"
Page
Discussion
View source
History
teams
Log in
 
Team:Grinnell CN/Notebook
From 2011.igem.org
Home
Team
Gallery
Project
Applications
Parts
DspB
Esp
RsaA
Notebook
Safety
Protocols
Recipes
Gels
DspB
Esp
Promoters
Attributions
Sponsors
Notebook
Figure 1: PCR products on
DNA gel.  Lane 1: ladder; Lane 2: rsaA from liquid culture Caulobacter; Lane 3: rsaA from plate culture Caulobacter; Lane 4: esp from S. epidermidis.
Figure 2: Gel results for transformation of ligations. Lane 1: ladder; Lane 2: digested plasmid with rsaA; Lane 3: digested plasmid with rsaA and esp; Lanes 4-8: digested plasmids from various colonies with esp.
Figure 3: lane 1: ladder; lanes 2-4: PCR product using VF2 and VR of plasmid containing esp ligated with rsaA that had been digested 20110606; lane 5: PCR product of plasmid containing esp; lanes 6-8: PCR product using VF2 and VR of plasmid containing esp ligated with rsaA PCR product from 20110605. Only lane 6 shows successful ligation of esp and rsaA.
Figure 4: lane 1: ladder; lane 2: PrsaA PCR product; lane 3: Pxyl PCR product
Figure 5: lane 1: ladder; lanes 2,3: PCR product of rsaA using new primers from chromosomal Caulobacter DNA; lane 4: PCR product of esp using new primers from chromosomal S. epidermidis DNA.
Week 1 May 31-June 5
PCR primers designed according to Biobrick specifications and ordered for rsaA C-terminal and WT esp. Because of the discrepancy in GC content between Staphylococcus epidermidis and Caulobacter, we also optimized the esp sequence for Caulobacter for later synthesis with a higher resulting GC content.
Prepared competent cells of E. coli Top10. ( Protocol )
Streaked plates of Caulobacter, Staphylococcus aureus, and S. epidermidis.
Template DNA for PCR prepared from colonies using GeneReleaser . ( Protocol )
Week 2 June 5-11
Performed PCR on esp and rsaA genes ( protocol ) and ran results on gel ( protocol ).  Results showed successful amplification ( Figure 1 ).
Performed a transformation ( protocol ) to test the efficiency of our competent cells make in Week 1.  Results were successful with about a 0.7 x 10^7 CFU/μg of pUC19.
Purified PCR product ( protocol ) and digested esp with EcoRI, SpeI, and PstI; rsaA with EcoRI, XbaI, and PstI; and pSB1C3 with EcoRI and PstI.  We then ligated esp, rsaA, and esp and rsaA to pSB1C3 and transformed into E. coli Top10. ( Protocol )
Inoculated liquid cultures with colonies from transformations.
Performed a Miniprep on the results from transformation, digested the plasmid and ran the results on a gel to check the success of our transformations ( Figure 2 ).
Transformed BBa_K081005 , a BioBrick part containing a gene for constitutive promotor and RBS, into E. coli Top10.
Week 3 June 12-18
We inoculated cultures of transformed BBa_K081005, then performed a Miniprep on the overnight cultures to obtain plasmid containing BBa_K081005. PCR amplified the miniprep product.  The product of this was cleaned and digested with Eco RI and SpeI to be used in later ligation with plasmid containing esp and rsaA.
Plasmid samples containing rsaA or esp were prepared and sent for sequencing. Results arrived on Friday showing that the coding sequences had been conserved.
Digested plasmid containing esp with SpeI and PstI and PCR product of rsaA with XbaI and PstI, then ligated these digests together and transformed into E. coli Top10.  Twelve colonies were picked from transformation plates and prepared streak plates and PCR template DNA by freeze-thaw ( protocol ).  PCR was conducted using plasmid primers VF2 and VR and the products were run on a gel, showing some success ( Figure 3 ).  The strain containing the successful ligation product was inoculated in chloramphenicol broth and the overnight culture was used for a miniprep of the plasmid.  The plasmid was then digested with Eco RI and XbaI and ligated with the BBa_K081005 fragment.  This was transformed into E. coli Top10 and successful colonies were streaked and used for colony PCR to see if the ligation was successful.
PCR amplified PrsaA , a constitutive promoter in Caulobacter and Pxyl , an inducible promoter in Caulobacter.  We purified our PCR product ( protocol ), but we lost our DNA in the process ( Figure ).  We re-PCRed the two promoters ( Figure 4 ) and performed an ethanol precipitation DNA clean-up ( protocol ) instead and received a low concentration of DNA yield.  This product was digested with Eco RI and Pst HF and ligated into pSB1C3 that had been previously digested with Eco RI and Pst HF.  The ligations were transformed into E. coli Top10.
After realizing that the combination of esp and rsaA created a stop codon between the two parts of the gene, we decided to start from scratch on the plasmids containing those genes and the plasmid containing them together.  We ordered new primers that solved this issue.
We performed PCR from chromosomal DNA again using the new primers and ran them on a gel, showing that our fragments had been amplified ( Figure 5 ).
Week 4 June 19-25
We started the process of recreating esp and rsaA C-terminal from template DNA and new primers that would not create an in-frame stop codon.  Template DNA from colonies was used for PCR successfully. PCR products were then digested for insertion into pSB1C3 individually and together as a three part ligation.  Colony PCR of the transformation products showed successful individual insertions, but unsuccessful three-part ligation.  Overnights and plates of successfully transformed strains were made.  Digestions of previously made PCR products and minipreps of esp and rsaA were made and used in ligations with each other and blank plasmids, pSB1C3 and pMR10, that had previously been digested.  Insertions of esp PCR product into rsaA C-terminal containing pSB1C3 were the most successful.
The transformation of promoters PrsaA and Pxyl were successful.  We picked 2 colonies from each transformed promotor, inoculated overnight cultures, and Miniprepped them.  We then performed a test digest but concluded that samples were contaminated by each other when we observed two bands in each lane corresponding to both sizes of promoters ( Figure ).
We made new overnights of PrsaA and Pxyl transformation colonies as well as a colony transformed with BBa_K081005, then Miniprepped and PCR amplified them this time using VF2 and VR so that the fragments would be large enough to see on a gel.  We also used One Taq instead of Phusion polymerase and our gel shows a lot of mispriming and extra bands. However, the PCR was sufficient to confirm the sizes of all three promoters ( Figure ). We sent PrsaA and Pxyl for sequencing and the results show successful isolation.
We digested the three promoters with SpeI and PstI to be ligated with esp plus rsaA and transformed.
This week is incomplete ( Figure , Figure , and Figure )
Week 5 June 26- July 2
Transformation plates of PrsaA and Pxyl with esp plus rsaA combination insert had a lot of growth, too many colonies to count and were unusually small.  We PCR amplified  with VF2 and VR but the gel showed that only the promotor region was present in the vector ( Figure and Figure ).
We decided to run some tests to see where we were going wrong.  We tested the restriction enzymes and the test showed that they work fine ( Figure ).  We plated competent cells without DNA on all plates with all antibiotics and plain L-Broth on plates.  We found that only our carb plates were contaminated with E. coli so we made new carb plates.  We also made new competent cells in case they were also contaminated.
We tried inserting the promotors into the vector containing esp plus rsaA rather than the other way around to see if we could get better results and got similar looking plates.  We also realized after looking at the gel ( Figure , Figure , and Figure ) that the size of one of the samples appeared be about 700bp, which is a size we would expect for rsaA alone.  We went back and examined the original gel image that led us to believe that it was both esp and rsaA together only to find that we had read the ladder wrong and we had in fact been working only with rsaA.
Week 6 July 3-9
We ran transformation tests to determine whether or not the method we were using to dry our plates prior to transformation plating was unnecessarily exposing them to contamination.  The results did not show any relation between the method of drying and the contamination.  What was suggested instead was that the stock chloramphenicol we had used in the previous batch of plates may be too weak to be entirely effective.  New chloramphenicol stock was made at 30mg/mL and new antibiotic plates were made using that.
Over the weekend, we gel extracted esp and rsaA in preparation for new ligation and transformation attempts.  The first round of tested transformants did not contain the desired ligation product, as shown by agarose gels .  A second round of tests on new colonies was started while we prepared new plasmid DNA for digests and gel extraction of both insert and target plasmid to reduce the likelihood of re-ligation.  The second round of tested colonies, however, showed one colony with the desired ligation product ( gel or gel ).
Overnights were made of the new esp + rsaA containing strain.  The DNA that was obtained from both PCR and gel extraction was later digested for ligation with promoter/RBS pieces.  Miniprep DNA was also prepared for sequencing to verify the sample.
We had optimized esp and dspB gene synthesized (G/C rich for Caulobacter) and transformed these two genes into E. coli Top10.
The BioBrick promotor was successfully added in front of the new esp+rsaA gene.
Wild Type dspB:
DspB gene arrived from the University of British Columbia 2010 iGEM Team in the form of a plasmid.  We will call this gene "WT dspB" This gene was transformed into E. coli Top10.  Transformation was successful with a lot of growth, more than usual.  But this is expected because this was a transformation of a purified plasmid rather than a ligation product which we are used to which makes  more growth understandable.
Overnights of 6 colonies of the transformed WT dspB were Miniprepped  and three colonies were digested with EcoRI and PstI to confirm size ( DspB Figure 1 ).  Once the size was confirmed, colony 3 was digested with XbaI and PstI and ligated and transformed into previously digested pSB1C3 plasmids containing the three promotors.
The transformations had a lot of small colonies, too many to count.  4 isolatable colonies from each transformation were prepared through freeze-thaw and PCR amplified to confirm sizes.  Sizes of four colonies were unclear because the promotors are so small that it is difficult to tell the difference between a band that contains only WT dspB and one that contains both WT dspB and a promotor ( DspB Figure 2 and the first four lanes of Esp Figure ). Therefore, a sample of WT dspB was PCR amplified as a standard for comparison and run on a gel against the ambiguous transformation results ( DspB Figure 3 ).  These gels confirmed the sizes of WT dspB + PrsaA and WT dspB + the BioBrick promotor BBa_K081005 but not WT dspB + Pxyl.
Four more colonies were freeze-thawed and PCR amplified and run on a gel against the standard WT dspB but none contained WT dspB + Pxyl ( DspB Figure 4 ).
Week 7 July 10-16
WT dspB: PrsaA + WT dspB and BBa_K081005 + WT dspB were miniprepped.  Will continue working on WT dspB when we recieve the sequencing results and determine whether we need to make new primers to eliminate the stop codon from the gene.  4 more colonies were picked from the transformation of WT dspB into Pxyl and two colonies showed the expected size ( Figure ).  This band was gel extracted because of the presence of another brighter band.  WT dspB + Pxyl was then digested with EcoRI and PstI and then ligated into pSB1C3 previously digested with EcoRI and PstI and then transformed into E. coli Top10.  The results of this transformation were confirmed on a gel ( Figure )
Optimized dspB: 4 colonies from the transformation of optimized dspB were PCR amplified and one colony showed the correct size ( Figure ).  An overnight culture of this colony was miniprepped, digested with SpeI and PstI and ligated with rsaA that had been previously gel extracted and digested with XbaI and PstI and then transformed.  4 colonies from the transformation plate were PCR amplified and 3 matched the expected size of optimized dspB + rsaA ( Figure ). Colony 2 was digested with XbaI and PstI and ligated into pSB1C3 containing each of the three promotors (PrsaA, Pxyl, and BBa_K081005) digested with SpeI and PstI and then transformed into E. coli Top10.  4 colonies from each transformation were picked, freeze-thawed, PCR amplified, and run on a gel.  Colony 3 for PrsaA + dspB + rsaA was successful ( Figure ) and colony 2 for BBa_K081005 + dspB + rsaA was successful ( Figure ). Both successful BBa_K081005 + dspB + rsaA and PrsaA + dspB + rsaA PCR products were PCR Purified; digested with EcoRI and PstI; ligated into pMR10 digested with EcoRI and PstI; and transformed. However, no colonies for Pxyl + dspB + rsaA showed the expected sizes, most of them contained only Pxyl ( Figure ).  4 more colonies ( Figure ) and then 6 more colonies  were picked, freeze-thawed and PCR amplified with no correct insert present ( Figure ).
Week 8 July 17-23
Optimized esp :
We ligated optimized esp with rsaA and two types of promotors: PrsaA and Pxyl. Later we transformed the ligate products into E.coli. Colony PCR showed that we had optimized esp gene alone in pSB1C3 plasmid, opt esp gene with rsaA gene in IDT-SMART Amp plasmid and opt esp behind two promotors in pSB1C3 plasmid. Overnights of E. coli with opt esp+PrsaA/Pxyl containing pSB1C3 plasmids were made and MiniPrep was done to the overnights so that rsaA gene (from gel extract) can be inserted behind optimized esp later. Additionally, optimized esp + rsaA combination were cut off from IDT-SMART plasmid and put into the standardizd pSB1C3 plasmid for sending to the registry as a BioBrick part. RsaA inserting into opt esp+PrsaA and opt esp+Pxyl containing plasmid were both proved by colony PCR to be successful. Later this week, Pxyl/PrsaA +optimized esp +rsaA were cut off from pSB1C3 and inserted into pMR10.
Optimized dspB:
Colonies from the transformation of Pxyl and BBa_K081005 with dspB optimized + rsaA in pMR10 were picked, freeze-thawed, and PCR amplified using M13F and M13R.  These were run on a gel ( Figure ).  PrsaA + dspB optimized + rsaA appears to have been transformed and BBa_K081005 + dspB optimized + rsaA may have been.  A conjugation was performed on PrsaA + dspB optimized + rsaA.  To check BBa_K081005 + dspB optimized + rsaA, the transformation product was Miniprepped and digested and run on a gel again but the digestion did not appear to have worked.  More colonies of BBa_K081005 + dspB optimized + rsaA were checked through colony PCR ( Figure ) and a miniprep of colony 1 was PCR amplified but showed the wrong size fragment ( Figure ). The transformation of Pxyl + dspB optimized + rsaA was redone with a fresh digest of Pxyl with SpeI and PstI from a new miniprep. 6 colonies from this transformation plate were picked, freeze-thawed, PCR amplified and run on a gel but showed no desired fragment ( Figure ).  8 more colonies were colony PCR amplified and run on a gel ( Figure ).  Colony 12 showed the correct band size and was digested with EcoRI and PstI and then transformed into pMR10.  Colonies from the transformation of Pxyl + dspB optimized + rsaA into pMR10 were colony PCR amplified and run on a gel ( Figure ).  Colony 3 appears to contain the right size fragment (around 1,800bp) and was therefore conjugated into Caulobacter.
Week 9 July 24-30
The insertions of Pxyl/PrsaA+optimized esp+rsaA into pMR10 were both successful ( Figure Pxyl ). In order to transfer the pMR10 plasmids into C. crescentus, two conjugations were performed.
Inserting optimized esp into BBa_K081005 plasmid was also proved to be right. We picked 8 colonies from from transformation cells and made overnight for each. The overnight culture was Miniprep to get the plasmid and the MiniPrep products were digested and then run on a gel. 6 colonies showed bright band just below 1kb, which was supposed to the right size for BBa_K081005+optimized esp. ( Figure )
Optimized dspB
More colonies of Bba_K081005 + dspB optimized + rsaA were PCR amplified and four colonies showed the expected size ( Figure ).  Colony 12 was then conjugated into Caulobacter.
Samples of Pxyl + dspB optimized + rsaA and PrsaA + dspB optimized + rsaA were prepared for running on a protein gel. This served as a pilot for future procedures in running protein gels.  However, the samples did not show up on the gel.
Retrieved from " http://2011.igem.org/Team:Grinnell_CN/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
