<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Grinnell_CN_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Grinnell_CN/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Grinnell_CN/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Grinnell_CN/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Grinnell_CN/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Grinnell_CN/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Grinnell CN/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="menu-bar"><UL><LI><A href="https://2011.igem.org/Team:Grinnell_CN">Home</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Gallery">Gallery</A></LI></UL><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Applications"><SPAN>Applications</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Parts">Parts</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Project/DspB">DspB</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Project/Esp">Esp</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Project/RsaA">RsaA</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Recipes">Recipes</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Notebook/Gels">Gels</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Notebook/Gels/DspB">DspB</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Notebook/Gels/Esp">Esp</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Notebook/Gels/Promoters">Promoters</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Attributions">Attributions</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell_CN/Attributions/Sponsors">Sponsors</A></LI></DIV><H1><SPAN class="mw-headline" id="Notebook">Notebook</SPAN></H1><P><UL class="figure"><LI>Figure 1: PCR products onDNA gel.  Lane 1: ladder; Lane 2: <I>rsaA</I> from liquid culture <I>Caulobacter</I>; Lane 3: <I>rsaA</I> from plate culture <I>Caulobacter</I>; Lane 4: <I>esp</I> from <I>S. epidermidis</I>.</LI><LI>Figure 2: Gel results for transformation of ligations. Lane 1: ladder; Lane 2: digested plasmid with <I>rsaA</I>; Lane 3: digested plasmid with <I>rsaA</I> and <I>esp</I>; Lanes 4-8: digested plasmids from various colonies with <I>esp</I>.</LI><LI>Figure 3: lane 1: ladder; lanes 2-4: PCR product using VF2 and VR of plasmid containing <I>esp</I> ligated with <I>rsaA</I> that had been digested 20110606; lane 5: PCR product of plasmid containing <I>esp</I>; lanes 6-8: PCR product using VF2 and VR of plasmid containing <I>esp</I> ligated with <I>rsaA</I> PCR product from 20110605. Only lane 6 shows successful ligation of <I>esp</I> and <I>rsaA</I>.</LI><LI>Figure 4: lane 1: ladder; lane 2: P<SUB>rsaA</SUB> PCR product; lane 3: P<SUB>xyl</SUB> PCR product</LI><LI>Figure 5: lane 1: ladder; lanes 2,3: PCR product of <I>rsaA</I> using new primers from chromosomal <I>Caulobacter</I> DNA; lane 4: PCR product of <I>esp</I> using new primers from chromosomal <I>S. epidermidis</I> DNA.</LI></UL></P><H3><SPAN class="mw-headline" id="Week_1_May_31-June_5">Week 1 May 31-June 5</SPAN></H3><P>PCR primers designed according to Biobrick specifications and ordered for <I>rsaA</I> C-terminal and WT <I>esp</I>. Because of the discrepancy in GC content between <I>Staphylococcus epidermidis</I> and <I>Caulobacter</I>, we also optimized the <I>esp</I> sequence for <I>Caulobacter</I> for later synthesis with a higher resulting GC content.
</P><P>Prepared competent cells of <I>E. coli</I> Top10. (<A href="/Team:Grinnell/Notebook/Protocols#Competent" title="Team:Grinnell/Notebook/Protocols">Protocol</A>)
</P><P>Streaked plates of <I>Caulobacter</I>, <I>Staphylococcus aureus</I>, and <I>S. epidermidis</I>.
</P><P>Template DNA for PCR prepared from colonies using <A href="http://www.bioventures.com/products.php?cid=10005|" class="external text" rel="nofollow">GeneReleaser</A>. (<A href="/Team:Grinnell/Notebook/Protocols#GeneReleaser" title="Team:Grinnell/Notebook/Protocols">Protocol</A>)
</P><H3><SPAN class="mw-headline" id="Week_2_June_5-11">Week 2 June 5-11</SPAN></H3><P>Performed PCR on esp and rsaA genes (<A href="/Team:Grinnell/Notebook/Protocols#cPCR" title="Team:Grinnell/Notebook/Protocols">protocol</A>) and ran results on gel (<A href="/Team:Grinnell/Notebook/Protocols#dGel" title="Team:Grinnell/Notebook/Protocols">protocol</A>).  Results showed successful amplification (<A href="https://2011.igem.org/File:20110606_PCRproduct.jpg">Figure 1</A>).
</P><P>Performed a transformation (<A href="/Team:Grinnell/Notebook/Protocols#Transformation" title="Team:Grinnell/Notebook/Protocols">protocol</A>) to test the efficiency of our competent cells make in Week 1.  Results were successful with about a 0.7 x 10^7 CFU/μg of pUC19.
</P><P>Purified PCR product (<A href="/Team:Grinnell/Notebook/Protocols#dClean" title="Team:Grinnell/Notebook/Protocols">protocol</A>) and <A href="/Protocols#Restriction_Digest" title="Protocols">digested</A><I>esp</I> with EcoRI, SpeI, and PstI; <I>rsaA</I> with EcoRI, XbaI, and PstI; and <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A> with EcoRI and PstI.  We then <A href="/Protocols#Ligation" title="Protocols">ligated</A><I>esp</I>, <I>rsaA</I>, and <I>esp</I> and <I>rsaA</I> to pSB1C3 and transformed into <I>E. coli</I> Top10. (<A href="/Team:Grinnell/Notebook/Protocols#Transformation" title="Team:Grinnell/Notebook/Protocols">Protocol</A>)
</P><P>Inoculated liquid cultures with colonies from transformations.
</P><P>Performed a <A href="/Team:Grinnell/Notebook/Protocols#MiniPrep" title="Team:Grinnell/Notebook/Protocols">Miniprep</A> on the results from transformation, digested the plasmid and ran the results on a gel to check the success of our transformations (<A href="https://2011.igem.org/File:20110610PlamidGel.jpg">Figure 2</A>).
</P><P>Transformed <A href="http://partsregistry.org/Part:BBa_K081005" class="external text" rel="nofollow">BBa_K081005</A>, a BioBrick part containing a gene for constitutive promotor and RBS, into <I>E. coli</I> Top10.
</P><H3><SPAN class="mw-headline" id="Week_3_June_12-18">Week 3 June 12-18</SPAN></H3><P>We inoculated cultures of transformed BBa_K081005, then performed a <A href="/Team:Grinnell/Notebook/Protocols#MiniPrep" title="Team:Grinnell/Notebook/Protocols">Miniprep</A> on the overnight cultures to obtain plasmid containing BBa_K081005. <A href="/Team:Grinnell/Notebook/Protocols#cPCR" title="Team:Grinnell/Notebook/Protocols">PCR</A> amplified the miniprep product.  The product of this was <A href="/Team:Grinnell/Notebook/Protocols#dClean" title="Team:Grinnell/Notebook/Protocols">cleaned</A> and <A href="/Protocols#Restriction_Digest" title="Protocols">digested</A> with Eco RI and SpeI to be used in later ligation with plasmid containing <I>esp</I> and <I>rsaA</I>.
</P><P>Plasmid samples containing <I>rsaA</I> or <I>esp</I> were prepared and sent for sequencing. Results arrived on Friday showing that the coding sequences had been conserved.
</P><P>Digested plasmid containing <I>esp</I> with SpeI and PstI and PCR product of <I>rsaA</I> with XbaI and PstI, then ligated these digests together and transformed into <I>E. coli</I> Top10.  Twelve colonies were picked from transformation plates and prepared streak plates and PCR template DNA by freeze-thaw (<A href="/Team:Grinnell/Notebook/Protocols#freezeThaw" title="Team:Grinnell/Notebook/Protocols">protocol</A>).  PCR was conducted using plasmid primers VF2 and VR and the products were run on a gel, showing some success (<A href="https://2011.igem.org/File:20110614_PCR135.jpg">Figure 3</A>).  The strain containing the successful ligation product was inoculated in chloramphenicol broth and the overnight culture was used for a <A href="/Team:Grinnell/Notebook/Protocols#MiniPrep" title="Team:Grinnell/Notebook/Protocols">miniprep</A> of the plasmid.  The plasmid was then <A href="/Protocols#Restriction_Digest" title="Protocols">digested</A> with Eco RI and XbaI and ligated with the BBa_K081005 fragment.  This was transformed into <I>E. coli</I> Top10 and successful colonies were streaked and used for colony PCR to see if the ligation was successful.
</P><P>PCR amplified <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K531000|" class="external text" rel="nofollow">P<SUB>rsaA</SUB></A>, a constitutive promoter in <I>Caulobacter</I> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K531002|" class="external text" rel="nofollow">P<SUB>xyl</SUB></A>, an inducible promoter in <I>Caulobacter</I>.  We purified our PCR product (<A href="/Team:Grinnell/Notebook/Protocols#dClean" title="Team:Grinnell/Notebook/Protocols">protocol</A>), but we lost our DNA in the process (<A href="https://2011.igem.org/File:20110614_Promoters.jpg">Figure</A>).  We re-PCRed the two promoters (<A href="https://2011.igem.org/File:20110614_PCR246.jpg">Figure 4</A>) and performed an ethanol precipitation DNA clean-up (<A href="/Team:Grinnell/Notebook/Protocols#dCleansmall" title="Team:Grinnell/Notebook/Protocols">protocol</A>) instead and received a low concentration of DNA yield.  This product was <A href="/Protocols#Restriction_Digest" title="Protocols">digested</A> with Eco RI and Pst HF and ligated into pSB1C3 that had been previously digested with Eco RI and Pst HF.  The ligations were transformed into <I>E. coli</I> Top10. 
</P><P>After realizing that the combination of <I>esp</I> and <I>rsaA</I> created a stop codon between the two parts of the gene, we decided to start from scratch on the plasmids containing those genes and the plasmid containing them together.  We ordered new primers that solved this issue. 
</P><P>We performed PCR from chromosomal DNA again using the new primers and ran them on a gel, showing that our fragments had been amplified (<A href="https://2011.igem.org/File:20110614_Promoters.jpg">Figure 5</A>).
</P><H3><SPAN class="mw-headline" id="Week_4_June_19-25">Week 4 June 19-25</SPAN></H3><P>We started the process of recreating <I>esp</I> and <I>rsaA</I> C-terminal from template DNA and new primers that would not create an in-frame stop codon.  Template DNA from colonies was used for PCR successfully. PCR products were then digested for insertion into pSB1C3 individually and together as a three part ligation.  Colony PCR of the transformation products showed successful individual insertions, but unsuccessful three-part ligation.  Overnights and plates of successfully transformed strains were made.  Digestions of previously made PCR products and minipreps of <I>esp</I> and <I>rsaA</I> were made and used in ligations with each other and blank plasmids, pSB1C3 and pMR10, that had previously been digested.  Insertions of <I>esp</I> PCR product into <I>rsaA</I> C-terminal containing pSB1C3 were the most successful.
</P><P>The transformation of promoters P<SUB>rsaA</SUB> and P<SUB>xyl</SUB> were successful.  We picked 2 colonies from each transformed promotor, inoculated overnight cultures, and Miniprepped them.  We then performed a <A href="/Protocols#Test_Digest" title="Protocols">test digest</A> but concluded that samples were contaminated by each other when we observed two bands in each lane corresponding to both sizes of promoters (<A href="https://2011.igem.org/File:20110620_promotorsTransformationGel.jpg">Figure</A>).  
</P><P>We made new overnights of P<SUB>rsaA</SUB> and P<SUB>xyl</SUB> transformation colonies as well as a colony transformed with BBa_K081005, then Miniprepped and PCR amplified them this time using VF2 and VR so that the fragments would be large enough to see on a gel.  We also used One Taq instead of Phusion polymerase and our gel shows a lot of mispriming and extra bands. However, the PCR was sufficient to confirm the sizes of all three promoters (<A href="https://2011.igem.org/File:20110622_PromotersGel2.jpg">Figure</A>). We sent P<SUB>rsaA</SUB> and P<SUB>xyl</SUB> for sequencing and the results show successful isolation.
</P><P>We digested the three promoters with SpeI and PstI to be ligated with <I>esp</I> plus <I>rsaA</I> and transformed.
</P><P>This week is incomplete (<A href="https://2011.igem.org/File:20110624_PrsaA_esp_rsaA.jpg">Figure</A>, <A href="https://2011.igem.org/File:20110624_Pxyl_esp_rsaA.jpg">Figure</A>, and <A href="https://2011.igem.org/File:20110624_BBa_K081005_esp_rsaA.jpg">Figure</A>)
</P><H3><SPAN class="mw-headline" id="Week_5_June_26-_July_2">Week 5 June 26- July 2</SPAN></H3><P>Transformation plates of P<SUB>rsaA</SUB> and P<SUB>xyl</SUB> with <I>esp</I> plus <I>rsaA</I> combination insert had a lot of growth, too many colonies to count and were unusually small.  We PCR amplified  with VF2 and VR but the gel showed that only the promotor region was present in the vector (<A href="https://2011.igem.org/File:20110627_esprsaA%2Bpromoters_1.jpg">Figure</A> and <A href="https://2011.igem.org/File:20110627_esprsaA%2Bpromoters_2.jpg">Figure</A>).
</P><P>We decided to run some tests to see where we were going wrong.  We tested the restriction enzymes and the test showed that they work fine (<A href="https://2011.igem.org/File:20110628_TestDigest.jpg">Figure</A>).  We plated competent cells without DNA on all plates with all antibiotics and plain L-Broth on plates.  We found that only our carb plates were contaminated with <I>E. coli</I> so we made new carb plates.  We also made new competent cells in case they were also contaminated.
</P><P>We tried inserting the promotors into the vector containing <I>esp</I> plus <I>rsaA</I> rather than the other way around to see if we could get better results and got similar looking plates.  We also realized after looking at the gel (<A href="https://2011.igem.org/File:20110629_PromoterInserts1.jpg">Figure</A>,<A href="https://2011.igem.org/File:20110629_PromoterInserts_PrsaA.jpg">Figure</A>, and <A href="https://2011.igem.org/File:20110629_PromoterInserts_Both.jpg">Figure</A>) that the size of one of the samples appeared be about 700bp, which is a size we would expect for <I>rsaA</I> alone.  We went back and examined the original gel image that led us to believe that it was both <I>esp</I> and <I>rsaA</I> together only to find that we had read the ladder wrong and we had in fact been working only with <I>rsaA</I>.
</P><H3><SPAN class="mw-headline" id="Week_6_July_3-9">Week 6 July 3-9</SPAN></H3><P>We ran transformation tests to determine whether or not the method we were using to dry our plates prior to transformation plating was unnecessarily exposing them to contamination.  The results did not show any relation between the method of drying and the contamination.  What was suggested instead was that the stock chloramphenicol we had used in the previous batch of plates may be too weak to be entirely effective.  New chloramphenicol stock was made at 30mg/mL and new antibiotic plates were made using that.
</P><P>Over the weekend, we gel extracted <I>esp</I> and <I>rsaA</I> in preparation for new ligation and transformation attempts.  The first round of tested transformants did not contain the desired ligation product, as shown by <A href="https://2011.igem.org/File:20110705_notCombo.jpg">agarose gels</A>.  A second round of tests on new colonies was started while we prepared new plasmid DNA for digests and gel extraction of both insert and target plasmid to reduce the likelihood of re-ligation.  The second round of tested colonies, however, showed one colony with the desired ligation product (<A href="https://2011.igem.org/Team:Grinnell/Notebook/Gels#20110706_MoreColoniesFollowup">gel</A> or <A href="https://2011.igem.org/File:20110706_MoreColoniesFollowup.jpg">gel</A>).
</P><P>Overnights were made of the new <I>esp</I> + <I>rsaA</I> containing strain.  The DNA that was obtained from both PCR and gel extraction was later digested for ligation with promoter/RBS pieces.  Miniprep DNA was also prepared for sequencing to verify the sample.
</P><P>We had optimized <I>esp</I> and <I>dspB</I> gene synthesized (G/C rich for Caulobacter) and transformed these two genes into E. coli Top10. 
</P><P>The BioBrick promotor was successfully added in front of the new <I>esp</I>+<I>rsaA</I> gene.
</P><P>Wild Type <I>dspB</I>:
</P><P><A href="/Team:Grinnell/Project/DspB" title="Team:Grinnell/Project/DspB"><I>DspB</I></A> gene arrived from the <A href="https://2010.igem.org/Team:British_Columbia" class="external text" rel="nofollow">University of British Columbia 2010 iGEM Team</A> in the form of a plasmid.  We will call this gene &quot;WT <I>dspB</I>&quot; This gene was transformed into <I>E. coli</I> Top10.  Transformation was successful with a lot of growth, more than usual.  But this is expected because this was a transformation of a purified plasmid rather than a ligation product which we are used to which makes  more growth understandable.
</P><P>Overnights of 6 colonies of the transformed WT <I>dspB</I> were Miniprepped  and three colonies were digested with EcoRI and PstI to confirm size (<A href="https://2011.igem.org/File:20110707_UBC_dspB_transformation_results_colonies_1_2_3.jpg">DspB Figure 1</A>).  Once the size was confirmed, colony 3 was digested with XbaI and PstI and ligated and transformed into previously digested pSB1C3 plasmids containing the three promotors.  
</P><P>The transformations had a lot of small colonies, too many to count.  4 isolatable colonies from each transformation were prepared through freeze-thaw and PCR amplified to confirm sizes.  Sizes of four colonies were unclear because the promotors are so small that it is difficult to tell the difference between a band that contains only WT <I>dspB</I> and one that contains both WT <I>dspB</I> and a promotor (<A href="https://2011.igem.org/File:20110708_dspB%2BPrsaA_Pxyl.jpg">DspB Figure 2</A> and the first four lanes of <A href="https://2011.igem.org/File:20110708_BBaGel.jpg">Esp Figure</A>). Therefore, a sample of WT dspB was PCR amplified as a standard for comparison and run on a gel against the ambiguous transformation results (<A href="https://2011.igem.org/File:20110708_dspB%2BpromotersVSdspB_std.jpg">DspB Figure 3</A>).  These gels confirmed the sizes of WT <I>dspB</I> + P<SUB>rsaA</SUB> and WT <I>dspB</I> + the BioBrick promotor BBa_K081005 but not WT <I>dspB</I> + P<SUB>xyl</SUB>.
</P><P>Four more colonies were freeze-thawed and PCR amplified and run on a gel against the standard WT <I>dspB</I> but none contained WT <I>dspB</I> + P<SUB>xyl</SUB> (<A href="https://2011.igem.org/File:20110709dspB%2Bpxyl_Colonies5-8.jpg">DspB Figure 4</A>).
</P><H3><SPAN class="mw-headline" id="Week_7_July_10-16">Week 7 July 10-16</SPAN></H3><P>WT <I>dspB</I>: PrsaA + WT <I>dspB</I> and BBa_K081005 + WT <I>dspB</I> were miniprepped.  Will continue working on WT <I>dspB</I> when we recieve the sequencing results and determine whether we need to make new primers to eliminate the stop codon from the gene.  4 more colonies were picked from the transformation of WT <I>dspB</I> into P<SUB>xyl</SUB> and two colonies showed the expected size (<A href="https://2011.igem.org/File:20110711_c9_10_12_15_from20110707transf-Pxyl%2BdspBWT.jpg">Figure</A>).  This band was gel extracted because of the presence of another brighter band.  WT <I>dspB</I> + P<SUB>xyl</SUB> was then digested with EcoRI and PstI and then ligated into pSB1C3 previously digested with EcoRI and PstI and then transformed into <I>E. coli</I> Top10.  The results of this transformation were confirmed on a gel (<A href="https://2011.igem.org/File:20110714_WTdspB%2BPxyl.jpg">Figure</A>)
</P><P>Optimized <I>dspB</I>: 4 colonies from the transformation of optimized <I>dspB</I> were PCR amplified and one colony showed the correct size (<A href="https://2011.igem.org/File:20110710_optGenes%2BC3.jpg">Figure</A>).  An overnight culture of this colony was miniprepped, digested with SpeI and PstI and ligated with <I>rsaA</I> that had been previously gel extracted and digested with XbaI and PstI and then transformed.  4 colonies from the transformation plate were PCR amplified and 3 matched the expected size of optimized <I>dspB</I> + <I>rsaA</I> (<A href="https://2011.igem.org/File:20110712_optdspB%2BrsaAc1-4_optdspB.jpg">Figure</A>). Colony 2 was digested with XbaI and PstI and ligated into pSB1C3 containing each of the three promotors (P<SUB>rsaA</SUB>, P<SUB>xyl</SUB>, and BBa_K081005) digested with SpeI and PstI and then transformed into <I>E. coli</I> Top10.  4 colonies from each transformation were picked, freeze-thawed, PCR amplified, and run on a gel.  Colony 3 for P<SUB>rsaA</SUB> + <I>dspB</I> + <I>rsaA</I> was successful (<A href="https://2011.igem.org/File:20110714_optdspB%2BrsaA%2BPrsaA_OR_Pxyl.jpg">Figure</A>) and colony 2 for BBa_K081005 + <I>dspB</I> + <I>rsaA</I> was successful (<A href="https://2011.igem.org/File:20110715_BBa%2BoptdspB%2BrsaA.jpg">Figure</A>). Both successful BBa_K081005 + <I>dspB</I> + <I>rsaA</I> and P<SUB>rsaA</SUB> + <I>dspB</I> + <I>rsaA</I> PCR products were PCR Purified; digested with EcoRI and PstI; ligated into pMR10 digested with EcoRI and PstI; and transformed. However, no colonies for P<SUB>xyl</SUB> + <I>dspB</I> + <I>rsaA</I> showed the expected sizes, most of them contained only P<SUB>xyl</SUB> (<A href="https://2011.igem.org/File:20110714_optdspB%2BrsaA%2BPrsaA_OR_Pxyl.jpg">Figure</A>).  4 more colonies (<A href="https://2011.igem.org/File:20110714_optdspB%2BrsaA%2BPxyl_c5-8.jpg">Figure</A>) and then 6 more colonies  were picked, freeze-thawed and PCR amplified with no correct insert present (<A href="https://2011.igem.org/File:20110715_Pxyl%2BoptdspB%2BrsaA_c9-14.jpg">Figure</A>).
</P><H3><SPAN class="mw-headline" id="Week_8_July_17-23">Week 8 July 17-23</SPAN></H3><P>Optimized <I>esp </I>:
</P><P>We ligated optimized esp with rsaA and two types of promotors: P<SUB>rsaA</SUB> and Pxyl. Later we transformed the ligate products into E.coli. Colony PCR showed that we had optimized esp gene alone in pSB1C3 plasmid, opt esp gene with rsaA gene in IDT-SMART Amp plasmid and opt esp behind two promotors in pSB1C3 plasmid. Overnights of E. coli with opt esp+PrsaA/Pxyl containing pSB1C3 plasmids were made and MiniPrep was done to the overnights so that rsaA gene (from gel extract) can be inserted behind optimized esp later. Additionally, optimized esp + rsaA combination were cut off from IDT-SMART plasmid and put into the standardizd pSB1C3 plasmid for sending to the registry as a BioBrick part. RsaA inserting into opt esp+PrsaA and opt esp+Pxyl containing plasmid
were both proved by colony PCR to be successful. Later this week, P<SUB>xyl</SUB>/P<SUB>rsaA</SUB> +optimized esp +rsaA were cut off from pSB1C3 and inserted into pMR10. 
</P><P>Optimized <I>dspB</I>: 
</P><P>Colonies from the transformation of Pxyl and BBa_K081005 with <I>dspB</I> optimized + <I>rsaA</I> in pMR10 were picked, freeze-thawed, and PCR amplified using M13F and M13R.  These were run on a gel (<A href="https://2011.igem.org/File:20110718_BBa_OR_PrsaA%2BoptdspB%2BrsaA_pMR10.jpg">Figure</A>).  P<SUB>rsaA</SUB> + <I>dspB</I> optimized + <I>rsaA</I> appears to have been transformed and BBa_K081005 + <I>dspB</I> optimized + <I>rsaA</I> may have been.  A <A href="/Team:Grinnell/Notebook/Protocols#Conjugation" title="Team:Grinnell/Notebook/Protocols">conjugation</A> was performed on P<SUB>rsaA</SUB> + <I>dspB</I> optimized + <I>rsaA</I>.  To check BBa_K081005 + <I>dspB</I> optimized + <I>rsaA</I>, the transformation product was Miniprepped and digested and run on a gel again but the digestion did not appear to have worked.  More colonies of BBa_K081005 + <I>dspB</I> optimized + <I>rsaA</I> were checked through colony PCR (<A href="https://2011.igem.org/File:20110722_BBa%2BodspB%2BrsaA%2BpMR10_c1,5-9.jpg">Figure</A>) and a miniprep of colony 1 was PCR amplified but showed the wrong size fragment (<A href="https://2011.igem.org/File:20110722_Pxyl%2BodspB%2BrsaA%2BpMR10.jpg">Figure</A>). The transformation of P<SUB>xyl</SUB> + <I>dspB</I> optimized + <I>rsaA</I> was redone with a fresh digest of P<SUB>xyl</SUB> with SpeI and PstI from a new miniprep. 6 colonies from this transformation plate were picked, freeze-thawed, PCR amplified and run on a gel but showed no desired fragment (<A href="https://2011.igem.org/File:20110719_Pxyl%2BodspB%2BrsaA_re-do_c1,3,4,5,6_std.jpg">Figure</A>).  8 more colonies were colony PCR amplified and run on a gel (<A href="https://2011.igem.org/File:20110720_Pxyl%2BodspB%2BrsaA_re-do_c7-14_WTdspB_c5.jpg">Figure</A>).  Colony 12 showed the correct band size and was digested with EcoRI and PstI and then transformed into pMR10.  Colonies from the transformation of P<SUB>xyl</SUB> + <I>dspB</I> optimized + <I>rsaA</I> into pMR10 were colony PCR amplified and run on a gel (<A href="https://2011.igem.org/File:20110722_Pxyl%2BodspB%2BrsaA%2BpMR10.jpg">Figure</A>).  Colony 3 appears to contain the right size fragment (around 1,800bp) and was therefore conjugated into <I>Caulobacter</I>.
</P><H3><SPAN class="mw-headline" id="Week_9_July_24-30">Week 9 July 24-30</SPAN></H3><P>The insertions of P<SUB>xyl</SUB>/P<SUB>rsaA</SUB>+optimized esp+rsaA into pMR10 were both successful (<A href="https://2011.igem.org/File:20110723_PrsaA%2Boesp%2BrsaA%2BpMR10.jpg">Figure Pxyl</A>). In order to transfer the pMR10 plasmids into <I>C. crescentus</I>, two conjugations were performed. 
</P><P>Inserting optimized esp into BBa_K081005 plasmid was also proved to be right. We picked 8 colonies from from transformation cells and made overnight for each. The overnight culture was Miniprep to get the plasmid and the MiniPrep products were digested and then run on a gel. 6 colonies showed bright band just below 1kb, which was supposed to the right size for BBa_K081005+optimized esp. (<A href="https://2011.igem.org/File:20110723_BBa%2Boesp.jpg">Figure</A>)
</P><P>Optimized <I>dspB</I></P><P>More colonies of Bba_K081005 + <I>dspB</I> optimized + <I>rsaA</I> were PCR amplified and four colonies showed the expected size (<A href="https://2011.igem.org/File:20110725_BBa%2BodspB%2BrsaA_c10-17.jpg">Figure</A>).  Colony 12 was then conjugated into <I>Caulobacter</I>.
</P><P>Samples of P<SUB>xyl</SUB> + <I>dspB</I> optimized + <I>rsaA</I> and P<SUB>rsaA</SUB> + <I>dspB</I> optimized + <I>rsaA</I> were prepared for running on a protein gel. This served as a pilot for future procedures in running protein gels.  However, the samples did not show up on the gel.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Grinnell_CN/Notebook">http://2011.igem.org/Team:Grinnell_CN/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Grinnell_CN/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Grinnell_CN/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Grinnell_CN/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Grinnell_CN/Notebook&amp;oldid=182070" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>