<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Grinnell_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Grinnell/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Grinnell/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Grinnell/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Grinnell/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Grinnell/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Grinnell/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>Grinnell Menubar</TITLE></P><DIV class="menu-bar"><UL><LI><A href="https://2011.igem.org/Team:Grinnell">Home</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Gallery">Gallery</A></LI></UL><LI><A href="https://2011.igem.org/Team:Grinnell/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Applications"><SPAN>Applications</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Parts">Parts</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Project/DspB">DspB</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Project/Esp">Esp</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Project/RsaA">RsaA</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Recipes">Recipes</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Notebook/Gels">Gels</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Notebook/Gels/DspB">DspB</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Notebook/Gels/Esp">Esp</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Notebook/Gels/Promoters">Promoters</A></LI><LI><A href="https://2011.igem.org/Team:Grinnell/Attributions">Attributions</A></LI></DIV><H1><SPAN class="mw-headline" id="Project">Project</SPAN></H1><H2><SPAN class="mw-headline" id="Project_Overview">Project Overview</SPAN></H2><P>Biofilms are cells encased in a hydrated extracellular polymeric substance (EPS) matrix that is composed of polysaccharides, proteins, nucleic acids, and lipids (Lu &amp; Collins 2007). Biofilms act as a protective umbrella for their inhabitants against various adverse conditions and can aid in communication between cells (Foley &amp; Gilbert 1996). Biofilms have recently become a concern in various fields, including health, food, and energy. The structure of biofilms make them difficult to remove once mature. By protecting the cells involved and facilitating horizontal gene transfer biofilms increase virulence of the incorporated bacteria.
</P><P>Synthetic biologists are beginning to tackle the problem of biofilms, as evidenced by the number of iGEM teams interested in the degradation and inhibition of biofilms in recent years.  These projects have been conducted using the workhorse of synthetic biology, <I>E. coli</I>, with a focus on finding ways to kill the bacteria in the biofilm before the biofilm is formed (inhibition) or by infiltrating the biofilm (degradation).  Our team approached this problem differently in two ways: we aimed to exploit the rigorous type I secretion pathway of <I>Caulobacter crescentus</I>, and we sought to degrade the EPS rather than kill the involved cells.
</P><P>We decided to utilize <I>Caulobacter</I> because it has many advatages over <I>E. coli</I> for our purposes.  The first of these is the robust TypeI secretion system that <I>Caulobacter</I> uses to secret its paracrystalline S-layer protein, RsaA, which makes up 10-12% of manufactured protein in lab strain CB15N (a strain which is deficient in producing a holdfast). <I>Caulobacter</I> is an aquatic bacterium and it grows well in low-nutrient environments.  Like <I>E. coli</I>, <I>Caulobacter</I> is gram-negative and has had its genome sequenced, however <I>Caulobacter</I> is safer for use around humans as it produces 100 times less endotoxin than <I>E. coli</I>, and is unable to survive in a human body.
To exploit the secretion pathway, we planned to attach the C-terminal secretion tag from RsaA to a biofilm inhibiting or degrading protein. This allows our system to produce and secrete large quantities of enzyme that are easy to isolate because there is no cell lysis that is necessary.
</P><P>For the biofilm degrading enzymes that we chose to have <I>Caulobacter</I> secrete, we focused our efforts on a serine protease, Esp (Iwase et al. 2010), from <I>Staphylococcus epidermidis</I>, and a hydrolase, DspB (Kaplan 2003), from <I>Aggregatibacter actinomycetemcomitans</I> that have both been shown to degrade biofilms.
</P><P>The general goals of our project were: 1) to introduce <I>Caulobacter</I> as another potential chassis for synthetic biology, especially in environmental and biomedical-related fields; 2) to create a toolbox of biobrick parts that enable easy exploitation of <I>Caulobacter's</I> typeI secretion system for any protein of interest through fusion to the C-terminal secretion tag; 3) and to develop a system for degrading biofilms by targeting the EPS.
</P><H2><SPAN class="mw-headline" id="Background"> Background </SPAN></H2><P><A class="titled" href="https://2011.igem.org/Team:Grinnell/Project/DspB">DspB</A><A class="titled" href="https://2011.igem.org/Team:Grinnell/Project/Esp">Esp</A></P><H3><SPAN class="mw-headline" id="Caulobacter_crecentus_and_Type_I_secretion"><I>Caulobacter crecentus</I> and Type I secretion</SPAN></H3><P><I>Caulobacter crescentus</I> is an aquatic gram-negative bacterium. It is widely used as a model organism for studying cell cycle, cell division and cell differentiation. <I>Caulobacter</I> divides asymmetrically to produce two morphologically different progeny, a swarmer and a stalked daughter cell (Laub et al. 2007). Only the stalked <I>Caulobacter</I> cells can initiate chromosome replication. The swarmer cells differentiate into stalked cells after a period of motility. The stalked cells produce an extremely strong polar adhesive called the holdfast (Toh et al. 2008) which can serve as a biofilm initiator. The <I>Caulobacter</I> strain we used (CB15N) is deficient in the gene that is responsible for producing the holdfast.  
</P><P>In the wild, <I>Caulobacter</I> is found in freshwater lakes and streams, where low nutrient conditions are common. However, <I>Caulobacter</I> cells are well equipped with various environmental sensors and transporters so that they are able to survive. For example, <I>Caulobacter</I> has many more TonB-dependent receptors than most bacteria, which are accountable for gathering carbohydrates from a variety of sources (Ryan et. al,  2010). <I>Caulobacter</I> is even capable of halting cell cycle progression in extreme dilute aquatic environment (Laub et al. 2007).
</P><P>One feature of <I>Caulobacter</I> that is often overlooked is its surface layer. The S-layer of <I>Caulobacter</I> is composed of a single protein—RsaA, which is secreted and assembled into a hexagonal crystalline array that covers the organism. RsaA provides <I>Caulobacter</I> a measure of protection for the cell from attacking agents such as proteases, viruses and parasitic bacteria (Nomellini et al. 2004). RsaA makes up approximately 10-12% of the total cell protein production. The synthesis of RsaA occurs without need for induction and the protein is produced continuously throughout the life cycle (Nomellini et al. 2004). The secretion system for RsaA is also well studied: secretion is ATP-driven and considered a type I secretion pathway given the fact that the C-terminal secretion signal is not cleaved off after secretion (Awram &amp; Smit 1998). The secretion signal has been shown to be within the C-terminal 82 amino acids of RsaA (Bingle, Nomellini and Smit 2000), and proteins as large as RsaA (98kDa, 1025 amino acids) (Gilchrist, Fisher and Smit 1992) can pass through the cell membrane successfully. 
</P><P>Thus, the reasons we chose <I>Caulobacter</I> as our chassis organism can be summarized as follows: 
	<I>Caulobacters</I> secrete recombinant proteins using the Type I secretion system, not the General Secretory Pathway (GSP), which is more common. The bacteria are obligate aerobes and grow to high densities in minimal media, up to OD 25-30. They do not secret other proteins and have a lipopolysaccharide (LPS) that has surprisingly low endotoxin potential. They are easily manipulated in lab and have a sequenced genome. And the yields are usually high for heterologous protein secretion (Nomellini et al. 2004).
</P><H3><SPAN class="mw-headline" id="Biofilms">Biofilms</SPAN></H3><P>Biofilms are a unique community of usually heterogeneous microorganisms that readily attach to rough and hydrophobic surfaces1. They are composed of cells encased in a hydrated extracellular polymeric substance (EPS) matrix that is composed of polysaccharides, proteins, nucleic acids, and lipids(Lu &amp; Collins 2007). 
</P><P>Due to its extracellular structure, a biofilm can act as a protective barrier for associated cells against various adverse conditions and can aid in the communication between cells(Foley &amp; Gilbert 1996). Cells within a biofilm are frequently found to be more resistant to antimicrobials compared to planktonic cells (Simões &amp; Vieira 2009). Moreover, the EPS serves as a bridge between cells, promoting intercellular communication that may facilitate cells’ (or the biofilm's as a whole) adaptation to changing environment (Davis et al. 1998). 
</P><P>These traits make the elimination of biofilms a challenging concern in a wide range of fields, especially in the food, energy, and biomedical industries1. Pathogens such as <I>E. coli</I> O157:H7 and <I>Staphylococcus aureus</I> will contaminate food products, microorganisms in biofilms will catalyze chemical and biological reactions causing metal corrosion in pipelines and tanks, and biofilms reduce heat transfer efficacy if it becomes too thick1 (Mittelman, 1998).
</P><H3><SPAN class="mw-headline" id="The_Experiments"> The Experiments </SPAN></H3><H4><SPAN class="mw-headline" id="Constructs">Constructs</SPAN></H4><P>Since <I>Caulobacter crescentus</I> has a G/C rich genome (over 60%) and both <I>esp</I> and <I>dspB</I> coding regions are G/C poor, we had the sequences codon optimized for expression in <I>Caulobacter</I>. 
</P><P>We were also interested to see the difference of gene expression in <I>Caulobacter</I> between the wild type and optimized genes, and we wanted to make the wild type <I>esp</I> gene available for other synthetic biologists to use. Therefore, we acquired the wild type <I>esp</I> gene from <I>Staphylococcus epidermidis</I> by PCR.  We obtained our wild type <I>dspB</I> gene from the University of British Columbia. 
</P><P>The promoters we used were P<SUB>xyl</SUB>, P<SUB>rsaA</SUB> and BBa_K081005, all functioning at a relatively high efficiency in both <I>E. coli</I> and <I>Caulobacter</I>. P<SUB>xyl</SUB> is a xylose-inducible promoter that originates from <I>Caulobacter</I>; P<SUB>rsaA</SUB> is a strong constitutive promoter also native to <I>Caulobacter</I> that is responsible for initiating transcription of <I>rsaA</I>; BBa_K081005 is a strong constitutive promoter that was designed by the 2008 iGEM team from the University of Pavia. 
</P><P>We knew from previously published reports that as few as 80 amino acids of the C-terminal end of RsaA function as a secretion tag for heterologous proteins. However, a larger secretion signal can increase the efficiency of secreting heterologous proteins, so we cloned the C-terminal 120 amino acids from <I>Caulobacter</I> strain CB15N. 
We engineered the following seven constructs:
<TABLE><TBODY><TR><TD>P<SUB>rsaA</SUB></TD><TD>P<SUB>xyl</SUB></TD><TD>BBa_K081005</TD></TR><TR><TD><I>dspB</I> (optimized)</TD><TD>✔</TD><TD>✔</TD><TD>✔</TD></TR><TR><TD><I>esp</I> (optimized)</TD><TD>✔</TD><TD>✔</TD><TD>✔</TD></TR><TR><TD><I>esp</I> (WT)</TD><TD>✖</TD><TD>✖</TD><TD>✔</TD></TR></TBODY></TABLE></P><P>All of the initial cloning was done in <I>E. coli</I> and after each construct was completed, we transferred the biobrick part from pSB1C3 plasmid to pMR10, which replicates in <I>Caulobacter crescentus</I> and <I>E. coli</I> at a low copy number. The pMR10 constructs were transferred into <I>Caulobacter</I> by conjugation.  
</P><P>One issue that gave us a hard time in the beginning of our project was that when we made the <I>esp</I> or <I>dspB</I> fusion with the rsaA C-terminal secretion signal using the standard biobrick prefix and suffix a stop codon was formed in the scar between the two genes that terminated the expression before the RsaA C-terminal secretion signal. Later, we solved the issue by modifying the suffix of the upstream gene and the prefix of the downstream gene so that the stop codon would no longer be in frame.
</P><H2><SPAN class="mw-headline" id="Data_.26_Results"> Data &amp; Results </SPAN></H2><P>The following results show that most of our chimeric proteins were expressed and secreted from <I>Caulobacter</I> and had significantly greater activity against <I>Staphylococcus aureus</I> biofilms than the WT <I>Caulobacter</I> strain. 
</P><P>Figure 1. Expression and secretion of chimeric proteins from <I>Caulobacter</I>. We cultured each of our <I>Caulobacter</I> secretion strains and collected the supernatants from equal numbers of cells. The proteins in the supernatants were precipitated with TCA, run on a 6-18% polyacrylamide gel, and silver stained. Results show that proteins of the expected size were secreted from P<SUB>rsaA</SUB> with chimeric Esp/RsaA C-terminal construct. However, even though we used gradient gel and silver staining, bands did not appear easily: only those constructs that presumably had a high yield of heterologous protein had bands corresponding to the desired biofilm inhibitor protein. 
</P><P>Figure 2. Expression and secretion of chimeric proteins from <I>Caulobacter</I>. We cultured each of our <I>Caulobacter</I> secretion strains and collected the supernatants from equal numbers of cells. The proteins in the supernatants were precipitated with TCA, run on a 6-18% polyacrylamide gel, and silver stained. Results show that proteins of the expected size were secreted from the following strains: both DspB and Esp constructs with P<SUB>xyl</SUB> inducible promoter and chimeric Esp with promoter <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K081005" class="external text" rel="nofollow">BBa_K081005</A>.
</P><P>Figure 3. Relative S. aureus biofilm growth after treatment with different engineered <I>Caulobacter</I> strains (three letters abbreviation for constructs: B for BBa_K081005, R for PrsaA, X for Pxyl, O for optimized, W for wild type, E for Esp,  D for DspB, and CC for WT <I>Caulobacter crescentus</I>). Relative biofilm growth is calculated by subtracting the ODmean of S. aureus only treatment from the OD of each well. 
</P><P>Compared to <I>S. aureus</I> only treatment, most of the other treatments had significantly lower biofilm growth (p&lt;0.001) except XOE construct (p=0.743) and wild type <I>Caulobacter</I> (p=0.085) treatments show no significant differences from the <I>S. aureus</I> treatment. Notably, the ROD construct induced <I>S. aureus</I> biofilm formation to some extent, which is worth further investigation.  The failure of XOE is explainable: xylose, the inducer of the Pxyl promoter of this construct, is usually consumed by the cells after 4h but we incubated cell cultures with xylose for more than 24 h. Therefore, Esp was not likely being produced during the biofilm inhibiting step. 
</P><P>The inhibition effect on biofilm growth of different proteins, the efficiency of different promoters and the expression level of wild type Esp and optimized Esp in <I>Caulobacter</I> were also compared among constructs.
</P><P>Figure 4. Here the standard group was composed of <I>S. aureus</I> only and WT groups. Both DspB and Esp can inhibit the biofilm growth. Though not statistically significant, the biofilm assay data showed a general trend that Esp, which is known to target <I>S. aureus</I> biofilm specifically, has a better inhibition effect than DspB, which is generally targeting most biofilms.
</P><P>Figure 5. All promoters were working great in <I>Caulobacter</I>. No significant difference among three promoters were shown.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Grinnell/Project">http://2011.igem.org/Team:Grinnell/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Grinnell/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Grinnell/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Grinnell/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Grinnell/Project&amp;oldid=203970" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>