<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Braunschweig_Notebook-content"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Braunschweig/Notebook-content">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Braunschweig/Notebook-content&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Braunschweig/Notebook-content&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Braunschweig/Notebook-content&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Braunschweig/Notebook-content" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Braunschweig/Notebook-content</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>E. cowli - Fighting Climate Change - iGEM 2014 Team Braunschweig</TITLE></P><DIV class="templatemo-top-menu"><DIV class="container"><DIV class="navbar navbar-default" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" id="templatemo-nav-bar"><UL id="nav" class="nav navbar-nav navbar-right" style="margin-top: 5px;white-space:nowrap;overflow:hidden"><LI id="navbarhome"><A href="https://2014.igem.org/Team:Braunschweig">Home</A></LI><LI id="navbarteam"><A href="https://2014.igem.org/Team:Braunschweig/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Team#profiles">Profiles</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Team-content#Collaborations">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Gallery">Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Team-content#Attributions">Attributions</A></LI><LI><A target="_blank" href="https://igem.org/Team.cgi?year=2014&amp;team_name=Braunschweig">Official Team Profile</A></LI></UL><LI id="navbarproject"><A href="https://2014.igem.org/Team:Braunschweig/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Project">Summary</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Project-content#Problem">Problem</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Project-content#Idea">Idea</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Project-content#Approach">Approach</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Project-content#Potential Impact">Impact</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Project-content#Engineering Principles">Engineering Principles</A></LI><LI id="navbarparts"><A href="https://2014.igem.org/Team:Braunschweig/HP">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/HP#spreading">Spreading the Word</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/HP#teaching">Teaching</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/HP#opinions">Opinions</A></LI><LI id="navbarmodeling"><A href="https://2014.igem.org/Team:Braunschweig/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Modeling-content#Modeling Approach">Modeling Approach</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Modeling-content#MM">Mathematical Model</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Modeling-content#Code">Code</A></LI><LI id="navbarnotebook"><A href="https://2014.igem.org/Team:Braunschweig/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Notebook-content#experiments">Experiments</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Notebook-content#protocols">Protocols</A></LI><LI id="navbarachievements"><A href="https://2014.igem.org/Team:Braunschweig/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Achievements-content#parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Braunschweig/Achievements-content#mr">Medal Requirements</A></LI><LI><A target="_blank" href="https://igem.org/2014_Judging_Form?id=1390">Judging</A></LI><LI id="navbarsafety"><A href="https://2014.igem.org/Team:Braunschweig/Safety">Safety</A></LI></DIV></DIV></DIV></DIV></DIV><DIV class="templatemo-welcome" id="templatemo-welcome"><DIV class="container"><DIV class="templatemo-slogan"><DIV class="container section"><DIV class="subpage"><DIV class="row"><H1>Experiments</H1><H2>1. Obtaining the sMMO genes</H2><P>
                        The very first step of our lab work was to obtain all genes encoding  the six subunits of the methane monooxygenase (mmoX, mmoY, mmoB, mmoZ, mmoD, mmoC) which catalyzes the conversion of the greenhouse gas methane to methanol, a natural intermediate of the cellular metabolism. Unfortunately, even though most of the MMO subunits  are described in the iGEM Registry, most of them were not available in stock. Therefore, we isolated them directly from <I>Methylococcus capsulatus</I> str. Bath via PCR using appropriate self-designed primers. However, the sequence of the initial subunits mmoC, mmoX, mmoY and mmoZ contained various restriction sites within the genes (mostly PstI). In order to receive the RFC10 standard  these were removed by using mutation PCR. We then cloned them into  the pSB1C3 shipping vector to put them at disposal of  the community of all future iGEM teams. 
                      For detailed information, take a look at our Lab notebook 	→<A href="https://static.igem.org/mediawiki/2014/f/fa/E._1_Isolation_of_sMMO_genes_from_M._capsulatus.pdf">E. 1 Isolation of sMMO genes from <I>M. capsulatus</I></A>.

                    </P></DIV><DIV class="row"><H2>2. His-tagging</H2><P>
                		Now that we had successfully isolated the genes for all the MMO subunits,  our next goal was to proof that we were able to produce all of them in the well characterized and easily manageable model organism <I>Escherichia coli</I>. In order to check for expression later on,  we added a His-tag  at the C-terminus of each subunit  by using appropriate PCR-primers. Afterwards, we put them under control of the inducible lac-promoter (R0011), a weak ribosome binding site (B0032) and a double terminator (B0015).
                      For detailed information, take a look at our Lab notebook 	→<A href="https://static.igem.org/mediawiki/2014/4/46/E._2_HIS-Tagging_of_mmo_genes.pdf">E. 2 HIS-Tagging of mmo genes</A>.
                	</P><DIV class="fig-caption-center"><UL><DIV class="ccaption"><LI>
Scheme of our His-tagged constructs flanked by a weak RBS and a double terminator under control of the inducible LacI+pL promoter.
</LI></DIV></UL></DIV></DIV><DIV class="row"><H2>3. Expression of His-construct</H2><P>
      Plasmids containing the  final His-tagged constructs were then transformed into chemo-competent <I>E. coli</I> cells and the single tagged subunits were expressed by induction with IPTG. To isolate the soluble and inclusion protein fractions,  the cells were harvested and pelleted by centrifugation.
SDS-PAGE followed by an immunostain via His-tag was performed in order to verify the expression of  the subunits. Unfortunately, only two subunits were detectable (MMOC and MMOX). 
For the purpose of producing all of the MMO subunits efficiently we tested the coexpression of different combinations of known chaperone proteins. With their help the <I>E. coli</I> cells were able to express each of the six MMO subunits in soluble form. Hence, we are the first to efficiently produce all of these MMO subunits in the heterologous organism <I>E.coli</I>. 
                      For detailed information, take a look at our Lab notebook 	→<A href="https://static.igem.org/mediawiki/2014/1/1e/E._4_Expression_of_HIS-constructs.pdf">E. 5 Expression of HIS-constructs</A>.

                	</P></DIV><DIV class="row"><H2>4. Assembly of the sMMO gene construct</H2><P>
                	After ensuring that all the subunits of the MMO were producible in <I>E. coli</I> using the right combination of chaperones, we decided to assemble all of them in one construct, thus creating our methane degrading <I>E. cowli</I>! To allow a subsequent induction of our final construct containing all six of the MMO subunit genes we again used the lac-promotor in combination with a weak RBS and a double terminator. To make sure that every subunit-encoding gene is translated, a single RBS is inserted upstream of every gene.
                	</P><DIV class="fig-caption-center"><UL><DIV class="ccaption"><LI>Composition of the MMO construct consisting of a combination of all six sMMO subunits and a weak RBS under control of the inducible LacI+pL promoter.
</LI></DIV></UL></DIV><P>
	In order to allow a subsequent induction of our final construct containing all six of the MMO subunit genes we again used the lac-promotor in combination with a weak RBS and a double terminator. In order to make sure that every subunit-encoding gene is translated, a single RBS is inserted upstream of every gene. 
                      For detailed information, take a look at our Lab notebook 	→<A href="https://static.igem.org/mediawiki/2014/f/f3/E._3_Cloning_of_final_construct.pdf">E. 3 Cloning of final construct</A>.

</P><H2>5. Expression of the sMMO protein complex</H2><P>
To analyse the functionality of the final construct we cultivated our <I>E. cowli</I> in a converted anaerobic jar filled with a 2% methane atmosphere for 6 h. In order to proof the activity of the enzyme complex we measured the decrease of the methane concentration over a time period of 3 h with an MQ-4 semiconductor sensor for natural gas, a kind gift from iGEM Team Aachen.
                      For detailed information, take a look at our Lab notebook 	→<A href="https://static.igem.org/mediawiki/2014/f/f0/E._4_Expression_of_mmo_construct.pdf">E. 5 Expression of mmo construct</A>.
                </P></DIV><DIV class="row"><H2>6. Preparation of competent cells with chaperones</H2><P>
                      For our coexpression experiments, we used different combination of chaperones. Therefore, we  created a stock of competent <I>E. coli</I> cells, that were made chemically competent after transformation with the different chaperone constructs.
                      For detailed information, take a look at our Lab notebook 	→<A href="https://static.igem.org/mediawiki/2014/3/3e/E._6_Preparing_of_competent_cells_with_chaperones.pdf">E. 6 Preparing of competent cells with chaperones</A>.
                      
                	</P></DIV><DIV class="row"><H2>7. Rumen fluid experiments</H2><P>
                To test our constructs under realistic conditions, we performed various rumen fluid experiments. Fresh rumen fluid was kindly provided by the Institute of Animal Nutrition, Friedrich-Loeffler-Institute (FLI), Federal Research Institute for Animal Health, Braunschweig and sterile filtrated. We then used this sterile fluid to analyse our <I>E. cowli</I>'s chances in the cow’s rumen.
                For detailed information, take a look at our Lab notebook →<A href="https://static.igem.org/mediawiki/2014/c/c7/E._7_Rumen_fluid_experiments.pdf">E. 7 Rumen fluid experiments</A></P></DIV><DIV class="row"><H2>8. Entrapping of <I>E. coli</I></H2><P>
                		Based on our first rumen fluid experiments, we have learnt that <I>E. coli</I> is not capable to grow on rumen fluid. To improve the chances of survivial of our <I>E. cowli</I>, as well as, to find a medium to transport our <I>E. cowli</I> into the cow's rumen we tested the impact of entrapment of <I>E. cowli</I> in alginate beads and analysed its growth kinetics.
                      For detailed information, take a look at our Lab notebook 	→<A href="https://static.igem.org/mediawiki/2014/4/4a/E._8_Entrapping_of_E._coli.pd">E. 8 Entrapping of <I>E. coli</I></A>.
                	</P></DIV><DIV class="row"><H2>9. Interlab Measurement Study</H2><P>
                		In this year’s iGEM competition, the first international Interlab Measurement Study was introduced to  obtain fluorescence data for three specific genetic devices expressing GFP. Even though we did not sign up for the track “Measurement” of the iGEM competition and therefore were not obliged to participate in this Interlab Measurement Study, we voluntarily took part as one of 45 teams worldwide. In addition to the general requirements we also completed the second “extra credit” assignment by building and testing all three devices expressing RFP instead of GFP and compared the fluorescence of both fluorescence proteins.
                      For detailed information, take a look at our Lab notebook 	→<A href="https://static.igem.org/mediawiki/2014/7/75/Interlab_Study.pdf">Interlab Study</A>.
                	</P></DIV></DIV></DIV><DIV class="container section"><DIV class="subpage" style="border-bottom:0px;"><DIV class="row"><H1>Protocols</H1><H3>Recipes for Solutions and Media</H3><TABLE class="protocol_table"><CAPTION>
				2x YT medium</CAPTION><TBODY><TR><TD>
						16 g</TD><TD>
						Bacto tryptone</TD></TR><TR><TD>
						10 g</TD><TD>
						Bacto yeast extract</TD></TR><TR><TD>
						5 g</TD><TD>
						NaCl</TD></TR></TBODY></TABLE><P>
			were dissolved in 1 L dH<SUB>2</SUB>O, the pH was adjusted to 7.0.
			For solid medium 12 g of Bacto agar (per Liter) was added.</P><TABLE class="protocol_table"><CAPTION>
				LB medium</CAPTION><TBODY><TR><TD>
						10 g</TD><TD>
						Bacto tryptone</TD></TR><TR><TD>
						5 g</TD><TD>
						Bacto yeast extract</TD></TR><TR><TD>
						10 g</TD><TD>
						NaCl</TD></TR></TBODY></TABLE><P>
			were dissolved in 1 L dH<SUB>2</SUB>O, the pH was adjusted to 7.0.
			For solid medium 12 g of Bacto agar (per Liter) was added.</P><TABLE class="protocol_table"><CAPTION>
				PBS</CAPTION><TBODY><TR><TD>
						0,8 % (w/v)</TD><TD>
						NaCl</TD></TR><TR><TD>
						0,02 % (w/v)</TD><TD>
						KCl</TD></TR><TR><TD>
						0,144 % (w/v)</TD><TD>
						Na<SUB>2</SUB>HPO<SUB>4</SUB>· 2 H<SUB>2</SUB>O</TD></TR><TR><TD>
						0,024 % (w/v)</TD><TD>
						KH<SUB>2</SUB>HPO<SUB>4</SUB></TD></TR></TBODY></TABLE><TABLE class="protocol_table"><CAPTION>
				NMS medium</CAPTION><TBODY><TR><TD>
						1 g</TD><TD>
						MgSO<SUB>4 </SUB>· 7 H<SUB>2</SUB>O</TD></TR><TR><TD>
						0.2 g</TD><TD>
						CaCl<SUB>2 </SUB>· 6 H<SUB>2</SUB>O</TD></TR><TR><TD>
						2 mL</TD><TD>
						Chelated Iron Solution</TD></TR><TR><TD>
						1 g</TD><TD>
						KNO<SUB>3</SUB></TD></TR><TR><TD>
						0.5 mL</TD><TD>
						Trace Element Solution</TD></TR><TR><TD>
						0.272 g</TD><TD>
						KH<SUB>2</SUB>PO<SUB>4</SUB></TD></TR><TR><TD>
						0.717 g</TD><TD>
						Na<SUB>2</SUB>HPO<SUB>4 </SUB>· 12 H<SUB>2</SUB>O</TD></TR><TR><TD>
						12.5 g</TD><TD>
						Purified Agar</TD></TR><TR><TD>
						1 L</TD><TD>
						Distilled deionized water</TD></TR></TBODY></TABLE><P>
			Adjust pH to 6.8.</P><TABLE class="protocol_table"><CAPTION>Chelated Iron Solution</CAPTION><TBODY><TR><TD>
						0.1 g</TD><TD>
						Ferric (III) ammonium citrate</TD></TR><TR><TD>
						0.2 g</TD><TD>
						EDTA, sodium salt</TD></TR><TR><TD>
						0.3 mL</TD><TD>
						HCl (concentrated)</TD></TR><TR><TD>
						100 mL</TD><TD>
						distilled deionizied water</TD></TR></TBODY></TABLE><TABLE class="protocol_table"><CAPTION>
				Trace Element Solution</CAPTION><TBODY><TR><TD>
						500 mg</TD><TD>
						EDTA</TD></TR><TR><TD>
						200 mg</TD><TD>
						FeSO<SUB>4</SUB>· H<SUB>2</SUB>O</TD></TR><TR><TD>
						10 mg</TD><TD>
						ZnSO<SUB>4 </SUB>· 7 H<SUB>2</SUB>O</TD></TR><TR><TD>
						3 mg</TD><TD>
						MnCl<SUB>2</SUB>· 4 H<SUB>2</SUB>O</TD></TR><TR><TD>
						30 mg</TD><TD>
						H<SUB>3</SUB>BO<SUB>3</SUB></TD></TR><TR><TD>
						20 mg</TD><TD>
						CoCl<SUB>2</SUB>· 6 H<SUB>2</SUB>O</TD></TR><TR><TD>
						1 mg</TD><TD>
						CoCl<SUB>2</SUB>· 6 H<SUB>2</SUB>O</TD></TR><TR><TD>
						2 mg</TD><TD>
						NiCl<SUB>2</SUB>· 6 H<SUB>2</SUB>O</TD></TR><TR><TD>
						3 mg</TD><TD>
						Na<SUB>2</SUB>MoO<SUB>4 </SUB>· 2 H<SUB>2</SUB>O</TD></TR><TR><TD>
						1 L</TD><TD>
						Distilled water</TD></TR></TBODY></TABLE><TABLE class="protocol_table"><CAPTION>
				50x TAE Buffer stock solution</CAPTION><TBODY><TR><TD>
						242 g</TD><TD>
						2 M Tris base</TD></TR><TR><TD>
						51,1 mL</TD><TD>
						glacial acetic acid (17,4 M)</TD></TR><TR><TD>
						200 mL</TD><TD>
						100 mM EDTA (pH 8)</TD></TR></TBODY></TABLE><P>
			were dissolved in 1 L dH<SUB>2</SUB>O.</P><P><STRONG>Loading buffer for gel electrophoresis</STRONG>
			A 0.2 M EDTA and 0.05 % (w/v) orange G or bromphenoleblue solution was prepared in 50 % glycerine.<STRONG>Ampicillin stock solution</STRONG>
			10 g ampicillin sodium salt was dissolved in 100 mL dH<SUB>2</SUB>O, sterile filtered, aliquoted and stored at -20 °C. </P><P><STRONG>Kanamycin stock solution</STRONG></P><P>
			5 g kanamycin sulfate was dissolved in 100 mL dH<SUB>2</SUB>O, sterile filtered, aliquoted and stored at -20 °C.<STRONG>Chloramphenicol stock solution</STRONG>
			3.4 g chloramphenicol was dissolved in 100 mL Ethanol, sterile filtered, aliquoted and stored at -20 °C.</P><P><STRONG>Tetracyclin stock solution</STRONG>
			5 g Tetracyclin was dissolved in 100 mL Ethanol, sterile filtered aliqoted and stored at -20 °C. </P><TABLE class="protocol_table"><CAPTION>
				TFB 1, pH 5.7 for preparation of competent cells</CAPTION><TBODY><TR><TD>
						0.3 g</TD><TD>
						CaCl<SUB>2</SUB></TD></TR><TR><TD>
						0.6 g</TD><TD>
						potassiumacetate</TD></TR><TR><TD>
						1.2 g</TD><TD>
						RbCl</TD></TR><TR><TD>
						1 g</TD><TD>
						MnCl<SUB>2 </SUB>· 4H<SUB>2</SUB>O</TD></TR><TR><TD>
						15 ml</TD><TD>
						glycerin</TD></TR></TBODY></TABLE><P>
			were dissolved in dH<SUB>2</SUB>O and diluted to a final volume of 100 mL. The pH was adjusted and the solution sterile filtered.</P><TABLE class="protocol_table"><CAPTION>
				TFB 2, pH 8.0 for preparation of competent cells</CAPTION><TBODY><TR><TD>
						2.2 g</TD><TD>
						CaCl<SUB>2</SUB></TD></TR><TR><TD>
						0.12 g</TD><TD>
						potassiumacetate</TD></TR><TR><TD>
				
		0.21 g</TD><TD>
						morphoslinopropanesulfuricacid</TD></TR><TR><TD>
						15 ml</TD><TD>
						glycerin</TD></TR></TBODY></TABLE><P>
			were dissolved in dH<SUB>2</SUB>O and diluted to a final volume of 100 mL. The pH was adjusted and the solution sterile filtered. </P><TABLE class="protocol_table"><CAPTION>
				5x Laemmli Buffer</CAPTION><TBODY><TR><TD>
						10 % (w/v)</TD><TD>
						SDS</TD></TR><TR><TD>
						50 % (w/v)</TD><TD>
						Glycerine</TD></TR><TR><TD>
						0.02 % (w/v)</TD><TD>
						Bromphenol blue</TD></TR><TR><TD>
						15 % (w/v)</TD><TD>
						Beta-Mercaptoethanol</TD></TR></TBODY></TABLE><TABLE class="protocol_table"><CAPTION>
				Acrylamide - Mix</CAPTION><TBODY><TR><TD>
						30 % (w/v)</TD><TD>
						Acrylamide</TD></TR><TR><TD>
						0.8 (w/v)</TD><TD>
						Bisacrylamide</TD></TR></TBODY></TABLE><P> </P><STRONG>List of used strains</STRONG><TABLE class="scientific_table"><TBODY><TR><TH>Strain</TH><TH>Genotype</TH><TH>Risk Group</TH><TH>Risk to Humans</TH></TR><TR><TD><I>E. coli</I> XL1-Blue-MRF‘</TD><TD>endA1 gyrA96(nalR) thi-1 recA1 relA1 lac glnV44 F'[ ::Tn10 proAB+ lacIq Δ(lacZ)M15 Amy CmR] hsdR17(rK- mK+)</TD><TD>S1</TD><TD>Non-pathogenic, may cause irritation to skin, eyes and respiratory tract, may affect kidneys</TD></TR><TR><TD><I>Methylococcus capsulatus</I></TD><TD>wildtype</TD><TD>S1</TD><TD>Non-pathogenic, may cause irritation to skin, eyes and respiratory tract, may affect kidneys</TD></TR><TR><TD><I>E. coli</I> BL21 DE3</TD><TD>F– ompT gal dcm lon hsdSB(rB- mB-) λ(DE3 [lacI lacUV5-T7 gene 1 ind1 sam7 nin5]</TD><TD>S1</TD><TD>Non-pathogenic, may cause irritation to skin, eyes and respiratory tract, may affect kidneys</TD></TR><TR><TD><I>E. coli</I> JM109</TD><TD>endA1 glnV44 thi-1 relA1 gyrA96 recA1 mcrB+ Δ(lac-proAB) e14- [F' traD36 proAB+ lacIq lacZΔM15] hsdR17(rK-mK+)</TD><TD>S1</TD><TD>Non-pathogenic, may cause irritation to skin, eyes and respiratory tract, may affect kidneys</TD></TR></TBODY></TABLE><STRONG>List of used Primers</STRONG><P><A href="https://static.igem.org/mediawiki/2014/0/0d/TU-BS_Primerlist.pdf">The primer sequences that were used.</A></P><STRONG><I>E. coli</I> Culture Conditions</STRONG><P>
             For the cultivation of <I>E. coli</I> 2xYT medium was used. If necessary 1 µL/mL of an antibiotic stock solution was added. The cultures were incubated at 37 °C and in case of liquid cultures shaken at 250 rpm.
         </P><STRONG><I>Methylococcus capsulatus</I> Culture Conditions</STRONG><P>
           For the cultivation of <I>M. capsulatus</I> NMS medium was used. The cultures were incubated at 42 °C at 50 % methane atmosphere.
         </P><STRONG>Measurement of Growth Curves</STRONG><P>
             Overnight cultures in 2xYT medium containing the appropriate antibiotic were inoculated directly from -80 °C glycerol cell stock. Cells were grown in non-baffled-flasks at 37 °C and 250 rpm.
The next day, main cultures in 2xYT medium containing the appropriate antibiotic were inoculated from overnight cultures to an OD<SUB>600</SUB> 0.1. Cells were grown in non-baffled flasks at 37°C and 250 rpm. 
Samples from each flask were taken at appropriate time points depending on the growth phase of the cells. Optical density was measured at a wavelength of 600 nm. Cells were left to grow until the stationary phase was reached.

         </P><STRONG>Preparation of chemically competent cells</STRONG><P>
             100 mL of 2xYT media containing required antibiotic were inoculated and incubated at 37 °C and 250 rpm until OD<SUB>600</SUB> 0.5. Subsequently the cells were incubated on ice for 15 minutes and then centrifuged at 4 °C and 400 rpm for 5 minutes. The excess was discarded and the pellets resuspended in 7.5 mL ice-cold TFB1. Afterwards, the solution is incubated on ice for 90 minutes and then centrifuged for 5 minutes at 4000 rpm. Finally, the pellets were resuspended, aliquoted and quick-frozen in liquid nitrogen. The vials were stored at -80°C.
         </P><STRONG>Cryopreservation</STRONG><P>
             The respective strain was grown on 2xYT medium at 37 °C and 250 rpm overnight. Afterwards a 20 % Glycerin solution was prepared with the liquid culture, aliquoted, shock frozen and stored at -80 °C.
         </P><STRONG>Minipreparation of Plasmid DNA</STRONG><P>
             Miniprep was done with Plasmid DNA purification Kit by Macherey-Nagel following the manufacturer’s protocol.
First 3-5 mL of the cell suspension were spun down for 10 minutes (5000 g).
For the alkaline lysis of the DNA the pellet was resuspended in 250 μL Resuspension Buffer A1. 250 μL of Lysis Buffer A2 was added and incubated at RT for 5 min. 300 μL of Neutralization Buffer A3 was added to stop the reaction. As final step of the alkaline lysis the mixture was centrifuged at 10.000 x g for 10 minutes.
Afterwards, the NucleoSpin Column was loaded with 750 μL lysate and then centrifuged for 1 minute (10.000 x g). Subsequently the DNA was washed with 500 μL Wash Buffer and centrifuged again for 1 minute (10.000 x g). The DNA was the precipitated with the Ethanol containing wash buffer and separated by another centrifugation (1 min, 10000 x g) and dried.
At last the DNA was eluted with 50 μL elution buffer, which was pre-heated to 60 °C, and incubated for 5 min. The eluted DNA was separated from the matrix by a final centrifugation step (1 min, 5000 x g).

         </P><STRONG>Colony PCR</STRONG><P>
             After successful transformation single colonies were picked and analyzed by colony PCR. Therefore each colony was picked from the original plate with a pipette tip and dipped into the PCR-mix. Afterwards the pipette tip was used to inoculate a second agar plate. PCR fragments were analyzed via gelectrophoresis.
         </P><TABLE class="protocol_table"><TBODY><TR><TD><STRONG>Component</STRONG></TD><TD><STRONG>μl per 10 μL reaction batch</STRONG></TD></TR><TR><TD>
						GoTaq Buffer flexi green</TD><TD>
						2</TD></TR><TR><TD>
						dNTP mix</TD><TD>
						0.2</TD></TR><TR><TD>
						MgCl<SUB>2</SUB> (25 mM)</TD><TD>
						0.8</TD></TR><TR><TD>
						Primer 1</TD><TD>
						0.5</TD></TR><TR><TD>
						Primer 2</TD><TD>
						0.5</TD></TR><TR><TD>
						Promega GoTaq polymerase</TD><TD>
						0.05</TD></TR><TR><TD>
						H<SUB>2</SUB>O</TD><TD>
						5.95</TD></TR></TBODY></TABLE><P> </P><TABLE class="protocol_table"><TBODY><TR><TD><STRONG>Temperature [°C]</STRONG></TD><TD><STRONG>Time</STRONG></TD><TD><STRONG>Cycles</STRONG></TD></TR><TR><TD>
						95</TD><TD>		2 min</TD><TD>
						1</TD></TR><TR><TD>
						94</TD><TD>
						15 s</TD><TD colspan="1" rowspan="3">
						30</TD></TR><TR><TD>
						52</TD><TD>
						20 s</TD></TR><TR><TD>
						72</TD><TD>
						60 s</TD></TR><TR><TD>
						72</TD><TD>
						10 min</TD><TD>
						1</TD></TR></TBODY></TABLE><TABLE class="protocol_table"><CAPTION>
				DNA amplification via PCR with NEB Phusion High-Fidelity DNA Polymerase</CAPTION><TBODY><TR><TD><STRONG>Component</STRONG></TD><TD><STRONG>μl per 20 μl reaction batch</STRONG></TD></TR><TR><TD>
						Nuclease- free water</TD><TD>
						add 20</TD></TR><TR><TD>
						5x Phusion HF</TD><TD>
						4</TD></TR><TR><TD>
						10 mM dNTPs</TD><TD>
						0.4</TD></TR><TR><TD>
						10 μM forward primer</TD><TD>
						1</TD></TR><TR><TD>
						10 μM reverse primer</TD><TD>
						1</TD></TR><TR><TD>
						Template DNA</TD><TD>
						Variable: 1 pg - 10 ng</TD></TR><TR><TD>
						Phusion DNA polymerase</TD><TD>
						0.2</TD></TR></TBODY></TABLE><P> </P><TABLE class="protocol_table"><TBODY><TR><TD><STRONG>Temperatur [°C]</STRONG></TD><TD><STRONG>Time</STRONG></TD><TD><STRONG>Cycles</STRONG></TD></TR><TR><TD>
						98</TD><TD>
						30 s</TD><TD>
						1</TD></TR><TR><TD>
						98</TD><TD>
						10 s</TD><TD colspan="1" rowspan="3">
						30</TD></TR><TR><TD>
						50 - 60 depending on primer</TD><TD>
						30 s</TD></TR><TR><TD>
						72</TD><TD>
						30 s/kb</TD></TR><TR><TD>
						72</TD><TD>
						10 min</TD><TD>
						1</TD></TR></TBODY></TABLE><P> </P><STRONG>Gel electrophoresis</STRONG><P>
             Depending on the size of the DNA fragments a 0.8 to 2 % agarose gel with 0.2 µg/mL ethidiumbromide was used. The gel was placed in an electrophoresis chamber and covered in 1x TAE buffer. Subsequently DNA samples mixed with loading buffer were loaded to the gel pockets and separated at 80-120 V for about 1 h. Last the gel was photographed for documentation. 
         </P><STRONG>Gel Slice Preparation</STRONG><P>
             10 µL of 6x loading buffer was added to digested DNA und mixed. The sample was loaded onto a 1 % agarose gel and separated via gel electrophoresis at 75 mA for 20-40 min depending on the size of the DNA. For gel extraction, the gel was visualized on a UV-lightsource and the appropriate bands were cut using the x-tracta™ Gel Extractor tool (Promega). Gel purification was carried out using the Macherey-Nagel Nucleospin Gel and PCR Clean-up Kit. 
         </P><H3>Cloning</H3><STRONG>Restriction digest</STRONG><P>
             For restriction of vector and insert DNA approximately 1000 ng purified DNA, 5 µL 10x buffer and 1 µL of each of the two restriction enzymes were mixed in a 1.5 mL reaction tube. The amount of DNA used depended on the size of the desired fragment. Water was added to a final volume of 50 µL. Restriction was carried out at 37 °C for 1 h. The reaction was stopped by inactivating the enzymes at 80 °C for 20 min.
         </P><UL><LI>PstI/EcoRI HF: NE 2.1</LI><LI>PstI/SpeI HF: NE 2.1</LI><LI>PstI/XbaI: NE 3.1</LI><LI>all other combinations: NEB CutSmart Buffer</LI></UL><STRONG>Dephosphorylation</STRONG><P>
             For the desphosphorylation of vector DNA 5.56 µL Antarctic Phos Reaction Buffer (NEB) and 0.5 µL Antarctic Phosphatase (NEB) were added to the digested DNA and mixed. DNA was incubated at 37 °C for 60 min. After 30 min of incubation 0.5 µL Antarctic Phosphatase was added to the DNA and mixed.
         </P><STRONG>DNA Purification</STRONG><P>
             After desphosphorylation the vector DNA was purified using the Macherey-Nagel Nucleospin Gel and PCR Clean-up Kit.
         </P><STRONG>Ligation</STRONG><P>
             For ligation of the constructed plasmid 50 ng vector DNA, 3 times as much insert DNA, 2 µL T4 Ligase Buffer (NEB), 0.5 µL T4 Ligase (NEB) and water added to a final volume of 20 µL were mixed in 1.5 mL reaction tube. Ligation was carried out at 16 °C overnight. The reaction was stopped by inactivating the enzymes at 65 °C for 20 min.
         </P><STRONG>Transformation of chemocompetent cells via heatshock</STRONG><P>
             One -80 °C glycerol stock of chemocompetent cells was thawed on ice. 10 µL of the ligated DNA was prepared in a 1.5 mL reaction tube. 50 µL of the cells were added and incubated for 20 min on ice. The mixture was then heat shocked at 42 °C for 1 min and cooled on ice for 2 min. 1 mL SOC-medium was added and the cells were incubated for 1 h at 37 °C while shaken at 600 rpm. 100 µL of the cell suspension were plated on a 2xYT agar plate with the according antibiotic (chloramphenicol or ampicillin) and incubated at 37 °C overnight. 
         </P><H3>Expression of the His-constructs</H3><P>
             50 ml 2xYT over night cultures of cells expressing HIS-tagged subunits of the MMO were inoculated from Glycerinstocks. After 24 h a main culture of 500 ml 2xYT-Ca-Glucose-Medium was inoculated to a starting OD<SUB>600</SUB> of 0,1 and cultivated at 37 °C and 250 rpm. Induction to a concentration of 100μM IPTG between OD 0,4-0,6. After induction the culutures were cultivated at 30 °C. 1mL of an arabinose solution (200 mg/mL) was added after 2 h and 4 h. After 6 h and 24 h after induction with IPTG 250 ml of the cells were harvested, centrifuged (13,000 g, 10 min) and the supernatant was discarded. The expression of the HIS-tagged subunits of the MMO was analyzed by using SDS-PAGE and Coomassie or Immunostaining respectively.
         </P><H3>Expression of the mmo-construct</H3><P>
             5 ml 2xYT overnight cultures of cells expressing the MMO were inoculated from Glycerinstocks. After 24 h a main culture of 50 ml 2xYT-Ca-arabinose (4 mg/mL)-Medium was inoculated to a starting OD<SUB>600</SUB> of 0,1 and cultivated at 37 °C and 250 rpm. Induction to a concentration of 100μM IPTG between OD 0,4-0,6. After induction, the cultures were cultivated at 30°C. 1mL of an arabinose solution (200 mg/mL) was added after 2 h and 4 h. The complete culture was harvested after 6 h.
         </P><H3>Isolation of the soluble and inclusion fraction</H3><P>
             The pellets harvested after 24 hours after induction with IPTG was resuspended in 5 ml PBS each. The cells were disrupted by sonification for 1 min and afterwards centrifuged for 15 min (800 g, 4 °C).
The supernatant was transferred into a new 15 ml falcon and centrifuged for 30 min (6000 g, 4°C).  The obtained supernatant contains MMO-subunits which are solubly produced. The pellet obtained after the first centrifugation step was resuspended in 10 ml PBS and again centrifuged for 15 min (6000 g, 4 °C). After the supernatant has been discarded this step has been repeated (resuspension in 10 ml PBS and centrifugation step). After the supernatant has been discarded the pellet was then resuspended in 2 ml PBS 8 M Urea added. The suspension was then transferred into a 2 ml Eppendorf tube and centrifuged for 20 min (11000 g, 4 °C). The obtained supernatant contains MMO-subunits which are produced in inclusion bodies.

         </P><P>
             The soluble fraction and inclusion body fraction was stored at -20 °C and later analysed using SDS-PAGE and Coomassie or Immunostaining respectively. 
         </P><H3>SDS-Page</H3><P>
             The samples were mixed with 1x Laemmli Buffer with beta-Mercaptoethanol and heated for 10 min to 95 °C. The Gel was loaded with 10 μl of the prepared samples and run at 200 V for 50 min.
         </P><H3>Coomassie Staining</H3><P>
             After the SDS-PAGE has been executed the gel is put into a glass petri dish filled with Coomassie Brilliant Blue coloring suspension and boiled up for a couple of second. The gel was incubated in the coloring suspension on a rocker for 10 min until the coloring was sufficient. Afterwards, the gel was destained using destaining suspension and again boiled up for a couple of seconds and incubated on the rocker for an hour. The gel was stored in water until it was fully destained. 
         </P><H3>Western Blot</H3><P>
             Blotting with Trans-blot SD semi-dry transfer cell: 
Order from top to bottom: 

         </P><OL><LI>filter paper</LI><LI>gel</LI><LI>blot-membrane</LI><LI>filter paper</LI></OL><P>
             Activation of blot-membrane with MeOH (100 %). Filter paper are moistened with running buffer 
Run of blot: 15V for 30 min

         </P><H3>Immunostain</H3><P>
             After having executed the Blot, the membrane was blocked with milk powder for 1 hour on a rocker. After 2 washing steps with PBS (0,1 %) (1x short, 1x for 5 min) the membrane was incubated for an hour in the first antibody solution on the rocker (Qiagen Anti Penta his (mouse) 1:2000). Again there was a washing step with PBS (1x short, 1x for 5 min). The membrane was then incubated for an hour in the second antibody solution (Dianova Goat anti-Mouse IgG AP-conjugated 1:30.000, Predilution of antibody already 1:30) on the rocker. After the incubation, the membrane was washed 3 times with PBST (2x short, 1x for 10 min) and twice washed with substrate buffer (1x short, 1 x for 5 min). The blot is then incubated in substrate buffer with NBT and BCIP (1: 100 in substrate buffer) until bands are visible (1-20 min). Afterwards, the blot was washed twice with water (1x short, 1x for 5 min) and dried between two filter papers.
         </P><H3>sMMO activity assay</H3><P>
             50 mL bacterial suspension was cultured as described in “Expression of the mmo-construct”. The culture was diluted to OD<SUB>600</SUB> 4 and centrifuged and washed twice with 0,9 % saline solution. The bacterial pellet was resuspended in 50 mL NMS-Medium. The methane measurement was performed in a modified anaerobic jar. Therefore, the solution was placed into the anaerobic jar and was treated with methane and compressed air in order to obtain an atmosphere with 2 % methane. The methane concentration was detected with an “MQ-4 Semiconductor Sensor“ for 10.000 sec.
A converted anaerobic jar in combination with a methane sensor was used for measurements.
</P><DIV class="row"><DIV class="col-sm-5 col-md-6"><DIV class="fig-caption" align="center" style="width:100%;"><UL><LI>Cables and tubing for methane atmosphere creation and measurement.</LI></UL></DIV></DIV><DIV class="col-sm-5 col-md-6"><DIV class="fig-caption" align="center" style="width:100%;"><UL><LI>Methane sensor in the anaerobic jar.</LI></UL></DIV></DIV></DIV><P>
For the measurements, we used the sensor MQ-4 Semiconductor Sensor for Natural Gas by Henan Hanwei Electronics. Its function is based on SnO2 whose low conductivity rises with higher concentrations of combustible gases like methane or propane. This sensitivity for methane and its low costs made it perfect for our do it yourself methane measuring system. The data from the sensor were processed by a Sainsmart nano V3.0 device, a smaller variant of the Arduino board.</P><DIV class="row"><DIV class="col-sm-5 col-md-6"><DIV class="fig-caption" align="center" style="width:100%;"><UL><LI>The methane sensor MQ-4 used in our experiment.</LI></UL></DIV></DIV><DIV class="col-sm-5 col-md-6"><DIV class="fig-caption" align="center" style="width:100%;"><UL><LI>The Sainsmart Arduino Nano v3 Board for processing the data coming from the MQ-4 sensor.</LI></UL></DIV></DIV></DIV><P>
To get the right methane concentration for each experiment we used flow controllers to set up the respective flow rates for methane, air and carbon dioxide which were then conducted through the anaerobic jar for 10 minutes until a homogenous methane atmosphere was formed.
</P><DIV class="row"><DIV class="col-sm-5 col-md-6"><DIV class="fig-caption" align="center" style="width:80%;"><UL><LI>Tubing system and flow controllers to generate a desired methane atmosphere.</LI></UL></DIV></DIV></DIV><P>
When the atmosphere needed was generated and all vents were closed the anaerobic jar was placed in an incubator set to correct temperature. For some experiments a magnetic stirrer was also placed in the incubator to keep the homogeneity of the culture and the atmosphere in the jar. The sensor was connected to the Arduino board which was then plugged into the PC outside the incubator via USB cable.
</P><DIV class="row"><DIV class="col-sm-5 col-md-6"><DIV class="fig-caption" align="center" style="width:100%;"><UL><LI>Complete setup of our methane measuring system.</LI></UL></DIV></DIV></DIV><P>
We used a modified version of the <A href="https://2014.igem.org/Team:Aachen/Collaborations/Methane_Sensor">Serial Client</A> programmed by Michael Osthege from iGEM Aachen converting the processed signal of the Arduino board into a visual graph and a exportable table file. 

For the experiments, we filled the jar with a defined atmosphere of 2% methane in 98% air until the sensor displayed a stable methane concentration which was then set as 2%.The subsequently measured data were considered in relation to this value. Measurements were started after the sensor had adapted to humidity and temperature in the anaerobic jar. After the measurement, the jar was filled with compressed air to define a zero-point.
         </P><H3>Entrapment of <I>E. coli</I></H3><P>
             50 mL of an overnight culture was pelletized. The pellet was washed twice in 0,9 % saline solution resuspended and diluted to OD<SUB>600</SUB> 4 in 10 mL 0,9 % saline solution. 1,5 g alginate was solved in 40 mL ddH<SUB>2</SUB>O and was briefly boiled up. The alginate solution was mixed with all required antibiotics and arabinose (4 mg/mL). The whole solution was cooled and mixed with 10 mL bacterial 0,9 % saline solution. Beads were dropped with a transfer pipette into a 0,05 M CaCl<SUB>2</SUB> solution. The CaCl<SUB>2</SUB> solution was decanted after 10 min.


         </P><H3>Counting of cells with Thoma counting plate</H3><P>     Cell culture is placed on the plate and covered with a cover plate. 
Sizes of the counting plate: depth: 0.02 mm area: 0.0025 mm².
(Volume of a C-field: 0,0005 mm³ ).
The number of cells per mL was calculated accordingly.

         </P><H3>Cultivation on rumen fluid agar plates</H3><P>
We prepared agar plates with different concentrations of sterile filtrated rumen fluid, each with and without supplementation of glucose (100 mM). One-half of each of these plates was inoculated with unsterile rumen fluid and the other one with <I>E. coli</I>. The plates were incubated overnight at 37°C.
         </P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="templatemo-line-header"><DIV class="text-center"><H4 align="center">Our Supporters</H4></DIV></DIV><DIV class="templatemo-footer"><DIV class="container"><DIV class="row"><DIV class="text-center"><A class="btn" href="#">Back To Top</A><DIV class="footer_bottom_content">Copyright © 2014 iGEM Team Braunschweig<A href="mailto:igem.braunschweig@gmail.com">Contact us</A></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Braunschweig/Notebook-content">http://2014.igem.org/Team:Braunschweig/Notebook-content</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Braunschweig/Notebook-content" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Braunschweig/Notebook-content" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Braunschweig/Notebook-content&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Braunschweig/Notebook-content&amp;oldid=366129" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>