<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_XMU-China_Content3"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:XMU-China/Content3">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:XMU-China/Content3&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU-China/Content3&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU-China/Content3&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:XMU-China/Content3" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:XMU-China/Content3</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>LinkUp - Multipurpose HTML Template</TITLE></P><DIV id="wrapper"><DIV id="separator"><DIV class="center-block"><H3>Background</H3></DIV></DIV><DIV class="center-block-page clearfix"><H4>Forming the Glee</H4><P>To be a good glee, first it needs a good song. The most intriguing song on earth is called OSCILLATION. If you look about, you will find it ubiquitous and extremely useful. After we taught each member this song, we need to help them sing it together, so oscillation became SYNCHRONIZED OSCILLATION. Thus how we form a glee!
</P><H4>Oscillation: The Magic Song of World</H4><P>Once upon the time, there was a magic song called oscillation, it permeated every corner of the world. It <B>regulated nearly everything</B> on the earth. The interaction of moon and ocean generated the magnificent tide oscillation, and the biological process that displays a 24-hour oscillation inside human body.</P><TABLE><TBODY><TR><TD>Fig. 1-1 The Tides depend on several factors, including where the sun and moon are relative to the Earth</TD><TD>Fig. 1-2 The circadian clock inside human body</TD></TR></TBODY></TABLE><P> 
Besides, oscillation also <B>plays an important part </B>in signal transition. Take lasers for example, which are known for their intensity and can be focused to a tight spot over long distance. These characteristics all owe to their spatial coherent in the frequency of the light source.
</P><TABLE><TBODY><TR><TD>Fig. 1-3 Helium-Neon laser demonstration at the Kastler-Brossel Laboratory at Paris VI: Pierre et Marie</TD></TR></TBODY></TABLE><H4>Oscillation in Bacteria</H4><P>
Synthetic biologists have done a lot of work to teach bacteria to sing this oscillation song by building artificial genetic circuit inside them, since oscillations can lead to fantastic applications and benefit our everyday life. Besides regulation and signal transmission, oscillations in living organisms can also react to its growing environment, which probably will activate the trigger of oscillation inside cells and generate oscillation signals. From collected signals we can tell how an environment factor influences the cells behavior, and in turn the environment factor can be indicated by the signals. Furthermore, if we interpret these kinds of oscillation signals like 1010 in binary system, then we could even make an electronic environmental factor detector, using signals from environment as an input, and numbers on computer screen as output (Fig. 1-4).
</P><TABLE><TBODY><TR><TD>Fig. 1-4 How to make a signal converter</TD></TR></TBODY></TABLE><P>Here is the progress of artificial synthetic biological oscillations have made.
In <B>the first generation of oscillation</B>, three transcriptional repressor systems that are not part of any natural biological clock are used to build an oscillating network, termed the repressilator, in <I>Escherichia coli</I> (Fig. 1-5). The network periodically induces the synthesis of green fluorescent protein as a readout of its state in individual cells. However, plasmids can hardly pass from generation to generation and this artificial clock is not robust enough and always displays noisy behavior.
</P><TABLE><TBODY><TR><TD>Fig. 1-5 The first generation oscillator, known as repressor</TD></TR></TBODY></TABLE><P>Scientists constructed the second generation oscillator rapidly. This <B>improved oscillator</B> is fast, robust and persistent; with tunable oscillatory periods as fast as 13 min (Fig.1-6). The oscillator is designed using a previously modeled network architecture comprising linked positive and negative feedback loops. But this circuit could only monitor oscillations in individual cells through multiple cycles, which means the song is still limited to single individual. 
</P><TABLE><TBODY><TR><TD>Fig. 1-6 2nd generation of oscillation consists of positive and negative feedback loops</TD></TR></TBODY></TABLE><P>Thinking that singing alone is lonely, scientists introduce quorum sensing to connect isolated cells and let them sing together, and technically it's called synchronized oscillation. The synchronization effect of quorum sensing, however, is limited to over tens of micrometer (Fig.1-7). If the synchronized colony could be enlarged, then the oscillation signal will be much stronger and steadier. Longer ranged synchronized oscillation is realized by a gas-phase redox signal H2O2 to the third generation circuit, which could achieve the communication among colonies at over millimeter scales and became the fourth generation. Oh yeah, finally our bacteria can sing the same oscillation at the same time, so they decided to form their own glee!
</P><TABLE><TBODY><TR><TD>Fig.1-7 3rd generation involved in quorum sensing</TD></TR></TBODY></TABLE><P>That's how our little <I>E. coli</I> friend formed their glee, a long but cheerful story. Thanks to hardworking synthetic biologists. Let's go and see how this fourth generation oscillation glee functions inside in <B>THREE PLASMIDS</B>!</P><P><I>Reference</I>
1.	http://en.wikipedia.org/wiki/Oscillation
2.	Elowitz M. B. &amp; Leibler S. A synthetic oscillatory network of transcriptional regulators. <I>Nature</I> 403, 335-338 (2000)
3.	Stricker J. et al. A fast, robust and tunable synthetic gene oscillator. <I>Nature</I> 456, 516-519 (2008)
4.	Danino T., Mondrago´n-Palomino O., Tsimring L. &amp; Hasty J. A synchronized quorum of genetic clocks. <I>Nature</I> 463, 326-330 (2010)</P></DIV></DIV><DIV id="wrapper1"><DIV id="separator"><DIV class="center-block"><H3>Mechanism</H3></DIV></DIV><DIV class="center-block-page clearfix"><H4>Three Parts in Circuit Designation</H4><P>
In a synchronized oscillatory system, three important parts should be included: the <B>oscillator</B>, which is the biochemical machinery that generate the oscillatory output; the <B>coupling</B> pathway that ensure the connection among cells; and output pathway, which is also known as a <B>reporter</B> that reflect the state of the oscillator to downstream targets (Fig. 2-1). 
</P><TABLE><TBODY><TR><TD>Fig. 2-1 Circuit working mechanism</TD></TR></TBODY></TABLE></DIV><DIV class="center-block-page clearfix"><H4>Oscillator</H4><P>
Quorum sensing (QS) is a cell-to-cell signaling mechanism that refers to the ability of bacteria to respond to chemical hormone-like molecules called autoinducers. When an autoinducer reaches a critical threshold, the bacteria detect and respond to this signal by altering their gene expression. In our circuit, QS (from <I>Vibrio fischeri</I>) is installed as a positive feedback while <I>aiiA</I> (from <I>Bacillus Thurigensis</I>) acts as a negative one to compose an oscillator together. In the quorum sensing part, the <I>luxI</I> gene is at low expression level and produces LuxI protein that synthesize a kind of acyl-homoserine lactone (AHL), which is a small molecule that can diffuse across the cell membrane and mediate intercellular coupling when it reaches the threshold as enough biomass accumulated. AHL will bind intracellular protein LuxR, which is also consecutively produced by <I>luxR</I> gene. The LuxR-AHL complex can activate the <I>luxI</I> promoter, and the positive feedback loop is built. At the same time, the <I>aiiA</I> gene, which is under control of the same promoter is expressed and produce a lactonase enzyme known as <I>aiiA</I> that hydrolyzes the lactone ring of AHL. (Fig. 2-1) In this system, the activator enhances the expression of both activator and repressor, which shares the common motif of many synthetic oscillators. 
</P></DIV><DIV class="center-block-page clearfix"><H4>Coupling</H4><P>
The communication range of quorum sensing, however, is limited by the diffusion rate of AHL, could only reach cells over tens of micrometers. So, we introduced the coding gene for NADH dehydrogenase II (<I>ndh</I>) and put it under the control of the same <I>luxI</I>promoter, which means <I>ndh</I> also has a periodical expression in accordance with the oscillator. NDH-2 is a membrane-bound respiratory enzyme that generates low level H<SUB>2</SUB>O<SUB>2</SUB> and superoxide (O<SUB>2</SUB><SUP>-</SUP>) and H<SUB>2</SUB>O<SUB>2</SUB> will permeate to neighboring colonies. Periodic production of H<SUB>2</SUB>O<SUB>2</SUB> changes the redox state of a cell immediately and interacts with the synthetic circuit through the native aerobic response control systems, including ArcAB, which has a binding site in the <I>lux</I> promoter region. Before the oxidizing condition is triggered, ArcAB is partially expressed, so <I>lux</I> is partially repressed. When the concentration of H<SUB>2</SUB>O<SUB>2</SUB> is increasing, ArcAB is gradually inactivated, and the repression of <I>lux</I> is relieved. With the periodically produced vapor phase H<SUB>2</SUB>O<SUB>2</SUB> that can diffuse quickly among colonies, the oscillation is not only strengthened but also expanded to millimeter scales(Fig. 2-2).</P><TABLE><TBODY><TR><TD>Fig. 2-2  H<SUB>2</SUB>O<SUB>2</SUB> functions as an activator to the quorum sensing promoter</TD></TR></TBODY></TABLE><P> 
When compared with the strong but short ranged coupling by QS, H<SUB>2</SUB>O<SUB>2</SUB> might be weaker but long ranged because its disperse characteristic. These two levels of communication between cells formed the basic oscillatory system inside our host. 
</P></DIV><DIV class="center-block-page clearfix"><H4>Reporter</H4><P>
A good reporter in an oscillatory circuit should be steadily expressed and can be easily detected by regular equipment in laboratory, so broadly used reporter green fluorescent protein (GFP) came into our mind. 
First, we built regular <I>gfp</I> gene into our circuit and it worked well. Its fluorescence can be observed by fluorescence microscopy under 1s of exposure time on the microfluidic array 1(Fig. 2-3).</P><TABLE><TBODY><TR><TD>Fig. 2-3 GFP under fluorescence microscopy on a microfluidic array</TD></TR></TBODY></TABLE><P>
However, there is a newly found GFP variant Superfolder GFP, which has shown outstanding performance in many ways. It has two advantages in observing: 
1)	Fold 3.5 to 4 times faster than regular GFP; 
2)	Yield up to four times more total Fluorescence than regular GFP.
The greater the fluorescence strength is, the shorter exposure time will be needed, thus can decrease the photobleaching (GFP will be eventually destroyed by the light exposure that tries to stimulate it into fluorescing) that will lead into lapses in data processing.
Besides, we also want to make a comparison between two different GFPs' performances in our oscillatory circuit. For example, peaks, troughs and periods in oscillatory curves. The following picture shows the difference between GFP and sfGFP both on colE1 backbone.(Fig. 2-4)</P><TABLE><TBODY><TR><TD>Fig.2-4 Difference between GFP and sfGFP both on colE1 backbone
(P. S. Thanks the Peking iGEM team 2013 for providing us the sfGFP part.)
</TD></TR></TBODY></TABLE><P>
Please come with us to see how we built our magic circuit in <I>E. coli</I> in Construction!</P></DIV></DIV><DIV id="wrapper2"><DIV id="separator"><DIV class="center-block"><H3>Circuit Construction</H3></DIV></DIV><DIV class="center-block-page clearfix"><H4>Overview</H4><P>
So we designed three plasmids to function as the three important parts in the oscillation circuit, and all of them are in the charge of the same quorum sensing promoter to make sure that all target genes share the same oscillatory period (Table 2-1).
</P><TABLE width="100%" border="1" cellspacing="0px" align="center"><CAPTION>
    Table 2-1 Three plasmids constructed as parts of our circuit
  </CAPTION><TBODY><TR><TH rowspan="2">No.</TH><TD rowspan="2">Plasmid</TD><TD rowspan="2">ReplicationOrigin</TD><TD rowspan="2">CopyNumber</TD><TD rowspan="2">Resistance</TD><TD colspan="2">Size(bp)</TD></TR><TR><TD>Insert</TD><TD>Backbone</TD></TR><TR><TH>A1</TH><TD>pSB1C3-gfp-luxI</TD><TD>pSB1C3</TD><TD>high(100~300)</TD><TD>Chloromycetin(Cm)</TD><TD>4052</TD><TD>2070)</TD></TR><TR><TH>A2</TH><TD>pSB1C3-sfgfp-luxI</TD><TD>pSB1C3</TD><TD>high(100~300)</TD><TD>Chloromycetin(Cm)</TD><TD>4046</TD><TD>2070)</TD></TR><TR><TH>A3</TH><TD>p3H-GFP-luxI</TD><TD>p3H</TD><TD>Middle(18~22)</TD><TD>Ampicillin
(Amp)</TD><TD>4052</TD><TD>/</TD></TR><TR><TH>A4</TH><TD>p3H-sfGFP-luxI</TD><TD>p3H</TD><TD>Middle(18~22)</TD><TD>Ampicillin
(Amp)</TD><TD>4046</TD><TD>/</TD></TR><TR><TH>B</TH><TD>pSB3T5-aiiA</TD><TD>p15A</TD><TD>Middle(10~12)</TD><TD>Tetracycline
(Tet)
</TD><TD>2135</TD><TD>2837</TD></TR><TR><TH>C</TH><TD>pSB4K5-ndh</TD><TD>pSC101</TD><TD>Low(5)</TD><TD>Kanamycin
(Kan)
</TD><TD>2658</TD><TD>3004</TD></TR></TBODY></TABLE><P>In a synchronized oscillation circuit, each A, B and C plasmid is necessary. The plasmid A expresses GFP <B>breporter</B> and LuxI proteins, which is the <B>positive feedback in oscillator</B>; the plasmid B expresses protein aiiA to degrade AHL and acts as the<B> negative feedback in oscillator</B>; the plasmid C expresses NAD-2 to generate H2O2 to communicate between colonies, is the <B>coupling part</B> in circuit (Fig 2-5). </P><TABLE><TBODY><TR><TD>Fig. 2-5 Plasmids A, B and C in oscillation circuit</TD></TR></TBODY></TABLE><P>You may have noticed that these three plasmids have different resistant genes and replication origins on their backbone. Different resistant genes are used mainly for selection. And since different replication origin means a different copy number of target genes in this plasmid. Using different replication origins is to produce different target proteins in an appropriate ratio, and only in this way can oscillation be observed. Just like a glee needs Soprano, Tenor and Bass to cooperate to finish a song. Besides, three artificial plasmids with same copy numbers always have a competitive relation in one cell and cannot be well expressed.</P><P>Plasmid A has two different backbones with high and middle copy numbers respectively. We built them to see the copy numbers' effect on oscillation. Results can be seen in <B>Improvement (For a better Glee)</B>. </P><P>All of the plasmids we constructed are confirmed by agarose gel electrophoresis, and result can be seen in <B>Parts</B>, where you can also get the link to parts we submitted. </P><P>After the onerous construction work, our glee is ready to perform the synchronized oscillation song, so we have to find them a stage. Let's go to see the beautiful opera in <B>Microfluidics (Stages)</B>. </P><P><I>Reference</I>
1.	Prindle, A., et al., A sensing array of radically coupled genetic 'biopixels'. <I>Nature</I> 481, 39-44, 2012;
2.	Pedelacq, J. et al., Engineering and characterization of a superfolder green fluorescent protein. <I>Nature 24</I>, 79-88, 2006;
3.	Waters C. M. &amp; Bassler B. L. Quorum Sensing: Cell-to-Cell Communication in <I>Bacteria. Annu. Rev. Cell Dev. Biol</I> 21, 319-46, 2005.</P></DIV></DIV></DIV><DIV id="wrapper"><DIV id="separator"><DIV class="center-block" style:=""><H3>MICROFLUIDIC &amp; MICROSCOPY</H3></DIV></DIV><DIV class="center-block-page clearfix"><H4>Microfluidics (Opera Houses)</H4><P>A Microfluidic array to cells is like an opera house to a glee that provides a cell culture environment and observation stage. To accompany the observation of microfluidic we used the fluorescence microscope, just like mass media to photograph the oscillation. We call them M &amp; M. What a lovely couple.</P><H4>Why M&amp;M?</H4><P>

If we want to capture gene expression dynamics and distributions reported by fluorescence, we need to create a chemostat that provide a near-constant environment for the growth of bacteria. The near-constant environment mainly includes two aspects: continuous fresh media to ensure the optimum growth state and constant small populations of bacteria to ensure single-cell resolution in observation. Steady growth state is needed because the oscillation curve is expected to be a near simple harmonic one. Besides, bacteria tend to form biofilm which will lead to the overlapping of cells and create a false impression that the fluorescence increases, and that's why we have to prevent this to maintain a constant bacteria population with single-cell resolution.
What we mentioned above might have covered all aspects we need to consider for communication in a single colony, but if we still want to observe the coupling between colonies we have to create an array that contains many colonies and allows H<SUB>2</SUB>O<SUB>2</SUB> to pass through freely.
Among many different techniques, microfluidic array seems to be our best choice with following advantages:
1)	Precisely control of flowing rate and cell populations when equipped with syringe pumps;
2)	Cells behavior including fluorescence strength can be captured by high resolution microscopy;
3)	Trap numbers and dimensions can be adapted to different experimental need;
4)	Productive: fewer resources and less time are required when compared with macroscopic cultures.</P></DIV><DIV class="center-block-page clearfix"><H4>Design of Microfluidic Array</H4><P><B>1.Pretests in &quot;Swimming Pool&quot;</B></P><P>Before our experiments on the microfluidic array we described before, a series of pretests were done on a chip whose structure looks like many swimming pool on the ground (Fig. 3-1).</P><TABLE><TBODY><TR><TD>Fig. 3-1 A &quot;Swimming pool&quot; chip in black field; those white squares (100μm×100μm×100μm) are chambers for bacteria</TD></TR></TBODY></TABLE><P>
From the sketch (Fig. 3-2) we can see that the chamber is like a swimming pool where bacteria swim happily in the fresh LB that running from up above, and newly generated cells would be washed away. 
</P><TABLE><TBODY><TR><TD>Fig. 3-2 Diagram for swimming pool model's working mechanism</TD></TR></TBODY></TABLE><P>A &quot;swimming pool&quot; chamber model would have a lager AHL concentration when compared with a &quot;passage&quot; chamber because the former one has a minor dilution. According to a published research that the oscillation period will be shorter as the AHL concentration increases. So we assume that it will be more time-consuming to use this &quot;swimming pool&quot; model in pretests.</P><P>However, this chamber is about 100 μm deep and could grow E. coli far more than monolayer. As time goes by, the oscillation in a single colony signaling by AHL could never go back to the original baseline, and the oscillation curves have a tendency like the following graph (Fig. 3-3).</P><TABLE><TBODY><TR><TD>Fig. 3-3 Simulation curve for our oscillation in &quot;Swimming Pool&quot;</TD></TR></TBODY></TABLE><P><B>2.Advanced &quot;Passage&quot;</B></P><P>According to what we have discussed above, the microfluidic device should be able to provide flowing fresh media and maintain a constant bacteria population with single-cell resolution. Because &quot;swimming pool&quot; has a cell stacking problem, we designed a new passage-like device as shown below (Video 1).</P><H4>Video 1</H4><P><B>Channels and Chambers</B></P><P>
This device is composed by two parts: the flowing channels and near 3,000 square trapping chambers, and their layout is like there are many passages with symmetry rooms distributed on both sides, so we call this array &quot;passage&quot; model. Firstly, cells are loaded from the cell port. When the loading stops, most cells would remain in channels, but if the loading cell density is dense enough, many cells would be pushed into chambers and form a monolayer of Escherichia coli growing in it. This is the time for running LB to get in. Fresh LB is loaded from the media port with Tween 20* and appropriate antibiotics. Flowing LB does not only permit a constant nutrition and exponentially growing cells for more than two days (that's the longest time we have tried), but also washes away newly generated cells to sustain a certain cell density, which is crucial for the determination of fluorescence strength.
</P><P><B>Size</B></P><P>
It has been stated before in the Mechanism part that most quorum sensing systems have a critical cell density requirement, so the trap size is designed neither too big nor too small to capture enough cells and allows for nutrients to permeate to every corner of the chamber. 
You may also notice that the height of the chamber is extremely small, which is really demanding for the researcher's skills, but it's important for the observation. The oscillation could only be seen when the height of the chamber is in a particular range as our modeling predicted for many complicated reasons. For more details, please refer to the Modeling part. </P><P><B>Material</B></P><P>
Polydimethylsiloxane (PDMS), a transparent polymer material, is chosen to build the devise because of its excellent breathability, which is necessary for the permeation of H<SUB>2</SUB>O<SUB>2</SUB>. Once H<SUB>2</SUB>O<SUB>2</SUB> diffused outside cells, it can travel from colony to colony and synchronize them. 
Our device is not only environmental friendly, but also a reusable one. After the flushing of a lysis buffer, most cells can be washed away, even those attached to walls. Then the device will be washed for three times by sterilized water before reuse.</P><TABLE><TBODY><TR><TD>Fig. 3-4 Microfluidic array- &quot;Passage Model&quot;</TD></TR></TBODY></TABLE><P><B>Image Capturing &amp; Data analysis</B></P><P>
The green fluorescence of bacteria in chambers is captured every 5 minutes by an inverted microscope (Eclipse Ti, Nikon) with a standard CCD camera manually and usually last more than 7 hrs. (This is really a big challenge for our team and very physically demanding since this is the only available fluorescence we could find. However, we managed to finish this data collecting process anyway. Good jobs, guys！)  
Data are transferred into oscillation curves by ImageJ and software designed by our teammate for digital image batch processing. (More detail for our self-designed software, please refer to protocol.) </P><TABLE><TBODY><TR><TD>Fig. 3-4 Microfluidic array- &quot;Passage Model&quot;</TD></TR></TBODY></TABLE><P>Data are transferred into oscillation curves by ImageJ and software designed by our teammate for digital image batch processing. (More detail for our self-designed software, please refer to <B>Software</B>.</P><P>Next, we are going to <B>Improvement (For a better Glee)</B> to see how our little friends do on stage.</P></DIV></DIV><DIV id="wrapper2"><DIV id="separator"><DIV class="center-block"><H3>Modeling</H3></DIV></DIV><DIV class="center-block-page clearfix"><H4>Mathematical model</H4><P>

We modeled the oscillate gene circuit (without <I>ndh</I> gene) based on the paper &quot;A synchronized quorum of genetic clocks&quot;<SUP>[12]</SUP>. In that, the author characterized the oscillator by using the four delay differential equations (DDEs), which are the basic of our modeling. DDE can perfectly simulate our circuit due to its powerful function about solving the dynamic change in biological processes.</P><P>
In the first two equations,  item is a Hill function equaling to , which describes the delayed reactions about the complex processes in the cells (transcription, translation, maturation, etc.). This Hill function takes the history of the system into account, i.e. the concentration of AHL at the time it binds to LuxR to form the activation complex.
This model does not include an equation for LuxR assuming that it is constitutively produced at a constant level. All parameters are normalized for the convenience of analysis.</P><H4>Modeling in MATLAB</H4><P>We used MATLAB to help us solve the DDEs and draw the graphs. In addition, we developed a GUI user interface after finishing the main DDE code. The user interface makes people easier to modify and adjust the parameters without recoding the .m files and any MATLAB language knowledge. We called this software &quot;Gene OS&quot; which is short for Gene oscillate simulator.</P><A href="https://2013.igem.org/Team:XMU-China/Software" style="font-size:20px">See more in Software</A></DIV></DIV><DIV id="wrapper"><DIV id="separator"><DIV class="center-block"><H3>Applications</H3></DIV></DIV><DIV class="center-block-page clearfix"><P>
After travelling a long way of circuit construction, you may wonder what we are going to do with those sparkling bacteria. Of course much thought was given, too, to make our oscillation circuit both showy and substantial as a safety guard in daily life. Here we are going to show you the potential of those shinny soldiers as<B style="font-size:20px"> biosensor, cancer cell killer and gene sinusoidal signal generator.</B></P></DIV><DIV class="center-block-page clearfix"><H4>Biosensor</H4><P>
To date there appears a great number of biosensors that are used for detecting certain factors, such as small organic molecules or metal ions. And we've noticed that each year many iGEM teams relate their projects to biosensors. Biological detection has strength in its low detection limit, high sensitivity and bioaffinity when compared with chemical detection methods. We can use the same gene sensing circuit to detect various metal ions or other determinant via changing different promoters which responds to a specific target molecule.
Compared with traditional single-cell biosensor, our oscillatory system biosensor has three advantages. 
1) Synchronized colonies will be more sensitive to concentration changes of the target molecule and the changes in fluorescence strength are easier to observe than single cell biosensors. 
2) Single cell biosensor usually works, or &quot;switches on&quot; when the concentration of the specific molecules which can be responded by the promoter reaches to the lowest value. Sometimes this kind of detection is limited when we want to narrow the range of detection. Because the oscillation will happen only when concentration of certain molecule is between two value, the maximum and the minimum, it can dwindle the detection range of the determinant. 
3) Oscillatory system is more robust and stable when used as a biosensor. These advantages will promote oscillatory gene circuits a better use in biosensing.</P><B>1. Arsenic sensing array</B><P>Jeff Hasty from UCSD used the same kind of oscillate circuit to engineer two kinds of arsenic-sensing macroscopic biosensors. These circuits are rewired by a native arsenite-responsive promoter, which is repressed by ArsR in the absence of arsenite. Due to the different mechanism, the sensor outputs are not the same. Figure 1 shows two constructed sensing modules and Figure 2 compares the outputs of two sensors<SUP>[1]</SUP>.</P><P>Figure 1.Two different constructed sensing modules.</P><P>Figure 2. Two different outputs of corresponding biosensors.</P><B>2. Cytotoxin sensor</B><P>Due to the accuracy and specific condition in the field of bio-medicine, the oscillatory biosensor needs to meet the requirement such as synchronization. For example, all cells expected to twinkle at the same time only when cytotoxin existed. In addition, the period or amplitude of the oscillation needs to indicate the concentration of the cytotoxin. Our circuit has the potential to accomplish this kind of job. By further exploration in the relationship between oscillatory parameters such as period and amplitude with other environmental factors such as concentration of antibiotic or cytotoxin, we may create a oscillatory biosensor which can sense different kinds of biological toxin.</P></DIV><DIV class="center-block-page clearfix"><H4>Cancer cell killer</H4><B>1.Cancer cell stalking </B><P>
Nowadays cancer becomes one of the greatest enemy of our human being. Our main methods used to diagnosis of cancer are based on tissue and organ level, such as MRI, CT, PET, etc. These are mainly physical diagnostic methods.
With the development of science and technology, we create various methods into molecule level concentrated on molecular recognition and signal transduction of tumor marker, such as AFP(alpha fetoprotein), CEA (carcino-embryonic antigen), PSA(Prostate Specific Antigen) and so on. Level of these special molecules in human bodies has become the main basis of diagnosis of cancer. These are mainly chemical diagnostic methods.
So how about using synthetic biology to do something for diagnosis of cancer? Our project can be a potential biological method to assist diagnosis of cancer.
ROS is reactive oxygen species in cells, which has been proved to be associated with several neurological disorders including Alzheimer's disease (AD) and Parkinson's diseases<SUP>[2]</SUP><SUP>[3]</SUP><SUP>[4]</SUP>. Concentration of ROS indicates the level of cell metabolism. Normally the stronger cell' metabolism is, the more ROS it will produce. In the recent decade scientist found that the occurrence of cancer is closely related to the level of ROS. But it is not a simple positive correlation as we expect<SUP>[5]</SUP>.</P><P>Figure 4. ROS Level</P><P>From this picture we can get to know it clearly. As we know, we use oxide stress to describe the level of ROS in our cell. There is a threshold of oxide stress which is the critical value between normal cell and cancer cell. As a normal cell, its oxide stress is lower than the critical value while the ROS production value is over the threshold. On the one hand, when the oxide stress in our internal environment increases, normal cells may mutate to cancer cells if the ROS value is higher than the threshold value. On the other hand, cancer cells may die because of the accumulation of ROS. So the key point is, when ROS value higher than the maximum, we can kill the cancer cell while ROS value lower than the threshold we may push normal cells to change into cancer cells. So the level of ROS is in a specific range which we can measure using chemical methods.
Think about a proper range of ROS's concentration. Is it familiar to you as we mentioned a similar problem when talking about advantages of oscillatory system as biosensors? H<SUB>2</SUB>O<SUB>2</SUB> is a kind of ROS. The oscillation will happen only when the concentration of is H<SUB>2</SUB>O<SUB>2</SUB> between the maximum and the minimum. By regulating our circuits to let it respond in the proper concentration range which matches the ROS range of cancer cells, we can use the oscillatory system as a biological cancer detector. </P><P>
In our experiments, we found that we can regulate flow rate and some parameters of microfluidic chip such as the depth of the well, the width of the channel and the concentration of H<SUB>2</SUB>O<SUB>2</SUB> to change the period and amplitude of the oscillation. So by changing parameters of microfluidic chip, we can get a proper concentration range of H<SUB>2</SUB>O<SUB>2</SUB> if the oscillation works. From then on, we can extract some cancer tissue to detect the ROS concentration in the microfluidic chips or get some suspected tissue on chips and diagnose whether it contains cancer cell.</P><P><B>2	Periodical Killer</B><I>2.1 Drug delivery</I>
Our circuit also has a great potential application in periodic drug delivery. It is amazing when we realize that the oscillate circuit can control the drug to release in a specific period and the concentration of the drug is related to the amplitude. It is well-known that nowadays drug-release is a very hot research field. Many efforts are focusing on how to modify or delivery drug using materials such as synthetic polymers, liposome or nano shell which can respond to differences between different intracellular environments. Our oscillatory gene circuits provide a more biofriendly way for drug release. Our future plan is to make some deep exploration in this field.<I>2.2 Protein Synthesis</I>
What else, a resonance may happen among the oscillators which results in a peak synthesis of protein under some conditions. This phenomenon can be used to increase the yield of some medical proteins such as antitoxic serum and antibody<SUP>[9]</SUP><SUP>[10]</SUP>.<I>2.3 Prevention pathogen</I>
In the applications of controlling pathogens, quorum sensing system plays an important role in regulating growth and reputation of microorganism. So we could restrain quorum sensing system to control microorganism to make our life more healthy and nice. The most important part in the metabolism of pathogens controlled by quorum sensing system is the synthesis of toxin and the formation of membrane system. If we can restrain these two kinds of effect may we decrease some potential diseases resources and improve the effects of antitoxins. What's more, it is a particularly attractive point to prevent pathogens through restraining their quorum sensing system instead of killing them directly as it offers the prospect of combating effectively infections resistant to all existing drugs. As we all know, just treat diseases via using antibotics or antitoxin can help these pathogens improve their resistance to various drugs, which is very dangerous for us human being. To date, although we've found that different fungus form different signal molecules, the quorum sensing system almost takes part in cell morphological transformation and information communication. We think our oscillatory gene circuits provide a very useful and meaningful model to promote more and more research in this field<SUP>[11]</SUP>.</P></DIV><DIV class="center-block-page clearfix"><H4>Gene sinusoidal signal generator</H4><P>
With the development of science and technology, computer had been here and there with larger RAM, faster CPU and smaller volume. All these changes owe to VLSI （Very Large Scale Integrated Circuites）, which consists of different electronic component.
Just image what if we can control cell as a biocomputer. Cellular computer is a device consists of different kinds genetically engineered bacteria, which can do the calculation, such as add and subtract like a normal computer. To finish such a device, scientists need various gene circuits to complete different operations. These circuits are just like electronic component working in an electronic computer.
At the beginning of this century, biocomputer is still a crazy dream far more than a possible blueprint. But now scientists have had the technology to make this &quot;impossible mission&quot; comes true. In recent months, scientists from Stanford constructed a new &quot;bio-crystal valve&quot; by DNA and RNA, which can do the logical operations in the living cells<SUP>[6]</SUP>.
Two months later, scientists from MIT used less than 3 gene devices to do the digital operation in division, logarithmic and square root. Thus, constructing gene circuits as electronic devices have become possible. From their research, their cellular calculator which uses analog computation has more advantages in computing efficiency in part count, speed and energy consumption compared with digital computation<SUP>[7]</SUP>.
</P><P>Figure 1: Different kinds of log-domain analog computation.</P><P>Due to the fact that our circuits can generate a steady oscillation, it is possible that using our circuits as a sinusoidal signal generator.
Signal generators, such as function generator (including sinusoidal signal generator), are electronic devices that generate repeating or non-repeating electronic signals. As an important part of basic circuit studies, they are generally used in designing, testing, troubleshooting, and repairing electronic or electroacoustic devices. 
What's more, Li Guang-bi from TUST reported that they have constructed two kinds of electronic circuits which well simulated the gene oscillator. This indicates that our circuit can develop to a new kind of gene sinusoidal signal generator<SUP>[8]</SUP>.</P><P>Figure 2.This picture shows a kind of electronic circuit generating the oscillation wave.</P><P>Figure 3.This graph shows the generated oscillation wave.</P></DIV><DIV class="center-block-page clearfix"><H4>Future Version Trilogy</H4><P>
We blueprint our next step developing oscillatory gene circuits in following steps:<B>Step One
Complete our project by cleaning every obstacle on our way to oscillation. </B>
Confirm that every part is functioning as expected;
Amplify the effect of different copy numbers;
Complete the comparison of different strains' effect on oscillation;
Complete the comparison of different EGFP &amp; sfGFP effect on oscillation;
Confirm most suitable microfluidic parameters for oscillation;
Optimize our model</P><P><B>Step Two
We will make our oscillation a controllable one by integrating the catalase gene into the circuit.</B>
Catalase is a common enzyme found in nearly all living organisms exposed to oxygen. It catalyzes the decomposition of hydrogen peroxide to water and oxygen. It is a very important enzyme in protecting the cell from oxidative damage by reactive oxygen species (ROS). Since our lab is studying <I>Shewanella oneidensis</I>, and its catalase protein has showed outstanding performance in catalyzing H<SUB>2</SUB>O<SUB>2</SUB>. So we thought we could integrate the catalase gene into our circuit and under an artificial inducible promoter, by which we can decide when to start this catalase gene and stop the oscillation.</P><P><B>Step Three
Realize everything we imagined in APPLICATION!</B>
Make our circuit function as biosensor, cancer cell killer and gene sinusoidal signal generator.</P><DIV class="center-block-page clearfix"><B><P>Reference</P></B><P>
1. Prindle, A., et al., A sensing array of radically coupled genetic 'biopixels'. <I>Nature</I> 481, 39-44 (2012).
2. Gaggelli, E. Kozlowski, H. Valensin, D. Valensin &amp; G. Copper ,Homeostasis and Neurodegenerative Disorders (Alzheimer's,Prion, and Parkinson's Diseases and Amyotrophic Lateral Sclerosis). <I>Chem. Rev.</I> 106, 1995–2044 (2006).
3. Scott, L. E. Orvig, C. Medicinal Inorganic Chemistry Approaches to Passivation and Removal of Aberrant Metal Ions in Disease. <I>Chem. Rev.</I> 109, 4885–4910 (2009).
4. Zheng, Z. Q. White, C. Lee, J. Peterson, T. S. Bush, A. I. Sun, G. Y. Weisman, G. A. &amp; Petris, M. J. Altered Microglial Copper Homeostasis in a Mouse Model of Alzheimer's Disease. <I>J. Neurochem.</I> 114, 1630–1638 (2010).
5. Dunyaporn T., Jerome A. &amp; Peng H., Targeting cancer cells by ROS-mediated mechanisms: a radical therapeutic approach? <I>Nature Reviews Drug Discovery</I> 8, 579-591 (2009).
6. Bonnet J, Yin P, Ortiz ME, Subsoontorn P &amp; Endy D, Amplifying Genetic Logic Gates. <I>Science</I> 340, 599-603 ,2013.
7. Ramiz D., Timothy K. Lu et al. Synthetic analog computation in living cells. <I>Nature</I> 497, 619-+ (2013).
8. Guang bi Li et al, Electronic Ciircuit Model Construction of A Synthetic Gene Network. <I>Journal of Tianjin University of Science &amp; Technology</I> 26, 22-25 (2011).
9. Hasty, J., Dolnik, M., Rottschafer, V. &amp; Collins, J. J. Synthetic gene network for entraining and amplifying cellular oscillations. <I>Physical Review Letters</I> 88, 148101, (2002).
10. Elowitz, M. B. &amp; Leibler, S. A. Synthetic oscillatory network of transcriptional regulators. <I>Nature</I> 403, 335 (2000).
11. Yan C., Quorum--Sensing System in Microbes and Its Application, School of Resources and Environment, Yuxi Normal University，Yuxi，Yunnan 653100.
12. Danino, T., Mondragon-Palomino, O., Tsimring, L. &amp; Hasty, J., A synchronized quorum of genetic clocks. <I>Nature</I> 463, 326-330 (2010).</P></DIV></DIV></DIV><DIV id="wrapper1"><DIV id="separator"><DIV class="center-block"><H3>Parts</H3></DIV></DIV><DIV class="center-block-page clearfix"><H4>Table 1. The Bio-bricks used in the study</H4><TABLE class="MsoTableGrid" border="1" cellspacing="0" cellpadding="0" width="950"><TBODY><TR><TD width="105" rowspan="2"><P><B>Part</B></P></TD><TD width="102" rowspan="2"><P><SPAN><B>Backbone</B></SPAN></P></TD><TD width="106" rowspan="2"><P><SPAN><B>Type</B></SPAN></P></TD><TD width="99" rowspan="2"><P><SPAN><B>Location</B></SPAN></P></TD><TD width="161" colspan="2"><P><SPAN><B>Size
(bp)</B></SPAN></P></TD><TD width="377" rowspan="2"><P><SPAN><B>Description</B></SPAN></P></TD></TR><TR style="mso-yfti-irow:1;height:7.85pt"><TD width="81"><P><SPAN><B>part</B></SPAN></P></TD><TD width="81"><P><SPAN><B>backbone</B></SPAN></P></TD></TR><TR style="mso-yfti-irow:2"><TD width="105"><P><SPAN><SPAN>BBa_K546000</SPAN></SPAN></P></TD><TD width="102"><P><SPAN>pSB1C3</SPAN></P></TD><TD width="106"><P><SPAN>Signaling</SPAN></P></TD><TD width="99"><P><SPAN><SPAN lang="EN-US">2013-P1-12D</SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN>1964</SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN>2070</SPAN></SPAN></P></TD><TD width="377"><P><SPAN><SPAN>Lux pL
controlled LuxR with lux pR autoinducing LuxI (lva tag)- AHL.</SPAN></SPAN></P></TD></TR><TR style="mso-yfti-irow:3"><TD width="105"><P><SPAN><SPAN>BBa_I763020</SPAN></SPAN></P></TD><TD width="102"><P><SPAN style="mso-bookmark:
_Hlk367271772"><SPAN>pSB1C3</SPAN></SPAN></P></TD><TD width="106"><P><SPAN style="mso-bookmark:
_Hlk367271772"><SPAN>Intermediate</SPAN></SPAN></P><P><SPAN style="mso-bookmark:
_Hlk367271772"><SPAN>RBS-GFP-TT</SPAN></SPAN></P></TD><TD width="99"><P><SPAN><SPAN lang="EN-US">2013-P3-11H</SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN>914</SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN>2070</SPAN></SPAN></P></TD><TD width="377"><P><SPAN><SPAN>RBS-GFP
(+LVA Tag)-Terminators</SPAN></SPAN></P></TD></TR><TR style="mso-yfti-irow:4"><TD width="105"><P><SPAN><SPAN>BBa_F2621</SPAN></SPAN></P></TD><TD width="102"><P><SPAN>pSB1A2</SPAN></P></TD><TD width="106"><P><SPAN>Signaling</SPAN></P></TD><TD width="99"><P><SPAN><SPAN lang="EN-US">2013-P2-21F</SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN>1158</SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN>2079</SPAN></SPAN></P></TD><TD width="377"><P><SPAN><SPAN>3OC6HSL
Receiver Device</SPAN></SPAN></P></TD></TR><TR style="mso-yfti-irow:5;height:39.75pt"><TD width="105"><P><SPAN><SPAN>BBa_K546001</SPAN></SPAN></P></TD><TD width="102"><P><SPAN><SPAN>pSB1C3</SPAN></SPAN></P></TD><TD width="106"><P><SPAN><SPAN>Device</SPAN></SPAN></P></TD><TD width="99"><P><SPAN><SPAN lang="EN-US">2013-P1-12F</SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN>2135</SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN>2070</SPAN></SPAN></P></TD><TD width="377"><P><SPAN><SPAN>AIIA</SPAN></SPAN></P><P><SPAN><SPAN>AHL Reporter
and Quencher</SPAN></SPAN></P></TD></TR><TR><TD width="105"><P><SPAN>BBa_J04450</SPAN></P></TD><TD width="102"><P><SPAN><SPAN>pSB4K5</SPAN></SPAN></P></TD><TD width="106"><P><SPAN><SPAN>Reporter</SPAN></SPAN></P></TD><TD width="99"><P><SPAN><SPAN><SPAN>2013-P5-5G</SPAN></SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN>1069</SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN>3419</SPAN></SPAN></P></TD><TD width="377"><P><SPAN><SPAN>RFP Coding
Device</SPAN></SPAN></P></TD></TR><TR style="mso-yfti-irow:7;height:13.15pt"><TD width="105"><P><SPAN><SPAN>BBa_J04450</SPAN></SPAN></P></TD><TD width="102"><P><SPAN><SPAN>pSB3T5</SPAN></SPAN></P></TD><TD width="106"><P><SPAN><SPAN>Reporter</SPAN></SPAN></P></TD><TD width="99"><P><SPAN><SPAN>2013-P5-7C</SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN>1069</SPAN></SPAN></P></TD><TD width="81"><P><SPAN><SPAN lang="FR">3241</SPAN></SPAN></P></TD><TD width="377"><P><SPAN><SPAN lang="FR">RFP Coding Device</SPAN></SPAN></P></TD></TR><TR><TD width="105"><P><SPAN><SPAN>BBa_J23022</SPAN></SPAN></P></TD><TD width="102"><P><SPAN>BBa_J23006</SPAN></P></TD><TD width="106"><P><SPAN>Composite</SPAN></P></TD><TD width="99"><P><SPAN>2013-P5-3H</SPAN></P></TD><TD width="81"><P><SPAN>234</SPAN></P></TD><TD width="81"><P><SPAN lang="FR">2356</SPAN></P></TD><TD width="377"><P><SPAN lang="FR">[key3d][TT]</SPAN></P></TD></TR></TBODY></TABLE><TABLE width="100%" border="1" cellspacing="0"><B><P>
    Favorite XMU-China 2013 iGEM Team Parts
  </P></B><TBODY><TR><TH scope="col">W/N</TH><TH scope="col">Name</TH><TH scope="col">Type</TH><TH scope="col">Description</TH><TH scope="col">Designer</TH><TH scope="col">Length</TH></TR><TR><TD>W</TD><TD>BBa_K1036000</TD><TD>Composite</TD><TD>lux pL controlled luxR with lux pR controlled ndh (LVA-tag) coding for NADH dehydrogenase II</TD><TD>Shengquan Zeng Zhaopeng Cheng</TD><TD>2687</TD></TR><TR><TD>W</TD><TD>BBa_K1036001</TD><TD>Coding</TD><TD>the ndh gene coding for respiratory NADH dehydrogenase II</TD><TD>Shengquan Zeng Zhaopeng Cheng</TD><TD>1366</TD></TR><TR><TD>W</TD><TD>BBa_K1036003</TD><TD>Composite</TD><TD>lux pL controlled luxR with lux pR controlled gfp (LVA-tag)</TD><TD>Xin GuoXi Xi</TD><TD>4052</TD></TR><TR><TD>W</TD><TD>BBa_K1036006</TD><TD>Composite</TD><TD>lux pL controlled luxR with lux pR controlled sfgfp (with LVA-tag)</TD><TD>Zhaopeng ChengXiyu Wu</TD><TD>4046</TD></TR></TBODY></TABLE><TABLE width="100%" border="1" cellspacing="0"><B><P>
    XMU-China 2013 iGEM Team Parts Sandbox
  </P></B><TBODY><TR><TH scope="col">W/N</TH><TH scope="col">Name</TH><TH scope="col">Type</TH><TH scope="col">Description</TH><TH scope="col">Designer</TH><TH scope="col">Length</TH></TR><TR><TD>W</TD><TD>BBa_K1036002</TD><TD>Composite</TD><TD>gfp (with LVA-tag) under plux R control</TD><TD>Xin Guo Xi Xi</TD><TD>2866</TD></TR><TR><TD>W</TD><TD>BBa_K1036004</TD><TD>Reporter</TD><TD>sfgfp with LVA-tag</TD><TD>Zhaopeng ChengXiyu Wu</TD><TD>753</TD></TR><TR><TD>W</TD><TD>BBa_K1036005</TD><TD>Composite</TD><TD>sfgfp (with LVA-tag) under plux R control</TD><TD>Zhaopeng ChengXiyu Wu</TD><TD>2880</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="footer-menu"><DIV class="center-block clearfix"><UL id="second-menu"><LI><A href="https://2013.igem.org/Team:XMU-China">Home</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Team1">Team</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Software">Software</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Human Practice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Safety">Safety</A></LI></UL><DIV class="copyright">© Copyright © 2013.XMU-China iGEM All rights reserved.</DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:XMU-China/Content3">http://2013.igem.org/Team:XMU-China/Content3</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:XMU-China/Content3" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:XMU-China/Content3" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:XMU-China/Content3&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:XMU-China/Content3&amp;oldid=330014" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>