"
Page
Discussion
View source
History
teams
Log in
 
Team:XMU-China/Content microfluidic
From 2013.igem.org
LinkUp - Multipurpose HTML Template
MICROFLUIDIC & MICROSCOPY
Microfluidics (Opera Houses)
A Microfluidic array to cells is like an opera house to a glee that provides a cell culture environment and observation stage. To accompany the observation of microfluidic we used the fluorescence microscope, just like mass media to photograph the oscillation. We call them M & M. What a lovely couple.
Why M&M?
If we want to capture gene expression dynamics and distributions reported by fluorescence, we need to create a chemostat that provide a near-constant environment for the growth of bacteria. The near-constant environment mainly includes two aspects: continuous fresh media to ensure the optimum growth state and constant small populations of bacteria to ensure single-cell resolution in observation. Steady growth state is needed because the oscillation curve is expected to be a near simple harmonic one. Besides, bacteria tend to form biofilm which will lead to the overlapping of cells and create a false impression that the fluorescence increases, and that's why we have to prevent this to maintain a constant bacteria population with single-cell resolution.
What we mentioned above might have covered all aspects we need to consider for communication in a single colony, but if we still want to observe the coupling between colonies we have to create an array that contains many colonies and allows H2O2 to pass through freely.
Among many different techniques, microfluidic array seems to be our best choice with following advantages: 1) Precisely control of flowing rate and cell populations when equipped with syringe pumps;
2) Cells behavior including fluorescence strength can be captured by high resolution microscopy;
3) Trap numbers and dimensions can be adapted to different experimental need;
4) Productive: fewer resources and less time are required when compared with macroscopic cultures.
Design of Microfluidic Array
1.Pretests in "Swimming Pool"
Before our experiments on the microfluidic array we described before, a series of pretests were done on a chip whose structure looks like many swimming pool on the ground (Fig. 3-1).
Fig. 3-1 A "Swimming pool" chip in black field; those white squares (100×100×100μm) are chambers for bacteria.
From the sketch (Fig. 3-2) we can see that the chamber is like a swimming pool where bacteria swim happily in the fresh LB that running from up above, and newly generated cells would be washed away.
Fig. 3-2 Diagram for swimming pool model's working mechanism
A "swimming pool" chamber model would have a lager AHL concentration when compared with a "passage" chamber because the former one has a minor dilution. According to a published research that the oscillation period will be shorter as the AHL concentration increases. So we assume that it will be more time-consuming to use this "swimming pool" model in pretests.
However, this chamber is about 100 μm deep and could grow E. coli far more than monolayer. As time goes by, the oscillation in a single colony signaling by AHL could never go back to the original baseline, and the oscillation curves have a tendency like the following graph (Fig. 3-3).
Fig. 3-3 Simulation curve for our oscillation in "Swimming Pool"
2.Advanced "Passage"
According to what we have discussed above, the microfluidic device should be able to provide flowing fresh media and maintain a constant bacteria population with single-cell resolution. Because "swimming pool" has a cell stacking problem, we designed a new passage-like device as shown below (Video 1).
Video 1
Channels and Chambers
This device is composed by two parts: the flowing channels and near 3,000 square trapping chambers, and their layout is like there are many passages with symmetry rooms distributed on both sides, so we call this array "passage" model. Firstly, cells are loaded from the cell port. When the loading stops, most cells would remain in channels, but if the loading cell density is dense enough, many cells would be pushed into chambers and form a monolayer of Escherichia coli growing in it. This is the time for running LB to get in. Fresh LB is loaded from the media port with Tween 20* and appropriate antibiotics. Flowing LB does not only permit a constant nutrition and exponentially growing cells for more than two days (that's the longest time we have tried), but also washes away newly generated cells to sustain a certain cell density, which is crucial for the determination of fluorescence strength.
Size
It has been stated before in the Mechanism part that most quorum sensing systems have a critical cell density requirement, so the trap size is designed neither too big nor too small to capture enough cells and allows for nutrients to permeate to every corner of the chamber.
You may also notice that the height of the chamber is extremely small, which is really demanding for the researcher's skills, but it's important for the observation. The oscillation could only be seen when the height of the chamber is in a particular range as our modeling predicted for many complicated reasons. For more details, please refer to the Modeling part.
Material
Polydimethylsiloxane (PDMS), a transparent polymer material, is chosen to build the devise because of its excellent breathability, which is necessary for the permeation of H2O2. Once H2O2 diffused outside cells, it can travel from colony to colony and synchronize them.
Our device is not only environmental friendly, but also a reusable one. After the flushing of a lysis buffer, most cells can be washed away, even those attached to walls. Then the device will be washed for three times by sterilized water before reuse.
Fig. 3-4 Microfluidic array- "Passage Model"
Image Capturing & Data analysis
Fig. 3-5 Microfluidic array- "Passage Model"
The green fluorescence of bacteria in chambers is captured every 5 minutes by an inverted microscope (Eclipse Ti, Nikon) with a standard CCD camera manually and usually last more than 7 hrs. (This is really a big challenge for our team and very physically demanding since this is the only available fluorescence we could find. However, we managed to finish this data collecting process anyway. Good jobs, guys！)
Data are transferred into oscillation curves by ImageJ and software designed by our teammate for digital image batch processing. (More detail for our self-designed software, please refer to protocol.)
Data are transferred into oscillation curves by ImageJ and software designed by our teammate for digital image batch processing. (More detail for our self-designed software, please refer to Software .
Next, we are going to Exploration (For a better Glee) to see how our little friends do on stage.
Home
Team
Project
Notebook
Software
Human Practice
Safety
Contact us
© Copyright 2013.XMU-China iGEM All rights reserved.
Retrieved from " http://2013.igem.org/Team:XMU-China/Content_microfluidic "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
