"
Page
Discussion
View source
History
teams
Log in
 
Team:XMU-China/Content5
From 2013.igem.org
LinkUp - Multipurpose HTML Template
Content
1 Circuit Construction
1. Material
2. PCR Polymerase Chain Reaction (PCR)
3. Digestion
4.  Ligation
5. Transformation
2 SDS-PAGE
1. Preparation
2. Bacteria culture
3. Protein electrophoresis
3 Microfluidic Protocol
1. Pre-procession of DH5α
2. Pre-procession of LB
3. Wetting the chip & Capturing images
1. Circuit Construction
Material
1.1 Culture medium: LB (Luria-Bertani)
LB medium: 5g Yeast Extract, 10g Tryptone and 10g NaCl, add ddH2O to 1L.
LB solid medium: 5g Yeast Extract, 10g Tryptone, 10g NaCl and 15 g agar, add ddH2O to 1L.
1.2 Antibiotic
Antibiotic
Stock conc.
Final conc.
Solvent
Sterilization
Ampicillin
100 mg/ml
50 ug/ml
ddH2O
filtration
Chloramphenicol
25 mg/ml
25 ug/ml
Absolute ethyl alcohol
 
Tetracycline
40 mg/ml
20 ug/ml
75% absolute ethyl alcohol
 
Kanamycin
25 mg/ml
25 ug/ml
ddH2O
filtration
Stock in -20°C
1.3 Plasmid (Biobrick)
Part
Backbone
Type
Location
Size (bp)
BBa_K546000
pSB1C3
Signaling
2013-P1-12D
1964
2070
BBa_I763020
pSB1C3
Intermediate RBS-GFP-TT
2013-P3-11H
914
2070
BBa_F2621
pSB1A2
Signaling
2013-P2-21F
1158
2079
BBa_K546001
pSB1C3
Device
2013-P1-12F
2135
2070
Ba_J04450
pSB4K5
Reporter
2013-P5-5G
1429
3409
BBa_J04450
pSB3T5
Reporter
2013-P5-7C
1429
3241
BBa_J04450
pSB3T5
Reporter
2013-P3-3H
1429
3241
1.4 Strain
Strain
Application
Source
E. coli DH5α
cloning
Lab stock
E. coli BL21(DE3)
 
Lab stock
E. coli BL21
 
From Taiwan
E. coli MG1655
 
 
1.5 Primer
Part
Backbone
Type
Location
Size (bp)
part
backbone
BBa_K546000
pSB1C3
Signaling
2013-P1-12D
1964
2070
BBa_I763020
pSB1C3
Intermediate RBS-GFP-TT
2013-P3-11H
914
2070
BBa_F2621
pSB1A2
Signaling
2013-P2-21F
1158
2079
BBa_K546001
pSB1C3
Device
2013-P1-12F
2135
2070
Ba_J04450
pSB4K5
Reporter
2013-P5-5G
1429
3409
BBa_J04450
pSB3T5
Reporter
2013-P5-7C
1429
3241
 
 
 
2013-P3-3H
 
 
2. PCR Polymerase Chain Reaction (PCR)
2.1 General PCR
Reaction system:
Template
1 μl
Forward Primer
2 μl
Reverse Primer
2 μl
dNTP Mixture
8 μl
10*Ex Buffer
5 μl
Ex Taq
0.5 μl
ddH2O
31.5 μl
Total Volume
50 μl
Reaction condition:
94 oC
4 min
 
94 oC
1 min
30 cycles
55 oC
0.5 min
72 oC
1.5 min
72 oC
10 min
 
4 oC
∝
 
2.2 Fusion PCR
Every biobrick has four sites: EcoR I, Xba I, Spe  I and Pst I. When two parts were ligated after digestion by Xba I and Spe I, an 8 bp-size fragment (TACTAGAG) will appear between two parts. If we want to add a fragment in the end of a CDS, for example a fast-degradation tag LVA, the additional 8 bp-size fragment will affect the expression of the CDS. Fusion PCR will resolve this problem. Firstly, two pair primers are designed: F1 and R1 for gene A, F2 and R2 for gene B. The 5'-end of R1 and F2 have about 21-bp homologous region. Secondly, gene A and gene B are amplified via PCR independently. Thirdly, the purified gene A and gene B are fused together via PCR with F1 and R2.
2.3 Colony PCR
Reaction system:
Template
colony
Forward   Primer
1   μl
Reverse   Primer
1   μl
dNTP   Mixture
4   μl
10*PCR   Buffer
2.5   μl
rTaq
0.25   μl
ddH2O
16.25 μl
Total   Volume
25   μl
Reaction condition:
94 oC
4 min
 
94 oC
1 min
25 cycles
55 oC
0.5 min
72 oC
1.5 min
72 oC
10 min
 
4 oC
∝
 
3. Digestion
3.1 Digestion for confirmation
Reaction System:
Single Digestion
Double Digestion
Plasmid
5
Plasmid
5
EcoR   I
1
EcoR   I
1
/
 
Pst I
1
10*H   buffer
1
10*M   buffer
1
ddH2O
3
ddH2O
2
Total   Volume
10
Total   Volume
10
Reaction condition: 37°C for 1 hours.
3.2 Digestion for ligation
Reaction System:
A + B
B + A
Volume (μl)
Volume (μl)
A
B
A
B
30
50
EcoR I
EcoR I
Xba I
Spe I
3
5
Spe I
Xba I
Pst I
Pst I
3
5
10*H buffer
10*M buffer
10*M buffer
10*H buffer
4
8
ddH2O
0
12
Total Volume
40
80
4. Ligation
 
Volume (μl)
Insert
V1
Vector
V2
10*ligation   buffer
1
T4   Ligase
1
Total Volume
10
5. Transformation
5.1  Chemical Competent Cell Preparation (E.coli)
Activate the E.coli strain on LB-plate from glycerol stock under the condition of 37oC for 12 hours.
Pre-culture single-colony in 10 mL LB medium under the condition of 37oC, 200 rpm for 12 hours.
Add 200 μl pre-culture into 20 mL LB medium under the condition of 37oC, 200 rpm for 2 hours.
Place on ice for 30 min. Aliquot into sterile 1.5 mL tubes and spin down at 4000 rpm for 10 min at 4 oC, then discard supernatant.
Gently re-suspend each pellet with 1 mL0.1 M ice cold CaCl2-MgCl2 solution. Centrifuge 4000 rpm for 10 min and discard supernatant. 
Re-suspend each pellet on ice in 200 μL 0.1 M ice cold CaCl2-MgCl2 solution
5.2 Chemical Competent Cell Preparation (E.coli with plasmid(s))
When you prepare the competent cell of E.coli with plasmid(s), you need add half-concentration of appropriate antibiotic(s) in LB.
5.3 General Transformation
Add 1 μL plasmid or 10 μL ligation system into 100 μL fresh chemical competent cells and mix gently. Leave on ice for 30 minutes.
Heat shock tubes in 42oC water bath for 90 seconds, then place them on ice for 10 minutes immediately.
Incubate for 1~2 hr with 200 rpm shaking at 37°C. When the antibiotic of vector is ampicillin, the time of incubate is 1 hr.
Add 600 μL fresh LB medium into each tube.
Spread 100 μL on an LB solid medium plate containing the appropriate antibiotics and incubate overnight at 37°C
Two or three plasmids Transformation
Usually we just transform one plasmid into host cell, but sometimes we need transform two or three plasmids. Plasmid compatibility is considerable. Generally different replication origin means they are compatibility. We transform plasmids one by one from the low copy number to high copy number.
2. SDS-PAGE
Preparation
2.1 Bacterial strains:
The experimental strains:E.coli
2.2 The main instruments and reagents
Instruments
autopipets, electrophoresis meter, Vortex oscillator, Labscan Scanner, decolorization table, desk centrifuge, temperature metal bath, electronic balance, acidometer, enzyme-labeled instrument, superclean bench, constant  temperature  vibrator,  shaking table.
Reagents
LB liquid medium: yeast powder 5.0 g ;Tryptone 10.0 g ;sodium chloride  10.0 g;deionized water to produce 1000 ml medium.
LB solid medium: Put 1.5 g agar into every 100 ml liquid medium.
Preparation of electrophoretic solution
ddH2O;4 X Running Gel Buffer (PH 8.8,1.5 M Tris-HCl);4 X Running Gel Buffer (pH 6.8,1.5 M Tris-HCl);Monomer Solution;10% sodium dodecylsulphate(SDS);10% Ammonium Persulfate(APS);N,N,N',N'- tetramethylethylenediamine(TEMED);loading buffer;Tank Buffer;industrial alcohol; Marker ; dying liquor;destainer.
(1) Preparation of 4 X Running Gel Buffer (pH 8.8,1.5 M Tris-HCl): 36.3 g Tris-base is dissolved in 150 ml deionized water. Then use HCl solution to adjust the pH of solution to 8.8. At last use deionized water to produce 200 ml solution.
(2) Preparation of 4 X Running Gel Buffer (pH 6.8,1.5 M Tris-HCl): 3.0 g  Tris-base is dissolved in 40 ml deionized water. Then use HCl solution to adjust the pH of solution to 6.8. At last use deionized water to produce 100 ml solution.
(3)Preparation of 10% SDS: 10g SDS is weighted before being dissolved . Use  distilled water to produce 100ml solution.
(4)Preparation of 10% APS: 1 g APS is weighted before being dissolved. Use  distilled water to produce 10ml solution.
(5) Preparation of 10X Tank Buffer: 30.28 g Tris-base, 144.13 g glycine and 10 g SDS is weighed before being dissolved. Use distilled water to produce 1000ml solution. Use the solution as-prepared 100 ml mixed with 900 ml distilled water to dilute 10 times.
(6) Preparation of Kaumas Coomassie brilliant blue staining solution: 0.5 g Coomassie Brilliant blue R250 is weighted before being dissolved in 800 ml industrial alcohol. After dissolving, 140 ml acetic acid is added and Use  distilled water to produce 2000ml solution
(7) Preparation of destainer: 200 ml industrial alcohol is mixed with 50 ml acetic acid. Use distilled water to produce 1000ml solution.
Preparation of 10% gel.
2.2. Bacteria culture
Strain activation
Pick Escherichia coli colony from LB plate and subculture it into 20 ml conical beaker with 50 ml LB broth followed anaerobic culture 12 h under 37 °C . Then culture 1% concentration into conical beaker with 50 ml LB broth to subculture, which is the activated germ liquid.
Strain culture
Subculture 1% activated germ liquid into every conical beaker with LB broth. Then culture the microbe in the shaking table ,under 37 °C ，200 rpm.
2.3 Protein electrophoresis
Preparation of Samples
- The samples to be tested are cultured in the Basal Medium with appropriate antibiotics, and take 200 μL bacteria liquid to determine its OD600 at appropriate time.
- Dilute or concentrate the next 200 μL bacteria liquid in order to let the OD600 equals to 4.0 while the computational formula is the actual OD600 * 200=2.0 * X, and X presents the total volume of the bacteria liquid after being diluted or concentrated while its unit is μL as well.
- Add 30 μL diluted or concentrated liquid into corresponding 1.5 mL centrifugal tubes, then mix up them with 10 μL loading buffer.
- Put these centrifugal tubes into metal bath and heat them in 100 °C  in around 5 to 8 min, then centrifuge them at the speed of 13000 rpm for 5 min, the supernatant is what we need.
Manufacture Albumen Gel
- Prepare a clear centrifuge tube in the capacity of 50 mL, and make running gel, high concentration one, following the formula below. Then mix up them and pour the mixture into a glass pane.
Running Gel
Final Gel Concentration (5 mL; 1 ea ;1.0 mm   thick; 10%)
ddH2O
2 mL
4 X Running Gel Buffer (pH 8.8,1.5 M Tris-HCl)
1.25 mL
Monomer Solution
1.65 mL
10% SDS
50 uL
10% Ammonium Persulfate
50 uL
TEMED
5 uL
- Add some absolute alcohol to planish the top of gel. There will be approximately 60 min for its solidification.
- After solidification, pour out the alcohol and make stacking gel following the formula below. Then mix them up, add the solution onto the running gel in the glass panes until it being filled up with the gel. Insert a clean comb into stacking gel. Wait for about 40 min for stacking gel solidification.
Running Gel
Final Gel   Concentration (5 ml; 1 ea ;1.0 mm thick; 10%)
ddH2O
2.7 ml
4   X Running Gel Buffer (PH 8.8,1.5 M Tris-HCl)
1.17 ml
Monomer   Solution
0.6 ml
10%   SDS
45 ul
10%   Ammonium Persulfate
45 ul
TEMED
5 ul
Electrophoresis
- Take out the glass pane with finished gel and then fasten it in an electrophoresis tank. Add some 1×Tank Buffer to detect whether liquid leak or not.     
- Take out the comb slowly and use pipette to add approximately 10 to 20 μL processed samples into the wells in stacking gel.
- Add 1×Tank Buffer until the liquid level is above the platinum line in the electrophoresis tank.
- Cover up the electrophoresis tank and connect it with the electrophoresis device. Set the program 80 V-30 min in one step and 120 V- 60 min in two step then start it up.
- When the blue marker band run to the bottom of running gel, stop the device.
Dyeing (Colloidal Coomassie Brilliant Blue)
- Take out the gel and put it into a clean petri dish. And add appropriate Commassie Blue Staining Solution. Please make sure that the solution can cover all the gel.
- Put the petri dish onto the orbital shaker and dye for approximate 1 h.
- Pour out the staining solution then add enough destaining solution. Destain about 30 min.
- Renew the destaining solution for about 2 or 3 times until the blue background of gel being taken off.
- Pour out the destaining solution and add appropriate water to clear it.
Scanning
Scan the processed gel and save the picture for analysis.
3. Microfluidic Protocol
Sterilization of microfluidic chips Pre-procession of DH5α
1. Briefly, 200 μl of an overnight culture was diluted in 20 ml of LB medium plus antibiotics the day of the experiment.
(Cm, 10 μl; tet, 25 μl for two plasmids gfp+aiiA and three plasmids gfp+aiiA+ndh)
2. When cells reached an optical density (OD600 nm) of 0.1 (for about 2 hr or longer), 12 ml cells were spun down and resuspended in 1.2 ml of fresh media and loaded into the device.
Pre-procession of LB
Filter 20 ml LB through 0.20 mm filter, and then add 0.075% Tween 20 (15 μl) to it. This step prevents the cells from sticking to chip walls without any noticeable harm to the cells. Depending on your experiment, make sure to add antibiotics to the media. So take the chip, your bacteria and your LB together to the florescent microscope.
Wetting the chip & Capturing images
1. Place the chip under microscope at 4×magnification and carefully examine the chip for dirt and collapsed channels. If the chip looks good proceed to wetting.
2. Wash syringe with 75% ethanol, MQ and LB in proper sequence, and repeat each cleaning for three times
3. Attach the plastic pipe about 15 to 20 cm in length to both syringes, and then fill one syringe with fresh LB, while the other one is filled with cells. Make sure there are no bubbles in the syringe or the pipe. Bubbles can be removed by flicking the syringe or the pipe with your finger.
4. Carefully insert the pipes into two ports: media and cell, and set syringes steady on the pumps.
5. Set the flow rate of the left pump (LB) to >0.115 ml/hr to wet the chip until there is no bubble can be seen on the chip (At 20×magnification). Then slow the flow rate of this pump which should be much slower than that of the right one. Media removal from the surface of the chip at the cell port can be best accomplished using a kimwipe.
6. Set the flow rate of the right pump (Cells) to 10~20 μl/hr to let cells fill in trapping region on the chip. When the cell density has reached a proper number, media flow should be 11.5~115 μl/hr and the right pump should be stopped.
7. Allow the cells to grow in the traps for 3-5 doublings, depending on the living condition and cell types this should take 1-2 h.
8. Set up the microscope software for manual image acquisition. Turn on the laser power and adjust the light to Blue. Choose an appropriate field with suitable cell density without any dirt or other disturb and keep capture an image every 5 minutes. 
Before the image taking, make sure you focus the right layer.
When the timer is beeping, make sure you press the button twice, or you can use the time on computer or your cell phone to record the time.
Home
Team
Project
Notebook
Software
Outreach
Safety
© Copyright © 2013.XMU-China All rights reserved.
Retrieved from " http://2013.igem.org/Team:XMU-China/Content5 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
