"
Page
Discussion
View source
History
teams
Log in
 
Team:XMU-China/Exploration
From 2013.igem.org
导航栏
Home
2011 XMU-China iGEM
2012 XMU-China iGEM
2013 XMU-China iGEM
Team
Our Team
Our Univercity
Our City
Our Album
Attribution
Project
Background
Three Plasmids
Microfluidic
Exploration
Modeling
Applications
Parts
Software
Gene OS
ImageMe
Examples & confirmation
Notebook
Wetlab Journal
Protocol
Calender
Outreach
Abstract
Human Practice
Collaborations
Surveys
Acknowledgement
Safety
LinkUp - Multipurpose HTML Template
Exploration (For a better glee)
Every famous star knows that there's no easy approach to fame and success. So does the way to realize the glee dream of E. coli. Their first performances are not perfect oscillation, let alone synchronized. So to help them we made some changes in hosts and copy numbers. Besides this, there was also an exploration for fast degrading tag LVA.
Strain casting- Changing the singing style
Their performance still couldn't be a hit, so our bacteria glee was thinking, may it be the BL21(DE3) style is not popular anymore? So they tried several kinds of styles, including DH5a, BL21 (wild type) and MG1655. Here is a table show some main reasons we took into consideration, including growing rate (crucial for quorum sensing), strain type and gene expression level.
Table 4-1 Strain choosing
Strain
Type
GrowingRate
Reasons we choose it
DH5α
K
Slow
(1)Plasmids construction was done in this strain;(2)Belong to K strain like MG1655
BL21 (DE3)
B
Faster
(1)competent in transformation and protein expression; (2)Ndh gene is originally expressed in BL21 (DE3)
BL21 (wild type)
B
Fastest
A BL21 strain with less artificial mordification, so it is expected to grow faster than BL21 (DE3)
MG1655
K
Data not shown
Shown oscillation in published research
Here comes the data of different strains by variety kinds of confirmation methods. (All plasmids have already been confirmed by agarose gel electrophoresis and sequencing.)
Strain: BL21 (DE3)
1. Single Plasmid A1, B and C Confirmation by SDS-PAGE In accordance with our circuit construction, following target proteins are expected(Table 4-2):
Table 4-2 Target Proteins in oscillation circuit
CDS
Size (bp)
Protein
Size (kDa)
luxR
750
LuxR
27.5
gfp-lva
753
GFP-LVA
27.6
aiiA-lva
783
aiiA-LVA
28.7
luxI-lva
612
LuxI-LVA
22.4
ndh-lva
1335
NDH-LVA
48.9
According to this SDS-PAGE and MALDI-TOF-MS we can confirm that the single pSB1C3-gfp-luxI plasmid can express GFP and LuxI proteins (Fig. 3-8). (Although the protein size seems a little bigger than we expected, but we assume it was the problem of Marker.) And pSB3T5-aiiA and pSB4K5-ndh plasmid do not have their target gene expressed without inducer AHL.
FIG.4-1 SDS-PAGE OF  THREE SINGLE PLASMIDS IN BL21(DE3)
2. Double Plasmids (A1 & B) & Triple Plasmids (A1 & B & C) Confirmation by SDS-PAGE From the SDS-PAGE (Fig. 4-2) we can tell
1) GFP and aiiA's existence are confirmed by MALDI-TOF-MS; 2) LuxI's existence can be confirmed by SDS-PAGE image; 3) Ndh gene is innate in BL21 (DE3), so we assume ndh had been produced since the band is darker in strain with three plasmids than others.
FIG.4-2 SDS-PAGE OF  SINGLE, DOUBLE AND TRIPLE PLASMIDS IN BL21(DE3)
Fig. 4-3  SDS-PAGE of plasmid pSB4K5-ndh in BL21
with 10-3 mM AHL induction
For fear this confirmation for ndh is not solid enough, we performed another experiment with only plasmid C and plasmid C with additional AHL as inducer. And the later one showed a broader band, so plasmid C function is proved (Fig. 4-3).
3. Double Plasmids A1 & B Confirmation by Microfluidic So we tried the BL21 (DE3) strain with our circuit on two different microfluidic arrays to test if it works well.
Swimming pool model
In spite of the effect of growing bacteria, a slight oscillation was still observed. However, this data was not solid enough since we only captured one period of oscillation because of the limited microscope using time (Fig. 4-4).
FIG. 4-4  AN OBSERVED CIRCLE OF OSCILLATION IN BL21 (DE3)
The oscillation period is about 120 minutes with a tendency of increasing fluorescence strength. However, it still proved that our circuit could function as an oscillation. BL21 (DE3) strain with three plasmids (pSB1C3-gfp-luxI & pSB3T5-aiiA & pSB4K5-ndh) has also been tried on this array once, but no obvious phenomenon could be captured since the cell density was not large enough for quorum sensing. There are also many other factors involved, for example trap size and flowing rate. The strict requirements for the observation of oscillation have already discussed Microfluidic part.
Passage Model
In passage model the distribution of bacteria can be seen in Fig. 4-5. And Fluorescence strength in every row were analyzed by ImageMe (Self-made software, please refer to Software) and an oscillation can be seen, even though the effect of noise was great (Fig. 4-6).
Fig. 4-5 Distribution of BL21 (DE3) with plasmids A and B
Fig. 4-6 Fluorescence strength analysis of BL21 (DE3) with plasmids A and B on Microfluidic (three rows are analyzed seperately)
Strain: DH5α
The second strain we tried was DH5a, in which we built our circuit initially. DH5a with plasmid A1 and B had been tested on both microfluidic arrays, but showed no obvious oscillation on either of them. (Data can be seen in Journal) So we performed a SDS-PAGE experiment to check functions of proteins (Fig. 4-7).
FIG.4-7 SDS-PAGE OF  SINGLE, DOUBLE AND TRIPLE PLASMIDS IN DH5a
From the SDS-PAGE data we can assume that the biggest problem of DH5α strain is the LuxI protein was not function properly as we expected, so the positive feedback couldn't be formed. Other target proteins have been well expressed.
Strain: MG1655
Since MG1655 strain had shown oscillations in many published researches and is known as a faster growing strain. So we got this strain and began our experiments. However, this strain grew even slower than DH5a and we guess this was the problem of long way transportation without low temperature protection.
Strain: BL21 (Wild type)
For faster growing rate, we found BL21 (wild type) and thought this may worth a try. Double Plasmids A1 & B Confirmation by Microfluidic (Fig. 4-8 (A))
                        FIG. 4-8 (A) FIGURE 4-8 RFU CURVE OF MICROFLUIDIC ASSAY ON PASSAGE ARRAY OF DOUBLE-PLASMID IN BL21:  (A) PSB1C3-GFP-LUXI & PSB3T5-AIIA
In BL21 (wild type), however, the oscillation had not been improved.
Conclusion for Host comparison
1.Different strains have different protein expression systems, so when you want build a circuit and hope it could function well; find a suitable host. 2.Growing rate is important for observation on microfluidic, if there is not enough bacteria inside a chamber, the quorum sensing promoter cannot be activated and the circuit won't work.
Copy Number Changing
It turned out that style does matter, but there are still other factors worth exploring, for example, the copy numbers for different plasmids. The quantities of different proteins have delicate relationships. If we can't make this relationship right, the oscillation won't be good. So we tried to lower the copy number of plasmid A1 (pSB1C3-gfp-luxI) by changing its backbone form pSB1C3 into p3H. Here comes A2 (p3H-gfp-luxI). The reason we lower it rather than higher it is because the GFP seemed remain longer than we expected and elongated the period.
Double Plasmids A2 & B Confirmation by Microfluidic
FIG. 4-8 (A) FIGURE 4-8 RFU CURVE OF MICROFLUIDIC ASSAY ON PASSAGE ARRAY OF DOUBLE-PLASMID IN BL21:  (A) PSB1C3-GFP-LUXI & PSB3T5-AIIA, (B) P3H-GFP-LUXI & PSB3T5-AIIA
No obvious oscillation improvement was shown in low-copy number oscillator (Fig. 4-8 (B)). However, there is something intriguing in this graph, the fluorescence strength in graph (A) is much lower than that in graph (B). We can assume that when copy number of luxI is high, the aiiA expression was also enhanced and the degradation of AHL became faster, then the fluorescence is lower, this explains why the pB1C3 backbone strain always showed a lower RFU.From this experiment, we can see that copy number does play an important role in oscillation, and its function still require further exploration.
Fluorescence Detection on Micro-well Plate
In order to help our glee finish their song, we did another experiment in 96 micro-well plate, a simulation of swimming pool model, and tested the fluorescence strength continuously. The total fluorescence on the graph A kept growing as what we found on microfluidic, however, the increasing rate curve (Fig. 4-9 (B)) showed an oscillation! Now, we guess the chief criminal is the fast degrading tag LAV we attached to our target protein didn't work well.
Figure 4-9  RFU curve of double-plasmid in BL21 with different copy number (A) and their increasing rate curve of RFU (B)
For now we have found that host, copy number and LVA-tag all have impact on oscillation. For further experiments please refer to Future work (Keep Practicing).
Home
Team
Project
Notebook
Software
Human Practice
Safety
Contact us
© Copyright 2013.XMU-China iGEM All rights reserved.
Home
Team
Project
Software
Notebook
Outreach
Safety
XMU-China | Welcome
Background
Three Plasmids
Microfluidic
Exploration
Modeling
Applications
Parts & Future
Retrieved from " http://2013.igem.org/Team:XMU-China/Exploration "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
