<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_XMU-China_Content_exploration"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:XMU-China/Content_exploration">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:XMU-China/Content_exploration&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU-China/Content_exploration&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU-China/Content_exploration&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:XMU-China/Content_exploration" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:XMU-China/Content exploration</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>LinkUp - Multipurpose HTML Template</TITLE></P><DIV id="wrapper"><DIV id="separator"><DIV class="center-block"><H3>Exploration (For a better glee)</H3></DIV></DIV><DIV class="center-block-page clearfix"><P>Every famous star knows that there's no easy approach to fame and success. So does the way to realize the glee dream of E. coli. Their first performances are not perfect oscillation, let alone synchronized. So to help them we made some changes in hosts and copy numbers. Besides this, there was also an exploration for fast degrading tag LVA.
</P><H4>Strain casting- Changing the singing style</H4><P>Their performance still couldn't be a hit, so our bacteria glee was thinking, may it be the BL21(DE3) style is not popular anymore? So they tried several kinds of styles, including DH5a, BL21 (wild type) and MG1655. Here is a table show some main reasons we took into consideration, including growing rate (crucial for quorum sensing), strain type and gene expression level.</P><TABLE width="80%" border="1" cellspacing="0"><CATION>Table 4-1 Strain choosing</CATION><TBODY><TR><TH scope="col">Strain</TH><TH scope="col">Type</TH><TH scope="col">GrowingRate</TH><TH scope="col">Reasons we choose it</TH></TR><TR><TD>DH5α</TD><TD>K</TD><TD>Slow</TD><TD>(1)Plasmids construction was done in this strain;(2)Belong to K strain like MG1655</TD></TR><TR><TD>BL21 (DE3)</TD><TD>B</TD><TD>Faster</TD><TD>(1)competent in transformation and protein expression; (2)Ndh gene is originally expressed in BL21 (DE3)</TD></TR><TR><TD>BL21 (wild type)</TD><TD>B</TD><TD>Fastest</TD><TD>A BL21 strain with less artificial mordification, so it is expected to grow faster than BL21 (DE3)</TD></TR><TR><TD>MG1655</TD><TD>K</TD><TD>Data not shown</TD><TD>Shown oscillation in published research</TD></TR></TBODY></TABLE><P>Here comes the data of different strains by variety kinds of confirmation methods. (All plasmids have already been confirmed by agarose gel electrophoresis and sequencing.)</P><B><P>Strain: BL21 (DE3)</P></B><P> 
1. Single Plasmid A1, B and C Confirmation by SDS-PAGE
In accordance with our circuit construction, following target proteins are expected(Table 4-2):
</P><TABLE width="80%" border="1" cellspacing="0"><CATION>Table 4-2 Target Proteins in oscillation circuit</CATION><TBODY><TR><TH scope="col">CDS</TH><TH scope="col">Size (bp)</TH><TH scope="col">Protein</TH><TH scope="col">Size (kDa)</TH></TR><TR><TD><I>luxR</I></TD><TD>750</TD><TD>LuxR</TD><TD>27.5</TD></TR><TR><TD><I>gfp-lva</I></TD><TD>753</TD><TD>GFP-LVA</TD><TD>27.6</TD></TR><TR><TD><I>aiiA-lva</I></TD><TD>783</TD><TD>aiiA-LVA</TD><TD>28.7</TD></TR><TR><TD><I>luxI-lva</I></TD><TD>612</TD><TD>LuxI-LVA</TD><TD>22.4</TD></TR><TR><TD><I>ndh-lva</I></TD><TD>1335</TD><TD>NDH-LVA</TD><TD>48.9</TD></TR></TBODY></TABLE><P>According to this SDS-PAGE and MALDI-TOF-MS we can confirm that the single pSB1C3-gfp-luxI plasmid can express GFP and LuxI proteins (Fig. 3-8). (Although the protein size seems a little bigger than we expected, but we assume it was the problem of Marker.) And pSB3T5-aiiA and pSB4K5-ndh plasmid do not have their target gene expressed without inducer AHL. </P><TABLE style="margin-left: 220px"><TBODY><TR><TD>FIG.4-1 SDS-PAGE OF  THREE SINGLE PLASMIDS IN BL21(DE3)</TD></TR></TBODY></TABLE><P>2.	Double Plasmids (A1 &amp; B) &amp; Triple Plasmids (A1 &amp; B &amp; C) Confirmation by SDS-PAGE
From the SDS-PAGE (Fig. 4-2) we can tell
</P><P>
1)	GFP and aiiA's existence are confirmed by MALDI-TOF-MS;
2)	LuxI's existence can be confirmed by SDS-PAGE image;
3)	Ndh gene is innate in BL21 (DE3), so we assume ndh had been produced since the band is darker in strain with three plasmids than others.
</P><TABLE style="float: left; margin-left: 40px;"><TBODY><TR><TD>FIG.4-2 SDS-PAGE OF  SINGLE, DOUBLE AND TRIPLE PLASMIDS IN BL21(DE3)</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Fig. 4-3  SDS-PAGE of plasmid pSB4K5-ndh in BL21 with 10<SUP>-3</SUP> mM AHL induction
</TD></TR></TBODY></TABLE><P>For fear this confirmation for ndh is not solid enough, we performed another experiment with only plasmid C and plasmid C with additional AHL as inducer. And the later one showed a broader band, so plasmid C function is proved (Fig. 4-3).</P><P>3. Double Plasmids A1 &amp; B Confirmation by Microfluidic
So we tried the BL21 (DE3) strain with our circuit on two different microfluidic arrays to test if it works well. </P><B><P>Swimming pool model</P></B><P>In spite of the effect of growing bacteria, a slight oscillation was still observed. However, this data was not solid enough since we only captured one period of oscillation because of the limited microscope using time (Fig. 4-4).</P><TABLE style="margin-left: 220px"><TBODY><TR><TD>FIG. 4-4  AN OBSERVED CIRCLE OF OSCILLATION IN BL21 (DE3) 
</TD></TR></TBODY></TABLE><P>The oscillation period is about 120 minutes with a tendency of increasing fluorescence strength. However, it still proved that our circuit could function as an oscillation. 
BL21 (DE3) strain with three plasmids (pSB1C3-gfp-luxI &amp; pSB3T5-aiiA &amp; pSB4K5-ndh) has also been tried on this array once, but no obvious phenomenon could be captured since the cell density was not large enough for quorum sensing. There are also many other factors involved, for example trap size and flowing rate. The strict requirements for the observation of oscillation have already discussed Microfluidic part.
</P><B><P>Passage Model</P></B><P>In passage model the distribution of bacteria can be seen in Fig. 4-5. And Fluorescence strength in every row were analyzed by ImageMe (Self-made software, please refer to Software) and an oscillation can be seen, even though the effect of noise was great (Fig. 4-6).</P><TABLE><TBODY><TR><TD>Fig. 4-5 Distribution of BL21 (DE3) with plasmids A and B
</TD><TD>Fig. 4-6 Fluorescence strength analysis of BL21 (DE3) with plasmids A and B on Microfluidic (three rows are analyzed seperately)
</TD></TR></TBODY></TABLE><B><P>Strain: DH5α</P></B><P>The second strain we tried was DH5a, in which we built our circuit initially. DH5a with plasmid A1 and B had been tested on both microfluidic arrays, but showed no obvious oscillation on either of them. (Data can be seen in Journal) So we performed a SDS-PAGE experiment to check functions of proteins (Fig. 4-7). </P><TABLE style="margin-left: 280px"><TBODY><TR><TD>FIG.4-7 SDS-PAGE OF  SINGLE, DOUBLE AND TRIPLE PLASMIDS IN DH5a
</TD></TR></TBODY></TABLE><P>From the SDS-PAGE data we can assume that the biggest problem of DH5α strain is the LuxI protein was not function properly as we expected, so the positive feedback couldn't be formed. Other target proteins have been well expressed.</P><B><P>Strain: MG1655</P></B><P>Since MG1655 strain had shown oscillations in many published researches and is known as a faster growing strain. So we got this strain and began our experiments. However, this strain grew even slower than DH5a and we guess this was the problem of long way transportation without low temperature protection.</P><B><P>Strain: BL21 (Wild type)</P></B><P>For faster growing rate, we found BL21 (wild type) and thought this may worth a try. Double Plasmids A1 &amp; B Confirmation by Microfluidic (Fig. 4-8 (A))</P><TABLE><TBODY><TR><TD>                        FIG. 4-8 (A)
FIGURE 4-8 RFU CURVE OF MICROFLUIDIC ASSAY ON PASSAGE ARRAY OF DOUBLE-PLASMID IN BL21: 
(A) PSB1C3-GFP-LUXI &amp; PSB3T5-AIIA

</TD></TR></TBODY></TABLE><P>In BL21 (wild type), however, the oscillation had not been improved.</P><B><P>Conclusion for Host comparison</P></B><P>1.Different strains have different protein expression systems, so when you want build a circuit and hope it could function well; find a suitable host.
2.Growing rate is important for observation on microfluidic, if there is not enough bacteria inside a chamber, the quorum sensing promoter cannot be activated and the circuit won't work.
</P><B><P>Copy Number Changing</P></B><P>It turned out that style does matter, but there are still other factors worth exploring, for example, the copy numbers for different plasmids. The quantities of different proteins have delicate relationships. If we can't make this relationship right, the oscillation won't be good. So we tried to lower the copy number of plasmid A1 (pSB1C3-gfp-luxI) by changing its backbone form pSB1C3 into p3H. Here comes A2 (p3H-gfp-luxI). The reason we lower it rather than higher it is because the GFP seemed remain longer than we expected and elongated the period.</P><P>Double Plasmids A2 &amp; B Confirmation by Microfluidic</P><TABLE><TBODY><TR><TD>FIG. 4-8 (A)
FIGURE 4-8 RFU CURVE OF MICROFLUIDIC ASSAY ON PASSAGE ARRAY OF DOUBLE-PLASMID IN BL21: 
(A) PSB1C3-GFP-LUXI &amp; PSB3T5-AIIA, (B) P3H-GFP-LUXI &amp; PSB3T5-AIIA
</TD></TR></TBODY></TABLE><P>No obvious oscillation improvement was shown in low-copy number oscillator (Fig. 4-8 (B)). However, there is something intriguing in this graph, the fluorescence strength in graph (A) is much lower than that in graph (B). We can assume that when copy number of luxI is high, the aiiA expression was also enhanced and the degradation of AHL became faster, then the fluorescence is lower, this explains why the pB1C3 backbone strain always showed a lower RFU.From this experiment, we can see that copy number does play an important role in oscillation, and its function still require further exploration.</P><B><P>Fluorescence Detection on Micro-well Plate</P></B><P>In order to help our glee finish their song, we did another experiment in 96 micro-well plate, a simulation of swimming pool model, and tested the fluorescence strength continuously. The total fluorescence on the graph A kept growing as what we found on microfluidic, however, the increasing rate curve (Fig. 4-9 (B)) showed an oscillation! Now, we guess the chief criminal is the fast degrading tag LAV we attached to our target protein didn't work well.</P><TABLE><TBODY><TR><TD>Figure 4-9  RFU curve of double-plasmid in BL21 with different copy number (A) and their increasing rate curve of RFU (B)  

</TD></TR></TBODY></TABLE><P>For now we have found that host, copy number and LVA-tag all have impact on oscillation. For further experiments please refer to <A href="https://2013.igem.org/Team:XMU-China/Parts1#future"><B>Future work</B></A> (Keep Practicing).</P></DIV></DIV><DIV id="footer-menu"><DIV class="center-block clearfix"><UL id="second-menu"><LI><A href="https://2013.igem.org/Team:XMU-China">Home</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Team1">Team</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Software">Software</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Human Practice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:XMU-China/Safety">Safety</A></LI><LI style="margin-left: 400px"><A href="https://2013.igem.org/Team:XMU-China/Team1#Attribution">Contact us</A></LI></UL><DIV class="copyright" style="margin-right: -150px">© Copyright 2013.XMU-China iGEM All rights reserved.</DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:XMU-China/Content_exploration">http://2013.igem.org/Team:XMU-China/Content_exploration</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:XMU-China/Content_exploration" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:XMU-China/Content_exploration" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:XMU-China/Content_exploration&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:XMU-China/Content_exploration&amp;oldid=350359" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>