<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Shenzhen_BGIC_ATCG_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Shenzhen_BGIC_ATCG/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Shenzhen_BGIC_ATCG/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Shenzhen_BGIC_ATCG/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Shenzhen BGIC ATCG/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="box" style="width: 700px; margin-left: 137px; padding: 5px; border: 3px solid #000; background-color: #fe2b33;"><DIV id="template" style="text-align: center; font-weight: bold; font-size: large; color: #f6f6f6; padding: 5px;">
This is a template page. READ THESE INSTRUCTIONS.
</DIV><DIV id="instructions" style="text-align: center; font-weight: normal; font-size: small; color: #f6f6f6; padding: 5px;">
You are provided with this team page template with which to start the iGEM season.  You may choose to personalize it to fit your team but keep the same &quot;look.&quot; Or you may choose to take your team wiki to a different level and design your own wiki.  You can find some examples <A href="https://2008.igem.org/Help:Template/Examples">HERE</A>.
</DIV><DIV id="warning" style="text-align: center; font-weight: bold; font-size: small; color: #f6f6f6; padding: 5px;">
You <STRONG>MUST</STRONG>have all of the pages listed in the menu below with the names specified.  PLEASE keep all of your pages within your teams namespace.  
</DIV></DIV><TABLE style="color:#1b2c8a;background-color:#0c6;" cellpadding="3" cellspacing="1" border="1" width="62%" align="center"><TBODY><TR><TH align="center"><A href="/Team:Shenzhen_BGIC_ATCG" title="Team:Shenzhen BGIC ATCG">Home</A></TH><TH align="center"><A href="/Team:Shenzhen_BGIC_ATCG/Team" title="Team:Shenzhen BGIC ATCG/Team">Team</A></TH><TH align="center"><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Shenzhen_BGIC_ATCG" class="external text" rel="nofollow">Official Team Profile</A></TH><TH align="center"><STRONG class="selflink">Project</STRONG></TH><TH align="center"><A href="/Team:Shenzhen_BGIC_ATCG/Parts" title="Team:Shenzhen BGIC ATCG/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><A href="/Team:Shenzhen_BGIC_ATCG/Modeling" title="Team:Shenzhen BGIC ATCG/Modeling">Modeling</A></TH><TH align="center"><A href="/Team:Shenzhen_BGIC_ATCG/Notebook" title="Team:Shenzhen BGIC ATCG/Notebook">Notebook</A></TH><TH align="center"><A href="/Team:Shenzhen_BGIC_ATCG/Safety" title="Team:Shenzhen BGIC ATCG/Safety">Safety</A></TH><TH align="center"><A href="/Team:Shenzhen_BGIC_ATCG/Acknowledgment" title="Team:Shenzhen BGIC ATCG/Acknowledgment">Acknowledgment</A></TH></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overall_project"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overall project</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Project_Details"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Details</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Actor_-_signal_peptides"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Actor - signal peptides</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Make-up_Artists_-_introns"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Make-up Artists - introns</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Make-up_Artists_-_Degradation_Peptide"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Make-up Artists - Degradation Peptide</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Editor_-_the_promoters"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Editor - the promoters</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Freezer_-_Sic1"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Freezer - Sic1</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Tool_-_Microfluidic"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Tool - Microfluidic</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overall_project"><B>Overall project</B></SPAN></H2><P>This year, our project named &quot;Cell Magic&quot;. We engineered the budding yeast in time level as well as space level. &quot;Cell Magic&quot;, like a movie,can be divided into two parts: the actor related and the time regulation related one. In the first part, our actor, signal peptides, were all made up with colorful clothes by fluorescent proteins. Furthermore, the intron and degradation biobricks as make-up artists can decide how the clothes be matched. Through these steps, our actors may wearing in green, yellow or even mixed, appear in their specific sub-locations of yeast cell.
With respect to the time level, we took advantages of the natural cell cycle in budding yeast to improve the time process to be more suitable for our movie.In general, the movie director, the promoters from five cycline gene, can decide the expression of downstream gene  in G1 and G2 phase phase, respectively. Also, freezer Sic1 help our movie stay a longer time in the G1 phase which is important for the actor performing their stories. And the last tools we utilized is the microfluidics, through which numerous cells can project our &quot;Cell Magic&quot;, thus ensuring our observation by naked eyes.
</P><H2><SPAN class="mw-headline" id="Project_Details"> Project Details</SPAN></H2><H3><SPAN class="mw-headline" id="Actor_-_signal_peptides"> Actor - signal peptides </SPAN></H3><P>　　A target peptide is a short (3-70 amino acids long) peptide chain that directs the transport of a protein to a specific region in cell, including nucleus, mitochondria, endoplasmic reticulums (ERs), chloroplasts, apoplasts, peroxisomes and plasma membrane. Targeting peptide can exists in both N-terminal, C-terminal and internal sequence of a precursor protein. And after transported, some target peptides are cleaved by signal peptidases.
　　In our project we utilized 19 peptides target to 9 sub-locations in yeast cells, and when combined with fluorescent proteins, such region can be marked by different colors.
</P><UL><LI> OMM
</LI></UL><P>　　Mitochondria
　　Though it accounts a small ratio in the cell space, mitochondria possess about 10% to 15% proteins encoded by nuclear genes in eukaryotic organisms. These proteins are synthesized in cytosol and then recognized by the membrane receptors of mitochondria. Translocases in the outer and inner membrane of mitochondria mediate the import and intra-mitochondrial sorting of these proteins. ATP is used as an energy source; Chaperones and auxiliary factors assist in folding and assembly of mitochondrial proteins into their native, three-dimensional structures. 
</P><P>　Figure 1 | Protein-import pathways for mitochondrial proteins. 
　　As shown in figure1, beta-barrel outer-membrane proteins (dark green), precursor proteins (brown) with positively charged amino-terminal presequences and multispanning inner-membrane proteins (blue) with internal targeting signals are recognized by specific receptors of the outer mitochondrial membrane (TOM) translocases Tom20, Tom22 and/or Tom70. The precursor proteins are then translocated through a small Tom proteins of the TOM complex, Tom40 pore, which the TOM complex contains two or three.
</P><UL><LI> peroxisomes 
</LI></UL><P>The import of post-translational matrix protein into peroxisomes depends on either of the two peroxisomal targeting signals (PTS), PTS1 and PTS2. PTS2-driven import is facilitated by a complex in the membrane. Under oleic acid-inducing growth conditions, there is a ternary core complex of approximate 150 kDa in the cytosol, which consists of Fox3p,Pex7p and Pex18p. Fox3p is imported as a dimer, while other two are bind in monomeric forms.
</P><P>
Figure2._Schematical_model_of_the_early_steps_of_PTS2-driven_import.png‎
</P><P>As study mentioned there are four steps involved in PTS2-driven import. The first step is the recognition of dimerized Fox3p by Pex7p through its PTS2 in the cytosol. In a second step, the Pex7p–Fox3p complex interacts with Pex18p, which targets the PTS2 pre-import complex to the peroxisomal membrane. The third step is the docking process, involving the interaction between Pex7p and the integral membrane protein Pex13p. As a final step of these early steps in the PTS2 import cascade, PTS2 receptor dissociation takes place during or after its assembly into large oligomeric complexes containing Pex14p and Pex13p. Pex18p remains at the peroxisomal membrane in the form of a large-molecular-weight complex in conjunction with Pex14p and/or Pex13p, from where it might be released to the cytosol.
</P><P>REFERENCE :【Peroxisomal Targeting of PTS2 Pre-Import Complexes in the Yeast Saccharomyces cerevisiae】
</P><UL><LI> Actin 
</LI></UL><P>In yeast, the cortical actin cytoskeleton seems to specify sites of growth of the cell surface. Because the actin-binding protein ABP1p is associated with the cortical cytoskeleton of Saccharomyces cerevisiae, it might be involved in the spatial organization of cell surface growth. ABP1p is localized to the cortical cytoskeleton and its overproduction causes assembly of the cortical actin cytoskeleton at inappropriate sites on the cell surface, resulting in delocalized surface growth. ABP1p is a novel protein with a 50 amino-acid C-terminal domain, which is very similar to the SH3 domain in the non-catalytic region of nonreceptor tyrosine kinases (including those encoded by the proto-oncogenes c-src and c-abl), in phospholipase C gamma and in alpha-spectrin. They also identified an SH3-related motif in the actin-binding tail domain of myosin-I. The identification of SH3 domains in a family of otherwise unrelated proteins that associate with the membrane cytoskeleton indicates that this domain might serve to bring together signal transduction proteins and their targets or regulators, or both, in the membrane cytoskeleton.
</P><P>REFERENCE: [Homology of a yeast actin-binding protein to signal transduction proteins and myosin-I]
</P><UL><LI> Nucler
</LI></UL><P>Histones are nuclear proteins package DNA into nucleosomes[Youngson, Robert M. (2006).Collins Dictionary of Human Biology. Glasgow: HarperCollins.], and they are responsible for maintaining the shape and structure of a nucleosome. One chromatin molecule is composed of at least one of each core histones per 100 base pairs of DNA.[The Nucleosome: From Genomic Organization to Genomic Regulation.] There are five families of histones known to date, termed H1/H5, H2A, H2B, H3, and H4.[3] H2A is considered a core histone, along with H2B, H3 and H4. Core formation first occurs through the interaction of two H2A molecules.[Lehninger Principles of Biochemistry.] Then, H2A forms a dimer with H2B; the core molecule is complete when H3-H4 also attaches to form a tetramer.
</P><UL><LI> vacuolar membrane
</LI></UL><P>The ZRC1 gene encodes a multicopy suppressor of zinc toxicity in Saccharomyces cerevisiae; however, previously it was reported that the expression of ZRC1 was induced when the intracellular zinc level was decreased. The COT1 and ZRC1 genes of Saccharomyces cerevisiae are structurally related dosage-dependent suppressors of metal toxicity. COT1 confers increased tolerance to high levels of cobalt; ZRC1 confers increased tolerance. 【Interactions between gene products involved in divalent cation transport in Saccharomyces cerevisiae.】Zrc1 has six putative transmembrane domains, and Zrc1-GFP fusion protein was localized to the vacuolar membrane. Zrc1 function as a mechanism to maintain the zinc homeostasis in yeast. 【The Zrc1 is involved in zinc transport system between vacuole and cytosol in Saccharomyces cerevisiae.】
</P><H3><SPAN class="mw-headline" id="Make-up_Artists_-_introns"> Make-up Artists - introns </SPAN></H3><P>In yeast cells, alternative splicing is a common process in before gene transcription. The splicing sites are located in 5'UTR of introns and can by recognized and spliced by some splicesome. Alternative splicing can produce two isoforms from one gene, thus can be used in our project to monitor whether the Sic1 system work efficiently. Here we find two introns with alternative splicing sites in yeast cells named SRCI intron and MER2 intron.
</P><UL><LI>Scr1 and Hub1
</LI></UL><PRE>   SRC1 intron has two 5' splicing site whose efficiency is regulated by protein hub1. Originally, the existence of the intron would produce two mature mRNA in proportion. After engineering, the existence of protein hub1 regulates to the preservation of intron precisely, which would make the following mRNA be expressed normally or not. As for the silencing of wild type gene HUB1, we choose CRISPRi that is comparably easy to use reversibly.
</PRE><P>
　　Alternative splicing substantially increases the gene product diversity and is a major source of cell type differentiation. A good example is the alternative splicing of Saccharomyces cerevisiae SRC1 pre-mRNA, which is promoted by the conserved ubiquitin-like protein Hub1. It can function through binding non-covalently to a conserved element termed HIND in the spliceosomal protein Snu66. Such binding makes the splicesome target sites change and moderately alters spliceosomal interactions. 
　　Hub1 is a ubiquitin-like modifier (UBL) that covalent modify the proteins. Interest enough, it harbors several different to other UBLs in which it possesses a C-terminal double tyrosine motif while others having a GG motif. The Snu66, a tri-snRNP in yeast spliceosome, possesses with two N-terminal HINDs (Hub1-INteraction Domain). The Hub1–HIND interaction comprises a strong salt bridge accompanied by several hydrophobic contacts and high affinity. Such binding modifies the spliceosome rather than modulating the properties of an individual binding partner. Hub1-controlled splicing occurs universally in eukaryotes. SR proteins and hnRNPs involved in spliceosome targeting do not seem to exist in S. Cerevisiae , and thus the Hub1-dependent mechanism may be evolutionarily older. 
Scr1 is a protein in yeast having alternative splicing sites in its intron. The characteristic differential Hub1 dependence of SRC1 alternative splicing requires the tandem arrangement of overlapping 5’ splice sites. The Hub1 binding spliceosome can splice the intron from both downstream 5’ sites as well as the upstream 5’ sites with preference to the former one.
</P><P>Showed in Figure.1 When cutting in the upstream splice sites, the exons flanking around it would be translated as a fusion protein Scr-S. However, when cutting at the down stream one, the left 4bp in intron would result in a frameshift, thus only the forward exon can express named Scr-L.
</P><UL><LI>dCas9 CRISPR  interface system:
</LI></UL><P>CRISPR shorts for Clustered Regularly Interspaced Palindromaic Repeats system, which can be targeted to DNA using RNA, enabling genetic editing of any region of the genome in many organisms.[Targeted genome engineering in human cells with the Cas9 RNA-guided endonuclease .] In the type II CRISPR/Cas system, a ribonucleoprotein complex formed from a single protein (Cas9), a crRNA, and a trans-acting CRISPR RNA (tracrRNA) can carry out efficient crRNA-directed recognition and site-specific cleavage of foreign DNA(CRISPR RNA maturation by transencoded small RNA and host factor RNase III.). After mutated the endonuclease domains of the Cas9 protein, it creates a programmable RNA-dependent DNA-binding protein. The sgRNA consists of three domains: a 20 nt complementary region for specific DNA binding, a 42 nt hairpin for Cas9 binding (Cas9 handle), and a 40 nt transcription terminator derived from S. Pyogenes. After translation, the Cas9 binds to sgRNA to form a protein-RNA complex, which can recognize target sites in the genome sequence and bind to it. Then, it could block RNA polymerase and transciption elongation.
</P><UL><LI>MER2:
</LI></UL><P>There are at least three separate but overlapping meiotic splicing regulatory programs in yeast cell [Integration of a splicing regulatory network within the meiotic gene expression program of Saccharomyces cerevisiae.] [Qiu, Z. R., Shuman, S. &amp; Schwer, B. An essential role for trimethylguanosine RNA caps in Saccharomyces cerevisiae  meiosis and their requirement for splicing of SAE3 and PCH2 meiotic pre-mRNAs]. And the most well-characterized one is regulated by MER1, which encodes an RNA-binding protein (Mer1) that is transcriptionally induced during initiation of the meiotic cycle [ a yeast gene required for chromosome pairing and genetic recombination, is induced in meiosis.]. The primary function of Mer1 is to activate splicing of four single-intron genes, thereby inducing their expression during Meiosis[Munding, E. M. et al.  Integration of a splicing regulatory network within the meiotic gene expression program of Saccharomyces cerevisiae .  Genes Dev.  24, 2693–2704 (2010)]Mer1 protein is the only meiosis-specific factor required for MER2 RNA splicing.  
And the MER2 is distinct with others in alternative splice sites features:
1) First, the MER2 intron is located at a considerable distance (316 nucleotides [nt]) downstream of the translation initiation site. 
2) the distance between the branchpoint sequence (UACUAAC) and the 39 splice site is only 10 nt
3) the 5’ splice site in MER2 (GUUCGU) differs from the consensus in yeast (GUAYGU) by an A-to-U change at the third position
MER2 intron provides cis-acting sequences that are sufficient for Mer1-dependent splicing in vivo.
</P><H3><SPAN class="mw-headline" id="Make-up_Artists_-_Degradation_Peptide"> Make-up Artists - Degradation Peptide </SPAN></H3><P>In yeast cell, protein degraded through several mechanisms, a major one is the ubquitin- pathway. In detail the signal in substrates can be recognized by the enzyme E3, which transport the ubiqutin from E2 to such protein. And through such way, the ubiquited protein would finally be degraded by the proteasome. 
<A href="http://upload.wikimedia.org/wikipedia/commons/5/5b/Ubiquitylation.svg" class="external autonumber" rel="nofollow">[1]</A></P><UL><LI>PEST 
</LI></UL><P>There are two types of cyclins in the budding yeast. One kind contains PEST sequence at C-terminal: Cln1, Cln2, Cln3, Pcl1, Pcl2. Another kind is Clb1~6 which posses 9 conserved amino acid sequence in N-terminal named D-box (destruction box). The D-box are necessary for later kind cyclins degradation in M phase and can function in same way when combined with other proteins. In our study, we utilized the D-box, PEST sequence and ubiquitin-mediated pathway for fluorescent proteins degradation. 【蛋白质的磷酸化作用和泛肽化降解作用与芽殖酵母的调控】
</P><UL><LI>Poly-ubiquitin
</LI></UL><P>Ubiquitin, a highly conserved 76-residue protein, was involved in the ubiquitin–proteasome pathway of protein proteolysis, which is a fundamental way for protein turnover, signal transduction and cell cycle control. For example, the degradation of CDC34 through such way is necessary for S phase start; another instance is that APC\C(anaphase-promoting complex\cyclosome) is degraded through ubiquitination, which is the foundation of M phase beginning. And both activated by Cdc28-cyclins.
</P><P>The mechanism of ubiquitin-mediated protein degradation can be separated into three continuous enzyme catalyze steps. First 1) a ubiquitin is activated by the E1 (ubiquitin-activating enzyme) through a thioester linkage，and then 2)added into the a small ubiquitin-carrier E2. Finally,3) through the E3 ubiquitin protein ligase, this ubiquitin complex was conjugate to the ε-amino group of lysine residues in substrate proteins, forming a glycyllysine isopeptide bond【Roles of ubiquitin-mediated proteolysis in cell cycle control.】In some conditions, even E3 enzymes themselves carry ubiquitin as a thioester 【A family of proteins structurally and functionally related to the E6-AP ubiquitin-protein ligase】
</P><P>In our project, we design a poly-ubiquitin(5 ubiquitin combined) biobricks which can directly degrade the proteins fused with it. 
</P><UL><LI>-D-box
</LI></UL><P>The best studied substrates of ubiquitin- and APC/C-mediated proteolysis are the mitotic cyclins【The role of cyclin synthesis and degradation in the control of maturation promoting factor activity 】Mitosis-exit inducer, a cyclin B, are highly dependent on its 90 residues for ubiquitin-mediated proteolysis. Analysis of its N-terminal region demonstrated the sequence essential for cyclin proteolysis, called ‘destruction box’(D-box). Meanwhile, when cyclin destruction started, substrates containing D-boxes were rapidly poly-ubiquitinated. The consensus motif in B-type cyclins is RXALGXIXN. For much of the cell cycle, the D-box may not be recognized with high affinity, and when it is recognized, the ‘bait’ construct becomes highly unstable. 【The role of the destruction box and its neighbouring lysine residues in cyclin B for anaphase ubiquitin-dependent proteolysis in fission yeast: defining the D-box receptor】
</P><P>When the D-box exists in the protein N-terminal, this protein is recognized more easily by the E1 and then captured into the ubiquitin-mediated degradation pathway. Thus, we combined it to the fluorescent proteins’ N-terminal to fasten their degradation and avoid veiling the following fluorescence.
</P><H3><SPAN class="mw-headline" id="Editor_-_the_promoters"> Editor - the promoters </SPAN></H3><P>Cell cycle is a complex process and can be separated into G0, G1, S, G2, M phase. In each phase, distinct transcription factors help the phase-specific gene express through recognizing their promoters. Thus, these promoters are phase-specific too and can be fused with other genes in order to express such gene in a defined cell cycle phase.
</P><P>As we all know, different proteins are planned to be expressed along the cell cycle. And their mRNAs are transcripted through the specific promoters. These promoters locate at their upstream sequences and can be recognized by the RNA polymerase, which can initiate such process. There are databases containing the upstream 600 amino acid sequence in the upstream of DNA, which can be transcript in the G1, S, G2 and M phase, respectively 
<A href="http://www.genome.jp/kegg-bin/show_pathway?org_name=sce&amp;mapno=04111&amp;mapscale=&amp;show_description=hide" class="external autonumber" rel="nofollow">[2]</A>. Previous study 【Candidate regulatory sequence elements for cell】demonstrated that cell cycle specific promoters posses their conserved five to six base pairs. Thus, we found the high-confidence 600 promoter-containing sequences in database that harbor the paper-mentioned 5/6 bp sequence. Consequently, we pick up the upstream 600bp sequence of cln2, cln3, clb2, clb5 and clb6.
The Clb2 gene is highly expressed in G2 phase, and the genes are very strongly induced by GAL-CLB2, whereas GAL-CLN3 appears somewhat repressive 【Comprehensive Identification of Cell Cycle–regulated Genes of the Yeast Saccharomyces cerevisiae by Microarray Hybridization】
CLN3 are G2 phase-specific gene.
</P><H3><SPAN class="mw-headline" id="Freezer_-_Sic1"> Freezer - Sic1 </SPAN></H3><P>As we known, the yeast cell cycle contains a huge and complex regulatory network in the transcription level, translation level as well as protein level. In our project we utilize Sic1 as a regulator to help elongate G1 phase in yeast cell.
</P><P>The activation of B-type cyclin (Clb5/6)+Cdc28 kinases is a necessary step for initiation of DNA replication in vivo. One of its inhibitor Sic1 can be phosphorylated by the activated Cln+Cdc28, thereby targeted for degradation. Over expression of the gene encoding p40, SIC1, produces cells with an elongated bud morphology(figure1)【An Inhibitor of Yeast Cyclin-Dependent Protein Kinase Plays an Important Role in Ensuring the Genomic Integrity of  Daughter Cells】. SIC1 deletion is viable and causes increasing frequencies of chromosomes broken and lost. The deletion strain segregates out many dead cells, which are primarily arrested at the G2 checkpoint in an asymmetric fashion. Therefore, it has an important role in ensuring genomic integrity, and that this role has a pronounced mother-daughter asymmetry.	
</P><P>After phosphorylation, phospho-Sic1 is specifically recognized by the F-box protein Cdc4, which leads Sic1 being ubiquitinated by the Cdc34±SCF complex (E3). The recognizing and binding by Cdc4 is based on the Sic1’s 9 Cdc4 phospo-degrons (CPDs, figure. a). Several phosphorylation sites contributed to Sic1 instability, with an order of Thr45, Ser76, Thr5,Thr33 and other less significant sites. The immunocipients after selectively phosphorylation as figure e showed suggested that at least six sites phosphorylation is necessary for the Cdc4 recognizing and binding with Sic1. Furthermore, the culture of GAL1-SIC1 constructs strain showed less than 6 sites are phosphorylated is not sufficient for SIC1 degradation in vitro【Multisite phosphorylation of a CDK inhibitor sets a threshold for the onset of DNA replication】.
</P><H3><SPAN class="mw-headline" id="Tool_-_Microfluidic"> Tool - Microfluidic </SPAN></H3><P>Based physically on the chip, microfluidics can automatically control the biology process. In our project, the microfluidics plays a role as the movie theater, for cell cycle control. We designed a channel which can capture the yeast cell and then alternatively inject medium of different concentrations. Therefore, the yeast cell cycle can be synchronized. Furthermore, due to its transparent attribution, the chip can project the fluorescent lights from the yeast cell sub-locations. Consequently, the magic film can be shown to us. 
　　
Microfluidic devices possess many advantages such as large-scale integration, fast analyses, and considerably reduced reagent consumption (Comparative study and improvement of current cell micro-patterning techniques). Nowadays many designs are based on cell loading and culturing in a microfluidic channel with particular trapping structures such as micro-well arrays, and micro-cup arrays and micro-chambers. In our project, we design a cell loading technique based on gas absorption of degassed polydimethylsiloxane (PDMS). Because the balance concentration of gas dissolved in PDMS is proportional to the partial pressure of the gas around it, so that one can degas a piece of PDMS by placing it in vacuum. 
Our cell loading device is a one level PDMS with an array of triangle cavities connected to one or both sides of the main fluidic channel. After degassed, yeast cells in the buffer solutions can be introduced into the device’s triangle cavities by gas absorption therein. Through changing the cell density in the buffer, we can control the number of the yeast cells loaded into each triangle cavity. Statistically, the ratio of cavities with a single cell can reach 40%. Consequently it is convenient to monitor the cell number and cell phenotypic variations because the yeast cells can grow in the micro-triangle-cavities in one layer.
</P><P>Figure 1. Masks for PDMS microfluidic device fabrication.a : Three micro-triangle-cavities at one side of the main channel. b : One single channel for cell culture (100 microcavities). c : Seven parallel channels for different cell culture environments
</P><P>REFRENCE :A Fast Cell Loading and High-Throughput Microfluidic System for Long-Term Cell Culture in Zero-Flow Environments
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Shenzhen_BGIC_ATCG/Project">http://2013.igem.org/Team:Shenzhen_BGIC_ATCG/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Shenzhen_BGIC_ATCG/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Shenzhen_BGIC_ATCG/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/Project&amp;oldid=130713" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>