<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Shenzhen_BGIC_ATCG_stories"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Shenzhen_BGIC_ATCG/stories">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Shenzhen_BGIC_ATCG/stories&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/stories&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/stories&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Shenzhen_BGIC_ATCG/stories" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Shenzhen BGIC ATCG/stories</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="menu"><UL class="menu_navi"><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board1">Consortium</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board2">Promoter Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board3">Degradation Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board4">Targeting Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board5">Regulator Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board6">Splicing Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board7">Microfluidic Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board8">Modeling Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board9">Instructors</A></LI></UL><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board1">The Magic</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board2">Cyclin Promoters</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board3">Reporter Modification</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board4">Degradation Tags</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board5">Targeting Peptides</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board6">Cell Synchronization</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board7">Alternative Splicing Device</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board8">Microfluidic Device</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/modeling#board1">Blueprint</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/modeling#board2">Cell Cycle</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/modeling#board3">Cell Synchronization</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/modeling#board4">Alternative Splicing by CRISPRi</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/modeling#board5">Degradation Rate</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board1">The Magic</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board2">Promoter Verification</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board3">Reporter Locating</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board4">Degradation Rate</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board5">Cell Synchronization</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board6">Alternative Splicing</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/humanpractice#board1">iGEMer Incubator</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/humanpractice#board2">Guest iGEMers</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/humanpractice#board3">APP: iGEMagic</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/humanpractice#board4">Friendship</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/parts#board1">Best BioBricks</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/parts#board2">Devices List</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/parts#board3">Basic Parts</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/parts#board4">Parts Sequencing</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/notes#board1">Timeline</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/notes#board2">Attributions</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/notes#board3">Protocols</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/notes#board4">References</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/notes#board5">Acknowledgements</A></LI></DIV><DIV class="hotspot1"><DIV class="text1"><P>Playing with my eyesaren't you?</P></DIV></DIV><DIV class="hotspot2"><DIV class="text2"><P>Hi I am Dr. Mage!A &quot;budding&quot; yeast cell!</P></DIV></DIV><DIV id="bodymain" class="stories"><DIV class="navleftholder"><UL class="navleft"><LI><A id="navleftsub1" class="navleftsub" href="#board1">The Magic</A></LI><LI><A id="navleftsub2" class="navleftsub" href="#board2">Cyclin Promoters</A></LI><LI><A id="navleftsub3" class="navleftsub" href="#board3">Reporter Modification</A></LI><LI><A id="navleftsub4" class="navleftsub" href="#board4">Degradation Tags</A></LI><LI><A id="navleftsub5" class="navleftsub" href="#board5">Targeting Peptides</A></LI><LI><A id="navleftsub6" class="navleftsub" href="#board6">Cell Synchronization</A></LI><LI><A id="navleftsub7" class="navleftsub" href="#board7">Alternative Splicing Device</A></LI><LI><A id="navleftsub8" class="navleftsub" href="#board8">Microfluidic Device</A></LI></UL></DIV><DIV id="box1" class="box"><H3>The Magic</H3><P>Life, the most brilliant magic in the universe, celebrated with the ability of reproduction and evolution. And the magic, is based on a sophisticated mechanism, called cell cycle. Most genetic reactions in a cell are regulated, directly or indirectly, by cell cycle. As we wish to build artificial lives, it is worthwhile to learn from the exquisite design of  creature, to make use of cell cycle. As a pioneer study, we try to performing a &quot;Cell Magic&quot;, by capturing cell cycle with fantastic reporters. By grasping the usage of cell cycle tools, we are promised to direct refined actions in a cell. For instance, when producing Paclitaxel (harmful to centrosome in M phase) with a cell factory, we may able to transport it out before it comes detrimental. </P><P>So this is our project, the &quot;Cell Magic&quot;, to engineer two versions of cell cycle based magic in both budding yeast and <I>E. coli</I>. We will present the blueprint of our stories by the yeast version. </P><H4>Version #1, Degradation Tag X, Targeting Peptides X, Cell Cycle Regulator X</H4><P>Promoters of cyclins, which can initial transcription in specific phase in a cell cycle, are selected to produce different fluorescence proteins in G1/SG2/M phases respectably. However, in our first version of story, with low natural degradation rate, fluorescence proteins would remain in cell after their promoters stop working. So colors for each phase cannot be distinguished, as the figure shows.</P><P>Figure. Version #1, cell magic without degradation tags, targeting peptides and cell cycle regulator. Animation and genetic circuits.</P><H4>Version #2, Degradation Tag √, Targeting Peptides X, Cell Cycle Regulator X</H4><P>In the second version, degradation tags are added to the reporters to solve this problem. All fluorescence proteins are thought to be degraded in 15 minutes which is far less than the period of a cell cycle.</P><P>Figure. Version #2, cell magic with degradation tags, without targeting peptides and cell cycle regulator. Animation and genetic circuits.</P><H4>Version #3, Degradation Tag √, Targeting Peptides √, Cell Cycle Regulator X</H4><P>While we are not sure if degradation tags can work as expected, we try to turn the time magic into a time-space one. Reporters are located to different cell structures by targeting peptides. Different phase with different color shines in different organelle, that is the third version.</P><P>Figure. Version #3, cell magic with degradation tags and targeting peptides, without cell cycle regulator. Animation and genetic circuits.</P><H4>Version #4, Degradation Tag √, Targeting Peptides √, Cell Cycle Regulator √</H4><P>To capture the magic macroscopically, we developed a microfluidic device to synchronize all cells into the same phase (G1). So synchronization devices was added to the forth version. </P><P>Figure. Cell magic in microfuidics device.</P><P>To make the magic more fantasy, alternative splicing device acting synchronously with the synchronization device (a translational unit of a cyclin regulator), will splice the targeting peptide tail when cell are being synchronized to G1 phase, so all cells turn green; while synchronization device not working, targeting peptide remains in the reporter so only mitochondria turn green. In this way, the cell phase and statues can be reported by fluorescences.</P><P>Figure. Version #4, cell magic with degradation tag, targeting peptides and cell cycle regulator. Animation.</P><P>Figure. Genetic circuits for version #4, in glucose medium, not synchronizing.</P><P>Figure. Genetic circuits for version #4, in galactose medium, synchronizing.</P><H4>Final Constructions</H4><H5>Budding Yeast Version</H5><H5><I>E. coli</I> Version</H5></DIV><DIV id="box2" class="box"><H3>Cyclin Promoters</H3><H4>Yeast Version</H4><P>  As we all know, different proteins are planned to be expressed along the cell cycle. And their mRNAs are transcribed through the specific promoters. These promoters locate at their upstream sequences and can be recognized by the RNA polymerase, which can initiate such process. There are databases containing the upstream 600 amino acid sequence in the upstream of DNA, which can be transcript in the G1, S, G2 and M phase, respectively</P><P>Previous study (Wolfsberg et al., 1999) demonstrated that cell cycle specific promoters posses their conserved five to six base pairs. Thus, we found the high-confidence 600 promoter-containing sequences in database that harbor the paper-mentioned 5/6 bp sequence. Consequently, we pick up the upstream 600bp sequence of cln2, cln3, clb2, clb5 and clb6.</P><P>The Clb2 gene is highly expressed in G2 phase, and the genes are very strongly induced by GAL-CLB2, whereas GAL-CLN3 appears somewhat repressive Spellman et al., 1998)</P><P>Figure. KEGG diagram for core cell cycle in yeast.</P><P>Figure. Selected promoters from multiple database and reviews.</P><H5>Measurement Devices</H5><H4><I>E. coli</I> Version</H4><P> Cell cycle is a complex process and can be separated into G0, G1, S, G2, M phase. In each phase, distinct transcription factors help the phase-specific gene express through recognizing their promoters. Thus, these promoters are phase-specific too and can be fused with other genes in order to express such gene in a defined cell cycle phase.The well-regulated cell cycle in E.coli consists of three key events: DNA replication, nucleon segregation and the initiation of cell division. The replication initiator is DnaA protein; the second step relates to MukB protein and the cell division activator is the FtsZ protein. In our project, we take full aadvantage of the two initiators DnaA and FtsZ - using their promoter to control the transcript of specific genes binding behind them in the first step or last step of <I>E. coli</I> cell cycle.</P><H5>DnaA protein</H5><P>As mutation strain demonstrated, dnaA gene is absolutely required for initiation at oriC. When the concentration of DnaA increases, there is increased initiation?, indicating that DnaA protein is the switch for initiation (Atlung, Løbner-Olesen, &amp; Hansen, 1987). Biochemical studies have shown that, with the help of accessory proteins, DnaA protein binds to five sites within oriC, therefore, leading to strand opening of a region containing three 13-mer repeats. This opening results in the begin of bidirectional replication (Atlung, Løbner-Olesen, &amp; Hansen, 1987).</P><P>Figure. dnaA promoter selected for S1 phase.</P><P>Figure. gyrB promoter selected for S2 phase.</P><H5>FtsZ</H5><P>The cell itself seems to regard FtsZ gene expression as the commitment to division, because of the fact that endogenous division inhibitors have FtsZ as target(Bi &amp; Lutkenhaus, 1990). Increasing FtsZ concentration resulted in excess cell division, with the cell size decreasing (Ward &amp; Lutkenhaus, 1985), vice verse. Thus the critical role of FtsZ in division is similar to that of DnaA in replication initiation. FtsZ protein exists randomly throughout cytoplasm during cell elongation, while polymerizes into a membrane-associated ring at the precise site of cell division (Bi &amp; Lutkenhaus, 1991). Just before the appearance of a visible constriction. The ring decreasing along with the septation and FtsZ is dispersed again at completion. Such ring formation is presumed to be essential for cell division, since it cannot be observed in the presence of the cell division inhibitors (Bi &amp; Lutkenhaus, 1993) but is always present when and where septa are formed.</P><P>Figure. ftsQ promoter selected for S2 phase.</P><H5>Measurement Devices</H5></DIV><DIV id="box3" class="box"><H3>Reporter Modification</H3><P> The XFP, as reporter in our project has been modified through</P><P>1) removed the stop codon in them, 2) modified with 23# prefix and suffix， and then 3) added the terminator within stop codon to them. </P><P> Furthermore, all BioBricks were added with 23# prefix and suffix, which ensures it to be standard.
     </P></DIV><DIV id="box4" class="box"><H3>Degradation Tags</H3><H4>Yeast Version</H4><P>In yeast cell, protein degraded through several mechanisms, a major one is the ubquitin- pathway. In detail the signal in substrates can be recognized by the enzyme E3, which transport the ubiqutin from E2 to such protein. And through such way, the ubiquited protein would finally be degraded by the proteasome. </P><H5>PEST</H5><P>There are two types of cyclins in the budding yeast. One kind contains PEST sequence at C-terminal: Cln1, Cln2, Cln3, Pcl1, Pcl2. Another kind is Clb1~6 which posses 9 conserved amino acid sequence in N-terminal named D-box (destruction box). The D-box are necessary for later kind cyclins degradation in M phase and can function in same way when combined with other proteins. In our study, we utilized the D-box, PEST sequence and ubiquitin-mediated pathway for fluorescent proteins degradation.</P><H5>Poly-ubiquitin</H5><P>Ubiquitin, a highly conserved 76-residue protein, was involved in the ubiquitin–proteasome pathway of protein proteolysis, which is a fundamental way for protein turnover, signal transduction and cell cycle control. For example, the degradation of CDC34 through such way is necessary for S phase start; another instance is that APC\C(anaphase-promoting complex\cyclosome) is degraded through ubiquitination, which is the foundation of M phase beginning. And both activated by Cdc28-cyclins</P><P>The mechanism of ubiquitin-mediated protein degradation can be separated into three continuous enzyme catalyze steps. First 1) a ubiquitin is activated by the E1 (ubiquitin-activating enzyme) through a thioester linkage，and then 2)added into the a small ubiquitin-carrier E2. Finally,3) through the E3 ubiquitin protein ligase, this ubiquitin complex was conjugate to the ε-amino group of lysine residues in substrate proteins, forming a glycyllysine isopeptide bond (Hershko, 1997).In some conditions, even E3 enzymes themselves carry ubiquitin as a thioester (Huibregtse et al., 1995).</P><P>Figure. Function of fusion ubiquitin.</P><P>In our project, we design a poly-ubiquitin(5 ubiquitin combined) BioBricks which can directly degrade the proteins fused with it.</P><H5>D-box</H5><P>The best studied substrates of ubiquitin- and APC/C-mediated proteolysis are the mitotic cyclins (Murray, Solomon, &amp; Kirschner, 1989) Mitosis-exit inducer, a cyclin B, are highly dependent on its 90 residues for ubiquitin-mediated proteolysis. Analysis of its N-terminal region demonstrated the sequence essential for cyclin proteolysis, called ‘destruction box’(D-box). Meanwhile, when cyclin destruction started, substrates containing D-boxes were rapidly poly-ubiquitinated. The consensus motif in B-type cyclins is RXALGXIXN. For much of the cell cycle, the D-box may not be recognized with high affinity, and when it is recognized, the ‘bait’ construct becomes highly unstable(Yamano, Tsurumi, Gannon, &amp; Hunt, 1998). </P><P>Figure. Function of destruction box.</P><P>When the D-box exists in the protein N-terminal, this protein is recognized more easily by the E1 and then captured into the ubiquitin-mediated degradation pathway. Thus, we combined it to the fluorescent proteins’ N-terminal to fasten their degradation and avoid veiling the following fluorescence.</P><H5>Measurement Devices</H5><H4><I>E. coli</I> Version</H4><P>In E.coli, the adaptor SspB tethers ssrAtagged substrates to the ClpXP protease, causing a modest increase in their rate of degradation. Which means, a variation of the WT SsrA tag sequence (K1051206, K1051207 and K1051208) will accelerate the degradation of proteins when fused to their C-terminal. Thus the degradation rates are dependent on concentration of proteases and binding mediators. In order to fuse degradation tags freely on the C-terminal of protein, we add TAATAA to the tail of tags.</P><H5>Measurement Devices</H5></DIV><DIV id="box5" class="box"><H3>Targeting Peptides</H3><P>A target peptide is a short (3-70 amino acids long) peptide chain that directs the transport of a protein to a specific region in cell, including nucleus, mitochondria, endoplasmic reticulums (ERs), chloroplasts, apoplasts, peroxisomes and plasma membrane. Targeting peptide can exists in both N-terminal, C-terminal and internal sequence of a precursor protein. And after transported, some target peptides are cleaved by signal peptidases.</P><P>In our project we utilized 19 peptides target to 9 sub-locations in yeast cells, and when combined with fluorescent proteins, such region can be marked by different colors.</P><H5>Mitochondria</H5><P>Though it accounts a small ratio in the cell space, mitochondria possess about 10% to 15% proteins encoded by nuclear genes in eukaryotic organisms. These proteins are synthesized in cytosol and then recognized by the membrane receptors of mitochondria. Translocases in the outer and inner membrane of mitochondria mediate the import and intra-mitochondrial sorting of these proteins. ATP is used as an energy source; Chaperones and auxiliary factors assist in folding and assembly of mitochondrial proteins into their native, three-dimensional structures. </P><P>Figure. Mitochrondria import pathway.</P><P>As shown in the figure above, beta-barrel outer-membrane proteins (dark green), precursor proteins (brown) with positively charged amino-terminal presequences and multispanning inner-membrane proteins (blue) with internal targeting signals are recognized by specific receptors of the outer mitochondrial membrane (TOM) translocases Tom20, Tom22 and/or Tom70. The precursor proteins are then translocated through a small Tom proteins of the TOM complex, Tom40 pore, which the TOM complex contains two or three.</P><H5>Peroxisomes </H5><P>The import of post-translational matrix protein into peroxisomes depends on either of the two peroxisomal targeting signals (PTS), PTS1 and PTS2. PTS2-driven import is facilitated by a complex in the membrane. Under oleic acid-inducing growth conditions, there is a ternary core complex of approximate 150 kDa in the cytosol, which consists of Fox3p,Pex7p and Pex18p. Fox3p is imported as a dimer, while other two are bind in monomeric forms.</P><P>Figure. Peroxisome import pathway.</P><P>As study mentioned there are four steps involved in PTS2-driven import. The first step is the recognition of dimerized Fox3p by Pex7p through its PTS2 in the cytosol. In a second step, the Pex7p–Fox3p complex interacts with Pex18p, which targets the PTS2 pre-import complex to the peroxisomal membrane. The third step is the docking process, involving the interaction between Pex7p and the integral membrane protein Pex13p. As a final step of these early steps in the PTS2 import cascade, PTS2 receptor dissociation takes place during or after its assembly into large oligomeric complexes containing Pex14p and Pex13p. Pex18p remains at the peroxisomal membrane in the form of a large-molecular-weight complex in conjunction with Pex14p and/or Pex13p, from where it might be released to the cytosol(Grunau et al., 2009).</P><H5>Actin </H5><P>In yeast, the cortical actin cytoskeleton seems to specify sites of growth of the cell surface. Because the actin-binding protein ABP1p is associated with the cortical cytoskeleton of Saccharomyces cerevisiae, it might be involved in the spatial organization of cell surface growth. ABP1p is localized to the cortical cytoskeleton and its overproduction causes assembly of the cortical actin cytoskeleton at inappropriate sites on the cell surface, resulting in delocalized surface growth. ABP1p is a novel protein with a 50 amino-acid C-terminal domain, which is very similar to the SH3 domain in the non-catalytic region of nonreceptor tyrosine kinases (including those encoded by the proto-oncogenes c-src and c-abl), in phospholipase C gamma and in alpha-spectrin. They also identified an SH3-related motif in the actin-binding tail domain of myosin-I. The identification of SH3 domains in a family of otherwise unrelated proteins that associate with the membrane cytoskeleton indicates that this domain might serve to bring together signal transduction proteins and their targets or regulators, or both, in the membrane cytoskeleton (Drubin, Mulholland, Zhu, &amp; Botstein, 1990; Khorasanizadeh, 2004).</P><H5>Nucler</H5><P>Histones are nuclear proteins package DNA into nucleosomes, and they are responsible for maintaining the shape and structure of a nucleosome. One chromatin molecule is composed of at least one of each core histones per 100 base pairs of DNA.[The Nucleosome: From Genomic Organization to Genomic Regulation.] There are five families of histones known to date, termed H1/H5, H2A, H2B, H3, and H4. H2A is considered a core histone, along with H2B, H3 and H4. Core formation first occurs through the interaction of two H2A molecules(Acid, 2004). Then, H2A forms a dimer with H2B; the core molecule is complete when H3-H4 also attaches to form a tetramer.</P><H5>Vacuolar Membrane</H5><P>The ZRC1 gene encodes a multicopy suppressor of zinc toxicity in Saccharomyces cerevisiae; however, previously it was reported that the expression of ZRC1 was induced when the intracellular zinc level was decreased. The COT1 and ZRC1 genes of Saccharomyces cerevisiae are structurally related dosage-dependent suppressors of metal toxicity. COT1 confers increased tolerance to high levels of cobalt; ZRC1 confers increased tolerance(Conklin, Culbertson, &amp; Kung, 1994).Zrc1 has six putative trans-membrane domains, and Zrc1-GFP fusion protein was localized to the vacuolar membrane. Zrc1 function as a mechanism to maintain the zinc homeostasis in yeast(Miyabe, Izawa, &amp; Inoue, 2001).</P><H4>Measurement Devices</H4></DIV><DIV id="box6" class="box"><H3>Cell Synchronization</H3><P>As we known, the yeast cell cycle contains a huge and complex regulatory network in the transcription level, translation level as well as protein level. In our project we utilize Sic1 as a regulator to help elongate G1 phase in yeast cell.</P><P>The activation of B-type cyclin (Clb5/6)+Cdc28 kinases is a necessary step for initiation of DNA replication in vivo. One of its inhibitor Sic1 can be phosphorylated by the activated Cln+Cdc28, thereby targeted for degradation. Over expression of the gene encoding p40, SIC1, produces cells with an elongated bud morphology(figure1)(Nugroho &amp; Mendenhall, 1994). SIC1 deletion is viable and causes increasing frequencies of chromosomes broken and lost. The deletion strain segregates out many dead cells, which are primarily arrested at the G2 checkpoint in an asymmetric fashion. Therefore, it has an important role in ensuring genomic integrity, and that this role has a pronounced mother-daughter asymmetry.</P><P>Figure. Over expression of Sic1 can stop all yeast cells to G1 phase.</P><P>After phosphorylation, phospho-Sic1 is specifically recognized by the F-box protein Cdc4, which leads Sic1 being ubiquitinated by the Cdc34±SCF complex (E3). The recognizing and binding by Cdc4 is based on the Sic1’s 9 Cdc4 phospo-degrons (CPDs, figure. a). Several phosphorylation sites contributed to Sic1 instability, with an order of Thr45, Ser76, Thr5,Thr33 and other less significant sites. The immunocipients after selectively phosphorylation as figure e showed suggested that at least six sites phosphorylation is necessary for the Cdc4 recognizing and binding with Sic1. Furthermore, the culture of GAL1-SIC1 constructs strain showed less than 6 sites are phosphorylated is not sufficient for SIC1 degradation in vitro(Nash et al., 2001).</P><P>Figure. N-terminal phosphorylation sites to be mutated.</P><H4>Measurement Devices</H4></DIV><DIV id="box7" class="box"><H3>Alternative Splicing Device</H3><P>In yeast cells, alternative splicing is a common process in before gene transcription. The splicing sites are located in 5'UTR of introns and can by recognized and spliced by some splicesome. Alternative splicing can produce two isoforms from one gene, thus can be used in our project to monitor whether the Sic1 system work efficiently. Here we find a intron with alternative splicing sites in yeast cells named SRCI intron.</P><H4>Src1 Intron and Hub1</H4><P>  SRC1 intron has two 5' splicing site whose efficiency is regulated by protein hub1. Originally, the existence of the intron would produce two mature mRNA in proportion. After engineering, the existence of protein hub1 regulates to the preservation of intron precisely, which would make the following mRNA be expressed normally or not. As for the silencing of wild type gene HUB1, we choose CRISPRi that is comparably easy to use reversibly.</P><P>Alternative splicing substantially increases the gene product diversity and is a major source of cell type differentiation. A good example is the alternative splicing of Saccharomyces cerevisiae SRC1 pre-mRNA, which is promoted by the conserved ubiquitin-like protein Hub1. It can function through binding non-covalently to a conserved element termed HIND in the spliceosomal protein Snu66. Such binding makes the splicesome target sites change and moderately alters spliceosomal interactions. </P><P>Hub1 is a ubiquitin-like modifier (UBL) that covalent modify the proteins. Interest enough, it harbors several different to other UBLs in which it possesses a C-terminal double tyrosine motif while others having a GG motif. The Snu66, a tri-snRNP in yeast spliceosome, possesses with two N-terminal HINDs (Hub1-INteraction Domain). The Hub1–HIND interaction comprises a strong salt bridge accompanied by several hydrophobic contacts and high affinity. Such binding modifies the spliceosome rather than modulating the properties of an individual binding partner. Hub1-controlled splicing occurs universally in eukaryotes. SR proteins and hnRNPs involved in spliceosome targeting do not seem to exist in S. Cerevisiae , and thus the Hub1-dependent mechanism may be evolutionarily older. </P><P>Scr1 is a protein in yeast having alternative splicing sites in its intron. The characteristic differential Hub1 dependence of SRC1 alternative splicing requires the tandem arrangement of overlapping 5’ splice sites. The Hub1 binding spliceosome can splice the intron from both downstream 5’ sites as well as the upstream 5’ sites with preference to the former one.</P><P>Figure. Principle for alternative splicing of Src1.</P><P>Showed in Figure.above When cutting in the upstream splice sites, the exons flanking around it would be translated as a fusion protein Scr-S. However, when cutting at the down stream one, the left 4bp in intron would result in a frame shift, thus only the forward exon can express named Scr-L.</P><H4>dCas9 CRISPR interface system</H4><P>CRISPR shorts for Clustered Regularly Interspaced Palindromaic Repeats system, which can be targeted to DNA using RNA, enabling genetic editing of any region of the genome in many organisms.(Cho, Kim, Kim, &amp; Kim, 2013). In the type II CRISPR/Cas system, a ribonucleoprotein complex formed from a single protein (Cas9), a crRNA, and a trans-acting CRISPR RNA (tracrRNA) can carry out efficient crRNA-directed recognition and site-specific cleavage of foreign DNA(Deltcheva et al., 2011). After mutated the endonuclease domains of the Cas9 protein, it creates a programmable RNA-dependent DNA-binding protein. The sgRNA consists of three domains: a 20 nt complementary region for specific DNA binding, a 42 nt hairpin for Cas9 binding (Cas9 handle), and a 40 nt transcription terminator derived from S. Pyogenes. After translation, the Cas9 binds to sgRNA to form a protein-RNA complex, which can recognize target sites in the genome sequence and bind to it. Then, it could block RNA polymerase and transciption elongation.</P><P>Figure. sgRNA structure.</P><P>Figure. Principle of the function of dCAS9 and guide RNA.</P><H4>Measurement Devices</H4></DIV><DIV id="box8" class="box"><H3>Microfluidic Device</H3><P>Based physically on the chip, microfluidics can automatically control the biology process. In our project, the microfluidics plays a role as the movie theater, for cell cycle control. We designed a channel which can capture the yeast cell and then alternatively inject medium of different concentrations. Therefore, the yeast cell cycle can be synchronized. Furthermore, due to its transparent attribution, the chip can project the fluorescent lights from the yeast cell sub-locations. Consequently, the magic film can be shown to us. </P><P>Microfluidic devices possess many advantages such as large-scale integration, fast analyses, and considerably reduced reagent consumption (Comparative study and improvement of current cell micro-patterning techniques). Nowadays many designs are based on cell loading and culturing in a microfluidic channel with particular trapping structures such as micro-well arrays, and micro-cup arrays and micro-chambers. In our project, we design a cell loading technique based on gas absorption of degassed polydimethylsiloxane (PDMS). Because the balance concentration of gas dissolved in PDMS is proportional to the partial pressure of the gas around it, so that one can degas a piece of PDMS by placing it in vacuum. </P><P>Our cell loading device is a one level PDMS with an array of triangle cavities connected to one or both sides of the main fluidic channel. After degassed, yeast cells in the buffer solutions can be introduced into the device’s triangle cavities by gas absorption therein. Through changing the cell density in the buffer, we can control the number of the yeast cells loaded into each triangle cavity. Statistically, the ratio of cavities with a single cell can reach 40%. Consequently it is convenient to monitor the cell number and cell phenotypic variations because the yeast cells can grow in the micro-triangle-cavities in one layer.</P><P>Figure. Masks for PDMS microfluidic device fabrication.a : Three micro-triangle-cavities at one side of the main channel. b : One single channel for cell culture (100 microcavities). c : Seven parallel channels for different cell culture environments(Luo et al., 2008).</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories">http://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Shenzhen_BGIC_ATCG/stories" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Shenzhen_BGIC_ATCG/stories" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/stories&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/stories&amp;oldid=345290" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>