"
Page
Discussion
View source
History
teams
Log in
 
Team:Shenzhen BGIC ATCG/notes
From 2013.igem.org
Consortium
Promoter Group
Degradation Group
Targeting Group
Regulator Group
Splicing Group
Microfluidic Group
Modeling Group
Instructors
The Magic
Cyclin Promoters
Reporter Modification
Degradation Tags
Targeting Peptides
Cell Synchronization
Alternative Splicing Device
Microfluidic Device
Blueprint
Cell Cycle
Cell Synchronization
Alternative Splicing by CRISPRi
Degradation Rate
The Magic
Promoter Verification
Reporter Locating
Degradation Rate
Cell Synchronization
Alternative Splicing
iGEMer Incubator
Guest iGEMers
APP: iGEMagic
Friendship
Best BioBricks
Devices List
Basic Parts
Parts Sequencing
Timeline
Attributions
Protocols
References
Acknowledgements
Playing with my eyes
aren't you?
Hi I am Dr. Mage!
A "budding" yeast cell!
Timeline
Attributions
Protocols
References
Acknowledgements
Timeline
Promoter Group
Week1: research about the previous work of iGEM, especially the work done by the European teams.
Week2~week4: project idea discussing and determination， group issue asssigned.
Week5~week6: research about the principle of promoter in E.coli and yeast.
Week7: Experiments process design, constructs the basic parts and test curcuits.
Week8: Design the primers needed to amplification the promotor
Week9: Workshop among several schools
Week10-week13: constructs the basic parts: BBa_K1051300, BBa_K1051301, BBa_K1051302,BBa_K1051303, BBa_K1051304, BBa_K1051305, BBa_K1051306
Week14-week18: Measurement designing
Week19-week22: Test measurement circuits and then data analysis
Week23-week25: results drawing and made the wiki
Degradation Group
Week1: previous team project data collection, especially the Latin America teams
Week2~week4: project idea decided，group issue divided into E.coli one and yeast one.
Week5~week6: research of principle of degradation peptide and design the experiment generally combining with the mechanism of cyclin protein degradation.
Week7: Experiments process design, constructs the basic parts and test curcuits.
Week8: Design the primers needed to amplification the degradation tags.
Week9: Workshop among several schools
Week10-week13: constructs the basic parts: K051200/K1051201/K1051202/K1051203/K1051204/K1051205/K1051206/K1051207
Week14-week18: Measurement circuits construction
Week19-week22: Test measurement circuits and then data analysis
Week23-week25: results drawing and made the wiki
Targeting Group
Week1: research about the previous work of iGEM, especially the work done by the Asia teams.
Week2~week4: project idea discussing and determination， group issue assigned.
Week5~week6: research about the principle of promoter in E.coli and yeast.
Week7: Experiments process design, constructs the basic parts and test circuits.
Week8: Design the primers
Week9: Workshop among several schools
Week10-week13: constructs the basic parts: BBa_K1051100 to BBa_K1051118
Week14-week18: Measurement designing
Week19-week22: Test measurement circuits and then data analysis
Week23-week25: results drawing and made the wiki
Synchronization Group
Week1: research about the previous work of iGEM, especially the work done by the European teams.
Week2~week4: project idea discussing and determination， group issue assigned.
Week5~week6: research about the principle of promoter in E.coli and yeast.
Week7: Experiments process design, constructs the basic parts and test circuits.
Week8: Design the primers needed to amplification the degradation tags.
Week9: Workshop among several schools
Week10-week13: constructs the basic parts: BBa_K1051500, etc.
Week14-week18: Measurement designing
Week19-week22: Test measurement circuits and then data analysis
Week23-week25: results processing
Modeling
Week 1-4 research about the previous best model
Week 4 Decided using the budding yeast cell cycle model as basic part
Week 4-6 Learning the cell designer，Matlab Simbiology as the modeling software and trying to make some test
model.
Week 4-6  Learning how to use the Matlab Simbiology part and input results within cell designer
Week 7-10 Understanding the cell cycle model in cell designer and using cell designer drawing  reaction network,
setting the parameters.
Week 11-15 Decided making three modeling: alternative splicing, Sic1 regulation and degradation tags. Do related research.
Week 15-20 Made the cell cycle circuits, find parameters
Week 23-25 combine with experiments data and made wiki
Attributions
Work Design
Gone Jianhui as a team leader and K2 as our instructor draft our project "Cell Magic".
Experiments Conduct
Li Xiang Li,Xu Yanhui,Wu Fanzi, Yu yang from SCU: responsible for targeting peptide,XFP,terminators design and experiments.
Chen Shihong,Gu Chenguang, Lu Yanping, Liang Jiale,from South China University of Technology, are responsible for the alternative splicing Src1 and Mer2 intron design and experiments.
Zhu Shuang, Lin Kequan from Wuhan University and Wei Wei, Zheng Bingwei, Yi Lan in HUST work together for the promotors.
Guan Rui from SEU and He funan, Wang Rui, Lin Li,Zhang Yaolei from UESTC made their efforts to the degradation parts.
Zhou Wanling,Zhang Aiping, Li Dongdong, the undergraduates in AHMU, joined the part one
Chen Yichun of SCNU, Zhong Na of JNU work for the microfulidic part.
The SCNU student: Chen Chengxuan, Lin Qiongfen, Xie Qiaolin, worked for the cell cycle regulator Sic1
Modeling
Liu Shuang Liu from SEU, Zhou Yang from SCUT, Jinchun Zhang from SCU, Qiu Bitao from BGI
Wiki Construction
Zhang Jinchun and Zhou Yang
Protocols
Protocol of microfluidic
1. design the pattern of the biochip.
2. print the pattern on the mask.
3. prepare the basic materials like clean silicon wafer, photoresist and so on.
4. spread the photoresist on the silicon wafer and make sure the thickness is 10 um.
5. with the help of the ultraviolet ray, the pattern of the mask can be transferred to the metamorphic photoresist.
6. (photographic fixing)Bake the silicon wafer to solidify the photoresist for about 2 hours.
7. (develop the pattern)wash off the photoresist which wasn’t exposure to the ultraviolet ray by using the special chemical reagent
8. Place the silicon wafer in the culture dish, then prepare the PDMS and pour it into the culture dish.
9. When the PDMS freeze, downcut it and fit it on a slide by surface plasma.
10. Connect the capillary tube and the chip.
Protocol of chip-based XFP degradation rate detection in E.coli.
First of all, E.coli will be measured after shaking to about OD2.0（600）in the chip which was washed by plasma water, vacuum pumping. The bacteria liquid was pushed into the chip, letting the cells enter the small triangle. Then use the constant flow pump culture medium into chip (the laboratory constant temperature, can not guarantee the training environment, 37°E. coli slower growth). The medium speed is about 200ul/h. Finally we test the data after yeast fulled in the triangles.Using the IPTG medium, the new RFP expression was stopped and we can regard the lights as degradation tags' efficiency.
Protocol of Enzyme - labelled meter detecting the fluorescent protein intensity
First of all, take a certain amount of bacteria liquid, recovery to around OD0.6(600), ensuring the bacteria in the logarithmic growth phase. Diluted and then the bacteria transferred to 96 well plates, measured their fluorescence intensity. Red fluorescent protein using an excitation wavelength of 584nm, and its emission wavelength is 607nm.When measuring, we first detect the OD600 of each strain, removing the factor of bacteria number difference. Thus the fluorescence intensity cannot be altered by bacteria quantity. Then the measurement mode switching for measurement of fluorescence, fluorescence intensity.
Protocol of Fluoresces Intensity Measurement via ImageJ
1. Install and open the ImageJ
2. Open the picture：File>open
3. Transit to 8bit grey style：Image>Type>8-bit
4. Invert white and black：Edit>Invert
5. Revise light intensity
6. Using Global calibration，revise the lilght intensity
7. Set pixel
8. Set Measurements: Area、Integrated density.
9. Set the threshold :Image>Adjust>Threshold
10. Meaure: Analyze>Measure
11. Record the results
Protocol of Yeast Synchronization with Microfluidics
With the micro-fluidic device described above, and based on the fact that the length of the G1 phase of budding yeast is more sensitive to starvation, we carried out a series of experiments with various modulation schemes of changing medium. The rich medium used in budding yeast modulation experiments was SC -Ura, the poor medium was a mixture of 1.5% SC -Ura and 98.5% PBS (phosphate-buffered saline, pH = 6.0). After a scheme of 60 min period of poor medium alternating with 90 min period of rich medium three times.
References
Acid, S. A. (2004). Lehninger principles of biochemistry.
Atlung, T., Løbner-Olesen, A., & Hansen, F. G. (1987). Overproduction of DnaA protein stimulates initiation of chromosome and minichromosome replication in Escherichia coli. Molecular and General Genetics MGG, 206(1), 51-59.
Bi, E., & Lutkenhaus, J. (1990). Interaction between the min locus and ftsZ. Journal of bacteriology, 172(10), 5610-5616.
Bi, E., & Lutkenhaus, J. (1991). FtsZ ring structure associated with division in Escherichia coli.
Bi, E., & Lutkenhaus, J. (1993). Cell division inhibitors SulA and MinCD prevent formation of the FtsZ ring. Journal of bacteriology, 175(4), 1118-1125.
Cho, S. W., Kim, S., Kim, J. M., & Kim, J.-S. (2013). Targeted genome engineering in human cells with the Cas9 RNA-guided endonuclease. Nat Biotechnol.
Conklin, D. S., Culbertson, M. R., & Kung, C. (1994). Interactions between gene products involved in divalent cation transport in Saccharomyces cerevisiae. Molecular and General Genetics MGG, 244(3), 303-311.
Deltcheva, E., Chylinski, K., Sharma, C. M., Gonzales, K., Chao, Y., Pirzada, Z. A., . . . Charpentier, E. (2011). CRISPR RNA maturation by trans-encoded small RNA and host factor RNase III. Nature, 471(7340), 602-607.
Drubin, D. G., Mulholland, J., Zhu, Z., & Botstein, D. (1990). Homology of a yeast actin-binding protein to signal transduction proteins and myosin-I. Nature, 343(6255), 288-290.
Engebrecht, J., & Roeder, G. S. (1990). MER1, a yeast gene required for chromosome pairing and genetic recombination, is induced in meiosis. Molecular and cellular biology, 10(5), 2379-2389.
Fuller, R. S., & Kornberg, A. (1983). Purified dnaA protein in initiation of replication at the Escherichia coli chromosomal origin of replication. Proceedings of the National Academy of Sciences, 80(19), 5817-5821.
Grunau, S., Schliebs, W., Linnepe, R., Neufeld, C., Cizmowski, C., Reinartz, B., . . . Erdmann, R. (2009). Peroxisomal Targeting of PTS2 Pre‐Import Complexes in the Yeast Saccharomyces cerevisiae. Traffic, 10(4), 451-460.
Hershko, A. (1997). Roles of ubiquitin-mediated proteolysis in cell cycle control. Current Opinion in Cell Biology, 9(6), 788-799.  Huibregtse, J. M., Scheffner, M., Beaudenon, S., & Howley, P. M. (1995).
A family of proteins structurally and functionally related to the E6-AP ubiquitin-protein ligase. Proceedings of the National Academy of Sciences, 92(7), 2563-2567.
Khorasanizadeh, S. (2004). The nucleosome: from genomic organization to genomic regulation. Cell, 116(2), 259-272.
Luo, C., Zhu, X., Yu, T., Luo, X., Ouyang, Q., Ji, H., & Chen, Y. (2008). A fast cell loading and high‐throughput microfluidic system for long‐term cell culture in zero‐flow environments. Biotechnology and bioengineering, 101(1), 190-195.
Miyabe, S., Izawa, S., & Inoue, Y. (2001). The Zrc1 Is Involved in Zinc Transport System between Vacuole and Cytosol in< i> Saccharomyces cerevisiae. Biochemical and Biophysical Research Communications, 282(1), 79-83.
Munding, E. M., Igel, A. H., Shiue, L., Dorighi, K. M., Treviño, L. R., & Ares, M. (2010). Integration of a splicing regulatory network within the meiotic gene expression program of Saccharomyces cerevisiae. Genes & Development, 24(23), 2693-2704.
Murray, A. W., Solomon, M. J., & Kirschner, M. W. (1989). The role of cyclin synthesis and degradation in the control of maturation promoting factor activity. Nature, 339(6222), 280-286.
Nash, P., Tang, X., Orlicky, S., Chen, Q., Gertler, F. B., Mendenhall, M. D., . . . Tyers, M. (2001). Multisite phosphorylation of a CDK inhibitor sets a threshold for the onset of DNA replication. Nature, 414(6863), 514-521.
Nugroho, T. T., & Mendenhall, M. D. (1994). An inhibitor of yeast cyclin-dependent protein kinase plays an important role in ensuring the genomic integrity of daughter cells. Molecular and cellular biology, 14(5), 3320-3328.
Qiu, Z. R., Shuman, S., & Schwer, B. (2011). An essential role for trimethylguanosine RNA caps in Saccharomyces cerevisiae meiosis and their requirement for splicing of SAE3 and PCH2 meiotic pre-mRNAs. Nucleic Acids Research, 39(13), 5633-5646.
Spellman, P. T., Sherlock, G., Zhang, M. Q., Iyer, V. R., Anders, K., Eisen, M. B., . . . Futcher, B. (1998). Comprehensive identification of cell cycle–regulated genes of the yeast Saccharomyces cerevisiae by microarray hybridization. Molecular biology of the cell, 9(12), 3273-3297.
Ward, J. E., & Lutkenhaus, J. (1985). Overproduction of FtsZ induces minicell formation in E. coli. Cell, 42(3), 941-949.
Wolfsberg, T. G., Gabrielian, A. E., Campbell, M. J., Cho, R. J., Spouge, J. L., & Landsman, D. (1999). Candidate regulatory sequence elements for cell cycle-dependent transcription in Saccharomyces cerevisiae. Genome Res, 9(8), 775-792.
Yamano, H., Tsurumi, C., Gannon, J., & Hunt, T. (1998). The role of the destruction box and its neighbouring lysine residues in cyclin B for anaphase ubiquitin-dependent proteolysis in fission yeast: defining the D-box receptor. The EMBO Journal, 17(19), 5670-5678.
Belle, A., Tanay, A., Bitincka, L., Shamir, R., & O'Shea, E. K. (2006). Quantification of protein half-lives in the budding yeast proteome. Proc Natl Acad Sci U S A, 103(35), 13004-13009. doi: 10.1073/pnas.0605420103
Belli, G., Gari, E., Aldea, M., & Herrero, E. (2001). Osmotic stress causes a G1 cell cycle delay and downregulation of Cln3/Cdc28 activity in Saccharomyces cerevisiae. Mol Microbiol, 39(4), 1022-1035.
Chen, K. C., Calzone, L., Csikasz-Nagy, A., Cross, F. R., Novak, B., & Tyson, J. J. (2004). Integrative analysis of cell cycle control in budding yeast. Mol Biol Cell, 15(8), 3841-3862. doi: 10.1091/mbc.E03-11-0794
Gilchrist, M. A., & Wagner, A. (2006). A model of protein translation including codon bias, nonsense errors, and ribosome recycling. J Theor Biol, 239(4), 417-434. doi: 10.1016/j.jtbi.2005.08.007
Mason, P. B., & Struhl, K. (2005). Distinction and relationship between elongation rate and processivity of RNA polymerase II in vivo. Mol Cell, 17(6), 831-840. doi: 10.1016/j.molcel.2005.02.017
Wang, Y., Liu, C. L., Storey, J. D., Tibshirani, R. J., Herschlag, D., & Brown, P. O. (2002). Precision and functional specificity in mRNA decay. Proc Natl Acad Sci U S A, 99(9), 5860-5865. doi: 10.1073/pnas.092538799
Acknowledgements
At the Beginning
The BGI-ATCG iGEM 2013 team is a huge and diversity group consists of undergraduate from ten universities. Our team members’ major is really different, such as biotechnology bioinformatics, physical and even mathematics. Our work is partly assigned into each group made up by the students from the same university, which you can find in the group part in this page. And you can also browse about who and how everyone contributed to the project in this page. We really appreciate all our advisors and instructors that have assisted us throughout this project, without whom the project could not been carried out. We would also like to thank all everyone else who has helped us to achieve our project through put up advice or providing DNA, seeds, or other materials. Their contributions have helped us enormously. For a full list of acknowledgments, please see the bottom of this page.
Financial Support
BGI College provides the totally funding including our team registration fee and competition travel fee.
BGI Research Institute covers the wet-lab costs and provides equipments and labs.
General Support
Unit of Synthetic Biology at BGI trains team members about the basic experiment technology and knowledge which was really necessary for us.
Cho-Kiu Wong at SMC, BGI Tech Solutions help us much to send and receive the BioBricks.
MathWorks provides free software of Matlab and SimBiology.
Material Support
Boeke Lab of John Hopkins Medical Institutions (Boeke Lab @ John Hopkins Medical) provide us pRS413, pRS414, pRS415, pRS416
Dr. Chi-Ming Wong (Dr. Chi-Ming Wong @ HKU) at Hong Kong University provides us the plasmid YEpLac195 YEpLac181
Kai Tian and Yong Li at BGI, helping us with the CRIPSR system.
SUSTC Biology Department and the BGI Unit of Synthetic Biology help us purchasing some materials.
WHU-China has provided us the E. coli version of CRISPRi system.
At Last
We really appreciate
1) the BGIC_0101 team for their cooperation,
2) the SCSTC help us for borrowing device and booking materials, and
3) SYSU, SUSTC borrowing us some experiments equipment.
Lab Support
Unit of Synthetic Biology at BGI supports us with the lab for the most important cell and molecular biology experiments.
Dr. Ming Ni, team leader in BGI cancer group, let us use his lab for the microfluidic experiments.
Pr. Lingling Shui at South China Normal University provides the microfluidic lab and materials for our experiments.
Cooperation College or University
Huazhong University of Science and Technology
Wuhan University
China University of Geosciences
South China University of Technology
South China Normal University
Jinan University
Sichuan University
University of Electronic Science and Technology of China
Southeast University
Qingdao University
University of Chinese Academy of Sciences
Retrieved from " http://2013.igem.org/Team:Shenzhen_BGIC_ATCG/notes "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
