<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Shenzhen_BGIC_ATCG_results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Shenzhen_BGIC_ATCG/results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Shenzhen_BGIC_ATCG/results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Shenzhen_BGIC_ATCG/results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Shenzhen BGIC ATCG/results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="menu"><UL class="menu_navi"><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board1">Consortium</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board2">Promoter Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board3">Degradation Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board4">Targeting Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board5">Regulator Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board6">Splicing Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board7">Microfluidic Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board8">Modeling Group</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/team#board9">Instructors</A></LI></UL><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board1">The Magic</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board2">Cyclin Promoters</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board3">Reporter Modification</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board4">Degradation Tags</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board5">Targeting Peptides</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board6">Cell Synchronization</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board7">Alternative Splicing Device</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/stories#board8">Microfluidic Device</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/modeling#board1">Blueprint</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/modeling#board2">Cell Cycle</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/modeling#board3">Cell Synchronization</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/modeling#board4">Alternative Splicing by CRISPRi</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/modeling#board5">Degradation Rate</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board1">The Magic</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board2">Promoter Verification</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board3">Reporter Locating</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board4">Degradation Rate</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board5">Cell Synchronization</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results#board6">Alternative Splicing</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/humanpractice#board1">iGEMer Incubator</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/humanpractice#board2">Guest iGEMers</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/humanpractice#board3">APP: iGEMagic</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/humanpractice#board4">Friendship</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/parts#board1">Best BioBricks</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/parts#board2">Devices List</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/parts#board3">Basic Parts</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/parts#board4">Parts Sequencing</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/notes#board1">Timeline</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/notes#board2">Attributions</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/notes#board3">Protocols</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/notes#board4">References</A></LI><LI class="subnavi"><A href="https://2013.igem.org/Team:Shenzhen_BGIC_ATCG/notes#board5">Acknowledgements</A></LI></DIV><DIV class="hotspot1"><DIV class="text1"><P>Playing with my eyesaren't you?</P></DIV></DIV><DIV class="hotspot2"><DIV class="text2"><P>Hi I am Dr. Mage!A &quot;budding&quot; yeast cell!</P></DIV></DIV><DIV id="bodymain" class="results"><DIV class="navleftholder"><UL class="navleft"><LI><A id="navleftsub1" class="navleftsub" href="#board1">The Magic</A></LI><LI><A id="navleftsub2" class="navleftsub" href="#board2">Promoter Verification</A></LI><LI><A id="navleftsub3" class="navleftsub" href="#board3">Reporter Locating</A></LI><LI><A id="navleftsub4" class="navleftsub" href="#board4">Degradation Rate</A></LI><LI><A id="navleftsub5" class="navleftsub" href="#board5">Cell Synchronization</A></LI><LI><A id="navleftsub6" class="navleftsub" href="#board6">Alternative Splicing</A></LI></UL></DIV><DIV id="box1" class="box"><H3>The Magic</H3><P>Our project “Cell Magic&quot;, a complex work has been completed partly,due to the time limitation. The results we have gotten are listed below:</P><P>We verified some of the cyclin promoters. The clb6 promoter, which is supposed to be expressed in G1 phase, has been verified perfectly.</P><P>Figure. clb6 promoter for G1 phase has been verified</P><P>Targeting peptides to three locations have been verified. Others are still being constructed and tested.</P><P>Figure. Test for targeting peptides to mitochondria, nucleus and vacuolar. </P><P>We got good data for degradation tags for <I>E. coli</I>. Construction of degradation devices for yeast have been finished and being tested. Degradation rate calculated  from both methods mates well.</P><P>Figure. Pictures for degradation tag testing devices with positive control.</P><P>Microfluidics device has been successfully developed for measurement and cell synchronization.</P><P>Figure. Cells being captured by microfluidic chip.</P><P>Figure. Degradation tags were tested in Microfluidic</P><P>Cell synchronization has been successful by changing the method to microfluidics based one.</P><P>Figure. All cells are synchronized to G1 phase by using the microfluidics method.</P><P>Alternative splicing device are still in construction, related results will be updated later.</P></DIV><DIV id="box2" class="box"><H3>Promoter Verification</H3><H4>Achievements</H4><P> Using the GFP as reporter and morphological alteration as cell cycle representation, we verified the Clb6 can be activated in G1 phase in the yeast.</P><P>Figure. Measurement device for clb6 promoter</P><P> As shown in the picture in normal lights, there are some yeast were budding. These budded yeast cells and small size budding yeasts (within the red circles) are assumed in the G1 phase of cell cycle. And the next picture was taken under activation light, thus the green lights can representing the Clb6 promoter expressing. The two pictures indicated the Clb6 were expressed in the G1 phase as we expected when designing the experiments </P><P> PS: we modified the contrast ratio to lower the lights of neighbor cells, thus our results looking better.</P><P>Figure. clb6 promoter for G1 phase has been verified.</P><P>Figure. Measurement device for cln3 promoter</P><P>We also captured fluorescence on cln3 promoter (S phase) test device, for the fluorescence is quite weak, we utilize micro plate reader and flow cytometry to verify the promoter.</P><P>Figure. Flow cytometry to verify S phase promoter cln3.</P><H4>Problems</H4><H5>Yeast Version</H5><P>While the promoters are natural ones without optimization, and our shuttle plasmid is pSR416 which is a single copied one, fluorescence is too weak to be captured by microfluidics device. That is a main reason for why we currently cannot show you a perfect cell magic movie. We are now constructing all devices with high copy number vectors, and try to optimize the junction near kozak sequence to make a better magic.</P><H5><I>E. coli</I>  Version</H5><P>Promoters for <I>E. coli</I> seems not to be successful yet. In our first version, original 5' utr of promoter's downstream gene is included in BioBrick's sequences. The wild-type promoter and RBS may be too weak, or the junction of RFC[23] may have negative effect to the RBS. So in the present version, original 5' utr has been removed and strong RBS B0030 is used. We will update related results later.</P></DIV><DIV id="box3" class="box"><H3>Reporter Locating</H3><H4>Reporter Modification</H4><P>Fluorescence proteins have been modified to RFC[23], and stop codons have been removed for fusion protein purpose.</P><P>Figure. Fluorescence proteins have been modified to RFC[23], and stop codons have been removed for fusion protein purpose</P><H4>Targeting Peptides</H4><P> The four pictures shows the reporters can rightly locate to Mitochondria, Nucleus and Vacuole，respectively. Picture A is the negative control, all yeast cells are lighted with GFP. And figure B is the reporter to the mitochondria, we can saw there are several light spots in one cells. Figure C is the reporter located in nucleus, the green spots are small and there is only one in a yeast cell.The last picture shows the reporter of Vacuolar membrane, the green lights were discrete in cells which was as expected.</P><P>Figure. Test for targeting peptides to mitochondria, nucleus and vacuolar.</P></DIV><DIV id="box4" class="box"><H3>Degradation Rate</H3><P>In every curcuits measurement, we firstly test the growth curve through detecting the absorbance. Taken small amount of bateria, inoculate into 400ul LB medium, ensuring the initial concentration between 0.02-0.05 (OD600). Cultured in the room temperature, detect the OD600 value every 20 minutes. Y axis represents the logarithm of bacteria number, X axis represents the growth time. </P><P>Figure. Growth curves have been drawn for all constructed devices.</P><P>In E.coli, the adaptor SspB tethers ssrA tagged substrates to the ClpXP protease, causing a modest increase in their rate of degradation. Which means, a variation of the WT SsrA tag sequence (K1051206, K1051207 and K1051208) will accelerate the degradation of proteins when fused to their C-terminal. Thus the degradation rates are dependent on concentration of proteases and binding mediators.</P><H4>Achievements</H4><P>We constructed the measurement pathway of each tag (K1051257, K1051258 and K1051259) to test the rates of degradation of tagged proteins respectively. J04450 was used as positive control because of the same promoter and fluorescent protein. </P><P>Figure. Measurement devices for degradation tags in <I>E. coli</I>, detected by microscope</P><P>Figure. Fluorescence ladder for degradation test devices. From left, the negative control,BBa_K1051257, BBa_K1051258, BBa_K1051259, J04450 as Positive Control. </P><P>As the pictures showed, the lights of RFP within three degradation tags are decreasing. </P><P>Enzyme - labelled meter detect the fluorescent protein intensity</P><P>First of all, take a certain amount of bacteria liquid, recovery to around OD0.6(600), ensuring the bacteria in the logarithmic growth phase. Diluted and then the bacteria transferred to 96 well plates, measured their fluorescence intensity. Red fluorescent protein using an excitation wavelength of 584nm, and its emission wavelength is 607nm.When measuring, we first detect the OD600 of each strain, removing the factor of bacteria number difference. Thus the fluorescence intensity cannot be altered by bacteria quantity. Then the measurement mode switching for measurement of fluorescence, fluorescence intensity.</P><P>Figure. The test results for K1051257, K1051258, K1051259 and positive control. 1a.Positive control in bright field; 1b.Positive control under exciting light; 2a.K1051257 in bright field; 2b. K1051257 under exciting light; 3a.K1051258 in bright field; 3b. K1051258  under exciting light; 4a. K1051259 in bright field; 4b. K1051259 under exciting light.</P><P>Figure. Fluorescence intensity calculated by ImageJ and degradation rate calculated by model of positive control and three tested degradation tags.</P><P>The degradation efficiency of three degradation tags are obvious.</P><P>While in simulation, we obtained the degradation rate by calculating with [P1_P]/[P2_P]</P><P>Figure. Measurement devices for degradation tags in <I>E. coli</I>, detected by microfluidics</P><P>Figure. The test results of BBa_K1051258 in chip. A,LB medium,O minuts; B, IPTG medium,9minutes; C, IPTG medium, 15 minutes</P><P>Using the IPTG medium, the new RFP expression was stopped and we can regard the lights as degradation tags' efficiency. Obviously, the lights are decreasing along with the time. Thus, it indicates that the degradation tags work effectively.</P><P>Figure. RFP with degradation tag's half life tested by microfluidics and fit by modeling.</P><P>Calculated by ImageJ and from experimental fitting curve we got its half-life of our protein 6.88 min. So the degradation rate should be: </P><P>Simulation results calculated and fit from both microscope and microfluidics mates well, which has verified our experimental results.</P><P>The datasheet of degradation tags and half-life of E. coli with degradation tags and positive control. </P><P>From these data, we can see that the two data from microscope and microfluidic mate very well. </P><H4>Problems</H4><H5>Yeast Version</H5><P>The test for yeast degradation tags has not been fully successful. Fluorescence has been captured in some occasions but not stable. It may caused by the same problem with promoter test devices - plasmid copy number. </P><H5><I>E. coli</I>  Version</H5><P>Data collected from microscope for different degradation tags did not fully meet performance expectation. Reasons may be multiple to increase systematic errors.</P><P>- Fluorescence is too weak for calculating</P><P>- Quenching of fluorescence, especially in RFP</P><P>- Sample size is small for ImageJ calculating</P><P>We will have more data detected by microfluidics and flow cytometry. Results will be updated later.</P></DIV><DIV id="box5" class="box"><H3>Cell Synchronization</H3><H4>Microfluidics Device</H4><P>We made the chip as a platform for watching and synchronize the cells. First step is capture the cells by the chip. As showed in the two figures, both E.coli and budding yeast can be captured by the chip successfully.</P><P>Figure. <I>E. coli</I> and yeast cells have been captured by the chip successfully.</P><P>As Sic1 method has not been successful, we tried another way by changing medium by microfluidics. Protocols can be found in Notes page.</P><P>Figure. All cells are synchronized to G1 phase by using the microfluidics method.</P><P>All yeast cells in microfluidics chip are in G1 phase, after budding.</P><H4>Sic1 Synchronization</H4><P>Seven phosphorylation sites are mutated by PCR to make Sic1 mutant. However, it has been taking us too long time that we have been still working on right clones. Experiments results will be updated later. 


</P></DIV><DIV id="box6" class="box"><H3>Alternative Splicing</H3><H4>Original Design</H4><P>In our project, we attempt to use intron as a switch. However, according to previous research, there are two splicing forms of SRC1 intron: one is complete splicing (5’S) while the other can leave 4bp at its 5’ end (5’L). The remaining 4-bp causes frame-shift and a stop codon in the region of adaptor, thus the following exon will not be translated. However, this kind of switch is not complete – the ratio of 5’L and 5’S varied between 40-60 and 85-15.</P><P>In order to make a complete switch, we abandoned the remaining 4bp of SRC1 intron to avoid producing 5’L mRNA. So the result comes out to be: when Hub1p are expressed, they bind to spliceosome to modify it, resulting to easy recognizing 5’S splice site and its splicing; when inhibit the expression of HUB1 by CRISPRi system, spliceosome can hardly recognize 5’S splice site, therefore no intron can be spliced. </P><H4>Further Plans</H4><P>However, things didn’t go as we expected. Our experiment showed that this 4bp plays important role in SRC1 intron splicing. So, to achieve our goal, we redesigned another three versions of SRC1 intron:
</P><P>1.	Intron with GG at its 5’ end</P><P>2.	Intron with CGG at its 5’ end</P><P>3.	Intron with its original 6bp at both end</P><P>Due to the lack of time, we are still  constructing these parts to test devices.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results">http://2013.igem.org/Team:Shenzhen_BGIC_ATCG/results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Shenzhen_BGIC_ATCG/results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Shenzhen_BGIC_ATCG/results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_ATCG/results&amp;oldid=343565" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>