<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_FAFU-CHINA_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:FAFU-CHINA/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid noma"><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:FAFU-CHINA">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/Applied_Design">Background</A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/InterLab">Interlab</A></LI><LI class="chooseone"><A href="https://2017.igem.org/Team:FAFU-CHINA/Plant">Module 1: Phyto-route</A></LI><LI class="choosetwo"><A href="https://2017.igem.org/Team:FAFU-CHINA/Experiments#sss2">Module 2: Metal-trap</A></LI><LI class="choosethree"><A href="https://2017.igem.org/Team:FAFU-CHINA/Experiments#sss3">Module 3: Safeguard</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Achievements </A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/Judging_Form">Judging Form</A></LI><LI class="dropdown active"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Extras </A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/Collaborations">Collaborations</A></LI><LI class="active"><A href="#">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/Human_Practices">Human Practice</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/Team">Team members</A></LI><LI><A href="https://2017.igem.org/Team:FAFU-CHINA/Attributions">Attributions</A></LI></DIV></DIV><DIV id="abodd"><DIV class="box131"><H1>Plant</H1><A href="#tiao1"><EM>Arabidopsis thaliana</EM></A><H1>PSB</H1><A href="#tiao2">Heavy metal stress experiment of Bacillus megaterium</A><A href="#tiao3">SDS-PAGE</A><A href="#tiao4">The expression of the bacillus bacillus bacillus</A><A href="#tiao5">The protoplasmic system preparation and transformation of the giant bacillus</A><A href="#tiao6">The bacterial total protein was cleaved by SDS</A><A href="#tiao7">Screening of phosphobacteria and separation</A><A href="#tiao8">The solubilizing capacity</A><A href="#tiao9">Protoplast preparation</A><H1>PSB Assisted Plant Absorption</H1><A href="#tiao10">PSB Assisted Plant Absorption</A></DIV></DIV><DIV style="background-color: #F1E6CE; width:100%;"><DIV id="bodydii"><DIV class="box2" id="tiao1"><H2><STRONG>Plant</STRONG></H2></DIV><DIV class="box1"><H2><EM>Arabidopsis thaliana</EM></H2><H1>Seeds sterilization</H1><P>-Weigh about 100 mg <EM>Arabidopsis thalian</EM>a seeds in 1.5 mL microcetrifuge tube. </P><P>-Prepare 75% ethanol 1 mL for washing <EM>Arabidopsis thaliana</EM> seeds and incubate for 20 min. </P><P>-Remove 75% ethanol and replace with 1 mL 95% ethanol. </P><P>-Wash 5x with sterile ddH2O to remove ethanol. </P><H1>1/2 Murashige and Skoog medium</H1><P>-Add 100x MS salts with micronutrients 10 mL and halve macronutrients to 5 mL. </P><P>-Add Sucrose 30 g and Agar 8 g. </P><P>-Add RO pure water to final volume of 1 L. </P><P>-Check and adjust pH to 5.7-5.8 using 1M NaCl. </P><P>-Autoclave for 20 min at 121 ℃</P><P>-Place the bottles on a stir plate at low speed and allow the agar medium to cool to 45–50 °C (until the container can be held with bare hands). </P><P><STRONG>Note</STRONG>:Starting from this step, perform all the steps in sterile conditions in a laminar fl ow hood.</P><H1>Heavy metal stress experiment</H1><P>-Prepare 100 mM CdCl2 by adding 183.1 mg CdCl2 to 10 mL ddH2O</P><P>-Prepare 10, 20, 50, 100 μM by transfering 20, 40, 100, 200 μL to 200 mL 100mM CdCl2 to 1/2 MS medium. </P><P>-Culture seeds to the heavy metal stress medium. </P><P>- Transfer plates into a growth chamber with a 16/8 h (22/18 ℃) day/night regimes at 120 μmolm<SUP>-2</SUP> s<SUP>-1</SUP> irradiation. to</P></DIV><DIV class="box2" id="tiao2"><H2><STRONG>PSB</STRONG></H2></DIV><DIV class="box1"><H1>beef extract peptone medium</H1><P>Beef extract 3 g</P><P>Peptone 10 g</P><P>NaCl 5 g</P><P>ddH<SUB>2</SUB>O 1L</P><P>pH 7.0-8.0</P><P><STRONG>Cadmium part</STRONG></P><P>- Prepare 10mL overnight cultures of <EM>Bacillus megaterium</EM> in beef extract peptone medium. </P><P>- Prepare 100 mg/mL Cd2+ solution by adding 16.308176 g CdCl2 to 100 mL ddH2O. </P><P>- Prepare 10,30, 70,90,100, 200 300 400 …… 1500 mg/L heavy metal stress medium by transferring 0.5, 1.5, 3.5, 4.5,5,10,…… 75μL Cd2+ solution to 5 mL beef extract peptone medium in 10mL tube. </P><P>- Transfer 100 μL overnight cultures(OD600=1) to 10mL tubes with Cd2+..</P><P>- Transfer tubes into a growth Incubator with 37℃/200rpm. </P><P>- Measure the OD600 every 2 hours and write down the results to make growth curve. </P><P>- After 24 hours, take out 100 μL of cultures to spread plate. </P><P><STRONG>Cadmium part</STRONG></P><P>- Prepare 10mL overnight cultures of Bacillus megaterium in beef extract peptone medium. </P><P>- Prepare 100 mg/mL Pb2+ solution. </P><P>- Prepare 10, 100, 300 500,700,1000 mg/L heavy metal stress medium by transferring 0.5, 5,15,25,35,50μL Cd2+ solution to 5 mL beef extract peptone medium in 10mL tube. </P><P>- Transfer 100 μL overnight cultures(OD600=1) to 10mL tubes with Pb2+..</P><P>- Transfer tubes into a growth Incubator with 37℃/200rpm. </P><P>- Measure the OD600 every 2 hours and write down the results to make growth curve. </P><P>- After 24 hours, take out 100 μL of cultures to spread plate. </P></DIV><DIV class="box1"><H2>SDS-PAGE</H2><P> 1. Selection of a SDS-PAGE gel. Typically 10% acrylamide gels are used for high molecular weight (MW) proteins (20-80 kD a), 12% gels for mid range MW proteins (12 - 60 kDa), and 15% gels for low MW proteins (10-40 kDa). Load 5 ul marker or 20 ul protein sample each lane. </P><P> 2. Run at 80V until samples enter the separation gel in gel running buffer (19.3 mM Glycine, 2.5 mM Tris base, 0.1% SDS), and then run at 120V. Electrophoresis is complete when the dye front migrates about 2 mm from the bottom of the gel.. </P><P id="tiao4"> 3. Stain with Coomassie brilliant blue for 1 h, and then destain in destain buffer (50% H2O, 20% AcOH, 30% methanol) for 1h. </P></DIV><DIV class="box1"><H1>Test protein production</H1><P>1. Prepare an overnight culture inoculated with B. megaterium single colonies from a plate (medium including antibiotic such as tetracycline or chloramphenicol) grown 14 h at 37 °C while agitation at 100 rpm. </P><P>2. Inoculate fresh medium with overnight culture in a dilution of 1:100. </P><P>3. Grow the recombinant B. megaterium cells in baffled flasks to an optical density (OD<SUB>578nm</SUB>) of 0.3 - 0.4 at 37 °C under vigorous shaking (250 rpm). </P><P>4. Take a sample as control before induction. </P><P>5. Induce the xylose inducible promoter by addition of 0.5% (w/v) of (D)-xylose. </P><P>6. Incubate at 37 °C with vigorous shaking at 250 rpm. </P><P>7. Withdraw samples every 30 to 60 minutes for OD<SUB>578nm</SUB>-measurement and protein analysis (up to 6 hours after induction). For extracellular protein analysis take 2 ml of cell culture. Intracellular protein analysis requires a higher volume than 2 ml of sample. </P><P>8. Centrifuge each sample to harvest cells and cell free supernatant. </P><P>9. For extracellular protein analysis store the cell free supernatant at 4 °C, and for intracellular protein analysis completely remove supernatant and freeze the cell pellet at -20 °C. </P><H1>Analysis of intracellular proteins</H1><P>1. Resuspend cells in 30 μl of lysis buffer. </P><P>2. Incubate for 30 min at 37 °C while shaking at 1,000 rpm in a thermomixer. An effective cell lysis can be obtained by whirling the samples every 10 minutes. </P><P>3. To separate insoluble fraction (pellet) from the soluble fraction (supernatant), centrifuge cell lysate for 30 min at 4 °C and 13,000 rpm. </P><P>4. Mix 27 µl of supernatant containing soluble proteins with 13 µl of SDS sample buffer. </P><P>5. Completely remove the supernatant and resuspend the pellet in 30 µl of 8 M urea (w/v). Centrifuge for 30 min at 4 °C and 13,000 rpm. </P><P>6. Mix 27 µl of the supernatant with 13 µl of SDS sample buffer. </P><P>7. Heat each sample for 5 min at 95 °C. </P><P id="tiao5">8. Load 7.5 µl of each sample onto an SDS-PAGE gel. </P></DIV><DIV class="box1"><H1>Protoplast preparation</H1><P>A preculture of B. megaterium  was grown overnight in 50 mL SMMP on a shaking platform (100 rpm) at 30 °C for a maximum of 16 h. SMMP (50 mL) was inoculated with 1 mL preculture, incubated at 37 °C, and shaken at 200 rpm until an OD578 of 1.0 was reached (~2 h).Cells were harvested by centrifugation at 2,683 × g and 4 °C for 15 min. The supernatant was removed, and cells were resuspended in 5 mL SMMP. Lysozyme (100 μL 5 mg mL−1dissolved in SMMP and ﬁlter sterilized) was added and incubated at 37 °C with shaking at 100 rpm for 10–20 min until at least 50% of cells were protoplasts (as estimated by use of a light microscope). Protoplasts were incubated on ice for 30 min and then harvested at 1,400 × g and 4 °C for 10 min. The supernatant was removed, and cells were resuspended in 5 mL SMMP.This process was repeated three times to remove excess lysozyme. After the ﬁnal resuspension in 5 mL SMMP, 652.5μL 100% glycerol were added and mixed. Aliquots (500 μL) wereprepared and used fresh or stored at −80 °C (viable for 2 month). </P><H1>Transformation protocol:（tet 10ul/ml）</H1><P>1. Combine 500 μl of protoplast suspension and 3-5 μg of plasmid DNA in a 15 ml tube, one for each transformation. DNA should be purified using a commercial preparation kit. Elute the DNA from the column using water. </P><P>2. Add 1.5 ml of PEG-P, mix gently, and incubate for 2 minutes at RT. </P><P>3. Add 5 ml SMMP and mix carefully by rolling the tube. </P><P>4. Harvest cells by gentle centrifugation (1,300 x g for 10 minutes at RT), discard the supernatant immediately after centrifugation. Supernatant does not have to be removed completely. 
				(Note: do not check for a cell pellet - most of the time it will be invisible) </P><P>5. Add 500 μl of SMMP to remaining supernatant (containing bacterial cells) and transfer to a 1.5 ml microcentrifuge tube. </P><P>6. Incubate at 30 °C for 90 minutes with gentle shaking or rolling of tubes (max. 100 rpm) or incubate for 45 min without shaking followed by another 45 min while shaking at 300 rpm. </P><P>7. Prepare 2.5 ml aliquots of CR5-top agar in sterile tubes. </P><P>8. After incubation at 30 °C add all cells to 2.5 ml top agar, mix gently by rolling the tube between both hands (do not vortex!), and pour onto a pre-warmed plate of LB containing desired antibiotic. </P><P>9. Incubate overnight at 30 °C - expect colonies of varying diameter because some will be covered with agar and others have easier access to air. 
				(Note: bacterial colonies on the top of the agar surface will be shiny) </P><P>10. Streak several single colonies (at least 3) on fresh plates within two days. </P><P>Suggest: </P><P>1. Negative control: protoplasts without DNA</P><P>This is a test reassuring that the protoplasts are not only fully viable but also free of contaminations before using them for transformation. Perform this test according to the transformation protocol as demonstrated below. After incubation at 30 °C, apply CR5-top agar to the protoplasts and split the sample in two portions. You may plate one sample on a LB plate with antibiotic such as tetracycline or chloramphenicol, and another one on a plate without any drug. In this case, bacterial colonies will grow only on a solid medium without antibiotics. </P><P>2.Positive control: protoplasts transformed with an empty plasmid</P><P>This is a test control for a successful transformation and should yield lots of colonies on the plates supplemented with an antibiotic (here: tetracycline or chloramphenicol). If this transformation works well, but you have problems with the plasmid containing your target gene, the problem is most likely associated with your construct. </P><H1>Protoplast Transformation Medium</H1><P><STRONG>2  SMM (solubilize in the given order in 150 ml of deion. water!)</STRONG></P><P>• 1.16 g maleic acid (40 mM) </P><P>• 800 mg NaOH (80 mM) </P><P>• 2.03 g MgCl<SUB>2</SUB> x 6H<SUB>2</SUB>O (40 mM) </P><P>• 85.58 g sucrose (1 M) </P><P>• mix and fill with deion. water to 250 ml</P><P>• sterilize by filtration</P><P><STRONG>2 x AB3 (Antibiotic Medium No. 3, DIFCO) </STRONG></P><P>• 2 g Peptone</P><P>• 0.6 g Yeast extract</P><P>• 0.6g `Lab-Lemco’ powder</P><P>• 0.2Glucose</P><P>• 1.4 g Sodium chloride</P><P>• 1.472gDipotassium hydrogen phosphate</P><P>•0.528gPotassium dihydrogen phosphate</P><P>• adjust pH 7.0 ± 0.2 </P><P>•in 200 ml deion. water</P><P>• autoclave 15 min</P><P><STRONG>PEG-P</STRONG></P><P>• solubilize 20 g PEG-6000 with 1  SMM and fill to 50 ml</P><P>• autoclave 11 min</P><P><STRONG>SMMP</STRONG></P><P>• 2  AB3 and 2  SMM 1:1 (freshly prepared!) </P><P><STRONG>CR5-top agar</STRONG></P><P>prepare separately for 500 ml: </P><P>solution A</P><P>• 51.5 g sucrose</P><P>• 3.25 g MOPS</P><P>• 300 mg NaOH</P><P>• add deionized water to 250 ml</P><P>• adjust pH to 7.3 with NaOH</P><P>• sterilize by filtration</P><P><STRONG>solution B</STRONG></P><P>• 2 g agar</P><P>• 100 mg casamino acids</P><P>• 5 g yeast extract</P><P>• add deion. water to 142.5 ml</P><P>• autoclave for 15 min</P><P><STRONG>8  CR5 salts</STRONG></P><P>• 1.25 g K<SUB>2</SUB>SO<SUB>4</SUB></P><P>• 50 g MgCl<SUB>2</SUB>  6 H<SUB>2</SUB>O</P><P>• 250 mg KH<SUB>2</SUB>PO<SUB>4</SUB></P><P>• 11 g CaCl<SUB>2</SUB></P><P>• solubilize in 625 ml deion. water</P><P>• autoclave for 15 min</P><P><STRONG>12% proline</STRONG></P><P>• 3 g L-proline</P><P>• add deion. water to 25 ml</P><P>• sterilize by filtration</P><P><STRONG>20% glucose</STRONG></P><P>• 5 g glucose</P><P>• add deion. water to 25 ml</P><P>• sterilize by filtration</P><P><STRONG>for a 2.5 ml portion of CR5-top agar add the following (in the given order!): </STRONG></P><P>• 1.25 ml solution A</P><P>• 288 μl CR5 salts</P><P>• 125 μl 12% proline</P><P>• 125 μl 20% glucose</P><P>90 minutes after transformation: </P><P>• boil solution B</P><P>• add 713 μl to the provided CR5-top agar</P><P>• immediately add the regenerated protoplasts, and put onto prewarmed agar plates containing appropriate antibiotic (tetracycline or chloramphenicol). </P><P><STRONG>lysis buffer</STRONG></P><P>• 100 mM Na<SUB>3</SUB>PO<SUB>4</SUB></P><P>• 5 mg/ml lysozyme</P><P>• pH 6.5 (adjust with H<SUB>3</SUB>PO<SUB>4</SUB>) </P><P>• add 1 µl of a 1 M MgSO<SUB>4</SUB> solution and</P><P>• 2 μl HS-Nuclease* (250 U/μl, cat.# GENUC10700-01, final concentration of 500 U/ml) per ml lysis buffer shortly before use. </P><P>6.4. Recombinant protein production and secretion </P><H1>Test protein production</H1><P>1. Prepare an overnight culture inoculated with B. megaterium single colonies from a plate (medium including antibiotic such as tetracycline or chloramphenicol) grown 14 h at 37 °C while agitation at 100 rpm. </P><P>2. Inoculate fresh medium with overnight culture in a dilution of 1:100. </P><P>3. Grow the recombinant B. megaterium cells in baffled flasks to an optical density (OD578nm) of 0.3 - 0.4 at 37 °C under vigorous shaking (250 rpm). </P><P>4. Take a sample as control before induction. </P><P>5. Induce the xylose inducible promoter by addition of 0.5% (w/v) of (D)-xylose. </P><P>6. Incubate at 37 °C with vigorous shaking at 250 rpm. </P><P>7. Withdraw samples every 30 to 60 minutes for OD578nm-measurement and protein analysis (up to 6 hours after induction). For extracellular protein analysis take 2 ml of cell culture. Intracellular protein analysis requires a higher volume than 2 ml of sample. </P><P>8. Centrifuge each sample to harvest cells and cell free supernatant. </P><P>9. For extracellular protein analysis store the cell free supernatant at 4 °C, and for intracellular protein analysis completely remove supernatant and freeze the cell pellet at -20 °C. </P><H1>Analysis of intracellular proteins</H1><P>1. Resuspend cells in 30 μl of lysis buffer. </P><P>2. Incubate for 30 min at 37 °C while shaking at 1,000 rpm in a thermomixer. An effective cell lysis can be obtained by whirling the samples every 10 minutes. </P><P>3. To separate insoluble fraction (pellet) from the soluble fraction (supernatant), centrifuge cell lysate for 30 min at 4 °C and 13,000 rpm. </P><P>4. Mix 27 µl of supernatant containing soluble proteins with 13 µl of SDS sample buffer. </P><P>5. Completely remove the supernatant and resuspend the pellet in 30 µl of 8 M urea (w/v). Centrifuge for 30 min at 4 °C and 13,000 rpm. </P><P>6. Mix 27 µl of the supernatant with 13 µl of SDS sample buffer. </P><P>7. Heat each sample for 5 min at 95 °C. </P><P id="tiao6">8. Load 7.5 µl of each sample onto an SDS-PAGE gel. </P></DIV><DIV class="box1"><H2>The bacterial total protein was cleaved by SDS</H2><P>(1) shake the culture medium and take 1mL in the centrifuge tube, 10000rpm centrifugation 10min. </P><P> (2) remove the supernatant, leaving about 50μl left, add a small amount of pH 7.4 PBS washed three times, leaving about 50μl. </P><P> (3) Add 50μL Loading Buffer, 0.31g DTT to suspend the suspension. </P><P> (4) Place the centrifuge tube in a water bath at 100 ° C for 10 min. </P><P> (5) 12000rpm centrifugation 5 ~ 10min, supernatant is extracted from the protein. </P><P id="tiao7"> (6) Move the supernatant to the new centrifuge tube, add a small amount of Loading Buffer (about 5μl) to sample. </P></DIV><DIV class="box1"><H2>Screening of phosphobacteria and separation</H2><H1> Soil sampling</H1><P>Bacillus genus in the legume root distribution,we select root soil samples as experiment materials. According to the scene, in the same piece of ground, randomly select 5 samples, two soil layers (0-10cm, 10-20cm) to collect soil samples, after the same levels of soil mix, dry, about taking 1kg about 2mm screenings, are assembled into a sterile plastic bag, indoor -20℃ refrigerator, and separated operations. </P><H1>Media</H1><P><STRONG>Organic phosphorus bacteria liquid medium</STRONG></P><P>Glucose         10g </P><P> (NH4)2SO4      0.5g  </P><P>NaCI           0.3g </P><P>KCl            0.3g </P><P>MgSO4·7H2O    0.3g </P><P>FeSO4·7H2O     0.03g </P><P>MnSO4· H20     0.0227g </P><P>CaCa3           5g </P><P> (NH4)H2PO4     0.5 g</P><P>Distilled water    1000mL</P><P>Egg yolk10 ml</P><P> (Egg yolk for sterile saline solution mixed with egg yolk 1:1, PH7). 0-7.5 (no sterilization) </P><P><STRONG>P solid culture medium</STRONG></P><P>Glucose          10g , </P><P> (NH4) 2SO4       0.5g , </P><P>NaCI             0.3g  </P><P>KCl              0.3g , </P><P>MgSO4·7H2O     0.3g , </P><P>Ca3 (PO4) 2       10g </P><P>FeSO4·7H2O      0.03g, </P><P>MnS04·4H2O      0.03g, </P><P>Agar              18~20g, </P><P>Distilled water 1000mL</P><P>PH                7.0-7.5</P><H1>Screening of phosphobacteria</H1><P>Take 1g powders joined to the 100mL of organic phosphorus of soil sample in liquid medium at 28 degrees, Shaker 48 h under 200~250 r/min. Dilute solubilizing bacteria by the plate dilution method: press 10-1, 10-2, 10-3, 10-4, 10-5, 10-6, 10-7 gradient of the diluted, for the last 5 dilution (10-3~10-7) taking in 200 uL supernatant, repeated for each dilution make 3, inorganic phosphorus on solid medium plate coating, at 28 ° cincubator training 3~5 d. </P><H1> Isolation of phosphate solubilizing bacteria</H1><P>Observe the growth plate, and phosphate. Various inorganic phosphobacteria solid medium colonies respectively crossed the plate separation and purification to obtain pure strain. </P><H1>Bacteria</H1><P id="tiao8">Derived from the pure culture of save the glycerol to final concentration of 15% tubes and inoculated on the inclined plane, and then placed 28~30℃ constant temperature incubator for 2 d, preserved at 4°c standby. </P></DIV><DIV class="box1"><H2>The solubilizing capacity</H2><H1>Determination of solubilizing capacity in solid culture</H1><P>Solubilizing method is used. Randomly picked a few(not less than 6)large solubilizing bacteria colonies with the medium, repeat 3 times, at 28°c incubator training in 7d, according to the phosphorus circle diameter and diameter ratio of colony (HD/CD)-solubilizing capability of size and identified various strains. &quot;HD for the phosphorus circle diameter, CD as colony diameter&quot; {solubilizing strains does not appear to do controlled trials. }</P><H1>determination of soluble phosphorus in liquid culture</H1><P>Picking test colonies of strain inoculated to inorganic phosphorus in 100mL liquid medium, 28, 200~250 r/min Shaker,every 24h take the supernatant, with molybdenum blue Colorimetry determination of effective phosphorus in culture medium. Simultaneous determination of 7d. {solubilizing strains does not appear to do controlled trials. }</P><P><STRONG>Molybdenum blue Colorimetry</STRONG></P><P>Under the 28℃ in liquid medium, standing take the filtrate of filter 5ml, add 15% H2S04, 10ml, 10ml adding tartaric acid solution, shake the solution of ammonium aluminum joined 4ml, 3/4 dilute to volume with water bottles, place in boiling water for 2 minutes, remove the reducing agent add 1 ml, color cool and dilute to scale. And color, according to phosphorus content in the filtrate is obtained standard curve. According to phosphorus content of phosphorus paint time curve. Using solubilizing strains to do controlled trials does not appear. </P><P><STRONG> Molybdenum blue colorimetric method measuring phosphorus content using the reagent preparation</STRONG></P><P>Concentrated sulphuric acid (proportion of 1.84, AR). </P><P>Catalyst: potassium (chemically pure) 100 g, copper sulfate (chemically pure) 10 grams and selenium powder (grade) 1 g, even after mixing abrasive through 80th sieve, bottle storage. </P><P>4% Sulfate of ammonium molybdate solution: 18 g of ammonium molybdate solution dissolved in 300 ml of concentrated sulfuric acid, add 420 ml distilled water.</P><P>Reducing agents: 15 grams of NaHS04 dissolved in 250 ml of water, add 1.5 g Na2SO3 and 0.5 g 1, 2, 4, Carl Zeiss-amino phenol sulfonic acid, dilute to 500 ml storage after the dissolution in the Brown bottle. </P><P>2% tartaric acid solution. </P><P> Standard solution of phosphorus: KH2P04 0.2196 grams of recrystallization, used 0.1N H2S04 dissolve and constant volume to 1 L, 1 ml of the solution containing 50 mg phosphorus. </P><P><STRONG>Standard curves curves</STRONG></P><P>Take 1~5 ml in phosphate standard solution in a 50ml bottle, join 15% H2SO4 10ml, tartaric acid solution 10ml, shake the solution of ammonium aluminum added 4ml, 3/4 dilute to volume with water bottle, placed in a boiling-water bath for 2 minutes, remove the quick add 1 ml of a reducing agent. Color, cool, and dilute to scale on a photoelectric colorimeter, wavelength 660 nm is better than color. With optical density as the ordinate, with concentrations as abscissa, draw standard curve. </P><H1> Molecular identification of phosphobacteria strains </H1><P><STRONG> PCR products electrophoresis</STRONG></P><P id="tiao9"><STRONG> Sequence analysis and construction of phylogenetic trees</STRONG></P></DIV><DIV class="box1"><H2>Protoplast preparation</H2><P>A preculture of B. megaterium  was grown overnight in 50 mL SMMP on a shaking platform (100 rpm) at 30 °C for a maximum of 16 h. SMMP (50 mL) was inoculated with 1 mL preculture, incubated at 37 °C, and shaken at 200 rpm until an OD578 of 1.0 was reached (~2 h).Cells were harvested by centrifugation at 2,683 × g and 4 °C for 15 min. The supernatant was removed, and cells were resuspended in 5 mL SMMP. Lysozyme (100 μL 5 mg mL−1dissolved in SMMP and ﬁlter sterilized) was added and incubated at 37 °C with shaking at 100 rpm for 10–20 min until at least 50% of cells were protoplasts (as estimated by use of a light microscope). Protoplasts were incubated on ice for 30 min and then harvested at 1,400 × g and 4 °C for 10 min. The supernatant was removed, and cells were resuspended in 5 mL SMMP.This process was repeated three times to remove excess lysozyme. After the ﬁnal resuspension in 5 mL SMMP, 652.5μL 100% glycerol were added and mixed. Aliquots (500 μL) wereprepared and used fresh or stored at −80 °C (viable for 2 month). </P><H1>Transformation protocol:（tet 10ul/ml）</H1><P>1. Combine 500 μl of protoplast suspension and 3-5 μg of plasmid DNA in a 15 ml tube, one for each transformation. DNA should be purified using a commercial preparation kit. Elute the DNA from the column using water. </P><P> 2. Add 1.5 ml of PEG-P, mix gently, and incubate for 2 minutes at RT. </P><P>3. Add 5 ml SMMP and mix carefully by rolling the tube. </P><P>4. Harvest cells by gentle centrifugation (1,300 x g for 10 minutes at RT), discard the supernatant immediately after centrifugation. Supernatant does not have to be removed completely.
				(Note: do not check for a cell pellet - most of the time it will be invisible) </P><P>5.  Add 500 μl of SMMP to remaining supernatant (containing bacterial cells) and transfer to a 1.5 ml microcentrifuge tube. </P><P>6.  Incubate at 30 °C for 90 minutes with gentle shaking or rolling of tubes (max. 100 rpm) or incubate for 45 min without shaking followed by another 45 min while shaking at 300 rpm. </P><P>7. Prepare 2.5 ml aliquots of CR5-top agar in sterile tubes. </P><P>8.  After incubation at 30 °C add all cells to 2.5 ml top agar, mix gently by rolling the tube between both hands (do not vortex!), and pour onto a pre-warmed plate of LB containing desired antibiotic. </P><P> 9. Incubate overnight at 30 °C - expect colonies of varying diameter because some will be covered with agar and others have easier access to air.
				(Note: bacterial colonies on the top of the agar surface will be shiny) </P><P>10. Streak several single colonies (at least 3) on fresh plates within two days. </P><P><STRONG> 2  SMM (solubilize in the given order in 150 ml of deion. water!) </STRONG></P><P>• 1.16 g maleic acid (40 mM) </P><P>• 800 mg NaOH (80 mM) </P><P>• 2.03 g MgCl2 x 6H2O (40 mM) </P><P>• 85.58 g sucrose (1 M) </P><P>• mix and fill with deion. water to 250 ml</P><P>• sterilize by filtration</P><P><STRONG> 2 x AB3 (Antibiotic Medium No. 3, DIFCO) </STRONG></P><P>• 2 g Peptone</P><P>• 0.6 g Yeast extract</P><P>• 0.6g `Lab-Lemco’ powder</P><P>• 0.2Glucose</P><P>• 1.4 g Sodium chloride</P><P>• 1.472gDipotassium hydrogen phosphate</P><P>•0.528gPotassium dihydrogen phosphate</P><P>• adjust pH 7.0 ± 0.2 </P><P>•in 200 ml deion. water</P><P>• autoclave 15 min</P><P><STRONG> PEG-P</STRONG></P><P>• solubilize 20 g PEG-6000 with 1  SMM and fill to 50 ml</P><P>• autoclave 11 min</P><P><STRONG> SMMP</STRONG></P><P>• 2  AB3 and 2  SMM 1:1 (freshly prepared!) </P><P><STRONG> CR5-top agar</STRONG></P><P>prepare separately for 500 ml: </P><P><STRONG> solution A</STRONG></P><P>• 51.5 g sucrose</P><P>• 3.25 g MOPS</P><P>• 300 mg NaOH</P><P>• add deionized water to 250 ml</P><P>• adjust pH to 7.3 with NaOH</P><P>• sterilize by filtration</P><P><STRONG> solution B</STRONG></P><P>• 2 g agar</P><P>• 100 mg casamino acids</P><P>• 5 g yeast extract</P><P>• add deion. water to 142.5 ml</P><P>• autoclave for 15 min</P><P><STRONG> 8  CR5 salts</STRONG></P><P>• 1.25 g K2SO4</P><P>• 50 g MgCl2  6 H2O</P><P>• 250 mg KH2PO4</P><P>• 11 g CaCl2</P><P>• solubilize in 625 ml deion. water</P><P>• autoclave for 15 min</P><P><STRONG> 12% proline</STRONG></P><P>• 3 g L-proline</P><P>• add deion. water to 25 ml</P><P>• sterilize by filtration</P><P><STRONG> 20% glucose</STRONG></P><P>• 5 g glucose</P><P>• add deion. water to 25 ml</P><P>• sterilize by filtration</P><P><STRONG> for a 2.5 ml portion of CR5-top agar add the following (in the given order!): </STRONG></P><P>• 1.25 ml solution A</P><P>• 288 μl CR5 salts</P><P>• 125 μl 12% proline</P><P>• 125 μl 20% glucose</P><P>90 minutes after transformation: </P><P>• boil solution B</P><P>• add 713 μl to the provided CR5-top agar</P><P>• immediately add the regenerated protoplasts, and put onto prewarmed agar plates containing appropriate antibiotic (tetracycline or chloramphenicol). </P><P><STRONG> lysis buffer</STRONG></P><P>• 100 mM Na3PO4</P><P>• 5 mg/ml lysozyme</P><P>• pH 6.5 (adjust with H3PO4) </P><P>• add 1 µl of a 1 M MgSO4 solution and</P><P>• 2 μl HS-Nuclease* (250 U/μl, cat.# GENUC10700-01, final concentration of 500 U/ml) per ml lysis buffer shortly before use. </P><P>6.4. Recombinant protein production and secretion </P><H1>Test protein production</H1><P>1. Prepare an overnight culture inoculated with B. megaterium single colonies from a plate (medium including antibiotic such as tetracycline or chloramphenicol) grown 14 h at 37 °C while agitation at 100 rpm. </P><P>2. Inoculate fresh medium with overnight culture in a dilution of 1:100. </P><P>3. Grow the recombinant B. megaterium cells in baffled flasks to an optical density (OD578nm) of 0.3 - 0.4 at 37 °C under vigorous shaking (250 rpm). </P><P>4. Take a sample as control before induction. </P><P>5. Induce the xylose inducible promoter by addition of 0.5% (w/v) of (D)-xylose. </P><P>6. Incubate at 37 °C with vigorous shaking at 250 rpm. </P><P>7. Withdraw samples every 30 to 60 minutes for OD578nm-measurement and protein analysis (up to 6 hours after induction). For extracellular protein analysis take 2 ml of cell culture. Intracellular protein analysis requires a higher volume than 2 ml of sample. </P><P>8. Centrifuge each sample to harvest cells and cell free supernatant. </P><P>9. For extracellular protein analysis store the cell free supernatant at 4 °C, and for intracellular protein analysis completely remove supernatant and freeze the cell pellet at -20 °C. </P><H1> Analysis of intracellular proteins</H1><P>1. Resuspend cells in 30 μl of lysis buffer. </P><P>2. Incubate for 30 min at 37 °C while shaking at 1,000 rpm in a thermomixer. An effective cell lysis can be obtained by whirling the samples every 10 minutes. </P><P>3. To separate insoluble fraction (pellet) from the soluble fraction (supernatant), centrifuge cell lysate for 30 min at 4 °C and 13,000 rpm. </P><P>4. Mix 27 µl of supernatant containing soluble proteins with 13 µl of SDS sample buffer. </P><P>5. Completely remove the supernatant and resuspend the pellet in 30 µl of 8 M urea (w/v). Centrifuge for 30 min at 4 °C and 13,000 rpm. </P><P>6. Mix 27 µl of the supernatant with 13 µl of SDS sample buffer. </P><P>7. Heat each sample for 5 min at 95 °C. </P><P>8. Load 7.5 µl of each sample onto an SDS-PAGE gel. </P></DIV><DIV class="box2" id="tiao10"><H2><STRONG>PSB Assisted Plant Absorption</STRONG></H2></DIV><DIV class="box1"><H1>Preparation</H1><P>Improved Hoagland Nutrient Solution</P><P>KNO3                   0.170g/L    </P><P>KH2PO4                0.120g/L</P><P>MgSO4*7H2O       0.163g/L、</P><P>Ca(NO3)2*4H2O   0.393g/L    </P><P>H3BO3                  2.067mg/L</P><P>MnSO4                 7.433mg/L </P><P>ZnSO4                  2.867mg/L</P><P>Na2MoO4             0.083mg/L </P><P>CuSO4                  0.008mg/L </P><P>Na2EDTA              2.480mg/L</P><P>Acid bucket configuration</P><P>Analysis of pure HNO3：HClO4＝7：1</P><H1>Sample collection and treatment of plant</H1><P>We contacted the teachers of the College of Resources and Environment to learn how to handle samples. With the help of teachers, we determinated the content of Cd/Pb by AAS </P><P>1.Immerse 15min in 20mmo1/L Na2-EDTA to remove the heavy metal adsorbed on the root surface</P><P>2.Dry and crush sample with smash meter with 120℃ for 30 mins,then turn to 65℃ for 8hours</P><P>3.According to the quality of the sample after grinding,transfer the sample and 6ml mixed acid（HNO3：HClO4＝5：1） into 50ml flask to heat in the electric heating with 120 ℃ for 8 hours </P><P>4.Cool and move sample into 15ml capacity bottle, and constant volume to mark line. </P><P>5.Take the Clarification solution after filtering flocculent by 0.45 nm filter</P><P>6..Measure the contents of Cd /Pb in deferent kinds of agrochemical by AAS</P><P>7. Data analysis</P><H1>Determination of Heavy Metals in Fertilizers </H1><P>The determination of the total amount of Fe, Mn, Cu, Pb, Zn and Cd was digested with HNO3-HClO4, </P><P>The sample was pulverized with the pulverizer, </P><P>A sample of 1.00 g of each sample was weighed into a 100 ml Erlenmeyer flask</P><P>B. Add 6ml of mixed acid (HNO3: HClO4 = 5: 1) and mix well, </P>
				Placed in the hot plate on the low temperature to produce a large number of white smoke, <P>The solution was yellow and then cooled, </P><P>Add a small amount of water, </P><P>Boil to remove residual nitric acid to produce white smoke. </P><P>C. repeat the treatment twice, let them let the cold, </P><P>Into the 250ml volumetric flask, </P><P>The residual liquid in the flask is washed and set to a graduated line. </P><P>After dilution, the solution contains floc, filter, take clear solution. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>