<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Peking_Design_PC_Reporter skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Peking/Design/PC Reporter</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="container"><HEADER class="clearfix"><DIV class="navbar navbar-default navbar-top"><DIV class="container"><DIV class="navbar-collapse collapse" style="padding-top:5px;padding-bottom:0"><UL class="nav navbar-nav navbar-right " style="padding-bottom:15px;padding-top:10px"><LI><A href="https://2015.igem.org/Team:Peking/JudgingCriteria">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Peking/JudgingCriteria">Judging Criteria</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Collaborations">Collaborations</A></LI></UL><LI><A class="active" href="https://2015.igem.org/Team:Peking/Design">Project</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design">Overview</A></LI><LI><A class="active" href="http://2015.igem.rg/Team:Peking/Design/PC_Reporter">Paired <SPAN style="text-transform:lowercase">d</SPAN>Cas9 Reporter</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design/Isothermal">Iso-<SPAN style="text-transform:lowercase">t</SPAN>hermal Amplification</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Device">Hardware</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design/Speculation">Speculation</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Modeling">Array Design</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Modeling/Analysis">Analysis Algorithm</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Background">Facts about TB</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Consultation">Consultation and Interview</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Engagement">Public Engagement</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Outreach">Ethics and Economics Issue</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Team">Lab</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Safety">Safety</A></LI></DIV></DIV></DIV></HEADER><DIV class="page-banner"><DIV class="container"><DIV class="row"><DIV class="col-md-6"><H2 style="font-size:20px; margin-bottom:5px; padding-bottom:0">P<SPAN style="text-transform:lowercase">roject</SPAN></H2><P style="margin-top:0px;font-size:14px" id="Introduction">It takes half your life before you discover life is a do-it-yourself project.</P></DIV><DIV class="col-md-6"><UL class="breadcrumbs"><LI><A href="https://2015.igem.org/Team:Peking">Home</A></LI><LI>Project</LI></UL></DIV></DIV></DIV></DIV><DIV id="content"><DIV class="container"><DIV class="row blog-page"><DIV class="col-md-3 sidebar left-sidebar"><DIV id="sidebar1" class="widget widget-categories"><H4 style="font-size:18px">PC R<SPAN style="text-transform:lowercase">eporter</SPAN></H4><UL><LI><A href="#Introduction">Introduction</A></LI><LI><A href="#Design">Design</A></LI><LI><A href="#Results">Results</A></LI><LI><A href="#Methods">Methods</A></LI></UL></DIV><DIV id="sidebar2" class="widget widget-categories"><H4 style="font-size:18px">PC R<SPAN style="text-transform:lowercase">eporter</SPAN></H4><UL><LI><A href="#Introduction">Introduction</A></LI><LI><A href="#Design">Design</A></LI><LI><A href="#Results">Results</A></LI><LI><A href="#Methods">Methods</A></LI></UL></DIV></DIV><DIV class="col-md-9 page-content" style="text-align:justify;"><DIV id="PCReporters"><H3 class="classic-title" style="margin-top:50px"><SPAN>Introduction</SPAN></H3><DIV style="margin-top:30px; margin-bottom:0"><P><BIG><B>Background</B></BIG></P><P>Tuberculosis (TB) ranks as the 2nd leading cause of death among various infectious diseases [1]. The fact that earlier diagnosis and treatment indicate better prognosis attaches great importance to the diagnostic services for TB patients. The current methods include tuberculin skin test, blood test and sputum culture. However, tuberculin skin test is not sensitive, blood test is risky and sputum culture needs a waiting period as long as 1-2 months which will delay timely treatment. Methods based on nucleic acid detection (NAD) are highly sensitive, safe and rapid; however, they are not widely applied to clinical practice. Why? </P><P>After consulting some front-line practitioners of TB control, we found that NAD has high false positive rate because non-specific amplification frequently occurs and current methods of NAD are not able to read out sequence-specific information; this makes them barely reliable. What' more, NAD requires expensive and clumsy instruments which are not accessible to rural areas where TB is epidemic (For more detailed information, see <A href="https://2015.igem.org/Team:Peking/Practices" style="color:#00afd1">Human Practice</A>). Therefore, Peking iGEM 2015 is devoted to building a new reporter system that enables NAD to be specific, reliable and less instrument-dependent.</P><P><BIG><B>CRISPR</B></BIG></P><P>To improve the specificity, we need a system that can read out sequence information directly. The first thing coming to our mind is the clustered regularly interspaced, short palindromic repeats (CRISPR)-associated protein 9 (Cas9) system, which is originally from prokaryotic immune system [2]. Cas9 is a sequence-specific, DNA cleavage protein guided by single-guide RNA (sgRNA); they form a complex and can be easily programmed to recognize any target sequence [3]; in previous studies it was engineered to be catalytically dead (dCas9), thus to form a programmable DNA binding complex [4]. But how can we convert the invisible sequence information into easily measurable signal?
               </P><P><BIG><B>Split luciferase</B></BIG></P><P>Split enzymes [5] were selected as the candidates to convert the presence of target DNA sequence into measureable signal. Each protein fragment by itself is inactive; when the fragments are reassembled, the enzymatic activity of the original protein would be reconstituted, thus providing easily measurable readout through enzymatic reaction. Among diverse split enzymes, we chose split luciferase which produces bioluminescence signal (Figure 1).</P><DIV id="figure1" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left" id="Design"><SMALL><B>Figure 1. Illustration of the working mechanism of split luciferase.</B> A functional luciferase is split into two inactive fragments (Nluc, orange and Cluc, yellow).  Each fragment is fused to an interacting domain (gray) that tends to dimerize. The interacting domains cause non-covalent complementation of Nluc and Cluc to reconstitute the enzymatic activity, thus to emit measurable bioluminescence.</SMALL></P></DIV></DIV></DIV></DIV><H3 class="classic-title" style="margin-top:50px"><SPAN>Design</SPAN></H3><DIV style="margin-top:30px; margin-bottom:0"><P>We combined dCas9 and split luciferase to obtain a paired dCas9 (PC) Reporter system that can not only extract the sequence information of target DNA, but also quantitatively visualize its abundance through bioluminescence emission. As shown in the schematic diagram (Figure 2), when N-luciferase-dCas9:sgRNA complex and C-luciferase-dCas9:sgRNA complex simultaneously bind to adjacent sites on a target DNA, the two fragments of luciferase are brought into proximity and the subsequent complementation brings measurable bioluminescence signal.</P><DIV id="picture2" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left" id="Results"><SMALL><B>Figure 2. Schematic of the paired dCas9 (PC) Reporter system.</B> dCas9 was fused with fragments of split luciferase (Nluc and Cluc), respectively, to form two kinds of split luciferase-dCas9:sgRNA complexes. In the presence of target DNA, the complexes, respectively, bind to two adjacent recognition sites, bringing Nluc and Cluc into proximity to produce bioluminescence signal.</SMALL></P></DIV></DIV></DIV><H3 class="classic-title" style="margin-top:50px"><SPAN>Results</SPAN></H3><DIV style="margin-top:30px; margin-bottom:0"><P><BIG><B>Validating the functional reconstitution of split luciferase </B></BIG></P><P>We began by examining whether our split luciferase could be really functionally reconstituted by protein fragment-assisted complementation (PFAC) [7,8]. For the complementation-assisting partners, we chose rapamycin-binding domain (FRB) of human mTOR that binds with high affinity to FK-506-binding protein 12 (FKBP); rapamycin is able to induce the dimerization to form a FRB-rapamycin-FKBP complex (Figure 3a).  A pair of widely-used split firefly (Photinus pyralis) luciferase fragments (Nluc 416/Cluc 398) was then selected to construct Nluc-FRB and FKBP-Cluc fusion proteins; they are, therefore, subjected to rapamycin induced complementation [9]. Our results of PFAC assay on our split luciferase fragments confirmed that the luciferase activity is able to be successfully reconstituted in a rapamycin-dependent manner (Figure 3b).</P><DIV id="picture3" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 3. Rapamycin-induced Nluc-FRB/FKBP-Cluc complementation.</B><B>(a)</B> The working mechanism of rapamycin induced dimerization. The interacting protein partners (FRB and FKBP) get closer and dimerize soon after rapamycin is added (40nM) [10], thus to reconstitute the enzymatic activity of luciferase. <B>(b)</B> The experimental data. Error bars denote s.d.; n=3. </SMALL></P></DIV></DIV></DIV><P><BIG><B>The location of His-tag</B></BIG></P><P>Despite the fact that His-tag is widely used for protein purification, it is well-known that His-tag might have some considerable impact on protein structure. Hence, we first tested the activity of C-terminally and N-terminally His-tagged split luciferase-dCas9 fusion proteins to see whether the location of His-tag matters. We found that when the His-tag is attached to Nluc or Cluc, the activity of fusion protein would dramatically decrease (See working mechanism of PC reporter in Figure 2); this is probably due to the interruption of luciferase reconstitution by His-tag (Figure 4). When the His-tag is attached to dCas9, the enzymatic activity was reconstituted significantly. Therefore, we chose split luciferase-dCas9 fusion proteins with His-tagged dCas9 for the following study.</P><DIV id="picture4" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 4. The impact of His-tag location on the reconstitution of luciferase activity.</B> When attached to dCas9, His-tag has minimal impact on the complementation of Nluc and Cluc.</SMALL></P></DIV></DIV></DIV><P><BIG><B>Construction and optimization of PC Reporter system</B></BIG></P><P>Considering that dCas9 is a huge protein and is structurally complex, we need to figure out the configuration of PC Reporter system. We first fused split luciferase to either N or C terminus of dCas9 (Figure 5a) to see its influence. Then, provided that initial binding of dCas9 depends on the protospacer adjacent motif (PAM, a short 3’ motif adjacent to target sequence) [11], four sets of sgRNA orientation settings were also tested (Figure 5b). One set placed two PAM sequences distal from the spacer sequence between the sgRNA pair, with the 5' end of the sgRNA adjacent to the spacer (PAM-out) while another set put both PAM sequences adjacent to the spacer (PAM-in). PAM-direct 1 and PAM-direct 2 combine both sgRNA orientations with one PAM sequence adjacent to and another distal from the spacer (Figure 5b).</P><P>In total, 16 pairs of split luciferase-dCas9:sgRNA complexes were constructed and tested (Figure 5c). Most of these pairs worked very well.  Among them we selected Nluc-dCas9/Cluc-dCas9, the most robust protein construct with minimal intergroup difference, and PAM-out, the sgRNA orientation exhibiting the strongest luminescence, for the following study. </P><DIV id="picture5" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 5. Construction and optimization of PC Reporter system.</B><B>(a)</B> 4 different split luciferase-dCas9 fusion strategies.  <B>(b)</B> 4 different sgRNA orientation settings. In orientation PAM-out, the pair of PAM sequences are distal from the spacer sequence, with the 5' end of the sgRNA adjacent to the spacer; in orientation PAM-in, the pair of PAM sequences are adjacent to the spacer sequence, with the 3' end of the sgRNA in proximity to the spacer; in orientation PAM-direct 1 and PAM-direct 2, one PAM sequence is adjacent to and another distal from the spacer. <B>(c)</B> The experimental results of PC Reporter systems using 16 different (4 different constructs X 4 different sgRNA orientation settings) configurations.</SMALL></P></DIV></DIV></DIV><P>As the reconstitution of enzymatic activity was constrained by the distance between Nluc and Cluc [11], we varied the spacer length from 5 bp to 107 bp (Figure 6) to find how it affects the performance of our PC Reporter system. Results showed that the PC Reporter system worked with a preference for spacer length of about 21 bp when two fusion proteins bind at the same side of the DNA double helix. As the spacer length increased, the luminescence intensity decreased. At the distance of 28 bp where two fusion proteins bind at the opposite side of the DNA double helix, the reconstitution was unstable, thus producing a lower luminescence. However, the luminescence went stronger again when two fusion proteins bind at the same side at the length of 33 bp. And finally it was too far for split luciferase to reconstitute again.</P><DIV id="picture6" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 6. The effect of spacer length variation on the performance of PC Reporter system.</B> Spacer is defined as the sequence between the sgRNA pairs. The spacer length varies from 5 bp to 107 bp. Note that the effect of spacer length exhibits a period of 10-11bp, which is consistent with the structure of DNA double helix.</SMALL></P></DIV></DIV></DIV><P>Taking all the above together, the final design of our PC Reporter system was determined: both fragments of split luciferase fused to N-terminus of dCas9, PAM-out sgRNA orientation, and spacer length of 21 bp. Next we set out to validate whether PC reporter can read out the sequence-specific information using crude PCR product instead of well-purified DNA samples. </P><P><BIG><B>Applying PC reporter to crude NAD product</B></BIG></P><P>We first carried out specificity test using PC reporter system to distinguish the target plasmid (<A href="http://parts.igem.org/Part:BBa_K909009" style="color:#00afd1">BBa_K909009</A>) from different kinds of non-specific controls (Figure 7a,b,c). The results showed that the luminescence intensity using the positive target is significantly higher than that of non-specific controls (Figure 7d).</P><DIV id="picture7" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 7. Specificity test of PC Reporter system.</B><B>(a)</B> Positive target with two sites recognized simultaneously by a pair of split luciferase-dCas9:sgRNA complexes. <B>(b)</B> Control with only one site recognized by &quot;half pair&quot;. <B>(c)</B> Negative control with no site to be recognized. <B>(d)</B> Experimental data. Error bars denote s.d.; n=3. </SMALL></P></DIV></DIV></DIV><P>Also, we need to test the sensitivity of PC Reporter system. As shown in Figure 8, PC reporter can still work at the concentration of ~0.1nM target for both plasmid (Figure 8a) and crude PCR product (Figure 8b). </P><DIV id="picture8" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 8. Exploring the sensitivity of PC reporter using plasmid and PCR product as the target.</B><B>(a)</B> Bioluminescence intensity at different concentrations of plasmid as the target. <B>(b)</B> Bioluminescence intensity at different concentrations of crude PCR product as the target. Both showed that PC reporter was able to detect target DNA at a low concentration (0.1 nM).</SMALL></P></DIV></DIV></DIV><P>However, we don’t know whether 0.1nM is the bottom limit of detection. We speculated that with multiple PC reporters recognizing more sites on one target, enhanced bioluminescence signal could be observed. In that way we could get a relatively high signal at a lower concentration of target DNA. The results confirmed our speculation (Figure 9).</P><DIV id="picture9" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 9. Bioluminescence intensity with multiple PC reporters.</B> One, three, and five PC reporters were, respectively, subjected to the measurement using PCR product of 0.01nM.</SMALL></P></DIV></DIV></DIV><P>After validating the specificity and sensitivity of PC Reporter system. We were eager to apply it to detect the real MTB genome. Fortunately, we got the absolutely safe MTB genomic DNA from Prof. LIU Cuihua (Chinese Academy of Sciences). </P><P>First, we used the most popular commercial TB NAD toolkit (TB-LAMP) to amplify MTB genome and found out that there was a large yield of non-specific product after amplification (Figure 10a,b), which could be mistakenly considered as a positive result if using the conventional diagnosis method. However, our PC Reporter system can easily distinguish the true from the false (Figure 10c).</P><DIV id="picture10" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 10.</B><B>(a)</B> Gel electrophoresis of amplification product of LAMP using MTB genome or control genome (E.coli). <B>(b)</B> Amount of amplified nucleic acid measured using the method provided in TB-LAMP toolkit. <B>(c)</B> Bioluminescence intensity generated by PC reporter, which is able to distinguish the true PCR product from the false.</SMALL></P></DIV></DIV></DIV><P>Then we set out to test whether our PC reporter could be applied to the clinical cases where patients might be in an early stage of TB thus the copies of MTB in their pathological samples could not be that many. The key factor is to figure out the minimum copy numbers of MTB genome in crude sample that can be detected by PC reporter when combined with PCR. We performed a series of dilution on the MTB genome before PCR amplification and found that, when the PCR products are subjected to the measurement, our PC reporter can easily reach the detection limit of only 1 genome copy per tube, which is sensitive enough for real TB diagnosis (Figure 11).</P><DIV id="picture11" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P id="array" class="text-left"><SMALL><B>Figure 11. The effect of genome copies/mL for NAD amplification on the bioluminescence intensity of PC reporter.</B></SMALL></P></DIV><DIV><P> </P><P> </P></DIV></DIV></DIV><P><BIG><B>From single marker to multi-marker array</B></BIG></P><P>We have demonstrated that our Paired dCas9 (PC) Reporter system works well in bench and could be applied to the detection of MTB genomic DNA. Particularly, when combined with PCR, our PC reporter is as sensitive as to confidently detect only 1 genome copy per tube.</P><P>However, in the clinical practice, the situations are much more complicated considering the fact that strain mutation, sample difference and other environmental factors would cause misdiagnosis for single test. So we got an idea of designing a diagnosis array to extract more sequence information for our PC Reporter to test multiple sites on the target genome at one time, thus  to guarantee reliability of diagnosis.</P><P>The core and also the first step of array design is to enumerate all the dCas9 binding site (PAM site) through the entire genome of MTB and to identify MTB-specific PAM sites as markers via <A href="https://2015.igem.org/Team:Peking/Modeling#Modeling-SSPD" style="color:#00afd1">SSPD</A>. After computational screening, we totally obtained 2791 MTB-specific markers. 72 sgRNA pairs was designed according to 72 markers out of 9 fragments on MTB genome (Figure 12). Then we utilized <A href="https://2015.igem.org/Team:Peking/Modeling#Modeling-OligoGenerator" style="color:#00afd1">Oligo Generator</A> to design oligos for Golden Gate Cloning in a high-throughput manner, built sgRNA Generators for cell-free transcription, and finally obtained 70 sgRNA pairs (2 pairs failed during cloning).</P><DIV id="picture12" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 12. 72 target sites (MTB-specific markers) out of 9 fragments on MTB genome, screened out using SSPD.</B></SMALL></P></DIV></DIV></DIV><P>Then we performed array-based high-throughput measurement using PC Reporter system (See <A href="#Methods" style="color">Methods</A>). Two separated assays were carried out using MTB and a control bacteria (E. coli), respectively. By data analysis, the results were presented as heat maps below (Figure 13). We can easily discriminate the results of MTB from that of control strain. Note that among 70 successfully constructed sgRNAs, 59 exhibit a significant performance for the detection of MTB; 11, however, fail to be positive. This is consistent with our concern that some factors might disrupt the performance of individual single test of PC reporter.  Reassuringly, the array-based PC reporter detection solved this problem (Figure 13).</P><DIV id="picture13" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 13. Results of high-throughput assay for MTB and control strain.</B> F denotes fragments obtained from MTB genome (a) or control strain (b); P denotes markers from each fragment.</SMALL></P></DIV></DIV></DIV><P>To validate the results in a quantitative way, we selected <A href="https://2015.igem.org/Team:Peking/Modeling/Analysis" style="color:#00afd1">Wilcoxon Rank Sum Test of Block Design</A> as a model, in which the sum of Wilcoxon Rank Sum statistics <I>W<SUB>j</SUB> (1≤j≤m)</I> of each block was calculated as</P><P>With the data of our experiment substituted, the statistics <I>W<SUB>BD</SUB></I> was calculated as</P><DIV><P id="Methods"> </P></DIV><P>Therefore MTB and control strain (E.coli) are significantly different in signal. That is, our PC Reporter system is able to distinguish the sequence of MTB genome from that of other non-specific strains.</P></DIV><H3 class="classic-title" style="margin-top:50px"><SPAN>Methods</SPAN></H3><P><BIG><B>Clone construction of split luciferase-dCas9 fusion protein </B></BIG></P><P>We cloned our split luciferase-dCas9 fusion protein in the pET-21a by Golden Gate assembly (Figure 14a,b).</P><DIV id="picture14" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 14. Plasmids of split luciferase-dCas9 fusion protein in pET-21a.</B><B>(a)</B> Plasmid of Nluc-dCas9 fusion architecture with T7 promoter, Cluc-dCas9 coding sequence and T7 terminator. <B>(b)</B> Plasmid of Cluc-dCas9 fusion architecture T7 promoter, Nluc-dCas9 coding sequence and T7 terminator.</SMALL></P></DIV></DIV></DIV><P><BIG><B>Selection of test target sequence</B></BIG></P><P>Before taking the sequence of <I>Mycobacterium tuberculosis</I>(MTB) genome as target of our PC Reporter system, we first performed our experiments within a test target gene, which must not only conform to the substrate requirements of dCas9, but also harmless and safe enough to its bacteria carrier after chemical transformation. After screening over iGEM Registry of Standard Biological Parts, we finally took Part: <A href="http://parts.igem.org/Part:BBa_K909009" style="color:#00afd1">BBa_K909009</A> (cDNA of UV-B sensing protein UVR8 from Arabidopsis thaliana (from iGEM12_ETH_Zurich) as the test target(Figure 15).</P><DIV id="picture15" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 15. Schematic of Part: BBa_K909009.</B></SMALL></P></DIV></DIV></DIV><P><BIG><B>sgRNA generation</B></BIG></P><P>To obtain sgRNA, we used <A href="https://2015.igem.org/Team:Peking/Modeling#Modeling-SSPD" style="color:#00afd1">SSPD</A> to screen out all guide sequences from the entire genome and <A href="https://2015.igem.org/Team:Peking/Modeling#Modeling-OligoGenerator" style="color:#00afd1">Oligo Generator</A> to generate oligos for Golden Gate Cloning, then inserted all guide sequences into sgRNA Generator backbone (Figure 16). Through PCR amplification and cell-free transcription using HiScirbe T7 Quick High Yield RNA Synthesis Kit (New England Biolabs), the sgRNA was produced and purified at last.We specially thank ZHAO Xuejin (Research Assistant, Institute of Microbiology, Chinese Academy of Sciences) for providing us with the sgRNA Generator!</P><DIV id="picture16" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 16. Schematic of sgRNA Generator backbone, containing T7 promotor, lacZa’, crRNA(tracrRNA) and T7 terminator.</B></SMALL></P></DIV></DIV></DIV><P><BIG><B>Testing PC Reporter system</B></BIG></P><P>We invented a new protocol for testing our PC Reporter system (Figure 17).</P><DIV id="picture17" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 17. Flow chart for testing PC Reporter system.</B> Since one dCas9 fusion protein would form a complex with a pair of sgRNAs simultaneously, which will decrease the complementation of Nluc and Cluc, we incubated two split luciferase-dCas9:sgRNA complexes separately at 25°C for 10 min. After adding target DNA into the mixture of two complexes and incubated at 37°C for 30 min, they simultaneously bind to two adjacent half-sites on target DNA, bringing Nluc and Cluc into close proximity, thus leading to recovery of its enzymatic activity. Then transfer the mixture to 96-well plates before adding Luciferase Assay Reagent (Promega) and promoptly measure the luminescence in Microplate Reader (Thermo Scientific).</SMALL></P></DIV></DIV></DIV><P><BIG><B>High-throughput array detection assay</B></BIG></P><P>Based on the results we acquired, single-site target detection presents relatively high specificity and sensitivity. However, taking strain mutations, sample differences and other environmental factors in clinical practices into consideration, we set an array (Figure 18) to detect multiple target sites (markers) of MTB to make our PC reporter quantitatively reliable.</P><DIV id="picture18" style="padding:bottom:10px"><DIV class="row"><DIV class="col-md-12"><P class="text-left"><SMALL><B>Figure 18. The 96-well plate layout in array detection assay.</B> 9 fragments (F1-F9) screened out by <A href="https://2015.igem.org/Team:Peking/Modeling#Modeling-SSPD" style="color:#00afd1">SSPD</A> from MTB genome were set in each line, and 8 MTB markers (M1-M8) in each fragment were respectively set in each sample pool. In total 8*9=72 markers were detected in one single assay. For each fragment, P9 meant a negative control with a mismatch sgRNA pair added. Two separated assays were carried out using MTB and a control strain (E. coli) as experimental subjects respectively.</SMALL></P></DIV></DIV></DIV></DIV><H3 class="classic-title" style="margin-top:50px"><SPAN>References</SPAN></H3><P>1. WHO. Global Tuberculosis Report 2014[M]. World Health Organization, 2014.</P><P>2. Rodolphe Barrangou, Christophe Fremaux, Hélène Deveau, et al. CRISPR provides acquired resistance against viruses. Science, 2007, 315: 1709-1712.</P><P>3. John P Guilinger, David B Thompson, David R Liu. Fusion of catalytically inactive Cas9 to FokI nuclease improves the specificity of genome modification. Nature biotechnology, 2014, 32, 6.</P><P>4. Lei S. Qi, Matthew H. Larson, Luke A. Gilbert et al. Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression. Cell, 2013, 152: 1173-1183.</P><P>5. Sujan S Shekhawat, Indraneel Ghosh. Split-protein systems: beyond binary protein-protein interactions. Current Opinion in Chemical Biology, 2011, 15: 789–797.</P><P>6. Taha Azad, Amin Tashakor, Saman Hosseinkani. Split-luciferase complementary assay: applications, recent developments, and future perspectives. Anal Bioanal Chem, 2014, 406: 5541-5560.</P><P>7. Rossi F, Charlton C, Blau. H. Monitoring protein-protein interactions in intact eukaryotic cells by beta-galactosidase complementation. Proc. Natl. Acad. Sci. 1994, USA 94: 8405–8410.</P><P>8. Remy I, Galarneau A, Michnick S W. Detection and visualization of protein interactions with protein fragment complementation assays. Methods Mol. Biol. 2002, 185: 447–459.</P><P>9. Kathryn E. Luker, Matthew C. P. Smith, et al. Kinetics of regulated protein–protein interactions revealed with firefly luciferase complementation imaging in cells and living animals. PNAS, 2004, 101: 12288-12293.</P><P>10. Ramasamy Paulmurugan and Sanjiv S. Gambhir. Combinatorial Library Screening for Developing an Improved Split-Firefly Luciferase Fragment-Assisted Complementation System for Studying Protein-Protein Interactions. Anal. Chem. 2007, 79: 2346-2353.</P><P>11. Jinek M, Chylinski K, Fonfara I et al. A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity. Science, 2012, 337(6096): 816-821.</P><P>12. Ramasamy Paulmurugan, Sanjiv S. Gambhir. An intramolecular folding sensor for imaging estrogen receptor-ligand interactions. The National Academy of Sciences of the USA, 2006, 103: 15883-15888.</P><P>13. Bruce Alberts et al. Molecular Biology of the Cell. Fifth edition.</P></DIV></DIV></DIV></DIV></DIV><FOOTER style="margin-top:30px; padding:0"><DIV class="container"><DIV class="row footer-widgets" style="padding:0:margin:0">&lt;<DIV class="col-md-3"><DIV class="col-md-12 footer-widget social-widget row" style="padding-top:0:padding-bottom:0;margin-top:0;margin-bottom:0"><H4>Share This</H4></DIV><DIV class="col-md-12 footer-widget social-widget row"><H4>Follow Us</H4></DIV></DIV><DIV class="col-md-6"><DIV class="footer-widget"><H4>Sponsors</H4></DIV></DIV><DIV class="col-md-3"><DIV class="footer-widget" style="padding:0:margin:0"><H4>Contact us</H4><DIV class="footer-widget contact-widget"><UL style="font-size:12px;color:#b4b4b4"><LI><SPAN>Email:</SPAN><A href="Mailto:pekingigem@gmail.com&amp;Subject=Hello&amp;Body=Hi, Peking iGEM!" style="color:#b4b4b4">pekingigem@gmail.com</A></LI><LI><SPAN>Website:</SPAN><A href="2015.igem.org/Team:Peking" style="color:#b4b4b4"> 2015.igem.org/Team:Peking</A></LI><LI><SPAN>Address:</SPAN> Peking University, No.5 Yiheyuan Road Haidian District, Beijing, P.R.China 100871</LI></UL></DIV></DIV></DIV><DIV class="copyright-section"><DIV class="row"><DIV class="col-md-12" style="padding:0;margin:0;height:30px"><P class="text-center">© 2015 Peking iGEM -  All Rights Reserved. Based on <A href="http://getbootstrap.com/" style="color:#b4b4b4">Bootstrap</A> and Margo template.</P></DIV></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>