<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Peking_Design_Speculation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Peking/Design/Speculation</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="container"><HEADER class="clearfix"><DIV class="navbar navbar-default navbar-top"><DIV class="container"><DIV class="navbar-collapse collapse" style="padding-top:5px;padding-bottom:0"><UL class="nav navbar-nav navbar-right " style="padding-bottom:15px;padding-top:10px"><LI><A href="https://2015.igem.org/Team:Peking/JudgingCriteria">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Peking/JudgingCriteria">Judging Criteria</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Collaborations">Collaborations</A></LI></UL><LI><A class="active" href="https://2015.igem.org/Team:Peking/Design">Project</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design/PC_Reporter">Paired<SPAN style="text-transform:lowercase"> d</SPAN>Cas9 Reporter</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design/Isothermal">Iso-<SPAN style="text-transform:lowercase">t</SPAN>hermal Amplification</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Device">Hardware</A></LI><LI><A class="active" href="https://2015.igem.org/Team:Peking/Design/Speculation">Speculation</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Modeling">Array Design</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Modeling/Analysis">Analysis algorithm</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Background">Facts about TB</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Consultation">Consultation and Interview</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Engagement">Public Engagement</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Outreach">Ethics and Economics Issues</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Team">Lab</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Safety">Safety</A></LI></DIV></DIV></DIV></HEADER><DIV class="page-banner"><DIV class="container"><DIV class="row"><DIV class="col-md-6"><H2 style="font-size:20px; margin-bottom:5px; padding-bottom:0">P<SPAN style="text-transform:lowercase">roject</SPAN></H2><P style="margin-top:0px;font-size:14px" id="Extension">It takes half your life before you discover life is a do-it-yourself project.</P></DIV><DIV class="col-md-6"><UL class="breadcrumbs"><LI><A href="https://2015.igem.org/Team:Peking">Home</A></LI><LI>Project</LI></UL></DIV></DIV></DIV></DIV><DIV id="content"><DIV class="container"><DIV class="row blog-page"><DIV class="col-md-3 sidebar left-sidebar"><DIV id="sidebar1" class="widget widget-categories"><H4 style="font-size:18px">S<SPAN style="text-transform:lowercase">peculation</SPAN></H4><UL><LI><A href="#Extension">Detection Extension</A></LI><LI><A href="#MB">Molecular Beacon</A></LI></UL></DIV><DIV id="sidebar2" class="widget widget-categories"><H4 style="font-size:18px">S<SPAN style="text-transform:lowercase">peculation</SPAN></H4><UL><LI><A href="#Extension">Detection Extension</A></LI><LI><A href="#MB">Molecular Beacon</A></LI></UL></DIV></DIV><DIV class="col-md-9 page-content" style="text-align:justify;"><DIV id="speculation"><H3 class="classic-title" id="Extension" style="margin-top:50px;"><SPAN>Detection Extension</SPAN></H3><DIV style="margin-bottom:30px;padding-bottom:30px"><P>Our paired dCas9 (PC) reporter system utilizes protein fragment complementation (PCA), where fragmented reporter protein reassembles to reconstitute the activity. A variety of reporter proteins have been rationally dissected into two pieces, including ubiquitin, dihydrofolate reductase(DHFR), β-Lactamase,  β-glactosidase, and GFP. </P><P>Our PC reporter system can be used for the detection of MTB, also applicable for other wide spread clinical pathogens such as Norovirus, Shigella dysenteriae, Vibrio cholera, Staphylococcus aureus. We could also proceed RNA specimen via reverse transcriptase, which generates DNA through reverse transcription. By introducing a separate PAM-presenting oligonucleotide(PAMer), our PC Reporter system could be used for the detection of single strand RNA virus, such as SARS, HCV, Ebola. Kleinstiver engineered Cas9 to overcome the constrains of conventional PAM(NGG) sequence, the evolved Cas9 can recognize distinct PAM sequence, that is to say, our PC Reporter system might work with all possible PAM sequences. Meanwhile the PC Reporter can also screen useful biomarkers associated with inherited diseases, such as trisomy 21 syndrome, albinism. That’s to say, our PC Reporter system could cover a wide range of disease detection.</P><P>Electrochemical biosensors may be based on biological catalysts or specific binding of ligands to generate electrochemical signals. The reaction typically generates a measurable current or potential; amperometry is based on the application of a constant potential, while in voltammetry, current is measured. In both techniques, the resulting current is proportional to the bulk concentration of electroactive species. For their ease to use, short assay time, and high sensitivity, they have emerged as an attractive method for molecular detection. </P><P>Here we could integrate PCA-based electrochemical reaction into our detection PC reporter system to further power the pathogen detection.  
                (1). β-lactamase catalyzes the hydrolysis of penicillin to penicillinoic acid, causing current changes which can be detected using A3 electrode system. β-Lactamase could be dissected between Gly196 and Leu198; furthermore we could obtain a more active and metabolically stable pattern by introducing a mutation (M182T), which disrupts an inactive molten-globule intermediate of β-Lactamase.
                 </P><P>(2). β-glactosidase catalyzes the hydrolysis of p-aminophenyl-β-D-galactopyranoside (PAPG) to p-aminophenol (PAP), which could be further oxidized near the working electrode held at 220mV versus the reference electrode. We could dissect LacZ as Δα, ∆ω segments to perform PCA.</P><P>(3). Dihydrofolate reductase(DHFR) catalyzes the NADPH-dependent reduction of dihydrofolate (H2folate) or folic acid to tetrahydrofolate (H4folate), which could be measured electrochemically. DHFR consists of an adenine binding domain (fragment [2]) and a discontinuous domain (fragment [1]/[3]). Fragments [1] and [2] contain folate-binding pocket and the NADPH-binding groove while fragment [3] has few contacts with the substrates. Here we choose to cleave ecDHFR at position 88 so as not to disrupt the active site and NADPH cofactor-binding sites.</P><P><BIG>References</BIG> Stains, C. I., Furman, J. L., Porter, J. R., Rajagopal, S., Li, Y., Wyatt, R. T., &amp; Ghosh, I. (2010). A general approach for receptor and antibody-targeted detection of native proteins utilizing split-luciferase reassembly. ACS chemical biology, 5(10), 943-952.
2 Zanardi, C., Baldoli, C., Licandro, E., Terzi, F., &amp; Seeber, R. (2012). Development of a gold-nanostructured surface for amperometric genosensors. Journal of Nanoparticle Research,14(10), 1-12.
3 Gonçalves, L. M., Callera, W. F., Sotomayor, M. D., &amp; Bueno, P. R. (2014). Penicillinase-based amperometric biosensor for penicillin G.Electrochemistry Communications, 38, 131-133.
4 Galarneau, A., Primeau, M., Trudeau, L. E., &amp; Michnick, S. W. (2002). β-Lactamase protein fragment complementation assays as in vivo and in vitro sensors of protein–protein interactions.Nature biotechnology, 20(6), 619-622.
5 Biran, I., Klimentiy, L., Hengge-Aronis, R., Ron, E. Z., &amp; Rishpon, J. (1999). On-line monitoring of gene expression. Microbiology, 145(8), 2129-2133.
6 Mohler, W. A., &amp; Blau, H. M. (1996). Gene expression and cell fusion analyzed by lacZ complementation in mammalian cells. Proceedings of the National Academy of Sciences, 93(22), 12423-12427.
7 Aiso, K., Nozaki, T., Shimoda, M., &amp; Kokue, E. (1999). Assay of dihydrofolate reductase activity by monitoring tetrahydrofolate using high-performance liquid chromatography with electrochemical detection. Analytical biochemistry,272(2), <SPAN id="MB">143-148</SPAN>.
8 Bystroff, C., &amp; Kraut, J. (1991). Crystal structure of unliganded Escherichia coli dihydrofolate reductase. Ligand-induced conformational changes and cooperativity in binding.Biochemistry, 30(8), 2227-2239.
9 Andrea R. (2015) CRISPR Craze Conquers the RNA World: Precise Manipulation of DNA and RNA Based on a Bacterial Defense System. Angewandte, 16(54), 4710-4712.
10 Benjamin, P.K., Michelle, S. P., Shengdar, Q. T. (2015). Engineered CRISPR-Cas9 nucleases with altered PAM specificities Nature, 523, 481-485.
</P></DIV><H3 class="classic-title" id="MB" style="margin-top:50px;"><SPAN>Molecular Beacon</SPAN></H3><DIV class="col-md-12" style="margin-bottom:30px;padding-bottom:30px"><P>Molecular beacon is hairpin-structured single-stranded oligonucleotide that can report the presence of specific nucleic acids through hybridization. It has been widely used for nucleic acid detection with high sensitivity and selectivity [1].This single-stranded DNA molecule consists of a stem-and-loop structure doubly labeled with a fluorophore and a quencher group on each end. In the “off” state, little fluorescence background is noted by the effect of quenching. However, upon binding with their targets, conformational changes open the hairpin structure; thereby fluorescence is turned “on”.</P><DIV id="fig1"><DIV class="col-md-12"><P class="text-center"><SMALL><B>Figure 1.</B> The working mechanism of molecular beacon. Q, quencher; F, fluorophore. </SMALL></P></DIV></DIV><P>To achieve higher sensitivity and specificity, a dual-MB system was designed in the beginning, which involves two fluorescence groups and two quenching groups [2]. These two fluorophores form a fluorescence resonance energy transfer (FRET) pair. When exciting the donor dye, the energy will be transferred to the acceptor dye, thus to elicit corresponding fluorescence signal. </P><DIV id="fig2"><DIV class="col-md-12"><P><SMALL><B>Figure 2.</B> The concept of dual-FRET molecular beacons. Hybridization of donor and acceptor molecular beacons to adjacent sites on the same single-strand DNA or mRNA target results in FRET between donor and acceptor fluorophores upon donor excitation. By detecting FRET signal, the concentration of target would be estimated.</SMALL></P></DIV></DIV><P>To further amplify the hybridization signal, the enzymatic reaction was introduced. We modified the dual-MB system by removing the quencher groups and replacing the FRET pair with the split firefly luciferase. [3] When the new dual-MB system hybridize to the target at the same time, the two parts of split luciferase will get closed to each other, thus to catalyze the luciferin oxidation to produce luminescent signal that could be detected by microplate reader. </P><P>One problem concerning the new dual-MB design is the conjugation of protein to DNA. The biotin-streptavidin interaction, one of the strongest noncovalent interactions in nature, was selected to synthesize the DNA-protein conjugates [4]. We fused the two parts of split luciferase, respectively, with the homotetrameric protein streptavidin, thus to obtain two fusion proteins, STV-FLucN416 and FlucC398-STV. </P><P>However, during protein expression and purification we found that most of the fusion proteins are insoluble in the cell. The inclusion bodies were isolated and then dissolved in 6M granidine hydrochloride, following by refolding via dilution into pH 8.0 Tris buffer. [5] Then the purified proteins were incubated, respectively, with one of the biotinylated MB, to synthesize the MB.</P><DIV id="fig3"><DIV class="col-md-12"><P><SMALL><B>Figure 3.</B> Using split luciferase to replace the fluorophores in dual-MB system. The two parts of split luciferase interact with each other after donor and acceptor MBs hybridize to adjacent sites on the same single-strand DNA or mRNA target. By detecting the luminescence signal produced by luciferin oxidation, the concentration of target would be estimated thereby.</SMALL></P></DIV></DIV><P>We verified the newly synthesized MBs by applying them to targeting a commonly mutated region of the K-ras gene involved in many cancers (the DNA was prepared by PCR amplification). To find out an appropriate test condition with a high signal-to-background ratio, we first determined the appropriate concentration of each protein and MB; then the signals generated at different concentrations of DNA or RNA target were also examined.</P><P>We conducted the experiment at different protein concentrations, with or without DNA or RNA target. As shown in Figure 4, significant signals could be detected only at a high protein concentration (2 μM). According to the previous studies, after the denaturation-refolding process the recovered activity of firefly luciferase ranges from 20%~60%, while the recovered activity of streptavidin could be only 5%. Therefore, the fraction of functionally complete proteins in the entire system is low while that of the functionally “false positive” protein is high; this results in a relatively high background and a low signal-to-background ratio.</P><DIV id="fig4"><DIV class="col-md-12"><P><SMALL><B>Figure 4.</B> Luminescence signals produced by different concentrations of each fusion protein with or without DNA or RNA target. When concentration of any fusion protein was lower than 1000 nM, there was no significant difference between the experiment group and the control group. Notable signals of experiment groups were detected when each protein concentration is 2000 nM. Signal-to-background ratio of 3.23 as the result of DNA target, 2.28 as the result of RNA target were obtained. DNA or RNA target concentration, 100 nM; each MB concentration, 400 nM.</SMALL></P></DIV></DIV><P>Higher concentration of MB would increase the binding with target nucleic acid. However, we found that, when the protein concentrations are fixed, higher MB concentration would result in the blocking of the target nucleic acid by protein-free MB; the signal would decrease instead. Fig.5 shows that 300 nM MB gave the best result when detecting DNA target. </P><DIV id="fig5"><DIV class="col-md-12"><P><SMALL><B>Figure 5.</B> Detection of the luminescence signals at different concentrations of each MB. (A) Each MB concentration varies from 400 nM to 1000 nM. As MB concentration gets higher, the signal of DNA target gets lower; however, the signal of RNA target gets higher, which indicates that higher concentration of MB is appreciated when detecting RNA target. The signal-to-background ratios are presented in (B). (C) Each MB concentration varied from 100 nM to 400 nM. 300 nM of each MB generated the most significant signal when detecting DNA target, which gave a ratio of 3.69. (D) The ratio of each group. Nucleic acid concentration was 100 nM for DNA or RNA.</SMALL></P></DIV></DIV><P>After determining the appropriate protein and MB concentration, the luminescence signals generated using different target concentrations were examined (Fig.6). We found that the signal increases as the target concentration improves. Even when there was only 10 nM target, significant signal could still be observed in our system with a ratio of 4.42 for DNA target, 3.12 for RNA target. </P><DIV id="fig6"><DIV class="col-md-12"><P><SMALL><B>Figure 6.</B> Detection of the signals using different target concentrations. The highest signals were detected when target concentration is 200 nM at each cases; the ratio is 7.07 for DNA target, 4.52 for RNA target. Even when the target concentration is only 10 nM, there was still significant difference compared to the control group; the ratio is 4.42 for DNA target, 3.12 for RNA target.</SMALL></P></DIV></DIV><P>It was reported that the gap size between the donor and acceptor fluorophores can significantly influence the FRET efficiency. In our system, the gap size would restrict the interaction of N-terminal and C-terminal luciferase. In our data shown above, the gap size is 4. We then created target sequences with a gap size ranging from 0 to 15 nucleotides and investigated their effect on the enzymatic reactions. Results showed that, for our system, a 4-nucleotide spacer is the most effective in maximizing the fluorescent signal.</P><DIV id="fig7"><DIV class="col-md-12"><P><SMALL><B>Figure 7.</B> The effect of spacer length on the luminescence signal. As the spacer length gets longer, the signal goes through an up-and-down process, which precisely matches our expectation. Spacer length of 4bp gave the highest signal.</SMALL></P></DIV></DIV><P>In summary, we have designed a dual-MB system using split luciferase to achieve the signal amplification. However, due to the limited activity of the protein after purification and the limitation of the working mechanism, our system didn’t exhibit outstanding sensitivity and signal-to-background ratio. </P><P><BIG>References</BIG></P><P>[1] Tan WH, Wang KM, Drake TJ (2004) Molecular beacons. Curr Opin Chem Biol 8(5):547–553
[2] Santangelo P J, Nix B, Tsourkas A, et al. Dual FRET molecular beacons for mRNA detection in living cells[J]. Nucleic acids research, 2004, 32(6): e57-e57.
[3] Paulmurugan R, Gambhir S S. Combinatorial library screening for developing an improved split-firefly luciferase fragment-assisted complementation system for studying protein-protein interactions[J]. Analytical chemistry, 2007, 79(6): 2346-2353.
[4] Niemeyer C M. Semisynthetic DNA–protein conjugates for biosensing and nanofabrication[J]. Angewandte Chemie International Edition, 2010, 49(7): 1200-1216.
[5] Sano T, Cantor C R. Expression of a cloned streptavidin gene in Escherichia coli[J]. Proceedings of the National Academy of Sciences, 1990, 87(1): 142-146.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><FOOTER style="margin-top:30px; padding:0"><DIV class="container"><DIV class="row footer-widgets" style="padding:0:margin:0"><DIV class="col-md-3"><DIV class="col-md-12 footer-widget social-widget row" style="padding-top:0:padding-bottom:0;margin-top:0;margin-bottom:0"><H4>Share This</H4></DIV><DIV class="col-md-12 footer-widget social-widget row"><H4>Follow Us</H4></DIV></DIV><DIV class="col-md-6"><DIV class="footer-widget"><H4>Sponsors</H4></DIV></DIV><DIV class="col-md-3"><DIV class="footer-widget" style="padding:0:margin:0"><H4>Contact us</H4><DIV class="footer-widget contact-widget"><UL style="font-size:12px;color:#b4b4b4"><LI><SPAN>Email:</SPAN><A href="Mailto:pekingigem@gmail.com&amp;Subject=Hello&amp;Body=Hi, Peking iGEM!" style="color:#b4b4b4">pekingigem@gmail.com</A></LI><LI><SPAN>Website:</SPAN><A href="2015.igem.org/Team:Peking" style="color:#b4b4b4"> 2015.igem.org/Team:Peking</A></LI><LI><SPAN>Address:</SPAN> Peking University, No.5 Yiheyuan Road Haidian District, Beijing, P.R.China 100871</LI></UL></DIV></DIV></DIV><DIV class="copyright-section"><DIV class="row"><DIV class="col-md-12" style="padding:0;margin:0;height:30px"><P class="text-center">© 2015 Peking iGEM -  All Rights Reserved. Based on <A href="http://getbootstrap.com/" style="color:#b4b4b4">Bootstrap</A> and Margo template.</P></DIV></DIV></DIV></DIV></DIV></FOOTER></DIV></BODY></HTML>