<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Peking_Modeling_test skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Peking/Modeling/test</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="container"><HEADER class="clearfix" id="fixedNav"><DIV class="navbar navbar-default navbar-top"><DIV class="container"><DIV class="navbar-collapse collapse" style="padding-top:5px;padding-bottom:0"><UL class="nav navbar-nav navbar-right " style="padding-bottom:15px;height:88px;padding-top:10px"><LI><A href="https://2015.igem.org/Team:Peking/JudgingCriteria">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Peking/JudgingCriteria">Judging Criteria</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Collaborations">Collaborations</A></LI></UL><LI><A href="https://2015.igem.org/Team:Peking/Design">Project</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design">Overview</A></LI><LI><A href="http://2015.igem.rg/Team:Peking/Design/Sequence-specific visualization">Sequence-specific Visualization</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design/Isothermal">Iso-thermal Amplification</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design/Array">Array-based diagnosis</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Device">Hardware</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design/Speculation">Speculation</A></LI><LI><A class="active" href="https://2015.igem.org/Team:Peking/Modeling">Modeling</A></LI><LI><A class="active" href="https://2015.igem.org/Team:Peking/Modeling">Array Design</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Modeling/Analysis">Analysis algorithm</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Background">Facts about TB</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Consultation">Consultation and Interview</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Engagement">Public Engagement</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Outreach">Ethics and Economics Issue</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Team">Lab</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Safety">Safety</A></LI></DIV></DIV></DIV></HEADER><DIV class="page-banner" style="background-color:#e0e0e0;margin-top:1%;margin-bottom:20px;"><DIV class="container"><DIV class="row"><DIV class="col-md-6"><H2><B>Modeling</B></H2><P>Specificity!!!!</P></DIV><DIV class="col-md-6"><UL class="breadcrumbs"><LI><A href="https://2015.igem.org/Team:Peking">Home</A></LI><LI>Modeling</LI></UL></DIV></DIV></DIV></DIV><DIV id="content"><DIV class="container"><DIV class="row blog-page"><DIV class="col-md-3 sidebar left-sidebar"><DIV id="sidebar1" class="widget widget-categories"><H4 style="font-size:18px">Marker Finder</H4><UL><LI><A href="#Modeling-Overview">Overview</A></LI><LI><A href="#Modeling-SSPD">SSPD Method</A></LI><LI><A href="#Modeling-OligoGenerator">Oligo Generator</A></LI></UL></DIV><DIV id="sidebar2" class="widget widget-categories"><H4 style="font-size:18px">Team </H4><UL><LI><A href="#Modeling-Overview">Overview</A></LI><LI><A href="#Modeling-SSPD">SSPD Method</A></LI><LI><A href="#Modeling-OligoGenerator">Oligo Generator</A></LI></UL></DIV></DIV><DIV class="col-md-9 page-content"><DIV id="Modeling-Marker Finder"><DIV id="Modeling-Overview" class="col-md-12" style="padding:0"><H3 class="classic-title"><SPAN>Overview</SPAN></H3><P>To increase the accuracy and specificity of the detection, we developed an assay over our Paired dCas9 Reporter (PC Reporter) System to get more sequence information from the target genome. The core as well as the first step of the design of the array is to screen over the entire genome and get paired specific sequences (CRISPR target sites) with high specificity as markers. 
                                    We develop a method named SSPD to achieve our aim, which is composed of 4 steps: <UL class="disc"><LI>Search for guide sequences of gRNA candidates</LI><LI>Specificity test for each candidate</LI><LI>Pair left and right target sites with optimal spacer length</LI><LI>Design PCR fragments</LI></UL>
                                    We will introduce each step in detail separately with analysis about Mycobacterium tuberculosis (MTB) genome as an example. After the target sites are chosen, we developed an Oligo Generator to turn the target sites into oligonucleotides sequences for following sgRNA construction combined with our gRNA generator (Part).
                                </P></DIV><DIV id="Modeling-SSPD" class="col-md-12" style="padding:0"><H3 class="classic-title"><SPAN>SSPD Methods</SPAN></H3><DIV id="SSPD-S1" class="col-md-12" style="margin:0; padding:0"><H4><EM>Search for guide sequences of gRNA candidates</EM></H4><P>Recall the structure of Paired dCas9 Reporter (PC Reporter) System, a protospacer adjacent motif (PAM) sequence in the form of 5’-NGG-3’ at 3’ end of guide sequence, usually 20bp, on the non-complementary strand. (Figure 1) As it was showed in our experimental results that PAM-out orientation (5’-CCNN20-…-N20NGG-3’) was highly efficient for PC Reporter system to work, thus our model would focus on this orientation. (However, it can be more convenient to adjust our program for guide sequence design also with other orientations. See more in <A href=""><B>Supplementary 1 </B></A>)
                                    </P><DIV id="Modeling_Fig1"><P>Figure 1. Schematic illustration of guide design in PAM-out orientation. Note the 20nt guide sequence is identical to target non-complementary strand.</P></DIV><P>We took advantage of Python 3.4.3 build-in regular expression to search for left guide sequences of gRNA (‘(?&lt;=cc).(?=.{20})’) and right guide sequences of gRNA (‘(?&lt;=.{20}).(?=gg)’) separately, which would be paired later for PC reporter system to function.
                                    </P><DIV id="Modeling_Table1"><TABLE border="1" style="margin:10px;padding:10px" class="col-md-6"><TBODY><TR><TD>Left gRNA candidates (CCNN20)</TD><TD>414962</TD></TR><TR><TD>Right gRNA candidates (CCNN20)</TD><TD>407371</TD></TR></TBODY></TABLE><P class="col-md-12">Table 1. The number of gRNA candidates in Mycobacterium Tuberculosis genome</P></DIV></DIV><DIV id="SSPD-S2" class="col-md-12" style="margin:0; padding:0"><H4><EM>Specificity test for each candidate</EM></H4><P>Specificity of guide sequence of gRNA here is defined as the probability of the gRNA binds to the corresponding target site instead of other similar non-target sites. It is measured by taking both quantity of potential off-target sites and similarity between off-target sites and the unique target site into consideration. Since sputum sample is commonly used in MTB detection, we compared our guide sequence candidates with Human Oral Meta-Genome (HOMG), and reserved the specific sequences orthogonal to oral meta-genome to avoid false positive signals in MTB detection. Here we adopted a BLAST-based 2-step filter approach to realize it. In general, the two steps are: a) Filter out guides with off-targets that have 12 bp PAM-proximal sequence identical to corresponding target; b) Score the reserved gRNA on specificity. The principle of the score-rule is that higher specificity should get higher score (see the detail below). Thus we can easily filter out guides with high off-target probability, which is indicated by a low score.
                                    </P><DIV id="SSPD-filter1" class="col-md-12" style="margin:0; padding:0"><H5><EM>a) PAM-proximal 12bp filtration by BLAST</EM></H5><P>Previous research has demonstrated that SpCas9 tolerates mismatches to a greater extent in the PAM-distal region than the PAM-proximal region, and the PAM-proximal 8-12nt of the target largely determine the specificity of the targets. Guide sequences of gRNA whose target has identical PAM-proximal 12nt off-targets is highly possible to hit off target, which suggest its fate of being eliminated. Remaining guides are uploaded to BLAST again for potential off-target site finding, target similar sequences found by BLAST are recorded as all off-target sites for the second filtration, those ignored are considered not contributing to off-target. 
                                    </P><DIV id="Modeling_Table2"><TABLE border="1" style="margin:10px;padding:10px" class="col-md-6"><TBODY><TR><TD>Left gRNA candidates (CCNN20)</TD><TD>55156</TD></TR><TR><TD>Right gRNA candidates (CCNN20)</TD><TD>54938</TD></TR></TBODY></TABLE><P class="col-md-12">Table 2. The number of gRNA candidates left after the first filtration in Mycobacterium Tuberculosis genome</P></DIV></DIV><DIV id="SSPD-filter2" class="col-md-12" style="margin:0; padding:0"><H5><EM>b) Score filtration </EM></H5><P>Our initial filtration significantly reduce computation by limiting guide sequence score operation to only most possible off-target sites rather than entire subject. We then use Zhang Lab<SUP>[4][5]</SUP> score methods to give an off-target effects measurement of the guide sequences. For each guide candidates, scoring process includes two steps: firstly，each of the individual off-target sites is assigned an “individual score”, then all individual scores are aggregated to an overall score of the given guide sequence.The algorithm used to score single off-target sites is as below.<SUP>[4]</SUP></P><P>
                                        In the first term, <I>e</I> runs over the mismatch positions set <I>M</I> between the guide and the off-target site, with <I>W</I> representing the experimentally-determined effect of mismatch position on targeting.<SUP>[5]</SUP> Term two refers to the effect caused by mean pairwise distance between mismatches(d). Term three represent a dampening penalty for highly mismatched off-targets, where n<SUB>m</SUB> refers to the number of mismatches. 
                                        </P><DIV id="Modeling_Fig2"><P>Figure 2. Schematic illustration of an off-target site. The 20 bps are numbered sequentially from the one most distant to PAM, and the red 7, 12, 19 represents the mismatches in the off-target sites.</P></DIV><P>
                                        Here we show an example (Figure 2) to help readers understand this formula better. Nucleotides are numbered sequentially from the one most distant to PAM, and mismatches in the off-target sites are emphasized by red sign. In this case, W(7)= 0.317, W(12)=0.508, W(19)=0.583; d<SUB>mean</SUB>=(d<SUB>(7,12)</SUB>+d<SUB>(7,19)</SUB>+d<SUB>(12,19)</SUB>)/3=7; n<SUB>m</SUB>=3. Therefore, the off-target site shown above has an individual score of 0.004415278985074628. A higher individual score for an off target site indicates a higher similarity to the target, and thus a higher likelihood of the CRISPR/Cas9 complex binding to the off-target site. Once individual off-target have been scored, each guide is assigned an overall score:
                                        </P><DIV><P><I>h</I> runs over the potential off-target set <I>T</I> of guide sequence </P></DIV><P>A higher overall score indicates a better guide with few or weak potential off-targets. Guides with individual off-target sites score 2% or above are eliminated, the remaining are ranked in the order of overall score from 100% to 0%, with a list of off-targets presented in individual score descending order. Guides having an overall score of greater than 45% are reserved finally while others were filtered out in case their lack of specificity.
                                        </P><DIV id="Modeling_Table3"><TABLE border="1" style="margin:10px;padding:10px" class="col-md-6"><TBODY><TR><TD>Left gRNA candidates (CCNN20)</TD><TD>54417</TD></TR><TR><TD>Right gRNA candidates (CCNN20)</TD><TD>54288</TD></TR></TBODY></TABLE><P class="col-md-12">Table 2. The number of gRNA candidates left after the first filtration in Mycobacterium Tuberculosis genome</P></DIV><DIV id="Modeling_Fig3"><P class="col-md-12">Figure 3. Analysis of the guide sequences scores after score filtration. (a)Overall score distribution of guide sequences after the second step filtration. Most sequences have scores more than 97. (b)Ratio of reserved guides (i.e. the number of guide sequences reserved after score filtration / the number of guide sequences after first filtration) is approximately 1 on higher score gRNA and 0 on lower score gRNA.ure 1. Schematic illustration of guide design in PAM-out orientation. Note the 20nt guide sequence is identical to target non-complementary strand.</P></DIV></DIV></DIV><DIV id="SSPD-P" class="col-md-12" style="margin:0; padding:0"><H4><EM>Pair left and right target sites with optimal spacer length</EM></H4><P>All reserved left and right target sites after two-step filtration are considered to be qualified for pairing. In this step, a left site and a right site with appropriate spacer length will be paired. The best spacer length is 19-23bp for split-luciferase dCas9 fusion system according to our experimental data (Figure 4, Link to CRISPR). Single sites that cannot pair with any other sites within the given range of the spacer length would be eliminated.
                                    </P><DIV id="Modeling_Fig4"><P>Figure 4. The effect of spacer length variation on the performance of PC reporter system. Orientation of sgRNA binding-site pairs used in the test is &quot;PAM-out&quot; and the spacer sequence length varies from 5bp to 107bp.</P><DIV id="Modeling_Table4"><TABLE border="1" style="margin:10px;padding:10px" class="col-md-6"><TBODY><TR><TD>Spacer Length</TD><TD>19</TD><TD>20</TD><TD>21</TD><TD>22</TD><TD>23</TD><TD>19-23</TD></TR><TR><TD>Marker Number</TD><TD>690</TD><TD>836</TD><TD>746</TD><TD>720</TD><TD>759</TD><TD>3751</TD></TR></TBODY></TABLE><P class="col-md-12">The number of gRNA pairs under the condition of different spacer length between left and right pair from 19 to 23 bps in Mycobacterium Tuberculosis genome</P></DIV></DIV></DIV><DIV id="SSPD-D" class="col-md-12" style="margin:0; padding:0"><H4><EM>Design PCR fragments</EM></H4><P>We provide two methods for determining PCR fragments. For the first, fix pair number k per fragment, search the adjacent but non-overlapped k pairs. Sort the results with fragment length. For another, fix maximal PCR fragment length, sort the results with pair numbers per fragment. Sorted results will be presented on user interface, enabling users to select fragments by themselves as needed. Selecting overlapping fragment is not allowable for array design.
                                    </P><DIV id="Modeling_Fig5"><P>Figure 5. Schematic illustration of PCR fragment determination method, taking 2 pairs per fragment as an example. The top xxx shows all left and right targets on given segment of pathogen genome, and the chart aside lists all pairs. Only adjacent but non-overlapped pairs can be deposited on one fragment. Users can choose one through the four optional PCR fragment listed below, since they are overlapped.</P></DIV><DIV id="Modeling_Fig6"><P>Figure 6. The result of 72 sequence markers chosen in MTB genome. They are divided into 9 fragments for PCR as shown above</P></DIV></DIV></DIV><DIV id="Modeling-OligoG" class="col-md-12" style="padding:0"><H3 class="classic-title"><SPAN>Oligo Generator</SPAN></H3><P>Using SSPD method mentioned above, we can easily find the reliable target sites on genome. However, designing multiple target sites into oligonucleotides sequences for following sgRNA construction manually can be laborious. Thus here we developed a supplementary program to facilitate oligo sequence generation, which is combined with our sgRNA generator (Link to Part xxx). Specifically, we used Golden Gate Cloning to make it more convenient to substitute guide sequences for different target sites. Detailed operation is explained on flow chart below. See more details in <A href=""><B>Supplementary 2</B></A>.
                                </P><DIV id="Modeling_Fig7"><P>Figure 7. Schematic illustration of a flow chart explaining the protocol of guide sequence substitution.</P></DIV></DIV><DIV id="Modeling-Ref" class="col-md-12" style="padding:0"><H3 class="classic-title"><SPAN>Reference</SPAN></H3></DIV></DIV></DIV></DIV></DIV></DIV><FOOTER style="margin-top:30px;padding-top:25px"><DIV class="container"><DIV class="row footer-widgets"><DIV class="col-md-3" style="margin-bottom:0;padding-bottom:0"><DIV class="col-md-12 footer-widget social-widget row"><H4>Share This</H4></DIV><DIV class="col-md-12 footer-widget social-widget row"><H4>Follow Us</H4></DIV></DIV><DIV class="col-md-6" style="margin-bottom:0;padding-bottom:0"><DIV class="footer-widget"><H4>Sponsors</H4></DIV></DIV><DIV class="col-md-3" style="margin-bottom:0;padding-bottom:0"><DIV class="footer-widget"><H4>Contact us</H4><DIV class="footer-widget contact-widget" style="font-size:12px"><UL><LI><SPAN>Email:</SPAN> pekingigem@gmail.com</LI><LI><SPAN>Website:</SPAN><A href="2015.igem.org/Team:Peking" style="color:#b4b4b4">2015.igem.org/Team:Peking</A></LI><LI><SPAN>Address:</SPAN> Peking University, No.5 Yiheyuan Road Haidian District, Beijing, P.R.China 100871</LI></UL></DIV></DIV></DIV></DIV><DIV class="copyright-section" style="margin-top:0;margin-bottom:0"><DIV class="row"><DIV class="col-md-6"><P>© 2015 Peking iGEM - All Rights Reserved.  Based on <A href="http://getbootstrap.com/">Bootstrap</A> and Margo template.</P></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>