<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Peking_Design_Isothermal skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Peking/Design/Isothermal</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="container"><HEADER class="clearfix"><DIV class="navbar navbar-default navbar-top"><DIV class="container"><DIV class="navbar-collapse collapse" style="padding-top:5px;padding-bottom:0"><UL class="nav navbar-nav navbar-right " style="padding-bottom:15px;padding-top:10px"><LI><A href="https://2015.igem.org/Team:Peking/JudgingCriteria">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Peking/JudgingCriteria">Judging Criteria</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Collaborations">Collaborations</A></LI></UL><LI><A class="active" href="https://2015.igem.org/Team:Peking/Design">Project</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design/PC_Reporter">Paired <SPAN style="text-transform:lowercase">d</SPAN>Cas9 Reporter</A></LI><LI><A class="active" href="https://2015.igem.org/Team:Peking/Design/Isothermal">Iso-<SPAN style="text-transform:lowercase">t</SPAN>hermal Amplification</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Device">Hardware</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Design/Speculation">Speculation</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Modeling">Array Design</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Modeling/Analysis">Analysis algorithm</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Background">Facts about TB</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Consultation">Consultation and Interview</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Engagement">Public Engagement</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Practices/Outreach">Ethics and Economics Issue</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Team">Lab</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Peking/Safety">Safety</A></LI></DIV></DIV></DIV></HEADER><DIV class="page-banner"><DIV class="container"><DIV class="row"><DIV class="col-md-6"><H2 style="font-size:20px; margin-bottom:5px; padding-bottom:0">P<SPAN style="text-transform:lowercase">roject</SPAN></H2><P style="margin-top:0px;font-size:14px">It takes half your life before you discover life is a do-it-yourself project.</P></DIV><DIV class="col-md-6"><UL class="breadcrumbs"><LI><A href="https://2015.igem.org/Team:Peking">Home</A></LI><LI>Practices</LI></UL></DIV></DIV></DIV></DIV><DIV id="content"><DIV class="container"><DIV class="row blog-page"><DIV class="col-md-3 sidebar left-sidebar"><DIV id="sidebar1" class="widget widget-categories"><H4 style="font-size:18px">I<SPAN style="text-transform:lowercase">so-thermal</SPAN> A<SPAN style="text-transform:lowercase">mplification</SPAN></H4><UL><LI><A href="#1">Multiple Displacement Amplification (MDA)</A></LI><LI><A href="#2">Loop-mediated Isothermal Amplification(LAMP)</A></LI></UL></DIV><DIV id="sidebar2" class="widget widget-categories"><H4 style="font-size:18px">I<SPAN style="text-transform:lowercase">so-thermal</SPAN> A<SPAN style="text-transform:lowercase">mplification</SPAN></H4><UL><LI><A href="#1">Multiple Displacement Amplification (MDA)</A></LI><LI><A href="#2">Loop-mediated Isothermal Amplification(LAMP)</A></LI></UL></DIV></DIV><DIV class="col-md-9 page-content" style="text-align:justify;"><DIV id="project"><DIV id="isothermal"><DIV style="margin-bottom:30px;padding-bottom:30px"><P>Polymerase chain reaction(PCR) was born in the year of 1985, which amplifies DNA through denaturation, annealing and extension, simulating <I>in vivo</I> DNA replication progress. PCR has radically altered molecular biology; via this technique, a trace amount of DNA can be amplified by tens of thousands of times, generating adequate products for analysis.</P><P>Several variations of PCR have been invented: (1) Nested PCR incorporates two pairs of primers flanking the same target sequence, which increases the sensitivity, i.e., the limit of detection (LOD); (2) RT-PCR monitors the fluorescence during the amplification, predicting the initial template’s abundance; (3) Multiplexed PCR amplifies multiple DNA fragments simultaneously. All these methods need a thermal cycler; Nested PCR and Multiplexed PCR need more than two pairs of primers, which may lead to the generation of primer-dimers; RT-PCT needs expensive CCD to capture signals in real time.</P><P>In order to overcome the disadvantages of the conventional PCR methods, effective, sensitive, and instrument-free amplification methods have been developed in the past decades: such as nucleic acid sequence-based amplification (NASBA), transcription mediated amplification (TMA), signal mediated amplification of RNA technology (SMART), strand displacement amplification (SDA), rolling circle amplification (RCA), loop-mediated isothermal amplification (LAMP), multiple displacement amplification (MDA), and helicase dependent amplification (HDA). All of them amplify nucleic acid sequence rapidly. To further ameliorate our detecting system, we could bring efficient isothermal PCR in the project. </P></DIV><H3 class="classic-title" id="1"><SPAN>Multiple Displacement Amplification (MDA)</SPAN></H3><DIV style="padding-bottom:20px"><P>MDA uses phi29 DNA polymerase and random primers to amplify template DNA isothermally at 30℃, which produces 10,000 fold amplification after overnight incubation. This reaction utilizes the strand displacement activity and great processivity of phi29 DNA polymerase, the former makes it possible to amplify DNA without denaturing, the latter allows it to synthesize DNA strands as long as 70,000bp in length without dissociation (Figure 1). </P><P>MDA generates DNA products robustly in an average length of over 10Kb even up to 100Kb (Figure 2); this robustness to replicate through a complex sequence and a wide coverage makes it possible to evenly amplify the target, regardless of DNA's secondary structure. In the end of amplification, there is about 20–30 ug products from as few as 1–10 copies of human genomic DNA. In previous studies it was suggested that pyrophosphatase be added to the reaction to eliminate the inhibitory accumulation of pyrophosphate for a higher yield. For phi29 DNA polymerase has 3’ to 5’ exonuclease activity, exonuclease-resistant primers should be exploited during the amplification, such as thiophosphate-modified random hexamer (5’-NpNpNpNpsNpsN-3’); this further augments the amount of DNA products.</P><P>MDA eliminates elaborate template preparation procedure; amplification could be carried out directly from crude cells, blood lysates, marrow aspirates, and buccal swabs, even stored DNA such as paraffin-embedded tissues. This also eliminates manual errors and is beneficial for district where there lacks technical guidance. The incubation time is from 2h to 16h, qualifying for rapid detection of pathogen. As we have shown, the required concentration of DNA samples for our PC Reporter is relatively low, also we could further reduce the incubation time. </P><DIV id="fig1" style="margin-bottom:20px"><DIV class="col-md-12"><P><SMALL><B>Figure 1.</B> In MDA system, when random primers anneal to the template DNA, phi29 DNA polymerase extends the strand, displaces downstream strands via its strand displacement activity and produces branched structures through its great processivity. This eventually produces hyperbranched products, which can be 10Kb, even exceeds 100Kb in length. </SMALL></P></DIV></DIV><DIV id="fig2"><DIV class="col-md-12"><P><SMALL><B>Figure 2.</B>MDA to amplify two different targets. A significant fraction of DNA products is too large to leave the sample well during electrophoresis. M for Marker, lane 1 for product of pSB1C3, lane 2 for pSB1C3, lane 3 for product of pET21a, lane 4 for pET21a, lane 5 for H<SUB>2</SUB>O negative control.</SMALL></P></DIV></DIV><P><BIG>References</BIG></P><P>
1. Dean, F. B., Hosono, S., Fang, L., Wu, X., Faruqi, A. F., Bray-Ward, P., ... &amp; Lasken, R. S. (2002). Comprehensive human genome amplification using multiple displacement amplification.Proceedings of the National Academy of Sciences,99(8), 5261-5266.
2. Esteban, J. A., Salas, M., &amp; Blanco, L. (1993). Fidelity of phi 29 DNA polymerase. Comparison between protein-primed initiation and DNA polymerization. Journal of Biological Chemistry,268(4), 2719-2726.
  3. Dean, F. B., Nelson, J. R., Giesler, T. L., &amp; Lasken, R. S. (2001). Rapid amplification of plasmid and phage DNA using phi29 DNA polymerase and multiply-primed rolling circle amplification. Genome research, 11(6), 1095-1099.4. Luthra, R., &amp; Medeiros, L. J. (2004). Isothermal multiple displacement amplification: a highly reliable approach for generating unlimited high molecular weight genomic DNA from clinical specimens. The Journal of Molecular Diagnostics,6(3), 236-242.5. Lasken, R. S., &amp; Egholm, M. (2003). Whole genome amplification: abundant supplies of DNA from precious samples or clinical specimens.Trends in biotechnology, 21(12), 531-535.6. Aviel-Ronen, S., Zhu, C. Q., Coe, B. P., Liu, N., Watson, S. K., Lam, W. L., &amp; Tsao, M. S. (2006). Large fragment Bst DNA polymerase for whole genome amplification of DNA from formalin-fixed paraffin-embedded tissues. BMC genomics, 7(1), 312.7. Spits, C., Le Caignec, C., De Rycke, M., Van Haute, L., Van Steirteghem, A., Liebaers, I., &amp; Sermon, K. (2006). Optimization and evaluation of single‐cell whole‐genome multiple displacement amplification. Human mutation, 27(5), 496-503.</P></DIV><H3 class="classic-title" id="2" style="margin-top:30px"><SPAN>Loop-mediated Isothermal Amplification (LAMP)</SPAN></H3><DIV style="padding-bottom:20px"><P> LAMP was first invented by Tsugunori in 2000, which uses Bst DNA polymerase and a set of primers targeting distinct sites on the template to amplify target DNA. It is the most popular and commercially available detection kit for the diagnosis of tuberculosis. Here is its working mechanism: in the beginning, the inner primer hybridizes to the template to initiate the stretch; then the outer primer binds to its complementary target site and displaces the synthesized single strand DNA, yielding the primary stem-loop DNA. This stem-loop DNA serves as new template for inner primers, which bind to the loop region of the products and continuously displace newly formed DNA strands. The final products appear to be inverted repeats and cauliflower-like structures. LAMP generates extremely large amounts of DNA, about 10^9 copies of target DNA within an hour. Later Eiken et al. introduced additional loop primers to accelerate the reaction, with amplification time less than half that of the original LAMP method. </P><P> LAMP can be used to detect various microorganisms, such as E.coli as few as 10 copies and HBV as few as 6 copies. As discussed above, LAMP is highly specific by using four to six primers, which bind six to eight sites of the target simultaneously. The final product can be easily analyzed by various means: 1, electrophoresis, ladder-shaped bands will appear on the agarose gel; 2, turbidity measurement, along with progress of the reaction the white precipitate appears, generating turbidity; 3, colorimetric discrimination, fluoresce probe serves as loop probe while cationic polymers (for example PEI) are used to hybridize LAMP products to precipitate final products and generate detectable color upon excitation, metal-sensitive molecules such as calcein with a shift from orange to green, or pH-sensitive dyes such as Phenol red, Cresol red Neutral red that change their color during reaction because of the release of protons and the drop of pH; 4, current response using Hoechst 33258 that binds DNA minor grooves, causing a significant drop in the current. </P><P>LAMP eliminates the need for thermal cycling equipment, as well as cost-consuming device and dye. It amplifies DNA with high specificity and rapidity and generates large amounts of DNA products. It is tolerant to inhibitors of PCR such as blood serum, urine; the reaction can be performed without the extraction of sample DNA. </P><P>However, there are some limitations for the LAMP. The primer design is complex because it utilizes four to six primers; Bst Large Fragment can catalyze primer-directed DNA synthesis even if there is no DNA template, resulting from the spurious amplification of the primers; the product it could elongate is restricted to less than 500bp. Moreover, the reaction is not suitable for quantitative analyses. </P><P>To overcome the problems, we incorporated palindromic sequence in the primer for efficient amplification, reducing the numbers of primers from six to two (Figure 3). In the initial step, primers would anneal to the template, the newly generated products serve as primers mutually, facilitating next round of amplification. The final products would be tens of thousands of tandem repeats of the target sequence.
                   </P><DIV id="fig3"><DIV class="col-md-12"><P><SMALL><B>Figure 3.</B>Our modified isothermal amplification using two primers. The two primers used to amplify target DNA can pair with each other. It generates initial products the same as conventional PCR; next, products with complementary sequence serve as primers mutually, allowing continuous amplification and producing a large amount of tandem repeats of the target sequence.</SMALL></P></DIV></DIV><P> As a proof of concept, we performed an isothermal amplification using our primers: </P><TABLE class="table" style="color:#666;font-size:14px"><TBODY><TR><TH>2×Reaction Mix（RM）</TH><TH>12.5μl</TH></TR><TR><TH>Distilled Water（DW）</TH><TH>9.5μl</TH></TR><TR><TH>Bst DNA Polymerase</TH><TH>1μl</TH></TR><TR><TH>Upstream Primer（40μM）</TH><TH>1μl</TH></TR><TR><TH>Downstream Primer（40μM）</TH><TH>1μl</TH></TR><TR><TH>DNA</TH><TH>2μl</TH></TR></TBODY></TABLE><P>Incubated at 63℃ or 60℃ for 60min; then 80℃ for 10min to deactivate Bst DNA Polymerase.</P><P> As shown in Figure 4, only the positive template generates ladder-like bands (a, b), which is an evidence that our primers work as expected. When the amplification products were subjected to our PC Reporter system, we obtained a significantly positive signal (c), demonstrating that our modified LAMP method really works. </P><DIV id="fig4"><DIV class="col-md-12"><P><SMALL><B>Figure 4.</B>Amplification using our modified LAMP method. (a), we could amplify DNA targets in different length; Lane 1 for 200 bp, Lane 4 for 400 bp, Lane 7 for 700 bp, and Lane 10 for 1000 bp; Lane 2, 5, 8, 11 and 14 for DNA negative control; Lane 3, 6, 9, and 12 for H<SUB>2</SUB>O negative control; Lane 13 for target DNA. (b), we reproduced the amplification using different incubation temperatures; 60 ℃ for 1~9, 63℃ for 10~18. The product length is 200 bp for Lane 1/10, 400 bp for Lane 4/13 and 1000 bp for Lane 7/16. The left 2 lanes are negative control. (c), detection of isothermal products using our PC reporter, we got a significantly positive signal even when the products was diluted by 5 fold (5× dilution of PCR products); 1×PCR products was directly used for the PC Reporter. mismatch target for the testing of PCR products with a non correct template, positive control for the detection of template DNA(1.5nM), negative control for no DNA samples. </SMALL></P></DIV></DIV><P>Compared with the conventional LAMP, our isothermal amplification method requires only two primers, making it easy for primer design. Besides, the amplification will not be constrained by the target length anymore; as shown in Figure 4, it could generate product over 1,000 bp. Combined with our PC Reporter system, we could imagine an improvement of sensitivity and reduction of turnover time for the nucleic acid detection in clinical practices.</P><P><BIG>References:</BIG></P><P>1. Notomi, T., Okayama, H., Masubuchi, H., Yonekawa, T., Watanabe, K., Amino, N., &amp; Hase, T. (2000). Loop-mediated isothermal amplification of DNA. Nucleic acids research, 28(12), e63-e63.
2. Nagamine, K., Hase, T., &amp; Notomi, T. (2002). Accelerated reaction by loop-mediated isothermal amplification using loop primers. Molecular and cellular probes, 16(3), 223-229.
3. Mori, Y., Nagamine, K., Tomita, N., &amp; Notomi, T. (2001). Detection of loop-mediated isothermal amplification reaction by turbidity derived from magnesium pyrophosphate formation. Biochemical and biophysical research communications, 289(1), 150-154.
4. Mori, Y., Hirano, T., &amp; Notomi, T. (2006). Sequence specific visual detection of LAMP reactions by addition of cationic polymers. BMC biotechnology, 6(1), 3.
5. Tomita, N., Mori, Y., Kanda, H., &amp; Notomi, T. (2008). Loop-mediated isothermal amplification (LAMP) of gene sequences and simple visual detection of products. Nature protocols, 3(5), 877-882.
6. Tanner, N. A., Zhang, Y., &amp; Evans Jr, T. C. (2015). Visual detection of isothermal nucleic acid amplification using pH-sensitive dyes. LAMP, 5, 6.
7. Safavieh, M., Ahmed, M. U., Tolba, M., &amp; Zourob, M. (2012). Microfluidic electrochemical assay for rapid detection and quantification of Escherichia coli. Biosensors and Bioelectronics, 31(1), 523-528.
8. Hill, J., Beriwal, S., Chandra, I., Paul, V. K., Kapil, A., Singh, T., ... &amp; Vats, A. (2008). Loop-mediated isothermal amplification assay for rapid detection of common strains of Escherichia coli. Journal of clinical microbiology, 46(8), 2800-2804.
9. Poon, L. L., Wong, B. W., Ma, E. H., Chan, K. H., Chow, L. M., Abeyewickreme, W., ... &amp; Peiris, J. M. (2006). Sensitive and inexpensive molecular test for falciparum malaria: detecting Plasmodium falciparum DNA directly from heat-treated blood by loop-mediated isothermal amplification. Clinical chemistry, 52(2), 303-306.10. Lage, J. M., Leamon, J. H., Pejovic, T., Hamann, S., Lacey, M., Dillon, D., ... &amp; Lizardi, P. M. (2003). Whole genome analysis of genetic alterations in small DNA samples using hyperbranched strand displacement amplification and array–CGH.Genome Research, 13(2), 294-307
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><FOOTER style="margin-top:30px; padding:0"><DIV class="container"><DIV class="row footer-widgets" style="padding:0:margin:0"><DIV class="col-md-3"><DIV class="col-md-12 footer-widget social-widget row" style="padding-top:0:padding-bottom:0;margin-top:0;margin-bottom:0"><H4>Share This</H4></DIV><DIV class="col-md-12 footer-widget social-widget row"><H4>Follow Us</H4></DIV></DIV><DIV class="col-md-6"><DIV class="footer-widget"><H4>Sponsors</H4></DIV></DIV><DIV class="col-md-3"><DIV class="footer-widget" style="padding:0:margin:0"><H4>Contact us</H4><DIV class="footer-widget contact-widget"><UL style="font-size:12px;color:#b4b4b4"><LI><SPAN>Email:</SPAN><A href="Mailto:pekingigem@gmail.com&amp;Subject=Hello&amp;Body=Hi, Peking iGEM!" style="color:#b4b4b4">pekingigem@gmail.com</A></LI><LI><SPAN>Website:</SPAN><A href="2015.igem.org/Team:Peking" style="color:#b4b4b4"> 2015.igem.org/Team:Peking</A></LI><LI><SPAN>Address:</SPAN> Peking University, No.5 Yiheyuan Road Haidian District, Beijing, P.R.China 100871</LI></UL></DIV></DIV></DIV><DIV class="copyright-section"><DIV class="row"><DIV class="col-md-12" style="padding:0;margin:0;height:30px"><P class="text-center">© 2015 Peking iGEM -  All Rights Reserved. Based on <A href="http://getbootstrap.com/" style="color:#b4b4b4">Bootstrap</A> and Margo template.</P></DIV></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>