<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTNU_Trondheim_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTNU_Trondheim/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTNU_Trondheim/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTNU_Trondheim/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTNU Trondheim/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="shadow-wrapper"><DIV id="menu"><UL><LI><A class="link top-heading" href="/Team:NTNU_Trondheim" title="NTNU Home Page">Home</A></LI></UL><UL><LI><H2 class="head top-heading">Team</H2></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Team" title="Team">About Us</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Contact Us" title="Contact Us">Contact Us</A></LI><LI><A class="link" href="https://igem.org/Team.cgi?year=2011&amp;team_name=NTNU_Trondheim" title="Official Team Profile">Official Team Profile</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Press" title="Press">Press</A></LI></UL><UL><LI><H2 class="head top-heading" href="/Team:NTNU_Trondheim/Project">Project</H2></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Project">The Project</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Modeling" title="Modeling">Modeling</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Journal" title="Protocols">Lab Journal</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Protocols" title="Protocols">Lab Protocols</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Journal_Modeling" title="Modeling Journal">Modeling Journal</A></LI></UL><UL><LI><H2 class="head top-heading">Data</H2></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Data">Data Page</A></LI><LI><H2 class="link">Our BioBricks</H2></LI><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/Stress_Sensor" title="Stress Sensor">Stress Sensor</A></LI><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/LacI+pLac+mCherry" title="LacI+pLac+mCherry">LacI+pLac +mCherry</A></LI><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/rrnB" title="rrnB">rrnB P1</A></LI><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/relA" title="relA">relA</A></LI></UL><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/Biobricks" title="BioBricks used">BioBricks Used</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Defective_Biobricks" title="Defective Biobricks">Defective Biobricks</A></LI><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/Modeling#Results" title="Modeling Results">Modeling Results</A></LI><UL><LI><A class="link head top-heading" href="/Team:NTNU_Trondheim/Safety" title="Safety">Safety</A></LI></UL><UL><LI><A class="link head top-heading" href="/Team:NTNU_Trondheim/Attributions" title="Attributions">Attributions</A></LI></UL><UL><LI><A class="link head top-heading" href="/Team:NTNU_Trondheim/Sponsors" title="Sponsors">Sponsors</A></LI></UL></DIV><DIV id="wikiContent"><H1><SPAN class="mw-headline" id="Modeling"> Modeling </SPAN></H1><P>To describe and understand the biological reactions and processes as the bacteria turns red under stress, we developed multiple mathematical and statistical models. As the most basic model we used a system of ordinary differential equations (ODE). This is a fully deterministic model, describing the change in concentration for all molecules involved. 
</P><P>Taking into account that the number of molecules in the cell, tends to be low and their movement random, one would have to use a model that allow randomness. This will result in a system of stochastic differential equations (SDE), which describe the underlying model from an observed pattern (random motion) in the number of molecules over time, given a concentration of ppGpp. A realization of such a model can be found under in figure in section for Results. Investigation of those models is done using Monte Carlo simulation in matlab and Dizzy.
</P><H2><SPAN class="mw-headline" id="Model_introduction"> Model introduction </SPAN></H2><P>At the heart of the modeling lies biological consistency and data integration. The modeling will be focused on interpretation simplicity and data consistency. That is to develop models that can be easily interpreted by biologist and mathematicians, but also describe what is observed at the laboratory. 
</P><P>The two main ways to model biological systems are deterministic and stochastic. In this project we will attempt to approach the problems in both ways. Using a deterministic model, with fixed parameters, and a stochastic model to integrate data more dynamically.
</P><H3><SPAN class="mw-headline" id="Process_description"> Process description </SPAN></H3><P>As described in the <A href="/Team:NTNU_Trondheim/Project" title="Team:NTNU Trondheim/Project">introduction</A>, ppGpp will repress the production of LacI and LacI represses the production of mCherry. These are the dominating processes. In short, when ppGpp are not present there will be little mCherry and when ppGpp are present the level of mCherry will be substantially higher. 
</P><P>In addition to these processes there are additional processes which might be of importance. First of all ppGpp affects the RNAP (RNA polymerase) and can therefore affect the production of mCherry as well as the production of LacI. As described in the introduction LacI is expected to be heavily downregulated by ppGpp. The production of mCherry can also in some cases lead to stress and therefore more ppGpp which in turn leads to more mCherry, in other words a positive feedback loop. These effects are assumed to be small, but might still affect the outcome. 
</P><P>Figure 1:<I>The processes involved in our system</I></P><H2><SPAN class="mw-headline" id="The_Models"> The Models</SPAN></H2><P>Two basic models where constructed, which will be described below. 
</P><H3><SPAN class="mw-headline" id="Systems_of_ODE"> Systems of ODE </SPAN></H3><P>Models based on Ordinary Differential Equations (ODEs) are one of the most used methods of describing genetic circuits, while the different processes and reactions taking place in the cell are described by a set of coupled differential equations. This method might give both qualitative and quantitative information about the system and can therefore be very useful. It is however dependent on accurate kinetic reaction parameters and in some cases one also has to take into account the stochastic nature of genetic circuits. The equations are then solved either in Dizzy or Matlab.
</P><H4><SPAN class="mw-headline" id="Basic_Model">Basic Model </SPAN></H4><P>Based on the model above the most important processes are, the symbols to the left of the equation denotes the kinetic parameter for the process:
</P><P>The first process describes ppGpp attaching to the RNA polymerase, <B>RNA<SUB>p</SUB><SUP>A</SUP></B> denotes the active part which is not repressed by ppGpp. <B>RNA<SUB>p</SUB><SUP>R</SUP></B> is the number of repressed RNA polymerase molecules. This process can also be reversed. The transcription of mRNA is described in one step. <B>D<SUB>0</SUB><SUP>1</SUP></B> is the promoter determining the production of LacI mRNA and <B>M<SUP>1</SUP></B> is LacI mRNA. The LacI mRNA in turn leads to production of LacI transcription factor denoted as <B>TF</B>. Both mRNA and LacI is degraded as well. LacI will then inhibit mCherry:
</P><P>Where <B>D<SUB>1</SUB><SUP>2</SUP></B> denotes promotors inhibited by LacI transcription factor. ξ is a measure of how strongly ppGpp affects mCherry directly, if 1 it is unaffected if 0 it is affected as strongly as LacI.
</P><H4><SPAN class="mw-headline" id="Steady_State">Steady State</SPAN></H4><P>Since it is difficult to find accurate parameters for all the processes involved and since it is often only the concentrations in steady state (long after the stress was first induced) that can be measured, all quantities involved were assumed to be constant. This assumption simplifies the system greatly and reduced the problem to these two equations:
</P><P>Where C<SUP>i</SUP>=α<SUB>M</SUB><SUP>i</SUP>α<SUB>p</SUB><SUP>i</SUP>RNAp<SUP>T</SUP>/γ<SUB>M</SUB><SUP>i</SUP>γ<SUB>p</SUB><SUP>i</SUP>, RNAp<SUP>T</SUP> is the total number of RNA-polymerase molecules in the cell and K<SUP>i</SUP>=q<SUP>i</SUP>/k<SUP>i</SUP>.
To check that the equations for steady state are correct a comparison was made between the numerical solution in Dizzy and the analytical solution for different levels of ppGpp. The comparison shown in the figure below shows that there is excellent agreement.
</P><P>Figure 2: <I>Comparison between numerical simulation and analytical solution in steady state for different values of ppGpp</I></P><H3><SPAN class="mw-headline" id="Stochastic_Differential_equations"> Stochastic Differential equations </SPAN></H3><P>In the cell, all movement of the molecules is random, this gives rise to the stochasticity observed in gene expression experiment. In this section we outline two methods where the randomness is accounted for; the Gillespie algorithm and the approximation  τ - leap algorithm.
</P><P>When the number of participating molecules are low (which do happen in the cell), then stochasticity really matters. If there are many molecules, then the behavior of the reactions goes &quot;smooth&quot; and looks like an ODE. However when to number of reactants gets small, the number of reactions in a small time frame varies, this gives rise to the irregularity seen in the time series for the number of created products in the cell.     
</P><H4><SPAN class="mw-headline" id="Gillespie">Gillespie </SPAN></H4><P>The Gillespie algorithm is a model that simulates the number of reactions and the time between the reactions exactly (under some assumptions). The most important assumptions is that the concentration of the reactants is distributed uniformly in the cell, the other important assumption is that the time between the reactions occur is markovian, that is it has no memory of how long the previous time step was.
</P><P>An improved version of the original algorithm, the Gibson-Bruck algorithm, was used with Dizzy.
</P><H4><SPAN class="mw-headline" id="Tau_-_leap_Algorithm">Tau - leap Algorithm </SPAN></H4><P>Instead of calculating the time between each reaction, we can fix a time frame and estimate how many reactions occur in that time frame. 
</P><P>It can be shown that setting the production and destruction rate from the ODE in as a rate parameter in an Poisson distribution, will give the same result. This is a nice way of reconfiguring the tau - leap such that it gives a clear and precise relation to the ODE models, and mathematical consistency.
</P><P>This approach was used to create a framework for a stochastic simulation using the computer language C++, there where an attempt to make this in a fully functional open source program. However this attempt stranded due to lack of time.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>This section describes the findings from the theoretical model presented in the equations above.
</P><P>As seen in figure 3 the level of LacI is reduced when ppGpp increases in the cell, which in turn leads to increased level of mCherry. The level of LacI reaches a saturation point, since there will be some production of LacI due to leakage, this sets a limit from above for mCherry. The stochastic realization fluctuates around the deterministic results, with the distributions shown in figure 4.
</P><P>Figure 3:<I>The number of molecules of the proteins for the stochastic and  the deterministic models are shown for different values of ppGpp</I></P><P>All the distributions are unimodal having only one peak, peaked around the steady state value from the deterministic model. The distributions for ppGpp=630 and ppGpp=1030 have heavy tails which indicates large deviations from the mean value, while the distributions for ppGpp=230 and ppGpp=1430 are more symmetric, resembling the normal distribution. 
</P><P>Figure 4:<I>The distribution of mCherry for t=2000s to t=10000s for different values of ppGpp. The distributions was calculated from the realizations in figure 3</I></P><P>As shown in figure 5 the variance increases linearly with increasing level of ppGpp, while the Fano factor (σ<SUP>2</SUP>/μ) is almost constant around 1.5 .
</P><P>Figure 5: <I>Variance and Fano factor for increasing values of ppGpp</I></P><H2><SPAN class="mw-headline" id="Model_Validation"> Model Validation </SPAN></H2><P>To see if the results agree with the theoretical model is vital in all parts of science. In this section we will give a short reasoning of why our models seems to fit the observed data. Here we will consider our model in two distinct parts, one deterministic model and one stochastic model, both of equally importance. 
</P><P>Because of the limited amount of data currently available it is diffucult to draw conclusions. In the modeling the level of ppGpp was assumed known, however we do not know at present the relationship between a stress level and the number of ppGpp and the relationship between the number of mCherry molecules and the flourescence level is also unknown. 
</P><P>However there are certain indications that the model predicts the behaviour of our system. For the deterministic model, we saw a significant increase in the concentration of mCherry as ppGpp was assumed present. This is consistent with the observation, described in figure 1 at <A href="/Team:NTNU_Trondheim/Stress_Sensor#Stress_sensor_characterization" title="Team:NTNU Trondheim/Stress Sensor">Stress sensor characterization</A>. 
</P><P>Using the distributions from figure 4, we can compare these to the spread in the flowcytometry, and see a close similarity in form of shape. Assuming scale is irrelevant (i.e. that there is a linear relationship between number of mCherry molecules and fluorescence count) one can come to a conclusion that our model seems to describe the actual biological system. 
</P><DIV><H4><SPAN class="mw-headline">Sponsors</SPAN></H4><DIV id="sponsor-box"><A class="sponsor" href="http://www.ntnu.edu/rector"><B>NTNU Rector's Office</B></A><A class="sponsor" href="http://www.ntnu.edu/bioinformatics/pbi"><B>Programme for Bioinformatics, NTNU</B></A></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:NTNU_Trondheim/Modeling">http://2011.igem.org/Team:NTNU_Trondheim/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTNU_Trondheim/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTNU_Trondheim/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Modeling&amp;oldid=149999" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>