<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTNU_Trondheim_Journal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTNU_Trondheim/Journal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTNU_Trondheim/Journal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Journal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Journal&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTNU_Trondheim/Journal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTNU Trondheim/Journal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="shadow-wrapper"><DIV id="menu"><UL><LI><A class="link top-heading" href="/Team:NTNU_Trondheim" title="NTNU Home Page">Home</A></LI></UL><UL><LI><H2 class="head top-heading">Team</H2></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Team" title="Team">About Us</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Contact Us" title="Contact Us">Contact Us</A></LI><LI><A class="link" href="https://igem.org/Team.cgi?year=2011&amp;team_name=NTNU_Trondheim" title="Official Team Profile">Official Team Profile</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Press" title="Press">Press</A></LI></UL><UL><LI><H2 class="head top-heading" href="/Team:NTNU_Trondheim/Project">Project</H2></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Project">The Project</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Modeling" title="Modeling">Modeling</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Journal" title="Protocols">Lab Journal</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Protocols" title="Protocols">Lab Protocols</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Journal_Modeling" title="Modeling Journal">Modeling Journal</A></LI></UL><UL><LI><H2 class="head top-heading">Data</H2></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Data">Data Page</A></LI><LI><H2 class="link">Our BioBricks</H2></LI><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/Stress_Sensor" title="Stress Sensor">Stress Sensor</A></LI><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/LacI+pLac+mCherry" title="LacI+pLac+mCherry">LacI+pLac +mCherry</A></LI><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/rrnB" title="rrnB">rrnB P1</A></LI><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/relA" title="relA">relA</A></LI></UL><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/Biobricks" title="BioBricks used">BioBricks Used</A></LI><LI><A class="link" href="/Team:NTNU_Trondheim/Defective_Biobricks" title="Defective Biobricks">Defective Biobricks</A></LI><LI><A class="link" href="https://2011.igem.org/Team:NTNU_Trondheim/Modeling#Results" title="Modeling Results">Modeling Results</A></LI><UL><LI><A class="link head top-heading" href="/Team:NTNU_Trondheim/Safety" title="Safety">Safety</A></LI></UL><UL><LI><A class="link head top-heading" href="/Team:NTNU_Trondheim/Attributions" title="Attributions">Attributions</A></LI></UL><UL><LI><A class="link head top-heading" href="/Team:NTNU_Trondheim/Sponsors" title="Sponsors">Sponsors</A></LI></UL></DIV><DIV id="wikiContent"><H1><SPAN class="mw-headline" id="Lab_Journal">Lab Journal</SPAN></H1><TABLE align="center" style="font-size: 10px;"><TBODY><TR><TH style="text-align:center;" colspan="7"><A href="https://2011.igem.org/wiki/index.php?title=Team:NTNU_Trondheim/Journal_Modeling/Calendar/June&amp;action=edit" class="external text" rel="nofollow">June</A></TH></TR><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun</TD></TR><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD></TR><TR><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD><TD>11
</TD><TD>12
</TD></TR><TR><TD>13
</TD><TD>14
</TD><TD>15
</TD><TD>16
</TD><TD>17
</TD><TD>18
</TD><TD>19
</TD></TR><TR><TD>20
</TD><TD>21
</TD><TD>22
</TD><TD><A href="#Thursday_23.2F6">23</A></TD><TD><A href="#Friday_24.2F6">24</A></TD><TD><A href="#Saturday_25.2F6">25</A></TD><TD><A href="#Sunday_26.2F6">26</A></TD></TR><TR><TD><A href="#Monday_27.2F6">27</A></TD><TD><A href="#Tuesday_28.2F6">28</A></TD><TD><A href="#Wednesday_29.2F6">29</A></TD><TD><A href="#Thursday_30.2F6">30</A></TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;"><TBODY><TR><TH colspan="7" style="text-align:center;"><A href="https://2011.igem.org/wiki/index.php?title=Team:NTNU_Trondheim/Journal/Calendar/July&amp;action=edit" class="external text" rel="nofollow">July</A></TH></TR><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD><A href="#Friday_1.2F7"> 1</A></TD><TD> 2
</TD><TD> 3
</TD></TR><TR><TD><A href="#Monday_4.2F7"> 4</A></TD><TD><A href="#Tuesday_5.2F7"> 5</A></TD><TD><A href="#Wednesday_6.2F7"> 6</A></TD><TD><A href="#Thursday_7.2F7"> 7</A></TD><TD><A href="#Friday_8.2F7"> 8</A></TD><TD><A href="#Saturday_9.2F7"> 9</A></TD><TD>10
</TD></TR><TR><TD><A href="#Monday_11.2F7">11</A></TD><TD><A href="#Tuesday_12.2F7">12</A></TD><TD><A href="#Wednesday_13.2F7">13</A></TD><TD><A href="#Thursday_14.2F7">14</A></TD><TD><A href="#Friday_15.2F7">15</A></TD><TD><A href="#Saturday_16.2F7">16</A></TD><TD><A href="#Sunday_17.2F7">17</A></TD></TR><TR><TD><A href="#Monday_18.2F7">18</A></TD><TD><A href="#Tuesday_19.2F7">19</A></TD><TD><A href="#Wednesday_20.2F7">20</A></TD><TD><A href="#Thursday_21.2F7">21</A></TD><TD><A href="#Friday_22.2F7">22</A></TD><TD><A href="#Saturday_23.2F7">23</A></TD><TD><A href="#Sunday_24.2F7">24</A></TD></TR><TR><TD><A href="#Monday_25.2F7">25</A></TD><TD><A href="#Tuesday_26.2F7">26</A></TD><TD><A href="#Wednesday_27.2F7">27</A></TD><TD><A href="#Thursday_28.2F7">28</A></TD><TD><A href="#Friday_29.2F7">29</A></TD><TD><A href="#Saturday_30.2F7">30</A></TD><TD><A href="#Sunday_31.2F7">31</A></TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;"><TBODY><TR><TH colspan="7" style="text-align:center;"><A href="https://2011.igem.org/wiki/index.php?title=Team:NTNU_Trondheim/Journal/Calendar/August&amp;action=edit" class="external text" rel="nofollow">August</A></TH></TR><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD><A href="#Monday_1.2F8">1</A></TD><TD><A href="#Tuesday_2.2F8">2</A></TD><TD><A href="#Wednesday_3.2F8">3</A></TD><TD><A href="#Thursday_4.2F8">4</A></TD><TD><A href="#Friday_5.2F8">5</A></TD><TD> 6
</TD><TD> 7
</TD></TR><TR><TD><A href="#Monday_8.2F8">8</A></TD><TD><A href="#Tuesday_9.2F8">9</A></TD><TD><A href="#Wednesday_10.2F8">10</A></TD><TD><A href="#Thursday_11.2F8">11</A></TD><TD><A href="#Friday_12.2F8">12</A></TD><TD><A href="#Saturday_13.2F8">13</A></TD><TD><A href="#Sunday_14.2F8">14</A></TD></TR><TR><TD><A href="#Monday_15.2F8">15</A></TD><TD><A href="#Tuesday_16.2F8">16</A></TD><TD><A href="#Wednesday_17.2F8">17</A></TD><TD><A href="#Thursday_18.2F8">18</A></TD><TD>19
</TD><TD>20
</TD><TD>21
</TD></TR><TR><TD>22
</TD><TD>23
</TD><TD>24
</TD><TD>25
</TD><TD>26
</TD><TD>27
</TD><TD>28
</TD></TR><TR><TD>29
</TD><TD>30
</TD><TD>31
</TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;"><TBODY><TR><TH colspan="7" style="text-align:center;"><A href="https://2011.igem.org/wiki/index.php?title=Team:NTNU_Trondheim/Journal/Calendar/September&amp;action=edit" class="external text" rel="nofollow">September</A></TH></TR><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD> 1
</TD><TD> 2
</TD><TD> 3
</TD><TD> 4
</TD></TR><TR><TD> 5
</TD><TD> 6
</TD><TD> 7
</TD><TD><A href="#Thursday_8.2F9">8</A></TD><TD><A href="#Friday_9.2F9">9</A></TD><TD>10
</TD><TD>11
</TD></TR><TR><TD><A href="#Monday_12.2F9">12</A></TD><TD><A href="#Tuesday_13.2F9">13</A></TD><TD><A href="#Wednesday_14.2F9">14</A></TD><TD>15
</TD><TD>16
</TD><TD>17
</TD><TD>18
</TD></TR><TR><TD>19
</TD><TD>20
</TD><TD>21
</TD><TD>22
</TD><TD>23
</TD><TD>24
</TD><TD>25
</TD></TR><TR><TD>26
</TD><TD>27
</TD><TD>28
</TD><TD>29
</TD><TD>30
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Thursday_23.2F6"> Thursday 23/6 </SPAN></H2><P>Lab equipment was prepared: Pipette tips, 1.5 ml tubes, toothpicks, SOC, LA with 100 µg/ml ampicillin/ 100 µg/ml spectinomycin (from 1988). Recipies are given in recipies section.
</P><H2><SPAN class="mw-headline" id="Friday_24.2F6"> Friday 24/6 </SPAN></H2><P>Biobricks were taken out from kit by resuspending them in sterile water followed by transformation 
to <I>E.coli</I> DH5 alpha. The biobricks that were taken out are given below:
</P><TABLE border="1"><TBODY><TR><TH>Abbreviation
</TH><TH>Name
</TH><TH>Part number
</TH><TH>Resistance
</TH></TR><TR><TD>P1
</TD><TD>rrnB P1
</TD><TD>BBa_K112118
</TD><TD>Spec
</TD></TR><TR><TD>lambda pR
</TD><TD>lambda pR mod
</TD><TD>BBa_R0051
</TD><TD>Amp
</TD></TR><TR><TD>RFP
</TD><TD>TetR + p(TetR)+RFP
</TD><TD>BBa_K092600
</TD><TD>Amp
</TD></TR><TR><TD>Lux
</TD><TD>TetR + p(TetR)+RFP+luxI
</TD><TD>BBa_K092700
</TD><TD>Amp
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Saturday_25.2F6"> Saturday 25/6 </SPAN></H2><P>Plates with transformants were moved to fridge.
</P><H2><SPAN class="mw-headline" id="Sunday_26.2F6"> Sunday 26/6 </SPAN></H2><P>Transformants were inoculated in 3 ml LB + proper antibiotics to prepare for isolation of plasmid.
</P><H2><SPAN class="mw-headline" id="Monday_27.2F6"> Monday 27/6 </SPAN></H2><P>Plasmids that contained P1, lambda pR, RFP and Lux biobricks were isolated by using miniprep kit from Promega.
</P><P>Consentrations of the biobricks were measured and were as follows:
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration (ng/µl)
</TH></TR><TR><TD>P1
</TD><TD>98.4
</TD></TR><TR><TD>lamda pR
</TD><TD>43.8
</TD></TR><TR><TD>RFP
</TD><TD>141.8
</TD></TR><TR><TD>Lux
</TD><TD>62.8
</TD></TR></TBODY></TABLE><P>
Transformants from Lux/RFP constructs were not red as expected. It was suggested that the phenotype could be explained by presence of TetR in the system. Since tetracycline (Tc) is an inhibitor of TetR we tried to grow the transformants in sublethal consentrations of Tc. <I>E. coli</I> with RFP were grown in 3 ml LB with 0, 0.1, 0.3, 0.6, 1.0, 1.5 and 100 µg/ml Tc.
</P><H2><SPAN class="mw-headline" id="Tuesday_28.2F6"> Tuesday 28/6 </SPAN></H2><P>It's Jon's birthday! Happy day!
</P><P>Growing the bacteria in Tc did not result in red color of the culture
</P><P>lambda pR was ligated to RFP backbone:
</P><UL><LI>RFP  and Luc construct were cut with EcoRI and XbaI.
</LI><LI>lambda pR was cut with EcoRI og SpeI
</LI></UL><P>Restriction digestion was done using <A href="http://partsregistry.org/Help:Protocols/Restriction_Digest" class="external text" rel="nofollow">iGEM's protocol</A></P><P>Since the restriction digestion gave unexpected fragments and we had not yet seen any pigmentation, we started looking for other designs that could give the result that we wanted. We came up with a different design that used pLac promoter and lacI repressor instead of the pTet/TetR system.
</P><P>To start constructing the alternative system 3 new biobricks were transformed to <I>E. coli</I>:
</P><TABLE border="1"><TBODY><TR><TH>Abbreviation
</TH><TH>Name
</TH><TH>Part number
</TH></TR><TR><TD>LacI
</TD><TD>LacI + RBS
</TD><TD>BBa_J24679</TD></TR><TR><TD>pLac
</TD><TD>Lac promoter hybrid
</TD><TD>BBa_R0011
</TD></TR><TR><TD>mCherry
</TD><TD>mCherry + RBS + term
</TD><TD>BBa_J06702
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Wednesday_29.2F6"> Wednesday 29/6 </SPAN></H2><P><B>Lambda Pr, RFP and Lux was cut:</B></P><UL><LI>RFP with XbaI and PstI
</LI><LI>Lux with XbaI and PstI
</LI><LI>Lambda Pr with SpeI and PstI
</LI></UL><P>The cutting gives RFP and Lux as insert and Lambda pR the backbone.
</P><P>The restriction fragments were separated on agarose gel:
</P><P>The restriction fragments for lambda pR were as expected but not for RFP or Lux.
</P><P>Lambda pR was isolated from gel (2110 nt) by using DNA gel extraction kit.
</P><P>DNA concentration of lambda pR was measured to 5.2 ng/µl 
</P><P>Biobricks that were transformed yesterday were inoculated in 3 ml LB with proper antibiotics to prepare for isolation of plasmids.
</P><P>Anders ordered a biobrick that should contain LacI + RBS + double terminator from iGEM HQ: <B>BBa_K292006</B></P><H2><SPAN class="mw-headline" id="Thursday_30.2F6">Thursday 30/6</SPAN></H2><P>Plasmid was isolated from lac-design biobricks. Concentration of DNA:
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration (ng/µl)
</TH></TR><TR><TD>pLac
</TD><TD>16.5
</TD></TR><TR><TD>lacI
</TD><TD>37.1
</TD></TR><TR><TD>mCherry
</TD><TD>36.7
</TD></TR></TBODY></TABLE><P>
pLac biobrick was cut with SpeI and PstI
mCherry was cut with XbaI and PstI
</P><P>Makes pLac backbone and mCherry insert
</P><P>Tried direct ligation of restriction fragments using following reaction mix:
</P><TABLE border="1"><TBODY><TR><TH>Compound
</TH><TH>Amount (µl)
</TH></TR><TR><TD>H20
</TD><TD>11
</TD></TR><TR><TD>Backbone
</TD><TD>2
</TD></TR><TR><TD>Insert
</TD><TD>2
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2
</TD></TR><TR><TD>Ligase
</TD><TD>1
</TD></TR></TBODY></TABLE><P>
Reaction mix were mixed and incubated 30 min at 16C. Heat destruction of enzymes for 20 min at 80C. 2 µl of ligation mix were used for transformation. Red colonies were to be selected from transformation.
Mix, spin down, 30 min 16C, heat kill 20 min 80C. Transform 2µL. Select for red colonies.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel picture of the separated mCherry insert (895 bp) and pLac backbone (2116 bp) fragments.</DIV></DIV></DIV><P>The rest of the ligation mix were separated on agarose gel and mCherry insert (895 bp) and pLac backbone (2116 bp) were isolated from gel using Quiaquick Gel Extraction Kit.
DNA concentration was measured:
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration (ng/µl)
</TH></TR><TR><TD>pLac
</TD><TD>8.5
</TD></TR><TR><TD>mCherry
</TD><TD>2.1
</TD></TR></TBODY></TABLE><P>
The directly ligated restriction fragments transformed to <I>E. coli</I> were plated out on LA + Amp and LA + IPTG and grown ON.
</P><P>The biobrick that has been ordered (BBa_K292006) was investigated and was found to be identical to the biobrick that we were planning to make, given that the sequence is as given in the registry. 
</P><P><B>relA</B></P><P>The nucleotide sequence of relA (codes for protein that synthesize ppGpp) was found in database. One hickup in making this gene a biobrick; there is a PstI site inside the gene (at 1383-1388 nt if added prefix). To overcome this problem we could use partial digestion or we could introduce a silent mutation that removes the restriction site. By changing CTGCAG → CTACAG or CTGCAA this could be done. 
</P><P><B>LacI-negative E.coli</B></P><P>To make the lac-system work we should use a strain that is lacI-negative. According to <A href="http://partsregistry.org/Part:BBa_K177038" class="external text" rel="nofollow">another iGEM group</A> the strain Top10 is lacI-negative.
</P><H2><SPAN class="mw-headline" id="Friday_1.2F7">Friday 1/7</SPAN></H2><P>mCherry insert and pLac backbone was isolated from gel using Quiquick Gel Extraction kit and then ligated using the following reaction mix:
</P><TABLE border="1"><TBODY><TR><TH>Compound
</TH><TH>Amount (µl)
</TH></TR><TR><TD>Insert
</TD><TD>14
</TD></TR><TR><TD>Backbone
</TD><TD>3
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2
</TD></TR><TR><TD>Ligase
</TD><TD>1
</TD></TR></TBODY></TABLE><P>
The ligation mix was ligated at 16C for 1 hour.
</P><P>Ligation mix was then transformed to LA + Amp, LA + Amp + IPTG and LA + Amp + more IPTG 
</P><P>Two new biobricks were also transformed and plated out on LA + Amp:
</P><TABLE border="1"><TBODY><TR><TH>Abbreviation
</TH><TH>Biobrick
</TH><TH>Part name
</TH></TR><TR><TD>GFP
</TD><TD>GFP + LVA
</TD><TD>BBa_K082003
</TD></TR><TR><TD>RBS
</TD><TD>Ribosome binding site
</TD><TD>BBa_B0034
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Monday_4.2F7"> Monday 4/7 </SPAN></H2><P>Plasmid isolation of GFP, RBS and LacP+mCherry. Concentrations were measured.
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration (ng/µl)
</TH></TR><TR><TD>GFP
</TD><TD>60.1
</TD></TR><TR><TD>RBS
</TD><TD>16.2
</TD></TR><TR><TD>pLac + mCherry
</TD><TD>29.1
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel picture of the separated fragments resulting from the  restriction cutting of the biobricks &quot;GFP&quot; and &quot;RBS&quot;. The squares indicate the fragments that where cut out of the gel</DIV></DIV></DIV><P><B>Restriction cutting of RBS and GFP:</B></P><UL><LI>RBS: SpeI and PstI making it the backbone
</LI><LI>GFP: XbaI and PstI making it the insert. 
</LI></UL><P><B>Gel Extraction:</B></P><P>Extracted RBS backbone, and GFP insert form the gel. 
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration (ng/µl)
</TH></TR><TR><TD>GFP
</TD><TD>3.0
</TD></TR><TR><TD>RBS
</TD><TD>4.6
</TD></TR></TBODY></TABLE><P><B>Plating:</B></P><P>Plating of pLac + mCherry on amp plates, amp plates with additional IPTG (7µL), amp + IPTG plates and amp + IPTG plates with additional IPTG (7 µL). Incubating on 30 and 37 degrees.
</P><H2><SPAN class="mw-headline" id="Tuesday_5.2F7">Tuesday 5/7</SPAN></H2><P>Ligation of RBS backbone and GFP insert.
</P><UL><LI>10μL Ligation Mix
</LI><LI>1.0 μL 10X T4 ligase buffer
</LI><LI>6:1 molar ratio of insert to vector (~10ng vector)
</LI><LI>Add (8.5 - vector and insert volume)μl ddH2O
</LI><LI>0.5 μL T4 Ligase
</LI></UL><P>Used 2,5 µL RBS backbone and 6 µL GFP insert.
Incubating at 16 degrees celsius, then heating at 80 degrees celsius. 
</P><P>PCR amplification of rrnB P1 from BioBrick
</P><P>Resuspend primers in dH20. Primers were designed by the team, and synthesized by <A href="http://www.sigmaaldrich.com" class="external text" rel="nofollow">Sigma-Aldrich</A>. Primers will amplify the whole rrnB P1 BioBrick (rrnB.FWD and REV), and only the assumed promoter sequence (pro.FWD and REV), and will add normal prefix and suffix instead of BBb format. 
</P><P>Primers used:
</P><TABLE border="1"><TBODY><TR><TH>Primer
</TH><TH>Type
</TH><TH>Sequence
</TH></TR><TR><TD>rrnB P1 F
</TD><TD>Forward
</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGAGACGTATCCTACGCCCGTGGT
</TD></TR><TR><TD>rrnB P1 R
</TD><TD>Reverse
</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTACGCCTTCCCGCTACAGAGTCA
</TD></TR><TR><TD>proL F
</TD><TD>Forward
</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGAGCCTCTTGTCAGGCCGGAATAACTCC
</TD></TR><TR><TD>proL R
</TD><TD>Reverse
</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTAGCGGCGTGTTTGCCGTTGTT
</TD></TR></TBODY></TABLE><P>
PCR mix:
</P><P>Work on ice
</P><UL><LI>Dillute P1 DNA 1µL + 9µL dH2O → ( 9,84 g/µL P1 DNA)
</LI><LI>2 x PCR tubes 0,5 µL DNA
</LI><LI>5 µL 10x PCR buffer 2 (w/MgCl)
</LI><LI>0,5 µL 10 mM dNTPs
</LI><LI>1 µL fwd primer (rrnBL og proL)
</LI><LI>1 µL rev primer (rrnBL og proL)
</LI><LI>0,5 µL polymerase    
</LI><LI>41,5 µL H20
</LI></UL><P>Heat cycles:
</P><P>Heated lid 103C
</P><UL><LI>Initial denaturation:     94C 2 min
</LI><LI>Denaturing:        94C 30 s
</LI><LI>Annealing        58C 30 s
</LI><LI>Elongation        72C 60 s
</LI><LI>Repeat 2-4 34 times
</LI><LI>Final elongation    72C 7 min
</LI><LI>Cooling         4 C HOLD
</LI></UL><P>Running products on 1,5 % agarose to find short fragment (promoter sequence). Should give 552 bp (rrnB) and  129 bp (proL).
The products were not so good, so we are running a new PCR tomorrow with different scheme.
</P><P>Inoculated &quot;LacI with RBS&quot;, TERM and LacP+MC biobrick. LacP+MC with and without IPTG and incubating at 30 and 37 degrees celsius.
</P><H2><SPAN class="mw-headline" id="Wednesday_6.2F7">Wednesday 6/7</SPAN></H2><P>Running PCR with rrnBS and L and proS and L on the rrnB P1 BioBrick.
5 cycles of 58C annealing temp, and 20 cycles of 68C annealing temp to try adjust for the massively changed annealing temperatures when primers with pre- and suffixes bind to PCR products.
The PCR products where separated by gel electrophoresis and cut out, ready for gel extraction tomorrow.
</P><P>Isolated plasmids containing the &quot;LacI with RBS&quot; and &quot;TERM&quot; biobricks. 
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration (ng/µl)
</TH></TR><TR><TD>LacI with RBS
</TD><TD>26.0
</TD></TR><TR><TD>TERM
</TD><TD>29.9
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel picture of the separated fragments resulting from the  restriction cutting of the biobricks &quot;LacI with RBS&quot; and &quot;TERM&quot;. The squares indicate the fragments that where cut out of the gel</DIV></DIV></DIV><P>Restriction cutting of &quot;LacI with RBS&quot; with EcoRI and SpeI, and &quot;TERM&quot; with EcoRI and XbaI. 
The fragments where separated using gel electrophoresis.
</P><P>The &quot;LacI with RBS&quot; insert and the backbone containing &quot;TERM&quot; where cut out of the gel, and extracted from it. 
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration (ng/µl)
</TH></TR><TR><TD>LacI with RBS insert
</TD><TD>1.6
</TD></TR><TR><TD>TERM backbone
</TD><TD>3.9
</TD></TR></TBODY></TABLE><P>
RBS+GFP transformants where inoculated in 3 mL LB+amp. 
</P><H2><SPAN class="mw-headline" id="Thursday_7.2F7">Thursday 7/7</SPAN></H2><P>Ligation of TERM backbone and LacI with RBS insert extracted from gel 6/7.
</P><P>10μL Ligation Mix:
</P><UL><LI>1.0 μL 10X T4 ligase buffer
</LI><LI>6:1 molar ratio of insert to vector (~10ng vector)
</LI><LI>Add (8.5 - vector and insert volume)μl ddH2O
</LI><LI>0.5 μL T4 Ligase
</LI></UL><P>Used 2,5 µL TERM backbone and 6 µL LacI with RBS insert.
Incubated at room temperature for 30 minutes, then heated at 65 C for 15 minutes. 
</P><UL><LI>Plasmid with GFP+RBS was isolated. Concentration was <B>26.7 ng/µl</B></LI></UL><P>Restriction cutting:
</P><P><B>rrnB P1 BioBrick</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">PCR products from rrnB P1 BioBrick separated on 1.5 % agarose. The marked products represent rrnB P1 with regular pre/sufix, and the promoter sequence w. pre/suffix</DIV></DIV></DIV><P>The rrnB P1 BioBrick found in the distribution was in a different format (BBb), and was therefore not compatible with the other BioBricks we are using. PCR was used to amplify the BioBrick itself, adding the prefix and suffix found <A href="http://openwetware.org/wiki/Synthetic_Biology:BioBricks/Part_fabrication" class="external text" rel="nofollow">here</A> making it compatible with the other parts, but losing the first 10 bp. We also amplified only the assumed promoter sequence, with the pre/suffixes. 
</P><P>Gel electrophoresis with the PCR products showed that DNA of the right sequence length had been amplified, as shown in the figure. The full-length rrnB P1 amplicon had an approximate size of 500 bp, and the shorter promoter-only amplicon was approx. 100 bp. The shorter fragments shown were created using shorter primers with only X and S restriction sites, in case the long primers failed. 
</P><P>The amplicons with the complete pre/suffixes were cut out, as shown in the figure, and purified with QIAquick Gel Extraction Kit. 
</P><P>Both promoter variants were cut with E and P and also with E and S to create insertable pieces. Our RBS+GFP BioBrick was cut with E and X to create a backbone, and the linearized backbones psB1A3 and psb1C3 were cut to accomodate the cut PCR products.
</P><H2><SPAN class="mw-headline" id="Friday_8.2F7">Friday 8/7</SPAN></H2><P><I>It's Friday, we excited...</I></P><P>The following ligations were made:
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">RBS + GFP insert fragments are shown in the black box.</DIV></DIV></DIV><TABLE border="1"><TBODY><TR><TH>Testing activity
</TH><TH>Ligate in BB plasmid
</TH></TR><TR><TD>rrnBL+mCherry
</TD><TD>rrnBL+backbone pSB1A3
</TD></TR><TR><TD>rrnBL+(RBS+GFP)
</TD><TD>rrnBL+backbone pSB1C3
</TD></TR><TR><TD>ProL+mCherry
</TD><TD>ProL+backbone psB1A3
</TD></TR><TR><TD>ProL+(RBS+GFP)
</TD><TD>ProL+backbone psB1C3
</TD></TR></TBODY></TABLE><P>
RBS + GFP was cut with Xbal and PstI, and the restriction fragments were separated on agarose gel. The insert fragment RBS + GFP were as expected (ca 750 bp), and the fragment was cut out from gel and extracted by using DNA gel extraction kit. The DNA concentration of RBS + GFP insert was measured to 2.1 ng/µl.
</P><P><B>PCR amplification of relA from chromosomal DNA</B></P><P>Resuspend primers in dH20. Primers were designed by the group and synthesized by Sigma-Aldrich. Primers will amplify the whole relA gene from E. coli and add normal prefix and suffix (biobrick standard).
</P><P>Primers used:
</P><TABLE border="1"><TBODY><TR><TH>Primer
</TH><TH>Type
</TH><TH>Sequence
</TH></TR><TR><TD>relA F
</TD><TD>Forward
</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGAGATGGTTGCGGTAAGAAGTGCACA
</TD></TR><TR><TD>relA R
</TD><TD>Reverse
</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTACTAACTCCCGTGCAACCGACG
</TD></TR></TBODY></TABLE><P>
PCR mix: 
Same as used earlier.
</P><P>Heat cycles:
</P><P>Heated lid 103C
</P><UL><LI>Initial denaturation:     94C 2 min
</LI><LI>Denaturing:        94C 30 s
</LI><LI>Annealing:        58C 30 s
</LI><LI>Elongation:        72C 150 s
</LI><LI>Repeat 2-4 4 times
</LI><LI>Denaturing:        94C 30 s
</LI><LI>Annealing:        68C 30s
</LI><LI>Elongation:        72C 150 s
</LI><LI>Repeat 6-8 19 times
</LI><LI>Final elongation    72C 7 min
</LI><LI>Cooling         4 C HOLD
</LI></UL><P>PCR products where put in the freezer for further work tomorrow.
</P><H2><SPAN class="mw-headline" id="Saturday_9.2F7"> Saturday 9/7 </SPAN></H2><P>Transformed new promoter system for our relA gene from biobrick plate and plated out on Kam plates:
</P><TABLE border="1"><TBODY><TR><TH>Abbreviation
</TH><TH>Biobrick
</TH><TH>Part name
</TH></TR><TR><TD>pBAD
</TD><TD>Inducible pBad/araC promoter
</TD><TD>BBa_I0500
</TD></TR></TBODY></TABLE><P>
Changed medium for our TOP10 culture
</P><P>Separated PCR products of the relA gene on gel and used QIAquick Gel Extraction Kit for cleanup.
Concentration: 3 ng/µL
</P><H2><SPAN class="mw-headline" id="Monday_11.2F7"> Monday 11/7 </SPAN></H2><P>The new relA biobrick was digested with EcoRI and PstI for insert in plasmid backbone pSB1A3. The following reaction mix was used:
</P><UL><LI> 17 μl DNA
</LI><LI> 2 μl buffer 3
</LI><LI> 1 μl EcoRI
</LI><LI> 1 μl PstI
</LI><LI> 0.5 μl BSA
</LI><LI> Incubation on a 37 °C water bath for 30 min
</LI><LI> Heat kill (80 °C for 20 min)
</LI></UL><P>Then the digested relA was ligeted with linearized pSB1A3 using the iGEM procedure for ligeting linearized plasmid backbones. The procedure can be found <A href="http://partsregistry.org/Help:Protocols/Linearized_Plasmid_Backbones" class="external text" rel="nofollow">here</A>.
</P><P>pSB1A3 containing relA was then transformed to competent DH5 α cells and incubated overnight.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel picture of the separated fragments resulting from the restriction cutting of the biobricks &quot;rrnBL in pSB1A3 plasmid&quot; and &quot;RBS+LacI+TERM&quot;. The squares indicate the fragments that where cut out of the gel</DIV></DIV></DIV><P>Plasmid Isolation:
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration (ng/µl)
</TH></TR><TR><TD>rrnBL in pSB1C3
</TD><TD>18,9
</TD></TR><TR><TD>ProL in pSB1C3
</TD><TD>41,6
</TD></TR><TR><TD>rrnBL in pSB1A3
</TD><TD>22,8
</TD></TR><TR><TD>ProL in pSB1A3
</TD><TD>29,6
</TD></TR><TR><TD>rrnBL+GFP
</TD><TD>47,5
</TD></TR><TR><TD>proL+GFP
</TD><TD>40,8
</TD></TR><TR><TD>rrnBL+MC
</TD><TD>85,3
</TD></TR><TR><TD>ProL+MC
</TD><TD>48,1
</TD></TR><TR><TD>RBS+LacI+Term
</TD><TD>53,9
</TD></TR><TR><TD>LacP+GFP
</TD><TD>26,6
</TD></TR><TR><TD>Lambda Pr
</TD><TD>38,4
</TD></TR></TBODY></TABLE><P>
Restriction Digestion:
Digested rrnBL in pSB1A3 with SpeI and PstI and the LacI construct (RBS+LacI-TERM) with Xba1 and PstI
The resulting fragments were separated using gel electrophoresis. rrnBL backbone and LacI insert were cut out of the gel and stored at -20
</P><P>Top10 cells: 
We would like to use top10 cells because it lacks a wild type LacI system. The top10 cells we have already contains a plasmid with Kan resistance. We have been incubating the top10 cells for a couple of days without selection so that some of the cells will loose the plasmid. Today we plated several dillutions of culture in order to get single colonies. The dilutions where 1:10, 1:100, 1:1000, 1:10000 and 1:100000. The plates were incubated over night.
</P><H2><SPAN class="mw-headline" id="Tuesday_12.2F7"> Tuesday 12/7 </SPAN></H2><P>Colonies from TOP10 plates were transferred to new plates with LA and LA + Kan to check wether the cells had lost their plasmid or not.
</P><P>Since the first GFP biobrick we tried did't work properly, we transformed two new variants of GFP. In addition, we also transformed the biobrick coding β-galactosidase, to be able to measure activities of different promotors by setting the promotor in front of the β-galactosidase biobrick, induce the promotor, destroy the cell and look at the absorbance when we add ONPG.
</P><TABLE border="1"><TBODY><TR><TH>Abbreviation
</TH><TH>Name
</TH><TH>Part number
</TH><TH>Resistance
</TH></TR><TR><TD>GFP 2
</TD><TD>GFP without degradation tag
</TD><TD>BBa_E0040
</TD><TD>Amp
</TD></TR><TR><TD>GFP ssra
</TD><TD>GFP with ssra degradation tag
</TD><TD>BBa_E0044
</TD><TD>Amp
</TD></TR><TR><TD>LacZ
</TD><TD>β-galactosidase
</TD><TD>BBa_I732019
</TD><TD>Amp
</TD></TR></TBODY></TABLE><P>
The fragments cut out of the gel on monday were extracted. The resulting consentrations were 1 ng/µL of rrnBL backbone and 1,6ng/µL of LacI insert.   
The parts were then ligated using 3,5µL backbone and 5µL insert. Then transformation and incubation on plate followed.
</P><H2><SPAN class="mw-headline" id="Wednesday_13.2F7"> Wednesday 13/7 </SPAN></H2><P>To see if our rrnB P1 + mCherry construct is working properly, it was made a dilution series from the cell culture, and the concentrations 10<SUP>-3</SUP>, 10<SUP>-4</SUP>, 10<SUP>-5</SUP> and 10<SUP>-6</SUP> was plated out on LA+Amp plates.
</P><P>The plasmids containing the pBad/AraC promotor, the relA gene and the rrnB P1 + mCherry construct was isolated using the miniprep kit from Promega. The concentrations of the plasmids was measured.
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration [ng/μl]
</TH></TR><TR><TD>pBad
</TD><TD>24,6
</TD></TR><TR><TD>RelA
</TD><TD>80,8
</TD></TR><TR><TD>P1+mC
</TD><TD>45,8
</TD></TR></TBODY></TABLE><P>
The P1+Lac construct, the LacZ gene and the two new GFP biobricks were inoculated.
</P><P>RelA-plasmid was digested with E and S to yield RelA insert (2234 bp)and was separated from its backbone (2155) and the larger piece was cut out with surgical precision. 
</P><P>RelA DNA was purified from the gel-slab and ligated into TERM backbone. The ligation mix was transformed into DH5-alpha cells.
</P><H2><SPAN class="mw-headline" id="Thursday_14.2F7"> Thursday 14/7 </SPAN></H2><P>Plasmids; pBAD/AraC, P1+Lac construct, the LacZ gene and the two new GFP biobricks were isolated, giving the following concentrations:
</P><TABLE border="2"><TBODY><TR><TD><B>BioBrick</B></TD><TD><B>Concentration ng/µL</B></TD></TR><TR><TD> pBAD/AraC</TD><TD>24,6
</TD></TR><TR><TD> lacZ</TD><TD>76,1
</TD></TR><TR><TD> GFP2</TD><TD>32,4
</TD></TR><TR><TD> GFP ssra</TD><TD>59,8
</TD></TR><TR><TD> P1+LacI</TD><TD>17,3
</TD></TR></TBODY></TABLE>cut iwth X+P
<P>
The lacZ construct was digested with E+X to yield a backbone for rrnB insert digested with E+S. The backbone digest was isolated from gel and saved.
</P><P>GFP2 and GFP ssra were digested with X+P, P1+lacI was digested with E+S, lacP+mCherry with E+X, and RBS+lacI+TERM with E+S. The fragments were separated and isolated from electrophoresis, and the DNA was purified from the gel-pieces.
</P><P>The concentrations of DNA after gel purification were as follows:
</P><H2><SPAN class="mw-headline" id="Friday_15.2F7"> Friday 15/7 </SPAN></H2><P>The LacZ backbone gel cut-out was isolated using QIAquick Gel Extraction Kit.
</P><P>Then, the following ligation mixes was made using the alternative ligation procedure on the <A href="https://2011.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocols page</A>.
</P><TABLE border="1"><TBODY><TR><TH>Insert
</TH><TH>Backbone
</TH><TH>Insert added [µl]
</TH><TH>Backbone added [µl]
</TH><TH>dH<SUB>2</SUB>O added [µl]
</TH></TR><TR><TD>P1+LacI
</TD><TD>LacP+mCherry
</TD><TD>1
</TD><TD>7.5
</TD><TD>0
</TD></TR><TR><TD>RBS+LacI+TERM
</TD><TD>LacP+mCherry
</TD><TD>4.5
</TD><TD>4
</TD><TD>0
</TD></TR><TR><TD>GFP2
</TD><TD>RBS
</TD><TD>6
</TD><TD>2.5
</TD><TD>0
</TD></TR><TR><TD>GFP ssra
</TD><TD>RBS
</TD><TD>6.5
</TD><TD>2
</TD><TD>0
</TD></TR><TR><TD>rrnB P1
</TD><TD>RBS+LacZ+TERM
</TD><TD>1
</TD><TD>3.6
</TD><TD>3.9
</TD></TR></TBODY></TABLE><P>
2 µl of each ligation mix was transformed to competent DH5α cells. The cells were plated out on LA + Amp plated and incubated on 37°C.
</P><H2><SPAN class="mw-headline" id="Saturday_16.2F7"> Saturday 16/7 </SPAN></H2><P>Transformed pBAD2+relA and religation to competent cells. 
</P><P>Plated out TOP10 cells in different dilutions (1:10 - 1:1000000).
</P><H2><SPAN class="mw-headline" id="Sunday_17.2F7"> Sunday 17/7 </SPAN></H2><P>Colonies from TOP10 plates were transferred to new plates with LA and LA + Kan to check wether the cells had lost their plasmid or not.
</P><P>Inoculated the following constructs; pBAD2+relA; P1+LacI+LacP+mCherry; RBS+LacI+TERM+LacP+mCherry; RBS+GFP2; RBS+GFPssra; rrnB P1+RBS+LacZ+TERM.
</P><H2><SPAN class="mw-headline" id="Monday_18.2F7"> Monday 18/7 </SPAN></H2><P>The following plasmids were isolated:
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel picture of the separated fragments resulting from the restriction cutting of the biobricks &quot;RBS+GFP2&quot; and &quot;RBS+GFPssrA&quot;. The squares indicate the fragments that where cut out of the gel</DIV></DIV></DIV><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration (ng/µl)
</TH></TR><TR><TD>RBS+GFP2
</TD><TD>59,0
</TD></TR><TR><TD>RBS+GFP ssrA
</TD><TD>61,6
</TD></TR><TR><TD>pBad+RelA
</TD><TD>55,0
</TD></TR><TR><TD>rrnB P1+RBS+LacZ+Term
</TD><TD>35,5
</TD></TR><TR><TD>RBS+LacI+TERM+LacP+MC
</TD><TD>38
</TD></TR><TR><TD>rrnB+RBS+LacI+TERM+LacP+MC
</TD><TD>22,6
</TD></TR></TBODY></TABLE><P>
RBS+GFP2 and RBS+GFP ssrA were cut with Xba1 and PstI.
the resulting fragments were separated on a gel and the insert fragments were cut out and extracted. 
</P><TABLE border="1"><TBODY><TR><TH>Insert
</TH><TH>consentration
</TH></TR><TR><TD>GFP2
</TD><TD>3,1
</TD></TR><TR><TD>GFP ssrA
</TD><TD>2,8
</TD></TR></TBODY></TABLE><P>The GFP's were ligated with promotors as follows:
</P><TABLE border="1"><TBODY><TR><TH>Insert
</TH><TH>µL
</TH><TH>Backbone
</TH><TH>µL
</TH></TR><TR><TD>GFP2
</TD><TD>7,3
</TD><TD>LacP
</TD><TD>1,2
</TD></TR><TR><TD>GFP2
</TD><TD>6,5
</TD><TD>Lambda Pr
</TD><TD>2,0
</TD></TR><TR><TD>GFP2
</TD><TD>3,5
</TD><TD>rrnB (P1)
</TD><TD>5
</TD></TR><TR><TD>GFP ssrA
</TD><TD>7,5
</TD><TD>LacP
</TD><TD>1,0
</TD></TR><TR><TD>GFP ssrA
</TD><TD>6,7
</TD><TD>Lambda Pr
</TD><TD>1,8
</TD></TR><TR><TD>GFP ssrA
</TD><TD>3,5
</TD><TD>rrnB (P1)
</TD><TD>5
</TD></TR></TBODY></TABLE><P>
Religations of the backbones were also made (Water instead of insert). 
</P><P>Different ways to stress the cells were discussed, and we decided that we could try making them stressed both by growing them in a nullmedium, a minimal a minimal medium, and in normal medium, but on higher and lower temperatures than 37°C
Nullmedium and minimal medium were prepared and autoclaved.
</P><P><B>Nullmedium:</B></P><UL><LI>0.9 % NaCl (9 g)
</LI><LI>1000 ml dH<SUB>2</SUB>O
</LI></UL><P><B>M9 minimal medium:</B></P><UL><LI>0.4 % glucose (4 g/l)
</LI><LI>2 mM MgSO<SUB>4</SUB>*7H<SUB>2</SUB>O (0.4922 g/l)
</LI><LI>0.1 mM CaCl<SUB>2</SUB> (0.0111 g/l)
</LI><LI>8.49 g/l Na<SUB>2</SUB>HPO<SUB>4</SUB>*2H<SUB>2</SUB>O
</LI><LI>3.84 g/l K<SUB>2</SUB>HPO<SUB>4</SUB></LI><LI>0.5 g/l NaCl
</LI><LI>1 g/l NH<SUB>4</SUB>Cl
</LI><LI>1000 ml dH<SUB>2</SUB>O
</LI><LI>pH was adjusted to 7.4 using HCl
</LI></UL><P>In addition, 2 L LA medium was prepared using the recipe on the <A href="https://2011.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocols page</A>, to make LA plates with kanamycin, ampicillin, and without antibiotics.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plasmid map of the complete construct. Promoters are shown in blue, and genes as inside arrows. Only single restriction sites are shown.</DIV></DIV></DIV><P>The RBS+LacI+TERM+LacP+mCherry inoculum was clearly red, indicating expression of mCherry and showing that it could be used as an indicator in liquid culture as well. 
</P><P>P1+LacI+LacP+mCherry was cut with three different enzyme pairs to test for the presence of the proper BioBricks, and size of the complete construct. The digestions are shown below.
</P><TABLE border="2"><TBODY><TR><TH>Enzyme-pair
</TH><TH>Expected fragments
</TH></TR><TR><TD>EcoRI, SpeI
</TD><TD>2056bp, 2771bp
</TD></TR><TR><TD>BsaAI, HpaI
</TD><TD>4083bp, 744bp
</TD></TR><TR><TD>BstBI, SpeI
</TD><TD>3284bp,934bp
</TD></TR></TBODY></TABLE><P>
Hopefully, the digestions will indicate a construct as shown in the plasmid map.
</P><H2><SPAN class="mw-headline" id="Tuesday_19.2F7">Tuesday 19/7</SPAN></H2><P>Prepared new M9 media, as the one yesterday was a complete FAIL. The media was prepared as shown in the <A href="https://2011.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocols</A>.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Gel picture of the separated fragments resulting from the restriction cutting of the pBad+RelA biobrick (EcoRI + XbaI), and the rrnB+LacI+Term+LacP+mC biobrick cutted with three different combination of restriction enzymes (EcoRI+SpeI, BsaAI+HpaI, BstBI+SpeI). The square indicate the fragment that was cut out</DIV></DIV></DIV><P>rrnB+LacZ and pBad+RelA was testcutted with different combinations of restriction enzymes:
</P><UL><LI> rrnB+LacZ:
</LI><LI> Cutted with HindIII and SacI → 4305 bp + 2615 bp
</LI><LI> Cutted with EcoRI and SpeI → 3754 bp (insert) + 3166 bp (backbone)
</LI><LI> Cutted with NdeI and SacI → 1017 bp + 5903 bp
</LI></UL><LI> pBad+RelA:
</LI><LI> Cutted with AflII and SexAI → 3201 bp + 2407 bp
</LI><LI> Cutted with EcoRI and XbaI → 5652 bp (backbone)
</LI><P>We then did gel electrophoresis for the five cut samples for rrnB+LacZ and pBad+RelA and the three samples of rrnB+LacI+TERM+LacP+mCherry that was cut yesterday with three different combination of enzymes. 
For the first gel, containing the three samples of rrnB+LacZ and the sample of pBad+RelA cut with AflII and SexAI, it was not possible to see any distinct difference between the sizes of the DNA pieces, but for the second gel, containing pBad+RelA cut with EcoRI and XbaI and the three samples of rrnB+LacI+TERM+LacP+mCherry, the gel looked better. pBad+RelA cut with EcoRI and XbaI gave a backbone we intend to use later, so this was cut out of the gel.
</P><H2><SPAN class="mw-headline" id="Wednesday_20.2F7"> Wednesday 20/7 </SPAN></H2><P>rrnB+LacZ and pBad+RelA was test-cut again with the same combination of enzymes that we did yesterday.
</P><P>We also transformed cells with the rrnB+lacZ construct that was test-cut, in case we need more DNA.
</P><P>We used gel electrophoresis to separate the test-cut plasmids. The pBad+relA-sample cut with EcoRI and XbaI gave a backbone, and was cut out of the gel, and the rrnB+lacZ-sample cut with EcoRI and SpeI gave an insert, and was also cutt out of the gel. 
The rrnB+lacZ-sample cutt with EcoRI and SpeI contained both an insert (3754 bp) and a backbone we are not going to use (3166 bp). Since these two DNA fragments were very similar in size, the gel electrophoresis had to proceed for about four hours.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Gel picture of the separated fragments resulting from the restriction cutting of the pBad+relA biobrick cut with AflII+SexAI and EcoRI+XbaI, respectively. The second sample, cut with EcoRI and XbaI, formed a backbone, and was cut out of the gel.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">The gel containing the three samples of rrnB+lacZ cut with three different combination of enzymes was run until it was possible to see that the sample cut with EcoRI and SpeI contained two pieces of DNA. The upper band (insert) was cut out of the gel.</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Gel picture of the separated fragments resulting from the restriction cutting of the rrnB+lacZ biobrick cut with HindIII+SacI, EcoRI+SpeI and NdeI+SacI. To be able to separate insert and backbone in the sample cut with EcoRI and SpeI, the gel was run longer (see picture to the right)</DIV></DIV></DIV></DIV><P>The six parallell RBS+lacI+TERM_lacP+mCherry plasmid and P1+lacI+TERM+lacP+mCherry were test-cut as follows.
</P><TABLE border="2"><TBODY><TR><TH>Enzymes
</TH><TH>Fragments
</TH></TR><TR><TD>EcoRI, NcoI
</TD><TD>2475 + 1851
</TD></TR><TR><TD>BsaAI, HpaI
</TD><TD>312 + 3894
</TD></TR><TR><TD>EcoRI, SpeI
</TD><TD>2056 + 2771
</TD></TR><TR><TD>BsaAI, HpaI
</TD><TD>4083 + 744
</TD></TR><TR><TD>BstBI, SpeI
</TD><TD>3284 + 934
</TD></TR></TBODY></TABLE><P>
lacP+GFP2 and lacP+GFPssra transformants were all glowing under UV-light. rrnBP1+GFP2 and rrnBP1+GFPssra were also glowing, but less bright than the lacP constructs. lambdaP constructs were not visibly glowing. 
Colonies from the rrnb P1 and lambdaP GFP-constructs were inoculated for plasmid isolation.
</P><H2><SPAN class="mw-headline" id="Thursday_21.2F7"> Thursday 21/7 </SPAN></H2><P><B>Stress-testing the final construct</B></P><P>Pre-cultures for biological stress-tests were inoculated in LB+amp at 37 C at 10:35. At 14:15 the cultures were pelleted down at 2000 rpm for 5 min and resuspended in remaining liquid. 
</P><P>One of the stress-experiments is to induce Heat-Shock. One culture will e grown at 42 C, and one will be grown at 37 C but will be heat-shocked at 42 C several times. The Heat-Shock cultures were inoculated at 14:35. Heat-shock was induced at 14:55 until 15:25.
</P><P>Two starvation stress-experiments were also performed, where the cells were resuspended and inoculated in M9 minimal media and in 0,9% NaCl. The incubated constructs included the P1-lacI+lacP-mCherry complete construct, and also the construct lacking the rrnB P1 promoter. Controls were also incubated in regular LB at 37C. 
The cultures were left over night in shaking incubators.
</P><P><B>GFP-construct plamids were isolated;</B></P><TABLE border="2"><TBODY><TR><TH>Construct
</TH><TH>Concentration (ng/µL)
</TH></TR><TR><TD>rrnB+GFP2 (1)
</TD><TD>23,3
</TD></TR><TR><TD>rrnB+GFP2 (2)
</TD><TD>24,1
</TD></TR><TR><TD>rrnB+GFPssra (1)
</TD><TD>19,7
</TD></TR><TR><TD>rrnB+GFPssra (2)
</TD><TD>23,2
</TD></TR><TR><TD>lamdbaP+GFP2
</TD><TD>27,0
</TD></TR><TR><TD>lamdaP+GFPssra
</TD><TD>31,4
</TD></TR></TBODY></TABLE><P>
The isolated plasmids were digested with E and S to yield inserts, to eventually put in front of our construct to be able to have green bacteria (in UV) as a positive indicator.
In addition the plasmids were digested with NcoI and StuI, to test for GFP insert (NcoI) and the ssra-tag (StuI). The fragments should be as follows:
</P><TABLE border="2"><TBODY><TR><TH>Construct
</TH><TH>Enzyme pair
</TH><TH>fragments
</TH></TR><TR><TD>rrnB P1+GFP2
</TD><TD>EcoRI, SpeI
</TD><TD>1242(insert) + 2132
</TD></TR><TR><TD>NcoI, StuI
</TD><TD>3374
</TD></TR><TR><TD>rrnB P1+GFPssra
</TD><TD>EcoRI, SpeI
</TD><TD>1281(insert) + 2132
</TD></TR><TR><TD>NcoI, StuI
</TD><TD>551 + 2682
</TD></TR><TR><TD>lambda Pr+GFP2
</TD><TD>EcoRI, SpeI
</TD><TD>798(insert) + 2056
</TD></TR><TR><TD>NcoI, StuI
</TD><TD>2854
</TD></TR><TR><TD>lambda Pr+GFPssra
</TD><TD>EcoRI, SpeI
</TD><TD>837(insert) + 2056
</TD></TR><TR><TD>NcoI, StuI
</TD><TD>2342 + 551
</TD></TR></TBODY></TABLE><P><B>rrnB P1 characterization construct:</B></P><P>The gel pieces that were cut out yesterday, containing rrnB+LacZ insert and pBad+RelA backbone, were purified using the QIAquick gel extraction kit, and the concentrations were determined:
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>pBad+RelA
</TD><TD>6.7
</TD></TR><TR><TD>rrnB+LacZ
</TD><TD>4.1
</TD></TR></TBODY></TABLE><P>
The pBad+RelA backbone and the rrnB+LacZ insert were ligated using the alternative ligation procedure on the <A href="https://2011.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocols page</A>. A religation mixture was also made. The following volumes were used:
</P><TABLE border="1"><TBODY><TR><TH>Backbone
</TH><TH>Insert
</TH><TH>Volume backbone [µl]
</TH><TH>Volume insert [µl]
</TH><TH>Volume water [µl]
</TH></TR><TR><TD>pBad+RelA
</TD><TD>rrnB+LacZ
</TD><TD>1.49
</TD><TD>7.01
</TD><TD>0
</TD></TR><TR><TD>pBad+RelA
</TD><TD> -
</TD><TD>1.49
</TD><TD> -
</TD><TD>7.01
</TD></TR></TBODY></TABLE><P>
The ligated plasmid and a religation was transformed to competent <I>E.coli</I> DH5α cells, and incubated on 37 °C.
</P><H2><SPAN class="mw-headline" id="Friday_22.2F7"> Friday 22/7 </SPAN></H2><TABLE border="2"><TBODY><TR><TH>Construct
</TH><TH>Enzyme pair
</TH><TH>Applied to well
</TH><TH>Fragment sizes (bp)
</TH></TR><TR><TD>rrnB P1+GFP2 1
</TD><TD>EcoRI, SpeI
</TD><TD>3
</TD><TD>1242(insert) + 2132
</TD></TR><TR><TD>NcoI, StuI
</TD><TD>2
</TD><TD>3374
</TD></TR><TR><TD>rrnB P1+GFP2 2
</TD><TD>EcoRI, SpeI
</TD><TD>5
</TD><TD>1242(insert) + 2132
</TD></TR><TR><TD>NcoI, StuI
</TD><TD>4
</TD><TD>3374
</TD></TR><TR><TD>rrnB P1+GFPssra 1
</TD><TD>EcoRI, SpeI
</TD><TD>7
</TD><TD>1281(insert) + 2132
</TD></TR><TR><TD>NcoI, StuI
</TD><TD>6
</TD><TD>551 + 2682
</TD></TR><TR><TD>rrnB P1+GFPssra 2
</TD><TD>EcoRI, SpeI
</TD><TD>11
</TD><TD>1281(insert) + 2132
</TD></TR><TR><TD>NcoI, StuI
</TD><TD>10
</TD><TD>551 + 2682
</TD></TR><TR><TD>lambda Pr+GFP2
</TD><TD>EcoRI, SpeI
</TD><TD>12
</TD><TD>798(insert) + 2056
</TD></TR><TR><TD>NcoI, StuI
</TD><TD>13
</TD><TD>2854
</TD></TR><TR><TD>lambda Pr+GFPssra
</TD><TD>EcoRI, SpeI
</TD><TD>14
</TD><TD>837(insert) + 2056
</TD></TR><TR><TD>NcoI, StuI
</TD><TD>15
</TD><TD>2342 + 551
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel with separated GFP-constructs. Wells are described in the table.</DIV></DIV></DIV><P>Top10 cells were plated out on agar plates were a grid was made. They were plated out on both LA plates and LA+Kan plates with equally looking grids, and will be investigated later to find out if any of the cells have lost their plasmid.
</P><P>Two samples of P1+GFP2, two samples of P1+GFPssra, one sample of λPr+GFP2 and one sample of λPr+GFPssra, all cut with E+S yielding inserts was run on gel, and DNA from the gel pieces was isolated using the QIAquick gel extraction kit. 
Concentration was measured:
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>P1+GFP2 1
</TD><TD>2.3
</TD></TR><TR><TD>P1+GFP2 2
</TD><TD>1.9
</TD></TR><TR><TD>P1+GFPssra 1
</TD><TD>3.3
</TD></TR><TR><TD>P1+GFPssra 2
</TD><TD>2.7
</TD></TR><TR><TD>λPr+GFP2
</TD><TD>3.0
</TD></TR><TR><TD>λPr+GFPssra
</TD><TD>4.3
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Saturday_23.2F7"> Saturday 23/7 </SPAN></H2><P>We have finally made E. coli TOP10 lose its plasmid!!
</P><P>Transformed the following constructs:
</P><TABLE border="1"><TBODY><TR><TH>Biobrick
</TH></TR><TR><TD>P1+GFP2 1 - RBS+LacI+TERM+LacP+RBS+mC
</TD></TR><TR><TD>P1+GFP2 2 - RBS+LacI+TERM+LacP+RBS+mC
</TD></TR><TR><TD>P1+GFPssra 1 - RBS+LacI+TERM+LacP+RBS+mC
</TD></TR><TR><TD>P1+GFPssra 2 - RBS+LacI+TERM+LacP+RBS+mC
</TD></TR><TR><TD>λPr+GFP2 - RBS+LacI+TERM+LacP+RBS+mC
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Sunday_24.2F7"> Sunday 24/7 </SPAN></H2><P>The transformations looked good, with many colonies on each plate. Negative control and religation control had 0 colonies. 4 colonies from each transformation were inoculated in 3 ml LB + Amp to prepare for plasmid isolation. 
</P><H2><SPAN class="mw-headline" id="Monday_25.2F7"> Monday 25/7 </SPAN></H2><P>The inoculated cells where isolated and test cut over night as described by the table below:
</P><TABLE border="2"><TBODY><TR><TH>Construct
</TH><TH>Sample
</TH><TH>Concentration [ng/µl]
</TH><TH>Enzyme pair
</TH></TR><TR><TD>P1+GFP2 + LacI-Construct
</TD><TD>1
</TD><TD>34,9
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR><TR><TD>2
</TD><TD>30,1
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR><TR><TD>3
</TD><TD>21,6
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR><TR><TD>4
</TD><TD>47,5
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR><TR><TD>P1+GFPssra + LacI-Construct
</TD><TD>1
</TD><TD>26,8
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR><TR><TD>2
</TD><TD>20,0
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR><TR><TD>3
</TD><TD>31,0
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR><TR><TD>4
</TD><TD>33,4
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR><TR><TD>P1 + LacI-Construct
</TD><TD>1
</TD><TD>64,8
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR><TR><TD>2
</TD><TD>56,5
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR><TR><TD>3
</TD><TD>59,5
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR><TR><TD>4
</TD><TD>55,1
</TD><TD>BsaAI + HpaI and EcoRI + SpeI
</TD></TR></TBODY></TABLE><P>
Colonies of TOP10 cells who had lost its plasmid (from saturday 23/7) were transferred to new plates with LA and LA + Kan, as well as LB medium.
</P><P>We found out that our RelA constructs were missing RBS :(  
We need to take some steps back and introduce an RBS to RelA+Term. This was done by cutting RelA+Term with X+P making it insert. Sinve RelA has a PstI site inside the gene, the plasmid needs to be partially digested with PstI. It was digested with X over night to prepare for partial digestion tomorrow.
</P><H2><SPAN class="mw-headline" id="Tuesday_26.2F7"> Tuesday 26/7 </SPAN></H2><P>1 ml TOP10 cells in LB medium grown over night were mixed with glycerol (400 µl, 60%) and stored in the freezer for use later.
</P><P>1 L M9 minimal medium were made.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel picture of the separated fragments resulting from the restriction cutting of the RelA+TERM plasmid. The first sample is cutted with XbaI+PstI, and the three next samples are cutted with XbaI, and then partially digested with PstI. The samples that was partially digested were made using a dilution series with decreasing concentration of PstI</DIV></DIV></DIV><P>The samples from yesterday was partially digested with PstI using a dilution series. The first sample was made according to the procedure for digestion cutting on the <A href="/Team:NTNU_Trondheim/Protocols" title="Team:NTNU Trondheim/Protocols"> protocols page</A>. The next two samples was also made according to this procedure, but with 10 µl less dH<SUB>2</SUB>O than usual. After adding PstI to the first sample, it was mixed, and 10 µl was transferred to the second sample. The second sample was mixed, and 10 µl transferred to the third sample.
The sample that was digested with XbaI and PstI was separated on gel alongside with the three samples from the dilution series from the partial digestion.
The gel picture didn't make sense, so we started concidering different reasons why the fragments were not as expected. We looked in the lab journal, to find out what had been done about RelA earlier, and we found out that when our PCR product was inserted in a plasmid, PstI and EcoRI was used, without partial digestion. 
It is likely that RelA was cut in two pieces when it was inserted in the plasmid, and that this is the reason why the fragments on the gel was smaller than expected.
If this assumption is correct, we have to start almost all over again, since it is most likely that the relA gene was destoyed in the first cutting step, where it was cloned into the plasmid.
</P><H2><SPAN class="mw-headline" id="Wednesday_27.2F7"> Wednesday 27/7 </SPAN></H2><P>RelA was amplified using PCR. The PCR product of our first amplification of RelA served as template DNA. Since the concentration of the template DNA was low, we amplified two different samples of it. In the first sample, we added 0.5 µl template DNA and 41.5 µl dH<SUB>2</SUB>O, and in the second sample, we used 5 µl of template DNA and 37 µl of dH<SUB>2</SUB>O. The rest of the volumes in the PCR mix corresponded to the standard PCR mix on the <A href="https://2011.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocols page</A>.
</P><P>Concentration was measured after the PCR:
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">The DNA fragments resulting from PCR of RelA</DIV></DIV></DIV><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>0.5 µl template DNA
</TD><TD>118.8
</TD></TR><TR><TD>5 µl template DNA
</TD><TD>103.4
</TD></TR></TBODY></TABLE><P>
To test if we had the right product, 5 µl of both samples was investigated using gel electrophoresis. 
The PCR-amplified RelA was digested with EcoRI and SpeI and purified. Concentration of the sample isolated from 0,5 µl template was 5,9 ng/µl while the one isolated from 5 µl was 4,3 ng/µl. The RelA insert cut with EcoRI and SpeI was ligated to Term backbone (cut with EcoRI and XbaI). 
The ligated plasmid was transformed to competent <I>E.Coli</I> DHα cells.
</P><P>Since the testcutting of our final construct did not look good we desided to try again using P1 as both insert and backbone, and to make the final construct with the lambda Pr promoter. We also made p1-MC and Lambda Pr-MC constructs in order to check the activity. 
When using P1 as insert we used the PCR product. The backbones and the cut PCR P1 was not run on gel but instead purified using the Qiagen PCR purification kit.
The LacI-LacP-MC insert were isolated from gel. Since the resulting insert and backbone after restriciton digestion  of LacI-LacP-MC (with XbaI and PstI) had almost the same length the backbone were cut with BglI as well to make two smaller backbone fragments. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">The resulting fragments after restriction digestion of LacI-LacP-MC with XbaI, PstI and BglI. The square indicates the insert fragment witch was cut out of the gel.</DIV></DIV></DIV><TABLE border="1"><TBODY><TR><TH>Part
</TH><TH>Consentration
</TH></TR><TR><TD>P1 Insert(PCR)
</TD><TD>1,6
</TD></TR><TR><TD>p1 backbone
</TD><TD>2,2
</TD></TR><TR><TD>lambda Pr backbone
</TD><TD>5,6
</TD></TR><TR><TD>LacI+LacP+MC backbone
</TD><TD>4,3
</TD></TR><TR><TD>LacI+LacP+MC insert
</TD><TD>1,7
</TD></TR><TR><TD>MC backbone
</TD><TD>6,1
</TD></TR></TBODY></TABLE><P>
The following ligations were made, and transformed. 
</P><TABLE border="1"><TBODY><TR><TH>Insert
</TH><TH>Backbone
</TH><TH>Insert added [µl]
</TH><TH>Backbone added [µl]
</TH></TR><TR><TD>P1
</TD><TD>LacI-LacP-MC
</TD><TD>6
</TD><TD>2,5
</TD></TR><TR><TD>P1
</TD><TD>MC
</TD><TD>7,5
</TD><TD>1
</TD></TR><TR><TD>LacI-LacP-MC
</TD><TD>P1
</TD><TD>5,5
</TD><TD>3
</TD></TR><TR><TD>MC
</TD><TD>P1
</TD><TD>4
</TD><TD>4,5
</TD></TR><TR><TD>LacI-LacP-MC
</TD><TD>Lambda Pr
</TD><TD>7,5
</TD><TD>1
</TD></TR><TR><TD>MC
</TD><TD>Lambda Pr
</TD><TD>6,5
</TD><TD>2
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Thursday_28.2F7"> Thursday 28/7 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">The resulting fragments after restriction digestion of LacI2 with SpeI and PstI and LacP-MC with XbaI and PstI. The square indicates the insert fragment witch was cut out of the gel.</DIV></DIV></DIV><P>In addition to the RBS-LacI-TERM we made ourself we orded another LacI part(BBa_K292006)containing RBS, LVA tag and a doubble terminator, from the iGem HQ.
The plasmid containing LacI2 was isolated giving the consentration 37,9 ng/µL. 
LacI2 was cut with SpeI and PstI making it the backbone, and LacP+MC was cut with XbaI and PstI making it the insert. 
</P><P>The digested plasmids were seperated on gel. 
The insert was cut out of the gel and extracted, and the backbone was purified with PCR purification (only 5 µL was run on the gel as a test) 
</P><TABLE border="1"><TBODY><TR><TH>Part
</TH><TH>consentration (ng/µL)
</TH></TR><TR><TD>LacI2
</TD><TD>5,0
</TD></TR><TR><TD>LacP-MC
</TD><TD>1,6
</TD></TR></TBODY></TABLE><P>
The parts were ligated
</P><TABLE border="1"><TBODY><TR><TH>Insert
</TH><TH>Bacbone
</TH><TH>Insert added [µL]
</TH><TH>Backbone added [µL]
</TH></TR><TR><TD>LacP-MC
</TD><TD>LacI2
</TD><TD>7
</TD><TD>1,5
</TD></TR></TBODY></TABLE><P>
Six colonies from the transformation of RelA+TERM (transformed yesterday) was inoculated to LB medium. 
</P><P>In order to test if the LacZ-gene works we also made some LA+Amp+Xgal plates, and transferred some colonies containing Xgal. These will be checked for colour tomorrow.
</P><P>The transformations looked good with many colonies on each plate, although there were many religations on the P1 plasmid. 6 colonies from the P1 plasmid + LacI-LacP-MC, 3 colonies from the Lambda Pr + LacI-Lacp-MC and 3 colonies from the P1 (PCR) + LacI-Lacp-MC were inoculated in 3 ml LB + Amp to prepare for plasmid isolation.
</P><P>We are going to make our TOP10 cells TSS competent. Transfection Storage Solution (TSS) was prepared (see  <A href="https://2011.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">recepies</A>).
More LA + Amp plates were also made.
</P><H2><SPAN class="mw-headline" id="Friday_29.2F7"> Friday 29/7 </SPAN></H2><P>Plasmids from P1+LacI+pLac+mCherry (where P1 was both backbone and insert in the ligation when the plasmid was made), λPr+LacI+pLac+mCherry and relA+TERM was isolated using the miniprep kit from Promega. Concentrations were measured:
</P><TABLE border="1"><TBODY><TR><TH>Plasmid
</TH><TH>Sample
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>P1+LacI+pLac+mCherry
</TD><TD>1 (backbone)
</TD><TD>18.7
</TD></TR><TR><TD>P1+LacI+pLac+mCherry
</TD><TD>2 (backbone)
</TD><TD>27.9
</TD></TR><TR><TD>P1+LacI+pLac+mCherry
</TD><TD>3 (backbone)
</TD><TD>20.1
</TD></TR><TR><TD>P1+LacI+pLac+mCherry
</TD><TD>4 (backbone)
</TD><TD>29.0
</TD></TR><TR><TD>P1+LacI+pLac+mCherry
</TD><TD>5 (backbone)
</TD><TD>33.0
</TD></TR><TR><TD>P1+LacI+pLac+mCherry
</TD><TD>6 (backbone)
</TD><TD>32.3
</TD></TR><TR><TD>P1+LacI+pLac+mCherry
</TD><TD>1 (insert)
</TD><TD>42.7
</TD></TR><TR><TD>P1+LacI+pLac+mCherry
</TD><TD>2 (insert)
</TD><TD>40.0
</TD></TR><TR><TD>P1+LacI+pLac+mCherry
</TD><TD>3 (insert)
</TD><TD>43.5
</TD></TR><TR><TD>λPr+LacI+pLac+mCherry
</TD><TD>1
</TD><TD>43.3
</TD></TR><TR><TD>λPr+LacI+pLac+mCherry
</TD><TD>2
</TD><TD>34.9
</TD></TR><TR><TD>λPr+LacI+pLac+mCherry
</TD><TD>3
</TD><TD>43.8
</TD></TR><TR><TD>RelA+TERM
</TD><TD>1
</TD><TD>88.1
</TD></TR><TR><TD>RelA+TERM
</TD><TD>2
</TD><TD>80.3
</TD></TR><TR><TD>RelA+TERM
</TD><TD>3
</TD><TD>65.6
</TD></TR><TR><TD>RelA+TERM
</TD><TD>4
</TD><TD>57.5
</TD></TR><TR><TD>RelA+TERM
</TD><TD>5
</TD><TD>76.1
</TD></TR><TR><TD>RelA+TERM
</TD><TD>6
</TD><TD>67.0
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fragments resulting from digestion of relA+TERM by XbaI and partial digestion by PstI. The frangments marked by the box was cut out of the gel and purified.</DIV></DIV></DIV><P>RelA+TERM sample 1 was used for digestion with XbaI and partial digestion with PstI. Three digestion mixtures were made. The first sample was made using 5.68 µl of DNA and 36.82 µl of dH<SUB>2</SUB>O, and the other two using 5.68 µl of DNA and 26.82 µl of dH<SUB>2</SUB>O. All three samples were digested with XbaI for 30 minutes. All the other ingredients were added according to the recipe on the <A href="/Team:NTNU_Trondheim/Protocols" title="Team:NTNU Trondheim/Protocols">protocols page</A>. 
Then, 1 µl of PstI was added to the first sample. This sample was mixed, and 10 µl transferred to the next sample. 10 µl of the second sample was transferred to the third sample. The three digestion samples were cut for 7 minutes, and the fragments resulting from the partial digestion were investigated using gel electroforesis.
The relA+TERM insert-fragment was identified and cut out of the gel, and then purified using the QIAquick gel extraction kit.
</P><P>The plasmids with P1-LacI-LacP-MC (made with P1 either as insert or backbone) and LambdaPr-LacI-LacP-MC was digested with enzymes in order to check that the plasmids were correct. On the resulting gel it looked like the samples with p1 as backbone had failed, but the other ones were correct.
</P><H2><SPAN class="mw-headline" id="Saturday_30.2F7"> Saturday 30/7 </SPAN></H2><P>Test cut the pSB1A3- and pSB1C3-plasmid with the rrnB P1 promoter insert, with BstBI.
</P><P>The concentration of gel-extracted RelA+Term was determined to 3,1 ng/µl. RelA + Term was ligated to 4,6 ng/µl RBS backbone using 4 µl  of RBS backbone and 11 µl of RelA+Term insert. After ligation for 30 min at 16 C 10 µl ligmix was transformed to DH5 alpha and plated out on LA + Amp. 
</P><H2><SPAN class="mw-headline" id="Sunday_31.2F7"> Sunday 31/7 </SPAN></H2><P>Transformed competent cells with a new ligation of rrnB P1 promoter and the pSB1C3- and pSB1A3-plasmid. Plated out.
</P><P>Inoculated 6 parallels of RBS+RelA+TERM.
</P><P>The following ligations were made and transformed:
</P><TABLE border="1"><TBODY><TR><TH>Insert
</TH><TH>Backbone
</TH><TH>Insert added [µl]
</TH><TH>Backbone added [µl]
</TH></TR><TR><TD>LacI2-LacP-MC
</TD><TD>pBad-AraC
</TD><TD>7,5
</TD><TD>1
</TD></TR><TR><TD>LacI-LacP-MC
</TD><TD>pBad-AraC
</TD><TD>7,5
</TD><TD>1
</TD></TR><TR><TD>MC
</TD><TD>pBad-AraC
</TD><TD>7,5
</TD><TD>1
</TD></TR><TR><TD>LacI2-LacP-MC
</TD><TD>LambdaPr
</TD><TD>7,5
</TD><TD>1
</TD></TR><TR><TD>P1 (PCR)
</TD><TD>LacI2-LacP-MC
</TD><TD>7,5
</TD><TD>1
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Monday_1.2F8"> Monday 1/8 </SPAN></H2><P>The six parallels of RBS+RelA+TERM inoculated yesterday was isolated using the promega miniprep kit. Concentrations were measured:
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>RBS+RelA+TERM 1
</TD><TD>82.0
</TD></TR><TR><TD>RBS+RelA+TERM 2
</TD><TD>35.9
</TD></TR><TR><TD>RBS+RelA+TERM 3
</TD><TD>74.6
</TD></TR><TR><TD>RBS+RelA+TERM 4
</TD><TD>24.6
</TD></TR><TR><TD>RBS+RelA+TERM 5
</TD><TD>58.9
</TD></TR><TR><TD>RBS+RelA+TERM 6
</TD><TD>53.8
</TD></TR></TBODY></TABLE><P>
All six samples were then cut with three different combination of enzymes. One digestion mixture contained EcoRI and XbaI. This digestion will yield a backbone for our next ligation, the ligation with pBad. The second digestion mixture contained PstI. This is to check if relA is ok this time, or if we have once again destroyed it. If relA is complete, this digestion should yield two fragments, due to the PstI restriction site inside the relA gene. If not, the plasmid should only be linearized. The third enzyme the samples were cut with was AhdI. The reason for this restriction cutting is to check that it really is relA+TERM that was ligated with RBS. The relA+TERM insert fragment that was cut out of gel on friday, was quite close to the pSB1AK3 backbone fragment. In the RBS+relA+TERM plasmid, pSB1A2 is backbone. In order to be sure that we didn't cut out any backbone as well when we cut out the relA+TERM insert, we digest with AhdI. Both plasmid backbones have an AhdI restriction site, so if everything is ok, we should get a linearized plasmid in this digestion. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fragments resulting from the restriction cutting of six parallels of RBS+RelA+TERM. The two squared fragments were cut out of the gel.</DIV></DIV></DIV><P>The 18 samples were separated using gel electrophoresis (see picture). The gel picture shows that sample 1 and sample 4 cut with EcoRI and XbaI probably consists of larger fragments than the other four samples do. From this we decided that sample 2, 3, 5 and 6 must all be religations with no insert. RelA+TERM was the insert of our last ligation, while RBS in pSB1A2 was backbone. We concluded that sample 2, 3, 5 and 6 doesn't contain RelA+TERM. This can also explain that when these four samples are cut with PstI, we get only a linearized plasmid. Sample 1 and 4 digested with PstI yields two fragments, which tells us that relA is intact. The backbones of sample 1 and sample 4 were cut out of the gel and purified using the QIAquick gel extraction kit.
Concentrations were measured as follows:
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>RBS+RelA+TERM 1
</TD><TD>5.9
</TD></TR><TR><TD>RBS+RelA+TERM 4
</TD><TD>5.7
</TD></TR></TBODY></TABLE><P><B>Top10</B></P><P>Two colonies of TOP10 that had lost their plasmids (1 and 3 on plate) were inoculated in parallels; 2x10 mL LB and 10mL LB+kan, for glycerol stock and to make TSS-competent cells, as well as checking that they have indeed lost their plasmids.
</P><P><B>Testing of system</B></P><P>To be able to test out if the system works, the construct will be set under control of inducible promoters. In addition to pBad/AraC the construct will also be tested in Pm/XylS-system. To do this the construct must first be cloned into a cloning vector (pUC128) and then to the Pm/XylS-plasmid (pVB20). The constructs (LacI+pLac+mc and LacI2+pLac+mc) digested with X+P was ligated to X+P digested pUC128. Relevant consentrations were estimated using NanoDrop:
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>pUC128 X+P backbone
</TD><TD>2.0
</TD></TR><TR><TD>LacI+pLac+mc insert
</TD><TD>7.8
</TD></TR><TR><TD>LacI2+pLac+mc insert
</TD><TD>7.5
</TD></TR><TR><TD>pVB20
</TD><TD>86.4
</TD></TR><TR><TD>pUC128
</TD><TD>26.3
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Tuesday_2.2F8"> Tuesday 2/8 </SPAN></H2><P>Plasmid Isolaton:
</P><TABLE border="1"><TBODY><TR><TH>Plasmid
</TH><TH>Consentration ng/µL
</TH><TH>Colour of liquid culture
</TH></TR><TR><TD>Lambda Pr-LacI2-LacP-MC 1
</TD><TD>44,7
</TD><TD>not red
</TD></TR><TR><TD>Lambda Pr-LacI2-LacP-MC 2
</TD><TD>32,7
</TD><TD>weak red
</TD></TR><TR><TD>P1-LacI2-LacP-MC 1
</TD><TD>26,4
</TD><TD>not red
</TD></TR><TR><TD>P1-LacI2-LacP-MC 2
</TD><TD>50,9
</TD><TD>weak red
</TD></TR><TR><TD>P1-LacI2-LacP-MC 3
</TD><TD>157
</TD><TD>red
</TD></TR><TR><TD>P1-LacI2-LacP-MC 4
</TD><TD>60,3
</TD><TD>weak red
</TD></TR><TR><TD>pBad-AraC-LacI-LacP-MC 1
</TD><TD>58,9
</TD><TD>not red
</TD></TR><TR><TD>P1-LacI2-LacP-MC 2
</TD><TD>23,0
</TD><TD>not red
</TD></TR><TR><TD>P1-LacI2-LacP-MC 3
</TD><TD>49,1
</TD><TD>not red
</TD></TR><TR><TD>pBad-AraC-MC 1
</TD><TD>24,9
</TD><TD>not red
</TD></TR><TR><TD>pBad-AraC-MC 2
</TD><TD>23,2
</TD><TD>not red
</TD></TR><TR><TD>pBad-AraC-MC 3
</TD><TD>22,4
</TD><TD>not red
</TD></TR><TR><TD>pBad-AraC-MC 4
</TD><TD>25,1
</TD><TD>not red
</TD></TR><TR><TD>pBad-AraC-LacI2-LacP-MC 1
</TD><TD>76,1
</TD><TD>strong red
</TD></TR><TR><TD>pBad-AraC-LacI2-LacP-MC 2
</TD><TD>74,9
</TD><TD>strong red
</TD></TR></TBODY></TABLE><P>
More LB medium was made.
</P><P>Both sample 1 and sample 4 of RBS+RelA+TERM backbone was ligated to pBad insert (already cut with EcoRI and SpeI, concentration; 3.0 ng/µl) according to the procedure on the <A href="/Team:NTNU_Trondheim/Protocols" title="Team:NTNU Trondheim/Protocols"> protocols page</A>. One religation test was made from each backbone sample. Volumes used in the ligations and religations is given below:
</P><TABLE border="1"><TBODY><TR><TH>Insert
</TH><TH>Backbone
</TH><TH>Backbone added [µl]
</TH><TH>Insert added [µl]
</TH><TH>dH<SUB>2</SUB>O added [µl]
</TH></TR><TR><TD>RBS+RelA+TERM 1
</TD><TD>pBad
</TD><TD>1
</TD><TD>7.5
</TD><TD> -
</TD></TR><TR><TD>RBS+RelA+TERM 1
</TD><TD> -
</TD><TD>1.69
</TD><TD> -
</TD><TD>6.81
</TD></TR><TR><TD>RBS+RelA+TERM 4
</TD><TD>pBad
</TD><TD>1
</TD><TD>7.5
</TD><TD> -
</TD></TR><TR><TD>RBS+RelA+TERM 4
</TD><TD> -
</TD><TD>1.75
</TD><TD> -
</TD><TD>6.75
</TD></TR></TBODY></TABLE><P>
The two ligations and religations were transformed to competent <I>E.Coli</I> DH5α cells, along with LacI+pLac+mCherry and LacI2+pLac+mCherry ligated yesterday. The transformed cells were incubated on 37 °C.
</P><P>TOP10 cells that were inoculated yesterday were made competent by adding TSS followed by snap freezing. 
</P><P>The PCR product of the rrnB P1 promoter was cut with EcoRI and PstI and ligated in to the linearized plasmids pSB1A3 and pSB1C3.
</P><H2><SPAN class="mw-headline" id="Wednesday_3.2F8"> Wednesday 3/8 </SPAN></H2><P>The two correct pBad+RBS+RelA+TERM plasmids transformed yesterday was inoculated to LB medium containing ampicillin and incubated on 37 °C.
</P><P>The following constructs were transformed in to Top10 cells. 
</P><TABLE border="1"><TBODY><TR><TH>construct
</TH><TH>Colony number
</TH></TR><TR><TD>pBad-LacI2-LacP-MC
</TD><TD>1
</TD></TR><TR><TD>P1-LacI2-LacP-MC
</TD><TD>1 and 3
</TD></TR><TR><TD>pBad-MC
</TD><TD>1
</TD></TR><TR><TD>P1-LacI-LacP-MC
</TD><TD>1
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Thursday_4.2F8"> Thursday 4/8 </SPAN></H2><P>Plasmids from the eight samples of pBad+RBS+RelA+TERM inoculated yesterday were isolated using miniprep kit from Promega. Concentrations were measured:
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>pBad+RBS+RelA+TERM 1(1)
</TD><TD>22.2
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 2(1)
</TD><TD>19.2
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 3(1)
</TD><TD>22.3
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 4(1)
</TD><TD>20.8
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 1(4)
</TD><TD>19.3
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 2(4)
</TD><TD>22.1
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 3(4)
</TD><TD>19.1
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 4(4)
</TD><TD>21.3
</TD></TR></TBODY></TABLE><P>
All eight pBad+RBS+RelA+TERM samples were digested with two combinations of enzymes; EcoRI + XbaI, and AflII + SexAI. In addition, rrnB+RBS+LacZ+TERM was digested with BstBI + HindIII.
The fragments resulting from the digestion didn't make sense, and we wanted to be sure we have the right backbone before we move on and ligate it with the rrnB+RBS+LacZ+TERM insert. So instead of isolating DNA from the gel piece, we inoculated eight new colonies and incubated them on 37°C overnight.
</P><P>Inoculated TOP10 cultures with the following constructs: P1-LacI-LacP-mC; P1-LacI(2)-LacP-mC; pBAD-mC; and pBAD-LacI(2)-LacP-mC.
</P><P>Inoculated five colonies of our pSB1A3 and pSB1C3 plasmids with P1-promoter.
</P><H2><SPAN class="mw-headline" id="Friday_5.2F8"> Friday 5/8 </SPAN></H2><P>Plasmids from the eight new colonies of pBad+RBS+RelA+TERM was isolated using miniprep kit from Promega. Concentrations were measured:
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>pBad+RBS+RelA+TERM 1(1)
</TD><TD>15.2
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 2(1)
</TD><TD>20.6
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 3(1)
</TD><TD>22.5
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 4(1)
</TD><TD>26.1
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 1(4)
</TD><TD>21.1</TD></TR><TR><TD>pBad+RBS+RelA+TERM 2(4)
</TD><TD>20.4
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 3(4)
</TD><TD>19.6
</TD></TR><TR><TD>pBad+RBS+RelA+TERM 4(4)
</TD><TD>23.0
</TD></TR></TBODY></TABLE><P>
Sample 4(1) and sample 1(4) were cutted with three different enzyme combinations according to the procedure on the <A href="/Team:NTNU_Trondheim/Protocols" title="Team:NTNU Trondheim/Protocols">protocols page</A>, yielding six digestion mixtures. Both samples were cut with EcoRI and XbaI, yielding backbones (5674 bp), AflII and SexAI, which should give who fragments on 2468 bp and 3221 bp, and BsaAI. RelA contains three restriction sites for BsaAI, so here we should get three fragments on respectively 354 bp, 849 bp and 4486 bp.  
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel picture of the fragments resulting from pBad+RBS+RelA+TERM digested with EcoRI and XbaI (well 2 and 3) and AflII and SexAI (well 4 and 5). The fragment resulting from digestion digest of sample 1(4) with EcoRI and XbaI was cut out of the gel.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel picture of the fragments resulting from pBad+RBS+RelA+TERM digested with BsaAI (well 2 and 3) and uncutted plasmids (well 4 and 5). In this picture it is possible to see two fragments in well 3, which is sample 1(4) digested with BsaAI.</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel picture of the fragments resulting from pBad+RBS+RelA+TERM digested with BsaAI (well 2 and 3) and uncutted plasmids (well 4 and 5). The digestion with BsaAI should have yielded three fragments. In this picture, the fragments are difficult to see.</DIV></DIV></DIV></DIV><P>
The fragments resulting from the digestion cutting of pBad+RBS+RelA+TERM sample 4(1) and sample 1(4) should have been similar, and unfortunately, they are not. The digestions of sample 1(4) didn't look so bad, except from the fact that the digestion with BsaAI should have resulted in three fragments instead of just two. Also, the fragment that that was digested with EcoRI and XbaI, and is supposed to be backbone is to large, according to the ladder. But nevertheless, it was cut out from the gel, in case we decide to use it. 
</P><P>
Plasmids from 5 colonies each of pSB1A3 with rrnB P1 and pSB1C3 with rrnB P1 was isolated using miniprep kit from Promega. Concentrations were measured:
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>pSB1A3 with rrnB P1 1
</TD><TD>26.8
</TD></TR><TR><TD>pSB1A3 with rrnB P1 2
</TD><TD>19.0
</TD></TR><TR><TD>pSB1A3 with rrnB P1 3
</TD><TD>33.7
</TD></TR><TR><TD>pSB1A3 with rrnB P1 4
</TD><TD>28.6
</TD></TR><TR><TD>pSB1A3 with rrnB P1 5
</TD><TD>23.1
</TD></TR><TR><TD>pSB1C3 with rrnB P1 1
</TD><TD>20.9
</TD></TR><TR><TD>pSB1C3 with rrnB P1 2
</TD><TD>39.6
</TD></TR><TR><TD>pSB1C3 with rrnB P1 3
</TD><TD>27.3
</TD></TR><TR><TD>pSB1C3 with rrnB P1 4
</TD><TD>23.6
</TD></TR><TR><TD>pSB1C3 with rrnB P1 5
</TD><TD>26.6
</TD></TR></TBODY></TABLE><P>
Tried to pick out red colonies from pUC128+LacI-Mc constructs, but they are not well separated so they were plated out on a new plate. In addition a new ligation was done, in case the plasmid cannot be isolated from last transformation.
</P><P>
Our construct where inoculated in M9 and LB medium to check how much the bacterial growth where affected. 
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>Medium
</TH><TH>Absorbance start
</TH><TH>Absorbance after X hours
</TH></TR><TR><TD>P1-LacI-LacP-mC 1
</TD><TD>M9+Amp
</TD><TD>0,0610
</TD></TR><TR><TD>P1-LacI-LacP-mC 2
</TD><TD>M9+Amp
</TD><TD>0,0894
</TD></TR><TR><TD>P1-LacI-LacP-mC 3
</TD><TD>M9+Amp
</TD><TD>0,0695
</TD></TR><TR><TD>P1-LacI-LacP-mC 1
</TD><TD>LB+Amp
</TD><TD>0,1142
</TD></TR><TR><TD>P1-LacI-LacP-mC 2
</TD><TD>LB+Amp
</TD><TD>0,1159
</TD></TR><TR><TD>P1-LacI-LacP-mC 3
</TD><TD>LB+Amp
</TD><TD>0,1179
</TD></TR></TBODY></TABLE><P><B>pBad/AraC constructs</B></P><P>The pBAD / AraC constructs were dilluted 1:100 in 5 mL arabinose + amp-containing LB at 11.00. Some samples were acidentally washed. These were incubated at 12.00. The concentrations of arabinose were 0%, 0.05%, 0.5% and 1%. 
</P><P>The aim of the tests is to show expression of mCherry from the pBad/AraC promoter, and also to show that the expression of mCherry should be increasingly inhibited by increasing expression of lacI from the pBad/AraC promoter.
</P><H2><SPAN class="mw-headline" id="Monday_8.2F8"> Monday 8/8 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">LacI2 plasmid cut with EcoR1 + PstI, BglI + BclI, BglI + EcoRV, BglI + BanII</DIV></DIV></DIV><P>Since test-cutting of the LacI2 constructs did not look that good, we preformed a test-cutting of the plasmid contoining the LacI2 biobrick that were sent to us. 
The plsamid was cut with EcoRI and PstI in order to look at the insert length, the expected fragments: 2053 and 1303. In adittion the plasmid was cut with  BglI + BclI, BglI + EcoRV, BglI + BanII. BglI cuts in the pSB1A2 plasmids while the rest cuts inside the LacI2 brick. The expected lengths: 2032 + 1324, 1760 + 1596, 1835 + 1521 bp. 
</P><P>As we can see on the resulting gel none of the restriction sites inn the LacI2 fragment appear to be present. The LacI2 insert looks smaller than it's supposed to be.
</P><P>Ligated pBad backbone (cons 7,6 ng/µL) and LacI-LacP-MC insert (from parallell 5 with cons 3,9 ng/µL) at 4 C over night. 
</P><P><B>LacI pLac mCherry in pUC128</B></P><P>Red colonies was isolated from transformation plates and colony was inoculated in LB+amp for plasmid isolation. No red colonies could be observed from the LacI2 variant.
</P><P><B>Testing of construct</B></P><P>The construct and variants of it was prepared for biological experiment. The experiment was run at 37 C over night by inoculating the cultures from plates.
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>LB Amp
</TH><TH>M9 Amp
</TH><TH>M9 Amp + IPTG
</TH></TR><TR><TD>Nonfluorescent culture
</TD><TD>X
</TD><TD>X
</TD></TR><TR><TD>Top10 with superbrick LacI version
</TD><TD>X
</TD><TD>X
</TD><TD>X
</TD></TR><TR><TD>Top10 with superbrick LacI2 version
</TD><TD>X
</TD><TD>X
</TD><TD>X
</TD></TR><TR><TD>DH5-alpha with superbrick LacI version
</TD><TD>X
</TD><TD>X
</TD></TR><TR><TD>DH5-alpha superbrick without P1
</TD><TD>X
</TD><TD>X
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Tuesday_9.2F8"> Tuesday 9/8 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">The Puc128 plasmid with LacI-LacP-MC cut with EcoRI. The square indicates the LacI-LacP-MC insert</DIV></DIV></DIV><P>LacI+LacP+MC in plasmid Puc128 was isolated (consentration 46,4 ng/µL)
In order to test the LacI-LacP-MC part we want to insert it in the pwb20 plasmid.
the pwb20 plasmid was opened using EcoRI and then treated with CIP to avoid religation. 
To retrive the LacI-LacP-MC insert the Puc128 plasmid was cut with EcoRI and the fragments was separated
on gel. The insert fragment was cut out and extracted from the gel (consentration 8,5 ng/µL) 
The pwb20 plasmid was purified using PCR purification kit (consentration 3,0 ng/µL) 
</P><P>Transformed pBad-AraC-LacI-LacP-MC (5)
</P><P>The test-cultures from yesterday were inspected. There was no growth in M9 minimal media. <B>This was due to direct inoculation from plates.</B> The M9 flasks were therefore inoculated from LB to M9. 
</P><P>pBad+RelA gel-piece was purified ( Concentration: 1.9 ng/µl)
</P><H2><SPAN class="mw-headline" id="Wednesday_10.2F8">Wednesday 10/8</SPAN></H2><P>Fluorescence was measured from the different samples.
</P><P>To do a new test, the construct with and without the rrnB P1 promoter in DH5 alpha was inoculated over night in 3mL LB + amp in a 13 mL tube.  
</P><H2><SPAN class="mw-headline" id="Thursday_11.2F8"> Thursday 11/8 </SPAN></H2><P>The inoculums with and without the rrnB P1 promoter were spun down and resuspended in 2 mL M9 minimal medium. 
Both were inoculated in shake flasks in triplicates; 100 µL in 10 mL LB, 10mL LB + 1mM IPTG, 10 mL M9 and 10 mL M9 + IPTG. All media were supplied with 200 µg/mL ampicillin. 
</P><P>The OD600 and fluorescence (ex 584 nm, em 620 nm) at time=0 were measured, and the flasks were incubated in 37C shaking incubator at 12.15.
</P><P>At 15.45 the OD600 and fluorescence were measured again.
</P><H2><SPAN class="mw-headline" id="Friday_12.2F8">Friday 12/8</SPAN></H2><P>The fluorescence and OD600 of the test-flasks was measured at 12.00.
</P><P>The constructs to be sequenced (P1-lacI-lacP-mCherry, P1-lacI2-lacP-mCherry, lacI-lacP-mCherry, lacI2-lacP-mCherry, RBS-relA-TERM, P1-lacZ-pBAD-relA) were re-streaked to give fresh colonies for plasmid isolation. 
</P><H2><SPAN class="mw-headline" id="Saturday_13.2F8">Saturday 13/8</SPAN></H2><P>Colonies were inoculated in 5 mL LB + amp for plasmid isolation to the sequencing reactions; 
3 x P1-lacI-lacP-mCherry, 3 x P1-lacI2-lacP-mCherry, 3 x lacI-lacP-mCherry, 3 x lacI2-lacP-mCherry, 3 x RBS-relA-TERM, 6 x P1-lacZ-pBAD-relA
</P><H2><SPAN class="mw-headline" id="Sunday_14.2F8">Sunday 14/8</SPAN></H2><P>Isolated plasmids from inoculums, and also Pm XylS- BioBrick plasmids 1-6.
</P><H2><SPAN class="mw-headline" id="Monday_15.2F8">Monday 15/8</SPAN></H2><P>Concentrations were measured for the six samples of rrnB+LacI+pLac+mCherry in Pm XylS:
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>1
</TD><TD>82.8
</TD></TR><TR><TD>2
</TD><TD>21.8
</TD></TR><TR><TD>3
</TD><TD>37.3
</TD></TR><TR><TD>4
</TD><TD>21.8
</TD></TR><TR><TD>5
</TD><TD>23.4
</TD></TR><TR><TD>6
</TD><TD>35.1
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fragments resulting from restriction digest of rrnB+LacI+pLac+mCherry in Pm XylS with AccI (well 2 - 7), PspOMI + NdeI (well 9 - 11 and 14 - 16) and DrdI + SexAI (well 18 - 23)</DIV></DIV></DIV><P>All six samples were cut with three different combinations of restriction enzymes to make sure that the insert is inserted the right way into the plasmid. To make sure that the insert is inserted in Pm XylS and not in pUC128, all six samples were cut with AccI. Pm XylS contains two restriction sites for AccI, while pUC128 only contains one. If the insert still is in pUC128, this restriction cutting should yield a linearized plasmid, while insert in Pm XylS should give two fragments on 2188 bp and 6696 bp, respectively.
To check which way the insert is inserted, we did two restriction digestions. One with PspOMI and NdeI, and one with DrdI and SexAI. If the insert is inserted the right way this digestions should yield two fragments on 624 bp and 8260 bp for the digestion with PspOMI and NdeI and fragments on 1027 bp and 7857 bp for the digestion with DrdI and SexAI.
If insert is inserted the wrong way we expect two fragments on 1689 bp and 7195 bp for the digestion with PspOMI and NdeI, and fragments on 2662 bp and 6222 bp for the digestion with DrdI and SexAI.
</P><P>The eighteen samples from the restriction digest were investigated using gel electrophoresis. 
In the digest with AccI we expected two fragments. This seems to be the case for all samples, except sample 1. In the digest with PspOMI and NdeI sample 3 and sample 5 looks correct, except the fact that the upper band of sample 3 looks like two bands. In the digest with DrdI and SexAI all samples except sample one is showing only one band. This indicates that only one of the enzymes has been cutting. One possible reason for this is that the site for the other enzyme might be damaged.
</P><H2><SPAN class="mw-headline" id="Tuesday_16.2F8">Tuesday 16/8</SPAN></H2><P>A new stress test was set up according to the following table:
</P><TABLE border="1"><TBODY><TR><TH>Number of parallels
</TH><TH>With (+) or without (-) promotor
</TH><TH>Type of medium
</TH></TR><TR><TD>3
</TD><TD> +
</TD><TD>LB
</TD></TR><TR><TD>3
</TD><TD> +
</TD><TD>LB + IPTG
</TD></TR><TR><TD>3
</TD><TD> -
</TD><TD>LB
</TD></TR><TR><TD>3
</TD><TD> -
</TD><TD>LB + IPTG
</TD></TR><TR><TD>3
</TD><TD> +
</TD><TD>M9
</TD></TR><TR><TD>3
</TD><TD> +
</TD><TD>M9 + IPTG
</TD></TR><TR><TD>3
</TD><TD> -
</TD><TD>M9
</TD></TR><TR><TD>3
</TD><TD> -
</TD><TD>M9 + IPTG
</TD></TR></TBODY></TABLE><P>Absorbance and OD<SUB>600</SUB> were measured at 0 hrs, 3.5 hrs and 24 hrs.
</P><P><B>rrnB P1 characterization:</B></P><P>rrnB-lacZ, pBAD-relA, and rrnB-lacZ-pBAD-relA constructs were transformed into TOP10 cells to prepare for beta-galactosidase assay. 
</P><P>The point being to demonstrate (hopefully) that the rrnB promoter gets downregulated when relA is overproduced. I.e. the rrnB-lacZ-pBAD-AraC cells should produce less lacZ when the cells get induced by arabinose.
</P><H2><SPAN class="mw-headline" id="Wednesday_17.2F8">Wednesday 17/8</SPAN></H2><P>We have started flow cytometry experiments to characterize the properties of our system.
</P><H2><SPAN class="mw-headline" id="Thursday_18.2F8">Thursday 18/8</SPAN></H2><P>rrnB-lacZ, pBAD-relA, and rrnB-lacZ-pBAD-relA TOP10 pre-cultures were inoculated for the beta-galactosidase assay.
</P><H2><SPAN class="mw-headline" id="Thursday_8.2F9">Thursday 8/9</SPAN></H2><P>In order for us to send in our biobricks to the registry we have to put them in the official shiping plasmid pSB1C3. The parts we want to send in are RBS+LacI+TERM+pLac+RBS+mCherry+TERM (BBa_K639000), RelA (BBa_K639001), rrnB P1 (BBa_K639002) and rrnB+RBS+LacI+TERM+pLac+RBS+mCherry+TERM (BBa_K639002). 
The lineraized pSB1C3 nedds to be digested with EcoRI and PstI prior to use. Since the RelA have a PstI restriction site in the middel of the sequence we retrived the pSB1C3 backbone from the rrnb P1 in the pSB1C3 plasmid that we already had made. 
For the other parts we usde the linerazied plasmid provided form the iGem HQ. 
</P><P>The following digestions were made: 
</P><TABLE border="1"><TBODY><TR><TH>BioBrick
</TH><TH>enzymes
</TH><TH>DNA (µL)
</TH><TH>dH20 (µL)
</TH></TR><TR><TD>RBS+LacI+TERM+pLac+RBS+mCherry+TERM
</TD><TD>EcoRI, PstI, BglI
</TD><TD>18,5
</TD><TD>24
</TD></TR><TR><TD>rrnB+RBS+LacI+TERM+pLac+RBS+mCherry+TERM
</TD><TD>EcoRI, PstI, BglI
</TD><TD>13,2
</TD><TD>29,3
</TD></TR><TR><TD>Lineriazed pSB1C3
</TD><TD>EcoRI, PstI
</TD><TD>20
</TD><TD>22,5
</TD></TR><TR><TD>rrnB p1 in psb1C3
</TD><TD>EcoRI, SpeI
</TD><TD>37,3
</TD><TD>5,2
</TD></TR><TR><TD>RelA
</TD><TD>EcoRI, SpeI, BglI
</TD><TD>12
</TD><TD>30,5
</TD></TR></TBODY></TABLE><P>Because of similar insert and backbone lengths some of the plasmids were cut with BglI wich digests the backbone.
The samples were digested over night. 
</P><H2><SPAN class="mw-headline" id="Friday_9.2F9">Friday 9/9</SPAN></H2><P>The digested samples from yeasterday were separated on gel and the rrnB+RBS+LacI+TERM+pLac+RBS+mCherry+TERM, RBS+LacI+TERM+pLac+RBS+mCherry+TERM and RelA pieces were cut out in addition to the pSB1C3 backbone separated from the rrnB P1 promoter. The gel pieces were extracted and the lineriazed pSB1C3 were PCR purified. We ended up with the following conestrations:
</P><TABLE border="1"><TBODY><TR><TH>BioBrick
</TH><TH>Consentration (ng/µL)
</TH></TR><TR><TD>RBS+LacI+TERM+pLac+RBS+mCherry+TERM
</TD><TD>2,1
</TD></TR><TR><TD>rrnB+RBS+LacI+TERM+pLac+RBS+mCherry+TERM
</TD><TD>4,4
</TD></TR><TR><TD>Lineriazed pSB1C3
</TD><TD>1,6
</TD></TR><TR><TD>pSB1C3 backbone (cut with EcoR1 and SpeI)
</TD><TD>2,9
</TD></TR><TR><TD>RelA
</TD><TD>3,4
</TD></TR></TBODY></TABLE><P>The following ligations were made:
</P><TABLE border="1"><TBODY><TR><TH>backbone
</TH><TH>µL
</TH><TH>Insert
</TH><TH>µL
</TH></TR><TR><TD>pSB1C3 linerized
</TD><TD>5
</TD><TD>RBS+LacI+TERM+pLac+RBS+mCherry+TERM
</TD><TD>3,5
</TD></TR><TR><TD>pSB1C3 linerized
</TD><TD>6
</TD><TD>rrnB+RBS+LacI+TERM+pLac+RBS+mCherry+TERM
</TD><TD>2,5
</TD></TR><TR><TD>pSB1C3 with EcorRI and SpeI overhangs
</TD><TD>4
</TD><TD>RelA
</TD><TD>4,5
</TD></TR></TBODY></TABLE><P>In addition religatons of the backbones were mader. 
</P><P>The ligation mixes were put incubated at 4 degrees celsius over the weekend.
</P><H2><SPAN class="mw-headline" id="Monday_12.2F9">Monday 12/9</SPAN></H2><P>Transformed rrnB+RBS+LacI+TERM+pLac+RBS+mCherry+TERM in pSB1C3, RBS+LacI+TERM+pLac+RBS+mCherry+TERM in pSB1C3, relA in pSB1C3 and rrnB P1 in pSB1C3 to competent <I>E.coli DH5α</I> cells. Religations of pSB1C3 and linearized pSB1C3 were also transformed.
</P><H2><SPAN class="mw-headline" id="Tuesday_13.2F9">Tuesday 13/9</SPAN></H2><P>rrnB+RBS+LacI+TERM+pLac+RBS+mCherry+TERM in pSB1C3, RBS+LacI+TERM+pLac+RBS+mCherry+TERM in pSB1C3, relA in pSB1C3 and rrnB P1 in pSB1C3 were inoculated in 3 ml LB + chloramphenicol to prepare for isolation of plasmids.
</P><H2><SPAN class="mw-headline" id="Wednesday_14.2F9">Wednesday 14/9</SPAN></H2><P>Two parallels of rrnB P1+RBS+LacI+TERM+pLac+RBS+mCherry+TERM, RBS+LacI+TERM+pLac+RBS+mCherry+TERM, relA and rrnB was isolated using the miniprep kit from Promega. The two parallels of each plasmid was eluted in the same tube in the end of the isolation. 50 µl of TE buffer was used in each elution, resulting in four 100 µl plasmid samples.
</P><P>Concentrations were measured: 
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>rrnB+RBS+LacI+TERM+pLac+RBS+mCherry+TERM
</TD><TD>120.1
</TD></TR><TR><TD>RBS+LacI+TERM+pLac+RBS+mCherry+TERM
</TD><TD>130.1
</TD></TR><TR><TD>relA
</TD><TD>127.4
</TD></TR><TR><TD>rrnB
</TD><TD>83.7
</TD></TR></TBODY></TABLE><P>After isolation, the plasmid was both test cutted and prepared for shipping to the Registry of standard biological parts. 
</P><P><I>Test cutting:</I></P><P>Since the cells containing the rrnB+RBS+LacI+TERM+pLac+RBS+mCherry+TERM and RBS+LacI+TERM+pLac+RBS+mCherry+TERM plasmids was red when the isolation started, we decided not to test cut these two plasmids, since the cells would not be red unless their plasmids is exactly as we think they are. 
</P><P>RelA was cut with PstI (expecting two fragments on 888 and 3417 bp, respectively) and BstBI + XbaI (expecting fragments on 1474 and 2831 bp)
</P><P>rrnB was cut with EcoRI + SpeI (expecting fragments on 516 and 2047 bp) and EcoRI + SacI (expecting fragments on 900 and 1663 bp)
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fragments resulting from the restriction digest of rrnB+RBS+LacI+TERM+pLac+RBS+ mCherry+TERM, RBS+LacI+TERM+pLac+RBS+ mCherry+TERM, relA and rrnB P1</DIV></DIV></DIV><P>The digestion mixtures were made according to the following table:
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>Enzymes
</TH><TH>Buffer
</TH><TH>Volume DNA added [µl]
</TH><TH>Volume dH<SUB>2</SUB>O added [µl]
</TH></TR><TR><TD>relA
</TD><TD>PstI
</TD><TD>Buffer 3 + BSA
</TD><TD>3.92
</TD><TD>38.58
</TD></TR><TR><TD>relA
</TD><TD>BstBI + XbaI
</TD><TD>Buffer 4 + BSA
</TD><TD>3.92
</TD><TD>38.58
</TD></TR><TR><TD>rrnB
</TD><TD>EcoRI + SpeI</TD><TD>Buffer 2 + BSA
</TD><TD>5.97
</TD><TD>36.53
</TD></TR><TR><TD>rrnB
</TD><TD>EcoRI + SacI
</TD><TD>Buffer 1 + BSA
</TD><TD>5.97
</TD><TD>36.53
</TD></TR></TBODY></TABLE><P>The digestion mixtures were incubated for one hour in 37°C.
The fragments resulting from the restriction digest were separated using gel electrophoresis.
</P><P><I>Preparation of plasmids for shipping to Registry of standard biological parts:</I></P><P>According to the concentrations previously measured, we calculated the amount of TE buffer we needed to add to get a concentration of 25 ng/µl for all of the samples. DNA and TE buffer were added according to the following table, and concentrations were measured:
</P><TABLE border="1"><TBODY><TR><TH>Sample
</TH><TH>DNA added [µl]
</TH><TH>TE buffer added [µl]
</TH><TH>Concentration [ng/µl]
</TH></TR><TR><TD>rrnB+RBS+LacI+TERM+pLac+RBS+mCherry+TERM
</TD><TD>10.40
</TD><TD>39.60
</TD><TD>26.6
</TD></TR><TR><TD>RBS+LacI+TERM+pLac+RBS+mCherry+TERM
</TD><TD>9.61
</TD><TD>40.39
</TD><TD>24.8
</TD></TR><TR><TD>relA
</TD><TD>9.81
</TD><TD>40.19
</TD><TD>23.7
</TD></TR><TR><TD>rrnB P1
</TD><TD>14.93
</TD><TD>35.07
</TD><TD>26.6
</TD></TR></TBODY></TABLE><DIV><H4><SPAN class="mw-headline">Sponsors</SPAN></H4><DIV id="sponsor-box"><A class="sponsor" href="http://www.ntnu.edu/rector"><B>NTNU Rector's Office</B></A><A class="sponsor" href="http://www.ntnu.edu/bioinformatics/pbi"><B>Programme for Bioinformatics, NTNU</B></A></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:NTNU_Trondheim/Journal">http://2011.igem.org/Team:NTNU_Trondheim/Journal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTNU_Trondheim/Journal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTNU_Trondheim/Journal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Journal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Journal&amp;oldid=154835" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>