"
Page
Discussion
View source
History
teams
Log in
 
Team:Nanjing China Bio/method
From 2012.igem.org
HOME
ABOUT US
PROJECTS
RESULTS
NOTEBOOK
OUTREACH
SAFETY
Method Home > projects
Background
Future
Method
Getting anaerobic promoters
We get these promoters in two ways. One is from the E.coli K12 which is found in papers. We use PCR reaction to get more promoter products for the following experiments. Another is from the iGEM committee. We get plasmids from E.coli and cut off promoters by restriction enzyme.
Using molecular cloning technology to construct new plasmids
After we have these promoters, we ligate each of them with luciferase by the molecular cloning technology. We use two different methods to link the promoter and reporter. One is to use the plasmids that have luciferase gene to be the plasmid bone of our parts. In this method, we use enzyme EcoRI and XbaI to switch off the plasmid that has luciferase gene, and use EcoRI and SpeI to cut off the promoter that was cloned from E.coli. The other method is used to deal with promoters that come from the iGEM committee. We use XbaI and PstI to cut the luciferase gene, and use SpeI and PstI to cut the plasmid with promoter. Then we use T4 DNA ligase to sew up the two strands. After this, we transform these plasmids into E.coli Top10 strains. And we get plasmid DNA production using alkaline lysis method.
Screening promoters with aerobic incubation and anaerobic incubation methods
We transform the constructed plasmids into E.coli for incubating , then we select monoclone to another medium to incubate again. After incubating these monoclones , we use alkaline lysis method to get new plasmids and transform them into S.typhimurium VNP20009 respectively with electroporation. Based on our experiments design, after aerobic and anaerobic incubation, we can know the expression level of each promoter in different incubation environments and select more advantageous promoters to do the following researches. From the results, we find out that the activities of these promoters were several times enhanced when the culture condition was exchanged from aerobic to anaerobic.
Electroporation instrument
Anaerobic incubation
Salmonella selection: methods
Using λ red recombination method to create mutations
The first step to knock out the genes of S. typhimurium VNP20009 was to introduce pkd46 into the bacteria so that the three genes, γ, β, and exo can be expressed. Gam inhibits the host RecBCD exonuclease v so that BET and EXO can gain access to DNA ends and to promote recombination. We then designed the PCR primers which provide the homology to the targeted gene(s). Primers were 60bp in length and were used to amplify a Kanamycin cassette from pKD4 with flanking DNA to the gene targeted for deletion. For the forward primer (RF1), select 40bp from ATG backwards, then add 20bp from vector backbone (CATATGAATATCCTCCTTA). For the reverse primer (RR1), select 24bp from the last nucleotide (stop codon) of gene inwards and 16bp after stop codon (downstream of gene). Reverse complement this sequence, then add 20bp of reverse-complemented vector backbone sequence (GTGTAGGCTGGAGCTGCTCC). In order to incorporate the linear DNA into the chromosome of salmonella, we electroporate the PCR product into the Salmonella harboring pKD46. With the help of the red recombinase, the linear DNA can incorporate into the chromosome. Finally we lose pKD46 by incubating the plate at 43°C since pKD46 contains a temperature sensitive replicon.
The steps to knock out gene(s) of VNP20009
Tumor implantation
The next resulted strain will be assayed of its ability to target tumors and the ratio of its number in tumors to that in normal tissues. B16F10 melanoma cells were purchased from the American Type Culture Collection (ATCC) and maintained in Dulbecco's modified eagle's medium supplemented with 10% fetal bovine serum by routine culture methods. For tumor implantation, 6-8 week female C57BL/6 mice were implanted subcutaneously (S.C.) on the mid-right side with 2×105 B16F10 cells in 0.1ML PBS. Normal tissues primarily infected by Salmonella include spleen and liver. Because of the size and accessibility of the liver, we generally use this organ as an indicator of the highest-level CFU in normal tissues.
1.C57BL6 mice were implanted subcutaneously with 5×105 B16F10 mouse melanoma cells and staged until the appearance of palpable tumors (1 WK).
2.Grow salmonella in LB overnight at 37°C.
3.On the following day, the bacteria are transferred to LB broth, subjected to further growth at 37°C on a rotator to OD600= 0.8, and placed on ice.
4.In the following growth, they are diluted to a concentration of 104 CFU/mL in PBS on ice, warmed to room temperature, and 0.2 mL injected intraperitoneally or intravenously into C57BL6 mice (n=5-10).
Cell culture passage
1. Wipe your work area and hands with 70% ethanol before starting.
2. Decant supernatant fluid from culture into a waste collection jar.
3. Washing with PBS.
4. Add 3ml of cold trypsin and examine under low magnification for 3-7 minutes.
5. It appears that most of the cells have rounded up, but have not yet completely detached. Then immediately return to the hood and mix the cells with pipette several times to help break up the clumps of cells.
6.Add some fresh DMEM media to three new flasks (if you are doing a 1:3 split). Suspend the cells with pipette and quickly dispense the aliquot into each flask containing fresh media .
Bacterial replication in various tissues
Isolate and titrate bacteria from blood, heart, liver, spleen, lung, kidney and tumor. Infected mice were euthanized and those tissues were removed aseptically and homogenized with 2ml PBS. Serially diluted homogenates were spread onto modified LB agar plates and incubated at 37°C for 24 h. The titer of bacteria was determined by counting colonies and dividing them by the weight of the tissue (colony-forming unit CFU/g tissue).
Retrieved from " http://2012.igem.org/Team:Nanjing_China_Bio/method "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
