"
Page
Discussion
View source
History
teams
Log in
 
Team:UQ-Australia/Project
From 2009.igem.org
Home
Team
Projects
Parts
Modelling
Notebook
Our Lab
Sponsors
Contents
1Mercury sequestration using a Multicomponent Operon and Bioprecipitation using P.syringae
1.1Project Abstract
1.2Bioaccumulation (Mercury) Project
1.2.1Project Outline
1.2.2Project Background
1.2.3Methods
1.2.3.1Transformation
1.2.3.2Picking Colonies
1.2.3.3Miniprep
1.2.3.4Glycerol Stock
1.2.3.5Digestions
1.2.3.6Agarose Gel
1.2.3.7In-gel extraction
1.2.3.8De-phosphorylation
1.2.3.9Ligation
1.2.3.10Sequencing
1.2.4Results
1.2.4.1Isolation of Ag43 gene
1.2.4.2Creation of BBa_I716101 Plasmid Stock, and Restriction Site Checking
1.2.4.3Preparation of Ag43 gene for compatability with BioBrick Standard 21
1.2.4.4Creation of BBa_J63010 Plasmid Stock, and Restriction Site Checking
1.2.4.5Mer Gene Isolation and Standard Compatability
1.2.4.6Cloning of MerT and MerP into BBa_J63010
1.2.5Conclusions
1.2.6References
2Bioprecipitation using P.syringae
Mercury sequestration using a Multicomponent Operon and Bioprecipitation using P.syringae
Project Abstract
Microbes such as Escherichia coli and Cuprivadis metallidurans have an endogenous multicomponent mercury (Hg2+) uptake and reduction operon, under the control of a metal responsive transcription factor, MerR.  By utilising elements of this pathway, with a novel recovery mechanism, mercury can be accumulated intracellularly and efficiently removed from the environment. The presence of mercury activates MerR, driving the expression of Antigen 43 (Ag43), a self-adhering surface protein. Coupling a mercury sensitive promoter to the expression of Ag43 enables cells to accumulate mercury then aggregate in solution.
P. syringae is a ubiquitous airborne bacterium which expresses a unique protein, InaZ. This protein acts as a scaffold for ice nucleation, inducing precipitation. Optimal growth of P. syringae occurs at 22oC. By introducing five heat-shock genes, the tolerance range will be increased to better suit the Australian climate.  This modification has the potential to increase the availability of Australia’s most precious resource; water.
Bioaccumulation (Mercury) Project
Project Outline
Water contamination is a key environmental issue for many countries around the world, both developed and developing.  In Queensland, Australia we have a particular problem with Mercury (Hg2+) contamination of water supplies around the major mining town of Mt Isa, and also from Airforce bases. After searching through the iGEM projects from previous years,  the arsenic detection system inspired us.  As the UQ 09' team, we wish to take this idea one step further and completely remove the offending heavy metal from water systems.
To do this we will be utilizing a strain of Escherichia coli, with their already established mercury uptake, reduction and efflux system and making a few modifications. One of our aims is to couple the detection of Mercury to the expression of a native bacterial protein, Antigen 43 (AG43).  This protein, when expressed, causes the bacteria to stick to one another. As the bacteria aggregate in clumps, they will fall to the bottom of the sample.  Our idea is for the bacteria to take up the mercury, activating  Ag43 expression, resulting in aggregation, and the Mercury-filled bacteria will fall to the bottom leaving clean water.
There are a number of parts that we hope to add to the registry. The first is Ag43 as a protein coding sequence and the MerR promoter sequence.  We will also add the completed mercury uptake and aggregation system as an operon.
Project Background
Many bacteria carry heavy metal resistance systems, either encoded on a plasmid or within their genome. For example, the bacterium Cupriavidis metallidurans, first isolated from a sludge tank contaminated with high levels of heavy metals, contains genes which encode for reistance systems to Ag(I), Cd(II), Co(II), Cr(IV), Hg(II), Ni(II), Pb(II), and Zn(II) [1] . These systems commonly contain a metal transporter, a metal binding protein, a reduction mechanism, and a metal responsive promoter. In the case of Hg(II), the currently accepted general system is outlined below. [2]
This system facilitates the transportation of Hg(II) into the cell, where MerA acts a reductase, producing Hg(0), which then diffuses out of the cell. However, our project focuses on the accumulation of mercury within the cell, so many elements of this pathway are unsuitable for our purpose. We envisage a system of two plasmids, one containing mercury resistance genes (mer genes).
Plasmid 1
This plasmid will allow for the transport of Hg(II) into the cell, where the protein metallothionien aids in intracellular accumulation. Once Hg(II) enters the cell, it activates MerR, a transcriptional regulator. When active, MerR will initiate the transcription of Antigen43 (Ag43).
Plasmid 2
A. DNA twisted conformation and transcription unable to occur B.Conformational change in MerR when Hg binds - transcription goes ahead C. Hypothetical situation in which MerD (in the endogenous system) binds preferentially to stop transcription (4) .
Ag43 is a surface expressed autotransporter [3] . This means all the information for suface targeting of the plasma membrane is contained within the protein. Since Ag43 is a native protein to E. Coli (specifically K12 strain), it must first be removed from the genome, then re-inserted as a plasmid so that its expression can be controlled. We were able to obtain a flu- E. coli strain, kindly donated by Associate Professor Mark Schembri from the University of Queensland. When two E. coli cells express Ag43, they aggregate together. Once enough of these cells aggregate, they will fall to the bottom of solution. This autoaggregation can be easily measured using a spectrophotometer.
Methods
Transformation
Electroporation
1uL of plasmid DNA added to a gene pulse cuvette with 50uL of defrosted electrocompetent E.coli and placed in electroporator at 2500V (2mm cuvette). 0.9mL of sterile SOC medium added in order to dilute the transformants and placed into the incubator at 180rpm for one hour at 37deg C. Agar plates loaded with the appropriate antibiotic warmed in incubator and 300-500uL transformed bacteria spread on plates.
Heat-shock
2uL of plasmid DNA added to 100uL of chemically competent E. coli and incubated on ice for 30 minutes. Bacteria were heat shocked for 30 seconds at 42deg C then incubated on ice for a further 2 minutes. Transformants were then shaken at 225rpm and 37deg C for one hour. 300-500uL of sample added to the agar plates loaded with the approporiate antibiotic and left to grow overnight at 37deg C.
Picking Colonies
Positive colonies were picked from plates, and cultured in 3mL of either LB broth or S.O.C media, supplemented with the appropriate antibiotic. These cultures were incubated overnight at 37deg C in a shaking incubator.
Miniprep
3mL of overnight culture was pelleted, and processed using an Invitrogen PureLinkQuick Plasmid Miniprep Kit.
1. Pellet resuspended in 250uL Resuspension Buffer with RNase A.
2. 250uL Lysis Buffer added to lyse cells.
3. Tubes incubated at room temperature for 5 minutes.
4. 350uL Precipitation Buffer added then centrifuged at 12000g for 10 minutes.
5. Supernatant extracted into a spin column then centrifuged once more at 12000g for one minute.
6. 700uL Wash Buffer with ethanol added to the column and incubated for 1 minute then centrifuged at 12000g for one minute.
7. Flow through discarded then spin column placed into a fresh 1.5mL recovery tube.
8. 75uL of pre-heated TE buffer added to column before incubation at room temperature for one minute.
9. Centrifugation of sample at 12000g for 2 minutes.
10. Recovery tube now contains purified plasmid DNA!
Product was Nanodropped to determine concentration, then stored at -20deg C.
Glycerol Stock
Glycerol stock solution was created by initially grown the original sample strain on an agarose gel plate overnight. Colonies grown overnight were then picked and placed into LB broth solution containing ampicillin and grown overnight at 37 deg C on the shaker. The cells were formed into a pellet and resuspended in a 50:50 glycerol and LB stock solution mixture. Cells were then stored for later use in a -80 deg C freezer.
Digestions
Samples contained a mixture of water, buffer plasmid and appropriate enzymes. Samples were incubated at 37deg C with NEB or Promega restriction enzymes for at least 3 hours to ensure complete digestion. Samples were then run on a 1% gel to determine the band size and if the digest had occurred.
Agarose Gel
DNA grade agarose powder was mixed into 1x TAE and heated until fully dissolved to create either a 1% or 2% gel depending on the experimental parameters. The heated mixure is then poured into a mold and allowed to set. Gels were post-stained using Ethidium Bromide, then visualized with a UV transilluminator. All gels were run with an appropriate ladder depending on the size of the fragments.
In-gel extraction
Bands were excised with a scalpel blade viewed using a UV box removing all of the band with as little of the agarose from the gel. These pieces were then placed into pre-weighed eppendorf tubes and re-weighed after to determine the mass of the excised piece of agarose. In-gel extraction was performed as per protocol from Invitrogen Purelink Gel Extraction Kit.
1. 3 volumes of Gel Solubilisation Buffer added to each volume of gel (preweighed).
2. Tube containing gel + buffer incubated in 50deg C waterbath for 10-15 minutes and inverted every 3 minutes to dissolve all gel.
3. 1 gel volume of isopropanol added for optimal DNA yields.
4. Dissolved gel mixture added to a Gel Extraction Column inside a wash tube.
5. Sample centrifuged at 12000g for one minute. Flow-through discarded.
6. 600uL Wash Buffer with ethanol added then centrifuged again for one minute at 12000g. Wash tube discarded and replaces with a Recovery tube.
7. 50uL Elution Buffer added and incubated at room temperature for one minute.
8. Purified DNA recovered from recovery tube after centrifugation at 12000g for one minute.
Product was Nanodropped to the determine concentration.
De-phosphorylation
10uL vector was incubated with 0.5uL Promega Shrimp Alkaline Phosphatase and 3uL 10XSAP reaction buffer at 37deg C overnight.
Ligation
Insert and de-phosphorylated vector were incubated at room temperature with NEB T4 DNA ligase for at least one hour before transformation.
Sequencing
Whole plasmid samples with appropriate primers were submitted to the Australian Genome Research Facility within the University of Queensland for sequencing.
Results
Isolation of Ag43 gene
Samples of E. coli MS427 containing either pBAD (empty vector) or pKKJ143 (Ag43+) were kindly donated by Associate Professor Mark Schembri, of The University of Queensland. [4]
Figure 1 | Plasmid DNA was extracted from cultures, and run on a 1% agarose gel. Bands were observed which corresponded to the expected size of pBAD (4,210bp) and pKKJ143 (7,205bp)
Creation of BBa_I716101 Plasmid Stock, and Restriction Site Checking
Plasmid backbone BBa_I716101 with part BBa_J04450 was obtained from the Spring parts distribution (Plate 1, Well 1C). This plasmid was transformed into E. coli, and red colonies were observed, indicating a positive result. Five clones were cultured further, then plasmid DNA extracted using a miniprep kit.
Figure 2 | DNA of five clones was digested overnight with either XhoI, BamHI, EcoRI or BglII. Only EcoRI appears to have been effective at cutting the plasmid. Lanes 1-5: Clones A->E, EcoRI. Lanes 6-10: Clones A->E, BglII. Lanes 11-15: Clones A->E, XhoI. Lanes 16-19: Clones B->E, BamHI.
Figure 3| repeat digestion using XhoI, BamHI, EcoRI or BglII was performed, with known control digests (pGfa2cLacZ - from Thomas lab). Again, only EcoRI appears to digest BBa_I716101. Lane 1: pGfa2cLacZ, EcoRI. Lane 2: BBa_I716101 clone A, EcoRI. Lane 3: pGfa2cLacZ, BglII. Lane 4: BBa_I716101, BglII. Lane 5: pGfa2cLacZ, XhoI. Lane 6: BBa_I716101 XhoI. Lane 7:pGfa2cLacZ, BamHI. Lane 8: BBa_I716101, BamHI. Lane 9: pGfa2cLacZ, undigested. Lane 10: BBa_I716101, undigested.
Figure 4| Digestion of BBa_J63010 A with PstI. Lane 1: BBa_J63010, undigested. Lane 2: BBa_J63010, PstI.
Preparation of Ag43 gene for compatability with BioBrick Standard 21
The sequence of Ag43 was analysed for incompatible restriction sites, and primers designed to add the appropriate restriction sites to allow Ag43 to be cloned into BBa_I716101.
Figure 5 | Primer sequences for addition of restriction sites to Ag43 gene. Green nucleotides are part of the known sequence, red nucleotides are new nucleotides to be added to the gene. Italic nucleotides are the prefix and suffix for Standard 21.
Figure 6 | Gel of PCR product; Lane 1 - AG43 with Standard 21 prefix and suffix attached (3,176bp).
Creation of BBa_J63010 Plasmid Stock, and Restriction Site Checking
Plasmid backbone BBa_J63010 with part BBa_J04450 was obtained from the Spring parts distribution (Plate 1, Well 3C). This plasmid was transformed into E. coli, and red colonies were observed, indicating a positive result. Five clones were cultured further, then plasmid DNA extracted using a miniprep kit.
Figure 7 | Agarose gel of BBa_J63010 clones. Lanes 1-5: BBa_J63010 clones A-E.
Figure 8 | Digestion of BBa_J63010 with EcoRI, XbaI, SpeI, PstI. Lane 1: Control 1, undigested. Lane 2: Control 1, EcoRI. Lane 3: Control 1, XbaI. Lane 4: Control 2, undigested. Lane 5: Control 2, SpeI. Lane 6: Control 2, PstI. Lanes 7-11: BBa_J63010 A; undigested, EcoRI, XbaI, SpeI, PstI. Lanes 12-16: BBa_J63010 B; undigested, EcoRI, XbaI, SpeI, PstI. Lanes 17-21: BBa_J63010 C; undigested, EcoRI, XbaI, SpeI, PstI. Lanes 22-26: BBa_J63010 D; undigested, EcoRI, XbaI, SpeI, PstI. Lanes 27-31: BBa_J63010 E; undigested, EcoRI, XbaI, SpeI, PstI.
Mer Gene Isolation and Standard Compatability
E. coli JM109 containing plasmids pSUTp (MerT, MerP), pGmpt (P. sativium Metallothionein)and pSUTp.pG.pmt were kindly donated by Professor David Wilson, from the Department of Molecular Biology and Genetics, Cornell University [5] . Cells were cultured, then plasmid DNA extracted using a miniprep kit.
Figure 9 | Lane 1: pSUTp.pG.pmt. Lane 2: pGpmt. Lane 3: pSUTp.
Since these plasmids are unsequenced, samples were sent to the AGRF for sequencing.
Figure 10| Alignment of MerT sequence results from pSUTp, against the original gene source, plasmid pDU1358 (from Serratia marcescens). Score = 304 bits (336), Expect = 1e-78, Identities = 623/901 (69%), Gaps = 35/901 (3%).
Figure 11 | Primer sequences for addition of restriction sites to MerT and MerP genes. Green nucleotides are part of the gene sequences, red nucleotides are new nucleotides to be added to the gene. Italic nucleotides are the prefix and suffix for Standard 23.
Cloning of MerT and MerP into BBa_J63010
PCR gene fragments and BBa_J63010 were digested overnight with EcoRI and PstI. Digestions were run on a gel, then in-gel extraction performed. BBa_J63010 was then de-phosphorylated using SAP. Gene was added to vector at 3:1 molar ratio for ligation with T4 DNA ligase. Ligation products were transformed into cells, supplemented with ampicillin, then positive colonies picked for micro-culture. Plasmid DNA was isolated using a miniprep kit, and a digestion using EcoRI and PstI performed.
Figure 12 | Two positive clones of MerT + BBa_J63010 were digested using EcoRI and PstI. Fragment size at 376pb is MerT gene.
Both clones have been submitted for sequencing (21/10/09). We are currently waiting on the results.
MerT was named part BBa_K205004 and submitted to the Registry of Standard Biological parts in plasmid backbone BBa_J63010.
Conclusions
Part BBa_I76101 is incorrect. The sequence supplied on the Registry of Standard Biological Parts site is not a true refelection of the physical DNA. The sequence was unable to be cut by BglII, XhoI or BamHI, which were predicted restriction sites from the sequence. It was cut by PstI, when the given sequence does not contain that site. This part does not fulfill the conventions of Standard 21, as it contains neither the correct suffix or prefix.
Ag43 was isolated from plasmid pKKJ143. However, because the intended vector BBa_I76101 did not conform to Standard 21, this part was taken no further.
Part BBa_J63010 contains the correct Standard 23 Biobrick suffix and prefix.
MerT and MerP were successfully isolated from plasmid pSUTp.
After cloning into vector BBa_J63010, a fragment of the correct size (corressponding to MerT) was observed after digest with appropriate restriction enzymes.
Part BBa_K205004 (MerT) was added to the Registry of Standard Biological Parts.
References
[1] Monchy, S., Benotmane, M. A., Janssen, P., Vallaeys, T., Taghavi, S., van der Lelie, D. and Mergeay, M. (2007). Plasmids pMOL28 and pMOL30 of Cupriavidus metallidurans are specialized in the maximal viable response to heavy metals. J Bacteriol189, 7417-25.
[2] Barkay, T., Miller, S. M. and Summers, A. O. (2003). Bacterial mercury resistance from atoms to ecosystems. FEMS Microbiol Rev27, 355-84.
[3] Kjaergaard, K., M. A. Schembri, et al. (2000). Antigen 43 from Escherichia coli induces inter- and intraspecies cell aggregation and changes in colony morphology of Pseudomonas fluorescens. J Bacteriol182(17): 4789-96.
[4] Ulett, G. C., Valle, J., Beloin, C., Sherlock, O., Ghigo, J. M. and Schembri, M. A. (2007). Functional analysis of antigen 43 in uropathogenic Escherichia coli reveals a role in long-term persistence in the urinary tract. Infect Immun75, 3233-44.
[5] Chen, S. and Wilson, D. B. (1997). Construction and characterization of Escherichia coli genetically engineered for bioremediation of Hg(2+)-contaminated environments. Appl Environ Microbiol63, 2442-5.
Bioprecipitation using P.syringae
Project Description
The arid climate of Australia has become notorious for causing widespread problems for agricultural industries.  That is, the scarcity of water hinders the production of crops and livestock, as well as forcing restrictions of daily water usage for households.  Team UQ Australia aims to solve this problem through the use of a bio-precipitation technique, thus increasing the availability of our most precious resource.
Pseudomonas syringae is a common bacterium, primarily found in colder climates (optimally at 22°C), and is well known for its biological ice nucleation properties, i.e. the formation of rain/snow.  This bacteria expresses an ice nucleation protein on its outer membrane (InaZ). InaZ acts as a scaffold for the formation of the ice crystals, this directly assists in the formation of clouds and speeds up the rain cycle. However, the bacteria are unable to survive in environmental temperatures above 28°C.  The ideal growing temperatures for syringae are 22°C - 26°C. By introducing heat shock proteins (DnaK, DnaJ, GroEL, and GroES) through plasmid insertion, UQ Australia aims to increase the optimal growth temperature available to P.syringae, thereby allowing bio-precipitation to occur in warmer (and drought stricken) climates. Although syringae already has this proteins encoded within the genome, they are not expressed at levels similar to bacteria which grow at higher temperatures (Susin, et al, 2006).
The heat shock proteins form two systems. GroEL and GroES form one dimer which acts as a chaperon. Together these proteins assists in lowering the mis-folding of proteins. At higher temperatures proteins become unstable and mis-fold at a higher rate.
Ideally, UQ Australia will be contributing a number of parts to the registry.  Firstly, a plasmid with DnaK, DnaJ, as well as an upstream promoter will drive the expression of these two genes.  Secondly, an additional plasmid will carry the genes for GroEL and GroES, also complemented by an upstream promoter to drive gene expression.
Method
After receiving the parts from Kristos we went through the process of transforming them into the bacteria. In order to insert two of the genes into the iGEM standards we have to perform mutagenesis. This process is detailed in the notebook. The basic concept is changing one base pair that doesn't affect the translated protein but removes a restriction site that would normally cut with the same enzyme that the standard restriction sites would. Once this was removed we performed PCR on each of the genes. After PCR each part was digested and ligated into the standard plasmid for iGEM standards 21 and 23. Unfortunately standard 21 was unavailable. There were issues
By taking measurements during the life cycle of P. syringae it should show a shift in the maximal growth peak. Over a 14 hour growth period absorbance readings were taken of the transformed bacteria. Singh et al performed this same experiment with syringae which grew optimally at lower temperatures. Image A shows the optical density of the bacteria population over their growth period. After an initial lag phase of 4 hours there were considerable differences in the absorbance readings. Absorbance readings or optical density can be directly correlated to the total population of bacteria. When bacteria grow and populate the sample the media will become more dense with bacteria. Thus you can compare the density to a blank sample. Distinct difference in the growth rate were evident. For our project we aim to change the maximum density reached and thus the total growth of the bacterial population. Growth measurement needs to be taken in the stationary phase as to get an accurate representation of bacteria that are dying as well as dividing and growing.
References
Singh et al (2009) Importance of trmE for Growth of the Psychrophile of Pseudomonas syringae at Low temperatures. Applied and Envrinomental Microbiology 75: 4419 - 4426
Susin et al (2006) GroES/GroEL and DnaK/DnaJ Have Distinct Roles in Stress Responses and during Cell Cycle Prograssion in Caulobacter crescents, Journal of Bacteriology 188: 8044 - 8053
Retrieved from " http://2009.igem.org/Team:UQ-Australia/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
