<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UQ-Australia_Notebook_Project1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UQ-Australia/Notebook/Project1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UQ-Australia/Notebook/Project1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UQ-Australia/Notebook/Project1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UQ-Australia/Notebook/Project1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UQ-Australia/Notebook/Project1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UQ-Australia/Notebook/Project1</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2009.igem.org/Team:UQ-Australia" style="color: white">Home
			</A></LI><LI><A href="https://2009.igem.org/Team:UQ-Australia/Team" style="color: white">Team
                        </A></LI><LI><A href="https://2009.igem.org/Team:UQ-Australia/Project" style="color: white">Projects
                        </A></LI><LI><A href="https://2009.igem.org/Team:UQ-Australia/Parts" style="color: white">Parts
                        </A></LI><LI><A href="https://2009.igem.org/Team:UQ-Australia/Model" style="color:white">Modelling
                        </A></LI><LI><A href="https://2009.igem.org/Team:UQ-Australia/Notebook" style="color: white">Notebook
                        </A></LI><LI style="width: 160px"><A href="https://2009.igem.org/Team:UQ-Australia/Lab" style="color: white">Our Lab
                        </A></LI><LI><A href="https://2009.igem.org/Team:UQ-Australia/Sponsors" style="color: white">Sponsors</A></LI></UL></DIV><H1><SPAN class="mw-headline" id="Bioacumulation_.28Mercury.29_Project">Bioacumulation (Mercury) Project</SPAN></H1><P><B>19/10/09: Nanodrop miniprep products of DDA2+</B></P><TABLE border="1"><CAPTION> Results:
</CAPTION><TBODY><TR><TH> Sample name </TH><TH> Concentration (ng/uL) </TH><TH> 260/280
</TH></TR><TR><TH> DDA2+MerT
</TH><TD> 23.78 </TD><TD> 1.74
</TD></TR><TR><TH> DDB2+MerT
</TH><TD>19.84
</TD><TD>19.84
</TD></TR></TBODY></TABLE><P><B>18/10/09: Miniprep of DDA2+MerT and DDB2+Mer P bacteria</B></P><P>Click HERE for detailed protocol of miniprep.
</P><P>2 samples were collected: DDA2+MerT and DDB+MerP plasmids.
</P><P><B>16/10/09: Microculture of DDA2  +MerT and DDB2+ MerT</B></P><P><B>15/10/09: Transformation of MerT and MerP into standard 23 plasmid:</B></P><P>Clik HERE for detailed protocol of transformation reaction.
</P><P>Result: No colony was shown on DDB2+MerP plates. 
</P><P><B>11/10/09-14/10/09: </B></P><UL><LI>Repeat the double digestion conducted on 10/09/10.
</LI><LI>In gel extraction for BBa , MerT and MerB
</LI><LI>Dephosphorylate BBa
</LI><LI>Ligation reaction.
</LI><LI>Transformation.
</LI></UL><P><B>10/10/09:</B></P><UL><LI> Experiment 1: Nanodrop for miniprep products of pSUTp and pSUTp.pG.pmt
</LI></UL><PRE>                           (Can you please put the link to Experiment 1 here?
</PRE><UL><LI>Experiment 2: Agarose gel for PCR products of MerT and MerP      (09/10/09)
</LI></UL><PRE>                                         and miniprep products of pSUTp and pSUTp.pG.pmt (09/10/09)
</PRE><PRE>                           (Can you please put the link to Experiment 2 here?
</PRE><UL><LI>Experiment 3: Double digestion for BBa_J63010
</LI></UL><PRE>                            (Can you please put the link to Experiment 3 here?
</PRE><UL><LI>Experiment 4: PCR MerT and MerP gene.
</LI></UL><PRE>                             (Can you please put the link to Experiment 4 here?
</PRE><P><I>Experiment 1: Nanodrop for miniprep products of pSUTp and pSUTp.pG.pmt</I></P><TABLE border="1"><CAPTION> Results:
</CAPTION><TBODY><TR><TH> Sample name </TH><TH> Concentration (ng/uL) </TH><TH> 260/280
</TH></TR><TR><TH> pSUTp Kan 1
</TH><TD> 21.6 </TD><TD> 1.67
</TD></TR><TR><TH> pSUTp Kan 2
</TH><TD> 22 </TD><TD> 1.77
</TD></TR><TR><TH> pSUTp Kan 3
</TH><TD> 23.7 </TD><TD> 1.56
</TD></TR><TR><TH> pSUTp Kan 4
</TH><TD> 21.6 </TD><TD> 1.67
</TD></TR><TR><TH> pSUTp.pG.pmt 1
</TH><TD> 62.9 </TD><TD> 1.77
</TD></TR><TR><TH> pSUTp.pG.pmt 2
</TH><TD> 51.7 </TD><TD> 1.78
</TD></TR><TR><TH> pSUTp.pG.pmt 3
</TH><TD> 57.7 </TD><TD> 1.75
</TD></TR><TR><TH> pSUTp.pG.pmt 4
</TH><TD> 58.4 </TD><TD> 1.65
</TD></TR></TBODY></TABLE><P><I>Experiment 2: Agarose gel for PCR products of MerT and MerP and miniprep products of pSUTp and pSUTp.pG.pmt</I></P><PRE> Set up the 1% agarose gel for miniprep products and 2% gel for Mer gene PCR products.
</PRE><P>Result: Click HERE for the gel image of the PCR product (File 6)
</P><PRE>           Lane 1: Ladder; Lane 2-4: pSUTp.pG.pmt 1-4; Lane 5-8: pSUTp 1-4
</PRE><PRE>          Click HERE for the gel.image of miniprep products (File 7)
</PRE><PRE>          Lane 1: Ladder; Lane 2:     ;Lane 3:     ; Lane 4:
</PRE><P><I>Experiment 3: Double digestion of BBa_J63010 and Mer gene PCR products from 09/10/09 with PstI and ECoRI</I></P><P>General formulae to set up the reaction (Add up to a total of 20uL in each reaction)
200ng of plasmid.
2uL of 10x Buffer.
1uL each enzyme.
X uL of water.
From the concentration of two samples of BBa_J63010 (BBa-J63010A and BBa_J63010B) find out the volume needed for the reaction.
Set up 6 tubes of double digestion for 6 samples of BBa-J63010A , BBa_J63010B, and 4 PCR products from 09/10/09 (Can you please put the link to 09/10/09 here?) :MerT , MerT +DMSO, MerP, MerP+DMSO.
In each tube, add the solution as indicated below:
</P><TABLE border="1"><CAPTION> Results:
</CAPTION><TBODY><TR><TH> Sample </TH><TH> Plasmid (uL) </TH><TH> Buffer (uL) </TH><TH> Enzyme PstI (ug) </TH><TH> Enzyme EcoRI (uL) </TH><TH> Water (ug)
</TH></TR><TR><TH> BBa_J63010_
</TH><TD> 2.7 </TD><TD> 2 </TD><TD> 1 </TD><TD> 1 </TD><TD> 13.3
</TD></TR><TR><TH> Mer_ gen
</TH><TD> 10 </TD><TD> 2 </TD><TD> 1 </TD><TD> 1 </TD><TD> 6
</TD></TR></TBODY></TABLE><P>Leave the reaction overnight.
Experiment 4 : PCR reaction of pSUTp with MerT and MerP primers
</P><P>Repeat PCR products for Mer_ gene.
Click HERE for the detailed protocol; however, only step 1,3 and 6 were followed.
4 tubes were set up :
</P><UL><LI>2x pSUTp. MerT
</LI><LI>2x pSUTp. MerP
</LI></UL><P><B>09/10/09:</B></P><UL><LI>Experiment 1: PCR reaction of pSUTp with MerT and MerP primers
</LI></UL><PRE>                           (Can you please put the link to Experiment 1 here?)
</PRE><UL><LI>Experiment 2: Miniprep pSUTp and pSUTp.pG.pmt colonies
</LI></UL><PRE>                           (Can you please put the link to Experiment 2 here?)
</PRE><P><I>Experiment 1: PCR reaction of pSUTp with MerT and MerP primers</I></P><P>The aim of this reaction is to amplify MerT and MerP gene from its plasmid.
</P><P>Prepare 10uM solution of each of four primers (MerT S, MerT AS, MerP S, MerP AS-received on 08/10/09)
</P><P>Click HERE for detailed protocol of PCR reaction (File 5).
</P><P>5 tubes collected from 5 different reactions were respectively named:
</P><UL><LI>Reaction 1: MerT
</LI><LI>Reaction 2: MerT+DMSO
</LI><LI>Reaction 3: MerP
</LI><LI>Reaction 4: MerP+ DMSO
</LI><LI>Reaction 5: Control
</LI></UL><P><I>Experiment 2: Miniprep pSUTp and pSUTp.pG.pmt colonies</I></P><P>The aim of this experiment is to isolate pure pSUTp and pSUTp.pG.pmt plasmids out of the bacteria.
</P><P>For each plasmid, four samples from four different colonies were collected, named : pSUTp  Kan 1-4 and pSUTp.pG.pmt Amp 1-4.
</P><P>Click HERE for detailed miniprep protocol.
</P><P>Click HERE for nanodrop results (Can you please put the link to Experiment 1 on 10/10/09 here?)
</P><P><B>08/10/09: Pick isolate colonies from pSUTp and pSUTp.pG.pmt cultures</B></P><P>Click HERE for detailed protocol of picking colonies.
</P><UL><LI>4 colonies from pSUTp culture were picked and transfered into new tubes named pSUTp  Kan 1-4
</LI><LI>4 colonies from pSUTp.pG.pmt were picked and transferred into new tubes named  pSUTp.pG.pmt Amp 1-4
</LI></UL><P>NOTE: MerT and MerP primers arrived.
</P><P><B>07/10/09:</B></P><UL><LI> Experiment 1: Gel extraction for PCR product of Ag43.
</LI></UL><PRE>                           (Can you please put the link to Experiment 1 here?)
</PRE><UL><LI> Experiment 2: Transformation of pSUTp. pG.pmt and pSUTp.
</LI></UL><PRE>                           (Can you please put the link to Experiment 2 here?)
</PRE><P>Experiment 1: Gel extraction for PCR product of Ag43
</P><P>Running PCR products of Ag43 collected from reaction1 (named PCR1) and 2 (named PCR2) (Click HERE for further details on 06/10/09 ) on 1% agarose gel.
Click HERE for the gel image. (File 4)
</P><P>Follow the procedure described in Quick Gel Extraction Kit Manual (Click HERE for more details)
</P><TABLE border="1"><CAPTION> Result: Nanodrop  of gel extraction ‘s final products:
</CAPTION><TBODY><TR><TH> Sample </TH><TH> Concentration (ng/ul)  </TH><TH> 260/280
</TH></TR><TR><TH> PCR1
</TH><TD> 40.04 </TD><TD> 1.99
</TD></TR><TR><TH> PCR2
</TH><TD> 27.05 </TD><TD> 1.96
</TD></TR></TBODY></TABLE><P>Experiment 2: Transformation of pSUTp. pG.pmt and pSUTp.
</P><P>The bacteria used for this transformation reaction was DH5α Max efficiency cells strain (chemical competent)
pSUTp. pG. pmt  and pSUTp solution collected from 02/10/09 (click HERE fore further details)                            (Can you please put the link to 02/10.09 nanodrop result here?) were diluted to 10ng/uL solution to use for the transformation reaction.
</P><P>Transformation control: pUC19
Click HERE for detailed protocol of transformation reaction.
While bacteria containing pSUTp.pG.pmt and pUC 19 were cultured on two separate Amp plates, bacteria with pSUTp was grown on Kan plate.
Incubate went overnight at 370C.
Results: Bacteria colonies were shown on plate in following morning
</P><P><B>06/10/09: PCR reaction of Ag43</B></P><P>The aim of this experiment was to find out the most efficient method to amplify Ag43 Plasmid with PCR reaction and also collect the products for further experiments.
</P><P>CLICK HERE for details of the protocol (File 3)
</P><TABLE border="1"><CAPTION> Results:
</CAPTION><TBODY><TR><TH> Sample </TH><TH> Concentration (ng/ul)  </TH><TH> 260/280
</TH></TR><TR><TH> M3 JM109 pSUTP. pG.pmt
</TH><TD> 11.58 </TD><TD> 1.62
</TD></TR><TR><TH> M4 JM109 pG.pmt
</TH><TD> 51.10 </TD><TD> 1.78
</TD></TR><TR><TH> M18 JM109 pSUTP
</TH><TD> 24.32 </TD><TD> 2.32
</TD></TR></TBODY></TABLE><P>
Primer preparation:
</P><P>Prepare the stocks of primers with concentration of 9.6 pmol/uL. Eight different primers (MerT seq1, MerT seq2, MerP seq1, MerP seq2, MT seq1, MT seq2) were diluted, following the protocol:
</P><P>Centrifuge samples at high speed for 30seconds.
Add nuclease free water to each sample as recorded on the label of the primer tube received from Sigma-Aldrich to obtain a 100uM solution.
</P><TABLE border="1"><CAPTION> Results:
</CAPTION><TBODY><TR><TH> Primer </TH><TH> Water volume (uL)  </TH><TH> Molecular Weigh
</TH></TR><TR><TH> MerT seq1
</TH><TD> 160 </TD><TD> 5781
</TD></TR><TR><TH> MerT seq2
</TH><TD> 149 </TD><TD> 5910
</TD></TR><TR><TH> MerP seq1
</TH><TD> 154 </TD><TD> 6100
</TD></TR><TR><TH> MerP seq2
</TH><TD> 164 </TD><TD> 5869
</TD></TR><TR><TH> MT seq1
</TH><TD> 155 </TD><TD> 6250
</TD></TR><TR><TH> MT seq2
</TH><TD> 159 </TD><TD> 6148
</TD></TR></TBODY></TABLE><UL><LI>Prepare 20uL of 10uM solution for each primer from their 100uM stocks.
</LI><LI>Prepare 20uL of 9.6uM solution for each primers by add 1.92uL of their 100uM stock to 18.08uL water à Used for DNA sequencing.
</LI><LI>Preprare solutions for DNA sequencing:
</LI></UL><P>3 samples with the listed component was prepared for sequencing.
</P><P>iGEM1: 10uL pG.pmt , 1uL MT seq1 (9.6uM) and 1uL MT seq2 (9.6uM)
iGEM2: 10uL pSUTp, 1uL MerT seq1 (9.6uM) and 1uL MerT seq2 (9.6uM)
iGEM3: 10uL pSUTp, 1uL MerT seq1 (9.6uM) and 1uL MerT seq2 (9.6uM).
We do not have the sequencing result , hence, pSUTp was chosen to continue the project. 
</P><P><B>02/10/09: Nanodrop Mer-plasmid miniprep and preparing MerT and MerP primers</B></P><TABLE border="1"><CAPTION> Results:
</CAPTION><TBODY><TR><TH> Sample </TH><TH> Concentration (ng/uL) </TH><TH> 260/280
</TH></TR><TR><TH> M3 JM109 pSUTP. pG.pmt
</TH><TD> 11.58 </TD><TD> 1.62
</TD></TR><TR><TH> M4 JM109 pG.pmt
</TH><TD> 51.10 </TD><TD> 1.78
</TD></TR><TR><TH> M18 JM109 pSUTP
</TH><TD> 24.32 </TD><TD> 2.32
</TD></TR></TBODY></TABLE><P><I>Primer preparation:</I></P><P>Stocks of primers with concentration of 9.6 pmol/uL were prepared. Eight different primers (MerT seq1, MerT seq2, MerP seq1, MerP seq2, MT seq1, MT seq2) were diluted, with the following protocol:
</P><P>Centrifuge samples at high speed for 30seconds.
Add nuclease free water to each sample as recorded on the label of the primer tube received from Sigma-Aldrich to obtain a 100uM solution.
</P><TABLE border="1"><CAPTION> Solution:
</CAPTION><TBODY><TR><TH> Primer </TH><TH> Water volume (uL)  </TH><TH> Molecular Weigh
</TH></TR><TR><TH> MerT seq1
</TH><TD> 160 </TD><TD> 5781
</TD></TR><TR><TH> MerT seq2
</TH><TD> 149 </TD><TD> 5910
</TD></TR><TR><TH> MerP seq1
</TH><TD> 154 </TD><TD> 6100
</TD></TR><TR><TH> MerP seq2
</TH><TD> 164 </TD><TD> 5869
</TD></TR><TR><TH> MT seq1
</TH><TD> 155 </TD><TD> 6250
</TD></TR><TR><TH> MT seq2
</TH><TD> 159 </TD><TD> 6148
</TD></TR></TBODY></TABLE><UL><LI>Prepare 20uL of 10uM solution for each primer from their 100uM stocks.
</LI><LI>Prepare 20uL of 9.6uM solution for each primer by adding 1.92uL of the 100uM stock to 18.08uL water. 
</LI><LI>Preprare solutions for DNA sequencing:
</LI></UL><P>3 samples with the listed components were prepared for sequencing.
</P><UL><LI>iGEM1: 10uL pG.pmt , 1uL MT seq1 (9.6uM) and 1uL MT seq2 (9.6uM)
</LI><LI>iGEM2: 10uL pSUTp, 1uL MerT seq1 (9.6uM) and 1uL MerT seq2 (9.6uM)
</LI><LI>iGEM3: 10uL pSUTp, 1uL MerT seq1 (9.6uM) and 1uL MerT seq2 (9.6uM).
</LI></UL><P>We do not have the sequencing result , hence, pSUTp was chosen to continue the project.
</P><P><B>24/09/09: Nanodrop and running Agarose gel for mini prep products from 23/09/09</B></P><P>A 1% agarose gel was set up with 1g agarose and 100ml TE buffer. The gel was run at 70V for 1 hour and post stained with ethidium bromide before visualisation.
</P><UL><LI>Lane 1: dying load (mixture of 5uL 1kb ladder + 1uL 6xLoading dye)
</LI></UL><UL><LI>Lane 2: pSUTP. pG. pmt (mixture of 10uL M3 JM109 pSUTP. pG.pmt + 2uL 6x loading dye).
</LI></UL><UL><LI>Lane 3: pG.pmt (mixture of 10uL M4 JM109 pG.pmt + 2uL 6x loading dye).
</LI></UL><UL><LI>Lane 4: pSUTP (mixture of 10uL M18 JM109 pSUTP + 2uL 6x loading dye).
</LI></UL><P>CLICK HERE for the picture of the gel (File 2).
</P><P><B>23/09/09: Mini prep Mer-gene containing bacteria</B></P><P>A mini prep was done to collect the plasmids that contain MerT and Mer P genes from the bacteria being grown on 24/09/09.
</P><P>Products were named corresponding to the plasmids expected , including M3 JM109 pSUTP. pG.pmt , M4 JM109 pG.pmt and M18 JM109 pSUTP.
</P><P>CLICK HERE for mini prep protocol.
</P><P>CLICK HERE for the result (Can u please put the link to 24/09/09 here?)
</P><P><B>22/09/09: Growing Mer-containing Bacteria</B></P><P>Three strains of bacteria containing MerT and MerP genes, including M3 JM109 pSUTP. pG.pmt (AmpR + KanR), M4 JM109 pG.pmt (AmpR) and M18 JM109 pSUT (AmpR) , were culture overnight at 370C on shaker.
</P><UL><LI>M18 JM109 pSUT : bacteria contains MerT and MerP gene  without introns.
</LI></UL><UL><LI>M4 JM109 pG.pmt : bacteria with plasmids containg both MerT and MerP with introns  in the sequence.
</LI></UL><UL><LI>M3 JM109 pSUTP. pG.pmt : bacteria containg both pSUTP and pG. pmt plasmid.
</LI></UL><P>CLICK HERE for details of medium preparation (File 1).
</P><P><I>Result:</I> There was growth of bacteria in all 3 tubes. However, the culture in tube 3 was less cloudy than the other.
</P><P><B>15/09/09 - Repeat Transformation of BBaI716101</B>
The overnight transformation from 14/09/09 was unsuccessful as no growth was observed on plates. It is probable that the cell death plasmid component (Part BBa_P1010) is functioning normally as it was believed that this component was 'faulty'. The same protocol as on 14/09/09 was used for the transformation with an extra positive control for comparison. A well characterised plasmid PUC18 (ampicillin resistance) was used as the positive control.
</P><P><B>14/09/09 - Transformation of Plate 1, Well C</B>
PLasmid BBa_I716101 with Part BBa_P1010 was transformed into XL-10 Gold Ultracompetent cells via heat shock and left in ampicillin plates at 37C overnight. DNA from plate 1, well C was stored in the Green iGEM box ('Fiona').</P><P><B>02/09/09 - Running Gel for Standard 23 </B>
2 gels were set up:
- A---&gt;1%, 140mL gel.Undigested and digested controls. Undigested/digested plasmid BBa_J63010 (Sample A, B and C)
- B---&gt;1%, 80mL gel. Undigested and Digested BBa_J63010 (Sample D,E) </P><P><B>01/09/09   - Agarose Gel and Nanodrop</B></P><UL><LI> Miniprepped BBa_J63010 plasmids ran on 1% Agarose gel (TAE buffer, pH8.5) at 200V, 70mA for 1hour. Post-stain: Ethidium Bromide. Click <A href="https://static.igem.org/mediawiki/2009/7/7e/UQ_-_Gel_1-09-09.tif" class="external text" rel="nofollow">HERE</A> for a picture of the gel. Lanes1,8 - empty, Lane2 - ladder, Lanes3-7 - Miniprepped DNA (colonies A-E)</LI><LI> Miniprepped BBa_J63010 plasmids (tubes A to E) analysed on nanodrop. (Data printout in IGEM_Mercury notebook)</LI><LI> New space created for BBa_J63010 tubes A to E: in &quot;Fiona&quot; freezer, UQ-Australia IGEM Team's <I>Green</I> box.</LI><LI> In the afternoon, digests of samples BBa_J63010 A-E (Standard 23) were run with EcoRI, AgeI, SpeI and PstI enzymes. Unfortunately NgoMIV enzymes were not in stock in the freezer so this digest could not be performed.
</LI><LI> The Standard 21 plasmid BBa_I716101 was again digested with PstI. All samples were left in the 37 degrees water bath overnight.</LI></UL><P><B>28/08/09   - DNA miniprep</B>
Mini prep DNA extractions performed for BBa_J63010 plasmid selected yesterday and incubated overnight.
Tubes stored in &quot;Fiona&quot; freezer, UQ-Australia IGEM Team's orange box.</P><PRE><I>EDIT (01/09/09): these tubes now in IGEM</I><I>Green Box (&quot;Fiona&quot; freezer, bottom shelf).</I></PRE><P><B>27/08/09   - Colony Picking</B>
5 colonies were picked and grown in ?mL of LB Broth and were left in the 37C incubator overnight. Samples were labelled 'BBa_J63010 A,B,C,D,E'</P><P><B>26/08/09   - Transformation of Plasmid BBa_J63010 with part BBa_J04450</B>
BBa_J04450 was found in plate 1, well C. Colonies were grown overnight and will be picked and grown tomorrow.</P><P>---&gt;<B>Digest Gel of BBa_I716101 with pGfa2cLac2 control</B>
A 1% agarose gel was run with 1kb ladder and samples with EcoRI, XhoI, BglII and BamHI restriction enzymes. The gel visualisation (image here) indicates that only EcoRI was able to slice the plasmid; it is possible that the other restriction sites are not on the plasmid.</P><P><B>25/08/09   - In-gel Extraction and Overnight Digests</B>
20 chloroamphenicol plates were prepared with LB agar. BBa_I716101 and control plasmid pGfa2cLac2 were each digested with EcoRI, XhoI, BglII and BamHI enzymes (total of 8 samples). Digests have been left to run overnight in the 37C waterbath.
</P><P><B>24/07/09   - DNA extraction and electrophoresis</B>
Mini prep DNA extractions performed for MS427-pBAD, MS427-pKKJ143 cultures incubated overnight.
--&gt; run on ethidium bromide gel to confirm DNA production.
Click <A href="https://2009.igem.org/Image:UQ_MercuryMiniprep_AgaroseGel_1.png" class="external text" rel="nofollow">HERE</A> for a picture of the gel.
</P><P><B>23/07/09   - Transformations with AG43 plasmids</B><I>E. coli</I> incubated overnight in duplicate under the following conditions:</P><PRE>Strain: MS427 
Plasmid: pBAD (empty plasmid -AG43) 
- 2mL LB broth 
- 2µL ampicillin </PRE><PRE>Strain: MS427 
Plasmid: pKKJ143 (plasmid w/ AG43) 
- 2mL LB broth 
- 2µL ampicillin</PRE><P><B>20/08/09  - Plasmid Digestion</B></P><P><B>19/08/09  - Miniprepping and Running PCR Gel + Nanodrop</B></P><P><B>18/08/09  - PCR Preparation and Picking Cultures</B></P><P><B>24/07/09  - Extraction of DNA from E.coli M5427</B></P><P><B>17/07/09  - MG1655 Stock Preparation</B>
MG1655 <I>E. coli</I> grown on pure LB stock. No growth was observed using LB + ampicillin medium.
Pelleted and resuspended in 50:50 glycerol:LB medium and stored at -80 degrees C.
---&gt;</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:UQ-Australia/Notebook/Project1">http://2009.igem.org/Team:UQ-Australia/Notebook/Project1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UQ-Australia/Notebook/Project1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UQ-Australia/Notebook/Project1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UQ-Australia/Notebook/Project1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UQ-Australia/Notebook/Project1&amp;oldid=158291" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>