Team:BNU-China/Notebook
Team:BNU-CHINA - 2016.igem.org
Toggle navigation
BNU-China
Home
Project
Background
Design
Results
Improvement
Modeling
Introduction
Modeling theory
Single Factor Analysis
Differential Equation
Visual Simulation
Achievements
Parts
Judging
Practices
Schedule
Human Practice
Safety
Team
Members
Attributions
Collaborations
Acknowledgement
Notebook
Protocol
Notebook
Notebook
Week 1
Wet Lab
We chose the plasmid of kit plate to transform and amplify. We designed parts by adding promoter (BBa_K206000) and RBS(BBa_B0034) to the upstream region of luciferase gene and adding prefix and suffix and we committed the GENEWIZ company to synthesize this gene.
We gathered and summarized the data of algal bloom case in North China.
Week 2
Wet Lab
We transformed the plasmid pSB1C3 (BBa_K592009/BBa_E1010/BBa_J04450) in DH5α and proceeded amplification, extracted the plasmid and used two kinds of enzyme (EcoR I, Pst I) to digest and verify.
We designed primers and committed the GENEWIZ company to synthesize these primers (nluc, nluc-pSB1C3, cluc, cluc-pSB1C3, β-tubulin, β-tubulin-pSB1C3, α-tubulin, α-tubulin-pSB1C3).
We borrowed expression plasmid pET21a(+) from the lab of Yang Dong to amplify and DNA template (nluc, cluc, mcf-7, hepg-2), and we amplified these fragments (nluc, cluc, α-tubulin, β-tubulin) by PCR.
Dry Lab
We guided FAFU about modeling.
Week 3
Wet Lab
We designed primers about TOPO (α-tubulin-TOPO-F, β-tubulin-TOPO-F, α-pET21a(+), β-pET21a(+), nluc-pET21a(+), cluc-pET21a(+)) and committed the GENEWIZ company to synthesize these primers.
We tested amplification products by PCR, and linked them to produce plasmids pSB1C3 and plasmids pET21a(+). Meanwhile we did topo clone.
We did PCR amplification by using efficient polymerase, did restriction enzyme (EcoR I/Xho I) digestion and ligation about PCR products(α-TOPO, β-TOPO, α-pET21a(+), β-pET21a(+), nluc-pET21a(+), cluc-pET21a(+)).
Dry Lab
We modified basic logistic growth curve combined with specific growing environment of Algae through reviewing literatures about Algae growthing modeling.
Week 4
Wet Lab
We did PCR amplification again by using efficient polymerase of TransgenTM company, digestion and ligation. (α- pET21a(+), β-pET21a(+), nluc-pET21a(+), cluc -pET21a(+))
We did PCR amplification again by using efficient polymerase (primeSTAR) of TAKARA company, digestion, ligation and transformation (α-pET21a(+), β-pET21a(+), nluc-pET21a(+), cluc -pET21a(+)).
We did bacterial colony PCR about transformation products. We inoculated 15 mL centrifuge tubes with the monoclonal colony to produce bacterium solution for plasmid extraction.
We transformed ligation product and plasmid extraction to BL21 to express.
Dry Lab
We built differential equation model about light, CO2, mineral substance, organic substance.
Week 5
Wet Lab
We exchanged plasmid pET30a(+).
We did PCR amplification again, digestion. We recycled gene segments following the agarose gel DNA extraction kit, then ligated.
We inoculated 15 mL centrifuge tubes with the monoclonal colony (YNE, YCE) to produce bacterium solution for plasmid extraction, digested(YNE: Xba I/Spe I; YCE: Xho I/Sac I) and We recycled gene segments following the agarose gel DNA extraction kit, then ligated.
Dry Lab
We applied MATLAB numerical calculation for differential equation model of FAFU, discussed its practical application and deficiency through video session with FAFU team member.
Week 6
Wet Lab
We extracted β plasmid and did bacterial colony PCR.
We designed primer of α, did PCR amplification again, changed cleavage sites, digested (Hind III/Xho I), ligated.
We committed company to do sequencing about PCR products (α-pET30a(+), β-pET30a(+), nluc-pET30a(+), cluc-pET30a(+)).
Others digested again (pET30a(+), β-pET30a(+), nluc-pET30a(+), cluc-pET30a(+)). We recycled the plasmid backbone (pET30a(+)) following the agarose gel DNA extraction kit, and gene segments following the DNA fragment purification kit, ligated, transformed, extracted plasmid, did restriction enzyme digestion and PCR to check the extracted plasmid.
Dry Lab
We discussed the application value of one-way analysis of variance for biological experiment.
Week 7
We inoculated 15 mL centrifuge tubes with the transformation product to produce bacterium solution for plasmid extraction, measured concentration through quantitated agarose gel, then did restriction enzyme digestion and PCR to check the extracted plasmid.
We transformed ligation products on 17/8 (α-tubulin, α-hep, β-pET30(+), nluc-pET30a(+), cluc–pET30a(+)) and plasmid pET30a(+) into transB expression strain.
Week 8
We designed primer and committed company to synthesis primer. We extracted plasmid, did PCR amplification (nluc, cluc, α, β), digested (pET30a(+), α, β(Hind III/Xho I); pET30a(+), nluc, cluc(EcoR I/Xho I)), ligated, transformed.
We planned to use MATLAB to do light intensity analysis after sample picture grey processing to match the numerical relationship between taxol concentration and light intensity of glowing.
Week 9
Wet Lab
We inoculated 15 mL centrifuge tubes with the transformation product to produce bacterium solution for plasmid extraction, then did PCR to check the extracted plasmid.
We did fusion PCR, digested, ligated, transformed.
Dry Lab
We reviewed literatures about biodynamics modeling of microtubule.
Week 10
Wet Lab
After the result of sequencing is correct, we inoculated 15 mL centrifuge tubes with the ligation product to produce bacterium solution for expression.
We inoculated 15 mL centrifuge tubes with the transformation product to produce bacterium solution for plasmid extraction, did PCR exam. We committed company to do sequencing which has correct result (α-YNE, α-YCE, YCE-α, cluc-α, β-YNE, α-cluc, β-YCE, YCE-β). The result of α-YNE, α-YCE, YCE-α, β-YCE, YCE-β, cluc-α is correct.
We did SDS-PAGE to test the expression protein.
LR reaction.
Dry Lab
We studied the taxol’s influence in the dynamic assembly process.
Week 11
Wet Lab
We did SDS-PAGE to test the expression protein.
We promoted parts: we extracted plasmid in kit 2 and did enzyme digestion. We chose BBa_K934001 to assemble parts by enzyme digestion, ligation, transformation and plasmid extraction and separately ligated with the Arab sugar induced promoter(BBa_10500) and the heat sensitive promoter(BBa_K873002).
We re-did fusion PCR, and the α-nluc had the objective band. Then we sequenced gene.
We used the product of the LR reaction β-nluc to do bacterial colony PCR, and it had a high positive rate. So we did plasmid extraction(β-nluc, cluc-β), enzyme digestion and ligation.
Dry Lab
We exchaged ideas for the development of the dynamical model microtubule dynamic instability.
The mechanism of the taxol in the microtubules dynamic assembly process had important significance. In order to well study the taxol's influence in the dynamic assembly process, we divided into four steps on the dynamic process modeling.
Week 12
Wet Lab
We assembled standard carrier: we ligated the target gene which had been assembled in pET30a(+) with pSB1C3 by PCR, enzyme digestion, and transformation.
Now
