<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BNU-China_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BNU-China/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:BNU-CHINA - 2016.igem.org</TITLE></P><DIV class="header-container"><NAV class="navbar navbar-fixed-top navbar-inverse nav-adjust" role="navigation" style="margin-top: 14px;"><DIV class="container-fluid nav-container scrolled-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" id="navbar-toggle"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" id="navbar-brand" href="https://2016.igem.org/Team:BNU-China">BNU-China</A></DIV><DIV class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:BNU-China" class="">Home</A></LI><LI class="dropdown "><A href="https://2016.igem.org/Team:BNU-China/Project" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Project">Background</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Proof">Results</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Description">Improvement</A></LI></UL><LI class="dropdown "><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling </A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Model">Introduction</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Model#model-theory">Modeling theory</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Model#model-factor">Single Factor Analysis</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Model#model-differential">Differential Equation</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Model#model-visual">Visual Simulation</A></LI><LI class="dropdown "><A href="./achievement.html" class="dropdown-toggle" data-toggle="dropdown">Achievements </A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Achievements#Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Achievements#Judging">Judging</A></LI><LI class="dropdown "><A href="#" class="dropdown-toggle" data-toggle="dropdown">Practices </A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Human_Practices">Schedule</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Practices">Human Practice</A></LI><LI class=""><A href="https://2016.igem.org/Team:BNU-China/Safety">Safety</A></LI><LI class="dropdown focus"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Team#">Members</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:BNU-China/Protocol">Protocol</A></LI></DIV></DIV></NAV></DIV><DIV class="main-container"><DIV id="page-heading" class="container-fluid page-heading" style="background-image: url(https://static.igem.org/mediawiki/2016/9/96/T--BNU-China--team.jpg);"><H3> Collaborations</H3></DIV><DIV><DIV class="page-story"><ARTICLE id="collaborations" class="col-lg-10 col-lg-offset-1 col-md-12 col-md-offset-0 col-sm-offset-0 col-sm-12"><HEADER class="page-header"><H1>Collaborations</H1></HEADER><H2>We helped others - the modeling</H2><H3>Introduction</H3><P> It’s essential for us to get accurate growth condition of <I>Chlamydomonas reinhardtii</I> in the natural environment to keep the concentration of toxin at a lethal level. But in fact, it is almost impossible to test concentration anywhere due to the lack of equipment and skills. Therefore, building the growth model can help determine the amount of Chlamydomonas <I>reinhardtii</I> they should use and when they need to add more. To build an accurate growth model, BNU-China team members who have much experience in the mathematics helped us to achieve it.</P><P>Contacting with data provided by wet laboratory, we can draw the diagram of variation trend of algae population. Then we can get the key point where rate of algae population increment meets the maximal value so that the results can guide to culture of algae in their wet part. To control quantity of aquatic larva of mosquito by applying expression of specific protein in algae. There is an impressive impact of establishing mathematic modeling in population of algae.</P><P> They helped us to establish a mathematic model to illustrate the whole temporal change of algae population. In general, it’s an original differential equations based on light intensity, mineral nutrient, organism and carbon dioxide, which are four main parameters in that. As for the temporal changing rate of population of algae growing in ideal conditions, there has been a lot of methods to solve this question. They referred to Huisman model and combined with practice factors. Then we got our deducted model. This model has a few parameters and it’s easy to get the solution.</P><P> We provided the data of wet laboratory for us, and they run the model to get result. Finally, these results can help us to complete experiment. </P><H3>Hypothesis of model</H3><P> The fundamental way that the algae grow is through photosynthesis, in which the inorganic carbon in the water (carbon dioxide) can be transformed into the organic carbon (carbohydrate). However, the photosynthesis of the algae, which is fixed in the water, is influenced and limited by lots of factor.</P><P> Not only in the laboratory but also the factory, Culturing algae in the stirred-well mixed culturing vessel with fixed volume is a common way. Under the circumstances, we believe that those important parameters, which is related to the growth of the algae, are all isotropy. In another word, by only considering the result which is dependent on time, we can meet the requirements of the experiments, or even the production.</P><P> Thus, after weighing up the actual conditions, based on the combination of the existing model of the first order ordinary differential equations, we set up some second order ordinary differential equations to simulate the growth velocity - the algae’s dry weight of the time derivative. Furthermore, the results, which the model has simulated the growth condition of the algae in limited light and nutritive substance, quite tally with the actual situation. In some limiting cases, such as limited light with unlimited nutritive substance or unlimited light with limited nutritive substance, these equations can be simplified into some common first order ordinary differential equations.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.1 The logical relationships among four parameters
                    </FIGCAPTION></FIGURE><P> Firstly, according to the flow figure below, we explain the logical relationship of the four important parameters of the model. </P><P> The growing speed of the alga in the incubator, A, which means the change of dry weight of the alga in unit interval is equal to the increase minus the decrease of the organic substance in the cells. The decrease is mainly based on two ways. One is the natural death and the other is the artificial separation of the useful mature alga.</P><P> The carbon dioxide in the water area is converted to saccharides by the photosynthesis of alga and then stored in the cells. This also shows the conclusion that the growth of the population density of the alga will accelerate the speed of the decrease of carbon dioxide. Hence the content of carbon is equal to the inflow minus the part which are converted by photosynthesis.</P><P> Similarly, the content of mineral substance in the water area M is also a factor which has its influence on photosynthesis. It will decrease faster when the amount of the alga is increasing as well.</P><P> In a certain space, the sum of the carbohydrate, S (exclusive that in the cell), can be considered as the result which the fixed sum of the photosynthesis subtracts the total sum of the increment of the dry weight and the decrement of the dry weight in the culturing medium (death, artificial extraction).</P><P> The quantitative relationships among four parameters shows below:</P><P> $$\dot{\mathcal{A}}=\alpha_{\dot{\mathcal{A}}}f_m(M)S-(D_r+h_0)\dot{\mathcal{A}}$$</P><P>$$\dot M = -k_2 \alpha_{\dot{\mathcal{A}}}f_m (M)S+I_m(t)$$</P><P>$$\dot S=\alpha_S C-k_3\alpha_{\mathcal{A}}f_m(M)S-(D_r+h_0)S$$</P><P>$$\dot C=-k_1\alpha_s C+I_c(t)$$</P><H3>Mathematic formulation</H3>
               According to the expression of photosynthesis which is the reaction of photosynthesis as following:

                       <P> $$6CO_2+6H_2O\longrightarrow(CH_2O)_6+6O_2$$</P><P>We know that the coefficient of the carbon dioxide which is consumed by carbohydrates is \(\frac{44}{33}\ g[CO_2]\ g[CH_2O]^{-1}\). If the increment of the carbohydrates’ dry weight is influenced only by the content of the organic matters and mineral substance, and the proportion of the two is 1:9 approximately, the coefficient of the consumed mineral substance, k2, is: </P><P>$$k_2=0.1\frac{g[M]}{g[\mathcal{A}]}$$</P><P>The coefficient of the consumed organic matters, k3, is:</P><P>$$k_3=0.9\cdot \frac{g[(CH_2 O)_6]}{g[\mathcal{A}]}$$</P><P>From the above, what can be seen is that the main idea of these ordinary differential equations is element conservation. They work out the growth velocity indirectly by analyzing the transform of the substances in the fixed space.</P><H3>Results</H3><FIGURE class="text-center"><FIGCAPTION>
                        Fig.2 The logical relationships among four parameters
                    </FIGCAPTION></FIGURE><H2>The help we received – experiments helped by FAFU</H2><H3>Introduction</H3><P>We want to detect paclitaxel by using microtubule which expressed in <I>E.coli</I>. As we all know, microtubule is the cytoskeleton in eukaryotes which is assembled by α-tubulin and β-tubulin, but <I>E.coli</I> is the prokaryote, so it is very important to verify whether microtubule assemble successfully or not in <I>E.coli</I>. If microtubule assembles successfully, it means α-tubulin and β-tubulin have biological activities and have interaction in <I>E.coli</I>. So, FAFU-CHINA plan to use Co-Immunoprecipitation(CoIP) to confirm their interaction.</P><H3>Design</H3><P>Since pET30a(+) has His protein tag, they plan to add HA protein tag and Flag protein tag to the down-stream of α-tubulin and β-tubulin respectively . The target fragments are amplified by PCR and cloned to T vector and sequenced. Then they link the confirmed gene to pET30a(+) (enzyme site: XhoI, HindIII) .After that，they transform the recombination vector into expression strains(BL21)for culturing. When the concentration of bacterium is appropriate (OD is 0.6-0.8), they induce the strains by 1 mmol IPTG. By using ultrasonic waves to break the cell of bacteria and centrifuging, they obtain the pellet and deal with the inclusion body for the further experiment of Co-Immunoprecipitation(CoIP).</P><H3>Results</H3><H4>Primer design</H4><P>FAFU-CHINA looked up αandβtublin seqences via NCBI GenBank and downloaded  cDNA sequences for primer design(table 1).</P><P style="text-align: center">Table 1  Primers for αandβtublin expression vectors</P><TABLE class="table table-bordered"><THEAD><TR><TH style="text-align:center">Primer name</TH><TH style="text-align:center">Primer sequence</TH><TH style="text-align:center">Enzyme site</TH></TR></THEAD><TBODY id="RN"><TR><TD align="center">α-tubulin-HA-S</TD><TD align="center">5'AAGCTTATGCGTGAGTGCATCTCCATCCACGTTG 3'</TD><TD align="center">HindIII</TD></TR><TR><TD align="center">α-tubulin-HA-A</TD><TD align="center">5'GTTAAGCGTAGTCTGGGACGTCGTATGGGTAGTATTCCTCTCCTTCTTCCTCACCCTC 3'</TD><TD align="center">XhoI</TD></TR><TR><TD align="center">β-tubulin-Flag-S</TD><TD align="center">5' AAGCTTATGAGGGAAATCGTGCACATCCAGGCTGG  3'</TD><TD align="center">HindIII</TD></TR><TR><TD align="center">β-tubulin-Flag-A</TD><TD align="center">5'CTCGAGTTACTTATCGTCGTCATCCTTGTAATCGGCCTCCTCTTCGGCCTCCTC 3'</TD><TD align="center">XhoI</TD></TR></TBODY></TABLE><H4>Gene amplification via PCR </H4><P>They amplified α-tubulin-HA, β-tubulin- Flag sequences based on the plasmids provided by BNU-China. Electrophoresis result showed the fragments were all in their correct sizes. Sequencing result furthered verified that target genes were successfully cloned.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.3 Electrophoresis result of α-tubulin-HA, β-tubulin- Flag fragments
                    </FIGCAPTION></FIGURE><P>They ligated the gene fragments to expression vectors and did dual digestions for verification. Electrophoresis result (Fig.2) showed 3 different bands (up to down: dual digested plasmid, non/single digested plasmid, gene fragment), indicating that target genes were ligated to the expression vectors successfully.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.4 Dual digestion of the recombinant vector 
                         <SMALL style="font-size:66%">left to right: marker, α-HA-PET30a, β-flag-PET30a  
                    </SMALL></FIGCAPTION></FIGURE><H4>Transformation of the target plasmids</H4><P>They transformed the target plasmids into BL21 strain. Colony PCR verified that target fragments were existed in the colonies.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.5 Electrophoresis result of colony PCR
                         <SMALL style="font-size:66%">A: verification of α-HA-PET30a plasmid transformation   B: verification of β-flag- PET30a plasmid transformation</SMALL></FIGCAPTION></FIGURE><DIV class="reference"><OL><LI>Jayaraman S K, Rhinehart R R. Modeling and Optimization of Algae Growth[J]. Industrial &amp; Engineering Chemistry Research, 2010, 54(33)</LI></OL></DIV></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>