<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Debrecen-Hungary_project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Debrecen-Hungary/project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Debrecen-Hungary/project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Debrecen-Hungary/project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Debrecen-Hungary/project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Debrecen-Hungary/project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Debrecen-Hungary/project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><TABLE><TBODY><TR valign="top" style="margin-right: 0px;"><TD width="1100px" style="padding: 0 0px 0px 0px; background-color:#ede8e2"><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team" style="color: white"> Team</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/meetteam"><SPAN><SPAN>Meet the Team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/gallery"><SPAN><SPAN>Gallery</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team#about"><SPAN><SPAN>About The team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team#Lab"><SPAN><SPAN>About The Lab</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team#deoec"><SPAN><SPAN>About The University</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team#debrecen"><SPAN><SPAN>About  Debrecen</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team#cube"><SPAN><SPAN>About  Rubiks's Cube</SPAN></SPAN></A></LI></UL><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project" style="color: white"> Project</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project#abs"><SPAN><SPAN>Abstract</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project#intro"><SPAN><SPAN>Introduction</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project#Results"><SPAN><SPAN>Results</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project#Plans"><SPAN><SPAN>Plans, Possibilities</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project#BBF_RFC_64_:_Building_Protein_Domain_Based_Composite_Biobricks_for_Mammalian_Expression_Systems"><SPAN><SPAN>RFC 64 suggestion</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/indicontibi"><SPAN><SPAN>Members Contribution</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts" style="color: white"> Parts</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#LBD"><SPAN><SPAN>LBD</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#DBD"><SPAN><SPAN>DBD</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#other"><SPAN><SPAN>Other basic parts</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#composite"><SPAN><SPAN>Composite Parts</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/favorite"><SPAN><SPAN>Favorite Parts</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/protocols" style="color: white"> Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/protocols#Contents"><SPAN><SPAN>Cloning Team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/protocols#Mini_Prep"><SPAN><SPAN>Molecular work team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/protocols#Gel_electrophoresis"><SPAN><SPAN>Tissue culture team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/protocols#Measuring_Luciferase_activity_with_the_Victor_plate_reader"><SPAN><SPAN>Luciferase Team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/sponsors" style="color: white"> Sponsors</A></LI><LI><A style="color: white"> Tools</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/sitemap"><SPAN><SPAN>Site Map</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals"><SPAN><SPAN>The Minimals</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/safety"><SPAN><SPAN>Safety</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team_meetings"><SPAN><SPAN>Team Meetings</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/links"><SPAN><SPAN>Links</SPAN></SPAN></A></LI><LI><A style="color: white"> Side Projects</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/arsenic"><SPAN><SPAN>Arsenic Project</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/AKG_team"><SPAN><SPAN>AKG team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/video_project"><SPAN><SPAN>Video Project</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/on_the_media"><SPAN><SPAN>On The Media</SPAN></SPAN></A></LI></DIV></TD></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Abstract"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Synthetic_Biology"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Synthetic Biology</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#New_Tools_for_Synthetic_Biology:_Nuclear_Receptors"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">New Tools for Synthetic Biology: Nuclear Receptors</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Cloning"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Basic_Parts"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">Basic Parts</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Composite_Parts"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">Composite Parts</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#Gal4-human_LBD_fusions"><SPAN class="tocnumber">3.1.2.1</SPAN><SPAN class="toctext">Gal4-human LBD fusions</SPAN></A></LI><LI class="toclevel-4 tocsection-10"><A href="#Gal4-Ecdysone_receptor_fusion"><SPAN class="tocnumber">3.1.2.2</SPAN><SPAN class="toctext">Gal4-Ecdysone receptor fusion</SPAN></A></LI><LI class="toclevel-4 tocsection-11"><A href="#Gal4-C._elegans_LBD_fusions"><SPAN class="tocnumber">3.1.2.3</SPAN><SPAN class="toctext">Gal4-C. elegans LBD fusions</SPAN></A></LI><LI class="toclevel-4 tocsection-12"><A href="#Estrogen_Receptor_from_basic_parts"><SPAN class="tocnumber">3.1.2.4</SPAN><SPAN class="toctext">Estrogen Receptor from basic parts</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Creating_expression_vector_from_pSB1A3"><SPAN class="tocnumber">3.1.3</SPAN><SPAN class="toctext">Creating expression vector from pSB1A3</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Western_Blots_of_Gal-fused_proteins"><SPAN class="tocnumber">3.1.4</SPAN><SPAN class="toctext">Western Blots of Gal-fused proteins</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Transients_cotransfection_assay_.2Ftwo-hybrid_system.2F"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Transients cotransfection assay /two-hybrid system/</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Activation_experiments"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Activation experiments</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Two_hybrid_system"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">Two hybrid system</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#BBF_RFC_64_:_Building_Protein_Domain_Based_Composite_Biobricks_for_Mammalian_Expression_Systems"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">BBF RFC 64 : Building Protein Domain Based Composite Biobricks for Mammalian Expression Systems</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Plans.2C_Possibilities"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Plans, Possibilities</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Contibution_to_project_by_team_members"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Contibution to project by team members</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Abstract"><B>Abstract</B></SPAN></H1><P>Eukaryotic synthetic biology has huge potential, yet it is still in need of more diverse molecular tools for defined gene regulation. 
Nuclear receptors are a conserved family of proteins responsible for sensing lipids; they may be viewed as lipid activated transcription factors. 
<B>We have successfully developed a kit with a variety of lipid responsive domains</B> (from H.sapiens, D.melanogaster and C.elegans) <B>for the rational construction of synthetic transcription factors.</B> The domains respond only to predefined lipids and selectively activate predetermined gene expression. 
To characterize theses domains, we used standardized protocols for comparable measurements. In vivo gene expression was measured as a function of ligand concentration using luciferase activity.
</P><P><B>The potential for these tools is immense; e.g. from the ultra sensitive detection of lipid contaminants in the environment to the opportunity of titration specific gene expression canges in patients undergoing gene therapy. Regarding applications in food quality control assessment, these system allows the measurement of Vitamin A, Vitamin A derivatives through RAR (retinoic acid receptor),  vitamin D and its derivatives through VDR (vitamin D receptor), but also the detection of steroids (antiinflamatory steroids, anabolic steroids, estrogens) through GR (glucocorticoid receptor), ER (estrogen receptors), AR (testosteron receptor). By simply copying the xenobiotic receptors of humans (e.g. PXR, CAR) we can assess the presence of antibiotics in food or wheter existing or new food additives will activate our endogenous xenobiotic system.</B></P><P><B>In summary the Nuclear Receptor Toolkit developped by us allows the (1) ultra sensitive detection of xenobiotic activators in the environment (e.g. oil derived traces in water and soil), (2)  the assessment of lipid soluble vitamins, steroids, xenobiotics in food and (3) the potential of inducing specific genetic programs in transfected cells or modell organisms with a possibility to destroy these modified cells by an independent tetracyclin induced apoptosis effector system.</B></P><H1><SPAN class="mw-headline" id="Introduction"><B>Introduction</B></SPAN></H1><H2><SPAN class="mw-headline" id="Synthetic_Biology"><I>Synthetic Biology</I></SPAN></H2><P><B>Synthetic biology</B> is a relatively new area of biological research; similar to many other new scientific fields it has many definitions. The best way to express the meaning of synthetic biology is to understand the desired end-result: <I>engineering of complex biological systems</I>. These systems are best thought of as analogues of everyday machinery: cogwheels, levers, timers, button and buzzers (in this case a clock), only in the case of biological systems (molecular ones) cells, DNA, proteins, lipids, sugars and RNA are the “parts” of the system.  
</P><P>Similar to mechanical engineering (or every other engineering branch) there is a need for standards (consensus way of doing things), abstraction (simple and unified way of thinking about the parts of a system), and modularity (how these parts interact to become the device, or several devices into a system). Thus a good definition for synthetic biology could be engineering of molecular (for the time being) biological systems <B>according to preset standard parts</B>. 
The international genetically engineered machines and associated parts registry are, to date, one of the largest registries for standard parts in use for synthetic biology. Free information can be found in the registry regarding parts, devices and modules all inputted by various teams worldwide.
</P><P><A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals#Drosophila_Melanogaster" class="external text" rel="nofollow">Eukaryotic synthetic biology</A> is still in its infancy. The large kingdom of metazoans includes all multicellular eukaryotes such as mammalians, arthropods and nematodes. No standard chassis (framework) exists for the animal kingdom which makes them far less popular then the famous E.coli. The protein modules derived from metazoans (like Drosophila or C. elegans) are functional in yeasts also. Very few iGEM teams (or even labs outside iGEM) have chosen to toggle the animal chassis (two notable examples are team Heidelberg 2009 and team Slovenia 2006).  The amount of available compatible parts is limited, which severely restricts the options of creating complex biological devices. Nearly no imagination is required for designing tools, since their analogues already exist in the bacterial chassis. The possible use of such systems is unlimited. Field’s such as of environment, medicine, energy and research all gain to profit from the development of animal synthetic biology.
</P><P>Systems requiring gene expression input in eukaryotic synthetic biology systems require a way to standardize gene expression, a complicated task. The way from gene to protein contains many steps of possible error: transcription factor binding, promoter strength, recruitment of auxiliary proteins, nuclear RNA synthesis and many more steps finally leading to translation, folding, cleaving and delivery (but hey, you have to start somewhere). 
</P><P><B>Our team was interested at designing eukaryotic synthetic biology tools related to PoPs</B>. PoPs (polymerase per second), the flying Dutchman of synthetic biology, is a number which represents the rate (base pair per second) at which RNA polymerase crosses past a given DNA position. Currently, no in vivo technique for measuring PoPS directly exists; it can be estimated indirectly by measuring other parameters (eg protein expression or enzyme activity). Nevertheless it is still a useful abstraction for thinking about transcription-based logic devices and it allows the engineer to define devices. Our aim, was not only to infer PoPs but to devise a way to titrate it remotely.
</P><H2><SPAN class="mw-headline" id="New_Tools_for_Synthetic_Biology:_Nuclear_Receptors"><I>New Tools for Synthetic Biology: Nuclear Receptors</I></SPAN></H2><P><A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals#Scientific_Background" class="external text" rel="nofollow">Nuclear receptors</A> (click to find out about nuclear receptors), best viewed as transcription factors which can be activated by extracellular cues, are unique in their ability to allow direct remote PoPs titration (both activation and repression). These receptor classes bears high homology to each other throughout the animal kingdom and are modular into distinct domains. All of these features attracted our attention to find a way to incorporate these tools in the parts registry, and characterize them in standardized methods.
</P><P>Many nuclear receptors, are among the primary targets of drug discovery because of their diverse biological actions. Nuclear receptors bear high homology to each other and are modular into distinct domains: N-terminal regulatory domain, DNA-binding domain (DBD), a Hinge region, Ligand binding domain (LBD) and a C-terminal domain. <A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals#Scientific_Background" class="external text" rel="nofollow">The LBD</A> (click to find out more about LBD's) is the segment of the receptor which changes its conformation upon lipophilic ligand binding; this also causes the exposure of a powerful transcriptional activation domain (which attracts the transcriptional machinery). This was our segment of interest since it links ligand binding with transcriptional activity. 
</P><P><B>Our team has generated a library of ligand binding domains</B> for the rational construction of synthetic titrateable transcription factors.  In our model hybrid receptor, this LBD was <B>fused to a Gal4 DNA binding domain</B>.  The sequence to which Gal4 binds is CGG-N11-CCG, where N can be any base. Although Gal4 is a yeast protein not normally present in other organisms it has been shown to work as a transcription factor in a variety of organisms such as Drosophila, and human cells, highlighting that the same mechanisms for gene expression have been conserved over the course of evolution.  
</P><P>The examination of these chimeric receptors was through a version of the <A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals#Cotransfection_Assay_Two_Hybrid_Assay" class="external text" rel="nofollow">two hybrid assay</A> (click to find out more about the two hybrid assay). For a more detailed description of our experimental system, please visit our <A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals#Cotransfection_Assay_Two_Hybrid_Assay" class="external text" rel="nofollow">&quot;Minimals&quot;</A> page.
</P><H1><SPAN class="mw-headline" id="Results"><B>Results</B></SPAN></H1><P>First we had to <B>generate expression vectors</B> that can be used in mammalian expression systems. The cloning work of Team Debrecen-Hungary resulted <A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#Ligand_binding_domains_-_Basic_Parts" class="external text" rel="nofollow">18 basic parts</A> in pSB1C3 and <A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#Chimeric_Nuclear_Receptors_-_Composite_Parts" class="external text" rel="nofollow">13 composite parts</A> in pSB1C3 or pSB1A3. All of them have been tested by restriction enzyme analysis and by sequencing. Furthermore, some basic and fused parts have been ligated into pCDNA3.1 containing a specific adaptor by which RFC-25 compatible parts can be inserted into this expression vector. pCDNA3.1 results constitutive expression of the inserted part in the cells so the investigated part can be tested at functional level. The other <A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#Eukaryotic_expression_vectors_-_Composite_Parts" class="external text" rel="nofollow">expression vector</A> is a fully SB (Synthetic Biology) compatible mammalian expression vector generated from pSB1A3. We inserted into the pSB1A3 the TRE (tetracycline response element) a strong CMV (cytomegalo virus) promoter and a PolyA element that is a terminator of transcription in mammalian cells. These four elements are available as individual parts in the parts registry and can be used in combination with other parts too.  
</P><H2><SPAN class="mw-headline" id="Cloning"><B>Cloning</B></SPAN></H2><H3><SPAN class="mw-headline" id="Basic_Parts">Basic Parts</SPAN></H3><P><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#Ligand_binding_domains_-_Basic_Parts" class="external text" rel="nofollow"><B>18 basic parts</B></A> have been inserted into pSB1C3 which contain 3 DBDs, 1 hinge, 11 LBDs, 2 regulators and 1 apoptosis protein. A Gal4 DBD which has the RFC25 N terminal linkers was designed and submitted to be used (BBa_K364319).
</P><H3><SPAN class="mw-headline" id="Composite_Parts">Composite Parts</SPAN></H3><P><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#Chimeric_Nuclear_Receptors_-_Composite_Parts" class="external text" rel="nofollow"><B>13 composite parts</B></A> have been created, ten of them consist of two basic parts, one of them has been built up from three basic parts, and an expression vector from standard biobrick parts has been created by using four basic parts. The backbone has been opened with AgeI and PstI restriction enzymes in order to ligate the part cut out with NgoMIV and PstI. The previous restriction digests have been repeated so each part could be inserted after the previous one.
</P><H4><SPAN class="mw-headline" id="Gal4-human_LBD_fusions">Gal4-human LBD fusions</SPAN></H4><P>Two human nuclear hormone receptor LBDs (Pregnane X Receptor LBD, Estrogen Receptor LBD) have been inserted into pSB1C3 after Gal4 was ligated into the backbone. The Gal4 (438 bp) is a modular protein consisting of a DNA-binding domain and an activation domain which can be used in <A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals#Cotranfection_Assay,_Two_Hybrid_Assay" class="external text" rel="nofollow">Two-Hybrid system</A> in order to identify interactions between two proteins or a protein with DNA. The length of PXR LBD is 732 bp while the ER LBD is 828 bp long, so the fusion leads to 1170 bp and 1266 bp long composite inserts.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:403px;"><DIV class="thumbcaption">pSB1C3-Gal4-PXR</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:374px;"><DIV class="thumbcaption">pSB1C3-Gal4-ER</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Both of them have been transferred into pCDNA3.1 to express them in certain cell lines.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:374px;"><DIV class="thumbcaption">pCDNA-Gal4-PXR</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:315px;"><DIV class="thumbcaption">pCDNA-Gal4ER</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Gal4-Ecdysone_receptor_fusion">Gal4-Ecdysone receptor fusion</SPAN></H4><P>The LBD of Ecdysone Receptor (639 bp) which can be found in arthropodes has been inserted after the Gal4 both in the pSB1C3 and pCDNA3.1 resulting an 1077 bp long insert in the vectors.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:354px;"><DIV class="thumbcaption">pSB1C3-Gal4-EcR</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">pCDNA-Gal4-EcR</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Gal4-C._elegans_LBD_fusions">Gal4-C. elegans LBD fusions</SPAN></H4><P>Five nuclear hormone receptor LBDs of C. elegans (NHR8, NHR23, NHR31, NHR80, DAF12) have been inserted after Gal4 in pSB1C3, resulting inserts with the lenght between 944 bp and 977 bp. All of the Gal4-C.elegans LBD constructs (e.g. Gal4-NHR8) except Gal4-NHR80 composite part have been fused into pCDNA3.1 expression vector to test them as a lipid sensor.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">pSB1C3-Gal4-NHR</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:381px;"><DIV class="thumbcaption">pCDNA-Gal4-NHR8</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Estrogen_Receptor_from_basic_parts">Estrogen Receptor from basic parts</SPAN></H4><P>Estrogen receptor DBD (318 bp) and LBD (828 bp) have been inserted together into pSB1C3. The two main domains of ER are linked together by the hinge region of Vitamin D Receptor (357 bp) resulting a 1503 bp long insert in the backbone.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:468px;"><DIV class="thumbcaption">pSB1C3-ER_DBD-VDRh-ER_LBD</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Creating_expression_vector_from_pSB1A3">Creating expression vector from pSB1A3</SPAN></H3><P>Three basic parts from pSB1C3 have been ligated into pSB1A3 consecutively by using chloramphenicol and ampicillin assembly in order to create an <A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#Eukaryotic_expression_vectors_-_Composite_Parts" class="external text" rel="nofollow">expression vector</A> from the ampicillin resistance carrying backbone. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:427px;"><DIV class="thumbcaption">pSB1A3-TRE-Gal4-PXR</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">pSB1A3-TRE-Gal4-EcR</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The first step was to insert 321 bp long TRE-CMV (Tetracyclin Response Element- Minimal Cytomegalovirus promoter) into pSB1A3 which has the length of 2157 bp. Then Gal4-PXR LBD (1170 bp) and Gal4-Ecr LBD (1077 bp) fusions were inserted after the TRE- CMV, before PolyA tail (191 bp) was ligated in the end. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:367px;"><DIV class="thumbcaption">pSB1A3-TRE-Gal4-PXR-PolyA</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">pSB1A3-TRE-Gal4-EcR-PolyA</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
As the negative control of the expression vector built up from standard biobrick parts TRE- CMV and PolyA tail were ligated together into pSB1C3 and pSB1A3. 
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">pSB1A3-TRE-PolyA</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Western_Blots_of_Gal-fused_proteins">Western Blots of Gal-fused proteins</SPAN></H3><P>The functionality of the constructs was assesed by Gal4 <B>Western Blot</B>.
293T cells were transfected. Western-blot was carried out using <B>rabbit anti-Gal4 polyclonal</B> primary antibody. High protein expression have been noticed when Gal4-PXR LBD composite construct was cloned into pCDNA3.1 as well as into the expression vector created from pSB1A3 (using standard BioBrick parts). Lower expression was detected in the case of Gal4-EcR LBD in both expression vectors. The Gal4-PXR and EcR-LBD fused proteins appeared at about <B>40 kDa</B> while the Gal4-C.elegans-LBD (NHR8, NHR31) chimeric proteins which have been intensely expressed, can be seen between <B>30-35 kDa</B> which corresponds to thier predicted molecular weight. 
</P><H2><SPAN class="mw-headline" id="Transients_cotransfection_assay_.2Ftwo-hybrid_system.2F">Transients cotransfection assay /two-hybrid system/</SPAN></H2><P>We <B>moved the composite parts generated in pSB1C3 into the expression vectors</B>, then transfered these fused parts into pCDNA and pSB1A3 vectors for further works. pCDNA is designed for high-level expression, purification and detection of recombinant protein in mammalian hosts. In addition we developed a new expression vector pSB1A3 with a CMV-promoter and poly-A tail. Our experiments proceeded on two different lines.  
</P><P>The first one is the presentation of the cloned constructs bearing the <B>ability to get activated</B> by ligand binding. These were the activation experiments.
</P><P>Furthermore we showed that our constructs had the <B>ability to interact with the previously published heterodimeric partner RXR</B>. To confirm this we used the mammalian two-hybrid system. Binding to RXR confirms the proper folding and ultimately the generation of a well designed LBD, and a functional mammalian expression vector.  The method used to design our parts is described as a proposed standard BBF <A href="http://tools.ietf.org/html/rfc64" class="external mw-magiclink-rfc">RFC 64</A><A href="https://static.igem.org/mediawiki/2010/7/74/BBF_RFC_64a.pdf" class="external autonumber" rel="nofollow">[1]</A>.
</P><H3><SPAN class="mw-headline" id="Activation_experiments">Activation experiments</SPAN></H3><P><A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals#Cos-1_cells" class="external text" rel="nofollow">COS- 1 cells</A> were transfected with the chimeric receptors and reporter constructs. 5-6 hours after transfection we exposed the cells to ligand treatment (at least 10uM of ligands).
The CMX-GAL4-PPAR gamma was the positive control during our experiments, showing a stable and robust activation upon Rosiglitazone treatment, according to the b-galactosidase-normalized <A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals#Luciferase" class="external text" rel="nofollow">luciferase activity</A> levels.
</P><P>We tested: 
</P><P><B>1)</B></P><P><I>pCDNA-GAL4-PXR-LBD</I></P><P>pSB1C3-GAL4-PXR-LBD</P><P>with Pregnenolone carbonitrile, Rifampicin. These proteins showed no response to the mentioned ligands. 
</P><P><B>2)</B></P><P><I>pCDNA-GAL4-Ecdysone receptor-LBD</I></P><P>pSB1A3-GAL4-Ecdysone receptor-LBD</P><P>with Ponasterone A a typical ecdysteroid. Adding 5 uM of Ponasterone A resulted the activation of the construct (red arrow).
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Test of pCDNA/pSB1A3-GAL4-PXR/EcR-LBD activity</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><I>pCDNA-GAL4-Estrogen receptor-LBD</I></P><P>with its known ligand, 17β-Estradiol. When we treated the cells with 100nM concentration of 17β-Estradiol, a 1.8 fold activation was detected, compare to the negative control (DMSO).  
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Test of pCDNA-GAL4-ER-LBD activity</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Two_hybrid_system">Two hybrid system</SPAN></H3><P>Expression plasmids of the various receptor constructs were co-transfected into COS1 cells together with luciferase reporter plasmids. Our question was wether these proteins still have the <B>ability to form RXR-heterodimers</B>, a key feature of several nuclear receptor superfamilies. In these experiments we used our newly fused parts and some characterized constructs that were partially <B>published by our lab</B>:
</P><P>CMX-VP-RXR-LBD
</P><P>CMX-GAL4-PPAR gamma-LBD 
</P><P>CMX-GAL4-RAR alpha-LBD 
</P><P>CMX-GAL4-PXR-LBD
</P><P><B>Our fused constructs:</B></P><P>
pCDNA-GAL4-PXR-LBD
</P><P>pSB1A3-GAL4-PXR-LBD 
</P><P>pCDNA-GAL4-Ecdysone receptor-LBD 
</P><P>
As a positive control we transfected the cells with <I>CMX-GAL4-PPARg-LBD or CMX-GAL4-PXR-LBD</I>. 
Our results shows that the constructed chimeric recptors (<I>pCDNA-GAL4-PXR-LBD and pSB1A3-GAL4-PXR-LBD</I>) are <B>able to interact</B> with RXR-LBD. LG268 is a potent synthetic RXR-agonist, that enhances the formation of RXR-heterodimers. The increased luciferase expression of the last columns shows that dimers are formed. This is one key aspect of NR-action, that <B>remained intact</B> during our cloning process.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Test of PXR:RXR interaction</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="BBF_RFC_64_:_Building_Protein_Domain_Based_Composite_Biobricks_for_Mammalian_Expression_Systems"><B>BBF <A href="http://tools.ietf.org/html/rfc64" class="external mw-magiclink-rfc">RFC 64</A> : Building Protein Domain Based Composite Biobricks for Mammalian Expression Systems</B></SPAN></H1><P>The method described below is RECOMMENDED to be used for the design of mamalian domains that are planned to be used in mammalian expression vectors. It contains detailed instruction which may ensure protein expression of your transfected gene. <A href="https://static.igem.org/mediawiki/2010/7/74/BBF_RFC_64a.pdf" class="external text" rel="nofollow">Click here to download the RFC</A></P><H1><SPAN class="mw-headline" id="Plans.2C_Possibilities"><B>Plans, Possibilities</B></SPAN></H1><P>The remotely activated transcription factor concept can be used to construct highly complex synthetic biological systems, for us it was a very appealing concept. Physicians may use fruit fly hormones in humans to titrate specific genes in gene therapy patient in selected tissue, such as dopamine receptor genes in schizophrenia. Worm species that can synthesize a different color reaction based on the amount of environmental pollutants in the soil they live. Scientists may induce stem cell pluripotency by titrating the exact amount of oncogenes needed for a fibroblast to turn into an embryo.
</P><P>The defenite next step would be to add DBD's to the registry, or more exact a way for users to synthesize a DNA binding domain according to their sequance of interest. DNA binding domains are themselves modular from sybdomains as: helix loop helix, leucine zippers, helix turn helix and zinc fingers (the most popular subdomains). Once our team assembles a zinc finger toolkit with carefull construction instructions, a user could in vitro or in silico assemble and predict where in the genome his artificial transcription factor/ nuclear receptor will act on gene expression. In the future this will allow full freedom of harnesing the potency of artificial titratable transcription factors. 
</P><P>In our system we would like to induce <A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals#Apoptosis" class="external text" rel="nofollow">apoptosis</A> (click to find out more about apoptosis) by increasing the amount of proapoptotic proteins such as BAX by producing an expression vector containing TRE-minimal CMV promoter, BAX or PUMA and polyA tail. The pro-apoptotic proteins are able to express only in a controlled form using the Tetracyclin-controlled transcriptional activation system. The overexpression of these proteins can lead to Mitochondrial Outer Membrane Permeabilisation (MOMP) leading Cytochorome c release which is a point of no return in the mitochondrial pathway of Programmed Cell Death (PCD). The goal is to “switch off” the cells, which were genetically modified by us, after they “finished their work” in the recent local environment combining the components of the tetracycline-inducible system in self-contained retroviral and plasmid vectors. 
</P><P>Furthermore human, arthropode or C. elegans nuclear hormone receptor composite constructs can be expressed in an inducible form using <A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals#The_Tetracycline_On/Off_Gene_Expression_System" class="external text" rel="nofollow">Tetracyclin-controlled transcriptional activation</A> (click to find out more about TRE) system rather than using a vector which ensures constitutive expression of the parts. In our system Tetracycline Response Element (TRE) is recognized and bound by the Tetracycline repressor (TetR) protein. The TRE consists of 7 repeats separated by spacer sequences and placed upstream of CMV minimal promoter that has basal expession in the abscence of bound TetR. Tetracycline derivatives (e.g. doxycycline) bind TetR and render it incapable of binding to TRE, thereby forcing the expression of target genes.
</P><H1><SPAN class="mw-headline" id="Contibution_to_project_by_team_members"><B>Contibution to project by team members</B></SPAN></H1><UL><LI><B>Students</B> (in alphabetical order) 
</LI><LI><B>Timea Beregi</B> -  Cloning work, wiki writing, team image, protocols writing
</LI><LI><B>Erika Berényi</B> - Prepared psB1A3-TRECMV-polyA plasmid constructs( cloning, transfection, PCR), and prepared western blots after transfection. Uploaded description from apoptosis parts to the wiki and partsregistry. 
</LI><LI><B>Shun Chieh Liu</B> - Took part in the cloning task initially, demonstrated some in silico work, scriptwriter of the video projects, modified the side project page in the wiki. 
</LI><LI><B>Lior Malka</B> - Wiki design and programming, Culture team, Wiki Video Project design, Parts Registry moderator. 
</LI><LI><B>Daniel Markovits</B> - Wiki design and graphics, Help for teams PCR, Cell Culture and Luciferase teams, Video Project Narrator
</LI><LI><B>Katalin Nagy</B> - Cloning work
</LI><LI><B>Lilla Ozgyin</B> - Cloning work, coordinator of transfections, actress in video protocols, protocol writing, uploading a part, Parts Registry work, some wiki work, making photos of labwork. 
</LI><LI><B>Katalin Sandor</B>- Cloning, luciferase measurements, extraction of contaminated environmental samples, presenter of PCR video protocol, uploading to the wiki, making photos about labwork.
</LI></UL><UL><LI><B>Student Team</B> Coordinators (in alphabetical order) 
</LI><LI><B>Bence Daniel</B>- Supervisor of the whole labwork, optimization of the luciferase measurements and the two hybrid system, 3D protein structures, wiki writing.
</LI><LI><B>Endre Kristof</B> - Cloning (e.g. new expression vector from standard Biobrick parts), transfections and coordination in the wet lab. Participating in fund raising, in silico design, video and media project. Uploading materials of project description, future possibilities etc. to the wiki and informations about the composite and some basic parts to the partsregistry.
</LI><LI><B>Yakir Guri</B> - Project selection and planning, part of cell culture team, cloning and in silico design. The supervisor of the <A href="https://static.igem.org/mediawiki/2010/8/8f/Links_to_OWW_and_YOUTUBE.pdf" class="external text" rel="nofollow">'Film in Lab'</A> video tutorial project (film making, editing, sound embedding) and <A href="http://www.youtube.com/user/debrecenigem2010" class="external text" rel="nofollow">Youtube channel</A> manager. Collecting, editing and reviewing protocols and uploading to the <A href="http://www.openwetware.org/wiki/Special:Contributions/Yakir_Guri" class="external text" rel="nofollow">'Open Wet Ware'</A> . Coordinator of PR and fundraising (media, images, writing <A href="https://static.igem.org/mediawiki/2010/2/2a/Medikuslap.pdf" class="external text" rel="nofollow">articles</A>, interviewing team members). 
</LI><LI><B>Ophir Keret</B> - Intiated project together with Balint.L.Balint, Team recruitment, project selection, work planning and coordination, team tutorials, labwork supervision, in silico design/modeling of the parts, construction of the expression vector, PCR work, Chief wiki editor, contributed to wiki writings (abstract, introduction, minimals, on the media).  
</LI></UL><UL><LI><B>Team Instructors</B>(in alphabetical order) 
</LI><LI><B>Balint L. Balint</B> - Initiated the project, made the team recruitment, fund raising, some of the PR, laboratory setup, helped shaping the project, seting up of the team.  I supervised the first lab experiments and planned some of the experiments later on, ordered parts and reagents, organized finances, organized administrative issues of the trip. Supervised wiki and partsregistry work. 
</LI><LI><B>Peter Brazda</B> - Supervised planing and evaluation of luciferase assays. Participated in wet-lab education and the wiki-works. Co-mentored students in discussing scientific problems and questions of NR-biology.
</LI><LI><B>Mate Demeny</B> -  Implementation of the general approach for the design of the BBF_<A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible parts. Supervised the cloning process and designed adaptors to the pCDNA3.1. Tutoring the cloning process. Troubleshooting for the cloning team.
</LI><LI><B>Gabor Zahuczky</B> - Supported the initiation of the project, fund raising, organization of finances and administration. Participated in the design of parts and wet lab education.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Debrecen-Hungary/project">http://2010.igem.org/Team:Debrecen-Hungary/project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Debrecen-Hungary/project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Debrecen-Hungary/project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Debrecen-Hungary/project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Debrecen-Hungary/project&amp;oldid=193134" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>