<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Debrecen-Hungary_minimals"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Debrecen-Hungary/minimals">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Debrecen-Hungary/minimals&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Debrecen-Hungary/minimals&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Debrecen-Hungary/minimals&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Debrecen-Hungary/minimals" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Debrecen-Hungary/minimals</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><TABLE><TBODY><TR valign="top" style="margin-right: 0px;"><TD width="1100px" style="padding: 0 0px 0px 0px; background-color:#ede8e2"><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team" style="color: white"> Team</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/meetteam"><SPAN><SPAN>Meet the Team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/gallery"><SPAN><SPAN>Gallery</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team#about"><SPAN><SPAN>About The team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team#Lab"><SPAN><SPAN>About The Lab</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team#deoec"><SPAN><SPAN>About The University</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team#debrecen"><SPAN><SPAN>About  Debrecen</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team#cube"><SPAN><SPAN>About  Rubiks's Cube</SPAN></SPAN></A></LI></UL><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project" style="color: white"> Project</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project#abs"><SPAN><SPAN>Abstract</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project#intro"><SPAN><SPAN>Introduction</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project#Results"><SPAN><SPAN>Results</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project#Plans"><SPAN><SPAN>Plans, Possibilities</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/project#BBF_RFC_64_:_Building_Protein_Domain_Based_Composite_Biobricks_for_Mammalian_Expression_Systems"><SPAN><SPAN>RFC 64 suggestion</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/indicontibi"><SPAN><SPAN>Members Contribution</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts" style="color: white"> Parts</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#LBD"><SPAN><SPAN>LBD</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#DBD"><SPAN><SPAN>DBD</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#other"><SPAN><SPAN>Other basic parts</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/parts#composite"><SPAN><SPAN>Composite Parts</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/favorite"><SPAN><SPAN>Favorite Parts</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/protocols" style="color: white"> Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/protocols#Contents"><SPAN><SPAN>Cloning Team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/protocols#Mini_Prep"><SPAN><SPAN>Molecular work team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/protocols#Gel_electrophoresis"><SPAN><SPAN>Tissue culture team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/protocols#Measuring_Luciferase_activity_with_the_Victor_plate_reader"><SPAN><SPAN>Luciferase Team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/sponsors" style="color: white"> Sponsors</A></LI><LI><A style="color: white"> Tools</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/sitemap"><SPAN><SPAN>Site Map</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/minimals"><SPAN><SPAN>The Minimals</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/safety"><SPAN><SPAN>Safety</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/Team_meetings"><SPAN><SPAN>Team Meetings</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/links"><SPAN><SPAN>Links</SPAN></SPAN></A></LI><LI><A style="color: white"> Side Projects</A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/arsenic"><SPAN><SPAN>Arsenic Project</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/AKG_team"><SPAN><SPAN>AKG team</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/video_project"><SPAN><SPAN>Video Project</SPAN></SPAN></A></LI><LI><A href="https://2010.igem.org/Team:Debrecen-Hungary/on_the_media"><SPAN><SPAN>On The Media</SPAN></SPAN></A></LI></DIV></TD></TR></TBODY></TABLE><DIV align="justify"><H1><SPAN class="mw-headline" id="Welcome_To_The_Minimals">Welcome To The Minimals</SPAN></H1><P>From year to year (and from one to jamboree to another) the world of synthetic biology exponentially expands. Some iGEMers may find niches of complex biological systems and use special model organisms or laboratory tools. Our philosophy is that a good project is one that can be kept simple and short. 
Thus we found it prudent to provide our fellow iGEM teams from abroad with brief overviews of our project background (which we call “the minimals”).  Our hope is that it may clear some of the molecular “mish mash” you may be experiencing and help point you in the right direction for a complete understanding of our project.
</P><H2><SPAN class="mw-headline" id="Contents"><B>Contents</B></SPAN></H2><P><B>Scientific Background</B></P><P><A href="/Team:Debrecen-Hungary/minimals#Cellular_signaling" title="Team:Debrecen-Hungary/minimals">Cellular signaling</A> - <A href="/Team:Debrecen-Hungary/minimals#Nuclear_Receptors" title="Team:Debrecen-Hungary/minimals">Nuclear Receptors</A> -  <A href="/Team:Debrecen-Hungary/minimals#Structure_of_NRs" title="Team:Debrecen-Hungary/minimals">Structure of NRs</A> -  <A href="/Team:Debrecen-Hungary/minimals#Apoptosis" title="Team:Debrecen-Hungary/minimals">Apoptosis</A> -  <A href="/Team:Debrecen-Hungary/minimals#The_Tetracycline_On.2FOff_Gene_Expression_System" title="Team:Debrecen-Hungary/minimals">The Tetracycline On/Off Gene Expression System</A></P><P><B>Model Organisms</B></P><P><A href="/Team:Debrecen-Hungary/minimals#Drosophila_Melanogaster" title="Team:Debrecen-Hungary/minimals">Drosophila Melanogaster</A> - <A href="/Team:Debrecen-Hungary/minimals#Caenorhabditis_elegans" title="Team:Debrecen-Hungary/minimals">Caenorhabditis elegans </A> - <A href="/Team:Debrecen-Hungary/minimals#Homo_sapiens" title="Team:Debrecen-Hungary/minimals">Homo sapiens </A></P><P><B>In The Laboratory (Techniques And Reagents)</B></P><P><A href="/Team:Debrecen-Hungary/minimals#Cotransfection_Assay.2C_Two_Hybrid_Assay" title="Team:Debrecen-Hungary/minimals">Cotransfection Assay, Two Hybrid Assay</A> - <A href="/Team:Debrecen-Hungary/minimals#Luciferase" title="Team:Debrecen-Hungary/minimals">Luciferase</A> - <A href="/Team:Debrecen-Hungary/minimals#Cos-1_cells" title="Team:Debrecen-Hungary/minimals">Cos-1 cells </A> - <A href="/Team:Debrecen-Hungary/minimals#Dose_response_curve" title="Team:Debrecen-Hungary/minimals">Dose response curve </A></P><H2><SPAN class="mw-headline" id="Scientific_Background">  Scientific Background  </SPAN></H2><H3><SPAN class="mw-headline" id="Cellular_signaling"> Cellular signaling </SPAN></H3><P>Cells have an innate ability to “listen” and correctly react to their local or even distant environment. Through time it has been observed that a complex systems of communication governs essential cellular activates and coordinates cell actions.<A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[1]</A> Today, it is well known that processes such as development, growth, tissue repair or death,  metabolic shifts and immunity are all governed, at the molecular level, by signaling.   By understanding cell signaling, diseases may be treated effectively and, theoretically, artificial tissues may be created.
Cells sense information from their local surroundings through a class of proteins known as receptors. Chemicals that activate (or  inhibit) receptors are often named hormones, growth factors,  cytokines or even neurotransmitters  yet their proper term is receptor ligands. 
Water soluble ligands have cell membrane penetration and thus mostly interact with trans-membranous receptors, whereas ligands with high lipid solubility easily penetrate the cell membrane 
</P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P><H3><SPAN class="mw-headline" id="Nuclear_Receptors"> Nuclear Receptors </SPAN></H3><P>
Nuclear receptors are ligand activated transcription factors. As such, they are able to regulate the expression of their target genes by direct DNA-binding, in a ligand-dependent manner. They play a central role in endocrine signaling, regulation of embryonic and adult development and differentiation <A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[2]</A><A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[3]</A>. Many nuclear receptors, are among the primary targets of drug discovery because of their diverse biological actions.
Nuclear receptors bear high homology to each other and are modular into distinct domains: N-terminal regulatory domain, DNA-binding domain, a Hinge region, Ligand binding domain (LBD) and a C-terminal domain. 
</P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P><H3><SPAN class="mw-headline" id="Structure_of_NRs"> Structure of NRs </SPAN></H3><P>Ligand binding domain (LBD) is a well conserved domain amongst various nuclear receptors whose structure usually referred to as an alpha helical sandwich fold. The LBD shows some diversity among nuclear receptors as it is a site for receptor-specific events. It possesses transactivation ability and contains a ligand-binding pocket as well as the main interaction surfaces for other proteins <A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[8]</A>. 
</P><P>The DNA-binding domain (DBD) contains two Zinc-finger motifs and is linked to the LBD by a highly flexible hinge region. This segment of the nuclear receptors holds the ability to recognize and bind to preferred or specific DNA-motifs, the response elements. 
</P><P>A partially unexpected and amazing feature of the nuclear receptors is that the above mentioned domains can be swapped. It was Vincent Giguere and his colleagues, who first devised a technique called cotransfection assay and developed it to the domain-swap technique.
</P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P><H3><SPAN class="mw-headline" id="Apoptosis"> Apoptosis </SPAN></H3><P>Apoptosis, the process of programmed cell death (PCD), is an elaborate cellular homeostasis mechanism that ensures correct development and function of multicellular organisms. Biochemical events lead to characteristic changes (morphology) and the death of cells. 
</P><P>It has a crucial role in maintaining tissue homeostasis as provides the appropriate rate of cellular turnover. Potentially dangerous, poorly differentiated or excess cells are removed by apoptosis without local inflammation from leakage of cell contents. Dying by apoptosis in many cases has primary importance: lens cells, which lost their nucleus, need the vision, the corneocytes without nucleus and organelles provide mechanical protection on the outer surface of the skin. 
</P><P>Apoptosis might occur in pathological conditions, mainly in secondary, as a part of the defensive mechanism, helping to eliminate the viral infected or malignant cells. The abnormality of the central molecules and genes of apoptosis may play a role in the development of certain diseases: degenerative disease, autoimmune disease, tumor formation and malformations.
</P><P>Many pathways and signals lead to apoptosis, but there is only one mechanism that actually causes the death of a cell. After a cell receives stimulus, it undergoes organized degradation of cellular organelles by activated proteolytic caspases. A cell undergoing apoptosis shows characteristic morphology.
</P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P><H3><SPAN class="mw-headline" id="The_Tetracycline_On.2FOff_Gene_Expression_System"> The Tetracycline On/Off Gene Expression System </SPAN></H3><P>The most commonly used inducible expression systems for research of eukaryote cell biology are named Tet-Off and Tet-On called togheter the Tetracyclin-controlled transcriptional activation. Gene expression is controlled as a result of binding of the Tet-Off (tetracycline transactivator) or Tet-On (reverse tetracycline transactivator) protein to tetracycline response elements (TREs) located within an inducible promoter such as minimal Cytomegalovirus (CMV) promoter. Each system consists of 2 components (a) that have been optimized for use in mammalian cells:
</P><P>/1/ A regulator vector which expresses one of the tetracycline-controlled transactivators.
</P><P>/2/ A response vector containing TRE within the promoter that controls expression of the gene of interest.
</P><P>The Tet-Off system (b) makes use of the tetracycline transactivator (tTA) protein created by fusing TetR (tetracycline repressor) found in E. coli with another protein, VP16 produced by the Herpes Simplex Virus. The tTA binds the DNA at a tet-o operator. Once bound to the tet-o the tTA will activate a promoter coupled to the tet-o operator, activating the transcription of the gene of interest. Tetracycline derivatives bind tTA and render it incapable of binding to TRE thereby preventing transactivation of target genes. The TetR can prevent alone the expression of the nearby gene and the presence of tetracylin can force the promoter effect before the gene of interest.
</P><P>The Tet-On system (c) works in the opposite way so the rtTA protein is capable of binding the operator only when bound by doxycycline. Thus the introduction of doxycyline to the system initiates the transcription of the genetic product. The Tet-On system is sometimes preferred for the faster responsiveness.
</P><P><I>The alternative version of the original Tet-On and Tet-Off Systems called Tet-Advanced have been optimized for improved expression in mammalian cells by utilizing human codon preferences and removing cryptic splice sites from the mRNA sequence. These enhancements lead to higher and more stable expression levels, minimize off-target effects and reduce toxicity.</I></P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P><H2><SPAN class="mw-headline" id="Model_Organisms"> Model Organisms </SPAN></H2><H3><SPAN class="mw-headline" id="Drosophila_Melanogaster"> Drosophila Melanogaster </SPAN></H3><P>Drosophila Melanogaster, also known as the common fruit fly, is one of the most frequently used model organisms in biological sciences, including studies in genetics, physiology, microbial pathogenesis and life history evolution.<A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[9]</A>   
The ecdysone receptor is a nuclear receptor found in D.Melanogaster, where it controls development and contributes to other processes such as reproduction. Its ligands are ecdysteroid which are secreted by the organism’s prothoracic gland.
</P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P><H3><SPAN class="mw-headline" id="Caenorhabditis_elegans"> Caenorhabditis elegans</SPAN></H3><P>Caenorhabditis elegans is a free-living, transparent nematode (roundworm), about 1 mm in length,<A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[10]</A>    which lives in temperate soil environments C. elegans is intensively studied as a model organism in biology for a variety of reasons. The developmental fate of every single somatic cell (959 in the adult hermaphrodite; 1031 in the adult male) has been mapped out.<A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[11]</A><A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[12]</A>     
The C.elegans genome harbors 284 nuclear receptors <A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[10]</A>    (a striking figure), which have been shown to control traits such as sex determination, larva development, life span, neuronal growth and identity and much more. As far as nuclear receptors go, they are a gold mine.
</P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P><H3><SPAN class="mw-headline" id="Homo_Sapiens"> Homo Sapiens </SPAN></H3><P>Homo sapiens are the only living species in the Homo genus of bipedal primates in  the great ape family. Nuclear receptors number up to 47 in humans, yet only few have been well characterized. They constitute the focus of medicinal reproductive technologies, hormonal medicine (endocrinology), immunology, drug interaction and much more.
</P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P><H2><SPAN class="mw-headline" id="In_The_Laboratory_.28Techniques_And_Reagents.29"> In The Laboratory (Techniques And Reagents) </SPAN></H2><H3><SPAN class="mw-headline" id="Cotransfection_Assay.2C_Two_Hybrid_Assay"> Cotransfection Assay, Two Hybrid Assay </SPAN></H3><P>Cotransfection assay requires two types plasmids to be cotransfected into a cell; an expression vector, coding a functional NR and and a reporter-plasmid, harboring an inducible promoter that regulates a reporter-gene the expression level of which can easily be detected and quantified. By this method the NR-activity and ligand-potency can be studied. As a result of the unique domain-structure of NRs, the domain-swap technique was developed, using chimeric proteins; the DBD of a known receptor is fused to any other receptor. This made ligand screening possible. 
</P><P>The two-hybrid assay is a technique in molecular biology which can be used to investigate protein-protein interactions<A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[13]</A><A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[14]</A>.
The interaction of a DNA-binding bait-protein and the prey-protein with a ligand binding and activating ability is shown by the activated reporter. 
</P><P>
Multiple applications of these methods have appeared. It is possible to investigate: 
</P><P>- the response elements by changing the sequence that is recognized by the DBD
</P><P>- potential dimer partners and co-regulators by creating specific bait-prey sets
</P><P>- potential ligands and activating factors
</P><P><B>Our main concept was to introduce the <I>iGEM-method</I> into this well established area of research. We showed that these systems (they were published before) can be reproduced with the use of our pre-designed Parts. We could also present that it is possible to take the next step: to investigate new samples as well, i.e. to test the effects of different biological samples on NR-action by our Kits.</B></P><P>This image shows the basic concept of the two-hybrid system: the DNA-binding bait-protein (Gal-fused, in our case), and the activator-domain fused to the prey-protein (mostly VP-fusion, in our case) and the reporter gene with the specific binding site. Yeast Gal4 is a common DBD used for this techniques purpose. Commonly used reporter genes include the product of the LacZ gene (Beta galactosidase) and Luciferase. 
</P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P><H3><SPAN class="mw-headline" id="Luciferase"> Luciferase </SPAN></H3><P>Luciferase is an enzyme class able to produce bioluminescence by oxidizing the substrate luciferin. &quot;Firefly luciferase&quot; as a laboratory reagent usually refers to P. pyralis luciferase. Its emission can be measured photometrically and hence used to deduce the protein enzyme concentration through standardized methods.
</P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P><H3><SPAN class="mw-headline" id="Cos-1_cells"> Cos-1 cells  </SPAN></H3><P>Cos-1 cells (acronym for CV-1 simian origin, SV-40 viral positive) is cell line derived from the African green monkey kidney cells. It is also often used to transfect cells in tissue culture conditions to produce recombinant proteins for molecular biology, biochemistry, and cell biology experiments. Two forms of COS cell lines commonly used are COS-1 and COS-7. The cell line was obtained by immortalizing the original CV-1 cells with SV-40 virus genome.  This allows the production of large T antigen but has a defect in genomic replication.<A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[15]</A></P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P><H3><SPAN class="mw-headline" id="Dose_response_curve"> Dose response curve</SPAN></H3><P>Dose response curve depicts a change in a measured effect on an organism caused by differing levels of exposure to a chemical in standardized measuring conditions. It may apply to either individuals or to populations. The curve is usually displayed in a simple X-Y graph (X being logarithm of dose, Y for effect). The half maximal effective concentration (EC50), a common feature of drug potency, is the chemical’s concentration which induces a response halfway between the baseline and maximum.<A href="/Team:Debrecen-Hungary/minimals#References" title="Team:Debrecen-Hungary/minimals">[16]</A></P><P><A href="/Team:Debrecen-Hungary/minimals#Welcome_To_The_Minimals" title="Team:Debrecen-Hungary/minimals">[TOP]</A></P></DIV><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><DIV style="text-align:justify;"><P>1.	^ Witzany, Guenther (2010). Biocommunication and Natural Genome Editing. Springer. <A href="/Special:BookSources/9789048133185" class="internal mw-magiclink-isbn">ISBN 9789048133185</A>. 
2.	^ Evans RM (1988). &quot;The steroid and thyroid hormone receptor superfamily&quot;. Science 240 (4854): 889–95. doi:10.1126/science.3283939. <A href="http://www.ncbi.nlm.nih.gov/pubmed/3283939?dopt=Abstract" class="external mw-magiclink-pmid">PMID 3283939</A>.   
3.	^ Olefsky JM (2001). &quot;Nuclear receptor minireview series&quot;. J. Biol. Chem. 276 (40): 36863–4. doi:10.1074/jbc.R100047200. <A href="http://www.ncbi.nlm.nih.gov/pubmed/11459855?dopt=Abstract" class="external mw-magiclink-pmid">PMID 11459855</A>. 
4.	^ a b Mangelsdorf DJ, Thummel C, Beato M, Herrlich P, Schutz G, Umesono K, Blumberg B, Kastner P, Mark M, Chambon P, Evans RM (1995). &quot;The nuclear receptor superfamily: the second decade&quot;. Cell 83 (6): 835–9. doi:10.1016/0092-8674(95)90199-X. <A href="http://www.ncbi.nlm.nih.gov/pubmed/8521507?dopt=Abstract" class="external mw-magiclink-pmid">PMID 8521507</A>. 
5.	^ a b Novac N, Heinzel T (2004). &quot;Nuclear receptors: overview and classification&quot;. Curr Drug Targets Inflamm Allergy 3 (4): 335–46. doi:10.2174/1568010042634541. <A href="http://www.ncbi.nlm.nih.gov/pubmed/15584884?dopt=Abstract" class="external mw-magiclink-pmid">PMID 15584884</A>. <A href="http://www.ingentaconnect.com/content/ben/cdtia/2004/00000003/00000004/art00002" class="external free" rel="nofollow">http://www.ingentaconnect.com/content/ben/cdtia/2004/00000003/00000004/art00002</A>. 
6.	^ a b c Nuclear Receptors Nomenclature Committee (1999). &quot;A unified nomenclature system for the nuclear receptor superfamily&quot;. Cell 97 (2): 161–3. doi:10.1016/S0092-8674(00)80726-6. <A href="http://www.ncbi.nlm.nih.gov/pubmed/10219237?dopt=Abstract" class="external mw-magiclink-pmid">PMID 10219237</A>. 
7.	^ a b Laudet V (1997). &quot;Evolution of the nuclear receptor superfamily: early diversification from an ancestral orphan receptor&quot;. J. Mol. Endocrinol. 19 (3): 207–26. doi:10.1677/jme.0.0190207. <A href="http://www.ncbi.nlm.nih.gov/pubmed/9460643?dopt=Abstract" class="external mw-magiclink-pmid">PMID 9460643</A>. 
8.	^ a b Wärnmark A, Treuter E, Wright AP, Gustafsson J-Å (2003). &quot;Activation functions 1 and 2 of nuclear receptors: molecular strategies for transcriptional activation&quot;. Mol. Endocrinol. 17 (10): 1901–9. doi:10.1210/me.2002-0384. <A href="http://www.ncbi.nlm.nih.gov/pubmed/12893880?dopt=Abstract" class="external mw-magiclink-pmid">PMID 12893880</A>. 
9.	^ a b Eric C. R. Reeve, ed (2001-06-23). &quot;Drosophila melanogaster: The Fruit Fly&quot;. Encyclopedia of genetics. USA: Fitzroy Dearborn Publishers, I. pp. 157. Retrieved 2009-07-01.
10.	WormBook - a free online compendium of all aspects of C. elegans biology, including laboratory protocols
11.	^ Sulston JE, Horvitz HR (March 1977). &quot;Post-embryonic cell lineages of the nematode, Caenorhabditis elegans&quot;. Dev. Biol. 56 (1): 110–56. doi:10.1016/0012-1606(77)90158-0. <A href="http://www.ncbi.nlm.nih.gov/pubmed/838129?dopt=Abstract" class="external mw-magiclink-pmid">PMID 838129</A>. 
12.	^ Kimble J, Hirsh D (June 1979). &quot;The postembryonic cell lineages of the hermaphrodite and male gonads in Caenorhabditis elegans&quot;. Dev. Biol. 70 (2): 396–417. doi:10.1016/0012-1606(79)90035-6. <A href="http://www.ncbi.nlm.nih.gov/pubmed/478167?dopt=Abstract" class="external mw-magiclink-pmid">PMID 478167</A>. 
13.	^ a b c d e f g h i j k l m n o p q r s t u v w x y z aa ab Joung J, Ramm E, Pabo C (2000). &quot;A bacterial two-hybrid selection system for studying protein-DNA and protein-protein interactions&quot;. Proc. Natl. Acad. Sci. U.S.A. 97 (13): 7382–7. doi:10.1073/pnas.110149297. <A href="http://www.ncbi.nlm.nih.gov/pubmed/10852947?dopt=Abstract" class="external mw-magiclink-pmid">PMID 10852947</A>. PMC 16554. <A href="http://www.pnas.org/cgi/content/full/97/13/7382" class="external free" rel="nofollow">http://www.pnas.org/cgi/content/full/97/13/7382</A>. 
14.	^ a b c d e f g h i j Hurt J, Thibodeau S, Hirsh A, Pabo C, Joung J (2003). &quot;Highly specific zinc finger proteins obtained by directed domain shuffling and cell-based selection&quot;. Proc. Natl. Acad. Sci. U.S.A. 100 (21): 12271–6. doi:10.1073/pnas.2135381100. <A href="http://www.ncbi.nlm.nih.gov/pubmed/14527993?dopt=Abstract" class="external mw-magiclink-pmid">PMID 14527993</A>. PMC 218748. <A href="http://www.pnas.org/cgi/content/full/100/21/12271" class="external free" rel="nofollow">http://www.pnas.org/cgi/content/full/100/21/12271</A>. 
15.	Gluzman Y (1981) SV40-transformed simian cells support the replication of early SV40 mutants. Cell 23: 175-182. <A href="http://www.ncbi.nlm.nih.gov/pubmed/6260373?dopt=Abstract" class="external mw-magiclink-pmid">PMID 6260373</A>
16.	Introducing doseresponse curves, Graphpad Software</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Debrecen-Hungary/minimals">http://2010.igem.org/Team:Debrecen-Hungary/minimals</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Debrecen-Hungary/minimals" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Debrecen-Hungary/minimals" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Debrecen-Hungary/minimals&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Debrecen-Hungary/minimals&amp;oldid=194556" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>