"
Page
Discussion
View source
History
teams
Log in
 
Team:Debrecen-Hungary/protocols/PCR
From 2010.igem.org
Team
Meet the Team
Gallery
About The team
About The Lab
About The University
About  Debrecen
About  Rubiks's Cube
Project
Abstract
Introduction
Results
Plans, Possibilities
RFC 64 suggestion
Members Contribution
Parts
LBD
DBD
Other basic parts
Composite Parts
Favorite Parts
Protocols
Cloning Team
Molecular work team
Tissue culture team
Luciferase Team
Sponsors
Tools
Site Map
The Minimals
Safety
Team Meetings
Links
Side Projects
Arsenic Project
AKG team
Video Project
On The Media
Contents
1Overview
2Reagents
3Equipment
4Step
5An example program
6References
7Links
Overview
PCR is an acronym for polymerase chain reaction. It is a method for amplifying DNA in vitro.
Reagents
• Expand High Fidelity Buffer with MgCl2
• PCR grade nucleotide mix MgCl 2
• Upstream primer
• Downstream primer
• Template DNA
• RNase free water
• Expand High Fidelity Enzyme Mix
Equipment
• Thermocycler
• Reaction tubes
Step
Prepare Master mix 1
1. Add RNase free water to reach the final volume of 25 microliter
2. Add 1 microliter of PCR grade nucleotide mix
3. Add downstream primer to reach the final concentration of 300 nanomolar
4. Add upstream primer in the same way.
5. Add Template DNA which contains 10 to 250 nanogram complex DNA
Prepare Master mix 2
6. Add 19.25 microliter of RNase free water
7. Add 5 microliter of Expand High Fidelity Buffer with MgCl2
8. Add 0.75 of Expand High Fidelity Enzyme Mix
9. Vortex the components in both Master mixes
10. Prepare the PCR tubes
11. Divide the mix1 into the tubes in a desired volume
12. Then add the mix2 into the PCR tubes which contain mix 1
13. Set up the Thermocycler
14. Place the PCR tubes into the thermocycler.
An example program
Program a standard thermocycler to run the reaction using the following parameters: Initial denaturation • Denature: 96°C, 5 mins This denatures any double stranded DNA. Thermocycling
• No. of cycles: 25
• Denature: 96°C, 1 min
• Anneal: 55°C, 30 secs
• Elongate: 72°C, 1 min
Termination
• Elongate: 72°C, 5 mins
• Hold: 4°C, until removed from machine
References
1. Arezi B, Xing W, Sorge JA, and Hogrefe HH. Amplification efficiency of thermostable DNA polymerases. Anal Biochem 2003 Oct 15; 321(2) 226-35. pmid:14511688. PubMed HubMed [Arezi-AnalBiochem-2003]
2. Saiki RK, Scharf S, Faloona F, Mullis KB, Horn GT, Erlich HA, and Arnheim N. Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia.Science 1985 Dec 20; 230(4732) 1350-4. pmid:2999980. PubMed HubMed [Saiki-Science-1985] original paper on PCR
3. Mullis K, Faloona F, Scharf S, Saiki R, Horn G, and Erlich H. Specific enzymatic amplification of DNA in vitro: the polymerase chain reaction. Cold Spring Harb Symp Quant Biol 1986; 51 Pt 1 263-73.pmid:3472723. PubMed HubMed [Mullis-1986] first public presentation on PCR
Links
Video I & II
Video III
Video IV
Video V
Video VI
Retrieved from " http://2010.igem.org/Team:Debrecen-Hungary/protocols/PCR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
