"
Page
Discussion
View source
History
teams
Log in
 
Team:METU Turkey/Delta
From 2010.igem.org
Home
 
HOME
TEAM
PROJECT Description | Planning | Results & Discussion | Conclusions | Future Work | Safety | References | Acknowledgements |
PARTS
SUPPORT ACTIONS Collaboration | Characterization of an Existing Part |
NOTEBOOK Alpha Team | Bravo Team | Charlie Team | Delta Team | Procedures
GALLERY
SPONSORS
LINKS
----
PROTEIN-DNA INTERACTIONS
Aim:
To show the interaction thermodynamics of COOA protein and target promoter pCooF and pCooM and changes in binding affinity of protein with respect to mutational changes in its target  DNA.
Methods
- Isothermal Titration Calorimetry (ITC)
- Electrophoretic  Mobility Shift Assay (EMSA)
- Intrinsic Tryptophan Fluorescence (ITF)
Both these methods require short oligonucleotides as DNA template and especially for ITC experiment we need high amounts of DNA template.  In order to propagate enough DNA, we design primers for promoters and sequences, which include response elements.
1.Isothermal Titration Calorimetry (ITC)
Scenarios for DNA component preparation:
30bp
The possible sequence of oligonucleotide that we are planning to use in ITC measurements:
GGA TAA CTG TCA TCT GGC CGA CAG ACG GGG
Response (RE) Response (RE)
Element       Element
-  MW :  18594.2 g/mole
- GC content: 60%
-  Length: 30bp
- Average MW of a base is 325 g/mole (our MW prediction is applicable acc to that info.)
- for an ITC experiment min 1uM DNA ( max: 100uM) is required in sample cell (Vcell 1.8mL)
- we will prepare 2.5 mL per experiment ( it is calculated as 2.5 nmole - 18kDa x 2.5nmole= 45ug)
- for 1uM DNA we require 45ug DNA in 2.5 mL
If we propagate our oligonucleotide w/ PCR:
- min 70 bp is required for using a QIAGEN PCR purification kit >>>it is  problem for us
- max yield is 10ug
-  so we will use 5 kit per experiment for getting 45ug
50bp
TAA AAA CTC TGG ATA ACT GTC ATC TGG CCG ACA GAC GGG GGC CGG GCT TT
- Length: 50bp
- MW: 30773g/mole ~31 kDa
- for ITC 1uM (Vcell = 2.5 mL)
1umol in 1 L ; so we need 2.5nmole for 2.5 mL
1 mole is nearly 31000 g ; so we need 77.7 ug - w/ invitrogen plasmid purification kit we get 800ug plasmids from 100ml culture - we can extract 800ug plasmid from 2ml culture - if we assume that our vector plasmid is nearly 5000bp long, it will be 100 fold of our fragment DNA (50bp) - 800/100 we get 8ug fragments after restriction digestion.
- w/ qiagen gel extraction kit we can get 10ug DNA once
- so, we need 8 kit gel extraction and 10 plasmid purification kit for one ITC experiment
- 50bp fragment recovery 50% (Fermentas)
- for 50 bp we require ~80ug
- one kit elutes 25x 0.05= 12.5ug  in once
- for 80ug we require 80/12.5= 6.4= 7  kits are required
Experimental plan [Best Case]
- Full plasmid is used as template
- Plasmids from  Mr Gene  : about 3 weeks
-Plasmids from Mr gene last year
-pAny-amphicilline
-2300bp+gene
-5 ug
- our DNA components is  137 bp >>>Total palsmid lenght is  about 2500 bp.
- average MW of a base is 325 g/mole
- MW: 325 x 2500 x 2 = 1625000 g/mole
- V(sample cell) = 2.5 ml
- we assume that we use 1 uM DNA
- so we need 2.5 nmole DNA
- for 2.5 nmole we require 2.5 x 1625000= 4062500x10^-9 ng = 1063 x 10^-6 = ~4mg
- transformation of the plasmids (2days)
- plasmid purification (3rd day)
- Invitrogen PureLink™ HiPure Plasmid Gigaprep Kit
- Cat. No. K2100-09
- Unit Size 2 preps
- DNA yield is 7.5mg- 10mg (depends on copy number type, size)
- Invitrogen PureLink™ HiPure Plasmid Megaprep Kit
- Cat. No. K2100-08
- Unit Size 4 preps
- DNA yield is 1.5mg-2.5mg (depends on copy number type, size)
- we can get ~4mg plasmid by using 2 x megakit or 1x gigakit
- confirmation of transformation w/ single cut and restriction digestion gel (3rd day)
- Gel extraction (4th day)
- PureLink™ Quick Gel Extraction and PCR Purification Combo Kit
- Cat. No. K220001
- Unit Size: 50 prep
- DNA yield is 40ug
- Unit size 50 prep
- we can get 4mg plasmid by using 100 kit :(
-  we search plasmid isolation kit which gives more yield than usual.
Richie Rich (50 bp case)
- 50 bp seq is 77.5 ug/mol (~80 ug/mol)
- we need 2.5 ml so 2.5 nmol for 1 uM in sample cell
- IDT sends 100 nmol (4 nmol after HPLC) ds 50 bp oligo which costs 200 USA dolars in total
- we can perform 2 experiments from one oligo order
- we need 2.5 nmol DNA per experiment. If we make optimization with WT Response Element,  we will make al least 8 experiment.
- w/ mutants 10 oligos in total
- 10 experiment x 2.5 nmol = 25 nmol (after HPLC)
- To be ordered 625 nmol >>> 1.750 TL >>> ~1250 USA dollars :(
IDT Contact
- 100 nmole  DNA is € 0.46 /base.
- Annealing is € 23,00/duplex.
- For 50 bp cost is € 69,00.
- Shipping to Turkey is € 50,00.
- lyophilized
- You should have 6 OD of duplex which is 12 nmole depending upon sequence composition.
Primer Design
pcooF
forward: 5' TAAAAACTCTGGATAAC
-      Avoid polyXs YES
-      3’ end: Avoid T (greater tolerance of mismatch) YES
-      3′ end: G or C (stronger clamping to the template) [Fernandez] YES
-      3′ end region: only 2 G or C in the last 5 [Sayler Lab] YES
-      Length: 17
-      GC : 29%
-      Tm : 37C
-      self complementarity  NONE
OR
forward: 5' ccgaattcgcggccgcttctagagCTCTGGATAAC >>>>is it better?
-      Avoid polyXs  NO
-      3’ end: Avoid T (greater tolerance of mismatch) YES
-      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
- 3′ end region: only 2 G or C in the last 5 [Sayler Lab] YES
-      Length: 11
-      GC : 45% YES
-      Tm:  32 C
-      self complementarity  NONE
reverse: 5' cctactagtagcggccgctgcagAAAAGCCCGGCCCCGTC
-      Avoid polyXs  YES
-      3’ end: Avoid T (greater tolerance of mismatch) NO
-      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
-      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO
-      Length: 17
-      GC : 71%
-      Tm: 54 C
-      self complementarity  NONE
-  All potential self-annealing sites are marked in red (allowing 1 mis-match):
5'     AAAAGCCCGGCCCCGTC 3'
3' CTGCCCCGGCCCGAAAA   5'
5'  AAAAGCCCGGCCCCGTC  3'
3' CTGCCCCGGCCCGAAAA   5'
OR
reverse: 5' cctactagtagcggccgctgcagGCCCGGCCCCGTC
-      Avoid polyXs  NO (yinr de gg cc  tekrarları var)
-      3’ end: Avoid T (greater tolerance of mismatch) NO
-      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
-      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO
-      Length: 13
-      GC : 92% NO
-      Tm: 50 C
-      self complementarity  NONE
-      hairpin: NONE
pCooM
forward: 5' ccgaattcgcggccgcttctagagACAGCATGACGAG
-      Avoid polyXs  NO
-      3’ end: Avoid T (greater tolerance of mismatch) NO
-      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
-      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO
-      Length: 13
-      GC : 54 % NO
-      Tm: 40 C
-      self complementarity  NONE
-      hairpin: NONE
reverse: 5' cctactagtagcggccgctgcagCGCCACCTGGGCG
-      Avoid polyXs  NO
-      3’ end: Avoid T (greater tolerance of mismatch) NO
-      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
-      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO
-      Length: 13
-      Tm: 48 C
-      self complementarity  NONE
-      hairpin: NONE
Short primer design
pcooF: >>> OK     CATAAAAACTCTGGATAACTGTCATCTGGCCGACAGACGGGGCCGGGCTTTTTGTCG
-      Length: 57
-      GC : 53%
-      Tm : 75 C
forward: 5' CATAAAAACTCTGG
-      Avoid polyXs  NO
-      3’ end: Avoid T (greater tolerance of mismatch) YES
-      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
-      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO
-      Length: 14
-      GC : 36%
-      Tm : 32C
-      hairpin: NONE
-      self dimer (free energy):  0
-      cross dimer (free energy): 0
reverse: 5' CGACAAAAAGC
-      Avoid polyXs  YES
-      3’ end: Avoid T (greater tolerance of mismatch) YES
-      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
-      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO
-      Length: 11
-      GC : 45%
-      Tm : 32C
-      MW: 3343 g/mole
-      hairpin: NONE
-      self dimer (free energy):  0
-      cross dimer (free energy): 0
pcooM:  >>> OK  GAGTGTCGGCCAAGTGACG
forward: 5' ACAGCATGAC
-      Avoid polyXs  YES
-      3’ end: Avoid T (greater tolerance of mismatch) YES
-      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
-      3′ end region: only 2 G or C in the last 5 [Sayler Lab] YES
-      Length: 10
-      GC : 50%
-      Tm : 30C
-      MW: 3021 g/mole
-      hairpin: NONE
-      self dimer (free energy):  -2.3
-      cross dimer (free energy): 0
reverse: 5' AGTACGCCAC
-      Avoid polyXs  YES
-      3’ end: Avoid T (greater tolerance of mismatch) YES
-      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
-      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO
-      Length: 10
-      GC : 60%
-      MW: 2997 g/mole
-      hairpin: NONE
-      self dimer (free energy):  -2.0
-      cross dimer (free energy): 0
Preparation of DNA Component
DNA Components to be produced
Promoters:
pCooM
pCooF
pCooF C4G
pCooF C9G
pCooF G2C
pCooF G7C
Response Elements (RE)
pCooM
pCooF
pCooF C4G
pCooF C9G
pCooF G2C
pCooF G7C
1.Transformation
2.Plasmid purification
It is started with 5ml overnight culture from glycerol stock
Fermentas Gene jet miniprep kit is used
Final volume is 50ul
3. Restriction digestion for confirmation of the plasmids
Fermentas fast digest kit is used
4.Concentration measurement with alpha UV spectrophotometer
5.PCR for promoters
Primers:
Primers for pCooF promoter/mutants:
Long promoter
Forward: CCGAATTCGCGGCCGCTTCTAGAGTTCGGCGTCTTTTCATACCC
Reverse:  CTACTAGTAGCGGCCGCTGCAGCCTGGCGCCGAGTAAGC
Short promoter
forward: TTCGGCGTCTTTTCATACCC
reverse: CCTGGCGCCGAGTAAGC
primers for pCooM promoter:
long promoter
forward: CCGAATTCGCGGCCGCTTCTAGAGCAGCATGACGAGTGTCGGCCAAG
reverse: CCTACTAGTAGCGGCCGCTGCAGGAAGAACGGGAGTACGCCACCTG
short promoter
forward: CAGCATGACGAGTGTCGGCCAAG
reverse: GAAGAACGGGAGTACGCCACCTG
PCR reaction mix (fermantas) for 1 reaction:
10X Taq Buffer                   3ul
dNTPs                            3ul
25mM MgCl2                       3ul
f-primer                         3ul
r-primer                         3ul
5 u/ul Taq polymerase  0.25ul
2.5u/ul Pfu polymerase 0.015ul
Template                         3ul
Nuc free H2O                   11.75ul
TOTAL:                           30ul
PCR reaction conditions (Techne TC-512)
Contents
1 6.Agarose gel electrophoresis
2 1. Agarose gel electrophoresis for RE
2.1 12. Gel extraction
2.2 13.Concentration measurement with alpha uv spectrophotometer
3 2. EMSA
4 4. ITF
4.1 Conclusion:
6.Agarose gel electrophoresis
- % 1 agarose
-  75V
-  45 min
pCooF promoter
pCooF mutant C4G
pCooF mutantC9G
pCooF mutant G2C
pCooF G7C
pCooM promoter(long)
7.Gel extraction
Gene jet gel extraction kit is used
8.Concentration of samples with vacuum concentrator
1hr in 200mbar for 100 ul  reduction
9.Concentration measurement with alpha uv spectrophotometer
10.PCR for response element
Primers
primers for pCooF/mutants RE
forward: CATAAAAACTCTGG
reverse: CGACAAAAAGC
primers for pCooM RE
forward: ACAGCATGAC
reverse:     AGTACGCCAC
PCR reaction mix (fermantas) for 1 reaction:
10X Taq Buffer                  3ul
dNTPs                           3ul
25mM MgCl2                      3ul
f-primer                        3ul
r-primer                        3ul
5 u/ul Taq polymerase           0.25ul
2.5u/ul Pfu polymerase          0.015ul
Template                        3ul
Nuc free H2O                    11.75
TOTAL:                          30ul
PCR conditions
1. Agarose gel electrophoresis for RE
- % 1 agarose
- 100V
- 75 min
12. Gel extraction
Gene jet gel extraction kit is used
13.Concentration measurement with alpha uv spectrophotometer
Oligonucleotides that include response elements are too short (57bp).  Although we reach the good amount of products from the Agarose gel electrophoresis, we lost most of our samples during the gel extraction process, because the recovery of the gel extraction kit reduces with oligos shorter than 100bp. Therefore, we had to use promoters (~100bp) rather than response elements.  Unfortunately, ITC experiments cannot be performed because of inadequate production of DNA and protein components.
2. EMSA
151010-EMSA (w/ Lightshift Chemiluminescence EMSA Kit (PIERCE))
Gel
8.3ml 30% polyacrylamide solution (29:1)
2.5ml 10X TBE
39.2ml ddH2O
50ul TEMED
500ul APS (10%)
DNA&PROTEIN:
pCooM:
MW: ~66kDa
330ng in 10ul
330x10-9 / 66 kDa= 5 pmol
pCooF:
MW: 57 kDa
91.6ng in 10 ul
91.6x10-9 / 57kDa= 1.7 pmol
CooA:
MW: 48 kDa
1ug/ul in 500 ul
Oxidized protein is untreated CooA
Reduced protein is 2ul DTT added into 1000 ul CooA
Reduced +CO is degassed, reduced with DTT and 5 min CO is sparged.
Binding Reaction Component (w/ Lightshift Chemiluminescence EMSA Kit (PIERCE))
Reactions:
Reaction is incubated 20min at 4C and 20 min at room temperature before running
Pre run: 100V 45min at 4C
Run: 100V 1 h at 4C
Staining: After run is compleated, gel is staining with Et-Br by shaking  30 min to stain DNA.
Results:
In conclusion, there is not a meaningful retardation. Binding conditions may not be sustained because gel is not anoxic.  Not only reaction buffer but also running buffer and gel should have to be reduced for proper binding of CooA to the DNA
EMSA[211010] [He at al. (1995)]
Gel
6% polyacryamide gel
8.3ml 30% polyacrylamide solution (29:1)
2.5ml 10X TBE
39.2ml ddH2O
50ul TEMED
500ul APS (10%)
3 ml 170 mM sodium dithionite
DNA&PROTEIN:
pCooM:
MW: ~66kDa
330ng in 10ul
330x10-9 / 66 kDa= 5 pmol
pCooF:
MW: 57 kDa
91.6ng in 10 ul
91.6x10-9 / 57kDa= 1.7 pmol
CooA:
MW: 48 kDa
1ug/ul in 500 ul
Reduced protein is 2ul DTT added into 1000 ul CooA.
Reduced +CO is degassed, reduced with DTT and 5 min CO is sparged.
Reactions:
Reaction is incubated 20min at 4C and 20 min at room temperature before running
Pre run: 100V 45min at 4C
Run: 100V 1. 5 hrs  at 4C
Staining: After run is completed, gel is staining with Et-Br by shaking 15 min to stain DNA.
Results:
In conclusion, there is a little retardation.   Binding conditions may not be sustained because gel is not anoxic.  Not only reaction buffer but also running buffer and gel should have to be reduced for proper binding of CooA to the DNA.
EMSA [231010]
Gel
8.3ml 30% polyacrylamide solution (29:1)
2.5ml 10X TBE
39.2ml ddH2O
50ul TEMED
500ul APS (10%)
Argon is sparged for 5 min to get an anoxic gel
45min pre-run is completed at 100V before reaction in order to remove APS/TEMED
1.7mM sodium dithionate is added in to the running buffer
DNA&PROTEIN:
pCooF:
MW: 57 kDa
200 ng in 8 ul
200x10-9 / 57kDa= 3.5 pmol
pCooM:
MW: ~66kDa
264ng in 8ul
264x10-9 / 66 kDa= 4 pmol
CooA:
MW: 48 kDa
1ug/ul in  500ul
12ul is added to reaction mix.
Reduced protein is 2ul DTT added into 1000 ul CooA.
Reduced +CO is degassed, reduced with DTT and 5 min CO is sparged.
Binding Reaction
Binding solution (2X)
80ul 1M TRIS-HCl  (ph:7.6)
1120ul 25mM MgCl2
40ul 5M KCl
40ul 1M DTT
100ul glycerol
20 ul 10mg/ml BSA
20ul 170 mM sodium dithionate
580ul pure water
Reactions:
Reaction is incubated 20min at room temperature before running
Run: 100V 1 h at 4C
Staining: After run is compleated, gel is staining with Et-Br by shaking for 15 min to stain DNA.
Results:
We cannot get a clear image of the gel because of uneven polymerization of the gel. We think the argon amount is too much so our gel cannot solidify properly. DNA marker which cannot be open accurately is the evidence for failed polymerization
4. ITF
Intrinsic Tryptophan Fluorescence (ITF)
Samples
DNA:    28 Nm ; 56 Nm ; 74 nM :   93 nM pCooF  (eluted with elution buffer )
Protein: 50 nM reduced –CO bounded; reduced; oxidized CooA
Buffers
For oxidized CooA :  2X Buffer
-40 mM Tris-HCl ( pH: 7.6)
- 14 mM MgCl2
- 100 mM KCl
-10 % glycerol
For reduced –CO bounded & reduced CooA : 2X buffer
-40 mM Tris-HCl (ph:7.6)
- 14 mM MgCl2
- 100 mM KCl
-10 % glycerol
-20 mM DTT
-3.4 mM sodium dithionite
Parameters:
Excitation: 280 nm
Emission: 300- 400 nm
Slit number: 4 nm ( 2 turn)
Temperature: 25 C
Blank: buffer (1X)
Measurements
1 -Blank: 1X buffer ( for oxidized  , reduced-CO  / reduced  CooA )
2- 50 nM oxidized CooA  in oxidized buffer
3- 50 nM oxidized CooA+ 28 nM pCOOF + 20. Min
4- Negative control :50 nM oxidized CooA + 10 ul elution buffer + 20. Min
5- 50 nM oxidized CooA+ 56 nM pCOOF + 40. Min
6- Negative control : 50 nM oxidized CooA + 20 ul  elution buffer + 40. Min
7- 50 nM reduced –CO bounded CooA in reduced buffer
8- 50 nM reduced –CO bounded CooA + 28 nM pCOOF + 20 min
9- Negative control : 50 nM reduced –CO bounded CooA + 10 ul elution buffer + 20. min
10-50 nM reduced –CO bounded CooA + 56 nM pCOOF + 40. Min
11-Negative control :50 nM reduced –CO bounded CooA + 20 ul elution buffer + 40. Min
12- 50 nM reduced –CO bounded CooA + 74 nM pCOOF + 60. Min
13-Negative control: 50 nM reduced –CO bounded CooA + 30 ul elution buffer + 60. Min
14-50 nM reduced –CO bounded CooA + 93  nM pCOOF + 80. Min
15- Negative control: 50 nM reduced –CO bounded CooA + 40 ul elution buffer + 80.min
16-50 nM reduced  CooA  in reduced buffer
17- 50 nM reduced CooA + 28 nM pCOOF + 20. Min
18-Negative control: 50 nM reduced CooA + 10 ul elution buffer + 20. Min
19- 50 nM reduced CooA + 56 nM pCOOF + 40.min
20-Negative control: 50 nM reduced CooA + 20 ul elution buffer + 40.min
Result :
pink: reduced –CO bounded CooA  + buffer
yellow : reduced –CO bounded CooA + 28 nM pCOOF ( 20 .min )
green: NC : reduced- CO bounded CooA +10 ul elution buffer  (20.min )
blue: reduced – CO bounded CooA +  56 nM pCooF  (40. min )
purple: NC : reduced –CO bounded CooA + 20 ul elution  (40. Min )
yellow : red/Co 28 nM pcoof 20 .min
Blue line : reduced –CO bounded CooA + 56 nM pCOOf
White line :oxidized CooA + 56 nM pCOOF
Conclusion:
The structure of the CooA protein has two residues of Tryptophan. Carbon monoxide binding to protein and DNA binding to protein can cause conformational change in the protein structure so tryptophan orientation change in the protein . Intrinsic tryptophan fluorescence experiment shows that any fluorescence intensity change and fluorescence shift resulting from the change of tryptophan.  We see a little difference between binding of promoter of oxidized and reduced CO bounded CooA protein. Decrease in the intensity is obtained.  The structure of CooA is given above :
Retrieved from " http://2010.igem.org/Team:METU_Turkey/Delta "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
