<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_METU_Turkey_Software"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:METU_Turkey_Software">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:METU_Turkey_Software&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:METU_Turkey_Software&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:METU_Turkey_Software&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:METU_Turkey_Software" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:METU Turkey Software</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="home" class="item"><DIV class="homepage"><DIV id="homecontainer"><H3>Abstract</H3><P>As Synthetic Biology field is on the rise, iGEM also grows up and number of parts in parts registry increase with submission of more complex constructs each year. Our first milestone was to perform more efficient standardization on parts entry due to facing some difficulty while running our algorithms on the parts registry. We also used Software Requirements Specification, Software Design Description and Quality Plan approaches to define requirements for each part and building blocks, risks and design art elements of the designed software program.</P><P>Next, we have used graph theoretic modeling to visualize relations between parts and to standardize representation of the parts as much as possible. It will help us when we try to find input-output relations between either biobrick parts or constructs. By this way, our program BIOGUIDE will act as a guide for searhing biobricks and will provide alternative pathway choices to users for construction of the most reliable devices with respect to given inputs and expected outputs.</P></DIV></DIV></DIV><DIV id="team" class="item"><DIV class="content2"><DIV class="text"><H2>Team</H2><P>METU Turkey Software is an interdisciplinary team of 7 students and 
			4 advisors from various backgrounds such as Molecular Biology, Bioinformatics, 
			Computer Engineering and Computer Education and Instructional Technology. 
			We have put our knowledge and experience in our fields together to bring 
			a much needed solution to a daily problem in field of synthetic biology 
			for iGEM 2010</P><DIV class="smallbox"><H2 class="black">Tolga Can</H2><DIV class="smalltext"><P>Tolga Can received his PhD in Computer Science at the University 
					of California at Santa Barbara in 2004. He is currently an Assistant 
					Professor of the Department of Computer Engineering, Middle 
					East Technical University, Ankara, Turkey. His main research 
					interests are in bioinformatics, especially prediction and analysis 
					of protein-protein interaction networks, and statistical methods 
					such as graphical models and kernel methods.</P></DIV></DIV><DIV class="smallbox"><H2 class="black">Yeşim Aydın-Son</H2><DIV class="smalltext"><P>Yeşim, has received her M.D in 1999 from HÜTF, Ankara and 
					completed her Ph.D at University of TN, Knoxville on Genome 
					Science and Technology in 2006. After working as a research 
					fellow at City of Hope National Medical Center, Duarte,CA , 
					she has recently accepted her current position at METU Informatics 
					Institute as an Assistant Professor of Medical Informatics. 
					She is also the acting coordinator of the Bioinformatics Graduate 
					Program in METU. Main focus of her research is Genomic Biomarker 
					discovery and applications of biomarker research in Personalized 
					Medicine . Her research group is working on building a new integrated 
					application for genome wide association of SNP biomarkers and 
					discovery of genes and pathways related to diseases, where SNP 
					genotyping data from both microarray and next generation sequencing 
					experiments can be analyzed in all-in-one step. </P></DIV></DIV><DIV class="smallbox"><H2 class="black">Ömer Nebil Yaveroğlu</H2><DIV class="smalltext"><P>Ömer Nebil Yaveroğlu is currently a PhD Student in Imperial 
					College, London. He has worked as a teaching assistant between 
					2008 – 2010 in the Computer Engineering Department of Middle 
					East Turkey. Throughout his MSc studies, he tried to understand 
					the orthological similarities between the protein interaction 
					networks of different species using graph theory. He helped 
					the group as an advisor in the computing related discussions</P></DIV></DIV><DIV class="smallbox"><H2 class="black">Burak Yılmaz</H2><DIV class="smalltext"><P>I am a recent graduate of METU Molecular Biology and Genetics 
					department and now studying towards my masters degree on Molecular 
					Bioengineering at METU. My interest in synthetic biology did 
					start during my undergraduate years and after graduation I started 
					up the Sentegen company which is the first biotechnology based 
					company focused on synthetic biology in Turkey. I continue my 
					research and training in synthetic biology while also contributing 
					to the development of the field in my country. We need new scientific 
					revolutions to solve huge problems of life and emerging field 
					of synthetic biology is best candidate for biotechnological 
					revolution. I am interested in synthetic biology applications, 
					along with Lab-on-a-Chip devices for molecular biology techniques, 
					and we are designing gene synthesis chips to produce biobricks 
					- raw materials of synthetic garage biology- faster and cheaper. 
					I enjoy snowboarding, cycling and write poems.</P></DIV></DIV><DIV class="smallbox" style="margin: 0 auto; float: none"><H2 class="black">Muhammed Akif Ağca</H2><DIV class="smalltext"><P>In bachelor he has worked on Software Development, Database Management,  Instructional Technologies, and Bioinformatics. He was developing the software and leading the team. 
A new page to his life is starting at 2011 after becoming a graduate of Computer Education and Instructional Technologies. He says that &quot; If we can use technology and science efficiently we can solve too many problems in the world, but if can use. That is if we can manage those we can solve the problems and organize the world with minimum problems &quot; which is a study field Science Technology and Management ( STM )  which is the field he will start to work in 2011 &quot;.</P></DIV></DIV><DIV class="smallbox"><H2 class="black">Cihan Taştan</H2><DIV class="smalltext"><P>2010 is the last year for me as B.S degree at Molecular Biology 
					and Genetics department. Moreover, I am studying at Computer 
					Engineering as my minor department. My research will be on Scientist 
					position at Cancer and Virology Relationship (Let's say Viral 
					Oncology) by integrating novel technices of Bioinformatics and 
					Synthetic Biology approaches. This is my second year at iGEM.</P></DIV></DIV><DIV class="smallbox"><H2 class="black">Hassan Salehe Matar</H2><DIV class="smalltext"><P>Ascended up the hills of Kilimanjaro, crossed over the savannah 
					of Serengeti and finally landed on the country of Istanbul my 
					name is Hassan Salehe. I'm a final year undergraduate student 
					at the Department of Computer Engineering, Middle East Technical 
					University. In Metu Turkey Software I'm a database administrator 
					and a core Software developer. I'm interested in software development, 
					database management, Computer networks and Bioiformatics I do 
					marathon, I like swimming and I'm fond of action movies.Oh, 
					I was about to forget to tell you that I also like traveling. 
					Thanks!</P></DIV></DIV><DIV class="smallbox"><H2 class="black">Ayub Rokhman Wakhid</H2><DIV class="smalltext"><P>From the country of thousand islands, across the ocean he 
					came to Ankara. Now he is finishing his undergraduate study 
					at the Department of Computer Education and Instructional Technology, 
					Middle East Technical University. This is the first time he 
					joined IGEM. He is in design team in Metu Turkey Software. He 
					has interest in animation, web development, and instructional 
					technologies.</P></DIV></DIV><DIV class="smallbox"><H2 class="black">Muhammad Fakhry Syauqy</H2><DIV class="smalltext"><P>A senior undergraduate student of Computer Education and 
					Instructional Technology at Middle East Technical University. 
					He came far away from Indonesia to Ankara, Turkey, to make a 
					great leap on his life. His role in this team is designer. Together 
					with Ayub, they designs the team's wiki, poster. He is interested 
					in 2D and 3D designing, web developing and animation. He loves 
					playing football and working with computer. His motto is &quot;Possibilities 
					are limitless&quot;</P></DIV></DIV><DIV class="smallbox"><H2 class="black">Saygın Karaaslan</H2><DIV class="smalltext"><P>Our multimedia support and the core of our design and animation 
					team. Saygın, is a senior in Biology department at METU and 
					about to launch his own scientific animation company. After 
					graduation he will be continuing his academic studies on medical 
					informatics, scientific data visualization and 3D molecular 
					animations. He has recently completed the production of OCW 
					for molecular biology laboratory lectures as video-notes. Never 
					says no to a good soccer game or mafia II. We look forward for 
					the day for the premier of his documentary on &quot;History of Science&quot;</P></DIV></DIV><DIV class="smallbox"><H2 class="black">Yener Tuncel</H2><DIV class="smalltext"><P>I have graduated from METU Molecular Biology and Genetics 
					department and just started the Bioinformatics Graduate Program 
					this fall. My main research interests is in systems biology, 
					and its applications. Currently I am focused on genome wide 
					association of SNP biomarkers, where we will utilize systems 
					biology approaches for discovering disease gene and pathway 
					associations after highthrough-put genotyping studies. During 
					the course of our research on the iGEM project this summer as 
					a Molecular Biologist I worked on the standardization of the 
					parts information for our applicaiton database. Also, as a Bioinformatician 
					in training, contributed to the development of the algorithms 
					for the BioGuide software. Besides research, I develop educational 
					tools for biology and bioinformatics education and getting used 
					to do couples dancing. </P></DIV></DIV></DIV></DIV></DIV><DIV id="motivation" class="item"><DIV class="content2"><DIV class="text"><H2>Motivation</H2><P>Since 2008, we have been participating in iGEM as METU ( Middle East 
			Technical University) wet-lab team, and each year we have noticed the 
			increasing number of teams participating, along with an increase in 
			biobricks entries at <A href="http://www.partregistry.org">partregistry.org</A>. 
			While having more choices of biobricks to choose from is incredible, 
			searching for and choosing the appropriate parts is becoming a challenge. 
			This year during the construction process of iGEM biobrick parts for 
			our new project, we felt the need for an application to find interacting 
			parts based on an input/output model to design the genetic constructs. 
			Using a specialized software for searching the parts registry to find 
			possible biobricks to include into our construct would be much easy, 
			fast and accurate than manual. We have shared our need with a group 
			of friends who are software engineers, and initiated the METU_Turkey_SOFTWARE 
			team where we worked together over this summer to build the BIO-Guide 
			software.</P></DIV></DIV></DIV><DIV id="scope" class="item"><DIV class="content2"><DIV class="text"><H2>Scope and Future Aspects</H2><P>The <A href="http://www.partregistry.org">partregistry.org</A> is 
			a continuously growing collection of standard genetic parts that can 
			be mixed and matched to build synthetic biology devices and systems. 
			The Registry is based on the principle of &quot;get some, give some&quot;. Registry 
			users benefit from using the parts and information available in the 
			Registry for designing their own genetically engineered biological systems. 
			In exchange, the expectation is that Registry users will contribute 
			back to the information and the data on existing parts and will submit 
			new parts they have designed in order to improve this community resource.
			</P><P>As an expanding database <A href="http://www.partregistry.org">partregistry.org</A> 
			needs to be more organized and the standardization template needs to 
			be improved. Additionally, the potential of multiple ways of using each 
			part in different construct combination brings out the necessity for 
			an application to search through the database. BioGuide is the first 
			designed software that organizes over 1000 parts in
			<A href="http://www.partregistry.org">partregistry.org</A> as possible 
			atomics parts to build new biological device and systems for specific 
			input and outputs based on graph theory. The requirement of similar 
			applications and software tools are now inevitable in the emerging field 
			of synthetic biology. The innovative approach that makes the
			<A href="http://www.partregistry.org">partregistry.org</A> easy to use 
			for synthetic biology applications is the collection of standardized 
			parts that can be used in any combination with minimal effort under 
			one database. But while working on our algorithm to search for possible 
			combinations of parts depending on the given input and output, we have 
			realized that present standards are inadequate and parts registry form 
			must be improved. </P><P>In very near future a new format for parts registry form is needed 
			and few additional features should be implemented to have more control 
			on the database. We are planning to suggest a new format and features 
			for the parts registry based on the survey results we have received. 
			And planning to build the next version of Bioguide based on the revised 
			parts registry form. Along with using new parts registry standards we 
			will be improving the algorithm, so that the software can search through 
			more complex relations and returns all possible functional constructs.
			</P></DIV></DIV></DIV><DIV id="project" class="item"><DIV class="content2"><DIV class="text"><H2>Project Introduction</H2><P>As the field of Synthetic Biology is on the rise, iGEM is growing 
			up very fast and the number of parts in the parts registry is increasing 
			with the addition of more complex parts each day. After facing some 
			difficulty while running our algorithms on the parts registry, the need 
			for more effective standardization of parts entry was apparent. We have 
			investigated the information on parts in iGEM’s 2010 distribution and 
			reorganized the information on the parts registry forms according to 
			the needs of our algorithm. Then we have used graph theoretic modeling 
			to visualize the relations between iGEM Parts and to standardize the 
			representation of the parts as much as possible by graph theoretical 
			methods. This helped us to find input output relations between the parts. 
			Furthermore, our program BioGuide is now able to provide alternative 
			pathways to construct the most reliable and functional Biobrick devices 
			with respect to given inputs and expected outputs as a guide to Biobricks 
			parts registry.</P></DIV></DIV></DIV><DIV id="notebook" class="item"><DIV class="content2"><DIV class="text"><H2>Notebook</H2><DIV id="notebook2"><DIV class="month">
					January</DIV><DIV class="general"><P>Brain Storming about the iGEM. </P><UL><LI>What is iGEM.</LI><LI>Previous Wet-Lab Projects developed at METU.</LI><LI>What kind of projects can be developed as a software 
						team.</LI></UL><P>NOTE: The first software team in Turkey...</P></DIV><DIV class="month">
					February</DIV><DIV class="general"><UL><LI>Reading articles about iGEM WetLab and Software team 
						projects.</LI><LI>Looking for the members of the team.</LI><LI>Looking for the instructors who can consult the team.
						</LI></UL></DIV><DIV class="month">
					March</DIV><DIV class="general"><UL><LI>Employing a member to team interested in Synthetic Biology.</LI><LI>Reading articles about Synthetic Biology, Bioinformatics 
						and Bio Engineering.</LI><LI>Founding the team [ An instructor, and student members 
						]</LI></UL></DIV><DIV class="month">
					April</DIV><DIV class="general">
					At this month we have started regular workshop about Synthetic 
					Biology, Bioengineering, and Bioinformatics.
					<UL><LI>This month biologists in the team telling the needed 
						basics to software group.</LI></UL></DIV><DIV class="lower"><DIV class="week1" style="height: 150px;"><DIV id="week">
							Week 1</DIV><P>Workshop -1</P><P>[Biology basics, What is Synthetic Biology?, and the 
						works in this field ]</P></DIV><DIV class="week2" style="height: 150px;"><DIV id="week">
							Week 2</DIV><P>Workshop -2</P><P>[What is Synthetic Biology?, and the works in this field 
						]</P></DIV><DIV class="week3" style="height: 150px;"><DIV id="week">
							Week 3</DIV><P>Workshop -3</P><P>[iGEM, Parts, Biobricks, and Devices ]</P></DIV><DIV class="week4" style="height: 150px;"><DIV id="week">
							Week 4</DIV><P>Workshop – 4</P><P>[iGEM, Parts, Biobricks, and Devices ]</P></DIV></DIV><DIV class="month">
					May</DIV><DIV class="general">
					At this month we have completed our workshops, and as the masters 
					in this field started meeting with instructors. /
					<UL><LI>Meanwhile, looking for sponsors [We have designed a 
						document telling the iGEM, previous project and our project 
						generally and started to send it private companies who can 
						found us.] </LI><LI>This month it is turn to software group; they are transferring 
						the basics of software concepts to biologists in the team.
						</LI><LI>Furthermore, we are discussing about how we can apply 
						the basics of computer engineering to synthetic biology 
						and iGEM parts. </LI></UL></DIV><DIV class="lower"><DIV class="week1" style="height: 180px;"><DIV id="week">
							Week 1</DIV><P>Meeting -1</P><P>[First, discussion on articles that have been selected 
						by consulters. Then, brain storming about the selected iGEM 
						projects from previous years and our project ]</P></DIV><DIV class="week2" style="height: 180px;"><DIV id="week">
							Week 2</DIV><P>Meeting -2[with consulters ]</P><P>[Tellingtheprevious projects to consulters and telling 
						our initial idea about project. Then, brainstorming about 
						our project. ]</P></DIV><DIV class="week3" style="height: 180px;"><DIV id="week">
							Week 3</DIV><P>Meeting -3</P><P>[Basic databaseconcepts and iGEM parts. ]</P></DIV><DIV class="week4" style="height: 180px;"><DIV id="week">
							Week 4</DIV><P>Meeting -4</P><P>[What is ER Diagram and How we can develop a database 
						model for iGEM parts with ER Model. ]</P></DIV></DIV><DIV class="month">
					June</DIV><DIV class="general"><P>At this month software group is going on to tell the basics 
					of software development and programming and computer engineering, 
					discussions about the computer engineering approaches are continuing.</P><UL><LI>Furthermore, we have constructed the design group for 
						web page, poster, presentation, and an attractive animation 
						telling us.</LI></UL></DIV><DIV class="lower"><DIV class="week1" style="height: 250px;"><DIV id="week">
							Week 1</DIV><P>Meeting – 5</P><P>[Graph theory, Graph theoretic modeling, and graphical 
						modeling of iGEM parts. Using Input Output loops on iGEM 
						parts. ]</P></DIV><DIV class="week2" style="height: 250px;"><DIV id="week">
							Week 2</DIV><P>Meeting – 6 [ With Consulters ]</P><UL><LI>Project Description. </LI><LI>Our tasks on holiday.</LI><LI>Telling our project and basic concepts to design 
							team.</LI><LI>Discussion on storyboard for animation.</LI></UL></DIV><DIV style="height: 250px; background-color: #1e6dd4; padding: 10px;"><DIV id="week">
							Week 3 and Week 4</DIV><P align="center">HAVE a NICE HOLIDAY</P><P align="center">SEE YOU ON JULY 1 as a POWERFUL TEAM;
						<EM>“METU TURKEY SOFTWARE”</EM>.</P></DIV></DIV><DIV class="month">
					July</DIV><DIV class="general"><P>At this month we have started to develop application. And 
					divided the team to 3 groups [ Software - Gene – Design ].</P><UL><LI>Gene group is providing row data to software group by 
						extracting it from the part registery and other resources.</LI><LI>Software group is developing application.</LI><LI>Designers are learning new design tools, and applying 
						those to our project [Not all Members of the group working 
						actively for the team].</LI></UL><P><U>NOTE: Members are not strictly assigned to a group; this 
					is just for organization of tasks.</U></P></DIV><DIV class="lower"><DIV class="week1" style="height: 600px;"><DIV id="week">
							Week 1</DIV><P>Meeting-7</P><UL><LI>Take stock for current situation.</LI><LI>Discussion on web, poster, animation design.</LI><LI>Discussion on storyboard for animation.</LI><LI>Task analyses for each group [ Software, Gene, Design 
							].</LI></UL></DIV><DIV class="week2" style="height: 600px;"><DIV id="week">
							Week 2</DIV><P>Meeting – 8</P><P>Checking the tasks of each group. </P><P>· Software Group</P><UL><LI>Database Design</LI><LI>Interface for DB.</LI><LI>Designing a basic SRS and SDD to be able to state 
							the requirements of application exactly .</LI></UL><P>· Gene Group</P><UL><LI>Extracting I/O information for each part in part 
							registry according to specified standards by Gene group.</LI><LI>Discussion about expectation from the software.
							</LI></UL><P>· Design Group</P><UL><LI>Team Logo</LI><LI>Web site</LI><LI>Poster</LI><LI>Animation</LI><LI>Presentation</LI></UL></DIV><DIV style="height: 600px; background-color: #1e6dd4; padding: 10px"><DIV id="week">
							Week 3 and Week 4
							<P>DOING THE TASKS.</P></DIV></DIV></DIV><DIV class="month">
					August</DIV><DIV class="general"><P>At this month we have started to apply graph theory on iGEM 
					parts. We have specified nodes, edges, graph types. Furthermore, 
					we have started to develop a new “ Part Registery Form ” to 
					be able to more standardize the part entry to be able to apply 
					some algorithms on the parts more efficiently.</P></DIV><DIV class="lower"><DIV class="week1" style="height: 350px;"><DIV id="week">
							Week 1</DIV><P>Meeting -8</P><P>/ </P><UL><LI>Node data extraction algorithm.</LI><LI>Node description. </LI><LI>Visualization of nodes.</LI><LI>Pathway finding according to specified I/O properties.</LI><LI>Representing the nodes with original images.</LI></UL></DIV><DIV class="week2" style="height: 350px;"><DIV id="week">
							Week 2</DIV><P>Meeting -9</P><UL><LI>Whichone is node part or subparts or both are nodes 
							in different graphs?</LI><LI>Part Combination rules.</LI><LI>Web site, Poster Content</LI><LI>Animation storyboard. </LI><LI>Survey for new “Part Registery Standarts”.</LI><LI>NewPart Registery Form.</LI></UL></DIV><DIV class="week3" style="height: 350px;"><DIV id="week">
							Week 3</DIV><P>Meeting -10</P><P>/ </P><UL><LI>Extraction of Part Combination Rules </LI><LI>Web, poster, presentation contents generally.
							</LI></UL></DIV><DIV class="week4" style="height: 350px;"><DIV id="week">
							Week 4</DIV><P>Meeting -11</P><UL><LI>USTC and Berkeley projects.[ https://2009.igem.org/Team:USTC_Software 
							and https://2009.igem.org/Team:Berkeley_Software </LI><LI>Graphical representation of node relations. </LI><LI>Part Combination Rules </LI><LI>Subpart Combinations Rules </LI><LI>Expectations from the software (SRS: Functional; 
							Requirements) </LI></UL></DIV></DIV><DIV class="month">
					September</DIV><DIV class="general">
					At this time all bases for application were nearly to finish, 
					the software group was waiting row data from the gene group. 
					Meanwhile, they were working on code bases.</DIV><DIV class="lower"><DIV class="week1" style="height: 350px;"><DIV id="week">
							Week 1 </DIV><P>DOING THE TASKS. </P></DIV><DIV class="week2" style="height: 350px;"><DIV id="week">
							Week 2</DIV><P>Meeting -12</P><UL><LI>Final Database </LI><LI>Final Graphs </LI><LI>GUI </LI><LI>Expectations from the software (SRS: Functional 
							Requirements) (Suggestions)</LI><LI>Survey details</LI></UL></DIV><DIV class="week3" style="height: 350px;"><DIV id="week">
							Week 3</DIV><P>Meeting -13</P><UL><LI>Animation (Storyboard, timeline) </LI><LI>Web site (Suggestions to web site) </LI><LI>Poster (How we can use 3D stereoscopic image, How 
							we can tell the development progress and our concepts 
							by 3D effect etc...) </LI><LI>Presentation ( Suggestions about presentation )</LI></UL></DIV><DIV class="week4" style="height: 350px;"><DIV id="week">
							Week 4</DIV><P>Meeting -14</P><UL><LI>Final Graphs </LI><LI>GUI ( about 70 % is over ) </LI><LI>How to send the software to other teams for collaboration 
							( just general ideas, details will be talked later ).
							</LI><LI>With survey or not , can it shade the software?
							</LI><LI>What to ask to teams while sending?</LI></UL></DIV></DIV><DIV class="month">
					October</DIV><DIV class="general"><P>Now everything is nearly over, it is time to put everything 
					together.</P><UL><LI>Gene group explaining the methods that we have used 
						during project.</LI><LI>Software group finalizing, software, testing it importing 
						new function according to collaboration results with METU 
						TURKEY wet lab team., trying to solve infinite bugs…</LI><LI>Design group putting all together…</LI><LI>Meanwhile, all team is writing the content for web, 
						poster, and presentation.</LI></UL></DIV><DIV class="lower"><DIV class="week1" style="height: 170px;"><DIV id="week">
							Week 1</DIV><P>Meeting -15</P><UL><LI>BioGuide Application, missing points.</LI><LI>Content writing</LI><LI>Web, poster, animation</LI></UL></DIV><DIV style="float: left; width: 540px; height: 170px; font-weight: normal; background-color: #3fff72; padding: 10px;"><DIV id="week">
							Week 2, Week 3, &amp; Week 4
							<P>GOOD NEWS Infinite meetings started :) </P><UL><LI>Writing content [shared it, tasks assigned to 
								members according to their fields.]</LI><LI>Software; debugging, testing, adding functions…</LI><LI>Designers web, poster, presentation, animation, 
								importing content…</LI></UL></DIV></DIV></DIV><DIV class="month">
					November</DIV><DIV class="general"><P align="center">November 1 – 12 is our ticket dates,</P><P align="center">See you in BOSTON …</P><P align="center">METU TURKEY SOFTWARE</P></DIV></DIV></DIV><H3>Follow our discussion topic at <A href="http://groups.google.com.tr/group/software_MetuTurkey/topics">our googlegroups</A></H3></DIV></DIV><DIV id="download" class="item"><DIV class="content2"><DIV class="text"><H2>Download Executable and Code</H2><P>You can download BioGUIDE at <A href="http://sourceforge.net/projects/bioguide">http://sourceforge.net/projects/bioguide</A></P><P>Download &amp; install it and feel it's power!!</P></DIV></DIV></DIV><DIV id="miscellaneous" class="item"><DIV class="content2"><DIV class="text"><H2>Collaboration</H2><H3>Part Registry Survey</H3><P><A href="http://www.kwiksurveys.com/online-survey.php?surveyID=HIMIGG_469f28c4&amp;ooo=HKDDJHO_95dcbe36">
			Click here to go to survey page..</A></P><H3>Results</H3><P>(including responses between 10th -22nd of Oct 2010)</P><H4>General Profile of Participants</H4><UL><LI>The following teams are dedicated as collaborators with more 
				than 60% team participation are </LI><UL><LI>INSA-Lyon</LI><LI>Lethbridge</LI><LI>WashU</LI></UL><LI>Out of 244 participants between 10 to 22.10.2010, 57% of the 
				participant had scientific degrees from B.Sc to Professor and 18% 
				had graduate degrees. 18% of participants are enrolled in their 
				teams as either Instructors or Advisors. </LI><LI>95 teams have responded to the survey while we are still waiting 
				to hear from 33 teams. 74% teams participated with one or more members 
				to the survey. </LI><LI>75% of participants were interested with synthetic biology field 
				for academic purposes.</LI></UL><H4>Browsing the Registry of Standard Parts </H4><UL><LI><STRONG>56% of participants think that it is not easy to search 
				for the parts in Registry of Standard Parts.</STRONG> Many comments 
				indicate a need for a better search engine and more flexible keyword 
				search options, especially excepting aliases. Also many are longing 
				for recognizable parts names, which will ease searching with keyword.
				</LI></UL><P><I>Partnership with Google and enforcing standardized parts names 
			are suggested </I></P><P><I>As a global organization iGEM can offer the Parts Registry in 
			different languages and more illustrations describing how the system 
			works.</I></P><H4>Content of Registry of Standard Parts </H4><UL><LI><STRONG>57% of participants agree that the number of parts registered 
				in the Registry of Standard Parts is not enough for their projects.</STRONG></LI><LI><STRONG>55% of participants think that there are enough and 
				useful parts distributed in iGEM Plates that we can use in our projects.
				</STRONG></LI></UL><P>Even though most agree the number of parts in the registry is impressive, 
			still they find it limited when it comes to design different devices 
			for diverse applications especially in different species other than 
			E. Coli. Participants believe that if there are more functional standardized 
			parts, especially protein coding sequences and promoter-RBS , they can 
			design devices according to the needs of the community instead of designing 
			what can simply be assembled into a device. </P><P><I>Encouraging development of vectors and standards for new species 
			and new standardized parts in different research areas is suggested.
			</I></P><P><I>Enforcing submission of right DNA sequences and working conditions 
			for each part is suggested.</I></P><P><I>Few recommend expanding iGEM into a collaborative effort rather 
			than an undergraduate tournament, which will increase the number and 
			the diversity of the parts designed and submitted all throughout the 
			year. </I></P><H4>Submission to the Parts Registry </H4><UL><LI><STRONG>52% of participants said that they have not encountered 
				difficulties during submitting parts.</STRONG> Even though participants 
				are satisfied with the web interface of the registry, most complains 
				about the pSB1C3 as the new standard plasmid to submit DNA. </LI><LI><STRONG>71% of participants are like minded with our team's 
				opinion, which is that The nomenclature of part IDs such as construct, 
				device, composite parts, protein generator, is confusing as there 
				is no consensus on how to use them correctly.</STRONG></LI></UL><P><I>Terminology and categorization used on iGEM’s Parts Registry should 
			be re-described and correct use of terminology should be enforced during 
			the submission process.</I></P><UL><LI><STRONG>75% of participants agree that different, specified 
				submission interfaces should be designed for contructs, promoter, 
				RBS, CDS and terminals is needed during Registry of Standard Parts.</STRONG> 
				But, there are very strong and valid arguments against it such as, 
				losing the flexibility of the registry will not allow future submission 
				of unclassified parts. </LI></UL><P><I>We suggest keeping the parts submission interface as is, until 
			these concerns are addressed.</I></P><UL><LI><STRONG>75% of participants agree that Out-dated, un-available 
				and not-characterized parts in the Registry of Standard Parts should 
				be removed to an archive after the consent of the designer.</STRONG></LI></UL><P>“It would be great to see some sort of organization like this! I 
			agree that unavailable parts should be followed up on and removed if 
			necessary. I also think that parts which are not sufficiently documented 
			should be highlighted in some way. Once these parts are identified, 
			teams can actively characterize them as part of their projects or as 
			side projects.” </P><P>“Think about these things: (i) who decides when a part is out-dated, 
			and how can that person know that an old part cannot have a novel use 
			in the future? (ii) likewise, an uncharacterized part may be both characterized 
			and used in the future” </P><P><I>We suggest building a backup system, such as an archive, to sort 
			out the rarely used, un-available and un-categorized parts until they 
			are in line with the enforced standards.</I></P><UL><LI><STRONG>91% of participants have same opinion with us, which 
				is that standardization of the nomenclatures used for each different 
				composition of parts is necessary.</STRONG></LI></UL><H4>Standards that should be enforced and Additional New Standards
			</H4><P>According to our survey, from high rated to low, these standards 
			have been rated which has been used while assigning a name to parts
			</P><UL><LI><STRONG>33% Type of part</STRONG></LI><LI><STRONG>17% Input</STRONG></LI><LI><STRONG>17% Output</STRONG></LI><LI><STRONG>14% Version</STRONG></LI><LI><STRONG>10% Year</STRONG></LI><LI><STRONG>9% Group</STRONG></LI></UL><P>Along with above, having short recognizable part names along with 
			function and performance , Genbank/EMBL link and organism information 
			is important. </P><UL><LI><STRONG>93% of participants have said that for the parts that 
				are marked as “WORKS” distinguishing the parts with quantitative 
				experimental validation vs parts without this information is important.</STRONG> 
				Most participants have encountered with similar problems about parts 
				that don’t work under their lab conditions or works but not they 
				were claimed for. </LI><LI><STRONG>89% of participants have same opinion with us, which 
				is that iGEM should sub-categorize the “WORKS” comment into 1) “Quantitative” 
				for parts which are characterized with experiments and 2) “Qualitative” 
				for parts which are not characterized will be an appropriate measure 
				for standardization of Biobrick database.</STRONG></LI></UL><P><I>In order to overcome these problems we suggest enforcing the working 
			conditions title for the registry entrance, in order to collect quantitative 
			experimental details on submitted parts, which might slow down the registration 
			process but will definitely increase the quality of the database.</I></P><UL><LI><STRONG>61% of participants agree that POPS (Polymerase Per 
				Second) should be assigned to every part or biobricks with a promoter, 
				where appropriate. - 57% of participants have been agree that RIPS 
				(Ribosome per Second) should be assigned to every part or biobricks 
				with a RBS brick.</STRONG></LI></UL><P>Though most participants agree the need for POPS and RBS information 
			, they are concerned about the workload it would bring to individual 
			labs. </P><P>“To do this, the Registry need to define a reliable and easy method 
			of determining the PoPS for teams to use. However, I would say that 
			there are better systems for quantifying promoter output than PoPS, 
			and they should be used instead, if possible”. </P><UL><LI><STRONG>67% of participants have thought that entering POPS 
				information should not be mandatory while submitting new parts. 
				Similarly, 65% of participants disagree that entering RIBS information 
				should be mandatory while submitting new parts </STRONG></LI></UL><P>Even though the researchers feeling the need for this information 
			they are shying away from requesting it as a mandatory title for parts 
			registry as it would be difficult for underfunded and inexperienced 
			groups to perform these measurements. </P><P><I>We strongly suggest starting a forum on how to quantify the performance 
			of promoters and genes to bring an easy to measure standard for the 
			efficiency of the parts. Additionally iGEM should the responsibility 
			and provide the measurements for the each promoter and gene included 
			in the distributions. The second choice would be even better in terms 
			of standardization as all the measurement will be performed by one center 
			under similar conditions and with experienced researchers, which will 
			allow user to compare and contrast the efficiencies of the parts more 
			accurately. </I></P><UL><LI><STRONG>82% of participants have thought that information on 
				working conditions of the parts should be mandatory while submitting 
				new parts.</STRONG> Most find submiting the detailed experimental 
				information and working conditions is crucial and even easier than 
				submitting measurements of POPS or RBS. </LI></UL><H4>Definitions you would like to see at the Registry of Standard Parts
			</H4><UL><LI><STRONG>Transcriptional efficiency 13%</STRONG></LI><LI><STRONG>Protein lifetime 10%</STRONG></LI><LI><STRONG>Ribosome binding efficiency 10%</STRONG></LI><LI><STRONG>mRNA lifetime 9%</STRONG></LI><LI><STRONG>Translation initiation and efficiency 9%</STRONG></LI><LI><STRONG>Protein concentration 9%</STRONG></LI><LI><STRONG>Cooperative effects with other molecules 9%</STRONG></LI><LI><STRONG>Protein-DNA binding rates and efficiencies 8%</STRONG></LI><LI><STRONG>RNA polymerase affects 8% </STRONG></LI><LI><STRONG>System copy count 8%</STRONG></LI><LI><STRONG>Protein multimerization 6%</STRONG></LI></UL><P>Additional titles includes: Catalytic rates and affinities for substrates, 
			leakiness of promoter in lack of stimulus, POPS at various inducer/repressor 
			concentrations. </P><H4>Efficiency of the Database Entries </H4><UL><LI><STRONG>86% of participants would like to see a ranking/rating 
				system for the parts by the other iGEM users which will be one indication 
				of if a part is working and how well in different laboratories.</STRONG> 
				Few had concerns about how well the rating system will work for 
				rarely used parts while the widely used parts would even more popular 
				due the the rating system. Still many believes this would be one 
				futher towards a peer-reviewed quality control system for the parts.
				</LI><LI><STRONG>61% of participants agreed that parts should be updated 
				regularly by the designers, where most agreed at least when there 
				is new information on the parts.</STRONG> It has also been suggested 
				to give permission to all the users of that part for updating information.
				</LI><LI><STRONG>73% of participants have been agree with us that excluding 
				the low ranking parts or the parts with negative feedback from the 
				future plates will increase efficiency of the system.</STRONG> The 
				major concern about excluding any part is losing the variety of 
				parts in the database. Few recommends excluding only the parts that 
				are not working. </LI></UL><P>“Efficiency shouldn't be top priority in a database. First and foremost, 
			data is the top priority. Excluding those parts would make the system 
			more efficient” </P><P>“Some parts may be rare or new and have low efficiency, but can be 
			very important! Getting rid of them would eliminate any chance of improvement 
			to these parts, which not only a qualifier for an iGEM gold medal, but 
			also one of the focuses of biobricks.” </P><P><I>We suggest excluding the parts not-working, low rated or with 
			negative feedbacks from the annual distribution plates but still archive 
			them and make their data available through the parts registry. So the 
			while the individuals labs are receiving plates with higher rated, fully 
			working parts for their projects, anyone who wants to work on a more 
			exotic part can search through the achieves and re-vitalize the parts 
			stored there. The challenge of re-vitalization of parts can be encouraged 
			as an collaborative effort.</I></P><H4>New Options for the Parts Registry Database </H4><UL><LI><STRONG>96% of participants are like minded with us that it 
				will be useful to have a link out to the gene/protein information 
				of the parts and - %97 of participants have been agree that they 
				would like to know if a part is also involved in known biological 
				pathways.</STRONG></LI></UL><P><STRONG>For receiving pathway information more participants have 
			voted for NCBI Cog (59%) than KEGG pathways (38%) when the responses 
			for both has been distributed among the choices according to response 
			rates.</STRONG> Adding the blast option to the parts registry has also 
			been suggested to locate parts of interest. We are sure all of us would 
			like to see gene-protein and pathway information if these information 
			was integrated into the database and offered automatically for each 
			entry in the database.</P><P><I>We are planning to provide this information about the parts to 
			all parts registry users as a build-in option in the next version of 
			BioGuide in iGEM 2011. </I></P><H2>New Parts Registry Form Suggested for The New Standards</H2><H3>Description</H3><P>Warning Boxes: </P><UL><LI>If Out-dated, un-available and not-characterized parts exist 
				in the Registry of Standard Parts, bring to an archive after the 
				consent of the designer. Divide archive into three title: Out-dated, 
				un-available and not-characterized parts </LI><LI>Besides shown as “works”, in the works box there should be explanation 
				whether the part is characterized or non-characterized. </LI><LI>Parts should be updated regularly by the designers </LI><LI>Excluding the low ranking parts or the parts with negative feedback 
				from the future plates </LI></UL><P>Characterization Boxes: </P><UL><LI>transcriptional efficiency </LI><LI>mRNA lifetime </LI><LI>ribosome binding efficiency </LI><LI>translation initiation and efficiency </LI><LI>protein lifetime </LI><LI>protein concentration </LI><LI>protein multimerization </LI><LI>protein-DNA binding rates and efficiencies </LI><LI>cooperative effects with other molecules </LI><LI>RNA polymerase effects </LI><LI>system copy count </LI></UL><H3>Desription</H3><P>Search box </P><UL><LI>with click options </LI><LI>options: searched parts are:
				</LI><LI>Available </LI><LI>Length OK </LI><LI>Building </LI><LI>Planning </LI><LI>Missing </LI><LI>Unavailable </LI></UL><P>according to the clicks of above options, search is modified</P><H3>Description</H3><P>Assume on the part image; </P><UL><LI>part DNA sequence is not confirmed, then tag with &quot;non-confirmed 
				DNA sequence&quot;</LI><LI>non-characterized parts in the Parts Registry are not characterized 
				further, then it will be tagged as &quot;deprecated&quot;</LI></UL><P>also: </P><UL><LI>comment box stated that any team can make comment about experiences 
				with the part is opened </LI><LI>boxes which had been not filled with the data are highlighted;
				</LI><LI>transcriptional efficiency </LI><LI>mRNA lifetime </LI><LI>ribosome binding efficiency </LI><LI>translation initiation and efficiency </LI><LI>protein lifetime </LI><LI>protein concentration </LI><LI>protein multimerization </LI><LI>protein-DNA binding rates and efficiencies </LI><LI>cooperative effects with other molecules </LI><LI>RNA polymerase effects </LI><LI>system copy count </LI></UL><LI>if the part is not characterized but &quot;works&quot; then a &quot;Qualitative 
				part&quot; tag is added </LI><LI>besides &quot;works&quot;, &quot;Characterized&quot; or &quot;non-characterized&quot; box 
				is added </LI><LI>ranking/rating stars for the parts voted by the other iGEM users 
				which indicate how well the parts perform in different laboratories 
				is added. For example 4.5 star voted by 27 teams (number of stars 
				and number of votes) </LI></DIV></DIV></DIV><DIV id="project2" class="item"><DIV class="content2"><DIV class="text"><H2>Design</H2></DIV></DIV></DIV><DIV id="download2" class="item"><DIV class="content2"><DIV class="text"><H2>User Guide</H2><P>For new users whom want to interest with synthetic biology and for experienced scientists, in our program you have two options for Input Output properties ;</P><P>To find input-output related biobrick parts or larger constructs! If you want to provide specific chemical or physical inputs from external environment, you may choose these inputs from the list. if you want other specific inputs which will be expressed from a coding sequence, the list can provide these specified protein names.</P><P>Wanted features of outputs can be selected by either from list or entering individually into text boxes.</P><P>Then click on the “Show Parts ” button.</P><P>Then the program will show you the parts have specified I/O properties</P><P>To see the properties of a part just click on it.</P><P>After clicking, you will see a highlighted pathway on the 2nd graph. Which are the subparts of the clicked part. Showing the sub graph in network.</P></DIV></DIV></DIV><DIV id="miscellaneous2" class="item"><DIV class="content2"><DIV class="text"><H2>Human Practices</H2><P>iGEM’s parts registry is the only current database that holds information 
			and DNA for over 3000 standard parts for the use of synthetic biology 
			community. BioGUIDE is the first designed software that organizes the 
			parts in iGEM spring 2010 distribution as possible atomics parts to 
			build new biological device and systems for specific input and outputs 
			based on graph theory. We are the first group who has applied a novel 
			algorithm to search for input/output relations between the parts to 
			reveal possible construct assemblies. This new approach will chance 
			how the parts registry is used by the scientific community. And the 
			availability of software implementing our algorithm with a very user 
			friendly graphical user interface will allow all the users of the parts 
			registry to explore new and novel constructs according to their parameter 
			with ease. Additionally as the BioGUIDE software is an open source ware 
			, any user can contribute to the development of the application. So, 
			BioGUIDE will be improving with a collaborative effort, which will make 
			it even more widely used among the synthetic biologists. </P><P>We also had many collabrations with the teams such as, INSA-Lyon, 
			Lethbridge, WashU through our survey and WARSAW as we have participated 
			in their survey. Also our sister team METU_TURKEY is our main collaborator. 
			They have provided feedback BioGUIDE at different stages of the development. 
			And they had the chance to analyze their constructs input/output relations 
			throughout this collaboration. </P><P><STRONG>ALL TEAMS</STRONG>: The Parts Registry Survey that has been 
			developed by METU_TURKEY_SOFTWARE got response from 253 of participant 
			from 94 teams. Analysis of the survey results we have received so far 
			indicates that we were able outreach to the iGEM community and help 
			them verbalize their concerns and suggestions for the parts registry 
			standards and the maintenance of the database.</P><P><STRONG>WARSAW</STRONG>: Our team members have participated in their 
			survey abour iGEM participants profile. </P><P><STRONG>METU_TURKEY</STRONG>: Beta testing of the algorithm developed 
			and the BioGUIDE software has been performed by METU_TURKEY on few case 
			studies. Additionally they have tested their construct for 2010 competition 
			with our algorithm. </P></DIV></DIV></DIV><DIV id="project3" class="item"><DIV class="content2"><DIV class="text"><H2>Material</H2><P>Our main data source for BIO Guide Software Program was the available 
			background information of parts distributed in 2010 iGEM plates (Total 
			of three384-well plates of dried DNA) to the wetlab teams. This data 
			was available through both the parts registry main website (<A href="http://partsregistry.org/Main_Page">http://partsregistry.org/Main_Page</A>) 
			in XML format and parts registry libraries (http://partsregistry.org/assembly/libraries.cgi?id=31) 
			in Excel format. Data from parts with specific part IDs have been parsed 
			with a custom code developed to modify SAX Parser. Then, the rest of 
			the data which needs to be standardized according to biological importance 
			have been extracted from the Registry of Parts Page manually. The chemical 
			(IPTG, galactose etc.) or physical (UV irradiation, temperature etc) 
			external inputs and proteins synthesized from a biobrick coding sequence 
			can affect promoters on the parts. These effectors are identified under 
			the title “Input”. And the “Output” s of these effectors are classified 
			as inducers (a molecule that starts gene expression), repressors (blocker 
			of attachment of RNA polymerase to promoter), activators (increasing 
			the rate of transcription) and inhibitor s(decreasing the rate of transcription). 
			These standardizations on the database helped us to build the algorithm 
			based on input/output relationships. MySQL Server is used for Database 
			development and organization. All of our illustrations for ER and algorithm 
			is created in SmartDraw (trial version). Java Programming Language, 
			and NetBeans Development environment is used or for software development. 
			The graphical visualization of the software is done with Cyctoscape 
			and yfiles libraries (trial version) are used for the presentation of 
			graphical events. We have utilized css Javascripts for our webdesign. 
			Autodesk Maya 2011 with Academic Licence, Adobe Creative Suite 5 Master 
			Collection (Trial Version) have been used for animations and illustrations. 
			Video tutorial for the BioGUIDE has been created by camstudio and trial 
			versions of Flash and After Effects are also used for the videos.
			</P></DIV></DIV></DIV><DIV id="download3" class="item"><DIV class="content2"><DIV class="text"><H2>Supporting Tools</H2><UL><LI>SAX Parser ( modified )to parse XML files</LI><LI>Java ProramingLanguage, NetBeans Developement environment for 
				software developement</LI><LI>MySQL Server for DataBase</LI><LI>cystoscape for graph visualization</LI><LI>yfiles library for graph events</LI><LI>SmartDraw for illutration of ER and Algorithm</LI><LI>maya &amp; Cinema 4D for 3D animation, Adobe Master Collection and 
				Microsoft Expression Studio for design</LI><LI>CSS, Java Script for web</LI></UL></DIV></DIV></DIV><DIV id="miscellaneous3" class="item"><DIV class="content2"><DIV class="text"><H2>Safety</H2><P>Synthetic biology has the potential to impact many areas of society. 
			Synthetic biologists may use artificial molecules to reproduce emergent 
			behavior from natural biology, with the goal of creating artificial 
			life or seeking interchangeable biological parts to assemble them into 
			devices and systems that function in a manner not found in nature (Benner 
			and Sismour 2005, Endy 2005, Heinemann and Panke 2006, Luisi 2007, Serrano 
			2007).There is possibility of causing intentional or accidental harm 
			to humans, agriculture or the environment. While deliberate damage is 
			dealt with under the heading biosecurity, the potential unintended consequences 
			have to be considered under the term biosafety. As a software developer 
			, we have to consider all possible maliciously use of synthetic biology 
			tools. However, it is diffucult to understand for which purposesour 
			tools will be used, bestway avoiding garage bioterrorism is all parts 
			before adding to partsregistry must be checked , looked for toxic affect 
			and any environmental or human harmness. After scanning possible candidate 
			parts, a committe should decide whether novel part can be added parts 
			registry or not.And we can only warn the user about our intention while 
			building the application: </P><P>“BioGUIDE v1.0 software is <STRONG>FOR RESEARCH USE ONLY</STRONG>, 
			no medical or diagnostic use for applications of the novel Biobrick 
			constructs generated through our software has been described “</P><P>“No military (defense or combat) applications will be allowed in 
			future”</P></DIV></DIV></DIV><DIV id="project4" class="item"><DIV class="content2"><DIV class="text"><H2>Methods</H2><H3>Part Extraction Standards</H3><P>All information about the parts that are essential in experimental 
			setup of iGEM projects has been utilized. The information for the parts 
			available provided with all three 384 well plates in Spring 2010 distribution 
			have been standardized. Our standardization criteria have been discussed 
			in detail under Database Standardization. ER diagram has been generated 
			which simply describes the organization of the data. Around 70% of the 
			parts information has been fetched by the custom parsing code from XML 
			and Excel files provided by iGEM. Rest of the data had to be collected 
			and organized manually as the organization of these data cannot be standardized 
			to generate an algorithm. This step was one of the most time consuming 
			steps in our project. For each construct and Biobrick the information 
			collected was; Activity, Inducer, Activator, Repressor and Inhibitor 
			for promoters and Inducer, Activator, Repressor and Inhibitor information 
			valid for synthesized molecules (mostly proteins and RNA fragments etc.)</P><H3>Combination</H3><P>Rules (Image Combinations) In order to build our input/output relations 
			graphs first we run our algorithm on the real combination dataset which 
			contains all few thousand different possible combinations of the biobricks. 
			But after performing all combinations for the first few hundred biobricks 
			application’s rate slowed downed tremendously, which also become very 
			time consuming for displaying biobricks graphs. To overcome this bottleneck 
			we have developed a new strategy, where we have only used the construct 
			combinations of the biobricks distributed within the plates. Moreover, 
			according to information gathered from the subparts of the constructs 
			distrubuted, we also collected the subpart assembly order, such as 1st: 
			promoter, 2nd:rbs, 3rd:coding seq, any internal parts and the Last: 
			terminator. Each specific Biobrick type has been assigned a number as 
			a unique image ID from 1 to 19. Gathering the information on subparts 
			was not a direct forward process. ImageID assembly orders for each construct 
			has been used to extract the type information for each subpart with 
			that construct. This innovative approach helped us to reveal 400 possible 
			brick combinations present within the 3x384 well plates distributed 
			by iGEM in Spring 2010.</P></DIV></DIV></DIV><DIV id="download4" class="item"><DIV class="content2"><DIV class="text"><H2>Contact</H2><P>For critics, suggestion, or appraisal, you can contact us on
			<A href="mailto:software_metuturkey@googlegroups.com">software_metuturkey@googlegroups.com</A></P></DIV></DIV></DIV><DIV id="miscellaneous4" class="item"><DIV class="content2"><DIV class="text"><H2>Future Plan</H2><P>The application we have developed can be used by all iGEM members. 
			As the iGEMs database expands and the recognition of the field of Synthetic 
			Biology increases, data resources from other biological databases such 
			as NCBI might be needed to be integrated to the application. In such 
			a situation, extendibility of the application is vital. New data resources 
			and new functions should be added easily. </P><P>Before planning ahead, feedback from other teams and iGEM headquarters 
			about the BioGuide 1.0 will be collected. That will help us to fill 
			in the missing features of the application and check the theories which 
			are basis of our algorithms. </P><P>As our application is not geared towards any commercial use and will 
			stay as an academic application, keeping track of the weekly developmental 
			process on the wiki notebook environment was satisfactory. If the need 
			for a commercial application emerges, we should be utilizing professional 
			software development approaches to determine the exact requirements 
			and to facilitate the use of a common language between interdiciplinary 
			members in the team. </P><P>For constructing BioGuide 2.0 we have some plans. </P><H3>Short Term plan:</H3><P>Next year we are planning to generate BioGuide 2.0 by using all parts 
			data but inorder to do this we will update our part database but easyway 
			is standardization and reorganizing all parts in partsregistrty.org 
			according to our suggestion because re-organizing and normalization 
			are crucial. We are planning to add new tools to improve graphs.Our 
			ultimate aim is finding best pathway based on automated construction 
			and input-output relation. BioGuide 2.0 will be more faster because 
			we are planning to use OODBMS and all all platform will support BioGuide 
			2.0. </P><H3>Long term plan:</H3><P>We want to improve our algoritm and add more parameter to make graphs 
			more effective. Our dream is embeding our software into partsregistry.org 
			so no mere iGEMers will choose parts in real time by using our software.
			</P><H3>Suggestions based on PartsRegistry Survey Results</H3><P>First suggesiton is offering fartnership with Google for easy search 
			and founding a committe to enforce a standardized nomenclature for terminology 
			and parts registry entries. </P><P>We strongly suggest starting a forum on how to quantify the performance 
			of promoters and genes to bring an easy to measure standard for the 
			efficiency of the parts. Additionally iGEM should assume the responsibility 
			and provide the measurements for the each promoter and gene included 
			in the distributions. The second choice would beeven better in terms 
			of standardization as all the measurement will be performed by one center 
			under similar conditions and with experienced researchers, which will 
			allow user to compare and contrast the efficiencies of the parts more 
			accurately. We suggest excluding the parts not-working, low rated or 
			with negative feedbacks from the annual distribution plates but still 
			archive them and make their data available through the parts registry. 
			So the while the individuals labs are receiving plates with higher rated, 
			fully working parts for their projects, anyone who wants to work on 
			a more exotic part can search through the achieves and re-vitalize the 
			parts stored there. The challenge of re-vitalization of parts can be 
			encouraged as an collaborative effort. We are sure, all of us would 
			like to see gene-protein and pathway information if these information 
			was integrated into the database and offered automatically for each 
			entry in the database. We are planning to provide this information about 
			the parts to all parts registry users as a build-in option in the next 
			version of BioGuide in iGEM 2011. </P></DIV></DIV></DIV><DIV id="project5" class="item"><DIV class="content2"><DIV class="text"><H2>Database Standardization</H2><P>Two main focuses of our project was the organization of the available 
			information about Biobricks on iGEM’s website and development of a software 
			application to help synthetic biologists at the experimental set-up 
			level by providing all available construct combinations for any given 
			input and output relations ,which they can utilize for their own project.</P><P>Normalization and re-organization of the part information at iGEM’s 
			web site was needed in order to develop our application, which will 
			automatically search the possible construct combinations. For the organization 
			and analysis of the Biobricks, we used part info for Spring 2010 distribution. 
			The information on all three 384 well plates distributed by iGEM scrutinized 
			and checked individually to specify the standards available and needed. 
			iGEM is providing so many parts within a hierarchical way, but there 
			is no order in the information flow and no common standards. Furthermore, 
			the information bulk is being used in an ineffective manner. Some of 
			the parts distributed are known to be nonfunctional. Web pages for parts 
			contain lots of information, but majority of them, are again not ordered. 
			Moreover, some additional information had to be removed or replaced 
			in such a way that the information for parts can be used effectively. 
			And removal of the redundant bulk information related with parts at 
			iGEM’s web site had been recommended for future. </P><P>Although, the final standardization, which we have suggested is not 
			for general public use and it was urgently needed in order to satisfy 
			the needs of our algorithm. But, still it will be a valuable resource, 
			since it summarizes the basic information about the parts.</P><P>As the first step to build the proposed standardization template, 
			the headings selected related to parts are listed on Table 1. Submission 
			of part IDs for individual parts is an accepted and quite valuable way 
			of tracking information. Although, every part has unique partID, for 
			every part there is a need to assign unique part names as official iGEM 
			names. Part names will have an important role as they will be providing 
			the short description about the part, which synthetic biologists can 
			immediately recognize and utilize during the construction of unique 
			Biobricks. Additionally unique part names will be helpful to identify 
			the devices with more than one Biobrick in their constructs. Assignment 
			of unique and distinct names for parts describing their nature and content 
			will be helpful to researchers for the recognition of and search for 
			the parts.</P><H3>Headings Selected From Previous Entry Forms for Indication of Standardized 
			Information</H3><P>=========================================</P><P>PartID:</P><P>PartName:</P><P>Bricks:</P><P>BrickIDs:</P><P>ImageIDs:</P><P>RFC10:</P><P>RFC21:</P><P>RFC23:</P><P>RFC25:</P><P>=========================================</P><P><SPAN>Table 1: The table above basically describes and designates 
			qualities of parts which identifies their compositions and demonstrates 
			the status of previously assigned standards. PartID refers to the unique 
			ID number for parts including atomic parts and assemblies. PartName 
			refers to the given unique names to parts. Bricks, refers to the shortcut 
			names which specifies atomic parts. ImageIDs, refers to individual or 
			combination of numbers that are assigned by us. RFCs refers to the states 
			of parts based on RFC standards.</SPAN></P><P>iGEM both provides individual, atomic parts and pre-combined constructs 
			such as devices and systems. Availability of combined constructs is 
			important to the researchers as combining individual bio-bricks one 
			at a time will be very time consuming. These previously merged constructs, 
			serve as the repository for puzzle and they can be used for different 
			purposes. Up to date the largest and most trustworthy source, for synthetic 
			biology and its components, is iGEM’s parts registry. In 2010, iGEM 
			provided over 1000 parts that have initiated many projects. Having more 
			atomic parts available in the iGEM’s repository, will lead to the design 
			of more complex and robust constructs, and we would have a better chance 
			to design different constructs for unique purposes. Also, for the parts 
			that are already available, extra steps needs to be taken for the quality 
			control and surveillance of these products. The quality control of the 
			information for the parts is essential for the future of iGEM and synthetic 
			biology. Even though we have found pre-determined RFC standards useful 
			and included those to our standardized template, some individual parts 
			still requires re-organization of the information as RFC standards alone 
			for the functionality of parts, does not satisfy the needs for wet lab 
			biologists.</P><P>Without a question there is an urgent need to build a distinct and 
			specific database well organized with its own standards for synthetic 
			biology; however, development of such a database is not an easy task.</P><H3>Contact Information of Part Owners and Qualitative Group Comments 
			about Parts</H3><P>=========================================</P><P>Designers: Mail:</P><P>GroupFavorite:</P><P>StarRating:</P><P>Parameters:</P><P>=========================================</P><P><SPAN>Table 2: The above table simply depicts information about possessors 
			of parts and their contact information and the popularity of the parts 
			for groups. Parameters heading, refers distinctive experimental details 
			unique to the usage of parts which should be decided by groups.</SPAN></P><P>Second step for building the standardized template was to get the 
			phylogenic information about the parts development process which includes 
			the name of the group, designer and contact information, along with 
			the comments from the group on the parts they have submitted. Contact 
			information is especially important for iGEM as other groups who need 
			extra information about the available part can reach to the required 
			information. Even though contacting with the designers of the individual 
			parts which are available is highly encouraged by iGEM, unavailability 
			of contact information points at out the fact that iGEM’s parts registry 
			needs strong re-organization in order to serve to the synthetic biology 
			community properly.</P><P>Additionally, the “group favorite” and “starRating” fields are also 
			important for individual evaluation of the parts, which doesn’t get 
			the deserved attention from the iGEM groups. “Group Favorite” defines 
			the confidence on the part by the designer group. “StarRating” defines 
			the related part in terms of popularity and usage efficiency among the 
			groups. According to our observations, most groups are not aware of 
			either of the fields or they are used incorrectly or ineffectively. 
			For example for a part with a full reporter which is known to be functional 
			and gives precise and expected results the StarRating should be at least 
			2 stars, but for most of the parts in 2010 distribution, it is very 
			difficult to observe a part whose “StarRating” is above one. For quick 
			determination of functionality of the parts these two evaluations are 
			important so they have been included in the proposed standardization 
			template. But, as they were not properly used up to now for the re-organization 
			of the parts information during the development of our software application 
			we had to include all parts to our queries regardless of their evaluations 
			based on “Group Favorites” and “ StarRatings”</P><P>Second step for building the standardized template was to get the 
			phylogenic information about the parts development process which includes 
			the name of the group, designer and contact information, along with 
			the comments from the group on the parts they have submitted. Contact 
			information is especially important for iGEM as other groups who need 
			extra information about the available part can reach to the required 
			information. Even though contacting with the designers of the individual 
			parts which are available is highly encouraged by iGEM, unavailability 
			of contact information points at out the fact that iGEM’s parts registry 
			needs strong re-organization in order to serve to the synthetic biology 
			community properly.</P><P>Additionally, the “group favorite” and “starRating” fields are also 
			important for individual evaluation of the parts, which doesn’t get 
			the deserved attention from the iGEM groups. “Group Favorite” defines 
			the confidence on the part by the designer group. “StarRating” defines 
			the related part in terms of popularity and usage efficiency among the 
			groups. According to our observations, most groups are not aware of 
			either of the fields or they are used incorrectly or ineffectively. 
			For example for a part with a full reporter which is known to be functional 
			and gives precise and expected results the StarRating should be at least 
			2 stars, but for most of the parts in 2010 distribution, it is very 
			difficult to observe a part whose “StarRating” is above one. For quick 
			determination of functionality of the parts these two evaluations are 
			important so they have been included in the proposed standardization 
			template. But, as they were not properly used up to now for the re-organization 
			of the parts information during the development of our software application 
			we had to include all parts to our queries regardless of their evaluations 
			based on “Group Favorites” and “ StarRatings”</P><H3>Input and Output Characteristics of Parts</H3><P>=========================================</P><P>Parameters:</P><P>-Input:</P><P>• Promoter:</P><P>• Activity:</P><P>• Inducer:</P><P>• Activator:</P><P>• Repressor:</P><P>• Inhibitor:</P><P>• Promoter2:</P><P>• Activity:</P><P>• Inducer:</P><P>• Activator:</P><P>• Repressor:</P><P>• Inhibitor:</P><P>-Output:</P><P>• Reporter:</P><P>• Reporter2:</P><P>• Regulator:</P><P>• Inducer:</P><P>• Activator:</P><P>• Repressor:</P><P>• Inhibitor:</P><P>• Regulator2:</P><P>• Inducer:</P><P>• Activator:</P><P>• Repressor:</P><P>• Inhibitor:</P><P>-Working Condition:</P><P>=========================================</P><P><SPAN>Table 3: The table above elaborately describes the input relations 
			based on promoters and the output products based on the functional genes 
			and RNAs which are included within the parts. Working condition simply 
			describes any influencing factor or circumstance which is directly related 
			with the functional properties of parts.</SPAN></P><P>Third part of our standardization template includes parameters of 
			contingent input and output elements. These parameters are classified 
			into two groups for simplicity as presented on Table 3. This final part 
			of the standardization template includes the upmost important information 
			about the Biobricks that are required for the BioGuide Software to run 
			its searching algorithm.</P><P>Briefly, BioGuide application is designed to catch the input and 
			output relations of individual parts to examine possible Biobricks pathways 
			for specific input and output queries. In other words, at pre-experimental 
			stage, it helps wet lab biologists to design their unique constructs 
			by revealing possible alternative options for pre-determined purposes, 
			along with the primary paths. Our ultimate goal is to improve the algorithm 
			designed for iGEM 2010 and present a new version of the BioGuide in 
			iGEM 2011, which will provide optimum design of constructs for predetermined 
			parameters.</P><P>Most of the parts are composed of functional and nonfunctional constructs 
			which are formed by atomic parts. And every part should carry the information 
			for all of its atomic parts within itself. The “input” heading actually 
			stands for promoters. Parts with one or more promoters can be found 
			at iGEM’s Parts Registry. Along with the information on which and how 
			many promoters a part might have, the activity level of promoters are 
			also important to distinguish between a constitutively active promoter 
			or a promoter activated by specific physiological processes or states 
			etc. This information was crucial for us to dissect in order to run 
			our algorithm as it directly affects which inputs can activate the devices 
			or the systems.</P><P>Throughout our investigations on the Parts Registry, we found out 
			that much of the terminology was being used ambiguously. Although this 
			might not be vital for synthetic biologists, it is still endeavoring 
			to understand the function of certain regulatory elements which also 
			becomes a time consuming task for the researcher. Thus, we recommend 
			that the explanations of certain regulatory elements should be redefined 
			and fixed especially for synthetic biology for easy communication, sharing 
			and searching of information.</P><P>Common misuses of the terminology can guide us to figure out how 
			to construct a standard nomenclature for synthetic biology. We claim 
			that a standard nomenclature is urgently needed for synthetic biology 
			for the following reasons. First of all, synthetic biology is an emerging 
			research discipline and an industrial application area which is highly 
			promising. Secondly, redefinition of the terminology to build a standard 
			nomenclature is needed as some of the terms are prone to be used instead 
			of another causing problems related to misuse for the global communication 
			about synthetic biology. Lastly, the nomenclature has major importance 
			for the construction of a persistent and trustworthy database for synthetic 
			biology which serves for the information exhibition and exchange globally. 
			For instance, there are obvious misunderstandings about the words which 
			are predominantly used for regulation process. We have noticed that, 
			the terms “inhibitor” and “repressor” are being used as equivocally 
			in the part information pages. Like the lactose inhibitor protein, a 
			widely used DNA-binding transcriptional repressor, that have been labeled 
			both as “inhibitor” and “repressor” at iGEM’s Parts Registry. Similar 
			problems resulting from ambiguous use of terminology also observed with 
			regulatory elements. To sum up, we investigated all input elements for 
			promoters and classify these elements in terms of their function, affect 
			and required input element for them. So, we suggest that terminology 
			used for regulation of transcription should be defined clearly on iGEM’s 
			website and correct use of terminology should be enforced.</P><P>The second group of parameters was collected under the title “Output”, 
			which refers to products of functional genes. In contradiction, the 
			term “reporter” has also been described within the same list. Reporters 
			are also genes whose products, can be used for screening as an output. 
			According to our group, the usage of the term “reporter” for genes is 
			unnecessary and cause extra complexity for information distribution 
			and gives rise to discrepancies. Instead of using the term “reporter”, 
			predefined “gene” description should be used for genes, which can function 
			as reporters. The special information which is related with the characteristic 
			of that gene should also be presented on part info web page.</P><P>Furthermore, the same terminology “reporter” was used for both atomic 
			parts and composite bio-bricks. Also the overall image descriptions 
			for these were defined as “reporters”. We want to point out that using 
			same nomenclature for both atomic genes and for whole functional constructs 
			contributes to the complexity and makes specific explorations difficult 
			through the Parts Registry. So, assigning “reporter” for both atomic 
			parts and for whole constructs is not a good practice. Instead, we are 
			suggesting the usage of other available terminology for the parts listed 
			as reporters, which most of the constructs, now known as reporters, 
			can be grouped into, such as “protein generators”, “composite parts” 
			or “inverters”.</P><P>Devices are whole constructs which are functional and have specific 
			and distinct functions. But, as we have observed, unfortunately, the 
			term “device” is also being used for parts which are not functional 
			and do not have specific functional at all. Moreover, within the classification 
			of devices, we argue that some terms are also being used unnecessarily 
			and ambiguously. Devices are classified into five types which are protein 
			generators, reporters, inverters, receivers and senders, measurement 
			devices. For example iGEM defines protein generators as:</P><P>Protein generator = promoter + rbs +gene + terminator</P><P>Though we accept the definition for protein generators, we observed 
			that there exist numerous parts which are defined as protein generators 
			but actually most of them do not fit to the definition provided above. 
			Although some parts are not functional and do not generate proteins 
			at all, they are classified as protein generators, which makes searching 
			for the parts difficult in the registry. Furthermore, there are also 
			numerous parts which are defined as “composite parts” but actually they 
			fit to the same definition with protein generators. In order to overcome 
			the problem of misuse of device type we have extracted related image 
			ID information for the composite parts. Image ID information helped 
			us to correctly categorize composite parts depending on its individual 
			atomic parts and identify the ones with more than one function, such 
			as being both inhibitor and activator. In other words, we used image 
			and part IDs in order to merge an input for its outputs.</P><P>Subtitle working conditions, includes all the detailed information 
			about the experimental properties of parts, and the details about the 
			working process of individual parts and complete devices. Additionally, 
			we marked the subtitle “Working Condition” in our standardization template 
			as potentially the most important title that helps synthetic biologist 
			to better understand the parts functions at iGEM’s part registry database. 
			The main problem we have encounter with the subtitle “working condition” 
			is within most of the parts the details about working process is not 
			enough and not provided regularly. </P><H3>Examples of Misuse of Terminology:</H3><H4>For Composite Parts:</H4><P>PartID: BBa_S04055</P><P>PartName: Synthetic lacYZ operon</P><P>This part is functional and responsible for the production of LacY 
			and LacZ proteins. This part partially fits the definition for “composite 
			part” but actually should be a protein generator as it fits fully to 
			the definition of “protein generators”.</P><H4>For Protein Generators:</H4><P>PartID: BBa_J45299</P><P>PartName: PchA &amp; PchB enzyme generator</P><P>The part which is illustrated above actually fits the definition 
			for “composite part” but in part registry it is classified as protein 
			generator. This part can be functional but it needs a promoter. Even 
			though this part is not functional and is not capable of producing protein, 
			part registry assigns this product as protein generator. We suggest 
			that all parts in the registry, which are composed of more than one 
			atomic part and which are not functional on their own but can be functional, 
			should be classified as “composite parts”.</P><H4>For Reporters:</H4><P>PartID: BBa_J04451</P><P>PartName: RFP Coding Device with an LVA tag</P><P>This functional part is classified as “Reporter” in the parts registry 
			database. It is very clear that this part fits the same description 
			as Protein Generator in Biobrick part registry standards. Although, 
			this part has specific and known functional role, characterizing this 
			part as a reporter is unnecessary and contributes to the level of complexity 
			of information provided. Instead, we suggest that this part should be 
			classified as “protein generator” and related detailed information about 
			the specific function of this part, should be provided in the part information 
			page.</P><P>In conclusion, as mentioned above we tried to reorganize and normalize 
			the information about parts which is provided in part registry for 2010 
			in order to develop our algorithm for the BioGuide application. During 
			this process, we encountered some inconsistencies and misuses of the 
			terminology being used and also inadequacies about the information provided 
			about parts. First of all, we claim that a standard nomenclature should 
			be constituted for future use in the field of synthetic biology. Based 
			on the information gathered according to new nomenclature a professional 
			database should be constructed to address the needs of synthetic biology. 
			This will enable easy information exchange and exhibition globally. 
			Secondly, although there are enough information about parts exists on 
			parts registry database, the information which is provided for parts 
			need to be ordered urgently. Furthermore, there should be new experimental 
			standards which must be introduced to groups in the part submission 
			process for the subtitle “working condition”. These experimental standards 
			will be important because the experimental details about parts are not 
			satisfying the needs of wet-lab biologists for the design and the construction 
			of new Biobricks.</P></DIV></DIV></DIV><DIV id="miscellaneous5" class="item"><DIV class="content2"><DIV class="text"><H2>Sponsors</H2></DIV></DIV></DIV><DIV id="project6" class="item"><DIV class="content2"><DIV class="text"><H2>Algorithm</H2><P>In this section, the step by step functioning of our application, 
			along with the encapsulation of the algorithmic concepts of ‘standardization’ 
			of functional iGEM devices are depicted in pictorial forms called flowcharts. 
			Rectangular boxes represent the encapsulation of implementations of 
			the computer programs to perform the particular tasks stated in that 
			box on the flowcharts. These boxes are sometimes called subprograms, 
			objects or packages in Object Oriented software Engineering context. 
			The diamonds represent decision branching and they are found between 
			two rectangular boxes. The arrows show the direction in which subprograms 
			work and communicate. The subprogram at the head of the arrow starts 
			executing after the termination of the subprogram at the tail of the 
			arrow. Following flowcharts are the high level representations of our 
			algorithms developed for the BioGuide software.</P><H3>1</H3><P><SPAN>Diagram 1. Flowchart of collection, formatting and storage 
			of devices data algorithm</SPAN></P><P>Information about the iGEM parts had to be collected in a standardized 
			format for our application to function properly. Following data collection 
			custom subprograms is needed to parse and forward the data the application’s 
			database. In order to achieve this we have designed and implemented 
			the algorithm shown in diagram 1. In this algorithm, the first stage 
			was to find the list of part IDs of devices which were supplied by iGEM 
			in Spring 2010 distribution. This information has been collected from 
			two sources 1) plate files in excel format which was available online 
			2) device data provided in xml format, both provided by iGEM. The last 
			step in the algorithm was to send the collected partID data to the application’s 
			database.</P><H3>2.</H3><P><SPAN>Diagram 2. Flowchart for BioGuide execution before and during 
			user interaction</SPAN></P><P>Diagram 2 presents the main algorithm, which shows how BioGuide application 
			works. In BioGuide the major components are device and Biobrick graphs. 
			While the device graph represents input-output (promoter-regulator) 
			compatibility combination of iGEM devices, the Biobrick graph represents 
			combinations of atomic parts assembled in a device or system. The flowchart 
			shows how these graphs are created and embedded into the program, which 
			displays both of the graphs to the user when launched. Application presents 
			few interactive options to the user when started, which were shown on 
			the flowchart under the horizontal, bolded line. As shown on the diagram 
			2, there are four interactive tasks BioGuide can do, where the device 
			and Biobricks graphs are utilized. Upon clicking a node on a devices 
			or Biobricks graph, that node changes in size and color and the various 
			functions shown on the flowchart can be performed then after.</P></DIV></DIV></DIV><DIV id="project7" class="item"><DIV class="content2"><DIV class="text"><H2>Graph Modeling</H2><H3>Graphical Modeling for Bio-Guide</H3><H4>Introduction</H4><P>Graphical Modeling Theory has been applied to construct four different 
			graphs where relations of atomic parts, devices and systems and the 
			functional combinations that can build new constructs are presented 
			for the iGEMs parts registry database. Three graphs are composed of 
			iGEM devices and one graph is based on Biobricks. Each graph comprises 
			a set of vertices or nodes and a set of edges. In the set of nodes each 
			node represents a device, while in the set of edges each edge represents 
			the input-output combination of the nodes. These graphs are directed 
			graphs as the edges are created according to input-output combination. 
			All compatibilities between a regulator and a promoter of an edge is 
			created, where the source of this edge is the device with the corresponding 
			regulator and target of the edge is the device with the promoter in 
			concern.</P><P><SPAN>Fig. 1: A node representing a device</SPAN></P><P><SPAN>Fig. 2: Arrow representing an edge between two nodes</SPAN></P><P>The atomic structures used in our graphical model have been represented 
			in Figures 1 and 2. A node is represented with a solid circle where 
			the label, the part/device ID according to iGEM standards, of the device 
			is marked on the foreground. The blue arrows between nodes connect the 
			related devices, representing the input-output connectivity. End style 
			of the arrow helps us to determine the direction of the node, like in 
			Figure 2 where the node labeled BBa_S03520 is the source and BBa_JO9250 
			is the target.</P><H3>Directivity</H3><P>All the four constructed graphs build for BioGuide are directed graphs. 
			So that, for every edge there must be a single source and a target. 
			There is no single edge which is bidirectional. In mathematical form 
			this can be represented as:</P><P>If an edge e has node v as source and node w as target then the edge 
			can be expressed as</P><P>For a directed graph the combination (v, w) is totally different 
			from (w, v). Therefore,</P><P>The direction of the edges has been represented with the arrows, 
			as explained in Figure 2.</P><H3>Connectivity</H3><P>The nodes forming their own sub-graphs disconnected from the rest 
			of the nodes have been recognized, which showed us the presence of incompatibility 
			between few regulators and promoters of the devices. We have observed 
			this disconnection in all four of our graphs. The basis of the disconnection 
			has been shown in Figure 3, where the two sub-graphs without any edge 
			that connects them to the main graph has been presented on the right 
			hand side of the diagram. These features classify our graphs as disconnected 
			graphs [1].</P><P><SPAN>Fig. 3: A zoomed in screenshot showing two sub-graphs within 
			the disconnected graph.</SPAN></P><H3>&quot;Semi-Simplicity&quot;</H3><P>A simple graph is a graph in which no more than one edge contains 
			the same set of nodes. So, in a simple graph it is not possible to find 
			more than one edge with the same source and the same target. Additionally, 
			an edge with the same source and target, forming a loop is not allowed. 
			But, in synthetic biology it is possible to construct a device consisting 
			of devices or bio bricks of the same species or type. Accordingly, our 
			graphs are simple graphs with an exception of possible self-containing 
			loops, where the edge starts from and ends on the same node. Our graphs 
			have an exception of having loops and due to this permitted flexibility 
			our graphs are &quot;semi-simple&quot;.</P><P>For general information about graphs refer to:</P><P><SPAN><A href="http://en.wikipedia.org/wiki/Graph_(mathematics)">
			[1] http://en.wikipedia.org/wiki/Graph_(mathematics)</A></SPAN></P></DIV></DIV></DIV><DIV id="project8" class="item"><DIV class="content2"><DIV class="text"><H2>Results</H2><UL><LI>In order to develop BioGuide algorithm, there was a need to 
				reorganize and normalize parts information database in partsregistry.org. 
				Information of the parts was processed and developed from partsregistry.org 
				which includes the PartID, PartName, RFC values, etc , designer 
				and contact information and parameters to build a standardized template 
				for our algorithm to run smoothly. </LI><LI>Database standardization is done by analyzing iGEM Spring 2010 
				distribution and collecting information needed for the standardized 
				parts. Also new parameters are assigned according to parts input-output 
				model like : Inducer, Activator, Repressor, Inhibitor</LI><LI>For each promoter we have assigned their inducers or repressors 
				as input and for protein generator parts we have assigned generated 
				proteins as output which maybe activator or repressor of other devices.
				</LI><LI>iGEM devices are depicted in pictorial forms called flowcharts, 
				encapsulation of implementations , decision branching and direction 
				are milestones of our algorithm. Following flowcharts are the high 
				level representations of our algorithms developed for the BioGuide 
				software.</LI><LI>We run our algorithm on the real combination dataset in order 
				to build our input/output relations graphs. We also collected the 
				subpart assembly order, such as 1st: promoter, 2nd:rbs, 3rd:coding 
				seq, any internal parts and the Last: terminator. Each specific 
				Biobrick type has been assigned a number as a unique image ID from 
				1 to 19. </LI><LI>ImageID assembly orders for each construct have been used to 
				extract the type information for each subpart with that construct. 
				This innovative approach helped us to reveal 400 possible brick 
				combinations present within the 3x384 well plates distributed by 
				iGEM in Spring 2010</LI><LI>Four different graphs were constructed based on graphical modeling 
				theory where relations of atomic parts, devices and systems and 
				the functional combinations that can build new constructs are presented 
				for the iGEMs parts registry database. Three graphs are composed 
				of iGEM devices and one graph is based on Biobricks.</LI><LI>Each graph comprises a set of vertices or nodes and a set of 
				edges. In the set of nodes each node represents a device, while 
				in the set of edges each edge represents the input-output combination 
				of the nodes.</LI><LI>We arrange a survey to analyze user needs and response, According 
				to our survey results, 244 participants are completed our survey 
				between 10 to 22.10.2010, 57% of the participant had scientific 
				degrees from B.Sc to Professor and 18% had graduate degrees. 18% 
				of participants are enrolled in their teams as either Instructors 
				or Advisors.INSA-Lyon, Lethbridge, WashU are dedicated as collaborators 
				with more than 60% team participation. You can learn more on survey 
				results on our collaboration page</LI></UL></DIV></DIV></DIV><DIV id="project9" class="item"><DIV class="content2"><DIV class="text"><H2>Attribution and Contributions</H2><P>The BioGUIDE software is a proud product of METU_TURKEY_SOFTWARE group where the students have presented the problem and come up with the solution with minor advising. This project was not a part of an ongoing project in any of the advisors.</P><P>The biologist of the team have done the preliminary database standardization by analyzing and collected data from the iGEM Spring 2010 distribution. All the algorithms have been developed by the soon-to-be computer scientist in the group. The Parts registry survey and standardization of the database information on parts have been developed through a colloborotative effort within the group. </P><P>Additionally all the logo, illustrations, animations, video and other visual media and web design has been created by the design team in the group.</P></DIV></DIV></DIV><DIV id="project10" class="item"><DIV class="content2"><DIV class="text"><H2>Development Plans and Project Management</H2><P>The application we are developing can be used by all iGEM members and it’s data is increasing very fast as the iGEM becomes a bigger organization.  Moreover, other data resources like NCBI can be imported to this application, hence the user group of application is gets bigger and bigger. In such a situation, extendibility of the application is vital. New data resources and new functions should be added easily. </P><P>Before starting to think about those, first we have to ask to other teams and iGEM headquarters about the BioGuide 1.0 to see the missing points in our idea, and to check our theories. Until developing this idea we were not using professional software approaches. We were planning our weekly tasks and doing those as in the notebook. </P><P>After finding a novel software project we have to change our approach. We have to use professional software development approaches to develop such an application to be able to state the requirements exactly and to use a common language between software oriented and biology oriented members in the team.</P><P>As we mentioned above, our software can be extended, and new data resources can be imported. The requirements are changing and the problem can be understood better in time.  Extreme Programming Approach is developed for such software projects. That is the approach we are going to use in BioGuide2.0</P></DIV></DIV></DIV><DIV id="menu"><UL class="navmenu"><LI><DIV class="menutop menusingle"><A class="panel" href="#home">Home</A></DIV></LI><LI><DIV class="menutop menusingle panel"><A class="panel" href="#team">Team</A></DIV></LI><LI><DIV class="menutop menusingle panel"><A class="panel" href="#motivation">Motivation</A></DIV></LI><LI><DIV class="menutop menusingle panel"><A class="panel" href="#scope">Scope</A></DIV></LI><LI><DIV class="menutop"><A class="panel" href="#project">Project</A><DIV class="toggle toggle-closed">
				+</DIV></DIV></LI><LI><A class="panel" href="#project">Introduction</A></LI><LI><A class="panel" href="#project2">Design</A></LI><LI><A class="panel" href="#project3">Material</A></LI><LI><A class="panel" href="#project4">Methods</A></LI><LI><A class="panel" href="#project5">Database Standardization</A></LI><LI><A class="panel" href="#project6">Algorithm</A></LI><LI><A class="panel" href="#project7">Graph Modeling</A></LI><LI><A class="panel" href="#project8">Results</A></LI><LI><A class="panel" href="#project9">Attribution and Contributions</A></LI><LI><A class="panel" href="#project10">Development Plans</A></LI></UL><LI><DIV class="menutop menusingle panel"><A class="panel" href="#notebook">Notebook</A></DIV></LI><LI><DIV class="menutop panel"><A class="panel" href="#download">Download</A><DIV class="toggle toggle-closed">
				+</DIV></DIV></LI><LI><A class="panel" href="#download">Executable and Code</A></LI><LI><A class="panel" href="#download2">User Guide</A></LI><LI><A class="panel" href="#download3">Supporting Tools</A></LI><LI><A class="panel" href="#download4">Contact</A></LI><LI><DIV class="menutop panel"><A class="panel" href="#miscellaneous">Miscellaneous</A><DIV class="toggle toggle-closed">
				+</DIV></DIV></LI><LI><A class="panel" href="#miscellaneous">Collaboration</A></LI><LI><A class="panel" href="#miscellaneous2">Human Practices</A></LI><LI><A class="panel" href="#miscellaneous3">Safety</A></LI><LI><A class="panel" href="#miscellaneous4">Future Plan</A></LI><LI><A class="panel" href="#miscellaneous5">Sponsors</A></LI></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:METU_Turkey_Software">http://2010.igem.org/Team:METU_Turkey_Software</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:METU_Turkey_Software" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:METU_Turkey_Software" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:METU_Turkey_Software&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:METU_Turkey_Software&amp;oldid=209677" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>