<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_METU_Turkey_Delta"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:METU_Turkey/Delta">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:METU_Turkey/Delta&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:METU_Turkey/Delta&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:METU_Turkey/Delta&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:METU_Turkey/Delta" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:METU Turkey/Delta</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><TITLE>Home</TITLE></P><DIV id="wrapper"><P> </P><DIV id="tab_nav"><UL id="topnav"><LI class="active"><A href="https://2010.igem.org/Team:METU_Turkey">HOME</A></LI><LI><A href="https://2010.igem.org/Team:METU_Turkey/Team">TEAM</A></LI><LI><A href="https://2010.igem.org/Team:METU_Turkey/Project">PROJECT</A><SPAN><A href="https://2010.igem.org/Team:METU_Turkey/Project">Description</A> |
                                <A href="https://2010.igem.org/Team:METU_Turkey/Planning">Planning</A> |
                                
                                <A href="https://2010.igem.org/Team:METU_Turkey/Results_Discussion">Results &amp; Discussion</A> 

|
                       <A href="https://2010.igem.org/Team:METU_Turkey/Conclusions">Conclusions</A> |

                       <A href="https://2010.igem.org/Team:METU_Turkey/Future_Work">Future Work</A> |

                       <A href="https://2010.igem.org/Team:METU_Turkey/Safety">Safety</A> |

                       <A href="https://2010.igem.org/Team:METU_Turkey/References">References</A> |

                       <A href="https://2010.igem.org/Team:METU_Turkey/Acknowledgements">Acknowledgements</A> |
         
                               
		            </SPAN></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2010&amp;group=METU_Turkey">PARTS</A></LI><LI><A href="https://2010.igem.org/Team:METU_Turkey/Support_Actions">SUPPORT ACTIONS</A><SPAN><A href="https://2010.igem.org/Team:METU_Turkey/Collaboration">Collaboration</A> |
                				<A href="https://2010.igem.org/Team:METU_Turkey/Characterization_of_an_existing_part">Characterization of an Existing Part</A> 

|</SPAN></LI><LI><A href="https://2010.igem.org/Team:METU_Turkey/Notebook">NOTEBOOK</A><SPAN><A href="https://2010.igem.org/Team:METU_Turkey/Alpha">Alpha Team</A> |
                				<A href="https://2010.igem.org/Team:METU_Turkey/Bravo">Bravo Team</A> |
               					<A href="https://2010.igem.org/Team:METU_Turkey/Charlie">Charlie Team</A> |
               					<A href="https://2010.igem.org/Team:METU_Turkey/Delta">Delta Team</A> |
                				<A href="https://2010.igem.org/Team:METU_Turkey/Procedures">Procedures</A></SPAN></LI><LI><A href="https://2010.igem.org/Team:METU_Turkey/Gallery">GALLERY</A></LI><LI><A href="https://2010.igem.org/Team:METU_Turkey/Sponsors">SPONSORS</A></LI><LI><A href="https://2010.igem.org/Team:METU_Turkey/Links">LINKS</A></LI></UL></DIV>----<H2>PROTEIN-DNA INTERACTIONS</H2> Aim:
 To show the interaction thermodynamics of COOA protein and target promoter pCooF and pCooM and changes in binding affinity of protein with respect to mutational changes in its target  DNA. 

 Methods
 - Isothermal Titration Calorimetry (ITC)
 - Electrophoretic  Mobility Shift Assay (EMSA)
 - Intrinsic Tryptophan Fluorescence (ITF)

Both these methods require short oligonucleotides as DNA template and especially for ITC experiment we need high amounts of DNA template.  In order to propagate enough DNA, we design primers for promoters and sequences, which include response elements.  
 1.Isothermal Titration Calorimetry (ITC)

 Scenarios for DNA component preparation:

 30bp
 The possible sequence of oligonucleotide that we are planning to use in ITC measurements:
 GGA TAA CTG TCA TCT GGC CGA CAG ACG GGG     
           Response (RE) Response (RE)
             Element       Element 

-  MW :  18594.2 g/mole
 - GC content: 60% 
 -  Length: 30bp
 - Average MW of a base is 325 g/mole (our MW prediction is applicable acc to that info.)
 - for an ITC experiment min 1uM DNA ( max: 100uM) is required in sample cell (Vcell 1.8mL)
 - we will prepare 2.5 mL per experiment ( it is calculated as 2.5 nmole - 18kDa x 2.5nmole= 45ug)
 - for 1uM DNA we require 45ug DNA in 2.5 mL   

 If we propagate our oligonucleotide w/ PCR: 
 - min 70 bp is required for using a QIAGEN PCR purification kit &gt;&gt;&gt;it is  problem for us
 - max yield is 10ug 
 -  so we will use 5 kit per experiment for getting 45ug   


 
50bp 
 TAA AAA CTC TGG ATA ACT GTC ATC TGG CCG ACA GAC GGG GGC CGG GCT TT 
 - Length: 50bp 
 - MW: 30773g/mole ~31 kDa
  - for ITC 1uM (Vcell = 2.5 mL)    
  1umol in 1 L ; so we need 2.5nmole for 2.5 mL    
  1 mole is nearly 31000 g ; so we need 77.7 ug - w/ invitrogen plasmid purification kit we get 800ug plasmids from 100ml culture - we can extract 800ug plasmid from 2ml culture - if we assume that our vector plasmid is nearly 5000bp long, it will be 100 fold of our fragment DNA (50bp) - 800/100 we get 8ug fragments after restriction digestion.
 - w/ qiagen gel extraction kit we can get 10ug DNA once 
 - so, we need 8 kit gel extraction and 10 plasmid purification kit for one ITC experiment 
 - 50bp fragment recovery 50% (Fermentas)
 - for 50 bp we require ~80ug
 - one kit elutes 25x 0.05= 12.5ug  in once
 - for 80ug we require 80/12.5= 6.4= 7  kits are required


<H3>Experimental plan [Best Case]</H3> - Full plasmid is used as template 
 - Plasmids from  Mr Gene  : about 3 weeks 
 -Plasmids from Mr gene last year  
  -pAny-amphicilline
 -2300bp+gene 
 -5 ug
  - our DNA components is  137 bp &gt;&gt;&gt;Total palsmid lenght is  about 2500 bp. 
  - average MW of a base is 325 g/mole
 - MW: 325 x 2500 x 2 = 1625000 g/mole
  - V(sample cell) = 2.5 ml
  - we assume that we use 1 uM DNA 
 - so we need 2.5 nmole DNA
 - for 2.5 nmole we require 2.5 x 1625000= 4062500x10^-9 ng = 1063 x 10^-6 = ~4mg 
 - transformation of the plasmids (2days)
 - plasmid purification (3rd day)
  - Invitrogen PureLink™ HiPure Plasmid Gigaprep Kit
              - Cat. No. K2100-09
              - Unit Size 2 preps
              - DNA yield is 7.5mg- 10mg (depends on copy number type, size)
  - Invitrogen PureLink™ HiPure Plasmid Megaprep Kit
               - Cat. No. K2100-08
               - Unit Size 4 preps
               - DNA yield is 1.5mg-2.5mg (depends on copy number type, size)
 - we can get ~4mg plasmid by using 2 x megakit or 1x gigakit


 - confirmation of transformation w/ single cut and restriction digestion gel (3rd day)
 - Gel extraction (4th day) 
 - PureLink™ Quick Gel Extraction and PCR Purification Combo Kit
             - Cat. No. K220001
             - Unit Size: 50 prep
             - DNA yield is 40ug 
             - Unit size 50 prep

 - we can get 4mg plasmid by using 100 kit :( 
 -  we search plasmid isolation kit which gives more yield than usual.  



 Richie Rich (50 bp case)
 - 50 bp seq is 77.5 ug/mol (~80 ug/mol)
 - we need 2.5 ml so 2.5 nmol for 1 uM in sample cell
 - IDT sends 100 nmol (4 nmol after HPLC) ds 50 bp oligo which costs 200 USA dolars in total
 - we can perform 2 experiments from one oligo order
 - we need 2.5 nmol DNA per experiment. If we make optimization with WT Response Element,  we will make al least 8 experiment. 
 - w/ mutants 10 oligos in total
 - 10 experiment x 2.5 nmol = 25 nmol (after HPLC)
 - To be ordered 625 nmol &gt;&gt;&gt; 1.750 TL &gt;&gt;&gt; ~1250 USA dollars :(

 IDT Contact 
 - 100 nmole  DNA is € 0.46 /base. 
 - Annealing is € 23,00/duplex.
 - For 50 bp cost is € 69,00. 
 - Shipping to Turkey is € 50,00.
 - lyophilized 
 - You should have 6 OD of duplex which is 12 nmole depending upon sequence composition.

 Primer Design  
 pcooF

 forward: 5' TAAAAACTCTGGATAAC 

      -      Avoid polyXs YES
      -      3’ end: Avoid T (greater tolerance of mismatch) YES
      -      3′ end: G or C (stronger clamping to the template) [Fernandez] YES
      -      3′ end region: only 2 G or C in the last 5 [Sayler Lab] YES 
      -      Length: 17
      -      GC : 29%
      -      Tm : 37C
      -      self complementarity  NONE
 OR
 forward: 5' ccgaattcgcggccgcttctagagCTCTGGATAAC &gt;&gt;&gt;&gt;is it better?
    
  -      Avoid polyXs  NO
  -      3’ end: Avoid T (greater tolerance of mismatch) YES
  -      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
  -      3′ end region: only 2 G or C in the last 5 [Sayler Lab] YES 
  -      Length: 11
  -      GC : 45% YES
  -      Tm:  32 C
  -      self complementarity  NONE

  reverse: 5' cctactagtagcggccgctgcagAAAAGCCCGGCCCCGTC
  -      Avoid polyXs  YES
  -      3’ end: Avoid T (greater tolerance of mismatch) NO
  -      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
  -      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO 
  -      Length: 17
  -      GC : 71%
  -      Tm: 54 C 
  -      self complementarity  NONE
  -  All potential self-annealing sites are marked in red (allowing 1 mis-match):

  5'     AAAAGCCCGGCCCCGTC 3'
  3' CTGCCCCGGCCCGAAAA   5'

  5'  AAAAGCCCGGCCCCGTC  3'
  3' CTGCCCCGGCCCGAAAA   5'

  OR
  reverse: 5' cctactagtagcggccgctgcagGCCCGGCCCCGTC
  -      Avoid polyXs  NO (yinr de gg cc  tekrarları var)
  -      3’ end: Avoid T (greater tolerance of mismatch) NO
  -      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
  -      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO 
  -      Length: 13
  -      GC : 92% NO
  -      Tm: 50 C
  -      self complementarity  NONE
  -      hairpin: NONE

  pCooM
  forward: 5' ccgaattcgcggccgcttctagagACAGCATGACGAG   
  -      Avoid polyXs  NO 
  -      3’ end: Avoid T (greater tolerance of mismatch) NO
  -      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
  -      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO 
  -      Length: 13
  -      GC : 54 % NO
  -      Tm: 40 C
  -      self complementarity  NONE
  -      hairpin: NONE

  reverse: 5' cctactagtagcggccgctgcagCGCCACCTGGGCG  
  -      Avoid polyXs  NO 
  -      3’ end: Avoid T (greater tolerance of mismatch) NO
  -      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
  -      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO 
  -      Length: 13
  -      Tm: 48 C
  -      self complementarity  NONE
  -      hairpin: NONE

  Short primer design 
  pcooF: &gt;&gt;&gt; OK     CATAAAAACTCTGGATAACTGTCATCTGGCCGACAGACGGGGCCGGGCTTTTTGTCG
  -      Length: 57
  -      GC : 53%
  -      Tm : 75 C
  forward: 5' CATAAAAACTCTGG
  -      Avoid polyXs  NO
  -      3’ end: Avoid T (greater tolerance of mismatch) YES
  -      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
  -      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO 
  -      Length: 14
  -      GC : 36%
  -      Tm : 32C
  -      hairpin: NONE
  -      self dimer (free energy):  0
  -      cross dimer (free energy): 0

  reverse: 5' CGACAAAAAGC
  -      Avoid polyXs  YES
  -      3’ end: Avoid T (greater tolerance of mismatch) YES
  -      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
  -      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO 
  -      Length: 11
  -      GC : 45%
  -      Tm : 32C
  -      MW: 3343 g/mole
  -      hairpin: NONE
  -      self dimer (free energy):  0
  -      cross dimer (free energy): 0

  pcooM:  &gt;&gt;&gt; OK  GAGTGTCGGCCAAGTGACG
  forward: 5' ACAGCATGAC
  -      Avoid polyXs  YES
  -      3’ end: Avoid T (greater tolerance of mismatch) YES
  -      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
  -      3′ end region: only 2 G or C in the last 5 [Sayler Lab] YES 
  -      Length: 10
  -      GC : 50%
  -      Tm : 30C
  -      MW: 3021 g/mole
  -      hairpin: NONE
  -      self dimer (free energy):  -2.3
  -      cross dimer (free energy): 0

  reverse: 5' AGTACGCCAC
  -      Avoid polyXs  YES
  -      3’ end: Avoid T (greater tolerance of mismatch) YES
  -      3′ end: G or C (stronger clamping to the template) [Fernandez]  YES
  -      3′ end region: only 2 G or C in the last 5 [Sayler Lab] NO
  -      Length: 10
  -      GC : 60%
  -      MW: 2997 g/mole
  -      hairpin: NONE
  -      self dimer (free energy):  -2.0
  -      cross dimer (free energy): 0



  Preparation of DNA Component

  DNA Components to be produced

  Promoters:

  pCooM
  pCooF
  pCooF C4G
  pCooF C9G
  pCooF G2C
  pCooF G7C

  Response Elements (RE)

  pCooM
  pCooF
  pCooF C4G
  pCooF C9G
  pCooF G2C
  pCooF G7C

  1.Transformation 

  2.Plasmid purification 
  It is started with 5ml overnight culture from glycerol stock
  Fermentas Gene jet miniprep kit is used
  Final volume is 50ul 

  3. Restriction digestion for confirmation of the plasmids
 Fermentas fast digest kit is used
<P>  4.Concentration measurement with alpha UV spectrophotometer
</P><P>  5.PCR for promoters
  Primers: 
  Primers for pCooF promoter/mutants: 
  Long promoter
</P><P>  Forward: CCGAATTCGCGGCCGCTTCTAGAGTTCGGCGTCTTTTCATACCC
  Reverse:  CTACTAGTAGCGGCCGCTGCAGCCTGGCGCCGAGTAAGC
</P><P>  Short promoter 
</P><P>  forward: TTCGGCGTCTTTTCATACCC
  reverse: CCTGGCGCCGAGTAAGC
</P><P>  primers for pCooM promoter:
</P><P>  long promoter
</P><P>  forward: CCGAATTCGCGGCCGCTTCTAGAGCAGCATGACGAGTGTCGGCCAAG
  reverse: CCTACTAGTAGCGGCCGCTGCAGGAAGAACGGGAGTACGCCACCTG
</P><P>   short promoter
</P><P>   forward: CAGCATGACGAGTGTCGGCCAAG
   reverse: GAAGAACGGGAGTACGCCACCTG
</P><P>  PCR reaction mix (fermantas) for 1 reaction:
</P><P>  10X Taq Buffer                   3ul
  dNTPs                            3ul
  25mM MgCl2                       3ul
  f-primer                         3ul
  r-primer                         3ul
  5 u/ul Taq polymerase  0.25ul
  2.5u/ul Pfu polymerase 0.015ul
  Template                         3ul
  Nuc free H2O                   11.75ul
</P><P>  TOTAL:                           30ul
</P><P>  PCR reaction conditions (Techne TC-512)

</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#6.Agarose_gel_electrophoresis"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">6.Agarose gel electrophoresis</SPAN></A></LI><LI class="toclevel-1"><A href="#1._Agarose_gel_electrophoresis_for_RE"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">1. Agarose gel electrophoresis for RE</SPAN></A></LI><LI class="toclevel-2"><A href="#12._Gel_extraction"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">12. Gel extraction</SPAN></A></LI><LI class="toclevel-2"><A href="#13.Concentration_measurement_with_alpha_uv_spectrophotometer"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">13.Concentration measurement with alpha uv spectrophotometer</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#2._EMSA"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">2. EMSA</SPAN></A></LI><LI class="toclevel-1"><A href="#4._ITF"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">4. ITF</SPAN></A></LI><LI class="toclevel-2"><A href="#Conclusion:"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Conclusion:</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="6.Agarose_gel_electrophoresis">6.Agarose gel electrophoresis</SPAN></H3><P>   - % 1 agarose
   -  75V
   -  45 min
</P><P>  pCooF promoter  

</P>  pCooF mutant C4G  

  pCooF mutantC9G

  pCooF mutant G2C

  pCooF G7C     

  pCooM promoter(long)

<P>  7.Gel extraction
  Gene jet gel extraction kit is used
</P><P>  8.Concentration of samples with vacuum concentrator
	1hr in 200mbar for 100 ul  reduction
</P><P>  9.Concentration measurement with alpha uv spectrophotometer
</P><P>  10.PCR for response element
  Primers
   primers for pCooF/mutants RE
     forward: CATAAAAACTCTGG
     reverse: CGACAAAAAGC
</P><P>   primers for pCooM RE
    forward: ACAGCATGAC    reverse:     AGTACGCCAC
</P><P>  PCR reaction mix (fermantas) for 1 reaction:
</P><P>  10X Taq Buffer                  3ul
  dNTPs                           3ul
  25mM MgCl2                      3ul
  f-primer                        3ul
  r-primer                        3ul
  5 u/ul Taq polymerase           0.25ul
  2.5u/ul Pfu polymerase          0.015ul
  Template                        3ul
  Nuc free H2O                    11.75
</P><P>  TOTAL:                          30ul
</P><P>  PCR conditions

</P><H2><SPAN class="mw-headline" id="1._Agarose_gel_electrophoresis_for_RE"> 1. Agarose gel electrophoresis for RE</SPAN></H2><P>  - % 1 agarose
  - 100V
  - 75 min

</P><H3><SPAN class="mw-headline" id="12._Gel_extraction">12. Gel extraction </SPAN></H3><P>  Gene jet gel extraction kit is used
</P><H3><SPAN class="mw-headline" id="13.Concentration_measurement_with_alpha_uv_spectrophotometer">13.Concentration measurement with alpha uv spectrophotometer</SPAN></H3><P>  Oligonucleotides that include response elements are too short (57bp).  Although we reach the good amount of products from the Agarose gel electrophoresis, we lost most of our samples during the gel extraction process, because the recovery of the gel extraction kit reduces with oligos shorter than 100bp. Therefore, we had to use promoters (~100bp) rather than response elements. 
Unfortunately, ITC experiments cannot be performed because of inadequate production of DNA and protein components. 
</P><H2><SPAN class="mw-headline" id="2._EMSA"> 2. EMSA</SPAN></H2><P>  151010-EMSA (w/ Lightshift Chemiluminescence EMSA Kit (PIERCE))
  Gel
  8.3ml 30% polyacrylamide solution (29:1)
  2.5ml 10X TBE 
  39.2ml ddH2O
  50ul TEMED
  500ul APS (10%)
  DNA&amp;PROTEIN:
  pCooM:  
  MW: ~66kDa
  330ng in 10ul
  330x10-9 / 66 kDa= 5 pmol
  pCooF:
  MW: 57 kDa
  91.6ng in 10 ul
  91.6x10-9 / 57kDa= 1.7 pmol
  CooA:
  MW: 48 kDa 
  1ug/ul in 500 ul 
  Oxidized protein is untreated CooA
  Reduced protein is 2ul DTT added into 1000 ul CooA
  Reduced +CO is degassed, reduced with DTT and 5 min CO is sparged. 
  Binding Reaction Component (w/ Lightshift Chemiluminescence EMSA Kit (PIERCE))

</P><P>  Reactions:

</P><P>  Reaction is incubated 20min at 4C and 20 min at room temperature before running
  Pre run: 100V 45min at 4C
  Run: 100V 1 h at 4C
  Staining: After run is compleated, gel is staining with Et-Br by shaking  30 min to stain DNA.
  Results:

</P><P>  In conclusion, there is not a meaningful retardation. Binding conditions may not be sustained because gel is not anoxic.  Not only reaction buffer but also running buffer and gel should have to be reduced for proper binding of CooA to the DNA
</P><P>  EMSA[211010] [He at al. (1995)]
   Gel
  6% polyacryamide gel 
  8.3ml 30% polyacrylamide solution (29:1)
  2.5ml 10X TBE 
  39.2ml ddH2O
  50ul TEMED
  500ul APS (10%)
  3 ml 170 mM sodium dithionite 
  DNA&amp;PROTEIN:
  pCooM:  
  MW: ~66kDa
  330ng in 10ul
  330x10-9 / 66 kDa= 5 pmol
</P><P>  pCooF:
  MW: 57 kDa
  91.6ng in 10 ul
  91.6x10-9 / 57kDa= 1.7 pmol
  CooA:
  MW: 48 kDa
  1ug/ul in 500 ul 
  Reduced protein is 2ul DTT added into 1000 ul CooA. 
  Reduced +CO is degassed, reduced with DTT and 5 min CO is sparged. 
</P><P>  Reactions:

</P>  Reaction is incubated 20min at 4C and 20 min at room temperature before running
  Pre run: 100V 45min at 4C
  Run: 100V 1. 5 hrs  at 4C
  Staining: After run is completed, gel is staining with Et-Br by shaking 15 min to stain DNA.
  Results: 

<P>  In conclusion, there is a little retardation.   Binding conditions may not be sustained because gel is not anoxic.  Not only reaction buffer but also running buffer and gel should have to be reduced for proper binding of CooA to the DNA. 
  EMSA [231010]
  Gel
  8.3ml 30% polyacrylamide solution (29:1)
  2.5ml 10X TBE 
  39.2ml ddH2O
  50ul TEMED
  500ul APS (10%)
  Argon is sparged for 5 min to get an anoxic gel 
  45min pre-run is completed at 100V before reaction in order to remove APS/TEMED
  1.7mM sodium dithionate is added in to the running buffer 
  DNA&amp;PROTEIN:
  pCooF:
  MW: 57 kDa
  200 ng in 8 ul
  200x10-9 / 57kDa= 3.5 pmol
  pCooM:  
  MW: ~66kDa
  264ng in 8ul
  264x10-9 / 66 kDa= 4 pmol
</P><P>  CooA:
  MW: 48 kDa
  1ug/ul in  500ul 
  12ul is added to reaction mix. 
  Reduced protein is 2ul DTT added into 1000 ul CooA. 
  Reduced +CO is degassed, reduced with DTT and 5 min CO is sparged. 
</P><P>  Binding Reaction 
  Binding solution (2X)
  80ul 1M TRIS-HCl  (ph:7.6)
  1120ul 25mM MgCl2
  40ul 5M KCl
  40ul 1M DTT
  100ul glycerol
  20 ul 10mg/ml BSA
  20ul 170 mM sodium dithionate 
  580ul pure water
  Reactions:

</P><P>  Reaction is incubated 20min at room temperature before running
  Run: 100V 1 h at 4C
  Staining: After run is compleated, gel is staining with Et-Br by shaking for 15 min to stain DNA.
  Results:

</P>  We cannot get a clear image of the gel because of uneven polymerization of the gel. We think the argon amount is too much so our gel cannot solidify properly. DNA marker which cannot be open accurately is the evidence for failed polymerization
<H2><SPAN class="mw-headline" id="4._ITF">4. ITF</SPAN></H2><P>  Intrinsic Tryptophan Fluorescence (ITF) 
  Samples
</P><PRE>  DNA:    28 Nm ; 56 Nm ; 74 nM :   93 nM pCooF  (eluted with elution buffer )
      Protein: 50 nM reduced –CO bounded; reduced; oxidized CooA 
</PRE><P>  Buffers 
  For oxidized CooA :  2X Buffer 
  -40 mM Tris-HCl ( pH: 7.6)
  - 14 mM MgCl2
  - 100 mM KCl 
  -10 % glycerol 
   For reduced –CO bounded &amp; reduced CooA : 2X buffer 
</P><P>   -40 mM Tris-HCl (ph:7.6)
   - 14 mM MgCl2
   - 100 mM KCl 
   -10 % glycerol 
   -20 mM DTT 
   -3.4 mM sodium dithionite
</P><P>  Parameters:
</P><PRE>   Excitation: 280 nm 
       Emission: 300- 400 nm 
       Slit number: 4 nm ( 2 turn) 
       Temperature: 25 C 
       Blank: buffer (1X) 
</PRE><P>  Measurements              
</P><PRE>   1 -Blank: 1X buffer ( for oxidized  , reduced-CO  / reduced  CooA ) 
         2- 50 nM oxidized CooA  in oxidized buffer 
         3- 50 nM oxidized CooA+ 28 nM pCOOF + 20. Min 
         4- Negative control :50 nM oxidized CooA + 10 ul elution buffer + 20. Min
         5- 50 nM oxidized CooA+ 56 nM pCOOF + 40. Min
         6- Negative control : 50 nM oxidized CooA + 20 ul  elution buffer + 40. Min
         7- 50 nM reduced –CO bounded CooA in reduced buffer 
         8- 50 nM reduced –CO bounded CooA + 28 nM pCOOF + 20 min 
         9- Negative control : 50 nM reduced –CO bounded CooA + 10 ul elution buffer + 20. min
         10-50 nM reduced –CO bounded CooA + 56 nM pCOOF + 40. Min
         11-Negative control :50 nM reduced –CO bounded CooA + 20 ul elution buffer + 40. Min
         12- 50 nM reduced –CO bounded CooA + 74 nM pCOOF + 60. Min
         13-Negative control: 50 nM reduced –CO bounded CooA + 30 ul elution buffer + 60. Min
         14-50 nM reduced –CO bounded CooA + 93  nM pCOOF + 80. Min
         15- Negative control: 50 nM reduced –CO bounded CooA + 40 ul elution buffer + 80.min
         16-50 nM reduced  CooA  in reduced buffer 
         17- 50 nM reduced CooA + 28 nM pCOOF + 20. Min
         18-Negative control: 50 nM reduced CooA + 10 ul elution buffer + 20. Min
         19- 50 nM reduced CooA + 56 nM pCOOF + 40.min
         20-Negative control: 50 nM reduced CooA + 20 ul elution buffer + 40.min
</PRE><P>  Result :

</P><P> pink: reduced –CO bounded CooA  + buffer 
</P><P> yellow : reduced –CO bounded CooA + 28 nM pCOOF ( 20 .min )
</P><P> green: NC : reduced- CO bounded CooA +10 ul elution buffer  (20.min ) 
</P><P> blue: reduced – CO bounded CooA +  56 nM pCooF  (40. min )
</P><P> purple: NC : reduced –CO bounded CooA + 20 ul elution  (40. Min )  yellow : red/Co 28 nM pcoof 20 .min
</P><P>  Blue line : reduced –CO bounded CooA + 56 nM pCOOf 
  White line :oxidized CooA + 56 nM pCOOF 
</P><H3><SPAN class="mw-headline" id="Conclusion:">Conclusion: </SPAN></H3><P>  The structure of the CooA protein has two residues of Tryptophan. Carbon monoxide binding to protein and DNA binding to protein can cause conformational change in the protein structure so tryptophan orientation change in the protein . Intrinsic tryptophan fluorescence experiment shows that any fluorescence intensity change and fluorescence shift resulting from the change of tryptophan.  We see a little difference between binding of promoter of oxidized and reduced CO bounded CooA protein. Decrease in the intensity is obtained.  The structure of CooA is given above :

</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:METU_Turkey/Delta">http://2010.igem.org/Team:METU_Turkey/Delta</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:METU_Turkey/Delta" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:METU_Turkey/Delta" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:METU_Turkey/Delta&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:METU_Turkey/Delta&amp;oldid=207912" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>