"
Page
Discussion
View source
History
teams
Log in
 
Team:CSU Fort Collins/Beer
From 2013.igem.org
Home
Home
Our Projects
Desalination
Self Healing and Inflating Tires
Gluten-free Beer
Parts Submitted to the Registry
Modeling
Team Notebook
Safety
Our Team and Sponsors
Meet the Members
Meet the Advisors
Official Team Page
Sponsors
Acknowledgements
Our Gluten-Free Mission
If you have Celiac Disease or are Gluten-Intolerant (and are over 21), stop what you are doing right now. Your prayers for a great tasting gluten-free beer have finally been answered, or at least are in the process of being answered. Think of it as a delay in the mailroom. All of those nights when your friends were taunting you with the ice cold, delicious, golden liquid that gives courage to even the most faint of heart are quickly coming to an end. Your desire to enjoy a "cold one" without having a gut-wrenching reaction should be fullfilled by the end of the summer. The CSU iGEM team is working on producing a "gluten-free" beer that can be brewed from wheat or barley so you can savor that same great taste you know and love. So cross your fingers, bust out your lucky rabbits foot, and watch out for black cats because we are gearing up for something big, and it has never been done before.
Project Summary
As Fort Collins is a major brewing hub, it was natural for our team to gravitate toward a beer-related project. Knowing full well the problems caused by Celiac disease, and the affinity many others have for reducing gluten in their diets, we decided to design and create a yeast strain capable of both fermenting quality beer, and breaking down gluten. To accomplish this we had to address several issues:
What is the immunological basis for gluten intolerance i.e. gluten’s antigenic qualities?
Celiac disease is an autoimmune disorder that affects the digestive system. People with Celiac disease suffer from a severe reaction when exposed to gluten, a protein found in wheat, rye, and barley. Glutamine and proline rich regions of gluten family proteins, which include gliadin in wheat and hordein in barley, are incredibly stable and cannot be broken down by the stomach. The antigenicity of gluten seems to be due to these proline and glutamine rich peptides, the most prevalent one being the 33-mer: LQLQPFPQPQLPYPQPQLPYPQPQLPYPQPQPF, found in gliadin. The sequence seems to be fairly conserved across barley and wheat. Three distinct patient-specific T cell epitopes identified previously in T cell proliferation assays are present in this peptide, namely, PFPQPQLPY, PQPQLPYPQ (three copies), and PYPQPQLPY (two copies). Upon reaching the intestines, these peptides are processed by a Celiac’s immune system, producing a response that damages villi in the small intestine and interferes with absorption of nutrients from food. [1] [2]
Where we found a viable enzyme
Our search for an enzyme capable of breaking down gluten and neutralizing its toxicity led us first to the enzyme mutated by the 2011 UW iGEM team. The modified Kumamolisin-As has a maximal activity at a pH of 4 and would work well in the pH range of 5.2-5.5 found in beer. For expression in yeast we had to account for codon bias, and optimized the sequence so it could more easily be moved from a prokaryotic system to a eukaryotic one. Another promising enzyme was AN-PEP. This protease already cleaves after proline residues and is stable at low pH. Used in the cocktail known as Brewers Clarex, AN-PEP has been shown to reduce gluten levels in beer when it is added to a final product. Unfortunately, the enzyme is protected by patent and was unavailable to us.
How the enzyme is secreted
Yeast have two mating types, called “a” and “
alpha
”. Yeast of the alpha mating type secrete mating factor-alpha (MF-
\alpha
), which is tagged with a secretion factor at the N-terminus of the protein. Experimental evidence shows that this signal sequence is cleaved in the golgi before MF-
\alpha
is exported from the cell. It is commonly used by researchers to mark foreign proteins for secretion in laboratory yeast. The tag’s sequence was placed directly upstream of the sequence for Kumamolisin. The DNA itself was synthesized by IDT, but we were also able to extract it from the yeast genome by PCR. [1] [2]
How we moved our system into yeast
Yeast can take up plasmids from the environment, but without selective pressure will drop them quickly. This would not be ideal for our purposes, as it would be unrealistic and potentially harmful to add antibiotics to every batch of beer just to ensure plasmid retention. As a proof of concept we introduced our system into yeast using plasmids, but we were also in the process of developing an integrative plasmid that would insert our sequence directly into the yeast genome.
Two plasmids were used:
pCM189: centromeric yeast plasmid, marker URA3, tetracycline repressed expression of target gene under control of tetO2, low copy number
pCM190: episomal yeast plasmid, marker URA3, tetracycline repressed expression of target gene under control of tetO7, high copy number
The integrative plasmid was constructed by adding the resistance gene for geneticin to pCM189/MF-alpha/Kuma-max.
Jamboree
At this years iGEM Jamboree, we will be presenting our method utilized for optimizing MF-alpha/Kuma-max and how both Kuma-max and the optimized protease, UltraK, were inserted into integrative yeast vectors. These vectors were then inserted using homologous recombination into WLP007, a common commercial beer brewing yeast, and used to brew beer for 5 days. Then, data was quantified using a RIDASCREEN 2nd generation Gliadin ELISA.
Retrieved from " http://2013.igem.org/Team:CSU_Fort_Collins/Beer "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
