<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2015.igem.org/Team:Macquarie_Australia">Home</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/ProjectOverview">Project</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Experiments">Experiments and Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Medals">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Modeling">ALA to PPIX Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Modeling/PhotoSysII">Photosystem II Modelling</A></LI><LI><A href="" https:="">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/Mythbusters">Chlorophyll Mythbusters</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/Implementation">Implementation Strategy</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/ModelingIntern">Internship</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/SYTYCS">SYTYCS</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/MqOpenDay">Open Day</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Team/Members">Students</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Team/Instructors">Instructors</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Sitemap">Sitemap</A></LI></DIV><TITLE>Experiments and Protocol</TITLE><DIV class="contentContainer"><FIGCAPTION><CENTER><I><B>Fig 1.</B> An overview of the methodology. A modified BioBrick Standard Assembly was used. Alkaline phosphatase on the target vector backbone for the genes; ChlM and those from Photosystem II. The Gibson Assembly method was used in order to insert composite part ChlH into the BioBrick vector plasmid.</I></CENTER></FIGCAPTION><H1 style="text-align: center;">Experiments and Protocol</H1><H4 style="text-align: center;">Making Competent Cells</H4><OL><LI>Using a sterile plastic loop, pick 10-12 large (2-3mm in diameter) colonies from the plate. Inoculate to 150mL of SOB medium in a 1L flask, and grow overnight at 18-22<SUP>o</SUP>C, 200-250rpm.</LI><LI>A600 should be 0.2-0.8 to harvest. Preferably, cells should be in mid log phase with A600 ~ 0.5.</LI><LI>Remove the flask from the incubator and place on ice for 10 minutes.</LI><LI>Transfer the culture to a 15mL centrifuge tube and spin at 2500 x g for 10 min at 4<SUP>o</SUP>C</LI><LI>Pour off and discard the supernatant, and immediately place the tube on ice.</LI><LI>Resuspend your cells in 1mL of ice-cold TB buffer, make sure there are no clumps of cells left, but also treat your cells gently and keep them cold.</LI><LI>Add ice-cold TB buffer to bring volume up to 1/5th of the original culture volume (~30mL in this case). Mix the tube by gently inverting 3 times.</LI><LI>Incubate the tube on ice for 10 minutes.</LI><LI>Centrifuge at 2,500 x g for 7 minutes at 4<SUP>o</SUP>C, discard the supernatant.</LI><LI>Gently resuspend the cells in ~1/20th of the original culture volume of ice-cold TB buffer. NOTE: 1/20th is based on and OD600 of 0.5, so adjust volume accordingly. e.g. if the culture OD600 was 0.1 then resuspend in 1/100th of original volume.</LI>Pre-chill 1.5ml Eppendorf tubes on ice. Add 930µl of your cell suspension, keeping the remainder on ice in the 15mL tube.<LI>Add 70µl of DMSO to the 930µl of cell suspension. Mix gently by swirling, and place on ice.</LI><LI>Aliquot 100µl of the competent cell/DMSO mixture into fresh microcentrifuge tubes. Label the tubes with: Date – Strain. Snap freeze with liquid nitrogen.</LI><LI>Repeat step 11-13 for the rest of your cell suspension in step 10. Store cells at -80<SUP>o</SUP>C</LI></OL><H6>TB Buffer</H6><B>Ingredients:</B> 3g PIPES, 10.9g MnCl2-4H2O, 2.0 g CaCl2-2H2O, 18.6 g KCl.<B>Methods:</B> All components (except for MnCl2-4H2O) were mixed and dissolved in 500 mL of water and pH adjusted to 6.7 with KOH. Then, MnCl2-4H2O, was dissolved in 300 mL of water, mixed and solution adjusted to 1 L. Sterilisation via filtration followed through a pre-rinsed 0.45 µm filter unit and stored at 4°C.
<H6>SOB Media (for competent cells)</H6><B>Ingredients:</B> 10g bacto-tryptone, 2.5g bacto-yeast, 1000µl NaCl, 417µL KCl, 1.205g Mg2SO4 &amp; 500mL Milli-Q water. <B>Methods:</B> The adjusted quantities were combined in a 1 L measuring column with constant stirring and then placed in the autoclave for sterilisation (121<SUP>o</SUP>C, 15 min, standard liquid cycle).<H4 style="text-align: center;"> Agarose Gel Electrophoresis</H4><OL><LI>Mix 1ul of 1kbp DNA ladder with 6ul of loading dye (bromophenol blue) and 4ul of 1x TAE buffer and load into first well </LI><LI>Mix 5ul of plasmid DNA with 1ul of loading dye and load into wells.</LI><LI>Run gel at 90V for 45minutes approximately</LI><LI>Gel image is taken using Biorad GelDoc</LI></OL><H6>EDTA Buffer</H6><B>Ingredients:</B> 37.22g EDTA solid, 180 mL of water and pH adjusted to 8.0 using 10M NaOH.<B>Methods:</B> Components were combined then pH adjusted.
<H6>50 X TAE Buffer</H6><B>Ingredients:</B> 121.2g Tris base (dissolved in water) with 28.55mL of glacial acetic acid &amp; 50mL 0.5M EDTA (pH 8.0).<B>Methods:</B> Components were combined then make up to 500ml with distilled water.
<H6>Making Agarose Gel</H6><B>Ingredients:</B> 1g agarose powder, 100ml of 1 x TAE<B>Methods:</B> Dissolve agarose in TAE by microwaving for 1 minute, wait till cool to touch, then add 2ul of GelRed and pour into the casette.
<H4 style="text-align: center;">Composite Part Ligation</H4><P>The assembly of composite parts from two existing BioBricks i.e. BioBrick plasmid A (parent vector) and BioBrick plasmid B (gene to be inserted) was performed through a restriction digest/ligation protocol. </P><OL><LI>200ng of each BioBrick plasmid was digested as follows: Plasmid BioBrick A with NEB restriction enzymes SpeI and PstI; plasmid BioBrick B with XbaI and PstI, according to supplier protocol. (1hr @37<SUP>o</SUP>C, then 20mins @80<SUP>o</SUP>C).</LI><LI>1µL of Fast alkaline phosphatase (Thermo Scientific) was added to reaction tube of plasmid BioBrick A to dephosphorylate the gene fragments and prevent re-ligation of the parent vector. Reaction tubes were incubated at 37<SUP>o</SUP>C for 60 mins, followed by a deactivation step at 80<SUP>o</SUP>C for 20 mins.</LI><LI>Ligation was then performed with an insert to vector molar ratio of 3:1. 1µL of T4 ligase (NEB) was added to the mix for ligation, according to supplier protocol. Ligation reactions were performed at 37<SUP>o</SUP>C for 60 mins and kept on ice for transformation into chemical competent cells.</LI></OL><H4 style="text-align: center;">Plasmid Prep</H4><OL><LI>Centrifuge @13,200 rpm for 10 min of 1.8mL of overnight cultures in 2mL Eppendorf’s </LI><LI>Discard supernatant and add 1.9 mL of culture and centrifuge again @13,200 rpm for 10 min </LI><LI>Re-suspend pelleted cells in P1 Buffer (250µl)  </LI><LI>Add 250µL of P2 buffer &amp; invert 4-6 times (turns homogenous blue) </LI><LI>Add 350µL of N3 &amp; mix by inverting (turns colourless) </LI><LI>Centrifuge @ 10min 13,000 rpm to obtain a pellet </LI><LI>Transfer supernatant in QIA Prep Spin Column by pipetting </LI><LI>Centrifuge 30-60 sec - discard flow through </LI><LI>Wash QIA Prep Spin Column with 0.5mL of PB </LI><LI>Centrifuge for 30-60 seconds - discard flow through </LI><LI>Wash spin column by adding 0.75 mL PE buffer </LI><LI>Centrifuge for 30-60 seconds - discard flow through and centrifuge for a further 1min </LI><LI>Place QIAPrep Column in clean Eppendorf </LI><LI>Elute DNA by adding 30µl of water, stand for 1min, centrifuge for 1min </LI><LI>Plasmid concentration is measured using NanoDrop 2000 spectrophotometer </LI></OL><H4 style="text-align: center;">General PCR Recipe</H4><UL><LI>4 µL of 5x phusion buffer. </LI><LI>0.4 µL dNTPs. </LI><LI>0.6 µL of DMSO. </LI><LI>0.2 µL of polymerase. </LI><LI>11.8 µL of water. </LI></UL><B>To this mixture, add:</B><UL><LI>1 µL of forward primer. </LI><LI>1 µL of reverse primer. </LI><LI>1 µL of template. </LI><LI>Total volume = 20 µL </LI></UL><H4 style="text-align: center;">PCR settings: A general program for PCR is as follows:</H4><UL><LI>Initial denaturation at 98<SUP>o</SUP>C for 30seconds.</LI></UL><B>Followed by 30 repeats of:</B><UL><LI>Denaturation – 98<SUP>o</SUP>C for 10seconds. </LI><LI>Annealing – 60<SUP>o</SUP>C for 10seconds. </LI><LI>Extension – 72<SUP>o</SUP>C for 2minutes. </LI><LI>Final extension – 72<SUP>o</SUP>C for 10minutes. </LI></UL><H4 style="text-align: center;">Transformation By Heat Shock</H4><OL><LI>Obtain competent cells from -80<SUP>o</SUP>C. </LI><LI>Defrost gently on ice. 100µl is sufficient for 2 transformations. </LI><LI>Add 1-10µl of plasmid DNA/ ligation mix to each tube. Incubate on ice for 30 min. </LI><LI>Heat shock in 42<SUP>o</SUP>C water bath for 30 seconds, then back on ice for 2 min. </LI><LI>Add 200µl of SOC media to each tube, and incubate in the 37<SUP>o</SUP>C shaker for 1h </LI><LI>For each tube of cells, spread 50µl onto one LB plate with appropriate antibiotic, and 100µl onto a second plate, using aseptic technique. Place your plate upside-down in the 37<SUP>o</SUP>C incubator. </LI></OL><H6>LB Agar Plate Preparation</H6><B>Ingredients:</B> 15g Bacto agar, 1000mL LB media <B>Methods:</B> Dissolve agar in LB media, autoclave for 15 mins at 121<SUP>o</SUP>C; wait until cool to touch, then add appropriate antibiotics before pouring into petri dishes. 
<H6>LB Media</H6><B>Ingredients:</B> Tryptone 10g, Yeast extract 5g, NaCl 10g, Milli-Q water to 1000mL.<B>Methods:</B> Dissolved 10g tryptone, 5g yeast extract and 10g NaCl in 800mL Milli-Q water, making use of a magnetic stirrer. Once dissolved, bring volume up to 1 L using Milli-Q water. Autoclave 1L of the solution (121<SUP>o</SUP>C, 15 min, standard liquid cycle). 
<H6>SOC Media </H6><B>Ingredients:</B> 10g bacto-tryptone, 2.5g bacto-yeast, 1000µl 5M NaCl, 417µL 3M KCl, 1.205g 20mM Mg2SO4, 805g 20mM D-glucose &amp; 500mL Milli-Q water. <B>Methods:</B> The adjusted quantities were combined in a 1 L measuring column with constant stirring and then placed in the autoclave for sterilisation (121<SUP>o</SUP>C, 15 min, standard liquid cycle).<H4 style="text-align: center;">Protein Expression</H4><OL><LI>Re-transform restriction-digest screened plasmids according to the heat shock transformation protocol. </LI><LI>Screen plates for transformant colonies and place in 5mL of ZYM-5052 broth with appropriate antibiotic concentration at 37<SUP>o</SUP>C with shaking until OD600¬ is 2.0 – 3.0. </LI><LI>Take whole volume and place in 45mL of ZYM-5052 broth<SUP>(1)</SUP> in a sterile conical flask containing appropriate antibiotic concentration. Incubate at 37<SUP>o</SUP>C with shaking. </LI><LI>Cell pellets were collected through centrifugation at 12000 rpm for 5 mins.  </LI><LI>Cell pellets were resuspended in re-suspension buffer (Containing: 50mM Tricine NaOH pH 8.0, 2mM MgCl2, 1mM DTT) and whole lysate was collected through a French Press for functional assays of operon protein products in lysate. </LI></OL><H6>Auto induction Medium (ZYM-5052) </H6><UL><LI>ZY media 9.57ml  </LI><LI>1m MgSO4 20ul </LI><LI>50x5052 200ul </LI><LI>50 x M 200ul </LI></UL><U>1. 50 x M</U><UL><LI>DI water 80ml </LI><LI>Na2HPO4 17.75g </LI><LI>KH2PO4 17g </LI><LI>NH4Cl 13.4 </LI><LI>Na2SO4 3.55g </LI></UL><U>2. 50 x 5052</U><UL><LI>Glycerol 25g (weigh in Beaker) </LI><LI>DI water 73ml </LI><LI>Glucose 2.5g </LI><LI>alpha lactose 10g </LI></UL><U>3. 1M MgSO4</U><UL><LI>MgSO4.7H2O 24.65g </LI><LI>DI water 87ml  </LI></UL><U>4. ZY Media (400mL)</U><UL><LI>Trypton 4g </LI><LI>Yeast extract 2g </LI></UL><H4 style="text-align: center;">MgPE Enzymatic Functional Assay</H4><OL><LI>Collect whole lysate, from induced operon 3 containing ChlM expressed proteins, using ChlM pET<SUP>(2)</SUP> as a negative control.  </LI><LI>Prepare a master mix of recombinant ChlM, 25µl of S-Adenosyl-L- methionine and 65µl of 2µM of Mg-protoporphyrin IX. </LI><LI>Incubate at 30℃ with gentle agitation for 8-10min </LI><LI>100µL of acetone was added to precipitate protein. </LI><LI>The sample was spun down, to separate the protein and supernatant.
</LI><LI>Supernatant was then subjected to UHPLC to and observed peaks.  </LI></OL><H4 style="text-align: center;">UHPLC </H4><P>The UHPLC was performed according to Sawicki and Willows<SUP>(3)</SUP>. Briefly, a Shimadzu HPLC system was used at 2ml/min with an Alltech C8 column. A 5 min linear gradient was used for separation of Mg-protoporphyrin IX and Magnesium-protoporphyrin IX mono-methyl ester. Compounds were detected at an excitation wavelength of excitation of 416nm and emission of 595nm.  </P><H4 style="text-align: center;">SDS-PAGE Electrophoresis</H4><OL><LI>Re-suspend pelleted bacterial cells in 200µL of Milli-Q-water </LI><LI>Transfer 50 µL of suspension into new Eppendorf tubes and combine with 50ul of 2xTruSep sample buffer. </LI><LI> Shear the cells using a Hamilton syringe. </LI><LI>Centrifuge the preparation for 3minutes @ 13,000 rpm. </LI><LI>Load 20 µL of the supernatant into gel. </LI><LI>Conduct electrophoresis at a constant voltage (200V) for 1 hour. </LI><LI>Coommassie Stain for ~30 minutes. </LI></OL><H4 style="text-align: center;"> Gun4 PPIX binding by fluorescence spectrophotometry </H4><H6>Preparation of PPIX</H6><OL><LI>Approximately 1mg of dried PPIX was dissolved in 5ul of 1M NaOH, and diluted with 200ul of deionized water.</LI><LI>Pellet any undissolved particulate by centrifugation, and dilute the supernatant 100-fold with 5% HCl.</LI><LI>The concentration of the PPIX solution was calculated by measuring the absorbance of the diluted solution at 404nm, using the Beer-Lambert equation with molar extinction coefficient of 278000.</LI><LI>A 2uM concentration solution of PPIX was used for the binding assay.</LI></OL><H6>Gun4 PPIX binding </H6><OL><LI>Over-expressed Gun4 containing cell lysates was prepared as described above.</LI><LI>The protein concentration was estimated using Bradford reagent, and the lysate diluted to approximately 0.2mg/ml.</LI><LI>100ul of the 2uM PPIX solution was added with varying amounts of protein lysate to a total volume of 200ul, and transferred to a clean quartz cuvette for fluorescence measurements in a Cory Eclipse fluorescence spectrophotometer (Agilent)</LI><LI> The excitation wavelength was set at 280nm and the emission at measure from 580nm to 800nm. The emission maxima for protein bound PPIX is at 635nm.</LI></OL><H5>References</H5><OL><LI>Studier, F. W. (2014). Stable expression clones and auto-induction for protein production in E. coli. In <I>Structural Genomics</I> (pp. 17-32). Humana Press.</LI><LI>Meinecke, L., Alawady, A., Schroda, M., Willows, R., Kobayashi, M. C., Niyogi, K. K., ... &amp; Beck, C. F. (2010). Chlorophyll-deficient mutants of Chlamydomonas reinhardtii that accumulate magnesium protoporphyrin IX. <I>Plant molecular biology, 72</I>(6), 643-658.</LI><LI>Sawicki, A., &amp; Willows, R. (2007). S-Adenosyl-L-methionine: magnesium-protoporphyrin IX O-methyltransferase from Rhodobacter capsulatus: mechanistic insights and stimulation with phospholipids. <I>Biochem. J, 406</I>, 469-478. </LI></OL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>