<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2015.igem.org/Team:Macquarie_Australia">Home</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/ProjectOverview">Project</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Experiments">Experiments and Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Medals">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Modeling">ALA to PPIX Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Modeling/PhotoSysII">Photosystem II Modelling</A></LI><LI><A href="" https:="">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/Mythbusters">Chlorophyll Mythbusters</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/Implementation">Implementation Strategy</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/ModelingIntern">Internship</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/SYTYCS">SYTYCS</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/MqOpenDay">Open Day</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Team/Members">Students</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Team/Instructors">Instructors</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Sitemap">Sitemap</A></LI></DIV><TITLE>Results</TITLE><DIV class="contentContainer"><H1 style="text-align: center;">Project Results</H1><P>On this page we describe the results of our project and our future plans.</P><H4 style="text-align: center;">Summary of achievements</H4><UL><LI>Successful completion of the chlorophyll-a synthesis pathway - all 13 genes required for chlorophyll-a synthesis in E. coli successfully placed into BioBricks</LI><LI>Functional characterisation of the ChlM (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1640018">BBa_K1640018</A>) enzyme. Functionality was demonstrated in converting Magnesium-Protoporphyrin IX (Mg-PPIX) to Magnesium-Protoporphyrin IX-Monomethyl Ester (Mg-PPIX-ME). The presence of ChlM was further confirmed by running of extracted protein on an SDS-PAGE gel</LI><LI>Improved the characterization of Gun4 (<A href="http://parts.igem.org/Part:BBa_K1080003">BBa_K1080003</A>) by showing PPIX binding with an alternate, simpler method and documented it as a contribution under the <A href="http://parts.igem.org/Part:BBa_K1080003:Experience">experience page</A></LI><LI>Successfully constructed BioBricks containing 14 of the 17 genes required for biosynthesis of Photosystem II</LI><LI>Modelling of pathway from 5-aminolevulinic acid to PPIX. This determined what concentration of 5-aminolevulinic acid (5-ALA) resulted in optimum yield of Protoporphyrin IX (PPIX)</LI></UL><DIV class="centreStuffInline"><FIGURE class="specialInline"><FIGCAPTION><I>Fig 1. A schematic representation of the four designed operons containing the 13 genes necessary for the synthesis of chlorophyll-a from protoporphyrin IX in E. coli</I>.</FIGCAPTION></FIGURE></DIV><P align="justify">We have successfully constructed BioBricks containing all 13 of the required genes for the synthesis of chlorophyll-a from protoporphyrin IX.
	This was achieved through the design and construction of four operons containing these 13 genes in BioBricks.
	The four operons were constructed by assembling our 13 BioBricks into composite parts using a <A href="https://2015.igem.org/Team:Macquarie_Australia/Experiments">restriction digestion and ligation protocol</A>.
	All of our assembled parts were sequenced for further confirmation of our success.</P><FIGCAPTION><I>Fig 2. Single (EcoRI) and double (EcoRI + PstI) restriction digests of four operon constructs containing 13 genes required for the chlorophyll-a synthesis pathway on a 1% agarose gel. A 1kb DNA ladder (NEB) is shown in the far left lane running alongside digest products.</I></FIGCAPTION><H4 style="text-align: center;">ChlH</H4><P align="justify">We successfully constructed the ChlH(BBa_K1080001)<B>(4166kb)</B> BioBrick. ChlH is part of operon 1 constituting the chlorophyll-a biosynthesis pathway in <I>E. coli</I>.</P><P align="justify">The ChlH construct’s purpose is to generate Mg-protoporphyrin IX from Protoporphyrin IX, as the very first step towards synthesising chlorophyll-a from Protoporphyrin IX<SUP>(1)</SUP>. The role of ChlH is to chelate Mg<SUP>2+</SUP> to Protoporphyrin IX, thereby synthesising MG-protoporphyrin IX <SUP>(1)</SUP>.</P><P align="justify">ChlH was synthesized in three fragments (figure 3), and assembled in two steps First G13, 3-6 and the KAN vector were combined via restriction digest and then P2 was added via Gibson assembly.</P><FIGCAPTION><I>Fig 3. Restriction digest of Gibson assembly product visualised on a 1% agarose gel stained with GelRed. Lane 1 is NEB 1 kb DNA ladder. Lane 2 is a single digest by EcoRI and Lane 3 is a double digest by EcoRI and PstI. A band of approximately 6 kb in lane 2 corresponds to the expected size of assembled ChlH in the KAN vector. Bands at approximately 2 kb and 4 kb correspond to the expected size of the KAN vector and ChlH gene respectively.</I></FIGCAPTION><H4 style="text-align: center;">Protein expression</H4><P align="justify">We have successful expressed the ChlM and GUN4 genes in <I>E. coli</I>. This was confirmed by running protein extracts on an SDS-PAGE gel, which showed distinctive gel bands corresponding to the theoretical sizes of these two proteins (figure 4).</P><FIGCAPTION><I>Fig 4. SDS-PAGE protein expression analysis of transformed <I>E. coli</I> cells each expressing ChlM and GUN4. Protein extracts were loaded onto 4-12% Bis-Tris gel. A unique band is present at the molecular weight of approximately 30kDa (theoretical mass of ChlM) signifying expression of S-Adenosyl-L-methionine:magnesium protoporphyrin IX methyltransferase. This was further confirmed by enzymatic functional assay. Overexpression of GUN4 was also observed (as highlighted), and functionality proven by a simple fluorescence spectrophotometric measurement. </I></FIGCAPTION><H4 style="text-align: center;">Functional assay of submitted parts</H4><P align="justify">We have demonstrated the functionality of ChlM enzyme encoding magnesium protoporphyrin IX methyltransferase (lac+ChlM: BBa_K1640018), an enzyme within the third operon in the chlorophyll-a biosynthesis pathway. ChlM catalyses the methylation of a carboxyl group in magnesium-protoporphyrin IX using cofactor S-Adenosyl-L-methionine (SAM), yielding magnesium-protophorphyrin IX mono-methyl ester <SUP>(3)</SUP>:


</P><P align="center">S-adenosyl-L-methionine + magnesium protoporphyrin IX ⇌ S-adenosyl-L-homocysteine + magnesium protoporphyrin IX 13-methyl ester.</P><FIGCAPTION><I>Fig 5. Catalysis of Mg-protoporphyrin IX to Mg-protoporphyrin IX monomethylester by ChlM(4).</I></FIGCAPTION><P align="justify">To demonstrate functionality of ChlM, we added a lac promoter creating a lac + ChlM composite part (figure 4). This was transformed into E. coli strains DH5a, K12 and XL-blue. Protein was induced using auto-induction media ZYM5052<SUP>(5)</SUP>.</P><H4 style="text-align: center;">Functional assay of ChlM</H4><P>Cell lysate containing over expressed ChlM was added with substrates and products were separated by UHPLC (figure 3).</P><FIGCAPTION><I>Fig 6. UHPLC chromatogram of the products of ChlM activity alongside positive and negative controls. The substrate MG-PPIX elutes at 3.04 mins, while the product Mg-PPIX mono-ester elutes at 4.22 mins.</I></FIGCAPTION><P align="justify">The UHPLC chromatogram data enabled determination of ChlM enzyme activity (figure 3). This data enabled further determination of levels of Mg-Protoporphyrin IX converted to Mg-Protoporphyrin IX-ME by ChlM. In this instance, 26% of the precursor molecule Mg-Protoporphyrin IX had been catalysed by the enzyme ChlM into Mg-Protoporphyrin IX-ME.</P><P align="justify">When compared to ChlM pET (positive control) expression, which successfully converted 12% of the precursor molecule, it is apparent that our expression of ChlM was very successful.</P><H4 style="text-align: center;">Functional Assay of GUN4</H4><P align="justify"> The gene GUN4 encodes a cofactor that binds to protoporphyrin IX facilitating the Mg-chelatase mediated insertion of an Mg2+ ion to form Mg-protoporphyrin IX. Protoporphyrin IX has fluorescent properties with an excitation spectrum that peaks at 404nm and an emission peak at 635nm.</P><P align="justify">The assay undertaken involved adding protoporphyrin IX to cell lysates overexpressing GUN4 and exposing these samples to light at a wavelength of 280nm. This excites tryptophan residues in GUN4 and, when bound, the the resulting 340nm fluorescence will excite protoporphyrin IX, resulting in an emission peak at 635nm that can be detected by a spectrophotometer.</P><FIGCAPTION><I>Fig 7. The emission spectra of GUN4 over-expressing lysates, heat treated lysates and BSA when excited at 280nm. Measured emission wavelengths ranged from 580nm to 800nm. Maximum emission of GUN4 bound to protoporphyrin IX is 635nm.</I></FIGCAPTION><P align="justify">For this assay, cells were lysed by French press and these lysates were diluted ten-fold to a concentration of approximately 0.2mg/ml. Protoporphyrin was prepared to a concentration of 2µM and added to samples of lysate containing 0µg, 5µg, 10µg, 15µg and 20µg of GUN4. A non-specific protein control (Bovine Serum Albumin) and heat treated GUN4 cell lysates (80 degrees Celsius for ten minutes) were used as controls.</P><P align="justify">When excited at 280nm, emission peaks at 635nm were detected through fluorescence spectrophotometry for each GUN4 lysate, with increased intensity as GUN4 concentration rose. BSA showed no fluorescent activity when excited at this wavelength and the heat treated lysates displayed a much lower level of fluorescence compared to the unheated samples (figure 11).</P><FIGCAPTION><I>Fig 8. Plots of fluorescence intensity at emission 635nm against protein concentration. Gun4 showed a clear concentration dependent relationship with PPIX binding, which was reduced after a short heat treatment. BSA acts as a negative control</I></FIGCAPTION><P align="justify">This demonstrates that GUN4 is biologically active, binding to protoporphyrin IX. Further, the linear increase in emission at 635nm with greater concentration of GUN4 is indicative of higher rates of binding. This corresponds with the modelling conducted by the Macquarie University team of 2014 that suggests a linear pattern of protoporphyrin IX/GUN4 complex formation. Our functional assay builds upon this framework as we found that this linear relationship only occurs at low protein concentrations, when the lysates were diluted to a concentration of 0.2mg/ml.</P><H4 style="text-align: center;">Photosystem II</H4><P align="justify">We designed 5 operons consisting a total of 17 genes from <I>C. reinhardtii</I> which encoded for essential proteins for Photosystem II (see table 1). These 17 genes were divided arbitrarily into 10 BioBricks.</P><FIGCAPTION><I>Fig 9. A schematic representation of 5 designed operons containing the 17 necessary genes for biosynthesis of Photosystem II in E. coli.</I></FIGCAPTION><P align="justify">Our team successfully constructed 8 of the 10 BioBricks designed for Photosystem II. These were confirmed by sequencing.</P><P align="justify">Table 2. The 10 BioBricks we attempted to create, whether we were able to detect a constituent gene fragment in a gel-electrophoresis and whether confirmed an accurate sequence from an extracted plasmid.</P><TABLE class="regularTable" style="width:40%" cellpadding="0" align="center"><TBODY><TR class="regularTable" style="height:24px"><TH class="regularCol20" style="text-align:center">Biobrick</TH><TH class="regularCol10" style="text-align:center">Operon</TH><TH class="regularCol10" style="text-align:center">Constructed</TH></TR><TR><TD>PsbE-L-J</TD><TD>2</TD><TD>Y</TD></TR><TR><TD>PsbQ-R</TD><TD>5</TD><TD>Y</TD></TR><TR><TD>PsbM-Z-H</TD><TD>4</TD><TD>Y</TD></TR><TR><TD>PsbA</TD><TD>1</TD><TD>Y</TD></TR><TR><TD>PsbW-K</TD><TD>4</TD><TD>Y</TD></TR><TR><TD>PsbP</TD><TD>5</TD><TD>Y</TD></TR><TR><TD>PsbO</TD><TD>5</TD><TD>Y</TD></TR><TR><TD>PsbC</TD><TD>1</TD><TD>Y</TD></TR><TR><TD>PsbT-B</TD><TD>3</TD><TD>N</TD></TR><TR><TD>PsbD</TD><TD>1</TD><TD>N</TD></TR></TBODY></TABLE><FIGCAPTION><I>Fig 10. Single (EcoRI) and double (EcoRI + PstI) restriction digests of eight BioBricks containing 14 genes required for the construction of PSII on a 1% agarose gel. A 1kb DNA ladder (NEB) is shown in the far left lane running alongside digest products.</I></FIGCAPTION><P align="justify"> If we achieve this we can then transform <I>E. coli</I> which we have engineered to develop chlorophyll and hydrogenase, this will allow the <I>E. coli</I> to produce a functioning PSII and allow the production of H<SUB>2</SUB>.</P><H4 style="text-align: center;">Modelling</H4><P align="justify"> ALA is the first precursor on the pathway to biosynthesis of both chlorophyll-a and heme<SUP>(6)</SUP>. As an organism which produces its own heme, E. coli itself produces 5-ALA and converts it to PPIX (the point of divergence between the heme and chlorophyll-a synthesis pathways) through a multi-step pathway for which it naturally possesses the necessary genes<SUP>(6)</SUP>.</P><P align="justify"> We successfully modelled the production of PPIX from 5-ALA in non-genetically transformed E. coli, comparing how different concentrations of the precursor 5-ALA affected the production of PPIX (figure 9).</P><FIGCAPTION><I>Fig 11. Concentration of PPIX produced from various initial concentrations of 5-ALA for E. coli cell culture concentrations of 0.2mM, with MgCl added at a concentration of 2mM.</I></FIGCAPTION><FIGCAPTION><I>Fig 12. A schematic representation of Protoporphyrinogen IX causing substrate inhibition of 5-ALA’S conversion to porphobilinogen via porphobilinogen synthase.</I></FIGCAPTION><H4 style="text-align: center;">Future Direction of our Project</H4><UL><LI>To demonstrate functionality of our constructed parts for chlorophyll-a synthesis, through successfully inducing protein expression of, and functionally characterising, all 13 genes in the pathway.</LI><LI>To finish the construction of Photosystem II. This would be through the construction of BioBricks for the remaining components (PsbD and PsbT-B), building of composite parts by joining the constructed BioBricks, and the development of a BioBrick containing all 5 operons.</LI><LI>To successfully biosynthesise a hydrogenase enzyme, allowing for the combination of electrons and protons for the synthesis of hydrogen gas. This would help us build a functional prototype.</LI></UL><H5>References</H5><OL><LI>Peter, E., Wallner, T., Wilde, A. &amp; Grimm, B. (2011). Comparative functional analysis of two hypothetical chloroplast open reading frames (ycf) involved in chlorophyll biosynthesis from Synechocystis sp. PCC6803 and plants. Journal of Plant Physiology, 168(12), 1380-1386. doi:10.1016/j.jplph.2011.01.014</LI><LI>Gibson, D.G., Young, L., Chuang, R.Y., Venter, J.C., Hutchison, C.A. &amp; Smith, H.O. (2009). Enzymatic assembly of DNA molecules up to several hundred kilobases. Nature Methods, 6(5), 343–345. doi:10.1038/nmeth.1318.</LI><LI>Sawicki, A., &amp; Willows, R. D. (2007). S-Adenosyl-L-methionine:magnesium-protoporphyrin IX O-methyltransferase from Rhodobacter capsulatus: mechanistic insights and stimulation with phospholipids. The Biochemical Journal, 406(3), 469–478. doi:10.1042/BJ20070284</LI><LI>Shepherd, M., Reid, J. D., &amp; Hunter, C. N. (2003). Purification and kinetic characterization of the magnesium protoporphyrin IX methyltransferase from Synechocystis PCC6803. Biochemical Journal, 371(2), 351–360. doi:10.1042/BJ20021394</LI><LI>Studier, F.W. (2014). Stable expression clones and auto-induction for protein production in E. coli. Methods in Molecular Biology, 1091, 17-32. doi: 10.1007/978-1-62703-691-7_2.</LI><LI>Ilag, L.L., Kumar, A.M. &amp; Soll, D. (1994). Light regulation of chlorophyll biosynthesis at the level of 5-aminolevulinate formation in Arabidopsis. The Plant Cell, 6(2), 265-275. doi10.2307/3869644</LI><LI>Patrick-Stanford, N.P.J, Capretta, A. &amp; Battersby, A.R. (1995). Expression, purification, and characterisation of the product from the Bacillus subtilis hemD gene, uroporphyrinogen III synthase. European Journal of Biochemistry, 231(1), 236-241. doi:10.1111/j.1432-1033.1995.0236f.x</LI></OL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>