<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_Nicky skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/Nicky</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;
The 4166 bp ChlH gene was engineered synthetically by Integrated DNA Technologies (IDT) into 3 gene blocks (Table 1). The gene sequence was originally from Chlamydomonas Reinhardtii and codon optimized for expression in Escherichia coli.
</P><H6><I>Table 1: Gene blocks</I></H6><TABLE class="regularTable" style="width:30%" cellpadding="0"><TBODY><TR class="regularTable" style="height:24px"><TH class="regularCol10" style="text-align:center">Master Digest Mix</TH></TR><TR><TD>1(G13)</TD><TD>1678 bp</TD></TR><TR><TD>2 (P2)</TD><TD>980 bp</TD></TR><TR><TD>3 (3-6)</TD><TD>1508 bp</TD></TR></TBODY></TABLE><H4>Thursday 6th August 2015 </H4><UL><LI> Attempted assembly of ChlH fragments into CAM and KAN backbones via Gibson Assembly </LI><LI> Amplified GA mix to check whether gblocks had successfully been assembled. 3 sets of primers were used (table 2). </LI><LI> Ran amplicons on a 1% agarose gel with GelRed. If assembly was successful, a band around 4.2 kbp is expected. </LI><UL><LI>No DNA evident on gel</LI></UL></UL><H6><I>Table 2: Primers used for PCR</I></H6><TABLE border="1" style="width:70%"><TBODY><TR><TD>1.</TD><TD>BB-VF2 (forward)
<SMALL>5’ - TGC CAC CTG ACG TCT AAG AA - 3’</SMALL></TD><TD>BB-VR (reverse)
<SMALL>5’ - ATT ACC GCC TTT GAG TGA GC - 3’</SMALL></TD></TR><TR><TD>2.</TD><TD>BB-for (forward)
<SMALL>5’ - CGA ATT CGC GGC CGC TTC TAG - 3’</SMALL></TD><TD>BB-rev (reverse)
<SMALL>5’ - GCT GCA GCG GCC GCT ACT AGT - 3’</SMALL></TD></TR></TBODY></TABLE><UL><LI> Transformed 2 uL KAN and CAM assembly products into <I> E. coli </I> cells and plated 100 uL and 150 uL of transformants out on nutrient agar plates with appropriate KAN or CAM antibiotic</LI><UL><LI>Colonies present on all plates (Table 3) but transformation efficiency was very low</LI></UL></UL><H6><I>Table 3: Colony counts of Gibson Assembly transformants</I></H6><TABLE border="1" style="width:50%"><TBODY><TR><TD colspan="2">100 uL plates</TD><TD colspan="2">150 uL plates</TD><TD> Total </TD></TR><TR><TD>pink</TD><TD>white</TD><TD>pink</TD><TD>white</TD></TR><TR><TD>CAM</TD><TD>6</TD><TD>8</TD><TD>3</TD><TD>3</TD><TD>20</TD></TR><TR><TD>KAN</TD><TD>1</TD><TD>1</TD><TD>0</TD><TD>1</TD><TD>3</TD></TR><TR><TD>Control</TD><TD>0</TD><TD>0</TD><TD>5</TD><TD>7</TD><TD>12</TD></TR></TBODY></TABLE><H4>Wednesday 12th August 2015</H4><UL><LI> Harvested white colonies and cultured them in LB broth with the appropriate antibiotics
</LI><LI>1 white KAN colony</LI><LI>5 white CAM colonies</LI></UL><H4>Thursday 13th August 2015</H4><UL><LI>Isolated plasmids from liquid cultures via GenElute Plasmid Miniprep Kit (Sigma-Aldrich) for PCR to check for successful cloning of ChlH. Quality and concentration of isolated DNA was assessed via NanoDrop and 1% agarose gel stained with GelRed (Table 4).</LI></UL><H6><I>Table 4: Concentration and purity of isolate plasmid DNA obtained from NanoDrop.</I></H6><TABLE border="1" style="width:35%"><TBODY><TR><TD>Concentration (ng/uL)</TD><TD>260/280</TD></TR><TR><TD>CAM 1</TD><TD>155.3</TD><TD>1.95</TD></TR><TR><TD>CAM 2</TD><TD>302.1</TD><TD>1.91</TD></TR><TR><TD>CAM 3</TD><TD>253.5</TD><TD>1.94</TD></TR><TR><TD>CAM 4</TD><TD>277.8</TD><TD>1.95</TD></TR><TR><TD>CAM 5</TD><TD>119.5</TD><TD>1.85</TD></TR><TR><TD>KAN 1</TD><TD>87.2</TD><TD>1.91</TD></TR></TBODY></TABLE><UL><LI> 260/280 values indicate isolated DNA was of fairly high quality.
</LI><LI>Performed first round of double restriction digests with EcoRI and EcoRI + PstI, and ran product on a gel to determine if ChlH had been successfully assembled by last weeks Gibson Assembly.</LI><LI>Ran restriction products on a 1% gel stained with GelRed. Gel indicated that assembly was unsuccessful.</LI><LI>Assembly may have failed due to poor overlap between part G13 and the KAN vector (11 bp) and the KAN biobrick vector. Potentially there is also insufficient overlap between the KAN vector and part 3-6 (19 bp). Optimal overlap for Gibson Assembly is 15 - 30 bp.</LI></UL><H4>Wednesday 19th August 2015</H4><UL><LI> Attempted new assembly of ChlH via restriction digest and ligation for parts G13, 3-6 and KAN vector </LI><UL><LI> 3-6-------------------------KAN vector-----------------------G13 (5.4 kb length)</LI><LI> Expect an approximately 5kb band on gel if assembly is successful </LI></UL><LI>Method:
	</LI><LI> 2x Master Digest Mix </LI><TABLE border="1" style="width:40%"><TBODY><TR><TD><U>Master Digest Mix</U></TD></TR><TR><TD>4 uL 10x Buffer for EcoRI and PstI
14 uL water
 1 uL EcoRI
 1 uL PstI</TD></TR></TBODY></TABLE><LI>Digest protocol</LI><UL><LI> 2.5 uL linearised vector </LI><LI> 1 uL each gblock (G13 and 3-6) </LI><LI> 4.5 uL digest master mix </LI><LI>Incubate at 37 C for 1 hour and heat inactivate at 80 C for 15 min</LI></UL><LI>Ligation Protocol
	</LI><LI> 9 uL 2x T4 Ligase buffer to digest mix </LI><LI> 1 uL T4 DNA ligase
		</LI><LI> incubate at 25 C for 20 min, 37 C for 20 min and heat inactivate at 80 C for 10 min </LI></UL><UL><LI>Ran digest products on a 1% agarose gel stained with GelRed</LI><LI>Approx 5kb band evident - seems as though gblocks G13 and 3-6 were ligated to the linearized KAN vector (figure 1).



</LI><LI> Attempted assembly of P2 with other gblocks and KAN vector via Gibson Assembly </LI><LI> Performed PCR on GA product, and ran amplicon on a gel to check for successful ChlH assembly </LI><UL><LI>Only primer dimers were visualised on the gel. Will check for assembly via restriction digest next week. </LI></UL></UL><H4> Thursday 20th August 2015 </H4><UL><LI> Transformed assembly products into competent E.coli cells and spread these cells onto plates with KAN antibiotic. </LI></UL><H4> Wednesday 26 August 2015 </H4><UL><LI> Prepared liquid cultures of the ChlH transformant colonies grown last thursday. </LI></UL><H4>Thursday 27th August 2015</H4><UL><LI>Performed plasmid preps on 20 of the 40 liquid cultures. DNA concentration and quality was assesed via NanoDrop (Table 5). </LI></UL><H6><I>Table 5: Concentration and purity of isolated plasmid DNA obtained from NanoDrop.</I></H6>
TABLE 6- DONE
<TABLE border="1" style="width:30%"><TBODY><TR><TD>Concentration (ng/uL)</TD><TD>260/280</TD><TD>260/230</TD></TR><TR><TD>1</TD><TD>67.3</TD><TD>1.84</TD><TD>1.68</TD></TR><TR><TD>2</TD><TD>66.7</TD><TD>1.85</TD><TD>1.73</TD></TR><TR><TD>3</TD><TD>76.2</TD><TD>1.82</TD><TD>1.60</TD></TR><TR><TD>4</TD><TD>90.0</TD><TD>1.86</TD><TD>2.02</TD></TR><TR><TD>5</TD><TD>75.1</TD><TD>1.82</TD><TD>1.72</TD></TR><TR><TD>6</TD><TD>54.3</TD><TD>1.89</TD><TD>2.01</TD></TR><TR><TD>7</TD><TD>89.9</TD><TD>1.89</TD><TD>1.99</TD></TR><TR><TD>8</TD><TD>196.5</TD><TD>1.59</TD><TD>0.83</TD></TR><TR><TD>9</TD><TD>57.9</TD><TD>1.82</TD><TD>1.33</TD></TR><TR><TD>10</TD><TD>130.6</TD><TD>1.69</TD><TD>0.97</TD></TR><TR><TD>11</TD><TD>72.2</TD><TD>1.95</TD><TD>2.11</TD></TR><TR><TD>12</TD><TD>71.2</TD><TD>1.92</TD><TD>1.71</TD></TR><TR><TD>13</TD><TD>58.8</TD><TD>1.97</TD><TD>2.03</TD></TR><TR><TD>14</TD><TD>70.5</TD><TD>1.98</TD><TD>2.11</TD></TR><TR><TD>15</TD><TD>75.0</TD><TD>1.98</TD><TD>2.09</TD></TR><TR><TD>16</TD><TD>57.7</TD><TD>2.01</TD><TD>2.18</TD></TR><TR><TD>17</TD><TD>94.3</TD><TD>1.95</TD><TD>2.17</TD></TR><TR><TD>18</TD><TD>82.2</TD><TD>1.92</TD><TD>2.28</TD></TR><TR><TD>19</TD><TD>72.9</TD><TD>1.93</TD><TD>2.08</TD></TR><TR><TD>20</TD><TD>89.1</TD><TD>1.94</TD><TD>2.12</TD></TR></TBODY></TABLE><UL><LI>Performed double restriction digests with EcoRI and EcoRI + PstI on the isolated plasmid DNA</LI><LI>Ran digest products on a gel to determine if ChlH had been successfully assembled by last week’s Gibson Assembly.</LI><UL><LI>We could not interpret the results of the gel, so will redo the digest next week </LI></UL></UL><H4>Wednesday 2nd September 2015</H4><UL><LI>Performed plasmid preps on the remaining 20 liquid cultures</LI><LI>Checked DNA concentration via NanoDrop (Table 6).
</LI></UL>
TABLE 7-done
<H6><I>Table 6: Concentration and purity of isolate plasmid DNA obtained from NanoDrop</I></H6><TABLE border="1" style="width:30%"><TBODY><TR><TD>Concentration (ng/uL)</TD><TD>260/280</TD><TD>260/230</TD></TR><TR><TD>1</TD><TD>10.3</TD><TD>2.53</TD><TD>1.72</TD></TR><TR><TD>2</TD><TD>96.2</TD><TD>1.81</TD><TD>1.53</TD></TR><TR><TD>3</TD><TD>95.3</TD><TD>1.84</TD><TD>1.80</TD></TR><TR><TD>4</TD><TD>107.0</TD><TD>1.88</TD><TD>1.44</TD></TR><TR><TD>5</TD><TD>81.2</TD><TD>1.87</TD><TD>1.85</TD></TR><TR><TD>6</TD><TD>90.5</TD><TD>1.78</TD><TD>1.69</TD></TR><TR><TD>7</TD><TD>79.1</TD><TD>1.77</TD><TD>1.42</TD></TR><TR><TD>8</TD><TD>74.5</TD><TD>1.85</TD><TD>1.79</TD></TR><TR><TD>9</TD><TD>72.5</TD><TD>1.82</TD><TD>1.85</TD></TR><TR><TD>10</TD><TD>67.0</TD><TD>1.81</TD><TD>1.69</TD></TR><TR><TD>11</TD><TD>62.2</TD><TD>1.84</TD><TD>1.57</TD></TR><TR><TD>12</TD><TD>119.1</TD><TD>1.85</TD><TD>1.81</TD></TR><TR><TD>13</TD><TD>59.2</TD><TD>1.89</TD><TD>1.88</TD></TR><TR><TD>14</TD><TD>76.4</TD><TD>1.80</TD><TD>1.38</TD></TR><TR><TD>15</TD><TD>68.8</TD><TD>1.85</TD><TD>1.68</TD></TR><TR><TD>17</TD><TD>116.7</TD><TD>1.83</TD><TD>1.61</TD></TR><TR><TD>18</TD><TD>43.2</TD><TD>1.84</TD><TD>1.76</TD></TR><TR><TD>19</TD><TD>63.5</TD><TD>1.85</TD><TD>1.66</TD></TR><TR><TD>20</TD><TD>134.8</TD><TD>1.94</TD><TD>2.08</TD></TR></TBODY></TABLE><H4>Thursday 3rd September 2015</H4><UL><LI>Performed double restriction digests on 38 isolated plasmids - 18 from yesterday, 28 from last week with EcoRI and EcoRI + PstI. 2 samples from yesterday (1 and 16) were disregarded due to low DNA concentration.</LI><LI>Ran digest products on a 1% agarose gel stained with GelRed to visualise whether ChlH was present. </LI><UL><LI><B>For sample 17, we obtained a single digest band at around 6 kb which corresponds to the linearized vector, and 2 double digest bands at 4 kb and 2 kb, corresponding to the expected size of ChlH and the vector respectively (Figure 4). Only sample 17 worked. 
</B></LI></UL></UL><UL><LI> Plated out the remaining liquid culture for sample 17 </LI><LI> Transformed cells with sample 17 isolated plasmid </LI><LI> Set up a sequencing reaction for sample 17 with primers BB-VF2 and BB-VR (Table 2) and sent it off for sequencing </LI></UL><H4>Monday 7th September 2015</H4><UL><LI> Performed plasmid preps on 5 colonies grown from the remaining liquid culture for sample 17</LI><LI> Checked quality and concentration of isolated DNA via NanoDrop (Table 7) </LI><H6><I>Table 7: Concentration and purity of isolate plasmid DNA obtained from NanoDrop</I></H6></UL>
TABLE 8 - 
<TABLE border="1" style="width:30%"><TBODY><TR><TD>Concentration (ng/uL)</TD><TD>260/280</TD><TD>260/230</TD></TR><TR><TD>1</TD><TD>366.2</TD><TD>1.90</TD><TD>2.27</TD></TR><TR><TD>2</TD><TD>339.6</TD><TD>1.91</TD><TD>2.28</TD></TR><TR><TD>3</TD><TD>332.0</TD><TD>1.91</TD><TD>2.28</TD></TR><TR><TD>4</TD><TD>258.8</TD><TD>1.89</TD><TD>2.21</TD></TR><TR><TD>5</TD><TD>261.8</TD><TD>1.89</TD><TD>2.27</TD></TR><LI> Carried out double restriction digest of sample 17 plasmid DNA with EcoRI and EcoRI + PstI </LI><LI> Ran digest products on a gel to confirm sample 17 was the correct sample with assembled ChlH </LI><H4>Thursday 10th September 2015 </H4><UL><LI>Results of sequencing from 3rd of September returned. 99% sequence was observed for the first and last _____ bp to the beginning of G13 and end of 3-6 respectively. A single point mutation was observed at _____. something to something.</LI><LI>Designed additional primers to generate sequence to cover the entire assembled ChlH gene (table 9)</LI></UL><H6>Table 8: Primers designed for checking sequence of assembled ChlH</H6><TABLE border="1" style="width:70%"><TBODY><TR><TD>1.</TD><TD>End of G13 to half of P2 (forward)
<SMALL>5’- GAT ACC GTC GTC TCA CTG ACC -3</SMALL></TD></TR><TR><TD>2.</TD><TD>Half of 3-6 to half of P2 (reverse)
<SMALL>5'- ACT CGC CGT AGG TCT TGA TG -3'</SMALL></TD></TR><TR><TD>3.</TD><TD>End of P2 to half of G13 (reverse)
<SMALL>5'- AGC AGA CGC ATC GGA TCG -3'</SMALL></TD></TR><TR><TD>4.</TD><TD>End of P2 to mid 3-6 (forward)
<SMALL> 5'- CGA CGC CAA AGA TCT GAC C -3'</SMALL></TD></TR><TR><TD>5.</TD><TD>537 bp from start of G13(forward)
<SMALL>5'- GTT TTC TAT GGC ACA GCT TGG TC -3’</SMALL></TD></TR></TBODY></TABLE><H4>Thursday 10 September</H4><UL><LI>Attempted Backbone switch for ChlH from KAN to CAM via standard 3A assembly</LI><LI>transformed into BL21 origami cells</LI><LI>Backbone switch showed no growth on LB-CAM.</LI></UL><H4>Monday 14 September</H4><UL><LI>Backbone switch for ChlH from KAN to CAM</LI><LI>500 ng of ChlH was digested with 25ng of psb1C3 (CAM) with EcoRI and PstI. Ligase was then added and the ligation products were transformed into competent cells. The cells were then plated out onto LB-CAM media.</LI><LI>Conducted PCR with primers for ChlH (table 8) to check that the part had been assembled in the correct order.</LI></UL><H4>Wednesday 16 September</H4><UL><LI>A gel of the primer tests for ChlH was run and it was clear that too much DNA had been loaded. The PCR was repeated with a diluted ChlH template and these products were run on another gel. However, no products were seen on the gel.</LI></UL>

Table with primer pair and expected sizes
</TBODY></TABLE></DIV></DIV></DIV></DIV></BODY></HTML>