Team:Macquarie Australia/Notebook2PSB
Home
Project
Description
Experiments and Protocols
Results
Achievements
Notebook
Safety
Modelling
ALA to PPIX Modelling
Photosystem II Modelling
Parts
Team Parts
Human Practices
Chlorophyll Mythbusters
Implementation Strategy
Internship
SYTYCS
Open Day
Collaborations
Team
Students
Instructors
Attributions
Sitemap
Notebook 2 PSB
Notebook - Photosystem II
This notebook includes the lab-work done to insert the genes to allow E. coli to build a Photosystem II protein complex.
Thursday 30 July
Restriction sites: Checked all part designs for appropriate prefix/suffix.
Operon 1:
Checked operon 1 for features that were present in the design to see how it was broken up (it was broken into three parts so it could actually be synthesised).
Operon 1 part 2:
Blastx of protein sequence confirmed correct protein for the gene, however gene sequence did not match that of the operon design. Divergence assumed to be the result of Rob changing the sequence (conserving AA sequence) to assist sequence synthesis. No RBS identified, however alternate RBS have have been used. Yet to be identified.
Thursday 6 August
Digested 7 gblocks with EcoRI + PstI:
psbD, psbC, psbA (3 parts of first operon)
psbELJ (operon 2)
psbTB (operon 3)
psbP, psbQR (2 parts of operon 5)
as well as CAM backbone
Ligated parts into plasmids
Transformed competent E. coli cells via heat shocking, and plate out onto Chloramphenicol plates - Incubated
Checked for update on part 2 of Operon 5, and Operon 4. Check with Rob.W. re: Part 1     of operon 1
Parts update:
Confirmed alternate RBS used in Operon 1 part 2. No further action required.
DNA Oligonucleotide Resuspension and Storage
Upon receiving newly synthesized oligonucleotides, researchers must decide how to resuspend and store the product. Here are some guidelines and recommendations.
Resuspension
Keep in mind most commercially synthesized oligonucleotides are shipped as lyophilized product. Dried DNA is usually very easy to resuspend in an aqueous solution.  However, not all oligonucleotides dry identically and some require more time to go into solution than others.  It is also possible for the dried oligonucleotide to become dislodged from the tube during shipping. Thus, it very important to spin down every oligonucleotide prior to opening the tube for resuspension.
Aqueous buffer
Resuspend oligos in TE buffer (10mM Tris; 0.1 mM EDTA; pH 8.0) as this buffer will maintain a constant pH.  Alternatively, use nuclease-free water. DEPC water will harm oligonucleotides and water from deionizing systems can be overly acidic, with a pH as low as 5.0.
Concentration
Oligonucleotides can be stored at a large range of concentrations.  However, concentrations <1 µM may change over time as some of the oligo can adhere to the plastic of the tube.  A 5-10 mM solution is generally the highest concentration at which an oligo will go into solution. Resuspension calculations can be made using yield information contained on IDT product specification sheets and on the oligo tube.
There you will find the actual yield of the oligonucleotide synthesis in three forms:
optical density units (OD)
mass (in mg)
and copy number (in nmole)
Resuspend dry oligonucleotides to a storage stock concentration of 100 µM and then dilute a portion of this to create working stock solutions.
To make a 100 µM concentration stock solution:  Take the number of nmoles in the tube and multiply that by 10. This will be the number of µL buffer to add to get a 100 µM solution.  For example, if you have 9 nmoles oligo, add 90 µL buffer to make a 100 µM solution. If you prefer to work in other units or to resuspend to a different concentration, a Dilution Calculator is available in the SciTools section of the IDT website.
Resuspension
For hard-to-suspend oligos, heat the oligonucleotide at 55°C for 1-5 minutes, then vortex thoroughly.  If there is still a visible precipitate in the tube,  the sample may contain silica which is a by-product of oligo synthesis. It will not affect the performance of the product, and may be removed through filtration or decanting the supernatant.
Long-term storage
If you would like to use a portion of the oligonucleotide immediately and store the remainder for future use, it is best to resuspend the entire product in Tris-EDTA (TE) buffer, pH 8.0 at the desired stock solution concentration. Take a sufficient volume for immediate use from the stock and dilute it to a working stock concentration. Divide the remaining stock solution into several small aliquots and store at –20°C.
Short-term storage
Oligonucleotides that have been resuspended in TE buffer, pH 8.0 can be stored at 4°C for up to 6 months.
Thursday 13 August
Last Week’s results
Successful colonies were observed on Chloramphenicol media for psbA, psbC, psbD, and psbQR.
No colonies were observed for psbTB.
Plasmid preparation re-done for psbTB. To improve transformant yield, the following changes were made to the procedure:
Incubation on ice increased from 10 minutes to 30 minutes
The ligation was performed at 37°C instead of 16°C
Fragment to vector ratio was increased, using 4:1 instead of 3:1
A plasmid miniprep was performed in triplicate for the successfully transformed samples.
DNA concentration from the miniprep was assessed using a nanodrop.
Samples were digested using EcoRI and PstI.
Gel electrophoresis performed with the 5 extracted samples, non digested samples, and the new and old prep of the psbTB: hard to read gel, as too much DNA was used. Will redo next week with less DNA, and single/double digests.
Week 3 image
Thursday 20 August
Digestion of newly received G-blocks (psbP, psbO, psbMZH, psbWK) with EcoRI and PstI
Ligation of digested G-blocks into Chloramphenicol backbone, and transformation into E. coli
Re-did gel electrophoresis of last week’s plasmid extracts, with a lower DNA concentration and single/double digests
Today’s results:
Gel electrophoresis shows expected banding for psbC 2 (expected 1456bp), psbQR colony 1 (expected 1092bp), psbELJ colony 2 (expected 756bp). Will send these for sequencing.
Thursday 27 August
Last week’s results:
Of the 4 transformed G-blocks, only PsbMZH grew colonies on Cam plates, with no colonies observed for psbP, psbO, and PsbWK
Sequencing data showed expected sequence for psbELJ
Incomplete sequence of psbQR - very short sequences returned (~600bp) for some reason. Have sent for further sequencing with more internal BBF and BBR primers.
Done today:
Screening of PsbA, PsbD, and PsbMZH colonies in triplicate, digested with EcoRI and EcoRI/PstI
Probable cause as to why no colonies were observed on PsbP, PsbO,and PsbWK was attributed to insufficient recovery time for competent cells after heat shock
Due to slow/few colony growth for Psb-MZH transformants, combined with the shortened recovery time from 2 hours to 40 minutes due to time constraints
Transformation of PsbP, PsbO and PsbWK was repeated
psbA(x2) and psbMZH(x3) sent for sequencing
Results:
psbA 1 & 2 (expected 1257bp), psbMZH 1, 2, 3 (expected 541bp, faint bands) appear as expected, will send these for sequencing.
Thursday 3 September
Last week’s results
Colonies observed for psbWK, psbP, and one colony of psbTB
Sequencing data for psbA and psbMZH shows expected sequence!
Done over the week
psbELJ colonies were cultured in induction media
Done today
Ligation/transformation of psbTB, PsbO and PsbC redone - into Cam backbone
psbTB redone in case sequencing of last week’s psbTB colony does not reveal successful incorporation of the biobrick
psbC redone as digestion of previous psbC colonies did not show successful incorporation of the biobrick
psb-O redone as no colonies have been isolated for this G-block
Miniprep of colonies containing psbTB (x1), psbP (x3), psbD (x5) and psbWK (x3)
Nanodrop of each eluted sample to assess DNA concentration
Gel electrophoresis in 1% agarose gel, using 10ng of DNA sample
Gel of both E and EP digestion
SDS-PAGE ran for psbELJ cultures
Identified that the incorrect concentrations of Cam backbone to inserts used - possible reason for no/few colonies using psb-TB and psbO
7.75ng/uL DNA solution, not 10.0ng/uL solution used in previous calculations
6 samples sent for sequencing - 3 x psbP, 3x psbWK
Results:
SDS-PAGE shows ladder, 5 lanes of induced sample with psbELJ plasmid, then 2 lanes of uninduced control. Target protein sizes are psbE=9.3kDa, psbL=4.4kDa, psbJ=4.2kDa. No protein expression is visible at these sizes, compared to uninduced control.
Thursday 10 September
Last week's results
Sequencing data showed expected sequence for both psbP and psbWK
Colonies observed for psbTB, psbO and psbC
Done Today
Ligation/transformation re-done for psbD into Cam backbone
Gel electrophoresis of psbO, psbTB and psbC in 1% agarose, with EcoRI and EcoRI/PstI digestions using 100ng of plasmid sample
PsbC and psbO sent for sequencing
Results:
Gel electrophoresis of psbC 1 & 2, psbTB 1, 2, 3, and psbO 1, 2, 3 with EcoRI and EcoRI/PstI digestion. PsbC 1 (expected 1456bp) and psbO 1, 2 & 3 (expected 894bp) appear as expected, and will be sent for sequencing.
Friday 11 September
Previous day's results
One colony of psbD observed
Done today
Transformation of psbD and psbTB ligation mixture, using 4µL ligation mix
Transformation of psbWK and psbP, using 2µL ligation mix
Sunday 13 September
Friday's results
Colonies for psbD, psbTB and psbWK observed
No colonies observed for psbP
Done today
Cultured seven colonies of psbTB and psbD
Cultured four colonies of psbC
Monday 14 September
Done today
Transformation of psbP and psbWK
Gel electrophoresis of psbTB, psbD and psbC with EcoRI and EcoRI/PstI digestions
Results:
Gel electrophoresis of psbD 1-7, psbTB 1-7 and psbC 1-4, with EcoRI and EcoRI/PstI digestion. No colonies displayed expected bands.
Wednesday 16 September
Done Today
Transformation of psbD re-done
Cultured seven colonies of psbTB
Cultured four colonies of psbO and psbC
Cultured three colonies of psbD
Thursday 17 September
Yesterday's results
Colonies observed for psbD
Done today
Gel electrophoresis of psbTB 1-7, psbO 1-4, psbD 1-3 and psbC 1-4 in 1% agarose, with EcoRI and EcoRI/PstI digestions
Results:
Gel electrophoresis of psbTB 1-7, psbO 1-4 psbD 1-3 and psbC 1-4. PsbO 1-4 (expected 908bp) and psbC 1-2 (expected 1456bp) appeared as expected, and will be sent for sequencing.
Friday 18 September
Done Today
Gel electrophoresis of the eight successfully constructed biobricks
Results:
Gel electrophoresis of psbC, psbA, psbELJ, psbMZH, psbWK, psbO, psbP, psbQR.
