<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_Notebook1CBP skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/Notebook1CBP</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2015.igem.org/Team:Macquarie_Australia">Home</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/ProjectOverview">Project</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Experiments">Experiments and Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Medals">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Modeling">ALA to PPIX Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Modeling/PhotoSysII">Photosystem II Modelling</A></LI><LI><A href="" https:="">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/Mythbusters">Chlorophyll Mythbusters</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/Implementation">Implementation Strategy</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/ModelingIntern">Internship</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/SYTYCS">SYTYCS</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Practices/MqOpenDay">Open Day</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Team/Members">Students</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Team/Instructors">Instructors</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Macquarie_Australia/Sitemap">Sitemap</A></LI></DIV><TITLE>Notebook 1 CBP</TITLE><DIV class="contentContainer"><H2>Notebook - Chlorophyll Biosynthesis Pathway</H2><P>This notebook includes the lab-work done to insert the genes to allow <I>E. coli</I> to transform protoporphyrin-IX to chlorophyll-<I>a</I>.</P><H5>Tuesday 30 June </H5>
Project status
<UL><LI>ChlM</LI><UL><LI>Synthesised as 1 gBlock</LI><UL><LI>Contains:</LI><UL><LI>Biobrick prefix &amp; suffix</LI><LI>Gene and Ribosome Binding Site</LI></UL></UL><LI>Need to digest and ligate into Biobrick Vector</LI></UL><LI>ChlH</LI><UL><LI>Synthesised as 3 gBlocks</LI><LI>Need to use Gibson Assembly to join them into whole gene sequence</LI></UL></UL>
Production of lab consumables
<UL><LI>Autoclaving pipette tips and eppendorf tubes.</LI><LI>Making new LB-agar media with the antibiotics kanamycin (KAN), chloramphenicol (CAM) and ampicillin (AMP).</LI><LI>Making fresh SOC media to be used for transformations.</LI><LI>Making fresh SOB media for use with competent cells.</LI></UL><H5>Wednesday 1 July </H5>
Primer design
<UL><LI>We designed forward and reverse sequencing primers for the 13 genes involved in the chlorophyll biosynthesis pathway; ChlH, Chli1, Chli2, ChlD, GUN4, ChlM, CTH1, YCF54, Plasto, POR, DVR1, ChlP and ChlG.</LI></UL>
Production of lab consumables
<UL><LI>We made dozens of LB-agar plates containing either Kan, Cam, Amp or no antibiotic.</LI><LI>We plated out the glycerol stocks from 2014 on to the correct media as well as plating out all the plasmids provided by Macquarie University team 2014.</LI></UL>
Screening of parts
<UL><LI>Sequencing reactions set up for the 2014 single and composite gene parts:</LI><UL><LI>Chli1+ChlD+GUN4</LI><LI>POR+DVR1+ChlP+ChlG</LI><LI>ChlP</LI><LI>YCF54</LI><LI>Plastocyanin (Plasto)</LI><LI>POR</LI><LI>ChlD</LI><LI>CTH1+YCF54</LI><LI>DVR1</LI><LI>CTH1</LI><LI>ChlG</LI><LI>Chli2</LI><LI>GUN4</LI><LI>YCF54</LI></UL></UL><H5>Thursday 2 July </H5>
Screening of parts
<UL><LI>PCR for all the single and composite gene parts from 2014 using the protocol shown below.</LI><UL><LI>NEB Q5 polymerase protocol</LI><UL><LI>5x Q5 reaction buffer           5ul</LI><LI>10mM dNTPs                       0.5ul</LI><LI>10um forward primer           1.25ul</LI><LI>10um reverse primer           1.25ul</LI><LI>Template DNA                     1ul</LI><LI>Q5 high fidelity polymerase  0.25ul</LI><LI>Nuclease free water             to 25ml</LI></UL><LI>Shorter PCR cycle for shorter fragments (ChlD, ChlG, ChlP, DVR1, POR, CTH1+YCF54+Plasto, CTH1+YCF54, DVR1+ChlP)</LI><UL><LI>Initial Denaturation   15m       95<SUP>o</SUP>C</LI><LI>Denaturation            30s        95<SUP>o</SUP>C</LI><LI>Anneal                     30s        52<SUP>o</SUP>C</LI><LI>Extend                    2m 30s   72<SUP>o</SUP>C</LI><LI>Final extend             5m         72<SUP>o</SUP>C</LI></UL><LI>Longer PCR cycle for larger fragments (Chli1+ChlD+GUN4, Chli1+ChlD, DVR1+ChlP+ChlG)</LI><UL><LI>Initial denaturation   15m   95<SUP>o</SUP>C</LI><LI>Denature                 30s   95<SUP>o</SUP>C</LI><LI>Anneal                    30s   52<SUP>o</SUP>C</LI><LI>Extend                 4m 30s 72<SUP>o</SUP>C</LI><LI>Final extend             5m    72<SUP>o</SUP>C</LI></UL></UL></UL>
Gel electrophoresis of the PCR products from the slower cycle.
<UL><LI>As seen below, ChlD did not amplify, the ChlG band appeared to be too small and the DVR1+ChlP band was also anomalous. All other bands appeared to be correct. (Note: the ladder was too concentrated and should be diluted in future.)</LI></UL>
IMAGE 1
Image 1: <I>Lanes 4-10, 12 all show band of expected size.</I><H5>Monday 6 July </H5>
Screening of products
<UL><LI>We ran a gel of the PCR products of the longer cycle from Thursday July 2. DVR1+ChlP+ChlG composite part amplified and there seems to be a problem with amplification of ChlD. Given ChlD high GC content, a high-GC enhancing buffer may be required.</LI></UL>
IMAGE 2
Image 2: <I>Lane 4 shows amplified expression of part.</I><H5>Tuesday 7 July </H5>
Screening of parts
<UL><LI>We carried out plasmid preps and nanodrops of the glycerol stock cultures from 2014.</LI><LI>We attempted to amplify the ChlD gene constructs again, this time using a GC-rich buffer.</LI></UL>
IMAGE 3
Image 3: Target genes are amplified but there are numerous other bands due to a long extension cycle.<I> NOTE: 1 + D + 4 refers to ChlI1 + ChlD + Gun4</I><UL><LI>We conducted PCR to amplify the G1-G3 and P2 fragments of ChlH that were produced by Macquarie University Team 2014 but it did not work with Q5 polymerase and a 60<SUP>o</SUP>C annealing temperature.</LI></UL>
Production of lab consumables
<UL><LI>We made competent cells for future transformations.</LI></UL>
Primer design
<UL><LI>We placed an order for the primers that had been designed on July 1 with Integrated DNA Technologies.</LI></UL><H5>Thursday 9 July</H5>
Construction of parts
<UL><LI>We attempted to construct a ChlH biobrick through Gibson Assembly following standard protocols and using the parts listed below.</LI><UL><LI>G1-3 PCR-amplified fragment from July 8.</LI><LI>P2 PCR-amplified fragment from July 8.</LI><LI>G4-6 gBlock</LI><LI>psb1K3 vector</LI></UL><LI>The Gibson assembly product was then transformed into competent cells and plated onto LB-KAN.</LI></UL>
Screening of parts
<UL><LI>We conducted analysis of the sequencing results from the summer break, finding that ChlP was incorrectly added to a composite part instead of ChlG.</LI></UL><H5>Friday 10 July</H5><UL><LI>Grew liquid cultures of ChlM and ChlH</LI><LI>Primer design</LI><UL><LI>We set up 41 PCRs to test the primers that we had designed on 1 July and ran the products on a gel. As seen in the images below, all the primers worked except for those designed for YCF54 and plasto.</LI></UL></UL>
IMAGE 4
Image 4: <I>Lanes 2 (POR), 7 (Chli1), 13 (Chli1), 14 (ChlD), 15 (GUN4), 17 (CTH1) &amp; 20 (CTH1) contain bands of the correct size</I>
IMAGE 5
Image 5: <I>Lanes 4 (ChlG), 5 (CTH1), 9 (Chli1), 11 (Chli1), 12 (ChlD), 13 (GUN4) and 14 (CTH1) have bands of the expected size.</I>
IMAGE 6
Image 6: <I>All lanes have bands of the expected size.</I><H5>Monday 13 July</H5>
Screening of parts
<UL><LI>Plasmid preps of our ChlM and ChlH liquid cultures from 10th July and checked the concentrations using nanodrop.</LI><UL><LI>Enzymatic digest (single digest with EcoRI; double digest with XbaI and PstI) and gel electrophoresis</LI></UL></UL>
IMAGE 7
Image 7: <I>ChlM seems to have been successfully inserted into BioBrick backbones; ChlH doesn’t.</I><UL><UL><LI>A number of our genes do not have promoters and for this reason both the lac promoter and T7 promoter from the iGEM kits were transformed into competent cells and plated out onto LB-CAM. If successful, we will attempt to attach these promoters to our genes.</LI><LI>We conducted PCR on 1/10, 1/100 and 1/1000 dilutions of the ChlH Gibson assembly product with a number of different primers (BB-VF2/BB-VR, G1F/G6R and BBF/BBR) as a further test of the success or failure of the biobrick construction attempted on 9 July.</LI><LI>A second round of testing was conducted for the gene specific primers we had previously designed.</LI></UL></UL><H5>Tuesday 14 July</H5>
Screening of parts
<UL><LI>We ran a gel of the PCR products of the ChlH Gibson assembly mix dilutions from yesterday and a further PCR of the 1/1000 dilution with the primers G1F and G6R was set up to run overnight.</LI><LI>We conducted PCR of ChlM as a further measure to screen for successful ligation into a BioBrick vector. Alongside this we conducted colony PCR of the lac and T7 promoter transformants from yesterday and a further PCR of our ChlD BioBrick with a high GC enhancing buffer and Hot Start polymerase.</LI><LI>These PCR products were then run on a gel and, as seen below, all of the ChlM and ChlD samples amplified successfully and produced bands of the expected size. Further the transformation of the lac promoter appears to have been successful although the transformation of the T7 promoter was not.</LI><LI>Liquid cultures of lac promoter transformant cells were made.</LI></UL><H5>Wednesday 15 July</H5>
Screening of parts
<UL><LI>Plasmid prep of lac promoter liquid culture that had been incubating overnight, followed by enzymatic digest (double digest with EcoRI and SpeI; double digest with SpeI and PstI) followed by gel electrophoresis (PCR product of ChlH also ran).</LI><LI>As seen in the image below, it appears that the Gibson assembly attempted on 9 July was unsuccessful. However the band sizes produced by the restriction digest of the lac plasmids is suggestive of a successful transformation. (Note: lane 1 contains ChlH PCR product and lanes 3 and 4 contain digested lac plasmid.)</LI></UL>
IMAGE 8<H5>Thursday 16 July</H5>
Construction of parts
<UL><LI>We attempted to construct composite parts consisting of the lac promoter and the following gene parts; ChlM, ChlD, and CTH1+YCF54.</LI><UL><LI>~200ng of lac promoter plasmids were digested with SpeI and PstI using Fast AP to prevent re-annealing.</LI><LI>~200ng of each of the following genes were digested with XbaI and PstI; ChlM, ChlD, and CTH1 + YCF54.</LI><LI>2µl of the lac promoter digest product was ligated with 4µl of each gene part and these were then transformed into competent cells and plated onto LB-CAM.</LI></UL></UL>
Screening of parts
<UL><LI>We sent our ChlM biobrick off for sequencing.</LI></UL><H5>Friday 17 July</H5>
Construction of parts
<UL><LI>We attempted to construct a further 3 composite parts consisting of the lac promoter and the following genes; CTH1 + YCF54, GUN4, and Chli1, following protocol from 16 July.</LI></UL>
Production of lab consumables
<UL><LI>We made more Cam LB-agar plates.</LI></UL><H5>Sunday 19 July</H5>
We prepared liquid cultures of the transformants from 17 July.<H5>Monday 20 July</H5>
Screening of parts
<UL><LI>Plasmid preps followed by enzymatic digest (double digest with EcoRI and SpeI), products run on gel.</LI><UL><LI>ChlM</LI><LI>ChlD</LI><LI>ChlI1</LI><LI>GUN4</LI><LI>CTH1+YCF54</LI></UL><LI>Results suggested that the attempted construction of composite parts had failed.</LI><LI>We then made further liquid cultures of ChlM, Chli1 and GUN4 for further screening.</LI><UL><LI>Protein expression</LI><LI>We began the process of testing for protein expression for a  number of the composite parts that were provided by Macquarie University Team 2014. To this end, we re-transformed plasmids containing the composite parts listed below and plated them onto LB-AMP.</LI><UL><LI>Chli1+ChlD+GUN4</LI><LI>POR+DVR1+ChlP+ChlG</LI><LI>POR+DVR1+ChlP</LI></UL></UL></UL><H5>Tuesday 21 July</H5>
Screening of parts
<UL><LI>We conducted a further PCR of  our ChlH Gibson Assembly mix from July 9 using the following primer pairs;</LI><UL><LI>BBVF2 and BBVR</LI><LI>BBVF2 and G6R</LI><LI>G1F and G6R</LI><LI>G1F and BBVR</LI></UL><LI>We conducted plasmid preps of yesterday’s liquid cultures, and, after determining their concentrations via nanodrop, these plasmids were again subject to restriction digests, this time with XbaI and PstI. These digests were then run on a gel for screening using the PCR products of the genes in question (CTH1+YCF54, GUN4, Chli1, ChlM and ChlD) as a control. Again, the results indicated an unsuccessful composite part construction.</LI><LI>We then attempted to construct these lac promoter composite parts using a modified Biobrick assembly protocol seen below.</LI><UL><LI>~500ng of the upstream part plasmid, in this case the lac promoter, was digested withEcoRI and SpeI.</LI><LI>~100ng of the downstream part plasmid, in this case one of the following genes; CTH1+YCF54, GUN4, Chli1, ChlM or ChlD, was digested with XbaI and PstI.</LI><LI>~100ng of the destination plasmid, in this case the psb1A3 linear vector, was digested with EcoRI and PstI.</LI><LI>All three parts were then ligated and transformed into competent cell before being plated out onto Amp media.</LI></UL></UL><H5>Wednesday 22 July</H5>
Of the biobrick construction plates from last night, only the lac+CTH1+YCF54 and the lac+ChlM transformants produced colonies. We made liquid cultures of these colonies and resuspended the lac+Chli1, lac+ChlD and lac+GUN4 cells in LB-Amp media to test for growth.<UL><LI>Growth was observed for these cultures and they were subsequently plated onto Amp media.</LI><LI>We conducted colony PCR of lac+CTH1+YCF54 and lac+ChlM transformants with both the gene specific primers we had designed and the biobrick primers. These PCR products were then run on gel for screening purposes.</LI><LI>We re-attempted the ligation of the lac+Chli1, lac+ChlD and lac+GUN4 biobrick assemblies at 37<SUP>o</SUP>C (it had previously been done at room temperature), transformed the ligation products into competent cells and plated them onto AMP media.</LI><LI>We induced expression in our Chli1+ChlD+GUN4, POR+DVR1+ChlP+ChlG and POR+DVR1+ChlP transformed cells using IPTG and ran a protein gel to test for expression. Uninduced cultures were used as a point reference to see if our genes were being correctly expressed.</LI></UL><H5>Thursday 23 July</H5>
Screening of products
<UL><LI>We made liquid cultures of the lac+Chli1, lac+ChlD and lac+GUN4 colonies that had been plated out yesterday.</LI><LI>We conducted colony PCR of the lac + Chli1, lac + ChlD and lac + GUN4 cultures using both the gene specific primers we designed and the biobrick primers. We ran the resulting PCR products on a gel for screening purposes.</LI><LI>We conducted plasmid preps of the lac+CTH1+YCF54 and lac+ChlM liquid cultures followed by enzymatic digest (single digest with EcoRI; double digest with EcoRI and SpeI)</LI><LI>Gel electrophoresis for screening purposes - the results suggesting that we were unsuccessful in constructing the required composite parts.</LI><LI>We cut the relevant bands from yesterday’s protein gel and conducted a trypsin digest on these proteins.</LI></UL><H5>Friday 24 July </H5>
Screening of parts
<UL><LI>We conducted plasmid preps of selected lac+Chli1, lac+GUN4 and lac+ChlD liquid cultures.</LI></UL>
Construction of parts
<UL><LI>We attempted once again to build composite parts incorporating the lac promoter, this time with the genes listed below.</LI><UL><LI>ChlD</LI><LI>ChlM</LI><LI>CTH1+YCF54</LI><LI>DVR1</LI><LI>Chli1</LI><LI>Plasto</LI></UL><LI>This time we used the following protocol</LI><UL><LI>~2µg of the lac promoter plasmids were digested with SpeI and PstI, using Fast AP to prevent re-annealing of the two resulting DNA fragments.</LI><LI>~ 100ng of each plasmid (one for each of the genes listed above) was digested with XbaI and PstI.</LI><LI>PCR cleanup of the digested lac plasmids, following the QIAGEN protocol, was undertaken in order to remove smaller DNA fragments and provide a greater chance that our promoter would anneal and ligate to the gene in question, rather than the smaller fragment of that same plasmid.</LI><LI>We then ligated the promoter and our genes using a 3:1 insert to backbone ratio, before transforming the ligation products into competent cells and plating them onto Cam media.</LI></UL><LI>We extracted the peptides from yesterday’s trypsin digests of our protein gel.</LI></UL><H5>Thursday 30 July</H5>
Screening of parts (to check if the gene has been successfully inserted)
<UL><LI>Plasmid prep and nanodrop</LI><UL><LI>lac + ChlM CAM (x4)</LI><LI>lac + Chli1 CAM (x4)</LI><LI>lac + ChlP AMP (x3)</LI><LI>lac + YCF54 AMP (x3)</LI><LI>lac + CTH1 AMP (x3)</LI><LI>lac + Plasto CAM (x4)</LI><LI>lac + ChlD CAM (x4)</LI><LI>lac + DVR1 CAM (x4)</LI><LI>lac + POK AMP (x3)</LI><LI>lac + ChlD CAM</LI><LI>lac + ChlM CAM</LI><LI>lac + DVR1 CAM</LI><LI>lac + Chli1 CAM</LI><LI>lac + plasto CAM</LI><LI>lac + CTH1+ YCF54 CAM</LI></UL><LI>Restriction digest</LI><UL><LI>lac + CTH1 + YCF54 CAM</LI></UL></UL>
Construction of composite parts<UL><LI>Enzymatic digest of 2μg of lac plasmid with SpeI and PstI followed by PCR cleanup.</LI><LI>Enzymatic digest of 100 ng of gene part with XbaI and PstI using fast alkaline phosphatase to prevent DNA religation.</LI><LI>Ligation of 50 ng of digested lac plasmid with gene part (3:1 molar ratio of gene and lac plasmid respectively)</LI><UL><LI>lac + ChlD CAM</LI><LI>lac + ChlM CAM</LI><LI>lac + DVR1 CAM</LI><LI>lac + Chli1 CAM</LI><LI>lac + plasto CAM</LI></UL></UL><H5>Tuesday 4 August </H5><UL><LI>Transformation of competent E. coli cells with constructed parts from the 30 July</LI><LI>Ran gel of lac+CTH1+YCF54 CAM restriction digests from 30 July</LI></UL>
IMAGE 9
Image 9: <I>Sample 4 shows bands of expected size.</I><H5>Wednesday 5 August </H5>
Restriction digest of plasmid preps from 30 July
<H5>Thursday 6 August </H5>
Screening of parts
<UL><LI>Plasmid prep from competent E. coli cells from 4 August and parts constructed using BioBrick assembly</LI><UL><LI>lac + ChlD CAM</LI><LI>lac + ChlM CAM</LI><LI>lac + DVR1 CAM</LI><LI>lac + ChlI1 CAM</LI><LI>lac + ChlI2 CAM</LI><LI>lac + plasto CAM</LI><LI>lac + Gun4 (B1) CAM</LI><LI>lac + Gun2 (B2) CAM</LI><LI>lac+Gun4 (B1(sh)) CAM</LI><LI>lac+Gun4 (B2(sh)) CAM</LI><LI>lac + Gun4 CAM</LI><LI>lac + GUN4 AMP</LI></UL><LI>Restriction digest; single digest with EcoRI, double digest with EcoRI and PstI</LI><LI>Gel electrophoresis to screen for successful transformants.</LI></UL>

IMAGE 10 and 11
Image 10 and 11:<I> Gel was loaded with single digest/double digest alternating. Where only one band is shown for double digest, parts have religated.</I><UL><LI>Ran gel of restriction digests from 5 August; single digest with EcoRI, double digest with EcoRI and PstI</LI></UL>
IMAGE 11
IMAGE 12
Image 12: <I>Gel results made no sense → parts redigested for a second round of screening for. PCR reaction conducted with gene specific primers to amplify expression for redigested parts.</I><UL><LI>Gel electrophoresis of second restriction digest products and PCR products.</LI></UL>
IMAGE 13
Image 13: <I>PCR products; ChlM (lanes 2-5), ChlP (lanes 8-10) show good gene specific primers.</I>

IMAGE 14/15
Image 14/15: <I>Showed gene insert at correct size for lac+CTH1+YCF54 (lanes 7 + 11) and lac+DVR1 (lanes 9 + 10)</I><H5>Thursday 13th August </H5>
Construction of composite parts with three different methods
<UL><UL><LI>lac + ChlD</LI><LI>lac + GUN4</LI><LI>lac + ChlM</LI><LI>plasto + ChlM</LI><LI>lac + ChlG</LI></UL><LI>BioBrick 3A Assembly method</LI><UL><LI>~5 μg of plasmid + lac promoter was digested with SpeI and PstI and ~8 μg was digested with EcoRI and PstI. These digested plasmids were gel purified for use in the construction of composite parts. A KAN backbone was used.</LI></UL><LI>PCR method</LI><UL><LI>PCR amplification followed by gel electrophoresis. PCR was used to make the gene insert which would be inserted later into a clean backbone (made by next method).</LI></UL><LI>XP/SR digest method</LI><UL><LI>Double digest (XbaI + PstI) followed by gel electrophoresis. This method was used in order to get a clean lac-promoter backbone part.</LI></UL></UL>
IMAGE 16, 17, 18
Image 16,17: <I>From the PCR, the ChlG product was successful. Gel purified product D (Image 17) was a successful backbone part which was then used to make lac + ChlG.</I><UL><LI>Checked glycerol stocks from 2013 of parts - Only the first two were good - unusable stocks were disposed of</LI></UL>
IMAGE 19<H5>Thursday 20 August </H5>
Screening of parts
<UL><LI>BioBrick Assembly method transformations showed success on two plates</LI><UL><LI>Lac + ChlM </LI><LI>Plasto + Chlm </LI></UL><LI>Colonies from these plates were screened using plasmid preps. Plasmid preps from BioBrick 3A Assembly had very low nucleic acid concentrations (3 - 8 ng/μL). PCR was performed to amplify DNA followed by gel electrophoresis.</LI><LI>Gel electrophoresis showed no products.</LI><LI>E. coli cells were transformed with the lac+ChlM gene part and lac+plasto+ChlM gene part to retest.</LI><LI>lac + ChlG made 13 August was plated onto LB-CAM. No colonies grew.</LI></UL>
Construction of composite parts via ligation reactions (25ng of lac plasmid)
<UL><LI>lac + ChlD CAM (75ng)</LI><LI>lac + GUN4 CAM (50 ng)</LI><LI>lac + ChlM CAM (50 ng)</LI><LI>lac + ChlG CAM (50 ng)</LI></UL>
Parts were transformed into competent cells and plated onto LB-CAM media<H5>Sunday 23 August </H5>
Screening of parts
<UL><LI>E. coli cells transformed with lac+ChlM and lac+plasto+ChlM on 20 August were observed.</LI><LI>No growth for lac+plasto+ChlM.</LI><LI> One colony for lac + ChlM</LI></UL><H5>Thursday 27 August </H5>
Construction of composite parts
<UL><LI>Restriction digest of plasmids containing:</LI><UL><LI>lac+ChlI1+ChlD</LI><LI>ChlI2</LI><LI>ChlD</LI></UL><LI>The purpose of this was to prepare these parts for ligation. The following parts were to be ligated:</LI><LI>lac+Chli1+ChlD with Chli2</LI><LI>ChlD with lac</LI></UL><LI>The purpose of this was to make up for the same systems which previously did not show growth of viable cells.</LI><LI>Ligated composite parts were made with both a CAM and an AMP backbone</LI><LI>Ligated composite parts were plated out onto LB-CAM and LB-AMP and incubated:</LI><UL><LI>lac+Chli1+ChlD ligated with Chli2</LI><LI>ChlD with lac</LI><LI>Chli2 with lac</LI><LI>ChlG with lac</LI><LI>ChlM with lac</LI><LI>ChlM - Chloramphenicol and Kanamycin (to extend our current stock)</LI></UL><LI>These were made up to ensure we have correctly expressed our gene parts into our competent E.coli cells (unless stated otherwise).</LI>
Screening of parts<UL><LI>Plasmid prep followed by restriction digest (single digest with EcoRI, double digest with EcoRI and PstI) and gel electrophoresis</LI><UL><LI>lac + ChlD CAM (x5)</LI><LI>lac + ChlM CAM (x6)</LI><LI>lac + ChlG CAM (x4)</LI><LI>lac + plasto + ChlM CAM (x4)</LI><LI>lac + plasto + ChlM KAN</LI></UL><LI>This was done to screen for successful composite parts from previous transformations.</LI></UL>
IMAGE 20
Image 20: <I>All parts were religated. To counteract, the backbone needs to be swapped from CAM to AMP or KAN. This should improve colony selection to give us more successful results.</I><H5>Sunday 30 August </H5>
Backbone switch<UL><LI>Swapped plasmid preps from 27 August from CAM backbones into KAN or AMP backbones.</LI><LI>This is to facilitate selective process for re-screening.</LI></UL>
Construction of parts
<UL><LI>No new parts from 27 August grew on their antibiotic media.
 Cells were replated onto fresh media with appropriate antibiotic and there was several colonies.</LI></UL><H5>Tuesday 1 September </H5>
Growing parts<UL><LI>Colonies from media plated 30 August were inoculated into liquid cultures for screening on Thursday.</LI></UL><H5>Thursday 3 September </H5>
Screening of parts<UL><LI>Plasmid prep followed by restriction digest and gel electrophoresis):</LI><UL><LI>lac+ChlM (4 colonies) in CAM</LI><LI>lac+Chli2 (4 colonies) in CAM</LI><LI>lac+ChlD (4 colonies) in CAM</LI><LI>This was done as previous attempts to make these composite parts had failed.</LI></UL></UL>
IMAGE 21
Image 21: <I>No successful parts shown.</I><UL><LI>Construction of composite parts</LI><UL><LI>Digested 300 ng of lac plasmid into KAN and AMP backbones with SpeI and PstI, plus 100ng of gene (from CAM backbone) with XbaI and PstI. 25ng of digested lac plasmid with ligated with a gene part (3:1 molar ratio of gene and lac plasmid respectively).
Different backbones were used to facilitate selective process.</LI><UL><LI>lac + Chli2</LI><LI>lac + GUN4</LI><LI>lac + ChlM</LI><LI>lac + ChlG</LI><LI>lac + DVR1</LI><LI>lac + Plasto</LI><LI>lac + ChlD</LI></UL><LI>The ligation products were then transformed into composite cells and plated onto the correct media.</LI></UL><LI>Backbone switch</LI><UL><LI>Successful parts were in KAN or AMP backbones, needed to be switched to CAM backbones for iGEM submission.</LI><LI>Enzymatic digest of 100 ng gene part with 25ng CAM backbone (EcoRI and PstI.
Ligation procedure followed by plating samples onto LB-CAM. No colonies grew.</LI></UL><LI>Protein expression</LI><UL><LI>A control gel with known pET-clones was run (Image 22).</LI></UL></UL>
IMAGE 22<UL><LI>Other gel was run with</LI><UL><LI>lac + ChlI1+ChlD+Gun4 (lane 3)</LI><LI>Control pET clones (lanes 4 &amp; 5)</LI><LI>lac + CTH1 + YCF54 (lanes 8-11)</LI></UL></UL>
IMAGE 23
Image 23: <I>pET clones (lanes 4&amp;5) were used as controls. Bands 1-8 were excised and analysed with mass spectrometry. ChlI1 (band 8) was successfully detected in MS. No other proteins were found via MS.</I><H5>Monday 7 September </H5>
Screening of parts<UL><LI>Enzymatic digest completed (single digest with EcoRI, double digest with EcoRI and PstI) of all single gene parts plus several composite parts with a molecular ratio of 3:1, with an excess of the insert.</LI></UL>
Screened via gel electrophoresis.<UL><LI>ChlH, DVR1 and POR+DVR1+ChlD+ChlG showed good results</LI></UL>
IMAGE 24-26<H5>Tuesday 8 September </H5>
Transformation of competent E. coli cells<UL><LI>lac + POR + DVR1 + ChLP +ChlG</LI><LI>lac + POR</LI><LI>lac + Chli1 + Chld + GUN4</LI><LI>lac + CTH1 + YCF54 + Plasto</LI></UL>
Screening of parts
<UL><LI>Used PCR amplification to test for successful ChlH gene insertion and correct assembly of lac+POR+DVR1+ChlP+ChlG</LI></UL>
Image 27
Image 27:<I>ChlH was not observed - problem with primers, need to be redesigned.</I><H5>Thursday 10 September </H5>
Backbone switch for lac+ChlH from KAN to CAM<UL><LI>This is to ensure the part is ready for submission to iGEM. This involved 100ng of gene part with 25ng of CAM backbone and 0.5mL of EcoRI and PstI followed by enzymatic digest and ligation reactions, plated onto LB-CAM.</LI></UL>
Protein expression<UL><LI>Induced lac-promoter to begin expression of</LI><UL><LI>ChlI1+ChlD+GUN4</LI><LI>CTH1 + YCF54</LI><LI>CTH1 + YCF54 + plasto</LI></UL><LI>Protein gel ran.</LI></UL>
Image 28
Image 28: <I>Clear expression of ChlI1 in band 1. No expression of other genes.</I><H5>Friday 11 September </H5>
Backbone switch for lac + ChlH from KAN to CAM<UL><LI>Backbone switch showed no growth on LB-CAM. This process was redone.</LI></UL><H5>Monday 14 September </H5>
Backbone switch for ChlH from KAN to CAM<UL><LI>500 ng of ChlH was digested in a tube with 25ng of psb1C3 with EcoRI and PstI. Ligase was then added and the ligation products were transformed into competent cells. The cells were then plated out onto LB-CAM media.</LI></UL>
Functional assay for lac + ChlM<UL><LI>Cells were harvested from eight 50 ml overnight cultures, four of which had been induced IPTG,  the other four were auto-induced. An assay was undertaken to test for the conversion of Mg-protoporphyrin IX to Mg-protoporphyrin monomethyl ester, separating the products using high performance liquid chromatography. None of our cultures showed evidence of Mg-protoporphyrin IX monomethyl ester production when introduced to the appropriate substrates (Mg-protoporphyrin IX and S-adenosyl methionine). It is likely that the E. coli strain used for the assay was not ideal for expression under the lac promoter.</LI></UL><H5>Tuesday 15 September</H5><UL><LI>The backbone switch of ChlH from KAN to CAM was attempted again, as it failed yesterday.</LI><LI>PCR was conducted for ChlM, lac + ChlM and transformed DH5 alpha cells to determine; (a) that the lac promoter was definitely present in the composite parts we were using and (b) whether or not these cells were indeed transformants. Once the PCR products had been run on a gel it was clear that the lac promoter was present but the DH5 alpha cells had not been successfully transformed.</LI><LI>We ran a ‘show gel’ of all the successfully constructed parts from the chlorophyll synthesis pathway to include in our results section (Image 29).</LI></UL>
IMAGE 29<UL><LI>We conducted PCR with primers for ChlH to check that the part had been assembled in the correct order.</LI><LI>lac + ChlM transformed DH5 alpha cells were induced and set up to incubate whilst shaking overnight for a second attempt at a functional assay for ChlM to be conducted tomorrow.</LI><LI>Further, XL Blue and K12 E.coli cells were made competent and transformed with our lac + ChlM plasmids and plated out. These will be used for a third attempt for the assay if the second also fails.</LI></UL><H5>Wednesday 16 September</H5><UL><LI>The composite parts lac + DVR1, lac + GUN4 and lac + Plasto were transformed into K12 and XL Blue E. coli cells. These cells were later induced in preparation for a protein gel to check for expression.</LI><LI>A gel of the primer tests for ChlH was run and it was clear that too much DNA had been loaded. The PCR was repeated with a diluted ChlH template and these products were run on another gel. However, no products were seen on the gel.</LI><LI>The functional assay for ChlM was repeated with induced DH5-alpha cells that had been transformed with our lac + ChlM plasmids. There was clear evidence that Mg-protoporphyrin monomethyl ester was produced.</LI></UL>
IMAGE 30
Image 30:<I> HPLC. From the graph above, we were able to calculate the amount of this precursor molecule that had been converted by ChlM. In this instance, 26% of the precursor molecule had been converted by the enzyme ChlM. The ChlM pET expression successfully converted 12% of the precursor molecule, it becomes quite apparent that our protein expression was very successful.</I><H5>Thursday 17 September</H5><UL><LI>XL Blue and K12 E. coli cultures containing the plasmids lac + GUN4, lac + DVR1 and lac + Plasto and lac + ChlM were induced. Samples were then lysed and run on a protein gel. Clear bands were observed for GUN4, Plasto and ChlM and these bands were cut from the gel for trypsin digest in preparation for MALDI-TOF mass spectrometry tomorrow.</LI><LI>A gel of all 13 chlorophyll a synthesis pathway genes was run.</LI><LI>Further functional assays were performed for ChlM, this time with K12 and XL Blue cells. Only low level enzymatic activity was observed.</LI></UL>
IMAGE 31
Image 31:<I>...</I></DIV></DIV></DIV></DIV></DIV></BODY></HTML>