<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UrbanTundra_Edmonton_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UrbanTundra Edmonton/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Urban Tundra | Intelligent Innovation</TITLE></P><DIV id="ute-page"><HEADER id="ute-header" role="banner"><DIV class="container"><DIV class="header-inner"><NAV role="navigation"><UL><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton">Home</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Our_Story">Our Story</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Contact">Contact</A></LI><LI class="cta"><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Research">Research</A></LI></UL></NAV></DIV></DIV></HEADER><DIV id="ute-services-section"><DIV class="container"><DIV class="row"><DIV class="col-md-3 side-nav"><DIV class="side-nav-container"><UL style="list-style-type: none;"><LI class="side-nav-toplink">TEAM</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Our_Story">Our Story</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Collaborations">Collaborations</A></LI></UL><LI class="side-nav-toplink">PROJECT</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Description">Background</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Design">Gene Design</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Experiments">Experimental</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Proof">Bio Reaction</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Demonstrate">O<SUB>2</SUB></A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Notebook">Notebook</A></LI></UL><LI class="side-nav-toplink">PARTS</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Parts">BioBrick</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Composite_Part">CLD-</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Part_Collection">Collection</A></LI></UL><LI class="side-nav-toplink">SAFETY</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Lab_Safety">Lab Safety</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Protocals">Protocols</A></LI></UL><LI class="side-nav-toplink">ATTRIBUTIONS</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Attributions">Support</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/FullCitations">Citations</A></LI></UL><LI class="side-nav-toplink">HUMAN PRACTICES</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/BroaderApplications">Broader Applications</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Engagement">Outreach</A></LI></UL><LI class="side-nav-toplink">ACHIEVEMENTS</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Awards">Awards</A></LI></UL></UL></DIV></DIV><DIV class="col-md-9 content-area"><DIV class="content-container"><DIV style="text-align: center"><H1>Designing an Expression plasmid for Chlorite Dismutase</H1></DIV><H2>The Expression Plasmid Backbone</H2><P>
  For the expression of chlorite dismutase (Cld) we selected a  commercially available expression plasmid called PBP-38-441 from DNA 2.0. We had worked with this plasmid earlier during our Molecular biology training period and were satified with its performance. As shown in Figure 1 The plasmid contains the coding sequence for the purple chromophore “Tinsel” under the control of two Lac operators that sraddle a strong promoter from the T4 bacteriophage. The lacI repressor, which normally shuts of gene expression until induced with IPTG, is present on the plasmid, along with a kanamycin resistance gene for plasmid selection. The Tinsel cassette consists of the protein coding sequence and an RBS that can be excised from the plasmid at two BsaI sites that leave non-complimentary non-palindromic ens as shown below. The ability to turn Cld on and off  was a desirable feature in our design because of the possibility that  continuous strong expression of Cld might pose a selective disadvantage for cell growth.
</P><DIV class="figure"><H5>Figure 1.  Shown is an annotated plasmid map for the commercially available, lacI regulated expression plasmid PCB-38-441 (from the DNA 2.O chromophore “Paint Box collection. The Tinsel Cassette situated between flanking BsaI sites includes the RBS and produce a strong purple chromophore when induced with IPTG. An expanded view of the sequences that border the coding sequence is also shown up to the flanking BsaI sites at either end. The sticky ends produced by BsaI digested are shown in red.</H5></DIV><H2>Constructing Cld Expression Plasmids</H2><H3>Results and Discussions</H3><H3 style="font-style: italic;">Cloning:</H3><P>
  Our cloning strategy was to replace the “Tinsel” chromophore gene cassette contained in the host plasmid CPB-38-441,with the G-Blocks for Cld(SP+) and Cld(SP-) using the compatible BsaI sticky-ends located at the ends of our G-Blocks and the Tinsel gene cassette as shown in Figure 2 below. When transformed and plated, we expected that  ligation mixtures containing equal molar concentrations of  digested G-Block and plasmid would produce an equal number of white colonies (Cld-containing plasmids) and purple colonies (Tinsel containing plasmids)in the presence of kanamycin. We also plated transformations that contained either no DNA or the host plasmid CPB-38-441. The absence of colonies on the former would verify that resulting colonies are not due to contamination. The presence of purple colonies on the latter would demonstrate that the lack of colonies on either Cld plate was not due to failed transformation but rather a problem upstream in the cloning procedure.
</P><P>
  The plating results are shown below (Figure 2). As expected white and purple colonies were present for both Cld(SP+) and Cld(SP-) ligations. The observation that no colonies were detected on the “No DNA&quot; control showed that these colonies were not contaminants but derived from the DNA that we added intentionally. The presence of purple colonies on the positive control plate showed that the transformation procedure was functional.  We observed, however, that on our recombinant DNA plates, there were fewer white colonies than purple colonies. Our supervisor suggested that this may be because of partially synthesized G-Block DNA that is missing the terminal BsaI sites.
</P><DIV class="figure"><H5>Figure 2. Shown above are the  expected fragments resulting from the BsaI digestion of the Cld G-Blocks and the CPB-38-441 plasmid. Both G-Blocks and plasmid have compatible stick-ends that assure their replacement with the Tinsel cassette in the correct orientation and at single copy.</H5></DIV><DIV class="figure"><H5>Figure 3. The schematic on the left display the two possible ligation products (gene replacement with Cld and Tinsel re-ligation). Images on the right show kanamycin LB agar plates for the “No DNA” negative control (-)ctrl, the host plasmid control [(+)ctrl], the Cld(-SP) ligation and the Cld(+SP) (bottom right).</H5></DIV><P style="font-style: italic;">Plasmid isolation and gel analysis:</P><P>
  While the presence of white colonies are consistent with the hypothesis that they contain plasmids with Cld inserts, they do not prove the hypothesis. White colonies could also arise from the rare ligation of the host plasmid backbone using non-compatible ends. To verify that they contain Cld inserts. The plasmids must be purified and shown to contain inserts that correspond to the sizes expected of Cld(SP+) and Cld(SP-).
</P><P>
  	We therefore purified (mini-prepped) the DNA from overnight cultures derived from six white colonies selected from the Cld(SP+) plate and six colonies selected from the Cld(SP-). Shown below (Tables 1 and 2) are the spectroscopic results for each plasmid isolates which provide a measure of their concentration (ng/uL) and purity (OD 260/280 ratio). Each measurement was made twice from separate aliquots.
</P><P>
  As shown below, duplicate values are very similar. For all plasmid isolates the concentration ranges from a low of 76ng/uL to a high of 98ug/uL while 260/280 ratios range from 1.77 to 1.83. The 260/280 ratio is a measure of the amount of DNA (260nm) relative to protein (280nm). Values of 1.8 are considered to be free of protein (Thermo Scientific Technical Bulletin T009). Therefore all plasmid isolates show minimal protein content.
</P><P>
  We compared the insert sizes of the 12 plasmid isolates relative to the insert (Tinsel) of the host plasmid, CPB-38-441. The inserts were released by digestion with BsaI, electrophoresed and imaged as shown below (Figure 3).The CPB plasmid backbone (CPB Bkbn) fragment (3.9 kBP), found opposite the 4.0 kBP marker. In the positive control column, the lowest band sits between the 0.7 kBP and 1.0 kBP marker, which is the 0.75 kBP Tinsel gene fragment. Lower bands on the Cld+sp gel are slightly above the Tinsel band, but still below the 1.0 kBP marker, indicating that the sequence is between 0.75 kBP to 1.0 kBP long. This corresponds to the 0.9 kBP sequence coding the Cld+sp gene. On the Cld-sp gel, lower bands are found between Tinsel bands and Cld+sp bands suggesting a sequence that is 0.75 kBP to 0.9 kBP long, likely the 0.8 kBP Cld-sp gene. This fragment is shorter than Cld+sp, which is expected of its shorter sequence without the signal peptide. All of these digestions show higher bands higher than the CPB backbone. Based on discussions with our supervisor we have tentatively assigned these bands as “nicked circular plasmid” (N.C.) and linear plasmid.
</P><P>
  	Based on these results we believe that we have successfully substituted the Tinsel gene cassette with the Cld(SP+) and Cld(SP-) cassettes in the expression plasmid CPB-38-441
</P><DIV class="figure"><H5>Figure 4. Shown are the reverse gel images for 12 plasmid isolates digested with BsaI. Resulting products are labeled on the left, while marker sizes that straddle positions of interest are shown on the left. The positive control lane one is a digest of the host plasmid CPB-38-441.    </H5></DIV><H3>Methods</H3><H3 style="font-style: italic;">Cloning:</H3><P>
  Each g-block was combined with the expression plasmid CPB-38-441 at a molar ratio of 1:1 in  a 20 uL reaction containing 1x Cut Smart buffer (NEB) and 1uL BsaI HF. The reactions were incubated at 37O C for one hour and the BsaI was heat-killed for 20 minutes at 80<SUP>O</SUP>C. Ligations using T4 DNA ligase (NEB) were done by adding to each reaction: 4uL of 10x ligation buffer, 1uL ligase and 15uL MilliQ H<SUB>2</SUB>O. Ligase reactions were incubated overnight at room temperature.
</P><P>
  	The next day the ligase was heat-killed as described above. For the transformations, 5uL of each reaction was combined with 100uL of chemically competent DH5α <EM>E. coli</EM> on ice. For the negative control, no DNA was added to cells and for the positive control, the original host plasmid CPB-38-441 added. Samples were then incubated on ice for 30 minutes. Following a 45 second heat-shock at 42O, cells were then placed on ice for 5 minutes and then combined with 1 mL of LB broth followed by incubation at 37OC for 1 hour. 100uL of each reaction was then plated on LB agar plates containing kanamycin. Plates were incubated overnight upside-down at 37<SUP>OC.
</SUP></P><H3 style="font-style: italic;">Plasmid recovery and gel analysis:</H3><P>
  	For gel analysis of Cld-containing plasmid candidates, 3 white colonies were selected from the each of the Cld(SP-) and Cld(SP+) plates to inoculate overnight cultures. Next day, Plasmids were mini-prepped from 2 mL of each culture using the QIAprep Miniprep protocol (QIAGEN). The purity and dsDNA concentrations of the extracted plasmid samples were assessed  by spectrophotometry (NanoDrop).
</P><P>
  For gel analysis 5uL of each miniprep was digested for 1 hour at 37<SUP>O</SUP>C with 1U (0.1uL) of Bsa in a 20 uL reaction containing Cut Smart buffer (1x). 4uL of load buffer was added and 10uL of each was loaded onto a Tris-acetate 1% agarose gel. Gene Ruler 1KB ladder (Thermo) was used as the molecular size standards. The gel was then stained with ethidium bromide and the photographed on a UV light box.
</P><H2>Gene Sequences:</H2><P>
  We obtained our gene sequences for Cld(SP+) and Cld(SP-) from the Arizona State University Collegiate iGEM team. We provided them with both our expression plasmids in addition to primers designed to sequence the forward and reverse strands of each gene cassette. The sequence we obtained the Cld(SP-) forward strand as shown below precisely matched our designed G-Block sequence detailed in <A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Design">Gene Design</A>.  In two independent sequencing reactions (Shown below, Fig. 5;  Seq-4 and Seq-5). The reverse strand sequence was of poorer quality and is not displayed. By comparison, the sequence of Cld(SP+) showed a single G to T substation corresponding to an Ala to Ser substitution at position 78 of the polypeptide chain (not shown). Based on our subsequent observation that Cld(SP+) protein was non-functional, it was not submitted to the registry.
</P><H2>Testing the toxic effects of perchorate and chlorite on <EM>E. coli</EM></H2><H3 style="font-style: italic;">Summary:</H3><P>
  In light of the toxity of perchlorate and chlorate, we decided first to investigate the toxic effects of the compounds on <EM>E. coli</EM> growth and survival. The goal of this experiment was to discover the range of concentrations of sodium perchlorate (NaClO<SUB>4</SUB>) and sodium chlorite (NaClO<SUB>2</SUB>) in LB broth. Two serial dilutions were performed (one each for sodium perchlorate and sodium chlorite) and <SPAN style="font-style: italic;">E.Coli</SPAN> viability was measured qualitatively by observation of solution turbidity. The range of compound concentrations that we selected were based on the previous study of  Kwolek-Mirek et al. (2011) who performed a similar analysis in the yeast <SPAN style="font-style: italic">Saccharomyces cerevisae</SPAN>. Our experiments revealed that <SPAN style="font-style: italic;">E.Coli</SPAN> is able to survive in a NaClO<SUB>4</SUB> solution at concentrations up to 0.1M, and are not at all viable in any of our tested concentrations of NaClO<SUB>2</SUB> (0.0125M and greater) (Figure 6. The results from this experiment will be applied to future work in which <SPAN style="font-style: italic"> E.coli</SPAN> will be transformed to express a plasmid that will allow them to create the chlorite dismutase and perchlorate reductase enzymes. If we know the range of concentrations that <SPAN style="font-style: italic">E.coli</SPAN> is able to survive, we will know at which concentrations of perchlorate and chlorite we should begin with to test our transformed <SPAN style="font-style: italic">E.coli</SPAN>. Our alternative strategy would be add these compounds to saturated cultures expressing the enzymes understanding that it will kill the cells but may still produce oxygen in a non sustainable way.
</P><H3>References</H3><P>
  Kwoleck-Mirek, M., Bartosz, G. and Spicket, C. (2011) <STRONG>Yeast</STRONG> 28 pp. 595-609
</P><H3 style="font-style: italic;">Experimental Design:</H3><P>
  Our treatment groups were the cultures grown in 0.2M, 0.1M, 0.05M, 0.025M, and 0.0125M solutions of perchlorate and chlorite ions in LB. To create these concentrated solutions, we performed serial dilutions through five different test tubes, each containing 5.0mL of the perchlorate/chlorite - LB solution. A sixth test tube was included as a control group, which contained only LB. Each serial dilution was performed twice to ensure precision in our results.
</P><H3 style="font-style: italic;">Results (Figure 6):</H3><H3 style="font-style: italic;"><STRONG>Methods:</STRONG></H3><DIV style="font-size: medium;font-family: " raleway=""><P style="font-style: italic">
  Proper precautions regarding the chemical hazards and bio hazards were followed.
</P><P style="font-style: italic">
  Preparation of the <SPAN style="font-style: none">E. Coli</SPAN> Stock Culture:
</P><UL><LI>Inoculate and incubate 5 mL <SPAN style="font-style: italic">E. coli K12</SPAN> culture. </LI></UL><P style="font-style: italic">
  Preparation of the Inoculated LB Broth Master Solution:
</P><OL><LI>Pipette 80 mL of Broth into a sterilized jar. </LI><LI>Inoculate the LB Broth with 320 μL of the prepared <SPAN style="font-style: italic">E. coli</SPAN> stock culture.</LI><LI>Gently swirl the contents of the jar.</LI></OL><P style="font-style: italic;"><STRONG>Investigation with Sodium Perchlorate Stock Solution:</STRONG></P><P style="font-style: italic;">Preparation of Sodium Perchlorate Stock Solution: </P><OL><LI>Measure 0.489756 g of sodium perchlorate.</LI><LI>Dissolve sodium perchlorate in 10 mL of LB Broth to make a 0.4 M stock solution. </LI></OL><P style="font-style: italic;">Serial Dilution of Sodium Perchlorate Solution with Stock Culture (Treatment Groups): </P><OL><LI>In each of 5 culture tubes, pipette 2.5 mL of the inoculated LB Broth master solution.</LI><LI>Pipette 2.5 mL of the sodium perchlorate stock solution into one culture tube. </LI><LI>Dilute solution by mixing contents. </LI><LI>Take 2.5 mL of diluted sodium perchlorate solution and pipette into the next culture tubes. Repeat for the remaining 3 culture tubes to achieve a serial dilution.</LI><LI>Label each tube according to its concentration. </LI></OL><P style="font-style: italic;">Preparation of the Control Group: </P><OL><LI>1.	Pipette 2.5 mL of the inoculated LB Broth Master Solution into the sixth culture tube. </LI></OL><P style="font-style: italic;">Incubation of Cultures: </P><OL><LI>Place prepared culture tubes into the incubator shaker overnight at 37°C along with the Sodium Chlorite cultures. </LI></OL><P style="font-style: italic;"><STRONG>Investigation with Sodium Chlorite Stock Solution: </STRONG></P><P style="font-style: italic;">Preparation of Sodium Chlorite Stock Solution: </P><OL><LI>Measure 0.361764 g of sodium chlorite.</LI><LI>Dissolve sodium chlorite in 10 mL of LB Broth to make a 0.4M solution.</LI></OL><P style="font-style: italic;">Serial Dilution of Sodium Chlorite Solution with LB Broth (Treatment Groups): </P><OL><LI>In each of 5 culture tubes, pipette 2.5 mL of the inoculated LB Broth master solution.</LI><LI>Pipette 2.5 mL of the sodium chlorite stock solution into one culture tube. </LI><LI>Dilute solution by mixing contents. </LI><LI>Take 2.5 mL of diluted sodium perchlorate solution and pipette into the next culture tubes. Repeat for the remaining 3 culture tubes to achieve a serial dilution.</LI><LI>Label each tube according to its concentration.</LI></OL><H3 style="font-style: italic;">Preparation of the Control Group:</H3><OL><LI>Pipette 2.5 mL of the inoculated LB Broth Master Solution into the sixth culture tube. </LI></OL><H3 style="font-style: italic;">Incubation of cultures:  </H3><OL><LI>1.	Place prepared culture tubes in the incubator shaker overnight at 37°C along with the Sodium Perchlorate cultures.</LI></OL></DIV><H2>Riley Broth: A novel growth media based on purified canine biowaste</H2><DIV class="figure"><H5>Figure 7</H5></DIV><P><STRONG>Overview</STRONG>
  A fundamental objective of our project was to develop a methodology for reusing Martian colonial biomass as a feedstock for growing our recombinant Escherichia coli (<SPAN style="font-style: italic">E.coli</SPAN>). Our idea was to eliminate the need for astronauts to bring containers of LB with them on the space shuttle and allow for other essential materials, such as food, to be brought on board. The current standard for growing <SPAN style="font-style: italic">E.coli</SPAN> (along with a large number of other bacteria) is lysogeny broth. This broth is made up of a combination of yeast extract, salt, tryptone, and water. These four simple ingredients provide a nutritionally complete medium for <SPAN style="font-style: italic">E.coli</SPAN> to grow on, and can be supplemented with antibiotics for selection of recombinant plasmids. We hypothesized that utilizing a simple drying and sterilization procedure would allow us to generate an equivalent nutritionally complete medium. Our results indicate that our media (which we dubbed “Riley Broth” after the canine companion who produced it) is capable of sustaining levels of growth for standard DH5α <SPAN style="font-style: italic">E.coli</SPAN> cells that are comparable to those of growth on LB.
</P><P><STRONG>Riley Broth Production</STRONG>
  The first step of the production of Riley Broth (RB) was to freeze dry the original sample of feces to remove all the moisture from the sample. This was done by placing the sample under vacuum for 36 hours at 2660 mT of pressure.
  
  After drying, the dry mass was crushed into a fine powder. The fine powder was then split into two separate samples for replication. Each of the samples were then dissolved in 500 mL of MilliQ water then autoclaved. The sample was then filtered through a standard Grade 1: 11 μm filter paper to remove the non-water soluble waste. It was then filtered again through a 0.22 μm filter. We hypothesized that our media might lack the appropriate levels of salts. In order to circumvent this issue, 15mL of 5 M NaCl was added to the purified media to establish a salt concentration of 0.15M.
</P><P><STRONG>Riley Broth Testing</STRONG>
  In order to test the capability of RB to sustain bacterial growth, we performed a simple overnight culture using an inoculant of DH5α cells. This strain was selected based on our usage of it as a chassis for recombinant gene expression. Five conditions were then established to test RB’s ability to sustain <EM>E. coli</EM> growth. A total volume of 5 mL was used for each condition. Luria broth was inoculated with 10 microliters of DH5α culture to ensure viability of the cells, as well as to provide a reference for normal overnight growth, as is shown on the rightmost side of the image (Figure 8.) To ensure the sterility of the medium and to observe any possible change in optical density of RB over time, one condition was composed of RB with no inoculant. Next, three experimental groups were established, which contained increasing amounts of RB. These groups contained RB diluted in water ¼, ½, and undiluted (Figure 8, central three cultures moving from left to right). Each of these experimental groups were inoculated in the same manner and grown at 37 °C overnight before measuring each sample’s OD600 (600 nm). The optical density at 600 nm was used to estimate the total amount of cells present in each culture following overnight growth based on a standard curve.  Measurements of optical density were taken after normalizing the spectrophotometer with a “blank” sample, which contained only fresh media.
</P><DIV class="figure"><H5>Figure 8<P><STRONG>Data Collection</STRONG></P><P><STRONG>Discussion</STRONG></P><P>
  Our experimental results demonstrated that showed that RB media can facilitate the proliferation of <EM>E. coli</EM> cells at a rate that is comparable to LB. The cell count measurement collected in undiluted RB (4.40 x10<SUP>9</SUP> cells/mL) grown overnight is quite close to that of undiluted LB (4.40 x 10<SUP>9</SUP> cells/mL) grown under the same conditions. Although this was only performed as a single replicate, the dose-dependent increase in cell count observed upon adding RB from the ¼ dilution (4.5 x 10<SUP>8</SUP> cells/mL) to the undiluted RB culture demonstrated that the growth of <EM>E.coli</EM> in the RB cultures was in fact dependent on the contents of the RB medium. Further replicates of these growth experiments, as well as an increase in the number of time points analyzed, will allow us to more rigorously compare the growth potential of our RB medium.
  
  We can conclude from this experiment that a suitable substitute for LB media can be prepared through the use of canis feces. Theoretically, feces are made up of roughly 75% water and 25% dry mass, where 30% of the dry mass is dead bacteria; 30% is indigestible matter; 10-20% contains cholesterol and other fats; 10-20% inorganic substances; and 2-3% contains protein1.  Although this composition likely varies from organism to organism, it is clear that it contains most of the components necessary to sustain bacterial life. While it is doubtful that Riley Broth will replace LB on planet Earth, we believe that it provides a viable alternative for growing microorganisms in extraterrestrial environments.
</P></H5><H4>References</H4><P><SUP>1</SUP><SPAN style="font-style: italic;"> Encyclopædia Britannica</SPAN> (Encyclopædia Britannica, 2016), s.v “Feces | biology” by The Editors of Encyclopædia Britannica, accessed October 14, 2016, https://www.britannica.com/science/feces.</P><H2>Our first attempt at Oxygen generation</H2><H3 style="font-style: italic">Experimental Design:</H3><P>
  To generate oxygen in our system, from plasmid-borne Clds in <EM>E. coli</EM>, first requires the addition of three components as discussed under Project Description. IPTG is required to induce Cld gene expression. Fe<SUP>2+</SUP> and levulinic acid are required as precursors for heme synthesis which functions as a catalytic co-factor in the Cld enzyme. Since heme is synthesized in the cytoplasm we expected to see O<SUB>2</SUB> produced in both Cld(SP+) and Cld(SP-) strains. We hoped that this experiment would shed light on relatively efficiency and capacity of O<SUB>2</SUB> in the periplasm Cld(SP+) relative to the cytosol. Following gene expression, chlorite would be added to a final concentration of 0.1M where we expected to see the generation of bubbles.
</P><H3 style="font-style: italic">Method:</H3><P>
Six 5 mL LB overnight cultures were set up using single colonies that were picked from the first 3 of each of the Cld-containing candidates  previously described in Fig.8 [Cld (SP+) 1,2 and 3; Cld(SP-) 1,2 and 3]. To each of the six cultures was added IPTG (1.0mM), levulinic acid (2.0mM) FeSO<SUB>4</SUB> (2.0mM). These concentrations approximate the concentrations reported by Thorell et al. (2003). Cells were incubated overnight at 37<SUP>O</SUP> C in a shaking incubator. The next day, each 5mL culture was pelleted and resuspended in 0.4mL of 10mM Tris, 1mM EDTA pH 8.0. The resuspended pellets were then transferred to plastic cuvettes as shown below in Fig.8). Chlorite (0.1M) was then added for a final concentration of 0.5M. </P><H3 style="font-style: italic">Results and Discussion:</H3><P>
  Cultures expressing Cld(SP-) were noticeably reddish brown in colour whereas those derived from Cld(SP+) showed the same colour of  conventional <EM>E col</EM>i. This suggests that heme-containing Cld(SP-) is being produced whereas heme-containing Cld(SP+) is not being produced. When chlorite was added to the resuspended cell pellets, the CLD(SP-) samples began to bubble almost instantaneously. In contrast, no bubbling was observed in the Cld(SP+) samples. The failure of  Cld(SP+) to produce O<SUB>2</SUB> is not clear. It might have resulted from the ala to ser mutation revealed from the gene sequence or alternatively the failure of the non-native MalE signal peptide.
</P><DIV class="figure"><H5>Figure 9</H5></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="ute-cta" style="background-image: url(https://static.igem.org/mediawiki/2016/5/5d/T--UrbanTundra_Edmonton--Footer_Image.jpg);"><DIV class="container"><DIV class="col-md-12 text-center animate-box"><H3>Explore With Us.</H3><DIV><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Research" class="btn btn-primary btn-outline with-arrow"> View Research </A></DIV></DIV></DIV></DIV><FOOTER id="ute-footer" role="contentinfo"><DIV class="container"><DIV class="col-md-3 col-sm-12 col-sm-push-0 col-xs-12 col-xs-push-0"><H3>Urban Tundra</H3><P style="font-family: &quot;Raleway&quot;, Arial, sans-serif;color: grey;">Twenty heads are better than one.</P><P><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Team" class="btn btn-primary btn-outline with-arrow btn-sm">Meet the Team </A></P></DIV><DIV class="col-md-6 col-md-push-1 col-sm-12 col-sm-push-0 col-xs-12 col-xs-push-0"><H3>Quick Links</H3><UL class="float"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton">Home</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Our_Story">Our Story</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Sponsors">Sponsors</A></LI></UL><UL class="float"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Research">Research</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Contact">Contact Us</A></LI><LI><A href="https://2016.igem.org/Main_Page">iGEM</A></LI></UL></DIV><DIV class="col-md-2 col-md-push-1 col-sm-12 col-sm-push-0 col-xs-12 col-xs-push-0"><H3>Get in Touch</H3></DIV><DIV class="col-md-12 ute-copyright text-center"><P style="margin-left: 475px; color: grey; font-family:&quot;Raleway&quot;, Arial, sans-serif;">Compiled by <A href="http://theimmortalstudioz.webs.com">Immortal Studioz</A></P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>