<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UrbanTundra_Edmonton_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UrbanTundra Edmonton/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Urban Tundra | Intelligent Innovation</TITLE></P><DIV id="ute-page"><HEADER id="ute-header" role="banner"><DIV class="container"><DIV class="header-inner"><NAV role="navigation"><UL><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton">Home</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Our_Story">Our Story</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Contact">Contact</A></LI><LI class="cta"><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Research">Research</A></LI></UL></NAV></DIV></DIV></HEADER><DIV id="ute-services-section"><DIV class="container"><DIV class="row"><DIV class="col-md-3 side-nav"><DIV class="side-nav-container"><UL style="list-style-type: none;"><LI class="side-nav-toplink">TEAM</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Our_Story">Our Story</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Collaborations">Collaborations</A></LI></UL><LI class="side-nav-toplink">PROJECT</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Description">Background</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Design">Gene Design</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Experiments">Experimental</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Proof">Bio Reaction</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Demonstrate">O<SUB>2</SUB></A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Notebook">Notebook</A></LI></UL><LI class="side-nav-toplink">PARTS</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Parts">BioBrick</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Composite_Part">CLD-</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Part_Collection">Collection</A></LI></UL><LI class="side-nav-toplink">SAFETY</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Lab_Safety">Lab Safety</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Protocals">Protocols</A></LI></UL><LI class="side-nav-toplink">ATTRIBUTIONS</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Attributions">Support</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/FullCitations">Citations</A></LI></UL><LI class="side-nav-toplink">HUMAN PRACTICES</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/BroaderApplications">Broader Applications</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Engagement">Outreach</A></LI></UL><LI class="side-nav-toplink">ACHIEVEMENTS</LI><UL class="sublist"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Awards">Awards</A></LI></UL></UL></DIV></DIV><DIV class="col-md-9 content-area"><DIV class="content-container"><DIV style="text-align: center"><H1><STRONG>Project Overview and Inspiration:</STRONG></H1></DIV><P>Stephen Hawking recognized that humans are like eggs in a basket - drop the basket and all is lost. Planet Earth is the basket, and over the long-term, a calamity - be it through climate change, disease, asteroid collisions, etc. - is a statistical certainty. Colonizing Mars is the first step in distributing the eggs to other baskets. However, colonies must be able to sustain themselves on what can be found locally. There isn’t much on Mars, but there is perchlorate, and that is why we feel that this project and others like it will contribute significantly to what is evolving into a major international effort. In short term, Mars missions will develop, and spin-off a range of creative technologies that will allow us all to live more comfortably, efficiently, and cleanly on planet Earth.
</P><P>Our project was hatched from what we learned of iGEM from the “Fusion” Science Expo we attended last fall, and the movie, <EM>The Martian</EM>. When we dug deeper, we stumbled on the work of Davila et al [1] who proposed that toxic concentrations of ClO<SUB>4</SUB> in Martian soil was actually a resource that could be exploited both as a source of rocket fuel and O<SUB>2</SUB>, while remediating the soil in the process. It was also shown that 2 enzymes from soil bacteria on earth could biodegrade ClO<SUB>4</SUB> to O<SUB>2</SUB>. Here, we plan to build on this work and the work of others [2], [3] by developing an integrated and inexpensive process for: 1) perchlorate extraction, purification and concentration using activated charcoal, and 2) using engineered <EM>E. coli</EM> that grows on colony bio-waste to efficiently convert ClO<SUB>4</SUB> to O<SUB>2</SUB>. A graphical overview of this proposed process is shown below. 
</P><P>Our process begins with a methodology for mining and extracting perchlorate. We hypothesize the usage of an activated charcoal column-based purification and concentration method. The concentrated perchlorate is then be shuttled to either be used as a base ingredient for the production of rocket fuel (ammonium perchlorate) or to the bioreactor. The bioreactor contains the engineered microbes, which act as biocatalysts to reduce perchlorate down to oxygen. The oxygen is then captured, transferred and made available to an enclosed Martian colony. Biowaste from the colony can then also be used as a feedstock for the bioreactor, as a method of reducing the load on the limited biomass resources available to the colony.
</P><DIV style="text-align: center"><H1><STRONG>Project Description:</STRONG></H1></DIV><P>A Martian colony must make efficient use of the planet’s limited resources. Martian soil contains high levels of perchlorate (ClO<SUB>4</SUB>) making it toxic. Our team is developing an automated prototype for ClO<SUB>4</SUB> extraction, its conversion to rocket fuel, and its genetically engineered biodegradation to oxygen. Remediated soil can then be recycled for construction and agriculture.Our goal was to take this perchlorate reducing pathway and to transform it into <EM>E. coli</EM> in order to effectively use it to remediate Martian regolith for use in a Martian colony. We want to show that Chlorite Dismutase when expressed in an <EM>E. coli</EM> chassis, is functional in regards to the conversion of chlorite ions into both oxygen and chloride ions. Our project is essentially a proof of concept, showing the viability of using biological systems as a method of oxygen production.
</P><P>Our project has two main themes: 1) the extraction, purification and concentration of ClO<SUB>4</SUB> from Martian soil, and 2) the bioconversion of ClO<SUB>4</SUB> to oxygen using genetically engineered <EM>E. coli</EM>. ClO<SUB>4</SUB> is highly soluble in water. We are testing the idea that ClO<SUB>4</SUB> can be highly enriched and concentrated inexpensively using activated charcoal, as previously shown for a similar compound. For O<SUB>2</SUB> production, we plan to synthesize and express genes from the soil bacterium <EM>Ideonella dechloratans</EM>, which is capable of degrading perchlorate to molecular oxygen. A schematic showing the mechanics of this process in <EM>Ideonella dechlorotans</EM>, is shown below:
</P><H5 style="font-size: 24px; text-align: center">Figure 1: The process by which soil bacteria degrade perchlorate to oxygen</H5><P>
This pathway utilizes two enzymes, perchlorate reductase (shown above in orange) and chlorite dismutase (shown above in purple). Perchlorate is first reduced to chlorate, releasing one molar equivalent of oxygen gas, through the action of perchlorate reductase, a two-subunit enzyme. Chlorate is then reduced completely down to one molar equivalent of oxygen gas and one molar equivalent of chloride ions by the action of chlorite dismutase. The entire reduction process takes place in the periplasmic space due to the toxicity of the perchlorate and chlorate oxyanions [1]. The pathway results in the net production of two molar equivalents of oxygen and one molar equivalent of chloride from one mole of perchlorate. 
</P><P>There exist over 50 different strains of bacteria that are capable of fully reducing perchlorate to oxygen gas. These bacteria are quite often facultative anaerobes, and often utilize perchlorate as the terminal electron acceptor in metabolic processes under anaerobic conditions, due to its high reduction potential [2]. The reduction of perchlorate is typically coupled to the oxidation of organic carbon sources or alternative electron donors such as hydrogen sulfide [2]. One of the original hypotheses surrounding perchlorate reduction was that it was simply cross-reacting with nitrate reducing catalytic machinery. However, when nitrate-reducing bacterial cell extracts were challenged with perchlorate, this always led to the accumulation of toxic amounts of the chlorite oxyanion product, suggesting that there exists a distinct pathway by which specific bacteria are capable of degrading perchlorate completely. The pathway by which perchlorate is reduced utilizes two enzymes as shown above (Figure 1), perchlorate reductase and chlorite dismutase. 
</P><P>Perchlorate reductase forms a heterodimer composed of subunits PcrA and PcrB that localizes to the periplasm. Although no structural information has been obtained on this enzyme, it is thought to be a part of the type II DMSO reductase family [1]. However, the genes encoding both subunits of this enzyme have not been successfully expressed in other bacterial species, such as <EM>E.coli</EM>, and so the level of characterization of this enzyme remains limited, although limited kinetic studies have been performed [3]. Deletion of the pcrA led to perchlorate sensitivity in <EM>D. aromatica</EM>, providing a strong implication that it is necessary for perchlorate metabolism [1]. 
</P><P>Chlorite dismutase is a single subunit enzyme that has been relatively well-characterized. It has been cloned and expressed in <EM>E.coli</EM> and extensive activity assays have been performed with it [4]. This enzyme is active in the periplasm as well, due to the toxicity of the substrate. Chloride dismutase is critical for ameliorating chlorite-mediated toxicity to the cell, by reducing it down fully to molecular oxygen and chloride ions. Additionally, chlorite dismutase is a heme-containing protein [5]. 
</P><P>Given the further level of characterization of chlorite dismutase, our team decided to attempt to clone and express the gene encoding this enzyme, Cld, before moving on to attempt the cloning and expression of the gene encoding perchlorate reductase, given that perchlorate reductase has never been functionally expressed.  Our strategy for expressing Cld is shown below (Figure 2):
</P>Figure 2: Expression of the Cld gene<P>
The gene for chlorite dismutase, which we engineered to either have the signaling peptide (Cld+SP) or not (Cld-SP), is placed under the control of the lac operator. The backbone also contains genes encoding kanamycin resistance and the lac repressor. To switch gene expression off, lac repressor proteins bind to the promoter region, preventing RNA polymerase (RNA pol) from docking onto the promoter region and starting gene transcription. Isopropyl β-D-1-thiogalactopyranoside (IPTG) is then added to turn on gene expression. IPTG is a lactose metabolite that can bind to the lac repressor, causing the lac repressor to release the promoter region. The release of the lac repressor then allows RNA polymerase to bind, triggering the expression of chlorite dismutase. After translation, the newly synthesized chlorite dismutase then incorporates a heme group, which is synthesized by the addition of levulinic acid and Fe<SUP>2+</SUP>. Depending on whether the signaling peptide is present or not, the enzyme is then trafficked to either the periplasm (Cld+SP) or secreted into the cytoplasm (Cld-SP). Chlorite dismutase then catalyzes the oxidation of chlorite to produce oxygen and chloride ions, as shown by the reaction scheme in the red box. 
</P><P>An additional advantage to beginning with the cloning of the Cld gene is that the measurement of oxygen is considerably easier than measuring the production of chlorite, which is the product of the perchlorate reductase. By measuring oxygen, we would be able to have some idea of the efficiency of our engineered biocatalyst. Although the introduction of the perchlorate reductase would dramatically affect the overall catalytic efficiency, this would provide us with a rough estimate of the production capacity of our microbes. In order to assess the viability of our process, we looked at the total amount of oxygen required by a single human to survive.
</P><P>The average adult inhales/exhales around 7 to 8 L of air/min.1 If we take the higher value:</P><P style="text-align: center">8L/(min)*1440  min/day = 11,520L</P><P>Given that the air is composed of ~20% oxygen and the air that is exhaled is roughly 15% oxygen, only about 5% of oxygen is consumed with each breath.6 Thus, the volume of oxygen consumed by the average adult is:</P><P style="text-align: center"> V = 11,520L * 20% = 576L</P><P>If we assume SATP, the relationship between the volume and molar amount is given by:
</P><P style="text-align: center">V = 24.8  L/mol</P><P>Then the amount of oxygen needed per day (n<SUB>O<SUB>2</SUB></SUB>) is given by:</P><P style="text-align: center">n<SUB>O<SUB>2</SUB></SUB>=v/V = (576L)/(24.8L/mol) = 23.2mol of O<SUB>2</SUB></P><P>Martian soil was found to have a perchlorate (ClO4-) content of 0.5 to 1%w/w.7 Perchlorate can be oxidized to produce oxygen using perchlorate reductase and chlorite dismutase,8 as shown by the equation below:</P><P style="text-align: center">ClO<SUB>4</SUB><SUPER>-</SUPER> yields 2O<SUB>2</SUB> + Cl<SUPER>-</SUPER>
Therefore, the mass of ClO4- required per day is given by:
</P><P style="text-align: center">m<SUB>ClO<SUB>4</SUB></SUB> = n<SUB>O<SUB>2</SUB></SUB> * (1 mol ClO<SUB>4</SUB>)/(2 mol of O<SUB>2</SUB>)*M<SUB>ClO<SUB>4</SUB></SUB></P><P style="text-align: center">m<SUB>ClO<SUB>4</SUB></SUB> = 23.2 mol of O<SUB>2</SUB>*(1 mol ClO<SUB>4</SUB>)/(2 mol of O<SUB>2</SUB> )*99.45  g/mol = 1.15 kg of ClO<SUB>4</SUB></P><P>Thus, the mass of Martian soil (msoil) required to supply one average adult with a day’s supply of oxygen, using the lowest possible concentration, is:</P><P style="text-align: center">m<SUB>soil</SUB> = m<SUB>ClO<SUB>4</SUB></SUB>/C<SUB>ClO<SUB>4</SUB></SUB> = (1.15 kg of ClO<SUB>4</SUB>)/(0.5%) = 230 kg of soil</P><P>This calculation is assuming 100% catalytic efficiency of conversion from perchlorate to oxygen. This provided us with a benchmark of what the goal would be for oxygen production. 
</P><P>Over the course of the last 4 months, our team was able to clone and express a synthesized form of the Cld gene in E.coli. The cloning was confirmed by gel electrophoresis and sequencing. We have shown preliminary data to suggest that our recombinant bacteria are capable of producing O2 from chlorite added to pure culture. Although the efficiency of oxygen production was low, we believe that there are multiple ways to optimize our production using a variety of means.  Additionally, we have also developed a method for the recycling of Martian colony biowaste into a highly enriched media and have demonstrated its capability to support bacterial growth. A detailed description of our data is presented in the results section.  
</P><H1>References:</H1><OL><LI>Bender KS, Shang C, Chakraborty R, Belchik SM, Coates JD, Achenbach LA. Identification, characterization, and classification of genes encoding perchlorate reductase. J Bacteriol. 2005;187:5090–5096. doi: 10.1128/JB.187.15.5090-5096.2005. </LI><LI>Coates JD, Achenbach LA (2004) Microbial perchlorate reduction: rocket-fuelled metabolism. Nat Rev Microbiol 2:569–580</LI><LI>S.W. Kengen, G.B. Rikken, W.R. Hagen, C.G. van Ginkel, A.J. Stams. Purification and characterization of (per)chlorate reductase from the chlorate-respiring strain GR-1. J. Bacteriol., 181 (1999), pp. 6706–6711</LI><LI>van Ginkel C. G.; Rikken G. B.; Kroon A. G.; Kengen S. W. (1996) Purification and characterization of chlorite dismutase: a novel oxygen-generating enzyme. Arch. Microbiol. 166, 321–326.10.1007/s002030050390</LI><LI>Hofbauer S., Schaffner I., Furtmüller P.G., and Obinger C. (2014c) Chlorite dismutases – a heme enzyme family for use in bioremediation and generation of molecular oxygen. Biotechnol J 9: 461–473.</LI><LI>http://health.howstuffworks.com/human-body/systems/respiratory/question98.htm. Accessed on October 19, 2016.</LI><LI>Davila, A. F.; Willson, D.; Coates, J. D.; McKay, C. P. Perchlorate on Mars: a chemical hazard and a resource for humans. International Journal of Astrobiology 2013, 12, 321-325.</LI><LI>Bender, K. S.; Shang, C.; Chakraborty, R.; Belchik, S. M.; Coates, J. D.; Achenbach, L. A. Journal of Bacteriology 2005, 187, 5090.</LI><DIV class="ute-cta" style="background-image: url(https://static.igem.org/mediawiki/2016/5/5d/T--UrbanTundra_Edmonton--Footer_Image.jpg);"><DIV class="container"><DIV class="col-md-12 text-center animate-box"><H3>Explore With Us.</H3><DIV><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Research" class="btn btn-primary btn-outline with-arrow"> View Research </A></DIV></DIV></DIV></DIV><FOOTER id="ute-footer" role="contentinfo"><DIV class="container"><DIV class="col-md-3 col-sm-12 col-sm-push-0 col-xs-12 col-xs-push-0"><H3>Urban Tundra</H3><P style="font-family: &quot;Raleway&quot;, Arial, sans-serif;color: grey;">Twenty heads are better than one.</P><P><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Team" class="btn btn-primary btn-outline with-arrow btn-sm">Meet the Team </A></P></DIV><DIV class="col-md-6 col-md-push-1 col-sm-12 col-sm-push-0 col-xs-12 col-xs-push-0"><H3>Quick Links</H3><UL class="float"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton">Home</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Our_Story">Our Story</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Sponsors">Sponsors</A></LI></UL><UL class="float"><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Research">Research</A></LI><LI><A href="https://2016.igem.org/Team:UrbanTundra_Edmonton/Contact">Contact Us</A></LI><LI><A href="https://2016.igem.org/Main_Page">iGEM</A></LI></UL></DIV><DIV class="col-md-2 col-md-push-1 col-sm-12 col-sm-push-0 col-xs-12 col-xs-push-0"><H3>Get in Touch</H3></DIV><DIV class="col-md-12 ute-copyright text-center"><P style="margin-left: 475px; color: grey; font-family:&quot;Raleway&quot;, Arial, sans-serif;">Compiled by <A href="http://theimmortalstudioz.webs.com">Immortal Studioz</A></P></DIV></DIV></FOOTER></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>