Team:UAM Poznan/Experiments
Home
PROJECT MAIN IDEA APPLICATIONS
TEAM ATTRIBUTIONS COLLABS
RESULTS PROMOTERS ORFs ACHIEVEMENTS BIOBRICK TESTING
PARTS
METHODS
NOTEBOOK PROMOTERS ORFs SAFETY AND ETHICS PATENTING
GALLERIES
HUMAN PRACTICES
UAM_Poznan Team
METHODS
MEDIA PREPARATION AND BACTERIAL CULTURES
LB medium (Lysogeny-Broth)
Tryptone        10g/L
NaCl            10g/L
Yeast extract   5g/L
Dissolve the components in 800ml of sterile deionized water, adjust the volume to 1 liter and sterilize by autoclaving. Solid media contain 1,4-1,5% agar and should be well mixed after autoclaving, then cooled to about 55°C  before the appropriate antibiotic is added. Then medium should be mixed well again and poured on plates as about 3mm layer without bubbles, under clean bench conditions.
2xLB medium (2xLysogeny-Broth)
Tryptone 20g/L
NaCl  10g/L
Yeast extract 10g/L
Dissolve the components in 800ml of sterile deionized water, adjust the volume to 1 liter and sterilize by autoclaving. Solid media contain 1,4-1,5% agar and should be well mixed after autoclaving, then cooled to about 55°C before the appropriate antibiotic is added. Then medium should be mixed well again and poured on plates as about 3mm layer without bubbles, under clean bench conditions.
SB-PKB medium (Super Broth-PKB)
20xPKB buffer/100ml
KH2PO4 4.6g
K2HPO4 24.3g
Weigh the components and dissolve in 100ml of water, filter and sterilize by autoclaving.
SB medium/L
Tryptone 12g
Yeast extract 24g
Glycerol  4ml
Dissolve in 950ml of water, sterilize by autoclaving. Prior to use, add 50ml of 20xPKB buffer.
SOC medium
Tryptone       2%
Yeast extract  0.5%
NaCl        0.05%
KCl        2.5mM
MgCl2 10mM
Glucose        20mM
Weigh the components and dissolve in water, add KCl, MgCl2 and sterilize by autoclaving. At the end add filter sterilized glucose (pore size 0.22µm).
5xM9 salts
Na2HPO4 64g/L
KH2PO4 15g/L
NaCl            2.5g/L
NH4Cl           5g/L
To 800ml of H2O add weighed components and stir until dissolved. Adjust to 1L with distilled water and sterilize by autoclaving.
M9 medium
Measure 770 ml of distilled sterile water and add 200ml of 5xM9 salts. Then add 2ml of 1M MgSO4, 20ml of 20% carbon source (e.g. glucose), 100µl of 1M CaCl2, 10ml of 1% thiamine and adjust to 1L with distilled water.
Plates with M9 medium
Recipe for 200ml of M9 medium~10 plates.
1. Melt 100ml of 3% agar in the microwave and transfer to a water bath 50°C.
2. Add 53ml of sterile water preheated to 50°C.
3. Add 40ml of 5xM9 salts, kept at the same temp.
4. Add 20μl of warm 1M CaCl2 and mix quickly.
5. Add 0.4ml of warm 1M MgSO4 and mix well.
6. Add 4ml of 20% filter sterilized solution of glucose or another carbon source ( final conc. 0.4%).
7. Add 2ml of 1% thiamine (filter sterilized), mix well, keep at 50°C
8. Add the appropriate antibiotic, mix well and pour the medium on plates.
Preparation of electro-competent Escherichia coli cells
1. Grow overnight a single colony of E. coli DH5α in 250ml flask with 100ml of LB medium (120 rpm, at 30-37°C).
2. Transfer 0.5ml of the bacterial culture to a new 250ml flask with 100ml of LB medium and grow the culture at 37°C to get OD600=0.5 or 0.6.
3. After 10-15 minutes of incubation on ice, centrifuge the culture at 4000 rpm for 20 min. at 4°C.
4. Suspend bacterial pellet in 100ml of deionized water (4°C) and centrifuge as above. Repeat this step one more time.
5. Suspend the cells in 40ml of cold deionized water with 1% of glycerol and centrifuge as before. (Keep tubes on ice!)
6. Suspend the pellet in 1-2ml of 10% glycerol, split the suspension into 50-100μl portions and freeze in liquid nitrogen.
7. Store at -80°C for up to 1 year.
Transformation of Escherichia coli via electroporation
1. Incubate electro-competent cells (50-100µl) on ice for 5-10 min. and let it thaw slowly.
2. Keep cells on ice and add 2-4μl of plasmid DNA (10-50ng) or 7μl of purified DNA template from CPEC to obtain 100-500 colonies.
3. Transfer the mixture to the cold electroporation cuvette 0.2mm. Make sure that electrodes of cuvette are dry.
4. Perform the electroporation at 2.5 kV. Pulse time should be close to 5ms.*
5. Immediately add 800μl of LB medium/SOC medium, slowly mix (pipet up and down) and transfer to a new tube.
6. Incubate at 37°C for 30-45 min with shaking 300-350 rpm.
7. Centrifuge the tube (3 min, 5000rpm) and suspend the pellet in 50-100µl of LB medium.
8. Spread the bacterial culture onto the 90mm plate with solid LB medium with the proper antibiotic.
9. Incubate overnight at 37°C.
* Remember to provide the proper density of competent cells. Too high density can cause that cuvette can explode due to an electric arc!
E.coli glycerol stocks
Bacterial culture       25ml
50% glycerol  40µl
LB medium               200µl
1. Grow a single colony of E. coli DH5α, after transformation, into a 100ml flask with 25ml of LB medium overnight, at 120 rpm, at 37°C.
2. Centrifuge the bacterial culture 5 min at 5000 rpm and suspend the pellet in solution of glycerol and LB medium.
3.  Split the suspension into 15µl portions to eppendorf tubes and freeze in liquid nitrogen.
4.  Store at -80°C.
CONSTRUCT DESIGN AND CLONING METHODS
DNA amplification
Primers for PCR were designed using Primer 3 [1,2] to have the same calculated melting temperature of 60°C. This way fragments of similar sizes can be amplified at the same time on one thermal cycler.
Phusion PCR- standard PCR mix
In a sterile PCR tube mix reagents in the following order:
5xHF buffer (NEB)                        10µl
H20                                      up to 50 µl
dNTP mix (10mM each)                     1 µl
Primer forward (100 µM)                  0.25µl
Primer reverse (100 µM)                  0.25µl
DNAtemplate (0.1-5ng)                    x µl
Phusion DNA polymerase (NEB)          0.5µl
Standarized PCR conditions
Remember to put the probes to preheated thermocycler to avoid non-specific product amplification.
Electrophoresis of DNA fragments in agarose gels
1. Weigh required amount of agarose and pour into a 250 ml screw cap Simax bottle.
2. Add appropriate volume of 1xTAE Buffet and mix.
3. Heat for up for 2 minutes in microwave oven until the agarose sol is homogeneous. Cool the bottle shortly under tap water. Add ethidium bromide (10mg/ml). Mix well.
4. Pour the agarose sol into a gel casting tray. Insert the appropriate comb. If necessary, clear the bubbles with a pipette tip. Wait until the gel is fully solidified.
5. Pour TAE buffer onto the gel, remove the comb and casting gates. Pour TAE buffer to buffer tanks to cover the gel with 1mm of the buffer.
6. Mix the samples with 6xloading dye in a 5:1 ratio. Put the samples into the wells. We were using two DNA molecular weight markers: 100 bp DNA Ladder (https://www.neb.com/products/n3231-100-bp-dna-ladder) and 1 kb DNA Ladder (https://www.neb.com/products/n3232-1-kb-dna-ladder) .
7. Close the lid and apply proper running conditions- 5V/cm.*
* Running time depends on the size of the gel and the expected resolution.
Gel extraction
1. After electrophoresis, photograph agarose gel on UV transiluminator and cut out the DNA fragments that you are interested in with a clean razor blade.
2. Place agarose block in a weighed eppendorf tube.
3.  Purify DNA fragments by using Zyppy silica columns according to the protocol provided by ZYMO RESEARCH (https://www.zymoresearch.com/dna/dna-clean-up/gel-dna-recovery/zymoclean-gel-dna-recovery-kit) .
4. Determine the final concentration of eluted DNA by NanoDrop.
Chromosomal DNA isolation from Escherichia coli
1. Spin down 0.5 ml of E. coli culture OD600=0.6 for 3 min at 5000g and remove the supernatant.
2. Suspend the pellet in 200μl of buffer A (150 mM NaCl,100 mM EDTA pH=8.0).
3. Add 10 μl of fresh lysozyme suspension (10 mg/ml), incubate at 37°C for 15 min.
4. Add 20 μl 10% SDS and 1μl proteinase K, mix well, incubate at 56°C for 15 min.
5. Cool down the mixture to RT, add 100 μl chloroform and 100 μl phenol pH=8.0, vortex for 1 min. Centrifuge for 5 min at 12 000 rpm.
6. Carefully transfer the upper water phase containing nucleic acids to a new tube, add equal volume of ethanol, mix well and centrifuge as above.
7. Remove the supernatant and wash the pellet twice with 70% ethanol, spin down for 1 min at 12 000 rpm, remove the supernatant carefully to leave the pellet only. Dry the pellet at RT for 5-10 min.
8. Add 50 μl H2O, incubate at 56°C for 2 min, mix well by pipetting.
9. Check the quality of DNA on 0.8% agarose gel. The main DNA band should represent molecules longer than 20 kb. 10. Use 0.1- 0.5μg of chromosomal DNA to amplify a gene of interest.
All promoter sequences we use derive from the chromosomal DNA of E. Coli K-12, DH5α.
Plasmid isolation and purification
TEG buffer/50ml
1M glucose                      1.5ml
250mM Na2EDTA pH=8.0            6ml
1M Tris-HCl pH=8.0                      0.75ml
H2O                                    41.75ml
NaOH-SDS buffer/40ml
10M NaOH                        0.8ml
10% SDS                         4ml
H2O                             35.2ml
KAc buffer/ 50ml
Glacial acetic acid             7ml
5M potassium acetate            36ml
H2O                             7ml
1. Transfer 25ml of bacterial culture grown overnight in LB medium at 37°C to a conical tube and centrifuge at 9 000 rpm at 4°C. Discard the supernatant.
2. Suspend bacterial pellet in 10ml of TEG buffer.
3. Add 10ml of NaOH-SDS buffer and mix by inverting the tube. Leave the mixture for no longer than 2 min to obtain a clear lysate.
4. Add 10ml of KAc buffer, vortex quickly and centrifuge at 9 000 – 12 000 rpm at 4°C.
5. Carefully transfer the supernatant to a new tube.
6. Add 0.6 – 0.7 V of isopropanol, mix and put into the fridge for minimum 30 min.
7. Centrifuge at 9 000 - 12 000 rpm for 30 min. at 4°C.
8. Discard the supernatant, suspend the pellet in 800μl of miliQ water, add RNase.
9. Incubate at 37°C for 15 min or keep in the fridge for overnight.
10. Just proceed to Zippy Plasmid Miniprep Kit (href="https://www.zymoresearch.com/dna/plasmid-dna-purification/zyppy-plasmid-miniprep-kit) procedure starting from step 1.
One silica column will bind up to 50μg of plasmid DNA at one binding session, so one can purify 200μg of high quality plasmid DNA on one mini-column. After checking the quality by agarose gel electrophoresis and A260 measurement, the plasmid DNA is ready for sequencing and RE digestions, cloning, and transfections.
Construct verification by colony PCR and Sanger DNA sequencing
Colony PCR
1. Pick up several colonies of interest with a sterile toothpick and streak them on one or two new selective plates with numbered fields. Wipe the remaining part of the colony on the bottom of a 1.5 ml Eppendorf tube. Use the same numbering for tubes as for fields on plates.
2.  Add 50-100µl of H2O to each tube, incubate at 96°C for 10 min. Cool down by centrifugation for 2 min at 12 000 rpm. Transfer 2µl to a new PCR tube.
3. Use Taq DNA polymerase for colony PCR. Prepare a standard PCR mix leaving 2µl of the total volume for the template. Consider an additional volume for a positive control, like just an empty plasmid.
4. Add 48µl of PCR mix to every tube. Prepare a positive control. Run a standard PCR.
5. Check the sizes of amplicons by gel electrophoresis in agarose gel.
Sanger sequencing
Sanger sequencing of constructs was performed in our Faculty facility on ABI Prism capillary system. We provided DNA, which were purified, with VF (TGCCACCTGACGTCTAAGAA) and VR (ATTACCGCCTTTGAGTGAGC) primers and primers used for amplification of parts of our constructs – these shorter ones, without overlaps. To obtain better results, i.e. longer sequences we purified the primers by electrophoresis in 15% polyacrylamide gel with 7M urea.
CONSTRUCT AND PART DESIGN
Primers design
Primers were designed by using the NEBuilder with a minimal length of overlaps of 18 bp. Parts of constructs were designed to be easily amplified with primers of the same melting temperature of 60°C. These PCR products of expected size were excised from agarose gels and reamplified with a pair of primers containing sequences of overlaps, and again purified from agarose gels. The initial primers we use to verify the structure of the final construct, as well as for sequencing. Verification of construct structure by Sanger sequencing was performed in our Faculty sequencing lab.
Construct assembly by CPEC
The purified PCR products containing overlaps were quantitated on DeNovix DS11 spectrophotometer. Sizes of these DNA fragments were checked by electrophoresis in agarose gels. Then fragments were mixed to obtain the molar ratio, like: 1(promoter): 1(CDS): 0.5(vector).
Optimal amounts of DNA fragments are as in example: Promoter (300bp) 30ng; CDS (1000bp) 100ng; PSB1C3 vector (2000bp) 100ng.
Preparation of a standard CPEC mix
In a sterile PCR tube mix reagents in the following order:
5xHF buffer (NEB)                          10µl
Mixture of fragments with overlaps         xµl
H2O                                 up to 50µl
dNTP mix (25mM)                            1µl
Phusion DNA polymerase (NEB)              0.5µl
The table below provides standard reaction conditions for CPEC reaction.
Purification of the CPEC products before electroporation:
CPEC products can be purified by two methods: phenol/chloroform extraction or by using silica columns Zymoclean Gel DNA Recovery Kit (www.zymoresearch.com) .
Phenol/chloroform method:
1. Add 150µl H2O to the sample after CPEC, 100µl of chloroform and 100µl of phenol (pH=8.0)/ vortex and centrifuge 3 min at 12 000 rpm.
2.  Collect the upper phase and transfer it to new eppendorf tube, add 100µl of chloroform, vortex and centrifuge as in the previous step.
3.     Collect the upper phase and transfer it to new DNA LoBindTube.
4.     Add 2µl of glycoblue, mix, add 0.1 V of 3M sodium acetate (pH=4.8), mix well, and 2.5-3 volumes of 99% ethanol and mix again.
5.  Cool at -20°C for at least 2 h. Centrifuge 40 min at 12 000 rpm (4°C). Discard the supernatant and suspend the blue pellet in 20 µl of sterile H2O. Use 7 µl for electroporation.
Purification by silica columns:
Proceed to Zymoclean Gel DNA Recovery Kit procedure starting from step 2.
Coding sequence optimization
The very high number of citations (777) of a paper published by Kane in 1987 indicates that may investigators faced difficulties with an efficient production of recombinant proteins in Escherichia coli and associate these difficulties with the presence of codons very rarely occurring in coding sequences of the host cell. Our first construct dedicated to this phenomenon was sfGFP with four rare arginine codons located before two stop codons TAA (sfGFP-4R). We wanted to check to what extent four such codons can be detrimental to the biosynthesis of a stable and soluble recombinant protein in E. coli. Surprisingly it appeared that there is no clearly visible difference between the rate of sfGFP and sfGFP-4R biosynthesis. Then we extended this motif to eight consecutive AGG / AGA codons, and we have found that sfGFP-8R biosynthesis rate is still almost identical to that of sfGFP without rare arginine codons. There are several codon optimization methods available, which are usually based on codon usage in the host cell, however the results of optimization are quite often disappointing. We decided to check what is the difference in the protein synthesis rate between two very different versions of the same reading frame, one composed of the most common codons (sfGFP-B, the best choice) in E. coli orfeome and another codon-monotonous as well but composed of the rarest codons (sfGFP-W, the worst case). In sfGFP-W coding sequence appeared 7 dispersed AGG codons, 10 isoleucine ATA codons, one neighbouring with AGG, 15 leucine CTA codons, one neighbouring with ATA, and two pairs of consecutive CTA codons. We have found and show it in results section, that in E. coli cells growing in rich media the introduction of 32 very rare codons to a 248 codons long sequence coding for a well soluble protein at a moderate level (like in pBAD systems) is not sufficient to observe any significant decrease in the rate of its translation + accumulation several hours after induction of the gene expression. We decided then to check if the same results will be observed in the case of a fluorescent protein of a different sequence. We have chosen mRFP. Looking for any other general way to optimize ORFs, we are working now on contrasting ORFs which are AT or GC rich, with codons optimization based on codon adaptation index and on most common and the rarest codon contexts.
ORFs
We have prepared four different rankings for different sets of coding sequences: a whole orfeome set generated for Escherichia coli str. K12 substr. MG1655 (NCBI: NC_000913.3) and sets of 300, 200 and 100 most abundant proteins for Escherichia coli based on Ishihama et al., 2008. The most abundant proteins with the highest log-copy number values were chosen (see Ishihama et al., 2008). The maximization and minimization of Codon Pair Bias was performed using CodonComposer for sfGFP and RFP open reading frames and default GA parameters. The additional constraints the algorithm was checking for were lysine-encoding A-homopolymers (that may cause ribosome sliding) and restriction enzyme sites of the RFC10 format. The mRNA secondary structures were predicted using the RNAfold server and the default parameters.
Codon context optimization
CodonComposer (https://github.com/MelaniaNowicka/CodonComposer) is a piece of software which allows to calculate a codon pair ranking for any set of ORFs and optimize the codon context considering restriction enzyme sites, A-homopolymers removal and maintenance of the effective number of codons at the same time. Codon context optimization is based on the ranking approach. The codon context ranking (scores for each possible codon pair) is calculated for any chosen set of open reading frames using the Codon Pair Bias measure proposed by Coleman et al., 2008. The overall score for the ORF of interest is calculated as the arithmetic mean of individual pair scores.
The implemented genetic algorithm (GA) allows to obtain close to optimal results and optimization is much less time-consuming than it would be for an exact algorithm when considering additional constrains. GA is an algorithm inspired by the processes of natural selection. The algorithm uses population-based approach which means that the optimization is performed for a large set of possible solutions called individuals. All steps mimic processes occurring in nature: individuals undergo selection, recombination and mutation to „evolve” a better solution during iterations called reproductive cycles. For more details, please check the CodonComposer GitHub repository.
MEASUREMENTS
Fluorescence measurements
To measure the strength of the promoters, we measured  the florescence of sfGFP which is used as reporter protein.
1. Drive the bacterial culture in 25ml of medium and induce the expression when the culture reaches OD600 =0.4.
2. Collect 500µl of bacteria after each hour during 6 hours of measurement and centrifuge immediately (4°C, 3 min, 14 000rpm).
3. Discard the supernatant and suspend the pellet in 0.5ml of buffer (5M urea, 50mM Tris-HCl pH=8.0, 20mM EDTA, 0.1% SDS).
4. Incubate the solution 5 min at 55°C and then centrifuge 4 min, 13 000 rpm.
5. Transfer the lysate to a new eppendorf tube.
6. To measure the fluorescence dilute the lysate 10 times in buffer (50mM TrisHCl pH=8.0, 5mM EDTA) and transfer the sample to the 96-wells black plate. The buffer provides the proper ionic strength, in contrast to water, so the fluorescence of sfGFP does not change upon dilution.
7. Besides samples, put on the plate: blank sample- the buffer and GFP solution- which provides max emission level of fluorescence (50 000 RFU). Thereby all results will be normalized to the one, known maximum value.
8. Put the plate into Infinite 200 PRO microplate reader (Tecan) (http://lifesciences.tecan.com/products/ reader_and_washer/microplate_readers/infinite_200_pro) and perform the measurement. Apply the excitation light wavelength to  485nm, at which GFP is maximally excited - and the emission light wavelength to 535nm, at which the GFP emission is the highest.
OD600 Measurements
1. To control the density of bacteria during fluorescence measurements use a spectrophotometer- we were using DeNovix. Collect 500µl of  bacterial culture, after each hour during 6 hours of measurement, and  immediately centrifuge (4°C, 3 min, 14 000 rpm).
2. Discard the supernatant and suspend the pellet in 0.5ml of 0.3M NaoAc pH=5.0. The acetate pH=5.0 causes that GFP protein stop emit the fluorescence, what is very important because the fluorescence of protein can reduce the OD600 values during measurement and provide unreliable data.
3. Dilute solution 10 times, compared to suspended in acetate pellet, and measure the OD600.
References
1. Koressaar T1, Remm M. (2007) Enhancements and modifications of primer design program Primer3. Bioinformatics 23(10):1289-91.
2. Untergasser A1, Cutcutache I, Koressaar T, Ye J, Faircloth BC, Remm M, Rozen SG. (2012) Primer3—new capabilities and interfaces. Nucleic Acids Res. 40(15):e115.
