<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UAM_Poznan_Notebook-promoters skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UAM Poznan/Notebook-promoters</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="background"><DIV id="page"><DIV id="header"><DIV id="navigation"><UL><LI id="menu_home"><A href="https://2016.igem.org/Team:UAM_Poznan">Home</A></LI><LI id="menu_project"><A href="#" class="noclick">PROJECT</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Description">MAIN IDEA</A><A href="https://2016.igem.org/Team:UAM_Poznan/Demonstrate">APPLICATIONS</A></SPAN></LI><LI id="menu_team"><A href="https://2016.igem.org/Team:UAM_Poznan/Team">TEAM</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Attributions">ATTRIBUTIONS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Collaborations">COLLABS</A></SPAN></LI><LI id="menu_results"><A href="#" class="noclick">RESULTS</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Results">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Results-ORF">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Achievements">ACHIEVEMENTS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Biobrick_testing" class="double-line">BIOBRICK TESTING</A></SPAN></LI><LI id="menu_parts"><A href="https://2016.igem.org/Team:UAM_Poznan/Basic_Part">PARTS</A></LI><LI id="menu_methods"><A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">METHODS</A></LI><LI id="menu_notebook"><A href="#" class="noclick">NOTEBOOK</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook-promoters">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Safety" class="double-line">SAFETY AND ETHICS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Patenting">PATENTING</A></SPAN></LI><LI id="menu_galleries"><A href="https://2016.igem.org/Team:UAM_Poznan/Galleries">GALLERIES</A></LI><LI id="menu_hp"><A href="https://2016.igem.org/Team:UAM_Poznan/HP/Silver" id="hp_menu" class="double-line">HUMAN PRACTICES</A></LI></UL></DIV></DIV><P><TITLE>UAM_Poznan Team</TITLE></P><DIV id="contents"><DIV class="box"><DIV><DIV id="news" class="body"><DIV class="sidebar"><H3>CALENDAR</H3><UL><LI><A href="#june">JUNE</A></LI><LI><A href="#july">JULY</A></LI><LI><A href="#august">AUGUST</A></LI><LI><A href="#september">SEPTEMBER</A></LI><LI><A href="#october">OCTOBER</A></LI></UL></DIV><DIV><DIV><DIV><H1><CENTER>NOTEBOOK - PROMOTERS</CENTER></H1><H2 id="june">JUNE</H2><SPAN><B>3<SUP>rd</SUP> week</B></SPAN><P align="justify">Since our exams at the university are over, this week we get to start our iGEM project! First of all: lab arrangement. We’ve prepared lots of 1xLB, 2xLB, SB-PKB and M9 media <A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">(see our Methods)</A>, and also Petri dishes with solid media. The whole laboratory had to be cleaned and tips and eppies autoclaved. Sometime before, we designed primers necessary for assembly of our constructs in NeBuilder and ordered them in (IDTDNA). We use overlaps so that fragments attach to each other during CPEC reaction - circular polymerase extension cloning, method which doesn't require additional enzymes but Phusion DNA polymerase. </P><SPAN><B>4<SUP>th</SUP> week</B></SPAN><P align="justify">This week started with the assembly of new constructs- pBAD-M5'UTR <A href="http://parts.igem.org/Part:BBa_K2014003">(BBa_K2014003)</A> and pxylS-M5'UTR <A href="http://parts.igem.org/Part:BBa_K2014004">(BBa_K2014004)</A>. We aim to introduce new 5’UTR modification into Arashort1 <A href="http://parts.igem.org/Part:BBa_K1741000">(BBa_K1741000)</A> and XylS <A href="http://parts.igem.org/Part:BBa_K1741009">(BBa_K1741009)</A> promoters- synthetic UTR without secondary structures and with well-positioned RBS, which was previously tested in Mel2 promoter <A href="http://parts.igem.org/Part:BBa_K1741004">(BBa_K1741004)</A>. For more details, please check our Biobricks’ pages. 
First of all we had to add overlaps to our fragments using PCR reaction. After amplifying all fragments we checked whether they have a proper length on the agarose gel. We cut appropriate bands and purified them on Zymo silica columns. Then we performed their assembly by CPEC. We purified CPEC products (properly assembled vectors with our promoters, let's hope) and used it for <I>E.coli</I> transformation by electroporation. We spread bacteria on Petri dishes with LB medium and appropriate antibiotic and obtained many single colonies the next day (we’ve got colonies for both pBAD-M5'UTR and pxylS-M5'UTR). We picked some colonies for both constructs and transferred them onto plates with M9 minimum medium containing xylose- to induce the promoters and allow the translation of sfGFP, to check whether promoters were assembled properly and if our 5’UTR modifications didn't ruin their activity. All chosen colonies had shined, so we inoculated some of them in liquid LB medium for the overnight culture. Next day flasks with bacteria were centrifuged and pellets were collected for plasmids isolation <A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">(see our Methods page)</A>.</P><H2 id="july">JULY</H2><SPAN><B>1<SUP>st</SUP> week</B></SPAN><P align="justify">This week we focused on preparing bacterial stocks (with all of our promoters!) for further usage during promoters activity measurements.
We still have to assemble many more constructs with various promoter modifications, so for sure other bacterial stocks will be prepared later on. In order to prepare bacterial stocks, we had to start overnight cultures of all needed promoters, centrifuge it the next day, add appropriate amount of antibiotic, glycerol and LB medium to the pellet and froze it. We keep all our stocks at -80°C. </P><SPAN><B>2<SUP>nd</SUP> week</B></SPAN><P align="justify">Next step in our project is the assembly of constructs containing xylose promoters. We worked on several promoters induced by xylose last year. New constructs include improvement of XylS* 5’UTR region by introducing two versions of enhancer from bacteriophage T7- RBS from gene 10. One version is with additional hairpin, second version is without the hairpin (pxylS-E2_5'UTR and pxylS-E1_5'UTR).
*XylS is a shortened version of XylWT promoter (native to <I>E.coli</I>), consisting of only XylA fragment with 5’UTR from constitutive promoter proD <A href="http://parts.igem.org/Part:BBa_K1741009">(see BBa_K1741009 biobrick page)</A>.
We added overlaps to promoters’ fragments and amplify them using PCR reaction. PCR products were checked on the agarose gel. Appropriate bands were cut and purified on columns. Then we performed their assembly by CPEC, purified CPEC products and used it for <I>E.coli</I> transformation by electroporation. The next day we obtained colonies for both xylose constructs. Some colonies were picked and transferred onto plates with M9 minimum medium containing xylose- to induce the promoters and allow the expression of sfGFP protein. Colonies for pxylS-E1_5'UTR shined (i.e. construct assembly went well), but colonies for pxylS-E2_5'UTR didn’t, which suggests that this additional hairpin in 5’UTR sequence have a rather negative impact on sfGFP translation.
We inoculated shining (Xyl-E1) and non-shining (Xyl-E2 with hairpin) colonies into liquid LB medium for the overnight culture. Next day flasks with bacteria were centrifuged and pellets were collected for plasmids isolation. We also prepared templates for sequencing of two constructs. Sequencing results showed that the assembly went perfectly. 
<SPAN><B>3<SUP>rd</SUP> week</B></SPAN></P><P align="justify">It is a measurement week! We want to check the activity and strength of all promoters that we’ve assembled so far. That would be all arabinose, rhamnose, xylose and melibiose inducible promoters, and we want to compare them with T7 promoter from pET system. 
The measurement is based on checking sfGFP fluorescence, as all of our constructs control sfGFP expression. The higher sfGFP fluorescence, the stronger promoters activity <A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">(see our Method page)</A>.
We measured our promoters’ strength during 6h <I>E.coli</I> (DH5α and BL21-DE3) culture in LB medium after induction with 0.4% of sugar and 0.4 as starting OD<SUB>600</SUB>. We observed and collected samples from 25ml culture every one hour. We treated collected samples as it is described in Methods (separately for OD measurement and separately for checking protein fluorescence). Bacterial supernatant with released sfGFP was later measured on Tecan (Infinite 200 PRO) using plate reader.</P><P align="justify">When it comes to the results- for rhamnose promoters, Rha1 (without EcoRI site) is the strongest, pBAD-M5'UTR (Ara1-UTR, version of AraC-pbad without AraC frame, with altered 5’UTR) is the strongest among arabinose promoters and Mel2 among melibiose promoters- although it’s still pretty weak. From xylose promoters, pxylS-E1_5'UTR (XylS-E1) turns out to be the strongest among all tested promoters- almost as strong as T7! 
Because of this promising results, we’re going to introduce this E1 enhancer with additional ribosome binding site into the strongest promoters from other sugar groups. </P><SPAN><B>4<SUP>th</SUP> week</B></SPAN><P align="justify">This week we performed the assembly of constructs containing E1_5’UTR - Arashort1-E1 (pBAD-E15'UTR, <A href="http://parts.igem.org/Part:BBa_K2014000">BBa_K2014000</A>), Rha1-E1 (prha1-E15'UTR, <A href=" http://parts.igem.org/Part:BBa_K2014001">BBa_K2014001</A>), Mel2-E1.
To perform the assembly of constructs with E1 enhancer, we had to amplify all fragments with overlaps-adding-primers. After PCR reaction we checked lengths of products on the gel, cut appropriate bands and perform their purification. Then we performed CPEC and transformed <I>E.coli</I> with assembled constructs. After transformation, bacteria were streaked on plates with solid LB medium. The next day we were able to observe colonies on plates with all constructs. Single colonies were chosen for transferring on plates with M9 minimal medium containing appropriate sugar (arabinose for Ara promoters, rhamnose for Rha promoters and melibiose for Mel promoters) to induce the promoter and check which colonies show sfGFP fluorescence, i.e. were properly assembled. Luckily, all transferred colonies for Arashort1-E1 were shining, almost all for Rha1-E1 and two colonies for Mel2-E1. We picked those shining ones and inoculated liquid LB medium with them for the overnight cultures, so that we would be able to isolate plasmids the next day. Isolated plasmids were send for sequencing. After sequencing we found out that Arash1-E1 was assembled properly, but Rha1-E1 has a deletion and Mel2-E1 has a substitution. </P><H2 id="august">AUGUST</H2><SPAN><B>1<SUP>st</SUP> week</B></SPAN><P align="justify">This week we took part in informatics workshops with IT specialist, who taught us a lot about html programming. We managed to start working on our wiki page- we’ve chosen web template and created homepage and subpages.</P><SPAN><B>2<SUP>nd</SUP> week</B></SPAN><P align="justify">Our expression system is not leaky (i.e. promoters are induced in the presence of specific sugars) and can be efficiently blocked by glucose, which we proved last year.  However, this year we want to repeat the measurements once again to be sure those newly assembled promoters are also tight. First we focused on xylose promoters that control sfGFP expression- we have collected and measured fluorescence during 6h <I>E.coli</I> cultures, every one or two hours, with or without induction with xylose  and with/without glucose.</P><SPAN><B>3<SUP>rd</SUP> and 4<SUP>th</SUP> week</B></SPAN><P align="justify">Two weeks of measurements. Activity measurements, comparisons to T7 promoter from pET system, measurements of promoters’ activity on different media. Also measurements of tightness of rhamnose, arabinose and melibiose promoters- we are checking whether they can be efficiently blocked by glucose (whether there is no further sfGFP production). We also perform fluorescence intensity measurements and data analysis.</P><H2 id="september">SEPTEMBER</H2><SPAN><B>1<SUP>st</SUP> and 2<SUP>nd</SUP> week</B></SPAN><P align="justify">To prove that our expression system is working perfectly, we aimed to produce sfGFP and mRFP proteins. We’ve chosen pBAD-M5'UTR <A href="http://parts.igem.org/Part:BBa_K2014003">(BBa_K2014003)</A>, our arabinose promoter with altered 5’UTR, to drive the expression of fluorescent proteins.
We started the overnight <I>E.coli</I> culture with our constructs controlling fluorescent ORFs expression in LB medium with the inducer- arabinose (final conc. 0.4%). The next day we centrifuged the cultures, collected pellets and started the process of protein purification. 
After protein purification with His-tag purification resin and buffers containing high imidazole concentrations, we checked the results using SDS-PAGE.</P><SPAN><B>3<SUP>rd</SUP> week</B></SPAN><P align="justify">This week we tried to assemble Rha1-E1 and Mel2-E1 one more time, as our previous constructs turned out to have mutations within their sequences. We amplified required fragments and performed CPEC reaction. After bacteria transformation, we picked some colonies and transferred them onto medium with inducer to check which colonies really took up plasmid- i.e. produce sfGFP and shine in green under UV light. We picked shining colonies, isolated plasmids out of bacteria and sent plasmid samples for sequencing. </P><SPAN><B>4<SUP>th</SUP> week</B></SPAN><P align="justify">Sequencing results proved that Rha1-E1 was properly assembled (luckily!), but we still have some problems with Mel2-E1 construct- sequencing showed that obtained sequence didn’t match the designed one.
We performed promoters’ activity measurements with E1 constructs, theirs wild-type versions and T7 promoter. We wanted to evaluate, whether adding E1 fragments improves promoter activity - and the results show that it most certainly does! Also Rha1-E1 happens to be almost 3-fold stronger than T7!</P><P align="justify">Flask on the left contains <I>E.coli</I> BL21-DE3 cells with T7-sfGFP and flask on the right <I>E.coli</I> DH5aplha sfGFP under Rha1-E1 promoter, which turned out to be the stronger one. Bacteria were cultured in LB medium for 6h with 0.4% lactose or rhamnose, respectively.</P><P align="justify">This picture shows comparison between promoters with E1_5’UTR and theirs wild-type versions. In all cases E1 modification increases promoters activity.</P><H2 id="october">OCTOBER</H2><SPAN><B>1<SUP>st</SUP> week</B></SPAN><P align="justify">It's time to start working on banner of our UAM_Poznan team. Many possibilities are being tested! Also it's about time to order T-shirts with our logo and to start putting content on our iGEM wiki.</P><SPAN><B>2<SUP>nd</SUP> week</B></SPAN><P align="justify">This week is super busy! We are finishing experiments, preparing lots of texts for our wiki page and uploading it all on iGEM servers. We are also working on our poster and presentation.</P><SPAN><B>3<SUP>rd</SUP> week</B></SPAN><P align="justify">Let’s pack our bags:) Boston here we come!</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>