<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UAM_Poznan_Biobrick_testing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UAM Poznan/Biobrick testing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="background"><DIV id="page"><DIV id="header"><DIV id="navigation"><UL><LI id="menu_home"><A href="https://2016.igem.org/Team:UAM_Poznan">Home</A></LI><LI id="menu_project"><A href="#" class="noclick">PROJECT</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Description">MAIN IDEA</A><A href="https://2016.igem.org/Team:UAM_Poznan/Demonstrate">APPLICATIONS</A></SPAN></LI><LI id="menu_team"><A href="https://2016.igem.org/Team:UAM_Poznan/Team">TEAM</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Attributions">ATTRIBUTIONS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Collaborations">COLLABS</A></SPAN></LI><LI id="menu_results"><A href="#" class="noclick">RESULTS</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Results">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Results-ORF">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Achievements">ACHIEVEMENTS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Biobrick_testing" class="double-line">BIOBRICK TESTING</A></SPAN></LI><LI id="menu_parts"><A href="https://2016.igem.org/Team:UAM_Poznan/Basic_Part">PARTS</A></LI><LI id="menu_methods"><A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">METHODS</A></LI><LI id="menu_notebook"><A href="#" class="noclick">NOTEBOOK</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook-promoters">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Safety" class="double-line">SAFETY AND ETHICS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Patenting">PATENTING</A></SPAN></LI><LI id="menu_galleries"><A href="https://2016.igem.org/Team:UAM_Poznan/Galleries">GALLERIES</A></LI><LI id="menu_hp"><A href="https://2016.igem.org/Team:UAM_Poznan/HP/Silver" id="hp_menu" class="double-line">HUMAN PRACTICES</A></LI></UL></DIV></DIV><TITLE>UAM_Poznan Team</TITLE><DIV id="contents"><DIV class="box"><DIV><DIV class="body"><H1><CENTER>BIOBRICK TESTING</CENTER></H1><P align="justify">We have decided to test several existing BioBrick Parts created by other iGEM teams, which were included in The 2016 DNA Distribution Kit. At first we have chosen 4 parts: L-rhamnose-inducible promoter <A href="http://parts.igem.org/Part:BBa_K914003">(BBa_K914003)</A>, amilCP blue chromoprotein <A href="http://parts.igem.org/Part:BBa_K592009">(BBa_K592009)</A>, Blue Fluorescent Protein (mTagBFP) <A href="http://parts.igem.org/Part:BBa_K592100">(BBa_K592100)</A> and Apple fragrance generator <A href="http://parts.igem.org/Part:BBa_K395602">(BBa_K395602)</A>. BFP and amilCP could have been compared with our sfGFP whereas the first chosen BioBrick would make a nice comparison to our rhamnose-inducible promoters.  We have chosen Apple fragrance generator just out of curiosity if <I>E. coli</I> could really smell like an apple.</P><P align="justify">We have transformed <I>E. coli</I> DH5α cells with provided constructs and performed plasmid isolation. Plasmids have been sequenced using Sanger method in the lab at our Faculty. Unfortunately obtained sequences were nowhere similar to sequences from iGEM Registry of Standard Biological Parts. BLAST analysis revealed partial similarity to different cloning vectors. Surprisingly, partial similarity to Rhizobium spp. genomes had also been found. 
We have then decided to perform PCR using VF and VR primers (they are designed for sequencing or amplifying the insert in pSB1C3 plasmid ) to check the size of inserts. We were expecting them to have lengths from 122 to 1813 bp. Picture below indicates that all inserts are about 1100 – 1400. 
 </P><P align="justify">After that we have decided to check another 4 Parts: enhanced yellow fluorescent protein <A href="http://parts.igem.org/Part:BBa_E0032">(BBa_E0032)</A>, Right facing promoter (for xylF) controlled by xylR and CRP-cAMP <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_I741018">(BBa_I741018)</A>, monomeric RFP <A href="http://parts.igem.org/Part:BBa_J06505:Design">(BBa_J06505)</A> and Orange Fluorescent Protein (OFP) <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K156009">(BBa_K156009)</A>. </P><P align="justify">However, this same obtained sequences have demonstrated no similarity to reference sequences as well. BLAST analysis revealed partial similarity to cloning and expressing vectors or even to thymidine kinase or lox66. Picture below shows colony PCR using all 8 tested inserts amplified with VF and VR primers. Only few bands had been obtained (for YFP and Apple fragrance generator) but none was at the appropriate height. </P><P align="justify">Gel electrophoresis revealed that neither of tested BioBrick Parts gave expected results and unfortunately none of them seem to be working.
Pictures below show chromatograms which prove that obtained DNA sequences have reliable quality. 
Comparison of obtained sequences to provided reference sequences resulted in insufficient similarity for further analysis.</P><P align="justify">Finally, we have decided to check another BioBrick, but this time we used amilCP BioBrick <A href="http://parts.igem.org/Part:BBa_K592009">(BBa_K592009)</A> from The 2013 DNA Distribution Kit. We have designed overlaps for amilCP and tag SUMO and decided to put it into pSB1C3 plasmid under arashort promoter. To assemble our construct, we carried out two PCRs – one to overlaps to the amilCP and SUMO-tag and a second to amplify a plasmid with arashort1 promoter. Then we performed CPEC reaction to put everything together into one piece. In the next step we transformed electrocompetent <I>E.coli</I> cells with obtained construct and after that we performed plasmid and protein isolation. Pictures below shows comparison of obtained sequence to reference sequence as well as chromatogram of obtained sequence.</P><P align="justify">After that, we induced protein expression in <I>E. coli</I> DH5α cells in overnight culture in liquid LB medium with 0,4% arabinose. We extracted our protein using urea, Tris, Triton and lysozyme containing buffer and purified it with cOmplete™ His-Tag Purification Resin by Roche.
Joining the SUMO-tag to amilCP gave us the opportunity to work with proteins where is not possible to introduce His-Tag. We have another construct with protease which cuts the Tag. Matching conditions will let us to work with proteins that are not easy to purify by simple color-testing (e.g. GFP or amilCP). But still it’s few things to do in this case! 
Pictures below show steps from solid bacteria culture to purified amilCP protein.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>