<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UAM_Poznan_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UAM Poznan/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="background"><DIV id="page"><DIV id="header"><DIV id="navigation"><UL><LI id="menu_home"><A href="https://2016.igem.org/Team:UAM_Poznan">Home</A></LI><LI id="menu_project"><A href="#" class="noclick">PROJECT</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Description">MAIN IDEA</A><A href="https://2016.igem.org/Team:UAM_Poznan/Demonstrate">APPLICATIONS</A></SPAN></LI><LI id="menu_team"><A href="https://2016.igem.org/Team:UAM_Poznan/Team">TEAM</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Attributions">ATTRIBUTIONS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Collaborations">COLLABS</A></SPAN></LI><LI id="menu_results"><A href="#" class="noclick">RESULTS</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Results">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Results-ORF">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Achievements">ACHIEVEMENTS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Biobrick_testing" class="double-line">BIOBRICK TESTING</A></SPAN></LI><LI id="menu_parts"><A href="https://2016.igem.org/Team:UAM_Poznan/Basic_Part">PARTS</A></LI><LI id="menu_methods"><A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">METHODS</A></LI><LI id="menu_notebook"><A href="#" class="noclick">NOTEBOOK</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook-promoters">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Safety" class="double-line">SAFETY AND ETHICS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Patenting">PATENTING</A></SPAN></LI><LI id="menu_galleries"><A href="https://2016.igem.org/Team:UAM_Poznan/Galleries">GALLERIES</A></LI><LI id="menu_hp"><A href="https://2016.igem.org/Team:UAM_Poznan/HP/Silver" id="hp_menu" class="double-line">HUMAN PRACTICES</A></LI></UL></DIV></DIV><P><TITLE>UAM_Poznan Team</TITLE></P><DIV id="contents"><DIV class="box"><DIV><DIV class="body"><H1><CENTER>RESULTS - PROMOTERS</CENTER></H1><P align="justify">Since last year we’ve been working on our expression system for <I>Escherichia coli</I> – a set of promoters induced by four different sugars: arabinose, rhamnose, xylose and melibiose. </P><P align="justify">E.co-Factory ensures independent induction of expression of at least two different genes in one cell, because our promoters use different sugars as inducers, and efficient blockage of all promoters by glucose. We provide many promoter versions, which differ in their strength, so one can choose either stronger promoter, or the one ensuring lower levels of recombinant protein production. Our system uses <I>E. coli</I> RNA polymerase, which is better than T7 phage RNA polymerase because is slower and more accurate. <B>This year we have focused on 5’UTR modifications, trying to select promoter/5’UTR combinations which give the highest protein expression upon induction. 
These year’s constructs are as follows: </B><B><A href="http://parts.igem.org/Part:BBa_K2014000">BBa_K2014000</A></B> - pBAD-E15'UTR-&gt;sfGFP – arabinose induced promoter with E1 5’UTR<B><A href="http://parts.igem.org/Part:BBa_K2014001">BBa_K2014001</A></B> - prha1-E15'UTR-&gt;htsfGFP - rhamnose induced promoter with E1 5’UTR<B><A href="http://parts.igem.org/Part:BBa_K2014002">BBa_K2014002</A></B> - pxylS-E1_5'UTR-&gt;sfGFP – xylose induced promoter with E1 5’UTR<B><A href="http://parts.igem.org/Part:BBa_K2014003">BBa_K2014003</A></B> - pBAD-M5'UTR-&gt;sfGFP – improvement of pBAD promoter (<A href="http://parts.igem.org/Part:BBa_K1741000">BBa_K1741000</A>) with M5’UTR<B><A href="http://parts.igem.org/Part:BBa_K2014004">BBa_K2014004</A></B> - pxylS-M5'UTR-&gt;sfGFP - improvement of XylS promoter (<A href="http://parts.igem.org/Part:BBa_K1741009">BBa_K1741009</A>) with M5’UTR.
All constructs contain sfGFP under different promoters/UTRs- as a marker of gene expression and protein synthesis/accumulation.</P><H2><CENTER>E1-5’UTR</CENTER></H2><P align="justify">E1 5’UTR contains an additional ribosome binding site from gene 10 of bacteriophage T7. This ribosome binding site forms an additional base-paired interaction with <I>E.coli</I> 16S rRNA and strongly stimulates translation efficiency. First time it was shown by Olins in 1987, then in 1992, tested on tac promoter by Lehmeier. We show that at least three non-phage promoters can be successively fused to a 39nt long part of this UTR, containing an additional 16S rRNA binding site, and in all cases we observe strong increase of protein expression. This mechanism seems to be universal and no factor of phage origin, except a short mRNA sequence, is necessary to enhance protein biosynthesis, most likely due to polysome takeover and more effective translation of mRNAs, which can better compete for initiation of translation. We fused this UTR to tightly controlled promoters to make new, better expression systems, which tightly controlled upon induction, will very effectively produce recombinant proteins in <I>E. coli</I>.<B>pBAD-E15'UTR-&gt;sfGFP (Ara1-E1, <A href="http://parts.igem.org/Part:BBa_K2014000">BBa_K2014000</A>)</B>
pBAD-E1_5'UTR is a fusion of pBAD (Arashort1, <A href="http://parts.igem.org/Part:BBa_K1741000">BBa_K1741000</A>), <I>Escherichia coli</I> K-12 arabinose promoter, with the E1_5’UTR.  This short promoter does not contain AraC ORF.<B>prha1-E15'UTR-&gt;htsfGFP (Rha1-E1,<A href="http://parts.igem.org/Part:BBa_K2014001">BBa_K2014001</A>)</B>
prha1-E1_5'UTR is a fusion of rha1 (<A href="http://parts.igem.org/Part:BBa_K1741005">BBa_K1741005</A>), <I>Escherichia coli</I> K-12 rhamnose promoter, and E1_5’UTR .<B>pxylS-E1_5'UTR-&gt;sfGFP (XylS-E1,<A href="http://parts.igem.org/Part:BBa_K2014002">BBa_K2014002</A>)</B>
pxylS-E1_5'UTR is a fusion of XylA part of <I>Escherichia coli</I> K-12 double sided xylose promoter, with E1_5’UTR.<B>All above promoters fused with 39 nt E1_5’UTRs have strongly improved activity</B> - at least 2-3-fold, when compared to their wild-type versions (Fig.2, 3). Among all promoters with E1_5’UTR, prha1-E15'UTR (Rha1-E1) is the strongest one and ensures the highest production of recombinant proteins. 
Furthermore, fusion promoters’ tightness is not affected (Fig. 5). We performed a test in which <I>E.coli</I> cells transformed with biobricks containing ara, rha and xyl promoters with E1_UTR and T7 promoter were cultured overnight in LB medium without inducer; there is a clear leakage in T7 promoter, whilst our new much stronger promoters appear to be still tightly regulated. </P><P align="justify"><B>Fig. 2. sfGFP fluorescence intensity comparison between constructs containing E1_5’UTR, their wild-type versions and T7-sfGFP.</B><I>E.coli</I> cells [DH5α transformed with E1_5’UTR fused promoter constructs (Arashort1-E1, Rha1-E1, Xyl-E1)  and their wild-types: AraWT, RhaWT, XylWT; <I>E. coli</I> BL21-DE3 transformed with  a construct with sfGFP under T7 promoter]  were cultured for 6h in LB medium supplemented with 0.4%  appropriate inducer: lactose; arabinose; rhamnose; xylose. <B>prha1-E15'UTR is the strongest among all of our promoters.</B>
Fusion of E1_5’UTR with Rha1 promoter improved Rha1’s strength- the amount of produced proteins increased about 15-times in comparison to wild-type version and about <U>3-times in comparison to strong T7 promoter (Fig. 2, 3, 4).</U>  Furthermore, introduced modification didn’t change promoter’s tightness. (Fig. 5)</P><P align="justify"><B>Fig. 3. Among all promoters with E1_5’UTR, prha1-E15'UTR (Rha1-E1) is the strongest one.</B> prha1-E15'UTR is also stronger than T7 promoter.  The picture shows comparison between T7 promoter and our constructs : promoters fused with E1_5’UTR (Arashort1-E1, Rha1-E1, Xyl-E1) and their wild-types: AraWT, RhaWT, XylWT. <I>E.coli</I> cells (DHα transformed with E1_5’UTR constructs and wild-types; BL21-DE3 transformed with sfGFP-T7) were cultured for 6h in LB medium supplemented with 0.4%  lactose; arabinose; rhamnose or xylose.</P><P align="justify"><B>Fig. 4. Fluorescence of prha1-E15’UTR (Rha1-E1) and sfGFP-T7.</B><I>E.coli</I> BL21-DE3 cells culture, transformed with T7-sfGFP, was grown in LB medium supplemented with 0.4% lactose and <I>E.coli</I> DH5α culture transformed with Rha1-E1 (<A href="http://parts.igem.org/Part:BBa_K2014001">BBa_K2014001</A>) was grown in LB medium supplemented with 0.4% rhamnose. Both constructs were assembled in the same pSB1C3 vector, cultures were grown in the same medium containing chloramphenicol (75 µg/mL) and the plasmid copy number per cell should be identical or at least similar. </P><P align="justify"><B>Fig. 5. Promoters’ tightness.</B> Fluorescence intensity of sfGFP produced in <I>E.coli</I> DH5α transformed with E1_5’UTR constructs and T7-sfGFP, grown overnight without inducers. </P><P align="justify"><B>Under Construction</B>
We are currently working on the assembly of pmel2-E1_5’UTR-&gt;sfGFP construct. It is a melibiose promoter controlling sfGFP expression, based on Mel2 (<A href="http://parts.igem.org/Part:BBa_K1741004">BBa_K1741004</A>), with E1_5’UTR .</P><H2><CENTER>Biobricks’ improvements</CENTER></H2><P align="justify">According to the criteria, that each group has to fulfill in order to get gold medal, we improved two biobricks, which we delivered, to the iGEM Registry in 2015. We improved one arabinose induced promoter- Arashort1 (araBAD;<A href="http://parts.igem.org/Part:BBa_K1741000">BBa_K1741000</A>), and one xylose induced promoter- XylS (xylA-proD5'UTR; <A href="http://parts.igem.org/Part:BBa_K1741009">BBa_K1741009</A>). 

Because there are many elements in promoter, UTR and cds sequences that influence protein expression in <I>E.coli</I> we had a tough nut to crack, to decide which of them we should take under consideration. It is well known, that the secondary structures in untranslated regions (UTRs) influence translation efficiency. Also the strength of mRNA binding to the ribosome i.e. the 3’-end of 16S rRNA with RBS (Ribosome Binding Site) is very important. In an effort to increase gene expression, several attempts have been made to optimize the complementarity between the Shine- Dalgarno (SD) sequences on the mRNA and the 3’ end of the 16S rRNA. But let’s go further, even the distance between the SD sequence and the translation initiation codon can affect expression levels. During this iGEM edition we have focused on 5’UTR modifications, trying to select those versions of promoter/5’UTR combinations which give the highest protein expression upon induction.  The results which we obtained turned up to be quite promising.<B>These year’s improved constructs are as follows: </B><A href="http://parts.igem.org/Part:BBa_K2014003"><B>BBa_K2014003</B></A> - pBAD-M5'UTR-&gt;sfGFP – improvement of pBAD promoter (<A href="http://parts.igem.org/Part:BBa_K1741000">BBa_K1741000</A>) with M5’UTR,<A href="http://parts.igem.org/Part:BBa_K2014004"><B>BBa_K2014004</B></A> - pxylS-M5'UTR-&gt;sfGFP - improvement of XylS promoter (<A href="http://parts.igem.org/Part:BBa_K1741009">BBa_K1741009</A>) with M5’UTR.
All constructs contain sfGFP under different promoters/UTRs- as a marker of gene expression and protein synthesis/accumulation.<B>pBAD-M5’UTR-&gt;sfGFP (<A href="http://parts.igem.org/Part:BBa_K2014003">BBa_K2014003</A>)</B>
Arashort1 is an arabinose induced promoter, from which, during last iGEM edition, we removed AraC ORF, present in the original <I>E. coli</I> AraC-pBAD promoter (<A href="http://parts.igem.org/Part:BBa_K1481002">BBa_K1481002</A>) and in commercially available pBAD vectors. Since we already knew that our unstructured synthetic M5’UTR, with well positioned, strong RBS enhances the strength of the melibiose induced promoter derived from <I>E. coli</I> genome - Mel2 (<A href="http://parts.igem.org/Part:BBa_K1741004">BBa_K1741004</A>) - we decided for its transplantation to Arashort1. By using specially designed primers, we have substituted the original 5’UTR from Arashort1 with the synthetic M5’UTR. </P><P align="justify"><B>Fig. 6. Synthetic evolution of arabinose induced promoter from <I>Escherichia coli</I>.</B> For more details please go to proper biobrick: AraWT (<A href="http://parts.igem.org/Part:BBa_K1481002">BBa_K1481002</A>), Arashort1 (<A href="http://parts.igem.org/Part:BBa_K1741000">BBa_K1741000</A>) and Ara1-UTR (<A href="http://parts.igem.org/Part:BBa_K2014003">BBa_K2014003</A>).<B>pxylS-M5'UTR-&gt;sfGFP (<A href="http://parts.igem.org/Part:BBa_K2014004">BBa_K2014004</A>)</B>
The promoter pxylS-M5'UTR (with a lab name: XylS-UTR ) is a modified xylose-induced promoter/5’UTR controlling sfGFP protein expression. To make it we exchanged the 5’UTR sequence of xylA-proD5'UTR in a biobrick (<A href="http://parts.igem.org/Part:BBa_K1741009">BBa_K1741009</A>) to a synthetic, unstructured M5’UTR derived from the Mel2 promoter (<A href="http://parts.igem.org/Part:BBa_K1741004">BBa_K1741004</A>).</P><P align="justify"><B>Fig. 7. Synthetic evolution of E. coli xylose induced promoters in our lab up to xylS-UTR.</B> For more details please go to a proper biobrick: XylWT (<A href="http://parts.igem.org/Part:BBa_K1741007">BBa_K1741007</A>), XylS (<A href="http://parts.igem.org/Part:BBa_K1741009">BBa_K1741009</A>) and XylS-UTR (<A href="http://parts.igem.org/Part:BBa_K2014004">BBa_K2014004</A>).</P><P align="justify"><B>Fig. 8. Efficiency of xylose and arabinose promoters of our collection based on the GFP fluorescence of a culture of <I>E. coli</I> DH5α transformed with:</B> Xyl-WT (<A href="http://parts.igem.org/Part:BBa_K1741007">BBa_K1741007</A>), XylA1 (<A href="http://parts.igem.org/Part:BBa_K1741008">BBa_K1741008</A>), XylS (<A href="http://parts.igem.org/Part:BBa_K1741009">BBa_K1741009</A>), XylS-UTR (<A href="http://parts.igem.org/Part:BBa_K2014004">BBa_K2014004</A>), AraWT (<A href="http://parts.igem.org/Part:BBa_K1481002">BBa_K1481002</A>), Arashort1 (<A href="http://parts.igem.org/Part:BBa_K1741000">BBa_K1741000</A>) or Ara1-UTR (<A href="http://parts.igem.org/Part:BBa_K2014003">BBa_K2014003</A>), grown in LB medium for 6h after induction with 0.4% xylose and 0.4% arabinose respectively.
The efficiency of the improved promoter Ara1-UTR is 2-3 fold higher, compared to its previous version- Arashort1, (<A href="http://parts.igem.org/Part:BBa_K1741000">BBa_K1741000</A>). The efficiency of the improved promoter pxylS-M5'UTR (<A href="http://parts.igem.org/Part:BBa_K2014004">BBa_K2014004</A>) is slightly higher, than all other versions of xylose responsive promoters we tested, and its induction seems to be faster (see 2h time-point). </P><P align="justify"><B>Fig. 9</B> The growth rate (OD<SUB>600</SUB>) of all constructs compared: Xyl-WT (<A href="http://parts.igem.org/Part:BBa_K1741007">BBa_K1741007</A>), XylA1 (<A href="http://parts.igem.org/Part:BBa_K1741008">BBa_K1741008</A>), XylS (<A href="http://parts.igem.org/Part:BBa_K1741009">BBa_K1741009</A>), XylS-UTR (<A href="http://parts.igem.org/Part:BBa_K2014004">BBa_K2014004</A>), AraWT (<A href="http://parts.igem.org/Part:BBa_K1481002">BBa_K1481002</A>), Arashort1 (<A href="http://parts.igem.org/Part:BBa_K1741000">BBa_K1741000</A>) or Ara1-UTR (<A href="http://parts.igem.org/Part:BBa_K2014003">BBa_K2014003</A>) is very similar. This comparison proves that promoters’ efficiency is close to its’ strength.<B>Legend</B>
AraC-araBAD– briefly called AraWT  (<A href="http://parts.igem.org/Part:BBa_K1741007">BBa_K1481002</A>)
araBAD briefly called Arashort1  (<A href="http://parts.igem.org/Part:BBa_K1741000">BBa_K1741000</A>)
pBAD-M5'UTR– briefly called Ara1-UTR  (<A href="http://parts.igem.org/Part:BBa_K2014003">BBa_K2014003</A>)
xylF-xylA - briefly called XylWT  (<A href="http://parts.igem.org/Part:BBa_K1741007">BBa_K1741007</A>)
xylF-xylA-proD5'UTR - briefly called XylA1   (<A href="http://parts.igem.org/Part:BBa_K1741008">BBa_K1741008</A>)
xylA-proD5'UTR - briefly called XylS  (<A href="http://parts.igem.org/Part:BBa_K1741009">BBa_K1741009</A>)
xylS-M5'UTR - briefly called XylS-UTR (<A href="http://parts.igem.org/Part:BBa_K2014004">BBa_K2014004</A>)<I>References:</I>
1.	Olins PO, Rangwala SH.;<I> A novel sequence element derived from bacteriophage T7 mRNA acts as an enhancer of translation of the lacZ gene in Escherichia coli.</I> J Biol Chem. 1989 Oct 15;264(29):16973-6.
2.	Davis J.H., Rubin A.J., Sauer R.T.;<I> Design, construction and characterization of a set of insulated bacterial promoters.</I> Nucleic Acids Research, 2011, Vol. 39, No. 3 1131–1141
3.	Haldimann A., Daniels L.L, Wanner B. L.;<I> Use of New Methods for Construction of Tightly Regulated Arabinose and Rhamnose Promoter Fusions in Studies of the Escherichia coli Phosphate Regulon.</I> Journal of Bacteriology, Mar. 1998, p. 1277–1286
4.	Holcroft C.C, Egan S.M. <I> Roles of Cyclic AMP Receptor Protein and the Carboxyl-Terminal Domain of the a Subunit in Transcription Activation of the Escherichia coli rhaBAD Operon.</I> Journal of Bacteriology, June 2000, p. 3529–3535
5.	Giacalone M.J. et.al.,<I> Toxic protein expression in Escherichia coli using a rhamnose-based tightly regulated and tunable promoter system.</I> BioTechniques 40:355-364 (March 2006)
6.	Song S., Park C.;<I> Organization and Regulation of the D-Xylose Operons in Escherichia coli K-12: XylR Acts as a Transcriptional Activator.</I> Journal of Bacteriology, Nov. 1997, p. 7025–7032
7.	Hans Peter Sorensen, Kim Kusk Mortensen.,<I> Advanced genetic strategies for recombinant protein expression in Escherichia coli.</I> Journal of Biotechnology, August 2004.
8.	Jason R. Newman, Clay Fuqua.,<I> Broad-host-range expression vectors that carry the L-arabinose-inducible Escherichia coli araBAD promoter and the araC regulator.</I> Gene, Nov. 1998.
9.	Lehmeier B, Amann E.<I> Tac promoter vectors incorporating the bacteriophage T7 gene 10 translational enhancer sequence for improved expression of cloned genes in Escherichia coli. </I>Journal of Biotechnology, Apr. 1992; 23(2):153-65.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>