<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UAM_Poznan_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UAM Poznan/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="background"><DIV id="page"><DIV id="header"><DIV id="navigation"><UL><LI id="menu_home"><A href="https://2016.igem.org/Team:UAM_Poznan">Home</A></LI><LI id="menu_project"><A href="#" class="noclick">PROJECT</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Description">MAIN IDEA</A><A href="https://2016.igem.org/Team:UAM_Poznan/Demonstrate">APPLICATIONS</A></SPAN></LI><LI id="menu_team"><A href="https://2016.igem.org/Team:UAM_Poznan/Team">TEAM</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Attributions">ATTRIBUTIONS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Collaborations">COLLABS</A></SPAN></LI><LI id="menu_results"><A href="#" class="noclick">RESULTS</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Results">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Results-ORF">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Achievements">ACHIEVEMENTS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Biobrick_testing" class="double-line">BIOBRICK TESTING</A></SPAN></LI><LI id="menu_parts"><A href="https://2016.igem.org/Team:UAM_Poznan/Basic_Part">PARTS</A></LI><LI id="menu_methods"><A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">METHODS</A></LI><LI id="menu_notebook"><A href="#" class="noclick">NOTEBOOK</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook-promoters">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Safety" class="double-line">SAFETY AND ETHICS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Patenting">PATENTING</A></SPAN></LI><LI id="menu_galleries"><A href="https://2016.igem.org/Team:UAM_Poznan/Galleries">GALLERIES</A></LI><LI id="menu_hp"><A href="https://2016.igem.org/Team:UAM_Poznan/HP/Silver" id="hp_menu" class="double-line">HUMAN PRACTICES</A></LI></UL></DIV></DIV><P><TITLE>UAM_Poznan Team</TITLE></P><DIV id="contents"><DIV class="box"><DIV><DIV id="news" class="body"><DIV class="sidebar"><H3>CALENDAR</H3><UL><LI><A href="#june">JUNE</A></LI><LI><A href="#july">JULY</A></LI><LI><A href="#august">AUGUST</A></LI><LI><A href="#september">SEPTEMBER</A></LI><LI><A href="#october">OCTOBER</A></LI></UL></DIV><DIV><DIV><DIV><H1><CENTER>NOTEBOOK - ORFs</CENTER></H1><H2 id="june">JUNE</H2><SPAN><B>1<SUP>st</SUP> week</B></SPAN><P align="justify">Our iGEM project started at the beginning of June. Our plan is to check how codon content of coding sequence influences the protein production. According to bioinformatics analysis of codons in <I>E.coli</I> genome we obtained a list of the most frequent and the rarest codons in <I>Escherichia coli</I>. This week we spent on computer to engineer two different variants (Best and Worst) of green fluorescent protein (GFP). They differ within all codons with the exception of first ten residues that correspond to a stable N-end- a 6-histidine tag. Those two variants of sfGFP and primers with overlaps were ordered in IDT at the end of the week- now we are waiting…</P><SPAN><B>3<SUP>rd</SUP> week</B></SPAN><P align="justify">Our exams at the university have finished and we can concentrate on iGEM project! All the IDT fragments and geneblocks came to our lab! We performed PCR to amplify two versions of sfGFP: best and worst. Two versions of sfGFP were amplified using primers with overlaps designed in NEBuilder. Those overlaps are necessary to CPEC (Circular polymerase extension cloning) reaction <A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">(See: Methods)</A>. We also amplified arabinose promoter (AraC-pBAD) <A href="http://parts.igem.org/Part:BBa_K1481002">(BBa_K1481002)</A> and visualized the PCR products under UV light after electrophoresis on 1% agarose gel.</P><P align="justify">We cut out the bands that represented the desired products and purified them on silica columns. We also amplified vector pSB1C3. When we obtained all fragments needed to assembly our constructs, we performed CPEC reaction to obtain two constructs.</P><SPAN><B>4<SUP>th</SUP> week</B></SPAN><P align="justify">This week we started from desalting the CPEC products on silica columns. The same day we also introduced the constructs to <I>E.coli</I> DH5α (from New England Biolabs) by electroporation. To check the intensity of two GFP versions we transferred a few randomly picked colonies on Petri dishes with 0.4% of arabinose and chloramphenicol to induce GFP expression. Below you can see the photo made under the Blue light.</P><H2 id="july">JULY</H2><SPAN><B>1<SUP>st</SUP> week</B></SPAN><P align="justify">To verify the sequence of our arabinose promoter and GFP_Best/Worst we isolated plasmids and sent them to sequencing laboratory. 
This week there was also a brainstorming meeting. Conversations over coffee and homemade cookies resulted in new ideas – we decided to order two versions of another protein (red fluorescent protein) that differs in structure from GFP. We got to work immediately and designed two versions of RFP according to the bioinformatics analysis of <I>E.coli</I> genome. We also designed primers with overlaps required for the CPEC reaction. Next day the order to IDT was already sent!</P><SPAN><B>2<SUP>nd</SUP> week</B></SPAN><P align="justify">This week started with good news- sequencing confirmed the sequences of genetic constructs of GFP_best and GFP_worst under arabinose promoter!</P><P align="justify">We introduced the isolated plasmids to <I>E.coli</I> DH5α by electroporation. Electroporation was very efficient and there were a lot of colonies on our Petri dishes! </P><P align="justify">We took one colony from each dish and inoculated 1xLB medium for overnight harvesting. Next day we centrifuged the bacteria and dissolved the pellet in 1xLB, chloramphenicol and glycerol and froze them immediately in liquid nitrogen. Glycerol stocks will be stored in -80°C.  </P><SPAN><B>3<SUP>rd</SUP> week</B></SPAN><P align="justify">If we have glycerol stocks we can start measurements. We used the cultures from the previous day to inoculate 1xLB and M9 minimal medium with 0.4% arabinose. After inoculation we collected samples from all the cultures every hour, 6 timepoints (See: Methods). Each harvest we mixed the bacterial culture sample and treated with extraction buffer containing Tris HCl, EDTA, SDS and urea then we incubated them in 60°C for 5 minutes. Next day we measured the fluorescence of the samples on Tecan <A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">(See: Methods)</A> to check the difference in protein production between two versions of GFP. The scheme of the procedure is shown below.</P><P align="justify">We were very surprised by the results- they showed that a simple translational circuit can easily adjust and produce GFP protein in richer medium irrespectively of codon content! We came up with an idea to check the intensity of GFP production in SB/PKB medium <A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">(See: Materials)</A>, which is a richer medium than 1xLB. We can’t wait till our further experiments!</P><SPAN><B>4<SUP>th</SUP> week</B></SPAN><P align="justify">It was a “planning week”- we were wondering, which constructs should be assembled. We found it interesting to check the difference between fluorescent protein production under promoters induced by another sugar. We chose two xylose promoters. The same day primers with overlaps were designed and ordered from IDT. 
At the end of the week geneblocks of RFP_Best and RFP_Worst came to our lab! We started from PCR to add overlaps that are essential for proper construct assembling in CPEC reaction. Then the products were separated on agarose gel to confirm the appropriate length. After cutting the bands from the gel and cleaning them on columns – we performed the CPEC reaction!</P><H2 id="august">AUGUST</H2><SPAN><B>1<SUP>st</SUP> week</B></SPAN><P align="justify">This week we repeated the experiment and compared three versions (sfGFP, sfGFP_Best, sfGFP_Worst) of green fluorescent proteins production on three different media: SB/PKB, 1xLB and M9 minimal medium. We prepared cultures (we started from OD<SUB>600</SUB>= 0.4) and add arabinose (inducer). We collected samples and measured the OD<SUB>600</SUB> each hour. Next day the fluorescence was measured on Tecan- and the results were similar to the previous ones. In the richest medium- SB/PKB there isn’t any change in fluorescence intensity between all sfGFP versions.
We also introduced two variants of RFP to <I>E.coli</I> DH5α by electroporation. We isolated plasmids and sent to sequencing laboratory to confirm the sequence. </P><SPAN><B>2<SUP>nd</SUP> week</B></SPAN><P align="justify">We have successfully obtained another two constructs: RFP_Best and RFP_Worst under the arabinose promoter (araBAD-araC) <A href="http://parts.igem.org/Part:BBa_K1481002">(BBa_K1481002)</A>. We introduced the confirmed plasmids to <I>E.coli</I> DH5α. Next day we took one colony from Petri dish and inoculated 1xLB medium for overnight. In the morning we started culturing the bacteria in 1xLB and M9 minimal media with 0.4% arabinose. RFP folds much slower than GFP that is why we didn’t collect samples every hour but just after 12h. We measured the fluorescence and we could observe differences in fluorescence between RFP_Best and RFP_Worst not only on M9 minimal medium but also on 1xLB. We checked the sequences in order to make sure that there weren’t any ribosomal binding sides or problematic secondary structures. We decided to create another two RFP variants- mixed RFPs. The first half of coding sequence is made of the most frequent codons and the second half is built from the rarest ones and vice versa (the scheme of the constructs is shown below). We created primers required for assembling this constructs. </P><SPAN><B>3<SUP>rd</SUP> week</B></SPAN><P align="justify">This week we received primers from IDT so we started from PCRs. We amplified GFP_Best and GFP_Worst by primers that add overlaps to different promoters: XylWT and XylE1. We cut out bands corresponded to GFP_Best and GFP_Worst (with appropriate overlaps) and purified them on columns. Then we performed CPEC reaction to assembly the constructs. Next day we started from purifying the CPEC products and electroporation of <I>E. coli</I> cells. The same week we also isolated plasmids and sent them to sequencing laboratory to confirm the sequence. </P><SPAN><B>4<SUP>th</SUP> week</B></SPAN><P align="justify">The results from sequencing laboratory confirmed sequences of GFP_Best and GFP_Worst under xylose-induced promoters (XylWT, XylE1)- we have another two constructs. We performed PCRs to amplify halves of RFP_Best and RFP_Worst with appropriate overlaps in order to create constructs built of mixed versions of RFP. We assembled the constructs by CPEC and introduced them to DH5α <I>E. coli</I> cells. Then we isolated plasmids from single colony and sent them to sequencing laboratory.</P><H2 id="september">SEPTEMBER</H2><SPAN><B>1<SUP>st</SUP> week</B></SPAN><P align="justify">In order to have glycerol stocks, which are necessary for further experiments we performed 17 electroporation with 17 different constructs!</P><P align="justify">It was a lot of work but hopefully everything went perfect and next day we obtained colonies on solid LB dishes. Next day we inoculated liquid LB media to prepare glycerol stocks. We also got ready to measurement week- we autoclaved tips and media: LB, SB/PKB and M9 salts.</P><SPAN><B>2<SUP>nd</SUP> week</B></SPAN><P align="justify">It was our measurement week! We compared the fluorescence of all variants of the green fluorescent protein (sfGFP, sfGFP_Best, sfGFP_Worst) under wild-type arabinose promoter (araBAD-araC) on 3 media: 1xLB, SB/PKB and M9 minimal medium.  To calculate a standard deviation next day we repeated the same experiment. At the end of the week we created charts according to fluorescence measurements on Tecan.</P><SPAN><B>3<SUP>rd</SUP> week</B></SPAN><P align="justify">We were continuing with measurements… Our instructor sent as a few interesting publications about the influence of sequences on 5’ or 3’ end of ORF on protein production and stability. Thus, we measured the fluorescence of GFP with three different tags. The constructs had been prepared earlier. On N-end of sfGFP was added methionine, leucine and two arginines (MLRR_sfGFP). Next constructs with introduced either 8 arginine residues (sfGFP_8R) or tagX (sfGFP_tagX) on C-end of sfGFP. The fluorescence was measured during 6h culture in three medias (1xLB, SB/PKB, M9 minimal medium) with 0.4% of arabinose.</P><SPAN><B>4<SUP>th</SUP> week</B></SPAN><P align="justify">This week we obtained the list of codon pair frequency rankings. Based on the special software- the Codon Pair Bias measure (created by our advisor- Melania Nowicka) she designed fluorescent proteins ORFs built of the most or least frequently codon contexts. In order to assemble constructs with those variants we designed primers with overlaps in NEBuilder. Both the geneblocks and the primers were ordered in IDT. </P><H2 id="october">OCTOBER</H2><SPAN><B>1<SUP>st</SUP> week</B></SPAN><P align="justify">To check the production of different versions of sfGFP under xylose promoters we inoculated 1xLB, SB/PKB and M9 medium with previously prepared stocks. Next day we prepared cultures (we started from OD<SUB>600</SUB>= 0.4) and add xylose to the final concentration 0.4%. The samples were collected every hour during 6h of culturing in the same way as already described <A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">(See Methods)</A>. The fluorescence was measured on Tecan. To check, whether the results are reproducible and calculate a standard deviation we repeated the experiment. </P><SPAN><B>2<SUP>nd</SUP> week</B></SPAN><P align="justify">We received the last delivery from IDT- geneblocks and primers required for assembling the constructs based on most or least abundant codon contexts. We started from PCRs to amplify the geneblocks with appropriate overlaps. We cut out the bands from the gel. We also performed the PCR to amplify the arabinose promoter and the vector (pSB1C3). The story will be continued…  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>