<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UAM_Poznan_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UAM Poznan/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="background"><DIV id="page"><DIV id="header"><DIV id="navigation"><UL><LI id="menu_home"><A href="https://2016.igem.org/Team:UAM_Poznan">Home</A></LI><LI id="menu_project"><A href="#" class="noclick">PROJECT</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Description">MAIN IDEA</A><A href="https://2016.igem.org/Team:UAM_Poznan/Demonstrate">APPLICATIONS</A></SPAN></LI><LI id="menu_team"><A href="https://2016.igem.org/Team:UAM_Poznan/Team">TEAM</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Attributions">ATTRIBUTIONS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Collaborations">COLLABS</A></SPAN></LI><LI id="menu_results"><A href="#" class="noclick">RESULTS</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Results">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Results-ORF">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Achievements">ACHIEVEMENTS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Biobrick_testing" class="double-line">BIOBRICK TESTING</A></SPAN></LI><LI id="menu_parts"><A href="https://2016.igem.org/Team:UAM_Poznan/Basic_Part">PARTS</A></LI><LI id="menu_methods"><A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">METHODS</A></LI><LI id="menu_notebook"><A href="#" class="noclick">NOTEBOOK</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook-promoters">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Safety" class="double-line">SAFETY AND ETHICS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Patenting">PATENTING</A></SPAN></LI><LI id="menu_galleries"><A href="https://2016.igem.org/Team:UAM_Poznan/Galleries">GALLERIES</A></LI><LI id="menu_hp"><A href="https://2016.igem.org/Team:UAM_Poznan/HP/Silver" id="hp_menu" class="double-line">HUMAN PRACTICES</A></LI></UL></DIV></DIV><TITLE>UAM_Poznan Team</TITLE><DIV id="contents"><DIV class="box"><DIV><DIV class="body"><H1><CENTER>METHODS</CENTER></H1><CENTER><H2>MEDIA PREPARATION AND BACTERIAL CULTURES </H2></CENTER><DIV class="clickable" onclick="toggle_visibility('content_1')"><FONT size="4">LB medium (Lysogeny-Broth)</FONT></DIV><DIV id="content_1" style="display: none"><P align="justify">
Tryptone        10g/L
NaCl            10g/L
Yeast extract   5g/L

Dissolve the components in 800ml of sterile deionized water, adjust the volume to 1 liter and sterilize by autoclaving. Solid media contain 1,4-1,5% agar and should be well mixed after autoclaving, then cooled to about 55°C  before the appropriate antibiotic is added. Then medium should be mixed well again and poured on plates as about 3mm layer without bubbles, under clean bench conditions.
 </P></DIV><DIV class="clickable" onclick="toggle_visibility('content_2')"><FONT size="4">2xLB medium (2xLysogeny-Broth)</FONT></DIV><DIV id="content_2" style="display: none"><P align="justify">
Tryptone	20g/L
NaCl		10g/L
Yeast extract	10g/L

Dissolve the components in 800ml of sterile deionized water, adjust the volume to 1 liter and sterilize by autoclaving.
Solid media contain 1,4-1,5% agar and should be well mixed after autoclaving, then cooled to about 55°C before the appropriate antibiotic is added. Then medium should be mixed well again and poured on plates as about 3mm layer without bubbles, under clean bench conditions.
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_3')"><FONT size="4">SB-PKB medium (Super Broth-PKB)</FONT></DIV><DIV id="content_3" style="display: none"><P align="justify"><U>20xPKB buffer/100ml</U>
KH<SUB>2</SUB>PO<SUB>4</SUB>            4.6g
K<SUB>2</SUB>HPO<SUB>4</SUB>            24.3g
Weigh the components and dissolve in 100ml of water, filter and sterilize by autoclaving. <U>SB medium/L</U>
Tryptone	12g
Yeast extract	24g
Glycerol 	4ml
Dissolve in 950ml of water, sterilize by autoclaving. Prior to use, add 50ml of 20xPKB buffer.
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_4')"><FONT size="4">SOC medium</FONT></DIV><DIV id="content_4" style="display: none"><P align="justify">
Tryptone       2%
Yeast extract  0.5%
NaCl	       0.05%
KCl	       2.5mM
MgCl<SUB>2</SUB>	       10mM
Glucose        20mM
Weigh the components and dissolve in water, add KCl, MgCl<SUB>2</SUB> and sterilize by autoclaving. At the end add filter sterilized glucose (pore size 0.22µm). 
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_5')"><FONT size="4">5xM9 salts</FONT></DIV><DIV id="content_5" style="display: none"><P align="justify">
Na<SUB>2</SUB>HPO<SUB>4   </SUB>      64g/L
KH<SUB>2</SUB>PO<SUB>4</SUB>          15g/L
NaCl            2.5g/L
NH<SUB>4</SUB>Cl           5g/L
To 800ml of H<SUB>2</SUB>O add weighed components and stir until dissolved. Adjust to 1L with distilled water and sterilize by autoclaving. 
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_6')"><FONT size="4">M9 medium</FONT></DIV><DIV id="content_6" style="display: none"><P align="justify">
Measure 770 ml of distilled sterile water and add 200ml of 5xM9 salts. Then add 2ml of 1M MgSO<SUB>4</SUB>, 20ml of 20% carbon source (e.g. glucose), 100µl of 1M CaCl<SUB>2</SUB>, 10ml of 1% thiamine and adjust to 1L with distilled water. 
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_7')"><FONT size="4">Plates with M9 medium</FONT></DIV><DIV id="content_7" style="display: none"><P align="justify">
Recipe for 200ml of M9 medium~10 plates.
1.	Melt 100ml of 3% agar in the microwave and transfer to a water bath 50°C.
2.	Add 53ml of sterile water preheated to 50°C.
3.	Add 40ml of 5xM9 salts, kept at the same temp.
4.	Add 20μl of warm 1M CaCl<SUB>2</SUB> and mix quickly.
5.	Add 0.4ml of warm 1M MgSO<SUB>4</SUB> and mix well.
6.	Add 4ml of 20% filter sterilized solution of glucose or another carbon source ( final conc. 0.4%).
7.	Add 2ml of 1% thiamine (filter sterilized), mix well, keep at 50°C
8.	Add the appropriate antibiotic, mix well and pour the medium on plates.</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_8')"><FONT size="4">Preparation of electro-competent <I>Escherichia coli</I> cells</FONT></DIV><DIV id="content_8" style="display: none"><P align="justify">
1.	Grow overnight a single colony of <I>E. coli</I> DH5α in 250ml flask with 100ml of LB medium (120 rpm, at 30-37°C).
2.	Transfer 0.5ml of the bacterial culture to a new 250ml flask with 100ml of LB medium and grow the culture at 37°C to get OD<SUB>600</SUB>=0.5 or 0.6.
3.	After 10-15 minutes of incubation on ice, centrifuge the culture at 4000 rpm for 20 min. at 4°C.
4.	Suspend bacterial pellet in 100ml of deionized water (4°C) and centrifuge as above. Repeat this step one more time.
5.	Suspend the cells in 40ml of cold deionized water with 1% of glycerol and centrifuge as before. (Keep tubes on ice!)
6.	Suspend the pellet in 1-2ml of 10% glycerol, split the suspension into 50-100μl portions and freeze in liquid nitrogen.
7.	Store at -80°C for up to 1 year.
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_9')"><FONT size="4">Transformation of <I>Escherichia coli</I> via electroporation</FONT></DIV><DIV id="content_9" style="display: none"><P align="justify">
1.	Incubate electro-competent cells (50-100µl) on ice for 5-10 min. and let it thaw slowly.
2.	Keep cells on ice and add 2-4μl of plasmid DNA (10-50ng) or 7μl of purified DNA template from CPEC to obtain 100-500 colonies.
3.	Transfer the mixture to the cold electroporation cuvette 0.2mm. Make sure that electrodes of cuvette are dry.
4.	Perform the electroporation at 2.5 kV. Pulse time should be close to 5ms.*
5.	Immediately add 800μl of LB medium/SOC medium, slowly mix (pipet up and down) and transfer to a new tube.
6.	Incubate at 37°C for 30-45 min with shaking 300-350 rpm.
7.	Centrifuge the tube (3 min, 5000rpm) and suspend the pellet in 50-100µl of LB medium.
8.	Spread the bacterial culture onto the 90mm plate with solid LB medium with the proper antibiotic.
9.	Incubate overnight at 37°C.
* Remember to provide the proper density of competent cells. Too high density can cause that cuvette can explode due to an electric arc!
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_10')"><FONT size="4"><I>E.coli</I> glycerol stocks</FONT></DIV><DIV id="content_10" style="display: none"><P align="justify">
Bacterial culture       25ml
50% glycerol		40µl
LB medium               200µl

1.	Grow a single colony of <I>E. coli</I> DH5α, after transformation, into a 100ml flask with 25ml of LB medium overnight, at 120 rpm, at 37°C.
2.	Centrifuge the bacterial culture 5 min at 5000 rpm and suspend the pellet in solution of glycerol and LB medium.
3.	 Split the suspension into 15µl portions to eppendorf tubes and freeze in liquid nitrogen.
4.	 Store at -80°C. </P></DIV><CENTER><H2>CONSTRUCT DESIGN AND CLONING METHODS</H2></CENTER><DIV class="clickable" onclick="toggle_visibility('content_11')"><FONT size="4">DNA amplification</FONT></DIV><DIV id="content_11" style="display: none"><P align="justify">
Primers for PCR were designed using Primer 3 [1,2] to have the same calculated melting temperature of 60°C. This way fragments of similar sizes can be amplified at the same time on one thermal cycler.<U>Phusion PCR- standard PCR mix </U>
In a sterile PCR tube mix reagents in the following order:

5xHF buffer (NEB)                        10µl
H<SUB>2</SUB>0                                      up to 50 µl
dNTP mix (10mM each)                     1 µl
Primer forward (100 µM)                  0.25µl
Primer reverse (100 µM)                  0.25µl
DNAtemplate (0.1-5ng)                    x µl
Phusion DNA polymerase (NEB)	         0.5µl<U>Standarized PCR conditions</U>Remember to put the probes to preheated thermocycler to avoid non-specific product amplification.

</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_12')"><FONT size="4">Electrophoresis of DNA fragments in agarose gels</FONT></DIV><DIV id="content_12" style="display: none"><P align="justify">
1.	Weigh required amount of agarose and pour into a 250 ml screw cap Simax bottle. 
2.	Add appropriate volume of 1xTAE Buffet and mix.
3.	Heat for up for 2 minutes in microwave oven until the agarose sol is homogeneous. Cool the bottle shortly under tap water. Add ethidium bromide (10mg/ml). Mix well.
4.	Pour the agarose sol into a gel casting tray. Insert the appropriate comb. If necessary, clear the bubbles with a pipette tip. Wait until the gel is fully solidified.
5.	Pour TAE buffer onto the gel, remove the comb and casting gates. Pour TAE buffer to buffer tanks to cover the gel with 1mm of the buffer.
6.	Mix the samples with 6xloading dye in a 5:1 ratio. Put the samples into the wells. We were using two DNA molecular weight markers: 100 bp DNA Ladder <A href="https://www.neb.com/products/n3231-100-bp-dna-ladder">(https://www.neb.com/products/n3231-100-bp-dna-ladder)</A> and 1 kb DNA Ladder <A href="https://www.neb.com/products/n3232-1-kb-dna-ladder">(https://www.neb.com/products/n3232-1-kb-dna-ladder)</A>.
7.	Close the lid and apply proper running conditions- 5V/cm.*
* Running time depends on the size of the gel and the expected resolution. 
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_13')"><FONT size="4">Gel extraction</FONT></DIV><DIV id="content_13" style="display: none"><P align="justify">
1.	After electrophoresis, photograph agarose gel on UV transiluminator and cut out the DNA fragments that you are interested in with a clean razor blade.
2.	Place agarose block in a weighed eppendorf tube.
3.	 Purify DNA fragments by using Zyppy silica columns according to the protocol provided by ZYMO RESEARCH <A href="https://www.zymoresearch.com/dna/dna-clean-up/gel-dna-recovery/zymoclean-gel-dna-recovery-kit">(https://www.zymoresearch.com/dna/dna-clean-up/gel-dna-recovery/zymoclean-gel-dna-recovery-kit)</A>.
4.	Determine the final concentration of eluted DNA by NanoDrop.
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_14')"><FONT size="4">Chromosomal DNA isolation from <I>Escherichia coli</I></FONT></DIV><DIV id="content_14" style="display: none"><P align="justify">
1.	Spin down 0.5 ml of <I>E. coli</I> culture OD<SUB>600</SUB>=0.6 for 3 min at 5000g and remove the supernatant.
2.	Suspend the pellet in 200μl of buffer A (150 mM NaCl,100 mM EDTA pH=8.0).
3.	Add 10 μl of fresh lysozyme suspension (10 mg/ml), incubate at 37°C for 15 min.
4.	Add 20 μl 10% SDS and 1μl proteinase K, mix well, incubate at 56°C for 15 min.
5.	Cool down the mixture to RT, add 100 μl chloroform and 100 μl phenol pH=8.0, vortex for 1 min. Centrifuge for 5 min at 12 000 rpm.
6.	Carefully transfer the upper water phase containing nucleic acids to a new tube, add equal volume of ethanol, mix well and centrifuge as above.
7.	Remove the supernatant and wash the pellet twice with 70% ethanol, spin down for 1 min at 12 000 rpm, remove the supernatant carefully to leave the pellet only. Dry the pellet at RT for 5-10 min.
8.	Add 50 μl H<SUB>2</SUB>O, incubate at 56°C for 2 min, mix well by pipetting.
9.	Check the quality of DNA on 0.8% agarose gel. The main DNA band should represent molecules longer than 20 kb.
10.	Use 0.1- 0.5μg of chromosomal DNA to amplify a gene of interest.
All promoter sequences we use derive from the chromosomal DNA of <I>E. Coli</I> K-12, DH5α.
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_15')"><FONT size="4">Plasmid isolation and purification</FONT></DIV><DIV id="content_15" style="display: none"><P align="justify"><U>TEG buffer/50ml</U>
1M glucose                      1.5ml
250mM Na<SUB>2</SUB>EDTA pH=8.0            6ml
1M Tris-HCl pH=8.0                      0.75ml
H<SUB>2</SUB>O                                    41.75ml<U>NaOH-SDS buffer/40ml</U>
10M NaOH                        0.8ml
10% SDS                         4ml
H<SUB>2</SUB>O                             35.2ml<U>KAc buffer/ 50ml</U>
Glacial acetic acid             7ml
5M potassium acetate            36ml
H<SUB>2</SUB>O                             7ml
1.	Transfer 25ml of bacterial culture grown overnight in LB medium at 37°C to a conical tube and centrifuge at 9 000 rpm at 4°C. Discard the supernatant. 
2.	Suspend bacterial pellet in 10ml of TEG buffer.
3.	Add 10ml of NaOH-SDS buffer and mix by inverting the tube. Leave the mixture for no longer than 2 min to obtain a clear lysate.
4.	Add 10ml of KAc buffer, vortex quickly and centrifuge at 9 000 – 12 000 rpm at 4°C.
5.	Carefully transfer the supernatant to a new tube.
6.	Add 0.6 – 0.7 V of isopropanol, mix and put into the fridge for minimum 30 min.
7.	Centrifuge at 9 000 - 12 000 rpm for 30 min. at 4°C.
8.	Discard the supernatant, suspend the pellet in 800μl of miliQ water, add RNase.
9.	Incubate at 37°C for 15 min or keep in the fridge for overnight.
10.	Just proceed to <B>Zippy Plasmid Miniprep Kit</B><A href="https://www.zymoresearch.com/dna/plasmid-dna-purification/zyppy-plasmid-miniprep-kit">(href=&quot;https://www.zymoresearch.com/dna/plasmid-dna-purification/zyppy-plasmid-miniprep-kit)</A> procedure starting from step 1. 
One silica column will bind up to 50μg of plasmid DNA at one binding session, so one can purify 200μg of high quality plasmid DNA on one mini-column. After checking the quality by agarose gel electrophoresis and A<SUB>260</SUB> measurement, the plasmid DNA is ready for sequencing and RE digestions, cloning, and transfections.
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_16')"><FONT size="4">Construct verification by colony PCR and Sanger DNA sequencing</FONT></DIV><DIV id="content_16" style="display: none"><P align="justify"><U>Colony PCR</U>
1.	Pick up several colonies of interest with a sterile toothpick and streak them on one or two new selective plates with numbered fields. Wipe the remaining part of the colony on the bottom of a 1.5 ml Eppendorf tube. Use the same numbering for tubes as for fields on plates.
2.	 Add 50-100µl of H<SUB>2</SUB>O to each tube, incubate at 96°C for 10 min. Cool down by centrifugation for 2 min at 12 000 rpm. Transfer 2µl to a new PCR tube.
3. Use Taq DNA polymerase for colony PCR. Prepare a standard PCR mix leaving 2µl of the total volume for the template. Consider an additional volume for a positive control, like just an empty plasmid.
4. Add 48µl of PCR mix to every tube. Prepare a positive control. Run a standard PCR.
5. Check the sizes of amplicons by gel electrophoresis in agarose gel.<U>Sanger sequencing</U>
Sanger sequencing of constructs was performed in our Faculty facility on ABI Prism capillary system. We provided DNA, which were purified, with VF (TGCCACCTGACGTCTAAGAA) and VR (ATTACCGCCTTTGAGTGAGC) primers and primers used for amplification of parts of our constructs – these shorter ones, without overlaps. To obtain better results, i.e. longer sequences we purified the primers by electrophoresis in 15% polyacrylamide gel with 7M urea.
</P></DIV><CENTER><H2>CONSTRUCT AND PART DESIGN</H2></CENTER><DIV class="clickable" onclick="toggle_visibility('content_17')"><FONT size="4">Primers design</FONT></DIV><DIV id="content_17" style="display: none"><P align="justify">
Primers were designed by using the NEBuilder with a minimal length of overlaps of 18 bp.
Parts of constructs were designed to be easily amplified with primers of the same melting temperature of 60°C. These PCR products of expected size were excised from agarose gels and reamplified with a pair of primers containing sequences of overlaps, and again purified from agarose gels. The initial primers we use to verify the structure of the final construct, as well as for sequencing. Verification of construct structure by Sanger sequencing was performed in our Faculty sequencing lab.
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_18')"><FONT size="4">Construct assembly by CPEC</FONT></DIV><DIV id="content_18" style="display: none"><P align="justify">
The purified PCR products containing overlaps were quantitated on DeNovix DS11 spectrophotometer. Sizes of these DNA fragments were checked by electrophoresis in agarose gels. Then fragments were mixed to obtain the molar ratio, like: 1(promoter): 1(CDS): 0.5(vector). 
Optimal amounts of DNA fragments are as in example: Promoter (300bp) 30ng; CDS (1000bp) 100ng; PSB1C3 vector (2000bp) 100ng.
<U>Preparation of a standard CPEC mix</U>
In a sterile PCR tube mix reagents in the following order:
5xHF buffer (NEB)                          10µl
Mixture of fragments with overlaps         xµl
H<SUB>2</SUB>O                                 up to 50µl
dNTP mix (25mM)                            1µl
Phusion DNA polymerase (NEB)              0.5µl

The table below provides standard reaction conditions for CPEC reaction.<U>Purification of the CPEC products before electroporation:</U>
CPEC products can be purified by two methods: phenol/chloroform extraction or by using silica columns Zymoclean Gel DNA Recovery Kit <A href="http://www.zymoresearch.com/">(www.zymoresearch.com)</A>.

Phenol/chloroform method: 
1.	Add 150µl H<SUB>2</SUB>O to the sample after CPEC, 100µl of chloroform and 100µl of phenol (pH=8.0)/
vortex and centrifuge 3 min at 12 000 rpm.
2.	 Collect the upper phase and transfer it to new eppendorf tube, add 100µl of chloroform,
vortex and centrifuge as in the previous step.
3.     Collect the upper phase and transfer it to new DNA LoBindTube.
4.     Add 2µl of glycoblue, mix, add 0.1 V of 3M sodium acetate (pH=4.8), mix well, and 2.5-3 volumes
of 99% ethanol and mix again.
5.	 Cool at -20°C for at least 2 h. Centrifuge 40 min at 12 000 rpm (4°C). Discard the supernatant
and suspend the blue pellet in 20 µl of sterile H<SUB>2</SUB>O. Use 7 µl for electroporation.

Purification by silica columns:
Proceed to Zymoclean Gel DNA Recovery Kit procedure starting from step 2.
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_19')"><FONT size="4">Coding sequence optimization</FONT></DIV><DIV id="content_19" style="display: none"><P align="justify">
The very high number of citations (777) of a paper published by Kane in 1987 indicates that may investigators faced
difficulties with an efficient production of recombinant proteins in <I>Escherichia coli</I> and associate these difficulties
with the presence of codons very rarely occurring in coding sequences of the host cell. Our first construct dedicated
to this phenomenon was sfGFP with four rare arginine codons located before two stop codons TAA (sfGFP-4R). We
wanted to check to what extent four such codons can be detrimental to the biosynthesis of a stable and soluble
recombinant protein in <I>E. coli</I>. Surprisingly it appeared that there is no clearly visible difference between the rate of
sfGFP and sfGFP-4R biosynthesis. Then we extended this motif to eight consecutive AGG / AGA codons, and we have
found that sfGFP-8R biosynthesis rate is still almost identical to that of sfGFP without rare arginine codons.
There are several codon optimization methods available, which are usually based on codon usage in the host cell,
however the results of optimization are quite often disappointing. We decided to check what is the difference in the
protein synthesis rate between two very different versions of the same reading frame, one composed of the most
common codons (sfGFP-B, the best choice) in <I>E. coli</I> orfeome and another codon-monotonous as well but composed
of the rarest codons (sfGFP-W, the worst case). In sfGFP-W coding sequence appeared 7 dispersed AGG codons, 10
isoleucine ATA codons, one neighbouring with AGG, 15 leucine CTA codons, one neighbouring with ATA, and two pairs of consecutive CTA codons. We have found and show it in results section, that in <I>E. coli</I> cells growing in rich media the introduction of 32 very rare codons to a 248 codons long sequence coding for a well soluble protein at a
moderate level (like in pBAD systems) is not sufficient to observe any significant decrease in the rate of its
translation + accumulation several hours after induction of the gene expression. We decided then to check if the
same results will be observed in the case of a fluorescent protein of a different sequence. We have chosen mRFP.
Looking for any other general way to optimize ORFs, we are working now on contrasting ORFs which are AT or GC
rich, with codons optimization based on codon adaptation index and on most common and the rarest codon
contexts.
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_20')"><FONT size="4">ORFs</FONT></DIV><DIV id="content_20" style="display: none"><P align="justify">
We have prepared four different rankings for different sets of coding sequences: a whole orfeome set generated for
<I>Escherichia coli</I> str. K12 substr. MG1655 (NCBI: NC_000913.3) and sets of 300, 200 and 100 most abundant
proteins for <I>Escherichia coli</I> based on Ishihama et al., 2008. The most abundant proteins with the highest log-copy
number values were chosen (see Ishihama et al., 2008). The maximization and minimization of Codon Pair Bias was
performed using CodonComposer for sfGFP and RFP open reading frames and default GA parameters. The additional
constraints the algorithm was checking for were lysine-encoding A-homopolymers (that may cause ribosome sliding)
and restriction enzyme sites of the RFC10 format. The mRNA secondary structures were predicted using the RNAfold
server and the default parameters.
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_21')"><FONT size="4">Codon context optimization</FONT></DIV><DIV id="content_21" style="display: none"><P align="justify">
CodonComposer <A href="https://github.com/MelaniaNowicka/CodonComposer">(https://github.com/MelaniaNowicka/CodonComposer)</A> is a piece of software which allows to
calculate a codon pair ranking for any set of ORFs and optimize the codon context considering restriction enzyme
sites, A-homopolymers removal and maintenance of the effective number of codons at the same time.
Codon context optimization is based on the ranking approach. The codon context ranking (scores for each possible
codon pair) is calculated for any chosen set of open reading frames using the Codon Pair Bias measure proposed by
Coleman et al., 2008. The overall score for the ORF of interest is calculated as the arithmetic mean of individual pair
scores.
The implemented genetic algorithm (GA) allows to obtain close to optimal results and optimization is much less
time-consuming than it would be for an exact algorithm when considering additional constrains. GA is an algorithm
inspired by the processes of natural selection. The algorithm uses population-based approach which means that the
optimization is performed for a large set of possible solutions called individuals. All steps mimic processes occurring in nature: individuals undergo selection, recombination and mutation to „evolve” a better solution during iterations
called reproductive cycles. For more details, please check the CodonComposer GitHub repository.
</P></DIV><CENTER><H2>MEASUREMENTS</H2></CENTER><DIV class="clickable" onclick="toggle_visibility('content_22')"><FONT size="4">Fluorescence measurements</FONT></DIV><DIV id="content_22" style="display: none"><P align="justify">
To measure the strength of the promoters, we measured  the florescence of sfGFP which is used as reporter protein. 
1.	Drive the bacterial culture in 25ml of medium and induce the expression when the culture reaches OD<SUB>600</SUB> =0.4.
2.	Collect 500µl of bacteria after each hour during 6 hours of measurement and centrifuge immediately (4°C, 3 min, 14 000rpm).
3.	Discard the supernatant and suspend the pellet in 0.5ml of buffer (5M urea, 50mM Tris-HCl pH=8.0, 20mM EDTA, 0.1% SDS).
4.	Incubate the solution 5 min at 55°C and then centrifuge 4 min, 13 000 rpm.
5.	Transfer the lysate to a new eppendorf tube.
6.	To measure the fluorescence dilute the lysate 10 times in buffer (50mM TrisHCl pH=8.0, 5mM EDTA) and transfer the sample to the 96-wells black plate. The buffer provides the proper ionic strength, in contrast to water, so the fluorescence of sfGFP does not change upon dilution.
7.	Besides samples, put on the plate: blank sample- the buffer and GFP solution- which provides max emission level of fluorescence (50 000 RFU). Thereby all results will be normalized to the one, known maximum value.
8.	Put the plate into Infinite 200 PRO microplate reader (Tecan) <A href="http://lifesciences.tecan.com/products/reader_and_washer/microplate_readers/infinite_200_pro">(http://lifesciences.tecan.com/products/ reader_and_washer/microplate_readers/infinite_200_pro)</A> and perform the measurement. Apply the excitation light wavelength to  485nm, at which GFP is maximally excited - and the emission light wavelength to 535nm, at which the GFP emission is the highest. 
</P></DIV><DIV class="clickable" onclick="toggle_visibility('content_23')"><FONT size="4">OD<SUB>600</SUB> Measurements</FONT></DIV><DIV id="content_23" style="display: none"><P align="justify">
1.	To control the density of bacteria during fluorescence measurements use a spectrophotometer- we were using DeNovix. Collect 500µl of  bacterial culture, after each hour during 6 hours of measurement, and  immediately centrifuge (4°C, 3 min, 14 000 rpm).
2.	Discard the supernatant and suspend the pellet in 0.5ml of 0.3M NaoAc pH=5.0. The acetate pH=5.0 causes that GFP protein stop emit the fluorescence, what is very important because the fluorescence of protein can reduce the OD<SUB>600</SUB> values during measurement and provide unreliable data.
3.	Dilute solution 10 times, compared to suspended in acetate pellet, and measure the OD<SUB>600</SUB>.
</P></DIV><P><B>References</B>
1.	Koressaar T1, Remm M. (2007) Enhancements and modifications of primer design program Primer3. Bioinformatics 23(10):1289-91.
2.	Untergasser A1, Cutcutache I, Koressaar T, Ye J, Faircloth BC, Remm M, Rozen SG. (2012) Primer3—new capabilities and interfaces. Nucleic Acids Res. 40(15):e115.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>