<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UAM_Poznan_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UAM Poznan/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="background"><DIV id="page"><DIV id="header"><DIV id="navigation"><UL><LI id="menu_home"><A href="https://2016.igem.org/Team:UAM_Poznan">Home</A></LI><LI id="menu_project"><A href="#" class="noclick">PROJECT</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Description">MAIN IDEA</A><A href="https://2016.igem.org/Team:UAM_Poznan/Demonstrate">APPLICATIONS</A></SPAN></LI><LI id="menu_team"><A href="https://2016.igem.org/Team:UAM_Poznan/Team">TEAM</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Attributions">ATTRIBUTIONS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Collaborations">COLLABS</A></SPAN></LI><LI id="menu_results"><A href="#" class="noclick">RESULTS</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Results">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Results-ORF">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Achievements">ACHIEVEMENTS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Biobrick_testing" class="double-line">BIOBRICK TESTING</A></SPAN></LI><LI id="menu_parts"><A href="https://2016.igem.org/Team:UAM_Poznan/Basic_Part">PARTS</A></LI><LI id="menu_methods"><A href="https://2016.igem.org/Team:UAM_Poznan/Experiments">METHODS</A></LI><LI id="menu_notebook"><A href="#" class="noclick">NOTEBOOK</A><SPAN class="submenu"><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook-promoters">PROMOTERS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Notebook">ORFs</A><A href="https://2016.igem.org/Team:UAM_Poznan/Safety" class="double-line">SAFETY AND ETHICS</A><A href="https://2016.igem.org/Team:UAM_Poznan/Patenting">PATENTING</A></SPAN></LI><LI id="menu_galleries"><A href="https://2016.igem.org/Team:UAM_Poznan/Galleries">GALLERIES</A></LI><LI id="menu_hp"><A href="https://2016.igem.org/Team:UAM_Poznan/HP/Silver" id="hp_menu" class="double-line">HUMAN PRACTICES</A></LI></UL></DIV></DIV><TITLE>UAM_Poznan Team</TITLE><DIV id="contents"><DIV class="box"><DIV><DIV class="body"><H1><CENTER>Introducing E.co-Factory project</CENTER></H1><P align="justify">Scientists all over the world use <I>Escherichia coli</I> as the host for recombinant protein production. The system is so popular because of its rapidity, low cost and well-characterized genetics. The choice of a proper promoter is a factor highly affecting protein expression. Right now, pET lactose/IPTG induced T7 RNA polymerase dependent system, which enables high efficiency of protein production, is the most popular one. However, it has some downsides: the system is leaky, so proteins toxic for the host cell cannot be produced. Its leakiness is very hard to control once the stable and highly active T7 RNA polymerase is produced. Once the system is induced, the high heterologous protein accumulation can turn into a problem with insoluble inclusion bodies. The typical phage T7 RNA polymerase tends to introduce more mistakes than its cellular counterpart. Considering that translation in prokaryotic cells occurs on just transcribed mRNAs, slower cellular RNA polymerase can be beneficial for an accurate translation of mRNAs with stronger secondary structures, which have lower chance to form before ribosomes will cover the nascent mRNA molecule.</P><P align="justify"><B>Our group, UAM_Poznan, aims to generate four sets of promoters induced by sugars: xylose, melibiose, arabinose and rhamnose, and give the user the possibility of introducing numerous genetic circuits into one cell, which expression can be independently controlled.</B>
We have reduced promoters' size – they are minimal but fully functional and tightly regulated. We have also introduced changes in 5’UTRs (untranslated regions), as 5’UTR regions with their secondary structures and 16S rRNA binding sites of different sequence and location regarding AUG codon are known to strongly influence the efficiency of translation.<B>Our latest activity focuses on trials to find out to what extent the structure of an open reading frame (ORF) can be optimized and what is the difference in translation rate of let say optimal, less optimal, and much less optimal reading frame.</B> We build our experimental model on sfGFP and mRFP – two soluble fluorescent proteins. We compare expression rate of contrasting variants of their coding sequences – optimized and the most deoptimized codon usage, codon adaptation usage, codon contexts and the highest or the lowest GC content. This simple comparison should allow to evaluate which method of ORF optimization does really make sense. </P><CENTER><H2>Strong, tight promoters</H2></CENTER><P align="justify"><B>Our multipromoter expression system for <I>E.coli</I> contains four sets of pSB1C3 vectors with promoters induced by non-toxic compounds - sugars: arabinose (Ara promoters), rhamnose (Rha promoters), xylose (Xyl promoters) and melibiose (Mel promoters).</B> Promoter is induced by a specific sugar and starts the expression of a gene. One can exchange the ORF of sfGFP to any other coding sequence of interest.. AraC, RhaS, RhaR, MelR, XylR are transcription factors- they bind specific sugars, by which they are activated, and start the transcription. These factors are dependent on the binding of the catabolite receptor protein (CRP)-cAMP complex to the promoter. In the presence of glucose, cAMP levels are low and CRP is unable to bind to the promoter, which results in blocked transcription.</P><P align="justify">Our multipromoter system provides <B>differential levels of protein expression,</B> so one can choose either stronger promoter, or the one ensuring lower levels of inducible expression. Since all four sets of <B>promoters are tightly regulated,</B> they can be useful in production of proteins toxic for <I>E.coli</I> cells. It is also worth taking under consideration that <I>E.coli</I> cells behave differently on different carbon sources, which results in contrasting bacterial growth rate and indirectly influence protein production. E.co-Factory ensures independent induction of expression of at least two different genes and efficient blockage of all promoters by glucose. <B>Moreover, cellular RNA polymerase is more accurate and introduces less mistakes in the transcribed sequence than that of phage origin from pET system,</B> which is important if one’s aim is to produce recombinant proteins for crystallography and medicine.</P><P align="justify">Our group modified arabinose, rhamnose, xylose and melibiose promoters, from which AraC-pBAD arabinose promoter is commercially available. All promoter sequences were copied from <I>E.coli</I> K-12 genome. <B>Modifications of promoters include sequence size reduction</B> with full functionality maintenance, because shorter promoters = smaller biobricks, which are more convenient for molecular cloning purposes. <B>We have substituted original 5’UTRs</B> of genes located downstream these promoters with synthetic sequences considering RNA structure, the RBS motif and its positioning. <B>Our biggest achievement this year is the experiment in which we have transplanted a 39nt fragment of 5'UTR of gene 10 of bacteriophage T7 to three of four (melibiose inducible promoter is still under construction) tightly regulated <I>E. coli</I> promoters.</B> It appears that all fusion promoters (araE1, xylE1, rhaE1) are still much tighter than the lactose induced T7 systems and all fusion promoters are much more responsive to inducers. One of them upon induction seems to give higher expression levels than T7 expression systems. Considering that the additional ribosome binding site in 5'UTR seems to be an universal mechanism which allows for an efficient polysome takeover in <I>E. coli</I> and possibly in other procaryotic systems, it opens new attractive opportunities, at least to develop very efficient expression systems for other bacteria like <I>Bacillus</I>, <I>Lactococcus</I>, and many other.</P><CENTER><H2>Better, and better ORFs</H2></CENTER><P align="justify">Degeneracy of the genetic code means that some amino acids are encoded by more than one codon, thus the same protein sequence can be translated from various mRNAs. The frequency of particular codons in the gene of interest can cause expression problems because of rarely occurring tRNAs. Variation in codon usage is considered to be a major problem in heterologous protein production. It can contribute to slowing down the translational process and induce error introduction. It is believed that by codon optimization one can substantially increase the gene expression and that the optimized gene will more effectively compete for cell resources and will be more accurately translated [Kane JK, 1995]. <B>We would like to check which approach to optimize a reading frame is the best and to what extent it can improve the expression of the optimized gene. We consider improvements of such traits as: codon usage, codon adaptation index, contexts of codons and secondary structures in coding sequences.</B></P><P align="justify">We intentionally started our comparisons from implementing general optimization rules, which effects can be easily compared in simple induced expression experiments.  We decided to check what is the difference in the protein synthesis rate between two very different versions of the same reading frame- sfGFP [Pedelacq JD, 2006], one <B>composed of the most common codons</B> (sfGFP-B, the best choice) in <I>E. coli</I> orfeome and <B>another codon-monotonous as well but composed of the rarest ones</B> (sfGFP-W, the worst case). According to bioinformatics analysis we indicated 7 codons that occur in <I>E.coli</I> genome with the least frequency rate. They encode Arginine (AGG), Leucine (CTA), Isoleucine (ATA), Glycine (GGA), Serine (TCA), Proline (CCC) and Threonine (ACA). These codons represent 40% of all codons of sfGFP_Worst ORF. sfGFP_B and sfGFP_W constructs differ within all codons with the exception of the first ten residues that correspond to a stable N-end as a 6-histidine tag, which is often used for affinity purification of recombinant proteins. 
Looking for any other general way to optimize ORFs, we are working now on contrasting ORFs, which are <B>AT or GC rich</B>, with codons optimization based on codon adaptation index. Further experiments concern <B>examining codon context</B> effect, a hypothesis claiming that the translational efficiency of a given codon depends highly on its neighboring codons. Therefore, using specially designed software designed by our advisor, Melania. She analyzed and chose codon pairs which occur most often and least often in <I>E.coli</I> transcriptome. We have prepared four different codon context rankings: one based on whole orfeome and three different sets of coding sequences for high abundant proteins. Using the rankings we have maximized and minimized codon context for two ORFs, sfGFP and mRFP to test if there is a correlation between single codon, codon pair frequencies and protein abundancy.</P><CENTER><H2>Biobricks’ improvements</H2></CENTER><P align="justify">According to the criteria that each group has to fulfill in order to get gold medal, we improved function of two biobricks, which we delivered to the iGEM Registry in 2015: one arabinose induced promoter- Arashort1 (araBAD; <A href="http://parts.igem.org/Part:BBa_K1741000">BBa_K1741000</A>), and one xylose induced promoter- XylS (xylA-proD5'UTR; <A href="http://parts.igem.org/Part:BBa_K1741009">BBa_K1741009</A>). <B>These year’s improved constructs are as follows: </B><B><A href="http://parts.igem.org/Part:BBa_K2014003">BBa_K2014003</A></B> - pBAD-M5'UTR-&gt;sfGFP – improvement of pBAD promoter <A href="http://parts.igem.org/Part:BBa_K1741000">(BBa_K1741000)</A> with M5’UTR,<B><A href="http://parts.igem.org/Part:BBa_K2014004">BBa_K2014004</A></B> - pxylS-M5'UTR-&gt;sfGFP - improvement of XylS promoter <A href="http://parts.igem.org/Part:BBa_K1741009">(BBa_K1741009)</A> with M5’UTR.
All constructs contain sfGFP under different promoters/UTRs- as a marker of gene expression and protein synthesis/accumulation.</P><P align="justify">For more information about these improvements see our <B><A href="https://2016.igem.org/Team:UAM_Poznan/Results">Results</A></B> and our BioBricks' main pages: <B><A href="http://parts.igem.org/Part:BBa_K2014003">BBa_K2014003</A></B> and <B><A href="http://parts.igem.org/Part:BBa_K2014004">BBa_K2014004</A></B></P><P align="justify"> We have also tested and slightly improved <B><A href="http://parts.igem.org/Part:BBa_K1481003">Ba_K1481003</A></B>- biobrick provided to iGEM in 2014 by Poznan_Bioinf team, named &quot;sfGFP with R4-tag under arabinose promoter&quot;. We have checked if four rare arginine codons (namely AGG and AGA) added to the 3' end of GFP reading frame will influence its translation. Surprisingly we have found that there is no obvious difference in expression comparing to gene without arginine tag. 

Thus, we improved the construct and added four rare arginine codons (AGA) under identical promoter (AraC-pBAD) with an identical 5’UTR- <A href="http://parts.igem.org/Part:BBa_K2014007"><B> BBa_K2014007 </B></A>. </P><P align="justify"><I>References:</I>
1.	Kane JF. (1995) <I>Effects of rare codon clusters on high-level expression of heterologous proteins in Escherichia coli.</I> Current Opinion in Biotechnology, 6:494-500
2.	Pédelacq JD, Cabantous S, Tran T, Terwilliger TC, Waldo GS. (2006) <I>Engineering and characterization of a superfolder green fluorescent protein.</I> Nat Biotechnol. 2006 Sep;24(9):1170.


3.      Olins PO, Rangwala SH.; <I>A novel sequence element derived from bacteriophage T7 mRNA acts as an enhancer of translation of the lacZ gene in Escherichia coli.</I> J Biol Chem. 1989 Oct 15;264(29):16973-6.

4.      Davis J.H., Rubin A.J., Sauer R.T.; <I>Design, construction and characterization of a set of insulated bacterial promoters.</I> Nucleic Acids Research, 2011, Vol. 39, No. 3 1131–1141

5.      Haldimann A., Daniels L.L, Wanner B. L.;<I> Use of New Methods for Construction of Tightly Regulated Arabinose and Rhamnose Promoter Fusions in Studies of the Escherichia coli Phosphate Regulon.</I> Journal of Bacteriology, Mar. 1998, p. 1277–1286

6.      Holcroft C.C, Egan S.M.  <I>Roles of Cyclic AMP Receptor Protein and the Carboxyl-Terminal Domain of the a Subunit in Transcription Activation of the Escherichia coli rhaBAD Operon.</I> Journal of Bacteriology, June 2000, p. 3529–3535

7.      Giacalone M.J. et.al., <I>Toxic protein expression in Escherichia coli using a rhamnose-based tightly regulated and tunable promoter system.</I> BioTechniques 40:355-364 (March 2006)

8.      Song S., Park C.; <I>Organization and Regulation of the D-Xylose Operons in Escherichia coli K-12: XylR Acts as a Transcriptional Activator.</I> Journal of Bacteriology, Nov. 1997, p. 7025–7032


9.      Hans Peter Sørensen, Kim Kusk Mortensen.,<I> Advanced genetic strategies for recombinant protein expression in Escherichia coli.</I> Journal of Biotechnology, August 2004.

10.      Jason R. Newman, Clay Fuqua., <I>Broad-host-range expression vectors that carry the L-arabinose-inducible Escherichia coli araBAD promoter and the araC regulator.</I> Gene, Nov. 1998.

11.      Lehmeier B, Amann E. <I>Tac promoter vectors incorporating the bacteriophage T7 gene 10 translational enhancer sequence for improved expression of cloned genes in Escherichia coli.</I> Journal of Biotechnology, Apr. 1992; 23(2):153-65.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>