"
Page
Discussion
View source
History
teams
Log in
Â 
Team:BU Wellesley Software/OptimusPrimer
From 2011.igem.org
BU-Wellesley iGEM Team: Optimus Primer
Team
Project
Overview
Clotho
G-nome Surfer Pro
Optimus Primer
Trumpet
Puppetshow
eLabNotebook
Wet Lab
Downloads and Tutorials
Process
Methodology
Safety
Notebook
Outreach
Tips and Tricks
Medal Fulfillment
Additional Info
Acknowledgement
Fun
Optimus Primer
JUMP TO...
Top
Tool Overview
Demo
Results
Future Work
Tool Overview
We implemented the OptimusPrimer for the desktop and the Microsoft tabletop Surface. Both the surface application and desktop GUI allow users to create a primer design from a selected gene. Users can then can BLAST designs and run tests based on alignment and Gibb's free energy. Both applications make use of the same algorithms and allow saving as BioBricks.
The OptimusPrimer GUI is written in Java with Swing and offers an interface for independent and desktop centered design. It is designed for integration with Clotho and allows users to save designs as BioBricks. Users can select a number of bases and run tests to determine the sustainability of their designs.
The surface application is written in C# for the Microsoft Surface. The system is accessible from the G-Nome Surfer interface. Users create designs by sliding the highlight on the gene's ends and selecting the primers.  Alignment results are generated using a .jar and BLAST results are generated using the NCBI BLAST software. Both are accessed from the command line, with BLAST results being written to file and parsed out for display and alignment results being pulled from common output for further calculation for hetero-dimer and self-dimer tests. The surface application provides a streamlined, intuitive interface for learning and collaborative design.
Optimus Primer is an environment for designing primers and saving genes as BioBricks. We have built two interfaces-a desktop GUI for independent research and design and the surface application for collaborative learning and design. Both allow users to run a number of tests on designs to determine their feasibility before saving as BioBricks.
Demo Video
Results
We successfully utilized this application to design primers for amplifying the MTb gene with attached restriction sites using PCR reaction. First of all, Optimus Primer was used through Clotho interface to generate 6 forward primers and 6 reverse primers. There was no need to determine the existence of restriction sites in the amplified sequence, because Optimus Primer will put a warning sign if that ever happens. Optimus Primer also informed us the length, melting temperature or free energy for each of the generated design. Furthermore, the bottom panel at the display window would turn pink if dimerization could occur. With the instantaneous data provided by Optimus Primer, we were able to design the required primer for MTb gene amplification easily. The primer was then ordered and used in the PCR reaction to insert TB transcription factors into the characterized genetic devices.
Future Work
There are many small usability issues to address within both interfaces in the future.  For the surface application specifically:
We would like to support more than the first 40 forward and reverse base pairs as we do now
When performing dimerization checks we would like to also display where they fail on the primer itself.
In the future we also plan to explore the advantages and disadvantages between both the Surface application and the desktop GUI. These vastly different interaction techniques can greatly affect the software's effectiveness in a laboratory setting. Using our findings, we will continue to look for ways on making the best Primer Designer.
Retrieved from " http://2011.igem.org/Team:BU_Wellesley_Software/OptimusPrimer "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
