"
Page
Discussion
View source
History
teams
Log in
 
Team:BU Wellesley Software/Wet Lab
From 2011.igem.org
BU-Wellesley iGEM Team: Wet Lab
Team
Project
Overview
Clotho
G-nome Surfer Pro
Optimus Primer
Trumpet
Puppetshow
eLabNotebook
Wet Lab
Downloads and Tutorials
Process
Methodology
Safety
Notebook
Outreach
Tips and Tricks
Medal Fulfillment
Additional Info
Acknowledgement
Fun
Wet Lab
JUMP TO...
Top
Background
Tools
Results
Problems
Future Works
Background
While some bacterial infections are seen as common place, others remain elusive to cure. Tuberculosis has currently infected a third of the world’s population, and 1.7                             million people                          died from it in 2009.  As shown, it is a worldwide problem that is prevalent in third                           world                            countries.
Tuberculosis exists in two stages. In its latent stage, the immune system has                           Tuberculosis under                           control and prevents the bacteria from reproducing. It switches into its active stage when a person's immune system becomes compromised. As it begins to multiply the person                           starts                            showing symptoms of the disease.
The genetic mechanisms that trigger the change between the two                           forms are not very well understood. The genetic network of tuberculosis is                            complicated and                           studying the interactions between genes can be both material and time consuming, requiring the                            construction of                           many plasmids to study only one small part of the network.  To look for new, faster ways of                           studying this problem we turned to synthetic biology.  Instead of creating many vectors                           to study the interactions of genes, we aimed to create a plasmid with a designed circuit that                           would control the transcription of several tuberculosis genes using invertases. We chose to                           use non-pathogenic Mycobacterium tuberculosis genes and transcription factors within non-                            pathogenic E.coli for safety reasons.
Invertases are recombinases that will bind to recognition sites. Once bound, the DNA between two sites will invert and flip horizontally. This effectively acts as an on/off switch for transcription of that region of DNA as it is not available for DNA polymerase to bind to. In terms of circuit design, it will act as a not gate.  Using the within a plasmid with several tuberculosis genes would allow us to check for regulatory measures such as negative feedback, etc using only one plasmid.
Tools
In addition to the novel cellular architecture, we also used a variety of software tools created by our computational teams (Boston University, Wellesley):
G-nome Surfer Pro: G-nome Surfer Pro is an integrated environment that allows for the viewing of prokaryotic genomic data and literature associated with the genome. As it is built on a Microsoft Surface, it encourages collaboration. We used Optimus Primer on the G-nome Surfer to design the primers for the tuberculosis genes and transcription factors.
Clotho: Clotho is used to mange, create and store new biological building blocks in community based repositories. It includes a suite of tools that include PuppetShow and Trumpet, which were designed specifically for this project.
The following are new Clotho applications that were made this summer:
Trumpet: Trumpet consists of a library of genes and promoters, which can be configured into any desired permutation or combination by treating the DNA with recombinases, which allows us to rewire the genes and switches and study all their combinations. It was used by the wetlab to help us map out where the invertases should go in our plasmids in order to correctly turn on and off the segments of DNA we want to study.
PuppetShow: This suite includes a high level programming language for specifying biological protocols commonly used in the laboratory, which are then executed by a liquid-handling robot with minimal user intervention. We used this in conjunction with the robot (described below) to create samples that were simultaneously created manually to compare the results.
OptimusPrimer: A primer designer used to create the primers for the Tuberculosis genes.
These are Clotho tools that were enhanced this summer to assist us in our program:
Feature Chomp: reads in APE and GENBANK files and takes the features found within the file to a feature database
Batterboard:allowed us to electronically represent physical samples in the lab
Other Clotho tools that were employed to store a variety of parts, backbones, and features electronically:
Bull Trowel
Spreadit Parts
Spreadit Vectors
Spreadit Features
Another way we looked into facilitating progress in studying genetic networks is through the use of automation:
Robot
The liquid handling robot was used to reproduce molecular biology protocols in a more efficient and accurate manner.  The robot is controlled by Puppeteer which creates code necessary to choose the correct series of commands in EVOware,the robot's software.
QIA Cube
We purchased a QIA Cube to automate frequently used procedures such as Gel Extraction and Mini-Prep. It could take care of up to twelve samples at a time, and would free us up to work on other needs in the lab.
Through the usage of novel cell architecture, software, and hardware we show that studying complex genetic networks can be done faster than has been done in the past. It reduces the amount of work needed to be done, as well as the demand on materials and time.
Results
Being associated with a large computational team, the wetlab team was unique as our results not only consisted of creating novel forms of DNA, but evaluating the usefulness of the software tools our computational team produced. Our success was not only measured in the number of colonies we could get to grow, but also in how well we could incorporate these tools and give informative feedback.
Clotho: Key to Creation
Clotho allows one to enter a variety of pieces of DNA including promoters, RBS, genes, and terminators into a database and then electronically form plasmids out of them. Besides pieces like the promoters which are considered parts in Clotho, the backbones can be saved as vectors and can be changed up in creating the plasmids. BullTrowel , Spreadit Parts , Spreadit Vectors , and Spreadit Features are a suite of tools that enable one to enter the parts, vectors, plasmids, and features into the database.
An example of a collection showing parts, vectors, and features
Collector Browser provides a way to view the library of everything already entered that allowed us to check out what parts were available to form a plasmid. Additionally, parts could be entered as features. In Sequence Viewer , features are highlighted to enable us to know where such things as restriction enzymes are in a plasmid. Once Clotho helped us to maximize the design of our plasmids, we then set out to physically construct them.
A DNA sequence opened in sequence viewer with features highlighted.
The first vectors we were interested in creating were fluorescent protein reporters devices. Our goal was to eventually fuse genes with the reporter devices in order to establish that the genes were being turned on or off.It is important to understand the behavior of each promoter before using it in a complex plasmid architecture, because a baseline is needed to establish how it behaves independently of any additional transcription factors. In the wet lab, various devices were created with either constitutive or inducible promoters. The constitutive promoters like Bba_R2000, Bba_I14033, and Bba_R0040 cause the genetic devices to produce fluorescent protein at all times. But the inducible promoters like Bba_I13453 need a factor such as arabinose in order to promote the transcription of the fluorescent protein gene. Besides figuring out which promoters work best, we also wanted to create a small arsenal of different fluorescent proteins to use. Our devices include one of the following: red, yellow, green, and blue fluorescent proteins.
Part
Diagram
Notes
BBa_K577000
BBa_K577001
Lacks the tetR gene to make this repressible
BBa_K577003
Acts as a constitutive promoter, but does not glow strongly.
BBa_K577004
Lacks the araC gene to make it glow
BBa_K577006
Lacks the araC gene to make it glow
BBa_K577881
This reporter part is a combination of a constitutive promoter, AraC gene, a pBad promoter (inducible with arabinose), and a GFP gene.
BBa_K577882
BBa_K577883
BBa_K577884
BBa_K577891
An expression vector of the TetR gene using a constitutive promoter, J23101
BBa_K577892
An expression vector of the TetR gene using the constitutive promoter, I14033
BBa_K577893
A repressed pTet promoter to a GFP gene. Its repression can be inhibited by addition of tetracycline or aTc
BBa_K577894
A repressed pTet promoter to a YFP gene. Its repression can be inhibited by addition of tetracycline or aTc
BBa_K577895
A repressed pTet promoter to a RFP gene. Its repression can be inhibited by addition of tetracycline or aTc
BBa_K577896
A reporter plasmid using pTet as a constitutive promoter in front of a GFP gene
BBa_K577898
A plasmid containing strongly enhanced blue fluorescent protein (SBFP2) combined with the double terminator
BBa_K577899
A reporter plasmid using pTet as a constitutive promoter in front of a BFP gene
BBa_K577900
A reporter plasmid using pCAt as a constitutive promoter in front of a BFP gene
The protocols we used to create these devices can be found here .
Obstacles/Trouble Shooting: Despite our plan to create a reconfigurable plasmid by the end of summer, we encountered some problems that prevented us from achieving this goal. In the first few days, we had extremely low DNA quantification values for our plasmid samples. While there seemed to be growth in the overnight cultures, there was no evidece of DNA in the gel after gel electrophoresis. In order to verify if the correct bacteria grew in the liquid culture, a small sample was taken for gram staining and observed under the microscope. There seemed to be other bacteria in these samples and we concluded that the antibiotic selection was not working or our lab equipment was contaminated.
As a result, we autoclaved the pipet tips and prepared a new batch of antibiotic-equipped LB agar plates. Then, we faced an issue with our ligations because there were no colonies on the transformed plates of the ligation reactions. We tried tinkering with the ligation protocols by using different insert to vector ratios, ethanol precipitations,  and various ligation enzymes. We even tried treating the backbone with the CIP enzyme to prevent the backbone from ligating to itself. After several unsuccessful troubleshooting approaches, we tried pelleting the cell to increase the concentration of the comptent cells made and then attempted to use a commercial line of competent cells. Fortunately, the second method worked and we could build our genetic devices!
The final problem involved the failure to induce the pBad promoter. After adding various concentrations of arabinose and using different protocols, it turned out that the commercial cell lines we were using to solve the ligation issues did not contain the AraC gene. Since the AraC gene has to interact with arabinose to activate the pBad promoter, we then found a biobrick part with the AraC gene and ligated it to the existing constructs. Our completed devices were then analyzed through the Fluorescence Activated Cell Sorting (FACS) machine. The data collected produced plots of arabinose concentration versus fluorescence intensity shown below.
From the FACS data on arabinose induction and the resulting fluorescent protein gene expression, it appears that each genetic devices behave differently. While the tested device contain so much more components than just the reporter gene, but these constructs use a similar set of parts. Hence, the fluorescent intensity can be narrowed down enough to characterize the promoter and fluorescent genes. Before you continue reading, please note that fluorescent intensity uses arbitrary units so that such value is only meaningful when compared with other fluorescence signals.
The first graph shows the correlation between arabinose concentration and the fluorescence intensity expressed by the pBad+GFPc device. There were 3 colonies tested with the FACS machine and the plots of their expression levels show a relatively high degree of consistency. It appears that the effect of arabinose addition starts to manifest when the concentration of arabinose used is somewhat larger than 0.01 mM. If arabinose is added with a concentration that is one order of magnitude larger, the increase in fluorescence becomes drastically steeper up until the x value reaches ~5 mM. Then further addition will keep on increasing the fluorescence intensity, albeit with a smaller effect. The plot from the third colony is a bit unusual because it shows a more drastic increase, before falling down at the x value of ~5 mM. The huge gap between the last points from the first 3 points allow us to determine that ~5500 points difference in fluorescence intensity can be used to define an activated pBad+GFPc device when 50 mM arabinose is used.
The second graph shows the high correlation between arabinose concentration and the fluorescence intensity produced by the pBad+RFPc device. The 3 plot lines come from the 3 colonies tested with the FACS machine and the huge overlap between the lines implies a consistent pattern of gene expression. In this case, a visible gene expression occurs when the arabinose added has a concentration of 0.001 mM or higher. If arabinose is added with a concentration that is one order of magnitude larger, the increase in fluorescence becomes drastically steeper up until the x value reaches ~5 mM. Unlike the previous graph, the data does not tell us what will happen if arabinose with concentration higher than 5mM is used. The huge gap between the last points from the first 2 points allow us to determine that ~500 points difference in fluorescence intensity can be used to define an activated pBad+RFPc device when 5 mM arabinose is used.
By combining the running averages from the previous two graphs into the third graph, several conclusions can be reached on how the 2 devices compare. First, arabinose with concentration 0.001 mM or lower does not seem to induce any gene expression. Second, a similar pattern of induction can be observed in the individual behavior of pBAD+GFPc and/or pBad+RFPc. Nevertheless, direct comparison between them shows a great discrepancy between their intensity values. The expression level from the pBad+GFPc device reaches a peak value of ~5500, about one order of magnitude higher than the ~500 value from the pBad+RFPc device. In case both devices are incorporated into larger plasmid architecture, their relative behavior has to be noted as a baseline while analyzing the overall gene expression. Furthermore, a fluorescent intensity plot that appears flat when juxtaposed with another plot using the same axis does not prevent the possibility that they have the same characteristics that only differs in the relative values. In the end, it is important to remember that the highest expression with 50 mM arabinose is not even visible unless the FACS machine is used. Hence, an instrument with high sensitivity is needed to determine the activity of our genetic devices.
Gnome Surfer Pro:
We used genome surfer pro on the Microsoft surface to easily visualize the plasmid of TB myobacterium and choose the transcription factors we would like to use in our basic device. It also is linked to GenBank files to tell us the sequence of the gene of interest and the amino acids it codes for. In addition, we looked up more information about the transcription factors such as finding it in publications from the PubMed database.  This information was crucial to learn more about the transcription factors we were going to handle in the lab.
Optimus Primer:
After we characterized the promoters with the FACS machine and researched the MTb genes using Gnome Surfer Pro, we could theoretically start building a genetic device with a MTb gene fused with the fluorescent protein gene. The first step was to design primers to amplify the MTb gene with desired restriction sites using PCR reaction. The tool Optimus Primer could be used on two different interfaces, in Clotho and on the Microsoft surface to design and visualize primers. We used Optimus Primer in Clotho to be able to visualize the primers and detect whether there were restriction sites in the primers. By clicking “Generate Primer”, 6 forward primers and 6 reverse primers were produced. For each set of the 2  types of primers (FF and FR), Optimus Primer calculated the differences in length, melting temperature, and free energy. Also the bottom panel will turn pink if the possible occurrence of dimerization is detected. We then chose the primers we would like to use for our PCR reaction from the two sets that were designed. The PCR reaction will help us in the process of inserting the TB transcription factors into the characterized genetic devices.
Trumpet:
Next, we used Trumpet as a design tool to map out how to manipulate our devices further. Trumpet will design a configurable device with our genes of interest as well as permutations/combinations we need.  The diagram will be organized in a way that will help us visualize the permutations in a specific order, so we are aware of how to build this elaborate construct in the lab.
Automated Creation: It was the best of the times; it was the worst of times
Using Puppeteer, the BioBrick construction process was automated. Ligation and restriction digest could be done using a liquid handling robot.  The robot could pipet the correct volumes from its specified solutions and combines the needed solutions to either digest a plasmid or ligate two digested parts of a plasmid.  The order of commands that the robot would perform in order to execute the desired protocol was written by HAL.  HAL was generated after Puppeteer (python-based high level language) analyzed how many samples, enzymes, water, and buffers needed to implement the desired protocol.  Then it analyzed the robot deck in which each sample would be placed and decided the best commands to generate on HAL to control the robot’s software.
Restriction Digest:
Restriction digest was tested first as it was successfully proven to work manually.  Three trials were run on the robot using BioBrick BBa_J52028(GFP). While the three trials were being performed a manual trial with the same part was done. After entering the desired volumes into Puppeteer a separate protocol displaying the total volume needed for each well was made.
The gene was cut out of the plasmid using EcoRI and SpeI.  As a cooling block was not available to contain the enzymes and buffer it was necessary to store them on ice until the robot was ready to start.  After cutting the plasmid, the parts were run on a one percent agarose gel and the outcome was positive.
1. Uncut GFP, 2. Cut manual GFP, 3. Cut robot trial 1 GFP, 4. Cut robot trial 2 GFP, 5. Cut robot trial 3 GFP
Ligation:
After successfully performing restriction digest, the next step was ligating parts together.  This was essential because the ultimate goal was to determine if a tuberculosis transcription factor could be turned off and on by the invertases.  In order to determine if it was being switched a fluorescent protein would be ligated to the transcription factors and signal the status by fluorescing or not.
As the BioBrick BBa_J23100 had a composition of RBS-RFP-Terminator within it, it was decided to cut it out of the plasmid and name it RFP composite (RFPc).  It was manually ligated with BBa_I14033(Pcat) or BBa_R0040(Ptet). Both gave good results as it fluoresced red when the DNA was transformed into Ecoli but Ptet was a better promoter.  Like the restriction digest, the correct volumes had to be entered which allowed Puppeteer to create  the code necessary to produce the correct commands on EVOware (robot’s software).  Then a set of directions were created by the robot which stated the total volumes needed from each solution and the location in which to put them.
Regular ligations with RFPc and Ptet were run on the robot along with ligations with double the volume in order to determine if the small volumes were causing the issues.  After they were transformed it was obvious that the robot was having issues pipetting 1ul of T4 ligase. The figure to the left shows the colonies for the double volume ligations.  The manual trial and the robot trials had about the same amount of colonies on the plate.  The right figure shows the colonies from the regular ligation.  There is obviously more colonies for the manual ligation than the robot ligation.
After the two parts were ligated it was essential to confirm that the correct parts were located inside the plasmid.  This was checked by doing restriction mapping on the plasmid preps of the colonies.  The following is the restriction mapping of the composite part Ptet-RFPc for one of the robot double volume ligations.
Problems
The first issue encountered when doing restriction digest was the fact the robot could only pipet whole values.  This became an issue as decimals occurred quite often for restriction digest and ligations.  The issue with decimals was fixed but it was clear that the robot could not pipet anything less than 1ul.  In order to fix this issue we had to dilute the BSA so pipetting 0.5ul would not be an issue.
At the end of the restriction digest the total amount of the final sample was off from 9ul to 12ul.  This was a big issue that was improved by changing the liquid class of the pipet from water to highly viscous liquids.  This gave the robot more time to pipet the desired liquid.
The volume issue came up again when doing ligations.  This time the problem was if the pipet could successfully pipet 1ul.  The total amount of colonies was always less from the robotic trials than the manual trials by a significant amount. There were obviously other issues, but the volume discrepancy did affect the total colonies.
Therefore double volume ligations were performed to see if the 1ul of T4 ligase was the issue.  This seemed to work properly as seen above.  There is still discretion in volumes but we realized it does not affect the outcome as much as the lack of ligase.
Future Work
Most of our project goals have been successfully accomplished at the end of summer, such as: created complete genetic devices, characterized each of their gene expression, submitted several new reporter devices to the Part Registry database, and integrated Clotho software suite along with automation software.
It is important, however, to build up from such achievements and start working on producing the prototype for a reconfigurable plasmid architecture. It will be a great step forward for synthetic biology if automation and permutation algorithm can be used to design a complex "flippable" device. That way, an interaction between various genes and/or transcription factors can be studied with a single genetic device.
In order to achieve that immediate goal, there are several future works that need to be done in our wet lab:
Design and build a simple reconfigurable genetic device
Determine the optimal amount of expression vector required to initiate plasmid reconfiguration
Utilize Trumpet to build a larger and more complex reconfigurable genetic device
Use Clotho to create genetic parts database at Boston University
Retrieved from " http://2011.igem.org/Team:BU_Wellesley_Software/Wet_Lab "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
