"
Page
Discussion
View source
History
teams
Log in
 
Team:BU Wellesley Software/G-nomeSurferPro
From 2011.igem.org
BU-Wellesley iGEM Team: G-nome Surfer Pro
Team
Project
Overview
Clotho
G-nome Surfer Pro
Optimus Primer
Trumpet
Puppetshow
eLabNotebook
Wet Lab
Downloads and Tutorials
Process
Methodology
Safety
Notebook
Outreach
Tips and Tricks
Medal Fulfillment
Additional Info
Acknowledgement
Fun
G-nome Surfer Pro
JUMP TO...
Top
Tool Overview
Demo
Results
Future Work
Tool Overview
G-nome Surfer Pro is an integrated environment for viewing prokaryotic genomic data and literature. Users can find genes on the chromosome wheel or search for a gene name or Genbank number, access Genbank notes and publications from Pubmed and view alignment and translation from the chromosome bar.  The extended desktop provides distraction free workspace for information processing and hypothesis formation. The Microsoft Surface allows users to collaborate in pairs and teams on design and research tasks.
G-nome Surfer Pro is written in C# for the Microsoft Surface using the Surface SDK . State is managed using the MVVM Architecture and time consuming processes, like retrieving publications and running BLAST, are run in threads behind the main UI thread.
After selecting a genome from the tree menu the chromosome bar is populated with genes from the GenBank file. Another genome can be selected at anytime from the pull down search menu.
The chromosome bar is infinitely scrolling and position on the chromosome bar is tied into the orientation of the chromosome wheel through the MVVM architecture allowing manipulation of one to affect change of the other. GenBank files are parsed for sequence, translation and any notes, all of which are available as information objects, which are accessible from any gene on the bar.  Users can take genes on the bar directly into the Optimus Primer surface application for primer design.
Information from any gene’s GenBank file are viewable as “ScatterViewItem” objects which can be moved and re-sized on the main “ScatterView” workspace. Users can view and annotate sequence/translation information by dragging and overlaying one over the other. All information objects can be moved to and from the extended desktop by dragging or tossing or erased by tossing them off the sides of the workspace.
Publications are populated in a scrollable list and can be dragged and dropped from the listBox to the main workspace or extended desktop through manipulation of the objects template.  The PubMed search results for the gene are fetched and parsed for abstracts, authors, and publication data.
By moving objects between ScatterViews, users can view and organize data in a distraction free environment which can be pulled up over the main workspace or hidden from view.
Publications are populated in a scrollable list and can be dragged and dropped from the listBox to the main workspace or extended desktop through manipulation of the objects template.  The PubMed search results for the gene are fetched and parsed for abstracts, authors, and publication data.
Demo Video
Results
We used G-nome Surfer Pro on the Microsoft surface to easily visualize the plasmid of TB myobacterium and choose the transcription factors we would like to use in our basic device. It also is linked to GenBank files to tell us the sequence of the gene of interest and the amino acids it codes for. In addition, we looked up more information about the transcription factors such as finding it in publications from the PubMed database.  This information was crucial to learn more about the transcription factors we were going to handle in the lab.
Future Work
Integrate with the Clotho database.
Expand annotations to include GenBank notes and publications.
Create save functionality for sessions and workspaces.
Allow users to export publications and annotations.
Retrieved from " http://2011.igem.org/Team:BU_Wellesley_Software/G-nomeSurferPro "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
