"
Page
Discussion
View source
History
teams
Log in
 
Team:BU Wellesley Software/Notebook/SiruiNotebook
From 2011.igem.org
BU-Wellesley iGEM Team: Welcome
Team
Project
Overview
Clotho
G-nome Surfer Pro
Optimus Primer
Trumpet
Puppetshow
eLabNotebook
Wet Lab
Downloads and Tutorials
Process
Methodology
Safety
Notebook
Outreach
Tips and Tricks
Medal Fulfillment
Additional Info
Acknowledgement
Fun
Kathy's Notebook
Contents
1Timeline
1.1Week of 5/31/2011
1.2Week of 6/6/2011
1.3Week of 6/12/2011
1.4Week of 6/19/2011
1.5Week of 7/4/2011
1.6Week of 7/11/2011
1.7Week of 7/18/2011
1.8Week of 7/25/2011
1.9Week of 8/1/2011
1.10Week of 8/29/2011
1.11Week of 9/5/2011
1.12Week of 9/12/2011
1.13Week of 9/19/2011
2Other Important Links:
2.1External Links
2.2Wiki Tools
Timeline
Week of 5/31/2011
Summary BU/Wellesley Synthetic Biology Bootcamp
Members of the BU/Wellesley iGEM come together and work on their "Hello World!" application in Clotho.
5/31-6/1: Team introductions; we learned the basics of CAD and basics of molecular biology and genetics.
6/2: We started learning the basics of Clotho, as well as more basic biology, focusing on the structure of a transcriptional unit.  We split into teams and started exploring the format of Clotho.
6/3: Time to get into more details: How is TB involved in our research? Suma gave a brief powerpoint presentation on the global effects of TB, and an overview on the transcriptional regulation of TB.  Also Traci talked to us more about the basics, such as the definition of a Biobrick, and objectives of the iGEM competition.  Also, she gave an overview outline on wetlab protocol for this summer.  Swapnil talked about how invertases work, and what would be a simple algorithm for computing potential permutations.
The comp team now has a good idea of what our goals are for the summer. Michelle and I will focus on observing the wet lab environment to see how a tangible user interface would potentially help research in a bioengineering lab.  We may talk to more people later on in the summer about certain features to improve on G-nomeSurfer, and adjust our goals based on the needs of the wetlab.
Week of 6/6/2011
Summary:End of training; beginning of Wet-Lab Observations and Background Research
6/8: 1st lab meeting at BU with the wet-lab.  Michelle and I spent a couple of hours in the wet-lab observing the wet-lab team.  We took notes on the organization of the lab, team interactions, and where HCI tools would come best into.
There were a couple of questions we had in mind:
How useful would publications be when the team is starting to do research?  Would they need a lot of background reading?
What potential sources would they go to for publications?  Would they prefer PubMed, GoogleScholar, or some other specific database?
Found out that publications are not the main focus at the moment.  They need to first learn about the equipment in the lab, and also how the Registry of Standard Biological Parts works.
A sample variety of BLAST visualization considered.
6/9: 1st HCI lab meeting at Wellesley- there was a general brainstorming session on BLAST visualization.  After doing some background research on existing BLAST visualization, I divided the designs into categories based on linear vs. circular format, color scheme, and content organization. The circular BLAST visualizations are interesting, and we'll look more into GView and CGViewer.  The team will keep the designs in m ind when considering how to design the paper prototype for the widget.
Work for the rest of the week: I need to start fiddling with Geneious, to see the pros/cons of the design.  What are aspects that are great that may inspire the next generation of G-nome Surfer?  What are some things that won't work on a Tangible User Interface? We need to first understand the basic functions of Geneious before we know what to tweak about G-nome Surfer.
Some members of the team are starting to develop the Extended Desktop; How do we envision this being used?
Week of 6/12/2011
Summary: Started designing widget; continued with wet-lab observations; devising a plan for BLAST visualization
6/13: Continued looking at the features of Geneious and looking up related works.  The circular chromosome design and the level of zoom in Geneious could potentially be interesting if we implement it in G-nome Surfer.  Now G-nome Surfer is called G-nome Surfer Pro because we are now focusing on E. coli chromosomes.
6/15: Wellesley lab meeting #2.  The team has divided into groups of two to work on aspects of the widget we pitched to Traci and the wet-lab.  Different people are working on the chromosome wheel, hands of the wheel, chromosome stream, and the chromosome bar.  Michelle and I will go to BU again.
Questions to consider for the widget (and to ask the BU wet-lab members):
Do we need a visualization level for the restriction sites as well?  Do we need to zoom down to the nucleotide level? If not, what components are necessary on the widget, and which are optional?
How would the wet-lab members interact with the surface?
Would it be mainly used for individual research, paired search with a post-doc, group research, or some other form of interaction we haven't considered, since there is a space restraint in the lab?
How do we look for genes?  What methods seem intuitive?  Do they want to look at specific restriction sites or genes?
How will the abstract layer work with the widget?  Do we use SBOL form for the abstract layer or not?
6/15: Wet-lab meeting #2 at BU with Suma.  The group talked briefly about transformation, and updates on what is happening in lab, and how fluorescent proteins are displayed in the samples.
A member of the wet-lab suggested that it would be cool to have barcodes for easier cataloguing.  The barcodes would go on the plastic, and ideally, after the bar codes are scanne we can use that to pull up specific protocols related to each tube with a barcode.  Michelle and I will look more into this.
6/16: Preparing for an extensive presentation on key features of Geneious.
Week of 6/19/2011
6/20-6/22: continued research on PGP (Personal Genome Project) and Prof. Church and K-nome.  Started drafting questions for the interview with Professor Church on the 22nd, as well as looking up more information on useful features of Geneious.
6/22: George Church interview with Orit and Michelle.  It was an amazing experience where I learned a lot more about related works in the area as well as the future of synthetic biology.  It was very interesting listening to Professor Church talk about the connection between “personalized medicine” and synthetic biology.  Interviewed the BU lab team about the notebook, and made further observations of the wet lab environment.  We began learning about fluorescent proteins and how they are indicators in parts.  Wet lab members were interviewed one by one after the lab meeting about improvements and suggestions for tools they think would be useful to have in the lab.  Members of the team suggested having a cataloging system for test tubes and samples, perhaps a barcode or QR code system.  Alberto mentioned putting up a timer, Kyle mentioned a personalized scheduling tool would be useful in time management, and Shannon and Evelyn suggested that a “status bar” for the progress of each “part” made would be incredibly useful.  Also, the team members would like to be able to search for specific parts at the end of the process, as well as access information on parts they are not currently working on, but are managed by other members of the wet lab.
6/23: Combined BU Comp/WetLab meeting. MIT lab meeting:
from the meeting we learned about many information sharing tools used by the MIT iGEM team to manage information.  The team used conferenceWIKI, Piazza, Stellar, Twitter, Geneious, AdGene, and PubMed extensively.  Perhaps we could look into different social media and research tools being used by other teams as well.   The team talked about having an assembly line for part creation the year before; while this emphasized parts creation, students were not able to acquire full skill sets, which is not the most educational approach to iGEM.  Instead, the tasks this year are divided between everyone doing their own wiki and are responsible for their own parts, and a few students managing specific aspects, such as media relations or parts documentation.   Suggestions from the MIT iGEM team on software to use:  QR codes for documenting parts; having “glove-only” computers;  perhaps even looking into a system that can be managed by gestures, and not by just touch; can a system be voice-activated?   Having a videocamera system so that students can see what techniques were good, and if there were mistakes made, what to improve on in the future
6/24 I did further research on SBOL, and how the form of SBOL would fit into our tool design.
6/25: everything in the widget for G-nome Surfer Pro is finally hooked together!  Now we need to implement actual sequences…
Over the weekend: I need to do some background research on AutoCAD and GenoCAD, and find information on the company Ginko, which may help us pursue the QR code idea.  Also, I need to do background research on GeneDesigner and see different existing designs for primer designers.   Furthermore, I need to look into Spectacles, and see the important features of Spectacles.  I need to start working on related works for a paper Orit is putting together! Also, need find out the size of tuberculosis plasmid backbones that may change how they are rendered on the widget.
6/27: BU comp team comes to visit.  Presented GnomeSurfer to them, as well as progress so far.  Presented paper prototype for the  notebook design.  Questions from today’s session:
Brainstorming on the LabBuddy/ electronic notebook system: where will it live?  What will it do?  Will it be an iPad/iPod/Android application? Can the notebook be monitored by the supervisor?  Can students use it as a data-collector, or as a sketchpad?   How will the information for the electronic notebook be organized? Talk about designs for PermuteTrumpet, Primer Design too!  Need to read further on invertases…  When it comes to gesture controls…. Kelsey and Consuelo are looking into Kinect.   Can we create a primer design as intuitive as the Cambridge 2010 design?  The BU team as split into teams, with Janoo on Puppetshow/Notebook, Craig on invertase/ Permute Trumpet design, Jenhan on a primer designer, and Chenkai on working with the wet lab in using the robot.  Now we know who to contact if we have specific questions for each part!
6/28: Teachers iGEM meeting Realized that Tinkercell may be used extensively for teaching in introductory biology classes, such as junior high or high school biology classes.  Need to do further research on related works besides Tinkercell that help with simple visuals. Can we make a tool that can be used by citizen scientists as well as undergraduates?  It’s an interesting thing to keep in mind
6/29: Lab meeting #4 at BU. Primer design overview with Suma.  We harvested interesting information about primer designing works.  Suma mentioned that a really useful software for designing primers was ApE.  I will spend the next few days fiddling with ApE to see how it handles restriction sites, and also organization.  Can we design something along the lines of ApE, but for a tangible user interface?
Review protocol with Traci.  We saw that each lab member presented their results and data to Traci individually, but there wasn’t a lot of group interaction involved.  What if the electronic lab notebook had a conference sharing/ powerpoint presentation-typed feature that allow the entire lab to exchange data?  That would be pretty cool!      Wet lab observations at MIT.  We interviewed iGEM members there about how the system works:  how do they do background research on parts, how do people interact in the lab, what is the physical set up of the lab, and what they would want improved in their lab. The MIT team has a little more emphasis on presentation and research of background information before the division of labor even starts; can we design the tool to cater to their need for a strong database to get background research from?
6/30: Began designing a main menu for updating wetlab observations.  Michelle and I start doodling on the board of idealized things we would want in the lab notebook.  Sky’s the limit!
7/1:Try to catch up on the notebook.  Also, work on related works, and try to figure Gene Designer and Tinkercell.  Also, record innovations of the LabBuddy in the notebook.  Work on related works.
Related work list NOW:  update the powerpoint: more on Tinkercell, GenoCAD, AutoCAD, figure out GeneDesigner, ApE, SBOL, Spectacles try to think of a scenario for using the tool, and write section on related works.
Week of 7/4/2011
Members of the Wellesley HCI lab testing out the Surface.
Continue designing notebook application
7/4: no research today; 4th of July!  Worked instead on the Lab Notebook.
7/5: Lots happened today:
Today the other half of the team presented in the Science Center CS Lecture Series.  The first group presented last week Tuesday on inspirations and brainstorming ideas for the design process.  This week, the lab presented on designs and future plans for beyond iGEM.   We continued designing the LabBuddy/ electronic notebook application.  Consuelo and I decided that instead of a checklist or a protocol list, a flowchart of some sort with annotations on the side could work, allotted that there is room for figures and the notes can be printed separately later.   Something also to consider is a function that displays lab reminders from Traci, Suma, and James, a “training mode” in protocols, that could help prevent wet lab errors and increase productivity.   Also, I did further background research on related works, either in the LabBuddy application or frequently-used tools in synthetic biology.  There are a few tools that are already in use for note taking in an iPod or iPad setting, but they are not specifically for lab usage, and are fairly pricy.
7/6: Continued writing the user scenario- should eventually incorporate all aspects of the GnomeSurfer Pro suite in the content Continue updating aspects of the lab notebook- what kind of pictures are needed?
Continue doing related works ppt and adding features 7/7-7/8:The research for related works continues still!  Michelle and I are refining the scenario for presentation to the BU team.
Week of 7/11/2011
7/11-7/12:  Michelle and I continue working on the user scenario.  Monday night we both went to a writing workshop, and used information compiled from all the interviews that we learned to continue writing the usability scenario.
7/13:  Kelsey, Michelle, Megan, and I sat down and worked on visuals for a primer design.  Kelsey and Casey ended up working on the primer design.  Since Orit gave us a new copy of the scenario to edit, Michelle and I ended up working on explaining a user scenario for the primer designer how is using a primer designer different for the MS Surface than for a computer?  How can collaboration techniques vary?  How will the team approach this project?  What kind of visuals would we need for the primer designer?
7/14: Went to BU for a joint meeting, and presented our designs (so far) to BU.  Learned more about enzyme interaction, and will do further readings about the biology behind enzyme interaction.  Continued doing related works with citations.  Met Avi Mosher today from the Weiss Institute, who gave us interesting ideas about how graduate level biology research is done, and how our designs would be used in a more grad-school level research environment.  This is something to keep in the back of our minds when we’re designing for the future.  Can we use the tool to do research and find out information on genes that were previously un-researched?
7/15: Finished the first draft of related work.  Need to figure out more information about citations, and also find other related works and streamline the writing!  Also, we need to start thinking about presenting a poster for the science center poster session.
Week of 7/18/2011
7/18: Continued editing the scenario, this time with notes from Swapnil and Traci.  Started thinking about people to recruit for the usability study: time to “stalk down” some biology/neuroscience students from around the science center…
7/19: Worked on abstracts for the science center summer research grant.    Continued recruiting people for the usability study, and work on the scenario.  Started formulating questions to ask neuroscience students, who have used a previous iteration of G-nome Surfer in the past.
7/20:Crafted a survey to ask neuroscience students who are off campus about their previous experience with G-nome Surfer for the paper.  Also, prepared for the interviews for tomorrow!!
7/21: Today we interviewed two neuroscience students about how they used G-nome Surfer in an intermediate neuroscience class.  Since I took this class before, Megan and I both interviewed these students.  They were a wealth of knowledge and really, really useful information!  We had a better idea of what to expect from designing a usability scenario after interviewing these students. 
7/22: started doing research on particular genes for the pilot testing and usability studies with biology students next week, while the rest of the lab did transcripts for the two interviews.  Should we bring in a teacher to be interviewed as well?
Week of 7/25/2011
This was a crazy week!  We managed to recruit 8 Wellesley biochemistry and biology students for usability studies on Wednesday and Thursday.  The students came in pairs in the afternoon, and learned how to use the G-nome Surfer Pro software after a quick tutorial on designing primers and navigating the MS Surface.  Most of the students have moderate to strong genetics backgrounds, and seemed to really enjoy using the tool!
A few students suggested for us to interview Professor Suzuki in the biology department, a developmental biologist who would have great ideas for improving the system.  We will do that, after we test the software tool at BU on Monday.  We’re taking the surface to BU!
Week of 8/1/2011
One of many iterations of G-nome Surfer Pro.
Deployment of G-nome Surfer Pro at BU photonics center for the wet lab members.  The students worked in groups of threes, while Consuelo, Heidi, Taili and I observed their interactions.  There are still a lot of kinks to work out on this iteration; sometimes GenBank notes don’t work, and other times the PubMed are blank!  Also, the extended workspace is confusing some students these are all small things to focus on for future designs.
8/2: interview with Professor Suzuki it was incredibly useful and Professor Suzuki was a wealth of information!  Consuelo and I learned about model organisms in developmental biology, and how a professor doing research would want the software to be different for learning and personal research.  Definitely a highlight of my summer research experience- having a professor say our software was really cool and potentially useful!  Perhaps we can one day even design it for seminars and for research for student theses?
8/3-8/4: Worked on designing the front-end of PermuteTrumpet, while the rest of the lab put together the posters for summer research.  We have THREE posters this year- a GreenTouch one, one on iGEM, and one on Trumpet and the eLabNotebook and future plans.  Now we have to practice for the presentations on Thursday!
8/4: Presentations!  Also, the BU comp team came over and we did brainstorming on things to do between the end of summer and the beginning of the school year.  Each member was given tasks to do for the website.
Week of 8/29/2011
8/30: first HCI lab meeting of the semester! :D What are issues we still need to address?  How’s the website coming along?  Are we thinking about the presentation yet?  How did the results for the usability studies go?
Week of 9/5/2011
Started finalizing the design for the website, after doing brainstorming.  Over the week of the 9th to the 12th we will focus on polishing the website and working on the brainstorms for the presentation itself.  Is there an overarching theme of some sort that we can focus on?  Perhaps the iterative science research process?
Week of 9/12/2011
Worked on making tutorial videos for the eLab Notebook prototype and also Optimus Primer. Somehow the format for the videos isn’t compatible with the wiki will figure this out.
Week of 9/19/2011
Wiki freeze on Wednesday! AAAHHH
9/26: methodology page was updated, with focus on user-centered design.
9/27: final touches on the wiki.  We started ordering team t-shirts for iGEM.  WIKI-FREEZE at midnight!
Other Important Links:
External Links
Follow BU_Wellesley IGEM Team on Twitter!
Registry of Standard Biological Parts
BU_Wellesley IGEM Lab Notebook
BU_Wellesley IGEM Home
Wiki Tools
Upload Files
Retrieved from " http://2011.igem.org/Team:BU_Wellesley_Software/Notebook/SiruiNotebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
