Team:Hong Kong HKUST/Sensing
HKUST iGEM Team 2017
Team Hong Kong - HKUST
Project
Project Overview
Sensing Module
Time Control Module
Recombination Module
Possible Future Application
Modelling
Single-cell
Model-Sensing
Model-Time Control
Model-Recombination
Population
References
Human Practices
Human Practices (Silver)
Integrated Human Practices
Education & Public Engagement
Medal Requirements
Collaborations
Interlab
About Us
Attributions
Our Team
Logbook & Protocols
Safety
Giant Jamboree Photo Gallery
Sensing Module
Overview / Rationale
In order to control the expression in a population of cells, it is essential that the signal can propagate from one         cell to another. It is therefore, important to consider the diffusion of signals. Signals should be able to pass         through cell membrane and cell wall, from cytoplasm to the outside environment, and vice versa. Adding a positive         feedback loop can enhance the transmission of the signal through the population
To ensure the knockout process occurs in every single cell, we select the diffusible signals that a bacterium can produce         quickly. This inducer is the 3O6CHSL, or AHL molecule.
Sensing Module Construct
Diagram 1. Sensing Module Construct
Positive feedback loop mechanism
Diagram 2. Positive feedback loop mechanism
Experimental Construct
Diagram 3. Experimental Construct
Our Design
The function of the sensing module is to detect the deliberately released stimulus and initiate the process of “knockout”.         Considering that the released stimulus can become diluted in an environment, a positive feedback loop is included         to amplify the signal. Thus, 3OC6HSL, which is a member of acyl-homoserine lactone (AHL) family, was chosen as the         inducer. 3OC6HSL, originally from V. fischeri, is a lipid molecule that can diffuse through bacterial cell         membrane, facilitating cell-to-cell communication.
The sensing module begins with the TetR repressible promoter or pTetR (BBa_R0040), which can be treated as constitutive         promoter of LuxR protein (BBa_C0062) under no repressor TetR. This segment of the biosensor works to produce an abundance         of LuxR. Once the 3OC6HSL is added to the cell environment, LuxR forms a complex with 3OC6HSL and then activates         downstream promoter, pLuxR (BBa_R0062).
After the activation of pLuxR promoter, LuxI protein (BBa_C0061) then expresses. LuxI is an autoinducer synthetase         which catalyzes 3OC6HSL from S-adenosyl-L-methionine (SAM) in cell. The entire part from pTetR to LuxI generates         a positive feedback loop, because it increases the concentration of 3OC6HSL/LuxR complex and hence induces pLuxR more strongly. The 3OC6HSL molecule can also diffuse back to the extracellular environment and affect the nearby         cells.
With its positive feedback loop, the sensing module is able to convert from signal receiver into signal emitter whenever         it receives 3OC6HSL molecule. Thus, it is expected to increase the efficiency of activation in a targeted environment.
Potential Issue
The issue that previous iGEM team encountered (See experience in BBa_F2620) is the leakiness of pLuxR promoter             (BBa_R0062). It is a situation when there is little expression of gene even without initial activation of AHL.             This time we improved by adding sequences of antisense RNA Binding regions and antisense RNA in which their interaction             can inhibit the translation of excess mRNA.
First, the antisense RNA Binding Region (ABR) is placed right before our targeted Ribosomal Binding Site (RBS),             which is upstream to the LuxI (BBa_C0061) and GFP (BBa_E0040). Second, the antisense RNA is placed downstream             the positive feedback loop and its reporter together with a medium strength promoter, pLuxL (BBa_R0063).
The antisense RNA we use has two important characteristics that will help reduce leakiness. Firstly, it comprises               of sequence that is complementary to the sequence of mRNA of ABR. The affiliation of antisense RNA to the complementary               ABR prevents ribosome from binding to the mRNA of the targeted RBS. Secondly, it has a Hfq binding site, which               is suspected to recruit RNase to degrade the targeted RNA chain. (Wagner, 2009)(Hoynes-O’Connor & Moon, 2016).               In consequence, translation of LuxI:GFP mRNA will be reduced, and so will the leakiness of pLuxR.
To see the theoretically expected behavior of the construct with and without asRNA, please refer to our model
Experiments & Results
Introduction & Background
To investigate whether our antisense RNAs can reduce basal level expression, we ligated GFP (BBa_E0240) in each               construct, and measured the fluorescence/OD600 unit produced: pSB1C3-BBa_T9002 (Annotated as w/o PFB): Plasmid               without positive feedback loop
Diagram 4. pSB1C3-BBa_T9002
pSB1C3-BBa_F2620-C0261-E0240 (Annotated as w/ PFB): Plasmid with positive feedback loop due to insertion of LuxI               gene (BBa_C0261)
Diagram 5. pSB1C3-BBa_F2620-C0261-E0240
pSB1C3-BBa_K2240000 (Annotated as as PFB + asRNA1): Plasmid resulted from insertion of antisense type I and antisense               binding region complementary to antisense RNA type I into the positive feedback loop system (pSB1C3-BBa_F2620-ABR1-C0261-ABR1-E0240-R0063-Anti1).               This part has GFP expresses in the construct.
pSB1C3-BBa_K2240003 (Annotated as as PFB + asRNA2): Construct with Anti II; Plasmid resulted from insertion of               antisense type II and antisense binding region complementary to antisense RNA type II into the positive feedback               loop system (pSB1C3-BBa_F2620-ABR2-C0261-ABR2-E0240-R0063-Anti2). This part has GFP expresses in the construct.
Diagram 6. pSB1C3-BBa_K40000 or pSB1C3-BBa_K40003
Antisense RNA reduces basal expression level
We first characterized the pSB1C3-BBa_T9002 (w/ PFB) to find suitable time for induction, which we found that fluorescence/OD600 can reach over than 600,000 GFP/OD600 after more than 3 hours. Thus, we used control variables               of 3 hours and [AHL] = 1.00E-05 M as an optimum level of induction.
Diagram 7. Characterization of pSB1C3-BBa_T9002
Results
AHL[M]
All constructs comparison
Antisense RNA comparison
0 M
Fig. 1 Error bar presents SD from 6 biological replicates.
Fig. 2 Error bar presents SD from 6 biological replicates.
1.00E-05 M
Fig. 3 Error bar presents SD from 6 biological replicates.
Fig. 4 Error bar presents SD from 6 biological replicates.
Discussions
Antisense RNAs can reduce basal expression level
The efficiency of basal level reduction for antisense RNA type I and II are compared prior to AHL induction ([AHL]=0M)               and after the induction ([AHL] = 1.00E-05 M) Unpaired t-test analysis suggests that the differences between               w/PFB and PFB+ asRNA2 and the difference between w/PFB and PFB+asRNA2 could be considered as highly statistically               significant.
Antisense RNA type II reduces basal level more significantly
To visualize the difference between antisense type I and type II better, Fig. 2 and 4 were illustrated. Using               unpaired t-test calculation, it suggests that there are significant differences between antisense RNA type               I and type II both before and after induction with 1.00E-05 M AHL, where Fig. 2 and 4. imply that construct               with antisense RNA type II can reduce basal expression level more significantly. This may be due to higher               GC content of its complementary sites (55% for asRNA type II comparing to 47.4% for asRNA type I) according               to our first hypothesis.
Positive Feedback Loop works after AHL induction
There are also notable statistical differences when comparing the changes before and after induction using two-tailed               paired t-test analysis for both PFB+asRNA1 (p <0.001) and PFB+asRNA2 (p<0.01). The average of fluorescence/OD600 for PFB+asRNA2 is 4,199.18 while the average after its                 induction is 6,121.58, showing an increase for around 2,000 GFP/OD600 (45.8% growth) after 3 hours. Whereas,                 PFB+asRNA1 also experiences an increase by 4,765 GFP/OD600 (65.7% growth) under the same controlled time (Fig. 5),                 suggesting that antisense RNA regulates translation inhibition more tightly. This supports our second hypothesis                 that there should be an increase in expression after induction by AHL.
Antisense type I (PFB+asRNA1)
Antisense type II (PFB+asRNA2)
Fig. 5 Error bar presents SD from 6 biological replicates.
Fig. 6 Error bar presents SD from 6 biological replicates.
Conclusions
Our antisense RNAs construct can verify the following hypotheses:
There is a significant decrease in basal expression level when antisense RNA type I and II are used.
Antisense RNA type II reduces basal level expression more efficiently.
Positive feedback loop remains function after activation by 1.00E-05 M of AHL after 3 hours.
Suggestions
Antisense RNA sequences can be further fine-tuned to investigate higher efficiency of basal level reduction. Insertion of a stabilizer element would increase antisense RNA’s half-life (Engdahl, 2001).
Higher concentration of AHL can be tested with PFB+asRNA1 and PFB+asRNA2 to investigate their effects on expression level in positive feedback loop.
The effect of AHL positive feedback loop can be tested in population dynamics
Additional comments/Notes
W/ PFB sample increases the expression by only 4,316.5 gfp/OD600 when induced by 1.00E-05 M AHL after 3                     hours, at the same time which fluorescence of w/o PFB increases by 10 folds.
Fig. 1 and 3 show that w/o PFB produces the highest fluorescence/OD600, while W/ PFB is significantly lower.                     This has been experienced by brown iGEM team 2007, where their hypotheses have been proven false due                     to an inverse relationship of GFP expression between the control (no feedback loop) and the one with                     LuxI (Positive feedback loop). Given that the concentration of SAM is constant, the unpredictable lower level of fluorescence may be due to high concentration of AHL and low concentration of luxI that cause the shift in equilibrium to the reactant (SAM) rather than to the product side. However, in our course of experiment, we were interested in the efficiency of two types                     of antisense RNA in reducing basal expression level, so we took W/ PFB construct for the comparison.
We have considered tuning pLuxR promoter instead of adding ABR and antisense RNA. However, we predicted                     that there will be a tradeoff. If the promoter is tuned to express low level of LuxI, that will minimize                     AHL production. Thus, the leakiness will be reduced. However, that means the impact of AHL diffusing                     across cell membrane will be impeded given that AHL concentration inside the cell affects the diffusion                     rate across cell membrane.
The adjustment of pLuxR promoter should satisfy the criteria that:
It will not constitutively express until LuxR:AHL complex induces
It can produce high LuxI product which catalyzes high concentration of AHL once induced.
Apparently, mutation of promoter to lower the leakiness cannot compensate the second criterion. Thus, the                     second suggestion for improvement is to use Gram-positive bacteria as a chassis.
CUHK's characterization
We sent our parts to CUHK to characterize and validate our sensing module’s constructs for the effect of                     antisense RNA on the basal expression level. However, their results are not consistent with our results.                     Because there was no increase in fluorescence/OD600 for our positive control I, we omit their results                     in our data processing and interpretation.
References
Andrew Camilli (2006) Bacterial Small-Molecule Signaling Pathways. Science. 311(5764), 1113–1116. doi:  10.1126/science.1121357
Brooke A. McDaniel (2006) Identification of a Mutation in the Bacillus subtilis S-Adenosylmethionine Synthetase Gene That Results in Derepression of S-Box Gene Expression.J Bacteriol.188(10), 3674–3681. doi:  10.1128/JB.188.10.3674-3681.2006
Engdahl HM (2001), Introduction of an RNA stability element at the 5'-end of an antisense RNA cassette increases the inhibition of target RNA translation. Antisense Nucleic Acid Drug Dev 11(1), 29-40. Doi: 10.1089/108729001750072100
Daniel J. Sayut (2007) Noise and kinetics of LuxR Positive feedback loops, Biochemical and biophysical Research Communities, 363(3), 667-673, doi: https://doi.org/10.1016/j.bbrc.2007.09.057
Hoynes-O’Connor, A., & Moon, T. (2016). Development of Design Rules for Reliable Antisense RNA Behavior in E. coli. ACS Synthetic Biology, 5(12), 1441-1454. http://dx.doi.org/10.1021/acssynbio.6b00036
Wagner, E. (2009). Kill the messenger: bacterial antisense RNA promotes mRNA decay. Nature Structural & Molecular Biology, 16(8), 804-806. http://dx.doi.org/10.1038/nsmb0809-804
Follow Us Now !!!
SiteMap
© iGEM 2017 Team HKUST. All rights reserved.
made with ♥ @ HKUST
