Team:Hong Kong HKUST/HP/Gold Integrated
HKUST iGEM Team 2017
Team Hong Kong - HKUST
Project
Project Overview
Sensing Module
Time Control Module
Recombination Module
Possible Future Application
Modelling
Single-cell
Model-Sensing
Model-Time Control
Model-Recombination
Population
References
Human Practices
Human Practices (Silver)
Integrated Human Practices
Education & Public Engagement
Medal Requirements
Collaborations
Interlab
About Us
Attributions
Our Team
Logbook & Protocols
Safety
Giant Jamboree Photo Gallery
Integrated Human Practices
MOTIVATION
As part of our project brain-storming process, we had been looking through some of the previous projects for inspiration. However, when we start looking into how teams tackle the problem of their project safety, we realized that most teams uses kill switch as their genetic containment strategies. However, there are no practical implementation and standardization of the strategy.  We wanted to create a universal safety switch that teams can easily implement it into their project in the future. We were inspired by the application of recombinases in eukaryotes where only a specific gene inside the cell will be knocked out without harming the cell itself. As a proof-of-concept, we therefore uses the Cre-lox system to illustrate how recombinase can excise the gene of interest in the DH10b strain of E.coli bacteria that are easy to grow and safe to use in the laboratory.
Having received feedbacks from people joining our organized activities in Hong Kong Science Park, where the majority is parents,                             it surprisingly shows that our samples view GMOs to be more safe if the genetically modified                             gene within an organism has been knocked out before the consumption. We, therefore, directed                             our focus on Genetically Modified Organisms (GMOs) with the addition of time delay controlling                             the expression of gene of interest for safety reason.
1
TOWARDS OUR APPLICATION
Feedbacks from the participants in our workshops surprisingly shows that the public view GMOs, specially food to be more                             safe if the genetically modified gene within an organism has been knocked out before consumption.                             We, therefore, directed our focus on Genetically Modified Organisms (GMOs) with the addition                             of time delay controlling the expression of gene of interest to satisfy the public’s concern.                             Time control mechanism can allow the genetically modified crops to grow under extreme conditions                             to increase crop yield while having this foreign gene removed before reaching the consumers so                             that these crops will be considered safer in the public’s point of view.
The introduction of GMOs, however, varies between countries in terms of regulation and stringency. In order to explore the                             situation of GMOs and their products in Hong Kong, we had contacted and interviewed with Dr.                             Terence Lau, a biosafety expert from the Hong Kong government. Dr. Lau shared with us about the                             current regulation and control of GMOs. To our surprise, there is yet no measure in preventing                             or mitigating the risks of GMOs being accidentally released in Hong Kong. The government only                             emphasizes on the control of genetically modified organism in some products such as vaccine and                             GM crops where the Centre for food safety had provided guidelines on voluntary labelling of Genetically                             Modified (GM) Food. In addition, industries will take risk to benefit ratios into account before                             considering GMOs as their profit-based business. Throughout this interview, the issue raised                             has inspired us to pursue deeper into the application of Genetic Containment Strategy.
2
WHAT IS GENETIC CONTAINMENT STRATEGY? HOW IS IT DIFFERENT FROM BIO-CONTAINMENT?
Genetic (Biological) Containment Strategy is a measure to safeguard recombinant DNA in microorganism such that it can only                             be used in lab experiment but not outside. If accidental release happens, growth of modified                             microbes and rDNA replication should be immediately inhibited.
On the other hand, biocontainment is a prevention of risk of genetically engineered microorganism (GEM) release by using                             physical containment.
Hong Kong laboratories mainly relies on biocontainment. But due to the demand for scaling up GEM,                             biocontainment may not be sufficient to contain large scale of GEM. Thus, we propose the use                             of genetic containment strategy as genetic safeguards can potentially prevent the escape of microbes                             from proliferating. 1
3
THE SENSOR'S POSITIVE FEEDBACK LOOP
Dr. Lau not only shared with us some knowledge regarding biosafety. He also raised questions concerning how we can achieve                             the purpose of knocking out in large population of cells.
We decided on using feedback loop to amplify signals such that sufficient receiver cells can be knocked out at time bound.                             The propagation of AHL signals can be simulated by modeling the rate of diffusion from the senders                             to the receivers and the rate of AHL being amplified again to affect peripheral cells. The positive feedback loop we found involves in the use of luxR cassette in E.coli. We characterized and found that there's basal expression level under no induction. We, therefore, addressed the issues of promoter leakiness and basal level expression through the use of antisense RNA based on our modelling analysis. Click to see our Modelling Page.
4
Feedbacks from conference (2017 iGEM Pan-Pearl conference organized by the CUHK 2018 team)
Q1: Is there a chance where random homologous recombination occurs between your recombinant plasmid and bacterial genome                             inside the cell?
Q2: Is there any enzymes that can digest your engineered plasmids instead of using these recombinases?
Q3: How can you eliminate the plasmid after it’s recombined?
Q4: How would you use your construct in particular scenarios. For example, how will your construct help if you accidentally                             spill the GM bacteria into environment?
5
ADDRESSING ISSUES
In addition to these feedback questions, we looked more into details of our constructs where improvement can further be made                             to reduce the safety issues. We conducted an interview with Prof. Matthew Bennett from Rice University                             in Houston, who is an expert in organism’s genetic system and circuits.
Q1: It was found that even though luxR sequences do not appear in E. coli in nature,                             there is a homologous gene found in E. coli genome called sdiA. The luxR gene, a family of sdiA gene, shares homologous sequence such that random recombination                             between the plasmid and genome may occur many times, although not frequent. We address this problem                             by finding a chassis that has the sdiA gene being knocked out. See strain JW1901-5 2
Q2: Prof. Matthew Bennett commented that endonucleases may be more efficient in targeting multiple restriction sites and                             we have explored some more nucleases that could do the job of DNA degradation. One efficient                             method to target DNA degradation is the use of CRISPR-Cas3 device, where CRISPR nuclease specifically                             degrades the targeted DNA, leaving the non-targeted DNA unaffected 3 There is also a nuclease that can be transiently transcribed in vivo such as Yeast homothallic                             switching endonuclease (HO endo) which recognizes HO recognition sites and cleaves DNA to become                             linearized 4.
Q3: As our project describes a method that uses recombinase to separate between the gene of interest and the plasmid containing                             the origin of replication and marker gene, we use modelling to simulate a scenario to measure                             the dilution of plasmids and how long the recombined plasmid can be degraded over time.
Nevertheless, after the interview and through further research, we conjectured that a more effective way to digest recombinant                             plasmid is through the use of endonuclease. Apart from considering endonucleases such as a DNAi                             device called CRISPR-Cas3 as a choice for further improvement in our project, our team has continued                             to find potential application that recombinase can be compatible with (See Application page).                             We also designed plasmid simulation where CRISPR-Cas3 replaces Cre-recombinase8,                             but we were not able to incorporate into our project due to the time limitation.
The system works by expressing Cas3 and CasABCDE elements that will complex together and are guided                             by CRISPR RNAs (crRNAs) with its spacer which will recognize PAM sequence correspnding with their                             complementary protospacer. The DNAi device will then cleave double stranded DNA, which will be                             degraded by exonuclease activity. The figure below shows the construct design (left and bottom)                             and how Cas3, CasABCDE and crRNAs target their sites (right)
We investigated the pros and cons of using 3 different knockout methods such as Cas9, Cas3 and recombinase,                             by listing out in the following tables
Cas9
Cas3-CasABCDE
Cre recombinase
Characteristic
Genome editing at specific interval
Remove the gene beyond the point of repair like DNA shredder
Genome editing at specific interval; Knockout genes in tissue-specific and time-controlled                                         fashion (1)
Efficiency
Depends on chassis:Up to 45% in iPSC cells (2)
Dependent on number of plasmids and spacers (3)
Variable in mouse strains (4)
Advantages
- Target different genome/plasmid with different crRNA design
- Works in all organisms
- Host cells kept intact
- Can trigger bacterial self-destruction
- Future application                                         on antibiotic resistance such as MRSA
-Host cells kept intact
- No cross recombination with sites that are not loxP (5)
-                                         Good temporal or tissue-specific control of genetic modification
- Knockout efficiency                                         is independent of target DNA length (6)
- Off-target not shown in this type of                                         chicken (7)
Limitations
- Different Cas9 homologs require different PAM sequences
- Mosaicism in mice (8)
-                                         Large fragment deletion is achieved at low efficiency (less than 10%) (9)
- Not a site-specific genome editing
- defective CRISPR
- Organism may tolerate                                         PAM sequences (PAM sequences should be properly designed)
- More targetable spacer:                                         Use more flexible PAM sequence
- Reduce off-target effects: Use more stringent                                         PAM sequence (10)
- Off-target effects
- Not yet experimented in mammal cells                                         due to large size
- Can only target bacterial cells because cas3 is too big to                                         enter mammalian cells (11)
- Mutations in target sequence
- Knockout efficiency is dependent of target DNA length                                         (12)
- Recombine sites that are a family of loxP e.g. LoxP, mLoxP
- Pseudo-loxP                                         sites may present in mammalian genome, leading to aberrant activity (13). Must be                                         careful on selecting strain
Common limitations among CRISPR
- Mutated target sequence (14)
- defective CRISPR
- Organism may tolerate PAM sequences                                         (PAM sequences should be properly designed)
- More targetable spacer: Use more                                         flexible PAM sequence
- Reduce off-target effects: Use more stringent PAM sequence                                         (11)
- Off-target effects
References:
(1) Loonstra, A., Vooijs, M., Beverloo, H. B., Allak, B. A., van Drunen, E., Kanaar, R.,                             Berns, A., & Jonkers, J. (2001). Growth inhibition and DNA damage induced by Cre recombinase                             in mammalian cells. Proceedings of the National Academy of Sciences, 98(16), 9209–9214. doi:                             10.1073/pnas.161269798
(2) Liang, X., Potter, J., Kumar, S., Ravinder, N., & Chesnut, J. D. (2017).                             Enhanced CRISPR/Cas9-mediated precise genome editing by improved design and delivery of gRNA,                             Cas9 nuclease, and donor DNA. Journal of Biotechnology, 241, 136-146. doi: https://doi.org/10.1016/j.jbiotec.2016.11.011
(3) Caliando, B. J., & Voigt, C. A. (2015). Targeted DNA degradation using a CRISPR device stably                             carried in the host genome. Nature Communications, 6, 6989. doi:10.1038/ncomms7989
(4) Garcia-Arocena,                             D. (2013). Cre-lox Myths Busted. The Jackson Laboratory. Retrieved from https://www.jax.org/news-and-insights/jax-blog/2013/september/cre-lox-myths-busted
(5) Anastassiadis, K., Fu, J., Patsch, C., Hu, S., Weidlich, S., Duerschke, K., Buchholz, F.,                             Edenhofer, F., & Stewart, A. F. (2009). Dre recombinase, like Cre, is a highly efficient site-specific                             recombinase in E. coli, mammalian cells and mice. Disease Models & Mechanisms, 2(9-10), 508-15.                             doi: 10.1242/dmm.003087
(6) Ullrich, S., & Schüler, D. (2010). Cre-lox-Based Method for Generation                             of Large Deletions within the Genomic Magnetosome Island of Magnetospirillum gryphiswaldense.                             Applied and Environmental Microbiology, 76(8), 2439-2444. doi: 10.1128/AEM.02805-09
(7) Leighton,                             P. A., Pedersen, D., Ching, K., Collarini, E. J., Izquierdo, S., Jacob, R., & van de Lavoir,                             M.-C. (2016). Generation of chickens expressing Cre recombinase. Transgenic Research, 25(5),                             609–616. doi: 10.1007/s11248-016-9952-6
(8) Yeadon, J. (2014). Pros and cons of ZNFs, TALENs,                             and CRISPR/Cas. Pros and cons of ZNFs, TALENs, and CRISPR/Cas. The Jackson Laboratory. Retrieved                             from https://www.jax.org/news-and-insights/jax-blog/2014/march/pros-and-cons-of-znfs-talens-and-crispr-cas
(9) Song, Y., Lai, L., & Li, Z. (2017). Large-scale genomic deletions mediated by CRISPR/Cas9                             system. Oncotarget, 8(4), 5647. doi: 10.18632/oncotarget.14543
(10) Larson, M. H., Gilbert, L. A., Wang, X., Lim, W. A., Weissman, J. S., & Qi, L. S. (2013).                             CRISPR interference (CRISPRi) for sequence-specific control of gene expression. Nature Protocols,                             8(11), 2180-2196. doi: 10.1038/nprot.2013.132
(11) Buhr, S. (2016). Move over Cas9, CRISPR-Cas3 might hold the key to solving the antibiotics                             crisis. TechCrunch. Retrieved from https://techcrunch.com/2016/12/21/move-over-cas9-crispr-cas3-might-hold-the-key-to-solving-the-antibiotics-crisis/
(12) Coppoolse, E. R., de Vroomen, M. J., van Gennip, F., Hersmus, B. J. M., & van Haaren,                             M. J. J. (2005). Size Does Matter: Cre-mediated Somatic Deletion Efficiency Depends on the Distance                             Between the Target lox-Sites. Plant Molecular Biology, 58(5), 687-698. doi:10.1007/s11103-005-7705-7
(13) Loonstra, A., Vooijs, M., Beverloo, H. B., Allak, B. A., van Drunen, E., Kanaar, R.,                             Berns, A., & Jonkers, J. (2001). Growth inhibition and DNA damage induced by Cre recombinase                             in mammalian cells. Proceedings of the National Academy of Sciences, 98(16), 9209–9214. doi:                             10.1073/pnas.161269798
(14) Bikard, D., Euler, C., Jiang, W., Nussenzweig, P. M., Goldberg, G. W., Duportet, X.,                             Fischetti, V.A., & Marraffini, L. A. (2014). Development of sequence-specific antimicrobials                             based on programmable CRISPR-Cas nucleases. Nature Biotechnology, 32(11), 1146-1150. doi: 10.1038/nbt.3043
(15) Larson, M. H., Gilbert, L. A., Wang, X., Lim, W. A., Weissman, J. S., & Qi, L. S. (2013).                             CRISPR interference (CRISPRi) for sequence-specific control of gene expression. Nature Protocols,                             8(11), 2180-2196. doi: 10.1038/nprot.2013.132
Q4: The desirable design for eliminating bacterial spill is to immediately knockout the construct                             containing GM gene without the delay of recombinase or endonuclease expression. Thus, one question                             raised would be ‘When and in which scenario should time delay be used?’ if time delay is not                             used in the accidental release of GM bacteria.
Time delay can be useful in laboratory experiments where GM bacteria should not last for so long                             time. This does not require modification of bacteria having genetic code dependent on some nutrients                             to survive nor require the activation of toxin killing GM organism in the culture. Thus, safer                             extraction of proteins (both secreted and non-secreted) can be achieved with no contamination                             of toxic compounds. Additionally, we would not need to worry about any changes in the organism’s                             genomic content..
6
♡
❀
FUTURE SUGGESTIONS
Suggest for an alternative way of resolving the leakiness of promoters,pluxR
We suggest that expressing our construct in gram-positive bacteria may avoid leaky expression                                 due to an auto-activation of pluxR. Gram-positive bacterium has SAM molecules that are conserved                                 in both gram-positive and gram-negative species. However, gram-positive bacteria do not use                                 the same communication pathway. It employs two component systems called oligopeptide auto-inducers,                                 instead of AHL molecules (luxI from Vibrio fischeri produces AHL with a distinct                                 side chain that can’t be found in Gram positive bacteria)6.
Thus, we may try to express pluxR promoter along with luxI synthases within the gram-positive bacteria such as Bacillus subtilis and test whether there is a reduction in promoter leakiness in such chassis.
Suggestion for more specific recombination
While Cre-loxP system allows                                     the study of spatial and temporal gene expression, CRISPR/Cas9 can target specific site                                     to mediate gene editing. According paper published in 2017,  CRISPR/Cas9 was shown to be able to work with loxP for the first time. This has an application on site-specific genome editing in human cells where using loxP mutant that contains sgRNA could allow a selective gene excision. 7
Nevertheless, future improvements need to be made. Fayu Yang and his research team suggested that loxP66 and loxP71 mutants (which we also employed in this iGEM project) may be more suitable for CRISPR/Cas9-loxP design because they contain PAM sequences which sgRNA can recognize.
Thus, this system may enhance effectiveness in knockout performance inside bacterial cells if two or more external genes are to be inserted and knocked out at the same time. Knocking out long sequences of target gene such as sensing and time-delay module may be done more easily especially when Cre recombinase fails to recognize lox site. Due to time limits, we could not explore more in details so we consider this method as our future suggestions and possible improvement for our project
7
REFERENCES
Karmella A Haynes (2016) Synthetic biology: Building genetic containment. Nature Chemical Biology, 12, 55–56. doi:10.1038/nchembio.2004
E. Coli Genetic Resources at Yale CGSC, The Coli Genetic Stock Center  (n.d.) Retrieved from: https://cgsc2.biology.yale.edu/Strain.php?ID=107863
Calianda BJ (2015) Targeted DNA degradation using a CRISPR device stably carried in the host genome. Nature Communications 6, 6989. doi :10.1038/ncomms7989
Chien-Ping Liang (1999) Targeted Linearization of  DNA in vivo. Elsevier, 17(2) 95-103, Retrieved from: https://doi.org/10.1006/meth.1998.0721
Chien-Ping Liang (1999) Targeted Linearization of  DNA in vivo. Elsevier, 17(2) 95-103, Retrieved from: https://doi.org/10.1006/meth.1998.0721
Andrew Camilli (2009) Bacterial Small-Molecule Signaling Pathways. Science, 311(5764), 1113-1116. Doi: 10.1126/science.1121357
Brooke A. McDaniel  (2006) Identification of a Mutation in the Bacillus subtilis S-Adenosylmethionine Synthetase Gene That Results in Derepression of S-Box Gene Expression. J. Bacteriol, 188(10), 3674-3681, Doi:  10.1128/JB.188.10.3674-3681.2006
Fayu Yang (2017) CRISPR/Cas9-loxP-Mediated Gene Editing as a Novel Site-Specific Genetic Manipulation Tool. Mol Ther Nucleic Acids 7, 378-386. Doi: 10.1016/j.omtn.2017.04.018
Brian J. Caliando & Christopher A. Voigt (2015) Targeted DNA degradation using a CRISPR device stably carried in the host genome. Nature Communications 6. Doi:10.1038/ncomms7989
Ref.
Follow Us Now !!!
SiteMap
© iGEM 2017 Team HKUST. All rights reserved.
made with ♥ @ HKUST
