<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong_HKUST_Recombination skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong HKUST/Recombination</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>HKUST iGEM Team 2017</TITLE></P><DIV id="page-wrapper"><HEADER id="header" style="margin-top:30px; color:	rgba(255, 255, 255, 0.9);"><H1 id="logo" style="font-family:  font-family: 'Coming Soon', cursive; font-weight: 100; font-size: 15pt;"><A href="https://2017.igem.org/Team:Hong_Kong_HKUST" style="color:white; text-decoration: none; font-size:16px; ">Team Hong Kong - HKUST</A></H1><NAV id="nav"><UL><LI><A>Project</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Overview">Project Overview</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Sensing">Sensing Module</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/TimeControl">Time Control Module</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Recombination">Recombination Module</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/FutureApplication">Possible Future Application</A></LI></UL><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model">Modelling</A></LI><LI><A>Single-cell</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/Sensing">Model-Sensing</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/TimeControl">Model-Time Control</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/Recombination">Model-Recombination</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/Population">Population</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Model/References">References</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/HP">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/HP/Silver">Human Practices (Silver)</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Engagement">Education &amp; Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Medal_Requirements">Medal Requirements</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/InterLab">Interlab</A></LI><LI><A>About Us</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Team">Our Team</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Logbook_and_Protocol">Logbook &amp; Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/Giant_Jamboree_Photo_Gallery">Giant Jamboree Photo Gallery</A></LI></NAV></HEADER><DIV id="main" style="padding-left: 8%; padding-right:8%"><DIV><HEADER class="major"><H2>Recombination Module</H2></HEADER><SECTION id="content"><CENTER><H3>Overview / Rationale</H3></CENTER><P>The final project in our module which activates after the delay produced by the Time Control module is the Recombination module. The purpose of this module is to create recombinase proteins which will cleave the plasmid at specific sites and separate the origin of replication from the gene of interest. The recombinase we chose to use in our project was Cre recombinase.
 </P><P>
Cre recombinase does not destroy the entire plasmid and allows the cell to live after the gene sequence is removed. This could be advantageous in eukaryotic organisms where the death of healthy cells is not favorable and only the unwanted gene portion needs to be removed.</P><P>
Cre recombinase is simple to use when two loxP sites are flanked around the sequence to be excised without the addition of other guided DNA/mRNA. It has been experimented with in mammalian organisms and extensively in mice. Similar recombinases such as Dre can be further added to splice other inserted gene sequences. The Dre recombinase recognizes its own specific sites so there will not be cross recombination between the two recombinases.</P><P>While Cre recombinase itself can only recombine about 55% of the plasmids to be in the recombined state due to the fact that it also catalyzes the reverse reaction back to the original state, our <A style="color:white; word-wrap: break-word;">model</A> shows that under when the population size is in equilibrium, the amount of the gene of interest is diluted after generations of cell replication and cell death.</P><CENTER><A class="image fit" style="color: #fff; width:100%; max-width:1000px;">Diagram 1. Cre recombinase targets lox mutant sites which will then recombine gene A, while Dre recombinase targets rox site which will recombine gene B </A></CENTER><P>The function of the Cre-lox&amp; Dre-rox is to knock out the gene of interest from the plasmid. Cre is a site-specific
        DNA recombinase derived from the P1 bacteriophage while Dre derives from P1-like transducing phage D6 isolated from
        Salmonella oranienburg. (Anastassiadis et al., 2009) (Nagy, 2000) Both of them are tyrosine family of site-specific
        recombinases.
      </P><CENTER><H3>Our Design</H3></CENTER><P>We designed recombinase expressing genes located just downstream of our time delay module under the regulation of <I>pPhlF</I>.</P><CENTER><A class="image fit" style="color: #fff; width:100%; max-width:1200px;">Diagram 2. Design of recombination system for application </A></CENTER><P>The application of Cre and Dre is one of our main focus in our project. Our design is to insert the lox and rox sites
        into the vector plasmid that flanks the origin of replication. When there is an expression of Cre and Dre protein,
        Cre and Dre will cause recombination of their corresponding target site. The Cre-lox and Dre-rox systems then induce
        deletion of the origin of replication, resulting in two occasions. One is the plasmid without the origin of replication.
        The other one is the plasmid with the origin of replication. The plasmid without the origin of replication cannot
        carry out DNA replication. In each generation, the population will contain only half of the plasmid with the gene
        of interest. After many generations, the amount of plasmid with the gene of interest will be diluted and eventually
        none of the cells will contain the plasmid with the gene of interest.
      </P><P>The second occasion, the population will contain the plasmids with the origin of replication only (because of deletion).
        However, as the cell will use its resource to replicate the plasmids with the origin of replication. The cell containing
        the plasmids with the origin of replication are less competitive than the wild-type. The population containing the
        plasmid flanked with origin of replication will be eventually wiped out due to selective pressure. As a result, the
        population would revert to its original state before the introduction of the plasmid. </P><P>Nevertheless, lox and rox sites are not necessarily flanked around the origin of replication. </P><CENTER><A class="image fit" style="color: #fff; width:100%; max-width:700px;">Diagram 3. Plasmid containing 3 modules construct: Sensing, Time control and recombination modules, with origin of replication and chloramphenicol antibiotic resistance gene </A></CENTER><CENTER><H3>Maximizing the efficiency of our knock out switch</H3></CENTER><P>Our team emphasizes on the safety of construct design. That is why maximizing the efficiency of our knock out switch
        is of paramount importance in our project. We have designed several measures to increase the efficiency of our knock
        out switch. First is the implementation of two recombination deletion systems: Cre-lox and Dre-rox systems. Due to
        the property of non-crossover recombination between Cre-rox or Dre-<I>loxP</I>, if one of our recombination systems fails,
        other recombination system can still induce the deletion of the origin of replication, thus termination of the expression
        of gene of interest and making our knockout switch doubly safe. Second is the use of mutant <I>loxP</I> sites, lox 66 and
        lox 71. After the recombination between lox 71 and lox 66 sites, the characteristic property of these target site
        prevents recombination back to the original state, increasing the efficiency of deletion. </P><CENTER><H3>Mechanism of Cre/lox system induced deletion</H3></CENTER><P>In the case of Cre protein, it recognizes 34 base pair sequences known as <I>loxP</I> sites. There are also mutants of the
        recognition site, for example lox 66 and lox 71 which have distinctive activities from <I>loxP</I> site. The lox sequence
        consists of asymmetric 8 bp sequences of spacer region (also called linker), which is flanked by 13 bp palindromic
        repeats. Because of the asymmetrical properties of linkers, it provides a control of directionality of recombination
        by Cre protein. When two lox sites have compatible linkers with identical orientations, Cre-mediated recombination
        causes a deletion of the sequence in between the two lox sites.</P><CENTER><H3>Mechanism of Dre/rox system induced deletion</H3></CENTER><P>Dre protein is a Cre-like site-specific recombinase (SSR) which had diverged sufficiently to recognize its own recombination
        site that is distinct from lox site named rox. The base pair of rox site is 32 (lox site has 34 bp). The rox site
        is structurally similar to lox site. It also has two palindromic repeats but are 14 bp (lox site has 13 bp repeats)
        and are separated by only 4bp linker sequence. Multiple research papers mention that there is no crossover recombination
        between Cre-rox or Dre-<I>loxP</I>. (Brian L, 2004) </P><CENTER><H3>References</H3></CENTER><OL><LI id="ref1"><A style="color:white; pointer-events:none;">Anastassiadis, K., Fu, J., Patsch, C., Hu, S., Weidlich, S., &amp; Duerschke, K. et al. (2009). Dre recombinase, like Cre, is a highly efficient site-specific recombinase in E. coli, mammalian cells and mice. Disease Models &amp; Mechanisms, 2(9-10), 508-515.</A><A style="color:white; word-wrap: break-word;" href="http://dx.doi.org/10.1242/dmm.003087"> http://dx.doi.org/10.1242/dmm.003087</A></LI><LI id="ref2"><A style="color:white; pointer-events:none;">Nagy, A. (2000). Cre recombinase: The universal reagent for genome tailoring. Genesis, 26(2), 99-109. </A><A style="color:white; word-wrap: break-word;" href="http://dx.doi.org/10.1002/(sici)1526-968x(200002)26:2%3c99::aid-gene1%3e3.0.co;2-b">http://dx.doi.org/10.1002/(sici)1526-968x(200002)26:2&lt;99::aid-gene1&gt;3.0.co;2-b</A></LI><LI id="ref3"><A style="color:white; pointer-events:none;">Patent US7915037 - Dre recombinase and recombinase systems employing Dre recombinase. (2017). Google Books. Retrieved 26 October 2017, from </A><A style="color:white; word-wrap: break-word;" href="https://www.google.com/patents/US7915037">https://www.google.com/patents/US7915037</A></LI></OL></SECTION></DIV></DIV><DIV><FOOTER style="opacity:0.7" id="footer"><UL class="icons"><LI><DIV style="display:flex; flex-direction:row; justify-content:center"><A href="https://www.facebook.com/hkustigem2017domilox/" style="font-size:15px; color:blue"> Follow Us Now !!!</A></DIV></LI><LI><DIV style="display:flex; flex-direction:row; justify-content:center"><A href="https://2017.igem.org/Team:Hong_Kong_HKUST/SiteMap" style="font-size:15px; color:red">SiteMap</A></DIV></LI></UL><UL class="copyright"><LI><B>© iGEM 2017 Team HKUST. All rights reserved.</B></LI><LI>made with <SPAN style="color: red">♥</SPAN> @ HKUST</LI></UL></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>