<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tianjin_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tianjin/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tianjin/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tianjin/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tianjin/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Team:Tianjin2014/Safety</TITLE></P><DIV class="fixed"><TABLE width="100%" border="5" cellpadding="0" cellspacing="0" bordercolor="#993300" bgcolor="#993300"><TBODY><TR bgcolor="#3A2108"><TD height="40"><SPAN class="STYLE78"><A href="#part1">Overview</A></SPAN></TD></TR><TR bgcolor="#3A2108"><TD height="40"><SPAN class="STYLE78"><A href="#part2">Background</A></SPAN></TD></TR><TR bgcolor="#3A2108"><TD height="40"><SPAN class="STYLE78"><A href="#part3">Design</A></SPAN></TD></TR><TR bgcolor="#3A2108"><TD height="40" class="STYLE78"><A href="#part4"><STRONG>Results</STRONG></A></TD></TR><TR bgcolor="#3A2108"><TD height="40" class="STYLE69"><A href="#part5"><STRONG>Future Work</STRONG></A></TD></TR><TR bgcolor="#3A2108"><TD height="40" class="STYLE78"><A href="https://2014.igem.org/Team:Tianjin/Project2#part6"><STRONG>Modeling</STRONG></A></TD></TR><TR bgcolor="#3A2108"><TD height="35" bgcolor="#663300"><SPAN class="STYLE78"><A href="#top">Go to the top</A></SPAN></TD></TR></TBODY></TABLE></DIV></DIV><TABLE width="1230" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="19%" bgcolor="#3A2108"><P> </P><P> </P><P> </P></TD><TD width="43%" bgcolor="#3A2108"><P> </P><P> </P><P> </P><P><SPAN class="STYLE43"><SPAN class="STYLE24"><SPAN class="STYLE77">T</SPAN></SPAN><SPAN class="STYLE76">rans<EM>f</EM> ibre</SPAN></SPAN></P><P> </P><P><SPAN class="STYLE40">Welcome to Team Tianjin! </SPAN></P></TD></TR></TBODY></TABLE><TABLE width="1230" border="0" cellpadding="0" cellspacing="0" bgcolor="#663300"><TBODY><TR><TD bgcolor="#663300"><SPAN class="STYLE45"><A href="https://2014.igem.org/Team:Tianjin">Home|</A></SPAN></TD></TR></TBODY></TABLE></DIV><DIV class="STYLE46" id="project" onmouseover="MM_showHideLayers('Layer8','','show')" onmouseout="MM_showHideLayers('Layer8','','hide')"><A href="https://2014.igem.org/Team:Tianjin/Project" class="STYLE48">Project|</A></DIV><DIV id="judging" onmouseover="MM_showHideLayers('Layer11','','show')" onmouseout="MM_showHideLayers('Layer11','','hide')"><TABLE width="110" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#663300"><A href="https://2014.igem.org/Team:Tianjin/Judging" class="STYLE47">Judging|</A></TD></TR></TBODY></TABLE></DIV><DIV id="notebook" onmouseover="MM_showHideLayers('Layer13','','show')" onmouseout="MM_showHideLayers('Layer13','','hide')"><TABLE width="110" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#663300"><A href="https://2014.igem.org/Team:Tianjin/Notebook" class="STYLE47">Notebook|</A></TD></TR></TBODY></TABLE></DIV><DIV id="team" onmouseover="MM_showHideLayers('Layer7','','show')" onmouseout="MM_showHideLayers('Layer7','','hide')"><DIV id="layer"><TABLE width="100" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#663300"><SPAN class="STYLE45"><A href="https://2014.igem.org/Team:Tianjin/Team">Team|</A></SPAN></TD></TR></TBODY></TABLE></DIV></DIV><DIV id="safety" onmouseover="MM_showHideLayers('Layer19','','show')" onmouseout="MM_showHideLayers('Layer19','','hide')"><DIV id="layer2"><TABLE width="100" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#663300"><SPAN class="STYLE45"><A href="https://2014.igem.org/Team:Tianjin/Safety">Safety|</A></SPAN></TD></TR></TBODY></TABLE></DIV></DIV><DIV id="human-practice" onmouseover="MM_showHideLayers('Layer22','','show')" onmouseout="MM_showHideLayers('Layer22','','hide')"><DIV id="layer3"><TABLE width="100" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#663300"><SPAN class="STYLE45"><A href="https://2014.igem.org/Team:Tianjin/Human_Practice">Human Practice|</A></SPAN></TD></TR></TBODY></TABLE></DIV></DIV><TABLE width="1230" border="0" align="center" cellpadding="0" cellspacing="0"><TBODY><TR><TD bgcolor="#C67B14"><SPAN class="STYLE62"><STRONG>①Overview</STRONG></SPAN></TD></TR><TR><TD bgcolor="#FFED97" class="STYLE33"><P>As  to lower organisms, there seems an indestructible barrier between biological  signals and electrical signals. There have been bits of research results on the  transformation from electrical signals to biological signals, for example, the  phosphoinositide phosphatase activity coupled to an intrinsic voltage sensor in  Ciona intestinalis. However, the opposite direction of transformation is almost  a blank space, especially in microbes. Thus, we want to build a bridge between  them. With this, microbes can accomplish detection with electronic output  signals, and even ‘operate’ machines!</P><P>The  bridge is the transformed curli fiber linked to nanogold. When inducer is added  in the system, ‘transfibers’ take shape and nanogolds are linked together as  wires, so the circuit is ‘on’. In terms of detection, it can be characterized  by electric current change. Meanwhile, other machines can be switched on by  means of this.</P><P>To learn more, follow us  and enjoy the travel of ‘TRANS’!</P></TD></TR></TBODY></TABLE><SPAN class="STYLE29"><SPAN class="STYLE62"><STRONG>②Background</STRONG></SPAN></SPAN><P class="STYLE33"><STRONG>Curli fiber</STRONG></P><P class="STYLE33"><STRONG>1.What is curli fiber?</STRONG></P><P class="STYLE33">Bacteria are able to integrate and survive  in a remarkably diverse collection of environments.</P><P class="STYLE33">In recent years, bacterial communities have  been better appreciated as an integral part of most microbial lifestyles. These  communities, or biofilms, are prominent during infections and are generally  characterized by an extracellular matrix that can help sculpt three-dimensional  structures, which promote the survival of its inhabitants in the face of  environmental stresses. Enteric bacteria such as Escherichia coli and  Salmonella spp. express proteinaceous extracellular fibers called “<STRONG>curli</STRONG>” that are involved in surface and  cell-cell contacts that promote community behavior and host colonization.</P><P class="STYLE33">Curli are the major proteinaceous component  of a complex extra-cellular matrix. Curli fibers are involved in adhesion to  surfaces, cell aggregation and biofilm formation. Curli also mediate host cell  adhesion and invasion, and they are potent inducers of the host inflammatory  response.</P><P class="STYLE33">Structurally and biochemically, curli  belong to a growing class of fibers known as amyloids. Amyloid fiber formation  is responsible for several human diseases including Alzheimer's, Huntington's, and prion  diseases. Curli share all of the biophysical properties of amyloids: ordered  β-sheet-rich fibers resistant to proteases and other harsh denaturants with a  capacity to bind the dyes CR and ThT. In contrast to disease-associated  amyloids, curli assembly is not the result of protein misfolding. Instead, curli  are the product of a dedicated biogenesis pathway providing a paradigm for  understanding controlled amyloidogenesis. </P><P class="STYLE33"><STRONG>2.Why do we choose curli？</STRONG></P><P class="STYLE33">Curli  assembly is guided by the products of seven curlispecific genes (csg) encoded  on two divergently transcribed operons, <EM>csgDEFG</EM> and <EM>csgBAC</EM>. CsgD is the master regulator of  curli biogenesis and is required for transcription of the csgBAC operon.</P><P class="STYLE33">Our model  system is based on the self-assembly of the secreted major curli subunit CsgA.  Secreted CsgA monomers are templated on CsgB, which is anchored to the cell  surface, to form curli fibre; moreover, CsgA secreted from one cell can  interact with CsgB on other cells. Situated at the cell surface, CsgB  nucleates soluble, unstructured CsgA into a highly ordered amyloid fiber. </P><P class="STYLE33">To  transport the major submit CsgA and the minor submit CsgB into the  extracellular milieu, the lipoprotein CsgG oligomerizes into a pore-like structure  in the outer membrane.
            In  addition, the secretion and localization of curli fiber subunits through CsgG are  mediated by three additional proteins: CsgC, CsgE and CsgF. CsgC was predicted  to have oxidoreductase activity, regulating CsgG outer membrane assembly and  pore activity. CsgE could be considered a CsgA-specific chaperone. Besides, CsgF  associates with the outer membrane and is required for cell association of the  minor curli fiber subunit CsgB.</P><P class="STYLE33">As  explained above, curli have two remarkable advantages. One is the self-assembly  of CsgA monomers into mature amyloids----curli. Apart from it, using synthetic  riboregulators, we can implement inducible transcriptional and translational  control over the expression of CsgA subunits engineered to display various  peptide tags, which can interface with inorganic materials, such as gold  nanoparticles, to form a conductive structure. Utilizing these advantages, we can design  a transducer which can convert the change of gene expression level that  stimulated by inducer directly into the electric signal via inductive synthesis（or destroy）of  nanowire between the electrodes.</P><P class="STYLE33"><STRONG>Gold Nanoparticles (AuNPs)</STRONG></P><P class="STYLE33">AuNPs are  the most stable metal nanoparticles, and they present fascinating aspects such  as their assembly of multiple types involving materials science, the behavior  of the individual particles, size-related electronic, magnetic and optical  properties (quantum size effect), and their applications to catalysis and  biology.</P><P class="STYLE33">Since we  can interface curli fiber with inorganic materials to produce amyloid-based  wires that are either externally controllable or undergo autonomous patterning,  a material with completely bio-compatibility, conductivity and stability is  required. AuNPs can be linked to some peptides, providing fascinating  propoties. For instance, the assembly of relatively short polypeptides on curli  can provide protein-like complex structures and conductivity in  conjugates with AuNPs.</P><P class="STYLE33"><STRONG>Nanowire-based devices</STRONG></P><P class="STYLE33">Devices  based on nanowires have emerged as one of the most powerful and general  platforms for ultrasensitive, direct electrical detection of biological and  chemical species and for building functional interfaces to biological systems.  With collective efforts and creation of researchers all around the world, there  comes many kinds of nanosensors for various uses such as ultrasensitive  detection of proteins and individual virus particles as well as recording,  stimulation, and inhibition of neuronal signals in nanowire–neuron hybrid  structures.</P><P class="STYLE33">The  advantages of nanowires lie in many ways. Firstly, nanowires are enabled  reproducible synthesis of nanowires of homogeneous composition and diameter  with controllable electronic and optical properties. Moreover, branched  nanowire structures with unique functions have been built in at the stage of  synthesis. Significantly, In general, the similarity in size of nanowires and  natural nanostructures in biological systems makes nanowires an obvious choice  for creating highly sensitive tools that can probe biological systems. The  great characteristic of direct and label-free readout (i.e., without the use of  bound dyes and fluorescent probes) is particularly attractive for many  applications in medicine and life sciences.</P><P class="STYLE33">An  important point about this detection process, which is quite distinct from  common optically based assays, is that it occurs in real time, and the binding  process can be viewed as it happens on a computer logging the conductance of  one or more devices.</P><P class="STYLE33"><STRONG>What is MMP-7?</STRONG></P><P class="STYLE33">Matrilysin,  also known as matrix metalloproteinase-7 (MMP-7), pump-1 protease (PUMP-1), or  uterine metalloproteinase is an enzyme in humans that is encoded by the MMP-7  gene. 
            The  primary role of cleaved/activated MMP-7 is to break down extracellular matrix  by degrading macromolecules including casein, type I, II, IV, and V gelatins,  fibronectin, and proteoglycan. Most MMP's are secreted as inactive proproteins  which are activated when cleaved by extracellular proteinases. The enzyme  encoded by this gene degrades proteoglycans, fibronectin, elastin and casein  and differs from most MMP family members in that it lacks a conserved  C-terminal protein domain.</P><P class="STYLE33">MMP-7  cleaves collagen III/IV/V/IX/X/XI and proteoglycan indicating that MMP  inhibitors can potentially be used in therapies that involved in inhibition  tissue degradation, remodeling, anti-angiogenesis and inhibition of tumor  invasion. The upregulation of MMP-7 is associated with many malignant tumors  including esophagus, stomach, colon, liver, pancreas, and renal cell  carcinomas. High MMP-7 expression facilitates cancer invasion and angiogenesis  by degrading extracellular matrix macromolecules and connective tissues. Theses  degradations are associated with many mechanisms including embryogenesis,  postpartum uterine involution, tissue repair, angiogenesis, bone remodeling,  arthritis, decubitus ulcer, and tumor metastasis/invasion. Benifiting from the  characteristic of MMP-7, it may be used to predict the level of cancer.</P><P class="STYLE33">Most MMP's are secreted as inactive  proproteins which are activated when cleaved by extracellular proteinases. The  enzyme encoded by this gene degrades proteoglycans, fibronectin, elastin and  casein and differs from most MMP family members in that it lacks a conserved  C-terminal protein domain.</P><P class="STYLE33"><STRONG>What is CheZ?</STRONG></P><P class="STYLE33">Bacterial chemotaxis results from the  ability of flagellated bacteria to control the frequency of switching between  smooth-swimming and tumbling episodes in response to changes in concentration  of extracellular substances. High levels of phosphorylated CheY protein are the  intracellular signal for inducing the tumbling mode of swimming. The CheZ  protein has been shown to control the level of phosphorylated CheY by  regulating its rate of dephosphorylation. In short, protein CheZ plays an  important role in bacterial chemotaxis signal transduction pathway by  accelerating the dephosphorylation of phosphorylated CheY (CheY-P).</P><P class="STYLE33"><STRONG class="STYLE48">References</STRONG></P><P class="STYLE33">[1] Blanco L P, Evans M L, Smith D R, et  al. Diversity, biogenesis and function of microbial amyloids[J]. Trends in  microbiology, 2012, 20(2): 66-73.</P><P class="STYLE33">[2] Barnhart M M, Chapman M R. Curli  biogenesis and function[J]. Annual review of microbiology, 2006, 60: 131.</P><P class="STYLE33">[3] Chen A Y, Deng Z, Billings A N, et al.  Synthesis and patterning of tunable multiscale materials with engineered  cells[J]. Nature materials, 2014.</P><P class="STYLE33">[4] Daniel M C, Astruc D. Gold  nanoparticles: assembly, supramolecular chemistry, quantum-size-related properties,  and applications toward biology, catalysis, and nanotechnology[J]. Chemical  reviews, 2004, 104(1): 293-346.</P><P class="STYLE33">[5] Patolsky F, Timko B P, Zheng G, et al.  Nanowire-based nanoelectronic devices in the life sciences[J]. MRS bulletin,  2007, 32(02): 142-149.</P><P class="STYLE33">[6] Wilipedia</P><P class="STYLE33">[7] Yokoyama Y, Grünebach F, Schmidt SM,  Heine A, H?ntschel M, Stevanovic S, Rammensee HG, Brossart P (2008).  &quot;Matrilysin (MMP-7) is a novel broadly expressed tumor antigen recognized  by antigen-specific T cells&quot;. Clin. Cancer Res. 14 (17): 5503–11. </P><P class="STYLE33">[8] Edman K, Furber M, Hemsley P, Johansson  C, Pairaudeau G, Petersen J, Stocks M, Tervo A, Ward A, Wells E, Wissler L  (2011). &quot;The discovery of MMP7 inhibitors exploiting a novel selectivity  trigger&quot;. ChemMedChem 6 (5): 769–73.</P><P class="STYLE33">[9] Sanna M G, Simon M I. In vivo and in  vitro characterization of Escherichia coli protein CheZ gain-and  loss-of-function mutants[J]. Journal of bacteriology, 1996, 178(21): 6275-6280.</P><SPAN class="STYLE29"><STRONG class="STYLE62">③Design</STRONG></SPAN><TABLE width="100%" border="3" cellpadding="0" cellspacing="0" bordercolor="#C67B14" bgcolor="#993300"><TBODY><TR><TD bgcolor="#FFED97"><P class="STYLE33">To  achieve the convert from bio-respond to electrical output, we designed the  transmitter &quot;transfibre&quot;, and the mechanics of which can be  summarized as follow: On appearance of the substance of interest, our bacteria  will inductively grow a large quantity of modified curli fibers which can  adsorb nanogold. These nanowires are  highly exclusive in conductivity from the culture, thus, they will conductively  bridges two separated electrodes that pre-arranged in the culture and the  change of current can be monitored, which present the approximate concentration  of the target substance. </P><P class="STYLE33">Our design work of this bio-electrical  bridge can be summarized into following parts. </P><P class="STYLE33"><STRONG>Part 1: the  controllable expression of the curli fiber</STRONG></P><P class="STYLE33">1.Background </P><P class="STYLE33">The delicate curli-assembly system in  E. coli K-12 is revealed lately and chose for our &quot;transfibre&quot;  prototype. Curli fibers, that play key role in establishment and persistence of  biofilm communities, are coded by the Csg gene cluster where seven  genes(CsgABCDEFG) located, and their function are listed here： 
                    CsgA: At the cell surface, CsgA, The main subunit, alone soluble  in vivo, that can be nucleated into an amyloid fiber by CsgB. 
                    CsgB: Cell-surface-associated  nucleation agent. 
                    CsgC: CsgC is less well understood  and it was predicted to have oxidoreductase activity. Within the periplasm, CsgC may regulate CsgG  outer membrane assembly and pore activity through modification of C230 in CsgG. 
                    CsgE: a small periplasmic protein, is  proposed to direct CsgA to the CsgG secretion complex and helps mediate  secretion specificity via an N-terminal sequence in the mature CsgA. CsgE also  inhibits CsgA polymerization in vitro, and hence CsgE could be considered a  CsgA-specific chaperone. 
                    CsgF: CsgF is a membrane-associated  protein, required for CsgB surface exposure, and is crucial for efficient CsgA  polymerization. 
                    CsgG: Once in the periplasm, the  lipoprotein CsgG is transported by the LOL (lipoprotein  outer-membrane localization) transport system to the outer membrane where it  oligomerizes into a pore-like structure that is required for secretion of CsgA  and CsgB into the extracellular milieu. </P></TD></TR><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig 1. This figure  demonstrates the cooperation of curli-associated genes when bacteria respond to  external environment change.</SPAN></TD></TR></TBODY></TABLE><P class="STYLE33">For the reason that curli fiber mostly acts  an emergency mechanism that could enhance bacterial survival in extreme  conditions such as low osmolality, low growth temperature (&lt;32°C) and hydroponic,  the regulation system attached strictly limited the expression of two curli  operons under normal circumstances that bio-sensing processes operating. </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig 2. Genes that  associated with curli fiber assembly are located on two operons in E coli. K-12 genome. They are under strict  control and low level of expression at normal cultivation conditions. </SPAN></TD></TR></TBODY></TABLE><P class="STYLE33">2. The design idea </P><P class="STYLE33">According  to the synthetic properties of curli fiber, CsgA interacts with molecular  chaperones  CsgE and  maintains it in a soluble state in  vivo, preventing CsgA from aggregation without CsgB protein. Thus, controllable  expression of CsgB can provide nucleation for CsgA rapid aggression and recruits CsgA monomers that  pre-secreted in culture into fibers, which meets the request of controllable  expression and immediate respond for biosensor output platform. 
                    Following this strategy, we devise an  inducible expression system of CsgB and a constitutive expression system of the  other Csg-genes (excluding CsgD).  Referring to the original curli assembly device, CsgA, the major subunit of  curli fiber, and CsgC are constructed in a high copy plasmid (pSB1C3) and  constitutive expression by a relatively strong promoter, while CsgEFG using a  weaker promoter and docked in pSB3K3. </P><P class="STYLE33">To  obtain target genes from Csg operons, method PCR was applied on the genome of  DH5alpha strains (The original genes of CsgA and CsgE has SpeI and PstI  restriction enzyme cutting sites. They have been substituted by synonym codon  according to codon usage bias). Then, construct expression vector of these  genes to transform E coli. BL21 and transT-1. (In BL21, CsgA and CsgB genes has  been knocked out by λ-red recombinase system to eliminate disturb.) </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig.3 The concept of bio-electrical transmit.  The modified CsgAs are aggregate on the inductive expression of CsgB and  bridges the circuit.</SPAN></TD></TR></TBODY></TABLE><P class="STYLE33">Two promoters, T7 promoter and Pbad  promoter, are selected whose inducers,  IPTG and L-arabinose respectively, are chosen to simulate the target inducing  process. Meanwhile, for CsgA and CsgE,  F, G, we also choose three promoters with different strength--- P1 (BBa_J23100), P2  (BBa_J23104) and P3(BBa_J23110) to screen out a better gene expression  combination through this method.  Thus, six parts are constructed with different levels of CsgA, C expression to work out the best fiber  polymerization. </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="559" colspan="2" valign="top"><P class="STYLE33">Biobricks for Promoter Strength    Trail</P></TD></TR><TR><TD width="253" valign="top"><P class="STYLE33">T7-CsgB-P1-CsgA, C</P></TD><TD width="307" valign="top"><P class="STYLE33">Pbad-CsgB-P1-CsgA, C</P></TD></TR><TR><TD width="253" valign="top"><P class="STYLE33">T7-CsgB-P2-CsgA, C</P></TD><TD width="307" valign="top"><P class="STYLE33">Pbad-CsgB-P1-CsgA, C</P></TD></TR><TR><TD width="253" valign="top"><P class="STYLE33">P2-CsgE, F, G</P></TD><TD width="307" valign="top"><P class="STYLE33">P3-CsgE, F, G</P></TD></TR></TBODY></TABLE><SPAN class="STYLE33">Table 1. Six parts with distinguished  strength of constitutive promoters were constructed. And combinations of them  were tested.  </SPAN><P class="STYLE33">In addition, we truncate 19 amino acids from CsgB molecule’s C-terminus and get protein CsgBtrunc. It has been tested to be  not outer membrane-associated, but secreted away. Soluble CsgBtrunc can also  assemble into fibers that bound to the amyloid-specific dyes Congo red, and  CsgBtrunc are able to seed soluble CsgA polymerization in vitro. Although it  displayed only modest nucleator activity in vivo, considering that our modules  could secrete much higher concentration of CsgA, it could acts relatively good nucleating capacity. </P><P align="left" class="STYLE33">The establishment of CsgBtrunc modules resembles with CsgB.  The difference is that, after the addition of inducers, the curli fiber will  grow suspendingly in liquid, not anchored on the outer membrane of E.coli. </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig 4.  This figure shows the concept of CsgBtrunc expression  and off-cell nucleating on CsgA. 
3. An alternative expression strategy</SPAN></TD></TR></TBODY></TABLE><P align="left" class="STYLE33">We can transform CsgA and CsgB into  separate bacteria and put them in same culture.  In this mode, one kind of E. coli acts as ‘material background’ that provide  constitutively expressed CsgBtrunc, and another acts as ‘inducible nucleator’ which CsgA is under induction. Detection acting at a  distance could be accomplished by this method. However, this method was not  accomplished and we don’t know its advantages clearly. </P><P class="STYLE33"><STRONG>Part 2: The method of  electric conduction</STRONG></P><P class="STYLE33">Our curli fiber is supposed to adsorb  nanogold, which is conductive and can act as wires when linked to each other.  However, the wild type CsgA monomer is unable to adsorb nanogold. Thus, some  measures should be taken to functionize it. 
                    According to  literature research, we  designed three methods to achieve this goal: inserting two histidine tags (one  before the first repeat domain and one after the last repeat domain in CsgA),  adding a cysteine tag and inserting an amino acid sequence of pentapeptide. The  principle of these three methods is the same: adding  ligands like sulfhydryl that have good affinity with gold to gain CsgA ability  of absorbing nanogold. 
                  As for the first method, CsgA-His is  constitutive expression whereas CsgB is under the control of Pbad promoter.  Once we adding the inducer, CsgA would quickly aggregates  and thus curli fiber with his  tags would generate immediately and absorbing  Au-NTA-Ni particle. We want to test  the affinity between Meanwhile, the  CsgA-His can be used for  CsgA protein detection and purification.  For the second method that adding a  cysteine tag, the integral mechanism of generating the curli fibre is same as  above, but the sulfhydryl of cysteine tags can link to the nanogold particles  itself because the monodisperse spherical gold nanoparticles are arranged to  chained structure due to the effect of these molecules. Besides, the citrate  layer on the metal surface is replaced by cysteine leading to a formation of  organic double layer structure, which enhances the absorption. As for the third  one, the composition of the pentapeptide is  Leu-Pro-Phe-Phe-Asp-OH (LPFFD-OH), the existence of N-donors containing  side-chains will help a lot in the attachment or binding to the surface of gold  nanoparticles, though the detailed mechanisms of the binding of the  pentapeptide was not fully resolved. </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig 5. The modified CsgA that obtains  the ability of absorbing nanogold will firm conductive wires between electrodes  on the respond of inducer.</SPAN></TD></TR></TBODY></TABLE><P class="STYLE33">In all three methods, the primary  concept is to produce conductible curli fiber and accomplish the bio-signal  transformation. We have already known the modified curli fiber were produced by  a certain kind of inducer (arabinose and IPTG  were used in our experiments) and then  attached to AuNPs. Since we assumed that the intensity of the induction could  be reflected by the quantity of the AuNPs-linked curli, the concentration of  the inducer could thus determine the resistance and consequent current of the  whole circuits. In this way, we transformed the bio-signal into the electric  signal. Furthermore, as the system is highly modularized, we can substitute the  promoter of other inducers so that we could extend this system to detect sorts  of subsidence apart from arabinose we used. </P><P class="STYLE33"><STRONG>Part 3: the upgrade  of idea-logic gate.</STRONG></P><P class="STYLE33">As mentioned, CsgA will aggregate  into curli fibers with the seduction by seducer (like arabinose). In order to  make the whole system a contenting device to characterize the concentration of  seducer, a NOT gate is added to sabotage the nanowire. Thus, the current will  decline and the change of concentration can still follow a kind of function,  which can be determined by experiments. 
                    AuNPs-based nanowire may be digested  by protease or tear into pieces under the influence of physical forces. Because of the endurance of CsgA to proteases,  we design a forces-destroy pathway. 
                    CheY and CheZ are antagonist in a  system regulating the motion and the resulting chemotaxis of E.coli:  Phosphorylated CheY motivates the rolling of the bacterium, and it is damped by  CheZ through prompting the dephosphorylation. 
                  We construct two plasmids. One is  BBa_K1361000, which has the stronger promoter P1, and the other has the weaker  promoter P2. CI is a repressor gene that can restrain the expression of CheZ,  and Plam acts as the promoter to start the gene of expression of cheZ. If the  concentration of the inducer increases, the Pbad promotes the gene expression  of CsgB as well as the repressor CI. So the effect of inhibition of CheZ increases as the curli fiber grows, because the repressor CI can bind to the Plam and inhibit the expression of CheZ.  Thus, CheZ gene is depressed. If the concentration of the inducer decreases,  the concentration of the repressor decreases. So the promoter Plam could start  the expression of CheZ in some degree. The  CheZ gene expresses and it can and,  afterwards, motivate the motion of flagellum, that naturally  destroyed curli fiber. So the intensity of the electric current will  descend. </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig 6. The mechanic of CheZ-Curli interaction  is demonstrated in the figure, hypothesizing that the increasing motility of  cells will destruct the curli fiber. </SPAN></TD></TR></TBODY></TABLE><P class="STYLE33">Because our conductive biofilm build  a bridge between bacteria and electricity, our E.coli is expected to operate  machine. As the concentration of inducer increases, the circuit can be turned  on, and it can switch almost any electrical devices and machines with or  without the help of signal amplifier. It, which may extend the application of transfibre. </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig 7. The schematic  diagram of coordination between bio-device and machines.</SPAN></TD></TR></TBODY></TABLE><P class="STYLE33"><STRONG>Part 4: The detection  device</STRONG></P><P class="STYLE33">Our  living cell system can be used to create environmentally switchable conductive  biofilms. We hypothesized that IPTG-or-L-arabinose-inducible production of CsgB  monomers by our cells would generate extracellular amyloid fibrils that  organize AuNPs into chains, and form a conductive biofilm network. 
                    Engineered biofilms were grown on  interdigitated electrodes deposited on glass slides, with our E.coli cultured  in the presence of AuNPs and in the presence  or absence of IPTG or L-arabinose inducer. </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig 8. The picture shows the making process  from test tube to our detection petri dish.
We designed the petri dish by  ourselves. For the convenience of depositing electrodes, we composed a  detachable culture dish. Firstly, we cut the test tube into a 3cm-long section  without bottom. Secondly, we cut another test tube with longer diameter into a  4cm-long section with a bottom. The agar is utilized to attach the 3cm-long  section to a glass slide to make the petri dish, and the 4cm-long section acts  as a cover to ensure sterility. </SPAN></TD></TR></TBODY></TABLE><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig 9. The real object of our  self-made petri dish.</SPAN></TD></TR></TBODY></TABLE><P class="STYLE33">The fabrication of electrodes  resembles the method of Synthesis and patterning of tunable multiscale  materials with engineered cells. Interdigitated electrodes for measuring biofilm  conductance were created by sputtering gold through shadowing masks onto glass  slides. </P><P class="STYLE33">Our circuit is designed by Yuanchi Ma  at College of Precision Instrument&amp; Opto-Electronics Engineering. </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"> </TD></TR></TBODY></TABLE><P class="STYLE33"><STRONG>Part 5: Stabilized  matrix sensor for MMP-7</STRONG></P><P class="STYLE33">The qualitative or quantitative  detection of proteases can be achieved by various methods such as mass  spectrum, two-dimensional gel electrophoresis and antigen-antibody complex  reaction. However, they are usually time-consumed, demanding and the data is  difficult to be stored and processed. Our project can transfer bio-signal to  electrical signal, and as a result, we try to make it a convenient device to  detect proteases without those aforementioned drawbacks. </P><P class="STYLE33">MMP-7 is the abbreviation of matrix  metalloproteinase-7, which belongs to the family of MMPs (a family of  extracellular zinc proteases that have long been associated with tumor invasion  and metastasis and thus may serve as important cancer biomarkers). It has be  proved that a helix peptide (Arg-Pro-Leu-Ala-Leu-Trp-Arg-Ser) is the substrate  of MMP-7. </P><P class="STYLE33">The substrate peptide of MMP-7 and  binding-site peptide of nano-gold particles (AuNPs) are modified with CsgA.  These two modules added to the C-terminal of the wild type CsgA will not  seriously interrupt the normal aggregation of CsgA. Consequently, we get  redesigned curli fibers with the capacity to detect the MMP-7: When no MMP-7  exists in the system with applied voltage, the most electrons flows through the  nanowire composed by AuNPs binding to the modified CsgAs; however, the  resistance of the fibers will skyrocket for the detrimental enzymolysis of the  substrate peptide by MMP-7, which result in the wreck of nanowire. Based on  this blueprint, the existence of MMP-7 can be detected and the concentration can  be calculated by the concentration function of current. </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig 10. Detection of MMP-7</SPAN></TD></TR></TBODY></TABLE><P class="STYLE33">Reference: 
                    [1] Dobson, C.M. (2005) Structural  biology: prying into prions. Nature 435,747–749 
                    [2] Collinson, S.K. et al. (1999)  Structural predictions of AgfA, the insoluble fimbrial subunit of Salmonella  thin aggregative fimbriae. J. Mol. Biol. 290, 741–756 
                    [3] Shewmaker, F. et al. (2009) The  functional curli amyloid is not based on in-register parallel beta-sheet  structure. J. Biol. Chem. 284, 25065–25076 
                    [4] Wang, X. et al. (2007) In vitro  polymerization of a functional scherichia coli amyloid protein. J. Biol. Chem.  282, 3713–3719 
                    [5] Barnhart, M.M. and Chapman, M.R.  (2006) Curli biogenesis and function. Annu. Rev. Microbiol. 60, 131–147 
                    [6] Wang, X. and Chapman, M.R. (2008)  Curli provide the template for understanding controlled amyloid propagation.  Prion 2, 57–60 
                    [7] Stathopoulos, C. et al. (2000)  Secretion of virulence determinants by the general secretory pathway in  Gram-negative pathogens: an evolving story. Microbes Infect. 2, 1061–1072 
                    [8] Taylor, J.D. et al. (2011) Atomic  resolution insights into curli fiber biogenesis. Structure 19, 1307–1316 
                    [9] Nenninger, A.A. et al. (2011)  CsgE is a curli secretion specificity factor that prevents amyloid fibre  aggregation. Mol. Microbiol. 81, 486–499 
                    [10] Narita, S. et al. (2004)  Lipoprotein trafficking in Escherichia coli. Arch. Microbiol. 182, 1–6 
                    [11] Epstein, E.A. et al. (2009)  Spatial clustering of the curlin secretion lipoprotein requires curli fiber  assembly. J. Bacteriol. 191, 608–615 
                    [12] Robinson, L.S. et al. (2006)  Secretion of curli fibre subunits is mediated by the outer membrane-localized  CsgG protein. Mol. Microbiol. 59, 870–881. 
                    [13] Wang, X. et al. (2010)  Gatekeeper residues in the major curlin subunit modulate bacterial amyloid  fiber biogenesis. Proc. Natl. Acad. Sci.U.S.A. 107, 163–168 
                    [14] Majzik, A. et al. (2010). Functionalization of gold  nanoparticles with amino acid, β-amyloid peptides and fragment. Colloids and Surfaces B:  Biointerfaces, 81(1), 235-241. 
                    [15] Liu G, Wang J, Wunschel DS, Lin  Y (2006) Electrochemical proteolytic beacon for detection of matrix  metalloproteinase activities. J Am Chem Soc 128(38):12382–12383.  doi:10.1021/ja0626638 
                    [16]  Hammer  N D, Schmidt J C, Chapman M R. The curli nucleator protein, CsgB, contains an  amyloidogenic domain that directs CsgA polymerization[J]. Proceedings of the  National Academy of Sciences, 2007, 104(30): 12494-12499. 
                  [17] Chen A Y, Deng Z, Billings A N, et  al. Synthesis and patterning of tunable multiscale materials with engineered  cells[J]. Nature materials, 2014. </P><SPAN class="STYLE33">[18] Scheibel T,  Parthasarathy R, Sawicki G, et al. Conducting nanowires built by controlled  self-assembly of amyloid fibers and selective metal deposition[J]. Proceedings  of the National Academy of Sciences, 2003, 100</SPAN><P class="STYLE62"><STRONG>④Result</STRONG></P><P class="STYLE79">Part 1.  Controllable Expression of Curli fiber</P><P class="STYLE33">Expression and secrete of CsgA, the major subunit of  curli fiber, was carried out in <EM>E coli.</EM> trans-T1 transformed with CsgA, CsgB expression cassette (BBa_K1361007 docked  into pSB1C3) and curli-secrete device (BBa_K1361005 docked into pSB3K3). We tracked  cell growth and total quantities of secrete protein in culture. Samples were  taken from culture and Bradford protein assay was adopted after centrifuging  and sterilization filtered through 0.22μm filter membrane. Meanwhile, another  sample taken at 2-hour interval was wall-broken and examined by SDS-PAGE, which  confirm the expression of CsgA-his.</P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR bgcolor="#FFED97"><TD class="STYLE33">Fig 1. The result of SDS-PAGE shows that  CsgA-his has been constitutive expressed in cell. Inducating that BBa_K1361007  works as anticipation.</TD></TR></TBODY></TABLE><P class="STYLE33">In  order to find the proper promoter strength to control the concentration of CsgA  monomer, Cell growth curve was drew to compare the CsgA secrete and self- aggregation  level when adapted with a strong promoter  BBa_J23100 and a  weaker one BBa_J23104. </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig  2. Cell growth curve of BBa_K1361007 and BBa_K1361004 that a relatively weaker  promoter was used to control the CsgA secrete level. </SPAN></TD></TR></TBODY></TABLE><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig 3. This figure shows the absorbency  of secrete protein in culture between BBa_K1361007 and parallel BBa_K1361004.  No significant divergence was found. And no self- aggregation was observed  without inducer.</SPAN></TD></TR></TBODY></TABLE><P class="STYLE33"> </P><P class="STYLE33">However,  there is no obvious divergence of the concentration of matrix soluble protein  between control samples (with empty plasmids) and cells that had been  transformed with both BBa_K1361007/BBa_K1361004 (Curli Fiber generator under  the control of Pbad promoter with CsgA modified by His tag) and BBa_K1361005 (the  outer membrane secrete device for curli fiber). Coupled with the failure of  CsgA-his purification from cell culture, suggesting that the outer membrane channel  protein CsgG and its assist proteins CsgE, F we using wasn’t functioning and  may get mutation.</P><P class="STYLE33"><STRONG>Part 2.  Congo Red (CR) Assay to Determine the Degree of Curli Fiber Polymerization</STRONG></P><P class="STYLE33">To  examine our controlling force on curli fiber inductively aggregation, we measured  the curli fiber polymerization properties after L-arabinose inducing by CR assay.  The experiments showed that adding of the inducer seemed not greatly affect the  curli fiber assembly. Regardless of induction, there would be decay at A480  within 10min after transferring cells into static condition.</P><P class="STYLE33">Speculating  the non-function of outer membrane secrete system, we restructured the out membrane  channel CsgEFG and send it to iGEM Headquarter (BBa_K1361005), while using it  to process further characterization. Result of which will be demonstrated in  our presentation. </P><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97"><SPAN class="STYLE33">Fig 4. The polymerization rate of curli fiber was determined  by CR assay.? After 1μM L- arabinose inducing at 0min, culture  medium was subpackaged  into glass tubes and cultivates under 37C, static. Samples were taken to filter  and suspended in 700μl 1XPBS; 300μl of cell suspension was used for OD600 cell  number normalization, and the remaining 400μl combined with 5X Congo Red (CR)  for a final concentration of 20μg/ml CR, and incubated for 5min at RT. The  cells and curli with bound CR were spun down at 15,000-x g for 5min, and 300μl  supernatant was removed for CR quantification. Concentration of CR in  supernatant was quantified by absorbance at 480nm. The amount of CR bound by  cells and curli were quantified by subtracting the Ab480nm of supernatant from  Ab480nm of 20μg/ml CR. </SPAN></TD></TR></TBODY></TABLE><TABLE width="100%" border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD height="115" bgcolor="#FFED97"><P class="STYLE33">Fig  5. The adding of 1μM L- arabinose inducer at 0min didn’t significantly affect  the CsgA aggregation in matrix judging by OD480. Which indicates the failure of  outer membrane secrete device. </P><P class="STYLE33"> </P></TD></TR></TBODY></TABLE><TABLE width="100%" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD bgcolor="#FFED97" class="STYLE33">Fig 6. This pic shows the color change of CR solution  after 1μM L- arabinose inducing. In the fist 20min, distinct  color change was observed. Nevertheless, this could be the expression of curli  fiber from <EM>E coli</EM> K-12’s own genome.</TD></TR></TBODY></TABLE><STRONG>⑤Future Work</STRONG><P class="STYLE33">We believe the enlightening idea of the  transformation from bio-signal to electrical signal will exert far-reaching  influence on the future society for the convenience and the help of highly  efficient computing and analyzing capacity of computers. Our project is born  ignited attributed to the belief, and as a result, we have devoted ourselves to  the theories and experiments constructing to the final project. However, some  work is expected to be done to perfect the project in the future due to the  limited time we have during the competition.</P><P class="STYLE33">Firstly, no ideal method have been found to  reverse the electroconductivity when the concentration of the seducer is lower  than the threshold value. Although we have paid great effort on searching information  on the problem and assumed that enzymolysis and the motivation of bacterial  flagella can destroy the nanowire composed by CsgA, no further experiments can  be carried on for the lack of time and grounded references.</P><P class="STYLE33">Secondly, the linker used in the experiment  of the detection of MMP-7 is obtained semiempirically. Thus, this linker will potentially  and probably effect the protein folding of CsgA and the substrate polypeptide, exert  negative influence on the aggregation of CsgA and the enzymolysis of the  polypeptide by MMP-7. Consequently, an optimal linker may be found through  future theoretical calculation and experiments to lead to better results.</P><P class="STYLE33">Finally, the application of the project  will be considered in the future. This project is endowed with innovative ideas  and constituted of various sub-modules, but the experiments only prove that the  system can be a great practical device to detect the existence of  protease(MMP-7). We believe the ideas embedded in the project can ignite our  ideas of the novel application of biosensors related to pharmacy, detection of  organics, medicine treatment, etc.</P><TABLE width="1230" border="5" cellpadding="0" cellspacing="0" bordercolor="#993300" bgcolor="#FFFFFF"><TBODY><TR><TD width="81%" bgcolor="#3A2108"><P><SPAN class="STYLE61">Tianjin University,Tianjin, China
Email:michaelss@tju.edu.cn</SPAN></P><P> </P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Tianjin/Project">http://2014.igem.org/Team:Tianjin/Project</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tianjin/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tianjin/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tianjin/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tianjin/Project&amp;oldid=356765" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>