"
Page
Discussion
View source
History
teams
Log in
 
Team:Tianjin/Notebook
From 2014.igem.org
Team:Tianjin2014/Safety
Protocols
    ·Genomic
    ·Vector
    ·Preparation  of competent
    ·Detection
Labnotes
Go to the top
 
 
 
 
 
 
Transf ibre
 
Welcome to Team Tianjin!
Home|
Project|
Judging|
Notebook|
Team|
Safety|
Human Practice|
①Protocol
1.Genomic
1.1 Extract genome
Add absolute ethyl alcohol to Buffer GD and  Buffer PW before using them.
(1)Obtain 1~5ml bacterium sample in centrifuge  tube, centrifuge at 10000rpm for 1 min and remove the supernatant.
(2)Add 200μl Buffer GA to the precipitate of bacteria, oscillate  it to suspension.
(3)Add 20μl Proteinase K.
(4)Add 220μl Buffer GB, oscillate for 15 sec, place it at 70℃ for 10 min and centrifuge shortly.
(5)Add 220μl absolute ethyl alcohol, oscillate for 15 sec and  centrifuge shortly.
(6)Transfer the obtained solution and precipitate  from the step above to a Spin Column CB3, centrifuge at 12000rpm for 30 sec and  discard the liquid.
(7)Add 500μl Buffer GD, centrifuge at 12000rpm for 30 sec and  discard the liquid.
(8)Add 600μl Buffer PW, centrifuge at 12000rpm for 30 sec and  discard the liquid.
(9)Repeat the eighth step.
(10)Put the Spin Column CB3 back into the  collection tube, centrifuge at 12000rpm for 2 min and discard the liquid. Place  it at room temperature for a few seconds to dry out the buffer.
(11)Transfer the Spin Column to a clean  centrifuge tube, add 50~200μl Buffer TE to the middle of column membrane, place it at room  temperature for 2~5 min, centrifuge at 12000rpm for 2 min and collect the  solution to the centrifuge tube.
(12)Detect the concentration of DNA.
1.2 PCR
1. Make up a master mix of everything into eachPCR  tube.
reaction system    1
20.0μL
5x Buffer
4.0μL
2.5 mMdNTP
2.0μL
Template 1
1.0μL
Forward primer
0.5μL
Reverse primer
0.5μL
Pfu DNA    polymerase
0.5μL
ddH2O
11.5μL
reaction system    2
20.0μL
5x Buffer
4.0μL
2.5 mMdNTP
2.0μL
Template 2
0.5μL
Forward primer
0.5μL
Reverse primer
0.5μL
Pfu DNA polymerase
0.5μL
ddH2O
12.0μL
2. Run the " PCR" program, and  adjust your extension time as described below:
Initial denaturation: 98°C for 5min
30cycles of:
95°C for 30 seconds
55°C for 30seconds
72°C for 30seconds
Final extension: 72°C for 10 min
2.Vector
2.1 Restriction Enzyme Digestion
To check if the two  selected restriction enzymes can perform effective catalysis in the same  solution
1. Mix DNA solution  with the suitable amount of the master mix.
Reaction system
25.0μL
DNA solution
17.0μL
10x FD Buffer
2.5μl
Restriction enzyme S
1.5μL
Restriction enzyme P
1.5μL
ddH2O
2.5μL
2. Pipette up and  down in the EP tube.
3. Incubate: 37°C for  30-40 min
2.2 Purification  or Gel Extraction of DNA
2.2.1 Purification
Add 60ml ethanol (96-100%) to Buffer PW before use  (see bottle label for volume). All centrifuge steps are in a conventional tabletop  microcentrifuge at room temperature (15-25°C).
1. Column equilibration: add 500ul Buffer BL to the  Spin Column CB2. Centrifuge for 1 min at 12,000 rpm (~13,400 × g) in a  table-top microcentrifuge. Discard the flow-throw, and then place Spin Column  CB2 in the collection tube.
2. Add 5 volumes of Buffer PB to 1 volume of the PCR  reaction or enzymatic reaction and mix. It is not necessary to remove mineral  oil or kerosene.
Note: For example, add 250μl of Buffer PB to 50μl  PCR reaction (not including oil).
3. Transfer the mixture to the Spin Column CB2. Let  it stand in refrigerator at 4℃  for 30  min. Centrifuge for 60s at 12,000 rpm (~13,400 × g) in a table-top microcentrifuge.  Discard the flow-throw, and then place Spin Column CB2 back into the same  collection tube.
Note: The maximum loading volume of the column is  800 μl. For sample volumes greater than 800 μl simply load again.
4. To wash, add 600 μl Buffer PW to the Spin Column  CB2 and place the Spin Column CB2 in refrigerator at 4℃ for 2 min, then centrifuge  for 60s at 12,000 rpm (~13,400 × g). Discard the flow-through, and place Spin  Column CB2 back in the same collection tube.
Note: If the purified DNA is used for the subsequent  salt sensitive experiments, such as ligation or sequencing experiment, it  suggests to let it stand for 2~5min after adding Buffer PW, and then  centrifuge.
5. Wash the Spin Column CB2 with 600 μlBuffer PW and  place the Spin Column CB2 in refrigerator at 4℃ for 2 min, then centrifuge for 60s  at 12,000 rpm (~13,400 × g). Discard the flow-through, and centrifuge for an  additional 2 min to remove residual wash buffer PW.
Note: Residual wash buffer will not be completely  removed unless the flow-through is discarded before this additional  centrifugation. Residual ethanol from Buffer PW may inhibit subsequent  enzymatic reactions.
6. Place the Spin Column CB2 in a clean 1.5 ml  microcentrifuge tube and open CB2 lid and stay at 60℃ temperature for 20 min. To elute  DNA, add 50 μl Buffer EB or deionized water (pH 7.0-8.5) to the center of membrane,  let the column stand for 15 minat 60℃,  and centrifuge for 2 min at 12,000 rpm (~13,400 × g).
7. Alternatively, for increased DNA concentration,  add the solution gained from step 6 to the center of membrane again, let the  columns stand for 15 minat 60℃,  and then centrifugefor 2 min at 12,000 rpm (~13,400 × g).
Note: The volume of Buffer  EB must be more than 30 ul, or it may affect recovery efficiency. What’s more,  the pH value of eluted buffer will have some influence in eluting, we suggest  chose buffer EB or distilled water (pH 7.0-8.5) to elute plasmid DNA. For  long-term storage of DNA, eluting in Buffer AE and storing at –20°C is  recommended, since DNA stored in water is subject to acid hydrolysis.
2.2.2 Gel Extraction of DNA
Column  balance step:Place a TIANGEN DNA column CA2 in a provided 2 ml collection tube.  Add 500μl Buffer BL to the TIANGEN DNA column, and centrifuge at 12,000 rpm for  1 min at room temperature. Discard liquid and place the TIANGEN DNA column back  into the same collection tube.
Cut  off the Single goal DNA band from the gel put it into the clean centrifuge  tube and weigh.
Add  same volume of Buffer PN more than the gel slice (0.1g gel account for  100μl). Incubate the mixture at 50°C for 10 min until the gel has  completely melted. During the time we can mix it by shaking or overtaxing the  tube.
Place  a TIANGEN DNA column CA2 in a provided 2 ml collection tube. Add the mixture in  last step to the TIANGEN DNA column, and place it at 4°C temperature for 30 min,then centrifuge at 12,000 rpm  for 60 sec . Discard liquid and place the TIANGEN DNA column back into the same  collection tube.
Add  600μl Buffer PW diluted with absolute ethanol into the TIANGEN DNA column and  place it at 4°C temperature for  2 min and centrifuge at 12,000 rpm for 60 sec at room temperature to wash the  column. Discard liquid and place the TIANGEN DNA column back into the same collection  tube.
Note:  Buffer PW Concentrate must be diluted with absolute ethanol before use. See  label for directions. If refrigerated, Buffer PW must be brought to room  temperature before use.
Repeat  the step 5.
Centrifuge  the empty TIANGEN DNA column at 12,000 rpm for 2 min to dry the column, and  place it at 60℃  temperature for 20 min. Do not skip this step, it is critical for the removal  of ethanol from the TIANGEN DNA column.
Place  the TIANGEN DNA column into a clean centrifuge tube. Add 50μl Buffer EB directly  into the middle of the column membrane and place it at 60℃  temperature for 15 min. Centrifuge for 2 min at 12,000 rpm to elute DNA. An  optional second elution will yield any residual DNA, though at a lower  concentration.
2.3 Ligation
1. Check the concentration  of DNA fragments and vector which are going to be ligated.
2. Calculate the  amount of part A/partB and vector added, based on the fragment length. Note  that a ligation using a molar ratio of 1:3 vector to inserts.
3. Add DNA/buffer and  ligase together in the EP tube.
Reaction system
10.0μL
Part A
A.0μL
Vector
V.0μl
10x T4 Ligase Buffer
1.0μL
T4 Ligase
1.0μL
Add ddH2O until the total    volume is 10.0μL
4. Mix the reaction  by pipetting up and down gently and microfuge briefly.
5. Incubate at 22°C  for 1 hour or at 16°C for overnight.
2.4 Transformation (Competent cell: E.coli DH5α)
(1)Thaw  DH5α cells on ice. Separate the cell into sterile precooling centrifuge tubes.
(2)Add  5µl of DNA to the cells. Mix the solution slightly and stand for 30 min on ice.
(3)Cells  are incubated for 30 seconds at 42℃. Put cells back on ice quickly and  stand for 2 min.
(4)Add 250µl  LB broth(without antibiotics) to each centrifuge tubes. Incubate for 45 min at  37℃ on 150 rpm  shaker.
(5)Spread  100µl onto the petric dish with LB agar(with antibiotics).
(6)Grow  overnight at 37℃.
2.5 Colony PCR
1. Make up a master mix of everything into one  microcentrifuge tube.
reaction system
15.0μL
dNTP
1.5μL
10xTaq Buffer
1.5μL
up primer
0.5μL
down primer
0.5μL
Taq DNA polymerase
0.15μL
ddH2O
10.85μL
2.Run the "Simple PCR" program, and adjust your  extension time as described below.
The "Simple PCR" program
Initial denaturation: 98°C for 5min
30cycles of:
95°C for 30 seconds
55°C for 30seconds
72°C for 35seconds
Final extension: 72°C for 10min
Plus.1 Luria Bertani Medium
Tryptone
10g/L
Yeast extract
5g/L
NaCl
10g/L
Solid Luria Bertani medium: Add 15g/L Agar into Luria    Bertani medium
Plus.2 DNA Agarose Gels
1. Prepare a 1%  weight-to-volume agarose gel(take 100ml as example)
2. Dilute stock of  50×TAE to 1×with ddH2O.
3. Measure 100 ml  of 1×TAE buffer.
4. Transfer 1×TAE  buffer to Erlenmeyer flask.
5. Weigh out  enough agarose to make 1% gel. (1% of 100mL is 1.0 g)
6. Transfer  agarose to Erlenmeyer flask.
7. Melt agarose in  microwave, stirring every 15-20 seconds until completely melted.
8. Allow gel to  cool until Erlenmeyer flask can be handled comfortably. Then add 1:20 volume  ratio (5μl) GelRed Nucleic acid dye to the gel and shake the Erlenmeyer flask  to dye the gel well.
9. Pour agarose  into gel tray, assemble gel pouring apparatus by inserting gate into slots.
Plus.3 Agarose Gel Electrophoresis
1. Allow agarose to cool, place the gel in the  apparatus rig with the wells facing the negative end (black-colored).
2. Fill the rig with 1x TAE buffer.
3. Load 5µL of DNA maker into lane.
4. Mix 1µL of 6x loading buffer with 5µL DNA sample,  load them into lane.
5. Run at 150V for 20 min.
6. Use Gel imaging system check gel.
7. Take picture for gel.
3.Preparation of competent
3.1 Competent  cell
1. Take bacterium  fluid in 10 ml centrifuge tube, centrifuge it at low speed ( 4000rpm ),for 5min  at 4℃.
2. Pour out on the  clear liquid, by adding 5ml10 % glycerine solution, bacteria will weight new  suspension. Then centrifuge it at the same speed and for the same time.
Repeat this step  three times.
3. In the end, add  500μL 10 %glycerine solution to the bacteria sediment to make the residue  resuspension.
4. Subpackage the  bacteria solution to 5 EP tubes, so there is 100μLcompetent cell solution in  each EP tube. Keep the competent cell solution in -80℃refrigerator.
3.2 Electrotransformation
1. Add 5μPCR  product into 100μLcompetent cell solution, then put this  mixture into 0.2cm electroporation cuvettes.
2. Then use BioRad  Gene PulsenⅡ to electroporate our bacteria. Set the parameter  as 25μF，2.5KV.
3. After electroporation  add 1mL LB nutrient solution into our bacteria, the put this bacteria system in  37℃，150rpm incubator for 2 hours to removePIJ790  from our system.
4. Besmear the recoveryed  bacteria liquidon the solid medium which has the corresponding resistance.
5. Culture the  bacteria overnight at 37℃
 
4.Detection
4.1 Cell and protein OD
Take  several bottles of liquid culture medium and add 5μL of bacteria in every one.  Meanwhile, add 1μL of inducing agent in them except the control group. Placed  the bottles in a shaking culture box.
After  one hour, take out the liquid culture medium and shake them. Remove 2500μL  bacteria under sterile conditions and add it in a clean cuvette, dilute it if  necessary.
Adjust  the absorbance of spectrophotometer of 600nm, then measure and record the ODs  of the removed bacteria.
Remove  1500μL bacteria from the cuvette and add it in a clean EP tube, then centrifuge  at 6300rpm for 10 min.
Extract  supernatant 500μL and use it to rinse Syringe Filters, then take out the  residual supernatant and add it in a new EP tube.
Remove 500μL supernatant and add it in a new  clean cuvette, then add 2500μL shook Kaumas Coomassie brilliant blue G-250 and  stand for 2 min.
Adjust  the absorbance of spectrophotometer of 595nm, then measure and record the ODs  of the csgA protein.
Repeat the step 2~7 until adequate data.
4.2  Polymerized curli fiber
1.add 5μL target glycerin bacteria to 100mL nutrient solution.Put  bacterium solution in 37℃，150rpmincubator , cultivate it for 7 hours.
2.Then add 1μL  1mol/LL arabinose to  100mL bacterium solution. Subpackage100mL bacterium solution into 10 test tubes  averagely. Put subpackaged bacterium solution into 37℃Constant temperature and humidity incubator.
3.Take out each test tube as the schedule showed below.
10     20       20         30       30        30        30        60    /（min）
4.Filter the bacteria solution in each test tube.Use 700L PBS solution to  make the residue resuspension. Add  0.8μLcongo red (10g/L) into 400μLdense medium, then water  bath this system at 25℃ for 5 min.
5.In the  end, centrifuge this system at 15000g centrifuge force for 5min, then measure  the absorbance of the supernate.
②Labnotes
June
 
We improved our initial idea and did more  research about our project, including finding more information about properties  of csgA and some particular application of this curli fibre. We made a rough  schedule to arrange the experiment in the summer. And most of the protocols  were established.
July
 
The first protocol we did was to extract  the genome of our E.coli strains DH5a, then we designed several different  primers in order to amplify the target gene sequences from the genome through  PCR device. Those primers were designed to add traditional iGEM suffix and  prefix on the initial sequences to introduce the restriction sites we need. The  PCR product was confirmed by gel electrophoresis. CsgA and other necessary  parts were confirmed and isolated and extracted. We attempted to ligate our  sequence on the iGEM chloramphenicol backbone on July 10th. We  need to add different promoters to those sequences and rearrange the orders of  our parts. And there would be 4-5steps before we got our ideal plasmids. Each  single step included ligation, transformation, confirming consequences and extracting  plasmid of this step. The first step to construct these parts went well and we  successfully added different promoters to the parts. However, we were trapped  in the second step. We can’t efficiently isolated the parts using restrict  enzyme. We finally found the reason on July 25th that we designed  the wrong primers---we mixed the order of the four restriction sites when we  design these primers. At the same time, we found that there was restrict site  EcoRI inside the sequence of csgA and csgE, which need to be repaired. So we designed  the new primers and started to reconstruct the parts we need on July 28th.
August
 
In august, our team started to reconstruct  our parts. We designed to put our target gene into two different kinds of  plasmids and transfer them into one BL21 cell. We put csgA, csgB into chloramphenicol  backbone pSB1C3 with three kind of inducible promoters T7 , Pbad and Plam and  two kind of constitutive promoters P1 and P2. We put csgE, csgF and csgG into  another kind of backbone pSB3K3 with two constitutive promoters P2 and P3. We  planned to construct 18 parts in total and completed several of them before  august 30th.
Most E.coli strains have csgA in their own  genome. Therefore, we began to knock out the csgA, csgB, csgC, csgE, csgF, csgG  in the genome of backbone cell BL21 on august 25th
September
 
We constructed the rest of the biobricks  before September 15th and then we began to transfer the E.coli BL21  we made with two different plasmids we had constructed and cultured the cells  at the LB plates that contained both chloramphenicol and kanamycin  to choose the colonies that were successfully transferred by two different  plasmids. At the same time, we began to contact the instruments and devices we  need. We designed a small device for csgA to properly assembling and we made  our own Au electrode.
October
 
We  sent some of our biobricks to iGEM on October 10th. And most parts  have been transferred into backbone cell so that we were able to measure some  properties of the protein. Our team began to examine the OD600 to make the cell  growth curve on October 15th and measured OD595 of efflux protein with  G-250 Coomassie brilliant blue. Then we use arabinose to induce the expression  of csgA with Pbad promoter and measured the polymerization of curli fibre with  congo red. Also, we designed electric device which could be used to measure  tiny current to examine electric conduction of modules that have affinity with  n ano gold. Our team established the modeling with results  of these tests and conducted some evaluation and analyze.
 
Tianjin University,Tianjin, China
Email:michaelss@tju.edu.cn
 
Retrieved from " http://2014.igem.org/Team:Tianjin/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
