<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Yale_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Yale/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:Yale"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Yale/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Yale/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:Yale/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Yale/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A></DIV><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Yale/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=Yale"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><H1>Modeling</H1><H3>Metabolic Modeling w/ Flux Balance Analysis (FBA)</H3><P>
    The goal of the Objective #2 was to engineer a metabolic pathway by which E. coli could metabolize one of two byproducts of PET degradation: Ethylene glycol (EG). For this objective, we heavily relied on metabolic modelling with Flux Balance Analysis (FBA) to understand what genetic changes to E. coli we needed to make for E. coli to metabolize EG.
</P><P>
    Also known as antifreeze, EG is a hazardous chemical used as an industrial coolant in car engines and airplanes. Antifreeze is often released into the environment via leaks in car engines or, more commonly, as runoff from airplane de-icing operations. In high doses, EG can cause kidney failure in humans and animals.
</P><P>
    E. coli has native enzymes (fucO and aldA) capable of degrading EG and metabolizing its products. However, the native enzymes are not enough to allow E. coli to grow using EG as a sole carbon source because the flux through the EG metabolic pathway is too small.
</P><P>
        Figure 1. EG Metabolism in E.coli. The fucO and aldA genes are critically important for degradation.
    </P><P>
    Over the course of 10 weeks, we used Flux Balance Analysis to model E. coli metabolism. Specifically, we used the COBRA Toolbox for MATLAB to simulate the metabolism of the ancestral E. coli strain in the presence of increased glycolate, a downstream metabolite of EG metabolism from fucO and aldA.
</P><P>
    This modelling allowed us to identify genes whose overexpression (or down-regulation) would increase flux through the EG metabolic pathway. We then subsequently used MAGE to overexpress these genes (by altering their ribosome binding sites to be closer to the consensus Shine-Dalgarno sequence) and adaptive evolution (coupled with EMS or UV mutagenesis) to evolve E. coli capable of growing on EG as a sole carbon source. MAGE is a genomic engineering technique that allows multiple sites in the genome to be targeted for precision editing at the same time.
</P><P>
        Figure 2. Metabolic flux diagram resulting from FBA of the starting E. coli strain’s metabolism in the presence of increased glycolate. Certain genes (boxed in red) were predicted to be expressed at levels greater than 2 orders of magnitude higher than basal expression levels. The TCA cycle represents the collective target of this data. FBA was also used to identify genes (boxed in green) that had moderately increased expression but are still required for succinyl CoA, supplementary to the TCA cycle.
    </P><P>
    Flux Balance Analysis allowed us to explore many potential avenues of upregulating flux through the natural propylene glycol (PG) metabolic pathway in E. coli. From previous literature, this compound and pathway were identified by researchers as a necessary precursor and target, respectively, to genetically engineer E. coli to degrade ethylene glycol (EG). Of the two glycols, both are very similar in structure. There is also pre-existing machinery in wild-type mutants to degrade propylene glycol as a function of the fucose degradation pathway.
</P><P>
        Figure 3. Chemical structure of propylene glycol (left) and ethylene glycol (right)
    </P><P>
    Our metabolic model, set to optimize cell growth/proliferation when grown in excess glycolate, identified many genes whose upregulation or downregulation would be useful to the metabolism of EG. Glycolate was used in the model because it is the product of PG and EG degradation after the action of the fucO and aldA enzymes.
</P><P>
    After identifying these genes, we designed Multiplex Automatable Genetic Engineering (MAGE) single-stranded oligonucleotides needed to upregulate genes (by improving their RBS’s) or to silence their expression altogether (by deleting their RBS’s). Electroporating these oligonucleotides into MAGE-compatible strains of E. coli would allow these changes to all be made simultaneously. THe MAGE-compatible strain of E. coli we decided to use was EcNR1. By doing this, we hoped to increase overall flux through the pathway and optimize EG degradation.
</P><P>
    MAGE, however, proved difficult to implement. Of the 10 genes we identified and attempted to target, few were knocked out. However, this low genomic editing efficiency is likely due to experimental error and unfamiliarity with the MAGE protocol. We plan to re-do this experiment again with a better understanding of the MAGE protocol and a more robust experimental design. Nonetheless, the metabolic modelling done here has greatly increased our understanding of the EG metabolic network and will be essential in our efforts to understand what mutations (in our EMS and UV mutated E. coli strains) led to an increased ability to utilize ethylene glycol as a sole carbon source.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>