<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Yale_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Yale/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:Yale"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Yale/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Yale/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:Yale/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Yale/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:Yale/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A></DIV><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Yale/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=Yale"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1>Description</H1><P>
    Polyethylene terephthalate (PET) is a polymer used to make plastic products ranging from synthetic fibers to water bottles. Although PET can be recycled, large amounts of PET end up accumulating in the environment as pollution. A recently discovered bacterium named Ideonella sakaiensis was found to degrade PET by using two enzymes, PETase and MHETase, to break PET into its two constitutive monomers: ethylene glycol (EG) and terephthalic acid (TPA). However, I. sakaiensis’ inability to breakdown PET on a practical time scale undermines its usefulness in eliminating PET pollution. Our project aimed to tackle PET pollution by genetically engineering a synthetic Escherichia coli and Aceintobacter baylyi co-culture to degrade and metabolize PET. We used the following three-pronged approach: (1) engineer E. coli to express and secrete PETase and MHETase for extracellular degradation of PET, (2) engineer E. coli to uptake and metabolize EG, and (3) engineer A. baylyi to uptake and metabolize TPA by expressing foreign TPA utilization genes from Comamonas sp. strain E6. Once individually completed, these three components will then be combined to create a synthetic bacterial co-culture of cooperative E. coli and A. baylyi amino acid auxotrophs. Since both E. coli and A. baylyi are more characterized than I. sakaiensis and also capable of high-throughput mutagenesis, PET degradation and metabolism pathways in an engineered synthetic E. coli and A. baylyi co-culture potentially could be optimized to be even more efficient than those natively found in I. sakaiensis.
</P></DIV><DIV class="column full_size"><H1>References</H1><UL><LI>
Bornscheuer, U. T. (2016). MICROBIOLOGY. Feeding on plastic. Science, 351(6278), 1154-1155. doi:10.1126/science.aaf2853
</LI><LI>
Boronat, A., Caballero, E., &amp; Aguilar, J. (1983). Experimental evolution of a metabolic pathway for ethylene glycol utilization by Escherichia coli. J Bacteriol, 153(1), 134-139.
</LI><LI>
Kint, D., &amp; Muñoz-Guerra, S. (1999). A review on the potential biodegradability of poly(ethylene terephthalate). Polymer International, 48(5), 346-352. doi:10.1002/(SICI)1097-0126(199905)48:53.0.CO;2-N
</LI><LI>
Romero-Silva, M. J., Méndez, V., Agulló, L., &amp; Seeger, M. (2013). Genomic and Functional Analyses of the Gentisate and Protocatechuate Ring-Cleavage Pathways and Related 3-Hydroxybenzoate and 4-Hydroxybenzoate Peripheral Pathways in Burkholderia xenovorans LB400. PLOS ONE, 8(2), e56038. doi:10.1371/journal.pone.0056038
</LI><LI>
Sasoh, M., Masai, E., Ishibashi, S., Hara, H., Kamimura, N., Miyauchi, K., &amp; Fukuda, M. (2006). Characterization of the terephthalate degradation genes of Comamonas sp. strain E6. Appl Environ Microbiol, 72(3), 1825-1832. doi:10.1128/aem.72.3.1825-1832.2006
</LI><LI>
Tanasupawat, S., Takehana, T., Yoshida, S., Hiraga, K., &amp; Oda, K. (2016). Ideonella sakaiensis sp. nov., isolated from a microbial consortium that degrades poly(ethylene terephthalate). Int J Syst Evol Microbiol, 66(8), 2813-2818. doi:10.1099/ijsem.0.001058
</LI><LI>
Wang, H. H., Isaacs, F. J., Carr, P. A., Sun, Z. Z., Xu, G., Forest, C. R., &amp; Church, G. M. (2009). Programming cells by multiplex genome engineering and accelerated evolution. Nature, 460(7257), 894-898.
</LI><LI>
Yang, Y., Yang, J., &amp; Jiang, L. (2016). Comment on &quot;A bacterium that degrades and assimilates poly(ethylene terephthalate)&quot;. Science, 353(6301), 759. doi:10.1126/science.aaf8305
</LI><LI>
Yoshida, S., Hiraga, K., Takehana, T., Taniguchi, I., Yamaji, H., Maeda, Y., . . . Oda, K. (2016a). A bacterium that degrades and assimilates poly(ethylene terephthalate). Science, 351(6278), 1196-1199. doi:10.1126/science.aad6359
</LI><LI>
Yoshida, S., Hiraga, K., Takehana, T., Taniguchi, I., Yamaji, H., Maeda, Y., . . . Oda, K. (2016b). Response to Comment on &quot;A bacterium that degrades and assimilates poly(ethylene terephthalate)&quot;. Science, 353(6301), 759. doi:10.1126/science.aaf8625
</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>