Team:Yale/Results
HOME
TEAM
Team Members
Collaborations
PROJECT
Description
Notebook
InterLab
Model
Results
Attributions
PARTS
Parts Overview
Basic Parts
HUMAN PRACTICES
Human Practices
JUDGING FORM ⇗
OBJECTIVE #1: Engineer E. coli to degrade PET
The goal of Objective #1 was to engineer E. coli to degrade PET by expressing and secreting PETase and MHETase, two genes originally found in Ideonella sakaiensis.
Plastic Degradation Constructs
Two separate plasmids were constructed in our lab which contained either PETase or MHETase. These plasmids were constructed with a T7 RNA polymerase promoter attached in order to encourage over-expression of the proteins. The PETase containing constructs utilized GFP as a reporter gene and the MHETase containing constructs utilized RFP. Both constructs were built with T1 terminators at their ends.
Within E. Coli, the over-expressed PETase or MHETase now had to be secreted from within the cell. To do this, the PETase or MHETase was fused to a secretion tag. Five different secretion tags were used in testing with the objective of finding the most efficient secretion tag. Four of these tags were derived from Pseudomonas (S.m Hemolysin, S.m Protease 1, S.m Protease 2, and P.a Protease) and one was found native to E. Coli (yebF). In the end, via a pNP assay, yebF was found to be the best candidate for the secretion of the proteins.
In order to assess the secretion of PETase, GFP was fused to PETase as a reporter gene.
In order to assess the secretion of MHETase, RFP was fused to MHETase as a reporter gene.
From reading previous literature and work with PETase and MHETase, it was evident that the most glaring problem that faced the project was the speed at which the two enzymes worked to degrade PET into its substituents of EG and TPA. Therefore, as part of our project, we strove to improve functionality of PETase and MHETase; the main way we decided to approach this issue was by combining the PETase and MHETase proteins into either a fusion protein or a polycistronic construct via Gibson Assembly.
Using the PETase containing construct as a foundation, we inserted MHETase directly after PETase using Gibson Assembly; this was in the hopes of creating a PETase-MHETase fusion protein to increase PET degradation efficiency. Additionally, we believed that creating a fusion protein would lessen the metabolic burden on E. Coli and also eliminate any concern about the regulation of each protein independently. The secretion tag that we utilized for this construct was yebF, as it was previously determined as the most efficient secretion tag.
Again, by using the PETase containing construct as a foundation, we inserted MHETase directly after PETase using Gibson Assembly; however, this time we attempted to create a polycistronic construct by not only inserting MHETase, but also another ribosomal binding site, the original RFP reporter gene, and another yebF secretion tag. We constructed this polycistronic plasmid in parallel to the fusion protein construct because we were not sure how creating a fusion protein would affect PETase/MHETase functionality.
A PCR colony screen was used to verify assembled plasmids. On the following gels: Gel 1 screened colonies containing a plasmid with PETase and MHETase expressed as a fusion protein. Gel 2 screened for colonies with PETase and MHETase expressed from a polycistronic transcript.
Construct Testing
Via the T7 promoter, PETase and MHETase were overexpressed in the colonies of E. Coli grown and subsequently checking for GFP/RFP under a blue light yielded positive results for expression of the construct.
A colorimetric para-nitrophenol (pNP) assay was used to evaluate the effectiveness of various secretion systems. In the presence of PETase, pNP is hydrolyzed and produces a yellow product that can be quantified with spectroscopy.
One approach we attempted to utilize in quantifying protein expression was running a protein gel in the hopes of identifying a corresponding band which fit our protein’s weight. Repeated attempts however, did not yield the desired band. We suspect that the protein may be insoluble and therefore difficult to view on the gel. Furthermore, the protein may be too large, getting stuck at the top of the well, or may simply be too diluted to appear on the gel as well.
Another approach we used to identify protein functionality was LC-MS. We hoped to find bands in supernatants of colonies grown with PET that corresponded to the bands TPA produces on the LC-MS. This would indicate PET degradation; however, a large problem we found working with TPA is that it is insoluble with water, and using DMSO/other organic solvents did not prove useful in the LC-MS.
Future Direction
Moving forward, we hope to obtain cleaner gel bands with our protein gels in order to finally obtain bands which match our desired protein size. This would be accomplished through retransformation of new colonies for our constructs as well as full gene sequencing in order to guarantee the success of our transformations.
The following reference curves were obtained while trying to identify degraded products in solution with LC-MS. As you can see, the scale of TPA is 10 times less than the BHET curve, making it difficult to view TPA on LC-MS results. In the future, we would like to find a way to properly view degraded products on the LC-MS in order to better verify protein functionality.
Finally, the last step we’d like to use to move forward would be mutation of the enzyme once protein secretion is identified. Since PETase and MHETase are still newly discovered proteins and not too far along evolutionary-wise, mutations and directed evolution may yield better protein secretion and degradation of PET plastic.
OBJECTIVE #2: Engineer E. coli to metabolize ethylene glycol (EG)
The goal of the Blue Team (Objective 2) was to engineering a metabolic pathway by which E. coli could metabolize one of two byproducts of PET degradation: Ethylene glycol. Ethylene glycol (EG), also known as antifreeze, is a hazardous chemical used as an industrial coolant in car engines and airplanes. Antifreeze is often released into the environment via leaks in car engines or, more commonly, as runoff from airplane de-icing operations. In high doses, EG can cause kidney failure in humans and animals.
E. coli has native enzymes (fucO and aldA) capable of degrading EG and metabolizing its products. We sought to upregulate this metabolic pathway by means of Multiplex Automated Genome Engineering (MAGE), EMS mutagenesis, UV mutagenesis, and adaptive evolution. Over the course of 10 weeks, we used Flux Balance Analysis to identify genes whose overexpression increased flux through the EG metabolic pathway. We used MAGE to overexpress these genes (by altering their ribosome binding sites) and adaptive evolution coupled with EMS or UV mutagenesis to evolve a strain of E. coli that is capable of growing on EG as a sole carbon source.
Approach
Figure 1. EG Metabolism in E.coli. The fucO and aldA genes are critical for degradation of EG. In normal E.coli cells, however, they are expressed in very small quantities. Source: BioCyc Database.
Previous literature suggested that E. coli mutants capable of digesting propylene glycol (PG) were more likely to evolve the ability to digest ethylene glycol (EG) compared to wild-type E.coli. Our experimental design reflects these findings. After cells displayed sufficient growth in minimal media M9 + PG, they were transferred to M9 + EG media for further rounds of selection.
We first attempted to increase flux through the natural E. coli ethylene glycol metabolic pathway by means of a high copy plasmid contain the genetic code for the fucO and aldA genes. After repeated attempts and successful transfection, we found that the presence of the plasmid did not increase PG or EG metabolism when compared to normal wild type E. coli. Both strains failed to grow in minimal media + 0.2% PG and minimal media + 0.2% EG.
Using FBA (Flux Balance Analysis) we identified genes whose upregulation or knock-out would increase flux through the EG metabolic pathway. We endeavored to upregulate or downregulate these genes by altering their ribosome binding sites towards consensus or deleting the from the genome entirely via MAGE (Multiplex Automatable Genetic Engineering). Mutation via Ethyl methanesulfonate (EMS) or Ultraviolet radiation was also employed to create mutant E. coli capable of metabolising PG and EG.
Metabolic Modeling
Link to Model Page
Multiplex Automatable Genetic Engineering (MAGE)
Single-stranded MAGE oligo libraries were designed to optimize RBS’s of the genes identified from FBA. The starting E. coli strain was then electroporated with the MAGE oligo library and recovered in a minimal media with EG as a sole carbon source multiple times.
Figure 2: Images detailing MAGE mechanism and methods of DNA insertion or deletion. Source: Wikimedia Commons.
Initial attempts with MAGE proved unsuccessful. It was difficult to ensure that MAGE oligos were in fact being inserted into the genome without the use of a lacZ MAGE oligo that knocked out the lac repressor, allowing visualization of successful oligo integrations on X-gal plates.
MAGE protocols were performed on the aforementioned genes 3 times to ensure a large population of altered cells. Despite successful rounds of MAGE, MAGE cells failed to grow in propylene glycol (PG) or ethylene glycol (EG), indicating the need for experimental redesign/troubleshooting.
Mutagenesis & Adaptive Evolution
In addition to utilizing MAGE to engineer EG metabolism, we also used two forms of mutagenesis to create strains capable of EG metabolism. Here the adaptive evolution design (in which cells capable of growing in PG were isolated and then moved into EG) was implemented as shown below.
We used 2 methods of mutagenesis:
(1) Ethyl methanesulfonate (EMS) mutagenesis: EMS is a carcinogenic agent capable of introducing point mutations in cells via nucleotide substitution. Often, G:C pairs can become A:T pairs. E. coli cells were exposed to two rounds of 1.5% EMS for 60 minutes.
(2) Ultraviolet mutagenesis: E. coli cells were subjected to 30,000uJ of UV radiation at 254 nm using a 2W Germicidal FG8T5 lamp before being placed into minimal media with 0.2% PG.
After mutagenesis, cells were grown in minimal media + 0.2% PG. Once sufficient growth had occurred, these cells were spun down, washed 3 times and plated on 0.2% EG plates. Colonies derived from these plates were introduced to minimal media + 0.2% EG liquid cultures for further growth. OD readings were monitored daily. Below is a figure depicting the growth of cells in EG.
Figure 3.  Growth of EMS and UV E. coli strains in M9 + EG media. The solid line represents an E. coli strain transformed with a T7 RNA polymerase promoter (E1T7) and exposed to 1.5% EMS mutagenesis. After two rounds of EMS, the cells were subjected to three rounds of selection. The cells were plated on M9 and EG plates for survival and subsequently grown up in M9 + EG liquid culture, with OD readings taken periodically. The dotted line represents an E. coli strain with a mutS KO (EcNR2) exposed once to 30,000uJ of UV radiation at 254 nm. The dashed line represents a negative control culture of EcNR2 grown in parallel with the other strains.
Several EMS and UV strains were identified that were able to metabolism EG as a sole carbon source. Their growth curves were analyzed in 0.2%EG + minimal media over the course 56 hours. The data is shown below.
Figure 4. EMS and UV cells growth curves in 0.2% EG (8.7.18). Growth profiles indicated improved EG metabolism when compared with wild type strains, which exhibited no growth.
Mutagenesis methods, coupled with adaptive evolution via the utilization of PG, then EG as a carbon source to select capable mutants, enabled us to evolve E.coli strains capable of metabolising ethylene glycol significantly better than wild type E. coli.
Bioprospecting
Ethylene glycol can be found in runoff from industrial plants and airports. In an interesting experiment, soil samples were collected from the Yale West Campus Oyster river and plated on minimal media containing ethylene glycol. Samples were washed several times with PBS and re-plated to ensure that all possible contaminants had been removed.
Remarkably, cells grew on the EG plates. Subsequent plating and further analysis using 16S RNA sequencing identified these soil bacteria as a strain of Pseudomonas. These soil cells were able to grow particularly well in 0.2%EG + minimal media. Further work must be done to understand the mechanisms that allow them to metabolise EG as a sole carbon source. Insights into this area might allow us to engineer better EG metabolism in E. coli mutants.
Figure 5. Pseudomonas cells grown in ethylene glycol minimal media. Cells were isolated from soil samples collected on the banks of the Yale West Campus Oyster river. Soil samples were washed multiple times in PBS, then plated on minimal media containing ethylene glycol (EG) as a sole carbon source. After three rounds of washing and plating, single colonies were isolated and sequenced using 16S rRNA oligos. Pseudomonas isolated from river soil samples demonstrates promising growth on EG. As such, it may prove useful in metabolizing the EG byproducts of PET degradation.
Compared to the UV mutagenized clones and the EMS mutagenized clones, the soil bacterium were able to grow the best in EG minimal media as can be seen from the following data:
Future Direction
In the future we plan to do the following: (1) Use more Flux Balance Analysis to model metabolites, identify unnecessary pathways that can be turned off, and modify resource availability in an E.coli model, (2) Sequence mutated strains to find mutations responsible for EG metabolism, (3) Continuously adaptively evolve E. coli in eVOLVER, and (4) Sequence bioprospected strain to look for EG metabolism machinery and consider porting it into E. coli.
OBJECTIVE #3: Engineer A. baylyi ADP1 to metabolize terephthalic acid (TPA)
The focus of objective #3 was to engineer a strain of bacteria to break down terephthalic acid (TPA). To do so, we would need two foreign gene clusters: (1) one from a strain of Comamonas that would allow for the degradation of TPA to an intermediate, protocatechuic acid (PCA), and (2) one from Acinetobacter baylyi ADP1 to convert PCA into succinate, where it would be fed into the citric acid cycle.
E. coli vs A. baylyi Growth in PCA
For this summer, we first worked on the second half of the pathway, converting PCA into succinate to feed into the citric acid cycle. This would allow us to feed in PCA as a sole carbon source to select for strains that better metabolized PCA. Initially, we were not sure whether or not to engineer everything to be housed in E. coli or to have A. baylyi ADP1 house the pathway instead. So, we first did a head-to-head comparison between E. coli and A. baylyi. We amplified the native  PCA degradation genes from A. baylyi ADP1 and cloned them into a vector with a T7 promoter for heterologous expression in E. coli. Then, we compared the growth of A. baylyi and E. coli in minimal media with PCA as a sole carbon source. Since the A. baylyi vastly outperformed the E. coli, we decided to proceed with using A. baylyi.
Improving PCA Metabolism
Having decided to pursue the sole usage of A. baylyi for downstream experiments, our next goal was to see if we could improve PCA metabolism. We did this using two methods: (1) an EMS mutagenesis approach and (2) a rationale genomic engineering approach. The EMS mutagenesis approach we took was essentially the same as the approach used in Objective #2. We exposed the ancestral strain of A. baylyi ADP1 to the EMS mutagen, then recovered the cells in PCA minimal media. After that, we back diluted continuously over a period of 3 weeks and then screened isogenic clones from the population to determine which ones grew the best in PCA minimal media. The genomic engineering approach was based off our knowledge of the pcaU regulator gene in the literature. PcaU strongly represses the PCA gene cluster in the absence of PCA but activates it in the presence of PCA. So, we wanted to see if we could improve PCA metabolism by replacing the pcaU regulator gene with a strong, constitutive Anderson promoter. To do this, we created a DNA cassette containing Anderson Promoter 119 and a CAT gene that would impart chloramphenicol resistance for selection. By attaching 50bp homology arms on flanking the cassette that were homologous to genomic regions flanking the pcaU regulator gene, we were able to use Lambda Red recombineering to knockout the pcaU regulator and replace it with a strong, constitutive promoter.
After generating the mutagenized adaptively-evolved and knockout strains, we then compared their growth profiles in PCA minimal media. Although knocking out the pcaU regulator appeared to marginally improve growth in PCA minimal media, the mutagenized adaptively-evolved strain grew the best in PCA minimal media by far. In an attempted to quickly pinpoint the mutations responsible for the improved growth in PCA, we performed targeted sequencing of key sites that might have been linked to improved growth. Regions we sequenced included: the pcaU regulator gene, the PcaU operator sequence, and the pcaK permease that transports PCA into the cell. We found no mutations in any of the regions above. Further sequencing will be needed to determine the mutations responsible for improved growth in PCA.
Designing a Co-Culture
Our next step was to design a co-culture between E. coli and A. baylyi. Because our project’s final vision hinged upon the ability of the two strains to grow together, we needed to find some way to engineer some form of cross-dependency between the strains. This would force both populations to co-exist without one strain out-competing the other. Because there was no established method for generating such a co-culture in the literature, we decided to utilize a system that was initially described for designing a co-culture between two E. coli strains. The underlying mechanism behind the system was a pair of complimenting amino acid auxotrophs. Since the system seemed like it would also work for other organisms, we decided to implement the system as a solution to our co-culture design challenges. Using Lambda Red recombineering, we knocked out the lysA gene in E. coli and the leuA gene in A. baylyi, making them lysine and leucine auxotrophs, respectively. This would make the E. coli dependent on the A. baylyi for lysine production and A. baylyi dependent on the E. coli for leucine production.
After generating the two knockout strains, we first verified that they were true auxotrophs by growing the strains in monocultures, with and without amino acid supplements. Since we only observed growth with the addition of amino acid supplements, we were confident that our strains were genuine auxotrophs. Next, we then inoculated both strains into minimal media containing glucose without any amino acid supplements. We observed that the strains were, in fact, growing when both present in the culture, showing that they were able to grow using the amino acids produced by the complementing strain.
Now that we had verified that the amino acid auxotrophs were able to grow by relying on each other’s amino acid production, we also wanted to study their population dynamics and to see how the complementing amino acid deficiencies would influence the ratio of E. coli and A. baylyi in the culture. We were able to assay for the population ratios by plating aliquots of the culture onto X-gal plates. Since E. coli has the lacZ gene, they would form blue colonies while A. baylyi would form white colonies.
After 60 hours of growth, we observed that in the control culture, in which we inoculated both the ancestral E. coli and ancestral A. baylyi strains, the E. coli completely outgrew the A. baylyi. In fact, we observed essentially zero white colonies from the control co-culture. In the auxotroph co-culture, however, after 60 hours of growth, we observed both blue and white colonies, showing that the complementing amino acid deficiencies necessitated the presence of both E. coli and A. baylyi to be present in sufficient numbers for the population to continue growing. We then also quantified the ratio of E. coli to A. baylyi by counting the CFUs on the X-gal plates.
Future Direction
An immediate next step for this objective would be to introduce the second half of the TPA degradation pathway. The second half of the pathway is encoded by genes from Comonas sp. strain E6. Already, we have had the genes synthesized as gBlocks by IDT and are in the process of cloning the genes into our mutagenized adaptively-evolved A. baylyi that can grow using PCA. Since the TPA gene cluster from Comonas converts TPA into PCA, we expect that heterologously expressing the gene cluster in A. baylyi will allow the A. baylyi to subsist off TPA as a sole carbon source.
With regards to the co-culture, our next step would be to grow the co-culture for a longer period of time while also monitoring population dynamics. We would also want to make further genomic modifications to make the co-culture more robust. For example, by increasing amino acid export and import, we could hopefully increase the growth rate of the co-culture.
