Team:Missouri Rolla/Description
Team Missouri Rolla
Menu
PROJECT
INTERLAB
TEAM
SAFETY
HUMAN PRACTICES
COLLABORATIONS
ATTRIBUTIONS
BTREE
From Idea to Project
We were inspired to join the fight against Emerald Ash Borers (EABs) - an invasive beetle to the Ash tree - because their infestation is a local problem in Missouri. We were familiar with the use of Bacillus Thuringiensis (BT) toxins in agricultural crops, and wanted to explore using BT toxins to help create Ash trees resistant to EABs.
Though it is unknown exactly how long Emerald Ash Borers have been in the US, they were first identified in Michigan in 2002. Since then, they have been identified in 30 U.S. states and have caused millions of dollars in damage. EABs are a beetle which originated in Asia; it is expected that these beetles came to the U.S. on infested wood materials. Because Asian ash trees coevolved with the beetle, they are immune, but American ash trees do not have this same benefit. Adult beetles lay their larva in the bark, where they begin to develop. As they grow, they feed on the tree and eat through the phloem. After generations, this is enough to kill the trees as nutrients can no longer travel into the canopy.
Ash trees are incredibly common in the U.S. Following the devastation caused by the Dutch Elm disease, urban foresters settled on ash trees to replace the fallen elms, because they had and have a great balance of quality attributes: fast growing, no smelly fruits, large canopies, and beautiful fall colors. This makes the disease especially devastating to urban areas, where ash trees can make up upwards of even 90% of trees in a city. Current methods for prevention of EABs can be too expensive for prophylaxis for many cities and neighbors. Additionally, often the beatles have caused too much damage by the time they are noticed for treatment to be effective as the trees are too damaged.
Emerald Ash Borer spread as of August 2017. Photo credit; Emerald Ash Borer Information Network
After focusing on EABs, we looked into viable options for genetically engineering resistance into trees. This may not protect trees that are already planted, but when speaking with urban foresters, we spoke about the importance of tree diversity. If every city were to have more varied tree species, a problem EABs or Dutch Elm could not be as devastating again. By leaving ash trees on the market, this would encourage urban foresters to keep this diversity and prevent this same problem from happening again and again. Though multiple methods seemed appropriate, we began investigating Bacillus thuringiensis (Bt) and, based on talks with Monsanto scientists, this would be especially appropriate if used alongside an RNAi mechanism. Bt is a rod shaped, Gram-positive, spore-forming bacterium. Bt strains produces crystal proteins, called δ-endotoxins or pore-forming toxins. These crystal proteins often show insecticidal abilities to specific species within the insect orders Lepidoptera, Diptera, and Coleoptera.  After ingestion, Cry proteins are solubilized in the midgut and are processed into active toxins by gut proteases. Activated Cry toxins bind specific receptors on the midgut epithelial cell brush border membrane. While searching for a Cry toxin that may be effective against EABs, we came across Dr. Leah Bauer and her team at US Department of Forestry Service and the University of Michigan. They showed that Adult Emerald Ash Borers could be impacted by a Bt SDS-502, which expresses Cry8Da. After ingestion of a formulation of Bt SDS-502 sprayed on leaves, 89% mortality was seen within 96 hours of feeding. Cry8Da has been currently approved for the treatment of multiple scarab beetles, the Japanese beetle, and multiple wood boring members of the genus Agrilus.
Once we identified Cry8Da, we explored methods to improve the specificity of this toxin for Emerald Ash Borers. From this, we discovered iGEM Wageningen 2016’s project who worked to engineer a Cry3a protein to kill a common honey bee pest - the mite varroa destructor. To prove the protein could potentially combat the pest, the team conducted a fluorophore release assay. This assay was used to determine how well the mutated Cry3a protein could lyse open Brush Border Membrane Vesicles by measuring released fluorescence levels from the vesicles.
Using these ideas, our group hoped to engineer the Cry8Da toxin to increase specificity towards adult Emerald Ash Borers which would be expressed within leaf tissues of Arabadopsis thaliana to target feeding adult EABs. To begin, we had the Cry8Da gene synthesized (UniProtKB - D5H3I7) from IDT after being codon optimized to remove internal restriction sites. Along with this, we had the Arabidopsis rbcS-1A promoter synthesized to be our model of tissue specific gene expression. We planned to utilize primer directed mutagenesis of suspected binding sites of Cry8Da for our targeting. The Cry8Da protein consists of three domains. Research suggests domain II alpha loop structures are responsible for host receptor binding in coleopteran insects. Further research has indicated that alpha loop 2 in domain II is likely associated with toxicity in adult beetles. Our group planned to target this region for mutagenesis. In the end, we were unable to assemble these parts into a functioning transcriptional unit following Golden Braid (GB) Assembly standard using iGEM supplied GB.2.0 plasmid backbones. Due to these issues, we were unable to conduct our mutagenesis experiments on our Cry8Da gene to attempt to screen for appropriate toxins. Our group was thankful to receive killed EAB’s from Robert C. Venette at the USDA Northern Research Station, but, due to handling issues in transit, the samples obtained were not able to be utilized for assembly of our BBMVs for future testing.
After meeting with Bayer scientists and discussing our project, our team has developed directions to advance our project in the future. We have connections to receive new EABs for assembly of BBMV for our testing. We are looking into RNAi gene silencing as a method of targeting EABs. Along with this, we are exploring the possibility of isolating BT spores from EABs to search for possible natural EAB specific Cry toxins. Lastly, our group plans to continue efforts to improve our gene construction ability to further develop our current method of treatment.
See attributions for references used and experts spoken to regarding the above information.
Members of iGEM Missouri Rolla plant arabidopsis seedlings
Notebook
29 September 2018
Lucas Dyer and Lynell Cunningham
Start: 1:00 PM
PCR of Transformation of Cry 8DA
Purpose: To prepare the transformed cells for a gel to determine the success of the digestion/ligation.
Protocol:
The Knight: Colony PCR protocol was followed. The following PCR tubes were created.
Component
Tubes 1-5
Tubes 6-10
Tube 11
Taq 2X
9 μL
9 μL
9 μL
VR
0.5 μL
0.5 μL
0.5 μL
VF2
0.5 μL
0.5 μL
0.5 μL
MilliQ
10 μL
10 μL
9 μL
DNA
Colony from Transfomation Plate 2%
Colony from Transformation Plate 20%
1 μL L1
Thermocycler Conditions:
Temperature (°C)
Time (s)
95
300
95*
30
56*
30
68*
180
68
300
4
Hold
* Denotes that this time/temperature will be cycled 30 times.
Stop: 2:30 PM
Products:
The eleven tubes from above were saved and moved to the freezer upon completion of the PCR.
Next:
A gel will be run of the PCR tubes to determine the success of the Digestion/Ligation.
28 September 2018
Lucas Dyer
Start: 1:30 PM
Transformation of Ligation of Cry 8DA
Purpose: To create cells with ligated segment of Cry DNA.
Protocol:
The Intact Genomics High Efficiency Transformation Protocol was followed to produce 54 μl of transformed cells. The transformation solution consisted of 50μL of competent cells, and 4μL of ligated DNA, taken from L1.
Stop: 3:00 PM
Results:
The transformation successfully produced live cells, further tests will determine their fidelity.
Products:
Two plates were produced: one with 20% of the solution, and one with 2%.
Plate
Label
1
Transformation 20% LD 09/28/18
2
Transformation 2% LD 09/28/18
Next:
The cells will be used in a colony PCR to eventually run a gel and determine if the digestion/ligation were successful.
24 September 2018
Lucas Dyer
Start: 5:00 PM
Gel Electrophoresis of Colony PCR of Cry 8DA Transformation
Purpose: To determine the success of the transformation of the Cry 8DA sequence into competent cells and the subsequent colony PCR.
Protocol:
The Knight: Colony PCR protocol was followed for reagent volumes and thermocycler conditions. The gel was run according to standard gel procedure.  Gel Lanes
Lane
Component
Volume (μ)
1
Ladder
10
2
PCR 9/21 - 1
10
3
PCR 9/21 - 2
10
4
PCR 9/21 - 3
10
5
PCR 9/21 - 4
10
6
PCR 9/21 - 5
10
7
PCR 9/21 - 6
10
8
PCR 9/21 - 7
10
9
PCR 9/21 - 8
10
10
PCR 9/21 - 9
10
11
PCR 9/21 - 10
10
12
PCR 9/21 - 11
10
Stop: 6:15 PM
Results:
The gel returned large smears in each lane containing sample DNA rather than distinct bands. This confirmed the presence of DNA, but did not allow for any identification. The ladder did work indicating that the gel was run correctly and the problem instead lay in either the preparation of the samples, the PCR, or the transformation itself.
Products:
The following gel was produced.
Next:
Troubleshooting of the smear problem and repeat gels will be run using alternate methods in order to identify and isolate the desired sequence.
<a
14 September 2018
Lucas Dyer
Start: 4:00 PM
Cry Toxin Restriction Digest
Purpose: To prepare digested pieces of plasmid for use in future ligation.
Protocol:
The following digests were run in order to prepare the individual parts for the ligation of the final plasmid.
Digestion 1: Plasmid Backbone
Component
Volume (μl)
O Buffer
2.5
DNA
12
EcoR1
1.0
Pst1
1.0
MilliQ Water
3.5
Digestion 2: Cry Toxin Sequence
Component
Volume (μl)
Tango Buffer
2.5
DNA
6
Xba1
1.0
Pst1
1.0
MilliQ Water
9.5
Digestion 3: Promoter + RBS
Component
Volume (μl)
CutSmart Buffer
2.5
DNA
2.5
Spe1
1.0
EcoR1
1.0
MilliQ Water
10
Stop: 6:00 PM
Products:
Three PCR tubes with the following contents.
Tube Label
Contents
D1
Plasmid Backbone
D2
Cry Toxin Sequence
D3
Promoter + RBS
Next:
A gel will be run to determine the success of the digestions. Sequences will be isolated from the gels to be used for ligation.
27 August 2018
Ryan Baumann
Start: 12:00pm
Golden Gate Assembly of Cry8Da and Rbsc-1A into the Universal Acceptor Plasmid
Purpose: To assemble basic parts into UAP for future cloning into transcriptional units
Protocol:
Golden Gate assembly was conducted using NEB Golden Gate assembly Master Mix with BsmBI. Protocols were as follows.  27Jul18 GG1
Component
Volume
UAP (5/5 MP1)
2.5 μL (100ng)
Rbsc_1A (IDT Resuspended)
6.2 uL (62 ng)
NEB T4 DNA Ligase
1 μL
NEB 10x Ligase Buffer
2 μL
BsmBI
1 μL
Sterile Water
7.3 μL
Total
20 μL
27Jul18 GG2
Component
Volume
UAP (5/5 MP1)
1.8 μL (75ng)
Cry8Da (IDT Resuspended)
10 μL (100ng)
NEB T4 DNA Ligase
1 μL
NEB 10x Ligase Buffer
2 μL
BsmBI
1 μL
Sterile Water
7.3 μL
Total
20 μL
Thermocycler Conditions:  20 seconds 37°C, (3 minutes 37°C, 4 minutes 16°C) X26 , 5 minutes 50°C, 5 minutes 80°C, 5 minutes 16°C  Following Assembly, 2 μL of each tube was transformed following Intact Genomics ig 5-Alpha Chemically Competent Cells High Efficiency Transformation protocol. Cells were plated on Kanamycin LB Agar Plates.
Stop: 8:00 pm
Results:
Colonies were seen on all transformation plates.
Next:
Colony PCR will be conducted on plates to check for properly assembled basic parts.
  23 August 2018 Lynell Cunningham Interlab Transformation Day 2 Purpose: Innoculate plates Protocol: 5 mL of LB Chloramphenicol were added to falcon tubes. 2 cell cultures were then added to these tubes. Tubes were put in the incubating shaker at 220 rpm. Next: Follow the day 3 protocol for interlab. 22 August 2018 Lynell Cunningham Interlab Transformation Day 1 Purpose: To start interlab transformations, woot woot. Protocol: In wells 4B, 4D, 4F, 4H, 4J, 4L, 4N, 4P on kit plate 7, DNA was resuspended with 10uL of dH2O. 5uL of this DNA was used for the transformation according to the Intact Genomics Chemically Competent Cells pamphlet. Cells were grown on LB chloramphenical plates. Next: Inoculate plates in LB Chloramphenicol.
22 August 2017
Ryan Baumann, Jessica Brooks, Luke Dyer
Start: 1400
 
Colony PCR's of previous GG assembly transformation plates to screen for proper clones
Purpose: To check if our Cry toxin, Promoter + RBS, and terminator are properly assembled into the Universal Acceptor Plasmid for assembly of transcriptional units.
Protocol:
30 Colony PCR's were conducted using NEB's Taq 2x Master mix protocol with the following layout.
Component
Volume
Taq 2x Master Mix
10 μL
VR
0.5 μL
VF2
0.5 μL
Sterile Water
9 μL
Colony (1-30)
N/A
Total Volume
20 μl
PCR 1 through 20 contain colonies from the Cry_RFC106 GG Assembly Transformation Plates  PCR 21 through 30 contain colonies from the Rbsc_1A_B0030 GG Assembly Transformation Plate  PCR 31 was conducted from a Miniprep of BBa_K1618037 labelled 9-13-17 MP3 which will be utilized as our plant terminator with the following layout.
Component
Volume
NEB Taq 2X Master Mix
12.5 μL
VR
0.5 μL
VF2
0.5 μL
9-13-17 MP3
0.5 μL
Sterile Water
11 μL
20 μL
PCR 1-20 were placed in a Thermocycler with the following conditions 95°C for 30s, 30 Cycles (95°C for 30s, 54°C for 60s, and 68 °C for 120s), 68°C for 5min, and 4°C Hold.  PCR 21-31 were placed in a Thermocycler with the following conditions 95°C for 30s, 30 Cycles (95°C for 30s, 54°C for 60s, and 68 °C for 20s), 68°C for 5min, and 4°C Hold.  
Notes:
Plates were marked with 1-30 to determine where colonies came from for future miniprep if assembled properly.
Stop: 1700
 
Results:
TBD after gels.
Next:
Gel of PCR's 1-31 to determine if properly assembled. If properly assembled, colonies will be inoculated in liquid media for plasmid purification.
2 August 2018
Lynell, Luke, Jessica
Start: 9:40pm
 
Inoculation of transformed colonies
Purpose: Grow colonies in liquid media
Protocol:
Used LB broth with chlor
Stop:
 
Products:
Eight tubes:  RBCS-100-1 RBCS-100-2 RBCS-200-1 RBCS-200-2  From GGW  Cry-100-1 Cry-100-2 Cry-200-1 Cry-200-2  From 7/31 GG1
Notes:
Was not completed, due to flooding. Also, should have done colony PCR instead of liquid cultures.
Next:
PCR with VR and VF2, then Gel, then Miniprep, then NanoDrop
2 August 2018
Jessica
Start: 7:10pm
 
Checking Transformation Plates
Purpose: Find colonies
Protocol:
 
Notes:
CT Plate#1: No colonies CT Plate#2: No colonies CT Plate#3: A few colonies CT Plate#4: A moderate amount of colonies CT Plate#5: Tons of colonies CT Plate#6: Even more colonies
Stop:
 
Next:
 
1 August 2018
Jessica
Start: 2:00pm
 
Blue/White Chlor Plates
Purpose: Make Blue/White Chlor X-Gal Screening Plates
Protocol:
Mix the following into 700 mL of MilliQ water: 6 g tryptone 3 g yeast extract 3 g NaCl 9 g agar  Stir solution using a magnetic stir bar and a stir plate.  Autoclave the media.  Allow flask to cool on bench or in a preheated water bath until it is approximately 50-55°C (the flask should be warm to the touch but cool enough to grasp for several seconds without burning your hands). Do not allow the solution to cool enough to solidify in the flask.  Add 725ul of 1000x Chloramphenicol 50mg/ml and 2900ul of 250x X-gal 10mg/ml to reach goal concentrations of 50mg/L and 40mg/L  Pour plates
Notes:
The amounts for tryptone, yeast extract, NaCl, and agar were changed from lab manual to align with corrected values on multiple LTP workshop sheets 2200 ul of our 250x X-gal stock and 700ul of Dr.W's 40mg/ul were used (we ran out) The broth spend a great deal of time in a 54-55 *C hot water bath after autoclaving and before pouring
Stop:
 
Results:
25 Blue/White Chloramphenicol Screening Plates, in large bag. Each plate is labeled with 3 orange lines (chlor), 1 yellow line (LB) and one purple line (X-gal). The sleeve is labeled 8/1/2018 LB+Chlor+X-Gal
Next:
 
1 August 2018
Jessica
Start: 12:30am
 
Redo of Arabidopsis Plates
Purpose: Previous plates were liquid, agar will be added to remedy this
Protocol:
Same as on 31 July 2018, with the addition of 7.5 g of agar. Will produce 18 plates labeled with three green lines, four of which will be used for planting (found using x/500=10.5/700)
Stop:
 
Next:
 
1 August 2018
Jessica
Start: 12:00am
 
Transformation of Cry Sequence and B0030
Purpose: Develop plates of UAP+Cry and UAP+B0030
Protocol:
Followed Intact Genomics Pamphlet
Notes: Both shaking incubators were set at 30*C, so that was the temperature during the 1 hour incubation. GG1 plates are B0030, GG- and GGW are Cry Sequence and Control (may not be in that order)
Use chlor plates
Stop:
 
Products:
Four plates, labeled:  CT1= from the GG- PCR tube, 100 ul CT2= from the GG- PCR tube, 200 ul CT3= from the GG W PCR tube, 100 ul CT4= from the GG W PCR tube, 200 ul CT5= from the 7/31 GG1 tube, 100 ul CT6= from the 7/31 GG1 tube, 200 ul  Three tubes of cells and recovery medium: GG- from GG- PCR GG W from GG W PCR GG1 from 7/31 GG1 PCR
Next:
 
31 July 2018
Jessica
Start: 9:45pm
 
Golden Gate: B0030
Purpose: To re-suspend B0030 ordered from IDT and to place it inside of a UAP
Protocol:
BsmBI Protocol  5/5 MP1 (UAP) = 2.5 ul or ~100ng IDT B0030 = 10 ul or ~100ng T4 Ligase = 1 ul 10x T4 Ligase Buffer = 2 ul BsmBI = 1 ul Water (for total of 20 ul) = 3.5 ul  Thermocycler Protocol: 20 seconds 37*C (3 minutes 37*C, then 4 minutes 16*C) X 26 cycles 5 minutes 50*C 5 minutes 80*C 5 minutes 16*C
Notes:
B0030 was re-suspended from IDT protocol
Stop: 10:30pm
 
Results:
Re-suspension and Golden Gate Assembly
Products:
1 Tube with IDT Label of 17-Jul-2018 Rbcs-1A_B0030  1 Tube 7/31 GG 1
Next:
Transformation
31 July 2018
Jessica and Lynell
Start: 8:00pm
 
Arabidopsis Plates
Purpose: Prepare the growth medium for the development of Arabidopsis in the lab
Protocol:
 
Component
Amount
Murashige and Skoog salts
2.15 g
1% Sucrose
5 ml
.05% MES
.25 ml
MilliQ Water
500 ml
KOH
N/A (see below)
 
Notes:
Murashige and Skoog salts were in the fridge KOH amount is until pH is 5.7. Due to the lack of a pH meter, pH strips were used and pH was determined to be 5.5-6, so no KOH was added
Stop: 9:00
 
Results:
18 Arabidopsis Plates (18 MS Mediums)
Next:
Plant!
15 July 2018
Jessica Brooks
Start: 5:20pm
 
Gel of BBa_B0015
Purpose: Confirm quality of DNA from 24/7/17, Labeled as 7/24/17 B0015
Protocol:
From Notebook
Notes:
Used 3 uL of ethidium bromide Used 10 uL of ladder in well closest to side of gel Used 10 uL of sample in well closer to center of gel Run at 130V  **Lynell ran an identical gel previous to this
Stop: 7:00
 
Results:
Ryan said lynell's gel is good
Products:
One gel
Next:
 
5 July 2018
Jessica Brooks
Start: 12:00pm
 
PCR of BBa_B0015
Purpose: Conduct a PCR and gel of tube 24/7/17 B0015 to check DNA quality
Protocol:
New England BioLabs protocol for Taq 2x Master Mx  Used 1 ul of DNA  30 Cycles of: 95* for 20 seconds 52* for 30 seconds 68* for 30 seconds
Notes:
Used VR and VF2, which may produce an extra band at 500 and hazy ones below (shorter and dimmer than expected product)
Stop: In PCR at 12:40pm
 
Products:
One tube, labeled 7/5 PCR 1
Next:
Run gel  Should see bright band at 500 and dim hazy bands under the 129 pb B0015
2 July 2018
Jessica, Lynell, Luke
Start: 12:00
 
InterLab
Purpose: Inoculate two colonies from previous transformation
Protocol:
Followed standard protocol
Stop: 3:00pm
 
Results:
16 tubes, labeled Device #-Colony Number
Next:
 
1July 2018
Lynell Cunningham, Jessica Brooks, and Lucas Dyer
Start: 11:00am
 
Interlab Cell Measurement Transformation
Purpose: To transform kit plate Test Device DNA into competent cells for further use in the interlab.
Protocol:
Eight wells of kit plate DNA were transformed into eight samples of competent cells according to the Intact Genomics High Efficiency Protocol. Each sample was then plated at 5% and 20% onto chloramphenicol plates and grown overnight.
Stop: 1:30pm
 
Results:
The cells grew overnight on plates.
Products:
A total of sixteen plates were created
Plate
Device
Part Number
Label
1
Negative Control
BBa_R0040
Interlab Negative Control 5%
2
Negative Control
BBa_R0040
Interlab Negative Control 20%
3
Positive Control
BBa_I20270
Interlab Positive Control 5%
4
Positive Control
BBa_I20270
Interlab Positive Control 20%
5
Test Device 1
BBa_J364000
Interlab Test Device 1: 5%
6
Test Device 1
BBa_J364000
Interlab Test Device 1: 20%
7
Test Device 2
BBa_J364001
Interlab Test Device 2: 5%
8
Test Device 2
BBa_J364001
Interlab Test Device 2: 20%
9
Test Device 3
BBa_J364002
Interlab Test Device 3: 5%
10
Test Device 3
BBa_J364002
Interlab Test Device 3: 20%
11
Test Device 4
BBa_J364003
Interlab Test Device 4: 5%
12
Test Device 4
BBa_J364003
Interlab Test Device 4: 20%
13
Test Device 5
BBa_J364004
Interlab Test Device 5: 5%
14
Test Device 5
BBa_J364004
Interlab Test Device 5: 20%
15
Test Device 6
BBa_J364005
Interlab Test Device 6: 5%
16
Test Device 6
BBa_J364005
Interlab Test Device 6: 20%
 
Next:
Cultures from plates will be inncoulated to LB+Chloramephenicol broth and grown over night.
28 June 2018
Jessica, Lynell, Luke
Start: 11:00am
 
InterLab
Purpose: Complete Calibration for Plate Reader/CFU
Protocol:
Completed Calibrations
Stop: 4:00pm
 
Next:
 
5 May 2018
Lynell Cunningham, Ryan Baumann, Lucas Dyer
Start: 1:30pm
 
Minipreps of phytobrick plasmids
Purpose: To obtain phytobrick plasmids
Protocol:
Three minipreps were performed according to the kitless miniprep procedure. Products were run through the nanodrop to determine concentration.
Stop: 4:30 pm
 
Results:
 
Sample
ng/μl
260/280
260/230
5/5 MP1
39.45
1.84
2.12
5/5 MP2
34.38
1.92
2.01
5/5 MP3
37.94
1.76
1.45
 
Products:
 
Label
Source
Description
5/5 MP1
5/2 BC3
UAP/BBa_P10500 in pSB1C3
5/5 MP2
5/2 BC4
Alpha 1/BBa_P10501 in pSB1K3
5/5 MP3
5/2 BC5
Alpha 2/BBa_P10503 in pSB1K3
 
Next:
Digest each miniprep with EcoRI and PstI run products on gel to verify identity
3 May 2018
Jessica Brooks, Lynell Cunningham, and Kent Gorday
Start: 16:15
 
Minipreps of amilCP and phytobrick plasmids
Purpose: To obtain amilCP in pSB1C3, along with the five (UAP + alpha and omega) phytobrick plasmids
Protocol:
Seven minipreps were performed according to the kitless miniprep procedure. Products were nanodropped to determine concentration.
Stop: 19:00
 
Results:
 
Sample
ng/μL
260/280
260/230
5/3 MP1
172.31
1.87
1.88
5/3 MP2
123.26
1.84
2.25
5/3 MP3
50.63
1.78
2.04
5/3 MP4
41.19
1.68
1.54
5/3 MP5
30.47
1.72
1.71
5/3 MP6
8.30
1.26
0.99
5/3 MP7
1.65
0.87
0.78
 
Products:
 
Label
Source
Description
5/3 MP1
5/2 BC1
amilCP in pSB1C3 (higher concentration?)
5/3 MP2
5/2 BC2
amilCP in pSB1C3 (cleaner prep?)
5/3 MP3
5/2 BC3
UAP/BBa_P10500 in pSB1C3
5/3 MP4
5/2 BC4
Alpha 1/BBa_P10501 in pSB1K3
5/3 MP5
5/2 BC5
Alpha 2/BBa_P10503 in pSB1K3
5/3 MP6
5/2 BC6
Omega 1/BBa_P10505 in pSB1K3
5/3 MP7
5/2 BC7
Omega 2/BBa_P10507 in pSB1K3
 
Next:
Digest each miniprep with EcoRI and PstI run products on gel to verify identity
2 May 2018
Ryan Baumann and Lucas Dyer
Start: 4:00
 
Broth cultures of cloning vectors
Purpose: To obtain good preps of several cloning vectors
Protocol:
One chloramphenicol broth culture and four kanamycin broth cultures were inoculated
Stop: 4:30
 
Products:
The following broth cultures were created
Label
Source
Description
5/2 UAP BC3
5/1 Transformation
UAP/BBa_P10500 in pSB1C3
5/2 Alpha 1 BC4
5/1 Transformation
Alpha 1/BBa_P10501 in pSB1K3
5/2 Alpha 2 BC5
5/1 Transformation
Alpha 2/BBa_P10503 in pSB1K3
5/2 Omega 1 BC6
5/1 Transformation
Omega 1/BBa_P10505 in pSB1K3
5/2 Omega 2 BC7
5/1 Transformation
Omega 2/BBa_P10507 in pSB1K3
 
Next:
Miniprep all broth cultures
2 May 2018
Kent Gorday
Start: 15:00
 
Broth cultures of amilCP in pSB1C3
Purpose: To obtain good preps of amilCP in pSB1C3
Protocol:
Two LB + chloramphenicol broth cultures of amilCP in pSB1C3 were inoculated
Stop: 15:20
 
Products:
 
Label
Source
Description
5/2 BC1
5/1 amilCP transformation
amilCP in pSB1C3
5/2 BC2
5/1 amilCP transformation
amilCP in pSB1C3
 
Next:
Miniprep amilCP in pSB1C3 from broth cultures
1 May 2018
Luke Dyer and Ryan Baumann
Start: 4:30
 
Transformations of 2018 Cloning vectors
Purpose: Transformation of cloning vectors and parts needed for summer research.
Protocol:
Transformation of Golden braid destination vectors, UAP, and AmilCP were conducted following Intact genomics 5 Alpha electro competent cells protocol.  UAP and AmilCP were plated on Chloramphenicol plates while GB Destination Vectors were plated on Kanamycin Plates
Stop: 7:30
 
Products:
 
Part Transformed
Alpha 1/BBa_P10501
Alpha 2/BBa_P10503
Omega 1/BBa_P10505
Omega 2/BBa_P10507
UAP/BBa_P10500
AmilCP/BBa_K592009
 
Next:
Grow Overnight and inoculate cultures in liquid media.
2-5-18
Ryan Baumann
Start: 10:00AM
 
DNA Purification of AmilCP and BBa_J04500
Purpose: To purify the promoter and chromoproterin for future miniprojects.
Protocol:
Plasmid minipreps were conducted following Missouri S&T iGEM's Lab manual 2.5 kitless miniprep protocol. DNA was resuspended in 40 μL TE buffer and concentration was tested with a Thermo Scientific Nanodrop 1000.  Nanodrop Results:
Name
DNA Conc.
260/280
1-22-18 MP1 (amilCP)
121.73
1.89
1-22-18 MP2 (amilCP)
158.92
1.89
2-1-18 MP1 (BBa_J04500)
119.82
1.87
2-1-18 MP2 (BBa_J04500)
127.71
1.90
 
Stop: 10:15 AM
 
Products:
 
Label
Source
Description
2-1-18 MP1
BBa_j04500 from 2014 kit plate 3
BBa_J04500 inducible promoter
2-1-18 MP2
BBa_j04500 from 2014 kit plate 3
BBa_J04500 inducible promoter
1-22-18 MP1
amilCP Chromoprotein from 2016 kit plate
amilCP blue Chromoprotein in pSB1C3
1-22-18 MP2
amilCP Chromoprotein from 2016 kit plate
amilCP blue Chromoprotein in pSB1C3
 
Next:
 
