<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Missouri_Rolla_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Missouri Rolla/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><A class="navbar-brand js-scroll-trigger" href="https://2018.igem.org/Team:Missouri_Rolla">Team Missouri Rolla</A><BUTTON class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
                Menu
                </BUTTON><DIV class="collapse navbar-collapse" id="navbarResponsive"><UL class="navbar-nav text-uppercase ml-auto"><LI class="nav-item"><A class="nav-link js-scroll-trigger" href="https://2018.igem.org/Team:Missouri_Rolla/Description">PROJECT</A></LI><LI class="nav-item"><A class="nav-link js-scroll-trigger" href="https://2018.igem.org/Team:Missouri_Rolla/InterLab">INTERLAB</A></LI><LI class="nav-item"><A class="nav-link js-scroll-trigger" href="https://2018.igem.org/Team:Missouri_Rolla/Team">TEAM</A></LI><LI class="nav-item"><A class="nav-link js-scroll-trigger" href="https://2018.igem.org/Team:Missouri_Rolla/Safety">SAFETY</A></LI><LI class="nav-item"><A class="nav-link js-scroll-trigger" href="https://2018.igem.org/Team:Missouri_Rolla/Human_Practices">HUMAN PRACTICES</A></LI><LI class="nav-item"><A class="nav-link js-scroll-trigger" href="https://2018.igem.org/Team:Missouri_Rolla/Collaborations">COLLABORATIONS</A></LI><LI class="nav-item"><A class="nav-link js-scroll-trigger" href="https://2018.igem.org/Team:Missouri_Rolla/Attributions">ATTRIBUTIONS</A></LI></UL></DIV></DIV><HEADER class="masthead"><DIV class="container"><DIV class="intro-text"><DIV class="intro-heading text-uppercase">BTREE</DIV></DIV></DIV></HEADER><SECTION class="bg-light" id="Description" style="padding-bottom:25px; padding-left:35px;"><DIV class="col-lg-12 text-center"><H2 class="section-heading text-uppercase">From Idea to Project</H2></DIV><P style="padding-bottom:25px; padding-left:35px; padding-right:35px; border-left:6px solid green;"> We were inspired to join the fight against Emerald Ash Borers (EABs) - an invasive beetle to the Ash tree - because their infestation is a local problem in Missouri. We were familiar with the use of Bacillus Thuringiensis (BT) toxins in agricultural crops, and wanted to explore using BT toxins to help create Ash trees resistant to EABs. </P><P style="padding-bottom:25px; padding-left:35px; padding-right:35px; border-left:6px solid green; background-color: #f8f9fa;"> 
Though it is unknown exactly how long Emerald Ash Borers have been in the US, they were first identified in Michigan in 2002. Since then, they have been identified in 30 U.S. states and have caused millions of dollars in damage. EABs are a beetle which originated in Asia; it is expected that these beetles came to the U.S. on infested wood materials. Because Asian ash trees coevolved with the beetle, they are immune, but American ash trees do not have this same benefit. Adult beetles lay their larva in the bark, where they begin to develop. As they grow, they feed on the tree and eat through the phloem. After generations, this is enough to kill the trees as nutrients can no longer travel into the canopy.
</P><P style="padding-bottom:25px; padding-left:35px; padding-right:35px; border-left:6px solid green; background-color: #f8f9fa;"> 
Ash trees are incredibly common in the U.S. Following the devastation caused by the Dutch Elm disease, urban foresters settled on ash trees to replace the fallen elms, because they had and have a great balance of quality attributes: fast growing, no smelly fruits, large canopies, and beautiful fall colors. This makes the disease especially devastating to urban areas, where ash trees can make up upwards of even 90% of trees in a city. Current methods for prevention of EABs can be too expensive for prophylaxis for many cities and neighbors. Additionally, often the beatles have caused too much damage by the time they are noticed for treatment to be effective as the trees are too damaged.</P><DIV class="col-lg-12 text-center"><H3 class="section-subheading text-muted">Emerald Ash Borer spread as of August 2017. Photo credit; Emerald Ash Borer Information Network</H3></DIV><P style="padding-bottom:25px; padding-left:35px; padding-right:35px; border-left:6px solid green; background-color: #f8f9fa;">After focusing on EABs, we looked into viable options for genetically engineering resistance into trees. This may not protect trees that are already planted, but when speaking with urban foresters, we spoke about the importance of tree diversity. If every city were to have more varied tree species, a problem EABs or Dutch Elm could not be as devastating again. By leaving ash trees on the market, this would encourage urban foresters to keep this diversity and prevent this same problem from happening again and again. Though multiple methods seemed appropriate, we began investigating Bacillus thuringiensis (Bt) and, based on talks with Monsanto scientists, this would be especially appropriate if used alongside an RNAi mechanism. Bt is a rod shaped, Gram-positive, spore-forming bacterium. Bt strains produces crystal proteins, called δ-endotoxins or pore-forming toxins. These crystal proteins often show insecticidal abilities to specific species within the insect orders Lepidoptera, Diptera, and Coleoptera.  After ingestion, Cry proteins are solubilized in the midgut and are processed into active toxins by gut proteases. Activated Cry toxins bind specific receptors on the midgut epithelial cell brush border membrane. While searching for a Cry toxin that may be effective against EABs, we came across Dr. Leah Bauer and her team at US Department of Forestry Service and the University of Michigan. They showed that Adult Emerald Ash Borers could be impacted by a Bt SDS-502, which expresses Cry8Da. After ingestion of a formulation of Bt SDS-502 sprayed on leaves, 89% mortality was seen within 96 hours of feeding. Cry8Da has been currently approved for the treatment of multiple scarab beetles, the Japanese beetle, and multiple wood boring members of the genus Agrilus. 
</P><P style="padding-bottom:25px; padding-left:35px; padding-right:35px; border-left:6px solid green; background-color: #f8f9fa;">Once we identified Cry8Da, we explored methods to improve the specificity of this toxin for Emerald Ash Borers. From this, we discovered iGEM Wageningen 2016’s project who worked to engineer a Cry3a protein to kill a common honey bee pest - the mite varroa destructor. To prove the protein could potentially combat the pest, the team conducted a fluorophore release assay. This assay was used to determine how well the mutated Cry3a protein could lyse open Brush Border Membrane Vesicles by measuring released fluorescence levels from the vesicles.
</P><P style="padding-bottom:25px; padding-left:35px; padding-right:35px; border-left:6px solid green; background-color: #f8f9fa;">
Using these ideas, our group hoped to engineer the Cry8Da toxin to increase specificity towards adult Emerald Ash Borers which would be expressed within leaf tissues of Arabadopsis thaliana to target feeding adult EABs. To begin, we had the Cry8Da gene synthesized (UniProtKB - D5H3I7) from IDT after being codon optimized to remove internal restriction sites. Along with this, we had the Arabidopsis rbcS-1A promoter synthesized to be our model of tissue specific gene expression. We planned to utilize primer directed mutagenesis of suspected binding sites of Cry8Da for our targeting. The Cry8Da protein consists of three domains. Research suggests domain II alpha loop structures are responsible for host receptor binding in coleopteran insects. Further research has indicated that alpha loop 2 in domain II is likely associated with toxicity in adult beetles. Our group planned to target this region for mutagenesis. In the end, we were unable to assemble these parts into a functioning transcriptional unit following Golden Braid (GB) Assembly standard using iGEM supplied GB.2.0 plasmid backbones. Due to these issues, we were unable to conduct our mutagenesis experiments on our Cry8Da gene to attempt to screen for appropriate toxins. Our group was thankful to receive killed EAB’s from Robert C. Venette at the USDA Northern Research Station, but, due to handling issues in transit, the samples obtained were not able to be utilized for assembly of our BBMVs for future testing. 
</P><P style="padding-bottom:25px; padding-left:35px; padding-right:35px; border-left:6px solid green; background-color: #f8f9fa;">
After meeting with Bayer scientists and discussing our project, our team has developed directions to advance our project in the future. We have connections to receive new EABs for assembly of BBMV for our testing. We are looking into RNAi gene silencing as a method of targeting EABs. Along with this, we are exploring the possibility of isolating BT spores from EABs to search for possible natural EAB specific Cry toxins. Lastly, our group plans to continue efforts to improve our gene construction ability to further develop our current method of treatment.
</P><P style="padding-bottom:25px; padding-left:35px; padding-right:35px; border-left:6px solid green; background-color: #f8f9fa;">
See attributions for references used and experts spoken to regarding the above information.
</P><DIV class="col-lg-12 text-center"><H3 class="section-subheading text-muted">Members of iGEM Missouri Rolla plant arabidopsis seedlings</H3></DIV></SECTION><SECTION id="Notebook" style="padding-bottom:25px; padding-left:35px;"><DIV class="col-lg-12 text-center"><H2 class="section-heading text-uppercase">Notebook</H2></DIV><DIV class="notebook"><H2>29 September 2018</H2><H4 class="h4left"><B>Lucas Dyer and Lynell Cunningham</B></H4><H4 class="h4right"><B>Start: 1:00 PM</B></H4><H4><B>PCR of Transformation of Cry 8DA</B></H4><B>Purpose:</B> To prepare the transformed cells for a gel to determine the success of the digestion/ligation.<H4><B>Protocol:</B></H4>The Knight: Colony PCR protocol was followed. The following PCR tubes were created. 

<TABLE><TBODY><TR><TD>Component</TD><TD>Tubes 1-5</TD><TD>Tubes 6-10</TD><TD>Tube 11</TD></TR><TR><TD>Taq 2X</TD><TD>9 μL</TD><TD>9 μL</TD><TD>9 μL</TD></TR><TR><TD>VR</TD><TD>0.5 μL</TD><TD>0.5 μL</TD><TD>0.5 μL</TD></TR><TR><TD>VF2</TD><TD>0.5 μL</TD><TD>0.5 μL</TD><TD>0.5 μL</TD></TR><TR><TD>MilliQ</TD><TD>10 μL</TD><TD>10 μL</TD><TD>9 μL</TD></TR><TR><TD>DNA</TD><TD>Colony from Transfomation Plate 2%</TD><TD>Colony from Transformation Plate 20%</TD><TD>1 μL L1</TD></TR></TBODY></TABLE>

Thermocycler Conditions:
<TABLE><TBODY><TR><TD>Temperature (°C)</TD><TD>Time (s)</TD></TR><TR><TD>95</TD><TD>300</TD></TR><TR><TD>95*</TD><TD>30</TD></TR><TR><TD>56*</TD><TD>30</TD></TR><TR><TD>68*</TD><TD>180</TD></TR><TR><TD>68</TD><TD>300</TD></TR><TR><TD>4</TD><TD>Hold</TD></TR></TBODY></TABLE>

* Denotes that this time/temperature will be cycled 30 times.<H4><B>Stop: 2:30 PM</B></H4><H4><B>Products:</B></H4>The eleven tubes from above were saved and moved to the freezer upon completion of the PCR.<H4><B>Next:</B></H4>A gel will be run of the PCR tubes to determine the success of the Digestion/Ligation.</DIV><DIV class="notebook"><H2>28 September 2018</H2><H4 class="h4left"><B>Lucas Dyer</B></H4><H4 class="h4right"><B>Start: 1:30 PM</B></H4><H4><B>Transformation of Ligation of Cry 8DA</B></H4><B>Purpose:</B> To create cells with ligated segment of Cry DNA.<H4><B>Protocol:</B></H4>The Intact Genomics High Efficiency Transformation Protocol was followed to produce 54 μl of transformed cells. The transformation solution consisted of 50μL of competent cells, and 4μL of ligated DNA, taken from L1.<H4><B>Stop: 3:00 PM</B></H4><H4><B>Results:</B></H4>The transformation successfully produced live cells, further tests will determine their fidelity. <H4><B>Products:</B></H4>Two plates were produced: one with 20% of the solution, and one with 2%.

<TABLE><TBODY><TR><TD>Plate</TD><TD>Label</TD></TR><TR><TD>1</TD><TD>Transformation 20% LD 09/28/18</TD></TR><TR><TD>2</TD><TD>Transformation 2% LD 09/28/18</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>The cells will be used in a colony PCR to eventually run a gel and determine if the digestion/ligation were successful.</DIV><DIV class="notebook"><H2>24 September 2018</H2><H4 class="h4left"><B>Lucas Dyer</B></H4><H4 class="h4right"><B>Start: 5:00 PM</B></H4><H4><B>Gel Electrophoresis of Colony PCR of Cry 8DA Transformation</B></H4><B>Purpose:</B> To determine the success of the transformation of the Cry 8DA sequence into competent cells and the subsequent colony PCR.<H4><B>Protocol:</B></H4>The Knight: Colony PCR protocol was followed for reagent volumes and thermocycler conditions. The gel was run according to standard gel procedure.

Gel Lanes
<TABLE><TBODY><TR><TD>Lane</TD><TD>Component</TD><TD>Volume (μ)</TD></TR><TR><TD>1</TD><TD>Ladder</TD><TD>10</TD></TR><TR><TD>2</TD><TD>PCR 9/21 - 1</TD><TD>10</TD></TR><TR><TD>3</TD><TD>PCR 9/21 - 2</TD><TD>10</TD></TR><TR><TD>4</TD><TD>PCR 9/21 - 3</TD><TD>10</TD></TR><TR><TD>5</TD><TD>PCR 9/21 - 4</TD><TD>10</TD></TR><TR><TD>6</TD><TD>PCR 9/21 - 5</TD><TD>10</TD></TR><TR><TD>7</TD><TD>PCR 9/21 - 6</TD><TD>10</TD></TR><TR><TD>8</TD><TD>PCR 9/21 - 7</TD><TD>10</TD></TR><TR><TD>9</TD><TD>PCR 9/21 - 8</TD><TD>10</TD></TR><TR><TD>10</TD><TD>PCR 9/21 - 9</TD><TD>10</TD></TR><TR><TD>11</TD><TD>PCR 9/21 - 10</TD><TD>10</TD></TR><TR><TD>12</TD><TD>PCR 9/21 - 11</TD><TD>10</TD></TR></TBODY></TABLE><H4><B>Stop: 6:15 PM</B></H4><H4><B>Results:</B></H4>The gel returned large smears in each lane containing sample DNA rather than distinct bands. This confirmed the presence of DNA, but did not allow for any identification. The ladder did work indicating that the gel was run correctly and the problem instead lay in either the preparation of the samples, the PCR, or the transformation itself.<H4><B>Products:</B></H4>The following gel was produced. 
<H4><B>Next:</B></H4>Troubleshooting of the smear problem and repeat gels will be run using alternate methods in order to identify and isolate the desired sequence.</DIV>&lt;a 

<DIV class="notebook"><H2>14 September 2018</H2><H4 class="h4left"><B>Lucas Dyer</B></H4><H4 class="h4right"><B>Start: 4:00 PM</B></H4><H4><B>Cry Toxin Restriction Digest</B></H4><B>Purpose:</B> To prepare digested pieces of plasmid for use in future ligation.<H4><B>Protocol:</B></H4>The following digests were run in order to prepare the individual parts for the ligation of the final plasmid.

<TABLE><TBODY><TR><TD>Digestion 1: Plasmid Backbone</TD></TR><TR><TD>Component</TD><TD>Volume (μl)</TD></TR><TR><TD>O Buffer</TD><TD>2.5</TD></TR><TR><TD>DNA</TD><TD>12</TD></TR><TR><TD>EcoR1</TD><TD>1.0</TD></TR><TR><TD>Pst1</TD><TD>1.0</TD></TR><TR><TD>MilliQ Water</TD><TD>3.5</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Digestion 2: Cry Toxin Sequence</TD></TR><TR><TD>Component</TD><TD>Volume (μl)</TD></TR><TR><TD>Tango Buffer</TD><TD>2.5</TD></TR><TR><TD>DNA</TD><TD>6</TD></TR><TR><TD>Xba1</TD><TD>1.0</TD></TR><TR><TD>Pst1</TD><TD>1.0</TD></TR><TR><TD>MilliQ Water</TD><TD>9.5</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Digestion 3: Promoter + RBS</TD></TR><TR><TD>Component</TD><TD>Volume (μl)</TD></TR><TR><TD>CutSmart Buffer</TD><TD>2.5</TD></TR><TR><TD>DNA</TD><TD>2.5</TD></TR><TR><TD>Spe1</TD><TD>1.0</TD></TR><TR><TD>EcoR1</TD><TD>1.0</TD></TR><TR><TD>MilliQ Water</TD><TD>10</TD></TR></TBODY></TABLE><H4><B>Stop: 6:00 PM</B></H4><H4><B>Products:</B></H4>Three PCR tubes with the following contents.

<TABLE><TBODY><TR><TD>Tube Label</TD><TD>Contents</TD></TR><TR><TD>D1</TD><TD>Plasmid Backbone</TD></TR><TR><TD>D2</TD><TD>Cry Toxin Sequence</TD></TR><TR><TD>D3</TD><TD>Promoter + RBS</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>A gel will be run to determine the success of the digestions. Sequences will be isolated from the gels to be used for ligation.</DIV><DIV class="notebook"><H2>27 August 2018</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 12:00pm</B></H4><H4><B>Golden Gate Assembly of Cry8Da and Rbsc-1A into the Universal Acceptor Plasmid </B></H4><B>Purpose:</B> To assemble basic parts into UAP for future cloning into transcriptional units
<H4><B>Protocol:</B></H4>Golden Gate assembly was conducted using NEB Golden Gate assembly Master Mix with BsmBI. Protocols were as follows. 
27Jul18 GG1
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>UAP (5/5 MP1)</TD><TD>2.5 μL (100ng)</TD></TR><TR><TD>Rbsc_1A (IDT Resuspended)</TD><TD>6.2 uL (62 ng)</TD></TR><TR><TD>NEB T4 DNA Ligase</TD><TD>1 μL</TD></TR><TR><TD>NEB 10x Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>Sterile Water</TD><TD>7.3 μL</TD></TR><TR><TD>Total</TD><TD>20 μL</TD></TR></TBODY></TABLE>

27Jul18 GG2
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>UAP (5/5 MP1)</TD><TD>1.8 μL (75ng)</TD></TR><TR><TD>Cry8Da (IDT Resuspended)</TD><TD>10 μL (100ng)</TD></TR><TR><TD>NEB T4 DNA Ligase</TD><TD>1 μL</TD></TR><TR><TD>NEB 10x Ligase Buffer</TD><TD>2 μL</TD></TR><TR><TD>BsmBI</TD><TD>1 μL</TD></TR><TR><TD>Sterile Water</TD><TD>7.3 μL</TD></TR><TR><TD>Total</TD><TD>20 μL</TD></TR></TBODY></TABLE>

Thermocycler Conditions: 
20 seconds 37°C,
(3 minutes 37°C, 4 minutes 16°C) X26 ,
5 minutes 50°C, 5 minutes 80°C,
5 minutes 16°C

Following Assembly, 2 μL of each tube was transformed following Intact Genomics ig 5-Alpha Chemically Competent Cells High Efficiency Transformation protocol. Cells were plated on Kanamycin LB Agar Plates. <H4><B>Stop: 8:00 pm</B></H4><H4><B>Results:</B></H4>Colonies were seen on all transformation plates. <H4><B>Next:</B></H4>Colony PCR will be conducted on plates to check for properly assembled basic parts.</DIV><DIV class="notebook"> <STRONG>23 August 2018 </STRONG><STRONG>Lynell Cunningham</STRONG><STRONG>Interlab Transformation Day 2</STRONG><STRONG>Purpose: </STRONG>Innoculate plates

<STRONG>Protocol: </STRONG>

5 mL of LB Chloramphenicol were added to falcon tubes. 2 cell cultures were then added to these tubes. Tubes were put in the incubating shaker at 220 rpm.

<STRONG>Next: </STRONG>Follow the day 3 protocol for interlab.

<STRONG>22 August 2018</STRONG><STRONG>Lynell Cunningham</STRONG><STRONG>Interlab Transformation Day 1</STRONG><STRONG>Purpose:</STRONG> To start interlab transformations, woot woot.

<STRONG>Protocol: </STRONG>

In wells 4B, 4D, 4F, 4H, 4J, 4L, 4N, 4P on kit plate 7, DNA was resuspended with 10uL of dH2O. 5uL of this DNA was used for the transformation according to the Intact Genomics Chemically Competent Cells pamphlet. Cells were grown on LB chloramphenical plates.

<B style="color: #000000"> Next: </B>Inoculate<SPAN style="color: #000000"> plates in LB Chloramphenicol.</SPAN><H2>22 August 2017</H2><H4 class="h4left"><B>Ryan Baumann, Jessica Brooks, Luke Dyer</B></H4><H4 class="h4right"><B>Start: 1400</B></H4> <H4><B>Colony PCR's of previous GG assembly transformation plates to screen for proper clones</B></H4><B>Purpose:</B> To check if our Cry toxin, Promoter + RBS, and terminator are properly assembled into the Universal Acceptor Plasmid for assembly of transcriptional units.
<H4><B>Protocol:</B></H4>
30 Colony PCR's were conducted using NEB's Taq 2x Master mix protocol with the following layout.
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Taq 2x Master Mix</TD><TD>10 μL</TD></TR><TR><TD>VR</TD><TD>0.5 μL</TD></TR><TR><TD>VF2</TD><TD>0.5 μL</TD></TR><TR><TD>Sterile Water</TD><TD>9 μL</TD></TR><TR><TD>Colony (1-30)</TD><TD>N/A</TD></TR><TR><TD>Total Volume</TD><TD>20 μl</TD></TR></TBODY></TABLE>
PCR 1 through 20 contain colonies from the Cry_RFC106 GG Assembly Transformation Plates

PCR 21 through 30 contain colonies from the Rbsc_1A_B0030 GG Assembly Transformation Plate

PCR 31 was conducted from a Miniprep of BBa_K1618037 labelled 9-13-17 MP3 which will be utilized as our plant terminator with the following layout.
<TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>NEB Taq 2X Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>VR</TD><TD>0.5 μL</TD></TR><TR><TD>VF2</TD><TD>0.5 μL</TD></TR><TR><TD>9-13-17 MP3</TD><TD>0.5 μL</TD></TR><TR><TD>Sterile Water</TD><TD>11 μL</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE>
PCR 1-20 were placed in a Thermocycler with the following conditions
95°C for 30s, 30 Cycles (95°C for 30s, 54°C for 60s, and 68 °C for 120s), 68°C for 5min, and 4°C Hold.

PCR 21-31 were placed in a Thermocycler with the following conditions
95°C for 30s, 30 Cycles (95°C for 30s, 54°C for 60s, and 68 °C for 20s), 68°C for 5min, and 4°C Hold.

 <H4><B>Notes:</B></H4>
Plates were marked with 1-30 to determine where colonies came from for future miniprep if assembled properly.
<H4><B>Stop: 1700</B></H4> <H4><B>Results:</B></H4>
TBD after gels.
<H4><B>Next:</B></H4>
Gel of PCR's 1-31 to determine if properly assembled. If properly assembled, colonies will be inoculated in liquid media for plasmid purification.

</DIV><DIV class="notebook"><H2>2 August 2018</H2><H4 class="h4left"><B>Lynell, Luke, Jessica</B></H4><H4 class="h4right"><B>Start: 9:40pm</B></H4> <H4><B>Inoculation of transformed colonies</B></H4><B>Purpose:</B> Grow colonies in liquid media
<H4><B>Protocol:</B></H4>
Used LB broth with chlor
<H4><B>Stop: </B></H4> <H4><B>Products:</B></H4>
Eight tubes:

RBCS-100-1
RBCS-100-2
RBCS-200-1
RBCS-200-2

From GGW

Cry-100-1
Cry-100-2
Cry-200-1
Cry-200-2

From 7/31 GG1
<H4><B>Notes:</B></H4>
Was not completed, due to flooding. Also, should have done colony PCR instead of liquid cultures.
<H4><B>Next:</B></H4>
PCR with VR and VF2, then Gel, then Miniprep, then NanoDrop

</DIV><DIV class="notebook"><H2>2 August 2018</H2><H4 class="h4left"><B>Jessica</B></H4><H4 class="h4right"><B>Start: 7:10pm</B></H4> <H4><B>Checking Transformation Plates</B></H4><B>Purpose:</B> Find colonies
<H4><B>Protocol:</B></H4> <H4><B>Notes:</B></H4>
CT Plate#1: No colonies
CT Plate#2: No colonies
CT Plate#3: A few colonies
CT Plate#4: A moderate amount of colonies
CT Plate#5: Tons of colonies
CT Plate#6: Even more colonies
<H4><B>Stop: </B></H4> <H4><B>Next:</B></H4> </DIV><DIV class="notebook"><H2>1 August 2018</H2><H4 class="h4left"><B>Jessica</B></H4><H4 class="h4right"><B>Start: 2:00pm</B></H4> <H4><B>Blue/White Chlor Plates</B></H4><B>Purpose:</B> Make Blue/White Chlor X-Gal Screening Plates
<H4><B>Protocol:</B></H4>
Mix the following into 700 mL of MilliQ water:
6 g tryptone
3 g yeast extract
3 g NaCl
9 g agar

Stir solution using a magnetic stir bar and a stir plate.

Autoclave the media.

Allow flask to cool on bench or in a preheated water bath until it is approximately 50-55°C (the flask should be warm to the touch but cool enough to grasp for several seconds without burning your hands). Do not allow the solution to cool enough to solidify in the flask.

Add 725ul of 1000x Chloramphenicol 50mg/ml and 2900ul of 250x X-gal 10mg/ml to reach goal concentrations of 50mg/L and 40mg/L

Pour plates
<H4><B>Notes:</B></H4>
The amounts for tryptone, yeast extract, NaCl, and agar were changed from lab manual to align with corrected values on multiple LTP workshop sheets
2200 ul of our 250x X-gal stock and 700ul of Dr.W's 40mg/ul were used (we ran out)
The broth spend a great deal of time in a 54-55 *C hot water bath after autoclaving and before pouring
<H4><B>Stop: </B></H4> <H4><B>Results:</B></H4>
25 Blue/White Chloramphenicol Screening Plates, in large bag. Each plate is labeled with 3 orange lines (chlor), 1 yellow line (LB) and one purple line (X-gal). The sleeve is labeled 8/1/2018 LB+Chlor+X-Gal
<H4><B>Next:</B></H4> </DIV><DIV class="notebook"><H2>1 August 2018</H2><H4 class="h4left"><B>Jessica</B></H4><H4 class="h4right"><B>Start: 12:30am</B></H4> <H4><B>Redo of Arabidopsis Plates</B></H4><B>Purpose:</B> Previous plates were liquid, agar will be added to remedy this
<H4><B>Protocol:</B></H4>
Same as on 31 July 2018, with the addition of 7.5 g of agar. Will produce 18 plates labeled with three green lines, four of which will be used for planting (found using x/500=10.5/700)
<H4><B>Stop: </B></H4> <H4><B>Next:</B></H4> </DIV><DIV class="notebook"><H2>1 August 2018</H2><H4 class="h4left"><B>Jessica</B></H4><H4 class="h4right"><B>Start: 12:00am</B></H4> <H4><B>Transformation of Cry Sequence and B0030</B></H4><B>Purpose:</B> Develop plates of UAP+Cry and UAP+B0030
<H4><B>Protocol:</B></H4>
Followed Intact Genomics Pamphlet
<H4><B>Notes: Both shaking incubators were set at 30*C, so that was the temperature during the 1 hour incubation. GG1 plates are B0030, GG- and GGW are Cry Sequence and Control (may not be in that order) </B></H4>
Use chlor plates
<H4><B>Stop: </B></H4> <H4><B>Products:</B></H4>
Four plates, labeled:

CT1= from the GG- PCR tube, 100 ul
CT2= from the GG- PCR tube, 200 ul
CT3= from the GG W PCR tube, 100 ul
CT4= from the GG W PCR tube, 200 ul
CT5= from the 7/31 GG1 tube, 100 ul
CT6= from the 7/31 GG1 tube, 200 ul

Three tubes of cells and recovery medium:
GG- from GG- PCR
GG W from GG W PCR
GG1 from 7/31 GG1 PCR
<H4><B>Next:</B></H4> </DIV><DIV class="notebook"><H2>31 July 2018</H2><H4 class="h4left"><B>Jessica</B></H4><H4 class="h4right"><B>Start: 9:45pm</B></H4> <H4><B>Golden Gate: B0030</B></H4><B>Purpose:</B> To re-suspend B0030 ordered from IDT and to place it inside of a UAP
<H4><B>Protocol:</B></H4>
BsmBI Protocol

5/5 MP1 (UAP) = 2.5 ul or ~100ng
IDT B0030 = 10 ul or ~100ng
T4 Ligase = 1 ul
10x T4 Ligase Buffer = 2 ul
BsmBI = 1 ul
Water (for total of 20 ul) = 3.5 ul

Thermocycler Protocol:
20 seconds 37*C
(3 minutes 37*C, then 4 minutes 16*C) X 26 cycles
5 minutes 50*C
5 minutes 80*C
5 minutes 16*C
<H4><B>Notes:</B></H4>
B0030 was re-suspended from IDT protocol
<H4><B>Stop: 10:30pm</B></H4> <H4><B>Results:</B></H4>
Re-suspension and Golden Gate Assembly
<H4><B>Products:</B></H4>
1 Tube with IDT Label of 17-Jul-2018 Rbcs-1A_B0030

1 Tube 7/31 GG 1
<H4><B>Next:</B></H4>
Transformation

</DIV><DIV class="notebook"><H2>31 July 2018</H2><H4 class="h4left"><B>Jessica and Lynell</B></H4><H4 class="h4right"><B>Start: 8:00pm</B></H4> <H4><B>Arabidopsis Plates</B></H4><B>Purpose:</B> Prepare the growth medium for the development of Arabidopsis in the lab
<H4><B>Protocol:</B></H4> <TABLE><TBODY><TR><TD>Component</TD><TD>Amount</TD></TR><TR><TD>Murashige and Skoog salts</TD><TD>2.15 g</TD></TR><TR><TD>1% Sucrose</TD><TD>5 ml</TD></TR><TR><TD>.05% MES</TD><TD>.25 ml</TD></TR><TR><TD>MilliQ Water</TD><TD>500 ml</TD></TR><TR><TD>KOH</TD><TD>N/A (see below)</TD></TR></TBODY></TABLE> <H4><B>Notes:</B></H4>
Murashige and Skoog salts were in the fridge
KOH amount is until pH is 5.7. Due to the lack of a pH meter, pH strips were used and pH was determined to be 5.5-6, so no KOH was added
<H4><B>Stop: 9:00</B></H4> <H4><B>Results:</B></H4>
18 Arabidopsis Plates (18 MS Mediums)
<H4><B>Next:</B></H4>
Plant!

</DIV><DIV class="notebook"><H2>15 July 2018</H2><H4 class="h4left"><B>Jessica Brooks</B></H4><H4 class="h4right"><B>Start: 5:20pm</B></H4> <H4><B>Gel of BBa_B0015</B></H4><B>Purpose:</B> Confirm quality of DNA from 24/7/17, Labeled as 7/24/17 B0015
<H4><B>Protocol:</B></H4>
From Notebook
<H4><B>Notes:</B></H4>
Used 3 uL of ethidium bromide
Used 10 uL of ladder in well closest to side of gel
Used 10 uL of sample in well closer to center of gel
Run at 130V

**Lynell ran an identical gel previous to this
<H4><B>Stop: 7:00</B></H4> <H4><B>Results:</B></H4>
Ryan said lynell's gel is good
<H4><B>Products:</B></H4>
One gel<H4><B>Next:</B></H4> </DIV><DIV class="notebook"><H2>5 July 2018</H2><H4 class="h4left"><B>Jessica Brooks</B></H4><H4 class="h4right"><B>Start: 12:00pm</B></H4> <H4><B>PCR of BBa_B0015</B></H4><B>Purpose:</B> Conduct a PCR and gel of tube 24/7/17 B0015 to check DNA quality
<H4><B>Protocol:</B></H4>
New England BioLabs protocol for Taq 2x Master Mx

Used 1 ul of DNA

30 Cycles of:
95* for 20 seconds
52* for 30 seconds
68* for 30 seconds
<H4><B>Notes:</B></H4>
Used VR and VF2, which may produce an extra band at 500 and hazy ones below (shorter and dimmer than expected product)
<H4><B>Stop: In PCR at 12:40pm</B></H4> <H4><B>Products:</B></H4>
One tube, labeled 7/5 PCR 1
<H4><B>Next:</B></H4>
Run gel

Should see bright band at 500 and dim hazy bands under the 129 pb B0015

</DIV><DIV class="notebook"><H2>2 July 2018</H2><H4 class="h4left"><B>Jessica, Lynell, Luke</B></H4><H4 class="h4right"><B>Start: 12:00</B></H4> <H4><B>InterLab</B></H4><B>Purpose:</B> Inoculate two colonies from previous transformation
<H4><B>Protocol:</B></H4>
Followed standard protocol
<H4><B>Stop: 3:00pm</B></H4> <H4><B>Results:</B></H4>
16 tubes, labeled Device #-Colony Number
<H4><B>Next:</B></H4> </DIV><DIV class="notebook"><H2>1July 2018</H2><H4 class="h4left"><B>Lynell Cunningham, Jessica Brooks, and Lucas Dyer</B></H4><H4 class="h4right"><B>Start: 11:00am</B></H4> <H4><B>Interlab Cell Measurement Transformation</B></H4><B>Purpose:</B> To transform kit plate Test Device DNA into competent cells for further use in the interlab.
<H4><B>Protocol:</B></H4>
Eight wells of kit plate DNA were transformed into eight samples of competent cells according to the Intact Genomics High Efficiency Protocol. Each sample was then plated at 5% and 20% onto chloramphenicol plates and grown overnight.
<H4><B>Stop: 1:30pm</B></H4> <H4><B>Results:</B></H4>
The cells grew overnight on plates.
<H4><B>Products:</B></H4>
A total of sixteen plates were created
<TABLE><TBODY><TR><TD><B>Plate</B></TD><TD><B>Device</B></TD><TD><B>Part Number</B></TD><TD><B>Label</B></TD></TR><TR><TD>1</TD><TD>Negative Control</TD><TD>BBa_R0040</TD><TD>Interlab Negative Control 5%</TD></TR><TR><TD>2</TD><TD>Negative Control</TD><TD>BBa_R0040</TD><TD>Interlab Negative Control 20%</TD></TR><TR><TD>3</TD><TD>Positive Control</TD><TD>BBa_I20270</TD><TD>Interlab Positive Control 5%</TD></TR><TR><TD>4</TD><TD>Positive Control</TD><TD>BBa_I20270</TD><TD>Interlab Positive Control 20%</TD></TR><TR><TD>5</TD><TD>Test Device 1</TD><TD>BBa_J364000</TD><TD>Interlab Test Device 1: 5%</TD></TR><TR><TD>6</TD><TD>Test Device 1</TD><TD>BBa_J364000</TD><TD>Interlab Test Device 1: 20%</TD></TR><TR><TD>7</TD><TD>Test Device 2</TD><TD>BBa_J364001</TD><TD>Interlab Test Device 2: 5%</TD></TR><TR><TD>8</TD><TD>Test Device 2</TD><TD>BBa_J364001</TD><TD>Interlab Test Device 2: 20%</TD></TR><TR><TD>9</TD><TD>Test Device 3</TD><TD>BBa_J364002</TD><TD>Interlab Test Device 3: 5%</TD></TR><TR><TD>10</TD><TD>Test Device 3</TD><TD>BBa_J364002</TD><TD>Interlab Test Device 3: 20%</TD></TR><TR><TD>11</TD><TD>Test Device 4</TD><TD>BBa_J364003</TD><TD>Interlab Test Device 4: 5%</TD></TR><TR><TD>12</TD><TD>Test Device 4</TD><TD>BBa_J364003</TD><TD>Interlab Test Device 4: 20%</TD></TR><TR><TD>13</TD><TD>Test Device 5</TD><TD>BBa_J364004</TD><TD>Interlab Test Device 5: 5%</TD></TR><TR><TD>14</TD><TD>Test Device 5</TD><TD>BBa_J364004</TD><TD>Interlab Test Device 5: 20%</TD></TR><TR><TD>15</TD><TD>Test Device 6</TD><TD>BBa_J364005</TD><TD>Interlab Test Device 6: 5%</TD></TR><TR><TD>16</TD><TD>Test Device 6</TD><TD>BBa_J364005</TD><TD>Interlab Test Device 6: 20%</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Cultures from plates will be inncoulated to LB+Chloramephenicol broth and grown over night.

</DIV><DIV class="notebook"><H2>28 June 2018</H2><H4 class="h4left"><B>Jessica, Lynell, Luke</B></H4><H4 class="h4right"><B>Start: 11:00am</B></H4> <H4><B>InterLab</B></H4><B>Purpose:</B> Complete Calibration for Plate Reader/CFU
<H4><B>Protocol:</B></H4>
Completed Calibrations
<H4><B>Stop: 4:00pm</B></H4> <H4><B>Next:</B></H4> </DIV><DIV class="notebook"><H2>5 May 2018</H2><H4 class="h4left"><B>Lynell Cunningham, Ryan Baumann, Lucas Dyer</B></H4><H4 class="h4right"><B>Start: 1:30pm</B></H4> <H4><B>Minipreps of phytobrick plasmids</B></H4><B>Purpose:</B> To obtain phytobrick plasmids
<H4><B>Protocol:</B></H4>
Three minipreps were performed according to the kitless miniprep procedure. Products were run through the nanodrop to determine concentration.
<H4><B>Stop: 4:30 pm</B></H4> <H4><B>Results:</B></H4> <TABLE><TBODY><TR><TD><B>Sample</B></TD><TD><B>ng/μl</B></TD><TD><B>260/280</B></TD><TD><B>260/230</B></TD></TR><TR><TD>5/5 MP1</TD><TD>39.45</TD><TD>1.84</TD><TD>2.12</TD></TR><TR><TD>5/5 MP2</TD><TD>34.38</TD><TD>1.92</TD><TD>2.01</TD></TR><TR><TD>5/5 MP3</TD><TD>37.94</TD><TD>1.76</TD><TD>1.45</TD></TR></TBODY></TABLE> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>5/5 MP1</TD><TD>5/2 BC3</TD><TD>UAP/BBa_P10500 in pSB1C3</TD></TR><TR><TD>5/5 MP2</TD><TD>5/2 BC4</TD><TD>Alpha 1/BBa_P10501 in pSB1K3</TD></TR><TR><TD>5/5 MP3</TD><TD>5/2 BC5</TD><TD>Alpha 2/BBa_P10503 in pSB1K3</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Digest each miniprep with EcoRI and PstI run products on gel to verify identity

</DIV><DIV class="notebook"><H2>3 May 2018</H2><H4 class="h4left"><B>Jessica Brooks, Lynell Cunningham, and Kent Gorday</B></H4><H4 class="h4right"><B>Start: 16:15</B></H4> <H4><B>Minipreps of amilCP and phytobrick plasmids</B></H4><B>Purpose:</B> To obtain amilCP in pSB1C3, along with the five (UAP + alpha and omega) phytobrick plasmids
<H4><B>Protocol:</B></H4>
Seven minipreps were performed according to the kitless miniprep procedure. Products were nanodropped to determine concentration.
<H4><B>Stop: 19:00</B></H4> <H4><B>Results:</B></H4> <TABLE><TBODY><TR><TD><B>Sample</B></TD><TD><B>ng/μL</B></TD><TD><B>260/280</B></TD><TD><B>260/230</B></TD></TR><TR><TD>5/3 MP1</TD><TD>172.31</TD><TD>1.87</TD><TD>1.88</TD></TR><TR><TD>5/3 MP2</TD><TD>123.26</TD><TD>1.84</TD><TD>2.25</TD></TR><TR><TD>5/3 MP3</TD><TD>50.63</TD><TD>1.78</TD><TD>2.04</TD></TR><TR><TD>5/3 MP4</TD><TD>41.19</TD><TD>1.68</TD><TD>1.54</TD></TR><TR><TD>5/3 MP5</TD><TD>30.47</TD><TD>1.72</TD><TD>1.71</TD></TR><TR><TD>5/3 MP6</TD><TD>8.30</TD><TD>1.26</TD><TD>0.99</TD></TR><TR><TD>5/3 MP7</TD><TD>1.65</TD><TD>0.87</TD><TD>0.78</TD></TR></TBODY></TABLE> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>5/3 MP1</TD><TD>5/2 BC1</TD><TD>amilCP in pSB1C3 (higher concentration?)</TD></TR><TR><TD>5/3 MP2</TD><TD>5/2 BC2</TD><TD>amilCP in pSB1C3 (cleaner prep?)</TD></TR><TR><TD>5/3 MP3</TD><TD>5/2 BC3</TD><TD>UAP/BBa_P10500 in pSB1C3</TD></TR><TR><TD>5/3 MP4</TD><TD>5/2 BC4</TD><TD>Alpha 1/BBa_P10501 in pSB1K3</TD></TR><TR><TD>5/3 MP5</TD><TD>5/2 BC5</TD><TD>Alpha 2/BBa_P10503 in pSB1K3</TD></TR><TR><TD>5/3 MP6</TD><TD>5/2 BC6</TD><TD>Omega 1/BBa_P10505 in pSB1K3</TD></TR><TR><TD>5/3 MP7</TD><TD>5/2 BC7</TD><TD>Omega 2/BBa_P10507 in pSB1K3</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Digest each miniprep with EcoRI and PstI run products on gel to verify identity

</DIV><DIV class="notebook"><H2>2 May 2018</H2><H4 class="h4left"><B>Ryan Baumann and Lucas Dyer</B></H4><H4 class="h4right"><B>Start: 4:00</B></H4> <H4><B>Broth cultures of cloning vectors</B></H4><B>Purpose:</B> To obtain good preps of several cloning vectors
<H4><B>Protocol:</B></H4>
One chloramphenicol broth culture and four kanamycin broth cultures were inoculated
<H4><B>Stop: 4:30</B></H4> <H4><B>Products:</B></H4>
The following broth cultures were created
<TABLE><TBODY><TR><TD>Label</TD><TD>Source</TD><TD>Description</TD></TR><TR><TD>5/2 UAP BC3</TD><TD>5/1 Transformation</TD><TD>UAP/BBa_P10500 in pSB1C3</TD></TR><TR><TD>5/2 Alpha 1 BC4</TD><TD>5/1 Transformation</TD><TD>Alpha 1/BBa_P10501 in pSB1K3</TD></TR><TR><TD>5/2 Alpha 2 BC5</TD><TD>5/1 Transformation</TD><TD>Alpha 2/BBa_P10503 in pSB1K3</TD></TR><TR><TD>5/2 Omega 1 BC6</TD><TD>5/1 Transformation</TD><TD>Omega 1/BBa_P10505 in pSB1K3</TD></TR><TR><TD>5/2 Omega 2 BC7</TD><TD>5/1 Transformation</TD><TD>Omega 2/BBa_P10507 in pSB1K3</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Miniprep all broth cultures

</DIV><DIV class="notebook"><H2>2 May 2018</H2><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 15:00</B></H4> <H4><B>Broth cultures of amilCP in pSB1C3</B></H4><B>Purpose:</B> To obtain good preps of amilCP in pSB1C3
<H4><B>Protocol:</B></H4>
Two LB + chloramphenicol broth cultures of amilCP in pSB1C3 were inoculated
<H4><B>Stop: 15:20</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>5/2 BC1</TD><TD>5/1 amilCP transformation</TD><TD>amilCP in pSB1C3</TD></TR><TR><TD>5/2 BC2</TD><TD>5/1 amilCP transformation</TD><TD>amilCP in pSB1C3</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Miniprep amilCP in pSB1C3 from broth cultures

</DIV><DIV class="notebook"><H2>1 May 2018</H2><H4 class="h4left"><B>Luke Dyer and Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 4:30</B></H4> <H4><B>Transformations of 2018 Cloning vectors</B></H4><B>Purpose:</B> Transformation of cloning vectors and parts needed for summer research.
<H4><B>Protocol:</B></H4>
Transformation of Golden braid destination vectors, UAP, and AmilCP were conducted following Intact genomics 5 Alpha electro competent cells protocol.

UAP and AmilCP were plated on Chloramphenicol plates while GB Destination Vectors were plated on Kanamycin Plates
<H4><B>Stop: 7:30</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD>Part Transformed</TD></TR><TR><TD>Alpha 1/BBa_P10501</TD></TR><TR><TD>Alpha 2/BBa_P10503</TD></TR><TR><TD>Omega 1/BBa_P10505</TD></TR><TR><TD>Omega 2/BBa_P10507</TD></TR><TR><TD>UAP/BBa_P10500</TD></TR><TR><TD>AmilCP/BBa_K592009</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Grow Overnight and inoculate cultures in liquid media.

</DIV><DIV class="notebook"><H2>2-5-18</H2><H4 class="h4left"><B>Ryan Baumann</B></H4><H4 class="h4right"><B>Start: 10:00AM</B></H4> <H4><B>DNA Purification of AmilCP and BBa_J04500 </B></H4><B>Purpose:</B> To purify the promoter and chromoproterin for future miniprojects.
<H4><B>Protocol:</B></H4>
Plasmid minipreps were conducted following Missouri S&amp;T iGEM's Lab manual 2.5 kitless miniprep protocol. DNA was resuspended in 40 μL TE buffer and concentration was tested with a Thermo Scientific Nanodrop 1000.

Nanodrop Results:
<TABLE><TBODY><TR><TD>Name</TD><TD>DNA Conc.</TD><TD>260/280</TD></TR><TR><TD>1-22-18 MP1 (amilCP)</TD><TD>121.73</TD><TD>1.89</TD></TR><TR><TD>1-22-18 MP2 (amilCP)</TD><TD>158.92</TD><TD>1.89</TD></TR><TR><TD>2-1-18 MP1 (BBa_J04500)</TD><TD>119.82</TD><TD>1.87</TD></TR><TR><TD>2-1-18 MP2 (BBa_J04500)</TD><TD>127.71</TD><TD>1.90</TD></TR></TBODY></TABLE> <H4><B>Stop: 10:15 AM</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD>Label</TD><TD>Source</TD><TD>Description</TD></TR><TR><TD>2-1-18 MP1</TD><TD>BBa_j04500 from 2014 kit plate 3</TD><TD>BBa_J04500 inducible promoter</TD></TR><TR><TD>2-1-18 MP2</TD><TD>BBa_j04500 from 2014 kit plate 3</TD><TD>BBa_J04500 inducible promoter</TD></TR><TR><TD>1-22-18 MP1</TD><TD>amilCP Chromoprotein from 2016 kit plate</TD><TD>amilCP blue Chromoprotein in pSB1C3</TD></TR><TR><TD>1-22-18 MP2</TD><TD>amilCP Chromoprotein from 2016 kit plate</TD><TD>amilCP blue Chromoprotein in pSB1C3</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4> </DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>