<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Alberta_Notebook_protocols_primer_design"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Alberta/Notebook/protocols/primer_design">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Alberta/Notebook/protocols/primer_design&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Notebook/protocols/primer_design&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Notebook/protocols/primer_design&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Alberta/Notebook/protocols/primer_design" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Alberta/Notebook/protocols/primer design</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="top-strip"><H2>TEAM ALBERTA</H2></DIV><DIV id="navbar"><UL><LI><A href="https://2010.igem.org/Team:Alberta" class=" nav-home">HOME</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Tour/start" class=" nav-software">AT A GLANCE</A></LI><LI class="headlink"><A href="https://2010.igem.org/Team:Alberta/project" class=" nav-project">PROJECT        </A></LI><LI><A href="https://2010.igem.org/Team:Alberta/project">Overview</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/biobyte2">BioBytes 2.0</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Software">Software</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/modelling">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Kit">The Kit</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Kit_components">Kit Components</A></LI></UL><LI class="headlink"><A href="https://2010.igem.org/Team:Alberta/Achievements/Overview" class=" nav-practices">ACHIEVEMENTS </A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Achievements/Overview">Overview</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Achievements">Medal Requirements </A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Achievements/Media">Media</A></LI><LI class="headlink"><A href="https://2010.igem.org/Team:Alberta/human_practices" class=" nav-practices">HUMAN PRACTICES</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/human_practices">Overview</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/human_practices/distribution_analysis">Distribution Analysis</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/human_practices/HighSchool">High School</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/human_practices/DiscoverElle">DiscoverElle Visit</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/human_practices/safety">Safety</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/parts" class=" nav-parts">PARTS</A></LI><LI class="headlink"><A href="https://2010.igem.org/Team:Alberta/Notebook" class="selected nav-notebook">TIMELINE &amp; PROTOCOLS</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook">Overview</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/protocols">General Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/ReusablePlates">Reusable Plates</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/BasePlasmids">Base Plasmids</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/CreatingParts">Creating Parts</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/TransformingCells">Transforming Cells</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/Anchor">Anchor</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/Beads">Beads</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/Optimizations">Optimizations</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/Software">Software</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/team" class=" nav-team">TEAM</A></LI><UL><DIV id="left-sidebar" class=""><H2><SPAN class="mw-headline" id="General_Protocols">General Protocols</SPAN></H2><UL><LI><A href="/Team:Alberta/Notebook/protocols/invitro_biobyte_assembly" title="Team:Alberta/Notebook/protocols/invitro biobyte assembly"> In Vitro BioByte Assembly</A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/LB" title="Team:Alberta/Notebook/protocols/LB"> LB Plates and Broth</A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/transformations" title="Team:Alberta/Notebook/protocols/transformations">Transformations</A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/overnight" title="Team:Alberta/Notebook/protocols/overnight">5mL Overnight </A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/glycerol" title="Team:Alberta/Notebook/protocols/glycerol"> Glycerol Stock </A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/miniprep" title="Team:Alberta/Notebook/protocols/miniprep"> Plasmid Miniprep </A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/digest" title="Team:Alberta/Notebook/protocols/digest"> Restriction Digest </A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/vector_dephos" title="Team:Alberta/Notebook/protocols/vector dephos"> Vector Dephosphorylation</A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/ligation" title="Team:Alberta/Notebook/protocols/ligation"> Ligation </A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/agarose_gel" title="Team:Alberta/Notebook/protocols/agarose gel"> Agarose Gel Electrophoresis </A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/gel_extraction" title="Team:Alberta/Notebook/protocols/gel extraction"> Gel Extraction </A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/pcr" title="Team:Alberta/Notebook/protocols/pcr"> PCR</A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/colony_pcr" title="Team:Alberta/Notebook/protocols/colony pcr"> Colony PCR </A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/pcr_purification" title="Team:Alberta/Notebook/protocols/pcr purification"> PCR Purification </A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/labelling" title="Team:Alberta/Notebook/protocols/labelling"> Sample Labelling Conventions</A></LI></UL><UL><LI><A href="/Team:Alberta/Notebook/protocols/sequencing" title="Team:Alberta/Notebook/protocols/sequencing"> Fluorescent Sequencing Reaction</A></LI></UL><UL><LI><STRONG class="selflink"> Primer Design</STRONG></LI></UL></DIV><DIV id="center-content" class=""><H2><SPAN class="mw-headline" id="Primer_Design">Primer Design</SPAN></H2><H3><SPAN class="mw-headline" id="Notes:">Notes:</SPAN></H3><H3><SPAN class="mw-headline" id="Typical_limits:">Typical limits: </SPAN></H3><UL><LI> delta Tm &lt; 5 C, hairpin dG &gt;= -2, dimer (hetero or homo) dG &gt;= -6 to -15, but stricter at 3'. 
</LI><LI> Some buffer to allow end enzymes to cut is required, NEB and Fermentas have charts for how much, 6 bases is typically sufficient for anything.
</LI><LI> Primer design limits: <A href="http://www.premierbiosoft.com/tech_notes/PCR_Primer_Design.html" class="external free" rel="nofollow">http://www.premierbiosoft.com/tech_notes/PCR_Primer_Design.html</A></LI></UL><P>primer 3' end hairpin -2 kcal/mol, -3kcal/mol elsewhere okay.
self dimer: -5 kcal/mol (3'), -6 elsewhere
heterodimer:
</P><UL><LI> Hot start helps melt strands.
</LI></UL><H3><SPAN class="mw-headline" id="VNTI_defaults:">VNTI defaults: </SPAN></H3><UL><LI> Tm difference 5 C, hairpin -1 max, dimers -15 max, or -3 at 3'. all checks only within first 10 bp of 3'
</LI><LI> Mike's criteria for PCR primer design:
</LI><LI> Forward primer should be same as coding strand, 5' -&gt; 3', reverse primer should be same as noncoding strand, so that forward primer complements noncoding strand and vice versa
</LI><LI> End in GC or GG, so end sticks down well
</LI><LI> ~50% GC content, so that it will anneal and deanneal well under temp shifts
</LI><LI> Should be 30-45 NTs long. 20-25 is too short
</LI><LI> Restriction site should cut exactly where you want
</LI><LI> Should be an overhang before restriction sites, so that restriction enzymes can do their work
</LI></UL><TABLE><TBODY><TR><TD>Repeat motifs  </TD><TD> Description &amp; Effect
</TD></TR><TR><TD>Simple repeats </TD><TD> A repeated sequence of four or more nucleotides, which is repeated: (...AATCGA...AATCGA...). Simple repeats can generate secondary binding sites for primers. Stable hybridization to secondary binding sites results in nonspecific amplification. Repeats &gt;3-4.
</TD></TR><TR><TD>Inverse repeats </TD><TD> A self-complementary sequence motif of four or more nucleotides (stem-loop or hairpin motifs): (...AATGGC...GCCATT...). Inverse repeats can cause inefficient priming because they lead to formation of stable hairpins in the binding region, or within the amplicon.
</TD></TR><TR><TD>Homopolymeric runs </TD><TD> A sequence of four or more identical nucleotides: (...AAAAA...). Homopolymeric runs can be considered a special case of direct repeats. These can cause ambiguous binding of primers to their target site (&quot;slippage effect&quot;). Poly (A) and poly (T) stretches should also be avoided because these will &quot;breathe&quot; and open up stretches of the primer-template complex. Additionally, runs of three or more G residues can cause problems due to intermolecular stacking.
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Procedure:">Procedure:</SPAN></H3><H3><SPAN class="mw-headline" id="For_Primer_Design:">For Primer Design:</SPAN></H3><P>1. Obtain a licensed version of Vector NTI.  
2. Download the E. coli K-12 MG16550 Genome from Gen Bank.
</P><UL><LI> To do this, type NC_000913.2 into the open Gen Bank file option under the tools panel.
</LI></UL><P>3. Obtain the following from the FTP.  It can be found in the Primer Design folder inside the Primers folder:
</P><UL><LI> Wayneâs Protocol: The Primer âadd onsâ containing the RE sites and 8bp extensions (download the correct set based on the instructions, which you receive with the gene list).
</LI><LI> A list of the genes which require primers and their locations.
</LI></UL><P>4. Type into the blank panel in the bottom right corner of the Vector NTI main window, the start and end locations of the gene of interest.
</P><UL><LI> Look to ensure that there is an ATG at one side of the sequence and a stop codon (TAA, TGA, TAG) on the other </LI><LI> The orientation of the ATG and stop codon varies depending on the orientation of the gene (forward or reverse).
</LI></UL><P>5. Determine the sequence numbers for the base pairs directly beside the start and stop codons (we do not need these as they are in the primers we are designing ourselves) and select the new sequence as in step 4.
</P><UL><LI> A quick way of doing this is just to add 3 to the start bp numbers and subtract 3 from the last bp number.
</LI></UL><P>6. With the sequence selected, click on the Primer Design tab and select âAmplify Selectionâ¦â.
7. In the lower left hand corner click on the load button and select Wayneâs Protocol which you downloaded from the FTP. 
8. In the sense and anti-sense boxes type in the sequence information which you downloaded from the FTP.
</P><UL><LI> Directionality is important! If the gene is encoded in a forward direction the sense sequence should contain the ATG site.
</LI><LI> The anti-sense sequence should contain the TTA stop codon.
</LI><LI> If the gene is encoded in the reverse direction the antisense sequence should contain the ATG start site.
</LI><LI> IMPORTANT: Do not modify these sequences all you have to do is copy and paste them in the boxes no alterations are required!!!
</LI></UL><P>9. Let the program run. Compare the resulting score of each primer sequence (found in the text box beside the length of the primer. Shoot for scores above 150. Write down this score in your spreadsheet).
</P><UL><LI> Underneath the score, it shows the area that was amplified.  Ensure that it did amplify the section that you wanted.
</LI></UL><P>10. Next check for dimers by right clicking on the sense primerâs sequence in the text box and selecting âthermodynamic propertiesâ.
11. Click the Analyze box and finally click on âDimers &amp; Hairpin Loopsâ.
12. Two boxes will appear, the top is for dimers, and the bottom for hairpins. Scroll through each and look at the dG values for each one (you can see the next one by clicking â&gt;&gt;â). Write down the lowest dG scores for both.
13. Repeat steps 10 â 12 for the antisense primer.
</P><UL><LI> It is important that the dG values for the dimers does not fall below -6 for any sequence or -5 if the dimer is in the 3â end of the sequence. For hairpins, the dG value should not be below -1.
</LI><LI> If it does fall below these values see step 14, if it does not then skip to step 15.
</LI></UL><P>14. If the primers will not work go back into the primer design tab and click on âDesign PCR Primers Inside Selectionâ¦â. Click the quality tab and lower the numbers in the boxes (from 9 to 7 and if it does not work again then to 5). Then repeat steps 9 â 13.
</P><UL><LI> Remember to switch these numbers back or reload Wayneâs PCR Program before doing another set of primers.
</LI></UL><P>15. If you cannot find any primers which actually fit into the sequences, pick the best possible primers you can produce.
16. Once you have found primers of interest write down the remaining information needed into the excel table.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Alberta/Notebook/protocols/primer_design">http://2010.igem.org/Team:Alberta/Notebook/protocols/primer_design</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Alberta/Notebook/protocols/primer_design" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Alberta/Notebook/protocols/primer_design" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Alberta/Notebook/protocols/primer_design&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Alberta/Notebook/protocols/primer_design&amp;oldid=171803" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></BODY></HTML>