"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Notebook June
From 2010.igem.org
(Redirected from Team:Alberta/Calendar )
genomikon
home | project | human practices | parts | software | notebook | team
May | June | July | August | September | October | November
June 2010
Su
Mo
Tu
We
Th
Fr
Sa
--
--
01
02
03
04
05
06
07
08
09
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
--
--
--
Notebook
Building Parts
Testing Parts
Assembly Method
Plates
Competent Cells
Software
iGEM 2010 Notebook
The lab notebook chronicles our journey in the creation of the Genomikon kit.  Many paths were woven together in space and time to reach this finished masterpiece.  To help you navigate through these trials with us we have laid out our notebook in a layered fashion.  This page gives a sketch of each project and how it interacts with each other.  Then follow the links to a projects page for time line of the major landmarks and accomplishments.  If you require more details on the project the links within that page will take you to our day-by-day work log.
Building Parts
The Building Parts project was responsible to first build a plasmid (plasmid 01)that contained our own specialized prefix and suffix nested inside of the standard BioBrick prefix and suffix.  After plasmid 01 existed we inserted the CcdB gene (the "death" gene) between our prefix and suffix removing the gene for Kanamycin resistance (plasmid 02).  Plasmid 02 is fantastic base plasmid from which we are able to amplify any part at all because it provides a positive selection marker when transformed into DH5α. At this point we were able to make parts en masse to put in our kit. After obtaining a particular part in a plasmid we PCRed the part and digested it ready to use in Assembly or to Test the plasmid.
Testing Parts
Before we were able to test parts we created 2 base testing plasmids (vector 01 and vector 02).  Vector 01 is designed to test Open Reading Frame parts, or parts that code for proteins.   The part is flanked by a promoter and the start codon on one side and a stop codon and terminator on the other.  Vector 02 is designed to test linker parts, or parts that control the expression of the Open Reading Frame parts they are next two.  In Vector 02 the part is flanked by two distinct reporter genes, that by comparing the relative expression of the 2 reporter genes we can determine the behavior of the linking part.
Assembly Method
Insert description here.
Plates
Insert description here.
Competent Cells
Insert description here.
Software
Insert description here.
June 1, 2010
Building Parts
KanA/B' and KanB/A' fragements PCRed on 11-05-2010 , digested with BsaI-HF at 37oC for 1.5hours, heat inactivated at 65oC for 30 minutes.  Tried to ligate KanA/B' fragments to each other and tried to ligate KanB/A' fragments to each other.  Also tried to ligate KanA/B' fragments with KanB/A'.  Ligated with T4 DNA ligase for 3 hours at 21oC.
Set up liquid cultures of KanRA/B'-Bsa and KanR B/A'-Bsa in pSB1C3 from plates streaked with on 30-05-2010 .
June 2, 2010
Building Parts | Software
Building Parts
Miniprepped liquid cultures from 01-06-2010 . Ran a 1% agarose gel of the ligations performed 01-06-2010 .
<-- image of gel-->
To optimize the Restriction and ligation of BsaI-HF, digested KanA/B' and KanB/A' fragements PCRed on 11-05-2010 with the following recipe:
Digestion Recipe:
14μL either A/B' or B/A' Kanamycin resistance cassette (approx. 100ng/μL)
5μL 1/10 dillution of 100X BSA
5μL 10X NEBuffer4
1.5&mu:L BsaI-HF
24.5μL MilliQ
Digested at 50oC for 1hour, heat inactivated the enzyme at 65oC for 20 minutes PCR purified the digests.
Software
Mike
* Got inline forms to work, and ajax to work
Steve
*  Added simple construct validation
* Added simple sequence annotation
* Added reverse strand display
June 3, 2010
Building Parts
Tried to ligate the KanA/B' fragments to itself.  Tried to ligate the KanB/A' fragments to itself.  Tried to ligate the KanA/B' fragments to the KanB/A' fragments.
Ligation Recipe:
8μL of digest from 02-06-2010 (either 8μL of one of the fragments or 4μL of each)
1μL T4 DNA ligase
6μL 5X Buffer
15μL MilliQ H2O
Took aliquots of ligations at varying times and ran 1% agarose gels to test ligation.
<--images-->
Digested some of Minipreps of the KanA/B' fragment inserted into pSB1C3 from 02-06-2010 .
Digestion Recipe:
14μL plasmid (approx 300-400ng/&mu:L)
5μL 10X NEBuffer 4
5μL 1/10 dilution of 100X BSA
1.5μL BsaI-HF
24.5μL MilliQ H2O
Construct of Amp Resistance in pSB1C3 Backbone
Made a 1/100 dilution of AmpR and TetR PCR Products from 27-05-2010 and performed PCR reactions to produce antibiotic inserts to make parts.
PCR Recipe:
35.5μL MilliQ H2O
1μL 10 uM dNTPs
5μL 10X PCR Buffer
2μL 50 uM MgCl2
2.5μL Primer + (ApR 1/10+ or TR 1/10+)
2.5μL Primer - (ApR 1/10- or TR 1/10-)
1μL Template (1/100 diluted AmpR or TetR)
0.5μL Taq Polymerase
PCR Program:
5 min-94oC
45 sec-94oC
1 min-60oC
1 min-72oC
Repeat 2 through 4 35 times
5 min-72oC
Experiment continue on 09-06-2010 .
June 4, 2010
Building Parts
Tried to test the limits of ligation reaction. Ligation Recipe of plasmids cut on 03-06-2010 :
8μL digestion mixture
6μL 5X T4 ligase buffer
1μL T4 ligase
15μL MilliQ
Tried to further reaction of KanA/B' fragments to KanB/A' fragments.  To the existing Ligase reactions from 03-06-2010 added:
1μL T4 ligase
6μL 5X T4 ligase buffer
23μL MilliQ
Tried to set limits of Kan fragments that would ligate:
24μL digestion from 02-06-2010 (either 24μL of one of the fragments or 12μL of each)
6μL 5X T4 ligase buffer
1μL T4 ligase
<--images-->
June 7, 2010
Software
Steve
*  Added more complicated custom sequence annotation
* Integrating Jacqueline's login stuff with construct & biobyte owner/admin permissions and junk
June 8, 2010
Testing Parts | Software
Testing Parts
To test the function of E.coli smell variant experiments for our kit, BBa_J45120 (Wintergreen) and BBa_J45200 (Banana) from the MIT 2006 BioBrick Registry were transformed in DH5α cells.
Software
Steve
*  Integrating construct designer with mike's electronic lab book back-end
*  Created a StepGenerator that generates a crude protocol based on the contents of an experiments associated constructs
June 9, 2010
Building Parts | Software
Building Parts
Construct of Amp Resistance in pSB1C3 Backbone
Restriction Digested AmpR and TetR inserts from 03-06-2010 to be ligated with psB1C3 vector later on.
Digestion Recipe for AmpR:
13.4μL MilliQ H2O
0.60&mu:L AmpR (333.3 ng/μL, determined by nanodrop)
2μL 10X BSA
2μL 10X Buffer 4
2μL BsaI
Digestion Recipe fro TetR:
13.3μL MilliQ H2O
0.7&mu:L TetR (571.3 ng/μL, determined by nanodrop)
2μL 10X BSA
2μL 10X Buffer 4
2μL BsaI
Both digestions were incubated at 50oC for 1hour, heat inactivated the enzyme at 70oC for 20 minutes.
Experiment continue on 10-06-2010 .
Software
Mike
* Added FlexImage plugin and integrated with lab book
June 10, 2010
Building Parts | Software
Building Parts
Double Digested Kan/Chlor minipreps to determine orientation.
Digestion Recipe:
5μL 10X Buffer 3
1μL XbaI
1μL PstI
5μL Kan/Chlor fragments
33μL MilliQ H2O
5μL 10X BSA
Incubated at 37oC for 1hour, heat inactivated the enzyme at 80oC for 20 minutes.
<---gel image of 10.06.10 Karina1--->
Construct of Amp Resistance in pSB1C3 Backbone
Restriction Digested A/B' psB1C3 vector.
Digestion Recipe:
10μL MilliQ H2O
5&mu:L psB1C3
2μL 10X BSA
2μL 10X Buffer 4
1μL BsaHF
Incubated at 37oC for 1hour, heat inactivated the enzyme at 80oC for 20 minutes.
Ligated digested A/B' psB1C3 with AmpR from 09-06-2010 .
Ligation Recipe:
9μL A/B' psB1C3 vector
5μL AmpR insert
6μL 5X Ligase Buffer
1μL Ligase
9μL MilliQ H2O
Incubated at room temperature for 45 minutes. Transformed with DH5α cells using 15μL of the ligated A/B' psB1C3 with AmpR.
Experiment continue on 11-06-2010 .
Software
Mike
*  Fixed all bugs with image uploading, cleaned up routing for lab book
* added inline upload forms to experiment view
June 11, 2010
Building Parts | Testing Parts | Software
Building Parts
Construct of Amp Resistance in pSB1C3 Backbone
We got colonies of psB1C3 with AmpR from the DH5α transformations from 10-06-2010 . We made overnights.
Experiment continue on 12-06-2010 .
Testing Parts
We experimented with L'Oreal skin toner LB media to see if BBa_J45120 plasmids from 08-06-2010 would produce a wintergreen odor. We incubated cultures in L'Oreal LB in a 5%, 10% and 15% concentration.
Software
Mike
*  synced up with Steve
* added ordering functionality to steps
* added ability to insert steps between other steps
June 12, 2010
Building Parts | Testing Parts
Building Parts
Construct of Amp Resistance in pSB1C3 Backbone
Miniprep of psB1C3 with AmpR was made from the overnights from 11-06-2010 .
Experiment continue on 16-06-2010 .
Testing Parts
L'Oreal LB cultures from 11-06-2010 had little growth success. Two of the 5% L'Oreal LB cultures grew.
June 14, 2010
Software
Mike
*  fixed up step model code
* improved javascript on experiment page
June 15, 2010
Software
Mike
*  added associations between users and experiments
* added code for changing views based on ownership
* added user profile page
To do:
* add publish experiment checkbox to profile page
* move javascript for experiment view to its own file
* fix weird bug that reappeared in form on Experiment view
* add user "notes" to steps
* add image caching to FlexImage stuff
June 16, 2010
Building Parts
Construct of Amp Resistance in pSB1C3 Backbone
psB1C3 with AmpR miniprep from 12-06-2010 ran undigested on a 1% agarose gel.
<--gel image of Karina's 16.06.10-->
We digested psB1C3 with AmpR to see if the AmpR insert would be released from the psB1C3 vector.
Digestion Recipe:
14.1μL MilliQ H2O
0.9μL psB1C3 with AmpR (222.5 ng/μL, determined by nanodrop)
2μL 10X BSA
2μL 10X Buffer 4
1μL BsaI
Incubated at 50oC for 1hour, heat inactivated the enzyme at 65oC for 30 minutes.
Also, we made more overnights of psB1C3 with AmpR.
Experiment continue on 17-06-2010 .
June 17, 2010
Building Parts
Construct of Amp Resistance in pSB1C3 Backbone
Digested psB1C3 with AmpR from 16-06-2010 was ran on a 1% agarose gel.
<--17.06.10 Anh-->
June 28, 2010
Building Parts
Transformed 2009 Cambridge color series parts:  Bba_K274100, BBa_K274200, BBa_K274002, BBa_K274003,  BBa_K274004, BBa_J23100.  Also transformed 2004 UTAustin BBa_M30109. Transformed using 5uL of DNA.
Made starter culture of Dbl3 in LB at 11:03am. Inoculated large overnight liquid culture as per specifications in Inoue method at 6:00pm
Performed an enzyme efficiency experiment to determine which of BbsI, BfuAI, BsaI, or BsaI-HF works most efficiently.
June 29, 2010
Building Parts
Made a glycerol stock of PL5 from 1.5mL liquid overnight made 28-06-2010 .
Took remaining transformation of Dbl3 cells with ccdB BfuA/B' and ccdB BfuB/A' from 28-06-2010 out of the fridge and plated 25, 50 and 100μL of each on chloramphenicol LB plates
From the chloramphenicol and chloramphenicol/Kanamycin plates streaked with ccdB BbsA/B', ccdB BbsB/A',ccdB BsaA/B' and  ccdB BsaB/A' on [[#28-06-2010|28-06-2010]], only ccdB BsaB/A' #6,8,10,11,12,14,15,18,  ccdB BbsA/B' #8,15 and 20 grew on only the chloramphenicol plate.  Made 5mL overnight liquid cultures with chloramphenicol of the some of the streaks that worked (ccdB BsaB/A' #6,8,10,11,12, ccdB BbsA/B' # 15, 20 and ccdB BsaB/A'#5 which didn't work but is a positive control).
Miniprepped liquid culture grown 28-06-2010 of Kan BsaA/B' #7.  Concentration is 293.4ng/μL.
All the transormations of Cambridge 1009 color series performed 28-06-2010 worked.  Made 5mL liquid cultures of each part.
Made competent Dbl3 cells by the Inuoe method from liquid cultures grown overnight. (set up on 28-06-2010 ). The OD of the culture was 0.775.
June 30, 2010
Building Parts
We Ran enzyme efficiency experiment with the same specifications as on 28-06-2010 but omitting BsaI-HF from the experiment because BsaI, BbsI and BfuAI appear to be the best of the enzymes.  Also we ran the experiment with 10 Units of each enzyme not 20 Units.
Miniprepped liquid cultures of ccdB Bbs A/B' # 15 and 20 and of ccdB BsaB/A'#6, 8, 10, 11, 12 and 5 (which did grow on both Kan/Chlor and Chlor plates, so it is a positive control for the incorrect plasmid) grown 29-06-2010 .  The concentrations of the minipreps were between 200 and 400ng/mu;L and the curves did not show contamination.   Digested approximately 500ng of minipreps with EcoRI and PstI with the following recipe:
19.5μL MilliQ H2O
2.5&mu:L Miniprep
3μL 10X BSA
3μL 10X NEBuffer 3
1μL EcoRI
1μL PstI
made 5mL culture of Kan BbsA/B' #1 from <--Find date-->
Ya! We got colonies of the ccdB BfuA/B' an ccdB BfuB/A' part plasmids that were plated 29-06-2010 .
Miniprep of liquid cultures made 29-06-2010 of transformations of Cambridge 2009 color series. Concentrations are as follows: J23100, 397.7 ng/μL; K274100, 388.9ng/μL; K274200, 556.5ng/μL; K274004, 106.1ng/μL; K130109, 82.5ng/μL; K274003, 161.0ng/μL.
Constructing Kan, Chlor and Tet Parts in pSB1A3 Backbone
Strategy: to insert Kan into pSB1A3 after cutting both with NotI. Then, digest with BsaI to take Kan out and insert chlor and tet parts.
Restriction Digest with NotI Protocol:
Kan AB PCR product (82.3 ng/ul) 1ul
pSB1A3 plasmid (30.5 ng/ul) 3ul
NotI enzyme 1ul
10x REact3 buffer 1ul
MilliQ H2O 4ul
Total 10ul
Incubated at 37C for 1 1/2 hours.
Ligation Protocol:
Digested pSB1A3 and Kan AB fragment 8ul
T4 DNA ligase 1ul
5X T4 DNA ligase buffer 6ul
MilliQ H20 5ul
Total 20ul
Incubated at room temperature for 1 hour.
Gel Electrophoresis:
Lanes 2-4 are digests of pSB1A3 and kan AB fragment with NotI. Lanes 5-7 are ligations of the digested pSB1A3 and kan fragments. Note that no ligated DNA shows up on the gel due to a small volume of digestion (3ul) loaded. Lanes 2-4 are digests of pSB1A3 and kan AB fragment with NotI. Lanes 5-7 are ligations of the digested pSB1A3 and kan fragments. Note that no ligated DNA shows up on the gel due to a small volume of digestion (3ul) loaded.
Transformation: Followed Inoue Protocol using 50ul DH5-alpha competent cells and 2ul of ligated pSB1A3 and kan AB part. Plated 100ul, 100ul, and 400ul on Kan/Amp plates overnight at 37C. Results:
Plate 1 (100ul): TNTC white colonies and ~20 red colonies
Plate 2 (100ul): TNTC white colonies and ~3 red colonies
Plate 3 (400ul): TNTC white colonies and ~30 red colonies
Experiment continues on 06-07-2010
site map | sponsors | contact us
Retrieved from " http://2010.igem.org/Team:Alberta/Notebook_June "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
