"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Notebook/protocols/labelling
From 2010.igem.org
TEAM ALBERTA
HOME
AT A GLANCE
PROJECT
Overview
BioBytes 2.0
Software
Modelling
The Kit
Kit Components
ACHIEVEMENTS
Overview
Medal Requirements
Media
HUMAN PRACTICES
Overview
Distribution Analysis
High School
DiscoverElle Visit
Safety
PARTS
TIMELINE & PROTOCOLS
Overview
General Protocols
Reusable Plates
Base Plasmids
Creating Parts
Transforming Cells
Anchor
Beads
Optimizations
Software
TEAM
General Protocols
In Vitro BioByte Assembly
LB Plates and Broth
Transformations
5mL Overnight
Glycerol Stock
Plasmid Miniprep
Restriction Digest
Vector Dephosphorylation
Ligation
Agarose Gel Electrophoresis
Gel Extraction
PCR
Colony PCR
PCR Purification
Sample Labelling Conventions
Fluorescent Sequencing Reaction
Primer Design
Sample Labelling Convention
Draft U of A iGEM Documentation Standards:
Sample naming:
Good labeling and storage of plasmids, glycerol stock, plates, intermediate constructs, etc. is absolutely crucial in enabling team members to find and identify the reagents they need.
In addition to being correct and complete in their descriptions, we also need to ensure that using the labeling and filing systems does not become the major work activity; systems must be effective but efficient.
The following system has been developed through many years of experience (both in the lab and in systems design) and considerable attention has been paid to making it match these criteria. However, these standards are intended to be dynamic and subject to the consensus of the project team.
Consider them suggestions for the start of an effective communications protocol.
Dates:
Dates are useful in tracing the history of the project, cross-referencing to lab notebooks, and assessing sample freshness.
Where space allows, We are using DD/MM/YYYY. The four digit year and the hyphens ensure unambiguous parsing. When put into a list, they also sort correctly as strings. Dates consisting of numbers separated by slashes may be in any of a wide variety of orders and should be considered ambiguous.
Eppendorf tubes: (microfuge tubes, 1.5 ml tubes, etc.)
Give every tube a label, even temporary ones.
Labels for permanent constructs should have a name or number that is described by an entry in a lab book or wiki. Naming conventions and allowable part numbers for a team are designated by iGEM HQ each year.
Following sequence verification, a colored label should be affixed to the top of a permanent construct and it should be stored in the appropriate -20ºC box.
Permanent DNA (i.e. any finished and verified construct) should have the date and a page reference number (page where final sequence was verified) on the side of the tube. For example, this might look like 00050-128; where the first 5 digits are the book number and the last three are the page number.
It is never necessary to write “miniprep” on a tube as this is the default. Other “in progress” markings may be used as follows (for 1.5 ml tubes):
“/” means “treated with enzymes…”. For example WM0302/EcoRI+PstI means WM0302 digested with enzymes EcoRI and PstI at the same time (i.e. a double digest). /EcoRI/PstI means digested with EcoRI then with PstI. This allows you to describe a string of treatments, including dephosphorylation, phosphorylation, blunt ending, etc.
PCR products should say “PCR” on the tab of the tube. It is not necessary to put primer names on the tube, though the tube should be cross-referenced to the lab book.
Gel cutouts (fragment in gel) should have the fragment size along with restriction enzymes used or “PCR” on the tube.
Gel-purified fragments should have a “P” on the tab. Digests or PCR used to produce the fragment should be noted.
Ligation mixes should have an “L” on the tab of the tube. Ligations may specify the two components being ligated, but it is better to give them a new construct name.
A series of “in progress” tubes (e.g. several plasmids all subjected to the same digest) may be numbered “1, 2, 3,…” or “a,b,c,…” or any other meaningful system. The first tube in a series should contain all the information expected for a permanent DNA tube. It should have a clear, descriptive label on the top and a date and lab book page reference on the side.
“In progress” tubes may be stored in plastic racks or in more permanent freezer boxes in the -20ºC freezer. In general, racks should only store tubes while the tube contents are under active (i.e. at least twice a week) use. Tubes left untouched in racks for over a week should be returned to their proper freezer boxes.
Freezer boxes (both -20 ºC and -80 ºC should be labeled by project NOT by person). The team should cooperate to develop project labels for boxes.
Enzyme names:
One letter abbreviations for enzymes are useful. For restriction enzymes not in this list, use the full REBASE name.
Abbreviation
Name
E
EcoRI
X
XbaI
N
NotI
S
SpeI
P
PstI
A
Alkaline Phosphatase
L
T4 DNA Ligase
People's initials:
Two or Three-letter abbreviations are useful for labeling lab books, samples, etc in a concise way. See the People page for what initials mean who.
Plates:
The stripe on the side indicates resistance. Multiple stripes have multiple resistance.
Default is high concentration, for high copy plasmids, double stripe indicates low concentration, for low copy plasmid or on chromosome resistance.
Color
Antibiotic
No color or Black
Ampicillin
Red
Kanamycin
Yellow
Tetracyclin
Blue
Chloramphenicol
Annotation:
Parts submitted to the registry must be annotated. Accurate annotation is useful both during construction and for downstream users of the part.
Use Genbank format, with a .gb extension, for defining sequences and annotations. Both APE and Vector NTI, as well as many other programs, can read and write Genbank.
Do not annotate restriction sites. Restriction sites can be found by the software.
Use meaningful labels.
Names for parts and constructs
Retrieved from " http://2010.igem.org/Team:Alberta/Notebook/protocols/labelling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
