"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Notebook May
From 2010.igem.org
genomikon
home | project | human practices | parts | software | notebook | team
May | June | July | August | September | October | November
May 2010
Su
Mo
Tu
We
Th
Fr
Sa
30
31
--
--
--
--
01
02
03
04
05
06
07
08
09
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
Notebook
Building Parts
Testing Parts
Assembly Method
Plates
Competent Cells
Software
iGEM 2010 Notebook
The lab notebook chronicles our journey in the creation of the Genomikon kit.  Many paths were woven together in space and time to reach this finished masterpiece.  To help you navigate through these trials with us we have laid out our notebook in a layered fashion.  This page gives a sketch of each project and how it interacts with each other.  Then follow the links to a projects page for time line of the major landmarks and accomplishments.  If you require more details on the project the links within that page will take you to our day-by-day work log.
Building Parts
The Building Parts project was responsible to first build a plasmid (plasmid 01)that contained our own specialized prefix and suffix nested inside of the standard BioBrick prefix and suffix.  After plasmid 01 existed we inserted the CcdB gene (the "death" gene) between our prefix and suffix removing the gene for Kanamycin resistance (plasmid 02).  Plasmid 02 is fantastic base plasmid from which we are able to amplify any part at all because it provides a positive selection marker when transformed into DH5α. At this point we were able to make parts en masse to put in our kit. After obtaining a particular part in a plasmid we PCRed the part and digested it ready to use in Assembly or to Test the plasmid.
Testing Parts
Before we were able to test parts we created 2 base testing plasmids (vector 01 and vector 02).  Vector 01 is designed to test Open Reading Frame parts, or parts that code for proteins.   The part is flanked by a promoter and the start codon on one side and a stop codon and terminator on the other.  Vector 02 is designed to test linker parts, or parts that control the expression of the Open Reading Frame parts they are next two.  In Vector 02 the part is flanked by two distinct reporter genes, that by comparing the relative expression of the 2 reporter genes we can determine the behavior of the linking part.
Assembly Method
Insert description here.
Plates
Insert description here.
Competent Cells
Insert description here.
Software
Insert description here.
May 30, 2010
Building Parts
From the transformation of DH5α cells with pSB1C3-KanR performed on 28-05-2010 , we took 12 distinct colonies of each KanA/B' and KanB/A', streaked them on a new chloramphenicol plate and inoculated 5ml liquid cultures with the appropriate antibiotics overnight at 37oC. We also picked colonies of pSB4A5-J04450, pSB4C5-J04450 and pSB3T5-J04450, streaked and made 5mL liquid cultures of them too.
May 31, 2010
Building Parts | Software
Building Parts
9/12 of the pSB1C3-KanA/B' Liquid cultures 30-05-2010 were successful and only 1/12 of the pSB1C3-KanB/A' liquid cultures were successful.  The pSB4A5, pSB3T5 and pSB4C5 liquid cultures worked.  Miniprepped all the liquid cultures that worked. However, the streaks on the plates worked.
Performed a restriction digest of an aliquot of the pSB1C3-KanA/B' and pSB1C3-KanB/A' minipreps.  Digested with XbaI at 37oC for one hour and then with EcoRI at 37oC for one hour.  Ran a 1% agarose gel of the digests to determine the orientation of the KanR fragments.
<--Need an image-->
KanA/B' and KanB/A' fragments PCRed on 11-05-2010 , digested with BsaI at 37oC for 1.5hours, heat inactivated at 65oC for 30 minutes.  Tried to ligate KanA/B' fragments to each other and tried to ligate KanB/A' fragments to each other.  Also tried to ligate KanA/B' fragments with KanB/A'.  Ligated with T4 DNA ligase overnight at 16oC.
Software
Mike
* working on lab book
* so far have working forms for adding new steps and experiments and navigation of experiments
* today was working on inline editing features and added some styles to the layout
* things to work on: get inline editing to working state, add images to experiment views and possibly image uploading
Jacqueline
* added user authentication
* added glossary entry form
* working on encyclopedia entry form
Steve
*  Working on parts designer javascript applet
o Functioning drag and drop
o Working on adding more features like sequence annotation
May 10, 2010
Building Parts
Construct of Kan Resistance Bsa in pSB1C3 Backbone
PCR Kanamycin resistance cassette fragment from p1003 containing either the A/B' ends or the B/A' ends. Fragments formed are called KanR A/B'-Bsa and KanR B/A'-Bsa respectively.
PCR Recipe:
1μL p1003 (approx. 1ng)
2.5μL prA_p1003+
2.5μL prB'_p1003-
5μL 10X PCR buffer
1μL 10uM dNTPs
2μL 50uM MgCl2
0.5μL Taq polymerase
35.5μL MilliQ H2O
Same recipe for KanR B/A'-Bsa except primers are prB_p1003+ and prA'_p1003-.
PCR Program:
5 min-94oC
45 sec-94oC
1 min-60oC
1 min-72oC
Repeat 2 through 4 35 times
5 min-72oC
<--Image of gel performed that day. 5μL of each PCR reaction, 1μL of 10X loading dye and 4μL MilliQ water in a 1% agarose gel -->
Experiment continues on 11-05-2010 .
May 11, 2010
Building Parts
Construct of Kan Resistance Bsa in pSB1C3 Backbone
PCR purification of KanR A/B'-Bsa and KanR B/A'-Bsa created 10-05-2010 with Qiagen PCR cleanup kit.
Determined concentrations by nanodrop:  KanA/B'-Bsa: 101.1ng/μL and KanB/A'-Bsa:89.6ng/μL
Experiment continues on 19-05-2010 .
May 17, 2010
Building Parts
Producing DH5α Competent Cells Stock
Innoculated 250mL overnight cultures with 10mL,4mL and 2mL of a starter culture of DH5α.  Left shaking at 18oC overnight.
Experiment continues on 18-05-2010 .
May 18, 2010
Building Parts
Producing DH5α Competent Cells Stock
Prepared DH5α E.Coli competent cells using the Inoue Method.
Transforming pSB1C3 Plasmid Backbone
Transformed DH5α cells with pSB1C3-J04450 and grew overnight at 37oC on Chloramphenicol plates
Experiment continues on 25-05-2010 .
May 19, 2010
Building Parts
Construct of Kan Resistance Bsa in pSB1C3 Backbone
From the transformation of DH5α cells with pSB1C3-J04450 performed on 18-05-2010 , we took 4 distinct colonies, streaked them on a new chloramphenicol plate and inoculated 5ml liquid cultures. These overnights will be used for minipreps for the plasmid backbone for the KanR Bsa insert.
Experiment continues on 20-05-2010 .
May 20, 2010
Building Parts
Construct of Kan Resistance Bsa in pSB1C3 Backbone
Performed a Miniprep of 3 of the 4 5ml liquid cultures of DH5α cells with pSB1C3-J04450 from 19-05-2010 .  Took a 1μL sample of the Miniprep solutions and digested with NotI at 37oC for 1 hour.
Digestion Recipe:
1μL Miniprep (between 153.2 ng/μl and 302.7ng/μl determined by nanodrop)
1μL NotI
1μL 10X ReACT 3
7μL MilliQ
Ran Digestion on a 1% agarose gel to check that the plasmid obtained with what we expected.
<-- Image of Gel-->
Experiment continues on 27-05-2010 .
May 25, 2010
Building Parts | Assembly Method
Building Parts
Transforming pSB1C3 Plasmid Backbone
Made 1.5mL LB liquid cultures of pSB1C3 from the plate streaked on 19-05-2010 and added chloramphenicol.
Experiment continues on 26-05-2010 .
Assembly Method
Discussed an assembly method. Will start with a polyA tail Anchor oligo which will connect to a polyT tail which is bound to the bead. The anchor will consist of a primer region, a BsaI cut site, a buffer region, and finally, an A prime or B prime end. A byte with an appropriate end can be added to this anchor, and another byte can be added onto that, etc... The last thing to be added to the construct is a Terminator byte consisting of a buffer region, bsaI cut site, primer region and a polyT tail. This second primer sequence must be as uncomplimentary as possible to the anchor's primer site so as to minimize primers binding to each other when PCRing the assembled construct. The primers melting point must be around 65 oC. After the construct is assembled on the bead, it is heated off and the polyA tail of the anchor binds to the polyT tail of the terminator and a plasmid is born.
We will purchase NEB oligo dT magnetic and cellulose beads which have a polyT tail (5' to 3') 25 nucleotides long. They are meant for mRNA isolation, but should work fine for our purposes. Used the IDT Analyzer software to determine what length of polyA tail is required for our anchor to have a melting temperature of 30 degrees Celcius (decided to change this melting temp later). This melting temp is sensitive to Na+ and Mg++ concentrations, so referred to ligase buffer and elution buffer for these concentrations.
The BsaI A end should look like this:<---diagram--->
Designed an anchor with a polyA10 tail with a primer (Tm=65.1 oC) region and the BsaI cut site with an A' end (ACCC).
May 26, 2010
Building Parts | Assembly Method
Building Parts
Transforming pSB1C3 Plasmid Backbone
Made 3 glycerol stocks of pSB1C3 from overnight made 25-05-2010 .
Assembly Method
Designed new anchors, ordered oligos. Designed new anchors with larger polyA tail (and without hairpins, or self dimerize-ness).
May 27, 2010
Building Parts | Assembly Method
Building Parts
Construct of Kan Resistance Bsa in pSB1C3 Backbone
Digested both A/B' and B/A' Kanamycin Resistance cassettes fragments from 11-05-2010 and pSB1C3 from 20-05-2010 with NotI at 37oC for 1 hour. Heat inactivated the NotI for 10 minutes at 65oC.  Ligated the Kanamycin Resistance cassettes into pSB1C3 at 16oC for 1 hour then took 15μL to room temperature for 2 hours.  Transformed 100μL of DH5α cells with 5μL of RT ligation reaction. Plated transformation on plates with both Chloramphenicol and Kanamycin.
Digestion Recipe:
1μL Miniprep (302.7ng/μl determined by nanodrop)
2μL either A/B' or B/A' Kanamycin resistance cassette (approx. 100ng/μL)
1μL NotI
1μL 10X ReACT 3
5μL MilliQ
Ligation Recipe:
10μL of Digest solution
1μL T4 DNA ligase
6μL 5X Buffer
13μL MilliQ H2O
Experiment continues on 28-05-2010 .
Transforming pSB4A5, pSB4C5 and pSB3T5 Plasmids
Transformed pSB4A5-J04450, pSB4C5-J04450 and pSB3T5-J04450 from the 2010 biobrick parts into DH5α cells.
Experiment continues on 28-05-2010 .
Constructing Amp, Chlor and Tet Resistance Inserts
Performed PCR reactions to create parts with antibiotic resistance with negative controls.
PCR Recipe:
3μL 10X PCR Buffer
1μL 10 uM dNTPs
2μL 50 uM MgCl2
17.5μL MilliQ H2O
0.5μL Taq Polymerase
1μL Template (psB4A5-J04450, psB4C5-J04450 or psB3T5-J04450)
2.5μL Primer + (PrA psB4A5 ApR+, PrA psB4C5 ChR+ or PrA psB3T5 TR+)
2.5μL Primer - (PrB psB4A5 ApR-, PrB psB4C5 ChR- or PrB psB3T5 TR-)
PCR Program:
5 min-94oC
45 sec-94oC
1 min-60oC
1 min-72oC
Repeat 2 through 4 35 times
5 min-72oC
<--gel images of PCR Products (Alina's) and ligated and pre-digested samples (Jeremy's)-->
Assembly Method
No binding capacity specified in the manual for the beads in terms of mols or mass of DNA or beads. Binding capacity is given in terms of cells (because the beads are meant for mRNA isolation from cells). Will have to quantify binding capacity later.
Derivation of mole matching equation: ssDNA is ~330 grams/(mol*bp).
moles = [concentration X volume] / [(330grams/mol X bp) X length]
moles1 = moles2
c1v1/[(330g/mol X bp) X l1] = c2v2/[(330g/mol X bp) X l2]
v1 = v2(c2 X l1/ c1 X l2)
Ordered new primers for anchor:
GCG CGC CCG GTC TCA TGG GTC ACC CTC CC
GGG AGG GTG ACC CAT GAG ACC GGC GCG C[A]12
May 28, 2010
Building Parts
Construct of Kan Resistance Bsa in pSB1C3 Backbone
We got colonies!! (it's a fantastic feeling) We then lovingly put them in the cold room to await our return from Calgary
site map | sponsors | contact us
Retrieved from " http://2010.igem.org/Team:Alberta/Notebook_May "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
