<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Alberta_biobyte2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Alberta/biobyte2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Alberta/biobyte2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/biobyte2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/biobyte2&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Alberta/biobyte2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Alberta/biobyte2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="top-strip"><H2>TEAM ALBERTA</H2></DIV><DIV id="navbar"><UL><LI><A href="https://2010.igem.org/Team:Alberta" class=" nav-home">HOME</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Tour/start" class=" nav-software">AT A GLANCE</A></LI><LI class="headlink"><A href="https://2010.igem.org/Team:Alberta/project" class="selected nav-project">PROJECT        </A></LI><LI><A href="https://2010.igem.org/Team:Alberta/project">Overview</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/biobyte2">BioBytes 2.0</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Software">Software</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/modelling">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Kit">The Kit</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Kit_components">Kit Components</A></LI></UL><LI class="headlink"><A href="https://2010.igem.org/Team:Alberta/Achievements/Overview" class=" nav-practices">ACHIEVEMENTS </A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Achievements/Overview">Overview</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Achievements">Medal Requirements </A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Achievements/Media">Media</A></LI><LI class="headlink"><A href="https://2010.igem.org/Team:Alberta/human_practices" class=" nav-practices">HUMAN PRACTICES</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/human_practices">Overview</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/human_practices/distribution_analysis">Distribution Analysis</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/human_practices/HighSchool">High School</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/human_practices/DiscoverElle">DiscoverElle Visit</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/human_practices/safety">Safety</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/parts" class=" nav-parts">PARTS</A></LI><LI class="headlink"><A href="https://2010.igem.org/Team:Alberta/Notebook" class=" nav-notebook">TIMELINE &amp; PROTOCOLS</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook">Overview</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/protocols">General Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/ReusablePlates">Reusable Plates</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/BasePlasmids">Base Plasmids</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/CreatingParts">Creating Parts</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/TransformingCells">Transforming Cells</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/Anchor">Anchor</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/Beads">Beads</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/Optimizations">Optimizations</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/Notebook/Software">Software</A></LI><LI><A href="https://2010.igem.org/Team:Alberta/team" class=" nav-team">TEAM</A></LI><UL><DIV id="left-sidebar" class=""><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Traditional_Methods"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Traditional Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Comparison"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Comparison</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Components_of_the_System"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Components of the System</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Iron_Micro_Beads"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Iron Micro Beads</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Anchor-Byte"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Anchor-Byte</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#BioBytes_2.0"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">BioBytes 2.0</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Cap"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Cap</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#BioBytes_2.0_Assembly_Process"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">BioBytes 2.0 Assembly Process</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Please_note_that_the_GENOMIKON_kit_comes_with_premade_anchor-Byte_constructs."><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Please note that the GENOMIKON kit comes with premade anchor-Byte constructs.</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#The_GENOMIKON_kit_supplies_plastic_micropipettes_than_can_dispense_25ul_per_drop."><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">The GENOMIKON kit supplies plastic micropipettes than can dispense 25ul per drop.</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Byte_Design"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Byte Design</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Byte_Construction"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Byte Construction</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#BioBytes_2.0_vs._BioBytes"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">BioBytes 2.0 vs. BioBytes</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV id="center-content" class=""><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>BioBytes 2.0 is the heart of the GENOMIKON kit.  BioBytes 2.0 is a method of creating novel plasmids through the sequential addition of functional units of DNA.  It is designed to be used in a high school setting but has potential applications in professional settings as well.  The assembly method we have created has some conceptual similarities to the original BioBytes Assembly System of the 2009 Alberta iGEM project. However, there are some striking differences between the two systems.
</P><H2><SPAN class="mw-headline" id="Traditional_Methods">Traditional Methods</SPAN></H2><P>The current assembly standard is the BioBrick method.  While the Registry of Parts and the assembly standard have allowed for effective construction of plasmids in a laboratory setting, it has numerous limitations prohibiting its use in high schools.  For example, common laboratory protocols such as transformations, ligations, and restriction digests require expensive materials and equipment not available to high schools. Another disadvantage is that it takes from days to weeks to assemble a complicated construct using this method. An experiment of such length far surpasses the average high school student’s attention span and time allotted in a curriculum.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:490px;"><DIV class="thumbcaption">Figure 1.1. Traditional BioBrick construction takes approximately 3 days to complete the addition of one BioBrick; from the digestion of the original constructs to transformation and confirmation.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Comparison">Comparison</SPAN></H4><P>The BioBytes 2.0 assembly method has provided a solution to these issues.  It is fast. The addition of one BioByte to a construct takes under 10 minutes. Therefore, creating a plasmid of desired specifications can happen in an afternoon, rather than the 3 or more days to create a plasmid through the traditional BioBrick method. It is accessible. The GENOMIKON kit is completely self-contained, requiring no additional equipment or reagents other than a hot plate and a beaker. This eliminates the need for the expensive equipment and reagents found in a traditional university laboratory.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:497px;"><DIV class="thumbcaption">BioByte version 2.0 construction.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Components_of_the_System">Components of the System</SPAN></H2><P>BioBytes 2.0 is composed of three main components.  The construct begins with an anchor-Byte piece which is attached to an iron micro bead.  Because of the magnetic nature of these beads, they can be positioned by using a simple magnet.  The BioBytes are added to the anchor-Byte one-at-a-time, in sequence.  This is possible due to the alternating overhang structure of the BioBytes.  Finally, a cap Byte is added allowing for circularization of the construct. No ligation is required for circularization.  The construct is now ready to transform.
</P><H4><SPAN class="mw-headline" id="Iron_Micro_Beads">Iron Micro Beads</SPAN></H4><P>Iron micro beads purchased from New England Biolabs have covalently attached poly-T tails. The bead allows us to orient the DNA with magnets, making washing and subsequent attachments easier.
</P><H4><SPAN class="mw-headline" id="Anchor-Byte">Anchor-Byte</SPAN></H4><P>The anchor is a part that attaches directly onto the iron micro beads. It is composed of:
</P><UL><LI>a poly-A tail
</LI><LI>a BsaI recognition site
</LI><LI>an A or B overhang
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:246px;"><DIV class="thumbcaption">The two components of the anchoring system.</DIV></DIV></DIV></DIV><P>We have designed several anchor-Byte constructs which can begin the process of assembly. An anchor-Byte is comprised of an anchor piece ligated to the first Byte of the assembly, usually a selectable marker. This allows for complete constructs to be selected for. As well, the incorporation of a BsaI cut site into the anchor, before the first Byte, gives versatility to the construct because the first Byte and the rest of the construct can be removed from the anchor, and used as a Byte on its own. This allows for multiplexing experiments.  
</P><P>Once the selectable marker is ligated to the first Byte, we anneal the anchor-Byte to the poly-T tails on the iron micro beads. We have created anchors with both AB and BA ends so that assemblies can begin with any type of Byte.
The results of one of our experiments is shown here. Note that the interaction between the anchor and the bead is non-covalent. The anchor along with the construct can be separated from the bead with heat. The gel shows the process of anchoring a construct. An anchor-Byte construct of 1kb was allowed to anneal to the magnetic beads. This was done in excess, and the supernatant is shown in the first lane. A subsequent wash step showed the absence of DNA, indicating that DNA construct is stably bound. The construct was melted from the scaffold at 75 degrees Celsius. The melted construct can be seen in the last lane.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure of the anchor system with the BioBytes 2.0 Assembly System.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="BioBytes_2.0">BioBytes 2.0</SPAN></H4><P>The BioBytes 2.0 assembly method has a number of unique features.  Each Byte has 4-base 5' overhangs on each side. These overhangs have been engineered to serve a number of purposes. The overhangs allow for ordered and sequential addition of parts to growing constructs.  
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">BioBytes come in two flavors: 'AB' and 'BA'.</DIV></DIV></DIV></DIV><P>The addition of each Byte to a growing construct takes about 7 minutes.
</P><H4><SPAN class="mw-headline" id="Cap">Cap</SPAN></H4><P>The cap Byte is analogous to the anchor.  The cap finishes the construct, where the anchor starts a construct.  The cap is comprised of a poly-T tail, complimentary to the poly-A tail of the anchor piece.  The cap also contains a BsaI cut site which allows its removal from the construct.  It can follow an AB or BA format. 
</P><P>When finishing a construct, the cap is ligated to the growing construct, in the same manner as another Byte.  The completed construct is then heated, to melt the anchor piece from the iron micro bead.  The solution is removed from the beads and allowed to cool.  The cap and anchor then have opportunity to anneal to each other.  The construct is now ready for transformation without ligation.
</P><H2><SPAN class="mw-headline" id="BioBytes_2.0_Assembly_Process">BioBytes 2.0 Assembly Process</SPAN></H2><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:478px;"><DIV class="thumbcaption">The entire construction process can be completed in an afternoon.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Please_note_that_the_GENOMIKON_kit_comes_with_premade_anchor-Byte_constructs.">Please note that the GENOMIKON kit comes with premade anchor-Byte constructs.</SPAN></H3><H3><SPAN class="mw-headline" id="The_GENOMIKON_kit_supplies_plastic_micropipettes_than_can_dispense_25ul_per_drop.">The GENOMIKON kit supplies plastic micropipettes than can dispense 25ul per drop.</SPAN></H3><DL><DD><OL><LI>	Mix the iron micro beads for 10 minutes.
</LI><LI>	Transfer a 25 ul (one drop) aliquot of iron micro beads to a 1.5 mL tube.
</LI><LI>	Pull the beads to the side using the magnetic tube rack supplied.
</LI><LI>	Remove and discard the supernatant.
</LI><LI>	Add 50 ul (2 drops) of supplied Wash Buffer to the beads. Flick gently to resuspend.
</LI><LI>	Pull the beads to the side using the magnetic tube rack. 
</LI><LI>	Remove and discard the supernatant. 
</LI><LI>	Repeat steps 5 to 7. 
</LI><LI>	Add 200 ng of a premade anchor-Byte construct to the beads and top off the volume to 25ul with the TE Buffer supplied. Flick gently to resuspend.
</LI><LI>	Allow annealing for 30 minutes, mixing by flicking every 5 minutes. Ensure that there are no droplets on the sides of the tube.
</LI><LI>	Repeat steps 6 and 7.
</LI><LI>	Repeat steps 5 to 7.
</LI><LI>	Add 200 ng of the next Byte of your construct, making sure that a BA Byte follows an AB Byte and vice versa. Add an appropriate amount of 2x QuickLigase Buffer, Quick Ligase and TE to a total volume of 25ul. 
</LI></OL></DD><DD>Example:
</DD></DL><TABLE style="margin-left:40px;"><TBODY><TR><TH> Reagent </TH><TH> Volume (ul)
</TH></TR><TR><TD> (40ng/ul) AB kan Byte </TD><TD> 5
</TD></TR><TR><TD> 2X Quick Ligase Buffer </TD><TD> 13
</TD></TR><TR><TD> TE Buffer </TD><TD> 6
</TD></TR><TR><TD> Quick Ligase </TD><TD> 1
</TD></TR><TR><TD> TOTAL </TD><TD> 25
</TD></TR></TBODY></TABLE><DL><DD>14.	Flick gently to resuspend.  Allow ligation for five minutes, flicking gently every minute.
</DD></DL><DL><DD>15.	Add 25 ul of Wash Buffer to the tube. Flick gently. 
</DD></DL><DL><DD>16.	Repeats steps 6 and 7.
</DD></DL><DL><DD>17.	Repeat steps 5 to 7 twice.
</DD></DL><DL><DD>18.	Repeat steps 13 to 17 for each subsequent Byte addition, including the last Byte. If the last Byte is cap, it must be added in 20 X molar excess.  
</DD></DL><DL><DD>19.	After the last Byte has been ligated and washed, addition of 25 ul of 75C Elution Buffer. The entire tube should be kept at 75C for ten minutes to allow the DNA construct to elute off the beads.
</DD></DL><DL><DD>20.	After ten minutes, put the tube into a magnetic rack in the 75C waterbath. Allow the beads to be pulled aside and collect the supernatant into a clean 1.5 mL tube. The supernatant will contain your construct.
</DD></DL><P>
Using this protocol, we were able to assemble an octamer in an afternoon!
</P><H2><SPAN class="mw-headline" id="Byte_Design">Byte Design</SPAN></H2><P>Because the overhangs in this year's project are only 4 bases long, and because the ends are malleable, they give additional versatility.  In this classification system, AB parts are &quot;ORF&quot; parts.  The &quot;ORF&quot; parts contain the coding sequence of genes.  However, these &quot;ORF&quot; parts do not contain a complete open reading frame. They lack a promoter, a RBS, a start codon, a stop codon and a terminator.  Therefore, an &quot;ORF&quot; part by itself has no function at all. We have coined this type of parts &quot;naked ORFs&quot;
The BA parts that can be ligated to the &quot;ORF&quot; parts will reconstitute these functions.  For this reason BA parts are now termed &quot;Linker&quot; Parts.  The &quot;Linkers&quot; can reconstitute the start and stop codons, the promoters and terminators, but they can also introduce, protein fusion linkers and protein tags.   This give this year's BioBytes 2.0 design added flexibility and potential for testing regulatory regions of DNA.  
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">The BioByte 2.0 ends are designed to be adaptable to protein linkers.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Byte_Construction">Byte Construction</SPAN></H2><P>New Biobytes are constructed with the aid of our Base Plasmids v.2.  The Base Plasmids v.2 have an RFP coding cassette (J04450) flanked by BsaI cut sites that leave the appropriate overhangs to create either an AB part or a BA part.  Because the BsaI restriction enzyme cuts at a site downstream of the recognition sequence, the overhang sequence can be engineered to be unique.  We have designed our own overhangs that define the AB and the BA parts.  
</P><P>
To create a new part PCR the part of interest, incorporating BsaI cut sites with appropriate overhangs onto each side. Then, cut the Base Plasmid v.2 and the PCR product with BsaI.  Ligate it together.    Again, because the overhangs are unique, the plasmid backbone can not re-ligate without an insert.  The insert can only be the new part of interest, or the RFP coding cassette that was originally in the base plasmid, leading to a total of 2 possible ligation products. When transformed, the ligation products can appear red (if the original RFP cassette is reinserted) or white (if the part of interest is inserted).  This provides a selection method for plasmids that have the part of interest in it.  
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:203px;"><DIV class="thumbcaption">Figure Base Plasmid v.2.  The Base plasmids can be cut leaving unique overhangs so that directional cloning of new parts can be inserted.  The lack of the RFP coding cassette provides a selection method for colonies containing new parts</DIV></DIV></DIV></DIV><P>Because all the parts are created in the same plasmid, mass production of parts for the kit is also a straight forward matter.  Using universal primers of our own design, which start about 100 base before the BsaI cut sites.  The PCR is then digested with BsaI.  Starting the PCR 100 bases outside the part allows for us to check that the part is completely digested.  Before parts are used in assembly, they are purified using a weak anion exchange column through HPLC (high performance liquid chromatography). 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:304px;"><DIV class="thumbcaption">Figure Part Mass Production.  New parts are PCRed, Digested with BsaI, then purified through an HPLC before being used in assembly</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="BioBytes_2.0_vs._BioBytes">BioBytes 2.0 vs. BioBytes</SPAN></H2><P>There are a few similarities between the original BioBytes 1.0 assembly method of Team Alberta 2009 and the new BioBytes 2.0 system that we have put forward this year.  However, if you look closely, there are a number of profound differences between the systems. 
</P><P>The similarities in the systems reside in the theory of alternating and complimentary ends.  The AB and BA parts can only ligate to each other, and AB parts can not ligate to themselves and BA parts can not ligate to themselves.  However, this year we are working with 4-basepair overhangs, not the 12-basepair overhangs used last year. Also, this year we are ligating they bytes together, whereas last year they were not ligated.  
</P><P>The iron micro beads with the poly-T tail are also unique to this year.  Last year, the constructs were anchored using a biotin and streptavidin process requiring an enzyme to cleave the construct.  This means that the release and circularizing of the constructs of each year are unique processes.  
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">The similarities and differences between the BioByte 2.0 and last year's BioByte 1.0</DIV></DIV></DIV></DIV><P>The &quot;BioBytes Version 2.0&quot; construction method has been shown to create plasmids from up to 8 separate parts in 
an afternoon's work. This is a vast improvement.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Alberta/biobyte2">http://2010.igem.org/Team:Alberta/biobyte2</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Alberta/biobyte2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Alberta/biobyte2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Alberta/biobyte2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Alberta/biobyte2&amp;oldid=207154" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></BODY></HTML>