"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Notebook/protocols/primer design
From 2010.igem.org
TEAM ALBERTA
HOME
AT A GLANCE
PROJECT
Overview
BioBytes 2.0
Software
Modelling
The Kit
Kit Components
ACHIEVEMENTS
Overview
Medal Requirements
Media
HUMAN PRACTICES
Overview
Distribution Analysis
High School
DiscoverElle Visit
Safety
PARTS
TIMELINE & PROTOCOLS
Overview
General Protocols
Reusable Plates
Base Plasmids
Creating Parts
Transforming Cells
Anchor
Beads
Optimizations
Software
TEAM
General Protocols
In Vitro BioByte Assembly
LB Plates and Broth
Transformations
5mL Overnight
Glycerol Stock
Plasmid Miniprep
Restriction Digest
Vector Dephosphorylation
Ligation
Agarose Gel Electrophoresis
Gel Extraction
PCR
Colony PCR
PCR Purification
Sample Labelling Conventions
Fluorescent Sequencing Reaction
Primer Design
Primer Design
Notes:
Typical limits:
delta Tm < 5 C, hairpin dG >= -2, dimer (hetero or homo) dG >= -6 to -15, but stricter at 3'.
Some buffer to allow end enzymes to cut is required, NEB and Fermentas have charts for how much, 6 bases is typically sufficient for anything.
Primer design limits: http://www.premierbiosoft.com/tech_notes/PCR_Primer_Design.html
primer 3' end hairpin -2 kcal/mol, -3kcal/mol elsewhere okay. self dimer: -5 kcal/mol (3'), -6 elsewhere heterodimer:
Hot start helps melt strands.
VNTI defaults:
Tm difference 5 C, hairpin -1 max, dimers -15 max, or -3 at 3'. all checks only within first 10 bp of 3'
Mike's criteria for PCR primer design:
Forward primer should be same as coding strand, 5' -> 3', reverse primer should be same as noncoding strand, so that forward primer complements noncoding strand and vice versa
End in GC or GG, so end sticks down well
~50% GC content, so that it will anneal and deanneal well under temp shifts
Should be 30-45 NTs long. 20-25 is too short
Restriction site should cut exactly where you want
Should be an overhang before restriction sites, so that restriction enzymes can do their work
Repeat motifs
Description & Effect
Simple repeats
A repeated sequence of four or more nucleotides, which is repeated: (...AATCGA...AATCGA...). Simple repeats can generate secondary binding sites for primers. Stable hybridization to secondary binding sites results in nonspecific amplification. Repeats >3-4.
Inverse repeats
A self-complementary sequence motif of four or more nucleotides (stem-loop or hairpin motifs): (...AATGGC...GCCATT...). Inverse repeats can cause inefficient priming because they lead to formation of stable hairpins in the binding region, or within the amplicon.
Homopolymeric runs
A sequence of four or more identical nucleotides: (...AAAAA...). Homopolymeric runs can be considered a special case of direct repeats. These can cause ambiguous binding of primers to their target site ("slippage effect"). Poly (A) and poly (T) stretches should also be avoided because these will "breathe" and open up stretches of the primer-template complex. Additionally, runs of three or more G residues can cause problems due to intermolecular stacking.
Procedure:
For Primer Design:
1. Obtain a licensed version of Vector NTI.   2. Download the E. coli K-12 MG16550 Genome from Gen Bank.
To do this, type NC_000913.2 into the open Gen Bank file option under the tools panel.
3. Obtain the following from the FTP.  It can be found in the Primer Design folder inside the Primers folder:
Wayne’s Protocol: The Primer ‘add ons’ containing the RE sites and 8bp extensions (download the correct set based on the instructions, which you receive with the gene list).
A list of the genes which require primers and their locations.
4. Type into the blank panel in the bottom right corner of the Vector NTI main window, the start and end locations of the gene of interest.
Look to ensure that there is an ATG at one side of the sequence and a stop codon (TAA, TGA, TAG) on the other
The orientation of the ATG and stop codon varies depending on the orientation of the gene (forward or reverse).
5. Determine the sequence numbers for the base pairs directly beside the start and stop codons (we do not need these as they are in the primers we are designing ourselves) and select the new sequence as in step 4.
A quick way of doing this is just to add 3 to the start bp numbers and subtract 3 from the last bp number.
6. With the sequence selected, click on the Primer Design tab and select “Amplify Selection…”. 7. In the lower left hand corner click on the load button and select Wayne’s Protocol which you downloaded from the FTP.  8. In the sense and anti-sense boxes type in the sequence information which you downloaded from the FTP.
Directionality is important! If the gene is encoded in a forward direction the sense sequence should contain the ATG site.
The anti-sense sequence should contain the TTA stop codon.
If the gene is encoded in the reverse direction the antisense sequence should contain the ATG start site.
IMPORTANT: Do not modify these sequences all you have to do is copy and paste them in the boxes no alterations are required!!!
9. Let the program run. Compare the resulting score of each primer sequence (found in the text box beside the length of the primer. Shoot for scores above 150. Write down this score in your spreadsheet).
Underneath the score, it shows the area that was amplified.  Ensure that it did amplify the section that you wanted.
10. Next check for dimers by right clicking on the sense primer’s sequence in the text box and selecting “thermodynamic properties”. 11. Click the Analyze box and finally click on “Dimers & Hairpin Loops”. 12. Two boxes will appear, the top is for dimers, and the bottom for hairpins. Scroll through each and look at the dG values for each one (you can see the next one by clicking “>>”). Write down the lowest dG scores for both. 13. Repeat steps 10 – 12 for the antisense primer.
It is important that the dG values for the dimers does not fall below -6 for any sequence or -5 if the dimer is in the 3’ end of the sequence. For hairpins, the dG value should not be below -1.
If it does fall below these values see step 14, if it does not then skip to step 15.
14. If the primers will not work go back into the primer design tab and click on “Design PCR Primers Inside Selection…”. Click the quality tab and lower the numbers in the boxes (from 9 to 7 and if it does not work again then to 5). Then repeat steps 9 – 13.
Remember to switch these numbers back or reload Wayne’s PCR Program before doing another set of primers.
15. If you cannot find any primers which actually fit into the sequences, pick the best possible primers you can produce. 16. Once you have found primers of interest write down the remaining information needed into the excel table.
Retrieved from " http://2010.igem.org/Team:Alberta/Notebook/protocols/primer_design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
