"
Page
Discussion
View source
History
teams
Log in
 
Team:Jilin China/NOTEBOOK
From 2014.igem.org
HOME
TEAM
OUR TEAM
ATTRIBUTION
PROJECT
OUTLINE
CONTRUST
TEST
BLUEPRINT
IDEA
BACKGROUND
DESIGN
PRACTICE
COORPERATION
OBSERVATION
PROPAGATE
SURVEY
ACKNOWLEDGEMENT
Welcome!
Team Jilin China
NOTEBOOK
Synthesis and Characterisation of the Mlr Promoter
Research background
A  novel pathway for degradation of the cyanobacterial heptapeptide hepatotoxin  microcystin LR was identified in a newly isolated Sphingomonas sp. (Bourne et al. 1996 Appl. Environ. Microbiol. 62:  4086–4094). And the gene cluster involved in bacterial degradation of the  cyanobacterial
toxin  microcystin LR were be reported by David G. Bourne in 2001. The cloning and  molecular characterisation of four genes from this Sphingomonas sp. that exist on a 5.8-kb genomic fragment and encode  the three hydrolytic enzymes involved in this pathway together with a putative  oligopeptide transporter. Situated immediately downstream of mlrA with  the same direction of transcription is a gene mlrD, whose conceptual  translation (MlrD, 442residues) shows significant sequence identity and similar  potential transmembrane spanning regions to the PTR2family of oligopeptide transporters. A gene mlrB is situated downstream of the mlrA and mlrD genes, but transcribed in the opposite direction. The gene encodes the  enzyme MlrB (402residues) which cleaves linear microcystin LR to a tetrapeptide  degradation product. This enzyme belongs to the “penicillin-binding  enzyme”  family of active site serine hydrolases. The final gene in the cluster mlrC,  is located upstream of the mlrA gene and is transcribed in the opposite  direction. It codes for MlrC (507 residues) which mediates further peptidolytic  degradation of the tetrapeptide. This protein shows significant sequence  identity to a hypothetical protein from Streptomyces coelicolor. It is  suspected to be a metallopeptidase based on inhibition by metal chelators. It  is postulated on the basis of comparison with other microorganisms that the  genes in this cluster may all be involved in cell wall peptidoglycan cycling  and subsequently act fortuitously in hydrolysis of microcystin LR.
In  this study, we  synthesized a RFP-promoter-GFP  sequence to detect the relationship between MLR promoter and microcystin LR.  This synthesized sequence using the promoter sequence between mlrA and mlrC, adding  the red fluorescent protein sequence in the upstream to instead mlrC gene, and adding  green fluorescent protein sequence in the downstream to instead mlrA gene.
Fig 1 Restriction  map of the 5.8 kb of sequence data and localisation of genes involved in  microcystin LR degradation.
methods
Synthetic  sequence
    BamHI
GCCTGCAGctataaaaataaatgatgtctaccttctgttctttcatattgttcaacaattgtataatcttcattatgtgatgtaatatctaacttagcatctacataataatatcctggtaattgaactggttttttagccatataaattgatttaaattcaactaaataatgtcctccgtcctttaattttagagctttatgtgtttctccctttaaaaccccgtctcttggatataatctttctgtactagcttcccatcccattgtttttttttgcataactggtccgtctgatggaaaattaaccccaataaatttaactttataaataaaacaaccatcttgtaatgatgaatcttgtgtaactgtagcaactccaccatcttcaaaattcataactctttcccatttaaatccttctggaaaagataattttttataatctggaatatcagctggatgtttaacgtaaacctttgatccatattgaaattgtggtgataaaatatcccacgcaaatggtaatggtcctccttttgtaacctttaatttaactgtattatgtccttcataCggtctaccttctccttctccctcaatttcaaattcatgtccattaactgtaccttccattctaactttaaatctcataaattctgtaataacattttctgatgaagccatAacacgagtcttcggtttgctgttgtttgcaccagctgctgcatcttcaccccataaatcaaccgaacggacagtcaattcttgttgaccatcctgggcctatctgaaatgttctgctgacagaacgggagaattgaaccatgagtaaaggagaagaacttttcactggagttgtcccaattcttgttgaattagatggtgatgttaatgggcacaaattttctgtcagtggagagggtgaaggtgatgcaacatacggaaaacttacccttaaatttatttgcactactggaaaactacctgttccatggccaacacttgtcactactttcggttatggtgttcaatgctttgcgagatacccagatcaCatgaaacagcatgactttttcaagagtgccatgcctgaaggttatgtacaggaaagaactatatttttcaaagatgacgggaactacaagacacgtgctgaagtcaagtttgaaggtgatacccttgttaatagaatcgagttaaaaggtattgattttaaagaagatggaaacattcttggacacaaattggaatacaactataactcacacaatgtatacatcatggcagacaaacaaaagaatggaatcaaagttaacttcaaaattagacacaacattgaagatggaagcgttcaactagcagaccattatcaacaaaatactccaattggcgatggccctgtccttttaccagacaaccattacctgtccacacaatctgccctttcgaaagatcccaacgaaaagagagaccacatggtccttcttgagtttgtaacagctgctgggattacacatggcatggatgaactatacaaataaGAATTCCG
EcoRI
Sequence analysis
 _Base Count : 1551 bp (489 A, 496 T, 314  C, 252 G)
 _Composition : 36% GC, 64% AT
Absent Sites
AarI AatII Acc65I AciI AclI AfeI AflII AgeI  AhdI AleI ApaI ApaLI AscI AseI AsiSI AvaI BamHI BanI BanII BbvCI BceAI BciVI  BclI BfuAI BglI BglII BlpI BmgBI BmrI BmtI Bpu10I BsaBI BsaHI BsaWI BseMII  BseRI BsgI BsiEI BsiHKAI BsiWI BsmI BsoBI BspCNI BspEI BspHI BspMI BspQI BsrBI  BsrDI BsrFI BssHII BstEII BstUI Bsu36I BtrI BtsI Cac8I ClaI DraIII EagI EarI  EciI Eco53KI EcoNI EcoO109I EcoRV FauI FseI FspAI FspI HaeII HgaI HhaI HinP1I  HindIII HpaII Hpy99I KasI KpnI MluI MmeI NaeI NarI NciI NdeI NgoMIV NheI NlaIV  NmeAIII NotI NruI NsiI NspI PacI PasI PciI PflFI PflMI PfoI PmeI PpuMI PshAI  PspOMI PspXI PstI PvuI RsrII SacI SacII SalI SanDI SbfI ScaI SexAI SfiI SfoI  SgrAI SmaI SnaBI SpeI SphI SrfI SspI StuI TauI TspMI Tth111I XbaI XcmI XhoI  XmaI XmnI ZraI
Unique Sites
Unique Sites
BamHI(3)
DdeI (92)
Esp3I (223)
SfcI (367)
BstXI (512) MslI (513)
BssSI (688) PleI (691)  BbsI (693)
MwoI (714)
BpmI (851)
BsmFI (859)
Bsp1286I (895)
BtgI (991) MscI (994)  Tsp45I (1006)
BsrGI (1101)
AflIII (1147) BsaAI  (1148)
BstZ17I (1273)
HpaI (1312)
BtgZI (1391)
AsuII (1446) BstYI (1452)  BsaI (1469) DrdI (1471)
SmlI (1486) BseYI (1504)
EcoRI (1543)
Building  Block Design
The maximum allowable assembly oligo length is  equal to the target assembly oligo length (60). This may cause some weird  behavior, especially in terms of overlap melting temperature.
2 building blocks were generated.
Building Block  mlrAC.1   789bp   1..789 
Left  - 5' GCCCTAGGCTATAAAAATAAATGATG  3'
Rght  - 5' ATAGGCCCAGGATGGTCAA  3'
RghtU - 5' ATAGGCCCAGGAUGGTCAA 3'
Sequence:
Assembly Oligos: average overlap Tm is 48°;average oligo length is 56bp.
GCCCTAGGCTATAAAAATAAATGATGTCTACCTTCTGTTCTTTCATATTGTTCAACAATTGTATAATCTTCATTATGTGATGTAATATCTAACTTAGCATCTACATAATAATATCCTGGTAATTGAACTGGTTTTTTAGCCATATAAATTGATTTAAATTCAACTAAATAATGTCCTCCGTCCTTTAATTTTAGAGCTTTATGTGTTTCTCCCTTTAAAACCCCGTCTCTTGGATATAATCTTTCTGTACTAGCTTCCCATCCCATTGTTTTTTTTTGCATAACTGGTCCGTCTGATGGAAAATTAACCCCAATAAATTTAACTTTATAAATAAAACAACCATCTTGTAATGATGAATCTTGTGTAACTGTAGCAACTCCACCATCTTCAAAATTCATAACTCTTTCCCATTTAAATCCTTCTGGAAAAGATAATTTTTTATAATCTGGAATATCAGCTGGATGTTTAACGTAAACCTTTGATCCATATTGAAATTGTGGTGATAAAATATCCCACGCAAATGGTAATGGTCCTCCTTTTGTAACCTTTAATTTAACTGTATTATGTCCTTCATACGGTCTACCTTCTCCTTCTCCCTCAATTTCAAATTCATGTCCATTAACTGTACCTTCCATTCTAACTTTAAATCTCATAAATTCTGTAATAACATTTTCTGATGAAGCCATAACACGAGTCTTCGGTTTGCTGTTGTTTGCACCAGCTGCTGCATCTTCACCCCATAAATCAACCGAACGGACAGTCAATTCTTGTTGACCATCCTGGGCCTAT
GCCCTAGGCTATAAAAATAAATGATGTCTACCTTCTGTTCTTTCATATTGTTCAACAAT      ATCTTCATTATGTGATGTAATATCTAACTTAGCATCTACATAATAATATCCTGGTAAT      GGTTTTTTAGCCATATAAATTGATTTAAATTCAACTAAATAATGTCCTCCGTCC          GAGCTTTATGTGTTTCTCCCTTTAAAACCCCGTCTCTTGGATATAATCTTTCTGTAC       CCATCCCATTGTTTTTTTTTGCATAACTGGTCCGTCTGATGGAAAATTAACCC           ACTTTATAAATAAAACAACCATCTTGTAATGATGAATCTTGTGTAACTGTAGCA          CTTCAAAATTCATAACTCTTTCCCATTTAAATCCTTCTGGAAAAGATAATTTTTTATAAT    AATATCAGCTGGATGTTTAACGTAAACCTTTGATCCATATTGAAATTGTGGTGATAAA      CACGCAAATGGTAATGGTCCTCCTTTTGTAACCTTTAATTTAACTGTATTATGTCCTTC     GTCTACCTTCTCCTTCTCCCTCAATTTCAAATTCATGTCCATTAACTGTACCTT          TTTAAATCTCATAAATTCTGTAATAACATTTTCTGATGAAGCCATAACACGA            CTGTTGTTTGCACCAGCTGCTGCATCTTCACCCCATAAATCAACCGAACGGA                                
                                 AGACAAGAAAGTATAACAAGTTGTTAACATATTAGAAGTAATACACTACATTATAGATTG    GTAGATGTATTATTATAGGACCATTAACTTGACCAAAAAATCGGTATATTTAACTAAATT    TTGATTTATTACAGGAGGCAGGAAATTAAAATCTCGAAATACACAAAGAGGG            AGAGAACCTATATTAGAAAGACATGATCGAAGGGTAGGGTAACAAAAAAAAACG          GCAGACTACCTTTTAATTGGGGTTATTTAAATTGAAATATTTATTTTGTTGGTAGAACAT    ACTTAGAACACATTGACATCGTTGAGGTGGTAGAAGTTTTAAGTATTGAGAAAGGG        GGAAGACCTTTTCTATTAAAAAATATTAGACCTTATAGTCGACCTACAAATTGC          TAGGTATAACTTTAACACCACTATTTTATAGGGTGCGTTTACCATTACCAGG            GAAATTAAATTGACATAATACAGGAAGTATGCCAGATGGAAGAGGAAGAGGG            AGTACAGGTAATTGACATGGAAGGTAAGATTGAAATTTAGAGTATTTAAGACATTATTGT    GACTACTTCGGTATTGTGCTCAGAAGCCAAACGACAACAAACGTGGTCGACG            GGTATTTAGTTGGCTTGCCTGTCAGTTAAGAACAACTGGTAGGACCCGGATA
CGGGATCCGATATTTTTATTTACTACAGATGGAAGACAAGAAAGTATAACAAGTTGTTAACATATTAGAAGTAATACACTACATTATAGATTGAATCGTAGAT
CGGGATCCGATATTTTTATTTACTACAGATGGAAGACAAGAAAGTATAACAAGTTGTTAACATATTAGAAGTAATACACTACATTATAGATTGAATCGTAGATGTATTATTATAGGACCATTAACTTGACCAAAAAATCGGTATATTTAACTAAATTTAAGTTGATTTATTACAGGAGGCAGGAAATTAAAATCTCGAAATACACAAAGAGGGAAATTTTGGGGCAGAGAACCTATATTAGAAAGACATGATCGAAGGGTAGGGTAACAAAAAAAAACGTATTGACCAGGCAGACTACCTTTTAATTGGGGTTATTTAAATTGAAATATTTATTTTGTTGGTAGAACATTACTACTTAGAACACATTGACATCGTTGAGGTGGTAGAAGTTTTAAGTATTGAGAAAGGGTAAATTTAGGAAGACCTTTTCTATTAAAAAATATTAGACCTTATAGTCGACCTACAAATTGCATTTGGAAACTAGGTATAACTTTAACACCACTATTTTATAGGGTGCGTTTACCATTACCAGGAGGAAAACATTGGAAATTAAATTGACATAATACAGGAAGTATGCCAGATGGAAGAGGAAGAGGGAGTTAAAGTTTAAGTACAGGTAATTGACATGGAAGGTAAGATTGAAATTTAGAGTATTTAAGACATTATTGTAAAAGACTACTTCGGTATTGTGCTCAGAAGCCAAACGACAACAAACGTGGTCGACGACGTAGAAGTGGGGTATTTAGTTGGCTTGCCTGTCAGTTAAGAACAACTGGTAGGACCCGGATA    .
Building Block mlrAC.2   775bp   777..1551 
Left  - 5' ATCCTGGGCCTATCTGAAATG  3'
LeftU - 5' ATCCTGGGCCTAUCTGAAATG 3'
Rght  - 5'  CGGAATTCTTATTTGTATAGTTCATCC 3'
Sequence:
Assembly Oligos: average overlap Tm is 49°;average oligo length is 53bp.
ATCCTGGGCCTATCTGAAATGTTCTGCTGACAGAACGGGAGAATTGAACCATGAGTAAAGGAGAAGAACTTTTCACTGGAGTTGTCCCAATTCTTGTTGAATTAGATGGTGATGTTAATGGGCACAAATTTTCTGTCAGTGGAGAGGGTGAAGGTGATGCAACATACGGAAAACTTACCCTTAAATTTATTTGCACTACTGGAAAACTACCTGTTCCATGGCCAACACTTGTCACTACTTTCGGTTATGGTGTTCAATGCTTTGCGAGATACCCAGATCACATGAAACAGCATGACTTTTTCAAGAGTGCCATGCCTGAAGGTTATGTACAGGAAAGAACTATATTTTTCAAAGATGACGGGAACTACAAGACACGTGCTGAAGTCAAGTTTGAAGGTGATACCCTTGTTAATAGAATCGAGTTAAAAGGTATTGATTTTAAAGAAGATGGAAACATTCTTGGACACAAATTGGAATACAACTATAACTCACACAATGTATACATCATGGCAGACAAACAAAAGAATGGAATCAAAGTTAACTTCAAAATTAGACACAACATTGAAGATGGAAGCGTTCAACTAGCAGACCATTATCAACAAAATACTCCAATTGGCGATGGCCCTGTCCTTTTACCAGACAACCATTACCTGTCCACACAATCTGCCCTTTCGAAAGATCCCAACGAAAAGAGAGACCACATGGTCCTTCTTGAGTTTGTAACAGCTGCTGGGATTACACATGGCATGGATGAACTATACAAATAAGAATTCCG
ATCCTGGGCCTATCTGAAATGTTCTGCTGACAGAACGGGAGAATTGAACCAT            AGAACTTTTCACTGGAGTTGTCCCAATTCTTGTTGAATTAGATGGTGATGTTAATGG       TTTTCTGTCAGTGGAGAGGGTGAAGGTGATGCAACATACGGAAAACTTACCCTT          GCACTACTGGAAAACTACCTGTTCCATGGCCAACACTTGTCACTACTTTCGG            ATGCTTTGCGAGATACCCAGATCACATGAAACAGCATGACTTTTTCAAGAGTGC          GGTTATGTACAGGAAAGAACTATATTTTTCAAAGATGACGGGAACTACAAGA           GTCAAGTTTGAAGGTGATACCCTTGTTAATAGAATCGAGTTAAAAGGTATTGATTTT     AGATGGAAACATTCTTGGACACAAATTGGAATACAACTATAACTCACACAATGTATA     TGGCAGACAAACAAAAGAATGGAATCAAAGTTAACTTCAAAATTAGACACAACATTG     GGAAGCGTTCAACTAGCAGACCATTATCAACAAAATACTCCAATTGGCGATGG         TTTACCAGACAACCATTACCTGTCCACACAATCTGCCCTTTCGAAAGATCC           GAGACCACATGGTCCTTCTTGAGTTTGTAACAGCTGCTGGGATTACACATG                               
                                CTTGCCCTCTTAACTTGGTACTCATTTCCTCTTCTTGAAAAGTGACCTCAACAG          AACTTAATCTACCACTACAATTACCCGTGTTTAAAAGACAGTCACCTCTCCC            TTGTATGCCTTTTGAATGGGAATTTAAATAAACGTGATGACCTTTTGATGGA            TGTGAACAGTGATGAAAGCCAATACCACAAGTTACGAAACGCTCTATGGGTC            TCGTACTGAAAAAGTTCTCACGGTACGGACTTCCAATACATGTCCTTTCTTGAT          TCTACTGCCCTTGATGTTCTGTGCACGACTTCAGTTCAAACTTCCACTATGG          CTTAGCTCAATTTTCCATAACTAAAATTTCTTCTACCTTTGTAAGAACCTGTG         ATGTTGATATTGAGTGTGTTACATATGTAGTACCGTCTGTTTGTTTTCTTACC         ATTGAAGTTTTAATCTGTGTTGTAACTTCTACCTTCGCAAGTTGATCGTCT           TTTTATGAGGTTAACCGCTACCGGGACAGGAAAATGGTCTGTTGGTAATGGAC         AGACGGGAAAGCTTTCTAGGGTTGCTTTTCTCTCTGGTGTACCAGGAAGAA           TCGACGACCCTAATGTGTACCGTACCTACTTGATATGTTTATTCTTAAGGC
TAGGACCCGGATAGACTTTACAAGACGACTGTCTTGCCCTCTTAACTTGGTACTCATTTCCTCTTCTTGAAAAGTGACCTCAACAGGGTTAAGAACAACTTAATCTACCACTACAATTACCCGTGTTTAAAAGACAGTCACCTCTCCCACTTCCACTACGTTGTATGCCTTTTGAATGGGAATTTAAATAAACGTGATGACCTTTTGATGGACAAGGTACCGGTTGTGAACAGTGATGAAAGCCAATACCACAAGTTACGAAACGCTCTATGGGTCTAGTGTACTTTGTCGTACTGAAAAAGTTCTCACGGTACGGACTTCCAATACATGTCCTTTCTTGATATAAAAAGTTTCTACTGCCCTTGATGTTCTGTGCACGACTTCAGTTCAAACTTCCACTATGGGAACAATTATCTTAGCTCAATTTTCCATAACTAAAATTTCTTCTACCTTTGTAAGAACCTGTGTTTAACCTTATGTTGATATTGAGTGTGTTACATATGTAGTACCGTCTGTTTGTTTTCTTACCTTAGTTTCAATTGAAGTTTTAATCTGTGTTGTAACTTCTACCTTCGCAAGTTGATCGTCTGGTAATAGTTGTTTTATGAGGTTAACCGCTACCGGGACAGGAAAATGGTCTGTTGGTAATGGACAGGTGTGTTAGACGGGAAAGCTTTCTAGGGTTGCTTTTCTCTCTGGTGTACCAGGAAGAACTCAAACATTGTCGACGACCCTAATGTGTACCGTACCTACTTGATATGTTTATTCTTAAGGC 
Primer synthesis
Primer synthesis order o Comate Bioscience Co.,Ltd.
Num
Sequence(5' to 3')
Base Count（bp）
Total Synthesis(OD)
Subpackage Pipe Counts
Purification Mode
AC101
GCCCTAGGCTATAAAAATAAATGATGTCTACCTTCTGTTCTTTCATATTGTTCAACAAT
59
1
1
PAGE
AC102
GTTAGATATTACATCACATAATGAAGATTATACAATTGTTGAACAATATGAAAGAACAGA
60
1
1
PAGE
AC103
ATCTTCATTATGTGATGTAATATCTAACTTAGCATCTACATAATAATATCCTGGTAAT
58
1
1
PAGE
AC104
TTAAATCAATTTATATGGCTAAAAAACCAGTTCAATTACCAGGATATTATTATGTAGATG
60
1
1
PAGE
AC105
GGTTTTTTAGCCATATAAATTGATTTAAATTCAACTAAATAATGTCCTCCGTCC
54
1
1
PAGE
AC106
GGGAGAAACACATAAAGCTCTAAAATTAAAGGACGGAGGACATTATTTAGTT
52
1
1
PAGE
AC107
GAGCTTTATGTGTTTCTCCCTTTAAAACCCCGTCTCTTGGATATAATCTTTCTGTAC
57
1
1
PAGE
AC108
GCAAAAAAAAACAATGGGATGGGAAGCTAGTACAGAAAGATTATATCCAAGAGA
54
1
1
PAGE
AC109
CCATCCCATTGTTTTTTTTTGCATAACTGGTCCGTCTGATGGAAAATTAACCC
53
1
1
PAGE
AC110
TACAAGATGGTTGTTTTATTTATAAAGTTAAATTTATTGGGGTTAATTTTCCATCAGACG
60
1
1
PAGE
AC111
ACTTTATAAATAAAACAACCATCTTGTAATGATGAATCTTGTGTAACTGTAGCA
54
1
1
PAGE
AC112
GGGAAAGAGTTATGAATTTTGAAGATGGTGGAGTTGCTACAGTTACACAAGATTCA
56
1
1
PAGE
AC113
CTTCAAAATTCATAACTCTTTCCCATTTAAATCCTTCTGGAAAAGATAATTTTTTATAAT
60
1
1
PAGE
AC114
CGTTAAACATCCAGCTGATATTCCAGATTATAAAAAATTATCTTTTCCAGAAGG
54
1
1
PAGE
AC115
AATATCAGCTGGATGTTTAACGTAAACCTTTGATCCATATTGAAATTGTGGTGATAAA
58
1
1
PAGE
AC116
GGACCATTACCATTTGCGTGGGATATTTTATCACCACAATTTCAATATGGAT
52
1
1
PAGE
AC117
CACGCAAATGGTAATGGTCCTCCTTTTGTAACCTTTAATTTAACTGTATTATGTCCTTC
59
1
1
PAGE
AC118
GGGAGAAGGAGAAGGTAGACCGTATGAAGGACATAATACAGTTAAATTAAAG
52
1
1
PAGE
AC119
GTCTACCTTCTCCTTCTCCCTCAATTTCAAATTCATGTCCATTAACTGTACCTT
54
1
1
PAGE
AC120
TGTTATTACAGAATTTATGAGATTTAAAGTTAGAATGGAAGGTACAGTTAATGGACATGA
60
1
1
PAGE
AC121
TTTAAATCTCATAAATTCTGTAATAACATTTTCTGATGAAGCCATAACACGA
52
1
1
PAGE
AC122
GCAGCTGGTGCAAACAACAGCAAACCGAAGACTCGTGTTATGGCTTCATCAG
52
1
1
PAGE
AC123
CTGTTGTTTGCACCAGCTGCTGCATCTTCACCCCATAAATCAACCGAACGGA
52
1
1
PAGE
AC124
ATAGGCCCAGGATGGTCAACAAGAATTGACTGTCCGTTCGGTTGATTTATGG
52
1
1
PAGE
AC201
ATCCTGGGCCTATCTGAAATGTTCTGCTGACAGAACGGGAGAATTGAACCAT
52
1
1
PAGE
AC202
GACAACTCCAGTGAAAAGTTCTTCTCCTTTACTCATGGTTCAATTCTCCCGTTC
54
1
1
PAGE
AC203
AGAACTTTTCACTGGAGTTGTCCCAATTCTTGTTGAATTAGATGGTGATGTTAATGG
57
1
1
PAGE
AC204
CCCTCTCCACTGACAGAAAATTTGTGCCCATTAACATCACCATCTAATTCAA
52
1
1
PAGE
AC205
TTTTCTGTCAGTGGAGAGGGTGAAGGTGATGCAACATACGGAAAACTTACCCTT
54
1
1
PAGE
AC206
AGGTAGTTTTCCAGTAGTGCAAATAAATTTAAGGGTAAGTTTTCCGTATGTT
52
1
1
PAGE
AC207
GCACTACTGGAAAACTACCTGTTCCATGGCCAACACTTGTCACTACTTTCGG
52
1
1
PAGE
AC208
CTGGGTATCTCGCAAAGCATTGAACACCATAACCGAAAGTAGTGACAAGTGT
52
1
1
PAGE
AC209
ATGCTTTGCGAGATACCCAGATCACATGAAACAGCATGACTTTTTCAAGAGTGC
54
1
1
PAGE
AC210
TAGTTCTTTCCTGTACATAACCTTCAGGCATGGCACTCTTGAAAAAGTCATGCT
54
1
1
PAGE
AC211
GGTTATGTACAGGAAAGAACTATATTTTTCAAAGATGACGGGAACTACAAGA
52
1
1
PAGE
AC212
GGTATCACCTTCAAACTTGACTTCAGCACGTGTCTTGTAGTTCCCGTCATCT
52
1
1
PAGE
AC213
GTCAAGTTTGAAGGTGATACCCTTGTTAATAGAATCGAGTTAAAAGGTATTGATTTT
57
1
1
PAGE
AC214
GTGTCCAAGAATGTTTCCATCTTCTTTAAAATCAATACCTTTTAACTCGATTC
53
1
1
PAGE
AC215
AGATGGAAACATTCTTGGACACAAATTGGAATACAACTATAACTCACACAATGTATA
57
1
1
PAGE
AC216
CCATTCTTTTGTTTGTCTGCCATGATGTATACATTGTGTGAGTTATAGTTGTA
53
1
1
PAGE
AC217
TGGCAGACAAACAAAAGAATGGAATCAAAGTTAACTTCAAAATTAGACACAACATTG
57
1
1
PAGE
AC218
TCTGCTAGTTGAACGCTTCCATCTTCAATGTTGTGTCTAATTTTGAAGTTA
51
1
1
PAGE
AC219
GGAAGCGTTCAACTAGCAGACCATTATCAACAAAATACTCCAATTGGCGATGG
53
1
1
PAGE
AC220
CAGGTAATGGTTGTCTGGTAAAAGGACAGGGCCATCGCCAATTGGAGTATTTT
53
1
1
PAGE
AC221
TTTACCAGACAACCATTACCTGTCCACACAATCTGCCCTTTCGAAAGATCC
51
1
1
PAGE
AC222
AAGAAGGACCATGTGGTCTCTCTTTTCGTTGGGATCTTTCGAAAGGGCAGA
51
1
1
PAGE
AC223
GAGACCACATGGTCCTTCTTGAGTTTGTAACAGCTGCTGGGATTACACATG
51
1
1
PAGE
AC224
CGGAATTCTTATTTGTATAGTTCATCCATGCCATGTGTAATCCCAGCAGCT
51
1
1
PAGE
AC1
GCCCTAGGCTATAAAAATAAATGATG
26
1
1
PAGE
AC2
ATAGGCCCAGGATGGTCAA
19
1
1
PAGE
AC3
ATCCTGGGCCTATCTGAAATG
21
1
1
PAGE
AC4
CGGAATTCTTATTTGTATAGTTCATCC
27
1
1
PAGE
The experimental procedure and  analysis
1、Dissolving  of primers
Sterile deionized water whose volume is  determined by primer’s molecular weight is used to dilute the solution of the  primer to 10um/L. And it is prepared for the following experiment.
2、Mixing  of primers
Oligonucleotides synthesized in powder  form need to be centrifuged for 1 min at 10000rpm to assemble powders to the  bottom of the EP tube.And be careful to open the cap.
Make sure that the synthesis report is  consistent to the OD number on the primer label,and check the number of the  dispensing tube before the dissolution of oligonucleotides.The volume of water  added to make 10umol/L oligonucleotides’ solution is calculated according to the  synthetic single report of each primer. Add sterile deionized water and keep it  at room temperature for 2min.And reverse the tube to accelerate the progress of  solubilization. Finally, collect substances at the bottom of the tube after  centrifugation.
Every six primers are combined to one  group.Add 4ulbeginning fiprimer, 1ulintermediate primer and 4ul end primerto a  new EP tube, then add sterile deionized water to 20ul, mix them up as spare.
组别
引物编号
吸取量
灭菌去离子水添加量
总体积
每PCR（50ul体系）吸取量
终浓度
A1
AC101
4
8
20
5ul
200nM
AC102
1
50nM
AC103
1
50nM
AC104
1
50nM
AC105
1
50nM
AC106
4
200nM
A2
AC107
4
8
20
5ul
200nM
AC108
1
50nM
AC109
1
50nM
AC110
1
50nM
AC111
1
50nM
AC112
4
200nM
A3
AC113
4
8
20
5ul
200nM
AC114
1
50nM
AC115
1
50nM
AC116
1
50nM
AC117
1
50nM
AC118
4
200nM
A4
AC119
4
8
20
5ul
200nM
AC120
1
50nM
AC121
1
50nM
AC122
1
50nM
AC123
1
50nM
AC124
4
200nM
A5
AC201
4
8
20
5ul
200nM
AC202
1
50nM
AC203
1
50nM
AC204
1
50nM
AC205
1
50nM
AC206
4
200nM
A6
AC207
4
8
20
5ul
200nM
AC208
1
50nM
AC209
1
50nM
AC210
1
50nM
AC211
1
50nM
AC212
4
200nM
A7
AC213
4
8
20
5ul
200nM
AC214
1
50nM
AC215
1
50nM
AC216
1
50nM
AC217
1
50nM
AC218
4
200nM
A8
AC219
4
8
20
5ul
200nM
AC220
1
50nM
AC221
1
50nM
AC222
1
50nM
AC223
1
50nM
AC224
4
200nM
DA-PCR
Every  six primers which are single-stranded oligonucleotidesare chemically  synthesized were combined to one group. Then they were going to be made as  longer double-stranded fragments of intermediate (Block) by DA-PCR.
Procedure of DA-PCR：
Mixed primer solutions                   5μl
Pfu DNA Polymerase  2.5U              0.5μl    
dNTP                                4μl
10×Pfu buffer（Mg2+）               5μl
ddH20add to 50μl
Procedure of DA-PCR：
94℃         2min
94℃         30S
50℃         30S     25 cycles
72℃         1min
72℃         10min
4℃preservation
End
DA-PCR splicing reaction product was  detected in 2% agarose gel electrophoresis,
DA-PCR product              5μl
10×Loading Buffer         0.6μl    spotting after mixed
100bp Marker              5μl    spotting directly
75V electrophoress for 1h。
4、OE-PCR
    Take the 5 double strand intermediate  fragment (Block) spliced by DA-PCR as templates, blocks were further connected  by OR-PCR, OE-PCR in a reaction system consisting of:
Block1                       1μl
Block2                       1μl
Block3                       1μl
Block4                       1μl
Block5                       1μl
Block6                       1μl
Block7                       1μl
Block8                       1μl
AC1                         1μl
AC4                         1μl
Pfu DNA Polymerase 2.5U       0.5μl
dNTP                        4μl
10×Pfu buffer（Mg2+）         5μl
Sterilized  ultrapure water   to 50ul
Procedure of OE-PCR：
94℃         2min
94℃         30S
50℃         30S    20 cycles
72℃        2min
72℃       10min
4℃        preservation
End
The full-length gene  spliced by OE-PCR was detected by a 1% agarose gel electrophoresis, specific  programs are as follows
OE-PCR  product           5μl
10×Loading Buffer        0.6μl    spotted after mixing well
100bp  Marker              5μl   spotted directly
80V electrophoresis for  1h。
5、Constructing  and sequencing of subcloning vector
We design a primer AC-B with Blp I site  and a primer AC-S with SgrA I site. Then we use RFP-promoter-GFP as template in  PCR. Then we use Blp I and SgrA I enzyme to digest the target fragment and  pET32a vector(37℃,3h) and link it overnight(16℃). After this step, transform the  linked vector to competent cell(E.coli BL21) and select the desirable colony by  using “Blue-White Selection” method.
Constitute of BlpⅠand SgrAⅠdouble  enzyme digestion reaction system:
Gene or plasmid                   10μl
BlpⅠ              0.3μl
SgrAⅠ0.3μl
10×NEBuffer 2.12μl
Sterilizing  ultrapure watertop up to20μl
Constitute of linking reaction system：
RFP-promoter-GFP                    10μl
pET32a                              3μl
T4  ligase                            0.5μl
10×T4  ligase Buffer                   2μl
Sterilizing  ultrapure water           top up to 20μl
Preparation process of competent cell E.coliBL21：
(1) Culturing strain E.coliBL21(37℃)overnight  to recovery culture, Adding overnight bacteria culture fluid 600uLto 30 mL LB  culture medium, and 37℃，200  r/min shake culturing 1-2 h, until OD600 is around 0.3-0.4.
(2) In the aseptic  condition, transfer 30mLbacteria fluid to the centrifugal tube in the ice 10  min to cooling the culture to 0℃.
(3)  Centrifuging 10 min in precooling centrifugal machine over 4000r/min, throw  supernate, and then add30mLprecooling 0.1mol/L CaCl2 solution to  resuspension precipitation in the ice 10 min.
(4) Centrifuging 4000r/min  in 4℃10min, throw  supernate,  each 30mL initial culture  resuspension cell precipitation with 1.0mL precooling 0.1mol/LCaCl2  solution, and then 200μL each one.
Heal  shock transformation process：
(1) Thawing freshly  prepared or -80℃ competentcellBL21suspension.
(2) Add constructed  recombinant cloning vector plasmidspSB1C3-A(no more than 50ng, 10μl), shake  gently, and then in the ice 30min.
(3)  Put EP tube in 42℃ water bath90s，and  then transfer it to ice water bath to cool cells1-2 min.
(4) add 800μL LB medium  to EP tube，transfer  it to 37℃  table，150r/min，to  recovery strain 45 min.
Spread plate and  Culture
(1)After culture in 37℃,  centrifuge the cell solution(5000rpm,5min) and concentrate it to 200μl.  Then coat it to LB culture with chloramphenicol and set a positive control.
(2) Wait until the  solution in culture is full absorbed by cell, then invert the plate and culture  it in 37℃  overnight.
Sequence test
On the pET32a vector(as shown in Fig 1), use the  RFP-promoter-GFP gene (1535bp) to replace the fragment between BlpⅠ site and SgrA I site (753bp) and then we get  the reconstructed plasmid, pET-mlr promoter.
Fig 3 Map of pET-mlr  promoter
Select the white colony on culture and culture it. Then we extract the plasmid and use it as template in PCR to identify the colony. In theory, the reconstructed vector with RFP-promoter-GFP fragment is 1728bp and the empty vector without mlrA fragment is 946bp. So, the result shows that the white colony we select has reconstructed vector with mlrA fragment in it.
Figure 3  PCR identification of recombinant vector pET-mlr promoter
Illustration of  the outcome：
1、L1,  L2 could be amplified to 1700bp products, the recombinant vector contained a  foreign gene RFP-promoter-GFP according to the result of the identification.
2、M  was 1000bp Ladder Marker
The positive clones which were proved to  contain the recombinant vectors pET-mlr promoter by PCR identification. were  sent to 吉林库美生物科技限公司to  be sequenced. The results showed (Figure 4), RFP-promoter-GFP sequence is  consistent with the design sequence.
Figure 4  Sequencing  results with the design sequence alignment
The  relationship detection between MLR promoter and microcystin LR
Single colony  containingthe recombinant vector pET-mlr promoterwas picked ,vigorously shaken  culture for 4-5h in LB liquid medium at 37 ℃，the  culturing stopped until OD600 reached 0.3 - 0.4, 10-fold dilution to 10-5-10-7with sterilized  water, taking 100ul to coat on the LB plate containing 10ug / L microcystin LR,  37 ℃  for overnight.
Single cultured colony was placed in the  UV lamp darkroom, we observed colony color and found that it luminesced green  light, indicating that when there is microcystin LR, mlr promoter can promote  the expression of GFP, which makes the single colony of E.coli BL21 luminescing  green fluorescence in the UV lamp.
Top
Retrieved from " http://2014.igem.org/Team:Jilin_China/NOTEBOOK "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
