"
Page
Discussion
View source
History
teams
Log in
 
Team:Jilin China/RESULT
From 2014.igem.org
HOME
TEAM
OUR TEAM
ATTRIBUTION
PROJECT
OUTLINE
CONTRUST
TEST
BLUEPRINT
IDEA
BACKGROUND
DESIGN
PRACTICE
COORPERATION
OBSERVATION
PROPAGATE
SURVEY
ACKNOWLEDGEMENT
Welcome!
Team Jilin China
NOTEBOOK
Synthesis of gene mlrA
The experimental scheme
1、The amino acid sequence of the protein
MlrA GenBank: AF411068 SOURCE  Sphingomonas sp. ACM-3962
MREFVRQRPLLCFYVLAILIALAAHALRAMSPTPLDPMFKMLQETHAHLNIITAVRSTFEYPGAYTLLLFPAAPMFAALIATGIGYGQAGFRELLSRCAPWRSPVSWRQGVTVIAVCFLAFFALTGIMWVQTYLYAPPGTLDRTFLRYGSDPVAIYVMLAASLLLSPGPLLEELGWRGFALPQLLKKFDPLTAAVILGIMWWAWHLPRDLPTLFSGAPGAAWSVIVKQLVITPGFIASTIIAVFVCNKLGGSMWGGVLTHAIHNELGVNVTAEWAPTVAGLGWRPWDLIEFAVAIGLVLICGRSLGAASPDNARLAWGNVPPKLPGGVGDKSGANA
2、The selection of codons
Lactococcus lactis subsp. cremoris SK11 [gbbct]: 2504 CDS's (696252 codons)
http://www.kazusa.or.jp/codon/cgi-bin/showcodon.cgi?species=272622
fields: [triplet] [frequency: per thousand] ([number])
UUU 36.1( 25117)  UCU 16.5( 11473)  UAU 28.4( 19782)  UGU  3.9(  2729)
UUC 12.0(  8326)  UCC  3.3(  2266)  UAC  8.1(  5665)  UGC  1.1(   771)
UUA 31.0( 21599)  UCA 21.1( 14698)  UAA  2.5(  1708)  UGA  0.7(   498)
UUG 21.2( 14782)  UCG  3.9(  2685)  UAG  0.4(   298)  UGG 10.1(  7000)
 
CUU 25.2( 17574)  CCU 11.9(  8253)  CAU 13.3(  9269)  CGU 14.6( 10159)
CUC  8.1(  5614)  CCC  2.9(  1990)  CAC  4.5(  3121)  CGC  4.5(  3122)
CUA  8.0(  5591)  CCA 14.5( 10061)  CAA 31.1( 21678)  CGA  5.9(  4140)
CUG  6.4(  4476)  CCG  2.9(  2010)  CAG  6.6(  4596)  CGG  2.3(  1570)
 
AUU 51.0( 35505)  ACU 20.4( 14218)  AAU 40.1( 27903)  AGU 14.7( 10220)
AUC 16.1( 11197)  ACC  7.4(  5135)  AAC 11.1(  7709)  AGC  6.2(  4328)
AUA  8.7(  6023)  ACA 22.1( 15386)  AAA 61.2( 42611)  AGA  8.0(  5557)
AUG 24.7( 17203)  ACG  7.1(  4919)  AAG 13.5(  9375)  AGG  1.7(  1152)
 
GUU 30.8( 21415)  GCU 30.1( 20949)  GAU 37.3( 25965)  GGU 23.4( 16287)
GUC 12.7(  8841)  GCC 12.1(  8400)  GAC 14.4( 10018)  GGC  8.5(  5906)
GUA 12.8(  8921)  GCA 22.3( 15521)  GAA 56.1( 39035)  GGA 24.5( 17046)
GUG  9.2(  6376)  GCG  8.2(  5708)  GAG 13.0(  9070)  GGG  8.2(  5732)
Coding GC 36.75% 1st letter GC 48.61% 2nd letter GC 34.45% 3rd letter GC 27.20%
The codon table used
* TAA 
A GCT
C TGT
D GAT
E GAA
F TTT
G GGA
H CAT
I ATT
K AAA
L TTA
M ATG
N AAT
P CCA
Q CAA
R CGA
S TCA
T ACT
V GTT
W TGG
Y TAT
3、The inverse translation sequences and the selection of restriction enzymes
EcoR I
CCGGAATTCATGCGAGAATTTGTTCGACAACGACCATTATTATGTTTTTATGTTTTAGCTATTTTAATTGCTTTAGCTGCTCATGCTTTACGAGCTATGTCACCAACTCCATTAGATCCAATGTTTAAAATGTTACAAGAAACTCATGCTCATTTAAATATTATTACTGCTGTTCGATCAACTTTTGAATATCCAGGAGCTTATACTTTATTATTATTTCCAGCTGCTCCAATGTTTGCTGCTTTAATTGCTACTGGAATTGGATATGGACAAGCTGGATTTCGAGAATTATTATCACGATGTGCTCCATGGCGATCACCAGTTTCATGGCGACAAGGAGTTACTGTTATTGCTGTTTGTTTTTTAGCTTTTTTTGCTTTAACTGGAATTATGTGGGTTCAAACTTATTTATATGCTCCACCAGGAACTTTAGATCGAACTTTTTTACGATATGGATCAGATCCAGTTGCTATTTATGTTATGTTAGCTGCTTCATTATTATTATCACCAGGACCATTATTAGAAGAATTAGGATGGCGAGGATTTGCTTTACCACAATTATTAAAAAAATTTGATCCATTAACTGCTGCTGTTATTTTAGGAATTATGTGGTGGGCTTGGCATTTACCACGAGATTTACCAACTTTATTTTCAGGAGCTCCAGGAGCTGCTTGGTCAGTTATTGTTAAACAATTAGTTATTACTCCAGGATTTATTGCTTCAACTATTATTGCTGTTTTTGTTTGTAATAAATTAGGAGGATCAATGTGGGGAGGAGTTTTAACTCATGCTATTCATAATGAATTAGGAGTTAATGTTACTGCTGAATGGGCTCCAACTGTTGCTGGATTAGGATGGCGACCATGGGATTTAATTGAATTTGCTGTTGCTATTGGATTAGTTTTAATTTGTGGACGATCATTAGGAGCTGCTTCACCAGATAATGCTCGATTAGCTTGGGGAAATGTTCCACCAAAATTACCAGGAGGAGTTGGAGATAAATCAGGAGCTAATGCTTAATAGCTGCAGTT
4、The sequence analysis
_Base Count : 1008 bp (270 A, 377 T, 158 C, 203 G)
 _Composition : 36% GC, 64% AT
Absent Sites
AarI AatII Acc65I AccI AciI AclI AcuI AfeI AflII AflIII AgeI AhdI AleI ApaI ApaLI AscI AseI AsiSI AsuII AvaI AvrII BaeGI BamHI BanI BbsI BbvCI BceAI BciVI BclI BfaI BfuAI BglI BglII BlpI BmgBI BmrI BmtI Bpu10I BpuEI BsaAI BsaHI BsaI BsaWI BseMII BseYI BsgI BsiEI BsiWI BslI BsmAI BsmBI BsmFI BsmI BsoBI BspCNI BspEI BspHI BspMI BspQI BsrBI BsrDI BsrFI BsrGI BssHII BstAPI BstBI BstEII BstUI BstZ17I Bsu36I BtgZI BtrI BtsI Cac8I ClaI CviQI DdeI DrdI EaeI EagI EarI EciI Eco31I EcoNI EcoO109I EcoRI EcoRV Esp3I FauI FseI FspAI FspI HaeII HaeIII HgaI HhaI HinP1I HincII HindIII HinfI HpaI HpaII Hpy99I HpyAV HpyCH4IV HpyCH4V KasI KpnI MaeI MaeII MfeI MluI MlyI MscI NaeI NarI NciI NdeI NgoMIV NheI NmeAIII NotI NruI NsiI NspI PacI PasI PciI PflFI PflMI PleI PmeI PpuMI PshAI PsiI PspOMI PspXI PstI PvuI RsaI RsrII SacII SalI SanDI SbfI ScaI SexAI SfaNI SfcI SfiI SfoI SgrAI SmaI SmlI SnaBI SpeI SphI SrfI StuI TaiI TatI TauI TfiI TspGWI TspMI TspRI Tth111I XbaI XhoI XmaI ZraI
Unique Sites
XmnI (6) MslI (25)
Tsp45I (89)
SwaI (142) SspI (147)
MspA1I (211)
DraIII (286)
BstXI (309)
BsaBI (439) Hpy188I (447)
Sau96I (501)
MboII (513)
BssSI (619)
SacI (646) XcmI (651) AlwNI (652)
NlaIV (821)
Hpy166II (901)
5、Sequence Split
The maximum allowable assembly oligo length is equal to the target assembly oligo length (60). This may cause some weird behavior, especially in terms of overlap melting temperature.
 
2 building blocks were generated.
Building Block .1   529bp   1..529 
Left  - 5' CCGGAATTCATGCGAGAATTTGTTCG 3'
Rght  - 5' ATTCTTCTAATAATGGTCCTGGTGATAATAATAATGAAG 3'
RghtU - 5' ATTCTTCTAAUAATGGTCCTGGTGATAATAATAATGAAG 3'
Sequence:
CCGGAATTCATGCGAGAATTTGTTCGACAACGACCATTATTATGTTTTTATGTTTTAGCTATTTTAATTGCTTTAGCTGCTCATGCTTTACGAGCTATGTCACCAACTCCATTAGATCCAATGTTTAAAATGTTACAAGAAACTCATGCTCATTTAAATATTATTACTGCTGTTCGATCAACTTTTGAATATCCAGGAGCTTATACTTTATTATTATTTCCAGCTGCTCCAATGTTTGCTGCTTTAATTGCTACTGGAATTGGATATGGACAAGCTGGATTTCGAGAATTATTATCACGATGTGCTCCATGGCGATCACCAGTTTCATGGCGACAAGGAGTTACTGTTATTGCTGTTTGTTTTTTAGCTTTTTTTGCTTTAACTGGAATTATGTGGGTTCAAACTTATTTATATGCTCCACCAGGAACTTTAGATCGAACTTTTTTACGATATGGATCAGATCCAGTTGCTATTTATGTTATGTTAGCTGCTTCATTATTATTATCACCAGGACCATTATTAGAAGAAT
 
Assembly Oligos: average overlap Tm is 47°;average oligo length is 58bp.
 
CCGGAATTCATGCGAGAATTTGTTCGACAACGACCATTATTATGTTTTTATGTTTTAGCTATTTTAATTGCTTTAGCTGCTCATGCTTTACGAGCTATGTCACCAACTCCATTAGATCCAATGTTTAAAATGTTACAAGAAACTCATGCTCATTTAAATATTATTACTGCTGTTCGATCAACTTTTGAATATCCAGGAGCTTATACTTTATTATTATTTCCAGCTGCTCCAATGTTTGCTGCTTTAATTGCTACTGGAATTGGATATGGACAAGCTGGATTTCGAGAATTATTATCACGATGTGCTCCATGGCGATCACCAGTTTCATGGCGACAAGGAGTTACTGTTATTGCTGTTTGTTTTTTAGCTTTTTTTGCTTTAACTGGAATTATGTGGGTTCAAACTTATTTATATGCTCCACCAGGAACTTTAGATCGAACTTTTTTACGATATGGATCAGATCCAGTTGCTATTTATGTTATGTTAGCTGCTTCATTATTATTATCACCAGGACCATTATTAGAAGAAT
CCGGAATTCATGCGAGAATTTGTTCGACAACGACCATTATTATGTTTTTATGTTTTAGCT              AGCTGCTCATGCTTTACGAGCTATGTCACCAACTCCATTAGATCCAATGTTTAAAATGTT              CTCATTTAAATATTATTACTGCTGTTCGATCAACTTTTGAATATCCAGGAGCTTATACTT              GCTGCTCCAATGTTTGCTGCTTTAATTGCTACTGGAATTGGATATGGACAAGCTGGATTT              ACGATGTGCTCCATGGCGATCACCAGTTTCATGGCGACAAGGAGTTACTGTTATTGCTGT               TTTTGCTTTAACTGGAATTATGTGGGTTCAAACTTATTTATATGCTCCACCAGGAACTTT                CGATATGGATCAGATCCAGTTGCTATTTATGTTATGTTAGCTGCTTCATTATTATTATCA                      
                                     ATAATACAAAAATACAAAATCGATAAAATTAACGAAATCGACGAGTACGAAATGCTC                 AATCTAGGTTACAAATTTTACAATGTTCTTTGAGTACGAGTAAATTTATAATAATGACGA              ACTTATAGGTCCTCGAATATGAAATAATAATAAAGGTCGACGAGGTTACAAACGAC                  AACCTATACCTGTTCGACCTAAAGCTCTTAATAATAGTGCTACACGAGGTACCGCT                  GTTCCTCAATGACAATAACGACAAACAAAAAATCGAAAAAAACGAAATTGACCTTAATAC                ATATACGAGGTGGTCCTTGAAATCTAGCTTGAAAAAATGCTATACCTAGTCTAGGTCAAC                TCGACGAAGTAATAATAATAGTGGTCCTGGTAATAATCTTCTTA
GGCCTTAAGTACGCTCTTAAACAAGCTGTTGCTGGTAATAATACAAAAATACAAAATCGATAAAATTAACGAAATCGACGAGTACGAAATGCTCGATACAGTGGTTGAGGTAATCTAGGTTACAAATTTTACAATGTTCTTTGAGTACGAGTAAATTTATAATAATGACGACAAGCTAGTTGAAAACTTATAGGTCCTCGAATATGAAATAATAATAAAGGTCGACGAGGTTACAAACGACGAAATTAACGATGACCTTAACCTATACCTGTTCGACCTAAAGCTCTTAATAATAGTGCTACACGAGGTACCGCTAGTGGTCAAAGTACCGCTGTTCCTCAATGACAATAACGACAAACAAAAAATCGAAAAAAACGAAATTGACCTTAATACACCCAAGTTTGAATAAATATACGAGGTGGTCCTTGAAATCTAGCTTGAAAAAATGCTATACCTAGTCTAGGTCAACGATAAATACAATACAATCGACGAAGTAATAATAATAGTGGTCCTGGTAATAATCTTCTTA 
 
Building Block .2   513bp   519..1031 
Left  - 5' ATTAGAAGAATTAGGATGGCGAGGATTTGC 3'
LeftU - 5' ATTAGAAGAAUTAGGATGGCGAGGATTTGC 3'
Rght  - 5' AAGACGTCCTATTAAGCATTAGCTCCTG 3'
Sequence:
ATTAGAAGAATTAGGATGGCGAGGATTTGCTTTACCACAATTATTAAAAAAATTTGATCCATTAACTGCTGCTGTTATTTTAGGAATTATGTGGTGGGCTTGGCATTTACCACGAGATTTACCAACTTTATTTTCAGGAGCTCCAGGAGCTGCTTGGTCAGTTATTGTTAAACAATTAGTTATTACTCCAGGATTTATTGCTTCAACTATTATTGCTGTTTTTGTTTGTAATAAATTAGGAGGATCAATGTGGGGAGGAGTTTTAACTCATGCTATTCATAATGAATTAGGAGTTAATGTTACTGCTGAATGGGCTCCAACTGTTGCTGGATTAGGATGGCGACCATGGGATTTAATTGAATTTGCTGTTGCTATTGGATTAGTTTTAATTTGTGGACGATCATTAGGAGCTGCTTCACCAGATAATGCTCGATTAGCTTGGGGAAATGTTCCACCAAAATTACCAGGAGGAGTTGGAGATAAATCAGGAGCTAATGCTTAATAGGACGTCTT
Assembly Oligos: average overlap Tm is 49°;average oligo length is 58bp.
ATTAGAAGAATTAGGATGGCGAGGATTTGCTTTACCACAATTATTAAAAAAATTTGATCCATTAACTGCTGCTGTTATTTTAGGAATTATGTGGTGGGCTTGGCATTTACCACGAGATTTACCAACTTTATTTTCAGGAGCTCCAGGAGCTGCTTGGTCAGTTATTGTTAAACAATTAGTTATTACTCCAGGATTTATTGCTTCAACTATTATTGCTGTTTTTGTTTGTAATAAATTAGGAGGATCAATGTGGGGAGGAGTTTTAACTCATGCTATTCATAATGAATTAGGAGTTAATGTTACTGCTGAATGGGCTCCAACTGTTGCTGGATTAGGATGGCGACCATGGGATTTAATTGAATTTGCTGTTGCTATTGGATTAGTTTTAATTTGTGGACGATCATTAGGAGCTGCTTCACCAGATAATGCTCGATTAGCTTGGGGAAATGTTCCACCAAAATTACCAGGAGGAGTTGGAGATAAATCAGGAGCTAATGCTTAATAGGACGTCTT
ATTAGAAGAATTAGGATGGCGAGGATTTGCTTTACCACAATTATTAAAAAAATTTGATCC            TGTTATTTTAGGAATTATGTGGTGGGCTTGGCATTTACCACGAGATTTACCAACTTTATT            AGGAGCTGCTTGGTCAGTTATTGTTAAACAATTAGTTATTACTCCAGGATTTATTGCTTC            TGTTTTTGTTTGTAATAAATTAGGAGGATCAATGTGGGGAGGAGTTTTAACTCATGC               AGGAGTTAATGTTACTGCTGAATGGGCTCCAACTGTTGCTGGATTAGGATGGCGA                 ATTTGCTGTTGCTATTGGATTAGTTTTAATTTGTGGACGATCATTAGGAGCTGCTTC                TTAGCTTGGGGAAATGTTCCACCAAAATTACCAGGAGGAGTTGGAGATAAATCAGGAGC                     
                                    TGTTAATAATTTTTTTAAACTAGGTAATTGACGACGACAATAAAATCCTTAATACACCAC            TGGTGCTCTAAATGGTTGAAATAAAAGTCCTCGAGGTCCTCGACGAACCAGTCAA                 ATAATGAGGTCCTAAATAACGAAGTTGATAATAACGACAAAAACAAACATTATTTAATCC            CCCTCCTCAAAATTGAGTACGATAAGTATTACTTAATCCTCAATTACAATGACGACTTAC            ACGACCTAATCCTACCGCTGGTACCCTAAATTAACTTAAACGACAACGATAACCTAATCA            TGCTAGTAATCCTCGACGAAGTGGTCTATTACGAGCTAATCGAACCCCTTTACAAGGTG               CTCAACCTCTATTTAGTCCTCGATTACGAATTATCCTGCAGAA
TAATCTTCTTAATCCTACCGCTCCTAAACGAAATGGTGTTAATAATTTTTTTAAACTAGGTAATTGACGACGACAATAAAATCCTTAATACACCACCCGAACCGTAAATGGTGCTCTAAATGGTTGAAATAAAAGTCCTCGAGGTCCTCGACGAACCAGTCAATAACAATTTGTTAATCAATAATGAGGTCCTAAATAACGAAGTTGATAATAACGACAAAAACAAACATTATTTAATCCTCCTAGTTACACCCCTCCTCAAAATTGAGTACGATAAGTATTACTTAATCCTCAATTACAATGACGACTTACCCGAGGTTGACAACGACCTAATCCTACCGCTGGTACCCTAAATTAACTTAAACGACAACGATAACCTAATCAAAATTAAACACCTGCTAGTAATCCTCGACGAAGTGGTCTATTACGAGCTAATCGAACCCCTTTACAAGGTGGTTTTAATGGTCCTCCTCAACCTCTATTTAGTCCTCGATTACGAATTATCCTGCAGAA
The experimental procedure and analysis
1、Dissolving of primers
Sterile deionized water whose volume is determined by primer’s molecular weight is used to dilute the solution of the primer to 10um/L. And it is prepared for the following experiment.
Num
Tm
MW(g/mole)
nmol/Tube
Add buffer for 10μM/uL
A101
68.3
18451
1.6
155
A102
65
17588.4
1.4
142
A103
69.7
18359
1.5
155
A104
64.2
18533
1.4
143
A105
66.3
18354
1.6
155
A106
68
17321.2
1.5
151
A107
71.1
18533
1.6
156
A108
70.9
17127.2
1.6
164
A109
73.8
18513
1.6
156
A110
66.3
18440
1.4
137
A111
67.6
18426
1.6
156
A112
69.7
18506
1.5
147
A113
67
18441
1.6
156
A114
64
13568.8
2.0
200
A201
66.3
18568
1.4
143
A202
64.9
18407
1.5
145
A203
68.3
18522
1.5
155
A204
71.8
16968
1.6
162
A205
68.3
18491
1.5
154
A206
64.9
18492
1.4
138
A207
67.9
17753.4
1.6
156
A208
68.3
18305
1.5
151
A209
72.6
17117
1.6
164
A210
70.4
18278
1.5
149
A211
68.6
17622.4
1.7
165
A212
72.6
18145.8
1.6
156
A213
71.9
18356.8
1.5
146
A214
66.9
13072.6
2.2
220
A1
62
8002.2
3.6
356
A2
61
12043.8
2.2
222
A3
62
9375
2.8
281
A4
62
8548.6
3.3
330
A5
62
8548.6
3.3
330
2、Mixing of primers
Oligonucleotides synthesized in powder form need to be centrifuged for 1 min at 10000rpm to assemble powders to the bottom of the EP tube.And be careful to open the cap.
Make sure that the synthesis report is consistent to the OD number on the primer label,and check the number of the dispensing tube before the dissolution of oligonucleotides.The volume of water added to make 10umol/L oligonucleotides’ solution is calculated according to the synthetic single report of each primer. Add sterile deionized water and keep it at room temperature for 2min.And reverse the tube to accelerate the progress of solubilization. Finally, collect substances at the bottom of the tube after centrifugation.
Every six primers are combined to one group.Add 4ulbeginning fiprimer, 1ulintermediate primer and 4ul end primerto a new EP tube, then add sterile deionized water to 20ul, mix them up as spare.
Group
Number of primer
Uptake
The volume of sterile deionized water
The total volume
Each PCR (50ul system) uptake
Final concentration
A1
A101
4
8
20
5ul
200nM
A102
1
50nM
A103
1
50nM
A104
1
50nM
A105
1
50nM
A106
4
200nM
A2
A107
4
8
20
5ul
200nM
A108
1
50nM
A109
1
50nM
A110
1
50nM
A111
1
50nM
A112
4
200nM
A3
A113
4
8
20
5ul
200nM
A114
1
50nM
A201
1
50nM
A202
1
50nM
A203
1
50nM
A204
4
200nM
A4
A205
4
8
20
5ul
200nM
A206
1
50nM
A207
1
50nM
A208
1
50nM
A209
1
50nM
A210
4
200nM
A5
A211
4
10
20
5ul
200nM
A212
1
50nM
A213
1
50nM
A214
4
200nM
3、DA-PCR
Every six primers which are single-stranded oligonucleotidesare chemically synthesized were combined to one group. Then they were going to be made as longer double-stranded fragments of intermediate (Block) by DA-PCR
Procedure of DA-PCR：
Mixed primer solutions                   5μl
Pfu DNA Polymerase  2.5U              0.5μl    
dNTP                                4μl
10×Pfu buffer（Mg2+）               5μl
ddH20add to 50μl
Procedure of DA-PCR：
94℃         2min
94℃         30S
50℃         30S     25 cycles
72℃         1min
72℃        10min
4℃         preservation
End
DA-PCR splicing reaction product was  detected in 2% agarose gel electrophoresis,
DA-PCR product              5μl
10×Loading Buffer         0.6μl    spotting after mixed
100bp Marker              5μl    spotting directly
75V electrophoress for 1h。
Figure1 DA-PCR  splicing results
Illustration of  the outcome:
1、A1,  A2, A3, A4 could getintermediates (Block1-4) which were made by six  single-stranded oligonucleotides splicedtogether, but the A3 and A4 groups  could also see by-products which were made by four single-stranded  oligonucleotides splicedtogether , and to be made a whole one after recycling  of Agarose gel.
2、A5 could  obtainintermediates(Block5) which were made by four single-stranded  oligonucleotides splicedtogether.
4、OE-PCR
Take the 5 double strand intermediate fragment (Block) spliced by DA-PCR as templates, blocks were further connected by OR-PCR, OE-PCR in a reaction system consisting of:
Block1                       1μl
Block2                       1μl
Block3                       1μl
Block4                       1μl
Block5                       1μl
A1                         1μl
A4                        1μl
Pfu DNA Polymerase 2.5U       0.5μl
dNTP                        4μl
10×Pfu buffer（Mg2+）         5μl
Sterilized  ultrapure water   to 50ul
Procedure of OE-PCR：
94℃         2min
         94℃         30S
47℃         30S    20 cycles
72℃        2min
72℃       10min
 4℃        preservation
End
The full-length gene  spliced by OE-PCR was detected by a 1% agarose gel electrophoresis, specific  programs are as follows
OE-PCR  product           5μl
10×Loading Buffer        0.6μl    spotted after mixing well
100bp  Marker              5μl   spotted directly
80V electrophoresis for  1h.
Illustration of  the outcome:
L1,  L2, L3 could get mlrA genome which were made by five single-stranded  oligonucleotides splicedtogether.
Wide  strip of the tape at 100bp was caused by excessive amount of amplification  primers A1 and A4.
5、Constructing and sequencing of subcoloning vector
After double  enzyme digestion reaction at 37℃ for 3h by EcoRⅠand PstⅠ, MlrA gene and pSB1C3 vector linked  together at 16℃  overnight. And then transformed to E.coli JM109 competent cell, and select the desirable colony by using “blue-white  selection” method.
Constitute of EcoRⅠand PstⅠdouble enzyme  digestion reaction system：
Gene or plasmid                       10μl
EcoRⅠ                             0.4μl
PstⅠ                               0.2μl
10×NEBuffer 3.1                      2μl
sterilizing ulturapure water         top up to 20μl
Constitute  of linking reaction system：
mlrA Gene                           10μl
pSB1C3                               3μl
T4 ligase                             0.5μl
10×T4 ligase Buffer                    2μl
sterilizing ulturapure water         top up to 20μl
Preparation process of competent cell E.coliBL21：
(1) Culturing strain E.coliBL21(37℃)overnight  to recovery culture, Adding overnight bacteria culture fluid 600uLto 30 mL LB  culture medium, and 37℃，200  r/min shake culturing 1-2 h, until OD600 is around 0.3-0.4.
(2) In the aseptic  condition, transfer 30mLbacteria fluid to the centrifugal tube in the ice 10  min to cooling the culture to 0℃.
(3)  Centrifuging 10 min in precooling centrifugal machine over 4000r/min, throw  supernate, and then add30mLprecooling 0.1mol/L CaCl2 solution to  resuspension precipitation in the ice 10 min.
(4) Centrifuging 4000r/min  in 4℃10min, throw  supernate,  each 30mL initial culture  resuspension cell precipitation with 1.0mL precooling 0.1mol/LCaCl2  solution, and then 200μL each one.
Heal  shock transformation process：
(1) Thawing freshly  prepared or -80℃ competentcellBL21suspension.
(2) Add constructed  recombinant cloning vector plasmidspSB1C3-A(no more than 50ng, 10μl), shake  gently, and then in the ice 30min.
(3)  Put EP tube in 42℃ water bath90s，and  then transfer it to ice water bath to cool cells1-2 min.
(4) add 800μL LB medium  to EP tube，transfer  it to 37℃  table，150r/min，to  recovery strain 45 min.
Spread plate and  Culture
(1)After culture in 37℃,  centrifuge the cell solution(5000rpm,5min) and concentrate it to 200μl.  Then coat it to LB culture with chloramphenicol and set a positive control.
(2) Wait until the  solution in culture is full absorbed by cell, then invert the plate and culture  it in 37℃  overnight.
Sequence test
Replace the fragment(45bp) between EcoR I site  and Pst I site with mlrA(1024bp) on pSB1C3 vector and then we get the  reconstructed plasmid, pSB1C3-A.
 
Fig3.Reconstructed vector pSB1C3-A
Select the white colony on culture and culture it. Then we extract the plasmid and use it as template in PCR to identify the colony. In theory, the reconstructed vector with mlrA fragment is 1293bp and the empty vector without mlrA fragment is 314bp. So, the result shows that the white colony we select has reconstructed vector with mlrA fragment in it.
                   Fig 4. The identification of reconstructed vector pSB1C3-A
Results:
1.L1 and L2 is 1300bp and it confirms that the reconstructed vector has mlrA on it.
2.M is the 100bp Ladde Marker.
Sequencing result shows that the sequence of synthetic mlrA gene is exactly same as sequence of designed sequence.
 
Fig5.Comparison of result of sequencing and designed sequence
 
Top
Retrieved from " http://2014.igem.org/Team:Jilin_China/RESULT "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
