Team:DTU-Denmark/Results-amilCP
Project
Description
Design
InterLab
Results
Choice of Organism
amilCP Expression
melA Expression
nat1 Expression
Ganoderma Protoplastation
Demonstrate
Modeling
Model
Growth Modeling
Design of Experiments
Modeling the Design
Structural Integrity
Hardware
Parts
Journal
Notebook
Experiments
People
Team Members
Attributions
Human Practices
Human Practices
Education and Engagement
Collaboration
Collaborations
BioBrick Tutorial
Safety
Hyphae Hackers
Project
Description Design InterLab Results Choice of Organism amilCP Expression melA Expression nat1 Expression Ganoderma Protoplastation Demonstrate
Modeling
Model Growth Modeling Design of Experiments Modeling the Design Structural Integrity Hardware
Parts Overview Journal
Notebook Experiments
People
Team Members Attributions
Safety Human Practices
Human Practices Education & Engagement
Collaborations
Collaborations BioBrick Tutorial
amilCP Expression
amilCP encodes a blue chromoprotein from the coral Acropora millepora commonly used as a biosensor circuit in iGEM. In order to investigate our ability to transform Aspergillus oryzae and change its color, we constructed an expression cassette for it. This was done with aim of being able to customize the color of the fungal materials produced.
Assembly
amilCP was acquired from the distribution kit and assembled into pSB1C3 with two different fungal/eukaryotic promoters – the Aspergillus nidulans derived promoter PgpdA and the cauliflower mosaic virus 35S promoter (PCaMV) - using 3A assembly. The Cauliflower Mosaic Virus 35S terminator was also added via 3A assembly resulting in cassettes 1623 and 1290 bp long respectively. Correct transformants were identified by colony PCR using the verification primers VF2 and VR (See fig. 1) and later confirmed by Sanger sequencing with the same primers. The constructs are shown in fig. 2 and 3.
Fig. 1: Colony PCR of transformants containing the amilCP expression cassette. The PCR was performed using two sets of primers, hence the alternating pattern. The odd numbered wells uses a primer set (BB_FW and SeqP_AmilCP_BW) which reads over the insertion site of a potential promoter. if no promoter is inserted, at band of ~130 bp. The even numbered wells use a primer set (SeqP_AmilCP_FW and SeqP_CaMVT_BW) that gives a band ~240 bp if amilCP is correctly assembled with The Cauliflower Mosaic Virus 35S terminator . Transformants identified as positive are marked with red. 1 KB + 100 bp ladder from NEB used.
Fig. 2: Plasmid map of BBa_K2799011 for expression of AmilCP.
Fig. 3: Plasmid map of BBa_k2799012 for expression of AmilCP.
Transformation
After confirmed assembly, the amilCP expression cassettes were then transformed into Aspergillus oryzae RIB40. They were plated onto transformation media using no selectable conditions, as none appropriate were available. Nonetheless, some strains, mainly those transformed using linearized BBa_K2799011, exhibited a change in spore coloration. Possible transformants exhibited green and white coloured spores, which Aspergillus oryzae plated on PDA does not, instead exhibiting yellow hued spores, see fig. 4:
Fig. 4: WT strain plated PDA compared to possible transformant plated on transformation media
We, however, suspected that this difference in coloration might be an artifact of different media. Therefore, we plated the WT strain onto transformation plates as well, see fig. 5. Here it showed light-green spore formation, but no white or dark-green spores. This was confirmed under a microscope (data not shown).
Fig. 5: WT plated onto transformation media.
Some variations in the color of transformants were additionally observed with some being a dark green and some showing a more yellowish hue, although all being green, see fig. 6:
Fig. 6: Plates from transformations. 1 and 2 exhibits a noticeably more green hue than 3 and 4.
Based on this we decided to confirm integration into the genome of transformants by PCR. Genomic DNA was purified from possible transformants exhibiting both the dark green and white spores. PCR was performed on the gDNA using primers (SeqP_AmilCP_FW and SeqP_CaMVT_BW) targeting a 227 bp region from the end of amilCP to the middle of the terminator, see fig. 7. Two samples was taken from each of the plates in fig. 6. One from a green area and one from a white are. Clear bands was produced from samples taken from green plates (1.1-2.2), while yellowish plates did not show any bands (3.1-4.2), thus indicating the darker green color as a possible byproduct of amilCP expression. No difference was seen between green and white spores from plate 1 and 2.
Fig. 7: PCR of possible transformants run on a 2% gel. 1.1-2.2 contains scrapings from green plates, while 3.1-4.2 contains scrapings from yellow plates. As a positive control PgpdA-amilCP-tCaMV:pSB1C3 was included. Ladder: 100 bp DNA ladder fra NEB.
Next, we repeated the PCR on sample 1.1-2.2, this time including two gDNA samples prepared from scrapings of WT plates, see fig. 8. Again clear bands were seen from samples 1.1-2.2, however, the WT isolates did not produce any bands demonstrating that the band is produced due to integration and not just because of suitable binding regions for the primers present in the genome of A. oryzae.
Fig. 8: PCR of samples 1.1-2.2 + wild type isolates on a 2% gel. 1.1-2.2 still produces clear bands in the correct size. Neither of the wild type isolates, marked by a red circle, shows bands. As a positive control PgpdA-amilCP-tCaMV:pSB1C3 was included. Ladder: 100 bp DNA ladder fra NEB.
To confirm integration in the positively identified transformants (1.1-2.2) samples have been submitted for Sanger sequencing using primer pointing into the constructs. Additionally, samples have also been submitted with primers pointing out of the construct in order to identify the site of integration in the genome. The primers used are: SeqP_AmilCP_FW, SeqP_CaMVT_FW and SeqP_CaMVT_BW. We are currently awaiting the sequencing results.
Brick Construction
To test whether the integration, and possible expression of amilCP, would lead to a visual change in materials produced, a small brick was inoculated using spores from the identified transformants. However, this did not produce any visual obvious change in brick color (data not shown).
Conclusion
A slight visual difference in spore colouration can be observed in possible transformants corresponding (see fig. 4) to integration of the expression cassette as demonstrated by PCR (see fig. 5 and 6). This indicates production of AmilCP as it is a blue chromoprotein. However, to definitely demonstrate this further test are needed. Given more time we would like to have demonstrated by qPCR as well as show the actual presences of AmilCP in the transformants by mass spectrometry. In order to change the color of the bricks produced from it higher expression seems to be necessary. This could feasibly be by codon-optimizing the sequence for expression in A. oryzae, including kozak sequences or by using another promoter.
Appendix: Primer list
Useful links
Home • Project description • Modeling • Parts overview • Notebook • Team • Safety • Human practices
About us
dtubiobuilders@gmail.com
Anker Engelunds Vej 1 Bygning 101A, 2800 Kgs. Lyngby, Denmark
Contact us
