<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_DTU-Denmark_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:DTU-Denmark/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><UL><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Description"><DIV class="item"><SPAN class="caption">Project</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Description">Description</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Design">Design</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/InterLab">InterLab</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Results">Results</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Results-choosing-organism">Choice of Organism</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Results-amilCP">amilCP Expression</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Results-melA">melA Expression</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Results-NAT-1">nat1 Expression</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Ganoderma-protoplastation">Ganoderma Protoplastation</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Demonstrate">Demonstrate</A></LI></UL></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Model"><DIV class="item"><SPAN class="caption">Modeling</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Model">Model</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/GrowthModelling">Growth Modeling</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/DesignOfExperiments">Design of Experiments</A></LI><LI style="padding-left:4em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/ModellingTheDesign">Modeling the Design</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/StructuralIntegrity">Structural Integrity</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Hardware">Hardware</A></LI></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Parts"><DIV class="item"><SPAN class="caption">Parts</SPAN></DIV></A></LI><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Notebook"><DIV class="item"><SPAN class="caption">Journal</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Notebook">Notebook</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Experiments">Experiments</A></LI></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Team"><DIV class="item"><SPAN class="caption">People</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Team">Team Members</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Attributions">Attributions</A></LI></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Human_Practices"><DIV class="item"><SPAN class="caption">Human Practices</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Human_Practices">Human Practices</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Public_Engagement">Education and Engagement</A></LI></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Collaborations"><DIV class="item"><SPAN class="caption">Collaboration</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Collaborations">Collaborations</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/BioBrickTutorial">BioBrick Tutorial</A></LI></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Safety"><DIV class="item"><SPAN class="caption">Safety</SPAN></DIV></A></LI><DIV id="mobiletopnav" class="topnav"><A href="https://2018.igem.org/Team:DTU-Denmark" class="activeMM">Hyphae Hackers</A><DIV id="myLinks"><A href="javascript:void(0);" onclick="myFunction1()">
    Project
  </A><DIV id="myLinks1" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Description">Description</A><A href="https://2018.igem.org/Team:DTU-Denmark/Design">Design</A><A href="https://2018.igem.org/Team:DTU-Denmark/InterLab">InterLab</A><A href="https://2018.igem.org/Team:DTU-Denmark/Results">Results</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/Results-choosing-organism">Choice of Organism</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/Results-amilCP">amilCP Expression</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/Results-melA">melA Expression</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/Results-NAT-1">nat1 Expression</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/Ganoderma-protoplastation">Ganoderma Protoplastation</A><A href="https://2018.igem.org/Team:DTU-Denmark/Demonstrate">Demonstrate</A></DIV><A href="javascript:void(0);" onclick="myFunction2()">
    Modeling
  </A><DIV id="myLinks2" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Model">Model</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/GrowthModelling">Growth Modeling</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/DesignOfExperiments">Design of Experiments</A><A style="padding-left:4em" href="https://2018.igem.org/Team:DTU-Denmark/ModellingTheDesign">Modeling the Design</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/StructuralIntegrity">Structural Integrity</A><A href="https://2018.igem.org/Team:DTU-Denmark/Hardware">Hardware</A></DIV><A href="https://2018.igem.org/Team:DTU-Denmark/Parts">Parts Overview</A><A href="javascript:void(0);" onclick="myFunction4()">
    Journal
  </A><DIV id="myLinks4" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Notebook">Notebook</A><A href="https://2018.igem.org/Team:DTU-Denmark/Experiments">Experiments</A></DIV><A href="javascript:void(0);" onclick="myFunction5()">
    People
  </A><DIV id="myLinks5" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Team">Team Members</A><A href="https://2018.igem.org/Team:DTU-Denmark/Attributions">Attributions</A></DIV><A href="https://2018.igem.org/Team:DTU-Denmark/Safety">Safety</A><A href="javascript:void(0);" onclick="myFunction6()">
    Human Practices
  </A><DIV id="myLinks6" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:DTU-Denmark/Public_Engagement">Education &amp; Engagement</A></DIV><A href="javascript:void(0);" onclick="myFunction7()">
    Collaborations
  </A><DIV id="myLinks7" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:DTU-Denmark/BioBrickTutorial">BioBrick Tutorial</A></DIV></DIV></DIV><DIV class="toppercontainer"><DIV class="headlinecontainer"><H1>Notebook</H1></DIV></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV id="notebookheight" class="container-fluid" style="width: 80%;margin-left: auto;
    margin-right: auto;position:relative;margin-top:0%;"><DIV class="row"><DIV class="col-xs-12"><BUTTON id="week5" class="collapsible">Week 5 (January 29 - February 4)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">Team Overview</H4><P>
								The DTU BioBuilders had the first official meeting. Fun team building activities were planned and the members got the first real taste of what iGEM is really all about.


							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week15" class="collapsible">Week 15 (April 9 - April 15)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">Team Overview</H4><P>
								The annual <A target="_blank" href="https://2018.igem.org/Team:DTU-Denmark/BioBrickTutorial">BioBrick Tutorial</A> was held and 89 exited and talented iGEM participants were gathered at DTU for an exciting weekend.


							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week17" class="collapsible">Week 17 (April 23 - April 29)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">Team Overview</H4><P>
								The team participated in <A target="_blank" href="https://2018.igem.org/Team:DTU-Denmark/Public_Engagement#eduscienceinforum">Science in Forum</A>, where they had a stand and told interested high school students about the wonders that is iGEM. The team based their explanations on the previous team's project and on the many project ideas that they had discussed during their time in iGEM.


							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week18" class="collapsible">Week 18 (April 30 - May 6)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">Team Overview</H4><P>
								Lo and behold! After many hours of discussing various projects and weighing the pros and cons of each of them, the DTU BioBuilders have now finally come to an agreement. This year the team will work on making a general tool box for uses of fungi. This entails ways of <A target="_blank" href="https://2018.igem.org/Team:DTU-Denmark/Description"> creating mycotextures to be used in the colonization of Mars</A>.


							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week23" class="collapsible">Week 23 (June 4 - June 10)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">Team Overview</H4><P>
								The team attended the <A target="_blank href=" https:="">Nordic iGEM Conference</A> hosted by Team Lund. Here they presented their project and showed off the amazing poster specially made for this event.
</P><DIV id="myModal" class="modal"><SPAN class="close">×</SPAN></DIV><P>
Needless to say that everyone had fun seeing and discussing the other groups' projects.


							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week24" class="collapsible">Week 24 (June 11 - June 17)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">Synthlab Overview</H4><P>
								The first day in the lab was used for creating media of different types and concentrations for the species to be grown on for two
reasons:</P></DIV></LI><LI><P>To illustrate an examine the growth rate (for modeling)</P></LI><LI><P>To multiply our pure species for use in liquid media</P></LI><P>
We used different PDA and MEA media for the species. It was also the plan to create
Vogel's medium as well as a mineral salt medium as certain species had shown in the previous study to grow well. However, we
did not have the proper materials to create those two and discarded the idea. We also came to this conclusion as we were
recommended not to spend time on specific media just yet.</P><BUTTON class="collapsible">More details here</BUTTON><DIV class="content"><P>All media was prepared before 11 am to be autoclaved. The actual weights of the different media was:</P><P><TABLE class="tg"><TBODY><TR><TH colspan="4" style="background-color:#27518e;color:#ffffff;text-align:center;">MEA Media</TH></TR><TR><TD class="tg-mhe8">Materials</TD><TD class="tg-0lax">Weight (g)</TD><TD class="tg-0lax">Weight (g)</TD><TD class="tg-0lax">Weight (g)</TD></TR><TR><TD class="tg-mhe8">Malt extract</TD><TD class="tg-0lax">47.6</TD><TD class="tg-0lax">49.9</TD><TD class="tg-0lax">52.5</TD></TR><TR><TD class="tg-mhe8">ZnSO4·7H2O</TD><TD class="tg-0lax">0.01</TD><TD class="tg-0lax">0.012</TD><TD class="tg-0lax">0.01</TD></TR><TR><TD class="tg-mhe8">CuSO4·5H2O</TD><TD class="tg-0lax">0.05</TD><TD class="tg-0lax">0.049</TD><TD class="tg-0lax">0.051</TD></TR><TR><TD class="tg-mhe8">Agar</TD><TD class="tg-0lax">14.99</TD><TD class="tg-0lax">14.99</TD><TD class="tg-0lax">15.04</TD></TR></TBODY></TABLE><TABLE class="tg"><TBODY><TR><TH style="background-color:#27518e;color:#ffffff;text-align:center;" colspan="4">PDB Media</TH></TR><TR><TD class="tg-mhe8">Potato dextrose extract</TD><TD class="tg-0lax">Weight (g)</TD><TD class="tg-0lax">Weight (g)</TD><TD class="tg-0lax">Weight (g)</TD></TR><TR><TD class="tg-mhe8">Malt extract </TD><TD class="tg-0lax">19.54</TD><TD class="tg-0lax">39.12</TD><TD class="tg-0lax">58.56</TD></TR><TR><TD class="tg-mhe8">ZnSO4·7H2O</TD><TD class="tg-0lax">0.012</TD><TD class="tg-0lax">0.012</TD><TD class="tg-0lax">0.0107</TD></TR><TR><TD class="tg-mhe8">CuSO4·5H2O </TD><TD class="tg-0lax">0.0056</TD><TD class="tg-0lax">0.0053</TD><TD class="tg-0lax">0.0049</TD></TR><TR><TD class="tg-mhe8">Agar</TD><TD class="tg-0lax">15.02</TD><TD class="tg-0lax">15.04</TD><TD class="tg-0lax">15.04</TD></TR></TBODY></TABLE></P><P>

All media were stirred with a magnetic stirrer at around 110 degrees Celsius and 1000 rpm until homologous.
The species available (<I>pleuratus ostreatus</I>) was inoculated on the solid PDA and MEA agar plates with a sterile toothpick
under a fume hood and left at 30 degrees Celsius. It was on Monday discovered that the sample given was of mycelia
without any spores and the process was redone.




</P></DIV></UL></DIV></DIV><BUTTON id="week25" class="collapsible">Week 24 (June 11 - June 17)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>SynthLab Overview</H4><P><B>20/6/2018</B><I>pleuratus ostreatus</I> was re-inoculated on the solid agar plates by placement of around 8 mycelia &quot;balls&quot; under a fume hood and left at 30 degrees Celsius. As to save time, fewer plates were inoculated and a YBD plate was included.
Today, we had gotten another species, <I>Schizophyllum commune</I>, in two forms: a wildtype as well as a mutant (<I>Δsc3</I>) which were also inoculated on PDB, MEA and YBD plates.
<B>22/6/2018</B>
The first sign of growth in our species was showing and pictures were taken of every plate as to count pixels and indicate the growth rate. Pictures were taken every morning for 5 days.



							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week26" class="collapsible">Week 26 (June 25 - July 1)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>SynthLab Overview</H4><P><B>26/6/2018</B><I>Aspergillus oryzae</I> was received and inoculated on MEA, PDB and YBD plates (5 days of photographs will follow this
species's growth in the same manner as the other species).
Today we decided that our species had grown enough and the samples were then removed and stored at a cooler
temperature excluding those used for later liquid media.

Liquid PDB (PDA) was prepared for growth in our &quot;brick&quot;-molds without the addition of agar. 500 mL was created for each species. The PDB concentration chosen was the
middle concentration in the table as these showed the strongest growth.

At the end of this process, we should be ready to start with the actual growth of our building material.


							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week27" class="collapsible">Week 27 (July 2 - July 8)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>2/7/2018</B>
								Boxes: Initialisation of tests to determine best substrates and conditions for formation of a brick made of fungi. Saw dust was autoclaved. 
Spore suspension: Spore suspensions are made for easy storage of spores. The spores are used for protoplastation and inoculation of boxes and plates. 
<I><I>A. oryzae</I></I> did not sporulate. It was incubated from June 26th to June 29th. Three days is not enough time for it to sporulate. 
<I>P. ostreatus</I> and <I>S. commune</I> did not sporulate either. 
 Reinoculation: Fungi strains were reinoculated onto new plates that all contained the same medium. 
Plates were checked for best growing conditions for the different fungi. PDA was in general the most promising medium and this was chosen as the medium used going forward. Three plates were inoculated for each strain with three-point inoculation. 
Depending on strain, different light and temperature conditions were used in incubation. The conditions were chosen based on knowledge of growth from research: 

</P></DIV></LI><LI><P><I>S. commune</I> (wildtype and <I>ΔSC3</I>): Two plates for 30°C light and one for 27°C light. </P></LI><LI><P><I>P. ostreatus</I>: Two plates for 28°C dark and for for 25°C light. </P></LI><LI><P><I><I>A. oryzae</I></I>: Two plates for 28°C dark and one for 30°C dark.  </P></LI></UL><P><B>3/7/2018</B>
								Boxes: <I><I>A. oryzae</I></I> and <I>S. commune</I> were used for inoculation of boxes. PDA and mycelium with attached agar mixed. Inoculated medium mixed well with saw dust in boxes and incubated at 30°C dark for a week. 
<B>5/7/2018</B>
								Boxes: Pipette boxes inoculated with fungus/hay mixture from Skyttegaarden. 4 boxes inoculated, two with parafilm and 2 with the lid. 
Inoculation of boxes with species of our own. 

<B>6/7/2018</B>
Photos of fungi: Photos were taken of the fungi every day to assess the growth. 
All species show growth although not all plates did. <I><I>A. oryzae</I></I> showed signs of sporulation. 
Boxes: <I>S. commune</I> show growth regardless of medium concentration and volume. However, higher concentration of medium means more growth. 
<I><I>A. oryzae</I></I> boxes moved to 28°C dark. 
<I>S. commune</I> all moved to 30°C light as they seemed to thrive there. 

							</P></DIV><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
								Up until this week, we have had a normal semester. Now all members, of drylab, are able to be working full time in the iGEM project, not counting scheduled personal holidays. This week Mathies worked on Poisson HMM (Hidden Marcow model) and investigating binomial distributions with regards to simulating hyphal growth and branching probabilities. Nicolai is manufacturing/welding growth boxes for the Wetlab group. They need these to be able to make consistently shaped fungi squares for drylab compression testing. Lau and Mathias are investigating other modeling possibilities; how to grow Mycelium on different substrates, and how we, as a group, can enhance our productivity and usefulness for the other subgroups. Hannah collected a lot of data, to be used for more precise modeling parameters, and is working on writing a program that will be able to calculate growth speed using image recognition. Collectively we made a safety assessment and handed for review for us to be able to gain access to proper material characterizations machines at the mechanical department at DTU.

							</P></DIV></LI></DIV></DIV><BUTTON id="week28" class="collapsible">Week 28 (July 9 - July 15)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>9/7/2018</B>
								Pictures of plates were taken everyday to follow growth and sporulation. 
Spore suspension: Spore suspension made from plates from last week following the <A target="_blank" href="https://2018.igem.org/Team:DTU-Denmark/Experiments#sporesuspension">spore suspension protocol</A>. 
Spore suspension ended up with a concentration of 1.69 * 10<SUP>8</SUP> spores/ml. 
<B>13/7/2018</B>
								Boxes: New substrates used: white and brown rice and coffee grounds. Rice are natural substrates for <I><I>A. oryzae</I></I> and coffee grounds are supposed to add to the texture of the mycelia. All substrates autoclaved before inoculation of boxes. Pipette boxes inoculated and incubated at 30°C dark.  
Baking of boxes: The baking or autoclavation of the boxes were to find a way to kill the mycelia, so the boxes could be brought outsite the laboratory. 
Boxes inoculated from Skyttegaarden and saw dust boxes were autoclaved. 
The saw dust boxes were very crumbly regardless of concentration before autoclavation. 
Weighing of boxes: Boxes were weighed everyday to follow the growth of the mycelia and to see if there was a difference over time. 

							</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
								We began the week with a meeting to set the course for the rest of the week. This week Nicolai is on holiday.
On the modeling side of things, the work on the HMM is coming along nicely but it is not clear how it can be related to real life. The model outputs the probabilities of branching for individual hyphae. The bigger picture is not clear for this model. Hannah is working on codon optimization after having dropped the imaging programme. The reason being that the color variation of the sample pictures was too great and writing a program to be able to take this into consideration proved to be too hard. An alternative program was used to do this, called <A target="_blank" href="http://www.robots.ox.ac.uk/~vgg/software/via/">VGG Image Annotator (VIA)</A>, where the radius of all the colonies were manually annotated. The area of each fungal colony was calculated and visualized in plots. Mathias and Lau made good progress in investigating live imaging of Mycelia. The purpose is to nail down real parameters spaces for use in simulations. We handed in the safety assessment to DTU Byg (Mechanical Department) for review/verification and got a short tour of the material lab. Many different people were contacted in regards to material supply and modeling/simulation. More growth boxes were manufactured. 

							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week29" class="collapsible">Week 29 (July 16 - July 22)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>20/7/2018</B>
Experiment: First insertion of genes into a backbone						

The purpose of this experiment is twofold. Firstly, to gain experience in using the basic techniques associated with molecular biology. Secondly, the scientific focus of the experiment is to perform an insertion of the <I>MelA</I> gene () and the CaMV 35S promoter (BBa_K1825004) into seperate backbone for amplification in <I>E. coli</I> This is followed by a miniprep to extract the plasmids.

The DNA delivered by IDT was suspended in EB buffer <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#sporesuspension”">protocol</A>.
Using part 1 and 2 from the BioBrick tutorial (Later adapted together with the <A target="_blank" href="https://2018.igem.org/Team:DTU-Denmark/Experiments#resuspensionofdna">resuspension</A> the parts were digested and ligated with the backbone from <I>RFP</I> (BBa_J04450)


							</P></DIV></LI><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>16/7/2018</B>
Reinoculation of plates: <I><I>A. oryzae</I></I> inoculated on new plates in three point inoculation with spores from spore suspension. So far, inoculation has been done with mycelia from earlier plates. Three plates were incubated as normally in 28°C and one at 4°C to see how the fungus fares at lower temperatures. 						

Boxes: Rice boxes were quite hard after only two days in the incubator, which was viewed as a very good sign. 

<B>17/7/2018</B>
Baking of boxes: After autoclavation the boxes were quite moist and therefore fell apart quite easily. In stead of autoclaving, microwaving or baking normally was suggested to avoid the bricks being too wet. 
Box from Skyttegaarden was put in the microwave. After 10 min it was still wet.  						

Protoplastation and transformation: Protoplastation was initialised according to the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#protoandtransfofasper”">Protoplastation and Tranformation protocol</A>.

<B>19/7/2018</B>
Protoplastation and transformation: Protoplastation was initialised according to the  <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#protoandtransfofasper”">Protoplastation and Tranformation protocol</A>.

<B>20/7/2018</B>


Boxes: Regardless of mycelia and media volume the brown rice inoculated with <I><I>A. oryzae</I></I> were crumbly and the mycelia had only grown on the stop of the brick. 
The white rice boxes (also with <I><I>A. oryzae</I></I>) were different according to volume of liquid media added to the rice. The less media, the sturdier the brick was to pressure from hands. 
Adding glucose to <I>S. commune</I> boxes with saw dust did not make a difference. The brick is still very crumbly. 

Baking of boxes: Rice boxes baked at 90°C and checked every 15 minutes. After the fire 15 minutes the paper surrounding the brick was very wet. 
After discussion a new plan was formed: Rice is cooked when in water and surroundings are very hot, so we try to cook them quickly at high temperatures. Therefore: One rice box is baked at 90°C for two hours and then 200°C for one hour. After that the brick is baked 70°C O/N. 
The other rice brick is only baked at 70°C O/N. Saturday they will be taken out and photographed. See designated photo entry for photos of the bricks. 
Small bricks: 
We have received ice cube trays to make smaller bricks, giving us the opportunity to make more bricks quicker and even different bricks at the same time. 
First small bricks are made with spores and different ratios of saw dust, white rice and coffee grounds to see how they work together. To avoid contamination every other well was inoculated instead of each well. See picture. 
</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
								This week Nicolai has come back. He assessed, based on the team's meetings with DTU Byg last week, that manufacturing metal inoculation boxes for Wetlab would become far too big of a task since it was unearthed that we would need over 100 boxes. It was decided that we order silicone ice-cube trays instead, both for us to focus on other pressing matters and for the consistency of the cubes. He also started work on a mission architecture including a design the brick for us to construct a martian hut. We obtained foam samples through a contact from last week. Hannah and Lau assisted the Myco-lab subgroup since they were struggling with their workload. At the end of the week, Nicolai began working on a <A target="_blank" href="https://2018.igem.org/Team:DTU-Denmark/Hardware">DIY hydraulic press</A>. We weren't going to be able to use the DTU byg machinery before a lab technician would return from his holiday in week 32.
Mathias started researching if there existed a gene regulating the frequency of hyphal fusion in the fungi. This was based on findings from polymer science, which showed that vulcanized rubber showed increased strength if the network of rubber was more interconnected or more elastic if the rubber was less interconnected. This would be possible if we could find a homolog of a known gene that regulates hyphal fusion.


							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week30" class="collapsible">Week 30 (July 23 - July 29)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>23/7/2018</B>
Experiment: First insertion of genes into a backbone						

Competent cells were transformed using the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#adaptedassemblyofcomposite”">transformation protocol</A>. After the transformation, the cells were plated on LB amp plates (which in retrospect is the wrong kind of plates) and left overnight. The plates from Monday 23rd were not successful, as we used amp plates instead of cam plates. The protocol has been rewritten accordingly.
The leftover cells were plated on some new LB cam plates. <B>25/7/2018</B>

Experiment:  Preparation of 2nd round of genes
The purpose of this experiment is to insert the DNA delivered from IDT into plasmids for progapagation and further use. The genes that are to be inserted are; <I>pGpdA</I>, <I>pTrpC</I>, <I>pTrpC-GFP</I> and<I>ScGPa</I>. This will be accomplished as per the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#adaptedassemblyofcomposite”">Adapted assembly protocol</A>.
<B>26/7/2018</B>
New Gene have arrived: CaMV 35S Terminator, <I>TS1</I> holology region, Hygromycin B. These were digested following the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#adaptedassemblyofcomposite”">adapted protocol</A>. The ligation results was transformed and plated on LB cam plates O/N.
 <B>27/7/2018</B>

Experiment:  First insertion of genes into a backbone
The prepared O/N cultures were miniprepped (see <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#QIAprep”">miniprep protocol</A>) in order to have the purified plasmids.

Experiment:  Preparation of 2nd round of genes
The O/N plates produced 3 transformants and a correct positive control, but the 4 older digestions produced no colonies, which suggests that digestions was unsuccesful. Consequently, the digestions, ligation and transformation was redone.  The transformation used 35 µL competent cells instead of the 50 µL the protocol requires. The liquid media was scaled down acordingly.
<B>28/7/2018</B>

Experiment:  Preparation of 2nd round of genes
The plates containing transformants was moved to the fridge.
</P></DIV></LI><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>23/7/2018</B>						

Boxes: New boxes made with different combinations of rice, saw dust and coffee. Tests with different kinds of treated rice initialised: porridge rice cooked, raw or grounded were autoclaved and so were jasime rice, raw (but cleaned), normally cooked, overcooked and grounded. 

<B>24/7/2018</B>
Another spore suspension of <I><I>A. oryzae</I></I> made according to the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#QIAprep”">spore suspension protocol</A>.

<B>25/7/2018</B>
Small bricks: More small bricks made. This time from differently treated rice: porridge rice, normally cooked (1), jasmine rice, overcooked (2) and jasmine rice, normally cooked (3). 
Rice mixed with spore inoculated medium and put into the wells in following pattern:
</P><P>Incubated at 28°C dark. 

<B>26/7/2018</B>
Small bricks: Same procedure for inoculating medium as 25/7. Other rice used:
</P></DIV></LI><LI><P>Ground jasmine rice</P></LI><LI><P>Porridge rice</P></LI><LI><P>Ground porridge rice</P></LI><LI><P>Washing jasmine rice</P></LI><LI><P>Finely ground rice</P></LI><LI><P>10 ml rice </P></LI><P>Added to two trays. First 5 ml inoculated rice was added, in the second inoculated rice was added until full.
<B>27/7/2018</B>
Test of contaminations found in boxes: Contaminated saw dust and PDA plated to ID them. The contaminated PDA might be <I><I>A. oryzae</I></I> and the contaminated saw dust might be <I>S. commune</I>. This is interesting because we might have found a way to get spores from <I>S. commune</I>. 
							</P><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
								This week Hannah and Lau were helping Myco-lab in growing “ice cube fungi”. They are being grown on a various substrates and inoculated at different times and temperatures.
Mathias was on holiday. Nicolai was “fighting” with DTU skylab, as their technicians bailed on appointments, making the progression of the DIY hydraulic press slow and wasteful. Late friday, the small project was completed, only missing some teflon tape to seal the thread. Over the weekend Nicolai made a short video showing the manufacturing process. Mathies planned the data structures for the experimental setup for DTU byg. The purpose is to minimize the necessary test that have to be done, to save time and resources. 


							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week31" class="collapsible">Week 31 (July 30 - August 5)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>30/7/2018</B>
Experiment: Preparation of 2nd round of genes						

O/N cultures have been prepaired. 2 for each plate 

Experiment: Extraction of <I>RFP</I> from the kit						
The purpose of this experiment is to transform the <I>RFP</I> gene (BBa_J04450) from the distribution kit into an DH-5-α compent E.coli and use a miniprep to extract it again. 
Using the transformation (subpart 3) from the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#adaptedassemblyofcomposite”">Adapted assembly protocol</A>, four transformations were made; 1 <I>RFP</I>, 1 negative and 2 positive with (2 µL) <I>A1</I>. The transformants were plated on LB cam plates and cultivated overnight.
<B>31/7/2018</B>

Experiment:  Preparation of 2nd round of genes
Overnight cultures were Miniprepped.
Experiments were performed by Joen, David and Jacob.
This concludes the preparation of 2nd round of genes experiment.
<B>1/8/2018</B>
Experiment: Extraction of <I>RFP</I> from the kit
Two colonies from the <I>RFP</I> transformant plate were picked and placed in a liquid O/N culture.
 <B>2/8/2018</B>
Experiment: Extraction of <I>RFP</I> from the kit
Overnight culture were miniprepped to extract the plasmids.
This concludes the Extraction of <I>RFP</I> from the kit experiment.
Experiment: Gibson assembly of basic toolkit
The purpose of this experiment is to assemble what we can of out basic toolkit using Gibson assembly. The parts to be assembled are; <I>TS1</I>, <I>pTrpC</I>, <I>hph</I>, CaMV 35S terminator and <I>TS2</I>. The Gibson assembly will be performed according to the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#hifiDNAassembly”">Hifi assembly protocol</A>. As the second homology region, <I>TS2</I>, had not arrived at this point, we began assembling the other genes into a composite.
Part 1 of the protocol was done (PCR amplification). Experiments performed by David, Joen and Jacob.</P></DIV></LI><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>30/7/2018</B>						

Reinoculation of plates: Cold <I><I>A. oryzae</I></I> grown to spores. Probably because of the cold incubator being broken. Temperature was around 20°C instead of 4°C. 
Baking of boxes: Ice cube trays from 20/07/2018 photographed and baked. Baked at 70°C for 4 hours and 40 minutes and then photographed. Put bad to bake O/N. 
Before baking: 
</P><P>After 4 hours and 40 minutes:</P><P><B>31/7/2018</B>
Baking of boxes: Ice tray taken out after being baked at 100°C for roughly one hour. 
Small bricks: New ice tray inoculated with spores suspension. Same method as last time. Incubated in fridge at 4°C. 

<B>2/8/2018</B>
Death test: After baking we need to make sure the mycelia in the bricks are actually dead. That's what we're doing here. 
For the large bricks: Tape is touched onto brick to catch potentially alive mycelia or spores and touched onto plate afterwards. 
For the small bricks: Blue inoculation needles are touched onto the brick and inoculated on plates according to marks from 20/07. 
Test of contamination found in boxes: Turned out to be nothing of interest. 

<B>3/8/2018</B>
Baking of boxes: As the large boxes from 20/07 had been unprotected while they were in the office (after their baking) spores from the office may have landed on them and started growing. They are baked again at 100°C for two and a half hours. 
Death test: Another death test in which the small bricks are broken in half and the inside is touched slightly on a PDA plate to see if the inside is dead as well.  </P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
								This week Mathias returned and Hannah went on holiday. The <A target="_blank" href="https://2018.igem.org/Team:DTU-Denmark/Hardware">DIY hydraulic press</A> was completely finished and ready to use. Nicolai restarted the mission architecture and brick design. Mathias reached out to a polymer expert (Mika Torkkeli, Postdoc) to discuss the role of the network structure/interconnectedness in the strength of polymer networks, it was confirmed that the sparse mycelium network could perhaps be analogous to the network structure of vulcanized rubber. But because of the macroscale of the network and the sparseness of the network the effects might be less drastic than those found in rubber. Another challenge was that the only gene that was identified that might increase hyphal fusioning also affected mycelium density negatively. It was deemed to be too uncertain if we could control these effects away and whether or not the effect size would be worth it. 

							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week32" class="collapsible">Week 32 (August 6 - August 12)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>6/8/2018</B>
Experiment: Gibson assembly of basic toolkit						

Adding the homology regions to <I>pTrpC</I>, CaMV Terminator and Melanin for assembly into the basic plasmid after it's contruction. This is done via PCR amplification with primers containing and extra 20 bp upstream from the binding part of the primer.

<B>7/8/2018</B>

Experiment: Gibson assembly of basic toolkit
PCR products was gel-purified and nano-dropped
</P><P>[note: the difference between &quot;top <I>TS2</I>&quot; and &quot;bottom <I>TS2</I>&quot; is that the gel had two bands for <I>TS2</I>] 

The Gibson mix arrived and we followed the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#hifiDNAassembly”">Gibson protocol</A> to assemble the basic plasmid. We assembled two different plasmids. One with TS2u and one with <I>TS2b</I>. The difference is from the gel electroforesis. One band was slightly further down than the other.We did not know which was the right one, so we collected both.
<B>8/8/2018</B>

Experiment: Extraction of <I>amilCP</I> from distribution kit
A liquid O/N culture were prepared from the plated <I>amilCP</I> transformants. Addionally, replating from the leftover media  from 7/8 were performed, as well as a clean streaking of a sucessfull transformant (from the LB Cam plates), as there were a low number of transformanants on the plates.


<B>9/8/2018</B>

Experiment: Gibson assembly of basic toolkit
Only one of the Mix 2 plates had colonies. We will continue to grow these plates. One colony was cleanstreaked and one was incubated in an overnight culture.
We will retry the Gibson assembly. This time we will start with 70 ng backbone and work from there.
We increased the incubation time to ~4 hours to maximize our chances of a successful assembly

Experiment: Extraction of <I>amilCP</I> from distribution kit
The liquid O/N culture was miniprepped and subsequently concentrations were determined by nanodrop. This concludes the Extraction of <I>amilCP</I> from distribution kit experiment.

<B>11/8/2018</B>

Experiment: Gibson assembly of basic toolkit
Gibson assembly products was transformed and plated. Different amounts of cell culture were used in the hopes of getting more colonies. The first batch of gibson products had grown and exhibited a red color, reminiscent of <I>RFP</I>. AN overnight culture was miniprepped.


<B>10/8/2018</B>

Experiment: Investigation of unsuccessful Gibson assemblies
The attempted Gibson assemblies have thus far been unsuccessful yielding only red, <I>RFP</I>-expressing, colonies. This indicates either: i) wrong PCR amplification of the backbone from BBa_E1010 (<I>A1</I> henceforth) or ii) that the assembly has not worked and that left-over BBa_E1010 is still present in the mix. Thus, <I>A1</I> were amplified again according to the PCR . This was then run on a 1% agarose gel together with the unsuccessful Gibson assemblies and BBa_E1010.

Conclusions from the gel: 

</P></DIV></LI><LI><P>The new PCR amplifications seems to have worked producing a clear bands, however some <I>A1</I> seems to be present in the mix still, thus calling for more stringent cutting during gel purification.</P></LI><LI><P>The Gibson assembly have not been sucessfull at assembling the total fragment</P></LI><LI><P><I>A1</I> seems to present in the Gibson assembly - explaining the red transformants</P></LI><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>6/8/2018</B>						

Baking of trays: Tray from 26/7 baked for two hours at 100°C. Both trays from 25/7 and 26/7 baked at 70°C O/N. 
Inoculation of boxes: As the mycelia has not been able to grow all the way to the bottom of the large boxes a hypothesis of air flow being the reason was raised. A box was supplyed with an air tube in the bottom of the box and inoculated as normally. Incubated at 28°C dark. 
<B>7/8/2018</B>
Small bricks: An ice cube tray was inoculated with different ratios of rice and sawdust to see how that affected the texture of the bricks. See the detailed notebook on Tueday 7/8 for further details. 

<B>8/8/2018</B>
Baking of trays: Tray from 25/7: In the bricks made with overcooked jasmine rice a strong and tough air pocket had been formed. It was tough to break by force with tools. This pocket was filled with air. For the porrigde rice it was filled with liquid and much more wet that the overcooked jasmine rice. 
Tray from 26/7: Dry ground rice from first tray seems to have grown well and is dry after baking. Second tray show less growth with many spores. 
All from 25/7 taken out of tray and baked individually because htye were not yet dry. This is attributed to them being made from cooked rice already contained a high amount of water. 

<B>9/8/2018</B>
We did more protoplasts according to the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#protoandtransfofasper”">Protoplastation and Tranformation protocol</A>, they were stored at -80°C.  

<B>10/8/2018</B>
Small bricks: We wanted to test difference rice with sawdust. From earlier test (7/8) it seemed that 50% rice and 75% rice gave the best bricks. These ratios will be used with different kinds of rice. White rice, granulated rice, porridge rice, grounded porridge rice were used and one tray was with 50% and another was with 75%. See detailed notebook for Friday 10/8 for schematic of the trays. Incubated at 28°C dark. 
Death test: Test from 3/8 showed no signs of mycelia growth. 

</P><P>Test from 8/8 showed signs of growth, although not mycelial. Plates put back in incubator for further inspection later. It seems like we are good with the baking. This means that the bricks that needed further drying should be dead as well - especially because they have had even longer in the oven than these tests. 

							</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P><B>6/8/2018</B>
It was decided today that we are going to pick up the mycelium growth simulations, but approaching it differently than the previous attempt involving hidden markov models. It started out with a discussion about how a mycelium develops over time and discussing parameters based on experimental measurements, such as branching frequency and branching extension rates. Work was begun on two complementary growth models: a detailed micro growth model with a focus on hyphal growth and a meso growth model based on a system of partial differential equations describing the change in biomass density as a function of time, nutrients, space and species characteristics.
 Nicolai and Lau collaborated in the sense that the brick design was finalized and Lau started simulating structures constructed using the brick design. Mathias and Nicolai tried to get a hold on the DTU Byg technician who could teach us to use their material properties machines. But he was still not available even though his schedule said otherwise. The first preliminary test was done using the “home built” hydraulic press. It initially showed a rough link between dryness and toughness. The more dry samples were generally more resistant to compression. 2 different kinds of foams were also tested to see if they were comparable to the material properties of the fungi. They were not, since they were actually tougher.



							</P></DIV></LI></UL></DIV></DIV><BUTTON id="week33" class="collapsible">Week 33 (August 13 - August 19)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>13/8/2018</B>
Experiment: Gibson assembly of basic toolkit						

After reviewing the available literature for troubleshooting Gibson assembly we have come to the conclusion that the problem probably arises from trying to assemble more than 5 fragments at once. Thus, we have decided to assemble the plasmid sequentially. Thus, Gibson assembly were performed of the <I>TS1</I> homology regions, the <I>pTrpC</I> promoter and <I>hph</I>, the Hygromycin B resistance gene (This product referred to as TH) Additionally assembly of <I>TS2</I> homology region and the CaMV terminator were performed (this product referred to as CT).

The assembly were performed according to the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#hifiDNAassembly”">Hi-Fi assembly protocol</A>.

<B>14/8/2018</B>

Experiment: Gibson assembly of basic toolkit
Another round of Gibson were performed according to the specifications given 13/8.
The resulting products from both rounds of assembly were amplified by PCR, utilizing primers a forward gibson-primer matching on end of the product and a reverse primer matching the other end, I.E. D0008, the forward primer for <I>TS1</I> and D0009 the reverse primer for <I>hph</I> were used.
The gibson assembled fragments were then run on a 1% agarose gel (the first number specifies round and the second number specifies duplicate):




</P><P>
PCR products have been inserted following the headline
<B>15/8/2018</B>

Experiment: Geleletrophoresis of PCR produts of Gibson assembly
After yesterdays nice looking gelelectrophoresis, we wanted to do a new electrophoresis (replicate):
This time we loaded 20 uL instead of 5 uL to get a sufficient amount of DNA for gel purification

</P><P>
PCR products have been inserted following the headline.
Given how vague the ladder is and how smeared the bands are, we assume the gel to be bad (given that is was also the very last drops of the gel bottle. 
The second gel electrophoresis was run on machine RunOne #8. We loaded 25 uL of DNA.

Based on online discussion we have identified overloading as the most likely problem




After the unsuccessful gelelectrophoresis, we realised that we would run out of PCR product of the 14/8 gibson assembly product after the next gel. For this reason we decided to make another 50 μL of TH1, TH2, CT1 and CT2. Due to the double banding the product, we decided to increased the cycle time to 5 min and 30 seconds (30 seconds is the minimum cycle, where 30 sec should be added for every kb. We added 5 mins).

<B>16/8/2018</B>

Experiment: Gel Electrophoresis of 15/8 PCR products 
Due to the hypothesis that too high DNA concentrations were the cause of smearing in the two previous gel electrophoresis, we decided to reduce the loaded amount to 5 μL

<B>18/8/2018</B><I>amilCP</I> PCR products.
11-12-21-22
</P><P>PCR products have been inserted following the headline
PCR composite 18/8<I>amilCP</I>, <I>GfaA</I>, <I>MelA</I>, A0, <I>hygB</I></P><P>PCR products have been inserted following the headline


<B>19/8/2018</B>

Tried to run gel for purification, unsuccessful, ran out of PCR products.</P></DIV></LI><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>14/8/2018</B>						

Death test: Test from 8/8: no sign of mycelial growth. Any growth present looks like bacteria or yeast. 

</P><P>
Baking of trays: Ice cube tray from 7/8 ready for baking. Put in for baking at 100°C for four hours. Turned down to 70°C for O/N baking. 
Boxes: Pipette boxes with tips for air checked: Only the rice/saw dust mix box had any noteworthy growth around the tips but not much growth in the bottom of the box. 
<B>15/8/2018</B>
Baking of trays: Bricks taken out and put in sterile pipette boxes for storage. Both completely sawdust and completely rice were not coherent. The mixes became more coherent the more rice there was. 
Boxes: Drylab made new boxes.
<B>17/8/2018</B>
Checked pipette boxes and ice trays. Rice + saw dust and rice + coffee should be baked in three days. All of them lacked strength which means no coherent results. 
Boxes: Drylab made new boxes.

</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P><B>13/8/2018</B>
This week Mathias, Hannah, and Mathies were successful in completing some initial growth simulations, in which mycelium grew from one spore, using their new models. After these results, the code was optimized in order to simulate growth from multiple spores and see their progression. A gif, visualizing the growth using the micro model, was made and it was <B>stunning</B>.
Nicolai and Mathias managed to get a hold of the DTU Byg technician and received training in the machinery in order to make really good compressional tests. The compression tests that our foam-like material is not really yielding the type of results we wanted e.g. we were expecting to see a clear “snap”  when the sample was under sufficient compression. We are not seeing any “snap/crack”, only an increasing amount of compressional force, until the machine stage bottoms out.  What we can do is to stop measuring at a given deformation, and look at HOW that acts.  
This is also taken into consideration for the simulation of the “hut design”. Since we have a finished design, it only needs to get realistic inputs. Lau applied physics (gravity, winds etc.) to the 3d model, but had problems with convergence.


</P></DIV></LI></UL></DIV></DIV><BUTTON id="week34" class="collapsible">Week 34 (August 20 - August 26)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>21/8/2018</B>
Experiment: 3A of genes w. camV Terminator						

The following genes, terminator and backbone have been digested: 
</P></DIV></LI><LI><I>MelA</I></LI><LI><I>MelA</I></LI><LI><I>A.oryzae</I></LI><LI><I>GfaA</I></LI><LI><I>amilCP</I></LI><LI><I>hygB</I></LI><LI>The CaMV 35S terminator</LI><LI><I>A1</I> backbone</LI></UL>

Of these, the <I>amilCP</I>, CaMV 35S terminator and <I>A1</I> came as pladsmids. The others were linearized fragments from IDT.

The resulting digestion mix had a concentration of 10 ng/µl. 

<B>22/8/2018</B>

Experiment: 3A of genes w. camV Terminator
Ligation: <I>MelA</I>,A0,<I>GfaA</I>,<I>amilCP</I>,<I>hph</I>.

Transformation: The ligations from yesterday and the ligations from the failed 3A PCR experiment were transformed into <I>E. coli</I> and placed at 37 C overnight.

Colony PCR: In parallel, we tested the colony using the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#colonypcr”">colony PCR protocol</A>. It seemed like diluting the colonies doesn't give good results.


<B>23/8/2018</B>
Experiment: 3A of genes w. camV Terminator:
The transformants was a success.

Colony PCR: 3 colonies from each plate was suspended in water. A small fraction of this suspension was used for a liquid overnight culture, that can be used for a miniprep. The rest was heat-treated for 10 min and used for a colony PCR afterwards.

The results show that the ligation products have the correct length. 

Experiment: Second round of colony PCR (multicolony PCR)
Explanation of procedure. 5 transformants plated on a new plate - and further transferred to the same tube. 

Two PCR reactions - one using VR/VF primers, the other using [Gene]_fw/CaMV_Term_bw primers.

Backup experiments: As a backup, we digested <I>MelA</I>, <I>MelA</I> - A.oryzae, <I>hph</I> and GfaA and prepaired the ligation.

New experiment: 3A of <I>amilCP</I>+CaMV Terminator with. promoters <I>amilCP</I> digestion: Digestion of: <I>pGpdA</I>, <I>pTrpC</I>, <I>PcamV</I>, <I>amilCP</I> and <I>A1</I><I>amilCP</I> ligation: Ligated with <I>pGpdA</I>, <I>pTrpC</I> and <I>PcamV</I><B>24/8/2018</B>

Experiment: 3A of genes w. camV Terminator 
Created gel of multicolony PCR. In this experiment the forward sequencing primers were used together with the backward sequencing primer of CaMV. It was conducted to test whether a multicolony PCR approach would work.

Backup Experiments: The backup genes were ligated and transformed into <I>E. coli</I>. The colonies were plated and left overnight.



Delivered assembly: We have recieved an assembly from IDT cocsisting of '<I>pTrpC-HygB-CaMV</I>- 35S term'. This assembly was digested and ligated to be transformed tomorrow. When the gene is ready and amplified, it can be handed over to Mycolab for transformation into <I>A. oryzae</I> and <I>G. lucidum</I>.


Experiment: 3A of <I>amilCP</I>+CaMV Terminator with. promoters
Assembly of <I>amilCP</I> and promoters. The successful ligation and colony PCR of the '<I>amilCP</I>-CaMV 35S term' assembly was digested and ligated with 3 different promoters: '<I>pGpdA</I>', '<I>pTrpC</I>' and '<I>pCaMV 35S</I>'. The ligation will be transformed tomorrow



<B>25/8/2018</B>
Experiment: 3A of <I>amilCP</I>+CaMV Terminator
Colony PCR of backup: The colony PCR was not successful. It is not clear why the gel showed no significant answers. The problems could include a failed ligation, failed PCR, failed gel or some other factor.

PCR amplification of delivered assembly: The delivered assembly was amplified with verification primers. The product has run on a gel and the results was mediocre (at best). The amplification is probably a success, but it cannot be confirmed from the gel. 

Experiment: 3A of <I>amilCP</I>+CaMV Terminator with. promoters
Tranformation: The <I>amilCP</I> + promoter assemblies and the delicered assembly has been transformed and plated on cam plates





<B>26/8/2018</B>

Experiment: 3A of <I>amilCP</I>+CaMV Terminator with. promoters
The different transformants (<I>amilCP</I> + promoter and <I>hph</I> marker) were colony PCR'ed.</DIV><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>20/8/2018</B>						

Baking of bricks: Bricks baked last weeked seemed to still be alive; they were put back into the oven at 70°C. 
Baking the pipette box of rice + saw dust; mycelia can be seen everywhere. It has a strong and spongy strucutre so far. O/N 70°C. 
Spore suspension: A new spore suspension was made from plates from 060818. 
<B>21/8/2018</B>
Protoplastation and transformation: Inoculated plates containing Hygromycin B to check if the concentration of Hygromycin B is enough too inhibit growth. 
Baking of bricks: Small bricks taken out and put back in containers. Large brick put back for more baking. Small part of large brick taken out. 
Reinoculation of plates: New plates were inoculated and incubated at 4°C to assess growth at low temperatures. 
Small bricks: Drylab made more bricks. 
<B>22/8/2018</B>
Baking of trays: Drylab baked trays. 
Protoplastation and transformation: Hygromycin B plates available will not work. <I><I>A. oryzae</I></I> not inhibited by hygromycin B. We try different approaches.
<B>23/8/2018</B>
Protoplastation and transformation: We try NTC as selection antibiotic instead. We try inoculating plates with different concentrations: 100 µg/ml, 200µg/ml, 400µg/ml, 600µg/ml and 800µg/ml. 
Could also try an in-house auxotroph of <I><I>A. oryzae</I></I>. 
<B>24/8/2018</B>
Protoplastation and transformation: Growth on all plates with NTC although the higher concentration of NTC in media, the slower growth rate. <I><I>Ganoderma</I></I> received from Ecovative. 
</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P><B>20/8/2018</B>
Great steps have been taken since last week, where we now have an approach to estimating the density of the mycelium. 
Still trouble with 3d convergence.



<B>21/8/2018</B>
Try to create much simpler model to just try out the software, now have trouble visualizing the deformation.
<B>24/8/2018</B>
Used the knowledge and confidence from simpler model to locate, that the error was in the visualization. Successfully created a test video, of a combination of 6 bricks(see drylab folder), created a design with round edges to allow rotation


</P></DIV></LI></DIV></DIV><BUTTON id="week35" class="collapsible">Week 35 (August 27 - September 2)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>27/8/2018</B>
Experiment: 3A of genes w. camV Terminator						

The colony PCR products was examined by gel elecroforesis and the following <I>amilCP</I> assemblies was identified as successful:
G3 (or G4 &amp; G7), C3, C5
Furthermore, we chose H2 and H4 to be an overnight culture. We chose G8 as a control.

</P><P style="text-align:center; font-size:14px;"> GelDoc 2018-08-27 13hr 06min - C1 -C2 -C3-C4-C5-C6-C7-C8-C9-C10-H1-H2-H3-H4-NEG </P><P style="text-align:center; font-size:14px;"> GelDoc 2018-08-27 G1-G2-G3-G4-G5-G6-G7-G8-G9-G10 </P><P style="text-align:center; font-size:14px;"> GelDoc 2018-08-27 T1-T2-T3-T4-T5-T6-T7-T8-T9-T10 </P><P>

The colonies were cultivated further in a liquid overnight cultre for miniprepping tomorrow. Lastly, we redid the gel for the <I>hph</I> selection marker. From this, it seems like H2 is the most viable selection marker.

</P><P style="text-align:center; font-size:14px;"> GelDoc 2018-08-27 16hr 13min H1 - H2 - H3 - H4 - Neg </P><P>

Based on these gels, we isolated specific cultures. 

It seems like the backup digestion and ligation does not work. We will therefore redo that experiment, this time with linear inserts. 
We think that if the inserts excist as plasmids, there is a posibility that they can aquire only the terminator and not the combined ligation. Therefore, only linear fragments will be used. This ensures that a colony is either;
1) red and contains the <I>RFP</I> gene or
2) white and contains the correctly assembled plasmid.

<B>28/8/2018</B>
Experiment: 3A assembly of CaMV terminator onto genes of interest 
The CaMV terminator was digested as downstream part, while the genes of interest, namely: GfaA, <I>MelA</I> and <I>MelA</I> optimized for <I><I>A. oryzae</I></I> were digested as upstream part.
They were inserted into <I>pSC1C3</I> (the <I>RFP</I> expressing variant was used for ease of identifying background transformants).



Experiment: 3A assembly of <I>hph</I> selection marker and <I>amilCP</I> constructs (containing) 
The <I>amilCP</I> constructs containing promoter and terminator as determined by the cPCR from 26-27/8 were digested as downstream part, while the <I>hph</I> selection marker from IDT were digested as upstream part. <I>pSC1C3</I> was used as backbone.

<B>29/8/2018</B>
Experiment: 3A of genes w. camV Terminator:

Based on the Gel Tuesday, we isolated the C3, C5, H2, H4, G3, G4, G7 and G8 cultures, I wanted to confirm the construct we have made. Given the differing migration lengths from the 27/8 gel electrophoresis, we are currrently not sure whether it is the C1 or the C3 that is the successful assembly of P. CaMV-<I>amilCP</I>-CaMV term or whether any of them actually are. Likewise for G3, G4 and G7. 

In this experiment I am to verify the sequences by amplifying the gene regions overlapping two adjacent gene fragments.Tomorrow the products from this PCR will be run on a gel, to hopefully either verify or disprove the existance of expxected gene fragments. 

The PCR used a ONETAQ polymerase. The PCR was run with an anealing temperature of 49C and a elongation time of 1 min.

<B>30/8/2018</B>

Experiment: 3A of genes w. camV Terminator 
Today we performed gel electrophoresis on the 29/9 verification PCR.

</P><P style="text-align:center; font-size:14px;"> G31-G32-G41-G42-G71-G72-G81-G82-C31-C32-C51-C52 </P><P style="text-align:center; font-size:14px;"> H21-H22-H41-H42-NEG3-NEG4 </P><P>

Experiment: Gibson assembly of gene cassettes 
PCR extention with homology regions to allow for Gibson assembly were performed on the genes of interest. This was done using specific primers and Q5 PCR. 

After the PCR the products were run on a gel. The lanes corresponds to the above chart in terms of product (So lane 1 contaings <I>pTrpC</I> with homology regions for <I>MelA</I> and so forth):

</P><P><B>31/8/2018</B>
The following genes, with their respective Gibson overhangs, were purified using the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#gelextraction”">gel extraction protocol</A> as they were the only sucessfull PCR amplified genes:</P></DIV></LI><LI><I>pTrpC</I> (<I>MelA</I>)</LI><LI><I>pTrpC</I> (GfaA)</LI><LI><I>MelA</I></LI><LI><I>MelA</I> (<I><I>A. oryzae</I></I>)</LI><LI>GfaA</LI><LI> CaMV terminator (<I>MelA</I>)</LI><LI>CaMV terminator (GfaA)</LI></UL>

Experiment: 3A assembly of <I>hph</I> selection marker and <I>amilCP</I> constructs (containing)
Transformants containing the 3A assembled <I>hph</I> selection marker were screened for the correct inserts by cPCR with the verification primers, VF2 and VR. 5 Colonies were chosen from each of the CaMV promoter containing transformants and 8 colonies were chosen from the <I>pGpdA</I> containin transformants, due to higher background (estimated by amount of <I>RFP</I>-expressing cells). A one-taq PCR amplification were then performed and the products run on a gel:

<P>

Based on the gel none of the screened mutants contained the right insert, which should be about 3200 bp, though g34 seem to containg some sort of additional assembly. Thus, this should be investigated. 

</P></DIV><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>27/8/2018</B>						

Protoplastation and transformation
Transformation media prepared. See specifications in the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#mediaforfungi”">protocol for fungi media</A>. 

Statistical study: Tray 2, 7, 8, 6 were placed into the oven.
<B>28/8/2018</B>
Statistical study: Tray 6 and 8 were removed from oven. 

Protoplastation and transformation
Plates with different concentrations of NTC prepared: 200, 400 and 800 and a positive. <I>E. coli</I> O/N cultures prepared with strains for MiniPrep. 

<B>29/8/2018</B>
Protoplastation and transformation: Mini-prepped <I>E. coli</I> cultures from yesterday according to  the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#QIAprep”"> QIAprep Spin Miniprep protocol</A> and nanodropped the minipreps. 

<B>30/8/2018</B>
Protoplastation and transformation: Transforming mini-prep into protoplasts to test protoplasts and miniprep. 5 transformations in 5 tubes. Each plated onto different plates with different NTC concentrations. Using protoplasts from 9/8. 

Statistical study: Tray 1 and trayy 5 placed in oven at 70°C. Tray 2 and 6 placed in oven at 100°C.
</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
The investigation for design of the dome have now been expanded into investigation of internal structures. That means Lau has also decided on what kind of brick forms should be used, as the result of his work revealed that he had to have at least two types of bricks designs. In regards of simulation microscopic mycelium growth, this week was spent investigating how to make the branch extension dependent on growth kinetics of the specific fungi. 
The mechanical compression tests were carried out, where Mathias and Mathies have been crushing many bricks. The mycelium bricks have a lot of moisture of them which influenced the bricks quite a lot in terms of pressure resistance. 

</P></DIV></LI></DIV></DIV><BUTTON id="week36" class="collapsible">Week 36 (September 3 - September 9)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>3/9/2018</B>
Experiment: Verification of PCR amplified camV terminator						

The camV. T. PCR amplificants were run on a gel to verify that the amplification had been successful. It seems that the amplificants seem successful.

</P><P>


The colonies were cultivated further in a liquid overnight cultre for miniprepping tomorrow. Lastly, we redid the gel for the <I>hph</I> selection marker. From this, it seems like H2 is the most viable selection marker.

</P><P>

Transformation of all the ligates from today were transformed into Dh5-α-competent cells following the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#proto3aassembly”">3A transformation protocol</A>

Experiment: Gibson Assembly of gene casettes. 
Since the previous PCR amplification with Gibson primers didn't yield any backbone to assemble into, as well as lacking some other amplifications, these were attempted again. This time both plasmid-inserted and linearized PCR amplification was performed. As can be seen from the gel below the plasmid amplifed backbone (PCR product 4 in the freezer) yielded an amplfied backbone.


The products ampflied are <I>pTrpC</I>-<I>MelA</I> (<I><I>A. oryzae</I></I>), <I>GfaA</I>, <I>CaMV-GfaA</I>, backbone from plasmid and backbone. The last three are from an investigation of a construct from the <I>hph</I>-selection marker 3a assembly:

</P><P><B>4/9/2018</B>
Experiment: Gibson Assembly of gene cassettes

As seen on the gel picture from Monday (September 3rd), we have two clear bands. These were gel extracted using the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#gelextraction”">protocol</A>. The resulting DNA concentrations are 38.1 and 38.6 ng/µl respectivly for band.

Experiment: Colony PCR
5 successful transformants from each plate was extracted and used for a colony PCR following the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#colonypcr”">colony PCR protocol</A>. We used the verification primers for this PCR to examine if length of the transformed constructs. This will be used to verify the digestion and ligation. In parallel, we experimented with an alternative technique for colony PCR. We scraped multiple colonies from the same plate off and treated it as if it was a colony. By doing it this way, we hope that we can screen plates faster, as we are looking for a band of the correct size in the gel.  A possible limitation of using this technique with the verification primers is that other plasmids, such as the <I>RFP</I> will show bright bands, while our gene of interest might be very dim. This can be avoided by using other primers, but we will discuss this at a later point.

<B>5/9/2018</B>
Experiment: Gel results of colony PCR
The colony PCR from 4/9 was run on a gel and gave rather dissapointing results

</P><P style="text-align:center; font-size:14px;"> GelDoc 2018-09-05 15hr 10min Colony PCR 11- 41 </P><P style="text-align:center; font-size:14px;"> GelDoc 2018-09-05 15hr 12min Colony PCR 42 - 72 </P><P style="text-align:center; font-size:14px;"> GelDoc 2018-09-05 15hr 14min Colony PCR 73 - 95 - Pos - Neg1 - Neg2 </P><P style="text-align:center; font-size:14px;"> GelDoc 2018-09-05 15hr 08min Colony PCR 1-9 </P><P>

Troubleshooting:</P></DIV></LI><LI>Premixing a 'master-mix' might be problematic?</LI><LI>The annealing temperature was too high</LI><LI>The extension time too long</LI><LI>Too long amplification sequences </LI><LI>An error was made mixing the master mixes for the colony PCR</LI><LI>The lysated contain insuficient DNA or otherwise make for a bad template</LI></UL>

PCR to test the lysates:
We have made a small PCR to test the viability of the lysates as template. In this PCR we have the colony lysates 1,2,3, a positive control using <I>RFP</I> (0023) and a negative control. We have reduced the annealing temperature to 49*C and otherwise we have tried to keep the parameters the same. Should we get good results, we will retry the colony PCR using the lysates and the new conditions. Should the results fail us, we can consider to try to use Q5 PCR on the lysates or make a new batch.

<B>6/9/2018</B>

Gel results from the lysate PCR test: The PCR results from the experiment on the 5/9 were run on a gel. We can see that there are some results, which suggests the lysates are not the problem. This could mean that it was the annealing temperature which was too high, that was causing the problems.

<B>9/9/2018</B>
Experiment: Colony PCR of transformants.
A colony PCR of the Gibson transformants was prepared, with 5 colonies picked from each plate. This was done early in the day, which may not have allowed the <I>RFP</I>-expressing tranformants to show yet, thus increasing risk of picking a wrong transformant. A one Taq-PCR was performed with verificiation primers at 53 degrees celcius.

</DIV><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>3/9/2018</B>						

Protoplastation and transformation
Transformation plates had already sporulated. They were replica plated onto minimum media plates each containing the same NTC concentration as the original transformation plate. 
<B>5/9/2018</B>
Protoplastation and transformation
Replicate plates phtographed and put in fridge. <I><I>A. oryzae</I></I> plates incubated at 20°C. 

<B>7/9/2018</B>
Protoplastation and transformation<I>Ganoderma resinaceum</I> protoplastation experiments started according to <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol”">protocol</A>. For incubation until Wednesday 12/9. 20°C <I><I>A. oryzae</I></I> plates checked. There is growth but it is very little. 
<B>9/9/2018</B>
Protoplastation and transformation.
Transformation of colour genes according to <A href="https://2018.igem.org/Team:DTU-Denmark/Experiments#protoandtransfofasper">protocol</A>. 

</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
This week the statistical DOE implemented into the compressive strength experiments by Mathias and Mathies. The pilot study was conducted for a screening of multiple substrates in the fungal lab by Mathias and Mathias. One conclusion was that the samples should definitely be dried for a longer time in the oven. Probably found explanation of the math behind geodesic domes.


</P></DIV></LI></DIV></DIV><BUTTON id="week37" class="collapsible">Week 37 (September 10 - September 16)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>10/9/2018</B>						

Experiment: Gibson Assembly of gene casettes
After two days in the cabinet incubator, the red colonies (<I>RFP</I>) were easily distinguishable from the white colonies (maybe succesful transformants). In total 12 colonies were to be found of the 3 agar plates, and these were isolated for a second round of colony PCR on the gibson transformants, Gel of 1st and 2nd round of colony PCT:

</P><P>

The colony PCR of the white colonies from the 10th of september (today) all seem to be succesful gibson transformants. Colony 1-4 have been prepared as an O/N culture for miniprep tomorrow.

<B>11/9/2018</B>
Experiment: Gibson Assembly of gene cassettes.
Colony 1-4 had been grown O/N and 0.5 ml was transfered to a glycerol stock and placed at -80 C. The rest of the O/N culture was miniprepped. The resulting concentration of plasmids can be seen here. Plasmid 0063 has been sent to sequencing.


Experiment: 3A assembly of composite parts. 
We PCR amplified sequnces of 0056, 0057 and 0058 to increase their concentration for sequencing. In addition, we have prepaired an O/N culture of 0056, 0057 and 0058, but it is not certain we will need them. The plasmids have been sent to sequencing to be verified. 

</P><P><B>12/9/2018</B>
Experiment: 3A assembly of NTC resistance gene
As the <I>hph</I> selection marker may not be a viable option in <I><I>A. oryzae</I></I>, we have decided to trying NTC as a selection marker. The <I>Nat1</I> gene has been PCR amplified to contain the BioBrick prefix and suffix in a previous experiment. Now we will assemble it into a plasmid with a promoter and terminator. As with a lot of the consructs, we are testing different promoters and we will also be doing that here.

Experiment: Digestion.
We started with assembling the <I>Nat1</I> gene together with <I>PcamV</I> (0028), <I>pTrpC</I> (0018) or <I>pGpdA</I> (0008). We are using <I>RFP</I> (0024) as the backbone. The reason for starting with the promoter, is that this unfinished construct could have some limited use in the finished product, while the gene and terminator has no direct application.


Experiment: Gibson Overhang Extension PCR
Homology regions for Gibson assembly was added via PCR extensions to specific fragments. This time the 5x round of the PCR were done at the intial annealing temperature of the primers and afterwards the annealing temperature was increased to account for the elongated DNA molecules present which can then bind the full primer (with the extension on). The products was run on a 1% agarose gel producing the following bands. Thus, all fragments was succesfully extended except for <I>GfaA</I>, which continues to be a problem. This could very well indicate a need for redesign of primers.

</P></DIV></LI><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>10/9/2018</B>						

Protoplastation and transformation.
Performed another transformation of colour genes. The <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#protoandtransfofasper”">protocol for protoplastation of the <I><I>A. oryzae</I></I> was initialised</A>. 

<B>12/9/2018</B>
Protoplastation and transformation 
New spore suspension. <I><I>A. oryzae</I></I> protoplastation according to <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#protoandtransfofasper”">protocol</A>. Next <I>Ganoderma</I> protoplast step taken. There was a human error and the samples were centrifuged instead of vortexed. 

<B>14/9/2018</B>
Protoplastation and transformation
Osmotic buffer and sorbitol solution made and autoclaved. This is the day that the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol”"><I>Ganoderma</I> protocol</A> was attempted changed for the first time. With no Driselase available we had to think of other ways to protoplast <I>Ganoderma</I>. After consulting Ecovative we tried the first time with higher concentrations of Glucanex  instead of both Glucanex and Driselase as originally in the protocol. 
</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
The sample from the pilot study was burning at different time intervals according to the optimal DOE. the samples were after the burning carried over to byg where the hydraulic testing of the cubes were commenced. Following the test resulted in the replicated DOE which was implemented into the DOE script and an optimal design was found which was carried out the same week. 
Doing the pilot sampling study, a huge part of the data (the fungus) were mushy and thus a lot of substrates were left out as optimal conditions. Another interesting results were that fungal cubes gets more brittle at higher temperatures. 
The first thoughts of implementing the Ganodermerma as a second brick option was considered due to the results of the pilot study.



</P></DIV></LI></UL></DIV></DIV><BUTTON id="week38" class="collapsible">Week 38 (September 17 - September 23)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>18/9/2018</B>						

Experiment: 3A assembly of NTC resistance gene
The plasmids containing a promoter and the <I>Nat1</I> gene were PCR amplified using verification primers and run on a gel to ensure a correct ligation.

</P><P>

These results look promising and we will continue with these plasmids. We are using the first of the four plasmids from each promoter construct.
The digestions were ligated and will be ready for transformation tomorrow.


Experiment: Gibson Assembly.
The Gibson products from 17/9 were transformed, plated and incubated at 37 C O/N.

Experiment: Existential Crisis
Have you ever considered that the parking lots built in your lifetime will still be there, when you are long gone and forgotten?

<B>19/9/2018</B>
Experiment:3A assembly of NTC resistance gene
The ligated products were transformed into E.coli and plated on LB cam plates.

Experiment: Gibson Assembly
The plated colonies may have been contaminated. We are solving this in two ways. We have redone the transformaiton and plated everything again We have spred the liquid culture on another set of plates.

Experiment: Gel of PCR amplified gene fragments for Gibson.
The PCR amplified gene fragments  for gibson from yesterday (18/9) were run on a gel. The resulting gel:

</P><P>

Given this result, samples 3, 4 &amp; 5 (The three <I>pTrpC</I>s with the right prefix) were isolated for gel extraction (performed 21/9).

Experiment: Preparation of <I>amilCP</I> for mycolab
For the transformaiton mycolab needs a steady supply of <I>amilCP</I>. We will provide that by producing multiple O/N cultures and puryfing the plasmids. We are also interested in discovering if the transformation is easier with linear DNA or with plasmids, so we will provide both. Multiple O/N cultures of C3 and G3 (See week 35) have been made from a glycerol stock.

<B>21/9/2018</B>
Experiment: 3A assembly of NTC resistance gen and Gibson Assembly
The colony PCR products were run on a gel. The results were rather disappointing.

</P><P>

Test of the colony PCR lysates: Due to the dissapointing results from the previous PCR, we wanted to test whehter the lysates were the problem, for this reason we selected the following lysates as a probe: F1 (<I>RFP</I> contaminated?), F2, C1, C2 and a positive control POS (<I>RFP</I> plasmid 0024).

</P><P><B>22/9/2018</B>
Experiment: 3A assembly of NTC resistance gene and Gibson Assembly. </P><P>

The only interesting results from this Gel electrophoresis were C4,D1-D4,E2-E4. 
The cultures for C4, (D2, D3?), E2, E3 where set to incubate O/N.

</P></DIV></LI><LI>C: <I>pCaMV+Nat1+CamC Term</I></LI><LI>D: <I>pTrpC+Nat1+CamC Term</I></LI><LI>E: <I>pGpdA+Nat1+CamC Term</I></LI></UL>
Given the migration patterns on the gel, I suspect that the the C,D,E constructs are too short to have both the promoter, the <I>NAT1</I> gene and the terminator. This can be investigated from the O/N cultures.

Concluding remarks: It seems that this round of gibson transformations was unsuccessful.


Experiment: PCR on the <I><I>A. oryzae</I></I> Lysates.
The PCR on the <I><I>A. oryzae</I></I> lysates didn't yield any good results.

<B>23/9/2018</B>
Experiment: Tranformation of <I><I>A. oryzae</I></I> with <I>amilCP</I>.<I><I>A. oryzae</I></I> was tranformed with <I>amilCP</I> cassettes, either containing the pCaMV promoter or the <I>pGpdA</I> promoter. Both cirucular and linear versions of the <I>PcamV</I> promoter constructs were used. 4 replicates of each was made and placed at 28 degrees celicus for growth.</DIV><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>17/9/2018</B>						

Protoplastation and transformation<I>Ganoderma</I> protoplastation: No Driselase found anywhere in the lab so trying out with different concentraions of Glucanex:</P></DIV></LI><LI>20 mg/ml</LI><LI>30 mg/ml</LI><LI>40 mg/ml</LI>

The 40 mg/ml is the same concentration as in the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#protoandtransfofasper”"><I>Aspergillus oryzae</I> protocol</A> which is what prior knowledge we had to draw from. Transformation with chromoprotein: No colour colonies which would indicate that there has been no transformation. But! The transformed spores are very green compared to the yellow-green spores of the wildtype which indicates some change. 

<B>18/9/2018</B>
Protoplastation and transformation
First part of the process for isolating protoplast done. Tubes labelle 1, 2 and 3 put in the fridge for finishing up the next day. 

<B>19/9/2018</B>
Protoplastation and transformation
Samples were centrifuged but no pellet was isolated. Samples were thrown out and the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#protoandtransfofasper”">protoplasting and transformation protocol</A> was replanned.

<B>20/9/2018</B> 
Protoplastation and transformation
Inoculated 15 PDA plates with <I>A. oryzae</I>. <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#protoandtransfofasper”">Ecovative <I>Ganoderma</I> protoplastation protocol</A> restarted. Next step is on Tuesday. 

<B>21/9/2018</B>
Protoplastation and transformation
Transformation of <I>PcamV</I> and PgdgA of <I>A. oryzae</I>. 

<B>23/9/2018</B>
Protoplastation and transformation
Transformation of <I>A. oryzae</I> with circular and linear fragments. Used 10 µl for all transformations. 
</DIV><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
Confirmed source found earlier was an explanation of the math behind geodesic domes. We will use the same math for calculating the size and placement of the pieces of the structure that should be made from fungi. Turns out it geodesic domes are based on a icosahedron, split into smaller triangles with corner points projected onto a circle, to ensure a more circular structure which ensures more equal distribution of structural load.
Grow <I>Ganoderma</I> from the Ecovative kit  To crush. 
Mathias spoke to Risø. They are going to irradiate them and see how resistant(growth rate) the spores are to gamma radiation. we are going to get a certificate as proof of all the technicalities of the test and results. they’ll test three strands. <I>Ganoderma</I>, Oreacce.
we had a meeting to determine the deadlines and assigned names to each wiki part.
Results from the replicated experiment of the best substrates from the pilot study came in and seemingly the lower temperatures of design was too low and so these were changed. Also seemingly pure rice gave the best results with regards to compression strength. The design was carried out once more with the new lower temperature. Also the <I>Ganoderma</I> experiments started using the same DOE as the <I>Aspergillus</I> fungi. 


</P></DIV></LI></DIV><BUTTON id="week39" class="collapsible">Week 39 (September 24 - September 30)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>24/9/2018</B>						
Experiment: PCR amplification of <I>amilCP</I> constructs (for transformation).
Anealing temperature of verification primers: 53 Celsius. We are amplifying <I>pGdp-amilCP-CaMV</I> and <I>pCaMV-amilCP-CaMV</I> which have a length of ~1600 bases and 1300 bases. We are running the PCR triplets. We are using 1 µl og DNA for each reaction. In total, we have 8 <I>amilCP</I> reactions, a positive with <I>RFP</I> and a negative.

Experiment: Prepartion of <I>RFP</I> in <I>pSCB1A3</I>.<I>RFP</I> was aquired from the kit and transformed into <I>E. coli</I> and plated on ampicillin-containing plates.

Experiment: Miniprep of G3 and C3.
4 cultures of each plasmid was miniprepped. They still need to be nano-dropped.<B>25/9/2018&gt;</B>
Experiment: Melanin 3A assembly.
We are very interested in testing our <I>MelA</I> and <I>MelA</I>-A.oryzae constructs work in a fungi. We will therefore try to assemble them using 3A assembly again.

Experiment: Purification of promoters for E.coli
BBa_J23119 is a strong constittutive promoter for E.coli from the Anderson promoter library. We want a promoter for E.coli to quickly verify that our <I>MelA</I> genes. We will be using the brick in two different backbones, <I>pSB1C3</I> and <I>pSB1A2</I>. These part are found on the distribution kit.

The following parts have been transformed into E.coli and plated on their respective plates:

</P></DIV></LI><LI>BBa_J23119 - <I>pSB1C3</I>: Plate 3, well 17O</LI><LI>BBa_J23119 - <I>pSB1A2</I>: Plate 4, well 17B</LI></UL><B>26/9/2018</B>
Experiment: Purification of promoters for E.coli.
A 17O culture has been transfered to LB cam liquid media and is being incubated O/N. The 17B transformants where wrongfully plates (as these contain the Amp resistance), but a new LB amp plate was prepared and the 17E cultures replated and let to incubate O/N.

Experiment: Melanin 3A assembly.
The amp resitance backbone pSB1A3 where minipreped and digested as backbone for the assembly.<B>27/9/2018</B>
Experiment: Melanin 3A assembly.
The ligations were transformed into E.coli and plated on LB amp plates. <I>RFP</I> (0025) was used for as a positive control and was plated on a LB cam plate.<B>28/9/2018</B>
Experiment: Melanin 3A assembly.Onetaq protocol using verification primers and a annealing time of 3 minutes per cylce.<B>30/9/2018</B>
Experiment: Gibson fragments.
The fragments intended for Gibson assembly showed terrible results on the gel. We need to retry the PCR to get usable results.

Experiment: Melanin 3A assembly.
Some of the colonies were transfered to an O/N culture and we can hopefully extract successful plasmids tomorrow.



</DIV><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>24/9/2018</B>						
Protoplastation and transformation.
Samples from transformatinos (21/09/18) taken for colony PCR. See synthlab notebook from this week for rest of this. <I>A. oryzae</I> protoplastation initialized. <B>25/9/2018</B>
Protoplastation and transformation.
No PDB available, so the <A href="https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol">Ganoderma protocol</A> was postponed to tomorrow to make the PDB. <B>26/9/2018</B>
Protoplastation and transformation.
Funnels prepared for protoplastation of <I>A. oryzae</I>. Digestion of <I>A. oryzae</I> was postponed to tomorrow. So the first couple of steps were done and then put in the fridge to be finished tomorrow. First steps of <A href="https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol">Ganoderma protocol</A> carried out as well.<B>27/9/2018</B><A href="https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol">Ganoderma protocol</A> continued. This time with varying concentrations of Glucanex, shaking speeds during digestion and digestion times. For details on this, see the protocol for protoplastation of <I>Ganoderma</I>.
No cell strainers are available so the samples that were only given 2-3 hours of digestion were not filtered correctly. Henceforth sterile funnels with miracloth were used instead. 
We continued working on the <A href="https://2018.igem.org/Team:DTU-Denmark/Experiments#protoandtransfofasper">protoplastation and transformation of A. oryzae</A>. <B>28/9/2018</B><A href="https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol"><I>Ganoderma</I> protoplastation</A> of samples that were given 19-24 hours of digestion continued. Following the protocol, all samples (Both 8 19-24 hour and 8 2-3 hour samples) were put into the fridge until Sunday for finalization of the protocol. <B>30/9/2018</B>
Finalised <A href="https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol">Ganoderma protocol</A> by centrifuging. No pellet, so no protoplast. Back to drawing board. 
</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
Succeeded in visualising that the corner points of the pieces placement projected onto a circle was calculated correctly. Growth model: A model of the fungi biomass. The top is going a bit down but that’s okay.  Risø gave three days of gamma radiation while fungi where grown. They grew as normal under radiation. Mathies have been creating the last bricks and the last tests on those will happen next week.<I>Ganoderma</I> tested as well. 


</P></DIV></LI></DIV><BUTTON id="week40" class="collapsible">Week 40 (October 1 - October 7)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>1/10/2018</B>						
Experiment: Amplified Insert Assembly of <I>MelA</I>.
The promoter-<I>MelA</I> assemblies seem to be successfully assembled, so we will try to add a terminator to the constructs.

We are using 0113-0116 as the <I>MelA</I> parts and 0014 was used as the terminator and backbone. We adapted the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#proto3aassembly”">3A assembly protocol</A> to just using digestion of the parts and then ligating them. If this doesn't work, we may have to try using PCR amplification to produce enough insert.The products were ligated using the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#proto3aassembly”">3A protocol</A> with the minor adjustment of just using two parts instead of three. The ligated DNA were then transformed into E.coli and plated 150 µl on LB cam plates.


Experiment: PCR verification of <I>NAT1</I> and Beta-tubulin in <I>A. oryzae</I>.
A Onetaq PCR was performed on the <I>A. oryzae</I> lysates PC1, PC2, PC3, PC4, PG1, PG2, PG3 and PG4 to verify the presence of <I>NAT1</I> and Beta-tubulin. Each of the lysate samples were amplified using the Seq_<I>NAT1</I>_in_fw/Seq_<I>NAT1</I>_in_bw or seqP_betatub_in_fw/seqP_betatub_in_bw primers. The resulting gels were dissapointing and the lysates declared unsuccesful End of Experiment: PCR verification of <I>NAT1</I> and Beta-tubulin in <I>A. oryzae</I>.

Experiment: Sequencing of biobricks.
Sequencing results:
</P></DIV></LI><LI>Confirmed the 0092,0093,0094 plasmids.</LI><LI>Showed that 0101,0102,0103 does not contain <I>NAT1</I> nor terminator. Only the promoters were present in the plasmids.</LI></UL><B>3/10/2018</B>
Experiment: Amplified Insert Assembly of <I>MelA</I>.
The plated cultures have grown overnight and 3 plates showed growth. We have therefore made liquid O/N cultures of 0113, 0114 and 0116. The plates continues to incubate at 37 degrees.


Experiment: Melanin 3A assembly.
Two cultures that showed positive results on the gel were grown O/N and miniprepped. They have been added to the spredsheet as 0127 and 0128.


Experiment: Sequencing of biobricks.
Some of our promoter-melanin constructs were sent for sequencing. We hope to determine if it is worth it to continue with this line of experiments, and we will therefore verify that we have the correct constructs.<B>4/10/2018</B>
Experiment: Amplified Insert Assembly of <I>MelA</I>.


Experiment: Assembly of <I>Nat1</I> plasmid.
The <I>Nat1</I> resistence gene has to be inserted into a backbone to be submitted to the registry. We will therefore cut it out and insert it into the backbone of <I>RFP</I> (0023).
The <I>Nat1</I> gene, with the prefix and suffix, and <I>RFP</I> (0023) was digestied with EcoRI and PstI. Following that they were ligated and transformed into E.coli.

Experiment: Sequencing of BioBricks.
The miniprepped promoter-<I>MelA</I>-terminator constructs were sent for sequencing.<B>5/10/2018</B>
Experiment: Assembly of <I>Nat1</I> plasmid.
The plated colonies didn't look great, so we chose the least red colonies and ran them on a colony PCR.

In parallel, since the results seemed very poor, we have redone the digestion and ligation, but this time we are using other enzymes. We suspect that PstI may not be able to cut our <I>Nat1</I> fragment, as our primer design hadn't allowed for extra bases following the cut site. We therefore used EcoRI and SpeI as our enzymes. In addition to using other enzymes, we also tried to use a linearized backbone from the distribution kit to see if that could increase the success-rate.

Experiment: Colony PCR on overnight cultures.
A total of 4 white colonies (N1,N2,N31 &amp; N32) were discovered on the three plates for <I>NAT1</I> transformants. A standard verification primer Onetaq was performed on these cultures and they were run on a gel.

<P>

Suprisingly, cultures N31 and N32 seem rather interesting and should be grown O/N.<B>6/10/2018</B>
Experiment: Colony PCR of the overnight.
A colony PCR was performed on the overnight <I>NAT1</I> transformants. The PCR used was a standard verification primer PCR with 60 seconds extension time.</P><P>

None on the colonies seem promising and looks mostly like selfligated backbones.

Experiment: PCR on N31 and N32.
A PCR with primers was performed on the N31 and N32 lysates from yesterday (5/10) using the primers.

Suprising something worked. However the results a both encouraging and deconsolate, as the two first samples shows promising 400 bp bands, however the negative test gives more or less a band in the same height, for which reason I believe we should run another gel tomorrow.<B>7/10/2018</B>
Experiment: Rerun of PCR on N31 and N32.
The verification PCR from yesterday was repeated using 51 C as annealing temperature instead of 64*C. The results are like those of yesterday and sadly rather inconclusive.

Experiment: Digestion, Ligation &amp; Transformation of <I>NAT1</I>.
A new digestion of <I>NAT1</I> was performed using EcoRI &amp; PstI was made. Each of the <I>NAT1</I> digests (N1,N2,N3) were ligated with both a digested backbone and a linear backbone resulting in the six ligates (N1B,N1L,N2B,N2L,N3B,N3L). These we subsequently transformed and plated alongside with the gibson transformants.

Experiment: Gibson assembly of <I>NAT1</I> into backbone.
Using the prefix/suffix homologous ends of the linear backbone and our <I>NAT1</I> PCR extracts, we performed gibson assembly for insertion of the <I>NAT1</I> gene into the backbone. These were transformed and plated for growth O/N.

Experiment: Overnight cultures of <I>NAT1</I>.
Two addtional O/N culture of N31 and N32 were prepared in liquied LB cam media and left of incubate.


</P></DIV><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>1/10/2018</B><A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol”"><I>Ganoderma</I> protocol</A> started.<I>Ganoderma</I> was inoculated on PDA plates and incubated at 30°C.<B>3/10/2018</B>
We inoculated 12 plates of <I>A. oryzae</I> on PDA and incubated them at 28°C and prepared sorbitol solution and funnels for the aforementioned <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol”"><I>Ganoderma</I> protocol</A>.<B>5/10/2018</B><A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol”"><I>Ganoderma</I> protocol</A> continued. This time with twice as many samples to try an up the amount of mycelium in the tubes. 1 ml og PDB was put into 32 tubes and inoculated with <I>Ganoderma</I>. Incubated at 30°C and at either 85 rpm and 150 rpm for 48 hours instead of 24, again to get more mycelium. Protoplastation of <I>A. oryzae</I> continued. <B>7/10/2018</B><A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol”"><I>Ganoderma</I> protocol</A> continued. The digest and start of isolation were done for the samples that were only given 2-3 hours and the digestion was started for the samples that need 19-24 hours. 

</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
							
Continued work on python code to calculate placement and size of pieces. A substrate grid was implemented to investigate the mycelium development dependent on substrate availability or depletion. Statistical modeling was implemented based on the DOE for both the replicated and unreplicated design. So far a Gamma model or a random effect model is seemingly the best choices.   



</P></DIV></LI></DIV><BUTTON id="week41" class="collapsible">Week 41 (October 8 - October 14)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>8/10/2018</B>
Colony PCR and Nat1 transformants were run. a miniprep of N31 and N32 was performed.

We had failed miniprep of A4 and A5 melA constructs as the A4 and A5 were incubated in media with a wrong antibiotic.<B>10/10/2018</B>
Miniprep of O/N cultures was performed and samples were sent to sequencing.<B>11/10/2018</B>
Results of the sequencing showed that all nat1 attempts have been unsuccessful.<B>14/10/2018</B>
Experiment: PCR Verification of A. oryzae transformants.
New gDNA was purified from 7 different A. oryzae plates. 4 plates contains a (probably) CAC from linear DNA transformation, There are 2 transformants from a GAC transformation and 1 MelA transformant.


Experiment: Transformation of A. oryzae with melA genes.
A. oryzae protoplasts were transformed with pcamV-melA-tcamv constructs (0131) and (0130) and left to incubated on transformations media containing 0,6 mg/mL tyrosine.

</P></DIV></LI><LI class="media"><DIV class="media-body"><H4>Mycolab Overview</H4><P><B>8/10/2018</B><I>Ganoderma</I> protocol continued: The 19-24 hours samples taken out and started the isolation procedure. Put in fridge with the other samples.<B>9/10/2018</B>
Looked at transformation plates with Kyle to see if something might have transformed: He thinks that the media being coloured and the spores looking greener than usual on the transformants might indicate that something has been transformed.  New bricks were made to see if they secrete the colour into the bricks.<B>10/10/2018</B>
The <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol”"><I>Ganoderma</I> protocol</A> was finished by centrifuging for 10 minutes at 10000 rpm. There was no pellet. 
Liquid cultures of the suspected transformants were inoculated and incubated at 28°C and 150 rpm. The cultures were inoculated with both green spores and white spores to see if they secreted any kind of colour.<B>11/10/2018</B>
As a last try, the <A href="“https://2018.igem.org/Team:DTU-Denmark/Experiments#ganodermaprotocol”"><I>Ganoderma</I> protocol</A> was restarted, this time with driselase as we hand received some the same day. There was no time to make new plates to incubate for 5 days, so we had to try with some of the old plates. Three tubes with 1 ml PDB was inoculated for continuation the next day.<B>12/10/2018</B>
The three tubes were taken through the digestion steps and put in the incubator at 30°C, 85 rpm.<B>13/10/2018</B>
The samples were taken out of the incubator. The incubator had been changed to 28°C, 100 rpm when we returned to the samples. The samples were run through the first steps for isolation of protoplasts.<B>14/10/2018</B>
Samples were centrifuged according to protocol but there was no pellet. 

</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
								
Decided that too many complications had arisen in calculating all side lengths and placements and implementing in comsol. Instead we approximate the structure as half a sphere, which is also a good baseline for future work and is more inline with the inaccuracies of our material properties. 



</P></DIV></LI></UL></DIV></DIV><BUTTON id="week42" class="collapsible">Week 42 (October 15 - October 21)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4>Synthlab Overview</H4><P><B>15/10/2018</B>
Experiment:PCR Verification of A. oryzae transformants.
The purified gDNA was PCR amplified using the AmilCO Forward primer and the tCaMV Backwards primer. This should reveal bands around 230 bp. (0092) was used as a positive control.

We see bands around 200 bp, but it is not certain if this is our gene of interest or not.</P><P>

We ran the PCR products on a 2% agerose gel and we got the following:

</P><P>

Here it could seem like we have a band on the first sample that matches the positive control. Another PCR was made to verify the results, but this showed signs of contamination, as the negative also showed bands that was alsom identical to the positive.

</P><B>16/10/2018</B>
Experiment: Confirmation of amilCP construct integration.
Another round gDNA preparation and PCR was performed to confirm integration of amilCP constructs in apparent transformants. Extra care was taken in order to ensure no leakage from wells as this had been a problem earlier. These gel can be seen in the <A target="_blank" href="https://2018.igem.org/Team:DTU-Denmark/Results-amilCP">results page for <I>amilCP</I></A>.

<B>17/10/2018</B>
Experiment: Sequencing of gDNA.
We would like to know where the inserted fragments are in the genome o A. oryzae. Therefore, we have send a sample of the most successful transformants to sequencing with primers that point out of the cassette. 
We tried to ethanol percipitate our previous samples, but we got very low DNA concentrations, which means we cannot expect resonable results. We have therefore performed a new extraction of gDNA from the same spot on the plate and disolved the DNA in 50, 100 and 200 µl of Milli-Q water.

</DIV></LI><LI class="media dry"><DIV class="media-body"><H4>Drylab Overview</H4><P>
							
Completed the simplified simulations in comsol and finished the corresponding wiki site writing.
The meso-growth model and the statistical model were finalized, plots were generated and the wiki was completed. 




</P></DIV></LI></UL></DIV></DIV><BUTTON id="week43" class="collapsible">Week 43 (October 22 - October 28)</BUTTON><DIV class="content"><DIV class="interlabspace"><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">Team Overview</H4><P>
								The DTU BioBuilders are ready and thrilled for Boston. Good luck to everyone.


							</P></DIV></LI></UL></DIV></DIV><DIV class="footcontent" style="height:100%; position:relative;"><FOOTER id="myFooter"><DIV class="footcontainer"><DIV id="footer_row" class="row"><DIV id="footer_useful_links" class="col-xs-4"><H5>Useful links</H5><UL id="footer-item"><P class="footer-links" style="font-size: 16px;"><A href="https://2018.igem.org/Team:DTU-Denmark">Home</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Description">Project description</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Model">Modeling</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Parts">Parts overview</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Notebook">Notebook</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Team">Team</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Safety">Safety</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Human_Practices">Human practices</A></P></UL></DIV><DIV id="footer_about_us" class="col-xs-4" style:=""><H5>About us</H5><UL id="footer-item"><LI><A style="font-size:16px;" href="mailto:dtubiobuilders@gmail.com">dtubiobuilders@gmail.com</A></LI><LI><A style="font-size: 16px;" href="https://www.google.dk/maps/@55.7854419,12.5196676,16z"> Anker Engelunds Vej 1 Bygning 101A, 2800 Kgs. Lyngby, Denmark</A></LI></UL></DIV><DIV id="footer_social" class="col-xs-3"><BUTTON type="button" class="btn btn-default" onclick="window.location.href='mailto:dtubiobuilders@gmail.com'">Contact us</BUTTON></DIV></DIV></DIV></FOOTER></DIV></BODY></HTML>