<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_DTU-Denmark_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:DTU-Denmark/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><UL><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Description"><DIV class="item"><SPAN class="caption">Project</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Description">Description</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Design">Design</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/InterLab">InterLab</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Results">Results</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Results-choosing-organism">Choice of Organism</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Results-amilCP">amilCP Expression</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Results-melA">melA Expression</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Results-NAT-1">nat1 Expression</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Ganoderma-protoplastation">Ganoderma Protoplastation</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Demonstrate">Demonstrate</A></LI></UL></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Model"><DIV class="item"><SPAN class="caption">Modeling</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Model">Model</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/GrowthModelling">Growth Modeling</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/DesignOfExperiments">Design of Experiments</A></LI><LI style="padding-left:4em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/ModellingTheDesign">Modeling the Design</A></LI><LI style="padding-left:2em"><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/StructuralIntegrity">Structural Integrity</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Hardware">Hardware</A></LI></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Parts"><DIV class="item"><SPAN class="caption">Parts</SPAN></DIV></A></LI><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Notebook"><DIV class="item"><SPAN class="caption">Journal</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Notebook">Notebook</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Experiments">Experiments</A></LI></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Team"><DIV class="item"><SPAN class="caption">People</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Team">Team Members</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Attributions">Attributions</A></LI></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Human_Practices"><DIV class="item"><SPAN class="caption">Human Practices</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Human_Practices">Human Practices</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Public_Engagement">Education and Engagement</A></LI></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Collaborations"><DIV class="item"><SPAN class="caption">Collaboration</SPAN></DIV></A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/Collaborations">Collaborations</A></LI><LI><A id="asub" href="https://2018.igem.org/Team:DTU-Denmark/BioBrickTutorial">BioBrick Tutorial</A></LI></DIV><LI><A href="https://2018.igem.org/Team:DTU-Denmark/Safety"><DIV class="item"><SPAN class="caption">Safety</SPAN></DIV></A></LI><DIV id="mobiletopnav" class="topnav"><A href="https://2018.igem.org/Team:DTU-Denmark" class="activeMM">Hyphae Hackers</A><DIV id="myLinks"><A href="javascript:void(0);" onclick="myFunction1()">
    Project
  </A><DIV id="myLinks1" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Description">Description</A><A href="https://2018.igem.org/Team:DTU-Denmark/Design">Design</A><A href="https://2018.igem.org/Team:DTU-Denmark/InterLab">InterLab</A><A href="https://2018.igem.org/Team:DTU-Denmark/Results">Results</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/Results-choosing-organism">Choice of Organism</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/Results-amilCP">amilCP Expression</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/Results-melA">melA Expression</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/Results-NAT-1">nat1 Expression</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/Ganoderma-protoplastation">Ganoderma Protoplastation</A><A href="https://2018.igem.org/Team:DTU-Denmark/Demonstrate">Demonstrate</A></DIV><A href="javascript:void(0);" onclick="myFunction2()">
    Modeling
  </A><DIV id="myLinks2" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Model">Model</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/GrowthModelling">Growth Modeling</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/DesignOfExperiments">Design of Experiments</A><A style="padding-left:4em" href="https://2018.igem.org/Team:DTU-Denmark/ModellingTheDesign">Modeling the Design</A><A style="padding-left:2em" href="https://2018.igem.org/Team:DTU-Denmark/StructuralIntegrity">Structural Integrity</A><A href="https://2018.igem.org/Team:DTU-Denmark/Hardware">Hardware</A></DIV><A href="https://2018.igem.org/Team:DTU-Denmark/Parts">Parts Overview</A><A href="javascript:void(0);" onclick="myFunction4()">
    Journal
  </A><DIV id="myLinks4" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Notebook">Notebook</A><A href="https://2018.igem.org/Team:DTU-Denmark/Experiments">Experiments</A></DIV><A href="javascript:void(0);" onclick="myFunction5()">
    People
  </A><DIV id="myLinks5" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Team">Team Members</A><A href="https://2018.igem.org/Team:DTU-Denmark/Attributions">Attributions</A></DIV><A href="https://2018.igem.org/Team:DTU-Denmark/Safety">Safety</A><A href="javascript:void(0);" onclick="myFunction6()">
    Human Practices
  </A><DIV id="myLinks6" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:DTU-Denmark/Public_Engagement">Education &amp; Engagement</A></DIV><A href="javascript:void(0);" onclick="myFunction7()">
    Collaborations
  </A><DIV id="myLinks7" class="topsubnav"><A href="https://2018.igem.org/Team:DTU-Denmark/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:DTU-Denmark/BioBrickTutorial">BioBrick Tutorial</A></DIV></DIV></DIV><DIV class="toppercontainer"><DIV class="headlinecontainer"><H1>Experiments</H1></DIV></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="container-fluid" style="width: 80%;margin-left: auto;
    margin-right: auto;position:relative;margin-top:0%;"><DIV class="row"><DIV class="interlabspace"><H2 style="text-align: left;margin-bottom:20px;">Synthlab protocols</H2><BUTTON id="resuspensionofdna" class="collapsible">Resuspension of DNA/primer</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P>
DNA from IDT will typically be delivered in a white flaky substance, which needs to be resuspended before the DNA is ready for use.
</P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><UL style="list-style-type:disc"><LI>Table centrifuge</LI><LI>EB/TE buffer or Milli-Q Water</LI><LI>Genes/Primers from IDT or other DNA provider</LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><H5>Resuspension</H5><OL><LI>Quickly spin the DNA down in the table centrifuge.</LI><LI>Calculate the amount of EB buffer needed to dilute the gBlocks to the desired concentration.
		<B>Important</B>: gene fragments and primers are not diluted to the same concentration:
		The concentration of gene fragments is usually 25 ng/μL  and for primers it is 100 μM.
		</LI><LI>Note 1: DNA from IDT usually comes in dried flakes of 500 or 1000 ng of DNA. To achieve the desired concentration (usually 25 ng/μL) the needed amount of EB buffer is  40 μL (for 1000 ng samples) or 20 μL (for 500 ng samples).</LI><LI>Note 2: In order to calculate the molar amount of primer, use the <A href="https://nebiocalculator.neb.com/#!/ssdnaamt">NEB calculator.</A></LI><LI>Add the calculated amount of EB buffer/Milli-Q water. </LI><LI>Store the resuspended DNA at -20°C. </LI></OL></DIV></DIV></DIV><BUTTON id="adaptedassemblyofcomposite" class="collapsible">Adapted Assembly of Composite Parts Into Backbone</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P><B>Timeline for procedure:</B>
Day 1: Minimum 4 ½ hours from start to finish
Day 2: Overnight culture from plating 
Day 3: Culture ready for MiniPrep
</P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><H5>Digestion materials</H5><UL style="list-style-type:disc"><LI>Milli-Q water</LI><LI>Buffer (e.g. cut-smart)</LI><LI>Gene or plasmid</LI><LI>Restriction enzyme (e.g. EcoRI)</LI><LI>Restriction enzyme (e.g. PstI)</LI></UL><H5>Ligation materials</H5><UL style="list-style-type:disc"><LI>Milli-Q water</LI><LI>T4 DNA ligase Buffer</LI><LI>Plasmid 3 kB</LI><LI>Cut part</LI><LI>T4 DNA ligase</LI></UL><H5>Transformation materials</H5><UL style="list-style-type:disc"><LI>Competent cells</LI><LI>Ice</LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><H5>Digestion</H5><OL><LI>Calculate the amount of gene fragment and plasmid backbone (A1) used for the digestion.
		</LI><LI>The recommended amount is 100 ng gene fragment and given that the standard concentration of resuspended DNA is 25 ng/μL, the needed amount is 4 μL.</LI><LI> Beware: For each digested gene fragment, one should prepare 50 ng of digested backbone (e.g. for 4 gene fragments one should prepare 200 ng of backbone). </LI><LI>Combine the contents of the table in PCR tubes:
		<TABLE><TBODY><TR><TH>Reagent</TH><TH>Value</TH></TR><TR><TD colspan="2">For each gene fragment</TD></TR><TR><TD>Milli-Q water</TD><TD>Up to 10 ul</TD></TR><TR><TD>Buffer</TD><TD>1 µl</TD></TR><TR><TD>Gene or A1</TD><TD>*</TD></TR><TR><TD>Restriction enzyme</TD><TD>0,2 µl</TD></TR><TR><TD>Restriction enzyme</TD><TD>0,2 µl</TD></TR><TR><TD>Total Volume</TD><TD>10 µl</TD></TR></TBODY></TABLE></LI><LI>Digest at 37°C for 1 hour.</LI><LI>Heat inactivate at 80°C for 20 min.</LI><LI>Store in the freezer or use for ligation.</LI></OL><H5>Ligation</H5><OL start="6"><LI>Calculate the volume of Digested DNA and Digested Backbone in the table below.<B>Volume of Digested DNA:</B> Open the <A href="https://nebiocalculator.neb.com/#!/ligation">NEB calculator</A>:
		</LI><LI>Use the Digested DNA-to-Plasmid-ratio 3:1. </LI><LI>We are using 50 ng backbone, as per a <A href="https://international.neb.com/Protocols/0001/01/01/dna-ligation-with-t4-dna-ligase-m0202">NEB protocol</A></LI><LI>Digestion concentration is 10 ng/µl</LI><TABLE><TBODY><TR><TH>Reagent</TH><TH>Volume</TH></TR><TR><TD colspan="2">For each of Digested DNA fragment</TD><TD colspan="2" style="border-left: 5px solid #000;">Standard DNA digest volumes*</TD></TR><TR><TD>Milli-Q water </TD><TD>up to 20 µl </TD><TD style="border-left: 5px solid #000;">Length of digested part</TD><TD>Volume </TD></TR><TR><TD>T4 DNA ligase Buffer </TD><TD>2.0 µl</TD><TD style="border-left: 5px solid #000;">500 bp</TD><TD>3.75 µl</TD></TR><TR><TD>Plasmid  3 kB </TD><TD>50 ng</TD><TD style="border-left: 5px solid #000;">1000 bp</TD><TD>7.5 µl</TD></TR><TR><TD>Digested part </TD><TD>x</TD><TD style="border-left: 5px solid #000;">1500 bp</TD><TD>11.25 µl</TD></TR><TR><TD>T4 DNA ligase </TD><TD>1  µl</TD><TD style="border-left: 5px solid #000;">2000 bp</TD><TD>15 µl</TD></TR><TR><TD>Total Volume </TD><TD>20  µl</TD></TR></TBODY></TABLE>
	* using the 2 kb A1 backbone.
	<LI>Mix the contents of the table above in a PCR tube.</LI><LI>Incubate at 25°C for 30 min.</LI><LI>Inactivate at 65°C for 20 min.</LI><LI>Put the ligation in the freezer or use it for transformation.</LI></OL><H5>Transformation</H5><OL start="11"><LI>5 µl of the ligated parts is transferred to an Eppendorf tube.</LI><LI>1 µl of the positive control is transferred to an Eppendorf tube.</LI><LI>50 µl competent cells is transferred to each tube. Competent cells must be held on ice!</LI><LI>Mix gently by rolling the tubes by hand.</LI><LI>Put the tubes back on ice for 30 minutes.</LI><LI>Make sure the heating block is heated up to 42°C and heat-shock the tubes for 30 seconds.</LI><LI>Place the tubes back on ice for 5 minutes.</LI><LI>Pipette 500 ul of sterile LB media to each tube.</LI><LI>Incubate for 2 hours for 37°C with shaking (this step can be shortened if an amp-backbone is used).</LI><LI>Shake the tube with the cells gently. Pipette 200 µl cell culture on each plate.</LI><LI>Incubate at 37°C O/N.</LI></OL></DIV></DIV></DIV><BUTTON id="proto3aassembly" class="collapsible">3A Assembly</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P><I>Adapted from <A href="http://parts.igem.org/Help:Protocols/3A_Assembly">iGEM 3A assembly and </A><A href="http://igem-victoria.pbworks.com/w/page/7992050/3A%20Assembly%20Protocol">iGEM Victoria 3A protocol</A>.</I></P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><H5>Digestion</H5><UL style="list-style-type:disc"><LI>Cutsmart buffer</LI><LI>SpeI, XbaI, PstI and EcoRI  restriction enzymes</LI><LI>Biobrick compatible genes</LI><LI>Milli-Q water</LI><LI>2 DNA fragments with verification primers</LI><LI>PCR tubes (2 per gene fragment: 1 for digestion, 1 for ligation)</LI></UL><H5>Ligation</H5><UL style="list-style-type:disc"><LI>T4 ligase buffer</LI><LI>T4 ligase</LI></UL><H5>Transformation</H5><UL style="list-style-type:disc"><LI>Competent cells</LI><LI>Ice</LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><H5>Digestion</H5><OL><LI>Mix the assembly parts following their respective tables a PCR tube.
		*100 ng is needed.
		<B>Upstream fragment</B><TABLE><TBODY><TR><TH>Reagent</TH><TH>Volume</TH></TR><TR><TD colspan="2">For each gene fragment</TD></TR><TR><TD>Milli-Q water </TD><TD>Up to 10 ul  </TD></TR><TR><TD>Cutsmart buffer</TD><TD>1 µl</TD></TR><TR><TD>Gene</TD><TD>*</TD></TR><TR><TD>SpeI</TD><TD>0,2 µl</TD></TR><TR><TD>EcoRI</TD><TD>0,2 µl</TD></TR><TR><TD>Total Volume </TD><TD>10 µl</TD></TR></TBODY></TABLE><B>Downstream fragment</B><TABLE><TBODY><TR><TH>Reagent</TH><TH>Volume</TH></TR><TR><TD colspan="2">For each gene fragment</TD></TR><TR><TD>Milli-Q water </TD><TD>Up to 10 ul  </TD></TR><TR><TD>Cutsmart buffer</TD><TD>1 µl</TD></TR><TR><TD>Gene</TD><TD>*</TD></TR><TR><TD>SpeI</TD><TD>0,2 µl</TD></TR><TR><TD>EcoRI</TD><TD>0,2 µl</TD></TR><TR><TD>Total Volume </TD><TD>10 µl</TD></TR></TBODY></TABLE><B>Backbone</B><TABLE><TBODY><TR><TH>Reagent</TH><TH>Volume</TH></TR><TR><TD colspan="2">For every other gene fragment</TD></TR><TR><TD>Milli-Q water </TD><TD>Up to 10 ul  </TD></TR><TR><TD>Buffer </TD><TD>1 µl</TD></TR><TR><TD>Gene</TD><TD>*</TD></TR><TR><TD>EcoRI</TD><TD>0,2 µl</TD></TR><TR><TD>PstI</TD><TD>0,2 µl</TD></TR><TR><TD>Total Volume </TD><TD>10 µl</TD></TR></TBODY></TABLE></LI><LI>Digest at 37°C for 1 hour.</LI><LI>Heat inactivate at 80°C for 20 min.</LI><LI>Store in the freezer or use for ligation.</LI></OL><H5>Ligation</H5><OL start="5"><LI>Mix the following in a PCR tube.
		<TABLE><TBODY><TR><TH>Reagent</TH><TH>Volume</TH></TR><TR><TD colspan="2"><B>For each of Digested DNA fragments</B></TD></TR><TR><TD>Milli-Q water </TD><TD>up to 20 µl </TD></TR><TR><TD>T4 DNA ligase Buffer </TD><TD>2.0 µl</TD></TR><TR><TD>Backbone</TD><TD>2 µl</TD></TR><TR><TD>Upstream fragment</TD><TD>2 µl</TD></TR><TR><TD>Downstream fragment</TD><TD>2 µl</TD></TR><TR><TD>T4 DNA ligase </TD><TD>1  µl</TD></TR><TR><TD>Total Volume </TD><TD>20  µl</TD></TR></TBODY></TABLE></LI><LI>Incubate at 25°C for 30 min.</LI><LI>Inactivate at 65°C for 20 min</LI></OL><H5>Transformation</H5><OL start="8"><LI>5 µl of the ligated parts is transferred to a chilled Eppendorf tube.</LI><LI>1 µl of the positive control is transferred to an Eppendorf tube.</LI><LI>50 µl competent cells is transferred to each tube. Competent cells must be held on ice!</LI><LI>Mix gently by rolling the tubes by hand.</LI><LI>Put the tubes back on ice for 30 minutes.</LI><LI>Make sure the heating block is heated up to 42°C and heat-shock for 30 seconds.</LI><LI>Place the tubes back on ice for 5 minutes.</LI><LI>Pipette 500 µl of sterile LB media to each tube.</LI><LI>Incubate for 2 hours for 37°C with shaking (this step can be shortened if an amp-backbone is used).</LI><LI>Next step is to plate it on to the plates.</LI><LI>Shake the tube with the cells gently. Pipette 100 µl cell culture on each plate.</LI><LI>Incubate at 37°C O/N.</LI></OL></DIV></DIV></DIV><BUTTON id="QIAprep" class="collapsible">QIAprep® Spin Miniprep Kit</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P><I>Adapted from Qiagen's <A href="https://www.qiagen.com/us/resources/download.aspx?id=56b0162c-23b0-473c-9229-12e8b5c8d590&amp;lang=en">QIAprep® Spin Miniprep Kit</A></I>
This protocol assumes using a centrifuge, and not vacuum manifold processing.
</P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><H5>Buffers</H5><UL style="list-style-type:disc"><LI>P1 buffer</LI><LI>P2 buffer</LI><LI>N3 buffer</LI><LI>LyseBlue reagent</LI><LI>PB buffer</LI><LI>PE buffer</LI><LI>EB buffer</LI></UL><H5>Tubes</H5><UL style="list-style-type:disc"><LI>Centrifuge tubes</LI><LI>1.5 mL Eppendorf tubes</LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><H5>Quick-start protocol</H5><OL><LI>Pellet 1–5 ml bacterial overnight culture by centrifugation at &gt;8000 rpm (6800 x g) for 3 min at room temperature (15–25°C). </LI><LI>Resuspend pelleted bacterial cells in 250 μl Buffer P1 and transfer to a microcentrifuge tube. </LI><LI>Add 250 μl Buffer P2 and mix thoroughly by inverting the tube 4–6 times until the solution becomes clear. Do not allow the lysis reaction to proceed for more than 5 min. If using LyseBlue reagent, the solution will turn blue. </LI><LI>Add 350 μl Buffer N3 and mix immediately and thoroughly by inverting the tube 4–6 times. If using LyseBlue reagent, the solution will turn colorless. </LI><LI>Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge. </LI><LI>Apply 800 μl supernatant from step 5 to the QIAprep 2.0 spin column by pipetting. Centrifuge for 30–60 s and discard the flow-through.</LI><LI>Recommended: Wash the QIAprep 2.0 spin column by adding 0.5 ml Buffer PB. Centrifuge for 30–60 s and discard the flow-through.</LI><LI>Wash the QIAprep 2.0 spin column by adding 0.75 ml Buffer PE. Centrifuge for 30–60 s and discard the flow-through. Transfer the QIAprep 2.0 spin column to the collection tube. </LI><LI>Centrifuge for 1 min to remove residual wash buffer. </LI><LI>Place the QIAprep 2.0 column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 μl Buffer EB (10 mM TrisCl, pH 8.5) or water to the center of the QIAprep 2.0 spin column, let stand for 1 min, and centrifuge for 1 min. </LI><LI>If the extracted DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel.</LI></OL></DIV></DIV></DIV><BUTTON id="hifiDNAassembly" class="collapsible">Hifi DNA Assembly and Transformation Protocol</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P><I>Adapted from IDT's <A href="https://international.neb.com/-/media/catalog/datacards-or-manuals/manuale2621.pdf">HiFi assembly protocol</A></I></P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><H5>Consumables</H5>
X is the number of reactions.
<UL style="list-style-type:disc"><LI>X PCR tubes for each reaction + 1 for positive control</LI><LI>X Eppendorf tube for each reaction + 1 for positive control</LI><LI>X selection plate for each reaction</LI><LI>1 Amp plate for positive control</LI></UL><H5>Chemicals</H5><UL style="list-style-type:disc"><LI>Hifi DNA assembly Master mix</LI><LI>Milli-Q water</LI><LI>Competent <I>E. coli</I> (e.g. DH5α)</LI><LI>Prepared DNA fragments for assembly (See information on primer construction)</LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><H5>Assembly protocol</H5><OL><LI>Set up the following reaction on ice:
		<TABLE><TBODY><TR><TH colspan="3">Recommended amount of fragments used for assembly</TH></TR><TR><TD><B>2-3 Fragments*</B></TD><TD><B>4-6 Fragments</B></TD><TD><B>Positive control</B></TD></TR><TR><TD><B>Recommended DNA Molar Ratio</B></TD><TD>Vector:insert = 1:2</TD><TD>Vector:insert = 1:1</TD></TR><TR><TD><B>Total amount of DNA fragments</B></TD><TD>0.03-0.3pmolsX uL</TD><TD>0.2-0.5 pmolsX uL</TD><TD>10 uL</TD></TR><TR><TD><B>NEB Hifi Assembly master mix</B></TD><TD>10 uL</TD><TD>10 uL</TD><TD>10 uL</TD></TR><TR><TD><B>Milli-Q water</B></TD><TD>10-X uL</TD><TD>10-x uL</TD><TD>0 uL</TD></TR><TR><TD><B>Total volume</B></TD><TD>20 uL**</TD><TD>20 uL**</TD><TD>20 uL</TD></TR></TBODY></TABLE>
		* If the inserts are less than 200 bp, use a 5-fold excess of inserts instead of a 2-fold excess.
		** If a  greater number of fragments are assembled, increase the volume of the reaction and use additional HiFi DNA assembly master mix.
	</LI><LI>Incubate the reaction samples in a thermocycler at 50°C for 15 minutes (when 2-3 fragments are assembled) or 60 minutes (when 4-6 fragments are assembled). Following incubation, store the reaction samples at -20°C for subsequent transformation.
		Note: Extended incubation up to 60 minutes can in some cases improve transformation efficiency.
	</LI></OL><H5>Transformation protocol</H5><OL start="3"><LI>Thaw chemically-competent cells on ice.</LI><LI>Add 2 µL of the chilled assembly product to the competent cells. Mix gently by pipetting up or down or by flicking the tube 4-5 times. <U>Do NOT vortex</U>.</LI><LI>Place the mixture on ice for 30 minutes. <U>Do not mix</U>.</LI><LI>Heat shock at 42°C for 30 seconds. <U>Do not mix</U>.</LI><LI>Transfer tubes to ice for 2 minutes.</LI><LI>Add 950 µL of room temperature SOC media to the tubes.</LI><LI>Incubate the tube for 37°C for 60 minutes. shake vigorously (250 rpm) or rotate.</LI><LI>Warm selection plates to 37°C.</LI><LI>Spread 100 µL of the cells onto the selection plates. Note: Use Amp plates for the positive control.</LI><LI>Incubate overnight at 37°C.</LI></OL></DIV></DIV></DIV><BUTTON id="gelextraction" class="collapsible">Gel Extraction Protocol</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P><I>Adapted from The Chen Laboratory's <A href="http://www.indiana.edu/~lchenlab/protocol_files/agarose_gel_extraction.pdf">QIAquick Gel Extraction Kit Protocol</A>.</I></P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><H5>Consumables</H5>
N is the number of reactions.
<UL style="list-style-type:disc"><LI>2*Ν  1.5 mL Eppendorf tubes</LI><LI>Ν spin columns with collection tubes</LI></UL><H5>Chemicals</H5><UL style="list-style-type:disc"><LI>Buffer QG</LI><LI>Buffer PE</LI><LI>Buffer EB</LI><LI>Isopropanol</LI><LI>(potentially: 3 M sodium acetate - see procedure step 4)</LI></UL><H5>Instruments</H5><UL style="list-style-type:disc"><LI>Thermocycler/thermoshaker</LI><LI>Scalpel</LI><LI>Table centrifuge for 1.5 mL microcentrifuge/Eppendorf tubes.</LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><H5>Gel extraction</H5><OL><LI>Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose. </LI><LI>Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 µl). 
		For example, add 300 µl of Buffer QG to each 100 mg of gel. For &gt;2% agarose gels, add 6 volumes of Buffer QG. The maximum amount of gel slice per QIAquick column is 400 mg; for gel slices &gt;400 mg use more than one QIAquick column.
	</LI><LI>Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation. IMPORTANT: Solubilize agarose completely. For &gt;2% gels, increase incubation time. </LI><LI>After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 µl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn to yellow. 
		Note: The adsorption of DNA to the QIAquick membrane is efficient only at pH ≤7.5. Buffer QG contains a pH indicator which is yellow at pH ≤7.5 and orange or violet at higher pH, allowing easy determination of the optimal pH for DNA binding.
	</LI><LI> Add 1 gel volume of isopropanol to the sample and mix.
		For example: if the agarose gel slice is 100 mg, add 100 µl isopropanol. This step increases the yield of DNA fragments &lt;500 bp and &gt;4 kb. For DNA fragments between 500 bp and 4 kb, the addition of isopropanol has no effect on yield. Do not centrifuge the sample at this stage.
	</LI><LI>Place a QIAquick spin column in a provided 2 ml collection tube. </LI><LI>To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min at ≥10,000 x g (~13,000 rpm). The maximum volume of the column reservoir is 800 µl. For sample volumes of more than 800 µl, simply load and spin again. </LI><LI>Discard flow-through and place QIAquick column back in the same collection tube. </LI><LI>Optional: Add 0.5 ml of Buffer QG to QIAquick column and centrifuge for 1 min ≥10,000 x g (~13,000 rpm). This step will remove all traces of agarose. It is only required when the DNA will subsequently be used for direct sequencing, in vitro transcription or microinjection. </LI><LI>To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min ≥10,000 x g (~13,000 rpm). Note: If the DNA will be used for salt-sensitive applications, such as blunt-end ligation and direct sequencing, let the column stand 2–5 min after the addition of Buffer PE, before centrifuging. </LI><LI>Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at ≥10,000 x g (~13,000 rpm). IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation. </LI><LI>Place QIAquick column into a clean 1.5 ml microcentrifuge tube. </LI><LI>To elute DNA, add 50 µl of Buffer EB (10 mM Tris·Cl, pH 8.5) or H<SUB>2</SUB>O to the center of the QIAquick membrane and centrifuge the column for 1 min at maximum speed. Alternatively, for increased DNA concentration, add 30 µl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min.
		<B>IMPORTANT</B>: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volume is 48 µl from 50 µl elution buffer volume, and 28 µl from 30 µl. Elution efficiency is dependent on pH. The maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C as DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE (10 mM Tris·Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions.</LI></OL></DIV></DIV></DIV><BUTTON id="q5pcramplification" class="collapsible">Q5 PCR amplification with Standard Primers</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P><I>Adapted from NEB Q5 <A href="https://international.neb.com/protocols/2012/12/07/protocol-for-q5-high-fidelity-2x-master-mix-m0492">High-Fidelity 2X Master Mix</A></I></P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><H5>Tubes</H5><UL style="list-style-type:disc"><LI>PCR tubes (1 per reaction)</LI></UL><H3>Reagent</H3><UL style="list-style-type:disc"><LI>Milli-Q water</LI></UL><H5>DNA</H5><UL style="list-style-type:disc"><LI>VR/VF primers</LI><LI>Ligation mix</LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><H5>Q5 PCR amplification with standard primers</H5><OL><LI>Determine the needed amount (x) of template DNA via the table below. Added volumes of about 1 µl are preferred, so dilute the sample if necessary with Milli-Q water.<B>Template amounts</B><TABLE><TBODY><TR><TH>Template source</TH><TH>Template amount</TH></TR><TR><TD>Genomic</TD><TD>1 ng - 1 μg</TD></TR><TR><TD>Plasmid or viral</TD><TD>1 pg - 1 ng</TD></TR></TBODY></TABLE></LI><LI>Mix the reagents in a PCR tube.<B>PCR reagents</B><TABLE><TBODY><TR><TH>Reagent</TH><TH>Volume</TH></TR><TR><TD colspan="2">For each of Digested DNA fragment</TD></TR><TR><TD>Milli-Q water </TD><TD>up to 50 µl </TD></TR><TR><TD>VF primer</TD><TD>2.5 µl</TD></TR><TR><TD>VR primer</TD><TD>2.5 µl</TD></TR><TR><TD>Template sample</TD><TD>x</TD></TR><TR><TD>Q5 2X Master Mix</TD><TD>25 µl</TD></TR><TR><TD>Total Volume </TD><TD>50  µl</TD></TR></TBODY></TABLE></LI><LI>For PCR machines without heated lid: overlay the sample with mineral oil.</LI><LI>Place in thermocycler using the following routine:<B>Thermocycler routine</B><TABLE><TBODY><TR><TH>Step</TH><TH>Temperature (°C)</TH><TH>Time</TH></TR><TR><TD>Initial Denaturation</TD><TD>98</TD><TD>30 sec</TD></TR><TR><TD>35 cycles</TD><TD>98</TD><TD>5-10 sec</TD></TR><TR><TD>66</TD><TD>10-30 sec</TD></TR><TR><TD>72</TD><TD>20-30 sec/kb*</TD></TR><TR><TD>Final extension</TD><TD>72</TD><TD>120 sec</TD></TR><TR><TD>Hold</TD><TD>4-10</TD></TR></TBODY></TABLE>
		*The recommended extension temperature is 72°C. Extension times are generally 20–30 seconds per kb for complex, genomic samples, but can be reduced to 10 seconds per kb for simple templates (plasmid, <I>E. coli</I>, etc.) or complex templates &lt; 1 kb. Extension time can be increased to 40 seconds per kb for cDNA or long, complex templates, if necessary.
		**When amplifying products &gt; 6 kb, it is often helpful to increase the extension time to 40–50 seconds/kb.
	</LI><LI>After thermocycling the product can be stored at -20°C or be used for gel electrophoresis.</LI></OL></DIV></DIV></DIV><BUTTON id="colonypcr" class="collapsible">Colony PCR</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P><I>By David Faurdal, adapted in part from NEB's one-taq protocol.</I>
The purpose of this protocol is to confirm correct insertion of fragments after 3A assembly/GIBSON assembly into the pSB1C3 backbone. This is done by PCR amplifying the insert one is interested in and confirming that it's the correct size (by gel electrophoresis). As the backbone comes equipped with verification primer-binding sites, these primers, VF2/VR (sequences can be found <A href="http://parts.igem.org/Primers/Catalog">here</A>), can simply be used if one does insertion into it. The protocol can, however, be used with different primers for different backbones. As the fragments run on the gel won't be used for cloning purposes there is no reason to use high-fidelity polymerases on this, just use one-taq.
</P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><UL style="list-style-type:disc"><LI>Transformants from whatever assembly method you fashion</LI><LI>Eppendorf tubes</LI><LI>Sterile toothpicks/inoculation lops/pipette tips for transferring colonies</LI><LI>Milli-Q water</LI><LI>LB media with appropriate antibiotics</LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><H5>Preparing the template DNA from the transformants</H5><OL><LI>Pick a number of transformants, typically 3-10, from each plate of interest and mark them on the back of the plate.</LI><LI>Set up 2 Eppendorf tubes for each colony and mark them accordingly:
		</LI><LI>Fill the first one (1) with 15 µl Milli-Q water.</LI><LI>The other one (2) remains empty for now.</LI></OL><LI>Transfer each colony to the Eppendorf containing 15 µl water using a sterile toothpick, inoculation loop or autoclaved pipette tip.</LI><LI>Transfer 5 µl of the water from (1) to the empty (2) tube. This tube (2) is now for safekeeping in case the colony PCR shows that the transformant in question contains the correct insertion.</LI><LI>Boil the (1) tubes for 10 minutes at 98 °C. Prepare the PCR mastermix, while the colonies are boiling.</LI><H5>Setting up the PCR itself</H5><OL><LI>Set up a 25 µl reaction for each colony to be screened, as per NEB'S one-taq PCR protocol (see the <A href="https://international.neb.com/protocols/2012/10/11/onetaqdnapolymerasem0480">protocol</A> for troubleshooting).
		<TABLE><TBODY><TR><TH>Component</TH><TH>25 µl reaction</TH></TR><TR><TD>5x OneTaq Standard Reaction Buffer</TD><TD>5 µl</TD></TR><TR><TD>10 mM dNTP</TD><TD>0,5 µl</TD></TR><TR><TD>10 µM Forward Primer</TD><TD>0,5 µl</TD></TR><TR><TD>10 µM Reverse Primer</TD><TD>0,5 µl</TD></TR><TR><TD>OneTaq  DNA Polymerase</TD><TD>0.125 µl</TD></TR><TR><TD>Template</TD><TD>1 µl from the (1) tube</TD></TR><TR><TD>Nuclease-Free Water</TD><TD>up to 25 µl</TD></TR></TBODY></TABLE></LI><LI>Run the PCR in the thermocycler (use this <A href="https://tmcalculator.neb.com/">website</A> to calculate temperatures used based upon the primers and polymerase used). </LI><LI>Run the products on a gel to check for correct insertion.</LI><LI>Prepare an O/N culture from the (2) tubes that have the correct insertions by adding 1 ml LB media to the tube, mixing it and transfer a W-tube containing 4 ml LB media. </LI></OL></DIV></DIV></DIV><BUTTON id="gDNAprotocol" class="collapsible">gDNA purification from <I>Aspergillus</I></BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P><I> Adapted by Jacob Mejlsted</I></P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><H5>Consumables</H5>
N is the number of samples.
<UL style="list-style-type:disc"><LI>Ν 2 mL Fastprep tubes</LI><LI>Ν 1.5 mL microcentrifuge/Eppendorf tube</LI></UL><H5>Chemicals</H5><UL style="list-style-type:disc"><LI>Lysis buffer <B>or</B> breaking buffer + LiAc </LI><LI>Small glass beads </LI><LI>5 M NaCl </LI><LI>Icecold 96% ethanol </LI><LI>Milli-Q water </LI></UL><H5>Instruments</H5><UL style="list-style-type:disc"><LI>FastPrep machine</LI><LI>Table centrifuge for 1.5 mL microcentrifuge/Eppendorf tubes.</LI><LI>Heating block</LI></UL></DIV><DIV class="col-xs-8"><H4>Procedure</H4><H5>gDNA purification</H5><OL><LI>Mix the following in a Fastprep tube: Scrape from plate colony and 500 µL lysis buffer <B>or</B> 500 µL breaking buffer + LiAc and 200 µL small glass beads.</LI><LI> Put in FastPrep machine at speed 4 for 40 seconds.</LI><LI>Spin down with a table centrifuge and transfer 150 µL of the supernantant to a new microcentrifuge tube. </LI><LI>Add 15 µL 5 M NaCl and 400 µL icecold 96% ethanol and mix </LI><LI>Spin 3 minutes at 10,000xg </LI><LI>Remove supernantant </LI><LI>Dry tubes on a heating block at 50 °C </LI><LI>Add 200 µL Milli-Q water and vortex </LI><LI><B>Optional:</B> Spin down and transfer 150 µL to a new tube. (This can give a cleaner solution) </LI></OL></DIV></DIV></DIV></DIV><DIV class="interlabspace"><H2 style="text-align: left;margin-bottom:20px;">Mycolab protocols</H2><BUTTON id="growthfungiagarplates" class="collapsible">Growth of Fungi on Agar Plates</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P>
Basic protocol on plating of the fungi. 
</P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><UL><LI><P>Agar plates with media of interest</P></LI><LI><P>Sterile toothpicks</P></LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><P>Plating using mycelia
</P><OL><LI><P>Open the plate containing the mycelia from the species of interest</P></LI><LI><P>With a sterile toothpick, scratch the surface of the mycelia.</P></LI><LI><P>Spike the new agar plate with the toothpick.</P></LI><LI><P>Repeat steps 1-3 times. Three inoculation points should be made in the agar forming a triangle. </P></LI><LI><P>Put the plate into a plastic growth bag. </P></LI><LI><P>Place growing bag into the incubator.  </P></LI></OL><P>Plating using spores
</P><OL><LI><P>Touch the sterile tooth pick in the spore suspension.</P></LI><LI><P>Spike the new agar plate with the toothpick.</P></LI><LI><P>Repeat steps 1-3 times. Three inoculation points should be made in the agar forming a triangle. </P></LI><LI><P>Put the plate into a plastic growth bag.  </P></LI><LI><P>Place growing bag into the incubator. </P></LI></OL></DIV></DIV></DIV><BUTTON id="mediaforfungi" class="collapsible">Media for Fungi</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P>
Minimal media used for protoplastation of <I>A. oryzae</I>. This is for 1 L of media. 
</P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><H5 class="media heading">Minimal media</H5><UL><LI><P>50 mL D-glucose (20% w/V)</P></LI><LI><P>50 mL nitrate salts (20x)</P></LI><LI><P>1 mL trace elements (1000x)</P></LI><LI><P>1 mL thiamine (1%)</P></LI><LI><P>20 g agar (SO.BI.GEL, solid) </P></LI></UL><H5 class="media heading">Transformation media</H5><UL><LI><P>342,30 g sucrose (solid)</P></LI><LI><P>50 mL nitrate salts (20x)</P></LI><LI><P>1 mL trace elements (1000x)</P></LI><LI><P>1 mL thiamine (1%)</P></LI><LI><P>20 g agar (SO.BI.GEL, solid)</P></LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><OL><LI><P>Mix in a blue cap flask.</P></LI><LI><P>Add water until the volume is 1 L. </P></LI><LI><P>Autoclave. </P></LI></OL></DIV></DIV></DIV><BUTTON id="minimalmediumsupplemented" class="collapsible">Minimal Medium Supplemented</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P>
Minimal medium: <I>Schizophyllum commune</I> SMM agar (1L) needed for <I>S. commune</I> plates 
</P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><UL><LI><P>Glucose - 20g</P></LI><LI><P>Monopotassium phosphate KH<SUB>2</SUB>PO<SUB>4</SUB> - 0.46g</P></LI><LI><P>Potassium phosphate dibasic K<SUB>2</SUB>HPO<SUB>4</SUB>  3H<SUB>2</SUB>O - 1.28g</P></LI><LI><P>Magnesium sulfate MgSO<SUB>4</SUB>  7H<SUB>2</SUB>O - 0.5g</P></LI><LI><P>Trace elements solution - 1mL </P></LI><LI><P>FeCl<SUB>3</SUB> solution - 1mL</P></LI><LI><P>L- Asparagin - 1.5g</P></LI><LI><P>Agar - 20g</P></LI><LI><P>Thiamine (10mg/100mL) - 1.2mL </P></LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><P>Preparation of the media
</P><OL><LI><P>Weigh and add the required solid components (except agar).</P></LI><LI><P>Add the required liquid components in a bottle.</P></LI><LI><P>Dissolve in demineralized water until the total volume is 900ml. </P></LI><LI><P>Stir the liquid using a magnet (no pH adjustment is required).</P></LI><LI><P>Add the required quantity of agar. </P></LI><LI><P>Add demineralized water until the desired total volume (1L). </P></LI><LI><P>Autoclave </P></LI><LI><P>After sterilization add 1.2 ml of filter sterilized thiamine.</P></LI></OL></DIV></DIV></DIV><BUTTON id="protoandtransfofasper" class="collapsible">Protoplastation and Transformation of <I>Aspergillus</I> spp.</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P>
Protocol provided by Fabiano Jares from DTU Bioengineering. The protocol is for species of <I>Aspergillus</I>. We used it for <I>Aspergillus oryzae</I>.
</P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><UL><LI><P><I>Aspergillus</I> protoplastation buffer (APB) 1L solution</P></LI><UL><LI><P>Final conc: 1.1 M MgSO<SUB>4</SUB> and 10 mM Na-phosphate buffer. (Hint: use 1 M solutions of Na<SUB>2</SUB>HPO<SUB>4</SUB> and NaH<SUB>2</SUB>PO<SUB>4</SUB> to prepare the sodium phosphate buffer). pH is adjusted with 2 N NaOH to 5.8.</P></LI></UL><LI><P>APB with Glucanex (40 mg Glucanex/ml APB) </P></LI><LI><P><I>Aspergillus</I> transformation buffer (ATB) 1 L solution</P></LI><UL><LI><P>Final conc: 1.2 M Sorbitol; 50 mM CaCl<SUB>2</SUB>·2 H<SUB>2</SUB>O; 20 mM Tris; and 0.6 M KCl. pH is adjusted with 2 N HCl to 7.2.</P></LI></UL><LI><P>PCT (200 mL stock solution) </P></LI><UL><LI><P>Final conc: 1.2 M Sorbitol; 50 mM CaCl<SUB>2</SUB>·2 H<SUB>2</SUB>O; 20 mM Tris; and 0.6 M KCl. pH is adjusted with 2 N HCl to 7.2.</P></LI><LI><P>Final conc: 50 % w/vol PEG 8000; 50 mM CaCl<SUB>2</SUB>;  20 mM Tris; and 0.6 M KCl. pH is adjusted with 2 N HCl to 7.5. Store PCT at 4 °C.</P></LI></UL></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><P>Protoplastation
</P><OL><LI><P>Collect the conidia from a plate by adding 5 mL of sterile liquid MM (or YPD for <I>A. niger</I>) with required supplements and firmly rubbing the colonies with a sterile Drigalski spatula. The conidial suspension is withdrawn from the plate and added to the shake flask containing 100 mL of media.</P></LI><LI><P>The culture is incubated over night (or 2 days for <I>A. niger</I> and other strains growing slower) at appropriate temperature and 150 rpm.</P></LI><LI><P>Harvest the mycelia and germlings by using Mira cloth.</P></LI><LI><P>Wash the mycelia with <I>Aspergillus</I> protoplastation buffer (APB) to remove the liquid media from the mycelia.</P></LI><LI><P>Resuspend the mycelium in 10-20 mL APB solution containing 40 mg Glucanex/mL APB. Homogenize mycelial and enzyme suspension gently to obtain the best possible digestion of the fungal cell wall.</P></LI><LI><P>Shake at the 30°C and 150 rpm for 2-3 hours.</P></LI><LI><P>Filter through Mira cloth and collect the flow through.</P></LI><LI><P>Add APB up to the total volume 40 mL.</P></LI><LI><P>Carefully make an overlay with 5 mL of 2 fold diluted <I>Aspergillus</I> transformation buffer (ATB). Dilute with sterile Milli-Q H2O.</P></LI><LI><P>Centrifuge at 3000xg (acceleration 9, deceleration 4) for 12 min.</P></LI><LI><P>Upon a successful protoplastation, a halo of white protoplast slurry is caught just below the surface. Collect protoplast slurry and transfer it to new tube.</P></LI><LI><P>Add ATB up to the total volume 40 mL.</P></LI><LI><P>Centrifuge at 3000xg (acceleration 9, deceleration 9) for 12 min. Discard supernatant.</P></LI><LI><P>Resuspend the protoplasts in approx. 1 mL ATB.</P></LI></OL><P>Transformation
</P><OL start="15"><LI><P>Gently mix DNA and protoplast in an eppendorf tube (for a simple transformation 50μL protoplast is enough).</P></LI><LI><P>Add 150 μL PCT.</P></LI><LI><P>Mix by inversion of the tube.</P></LI><LI><P>Incubate 10 min at room temperature.</P></LI><LI><P>Add 250 μL ATB and mix.</P></LI><LI><P>Plate on osmotic stabilized, selective media.</P></LI></OL></DIV><DIV class="col-xs-12"><H4 class="media heading">Notes</H4><OL><LI><P> The volume of DNA in water should preferably be kept below 25 % of the total volume of DNA-protoplast mix to avoid osmotic stress in protoplasts. The amount of DNA needed for a successful transformation varies with the type of DNA substrate.</P></LI><UL><LI><P>For linear DNA, add ≈10 μL linearized plasmid</P></LI><LI><P>Use self-replicating plasmids as positive controls to test the competence of the protoplasts and evaluate the success of transformation. AMA1 plasmids (pLAT4-3): add ≈2 μL miniprep.</P></LI><LI><P>Remember to include a negative control plate, where protoplasts are plated without any DNA.</P></LI></UL></OL></DIV></DIV></DIV><BUTTON id="ganodermaprotocol" class="collapsible">Protoplastation of <I>Ganoderma</I></BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P>
Protocol for protoplastation of <I>Ganoderma resinaceum</I> provided by Ecovative Designs. This is the original protocol that we received from Ecovative. Modifications have been made every single time it has been used due to limitations in our own lab.  
</P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><UL><LI><P>Glucanex (Sigma L1412)</P></LI><LI><P>Driselase (Sigma D9515)</P></LI><LI><P>Potato Dextrose Agar (pre-mix) + 5g/L Bacteriological Agar</P></LI><LI><P>Potato Dextrose Broth (pre-mix)</P></LI><LI><P>Sterile Millipore Water</P></LI><LI><P>Osmotic Buffer (500mL)</P></LI><UL><LI><P>74.04g MgSO<SUB>4</SUB> (0.6M)</P></LI><LI><P>1.05g MOPS (10mM)</P></LI><LI><P>Bring volume to 500mL with MilliQ water</P></LI><LI><P>Autoclave</P></LI></UL><LI><P>Sorbitol Solution (500mL)</P></LI><UL><LI><P>91.1g D-sorbitol (1M)</P></LI><LI><P>1.05g MOPS (10mM)</P></LI><LI><P>Bring volume to 500mL with Millipore water</P></LI><LI><P>Autoclave</P></LI></UL><LI><P>Lysing Solution (10mL)</P></LI><UL><LI><P>Add 100mg of Driselase to 5mL osmotic buffer in a 50mL centrifuge tube.</P></LI><LI><P>Allow to dissolve for 15-20min at room temperature with gentle swirling.</P></LI><LI><P>Add 100mg of Lysing Enzyme from <I>Trichoderma harzianum</I> (Glucanex) to 5mL osmotic buffer in a 50mL centrifuge tube.</P></LI><LI><P>Allow to dissolve for 15-20min at room temperature with gentle swirling.</P></LI><LI><P>Centrifuge the Driselase solution at 10,000xg for 1min to pellet the starch component.</P></LI><LI><P>Pour the supernatant into the tube with the Glucanex solution.</P></LI><LI><P>Attach a 30mm diameter 0.22um syringe filter to a 5mL syringe.</P></LI><LI><P>Fill the syringe with the lysing solution and filter into a 15mL centrifuge tube.</P></LI><LI><P>Pour the rest of the lysing solution into the syringe and filter.</P></LI><LI><P>Final, sterile lysing solution is 10mg/mL Driselase and 10mg/mL Glucanex in osmotic buffer.</P></LI></UL></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><P>Tissue Generation
</P><OL><LI><P>Inoculate potato dextrose agar plates with a single piece of tissue from a stock plate.</P></LI><LI><P>Parafilm and incubate at 30°C for 5 days.</P></LI><LI><P>Scrape tissue from the agar surface and place into 1mL of potato dextrose broth in a microcentrifuge tube.</P></LI><LI><P>Avoid bringing any agar into the tube.</P></LI><LI><P>Vortex the tubes at full speed for 30-60sec.</P></LI><LI><P>Place tubes in a tube rack and secure with tape.</P></LI><LI><P>Place the tube rack in a shaker incubator so that the tubes are completely horizontal.</P></LI><LI><P>Incubate at 30°C and 85rpm for 24hrs.</P></LI></OL><P>Enzymatic digest
</P><OL start="9"><LI><P>Centrifuge tubes at 10,000xrcf for 5min.</P></LI><LI><P>Gently discard supernatant.</P></LI><LI><P>Re-suspend tissue pellet in 1mL sterile water.</P></LI><LI><P>Invert tube several times and vortex briefly. Do not vortex too long or mycelium will begin to shear.</P></LI><LI><P>Centrifuge tubes at 10,000xrcf for 5min.</P></LI><LI><P>Gently discard supernatant.</P></LI><LI><P>Re-suspend tissue pellet in 1mL osmotic buffer.</P></LI><LI><P>Invert tube several times and vortex briefly.</P></LI><LI><P>Centrifuge at 10,000xrcf for 5min.</P></LI><LI><P>Gently discard supernatant.</P></LI><LI><P>Re-suspend tissue pellet in 1mL osmotic buffer.</P></LI><LI><P>Invert tube several times and vortex briefly.</P></LI><LI><P>Centrifuge at 10,000xrcf for 5min.</P></LI><LI><P>Gently discard supernatant.</P></LI><LI><P>Re-suspend tissue pellet in 1mL lysing solution.</P></LI><LI><P>Invert tube several times, but do NOT vortex.</P></LI><LI><P>Place tubes in a tube rack and secure with tape.</P></LI><LI><P>Place the tube rack in a shaker incubator so that the tubes are completely horizontal.</P></LI><LI><P>Incubate at 30°C and 85rpm for 19-24hrs.</P></LI></OL><P>Isolation of Protoplasts
</P><OL start="28"><LI><P>Invert tube several times and vortex to detach and protoplasts from the central cell mass.</P></LI><LI><P>Place a 40µm cell strainer in a new 50mL centrifuge tube.</P></LI><LI><P>Use a 1000µL pipette to further suspend the lysate. Pipette the lysate into the cell strainer and allow to drain fully.</P></LI><LI><P>Pipette 1mL of the sorbitol solution into the microcentrifuge tube used in the digestion and invert for 1min to rinse any additional protoplasts.</P></LI><LI><P>Pipette this volume through the same cell strainer and allow to drain fully. Cell strainer can be gently spun to remove more of the solution.</P></LI><LI><P>Repeat steps 3-5 for remaining protoplast preps (up to 5 additional 1mL preps).</P></LI><LI><P>Pour an additional 6-10mL of the sorbitol solution into the cell strainer and allow to fully drain.</P></LI><LI><P>Remove the cell strainer from the centrifuge tube.</P></LI><LI><P>Gently invert the tube 5-10 times to thoroughly mix the solutions.</P></LI><LI><P>Place the tube in a tube rack at 4 °C and incubate for at least 24hrs.</P></LI><LI><P>Centrifuge at 10,000xg for 10min.</P></LI><LI><P>Carefully pour the supernatant into a new 50mL centrifuge tube. Try to avoid disturbing the pellet along the side and bottom of the tube. The supernatant can be stored at 4 °C and more protoplasts may be able to be isolated at a later date, if needed.</P></LI><LI><P>Re-suspend the pellet into 1mL of sorbitol solution and transfer to a new 1.5mL microcentrifuge tube.</P></LI><LI><P>Prepare a 1:10 dilution in sorbitol solution and count protoplasts via hemocytometer.</P></LI><LI><P>Store protoplast suspension at 4 °C.</P></LI></OL></DIV><DIV class="col-xs-12"><H4 class="media heading">Modifications and notes</H4><OL><LI><P> There were no Driselase in our lab, so we did the protocol with three different concentrations of Glucanex: 20 mg/mL, 30 mg/mL and 40 mg/mL for the lysing solution after consulting Ecovative. </P></LI><LI><P>Second time we tried combining the protoplastation protocol we already had for <I>A. oryzae</I>. We used different conditions in three places with all in all 16 samples: 8 of them were incubated with a shaking speed of 85 rpm and 8 at a speed of 150 rpm. Of these 8, 4 of them were incubated for 2-3 hours and the 4 others for 19-24 hours. Each of the four samples had a different concentration of Glucanex for the digestion: 10 mg/mL, 20 mg/mL, 30 mg/mL and 40 mg/mL.  As there were no cell strainers in the lab, autoclaved funnels with mira cloth were used instead. </P></LI><LI><P>The third time we worked with 32 samples instead of 16. The conditions were still the same as the previous run but for an increased amount of mycelium we ran 2 two samples for each unique combination of conditions. Aside from this we also let the samples incubate for twice as long in the tissue generation part.</P></LI><LI><P>The fourth and last time we tried the protocol we had acquired Driselase and were, therefore, able to do the lysing solution according to the protocol. We still had no cell strainers, though, so funnels were still used. Aside from this 3 samples were made, all the same, and they were pooled together during the isolation of the protoplasts.</P></LI></OL><P>Posted with permission from Ecovative Designs</P></DIV></DIV></DIV><BUTTON id="sporesuspension" class="collapsible">Spore Suspension</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P>
To obtain the spores generated from a plate culture in agar. 
</P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><UL><LI><P>Milli-Q water </P></LI><LI><P>Mira-cloth</P></LI><LI><P>Autoclaved Funnels</P></LI><LI><P>Falcon tubes (50 mL)</P></LI><LI><P>Drigalski spatula</P></LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><P>Spore suspension
</P><OL><LI><P>Pour Milli-Q water into the fungal culture with plates. </P></LI><LI><P>Rub the plate with the Drigalski spatula. Make sure the spores get into solution. Spores are hydrophobic and can spread easily into the surroundings. </P></LI><LI><P>After spores are in suspension, if more plates with the same spores are to be harvested, pour water and spores to the next plate. Repeat steps 1-3 for as many plates you have. </P></LI><LI><P>Set a filter (funnel + mira cloth) on a Falcon tube. </P></LI><LI><P>Pour Milli-Q water + spores onto the mira cloth and filter solution. Make sure all liquid goes through the filter. </P></LI><LI><P>Do a second filtration on a new Falcon tube and filter. Make sure all liquid goes through. </P></LI></OL><P>Spore counting
</P><OL><LI><P>Make 1:100 dilution with the spore suspension (only 10 µL needed). </P></LI><LI><P>With a counting chamber under the optical microscope, put 5 µL of the dilution into the center of the chamber. Count spores in one of the squared cells. </P></LI><LI><P>Calculate spore concentration. </P></LI></OL></DIV></DIV></DIV></DIV><DIV class="interlabspace"><H2 style="text-align: left;margin-bottom:20px;">Building Experiments</H2><BUTTON id="fungalbrickprotocol" class="collapsible">Fungal brick protocol</BUTTON><DIV class="content"><DIV class="row"><DIV class="col-xs-12"><P>

The complete guide how to make and break fungal bricks for compression experiments
</P></DIV><DIV class="col-xs-4"><H4 class="media heading">Materials</H4><UL style="list-style-type:disc"><LI>Vortex</LI><LI>PDA</LI><LI>50 ml falcon tubes</LI><LI>Slicone ice-cube tray</LI><LI>Spore suspension</LI><LI>Substrate of interest</LI><LI>Incubator</LI><LI>Hydralic press</LI></UL></DIV><DIV class="col-xs-8"><H4 class="media heading">Procedure</H4><H5>Creating the bricks</H5><OL><LI>Fill 20 microliter sporesuspension in falcon tube</LI><LI>Mix sporesuspension with 50ml PDA.</LI><LI>Transfer 10 ml PDA/spore mix to 30 ml sustrate mix </LI><LI>Vortex the substrate PDA/spore mix for 30 seconds </LI><LI>Transfer mixture to the silicon tray  </LI><LI>Place tray in incubator for at least 1 week  </LI></OL><H5>Compression of bricks</H5><OL><LI>Wrap trays in aluminum foil</LI><LI>Poke holes all around the foil</LI><LI>Place the wraped trays in an oven high temperature for atleast 12 hours</LI><LI>Remove trays and press each cube with a Hydralic press</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footcontent" style="height:100%; position:relative;"><FOOTER id="myFooter"><DIV class="footcontainer"><DIV id="footer_row" class="row"><DIV id="footer_useful_links" class="col-xs-4"><H5>Useful links</H5><UL id="footer-item"><P class="footer-links" style="font-size: 16px;"><A href="https://2018.igem.org/Team:DTU-Denmark">Home</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Description">Project description</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Model">Modeling</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Parts">Parts overview</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Notebook">Notebook</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Team">Team</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Safety">Safety</A>•<A href="https://2018.igem.org/Team:DTU-Denmark/Human_Practices">Human practices</A></P></UL></DIV><DIV id="footer_about_us" class="col-xs-4" style:=""><H5>About us</H5><UL id="footer-item"><LI><A style="font-size:16px;" href="mailto:dtubiobuilders@gmail.com">dtubiobuilders@gmail.com</A></LI><LI><A style="font-size: 16px;" href="https://www.google.dk/maps/@55.7854419,12.5196676,16z"> Anker Engelunds Vej 1 Bygning 101A, 2800 Kgs. Lyngby, Denmark</A></LI></UL></DIV><DIV id="footer_social" class="col-xs-3"><BUTTON type="button" class="btn btn-default" onclick="window.location.href='mailto:dtubiobuilders@gmail.com'">Contact us</BUTTON></DIV></DIV></DIV></FOOTER></DIV></BODY></HTML>