Team:HZAU-China/Model
Project Description Design Results Demonstrate
Wetlab Experiments Improve InterLab Notebook
Drylab Model Software
Human Practices Safety Human Practices Public Engagement
About Us Team Attributions Collaborations
Parts Basic Composite
Salmonella infection model
We want to simulate the situation that tumor cells and Salmonella together in a liquid                     environment.                     We used the law of mass action to establish a model for the infection process of Salmonella,                     which                     is dimensionless.
$$N_{normal} + S_{almonella} \overset{Aw}{\rightarrow} N_{w} + S_{al\_normal}$$
$$N_{tumor} + S_{almonella} \overset{As}{\rightarrow} N_{s} + S_{al\_tumor}$$
$$S_{almonella} = S_{almonella}(t0) - N_{normal\_cell} - N_{tumor}$$
$$\dfrac {dN_{w}} {d_{t}} = A_{w} S_{almonella} N_{w}$$
$$\dfrac {dN_{s}} {d_{t}} = A_{s} S_{almonella} N_{s}$$
$$\dfrac {dS_{almonella}} {d_{t}} = - \dfrac {dN_{w}} {d_{t}} - \dfrac {dN_{s}} {d_{t}}$$
\(N_{normal\_cell}\): The density of normal cells.
\(S_{almonella}\): The density of Salmonella in the liquid environment.
\(N_{tumor}\): The density of tumor cells.
\(N_w\): The number of Salmonella in the normal cells.
\(N_s\): The number of Salmonella in the tumor cells.
\(A_w\): The affinity between Salmonella and normal cells.
\(A_s\): The affinity between Salmonella and tumor cells.
\(S_{al\_normal}\): The density of infected normal cells.
\(S_{al\_tumor}\): The density of infected tumor cells.
Salmonella begins to replicate two hours after infection1 .
$$\dfrac {dN_{sal}} {d_{t}}(t) =$$
$$\dfrac {dS_{al}} {d_{t}}(t) + S_{al}(t-2) 2^{\dfrac {t-2} {T}} \ln{2}\dfrac {1} {T} $$
\(T\): Cell cycle.
Identification of infection time
According to our experimental results, we noticed that Salmonella might follow Poisson                     distribution in                     cells, so we use Matlab to judge the distribution of bacteria in the cells. We assume                     that the area less than 1 in the Poisson distribution is a part of cells which are not infected by Salmonella. According to our experimental results, cells which are infected by only one Salmonella can also die of pyroptosis. Based on this feature, we divide cells into uninfected and infected                     cells. When the average number of bacteria in the cell changes, which means that the λ of Possion                     distribution changes, the ratio of the two kind of cells will change. In summary, when the average                     number of Salmonella in cells changes, the proportion of dead cells will change.
Figure 1. Poisson distribution and Salmonella infection results. a. Based on                     statistics of                     experimental results, we proved that the Salmonella follows Poisson distribution in normal                     cells. b. We assume that the area less than 1 in the Possion distribution is a part of cells which                     are not infected by Salmonella. When the λ of Possion distribution changes, which means the                     average number of Salmonella in cells changes, the proportion of infected cells changes. c. Cells                     which are infected by only one Salmonella can also die of pyroptosis.
Infection in tumor cell culture experiments
We hope that the mathematical model can help the Salmonella infection experiment. In our                     final                     phenotypic experiment, the cells carrying the GSDMD gene are induced by ATc, and we hope that the                     observed result is that the proportion of ATc-induced cell death is more than that of not induced                     to prove the ATc promoter is effective. In this experiment, the error may be big if the proportion                     of cells infected by Salmonella is different.
Figure 2. Results caused by efficiency                     differences of infection.
If the proportions of infection are different, the experimental results may not be able to prove that pyroptosis is induced by atc promoter. Figure 2 showed that the ATc promoter is induced and 90% cells death are caused and 70% cells die because of the promoter disclosure, but the difference of the proportions of infected cells is so big that the experimental results cannot reflect the real situation. We can reduce the difference by letting the infection proportions of the two kinds of cells both close to 100%.
We solved this problem by predicting the proportion of cells infected with bacteria over time.
Based on these, we designed an App with MatLab ( https://github.com/cccoolll/Pyroptosis.git )                    . In this App, different parameters obtained from measurement experiments can be input to predict the optimal infection time.                     Therefore, this App can provide guidance to the design of our experiments.
Figure 3. Salmonella infection prediction tool (for tumor cell).
A1: The predicted Salmonella numbers in tumor cell in the single cell                     infection experiment.
A2: The predicted proportions of infected tumor cells.
A3: The concentration of added Salmonella.
A4: The density of tumor cells;
A5: Rate constant of Salmonella infecting tumor cells.
A6: The proportion of tumor cells expected to be infected.
A7: The time to reach the wanted proportion of infected cells.
Figure 4. Salmonella infection prediction tool (for mixed culture of tumor cell and normal cell).
B1: The predicted numbers of Salmonella in tumor cell (red) and normal cell (blue) in a single cell infection                     experiments.
B2: The predicted proportions of infected cells (red for tumor cell and blue for normal cell).
B3: The concentration of added Salmonella.
B4: The density of tumor cells.
B5: Rate constant of Salmonella infecting tumor cells.
B6: The density of normal cells.
B7: Rate constant of Salmonella infecting normal cells.
B8: The predicted optimal infection time.
The parameters Nsal, Tumor and As are measured from our experiments.
Guidance to tumor cell infection experiments
Figure 5 (a part of Figure 3). Guidance for tumor cells infection experiments.
According to our experiment protocol, the MOI (multiplicity of infection) is 100, corresponding to the concentration of cells. If we want 98% of the tumor cells to be infected, the prediction result show that the infection time should be at least 2 hours to eliminate unnecessary variables.
Guidance to mixed culture experiments
Figure 6 (a part of Figure 4). The predicted proportions of infected cells (red for tumor                     cell and blue for normal cell).
In order to reflect the affinity of Salmonella to tumor cells and to normal cells, we                     hope that the difference between experimental results of tumor cells and normal cells is obvious.                     However, the numbers of bacteria in different host cells are difficult to count, and we can only count the                     number of infected cells and calculate their the proportions. Therefore, we need to                     predict the time when the difference is most obvious. Our App just can do this for us.
Chemical control model
Profile
The Tet repressor protein (tetR) regulates transcription of tetracyclines resistance protein, tetA.                     The expression of tetA must be strictly regulated since tetA is a membrane-spanning H+-[Tc-Mg]2+ antiporter which means it can lower the pH environment of cytoplasm. As a result, the natural                     circuit of tetracyclines regulation is a negative-feedback circuit2. Tc is the inducer,                     which shows                     high affinity to tetR protein. The tetR protein binds to tetO sequence on DNA specifically, thus                     inhibits the expression of Tet promoter. When Tc or other similar molecules like ATc                     (anhydrotetracycline) diffuse into bacteria, they will bind to tetR protein and unleash the tetR                     protein from DNA, and thus relieve the inhibition and start the expression of Tet promoter.
In our project, we choose ATc (anhydrotetracyclines) as the inducer. ATc is less harmful to                     bacteria than Tc and about 100-fold higher affinity to tetR than Tc2.
The ATc model aims to predict and solve two problems: first, how fast does the circuit react to                     ATc; second, how many target gene will express in the bacteria community under a certain                     concentration of ATc.
Hypothesis
There are two tetO sites on the Tet promoter and both can bind to tetR protein randomly and inhibit                     the promoter’s expression independently. To make the condition simple, we consider the two tetO                     sites into one as we just want to explain the relationship between the promoter inhibition and                     the tetR protein expression.
In our project, the ATc concentration in our incubation environment is uniform, and the diffusion                     rate of anhydrotetracycline can be ignored3. In spite of this, the degradation rate of                     ATc under 37℃ must be taken into account as reported4.
Based on these facts, we give the following hypotheses:
1. Regard two tetO operons as one equivalently.
2. Ignore the diffusion of ATc.
3. The reaction time between ATc and tetR, tetR and DNA is much shorter than transcription and                     translation.
Description and Equation
Reactions implicated:
$$tetR + [tetR - ATc_2] = tetR_{total}$$
$$tetR + 2 \times ATc = [tetR - ATc_2]$$
$$P_{tet} + [tetR_2 - P_{tet}] = [P_{tet}]_{total}$$
$$2 \times tetR + P_{tet} = [tetR_2 - P_{tet}]$$
$$$$
Equations5:
Based on Hill function, we can determine the amount of activated tetR, tetRact:
$$ tetR_{act} + n \times S_x(t) \rightarrow [tetR - S_x(t)_n] $$
$$ K_X = \dfrac {tetR_{act} \times S_x^n (t)} {[tetR - S_x(t)_n]} $$
$$ tetR = tetR_{act} + [tetR - S_x(t)_n] $$
$$ tetR_{act} = \dfrac {tetR} {1 + \dfrac {S_x^n (t)} {K_{X}}} $$
Based on Hill function, we can determine the amount of activated promoter, with which we can                     calculate the total transcription speed of all promoters per cell:
$$ P_{tet\_act} + n\cdot tetR_{act} \rightarrow [P_{tet} - (tetR_{act})_n] $$
$$ P_{tet\_copy} = P_{tet\_act} + [P_{tet} - (tetR_{act})_n] $$
$$ K_d = \dfrac {P_{tet\_copy} \times tetR^n_{act} } {[P_{tet} - (tetR_{act})_n]} $$
$$ A_{mRNA} = P_{tet\_act} \times beta $$
$$ A_{mRNA} = \dfrac {P_{tet\_copy} \times beta } { 1 + \dfrac {tetR^n_{act}} {K_{d}}} $$
Kinetic equations of transcription and translation:
$$ \dfrac {dmRNA} {dt} = A_{mRNA} - K_{deg\_mRNA} \times mRNA $$
$$ \dfrac {dtetR} {dt} = K_{trans\_tetR} \times mRNA - K_{deg\_tetR} \times tetR $$
$$ \dfrac {dGSDMD} {dt} = K_{trans\_GSDMD} \times - K_{deg\_GSDMD} \times GSDMD $$
Degradation function of ATc by time3:
$$ \dfrac {dS_x(t)} {d_t} = -K_{deg\_ATc} \times S_x(t) $$
$$ \ln(S_x(t)) = \ln(S_x(0)) - K_{deg\_ATc} \times t $$
Growth curve of bacteria based on logistics model from P. F. Verhulst:
$$ N(t) = \dfrac {K_{max}} {1 + C \cdot e^{-rt}} $$
Total GSDMD expressed in bacteria community:
$$ GSDMD_{total-amount} = N(t) \cdot GSDMD\cdot{CFU}\cdot{diluted-ratio}\cdot{V_{Bacteria volume}}$$
The symbols in the equations:
\(S_x(t)\): concentration of ATc, as a function of time.
\(tetR_{act}\): concentration of activated tetR.
\(tetR \): concentration of total tetR.
\(GSDMD \): concentration of GSDMD.
\(A_{mRNA} \): transcription rate constant of the promoter.
\(P_{tet\_copy} \): plasmid copy number.
\(K_X \): disassociation rate constant of tetR and ATc.
\(K_d \): disassociation rate constant of tetR and DNA.
\(beta \): original (unrepressed) transcription rate constant of the promoter.
\(K_{deg\_mRNA} \): degradation rate constant of mRNA.
\(K_{deg\_tetR} \): degradation rate constant of tetR.
\(K_{trans\_tetR} \): translation rate constant of tetR.
\(mRNA \): concentration of mRNA.
\(K_{deg\_GSDMD} \): degradation rate constant of GSDMD.
\(K_{trans\_GSDMD} \): transcription rate constant of GSDMD.
\(K_{deg\_ATc} \): degradation rate constant of ATc.
\(n \): Hill coefficient.
\(N(t) \): initial OD600 value of the bacteria.
\(r \): growth rate of the bacteria.
\(K_{max} \): maximum OD of the bacteria in cultivation.
Suggestions to our experiments (see Results )
As is hard to obtain the initial parameters in the equations above on our own without any                     experiments, the only way to obtain these parameters is to look up in former research or other                     teams work. Fortunately we got a copy of these parameters from team William and Mary iGEM 20166.                     These parameters include \(K_X = 0.36 \), \(K_d = 0.1 \), \(beta = 0.0023 \), \(K_{deg\_mRNA} =                     0.009 \), \(K_{deg\_tetR} = 0.631 \), \(K_{trans\_tetR} = 235.5 \) (All units are combined of                         nM and s). Considering that both Salmonella and E. coli are in Enterobacteriaceae, we assumed that in Salmonella these                     parameters are the same                     with those in E. coli since we just wanted to figure out a useful instruction to wet lab.
To gain the parameters in bacterial growth curve, we carried out an experiment to measure the growth                     of Salmonella. Then we fitted the obtained data into a logistics model. By doing these we                     figured out that                     \(r = 60min^{-1} \), \(K_{max} = 0.9997 \) and \(C = 7.2319 \). Results and diagram are shown below                     (Figure 7):
Figure 7. Bacterial growth curve.
After complete the work above, we used MATLABTM to solve the equations above and                     acquired a series of diagrams which visually demonstrated the relationships and helped the wet lab                     group get                     an overall view of how ATc influences the expression of GSDMD. We assumed that \(P_{tet\_copy} =                     4                     \), \(K_{deg\_GSDMD} = 0.8 \), \(K_{trans\_GSDMD} = 200 \), \(K_{deg\_ATc} = 0.0007 \) (All unites                     are                     combined of nM and s). In this action we didn’t take the growth of bacteria into account. From the diagrams, we can figure out that as ATc added increase, the concentration of GSDMD will also rise. Results are shown below (Figures 8, 9, 10.):
Figure 8. Concentration of tetR (nM) -                     time (s).
Figure 9. Concentration of GSDMD (nM) -                     time (s).
Figure 10. Max concentration of GSDMD (nM)                     -                     ATc Concentration (nM).
With this software, one can adjust all the parameters needed in the equations above and                     attain the diagrams which indicates the relations between concentration of GSDMD and time,                     concentration tetR and time and the maximum value of GSDMD and the initial concentration of ATc.                     The                     program will also generate a function describing the relationship between the maximum concentration                     of                     GSDMD and the concentration of ATc. With the help of this program, members in wet lab group can                     conveniently decide how much ATc should be added into cultivation environment according to their                     requirements (Figures 11, 12). Be advised that users must multiply the CFU number, bacteria cell volume and diluted ratio to the data obtained from this app to gain a final result.
Figure 11. Software parameters.
Figure 12. Software diagrams.
Significance
The model of ATc induced circuit is very common and well-known to biology researchers. The                     common-known significance to this model is that it can demonstrate the relationship between                     concentration of target gene and concentration of inducer added, which can instruct the researchers                     modulate their circuit precisely. In our project, this model will tell the members in wet lab group                     that how much GSDMD will be expressed under a certain concentration of ATc in the Salmonella community formed in the tumor cell.                     Another significance for this model is that, the response time is very short and the response                     speed is extremely fast. We anticipate that just minutes are needed to induce the fluorescence.                     This phenomenon is also verified in our experiment. In less than 10 minutes, fluorescence can be                     detected under fluorescence microscope.                     Especially, a remarkable significance to our project is that it’s a self-destructive system, which                     means, without any further operation, the process of induction can be self-terminated. As ATc                     degrades, the expression of GSDMD will significantly decreases, thus the process of pyroptosis can                     be inhibited. Based on the features, we think that the cytokine storm caused by pyroptosis                     is controllable.
The source code of the software and the scripts used above can be found following this link:
https://github.com/tom13amy/atc_modelling_software
Reference
1. I. Hautefort, A. Thompson, et al. During infection of epithelial cells Salmonella enterica                     serovar Typhimurium undergoes a time-dependent transcriptional adaptation that results in                     simultaneous expression of three type 3 secretion systems. Cellular Microbiology 10(4), 958–984                     (2008).
2. Berens, C. & Hillen, W. Gene regulation by tetracyclines: Constraints of resistance regulation in                     bacteria shape TetR for application in eukaryotes. Eur. J. Biochem. 270, 3109–3121 (2003).
3. Nevozhay, D., Adams, R. M., Murphy, K. F., Josic, K. & Balazsi, G. Negative autoregulation                     linearizes the dose-response and suppresses the heterogeneity of gene expression. Proc. Natl. Acad.                     Sci. 106, 5123–5128 (2009).
4. Politi, N. et al. Half-life measurements of chemical inducers for recombinant gene expression.                     J. Biol. Eng. 8, 1–10 (2014).
5. Alon, U. An Introduction to Systems Biology: Design Principles of Biological Circuits (Chapman & Hall/CRC Mathematical and Computational Biology).Pdf.
6. William and Mary iGEM 2016. A Kinetic Model of Molecular Titration. 1–11 (2016).
Model
Salmonella infection model
Chemical control model
Reference
Back to Top
Software
