<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HZAU-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HZAU-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="daohang"><UL class="caidan"><LI class="hiLight shortName"><A class="nav_head" href="https://2018.igem.org/Team:HZAU-China"><SPAN>Project</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Description">Description</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Design">Design</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Results">Results</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Demonstrate">Demonstrate</A></LI><LI class="hiLight shortName"><A class="nav_head" href="#"><SPAN>Wetlab</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Experiments">Experiments</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Improve">Improve</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/InterLab">InterLab</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Notebook">Notebook</A></LI><LI class="hiLight shortName"><A class="nav_head" href="#"><SPAN>Drylab</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Model">Model</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Software">Software</A></LI><LI class="hiLight longName"><A class="nav_head" href="#"><SPAN>Human Practices</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Safety">Safety</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Human_Practices">Human Practices</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Public_Engagement">Public Engagement</A></LI><LI class="hiLight shortName"><A class="nav_head" href="#"><SPAN>About Us</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Team">Team</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Attributions">Attributions</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Collaborations">Collaborations</A></LI><LI class="hiLight shortName"><A class="nav_head" href="#"><SPAN>Parts</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Basic_Part">Basic</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Composite_Part">Composite</A></LI></UL></NAV></P><DIV class="neirong clearfix"><DIV class="zhengwen"><DIV id="float01" class="cur"><DIV class="h1">Chassis</DIV><DIV class="h2"><I>sifA</I> knock out</DIV><P>SifA maintains the integrity of <I>Salmonella</I>-containing vacuole (SCV) where <I>Salmonella</I>
                    survive and
                    replicate<SUP>1</SUP>. The existence of SCV limits the releasing of GSDMD-N275 into cytoplasm. In
                    addition,
                    the growth of inhibition of Δ<I>sifA</I> mutant in macrophage is remarkable<SUP>2</SUP>. Thus, we
                    knocked out the
                    <I>sifA</I> gene in order to decrease the stability of SCV and reduce the virulence of <I>Salmonella</I>.
                </P><P>Δ<I>sifA</I> mutant was constructed by using gene editing system based on two-step allelic exchange<SUP>3</SUP>.</P><P>PCR verification indicated that the chromosomal gene <I>sifA</I> was knocked out (<B>Figure 1</B>).
                    Primers named
                    DSIFA F and DSIFA R were used in this PCR (<B>Figure 2</B>).</P><P><B>Figure 1.</B> PCR verification of <I>sifA</I> gene knock out. Lane 1 refers to Δ<I>sifA</I> mutant.
                    Lane 2 refers
                    to WT <I>Salmonella enterica</I> serovar Typhimurium strain SL1344 as a control.</P><P><B>Figure 2.</B> DSIFA F and DSIFA R are the primers in the ORF of <I>sifA</I>. This pair of primers
                    can produce a 436bp product in WT.</P><DIV class="h2">Safety</DIV><P><I>SifA</I> is essential for maintaining vacuolar membrane stability. Cytosolic bacterium can be generated
                    by the function loss of SCV. This population of bacterium was
                    easily
                    recognized and cleaned up by macrophage. Hence, Δ<I>sifA</I> mutant decreases the toxicity to the
                    host.
                    Microscopy demonstrated that Δ<I>sifA</I> mutant was defective for replication in macrophage (<B>Figure
                        3</B>).
                </P><P><B>Figure 3.</B> Microscopy of immortalized bone-marrow-derived macrophages (iBMDM) infected with
                    the Δ<I>sifA</I>
                    mutant and WT <I>Salmonella</I> SL1344, respectively. These strains contain high copy number plasmids to
                    express RFP constitutively.</P><DIV class="collapseDiv"><LABEL for="zhedie-toggle1">Method</LABEL><DIV id="zhedie1" class="text-success text-left"><B>Preparation of Cells for Infection</B>
                        1. Grow iBMDMs in a humidified 37 °C, 5% CO<SUB>2</SUB> tissue-culture incubator.
                        2. Count the cells using a hemocytometer. Seed in 24-well (5×10^4 per well) and grow
                        overnight .<B>Preparation of the Bacterial Cells</B>
                        1. Grow bacterial cells overnight 16 h in 2 mL LB in a 15-mL tube. Incubate at 37 °C in a shaking
                        incubator (200 rpm).
                        2. Subculture bacterial cells by transferring 300 μL of the overnight culture into 5 mL of LB in a
                        loosely capped 50-mL tube. Incubate at 37 °C in a shaking incubator (200 rpm) to late log
                        phase.
                        3. Pellet 1 mL of the <I>Salmonella</I> bacteria cells subculture by centrifugation at 1,000×g in a microfuge
                        for 2
                        min at room temperature.
                        4. Remove 900 μL of supernatant and gently resuspend the pellet in 900 μL PBS.<B>Infection</B>
                        1. Aspirate media and rinse the monolayer twice with PBS.
                        2. Inoculate cells with bacterial cells (MOI = 20) by adding bacterial cells directly to the cell-culture
                        supernatant. Centrifugate at 110 g for 5 min at room temperature.
                        3. Incubate for 25 min at 37 °C in 5% CO<SUB>2</SUB>.
                        4. Aspirate media and rinse the monolayer twice with PBS, to remove extracellular bacterial cells.
                        5. Add fresh GM containing 100 μg/mL gentamicin and incubate for 2 h at 37 °C in 5% CO<SUB>2</SUB>.
                        6. Replace GM with fresh GM containing 20 μg/mL gentamicin for remainder of experiment.
                        Observation is taken after 11 h.</DIV></DIV></DIV><DIV id="float02"><DIV class="h1">Targeting</DIV><P>RGD motif can specifically bind to alpha V beta 3 (αvβ3), a well-known biomarker on the surface of
                    tumor cells. We express RGD motif fused with OmpA to surface display it on the outer membrane of
                    bacterial cells. Microscopy shows that bacterial cells expressed Lpp-OmpA-RGD induced by 0.1 mM IPTG can bind to
                    αvβ3-positive MDA-MB-231 cell line (<B>Figure 4</B>), but cannot bind to αvβ3-negative MCF7 cell line (<B>Figure 5</B>). These results demonstrated that the bacterium gain the function of tumor targeting though display RGD motif. </P><P><B>Figure 4.</B> Microscopy of αvβ3-positive MDA-MB-231 cell line were incubated with <I>E. coli</I> 
                    which
                    constructively expressed RFP and inductively expressed RGD motif under the control of <I>lac</I> promoter. The locations of bacteria cells are pointed by red arrow. </P><P><B>Figure 5.</B> Microscopy of αvβ3-negative MCF7 cell line were incubated with <I>E. coli</I> which
                    constructively
                    expressed RFP and inductively expressed RGD motif under the control of <I>lac</I> promoter.</P><DIV class="collapseDiv"><LABEL for="zhedie-toggle2">Method</LABEL><DIV id="zhedie2" class="text-success text-left"><B>Preparation of Cells</B>
                        1. Grow MDA-MB-231 and MCF7 in a humidified 37 °C, 5% CO<SUB>2</SUB> tissue-culture incubator.
                        2. Count the cells using a hemocytometer. Seed in 24-well (1.5× 10^5 per well) and grow
                        overnight.<B>Preparation of Bacterial cells</B>
                        1. Grow bacterial cells overnight 14 h in 2 mL LB in a 15-mL tube. Incubate at 37 °C in a shaking
                        incubator (200 rpm).
                        2. Subculture bacterial cells by transferring 300 μL of the overnight culture into 5 mL of LB
                        containing 0.1 mM IPTG in a loosely capped 50-mL tube. Incubate at 37 °C in a shaking incubator
                        (200 rpm) to early stationary phase.
                        3. Pellet 1 mL of the bacterial cells subculture by centrifugation at 1000 g in a microfuge for 2 min
                        at room temperature.
                        4. Remove 900 μL of supernatant and gently resuspend the pellet in 900 μL PBS.<B>Infection</B>
                        1. Aspirate media and rinse the monolayer twice with PBS.
                        2. Inoculate cells with bacterial cells (MOI = 100) by adding bacterial cells directly to the cell-culture
                        supernatant.
                        3. Incubate for 1.5 h at 37 °C in 5% CO<SUB>2</SUB>.
                        4. Aspirate media and rinse the monolayer three times with PBS.
                        Observation is taken immediately.</DIV></DIV></DIV><DIV id="float03"><DIV class="h1">Function of GSDMD</DIV><DIV class="h2">The N-terminal of GSDMD performs the function of cell pyroptosis</DIV><P>We fused eGFP with GSDMD-N275 (N-terminal 275 amino acids) and GSDMD FL (full length), respectively. Then the corresponding
                    plasmids were transfected into Hela GSDMD knockout (KO) cell. Cell microscopy showed that the cells
                    transfected with GSDMD-N275 underwent pyroptosis while the cells with GSDMD FL did not (<B>Figure 6</B>).
                    We also tested the cell viability through an ATP assay (CellTiter-Glo<SUP>®</SUP> Luminescent Cell Viability
                    Assay) and demonstrated that GSDMD-N275 and mutants of GSDMD FL have different abilities to induce
                    pyroptosis (<B>Figure 7</B>).</P><P><B>Figure 6.</B> Microscopy of the Hela GSDMD KO cells transfected with pCS2-eGFP-GSDMD FL (above) and
                    pCS2-eGFP-GSDMD-N275 (below), respectively. Pyroptotic cells are pointed by red arrow.</P><DIV class="collapseDiv"><LABEL for="zhedie-toggle3">Method</LABEL><DIV id="zhedie3" class="text-success text-left"><B>Preparation of Cells for transfection</B>
                        1. Grow Hela GSDMD KO cells in a humidified 37 °C, 5% CO<SUB>2</SUB> tissue-culture incubator.
                        2. Count the cells using a hemocytometer. Seed in 24-well (5 × 10^4 per well) and grow.<B>Transfection</B>
                        1. Dilute 0.5 μg DNA into 50 μl jetPRIME<SUP>®</SUP> buffer (supplied). Mix by vortexing.
                        2. Add 1 μl jetPRIME<SUP>®</SUP>, vortex for 10 s, spin down briefly.
                        3. Incubate for 10 min at RT.
                        4. Add 50μl of transfection mix per well drop wise onto the cells in serum containing medium,
                        and distribute evenly.
                        5. Gently rock the plates back and forth and from side to side.
                        6. If needed, replace transfection medium after 4 h by cell growth medium and return the plates
                        to the incubator.
                        Observation is taken after 1.5 h.</DIV></DIV><P><B>Figure 7.</B> Cell viability of the 293T cells transfected with pCS2-Flag-GSDMD FL,
                    pCS2-Flag-GSDMD-N275, pCS2-Flag-GSDMD L290D, pCS2-Flag-GSDMD Y373D, pCS2-Flag-GSDMD A377D,
                    respectively. Asterisks indicate the statistically significant differences. ATP-based cell
                    viability was measured (n=6). </P><DIV class="collapseDiv"><LABEL for="zhedie-toggle4">Method</LABEL><DIV id="zhedie4" class="text-success text-left"><B>Preparation of Cells for ATP assay</B>
                        1. Grow HEK293T cells in a humidified 37 °C, 5% CO<SUB>2</SUB> tissue-culture incubator.
                        2. Count the cells using a hemocytometer. Seed in 96-well (1 × 10^4 per well) and grow
                        overnight.
                        3. Transfect 0.5 μg DNA per well.
                        4. Equilibrate the plate and its contents at room temperature for approximately 30 minutes
                        after 20 h.
                        5. Add a volume of CellTiter-Glo<SUP>®</SUP> Reagent equal to the volume of cell culture medium present in
                        each well. (add 100 μl of reagent to 100 μl of medium containing cells for a 96-well plate).
                        6. Mix contents for 2 minutes on an orbital shaker to induce cell lysis.
                        7. Allow the plate to incubate at room temperature for 10 minutes to stabilize luminescent
                        signal.
                        8. Record luminescence.</DIV></DIV></DIV><DIV id="float04"><DIV class="h1">Induce the expression of GSDMD-N275</DIV><DIV class="h2">Intracellular environment-dependent expression</DIV><P>P<SUB>sifA</SUB> is a intracellular environment-dependent promoter. SipD is required for bacterial
                    internalization.
                    Δ<I>sipD</I> mutant cannot enter host cells. Thus, we use this strain as a control. Microscopy
                    suggested
                    that only intracellular bacteria express eGFP which is under the control of P<SUB>sifA</SUB> (<B>Figure
                        8</B>). </P><P><B>Figure 8.</B> Microscopy of hela GSDMD KO cells infected with the Δ<I>sipD</I> or Δ<I>sifA</I>
                    mutant, respectively.
                    These mutants contain low copy number plasmids to express eGFP which was regulated by P<SUB>sifA</SUB>.</P><DIV class="collapseDiv"><LABEL for="zhedie-toggle5">Method</LABEL><DIV id="zhedie5" class="text-success text-left"><B>Preparation of Cells for Infection</B>
                        1. Grow Hela GSDMD KO cells in a humidified 37 °C, 5% CO<SUB>2</SUB> tissue-culture incubator.
                        2. Count the cells using a hemocytometer. Seed in 24-well (9× 10^4 per well) and grow
                        overnight.<B>Preparation of Bacterial Cells</B>
                        1. Grow bacterial cells overnight 16 h in 2 mL LB in a 15-mL tube. Incubate at 37 °C in a shaking
                        incubator (200 rpm).
                        2. Subculture bacterial cells by transferring 300 μL of the overnight culture into 5 mL of LB in a
                        loosely capped 50-mL tube. Incubate at 37 °C in a shaking incubator (200 rpm) to late log
                        phase.
                        3. Pellet 1 mL of the <I>Salmonella</I> subculture by centrifugation at 1000 g in a microfuge
                        for 2
                        min at room temperature.
                        4. Remove 900 μL of supernatant and gently resuspend the pellet in 900 μL PBS.<B>Infection</B>
                        1. Aspirate media and rinse the monolayer twice with PBS.
                        2. Inoculate cells with bacterial cells (MOI = 100) by adding bacterial cells directly to the cell-culture
                        supernatant.
                        3. Incubate for 3 h at 37 °C in 5% CO<SUB>2</SUB>.
                        4. Aspirate media and rinse the monolayer twice with PBS.
                        5. Add fresh GM containing 100 μg/mL gentamicin and incubate at 37 °C in 5% CO<SUB>2</SUB>.
                        Observation is taken after 2 h.</DIV></DIV><DIV class="h2">ATc-dependent expression</DIV><DIV class="h3">GSDMD-N275 can lyse bacteria</DIV><P>Expression of the N-terminal of GSDMD fused with eGFP (eGFP-GSDMD-N275) in <I>Salmonella enterica</I>
                    serovar Typhimurium str. SL1344 Δ<I>sifA</I> is under the control of P<SUB>tet</SUB>. The
                    colony-forming unit (CFU)
                    was measured for counting the number of viable bacterial cells (<B>Figure 9</B>). This result shows that
                    eGFP-GSDMD-N275 exhibits cytotoxicity in bacteria.</P><P><B>Figure 9.</B> CFU comparison between the SL1344 Δ<I>sifA</I> cells with eGFP-GSDMD-N275 plasmid
                    and with the
                    empty vector. In each group, ATc (15μg/ml) was added into medium when bacterium grew to logarithmic
                    phase (OD = 0.6~0.8). Vector refers to bacterium containing a high copy number plasmid which only
                    expresses TetR under the control of P<SUB>tet</SUB>. CFU for vector
                    and eGFP-GSDMD-N275 are shown in the logarithmic form (log10) (n=3).</P><DIV class="collapseDiv"><LABEL for="zhedie-toggle6">Method</LABEL><DIV id="zhedie6" class="text-success text-left">
                        1. Bacteria are cultured overnight in LB broth containing corresponding antibiotics, and
                        dilute each 1 volume overnight cultures with 100 volume fresh LB containing antibiotics.
                        Culture in 37℃ 200 rpm.
                        2. When OD reaching to 0.6-0.8, add anhydrotetracycline with final concentration of
                        μg/ml to induce the expression of EGFP-GSDMD-N275.
                        3. Take 100 μl diluted culture to plate on LB agar plates containing appropriate concentration
                        of antibody after 1.5 hours of induce.
                        Observation is taken overnight.</DIV></DIV><DIV class="h3">GSDMD-N275 from lytic bacteria induces host cell pyroptosis</DIV><P>Expression of the N-terminal of GSDMD fused with eGFP (eGFP-GSDMD-N275) is under the control of tet
                    promoter in Δ<I>sifA</I> SL1344. Hela GSDMD KO cells were infected with Δ<I>sifA</I> SL1344.
                    Inducer ATc
                    (16μg/mL) were added 3h after infection. Microscopy shows that eGFP-GSDMD-N275 located in cytoplasm
                    after 5 min of induction and triggered pyroptosis after 30 min of induction (<B>Figure 10</B>). After
                    1.5 h
                    of induction, Hela GSDMD KO cells underwent second necrosis caused by bacterial infection without
                    inducer. Morphology of this process is similar to pyroptosis<SUP>4</SUP>. Thus, the population of
                    ruptured
                    cells was counted. There is 1.96 fold change between control group and induced group (<B>Figure 11</B>).
                    So
                    the pyroptosis of host cell in the induced group was triggered by eGFP-GSDMD-N275 not by bacterial
                    infection.</P><P>In these experiments, the choice of MOI (multiplicity of infection) and infection time were directed by <A href="https://2018.igem.org/Team:HZAU-China/Model">Modeling</A>.</P><P><B>Figure 10.</B> Hela GSDMD KO cells were infected with Δ<I>sifA</I> SL1344 containing high copy
                    number plasmids
                    which express eGFP-GSDMD-N275 under the control of ATc. Photographs were captured 5 min, 30 min, 90 min
                    after induction, respectively.</P><P><B>Figure 11.</B>  Ruptured cells in a
                    field of
                    view were counted. </P><DIV class="collapseDiv"><LABEL for="zhedie-toggle7">Method</LABEL><DIV id="zhedie7" class="text-success text-left"><B>Preparation of Cells for Infection</B>
                        1. Grow Hela GSDMD KO cells in a humidified 37 °C, 5% CO<SUB>2</SUB> tissue-culture incubator.
                        2. Count the cells using a hemocytometer. Seed in 24-well (5 × 10^4 per well) and grow
                        overnight.<B>Preparation of Bacteria</B>
                        1. Grow bacteria overnight 16 h in 2 mL LB in a 15-mL tube. Incubate at 37 °C in a shaking
                        incubator (200 rpm).
                        2. Subculture bacteria by transferring 300 μL of the overnight culture into 5 mL of LB in a
                        loosely capped 50-mL tube. Incubate at 37 °C in a shaking incubator (200 rpm) to late log
                        phase.
                        3. Pellet 1 mL of the <I>Salmonella</I> subculture by centrifugation at 1,000×g in a microfuge
                        for 2
                        min at room temperature.
                        4. Remove 900 μL of supernatant and gently resuspend the pellet in 900 μL PBS.<B>Infection</B>
                        1. Aspirate media and rinse the monolayer twice with PBS.
                        2. Inoculate cells with bacteria (MOI = 100) by adding bacteria directly to the cell-culture
                        supernatant.
                        3. Incubate for 2 h at 37 °C in 5% CO<SUB>2</SUB>.
                        4. Aspirate media and wash.
                        5. Add fresh GM containing 100 μg/mL gentamicin and 16 μg/mL incubate at 37 °C in 5% CO<SUB>2</SUB>.
                        Observation is taken after 5 min, 30 min, 90min.</DIV></DIV></DIV><DIV id="float05"><DIV class="h1">Reference</DIV><P>1. Dumont, A. et al. SKIP, the host target of the <I>Salmonella</I> virulence factor SifA, promotes
                    kinesin-1-dependent vacuolar membrane exchanges. Traffic 11, 899-911,
                    doi:10.1111/j.1600-0854.2010.01069.x (2010).
                </P><P>2. Thurston, T. L. et al. Growth inhibition of cytosolic <I>Salmonella</I> by caspase-1 and
                    caspase-11
                    precedes host cell death. Nature communications 7, 13292, doi:10.1038/ncomms13292 (2016).</P><P>3. Hmelo, L. R. et al. Precision-engineering the Pseudomonas aeruginosa genome with two-step allelic
                    exchange. Nat Protoc 10, 1820-1841, doi:10.1038/nprot.2015.115 (2015).</P><P>4. He, W. T. et al. Gasdermin D is an executor of pyroptosis and required for interleukin-1beta
                    secretion. Cell research 25, 1285-1298, doi:10.1038/cr.2015.139 (2015).</P></DIV></DIV><DIV class="cebian"><A class="daohangyi" href="https://2018.igem.org/Team:HZAU-China/Description"><SPAN class="biaoti">Description</SPAN></A><A class="daohangyi" href="https://2018.igem.org/Team:HZAU-China/Design"><SPAN class="biaoti">Design</SPAN></A><DIV class="daohangyi"><SPAN class="biaoti">Results</SPAN></DIV><DIV class="floatCtro"><P class="daohanger">Chassis</P><P class="daohanger">Targeting</P><P class="daohanger">Function of GSDMD</P><P class="daohanger">Induce the expression of GSDMD-N275</P><P class="daohanger">Reference</P><A><SPAN>Back to Top</SPAN></A></DIV><A class="daohangyi" href="https://2018.igem.org/Team:HZAU-China/Demonstrate"><SPAN class="biaoti">Demonstrate</SPAN></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>