Team:HZAU-China/Results
Project Description Design Results Demonstrate
Wetlab Experiments Improve InterLab Notebook
Drylab Model Software
Human Practices Safety Human Practices Public Engagement
About Us Team Attributions Collaborations
Parts Basic Composite
Chassis
sifA knock out
SifA maintains the integrity of Salmonella-containing vacuole (SCV) where Salmonella survive and                     replicate1. The existence of SCV limits the releasing of GSDMD-N275 into cytoplasm. In                     addition,                     the growth of inhibition of ΔsifA mutant in macrophage is remarkable2. Thus, we                     knocked out the sifA gene in order to decrease the stability of SCV and reduce the virulence of Salmonella.
ΔsifA mutant was constructed by using gene editing system based on two-step allelic exchange3.
PCR verification indicated that the chromosomal gene sifA was knocked out (Figure 1).                     Primers named                     DSIFA F and DSIFA R were used in this PCR (Figure 2).
Figure 1. PCR verification of sifA gene knock out. Lane 1 refers to ΔsifA mutant.                     Lane 2 refers                     to WT Salmonella enterica serovar Typhimurium strain SL1344 as a control.
Figure 2. DSIFA F and DSIFA R are the primers in the ORF of sifA. This pair of primers                     can produce a 436bp product in WT.
Safety
SifA is essential for maintaining vacuolar membrane stability. Cytosolic bacterium can be generated                     by the function loss of SCV. This population of bacterium was                     easily                     recognized and cleaned up by macrophage. Hence, ΔsifA mutant decreases the toxicity to the                     host.                     Microscopy demonstrated that ΔsifA mutant was defective for replication in macrophage (Figure                         3).
Figure 3. Microscopy of immortalized bone-marrow-derived macrophages (iBMDM) infected with                     the ΔsifA mutant and WT Salmonella SL1344, respectively. These strains contain high copy number plasmids to                     express RFP constitutively.
Method
Preparation of Cells for Infection
1. Grow iBMDMs in a humidified 37 °C, 5% CO2 tissue-culture incubator.
2. Count the cells using a hemocytometer. Seed in 24-well (5×10^4 per well) and grow                         overnight .
Preparation of the Bacterial Cells
1. Grow bacterial cells overnight 16 h in 2 mL LB in a 15-mL tube. Incubate at 37 °C in a shaking                         incubator (200 rpm).
2. Subculture bacterial cells by transferring 300 μL of the overnight culture into 5 mL of LB in a                         loosely capped 50-mL tube. Incubate at 37 °C in a shaking incubator (200 rpm) to late log                         phase.
3. Pellet 1 mL of the Salmonella bacteria cells subculture by centrifugation at 1,000×g in a microfuge                         for 2                         min at room temperature.
4. Remove 900 μL of supernatant and gently resuspend the pellet in 900 μL PBS.
Infection
1. Aspirate media and rinse the monolayer twice with PBS.
2. Inoculate cells with bacterial cells (MOI = 20) by adding bacterial cells directly to the cell-culture                         supernatant. Centrifugate at 110 g for 5 min at room temperature.
3. Incubate for 25 min at 37 °C in 5% CO2.
4. Aspirate media and rinse the monolayer twice with PBS, to remove extracellular bacterial cells.
5. Add fresh GM containing 100 μg/mL gentamicin and incubate for 2 h at 37 °C in 5% CO2.
6. Replace GM with fresh GM containing 20 μg/mL gentamicin for remainder of experiment.
Observation is taken after 11 h.
Targeting
RGD motif can specifically bind to alpha V beta 3 (αvβ3), a well-known biomarker on the surface of                     tumor cells. We express RGD motif fused with OmpA to surface display it on the outer membrane of                     bacterial cells. Microscopy shows that bacterial cells expressed Lpp-OmpA-RGD induced by 0.1 mM IPTG can bind to                     αvβ3-positive MDA-MB-231 cell line (Figure 4), but cannot bind to αvβ3-negative MCF7 cell line (Figure 5). These results demonstrated that the bacterium gain the function of tumor targeting though display RGD motif.
Figure 4. Microscopy of αvβ3-positive MDA-MB-231 cell line were incubated with E. coli which                     constructively expressed RFP and inductively expressed RGD motif under the control of lac promoter. The locations of bacteria cells are pointed by red arrow.
Figure 5. Microscopy of αvβ3-negative MCF7 cell line were incubated with E. coli which                     constructively                     expressed RFP and inductively expressed RGD motif under the control of lac promoter.
Method
Preparation of Cells
1. Grow MDA-MB-231 and MCF7 in a humidified 37 °C, 5% CO2 tissue-culture incubator.
2. Count the cells using a hemocytometer. Seed in 24-well (1.5× 10^5 per well) and grow                         overnight.
Preparation of Bacterial cells
1. Grow bacterial cells overnight 14 h in 2 mL LB in a 15-mL tube. Incubate at 37 °C in a shaking                         incubator (200 rpm).
2. Subculture bacterial cells by transferring 300 μL of the overnight culture into 5 mL of LB                         containing 0.1 mM IPTG in a loosely capped 50-mL tube. Incubate at 37 °C in a shaking incubator                         (200 rpm) to early stationary phase.
3. Pellet 1 mL of the bacterial cells subculture by centrifugation at 1000 g in a microfuge for 2 min                         at room temperature.
4. Remove 900 μL of supernatant and gently resuspend the pellet in 900 μL PBS.
Infection 1. Aspirate media and rinse the monolayer twice with PBS.
2. Inoculate cells with bacterial cells (MOI = 100) by adding bacterial cells directly to the cell-culture                         supernatant.
3. Incubate for 1.5 h at 37 °C in 5% CO2.
4. Aspirate media and rinse the monolayer three times with PBS.
Observation is taken immediately.
Function of GSDMD
The N-terminal of GSDMD performs the function of cell pyroptosis
We fused eGFP with GSDMD-N275 (N-terminal 275 amino acids) and GSDMD FL (full length), respectively. Then the corresponding                     plasmids were transfected into Hela GSDMD knockout (KO) cell. Cell microscopy showed that the cells                     transfected with GSDMD-N275 underwent pyroptosis while the cells with GSDMD FL did not (Figure 6).                     We also tested the cell viability through an ATP assay (CellTiter-Glo® Luminescent Cell Viability                     Assay) and demonstrated that GSDMD-N275 and mutants of GSDMD FL have different abilities to induce                     pyroptosis (Figure 7).
Figure 6. Microscopy of the Hela GSDMD KO cells transfected with pCS2-eGFP-GSDMD FL (above) and                     pCS2-eGFP-GSDMD-N275 (below), respectively. Pyroptotic cells are pointed by red arrow.
Method
Preparation of Cells for transfection
1. Grow Hela GSDMD KO cells in a humidified 37 °C, 5% CO2 tissue-culture incubator.
2. Count the cells using a hemocytometer. Seed in 24-well (5 × 10^4 per well) and grow.
Transfection
1. Dilute 0.5 μg DNA into 50 μl jetPRIME® buffer (supplied). Mix by vortexing.
2. Add 1 μl jetPRIME®, vortex for 10 s, spin down briefly.
3. Incubate for 10 min at RT.
4. Add 50μl of transfection mix per well drop wise onto the cells in serum containing medium,                         and distribute evenly.
5. Gently rock the plates back and forth and from side to side.
6. If needed, replace transfection medium after 4 h by cell growth medium and return the plates                         to the incubator.
Observation is taken after 1.5 h.
Figure 7. Cell viability of the 293T cells transfected with pCS2-Flag-GSDMD FL,                     pCS2-Flag-GSDMD-N275, pCS2-Flag-GSDMD L290D, pCS2-Flag-GSDMD Y373D, pCS2-Flag-GSDMD A377D,                     respectively. Asterisks indicate the statistically significant differences. ATP-based cell                     viability was measured (n=6).
Method
Preparation of Cells for ATP assay
1. Grow HEK293T cells in a humidified 37 °C, 5% CO2 tissue-culture incubator.
2. Count the cells using a hemocytometer. Seed in 96-well (1 × 10^4 per well) and grow                         overnight.
3. Transfect 0.5 μg DNA per well.
4. Equilibrate the plate and its contents at room temperature for approximately 30 minutes                         after 20 h.
5. Add a volume of CellTiter-Glo® Reagent equal to the volume of cell culture medium present in                         each well. (add 100 μl of reagent to 100 μl of medium containing cells for a 96-well plate).
6. Mix contents for 2 minutes on an orbital shaker to induce cell lysis.
7. Allow the plate to incubate at room temperature for 10 minutes to stabilize luminescent                         signal.
8. Record luminescence.
Induce the expression of GSDMD-N275
Intracellular environment-dependent expression
PsifA is a intracellular environment-dependent promoter. SipD is required for bacterial                     internalization.                     ΔsipD mutant cannot enter host cells. Thus, we use this strain as a control. Microscopy                     suggested                     that only intracellular bacteria express eGFP which is under the control of PsifA (Figure                         8).
Figure 8. Microscopy of hela GSDMD KO cells infected with the ΔsipD or ΔsifA mutant, respectively.                     These mutants contain low copy number plasmids to express eGFP which was regulated by PsifA.
Method
Preparation of Cells for Infection
1. Grow Hela GSDMD KO cells in a humidified 37 °C, 5% CO2 tissue-culture incubator.
2. Count the cells using a hemocytometer. Seed in 24-well (9× 10^4 per well) and grow                         overnight.
Preparation of Bacterial Cells
1. Grow bacterial cells overnight 16 h in 2 mL LB in a 15-mL tube. Incubate at 37 °C in a shaking                         incubator (200 rpm).
2. Subculture bacterial cells by transferring 300 μL of the overnight culture into 5 mL of LB in a                         loosely capped 50-mL tube. Incubate at 37 °C in a shaking incubator (200 rpm) to late log                         phase.
3. Pellet 1 mL of the Salmonella subculture by centrifugation at 1000 g in a microfuge                         for 2                         min at room temperature.
4. Remove 900 μL of supernatant and gently resuspend the pellet in 900 μL PBS.
Infection
1. Aspirate media and rinse the monolayer twice with PBS.
2. Inoculate cells with bacterial cells (MOI = 100) by adding bacterial cells directly to the cell-culture                         supernatant.
3. Incubate for 3 h at 37 °C in 5% CO2.
4. Aspirate media and rinse the monolayer twice with PBS.
5. Add fresh GM containing 100 μg/mL gentamicin and incubate at 37 °C in 5% CO2.                         Observation is taken after 2 h.
ATc-dependent expression
GSDMD-N275 can lyse bacteria
Expression of the N-terminal of GSDMD fused with eGFP (eGFP-GSDMD-N275) in Salmonella enterica serovar Typhimurium str. SL1344 ΔsifA is under the control of Ptet. The                     colony-forming unit (CFU)                     was measured for counting the number of viable bacterial cells (Figure 9). This result shows that                     eGFP-GSDMD-N275 exhibits cytotoxicity in bacteria.
Figure 9. CFU comparison between the SL1344 ΔsifA cells with eGFP-GSDMD-N275 plasmid                     and with the                     empty vector. In each group, ATc (15μg/ml) was added into medium when bacterium grew to logarithmic                     phase (OD = 0.6~0.8). Vector refers to bacterium containing a high copy number plasmid which only                     expresses TetR under the control of Ptet. CFU for vector                     and eGFP-GSDMD-N275 are shown in the logarithmic form (log10) (n=3).
Method
1. Bacteria are cultured overnight in LB broth containing corresponding antibiotics, and                         dilute each 1 volume overnight cultures with 100 volume fresh LB containing antibiotics.                         Culture in 37℃ 200 rpm.
2. When OD reaching to 0.6-0.8, add anhydrotetracycline with final concentration of                         μg/ml to induce the expression of EGFP-GSDMD-N275.
3. Take 100 μl diluted culture to plate on LB agar plates containing appropriate concentration                         of antibody after 1.5 hours of induce.
Observation is taken overnight.
GSDMD-N275 from lytic bacteria induces host cell pyroptosis
Expression of the N-terminal of GSDMD fused with eGFP (eGFP-GSDMD-N275) is under the control of tet                     promoter in ΔsifA SL1344. Hela GSDMD KO cells were infected with ΔsifA SL1344.                     Inducer ATc                     (16μg/mL) were added 3h after infection. Microscopy shows that eGFP-GSDMD-N275 located in cytoplasm                     after 5 min of induction and triggered pyroptosis after 30 min of induction (Figure 10). After                     1.5 h                     of induction, Hela GSDMD KO cells underwent second necrosis caused by bacterial infection without                     inducer. Morphology of this process is similar to pyroptosis4. Thus, the population of                     ruptured                     cells was counted. There is 1.96 fold change between control group and induced group (Figure 11).                     So                     the pyroptosis of host cell in the induced group was triggered by eGFP-GSDMD-N275 not by bacterial                     infection.
In these experiments, the choice of MOI (multiplicity of infection) and infection time were directed by Modeling .
Figure 10. Hela GSDMD KO cells were infected with ΔsifA SL1344 containing high copy                     number plasmids                     which express eGFP-GSDMD-N275 under the control of ATc. Photographs were captured 5 min, 30 min, 90 min                     after induction, respectively.
Figure 11. Ruptured cells in a                     field of                     view were counted.
Method
Preparation of Cells for Infection
1. Grow Hela GSDMD KO cells in a humidified 37 °C, 5% CO2 tissue-culture incubator.
2. Count the cells using a hemocytometer. Seed in 24-well (5 × 10^4 per well) and grow                         overnight.
Preparation of Bacteria
1. Grow bacteria overnight 16 h in 2 mL LB in a 15-mL tube. Incubate at 37 °C in a shaking                         incubator (200 rpm).
2. Subculture bacteria by transferring 300 μL of the overnight culture into 5 mL of LB in a                         loosely capped 50-mL tube. Incubate at 37 °C in a shaking incubator (200 rpm) to late log                         phase.
3. Pellet 1 mL of the Salmonella subculture by centrifugation at 1,000×g in a microfuge                         for 2                         min at room temperature.
4. Remove 900 μL of supernatant and gently resuspend the pellet in 900 μL PBS.
Infection
1. Aspirate media and rinse the monolayer twice with PBS.
2. Inoculate cells with bacteria (MOI = 100) by adding bacteria directly to the cell-culture supernatant.
3. Incubate for 2 h at 37 °C in 5% CO2.
4. Aspirate media and wash.
5. Add fresh GM containing 100 μg/mL gentamicin and 16 μg/mL incubate at 37 °C in 5% CO2.
Observation is taken after 5 min, 30 min, 90min.
Reference
1. Dumont, A. et al. SKIP, the host target of the Salmonella virulence factor SifA, promotes                     kinesin-1-dependent vacuolar membrane exchanges. Traffic 11, 899-911,                     doi:10.1111/j.1600-0854.2010.01069.x (2010).
2. Thurston, T. L. et al. Growth inhibition of cytosolic Salmonella by caspase-1 and                     caspase-11                     precedes host cell death. Nature communications 7, 13292, doi:10.1038/ncomms13292 (2016).
3. Hmelo, L. R. et al. Precision-engineering the Pseudomonas aeruginosa genome with two-step allelic                     exchange. Nat Protoc 10, 1820-1841, doi:10.1038/nprot.2015.115 (2015).
4. He, W. T. et al. Gasdermin D is an executor of pyroptosis and required for interleukin-1beta                     secretion. Cell research 25, 1285-1298, doi:10.1038/cr.2015.139 (2015).
Description
Design
Results
Chassis
Targeting
Function of GSDMD
Induce the expression of GSDMD-N275
Reference
Back to Top
Demonstrate
