<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HZAU-China_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HZAU-China/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="daohang"><UL class="caidan"><LI class="hiLight shortName"><A class="nav_head" href="https://2018.igem.org/Team:HZAU-China"><SPAN>Project</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Description">Description</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Design">Design</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Results">Results</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Demonstrate">Demonstrate</A></LI><LI class="hiLight shortName"><A class="nav_head" href="#"><SPAN>Wetlab</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Experiments">Experiments</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Improve">Improve</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/InterLab">InterLab</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Notebook">Notebook</A></LI><LI class="hiLight shortName"><A class="nav_head" href="#"><SPAN>Drylab</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Model">Model</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Software">Software</A></LI><LI class="hiLight longName"><A class="nav_head" href="#"><SPAN>Human Practices</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Safety">Safety</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Human_Practices">Human Practices</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Public_Engagement">Public Engagement</A></LI><LI class="hiLight shortName"><A class="nav_head" href="#"><SPAN>About Us</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Team">Team</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Attributions">Attributions</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Collaborations">Collaborations</A></LI><LI class="hiLight shortName"><A class="nav_head" href="#"><SPAN>Parts</SPAN></A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Basic_Part">Basic</A><A class="item" href="https://2018.igem.org/Team:HZAU-China/Composite_Part">Composite</A></LI></UL></NAV></P><DIV class="neirong clearfix"><DIV class="zhengwen"><DIV id="float01" class="cur"><DIV class="h1"><I>Salmonella</I> infection model</DIV><P>We want to simulate the situation that tumor cells and <I>Salmonella</I> together in a liquid
                    environment.
                    We used the law of mass action to establish a model for the infection process of <I>Salmonella</I>,
                    which
                    is dimensionless.</P><P>
                    $$N_{normal} + S_{almonella} \overset{Aw}{\rightarrow} N_{w} + S_{al\_normal}$$

                </P><P>
                    $$N_{tumor} + S_{almonella} \overset{As}{\rightarrow} N_{s} + S_{al\_tumor}$$
                </P><P>
                    $$S_{almonella} = S_{almonella}(t0) - N_{normal\_cell} - N_{tumor}$$
                </P><P>
                    $$\dfrac {dN_{w}} {d_{t}} = A_{w} S_{almonella} N_{w}$$
                </P><P>
                    $$\dfrac {dN_{s}} {d_{t}} = A_{s} S_{almonella} N_{s}$$
                </P><P>
                    $$\dfrac {dS_{almonella}} {d_{t}} = - \dfrac {dN_{w}} {d_{t}} - \dfrac {dN_{s}} {d_{t}}$$
                </P><P>
                    \(N_{normal\_cell}\): The density of normal cells.
                    \(S_{almonella}\): The density of <I>Salmonella</I> in the liquid environment.
                    \(N_{tumor}\): The density of tumor cells.
                    \(N_w\): The number of <I>Salmonella</I> in the normal cells.
                    \(N_s\): The number of <I>Salmonella</I> in the tumor cells.
                    \(A_w\): The affinity between <I>Salmonella</I> and normal cells.
                    \(A_s\): The affinity between <I>Salmonella</I> and tumor cells.
                     \(S_{al\_normal}\): The density of infected normal cells.
                    \(S_{al\_tumor}\): The density of infected tumor cells.</P><P><I>Salmonella</I> begins to replicate two hours after infection<SUP>1</SUP> .</P><P>
                    $$\dfrac {dN_{sal}} {d_{t}}(t) =$$
                </P><P>
                        $$\dfrac {dS_{al}} {d_{t}}(t) + S_{al}(t-2) 2^{\dfrac {t-2} {T}} \ln{2}\dfrac {1} {T} $$
            </P><P> 
                   \(T\): Cell cycle.</P><DIV class="h2">Identification of infection time</DIV><P>According to our experimental results, we noticed that <I>Salmonella</I> might follow Poisson
                    distribution in
                    cells, so we use Matlab to judge the distribution of bacteria in the cells. We assume
                    that the area less than 1 in the Poisson distribution is a part of cells which are not infected by
                    <I>Salmonella</I>. According to our experimental results, cells which are infected by only one <I>Salmonella</I>
                    can also die of pyroptosis. Based on this feature, we divide cells into uninfected and infected
                    cells. When the average number of bacteria in the cell changes, which means that the λ of Possion
                    distribution changes, the ratio of the two kind of cells will change. In summary, when the average
                    number of <I>Salmonella</I> in cells changes, the proportion of dead cells will change.</P><P><B>Figure 1. Poisson distribution and <I>Salmonella</I> infection results.</B><B>a.</B> Based on
                    statistics of
                    experimental results, we proved that the <I>Salmonella</I> follows Poisson distribution in normal
                    cells.
                    <B>b.</B> We assume that the area less than 1 in the Possion distribution is a part of cells which
                    are not infected by <I>Salmonella</I>. When the λ of Possion distribution changes, which means the
                    average
                    number of <I>Salmonella</I> in cells changes, the proportion of infected cells changes. <B>c.</B>
                    Cells
                    which are infected by only one <I>Salmonella</I> can also die of pyroptosis.</P><DIV class="h2">Infection in tumor cell culture experiments</DIV><P>We hope that the mathematical model can help the <I>Salmonella</I> infection experiment. In our
                    final
                    phenotypic experiment, the cells carrying the GSDMD gene are induced by ATc, and we hope that the
                    observed result is that the proportion of ATc-induced cell death is more than that of not induced
                    to prove the ATc promoter is effective. In this experiment, the error may be big if the proportion
                    of cells infected by <I>Salmonella</I> is different.</P><P style="width: 100%; text-align: center !important;"><B>Figure 2.</B> Results caused by efficiency
                    differences of infection. </P><P>If the proportions of infection are different, the experimental results may not be able to prove that pyroptosis is induced by atc promoter. Figure 2 showed that the ATc promoter is induced and 90% cells death are caused and 70% cells die because of the promoter disclosure, but the difference of the proportions of infected cells is so big that the experimental results cannot reflect the real situation. We can reduce the difference by letting the infection proportions of the two kinds of cells both close to 100%.</P><P>We solved this problem by predicting the proportion of cells infected with bacteria over time.</P><P>Based on these, we designed an App with MatLab (<A href="https://github.com/cccoolll/Pyroptosis.git">https://github.com/cccoolll/Pyroptosis.git</A>)
                   . In this App, different parameters obtained from measurement experiments can be input to predict the optimal infection time.
                    Therefore, this App can provide guidance to the design of our experiments.</P><P style="width: 100%; text-align: center !important;"><B>Figure 3.</B> Salmonella infection prediction tool (for tumor cell). </P>                
                    A1: The predicted <I>Salmonella</I> numbers in tumor cell in the single cell
                    infection experiment.
                    A2: The predicted proportions of infected tumor cells.
                    A3: The concentration of added <I>Salmonella</I>.
                    A4: The density of tumor cells;
                    A5: Rate constant of <I>Salmonella</I> infecting tumor cells.
                    A6: The proportion of tumor cells expected to be infected.
                    A7: The time to reach the wanted proportion of infected cells.<P style="width: 100%; text-align: center !important;"><B>Figure 4.</B><I>Salmonella</I> infection prediction tool (for mixed culture of tumor cell and normal cell). </P>
                    
                    B1: The predicted numbers of <I>Salmonella</I> in tumor cell (red) and normal cell (blue) in a single cell infection
                    experiments.
                    B2: The predicted proportions of infected cells (red for tumor cell and blue for normal cell).
                    B3: The concentration of added <I>Salmonella</I>.
                    B4: The density of tumor cells.
                    B5: Rate constant of <I>Salmonella</I> infecting tumor cells.
                    B6: The density of normal cells.
                    B7: Rate constant of <I>Salmonella</I> infecting normal cells.
                    B8: The predicted optimal infection time.
                    <P>The parameters Nsal, Tumor and As are measured from our experiments.</P><DIV class="h3">Guidance to tumor cell infection experiments</DIV><P><B>Figure 5 (a part of Figure 3).</B> Guidance for tumor cells infection experiments.
                   According to our experiment protocol, the MOI (multiplicity of infection) is 100, corresponding to the concentration of cells. If we want 98% of the tumor cells to be infected, the prediction result show that the infection time should be at least 2 hours to eliminate unnecessary variables.</P><DIV class="h3">Guidance to mixed culture experiments</DIV><P><B>Figure 6 (a part of Figure 4).</B> The predicted proportions of infected cells (red for tumor
                    cell and blue for normal cell). 
                    In order to reflect the affinity of <I>Salmonella</I> to tumor cells and to normal cells, we
                    hope that the difference between experimental results of tumor cells and normal cells is obvious.
                    However, the numbers of bacteria in different host cells are difficult to count, and we can only count the
                    number of infected cells and calculate their the proportions. Therefore, we need to
                    predict the time when the difference is most obvious. Our App just can do this for us. </P></DIV><DIV id="float02"><DIV class="h1">Chemical control model</DIV><DIV class="h2">Profile</DIV><P>
                    The Tet repressor protein (tetR) regulates transcription of tetracyclines resistance protein, tetA.
                    The expression of tetA must be strictly regulated since tetA is a membrane-spanning H<SUP>+-</SUP>[Tc-Mg]<SUP>2+</SUP>
                    antiporter which means it can lower the pH environment of cytoplasm. As a result, the natural
                    circuit of tetracyclines regulation is a negative-feedback circuit<SUP>2</SUP>. Tc is the inducer,
                    which shows
                    high affinity to tetR protein. The tetR protein binds to tetO sequence on DNA specifically, thus
                    inhibits the expression of Tet promoter. When Tc or other similar molecules like ATc
                    (anhydrotetracycline) diffuse into bacteria, they will bind to tetR protein and unleash the tetR
                    protein from DNA, and thus relieve the inhibition and start the expression of Tet promoter.
                </P><P> In our project, we choose ATc (anhydrotetracyclines) as the inducer. ATc is less harmful to
                    bacteria than Tc and about 100-fold higher affinity to tetR than Tc<SUP>2</SUP>.
                </P><P> The ATc model aims to predict and solve two problems: first, how fast does the circuit react to
                    ATc; second, how many target gene will express in the bacteria community under a certain
                    concentration of ATc.
                </P><DIV class="h2">Hypothesis</DIV><P>
                    There are two tetO sites on the Tet promoter and both can bind to tetR protein randomly and inhibit
                    the promoter’s expression independently. To make the condition simple, we consider the two tetO
                    sites into one as we just want to explain the relationship between the promoter inhibition and
                    the tetR protein expression.
                </P><P> In our project, the ATc concentration in our incubation environment is uniform, and the diffusion
                    rate of anhydrotetracycline can be ignored<SUP>3</SUP>. In spite of this, the degradation rate of
                    ATc under 37℃ must be taken into account as reported<SUP>4</SUP>.
                </P><P> Based on these facts, we give the following hypotheses:
                </P><P> 1. Regard two tetO operons as one equivalently.
                    2. Ignore the diffusion of ATc.
                    3. The reaction time between ATc and tetR, tetR and DNA is much shorter than transcription and
                    translation.</P><DIV class="h2">Description and Equation</DIV><DIV class="h3">Reactions implicated:</DIV><P>$$tetR + [tetR - ATc_2] = tetR_{total}$$</P><P>$$tetR + 2 \times ATc = [tetR - ATc_2]$$</P><P>$$P_{tet} + [tetR_2 - P_{tet}] = [P_{tet}]_{total}$$</P><P>$$2 \times tetR + P_{tet} = [tetR_2 - P_{tet}]$$</P><P>$$$$</P><DIV class="h3">Equations<SUP>5</SUP>:</DIV><P>Based on Hill function, we can determine the amount of activated tetR, tetR<SUB>act</SUB>:</P><P>$$ tetR_{act} + n \times S_x(t) \rightarrow [tetR - S_x(t)_n] $$</P><P>$$ K_X = \dfrac {tetR_{act} \times S_x^n (t)} {[tetR - S_x(t)_n]} $$</P><P>$$ tetR = tetR_{act} + [tetR - S_x(t)_n] $$</P><P>$$ tetR_{act} = \dfrac {tetR} {1 + \dfrac {S_x^n (t)} {K_{X}}} $$</P><P>Based on Hill function, we can determine the amount of activated promoter, with which we can
                    calculate the total transcription speed of all promoters per cell:</P><P>$$ P_{tet\_act} + n\cdot tetR_{act} \rightarrow [P_{tet} - (tetR_{act})_n] $$</P><P>$$ P_{tet\_copy} = P_{tet\_act} + [P_{tet} - (tetR_{act})_n] $$</P><P>$$ K_d = \dfrac {P_{tet\_copy} \times tetR^n_{act} } {[P_{tet} - (tetR_{act})_n]} $$</P><P>$$ A_{mRNA} = P_{tet\_act} \times beta $$</P><P>$$ A_{mRNA} = \dfrac {P_{tet\_copy} \times beta } { 1 + \dfrac {tetR^n_{act}} {K_{d}}} $$</P><P>Kinetic equations of transcription and translation:</P><P>$$ \dfrac {dmRNA} {dt} = A_{mRNA} - K_{deg\_mRNA} \times mRNA $$</P><P>$$ \dfrac {dtetR} {dt} = K_{trans\_tetR} \times mRNA - K_{deg\_tetR} \times tetR $$</P><P>$$ \dfrac {dGSDMD} {dt} = K_{trans\_GSDMD} \times - K_{deg\_GSDMD} \times GSDMD $$</P><P>Degradation function of ATc by time<SUP>3</SUP>:</P><P>$$ \dfrac {dS_x(t)} {d_t} = -K_{deg\_ATc} \times S_x(t) $$</P><P>$$ \ln(S_x(t)) = \ln(S_x(0)) - K_{deg\_ATc} \times t $$</P><P>Growth curve of bacteria based on logistics model from P. F. Verhulst:</P><P>$$ N(t) = \dfrac {K_{max}} {1 + C \cdot e^{-rt}} $$</P><P>Total GSDMD expressed in bacteria community:</P><P>$$ GSDMD_{total-amount} = N(t) \cdot GSDMD\cdot{CFU}\cdot{diluted-ratio}\cdot{V_{Bacteria volume}}$$</P><DIV class="h3">The symbols in the equations:</DIV><P> \(S_x(t)\): concentration of ATc, as a function of time.
                    \(tetR_{act}\): concentration of activated tetR.
                    \(tetR \): concentration of total tetR.
                    \(GSDMD \): concentration of GSDMD.
                    \(A_{mRNA} \): transcription rate constant of the promoter.
                    \(P_{tet\_copy} \): plasmid copy number.
                    \(K_X \): disassociation rate constant of tetR and ATc.
                    \(K_d \): disassociation rate constant of tetR and DNA.
                    \(beta \): original (unrepressed) transcription rate constant of the promoter.
                    \(K_{deg\_mRNA} \): degradation rate constant of mRNA.
                    \(K_{deg\_tetR} \): degradation rate constant of tetR.
                    \(K_{trans\_tetR} \): translation rate constant of tetR.
                    \(mRNA \): concentration of mRNA.
                    \(K_{deg\_GSDMD} \): degradation rate constant of GSDMD.
                    \(K_{trans\_GSDMD} \): transcription rate constant of GSDMD.
                    \(K_{deg\_ATc} \): degradation rate constant of ATc.
                    \(n \): Hill coefficient.
                    \(N(t) \): initial OD600 value of the bacteria.
                    \(r \): growth rate of the bacteria.
                    \(K_{max} \): maximum OD of the bacteria in cultivation.</P><DIV class="h2">Suggestions to our experiments (see <A href="https://2018.igem.org/Team:HZAU-China/Results">Results</A>)</DIV><P>As is hard to obtain the initial parameters in the equations above on our own without any
                    experiments, the only way to obtain these parameters is to look up in former research or other
                    teams work. Fortunately we got a copy of these parameters from team William and Mary iGEM 2016<SUP>6</SUP>.
                    These parameters include \(K_X = 0.36 \), \(K_d = 0.1 \), \(beta = 0.0023 \), \(K_{deg\_mRNA} =
                    0.009 \), \(K_{deg\_tetR} = 0.631 \), \(K_{trans\_tetR} = 235.5 \) <B>(All units are combined of
                        nM and s)</B>. Considering that both <I>Salmonella</I>
                    and <I>E. coli</I> are in <I>Enterobacteriaceae</I>, we assumed that in <I>Salmonella</I> these
                    parameters are the same
                    with those in <I>E. coli</I> since we just wanted to figure out a useful instruction to wet lab.</P><P>To gain the parameters in bacterial growth curve, we carried out an experiment to measure the growth
                    of <I>Salmonella</I>. Then we fitted the obtained data into a logistics model. By doing these we
                    figured out that
                    \(r = 60min^{-1} \), \(K_{max} = 0.9997 \) and \(C = 7.2319 \). Results and diagram are shown below
                    (<B>Figure 7</B>):</P><P style="width: 100%; text-align: center !important;"><B>Figure 7.</B> Bacterial growth curve.</P><P>After complete the work above, we used MATLAB<SUP>TM</SUP> to solve the equations above and
                    acquired a series of diagrams which visually demonstrated the relationships and helped the wet lab
                    group get
                    an overall view of how ATc influences the expression of GSDMD. We assumed that \(P_{tet\_copy} =
                    4
                    \), \(K_{deg\_GSDMD} = 0.8 \), \(K_{trans\_GSDMD} = 200 \), \(K_{deg\_ATc} = 0.0007 \) (All unites
                    are
                    combined of nM and s). In this action we didn’t take the growth of bacteria into account. From the diagrams, we can figure out that as ATc added increase, the concentration of GSDMD will also rise. Results are shown below (<B>Figures 8, 9, 10.</B>):</P><P style="width: 100%; text-align: center !important;"><B>Figure 8.</B> Concentration of tetR (nM) -
                    time (s).</P><P style="width: 100%; text-align: center !important;"><B>Figure 9.</B> Concentration of GSDMD (nM) -
                    time (s).</P><P style="width: 100%; text-align: center !important;"><B>Figure 10.</B> Max concentration of GSDMD (nM)
                    -
                    ATc Concentration (nM).</P><P> With this software, one can adjust all the parameters needed in the equations above and
                    attain the diagrams which indicates the relations between concentration of GSDMD and time,
                    concentration tetR and time and the maximum value of GSDMD and the initial concentration of ATc.
                    The
                    program will also generate a function describing the relationship between the maximum concentration
                    of
                    GSDMD and the concentration of ATc. With the help of this program, members in wet lab group can
                    conveniently decide how much ATc should be added into cultivation environment according to their
                    requirements (<B>Figures 11, 12</B>). <B>Be advised that users must multiply the CFU number, bacteria cell volume and diluted ratio to the data obtained from this app to gain a final result.</B></P><P style="width: 100%; text-align: center !important;"><B>Figure 11.</B> Software parameters.</P><P style="width: 100%; text-align: center !important;"><B>Figure 12.</B> Software diagrams.</P><DIV class="h2">Significance</DIV><P>The model of ATc induced circuit is very common and well-known to biology researchers. The
                    common-known significance to this model is that it can demonstrate the relationship between
                    concentration of target gene and concentration of inducer added, which can instruct the researchers
                    modulate their circuit precisely. In our project, this model will tell the members in wet lab group
                    that how much GSDMD will be expressed under a certain concentration of ATc in the <I>Salmonella</I>
                    community formed in the tumor cell.
                    Another significance for this model is that, the response time is very short and the response
                    speed is extremely fast. We anticipate that just minutes are needed to induce the fluorescence.
                    This phenomenon is also verified in our experiment. In less than 10 minutes, fluorescence can be
                    detected under fluorescence microscope.
                    Especially, a remarkable significance to our project is that it’s a self-destructive system, which
                    means, without any further operation, the process of induction can be self-terminated. As ATc
                    degrades, the expression of GSDMD will significantly decreases, thus the process of pyroptosis can
                    be inhibited. Based on the features, we think that the cytokine storm caused by pyroptosis
                    is controllable.
                </P><DIV class="h2">The source code of the software and the scripts used above can be found following this link:</DIV><P><A href="https://github.com/tom13amy/atc_modelling_software">https://github.com/tom13amy/atc_modelling_software</A></P></DIV><DIV id="float03"><DIV class="h1">Reference</DIV><P>1. I. Hautefort, A. Thompson, et al. During infection of epithelial cells Salmonella enterica
                    serovar Typhimurium undergoes a time-dependent transcriptional adaptation that results in
                    simultaneous expression of three type 3 secretion systems. Cellular Microbiology 10(4), 958–984
                    (2008).
                </P><P>2. Berens, C. &amp; Hillen, W. Gene regulation by tetracyclines: Constraints of resistance regulation in
                    bacteria shape TetR for application in eukaryotes. Eur. J. Biochem. 270, 3109–3121 (2003).

                </P><P> 3. Nevozhay, D., Adams, R. M., Murphy, K. F., Josic, K. &amp; Balazsi, G. Negative autoregulation
                    linearizes the dose-response and suppresses the heterogeneity of gene expression. Proc. Natl. Acad.
                    Sci. 106, 5123–5128 (2009).
                </P><P> 4. Politi, N. et al. Half-life measurements of chemical inducers for recombinant gene expression.
                    J. Biol. Eng. 8, 1–10 (2014).
                </P><P> 5. Alon, U. An Introduction to Systems Biology: Design Principles of Biological Circuits (Chapman &amp;
                    Hall/CRC Mathematical and Computational Biology).Pdf.
                </P><P> 6. William and Mary iGEM 2016. A Kinetic Model of Molecular Titration. 1–11 (2016).
                </P></DIV></DIV><DIV class="cebian"><DIV class="daohangyi"><SPAN class="biaoti">Model</SPAN></DIV><DIV class="floatCtro"><P class="daohanger">Salmonella infection model</P><P class="daohanger">Chemical control model</P><P class="daohanger">Reference</P><A><SPAN>Back to Top</SPAN></A></DIV><A class="daohangyi" href="https://2018.igem.org/Team:HZAU-China/Software"><SPAN class="biaoti">Software</SPAN></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>