"
Page
Discussion
View source
History
teams
Log in
 
Team:Shenzhen BGIC 0101/Next version
From 2013.igem.org
Home
Team
Software »
Overview
Modules
Next Version
New Standard
Compatibility
Tutorial
Web-Based Trial
Downloads
Requirements »
Human Practices
Safety
Collaboration
Medal Criteria
Contact
Next Version
In general
Add restriction on parameters of plugin or even change the plugin structure so that Genovo has a wider applied range.
Add encrypted water marks.
Module1 Neochr
For genes in new pathway:
We can import exogenous genes into the new pathway, using the BLAST+ to classify and replace these exogenous genes orthologous relationship with original genes.
For relationship between these genes:
In the next version of module, we also can rewire genes’ relationship logically in the pathway, such as: the transformation of activation/inhibition, the transformation of Negated AND gate and the addition of bistable state and oscillation circuit control relationship.
For human control over genes’ expression:
Artificial add chemicals to control regulatory sequence of expression, make certain genes under control.
For the presentation of genes’ relationship:
In the next version for the presentation of genes’ relationship, we would have two improvements.
First, we will add the pathway which with indirectly relations between genes and present it in the pattern of gene-compound-gene, We will store the information of genes in the following structure. It will separate the substrates and products attribution of one gene in substance element which relate to reaction element.
Every element will have id attribution which will be the unique identification of it. This utility will also use python program language to grab the information that show above and then convert xml to JSON. The results seem to like following:
Circle represents genes and triangle represents chemical substances. It means that TPI1 and TDH3 connect with this kind of indirectly relationships.
Second, we will combine gene with pathway automatically whenever they have directly or indirectly relationships.We will store the genes’ relationship in the following ways, add one more values, link, in type attribution which in entry element.
Module2 NecleoMod
Since the integration of segments into whole do not work in proportion, so we’re obliged to identify the success of assembly. We hope to implement this by adding pcr tags to cds region of the genes. We mainly consider the melting temperature and minimum free energy by using bioperl
Module3 SegmMan
As the assembly from 2k minichunks to 10k chunks(link), then to 30k megachunks(link), Gibson & Goldengate Strategies are used, the process from 600bp produced by complementary OLSDesign to 2k minichunks may use Goldengate strategy but IIA type instead of IIB type*.
Another new way to construct new chromosome donovo
Because of the high price of gene synthesis, we would add such a function that users can construct their a small pathway by assembly genes one by one through biobrick stand or 3A standard.
*Class describes the cutting behavior of an enzyme. The classes used by
GeneDesign uses a generalized subset of the classes as described at Rebase - for the purposes of enzyme editing, three classes have so far proven to be enough. See http://rebase.neb.com/cgi-bin/sublist for the full description of enzyme classes.
IIP : This enzyme has a symmetric target and a symmetric cleavage site; this usually means that the enzyme cleaves inside its own recognition site.         This is not the same as overhang palindromy!
IIA : This enzyme has an asymmetric recognition site and usually cleaves         outside of it.
IIB : This enzyme has one recognition site and two cleavage sites, one on         either side of the recognition site, and thus cuts itself out of         sequence.
Retrieved from " http://2013.igem.org/Team:Shenzhen_BGIC_0101/Next_version "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
