<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Shenzhen_BGIC_0101_Tutorial_segmman"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Shenzhen_BGIC_0101/Tutorial/segmman">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Shenzhen_BGIC_0101/Tutorial/segmman&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_0101/Tutorial/segmman&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_0101/Tutorial/segmman&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Shenzhen_BGIC_0101/Tutorial/segmman" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Shenzhen BGIC 0101/Tutorial/segmman</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="navigator"><UL class="sf-menu sf-navbar"><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101">Home</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Team">Team</A></LI><LI><A class="sf-with-ul" href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Software">Software<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Software">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Modules">Modules</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Next_version">Next Version</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/NewStandard">New Standard</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Compatibility">Compatibility</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Tutorial">Tutorial</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Web-based_trial">Web-Based Trial</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Downloadsl">Downloads</A></LI></UL><LI><A class="sf-with-ul" href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Requirements">Requirements<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Requirements#HumanPractices">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Requirements#Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Requirements#collaboration">Collaboration</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Criteria">Medal Criteria</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Contact">Contact</A></LI></DIV><B><H1>Tutorial</H1></B><H1> SegmMan </H1><P>This module will cut chromosome into pieces with different sizes with Gibson, Goldengate, Homologous adaptors to them so that they are able to be assembled into whole experimentally.</P><H2>Plugin Scripts</H2><H3>3-1. 01.whole2mega.pl</H3><P>This utility can split the whole chromosome ( at least 90kbp long ) into about 30k segments and add homologous overlap and adaptors, so that these fragments can be integrated into whole experimentally.</P><P><B>Internal operation</B></P><P>First, this utility searches for the location of centromere and ARSs (autonomously replicating site). The minimal distance between centromere and ARS should NOT be larger than a defined megachunk which is about 30k long. 
Second, this utility cuts out the first 30k sequence window containing the centromere and its adjacent ARS, and then adds this megachunk with two original markers and left, right telomeres.
Thirdly, this utility continues to cut more megachunks from the original one to both ends. But these megachunks are not independent, they all have about 1kbp overlaps. Moreover, these new splited window can be given only one marker alternately and only left or right telomere.
The output file will be dealed with 02.globalREmarkup.pl
For more information about segmentation design, please refer to the page ASSEMBLY DESIGN PRINCIPLE .</P><P><B>Example (command line)</B></P><P>perl 01.whole2mega.pl –gff sce_chrI.gff -fa sce_chr01.fa -ol 1000 -ck 30000 -m1 LEU2 -m2 URA3 -m3 HIS3 -m4 TRP1 -ot sce_chrI.mega</P><P><B>Parameters</B></P><TABLE border="1"><TBODY><TR><TH>default</TH><TH>Option</TH></TR><TR><TH>gff</TH><TH>The gff file of the chromosome being restriction enzyme sites parsing</TH></TR><TR><TH>fa</TH><TH>The fasta file of the chromosome being restriction enzyme sites parsing
(The length of the chromosome is larger than 90k)</TH></TR><TR><TH>ol</TH><TH>The length of overlap between megachunks</TH><TH>1000bp</TH></TR><TR><TH>ck</TH><TH>The length of megachunks</TH><TH>30kbp</TH></TR><TR><TH>m1</TH><TH>The first marker for selection alternately</TH><TH>LEU2 (1797bp)</TH><TH>LEU2/URA3HIS3/TRP1</TH></TR><TR><TH>m2</TH><TH>The second marker for selection alternately
</TH><TH>URA3 (1112bp)</TH><TH>LEU2/URA3/HIS3/TRP1</TH></TR><TR><TH>m3</TH><TH>The first marker orinally residing in first 30k segmentation</TH><TH>HIS3 (1774bp)</TH><TH>LEU2/URA3/HIS3/TRP1</TH></TR><TR><TH>m4</TH><TH>The second marker orinally residing in first 30k segmentation</TH><TH>TRP1 (1467bp)</TH><TH>LEU2/URA3/HIS3/TRP1</TH></TR><TR><TH>ot</TH><TH>The output file </TH><TH>Prefix(fa filename)+ suffix(.mega)</TH></TR></TBODY></TABLE><P><B>The format of output:</B></P><P>The output file is stored in /the path where you install GENOVO/Result/ 01.whole2mega.
Besides, there is screen output about the process state and result.
1. Screen output
2. 01.state  Store the segmentation information<TABLE border="1"><TBODY><TR><TH>Megachunk_ID</TH><TH>Corresponding location in the designed chromosome</TH></TR><TR><TH>Part ID</TH><TH>Location in the segmentation</TH></TR></TBODY></TABLE></P>
3 *.mega Store the fasta information of the 30k segments<H3>3-2. 02.globalREmarkup.pl</H3><P>This utility will parse the exited restriction enzyme sites residing in the chromosome.</P><P><B>Internal operation</B></P><P>This utility searches the exited restriction enzyme sites along the chromosome both plus strand and minus strand, after users define the list of enzymes.
Besides, we tried to find out all the potential restriction enzyme sites, so that maybe some unusual restriction enzyme sites can be created and let segmentation go. But because it had low efficiency, we’re still working on that.
The output file will be dealed with 03.mega2chunk2mini.pl
For more information about segmentation design, please refer to the page ASSEMBLY DESIGN PRINCIPLE .
</P><P><B>Example (command line)</B></P><P>perl 02.globalREmarkup.pl -sg 01.whole2mega/sce_chrI.mega -re standard_and_IIB -ct Standard.ct –ot sce_chrI.mega.parse</P><P><B>Parameters</B></P><P><TABLE border="1"><TBODY><TR><TH>default</TH><TH>Option</TH></TR><TR><TH>sg</TH><TH>The fasta file of the 30k segmentation, the output of 01.wh2mega.pl</TH></TR><TR><TH>ps</TH><TH>The markup file of the 30k segmentation, the output of 02.globalREmarkup.pl</TH></TR><TR><TH>re</TH><TH>The restriction enzyme sites list. It is devided by different standards, type (IIP, IIA, IIB), cost (standard, nonexpensive) and etc.</TH><TH>Standard_and_IIB</TH><TH>IIP/IIA/IIB/Standard/
Nonexpensive/
Standard_IIB
Nonexpensive_IIB</TH></TR><TR><TH>a2</TH><TH>2k to 10k assembly strategy (Gibson or Goldengate)</TH><TH>Gibson</TH><TH>Gibson/ Goldengate</TH></TR><TR><TH>a10</TH><TH>10k to 30k assembly strategy (Gibson or Goldengate)</TH><TH>Goldengate</TH><TH>Gibson/ Goldengate</TH></TR><TR><TH>ckmax2</TH><TH>The maximum length of minichunks</TH><TH>2200 bp</TH></TR><TR><TH>ckmin2</TH><TH>The minimum length of minichunks </TH><TH>1800 bp</TH></TR><TR><TH>cknum</TH><TH>The number of minichunks in a chunk</TH><TH>5</TH></TR></TBODY></TABLE><PRE>
Codon table list:
1. The Standard Code
2. The Vertebrate Mitochondrial Code
3. The Yeast Mitochondrial Code
4. The Mold, Protozoan, and Coelenterate Mitochondrial Code and the Mycoplasma/Spiroplasma Code
5. The Invertebrate Mitochondrial Code
6. The Ciliate, Dasycladacean and Hexamita Nuclear Code
7. The Echinoderm and Flatworm Mitochondrial Code
8. The Euplotid Nuclear Code
9. The Bacterial, Archaeal and Plant Plastid Code
10. The Alternative Yeast Nuclear Code
11. The Ascidian Mitochondrial Code
12. The Alternative Flatworm Mitochondrial Code
13. Blepharisma Nuclear Code
14. Chlorophycean Mitochondrial Code
15. Trematode Mitochondrial Code
16. Scenedesmus Obliquus Mitochondrial Code
17. Thraustochytrium Mitochondrial Code
18. Pterobranchia Mitochondrial Code
19. Candidate Division SR1 and Gracilibacteria Code
</PRE></P><P><B>The format of utput</B></P><P>The output file is stored in /the path where you install GENOVO/Result/. 02.globalREmarkup.
Besides, there is screen output about the process state and result.
1. Screen output
2. *.parse
Store the exited enzyme recognition site in the megachunks
Enzyme ID	Start	End	Recognition site	Real site</P><P><B>3-3. 03.chunk_30k_10k_2k.pl</B></P><P>This utility can produce 2k minichunks with Gibson adaptors and 10k chunks with goldengate adaptors.</P><P>Internal operation</P><P>This utility will segmentate the megachunk produced by 03.mega2chunk2mini.pl into 2k minichunks with Gibson assembly adaptors, so that they can be put together into 10k chunks.
First, this bin will search the inexistent restriction enzyme sites locally, and then decide the size of the minichunks according to the requirements from users, and add two same Gibson adaptors to each sides of minichunks.
Secondly, the second part of this bin will define the start and end point of the chunks as users asked and design goldengate assembly adaptors for the chunks.
The output file can be sent in gene synthesis company after human attention and double check.
For more information about segmentation design, please refer to the page ASSEMBLY DESIGN PRINCIPLE .</P><P><B>Example (command line)</B></P><P>perl 03.mega2chunk2mini.pl -re standard_and_IIB -sg 01.whole2mega/sce_chr01_0.mega -ps 02.globalREmarkup/sce_chr01_0.parse  -ot 03.mega2chunk2mini</P><P><B>Parameters</B></P><P><TABLE border="1"><TBODY><TR><TH>default</TH><TH>Option</TH></TR><TR><TH>sg</TH><TH>The fasta file of the 30k segmentation, the output of 01.wh2mega.pl</TH></TR><TR><TH>ps</TH><TH>The markup file of the 30k segmentation, the output of 02.globalREmarkup.pl</TH></TR><TR><TH>re</TH><TH>The restriction enzyme sites list. It is devided by different standards, type (IIP, IIA, IIB), cost (standard, nonexpensive) and etc.</TH><TH>Standard_and_IIB</TH><TH>IIP/IIA/IIB/Standard/
Nonexpensive/
Standard_IIB
Nonexpensive_IIB</TH></TR><TR><TH>a2</TH><TH>2k to 10k assembly strategy (Gibson or Goldengate)</TH><TH>Gibson</TH><TH>Gibson/ Goldengate</TH></TR><TR><TH>a10</TH><TH>10k to 30k assembly strategy (Gibson or Goldengate)</TH><TH>Goldengate</TH><TH>Gibson/ Goldengate</TH></TR><TR><TH>ckmax2</TH><TH>The maximum length of minichunks</TH><TH>2200 bp</TH></TR><TR><TH>ckmin2</TH><TH>The minimum length of minichunks </TH><TH>1800 bp</TH></TR><TR><TH>cknum</TH><TH>The number of minichunks in a chunk</TH><TH>5</TH></TR></TBODY></TABLE>
If parameter a2 is Gibson, then there are additional parameters:<TABLE border="1"><TBODY><TR><TH>ol2</TH><TH>The length of overlap</TH><TH>40 bp</TH></TR><TR><TH>tmax2</TH><TH>The maximum melting temperature of the overlap of minichunks</TH><TH>60℃</TH></TR><TR><TH>tmin2</TH><TH>The minimum melting temperature of the overlap of minichunks</TH><TH>56℃</TH></TR><TR><TH>fe2</TH><TH>The minimum free energy of the overlap of minichunks</TH><TH>-3</TH></TR><TR><TH>ex2</TH><TH>The type of exonuclease used for minichunks</TH><TH>T5</TH><TH>T5/T3</TH></TR><TR><TH>lo2</TH><TH>The minimum distance between minichunks overlap and loxpsym</TH><TH>40 bp</TH></TR><TR><TH>en2</TH><TH>The type of enzyme flanking minichunks</TH><TH>IIP</TH></TR><TR><TH>et2</TH></TR><TR><TH>ep2</TH><TH>The maximum unit price of enzyme used in minichunks digestion</TH><TH>0.5 $/unit</TH></TR></TBODY></TABLE>
If parameter a10 is Goldengate, then there are additional parameters:<TABLE border="1"><TBODY><TR><TH>en10</TH><TH>The type of enzyme flanking chunks</TH><TH>IIB</TH><TH>IIA/IIB</TH></TR><TR><TH>et10</TH><TH>The temperature of enzyme used in chunks digestion</TH><TH>37℃</TH></TR></TBODY></TABLE></P><P><B>The format of ouput</B></P><P>The output file is stored in /the path where you install GENOVO/Result/. 03.mega2chunk2mini.
Besides, there is screen output about the process state and result.
1. Screen output
2. *.2kstate
Store the minichunks states.<TABLE border="1"><TBODY><TR><TH>Left IIP enzyme site</TH><TH>Right IIP enzyme site</TH><TH>Start</TH><TH>End</TH><TH>Size of minichunks</TH><TH>Melting temperature of overlap</TH></TR></TBODY></TABLE>
3. *.10kstate
Store the chunks states<TABLE border="1"><TBODY><TR><TH>Left IIB enzyme site</TH><TH>Right IIB enzyme site</TH><TH>Start</TH><TH>End</TH><TH>Size of chunks</TH></TR></TBODY></TABLE>
4. *.mini
Store the fasta of designed minichunks.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Shenzhen_BGIC_0101/Tutorial/segmman">http://2013.igem.org/Team:Shenzhen_BGIC_0101/Tutorial/segmman</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Shenzhen_BGIC_0101/Tutorial/segmman" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Shenzhen_BGIC_0101/Tutorial/segmman" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_0101/Tutorial/segmman&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_0101/Tutorial/segmman&amp;oldid=343501" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>