<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Shenzhen_BGIC_0101_Tutorial_nucleomod"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Shenzhen_BGIC_0101/Tutorial/nucleomod">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Shenzhen_BGIC_0101/Tutorial/nucleomod&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_0101/Tutorial/nucleomod&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_0101/Tutorial/nucleomod&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Shenzhen_BGIC_0101/Tutorial/nucleomod" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Shenzhen BGIC 0101/Tutorial/nucleomod</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="navigator"><UL class="sf-menu sf-navbar"><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101">Home</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Team">Team</A></LI><LI><A class="sf-with-ul" href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Software">Software<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Software">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Modules">Modules</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Next_version">Next Version</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/NewStandard">New Standard</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Compatibility">Compatibility</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Tutorial">Tutorial</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Web-based_trial">Web-Based Trial</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Downloadsl">Downloads</A></LI></UL><LI><A class="sf-with-ul" href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Requirements">Requirements<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Requirements#HumanPractices">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Requirements#Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Requirements#collaboration">Collaboration</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Criteria">Medal Criteria</A></LI><LI><A href="https://2013.igem.org/Team:Shenzhen_BGIC_0101/Contact">Contact</A></LI></DIV><B><H1>Tutorial</H1></B><H1>NucleoMod </H1><P>NucleoMod can modify CDS based on synonymous mutation. It has 5 applications. Firstly, NucleoMod is used to design CRISPR sites on NeoChr so that we can silence the wild type genes. Secondly, it can erase specific enzyme sites according to the users' selection. Thirdly, users can create an enzyme site in selected region of specific genes. Fourthly, it can optimize the codon efficiency to increase the expression level. Finally, it can smash the tandem repeat bases to reduce the synthesis difficulty.</P><H2> Plugins </H2><P>This module contains 5 plugins: CRISPR design, erase enzyme site, create enzyme site, codon optimization, repeat smash. All plugins are included in the main program.</P><H3>2.1	CRISPR design</H3><P>This plugin is used to design CRISPR site of NeoChr genes so that we can silence the wild type genes. We use blast+ to ensure the uniqueness of CRISPR sites. If you are using more than one plugin at the same time, this plugin will start firstly and deliver the data to next plugin. Otherwise it will generate a new fasta file for sequence and gff file for annotation.</P><P><B>2.1.1 Internal operation</B></P><P>First, this plugin extracts sequence and annotation from the NeoChr FASTA file and GFF3 file, respectively. Regular expression will be applied to find the 23bp basic structure of CRISPR site, with a head of ‘G’ then following 20 facultative bases and finally followed by ‘GG’. All the sequences and locus will be record in an array. 
Second, the blast+ will be used to check whether the 12bp sequences (from 9th to 20th) are uniq in the wild type genome. Only uniq sites will be reserved. 
Third, synonymous substitution method will be applied to change one base between the 9th to 20th bases of the CRISPR structure. The result will be record in GFF as an element of gene. If –verbose is set, the designed number will be report in STDOUT.
Finally, if this plugin is the last module, the sequence and annotation information will be recreated in FASTA and GFF format.</P><P><B>2.1.2	Example</B></P><P>We have two input forms to execute the plugin:
Run CRISPR design plugin only:
perl NucleoMod.pl -inputfa NeoChr.fa -inputgff NeoChr.gff -outputgff new_annotation.gff -outputfa new_chr.fa -crisprnum 2 -database saccharomyces_cerevisiae_chr.fa</P><P><B>2.1.3	Parameters</B></P><TABLE border="1"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Default</TH><TH>Selectable range</TH></TR><TR><TH>inputfa</TH><TH>The NeoChr sequence file in FASTA format</TH><TH>string</TH></TR><TR><TH>inputgff</TH><TH>The NeoChr annotation file in GFF3 format</TH><TH>string</TH></TR><TR><TH>outputgff</TH><TH>Output of new chromosome annotation in GFF3 format</TH><TH>string</TH></TR><TR><TH>outputfa</TH><TH>Output of new chromosome sequence in FASTA format</TH><TH>string</TH></TR><TR><TH>verbose</TH><TH>Output the detailed information in STDOUT</TH><TH>none</TH><TH>option</TH></TR><TR><TH>crisprnum</TH><TH>Number of CRISPR site to be design per gene</TH><TH>Int (&gt;0)</TH></TR><TR><TH>database</TH><TH>The sequence of reference genome, used as blast+ database</TH><TH>string</TH></TR><TR><TH>help</TH><TH>Show help information</TH></TR></TBODY></TABLE><P><B>2.1.4	The format of output file</B></P><P>The output files are standard GFF and FASTA format files.
1. GFF file
2. FASTA file
3. Detailed information in STDOUT</P><H3>2.2 Erase enzyme site</H3><P>Given a list of restriction enzyme information, this plugin will erase the restriction sites in every gene. If you are using more than one plugin at the same time, this plugin will start after CRISPR design and deliver the data to next plugin. Otherwise it will generate a new fasta file for sequence and gff file for annotation.</P><P><B>2.2.1	Internal operation</B></P><P>The enzyme information will be extracted. (If the –borbrickstandard parameter is set, it will also remove EcoRI, XbaI, SpeI, PstI and NotI) The recognize site will be reformatted to regular expression and searched in the CDS regions.
Once a restriction site is matched, synonymous substitution method will be applied to try to erase the enzyme site. When the substitution is finished, the plugin will restart the next search from 1 base after the last matched position.
If this plugin is the last module, the sequence and annotation information will be recreated in FASTA and GFF format.</P><P><B>2.2.2	Example</B></P><P>perl NucleoMod.pl -inputfa NeoChr.fa -inputgff NeoChr.gff -outputgff new_annotation.gff -outputfa new_chr.fa –biobrickstandard [-delenzymelist enzyme.list ]
	Format of enzyme.list:
	Company   enzyme_name   enzyme_site   …
　　Eg. NEB       BamHI        G/GATCC</P><P><B>2.2.3	Parameters</B></P><TABLE border="1"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Default</TH><TH>Selectable range</TH></TR><TR><TH>inputfa</TH><TH>The NeoChr sequence file in FASTA format</TH><TH>string</TH></TR><TR><TH>inputgff</TH><TH>The NeoChr annotation file in GFF3 format</TH><TH>string</TH></TR><TR><TH>outputgff</TH><TH>Output of new chromosome annotation in GFF3 format</TH><TH>string</TH></TR><TR><TH>outputfa</TH><TH>Output of new chromosome sequence in FASTA format</TH><TH>string</TH></TR><TR><TH>verbose</TH><TH>Output the detailed information in STDOUT</TH><TH>none</TH><TH>option</TH></TR><TR><TH>biobrickstandard</TH><TH>Erase the biobrick standard enzyme site</TH><TH>none</TH><TH>option</TH></TR><TR><TH>delenzymelist</TH><TH>The file of enzyme going to delete</TH><TH>string</TH></TR><TR><TH>detail</TH><TH>Show the erased enzyme site in new gff</TH><TH>none</TH><TH>option</TH></TR><TR><TH>help</TH><TH>Show help information</TH></TR></TBODY></TABLE><P><B>2.2.4	The format of output</B></P><P>The output files are standard GFF and FASTA format.
1. GFF file

2. FASTA file

3. Detailed information in STDOUT</P><H3>2.3	Create enzyme site</H3><P>Given a list of restriction enzyme information, this plugin can create a new enzyme site in specific region of selected gene. If you are using more than one plugin at the same time, this plugin will start after erase enzyme site and deliver the data to next plugin. Otherwise it will generate a new fasta file for sequence and gff file for annotation.</P><P><B>2.3.1	Internal operation</B></P><P>First, information of enzyme site will be extracted. According to 3 reading frames, a searching tree will be constructed and converted to regular expression. 
The plugin will search the selected regions and then change the sequence to enzyme site by synonymous substitution method.
If this plugin is the last module, the sequence and annotation information will be recreated in FASTA and GFF format.</P><P><B>2.3.2	Example</B></P><P>perl NucleoMod.pl -inputfa NeoChr.fa -inputgff NeoChr.gff -outputgff new_annotation.gff -outputfa new_chr.fa -addenzymelist enzyme.list -addenzymeconfig gene_id,start_pos,end_pos,enzyme_name</P><P><B>2.3.3	Parameters</B></P><P><TABLE border="1"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Default</TH><TH>Selectable range</TH></TR><TR><TH>inputfa</TH><TH>The NeoChr sequence file in FASTA format</TH><TH>string</TH></TR><TR><TH>inputgff</TH><TH>The NeoChr annotation file in GFF3 format</TH><TH>string</TH></TR><TR><TH>outputgff</TH><TH>Output of new chromosome annotation in GFF3 format</TH><TH>string</TH></TR><TR><TH>outputfa</TH><TH>Output of new chromosome sequence in FASTA format</TH><TH>string</TH></TR><TR><TH>verbose</TH><TH>Output the detailed information in STDOUT</TH><TH>none</TH><TH>option</TH></TR><TR><TH>addenzymelist</TH><TH>The file of enzyme to get enzyme site information</TH><TH>string</TH></TR><TR><TH>addenzymeconfig</TH><TH>A array of string to specify enzyme and regions</TH><TH>string,int,int,string</TH></TR><TR><TH>help</TH><TH>Show help information</TH></TR></TBODY></TABLE></P><P><B>2.3.4	The format of ouput</B></P><P>The output files are standard GFF and FASTA format.
1. GFF file

2. FASTA file

3. Detailed information in STDOUT</P><H3>2.4 Codon optimization</H3><P>Given a codon priority list, this plugin is used to optimize the codon so that we can increase the expression of selected genes. If you are using more than one plugin at the same time, this plugin will start after create enzyme site and deliver the data to next plugin. Otherwise it will generate a new fasta file for sequence and gff file for annotation.</P><P><B>2.4.1	Internal operation</B></P><P>The codon with same amino acid will be separated into 3 ranks, best normal and worst. Every codon of selected gene will be check whether the codon is in best rank. The codon in normal or worst will be change to best rank by synonymous substitution method.
If this plugin is the last module, the sequence and annotation information will be recreated in FASTA and GFF format.</P><P><B>2.4.2	Example</B></P><P>perl NucleoMod.pl -inputfa NeoChr.fa -inputgff NeoChr.gff -outputgff new_annotation.gff -outputfa new_chr.fa -codonoptimize CodonPriority.txt -optimizeallgene [-optimizegenelist gene1,gene2,gene3 ]</P><P><B>2.4.3	Parameters</B></P><P><TABLE border="1"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Default</TH><TH>Selectable range</TH></TR><TR><TH>inputfa</TH><TH>The NeoChr sequence file in FASTA format</TH><TH>string</TH></TR><TR><TH>inputgff</TH><TH>The NeoChr annotation file in GFF3 format</TH><TH>string</TH></TR><TR><TH>outputgff</TH><TH>Output of new chromosome annotation in GFF3 format</TH><TH>string</TH></TR><TR><TH>outputfa</TH><TH>Output of new chromosome sequence in FASTA format</TH><TH>string</TH></TR><TR><TH>verbose</TH><TH>Output the detailed information in STDOUT</TH><TH>none</TH><TH>option</TH></TR><TR><TH>codonoptimize</TH><TH>Codon priority list to get the ranking information</TH><TH>string</TH></TR><TR><TH>optimizeallgene</TH><TH>Optimize all genes in inputgff</TH><TH>option</TH></TR><TR><TH>optimizegenelist</TH><TH>A list of gene going to optimize,  separate by comma</TH><TH>string,string,string,...</TH></TR><TR><TH>detail</TH><TH>Show the optimization sequence in new gff</TH><TH>none</TH><TH>option</TH></TR><TR><TH>help</TH><TH>Show help information</TH></TR></TBODY></TABLE></P><P><B>2.4.4	The format of ouput</B></P><P>The output files are standard GFF and FASTA format.
1. GFF file
2 .FASTA file
3. Detailed information in STDOUT</P><H3>2.5 Repeat smash</H3><P>This plugin go through the CDS region to find out the tandem repeat bases. Synonymous substitution method will be applied to break long tandem repeat base to reduce the synthesis difficulty. If you are using more than one plugin at the same time, this plugin will start finally and then it will generate a new fasta file for sequence and gff file for annotation.</P><P><B>2.5.1	Internal operation</B></P><P>Regular expression is used to find out the tandem repeat bases longer then specified length (usually longer than 5bp). From the third of the matched sequence, synonymous substitution method will be applied to break the tandem repeat bases. 
If the substitution is successful and the rest sequence is still longer than the cutoff, then it will move to next 3 bases and do the same thing. 
The sequence and annotation information will be recreated in FASTA and GFF format.</P><P><B>2.3.2	Example</B></P><P>perl NucleoMod.pl -inputfa NeoChr.fa -inputgff NeoChr.gff -outputgff new_annotation.gff -outputfa new_chr.fa -repeatsmash 5</P><P><B>2.3.3	Parameters</B></P><P><TABLE border="1"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Default</TH><TH>Selectable range</TH></TR><TR><TH>inputfa</TH><TH>The NeoChr sequence file in FASTA format</TH><TH>string</TH></TR><TR><TH>inputgff</TH><TH>The NeoChr annotation file in GFF3 format</TH><TH>string</TH></TR><TR><TH>outputgff</TH><TH>Output of new chromosome annotation in GFF3 format</TH><TH>string</TH></TR><TR><TH>outputfa</TH><TH>Output of new chromosome sequence in FASTA format</TH><TH>string</TH></TR><TR><TH>verbose</TH><TH>Output the detailed information in STDOUT</TH><TH>none</TH><TH>option</TH></TR><TR><TH>repeatsmash</TH><TH>The tandem repeat bases longer or equal to this cutoff will be smashed</TH><TH>int</TH></TR><TR><TH>detail</TH><TH>Show the repeat smash result in new gff</TH><TH>none</TH><TH>option</TH></TR><TR><TH>help</TH><TH>Show help information</TH></TR></TBODY></TABLE></P><P><B>2.3.4	The format of ouput</B></P><P>
The output files are standard GFF and FASTA format.
1. GFF file
2. FASTA file
3. Detailed information in STDOUT</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Shenzhen_BGIC_0101/Tutorial/nucleomod">http://2013.igem.org/Team:Shenzhen_BGIC_0101/Tutorial/nucleomod</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Shenzhen_BGIC_0101/Tutorial/nucleomod" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Shenzhen_BGIC_0101/Tutorial/nucleomod" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_0101/Tutorial/nucleomod&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Shenzhen_BGIC_0101/Tutorial/nucleomod&amp;oldid=343512" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>