"
Page
Discussion
View source
History
teams
Log in
 
Team:Shenzhen BGIC 0101/Modules
From 2013.igem.org
Home
Team
Software »
Overview
Modules
Next Version
New Standard
Compatibility
Tutorial
Web-Based Trial
Downloads
Requirements »
Human Practices
Safety
Collaboration
Medal Criteria
Contact
Neochr - New Chromosome Designer
The Neochr module contains three plugins: Decouple, Add and delete.
Neochr module uses public data from Saccharomyces Genome Database, Kyoto Encyclopedia of Genes and Genomes , E.coli genome project , Virulence Factors of Pathogenic Bacteria .
NucleoMod - Nucleotide Sequence Modifier
When we extract gene from wild type genome to create a new chromosome, We need to silence the original wild type gene. The NucleoMod module can design CRISPR site to reach this goal.And the module can optimize the codon to increase expression level of genes.
SegmMan - Split for Assembly
The synthesizer or synthesis chip can up to 3kb DNA sequence with high accuracy, but chromosome is not that short.
SegmMan can settle this problem, it splits chromosome into 30k fragments, after parsing its exited enzyme sites, continues segmentation into 10k and 2k fragments. In 10k and 2k level, its will add vector homologous region and design enzyme sites.
OLS(Oligo library synthesis) Designer
For the users who synthesize DNA in chip can utilize OLS(Oligo library synthesis) Designer to design oligos and enzyme sites, primers to assemble 500-800 fragments.
(a) Pre-designed oligonucleotides (no distinction is made between dsDNA and ssDNA in the figure) are synthesized on a DNA microchip.
(b) cleaved to make a pool of oligo nucleotides.
(c) Plate-specific primer sequences (yellow or brown) are used to amplify separate plate subpools (only two areshown), which contain DNA to assemble different genes(only three are shown for each plate subpool).
(d) Assembly-specific sequences (shades of blue) are used to amplify assembly subpools that contain only the DNA required to make a single gene.
(e) The primer sequences are cleaved using either type IIS restriction enzymes (resulting in dsDNA) or by DpnII/USER/λ exonuclease processing (producing ssDNA).
(f) Construction primers(shown as white and black sites flanking the full assembly) are then used in an assembly PCR reaction to build a gene from each assembly subpool.
Others
Presentation from KGML
This module will grab genes’ detail information in KEGG Makeup Language (KGML) file which can be downloaded in KEGG or get through KEGG API, and it will establish a new standard for data transmission which will convert XML format into JSON format and simplify structures. Furthermore, this module will export genes’ list and its relationships. Choose one pathway and this module will visualize the pathway and rebuild it in the level of genes.
Retrieved from " http://2013.igem.org/Team:Shenzhen_BGIC_0101/Modules "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
