Team:BIT-China/Notebook
Medal fullfillments
Parts Submission
Description
Design
Results
Proof
Lab Book
Protocol
Equipments & Materials
Safety
Members
Collaborations
Attributions
Gallery
Gaint Jamboree
HP_Gold
HP_Silver
NoteBook
March [Expand]
We got a good idea about plasmid segragational instability of our project in 2016.
07/03/2016--26/03/2016 Mar 7th
1. After discussion, the main goal of our project was to implement the self-killing of bacteria while the plasmid number reduces to a threshold.
27/03/2016—02/04/2016 Mar 27th
1. Divided into three wet experiment group according to our project design. Killer group was to test the lethal efficiency of different killer genes. Inhibitor group was to test the threshold quantitatively. Integration group was to integrate the designed part into the genome.
April [Expand]
To get these parts what we need in kit box and preliminary build the validation devices.
03/04/2016--09/04/2016 Apr 3rd
1. Optimize the group division and prepare the some experimental reagents and equipment.
10/04/2016 Apr 10th
1. Decide to employ the CRISPR Cas9 coupled λ-Red recombination to improve the integration efficiency and simplify the selection process.
2. We got new parts pBAD (2015kit3 14A) andB0010 (2015 kit4 16G) from iGEM parts library. We transformed pBAD into TOP 10 and seeded/incubated in LB plates added chloramphenicol and transformed B0010 into TOP 10 and seeded/incubated in LB plates added Ampicillin.
3. This weekend，we designed the experiment of CRISPER/Cas9 team and made sure what circuits we need to construct.
11/04/2016 Apr 11st
1. Colony PCR：They had a positive result. And we incubated it in LB（Add antibiotics by 1000:1）.
2. We chosen TOP 10 as our host in this project, and looked for some genome information for preparation of recombination .
3. We chosen TOP 10 as our host in this project, and looked for some genome information for preparation of recombination .
12/04/2016 Apr 12nd
1. And Send the plasmids for sequencing in order to confirm the parts. (pBAD,B0010)
2. Choosing lacZ as target gene, we designed the sgRNA of this was site through online tool.
3. Choosing lacZ as target gene, we designed the sgRNA of this site through online tool.
13/04/2016 Apr 13rd
1. Design the primers of pBAD+B0034+LacIq
2. Through different online tool, we get different groups of sgRNA, after compare with each other, we chosen the best we thought.
14/04/2016 Apr 14th
1. The result of pBAD was positive，and B0010 was nagetive.
15/04/2016 Apr 15th
1. So We got new parts B0010 (2014 kit 4  16G) again, we decided to repeat transformation again.
2. We designed the primers and sent order to company, these primers will be used to construct the circuit EcoRI－N20-sgRNA－B0015-XbalI.
16/04/2016 Apr 16th
1. Colony PCR：Theyhad a positive result. And we incubated it in LB.
17/04/2016 Apr 17th
1． And Send the plasmids for sequencing in order to confirm the parts. (B0010)
2． We get the strains containing the sgRNA backbone plasmids, after bacilliculture, we extracted this plasmid. Plasmid PUC19 were transformed into TOP 10 and incubated in LB plates added Ampillin.
18/04/2016 Apr 18th
1. Run PCR for amplification of LacIq
2. Run PCR for amplification of LacIq
3.  LacIq using pfu PCR program.
4. DNA clean/purification:
5. Purification of RCR/Restriction digestion production by running agarose gel electrophoresis. Extract the right brand by TIANgel Midi Purification Kit.
6. Maked sure that PUC19 had been successfully transformed into TOP 10 vie colony PCR.
7. At the same time the another colony PCR result showed that the sgRNA was successfully ligated into plasmid PUC57.
8. Maked sure that PUC19 had been successfully transformed into TOP 10 vie colony PCR.
19/04/2016 Apr 19th
1. The result was negative.
2. To combine sgRNA and B0015 these two parts together, we run PCR for amplification of these two part separately, the result of agarose gel electrophoresis showed that sgRNA fragment was right but there was no B0015 fragment .
20/04/2016 Apr 20th
1. Designing the primers of B0010.
2. Purification of PCR production to get B0015 fragment, then run OE RCR to connect it with sgRNA fragment, so we get EcoRI－N20-sgRNA－B0015-XbalI this circuit.
21/04/2016 Apr 21st
1. We got plasmid contain λ-red gene and transformed it into TOP 10.
2. Purification of PCR production to get B0015 fragment, then run OE-PCR to connect it with sgRNA fragment, so we get EcoRI－N20-sgRNA－B0015-XbalI this circuit.
3. We get plasmid contain λ-red gene and transformed it into TOP 10.
22/04/2016 Apr 22nd
1. Connect B0010 with primers，Purification of RCR/Restriction digest production by running agarose gel electrophoresis. Extract the right brand by TIANgel Midi Purification Kit.
23/04/2016 Apr 23rd
1. Connect pBAD+B0034+lacIq with primers,Purification of RCR production by running agarose gel electrophoresis. Extract the right brand by TIANgel Midi Purification Kit.
2. We confirmed the positive colony of circuit EcoRI－N20-sgRNA－B0015-XbalI, after germiculture, these bacterial were sent to sequence.
24/04/2016 Apr 24th
1. Overlap-extension PCR to connect （pBAD+B0034+lacIq） with B0010.
2. Purification of RCR/Restriction digest production by running agarose gel electrophoresis. Extract the right brand by TIANgel Midi Purification Kit.
24/04/2016 Apr 24th
1. Overlap-extension PCR to connect （pBAD+B0034+lacIq） with B0010.
2. Purification of RCR/Restriction digest production by running agarose gel electrophoresis. Extract the right brand by TIANgel Midi Purification Kit.
25/04/2016 Apr 25th
1. Restriction digestion of gene circuit (pBAD+B0034+LacIq+B0010) with EcoRI and PstI at 37℃ for 4 hours.
2. Ligation of (pBAD+B0034+lacIq+B0010) with pSB1C3 by T4 ligase.
Ligation products (pBAD+B0034+LacIq+B0010) were transformed into TOP 10   and seeded in LB plates.
3. To design the left and right homologous arms, we needed exact sequence of lacZ gene. But in TOP 10, this gene was modified, so we design the primers contain EcoRI and PstI to get the lacZ gene, and got it sequence vie sequen
4. To design the left and right homologous arms, we need exact sequence of lacZ gene.But in TOP 10, this gene was modified, so we design the primers contain EcoRI and PstI to get the lacZ gene, and get it sequence vie sequencing.
26/04/2016 Apr 26th
1. Use Colony PCR to check whether the ligation worked as expected.
2. Colony PCR：They had a positive result. And we incubated it in LB.
3. Run PCR for amplification of lacZ gene, after purification, the fragment was digested by EcoRI and PstI, we also digested the vector PUC19 with same restriction enzyme.After 4 hours digestion, we ligated these two fragment.
27/04/2016 Apr 27th
1. And Send the plasmids for sequencing in order to confirm the parts. (pBAD+B0034+LacIq+B0010 and pTAC)
2. PCR was carried out on RFP and pTac using pfu PCR program.
3. Transformed the ligation produce of yesterday, and incubate the bacterial at 37℃.
The sequence result of 20th April showed that device 1 was right.
28/04/2016 Apr 28th
1. Purification of RCR/Restriction digest production by running agarose gel electrophoresis. Extract the right brand by TIANgel Midi Purification Kit.
2. The result was negative.
3. Transformed the ligation produce of yesterday, and incubate the bacterial at 37℃. The sequence result of 20th April showed that device 1 was right.
4. Run colony PCR to get the positive colony from yesterday’s plate. Incubate it and planed to sent it to sequence.
29/04/2016 Apr 29th
1. PCR was carried out on RFP and pTac using pfuPCR program again.
2. Purification of RCR/Restriction digest production by running agarose gel electrophoresis. Extract the right brand by TIANgel Midi Purification Kit.
3. The result was positive.
4. Ligation of them by T4 ligase.
5. Run colony PCR to get the positive colony from yesterday’s plate. Incubate it and planned to send it to sequence.
6. Sent the bacterial to sequence.
30/04/2016 Apr 30th
1. Ligation products (pTAC and RFP) were transformed into TOP 10  and seeded in LB plates.
2.  We decided to use mazF and HokD as our killer gene.
We failed to get mazF from TOP 10 via PCR and finally decided to use the original bacteria MG1655.
3. 3A assembly: pBAD-B0032-HokD, successful.                                                 pBAD+B0032+mazF, unsuccessful.
May [Expand]
Building gene circuits in May.
02/05/2016 May 2nd
1.  Designing the primers of which included B0034 to connect B0034 with pTAC+B0034+RFP.
04/05/2016 May 4th
1. We got the sequence result, just as we guessed, there were some difference in lacZ gene between TOP 10 and MG1655.
05/05/2016 May 5th
1. For the low quality of sequence, To confirm the sequence, we sent the bacterial to sequence again.
06/05/2016 May 6th
1. We called the company sold the BM TOP 10 for the exact sequence, but they rejected us.
07/05/2016 May 7th
1. Overlap-extension PCR to connect （pTAC+B0034） with RFP.
2. Purification of RCR production by running agarose gel electrophoresis. Extract the right brand by TIANgel Midi Purification Kit.
08/05/2016 May 8th
1. Ligation of them by T4 ligase.
2. 3A assembly: pBAD-B0032-HokD, successful.
3. pBAD+B0032+mazF, unsuccessful.
09/05/2016 May 9th
1. Ligation products (pTAC+B0034+RFP) were transformed into TOP 10  and seeded in LB plates.
2. The sequence quality was bad still. We designed the primers again, plant to sequence it again.
10/05/2016 May 10th
1. Colony PCR：They had a negative result.
2. During this time, we got the exact sequence of lacZ, we designed the homologous arm referring to this sequence.
3. We began to constrict the Device3 and Device4.
Device3:  PstI－pBAD－B0034－λ_Red－B0015－BamH
Device4:  BamHI－pBAD－B0034－Cas9－B0015－KpnI
These two devices would be constricted in vector pSTV29.
11/05/2016 May 11st
1. Run PCR for amplification of pTAC and RFP. Purification of RCR production by running agarose gel electrophoresis.
2. we had positive result of pTAC and had negative result of RFP.
3. Extract the right brand pTAC by TIANgel Midi Purification Kit.
12/05/2016 May 12nd
1. Run PCR for amplification of pTAC and RFP. Purification of RCR production by running agarose gel electrophoresis.
2. we had positive result of pTAC and had negative result of RFP.
3. Extract the right brand pTAC by TIANgel Midi Purification Kit.
13/05/2016 May 13rd
1. Run PCR for amplification of pTAC and RFP. Purification of RCR production by running agarose gel electrophoresis.
2. we had positive result of pTAC and had negative result of RFP.
3. Extract the right brand pTAC by TIANgel Midi Purification Kit.
14/05/2016 May 14th
1. Ligation products (pBAD+B0034+LacIq+ B0015) were transformed into TOP 10 and seeded in LB plates.
2. We began to constrict the Device3 and Device4.
3. Device3:  PstI－pBAD－B0034－λ_Red－B0015－BamH
4. Device4:  BamHI－pBAD－B0034－Cas9－B0015－KpnI
5. These two devices would be constricted in vector pSTV29.
15/05/2016 May 15th
1. PBLB’s bacteria didn’t grew up.
2. Run PCR for amplification of pTAC-RFP again. Purification of RCR production by running agarose gel electrophoresis.
3. we had negative result of pTAC-RFP
17/05/2016 May 17th
1. Ligation products PBLB (pBAD+B0034+LacIq+ B0015) were transformed into TOP 10 and seeded in LB plates again.
18/05/2016 May 18th
1. Use Colony PCR to check whether the ligation worked as expected.
2. Colony PCR：We had a negative result.
19/05/2016 May 19th
1. Run PCR for amplification of PBLB (pBAD+B0034+lacIq+B0015) with Universal Primers
20/05/2016 May 20th
1. Use the TIANprep plasmid Kit to extract plasmids.
2. Send the plasmids for sequencing in order to confirm the parts.
22/05/2016 May 22nd
1. We got the pSTV29 vector and transformed it into host DH5αto enrich it.
23/05/2016 May 23rd
1. Run PCR for amplification of λ_Red gene and purify it.
24/05/2016 May 24th
1. Run OE PCR for connection of device 3 and device 4 separately.
25/05/2016 May 25th
1. We had a positive result. (PBLB)
2. Added the homologous arms to kana resistance gene, after digestion, it was ligated with pUC19 overnight.
26/05/2016 May 26th
1. Run PCR for amplification of PBR（PTAC+B0034+RFP）.
2. Transfer the ligation product into TOP 10, incubate it overnight.
3. Send the plasmids for sequencing in order to confirm the parts (RFP).
27/05/2016 May 27th
1. The concentration of sample was too low to test.
2. Run colony PCR and get the positive colony, after incubation, it would be sent to sequence.
28/05/2016 May 28th
1. Design the primers of pTAC+B0034 and add restriction site.
June [Expand]
We finished a part of building jobs and continued to build other circuits.
1/06/2016 Jun 1st
1. Run PCR for amplification of pTAC and RFP with primers.
2. Purification of RCR production by running agarose gel electrophoresis.
3. we had negative result of pTAC.
4. And Restriction digestion of RFP with XbaI and PstI.
2/06/2016 Jun 2nd
1. Restriction digestion of pTac with EcoRI and SpeI
2. Restriction digestion of RFP with XbaI and PstI.
3. Ligation of pTac +RFP with pSB1K3 by 3A-assembly.
4. Device 3 fragment was get vie OE-PCR, after digestion, It was ligated with PSTV29 vector overnight.
03/06/2016 Jun 3rd
1. Use Colony PCR to check whether the ligation worked as expected.
2. Colony PCR：We had a negative result.
3. Restriction digestion of pTac +RFP with EcoRI and SpeI
4. And we got negative result.
5. Transformed the ligation product into TOP 10, the result of colony PCR showed that there were no positive colony.
04/06/2016 Jun 4th
1. Run PCR for amplification of pTAC and RFP with primers.
2. Purification of RCR/Restriction digestion production by running agarose gel electrophores was. We had negative result of pTAC.
3. We checked the sequence of promoter and found the restriction site of enzyme we used, so we designed the promoter and changed the enzyme we used.
05/06/2016 Jun 5th
1. Run PCR for amplification of pTAC and RFP with primers again.
2. Purification of RCR production by running agarose gel electrophoresis.
3. Restriction digestionof pTAC with EcoRI and SpeI. We had negative result of pTAC.
4. We designed the primers to standardizing the device 1.
5. PCR: pTac+B0032+MazF, sessessful.
6. 3A assembly: pBAD+B0034+lacIq+B0015 and pTac+B0032+MazF with PSB1C3, and than, transformed it into TOP 10 and seeded in LB with chloramphenicol, unsuccessful.
06/06/2016 Jun 6th
1. During this time, we finished the standardization of device1. There were some problems still existed in construction of device 3 and device 4, we guessed that’s because these two devices were too long, and the length difference between these two devices was two large.
08/06/2016 Jun 8th
1. Run PCR for amplification of pTAC and RFP with primers again.
2. Purification of RCR/Restriction digest production by running agarose gel electrophoresis.
3. Restriction digestion of pTAC with EcoRI and SpeI.
4. we had positive result of pTAC.
09/06/2016 Jun 9th
1. 3A-assembly: Ligation of pTAC+RFP with pSB1K3 by 3A-assembly.
10/06/2016 Jun 10th
1. Ligation production pTAC+RFP were transformed into TOP 10  and seeded in LB plates
11/06/2016 Jun 11st
1. Use Colony PCR to check whether the ligation worked as expected.
2. Colony PCR: select the appropriate restructuring bacteria from 6.10 contained plasmids.
3. We found pBAD promoter always being closed without araC, so we decided replace pBAD to araC-pBAD.
12/06/2016 Jun 12nd
1.  Use the TIANprep plasmid Kit to extract plasmids.
20/06/2016 Jun 20th
1. Ligation products PBLB(pBAD+B0034+LacIq+B0015)                                                 were transformed into TOP 10 and seeded/incubated in LB plates again.
23/06/2016 Jun 23rd
1. Ligation of pTAC and RFP with pSB1K3 by 3A-assembly.
24/06/2016 Jun 24th
1. Ligation products pTAC and RFP were transformed into TOP 10 and seeded/incubated in LB plates .
25/06/2016 Jun 25th
1. Use Colony PCR to check whether the ligation worked as expected.
2. Colony PCR: select the appropriate restructuring bacteria from 6.24 contained plasmids.
26/06/2016 Jun 26th
1. Use the TIANprep plasmid Kit to extract plasmids.
2. Send the plasmids for sequencing in order to confirm the parts.
27/06/2016 Jun 27th
1. We designed the primers of device4’s constriction, planned to divided the cas9 gene into two fragment and constrict it by two steps.
28/06/2016-04/07/2016 Jun 28th
1. We finished the constriction of first plasmid and added the araC gene in front of pBAD promoter. The constriction of device4 was finished.
July [Expand]
Testing the killer efficiency of the toxin gene,and we changed the recombination plan.
05/07/2016 Jul 5th
1. There were something wrong still exited in constriction of device 3, we wanted to constricted it by Gibson assembly.
2. OE: araC-pBAD-B0034 + LacIq-B0015
araC-pBAD-B0032 + MazF
araC- pBAD-B0032 + HokD
08/07/2016--12/07/2016 Jul 8th
Device 2 standardization
After finishing the construction of device 2, we have to make it a standard part. our main works were just like showed below.
1. check the sequence of device 2, make sure that there were no recognition sites of EcoR1、XbaI、SpeI、PstI these four enzyme. The result from DNAman showed that no recognition site should been mutated
2. design the primers which contained the four enzymes’ recognition sites, the primers were showed below;
5-GTT TCT TCG AAT TCG CGG CCG CTT CTA GAG TTG ACA GCT A GC TCA GTC CT-3
5-GTT TCT TCC TGC AGC GGC CGC TAC TAG TA  TAT AAA CGC AGA AAG GCC CA-3
3. we used these two primers clone device2,by this way, after purification and recovery, we got the device2 fragment with prefix and surfix.
4. Cut both modified device2 fragment and pSB1C3 with EcoR1 and PstI;
5. Ligate device2 fragment and pSB1C3 with ligase;
6. Transfer the last step product into competent cell TOP 10;
7. Spread the last step product on the LB plate contain chl, 37℃ overnight incubated ；
8. Check the single colony via COLONY PCR , and sequence the positive one.
Because some of us were not familiar enough with operating steps, we failed many times.
Fortunately, at last we got the plasmid pSB1C3 contains device2,
9. transferred araC-pBAD-B0032-HokD into cells
OE: araC-pBAD-B0034 + LacIq-B0015, unsuccessful
araC-pBAD-B0032 + MazF, unsuccessful
10. araC-pBAD-B0032-HokD colony PCR, 4&5positive result, picked up the thallus and preserve recovered araC-pBAD-B0032-MazF, unsuccessful
araC-pBAD-B0032-RFP (13.4ng/μl)
araC-pBAD-B0032-HokD (1.6ng/μl)
araC-pBAD-B0034-LacIq-B0015 (19.8ng/μl)
11. araC-pBAD-B0032-HokD distracted plasmids, genetic sequencing
araC-pBAD-B0032-MazF distracted plasmids, genetic sequencing
araC-pBAD-B0034-LacIq-B0015distracted plasmids, genetic sequencing
12. Ran colony PCR of araC-pBAD-B0032-MazF fragment
13/07/2016 Jul 13rd
1. We selected the constructed bacteria through COLONY PCR . But the result was all negative.
2. In order to connect the gene circuit araC-pBad-B0034-LacIq-pTac-B0032-RFP through GOLDEN GATE assembly, we added restricted sites to three parts, araC-pBad, B0034-LacIq-B0015, pTac-B0032-RFP, respectively through OE-PCR. And we transformed these three parts into E.coil strain TOP 10, via three vectors P O T.
3. (gram-positive bacteria on July 12th) araC-pBAD-B0032-MazF distracted plasmids, genetic sequencing
14/07/2016 Jul 14th
1. In order to test the efficiency of RFP, we added IPTG into our LB medium when bacteria’s OD had arrived 0.6. But the experimental group was the same as group control. The result elucidated that the RFP can not work.
2.   GOLDEN GATE ASSEMBLY : use restriction enzyme BsaI to splice these three parts we got on 7.13. And integrate into P O T plasmids. Transform into E.coli .
3. PCR recovery:
araC-pBAD-B0032 & MazF
araC-pBAD-B0032 & HokD
15/07/2016 Jul 15th
1. Extract the plasmid: extract the plasmid from the bacteria tested on 7.14. And send for DNA sequencing.
2. GOLDEN GATE ASSEMBLY: select the bacteria via COLONY PCR  and inoculate it into LB medium.
3. OE-PCR: connect the gene circuit araC-pBad-B0034-LacIq-B0015-pTac-B0032-RFP, the gel result was positive!
4. Digested/Ligated/Transformed:
araC-pBAD-B0032-MazF
araC-pBAD-B0032-HokD
16/07/2016 Jul 16th
1. Transformation:  get the part K864400 (pTac) from iGEM kit library for exchanging the promoter before.
2. Transformation: integrate the gene circuit made on 7.15 into plasmid pSB1C3 by restriction digestion and ligation. Then transform it into E.coli strain TOP 10.
3. GOLEDEN GATE ASSEMBLY:  Build the gene circuit araC-pBad-B0034-LacIq-B0015-pTac-B0032-RFP and integrate it into the plasmid, POT, by restriction enzyme BsmBI. Then transform the constructed plasmid into E.coli strain TOP 10.
17/07/2016 Jul 17th
1. GOLDEN GATE ASSEMBLY: select the bacteria through COLONY PCR . But the result was negative. Because the gene LacIq contains the BsmBI recognition sites.
2. OE-PCR: the transformation performed on 7. 16  was failing. We had to connect the gene circuit   araC-pBad-B0034-LacIq-B0015-pTac-B0032-RFP again.
3. ONE-STEP MUTATION: mutate the B0032 into B0034 by PCR in order to improving the expression of RFP for testing easily.
4. Inoculated the bacteria contained pTac(K864400).
18/07/2016 Jul 18th
We decided to reconbinate the killer device by λ Red recombination system. The key protein λRed came from Pkd46.
To test this system. We chose lacZ as the target gene, the kana resistance gene contain 500bp homologous arm as donor DNA.
1. We design three pairs of primers to clone two 500bp homologous arms from TOP 10 genome and the kana resistance gene from device 2;
2. After recovering the product of last step, we linked them one by one via OE PCR, and we recovered the product of this step.
3. We got Pkd46 from doctor Bo Lv and transferred it into TOP 10.
4. Incubate TOP 10 get from last step, induce TOP 10 express Beta, Exo, Gam(three key proteins of this system) by L-arabinose(10mmol/L)
5. Transfer the donor DNA into TOP 10 via electroporation, 37℃ incubated 1hour,during this time, add IPTG to induce expression of kana resistance gene;
6. Spread the product on the plate contain kana,
7. After 16 hours’ incubation, there were some colonies on the plate. But the result of colony PCR  showed that there was no positive result.
There were some factors maybe impact the result:
7.1. the concentration of donor DNA was too low;
7.2. the genome of TOP 10 was modified but we still design experiment plan refer to the genome of MG1655;
7.3. We found that the TM of primers designed last time differs too large, that may be the reason of strange PCR result.
8. EXTRACT THE PLASMID:  Extract the plasmid contained the pTac from the bacteria.
9. ONE-STEP MUTATION: Nitrification the PCR product by DpnI.
20/07/2016 Jul 20th
1. OE- PCR:  Build the new gene circuit pTac-B0032-RFP.
2. Get parts cI (C0051) and pR (R0051) from the iGEM Kit library. And transform into E.coli strain TOP 10
3. Determined growth curve in host cells(the antibiotic in controlled plates might have lost efficiency)：
araC-pBAD-B0032-HokD, null vector transfection as the control (induced&unducerd)
araC-pBAD-B0032-MazF, null vector transfection as the control (induced&unducerd)
21/07/2016 Jul 21st
1. EXTRACT THE PLASMID: get the plasmid contained the parts cI and pR .
2. COLONY PCR : Test the pTac-B0032-RFP by COLONY PCR . But the result was negative.
3. Digested/Degradation/Transformed：
araC-pBAD-B0031-MazF
araC-pBAD-B0034-MazF
araC-pBAD-B0031-HokD
araC-pBAD-B0034-HokD
22/07/2016 Jul 22nd
1. OE-PCR: Connect the gene circuit araC-pBad-B0034-cI-b0015-pR-B0032-RFP.
2. Inoculate the constructed bacteria contained gene circuit pTac-B0032-RFP.
3. Ran colony PCR:
araC-pBAD-B0031-MazF
araC-pBAD-B0034-MazF
araC-pBAD-B0031-HokD, unsuccessful
araC-pBAD-B0034-HokD, unsuccessful
23/07/2016 Jul 23rd
1. OE-PCR: Connect the gene circuit araC-pBad-B0034-cI-B0015-pR-B0032-RFP.
2. Inoculate the constructed bacteria contained gene circuit pTac-B0032-RFP.
24/07/2016 Jul 24th
1. EXTRACT THE PLASMID: Extract the plasmid from bacteria inoculated on 7.23. And the gel result was positive. And we sended the sample for sequencing.
2. OE-PCR: Construct the new gene circuit   araC-pBad-B0034-cI-B0015-pR-B0032-RFP.
3. So we changed the primers and carried out the plan again. There were better result of PCR, but no positive result still. So we decided to give up lacZ as target gene.
25/07/2016 Jul 25th
1. 3A ASSEMBLY: Connect the   araC-pBad-B0034-cI-B0015-pR-B0032-RFP by 3A assembly. And we also build the circuit by OE-PCR at the same time.
2. Get the part tetR(C0040) and pTet-B0034-RFP-B0015(I13521) from the iGEM kit library. And transform these plasmids into TOP 10 for accumulating plasmid.
26/07/2016 Jul 26th
1. 3A ASSEMBLY: Connect the araC-pBad-B0034-cI-B0015-pR-B0032-RFP by 3A assembly.
2. Inoculate the bacteria contained the tetR and pTet-B0034-RFP-B0015 for accumulation.
3. Determined growth curve in host cells
araC-pBAD-B0032-HokD, null vector transfection as the control (induced&unducerd）
araC-pBAD-B0032-MazF, null vector transfection as the control (induced&unducerd)
27/07/2016 Jul 27th
1. EXTRACT THE PLASMID: Extract the plasmid contained tetR and pTet-B0034-RFP-B0015.
3A ASSEMBLY: Connect the gene circuit araC-pBad-B0034-cI-b0015-pR-B0032-RFP.
2. Designing the primers of PBTB（pBAD+B0034+tetR+B0015）
3. SDS page
4. Designing the primers ofPBTB（pBAD+B0034+tetR+B0015）
28/07/2016 Jul 28th
1. We got new parts tetR and（pTet+B0034+RFP+B0015）from iGEM parts library. We transformed them into TOP 10 and seeded/incubated in LB plates .
2. 3A ASSEMBLY:  The bacteria which only had the gene circuit pR-B0032-RFP did not express the RFP. And we extracted the plasmid for the further connection. But the result of the restriction digestion about  araC-pBad-B0034-cI-b0015 was incorrect. The restriction site SpeI was wrong.
3. The sequencing about araC-pBad-B0034-LacIq-B0015 was incorrect. So we had to build this gene circuit again. And we built the gene circuit  araC-pBad-B0034-tetR-B0015.
4. SDS page
araC-pBAD-B0034-MazF Ran colony PCR,unsuccessful without mutation.
5. We got new parts tetR and（pTet+B0034+RFP+B0015）from iGEM parts library. We  transformedtheminto TOP 10 and seeded/incubated in LB plates .
29/07/2016 Jul 29th
1. Use Colony PCR to check whether them worked as expected. Incubate the positive bacteria in LB medium.
2. SDS page
3. Use Colony PCR to check whether themworked as expected.
Incubate the positive bacteria in LB medium.
30/07/2016 Jul 30th
We made a new plan, chose lacZ as the target gene and MRFP Expression Cassettes sandwiched by 555bp right and left homologues arms as donor DNA, we hoped that once donor DNA was recombine into genome of TOP 10, the color of red fluorescence protein would expose the positive recon.
1. we cloned the left arm and right arm from genome of TOP 10, cloned the MRFP Expression Cassettes from plasmid tested by the Inhibitor group. After recover these three fragment, we assemble them vie OEPCR, at last, we recovered the product of OEPCR;
2. the TOP 10 contain Pkd46 were prepared in the same time;
3. Transfer the donor DNA into TOP 10 vie electroporation method.
4. Spread the product of last step on blank plate and plate contain ampicillin, incubate them in 37℃ overnight.
Next day, we found that there were so many bacteria grow on the plate that even no single colony can be separated, what’s more, there was no red colony on the plate.
We then tried this plan one more time, and dilute the last product before spreading them on the plate.  After COLONY PCR , there was still no positive colony.
5. Use the TIANprep plasmid Kit to extract plasmids.
6. Construct the the gene circuits
araC-pBad-B0034-cI-b0015-pR-B0032-RFP
araC-pBad-B0034-tetR-B0015-pTet-B0032-RFP
araC-pBad-B0034-RFP.
In order to test the RFP , we transformed the RFP into T7 vector.
31/07/2016 Jul 31th
1. Prime Star :
①araC pBAD+tetR
②tetR+B0015
③araC pBAD+tetR+B0015
We had positive results.
2. Prime Star :
①+B0015
araC-pBAD+②
③araC pBAD+tetR+B0015
Purification of RCR/Restriction digest production by running agarose gel electrophoresis. Extract the right brand by TIANgel Midi Purification Kit.
We had positive result. (AraC-pBAD+② and ③araC-pBAD+tetR+B0015)
August [Expand]
The task of recombination was finished and we continued to build new gene citcuits what we need.
02/08/2016 Aug 2nd
1. PCR:
pR-B0032&B0032-MazF
pTac-B0032&B0032-MazF
2. Ligation of araC+ pBAD+tetR+B0015 and RFP with pTet+RFP by 3A-assembly.
3. urification of RCR/Restriction digest production by running agarose gel electrophoresis. Extract the right brand by TIANgel Midi Purification Kit.
03/08/2016 Aug 3rd
1. PCR:
pR-B0032&B0032-MazF
pTac-B0032&B0032-MazF
2. Use Colony PCR to check whether the ligation worked as expected
Incubate the positive bacteria in LB medium
3. We build the gene circuits:
araC-pBad-B0034-LacIq-b0015-pTac-B0032-RFP
araC-pBad-B0034-CI-B0015-pR-B0032-RFP
araC-pBad-B0034-tetR-b0015-pTet-B0032-RFP
But we can not get the recombine the bacteria all the time until we test the primer sequence. We use the 3A assembly to connect the gene circuits because the result of OE-PCR was always negative. The primer sequence was got from iGEM. But the design of restriction sites was not considered the protection base so that the restriction enzyme can not work efficiently. So we design the new primer with the consideration of the protection base and build our all gene circuit again. Luckily we get a positive gel result and we send the sample for DNA sequencing.
But the result of sequencing shows that the device, araC-pBad-B0034-CI-B0015, has a base mutation in B0015. And the device, araC-pBad-B0034-tetR-b0015, has a base mutation in tetR so that it becomes termination codon. But the adviser suggests we continue to do the further test because there was something mistake in it probable. The LacIq was completely wrong. We have to build the gene circuit araC-pBad-B0034-LacIq-b0015 again. The results of pTac-B0032-RFP-B0015 and pR-B0032-RFP-B0015.
4. We then decided to select the recon by antibiotic resistance. So we added a Chloramphenicol resistance Expression Cassettes into donor DNA, so after recombination, the right recon will have Chloramphenicol resistance and red look.But before we carry out a new plan, we plan to check competent cell, and make sure pkd46 was transferred into it. We extract plasmid from competent, the result of electrophoresis showed that no pkd46 in it, we decided to use new competent cell.
Through paper, we knew that the concentration of donor DNA must be high, and that’s one factor we will change in next few days.
We colon Chloramphenicol resistance Expression Cassettes from pkd3 and linked it with mRFP Expression Cassettes vie OEPCR,and recover the product of OEPCR;
Prepare the competent cell;
Transfer the new donor DNA into competent cell vie electroporation method , and spread it on the plate contain Chloramphenicol .
Next day, there was no colony on the plate, there must be something wrong, but had not found it. What we can do was improve the concentration of donor DNA and try this plane again, but there was still no positive result.
04/08/2016 Aug 4th
1. recovery：
pR-B0032
pTac-B0032
2. Use the TIANGEN plasmid Kit to extract plasmids. Send the plasmids for sequencing in order to confirm the parts.
05/08/2016 Aug 5th
1. Ran primer star PCR:
pR-B0032&B0032-MazF
pTac-B0032&B0032-MazF
07/08/2016 Aug 7th
1. Got GFP plasmids from kit box, transformation
08/08/2016 Aug 8th
1. Transformation on August 7th, unsuccessful
PCR&recovery&tetOE: araC-pBAD-B0032 (GFP)
pTet-B0032
GFP recovery, unsuccessful
2.   Contains PKD3 plasmids and PKD46 plasmids bacteria. Transfer into liquid culture medium. Put it in the shaking table for 12 hours. Preserve the bacteria then extract PKD3 plasmids and PKD46 plasmids.PKD46 provide theλ-red gene and PKD3 provide the chl gene.
Putting the PKD3 plasmids into TOP 10 cells. Lay over TOP 10 cells in the Amp plate. Culture it in the 30℃ incubator overnight. Because the PKD46 plasmid was a kind of temperature sensitive plasmid, we culture it in the 30℃ incubator.
Use PCR technology enrich –Ptet-mPFR- B0010 – snippet. After electrophoresis we get the right stripe then we use kit recovery the snippet and the last we verify the snippet. The result was correct.
09/08/2016—10/08/2016 Aug 9th
1. repeated OE&PCR
araC-pBAD-B0032 (GFP), unsuccessful
pTet-B0032
B0032-MazF(pTET), ran OE PCR, recovered, unsuccessful
GFP ran colony PCR/Inoculation
2. GFP distracted plasmids, unsuccessful
GFP repeated inoculation
Ran PCR, the validation result of GFP was unsuccessful
GFP distracted plasmids
3. Ran primer star PCR/Recovery:
araC+pBAD+B0032 with four different templates to do parallel experiments
(new template/old template/products recovered on August 9/water)
successful
Ran PCR:B0032+GFP, unsuccessful
4. Ran pfu PCR/recovery/Ran OE PCR: B0032+GFP with new template (sequenced) unsuccessful
5.  Get Amp Plate out of incubator. Use colony PCR  find the positive results. But we fail.
We design the primers for –chl- snippet and –Ptet-mPFR- B0010 – snippet, uses two primers to get two snippets. After electrophoresis we get the right stripe then we use kit recovery the snippets and the last we verify the snippets. The results were correct.
Connect two pieces through PCR technology. After electrophoresis we get the right stripe then we use kit recovery the snippets and the last, we verify the snippet. The result was correct.
6. Use colony PCR  find the positive results .And we fail again.
We don’t know what’s wrong, so we decided to make a contrast. Use DH5α cell as host cell. DH5αcell has PKD46 plasmids so we just lay over DH5α  cell in the Amp plate. As contrast, we lay over TOP 10 cell in the Amp plate as well. Culture it in the 30℃ incubator overnight.
11/08/2016—12/08/2016 Aug 11st
1. In the TOP 10 plate ,there no bacteria grow up, but in the DH5αplate ,there were some single colonies. Transfer it into liquid culture medium and induce the promoter with 10%Ara.
We use DH5αcell to make competent cell.
After a day we find some single colonies in the TOP 10 plate. So we use TOP 10    cell to make competent cell. To make sure the PKD46 plasmid transfer into the TOP 10 cell, we extract plasmid and use electrophoresis technology to verify, the result was correct. It means PKD46 plasmid was in the TOP 10 cell.
2. Until now, we have one contrast.DH5αcell and TOP 10 cell. In order to analysis how many homologous arms were more appropriate, we design two homologous arms, one was 555bp and another was 49bp. We have two kinds of aim snippets. Now, we have two contrasts.
DH5αcell
A.555bp homologous arms in the both sides of aim snippet.
B.49bp homologous arms in the both sides of aim snippet
TOP 10 cell
C.555bp homologous arms in the both sides of aim snippet
D.49bp homologous arms in the both sides of aim snippet                                             Here’s schematic diagram
3.We use electricity conversion to transfer two aim snippets into the two kinds of competent cell. Lay over B &D in the Chl plates and others in the Amp plates.
4. Culture it in the 30℃ incubator over ni
13/08/2016—20/08/2016 Aug 13th
1. Ran OE-PCR with different systems and annealing temperatures, successful
Recovery, unsuccessful.
2. PCR/Recovery: B0032-GFP
Ran PCR with new system and produce: araC+pBAD+B0032+GFP
3. Recovery, unsuccessful
NANO DROP4.8ng/μl
4. Troubleshooted, primers’ error
Sorted primers/Ran PCR
5. Ran PCR: araC-pBAD-B0032&GFP
AraC-pBAD-B0034&GFP
6. The result was that too many colonies in the A and C plate, but none bacteria grow up in the D plate. The bacteria only grow up in the B plate. By the verification of colony PCR , the result doesn’t have any positive bacteria.
We use different primer for colony PCR . First, we use the chl upstream and downstream primers, after electrophoresis we get the right stripe. Second, we use the mRFP upstream and downstream primers, after electrophoresis we get the wrong stripe.
We get the Chl gene but we don’t get the mRFP gene, it shows that we did not connect snippet on the cell genome. When we do it, we also do a contrast experiment .We put mRFP complete and it primers together, after colony PCR  and the electrophoresis we get the right stripe. This suggests that the primer was correct.
14/08/2016 Aug 14th
1. We use the mRFP upstream and downstream primers, after electrophoresis we get the wrong stripe again.
2. So we decide to check the whole snippet. We use VF and VR to copy the whole snippet, the whole snippet was about 2053bp, but the snippet we get was about 300bp , that was clearly wrong.
15/08/2016 Aug 15th
1. Now we were confused. Why always can't successfully transfer into the TOP 10 cell? Why did it grow up in the B plate but doesn’t have positive results?
2. We thought about the above wassues, the only way to make it have Chl gene was transfer the PKD3 plasmids into the cell by mistake. In order to get the Chl gene, we use PCR technology to get it from PKD3 plasmids. So we make a verification experiment.
3. We extract the plasmid from B plate, use it to be the complete for cat gene, and we succeed. And we also do electrophoresis to compare with PKD3 plasmid, we get the same stripe. It shows that the Chl gene was brought by PKD3. To solve this problem, we nitrify after we copy the whole snippet. It will eliminate plasmids.
16/08/2016 Aug 16th
1. We copy the Chl gene snippet and mRFP gene snippet by PCR technology.  After electrophoresis we get the right stripe then we use kit recycle the snippet and the last we verify the snippet.
2. Use OE-PCR link two snippets.
17/08/2016 Aug 17th
1. Copy the whole snippet through PCR technology, after electrophoresis we get the wrong stripe.
2. Use the bacteria we preserved at August 8. Transfer into liquid culture medium. Make it competent cell.
18/08/2016 Aug 18th
1. Copy the whole snippet through PCR technology, after electrophoresis we get the right stripe. Then we use kit recovery the snippet and verify the snippet. Nitrify the whole snippet for two hours. But the product concentration was too low
2. Design a 49bp new homologous arm for Chl gene.
3. Because we purpose fragment replacement LacI locus, we use  VF and VR to copy the lacI snippet, after electrophoresis we get the right stripe. It told us that our experimental thinking was correct, so we can replace this locus.
4. The product concentration was too low. We use PCR technology again to enrich the snippet. We use the already nitrification fragments as templates so we don't need to nitration after PCR.
19/08/2016 Aug 19th
1. Use electrophoresis to verify the whole snippet, but it was not correct.
2. Use OE-PCR connects two snippets again. This time we made it. After electrophoresis we get the right stripe then we use kit recovery the snippet and the last we verify the snippet. The result was correct.
3. Use electric turn to transfer it and lay over in the Chl plate.
20/08/2016 Aug 20th
1. We inspect the Chl plate, there was no single colony.
2. Do the paper work and prepare for tomorrow’s experiment.
3. Culture it in the 30℃ incubator overnight.
21/08/2016 Aug 21st
1. Before we make competent cell, we use 1mL 10%Ara induceλ-red Reorganization system for 2 hours, when we make competent cell, we make two kind of concentrations, one was diluted 10 times, another was diluted 100 times.
2. Recover for one hour.
3. Lay over the Chl plate. Culture it in the 30℃ incubator overnight.
22/08/2016 Aug 22nd
1. Use colony PCR  find the positive results.
2. After electrophoresis we get negative results except one have two snippets, one was 1000bp and another was 2000bp.Transfer this bacteria into liquid culture medium. Mark①.
3. There were some new single colonies in the Chl plate, so we use colony PCR  find the positive results again.
After electrophoresis, we get werid results. Expect number 11 and number 15,      they all have two snippets, one was 1000bp another was 2000bp. May be the false positive. But we still need to verify.
4.   We pick these single  colonies and transfer into  liquid culture medium to see if it lose Chl resistance. If so, the aim snippet did not link to DH5α genome.
We suspect the aim snippet did not integrate in the DH5α genome. But transfer in the cell. The snippet can live in the cell for a while. It may be the reason we get the 2000bp snippet.
5. We measured the concentration:
araC-pBAD-B0034-LacIq-B0015,
araC-pBAD-B0034-CI-B0015,
Ptac-B0032-MazF,
Pr-B0032-MazF, and PSB1C3.
Restriction digest of araC-pBAD-B0034-CI-B0015 with EcoRI and SpeI, Pr-B0032-MazF with XbaI and PstI, and PSB1C3 with EcoRI and PstI at 37°C for three hours. And then put them into 80°C water bath for 20min to stop the reactions.
23/08/2016 Aug 3rd
1. Observe bacteria①under Fluorescence microscope. No red fluorescence.
2. Observe bacteria①under Fluorescence microscope. No red fluorescence.
3. Use colony PCR find the positive results.
We finally get the right fragment.
4. Used pfu PCR program run PCR:
araC-pBAD-B0034 and B0032-GFP, unsuccessful.
○2we ran overlap-extension PCR to connect pTac-B0032 with MazF, successful.
○3araC-pBAD-B0034-CI-B0015, Pr-B0032-MazF and PSB1C3 were transformed into TOP 10 and seeded in LB with chloramphenicol.
24/08/2016—26/08/2016 Aug 24th
1. Connected the purpose fragment to T carrier. Transfer into DH5αcell. Lay over DH5αcells in the CHL plate.
2. Though Colony PCR to texting. The result was right. Transfer it into liquid culture medium.
3. Sent to sequencing.
4. Ran colony PCR  to validate the transform result: araC-pBAD-B0034-CI-B0015-Pr-B0032-MazF -PSB1C3, unsuccessful.
5. PCR was carried out on araC-pBAD-B0034 and B0032-GFP using pfu PCR program, unsuccessful.
6. We ran PCR for amplification of pTac-B0032-MazF and pR-B0032-MazF, successful.
7. We cultured bacteria which contain araC-pBAD-B0032-HokD into LB media with chloramphenicol overnight.
8. Kept the bacteria which contain araC-pBAD-B0032-HokD and abstracted its plasmid, successful.
9. We recovered the gelatin which contains araC-pBAD-B0034 and B0032-GFP, low concentration.
10. We cultured bacteria which contain araC-pBAD-B0032-MazF into LB media overnight.
11. Kept the bacteria which contain araC-pBAD-B0032-MazF and extracted its plasmid, successful.
12. we ran two overlap-extension PCR to connect pTac-B0032/pR-B0032 with MazF, successful.
27/08/2016 Aug 27th
1. .Observe bacteria under Fluorescence microscope. No red fluorescence.
28/08/2016 Aug 28th
1. Observe TA cloning bacteria under Fluorescence microscope. It has red fluorescence.
2. Restriction digest of pTac-B0032- MazF and pTet--B0032- MazF with XbaI and PstI, at 37°C for three hours. And then put them into 80°C water bath for 20min to stop the reactions.
3. Ligated of araC-pBAD-B0034-CI-B0015 (reverse consequence) and Pr-B0032-MazF with PSB1C3 by T4 ligase.
4. araC-pBAD-B0034-CI-B0015 (reverse consequence), pR-B0032-MazF and PSB1C3 were transformed into TOP 10 and seeded in LB with chloramphenicol, unsuccessful.
5. Ligation by T4 ligase overnight:
6. araC-pBAD-B0034-LacIq-B0015(reverse consequence) and pTac-B0032-MazF with PSB1C3; araC-pBAD-B0034-TetR-B0015(reverse consequence) and pTet-B0032- MazF with PSB1C3.
29/08/2016 Aug 29th
1. Help other group. We have one circuit.
2. Restriction digest:
3. pR-B0032 with MazF, and then put them into 80°C water bath for 20min to stop the reactions.
4. 3A assembly:
araC-pBAD-B0034-CI-B0015(reverse consequence) and pR-B0032-MazF with PSB1C3.
5. Transformed into TOP 10 and seeded in LB with chloramphenicol:
araC-pBAD-B0034-LacIq-B0015(reverse consequence) and pTac-B0032-MazF with PSB1C3; araC-pBAD-B0034-TetR-B0015(reverse consequence) and pTet-B0032- MazF with PSB1C3.
29/08/2016 Aug 29th
1. Design new primers for three circuits.
PR-B0034-Yfp-B0010+B0012   -cat-
PR-B0032-mRFP-B0010+B0012   -cat-
Plac-B0034-eYfp-B0010+B0012    -cat-
Cat gene was for bolting.
2. colony PCR :
araC-pBAD-B0034-LacIq-B0015(reverse consequence)-pTac-B0032-MazF-PSB1C3; araC-pBAD-B0034-TetR-B0015(reverse consequence)-pTet-B0032-MazF-PSB1C3, unsuccessful.
3. Transformed into TOP 10 and seeded in LB with chloramphenicol:
araC-pBAD-B0034-CI-B0015(reverse consequence)-pR-B0032-MazF-PSB1C3.
31/08/2016 Aug 31st
1. PCR and recovered the gelatin: pR-B0032-MazF,
pTac-B0032-MazF,
pTet-B0032-MazF, successful.
2. colony PCR :
araC-pBAD-B0034-CI-B0015(reverse consequence)-pR-B0032-MazF-PSB1C3.
3. Extracted plasmid:
araC-Pbad-B0031-MazF,
araC-pBAD-B0034-MazF,
araC-pBAD-B0031-HokD,
araC-pBAD-B0034-HokD, successful.
September [Expand]
Getting a lot of gene circuits and testing results.
01/09/2016 Sep 1st
1. Get these three fragments through PCR technology.
2. After electrophoresis we get the right stripe then we use kit recovery the snippets and the last, we verify the snippet. The result was correct.
3.Culture glycerin bacteria(DH5α) into liquid culture medium.
4. Restriction digest:
pR-B0032-MazF,
pTac-B0032-MazF,
pTet-B0032-MazF.
5. 3A assembly:
araC-pBAD-B0034-LacIq-B0015(reverse consequence) and pTac-B0032-MazF with PSB1C3; araC-pBAD-B0034-TetR-B0015(reverse consequence) and pTet-B0032- MazF with PSB1C3,and than, transformed it into TOP 10 and seeded in LB with chloramphenicol.
6. Sequenced:
araC- pBAD -B0031-MazF
araC-pBAD-B0034-MazF
araC-pBAD-B0031-HokD
araC-pBAD-B0034-HokD
7. We cultured bacteria which contain PSB1C3 into LB media overnight.
02/09/2016 Sep 2nd
1. Through OE PCR to link two fragments. After electrophoresis we get the right stripe then we use kit recovery the snippets and the last, we verify the snippet. The result was correct.
3. Culture the glycerol DH5αbacteria(contain PKD46 plasmids) into liquid culture medium.
4. Kept the bacteria which contain PSB1C3 and extracted its plasmid, successful.
5. Restriction digest and recovered the gelatin:
PSB1C3 ,araC-pBAD-B0034-CI-B0015(reverse consequence). And then put them into 80°C water bath for 20min to stop the reactions.
6. colony PCR :
araC-pBAD-B0034-LacIq-B0015(reverse consequence)-pTac-B0032-MazF-PSB1C3, unsuccessful; araC-pBAD-B0034-TetR-B0015(reverse consequence)-pTet-B0032- MazF-PSB1C3, successful.
7. We cultured bacteria which contain araC-pBAD-B0034-TetR-B0015(reverse consequence)-pTet-B0032- MazF-PSB1C3 into LB media overnight.
8. Ligation by T4 ligase overnight:
araC-pBAD-B0034-LacIq-B0015(reverse consequence)   and pTac-B0032-MazF with PSB1C3;
9. araC-pBAD-B0034-CI-B0015(reverse consequence) and pR-B0032-MazF with PSB1C3.
03/09/2016 Sep 3rd
1. Use colony PCR to find the positive result in crossed bacteria (TA colony)。
2. After electrophoresis we get the right stripe. Last time we text this fragment, it showed that the bacteria was not single clone. We pick other bacteria and text them, to make sure we have single colony.
3. Add TA colony bacteria into liquid culture medium.
4. Extract plasmids and run the plasmids, the result have two stripes. So we extract genome.
5. Make competent cell. Transfer these three fragments
(PR-B0034-Yfp-B0010+B0012+cat
PR-B0032-mRFP-B0010+B0012+cat
Plac-B0034-eYfp-B0010+B0012+cat)
into DH5αcell.
6. They were spread on LB agar plates with CHL on at 37℃.
7. Transformed into TOP 10 and seeded in LB with chloramphenicol:
araC-pBAD-B0034-LacIq-B0015(reverseconsequence)-pTac-B0032-MazF-PSB1C3;
araC-pBAD-B0034-CI-B0015(reverse consequence)-pR-B0032-MazF-PSB1C3;
araC-pBAD-B0031-HokD;
araC-pBAD-B0034-HokD.
8. Kept the bacteria which contain :araC-pBAD-B0034-TetR-B0015(reverseconsequence)-pTet-B0032- MazF-PSB1C3 and extracted its plasmid, successful.
9. PCR:
araC-pBAD-B0031-MazF;
araC-pBAD-B0034-MazF.
04/09/2016--05/09/2016 Sep 4th
1. Bacteria have grown up so we use colony PCR to find the positive results.
2. We use different primers and no positive results.
3. colony PCR :
araC-pBAD-B0034-LacIq-B0015(reverse consequence)-pTac-B0032-MazF-PSB1C3(unsuccessful);
araC-pBAD-B0034-CI-B0015(reverse consequence)-pR-B0032-MazF-PSB1C3(unsuccessful);
araC-pBAD-B0031-HokD(successful);
araC-pBAD-B0034-HokD(successful).
4. Sequenced: araC-pBAD-B0034-TetR-B0015(reverse consequence)-pTet-B0032- MazF-PSB1C3
5. Cultured bacteria into LB media overnight with chloramphenicol:
araC-pBAD-B0031-MazF; araC-pBAD-B0034-MazF;
araC-pBAD-B0031-HokD; araC-pBAD-B0034-HokD;
araC-pBAD-B0034-TetR-B0015(reverse consequence)-pTet-B0032- MazF-PSB1C3(without arabinose).
6. 3A assembly:
araC-pBAD-B0034-LacIq-B0015(reverse consequence) and pTac-B0032-MazF with PSB1C3;
araC-pBAD-B0034-CI-B0015(reverse consequence) and pR-B0032-MazF with PSB1C3.
and then, transformed into TOP 10 and seeded in LB with chloramphenicol.
7. Restriction digest and recovered the gelatin:
EcoR.I&SpeI: araC-pBAD-B0034-LacIq-B0015(reverse consequence)
araC-pBAD-B0034-CI-B0015(reverse consequence)
XbaI&PstI: pTac-B0032-MazF; pR-B0032-MazF, unsuccessful.
06/09/2016 Sep 6th
1. Use colony PCR to find the positive results again. Results: no positive results.
Analysis: we should digest the fragments before we transfer it.
2. Digest the fragments.
3. Use OE PCR to connect two fragments. After electrophoresis we get the right stripe then we use kit recovery the snippets and the last, we verify the snippet. The result was correct.
4. Sequenced:
araC- pBAD -B0031-MazF
araC-pBAD-B0034-MazF
araC-pBAD-B0031-HokD
araC-pBAD-B0034-HokD, successful.
5. 3A assembly:
araC-pBAD-B0034-LacIq-B0015(reverse consequence) and pTac-B0032-MazF with PSB1C3;
araC-pBAD-B0034-CI-B0015(reverse consequence) and pR-B0032-MazF with PSB1C3,and then, transformed into TOP 10 and seeded in LB with chloramphenicol.
6. First round of PCR: For amplification of mutated promoters at 65°/67° with upstream primers JB1, JB2, JB3. [Pfu protocol]
No target fragment [1023bp] observed.
Second round of PCR: For amplification of mutated promoters at 63°/65° with upstream primers JB1, JB2, JB3. Reduce the annealing temperature. [Pfu protocol]
No target fragment [1023bp] observed.
07/09/2016 Sep 7th
1. The liquid culture medium has been contaminated. We can’t transfer our bacteria. We make up culture medium and sterilize.
2. When the culture medium was ready. We transfer our bacteria and culture it overnight.
3. olony PCR :
araC-pBAD-B0034-LacIq-B0015(reverse consequence)-pTac-B0032-MazF-PSB1C3;
araC-pBAD-B0034-CI-B0015(reverse consequence)-pR-B0032-MazF-PSB1C3,
successful.
4. Cultured bacteria into LB media overnight with chloramphenicol :
araC-pBAD-B0034-LacIq-B0015(reverse consequence)-pTac-B0032-MazF-PSB1C3;
araC-pBAD-B0034-CI-B0015(reverse consequence)-pR-B0032-MazF-PSB1C3,
but invaded by other bacteria.
08/09/2016 Sep 8th
1. Culture the bacteria into another culture medium for 2.5 hours.
2. Make the competent cell. Transfer our fragments into competent cell.
3. They were spread on LB agar plates with CHL on at 37℃.
4. The sequencing results came out. Almost right except some base mutated.
5. Culture TA colony bacteria into liquid culture medium.
6. Explanation: The bacterium was transferred to the inhibitor group for texting.
7. Restriction digest:
XbaI&PstI: pTac-B0032-MazF; pR-B0032-MazF, successful.
8. PCR for amplification:
pTac-B0032-MazF;
pR-B0032-MazF, successful.
9. Replace the PCR machine/the template, and do it again.
ddH2O
15ul
Buffer
2.5ul
dNTP
2ul
template
1ul
Upstream primer
2ul
Downstream primer
2ul
Enzyme(Pfu)
0.5ul
No target fragment observed.
Analyze the reason, plan to replace the polymerase enzyme and increase the dNTPs.
09/09/2016 Sep 9th
1. Use colony PCR to find the positive results.
No positive results.
Analysis: fragments concentration was too low.
2. Make TA colony bacteria into competent cell. The plasmid was constructing. But the cell concentration was too low. We will do it again next day.
3. Add TA colony bacteria and DH5αcell into liquid culture medium.
4. 3A assembly:
araC-pBAD-B0034-LacIq-B0015(reverse consequence) and pTac-B0032-MazF with PSB1C3;
araC-pBA araC-pBAD-B0034-CI-B0015(reverse consequence) and pR-B0032-MazF with PSB1C3,
araC-pBAD-B0034-TetR-B0015(reverse consequence) and pTet-B0032-MazF with PSB1C3,
and then, transformed into TOP 10 and seeded in LB with chloramphenicol.
5. PCR was carried out on [pTet+B0034+RFP+B0015] circuit using Fast Pfu Fly Program. Extension time was set to 50s and annealing temperature was 67°.
ddH2O
15ul
Buffer
2.5ul
dNTP
2ul
template
1ul
Upstream primer
2ul
Downstream primer
2ul
Enzyme(Pfu)
0.5ul
Target Fragment observed through Gel electrophoresis.
Purification with kit from TIANGEN Biotech.
Digestion and ligation:
6. Restriction digestion of JP1/JP2/JP3/pSB1C3 with EcoRI and PstIat 37℃ for 2h20min. 80°C water bath for 20min to stop the reaction.
7. Ligation of JP1/JP2/JP3 with pSB1C3 by T4 ligase.
10/09/2016 Sep 10th
1. Culture them again. Use Arabinose to induce DH5αbacteria.
2. Make TA colony bacteria and DH5αbacteria into competent cell.
3. Put TA colony bacteria competent cell in the -80℃.
4. Transfer our fragments into competent cell.
5. They were spread on LB agar plates with CHL on at 37℃.
6. colony PCR :
araC-pBAD-B0034-LacIq-B0015(reverse consequence)-pTac-B0032-MazF-PSB1C3,unsuccessful;
araC-pBAD-B0034-CI-B0015(reverse
consequence)-pR-B0032-MazF-PSB1C3,unsuccessful;
araC-pBAD-B0034-TetR-B0015(reverse consequence)-pTet-B0032-MazF-PSB1C3, successful.
7. Cultured bacteria into LB media overnight with chloramphenicol:
araC-pBAD-B0034-TetR-B0015(reverse consequence)-pTet-B0032-MazF-PSB1C3,
araC- pBAD -B0031-MazF;
araC- pBAD -B0032-MazF;
araC-pBAD-B0034-MazF; and PSB1C3.
8. Restriction digestion of JP1/JP2/JP3 with EcoRI and PstIat 37℃ for 2h20min.  Purification of Restriction digest production by running agarose gel electrophoresis. Extract the right the right brand by TIANgel Midi Purification Kit.
9. Restriction digestion of pSB1C3 with EcoRI and PstIat 37℃ for 2h20min. urification of Restriction digest production by running agarose gel electrophoresis. Extract the right the right brand by TIANgel Midi Purification Kit.
10. Ligation of JP1/JP2/JP3 with pSB1C3 by T4 ligase.
11. Ligation products JP1/JP2/JP3 were transformed into TOP 10  and incubated in LB plates added Chlorine.
11/09/2016 Sep 11st
1. Use colony PCR to find the positive result.
2. Enrich fragments to improve concentration. After electrophoresis we don’t get the stripe.
Analysis: Pfu enzyme efficiency was too low.
3. Culture DH5αcell into liquid culture medium.
4. Make DH5αcell into competent cell.
5. Transfer our three fragments into DH5α competent cell.
6. They were spread on LB agar plates with CHL on at 37℃.
7. Culture it overnight.
8. 11 Measured growth curve(induced and uninduced)：
araC- pBAD -B0031-MazF;
araC- pBAD -B0032-MazF;
araC-pBAD-B0034-MazF;
PSB1C3.
9. We Used Colony PCR to check whether the ligation worked as expected. For amplification of mutated promoters at 55℃ with Single colony.
Upstream primer
0.5ul
Downstream primer
0.5ul
MIX
7.5ul
ddH2O
6.5ul
Target Fragment observed through Gel electrophoresis. Then we Incubated the bacteria in LB medium with chloramphenicol.
10. Send the plasmids for sequencing in order to confirm the parts.
12/09/2016 Sep 12nd
1. Use colony PCR to find the positive result. And we succeed.
2. The positive results were sent   to sequencing.
3. Cultured bacteria into LB media overnight with chloramphenicol:
araC-pBAD-B0031-HokD;
araC-pBAD-B0032-HokD;
araC-pBAD-B0034-HokD;
PSB1C3.
4． Colony PCR: select the appropriate restructuring bacteria from Sept 11. contained plasmids. Only
5． araC-pBad-CI-B0015-PR-B0032-RFP-B0015 and araC-pBad--B0034-Laciq-B0015-pLac-b0032-YFP-B0015 had positive results. And incubate NO.24(laciq) and NO.32(CI) in LB with kanamycin.
13/09/2016 Sep 13rd
1.   Measured growth curve(induced and uninduced)：
araC-pBAD-B0031-HokD;
araC-pBAD-B0032-HokD;
araC-pBAD-B0034-HokD;
PSB1C3.
2. Ligation products (araC-pBad-tetR-B0015-Ptet-B0032-RFP-B0015, J23119-B0032-CI-B0015,
J23119-B0032-tetR-B0015,
J23119-B0034-Laciq-B0015,)
were transformed into TOP 10 and seeded/incubated in LB plates added chloramphenicol.
3. Extract the plasmids:
araC-pBad-CI-B0015-PR-B0032-RFP-B0015 araC-pBad--B0034-Laciq-B0015-pLac-b0032-YFP-B0015
incubated on 9.12.
4. Incubate araC-pBad-CI-B0015-PR-B0032-RFP-B0015 and araC-pBad--B0034-Laciq-B0015-pLac-B0032-YFP-B0015 into LB follow the form:
Incubate overnight.
14/09/2016 Sep 14th
1. Get the PCP20 plasmid. Next step was to transfer this plasmid into the fragment that we already sequencing.
2. Use the TIANprep plasmid Kit to extract plasmid then we verify the plasmid. It was correct.
3. Design the experiment to eliminate the CHL resistance and the PCP20 plasmids.
4. Transer the DHαcell what including our aim fragment and PKD46 plasmid in the culture medium at 37℃ for 2 hours and 42℃ overnight. In order to eliminate the PKD46 plasmid, reduce the burden of bacteria.
5. The plates did not grow except J23119-B0034-Laciq-B0015(only one).
6. OE-PCR: connect the araC-pBad with tetR-B0015 again. fast PFU fly program. And the PCR results were positive. And we purified the PCR production by TIANgel Midi Purification Kit.
Digestion the PCR production, vector(pSB1K3) by EcoRI and PstI and ligate. And we transformed ligation production into TOP 10.
7. Group A and group B incubated last night did not show any differences incubated last night. The same result as C and D. But the control, group E incubated pSB1C3, did not grow and there were some deposition in LB medium.
After 17 hours, group A and group B showed a good result. Group B showed stronger fluorescence. So we incubated pSB1K3 and araC-pBad-CI-B0015-PR-B0032-RFP-B0015 (reverse) for further test.
15/09/2016 Sep 15th
1. Streak of single colony.
2. Use the TIANprep plasmid Kit to extract plasmid. Run PCR amplification of this plasmid. The plasmid was already eliminated.
3. Continue verification.
4. We tested the promoter strength.
5. Test: araC-pBad-CI-B0015-PR-B0032-RFP-B0015 (reverse)
6. Colony PCR: araC-pBad-tetR-B0015 had a positive result. And we incubated it in LB.
16/09/2016 Sep 16th
1. Use colony PCR to verify the PKD46 plasmid.
Still have some plasmids did not eliminated.
2. Culture the bacteria in LB medium
Cultured bacteria into LB media overnight with chloramphenicol:
araC- pBAD -B0031-MazF;
araC- pBAD -B0032-MazF;
araC-pBAD-B0034-MazF;
PSB1C3.
3. first round of PCR: For amplification of mutated promoters at 67℃ with upstream primers SJTB. [Pfu protocol]
No target fragment [1023bp] observed.
Second round of PCR: For amplification of mutated promoters at 67℃ with upstream primers SJTB .                                             Reduce the annealing temperature. [Pfu protocol]
4. Target Fragment observed through Gel electrophoresis.
4. Purification with kit from TIANGEN Biotech.                                             5. OE-PCR:
build the
J23119-B0034-Laciq-B0015
J23119-B0032-tetR-B0015 (reverse)
And the PCR production was purified by TIANgel Midi Purification Kit.
6. Digestion:
J23119-B0032-Laciq-B0015 (reverse)
J23119-B0034-tetR-B0015 (reverse)
J23119-B0032-CI-B0015 (reverse)
J23119-B0034-CI-B0015 (reverse)
And ligate these with vector, pSB1K3. After 1.5 hour, we transformed ligation production into TOP 10.
17/09/2016 Sep 17th
1. Make competent cell and transfer the PCP20 plasmid.
2. Incubate the bacteria in LB medium with AMP.
3. Culture it overnight.
4. Measured growth curve(induced and uninduced)：
araC- pBAD -B0031-MazF;
araC- pBAD -B0032-MazF;
5. Colony PCR:
J23119-B0032-Laciq-B0015 (reverse)
J23119-B0034-tetR-B0015 (reverse)
J23119-B0032-CI-B0015 (reverse)
J23119-B0034-CI-B0015 (reverse)
All results were negative.
6. Digestion:
J23119-B0034-Laciq-B0015
J23119-B0032-tetR-B0015 (reverse)
And ligate these with vector, pSB1K3. After 2 hours ,we transformed ligation production into TOP 10.
18/09/2016 Sep 18th
1. The concentration of the competent cell was too high, so the bacteria grow all the AMP plates.
2. We do it again and culture it overnight.
3. We use Fluorescence microscope to observe the bacteria and find they all emit red fluorescence. But two of them supposed to emit yellow fluorescence. We decide to limit the gene circuit again.
PCR was carried out on plac+Yfp ,PR+mRFP, PR+Yfp using fastpfu fly PCR program.
4. Colony PCR:
J23119-B0034-Laciq-B0015
J23119-B0032-tetR-B0015 (reverse)
The all results were negative.
5. Test the araC-pBad-CI-B0015-PR-B0032-RFP-B0015 again. But we found our sample was polluted by other bacteria. So we tested the arabinose (only add arabinose into LB ).
6. We got new parts J23109, J23116 and J23106 from iGEM parts library. And transform them into TOP 10. And we also got new vector pSB3K3, pSB4K4 at the same time.
19/09/2016 Sep 19th
1. Use OE PCR to link three fragments with CHL resistance gene.
2. After electrophoresis we get the right stripe then we use kit recovery the snippets and the last, we verify the snippet. The result was correct.
3. Make competent cell and transfer the fragments.
4. Incubate the bacteria in LB medium with CHL.
5. Culture it overnight.
6. Digestion and ligation:
Restriction digestion of SJTB with EcoRI and PstI at 37℃ for 2h20min. Purification of Restriction digest production by running agarose gel electrophoresis. Extract the right the right brand by TIANgel Midi Purification Kit.
Ligation of SJTB with pSB1C3 by T4 ligase.
Ligation products SJTB were transformed into TOP 10 and seeded/incubated in LB plates added Cholrine.
7. We Used Colony PCR to check whether the ligation worked as expected. For amplification of mutated promoters at 55℃ with Single colony.
8. Target Fragment observed through Gel electrophoresis. Then we incubated the bacteria in LB medium with chloramphenicol.
9. Some fungi had grown in the LB which was added only arabinose on 9.18. So we can confirm that the arabinose was the origin of pollution.
10. PCR:
J23119-B0032-tetR-B0015 (reverse)
J23119-B0032-Laciq-B0015 (reverse)
J23119-B0034-tetR-B0015 (reverse)
J23119-B0032-CI-B0015 (reverse)
J23119-B0034-CI-B0015 (reverse)
The results were all negative. The possible reason was that we set the annealing temperature was not high enough.
11. OE-PCR:
araC-pBad and tetR-B0015-Ptet-B0032-RFP-B0015
J23119 and B0034-tetR-B0015
The gel results were all positive.
20/09/2016 Sep 20th
1. Test the araC-pBad-RFP-B0015 to simulate the inhibitor.
2. PCR:
J23119-B0032-tetR-B0015 (reverse)
J23119-B0032-Laciq-B0015 (reverse)
J23119-B0034-tetR-B0015 (reverse)
J23119-B0032-CI-B0015 (reverse)
J23119-B0034-CI-B0015 (reverse)
J23119-B0034-Laciq-B0015
araC-pBad-tetR-B0015
The results were all positive.
So we digested all PCR production by EcoRI and PstI. And ligate them with vector, pSB1K3, overnight.
21/09/2016 Sep 21st
1. Transformation:
J23119-B0032-tetR-B0015 (reverse)
J23119-B0032-Laciq-B0015 (reverse)
J23119-B0034-tetR-B0015 (reverse)
J23119-B0032-CI-B0015 (reverse)
J23119-B0034-CI-B0015 (reverse)
J23119-B0034-Laciq-B0015
araC-pBad-tetR-B0015
into TOP 10.
22/09/2016 Sep 22nd
1. Colony PCR:
The results of transformation made on 9.20 were negative.
23/09/2016 Sep 23rd
1. Make competent cell. Transfer these three fragments   (PR-B0034-Yfp-B0010+B0012+cat
PR-B0032-mRFP-B0010+B0012+cat
Plac-B0034-eYfp-B0010+B0012+cat)
into DH5αcell.
2. They were spread on LB agar plates with CHL on at 37℃.
2. Bacteria have grown up so we use colony PCR to find the positive results.
3. Succeed.
4. Send to sequencing.
Result:PR+mRFP+cat was correct.
5. PCR was carried out on plac+Yfp+cat,PR+mRFP+cat,PR+Yfp+cat and Ptet+mRFP+cat using Primer Star PCR program.
7. After electrophoresis we get the right stripe then we use kit recovery the snippets and the last, we verify the snippet. The result was correct.
8. Colony PCR:
The results of transformation made on 9.20 were negative.
9. 3A assembly (9.22-9.23):
J23109-B0032-tetR-B0015
J23109-B0034-tetR-B0015
J23109-B0032-CI-B0015
J23109-B0034-CI-B0015
J23116-B0032-tetR-B0015
J23116-B0034-tetR-B0015
J23116-B0032-CI-B0015
-B0034-CI-B0015
J23106-B0032-tetR-B0015
J23106-B0034-tetR-B0015
J23106-B0032-CI-B0015
J23106-B0034-CI-B0015
And ligate the all gene circuits with pSB1K3 (high copy), pSB3K3 (medium copy) and pSB4K5 (low copy).
Transform the all gene circuit into TOP 10.
1. No bacteria grow in the LB culture medium
2. The sequence results were wrong. We decided to do it again.
3. Us OE PCR to enrich the fragments.
4. Culture the DH5αand TOP 10 bacteria in the LB culture medium.
5. araC-pBAD-B0034-CI-B0015 (reverse consequence) and pR-B0032-MazF with PSB1C3,     successful.
Send the plasmids for sequencing in order to confirm the parts.
24/09/2016 Sep 24th
1. Restriction digestion of plac+Yfp,PR+mRFP,PR+Yfp and Ptet+mRFP with EcoⅠand SpeⅠat 37℃ for 3 hours.
2. Then we use kit recovery the snippets and the last, we verify the snippet.
3. Ligate of plac+Yfp,PR+mRFP,PR+Yfp and Ptet+mRFP with PSB1C3 by T4 ligase. Transfer it in the TOP 10 cell.
4. Measured growth curve(induced and uninduced)：
araC-pBAD-B0034-HokD;
PSB1C3.
5. Test the araC-pBad-CI-B0015-PR-B0032-RFP-B0015 again for confirming the system’s function. (9.24-9.25)
6. Colony PCR (9.24):
The all results were positive except
J23116-B0032-CI-B0015 (pSB1K3)
J23106-B0034-tetR-B0015 (pSB3K3)
J23116-B0034-tetR-B0015 (pSB4K5)
And we extract the plasmids from bacteria whose results were right .( 9.25)
7. OE-PCR: restructure the gene circuit araC-pBad-tetR-B0015.(9.25-9.26)
25/09/2016 Sep 25th
1. Use colony PCR to find the positive results. Succeed.
2. This was inhibitor team’s circuit.
We were concerned that the strong promoter will affect the expression of toxin protein, so we would reverse the J23119 promoter. This was the reverse circuit.
3. PCR was carried out on this circuit using Primer Star PCR program. After electrophoresis we get the right stripe then we use kit recovery the snippets and the last, we verify the snippet. The result was correct.
4. Transfer it into TOP 10 cell.
5. Not succeed.
6. 3A assembly: araC-pBAD-B0034-lacIq-B0015(reverse consequence) and pTac-B0032-MazF    with PSB1C3,
unsuccessful.
7. For amplification of mutated promoters at 67℃ with upstream primers SJTB . Reduce the annealing temperature. [Pfu protocol]
Target Fragment observed through Gel electrophoresis.
8. Purification with kit from TIANGEN Biotech.
9. Digestion and ligation:
Restriction digestion of SJTB with EcoRI and PstIat 37℃ for 2h20min. Purification of Restriction digest production by running agarose gel electrophoresis. Extract the right the right brand by TIANgel Midi Purification Kit.
10. Ligation of SJTB with pSB1C3 by T4 ligase.
11. Ligation products SJTB were transformed into TOP 10  and seeded/incubated in LB plates added Cholrine.
12. Transformation :
J23116-B0032-CI-B0015 (PSB1K3)
J23106-B0034-tetR-B0015 (PSB3K3)
araC-pBad-tetR-B0015 (after digestion and ligation)
Into TOP 10.
27/09/2016 Sep 27th
1. Restricition digestion of J23119-B0032/34-tetR/CⅠ-B0015 with ECORⅠ and PSTⅠat 37℃ for 6 hours.
2. Use kit to recovery it.
3. Ligation of J23119-B0032/34-tetR/CⅠ-B0015 with PSB1K3 by T4 ligase.
4. Transfer it into TOP 10 cell. Incubate the bacteria in LB medium with Kan.
5. Not succeed.
DO it again.
Not succeed.
6. Measured growth curve(induced and uninduced)：
araC-pBAD-B0034-HokD;
PSB1C3,.
7. 3A assembly:
J23119-B0034-CI-B0015(reverse consequence) and pR-B0032-MazF with PSB1C3,
J23119-B0034-TetR-B0015(reverse consequence) and pTet-B0032-MazF with PSB1C3,
J23119-B0032-CI-B0015(reverse consequence) and pR-B0032-MazF with PSB1C3,
J23119-B0032-TetR-B0015(reverse consequence) and pTet-B0032-MazF with PSB1C3,
and then, transformed into TOP 10 and seeded in LB with chloramphenicol, unsuccessful.
8. We Used Colony PCR to check whether the ligation worked as expected. For amplification of mutated promoters at 55℃ with Single colony.
Target Fragment observed through Gel electrophoresis.
Negative results.
9. Colony PCR:
J23106-B0034-tetR-B0015
araC-pBad-tetR-B0015
The results were positive.
10. But the 1-11 didn’t grow.
28/09/2016—02/10/2016 Sep 28th
1. We got these four fragments from PSB1K3 by restriction digestion, using EcoⅠand SpeⅠ.
J23119-B0032-tetR-B0015
J23119-B0034-tetR-B0015
J23119-B0032-CⅠ-B0015
J23119-B0034-CⅠ-B0015
We integrate each of these four fragments into three different copy number plasmids: 1K3, 3K3 and 4K5 separately.
2. Restricition digestion of J23119-B0032-tetR-B0015,J23119-B0034-tetR-B0015,J23119-B0032-CI-B0015 and                                             J23119-B0034-CⅠ-B0015 with ECORⅠ and PSTⅠat 37℃ for 6 hours.
3. Use kit to recovery it.
4. Ligation of J23119-B0032-tetR-B0015,J23119-B0034-tetR-B0015,J23119-B0032-cⅠ-B0015 and                                             J23119-B0034-CI-B0015 with PSB1K3,PSB3K3 and PSB4k5 by T4 ligase.
5. Transfer them into TOP 10 and incubated in LB plates add KAN.
7. Use colony PCR to verify it. Correct.
8. Send to sequence.
9. Restriction digestion of SJTB with EcoRI and PstIat 37℃ for 2h20min. Purification of Restriction digest production by running agarose gel electrophoresis. Extract the right the right brand by TIANgel Midi Purification Kit.
Ligation of SJTB with pSB1C3 by T4 ligase.
Ligation products SJTB were transformed into TOP 10  and seeded/incubated in LB plates added Cholrine.
10. Test the gene circuit:
araC-pBad-B0034-RFP-B0015
fluorescence measurement
11. Construction:
B0015- araC-pBad-tetR-B0015( reverse)
The forward B0015 was designed to avoid the promoter which expressed  araC  impacted the downstream genes.
And transform this gene circuit into TOP 10.
October [Expand]
we finished the final testing and got good results.
02/10/2016--03/10/2016 Oct 2nd-3rd
1. Make competent cell and transfer J23119-B0032-tetR-B0015 and J23119-B0034-tetR-B0015 into DH5αcell which contain Ptet-B0032-mRFP-B0015 fragment, transfer J23119-B0032-CⅠ-B0015 and J23119-B0034-CⅠ-B0015 into DH5αcell which contain PR-B0032-mRFP-B0015 fragment.
2. There were too many bacteria grow in the KAN plate, so we make plate streaking.
3. Measured growth curve(induced and uninduced)：
araC-pBAD-B0031-HokD;
araC-pBAD-B0032-HokD;
araC-pBAD-B0034-HokD;
PSB1C3.
4. We Used Colony PCR to check whether the ligation worked as expected. For amplification of mutated promoters at 55℃ with Single colony.
Upstream primer
0.5ul
Downstream primer
0.5ul
MIX
7.5ul
ddH2O
6.5ul
Target Fragment observed through Gel electrophoresis. Then we Incubated the bacteria in LB medium with chloramphenicol.
5. Send the plasmids for sequencing in order to confirm the parts.
6. Colony PCR:
B0015-tetR-pBad-araC-B0015-PR-B0034-RFP-B0015
The results were negative. So we decided to repeat transformation again.
7. Colony PCR:
B0015-tetR-pBad-araC-B0015-PR-B0034-RFP-B0015
The results were positive! The number 2, 5, 6 bacteria were the positive!
4/10/2016 Oct 4th
1. Use colony PCR to verify it. Not succeed.
2. We culture our competent cell in KAN plate, it grow up. It was strange, because our competent cell don’t have KAN resistance. We culture other glycerol bacteria in KAN plate, in order to see if they have KAN resistance.
3. Test the gene circuit araC-pBad-RFP-B0015.
Measure the fluorescence strength.
05/10/2016 Oct 5th
1. The glycerol bacteria grow in the KAN plate, we need to change the host cell and do it over again.
2. PCR was carried out on plac+Yfp+cat,PR+mRFP+cat,PR+Yfp+cat and Ptet+mRFP+cat using Primer Star PCR program. These fragment was use to recombine.
In the mean time, PCR was carried out on plac+Yfp+cat,PR+mRFP+cat,PR+Yfp+cat and Ptet+mRFP+cat using Primer Star PCR program. These fragments were biobricks.
3. After electrophoresis we get the right stripe then we use kit recovery the snippets and the last, we verify the snippet. The result was correct
4. Restriction digestion of plac+Yfp+cat,PR+mRFP+cat,PR+Yfp+cat ,Ptet+mRFP+cat with DPNⅠat 37℃ for 4 hours. Restriction digestion of plac+Yfp+cat,PR+mRFP+cat,PR+Yfp+cat ,Ptet+mRFP+cat  with  EcoRⅠand PstⅠat 37℃ for 4 hours.
5. Use kit to recovery all the fragments.
6. Ligation of biobricks with PSB1C3 by T4 ligase.
7. Transfer PKD46 plasmids into DH5αcompetent cell.
8. Use colony PCR to verify it.
Correct.
9. SDS-PAGE: test the Jilin University’s device.
10. Digestion:
B0015-tetR-pBad-araC-B0015-PR-B0034-RFP-B0015 and
B0015-tetR-pBad-araC-PR-B0034-RFP-B0015 together.
And the result was positive. We have added the B0015 successfully .
11. Ligation(standardization):
J23106-B0032-tetR-B0015
J23106-B0034-tetR-B0015
J23116-B0032-tetR-B0015
J23116-B0034-tetR-B0015
J23106-B0034-cI-B0015
J23109-B0032-cI-B0015
J23109-B0034-cI-B0015
J23116-B0034-cI-B0015
Ligate with pSB1C3. And transform into TOP 10.
06/10/2016 Oct 6th
1. Transfer the products of ligation into TOP 10 and incubated in LB plates add CHL.
2. Incubate the DH5α in LB medium with CHL. Culture it overnight.
3. Fluorescence measurement:
B0015-cI-pBad-araC-B0015-PR-B0034-RFP-B0015
3. Standardization:
PR-B0032-RFP-B0015
araC-pBad-B0034-tetR-B0015
araC-pBad-B0034-LacIq-B0015
B0015--LacIq- B0034-pBad- araC-pLac-RFP-B0015
B0015-cI-pBad-araC-B0015
B0015-cI-pBad-araC-B0015-PR-B0034-RFP-B0015
Ligate with pSB1C3 and transform into TOP 10.
4. Colony PCR:
J23106-B0032-tetR-B0015
J23116-B0032-tetR-B0015
J23116-B0034-tetR-B0015
J23106-B0034-cI-B0015
J23109-B0034-cI-B0015
J23116-B0034-cI-B0015
The result were positive, but there were two gene circuit with negative result:
J23106-B0034-tetR-B0015
J23109-B0032-cI-B0015
07/10/2016 Oct 7th
1. Incubate in a new LB medium.
2. Make competent cell and transfer PR+mRFP+cat, Ptet+mRFP+cat into competent cell.
3. Incubated in LB plates add CHL.
4. Incubate the DH5α in LB medium with CHL. Culture it overnight.
5. For amplification of mutated promoters at 67℃ with upstream primers SJTB.
ddH2O
12ul
Buffer
5ul
dNTP
3ul
Template
1ul
Upstream primer
2ul
Downstream primer
2ul
Enzyme(Easy Pfu )
1ul
6. Purification with kit from TIANGEN Biotech.
Digestion and ligation:
Restriction digestion of SJTB with EcoRI and PstIat 37℃ for 2h20min. Purification of Restriction digest production by running agarose gel electrophoresis. Extract the right the right brand by TIANgel Midi Purification Kit.
Ligation of SJTB with pSB1C3 by T4 ligase.
7. PCR:
Construct the gene circuit:
B0015-tetR-B0034-pBad-araC-B0015
And connect with B0015 by 3A assembly after digestion. Then transform into TOP 10.
8. Colony PCR:
PR-B0032-RFP-B0015
araC-pBad-B0034-tetR-B0015
araC-pBad-B0034-LacIq-B0015
J23106-B0034-tetR-B0015
J23109-B0032-cI-B0015
All were positive .
8/10/2016 Oct 8th
1. There was no bacteria grow in the CHL plates.
2. Incubate in a new LB medium.
3. Make competent cell and transfer PR+mRFP+cat, Ptet+mRFP+cat into competent cell.
4. Incubated in LB plates add CHL.
5. Incubate the DH5α in LB medium with CHL. Culture it overnight.
6. Fluorescence measurement again:
B0015-cI-pBad-araC-B0015-PR-B0034-RFP-B0015
7. Colony PCR:
B0015-tetR-B0034-pBad-araC-B0015
The result was positive.
B0015-cI-B0034-pBad-araC
B0015--LacIq- B0034-pBad- araC-pLac-RFP-B0015
The result were negative .
09/10/2016 Oct 9th
1. There was no bacteria grow in the CHL plates.
2. Incubate in a new LB medium.
3. Make competent cell and transfer PR+mRFP+cat, Ptet+mRFP+cat into competent cell.
4. Incubated in LB plates add CHL.
5. Construction:
B0015-tetR-B0034-pBad-araC-B0015-pTet-RFP-B0015
After digestion and ligation, transform into TOP 10.
10/10/2016 Oct 10th
1. There was no bacteria grow in the CHL plates.
2. Ligation of J23119-B0032/B0034-CⅠ/tetR+B0015 with PSB1C3 by T4 ligase.
3. Transfer them into TOP 10 and incubate in LB medium with CHL.
4. Standardization:
J23109-B0032-tetR-B0015
J23106-B0032-cI-B0015
J23116-B0032-cI-B0015
5. Colony PCR:
B0015-tetR-B0034-pBad-araC-B0015-pTet-RFP-B0015
The result was negative.
11/10/2016 Oct 11st
1. Digestion:
B0015-cI-pBad-araC-B0015-PR-B0034-RFP-B0015
And ligate with pSB3K3, pSB4K5 for reconfirm our system’s function.
2.Colony PCR:
J23109-B0032-tetR-B0015
J23106-B0032-cI-B0015
J23116-B0032-cI-B0015
All were positive.
Breakthroughs
Project Idea
Validation Circuits
Growth Curves
Build Circuits
Protocol
P-SLACKiller
· Work or Die
· Monitor Plasmid Numbers
· Kill the Slacker
· Maintain the Better Biosystem
Useful Links
Achievements
Results
Design
Proof
HP
Sponsors
Get in touch
Email: iGEM_BIT_China@163.com
Location: No.5 Zhongguancun South Street
BIT-China © Copyright 2016
Powered by Bootstrap.
