<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT-China_Promoter skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT-China/Promoter</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="col-sm-12" id="content-wrapper" style="padding: 0;margin: 0;"><DIV class="header clearfix" id="header" style="width: 100%;position: relative;padding-bottom: 20px;background-size:cover;
                                background-image: url(https://static.igem.org/mediawiki/2016/9/9a/T--BIT-China--img--top_bg.png);"><DIV class="col-sm-12 clearfix " style="padding: 0;height: 106px;"><DIV class="header-container clearfix home-img" id="header-container" style="margin-top: 20px;z-index: 10"><DIV class="header-navigation col-sm-11"><UL class="pull-right"><LI><A href="https://2016.igem.org/Team:BIT-China/Medals">Medal fullfillments</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Parts">Parts Submission</A></LI></UL><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Description">Description</A></LI><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Design">Design</A></LI><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Results">Results</A></LI><LI class="tabdown-submenu"><A href="https://2016.igem.org/Team:BIT-China/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Notebook">Lab Book</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/E_M">Equipments &amp; Materials</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Team/Gallery">Gallery</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Team/Gaint_Jamboree">Gaint Jamboree</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/HP/Gold">HP_Gold</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/HP/Silver">HP_Silver</A></LI></DIV></DIV></DIV></DIV></DIV><DIV class="col-sm-12 clearfix" style="padding: 0;padding-top:30px;
background: url(https://static.igem.org/mediawiki/2016/4/45/T--BIT-China--content_bg.jpg)"><DIV class="content-right" style="float: left;width:100%;padding: 10px;"><DIV class="block-border-outer" id="content-right-border"><DIV class="overview-content clearfix"><DIV class="content-title col-sm-12">
                    Promoter mutation
                </DIV><DIV id="background" class="block-title">Background</DIV><DIV class="block-content"><DIV class="block-content-item"><DIV class="block-content-item-block"><DIV>
                                The regulation of gene expression is always the key point for most scientists to study. Traditionally, the functional gene is recognized to be controlled by the transcriptional promoter. And the promoter DNA interacting with RNA polymerase during the initiation of transcription has been extensively researched over recent decades.  Among the defined findings, the binding affinity and the open rate of complex formation are two key factors governing the promoter strength.
                                There are two regions predicting to play essential roles in prokaryotic transcription initiation, which are located at -10 and -35 regions. According to the specific effects of these two conserved sequence, plenty of mutations were built to investigate. It is important to make the mutational analysis to establish library with varying strengths promoters and to make the basic biological research easier.
                            </DIV></DIV></DIV></DIV><DIV id="mechanism" class="block-title">Mechanism</DIV><DIV class="block-content"><DIV class="block-content-item"><DIV class="block-content-item-block"><DIV>
                                As the first step of gene expression in prokaryote, the promoter is the DNA sequence affecting the frequency and location of transcription initiation through interaction with RNA polymerase holoenzyme. Two conserved regions about the -35 region and the -10 region were regarded as carriers containing those properties.
                                －35 Region:
                                During the transcriptional initiation, RNA polymerase and its associated Sigma factor will recognize this element. The RNA polymerase attaches itself to the template DNA strand and proceeds to slide the entire strand from 3’ to 5’ direction. Defined by sequence data, the overall strength of a promoter is important while various mutational analysis showed this region is involved in the initial binding affinity of holoenzyme. For a mutational analysis, kinds of strength revealed promoters have been constructed by replace each base. Higher-strength the natural promoter is, the wider range of efficient hybrid promoter will be get. Considered there is 6 bases of a group comprised in -35 region, 4096 mutations are the upper limit.
                                Base substitutions at position —31 gave only a little effect; base substitution sat position -32 caused significant reduction independent of which base replaced cytosine; base substitutions at position —35 also led to reduction, although the effects were somewhat smaller than those of -32 base substitutions; the effects of base substitutions at position -33, -34 and -36 were variable depending on the actual base introduced. The rate of open complex formation (parameter II) was slower for most variant promoters than for the wild-type promoter, except for the 34G (consensus) and the 33G promoters. Promoters containing altered bases at position —31 were less affected.
                                </DIV><DIV>-10 Region:
                                Similar to -35 region of a promoter, -10 region is another conserved region. Generally, it is involved in DNA melting efficiency—the rate of open promoter complex formation.
                                The distance between the -35 and -10 regions:
                                For about 92% of promoters had inter-region spacing of 17bp, plus or minis 1bp. The length of it also plays a role in promoter activity and strength.
                            </DIV></DIV></DIV></DIV><DIV id="application" class="block-title">Application in our project</DIV><DIV class="block-content"><DIV class="block-content-item"><DIV class="block-content-item-block"><DIV>
                                pTet: tccctatcagtgatagaga ttgaca tccctatcagtgatagagatactgagcac
                                In our basic circuit, we aim to find out the relationship between plasmid numbers and inhibitor concentration. However, when plasmids lost, inhibitor drops to a threshold, and resulted in the in-promoter is enough to express killer gene. So what if we want to kill the “slacker” by using different thresholds after well-constructing our circuits? Then we come to devote us to mutation in-promoters context.
                                Alteration or removal of -35 region sequences of the tet promoter can greatly affect in vivo promoter activity，by targeting the cause, we choose to mutate pTet to set promoters with varying strength. Amongst all the base pairs, the natural promoter was coincident with Darwinist survival of the fittest, it’s because of homological regions.
    Promoter mutations construction described shows the function of each base, especially those in conserved regions and played roles in gene expression. On the other hand, promoters like pTac have been widely discussed, pTac and pTet DNA sequences are bending to RNA polymerase with same sigma factor. For our goal of mutating pTet, we decide to mutate it site-directed firstly refers to the results of pTac mutations.
                                Secondly, we will create new regions to attempt at changing the strength, adjacent to -35 region and -10 region, or altering the spacing length.
                                However, these two steps are based on our rational design, in fact, we also need to use error-prone pcr to explore if there is somewhere new could affect the gene expression.
                            </DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="nav-left" id="nav_left_wrapper" style="float: left;margin-left:-100%;width: 200px;"><DIV class="nav-left-title" style="position:relative;width: 200px;padding: 0;"><SPAN style="color: #FFFFFF;font-size: 20px;font-weight: bold;position: absolute;
        right: 40px;bottom: 20px;">Inhibitor</SPAN></DIV><UL class="nav_left_ul" style="padding-left: 50px;"><LI><A href="#background" class="nav_point"><SPAN class="nav_left_txt">Background</SPAN></A></LI><LI><A href="#mechanism" class="nav_point"><SPAN class="nav_left_txt">Mechanism</SPAN></A></LI><LI><A href="#application" class="nav_point"><SPAN class="nav_left_txt">Application</SPAN></A></LI></UL></DIV></DIV><DIV class="col-sm-12" style="padding: 0;"><FOOTER class="footer dark"><DIV class="footer-widgets md-padding-top"><DIV class="container"><DIV class="row"><DIV class="col-sm-3"><DIV class="widget widget-about"><H4 class="widget-title" style="margin-bottom: 43px">P-SLACKiller</H4><P><B>· </B>Work or Die<B>· </B>Monitor Plasmid Numbers<B>· </B>Kill the Slacker<B>· </B>Maintain the Better Biosystem</P></DIV></DIV><DIV class="col-sm-3"><DIV class="widget widget-categories"><H4 class="widget-title" style="margin-bottom: 43px">Useful Links</H4><UL><LI><A href="https://2016.igem.org/Team:BIT-China/Medals" target="_blank">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Results" target="_blank">Results</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Design" target="_blank">Design</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/Proof" target="_blank">Proof</A></LI><LI><A href="https://2016.igem.org/Team:BIT-China/HP/Gold" target="_blank">HP</A></LI></UL></DIV></DIV><DIV class="col-sm-3"><DIV class="widget"><H4 class="widget-title" style="margin-bottom: 43px">Sponsors</H4></DIV></DIV><DIV class="col-sm-3"><DIV class="widget"><H4 class="widget-title" style="margin-bottom: 43px">Get in touch</H4><P><STRONG>Email:</STRONG> iGEM_BIT_China@163.com <STRONG>Location:</STRONG> No.5 Zhongguancun South Street </P></DIV></DIV></DIV></DIV></DIV><DIV class="footer-meta"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><DIV class="copyright">BIT-China © Copyright 2016  Powered by Bootstrap.</DIV></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>