<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SUSTC-Shenzhen-B_protocol1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SUSTC-Shenzhen-B/protocol1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SUSTC-Shenzhen-B/protocol1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen-B/protocol1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen-B/protocol1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SUSTC-Shenzhen-B/protocol1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SUSTC-Shenzhen-B/protocol1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Title</TITLE></P><DIV id="bodychild"><DIV id="outercontainer"><DIV id="outerheader"><HEADER id="top"><SECTION id="navigation"><NAV><UL id="topnav" class="sf-menu"><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B">Home</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/project.overview">Project Overview</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/introduction">Software</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/introduction">Overview</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/algorithm">Algorithm</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/achievements">Results</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/Download">Download</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/comment">Comment</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/Tutorial">Tutorial</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/SBOL">SBOL</A></LI></UL><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/lab introduction">WetLab</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/lab introduction">Overview</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/protocol1">Protocol</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/lab results">Lab Results</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/lab.sbol">SBOL Document</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/RFC">Technical Standard</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/members">Team</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/members">Members</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/advisers">Advisers</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/photos">Photo Gallery</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/Contributions">Contributions</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/Acknowledgement">Acknowledgement</A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen-B/protocol1&amp;action=edit">Edit</A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen-B/protocol1&amp;action=history">History</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/before.june">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/before.june">Before June</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/july">July</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/august">August</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/September">September</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/hp.intro">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/hp.intro">Overview</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/open.class">Open Classes</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/high.school.visits">High School Visits</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/SynBio.intro">SynBio Intro</A></LI></NAV></SECTION></HEADER></DIV><DIV id="outermain"><DIV id="maincontent"><SECTION id="mainthecontent"><ARTICLE><H1><P>Lab Protocol</P></H1><FONT face="Arial, Helvetica"><P><A href="#Site-Directed_Mutagenesis">1.  Site-Directed Mutagenesis</A></P><P><A href="#Restriction">2.  Mutation Verification by Restriction Enzyme Digestion</A></P><P><A href="#Media">3.  Media Preparation</A></P><P><A href="#Bacterial">4.  Bacterial Transformation</A></P><P><A href="#Colony">5.  Colony PCR for Verification</A></P><P><A href="#Culture">6.  Cultivate the Bacteria</A></P><P><A href="#Plasmid">7.  Plasmid DNA Isolation</A></P><P><A href="#Restriction_2">8.  Mutation Verification by Restriction Enzyme Digestion </A></P><P><A href="#Amplify">9.  Polymerase Chain Reaction and Electrophoresis</A></P><P><A href="#Electrophoresis">10. Double Restriction Enzyme Digestion and  Electrophoresis </A></P><P><A href="#Ligation">11. Ligation</A></P><P><A href="#Bacterial_Transformation">12. Bacterial Transformation</A></P><P><A href="#bacterial_colony">13. Bacterial Colony PCR</A></P><P><A href="#Culture_the">14. Cultivate the Bacteria</A></P><P><A href="#Plasmid_DNA">15. Plasmid DNA Isolation</A></P><P><A href="#Restriction_Enzyme">16. Restriction Enzyme Digestion and Electrophoresis</A></P><P><A href="#Ligation1">17.  Ligation</A></P><P><A href="#Bacteria">18.  Bacteria Transformation</A></P><P><A href="#Cultivate">19.  Cultivate the Bacteria</A></P><P><A href="#Flow">20.  Flow Cytometer Analysis</A></P><P><A href="#Fluorescence">21.  Fluorescence Microscope </A></P><H3><FONT face="Arial, Helvetica"><B><FONT color="#0000FF">Site-Directed Mutagenesis</FONT></B></FONT></H3>

Plasmid pSB1A3 was chosen as a backbone vector for cloning. The Pst I site in pSB1A3 was mutated to Afl II site to facilitate following cloning processes. Proper primers were designed and PCR-based site-directed mutageneis were carried out to generate this mutation as descbibed below.  <STRONG>Method:</STRONG>

1.  Set up PCR assay tubes as described below:

Total: 25 μl
+  0.25 μl of Ex Taq polymerase 
 +  2.5 μl of 10× Taq reaction buffer
 +  2.0 μl  of dNTP(2 mM)  
+  1.0 μl of template (E.coli plasmid 817) 
+  1.0 μl of oligonucleotide primer PtoA-F*
 +  1.0 μl of oligonucleotide primer PtoA-R* 
+  18.25 μl of ddH2O  
*The sequences of primer pair PtoA-F and PtoA-R.
 PtoA-F  5'-CCACCTGACGTCTAAGAAAC-3'
 PtoA-R  5'-ATGATCATCGCCGGCGAATTCAGGC-3' 

 2.    Set parameters for PCR to amplify desired products.  <TABLE><TBODY><TR><TD>Temperature</TD><TD>Time</TD><TD>Cycle</TD></TR><TR><TD>94˚C</TD><TD>5min</TD><TD>1</TD></TR><TR><TD>94˚C</TD><TD>1min</TD><TD>30</TD></TR><TR><TD>55˚C</TD><TD>1min</TD><TD>30</TD></TR><TR><TD>72˚C</TD><TD>1min 20sec</TD><TD>30</TD></TR><TR><TD>4˚C</TD><TD>7hrs</TD><TD>1</TD></TR></TBODY></TABLE><H3><FONT face="Arial, Helvetica"><B><FONT color="#0000FF">Restriction Enzyme Digestion for Verification</FONT></B></FONT></H3>
To verify whether PstI site in pSB1A3 was successfully mutated to AflII site, we performed restriction enzyme digestion experiments. 

Bacterial plasmids are double-stranded circular DNA molecules and uncut plasmid DNA can be in any of three forms - nicked circular, linear, closed supercoiled. When run on an agarose gel one frequently will see these forms as different bands with closed supercoiled form migrates the fastest, linear form migrates the slowest, and nicked circular migrates in between. If the PStI site was successfully mutated to AflII site, we expected to see increased linear form of pSB1A3 when digested with AflII. SpeI-cut pSB1A3 was served as a positive control. <STRONG>Method</STRONG>
  1.  Set up Pst I digestion in 1.5 ml eppendorf tubes as described below:
  Total: 10μl
  + 0.5μl of Pst I  restriction enzyme (company :Takara)
  + 1μl of 10XH buffer
  + 1μl of plasmid DNA
  + 7.5μl of ddH2O 
  2.  Set up Afl II digestion in 1.5 ml eppendorf tubes as described below:
  Total: 10μl
  + 0.5μl of Afl II  restriction enzyme, (company :Takara)
  + 1μl of 10XM buffer
  + 1μl of plasmid DNA
  + 1.0μl of 0.01% BSA
  + 6.5μl of ddH2O
 3. Incubate the eppendorf tubes in 37℃ water bath for 1-2 hr.  
4. Prepare 1% agarose gel in Conical flask. Weigh 0.6 g agarose and add 60 ml 1x TAE (diluted from 50x TAE). Cover the Conical flask with silver paper to avoid the loss of water vapor. Place the Conical flask in the microwave and microwave for 1 minute with a middle power. Take it out and shake gently till the solution is homogeneous,(BE CAREFUL to watch the solution closely when shake it–it superheats and can boil over and cause severe burns). Continue microwave and swirl until solution is seen clear and homogeneous with no existence of solid. After cool down the agarose gel briefly, add 3 μl of Gelred (10000x ) and mix well. Pour the agarose gel in gel casting apparatus and insert combs.
 5.  By inserting the pipette tip below the TAE liquid and into the well, add 5 μl of 1kb DNA ladder solution to first (and last if desired) well, skip one well, then begin adding the 5μl of digested DNA solutions mixed with 1 μl loading buffer (6x) to the wells. 
6.  Place the cover on the electrophoresis unit, plug into the power source, and turn on voltage to 120V, set time to 30 minutes, and press the start button twice,until the bubbles are seen. DNA separation can be observed as time goes on by turning off the power supply then gently removing the basin from the electrophoresis unit (be careful not to let the gel slip out of the basin) and placing on the UV transilluminator to see DNA bands. 
 7.  When the desired level of separation is obtained, the basin can be placed on the transilluminator for picture taking(Of the absence of transilluminator,we use camera to take pictures with the UV light ). 
8.  Cut the gel of specific position and collect it in tubes that have measured weight.  
9.  Use DNA Gel Extraction Ki to purify the plasmid DNA mutant-pSB1A3.
RPF (SpeI/AflII-digested), GFP (SpeI/AflII-digested) fragments were ligated into this vector, which was followed by insertion of designed terminator sequences between RFP and GFP, respectively.<FONT face="Arial, Helvetica">
(Figure 1  : This figure shows that the site-directed mutagenesis succeed ,we successfully  change a restriction enzyme cutting site named Pst I to Afl II. Lane 1  represents the plasmid mutant-pSB1A3,  lane 2 shows that the mutant-pSB1A3  cannot be digested by restriction enzyme Spe I ,lane 3 shows that mutant-pSB1A3 can be digested by restriction enzyme Afl  II.) 
<H3><FONT face="Arial, Helvetica"><B><FONT color="#0000FF">Media Preparation</FONT></B></FONT></H3>
For all experiments  involving the bacterial biomass and experimentation, proper media is chosen to grow  the cells. Commonly,we use Lysogeny broth media for <EM>E. coli</EM>. The following is the media compositions and their  quantities.<STRONG>Method:</STRONG>

1.Prepare the Lysogeny Broth (LB) liquid media (1 L) as indicated below:	
+ Bacto-Tryptone - 10 g
+ NaCl - 10 g
+ Yeast Extract - 5 g
Add ddH2O and 5mmol/L Tris Buffer in a measuring cylinder to ensure accuracy, to make a total of 1 liter and pH is 8.0.
2.Prepare the Lysogeny Broth (LB) solid media (1 L) as indicated below:
+ Bacto-Tryptone - 10 g
+ NaCl - 10 g
+ Yeast Extract - 5 g
+ Difco Agar - 15g
Add ddH2O and 5mmol/L Tris Buffer in a measuring cylinder to ensure accuracy, to make a total of 1 liter and pH is 8.0.
3. Autoclaving
Autoclave at 121 °C for 60 minutes. After the media cooling down enough, antibiotics Ampicillin(100mg of Ampicillin per 1ml of the media) are added. At last the media are poured 15ml on each plate and become solid.Store the plate at 4℃ refrigerator.
 <H3><FONT face="Arial, Helvetica"><B><FONT color="#0000FF">Bacterial Transformation</FONT></B></FONT></H3>

Transformation is commonly used to introduce recombinant plasmid DNA into bacterial strains which can transform naturally or can be made competitive for transformation by artificial means. The purpose of this technique is to introduce a recombinant plasmid DNA into a bacterial strains and to use bacteria strains to amplify the plasmid mutant-pSB1A3 for further plasmid construction.

  <STRONG>Method</STRONG>
  1. Take out an  appropriate number of tubes that contain competent cells(100μl ) from the freezer. Immediately place the tubes on ice, so that all  but the cap is surrounded by ice. Allow the cells to thaw on ice for 2-5 min.
  2.  Visually check the cells to see whether they  have thawed and gently flick the cells 1-2 times to evenly resuspend the cells.
  3. Add 10μl PCR products(mini-prep purified) to the competent cells DH-5α.  Stir gently to mix and return the tube to the ice, making sure that the tube is  surrounded by ice except for the cap. Repeat for additional two times for the  same samples.
  4. Incubate the  tubes on ice for 30 min.
  5. Place the tubes  in a 42°C  water bath for exactly 90 sec; do not shake.
  6. Place the tubes  on ice for 2 min to cool down.
  7. Add 800 μl of room temperature LB medium to each tube.
  8. Shake the tubes  vigorously at 37°C for 45-60 min.
  9. Centrifuge the  tubes at 3K RPM for 1 min. Discard the supernatant liquor and leave 100-200 μl of the mixtures.Mix the contents and spread  the whole liquid on LB agar plates containing the appropriate antibiotic ampicillin  for the plasmid.
10. Place the  plates on the bench for several min to allow excess liquid to be absorbed, and  then invert and incubate overnight at 37°C (12-16 h).
<H3><FONT face="Arial, Helvetica"><B><FONT color="#0000FF">Colony PCR for Verification</FONT></B></FONT></H3>

Colony PCR is used to identify and select cell colonies that have the correct plasmid inserted. The procedure is a way to do several PCR operations on cell colonies in parallel, to evaluate the results and select the corresponding positive cell colonies. After an overnight growth of E.coli, we can pick up several colonies from the plate and do a colony PCR verification. Besides, the colonies we choose and should also be stored, we can incubate these colonies in one plate after every colony has been marked.<STRONG>Method</STRONG>
1. Prepare the sample reaction as  indicated below:
Total: 25μl 
      +  0.25 μl of Ex Taq polymerase (company:Takara)
      +  2.5 μl of 10× Taq reaction buffer
       +  1.0 μl of R-NPS-F*
      +  1.0 μl of G-SXA-R*
      +  1.0 μl of plasmid mutant-pSB1A3
      +  2.0 μl of dNTP( 25mM  ) 
      +  17.25 μl of ddH2O 
Note: The sequences of primers R-NPS-F  , G-SXA-R 
R-NPS-F   5'-TATAGCGGCCGCCTTAAGTAAGTAAGAGTATACG-3'  
G-SXA-R   5'-TCTCCGACTTAAGGGATCCTATAAACGCAG-3'

2.    Set parameters for PCR to amplify desired products.  
				
<TABLE><TBODY><TR><TD>Temperature</TD><TD>Time</TD><TD>Cycle</TD></TR><TR><TD>94˚C</TD><TD>5min</TD><TD>1</TD></TR><TR><TD>94˚C</TD><TD>1min</TD><TD>30</TD></TR><TR><TD>55˚C</TD><TD>1min</TD><TD>30</TD></TR><TR><TD>72˚C</TD><TD>1min 20sec</TD><TD>30</TD></TR><TR><TD>4˚C</TD><TD>7hrs</TD><TD>1</TD></TR></TBODY></TABLE>

3.   Electrophorese the total system and observe the lane separation. 
<H3><FONT color="#0000FF" face="Arial, Helvetica"><B>Cultivate the Bacteria</B></FONT></H3>
              According  to the results of the   PCR detection,  positive colonies are chosen and transferred them to 5ml LB liquid media ( 5μl of ampicillin added) stored in  12.5ml centrifuge tubes. Put the  centrifuge tubes in 37℃ gas bath  overnight. 
<H3><FONT color="#0000FF" face="Arial, Helvetica"><B>Plasmid DNA Isolation</B></FONT></H3>
             Use E.Z.N.A.TM  Plasmid Mini I to realize plasmid DNA isolation.<STRONG>Method</STRONG><OL><LI>Transfer 5 ml of overnight culture into a  1.5-ml eppendorf tube labeled with group number.  </LI><LI>Centrifuge the sample at max. speed of desk  top centrifuge and RT for 1min to pellet the cells.</LI><LI>Discard the supernatant. Remove as much of  the supernatant as possible without disturbing the cell pellet. </LI><LI>Repeat step 1 and 2 twice. </LI><LI>Resuspend the pellet completely in 250 ml of Solution I (containing RNase A) by vortexing  the samples vigorously . No clumps should be visible in the tube. </LI><LI>Add 250 ml of Solution II and mix the sample by gently inverting the  tube 4 to 6 times. Do not vortex or shake the sample vigorously. The bacterial  suspension should begin to clear which have lysed the bacterial cells in this  step. <STRONG>Warning: </STRONG>Do not stop here for more than five min, as the high pH  hurts your DNA! </LI><LI>Add 350 ml of Solution III and mix by gently inverting the tube 4 to  6 times until a flocculent white precipitate forms. Do not shake vigorously, as  it might break the genomic DNA. </LI><LI>Centrifuge at maximum speed for 10 min at  room temperature to pellet the cell debris. You should see a white precipitate  in the tube after the centrifugation. </LI><LI>While the samples are centrifuging, for  each sample, label a clean HiBind Miniprep Column which is to assembled in a  2-ml collection tube</LI><LI>Apply the supernatants from step 8 to the  columns. </LI><LI>Centrifuge at maximum speed for 1 min at  RT. Discard the flow-through in the collection tube. </LI><LI>Add 500 ml of Buffer HB to wash the Hibind Miniprep Column.  Centrifuge at maximum speed for 1 min at RT. Discard the flow-through in the  collection tube.  </LI><LI>Wash the  column by adding 700 ml of DNA  Wash Buffer diluted with absolute ethanol. Centrifuge at maximum speed for 1  min at room temperature and discard the flow-through.</LI><LI>Then centrifuge the tubes again for 2 min  to remove all the moisture.</LI><LI>Place the column in a clean 1.5 ml  eppendorf tube that is labeled with the plasmid name and group number. To elute  the DNA, add 50 ml of Elution Buffer  to the center of each column. Let the samples stand for 2 or more minutes at  RT, and then centrifuge for 1 min. The sample in the centrifuge tube (bottom)  is your plasmid DNA. </LI><LI>Discard the column and save the sample in  the eppendorf tube by placing it in the freezer (-20°C).  </LI></OL><H3><FONT face="Arial, Helvetica"><B><FONT color="#0000FF">Restriction Enzyme Digestion and Electrophoresis</FONT></B></FONT></H3>
              
Because  the colony PCR test is so sensitive and affect markedly by environment factors.  So we do a restriction enzyme digestion to ensure that the isolated plasmid is  the site-directed mutated plasmid.<STRONG>Method</STRONG>
1.  Prepare the control  reaction as indicated below:
Total: 10μl
      + 0.5μl of Pst I  restriction enzyme(company :Takara)
      + 1μl of 10XH buffer
      + 1μl of plasmid DNA
      + 7.5μl of ddH2O 
2.   Prepare the sample reaction as indicated below:
Total: 10μl
     + 0.5μl of Afl II  restriction enzyme, (company :Takara)
     + 1μl of 10XM buffer
     + 1μl of plasmid DNA
     + 1.0μl of 0.01% BSA
     + 6.5μl of ddH2O 
3.   Electrophorese the total system and observe the lane separation.
<H3><FONT face="Arial, Helvetica"><B><FONT color="#0000FF">Polymerase Chain  Reaction(GFP &amp; RFP) and Electrophoresis</FONT></B></FONT></H3>
              
              GFP and RFP DNA fragments are the insert  which need to be ligate to the plasmid mutant-pSB1A3. Do a PCR amplification can get enough quantities  for the following reactions.<STRONG>Method </STRONG>
１．Prepare the sample reaction as  indicated below:
Total: 100μl ( PCR amplification of GFP fragments） 
    + 1.0μl of Taq DNA  polymerase,#EP0402
    + 10μl of 10XTaq buffer 
    + 10μl of MgCl2(25mM)
    + 10μl of dNTP(2mM)
    + 2μl of G-SXA-R* 
    + 2μl of G-SXA-F*
    + 4μl of DNA template
    + 61μl of ddH2O 
Total: 100μl（PCR amplification of RFP fragments） 
    + 1.0μl of Taq DNA  polymerase, EP0402
    + 10μl of 10XTaq buffer 
    + 10μl of MgCl2(25mM)
    + 10μl of dNTP(2mM)
    + 2μl of R-NPS-R* 
    + 2μl of R-NPS-F*
    + 4μl of DNA template
    + 61μl of ddH2O 

Note: The sequences of primers R-NPS-F , R-NPS-R , G-SXA-F ,G-SXA-R 
R-NPS-F   5'-TATAGCGGCCGCCTTAAGTAAGTAAGAGTATACG-3'  
R-NPS-R   5'-CGGAGACTAGTCTGCAGATCACATAAGTAAAGTGATAATC-3' 
G-SXA-F   5'-CTAGACTAGTTCTAGAGGCGGACTCACTATAGA-3'  
G-SXA-R   5'-TCTCCGACTTAAGGGATCCTATAAACGCAG-3'

 2.    Set parameters for PCR to amplify desired products.  <TABLE><TBODY><TR><TD>Temperature</TD><TD>Time</TD><TD>Cycle</TD></TR><TR><TD>94˚C</TD><TD>5min</TD><TD>1</TD></TR><TR><TD>94˚C</TD><TD>1min</TD><TD>30</TD></TR><TR><TD>55˚C</TD><TD>1min</TD><TD>30</TD></TR><TR><TD>72˚C</TD><TD>1min 20sec</TD><TD>30</TD></TR><TR><TD>4˚C</TD><TD>7hrs</TD><TD>1</TD></TR></TBODY></TABLE>


              3. Use  DNA Gel Extraction Kit  to purify the GFP and RFP DNA fragments after the Electrophoresis.
         
              
              (Figure 2  : This figure shows that The PCR reaction system can amplify large quantities  of GFP and RFP DNA fragments. The digestion based on the GFP and RFP DNA  fragments can be done to prepare for the ligation. Lane 1 represents the  template E.coli 817 can amplify the GFP and RFP DNA fragments , lane 2  represents the template E.coli 817(355.5) can also amplify the GFP and RFP DNA  fragments.)<H3><FONT face="Arial, Helvetica"><B><FONT color="#0000FF">Double Restriction  Enzyme Digestion and Electrophoresis.</FONT></B></FONT></H3><FONT face="Arial, Helvetica">
              Use specific restriction enzymes to digest  plasmid mutant-pSB1A3,GFP and  RFP to get sticky ends and purify the DNA fragment after the Electrophoresis.<STRONG>Method:</STRONG></FONT>

              1. Digestion of plasmid mutant-pSB1A3 
              Prepare the sample reaction as indicated below:
                Total: 50μl 
                + 3.0μl of Not I restriction enzyme, #ER0591
                + 3.0μl of Afl II restriction enzyme, #ER0831
                + 5.0μl of 10X buffer O 
                + 1.0μl of mutant-pSB1A3 plasmid 
                + 37.0μl of ddH2O

                2.  Digestion of PCR  product GFP
                Prepare the sample reaction as indicated below:
                Total: 50μl 
                + 3.0μl of Not I restriction enzyme, #ER0591
                + 3.0μl of Spe I restriction enzyme, #ER1251 
                + 5μl of 10X buffer Tango
                + 10μl of PCR products GFP
                + 29μl of ddH2O

                3.  Digestion of PCR  product RFP
                Prepare the sample reaction as indicated below:
                Total: 50μl 
                + 3.0μl of Afl II restriction enzyme, #ER0831
                + 3.0μl of Spe I restriction enzyme, #ER1251 
                + 5μl of 10X buffer Tango
                + 10μl of PCR products RFP
                + 29μl of ddH2O

                4.    Put the  tubes in 37℃ environment for 4-8 hours 

                5.   Use DNA Gel Extraction Kit to purify the  mutant-pSB1A3 fragment,  GFP and RFP after digestion and named them by mutant-pSB1A3 (NA) ,GFP(NS) and RFP(AS) after the  Electrophoresis.
<H3><FONT face="Arial, Helvetica"><B><FONT color="#0000FF">Ligation</FONT></B></FONT></H3>
              
              Ligation is the process that target DNA gene is inserted  into a plasmid. Both the vector and insert are prepared to have the sticky ends.  These two kinds of DNA pieces are placed in a reaction tube and the proper DNA  ligase, buffer, and cofactors are added for the reaction to take place. When  done properly, the ligation will result in a successful combination of the  insert and plasmid into one plasmid.Based on the digestion of mutant-pSB1A3,GFP and RFP DNA fragments,also ligation  can be done. We ligate mutant-pSB1A3  vector and sticky GFP and RFP DNA fragments to construct an new plasmid  mutant-pSB1A3-GR.  

                1.  Prepare the control  reaction as indicated below:
                Total: 10μl 
                + 1.0μl of plasmid mutant-pSB1A3  (NA) 
                + 3.0μl of GFP(NS) 
                + 3.0μl of RFP(AS) 
                + 2.0μl of T4  DNA Ligase,#EL0011 
                + 1.0μl of 10XT4  Ligase buffer
                Note: GFP(NS)  means the product of GFP DNA fragments digested by restriction enzyme Not I and  Spe I. 
                2.  Prepare the sample  reaction as indicated below:
                Total: 10μl 
                + 1.0μl of plasmid mutant-pSB1A3  (NA) 
                + 2.0μl of T4  DNA Ligase, #EL0011   
                + 1.0μl of 10XT4  Ligase buffer
                + 6.0μl of ddH2O
                3.  Put  the tubes in 22℃ water bath, react for 8-12 hours.
<H3><FONT face="Arial, Helvetica"><B><FONT color="#0000FF">Bacterial Transformation</FONT></B></FONT></H3><FONT face="Arial, Helvetica">
              Transform the ligation products into  the DH-5α competent cells, put the plate on 37℃ gas bath for 12-16  hours. 
</FONT><H3><FONT face="Arial, Helvetica"><B><FONT color="#0000FF">Bacterial Colony PCR</FONT></B></FONT></H3>
              
              Colony PCR is used  to identify and select cell colonies that have the correct plasmid insert. This  procedure is a way to do several PCR operations on cell colonies in parallel,  to evaluate the results and select the corresponding good cell colonies. After an  overnight growth of E.coli, we can pick up some colonies from the plate and do  a colony PCR verification. Besides, the colonies we choose and should also be  stored, we can incubate these colonies in one plate after every colony has been  marked.<STRONG>Method</STRONG>
1. Prepare the sample reaction as  indicated below:
Total: 20μl 
     +  0.25 μl of Ex Taq polymerase,#EP0402 
     +  2.0 μl of 10× Taq reaction buffer
     +  1.0 μl of R-NPS-F* 
     +  1.0 μl of G-SXA-R*
     +  5.0 μl of bacterial colony
     +  2.0 μl of dNTP( 25mM  ) 
     +  8.75 μl of ddH2O 
Note: The sequences of primers R-NPS-F , R-NPS-R , G-SXA-F ,G-SXA-R 
R-NPS-F   5'-TATAGCGGCCGCCTTAAGTAAGTAAGAGTATACG-3'  
R-NPS-R   5'-CGGAGACTAGTCTGCAGATCACATAAGTAAAGTGATAATC-3' 
G-SXA-F   5'-CTAGACTAGTTCTAGAGGCGGACTCACTATAGA-3'  
G-SXA-R   5'-TCTCCGACTTAAGGGATCCTATAAACGCAG-3'

 2.    Set parameters for PCR to amplify desired products.  <TABLE><TBODY><TR><TD>Temperature</TD><TD>Time</TD><TD>Cycle</TD></TR><TR><TD>94˚C</TD><TD>13.5min</TD><TD>1</TD></TR><TR><TD>94˚C</TD><TD>1min</TD><TD>30</TD></TR><TR><TD>55˚C</TD><TD>1min</TD><TD>30</TD></TR><TR><TD>72˚C</TD><TD>1min 20sec</TD><TD>30</TD></TR><TR><TD>4˚C</TD><TD>7hrs</TD><TD>1</TD></TR></TBODY></TABLE>

              3. Electrophorese the  total system and observe the lane separation. 
              (Figure 3: The lane on the figure are 2k DNA fragments, it shows the GFP and RFP DNA fragments are ligated to the vectors which were isolated from the bacterial colonies.) 
<H3><FONT color="#0000FF" face="Arial, Helvetica"><B>Cultivate the Bacteria</B></FONT></H3>
              According to the results of the   PCR detection, we choose positive colonies and  transfer them to 5ml LB liquid media ( 5μl of ampicillin has added) stored  in 12.5ml centrifuge tubes. Put the centrifuge tubes in 37℃  gas bath overnight.
<H3><FONT color="#0000FF" face="Arial, Helvetica"><B>Plasmid DNA Isolation</B></FONT></H3><FONT face="Arial, Helvetica">Use E.Z.N.A.TM Plasmid Mini I to isolate the constructed plasmid mutant-pSB1A3-GR. </FONT><H3><FONT color="#0000FF" face="Arial, Helvetica"><B>Restriction Enzyme  Digestion and Electrophoresis</B></FONT></H3>
              From the last step, we got the certain  quantities of isolated plasmids. In this step, we do two restriction enzyme  digestion reactions, one to prove that the plasmid is construct correctly (  mutant-pSB1A3-GR ), one  to get sticky ends preparing for the ligation.<STRONG>Method</STRONG>
              1.Restriction Enzyme Digestion to prove that plasmid is  constructed correctly
              
              1.1.  Prepare the sample  reaction as indicated below:
                Total: 10μl
                + 1.0μl of Not I  restriction enzyme,#ER0591
                + 1.0μl of Spe I  restriction enzyme,#ER1251     
                + 2.0μl of Buffer Tango( 10X )
                + 1.5μl of plasmid  mutant-pSB1A3-GR
                + 14.5μl of ddH2O 
                1.2.   Electrophorese the total system and observe the lane separation. 

              2. Restriction Enzyme Digestion to get sticky ends preparing  for the ligation.

              2.1.  Prepare the sample  reaction as indicated below:
                Total: 50μl
                + 5.0μl of Pst I  restriction enzyme, #ER0611
                + 5.0μl of Xba I restriction enzyme, #ER0681    
                + 3.0μl of Buffer Tango( 10X )
                + 5.0μl of plasmid  mutant-pSB1A3-GR
                + 32.0μl of ddH2O 
                2.2.   Electrophorese the total system and observe the lane separation.
                2.3.   Cut the gel of specific position and collect it in tubes that have  measured weight. 
                2.4. Use DNA  Gel Extraction Ki to purify the plasmid  DNA mutant-pSB1A3-GR(PX) .
                Note: Mutant-pSB1A3-GR(PX) means plasmid Mutant-pSB1A3-GR digested by Pst I and Xba I.

             
             
              (Figure 4 : The double  digestion of mutant-pSB1A3  forms a linear DNA fragments and it runs slower than circle DNA fragments. This  suggest that the double digestion of mutant-pSB1A3 works in a high efficiency, and desired sticky  ends are formed.1,3,5  are plasmids  digested by restriction enzyme Pst I and Xba I from different colonies, 2,5,6  are pure plasmid mutant-pSB1A3-GR,  7 is the plasmid mutant-pSB1A3.  )
 <H3><FONT color="#0000FF" face="Arial, Helvetica"><B>Ligation</B></FONT></H3>
              Ligation is the process that target DNA gene is inserted  into a plasmid. Both the vector and insert are prepared to have the sticky ends.  These two kinds of DNA pieces are placed in a reaction tube and the proper DNA  ligase, buffer, and cofactors are added for the reaction to take place. When done  properly, the ligation will result in a successful combination of the insert  and plasmid into one plasmid.Based on the digestion of mutant-pSB1A3-GR, terminator DNA fragments,ligation can  be done. We ligate mutant-pSB1A3-GR  vector and sticky terminator DNA fragments to construct an new plasmid  mutant-pSB1A3-GR-t.By  detecting the quantities of GFP and RFP, terminator efficiency can be  calculated. 
1.  Prepare the control  reaction as indicated below:
Total: 10μl 
      + 1.0μl of plasmid mutant-pSB1A3  (NA) 
      + 6.0μl of terminator
      + 2.0μl of T4  DNA Ligase , #EL0011 
      + 1.0μl of 10XT4  Ligase buffer
2.  Put  the tubes in 22℃ water bath, react for 8-12 hours. 
<H3><FONT color="#0000FF" face="Arial, Helvetica"><B>Bacteria  transformation</B></FONT></H3>
              Transform the ligation products into  the DH-5α competent cells, put the plate on 37℃ gas bath for 12-16  hours. 
<H3><FONT color="#0000FF" face="Arial, Helvetica"><B>Cultivate the Bacteria</B></FONT></H3>
              According to the results of the   PCR detection, we choose positive colonies and  transfer them to 5ml LB liquid media ( 5μl of ampicillin has added)  stored in 12.5ml centrifuge tubes. Put the  centrifuge tubes in 37℃ gas bath  overnight.
<H3><FONT color="#0000FF" face="Arial, Helvetica"><B>Flow Cytometer Analysis</B></FONT></H3>
              Flow cytometry is a laser based, biophysical technology employed in cell counting, sorting, biomarker detection and protein engineering, by suspending cells in a stream of fluid and passing them by an electronic detection apparatus. It allows simultaneous multiparametric analysis of the physical and/or chemical characteristics of up to thousands of particles per second. 
   Each fluorophore has a characteristic peak excitation and emission wavelength, and the emission spectra often overlap. Consequently, the combination of labels which can be used depends on the wavelength of the lamp(s) or laser(s) used to excite the fluorochromes and on the detectors available. It is practical that the expression of RFP and GFP can be measured at the same time.<STRONG>Method:</STRONG>
1. Materials:  1.1. NaCl solution( 0.9% ,M/V)  1.2. 75% ethanol
2. Procedures:  2.1. Bacterium are harvested by centrifuge at 10000g for 30s, then discard the supernatant. Repeat this step again until we got adequate bacterium.  2.2. Suspend the bacterium With NaCl solution(0.9%) and vibrate the centrifuge tubes until the bacterium are distributed homogeneous.  2.3. Load the bacteria containing pSB1A3 to Flow Cytometer (Beckman), set the parameters. Measure the GFP-FL1 and RFP-FL2 by Cytometer. 
<H3><FONT color="#0000FF" face="Arial, Helvetica"><B>Fluorescence  Microscope</B></FONT></H3>
A fluorescence microscope is an optical microscope that uses fluorescence and phosphorescence  reflection and absorption to study properties of organic or inorganic substances and generate an image.Fluorescence microscope detection are used to confirm the expression of GFP and RFP which are illuminated with light of a wavelength which excites fluorescence in the sample.
<STRONG>Method</STRONG>
Add 10μl of bacteria solution to micro slide and cover with coverslip.
Placed the micro slide on the Fluorescence Microscope. Set parameters to detect GFP and RFP. 
Generate an image of fluorescence protein and save it.
</FONT></FONT></ARTICLE></SECTION></DIV></DIV><DIV id="outerfooter"><DIV id="footercontainer"><FOOTER id="footer">South University of Science and Technology of China</FOOTER></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:SUSTC-Shenzhen-B/protocol1">http://2012.igem.org/Team:SUSTC-Shenzhen-B/protocol1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SUSTC-Shenzhen-B/protocol1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SUSTC-Shenzhen-B/protocol1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen-B/protocol1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen-B/protocol1&amp;oldid=290274" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>