<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SUSTC-Shenzhen-B_protocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SUSTC-Shenzhen-B/protocol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SUSTC-Shenzhen-B/protocol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen-B/protocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen-B/protocol&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SUSTC-Shenzhen-B/protocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SUSTC-Shenzhen-B/protocol</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>SUSTC iGEM Theme - Free CSS Template</TITLE></P><DIV id="templatemo_body_wrapper"><DIV id="templatemo_wrapper"><DIV id="templatemo_header"><DIV id="templatemo_menu" class="ddsmoothmenu"><UL id="none"><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B" class="selected">Home</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/introduction">Software</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/introduction">Introduction</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/algorithm">Algorithm</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/achievements">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/future plan">Future Plan</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/Download">Download</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/comment">User Comments</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/Tutorial">Tutorial</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/SBOL">SBOL</A></LI></UL><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/lab introduction">Wet lab</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/lab introduction">Introduction</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/protocol1">Lab Protocol</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/lab results">Lab Results</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/lab.sbol">SBOL Document</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/future work">Future Works</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/RFC">Technical Standard</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/members">Team</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/members">Members</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/advisers">Advisers</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/photos">Photo Gallery</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/Contributions">Contributions</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/before.june">Before June</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/july">July</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/august">August</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/September">September</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/hp.intro">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/hp.intro">Introduction</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/open.class">Open Classes</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/high.school.visits">High School Visits</A></LI><LI><A href="https://2012.igem.org/Team:SUSTC-Shenzhen-B/SynBio.intro">SynBio Intro</A></LI></DIV></DIV><TITLE>SUSTC iGEM Theme - Free CSS Template</TITLE><DIV id="templatemo_main"><H2>Lab Protocol</H2><H3><B>Brief Process</B></H3><P>1.  Site-Directed Mutagenesis </P><P> 2.  Restriction enzyme digestion and electrophoresis (proof)</P><P> 3.  Media Preparation</P><P> 4.  Bacterial Transformation</P><P> 5.  Colony PCR for verification</P><P> 6.  Culture the bacteria  </P><P> 7.  Plasmid DNA Isolation</P><P> 8.  Restriction Enzyme Digestion and Electrophoresis(proof) </P><P> 9.  Amplify GFP &amp; RFP</P><P> 10. Electrophoresis and Restriction Enzyme Digestion.</P><P>11. Ligation</P><P>12. Bacterial Transformation</P><P>13. bacterial colony PCR for verification</P><P> 14. Culture the bacteria</P><P> 15. Plasmid DNA Isolation</P><P> 16. Restriction Enzyme Digestion and Electrophoresis</P><H3><B>Site-Directed Mutagenesis</B></H3><P>We choose plasmid psb1a3 to be the vector that ligate GPF and RFP fragments.To protect the structural integrity of the constructed plasmid, we need to mutate a restriction enzyme cutting site named Pst I to Afl II. Proper primer are designed for this purpose.</P><P>PtoA-F:5'-CCACCTGACGTCTAAGAAAC-3'</P><P>PtoA-R:5'-ATGATCATCGCCGGCGAATTCAGGC-3' </P><P>1.  Prepare the sample reaction as indicated below:</P><P>Total: 25μl</P><P>+  0.25 μl of Ex Taq polymerase(酶的公司名称)</P><P>+  2.5 μl of 10× Taq reaction buffer</P><P>+  2.0 μl  of dNTP(2mM)</P><P>+  1.0 μl of template </P><P>+  1.0 μl of oligonucleotide primer P1</P><P>+  1.0 μl of oligonucleotide primer P2</P><P>+  18.25 μl of ddH2O</P><P>2.  Set thermocycler temperatures and the time. Procedure on the thermocycler are listed below:</P><P>①	 94˚C for 5 min</P><P>②	30 cycle</P><P>a.	94˚C for 1 min</P><P>b.	55˚C for 1 min</P><P>c.	72˚C for 1 min20sec</P><P>③	 72˚C for 10 min</P><P>④	 4˚C for stock</P><H3><B>Restriction Enzyme Digestion for Verification</B></H3><P>After the PCR mutation we have to do a restriction enzyme digestion to test whether the mutation is successful. The reaction system is as follows:</P><P>1.  Prepare the control reaction as indicated below:</P><P>Total: 10μl</P><P>+ 0.5μl of Pst I restriction enzyme</P><P>+ 1μl of 10XH buffer</P><P>+ 1μl of plasmid DNA</P><P>+ 7.5μl of ddH2O </P><P>2.  Prepare the sample reaction as indicated below:</P><P>Total: 25μl</P><P>+ 0.5μl of Afl II restriction enzyme, #ER0831</P><P>+ 1μl of 10XM buffer</P><P>+ 1μl of plasmid DNA</P><P>+ 1.0μl of 0.1% BSA</P><P>+ 6.5μl of ddH2O</P><P>After the reaction, do an electrophoresis to see the 2k bp cyclic plasmid DNA change into linear. The process is as follows:</P><P>3. Prepare electrophoresis gel by adding 0.6g agarose to 60ml TAE (1% solution,1X,diluted from 50X TAE). Pour on conical flask and cover the Conical flask sealing surface with silver paper to avoid the loss of water vapor. Place in the microwave and microwave on middle for 1 minute at a time, pulling it out and swirling until solution is homogeneous again, then repeating (BE CAREFUL to watch the solution closely when swirling–it superheats and can boil over and cause severe burns). Continue until solution is seen clear and homogeneous with no exsistence of solid.Add 3 μl of (神马情况！！！)</P><P>4. By inserting the pipette tip below the TAE liquid and into the well, add 5μl of 1kb DNA ladder solution to first (and last if desired) well, skip one well, then begin adding the 5μl of digested DNA solutions mixed with 1μl loading buffer (6X) to the wells.</P><P>5. Place the cover on the electrophoresis unit, plug into the power source, and turn on voltage to 120V, set time to 30 minutes, and press the start button twice,leave when bubbles are seen. DNA separation can be observed as time goes on by turning off the power supply then gently removing the basin from the electrophoresis unit (be careful not to let the gel slip out of the basin) and placing on the UV transilluminator to see DNA bands. </P><P>6. When the desired level of separation is obtained, the basin can be placed on the transilluminator for picture taking(Of the absence of transilluminator,we use camera to take pictures with the UV light ).</P><H3><B>Media Preparation</B></H3><P>For all experiments involving the bacterial biomass and experimentation, proper media is chosen to grow the cells. Commonly,we use Lysogeny broth media for E. coli. The following is the media compositions and their quantities.</P><P><B>Lysogeny Broth (LB) liquid media (1 L)</B></P><P>Measure out these following:	</P><P>Bacto-Tryptone - 10 g</P><P>NaCl - 10 g</P><P>Yeast Extract - 5 g</P><P>Add ddH2O and 5mmol/L Tris Buffer in a measuring cylinder to ensure accuracy, to make a total of 1 liter and pH is 8.0.</P><P><B>Lysogeny Broth (LB) solid media (1 L)</B></P><P>Measure out these following:</P><P>Bacto-Tryptone - 10 g</P><P>NaCl - 10 g</P><P>Yeast Extract - 5 g</P><P>Difco Agar - 15g</P><P>Add ddH2O and 5mmol/L Tris Buffer in a measuring cylinder to ensure accuracy, to make a total of 1 liter and pH is 8.0.</P><P><B>Autoclaving</B></P><P>Autoclave at 121 °C for 45 minutes (liquid setting, 0 minutes drying time). For making plates, after the media cool enough, antibiotics Ampicillin(0.6*10^-3 volume of media ) are added. At last media are poured 15ml on each plate and become solid.</P><H3><B>Bacterial Transformation</B></H3><P>Introduction of exogenous DNA into cells using non-viral methods is called “Transformation”.Transformation is commonly used to introduce recombinant plasmid DNA into bacterial strains which can transform naturally or can be made competitive for transformation by artificial means.</P><P>Depending on the expected transformation efficiency, there are two main types of competent cells that can be used for transformation.</P><P>1.Chemically competent cells</P><P>Chemically induced competent cells are calcium chloride-treated to facilitate attachment of the plasmid DNA to the competent cell membrane. During chemical transformation, the cells are heat-shocked in a water bath; which opens the pores of the cell membrane allowing entry of plasmid DNA from the buffer. </P><P>2.Electrocompetent cells</P>
Electrocompetent cells are prepared for transformation using electroporation, a method that uses an electrical pulse to create pores through which genetic material enters the cells. This method usually has high transformation efficiency. <P><B>Procedure</B></P><P>1. Take out an appropriate number of tubes that contain competent cells(100μl ) from the -80℃ refrigerator. Immediately place the tubes on ice, so that all but the cap is surrounded by ice. Allow the cells to thaw on ice for 2-5 min.</P><P>2.  Visually check the cells to see whether they have thawed and gently flick the cells 1-2 times to evenly resuspend the cells.</P><P>3. Add 10μl PCR products(mini-prep purified) to the competent cells DH-5α. Stir gently to mix and return the tube to the ice, making sure that the tube is surrounded by ice except for the cap. Repeat for additional two times for the same samples.</P><P>4. Incubate the tubes on ice for 30 min.</P><P>5. Place the tubes in a 42°C water bath for exactly 90 sec; do not shake.</P><P>6. Place the tubes on ice for 2 min to cool down.</P><P>7. Add 800 l of room temperature LB medium to each tube.</P><P>8. Shake the tubes vigorously at 37°C for 45-60 min.</P><P>9. Centrifuge the tubes at 3K RPM for 1 min. Discard the supernatant liquor and leave 100-200 μl of the mixtures.Mix the contents and spread the whole liquid on LB agar plates containing the appropriate antibiotic ampicillin for the plasmid.</P><P>10. Let the plates sit on the bench for several min to allow excess liquid to be absorbed, and then invert and incubate overnight at 37°C (12-16 h).</P><P>Make sure that all these steps should be done in super clean bench to avoid contamination.</P><H3><B>bacterial Colony PCR for Verification </B></H3><P>After the growth of E.coli, then pick up some colonies from the plate and do a colony PCR verification. (体系！！！)Do an electrophoresis to test which colony is positive.  Besides, the colonies we choose should also be stored, we can incubate these colonies in one plate after every colony has been marked.</P><H3><B>Culture the Bacteria</B></H3><P>Pick up the positive colonies and throw it to 5ml LB liquid media using 12.5ml centrifuge tube. Then culture the bacteria in 37℃ overnight.</P><H3><B>Plasmid DNA Isolation</B></H3><H3><B>Restriction Enzyme Digestion and Electrophoresis(proof)</B></H3><P>Because the colony PCR test is so sensitive that we still need a digestion to proof that the mutation is successful again.  </P><P>Procedure </P><P>1.  Prepare the control reaction as indicated below: </P><P>Total: 10μl </P><P>+ 0.5μl of Pst I restriction enzyme </P><P>+ 1μl of 10XH buffer </P><P>+ 1μl of plasmid DNA </P><P>+ 7.5μl of ddH2O  </P><P>2.  Prepare the sample reaction as indicated below: </P><P>Total: 25μl </P><P>+ 0.5μl of Afl II restriction enzyme, #ER0831 </P><P>+ 1μl of 10XM buffer </P><P>+ 1μl of plasmid DNA </P><P>+ 1.0μl of 0.1% BSA </P><P>+ 6.5μl of ddH2O </P><P>Then do an electrophoresis. </P><H3><B>Amplify GFP &amp; RFP using PCR</B></H3><P>GFP and RFP is the fragments which need to be ligate to the plasmid.PCR amplification can get enough quantities for the following reactions. </P><P>１．Prepare the sample reaction as indicated below:</P><P>Total: 100μl ( PCR amplificationｏｆ　ＧＦＰ）</P><P>+ 1.0μl of Taq DNA polymerase</P><P>+ 10μl of 10XTaq buffer </P><P>+ 10μl of MgCl2(25mM)</P><P>+ 10μl of dNTP(2mM)</P><P>+ 2μl of G-SXA-R </P><P>+ 2μl of G-SXA-F</P><P>+ 4μl of DNA template</P><P>+ 61μl of ddH2O </P><P>Total: 100μl（ＰＣＲ amplification ｏｆ　ＲＦＰ）</P><P>+ 1.0μl of Taq DNA polymerase</P><P>+ 10μl of 10XTaq buffer </P><P>+ 10μl of MgCl2(25mM)</P><P>+ 10μl of dNTP(2mM)</P><P>+ 2μl of Ｒ－ＮＰＳ－Ｆ </P><P>+ 2μl of Ｒ－ＮＰＳ－Ｒ</P><P>+ 4μl of DNA template</P><P>+ 61μl of ddH2O </P><P>R-NPS-F:5'-TATAGCGGCCGCCTTAAGTAAGTAAGAGTATACG-3'  </P><P>R-NPS-R:5'-CGGAGACTAGTCTGCAGATCACATAAGTAAAGTGATAATC-3' </P><P>G-SXA-F:5'-CTAGACTAGTTCTAGAGGCGGACTCACTATAGA-3'  </P><P>G-SXA-R:5'-CCGACTTAAGGGATCCTATAAACGCAG-3'</P><P>２．Procedure on the thermocycler are listed below:</P><P>① 94˚C for 5 min</P><P>② 30 cycle</P><P>a. 94˚C for 1 min</P><P>b. 55˚C for 1 min</P><P>c. 72˚C for 1 min20sec</P><P>③ 72˚C for 10 min</P><P>④ 12˚C for for long time</P><P>3. Use  DNA Gel Extraction Kit to purify the GFP and RFP DNA fragments after the Electrophoresis.</P><H3><B>Restriction Enzyme Digestion and Electrophoresis.</B></H3><P>Use specific restriction enzymes to digest plasmid mutant-psb1a3,GFP and RFP to get sticky ends and purify the DNA fragment after the Electrophoresis.</P><P>Menthod:</P><P>1.  Digestion of plasmid mutant-psb1a3,Prepare the sample reaction as indicated below:</P><P>Total: 50μl </P><P>+ 3.0μl of Not I restriction enzyme, #ER0591</P><P>+ 3.0μl of Afl II restriction enzyme, #ER0831</P><P>+ 5.0μl of 10X buffer O </P><P>+ 1.0μl of mutant-mutant-psb1a3 plasmid</P><P>+ 37.0μl of ddH2O</P><P>2.  Digestion of PCR product GFP, Prepare the sample reaction as indicated below:</P><P>Total: 50μl </P><P>+ 3.0μl of Not I restriction enzyme, #ER0591</P><P>+ 3.0μl of Spe I restriction enzyme, #ER1251 </P><P>+ 5μl of 10X buffer Tango</P><P>+ 10μl of PCR products GFP</P><P>+ 29μl of ddH2O</P><P>3.  Digestion of PCR product RFP</P><P>Prepare the sample reaction as indicated below:</P><P>Total: 50μl </P><P>+ 3.0μl of Afl II restriction enzyme, #ER0831</P><P>+ 3.0μl of Spe I restriction enzyme, #ER1251 </P><P>+ 5μl of 10X buffer Tango</P><P>+ 10μl of PCR products RFP</P><P>+ 29μl of ddH2O</P><P>4.   Put the tubes in 37℃ environment for 4-8 hours</P><P>5.  Use DNA Gel Extraction Ki to purify the mutant-psb1a3 fragment, GFP and RFP after digestion and named them by mutant-psb1a3 (NA) ,GFP(NS) and RFP(AS) after the Electrophoresis.</P><H3><B>Ligation</B></H3><P>Ligation is the process by which target DNA gene is inserted into a plasmid. Both the plasmid and insert are prepared to have the sticky ends. These two kinds of DNA pieces are placed in a reaction tube and the proper DNA ligase, buffer, and cofactors are added for the reaction to take place. When done properly, the ligation will result in a successful combination of the insert and plasmid into one plasmid. 
</P><P><B>Procedures</B></P><P>1. Prepare the control reaction as indicated below:</P><P>Total: 10μl </P><P>+ 1.0μl of plasmid mutant-psb1a3 (NA)</P><P>+ 3.0μl of GFP(NS)</P><P>+ 3.0μl of RFP(AS)</P><P>+ 2.0μl of T4 DNA Ligase , 5u/μl,#EL0011 </P><P>+ 1.0μl of 10XT4 Ligase buffer</P><P>Prepare the sample reaction as indicated below:</P><P>Total: 10μl </P><P>+ 1.0μl of plasmid mutant-psb1a3 (NA)</P><P>+ 2.0μl of T4 DNA Ligase,5u/μl  </P><P>+ 1.0μl of 10XT4 Ligase buffer</P><P>+ 6.0μl of ddH2O</P><P>2. Put the tubes in 16℃ environment for 8-12 hours.</P><H3><B>Bacterial Transformation</B></H3><P>Transform the ligation products into the DH5α competent cell, put the plate overnight.</P><H3><B>Bacterial Colony PCR for Verification</B></H3><P>After the growth of E.coli, then pick up some colonies from the plate and do a colony PCR verification. (体系！！！)Do an electrophoresis to test which colony is positive.  Besides, the colonies we choose should also be stored, we can incubate these colonies in one plate after every colony has been marked.</P><H3><B>Culture the Bacteria</B></H3><P>Pick up the positive colonies and throw it to 5ml LB liquid media using 12.5ml centrifuge tube. Then culture the bacteria in 37℃ overnight.</P></DIV></DIV></DIV></DIV><DIV id="templatemo_bottom"><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:SUSTC-Shenzhen-B/protocol">http://2012.igem.org/Team:SUSTC-Shenzhen-B/protocol</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SUSTC-Shenzhen-B/protocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SUSTC-Shenzhen-B/protocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen-B/protocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SUSTC-Shenzhen-B/protocol&amp;oldid=160600" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>