<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UI_Indonesia_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UI Indonesia/Parts</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-inner"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2018.igem.org/Team:UI_Indonesia">Home</A></DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown navbar-project"><A href="https://2018.igem.org/Team:UI_Indonesia/Project">Project</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Project#">Overview</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Project#ourproject">Our Project</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Project#resultsanddiscuccions">Results and Discussions</A></LI></UL><LI class="navbar-parts current"><A href="https://2018.igem.org/Team:UI_Indonesia/Parts">Parts</A></LI><LI class="navbar-safety"><A href="https://2018.igem.org/Team:UI_Indonesia/Safety">Safety</A></LI><LI class="dropdown navbar-interlab"><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#intro">Introduction</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#materials">Materials and Equipment</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#methods">Methods</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#results">Results and Discussions</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#conclusions">Conclusions</A></LI><LI class="navbar-model"><A href="https://2018.igem.org/Team:UI_Indonesia/Model">Model</A></LI><LI class="dropdown navbar-humanpractice"><A href="https://2018.igem.org/Team:UI_Indonesia/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Human_Practices">Integrated Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Human_Practices#catalogue">Human Practice Catalogue</A></LI><LI class="navbar-improve"><A href="https://2018.igem.org/Team:UI_Indonesia/Improve">Improve</A></LI><LI class="navbar-team"><A href="https://2018.igem.org/Team:UI_Indonesia/Team">Team</A></LI><LI class="navbar-collaborations"><A href="https://2018.igem.org/Team:UI_Indonesia/Collaborations">Collaborations</A></LI><LI class="dropdown navbar-attributions"><A href="https://2018.igem.org/Team:UI_Indonesia/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/MedalCriteria">Medal Criteria</A></LI></DIV></DIV></DIV><DIV class="bgimg-3 w3-display-container w3-opacity-min" id="affitoxin"><DIV class="w3-display-middle" style="white-space:nowrap;"><SPAN class="w3-center w3-padding-large w3-black w3-xlarge w3-wide w3-animate-opacity">DIPHTOX</SPAN></DIV></DIV><DIV class="w3-content w3-container w3-padding-64"><H5>Using the <I>wild-type</I> diphtheria exotoxin to characterize HB-EGF/Tar receptor could be harmful due to biosafety reason. To tackle this problem, our team design a much simplified diphtheria toxin by removing the domain that is deadly to the cell. This domain, named domain C, will be translocated into the cell with the aid of other domains in toxin called domain R and domain T. Additionally, R domain recognized the natural HBEGF receptor, and T domain will insert the C domain into the cell. Thus, C domain would catalyze NAD-dependent ADP-ribosylation of EF-2 and leads to cellular apoptosis<SUP>1</SUP>. This remodeled toxin, coined DiphTox, is incorporated in the plasmids (such as pSB1C3 and pEQ80L) along with the following parts.</H5><DIV class="w3-row w3-center"><H6><B>Figure 1.</B> DiphTox Biobrick.</H6></DIV><H5>Amplification of single biobrick are done via PCR cloning using ‘self-made’ universal forward (Primer Cloning Fwd) and reverse primers (Primer Cloning Rev). The following sequences are the primers for PCR cloning.</H5><DIV class="w3-row w3-center"><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence</TH><TH>Melting Temperatures (Tm)</TH><TH>GC Content (%)</TH><TH>Hairpin Structure Energy Formation (kcal/mol)</TH><TH>Self-dimer Energy Formation (kcal/mol)</TH></TR><TR><TD>Fwd Cloning</TD><TD>5’ AGTTCAAGTGTCCGAGAA 3’</TD><TD>60.2<SUP>0</SUP>C</TD><TD>44.4%</TD><TD>0.64</TD><TD>-3.61</TD></TR><TR><TD>Rev Cloning</TD><TD>5’ TAAGCGAGTGCCGTATTA 3’</TD><TD>60.1<SUP>0</SUP>C</TD><TD>44.4%</TD><TD>-1.36</TD><TD>-3.61</TD></TR></TBODY></TABLE><H6><B>Table 1.</B> PCR Cloning Amplification Primers</H6></DIV><H5>Note: Both heterodimer energy formations are -3.61 kcal/mol. Additionally, the predicted specifications are based on <I>IDT Oligo Analyzer 3.1</I><A href="https://sg.idtdna.com/calc/analyzer" style="color:blue">server.</A> PCR solution is predicted to contain 0.2 mM dNTP, 50 mM Na<SUP>+</SUP>, 5 mM Mg<SUP>2+</SUP>, and 0.3 mM oligos.
  </H5><H5>Confirmation of gBlocks insertion into plasmid could be confirmed using PCR colony method using following primer. Note that HbT1 Fwd and HbT2 Rev is the same primer used in HT fragments to do in DipthTox gBlocks.</H5><DIV class="w3-row w3-center"><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence</TH><TH>Melting Temperatures (Tm)</TH><TH>GC Content (%)</TH><TH>Hairpin Structure Energy Formation (kcal/mol)</TH><TH>Self-dimer Energy Formation (kcal/mol)</TH></TR><TR><TD>HbT1 Fwd</TD><TD>5’ CTTACGCTTCTGCCACAT 3’</TD><TD>61.7<SUP>0</SUP>C</TD><TD>50%</TD><TD>-0.84</TD><TD>-3.61</TD></TR><TR><TD>HbT2 Rev</TD><TD>5’ CCCCTGACTGAGCATGAT 3’</TD><TD>62.8<SUP>0</SUP>C</TD><TD>55.6%</TD><TD>0.57</TD><TD>-5.38</TD></TR></TBODY></TABLE><H6><B>Table 2.</B> Detailed data for DiphTox PCR colony primers</H6></DIV><H5>Note: Both heterodimer energy formation exhibits -5.04 kcal/mol free energy.</H5><H5>For iGEM biobrick submission, we would use prefix and suffix that contain EcoRI and PstI. Further characterization of the biobrick of DiphTox could be accessed via <A href="http://parts.igem.org/Part:BBa_K2607000" style="color:blue">http://parts.igem.org/Part:BBa_K2607000</A>. HindIII site and BamHI are inserted upstream from prefix and downstream from suffix respectively. The DiphTox contains only the last 54 amino acid from the R domain which has no cytotoxicity and significant enough to bind with HBEGF receptor<SUP>2</SUP>. At the downstream of the sequence, designing six histidine amino acids is essential for His-tag protein purification, as well as characterizing the DiphTox and HB-EGF/Tar receptor kinetics. Ribosome Binding Site (RBS) is located upstream within the gBlocks containing Shine-Dalgarno sequence as follow.
  RBS : 5' GAGCGGATTATATAAGGAGGTTAATC 3’
  </H5></DIV><DIV class="bgimg-3 w3-display-container w3-opacity-min" id="hbegftar"><DIV class="w3-display-middle" style="white-space:nowrap;"><SPAN class="w3-center w3-padding-large w3-black w3-xlarge w3-wide w3-animate-opacity">HB-EGF/Tar</SPAN></DIV></DIV><DIV class="w3-content w3-container w3-padding-64"><H5>HB-EGF/Tar is a synthetically combined receptor used as project’s diagnostic tool system. Since there was limitation in its biobrick construction (for its high DNA complexity and base pairs length as <I>gBlocks</I>), we separate the HB-EGF/Tar sequence into two fragments called HB-EGF/Tar Fragment 1 (HT-1) in the upstream, and the other one would be HB-EGF/Tar Fragment 2 (HT-2). The schematic structure of HT-1 biobrick is as follow.</H5><DIV class="w3-row w3-center"><H6><B>Figure 2.</B> HB-EGF/Tar Fragment 1 Biobrick.</H6></DIV><H5>The HT-1 fragment consist of 981 bp sequence that include <I>SalI</I> cutting site and specified PCR colony reverse primer (HbT1 rev). Amplification of single biobrick are done via PCR cloning using ‘self-made’ universal forward (Primer Cloning Fwd) and reverse primers (Primer Cloning Rev), which are the same as cloning primer for DiphTox gBlocks.</H5><H5>For confirmation of this biobrick existence inside <I>E. coli</I>, PCR colony is essential, using specific primers named “PCR colony HbT1 fwd” for forward primer and “PCR colony HbT1 rev” for reverse primer. They are constructed using NCBI primer BLAST <A href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/" style="color:blue">website.</A> The sequences are served as follows.</H5><DIV class="w3-row w3-center"><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence</TH><TH>Melting Temperatures (Tm)</TH><TH>GC Content (%)</TH><TH>Hairpin Structure Energy Formation (kcal/mol)</TH><TH>Self-dimer Energy Formation (kcal/mol)</TH></TR><TR><TD>HbT1 Fwd</TD><TD>5’ CTTACGCTTCTGCCACAT 3’</TD><TD>61.7<SUP>0</SUP>C</TD><TD>50%</TD><TD>-0.84</TD><TD>-3.61</TD></TR><TR><TD>HbT1 Rev</TD><TD>5’ TTCTTACGCTTCCCATGTTC 3’</TD><TD>62.1<SUP>0</SUP>C</TD><TD>45%</TD><TD>0.64</TD><TD>-5.38</TD></TR></TBODY></TABLE><H6><B>Table 3.</B> Detailed data for HT1 PCR colony primers</H6></DIV><H5>Note: Both heterodimer energy formations are -4.77 kcal/mol.</H5><H5>Biobrick RFC10 prefix is necessary and would be cut using EcoRI enzyme for submission. Active gene transcription includes highly expressing T7 constitutive promoter (Bba_I712074) located in the upstream part of HT-1. Ribosomal binding site (RBS) (Bba_B0034) functions to initiate direct HB-EGF/Tar translation (HbTf1-RBS). It is 18 bp in length with the following sequence, 5' AAAGAGGAGAAATACTAG 3’, conformed with Shine-Dalgarno sequence
  
  The next fragment of HB-EGF/Tar Sequence, called HB-EGF/Tar Fragment 2 (HT-2), will be ligated exactly next to the first one via SalI restriction site. Schematic representation of HB-EGF/Tar Fragment 2 is as follow.</H5><DIV class="w3-row w3-center"><H6><B>Figure 3.</B> HB-EGF/Tar Fragment 2 Biobrick.</H6></DIV><H5>To amplify this part, we would use universal PCR cloning forward and reverse primers with their sequences same as HT-1 and DiphTox gBlocks. PCR colony, with a specific primer named “HbT2 fwd” for forward primer and “HbT2 Rev” for reverse primer, could be used to confirm any successful transformation of ligated insert. Both primers are constructed with NCBI primer BLAST website and have the sequence detailed below.</H5><DIV class="w3-row w3-center"><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence</TH><TH>Melting Temperatures (Tm)</TH><TH>GC Content (%)</TH><TH>Hairpin Structure Energy Formation (kcal/mol)</TH><TH>Self-dimer Energy Formation (kcal/mol)</TH></TR><TR><TD>HbT2 Fwd</TD><TD>5’ GTTAGCGGTCTCAGAAATGG 3’</TD><TD>62.2<SUP>0</SUP>C</TD><TD>50%</TD><TD>-1.15</TD><TD>-3.61</TD></TR><TR><TD>HbT2 Rev</TD><TD>5’ CCCCTGACTGAGCATGAT 3’</TD><TD>62.8<SUP>0</SUP>C</TD><TD>55.6%</TD><TD>0.57</TD><TD>-5.38</TD></TR></TBODY></TABLE><H6><B>Table 4.</B> detailed data for HT2 PCR colony primers</H6></DIV><H5>Note: Both heterodimer energy formations are -6.73 kcal/mol.</H5><H5>In this fragment, SalI and NdeI are in upstream and downstream respectively. This arrangement would enable that HB-EGF/Tar frag 1 and HB-EGF/Tar frag 2 be next to each other and transcribed as a unity. At the end part of the coding region we add double terminator rrnB T1 and T7TE terminator (Bba_B0015). These terminators have been known to effectively terminate transcription of any gene. We shall submit the overall HB-EGF/Tar biobrick sequence by inserting iGEM registered RFC10 suffix that contain PstI restriction site at the downstream of the coding region. The prefix, as mentioned earlier, is inserted in the first fragment of the HB-EGF/Tar sequence. Further characterization of the biobrick of complete HB-EGF/Tar synthetic gene could be accessed via Further characterization of the biobrick of DiphTox could be accessed via <A href="http://parts.igem.org/Part:BBa_K2607001" style="color:blue">http://parts.igem.org/Part:BBa_K2607001</A>.
  </H5><H5><B>Reference</B><OL class="ref"><LI>Gillet D, Barbier J. Diphtheria toxin. The Comprehensive Sourcebook of Bacterial Protein Toxins. 2015;:111-132.</LI><LI>2.  Rolf J, Eidels L. Characterization of the diphtheria toxin receptor-binding domain. Molecular Microbiology. 1993;7(4):585-591.</LI></OL></H5></DIV><FOOTER class="w3-center w3-green w3-padding-64 w3-opacity w3-hover-opacity-off"><DIV class="w3-xlarge w3-section">Team UI Indonesia igemui2018@gmail.com
  </DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>