Team:UI Indonesia/Parts
Home
Project
Overview
Our Project
Results and Discussions
Parts
Safety
InterLab
Introduction
Materials and Equipment
Methods
Results and Discussions
Conclusions
Model
Human Practices
Integrated Human Practice
Education and Public Engagement
Human Practice Catalogue
Improve
Team
Collaborations
Attributions
Medal Criteria
DIPHTOX
Using the wild-type diphtheria exotoxin to characterize HB-EGF/Tar receptor could be harmful due to biosafety reason. To tackle this problem, our team design a much simplified diphtheria toxin by removing the domain that is deadly to the cell. This domain, named domain C, will be translocated into the cell with the aid of other domains in toxin called domain R and domain T. Additionally, R domain recognized the natural HBEGF receptor, and T domain will insert the C domain into the cell. Thus, C domain would catalyze NAD-dependent ADP-ribosylation of EF-2 and leads to cellular apoptosis1. This remodeled toxin, coined DiphTox, is incorporated in the plasmids (such as pSB1C3 and pEQ80L) along with the following parts.
Figure 1. DiphTox Biobrick.
Amplification of single biobrick are done via PCR cloning using ‘self-made’ universal forward (Primer Cloning Fwd) and reverse primers (Primer Cloning Rev). The following sequences are the primers for PCR cloning.
Primers
Sequence
Melting Temperatures (Tm)
GC Content (%)
Hairpin Structure Energy Formation (kcal/mol)
Self-dimer Energy Formation (kcal/mol)
Fwd Cloning
5’ AGTTCAAGTGTCCGAGAA 3’
60.20C
44.4%
0.64
-3.61
Rev Cloning
5’ TAAGCGAGTGCCGTATTA 3’
60.10C
44.4%
-1.36
-3.61
Table 1. PCR Cloning Amplification Primers
Note: Both heterodimer energy formations are -3.61 kcal/mol. Additionally, the predicted specifications are based on IDT Oligo Analyzer 3.1 server. PCR solution is predicted to contain 0.2 mM dNTP, 50 mM Na+, 5 mM Mg2+, and 0.3 mM oligos.
Confirmation of gBlocks insertion into plasmid could be confirmed using PCR colony method using following primer. Note that HbT1 Fwd and HbT2 Rev is the same primer used in HT fragments to do in DipthTox gBlocks.
Primers
Sequence
Melting Temperatures (Tm)
GC Content (%)
Hairpin Structure Energy Formation (kcal/mol)
Self-dimer Energy Formation (kcal/mol)
HbT1 Fwd
5’ CTTACGCTTCTGCCACAT 3’
61.70C
50%
-0.84
-3.61
HbT2 Rev
5’ CCCCTGACTGAGCATGAT 3’
62.80C
55.6%
0.57
-5.38
Table 2. Detailed data for DiphTox PCR colony primers
Note: Both heterodimer energy formation exhibits -5.04 kcal/mol free energy.
For iGEM biobrick submission, we would use prefix and suffix that contain EcoRI and PstI. Further characterization of the biobrick of DiphTox could be accessed via http://parts.igem.org/Part:BBa_K2607000 . HindIII site and BamHI are inserted upstream from prefix and downstream from suffix respectively. The DiphTox contains only the last 54 amino acid from the R domain which has no cytotoxicity and significant enough to bind with HBEGF receptor2. At the downstream of the sequence, designing six histidine amino acids is essential for His-tag protein purification, as well as characterizing the DiphTox and HB-EGF/Tar receptor kinetics. Ribosome Binding Site (RBS) is located upstream within the gBlocks containing Shine-Dalgarno sequence as follow.
RBS : 5' GAGCGGATTATATAAGGAGGTTAATC 3’
HB-EGF/Tar
HB-EGF/Tar is a synthetically combined receptor used as project’s diagnostic tool system. Since there was limitation in its biobrick construction (for its high DNA complexity and base pairs length as gBlocks), we separate the HB-EGF/Tar sequence into two fragments called HB-EGF/Tar Fragment 1 (HT-1) in the upstream, and the other one would be HB-EGF/Tar Fragment 2 (HT-2). The schematic structure of HT-1 biobrick is as follow.
Figure 2. HB-EGF/Tar Fragment 1 Biobrick.
The HT-1 fragment consist of 981 bp sequence that include SalI cutting site and specified PCR colony reverse primer (HbT1 rev). Amplification of single biobrick are done via PCR cloning using ‘self-made’ universal forward (Primer Cloning Fwd) and reverse primers (Primer Cloning Rev), which are the same as cloning primer for DiphTox gBlocks.
For confirmation of this biobrick existence inside E. coli, PCR colony is essential, using specific primers named “PCR colony HbT1 fwd” for forward primer and “PCR colony HbT1 rev” for reverse primer. They are constructed using NCBI primer BLAST website. The sequences are served as follows.
Primers
Sequence
Melting Temperatures (Tm)
GC Content (%)
Hairpin Structure Energy Formation (kcal/mol)
Self-dimer Energy Formation (kcal/mol)
HbT1 Fwd
5’ CTTACGCTTCTGCCACAT 3’
61.70C
50%
-0.84
-3.61
HbT1 Rev
5’ TTCTTACGCTTCCCATGTTC 3’
62.10C
45%
0.64
-5.38
Table 3. Detailed data for HT1 PCR colony primers
Note: Both heterodimer energy formations are -4.77 kcal/mol.
Biobrick RFC10 prefix is necessary and would be cut using EcoRI enzyme for submission. Active gene transcription includes highly expressing T7 constitutive promoter (Bba_I712074) located in the upstream part of HT-1. Ribosomal binding site (RBS) (Bba_B0034) functions to initiate direct HB-EGF/Tar translation (HbTf1-RBS). It is 18 bp in length with the following sequence, 5' AAAGAGGAGAAATACTAG 3’, conformed with Shine-Dalgarno sequence
The next fragment of HB-EGF/Tar Sequence, called HB-EGF/Tar Fragment 2 (HT-2), will be ligated exactly next to the first one via SalI restriction site. Schematic representation of HB-EGF/Tar Fragment 2 is as follow.
Figure 3. HB-EGF/Tar Fragment 2 Biobrick.
To amplify this part, we would use universal PCR cloning forward and reverse primers with their sequences same as HT-1 and DiphTox gBlocks. PCR colony, with a specific primer named “HbT2 fwd” for forward primer and “HbT2 Rev” for reverse primer, could be used to confirm any successful transformation of ligated insert. Both primers are constructed with NCBI primer BLAST website and have the sequence detailed below.
Primers
Sequence
Melting Temperatures (Tm)
GC Content (%)
Hairpin Structure Energy Formation (kcal/mol)
Self-dimer Energy Formation (kcal/mol)
HbT2 Fwd
5’ GTTAGCGGTCTCAGAAATGG 3’
62.20C
50%
-1.15
-3.61
HbT2 Rev
5’ CCCCTGACTGAGCATGAT 3’
62.80C
55.6%
0.57
-5.38
Table 4. detailed data for HT2 PCR colony primers
Note: Both heterodimer energy formations are -6.73 kcal/mol.
In this fragment, SalI and NdeI are in upstream and downstream respectively. This arrangement would enable that HB-EGF/Tar frag 1 and HB-EGF/Tar frag 2 be next to each other and transcribed as a unity. At the end part of the coding region we add double terminator rrnB T1 and T7TE terminator (Bba_B0015). These terminators have been known to effectively terminate transcription of any gene. We shall submit the overall HB-EGF/Tar biobrick sequence by inserting iGEM registered RFC10 suffix that contain PstI restriction site at the downstream of the coding region. The prefix, as mentioned earlier, is inserted in the first fragment of the HB-EGF/Tar sequence. Further characterization of the biobrick of complete HB-EGF/Tar synthetic gene could be accessed via Further characterization of the biobrick of DiphTox could be accessed via http://parts.igem.org/Part:BBa_K2607001 .
Reference
Gillet D, Barbier J. Diphtheria toxin. The Comprehensive Sourcebook of Bacterial Protein Toxins. 2015;:111-132.
2.  Rolf J, Eidels L. Characterization of the diphtheria toxin receptor-binding domain. Molecular Microbiology. 1993;7(4):585-591.
Team UI Indonesia
igemui2018@gmail.com
