<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UI_Indonesia_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UI Indonesia/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-inner"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2018.igem.org/Team:UI_Indonesia">Home</A></DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown navbar-project"><A href="https://2018.igem.org/Team:UI_Indonesia/Project#">Project</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Project#">Overview</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Project#ourproject">Our Project</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Project#resultsanddiscuccions">Results and Discussions</A></LI></UL><LI class="navbar-parts"><A href="https://2018.igem.org/Team:UI_Indonesia/Parts">Parts</A></LI><LI class="navbar-safety"><A href="https://2018.igem.org/Team:UI_Indonesia/Safety">Safety</A></LI><LI class="dropdown navbar-interlab"><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#intro">Introduction</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#materials">Materials and Equipment</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#methods">Methods</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#results">Results and Discussions</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#conclusions">Conclusions</A></LI><LI class="navbar-model current"><A href="https://2018.igem.org/Team:UI_Indonesia/Model">Model</A></LI><LI class="dropdown navbar-humanpractice"><A href="https://2018.igem.org/Team:UI_Indonesia/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Human_Practices">Integrated Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Human_Practices#catalogue">Human Practice Catalogue</A></LI><LI class="navbar-improve"><A href="https://2018.igem.org/Team:UI_Indonesia/Improve">Improve</A></LI><LI class="navbar-team"><A href="https://2018.igem.org/Team:UI_Indonesia/Team">Team</A></LI><LI class="navbar-collaborations"><A href="https://2018.igem.org/Team:UI_Indonesia/Collaborations">Collaborations</A></LI><LI class="navbar-attributions"><A href="https://2018.igem.org/Team:UI_Indonesia/Attributions">Attributions</A></LI></DIV></DIV></DIV><DIV class="bgimg-1 w3-display-container w3-opacity-min" id="intro"><DIV class="w3-display-middle" style="white-space:nowrap;"><SPAN class="w3-center w3-padding-large w3-black w3-xlarge w3-wide w3-animate-opacity">MODELLING</SPAN></DIV></DIV><H1 align="center"> Structural Modelling </H1><H3 align="center"> Chimera Combination </H3><DIV class="w3-content w3-container w3-padding-64"><H5>Our first steps in modelling the subsequent parts of <I>Finding Diphthy</I> iGEM-UI 2018 <I>in silico</I> is by 
  constructing all 3D models via <I>I-Tasser</I> server.<SUP>1,2,3</SUP> The extension of the product file is <I>.pdb</I>, that 
  could be read by the server. The chimera molecules which we need to predict their modelling are HB-EGF/TAR 
  (<I>Heparin Binding Epidermal Growth Factor-</I> TAR chemotaxis), CheA signalling protein, Che-Y signalling protein, 
  LuxAB dimerized luciferase subunits, and eYFP (<I>enhanced yellow fluorescent protein</I>), as well as DiphTox
  (modified diphtheria exotoxin). Since CheA and CheY are required to be linked with LuxB or eYFP, 
  we have cited one of the universal linker, that is ‘GGGSGGGGSGGGGSG’ peptides, according to <I>Sun S et al</I>.
  
Our signalling part of the project is referred these sequences of all chimera combinations. 
<UL><LI>LuxB-CheY                                             
</LI><LI>LuxB-CheA 
</LI><LI>CheY-eYFP 
</LI><LI>CheA-eYFP
</LI></UL></H5><H5>In choosing the best combination, we use <I>FoldX</I> option via <I>YASARA molecules viewer</I>
  to calculate the ∆G of each molecule, searching for the smallest free energy 
  (regarding its stability in <I>vivo</I>). All those sequences are also submitted to 
  <I>I-Tasser</I> server for projecting their 3D models qualitatively. The following 
  results would conclude that our <I>cytoplasmic</I> signalling combinations are 
  CheY-eYFP and LuxB-CheA. </H5><DIV align="center"><H6><B>Table 1.</B> Specific Gibbs Energy within Each Protein Combination.</H6><TABLE><TBODY><TR><TH width="120px">Combination</TH><TH width="120px"><P align="center">∆G</P></TH></TR><TR><TD><B>LuxB-CheY</B></TD><TD><P align="right">58.15 kcal/mol</P></TD></TR><TR><TD><B>LuxB-CheA</B></TD><TD><P align="right">1355.46 kcal/mol</P></TD></TR><TR><TD><B>CheY-eYFP</B></TD><TD><P align="right">36.48 kcal/mol</P></TD></TR><TR><TD><B>CheA-eYFP</B></TD><TD><P align="right">36.48 kcal/mol</P></TD></TR></TBODY></TABLE></DIV><H5>Characterisation or purification of those proteins would promote the usage 
  of <I>His-tag</I>; therefore, insertion of His-tag inside the sequence is essential. 
  To ensure the slightest change of tertiary structures of each protein, 
  we would need to find out the secondary structure and surface accesibility 
  via <I>NetSurfP ver. 1.1</I> analyser <A href="http://www.cbs.dtu.dk/services/NetSurfP/" style="color:blue">http://www.cbs.dtu.dk/services/NetSurfP/</A>

  We would insert <I>His-tag</I> sequence in either no available specific protein 
  domain or the coiled secondary structure of protein to minimize any 
  interruptions. Here is our DiphTox data from <I>NetSurfP</I> server.Result from the <I>NetSurf server</I>, we choose C-terminus side, 
  because it most likely turns/coils around (indicated by 
  has high number on the most right column is closest to 1), 
  and it is freely exposed (indicated by most left column has E alphabet)</H5><DIV align="center"><H6><B>Table 2.</B> Coiling probability of DipThox’s specific domain.</H6><TABLE><TBODY><TR><TH>Class assignment</TH><TH>Amino acid</TH><TH><P align="right">Amino acidnumber</P></TH><TH><P align="right">Probabilityfor Coil</P></TH></TR><TR><TD><B>B</B></TD><TD>I</TD><TD><P align="right">54</P></TD><TD><P align="right">0.223</P></TD></TR><TR><TD><B>E</B></TD><TD>K</TD><TD><P align="right">55</P></TD><TD><P align="right">0.669</P></TD></TR><TR><TD><B>E</B></TD><TD>S</TD><TD><P align="right">56</P></TD><TD><P align="right">0.994</P></TD></TR></TBODY></TABLE></DIV><H5>Performing structural similarity between original molecule and 
  the one inserted with <I>His-tag</I> sequence have been done by <I>MUSTANG</I> 
  server that built in via <I>YASARA molecule viewer.<SUP>5</SUP></I><SUP><SUP> The output would be 
  distance calculation between interacting atoms called RMSD 
  (Root-mean-square deviation). Following tables are summaries of the 
  molecular similarity analysis. From the data that described above, all the combinations are acceptable,
  except LuxA, since its possible combination has high RMSD. The threshold 
  is relative, but several literatures define the RMSD value of 2 as 
  threshold for structure similarity.<SUP>5,6,7</SUP></SUP></SUP></H5><DIV align="center"><H6><B>Table 1.</B> RMSD Calculation within Several Protein Linked with His-tag.</H6><TABLE><TBODY><TR><TH width="300px"><P align="center">Similarities between</P></TH><TH width="120px"><P align="center">RMSD</P></TH></TR><TR><TD><B>LuxA with LuxA + His</B></TD><TD>2.203 Å</TD></TR><TR><TD><B>LuxC with LuxC + His</B></TD><TD>0.985 Å</TD></TR><TR><TD><B>LuxD with LuxD + His</B></TD><TD>0.1777 Å</TD></TR><TR><TD><B>LuxE with LuxE + His</B></TD><TD>0.800 </TD></TR><TR><TD><B>CheY-eYFP with CheY-eYFP+his</B></TD><TD>0.108 Å</TD></TR><TR><TD><B>eYFP with eYFP + His</B></TD><TD>0.315 Å</TD></TR><TR><TD><B>CheA with CheA + His</B></TD><TD>0.134 Å</TD></TR></TBODY></TABLE></DIV><H3> HB-EGF/Tar Receptor Modelling </H3><H5>In HB-EGF, the part that serves as binding domain for diphtheria exotoxin 
  predominantly located in the extracellular environment. Therefore, 
  the domain, expands between 20<SUP>th</SUP> – 160<SUP>th</SUP> amino acid, was selected from 
  natural HB-EGF protein. On the other hand, the Tar domain that are 
  functions to establish intracellular chemotactic signalling includes 
  NdeI cutting-site (around 257<SUP>th</SUP> amino acid) until the utmost C-terminal 
  of the protein (the 553<SUP>rd</SUP> amino acid).8-11 By those factors, our team also 
  selected Tar domains involving the 1st – 33<SUP>rd</SUP> and 191<SUP>st</SUP> – 
  553<SUP>rd</SUP> amino acid as part of chimeric protein.</H5><DIV align="center"><H6><B>Figure 1.</B> The selected segment of Tar protein. The functional 
  intracellular domain of Tar is shown as yellow box, blue box is 
  transmembrane domain and orange box is periplasmic domain. Selected Tar 
  domain expands from 1st -33<SUP>rd</SUP> amino acids and 191<SUP>st</SUP> -553<SUP>rd</SUP> amino acids. 
  Modification of binding domain is located between 33<SUP>rd</SUP> – 191<SUP>st</SUP> amino acids</H6></DIV><H5>Our team have predicted the HB-EGF/Tar protein orientation in the 
  <I>Escherichia coli</I> membrane. For this purpose, server <I>TMHMM</I> and <I>OPM Membrane</I>, 
  are utilized to predict protein orientation.<SUP>12,13</SUP> Conceptual hypothesis 
  about the chimera protein is that it should begin its orientation of 
  C-terminus in cytoplasm, then continued to fold into transmembrane and 
  extracellular sites, as well as re-folding towards cytoplasm.</H5><H6><B>Figure 2.</B> The graph above explains the result of HB-EGF/Tar 
  orientation, which began from C-terminus (left) to N-terminus (right).<SUP>12</SUP> 
  Y-axis pictured the possibility of n<SUP>th</SUP> amino acid on protein located somewhere 
  between transmembrane (red part), intracellular (blue line), and 
  extracellular (pink line). There is also a diagram located above the graph 
  that represent the most possible location of each domain (with elongated box).</H6><H5>From the results, it could be concluded that the protein was oriented 
  as expected in the hypothesis. Therefore, the usage of chimera protein is 
  predicted to be functional anatomically. </H5><H6><B>Figure 3.</B>Molecular comparation of HB-EGF native protein (left) 
  with the HB-EGF/Tar fusion (right).<SUP>13,14</SUP> The pink-coloured 
  domain is intracellularly located as the N-terminus, yellow-coloured 
  domain for the transmembrane one. Then, purple-coloured could be a sign 
  as the extracellular domain, finally folding into transmembrane and back 
  to cytoplasm with orange-coloured and cyan-coloured domain respectively.</H6><H6><B>Figure 4.</B>3d models of HBEGF-TAR chimera on membrane.</H6><H5>After deciding sequence combination of amino acids in modelled chimera 
  HB-EGF/Tar protein, analyzing the interaction of both fusion protein and 
  diphtheria exotoxin <B>(Figure 5)</B> is extremely important to ensure functional 
  ligand-receptor system. The basic concept of interaction modelling is 
  that the protein will be bound to each other well if it causes the 
  ‘environment’ energy (termed by E parameter; calculated by formula below)
  being lowered down. In this part, our team sent the respective sequence to 
  ClusPro website for further analyzing.<SUP>15</SUP></H5><H6><B>Figure 5.</B> 3d model of proposed diphtox</H6><DIV><H4 align="center">
  E = 0.4E<SUB>rep</SUB> + -0.40E<SUB>att</SUB> + 600E<SUB>elec</SUB> + 1.00E<SUB>DARS</SUB></H4></DIV><H6>Note: E<SUB>rep</SUB> and E<SUB>attr</SUB> denote as repulsive and attractive contributions 
  to the <I>van der Waals</I> interaction energy. Additionally, E<SUB>elec</SUB> means an 
  electrostatic energy that occur during both protein interaction. E<SUB>DARS</SUB> 
  is a pairwise structure-based potential constructed by the Decoys of 
  the Reference State (DARS) approach, and it primarily represents 
  desolvation contributions, i.e., the free energy change due to the 
  removal of the water molecules from the interface.<SUP>15</SUP></H6><H6><B>Table 4.</B> Comparation of E parameter of native and chimera protein of HB-EGF interacted with DiphTox.</H6><TABLE><TBODY><TR><TH><P align="center">HB-EGFProtein</P></TH><TH><P align="center">Median Energy (kcal/mol)</P></TH><TH><P align="center">Lowest Energy (kcal/mol)</P></TH></TR><TR><TD><B>Native</B></TD><TD><P align="center">-944.3</P></TD><TD><P align="center">-994.3</P></TD></TR><TR><TD><B>Chimera</B></TD><TD><P align="center">858.2</P></TD><TD><P align="center">934.4</P></TD></TR></TBODY></TABLE></DIV><H6><B>Figure 6.</B> HB-EGF/Tar receptor-DiphTox 3D interaction modelling result.</H6><H6><B>Figure 7.</B>HB-EGF natural receptor and DiphTox 3D interaction modelling result.</H6><H5>The result of interaction modelling is quantified as energy score based on 
  the formula above. Referring to <B>figure 6</B> and <B>7</B>, we might expect that the 
  DiphTox (cyan) would bind to both native and chimeric HB-EGF receptor that 
  are both located in the extracellular (green). It is indicated by higher 
  energy score of interaction between chimeric HB-EGF/Tar receptor-DiphTox
  than that of to HB-EGF natural receptor-DiphTox (<B>Table 4</B>). This means 
  that the chimeric receptor could bind towards DiphTox as good 
  (or even better) than the original one. </H5><H5>Beside the cell’s ability to detect toxin, our team also need to ensure 
  the signaling machine works well. Our team also modelled the interaction 
  between LuxA dan LuxB (that we fused with CheA). From <B>figure 8</B> and <B>9</B>, 
  we might expect that both proteins are still able to interact normally 
  after combining them with FRET unit (CheA or CheY protein). </H5><DIV align="center"><H6><B>Table 5.</B> Comparation of E parameter of native and His-tagged protein of LuxB-CheA.</H6><TABLE><TBODY><TR><TH><P align="center">LuxAB</P></TH><TH><P align="center">Median Energy (kcal/mol)</P></TH><TH><P align="center">Lowest Energy (kcal/mol)</P></TH></TR><TR><TD><B>Native</B></TD><TD><P align="center">-1515.4.3</P></TD><TD><P align="center">-1553.2</P></TD></TR><TR><TD><B>Chimera</B></TD><TD><P align="center">-1220.9</P></TD><TD><P align="center">-1290.7</P></TD></TR></TBODY></TABLE></DIV><H6><B>Figure 8.</B>LuxA and LuxB-CheA (LuxAB-CheA) 3D interaction modelling result.</H6><H6><B>Figure 9.</B>.LuxA and LuxB 3D interaction modelling result.</H6><H5><B>Reference:</B></H5><OL align="justify"><LI>J Yang, R Yan, A Roy, D Xu, J Poisson, Y Zhang. The I-TASSER Suite: Protein structure and function prediction. Nature Methods, 12: 7-8 (2015)</LI><LI>A Roy, A Kucukural, Y Zhang. I-TASSER: a unified platform for automated protein structure and function prediction. Nature Protocols, 5: 725-738 (2010)</LI><LI>Y Zhang. I-TASSER server for protein 3D structure prediction. BMC Bioinformatics, vol 9, 40 (2008)</LI><LI>Sun, S., Yang, X., Wang, Y., Shen, X., 2016. In Vivo Analysis of Protein–Protein Interactions with Bioluminescence Resonance Energy Transfer (BRET): Progress and Prospects. International Journal of Molecular Sciences 17, 1704. https://doi.org/10.3390/ijms17101704</LI><LI>MUSTANG: A multiple structural alignment algorithm Konagurthu AS, Whisstock JC, Stuckey PJ, Lesk AM (2006) Proteins 64,559-574</LI><LI>Bordogna, A., Pandini, A., Bonati, L., 2010. Predicting the accuracy of protein-ligand docking on homology models. Journal of Computational Chemistry 32, 81–98. https://doi.org/10.1002/jcc.21601</LI><LI>Carugo, O., 2003. How root-mean-square distance (r.m.s.d.) values depend on the resolution of protein structures that are compared. Journal of Applied Crystallography 36, 125–128. https://doi.org/10.1107/s0021889802020502</LI><LI>Kanchan, K., Linder, J., Winkler, K., Hantke, K., Schultz, A. and Schultz, J. (2009). Transmembrane Signaling in Chimeras of the Escherichia coli Aspartate and Serine Chemotaxis Receptors and Bacterial Class III Adenylyl Cyclases. <I>Journal of Biological Chemistry</I>, 285(3), pp.2090-2099.</LI><LI>Ward, S., Delgado, A., Gunsalus, R. and Manson, M. (2002). A NarX-Tar chimera mediates repellent chemotaxis to nitrate and nitrite. <I>Molecular Microbiology</I>, 44(3), pp.709-719.</LI><LI>Melchers, L. S., Regensburg-Tuïnk, T. J., Bourret, R. B., Sedee, N. J., Schilperoort, R. A. and Hooykaas, P. J. (1989). Membrane topology and functional analysis of the sensory protein VirA of Agrobacterium tumefaciens. The <I>EMBO Journal</I>, 8(7), pp.1919-1925.</LI><LI>Weerasuriya, S., Schneider, B. M. and Manson, M. D. (1998). Chimeric Chemoreceptors in <I>Escherichia coli</I>: Signaling Properties of Tar-Tap and Tap-Tar Hybrids. <I>Journal of Bacteriology</I>, 180(4), pp.914-920.</LI><LI>Cbs.dtu.dk. (2018). <I>TMHMM Server</I>, v. 2.0. [online] Available at: <A href="http://www.cbs.dtu.dk/services/TMHMM/" style="color:blue">http://www.cbs.dtu.dk/services/TMHMM/</A> [Accessed 22 Jul. 2018].</LI><LI>Lomize M.A., Pogozheva I,D, Joo H., Mosberg H.I., Lomize A.L. OPM database and PPM web server: resources for positioning of proteins in membranes. Nucleic Acids Res., 2012, 40(Database issue):D370-6</LI><LI>Kelley LA et al. (2015). The Phyre2 web portal for protein modeling, prediction and analysis. <I>Nature Protocols</I> 10, pp.845-858 </LI><LI>Kozakov D, Hall DR, Xia B, Porter KA, Padhorny D, Yueh C, Beglov D, Vajda S. The ClusPro web server for protein-protein docking. <I>Nature Protocols</I>.2017 Feb;12(2):255-278 ; pdf </LI><LI>Kozakov D, Beglov D, Bohnuud T, Mottarella S, Xia B, Hall DR, Vajda, S. How good is automated protein docking? <I>Proteins: Structure, Function, and Bioinformatics</I>, 2013 Aug ;  pdf </LI><LI>Kozakov D, Brenke R, Comeau SR, Vajda S. PIPER: An FFT-based protein docking program with pairwise potentials. <I>Proteins</I>. 2006 Aug 24;  pdf </LI><LI>Comeau SR, Gatchell DW, Vajda S, Camacho CJ. ClusPro: an automated docking and discrimination method for the prediction of protein complexes.<I>Bioinformatics</I>. 2004 Jan 1;  pdf </LI><LI>Comeau SR, Gatchell DW, Vajda S, Camacho CJ. ClusPro: a fully automated algorithm for protein-protein docking <I>Nucleic Acids Research</I>. 2004 Jul 1;  pdf </LI><LI>Högbom, M., Eklund, M., Nygren, P. Å., &amp; Nordlund, P. (2003). Structural basis for recognition by an in vitro evolved affibody. Proceedings of the National Academy of Sciences, 100(6), 3191-3196.</LI><LI>2015.igem.org. (2018). <I>Team:Stockholm/Description - 2015.igem.org</I>. [online] Available at: https://2015.igem.org/Team:Stockholm/Description [Accessed 22 Jul. 2018].</LI></OL></DIV><DIV class="w3-content w3-container w3-padding-64"><H1 align="center"> Diphteria Toxin </H1><H5>To ensure the DiphTox has no toxicity, we performed literature search. First, we need to know every domain that construct the whole Diphteria Toxin (DT). We were using online databases such as Uniprot (<A href="https://www.uniprot.org/uniprot/Q5PY51" style="color:blue">https://www.uniprot.org/uniprot/Q5PY51</A>) and Interpro (<A href="https://www.ebi.ac.uk/interpro/protein/Q5PY51" style="color:blue">https://www.ebi.ac.uk/interpro/protein/Q5PY51</A>).[1,2] We found that DT consists of three domains (Figure 1), which each of them has specific function.[3]</H5><H6><B>Figure 1.</B>.Diphteria toxin domain.</H6><H5>DT is classified into AB toxin which A fragment consists of C (catalytic) domain and B fragment consists of T (translocation) and R (recognition) domain (<B>Figure 2</B>).[3]

</H5><H6><B>Figure 2.</B>Diphteria toxin as AB toxin.</H6><H5>To cause cell intoxication, first DT binds to proHBEGF on surface membrane. EGF like domain on proHBEGF receptor binds the R domain of DT. Receptor bound toxin is concentrated in clathrin coated pits and internalized into clathrin coated vesicle. With the increase of vacuolar ATPase activity, the pH inside vesicle is decreased. The decrease of pH will change the conformational structure of DT, especially T domain. T domain function is to translocate C domain to cytosol. There are many hypotheses about the mechanism of C domain translocation from lumen to cytosolic space. After C domain is delivered to cytosol, it undergoes structural change into enzymatically active conformation and catalyzes NADdependent ADP ribosilation of EF-2, leading to cellular death (apoptosis) via protein synthesis inhibition. One publication said that only A substance can lead to cellular death. The mechanism of intoxication is summarized in <B>figure 3.</B>[4,5]
</H5><H6><B>Figure 3.</B>. Intoxication of DT.</H6><H5>From this theory, we conclude that A fragment or C domain is the main cause of cytotoxic activity. T and R domain function is only for C domain delivery into cytosol. In order to maximize the safety, we want to take the toxin as small as we can but still retain the binding activity of DT. Publication by John M. Rolf and Leon Eidels said that last 54 amino acids (482-
535) are sufficient enough to make HT and HBEGF bind. These 54 amino acids is part of R domain which has no cytotoxicity effect.[6]
</H5><H5><B>Reference</B><OL><LI><A href="https://www.uniprot.org/uniprot/Q5PY51" style="color:blue">https://www.uniprot.org/uniprot/Q5PY51</A></LI><LI><A href="https://www.ebi.ac.uk/interpro/protein/Q5PY51" style="color:blue">https://www.ebi.ac.uk/interpro/protein/Q5PY51</A></LI><LI> Gillet D, Barbier J. Diphtheria toxin. The Comprehensive Sourcebook of Bacterial Protein Toxins. 2015;:111-132.
</LI><LI> Murphy J. Mechanism of Diphtheria Toxin Catalytic Domain Delivery to the Eukaryotic Cell Cytosol and the Cellular Factors that Directly Participate in the Process. Toxins. 2011;3(3):294-308.
</LI><LI> Yamaizumi, M.; Mekada, E.; Uchida, T.; Okada, Y. One molecule of diphtheria toxin fragment A introduced into a cell can kill the cell. Cell 1978, 15, 245–250.
</LI><LI> Rolf J, Eidels L. Characterization of the diphtheria toxin receptor-binding domain.Molecular Microbiology. 1993;7(4):585-591.
</LI></OL></H5><H1 align="center"> Kinetical Modelling </H1><DIV class="w3-content w3-container w3-padding-64"><H5>In synthesizing any designed system of engineered protein, one must able to oversee and predict regarding its performance and behaviour. In analysing the protein, one can apply principles of mathematics and engineering in doing so. Reaction kinetics study can be largely described as the study of reactions regarding their performance such as rates, effects of various variables, etc. In this report, the kinetics of the designed protein will be evaluated based on existing literature. Mass transportation principle, such as Langmuir-Hill or Michaelis-Menten equation, will be loosely applied and adequately correlated for the resulting models. Mathematical models and non-linear regression will play roles for the protein. Out team hopes that this mathematical modelling analysis for the protein will shed a new perspective for the synthesized protein’s function and enlighten some parts of our project.</H5><H5><B>Qualitative Overview</B></H5><DIV align="center"><H6><B>Figure 1.</B> The figure above describes the event of diphtheria toxin initial binding to the HB-EGF receptors in human epithelial cells. Attached toxin will inhibit protein synthesis and result in apoptosis.</H6></DIV><DIV align="center"><H6><B>Figure 2.</B> The figure above describes the difference between the chimeric protein performance’s regarding of its reaction with diphtheria toxin.</H6></DIV><H5>Thus, in overseeing the entire performance, it can be divided into three parts:</H5><UL><LI>HB-EGF binding to the diphtheria toxin</LI><LI>TAR protein’s binding performance</LI><LI>Reaction rates of the auto-phosphorylation process</LI></UL><H5><B>Quantitative Overview</B></H5><H5>In calculating reaction rates of either association or dissociation rate, the following mathematical equations are applied:</H5><H5><I>The surface concentration Gamma (G), for a protein can be calculated by the following formula (3): One Response Unit (RU) in the Blacore 2000/3000 machine corresponds to a surface coverage of 10<SUP>-6</SUP> g m<SUP>-2</SUP> for a typical protein. A 100 kDa protein generating a response of 1000 RU, corresponds to a surface coverage of 10<SUP>-8</SUP> moles m<SUP>-2</SUP>.</I></H5><H5><B>Formulas:</B></H5><H5>Response unit, which is the symbol for reaction rates’ response that are given directly by the response machine will first be converted into gamma (surface concentration), which will then be converted into concentration of complex with the following formula:</H5><H5>Therefore, reaction rates can be calculated with following formula:</H5><H5>The formula is synthesized by following the mass-transportation-based reaction rates, which is based on chemical kinetics of the reaction’s constituent.</H5><H5><B>HB-EGF Receptor Activity</B></H5><H5><I>Part I: Association and Dissociation Rates Based on Time</I></H5><H5>This part of analysis is mainly referenced to an article by Brooke et.al. [1], in which they model binding of immobilized HB-EGF with diphtheria toxin response unit versus time, which will be shown below:</H5><DIV align="center"><H6><B>Figure 3.</B>Binding of DT to immobilized hHB-EGF. The first arrow represents the start of the injection of DT over the immobilized hHB-EGF. The arrow with a ball represents the end of the DT injection and the beginning of the flow running buffer. From the bottom curve to the top curve, the respective binding curves for DT of 400 nM, 500 nM, 600 nM, 700 nM, 800 nM and 1 μM concentrations are shown. The number ①, ②, and ③ represent the baseline of running buffer, the association phase of the binding of DT to hHB-EGF, and the dissociation phase of the release of DT from hHB-EGF, respectively, RU, resonance units. This figure shows a representative experiment (use Materials and Methods).</H6></DIV><H5>Therefore, the prediction of both association rates and dissociation rates are shown below, by comparing our chimeric protein’s kinetics with literature, the x axis represents time, while the y axis represents response in Response Unit (RU).</H5><H5><OL><LI>Association Rates</LI></OL></H5><H6><B>Table 1.</B>HB-EGF Receptor Properties</H6><H6><B>Table 2.</B>Calculation Table of Diphtheria Toxin and HB-EGF Association Rate</H6><H5>Therefore, the data regression will be plotted as below:</H5><H6><B>Figure 4.</B>Association Response of Diphtheria Toxin towards HB-EGF Receptor.</H6><H6><B>Figure 5.</B>Reaction Rate of Diphtheria Toxin towards HB-EGF Receptor.</H6></DIV><LI>Dissociation Rates</LI><DIV align="center"><H6><B>Table 3.</B>HB-EGF Receptor Properties</H6><H6><B>Table 4.</B>Calculation Table of Diphtheria Toxin and HB-EGF Dissociation Rate</H6><H5>Therefore, the data regression will be plotted as below:</H5><H6><B>Figure 6.</B>Dissociation Response of Diphtheria Toxin towards HB-EGF Receptor.</H6><H6><B>Figure 7.</B>Reaction Rate of Diphtheria Toxin towards HB-EGF Receptor.</H6></DIV><H5><I>Part II: Rates Based on Concentration</I></H5><H5>The figure/literature used is still the same as the association/dissociation rates above; therefore, by remodelling and regressing the data based on the literature, one can predict the association and dissociation rates on certain times, with the concentration of the toxin as the independent variable, which is shown at the interactive Excel with this interface:</H5><DIV align="center"><H6><B>Table 5.</B>HB-EGF Receptor Properties</H6><H6><B>Table 6.</B>Association and Dissociation Constants of HB-EGF Receptor.</H6><H6><B>Table 7.</B>Association and Dissociation Response Unit and Concentration of Natural Diphtheria Toxin and HB-EGF Binding.</H6><H6><B>Table 8.</B>Calculation of Binding Reaction Rate of HB-EGF and Diphtheria Toxin Based on Concentration.</H6><H6><B>Table 8.</B>Calculation of Binding Reaction Rate of HB-EGF and Diphtheria Toxin.</H6><H6><B>Figure 8.</B>The Regression Graph for Response Unit vs Toxin Concentration</H6><H5>The programmed Excel will calculate the reaction rates just by inserting desired concentration.</H5></DIV><H5><I>Part III: Effects of pH and Temperature</I></H5><H5><OL><LI>Effects of pH</LI></OL></H5><H5>Based on throughout literature review, the closest thing for binding of diphtheria toxin is for Vero Cells [2], which is shown below:</H5><DIV align="center"><H5>Based on graph above, it can be assumed that the best temperature for diphtheria binding is 4°C. Assumed that the chosen temperature is 4°C, the regression model for said temperature for graph of time versus reaction rate, is shown below (at toxin concentration of 0.06 μM). All of the data and assumption are based on journal.</H5><H6><B>Table 10.</B>Concentration Counts of Diphtheria Toxin and HB-EGF Complex versus Time</H6><H5>The proposed model will follow arbitrary model based on Hill-Langmuir model, which is:</H5><H5>Our team formulated models (based on utilization of <I>Polymath 6.0</I> software) based on that with trial and error fit to the data, and the best fitted model would be:</H5><H6><B>Figure 10.</B>The Estimated Graphs for DT-Vero Cells Binding (Auth’s Personal Data)</H6><H5><B>Model</B></H5><DIV align="center"><H6><B>Table 11.</B>Result of Polymath 6.0</H6></DIV><H5><B>Non-linear regression settings</B></H5><H5>Max # iterations = 64</H5><H5><B>Precision</B></H5><DIV align="center"><H6><B>Table 12.</B>Result of Polymath 6.0 (cont’d).</H6></DIV><H5><B>General</B></H5><DIV align="center"><H6><B>Table 13</B>Result of Polymath 6.0 (cont’d).</H6></DIV><H5><B>The comparison table is shown as below:</B></H5><DIV align="center"><H6><B>Table 14</B>Comparison Data between Literature and Model Regression of Concentration Counts of Diphtheria Toxin and HB-EGF Complex versus Time.</H6></DIV></DIV><LI>Effects of Temperature</LI><DIV align="center"><H6><B>Figure 11.</B>Effect of pH on diphtheria toxin cytotoxicity and association of 125I-labeled diphtheria toxin with cells. Maintenance medium was replaced with complete Hanks’ 199 plus 25 mM HEPES buffer titrated to the pH indicated. For cytotoxicity assay, cells were incubated 3 h with 5 ng/mL of toxin, washed three times with normal media, and incubated a further 48 h. Cytotoxicity (█) was determined as previously described (6). For effects on association, titrated media was added to cells, followed by 125I-toxin (0.03 μg/mL) or 125I-toxin plus unlabelled toxin (0.03 μg/mL). after 2 hours at 37℃ (○) or 12 h at 4℃ (𝛥) cells were washed and radioactivity was assayed as usual.</H6><H6><B>Figure 12.</B>Dissociation of DT from immobilized hHB-EGF in running buffers of decreasing pH. The results for DT of 600 nM concentration in running buffers of pH 6.9, 6.4 and 5.8 are shown. The association phase (pH 7.4) of these curves is not shown. The origin represents the end of the DT injection and is the time at which the running buffer of specific pH has started flowing over the sensorchip, RU, resonance untis. This figure shows a representative ecperiment (see Materials and Methods).</H6></DIV><H5>Based on the graphs above, one can infer that:</H5><UL><LI>In smaller pH (more acidic), the rates of dissociation is faster; nevertheless, once the pH is below 5.3, there will be no change in the rate since the intoxicated cells have already broken (lysis).</LI><LI>In the human epithelial tissues, the intoxicated cells will not be destroyed although the pH is 5.3 as the environment temperature is close to 370C.</LI><LI>For research insight, the recommended temperature for testing binding rates is 25°C with range of pH between 5.6-5.9.</LI><LI>Can’t be mathematically modelled since the data is not sufficient for accurate modelling. The decision is made by engineering rule of thumb, in which data less than 5, is not reliable enough to be modelled into linear or non-linear regression, since the potential deviation is large.</LI></UL><H5><B>Tar Binding</B></H5><H5>The Tar binding model would be based on two research articles with different models, which are from [3] and [4]. Modelling based on [3] is as below:</H5><DIV align="center"><H6><B>Figure 13.</B>Time versus Response for TAR16 Binding to TAR*16 [3]</H6></DIV><H5><OL><LI>Association Rate</LI></OL></H5><H6><B>Table 15.</B>Properties of TAR Receptor</H6><H6><B>Table 16.</B>TAR Dissociation and Association Constant and Concentration Receptor</H6><H6><B>Table 17.</B>Calculation Table of TAR Binding Association Rate</H6><H6><B>Figure 15.</B>Association Response of Tar Binding.</H6><H6><B>Figure 16.</B>Association Reaction Rate of Tar Binding.</H6></DIV><LI>Dissociation Rate</LI><DIV align="center"><H6><B>Table 18.</B>Properties of TAR Ligand (where TAR is ligand, and TAR16 is the receptor).</H6><H6><B>Table 19.</B>TAR Receptor Dissociation and Association Constant and Concentration</H6><H6><B>Table 20.</B>Calculation Table of TAR Binding Dissociation Rate</H6><H6><B>Figure 17.</B>Dissociation Response of Tar Binding.</H6><H6><B>Figure 18.</B>Dissociation Reaction Rate of TAR Binding</H6></DIV><LI>Alternative Modelling</LI><H5>Modelling based on [4] is described as below:</H5><DIV align="center"><H6><B>Figure 19.</B>Time versus Frequency henge for TAR to Tat Binding Process. [4]</H6></DIV><H5>This model use different unit than before, in which the response is measured by the frequency change in the biosensor, therefore, the initial equation that are used is:</H5><H5>
        where,
        
        Δf  = the observed frequency change (Hz)
        Δm  = the change in mass per unit area (in g/cm2)
        Cf  = the sensitivity factor for the crystal used (i.e. 56.6 Hz μg-1 cm2 for a 5 MHz AT-cut quartz crystal at room temperature.)
        After finding the mass per unit area, the response unit can be calculated, and from that, the calculation is the same as before.
        <OL type="a"><LI>Association Rates</LI></OL></H5><H6><B>Table 21.</B>Properties of Tar Ligand (where TAR is ligand, and TAR16 is the receptor).</H6><H6><B>Table 22.</B>Tar Receptor Dissociation and Association Constant and Concentration</H6><H6><B>Table 23.</B>Tar Binding Association Rates Calculation Table.</H6><H6><B>Figure 20.</B>Association Frequency Change of Tar Binding.</H6><H6><B>Figure 21.</B>Association Reaction Rate of Tar Binding.</H6></DIV><H5>The similar method of finding reaction rates as association rates is applied on the calculation of dissociation rates: </H5><LI>Dissociation Rates</LI><DIV align="center"><H6><B>Table 24.</B>Tar Binding Dissociation Rates Calculation Table.</H6><H6><B>Figure 22.</B>Dissociation Response of Tar Binding.</H6><H6><B>Figure 23.</B>Dissociation Reaction Rate of Tar Binding.</H6></DIV><H5><B>Tar Binding Rates Based on Concentration</B></H5><H5>As for the same with HB-EGF binding above, we formulate Excel so it can predict the reaction rates based on the input of concentration, as shown below:</H5><H5><B>CheA Autophosphorylation</B></H5><H5>The source of the model is from [5]. The literature elaborates on the quantitative modelling of CheA autophosphorylation process, in which graph below left, models on the ratio of concentration of CheA-P at a certain time (P-CheA<SUB>t</SUB> or can be modelled as (〖dC〗_(〖P-CheA〗_t ))) to the maximum concentration of CheA-P that can be achieved. The graph on the right, shows the ratio of concentration of CheA-P at a certain time (P-CheA<SUB>t</SUB> or can be modelled as 〖dC〗_(〖P-CheA〗_t )/t) to the initial concentration of CheA-P:</H5><DIV align="center"><H6><B>Figure 24.</B>Ratio Between Concentration of CheA-P at a Certain Time to Concentration of CheA-P at a Threshold Tike (Left: Maximum Right: Minimum) versus Time [5]</H6></DIV><H5>With the similar approach to the modelling of association rate of Diphtheria toxin and HB-EGF, the overview and thus prediction of the autophosphorylation process can be modelled, as below:</H5><DIV align="center"><H6><B>Table 25.</B>CheA Properties and Variables Needed.</H6><H6><B>Table 26.</B>Ratio of The Change of CheA-P Concentration Against Time to the Maximum Concentration of CheA-P Achieved versus Time</H6><H5>Based on derived Langmuir-Hill model as explained above, the model used for above data is:</H5><H6><B>Figure 25.</B>The Estimated Graphs of Ratio between Concentration of CheA-P at a Certain Time to Maximum Concentration of CheA-P versus Time (Auth’s Pers Data).</H6><H6><B>Table 27.</B>Result of Polymath 6.0/h6&gt;
        <B>Nonlinear regression settings</B>
        Max # iterations = 64
        <B>Precision</B></H6><H6><B>Table 28.</B>Result of Polymath 6.0</H6></DIV><B>General</B><DIV align="center"><H6><B>Table 29.</B>Result of Polymath 6.0</H6></DIV>
        Source data points and calculated data points.
        <DIV align="center"><H6><B>Table 30.</B>Comparison of Data between Literature and Model Regression of Ratio of The Change of CheA-P Concentration Against Time to the Maximum Concentration of CheA-P Achieved versus Time.</H6></DIV><H5><B>ANALYSIS</B><B>General Analysis</B>
    Most of the base literature and reference model for all of the kinetics, be it DT and HB-EGF, Tar-agent, or autophosphorylation process, are based on the fundamental understanding of macro and micro mass-transfer in accordance of engineering’s perspective, in which:
    
    Which means that the rate of mass transfer (in this case): toxin/agent transport to be bound by the receptor, is affected by the agent’s own mass constituent (for example: concentration, or mass ratio) times its multiplier constant (sometimes called rate-limiting step), therefore, if one seeks to model the equation above, the first step to do is to linearize the differential equation by integrating it.
    
    The resulting graph, if it is in linear progression, will result in f(x) = ln(x) graph, like below:
    </H5><H6><B>Figure 26.</B>The graph of f(x) = ln(x).</H6></DIV>
    The resulting shape signifies that mass transportation will progress quickly in the early stage and decline slowly in the later stage, which can be explained by basic science and engineering, in which a group of mass will transport faster when there are bulk of them, signifying a push force between them, causing them to move faster, and then will gradually decline when the mass is mostly transferred, since there are no other force to move them, until it finally hits hypothetical saturated condition when it finally hits its peak.
    
    As for the reaction rates, the model will follow the engineering principle of chemical kinetics, in which the reaction rates will be maximum in the beginning, since the binding process happens maximum in the beginning, and gradually becomes much slower as the binding process becomes progressively slower too.
    
    Other base modelling for the mass-transport is Langmuir-Hill model, in which the binding of a ligand to a receptor/macromolecule will be faster in the presence of another ligand in close proximity. The resulting graph will have similar shape to the f(x) = ln(x) graph since the models basically have similar fundamental base, which can be found in the autophosphorylation graph.
    
    In reality, especially in micro-molecular protein complex, of course, there are difference and complex’s own signature characteristics, regarding their rules in mass transportation, that will be explained in detail as follows:
    <OL type="a"><LI>HB-EGF Binding</LI><DIV align="center"><H6><B>Figure 27.</B>Association Response of Diphtheria Toxin to HB-EGF.</H6><H6><B>Figure 28.</B>Reaction Rate of Diphtheria Toxin to HB-EGF.</H6><H6><B>Figure 29.</B>Dissociation Response of Diphtheria Toxin to HB-EGF.</H6><H6><B>Figure 30.</B>Reaction Rate of Diphtheria Toxin to HB-EGF.</H6></DIV>
      Both response unit and reaction rates for both association and dissociation reaction roughly follows the same mass transfer principle, in which the response unit (the existence of the mass) will grow faster in the early stage and then will gradually become slower in the later stage. The reaction rates will follow directly from the response unit, therefore, the reaction rates will reach its peak in the beginning and begins to falter until it reaches the maximum. The literature base [1] confirms and give another insight to the analysis, in which the Diphtheria toxin-association is environment dependent, while the release of Diphtheria Toxin (Dissociation) is rate-limiting step. The paper also elaborates that there might another additional factor regarding of interaction of toxin to HB-EGF.
      
      The final regression of the data ultimately utilizes polynomial equation since it gives the largest R<SUP>2</SUP> which means the smallest deviation, so it can be utilized for future interpolation or extrapolation purpose.
      <B>Effects of pH and Temperature on HB-EGF Binding</B><I>Effects of pH</I><DIV align="center"><H6><B>Figure 31.</B>The Estimated Graphs for DT-Vero Cells Binding (Auth’s Personal Data)</H6></DIV>
      Since the base literature on the effects of pH to the HB-EGF binding is limited, the consensus of the best temperature for the binding process will be elaborated on our own experimentation, meanwhile the literature result in temperature of 4°C for binding process graph is shown above, in which the shape of the graph follows Langmuir-Hill principle. The final regression of the data utilizes the modified version of Langmuir-Hill equation, in which the team combines both trial and error and iteration method to achieve the highest R<SUP>2</SUP>.
      <LI>TAR-Binding Kinetics</LI><I>Effects of Temperature</I>
      Since the literature reference of pH influence on the binding process is also limited, this is the analysis and our consensus regarding of the pH effects.
      <UL><LI>In smaller pH (more acidic), the rates of dissociation is faster; nevertheless, once the pH is below 5.3, there will be no change in the rate since the intoxicated cells have already broken (lysis).</LI><LI>In the human epithelial tissues, the intoxicated cells will not be destroyed although the pH is 5.3 as the environment temperature is close to 37°C.</LI><LI>For research insight, the recommended temperature for testing binding rates is 25°C with range of pH between 5.6-5.9.</LI></UL>
      Can’t be mathematically modelled since the data is not sufficient for accurate modelling. The decision is made by engineering rule of thumb, in which data less than 5, is not reliable enough to be modelled into linear or non-linear regression, since the potential deviation is large.
      
      Can’t be mathematically modelled since the data is not sufficient for accurate modelling. The decision is made by engineering rule of thumb, in which data less than 5, is not reliable enough to be modelled into linear or non-linear regression, since the potential deviation is large.
      
      The Tar binding kinetics use the same principle and method of calculation as the DT-HBEGF binding reaction, therefore the graph:
      <DIV align="center"><H6><B>Figure 32.</B>Association Response of Tar Binding.</H6><H6><B>Figure 33.</B>Reaction Rate of Tar Binding.</H6><H6><B>Figure 34.</B>Dissociation Response of Tar Binding.</H6><H6><B>Figure 35.</B>Reaction Rate of Tar Binding.</H6><H6><B>Figure 36.</B>Association Frequency Change of TAR Binding</H6><H6><B>Figure 37.</B>Association Reaction Rate of Tar Binding.</H6><H6><B>Figure 38.</B>Dissociation Response of Tar Binding.</H6><H6><B>Figure 39.</B>Dissociation Reaction Rate of Tar Binding.</H6></DIV>
      The alternative modelling is done to further prove that the TAR-binding kinetics toward ligands will also follow the mass-transfer principle. The polynomial regression is used since it give the biggest R<SUP>2</SUP> (smallest deviation)
      <LI>Auto-phosphorylation of CheA</LI><DIV align="center"><H6><B>Figure 40.</B>The Estimated Graphs of Ratio between Concentration of CheA-P at a Certain Time to Maximum Concentration of CheA-P versus Time (Auth’s Pers Data)</H6></DIV>
      This graph also closely follow the Langmuir Hill principle, even though the formula linearization and conception is not as simple as the aforementioned process, since autophosphorylation process has more elaborate steps, which is:
      
      The process above has more elaborate formulation of model in which the final model will roughly constitute of this equation:
      
      x = time
      x = initial time
      K = total constant of reactions
      When one sees the formula, it can be roughly estimated to form a similar ln graph with similar shapes to the Langmuir-Hill approximation too, therefore the final and simplified model of the autophosphorylation process will be fitted to modified Langmuir-Hill equation by trial and error and iteration to achieve the highest R<SUP>2</SUP>.
    </OL></DIV><DIV class="w3-content w3-container w3-padding-64"><H5><B>Reference :</B><OL align="justify"><LI>Brooke, J. S., Cha, J. H., &amp; Eidels, L. (1998). Diphtheria toxin: receptor interaction: association, dissociation, and effect of pH. Biochemical and biophysical research communications, 248(2), 297-302.</LI><LI>Middlebrook, J. L., Dorland, R. B., &amp; Leppla, S. H. (1978). Association of diphtheria toxin with Vero cells. Demonstration of a receptor. Journal of Biological Chemistry, 253(20), 7325-7330.</LI><LI>Nair, T. M., Myszka, D. G., &amp; Davis, D. R. (2000). Surface plasmon resonance kinetic studies of the HIV TAR RNA kissing hairpin complex and its stabilization by 2-thiouridine modification. Nucleic Acids Research, 28(9), 1935–1940</LI><LI>Tassew, N., &amp; Thompson, M. (2003). Kinetic characterization of TAR RNA–Tat peptide and neomycin interactions by acoustic wave biosensor. Biophysical chemistry, 106(3), 241-252.</LI><LI>Tawa, P., &amp; Stewart, R. C. (1994). Kinetics of CheA autophosphorylation and dephosphorylation reactions. Biochemistry, 33(25), 7917-7924</LI></OL></H5></DIV><FOOTER class="w3-center w3-green w3-padding-64 w3-opacity w3-hover-opacity-off"><DIV class="w3-xlarge w3-section">Team UI Indonesia igemui2018@gmail.com
  </DIV></FOOTER></DIV></BODY></HTML>