<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UI_Indonesia_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UI Indonesia/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-inner"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2018.igem.org/Team:UI_Indonesia">Home</A></DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown navbar-project current"><A href="https://2018.igem.org/Team:UI_Indonesia/Project#">Project</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Project#">Overview</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Project#ourproject">Our Project</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Project#resultsanddiscuccions">Results and Discussions</A></LI></UL><LI class="navbar-parts"><A href="https://2018.igem.org/Team:UI_Indonesia/Parts">Parts</A></LI><LI class="navbar-safety"><A href="https://2018.igem.org/Team:UI_Indonesia/Safety">Safety</A></LI><LI class="dropdown navbar-interlab"><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#intro">Introduction</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#materials">Materials and Equipment</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#methods">Methods</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#results">Results and Discussions</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/InterLab#conclusions">Conclusions</A></LI><LI class="navbar-model"><A href="https://2018.igem.org/Team:UI_Indonesia/Model">Model</A></LI><LI class="dropdown navbar-humanpractice"><A href="https://2018.igem.org/Team:UI_Indonesia/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Human_Practices">Integrated Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/Human_Practices#catalogue">Human Practice Catalogue</A></LI><LI class="navbar-improve"><A href="https://2018.igem.org/Team:UI_Indonesia/Improve">Improve</A></LI><LI class="navbar-team"><A href="https://2018.igem.org/Team:UI_Indonesia/Team">Team</A></LI><LI class="navbar-collaborations"><A href="https://2018.igem.org/Team:UI_Indonesia/Collaborations">Collaborations</A></LI><LI class="dropdown navbar-attributions"><A href="https://2018.igem.org/Team:UI_Indonesia/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:UI_Indonesia/MedalCriteria">Medal Criteria</A></LI></DIV></DIV></DIV><DIV class="bgimg-3 w3-display-container w3-opacity-min" id="overview"><DIV class="w3-display-middle" style="white-space:nowrap;"><SPAN class="w3-center w3-padding-large w3-black w3-xlarge w3-wide w3-animate-opacity">OVERVIEW</SPAN></DIV></DIV><DIV class="w3-content w3-container w3-padding-64"><H5>Diphtheria is becoming a prominent issue in Indonesia as its incidence is increasing recently. It also causes various complications, leading to morbidity and mortality. We realized the urgency of fast, reliable, and cheap early detection method for diphtheria infection as one of means necessary for its eradication. Therefore, we created a chimeric between native <I>Escherichia coli</I> Tar chemotaxis receptor and human HB-EGF receptor so the bacterium may recognize diphtheria toxin. Moreover, we combined CheA and CheY in <I>E. coli</I> chemotaxis system with LuxAB and eYFP, respectively. When in contact, LuxAB and eYFP create resonance energy transfer system in which LuxAB gives its emission to eYFP. Without diphtheria toxin, CheA will be in phosphorylated state, allowing interaction with CheY and energy transfer, resulting in yellow color. Toxin binding into chimeric receptor will inhibit CheA phosphorylation, hindering interaction with CheY and energy transfer, resulting in blue color (i.e. LuxAB native color).</H5><H3><B>Pathogenesis of Diphtheria: How Does <I>Corynebacterium diphtheriae</I> Cause the Disease?</B></H3><H5><I>Corynebacterium diphtheriae</I> is a Gram-positive rod bacterium that causes diphtheria. It produces exotoxin with two fragments (AB toxin). Fragment B facilitates toxin internalization within host cell via endocytosis upon binding HB-EGF receptor,while fragment A catalyzes modification of ribosomal elongation factor-2; therefore, it suppresses protein synthesis</H5><H3><B>Tar-mediated Chemotaxis System in <I>Escherichia coli</I></B></H3><H5> In <I>E. coli</I>, chemotaxis mediated by methyl-accepting chemotaxis proteins (MCPs) has been widely studied. MCPs are transmembrane chemoreceptors with periplasmic ligand binding domain and cytoplasmic signaling domain. To date, four different <I>E. coli</I> MCPs have been identified: Tar, Tsr, Trg and Tap chemoreceptors.</H5><H5>Tar chemoreceptor mediates <I>E. coli</I> movement towards aspartate and maltose (attractant molecules). It undergoes conformational change upon attractant molecule binding, leading to inhibition of CheA protein autophosphorylation and subsequent CheY activity. This causes the flagellum facilitates the bacterium to swin towards attractant substance via opposite motor protein FliM rotation. On the other hand, CheA autophosphorylation would cause prominent rotation of motor protein to make tumbling effect.</H5><H3><B>LuxAB-eYFP Fluorescence Resonance Energy Transfer (FRET) System</B></H3><H5>Basically, a molecule is excited to higher energy state when it absorbs a photon energy. This molecule relaxes back to ground state when the energy is emitted back to the environment or transferred into another molecule. FRET is a phenomenon in which non-radioactive energy is transferred from excited donor molecule to acceptor molecule via dipole-dipole interactions. Molecules involved in this phenomenon are called fluorophores as they emit fluorescence according to their respective emission spectrum after absorbing higher photon energy. LuxAB and eYFP are one of the most widely studied paired fluorophores. In this case, LuxAB is the donor fluorophore as it emits cyan colored light with relatively high energy (peak emission at 490 nm). eYFP serves as the acceptor fluorophore when in close contact with LuxAB, as it absorbs high energy from LuxAB that is overlapped with its own absorption spectrum and emits yellow colored light with lower energy (peak emission at 530 nm). To be utilized in macromolecules interaction studies, LuxAB and eYFP should be incorporated with the molecules of interest. When the molecules of interest are in contact, energy transfer between LuxAB and eYFP will happen and its efficiency can be measured with fluorescence-lifetime imaging microscopy method.</H5></DIV><DIV class="bgimg-4 w3-display-container w3-opacity-min" id="PROTOCOLS"><DIV class="w3-display-middle" style="white-space:nowrap;"><SPAN class="w3-center w3-padding-large w3-black w3-xlarge w3-wide w3-animate-opacity">OUR PROJECT</SPAN></DIV></DIV><DIV class="w3-content w3-container w3-padding-64"><H3><B>Genetic circuit</B></H3><H5>The gene for the parts used in our project are all expressed through a inducible promoter. The product of the genes do not interact with and regulate the expression of the gene of another part itself. Product of genes only interact with products of other genes. </H5><H5>The system made use of the chemotactic pathway in <I>E. coli</I> used to detect the presence of amino acids. The first part of the original pathway is the Tar receptor which detected aspartate. In our project, the receptor domain of this protein is modified by inserting the receptor domain of HB-EGF, the receptor which diphtheria toxin binds in human cells. The design of the chimeric protein, which we named HBEGF-Tar (HT), is explained more deeply in the modelling page.</H5><H5>In the native pathway, when no aspartate binds the Tar receptor, the CheA protein autophosphorylates and interacted with the CheY protein. CheY then interacts with FliM protein in the motor section of the flagella. This interaction results in the tumbling movement of the flagella. When aspartate binds with the Tar receptor, the autophosphorylation of CheA stops, thus stopping the interaction of CheA with CheY. The interaction between CheA and CheY is what our team use to detect the binding of diphtheria toxin through the Lux-BRET (Bioluminescence Resonance Energy Transfer) system. For this purpose, we linked CheA to LuxAB protein and CheY to enhanced yellow fluorescence protein (eYFP).</H5></DIV><DIV align="center"><H7><B>Fig. 1 </B> The native Tar chemotactic pathway in E. coli</H7></DIV><DIV class="w3-content w3-container w3-padding-64"><H5>When diphtheria toxin (in our project, a modified version of it) binds with the HB-EGF binding domain in HBEGF-Tar, the interaction between CheA and CheY is supposed to stop as if there’s aspartate binding. The input which is the binding of diphtheria toxin with HBEGF-Tar will create an output of blue color as a result of the lack of interaction between CheA and CheY. When there is no binding of diphtheria toxin, CheA and CheY could interact, thus resulting in LuxAB and eYFP being brought into close proximity, resulting in the fluorescence of eYFP which resulted in an output of yellow color.</H5></DIV><DIV align="center"><H7><B>Fig. 2</B> Chimeric proteins involved in the final system </H7><H7><B>Fig. 3 </B> Illustration of the BRET system used in our project </H7></DIV><DIV class="w3-content w3-container w3-padding-64"><H3><B>Lab Notes and Protocol</B></H3><H5>here we attached our protocol used during the lab work and some notes that might be usefull in the future</H5><UL><H5><LI>Our Protocols can be accessed <A href="https://static.igem.org/mediawiki/2018/6/6a/T--UI_Indonesia--protocol.pdf">here</A></LI><LI>Our Lab Notes can be accessed <A href="https://static.igem.org/mediawiki/2018/5/5d/T--UI_Indonesia--labnotes.pdf"> here</A><DIV class="bgimg-5 w3-display-container w3-opacity-min" id="resultsanddiscuccions"><DIV class="w3-display-middle" style="white-space:nowrap;"><SPAN class="w3-center w3-padding-large w3-black w3-xlarge w3-wide w3-animate-opacity">RESULTS AND DISCUSSIONS</SPAN></DIV></DIV></LI></H5><H3><B>Introduction</B></H3><H5>Managing the realization of the usage of synthetic diphtheria exotoxin (i.e. <I>DiphTox</I>/DT) and chimeric HB-EGF/Tar (i.e. HT) receptor, our team proposed a focus on characterizing the part of each BioBrick’s function. This year, our team, would be more focusing on testing the binding ability of both BioBricks in living <I>E. coli</I> spheroplast. FRET system would proceed in the later project of Finding Diphthy next in iGEM 2019. Before heading on to the lab results, it is recommended to fully understand the structure of DT and HT gBlocks and their respective protein functions in Parts (<A href="https://2018.igem.org/Team:UI_Indonesia/Parts" style="color:blue">https://2018.igem.org/Team:UI_Indonesia/Parts</A>) and Model domain (<A href="https://2018.igem.org/Team:UI_Indonesia/Model" style="color:blue">https://2018.igem.org/Team:UI_Indonesia/Model</A>) of UI_Indonesia Wiki page.</H5><DIV class="w3-content w3-container w3-padding-64"><H3><B>Materials and Equipment</B></H3><H5>Some of the materials are provided by IHVCB lab, as well as Promega sponsored kit and IDT.</H5><H5><I>Materials from 2018 DNA Distribution Kit</I></H5><H5><UL><LI>Part BBa_J04450 containing mRFP BioBrick in the pSB1C3 backbone
</LI></UL></H5><H5><I>Materials provided by IHVCB lab UI</I></H5><H5><UL><LI>ddH<SUB>2</SUB>O or nuclease-free water (NFW)
</LI><LI>Chemical substances, such as MgCl<SUB>2</SUB>, NaCl, CaCl<SUB>2</SUB>, etc
</LI><LI>96-well plate
</LI><LI>96-well plate reader (<B>GloMax<SUP>TM</SUP>– Multi Detection System</B>) 
</LI><LI>Luria Bertani (LB) liquid or agar media
</LI><LI>Competent cell of <I>Escherichia coli</I> strain K12 derivates, such as TOP10 and DH5α, also strain B derivate such as BL21(DE3).
</LI><LI>Antibiotics consisting of chloramphenicol (Cam) and ampicillin (Amp)
</LI><LI>Polymerase chain reaction (PCR) thermal cycler
</LI><LI>Micropipettes and tips
</LI><LI>Microtubes and centrifuge tubes
</LI><LI>Ice and buckets
</LI><LI>Electrophoretic horizontal (DNA) and vertical (protein) chamber, power supply
</LI><LI>Vortex 
</LI><LI>Minicentrifuge and ultracentrifuge
</LI><LI>Gel electrophoresis reader (<B>Gel Doc<SUP>TM</SUP> XR+ Gel Documentation System</B>) and sodium dodecyl sulphate - polyacrilamide gel electrophoresis (SDS-PAGE) reader (<B>ImageQuant<SUP>TM</SUP></B>)
</LI><LI>Buffers and enzymes (restriction enzymes such as EcoRI and PstI, ligase, etc.)
</LI><LI>Static incubator and roto-shaker
</LI><LI>Heatblock machine
</LI><LI>Autoclave and microwave machine
</LI></UL></H5><H3><B>Methods</B></H3><H5>Our lab team started to use traditional cloning for insertion of desired gene into backbone plasmids for expression and iGEM 2018 submission. The traditional cloning would involve the methods of inserting gene fragments into MCS (multiple site cloning) of backbone or between prefix-suffix. The whole processes of gene cloning for further assays in this project are described in the <B>Figure 1 and 2</B>.
</H5><H6><B>Figure 1.</B> Principle methods of traditional cloning</H6><H6><B>Figure 2.</B>. Original plan of how our lab team conducted the whole experiment in establishing parts for iGEM submission and characterizing our parts: DiphTox or DT (BBa_K2607000) and HB-EGF/Tar chimeric receptor or HT (BBa_K2607001).</H6><H3>Part I: DT and HT <I>Cloning</I></H3><H5>Upon receiving our parts (in gBlocks) from Integrated DNA Technologies, Inc. (IDT), we performed PCR to amplify the gBlocks. PCR cloning for all gBlocks used the hand-made designed forward (Fwd) and reverse (Rev) cloning primers (i.e. specifications could be seen in Parts page). Furthermore, cycling formula for PCR cloning and confirmation could be accessed in the lab notes, as we applied GoTaq<SUP>TM</SUP> Long PCR enzyme as the Hi-Fi polymerase. The amplified gBlocks were then used as inserts to plasmid vectors. For HT BioBrick, IDT was unable to yield the full sequence in high purity, so we had to split HT into two fragments (HT-1 and HT-2), which would later be amplified, restricted with SalI, and ligated to obtain complete HT fragment

On the other hand, we also prepared vectors for carrying our parts. Backbone pSB1C3-mRFP (BBa_J04450) has been used widely in our process of traditional cloning, for it provides much sensitive selection upon transformed recombinant plasmids. Since this plasmid does not contain any available expression promoter for the designed BioBrick, our supervisor suggested the usage of pQE80L expression vector belonged to IHVCB lab for functional assays and analyses. Therefore, we used pSB1C3 as cloning vector for submission to iGEM Headquarters and pQE80L as cloning vector for expression

We conducted traditional cloning (restriction-ligation) method to introduce our previously amplified inserts into prepared vectors. Restriction digestion was done sequentially with EcoRI and PstI in total of 8 hours by using the same buffer (i.e. EcoRI buffer and bovine serum antigen (BSA) 1X) with a minimum DNA template of 10 µg. Desalting and low-melting agarose (LMA) 1% electrophoresis purification was done to further remove any possible contaminating enzymes and undesired polynucleotides. Ligation of both vectors and inserts were conducted by adding T4 ligase and its respective buffers to be later incubated 16<SUP>0</SUP>C overnight

Transformation of resultant recombinant plasmids was done in <I>wild-type E. coli</I> K-12 (for submission purpose) and BL21(DE3) (for characterization and validation purpose) via “transformation protocol” methods in the other page. To enhance selection of recombinant <I>E. coli</I>, the transformed products were spread into selective LB agar containing appropriate antibiotic. Antibiotic formulation was complied to the lab’s proven antibiotics sensitivity test. We solubilized the powdered chloramphenicol in ethanol 95% and ampicillin in distilled water until final concentration of 25 mg/ml and 100 mg/ml, respectively. They were then added into LB media with ratio of 1:1000. After spread into LB agar, the transformed products were then incubated at 370C overnight.

In the case of transformation with pSB1C3, to select the colony with desired inserts, we performed red-white screening. If the grown colonies were red, it indicated that the colonies were transformed by native pSB1C3-mRFP and we excluded the colonies. We only picked white colonies (indicated that mRFP had been successfully removed from pSB1C3 and possibly replaced by insert) to be further confirmed for desired insert presence by colony PCR. We used VF2 and VR primers (i.e. iGEM standard primers) for confirmation of inserts in pSB1C3, while we used our hand-made designed primers for confirmation of inserts in pQE80L. 

Finally, we performed mini-prep plasmid isolation for any confirmed colonies with desired inserts in pSB1C3. We grew the colonies in LB liquid medium at 37<SUP>0</SUP>C shaken overnight. Sequencing was performed to confirm the sequence of inserts before submitted to iGEM Headquarters.</H5><H3>Part II: <I>SDS-PAGE Confirmation of Expressing BioBricks</I></H3><H5>Confirmation of any expressing DT and HT protein in recombinant <I>E. coli</I> BL21(DE3) was done via SDS-PAGE after isopropyl-D-1-thiogalactopyranoside (IPTG) induction for 4 hours in 370C in terrific broth (TB) medium with ampicillin. Subsequent lysis of <I>E. coli</I> to expose the desired proteins was done chemically via ionic and temperature induction. For DT containing His-Tag at the C-terminus of the protein, we managed to do His-Tag purification using magnetic beads. Binding of the DT protein into the beads would be enhanced by adding NaCl 500 mM. Incubation and washing were done 3X to remove any protein debris. Elution of the beads would generate the purified DT protein to be analyzed in the SDS-PAGE.</H5><DIV align="justify"><H3>Part III: DT-HT <I>Binding Assay</I></H3></DIV><H5>Prior to this step, our team expressed HT in transformed <I>E. coli</I> BL21(DE3) with pQE80L-HT by IPTG induction. In addition, we also had to remove outer membrane of the <I>E. coli</I>. The membrane removal would enable the HT receptor (in inner membrane) exposed directly towards extracellular environment, and possibly detecting DT.

Binding assays of DT and HT was conducted within 96-well plates by measuring the absorbance of 600 nm. This absorbance index indicates amounts of <I>E. coli</I> spheroplasts that successfully bound into DT in various environmental conditions (i.e. pH, temperatures, and DT concentration variables). Incubation was done within 60 minutes and purified magnetically using the available His-Tag. The amount of HT receptor binds to DT correlates positively with the amount of spheroplasts available in the eluents. Therefore, OD<SUB>600</SUB> is used as the primary quantification of spheroplasts amounts in the eluents. Specific details regarding methods of binding assays could be accessed via protocol page</H5><H3>Part IV: <I>Luminescence (ADP-Glo<SUP>TM</SUP> Kinase) Assays</I></H3><H5>To prepare the samples, we incubated <I>E. coli</I> BL21(DE3) transformed with empty pQE80L and pQE80L-HT cultures in LB liquid medium with ampicillin (1000:1) overnight. We created four replicates for each culture. On the following day, 1 mL from each replicate (eight in total) was aliquoted into 4 mL fresh TB medium with ampicillin (1000:1) and 4 µL IPTG 1 M. The replicates were incubated at 37<SUP>o</SUP>C, 220 rpm for four hours. Their OD<SUB>600</SUB> were then determined and the replicates were subsequently pelleted at 12,000 rpm for one minute. The pellets were then lysed using Promega FastBreak<SUP>TM</SUP> cell lysis reagent according the manufacturer protocol.

The following luminescence assay procedures is based on kit protocol with some modifications. First, we created standard curve according to the kit protocol to estimate ATP-to-ADP conversion rate from luminescence data. This performed by creating series of 1 mM ATP+ADP mixture with varying percentage of ADP relative to ATP+ADP (<B>Table 1</B>) in first row of microplate well. Series of 100 µM, 10 µM, and 1 µM were created by serial dilution in subsequent rows. The luminescence was then determined using Promega GloMax®-Multi Detection System.</H5><DIV align="center"><H6><B>Table 1.</B>ATP+ADP mixture with varying percentage of ADP relative to ATP+ADP in microplate wells for standard curve.</H6><H5>Next, the reaction between lysed cells and purified DT with the amount of 180 nM (i.e. from Part III we found that Kd for HB-EGF/Tar and DT binding is 88.43 nM in 250C, then the maximum binding activity in 250C is expected to be achieved at twice of Kd) was carried out under 1x kinase reaction buffer consisting of 40 mM Tris pH 7.5, 20 mM MgCl2, and 0.1 mg/mL BSA. Ten µL from each reaction was aliquoted into wells in microplate and 10 µL of ADP-GloTM reagent was added to each well to stop the reaction and deplete the remaining ATP in the samples. After 40 minutes of incubation, 20 µL of kinase detection reagent was added to each well to convert ADP to ATP which will be used to generate luminescence and introduce substances (i.e. luciferin and luciferase) for luminescence. After 30 minutes of incubation, the luminescence was measured.  Further details regarding ADP-Glo methods could be accessed in the protocol page.</H5><H3><B>Result and Discussion</B>
Part I: <I>Gel Electrophoresis Confirmation of DT and HT cloning</I></H3><H5><U>Confirmation Insertion of DT into pQE80L and pSB1C3</U>
Insertion of DT into pQE80L (for assays) and pSB1C3 (for submission) was done via traditional cloning using <I>EcoRI</I> and <I>PstI</I> restriction endonucleases. We have managed to maintain the workflow as preceding flowchart mentioned in methods. Transformation of recombinant plasmids is done into E. coli strain BL21(DE3) for pQE80L and TOP10 for pSB1C3, respectively. BL21(DE3) was chosen rather than TOP10 for assays, as it maintains anti-protease activity which would enable DT proteins undegraded intracellularly. In addition, BL21(DE3) is also a strain compatible for expression under T7 promoter (which is present in HT gBlock). Electrophoresis of colony PCR in the chamber utilized agarose 1% m/v soaked with TAE 1X, powered within 100 V for 30 minutes. <B>Figure 3</B> and <B>4</B> shows the process of how we selected the colonies with recombinant plasmids, while <B>Figure 5</B> and <B>6</B> were the final results of DT colony PCR confirmation.</H5><DIV align="center"><H6><B>Figure 3.</B> Colonies of E. coli BL21(DE3) with pQE80L-DT in LB agar containing ampicillin</H6><H6><B>Figure 4.</B> Ultraviolet (UV) illumination of E. coli transformed with pSB1C3-DT. Red-white screening could be utilized since our team used pSB1C3-mRFP as initial backbone vector.</H6><H6><B>Figure 5.</B> Gel analysis of colony PCR on pQE80L-DT transformed into E. coli BL21(DE3).  PCR colony primer would amplify ~250 bp bands that located inside the DT BioBrick, indicating that the DT fragment was successfully cloned into pQE80L.</H6><H6><B>Figure 6.</B> Gel analysis of PCR colonies on pSB1C3-DT transformed E. coli TOP10. Universal flanked primers of VF2 and VR in pSB1C3 would amplify ~600 bp bands that were found in several following colonies, indicating that the DT fragment was successfully cloned into pSB1C3.</H6><H5>From these results, we concluded that DT BioBrick was successfully inserted into pSB1C3 and pQE80L backbone.</H5><H5><U>Confirmation Insertion of HB-EGF/Tar (HT) into pQE80L and pSB1C3</U></H5><H5> The methods for inserting HT complete fragment were quite challenging, since it must be linearly ligated in the first place prior to recombination into plasmid vector. Linear ligation of HT-1 and HT-2 fragments were conducted by using SalI restriction enzyme, yielding approximately 1954 bp complete HT BioBrick. Unfortunately, PCR amplification in the beginning using hand-made designed Fwd and Rev cloning primers could not generate specific bands (i.e. the primers anneals unspecific in various length of both HT fragments, see Collaborations page). Therefore, the gBlocks were shipped to Nanyang Technology University, Singapore (NTU-Singapore) team for ligation into pcDNA3 and pSB1C3. Our team conducted immediate transfer of complete HT BioBricks into pQE80L, while waiting for NTU finishing the cloning of the HT complete fragments into pSB1C3. The transfer of completed HT into pQE80L is shown in <B>Figure 7</B> and <B>8</B>, while the transfer of completed HT into pSB1C3 (done by NTU-Singapore) is shown in <B>Figure 9.</B></H5><DIV align="center"><H6><B>Figure 7.</B> Gel analysis of colony PCR on pQE80L-HT transformed E. coli BL21(DE3). The subsequent ~600s bp bands were found in the following colonies, indicating that the HT fragment was successfully inserted</H6><H6><B>Figure 8.</B> PCR colonies of pQE80L-HT in LB + Amp agar medium</H6><H6><B>Figure 9.</B> Gel analysis of colony PCR of pSB1C3-HT. Universal VF2 and VR primers would amplify ~2000s bp bands that located inside the BioBrick HT, indicating that the HT fragment was successfully cloned into pSB1C3. Note: this datum was provided by team NTU-Singapore as gratitude for their aid in submission cloning for team UI_Indonesia’s HT fragments.</H6></DIV><H5>These results suggested that HT BioBrick was successfully cloned into pSB1C3 and pQE80L backbone</H5><DIV align="justify"><H3>Part II: <I>SDS-PAGE Confirmation of Expressing Biobricks</I></H3><H5>After insertion of both BioBricks into pQE80L, the assays could begin with expression confirmation. For transcription initiation of BioBricks require lac promoter provided by the vector, induction of IPTG was essential. Identification of positive control using E. coli TOP10 transformed with pBluescript KS(-) could be important in determining whether our IPTG used was expired or not. Wild-type E. coli BL21(DE3) and E. coli BL21(DE3) inserted with empty pQE80L were used as negative control. Furthermore, purification of DT was conducted to increase sensitivity of expression yield. <B>Figure 10</B> and <B>11</B> shows the SDS-PAGE performed to confirm DT and HT expression, respectively.</H5></DIV><DIV aign="center"><H6><B>Figure 10.</B> SDS-PAGE analysis (photographed via ImageQuant) of pQE80L-DT expression in E. coli BL21(DE3). White arrow indicates LacZα (size ~20.7 kDa) protein expression due to IPTG induction, indicating our IPTG was in good condition. On the other hand, black arrow shows DT (size ~7 kDa) protein expression as it is induced with IPTG within 4 hours. Note: pKS(0) = E. coli TOP10 transformed with pBluescript KS(-) with no IPTG induction; pKS(4) = E. coli TOP10 transformed with pBluescript KS(-) after 4 hours of IPTG induction in 370C; BL21(DE3) = wild-type E. coli BL21(DE3); BL21(DE3) w/ pQE80L = E. coli BL21(DE3) containing  empty pQE80L; pQE80L-DT(0) = E. coli BL21(DE3) containing recombinant pQE80L-DT with no IPTG induction; pQE80L-DT(0)p = purified protein of E. coli BL21(DE3) containing recombinant pQE80L-DT with no IPTG induction; pQE80L-DT(4) = E. coli BL21(DE3) containing recombinant pQE80L-DT with 4 hours of IPTG induction; pQE80L-DT(4)p = purified protein of E. coli BL21(DE3) containing recombinant pQE80L-DT after 4 hours of IPTG induction.</H6><H6><B>Figure 11.</B> SDS-PAGE analysis (photographed via ImageQuant) of pQE80L-HT expression in E. coli BL21(DE3). White arrow indicates LacZα (size ~20.7 kDa) protein expression due to IPTG induction, while black arrow shows increasing HT (size ~57.8 kDa) protein expression as it is induced with IPTG within 4 hours. Note: pKS(0) = E. coli TOP10 transformed with pBluescript KS(-) with no IPTG induction; pKS(4) = E. coli TOP10 transformed with pBluescript KS(-) after 4 hours of IPTG induction in 370C; BL21(DE3) = wild-type E. coli BL21(DE3); BL21(DE3) w/ pQE80L = E. coli BL21(DE3) containing empty pQE80L; pQE80L-HT(0) = E. coli BL21(DE3) containing recombinant pQE80L-HT with no IPTG induction; pQE80L-HT(4) = E. coli BL21(DE3) containing recombinant pQE80L-HT after 4 hours of IPTG induction.</H6><H5>Extraction of the protein DiphTox from recombinant E. coli was done by prominent lysis of cells grown that is grown overnight. Native lysis buffer was utilized to maintain specific protein configuration for its function for binding. To measure protein concentration, our team would use Bradford assay using Coomassie Brilliant Blue solution and placed within plate reader for 595 nm absorbance test. Standard curve of the assay was produced using Bovine Serum Albumin (BSA) as concentration standards.</H5><DIV align="center"><H6><B>Figure 12.</B> Standard curve of BSA in Bradford assay. 
</H6><H3>Part III: <I>DT-HT Binding Assays</I></H3></DIV><H5><U>Effect of Temperatures on DT-HT Binding Kinetics</U></H5><H5>Upon confirmation of DT and HT expression, our team would like to testify the interaction of those proteins. Prior to binding assay, the recombinant E. coli possessing HT expression should be uncoated from the outer membrane layer. This lets huge molecules or proteins accessing the periplasmic layer or inner plasma membrane of the bacteria. Transforming E. coli into intact spheroplast could be a disadvantageous for the cell itself, since the membrane is more fragile to extracellular extremes. This would be one of the major challenges of the binding assays in determining specific pH and temperatures for keeping the spheroplasts alive. Methods for making spheroplasts could be accessed in the protocol page

Spheroplasts were subjected to different DT concentration during one hour incubation with different temperatures (<B>Table 2</B>). Binding of HT receptor towards intact DT in Magne-His beads would prevent spheroplasts elimination during washing process. Elution of spheroplasts would be the final variable in quantifying DT-HT binding strength.</H5><DIV align="center"><H6><B>Table 2.</B> Net OD<SUB>600</SUB> (minus blank: elution buffer) results of DT-HT binding assays in different temperatures and various DT concentrations. Triplicates were done to minimize bias of absorbance data.</H6><H6><B>Figure 13.</B> Average amount of <I>E. coli</I> spheroplasts (expressed in OD<SUB>600</SUB>) remained in the eluent after magnetic binding with DT in different DT concentrations and temperatures. OD600 is expressed within arbitrary unit. Standard error of estimates is shown in between 0.005-0.008.</H6></DIV><H5>Plotting of the OD600 in the graph (Figure 13) could be referred classically as ligand-receptor dynamics. The increasing of DT concentration would enable huge amount binding of spheroplasts in the incubation. Therefore, the logarithmic trend occurs in the beginning. Nevertheless, there would be an amount of DT concentration where the number of spheroplast binding would be reaching its maximum. Affinity constant of the receptor towards ligands could be determined as the amount of substrate causing saturation of 50% receptors. Reverse plotting of Lineweaver-Burke would be required to determine the values of HT-DT affinity constant and maximum saturation. Modified receptor-ligand formulae could be further transformed as shown in Figure 14. (Note: y represents 1/OD600, and x represents 1/substrate concentration in linear equation)</H5><DIV align="center"><H6><B>Figure 14.</B> P-P plot regression analysis to determine its Lineweaver-Burke linear equation. R square values and linear equations are shown within regression in each temperature via SPSS Basic Statistics v.21. Therefore, standardized values of Kd and OD600 max could be determined in different temperature conditions.</H6></DIV><H5>Different temperatures exhibit different affinity constant and maximum binding (Table 3). It is natural that HT receptor possesses optimum tertiary structure facilitating binding of DT in certain environmental condition. Highest affinity of HT towards DT occurs in temperature of 250C. Additionally, maximum binding of HT towards DT happens highest in 40C and 370C. Low temperature might increase the possibility of HT-DT binding, since the molecular kinetical energy is low. Therefore, extrapolation of affinity and maximum binding from different temperatures into single human body temperatures (370C) would be required in future research.</H5><DIV align="center"><H6><B>Table 3.<B> Effect of temperatures towards binding kinetics of DT and HT.</B></B></H6></DIV><H5>In conclusion, positive binding of HT and DT shows that the designed ligand-receptor could be further used as predicted models of diphtheria toxins interaction in human body.
<U>Effect of Extracellular pH Towards Optimization of DT-HT Binding.</U></H5><H5>Profound understanding of the natural HT and DT binding could be analyzed by measuring the binding activities in different pH concentration. This could serve as potential basis for prediction models of environment that supports the interaction. Principles of measuring the activities are the same as previous one. Triplicates are subjected to different pH solution, ranging from 4-8.5, during an hour incubation prior to elution (<B>Table 4</B>). The used concentration of DT in this assay was 180 nM (as it referred to maximum binding of DT concentration towards HT in 25<SUP>0</SUP>C).</H5><DIV align="center"><H6>Table 4. Net OD<SUB>600</SUB> (minus blank: elution buffer) results of DT-HT binding in different extracellular pH. Triplicates were conducted to minimize bias effect of absorbance index.</H6><H6><B>Figure 15.</B> This clustered column represents amount of spheroplasts remained in the eluent buffer after washing in different pH solution. Triplicates with standard bar errors indicate any data bias optimized as samples. Different alphabetic indexes top of the column express data significances, such as a-b (p&lt;0.05), a-c (p&lt;0.05), and b-c (p&lt;0.01).</H6></DIV><H5>Optimal pH for the binding interaction occurs at 7 to 7.5 (<B>Figure 15</B>). This is exactly the physiological pH condition inside human body. Significant drop of binding activity observed at pH 5.5 could indicate early spheroplast autolysis, since it triggers intracellular cascade of lysis protein according to Raam R, et al.  Therefore, lab models of binding HT-DT could be conducted optimally within pH range of 7 to 7.5.</H5><H3>Part IV: <I>Luminescence (ADP-GloTM Kinase) Assays</I></H3><DIV align="center"><H6><B>Figure 16.</B> Standard curve for percent ATP-to-ADP conversion from luminescence with varying concentrations of ATP+ADP mixture (1 µM, 10 µM, 100 µM, and 1 mM). RFU = relative light units.</H6></DIV><H5>The standard curve obtained for percent ATP-to-ADP conversion from luminescence data is shown in Figure 16. With this standard curve, we can estimate how much ATP is converted into ADP (or how much ATP is conserved in HT-DT reaction compared to normal) based on given luminescence data later in the experiment.</H5><DIV align="center"><H6><B>Table 5.</B> Net OD<SUB>600</SUB> (minus blank) and luminescence (in RFU) after reaction with DT.</H6><H6><B>Figure 17.</B> Luminescence per OD<SUB>600</SUB> (RFU) of control (transformed <I>E. coli</I> BL21(DE3) with empty pQE80L) and experimental (transformed <I>E. coli</I> BL21(DE3) with pQE80L+HT) group. * indicates significant difference compared to control (p = 0.001).
</H6><H5><B>Table 5</B> shows net OD<SUB>600</SUB> (after deducted by blank Abs<SUB>600</SUB>) and their luminescence after reaction with DT for each replicate, while <B>Figure 17</B> shows comparison between luminescence per OD<SUB>600</SUB> for control (<I>E. coli</I> BL21(DE3) transformed with empty pQE80L) and experimental (<I>E. coli</I> BL21(DE3) transformed with pQE80L+ HT) group upon reaction with DT. We found that luminescence level in experimental group was significantly lower than control group (p = 0.001). This result suggested that experimental group has lower ADP concentration than control group, which likely to be caused by HT interaction with DiphTox inhibits CheA phosphorylation, ultimately leading to more ATP conserved and less ADP produced. Therefore, upon reaction with kinase detection reagent, less ADP will be converted into ATP in experimental group, causing less luminescence compared with control group.

From the experiment, we did not know the initial ATP+ADP concentration from each control and experimental group. Assuming the initial ATP concentration was 1 mM (mean ATP concentration in E. coli is 1.56 ± 1.27 mM according to Yaginuma et al.), with the help of standard curve from Figure 1, we can estimate that percent of ATP-to-ADP conversion inhibition upon HT-DT interaction was around 1%. This is due to luminescence of standard 0% ADP-ATP mixture was 1.69 x 10<SUP>6</SUP>, similar with mean luminescence of experimental group (1.55 x 10<SUP>6</SUP>), while luminescence of standard 1% ADP-ATP mixture was 2.2 x 10<SUP>6</SUP>, similar with mean luminescence of control group (2.1 x 10<SUP>6</SUP>). One note to be considered that E. coli cells may have varying level of intracellular ATP and ADP, as stated by large standard deviation of ATP concentration from Yaginuma et al. study. However, we grew and treated E. coli used in this experiment equally, also we created replicates for each group to minimize the possible bias.

Hence, our present result suggested that our HT works as expected to inhibit phosphorylation, shown by less luminescence generated compared with control, indicating that less ADP was produced upon HT-DT interaction.</H5><H5><B>Reference</B><OL><LI>Mohan, R. R., Kronish, D. P., Pianotti, R. S., Epstein, R. L., &amp; Schwartz, B. S. (1965). Autolytic Mechanism for Spheroplast Formation in Bacillus cereus and Escherichia coli. Journal of Bacteriology, 90(5), 1355–1364.
</LI><LI>Yaginuma H, et al. Diversity of ATP concentrations in a single bacterial cell population revealed by quantitative single-cell imaging. 2014. Available from: <A href="https://www.nature.com/articles/srep06522" style="color:blue">https://www.nature.com/articles/srep06522</A></LI></OL></H5></DIV><FOOTER class="w3-center w3-green w3-padding-64 w3-opacity w3-hover-opacity-off"><DIV class="w3-xlarge w3-section">Team UI Indonesia igemui2018@gmail.com
  </DIV></FOOTER></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>