Team:AHUT China/Parts
Parts
Home HomeHome
Team Team Team
Menbers MenbersMenbers
Instructions InstructionsInstructions
Attributions AttributionsAttributions Acknowledgement AcknowledgementAcknowledgement
Project Project Project
Pathway and DesignPathway and DesignPathway and Design
ResultResultResult
Future WorkFuture WorkFuture Work
Parts Parts Parts
Biobrick BiobrickBiobrick
ProtoolsProtoolsProtools
Human Practices Human PracticesHuman Practices
Society SocietySociety
New PowerNew PowerNew Power
QestionnaireQestionnaireQestionnaire
MOOC MOOCMOOC
Art ArtArt
Safety SafetySafety
Safety SafetySafety
NoteBook NoteBookNoteBook
NotebookNotebookNotebook
Lab PicturesLab PicturesLab Pictures
MedalMedalMedal
bronzebronzebronze
silversilversilver
goldgoldgold
Parts
Our submission parts
BBa_K1341000
We use this device as a part of a switch device designed for our bio-compass.It can help us screen out the possible path way which include the start point,terminal point, and the particular intermediate nodes(such as the museum(No.5 node)in our story).It has a LTT Promoter,with elements of the tet, lac, they will compose as a device and is responsive to the commonly used inducers IPTG and aTc, producing GFP as an output signal.If this device and tetR(BBa_K1341003)&lacI(BBa_K1341002)successful inserted in the path way by using the typically enzyme site. It will drive expression of green fluorescent protein (GFP). This characteristic will help us to screen out the possible way we want to find.PS: We build a measurement system to test the screen functional about this device in our project(BBa_K1341001)
BBa_K1341001
This is a measurement system comprised of a switch device(output GFP as signal)and a specialized path way sequence.This system can prove that we have insert the BBa_K1341000 in the testing path way successfully(we designed a AFL III Restriction Enzyme cutting site in the No.5 node sequence. Before we insert the device, we linked the matching AFL III Restriction Enzyme cutting site sequence on switch device's(BBa_K1341000)ends by using BioBrick assembly method.).Besides, this system also has a BamH I Restriction Enzyme cutting site in the No.2(start) node's sequence, a Nco I Restriction Enzyme cutting site in the No.8(terminal） node's sequence. With the same method, the TetR device (BBa_K1341003) and LacI device (BBa_K1341002) can insert in this measurement system according to the requirement. By using the aTc and IPTG, we can test whether the path way is the right result exactly such as this(2-4-5-7-8).
Improve part BBa_K091101 We change this part as an LTT pTet_ Lac hybrid promoter in our device BBa_K1341000,it will have the more active GFP expression than this kind of TTL promoters(BBa_K091101) according to model figure in result page. So we can test the change of the GFP expression more easily.
Protocols
1.Bridge PCR
Experimental idea
We plan to put all sites and lines together into the PCR tube, so the free combination is performed on each sites and lines through complementary base pairing, forming all possible DNA fragments (i.e. different pathways). Then add DNA polymerase and dNTP to fill the combination of single-stranded fragments at a suitable temperature.
Set up the following reaction:
The PCR reaction procedure is as follows:
Procedure
1. Place the diluted sites and fragments into the PCR tube (the reaction system is as shown in Table 1)
2. Carry out two groups of experiments
3. Place the samples in the PCR instrument (the PCR procedures are as shown in Figure 1).
2.DNA Ligation
Experimental idea
As the elongation of the newly forming DNA strand is in a 5'-3' direction, there is still no connection between sites and lines, despite the single strand of the free combination pathways have been filled in the DNA polymerization. Therefore, before the PCR, DNA ligase is added to connect them.
Set up the following reaction:
Procedure
1. Conduct the DNA Gel extraction of the DNA polymerization products (the protocol will be explained later)
2. Place the DNA polymerization products into the PCR tube (the reaction system is as shown in Table 2)
3. Incubate for 12 hours at 16℃.
3.DNA Gel extraction
Experimental idea
After the Bridge PCR and DNA Ligation, there are many impurities in the products such as different enzymes and reaction leftovers, which will affect the subsequent experiments. Therefore, the DNA gel extraction is needed, by extracting directly from the solution instead of gel electrophoresis to reduce losses.
Procedure
1. Add deionized water to 100 µl volume, add 300 µl Buffer DE-A.
2. Add 100 µl Buffer DE-B Buffer and mix, then add 20% isopropanol.
3. Place Spin Column of the kits on the Collection Tube.
4. Transfer the solution in the step 2 to Column Spin, centrifuge at 12000
×g for 1 minute (if there remains liquid in the Column Spin,
the centrifugal rate can be appropriately increased and centrifuge for another 1 minute),
then discard the filtrate.
Note: The recovery will improve if the filtrate can be added
to the Column Spin once again.
5. Add 500 µl Buffer W1 to Spin Column, centrifuge at 12000×g for 30 seconds,
then discard the filtrate.
6. Add 700 µl Buffer W2 to Spin Column, centrifuge at 12000×g for 30 seconds,
then discard the filtrate.
7. Repeat the step 6, centrifuge at 12000×g for 1 minute.
8. Transfer the Spin Column into a clean 1.5ml microfuge tube and let it stand for 3 minutes.
To elute the DNA, add 30 µl Eluent or 60℃ water. Let it
stand for 1 minute at room temperature.
9. Centrifuge at 12000×g for 1 minute for DNA elution.
10. Suck out the liquid from the microfuge tube and add in the Spin Column.
Repeat the step 9 to get the products.
4.The design of PCR primers
Experimental idea
According to our design, the starting point of all free combination pathways is site 2, the ending point is site 8. In order to obtain a large number of target genes, all of the DNA fragments that start at the site 2 and end at the site 8 need to be amplified. So the special primers are designed according to the sequence of site 2 and site 8, and restriction sites (respectively, Pst I and EcoR I -HF)are added in two primers according to what we want to import plasmid. The sequences of primers are as follows:
The sequence of site 2:
5’GTAATGATCTCCTAGGAGATACATTCGATCGATCATGCTA3’
The sequence of site 8:
5’GTACAGTACGGTACCGTACCCGTGACGTACGTGATGACTG3’
PrimeF:
5'GTTTCTTCGAATTCGCGGCCGCTTCTAGAGGTAATGATCTCCTAGGAGAT3’
PrimeR:
5’GTTTCTTCCTGCAGCGGCCGCTACTAGTACAGTCATCACGTACGTCACG3’
EcoRI_site:   GAATTC PstI_site:    CTGCAG
5. PCR
The PCR reaction procedure is as follows:
Divide the extraction products that conducted the DNA ligation into four PCR tubes, whose system is as shown in Table 3. PCR reactions are under different temperatures, respectively, at 50℃.53℃.55℃.58℃. The procedures are as shown in figure 3.
Procedure
1. Denaturation step: The high temperature causes DNA melting of the DNA template, yielding single-stranded DNA molecules.（98℃  10s）
2. Annealing step: The low temperature allows annealing of the primers to the single-stranded DNA template and hybridization of the primer to the strand.（50℃  30s）
3. Extension step: The DNA polymerase synthesizes a new DNA strand complementary to the DNA template strand by adding Mg2+, dNTPs that are complementary to the template in 5' to 3' direction. （72℃ ， 30s）
The processes of denaturation, annealing and extension constitute of one cycle. Each cycle of the product can be used as a template for the next cycle, the number of DNA copies after a given n of cycles repeated is, 2n of cycles.
The procedure is as follows:
6. Agarose Gel Preparation
Procedure
1. Prepare the 1X TAE Buffer that for electrophoresis and gel.
2. According to the amount and the concentration of gel, accurately weigh 0.75 g agarose powder and add in the conical flask.
3. Mix the agarose with the 25ml 1X TAE Buffer.
4. Heat the mixture until all the agarose is dissolved.
5. Leave the mixture to cool at 60℃.
6. Add TypeⅡnucleic acid dye and swirl the flask.
7. Pour agarose onto gel plate in a setting tray with appropriately-sized combs already fixed onto it.
8. Solidify at room temperature. It can be stored at 4℃ for a short period of time when there is no immediate use.
7. Gel extraction
Experimental idea
After PCR amplification, in theory we can get all possible pathways, namely, all fragments, as shown in Table 5, which begin with site 2 and end up with site 8. Then carry out the gel electrophoresis and gel extraction of PCR products, respectively cut gel volume of 200 bp (150 bp and 200 bp are so close as to cut together), 250 bp and 350 bp (300 bp and 350 bp are so close as to cut together), then extract separately.
Procedure
1. Cut out the desired DNA band (use UV lamp) and use paper towel to suck the liquid of gel surface.
Note:
1. At this time attention should be paid to remove the gel that contains no desired DNA, minimize gel volume as far as possible to improve the DNA recovery.
2. Minimize UV exposure time to DNA, in order to prevent DNA damage.
2. Shred the gel to speed up the step 6 of melting time, which improves the recovery rate of DNA.
3. Weigh out the gel, calculate the volume. Carry out the calculation with 1 mg=1 L.
4. Add 3 volumes of Buffer DE-A, 1.5 volumes of Buffer DE-B and 1 volume of isopropanol to the gel.
5. Heat the gel at 75℃( the agarose gel with a low melting point can be heated at 45℃). At this time, vortex intermittently to mix, making the gel melt fully (about 6 ~ 10 minutes).
Note:
The gel must be fully melted, otherwise it will seriously affect the recovery rate of DNA.
6. Add Buffer DE-A, Buffer DE-B (the volume of the latter is 1/2 of the former) to the gel melting
liquid and mix. When separating the DNA fragments that are shorter than 400 bp , add isopropyl whose
volume is 1/3 of the Buffer DE-A.
7. Place Spin Column of the kits on the Collection Tube.
8. Transfer the previous solution of step 6 into Spin Column, centrifuge at 12000×g for 1 minute
(if there remains liquid in the Column Spin, the centrifugal rate can be appropriately increased and
centrifuge for another 1 minute),
then discard the filtrate.
Note:
The recovery rate of DNA can be improved if the filtrate can be centrifuged in Spin Column once again.
9. Add 500 µl Buffer W1 into Spin Column, centrifuge at 12000×g for 30 seconds, then discard the filtrate. 10. Add 700 µl Buffer W2 into Spin Column, centrifuge at 12000×g for 30 seconds, then discard the filtrate.
11. Repeat step 7,centrifuge at 12000×g for 1 minute.
12. Transfer the Spin Column into a clean 1.5ml microfuge tube and let it stand for 3 minutes. To elute
the DNA, add 30 µl Eluent or 60℃ water. Let it stand for 1 minute at room temperature.
13. Centrifuge at 12000×g for 1 minute for DNA elution.
14. Suck out the liquid from the microfuge tube and add in the Spin Column. Repeat the step 10 to get the
products.
8.Transformation and filtering of target genes
Procedure
Carry out the enzyme digestion operation after getting the target genes. According to  the design of PCR primers, both ends of target genes contain the Pst I and EcoR I -HF enzyme sites, while the plasmid also contains this two enzyme sites. Therefore, the enzyme digestion is implemented simultaneously on target genes and plasmid for 2 hours(the enzyme digestion system is as shown in Table 6).
Import the target genes into the plasmids, so under the same system, they connect for 12 hours( as shown in Table 7).
Add the plasmid to competent cells, namely, the plasmid assembled with fragments of different bp are put into different tubes, with 50 competent cells each at room temperature for 20min. Then incubate in water bath at 42℃ for 40s. On the super net work, coat the cells on the solid medium.
1. Take e. coli that contains no target genes to Chl-  medium, coating evenly.
2. Take e. coli that contains no target genes to Chl+ medium, coating evenly.
3. Take e. coli that contains the after-enzyme-digestion plasmids to Chl+ medium, coating evenly.
4. Take e. coli that contains target genes to Chl+ medium, coating evenly.
After coating, incubate for around 16h.
After cultivating, observe the growth of e. coli in plate, pick out the colonies that we need to a liquid medium, then put it into the shaker in oscillation, oscillation for 12-16 hours.
9.Preparation of LB culture medium
We need to prepare 2 bottles of 200ml liquid medium and 2 bottles of 200ml solid medium.
1. Prepare 900ml liquid medium at first, according to the formula shown in Table 8.
2. Each take 200ml from the culture medium and put separately into four conical flask.
3. Label two bottles as solid culture medium, add respectively 3g agarose powder and vortex.
4. Seal four conical flasks with kraft paper and carry out high-pressure steam sterilization with culture dishes.
5. After sterilization, pour plate on the clean workbench, namely, pour the solid medium into Petri dishes, with about 10-15ml medium in each dish.
6. Preparation of 5 plates without chloramphenicol and 15 plates containing chloramphenicol (35 µg/ µL) tablet.
10. Plasmid purification
Procedure
After the multiplication of e. coli , we need to extract the plasmid in e. coli, the purification procedure is as follows:
1. Transfer the medium that in the test tube into the corresponding centrifugal tube, fill and cover the tube to centrifuge at 12000×g for 2 min.
2. Discard the supernatant fluid carefully.
3. Fill in the centrifugal pipe, centrifuge and discard repeatedly until the medium finished. At this time, there is bacteria depositing at the bottom of the centrifuge tube, so the following operation is as follows:
1. Add in 250µL Buffer S1 (dangling sample) and vortex until the precipitation disappears(a confirmation of having added RNase A in the Buffer S1).
2. Add in 250µL Buffer S2, mildly and fully upside down 4-6 times, mix evenly to make bacteria sufficient cracked (this step should not be more than 5 min).
3. Add in 350µL Buffer S3, mildly and fully upside down 6-8 times, mix evenly and centrifuge at 12000×g for 10 min.
4. Draw the supernatant fluid in step 3 and transfer to the preparation tube (put in 2 min centrifugal tube), centrifuge at 12000×g for 1 min, discard the filtrate.
5. Place the preparation tube back to the centrifugal tube, add 500µL Buffer W1, centrifuge at 12000×g for 1 min, discard the filtrate.
6. Place the preparation tube back to the centrifugal tube, add 700µL Buffer W2, centrifuge at 12000×g for 1 min, discard the filtrate.
7. Place the preparation tube back to the centrifugal tube, add 700µL Buffer W2, centrifuge at 12000×g for 1 min, discard the filtrate.
8. Place the preparation tube back to the centrifugal tube, empty from 1 min.
9. Place the preparation tube back to the new 1.5 ml centrifugal tube, then add 30µL ddH2O and let it stand for 2 min. Centrifuge at 12000×g for 2 minutes. Then draw the fluid back to the centrifugal tube and centrifuge at 12000×g for 2 minutes.
10. Plasmid cryopreservation.
After getting the target genes, send to the company for sequencing.
TEAM
Members
Instructors
Attributions
Acknowledgement
PROJECT
Pathway and Design
Result
Future Work
PARTS
Biobrick
Protools
HUMAN
PRACTICES
Society
New Power
Questionnaire
MOOC
Art
SAFETY
Safety
NOTEBOOK
Notebook
Lab Pictures
Copyright 2016 © AHUT iGEM Team , All rights reserved .
