Team:AHUT China/Project/Notebook
Document
Notebook
Notebook
Experimental preparation
1. The synthesis of all sites and lines (single-stranded DNA fragments synthesized by the company)
2. The design of PCR primers (the design of 2.8 primer )
3. The purchase of gel extraction kit
Experimental procedure
1. Dilute all sites and lines to 50μmol/l.
2. Bridge PCR, namely, the sites and lines were connected by complementary base pairing, and the rest single fragments were then filled.
3. DNA ligation, that is, the "gap" on the double-stranded DNA was connected with T4 DNA ligase.
4. PCR reaction.
5. DNA gel electrophoresis.
6. Find out the corresponding pathways (or the bp length), gel extraction.
7. Enzyme digestion was performed on the extraction productsof different length by PstⅠand EcoRⅠ-HF.
8. The same enzyme digestion was carried out on the plasmids.
9. After the enzyme digestion, the DNA fragments were mixed and connected with the corresponding plasmids.
10. Transformation, that is, the reconnected plasmids were delivered into E.coli.
11. Screening. Screen out of the strains that we need in the medium containing chloramphenicol.
12. The screened strains were amplified in liquid medium, and then take the plasmids out of E.coli.
13. Identify the plasmids by PCR and enzyme digestion.
14. Sequencing (send to the company).
Experimental records
2016.6.2
Dilution of eight sites and thirteen lines: Firstly, the samples were centrifuged at 12000×g, then we added deionized water, diluting it to 50µmol/l and centrifuged again. Finally, sample cryopreservation was neededfor the future experiments.
2016.6.3
Morning:
Thaw the samples to do two experiments ofBridge PCR: 1-8 sites and 13 lines of each were taken 1µl in the A.B test tubes and matched into 50µl system, the specific data are as follows:
Thaw the samples to do two experiments ofBridge PCR: 1-8 sites and 13 lines of each were taken 1µl in the A.B test tubes and matched into 50µl system, the specific data are as follows:
Afternoon：
Gel extraction of DNA:
A.B groups were combined, then we added 300µl Buffer DE-A  Do the following steps in order:
1. Add in 100µl Buffer DE – B
2. Add in 100µl isopropyl alcohol (as the experimental DNA fragments are short)
3. Add the liquid into DNA preparation tube (put in 2µl centrifuge tube) and centrifuge at 12000×g for 1 minute, then discard the filtrate.
4. Put the preparation tube back to centrifuge tube, add in 500µl Buffer W1 and centrifuge for 1 minute, then discard the filtrate.
5. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then discard the filtrate.
6. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, thencentrifuge again for 2 minutes, put the preparation tube to the 1.5µl centrifuge tube and open the reagent tube cap to volatilize ethanol for 3 minutes .
7. Add 30µl deionized water, centrifuge for 1 minute.
8. Suck out the sample and put into the column again, centrifuge for 1 minute.
9. Sample cryopreservation
DNA ligation experiment:
Take 10µlreaction system, the specific systemis as follows:
2016.6.4
DNA gel extraction: recycle the ligation products of yesterday.
90µl deionized water was added to the 10µl products to be recycled and matched into 100µl system
1. Add in 300µlBuffer DE - A
2. Add in 100µl Buffer DE – B
3. Add in 100µl isopropyl alcohol (as the experimental DNA fragments are short)
4. Add the liquid into DNA preparation tube (put in 2µl centrifuge tube) andcentrifuge at 12000×g for 1 minute, then discard the filtrate.
5. Put the preparation tube back to centrifuge tube, add in 500µl Buffer W1 and centrifuge for 1 minute, then discard the filtrate.
6. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then discard the filtrate.
7. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then centrifuge againfor 2 minutes, put the preparation tube to the 1.5µl centrifuge tube, open the reagent tube cap to volatilize ethanol for 3 minutes.
8. Add 30µl deionized water, centrifuge for 1 minute.
9. Suck out the sample and put into the column again, centrifuge for 1 minute.
10. Sample cryopreservation.
PCR reaction:
We took the extraction samples and matched into 50µl reaction system. The reaction system is as follows:
PCR reactions were carried out under different temperatures, respectively, at 50℃, 53℃, 55℃, 58℃.
PCR reaction procedure is as follows:
The PCR flow chart is as follows:
DNA gel electrophoresis:
Preparation of 3.0% gel, we took 0.75g agarose powder, added 25mlTAE to make up 25ml system. When adding the sample, 5µl product, 1µl loading and 5µl marker were added.
Here is the picture of DNA electrophoresis:
PCR reactions were carried out under different temperatures, respectively, at 50℃, 53℃, 55℃, 58℃.
2016.7.14
Repeat the first operation，1-8 sites and 13 lines of each were taken 1µl in the A.B test tubes and matched into 50µl system, the specific data are as follows:
The following reaction was carried out in the PCR instrument, the setting procedure is as follows:
TThe PCR flow chart is as follows:
Gel extraction of DNA:
A.B groups were combined, then we added 300µlBufferDE-A  Do the following steps in order:
1. Add in 100µl Buffer DE – B
2. Add in 100µl isopropyl alcohol (as the experimental DNA fragments are short)
3. Add the liquid into DNA preparation tube (put in 2µl centrifuge tube) and centrifuge at 12000×g for 1 minute, then discard the filtrate.
4. Put the preparation tube back to centrifuge tube, add in 500µl Buffer W1 and centrifuge for 1 minute, then discard the filtrate.
5. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then discard the filtrate.
6. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, thencentrifuge again for 2 minutes, put the preparation tube to the 1.5µl centrifuge tube and open the reagent tube cap to volatilize ethanol for 3 minutes .
7. Add 30µl deionized water, centrifuge for 1 minute.
8. Suck out the sample and put into the column again, centrifuge for 1 minute.
9. Sample cryopreservation
DNA ligation experiment:
Take 10µlreaction system, the specific systemis as follows:
2016.7.15
DNA gel extraction: recycle the ligation products of yesterday.
90µl deionized water was added to the 10µl products to be recycled and matched into 100µl system
1. Add in 300µlBuffer DE - A
2. Add in 100µl Buffer DE – B
3. Add in 100µl isopropyl alcohol (as the experimental DNA fragments are short)
4. Add the liquid into DNA preparation tube (put in 2µl centrifuge tube) andcentrifuge at 12000×g for 1 minute, then discard the filtrate.
5. Put the preparation tube back to centrifuge tube, add in 500µl Buffer W1 and centrifuge for 1 minute, then discard the filtrate.
6. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then discard the filtrate.
7. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then centrifuge again for2 minutes, put the preparation tube to the 1.5µl centrifuge tube, open the reagent tube cap to volatilize ethanol for 3 minutes.
8. Add 30µl deionized water, centrifuge for 1 minute.
9. Suck out the sample and put into the column again, centrifuge for 1 minute.
10. Sample cryopreservation.
PCR reaction:
We took the extraction samples and matched into 50µl reaction system.
The reaction system is as follows:
PCR reactions were carried out under different temperatures, respectively, at 50℃, 53℃, 55℃, 58℃.
PCR reaction procedure is as follows:
The PCR flow chart is as follows:
DNA gel electrophoresis:
Preparation of 3.0% gel, we took 0.75g agarose powder, added 25mlTAE to make up 25ml system. When adding the sample, 5µl product, 1µl loading and 5µl marker were added. Here is the picture of DNA electrophoresis:
2016.7.19
1.The gel electrophoresis was conducted on the PCR products. We chose the products in 50℃of the first experiment (4, June) and the products in 58℃of the second experiment (15, July) respectively (50µl product +5µl loading, marker 8µl). After that, we cut 200 bp, 250 bp and 350 bp of the gel fragments in these two temperatures. Then they were divided into six recycling tubes. (The order of adding sample is 50 ℃, 58 ℃)
Here is the picture of electrophoresis:
Gel extraction steps:
Firstly weigh out the weight of each gel respectively,
1. Add 3 gel Buffer volume DE - A
2. Add 1.5 gel Buffer volume DE - B
3. Add 1 gel volume isopropyl alcohol (as the experimental DNA fragments are short)
4. Add the liquid into DNA preparation tube (put in 2µl centrifuge tube) andcentrifuge at 12000×g for 1 minute, then discard the filtrate.
5. Put the preparation tube back to centrifuge tube, add in 500µl Buffer W1 and centrifuge for 1 minute, then discard the filtrate.
6. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then discard the filtrate.
7. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then centrifuge again for2 minutes, put the preparation tube to the 1.5µl centrifuge tube, open the reagent tube cap to volatilize ethanol for 3 minutes.
8. Add 30µl deionized water, centrifuge for 1 minute.
9. Suck out the sample and put into the column again, centrifuge for 1 minute.
10. Sample cryopreservation.
2016.7.20
1. PCR was conducted on the six different bp length fragments of yesterday’s gel extraction.
The PCR system is as follows:
The extraction products matched into 50µl system as the table above.
PCR settings are as follows:
The PCR flow chart is as follows:
After the PCR, we carried out the gel electrophoresis. The order of adding sample is as follows:
Here is the picture of electrophoresis:
2. Preparation of culture medium: preparation of 2 bottles of liquid medium and two bottles of 200ml solid culture medium. We prepared 900ml liquid culture medium at first.
Formula is as follows:
Each medium of 200 ml was taken and divided among four conical flasks, including two were labeled as solid culture medium. Then we added respectively 3g agarose powder and oscillated. Then we sealed the bottle with kraft paper and had high-pressure steam sterilization with culture dishes. At the same time we opened the ultraviolet sterilizing lamp of super clean workbench.
Two hours later, we took out the medium and petri dishes that have been sterilized and then poured the plates: firstly the UV lamp was turned off and the fan was opened, then the hands and instruments were disinfected with alcohol. The five medium without chloramphenicol were poured at first, then the 15 plates containing chloramphenicol (the concentration is 35µg/µl). After the solidification of the medium, every five medium was wrapped with plastic wrap and put into the refrigerator to prepare for later use.
2016.7.23
Enzyme digestion was performed on linear carriers and three PCR fragments (200bp, 250bp, 350bp under 58℃) that were extracted in July 20.
Enzyme digestion system is as follows:
After the enzyme digestion, gel electrophoresis was performed. (3.0 % of the gel, with 30ml)
The order of adding sample is as follows:
Here is the picture of electrophoresis:
Then gel extraction, which means cutting out the gel fragments of  corresponding bp length: The bright part of 200bp, 250bp and 350bp in correspondence with pieces of 200bp, 250bp and 350bp respectively. The next is to weigh out the the weight of each gel and put them into 5 recycling tubes.
1. Add 3 gel Buffer volume DE - A
2. Add 1.5 gel Buffer volume DE - B
3. Add 1 gel volume isopropyl alcohol (as the experimental DNA fragments are short)
4. Add the liquid into DNA preparation tube (put in 2µl centrifuge tube) andcentrifuge at 12000×g for 1 minute, then discard the filtrate.
5. Put the preparation tube back to centrifuge tube, add in 500µl Buffer W1 and centrifuge for 1 minute, then discard the filtrate.
6. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then discard the filtrate.
7. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then centrifuge again for2 minutes, put the preparation tube to the 1.5µl centrifuge tube, open the reagent tube cap to volatilize ethanol for 3 minutes.
8. Add 30µl deionized water, centrifuge for 1 minute.
9. Suck out the sample and put into the column again, centrifuge for 1 minute.
10. Sample cryopreservation.
2016.7.24
1. The extraction fragments of yesterday’s enzyme digestion and linear carriers were performed DNA gel electrophoresis, with 3.0 % of the gel, 25ml.
The order of adding sample is as follows:
Here is the picture of electrophoresis:
The experiments of connecting the plasmids and pieces of different bp length were carried out and there was a group that only added linear carrier.
The ligation system is as follows:
2. The transformation of competent cells, that is, to deliver the reconnected plasmid into E. coli, 6 groups of experiment are as the following:
Containing 200bp plasmid    + 50µl competent cells
Containing 250bp plasmid    + 50µl competent cells
Containing 350bp plasmid    + 50µl competent cells plasmid                   + 50µl competent cells
No chloramphenicol         + 50µl competent cells
Containing chloramphenicol   + 50µl competent cells
They were placed at room temperature for 20 min, then put into 42℃ water bath, heat insulation for 40 s. 400 ml liquid medium were added respectively to 10 tubes of 25 ml , then put these tubes in the shaker with rotation speed of 249rpm, temperature of 37.5℃ for 45 min.
The ultraviolet sterilization lamp of super clean workbench was opened for 1 hour in advance, petri dishes were taken out with 1 without chloramphenicol and 5 containing chloramphenicol to do 3 groups of control experiments. Alcohol disinfection at first, then delivered the connected plasmid into the medium with a gun, and uniformly coated with a coating device
that went into a flame to burn sterilization every time. Specific coating is as follows:
Take E.coli without target genes into Chl- medium, coating evenly.
Take E.coli without target genes into Chl+ medium, coating evenly.
Take E.coli containing constructed recombinant plasmid into Chl+ medium, coating evenly.
Take E.coli containing 200bp plasmid into Chl+ medium, coating evenly.
Take E.coli containing 250bp plasmid into Chl+ medium, coating evenly.
Take E.coli containing 350bp plasmid into Chl+ medium, coating evenly.
After coating evenly, put all dishes into the bacterial incubator for one night.
2016.7.25
1. The extraction products that connected in July 15 were performed PCR, respectively in different temperature at 50 ℃, 53 ℃, 55 ℃, 58 ℃. PCR system is as follows:
PCR settings are as follows:
The PCR flow chart is as follows:
After PCR, the products were performed gel electrophoresis, with 3.0 % of the gel, 30ml.
Here is the picture of electrophoresis:
After the gel electrophoresis, we carried out the gel extraction. 150bp, 200bp, 250bp, 350bp strips were cut off at each temperature and put into 16 recycling tubes which was clearly marked in temperature and bp length. Weigh out the weight of gel and record. The gels of the same bp length recovered together, using one preparation tube.
1. Add 3 gel Buffer volume DE - A
2. Add 1.5 gel Buffer volume DE - B
3. Add 1 gel volume isopropyl alcohol (as the experimental DNA fragments are short)
4. Add the liquid into DNA preparation tube (put in 2µl centrifuge tube) andcentrifuge at 12000×g for 1 minute, then discard the filtrate.
5. Put the preparation tube back to centrifuge tube, add in 500µl Buffer W1 and centrifuge for 1 minute, then discard the filtrate.
6. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then discard the filtrate.
7. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then centrifuge again for2 minutes, put the preparation tube to the 1.5µl centrifuge tube, open the reagent tube cap to volatilize ethanol for 3 minutes.
8. Add 30µl deionized water, centrifuge for 1 minute.
9. Suck out the sample and put into the column again, centrifuge for 1 minute.
10. Sample cryopreservation.
2.We opened the bacterial incubator and took out the petri dishes, observed the growth of E. coli on each dish, finding the result was as same as we had expected.
There were a large number of E. coli grown in the Chl- culture medium that added E.coli without target genes
No E.coli grew in the Chl+ culture medium that added E.coli without target genes
No E.coli grew in the Chl+ culture medium that added E.coli containing constructed recombinant plasmid
2 bacterial colonies in the Chl+ culture medium containing E.coli with 200bp plasmid No colony in Chl+ culture medium containing E.coli with 250bp plasmid
3 colonies in the Chl+ culture medium containing E.coli with 350bp plasmid The picture is as follows:
After observing the petri dish, as the results were the same as had expected, we proceeded to the next step: select a single colony on the plates, then let it thrive in the liquid medium. Open the ultraviolet sterilizing lamp of super clean workbench before the experiment. During experiment, we closed the lamp and turned on the fan, took out the 5 tubes of marked liquid medium. The instruments and hands were disinfected before the experiment, then we opened the petri dishes, with tweezers holding the spearhead to pick E. Coli, then put the E.coli together with the spearhead into the corresponding marked liquid medium. (2 colonies were selected from 200bp fragments, 3 colonies from 350bp fragments and marked as 1.2.3 of 200bp, and so on). After choosing colonies, we put 5 tubes into the shaker in oscillation and oscillated for one night.
2016.7.26
1.Enzyme digestion was performed on the 4 bp fragments of yesterday’s gel extraction products, the enzyme digestion system is as follows:
The reaction system of enzyme digestion was under 37℃ for 3-4 hours.
2. The extraction of plasmids in E. coli.
We closed the shaker, took out the 5 test tubes on the experimental  platform, then took 5 centrifuge tubes, labeling with the same number of  test tubes. Medium in the test tubes were poured into the corresponding centrifuge tubes. When filled, the centrifuge tubes were covered and centrifuged at 12000×g for 2 min. After centrifugation, we carefully poured out the supernatant fluid, then filled the centrifuge pipe to centrifuge again, repeated until the test tube medium finished. At that time, there were bacteria sedimentation at the bottom of the centrifuge tube, so the next operation is as follows:
1. Add 250µl Buffer S1 (dangling add samples) in test tubes, oscillate  until the precipitation disappeared (it confirmed that RNase A has been added in Buffer S1).
2. Add in 250µl Buffer S2, mildly and fully upside down 4 to 6 times, mix evenly to make bacteria sufficient cracked, this step should not be more than 5 min.
3. Add in 350µl Buffer S3, mildly and fully upside down 6-8 times, mix evenly, centrifuge at 12000×g for 10 min.
4. Aspirate the supernatant fluid that centrifuged in step 3 and transfer it to the preparation tube (put in 2 min centrifuge tube), centrifuge at 12000 ×g for 1 min, then discard the filtrate.
5. Put the preparation tube back to centrifuge tube, add in 500µl Buffer W1 and centrifuge at 12000×g for 1 minute, then discard the filtrate.
6. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge at 12000×g for 1 minute, then discard the filtrate.
7. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge at 12000×g for 1 minute, then discard the filtrate.
8. Put the preparation tube back to centrifuge tube, then centrifuge again for 1minute.
9. Put the preparation of pipe back to the new 1.5 ml centrifuge tube, add 30µl ddH2O, let it stand for 2 min, centrifuge at 12000×g for 2 min. Suck the liquid back into the preparation tube, centrifuge at 12000×g for 2 min.
10. Sample cryopreservation
3.The extraction products above performed PCR identification, PCR system is as follows (25µl system):
PCR settings are as follows:
The PCR flow chart is as follows:
After the PCR, we carried out the gel electrophoresis. The order of adding sample is as follows:
Here is the picture of electrophoresis:
2016.7.27
1. 4 fragments of yesterday’s enzyme digestion were performed DNA gel electrophoresis, with 3.0% of the gel, 30ml.
The order of adding sample is as follows:
Here is the picture of electrophoresis:
After the gel electrophoresis, we cut corresponding fragments respectively in bp length: fragments of 150 bp correspond to bright strips of 150bp. Then we put them in the marked centrifuge tube and weigh out the weight of gel. The steps of gel extraction are as follows:
1. Add 3 gel Buffer volume DE - A
2. Add 1.5 gel Buffer volume DE - B
3. Add 1 gel volume isopropyl alcohol (as the experimental DNA fragments are short)
4. Add the liquid into DNA preparation tube (put in 2µl centrifuge tube) andcentrifuge at 12000×g for 1 minute, then discard the filtrate.
5. Put the preparation tube back to centrifuge tube, add in 500µl Buffer W1 and centrifuge for 1 minute, then discard the filtrate.
6. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then discard the filtrate.
7. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge for 1 minute, then centrifuge again for2 minutes, put the preparation tube to the 1.5µl centrifuge tube, open the reagent tube cap to volatilize ethanol for 3 minutes.
8. Add 30µl deionized water, centrifuge for 1 minute.
9. Suck out the sample and put into the column again, centrifuge for 1 minute.
10. Sample cryopreservation.
2. The ligation experiment then was performed on the enzyme-digested products after the gel extraction. The ligation system is as follows:
3. We took out the plasmids (5 tubes) of the gel extraction in July 26 and conducted the enzyme identification experiment, which identify whether the imported fragments in plasmids are what we need. The enzyme digestion system is as follows:
The enzyme digestion was under 37℃ for 3-4 hours.
After the enzyme digestion, gel electrophoresis was performed. (3% of the gel, with 50ml)
The order of adding sample is as follows:
Here is the picture of electrophoresis:
After PCR and enzyme digestion identification, we found that there were the required fragments in the plasmids , so 200bp 1, 350bp 1 and 350bp 2 were taken respectively 10µl and installed in centrifuge tube to send to the company for sequencing.
2016.7.29
The transformation of competent cells, that is, to deliver the reconnected plasmid in July 27 into E. coli.
4 groups of experiments are as the following:
Containing 150bp plasmid + 50µl competent cells
Containing 200bp plasmid + 50µl competent cells
Containing 250bp plasmid + 50µl competent cells
Containing 350bp plasmid + 50µl competent cells
They were placed at room temperature for 20 min, then put into 42℃ water bath and insulated for 40 s. Liquid medium were filled in 5 tubes of 25 ml, then put these tubes in the shaker with rotation speed of 249rpm, temperature of 37.5℃ for 45 min.
The ultraviolet sterilization lamp of super clean workbench was opened 1 hour in advance, petri dishes were taken out with four containing chloramphenicol. Alcohol disinfection at first, then delivered the connected plasmid into the medium with a gun, and uniformly coated with a coating device that went into a flame to burn sterilization every time. Specific coating is as follows:
The 150bp plasmid was added into the medium of Chl+, coating evenly.
The 200bp plasmid was added into the medium of Chl+, coating evenly.
The 250bp plasmid was added into the medium of Chl+, coating evenly.
The 350bp plasmid was added into the medium of Chl+, coating evenly.
After coating evenly, put all dishes into the bacterial incubator in one night.
2016.7.30
1. We opened the bacterial incubator and took out the petri dishes, observing the growth of E. coli on each dish. The results are:
No colony in Chl+ culture media containing 150bp plasmid
1 bacterial colony in the Chl+ culture medium containing 200bp plasmid
4 colonies in Chl+ culture media containing 250bp plasmid
No colony in the Chl+ culture medium containing 350bp plasmid
2. After observing the petri dishes, we proceeded to the next step: select a single colony on the plates, then let it thrive in the liquid medium. The ultraviolet sterilizing lamp of super clean workbench was opened before the experiment. During experiment, we closed the lamp and turned on the fan, the 5 tubes of marked liquid medium was taken out. The instruments and hands were disinfected before the experiment, then we opened the petri dishes, with tweezers holding the spearhead to pick E. Coli. Then put the E.coli together with the spearhead into the corresponding marked liquid medium (1 colony was selected from 200bp fragments, 4 colonies from 250bp fragments, each marked as 1 of 200bp, 1.2...of 250bp and so on). After choosing colonies, put 5 tubes into the shaker in oscillationand oscillated for one night.
2016.7.31
1. The extraction of plasmids in E. coli.
We closed the shaker and took out the 5 test tubes on the experimental  platform, then took 5 centrifuge tubes, labeling with the same number of  test tubes. Liquid medium in the test tubes were poured into the corresponding centrifuge tubes. When filled, the centrifuge tubes were covered and centrifuged at 12000×g for 2 min. After centrifugation, we carefully poured out the supernatant fluid, then filled the centrifuge pipe to centrifuge again, repeated until the test tube medium finished. At that time there were bacteria sedimentation at the bottom of the centrifuge tube, so the next operation is as follows:
1. Add 250µl Buffer S1 (dangling add samples) in test tubes, oscillate  until the precipitation disappeared (it confirmed that RNase A has been added in Buffer S1).
2. Add in 250µl Buffer S2, mildly and fully upside down 4 to 6 times, mix evenly to make bacteria sufficient cracked, this step should not be more than 5 min.
3. Add in 350µl Buffer S3, mildly and fully upside down 6-8 times, mix evenly, centrifuge at 12000×g for 10 min.
4. Aspirate the supernatant fluid that centrifuged in step 3 and transfer it to the preparation tube (put in 2 min centrifuge tube), centrifuge at 12000 ×g for 1 min, then discard the filtrate.
5. Put the preparation tube back to centrifuge tube, add in 500µl Buffer W1 and centrifuge at 12000×g for 1 minute, then discard the filtrate.
6. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge at 12000×g for 1 minute, then discard the filtrate.
7. Put the preparation tube back to centrifuge tube, add in 700µl Buffer W2 and centrifuge at 12000×g for 1 minute, then discard the filtrate.
8. Put the preparation tube back to centrifuge tube, then centrifuge again for 1minute.
9. Put the preparation of pipe back to the new 1.5 ml centrifuge tube, add 30 µlddH2O, let it stand for 2 min, centrifuge at 12000×g for 2 min. Suck the liquid back into the preparation tube, centrifuge at 12000×g for 2 min.
10. Sample cryopreservation
2.The extraction products above performed PCR identification, PCR system is as follows (25µl system):
PCR settings are as follows:
The PCR flow chart is as follows:
After the PCR, we carried out the gel electrophoresis (3.0 % of the gel, with 25ml).
The order of adding sample is as follows:
Here is the picture of electrophoresis:
2. We took out the plasmids (5 tubes) of the gel extraction and conducted the enzyme identification experiment, which identify whether the imported fragments in plasmids are what we need. The enzyme digestion system is as follows:
The enzyme digestion was under 37℃ for 3-4 hours.
After the enzyme digestion, gel electrophoresis was performed. (3.0 % of the gel, with 25ml)
The order of adding sample is as follows:
Here is the picture of electrophoresis:
After PCR and enzyme digestion identification, we found that there were the required fragments in the plasmids , so 200bp 1,250bp 1,250bp 2 ,250bp 3 and 250bp 4were taken respectively 10µl and installed in centrifuge tube to send to the company for sequencing.
AHUT_CHINA
Copyright 2016 © AHUT iGEM Team , All rights reserved .
