<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_AHUT_China_Project_Result skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:AHUT China/Project/Result</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Document</TITLE></P><DIV id="nav"><DIV id="banner"><DIV id="menu"><DIV id="name"><P>Result</P></DIV></DIV></DIV><DIV id="content"><DIV id="con_inner"><P style="position:absolute;margin:10px;font-size:3em;font-family:Segoe UI;font-weight:100;position:relative;letter-spacing:0em;margin-left:0%">Result</P><P style="font-size:2.5em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">The acquisition of target genes(bridge-PCR)</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">Firstly, DNA polymerization shall be conducted, namely linking the “site” and the “line” through complementary base pairing and then filling the rest single fragment ares. Secondly, DNA ligation, that is connecting the “gap” on DNA double-strands with T4 DNA ligase. Then we recycle the ligated products and subsequently amplify target genes through PCR using the recycled fragments from gel extraction as templates， which is followed by DNA Gel Electrophoresis. The following figure is the result:</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">
On the left side of the chart is the Marker, lane 1、2、3、4 represent DNA fragments under the annealing temperature of 50℃、53℃、55℃、58℃ respectively. If bright bands were detected from 150bp to 300bp, then it proves the existence of target genes.
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">The DNA Gel Electrophoresis shall be conducted on the PCR products that were obtained previously. We will choose the 50℃ and 58℃ of the first experiment respectively ( 50ul products+50ul loading, marker 8ul), and cut gel of 200bp、250bp、350bp under both temperatures of 50℃ and 58℃. Then put them into 6 recycling tubes to do the gel extraction.
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">Here are the pictures of Gel Electrophoresis:
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem"> Using the 6 pieces of different DNA fragments as templates, PCR is conducted by the site 2 and 8-specific primers and followed by gel electrophoresis to check if the target genes exist. Here is the figure.
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">Extract plasmids using
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">From 150bp to 350bp, the 6 bands have all shown in the above picture, which proved the existence of DNA fragments of 200bp, 250bp and 350bp, namely, the existence of target genes.</P><P style="font-size:2.5em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">The acquisition of recombinated plasmids</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">We then recycle the DNA bands of correspondent length（58℃200bp, 250bp, 350bp） and conduct enzyme digestion on the recycled DNA fragments and plasmids pSB1C3 with Pst1 and EcoR1-HF under 37 ℃ respectively. This lasted for 3-4 hours and after that we will do gel electrophoresis. (3% gel, 30ml)
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">Here is the picture:
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem"> Because of the small quantity, lane 1、2、3 are hard to be found through eyes, however, they do exist as correspondence to bands of specific length. 
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">    Then we extracting DNA from agarose gel, which means cutting out gel of correspondent length of DNA: The bright band of 200bp, 250bp and 350bp in correspondence with pieces of 200bp,250bp and 350bp respectively. The next is to weigh the three gels and put them into 3 recycling tubes to recycle.
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">Also, in order to test the efficiency of enzyme digestion, DNA Gel Electrophoresis shall be conducted on the 3 pieces of DNA and the digested plasmids that have obtained through enzyme digestion and gel recycling, with 25ml of 3% gel.
</P><P style="font-size:2.5em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">Picture of Gel Electrophoresis</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">Owing to the small quantity, it is hard to be seen, the following part is the experiment of connecting the plasmids and pieces of different length of DNA, namely, inserting DNA pieces of different length into plasmids. And among them, one group will only be added linear vectors. Keep it connecting under 16℃ for overnight. 
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">Then we transformed the ligated plasmids to E. coli and put them in the bacteriological incubator to cultivate for about 12-16 hours.
 </P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">   After cultivation, we open the bacteriological incubator and fetch the culture plates to observe their growth. The results turned out to meet our expectations:
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">(1)	No E. coli grew on the Chl+ medium with Ecoli that no plasmids were transformed into it.
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">(2)	No E. coli grew on the Chl+ medium with Ecoli that linearized plasmids pSB1C3 were transformed into the Ecoli, which excludes the influence of the self-ligating of linearized plasmids 
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">(3) Two bacteria colonies were found in the Chl+ medium where Ecoli with plasmids of 200bp were imported.
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">(4) Three bacteria colonies were found in the Chl+ medium where Ecoli with plasmids of 350bp were imported.
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">(5) No Ecoli grow on the Chl+ medium where Ecoli with plasmids of 250bp were imported.
</P><P style="font-size:2.5em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">Here are the pictures:</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">
Having observed the culture plates and got the expected results, the next step can be taken. We will pick out the single bacterial colony on the solid cultural medium(two bacterial colonies that were on medium with 200bp will be labeled 200bp1, 200bp2, three bacterial colonies that were on medium with 350bp will be labeled 350bp1, 350bp2, 350bp3). Then single bacterial colony shall be picked out and monoclonal strains will be inoculated into LB liquid cultural medium to amplify and then get the recombinated plasmids.</P><P style="font-size:2.5em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">The verification of recombinated plasmids </P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">Extract plasmids using Axyprep Plasmid Miniprep Kit, followed by verification of the recombinated plasmids via enzyme digestion and PCR, and at last, sequencing of the correct recombinated plasmids will be performed. The following are the pictures:</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">During PCR verification, 5 expected bands in corresponding lengths are seen, which means that target genes have already been inserted into the plasmids. In the picture of Electrophoresis of gel extraction, the bright band on the top are plasmids and target genes are in the bottom which were not obvious but visible and the bands are corrected. The verification of recombinated plasmids proves the existence of target genes.

</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">After the verification of PCR and enzyme digestion, the corrected plasmids were delivered to the Nanjing genscript biotechnology co., LTD to do sequencing analysis.
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">the sequencing results were as follows：
</P><P style="font-size:2em;font-weight:lighter;font-family:Segoe UI;line-height:1.5;text-indent:2rem">The results of the sequencing analysis reflected the paths and nodes information on the two DNA strands. It can be seen that the two paths are &lt;2-4-8&gt; and &lt;2-6-8&gt;. After transforming the path lengths to decimal digits and calculating, we get the actual path lengths of the two respectively: 28mile for &lt;2-4-8&gt; and 26mile for &lt;2-6-8&gt;. In this project, the nodes of &lt;2-6-8&gt; are the least and the path is also the shortest, thus, the optimal solution is &lt;2-6-8&gt;.
</P></DIV></DIV><DIV id="footer"><DIV class="conent"><P style="font-size:6em;color:rgba(0,0,0,0.8);position:absolute">AHUT_CHINA</P><P id="foot_copyright">Copyright 2016 © AHUT iGEM Team , All rights reserved . </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>