<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Johns_Hopkins_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Johns_Hopkins/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Johns_Hopkins/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Johns_Hopkins/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Johns Hopkins/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><UL id="bar-nav"><LI><A href="https://2010.igem.org/Team:Johns_Hopkins">Home</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Team">Team</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Project">Project</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Modeling">Modeling</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Parts">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Device">Device</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Notebook">Notebook</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Safety">Safety</A></LI></UL></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Abstract"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#The_Voltage-Gated_Calcium_Channel"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">The Voltage-Gated Calcium Channel</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Assumptions"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Assumptions</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Constant_Instantiation"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Constant Instantiation</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#The_Calculation_of_Voltage_Experienced_at_the_Cellular_Level"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">The Calculation of Voltage Experienced at the Cellular Level</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Calculation_of_Charge_Intake"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Calculation of Charge Intake</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Derivation_of_Calcium_Intake_and_Post-Stimulation_Calcium_Concentration"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Derivation of Calcium Intake and Post-Stimulation Calcium Concentration</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Calcium-Induced_YFP_Transcription.2FTranslation.3D"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Calcium-Induced YFP Transcription/Translation=</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Assumptions_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Assumptions</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Setup_of_the_System_and_Solution_.28YFP_concentration.29"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Setup of the System and Solution (YFP concentration)</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H2><P>The expressed purpose of this model is to take the data obtained through experiment and simulate the actions of <I>saccharomyces cerevisiae</I> at different voltages. The yeasts, in SC complete media, were electroporated at various voltage to induce calcium ion intake through L-type voltage-gated calcium channels.<SUP>[1]</SUP> As yeasts have no sarcoplasmic reticulum, calcium will be imported largely from outside the cell. by following precedents set by the Reuter-Stevens (RS) and the Goldman-Hodgkin-Katz (GHK) models, the total charge in calcium imported into the cell during the period of electric stimulation was derived from the current-voltage relationship. The concentration of calcium taken into the cell was derived from the total charge imported, and used to determine the output of yellow fluorescent protein (YFP). the model consists of two major parts: the calculation of calcium intake and the determination of transcription/translation levels induced by calcium. These results fit the data presented in the notebook.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 0. A mathematical model of voltage-gated calcium-controlled transcription and expression in <I>S. cerevisiae</I>. Note: Degradation constants not shown.</DIV></DIV></DIV></DIV><P>Like the Valencia team in 2009, we utilized the stochastic approach through the analysis of gating through probabilistic variables. In fact, the stochastic model they utilized is not entirely accurate; recent literature suggests that the <I>f<SUB>2</SUB></I> inactivation gate does not play a role in yeast voltage-gated inactivation as much as in mammalian systems. By far, the largest improvement upon the mathematical modeling done by Valencia exists as that of the gene-to-protein modeling. Using the control system depicted in this model, we are able to predict the expression, and therefore, the luminescence of the yeast. The next step in advancing this prediction would be to fine-tune the control mechanism, preferably with Simulink.
</P><H2><SPAN class="mw-headline" id="The_Voltage-Gated_Calcium_Channel">The Voltage-Gated Calcium Channel</SPAN></H2><H3><SPAN class="mw-headline" id="Assumptions">Assumptions</SPAN></H3><UL><LI>[Ca<SUP>2+</SUP>]<SUB>out</SUB> is in excess compared to [Ca<SUP>2+</SUP>]<SUB>in</SUB>.<SUP>[12]</SUP></LI><LI>The experiment takes place at room temperature (25<SUP>o</SUP>C).
</LI><LI>the L-type Ca<SUP>2+</SUP> (Cch1) channel is homologous to those found in other eukaryotes.<SUP>[3]</SUP></LI><LI>Any Ca<SUP>2+</SUP> that enters into the channel will exit into the cell.
</LI><LI>The channel is specific and selective for Ca<SUP>2+</SUP>.
</LI><LI>The cell is at steady state previous to stimulation.
</LI><LI>There are two activation gates ('a') and two inactivation gates ('i').
</LI><LI>Gates of the same type are identical and independent in action.
</LI><LI>There are approximately 100 calcium channels per yeast cell.[1]
</LI></UL><H3><SPAN class="mw-headline" id="Constant_Instantiation">Constant Instantiation</SPAN></H3><P>the overall model of the calcium channel can be subdivided into its constituent gates: the activation gate and the inactivation gate. Collapsed into chemical formulas, the channel's interactions with the calcium can be thought of as such:</P><P>Based off the assumption that any bound calcium must travel inside, the result:</P><P>These may be further subdivided into the activation and the inactivation gates:</P><DIV style="text-align:center;"> and </DIV><P>The rate constants are provided below:</P><H3><SPAN class="mw-headline" id="The_Calculation_of_Voltage_Experienced_at_the_Cellular_Level">The Calculation of Voltage Experienced at the Cellular Level</SPAN></H3><P>Although the electrodes registered voltages in volts, the individual changes felt by the individual cells in the SC media are estimated to be in millivolts. In particular, a recursive algorithm was used to calculate the optimal voltage wherein charge (from calcium) transport across the membrane occurred at the greatest rate. The MATLAB function fsolve was utilized to find this point. It reflects the observed data as seen in the notebook: the optimal point was found to exist at roughly 0.007-0.008 V, or 7-8 mV.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:257px;"><DIV class="thumbcaption">Figure 1.c. MATLAB algorithm for the deduction of voltage of greatest charge transport</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Calculation_of_Charge_Intake">Calculation of Charge Intake</SPAN></H3><P>The charge intake is usually dependent on two variables: stimulation time and stimulation voltage (received).[6] The dependence can be reduced to one variable by varying the stimulation voltage at fixed stimulation times. Simultaneously, it is useful to construct, as per the RS model, a relationship between the calcium current and membrane voltage, at various stimulation times. From Ohm’s Law, it is known that V = IR. Therefore, the equation may also be written as:</P><P>In this case, the current would consist solely of calcium ions moving into the cell.[12] Also, it is required that the term GV be scaled by the probability of the channel being open, and also by the number of channels per cell. Thus, current may be written as:<SUP>[2],[13],[14]</SUP></P><P>or,</P><P>The Nernst Potential of calcium can be considered to be roughly 123 mV, under physiological conditions. 
exists as the probability that both inactivation gates and both activation gates are open. Therefore,  may be modeled as such:
</P><P>Bear in mind that this, as is the RS model, is a stochastic model. By calculating the steady state values of and from the differential equations:
</P><P>And
</P><P>At steady state, or  and ,
</P><P>The solution of the differential equations above takes the form:
</P><P>Therefore, the current may be written as:
</P><P>The solutions are plotted as such:
</P><P>The reason that the current is negative is that by convention, the inward diffusion of calcium ions is called inward current, since it diffuses into the cell.<SUP>[5]</SUP> Therefore, one must remember to take the negative of the current to obtain its magnitude. The relatively small magnitudes of and render the channel relatively insensitive to inactivation, and as such, the action is mitigated by negative feedback from the calcium binding to calmodulin and calcineurin. This will be discussed further in the Calcium-Induced YFP Transcription/Translation.
</P><H3><SPAN class="mw-headline" id="Derivation_of_Calcium_Intake_and_Post-Stimulation_Calcium_Concentration">Derivation of Calcium Intake and Post-Stimulation Calcium Concentration</SPAN></H3><P>The amount of calcium taken into the cell, and therefore the post-stimulation concentration, can be derived from the current taken at the ideal stimulation voltage (8 mV), during the ideal shock time (60 s). The idea behind the equation is simple: the total instantaneous charge will be taken, divided by 2x elementary charge, then divided by Avogadro’s number to obtain the number of moles taken in. This is then divided by the volume of the cell (1.25e-13 L) to obtain the post-stimulation concentration.<SUP>[1],[10]</SUP></P><P>This concentration of calcium will be the input from which YFP transcription is derived.
</P><H2><SPAN class="mw-headline" id="Calcium-Induced_YFP_Transcription.2FTranslation.3D">Calcium-Induced YFP Transcription/Translation=</SPAN></H2><H3><SPAN class="mw-headline" id="Assumptions_2">Assumptions</SPAN></H3><UL><LI>Entry into the nucleus occurs by diffusion, on the scale of several minutes.
</LI><LI>YFP takes more than a day to significantly degrade.
</LI><LI>Calcium first binds to Calmodulin; it is assumed that all of this bound calcium will bind to Calcineurin.
</LI><LI>The cells used are diploid yeast cells.
</LI></UL><P>===Constant Instantiation===<SUP>[4],[7],[8],[11],[12]</SUP></P><H3><SPAN class="mw-headline" id="Setup_of_the_System_and_Solution_.28YFP_concentration.29">Setup of the System and Solution (YFP concentration)</SPAN></H3><P>The system as pictured in the abstract can be represented as a system of differential equations:
</P><P>This equation can easily be placed in vector-matrix form, which can be solved numerically by MATLAB. Observe:
</P><P>The MATLAB code is as follows:
</P><P>The solution of the problem:
</P><P>From this graph, it is evident that the YFP concentration will grow to detectable levels by the time it is imaged, 3-8 hours later. The linearly increasing portion of the YFP concentration curve will taper off and decrease after a day or so, once degradation begins to take effect.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>1. <B>Cch1 Restores Intracellular Ca2+ in Fungal Cells during Endoplasmic Reticulum Stress</B></P><P><I>J.Biol. Chem. 2010 285: 10951-10958.First Published on February 1, 2010</I></P><P>2. <B>Comparison of Greenhouse Grown, Containerized Grapevine Stomatal Conductance Measurements Using Two Differing Porometers</B></P><P><I>Thayne Montague*, Edward Hellman, and Michael Krawitzky</I>
Texas AgriLife Research and Extension Center, Lubbock, Texas, U.S.A. 79403-6603
Department of Plant and Soil Science, Texas Tech University, Lubbock, Texas, U.S.A.</P><P>3. <B>Comparative Analysis of the Kinetic Characteristics of L-Type Calcium Channels in Cardiac Cells of Hibernators</B></P><P><I>Alexey E. Alekseev,*t Nick 1. Markevich,* Antonina F. Korystova,* Andre Terzic,* and Yuri M. Kokoz*</I>
Institute of Theoretical and Experimental Biophysics,* Russian Academy of Sciences, Pushchino, Moscow Region, Russia; and *Division of Cardiovascular Diseases, Departments of Internal Medicine and Pharmacology, Mayo Clinic, Mayo Foundation, Rochester, Minnesota, USA Biophysical Journal Volume 70 February 1996 786-797
</P><P>4. <B>Determination of in-vivo cytoplasmic orthophosphate concentration in yeast</B></P><P><I>Uwe Theobald, Jochen Mohns and Manfred Rizzi From the issue entitled &quot;Published in conjunction with Biotechnology Letters&quot; BIOTECHNOLOGY TECHNIQUES</I>
Volume 10, Number 5, 297-302
</P><P>5. <B>Functional Expression of a Vertebrate Inwardly Rectifying K+ Channel in Yeast</B></P><P><I>Weimin Tang, Abdul Ruknudin, Wen-Pin Yang, Shyh-Yu Shaw,Aron Knickerbocker, and Stephen Kurtz'Molecular Biology of the Cell Vol. 6, 1231-1240, September</I>
1995
</P><P>6. <B>Increased large conductance calcium-activated potassium (BK) channel expression accompanied by STREX variant downregulation in the developing mouse CNS</B></P><P><I>Stephen H-F MacDonald1,4 , Peter Ruth2 , Hans-Guenther</I>
Knaus3 and Michael J Shipston1 1 Centre for Integrative Physiology, School of Biomedical Science, Hugh Robson
Building, University of Edinburgh, Edinburgh, Scotland, EH8 9XD, UK BMC Developmental Technology, 2006.
</P><P>7. <B>Mathematical modeling of calcium homeostasis in yeast cells</B></P><P><I>Jiangjun Cui, Jaap A. Kaandorp Section Computational Science, Faculty of Science, University of Amsterdam, Kruislaan 403, 1098 SJ Amsterdam, The Netherlands</I></P><P>8. <B>Measurement of the binding of transcription factor Sp1 to a single GC box recognition sequence.</B></P><P><I>J Letovsky and W S Dynan Department of Chemistry and Biochemistry, University of Colorado, Boulder 80309. Nucleic Acids Res. 1989 April 11; 17(7): 2639–2653.</I></P><P>9. <B>Metabolic consequences of a species difference in Gibbs free energy of Na+/Ca2+ exchange: rat versus guinea pig</B></P><P><I>P. J. Cooper, M.-L. Ward, P. J. Hanley, G. R. Denyer, and D. S. LoiselleDepartment of Physiology, Faculty of Medicine and Health Science, University of Auckland, Private Bag 92019, Auckland, New Zealand</I>
Am J Physiol Regul Integr Comp Physiol 280 Vol. 280, Issue 4, R1221-R1229, April 2001
</P><P>10.<B>Permeation as a Diffusion Process</B></P><P><I>August 27, 2000 Bob Eisenberg Dept. of Molecular Biophysics and Physiology Chapter 4 in <B>Biophysics Textbook On Line &quot;Channels, Receptors, and Transporters&quot;</B> Louis J. DeFelice, Volume Editor <A href="http://biosci.umn.edu/biophys/OLTB/Channel.html" class="external free" rel="nofollow">http://biosci.umn.edu/biophys/OLTB/Channel.html</A></I></P><P>11. <B>Promoter escape limits the rate of RNA polymerase II transcription and is enhanced by TFIIE, TFIIH, and ATP on negatively supercoiled DNA</B></P><P><I>Jennifer F. Kugel and James A. Goodrich*</I>
Department of Chemistry and Biochemistry, University of Colorado at Boulder, Campus Box 215, Boulder, CO 80309-0215
<I>Communicated by Olke C. Uhlenbeck, University of Colorado, Boulder, CO, June 3, 1998</I></P><P>12. <B>Protein phosphatase type 2B (calcineurin)-mediated, FK506-sensitive regulation of intracellular ions in yeast is an important determinant for adaptation to high salt stress conditions.</B></P><P><I>T Nakamura, Y Liu, D Hirata, H Namba, S Harada, T Hirokawa, and T Miyakawa</I></P><P>13. <B>Quantitative Modeling of Chloride Conductance in Yeast TRK Potassium Transporters</B></P><P><I>Alberto Rivetta,<SUP>*</SUP> Clifford Slayman,* and Teruo Kuroda<SUP>†*</SUP>Department of Cellular and Molecular Physiology, Yale School of Medicine, New Haven, Connecticut; and <SUP>†</SUP>Department of Genomics and Applied Microbiology, Graduate School of Medicine, Dentistry and Pharmaceutical Sciences, Okayama University, Okayama, Japan Biophys J. 2005 October; 89(4): 2412–2426. Published online 2005 July 22. doi: 10.1529/biophysj.105.066712.</I></P><P>14. <B>Sodium channels in cultured cardiac cells</B></P><P><I>A B Cachelin , J E Peyer, S Kokubun<B> July 1, 1983 The Journal of Physiology, </B></I><B>340<I>, 389-401.</I></B></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Johns_Hopkins/Modeling">http://2010.igem.org/Team:Johns_Hopkins/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Johns_Hopkins/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Johns_Hopkins/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Modeling&amp;oldid=207045" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>