<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Johns_Hopkins_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Johns_Hopkins/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Johns_Hopkins/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Johns_Hopkins/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Johns Hopkins/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><UL id="bar-nav"><LI><A href="https://2010.igem.org/Team:Johns_Hopkins">Home</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Team">Team</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Project">Project</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Modeling">Modeling</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Parts">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Device">Device</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Notebook">Notebook</A></LI><LI><A href="https://2010.igem.org/Team:Johns_Hopkins/Safety">Safety</A></LI></UL></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Experiments"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Experiments</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Crz1-GFP_Experiment"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Crz1-GFP Experiment</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Conclusions_and_discussion"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Conclusions and discussion</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-4"><A href="#CDRE_Optimization_Experiment"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">CDRE Optimization Experiment</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Results"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Conclusions_and_Discussion_2"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Conclusions and Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#CDRE_hone_in_Experiment"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">CDRE hone in Experiment</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Observations"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Observations</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Results_2"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Conclusions"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Conclusions</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Characterization_Of_CDRE-RFP_system_in_wild_type_yeast_and_PMC1-YFP_system_in_Vesicular_Pump_Negative_Yeast_Strain"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Characterization Of CDRE-RFP system in wild type yeast and PMC1-YFP system in Vesicular Pump Negative Yeast Strain</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Results_3"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Protocol"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Plasmid_Extraction"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Plasmid Extraction</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Reagents"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Reagents</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Protocol_2"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Yeast_Transformation"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Yeast Transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Reagents_2"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Reagents</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Protocol_3"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#CDRE_Optimization"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">CDRE Optimization</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Protocol_4"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#CDRE_Hone_in"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">CDRE Hone in</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Procedure"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Characterization_Of_CDRE-RFP_System_in_Wild_Type_Yeast_and_PMC1-YFP_system_in_Yeast_Strain_Lacking_Vesicular_Calcium_Pumps"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Characterization Of CDRE-RFP System in Wild Type Yeast and PMC1-YFP system in Yeast Strain Lacking Vesicular Calcium Pumps</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Procedure_2"><SPAN class="tocnumber">2.5.1</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Characterization_Of_CDRE-RFP_in_wild_type_yeast_and_Yeast_Strain_Vesicular_Calcium_Pump_Negative_and_the_PMC1-YFP_system_in_wild_type_yeast_and_Vesicular_Negative_Yeast"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Characterization Of CDRE-RFP in wild type yeast and Yeast Strain Vesicular Calcium Pump Negative and the PMC1-YFP system in wild type yeast and Vesicular Negative Yeast</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Procedure_3"><SPAN class="tocnumber">2.6.1</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Plasmid_Extraction_2"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Plasmid Extraction</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Reagents_3"><SPAN class="tocnumber">2.7.1</SPAN><SPAN class="toctext">Reagents</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Protocol_5"><SPAN class="tocnumber">2.7.2</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Yeast_Transformation_2"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Yeast Transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Reagents_4"><SPAN class="tocnumber">2.8.1</SPAN><SPAN class="toctext">Reagents</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#Protocol_6"><SPAN class="tocnumber">2.8.2</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Plasmid_Mini-Prep"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">Plasmid Mini-Prep</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#Reagents_5"><SPAN class="tocnumber">2.9.1</SPAN><SPAN class="toctext">Reagents</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#Protocol_7"><SPAN class="tocnumber">2.9.2</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#SC_Media"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">SC Media</SPAN></A></LI><LI class="toclevel-3 tocsection-38"><A href="#Reagents_6"><SPAN class="tocnumber">2.10.1</SPAN><SPAN class="toctext">Reagents</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Digestion_and_ligation_of_plasmids"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">Digestion and ligation of plasmids</SPAN></A></LI><LI class="toclevel-3 tocsection-40"><A href="#Plates"><SPAN class="tocnumber">2.11.1</SPAN><SPAN class="toctext">Plates</SPAN></A></LI><LI class="toclevel-3 tocsection-41"><A href="#Protocol_8"><SPAN class="tocnumber">2.11.2</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Experiments">Experiments</SPAN></H2><H3><SPAN class="mw-headline" id="Crz1-GFP_Experiment">Crz1-GFP Experiment</SPAN></H3><P><I>Reason for doing the experiment:</I> To establish whether shocking the cells induces calcium influx leading to CRZ1 nuclear localization.
</P><P><I>Date:</I> 21/7/10
</P><P><I>Location:</I> Imaging Center
</P><P><I>Name of S. cerevisiae strain:</I>
Delta VCX1 Delta PMC1 with Crz1 tagged with GFP
</P><P>Representative images:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Yeast expressing Crz1-GFP</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Similar picture of Crz1-GFP expression</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Conclusions_and_discussion">Conclusions and discussion</SPAN></H4><P>From this experiment it is clear that there is a calcium influx into the cell on application of a voltage across the cell membrane. We have also established that there is a significant influx of Crz1 into the nucleus associated with this calcium influx as we expected. We hypothesis that the reason we saw Crz1 nuclear localisation only in cells very close to the live electrode because the voltage provided by the function generator was too low to reach across the gap, and so a hypothetical ground gets formed near the live electrode. Another reason may be that as our device has the electrodes at a large distance from each other, comparable x.
</P><H3><SPAN class="mw-headline" id="CDRE_Optimization_Experiment">CDRE Optimization Experiment</SPAN></H3><P><I>Reason for doing the experiment:</I>To find an optimum voltage amplitude and shocking time for yeast containing the CDRE-RFP plasmid.<I>Date:</I>10/7/10<I>Location:</I> Imaging Center<I>Name of </I>S. cerevisiae<I> strain:</I>Delta VCX1 Delta PMC1 W303 with CDRE-RFP
</P><P>Representative Images:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">No electrostimulus on yeast without vesicular pumps</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">6 Volts electrostimulus on yeast without vesicular pumps</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">8 Volts electrostimulus on yeast without vesicular pumps</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">10 Volts electrostimulus on yeast without vesicular pumps</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Results">Results</SPAN></H4><TABLE border="1" cellpadding="5" cellspacing="0" style="background: #ffffff;"><TBODY><TR><TH>2s </TH><TH>5s </TH><TH>10s </TH><TH>20s </TH><TH>40s</TH><TH>80s
</TH></TR><TR><TD> 10V </TD><TD> None </TD><TD>None </TD><TD>None </TD><TD>None </TD><TD>None </TD><TD>N/A
</TD></TR><TR><TD> 8V </TD><TD> None </TD><TD>None </TD><TD>None </TD><TD>None </TD><TD>High </TD><TD>High
</TD></TR><TR><TD> 6V </TD><TD> None </TD><TD>None </TD><TD>None </TD><TD>None </TD><TD>Moderate </TD><TD>High
</TD></TR><TR><TD> 4V </TD><TD> None </TD><TD>None </TD><TD>None </TD><TD>None </TD><TD>Low </TD><TD>Low
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Conclusions_and_Discussion_2">Conclusions and Discussion</SPAN></H4><P>First, observe that the channels are inactive for the lowest stimulus durations. Some of these durations are on cellular diffusive timescales, so not enough calcium can enter the cells. Also, cells can quickly move to extrude the excess calcium.
</P><P>Second, consider the channel mechanics. Calcium channel gates alternate between open and closed states based on first order kinetics, where the rate constants are exponential functions of voltage. The open probability curve increases rather steeply, made even steeper since having n channels raises the probability curve to the nth power. This curve was found to rise rapidly between 4V and 8V to a maximum around 8V. At 10V, however, shorter stimulus durations are insufficient to trigger calcineurin, but longer durations kill the cells. 
</P><P>From our experiments we found that there was significant expression of RFP after 40 seconds of shocking. We can also establish that a ten volt voltage amplitude kills most of the cells, which from literature (REFERENCE) is probably due to membrane disruption. The maximum expression we can get without damaging the cells is with an 8V shocking voltage amplitude.
</P><H3><SPAN class="mw-headline" id="CDRE_hone_in_Experiment">CDRE hone in Experiment</SPAN></H3><P><I>Reason for doing the experiment:</I>To hone in on the CDRE systems region of activation with respect to time of shocking. Also to compare yeast without PMC1 and VCX1 knocked out versus yeast with it knocked out to determine effect of the knockout on the CDRE systems expression levels.<I>Date:</I>10/7/10<I>Location:</I>Imaging Center<I>Name of S. cerevisiae strain:</I>Delta VCX1 Delta PMC1 W 303 with CDRE-RFP
</P><H4><SPAN class="mw-headline" id="Observations">Observations</SPAN></H4><P>From the imaging of the shocked yeast cells lacking vesicular calcium pumps we find that the cells start expressing RFP after 90 seconds of shocking. We saw expression in all the samples between 90 and 130 seconds with a definite trend of increase. The overall expression of RFP was pretty low, however, and only a few of the total cells expressed. It was also difficult to establish cytosol localization of RFP due to clumping of cells. We saw no expression of RFP in the 0 second shock trial.
</P><P>From imaging the yeast samples with vesicular calcium pumps we found very high expression levels overall. There was constitutive expression in the negative control i.e. the 0 second shock trial, and the expression of RFP increased rapidly with increased shocking time, with maximum saturation at 50 seconds. After this it was not possible to visually distinguish weather there was any increase in expression levels. There was definite localization of RFP in the cytoplasm. In the samples with vesicular calcium pumps, most of the cells expressed RFP. Another observation made during imaging was that the cells with vesicular pumps were a lot healthier looking and bigger than the ones without.
</P><H4><SPAN class="mw-headline" id="Results_2">Results</SPAN></H4><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:361px;"><DIV class="thumbcaption">This graph shows the increasing linear relationship that electrostimulation time has with reporter gene expression.  In this experiment, the promoter incorporated the CDRE from FKS2 and had vesicular calcium pumps present.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:420px;"><DIV class="thumbcaption">This graph shows the increasing linear relationship that electrostimulation time has with reporter gene expression.  In this experiment, the promoter incorporated the CDRE from PMC1 and was null for vesicular calcium pumps.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">0 seconds of electrostimulus on yeast with vesicular pumps</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">60 seconds of electrostimulus on yeast with vesicular pumps</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">90 seconds of electrostimulus on yeast with vesicular pumps</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">110 seconds of electrostimulus on yeast with vesicular pumps</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">60 seconds of electrostimulus on yeast without vesicular pumps</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">80 seconds of electrostimulus on yeast without vesicular pumps</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">90 seconds of electrostimulus on yeast without vesicular pumps</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">110 seconds of electrostimulus on yeast without vesicular pumps</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">130 seconds of electrostimulus on yeast without vesicular pumps</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Conclusions">Conclusions</SPAN></H4><P>From this experiment we can conclude that the CDRE–RFP plasmid was successfully inserted into the PMC1, VCX1 yeast strain lacking calcium vacuoles. We can also establish that the activation region of the CDRE–RFP system is between 90 and 130 seconds of shocking, with a definite increase in expression levels with an increase in time shocked. We can also establish that there is no constitutive expression of CDRE–RFP from the fact that there is no RFP seen in the 0 second control. The overall expression of the system is weak however and it is only after 110 seconds that more than half of the cells in the frame start to express. We also observed that the cells lacking calcium pumps are smaller and less healthy looking than the cells with the pumps, which is probably because they are less able to deal with calcium shock induced by shocking and so more strained and unhealthy.
</P><P>From the images of the vesicular pump lacking yeast we can see that there is significantly higher expression levels with constitutive expression. We also can see that the rate of increase in expression with time is much higher and the region where the CDRE gets activated is much earlier than with the yeast without the pumps. We theorize that the reason for this increased expression in yeast containing vesicular pumps could be because of a combination of two reasons. First, because they do not have PMC1 and VCX1 knocked out they have the ability to pump calcium out of the cytosol and into vacuoles making them more resistant to calcium shock and hence more healthy and better able to express the RFP. The other possible reason is that PMC1 and/or VCX1 may have feedback loops, and as PMC1 does have constitutive expression the vesicular pump containing cells may be having their RFP expression rates amplified by this effect. We believe that a combination of these two effects causes the increased expression rates.
</P><H3><SPAN class="mw-headline" id="Characterization_Of_CDRE-RFP_system_in_wild_type_yeast_and_PMC1-YFP_system_in_Vesicular_Pump_Negative_Yeast_Strain">Characterization Of CDRE-RFP system in wild type yeast and PMC1-YFP system in Vesicular Pump Negative Yeast Strain</SPAN></H3><P><I>Reason for doing the experiment:</I>To characterize the relationship between electrostimulation times and expression of florescent protein of the synthetic CDRE promoter sequence and the PMC1 promoter sequence. Also to demonstrate the effect of knocking out the PMC1 and VCX1 genes in yeast.<I>Date:</I>10/7/10<I>Location:</I>Ostermeier Complex<I>Name of</I> S. cerevisiae <I>strain:</I>W303 with CDRE-RFP, Delta VCX1 Delta PMC1 W303 with PMC-YFP
</P><H4><SPAN class="mw-headline" id="Results_3">Results</SPAN></H4><H2><SPAN class="mw-headline" id="Protocol">Protocol</SPAN></H2><H3><SPAN class="mw-headline" id="Plasmid_Extraction">Plasmid Extraction</SPAN></H3><H4><SPAN class="mw-headline" id="Reagents">Reagents</SPAN></H4><OL><LI><B>Buffer P1 - Resuspension Buffer</B> 50mM Tris-Cl, pH 8.0, 10mM EDTA, 100ug/mL RNase A Storage condition - 4oC after adding RNase A Prep - Dissolve 6.06g Tris base, 3.72g EDTA-2H20 in 800mL dH20. Adjust the pH to 8.0 with HCl.Adjust the volume to 1 liter with dH2O. Add 100mg RNase A per liter of P1.
</LI><LI><B>Buffer P2 - Lysis Buffer</B>200mM NaOH, 1% SDSStorage condition - RTDissolve 8.09g of NaOH pellets in 950mL dH2O, 50mL 20% SDS solution. The final volume should be 1 liter.
</LI><LI><B>Buffer N3 - Neutralization Buffer</B> for spin columns.Composition unknownStorage condition - RT
</LI><LI><B>PB Buffer - Binding Buffer</B>Composition Unknown (Proprietary)Storage condition - RT
</LI><LI><B>Buffer PE - Wash Buffer</B>Composition unknownStorage condition - RT
</LI></OL><H4><SPAN class="mw-headline" id="Protocol_2">Protocol</SPAN></H4><OL><LI> Innoculate 3-4 mL of the E. coli cells in glass tubes with the right selection resistance in LB (Luria Bertani) medium.
</LI><LI> Transfer solution to a 2 mL Eppendorf tube and spin down at max speed (~13,000         rpm) for 30 seconds.
</LI><LI> Pour out LB medium and repeat until all of the solution from the innoculation tube is pelleted at the bottom of the Eppendorf tube. Each spin down after the first requires one full minute.
</LI><LI> Re-suspend the pelleted E. coli in 250 µL Buffer P1 and transfer solution to micro-centrifuge tube provided by the Qiaprep Kit. (This can be the same tube as the 2 mL Eppendorf tube) This step requires a strong vortex in order to fully re-suspend the pelleted cells. May require 3-4 minutes of continuous agitation.
</LI><LI> Add 250 µL of Buffer P2 and mix gently by inverting the tube 4-6 times. Be very gentle! Shear stresses can ruin the experiment after the lysis of cells from P2 Buffer.
</LI><LI> Add 350 µL of Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times.
</LI><LI> Spin down at max speed for 10 minutes. A white pellet near the bottom and side of the walls will form.
</LI><LI> Extract the supernatant and add 500 µL of PB buffer. Place mixture into the Qiaprep spin column.
</LI><LI> Centrifuge mixture for 1 minute, max speed.
</LI><LI> Remove liquid at bottom.
</LI><LI> Add 750 µL of PE Buffer to Qiaprep spin column.
</LI><LI> Centrifuge mixture for 1 minute, max speed.
</LI><LI> Remove liquid at bottom and centrifuge again for 1 minute at max speed to remove more of the PE buffer.
</LI><LI> Transfer Qiaprep spin column to sterile Eppendorf tube and add 50 µL of dH2O and let the spin column equilibrate for 10 minutes with the dH2O.
</LI><LI> Spin down column for 1 minute, max speed.
</LI><LI> Confirm size with gel electrophoresis with 1% agarose gel and appropriate DNA Ladder. (Remember circular plasmids can become supercoiled and produce smaller apparent sizes on the gel)
</LI><LI> Use Nanodrop™ to confirm DNA concentration.
</LI></OL><H3><SPAN class="mw-headline" id="Yeast_Transformation">Yeast Transformation</SPAN></H3><P>To transform a desired plasmid into a host yeast cell.Date: 8/25/10 Used to transform ref-GFP
</P><H4><SPAN class="mw-headline" id="Reagents_2">Reagents</SPAN></H4><OL><LI><B>PEG-TE LiOAc</B>10 mL 10x TE 10 mL 1 M LiOAc 80 mL 50% PEG (3350 mw)Mix with stir bar, filter sterilize, store at 4 °C
</LI><LI><B>TE-LiOAc</B>10 mL 10x TE10 mL 1 M LiOAc80 mL H<SUB>2</SUB>O</LI><LI><B>10x TE</B>70 mL Tris Base (1M)30 mL 1M Tris-Cl20 mL 0.5 M EDTA: pH 8.0 880 mL diH<SUB>2</SUB>OAutoclave: 45 min
</LI><LI><B>SsDNA</B>-(Stratagene #201190-81)Sonicated Salmon Sperm DNAssDNA: 10mg/mL; 1 mL vialStore at -20°C
</LI><LI><B>1 M Lithium Acetate</B>-(LiOAc: Sigm L-6883)102.0 g LiOAcBring to 1L diH<SUB>2</SUB>OAutoclave 45 min.
</LI><LI><B>50% PEG-3350</B>-(Sigma P-4338)250g PEG to 250 mL warm diH<SUB>2</SUB>OBring to 500 mL diH<SUB>2</SUB>OFilter Sterilize
</LI></OL><H4><SPAN class="mw-headline" id="Protocol_3">Protocol</SPAN></H4><OL><LI> Grow yeast cells to log phase in 5 mL YPD medium overnight @ 30 degrees.
</LI><LI> Harvest (5 min, 2000 rpm);aspirate supernatant (sterile tip).
</LI><LI> Wash cell pellet 1X with 1mL TE-LiOAc; transfer to eppendorph tubes.
</LI><LI> Resuspend in 100 µL of TE-LiOAc.
</LI><LI> Add (mix after each):
<DL><DD>2.5µL Fresh Boiled ssDNA (10 mg/mL)
</DD><DD>1µL Mini-Prep Plasmid DNA
</DD><DD>800µL PEG - TE -LiOAc
</DD></DL></LI><LI> Incubate @ 30 C; 30 min. Heat Shock 42 C; ~20 minutes.
</LI><LI> Harvest cells (30 sec,~14000 rpm), aspirate supernatant.
</LI><LI> Wash pellet 1x with 1 mL of YPD.
</LI><LI> Resuspend in 100-250  µL of YPD. Plate onto YPD. Incubate plates 2 days at 30 C.(Optional: plate onto YPD for 1 day and then replica-plate onto selective medium)
</LI></OL><H3><SPAN class="mw-headline" id="CDRE_Optimization">CDRE Optimization</SPAN></H3><H4><SPAN class="mw-headline" id="Protocol_4">Protocol</SPAN></H4><P>A 96 well plate was set up with 2 rows of PMC1 and VCX1 vacuole pump lacking yeast cells, with the CDRE–RPF plasmid in them. Two rows below that were filled with yeast cells with their pumps intact and the CDRE–RFP plasmid inserted in it.
We also set aside 8 wells at the bottom as cleaning wells and filled them with ethanol.
</P><P>The shocking of the cells was carried out using a coaxial electrode system, consisting of 8 sets of gold plated electrodes that fit into the 8 wells of a 96 well plate. An oscilloscope was connected in parallel to the electrode. The shocking was done at 8 V and at 20HZ frequency. The pulses were all sinusoidal.
</P><P>The cells were shocked for increasing time intervals from 30 seconds to 130 seconds in ten second increments. There was also a 0 second control for every row of cells. Between each shocking the electrodes were first repeatedly dipped in the ethanol wells to clean them and then dipped in a water bath to remove any residual ethanol, then dried using Kim wipes.
</P><P>After shocking the 96 well plate was put in a shaker at 37C for 8 hours.  They were then imaged using a Meta 510 confocal microscope. A 60X oil objective was used and Rodamine and FIT-C filters were applied.
</P><P>After re suspending any settled cells in the well being tested, 20ul of the cell solution was pipetted onto a slide and a cover slip was placed over it. Some oil was spread on the cover slip and the cells were imaged. At least 2 pictures were taken from each sample.
</P><H3><SPAN class="mw-headline" id="CDRE_Hone_in">CDRE Hone in</SPAN></H3><H4><SPAN class="mw-headline" id="Procedure">Procedure</SPAN></H4><P>A 96 well plate was set up with 2 rows of PMC1 and VCX1 vacuole pump lacking yeast cells, with the CDRE – RPF plasmid in them. Two rows below that were filled with yeast cells with their pumps intact and the CDRE – RFP plasmid inserted in it. We also set aside 8 wells at the bottom as cleaning wells and filled them with ethanol.
</P><P>The shocking of the cells was carried out using a coaxial electrode system, consisting of 8 sets of gold plated electrodes that fit into the 8 wells of a 96 well plate. An oscilloscope was connected in parallel to the electrode. The shocking was done at 8 V and at 20HZ frequency. The pulses were all sinusoidal.
</P><P>The cells were shocked for increasing time intervals from 30 seconds to 130 seconds in ten second increments. There was also a 0 second control for every row of cells. Between each shocking the electrodes were first repeatedly dipped in the ethanol wells to clean them and then dipped in a water bath to remove any residual ethanol, then dried using Kim wipes.
After shocking the 96 well plate was put in a shaker at 37C for 8 hours. They were then imaged using a Meta 510 confocal microscope. A 60X oil objective was used and Rodamine and FIT-C filters were applied.
</P><P>After re suspending any settled cells in the well being tested, 20ul of the cell solution was pipetted onto a slide and a cover slip was placed over it. Some oil was spread on the cover slip and the cells were imaged. At least 2 pictures were taken from each sample.
</P><H3><SPAN class="mw-headline" id="Characterization_Of_CDRE-RFP_System_in_Wild_Type_Yeast_and_PMC1-YFP_system_in_Yeast_Strain_Lacking_Vesicular_Calcium_Pumps">Characterization Of CDRE-RFP System in Wild Type Yeast and PMC1-YFP system in Yeast Strain Lacking Vesicular Calcium Pumps</SPAN></H3><H4><SPAN class="mw-headline" id="Procedure_2">Procedure</SPAN></H4><P>A 96 well plate was set up with 1 rows of W303 yeast cells, with the CDRE – RPF plasmid in them. One row below that were filled with yeast cells with PMC1 and VCX1 without their calcium vacuole pumps the PMC1 - YFP plasmid inserted in it.
</P><P>The shocking of the cells was carried out using a coaxial electrode system, consisting of 8 sets of gold plated electrodes that fit into the 8 wells of a 96 well plate. An oscilloscope was connected in parallel to the electrode. The shocking was done at 8 V and at 20HZ frequency. The pulses were all sinusoidal.
</P><P>The cells were shocked for increasing time intervals from 40 seconds to 150 seconds in ten second increments. There was also a 0 second control for every row of cells. Between each shocking the electrodes were first repeatedly dipped in the ethanol wells to clean them and then dipped in a water bath to remove any residual ethanol, then dried using Kim wipes.
After shocking the 96 well plate was put in a shaker at 37C for 8 hours.
</P><P>These cells were then put in a plate reader that measured OD, as a measure of cell growth in the samples over the expression period of 8 hours. After this the plate was put in a plate florometer and RFP and YFP intensities were measured. The following data was collected.
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="background: #ffffff;"><TBODY><TR><TH> Time/FIU/OD </TH><TH>CDRE </TH><TH>PMC1
</TH></TR><TR><TD> 0 </TD><TD> 17.85119048 </TD><TD> 263121.1632
</TD></TR><TR><TD>40 </TD><TD> 26.10337553 </TD><TD> 338282.2967
</TD></TR><TR><TD>50 </TD><TD> 19.86680328 </TD><TD> 293471.7742
</TD></TR><TR><TD>60 </TD><TD> 29.04337349 </TD><TD> 456021.0843
</TD></TR><TR><TD>70 </TD><TD> 28.32403433 </TD><TD> 484804.1237
</TD></TR><TR><TD>80 </TD><TD> 33.50247525 </TD><TD> 505993.6102
</TD></TR><TR><TD>90 </TD><TD> 34.32098765 </TD><TD> 544573.5294
</TD></TR><TR><TD>100 </TD><TD> 29.03455724</TD><TD> 311173.5358
</TD></TR><TR><TD>110</TD><TD>34.65552699</TD><TD>262445.7237
</TD></TR><TR><TD>120</TD><TD>32.08673469</TD><TD>506424.9201
</TD></TR><TR><TD>130</TD><TD>41.87246377</TD><TD>485111.8881
</TD></TR><TR><TD>140</TD><TD>37.71948052</TD><TD>360177.9279
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Characterization_Of_CDRE-RFP_in_wild_type_yeast_and_Yeast_Strain_Vesicular_Calcium_Pump_Negative_and_the_PMC1-YFP_system_in_wild_type_yeast_and_Vesicular_Negative_Yeast">Characterization Of CDRE-RFP in wild type yeast and Yeast Strain Vesicular Calcium Pump Negative and the PMC1-YFP system in wild type yeast and Vesicular Negative Yeast</SPAN></H3><H4><SPAN class="mw-headline" id="Procedure_3">Procedure</SPAN></H4><P>Two 96 well plates were set up with  one having 4 rows of W303 yeast cells, with the CDRE – RPF plasmid in them. 4 rows below them were filled with yeast cells with PMC1 and VCX1 knocked out and the CDRE-RFP plasmid inserted in them.
</P><P>The other has  4 rows of W303 yeast cells, with the CDRE – RPF plasmid in them. 4 rows below them were filled with yeast cells with PMC1 and VCX1 knocked out and the CDRE-RFP plasmid inserted in them. 
</P><P>The shocking of the cells was carried out using a coaxial electrode system, consisting of 8 sets of gold plated electrodes that fit into the 8 wells of a 96 well plate. An oscilloscope was connected in parallel to the electrode. The shocking was done at 8 V and at 20HZ frequency. The pulses were all sinusoidal.
</P><P>The cells were shocked for increasing time intervals from 40 seconds to 150 seconds in ten second increments. There was also a 0 second control for every row of cells. Between each shocking the electrodes were first repeatedly dipped in the ethanol wells to clean them and then dipped in a water bath to remove any residual ethanol, then dried using Kim wipes.
After shocking the 96 well plate were allowed to grow at room temperature.
</P><P>These cells were then put in a plate reader that measured OD, as a measure of cell growth in the samples over the expression period of 8 hours. After this the plate was put in a plate florometer and RFP and YFP intensities were measured. The growth and expression was so measured at one hour intervals from three hours later to 8 hours later. The following data was collected.
</P><H3><SPAN class="mw-headline" id="Plasmid_Extraction_2">Plasmid Extraction</SPAN></H3><H4><SPAN class="mw-headline" id="Reagents_3">Reagents</SPAN></H4><OL><LI><B>Buffer P1 - Resuspension Buffer</B> 50mM Tris-Cl, pH 8.0, 10mM EDTA, 100ug/mL RNase A Storage condition - 4oC after adding RNase A Prep - Dissolve 6.06g Tris base, 3.72g EDTA-2H20 in 800mL dH20. Adjust the pH to 8.0 with HCl.Adjust the volume to 1 liter with dH2O. Add 100mg RNase A per liter of P1.
</LI><LI><B>Buffer P2 - Lysis Buffer</B>200mM NaOH, 1% SDSStorage condition - RTDissolve 8.09g of NaOH pellets in 950mL dH2O, 50mL 20% SDS solution. The final volume should be 1 liter.
</LI><LI><B>Buffer N3 - Neutralization Buffer</B> for spin columns.Composition unknownStorage condition - RT
</LI><LI><B>PB Buffer - Binding Buffer</B>Composition Unknown (Proprietary)Storage condition - RT
</LI><LI><B>Buffer PE - Wash Buffer</B>Composition unknownStorage condition - RT
</LI></OL><H4><SPAN class="mw-headline" id="Protocol_5">Protocol</SPAN></H4><OL><LI> Innoculate 3-4 mL of the E. coli cells in glass tubes with the right selection resistance in LB (Luria Bertani) medium.
</LI><LI> Transfer solution to a 2 mL Eppendorf tube and spin down at max speed (~13,000         rpm) for 30 seconds.
</LI><LI> Pour out LB medium and repeat until all of the solution from the innoculation tube is pelleted at the bottom of the Eppendorf tube. Each spin down after the first requires one full minute.
</LI><LI> Re-suspend the pelleted E. coli in 250 µL Buffer P1 and transfer solution to micro-centrifuge tube provided by the Qiaprep Kit. (This can be the same tube as the 2 mL Eppendorf tube) This step requires a strong vortex in order to fully re-suspend the pelleted cells. May require 3-4 minutes of continuous agitation.
</LI><LI> Add 250 µL of Buffer P2 and mix gently by inverting the tube 4-6 times. Be very gentle! Shear stresses can ruin the experiment after the lysis of cells from P2 Buffer.
</LI><LI> Add 350 µL of Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times.
</LI><LI> Spin down at max speed for 10 minutes. A white pellet near the bottom and side of the walls will form.
</LI><LI> Extract the supernatant and add 500 µL of PB buffer. Place mixture into the Qiaprep spin column.
</LI><LI> Centrifuge mixture for 1 minute, max speed.
</LI><LI>Remove liquid at bottom.
</LI><LI> Add 750 µL of PE Buffer to Qiaprep spin column.
</LI><LI> Centrifuge mixture for 1 minute, max speed.
</LI><LI> Remove liquid at bottom and centrifuge again for 1 minute at max speed to remove more of the PE buffer.
</LI><LI> Transfer Qiaprep spin column to sterile Eppendorf tube and add 50 µL of dH2O and let the spin column equilibrate for 10 minutes with the dH2O.
</LI><LI> Spin down column for 1 minute, max speed.
</LI><LI> Confirm size with gel electrophoresis with 1% agarose gel and appropriate DNA Ladder. (Remember circular plasmids can become supercoiled and produce smaller apparent sizes on the gel)
</LI><LI> Use Nanodrop™ to confirm DNA concentration.
</LI></OL><H3><SPAN class="mw-headline" id="Yeast_Transformation_2">Yeast Transformation</SPAN></H3><P>To transform a desired plasmid into a host yeast cell.Date: 8/25/10 Used to transform ref-GFP
</P><H4><SPAN class="mw-headline" id="Reagents_4">Reagents</SPAN></H4><OL><LI><B>PEG-TE LiOAc</B>10 mL 10x TE 10 mL 1 M LiOAc 80 mL 50% PEG (3350 mw)Mix with stir bar, filter sterilize, store at 4 °C
</LI><LI><B>TE-LiOAc</B>10 mL 10x TE10 mL 1 M LiOAc80 mL H<SUB>2</SUB>O</LI><LI><B>10x TE</B>70 mL Tris Base (1M)30 mL 1M Tris-Cl20 mL 0.5 M EDTA: pH 8.0 880 mL diH<SUB>2</SUB>OAutoclave: 45 min
</LI><LI><B>SsDNA</B>-(Stratagene #201190-81)Sonicated Salmon Sperm DNAssDNA: 10mg/mL; 1 mL vialStore at -20°C
</LI><LI><B>1 M Lithium Acetate</B>-(LiOAc: Sigm L-6883)102.0 g LiOAcBring to 1L diH<SUB>2</SUB>OAutoclave 45 min.
</LI><LI><B>50% PEG-3350</B>-(Sigma P-4338)250g PEG to 250 mL warm diH<SUB>2</SUB>OBring to 500 mL diH<SUB>2</SUB>OFilter Sterilize
</LI></OL><H4><SPAN class="mw-headline" id="Protocol_6">Protocol</SPAN></H4><OL><LI> Grow yeast cells to log phase in 5 mL YPD medium overnight @ 30 degrees.
</LI><LI> Harvest (5 min, 2000 rpm);aspirate supernatant (sterile tip).
</LI><LI> Wash cell pellet 1X with 1mL TE-LiOAc; transfer to eppendorph tubes.
</LI><LI> Resuspend in 100 µL of TE-LiOAc.
</LI><LI> Add (mix after each):
<DL><DD>2.5µL Fresh Boiled ssDNA (10 mg/mL)
</DD><DD>1µL Mini-Prep Plasmid DNA
</DD><DD>800µL PEG - TE -LiOAc
</DD></DL></LI><LI> Incubate @ 30 C; 30 min. Heat Shock 42 C; ~20 minutes.
</LI><LI> Harvest cells (30 sec,~14000 rpm), aspirate supernatant.
</LI><LI> Wash pellet 1x with 1 mL of YPD.
</LI><LI> Resuspend in 100-250  µL of YPD. Plate onto YPD. Incubate plates 2 days at 30 C.(Optional: plate onto YPD for 1 day and then replica-plate onto selective medium)
</LI></OL><H3><SPAN class="mw-headline" id="Plasmid_Mini-Prep">Plasmid Mini-Prep</SPAN></H3><H4><SPAN class="mw-headline" id="Reagents_5">Reagents</SPAN></H4><OL><LI><B>Solutions 1 (Glucose/Tris/EDTA Solution):</B>2.3 mL 40% Glucose (50 mM Glucose)2.5 mL Tris-Cl (25 mM Tris-Cl pH 8.0)2 mL EDTA .5M (10 mM EDTA)Bring to 100 mL with diH2Okeep refrigerated</LI><LI><B>Solution II (NaOH/ SDS solution):</B>Mix (in order):Make 10 mL fresh each time:7.5 mL diH202 mL of 1N NaOH0.5 mL of 20% SDSAdjust the volume to 10 mL with diH2O after both SDS and NaOH have been added</LI><LI><B>Solution III (KOAc solution):</B>to make 100 mL; add:60 mL 5 M KOAc11.5 mL glacial acetic acid28.5 mL diH2Okeep refrigeratedpH should be around 6.0</LI></OL><H4><SPAN class="mw-headline" id="Protocol_7">Protocol</SPAN></H4><OL><LI> Innoculate 5 mL of sterile growth medium containing appropriate antibiotic with a single bacterial colony. Incubate at 37 C in the incubator shaker. Allow to grow until a stationary phase is reached (overnight is ideal).
</LI><LI> Transter 2 x 750 µL aliquots from each overnight culture into a labeled eppendorf tube. Pellet the cells by spinning for 3 to 5 minutes in the microfuge at maximum speed.
</LI><LI> Aspirate as much of supernatant as possible.
</LI><LI> Resuspend the cell pellet in 100 µL of Solution I. Use pipetman to suck the solution up and down until the pellet is completely resuspended in the lysis buffer.
</LI><LI> Let samples sit at room temp for 5 minutes. (Unless using TE, then skip this step)
</LI><LI> Add 200 µL of Solution II (make fresh each time). Mix well, vortex if necessary. Let stand on ice for 5 minutes. The samples should get gooey and might have stringy webs when opened.
</LI><LI> Add 150 µL of cold Solution III and vortex at highest speed for 2 sec to mix, let stand on ice for 5 minutes. (Here the chromosomal DNA and the cellular proteins appear as a precipitate. The plasmid DNA renatures and remains in solution.) The tubes can stand on ice for several hours if necessary.
</LI><LI> Spin for 5 to 10 minutes in the cold room microfuge to pellet the cell debris and chromosomal DNA.
</LI><LI> While the tubes are spinning, set up fresh labeled tubes each with 2 volumes of cold 100% EtOH. If you had 400 µL of supernatant from the above step, use 800 µL of ethanol.
</LI><LI> Carefully remove the top aqueous layer of each tube from step 8 with a 1 mL pipetman (usually about 400 µL) and dispense into the tubes containing the cold 100% ethanol.
</LI><LI> Let stand on ice for 5 minutes (or longer) to precipitate the nucleic acids. Alternatively, place in the freezer for a longer period. The DNA is stable in ethanol and can be held at this step for as long as necessary.
</LI><LI> Spin for 5 to 10 minutes (or longer) in the microfuge to pellet the plasmid DNA and RNA.
</LI><LI> Remove and discard the supernatant, being careful to not disturb the pellet and dry on a kimwipe if needed.
</LI><LI> Add 1 mL of 70% EtOH to rinse DNA. Spin for 5 minutes
</LI><LI> Remove and discard the supernatant.
</LI><LI> Dry the pellets by placing the samples in the Speed-Vac and dry under vacuum for 5 to 10 minutes (make sure pellet is dry).
</LI><LI> Dissolve pellet in 30 µL of H2O (or TE). Gently agitate the tube by tapping on the side to insure that the pellet has been resuspended. The tubes can be incubated at 37 C to aid in resuspension.
</LI><LI> Refrigerate until ready to use.
</LI></OL><H3><SPAN class="mw-headline" id="SC_Media">SC Media</SPAN></H3><H4><SPAN class="mw-headline" id="Reagents_6">Reagents</SPAN></H4><TABLE border="1" cellpadding="5" cellspacing="0" style="background: #ffffff;"><CAPTION>Start with 600 mL of autoclaved H2O and add the following:
</CAPTION><TBODY><TR><TH>gms</TH><TH>mg/L</TH><TH>gms</TH><TH>mg/L
</TH></TR><TR><TD>Adenine sulfate </TD><TD>12</TD><TD>20</TD><TD>L-Isoleucine</TD><TD>18</TD><TD>30
</TD></TR><TR><TD>Uracil</TD><TD>12</TD><TD>20</TD><TD>L-Lysine</TD><TD>18</TD><TD>30
</TD></TR><TR><TD>L-Tryptophan</TD><TD>12</TD><TD>20</TD><TD>L-Phenylalanine</TD><TD>30</TD><TD>50
</TD></TR><TR><TD>L-Leucine</TD><TD>12</TD><TD>20</TD><TD>L-Glutamic acid</TD><TD>60</TD><TD>100
</TD></TR><TR><TD>L-Histadine</TD><TD>12</TD><TD>20</TD><TD>L-Aspartic acid</TD><TD>60</TD><TD>100
</TD></TR><TR><TD>L-Arginine</TD><TD>12</TD><TD>20</TD><TD>L-Valine</TD><TD>90</TD><TD>150
</TD></TR><TR><TD>L-Methionine</TD><TD>12</TD><TD>20</TD><TD>L-Threonine</TD><TD>120</TD><TD>200
</TD></TR><TR><TD>L-Tyrosine</TD><TD>18</TD><TD>30</TD><TD>L-Serine</TD><TD>240</TD><TD>400
</TD></TR><TR><TD>L-Leucine</TD><TD>36</TD><TD>60</TD><TD>Glucose</TD><TD>12</TD><TD>20
</TD></TR><TR><TD>L-Isoleucine</TD><TD>18</TD><TD>30</TD></TR></TBODY></TABLE><P>Selective media is made by omitting the desired amino acid(s).
</P><H3><SPAN class="mw-headline" id="Digestion_and_ligation_of_plasmids">Digestion and ligation of plasmids</SPAN></H3><H4><SPAN class="mw-headline" id="Plates">Plates</SPAN></H4><TABLE border="1" cellpadding="5" cellspacing="0" style="background: #ffffff;"><TBODY><TR><TH>YPD </TH><TH>ml </TH><TH>SC- </TH><TH>ml
</TH></TR><TR><TD> 40% Dextrose</TD><TD> 30 </TD><TD>40% Dextrose </TD><TD>30
</TD></TR><TR><TD> 2X YEP </TD><TD> 300 </TD><TD>Selective Media </TD><TD>300
</TD></TR><TR><TD> 4% Agar </TD><TD> 300 </TD><TD>4% Agar </TD><TD>300
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Protocol_8">Protocol</SPAN></H4><TABLE border="1" cellpadding="5" cellspacing="0" style="background: #ffffff;"><CAPTION>Prepare the following solutions:
</CAPTION><TBODY><TR><TH>Plasmid\Reagents (µL)</TH><TH>DNA</TH><TH>Xba1</TH><TH>Pst1</TH><TH>Buffer</TH><TH>BSA</TH><TH>Water
</TH></TR><TR><TD>pSB1C3 (50ng/µL)</TD><TD>33</TD><TD>4</TD><TD>0.07</TD><TD>10</TD><TD>1</TD><TD>51.88
</TD></TR><TR><TD>GTGGCTG (283ng/µL)</TD><TD>0.71</TD><TD>30.84</TD><TD>0.55</TD><TD>40</TD><TD>4</TD><TD>323.91
</TD></TR><TR><TD>GAGGCTG (343ng/µL)</TD><TD>0.58</TD><TD>3.0.84</TD><TD>0.55</TD><TD>35</TD><TD>3.5</TD><TD>279.53
</TD></TR><TR><TD>GAGGCTC (324ng/µL)</TD><TD>0.62</TD><TD>30.84</TD><TD>0.55</TD><TD>35</TD><TD>3.5</TD><TD>279.5
</TD></TR><TR><TD>GAGGCGG (235ng/µL)</TD><TD>0.93</TD><TD>33.92</TD><TD>0.61</TD><TD>35</TD><TD>3.5</TD><TD>276.04
</TD></TR><TR><TD>GAGGCTT (356ng/µL)</TD><TD>0.62</TD><TD>33.92</TD><TD>0.61</TD><TD>35</TD><TD>3.5</TD><TD>276.36
</TD></TR><TR><TD>CAGGCTG (345ng/µL)</TD><TD>0.64</TD><TD>33.92</TD><TD>0.61</TD><TD>35</TD><TD>3.5</TD><TD>276.34
</TD></TR><TR><TD>GGGGCTG (293ng/µL)</TD><TD>0.75</TD><TD>33.92</TD><TD>0.61</TD><TD>35</TD><TD>3.5</TD><TD>276.22
</TD></TR></TBODY></TABLE><P>These solutions are incubated at 37ºC for one hour.
Gel purification used 0.8% agarose TAE gel, visualized with ethidium bromide and UV light. Qiagen’s gel purification kit protocol was followed, with the exception of the ethanol-wash step, which was repeated twice.
Zymogen’s Clean and Concentrate protocal was followed to further purify the DNA.
</P><TABLE><TBODY><TR><TH>CDRE 1 </TH><TH>ng </TH><TH>µl
</TH></TR><TR><TD> pSB1C3 (4.5ng/µL)   </TD><TD> 50 </TD><TD>11.11
</TD></TR><TR><TD> GAGGCTC (2.8 ng/µL) </TD><TD> 4.95 </TD><TD>1.77
</TD></TR><TR><TD> sterile dH2O </TD><TD> ### </TD><TD>4.12
</TD></TR><TR><TD> 10x buffer </TD><TD> ### </TD><TD>2
</TD></TR><TR><TD> Ligase </TD><TD> ### </TD><TD>1
</TD></TR></TBODY></TABLE><TABLE border="1" cellpadding="5" cellspacing="0" style="background: #ffffff;"><TBODY><TR><TH>CDRE  2 </TH><TH>ng </TH><TH>µl
</TH></TR><TR><TD> pSB1C3 (4.5ng/µL)</TD><TD> 50 </TD><TD>11.11
</TD></TR><TR><TD> GGGGCTG(11.1ng/µL)</TD><TD> 4.95 </TD><TD>0.46
</TD></TR><TR><TD> sterile dH2O </TD><TD> ### </TD><TD>5.44
</TD></TR><TR><TD> 10x buffer </TD><TD> ### </TD><TD>2
</TD></TR><TR><TD> Ligase </TD><TD> ### </TD><TD>1
</TD></TR></TBODY></TABLE><TABLE border="1" cellpadding="5" cellspacing="0" style="background: #ffffff;"><TBODY><TR><TH>CDRE  3 </TH><TH>ng </TH><TH>µl
</TH></TR><TR><TD> pSB1C3 (4.5ng/µL) </TD><TD> 50 </TD><TD>11.11
</TD></TR><TR><TD> GAGGCTT (4.3ng/µL)</TD><TD> 4.95 </TD><TD>1.15
</TD></TR><TR><TD> sterile dH2O </TD><TD> ### </TD><TD>4.74
</TD></TR><TR><TD> 10x buffer </TD><TD> ### </TD><TD>2
</TD></TR><TR><TD> Ligase </TD><TD> ### </TD><TD>1
</TD></TR></TBODY></TABLE><TABLE border="1" cellpadding="5" cellspacing="0" style="background: #ffffff;"><TBODY><TR><TH>CDRE  4 </TH><TH>ng </TH><TH>µl
</TH></TR><TR><TD> pSB1C3 (4.5ng/µL) </TD><TD> 50 </TD><TD>11.11
</TD></TR><TR><TD> GTGGCTG (4.1ng/µL)</TD><TD> 4.95 </TD><TD>1.21
</TD></TR><TR><TD> sterile dH2O </TD><TD> ### </TD><TD>4.68
</TD></TR><TR><TD> 10x buffer </TD><TD> ### </TD><TD>2
</TD></TR><TR><TD> Ligase </TD><TD> ### </TD><TD>1
</TD></TR></TBODY></TABLE><TABLE border="1" cellpadding="5" cellspacing="0" style="background: #ffffff;"><TBODY><TR><TH>CDRE  5 </TH><TH>ng </TH><TH>µl
</TH></TR><TR><TD> pSB1C3 (4.5ng/µL) </TD><TD> 50 </TD><TD>11.11
</TD></TR><TR><TD> GAGGCTG (1.8ng/µL)</TD><TD> 4.95 </TD><TD>2.75
</TD></TR><TR><TD> sterile dH2O </TD><TD> ### </TD><TD>3.14
</TD></TR><TR><TD> 10x buffer </TD><TD> ### </TD><TD>2
</TD></TR><TR><TD> Ligase </TD><TD> ### </TD><TD>1
</TD></TR></TBODY></TABLE><TABLE border="1" cellpadding="5" cellspacing="0" style="background: #ffffff;"><TBODY><TR><TH>CDRE  6 </TH><TH>ng </TH><TH>µl
</TH></TR><TR><TD> pSB1C3 (4.5ng/µL) </TD><TD> 50 </TD><TD>11.11
</TD></TR><TR><TD> CAGGCTG (2.5ng/µL)</TD><TD> 4.95 </TD><TD>1.98
</TD></TR><TR><TD> sterile dH2O </TD><TD> ### </TD><TD>3.91
</TD></TR><TR><TD> 10x buffer </TD><TD> ### </TD><TD>2
</TD></TR><TR><TD> Ligase </TD><TD> ### </TD><TD>1
</TD></TR></TBODY></TABLE><TABLE border="1" cellpadding="5" cellspacing="0" style="background: #ffffff;"><TBODY><TR><TH>CDRE  7 </TH><TH>ng </TH><TH>µl
</TH></TR><TR><TD> pSB1C3 (4.5ng/µL) </TD><TD> 50 </TD><TD>11.11
</TD></TR><TR><TD> GAGGCGG (0.8ng/µL)</TD><TD> 4.95 </TD><TD>6.19
</TD></TR><TR><TD> sterile dH2O </TD><TD> ### </TD><TD>4.2
</TD></TR><TR><TD> 10x buffer </TD><TD> ### </TD><TD>2
</TD></TR><TR><TD> Ligase </TD><TD> ### </TD><TD>1
</TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Johns_Hopkins/Notebook">http://2010.igem.org/Team:Johns_Hopkins/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Johns_Hopkins/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Johns_Hopkins/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook&amp;oldid=208388" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>