"
Page
Discussion
View source
History
teams
Log in
 
Team:ESBS-Strasbourg/Results/Biobricks
From 2010.igem.org
{|
ESBS - Strasbourg
  HOME
  TEAM
Overview
Students
Advisors
Instructors
Strasbourg
Collaboration
  PROJECT
Overview
Strategy
Visual Description
Application
Acknowledgment
Reference
  RESULTS
Biobricks
Biobrick Assembly Technique
Characterization
Modeling
  NOTEBOOK
Synthetic Photoreceptors
Microfluidics
Lighting device
Lab-book
HUMAN PRACTICE
Organisation
Survey
The ClpX video
The ClpX game
Project Safety
     SPONSORS
Biobricks
PIF3
PIF6
PHY B 900
PHY B 650
∆N-ClpX
linker
LAA Tag
DAS Tag
Lambda Tag
GFP (super fold)
PhyB642-(linker-∆NClpX)3
PhyB908-(linker-∆NClpX)3
Full-length ClpX
∆NClpX-linker-∆NClpX-linker-∆NClpX
(linker-∆NClpX)3
Our Biobrick Assembly Technique
  
Let me guide you
Parts Submitted to Registry
  
   
Part    Number:
Part    Name:
Plasmid/Resistance
Status
BBa_K365000
Phytochrome Interacting Factor-3    (PIF3)
pSB1C3 / Chloramphenicol
Sequenced
BBa_K365001
Phytochrome Interacting Factor-6    (PIF6)
pSB1C3 / Chloramphenicol
Sequenced
BBa_K365002
Phytochrome B (aa 1-908)
pSB1C3 / Chloramphenicol
Sequenced
BBa_K365003
Phytochrome B (aa 1-642)
pSB1C3 / Chloramphenicol
Sequenced
BBa_K365004
∆N-ClpX (aa 61-425)
pSB1C3 / Chloramphenicol
Sequenced
BBa_K365005
Linker (aa 20)
pSB1C3 / Chloramphenicol
Sequenced
BBa_K365006
LAA tag
pSB1C3 / Chloramphenicol
Sequenced
BBa_K365007
DAS tag
pSB1C3 / Chloramphenicol
Sequenced
BBa_K365008
Lambda tag
pSB1C3 / Chloramphenicol
Sequenced
BBa_K365009
GFP (super fold)
pSB1C3 / Chloramphenicol
Sequenced
BBa_K365010
PhyB642-(linker-∆NClpX)3
pSB1C3 / Chloramphenicol
Assembled
BBa_K365011
PhyB908-(linker-∆NClpX)3
pSB1C3 / Chloramphenicol
Assembled
BBa_K365012
Full-length ClpX
pSB1C3 / Chloramphenicol
Sequenced
BBa_K365013
∆NClpX-linker-∆NClpX-linker-∆NClpX
pSB1C3 / Chloramphenicol
Assembled
BBa_K365014
(linker-∆NClpX)3
pSB1C3 / Chloramphenicol
Assembled
Colored restriction sites :
EcoRI NotI XbaI NgoMIV
AgeI SpeI NotI PstI
Phytochrome B (aa 1-908) - BBa_K365002
Background:
Phytochromes characterised by a red/far-red photochromicity. Through red-light (650–670 nm) absorption the phytochrome undergoes a rapid conformational change from its ground state Pr to its active state Pfr. The structural change allows the binding of the PIF. This light-sensitive interaction has been mapped to the 650-residue amino-terminal photosensory core of PhyB [30] . The process is completely reversible through absorption in the near infra-red spectrum (705–740 nm).
The photoreceptor protein PhyB serves for the light-dependent activation of the system, therefore it will be fused to the N-teminal of the ClpX-trimer.
Conception:
In in-vivo applications it has been shown that the PIF-interaction with the PhyB photosensory core (residues 1–650) is irreversible in infrared light. [42] demonstrated by assaying PIF6 (which has the strongest interactions of all previously reported PIF domains) against different variants of PhyB that the tandem C-terminal PAS domains (residues 1-908)of plant phytochromes are necessary to confer rapid photoreversibility under infrared light. The original sequence contains a SpeI restriction within the first 908 residues.
The plasmid containing the PhyB-sequence was provided by the laboratory of Wilfried Weber from the University of Freiburg. To create the BioBrick part the sequence was amplified with primers containing the standard prefix with ATG and the fusion suffix of the Fusion Protein Assembly Standard.
Forward primer (5’->3’): (41bp)
GGATCCgaattcgcggccgcttctagATGGTTTCCGGAGTC
Reverse primer (5’->3’): (52 bp)
CAGCTGctgcagcggccgctactagtattaaccggtGCTCGGGATTTGCAAG
In order to get a sequence without an internal restriction sites of one of the BioBrick standards the SpeI-restriction site was altered without changing the encoded amino acid (ACT=Threonine (AC(T,A,G,C)).
Primers for Pfu-mutagenese:
Forward primer (5’->3’): (28 bp)
GGACAAGACGTTACGAGTCAGAAAATCG
Reverse primer (5’->3’): (28 bp)
CGATTTTCTGACTCGTAACGTCTTGTCC
Phytochrome B (aa 1-642) - BBa_K365003
Background:
Phytochromes characterised by a red/far-red photochromicity. Through red-light (650–670 nm) absorption the phytochrome undergoes a rapid conformational change from its ground state Pr to its active state Pfr. The structural change allows the binding of the PIF. This light-sensitive interaction has been mapped to the 650-residue amino-terminal photosensory core of PhyB [42] . The process is completely reversible through absorption in the near infra-red spectrum (705–740 nm).
The photoreceptor protein PhyB serves for the light-dependent activation of the system, therefore it was fused to the N-teminal of the ClpX-trimer.
Conception:
As mentioned before it has been shown that the PIF-interaction with the PhyB photosensory core (residues 1–650) is irreversible in infrared light in in vivo-application [42] . Nevertheless, the binding strength and kinetic parameters depend on the composition and nature of the individual system, so we decided to include also this shorter variant of PhyB in our tests.
The plasmid containing the PhyB-sequence was provided by the laboratory of Wilfried Weber from the University of Freiburg. To create the BioBrick part the sequence was amplified with primers containing the standard prefix with ATG and the fusion suffix of the Fusion Protein Assembly Standard.
Forward primer (5’->3’): (42 bp)
GGATCCgaattcgcggccgcttctagATGGTTTCCGGAGTC
Reverse primer (5’->3’) : (53 bp)
CAGCTGctgcagcggccgctactagtattaaccggtCCCCGCCATATCCCTAC
Phytochrome Interacting Factor-3 (PIF3) - BBa_K365000
Background:
PIF3 is a downstream transcription factor in a well studied signaling pathway of A. thaliana, upon stimulation with red (650 nm) light, it binds directly to PhyB and translocates to the nucleus as a heterodimer where it modulates the transcription of response genes. PIF3 binds only the red-light-exposed form of phytochrome, Pfr, and shows no measurable binding affinity for the dark- or infrared-exposed Pr state.
In our system target proteins are fused to PIF3 and tagged with the DAS degradation sequence which, through light activation, brings the degradation tag in proximity to ClpX.
Conception:
The light-sensitive interaction with PhyB has been mapped to the first 100-residue N-terminal activated phytochrome binding (APB) domain of PIF3 [42].
We chose this sequence, as it has already been successfully used in different synthetic in vitro applications that benefitted from its light-sensitive interactions with PhyB. The original sequence contains an XbaI restriction site.
There is no PDB structure for the PIF3
The stucture below is modeled by homology using the server I-TASSER
The plasmid containing the PIF3-sequence was provided by the laboratory of Stephan Kircher from the University of Freiburg. For the synthesis of the BioBrick part primers containing the sites of the Fusion Protein BioBrick Assembly Standard were used.
Forward primer (5’->3’): 51 bp
GGATCCgaattcgcggccgcttctagatggccggcATGCCTCTGTTTGAGC
Reverse primer (5’->3’): 51 bp
ctgcagcggccgctactagtattaaccggtATGATGATTCAACCATGGAAC
In order to get a sequence without an internal restriction sites of one of the BioBrick standards the XbaI-restriction site was altered without changing the encoded amino acid(TCT=Serin (TC(T,A,G,C)).
Primers for Pfu-mutagenese:
Forward primer (5’->3’) (24 bp)
GCAAACTCTTCAAGAGCTAGAGAG
Reverse primer (5’->3’) (24 bp)
CTCTCTAGCTCTTGAAGAGTTTGC
Phytochrome Interacting Factor-6 (PIF6) - BBa_K365001
Background:
For the design of the first engineered system that achieved to enable the spatiotemporal control of PhyB-PIF interactions in in-vivo experiments, [42]. screened multiple potential  phytochrome–PIF pairs by a fluorescence translocation assay in NIH3T3 cells. They measured the red-light-induced translocation of yellow fluorescent protein (YFP) fused to PIF domains to co-expressed phytochrome domains fused through a flexible linker to mCherry and localized to the plasma membrane by a carboxyterminalpolybasic, prenylation sequence from Kras.  Of all previously reported PIF domains, only the N terminus of PIF6 is strong enough to cause significant translocation of YFP to the membrane.
Fig.1: Implied system of Lim and Voigt (2009) to screen potential phytochrome-PIF
pairs in a fluorescence translocation assay.
Conception:
We chose used the same sequence of the last 100-residue N-terminal activated phytochrome binding (APB) domain of PIF6, which was already successfully used by [42]. .
There is no PDB structure for the PIF6
The stucture below is modeled by homology using the server I-TASSER
The plasmid containing the PIF6-sequence was provided by the laboratory of Wilfried Weber from the University of Freiburg. For the synthesis of the Pif6 BioBrick primers containing the sites of the Fusion Protein BioBrick Assembly Standard were used.
Forward primer (5’->3’): 54 bp
GGATCCgaattcgcggccgcttctagatggccggcATGATGTTCTTACCAACCG
Reverse primer (5’->3’): 58 bp
CAGCTGctgcagcggccgctactagtattaaccggtGTCAACATGTTTATTGCTTTCC
∆N-ClpX (aa 61-425) - BBa_K365004
Background
ClpXP is a part of an E.coli protease which consists of three parts, the hexametric ClpX and two heptametrical ClpP subunits. ClpX consists of six identical subunits, each 1092bp long. ClpX recognizes and unfolds protein containing certain tags like LAA and leading them into the catalytic center of this protein complex, the two ClpP units. ClpX has two internal restriction sides for EcoRI and two restriction sides for AgeI.
Conception
ClpX has two internal restriction sides for EcoRI and two restriction sides for AgeI. The purpose of this first experimental part was to extract the ClpX gene out of the E.coli genome, to alter the internal EcoRI and AgeI sides in the ClpX gene and to fuse iGEM fusion pre- and suffixes to the ClpX sequence in order to get an iGEM Biobrick with standard prefix and suffix standard without internal EcoRI, Not, XbaI, AgeI, SpeI and PstI sides.
The sequence was obtained from the following database for DH5α E.coli cells:
1- http://www.ncbi.nlm.nih.gov/gene/945083
2- http://www.ncbi.nlm.nih.gov/nuccore/49175990?from=456650&to=457924&report=gbwithparts
Problem: 2 AgeI and 2 EcoRI sides
Primers for cloning ClpX out of the E.Coli genome.
These primes were used to amplificate ClpX from the E.coli genome.
Forward primer (5’->3’) : 31bp
CGCAGTGCGCTACCGACGCCGCATGAAATTC
Reverse primer (5’->3’) : 32bp
TTCACCAGATGCCTGTTGCGCTTCCGGCTTGC
Primers for Pfu-mutagenese
1. ACC=Thr AC(A,T,G,C), GGT=Gly GG(A,T,G,C)       AgeI site
Forward primer (5’->3’) (31 bp)
CTGATCGGTCCGACTGGTTCCGGTAAAACGC
Reverse primer (5’->3’) (31 bp)
GCGTTTTACCGGAACCAGTCGGACCGATCAG
2. GAA=Glu GA(A,G), TTC=Phe TT(C,T)       EcoRI site
Forward primer (5’->3’) (28 bp)
CATCCGCAGCAGGAGTTCTTGCAGGTTG
Reverse primer (5’->3’) (28 bp)
CAACCTGCAAGAACTCCTGCTGCGGATG
3. GAA=Glu GA(A,G), TTC=Phe TT(C,T)       EcoRI site
Forward primer (5’->3’) (25 bp)
CGTGGATCTGGAGTTCCGTGACGAG
Reverse primer (5’->3’) (25 bp)
CTCGTCACGGAACTCCAGATCCACG
4. ACC=Thr AC(A,T,G,C), GGT=Gly GG(A,T,G,C)       AgeI site
Forward primer (5’->3’) (24 bp)
GGCGCGTAAAACTGGTGCCCGTGG
Reverse primer (5’->3’) (24 bp)
CCACGGGCACCAGTTTTACGCGCC
Primers for amplification of ClpX with fusion pre- and suffix
After mutagenesis of internal restriction sides, the fusion pre- and suffixes were added to the ClpX gene.
Forward primer (5’->3’):
GGATCCgaattcgcggccgcttctagatggccggcCGCAGTGCGCTACCGACGCCGC
Reverse primer (5’->3’):
CAGCTGctgcagcggccgctactagtattaaccggtTTCACCAGATGCCTGTTGCGC
Linker (aa 20) - BBa_K365005
Background:
The linker biobrick is used to join the three ClpX subunits covalently in order to build a ClpX trimer and to link the degradation tags and PIF3/6 with the protein destined for degradation.
Conception:
We chose to use the same linker, which was already successfully used by [2] to construct the ClpX trimer. It is a twenty amino acid linker (ASGAGGSEGGGSEGGTSGAT). The codon usage of E. coli (http://www.geneinfinity.org/sp_codonusage.html) has been used to decide the DNA sequence, in addition RFC 25 fusion prefix and suffix have been added to the sequence.
The linker has been order as six separate, EcoRI+AgeI precut primers, which were hybridized in order to obtain the complete linker sequence.
Forward strand (5’->3’) : (89 bp)
aattcgcggccgcttctagatggc|cggcGCGAGCGGCGCGGGCGGCAGCGAAGGCGGCGGCAG|CGAAGGCGGCACCAGCGGCGCGACCa
Reverse strand (5’-3’) : (89 bp)
ccggtGGTCGCGCCG|CTGGTGCCGCCTTCGCTGCCGCCGCCTTCGCTGCC|GCCCGCGCCGCTCGCgccggccatctagaagcggccgcg
Ordered primer:
Forward:         5’aattcgcggccgcttctagatggc 3’
Forward:         5’cggcGCGAGCGGCGCGGGCGGCAGCGAAGGCGGCGGCAG  3’
Forward:         5’CGAAGGCGGCACCAGCGGCGCGACCa 3’
Reverse:          5’ccggtGGTCGCGCCG  3’
Reverse:          5’CTGGTGCCGCCTTCGCTGCCGCCGCCTTCGCTGCC 3’
Reverse:          5’GCCCGCGCCGCTCGCgccggccatctagaagcggccgcg3’
LAA tag - BBa_K365006
Background:
LAA tag is a C-terminal region of the natural ssrA-recognition sequence of E. Coli that interacts with the ClpXP protease. A protein fused with this tag will be preferentially degraded by the ClpX protease without need of an adaptor protein [6] . This tag serves as positive control for the functionality of the composed ClpXP and the PhyB-ClpXP fusion protein.
Conception:
We chose to use the same tag, which was already successfully used by used by [6] . It is a eleven amino acid tag (AANDENYALAA). At the end of the coding sequence a double stop codon was added (ACTAGT). The codon usage of E. coli (http://www.geneinfinity.org/sp_codonusage.html) has been used to decide the DNA sequence and NgoMIV and Pst1 restriction sites have been added to the sequence.
The LAA-tag has been order as two separate, NgoMIV + Pst1 precut primers, which were hybridized in order to obtain the degradation sequence.
LAA Tag
Ordered primer
Forward:         5’CCGGCGCGCTGGCGGCGTAATAATACTAGTAGCGGCCGC3’
Reverse:          5’ TGCAGCGGCCGCTACTAGTATTATTACGCCGCCAGCGCG 3’
For the synthesis of the PstI-site a mistake occurred in the command of the primers, as we did not consider that PstI cuts in the (3’ -> 5’) sens, contrary to the other restriction enzymes of the BioBrick standard. A supplementary step of ligation digestion in the experimental procedure can fix this mistake.
DAS tag - BBa_K365007
Background:
The DAS tag presents a C-terminal recognition  sequence that has been artificially altered so that it has weakened interactions with ClpXP and depends on an adaptor [6] . In E. coli, the adaptor SspB tethers specifically tagged substrates to the ClpXP protease, causing a modest increase in their rate of degradation. In our system, the role of the adaptor-protein SspB has been assumed by Pif3/6. So only light-induced activation can lead to binding and efficient degradation of DAS bearing constructs.
Conception:
We decided to use the same tag, which was already successfully used by [6] . It is a eleven amino acid tag (AANDENYADAS). At the end of the coding sequence a double stop codon was added (ACTAGT). The codon usage of E. coli (http://www.geneinfinity.org/sp_codonusage.html) has been used to decide the DNA sequence, in addition NgoMIV and PST1 restriction sites have been added to the sequence.
The DAS-tag has been order as two separate, NgoMIV + Pst1 precut primers, which were hybridized in order to obtain the degradation sequence.
Ordered primer
Forward:         5’ CCGGCGCGGATGCGAGCTAATAATACTAGTAGCGGCCGC3’
Reverse:          5’ TGCAGCGGCCGCTACTAGTATTATTAGCTCGCATCCGCG 3’
For the synthesis of the PstI-site a mistake occurred in the command of the primers, as we did not consider that PstI cuts in the (3’ -> 5’) sens, contrary to the other restriction enzymes of the BioBrick standard. A supplementary step of ligation digestion in the experimental procedure can fix this mistake.
Lambda tag - BBa_K365008
Background:
The λO- tag is the N-terminal equalent to the DAS tag. Degradation of proteins bearing the N-terminal λO- tag normally requires the N-domain of ClpX, which is missing in the PhyB-linker-[ClpX]3 variant.
[2] used this tag to test an artificial tethering system and demonstrated that it can serve as degradation signal for substrates that are tethered to ClpX.
Conception:
We chose to use the same sequence, which was already successfully used by used by [2] : NH2-TNTAKILNFGR. The codon usage of E. coli (http://www.geneinfinity.org/sp_codonusage.html) has been used to decide the DNA sequence, in addition NgoMIV and AgeI restriction sites have been added to the sequence.
The tag has been order as two separate, NgoMIV + Pst1 precut primers, which were hybridized in order to obtain the whole degradation tag.
Ordered primer
Forward:         5’AATTCGCGGCCGCTTCTAGATGACCAACACCGCGAAAATTCTGAACTTTGGCCGCA 3’
Reverse:          5’CCGGTGCGGCCAAAGTTCAGAATTTTCGCGGTGTTGGTCATCTAGAAGCGGCCGCG 3’
GFP (super fold) - BBa_K365009
Background
The GFP super fold protein originated from the BBa_I746915. This old biobrick contains the normal standard prefix and suffix. When this GFP is now fused to another biobrick a stop codon is created which would terminate the transcription of the fused biobrick. To avoid this inconvenience, we improved this biobrick to the fusion standard BBF RFC 25. This new biobrick can now be added to any biobrick with compatible restriction sides without creating a stop codon.
Conception
Standard PCR techniques were used. The Primer for amplification contained the fusion pre- and suffix and compatible base pair for the GFP gene.
PhyB642-(linker-∆NClpX)3 - BBa_K365010
Background:
This final construct will be the inducible part of the protease system. This permits the linking of the phytochrome B to the Clpx trimer which forms the light-dependent activation part of our degradation system.
Conception:
The standards cloning methods were used. This includes digestion of the vector containing linker- ClpX trimer and the PhyB part then the ligation of them and to finish the transformation of competente cells with this plasmid.
PhyB908-(linker-∆NClpX)3 - BBa_K365011
Background:
This final construct will be the inducible part of the protease system. This permits the linking of the phytochrome B to the Clpx trimer which forms the light-dependent activation part of our degradation system.
Conception:
The standards cloning methods were used. This includes digestion of the vector containing linker- ClpX trimer and the PhyB part then the ligation of them and to finish the transformation of competente cells with this plasmid.
Full-length ClpX - BBa_K365012
Background
ClpXP is a part of an E.coli protease which consists of three parts, the hexametric ClpX and two heptametrical ClpP subunits. ClpX consists of six identical subunits, each 1092bp long. ClpX recognizes and unfolds protein containing certain tags like LAA and leading them into the catalytic center of this protein complex, the two ClpP units. ClpX has two internal restriction sides for EcoRI and two restriction sides for AgeI.
Conception
ClpX has two internal restriction sides for EcoRI and two restriction sides for AgeI. The purpose of this first experimental part was to extract the ClpX gene out of the E.coli genome, to alter the internal EcoRI and AgeI sides in the ClpX gene and to fuse iGEM fusion pre- and suffixes to the ClpX sequence in order to get an iGEM Biobrick with standard prefix and suffix standard without internal EcoRI, Not, XbaI, AgeI, SpeI and PstI sides.
The sequence was obtained from the following database for DH5α E.coli cells:
1- http://www.ncbi.nlm.nih.gov/gene/945083
2- http://www.ncbi.nlm.nih.gov/nuccore/49175990?from=456650&to=457924&report=gbwithparts
Problem: 2 AgeI and 2 EcoRI sides
Primers for cloning ClpX out of the E.Coli genome.
These primes were used to amplificate ClpX from the E.coli genome.
Forward primer (5’->3’) : 31bp
CGCAGTGCGCTACCGACGCCGCATGAAATTC
Reverse primer (5’->3’) : 32bp
TTCACCAGATGCCTGTTGCGCTTCCGGCTTGC
Primers for Pfu-mutagenese
1. ACC=Thr AC(A,T,G,C), GGT=Gly GG(A,T,G,C)       AgeI site
Forward primer (5’->3’) (31 bp)
CTGATCGGTCCGACTGGTTCCGGTAAAACGC
Reverse primer (5’->3’) (31 bp)
GCGTTTTACCGGAACCAGTCGGACCGATCAG
2. GAA=Glu GA(A,G), TTC=Phe TT(C,T)       EcoRI site
Forward primer (5’->3’) (28 bp)
CATCCGCAGCAGGAGTTCTTGCAGGTTG
Reverse primer (5’->3’) (28 bp)
CAACCTGCAAGAACTCCTGCTGCGGATG
3. GAA=Glu GA(A,G), TTC=Phe TT(C,T)       EcoRI site
Forward primer (5’->3’) (25 bp)
CGTGGATCTGGAGTTCCGTGACGAG
Reverse primer (5’->3’) (25 bp)
CTCGTCACGGAACTCCAGATCCACG
4. ACC=Thr AC(A,T,G,C), GGT=Gly GG(A,T,G,C)       AgeI site
Forward primer (5’->3’) (24 bp)
GGCGCGTAAAACTGGTGCCCGTGG
Reverse primer (5’->3’) (24 bp)
CCACGGGCACCAGTTTTACGCGCC
∆NClpX-linker-∆NClpX-linker-∆NClpX - BBa_K365013
Background:
Phytochrome B needs to be fixed to the Clpx hexametric part of the protease. It is possible to link one phytochrome B per ClpX monomer but this could lead to steric problems. So the decision was made to follow the idea of the publication of Tanja Baker 2009. This permits to have only one phytochrome B for three ClpX units. Moreover the publication proved that the speed of the assembly of two ClpX trimers is quite the same than with ClpX monomers. So the Clpx hexameric part is composed of two ClpX trimers each one couple with a phytochrome B.
Conception: ClpX primer was constructed due to cloning methods. The new approach in this method is that the primer was built from a PCR product. The construction of the ClpX dimer was conducted with standard methods. Then in a PCR reaction the ClpX primers located inside of the dimer were used to construct this new primer. The product of this PCR was used as primer for constructing the ClpX trimer.
(linker-∆NClpX)3  - BBa_K365014
Background:
This construct was the final step for the ClpX hexamer creation. The two trimers will assemble to the full ClpX unit. Then the phytochrome B was cloned to this trimer. It will also be possible to add any other activation system like hormone receptors to this biobrick instead of phytochrome B, giving this part a huge potential of further applications.
Conception:
The standards cloning methods were used. This includes digestion of the vector containing ClpX trimer and the linker part then the ligation of them and to finish the transformation of competente cells with this plasmid.
Retrieved from " http://2010.igem.org/Team:ESBS-Strasbourg/Results/Biobricks "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
