<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ESBS-Strasbourg_Results_Characterization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ESBS-Strasbourg/Results/Characterization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ESBS-Strasbourg/Results/Characterization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/Results/Characterization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/Results/Characterization&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ESBS-Strasbourg/Results/Characterization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ESBS-Strasbourg/Results/Characterization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P>
 {|






</P><P><TITLE>ESBS - Strasbourg</TITLE></P><DIV class="header"><DIV class="menu"><DIV class="outer"><UL id="nav"><LI><P><A href="https://2010.igem.org/Team:ESBS-Strasbourg">  HOME</A></P></LI><LI><P><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Team">  TEAM</A></P></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Team">
				Overview</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Team#under">
				Students</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Team#advisors">
				Advisors</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Team#instructors">
				Instructors</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Team#uni">
				Strasbourg</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Team#collaboration">
				Collaboration</A></LI></UL><LI><P><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Project">  PROJECT</A></P></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Project">
				Overview</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Project/Strategy">
								Strategy</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Project/visual">
								Visual Description</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Project/Application">
								Application</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Project/Acknowledgment">
				Acknowledgment</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Project/Reference">
				Reference</A></LI><LI><P><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Results/Biobricks">  RESULTS</A></P></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Results/Biobricks">
				Biobricks</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Results/Assembly"><FONT size="3">Biobrick Assembly Technique</FONT></A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Results/Characterization">
								Characterization</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Results/Modelling">
				Modeling</A></LI><LI><P><A>  NOTEBOOK</A></P></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Notebook/Syntethic">
				Synthetic Photoreceptors</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Notebook/Microfluidics">
								Microfluidics</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Results/Device">Lighting device</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Notebook/Labbook">
				Lab-book</A></LI><LI><P><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Humanpractice">
			HUMAN PRACTICE</A></P></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Humanpractice#organisation">
				Organisation</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Humanpractice#survey">
				Survey</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Humanpractice#video">
								The ClpX video</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Humanpractice#game">
								The ClpX game</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Humanpractice#safety">
				Project Safety</A></LI><LI class="last"><P><A href="https://2010.igem.org/Team:ESBS-Strasbourg/Sponsors">     SPONSORS</A></P></LI></DIV></DIV></DIV></DIV><P><TABLE cellspacing="0"><TBODY><TR><DIV id="leftmenu"><TD width="210" rowspan="3" bgcolor="#414141" valign="top"><DIV class="heading">Characterization</DIV><DIV class="desc"><UL><LI><A href="#PLG">PIF6-linker-GFP</A></LI><LI><A href="#PhyB">Phytochrome B</A></LI></UL></DIV></TD></DIV></TR></TBODY></TABLE></P></DIV></DIV><DIV id="windowbox" style="position:fixed; top:50%; left:20px; width:11%;"><SPAN style="color:ivory;">  
Let me guide you</SPAN><DIV class="desc"><DIV class="heading">PIF6-linker-GFP</DIV>
In the course of our experiments, we were led to express PIF-linker-GFP to check if this fusion protein was fluorescent before going into further characterization of our system. Accordingly to the green fluorescence of the strain, we successfully expressed PIF6-linker-GFP under constitutive (promotor J23100 and RBS B0034) control (PIF3-linker-GFP is still in progress). This means this upstream tagging sequence does not prevent translation of the downstream protein. 
One really surprising result yet was that colonies expressing PIF6-linker-GFP were from far more fluorescent than those expressing GFP only. This came to a surprise since the expression of both proteins are controlled by the same promoter/RBS and that GFP-fusion proteins are more likely to be less fluorescent than GFP alone!
<CENTER>
Figure 1 : Pellets of the different strains. On the left is the negative 
control which expresses no GFP. In the middle is the positive control 
consisting of E. coli expressing the GFP. On the right is the culture of 
E. coli expressing PIF6-linker-GFP. 
</CENTER>
This led us to make two hypotheses:
-Either the fusion protein had an increased fluorescence efficiency due to a better folding/environment around its fluorophore.
-Or, most probably, PIF6-linker-GFP expression is stronger than GFP expression, due to the PIF6-linker sequence in Nter of GFP.
In order to find an answer to this question, we compared normalized fluorescence and normalized amount of fluorescent protein in each of these strains. 

First we measured fluorescence of the three different lysats (GFP negative strain, GFP expressing strain and  PIF6-linker-GFP expressing strain). Cells were resuspended and sonicated and the lysate was analyzed with a fluorimetre (absorption wavelenght=490nm, emission wavelenght=510nm). The test was done 3 times to give an average measurement. The table below shows the raw results. We then calculated the fluorescence over OD600 ration, the results are also reported in the table.
By normalizing the GFP strain fluorescence/OD600 ratio to 1, the PIF6-linker-GFP strain had a ratio of 15,5 (see chart 1). That is to say PIF6-linker-GFP expressing strains are 15,5 times more fluorescent than GFP expressing ones.
<CENTER>
Chart 1 : Relative specific fluorescence of GFP-, GFP and PIF6-linker-GFP expressing strains. 
</CENTER>
Simultaneously, we wanted to quantify the amount of overexpressed protein (either GFP or PIF6-linker-GFP) with a SDS-PAGE (see figure 2).<CENTER>
Figure 2 : Pictures of the SDS gel (on the left), and the corresponding immunoblot (on the right). We used rabbit anti-GFP  antibodies for the immunoblot.
</CENTER>
The GFP strain showed a new (or stronger) spot at around 27 kDa compared with the GFP negative control, which perfectly fits GFP's molecular weight. While the expected molecular weight of PIF6-linker-GFP was around 43 kDa,  the PIF6-linker-GFP strain showed several new spots when compared with the negative control, ranking from 43 to 32 kDa, which could be interpreted as truncated proteins. We thus performed an immunoblot assay using anti-GFP antibodies in order to see whether these spots were all responsible for the fluorescence within the PIF6-linker-GFP strain.

The immunoblot clearly showed a strong expression of different GFP containing proteins for the PIF6-linker-GFP construct. We can see that about 8 spots are recognized by anti-GFP antibodies. The GFP expressing strain shows the expected spot for GFP (red boxes). Nervertheless this spot is surprisingly detectable in the negative control. If it was also present in the PIF6-linker-GFP expressing strain, one could have assumed this spot was aspecific. Here most probably, this spot in the negative controle is a contamination.

We measured the intensities of the different bands between 43 and 32 kDa in the PIF6-linker-GFP lane using imageJ, corrected them accordingly to their molecular weights, and compared the cumulated value to the corrected intensity of the GFP spot. As a result, there was more than 30 times more fluorescent (GFP containing) proteins within the PIF6-linker-GFP strain than GFP in the GFP strain, wherease it is &quot;only&quot; 15,5 times more fluorescent.
Thus specific fluorescence is actually 2 times lower in PIF6-linker-GFP than in GFP, but this would be counterbalanced by a huge expression. 


We can conclude that PIF6-linker-GFP leads to the overexpression of different N-ter truncated GFP-containing proteins, whose cumulated quantity is greater than GFP in GFP expressing bacteria.

Presence of such truncated proteins could be explained by the high number of methionine codons within the PIF6 sequence, which would lead to several internal initiation of translation. This would mean that translation initiation is a limiting factor for expression of certain proteins such as GFP, a limit here possibly by-passed by several simultaneous initiations.  

More related to our project, fusing PIF6 sequence downstream GFP instead of upstream would prevent one to yield such truncated proteins.
</DIV><DIV class="heading">
Phytochrome B
</DIV><DIV class="desc"><P><B>His-tagging of Phytochrome B</B></P>
To facilitate further characterization steps the two length-variant phytochromes B, Phytochrome B 642 (PhyB 642, 
<A href="https://2010.igem.org/Team:ESBS-Strasbourg/Results/Biobricks#PHYB642">BBa_K365003</A>) and Phytochrome B 908 (PhyB 908, 
<A href="https://2010.igem.org/Team:ESBS-Strasbourg/Results/Biobricks#PHYB908">BBa_K365002</A>), were His-tagged at their C-terminal end.
<P><B>Phytochrome B expression</B></P>


In order to characterize the light-sensing part of our system the short PhyB 
642-His tag has been cloned into an expression plasmid under the control of the 
tetracycline-inducible promoter 
<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J13002">BBa_J13002</A> and verified by sequencing. 
Unfortunately, the cloning of the long PhyB 908-His tag did not succeed and was 
abandoned after multiple essays due to time constrains. Expression of PhyB 642 
in the ΔClpX W3110 cells was attempted over several weeks. All possible 
parameters have been altered using different anhydrotetracycline inducer 
concentration (from 50 ng/µl to 1 µg/µl), different expression temperatures 
(30°C and 37°C) and different sampling time points (3h, 6h, 9h and 16h after 
induction).




One possible reason for the expression failure could be the non-optimized 
codon usage of the Arabidopsis thaliana gene in E. coli. The analysis of the 
codon usage in E. coli for the PhyB 642 protein revealed a total of 7% of rarely 
used codons (arginine (AGA, AGG, and CGA), glycine (GGA) and proline (CCC)). In 
particular, there are four of these seldom codons (R31, R32, G33 and G34) in a 
row, which may lead to translation interruption and premature termination.

The expression system has been changed and three different Bl21 strains (Bl21 
wild type, Bl21 Star™ and Bl21 CodonPlus™) have been tried for expression. 
Especially the latter was promising in order to bypass the codon usage problem 
mentioned above. However, even with these three strains there was no PhyB642 
expression detectable.

<P><B>Further analysis of active Phytochrome B</B> (Theoretical part, as PhyB642 expression was not successful)</P><B>Zinc staining</B>


In the case, that PhyB is expressed in an active conformation, it is able to 
auto catalyze the covalent bonding of Phycocyanobilin (PCB), when PCB is added 
to the medium (final concentration 5µM typically). This covalent bond links the 
tetrapyrrole chromophore with a conserved cysteine residue of the phytochrome B 
<A href="https://2010.igem.org/Team:ESBS-Strasbourg/Project/Reference"><FONT>[37]</FONT>[Hill C et al., 1994]</A>. It can be easily detected by zinc staining followed by 
fluorescence analysis under UV light, whether PCB is attached to the phytochrome. 
Therefore, the protein extract is run on sodium dodecyl sulfate polyacrylamide 
gel electrophoresis (SDS-PAGE) to separate protein bound PCB from unbound PCB, 
incubated with zinc acetate and analyzed under UV light 
<A href="https://2010.igem.org/Team:ESBS-Strasbourg/Project/Reference"><FONT>[17]</FONT>[Mukougawa K et al., 
2006]</A>.
<P><B>Quantification of Phytochrome B activity by difference spectroscopy</B></P>


Difference spectroscopy can be used to analyze the light switching mechanism 
of the phytochrome B. The background properties, which are used in this test, 
are the two spectrally different forms of phytochrome, Pr and Pfr, as response 
to red and far-red light exposure. The difference spectrum is used to determine 
the amount of spectrally active phytochromes in the sample. Therefore, the Pfr 
absorption spectrum is subtracted from the Pr absorption spectrum, which results 
in the difference spectrum. Out of the difference between the minimum and the 
maximum of this spectrum, the active phytochrome concentration is calculated 
<A href="https://2010.igem.org/Team:ESBS-Strasbourg/Project/Reference">[21][McDowell MT and Lagarias JC, 2002]</A>.
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:ESBS-Strasbourg/Results/Characterization">http://2010.igem.org/Team:ESBS-Strasbourg/Results/Characterization</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ESBS-Strasbourg/Results/Characterization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ESBS-Strasbourg/Results/Characterization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/Results/Characterization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/Results/Characterization&amp;oldid=208157" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>