<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ESBS-Strasbourg_proteolux_scientific"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ESBS-Strasbourg/proteolux/scientific/">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ESBS-Strasbourg/proteolux/scientific/&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/proteolux/scientific/&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/proteolux/scientific/&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ESBS-Strasbourg/proteolux/scientific/" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ESBS-Strasbourg/proteolux/scientific/</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P>
 {|



</P><DIV style="position: absolute; width: 965px; height: 20px; z-index: 1; top: -40px; background-color:#222222" id="layer1"><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/contact" style="text-decoration: none"><FONT color="#FFFFFF">Contact</FONT></A>   <A style="text-decoration: none"><FONT color="#FFFFFF">|</FONT></A>   <A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/sitemap" style="text-decoration: none"><FONT color="#FFFFFF">Sitemap</FONT></A><P> </P></DIV><P><TITLE>ESBS-Strasbourg</TITLE></P><DIV class="header"><DIV class="menu"><DIV class="outer"><UL id="nav"><LI><P><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux">         HOME               |</A></P></LI><LI><P><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/overview">Overview             |</A></P></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/overview/proteoluxbasic">
				ProteOlux Basic</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/overview/proteoluxplus">
				ProteOlux Plus</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/overview/proteoluxpro">
				ProteOlux Pro</A></LI></UL><LI><P><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/scientific">
			Scientific Background             |</A></P></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/scientific/proteoluxbasic">
				ProteOlux Basic</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/scientific/proteoluxplus">
								ProteOlux Plus</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/scientific/proteoluxpro">
								ProteOlux Pro</A></LI><LI><P><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/application">          Application</A></P></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/application/cancer">
				Cancer</A></LI><LI><A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/application/neuro"><FONT size="3">Neurodegenerative Diseases</FONT></A></LI></DIV></DIV></DIV><P><TABLE cellspacing="0"><TBODY><TR><TD width="210" rowspan="20" bgcolor="#414141" valign="top"><DIV class="heading">ProteOlux Basic</DIV><DIV class="desc"><UL><LI><A href="#degradation">Degradation System</A></LI><LI><A href="#light">Light detection system</A></LI><LI><A href="#system">Final construction</A></LI><LI><A href="#chassis">Introduction to various chassis</A></LI></UL></DIV></TD></TR></TBODY></TABLE></P></DIV></DIV><DIV class="desc"><DIV class="heading">
 Degradation system
</DIV><P><B>Description</B></P>
ClpXP is an AAA protease present in bacteria, consisting of two main components, ClpX and ClpP. The ClpX is a hexamer consisting of six identical subunits. It recognizes specific degradation tags of target substrate proteins, unfolds them in an ATP-consuming hydrolysis reaction, and uses additional cycles of ATP hydrolysis to translocate the unfolded polypeptide into an interior chamber of ClpP, where proteolysis takes place. ClpP is a multi-subunit serine peptidase, in which the proteolytic active sites reside within a barrel-shaped structure. 
<CENTER>                      </CENTER>
In E. coli, the adaptor SspB tethers ssrA-tagged substrates to the ClpXP protease, causing a modest increase in their rate of degradation. 

<P><B>The recognition sequence</B></P>
In the native organism, the SsrA tag is added to incomplete proteins whose translation has been aborted. Thus, misfunctionnal proteins do not accumulate inside the cell. The ssrA tag is a natural well-characterized recognition for ClpXP-degradation sequence in E. Coli. It is composed of the 11 amino acid sequence AANDENYALAA, localized at the C-terminal of the target protein. ClpX recognizes the last three residues LAA(Leu9, Ala10 and Ala11). Proteins with C-terminal LAA- tags are degraded rapidly in the cell, even without presence of SspB.

To engineer controlled degradation, Baker and Sauer (2006) designed a series of modified SsrA tags that have weakened interactions with ClpXP. The DAS-tag presents one of these artificial sequence; its Kd value is significantly higher than the one of wild type SsrA, thus degradation of DAS-tagged proteins is not significant within the range of physiological concentrations. There is an N-terminal equivalent to the DAS-tag, the λO- tag. In the absence of SspB, substrates bearing the artificially altered tags are stable; however, through the action of the adaptor protein SspB, DAS- or λO –tagged proteins are significantly degraded.
<P><B>Substitution of the adaptor</B></P>
The role of the adaptor-protein SspB has been assumed by Pif6 in our system, only light-induced activation can lead to binding and efficient degradation of DAS bearing constructs. 
<P><SPAN>The target protein will be fused to Pif6 and to the specific degradation tag.To avoid problems with the accessibility or an increased activity of the target protein we provide two different variants: the C-terminal degradation tags, with the target protein construct [PIF6-linker-Protein-DAS] and the N-terminally fused λO-tag resulting in the construct [λO-Protein-linker-PIF6]. </SPAN></P>
Most bacterial proteins tolerate C-terminal fusions and the C-termini of most proteins are solvent accessible. The C-terminal fusion of PIF6 using the N-terminal degradation tag λO is consequently more promising to succeed, as the specific degradation sequence consists of very few amino acids (3 for the DAS-tag and 11 for the λO –tag) which do not perturb protein activity in most of the cases.
Nevertheless, if possible, we recommend to test the activity of the target protein within the two constructs before the implementation of Proteolux.


</DIV><DIV class="heading">
 Light detection system
</DIV><DIV class="desc">
As previously mentioned, the Phytochrome/PIF system has been chosen as light detection system. There are several advantages of this system: it offers a second timescale control which is significantly faster than chemically induced translocation systems, further it is perfectly reversible as it has been proven to be robust being cycled over a hundred times by alternating red and infrared illumination with no measurable decrease in recruitment ratios over time [41]. 
<P><B>Description</B></P>
Phytochromes are photoreceptive signaling proteins responsible for mediating many light-sensitive processes in plants. They detect red and near-infrared light through the photoisomerization of a covalently bound tetrapyrrole chromophore such as phycocyanobilin (PCB) for plant phytochromes. This photoisomerization event is coupled to an allosteric transition in the phytochrome between two conformational states called Pr (red-absorbing) and Pfr (far-red-absorbing). 
Upon stimulation with red light (650 nm), the phytochrome B (PhyB) protein binds directly to a downstream transcription factor, the phytochrome interaction factor (PIF). PIF is a nuclear-localized, basic helix–loop–helix (bHLH) factor initially isolated as interacting with the non-photoactive, C-terminal domain of Arabidopsis PhyB. The process is completely reversible through absorption in the near infra-red spectrum (705-740nm).

All plant phytochromes can be divided into an N-terminal photo sensory domain and a C-terminal dimerization domain. The N-terminal photo sensory domain comprises four consecutive subdomains called P1, P2/PAS, P3/GAF, and P4/PHY (named sequentially from the N terminus), the C-terminal domain consists of two subdomains, the PAS-A and PAS-B domains and the histidine kinase–related domain (HKRD) [45].
<P><SPAN>The P1 domain is not essential for the function of PHYB. Deletion of amino acids 1–57 of Arabidopsis PHYB yields a protein with full activity [30]. In contrast, the P2/PAS and P3/GAF domains form the essential photo sensory core domain. These domains contain a bilin lyase activity, which is responsible for the binding of the chromophore to a cysteine residue in the P3/GAF domain. The P2/PAS and P3/GAF domains play critical roles in photo sensing, whereas the P4/PHY domain is necessary for fine tuning phytochrome activity. The autophosphorylation and phytochrome-directed phosphorylation of other proteins, such as PIF3 is attributed to a serine/threonine kinase domain located in the N-terminal [1]. Deletion of the P4/PHY domain increases the dark reversion rate (i.e., the instability of the Pfr conformation) and causes a blue shift in absorption by both Pr and Pfr. The PAS-A and PAS-B domains of PHYB are necessary for dimerization and nuclear localization, whereas PAS-A, PAS-B, and the HKRD domains are necessary for nuclear speckle formation [1]. Dimerization is required for PhyB full activity.  </SPAN></P><P><B>Construction Choice</B></P>
To avoid potential sterical hindrance for ClpX activity we attempted to reduce its size as much as possible.

The light-sensitive interaction between PHYB and PIF3 has been mapped to the 650-residue amino- terminal photosensory core of PHYB [30]. The improved understanding of these mechanisms has been helpful for the design of the first engineered photoreceptors that utilized the interaction between PhyB and PIF3 to achieve light regulation of target proteins.
<P><SPAN>Leung et al. put the association of the GTPase Cdc42 with its effector protein WASP under red-light control. When in complex with PhyB-Cdc42, PIF3-WASP promoted actin polymerization in vitro; use in vivo was not demonstrated. Lastly, Voigt and colleges employed the light-dependent interaction between PhyB and PIF6 to activate target proteins in vivo. Thereby they have shown that the PIF-interaction with the PhyB photo sensory core (residues 1–642) is irreversible in infrared light. By assaying PIF6, which has the strongest interactions of all previously reported PIF domains, against different variants of PhyB they demonstrated that the tandem C-terminal PAS domains (residues 1-908) of PhyB are necessary to confer rapid photo reversibility under infrared light. </SPAN></P>
We have tested different variants for the implementation of our system: the residues 1-908 and the photo sensory core domain residues 1-642 for the phytochrome B constructs in combination with PIF3 (residues 1-100) and PIF6 (residues 1-100).
Our tests confirmed the findings of Voigt et al. that the tandem C-terminal PAS domains of PhyB are necessary to maintain the reversibility of the system. The Proteolux system therefore uses the PhyB residues 1-908 in combination with PIF6. The restriction of enzyme activity due to sterical hindrances could be reduced to less than 5% for the optimized final system. <P><B>The chromophore</B></P>

Since the plant phytochromes PhyA and PhyB employ the modified tetrapyrroles PCB or PΦB which are not available in most tissues and cell types, these chromophores must be supplied either exogenously or endogenously. It is possible to produce the holophytochrome in E. coli by co-expressing two genes from Synechocystis for chromophore biosynthesis together withcyanobacterial chromophore 1(Cph1) from the same organism [15],[35]. Heme oxygenase converts host heme to biliverdin IXK which is then reduced to phycocyanobilin via phycocyanobilin:ferredoxin oxidoreductase. The Cph1 apophytochrome is able to autoassemble with the phycocyanobilin in vivo to form the fully photoreversible holophytochrome.
Nevertheless, we decided to use the exogenous way by adding exogenous PCB as the endogenous way to produce the holoenzyme can lead to the production of toxic side-products. 



</DIV><DIV class="desc"><DIV class="heading">
 Final construction: Light controllable protease 
</DIV>
PhyB is fused to the N terminus of a trimeric form of ClpX-N in which the subunits were connected with a flexible linker to stabilize the enzyme. So the ClpX-hexamer is composed of two trimers fused to PhyB at its N-terminal.

The reason for this particular design is that N-domain dimerization is needed to stabilize the active hexameric form of ClpX [14]. So replacing this domain, as in our construction, would result in weaker hexamerization. <P> </P><P><SPAN>In our system the adaptor-role is assumed by PIF, which will bind to PhyB following light-induction. Target proteins are fused to PIF and tagged with the specific degradation sequence which, through light activation, brings the degradation sequence in proximity to ClpX and guides them to the catalytic core of the protease. Therefore a specific degradation of proteins containing the degradation sequence can be induced by a light signal. 
The system can be constitutively expressed in the chassis but it remains inactive until light-induction. However, it is expected to stay active for the background of naturally SsrA-tagged proteins, creating no interference with the natural occurring proteins.
</SPAN></P><P> </P></DIV><DIV class="desc"><DIV class="heading">
 Introduction to various chassis 
</DIV>
The Proteolux system can be applied to various different chassis. To optimize the expression we developed different versions of Proteolux based on an improved codon usage especially adapted to the target host system : 
<CENTER><UL><LI>Proteolux bacteria </LI><LI>Proteolux eucaria </LI><LI>Proteolux mammalia </LI></UL></CENTER>


Proteolux bacteria has been designed for an optimized expression in bacteria, the codon usage is primarily adapted to the class of proteobacteria. Proteobacteria are a major group of bacteria. They include a wide variety of pathogens, such as Escherichia, Salmonella, Vibrio, Helicobacter and many other notable genera. Others are free-living, and include many of the bacteria responsible for nitrogen fixation or involved in the carbon cycle.  


Proteolux bacteria has been successfully implemented in various representatives of proteobacteria, including Caulobacter crescentus, Escherichia Coli, Pseudomonas fluorescens , Salmonella enterica and Vibrio fischeri.


Proteolux eucaria has been designed for the use in eukaryotic microorganism as Saccharomyces cerevisiae that represents one of the most intensively studied unicellular eukaryotic model organisms in molecular and cell biology. Many proteins important in human biology were first discovered by studying their homologs in S. cerevisiae; these proteins include cell cycle proteins, signaling proteins, and protein-processing enzymes. 


Proteolux eucaria has further been tested in Schizosaccharomyces pombe, another yeast species and Ashbya gossypi, a model organism for filamentous fungi. 


Proteolux mammalia aims at the use in the medical research sector, its codon usage has been adapted to the human species. 

The system can be implemented in different mammalian cell lines in vitro, including the human cell lines HEK293, K562 and HeLa, embryonic mouse fibroblast cells 3T3 as the baby hamster kidney cell line BHK-21. 


For in vivo implementations you should consider the use Proteolux Pro, which presents a special variant of Proteolux mammalia. 



The introduction of Proteolux can be realized with common transformation, transfection or transduction methods. It requires the introduction of the light inducible degradation complex ClpX-PhyB as the ClpP-gene for non-bacterial systems. 

In proteobacteria species encode the ClpXP protease in their proper genome. The naturally present ClpP-subunit assembles auto-catalytically with ClpX, therefore the ClpP-gene has not to be added in proteobacteria, 

The target gene needs to be fused with Pif6 and the specific degradation sequence. We provide two different variants to avoid problems with the accessibility or an increased activity of the target protein: The N-terminally fused λO-tag, resulting in the target protein construct [λO-protein-linker-PIF6], and the C-terminal degradation tag DAS, resulting in the target protein construct  [PIF6-linker-Protein-DAS]. 

Most bacterial proteins tolerate C-terminal fusions and the C-termini of most proteins are solvent accessible. Consequently, the C-terminal fusion of Pif6 using the N-terminal degradation tag λO is more promising to succeed, as the specific degradation sequence consists of very few amino acids (3 for the DAS-tag and 11 for the λO–tag) which do not perturb protein activity in most of the cases. 
If possible, we recommend to test the activity of the target protein within the two constructs before the implementation of Proteolux, as the integration of the Pif6- and degradation tag-flanked target protein cassette presents a more time-consuming step for analytical purposes. 
The analysis of a specific target protein requires its fusion to the adaptor Pif6 and the specific degradation tag λO/DAS. To simplify the addition of these two sequences to your target gene we provide a special plasmid containing standard restriction sides for the integration of the target gene. Thus the integration requires no more that the amplification of your target gene with the standard restriction sites and its ligation into the plasmid. 

To avoid interferences with the original protein it is necessary to silence the native gene. Therefore it is necessary to a gene-knock-out using the common gene targeting approaches based on homologous recombination. We recommend to integrate the Pif6- and degradation tag-flanked target protein cassette directly into the native gene or to substitute it with per gene-knock-in. 

This requires a further amplification step adding the specific “homology arms” which must match which must match the genomic DNA of the cell line being targeted, so in our case the native target gene sequence. These arms drive the homologous recombination event that results in insertion of the construct into the desired locus.

It is the client’s responsibility to design and engineer the targeting construct. See our tips on <A href="https://2010.igem.org/Team:ESBS-Strasbourg/proteolux/scientific/proteoluxbasic/constructdesign" style="text-decoration: none"><FONT color="#FF9900">targeting construct design.</FONT></A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:ESBS-Strasbourg/proteolux/scientific/">http://2010.igem.org/Team:ESBS-Strasbourg/proteolux/scientific/</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ESBS-Strasbourg/proteolux/scientific/" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ESBS-Strasbourg/proteolux/scientific/" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/proteolux/scientific/&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/proteolux/scientific/&amp;oldid=209189" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>