"
Page
Discussion
View source
History
teams
Log in
 
Team:ESBS-Strasbourg/Project/Strategy
From 2010.igem.org
{|
ESBS - Strasbourg
  HOME
  TEAM
Overview
Students
Advisors
Instructors
Strasbourg
Collaboration
  PROJECT
Overview
Strategy
Visual Description
Application
Acknowledgment
Reference
  RESULTS
Biobricks
Biobrick Assembly Technique
Characterization
Modeling
  NOTEBOOK
Synthetic Photoreceptors
Microfluidics
Lighting device
Lab-book
HUMAN PRACTICE
Organisation
Survey
The ClpX video
The ClpX game
Project Safety
     SPONSORS
PROJECT INDEX
Our device construction is divided into 4 components :
1. Degradation system
2. Light detection system
3. Protein Tagging
4. Light controllable protease
Biobricks
  
Let me guide you
Introduction
The work of the former IGEM Team had a great influence on our choice, particularly those of The ESBS 2008 Team . Their goal was to control cells’ state by linking the natural variability of life to the binary system of computing. In the project of The ESBS 2008 Team , the incrementing from one bit to another required a protease which was expressed during the mitosis in yeast. Renaud Renault of the actual iGEM team had thus the idea of inventing a degradation system which would be not only temporally controllable but also specific.
Based on this idea we chose to create a light-controllable specific protein degradation system. The system contains several parts: the bacterial ClpXP protease from E. Coli and the specific recognition sequence (DAS-tag) for ClpX for the degradation part as the photoreceptor protein phytochrome B (PhyB) and the phytochrome interacting factor (PIF3/6) from A. thaliana for the light-dependent part of the system.
The different parts, their basic ideas and their strategic development will be discussed in detail in the following separated parts. The choice of the host organism will also be explained, followed by the final structure of the light controllable protease.
Degradation system
The ClpXP from E.coli protease rapidly established itself as an evident choice. Indeed, this protease has been very well studied, notably by the Tania Baker Team from the MIT.
Description
Basically, ClpXP is an AAA protease present in bacteria, consisting of two main components, ClpX and ClpP. The ClpX is a hexamer consisting of six identical subunits. It recognizes specific degradation tags of target substrate proteins, unfolds them in an ATP-consuming hydrolysis reaction, and uses additional cycles of ATP hydrolysis to translocate the unfolded polypeptide into an interior chamber of ClpP, where proteolysis takes place. ClpP is a multi-subunit serine peptidase, in which the proteolytic active sites reside within a barrel-shaped structure.
                      
How using the ClpXP protease ? : The question of the use of an adaptator
This question was really critical. We focused on different strategies before making a definitive choice.
The publication of Tania Baker [7] based on the ClpXP protease of E. Coli which degrades substrates bearing the specific SsrA recognition sequence, has been the starting point of our reflection. In this work Baker and colleagues designed a series of modified ssrA tags which have weakened interactions with ClpXP to engineer controlled degradation. In E. coli, the adaptor SspB tethers ssrA-tagged substrates to the ClpXP protease, causing a modest increase in their rate of degradation. In the absence of SspB , substrates bearing the artificially altered DAS-tag were stable, in contrast the degradation of substrates bearing these engineered peptide tags was 100-fold more efficiently when SspB was present.
Upon these findings our first idea consisted in using the native ClpX with the mutated tag and a modified SspB whose binding to ClpX should be controlled by light in order to control protein degradation.  Forcing cells to produce an inactive form of an adaptator seem to be a good solution to  be able to stop the degradation at a certain point. This could be realized by producing two parts of the adaptator which could interconnect them after light-induction by fusing them to proteins which had this capacity, for instance the couple Phytochrome/PIF.
However, the use of the adaptor-based system posed some major problems concerning the complexity. Subsequent to another finding of Baker et. al., we decided to fuse our phytochrome directly to the N-terminal of ClpX, as it is not required for the basic enzymatic functions of ClpX [7] .
Further, we decided to fuse the target protein, additionally to the specific degradation tag, with PIF which will assume the role of the adaptor protein SspB .
Light detection system
 
 
As told before, the Phytochrome/PIF system has been chosen as light detection system. Many reasons have motivated our choice: it is well-characterized, offers a second timescale control which is an order of magnitude faster than previous chemically induced translocation systems and are very near the physical limits for whole-cell diffusion. It has  also been proven to be robust being cycled over a hundred times by alternating red and infrared illumination with no measurable decrease in recruitment ratios over time [41] .
 
 
Description
Phytochromes are photoreceptive signaling proteins responsible for mediating many light-sensitive processes in plants, including seed germination, seedling de-etiolation and shade avoidance. They detect red and near-infrared light through the photoisomerization of a covalently bound tetrapyrrole chromophore such as phycocyanobilin (PCB) for plant phytochromes. This photoisomerization event is coupled to an allosteric transition in the phytochrome between two conformational states called Pr (red-absorbing) and Pfr (far-red-absorbing).
Upon stimulation with red light (650 nm), the phytochrome B (PhyB) protein binds directly to a downstream transcription factor, the phytochrome interaction factor (PIF). PIF is a nuclear-localized, basic helix–loop–helix (bHLH) factor initially isolated as interacting with the non-photoactive, C-terminal domain of Arabidopsis PhyB.
Construction choice
We needed to study the different domains of the phytochrome B in order to try to reduce its size which can be a potential sterical hindrance for ClpX activity.
All plant phytochromes can be divided into an N-terminal photo sensory domain and a C-terminal dimerization domain. The Nterminal photo sensory domain comprises four consecutive subdomains called P1, P2/PAS, P3/GAF, and P4/PHY (named sequentially from the N terminus), the C-terminal domain consists of two subdomains, the PAS-A and PAS-B domains and the histidine kinase–related domain (HKRD) [45] . The PAS domain is named after three proteins in which it occurs: Per (period circadian protein), Arn (Ah receptor nucleartranslocator protein), and Sim (single-minded protein) [1] .
The P1 domain is not essential for the function of PHYB. Deletion of amino acids 1–57 of Arabidopsis PHYB yields a protein with full activity [30] . In contrast, the P2/PAS and P3/GAF domains form the essential photo sensory core domain. These domains contain a bilin lyase activity, which is responsible for the binding of the chromophore to a cysteine residue in the P3/GAF domain. The P2/PAS and P3/GAF domains play critical roles in photo sensing, whereas the P4/PHY domain is necessary for fine tuning phytochrome activity. Deletion of the P4/PHY domain increases the dark reversion rate (i.e., the instability of the Pfr conformation) and causes a blue shift in absorption by both Pr and Pfr. A serine/threonine kinase domain that governs phytochrome autophosphorylation and phytochrome-directed phosphorylation of other proteins, such as the phytochrome interacting factor (PIF3) has also been located in the N-terminal domain [1] . The PAS-A and PAS-B domains of PHYB are necessary for dimerization and nuclear localization, whereas PAS-A, PAS-B, and the HKRD domains are necessary for nuclear speckle formation [1] . Dimerization is required for PhyB full activity.
The light-sensitive interaction between PHYB and PIF3 has been mapped to the 650-residue amino-terminal photosensory core of PHYB [30] .
The improved understanding of these mechanisms has been helpful for the design of the first engineered photoreceptors. By now there have been three illuminating studies that utilized the interaction between PhyB and PIF3 to achieve light regulation of target proteins.
Leung et al. put the association of the GTPase Cdc42 with its effector protein WASP under red-light control. When in complex with PhyB-Cdc42, PIF3-WASP promoted actin polymerization in vitro; use in vivo was not demonstrated. Based on the interaction between PhyB and PIF3, Muir and Tyszkiewicz established a protein-splicing system that was moderately regulated by red light in vitro65. Lastly, Voigt and colleges employed the light-dependent interaction between PhyB and PIF6 to activate target proteins in vivo. The nucleotide exchange factors Tiam and intersectin were recruited to the plasma membrane in a red-light-controlled manner where they activated their GTPase effectors Rac1 and Cdc42, respectively. In their activated form, the GTPases promoted formation of cell protrusions, and thus the motility of fibroblasts could be controlled by red light.
With this first successful in-vivo application Voigt et al. have shown that the PIF-interaction with the PhyB photo sensory core (residues 1–642) is irreversible in infrared light. By assaying PIF6, which has the strongest interactions of all previously reported PIF domains, against different variants of PhyB they demonstrated that the tandem C-terminal PAS domains (residues 1-908) of PHYB are necessary to confer rapid photo reversibility under infrared light. The interaction with PIF3 has been too weak to cause significant translocation.
 
Upon these findings, we decided to tests different variants for the implementation of our system: for the phytochrome constructs we choose PhyB residues 1-908 (PhyB900) and the photo sensory core domain residues 1-642 (PhyB650) in combination with PIF3 (residues 1-100) and PIF6 (residues 1-100).
The choice to include the shorter PhyB650 variant in our tests is reasoned by the risk of a potential sterical hindrance of ClpX due to the fusioned PhyB. Due to its reduced size, the PhyB650 construct provides an additional control for the case that the system implementation using PhyB950 does not produce results. Further, binding strength and kinetic parameters depend on the composition and nature of the individual system, so we thought it a good idea to test the findings of Voigt et al. in a novel background.
The question of the chromophore
Since the plant phytochromes PhyA and PhyB employ the modified tetrapyrroles PCB or PΦB which are not available in most tissues and cell types, these chromophores must be supplied either exogenously or endogenously. It is possible to produce the holophytochrome in E. coli by co-expressing two genes from Synechocystis for chromophore biosynthesis together withcyanobacterial chromophore 1(Cph1) from the same organism [15],[35] . Heme oxygenase converts host heme to biliverdin IXK which is then reduced to phycocyanobilin via phycocyanobilin:ferredoxin oxidoreductase. The Cph1 apophytochrome is able to autoassemble with the phycocyanobilin in vivo to form the fully photoreversible holophytochrome.
Nevertheless, we decided to use the exogenous way by adding exogenous PCB and to focus on our main objective to prove the functionality of our system. Beside that it has been shown that the endogenous way to produce the holoenzyme leads to the production of toxic side-products.
Protein Tagging
Construction choice
 
The addition of the tag could lead to some problems as compromised folding or increased activity of the target protein.  Most bacterial proteins tolerate C-terminal fusions and the C-termini of most proteins are solvent accessible. Chances are, that no major problems occur when we add our tag.
For the C-terminal degradation tags, the target protein construct will be PIF-linker-Protein-Tag. To avoid problems with the accessibility of the tag we decided to test a further contruction with a N-terminally fused degradation tag (Tag-Protein-linker-PIF).
We need then to choose an appropriate tag. It was also a critical step.
The LAA Tag
The ssrA tag is a natural well-characterized recognition for ClpXP-degradation sequence in E. Coli. It is composed of the 11 amino acid sequence AANDENYALAA, localized at the C-terminal of the target protein. At least five ClpX-recognizing motifs have been determined: three located at the N-terminus and two at the C-terminus [34] . Here we are concentrating on those located at the C-terminus where ClpX recognizes the last three residues (Leu9, Ala10 and Ala11).
ClpX alone is able to interact with the ssrA-tagged substrates and delivers them to ClpP protease. However, it has been shown that the adaptor protein, SspB, markedly enhances the recognition of the ssrA tag. So, in our case the PIF-fused target should also favor the recognition of the ssrA tag.
Baker et al. have shown that proteins with C-terminal LAA- tags are degraded rapidly in the cell, even without presence of SspB. This tag serves as positive control for the functionality of the composed ClpXP and the PhyB-ClpXP fusion protein.
The DAS Tag
Replacing two residues in the ssrA tag weakens ClpX binding. Ser was chosen as an allowed but not preferred C-terminal residue and Asp as the antepenultimate residue because this substitution decreases ClpXP and ClpAP degradation modestly. The ssrA tags with these mutations are referred as DAS tags [2] . Besides, no other proteases than ClpXP degrades DAS-tagged proteins. Therefore we chose the DAS tag  as specific recognition sequence for our system. As the role of the adaptor-protein SspB has been assumed by Pif3/6 in our system, only light-induced activation can lead to binding and efficient degradation of DAS bearing constructs.
The Lambda Tag
The λO- tag is the N-terminal equivalent to the DAS-tag. Degradation of proteins bearing the N-terminal λO- tag normally requires the N-domain of ClpX, which is missing in the PhyB-linker-[ClpX]3 variant.
Baker and colleges used this tag to test an artificial tethering system and demonstrated that it can serve as degradation signal for substrates that are tethered to ClpX.
Light controllable protease
The choice of the host organism
Another important point was the choice of the host organism. At the beginning we considered three different options: Saccharomyces cerevisae that was used by the former iGEM Team of the ESBS, Bacillus subtilis and Escherichia coli. The advantage of yeast was that it is a eukaryotic organism which would have been interesting for further applications in eukaryotic cells. At the other site this would impede the control of transcriptional regulation, which one of the initial application ideas of our protease, due to the nucleus’ membrane. The disadvantages are that the ClpP-coding sequence lacks in their genome which would have to be additionally cloned in. Moreover the growth rate of the cells is lower than the one of E.coli which would have slowed the laboratory work. The disadvantages of E.coli as host are the necessary knock-out of the ClpX gene for the control and the risk of interference of host-proteins with the synthetic system. This would have been an advantage for the use of B. subtilis but here the impact of a synthetic protease system is far less characterized than in E.coli.
Thus, the final choice of the host organism was E.coli as this organism is best characterized for the work in molecular genetics and also tested for synthetic engineering of ClpX andthe presence of the ClpP-subunit that assembles auto-catalitically with ClpX.
The Linker
The linker has been designed for the use in different construction to avoid steric hindrance. The sequence is composed of the twenty amino-acids ASGAGGSEGGGSEGGTSGAT.
Final construction
 
 
 
Even if we had a clear idea of our system, we needed more information about the constructional design. During our investigations, we found an extremely helpful article for this issue: "Engineering synthetic adaptors and substrates for controlled ClpXP degradation" from Tania Baker and collegues (2009).
 
 
In this work Baker et al. probed minimal biochemical functions necessary for efficient degradation by designing and characterizing variant substrates, adaptors and ClpX-enzymes. They implemented a rapamycin-dependent tethering system and demonstrated that artificial tethering can support substrate delivery.Therefore they constructed a ClpX variant lacking the N-domain that contained the human FKBP12 protein. The FKBP12 was fused to the N terminus of a trimeric form of ClpX-N in which the subunits were connected with a flexible linker to stabilize the enzyme. The reason for this particular design is that N-domain dimerization is needed to stabilize the active hexameric form of ClpX [14] so that replacing this domain would probably result in weaker hexamerization. Upon these findings we decided to construct the PhyB-ClpX fusion protein in the same manner.
For the adaptor-mediated delivery they fused an altered SspB variant to the FRB domain from rat m. In the presence of the small molecule rapamycin the FKBP12 protein and the FRB domain bind to each other with high affinity, consequently the adaptor-mediated delivery of their system depends on rapamycin-induction. The experiments resulted in efficient adaptor-dependent degradation.
In our system the adaptor-role is assumed by PIF, which will bind to PhyB following light-induction. Target proteins are fused to PIF and tagged with the DAS-degradation sequence which, through light activation, brings the degradation sequence in proximity to ClpX and guides them to the catalytic core of the protease. Therefore a specific degradation of proteins containing the degradation sequence can be induced by a light signal.
The system can be constitutively expressed in the chassis but it remains inactive until light-induction. However, it is expected to stay active for the background of naturally SsrA-tagged proteins, creating no interference with the natural occurring proteins of E.coli.
The E. Coli stem
The native ClpXP in E.coli would interfere with the measurements of the degradation and the degradation itself, so we decided to work in a ClpX-deficient E. Coli strain.
Prediction of possible problems
One problem could be a too slow protease activity of the ClpXP to clear the tagged protein. Endogenous levels of ClpXP are capable of degrading more than 100,000 copies of a tagged substrate per generation in E. coli [18] .
All in all, we had a precise and feasible project to start working.
Retrieved from " http://2010.igem.org/Team:ESBS-Strasbourg/Project/Strategy "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
