Team:Hong Kong UCCKE/Experiments
Toggle navigation
About
Team
Collaborations
Contact
Project
Description
Experiments
Results
Notebook
Parts
Parts
Basic Parts
Composite Parts
Safety
Attributions
Human Practices
Human Practices
Engagement
Art & Design
Logo
Web Design
TOUR
UCCKE / Experiments
In this page, you can know about what experiments have we done and our protocol used.
Experiments
Assay
We have done assays on pure chemicals and bacteria synthesized chemicals.
The aim of doing these assays was to test whether the substances are attractants or repellents to C. elegans. And we used response index to describe whether the chemicals are attractants or repellents to C. elegans.
Below I explain the steps and calculations involved in these assays in detail.
General Steps
As shown in the picture below, we mark the substance loading spots on the lid of a petri dish where we put volatile substances. For droplet test, same labelling were done on the plate.
Mark the same spots with extra circles around each loading spot. Label two opposite area as ‘T’ and the other two as ‘C’. they stand for ‘test’ and ‘control’ respectively.
Load 10 ul of medium containing about 200 worms on the centre of the the agar.
Loading chemicals
For volatile test Pipetted 2 ul of tested volatile substance on the loading spot in the two ‘T’ areas on the lid. And load the same volume of water on the ‘C’ spots on the lid.
For droplet test Pipetted 5ul of tested liquid on the loading spot in the two ‘T’ areas on the agar plate. And load the same volume of water on the ‘C’ spots on agar too.
Then, we put the assay plate aside and leave it undisturbed for 30 minutes. After 30 minutes, we obtained the total number of worms in the ‘T’ areas and ‘C’ areas.
Responsive Index
The response index was given by formula [(T-C)/(T+C)]. T is the number of worms in the ‘T’ areas. C is the number of worms in the ‘C’ areas, which means the number of worms that entered the areas by chance. The denominator [T+C] represents the total number of worms in the counted areas, that is the number of worms inside the 4 circles. The numerator [T-C] is the number of worms which made response due to the effect of the tested substances.
The index value will range between -1 and 1. If the index is negative, it means the tested chemical is a repellent; in contrast, positive index means the tested chemical is an attractant.
Positive Control
We tested our assay and response index by chemical diacetyl, a known strong attractant to C. elegans. And the result is promising that the Response index of volatile is close to +1.00, which means that assay and response index can show diacetyl is a strong attractant.
Negative Control
We also tried using water as negative control for the assay. The Response index is +0.07. Which means the assay can show that the worms have no significant response, neither attraction or repulsion, to water.
Assay on pure substances
We tested on pure chemicals of cinnamaldehyde and phenylpyruvic acid to check if our assumptions about the attractiveness were correct.
The result is that index of cinnamaldehyde is -0.81 when concentration is 1. Phenylpyruvic acid’s index is +0.34. With the evidence provided by the tests, the results are clearly shown. Cinnamaldehyde is a strong repellent and phenylpyruvic acid is a moderate attractant. Please refer to the result page for a more detailed assay results.
Assay on bacteria synthesized chemicals
We did a similar assay on bacteria synthesized cinnamaldehyde and phenylpyruvic acid. Other than testing the bacteria synthesized chemicals, we also conducted assay to test on bacteria containing backbone-only plasmid as a negative control with water remaining as a control. This is to ensure it is the bacteria synthesized chemical which affect the assay result instead of the plasmid or bacteria itself involving in the results. The response indexes taken in assay on bacteria synthesized phenylpyruvic acid and backbone-only plasmid bacteria are +0.83 and +0.63 respectively. Please refer to the result page for a more detailed assay results. The result indicates that the plasmid and bacteria itself is already an attractant to C.elegans. However, on top of the attractiveness produced by the bacteria with plasmid backbone, phenylpyruvic had increased the overall attractiveness. For cinnamaldehyde, we were not able to clone the genes into a plasmid and do transformation and thus we did not conduct any assay on bacteria synthesized cinnamaldehyde.
Cloning
Cinnamaldehyde
Based on the assays done on pure chemicals, cinnamaldehyde is a repellent while phenylpyruvic acid is an attractant. Hence, it would be more convenient for us to choose parts that are registered on the iGEM. We found a synthesis pathway of cinnamaldehyde with genes in the iGEM parts library, given by the 2013 UBC team [1] . The pathway is constructed with 3 genes, where the starting substrate is phenylalanine, an amino acid. With the sequence of the three genes provided, we designed and ordered the 3 oligo DNA from IDT, the Integrated DNA Technologies, Inc..
Each ordered oligo DNA sequence encloses a constitutive promoter J23100 [2] , medium strength RBS B0034 [3] , a gene in the cinnamaldehyde synthesis pathway and a double terminator B0015 [4] , with iGEM prefix and suffix [5] for a more convenient use in 3A assembly cloning steps.
We were supposed to sub-cloned these genes into 3 separate plasmid backbones then clone the 3 parts into one plasmid backbone. And lastly transform the plasmid into bacteria. Unfortunately, we were not successful in doing it.
Phenylpyruvic acid
Phenylpyruvic acid is an attractant. However, we had difficulties find a protein sequence that enables the production of phenylpyruvic acid in the iGEM community. As such, we decided to synthesize it ourselves. We discovered a protein sequence for producing phenylpyruvic acid [6] on NCBI website. We again ordered the oligo DNA from IDT. The ordered sequence also encloses a constitutive promoter J23100 [7] , medium strength RBS B0034 [8] , the target gene and a double terminator B0015 [9] , with iGEM prefix and suffix [10] . Our new gene is an amino acid deaminase gene [11] .
After successful cloning bacteria with phenylpyruvic acid producing gene, we used this composite part to perform our assays.
Protocols
3A Assembly
Download
Transformation
Download
Reference
[1] : UBC Team (2013) 2013 UBC Team, Available at:https://2013.igem.org/Team:British_Columbia (Accessed: 19th October 2016).
[2] [7] : iGEM (2016) Promoter J23100, Available at: http://parts.igem.org/Part:BBa_J23100(Accessed: 19th October 2016).
[3] [8] : iGEM (2016) RBS B0034, Available at: http://parts.igem.org/Part:BBa_B0034(Accessed: 19th October 2016).
[4] [9] : iGEM (2016) Terminator B0015, Available at: http://parts.igem.org/Part:BBa_B0015(Accessed: 19th October 2016).
[5] [10] : iGEM (2016) Prefix and suffix, Available at: http://parts.igem.org/Help:Prefix-Suffix(Accessed: 19th October 2016).
[6] [11] : Appl Microbiol Biotechnol. 2016 Mar;100(5):2183-91. doi: 10.1007/s00253-015-7048-5. Epub 2015 Nov 10.
Previous Page:
Description
Next Page:
Results
Contact
E-mail: bioresearchteam@gmail.com
Address
United Christian College (Kowloon East),
2 Lee On Lane,
Kwun Tong, HK
Follow Us
Content by UCCKE team. Designed and programmed in Hong Kong with by Gabriel Lee.
