"
Page
Discussion
View source
History
teams
Log in
 
Team:RHIT/Project
From 2012.igem.org
iGEM Home Login
iGEM Home   My Account   Log Out
Home
Team
Project
BioBricks
Modeling
Notebook
Outreach
Overview
Planning
Detailed Description
Simple Science
Basic Description
The Checkmate project is designed to address an existing challenge for yeast researchers. The typical test to determine the mating type of haploid cells is laborious and takes about 40 hours. Our goal is to streamline this process and reduce the time required to about four hours.
Background
Haploid and diploid yeast cells both use mitosis to reproduce and grow vegetatively. Diploid cells can also use meiosis to sporulate and produce four haploid spores. To leverage this facile genetic system, it is often necessary to determine the mating type of haploid cells, which are one of two mating types, MATa or MATalpha. Each secretes its own type of mating pheremone and has receptors for the opposite type mating pheromone on its surface. When haploids of opposite mating type encounter one another they costimulate, which activates the mating pheromone response pathway in each. The key transcription factor Ste12 is activated as part of this response. It subsequently binds to Ste12-binding elements to induce expression of other genes necessary for mating and diploid formation.
Current Test
The current test for mating type takes several days. It involves mixing the unknown strain with two known tester strains, each of which have a known auxotrophic deficiency, which is different from the auxotrophic deficiency of the known strain. The mixes are then plated on media that is deficient in both substances that are unable to be produced by the haploid strains, such that neither tester strain nor unknown strain can survive as a haploid. If the cells mate, then they get a working copy of their defective gene, so the diploid can survive.  Below is a graphic illustrating the current test process.
Our Test
Our test would involve mixing the unknown strain with each of our tester strains and then plating on YPD media. If mating occurs, the pheromone response pathway will initiate transcription of a fluorescent protein with a DNA-binding domain that binds to its own promoter, effectively forming a positive feedback loop. Our system is also known as a “latch circuit”, one that responds to an initial stimulus and then maintains its response indefinitely. Over several hours, fluorescence will become visible under a fluorescence microscope, thus signaling that mating occurred. Two different fluorescent proteins were used, one for each mating type, so that results would be unambiguous if the plates got mixed up. Below is a graphic illustrating how our test improves upon the current test.
Documentation: Planning Process
Under the leadership of advisor Dr. Yosi Shibberu, Rose-Hulman’s iGEM team started off the year with an iterative process of project planning. The method using the five steps found in chapter three of David Allen’s book, “Getting Things Done”, titled Getting Projects Creatively Under Way: The Five Phases of Project Planning. Allen proposes a five step process to focus all members on the same goal of the project: defining a purpose, visioning, brainstorming, organizing, and identifying next actions.       The group went through three iterations of this method, resulting in the purpose and principles that follow.
Purpose
Determine the mating type of yeast, Saccharomyces cerevisiae, through fluorescence measurements more easily and quickly than the current method, with a goal time of four hours or less. In addition, constitutively label a and alpha test strains. Go for the Gold!
Principles
Execute work that produces quality long-term information with the goal of being open source.
Learn from past teams' mistakes.
Boost each other up; don't bring anybody down.
Once the group agreed on a common purpose and principles, the team brainstormed ideas for the needs of the project. These ideas were then organized into categories based on similar themes. A few examples of the categories are team practices, mathematical modeling, presentation, laboratory, wiki, and gold criteria. Ideas were then flushed out by a group discussion, resulting in modifications to ideas’ category locations. This part served as the brainstorming and organization of Allen’s method.
Then, the team identified next actions for the project based off of the brainstormed ideas. Next actions are described as steps taken by the team to complete a specific portion of the project, and Allen’s method says that a project is only considered fully developed when all portions of the project have their potential next actions assigned. Once next actions were issued, all members of the team started to research how the task could be completed.
During the research phase, Devon found out that the team’s idea was previously attempted using a different method by the 2008 Johns Hopkins iGEM team. The team then switched gears into finding out what the Johns Hopkins team accomplished by examining their wiki, presentation, and poster. After learning helpful information about the project, the team decided to stick with their original idea.
Around the same time, research on possible pigment pathways, fluorescent colors, and promoters occurred. Robert then found the next big breakthrough on the project when he found a positive feedback loop in yeast cells form the work of Ajo-Franklin, Drubin, Eskin, et al. in their paper titled Rational Design of Memory in Eukaryotic Cells. Further research showed that many teams have used a very similar pathway in previous years of the iGEM competition, solidifying the part as a key component of the project design.
The next step in the process was finding DNA sequences that correlated to each component of the scheme. At this time, a modification from the initial layout occurred. All references and teams previously used a two-tiered system for the positive feedback loop, found in Figure 1. The team proposed to combine the two tiers into one, found in Figure 2.
Figure 1: Example fluorescence pathway based off literature and previous projects.
Figure 2: Proposed fluorescence pathway for RHIT’s iGEM 2012 competition.
The switch was driven by three major factors: cost, ease, and simplification of the pathway. The cost of production decreased proportionally as the size of the sequence was reduced. Also, by having a shorter sequence, fewer ligations were necessary. Finally, by decreasing the number of components, the sequence was simplified.
During this discussion, some possible problems were brought up. The spacing of the STE 12 binding site compared to the position of LexA-Regulation  promoter was called into question as well as the concern of competition between STE12 and LexA once the process was activated. The spatial concerns were addressed when laying out the DNA sequence by modifying the space between the components. The team hypothesized that the inhibition, if it occurs, would not greatly affect the positive feedback loop, resulting in no action taken regarding this concern.
After all portions of the DNA sequences were assembled, Adam created a program in MATLAB that codon optimized the sequences for yeast. The sequences were then manually compared to their reference sources to ensure correct amino acids.
Molecular Maya Animation
Description of project
The goal of the Rose-Hulman iGEM team’s project was to devise a cellular circuit that would allow for the determination of the mating type of Saccharomyces cerevisiae. This mating type sensor was created by introducing a self-perpetuating fluorescent heteroprotein by means of a plasmid vector. The heteroprotein contained several distinct segments including an Ste12 responsive element, a LexA-reg element, two separate fluorescent domains, a LexA binding domain, a VP64 activator domain, a nuclear localization sequence, and a terminator, as illustrated below. This protein was contained on a HIS3 plasmid.
Figure 1: Designed construct.
Ste12 is a transcription factor that is activated as part of the pheromone response pathway. The initial production of the heteroprotein is controlled by the Ste12 responsive element, taken from the Fus1 gene. Once the protein is produced, the LexA binding domain binds to the LexA-regulatory element, and in conjunction with the VP64 activator domain, facilitates the further production of the heteroprotein in the form of a positive feedback loop.
In addition to the planning described in the project planning section, this circuit was rationally designed with several potential problems in mind. The first problem that often arises in similar circuits is the “leaky” nature of some promoters. Extensive research was performed on the various proteins involved in the yeast pheromone response pathway in order to choose one that is tightly regulated, and is only activated in presence of mating factor. Furthermore, the use of a non-native activator as the predominant control mechanism reduces the probability of interactions between other similar transcription factors.
As a secondary application of the project, two copies of this construct were implemented, each containing a different fluorescent protein. One of these constructs would eventually be integrated into the genome of one mating type of yeast. The purpose of this would be to allow for easy identification of the mating type of the unknown strain.
Synthetic Biology
Synthetic biology combines DNA sequences discovered in nature and synthetic DNA sequences designed in the laboratory (parts) to produce new functions in living cells (machines). Different types of regulatory and protein-encoding parts are used to engineer useful machines. This approach is being applied to produce various things, including insulin from bacteria and biofuels from algae. It can also be used to address many of the world's large problems, from hunger to disease epidemics and alternative energy.
Yeast background
Yeast is a single-celled eukaryote, which means that it shares many properties with cells of multi-cellular organisms, including humans. It is commonly used for laboratory research and commercial applications. Yeast can exist as diploid cells, which have two copies of each chromosome, like most animal cells, or as haploid cells, which have only one copy of each chromosome.
This cell is diploid, as it has two copies of each chromosome.
This cell is haploid, as it has one copy of each chromosome.
Haploid cells also are one of two mating types, MATa or MATalpha. They are, therefore, similar in these respects to animal eggs and sperm. Diploid yeast cells are produced when haploid cells of opposite mating type sense one another and fuse together. What each haploid senses is mating pheromone produced by cells of the opposite mating type. MATa cells produce a-pheromone that binds a-receptors on MATalpha cells, while MATalpha cells produce alpha-pheromone, which binds alpha-receptors on MATa cells. This "cross-signalling" activates the mating pheromone response pathway (MPRP), leading to fusion of the two haploids and formation of a diploid cell.
The Checkmate Project
Yeast researchers must often determine the mating type of yeast haploids. This is a tedious and time-consuming task, which can take at least 40 hours. To streamline the process, we designed a cellular system called Checkmate, which produces a colorful protein in response to pheromone secreted by cells of the opposite mating type. We hope it will simplify the process and reduce the time necessary to identify a haploid's mating type. The system uses a genetic circuit that is turned on when the MPRP of a cell is activated. Once turned on, a positive feedback mechanism will maintain production of the colorful protein, even after the MPRP shuts off. Typical mating type testing will be done by mixing Checkmate mating type detector cells with unknown haploids and examining them for a colorful response. Such a response indicates that the mating type of the unknown is opposite that of the Checkmate detector used in the test.
Figure 1: Mating Type Test Comparison. The Checkmate test takes about four hours, while traditional tests requiring diploid selection take about 40 hours.
Retrieved from " http://2012.igem.org/Team:RHIT/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
