<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BYU_Provo_Notebook_Biofilm_septoct"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BYU_Provo/Notebook/Biofilm/septoct">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BYU_Provo/Notebook/Biofilm/septoct&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/septoct&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/septoct&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BYU_Provo/Notebook/Biofilm/septoct" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BYU Provo/Notebook/Biofilm/septoct</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1 style="color:#FFFFFF">BYU 2014 Notebook </H1><P style="color:#FFFFFF"><A href="https://2014.igem.org/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/septoct&amp;action=edit" style="color:#FFFFFF">Edit September October</A></P><TABLE width="100%"><TBODY><TR heigth="30px"><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo" style="color:#002255">Home </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Team" style="color:#002255"> Team </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=BYU_Provo" style="color:#002255"> Official Team Profile </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Project" style="color:#002255"> Project</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Parts" style="color:#002255"> Parts</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Modeling" style="color:#002255"> Modeling</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Notebook" style="color:#002255"> Notebook</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Safety" style=" color:#002255"> Safety </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Attributions" style="color:#002255"> Attributions </A></TD></TR></TBODY></TABLE><BLOCKQUOTE><H2>2 September 2014</H2><P>Alpha Amylase: Based off the sequencing results from the latest round of colony PCR of alpha amylase, it looks like colony 3 contained a mutated PstI restriction site. From the streak plate and overnight was started. (JB)</P><H2>3 September 2014</H2><P>Alpha Amylase: Performed a plasmid prep of the colony 3 mutant following the SpinSmart High-copy plasmid purification. I took the prep and measured the plasmid concentration by nano drop and it only read around 20 ng/uL. This is definitely too low to get any sequencing from. I started a new overnight from the colony 3 plate streak. (JB)</P><P>Aii: I obtained colonies on all of the plates that I used to transform the Aii and TolB into E. coli, which I hope is a good sign. My next step in the process is to do colony, which I finished today. I used the plasmid specific primers.(CZ)</P><P> Dispersin: Today I did the spinsmart plasmid purification protocol on the on the cells that were overnighted last Wednesday and Pelleted on Thursday. (JM) </P><H2>4 September 2014</H2><P>Alpha Amylase: Performed a plasmid prep of the colony 3 mutant again. This time I got a much larger pellet of cells from the overnight. The concentration was around 60 ng/uL however. Not sure why I have been getting such low plasmid concentrations lately. (JB)</P><H2>5 September 2014</H2><P>Aii: Today I am running my PCR products from Wednesday out on a standard agarose gel. I have a long series of strong bands at the correct size, so I will be purifying two of those colonies and using them for sequencing. (CZ)</P><P>Alpha Amylase: Today I started four overnights from the alpha amylase mutant colony 3 streak. Hopefully we will be able to obtain a sufficient concentration of the plasmid since the last few rounds of plasmid prep have yielded unusually low concentrations. (JB)</P><P>Dispersin: I took my purfied plasmid of pET15b and PIG97 and set up a restriction enzyme digest using the EcoR1 and  Pst1 restriction sites.(JM)</P><H2>6 September 2014</H2><P>Alpha Amylase: Today I combined the four overnights down and pelleted them. (JB)</P><H2>8 September 2014</H2><P>Alpha Amylase: Today I performed a plasmid prep following the SpinSmart High-copy plasmid purification protocol and materials. Despite the four combined liquid cultures I was only able to get a concentration of 114 ng/uL. This should be sufficient enough though for sequencing. I prepped for sequencing of the mutant colony using the forward and reverse pSB1C3 primers and the new internal primer that was created a few weeks ago. We can also start assaying the amylase in pet15b to determine the pre-mutant efficiency of the protein and later assay the mutant amylase to be sure that the mutation does not decrease the efficacy of the protein. I tested the pH of the biofilm samples to see if we would need to add buffer to them so that the protein would not denature if the pH was hostile. Luckily, the pH range for all of the samples was in the 7.1-7.5 range so we will not need to add any buffer when we start sequencing. (JB)</P><P> Dispersin: After digesting I quickly took 5 microliters from each and ran them out on a gel. I was pleased to find bands of the appropriate size for the PIG97 plasmid with the cut out Dispersin B gene as well as a large band for the pET15b plasmid. So I used the full reaction mixtures and ran them out on a low melt gel. Unfortunately they didn't show up under a low power UV light so I needed to use the high power imager in order to remove the appropriate sections from the gel. I then set up a ligation reaction. (JM) </P><P>Aii: Today was mostly spent in growing up overnights of my bacteria from two colonies, Plate 2, numbers 1 and 2. These were the two brightest bands on the gel, and were of the correct size. I also worked on the fundraising for our tickets. (CZ)</P><H2>9 September 2014</H2><P>Aii: My colonies had grown up fairly well after 24 hours. I spun them down, and also did some work with Jared's Dispersin overnights and plates.</P><H2>10 September</H2><P>Alpha Amylase: Today a majority of the time was spent up in Salt Lake City at the University of Utah retrieving <EM>N. multiformis</EM> from Ananda Shankar, a graduate student at the university. He was very generous and gave us a culture that he had been growing for the past couple of months, frozen stock, and very specific instructions on how to make the media needed for the <EM>N. multiform is</EM> to grow.</P><P>Sequencing data also came back today for the amylase mutant plasmid prep. (JB)</P><P> Dispersin: Today I transformed DH5Alpha cells with 7 microliters of my ligation mixture. Given that the bands were difficult to see and cut out however, I also began 4 overnights (two for each) of the PIG97 and pET15 colonies in case the slices I cut from the gel do not have the DNA concentrations necessary to have a successful ligation and subsequent transformation. Lastly I used the DNA gel extraction kit on my gel slices so that if my transformation is unsuccessful I can check and see if I have the necessary DNA by running 5 microliters out on Friday. (JM) </P><P>Aii: Today I purified the 2 overnights that I did on Monday and spun down yesterday. I used the SpinSmart High-copy plasmid purification protocol, and then Nanodropped the results. It turns out that in both tubes there was a very low count of DNA (approximately 20 ng/uL). Nonetheless I submitted both tubes for sequencing.(CZ)</P><H2>11 September</H2><P>Alpha Amylase: Today I spent some more time reviewing the sequencing results. It has been a bit difficult going through and piecing everything together to see if there are or are not any additional mutations to the alpha amylase gene. The coverage was very good and very accurate, but there was a section of the internal primer sequencing that was a bit spotty. There were extra base pairs sometimes, a missing base pair others, or a base pair that could not be read. But the mutation is there! (JB)</P><H2>12 September 2014</H2><P>Alpha Amylase: Since there was the spotty section of sequencing where none of the primers picked up the section I prepped two more instances for sequencing, one using the amylase specific forward primer and the other using the internal primer again. (JB)</P><P>Dispersin: Unfortunately my ampicilin plate didn't have any colony growth which means that the DH5Alpha cells likely did not pick up the pET15b plasmid which has amp resistance. Which means my ligation probably didn't work. After running out 5 microliters of my extracted DNA from Wednesday I was unable to see any DNA at all, which confirmed my suspicions that my ligation was unsuccessful. It seems I didn't extract it properly from the low melt on Monday. So today I again ran the spinsmart plasmid purification procedure on the overnights. Monday morning I will digest them and in the afternoon run them out on lowmelt. Hopefully this time by doubling the colonies contributing the plasmid the concentration will be high enough to not run into the same difficulties purifying the vector and insert for ligation. (JM) </P><P>Aii: I redid my overnights for the same colonies. We believe that our LB+CAM has broken down, but I had colonies this time that will hopefully have a higher expression rate. (CZ)</P><H2>15 September 2014</H2><P>Alpha Amylase: Sequencing came back and all the rounds of sequencing combined show a complete amylase with signaling sequence for <EM>multiformis</EM>, a mutated, no longer functioning PstI site, and no additional mutations! This means we can start assaying amylase pre- and post-mutation of the PstI restriction site!</P><P>The rest of my time was spent preparing the growth media for <EM>multiformis</EM> following Ananda's protocol he shared with us. There were some issues with the pH however. It was supposed to be 7.2-7.4. It was reading around 7.8 and when I added the Na<SUB>2</SUB>CO<SUB>3</SUB> it made the pH even more basic. Will need to discuss what to do about this. (JB)</P><P>Aii: I got my sequencing results back, but it doesn't look like anything turned up, which is worrisome. There a few explanations for that result. There could have been a problem with our LB+CAM broth. Our incubator has also not been working at the correct temperature. Lastly, the plasmid prep kid is old, and we think there may be problems with that as well. However, I grew up new overnights to try to fix the problem. (CZ)</P><P>Dispersin: I was sick today but I called in to have Cam digest my plasmids but when he tested them with a nano-dropper they had a minimal amount of DNA too low for any work to be done. (JM) </P><H2>16 September 2014</H2><P>Alpha Amylase: Today not much was done for amylase as I spent most of the time trying to fine-tune the pH of the <EM>multiformis</EM> media. I did so, as well as pellet down the old cells that Ananda gave us and transfer it to this new media. Also, as <EM>multiformis</EM> is a very sensitive bacteria and likes to flock together it is important not to re-suspend repeatedly or with a lot of force. I made this mistake, but luckily you could still see clusters of the bacteria, so hopefully it will not upset them too much. (JB) </P><BLOCKQUOTE>Left: <EM>multiformis</EM> media</BLOCKQUOTE><BLOCKQUOTE>Right: Pelleted <EM>multiformis</EM></BLOCKQUOTE><H2>17 September 2014</H2><P>Alpha Amylase: We need to move alpha amylase into the conjugation plasmid pIG111 in order to be able to test it in <EM>N. multiformis</EM>. However, we only have around 48 uL of purified, mutated amylase plasmid and we need to have enough for the transformation into pIG111, for the iGem registry, and for our own personal stock. To address this I transformed 5 uL of the purified, mutated plasmid into DH5Î±. After, I will be able to have a greater stock of purified plasmid and will be able to meet all the requirements for amount of plasmid needed for all these assignments. (JB) </P><P>Aii: Today was mostly spent on fundraising things. I pulled my overnights out of the incubator yesterday, but when I did the plasmid prep, we didn't get a much higher cell count (it was 8 ng/uL before, and 40 ng/uL now). This time I'm going to make three tubes of bacteria and purify them on one filter to get a higher count. (CZ)</P><P>Dispersin: Due to the failure of my plasmid preps, I again set up overnight Colonies, because I cant do anything until those cultures grow I also helped Cam with his overnights.

</P><H2>18 September 2014</H2><P>Alpha Amylase: The transformed colonies grew up very well last night. I started two overnights in 5 uL of LB + cam for a plasmid prep tomorrow and digest and ligation into pIG111.</P><P>Additionally, things are looking a little brighter for our <EM>N. multiformis</EM> we have growing. Yesterday it looked like the celled that had been pelleted and transferred to fresh media we clumping up and not looking like the small, white flocks we saw when we picked it up from Ananda. However, today they had separated out more and were forming more of those white flocks Ananda instructed us to be looking for. As well, the frozen culture I started yesterday is looking good. There were one or two small flocks that appeared to have formed since yesterday when I started this batch. (JB)</P><P>Aii and Dispersin: I did another plasmid prep for both my three test tubes of Aii, as well as Jared's pET15b and pIG 97. We are hoping that the plasmid prep kit is still working correctly. (CZ)</P><H2>19 September 2014</H2><P>Alpha Amylase: The plasmid prep turned out alright. We made new LB + cam, so maybe we are still getting low yields due to something in the kit. From the two overnights I was able to get a concentration of 132.7 ng/uL. This is good enough so I went ahead and used all of it for a restriction digest so I can ligate into the pIG111 conjugation plasmid for testing in <EM>multiformis</EM>. I followed the regular protocol for this, however, I substituted the 14 uL of ddH<SUB>2</SUB>O in the insert protocol for 14 uL of alpha amylase. There was only ~30 uL of the pIG111 so I used the normal recipe for that procedure. The old plasmid prep of the amylase mutant will be used for personal stock and for sending to the iGem parts database. Also, there are several far more significant flocks in the freezer stock of <EM>multiformis</EM> and it is appearing more and more like the batch Ananda gave us that had been growing for a while. (JB) </P><P>Dispersin: I ran a nano-dropper test on the purified plasmid Cam prepped for my from my overnights on Thursday unfortunately he placed each overnight in a separate binding column. So my pET15b plasmid concentration is low in both tubes, only 18ng/ul and 14ng/ul. I will just have to fill a larger low melt well with as much as I can from both solutions and hope the end concentration I get from cutting out the band will be enough. 
I also ran a PCR reaction using the B1307 and B1308 primers which target 200 bp up and downstream of the insert site of the PIG97 plasmid using some of my purified PIG97 plasmid so that I will have the Dispersin B gene along with all the standard igem restriction sites to work with in the future should I need it. (JM) </P><H2>20 September 2014</H2><P>Alpha Amylase: Ran the the restricted pIG111 and mutated alpha amylase with signaling sequence on low melt gel. The alpha amylase came our really strong, but the pIG111 was far fainter. I combined the two for a ligation reaction and will leave it overnight. (JB)</P><H2>21 September 2014</H2><P>Alpha Amylase: Performed a transformation into DH5Î± with 3 uL of the ligation (pIG111 + mutated alpha amylase w signaling sequence) and plated it on a chloramphenicol LB plate. (JB)</P><H2>22-29 September 2014</H2><P>I'll just do a synopsis of what I did, since my journal didn't save. Most of this week was spent on re-doing my transformation and resequencing. Most of my hours, however, were spent coordinating the fundraising and making calls to our friends and family about having us stay with them. I submitted new plasmid for sequencing at a much higher count (150 ng/uL), and I'm hoping that it works this time. (CZ)</P><H2>22 September 2014</H2><P>Alpha Amylase: I missed the fact that the pIG111 plasmid has ampicillin resistance, not resistance to chloramphenicol. I redid the transformation using the ligation piece and plated it on the correct type of plate.</P><P>I also looked at the <EM>multiformis</EM> we have growing (both what Ananda gave us that had been growing previously and from the frozen stock) under the microscope. It was far more difficult to find anything from the old batch (but that may have been due to just not taking sample from an area that had the flocks), but in the new batch you could see many cells that appeared to have somewhat of a kidney bean shape as Ananda described it, and they appeared to be flocculating together as we would expect to see from what we know of <EM>multiformis'</EM> growth patterns. Below are picture taken at x40 magnification. (JB)</P><P>Dispersin B: I ran a restriction digest on the 14ng/ul and 18ng/ul pET15b plasmid preps, as well as on the PIG97 plasmid prep. I used EcoR1 and the Pst1 restriction enzymes. After waiting 1.5 hours I filled a single low melt gel well with both the digested pET15b preps and another well with the digested pIG97. I was able to see bands of the appropriate size I(~1100bp for Dispersin B and ~3000+bp for pET15b) for both lanes under a low intensity UV light so I cut them out of the gel. (JM)</P><H2>23 September 2014</H2><P>Alpha Amylase: The transformations from last night did not work so we will have to skip the conjugation of amylase into <EM>multiformis</EM>. But we have plenty to do with the assay of amylase pre- and post-mutation so I will start that tomorrow. (JB)</P><H2>24 September 2014</H2><P>Alpha Amylase: Since no one else has been able to transform their pieces into pIG111, the conjugation plasmid, we needed to make more conjugation plasmid. What was left was a really low concentration (which means what had been used was very likely also low). I transformed the plasmid into DH5Î± and plated it on an ampicillin resistant plate and put the rest of the culture in 5 mL of LB + ampicillin to let it grow overnight. I still have my insert mut alpha amylase from the low melt gel cut, so once we have a good concentration of pIG111 plasmid and run it out on low melt gel, we will all be able to use that as vector and ligate our inserts into the conjugation plasmid.</P><P>I also checked the pHs of the media in which we have our old and new <EM>multiformis</EM> cultures growing up and they both appear to still be in the &gt;7 range, so that is good (however the media in which the new stock was in was getting close to 7, so I will recheck the pH this weekend and then later next week to make sure it stays in the &gt;7 pH range. But we have lots of cells growing it appears, so that is really good news! (JB)</P><P>Dispersin B: Today I ran a ligation reaction with my two gel slices and then transformed the ligation mixture into DH5alpha cells which were plated on plates with Ampicillin, the antibiotic for which resistance is conferred by the pET15b plasmid. (JM)</P><H2>25 September 2014</H2><P>Alpha Amylase: More issues with the pIG111 conjugation plasmid. I checked the temperature of the 35<SUP>o</SUP>C incubator that the liquid culture was in and it was only reading 32<SUP>o</SUP>C. I took it upstairs to Dr. Grose's other incubator and let it shake in there for the afternoon. Then I did the plasmid prep, followed all the instructions exactly, and nano-dropped the plasmid and it only showed a concentration of 24.9 ng/uL with a 260/280 reading of 1.92 and a 260/230 reading of 0.98 (indicating a contaminant that absorbs light at 230 nm wavelength). I did some sleuthing and have some theories. Yesterday we through out some old lysis buffer, so it shouldn't be that. But I talked to Julie and she mentioned that there were suspected issues with the resuspension buffer and so she had used another batch of it and had a good plasmid prep, but we have consistently had poor yields with the plasmid prep kit we have been using as of late. But then today she used the resuspension buffer and it gave good readings. The only things left I can think of are either the conjugation plasmid is a low copy plasmid or there is another ingredient that is having issues. Julie and I are pretty sure we used different wash buffers (we have two in the kit) and that is where the issue lies. I will talk to Dr. Grose and see if she has any further insight. (JB)</P><H2>26 September 2014</H2><P>Alpha Amylase: Dr. Grose said that sometimes if there is not enough ethanol in the wash buffer that the plasmid will not stick to the filter during the wash step. I used the kit upstairs and got a good concentration (96 ng/uL), but still low purity readings on the 260/230 reading. Dr. Grose said that might be due to not re-suspending all the cells or not lysing all the cells. I performed the restriction digest using the Xba and Eco restriction enzymes, ran it on low melt gel, and cut it out and aliquoted it out for those who will need it. The plasmid gave a really strong band under low powered UV, which is a good sign. (JB) </P><P>Dispersin B: I had Cameron pull out my plates yesterday, but I came in to check them. I have colony growth, which is exciting because it means that my restriction digest and ligation probably worked.(JM)</P><H2>27 September 2014</H2><P>Alpha Amylase: Today I performed the ligation of the alpha amylase mutant insert with the pIG111 vector. I will leave it overnight and perform the transformation into E. coli tomorrow. I also checked the pHs of the <EM>multiformis</EM> cultures and they are both in the basic range still, so that is a good sign, they still have plenty of NaCO<SUB>3</SUB> to continue their growth. (JB)</P><H2>29 September 2014</H2><P>Alpha Amylase: I transformed the ligation into DH5Î± and plated it on a Ampicillin plate. Tomorrow I will pick off a successful colony and start an overnight of it so it will be ready for conjugation on Wednesday! I also looked into last year's iGem team page to look at how they assayed amylase and the other biofilm degraders and worked on a write-up so we can start testing our constructs and their efficacy. (JB)</P><P>Dispersin B: I selected six colonies and labeled them in tubs A-F and dipped each individually into 50uL of ddH2O and then streaked each into a labled segment on a plate with Ampicillin. I then boiled the tubes to extract the plasmid and performed colony PCR using taq polymerase and primers for Dispersin B to check if the gene was there.(JM) </P><P>Aii: Most of today was spent helping other people. I helped Cameron for a while, and also worked on the collaboration project (CZ)</P><H2>30 September 2014</H2><P>Alpha Amylase: There were five colonies total that grew up from the transformation. I picked on and suspended it in LB + Amp and left in the 37<SUP>o</SUP> shaker overnight so that we can use it in the conjugation tomorrow. (JB)</P><H2>1 October 2014</H2><P>Alpha Amylase: I forgot to run colony PCR of the colonies because just because they show up as white does not mean that they successful cloned in the gene of interest, but that the RFP has been cut out. I ran TAQ PCR on the five colonies and ran their products out on gel to see if the mutated amylase was successfully cloned in. I also worked on writing up the protocol for assaying the efficacy of amylase at preventing biofilm build-up and breaking down pre-existing biofilm. (JB) </P><P>Dispersin B: Unfortunately after running the results out on gel it looks like the colonies I picked didn't have any primer. Additionally, only one colony of the six grew on the plate. So I selected, 6 new colonies and repeated the Colony PCR proceedure I followed on Monday. Only this time I added a lane as a positive control using purified PIG97 plasmid. (JM) </P><P>I am still doing the collaboration project. They seem to have numbered everything incorrectly, and their project description is strange. I worked on overnights and plating everything out. (CZ)</P><H2>2 October 2014</H2><P>Alpha Amylase: There was an issue with the gel last night and it did not turn out very well. I remade a gel and ran the PCR products on the new gel. There were still issues however. I had done two rounds of PCR and on the first nothing showed up, and the second had contamination it looked like, as even the control was showing smearing. So it does not look like the ligation and transformation of mutant amylase into the conjugation plasmid did not work. So it does not look like amylase will be ready for conjugation into <EM>multiformis</EM>.</P><P>But, in more optimistic news, the assay protocol for assaying amylase and dispersin in about ready! I sent in a draft to Dr. Grose and she said it looks good, just need to make final edits and we will be ready to begin testing next week! (JB)</P><P>Aii: I got my sequencing back. I will check that later, but I did plasmid preps for the 2 types of plasmid that grew up. The other two are not working correctly.</P><BLOCKQUOTE><P>Gel order on first PCR is control 1, colony 1, control 2, colonies 2-5, then Jared's dispersin colony PCR.</P><P>Gel order on second PCR is control 1, colony 1, control 2, colonies 2-5.</P></BLOCKQUOTE><H2>3 October 2014</H2><P> Dispersin B: Again, after running the PCR tubes out on gel, there was no indication that the plasmid containing Dispersin B was in the bacteria colonies. My positive control did show me a band of the appropriate size so I can safely assume that if the bacteria had the plasmid it would show the bands. I suspect that the plates I used to originally plate the bacteria may not have had an appropriate concentration of Ampicillin, and that the colonies I've been sampling have either kicked the plasmid out, or they never had it in the first place. Under that suspicion, I used my ligation mixture and retransformed and plated a new set of DH5 Alpha cells. (JM) </P><H2>6 October 2014</H2><P>Alpha Amylase: We got back Dr. Grose's suggestions she provided concerning the biofilm assay procedure and made the necessary corrections. We posted the procedure on the &quot;Modeling&quot; page of our wiki and started the procedure as outlined using purified amylase from last year's iGem team. </P><P>Below is an image of the protein (amylase) we are using from last year's team on gel. The order is ladder, elution 1, elution 2, and elution 3. We used elution 3 in the assay we started today. (JB)</P><P>Collaboration Project: Most of my time today was spent on the collaboration project. Part of the problem is that our purification kit wasn't working, so I needed to redo most of the experiment with the new kit. (CZ)</P><P>Dispersin B: There was absolutely no growth on my new plates. I suspect that my ligation didn't work and that the low concentration of pET15b plasmid that I was using was the issue. So, I took a plasmid prep made by Cameron of pET15b on the 23rd of September and ran a restriction digest using EcoRI and PstI restriction enzymes after testing it with the nano-dropper and finding that it had a satisfactory concentration of 69 ng/ul. I also aided Jordan in setting up the biofilm degradation assay. (JM) </P><H2>7 October 2014</H2><P>Alpha Amylase: I checked the first batch of the assay and it doesnât really look like anything is happening, the biofilm comes out just as easy in all of the tubes. I am wondering if the biofilm doesnât really attach very well to the tubes - may just be the biofilm and how it acts. I had an idea though, maybe we could start all the samples up and keep them in cuvettes, that way we can take several readings and do pre and post amylase readings and see how the absorbance changes. I would assume that as the amylase breaks up the biofilm the absorbance would decrease. We will have to discuss this over with Dr. Grose and Desi tomorrow and see what they think. 
</P><P>Also, I checked the pH of the media the <EM>multiformis</EM> is growing in and they are still both in the basic range. We will need to transfer the cells into new media though in the next week or two. (JB)</P><H2>8 October 2014</H2><P>Alpha Amylase: We discussed the assay with Dr. Grose and Desi. Likely, the biofilm needs more time in order to attach to the sides of the tube. That, or the biofilm is unable to attach to that type of plastic. We will have to modify the procedure once we figure out what is best for this assay. We will leave the one round of the assay continue to run and we are using the tubes that just had biofilm to check when has been long enough for the biofilm to adhere. We are also testing biofilm in 2 other types of 96-well plates as well as on glass slides so see if there is a better material that will allow the biofilm to adhere to it. We also prepped amylase for submission! Finally, it will be on its way! (JB)</P><P>I am still working on the collaboration. We did a low-melt gel that didn't display strong bands, so we nanodropped the purifications. They were at around 50 ng/uL each, so they didn't work. I picked new colonies and will try to regrow them and see if we can get higher expression. (CZ)</P><P> Dispersin B: I ran my digest on low-melt gel and cut out the band corresponding to the pET15b plasmid. Jordan and I also worked on the wiki page, as well as refined our Biofilm assay with the help of Dr. Grose. (JM)</P><H2> 10 October 2014 </H2><P> Dispersin B: Today using the Dispersin B purified insert made last week and my newly cut slice of gel containing the pET15b plasmdid I ran a ligation reaction. I then used a batch of DH5Alpha cells and transformed them with my new ligation mix. After giving the cells time to recover I plated them on a plate with ampicillin resistance. Cam said he would take the plate out for me tomorrow.(JM) </P><H2>13 October 2014</H2><P>Alpha Amylase: The assay worked! Both the controls worked, all the biofilm adhered to the tip of the tube after doing the double inverted tap, while the tubes with biofilm and 12.5 uL of amylase had some of the biofilm fall out when tapped. Below are pictures and tables of the results.</P><P>We started a new round of assays, this time using purified Dispersin B as well and we did the 12.5 uL tests of the enzymes as well as tests with 25 uL of the enzymes for better visualization of the biofilm dispersion. Below is a table of the new layout of the testing. </P><P>Please refer to <A href="https://2014.igem.org/Team:BYU_Provo/Modeling">Modeling page</A> for results and write-up of revised assay for the round of testing we started today.(JB) </P><P>Dispersin B: There was growth on my plate! This time the colonies appear the right size, better than the smaller satellite growths I got from the old plates I attempted to use earlier, so hopefully this time it worked. (JM)</P><H2>17 October 2014</H2><P>Alpha Amylase/DispersinB: Checked on the results of the biofilm assay, results look promising! Please refer to modeling page for data. (JB)</P></BLOCKQUOTE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BYU_Provo/Notebook/Biofilm/septoct">http://2014.igem.org/Team:BYU_Provo/Notebook/Biofilm/septoct</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BYU_Provo/Notebook/Biofilm/septoct" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BYU_Provo/Notebook/Biofilm/septoct" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/septoct&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/septoct&amp;oldid=364337" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>