"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Project
From 2014.igem.org
Project: Reclaiming Wastewater Reclamation
Click here  to edit this page!
Home
Team
Official Team Profile
Project
Parts
Modeling
Notebook
Safety
Attributions
Overall Project Description
We are working to optimize the wastewater treatment process. Currently we are addressing some of the difficulties faced by the working microbial community in the bioreactor. These include: the buildup of biofilm, the destruction of the "working class" bacteria by phage, antibiotic discharge into the wastewater, and nitrate production. We will be building our machine in the bioreactor bacterium N. multiformis and optimizing it to address the aforementioned obstacles.
Project: Antibiotic Breakdown
Project Background:
In the current sewage treatment processes antibiotics present a serious problem to the downstream environment. Many of these and other similar chemical compounds, because they are not regulated or easily degraded, are being released into the environment at a growing rate. We identified the most commonly prescribed antibiotics and chose to address the breakdown of such antibiotics. We identified amoxicillin based and erythromycin based antibiotics among the most prevalent. The enzymes beta-lactamase and erethryomycin esterase B were identified as candidates for the breakdown of our antibiotics. We used the EreB gene from the iGEM parts database, and amplified the Bla gene from the iGEM plasmid pSB1A3.
Antibiotic Breakdown Team:
Bri Keele and Julie Roberts
References:
http://www.chrt.org/publications/price-of-care/issue-brief-2011-02-antibiotic-prescribing-and-use/
Project: Biofilm Annihilation
Project Background:
Biofilm production by bacteria is a major concern in activated sludge processors (ASPs). The buildup of these bacteria in these processors inhibits the helpful bacteria from being able to effectively break down several components. Of main concern are the biofilms of bacteria such as Nocardia spp., Thiothrix spp., Sphaerotilus natans, and several others. In order to solve this issue we cloned the genes for Alpha Amylase and DispersinB, both which break down the polysaccharide matrix of biofilms, as well as the gene for AiiA, a quorum-sensor blocker, into Nitrosospira multiformis, one of the helpful bacteria in the activated sludge processors. Attached to these genes in their respective chassis is a signaling sequence which dictates the expression of these gene products extracellularly. With the expression of these genes outside the cells there should be a significant decrease in the amount of biofilm buildup by these biofilm-creating bacteria and bacteria such as N. multiformis will be less restricted in breaking down the interested components of ASPs.
AiiA, Alpha Amylase, DispersinB, A visual representation of biofilm inhibition and degradation.
Biofilm Team:
Cam Zenger, Jared McOmber, and Jordan Berg
The focus of the biofilm team was to insert the genes for Alpha Amylase, DispersinB, and AiiA with a signaling sequence for extracellular expression into the pSB1C3 chassis and to assay the efficacy of these genes in biofilm reduction in ASPs. Additionally, site-directed mutagenesis was performed on Alpha Amylase to remove the PstI restriction site found within the gene.
References:
Donelli G, Francolini I, Romoli D, Guaglianone E, Piozzi A, Ragunath C, Kaplan JB. 2007 Aug. Synergistic Activity of Dispersin B and Cefamandole Nafate in Inhibition of Staphylococcal Biofilm Growth on Polyurethanes. Antimicrobial Agents and Chemotherapy. [accessed 27 Mar 2014]; 51(8):2733–2740. http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1932551/pdf/1249-06.pdf.
Kalpana BJ, Aarthy S, Pandian SK. 2012 Feb. Antibiofilm Activity of α-Amylase from Bacillus subtilis S8-18 Against Biofilm Forming Human Bacterial Pathogens. Appl Biochem Biotechnol. [accessed 27 Mar 2014]; 167:1778–1794. http://download.springer.com/static/pdf/846/art%253A10.1007%252Fs12010-011-9526-2.pdf?auth66=1407375774_bcd3eaa94748c4bb0a1ec0553c81fa5c&ext=.pdf.
Brown-Elliott BA, Brown JM, Conville PS,Wallace RJ. 2006 Apr. Clinical and Laboratory Features of the Nocardia spp. Based on Current Molecular Taxonomy. Clin Microbial Rev. [accessed 14 Feb 2014]; 19(2):259-282. http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1471991/.
Fazekas E, Kandra L, Gyémánt G. 2012 Dec. Model for β-1,6-N-acetylglucosamine oligomer hydrolysis catalysed by DispersinB, a biofilm degrading enzyme. Carbohydr Res. [accessed 14 Feb 2014]; 363:7-13. http://www.ncbi.nlm.nih.gov/pubmed/23103508.
Kaplan JB, Ragunath C, Ramasubbu N, Fine DH. 2013. Detachment of Actinobacillus actinomycetemcomitans biofilm cells by an endogenous beta-hexosaminidase activity. J Bacteriol. [accessed 14 Feb 2014]; 185:4693-4698. http://www.uniprot.org/citations/12896987.
Project: Crispy CRISPR's
Project Background:
Nitrosospira multiformis, a key microbe in sewage treatment centers using the activated sludge process, suffers a high rate of bacteriophage induced lysis(7). Between the high titer of bacteriophage found in sewage, prophage incorporated into its genome, and its natural slow rate of growth, N. multiformis does not reach high concentrations in bio-reactors(7). To combat the prevalence of bacteriophage infection we have cloned the CRISPR 3 system found in Streptococcus thermophilus LMD-9 into N. multiformis(6). Using a novel spacer region we will specifically target three prophage that reside in N. multiformis' genome. The CRISPR 3 system has been shown to acquire additional phage spacers and will provide adaptive immunity to N. multiformis(1).
CRISPR Mediated Acquisition and Silencing of Phage DNA
https://upload.wikimedia.org/wikipedia/commons/5/5f/Crispr.png
CRISPR TEAM:
Garrett Jensen, Mike Abboud, Michael Linzey.
References:
Rimantas Sapranauskas, et. Al. The Streptococcus thermophilus CRISPR/Cas system provides immunity inEscherichia coliNucl. Acids Res. (2011) 39 (21): 9275-9282 first published online August 3, 2011doi:10.1093/nar/gkr606
Hongfan Chen, Jihoon Choi, and Scott Bailey. Cut Site Selection by the Two Nuclease Domains of the Cas9 RNA-guided EndonucleaseJ. Biol. Chem. jbc.M113.539726. First Published on March 14, 2014,doi:10.1074/jbc.M113.539726
Shah SA, Erdmann S, Mojica FJ, Garrett RA. Protospacer recognition motifs: Mixed identities and functional diversity. RNA Biology 2013; 10:891 - 899; PMID: 23403393; http://dx.doi.org/10.4161/rna.23764
"Addgene: Addgene's CRISPR Guide." Addgene: Addgene's CRISPR Guide. Web. 8 Apr. 2014.
Krzysztof Chylinski, Kira S. Makarova, Emmanuelle Charpentier,and Eugene V. Koonin. Classification and evolution of type II CRISPR-Cas systemsNucl. Acids Res. first published online April 11, 2014 doi:10.1093/nar/gku241
"Streptococcus Thermophilus LMD-9, Complete Genome." National Center for Biotechnology Information. U.S. National Library of Medicine, 24 Oct. 2006. Web. 8 Apr. 2014.
.
Choi, Jeongdong, Shireen M. Kotay, and Ramesh Goel. "Various Physico-chemical Stress Factors Cause Prophage Induction in Nitrosospira Multiformis 25196- an Ammonia Oxidizing Bacteria." Science Direct. Water Research, 4 Aug. 2010. Web. 1 Feb. 2014.
.
Project: Denitrification
Project Background:
Nitrification is the oxidative pathway that converts ammonia to nitrite and then nitrate. This process occurs naturally in sewage bioreactors due to ammonia-oxidizing bacteria (AOB) like N. multiformis that thrive in regions of anaerobic conditions. Since ammonia is toxic to many aquatic and terrestrial lifeforms, AOBs play a critical role in cleaning wastewater by effectively removing the majority of the ammonia that enters the treatment facility. According to chemical analysis performed on the influent and effluent at the Silver Creek Water Reclamation Facility in Park City, Utah, ammonia enters the facility at a concentration of 34.0 mg/l and leaves at a concentration of 0.50 mg/l. These values display that current practices in sewage treatment are quite effective in ammonia removal, cleaning out about 99% of the ammonia deposited and produced in sewage. At the same time though, nitrate, the end product of ammonia oxidation, is released from these facilities at high concentrations. Although not as toxic as ammonia, nitrate also has harmful impacts on the environment including the acidification of waterways, unbalanced expansion of primary producers in the ecosystem causing eutrophication (see photograph below), and even growth and developmental challenges for several organisms. The chemical analysis of the effluent from the Silver Creek Water Reclamation Facility revealed that nitrate and nitrite, a less prevalent but even more toxic compound, entered the environment at an average concentration of 23.9 mg/l. This concentration of nitrate over doubles the limit set by the United States Environmental Protection Agency for safe drinking water (10 mg/l) and is sufficient to retard the development of multiple species of freshwater fish, illustrating just how harmful this water actually is as it first enters the waterways.
To prevent eutrophication downstream from water treatment facilities, we have designed a plan to engineer N. multiformis to also perform denitrification, the process of converting nitrate into nitrogen gas, as a means of removing nitrogen without negative effects on the environment. According to scientific literature (see below), denitrification can be accomplished by the proteins coded by the nirS, norB, norC, and nosZ genes from Pseudomonas aeruginosa PAO1. These genes convert nitrite (NO2-) to nitric oxide (NO), then to nitrous oxide (N2O), and finally to nitrogen gas (N2). The nitrification pathway in N. multiformis converts ammonia (NH3) to nitrite (NO2-) and then to nitrate (NO3-), meaning that the denitrification pathway begins with an intermediate of the nitrification process; a diagram of both pathways can be seen below. The enzyme in N. multiformis that converts nitrite to nitrate in nitrification performs a reversible reaction, so Le Chatelier's principle suggests that the denitrification pathway will work because equilibrium will drive the reaction to produce nitrogen gas. Knocking out the nitrate reductase in N. multiformis will also ensure that our chassis is forced to proceed through the denitrification pathway instead of producing nitrate.
Denitrification Team:
Cameron Sargent and Julie Roberts
References:
J. A. Camargo, A. Alonso, Ecological and toxicological effects of inorganic nitrogen pollution in aquatic ecosystems: A global assessment. Environment international 32, 831 (Aug, 2006).
United States Environmental Protection Agency, Basic Information about Nitrate in Drinking Water. Water: Basic Information about Regulated Drinking Water Contaminants (Feb, 2014).
M. D. McGurk, F. Landry, A. Tang, C. C. Hanks, Acute and chronic toxicity of nitrate to early life stages of lake trout (Salvelinus namaycush) and lake whitefish (Coregonus clupeaformis). Environmental toxicology and chemistry / SETAC 25, 2187 (Aug, 2006).
Z. Chen et al., Differentiated response of denitrifying communities to fertilization regime in paddy soil. Microbial ecology 63, 446 (Feb, 2012).
H. Arai, Regulation and Function of Versatile Aerobic and Anaerobic Respiratory Metabolism in Pseudomonas aeruginosa. Frontiers in microbiology 2, 103 (2011).
V. Kathiravan, Pseudomonas aeruginosa and Achromobacter sp.: nitrifying aerobic denitrifiers have a plasmid encoding for denitrifying functional genes. World journal of microbiology & biotechnology 30, 1187 (2014).
Project: Serine Auxotrophy
Project Background:
The auxotrophy project aims to create a safety mechanism that will provide a barrier between the proposed modified organism and the environment at large. As a precaution, we have engineered our chassis to be dependent on the environment found in the bioreactor, thus preventing it from escaping and spreading uncontrollably. We found that the amino acid serine is abundant in the water of the bioreactor and is not present in the effluent. We also found that the serA gene is essential to Nitrosospira multiformis production of serine. By removing the serA gene Nitrosospira multiformis will be unable to produce its own serine, which is essential for growth outside of the bioreactor.
Biofilm Team:
Christian Boekweg, Tanner Robinson, and Mark Murdock
The focus of the auxotrophy team was to knock out the gene responsible for the biosynthesis of serine. We found the serA gene to be crucial to Nitrosospira multiformis production of serine.
References:
Kahn, L., and Wayman, C. (1964). Amino Acids in Raw Sewage and Sewage Effluents. J. Water Pollut. Control Fed. 36, 1368–1371.
Norton, J.M., Klotz, M.G., Stein, L.Y., Arp, D.J., Bottomley, P.J., Chain, P.S.G., Hauser, L.J., Land, M.L., Larimer, F.W., Shin, M.W., et al. (2008). Complete Genome Sequence of Nitrosospira multiformis, an Ammonia-Oxidizing Bacterium from the Soil Environment. Appl. Environ. Microbiol. 74, 3559–3572.
Subrahmanyam, P.V.R., Sastry, C.A., Rao, A.V.S.P., and Pillai, S.C. (1960). Amino Acids in Sewage Sludges. J. Water Pollut. Control Fed. 32, 344–350.
Fishman, N. (2006) Antimicrobial stewardship. Am. J. Infect. Control 34, S55–63
Collaboration with Cornell University
We had the opportunity to collaborate with Cornell University on their project. They requested 50mL water samples from some of our nearby lakes and rivers. We collected a sample from the Provo River (shown below), which they used to test their heavy metal water filtration system. We are grateful for the chance to work with them.
Policy and Practice: Wastewater Treatment Plant Outreach
One of the first steps of our project was to visit the Silver Creek Water Reclamation Facility. This experience was integral in helping us understand the mechanisms of water reclamation. We owe a huge thank you to Gary Hill for giving us a tour of the facility, and for helping us to find ways in which we could help the process to become more efficient. As well, we were able to help those at the facility become more aware of more improvements of wastewater treatment of nitrates and antibiotics which we hoped to address with our engineered biological machine. They appreciated our ideas and the questions and concerns we were addressing. He also provided us with important data about the treatment plants for our experiments.
Retrieved from " http://2014.igem.org/Team:BYU_Provo/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
