<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BYU_Provo_Notebook_Biofilm_julyaug"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BYU_Provo/Notebook/Biofilm/julyaug">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BYU_Provo/Notebook/Biofilm/julyaug&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/julyaug&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/julyaug&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BYU_Provo/Notebook/Biofilm/julyaug" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BYU Provo/Notebook/Biofilm/julyaug</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1 style="color:#FFFFFF">BYU 2014 Notebook </H1><P style="color:#FFFFFF"><A href="https://2014.igem.org/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/julyaug&amp;action=edit" style="color:#FFFFFF"> Edit July August</A></P><TABLE width="100%"><TBODY><TR heigth="30px"><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo" style="color:#002255">Home </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Team" style="color:#002255"> Team </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=BYU_Provo" style="color:#002255"> Official Team Profile </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Project" style="color:#002255"> Project</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Parts" style="color:#002255"> Parts</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Modeling" style="color:#002255"> Modeling</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Notebook" style="color:#002255"> Notebook</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Safety" style=" color:#002255"> Safety </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Attributions" style="color:#002255"> Attributions </A></TD></TR></TBODY></TABLE><BLOCKQUOTE><H2>1 July 2014</H2><P>Alpha Amylase: The previous transformation gave us several hundred colonies. This will make it difficult to find one that has the mutated Alpha Amylase plasmid. We will need to re-transform the PCR product into DH5a. (JB)</P><H2>2 July 2014</H2><P>Today we took our field trip to the water treatment plant in Park City, UT.</P><P>Alpha Amylase: I prepped a DpnI digest with cutsmart buffer, one from original mut product the second time around (when we had a good concentration of plasmid to start with) and one that was from that same PCR product but had already been digested, so it was a redigest. One only had about 1.5 uL of the PCR product to digest though, I believe it was the one with just the original mutagenesis product (hadn’t been digested already). (JB)</P><P>Dispersin B:
Today we went up to the a sewage treatment facility near Park City. One of the problems that Gary the man who gave us the tour mentioned was biofilm build-up but unlike the biofilm problems mentioned by the University of Utah it was not formed by filamentous bacteria. Inspecting the plant itself the build-up was not as drastic as it might have been. Gary said that the film build-up would vary and that it went through cycles of high and low and that traditionally the plant managed it by small tweaks to the oxygen exposure, and rapidity of the plant waste cycling. We were able to obtain some samples from the plant of the biofilm which we can use to start testing Dispersin and Amylase.&gt;
(JM)</P><H2>3 July 2014</H2><P>Alpha Amylase: Transformed digests into DH5α. (JB)</P><H2>4 July 2014</H2><P>Alpha Amylase: No colonies again! (JB)</P><H2>7 July 2014</H2><P>Alpha Amylase: Dr. Grose said that the likely reason for no colonies was due to a diluted amount of PCR product being used since I only used 10 uL of the original mutated PCR product and added water, buffer, and DpnI for the digest I tried. She suggested transforming 10 uL of the PCR product + buffer mix in order to have a better chance of getting colonies with the mutated plasmid. (JB)</P><P> Dispersin B: I finished checking the results of my sequencing and to my relief Dispersin B appears to have been successfully transformed in both Colonies A and F. I set up a PCR reaction using the PIG 97 Dispersin Backbone snf the B1 107 and B1 108 Dispersin B primers because my next step is to transform the gene into the pet15b plasmid so I can start testing Dispersin's effectiveness quantitatively against the Biofilm. -JM</P><H2>8 July 2014</H2><P>Alpha Amylase: Transformations yesterday do not appear to have worked. Desi said that we could take the previous plate where we got several hundred colonies, pick from single colonies, streak them on a plate to grow them up and make sure we can obtain single colonies since they were so small on the last plate, and run PCR on them to see if there are any that look like they have the plasmid in them. (JB)</P><H2>9 July 2014</H2><P>Alpha Amylase: We discovered that the box of DH5α we had been using last week was not actually DH5α as someone had put the wrong tubes in the box. Hopefully this means that the transformations are working (given that we are actually transforming it into DH5α!). The mutated alpha amylase was retransformed into DH5a using 10 uL of PCR product as suggested last time by Dr. Grose since the PCR product was diluted during DpnI digest. (JB)</P><P> Dispersin B: Unfortunately after running my PCR product out on gel it doesn't look like I had any product. I had a very small amount of PIG 97 plasmid left so I only used .5 microliters for the PCR reaction. Next week I'm going to take the E. Coli cells I transformed with the PIG 97 Dispersin plasmid on the 20th of May and do a plasmid prep and purification so that I can have more PIG 97 to work with. Instead today I attempted to transform E. Coli cells with the Pet15b backbone so I would have enough to work with. In addition to plating 100 microliters of the transformed cells I diluted some into 5 mL of LB with Ampicillin and 100 microliters into 5 mL of LB with Canamycin in an attempt to grow up liquid cultures. The Canamycin being a negative control. Unfortunately neither the plate nor the cultures showed any growth so the transformation appears to have been unsuccessful. -JM </P><H2>10 July 2014</H2><P>Alpha Amylase: Finally, it looks like we may have some progress coming, despite accidentally spilling half of my transformation in the plating process. There was one colony, so it is looking more promising that the digest was effective and we may have one colony that was successfully mutated! </P><P>A liquid culture overnight in chloramphenicol was prepared from the single colony and the leftover on the stick was streaked on a chloramphenicol plate. A plasmid prep will be performed on the overnight and this will be sequenced in order to determine if this colony contains the mutant plasmid or the wild-type plasmid. (JB)</P><P>AiiA: Today I completed the ligation and transformation of Aii into DH5α. Because my sequencing returned an empty backbone last time, I am hoping that it works this time around. I am planning on picking up the plates tomorrow to prepare for colony PCR. (CZ)</P><H2>11 July 2014</H2><P>AiiA: I checked the plates today, and nothing had grown! Because we know it is no longer a problem with the E.coli, I am planning to recheck the digest of the backbone that I did next. (CZ)</P><P>Alpha Amylase: Busy day, so I just pelleted the cells down. (JB)</P><H2>16 July 2014</H2><P>Alpha Amylase: Today a plasmid prep was performed following the Denville procedure on the mutated alpha amylase colony from last week and this plasmid was then prepped for sequencing tomorrow. Each reaction consists of 2 uL of the mutated plasmid and 1 uL of either the forward signaling sequence alpha amylase primer or the reverse alpha amylase primer. (JB)</P><P>Aii: Today we decided with Dr. Grose to redo the amplification of Aii, since we got out a an empty backbone. I used some of the leftover amplified and purified insert that we had obtained previously, and redid the PCR. (CZ)</P><H2>18 July 2014</H2><P>Aii: There was a crack in the lid of my PCR tube, so I lost a fair amount of the amplified product. I redid the PCR twice, with one tube using the PCR from Wednesday as template, and another tube with my insert I purified in June.(CZ)</P><P>Alpha Amylase: We got back the sequencing results. The PstI restriction site located within the amylase gene remained unchanged meaning the mutagenesis did not work. We will have to redo the mutagenesis procedure on Monday. (JB)</P><H2>21 July 2014</H2><P>Alpha Amylase: Today we spent re-doing the site-directed mutagenesis and PCR digest since the previous two tries have been ineffective. Later that evening I added 1 uL of the DpnI digest and placed it in the 37 deg incubator to sit overnight. (JB)</P><P>Aii: Today I spent the time running a gel to check my PCR products. I also updated the journal. Although the bands were pretty smeary, there seemed to be a brighter band at the correct size. (CZ)</P><P> Dispersin B: I had a little over a uL of PIG 97 plasmid to work with so I diluted it in 10 uL of ddH2O and used 1uL to run PCR. The primers I used to run PCR were B1,107 and B1,108, forward and reverse primers for Dispersin B. I followed the Phusion PCR protocol- see common procedures. I also transformed DH5 Alpha cells with 1 uL of my diluted Dispersin and plated on Cam and Amp resistant plates. -(JM) </P><H2>22 July 2014</H2><P>Alpha Amylase: I transformed 5 uL of the digested mutagenesis PCR product into 50 uL of DH5α following our standard transformation procedure. This was plated on a chloramphenicol plate and will let sit in the 35 deg incubator overnight. (JB)</P><H2>23 July 2014</H2><P>Alpha Amylase: The transformation done yesterday yielded no colonies. Dr. Grose had this same problem with the four other reactions she did yesterday as well with the same kit. She suggested this is likely due to an issue with the competent cells used as the kit we used was relatively new, so I redid the transformation of 5 uL PCR mutagenesis product in 100 uL of the XL 10-Ultra Competent Cells provided in the Quikchange II Site-Directed Mutagenesis Kit, followed our standard transformation procedure, plated these on a chloramphenicol plate, and will let set in the incubator overnight. (JB)</P><P>Aii: I began today by doing a PCR purification for both of my PCR products. That took some time, and afterwards I carried out a restriction digest of both my inserts and the backbone. I also CIPed the backbone. On Saturday I will continue with ligation. (CZ)</P><P> Dispersin B: Unfortunately,when I ran the gel of Monday's PCR product I didn't get any banding aside from what appeared to be the primers at the bottom of the gel. The machine had melted the top of my PCR tube, but it's unlikely that such was the cause of my no product. I fear that the PIG 97 plasmid is too dilute for me to get a product from my PCR. Today I set up the same Phusion PCR reaction as Monday to try one more time and see if I just didn't pipette carefully enough on Monday. If it doesn't work again I did have colony growth on my Cam plate, so it's likely the plasmid successfuly transformed, so I will see if I can do a plasmid prep from a liquid culture of those colonies and use that new plasmid prep to PCR. I set up a liquid overnight culture from one of the colonies on the CAM plate,using 5mL of LB+ Cam solution. The last thing I did today was transform another group of DH5 Alpha cells with Pet15B plasmid and then I plated it on an Ampicilin agar plate. This was done so that when I get enough Dispersin B PIG97 plasmid to digest for an insert I will also have enough vector plasmid to digest it into. At which point I can ligate and transform the Pet15b with Dispersin B into E. Coli and begin testing it's biofilm degredation quantitatively. -(JM)</P><H2>24 July 2014</H2><P>Alpha Amylase: The transformation seemed to have been ineffective. However, the four other transformations Dr. Grose did with the mutagenesis products also did not work. Desi said that there were more issues with the 50 uL DH5α but that the box of 100 uL DH5α was working so I will try the transformation over with that tomorrow. (JB)</P><H2>28 July 2014</H2><P>Alpha Amylase: Today I re-transformed the mutated plasmid into a different batch of XL 10-Gold Ultra Competent Cells since the previous transformation seemed to be ineffective. I followed the standard transformation procedure to do this. (JB)
</P><P>Aii: I was finally able to come in to do my transformations. Everything seemed to be working well except that I had to heat the gel to a higher temperature, more like 70 degrees Celsius. I will get the results on Wednesday (CZ)</P><P>Dispersin B: My overnight culture of the PIG97 transformed cells had growth so I ran through the high copy plasmid prep and purification procedure with the lab kit. I also set up an overnight culture of the pET15b cells, that were plated last week. I have four small colonies, but given that our incubator has been running less than optimally and pET15b is not a high copy plasmid this isn't entirely unexpected. Nevertheless I used one of those colonies to set up a liquid overnight for Wednesday. The results of the gel electrophoresis from concerning the PCR I set up last week showed smearing and no banding so I was again unsuccessful. On Wednesday this week I will redo the PCR but this time using the PIG97 plasmid that I purified today. (JM) </P><H2>29 July 2014</H2><P>Alpha Amylase: The transformation plate did not have any colonies on it again. I am unsure what is going on this process, I will need to troubleshoot with Dr. Grose tomorrow. (JB)</P><H2>30 July 2014</H2><P>Alpha Amylase: I left the Alpha Amylase mutant transformation plate out again last night just to see what would happen and there were two colonies (the second colony is hidden in the corner of the plate)! I started an overnight of both in LB with chloramphenicol from the two colonies and will perform a plasmid prep on both and prep both for sequencing tomorrow morning. (JB)</P><P>Aii: My Transformation came back with basically a lawn of red colonies. I'm not exactly sure what happened, but obviously the ligation was ineffective. Unfortunately we also need to start over with a new signal sequence, so I am starting from square one. I will spend the rest of the week finding the sequence and getting new primers to start next week. (CZ)</P><P> Dispersin B: Today we were informed that we would be switching our focus from <I> Nitrosospira multiformis </I> to <I> Nitrosomonas europaea </I> or <I> Nitrosomonas eutropha </I> because we have been as of yet unable to acquire a sample of the multiformis that would grow. This means that we need a new signal sequence so a significant portion of the lab time was spent attempting to isolate a signal sequence, with the hope and goal being that we might find one which would work for the europaea or the eutropha. I took another colony from my pET15b plate and set up an overnight in LB AMP because my previous overnight attempt wasn't as successful as I wanted it to be for digestion because Pet15b is such a low copy plasmid. (JM) </P><H2>31 July 2014</H2><P>Alpha Amylase: This morning I performed a plasmid prep on the two colonies that showed up on my mutagenesis transformation plate. I then took these preps and set up reactions for sequencing of these colonies to check to see if they contain the mutated plasmid. (JB) </P><H2>2 August 2014</H2><P>Alpha Amylase: The results came back from the sequencing. It appears that the colonies that grew up were just contaminants. (JB) </P><H2>4 August 2014</H2><P>Alpha Amylase: Today we designed a new forward primer. We needed to do this for each of our constructs due to the issues we have been having with finding and growing up <EM>N. multiformis</EM>. Instead we will be obtaining <EM>N. eutropha</EM> and <EM>N. europaea</EM> from USU. Since the signaling sequence we were using previously is not compatible with <EM>N. multiformis</EM> we needed to find one that would work for all three types of bacteria that we will hopefully be using (assuming we are able to get our hands on <EM>N. multiformis</EM> finally). We found a very strong signaling sequence in the TolB protein that is exported extracellularly. This signaling sequence is found in all three strains of bacteria we are working with which will be very helpful. Below is the new forward primer containing a ccc spacer, an XbaI restriction site, the TolB signaling sequence as found in <EM>N. eutropha</EM>, and the first 20 base pairs of alpha amylase.</P><P>I also talked with Dr. Grose about the issues with the mutagenesis. I decided that I will just wait until we have the new plasmid with the new signaling sequence and then worry about getting the mutagenesis step to work. Dr. Grose stressed the importance of making sure the plasmid prep for mutagenesis is very clean and abundant and that generally tends to help give better results with the mutagenesis reaction. (JB)</P><P><TABLE align="center" border="2"><THEAD><TR><TH colspan="2">Alpha Amylase Primers</TH></TR><TR><TD>Forward Signaling Sequence (TolB)</TD><TD>5' CCCTCTAGATGCGTAATTTTTTGTATTGTACTGGTGTGTTTCTGTTGTTATGGATGAATACACCGCTACAAGCTGCTATGCGTAATCCCACGCTGAT 3'</TD></TR></THEAD></TABLE></P><P>Aii: I spent most of today looking for an adequate signalling sequence that would be effective for all three types of bacteria (<EM>N. multiformis, N. eutropha, N. europaea</EM>). Jordan and I BLASTed about 20 sequences apiece, as well as checking them on SignalP. Finally we found one that will work, a TolB translocation protein. I made a new forward primer for PCR with primer tails, the iGEM prefix, the 69 bp signalling sequence, and 20bp of my sequence. (CZ)</P><P><TABLE align="center" border="2"><THEAD><TR><TH colspan="2">Aii Forward Primer</TH></TR><TR><TD>Forward Signaling Sequence (TolB)</TD><TD>5' cgcTCTAG ATGCGTAATTTTTTGTATTGTACTGGTGTGTTTCTGTTGTTATGGATGAATACACCGCTACAAGCTGCTACAGTAAAGAAGCTTATTT 3'</TD></TR></THEAD></TABLE></P><P>
 Dispersin B: Today I did a Plasmid prep on the pET15b overnight colony from last week. I also submitted new primers to Dr. Grose.
(JM)</P><H2>5 August 2014</H2><P>Today I came in to run the PCR product that Jared started yesterday on a standard gel. I was also checking the pET15b and pIG 97. We had a couple problems, but we figured them out pretty quickly. (CZ)</P><H2>6 August 2014</H2><P>Alpha Amylase: Since we are going to want to have amylase ready for all three bacteria options, I will need to continue with finished the mutated amylase with the <EM>multiformis</EM> signaling sequence. And we finally discovered the reason why the mutagenesis has not been working! The mutagenesis primer #1 was fine but the primer #2 was only the complement, not the reverse complement! Finally, some clarity! The new primer should be here by Monday so then we can run mutagenesis PCR and have some mutant PstI-removed Alpha Amylase plasmid! (JB)</P><P> Dispersin B: I ran an electrophoresis using the pET15b purified plasmid prep, the PIG97 purified plasmid prep, and the PCR reaction. The gel results were unfortunately not what we wanted to see.. I got a band that was likely the primers in the PCR lane and the banding I got in the pET15b and the PIG97 lanes were inconsistent with the plasmid size that I was expecting. Both plates that had colonies were less than ideal so I ended up retransforming both PIG97 and pET15b. I then plated 100ul on cam and amp plates respectively. (JM)</P><P>Aii: Dr. Grose suggested that we continue with our work on the sequence that is specific to <EM>N. multiformis</EM>. Since I had gotten a lawn of red colonies last time around in the transformation, we thought that probably my restriction digest of the iGEM backbone must have been incorrect. Therefore I borrowed some backbone from the Auxotrophy group, which had worked in the past. I redid the ligation and transformation. (CZ)</P><H2>7 August 2014</H2><P>Aii: I came back in today to check my plate, and I had only grown one colony, which was a little pinkish. I therefore tried to grow it up further in 5uL of LB+CAM broth. I also transferred Jared's colonies into overnights as well. (CZ)</P><H2>8 August 2014</H2><P>Aii: I came in to check in on our overnights. My cells were visibly red. Therefore, I must have somehow purified the rfp of the iGEM plasmid and used it as insert. Jared's cells looked good, and I spun down 1 uL of each and froze them. (CZ)</P><H2> 11 August 2014 </H2><P>Dispersin B: Today I took the cells that Cameron had set up overnight colonies for from my plates and ran the spinspart plasmid purification kit with them to create purified PIG97 and pET15b plasmid. Using PIG97, I set up another PCR with the dispersin B primers and the phusion polymerase procedure and a control without a plasmid. Cameron also helped out with the plasmid purification, because he was waiting for his primers.
(JM+CZ) </P><H2>12 August 2014</H2><P>Alpha Amylase: The corrected mutagenesis primer came in last Friday, so today Dr. Grose and I set up the mutagenesis reaction for Alpha Amylase, sent it through the PCR machine, and then I added 1 uL of DpnI digest to the mix in the evening and let it incubate at 37° overnight. (JB)</P><H2>13 August 2014</H2><P>Alpha Amylase: Today I transformed the digested mutant product into 25 uL of DH5α and plated on chloramphenicol and will let it set overnight.</P><P>The Amylase primer for the TolB signal sequence also came in -I will run Q5 PCR with these primers after we have purified the mutated Amylase plasmid so that we can have colonies of <EM>N. eutropha</EM> with the appropriate signaling sequence and with the PstI restriction site removed. (JB)</P><P>Dispersin B: I ran the results of my PCR out on gel today and unfortunately the only thing that appeared were faint bands towards the bottom that appeared to be primers, the one with the plasmid looked the same as the lane without. Operating under the grounds that perhaps it is the phusion polymerase that is having problems I set up 5 PCR reactions today. I set up a taq polymerase reaction and a control, a Q5 polymerase reaction without the enhancer and a control, and lastly I set up another Q5 reaction, but this time I used 1ul of the old diluted PIG97 to test to see if the problem is that I dont have any purified plasmid in my reaction. Hopefully after today's results I will know what is going wrong with my PCR and why I'm not getting any product. (JM) </P><P>Aii: I started from scratch, since my previous attempts were not working. I began by purifying the iGEM parts. (CZ)</P><H2>14 August 2014</H2><P>Alpha Amylase: Well, the good news is that we have the correct primers this time and we got colonies. The slightly less good news is that there are hundreds of colonies. This likely means that the DpnI digest was not as effective as we might have hoped it would have been, but maybe with the high concentration of plasmid this is more explainable. Today I prepped colony PCR using TAQ polymerase in order to see whether or not there are any colonies containing the Amylase plasmid. (JB)</P><P>Aii: I transformed the purified plasmid into DH5-alpha. I learned this after, but I misread the type of vector the Aii was in, and plate the cells onto CAM plates instead of AMP. Luckily, Jordan was able to get more Aii out of the wells. (CZ)</P><H2>15 August 2014</H2><P>Alpha Amylase: The PCR products turned out really well. Each of the sixteen colonies I picked showed strong bands coming in around 2000 bps, which should be correct (200 bp plasmid forward primer + 100 bp signaling sequence + 1488 bp alpha amylase + 200 bp plasmid reverse primer). I prepped sequencing from colonies 5,6, and 8 using the forward and reverse pSB1C3 primers. (JB)</P><H2>18 August 2014</H2><P>Alpha Amylase: The sequencing results came back. Colony 5 was wild type and colonies 6 and 8 were unable to be sequenced up to the mutation point. We ordered an internal primer that will help us sequence closer to the mutation point. I also sent in colonies 6 and 7 for sequencing with the Amylase BI259 forward primer which will get us 200 bps further into the gene so we might be able to see the mutation point. (JB)</P><P>Aii: Today transformations were done of BBa_I729006 and BBa_C0060 from the iGem parts kits pieces in order to recover Aii to be used further. (JB)</P><H2>19 August 2014</H2><P>Aii: The transformation plate for BBa_I729006 did not have any colonies. This is likely due to there not being any plasmid left in the iGem parts kit (it was a faint pink color when resuspended in water). The transformation plate for BBa_C0060, however, did work. Two colonies from that plate were suspended in 5 mL of LB + Chloramphenicol to grow us the colonies for plasmid amplification and purification. We will need to remember to store some of this plasmid for our iGem plasmid collection. (JB)</P><P>Dispersin B: Unfortunately, the two control reactions evaporated during the PCR reaction. The three other reactions were run on gel. (JB) </P><P>Lane Order: 5kb Ladder, pIG97 with Q5 polymerase, pIG97 with Taq Polymerase, old diluted PIG97 with Q5 polymerase</P><H2>20 August 2014</H2><P>Alpha Amylase: Today I prepped more colonies for sequencing from the mutagenesis transformation. I mixed 2 uL of PCR product from colonies 3, 6, 7, 9, 11, 13, 15, and 16 with 1 uL of the internal amylase primer that we made. (JB)</P><P>Aii: We performed a plasmid purification on the BBa_C0060 Aiia and set up PCR reactions with the TolB signaling sequence forward primer and reverse primer. Two iterations of this were done - one with Taq polymerase and one with Q5 and no Q5 enhancer in case one of them does not work with Aii since there have been issues in the past. As seen in the gel image below, the Taq reaction appears to have worked (~700 bp Aii + ~100 bp TolB signaling sequence). (JB)</P><P>Well order: Q5 control, Aii BBa_C0060 with Q5 polymerase, Taq control, Aii BBa_C0060 with Taq polymerase</P><P>Dispersin B: From the Q5 PCR product of pIG97 (well 2 from the last round of PCR), a restriction digest was performed using the XbaI and SpeI restriction enzymes. Using the same enzymes, a restriction digest was performed on the Pet15B plasmid, and after 60 minutes of incubation, was 1 uL of CIP was added and allowed to incubate another 30 minutes. The restriction digest was performed following the standard procedure found in our &quot;Common Procedures&quot; page. Strangely enough, after running the insert and vector in low melt gel for 60 minutes, in the lane that had the insert there was only one band and in the lane that had the vector there were no bands. This is very confusing, especially in the case of the vector as even if the restriction enzymes were incorrect or did not work, there should have still been at least one band from the uncut plasmid. (JB)</P><H2>22 August 2014</H2><P>Aii: I'm back from my vacation! I took the sequence that Jordan made with TolB, and carried out a restriction digest. I put in the XbaI and SpeI enzymes, and then waited approximately 2 hours. I saved 8 uL as backup(CZ)</P><H2>23 August 2014</H2><P>Aii: Today I used a low-melt gel to purify the the digested insert (we already have purified and CIPed vector). I ran it out, and cut out the band of the correct size.(CZ)</P></BLOCKQUOTE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BYU_Provo/Notebook/Biofilm/julyaug">http://2014.igem.org/Team:BYU_Provo/Notebook/Biofilm/julyaug</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BYU_Provo/Notebook/Biofilm/julyaug" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BYU_Provo/Notebook/Biofilm/julyaug" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/julyaug&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/julyaug&amp;oldid=352080" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>