"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/Biofilm/febapr
From 2014.igem.org
BYU 2014 Notebook
Edit February April
Home
Team
Official Team Profile
Project
Parts
Modeling
Notebook
Safety
Attributions
14 February 2014
We worked on some preliminary research regarding the biofilm aspect of our project idea. A good potential gene to look into is DispersinB which last year's BYU iGem team worked with some, so we can talk to team members from last year regarding that. Below are links for papers worth looking into more. (JB)
http://www.ncbi.nlm.nih.gov/pubmed/23103508
http://www.uniprot.org/citations/12896987
http://www.nlm.nih.gov/medlineplus/ency/article/000679.htm
19 February 2014
Today we read some more on the biofilm being formed in activated sludge processors like the one in Park City, UT. We are searching for different enzymes that could be utilized to break down the biofilm buildup found in these types of places.
11 March 2014
We emailed BYU's iGem team from last year which had done some work on biofilm degradation with alpha amylase and dispersin B. They gave us some good papers to look into regarding this aspect of our project. Another gene we have been looking into is Aiia, which is a quorum sensing blocker, which would be helpful in preventing aggregation of bacteria into biofilms. (JB)
17 March 2014
Today we went over our respective presentations for our subprojects. We will need to be able to insert our modified plasmid with our biofilm inhibiting genes into the N. multiformis bacteria which will then produce the enzymes and quorum sensing components and will affect the other bacteria forming the biofilms in the ASPs. We will need to look at fitness costs later on down the road and select for the most fit mutants with our biobrick in it. (JB)
Some possible candidates for degrading enzymes are Dispersin B and NucB
Sources:
Modeling and biochemical analysis of the activity of antibiofilm agent Dispersin B. Kerrigan JE1, Ragunath C, Kandra L, Gyémánt G, Lipták A, Jánossy L, Kaplan JB, Ramasubbu N. doi: 10.1556/ABiol.59.2008.4.5.
In vitro evaluation of DispersinB on methicillin-resistant Staphylococcus pseudintermedius biofilm. Turk R1, Singh A, Rousseau J, Weese JS. doi: 10.1016/j.vetmic.2013.07.011. Epub 2013 Jul 20.
Detachment of Actinobacillus actinomycetemcomitans biofilm cells by an endogenous beta-hexosaminidase activity. Kaplan JB1, Ragunath C, Ramasubbu N, Fine DH.
(JM)
19 March 2014
Today we started doing research as to how we will insert our genes of interest into the plasmid. Hopefully our genes will be compatible with the standard iGem E. coli plasmid pSB1C3. We need to find the gene sequences that will be used and design primers with restriction sites on them so that we can amplify the gene and insert it into the plasmid. We will also need to start thinking of experimental methods of how to test the efficacy of the different genes in biofilm dispersal and inhibition. I will be doing the research for alpha amylase.
BYU 2013 iGem team’s alpha amylase part: BBa_K1195001
Uniprot: I6S010 (E. coli alpha amylase)
As E. coli is a gammaproteobacteria and N. multiformis is in the betaproteobacteria class, if there are issues that arise from the E. coli gene being incompatible with the N. multiform is, we will need to consider using the betaproteobacteria Alpha Amylase found on Uniprot (Uniprot:Q47IJ1), but this would require isolating this gene ourselves from a betaproteobacterium. Alpha Amylase requires calcium for activation. We will need to verify its presence in the ASPs. (JB)
Dispersin B does not have any of the four restriction sites EcoRI, XbaI, SpeI, or PbaI inside its code so it is compatible with all four. It is also a part of the igem registry, however the signal protein was removed to allow for intracellular expression.  Will this removal affect our designs for the enzyme, namely to allow for extracellular expression to attack external biofilms? (JM)
21 March 2014
We discussed as a group a game plan for our next presentation. We know that the alpha amylase from E. coli that is in the iGem registry will be compatible with the pSB1C3 plasmid. We will need to discuss further how to link all 3 genes together and we will also need to determine if the genes will need to go on the plasmid in a certain order or if they will need anything additional in order to complete their function. We also discussed some preliminary ideas as to how we will test the efficacy of the biofilm inhibition by our components and also the concern whether or not they will be effective at degrading existing biofilms.
Alpha Amylase: I began designing my primers with restriction sites today for the Alpha Amylase gene that is from the BYU iGem registry. The restriction sites used will also depend on the order of linkage of the other genes that will go in our plasmid. (JB)
Dispersin B: One proposed method was cloning in each of the three enzymes and have them expressed in separate batches. Test each individually on biofilm and measure the results. Then test all three together and measure results.  Using said data, determine if the functionality could be maintained with the removal of one of the enzymes  to so as to possibly decrease the fitness cost on our target bacteria N. Multiformis.  Possible methods of measuring biofilm breakdown: Use of a spectrophotometer to measure adsorption before and after treatment.  Visual staining? (JM)
24 March 2014
Alpha Amylase: Today I continued to work on designing the primers for Alpha Amylase. They are all done except for the promoter/Shine Dalgarno sequence. Dr. Grose showed us how to search for signaling sequences. (JB)
Dispersin B: Worked on primers for Dispersin B. Also found an article with a possible assay method for biofilm degradation.
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3001887/ (Dispersal of Biofilms by Secreted, Matrix Degrading Bacterial DNase)
26 March 2014
Alpha Amylase: Today we finished preparing our presentation that we are giving later today. I finished the primer designs but there were a lot of potential hairpins and self-annealing sites that we will have to take a closer look at. As well, the PstI restriction site is contained with the alpha-amylase so we will not be able to use that restriction enzyme with Alpha Amylase until we remove the site with site-directed mutagenesis. (JB)
The dispersin B gene in the igem registry doesn’t have a signal sequence. A signal sequnece is necessary to accomplish what we want with all of our enzymes, AiiD, Amylase, and Dispersin, namely, export them outside the cell once they are made. So we will need to include a signal sequence for Nitrosospira Multiformis inside our primer design. The Signal Sequence Detection Program SignalP was used and the following Signal Sequence Proteins are found in Nitrosospira Multiformis OmpA, DsbA, TolB. Using SignalP program the following sequence was the best candidate for a signal sequence atccgcgcga gcaggggaaa ttgacggaaa acctataccg gccagcaaca ggaatgccgtaagcagccgc agtaaattca t  coming from the DsbA protein. (JM)
27 March 2014
Alpha Amylase: Today I did some research into methods other scientists have employed for measuring biofilm dispersal and degradation. (JB)
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1932551/pdf/1249-06.pdf
http://download.springer.com/static/pdf/846/art%253A10.1007%252Fs12010-011-9526-2.pdf?auth66=1396038196_95251369f639464eb174e81ed448c024&ext=.pdf
28 March 2014
Alpha Amylase: Today we talked to Dr. Grose about designing our primers. I also looked into oligocalc to see if my primers for a-amylase will be viable. The forward primer is fine, the reverse primer needs more reverse complementary base pairs added to the beginning to increase the melting temperature. Dr. Grose also told us to only use the Xba portion of the prefix to cut down on primer length and only use SpiI so that we can clone a-amylase without the PstI restriction site for initial cloning of the gene (we have to do this because a PstI restriction site is found in the middle of the a-amylase gene). Below are my primers. (JB)
Alpha Amylase Primers
Forward Signaling Sequence
5' ccctctagatgatccgcgcgagcaggggaaattgacggaaaacctataccggccagcaacagg 3'
Reverse Signaling Sequence
5' aacagcgtgggattacgcatatgaatttactgcggctgcttacggcattcctgttgctggccggtatagg 3'
Reverse
5' ccccactagtattaaatcacctcttcgataaccc 3'
Mutagenesis 1
5' gtttgatgcgccgctccagatgaaattccat 3'
Mutagenesis 2
5' caaactacgcggcgaggtctactttaaggta 3'
Dispersin B:  Unfortunately with the addition of the signal sequence our primers are incredibly long. Running them through a program which checks for hairpins and other self-interacting segments (Oligocalculator) found numerous areas where our forward primer would form problematic secondary structures with itself and inhibit to a degree primer annealing. The main concern is that the primer forms self-dimers.On Wednesday we started the initial process of trying to fix the primers and began making small changes to the nucleotide sequence to see if that would help, but ultimately there were too many regions to fix in this manner. The other option we were looking at is to split the primer into chunks between the 4 restriction sites.
Below is a theoretical primer idea before splitting.
[Restriction Site][Promoter BBa_J23119][Shine-Dalgarno][Signal Sequence*][~20 first BP Enzyme (preferably ending on a G/C  bond)]
*used to export expressed protein enzyme outside of the cell
REVERSE: [Restriction Site][~Reversed compliment of ~20 last BP Enzyme]
Here is an example after the possible split.  [Restriction Site][Promoter BBa_J23119][Shine-Dalgarno][Signal Sequence][Restriction Site][First 20 BP Enzyme] From checking out various sites that will sell Primers I have learned that we could make a sequence that would anneal to the target sequence at a higher temperature than it will form secondary structures at.
(JM)
31 March 2014
Alpha Amylase: Today we learned about site-directed mutagenesis
Dispersin B: Primer Design:
Dr. Grose mentioned that we would be using an igem plasmid with promoter and shine-dalgarno already integrated. She also said that the signal sequence would just hang out in the first few PCR runs and not cause problems if it self-annealed.
Forward. (Spacer)GCG(Restriction)CTAG(Start)ATG(signalSequence)ATCCGCGCGAGCAGGGGAAATTGACGGAAAACCTATACCG GCCAGCAACAGGAATGCCGTAAGCAGCCGCAGTAAATTCAT (20 Nucleotides):AATTGTTGCGTAAAAGGCAA
Reverse AGACGAATGGGGATGAGTGA
Flipped Compliment  (Space)CGC(Restriction)CTAG(20Nucleotides)TCACTCATCCCCATTCGTCT (JM)
2 April 2014
Alpha Amylase: Today I continued to work on the procedure write-up for Alpha Amylase. (JB)
Dispersin B: Today the time was spent writing up and designing our lab protocol and procedures. (JM)
4 April 2014
Today I continued to work on fine-tuning the protocol for a-Amylase. Received some clarification on the steps needed in order to both insert the signaling sequence into the gene as well as use site-directed mutagenesis to remove the PstI restriction site from the a-Amylase gene from BYU. (JB)
Dispersin B: Today was also spent writing up the lab protocol. We intend to finish it individually and combine on Monday with the most accurate set up and design. Our cloning procedures for the three enzymes should be fairly similar with the exception that Amylase will need to be point mutated to fix the PstI restriction site in the gene. The testing procedure for the Amylase and the Dispersin can also be the same. However the Aiia will need a separate procedure as it does not break down biofilm but will hopefully rather prevent its formation. One possible assay for the Aiia would be to begin colonies of biofilm forming bacteria under different concentrations of Aiia and see how much the biofilm formation varies between the test samples. (JM)
7 April 2014
Today we spent the majority of the time working on compiling the different aspects of our protocols paper as a group.
9 April 2014
Alpha Amylase: I found a possible contact for obtaining a Nocardia sample in case we want to test the biofilm degradation since Nocardia is a large component of the biofilm found in Activated Sludge Processors. (JB)
We also transformed several of the iGem promoters into E. coli. This was done in order to test the efficacy of the different promoters so that we can known which ones are the best to use for our project. We followed the cloning procedure found in the iGem protocols packet Dr. Grose gave us at the beginning of the semester. (JM)
11 April 2014
Today we performed a plasmid prep on the plasmids containing the promoters that we were working with the other day. We followed the Denville Plasmid Prep protocol #5 (SpinSmart Plasmid Purification Protocol: High-copy plasmid DNA from E. coli). (JB)
14 April 2014
Today we compiled the information that we have gathered for our final protocol and started planning our powerpoint for the presentation on Monday we will be giving to our team.
21 April 2014
Today we presented our biofilm degradation presentation to the team.
29 April 2014
Today we worked on the transformation of the plasmids containing the different promoters into DH5α.
For alpha-amylase we are testing both PIG92 and PIG98 to see if one or both of them are the alpha amylase and then we will transform whichever works. Tomorrow after PCR is done we will run 5 uL in gel. Along with the two potential alpha amylase plasmids we also performed PCR on the DispersinB plasmid in the iGem catalog and we performed a transformation of Aiia into DH5α.
PCR Set up and Part Transformation.  Today for Dispersin B and Alpha Amylase we set up and ran PCR because we had the template Samples of the DNA from last year’s BYU Igem team. This will enable us to run a restriction digest and begin transforming our inserts into E. Coli for expression testing on the 1st of May. The Aiia gene was part of the 2013 igem kit so we hydrated the part and then transformed it into e-coli. We hope to begin expression testing on that part on the 1st as well. Protocol for PCR of Dispersin and Amylase
Reaction mixture :
10uL of 5x Q5 Rxn Buffer
1 uL of dNTPs
1 uL of Forward Primer
1 uL of Reverse Primer
10uL of Q5 enhancer
23uL of ddH2O
1-2uL template DNA
We then PCRed the mixture for 2 & ½ hours.  We also transformed E. Coli DH5 Alpha with the Aiia gene plasmid from the igem 2013 kit Well 2k plate 3. We hydrated the part then added 5 uL to 100 uL of DH5 Alpha cells. We then put on ice for 2 minutes, heat shocked at 42 Degrees Celcius for 1 minute and then back on ice for 2 minutes. We then added 100 uL of LB solution and incubated at 72 degrees over night. Cells were removed the following day.(JM)
Retrieved from " http://2014.igem.org/Team:BYU_Provo/Notebook/Biofilm/febapr "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
