"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/CRISPR/febapr
From 2014.igem.org
BYU 2014 Notebook
Edit February April
Home
Team
Official Team Profile
Project
Parts
Modeling
Notebook
Safety
Attributions
Week of March 16
Michael Linzey
Exclusion genes
- Defective prophage is the often the exclusion gene
- E14 containts the lit gene, lit protein interacts with the major head protein of T4 casing translation inhibition
- Lit and gol interact causing the cleavage of the translation elongation factor, this kills the phage and the bacteria
- Rex gene from the lambda phage
- Also the pif gene on the F plasmid inhibits T7 and other related phages
- Lit inhibits most even T phages
- In unstructured environment the abortive system is not very effective, however in a structured environment the abortive system is helpful.
Michael Linzey
- Instructed by Dr. Grose to focus on crispr system
Research on CRSPRS
- 3 types of CRSPR systems
- http://www.genome-engineering.org/crispr/
- http://crispr.u-psud.fr/cgi-bin/crispr/SpecieProperties.cgi?Taxon_id=323848
- There are for sure no crisprs in this genome
- http://crispr.u-psud.fr/crispr/CRISPRProperties.php?RefSeq=NC_008344&Taxon=335283
- Nitrosomonas eutropha gives the location of 3 crisprs
- Crisprs are subject to horizontal gene transfer
Found a paper that included the crispr coordinates for a bunch of different bacteria
3/21/14 Michael Linzey
Continued research on crisprs
- A complex of Cas proteins 5, 6, and 7 is required for the biogenesis and stability of  crRNAs in Haloferax volcanii
- One of the most important genes/proteins is cas6. Cas 6 produces the crRNA’s, which directs the invader - degrading Cas protein complex to the invader
- Cas5, cas6, and cas 7 in a 1.7,1,8.5 relationship
Which genes are essential for crispr
- Cas 5, Cas 6, and Cas 7(see above)
- Cas 9 endonuclease, cuts up viral DNA
- Cas 4: grabs the invading DNA and incorporates it into the host genome
Structural basis for CRISPR RNA-guided DNA recognition by Cascade
- Cascade is a 405-kDa complex comprising five functionally essential CRISPR-associated (Cas) proteins (CasA1B2C6D1E1) and a 61-nucleotide CRISPR RNA (crRNA) with 5′-hydroxyl and 2′,3′-cyclic phosphate termini.
- This website gives hyperlinks to all of the proteins in the cascade complex, it links to the ncbi
Week of March 14-21, 2014: Michael Abboud
The following information was found from literature research:
- Genes required for different CRISPR systems that may be transformed into N. multiformis:
- E. coli K12 - cas3 (predicted HD-nuclease fused to a DEAD-box helicase), five genes designated casABCDE, cas1 (predicted integrase), and the endoribonuclease gene cas2.
- Streptococcus thermophilus – cas5, cas1, cas6, cas7
- Type II Cas system – tracr, cas9, cas1, cas2, csn2
All of these are followed by the CRISPR array
Week of March 23
Week of March 21-28, 2014: Michael Abboud
The following information was found from literature research and is relavent to our project:
- N. multiformis was BLASTed with the CRISPR system of S. thermophilus as registered in iGEM. This revealed very little commonality and confirms that N. multiformis has no Type II CRISPR.
- BLASTing the CRISPR system of S. thermophilus as registered in iGEM shows the following strains as hits (They probably have the same system).
• S. thermophilus MN-ZLW-002
• S. thermophilus ND03
• S. thermophilus LMD-9 (this is the strain that iGem uses)
- Csn1 and Cas 5 are synonyms for Cas9 (Q03JI6).
- Nitrosomonas strains contain no Csn1/Cas9 endonuclease (No Type II CRISPR)
- Burkholderia doesn't have well-defined CRISPRs, those I found did not seem to use Type II, though they tended to use CSE genes instead of CAS or CSN genes. I did this because Burkholderia phages infect multiformis so they might have something similar there.
- PAM (proto-spacer adjecent motif) sequences are necessary for Cas9 to "cut".
Week of March 30
Week of March 28-April 4, 2014: Michael Abboud
We chose to use the Type II CRISPR system of Streptococcus thermophilus DGCC7710. BLASTing the sequence revealed several “forbidden” restriction sites. Fortunately, XBaI and PstI were not present so they can be used as restriction sites (SpeI is compatible cohesive end for XBaI). I designed a primer for site-directed mutation of a SpeI site on the CRISPR system. We chose the iGEM plasmid pSB1C3 in which to insert the CRISPR system. We will insert both the mutated and unmutated version of the CRISPR plasmid into E. coli and N. multiformis and compare the expression. A comparison of promoters should be conducted as well.
March 31, 2014 Michael Linzey
Cas 1 and Cas 2 Proteins
CRISPR-associated protein Cas2 subtype (IPR010152)
CRISPR-Cas Systems: RNA-mediated Adaptive Immunity in Bacteria and Archaea (it looks like a book)
- Not a lot is know about the cas 1 and cas 2 other then they are adaptive
- Thought to be part of the adaptation mechanisms of  the crispr system
- Spacer acquisition
- Cas 1
- High nucleic acid binding affinity, metal dependent
- Produces dsDNA sequences about 80 base pairs in length
- Cas 2
- Small proteins
- Act as an endoribonuclease (looks for uracil rich regions), also metal dependent
- Facilitate space selection, integrate new spacers, and theoretically could degrade phage transcripts, or inhibit phage global transcription through RNA cleavage
The basic building blocks and evolution of CRISPR–Cas systems
- Cas1 is extremely important, provides adaptation
- Cas2 is a little more ambiguous
- Need both for space integration but don’t really know why we need cas2
- However, Type II systems use a seemingly unrelated mechanism that involves cellular RNAse III that is not encoded within the CRISPR–Cas loci and performs unrelated functions in RNA processing [37], a separately encoded small tracrRNA (transactivating crRNA) that is homologous with a cognate CRISPR repeat, and an unknown domain of Cas9, the multidomain protein that is the signature of the Type II CRISPR–Cas
- It has been shown that both proteins and a single CRISPR repeat are required and sufficient for spacer integration
- This is important because we will need to add in at least one specific spacer in order for it to work
We spent the rest of the day designing our primers and restriction sites in the genes that we need to mutate
April 2 Michael Linzey
I am designing 2 primers for 2 restriction sites on our genes
- Used neb cutter to find the restriction sites
- The sites I will be cutting are ecor1 (5843) and spe1 (6399)
- EcoR1
- gct cagttagtca ag gaattc ta tcagtatgat gaa (original sequence
- Forward Primer: gctcagttagtcaag gagttc tatcagtatgatgaa
- Reverse Primer:  ttcatcatactgatagaactccttgactaactgagc (one possible hair pin but making it longer made it worse.)
- I used http://web.expasy.org/translate/ to translate the gene so that I could know which nucleotide to change without changing the amino acid it sequenced
- To find the reverse compliment I used an internet provider
- Spe1
- Same process as above
- tgatgcc ctataata actagt aaatta gtaataagt
- Forward Primer: tgatgccctataata attagt aaattagtaataagt (one hair pin and multiple self-annealing sites
- Reverse Primer: acttattactaatttactaattattatagggcatca (3 hair pins and multiple self-annealing sites)
- Both of these get worse the longer that they get
Week of April 6
Week of April 4-11, 2014: Michael Abboud
We still need to design primers for inserting our own repeat-spacer sequence for acquired immunity in N. multiformis. We will do that later, but most likely it will involve “stitch” primers that overlap each other. The rest of the circuit has been designed so that we can insert the Cas genes necessary for the CRISPR system into our organism. Promoter analysis is ongoing, but so far we have purified the promoter plasmid from E. coli.
4/07/2014 - Garrett Jensen.
- This week we worked Monday on our circuit design paper. The assignment is saved on my computer in the igem folder.
April 7 Michael Linzey
- Today we spent pretty much all of the time finalizing which primers we are going to use
- We also figured out the protocol that we will need to use to complete our project
- We put our group objectives into words so we can focus on accomplishing those things
- Worked as a group to write a paper detailing what our project is going to be about and how to complete it.
4/09/2014 - Garrett Jensen.
- today we worked on the circuit assignment more. We did not make primers for inserting our own spacers because dr grose told us not to just yet.
- Started doing plasmid preps of the promoters in the igem registry. We removed some of the primers from the igem plates and transformed them into E. coli. After this we plated them on lb/amp and incubated overnight. Friday we will harvest our plasmids from these cells in preparation for moving them into n multiformis.
April 9 Michael Linzey
- Finished up the paper that we were writing on Monday
Transformations
- I transformed 2 types of igem plasmid into E Coli
o The difference in types were caused by different promoters
- Protocol
o Placed 3 ul of plasmid with a specific promoter into 25 ul of E Coli
o Heat shocked for 1 minute at 42 C
o Iced for at least 3 minutes
o Added .5 ml of LB
o Placed into an incubator for 30 minutes at 37 C
o Plated the E Coli
o Allowed to grow overnight in an incubator at 37 C
- The reason we are doing this is because it will allow us to test to see which promoters are consensus promoters and are strong promoters. It will also allow us to see which should be used by less active genes
- This is also good practice using iGEM products
- This will help our main project because we will know which promoter we should use to express our CRISPR genes
4/11/2014 - Garrett Jensen
-today we are harvesting the plasmids from E. coli usig the plasmid DNA purification kit. We are following the instructions desi gives us to the letter!
April 11, 2014 Michael Linzey
Purified plasmid
- Today we purified the plasmid that we transformed into E Coli
- We did this by breaking the cell membrane and causing the cell to lysis
- We then centrifuges it and caused the cell parts to pellet at the bottom of our Eppendorf tubes
- We then filtered the plasmids out and used several washed to clean it
- We then put the filter into a new tube and added one last buffer that allowed the plasmid to come out of the filter and into the new tube as we centrifuge it again
Week of April 27
4/27/14 - Garrett Jensen. Today we worked on amplifying our plasmid from the igem registry so that we have it ready to use when we get our crispr ready to move into e coli.
- Using E coli DH5α we added in 1 µL of igem plasmid, incubated on ice for 2 min
- Heat shock at 42C for 60 sec.
- Recovered on Ice for 5 minutes
- Incubated for 30 minutes at 37C
- Plated 100 µL on a plate with chloramphenicol. 400 µL on another plate with chloramphenicol. The plasmid is very concentrated, so transformation is usually very efficient.  Plating all 500 µL of our bacteria would give us a whole lawn of bacteria, so we only will plate 100 so we can get individual colonies. We will need to pick a colony and start an overnight from that to do plasmid preps from next class
4/28/14- Garrett Jensen.
-I went in today to start our overnight, but no bacteria grew.  Desi said we might be able to plate our cells after 30 minutes incubation but she always does 90.  30 must not have worked because none of our cells grew, but Jordan's did and he incubated his for longer.
○ ***NOTE:  when transforming e coli with a plasmid that has resistance to anything but ampicillin you need to incubate it for 60-90 minutes before plating.
April 29 Michael Linzey
Today we worked on transforming our plasmid into E Coli
- We did this so that when our s. thermophilus arrives we will have a large quantity of vector that we can use to transfer our CRISPR system into
- This is done by heat shocking a sample of E Coli with the plasmid added to it so that the E Coli will take up the plasmid
- We then incubated it at 37 C for 30 minutes
- Then we plated them over night
Retrieved from " http://2014.igem.org/Team:BYU_Provo/Notebook/CRISPR/febapr "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
