"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/Metabolism/febapr
From 2014.igem.org
BYU 2014 Notebook
Edit Feb April
Home
Team
Official Team Profile
Project
Parts
Modeling
Notebook
Safety
Attributions
Week of March 22nd
March 17, 2014
Researched articles on the effects of heavy metals on waste water treatment plants and the effects of heavy metals on bacteria to prepare for our presentation on N.multiformis metabolism optimization. Searched for common bacteria with heavy metal resistance genes that could be possibly insert into our bacterial chassis; finding sequences and reading about success rates in data of those that had been transferred.
--CS-- Researched metabolism options more.
March 18, 2014
--CS-- Reviewed all of our literature findings so far. In doing so, identified the specific focuses for our group: inserting the denitrification genes into N. multiformis, making N. multiformis more resistant to pH changes, and making N. multiformis more resistant to heavy metals.
March 19, 2014
--CS-- Presented our ideas for improving the metabolism of N. multiformis and received feedback from the class on them. Confirmed plan to insert the denitrification pathway into N. multiformis. Decided to forego other original goals and instead insert genes that would break down antibiotics.
March 20, 2014
Searched for the most commonly prescribed antibiotics in the United States. Top prescribed antibiotics include penicillins and macrolides according to the New England Journal of Medicine (2013)
U.S. Outpatient Antibiotic Prescribing, 2010
March 21, 2014
Searched articles on the effectiveness of macrolide and beta-lactam degradation enzymes. Researched bacteria with a known gene sequences to degrade both types of antibiotics.
--CS-- Continued researching denitrification pathway. Discovered that there is a BioBrick for denitrification already in the iGEM registry but that it appears to be incomplete (BBa_K1067006).
Week of March 29th, 2014
March 24, 2014
Investigated macrolide antibiotic degradation, settling on the ethryomycin esterase as the enzyme. Found part BBa_K1159000 in the IGEM registry which contains the Erythromycin Esterase Type II (EreB) gene that degrades macrolides.
--CS-- Continued researching denitrification ideas.
March 26, 2014
--CS--Presented the circuits for our group and decided to clone the genes ourselves from Pseudomonas aeruginosa instead of using BBa_K1067006.
March 28, 2014
Used the Anderson Promoter Collection to determine which promoters have the highest rate of expression. Antibiotic degradation genes would need medium to strong expression to be useful to the bacteria
--CS-- Reviewed literature about the genes involved in denitrification. Also started looking into promoters to use for these genes. Decided to use a constitutive promoter with medium expression for now and possibly come back and use a nitrate/nitrite-dependent promoter once everything is working right.
Complete Genome Sequence of Nitrosospira multiformis, an Ammonia-Oxidizing Bacterium from the Soil Environment
Nitrous oxide production and methane oxidation by different ammonia-oxidizing bacteria.
Comparison of Nitrosospira strains isolated from terrestrial environments.
Week of April 5th
March 31, 2014
--CS-- Reviewed more literature about the denitrification genes. Used NCBI BLASTn to confirm that the denitrification genes from Pseudomonas aeruginosa or homologues are not in the N. multiformis genome; only one of the genes (qnorB) has an E value of any significance (3e-22). Located the different denitrification genes in the Pseudomonas aeruginosa PAO1 genome.
Pseudomonas aeruginosa and Achromobacter sp.: nitrifying aerobic denitrifiers have a plasmid encoding for denitrifying functional genes.
April 1, 2014
Contacted the 2013 Technical University of Munich IGEM team to inquire about the EreB plasmid since because the registry said that it was not available. Received a response that the part would be available for 2014. Also contacted IGEM to request the part in the 2014 plate.
April 2, 2014
--CS-- Reviewed more literature about denitrification. Found that P. aeruginosa, N. multiformis, and E. coli are all gram negative, so the nitric oxide reductase, which works in the periplasm, should theoretically work in all three bacteria. Identified nirS, norB, norC, and nosZ as the genes needed to insert the denitrification pathway into N. multiformis. Also found that the enzyme that converts nitrite to nitrate in the nitrification process of N. multiformis does so in a reversible reaction, so inserting these genes should force equilibrium through to nitrogen gas.
Differentiated Response of Denitrifying Communities to Fertilization Regime in Paddy Soil.
April 3, 2014
Researched scholarly articles about denitrifying genes to determine which particular enzymes are the most important. The paper describes several experiments with these enzymes in soil denitrifiers, the genes required to denitrify, and the importance of each gene present in soil bacteria.
Differentiated Response of Denitrifying Communities to Fertilization Regime in Paddy Soil.
April 4, 2014
Checked denitrifying genes for internal restriction enzyme sequences.
Week of April 12th, 2014
April 7, 2014
Prepared primer sequences to perform mutagenesis to exchange nucleotides and change the restriction site within the gene. Primers were designed for the denitrification norB gene that contained the IGEM plasmid restriction site EcoR1. Those primers were:
5’-CCGACCACGTACTGAAGGCCCATGATC-3’
5’-GATCATGGGCCTTCTGTACGTGGTCGG-3’
5’-TGCAGCCAGTCCTGTAGCACCCCG-3’
5’-CGGGGTGCTACAGGTCTGGCTGCA-3’
--CS-- Designed primer sequences for denitrification genes. Forward primers include 3 hanging nucleotides, the XbaI restriction site, and the first several nucleotides of the gene. Reverse primers include 3 hanging nucleotides, the SpeI restriction site, and the last several nucleotides of the gene. The primers were:
nirS Forward: 5’-CCGTCTAGATGCCATTTGGCAAGCCACTGGTG-3’
nirS Reverse: 5’-CCGACTAGTTCAGTACACGTCGTGCTGGGTGTT-3’
norB Forward: 5’-CCGTCTAGATGATGTCGCCCAATGGCTCCCTGA-3’
norB Reverse: 5’-CCGACTAGTTCAGGCGGCCGCCTTGCCGCGCCGG-3’
norC Forward: 5’-CCGTCTAGATGTCCGAGACCTTTACCAAAGGC-3’
norC Reverse: 5’-CGGACTAGTTCAACCCTCCTTGTTCGGCGGCCA-3’
nosZ Forward: 5’-CCGTCTAGATGAGCGACGACACGAAAAGCCCCC-3’
nosZ Reverse: 5’-CCGACTAGTTCAAGCCTTTTCCACCAGCATCCGC-3’
Also began researching assay techniques to use in testing the different steps of denitrification.
April 9, 2014
Finished the circuit write up for macrolide degradation and outlined a protocol to test the function of the gene. Following the write-up, we transformed the IGEM constitutive promoter BBa_J23109 to test its functionality in competent E.coli.
--CS-- Searched online for more assay techniques to use in testing the denitrification genes. Transformed the IGEM constitutive promoters BBa_J23117 and BBa_J23118 into chemically competent E.coli.
Regulation and Function of Versatile Aerobic and Anaerobic Respiratory Metabolism in Pseudomonas aeruginosa.
April 11, 2014
Performed plasmid preps with the transformed bacteria according to our Common Procedures .
--CS-- Performed plasmid preps for BBa_J23104, BBa_J23105, and BBa_J23106.
Week of April 19th
April 14, 2014
--CS-- Decided that instead of finding various assays to test the denitrification genes individually after they have been cloned and transformed, we would first combine all of the denitrification genes into a single plasmid, transform it into the bacteria, and then test the ability of the bacteria to convert nitrate to nitrogen.
Week of April 26th
April 21, 2014
Our team prepared a semester final on our N.multiformis metabolism optimization processes.
Retrieved from " http://2014.igem.org/Team:BYU_Provo/Notebook/Metabolism/febapr "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
