<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BYU_Provo_Notebook_CRISPR_septoct"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BYU_Provo/Notebook/CRISPR/septoct">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BYU_Provo/Notebook/CRISPR/septoct&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CRISPR/septoct&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CRISPR/septoct&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BYU_Provo/Notebook/CRISPR/septoct" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BYU Provo/Notebook/CRISPR/septoct</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1 style="color:#FFFFFF">BYU 2014 Notebook </H1><P style="color:#FFFFFF"><A href="https://2014.igem.org/wiki/index.php?title=Team:BYU_Provo/Notebook/CRISPR/septoct&amp;action=edit" style="color:#FFFFFF">Edit September October</A></P><TABLE width="100%"><TBODY><TR heigth="30px"><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo" style="color:#002255">Home </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Team" style="color:#002255"> Team </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=BYU_Provo" style="color:#002255"> Official Team Profile </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Project" style="color:#002255"> Project</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Parts" style="color:#002255"> Parts</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Modeling" style="color:#002255"> Modeling</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Notebook" style="color:#002255"> Notebook</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Safety" style=" color:#002255"> Safety </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Attributions" style="color:#002255"> Attributions </A></TD></TR></TBODY></TABLE><BLOCKQUOTE><H2>Week of August 31</H2><P><U>September 2, Michael Abboud</U>Used DNA purification kit on low-melt gel of PCR product #1 from first sewing PCR.

</P><P><U>September 3, Michael Abboud</U>Ran three sewing PCR reactions to combine Primer #4 with the first sewing PCR, Primers #5,6: 1) Control 2) Purified low-melt cut-out 3) Original PCR product.


</P><H2>Week of September 7</H2><P><U> Michael linzey 9/8/2014 </U>-I took a sample of the CRISPR mutagenisis and streaked it out on another plate
-This will allow us to have spare plasmid to test our mutated CRISPR system
-I left the plates in overnight to allow them to grow up
-Helped Garrett prepare for the intial CRISPR tests </P><P><U>September 8, Michael Abboud</U>Ran the last sewing PCR reactions on gel.  The purified, low-melt cut-out did not produce enough DNA for subsequent sewing PCR reactions. Therefore, sewing PCR reactions will directly serve as the primer for the next reaction without running on low-melt gel or purification.


</P><P><U> Michael Linzey 9/9/14 </U>-I used the plasmid purification kit to extract the mutated plasmid
-I prepared a sequencing test so we could see if a mutation had taken place </P><P><U>September 10, Michael Abboud</U>Sewing PCR: Primer #3 added to Product #4,5,6 from original batch (did not go through low-melt and purified).



</P><P><U> - Garrett Jensen. 9/10/2014</U> - On Monday we decided to focus on testing the effectiveness of our CRISPR system as well as performing mutagenesis of our CRISPR so that we can submit it to the iGEM registry. These are our priorities as we have to submit the part to iGEM by mid october.  - Today we performed a 1:10 serial dilution of E coli phage T7 out to 10e-8. We will use this on Wednesday to test the effectiveness of our CRISPR system in E. coli. We are not completely sure if T7 will infect E coli Dh5a so we are going to test that as well in comparison with its normal host strain, E. coli B. 
 - Using the plates we have streaked our E. coli on that contains the CRISPR and the T7 spacer we started overnight bacterial cultures using 20 mL of LB broth with the antibiotics ampicillin and chloramphenicol. I also started overnights of DH5a and B. The CRISPR containing microbes are labelled 1 and 7.
 Today I made freezer stocks from our E. coli to preserve the strain that has the CRISPR for future use. They are in Cryo tubes in out iGEM group freezer box. 
 - Today we plated the E. coli and incubated the T7 phage with the host as well as several tests where we have done a spot test on the E. coli. The Table below details what strains we used, what concentrations of phage were used, and what type of test was performed. 

<CENTER><TABLE border="1" align="center" style="margin-top:20px; margin-left:35px; margin-right:35px"><TBODY><TR><TD>Plate Label</TD><TD>CRISPR Set T7</TD><TD>Phage Set</TD><TD>Concentration</TD></TR><TR><TD>1</TD><TD>#1	</TD><TD>original</TD><TD>1.00E+00</TD></TR><TR><TD>2</TD><TD>#1</TD><TD>original</TD><TD>1.00E-01</TD></TR><TR><TD>3</TD><TD>#1</TD><TD>original</TD><TD>1.00E-02</TD></TR><TR><TD>4</TD><TD>#1	</TD><TD>original</TD><TD>1.00E-03</TD></TR><TR><TD>5</TD><TD>#1</TD><TD>original</TD><TD>1.00E-04</TD></TR><TR><TD>6</TD><TD>#1</TD><TD>original</TD><TD>1.00E-05</TD></TR><TR><TD>7</TD><TD>#1	</TD><TD>new</TD><TD>1.00E+00</TD></TR><TR><TD>8</TD><TD>#1	</TD><TD>new</TD><TD>1.00E-01</TD></TR><TR><TD>9</TD><TD>#1</TD><TD>new	</TD><TD>1.00E-02</TD></TR><TR><TD>10</TD><TD>#1</TD><TD>new</TD><TD>1.00E-03</TD></TR><TR><TD>11</TD><TD>#1</TD><TD>new</TD><TD>1.00E-04</TD></TR><TR><TD>12</TD><TD>#1</TD><TD>new</TD><TD>1.00E-05</TD></TR><TR><TD>13</TD><TD>#1</TD><TD>CONTROL</TD></TR><TR><TD>14</TD><TD>#7</TD><TD>original</TD><TD>1.00E+00</TD></TR><TR><TD>15</TD><TD>#7</TD><TD>original</TD><TD>1.00E-01</TD></TR><TR><TD>16</TD><TD>#7</TD><TD>original</TD><TD>1.00E-02</TD></TR><TR><TD>17</TD><TD>#7</TD><TD>original</TD><TD>1.00E-03</TD></TR><TR><TD>18</TD><TD>#7</TD><TD>original</TD><TD>1.00E-04</TD></TR><TR><TD>19</TD><TD>#7</TD><TD>original</TD><TD>1.00E-05</TD></TR><TR><TD>20</TD><TD>#7</TD><TD>new</TD><TD>1.00E+00</TD></TR><TR><TD>21</TD><TD>#7</TD><TD>new</TD><TD>1.00E-01</TD></TR><TR><TD>22</TD><TD>#7</TD><TD>new</TD><TD>1.00E-02</TD></TR><TR><TD>23</TD><TD>#7</TD><TD>new</TD><TD>1.00E-03</TD></TR><TR><TD>24</TD><TD>#7</TD><TD>new</TD><TD>1.00E-04</TD></TR><TR><TD>25</TD><TD>#7</TD><TD>new</TD><TD>1.00E-05</TD></TR><TR><TD>26</TD><TD>#7</TD><TD>CONTROL</TD></TR><TR><TD>27</TD><TD>DH5a test</TD><TD>Control</TD></TR><TR><TD>28</TD><TD>DH5a Test</TD><TD>Spot test new</TD><TD>Full Strength, 10e-1, 10e-2, 10e-3.</TD></TR><TR><TD>29</TD><TD>Dh5 control</TD><TD>Spot test old</TD><TD>Full Strength, 10e-1, 10e-2, 10e-3.</TD></TR><TR><TD>30</TD><TD>B test</TD><TD>Spot test old</TD><TD>Full Strength, 10e-1, 10e-2, 10e-3.</TD></TR><TR><TD>31</TD><TD>B test</TD><TD>Spot test new</TD><TD>Full Strength, 10e-1, 10e-2, 10e-3.</TD></TR><TR><TD>32</TD><TD>B control</TD><TD>control</TD></TR><TR><TD>1</TD><TD>#1 spot test</TD><TD>Old</TD><TD>Full Strength, 10e-1, 10e-2, 10e-3.</TD></TR><TR><TD>2</TD><TD>#1 spot test</TD><TD>New</TD><TD>Full Strength, 10e-1, 10e-2, 10e-3.</TD></TR><TR><TD>3</TD><TD>#7 spot test</TD><TD>Old</TD><TD>Full Strength, 10e-1, 10e-2, 10e-3.</TD></TR><TR><TD>4</TD><TD>#7 spot test</TD><TD>New</TD><TD>Full Strength, 10e-1, 10e-2, 10e-3.</TD></TR></TBODY></TABLE></CENTER></P><P><U> Michael Linzey </U>-We worked as a team and set up the CRISPR test
-So I was involved in everything that Garrett talked about above </P><P><U> Garrett Jensen - 9/11/2014</U> - Our plates had some interesting results. The controls did not grow, but that was because we plated both strain B and DB5a on Cam+Amp plates. The dilute phage plates did not have much growth. T7 phage pretty well wiped out the E. coli. The spot test plates also had big plaques. The interesting result was that there were many colonies growing inside of the plaques on every single plate.  - The lack of CRISPR function could be attributed to a number of different things: We could have designed the T7 spacer region wrong, the T7 phage we used may have a mutation where we chose our spacer, the CRISPR could be non functional. 
 - My hypothesis to explain these colonies is that they grew up from the cells that managed to create their own spacer for T7. This type II CRISPR is adaptive and a possible explanation for our results is that it did adapt to the phage infection. 
 - The way for us to tell if this is true is to redo the experiment using E. coli from these colonies and run DH5a with the empty iGEM plasmid pSB1C3 as a control. If we do not get plaques on the first and do not see similar results to our last test then we can be confident that our CRISPR is working!
</P><H2>Week of September 14</H2><P><U> Michael Linzey 9/15/2014 </U>-I got the sequencing results back today and I spend an hour or so trying to figure them out
-I consulted with Dr. Grose and we determined that the forward primer didn't work and I needed to redo the experiment
-In order to do this I was going to need more CRISPR plasmids that had their restriction sites mutated. Luckily I had streaked the bacteria onto new plates so I had ample bacteria to use. I then used the Plasmid Purification Kit to extract the purified plasmid. </P><P><U>September 15, Michael Abboud</U>Ran last sewing PCR on gel, there was a problem with the gel.  No bands were found even from the ladder, so I suspect the gel had no ethidium bromide.


</P><P><U> Garrett Jensen - 9-16-2014</U> - Yesterday the CRISPR would not grow up. I inoculated the LB/Amp/Chloro broth twice but it never grew up. I thought it might be the broth so I tried growing it up in just LB/Chloro broth. Today that had worked. It seems a little weird that the CRISPR would grow up on a Chloro/Amp plate but not in the broth. I think that something is wrong with our broth. So today I made up a new batch and I am trying to grow up our CRISPR again. 
 - Today I did plate the CRISPR that grew up and tested it. I have spot tested 5 ÂµL of T7 dilutions from 10e-3 -&gt; 10e-10. As a control I have run Dh5a and the empty iGEM plasmid pSB1C3 with T7 phage infection to ensure that the plasmid is not conferring phage resistance. Cross your fingers!!!
</P><P><U>Michael Linzey 9/17/2014 </U>-Today I spent some time preparing plasmids for sequencing. I put two ul of plasmid with one ul of a forward or reverse primer. I brought these up to another lab where they will be put through a DNA sequencer. This will allow us to see if the mutations worked. 
- I also helped Garrett set up the next round of test for the CRISPR system, this mostly included trying to understand the results that we got and setting up another spot test. </P><P><U> Garrett Jensen 9/18/2014</U> - The test from Yesterday Didnt work. The plates with the phage on it were a little weird. There was only growth on half of the plate but not the other half. So I redid the experiment. Hopefully tomorrow we can see something. 
 - To make sure that its recorded, the small colonies that I tested from the plaques earlier did not have the CRISPR. I did a control experiment testing Dh5a and the iGEM plasmid pSB1C3. Both of these also gave the small colonies inside of the plaques. So I went back to the start and tested again. 
</P><P><U> Garrett Jensen 9/19/2014</U> - So today I got the results of the CRISPR test. Unfortunately it did not work. The plaques on the CRISPR plate were smaller than those from the iGEM plasmid control, but there were plaques. 
 - Talking with Dr. Grose the only things that we did differently than the paper we referenced for our work is that we put our guide region and CRISPR on different plasmids. We are going to try just cloning the entire CRISPR in and see if that works. I designed a new primer today to do that. The new primer is 5'-TACTAGTAGCGGCCGCTGCAGGCTGGATATTCGTATAACATGTTCCTTCTC-3'and is the reverse primer to remove the entire CRISPR set from LMD-9. Dr. Grose ordered it today and hopefully it will work!!
</P><H2>Week of September 21</H2><P><U> Garrett Jensen 9/21/2014</U> - Today I re read through the paper about the CRISPR we have used  (http://nar.oxfordjournals.org/content/39/21/9275.long#ref-14)  We have used the same protocol as they did so the problem is not with our experimental setup. 
 - This means that there could be a problem with our T7 spacer. We may have chosen the wrong DNA target for the CRISPR to target, also we could have chosen a region of the genome that has mutated. 
 - Reading through the paper I did notice that they said that not all cells that have a CRISPR are immune, but those that do acquire their own spacers will be immune to phage infection. We would expect this to give the small colonies that grow up after the infection. Since DH5a gives these naturally we cannot test these because most of them would be from the host bacteria. We are going to test E. coli strain B and see if it produces these satellite colonies. 
 - Today I started a plate of strain B and an overnight culture to transform the CRISPR and T7 spacer and also test to make sure that strain B doesnt produce satellite colonies. 
</P><P><U>September 22, Michael Abboud</U>Ran last sewing PCR on gel, this time it worked and product found.

</P><P><U> Michael Linzey 9/22/2014 </U>-Got the sequencing results back, they still did not work, there just had to be something wrong with the forward primers I have been using.
- This led me to a discussion with Dr. Grose, we discussed what could be going wrong. 
-We decided to try again but we needed to do grow up a liquid culture of the E Coli overnight and do a plasmid prep.
-I started an overnight liquid culture. </P><P><U> Garrett Jensen 9/23/2014</U> - Today Dr. Grose and I electroporated E. coli strain B and transformed it with the CRISPR and T7 spacer. I plated it on an LB+CAM+AMP plate and tomorrow we will be able to see if it worked. As a control we transformed the T7 spacer with the empty iGEM plasmid. Today I also plated E. coli B and spot tested dilutions of T7 phage out to 10e-10. Tomorrow I will be able to see if strain b gives the satellite colonies. Tomorrow I will also test the CRISPR in strain B and compare it to the empty iGEM plasmid. Hopefully we see some results!!!!
</P><P><U> Michael Linzey 9/23/2014 </U>-I did a plasmid prep and purified the mutated plasmid. </P><P><U>September 24, Michael Abboud</U>Sewing PCR: Primer #2 added to Product #3,4,5,6.  Ran gel and found product.

</P><P><U> Michael Linzey 9/24/2014 </U>- I prepared the DNA for sequencing
- I assisted other groups as they needed help

</P><H2>Week of September 28</H2><P><U> Michael Linzey 9/29/2014 </U>-Checked the sequencing results again...still nothing.
-I talked with Dr. Grose and she looked at the first primer I used, which I only tried once and suggested to try it again one last time. 
-So I prepared another round of DNA for sequencing. 
-Helped prepare a phage spot test with Garrett to test out our CRISPR system. </P><P><U> Michael Linzey 10/1/2014 </U>-The CRISPR system did not work.
-We looked at the T7 spacer and compared it to the spacer that the researchers in the paper that we have been studying from used.(http://nar.oxfordjournals.org/content/39/21/9275.long#ref-14) We realized that the T7 spacer was not right and that in the paper they put the CRISPR system and the spacer on the same plasmid. So we worked with Dr. Grose to create new primers and hopefully we will be able to get this new system test and characterized soon. </P><H2>Week of October 5</H2><P><U> Garrett Jensen 10/07/2014</U> - I requested a part number from iGEM for our mutated CRISPR. It is BBa_K1356001.   We will submit the CRISPR to iGEM using this part number. 
 - Today I performed a plasmid prep. on the 8 colonies we amplified from the mutagenesis. I also prepared a sequencing reaction for each of these and submitted them today for sequencing to make sure that the mutagenesis worked. They were submitted in the following order:  1A,1B,3A,3B,2A,2B,4A,4B. When we get them back from the sequencing center we will continue doing mutagenesis to see remove all of the forbidden restriction sites!
 - Today I also started a new PCR to amplify the CRISPR with the Spacer region as well. This will also put in a BamHI site that we can clone other spacers into for future use. There are 5 PCR reactions and 1 control. 
</P><P><U> Michael Linzey 10/11/2014</U>-This week I spent a lot of time working on creating the t7 spacer. We have run several pcr reactions but when we run them out on low melt gels our results have been inconclusive. We have run this experiment several times and it has not been working so we are going to work more closely with Dr. Grose.

</P><H2>Week of October 12</H2><P><U> Garrett Jensen 10/12/14</U> - This week we did a soeing PCR for the new T7 spacers, one for going on a separate plasmid fromt he CRISPR and the other that we will insert into the existing CRISPR spacer region. None of these reactions has worked so far. We are continuing to set these up and hopefully it works soon. Michael set up a reaction yesterday for the T7 spacer that will go into the CRISPR system yesterday that we have not yet run out on a gel. 
 - We ran a PCR of the S. thermophilus gDNA to amplify a set of the CRISPR that has part of the spacer region attached. Dr. Grose engineered in a restriction site into the middle of an existing spacer that we can use to clone additional spacers into the CRISPR. We were planning on cutting off the end of the CRISPR system at the SnaBI site present in the CRISPR and then ligating that last little bit inluding the spacer region into our existing CRISPR. Unfortunately, after several attempts at cutting our existing CRISPR system using the SnaBI site that it supposedly there we have been unsuccessful. Our sequencing fo the CRISPR did not have good coverage of that area of the DNA, and after running our product out on a gel the band is the size we expect the uncut CRISPR plasmid to be. My hypothesis for this has been that our restriction site has a mutation and cannot be cut by SnaBI. 
 - After talking to Dr. Grose we are just going to use the new CRISPR system that we have done PCR with instead of the old one. Hopefully we can get this to work soon!!!
</P><P><U>Michael Linzey 10/14/2014 </U>- Worked with Dr. Grose and set up another sewing PCR. This will create two T7 spacers. One will be put on a separate plasmid and the other will be ligated directly into the CRISPR system repeat. 
-After the PCR ran we used XBAI and SPEI to cut these fragments of DNA. 

</P><P><U> Michael Linzey 10/15/2014 </U>-I ran a low melt gel to remove the restriction digest enzymes from our product.
-We realized that we may have used the wrong primers to cut one of the t7 spacers. We should have cut one with BAMHI and PSTI. 




</P></BLOCKQUOTE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BYU_Provo/Notebook/CRISPR/septoct">http://2014.igem.org/Team:BYU_Provo/Notebook/CRISPR/septoct</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BYU_Provo/Notebook/CRISPR/septoct" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BYU_Provo/Notebook/CRISPR/septoct" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CRISPR/septoct&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CRISPR/septoct&amp;oldid=354485" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>