<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BYU_Provo_Notebook_Biofilm_mayjune"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BYU_Provo/Notebook/Biofilm/mayjune">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BYU_Provo/Notebook/Biofilm/mayjune&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/mayjune&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/mayjune&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BYU_Provo/Notebook/Biofilm/mayjune" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BYU Provo/Notebook/Biofilm/mayjune</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1 style="color:#FFFFFF">BYU 2014 Notebook </H1><P style="color:#FFFFFF"><A href="https://2014.igem.org/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/mayjune&amp;action=edit" style="color:#FFFFFF"> Edit May June</A></P><TABLE width="100%"><TBODY><TR heigth="30px"><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo" style="color:#002255">Home </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Team" style="color:#002255"> Team </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=BYU_Provo" style="color:#002255"> Official Team Profile </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Project" style="color:#002255"> Project</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Parts" style="color:#002255"> Parts</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Modeling" style="color:#002255"> Modeling</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Notebook" style="color:#002255"> Notebook</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Safety" style=" color:#002255"> Safety </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Attributions" style="color:#002255"> Attributions </A></TD></TR></TBODY></TABLE><BLOCKQUOTE><H2>1 May 2014</H2><P>Aiia: Today a plasmid prep of the Aiia plasmid was performed following the Denville SpinSmart Plasmid Purification protocol.</P><P>Alpha Amylase: Today sewing PCR was performed on the alpha amylase forward primer pieces. In order to PCR with the overlap extension primers that contain our signal sequence we will need to PCR with just the forward and reverse with the signal sequence and no template so that we have a combined forward primer to use. Then we will use that combined primer as a forward primer in a normal Q5 PCR reaction with the template. The only difference is that 2 uL of each primer should be used (the forward signaling sequence primer and the reverse signaling sequence primer). (JB)</P><P>Dispersin B: On Tuesday we unfortunately did not PCR correctly with our primers. Our primers exceeded the length that the company could provide so we actually will need to perform a “sewing PCR” reaction first to generate a forward primer that contains the signaling sequence and our first 20 nucleotides. This necessitated a repeat of the protocol followed on Tuesday for the PCR but this time using  1 uL of the Forward primer with SS and 1 uL of the Reverse primer with SS. We also set this up for Aiia. (JM)</P><H2>2 May 2014</H2><P>Today we ran our sewing PCR products on agarose gel and each got a product around 150 base pairs as it should have if the reaction worked properly. We then set up the next reaction with the newly sewn forward primers with the signaling sequence and the reverse primers for each gene as well as the template. We have a 5 uL control for each and then a reaction for each of the parts (including the two Amylase parts we are testing). The tube labeled Amy1 is pIG 92 and the tube labeled Amy2 is pIG98.</P><H2>5 May 2014</H2><P>Today we ran the gel of our PCR products of the enzyme plasmids with our sewn forward and regular reverse primers.</P><P>Aiia had a product that was around 2500 bp, where is should have only been around 800 so we need to figure out if it amplified the whole plasmid.</P><P>DispB showed smearing so either we need to play with the annealing temperatures because the primers are either not attaching at this temperature we used or the plasmid needs to be purified more.</P><P>Amy2 was the best band. This was from PIG98. We will need to digest this and the backbone and run it on low melt gel.</P><BLOCKQUOTE>Left: Gel image of the sewing PCR reaction (Lane order: 5 kb ladder, Alpha Amylase, Aiia, DispersinB)</BLOCKQUOTE><BLOCKQUOTE>Right: Gel image of finished gene+signaling sequence PCR products (Lane order: 5 kb ladder, Aiia, Amylase1/pIG92, DispersinB, Amylase2/pIG98)</BLOCKQUOTE><H2>6 May 2014</H2><P>Alpha Amylase: Today I purified the PCR product from the Alpha Amylase pIG98 plasmid (Amylase2) since it appeared to have the more solid band of the two on the gel. I then prepped a vector and insert digest from the purified PCR product and the iGem backbone. Thursday I will run it out on a low melt gel in order to perform a ligation of the two together. (JB)</P><P>We also ran our newly Pcr’ed primer out on Gel, Alpha Amylase looked like the vector was successfully produced but for Dispersin and Aiia there bands in our PCR smeared or were very faint respectively and so to be safe we decided to re-PCR those primers again from the beginning. 
Cam and I again followed the Sewing PCR protocol while Jordan did a ligation and digest of his inserts and vector plasmid.
Sewing PCR protocol<UL><LI>-24.5 uL of ddH2O</LI><LI>-10 uL Q5 enhancer</LI><LI>-10 uL Q5 buffer</LI><LI>-2 uL F primer with signal sequence</LI><LI>-2 uL R primer with signal sequence</LI><LI>-1 uL dNTPs</LI><LI>-.5 uL Q5 polymerase</LI>
(JM)</UL></P><H2>8 May 2014</H2><P>Alpha Amylase: Today I ran the low melt gel of the psb1c3 backbone and AA insert. It turned out well and I was able to see that a segment corresponding to the length of the RFP gene was separated from the plasmid. I followed our standard ligation procedure and will allow it sit overnight in order to get a better ligation. Tomorrow I will transform it into E. coli. (JB)</P><P>Dispersin B: Today Cam and I finished PCRing our primers. We also ran out a gel to check to see if our our sewing PCR reaction set up worked.(JM)</P><UL>We created a reaction mixture: 
<LI>24.5 uL ddH_2O</LI><LI>10 uL Q5 buffer</LI><LI>10 uL Q5 enhancer</LI><LI>1 uL newly made sewing PCR primer</LI><LI>1 uL Reverse Primer</LI><LI>1 uL dNTPS</LI><LI>.5 uL Q5</LI></UL>
We then took 5 uL of this mix for a control 
To the remaining mixture we added 1 uL of our template DNA and PCRed the reaction for 2 &amp; ½ Hours.
(JM)<H2>10 May 2014</H2><P>Alpha Amylase: Today the Alpha Amylase insert was ligated into the iGem plasmid vector and this was then transformed in DH5α following the standard procedures for ligation and transformation. (JB)</P><H2>13 May 2014</H2><P>Alpha Amylase: The E. coli that I transformed last weekend turned out really well. There were a lot of colonies covering the entire plate and only a few red colonies (where the RFP gene had not been excised meaning the insert did not properly work in these colonies). Today I picked eight colonies from the plate and mixed each colony that I picked into 50 uL of ddH2O, then wiped that pick on a new plate in its designated spot (numbered 1-8 to correspond with the tubes). I then boiled the tubes and the DNA acquired from these tubes will serve as template for colony PCR. I then prepped RedTAQ PCR and placed each template in its respective tube along with a control. I will run these on gel tomorrow to test the colonies in order to discover whether or not the insertion/transformation worked. I followed the standard RedTAQ procedure for the colony PCR and used 2 uL of boiled colonies as template.  (JB)</P><P>Dispersin B: Unfortunately with the reactions that we ran again last week the banding in the gels for Aiia and Dispersin B didn’t show the results we wanted. It appeared that while our forward and reverse primers with the signal sequence had annealed successfully, when that product was PCR’ed as the total former primer along with the reverse primer for the target DNA sequence of Dispersin or Aiia, we didn’t get banding that showed binding to the template. So today we attempted a new procedure. We placed the forward primer with the signal sequence the reverse primer signal sequence and the and the reverse primer for the target sequence as well as the template in a single PCR reaction (RXN 2). We also tried again to get binding from our pre-combined forward primer and the reverse (RXN 1). We ran a control for RXN 2 without Q5 polymerase, and we ran a control for RXN without template DNA. We will be running these on 2% agarose gel to better ascertain if we are getting the banding patters we need. Specifically to tell if our primers are annealing properly. 
</P><P>
RXN 1 Set up
-24.5 uL DDH2O
-10 uL Q5 Reaction Buffer
-10 uL Q5 Enhancer
-1 uL combined Forward and Reverse Primer with SS
-1 uL Reverse primer 
-1 uL dNTPs
-1 uL Dispersin B Template * Control reaction had no Template
.5 uL Q5 Polymerase
</P><P>
RXN 2 Set up
-23.5 uL DDH2O
-10 uL Q5 Reaction Buffer
-10 uL Q5 Enhancer
-1 uL Forward Primer with SS
-1 uL Reverse Primer with SS
-1 uL Reverse primer
-1 uL dNTPs
-1 uL Dispersin B Template
.5 uL Q5 Polymerase * Control reaction had no Q5
</P>
(JM)<H2>14 May 2014</H2><P>I ran the gel from the RedTAQ PCR yesterday and there were no bands from any of the boiled samples or control so I re-prepped a new batch of PCR and ran it. I will run the gel tomorrow. However, all of the swabs of the colonies on the plate produced good streaks. (JB)</P><H2>15 May 2014</H2><P>Alpha Amylase: Below is the image from the gel run to verify the presence of the Alpha Amylase plasmid in the colonies. Again, no bands showed up indicating Alpha Amylase. Today I retransformed the Alpha Amylase ligation into E. coli so that I can make more colonies. I will need to then repeat the steps of colony PCR. Desi suggested picking from 16 colonies this time for a better chance of getting a colony that will work. She also suggested using a positive control alongside the negative control, which would be the plasmid with alpha Amylase itself. This will help ensure that all the reagents are working. (JB)
</P><P>
Dispersin B:
After running Tuesday’s work out on a gel we were able to see some annealing happen but it appears that only our forward primers annealed in both reactions and we didn’t get any binding to the template. So we are now directing our efforts to resolving that issue. Cameron and I each set up 6 reaction mixtures, three controls and three tests. The tubes will be labeled with the numbers 1-6 and will have either D or A referencing Dispersin or Aiia respectively.

</P><P><UL>
All six tubes have 
	<LI>-26.5 uL ddH2O</LI><LI>-10 uL of Q5 enhancer</LI><LI>-10 uL of Q5 reaction buffer</LI><LI>-1 uL of dNTPs</LI><LI>-.5 uL of Q5 polymerase</LI><LI>-1 uL of Reverse Primer</LI><LI>Tubes 1,3,5 have 1 uL of dispersin B template</LI><LI>Tubes 2,4,6 are controls without template</LI><LI>Tubes 1,2 have 1 uL from RXN 1 set up on Tuesday acting as our forward Primer</LI><LI>Tubes 3,4 have 1 uL from RXN 2 set up on Tuesday acting as our forward Primer</LI><LI>Tubes 5,6 have only the Reverse Primer with SS acting as the forward Primer (the back half of the overall forward primer with signal sequence)</LI>

Tubes 5,6 are to see if we can get any annealing to the template DNA with our primers, by leaving out the full Signal Sequence we hope to increase the likelyhood that the primer piece meant to bind to the template will do so. 
(JM)</UL></P><H2>19 May 2014</H2><P>Alpha Amylase: Some good news finally - I froze down my DH5α + ligation transformation from last time so I was able to save some time and just replate some of that. Tomorrow I will pick from colonies and do a 16-colony PCR to see if the gene was inserted into the DH5α. Dr. Grose said it was likely issues with PCR that caused the lack of bands last time since many of the colonies were no longer red last time which indicates that the transformation did work. (JB)</P><H2>20 May 2014</H2><P>Alpha Amylase: The E. coli colonies from yesterday turned out really well. There were not as many colonies as last time but there was only one colony that I could see that was a pinkish color. I prepped colony PCR from 16 different colonies which I labelled on the plate. I then used those picks as template and streaked them in their respective quadrants. I will run the 16 colonies and control tomorrow on gel after PCR. (JB)</P><P>Dispersin B: The gel results from last Thursday were cloudy, but unfortunately it appeared that we didn’t get the band sizes that we needed. As a double check Cam and I ran the products again, along with a control of just the plasmid itself, because the only bands that appeared to be showing up were approximately the size of our plasmid. While this was running Dr. Grose pointed out there was significant smearing in the lanes for Dispersin on the cloudy gel image and she suggested it may be because we had a dirty plasmid prep so she had me perform a transformation into a new batch of E. Coli which were plated on Ampicillin and Kanamycin plates. 
Desi provided me with the sequenced DspB gene and told me to check it for errors  between it and the NCBI and to see if my primers matched the actual sequence (JM) </P><H2>21 May 2014</H2><P>Alpha Amylase: The colony picks all grew up well overnight. Colony 13’s PCR tube burned out - I think the cap had been mashed up a bit when I was trying to put the lid on so it probably did not close all the way.</P><P>No bands with PCR again. Desi suggested running a positive control this next time. This will consist of the Alpha Amylase plasmid with the Amylase reverse and Amylase forward signaling sequence if it has Alpha Amylase in it. If not I will need to grab another forward from the registry. (JB)</P><H2>22 May 2014</H2><P>Alpha Amylase: I will be using BI259 as an Alpha Amylase forward for a positive control.</P><P>Today I repeated the setup for colony PCR. As template I just reused colonies 1-8 that I had picked and streaked the other day. I made sure everything was thoroughly mixed together this time and that the polymerase was added only shortly before the PCR cycles were begun. I also created a positive control using the PIG98 plasmid and the BI259 forward primer. This primer was used in all the mixes for this round in order to determine if the Alpha Amylase was successfully transformed into the DH5α. (JB)</P><P>
After checking the sequenced DspB gene I had some concerns that the primers were incorrect but it was explained to me as an error in the sequencing machine code and didn’t constitute a discrepancy between my primers and the sequence. 
The new gel that Cam and I ran on Tuesday showed conclusively what we thought was happening with the cloudy gel results, namely that the bands we were seeing were in fact our plasmids and that our primers weren’t annealing. Since the control bands (those with just the template) were clear and showed no smearing it also meant that the plasmid prep I was working with for Dispersin B was not impure. 
Dr. Grose was fearful that we weren’t being careful enough in setting up our reaction mixture so Cam and I each ran control alongside our reaction today with primers that target the pSB1C3 plasmid and cover the insert area. We should get a product about 400 base pairs larger than our desired insert size in the control experiment . The idea being that these primers are known to work so if we don’t obtain a product then it means that we aren’t setting up our reaction mixture correctly. We also are doing it without the Q5 enhancer, and we ran a primer clean up reaction to purify our forward primers.
(JM)</P><H2>26 May 2014</H2><P>Alpha Amylase: I ran the PCR from last week on gel. Same thing happened as before where there were no bands on the template runs. There was faint banding on the positive control so we know the reaction is working at least in a weak manner. There also appears to be multiple bands in the positive control. I will need to discuss this all with Desi tomorrow during class in order to determine what is going on. (JB) </P><H2>27 May 2014</H2><P>Alpha Amylase: Since the RedTAQ did not work again so we ran Taq and RedTaq PCR side by side from templates 9-16 from my colony plate in order to see if the RedTAQ polymerase is faulty. And it worked with regular TAQ polymerase! Now I will start overnights from two of the colony streaks, do a plasmid prep tomorrow, sequence it, and then I can start mutagenesis! I started overnights from colonies 13 and 15 (gel positions 5 and 7 respectively on the lower well row). (JB)</P><P>Today Cam and I ran our PCR products from last Thursday out on a gel. I was pleased to see exactly what I was looking for. The gel showed a band that was between 1,000 and 1,500 bp long, which is the size I was anticipating for the dispersin B gene with attached signal sequence.  As a further confirmation the control experiment showed a band that was a couple hundred basepairs larger meaning that the plasmid targeting primers properly worked and copied the gene as well. With this new evidence I got the go ahead from Dr. Grose to make preparations for my digestion and insert. 
Using the sigma PCR cleanup kit I purified my PCR product.
(JM)</P><H2>28 May 2014</H2><P>Alpha Amylase: Today I worked on the plasmid prep from my transformed Alpha Amylase gene with signaling sequence. I also prepped for sequencing tomorrow. Below are what each PCR tube contains:</P><BLOCKQUOTE><P>1. Colony 13 Plasmid + AA w/ SS F</P><P>2. Colony 13 Plasmid + AA R</P><P>3. Colony 13 Plasmid + psB1C3 F (307)</P><P>4. Colony 13 Plasmid + psB1C3 R (308)</P><P>5. Colony 15 Plasmid + AA w/ SS F</P><P>6. Colony 15 Plasmid + AA R</P><P>7. Colony 15 Plasmid + psB1C3 F (307)</P><P>8. Colony 15 Plasmid + psB1C3 R (308)</P></BLOCKQUOTE><P>For sequencing prep, add 2 uL plasmid and 1 uL either forward or reverse primer to PCR tube and enter it on the sequencing spreadsheet downstairs in Dr. Grose's main lab, then put the tubes in the sequencing tube holder in the order you entered them in in the spreadsheet. (JB)</P><H2>29 May 2014</H2><P>Today a majority of the time was spent going over the grant proposal for SYNENERGENE.</P><P>Dispersin B: Today the class time was spent revising and editing a grant proposal. However, I came back into class in the late afternoon and was able to set up a restriction enzyme digest for my insert. I plan on using some of Jordan’s plasmid vector digest for my ligation on Tuesday since his project has already advanced and it is confirmed that his digest and ligation worked. That way I only have to worry about half of the reaction.  I put the digestion in the incubator overnight and Jordan said that he would be taking it out for me tomorrow.(JM)</P><H2>3 June 2014</H2><P>Alpha Amylase: It appears the cloning sites hybridized together and there is no amylase found in the plasmid. We talked to Skip about what to do and he suggested re-digesting the pSB1C3 vector and adding phosphatase to the digest in order to prevent the Xba and Spe sites from hybridizing again. </P><P>Procedure:</P><P>Prep vector digest as normal, allow to incubate for 1 hr at 37 deg.</P><P>Add 1 uL phosphatase (CIP) and allow to incubate for 30 mins at 37 deg</P><P>Run on low melt gel and purify</P><P>Also, a possible issue that may have occurred is the primers in the colony PCR were binding to the amylase that is naturally occurring in E. coli and that may be why we saw bands in there. In order to try and solve this we are going to redo colony PCR using the plasmid specific primers instead. This way, if our modified plasmid was successfully transformed into the E. coli we will be able to determine this. (JB)</P><P>Dispersin B: It was discovered that Jordan’s vector digest might have not been as effective as it could have been and that it would need to be re-set up. The good news is that I hadn’t yet begun my ligation so this didn’t set me back at all. Today I simply set up a low melt gel and ran it for purification of my insert. After the gel was run a UV light was used to locate my purified product which again matched the appropriate size for Dispersin B when compared with the ladder. After it was located the product was cut out of the gel and frozen.(JM)</P><H2>4 June 2014</H2><P>Alpha Amylase: Ran the gel from yesterdays PCR of the transformed E. coli colonies with the Amylase plasmid using the plasmid primers. All of the colonies were showing 400 bp fragments which means the primers were just cutting out the plasmid in that region and there was no amylase there. This means that most likely last time the PCR results were in fact showing amylase but that which naturally is produced by <I>E. coli</I> and not from the transformation. Today I did a plasmid prep from the <I>E. coli</I> DH5α with pSB1C3 transformed into it. 100 uL were eluted and we performed a nano-drop to determine the concentration of the elution. The concentration was 154.4 ug/uL which is average. I then performed a restriction digest of the vector and changed the recipe to use 30 uL of vector and 12 uL of ddH2O since the concentration of the vector was just average. At 1 hr into the digest, CIP will be added, then after 90 minutes of digest the vector solution will be run on low melt gel and isolated. Then all of this will be ligated and we will be able to transform this into DH5α tomorrow. (JB)</P><H2>5 June 2014</H2><P>Alpha Amylase: Today I transformed the ligation into DH5α. I found out that I mis-prepared my ligation mix yesterday though so I re-prepped a new ligation mix, let it sit for about an hour, and then transformed this into DH5α as well. Both are labeled and we will see which turns out. (JB)</P><P>Dispersin B: A new batch of vector was made and digested between the interim of Tuesday and today which I used for my ligation reaction. (JM)</P><H2>9 June 2014</H2><P>Alpha Amylase: Plates from over the weekend of the transformed DH5α turned out okay. The 1st ligation (not 100% according to recipe) showed a good amount of colonies and there were several pink colonies. The 2nd, correct ligation only had 5-10 colonies but none of them were pink. I ran PCR on a selection of colonies from both ligation transformations and here are the gel results:</P><BLOCKQUOTE>Gel well order: 5 kb ladder, negative control, positive control, colonies 1-4 from first ligation, colonies 5-8 from second ligation.</BLOCKQUOTE><P>Expected band would be around 2000 bps (1500 bps Alpha Amylase + 100 bps signaling sequence + 200 bps + 200 bps from plasmid primers) and colonies 6 and 8 appear to be the correct length and very strong. I will send these two colonies for sequencing. (JB)</P><H2>10 June 2014</H2><P>Alpha Amylase: Labeling of PCR tubes for sequencing:</P><LI>AA6a: Colony 6 Plasmid + Alpha Amylase with forward signaling sequence primer</LI><LI>AA6b: Colony 6 Plasmid + Alpha Amylase reverse primer</LI><LI>AA6c: Colony 6 Plasmid + psB1C3 forward primer (307)</LI><LI>AA6d: Colony 6 Plasmid + psB1C3 reverse primer (308)</LI><LI>AA8a: Colony 8 Plasmid + Alpha Amylase with forward signaling sequence primer</LI><LI>AA8b: Colony 8 Plasmid + Alpha Amylase reverse primer</LI><LI>AA8c: Colony 8 Plasmid + psB1C3 forward primer (307)</LI><LI>AA8d: Colony 8 Plasmid + psB1C3 reverse primer (308)</LI><P>I added 2 uL plasmid and 1 uL either forward or reverse primer to each PCR tube.</P><P>Tubes were sent for sequencing today. (JB)</P><P>Dispersin B: My ligation and transformation seems to have worked as I had some colony growth on my plate from last week. There were some pink colonies and some white ones. Today I took 6 of those colonies and ran a PCR to check to make sure I have an insert band that is the right size so that we can send the DNA off for sequencing next week.
(JM)</P><H2>12 June 2014</H2><P>Alpha Amylase: Sequencing looked fairly good. Need to be sure to pipette carefully the full 2 uL of plasmid and 1 uL of primer, make sure no drops left on side of tube or in pipette cap. We will have to see this next round of sequencing if there were in fact no unintentional mutations. (JB)</P><P>Dispersin B:Today I took the results of the colony PCR and ran them out on gel. 4 of the colonies showed banding of the appropriate size ~1500 bp. I selected two of the bands (A and F) that were the most clear and using the corresponding labeled colonies on the second Cam plate I prepared on Tuesday, I set up liquid colonies so that we could have things ready to sequence next week. (show gel picture)

I prepared two liquid colony tubes using 5mL of LB with Cam each and swabs from the A and F colonies. These were then placed into the 37 degrees C incubator and allowed to grow overnight. They will be pelleted, removed, and frozen, to be used next week for sequencing.
(JM)</P><H2>17 June 2014</H2><P>Alpha Amylase: Plating of the transformation of Alpha Amylase mutant showed no colonies. Today I am going to plate the other 450 uL of the previous transformation and re-transform DH5α with 5 uL of Alpha Amylase mutant product this time and then plate all 550 uL of the transformation.</P><P>Below are pictures of my colonies and transformation plate that I used in preparation for mutagenesis. The four colonies on the bottom are the colonies that were used. (JB)</P><P>Dispersin B: The bacteria pellets were re-suspended and the plasmids were purified using a plasmid purification kit. The following mixtures were then set up to be sent off for sequencing.
Tube labels:

DspB 1A—contained 2 uL of Colony A purified plasmid and 1 uL of forward ss DspB Primer 
DspB 2A—contained 2 uL of Colony A purified plasmid and 1 uL of reverse DspB primer
DspB 3A—contained 2 uL of Colony A purified plasmid and 1 uL of 307 forward primer
DspB 4A—contained 2 uL of Colony A purified plasmid and 1 uL of 308 reverse primer
DspB 1F—contained 2 uL of Colony F purified plasmid and 1 uL of forward ss DspB Primer
DspB 2F—contained 2 uL of Colony F purified plasmid and 1 uL of reverse primer
DspB 3F—contained 2 uL of Colony F purified plasmid and 1 uL of 307 forward primer
DspB 4F—contained 2 uL of Colony F purified plasmid and 1 uL of 308 reverse primer

These will be sent off for sequencing.
(JM)</P><H2>18 June 2014</H2><P>Alpha Amylase: There were not any colonies on either of the transformed mutation plates. (JB)</P><H2>19 June 2014</H2><P>Alpha Amylase: Today I worked on the background and team info for our iGem wiki and researched what other teams have done in the past for their wikis. (JB)</P><P>Dispersin B: Still waiting on sequencing, I helped Cam set up a liquid culture for one of his lines of bacteria.(JM)</P><H2>20 June 2014</H2><P>Alpha Amylase: Concentrations of plasmid preps for mutagenesis were very low - in the thirties. But at least they were very clean samples. Desi said it was surprising that we got such good sequencing from it since it usually doesn't sequence that well at such a small concentration. She suggesting starting a liquid overnight again from the colonies, let it incubate for 16-18 hours, pellet all of the bacteria out for use in prep, elute into 50 uL and make sure it all goes on the filter. (JB)</P><H2>23 June 2014</H2><P>Alpha Amylase: Finished the plasmid prep of colony 8 and had a concentration of 375 ng/uL and the purity readings were around 1.8 so it was decent.</P><P>We will prep and run mutagenesis today and transform tomorrow. (JB)</P><P>Today was spent working on a semester progress review.(JM)</P><H2>26 June 2014</H2><P>Alpha Amylase: Digest did not work. Desi mentioned that you can't always add the DpnI by itself. Sometimes it is helpful to mix 10 uL PCR product, 11.5 uL ddH2O, 2.5 uL Cutsmart buffer, and 1 uL DpnI. (JB)</P><P>Dispersin B: After to listening to everyone else’s progress and projects it seems like things are moving forward for most of the groups. The crispr system group is understandably running into more obstacles as their project is likely the hardest with the metabolism group right behind them. 

We have ordered N. Multiformis from a company since the samples we receive from the University of Utah are not growing. Once that grows the auxotrophy group should be on their final stage. Similarly one of the antibiotic breakdowns systems is complete. So things are getting to the stage where our groups will start to merge as we help each other complete our respective projects.
(JM)</P><H2>27 June 2014</H2><P>Alpha Amylase: Added DpnI to the PCR product for the digest of any plasmid that was not mutated properly and placed it in the 37 degree incubator. We will transform this digested plasmid into DH5a tomorrow. (JB)</P><H2>30 June 2014</H2><P>Alpha Amylase: Today the colony 8 PCR product for the Alpha Amylase mutant was transformed into DH5a using the basic transformation protocol as listed in the “Protocols” page. Tomorrow colonies with mutant Alpha Amylase pSB1C3 that survived the digest will be isolated and grown, then overnights can be made from these colonies for further work.</P><P>I also did some research for grants that we could apply for in order to receive more funding. Below are some links that may be useful to the team and others. (JB)</P><BLOCKQUOTE>http://www.werf.org/i/Funding/Open_RFPs/a/o/rfp.aspx?hkey=05bda2a1-23af-4891-badf-815b2960d4f3</BLOCKQUOTE><BLOCKQUOTE>http://www2.epa.gov/education/environmental-education-ee-grants</BLOCKQUOTE><BLOCKQUOTE>http://www.nifa.usda.gov/fo/waterquality.cfm</BLOCKQUOTE><BLOCKQUOTE>https://experiment.com/start </BLOCKQUOTE></BLOCKQUOTE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BYU_Provo/Notebook/Biofilm/mayjune">http://2014.igem.org/Team:BYU_Provo/Notebook/Biofilm/mayjune</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BYU_Provo/Notebook/Biofilm/mayjune" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BYU_Provo/Notebook/Biofilm/mayjune" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/mayjune&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Biofilm/mayjune&amp;oldid=351742" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>