<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BYU_Provo_Parts"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BYU_Provo/Parts">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BYU_Provo/Parts&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Parts&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Parts&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BYU_Provo/Parts" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BYU Provo/Parts</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1 style="color:#FFFFFF">BYU 2014 Team Parts Database</H1><P style="color:#FFFFFF"><A href="https://2014.igem.org/wiki/index.php?title=Team:BYU_Provo/Parts&amp;action=edit" style="color:#FFFFFF"> Click here  to edit this page!</A></P><TABLE width="100%"><TBODY><TR heigth="75px"><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo" style="color:#002255">Home </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Team" style="color:#002255"> Team </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=BYU_Provo" style="color:#002255"> Official Team Profile </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Project" style="color:#002255"> Project</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Parts" style="color:#002255"> Parts</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Modeling" style="color:#002255"> Modeling</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Notebook" style="color:#002255"> Notebook</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Safety" style=" color:#002255"> Safety </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Attributions" style="color:#002255"> Attributions </A></TD></TR></TBODY></TABLE><TABLE border="4" align="center" style="margin-top:20px; margin-left:35px; margin-right:35px"><TBODY><TR><TD>BBa_K1356000</TD><TD><B>Alpha Amylase with Signaling Sequence and PstI Site Removed</B></TD><TD>DNA in pSB1C3 plasmid backbone</TD><TD>Created by: Jordan Berg</TD></TR><TR><TD colspan="4"><H3>Description</H3><P>The Alpha Amylase for this part was taken from part BBa_K1195001. Attached to it is a DsbA signaling sequence required in order to express the gene product extracellularly in <I>N. multiformis</I> in the break down of biofilm in wastewater treatment plants. Additionally, the PstI site originally found in the BBa_K1195001 part was removed using site-directed mutagenesis. The restriction site was changed from &quot;CTGCAG&quot; to &quot;CTCCAG&quot; in order to remove this site. This gene is contained within the standard iGEM pSB1C3 plasmid backbone.</P><P>Amylase is an enzyme naturally synthesized by bacteria, such as E. coli, fungi, and even in humans in saliva and the pancreas. This enzyme catalyzes the hydrolysis of starches into sugars and breaks down the components of bacterial extracellular polymeric substance (EPS), which contains extracellular DNA, polysaccharides, and proteins. These components are a primary part of most bacterial biofilms and it is hoped that the enzyme being expressed extracellularly will allow for more biofilm break down so that <I.N. multiformis=""> can more effectively breakdown wastewater components to make wastewater treatment plants more effective. It has been shown in other studies that amylase is an effective degrader of several other types of biofilms and we hope to show that it is equally effective at breaking down wastewater biofilm.</I.N.></P><H3>Design Notes</H3><P>The DsbA signaling sequence was synthesized using RNA primers overlap-extension PCR owing it the signaling sequence's large size. The mutation to remove the iGem-illegal PstI site was done through mutagenic PCR.</P></TD></TR><TR><TD><CENTER><U>Part Plasmid Schematic</U></CENTER></TD><TD><CENTER><U>RaptorX Protein Structure and Function Prediction</U></CENTER></TD></TR></TBODY></TABLE><H3>Source</H3><P>The original amylase we modified was from part BBa_K1195001 in the iGem parts registry.</P><TABLE border="4" align="center" style="margin-top:10px; margin-left:35px; margin-right:35px"><TBODY><TR><TD>BBa_K1356001</TD><TD><B>LMD-9 CRISPR 3 System</B></TD><TD>DNA in pSB1C3 plasmid backbone</TD><TD>Created by: Garrett Jensen, Mike Abboud, Michail Linzey.</TD></TR><TR><TD colspan="4"><H3>Description</H3><P>This is the Type II CRISPR3 system taken from Streptococcus thermophilus LMD-9. It is the Cas9, Csn1, Cas1, and Cas2 proteins along with the tracrRNA but . It may be used with a novel spacer/repeat region to target bacteriophage, plasmids, or any other form of incoming DNA. Cas9 is an endonuclease/exonuclease type protein and is the agent that inactivates incoming DNA. Csn1, Cas1, and Cas2 are involved in additional spacer acquisition, though the method is unknown. Cas9 can be directed by specially designed spacers or by spacers acquired by the CRISPR. The adaptive nature of the CRISPR3 makes it useful as an adaptive immune system for bacteria. It has been shown to be effective in a diverse range of microbes and can be used in any microbe.</P><H3>Design Notes</H3><P>This part has many restriction sites present. We have removed 3 EcoRI sites and 2 SpeI sites from this CRISPR so that it can be used with the iGEM plasmid. This CRISPR has also been engineered with a BamHI restriction site in the third spacer following the CRISPR protein set. This can be used to insert custom spacers into the existing spacer-repeat region.</P><H3>Source</H3><P>Streptococcus thermophilus LMD-9 genomic DNA. GenBank Accession Number: NC_008532.1
</P><H3>References</H3><P><OL><LI>Rimantas Sapranauskas, et. Al. The Streptococcus thermophilus CRISPR/Cas system provides immunity inEscherichia coliNucl. Acids Res. (2011) 39 (21): 9275-9282 first published online August 3, 2011doi:10.1093/nar/gkr606</LI><LI>Hongfan Chen, Jihoon Choi, and Scott Bailey. Cut Site Selection by the Two Nuclease Domains of the Cas9 RNA-guided EndonucleaseJ. Biol. Chem. jbc.M113.539726. First Published on March 14, 2014,doi:10.1074/jbc.M113.539726</LI><LI>Shah SA, Erdmann S, Mojica FJ, Garrett RA. Protospacer recognition motifs: Mixed identities and functional diversity. RNA Biology 2013; 10:891 - 899; PMID: 23403393; http://dx.doi.org/10.4161/rna.23764</LI><LI>&quot;Addgene: Addgene's CRISPR Guide.&quot; Addgene: Addgene's CRISPR Guide. Web. 8 Apr. 2014.</LI><LI>Krzysztof Chylinski, Kira S. Makarova, Emmanuelle Charpentier,and Eugene V. Koonin. Classification and evolution of type II CRISPR-Cas systemsNucl. Acids Res. first published online April 11, 2014 doi:10.1093/nar/gku241</LI><LI>&quot;Streptococcus Thermophilus LMD-9, Complete Genome.&quot; National Center for Biotechnology Information. U.S. National Library of Medicine, 24 Oct. 2006. Web. 8 Apr. 2014. <HTTP:>.</HTTP:></LI><LI>Choi, Jeongdong, Shireen M. Kotay, and Ramesh Goel. &quot;Various Physico-chemical Stress Factors Cause Prophage Induction in Nitrosospira Multiformis 25196- an Ammonia Oxidizing Bacteria.&quot; Science Direct. Water Research, 4 Aug. 2010. Web. 1 Feb. 2014. <HTTP:>.</HTTP:></LI></OL></P></TD></TR></TBODY></TABLE><TABLE border="4" align="center" style="margin-top:10px; margin-left:35px; margin-right:35px"><TBODY><TR><TD>BBa_K1356002</TD><TD><B>BBa K1356002</B></TD><TD>Dispersin B with DsbA Signaling Sequence</TD><TD>Created by: Jared McOmber</TD></TR><TR><TD colspan="4"><H3>Description</H3><P>Dispersin B is a hydrolase protein that aids in the degradation of biofilms. This part contains the Dispersin B gene from part BBa_K1195000, with a signal sequence attached for export from the bacteria Nitrosospira Multiformis for the purpose of aiding in the breakup of biofilms in waste water management facilities. 

A biofilm is a complex matrix composed of extracellular polymeric substances (EPS) and bacterial cells. They are generated as a defense mechanism by certain bacterial species. Although researchers have had difficulty isolating and identifying specific components of various biofilms it is generally accepted that biofilms are composed of: polysaccharides, proteins and extracellular DNA. 

Dispersin B is an enzyme that targets polysaccharides in the matrix. It specifically targets &quot;the glycosidic linkages of poly-β-1,6-GlcNAc&quot; for hydrolysis (Itoh,Hinnebusch, Preston, &amp; Romeo, 2005).

<I>N. multiformis</I> is an amonia-oxidizing bacterium common to waste-water treatment facilities. Using NCBI we blasted it's genome for a signaling sequence found on the DsbA protein, common to <I>E. Coli.</I>, and were able to find a match. The sequence will enable Dispersin to be exported from <I> N. multiformis </I> and act to degrade problematic biofilm in the reactors.</P><H3>Design Notes</H3><P>The DsbA signaling sequence was attached to the Dispersin B gene using PCR. Due to the length of the forward primer, overlap extension PCR was used to piece it together prior to attaching it to the Dispersin Gene.
</P><P align="center"><U>Part Schematic</U></P><P align="center"><U>Ribbon Diagram</U></P><H3>Source</H3><P>The Dispersin B DNA was aquired from part BBa_K1195000. The signaling sequence was simply ordered along with our primers.
Dispersin B was originally discovered in <I>Aggregatibacter actinomycetemcomitans</I>, a pathogenic bacteria which causes gum disease. The bacteria uses excreted Dispersin B protein to disperse it's own biofilm formations, and enable it to spread to other locations (Kaplan, Ragunath,Ramasubbu,&amp; Fine, 2003).</P><H3>References</H3><OL><LI>Itoh, Y., Wang, X., Hinnebusch, B. J., Preston, J. F., &amp; Romeo, T. (2005). Depolymerization of -1,6-N-Acetyl-D-Glucosamine Disrupts the Integrity of Diverse Bacterial Biofilms. Journal of Bacteriology. doi:10.1128/JB.187.1.382-387.2005</LI><LI>Kaplan, J. B., Ragunath, C., Ramasubbu, N., &amp; Fine, D. H. (2003). Detachment of Actinobacillus actinomycetemcomitans Biofilm Cells by an Endogenous Hexosaminidase Activity. Journal of Bacteriology. doi:10.1128/JB.185.16.4693-4698.2003</LI></OL></TD></TR></TBODY></TABLE><TABLE border="4" align="center" style="margin-top:10px; margin-left:35px; margin-right:35px"><TBODY><TR><TD>BBa_K1356003</TD><TD><B>Nitrite Reductase (<I>nirS</I>) from <I>Pseudomonas aeruginosa</I> PAO1</B></TD><TD>DNA in pSB1C3 plasmid backbone</TD><TD>Created by: Cameron Sargent</TD></TR><TR><TD colspan="4"><H3>Description</H3><P>This gene codes for the nitrite reductase (<I>nirS</I>) that converts nitrite (NO<SUB>2</SUB><SUP>-</SUP>) into nitric oxide (NO). This conversion is the first step in the denitrification pathway from nitrite (NO<SUB>2</SUB><SUP>-</SUP>) to nitrogen gas (N<SUB>2</SUB>). Please refer to <A href="https://static.igem.org/mediawiki/2014/d/d4/DenitrificationSchematic.png">this image</A> for a schematic of the denitrification pathway.</P><H3>Design Notes</H3><P> This gene was cloned from <I>Pseudomonas aeruginosa</I> PAO1 genomic DNA into pSB1C3 using the <I>Xba</I>I and <I>Spe</I>I restriction sites. Correct sequence and orientation were confirmed using 454 Pyrosequencing (BYU).</P><H3>Source</H3><P>This gene was cloned from <I>Pseudomonas aeruginosa</I> PAO1 genomic DNA, which was isolated from a bacterial stock provided by Dr. Stephen Lory at Harvard Medical School in Boston.</P><H3>References</H3><OL type="1"><LI>Z. Chen et al., Differentiated response of denitrifying communities to fertilization regime in paddy soil. <I>Microbial ecology</I><B>63</B>, 446 (Feb, 2012).</LI><LI>H. Arai, Regulation and Function of Versatile Aerobic and Anaerobic Respiratory Metabolism in Pseudomonas aeruginosa. <I>Frontiers in microbiology</I><B>2</B>, 103 (2011).</LI><LI>V. Kathiravan, Pseudomonas aeruginosa and Achromobacter sp.: nitrifying aerobic denitrifiers have a plasmid encoding for denitrifying functional genes. <I>World journal of microbiology &amp; biotechnology</I><B>30</B>, 1187 (2014).</LI></OL></TD></TR></TBODY></TABLE><TABLE border="4" align="center" style="margin-top:10px; margin-left:35px; margin-right:35px"><TBODY><TR><TD>BBa_K1356004</TD><TD><B>Nitric oxide reductase (<I>norC</I>) from <I>Pseudomonas aeruginosa</I> PAO1</B></TD><TD>DNA in pSB1C3 plasmid backbone</TD><TD>Created by: Cameron Sargent</TD></TR><TR><TD colspan="4"><H3>Description</H3><P>This gene codes for one of the nitric oxide reductase subunits (<EM>norC</EM>) that, in connection with the other subunit (<EM>norB</EM>), converts nitric oxide (NO) into nitrous oxide (N<SUB>2</SUB>O). This conversion is the second step in the denitrification pathway from nitrite (NO<SUP>2-</SUP>) to nitrogen gas (N<SUB>2</SUB>). Please refer to <A href="https://static.igem.org/mediawiki/2014/d/d4/DenitrificationSchematic.png">this image</A> for a schematic of the denitrification pathway.</P><H3>Design Notes</H3><P>This gene was cloned from <EM>Pseudomonas aeruginosa</EM> PAO1 genomic DNA into pSB1C3 using the <EM>Xba</EM>I and <EM>Spe</EM>I restriction sites. Correct sequence and orientation were confirmed using 454 Pyrosequencing (BYU).</P><H3>Source</H3><P>This gene was cloned from <I>Pseudomonas aeruginosa</I> PAO1 genomic DNA, which was isolated from a bacterial stock provided by Dr. Stephen Lory at Harvard Medical School in Boston.</P><H3>References</H3><OL type="1"><LI>Z. Chen et al., Differentiated response of denitrifying communities to fertilization regime in paddy soil. <I>Microbial ecology</I><B>63</B>, 446 (Feb, 2012).</LI><LI>H. Arai, Regulation and Function of Versatile Aerobic and Anaerobic Respiratory Metabolism in Pseudomonas aeruginosa. <I>Frontiers in microbiology</I><B>2</B>, 103 (2011).</LI><LI>V. Kathiravan, Pseudomonas aeruginosa and Achromobacter sp.: nitrifying aerobic denitrifiers have a plasmid encoding for denitrifying functional genes. <I>World journal of microbiology &amp; biotechnology</I><B>30</B>, 1187 (2014).</LI></OL></TD></TR></TBODY></TABLE><TABLE border="4" align="center" style="margin-top:10px; margin-left:35px; margin-right:35px"><TBODY><TR><TD>BBa_K1356005</TD><TD><B>Nitric oxide reductase (<I>norB</I>) from <I>Pseudomonas aeruginosa</I> PAO1</B></TD><TD>DNA in pSB1C3 plasmid backbone</TD><TD>Created by: Cameron Sargent</TD></TR><TR><TD colspan="4"><H3>Description</H3><P>This gene codes for one of the nitric oxide reductase subunits (<I>norB</I>) that, in connection with the other subunit (<I>norC</I>), converts nitric oxide (NO) into nitrous oxide (N<SUB>2</SUB>O). This conversion is the second step in the denitrification pathway from nitrite (NO<SUP>2-</SUP>) to nitrogen gas (N<SUB>2</SUB>). Please refer to <A href="https://static.igem.org/mediawiki/2014/d/d4/DenitrificationSchematic.png">this image</A> for a schematic of the denitrification pathway.</P><H3>Design Notes</H3><P>This gene was cloned from <I>Pseudomonas aeruginosa</I> PAO1 genomic DNA into pSB1C3 using the <I>Xba</I>I and <I>Spe</I>I restriction sites. Correct sequence and orientation were confirmed using 454 Pyrosequencing (BYU). The original sequence contained <I>Pst</I>I sites starting at bases ￼115 and 1,231. These sequences were changed to CTTCAG and CTACAG, respectively, using site-specific mutagenesis; the mutant sites were verified to code for the same amino acids. Mutagenesis was also confirmed using 454 Pyrosequencing (BYU). </P><P align="center"><U>pSB1C3 Plasmid with <I>norB</I> insert:</U></P><H3>Source</H3><P>This gene was cloned from <I>Pseudomonas aeruginosa</I> PAO1 genomic DNA, which was isolated from a bacterial stock provided by Dr. Stephen Lory at Harvard Medical School in Boston.</P><H3>References</H3><OL type="1"><LI>Z. Chen et al., Differentiated response of denitrifying communities to fertilization regime in paddy soil. <I>Microbial ecology</I><B>63</B>, 446 (Feb, 2012).</LI><LI>H. Arai, Regulation and Function of Versatile Aerobic and Anaerobic Respiratory Metabolism in Pseudomonas aeruginosa. <I>Frontiers in microbiology</I><B>2</B>, 103 (2011).</LI><LI>V. Kathiravan, Pseudomonas aeruginosa and Achromobacter sp.: nitrifying aerobic denitrifiers have a plasmid encoding for denitrifying functional genes. <I>World journal of microbiology &amp; biotechnology</I><B>30</B>, 1187 (2014).</LI></OL></TD></TR></TBODY></TABLE><TABLE border="4" align="center" style="margin-top:10px; margin-left:35px; margin-right:35px"><TBODY><TR><TD>BBa_K1356006</TD><TD><B>Nitrous oxide reductase (<I>nosZ</I>) from <I>Pseudomonas aeruginosa</I> PAO1</B></TD><TD>DNA in pSB1C3 plasmid backbone</TD><TD>Created by: Cameron Sargent</TD></TR><TR><TD colspan="4"><H3>Description</H3><P>This gene codes for the nitrous oxide reductase (<I>nosZ</I>) that converts nitrous oxide (N<SUB>2</SUB>O) into nitrogen gas (N<SUB>2</SUB>). This conversion is the third and final step in the denitrification pathway from nitrite (NO<SUP>2-</SUP>) to nitrogen gas (N<SUB>2</SUB>). Please refer to <A href="https://static.igem.org/mediawiki/2014/d/d4/DenitrificationSchematic.png">this image</A> for a schematic of the denitrification pathway.</P><H3>Design Notes</H3><P>This gene was cloned from <I>Pseudomonas aeruginosa</I> PAO1 genomic DNA into pSB1C3 using the <I>Xba</I>I and <I>Spe</I>I restriction sites. Correct sequence and orientation were confirmed using 454 Pyrosequencing (BYU). This sequence contains a <I>Pst</I>I site starting at base 1845.</P><H3>Source</H3><P>This gene was cloned from <I>Pseudomonas aeruginosa</I> PAO1 genomic DNA, which was isolated from a bacterial stock provided by Dr. Stephen Lory at Harvard Medical School in Boston.</P><H3>References</H3><OL type="1"><LI>Z. Chen et al., Differentiated response of denitrifying communities to fertilization regime in paddy soil. <I>Microbial ecology</I><B>63</B>, 446 (Feb, 2012).</LI><LI>H. Arai, Regulation and Function of Versatile Aerobic and Anaerobic Respiratory Metabolism in Pseudomonas aeruginosa. <I>Frontiers in microbiology</I><B>2</B>, 103 (2011).</LI><LI>V. Kathiravan, Pseudomonas aeruginosa and Achromobacter sp.: nitrifying aerobic denitrifiers have a plasmid encoding for denitrifying functional genes. <I>World journal of microbiology &amp; biotechnology</I><B>30</B>, 1187 (2014).</LI></OL></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BYU_Provo/Parts">http://2014.igem.org/Team:BYU_Provo/Parts</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BYU_Provo/Parts" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BYU_Provo/Parts" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BYU_Provo/Parts&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BYU_Provo/Parts&amp;oldid=400227" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>