<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BYU_Provo_Notebook_Metabolism_septoct"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BYU_Provo/Notebook/Metabolism/septoct">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BYU_Provo/Notebook/Metabolism/septoct&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Metabolism/septoct&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Metabolism/septoct&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BYU_Provo/Notebook/Metabolism/septoct" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BYU Provo/Notebook/Metabolism/septoct</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1 style="color:#FFFFFF">BYU 2014 Notebook </H1><P style="color:#FFFFFF"><A href="https://2014.igem.org/wiki/index.php?title=Team:BYU_Provo/Notebook/Metabolism/septoct&amp;action=edit" style="color:#FFFFFF">Edit September October</A></P><TABLE width="100%"><TBODY><TR heigth="30px"><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo" style="color:#002255">Home </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Team" style="color:#002255"> Team </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=BYU_Provo" style="color:#002255"> Official Team Profile </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Project" style="color:#002255"> Project</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Parts" style="color:#002255"> Parts</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Modeling" style="color:#002255"> Modeling</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Notebook" style="color:#002255"> Notebook</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Safety" style=" color:#002255"> Safety </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Attributions" style="color:#002255"> Attributions </A></TD></TR></TBODY></TABLE><BLOCKQUOTE><H2>Week of September 6th</H2><H3>September 3, 2014</H3><P>--CS-- Today I checked the sequencing results from all of last week's sequencing. The samples that I submitted for <I>nirS</I> (3-5) and <I>norC</I> (3-2 and 3-4) all turned out great! The results showed that the promoters were inserted into the plasmids upstream of the gene sequences, just as we wanted them. I also checked the <I>norB</I> mutagenesis results. It appears that they did not work properly, so we'll have to try those all over again. The PCR hasn't been working properly, so I think I should try starting over from there instead of transforming and everything all over again. Julie and I also talked and decided that Julie would take over with constructing the ginormous plasmid containing all four of the denitrification genes since she is pretty much done with her antibiotics stuff now.</P><P>--JR--Plan for designing and constructing denitrification plasmid: Digest NorC at Xba and PstI so to insert it in front of NirS. Then, we need to get promoters in front of NorB and NosZ. The plan is to digest each of those plasmids at EcoRI and XbaI sites so as to insert the pomoter J23101 in front of each of the genes. Set up overnights to obtain necessary plasmids. </P><P>--BRK--I thawed out my plasmid preps from a couple weeks prior and set up a digest of the vector (the J23111 promoter) and insert (EreB) according to our <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">common procedures</A> and placed the product in the freezer to preserve it.</P><H2>Week of September 13th</H2><H3>September 8, 2014</H3><P>--CS-- Today I reviewed where everything is at right now with the denitrification project. I <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">transformed</A> 2 μl and 4 μl of the <I>Dpn</I>I-digested <I>nosZ</I> into DH5α, letting it incubate at 37°C for 1.5 hours, and then plated it onto LB+Cam plates, putting them in the old 37°C incubator overnight. </P><P>--JR--Set up restriction digests for denitrification genes. according to <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">RD Protocol</A> NosZ and NorB were digested with EcoRI and XbaI, expected digest sizes are 4000 and 3400 base pairs respectively. NorC was digested with XbaI and PstI, expected digest size of 500bp. NirS was digested with SpeI and PstI, with expected digest size being approximately 3700bp.</P><P>--BRK--I thawed the digest products from last week and prepared a low-melt agarose gel to separate the plasmid parts for ligation using directions from the<A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">common procedures</A>. After the gel ran for about 50 minutes, I removed it from the box and observed it under UV light to remove the bands for ligation but did not see any. Freezing the plasmids before ligating them may have been a bad idea so now I will have to go back and redo my last few weeks.</P><H3>September 9, 2014</H3><P>--CS-- The <I>nosZ</I> plates that I grew up overnight didn't have any colonies, so I will have to start the mutagenesis reaction all over since it hasn't worked for the past few times. Today I did a new <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">colony PCR</A> reaction for the <I>norB</I> mutagenesis reactions since the past few times haven't really worked.</P><P>--JR--Ran a low melt gel on digested products, cut out respective bands. Set up <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">ligation reaction</A> according to protocol. Ligation with previously digested J23101 and NorB and NosZ digests each respectively. Also a ligation was set up with the NorC and NirS digests done yesterday.</P><H3>September 10, 2014</H3><P>--CS-- Today I ran an <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">analytical gel</A> of the <I>norB</I> PCR products. My gel turned out great this time around! Here is the image:</P><P>I will be able to sequence a few of these this time and hopefully will be able to tell if the mutagenesis actually worked or not since all of the other tests failed to have very reliable results.</P><P>I also went back to old plates and picked a colony for <I>nosZ</I> from before mutagenesis but after successful cloning to use for plasmid preps prior to redoing the mutagenesis reaction. I also whipped up a new <I>P. aeruginosa</I> PAO1 stock plate just in case we need that anymore the rest of the semester since the stock plates we have are getting old. I also contacted Brother Lee about getting some Durham tubes made up.</P><P>--JR--Did colony PCR. The NirS/NorC ligation appears to have worked, but discovered I have used the wrong NorB and NosZ for my plasmids which needed to be digested. Disappointing setback. Set up some new correct overnights.</P><P>--BRK--Today I selected colonies from my previous overnight plate and added each to a test tube with 3mL of LB/CAM to grow overnight at 37C again. In addition, I selected a colony from an old plate that contained DH5α with the <A href="http://parts.igem.org/Part:BBa_J23111">constitutive promoter BBa_J23111</A> and also grew it overnight in 3mL of LB/CAM.</P><H3>September 11, 2014</H3><P>--CS-- The <I>nosZ</I> overnight appeared a little faint, so I let it grow up another day to make sure I have plenty of stuff to work with for plasmid preps.</P><P>--JR--Did plasmid preps for NorB and NosZ</P><H3>September 12, 2014</H3><P>--CS-- I didn't have a ton of time today so I pelleted down the <I>nosZ</I> overnights and put them in the freezer. I also prepared <I>norB</I> samples for sequencing by putting 2 μl of the PCR product from earlier in the week with 1 μl of both the forward and reverse primers in PCR tubes and submitting them to Desi.</P><P>--JR--Set up <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">restriction digest</A> again for NorB and NosZ with EcoRI and XbaI according to protocol.</P><P>--BRK--Removed my plates from the incubator and parafilm wrapped them so they could be stored in the refrigerator</P><H3>September 13, 2014</H3><P>--CS-- I reviewed the sequencing results for <I>norB</I>. The first site appeared to be fixed conclusively, as shown by the image below that has all 4 of the sequences matching the desired mutated sequence at the top. The second site though is still inconclusive; the sequencing results were very uncertain in the region of the mutagenesis site, as shown below.</P><P>So it appears that I should probably resubmit the samples to see if I can get better results for the second site. But so far things are looking positive!</P><P>--JR--Ran lowmelt on digests. Set up <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">ligation reaction</A>.</P><H2>Week of September 20th</H2><H3>September 15, 2014</H3><P>--CS-- Today I first helped Julie out by <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">transforming</A> the ligations that she had made for <I>norB</I> and <I>nosZ</I> with the promoters. I then tried to round up some Durham tubes but was unsuccessful; I arranged, however, to have some made for us tomorrow. I completed the plasmid prep of the <I>nosZ</I> that I had grown up last week using the kit. I then took some of this and did a single-site mutagenesis PCR with Dr. Grose. I also submitted some of the purified <I>nosZ</I> plasmid for sequencing with the internal primers. I also submitted <I>norB</I> samples 4-7 with the vector reverse and gene reverse primers again since the last time they had failed to show anything in the sequencing results.</P><P>--JR--Set up <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">transformation</A> according to protocol for NorB and NosZ, with Cameron's help. </P><P>--BRK--Spun down my overnight media and did a plasmid prep using <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">Qiagen QIAprep Spin Miniprep Kit</A>. I took each sample to Nano-drop the plasmids and see if they had been properly amplified and then I added more data to the wiki.</P><H3>September 16, 2014</H3><P>--CS-- Today I did the <I>Dpn</I>I digest of the PCR product from the single-site mutagenesis that Dr. Grose and I did yesterday. I added 1 μl of <I>Dpn</I>I to the PCR product, mixed briefly, and incubated at 37° for 1 hour. I then <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">transformed</A> 2 μl of this into DH5α, letting it incubate at 37°C for 1.5 hours, and then plated it onto LB+Cam plates, putting them in the old 37°C incubator overnight.</P><P>--JR--Took out plates from incubator. Picked colonies for colony PCR </P><H3>September 17, 2014</H3><P>--CS-- I got a pretty good amount of colonies on my <I>nosZ</I> plate today for a mutagenesis so I went ahead and did the <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">colony PCR</A> of 16 of these samples, streaking out plates of the picked colonies as I went. When these were done I ran an <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">analytical gel</A> of the PCR products. The images are shown below:</P><P>These gels showed that my PCR did not work since all of the primers are all down at the bottom of the gel. I am pretty certain that I added everything to the PCR mix, so it seems that these colonies not only lack the mutagenic <I>nosZ</I> but also lack the pSB1C3 plasmid altogether since I used the vector forward and reverse primers (307/308). This seems strange since I grew them up on LB+Cam plates, so I will consult Desi or Dr. Grose about this. While waiting for everything I also did a massive cleanup of all of my samples in the fridge and freezer to discard anything that is not needed anymore. We also received the knockout <I>E. coli</I> for testing our denitrification genes <I>in vivo</I> so I streaked the different strains out on LB and put them in the 37°C incubator overnight.</P><P>--JR--Set up new overnights of NorB and NosZ.</P><P>--BRK--I prepared my digest products from last week as well as another low-melt agarose gel to separate the plasmid parts for ligation using directions from the <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">common procedures</A>. I observed the gel under UV light and removed the bands with a razor blade for ligation. I placed them in microcentrifuge tubes and put them in the refrigerator for storage.</P><H3>September 18, 2014</H3><P>--CS-- Today I submitted <I>norB</I> samples 4-7 for sequencing. I added 2 μl PCR product to 1 μl reverse primer; for each sample I did one reaction with the gene reverse primer and one reaction with the vector reverse primer.</P><H3>September 20, 2014 </H3><P>--JR--Did plasmid preps on NorB and NosZ. This time they had good concentrations. ~150ng/ul each. </P><P>Set up <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">restriction digests</A> again according to protocol. With EcoRI and XbaI.</P><H2>Week of September 27th</H2><H3>September 22, 2014</H3><P>--CS-- Today I reviewed the sequencing results from the <I>norB</I> samples that I submitted last week. They did not have very confidant outputs, and when I aligned the sequences to the desired sequence it showed that the site was not actually mutated. These results were certainly not completely reliable due to low quality reads, but they suggest that mutagenesis did not work at the second site. The image of the sequencing alignment is shown below:</P><P>Neither the gene nor vector reverse primers worked. Skip suggested that instead of trying sequencing over and over again I could do a <I>Pst</I>I digest of the purified plasmid and run it on a gel to determine how many cut sites there are in the whole plasmid based on the number of bands. To try this out I picked some colonies for <I>nosZ</I> (1, 5, 6, and 9) and <I>norB</I> (4, 5, 6, and 7) and put them in 5 ml LB to grow up overnight cultures in the 37°C shaker. I also realized that although we had successfully clones <I>nirS</I> and <I>norC</I> into the pSB1C3 we had not prepped those for submission as BioBricks, so I picked some colonies from the plates that had the proper clones of those genes and grew up overnights of those as well using 5 ml LB.</P><P>--JR--Set up <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">ligation reaction</A> for promoter J23101 and NorB and NosZ respectively, according to protocol. </P><P>--BRK--I melted my gel slices and prepared them for ligation using <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">common procedures</A>.</P><H3>September 23, 2014</H3><P>--CS-- Today I started by doing the plasmid preps of the overnight cultures I made for all 4 genes. I then checked the DNA concentrations and purity of these using the <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">Nano-Drop</A>. The <I>nirS</I> and <I>norC</I> concentrations were in the twenties, so I will have to redo those. All but 2 of the other plasmid preps resulted in concentrations in the hundreds and 260/280 readings near 2, and the 2 were above 50 ng/μl so they should be alright. I set up the <I>norB</I> and <I>nosZ</I> plasmids for <I>Pst</I>I digests based on our <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">restriction digest</A> protocol. For the samples that had more than 100 ng/μl, I mixed 5 μl of the samples with 5 μl buffer, 38 μl ddH<SUB>2</SUB>0, and 2 μl Taq polymerase. For the samples that had less than 100 ng/μl, I mixed 10 μl of the samples with 5 μl buffer, 33 μl ddH<SUB>2</SUB>0, and 2 μl Taq polymerase. I then put these on the shaker in the 37° incubator overnight.</P><P>--JR--Set up colony PCR. Results didn't look great. No product. Went back to check plasmid concentration from NorB and NosZ--they weren't good, explaining why these transformations wouldn't have worked right. Set up some more plasmid preps. </P><H3>September 24, 2014</H3><P>--CS-- Today I ran the <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">analytical gels</A> of the <I>Pst</I>I digests. The image of my gel is below:</P><P>For some reason whenever I print or save the images of my gels, they never look as good as what I actually see on the screen. Unfortunately I had already tossed my gel, but I reviewed the results with Dr. Grose and she pointed out that since the band would be under 200 bp for either of the genes if the <I>Pst</I>I sites were still in the genes I probably wouldn't be able to see the small bands anyway. So since this test did not really confirm anything Dr. Grose suggested that I just try the sequencing again to check the mutagenesis. I submitted all 4 samples from both genes to Desi with the vector reverse primers for sequencing.</P><P>--JR--<A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">Transformed</A>NorB and NosZ ligations.
</P><P>--BRK--Ligated colonies grew strongly on the LB/CAM plates, but now I will have to test them against erythromycin to see if the resistance gene actually worked using the same dilution protocol that I used previously.</P><H3>September 26, 2014</H3><P>--CS-- Today I reviewed the sequencing results from my <I>norB</I> and <I>nosZ</I> samples. It looks like my <I>norB</I> is now properly mutagenized! The <I>nosZ</I> sequence still has the <I>Pst</I>I site in it though unfortunately. The sequencing results were pretty good (everything has some HQ data and some were even as high as 20%, a significant improvement to the results with 0% HQ that I've had recently), so I think that I will just do plasmid preps for sequencing from here on out and not even bother with trying the colony PCR product. The images of the alignments are shown below, <I>norB</I> on the left and <I>nosZ</I> on the right, with the <I>Pst</I>I site highlighted:</P><P>I also tried playing around with the wiki formatting some to try making our pages look better but wasn't really successful with anything, so I just left it as it originally was.</P><P>--JR--Set up colony PCR to verify transformants. This time it appears 2-3 of the NosZ transformations worked, and all the NorB colonies picked appear to have worked also! Hooray! Finally! Set up overnights on 2 of the NosZ working colonies and 2 of the NorB working colonies. </P><H3>September 27, 2014</H3><P>--JR--Did a plasmid prep on NorB and NosZ. Still getting relatively low plasmid concentrations. Approximately 50ng/ul. I even documented which buffers I used, using a different combination on duplicate plasmid preps, no apparent difference was made. Set up restriction digests using such plasmids. Digested the NorB+Promoter plasmid using EcoRI+XbaI, and the new NosZ+Promoter at EcoRI+SpeI. </P><H2>Week of October 4th, 2014 </H2><H3>September 29, 2014</H3><P>--JR--Ran digests out on gel. The bands looked good (brightness), but the sizes did not appear to be what I expected. I expected the NorB digest to be approximately 3400bp as it would be the psB1C3 plasmid with the norB gene and promoter (minus the RFP gene), and then the NosZ digest we wanted the ~2000bp product as we just wanted the NosZ gene and promoter from the plasmid. However it appears that both products looked as though they were around 3500bp, with the NosZ products appearing larger than the NorB.</P><P>--CS-- Today I looked over my sequencing results a little bit more. Dr. Grose and I then did another mutagenesis reaction using her kit for <I>nosZ</I> since the sequencing results showed that the unwanted <I>Pst</I>I site has yet to be mutagenized. I also did some planning to figure out what all I need to take care of this week in order to be able to submit all of the genes as BioBricks next week. It's getting close but hopefully we can make it all work out!</P><P>--BRK--To prepare for my dilution test, I also had to prepare a better control. I transformed DH5α with the empty iGEM backbone plasmid using <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">common procedures</A> and plated the bacteria on LB/CAM overnight at 37C</P><H3>September 30, 2014</H3><P>--JR--Did plasmid preps for NorB and NosZ, got decent, but not great plasmid concentrations ~50ng/ul. Set up <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">restriction digests</A> for these plasmids. This time set it up in a way that would digests NosZ so as to produce to differently sized bands. Digested NorB using EcoRI and SpeI so as to produce a ~1400bp fragment which will be cloned into the digested NosZ plasmid, which will be digested using EcoRI and XbaI producing a ~4000bp fragment. Things went well until I ran these out on gel. All the products looked to be the same size, but yet again the ladder looked weird. Not sure if there is a problem with the ladder or the LM gel.</P><P>--CS-- Today I did a <I>Dpn</I>I digest of the mutagenesis PCR product from yesterday (~25 μl) by adding 1 μl <I>Dpn</I>I and 3 μl 10X CutSmart buffer (total ~29 μl). After incubating at 37°C for 1 hour I <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">transformed</A> 2 and 4 μl of it into DH5α. After an hour and 20 minute recovery period I then plated this transformed bacteria onto LB+Chloramphenicol plates and incubated them overnight in the 37°C incubator.</P><H3>October 1, 2014</H3><P>--JR--Resetting up restriction digests. According to previous plan as stated in September 30. </P><P>--CS-- Today I did <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">colony PCR</A> for 16 of the <I>nosZ</I> colonies that were on the 2 μl plate (the 4 μl plate didn't have any colonies grow) and 8 colonies from <I>nirS</I> and <I>norC</I> plates from a little bit ago since my last attempt at preparing BioBricks for those 2 genes didn't work out. After PCR I ran an <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">analytical gel</A> of the PCR products. My images showed that only one of the <I>norC</I> reactions worked. Since time is so short though I am just going to have to go ahead and do plasmid preps and see if sequencing works for those since my PCR didn't work (I suspect I'm not doing something right with the colonies when I'm putting them in the water to boil for template but I won't have time to figure out what exactly is going on). But anyway, so when I streaked plates out when I went through and picked colonies for the PCR I also added some to 200 μl aliquots of LB+Cam so I was able to go back to these and add them to 5 ml LB+Cam for growing overnight cultures.</P><P>--BRK--Today I prepared liquid overnights of my EreB+111 bacteria and the newly transformed control in LB/CAM.</P><H3>October 2, 2014</H3><P>--CS-- Today I did plasmid preps using the kit. I noticed that my pellets were rather small and Nano-Drop results ended up being pretty low. My <I>nirS</I> and <I>norC</I> overnights probably didn't work too well since they were from old plates, but since I have streaked them again they should hopefully work better if I need to do them again after today. As for my <I>nosZ</I> results, I'm not sure why they were as low as they were. They were certainly better than the other samples, but still not ideal. I think they might have not been able to grow long enough since I did my plasmid preps early in the morning but I'm not sure. I submitted some samples from each of the 3 genes to Desi for sequencing to make sure that I'm 100% sure that I picked the right colonies for <I>nirS</I> and <I>norC</I> and to make sure that the mutagenesis worked.</P><H3>October 3, 2014</H3><P>--CS-- Today I reviewed the sequencing results. The results confirmed that <I>nirS</I> and <I>norC</I> were indeed correct, so I just need to grow those up again to get a higher concentration of my plasmid for submission. The results showed that <I>nosZ</I> mutagenesis did not work again. The image comparing the sequencing results with the desired sequence is below:</P><P>The third nucleotide in the highlighted 6-nucleotide sequence showed up as a G in all of the sequencing results, and the results were decent quality, so it appears that the mutagenesis did not work again.</P><H2>Week of October 11th, 2014 </H2><H3>October 6, 2014</H3><P>--CS-- Today I ran the sequencing results from last week by Dr. Grose and she suggested that I try some more plasmid preps since the reaction is just a low-efficiency one and we're bound to find a mutated one sooner or later. I set up 5 new overnights with 5 ml LB+Cam for other <I>nosZ</I> colonies that I hadn't prepped previously and Desi kindly agreed to set up sequencing reactions tomorrow. I also set up some overnights for <I>nirS</I> and <I>norC</I> since they had such low yields last time.</P><P>--JR--Set up overnights to do new plasmid preps. We decided to get a new kit as plasmid prep numbers have been consistently low for myself and others.</P><H3>October 7, 2014</H3><P>--CS-- Today I did plasmid preps of the <I>nirS</I>, <I>norC</I>, and <I>nosZ</I> overnight cultures that I grew up yesterday. I used the new kit that Dr. Grose got us and it made quite a difference. I got much higher yields, especially on the <I>nirS</I> and <I>norC</I> preps. I submitted the <I>nosZ</I> preps to Desi for sequencing.</P><P>--JR--Did plasmid preps. Still got relatively low concentration numbers. Between 40-80ng/ul.</P><P>--BRK--Today I set up a serial dilution of overnight media of transformed EreB-111 DH5α and iGEM backbone DH5α
control to compare the effectiveness of the plasmid gene against the concentration of antibiotic in solution. In 2 microcentrifuge tubes I added .5mL of LB/CAM media, and subsequently added .5uL of erythromycin solution. In 10 microcentrifuge tubes, I added 800uL of LB/CAM to do a 1:5 serial dilution for each overnight media. To the 2 initial tubes, I added .5mL of overnight bacteria solution, inverted the tube a few times, and subsequently added 200uL of media into the next tube and carried out the dilution series. All tubes were placed in an incubator overnight at 37C.</P><H3>October 8, 2014</H3><P>--CS-- Today I reviewed the sequencing results from this last batch of <I>nosZ</I> samples. The sequencing quality was high enough to be reliable, and none of the results showed that the desired mutation actually occurred. Here is an image of the results compared to the desired sequence with the six nucleotides of the <I>Pst</I>I site highlighted:</P><P>All of the sequences still have a G where we were hoping to see an A. Dr. Grose said that we should just keep checking colonies to see if something worked on one of them under the assumption that the primers and this specific reaction just have an abnormally low efficiency. I double checked the primers again too to ensure that they were correct; they were, so something else must be going on wrong in the reaction.</P><P>I also submitted all of my BioBricks to Jordan to prep for mailing in and submitted the information into the BioBrick submission pages and our parts page.</P><H3>October 9, 2014</H3><P>--CS-- Since none of our sequencing results have shown that the mutagenic PCR worked yet, Dr. Grose told me to keep making plasmid preps and sequencing them until we find a mutant. So today I started overnights for a bunch of <I>nosZ</I> colonies from the plate that I grew up last time we did the mutagenic PCR. I also fixed some things on our wiki.</P><P>--JR--Set up a <A href="https://2014.igem.org/Team:BYU_Provo/Notebook/CommonProcedures">restriction digest</A> according to protocol. With <I>NorB</I> being digested with <I>Eco</I>RI and <I>Spe</I>I, and <I>NosZ</I> being digested with <I>Eco</I>RI and <I>Xba</I>I.</P><H3>October 10, 2014</H3><P>--CS-- Today I pelleted down the bacteria from my overnight cultures and stuck them in the freezer for Monday.</P><H2>Week of October 18th, 2014 </H2><H3>October 13, 2014</H3><P>--BRK--Added to the wiki page under <A https:="">Attributions</A>. Digested plasmids from the EREB(111) overnight using <A https:="">plasmid mini prep kit</A> and placed 50ul solutions in freezer. Also worked with another team member to finish the attributions page on our wiki. </P><P>--CS-- Today I did plasmid preps on 16 of the samples that I had grown up last week using the kit in the lab. I got great yields from all of them and submitted them to Desi for sequencing.</P><H3>October 14, 2014</H3><P>--CS-- Today I worked on the wiki.</P><P>--BRK--Worked on updating the wiki as well.</P><H3>October 15, 2014</H3><P>--CS-- Today I worked some more on the wiki. I also found some really good sources to use in talking about eutrophication:</P><P><A href="http://www.ncbi.nlm.nih.gov.erl.lib.byu.edu/pubmed/16781774"><I>Ecological and toxicological effects of inorganic nitrogen pollution in aquatic ecosystems: A global assessment.</I></A></P><P><A href="http://water.epa.gov/drink/contaminants/basicinformation/nitrate.cfm"><I>Basic Information about Nitrate in Drinking Water</I></A></P><P><A href="http://www.ncbi.nlm.nih.gov.erl.lib.byu.edu/pubmed/?term=Acute+and+chronic+toxicity+of+nitrate+to+early+life+stages+of+lake+trout+(Salvelinus+namaycush)+and+lake+whitefish+(Coregonus+clupeaformis"><I>Acute and chronic toxicity of nitrate to early life stages of lake trout (Salvelinus namaycush) and lake whitefish (Coregonus clupeaformis).</I></A></P><H3>October 17, 2014</H3><P>--CS-- Today I looked at the sequencing results for the <I>nosZ</I> mutagenesis. As shown in the comparison of the sequences to the desired sequence in the images below, none of the colonies actually had the mutation in them unfortunately:</P></BLOCKQUOTE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BYU_Provo/Notebook/Metabolism/septoct">http://2014.igem.org/Team:BYU_Provo/Notebook/Metabolism/septoct</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BYU_Provo/Notebook/Metabolism/septoct" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BYU_Provo/Notebook/Metabolism/septoct" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Metabolism/septoct&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Metabolism/septoct&amp;oldid=359151" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>