"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/Auxotrophy/mayjune
From 2014.igem.org
BYU 2014 Notebook
Edit May June
Home
Team
Official Team Profile
Project
Parts
Modeling
Notebook
Safety
Attributions
Week of May 3rd
--CB-- /--TR-- This week in the lab we started our experiment to remove the SerA gene from N multiformis. We attempted to gain access to the genomic DNA of N multiformis by boiling the organism for 5 min to lyse the cell, this technique has worked on similar organisms in the past and due to its simplicity we opted for this approach. After boiling we added our primers and proceeded to perform PCR.   Then run our PCR product on a gel. We did this by adding our DNA samples (both forward primers and reverse primers) and a DNA ladder of known size to an agarose gel that had been stained with ethidium bromide and electrophorese for 30-40 min. After isolating our band of DNA we purified it using a freeze and squeeze method.  This method involves excising a band of DNA from the agarose gel and the gel slice cut into small pieces and placed into a micro centrifuge tube. This tube is then placed in a -20C freezer for 20 minutes then removed and immediately centrifuged at 12,000 for 5 minutes at room temp. Agarose debris is will be forced to the bottom of the cup and our now purified DNA is floating on top ready for removal.
Week of May 10th
May 5, 2014
--CB TR-- We finished out the freeze and squeeze experiment.  We took the agarose gel from the freezer that held our PCR product and centrifuged it for 5 min at max speed the added to the following protocol for a freeze and squeeze to get our purified product. Freeze and Sqeeze 2ul fragment one from PCR 2ul fragment two from PCR 4ul 5x buffer 4ul 5x enhancer .5ul dNTP .5ul primer on left side of left homology block .5ul primer on right side of right homology block .2ul Q5 enzyme QA 20uL ddH2O  Because Desi said that doing a freeze and squeeze was probably an unnecessary step we also ran a normal where we do not do a freeze and squeeze, rather we run straight to the SOEing part of it. SOEing protocol 1ul fragment one from PCR 1ul fragment two from PCR 4ul 5x buffer 4ul 5x enhancer .5ul dNTP .5ul primer on left side of left homology block .5ul primer on right side of right homology block .2ul Q5 enzyme QA 20uL ddH2O  We got our SOEing product and ran it on a gel (picture taken and to be included once we scan it). We also performed a restriction digest and plasmid prep.
Week of May 17th
May 13, 2014
-- TR CB--  The planned schedule  Week 1 Ligation, Transform Week 2 Conjugate (long time) • Grow the transformed E coli (s17) • Grow N multiformis • Mix together, turn off the lights, but on some Barry White and wait Week Work on getting funding * experiment.com • Fancy black card man•  Week4 Select for knock out with Kanamycin  Select  with sucrose  Grow in serine rich, low, and no environments      Thursday:  Today we took the product from  our ligation and transformed it. (protocol to be added) We made 3 tubes Tube 1: 200uL s17, 1uL of 93.1 ng/uL mini-prep pSR47s Tube 2: 200uL of our ligation. Tube 3: 300ul of LB, 90ul ddH2O, and 1ul of S17 • S17  is the bacteria that makes our suicide plasma grow. All 3 tubes were placed on a shaker at 37 degrees C. Tuesday: Today (Tuesday) we did ran a gel of the plasmid prep (low melt) of the plasmid digest and the PCR digest (pics to be uploaded) We also did a mini-prep with the pSR47s plasmid. And a ligation
Week of May 24th
--CB TR-- The past week I have spent approximately 6+ hours researching funding opportunities for our team and estimating costs. Opportunities include • Rathnau Instituut Grant  • Departmental Funding • Crowd founding initiatives o I have started to build a profile on Experiment.com • Various Donors  Verified our clones via colony PCR We did this by taking samples from 8 colonies to test for our clone.  Alongside the 8 colonies we also ran a negative control (just the plasmid with no insert) and a positive control (our original SOEing product) Protocol: 16.5 ddH2O 2.5 REdtaq Buffer 1 DNTP 1 Primer A 1 Primer B 1.25 Redtaq 2 Boiled template (colony) These we PCR’d and will check on Thursday.
Week of May 31st
I spent a ton of time this week writing the Rathenau Instituut grant… like LOADS of time. And after it was run through the class refiners fire I think it came out pretty good.  I also ran a gel of our new cloning PCR to verify that our TAQ pcr worked. Tuesday morning we found that the PCR we ran last week failed. We are now rerunning the clone verification using regular TAQ polymerase instead if RedTAQ polymerase.  Another group (one of the Cameron’s) also had their RedTAQ PCR fail.  By cross checking with a different polymerase we hope to verify whether or not the polymerase is to blame for the failed experiment. When boiling template we used 50 uL of ddH2O with our template.
Week of June 7th
--CB TR-- This week started out a little bit rough. We found that not only did Tanner forget to pull the plate from the incubator (again) I foolishly left the gel I ran Thursday in the electrophoresis dish without taking a picture (because I was under the delusion that that would be okay) and so our product diffused all over the place so this week we are re-re-doing out clone  . We poured a gel, placed in 8 colonies and are running it now. If this works we hope to jump into the conjugation with N. multiformis. So bad news, it didn’t work.  The bands we saw from our + control and our colonys showed at 600bp instead of the 1000bp where they should have been. We are not certain why this happened but our best guess would be that we cut out the wrong bands and placed them in our plasmid.  So now we are going to re do out SOE to get new SOEing product that we will remove from a gel and put into our plasmid. Redo of SOEing. 1ul fragment one from PCR 1ul fragment two from PCR 4ul 5x buffer 4ul 5x enhancer .5ul dNTP .5ul primer on left side of left homology block .5ul primer on right side of right homology block .2ul Q5 enzyme QA 20uL ddH2O
Week of June 14th
June 10, 2014
--CB TR--Looked at iGEM webpage, tried to figure out some code things….
This week was pretty chill.  I got back some bad news from the people at iGEM, our proposal was not accepted. Here is some feedback from K.Drinkwater  Dear BYU Provo Team,  Thank you for submitting a proposal for a SYNENERGENE Grant-Funded Collaboration. We received many proposals of high quality. Unfortunately, because of the number of proposals and the limits on our funding capacity, we are not able to fund your proposal this year. We encourage you to seek other opportunities to enrich your project by working with experts and potential end-users, and by engaging with the public -- as well as by applying for SYNENERGENE support again in future years. For more general iGEM team funding, we have posted some help and opportunities at
.  Your project proposal represents an interesting and novel approach to the problem of antibiotic resistance, and the proposal to bring about legislative change is intriguing, though we are curious what kind of legislative changes you would propose. We would suggest that you place more emphasis on ensuring and demonstrating the environmental safety of your project, before you address commercial viability or regulatory approval. For example, you could look at several different systems of biological containment, beyond just an auxotrophic chassis, and give some quantitative idea of their efficacy and usefulness for your project.  We wish you every success in your iGEM project, and we look forward to seeing you at the Giant Jamboree! Thank you, again, for submitting a proposal.  Best regards, Kelly Drinkwater iGEM HQ So that was a bumer. I have spent a little time this week perusing the igem website and trying to understand what all we are required to do for the competition. (outside of lab work) I feel that we as a team haven’t spent enough time doing things outside of the lab to contribute to a competitive showing in boston.
Week of June 21st
June 17, 2014
--CB TR-- Today we boiled our transformed S17 ecoli that we plated yesterday. We are now running a PCR to verify our transformed colonies. And will run the gel to verify our colonies either Friday or Monday Did a step 3 transformation today using the remaining product we have in our box. Our primers we ordered arrived today which is great news. The next step we are going to take isto PCR our product.
Week of June 28th
June 24, 2014
--CB TR-- We grew cultures of our final product (this time using the proper primers) and after performing a clone verification we discovered that we had bands at 1000 base pairs!  Yay!! That means that it works and we are not failures of scientists. Now we need to sequence the DNA and we plan on doing plasmid prep as soon as someone returns the missing reagent to the plasmid prep box. We presented to the class on Wednesday and also ran a plasmid prep now that we know that reagent 1 is kept in the fridge    Desi was totally awesome and did our sequencing for us so thanks Desi. The results are shown below
RESULTS
SerA-2F
AWWGRWCGMWTCTCTWTSYKTCSACGGTATCSATAAGCTTGATATCGAATTCCTGCASCCCGGGGGATCCGGCGTAATGGMACWCATGATCAAAACGGCGGCGGATGCCGAAGCGGCC
GTAAACKCAGTATATTACCCTCCACGCGGACAACGTGGAGTCGGGCTTGCACGCGCCCAGGGGTATGGTGCGCGATTTCAGCAATATCGCCACTGGCTGGAAGAAAATGCCGT
CATCATTGCCATGATCGAGCATATCGATGCAGTCGATGCCATCGATTCGATTCTTTCCGTTCCGGGAATCGATGCTTATATCATAGGTCCCTACGATCTTTCAGGATCGCTCG
GCCGTCCCGGAGAACTCGCTGATCCGGAAGTCCAGGCTGCGGTAGAACGGGTAAAAGATGCCGGGCGACGCGCGGGCAAGGCAGGCGGCATTCATGTAGTCGAACCCAATCCG
GAACAATTGCGCCGCAATATCGAGGCGGGCTTCAGTTTTCTTGGCTATGGCCTGGATATCCGCATTCTCGATACCGTCTGCCGCAGTCATCTGCAAAACATCAGGGAAGCTCT
ATGAACAAACTTGCGATTTCCACCTCGTCATTCGATGTCAGCATCGAAGCCGCTATGGCTCGCTCTATGACTAGTTCTAGAGCGGCCGCCACCGCGGTGGAGCTCCAGCTTTT
GTTCCCTTTAGTGAGGGTTAATTTCGAGCTTGGCGTAATCATGGTCATAGCTGTRTCCTGTGTGAAATTGTTATCCGCTCACAATTCCACACAACATACGAGCCGGAAGCATA
AAAGTGTAAAGCCTGGGGTGCCTAATGAGTGAGCTAACTCACATTAAWTTGCGTTGCGCTCACTGCCCGCTTTCCAGTCGGGAAMCCTGTCGTGCCAGCTGSATTAATGAATC
GGCCAACGCTARAWTTCCCATGTCAGCCGTTAAGTGTTCCTGTGTCACTCAAAATTGCTTGARAGGSTCTAARGGCTTCTCARTGCGTTACATCCCTGGSTTGTKGTCCRCAC
CGTTAAACCTTAAAAGCTTRARAGGCTTATATATCTTTTTTCTWAWAAAMCTAAAACCTARRRGCTATTAGTTGCTGAATTTATATAAGTTAATGGTCARACATGAGAGCTAA
GWASGGGAAAMTGGARAGCGTAGTACGTAGCCATGAAAGCTTAGTACGTAGCCAGGAGGGGTTAARGTCGTACATGAAGCTTAGTACGTAACCGTGAGAGCCTATAAMCGTGA
AAGCGTGGATARGCCGGAT
Retrieved from " http://2014.igem.org/Team:BYU_Provo/Notebook/Auxotrophy/mayjune "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
