<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BYU_Provo_Notebook_CRISPR_julyaug"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BYU_Provo/Notebook/CRISPR/julyaug">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BYU_Provo/Notebook/CRISPR/julyaug&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CRISPR/julyaug&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CRISPR/julyaug&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BYU_Provo/Notebook/CRISPR/julyaug" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BYU Provo/Notebook/CRISPR/julyaug</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1 style="color:#FFFFFF">BYU 2014 Notebook </H1><P style="color:#FFFFFF"><A href="https://2014.igem.org/wiki/index.php?title=Team:BYU_Provo/Notebook/CRISPR/julyaug&amp;action=edit" style="color:#FFFFFF"> Edit July August</A></P><TABLE width="100%"><TBODY><TR heigth="30px"><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo" style="color:#002255">Home </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Team" style="color:#002255"> Team </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=BYU_Provo" style="color:#002255"> Official Team Profile </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Project" style="color:#002255"> Project</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Parts" style="color:#002255"> Parts</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Modeling" style="color:#002255"> Modeling</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Notebook" style="color:#002255"> Notebook</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Safety" style=" color:#002255"> Safety </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Attributions" style="color:#002255"> Attributions </A></TD></TR></TBODY></TABLE><BLOCKQUOTE><H2>Week of July 6</H2><P><U>7/07/2014 - Garrett Jensen.</U>- Today I looked around for the M17 media that supposedly was delivered 2 weeks ago. Brother Lee and Dr. Robison both said that it was not in their lab but they are the only ones who signed out a package from Fischer Scientific 2 weeks ago. 
</P><P><U>July 7th Michael Linzey</U>Redo PCR
-	I realized that I was using the wrong primers for the PCR to check in my spacer was in the region I was trying to do
-	I got the primers that I ordered on the 30th
-	I set up the PCR
o	I used a Taq polymerase because our red Taq is not working really well
-	Let the reaction go over night
-	I also prepared a gel for gel electrophoresis </P><P><U>7/09/2014 - Garrett Jensen.</U>- Today I ran a PCR on the DNA that Skip isolated from S. thermophilus on June 26 using the universal 16S ribosomal primers. I used taq polymerase instead of Redtaq because the redtaq has been having some issues lately. If there is microbial DNA present then we may be able to get PCR for the CRISPR to work from it. In the meantime I am waiting for our M17 media to show up. 
</P><P><U>July 9th Michael Linzey</U>Came in to check to see if it worked
-	I ran the 8 PCR products in the gel that I had prepared
-	Used a blacklight to see if I got anything
o	There was definitely PCR product
o	Talked to Dr. Grose and she said that it looked good
-	Prepared an overnight for the two darkest bands, lane 2 and 3 </P><P><U>July 9, Michael Abboud</U>Set up two separate sewing PCR reactions using Phusion for the CRISPR spacer-repeat region.  Let it go overnight.




</P><P><U>7/10/2014 - Garrett Jensen. </U>- I ran out that PCR on a gel and there was a Band!!!! That means that there is good DNA present. I started a PCR reaction using taq polymerase and our Crispr primers to see if I can get a product for the CRISPR system.  If I can then I will set it up again with phusion. We still have not received the m17 media however.
</P><P><U>July 10th Michael Linzey</U>
Used a DNA prep kit to extract my plasmid from the DNA 
-	Same  protocol that I have followed before
-	Just followed the kit</P><P><U>July 11, Michael Abboud</U>Ran gel with the two PCR products and two controls. In the gel imager, both products showed smears in the expected region for the sewing PCR.

</P><H2>Week of July 13</H2><P><U>July 14, Michael Abboud</U>Moved to new Life Science Building. Showed Dr. Grose the gel bands for the sewing PCR and said the first PCR did a better job so use that one.


</P><P><U>July 14th Michael Linzey </U>Prepared my samples of DNA to get sequenced to make sure that the T7 spacer that we designed was there
We also helped move our lab to a new life science building</P><P><U>7/16/2014 - Garrett Jensen. </U> - On monday (7/14/2014) we moved our lab to the new life sciences building and were not able to do any work.
 - I ran out the PCR attempt to amplify the CRISPR from the LMD-9 DNA that Skip made but no bands were visible. 
 - After talking with Dr. Grose and Desi we decided to cancel our order of the M17 media. It apparently was back ordered but the company sent a shipping document anyways. We have all the materials to make M17 media except for beef extract. Desi found a place to order it from for pretty cheap and sent the information to Dr. Grose. We were able to find ascorbic acid (vitamin C) in Dr. O'Neil's lab. Once the beef extract comes in I can grow LMD-9 and purify its DNA. 
</P><P><U>July 16, Michael Abboud</U>Purified the sewing PCR DNA using a kit. Mike made me a low melt gel.


</P><P><U>July 17th Michael Linzey </U>Helped Mike Abboud with the Sewing PCR
-	We took his successful PCR product of the sewing PCR for N. Multiformis and used a cleaning kit on it to purify it
-	We also prepared a low melt gel to extract just the DNA from the sample 
</P><P><U> 7/18/2014 Garrett Jensen.</U> - Today the M17 agar Dr. Grose ordered a while back arrived. I poured several plates and let them cool so that I could plate our LMD-9 and let it incubate over the weekend.
 - Using one of our freezer stocks from the last time we attempted to grow up LMD-9 I pippetted 200 microliters onto 4 separate plates and streaked with a loop. I have them growing in the phage hunters lab incubator at 42 C. To prevent the plates from drying out I parafilmed all but a small section of the plate so that oxygen can still get in.  I also placed several beakers of water in the incubator to add some humidity to the air. 
</P><P><U>July 18, Michael Abboud</U>Ran low-melt, Gel slice for spacer DNA.

</P><H2>Week of July 20</H2><P><U> 7/21/2014 Garrett Jensen. </U> - Today I checked my plates that were incubating over the weekend and I had 4 that had colonies on them!!!! That was exciting. 
 - I picked two colonies off of my plates and streaked them out on a fresh M17 plate in a patch so that I can get enough bacteria to do a genomic DNA prep from 
 - I put them in the phage hunters incubator using the same circumstances as the last entry for me. They should grow up in the next 24-48 hrs. 
 - Mike and I ran a low melt gel of our plasmid for the T7 spacer and extracted the DNA.  On wednesday we will do the ligation and insert the T7 spacer into it and then transform it into E. coli for testing. 
 - On wednesday we will also do a genomic DNA prep from our LMD-9 and try a PCR for our CRISPR system!
</P><P><U>July 21 Michael Linzey </U>-	Desi gave me the results for the sequencing of the T7 spacer
-	I ran the results using a blast against what it should have been, there were no matches
-	I think this was because I dropped the tube with the DNA and some got on the sides, this prevented a good mixing of the primers with the DNA that hopefully contains the insert. </P><P><U>July 21, Michael Abboud</U>Ran low-melt for plasmid, Gel slice for plasmid.



</P><P><U> July 23 Michael Linzey </U>-	Mike Abboud was not able to come to class so I took his sewing PCR product and a cut plasmid and ligated them together
-	First I needed to melt the low melt gel using a water bath(I think out bath is broken because it took forever) 
-	Then we add 3 microliters of each gel to the master mix (water and ligase buffer)
-	Final step is to add the ligase
-	I had to leave so Garret transformed the plasmid into E Coli and grew it up on an Ampicillin plate </P><P><U>July 25 Michael Linzey </U>-	There was no growth
-	This is my fault because I incubated the ligase reaction at 35 C for a while, I should not have done that
-	So I am ligating the cut plasmid and the pcr product together for a second time
-	Then I transformed the plasmid into E Coli and plated it </P><P><U>7/25/2014 - Garrett Jensen. </U> - I started another plate of LMD-9 to get a better patch growing since the last one didnt work. I picked about 10 colonies from my other plates and streaked them out on a new M17 agar plate and incubated them at 42C. After 24 hrs I could see a good patch coming in so I streaked it out a little farther just to make sure that I would get good growth. I incubated it for 3 days total (friday to monday) and had a great patch of cells!
</P><H2>Week of July 27</H2><P><U> 7/28/2014 - Garrett Jensen. </U> - Over the last weekend I grew up a plate of LMD-9 to extract DNA from. 
 - Today I did the first half of the DNA prep, incubating with lysis buffer, proteinase K, and SDS.  I will do the rest of the DNA prep tomorrow.</P><P><U>July 28, Michael Abboud</U>No E. coli to transform. Put Eppendorf tubes into racks, covered with foil, and placed into autoclave for DH5α.


</P><P><U> 7/29/2014 - Garrett Jensen </U> - Today I finished the DNA prep from LMD-9. 
 - There actually was a pellet at the end of the DNA prep, which is AWESOME!  Never had that before.  There were some hiccups along the way though. At first there was no aqueous layer to extract after phenol/chloroform. So I had to add in 300 microliters more lysis buffer. 
 - Notes about DNA extraction
<UL>- Make sure to pull from the bottom layer of the phenol/chloroform bottle! The top layer is just a buffer, the bottom is the actual chloroform
- After you precipitate the DNA using isopropanol, dry the tube
- The sodium acetate step is a wash step to clean up the DNA a bit, ethanol will re-precipitate the DNA
- Resuspend the DNA in TE buffer.</UL>- Tomorrow I will run a gel to see if there actually is DNA. If there is then I will try to do a new PCR. If there is not then I will use the rest of the LMD-9 plate I made and do a new DNA prep.  
</P><P><U> 7/30/2014 - Garrett Jensen.</U> - I ran out 5 micL of my DNA prep from yesterday and there was a small band that was quite large! Just Like I'd expect the DNA from LMD-9 to be!
 - The band in the center lane is ours.  I asked Dr. Grose and she said that it was enough DNA to do PCR on. 


</P><H2>CRISPR GEL</H2> - I started a PCR using taq polymerase just to see if I can get it to work. I will run it out on a gel tomorrow to see if it works. Crossing my fingers! 
<P><U>July 30th Michael Linzey</U>-	I got the results back for the sequencing for the t7 spacer 
-	The spacer was not there
o	I consulted Desi and she looked over my primers
o	As we talked she told me that sometimes you need several hundred base pairs of strand in front of the area of interest to get a result
o	My primers were just about 100 base pairs away, so the sequencing may have overshot the t7 spacer, so it still may be in the plasmid
-	I did have to redesign new primers 
o	5’ GCG GCC TTT TTA CGG TTC CTG GC 3’ reverse
o	 5’ CAT TTA TCA GGG TTA TTG TCT CAT GAG CGG 3’ forward </P><P><U>July 30, Michael Abboud</U>Set up sewing PCR DNA transformation into E. coli. Plated and incubated overnight.

</P><H2>Week of August 3</H2><P><U> 8/04/2014 - Garrett Jensen. </U> - I ran the PCR out I did last weekend today. THere was a faint band for all three reactions!  It totally worked!! We have a CRISPR system in DNA! 
 - I purified the PCR using the DNA prep kit. This removed all of the PCR reaction mix and left me with just pure DNA. 
 - Mike Linzey and I prepared the nuclease reaction to cut the PCR product and plasmid vector and ligate them together. We will just incubate it overnight at 37 C as we dont have time today to actually do the CIP part of it. TOmorrow we will come in and finish the ligation and transform it into E. coli! Totally awesome that this is actually working. 
</P><P><U>August 4 Michael Linzey </U>-	Garrett was able to PCR the CRISPR system out of S. Thermophilus
o	Huge step forward
-	We then ran a restriction digest of Xba1 and Spe1 to give the CRISPR system the appropriate sticky ends
-	We also ran a restriction digest on an Igem plasmid with Chloramphenicol resistance
-	I prepared a low melt gel to purify the restriction digest products </P><P><U>August 4, Michael Abboud</U>The transformation did not work. The control Eppendorf tube was contaminated. I prepared a new control ligation and tried the transformation one more time.

</P><P><U>August 5	Michael Linzey</U>-	I treated the CRISPR product with CIP  
-	We ran the products on the gel and got bands of the correct length
-	I then cut the bands out of the low melt gel and melted the gel at 65 C 
o	We ran a ligation reaction and let it go over night </P><P><U> 8/06/2014 - Garrett Jensen. </U> - Yesterday Mike Linzey and I came in, ran the CIP digest, low melt gel purification, and started the ligation. Today we found out that you cant run CIP on the insert, only the plasmid. So we have to do this all over again. 
 - So today we started the restriction digest again. Once that is done we will redo the CIP digest and run the low melt gel then do the ligation. We will make sure to run the CIP digest on only the plasmid this time. 
 - Since we used all of the DNA I had from doing PCR to amplify the CRISPR DNA I am also starting a new PCR reaction so that we have some left over. 
 - Soon I will make up M17 broth as we got the beef extract in this week. I will grow up a liquid culture batch of LMD-9 and freeze it down in glycerol or DMSO so we have some stored if we need it later. 
</P><P><U>August 6, Michael Abboud</U>The second transformations did not work. There may have been something wrong with the ligation. I suspect that the gel slices are not low-melt so purification is necessary. Purified sewing PCR DNA and plasmid from the low-melt gel cutouts using the kit.

</P><P><U>August 6 Michael Linzey </U>-	It turns out that treating the CRISPR insert with CIP is the wrong thing to do, you only treat the vector with CIP…this meant we needed to start over
-	We set up the same Restriction Digest reaction that we had had before
-	I also set up another low melt gel, the last one we used was not as stable as I like to work with
-	This time I put CIP into the Vector so that should all work out now. </P><P><U> 8/08/2014 - Garrett Jensen </U> - Today we are doing colony PCR on our CRISPR! The colonies I transformed ysterday worked and grew up today!  They were small colonies but very isolated. Its not really suprising that they grew slowly because the CRISPR and the plasmid are quite big. In our PCR tubes 1-8 are vector+insert colonies. 9-12 are from the control plates. I'll check these plates tomorrow and see how they grew. Next monday I will run these out on a gel to see if the CRISPR insert made it into the plasmid!
</P><P><U> August 8 Michael Lnzey</U>-Helped Garrett With a colony PCR </P><H2>Week of August 10</H2><P><U> 8/11/2014 - Garrett Jensen</U> - When I grabbed my plates on Saturday they were all purple. Also, after placing the plates in the fridge over the weekend almost all of the colonies have turned purple! Apparently that happens sometimes. 
 - There were a few colonies that were not purple so we picked them and redid the colony PCR and streak. 
 - To be on the safe side I redid the ligation and plated that today. It should be ready to go tomorrow for another round of colony PCR if this one we did today does not work. 
 - The other thing I did today was to prepare the liquid media for LMD-9. I am going to grow up a liquid culture batch so that we can make a freezer stock of it for later. The incubator on the tabletop near our gel boxes wont get up to 42 C yet so I have inoculated the media yet. I'll wait for Dr. Grose to change the incubator and make sure it gets up the the right temp before I start it. 
</P><P><U> August 11 Michael Linzey</U>-Got results back for sequencing of T7 spacer
-The results looked good, it was a better sequencing than before and I found the repeat regions in the sequence!
-However it looked like the insert was not in the sequence, it looked like it had somehow perfectly skipped the sequence
-Consulted Dr. Grose, she said that if part of the insert was in the whole insert had to be in. She said that sometimes large repeat regions can cause the sequencing to skip around. 
-So it looks like we have the sequence, now we just need a functional CRISPR system</P><P><U>August 11, Michael Abboud</U>The first primer in the sewing PCR to construct the N. multiformis repeat-spacer array containing the XBaI site was found to be wrong.  Ordered correct primer.

</P><P><U>August 12 Michael Linzey</U>-Ran a CRISPR colony PCR on a gel, no CRISPR
-So I started the process over
- I cut open a new chloramphenicol resistant plasmid using Xba1 and Spe1
-After it was opened I used CIP to prevent re-connection of the plasmid
  - I ran the product through a low melt gel to purify the sample of the various protiens
  - I performed a ligation and ligated the CRISPR insert into the Igem Plasmid, I let this reaction run overnight</P><P><U> 8/13/2014 - Garrett Jensen.</U> - Yesterday I inoculated a flask of M17 media with LMD-9 and placed it in a 42C incubator on a stir plate. Today it had grown up to a high enough concentration to make freezer stocks from. 
 - I placed 1 mL of LMD-9 culture into a cryo-tube and put in 200 microliters of DMSO. I made 3 tubes and placed two in our CRISPR team freezer box and gave one to Dr. Grose for her freezer. 
 - I placed my CRISPR transformation plates in the fridge yesterday to give the colonies time to turn purple. 
 - Today most of  the colonies had turned purple but there were 7 that were clear.  - Mike and I started a colony PCR from those clear colonies. We also streaked them out. Hopefully these ones work
 - If the colony PCR doesnt work it may be because it can be difficult to get colony PCR to work for large products. Dr. Grose is ordering a new set of primers that will help to identify the CRSPR system in E. coli. 

</P><P><U>August 13, Michael Abboud</U>The following sewing Q-5 PCR sequence will be performed on six primers to construct the Repeat-spacer sequence for N. multiformis:
 - Primers #5 and #6
 - Primer #4 added to #5,6
 - Primer #3 added to #4,5,6
 - Primer #2 added to #3,4,5,6
 - Primer #1 added to #2,3,4,5,6
The sequence is backwards because we are waiting for the new Primer #1 to be delivered.
Ran the first sewing PCR reaction adding Primer #5 to Primer #6.
</P><P><U> August 13 Michael Linzey </U> - Today I used the ligation to transform the plasmid into E Coli
  - I plated them on Chloramphenicol resistant plates
  - I also helped garrett set up another colony PCR to check some colonies that he thinks may have a CRISPR system. </P><P><U> 8/15/2014 Garrett Jensen.</U> - I checked the plates from the colony PCR mike and I made on wednesday, but none of the plates grew anything at all. Sad day.  
 - I took the plates that Mike prepared yesterday and there were a few white colonies on them. I set up another colony PCR and streaked them out on plates with chloramphenicol resistance. 

</P><H2>Week of August 17</H2><P><U> 8/18/2014 Garrett Jensen/ </U> - The colonies from Mike's transformation last week worked great! We had 6 colonies streak out and stay white, one of them turned purple, but we have six that could have the CRISPR in them!
 - Dr. Grose wants to know ASAP if these have the CRISPR in them, so I did a DNA prep on 3 of the colonies today and submitted it for sequencing. We should have those results by friday. 
 We also have new primers ordered now for doing the colony PCR and setting up the gRNA region for N. eutropha. Dr. Grose has also ordered several sets of primers for amplifying small sections throughout the CRISPR system.
</P><P><U>August 18, Michael Abboud</U>Ran gel for the three PCR reactions.  All three reactions worked but the reaction #1 was the best result and will be used for future sewing


</P><P><U> 8/20/2014 Garrett Jensen</U> - Today our primers came in! So i set up PCR reactions for the new gRNA region (a soeing PCR) using both Phusion and Q5. I also set up a colony PCR for the colonies that were white after our transformation. Tomorrow I will run those out and see how it worked!
 - Useful tidbit of information: When doing a soeing PCR it works better if you use 2 of the primers and do a PCR reaction, then use that product as a primer in the next reaction. Do this until you have all of the primers used and it tends to work better than doing all of them at the same time. 
</P><P><U> 8/21/2014 Garrett Jensen.</U> - Today I ran out the PCR from yesterday. Nothing showed up in any of the lanes of the gel. I think I made a mistake setting up the PCR. I will carefully set it up again tomorrow. 
</P><P><U> 8/22/2014 Garrett Jensen</U> - Today  I set up a new PCR reaction. 
<UL> - Tubes 1-3 are a soeing PCR for the N. eutropha gRNA region (I used primers BI408 and BI409 for this), tube 4 is the control for this reaction.This reaction uses Q5 polymerase in a 50 microliter reaction. 
 - Tubes 5-7 are the soeing PCR for the N. multiformis gRNA region (I used primers BI384 and 385 for this) and tube 8 is the control. 50 microliter reaction using Q5. 
 - Tubes 1-8 labelled colony are the colony PCR reactions for the colonies that we have that have remained white. 1-6 are the colonies in numerical order from the plates that were white, tube 7 is the one purple colony, and tube 8 is the contol. (I used priemrs BI412 and 413 for this reaction.)Taq polymerase with a 25 microliter reaction was used. 
</UL></P><H2>Week of August 24</H2><P><U> 8/25/2014 Garrett Jensen</U> - Today I got the sequencing results back from the CRISPR transformation. Colonies 3 and 5 both have the CRISPR present!!! Tomorrow I'll meet with Dr. Grose to start mutagenesis of the forbidden restriction sites. 
 - We can also start testing the effectiveness of our CRISPR soon as well. We have the T7 spacer region ready and can transform that into a cell that also has the CRISPR and test its effectiveness at preventing T7 phage infection. 
</P><P><U> 8/26/2014 Garrett Jensen</U> - I met with Dr. Grose today and we started the mutagenesis reaction of the CRISPR. We used We set up two independent reactions using all forward primers in one and all reverse in the other. 
 - We used primers made to mutate the site at 1717(F:BI361 R:BI362), 1076(F:BI359 R:BI360), and 1863 (F:BI365 R:BI366)
 - When we sequence these reactions after transforming them we will see which direction worked the best and continue mutating that plasmid. 
 - Our reaction mix was 17.5 µL H2O, 2.5 µL Buffer, 0-0.5 µL Quick solution, 1.5 µL template DNA, .5 µL each primer, 1 µL dNTP, and 1 µL multi enzyme mix. *** We put .5 µL Quick solution buffer into the reverse reaction mix, 0 in the forward mix. If one works better than the other we will play with the concentration of the quick solution buffer. 

</P><P><U>August 27, Michael Abboud</U>Ran low-melt gel on PCR product #1 from the first sewing PCR. Cut out 200bp band and put in freezer.


</P></BLOCKQUOTE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BYU_Provo/Notebook/CRISPR/julyaug">http://2014.igem.org/Team:BYU_Provo/Notebook/CRISPR/julyaug</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BYU_Provo/Notebook/CRISPR/julyaug" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BYU_Provo/Notebook/CRISPR/julyaug" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CRISPR/julyaug&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CRISPR/julyaug&amp;oldid=352863" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>