"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/Metabolism/mayjune
From 2014.igem.org
BYU 2014 Notebook
Edit May June
Home
Team
Official Team Profile
Project
Parts
Modeling
Notebook
Safety
Attributions
Week of May 3rd
April 29, 2014
--CS-- /--BRK--Today we tried to get started but realized that we actually didn’t have anything that we needed. Dr. Grose could not find any P. aeruginosa PAO1 here at BYU, so we sent an email to Dr. Romesberg at The Scripps Institute requesting DNA from them. We also discovered that the antibiotic plasmid BioBricks that we need to transform are actually not in the iGEM kits that we already have, so we sent emails to the teams that submitted them last year to try to acquire the plasmids from them.
May 1, 2014
--CS-- Today we sent out some more emails to people regarding P. aeruginosa PAO1 DNA samples. Since we hadn’t heard anything from Dr. Romesberg, I sent an email to Stephen Lory at Harvard (www.pseudomonas.com says that he is the go-to guy for P. aeruginosa PAO1 samples) and Bri sent one to somebody else. Julie and I also wrote descriptions of our primers on the BYU database spreadsheet that Dr. Grose sent us. Bri also contacted the iGEM Registry people to try to track down the antibiotic BioBricks we need.
--BRK--Wrote another email to Dr. Morath at the Technische Universität München about acquiring a plasmid for erythromycin esterase type 2 (EreB) since it was not included in the 2013 IGEM Biobrick. He agreed to send dried plasmids on paper to us.
May 2, 2014
--BRK-- Emailed Dr. Olsen at the University of Washington about a sample of P.aeruginosa PAO1 DNA but he is retired and no longer affiliated with research.
Week of May 10th
May 5, 2014
--BRK--Emailed Dr. Lory at Harvard about acquiring a sample of P. aeruginosa PAO1 DNA.
May 6, 2014
--JR--Nano-dropped the Promoter plasmids that were recently purified. Almost all of the plasmids were over 100ng/ul concentration.
--CS-- We are still trying to get ahold of people for P. aeruginosa PAO1. Yesterday Bri sent the guy from www.pseudomonas.com an email and today Dr. Grose sent him one. Hopefully we will get a response from him in the near future about how we can go about getting our hands on some of that bacteria. Besides that we worked on the promoter library we have. We transformed the remaining plasmid from J23100, J23108, and J23110 into DH5α and plated that bacteria according to the protocol that we have. We also used the NanoDrop to measure the DNA concentrations of the plasmids that had already been isolated from the plasmid preps that we did as a class. All of the plasmids had pretty good concentrations and 260/280 ratios.
--BRK--Helped with nano dropping the promoter plasmid concentrations and received a response from the IGEM Part Request department that the EreB plasmid would be added to the 2014 BioBrick.
May 7, 2014
--CS-- Today we planned on doing plasmid preps for the J23100, J23108, and J23110 bacteria that we cloned yesterday but only the J23110 bacteria ended up growing and expressing RFP (which is the plasmid that we actually already have isolated— somebody had temporarily misplaced it). J23108 had a few colonies grow but none express RFP, and J23100 had nothing grow at all, so we decided to just refrigerate them and see what Desi thinks about them tomorrow.
May 8, 2014
--JR--Set up PCR to amplify Bla gene. Q5 protocol link Used psB1A3 plasmid as template with primers BI374 and BI375.
--CS-- We were planning on doing plasmid preps today for J23108 but realized when we got here that we needed to inoculate the bacteria overnight in order to do those. So Julie picked a colony and grew some bacteria up from it and will take care of the plasmid prep tomorrow (Bri and I will both be gone).
May 9, 2014
--JR--Ran a gel to check PCR products. Looks great!  Did PCR-up of PCR product using GenElute Protocol Kit link Ran restriction digests on vectors and insert. Chose the IGEM plasmids containing a strong, and medium-strong promoter strength respectively)
May 10, 2014
--JR--Restriction digests were run on the following samples: Promoter part plasmids BBa_J23102, and BBa_J23118, and Bla gene PCR product. SpeI HF, and XbaI were used with NEB buffer 4 in this protocol for each of the three samples. Restriction digest protocol
Week of May 17th
May 13, 2014
--JR--Figured out that we didn’t want to use the promoter vectors that I digested. We tried using pIG91 from the freezer, but the RD gel did not show any DNA present. Also did Sigma-Aldrich plasmid prep protocol according to the Sigma-Aldrich kit, for new pIG91.
--CS-- So today we continued work on our primers and Julie’s antibiotic part. We forgot to do plasmid preps over the weekend of the bacteria that we had grown up so we grew up some new bacteria from a colony on our J23108 and J23110 plates. To do so we used a plastic swab thing to pick a colony off the plate and add it to 5 ml LB+Amp in an incubation tube (yellow cap) and put that on a shaker in the 37°C incubator room overnight. For the antibiotic, Julie had already ran a restriction digest for the β-Lactamase part (BLa) and two different promoter plasmids. We then ran the BLa on a low-melt gel and cut out the band of our digested DNA. Since Julie cut the promoter plasmids with XbaI and SpeI we put 1 μl of calf intestine alkaline phosphatase in the digested vector and let it incubate for a while. We then realized that the promoter plasmids are in the J61002 plasmid and not the pSB1C3 plasmid, which is the plasmid backbone required for BioBrick parts. Some of the plasmid is in the freezer (called pIG91 in the plasmids II box) and the prophage group is also growing up more of the plasmid in some E. coli. The prophage pros had pelleted the bacteria, so Julie and Bri did plasmid preps to isolate the plasmid.
May 14, 2014
--CS-- Today we did the plasmid preps of the J23108 and J23110 bacteria that we grew up overnight. I also added 1 μl CIP to the restriction digest of the pIG91 vector for an hour and ran it on a low-melt gel. When we went to cut out the DNA though there was no DNA in the well for our plasmid (the ladder had DNA though so the gel worked fine), so it appears that the freezer sample of pIG91 DNA must not have been a good sample. We will try the procedure again with some of the pIG91 plasmid that was isolated yesterday.
May 15, 2014
--JR--Assisted in setting up new RD of pIG91.
Assisted Nano-dropping the recently purified pIG91 plasmisds, all concentrations were approximately 100ng/ul.
Assisted in setting up ligation of digested pIG91 (treated with CIP), with the digested Bla. Both were digested using SpeI HF and XbaI. This is so that we can then have our part in the iGEM registry plasmid to allow for quick and easy future cloning.
--CS-- Today we redid the restriction digest of pIG91 using some of the plasmid that Julie and Bri isolated on Tuesday. We followed the protocol and used the XbaI and SpeI enzymes. We added 1 μl CIP to the digest an hour before running it on a low-melt gel at 80V for about 50 minutes. We then checked the concentrations of the J23108, J23110, and pIG91 plasmids that we had isolated during the week. Everything checked out pretty well in terms of concentrations and purity.
Week of May 24th
May 20, 2014
--JR--Set up colony PCR. Chose 8 colonies from the transformation and streaked colonies onto another plate. Colonies 1-7 were white colonies. We chose colony 8 as a red colony to act as a negative control. Made a master mix (10x) the Taq PCR Protocol Taq PCR protocol
--CS-- Today we did the colony PCR for the transformed BLa E. coli that Julie had made at the end of last week. We followed the given protocol and used 7 white colonies and 1 red colony (definitely not recombined) in the PCR. Realizing that I would need a ton of the iGEM backbone plasmid (pIG91) for all of the different genes that I would be making standard parts for, I also transformed 2 μl of the previously isolated pIG91 into thawed DH5α according to the given protocol and grew it up in 5 ml LB+Cam overnight in the 37°C incubator shaker.
May 21, 2014
--JR--Ran a gel to verify PCR. Colonies 1, and 4-7 look good. Set up overnights.
--CS-- Today I did the plasmid preps of the pIG91-transformed DH5α that I grew up yesterday according to the protocol in the kit. I also confirmed that there was DNA in the products by measuring the DNA concentration with the NanoDrop. All of the samples had concentrations about 300 ng/μl and 260/280 ratios near 1.8 so they looked great. I recorded the concentrations on the tubes and put them in the freezer. I also ran our PCR products from our colony PCR on gels. Our colonies 1, 4, 5, 6, and 7 all had bands, meaning that these colonies were a success in terms of cloning and transformation. Julie also started overnight liquid cultures of these colonies from the streak plates that she made.
May 22, 2014
--JR--Did plasmid preps on overnights from colonies 4-7.
Nanodropped:
206.1 ng/ul 260/280: 1.89
215.4 ng/ul 260/280: 1.89
400.2 ng/ul 260/280:1.85
170.0 ng/ul 260/280: 1.84
--JR--Set up restriction digests on New Plasmid (pIG91+BlaGenepIG102), and promoter plasmids to build final construct with promoter and the bla gene together.
RD 5-221M, restriction digest of pIG102 cut with EcoR1 and Xba. Restriction digest protocol
RD 5-222M, and RD 5-223M
Same as above except: Enzymes used are EcoR1 HF and Spe1 HF Buffer 4 Used Two reactions, one containing promoter plasmid J23104 (5-222M), and the other J23111 (5-223M)  Ran Low Melt Gel at 90V for 46 min.
--JR--Each of the Plasmids showed up, which was good for pIG102, where we want that to be our vector. Other RD only saw the plasmids and not the promoter part which we are trying to insert into pIG102….
--CS-- Today I helped Julie a little with plasmid preps of the overnights that she had made from our successful colonies. Since P. aeruginosa came in today, we streaked out some of the bacteria on a normal LB plate. We also isolated DNA directly from stock by sticking a tip into the bacterial stock, putting that in 50 μl water, and boiling the water; we also streaked out a plate using the tip that we pulled the bacteria out of the stock with. We then used PCR to amplify the 4 denitrification genes (nirS, norB, norC, and nosZ) in 4 separate PCR reactions. We used Q5 as the polymerase since it is high-fidelity.
May 23, 2014
--CS-- Today I took my streak plate of P. aeruginosa PAO1 out of the 30°C incubator, parafilmed it, and stuck it in the fridge. There were lots of colonies on the plate so things are looking good!
Week of May 31st
May 27, 2014
--JR--Set up RD  of promoter plasmids J23101, and J23106 Used 3 RD enzymes because we will not be able to run on low melt and get out promoter part, so we want to ruin the plasmid the promoter is in so that it doesn’t relegate. Used Enzymes EcoRI, PstI, SpeI
Restriction digestst ran on promoter parts J23101, and J23106 according to restriction digest protocol. Restriction digest protocol
Because product we wanted from digest was so small, running a low melt gel would not be plausible. Thus we instead chose to inactivate enzymes thermally.To destroy enzymes reaction mix was incubated at 80C for 20 minutes, following restriction digest protocol and incubation.
Promoter digests and plasmid vector were each ligated together and incubated at room temperature according to ligation protocol ligation protocol
--JR--Ligation mixtures were each transformed into DH5alpha competent cells according to transformation protocol and then plated on both LB+Cam, and LB+Cam+Amp plates. Plates with both chloramphenicol and ampicillin anitibiotics we expect will select for successful transformants, those which have the standard plasmid backbone and a functioning promoter and beta-lactamase gene
--CS-- Today I ran a gel of the PCR products that I got from last week using a normal gel running at 175 V for about 20 minutes. I included a picture of my gel below:
All of my genes amplified as they should! They are all the appropriate length and have nice defined bands so I continued by using the PCR purification kit to clean up my DNA. I then ran restriction digests of my genes and the pIG91 vector with XbaI and SpeI, using 1 μl CIP to keep the vector from self-annealing. I loaded the digested genes onto a low-melt gel, ran it for about 45 minutes, and then cut out the sections that contained my desired genes. The bands were very faint in my low-melt gel though so I am not sure how well they will actually turn out. I then followed the ligation procedure in the protocol to insert my genes into the backbones. I let these go at room temperature overnight.
I also transformed some promoter plasmids (J23102, J23104, J23111, and J23118) into DH5α and plated it out on some LB+Amp plates overnight to replenish some of the promoter plasmid stocks that Julie had used up in her experiments.
May 28, 2014
--CS-- Today I picked some colonies from the DH5α that I had transformed with the 4 promoter plasmids and put it in 5 ml of LB+Amp to grow overnight. I then transformed DH5α with the ligations of the 4 denitrification genes and plated that out to grow overnight on LB+Cam plates.
May 29, 2014
--JR--Took 8 colony samples from each of the LB+Amp+Cam plates and set up colony PCRs. Did according to Taq colony PCR protocol Used primer bIG307 pSB1C3 forward and bIG375 which is for Bla reverse.
--CS-- Today I did the plasmid preps of the 4 promoters that I had grown up; I will need to do the NanoDrop of these later. I also did colony PCR for all 4 of the denitrification genes that I have been working with this week. I used Taq polymerase and made the reaction according to the protocol. I used the forward pSB1C3 primer (#307) and the reverse primer for each of my genes to check that the gene had been inserted into the plasmid in the correct orientation and transformed correctly into the bacteria. While picking plaques for the colony PCR I also streaked out the different colonies on LB+Cam plates and grew them up in the 37°C overnight.
May 30, 2014
--CS-- Today I got my streak plates out of the incubator and put them into the fridge. Pretty much all of my colonies streaked out and did not show signs of RFP, so things are looking pretty good so far. I will run the colony PCR gel next week to see exactly what happened.
Week of June 7th
June 2, 2014
--CS-- Today I ran the gels of the colony PCR product that I made last week. The images are below:
Images on top from left to right: nirS and norB. Images on bottom from left to right: norC and nosZ.
These digital images are awful, but there were some bands that showed up for nirS and norB. Only one of these actually ended up being the appropriate length though when compared with the ladder, so I need to start all over with the cloning PCR to amplify the genes from the P. aeruginosa DNA.
June 3, 2014
--JR--Confirmed colonies via plating and colony PCR.  Colony PCR showed colonies positive for colonies 1-4, 5, 6, and 8 for transformation of ligation 5-273M. However, on the plate only colony 8 showed up without RFP, thus I selected this colony as one that would be considered a final construct. Same process was repeated for transformed ligation 5-274M, where colonies 1, and 6 showed up on colony PCR, and where colony 1 was white—colony 1 was selected as a final construct. Overnights were set up and grown at 37C.
--CS-- Bri and I mixed up some Q5 reaction stuff and did the PCR for norB, norC, and nosZ. I then ran the PCR products on a normal gel. The gel looked good and it appeared that all of our genes were amplified. An image is below:
ADD IMAGE HERE
I then did restriction digests of my PCR products. I followed the protocol but made changes in that I used all 50 μl of my PCR product (Dr. Grose said she usually uses all of her DNA and I wouldn’t have enough leftover otherwise so I figured I’d try it), didn’t add any water, added 2 μl of each restriction enzyme (usually do 1.5 μl), and added the enzymes very last after mixing up everything else first (last time I made a master mix including the restriction enzymes but I noticed this time that the protocol emphasized putting the enzymes in last). I then set my reactions in the 37°C incubator overnight (last time I think I might have left them at room temperature overnight, so that might be another reason why they didn’t work well).
June 4, 2014
--JR--Plasmid Preps were ran on the overnight samples.
--CS-- Today I ran a low-melt gel of the restriction digest products for norB, norC, nosZ, and pIG91. This time everything came out with nice clear, bright bands so I think things have worked so far this time around. I cut out the bands from the gel and stuck them in the freezer.
June 5, 2014
Plasmids were nano-dropped.
Lig 274M-1 415.2ng/ul and 260/280: 1.86
Lig273M-8  515.6ng/ul and 260/280: 1.86
I also renamed these plasmids and submitted them to our BYU iGEM parts database.
Ligation273M-8 will be pIG105
Ligation 273M-9 will be pIG106
--CS-- Today I did the ligations of norB, norC, and nosZ into pIG91. If these don’t work I have plenty of stuff left over from my low-melt that I’m very certain of, so I can go back to this point later if needs be. I then transformed my ligated plasmids into DH5α and plated them on LB+CAM plates overnight according to the protocol.
June 5, 2014
--CS-- Today I did the colony PCR of my transformed bacteria. I then ran a gel of the PCR products. My gels looked like this:
Image on left is gel including DNA ladder, the 8 selected norB colonies, another DNA ladder, and the first 4 selected nosZ colonies. Image on right is gel including DNA ladder, the 8 selected norC colonies, another DNA ladder, and the last 4 selected nosZ colonies.
Based on the gene lengths (norB is 1400 bp, norC is 440 bp, and nosZ is 1910 bp), it appears that my norC gene worked well but I’m not really sure on the other ones. I will talk to Dr. Grose about it next week. I also streaked out plates with the colonies that I had picked for PCR and put those in the 37°C incubator.
Week of June 14th
June 10, 2014
--JR--Looked at iGEM webpage, tried to figure out some code things….
--CS-- So I talked to Dr. Grose about my colony PCR results and since there were some solid bands for norB and nosZ but the bands were not in the spot they are supposed to be (both appear to be shorter than they should be), I did another colony PCR with the DNA from the colonies. This time I used the pSB1C3 forward and reverse primers (307 and 308) to check since those primers are known to work and the primers appeared to have caused the problem with my PCR (there was a laddering pattern observed in the gel, suggesting that the binding wasn’t very good). I then ran the PCR products on a gel. My gels looked like this, with 4 DNA ladders spread throughout the 8 selected norB colonies, 7 of the selected nosZ colonies, and a norC positive control from a previous experiment that had worked properly:
All of these bands are for rather short sequences, suggesting that the pSB1C3 vectors closed in on themselves instead of adding the genes like we hoped they would during the incubation. I used CIP during the restriction digest, so this should not have been a problem but apparently it was.
June 12, 2014
--JR--Set up Q5 PCR for genes NorB, NosZ according to the Q5 PCR protocol Q5 PCR protocol using P.Aeroginosa as template and NorB forward and NorB reverse primers, BI335 and BI336 respectively.
--CS-- Since my restriction digest appeared not to work, I started everything all over again today for norB and nosZ. Julie and I used Q5 to amplify these genes from the P. aeruginosa genome after putting a colony in 50 μl water and boiling it to release the DNA. The gel of the PCR looked something like this:
The first lane after the ladder is norB, which looks right since it is a 1400 bp gene. Then there were 3 bands that showed up for nosZ in the right lane. Since one of these bands is where it should be, we will continue with the restriction digest though and use the low-melt gel to pick the portion we want if these extra bands show up there too.
June 13, 2014
--JR--Restriction digest of PCR products of NorB and NosZ according to restriction digest protocol Restriction digest protocol After incubation for 2 hours restriction digest was treated with CIP to prevent vector from religating
Transformation
--JR--Transformed ligation mixes into competent DH5alpha according to transformation protocol transformation protocol except used 4ul of ligation mix instead of 2.
--CS-- Today Julie and I ran a restriction digest of norB and nosZ using XbaI and SpeI enzymes according to the protocol. We added 1 μl CIP 1 hour prior to running the low-melt gel to prevent self-annealing of the backbone. We then ran the low-melt gel and cut out the bands that contained our genes and the vector. Here is an image of the low-melt gel:
The pIG91 digest turned out really well but the other ones were a lot fainter. We then melted the cutout bands and did ligations according to protocol. Julie then transformed these into bacteria and plated them on LB+CAM and put them in the 37°C incubator overnight.
Week of June 21st
June 17, 2014
--CS-- Today I ran the colony PCR of the transformed bacteria that Julie had grown up last week. The gel of my PCR products looked like this:
The image on the left has 7 norB colonies with a negative control (no insert) and the image on the right has 7 nosZ colonies with a negative control. Bet you can’t tell which lane is the negative control! So it appears that this round of cloning didn’t work either. I went through the whole procedure with Julie and we were unable to figure it all out, so we think it might be a primer problem but we really have no idea.
June 19, 2014
--CS-- So things haven’t been working well for norB and nosZ. For some reason I constantly get gels that look like this:
The norB lane (the middle lane) seems to be blurry the majority of the time, and the nosZ lane (the right lane) seems to always have 3 bands in it. This makes it seem that the primers I have for these two genes are screwy, so I looked at my primer sequences again. Both of the primers have the correct sequences and have 24-25 bp of the gene start/finish in addition to the correct restriction digest sites and the extra 3 bp that help the restriction enzymes bind. I blasted the P. aeruginosa PAO1 genome with the primer sequences and found that there are actually quite a lot of sites in the genome that match the sequences. Besides the appropriate sites of the genes that the primers were designed for, these matches are much weaker though, having no more than 15 aligning nucleotides for the most part. I tried to look for some pairs of sites that would result in the extra bands found in the nosZ lanes, but I couldn’t identify anything obvious that would work for that. So I am unsure as to why norB always smears and nosZ always has 3 bands and why neither of them works. I can amplify both of them from the P. aeruginosa genome but somewhere after that things don’t work.
Week of June 28th
June 24, 2014
--CS-- Today I discovered the source of my cloning PCR problem. The reverse primers for all 4 of my denitrification genes were wrong! I don’t know how it happened but somehow the beginning of each of the gene sequences in the primers got switched around; the following shows what the primers are and what they should be:
nirS reverse (with SpeI site)
Incorrect primer sequence: ccgACTAGTATCAGTACACGTCGTGCTGGGTGTT
Correct primer sequence: ccgACTAGTTCAGTACACGTCGTGCTGGGTGTT
norB reverse (with SpeI site)
Incorrect primer sequence: ccgACTAGTATCAGGCGGCCGCCTTGCCGCGCCGG
Correct primer sequence: ccgACTAGTTCAGGCGGCCGCCTTGCCGCGCCGG
norC reverse (with SpeI site)
Incorrect primer sequence: cggACTAGTATCAACCCTCCTTGTTCGGCGGCCA
Correct primer sequence: cggACTAGTTCAACCCTCCTTGTTCGGCGGCCA
nosZ reverse (with SpeI site)
Incorrect primer sequence: ccgACTAGTATCAAGCCTTTTCCACCAGCATCCGC
Correct primer sequence: ccgACTAGTTCAAGCCTTTTCCACCAGCATCCGC
I compared the forward and mutagenic primers that we have to what I originally designed and they were all the same, so the problem only exists with these reverse primers. I went through the primers I originally designed too and double checked to make sure they were good for the genes and restriction sites, and they all seem to be alright in those regards. So it is interesting that norC and nirS actually worked despite the error. I sent Dr. Grose a summary of the primer issue and she will order some new primers so that we can get working on cloning the genes with the right primers next week.
June 25, 2014
--CS-- Today we prepared our presentations. We then gave them.
June 26, 2014
--CS-- Today I talked to Dr. Breakwell about our project to get some ideas about how to make the denitrification pathway work better in N. multiformis. He talked to me a lot about how the type of bacteria that N. multiformis is has a lot to do with whether or not adding the denitrification genes into it will work. We went through the redox reactions that are involved in ammonia oxidation (what N. multiformis already does) and denitrification (what we want N. multiformis to do) and found that the energy produced by the bacteria when using O2 as an electron acceptor (aerobic respiration) is more than that which would be produced when converting the nitrate all the way to nitrogen gas (which is anaerobic); this means that the bacteria will likely not prefer producing nitrogen over nitrate. He said that we would likely need to find a way of reducing the oxygen in the activated sludge as part of a cycle to promote the N. multiformis to do denitrification. He also said that knocking out the gene that converts nitrite to nitrate might work in forcing the pathway through to completion but we really don’t know until we try. He also said that having a constitutive promoter on the plasmid with all of the necessary genes might also do the trick; if the proteins are being made irrespective to the bacteria’s demands then the bacteria might just go ahead and use them to get more energy. I also ran my ideas for assaying the denitrification plasmid in E. coli and later in N. multiformis; he thought the Durham test would be a good test and also suggested we test the nitrate level of the broth before and after. Doing both will allow us to confirm if nitrate is being used up and if nitrogen gas is being produced as we designed. He also informed me that E. coli naturally denitrifies in anaerobic conditions when nitrate is present, so for testing our plasmid out in E. coli we will need to use a strain that is knocked out for denitrification genes.
Retrieved from " http://2014.igem.org/Team:BYU_Provo/Notebook/Metabolism/mayjune "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
