"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/CRISPR/mayjune
From 2014.igem.org
BYU 2014 Notebook
Edit May June
Home
Team
Official Team Profile
Project
Parts
Modeling
Notebook
Safety
Attributions
Week of April 27
Week of April 25-May 2, 2014: Michael Abboud
Wet-lab work for our project was delayed because we still need the correct strain of Streptococcus thermophilus for template DNA. E-mails were sent to several different labs around the world that work with this particular strain to send some to us. The iGEM plasmid transformations were performed as given in the Method section of this journal entry with a caveat. The first time we did the transformations, it failed because we let the bacteria incubate for only 30 minutes which was not enough time for the antibiotic resistance to be expressed. The second time, we followed the method exactly. Success of the transformation will be determined next week.
5/01/14 - Garrett Jensen.
- Today we redid our transformation of the igem plasmid into E coli DH5α. The protocol is the same as from 4/27/14 EXCEPT we incubated for 90 minutes instead of 30.
- Mike is going to check them tomorrow and start he overnights.
- Today we also emailed several universities that have used S. thermohilus DGCC7710 and asked for a sample of it so that we can take the CRISPR from it and move it into N. mutiformis. Mike Linzey had one email him back already and is working with them to get it.
- I read up on how to design spacers for the crispr to use, the resources are found on the website http://www.addgene.org/CRISPR/guide/  under the section heading designing spacers.
- We will need to take the most consvered regions of our phage and run them through the online programs they use to find a good region to choose as a spacer for each of our phages.
May 1 Michael Linzey
-We found out that we should have incubated it for 90 minutes…no E Coli survived the chloramphenicol
oWe repeated the process from the 29th but included the appropriate amount of time for the incubation
-We also contacted and got responses from laboratories that could provide the s. thermophilus
May 2 Michael Linzey
- I prepared an overnight for the bacteria
o I swabbed a bacteria colony (it worked this time) and put it into 5 ml of LB so that it can grow over night.
Week of May 4
Week of May 2-9, 2014: Michael Abboud
All labs that were contacted for ordering S. thermophilus DGCC7710 got it from the same lab, so we had to order from that one. While it is in transit, the repeat-spacer sewing primers are being designed. I am in the process of finishing up the primer design.
5/06/14 - Garrett Jensen.
- Today we are working on designing our spacers for our PAM sequence.
- They need to be in a well conserved region of the gene, so we need to BLAST them and look for well conserved regions of that gene, these are found by identifying the regions that are aligned more completely with less mutations across all of the BLAST hits.
- First we are working on the capsid protein of phage 1.  the work for this is stored on garetts computer, under the tab gRNA in onenote.  On that page the PAM sequences are highlighted as well and I have a raw copy of the DNA for the capsid protein of phage 1.  I will get all this information for all of our phage proteins we use as well.
May 6 Michael Linzey
-started the long process of creating the primers for the Multiformis spacer
-to create the space we needed to choose several possible DNA sequences from conserved proteins
-I chose a structural protein, a tail fiber protein, and a lysosyme protein as an alternate
-Each primer was 20 base pairs that preceded a PAM sequence
-Each primer also needed the complete palindromic repeat on one side and half the repeat on the other side, this would allow us to use sewing PCR to bind them all together.
-These spacer region is what gives the CRISPR system its unique abilities to cut viral DNA
Structure Protein Primer:5’ CCCCTACCTATAATCGTCATTTTGTAGTGAAC-pam 3’
Tail Fiber Protein:5’ gacagcttaaccatcgcgcaaggaggacgag-pam 3’
5/08/14 - Garrett Jensen.
- Our assignment is due may 22.  we need to outline what we want to accomplish this semester.
- Today I worked on making primers for our gRNA spacers.
- My work so far is found under the gRNA tab.  I have a primer made for the integrase and primase of phage 1.
○ Phage 1 is in the region 53982-71088 on N. multiformis ATCC 25196 genome.
○ Integrase: is found between nucleotides 69640-70902.
PRIMER: 3'-gaaagagctaggacctagcatgctacaaac-5'
Target Sequence 5'-ctttctcgatcctggatcgtacgatgtttg-3'
○ Primase: found between nucleotides 66910-67335.
PRIMER:3'-acctccaaagcgtagggcgcttcagtacgt-5'
Target Sequence: 5'-tGGAggtttcgcatcccgcgaagtcatgca-3'
- **I still need to add the rest of the primer.  These primers will be used to do a sewing PCR to make our CRISPR repeat/spacer sequence.  These primers are specifically for the target sequence that we want the CRISPR to attack.
May 8 Michael Linzey
- Started an overnight
o Used the wooden dowel to put some E Coli in 5 ml of LB
o Left in 37 C incubator overnight
May 9 Michael Linzey
- Collected the E Coli cells
o Put the E Coli and LB mix in centrifuge tubes and centrifuged them
o Poured out the broth on top and collected the pellet of E Coli
o Did this 3 times with 5 test tubes
o Put in the freezer
Week of May 11
Week of May 9-16, 2014: Michael Abboud
Literature study has revealed that Streptococcus thermophilus LMD-9 has a CRISPR system that is incredibly similar. It has the exact same Type II, CRISPR03 system with Cas9 endonuclease as DGCC7710. The repeat sequences and PAM sequence is the exact same. The only difference is that the tracrRNA is a little more downstream so a new primer has been designed to account for that. The “forbidden” restriction sites are in the exact same spot so we can keep the primers already designed for point mutations. We also finished up our primer designs for the repeat-spacer array, targeting highly conserved areas of the prophage that affect N. multiformis immediately before the PAM: NGGNG.
5/13/14 - Garrett Jensen.
- Today I found out through emailing  between Dr. Barrangou and Dr. Grose that the strain of S. thermohpilus we wanted to use, DGCC7710, is a commercially owned strain by DuPont and we are not able to use it. We will need to use a new strain of S. thermophilus.
- We are going to use strain LMD-9. it has the same CRISPR system as DGCC7710 does. Blast showed a 99% identity with the CRISPR from DGCC7710. LMD-9 is also a publicly available strain.
- The repeats are the same ( 5'-gtttta gagctgtgtt gtttcgaatg gttccaaaac-3'), the PAM sequence is the same (NGGNG), the tracrRNA is NOT in the same place however.
- We are designing a new primer for this strain to be able to PCR the crispr system and also get the tracrRNA which is required for CRISPR spacer acquisition. Michael Linzey is working on the new primer.
- We emailed several different research labs today asking If we can ge ta sample of S. thermophilus LMD-9 from them.
May 13 Michael Linzey
- Today we found out that we were not going to be able to get the s. thermophilus strain that we were hoping to get to we needed to switch
- Streptococcus Thermophilus LMD-9 became our new strain
o Spent the class time seeing if we were going to need to change our primers
o All were good except the reverse primer
o We also designed the spacer primers that we will need to put in the crispr array
5/15/14 - Garrett Jensen.
-Our forward primer for taking the CRISPR from LMD-9 actually bind to the bottom strand of the genome. LMD-9 has the crispr system on the bottom strand of DNA, not the top strand. Because of this our reverse primer will be from the top strand onf LMD-9
- We sent out new reverse primer for the crispr array to Dr. Grose today for her to check
- Mike Linzey also found a professor that will send us a sample of LMD-9.  He sent Dr. Grose's address to him yesterday, so our LMD-9 should be on its way soon.
- I have primerse designed for making our phage spacers.  One if for targeting the integrase, the other is for targeting the primase.
○ Integrase:  This primer binds to next to the PAM at nucleotide 885 of N. multiformis. It binds downstream of this PAM sequence so that when Cas9     ○ 5'-gttttggaaccattcgaaacaacacagctctaaaac caaacatcgtacgatccaggatcgagaaag gttttggaaccattcgaaac-3'
- We also started some overnights today so that we can do plasmid preps tomorrow so that we wil have our plasmids ready for when our LMD-9 gets here.
May 15 Michael Linzey
- Started an overnight to produce more plasmids
- Made sure that everything was ready to send to Prof. Grose so she can order our primers
- We also designed a crispr primer for T7 so we have something to use on the E Coli that we will test this on
May 16 Michael Linzey
- I did a plasmid prep and spun some of the overnights down
o Purified 2 eppendorf tubes worth of plasmid
Week of May 18
Week of May 16-22, 2014: Michael Abboud
Streptococcus thermophilus LMD-9 has been ordered from a French lab and is on its way. Aligning the primers in Notepad revealed that we had used reverse complements of some of the repeats in the primers and that would have ruined the PCR. Corrections were made and the sewing primers now line up how we want them to. Unfortunately, this is the most ideal PCR and we will likely get dimerization and incorrect lining up of primers. Sequencing the DNA will help us see if we got the correct alignment.
5/19/14 - Garrett Jensen.
- Today we kept working on our primers.  We made them all with Dr. grose for the soeing PCR for making the CRISPR repeat region for multiformis.  We talked about how to make these primers, and then we put them on the googledoc primer sheet.  We also received our new reverse primer for removing the CRISPR from LMD-9.
- Our primers are
○ Primer with repeat:5'-ttatctagaggttttggaaccattcgaaacaacacagctctaaaac caaacatcgtacgatccaggatcgagaaag   gttttggaaccattcgaa-3'
5'- TTCGAATGGTTCCAAAAC    TGGAGGTTTCGCATCCCGCGAAGTCATGCA  GTTTTAGAGCTGTGTTGTTTCGAATGGTTCCAAAAC-3'
5'-gttttggaaccattcgaaacaacacagctctaaaac
ATAAGCATGGTGTCCAGCTCAAATTGATTG(PAM)
gttttggaaccattcgaa-3'
5'-TTCGAATGGTTCCAAAAC
GCGCAAATTGGAACACAAGGTAATGACCTT
GTTTTAGAGCTGTGTTGTTTCGAATGGTTCCAAAAC-3'
5’ gttttggaaccattcgaaacaacacagctctaaaac GTTCACTACAAAATGACGATTATAGGTAGGGG gttttggaaccattcgaa 3’
5'-attactagtaTTCGAATGGTTCCAAAAC     GTATTTTACTGGACCAGGCACAGCTGCATT GTTTTAGAGCTGTGTTGTTTCGAATGGTTCCAAAAC -3'
- I also made a primer for the T7 phage to put into the spacer region so that we can test the CRISPR's effectiveness in E coli.
- After this we worked on our class assignment with goals for the rest of the semester and sent that to Desi
hope to receive our S. thermophilus LMD-9 by May 27
We will PCR our CRISPR from LMD-9 on May 27 and run a quality check gel to make sure it is the right size.
We will ligate the CRISPR into the iGEM plasmid on  May 28
Place T7 phage spacer into CRISPR repeat region May 27-29
Clone T7 spacer plasmid and the Crispr Plasmid into E. coli May 27-29
June 2-6. We will test the effectiveness of our CRISPR against T7 phage. We will talk to Dr. Grose/Desi/Skip at this point to see what methods are the best for evaluating its effectiveness, whether spot test or incubating with phage and counting pfu/mL or another method. We will compare our tests to controls that do not have the CRISPR system.
Make N. multiformis spacer region against its prophages on June 10. Place Repeat Spacer Region into Plasmid - June 12
Clone our completed CRISPR system into E. coli for conjugation into multiformis on June 13
We will spend the rest of the semester transferring our finished crispr system into N.  multiformis.
May 20 Michael Linzey - Today we worked on our outline and the different goals we have for our project o We put dates to each experiment that we would like to perform and when we would like to perform them - We also finished up our CRISPR array primers o We had to spend a long time working on them because they need to fit together just right so that the primers will “sew” themselves together so our 6 primers can become 1 long pcr product. Since then we have been working on supporting other groups as we wait for our s. thermophilus to show up from a laboratory in Belgium.
Week of May 25
Week of May 22-30, 2014: Michael Abboud
Streptococcus thermophilus LMD-9 arrived after class on May 27. The following class period on May 29, a skim milk agar solution was made following the media recipe in Dr. Breakwell’s book but cutting the amount in half. During the boiling phase of the recipe, the solution boiled over on the hot plate and we lost most of it. Fortunately, there was still enough to use for our purposes. It was autoclaved, poured into plates, and inoculated with LMD-9. A summary of our project was submitted to the group working on the grant. We also assisted in revising the grant paper. 5/27-5/29 - Garrett Jensen.
- On Tuesday we received our sample of S. thermophilus LMD-9 and we ordered M17 media to grow it in.  I helped Jordan Berg do some PCR reactions for his project that day as well before our strain came in.
- On Thursday we spent the majority of class editing our grant proposal.
- The remainder of class I spent making milk agar to try to grow our S. thermophilus on.  It didn’t turn out well, the milk turned brown and chunky.  I think the autoclave ran two cycles and overcooked the milk
- On Friday I came in and plated some LMD-9 on our plates and then made more media to make additional plates.
- The second batch of milk agar did not turn brown or chunky, so I made a sleeve of plates and Michael Linzey came in on Saturday to plate more S. thermophilus
- Milk agar recipe
○ 15 g Agar
○ 5 g peptone
○ 2.5 g yeast extract
○ 1 g glucose
○ 100 mL Skim milk solution
- Milk solution:  add skim  milk solids to distilled water and mix.  Autoclave for 15 min at 15 PSI at 121 C.
- Medium: add all components except the milk solution. Mix, gently heat and boil, then autoclave for 15 min at 15 PSI at 121 C.  Cool to 50 C and add the milk solution.
- Growing S. thermophilus is best done between 40-45 C and purportedly in anaerobic conditions.  I am not sure where an anaerobic chamber that can be heated to 45 C is at, so we will need to ask.
May 27 Michael Linzey
- We worked on finalizing all of our plans for our project
- We were still waiting on s. thermophilus to show up from France
- I emailed the French team to ask if they sent it yet
- Received an email saying they sent it
May 29 Michael Linzey
- S. Thermophilus came!
- We needed to make an agar plate to grow up the bacteria
- We found that we could use a milk agar plate to grow it up on, we also ordered another media for s. thermophilus growth
- To make the milk agar we needed:
- Dry milk
- Water
- Pentone
- Glucose
- Yeast
- Agar
- You mix the dry milk and water in one flask and autoclave it seperatly from the rest, then you mix it together
- I think someone may have set our autoclave to run twice, the milk turned brown and chunky
- Garrett remade the agar plates
Week of June 1
Week of May 30-June 6, 2014: Michael Abboud
Our initial Q9 PCR reaction did not work. We ran a gel and no bands showed up in either the control or the product. We later found out with that our CRISPR system being around 7000bp, the elongation time on the PCR cycle should be much longer. Oops. The new PCR reaction (using Phusion) was set up and is currently running. An overnight culture of DH5alpha was also produced. The next steps are to clone the CRISPR into DH5alpha along with our T7 repeat-spacer (a small array with only one repeat targeting T7 phage and needs no PCR) and test the effectiveness before cloning into N. multiformis.
May 31 Michael Linzey
• Plated some of the bacteria, now we are just hoping for growth.
June 3 Michael Linzey
- We found s. thermophilus colonies on our milk agar plates
- Dr. Grose found more errors in our primers so we had to redo a couple of them
o Now we know that they are as good as we are going to be able to get them
- Set up a PCR, Q5 polymerase
June 5 Michael Linzey
- Ran a gel
o PCR didn’t work
- It turns out that because we are pcring a large segment of DNA the PCR will take longer, about 6 hours
longer
o Our segment of interest never even had a chance to really elongate
- Redid PCR with phusion this time
- Transformed a new igem plasmid into E Coli, need amp resistance for our second plasmid
o Plated it
Week of June 6-13, 2014: Michael Abboud
Both the Q9 and Phusion PCR reactions did not work again. The Phusion ended up giving us a long streak down the gel and the other wells were completely blank. Since Streptococcus thermophilus is gram-positive, boiling may not be effective in retrieving the template DNA. Next week we will prepare the DNA using a genomic-prep kit and run the PCR again.
6/06/2014 - Garrett Jensen.
• This week we got to work doing PCR on our LMD-9 to get our CRISPR system.
• On Tuesday we boiled a small sample of our LMD-9 that we grew over the weekend and then did PCR with Q5 polymerase
• On Thursday we ran the PCR on a gel to verify that it worked, but no band was visible.  We found out from desi later that the longer you insert is the longer the thermocycler needs to run for. Our cycle needs to be 14 minutes long, not 3.  So we redid out PCR and Michael Linzey is coming in on Friday to check for product.
• We also learned that we will be able to do our mutation PCR all at the same time, all 5 of them.  If we use only the forward or reverse primers we can do it all in one batch, this PCR will take a long time however, we are estimating 24 hours.
• On Thursday we also transfored some E. coli DH5 alpha with another iGEM plasmid, pSB1A3, because we will need another plasmid with a different selectable marker from pSB1C3.  Our spacer region will not be on the same plasmid as our CRISPR, so we needed to be able to select for both pieces.
• Next week we hope that our primers for our repeat region will arrive and that we will be able to transform our CRISPR system and the repeat region into E. coli for testing.
June 6 Michael Linzey
- I came in to run a gel
- We lost some of the product in the template tube
- Someone suggested that I centrifuge it down
- That didn’t work, somehow I spun all of the pcr product out of the tubes.
- So I reset up PCR again and ran it again
- Set up an overnight with the E Coli
- Also plated more s. thermophiles
June 7 Michael Linzey
- Ran gel
- Spun all of the e coli down
Week of June 8
6/10/2014 - Garrett Jensen.
- On Thursday last week our PCR did not work again. Today we are doing it again, but we are running the reaction with 2 different polymerases, Q5 and Phusion.
- Mike prepared the PCR reaction with phusion and I prepared it with Q5.
- Michael prepared the igem plasmid pSB1A3 over the weekend for the T7 spacer repeat region.
- Today the T7 spacer primers came in and he is ligating them into the plasmid.
- Once the CRISPR region is ready we will ligate it into the pSB1C3 plasmid.
- After that we are going to transform both of these plasmids into E. coli and test them
June 10 Michael Linzey
- Grew bacteria that contained Igem plasmid A3 which is amp resistant
- Removed the bacteria
- Used a restriction digest (xba and spe) to cut the plasmid
- Ligated in our T7 spacer
June 11 Michael Linzey
- Transformed our T7 spacer plasmid into E Coli
- Plated it and allowed to grow over night
June 12 Michael Linzey
- Growth on plate, but digested and undigested vectors were present in bacteria, red and white colonies
- Plated 8 white colonies on a single plate to grow
- Placed a sample of each colony into a pcr tube
o Use taq pcr to check to see if any of the bacteria had a t7 spacer added into it
- Used sewing PCR to prepare the spacer region for the crispr system that we will put into N. Multiformis
June 13 Michael Linzey
- Ran a gel to see if sewing pcr worked
- It worked!
-We expected to see a thick fuzzy band around the 500 or 600 base pair band on the ladder because of the possibility of the sewing PCR fitting together in different comfirmations
-However the PCR product was accidentally discard so it had to be repeated
Week of June 15
6/17/2014 - Garrett Jensen.
- Today mike and I did a genomic DNA prep of S. thermo with Skip because the PCR after boiling it was not working. We had prepared a plate of S. thermo last week and let it incubate for 4 days, however it did not grow well. We don’t think that there was enough cells in our prep to get enough DNA for PCR.
- Mike abboud prepared PCR with that genomic prep anyways to see if it would work.
- I prepared a liquid culture of S. thermo just in case the PCR didn’t work for Mike.
- Michael Linzey is still working on the T7 spacer plasmid for the CRISPR.
- He ran his PCR on a gel and found that it had worked. He also did a low melt gel to harvest his plasmid-T7 DNA.
6/19/2014 - Garrett Jensen.
- Ran CRISPR PCR on a gel, no band showed up again.
- Using the liquid cultures that I made on Tuesday we did another genomic DNA prep with Skip, but no pellet was visible
- Genomic DNA Prep Protocol
- Resuspend cells in 700 micL P1 buffer
- Add 7 micL lysozyme
- Incubate at 37 C for 30 min
- Add 7.5 micL 10% SDS, invert 4 times
- Heat at 65 C for 5 min
- Add 100 micL 5M NaCl
- Add 500 micL CHCl3. shake vigorously
- Spin 10 min max speed
- Combine supernatant with equal volume isopropanol
- Spin 5 min max speed
- Remove super and dry well. Resuspend in T3E3 Buffer
- I am going to do a RedTaq PCR using our CRISPR primers as well as a set of universal 16S Ribosomal primers that Skip gave me to see if there is even any DNA present in our sample.
- PCR Protocol for Red Taq
○ 2.5 micL 10x Buffer
○ 1 micL dNTP
○ 1 micL each primer
○ 1 micL Template DNA
○ 17 micL H2O
○ 1.5 micL red taq
- *** Polymerase needs 2 minutes for every 1000 bp of the product.  The product from Skip's primers should be 600 bp long, the product from the CRISPR primers should be 7,000 bp long.
- Tube 1 is the control using Skips primers
- Tube 2 has genomic DNA and skips priemrs
- Tube 3 and 4 both have genomic DNA and our CRISPR primers
- Tube 5  (the standalone tube) has no template DNA and our CRISPR primers
- The PCR did not work.  I ran the gel out on 6/20/2014 and there were no bands with either the CRISPR primers or Skips universal primers
CRISPR GEL
- Skip is trying a different DNA isolation method that will hopefully work
- We have 3 more tubes in the freezer with S. thermophilus LMD-9 in them in the CRISPR box
Week of June 22
6/23/2014 - Garrett Jensen.
- Protocol for isolating genomic DNA from S. thermophilus
- http://www.idosi.org/mejsr/mejsr11%281%2912/3.pdf
- Today Skip wasn’t here. Since the PCR I did last week didn’t work I looked for other protocols for isolating genomic DNA from S. thermophilus and found the paper above that is specifically for isolating DNA for PCR reactions.
- Today I isolated the DNA from LMD-9
- Using the frozen overnight cultures I made previously I discarded the supernatant
- Resuspend cells in 200 micL lysis buffer (6.7% sucrose; 50mM Tris/HCl, pH7.0; 1mM EDTA)
- Immediately add 10 micL proteinase K (10mg/mL)
- Incubate for 30 min at 37 C   - Vortex to break up chunks,
- Add 7 micL 20% SDS
- Incubate for 60 min at 60 C, invert every 15 minutes
- Extract with phenol chloroform/isoamyl alcohol (25:24:1)
- Spin 5 min 10,000 RPM
- Transfer supernatant to new tube
- Ethanol precipitate
- Resuspend in 1/10 volume of 3M sodium acetate and 2 volume ice cold ethanol.
- Mix gently and resuspend in 50 micL TE buffer
June 23 Michael Linzey
- The bacteria that I grew up over the weekend did not work
o One plate dried out
o The other plate put out a lot of satellite colonies which is common with amp resistance plates
- I regrew E Coli with the T7 spacer plasmid
o Put it into an amp plate over night
6/24/2014 - Garrett Jensen
- I started another liquid culture of S.thermophilus LMD-9. It is in the 42C water bath in Dr. Groses lab. This one is 10mL instead of 5 (I am following the directions from the paper I talked about above for 6/23/2014)
- Using one of our previously frozen samples of LMD-9 I followed the protocol of the paper listed above and I think I have some DNA.  The extraction looked different than the others I have done with Skip before. The top layer in the tube after mixing in the phenol/chloroform was the murky layer, usually it has been the bottom. To make sure I didn’t miss the DNA I did the full extraction on  both layers but the bottom layer produced a brown pellet while the top produced a white.
- Today I started a redtaq pcr of the DNA I isolated yeterday. There are 2 groups, one with our CRISPR primers and another with Skips 16S ribosomal primers.
- The tubes labelled 1&2 are the crispr primers.  3&4 are Skips primers.
- Tomorrow I will run these out on a gel to see if they worked on this batch of DNA or not.
- If the PCR didn’t work I will need to trouble shoot with Dr. Grose and Desi. If the problem is the DNA then the liquid culture will be ready tomorrow and I can extract the DNA from LMD-9 and hopefully get a higher yield.
June 24
- I chose 7 colonies that grew that had the rfp gene knocked out, and one control colony
- I set a colony check pcr to see if we had our t7 spacer in the plasmid
o Used redtaq
6/25/2014 - Garrett Jensen
- I isolated the genomic DNA of my LMD-9 liquid culture today using the protocol listed above. No DNA was visible in the isolation however.
- I ran the PCR from 6/24 out on a gel, nothing appeared again.
June 25 Michael Linzey
- Final presentation for spring term
- Then I ran the PCR overnight
June 26 Michael Linzey
- Ran the pcr product on the gel
- nothing was on the gel
- I have been told that other groups that used redtaq have had some problems
- Try one more time with a different polymerase before I restart the whole process
6/27/2014 - Garrett Jensen.
- Today I talked with Skip about the problems I have been having. We are going to try once more to isolate DNA from S. thermophilus. This time we are going to incubate it in a shaker. I have been growing it in a 42C water bath but there hasn’t been any aeration or shaking. He has an incubator that we put it in and we will let it grow over the weekend and on Monday will try to isolate its DNA again.
June 27 Michael Linzey
- Got 8 more colonies
o Boiled them to get the DNA out
o I couldn’t find the plasmid primers anywhere, so I froze the pcr tubes with the boiled DNA and will run the pcr on Monday.
Retrieved from " http://2014.igem.org/Team:BYU_Provo/Notebook/CRISPR/mayjune "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
