<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BYU_Provo_Notebook_Auxotrophy_febapr"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BYU_Provo/Notebook/Auxotrophy/febapr">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BYU_Provo/Notebook/Auxotrophy/febapr&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Auxotrophy/febapr&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Auxotrophy/febapr&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BYU_Provo/Notebook/Auxotrophy/febapr" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BYU Provo/Notebook/Auxotrophy/febapr</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1 style="color:#FFFFFF">BYU 2014 Notebook </H1><P style="color:#FFFFFF"><A href="https://2014.igem.org/wiki/index.php?title=Team:BYU_Provo/Notebook/Auxotrophy/febapr&amp;action=edit" style="color:#FFFFFF">Edit February April</A></P><TABLE width="100%"><TBODY><TR heigth="30px"><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo" style="color:#002255">Home </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Team" style="color:#002255"> Team </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=BYU_Provo" style="color:#002255"> Official Team Profile </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Project" style="color:#002255"> Project</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Parts" style="color:#002255"> Parts</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Modeling" style="color:#002255"> Modeling</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Notebook" style="color:#002255"> Notebook</A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Safety" style=" color:#002255"> Safety </A></TD><TD style="border:1px solid black; border-radius: 5px;" align="center" height="45px" onmouseover="this.bgColor='#f9f1e3'" onmouseout="this.bgColor='#c5af7d'" bgcolor="#c5af7d"><A href="https://2014.igem.org/Team:BYU_Provo/Attributions" style="color:#002255"> Attributions </A></TD></TR></TBODY></TABLE><BLOCKQUOTE><H2 style="003468">Week of March 16th</H2><H3>March 10, 2014</H3><P>--CB MM TR--Our group is assigned the kill switch for the Nitrosospira multiformis organism we’ll be creating. We want to make sure it can’t get out into nature because that’s irresponsible, or whatever. Didn’t want to reinvent the wheel so I looked on the iGEM registry for kill switches and found one using a holin, one using a lysozyme (these two from T4 bacteriophage) and one using three genes (S, R,and Rz), this one from a lambda virus. They’re labeled and listed in the Kill Switch powerpoint presentation in the iGEM folder. The ones on the registry will have to be attached to a special promoter that is useful for our purposes. We’ll want one that turns on if the organism escapes the bioreactor, so a promoter turning on with low metal cation or ammonia concentration will do nicely. (I actually found one that works on ammonia concentration, so we’ll try that first.) Will either have a simple one promoter one kill switch method or, if things are easier by random chance of the parts we have available, a promoter that turns on in the bioreactor, making a repressor for the kill promoter, turning off when it leaves the bioreactor. 

</P><H3>March 12, 2014</H3><P>--CB MM TR-- Researched possible kill switch options for <I>N. multiformis</I>. Focused on the environment of the bio-reactor to control spread of <I>N. multiformis</I>.</P><P>Paper about riboregulators creating a fast, powerful, programmable kill switch for microbes. Seems unnecessarily intricate for what we’re doing though. 
http://www.pnas.org/content/107/36/15898.short </P><P>Originally we considered designing a trigger that is specific to the environment of the bioreactor that, when removed from the environment would express a gene that would kill the bacteria. However we realized that such a design would be problematic due to the high mutation rate of <I>N. multiformis</I>. </P><H2>Week of March 23rd</H2><H3>March 17, 2014</H3><P>--CB MM TR-- Researched articles on the environment of waste water treatment plants and the possible genes we could knock out to prepare for our presentation on <I>N.multiformis</I> metabolism optimization.</P><P>http://www.jstor.org/stable/25035171</P><P>http://www.jstor.org/stable/25034119</P><P>http://aslo.org/lo/toc/vol_17/issue_5/0749.pdf</P><P>The first paper makes leucine out to be present in both raw sewage and sewage effluent. The second paper confirms this.</P><P>Focused our efforts on a unique gene knock out that would make <I>N. multiformis</I> reliant on the environment of the bio-reactor to grow.</P><P>The 2 gene kill switch is a terrible idea. If any mutations are made in the kill switch, the whole thing will fail. If any mutations are made in the repressor gene, the thing will die, meaning that evolution will keep this thing having the repressor switch I guess. In the one gene kill switch model, if any mutations are made, it will also fail. So yeah, a better plan is needed. 
That plan is….auxotrophy! We’ll have to knock a whole gene out, which will be very hard for it to replace. This gene will be something that makes an amino acid, likely. This amino acid will then be derived from the bioreactor during its normal life span, so that if it ever leaves the bioreactor, it will starve to death. Dr. Grose said that glutamine would be a good AA to try, so we’ll have to dig some and talk to the metabolic team to see what kind of glutamine needs/synthesis capabilities this thing has. 
</P><P>Reading over a few articles now to see what the environment inside the bioreactor is like:</P><P>Asparagine has two pathways to create it. Asn supposedly has function in metabolism and storage of Nitrogen. That being said, we don’t want to knock it out.</P><P>http://aem.asm.org/content/74/11/3559.full#ref-5</P><P>Here is the whole genome of <I>N multiformis</I>:</P><P>Norton JM, Klotz MG, Stein LY, Arp DJ, Bottomley PJ, Chain PS, Hauser LJ, Land ML, Larimer FW, Shin MW, Starkenburg SR. 2008. Complete genome sequence of Nitrosospira multiformis, an ammonia-oxidizing bacterium from the soil environment. Appl. Environ. Microbiol. 74:3559–3572.</P><H3>March 18, 2014</H3><P>--CB MM TR-- Designing the auxotrophy feature: Lysine, aspartic acid, serine
</P><P>Lysine:</P><P>Has three pathways for lysine: Biosynthesis 1, 3, 5.</P><P>http://metacyc.org/META/NEW-IMAGE?type=ORGANISM&amp;object=TAX-323848</P><P>Don’t bother with lysine. Too many biosynthetic pathways, can’t find one gene that would knock out all three pathways definitively.  

</P><P>Aspartate:</P><P>Has only one biosynthetic pathway, which is good.</P><P>This is the gene we want to knock out: http://metacyc.org/META/NEW-IMAGE?type=GENE-IN-PWY&amp;object=EG10096&amp;detail-level=2</P><P>Unfortunately it seems this gene, aspC, is also used in making phenylalanine, tyrosine, and kynurenate. Those are not very present in the bioreactor, so aspC would be a bad gene to knock out.</P><P>Also, it seems there is a metabolic mechanism for changing arginine into aspartate, so if there is arginine in the effluent it would likely be able to survive: not a good auxotroph. (Unfortunately, studies show arginine to be detectably present in sewage effluent, don’t use arpartate).</P><P>Serine:</P><P>Christian has had a lot more luck with serine. He has found there is one enzyme made from serA that is definitely needed to make serine, it doesn’t seem to be important to any other metabolic pathway. Should look up the gene sequence (probably for E. coli) and blast it against the N. multiformis genome so we can find the location in the genome and the exact nucleotide sequence. Christian had a crash course in bacterial gene knockout from Desi, should be in the slides for this week. 
</P><H2>Week of March 27th, 2014</H2><H3>March 24, 2014</H3><P>--CB MM TR-- What other investigators have used as an antibiotic selector for multiformis knock ins:</P><P>http://dnasu.org/DNASU/GetCloneDetail.do?cloneid=338876 </P><P>http://dnasu.org/DNASU/GetCloneDetail.do?cloneid=585194</P><P>http://dnasu.org/DNASU/GetCloneDetail.do?cloneid=597046</P><P>According to these protocols, we’re going to go ahead and use ampicillin for our selector before we take it back out with the FLPase. 
</P><H3>March 27, 2014</H3><P>--CB MM TR-- Dr. Grose gave us some great protocols to follow as we’re thinking about the knockout and the FRT’s for the
FLPase. FRT protocol, follow this when you are designing the primer.</P><P>http://rothlab.ucdavis.edu/protocols/frt-rules.html</P><P>Gene Knockout Protocol</P><P>http://rothlab.ucdavis.edu/protocols/Lin.Transform.html#uni</P><H2>Week of April 5th</H2><H3>March 31, 2014</H3><P>--CB MM TR--Drafted an email to send to NEB in order to obtain the HindIII RM genes to put into N. multiformis for prevention of lateral gene transfer.</P><P>http://blast.ncbi.nlm.nih.gov/Blast.cgi#alnHdr_490282362 : We did a protein BLAST of the gene immediately downstream from the Nmul_A0428(serA) gene and found that is most likely involved in UDP-glucose epimerase production.  We could find evidence either way for how vital it is for life of the cell, so we don’t want to include our own promoter on the gene we insert using homologous recombination in case that messes up the cell and causes loss of viability.</P><P>Homologous Recombination primers:</P><P>Forward - ATGAACAAACTTGCGATTTCCACCTCGTCATTCGATGTCAGCCATAATAG + 20-25bp Promoter,SD,amp gene sequence</P><P>Reverse - 20-25bp amp gene complement + TTAACCTTTTTCCATTACACCGGCTTCGATCAATCCGCGCACCAGATTTT
</P><H3>April 1, 2014</H3><P>--CB MM TR--Dr. Grose received an email back from the NEB representative for our area which said we could get the requested plasmid if we filled out a Materials Transfer Agreement.  We decided that since we wouldn’t be able to put our work in the iGEM registry, and we should have all of our genes in the chromosome, that we are just going to forget about the lateral gene transfer prevention.</P><H3>April 2, 2014</H3><P>--CB MM TR--Because we are not going to be able to include our own promoter, we have decided against switching the serA gene with a gene we would need to put in anyway.  Instead we are going to use an FRT protocol to just knock out the gene and then use a flippase get rid of the in between nucleotides.</P><P>http://www.ncbi.nlm.nih.gov/nuccore/NC_007614.1?report=fasta&amp;from=474841&amp;to=476072&amp;strand=true : This is the Nmul_A0428 gene with 50bp up and downstream of the start and stop codons.
</P><H3>April 4, 2014</H3><P>--CB MM TR--http://rothlab.ucdavis.edu/protocols/frt-rules.html : Looked over the FRT primer design from Rothlabs at UC Davis</P><H2>Week of April 12th, 2014</H2><H3> April 7, 2014</H3><P>--CB MM TR--Worked on our circuit design for making N multiformis auxotrophic for serine.  Finalized the primer designs for our recombinase.  After we have inserted the amp resistance in place of the serA gene into the chromosome, we can add a flippase to the cells, which will cut out the amp gene.</P><H3>April 8, 2014</H3><P>--CB MM TR--Finalized our circuit design for Wednesday.  Finished finding and inserting protocols for each of the steps we are going through.</P><H3> April 9, 2014</H3><P>--CB MM TR--Transformed two promoters (J23115 and J23116) into DH5α E. coli by heat shock transformation.  Took 3ul purified plasmid into 25ul DH5α competent cells.  Kept on ice for 5 min, heat shock at 42˚C for 1 min, ice for 3 min.  Added 500ul LB, incubate by aeration at 37˚C for 45 min.  Plate all 500ul and incubate overnight at 37˚C.  We are testing 20 different promoters from the iGEM Registry to see the activity level of each in order to decide what promoters will be bet to use for the different parts of our project.</P><H3> April 11, 2014</H3><P>--CB MM TR--Performed a plasmid prep of the promoter plasmids we transformed into E coli on Wednesday.  Broke into groups of 2 and each group did 3 samples.</P><H2>Week of April 19th</H2><H3> April 13, 2014</H3><P>--CB MM TR-- As it happens, our entire auxotrophy approach must be completely reformed. The Methods in Enzymology book says that N. multiformis CANNOT be electroporated, neither is it chemically competent, leaving Conjugation as the only viable transformation method. From E. coli to multiformis, we will pass a plasmid, meaning we do have to put our homologous recombination pieces into a plasmid. This plasmid needs to be a suicidal plasmid so that we can get rid of it after the recombination event. This being the case, we now must design primers to create 500bp regions of homology on either side of the serA gene and SOE them together onto a plasmid to be transferred to multiformis and later removed via homologous recombination. Skip helped us to design the plasmid which we would need, since Dr. Grose is ordering plasmids tomorrow. They are as follows:</P><P>SUICIDE PLASMID for the Conjugation of E. coli plasmids into N. multiformis</P><P>Front and Back halves of our primer (SOEing)</P><P>CCTCGTCATTCGATGTCAGC ATCGAAGCCGGTGTAATGGA</P><P>Reverse complement of above (SOEing)</P><P>TCCATTACACCGGCTTCGAT GCTGACATCGAATGACGAGG</P><P>Primers from front end of the front 500 homology block</P><P>[overhang compatibility] (bamHI) [GAT](GGATCC)ggcgtaatggtacccatgatcaaaac</P><P>Primer from back end of the back 500 homology block, reverse complement. </P><P>[overhang compatibility] (speI) [ATC](ACTAGT)catagagcgagccatagcgcaaaat </P><H3> April 15, 2014</H3><P>--CB MM TR-- We got the primers created on Monday ordered after verifying them with Skip and Dr. Grose again.  Skip is going to give us the pSR47s plasmid to use as our vector for conjugation.</P><H2>Week of April 26th</H2><H3>April 21, 2014</H3><P>--CB MM TR--Our team prepared a semester final on our <I>N.multiformis</I> Serine Auxotrophy Process.</P></BLOCKQUOTE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BYU_Provo/Notebook/Auxotrophy/febapr">http://2014.igem.org/Team:BYU_Provo/Notebook/Auxotrophy/febapr</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BYU_Provo/Notebook/Auxotrophy/febapr" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BYU_Provo/Notebook/Auxotrophy/febapr" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Auxotrophy/febapr&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Auxotrophy/febapr&amp;oldid=351179" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>