"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/Metabolism/julyaug
From 2014.igem.org
BYU 2014 Notebook
Edit July August
Home
Team
Official Team Profile
Project
Parts
Modeling
Notebook
Safety
Attributions
Week of July 5th
June  30, 2014
--BRK--I chose one colony of E.coli from four separate LB and CAM plates containing transformed bacteria with EreB and a promoter of either 101, 108, 111, or 119. These four colonies were streaked on four separate LB, CAM, and Erythromycin plates and grown overnight at 37C.
--CS-- We got our new reverse primers in today so I first checked all of the sequences to make sure they were correct and then ran Q5 PCR for all 4 genes. Julie then ran a gel of the PCR products. It looked like this:
The lanes are, from left to right, nirS, the DNA ladder, norB, norC, and nosZ. The most prevalent bands in each lane appear to be in the appropriate places based on length so it looks good to continue.
July 1, 2014
--BRK--The streaked colonies grew well on the LB/CAM/Erythromycin plates, indicating that the plasmid with the EreB gene had been successfully transformed. I took pictures of all the plates to compare the growth and use in the future to determine the effect of the different promoters.
--CS-- Today I did PCR cleanup, the restriction digest (XbaI and SpeI), the low-melt gel, the ligation, and the transformation of all 4 of the denitrification genes since we got the new reverse primers yesterday. My low-melt gel looked like this:
The lanes are, from left to right, the DNA ladder, nirS, norB, norC, nosZ and pIG91. I put my plates in the 37°C incubator overnight.
July 2, 2014
--CS-- Today we went to the wastewater reclamation facility in Park City. I learned a lot about the entire treatment process; I almost wish that we had gone to see the facility even earlier on to get a better idea of what actually happens. The visit gave me some more hope about the feasibility and importance of adding the denitrification pathway to N. multiformis. At first the worker who gave us the tour made it seem that the majority of the ammonia was converted to nitrogen during the activated sludge carousel stage, but the algal bloom observed in the final effluent stage and the high proportion of nitrite/nitrate released in the effluent show that there certainly is a need for improvement in nitrate removal. The readings they took in March (they provided us a copy of their report sheet) showed that of the 34 mg/l ammonia that enter the facility, about .5 mg/l ammonia and 25 mg/l nitrite/nitrate leave the facility. This shows that although the ammonia is removed very effectively (99%) there is still quite a bit of room for improvement in removing the nitrate from the effluent to reduce the occurrence of eutrophication. We got some pictures of the water leaving the facility where algae was growing all over despite the rather low flow of water. Another thing that I learned was that the conditions of the activated sludge carousel might actually be in favor of forcing N. multiformis to do both nitrification and denitrification since it might not be too favorable from a metabolic standpoint. As the waste water enters the carousel it is aerated to activate it. As it moves around the carousel though the oxygen supply is depleted, resulting in a portion of the carousel in which the conditions are anoxic. Denitrification occurs during this stage. Here is a diagram:
This change from activated to anoxic conditions works well with our project since, as Dr. Breakwell discussed when I met with him last week, this will help convince our engineered N. multiformis to undergo both nitrification (aerobic) and denitrification (anaerobic) since the different pathways are favorable under different oxygen conditions.
July 3, 2014
--BRK--Our team went to the wastewater treatment plant to learn more about the entire treatment process. I collected the biofilm and effluent water run-off from the plant to bring back to the lab for testing.
Week of July 12th
July 7, 2014
--BRK---Searched for a process to detect antibiotics in water. A few processes were discussed in a powerpoint written by Dr. Rosenfeldt from the University of Massachusetts. http://k12s.phast.umass.edu/sess/emerging/
Week of July 19th
July 14, 2014
--CS--/--BRK--Today we moved the lab equipment and supplies over to the new building.
July 17, 2014
--CS-- Today I did colony PCR for all 4 of the genes using the pSB1C3 forward and reverse primers (307 and 308). And the PCR worked great! The only problem is that all of the bands showed up under 500 bp, meaning none of the colonies had their genes in them. So it appears that we need to go all the way back to the start again. I didn't bother saving an image because it was very clear that all of the bands were too short.
--BRK--After finding a faulty control plate from last week, I regrew all of my colonies on new plates overnight.
July 19, 2014
--CS-- Today I worked on the team wiki.
--BRK-- Overnight plates failed to restrict the growth of bacteria on the erythromycin agar plates. I added 3 EreB transformed colonies from a previous plate to 3 separate tubes containing 5ml of LB/CAM/EREB media to grow overnight. In addition, I also grew overnight 3 separate colonies transformed with an empty backbone into 3 tubes with LB/CAM to use as controls. All transformations were done according to the common procedures.
Week of July 26th
July 21, 2014
--CS-- Today I worked a little bit on the team wiki. I also helped Julie get things set up for the PCR since we are going to have her run through the cloning process with all the genes one time to see if she can get things to work. And then there was a chemical spill alarm in the building. After that I finished up the preliminary safety sheet for our team.
--JR--Set up a Q5 PCR for 5 denitrification genes: NirS, NorB, NorC, NosZ. Set up according to Q5 PCR protocol with the following primers for each gene respectively NirS, BI333, BI334; NorB BI335, BI336; NorC BI337, BI338; NosZ BI339, BI340; Negative control with primers BI339, and BI340, but with no template.
--BRK-- Due to the failure of the previous plates, I doubled the erythromycin solution in the plate agar. Once the plates were set, I pipetted a sequence of 3 plates with 20ul, 30ul, and 50ul of liquid overnight bacteria media. In addition, I pipetted the same amount of control transformed bacteria into LB/CAM plates.
July 22, 2014
--BRK--Colonies grew on all plates, both EreB transformed and control, indicating that the plates did not contain enough erythromycin to provide good data and results.
July 23, 2014
--CS-- Today I worked some more on the team wiki.
--JR--Did PCR cleanup according to the GE cleanup kit protocol for the 5 denitrification genes. Then set up restriction digest of the genes according to restriction digest protocol . Used all 50 ul from the PCR reactions, also used 50ul for the plasmid digest as well. Decreased the water amount in reaction to compensate for added DNA content. Used XbaI,and SpeHF with NEB buffer 4 and BSA.
July 24, 2014
--CS-- Today I worked some more on the team wiki.
Week of August 2nd
July 28, 2014
--JR--Treated our plasmid restriction digest with CIP to prevent plasmid from religating to itself. Ran restriction digests out on low melt gel. Bands all visible, however NosZ was very faint. This band was right next to the plasmid digest, and band was in the same position, which aided in cutting out the band from the low melt.
--BRK--To avoid pouring new plates because we had an immense supply of LB/CAM, I added erythromycin solution to the top of the plate agar. I increased the amount of solution added by 50ul per plate to insure a high concentration of antibiotic. I let the plates sit for an hour before pipetting 50ul of transformed bacteria to each and plating with glass beads. There were 8 plates in total, 4 DH5α with EreB plasmids and 4 control DH5α. The plates were then incubated at 37C overnight.
July 29, 2014
--CS-- Today I worked some more on the team wiki.
--JR--Melted restriction digests and set up ligation reaction according to ligation protocol. Used 5ul of vector for the NosZ gene since band was very faint, and reduced water content to 4.5ul. All others according to protocol with respective genes and our digested pIG91. Ligation was allowed to run 2+hrs. Transformation was then set up according to transformation protocol.
--BRK-- There was growth again on both sets of resistant and control plates. The solution of erythromycin seems to be inadequately made. I have talked with Skip and asked his advice on re-creating a solution for future tests. We decided to make a 1000X stock solution using the formula of 20ug of erythromycin to 1mL of 96% ethanol according to the protocol published by Bio-ITech ELabProtocols
July 24, 2014
--CS-- Today I selected colonies from the plates that Julie made for the colony PCR. I added each colony to 50 μl water and boiled that; I also streaked out these colonies on new LB+Cam plates. I then started looking into the genomes of Nitrsomonas eutropha and Nitrosomonas europaea to see if they already have denitrification genes in them since we might be switching over to one of them as our chassis. I did some BLASTn comparisons and none of the 4 denitrification genes from P. aeruginosa PAO1 came up with any hits to either genome. I did BLASTx too though and although nits and nosZ did not have any hits with either of the new bacteria, both norB and norC matched the genomes. The norB gene had E values of 8e-128 and 3e-126 for N. europaea and N. eutropha, and the norC gene had E values of 3e-51 and 2e-50 for N. europaea and N. eutropha. It appears from these results that the two new bacteria that we might switch over to actually have some of these genes. I found a paper in PubMed about N. eutropha but I did not have enough time to look through it thoroughly. I will go back and try to find more papers to figure out whether or not these bacteria already do denitrification.
Whole-genome analysis of the ammonia-oxidizing bacterium, Nitrosomonas eutropha C91: implications for niche adaptation.
Week of August 2nd
July 30, 2014
--JR-- Set up colony PCR reactions. Multiplied and did master mix 45X's the single Taq protocol . Used primers BI307 and BI308 for the plasmid forward and reverse. We chose to use these primers just to ensure that we would get PCR product, as it has often not worked at this stage before. Then those that we get product on we plan to do further checks to confirm proper directionality of our genes.
Week of August 9th
August 4, 2014
--JR--Set up and ran a gel on our colony PCR products. NorC looks great! NirS worked, NorB appears to have worked. NosZ may have worked, but is in question. We will be setting up a few more colony PCRs to check the working ones for directionality. Then we plan to do some sequencing of these plasmids.
--CS-- Today Julie and I ran the gels from the colony PCR that she did last week. And we got some great results! The images are below:
The image on the left is the gel of nirS, norB, and norC in that order with a DNA ladder, nirS 1-8, and norB 1-5 on top and a DNA ladder, norB 6-8 and norC 1-8 on bottom. The image on the right is the gel of nosZ 1-16 except 9 with DNA ladders intermixed. There were great bands at the appropriate places for all of the bands except for nosZ, which had one band show up kinda close to where it should have. We talked to Dr. Grose about our results and she said that we should first do a directionality PCR (vector primer on one end and gene primer on the opposite to test that they are inserted in the proper direction since we used the compatible XbaI and SpeI sites for inserting the gene into the vector) and then proceed in sequencing vectors that have the gene inserted in them in the proper direction. So we made some great progress!
I also continued reading the article I referenced last time. I still need to look at it more thoroughly though to figure out exactly what the new bacteria do in terms of denitrification.
--BRK--Grew overnights of transformed DH5α and control DH5α in LB/CAM to prepare for testing, and then worked on the wiki.
August 6, 2014
--JR--Set up some more colony PCR to check directionality of colonies that produced bands. PCR this time was done with the pSB1C3 MCS Forward primer (BI307) and was done with each genes respective primer. Chose colonies 4,5,7,8 of NirS; 1,2,3,5,8 of NorB, and 1,2,3,7 of NorC. Did according to Taq protocol.
--BRK--Set up a serial dilution of overnight media of transformed DH5α and control to compare the effectiveness of the plasmid gene against the concentration of antibiotic in solution. In 2 microcentrifuge tubes I added .5mL of LB/CAM media, and subsequently added .5uL of erythromycin solution. In 10 microcentrifuge tubes, I added 800uL of LB/CAM to do a 1:5 serial dilution for each overnight media. To the 2 initial tubes, I added .5mL of overnight bacteria solution, inverted the tube a few times, and subsequently added 200uL of media into the next tube and carried out the dilution series. All tubes were placed in an incubator overnight at 37C.
August 7, 2014
--JR--Ran a gel for the PCR's set up yesterday. Looks like we have NirS, NorB, and NorC in the plasmid AND in the right direction! Denitrifying here we come! Also, set up overnights for the colonies which were indicated by the PCR to contain the insert.
August 8, 2014
--JR--Purified plasmids from positive colonies.
Week of August 16th
August 11, 2014
--JR--Nanodropped plasmid preps today. They had extremely low concentrations, which I suspect is a result of me not letting the elution reaction incubate long enough before centrifuging. However, this is a good thing because I overnighted the wrong colonies anyways. Thus I restarted and made sure to set up overnights of the correct colonies.
--CS-- Today I got caught up with Julie on the progress of the denitrification gene cloning process. She started overnights today and I will take over with plasmid preps tomorrow before submitting them to Desi for sequencing.
--BRK--The effect of the antibiotic added to the overnight dilution series test completely cleared all of the tubes of all bacteria and cell death was highly evident by a small layer of dead cells in the bottom of the tube. There are two conclusions that I have drawn from this failure: that either the plasmids failed to digest and ligate the EreB gene but remained resistant to CAM due to the backbone resistance gene, or that the concentration of erythromycin was too high. I have decided to go back to the original transformation of the EreB plasmid into DH5α and try ligating a promoter back into the plasmid and start over.
August 12, 2014
--CS-- Today I did plasmid preps of the the overnights that Julie started yesterday using the kit in the lab.
August 13, 2014
--CS-- Today Julie and I checked the concentrations of the plasmid preps with the Nano-Drop and they all appeared pretty good with concentrations around 100-200 ng/μl and 260/280 ratios around 2. We then prepared these for Desi to sequence by adding 2 μl of the vector to 1 μl of the primer in PCR tubes; we did one reaction with the forward primer and one with the reverse primer for each gene. We originally use each of the gene forward and reverse primers but then I remembered that we should use the vector forward and reverse primers (307/308) since those seem to work better than our gene primers. Desi will sequence those this week and get us our results.
I also decided that it would be a good idea to make a freezer stock of P. aeruginosa PAO1 just in case anything happens to the plates that we have made right now, so today I picked a colony and stuck it in 5 ml LB before placing that in the 37°C incubator shaker overnight. I also streaked out a new plate to store in the fridge since the other one is coming up on a month old.
--BRK-- I started adding protocols to the wiki page Common Procedures . I also selected a colony from an old plate that had the original EreB transformed DH5α and restreaked it onto a fresh LB/CAM plate and placed it in the incubator at 37C overnight.
August 14, 2014
--CS-- Today I pulled the new P. aeruginosa PAO1 streak plate out of the incubator and stored it in the fridge. I also made freezer stock of the P. aeruginosa PAO1 overnight liquid culture that I made by putting 1 ml of the liquid culture and 200 μl of DMSO in cryotubes.
Week of August 23rd
August 18, 2014
--CS-- Today I reviewed the sequencing results of our 4 denitrification genes. They looked great! I used Geneious to align the sequences to the published sequence to compare them. It appears that nirS 3 and norC 3 and 7 were perfect and that norB 5 and
2 should be good to move forward with even though their sequencing results weren't quite perfect.
--BRK--I picked three colonies from my last overnight plate and added each to a test tube with 3mL of LB/CAM to grow overnight at 37C. In addition, I selected a colony from an old plate that contained DH5α with the constitutive promoter BBa_J23111 and also grew it overnight in 3mL of LB/CAM.
August 19, 2014
--CS-- Today Julie and I worked with Dr. Grose to set up and run the mutagenesis PCR reactions for norB (2 sites) and nosZ (1 site) using a kit that she has in her lab.
--BRK--Finished up the Common Procedures . Spun down my overnight media and did a plasmid prep using Qiagen QIAprep Spin Miniprep Kit and stored them in the freezer.
August 20, 2014
--CS-- Today I did finished up the mutagenesis stuff by doing a dpn1 digest, adding 1 μl dpn1 to the PCR products, mixing, and incubating for over an hour at 37°C; transforming 3 μl into DH5α, letting recover in .5 ml LB for 1.5 hours, and plating onto LB-Cam plates.
I also began the process of assembling the promoters with the genes to create a plasmid that contains all of our genes. Dr. Grose advised us to put promoters in front of every individual gene instead of just one in front of all of the genes (what I originally planned). I started today by first performing a restriction digest of the nirS 3 and norC 3 purified plasmids. I then purified the digests using a low-melt gel . Julie had some leftover J23101 promoter that had already been digested with EcoRI and SpeI, so I was able to just take some of that and ligate it into the melted low-melt slices of my digested gene vectors. After incubating this reaction I then transformed some of it into DH5α before letting it recover and plating it on LB+Cam plates.
August 21, 2014
--CS-- Today I checked my mutagenesis and restriction digest reactions from yesterday. Since I had transformed both into DH5α and grown overnight, I first picked colonies for the mutagenesis and the RD and streaked them onto new LB+Cam plates while also boiling their DNA in 50 μl of water. I then set up Taq PCR to check everything, setting up the following reactions:
norB Forward Mutagenesis with vector forward and vector reverse primers
norB Forward Mutagenesis with gene forward and gene reverse primers
norB Reverse Mutagenesis with vector forward and vector reverse primers
norB Reverse Mutagenesis with gene forward and gene reverse primers
nosZ Mutagenesis with vector forward and vector reverse primers
nosZ Mutagenesis with gene forward and gene reverse primers
nirS 3 Restriction Digest with vector forward and vector reverse primers
nirS 3 Restriction Digest with vector forward and gene reverse primers
norC 3 Restriction Digest with vector forward and vector reverse primers
norC 3 Restriction Digest with vector forward and gene reverse primers
I did so many different reactions to make sure that I covered all of my bases in one swing. Doing PCR with the vector primers will give me the best chance of knowing if stuff worked and something is in the plasmids and doing PCR with the gene primers will help me confirm that it is my gene that is in there. I ran these reactions in the PCR machine. I then used analytical gels to determine the results of my PCR reactions. Here are images of my gels, with the reaction type and the PCR primers specified (V for vector, G for gene, F for forward, and R for reverse):
So it appears that something didn't work along the way for pretty much all of the genes. When I plated out norB and nosZ on a LB+Cam plate, norB had a good amount of colonies, suggesting that the DpnI digest might not have worked properly, while nosZ only had a few colonies, suggesting that everything was probably alright for that gene. I think the problem with my nirS and norC though had to do with the plates. I used some older plates that I wasn't sure of, and only tiny colonies grew up that didn't seem quite reliable. I saved most of my recovered DH5α for all of my colonies though.
August 22, 2014
--CS-- Since the majority of the PCR reactions didn't seem to work (either because of the PCR itself or because of the mutagenesis or transforming or whatever), I was only able to submit PCR product from a few of the norB mutagenesis colonies for sequencing. I picked colonies 1 and 4 from the forward mutagenesis reaction and colonies 5 and 6 from the reverse mutagenesis reaction. I added 2 μl of the PCR products from these to 1 μl of each the forward and reverse vector primers (separately) and submitted those to Desi for sequencing. I also plated the leftover transformed DH5α that I had for all of my genes on good LB+Cam plates, which I grew up overnight in the 37°C incubator.
August 23, 2014
--CS-- My plates from yesterday appeared to grow well, so I selected colonies from the nosZ 2, nirS 3, and norC 3 plates and set up colony PCR using the vector forward and reverse primers.
Week of August 30th
August 25, 2014
--CS-- Today we spent a lot of time working on the team abstract. I then reviewed my sequencing results for the norB mutagenesis reactions. None of the 4 colonies that I picked appeared to work, so we will sequence more colonies.
--BRK--Am on vacation and working on the wiki this week.
August 26, 2014
--CS-- Today Dr. Grose and I tried the DpnI digest again for all 3 of the mutagenesis reactions and then I transformed 3 μl of each of those into DH5α and plated on LB+Cam plates overnight. I also ran an analytical gel of my PCR products from the nosZ 2, nirS 3, and norC 3 colony PCR. My gels were pretty poor quality (the first one actually even melted). Here are images:
These images show that I was not very successful in either the mutagenesis or the restriction digest and ligation for any of these. Since the nirS sequence is about 1700 bp long, the band for colony 5 looks pretty good, and since the norC sequence is about 450 bp long, the band for colony 4 looks like it could possibly be correct. I will submit both of these for sequencing to see if we got our promoters inserted or not!
August 27, 2014
--CS-- Today I did colony PCR for the norB mutants; 1 norB plate had about 15 colonies, the other had 1, and the nosZ plate had none, so I picked 7 colonies from the first norB plate and used those with the 1 from the other. I ran an analytical gel of my PCR products. Here is the image:
Based on this gel only one of my colonies had the norB gene in it, which seems a little funny since they were growing up on the LB+Cam plates. Desi suggested that I try doing plasmid preps of some of these colonies, doing PCR of that, and then sequencing that product. I also tried transforming the DpnI digested nosZ mutated plasmid again and plated it on LB+Cam plates and put it in the 37°C incubator overnight.
August 28, 2014
--CS-- Today I started up overnights for norB 1-1, 2-4, 2-6, and 2-7 for plasmid preps. I also submitted a bunch of PCR products to Desi for sequencing. My nosZ plate also failed to grow up again, so I will have to transform that again sometime.
August 29, 2014
--CS-- Today I did the plasmid preps for the overnights that I had grown up. The DNA concentrations of these were actually really low, so hopefully the PCR will still work and we can get some good stuff for sequencing. I also cleaned out all of the gel boxes so that the 442 lab can take them.
Retrieved from " http://2014.igem.org/Team:BYU_Provo/Notebook/Metabolism/julyaug "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
