<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HAFS_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HAFS/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>﻿
</P><H1 class="logo">iGEM HAFS</H1><NAV class="menu"><UL><LI><A href="#">PROJECT ▾</A></LI><LI><A href="https://2018.igem.org/Team:HAFS/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:HAFS/Applied_Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:HAFS/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:HAFS/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:HAFS/Improve">Parts</A></LI></UL><LI><A href="#">HUMAN PRACTICES ▾</A></LI><LI><A href="https://2018.igem.org/Team:HAFS/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HAFS/Public_Engagement">Engagement</A></LI><LI><A href="https://2018.igem.org/Team:HAFS/InterLab">INTERLAB</A></LI><LI><A href="#">TEAM ▾</A></LI><LI><A href="https://2018.igem.org/Team:HAFS/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:HAFS/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:HAFS/Collaborations">Collaborations</A></LI></NAV><P style="font-size:5em; color: #6C5B7B; text-align: center; font-family: 'Quicksand', sans-serif; font-weight: bold; padding:0; margin:0">Model</P><DIV class="sublinks"><H2><A href="#Lysis">Bile salt induced lysis</A></H2><H2><A href="#Insulin">insulin</A></H2></DIV><H2 class="title-conteudo" id="lysis">Bile salt induced lysis</H2><DIV class="conteudo"><DIV style="padding:3%"><H3>Background</H3>   In order our project design, it is necessary for the minicells to lyse in the intestine in the presence of bile salt so that the insulin can be absorbed through intestinal cell after food is taken. In doing so, it is important to make sure that the cells will not lyse either too early (before food intake) or at a location other than intestine. In order to make this possible we decided to use bile salt inducible acrA promoter to regulate the secretion of T4 endolyin which induced cell lysis, considering that bile salt is secreted to intestine, and is secreted after food intake. 
  <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1962010"> BBa_K1962010 </A>, submitted by the <A href="https://2012.igem.org/Team:Wisconsin-Madison"> 2012 Wisconsin – Madison team </A>, encodes acrA promoter, which activates downstream gene in the presence of bile salt. acrA promoter is originally found in acrAB operon from Salmonella enterica, which is involved in multidrug efflux pump. Bacteria that live in intestine should withstand the toxicity of bile salt. The efflux pump  endows bile salt resistance capability by enabling the bacteria to transport a group of similar chemical compounds (including bile salt) outside the cell. Two proposed pathways account for the mechanism of acrA promoter and acrAB operon in relation to bile salt. 
<P class="figure-caption" style="text-align:center">Figure 1. acrA pathway involving RamA. Bile salt binds to RamA to change its low activity state to high activity state. RamA, in turn, induces promoter acrA. </P>   The first pathway is derived from Salmonella enterica (Figure 1), which requires the presence of RamA protein: The binding of bile salt to protein RamA converts the protein from a low activity to high activity state, which binds to the upstream activation site of acrA and acrB promoter. According to this model, the overexpression of protein RamA, even without the activation by bile salt, can lead to induction of acrA promoter. [1]
<P class="figure-caption" style="text-align:center">Figure 2.  acrA pathway involving Rob. Bile salt works as an effector that binds to activate Rob </P>   The multidrug efflux pump regulated by acrAB operon is present in E. coli – which is the chassis used by our team – as well (Figure 2). In E. coli, however, protein RamA is not involved in the acrAB related pathway. Instead, the protein Rob binds to effector such as bile salt which leads to conformational change of the Rob and subsequent expression of acrAB operon. Unlike RamA, Rob remains in entirely inactive and is expressed constitutively. [2]  

    In fact, the work done by <A href="https://2016.igem.org/Team:Dundee/Results"> 2016 Dundee team </A> provides deeper insight into the workings of acrA promoter. As a part of their project, Team Dundee cloned gfp into pSB1C3 downstream of the acrRA promoter, and RamA into the pUniprom vector downstream of the constitutive tat promoter (Fig.3) <P class="figure-caption" style="text-align:center">Figure 3. (a) gfp downstream of acRA promoter (b) RamA downstream of constitutive promoter </P>   Then, they transformed <I>E. coli</I> MG1655 cells with the two constructs, PacRA-gfp and RamA, with different combinations with empty vectors.Figure 4 suggests that the promoter construct (when transformed with acRA-gfp and RamA) was active with and without the addition of sodium cholate (a type of bile salt), though the activity greatly increased in the presence of sodium cholate. This suggests that the low activity RamA that isn't bounded to bile salt still functions to activate the acRA promoter. Furthermore Figure 5 suggests that the acRA promoter is active even without the RamA construct, which gives evidence to the endogenous transcription factor that binds to bile salt and activates acRA, or Rob. <P class="figure-caption" style="text-align:center">Figure 4. 96 well plate reader experiment, measuring OD600nm and GFP fluorescence over 20h.  Showing the difference in GFP fluorescence per unit absorbance when pSB1C3-PacrRA-gfp is grown in the presence or absence of Sodium cholate (10µg/ml)</P><P class="figure-caption" style="text-align:center">Figure 5.  Microscopy fluorescence imaging for PacrRA-gfp with and without RamA transcription factor on MacConkey agar plates.</P><H3>Model</H3>   Based on this background knowledge, we built two pathways that mediate acRA promoter with and without RamA producing plasmid. We used a CAD tool named TinkerCell to develop the model [3]. TinkerCell allows the users to easily design and model genetic networks and other complex sets of reactions. <P class="figure-caption" style="text-align:center">Figure 6. (a) Model that employs endogenic Rob pathway (b) Model that employs both endogenic Rob pathway and RamA pathway introduced through plasmid. Both models were created by TinkerCell </P>   While building the model, we assumed the half-life of the Insulin to be 184.7 minute based on a previous research, which reported that the half-life of insulin increased to from 14.5 minutes to 184.7 minutes when associated with D-Penetratin. [4] Furthermore, the model assumes that the activity of RamA decreases by half in the absence of bile salt. This is done by creating two types of RamA, one that needs bile salt to be activated and one that does not. The production of the protein endolysin is proportional to the strength of promoter and activation site, which is in turn proportional to the concentration of corresponding transcription factor. <P class="figure-caption" style="text-align:center">Figure 7. (a) Stochastic prediction of Rob pathway when bile salt is absent (b)  Stochastic prediction of Rob pathway when bile salt is present (10uM) </P>   We used stochastic model to predict the protein production in model that involves Rob pathway. Figure 7 (a) suggests that when the bile salt is absent, great majority of Rob remains inactivated. As a result, no T4 endolysin is produced, eliminating the chance of the kill switch being activated. On the other hand, when bile salt is present, the bile salt binds to Rob and activates acRA, leading to massive production of T4 endolysin (Figure 7 (b)). This would induce the cells to lyse in practical situations. 

<P class="figure-caption" style="text-align:center">Figure 8. (a) Stochastic prediction of Rob+RamA pathway when bile salt is absent (b)  Stochastic prediction of Rob+RamA pathway when bile salt is present (10uM) </P>   Similarly, we produced stochastic prediction of the model that involves both Rob and RamA, which represents the situation in which the <I>E.coli</I> is transformed with RamA producing plasmid. Figure 8 (a) suggests that without the presence of bile salt, both RamA and Rob activated by bile salt remain in a very low concentration. However, the low activity RamA (represented by red color) is present in considerable amount, which lead to the activation of acRA and subsequent production of T4 endolysin (represented by magenta). Figure 8 (b) demonstrates that the bile salt, when present, will bind and activate both Rob and RamA, resulting in additional production of T4 endolysin.    According to our modelling results, the constitutive production of the transcription factor RamA might cause unintended lysis of cells, even before it reaches the intestine where bile salt is present. In order to ensure the lysis based secretion to happen as designed, we decided that it was necessary to transform only with the acRA-endolysin construct, not acRA. 


[1] Nikaido, E., et al. “Regulation of the AcrAB Multidrug Efflux Pump in Salmonella Enterica Serovar Typhimurium in Response to Indole and Paraquat.” Microbiology, vol. 157, no. 3, 2010, pp. 648–655., doi:10.1099/mic.0.045757-0. 
[2] Blanco, Paula, et al. “Bacterial Multidrug Efflux Pumps: Much More Than Antibiotic Resistance Determinants.” Microorganisms, vol. 4, no. 1, 2016, p. 14., doi:10.3390/microorganisms4010014. 
[3] Chandran, Deepak, et al. “TinkerCell: Modular CAD Tool for Synthetic Biology.” Journal of Biological Engineering, vol. 3, no. 1, 2009, p. 19., doi:10.1186/1754-1611-3-19.  
[4] Khafagy, El-Sayed, et al. “Region-Dependent Role of Cell-Penetrating Peptides in Insulin Absorption Across the Rat Small Intestinal Membrane.” The AAPS Journal, vol. 17, no. 6, 2015, pp. 1427–1437., doi:10.1208/s12248-015-9804-y.


<H3 style="color:#4b5ea4; text-align:center; margin-top: 5px"><A href="#top">▲ top</A></H3></DIV></DIV><H2 class="title-conteudo" id="insulin">insulin structure and interactions</H2><DIV class="conteudo"><DIV style="padding:3%"><H3>Prediction of tridimensional structure using the insulin sequences used in our project</H3>   Since, Winsulin has different linker sequence as compared to conventional insulin, we wanted to predict the 3D protein structure of Winsulin. To start with, we used Swiss-Model  (available in http://swissmodel.expasy.org) to simulate the 3D structure of Winsulin based on the amino acid sequence. The 'first approach mode' from Swiss Model allows the users to obtain a 3D structure of a protein from the amino acid sequence based on available template structures. The following is the structure model of Winsulin that we built from the sequence of Winsulin:<P class="figure-caption" style="text-align:center">Figure 1. Predicted 3D structure of Winsulin</P>   The model built had 90% similarity with the template identified by Swiss-Model, which is M2PI an active single chain mini-proinsulin with about 50% of native insulin binding activity. It contains linker of 9 amino acids  between A and B chain. [1] We compared the 3D structure of M2PI and Winsulin using MatchMaker tool from Chimera software (available at https://www.cgl.ucsf.edu/chimera/) <P class="figure-caption" style="text-align:center">Figure 2. (a) 3D structure of M2PI (b) Comparison between the structure of Winsulin and M2PI </P> 

Using the same MactchMaker tool, we compared the protein structure of Winsulin with different types of insulin:<UL><LI>Insulin is composed of two peptide chains: A chain of 21 amino acids and B chain of 30 amino acids. These chains are linked together by two disulfide bonds. [2]
</LI><LI> Insulin Glargine is one of the most commonly used long acting insulin analog. The glycine residue at position 21 of the A-chain is substituted to asparagine, which renders the protein insoluble at physiological pH. After subcutaneous injection, insulin glargine precipitates. Then, the drug is released from the precipitates slowly, giving its characteristic long duration of action (up to 24 hours) [3,4]  </LI><LI> Insulin Lispro, otherwise known as Homolog, is the most widely used short acting insulin analog. The rapid in vivo absorption is achieved by inverting propyl, lysyl sequence at the C-terminal end of the B-chain, which results in decreased association of monomers to dimers. [5]</LI></UL><P class="figure-caption" style="text-align:center">Figure 3. (a) Structure comparison between Human Insulin and Winsulin (b) Structure comparison between Insulin Glargine and Winsulin (c) Structure comparison between Insulin Lispro and Winsulin</P><H3>Simulating interactions between insulin and other drugs</H3>   In order to clarify that Winsulin does not interrupt with anti-Diabetic medications, we simulated the interaction between insulin and commonly used anti-Diabetic drugs. This modelling was also done by Chimera software. We used a tool named Autodock Vina, which is normally used to calculate the energy score of the interaction between protein and ligand. <P class="figure-caption" style="text-align:center">Figure 4. Interaction between Metformin and human insulin and Winsulin</P>   First of all, we considered the interaction between Winsulin and Metformin, a first-line agent to treat type 2 diabetes. The structure of Metformin was obtained from NIH open chemistry database.  The highest score of interaction was 2.1kcal/mol for Winsulin and 2.0kcal/mol for insulin. This result demonstrate that the interaction between Metformin and Winsulin is not only very low but also similar to conventional Insulin. Hence, Winsulin will likely not interfere with the workings of Metformin. <P class="figure-caption" style="text-align:center">Figure 5. Interaction between Sulfonylureas and human insulin and Winsulin</P>   Sulfonylureas is also a widely used antidiabetic medication for Type 2 Diabetes that increases insulin release from the beta cells in the pancreas. This time, the model suggested that the docking score between Winsulin and Sulfonylureas was 2.6 kcal/mol, which is higher than the score between Human Insulin and Sulfonylureas, 2.0kcal/mol. However, the strength of interaction seems to be negligible, unlikely to cause any complication.  




[1] Cho, Y. et al., Solution Structure of an Active Mini-Proinsulin, M2PI: Inter-chain Flexibility is Crucial for Insulin Activity. J.Biochem.Mol.Biol. (2000).
[2]  Timofeev, V. I., et al. “X-Ray Investigation of Gene-Engineered Human Insulin Crystallized from a Solution Containing Polysialic Acid.” Acta Crystallographica Section F Structural Biology and Crystallization Communications, vol. 66, no. 3, 2010, pp. 259–263., doi:10.1107/s1744309110000461
[3] Yang, Y. et al., Dynamic repair of an amyloidogenic protein: Insulin fibrillation is blocked by tethering a nascent alpha-helix. To be Published 
[4] Chatterjee S, Tringham JR, Davies MJ: Insulin glargine and its place in the treatment of Types 1 and 2 diabetes mellitus. Expert Opin Pharmacother. 2006 Jul;7(10):1357-71. 
[5] Ciszak, Ewa, et al. “Role of C-Terminal B-Chain Residues in Insulin Assembly: the Structure of Hexameric LysB28ProB29-Human Insulin.” Structure, vol. 3, no. 6, 1995, pp. 615–622., doi:10.1016/s0969-2126(01)00195-2.

<H3 style="color:#4b5ea4; text-align:center; margin-top: 5px"><A href="#top">▲ top</A></H3></DIV></DIV><P><TITLE>Team: AQA_Unesp</TITLE></P><DIV id="footer"><DIV id="contact" class="btm_box"><P>
			Hankuk Academy of Foreign Studies 
			International studies
			Gyeong gi, South Korea
			kyuhee0622@gmail.com</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>