Team:MIT/WetlabDrylab
Home
Project
The Co-Culture
C. Hutchinsonii
Biodiesel Production
Parts
Circuit
Modeling
Promoter Characterization
Human Practices
Consolidated Bioprocessing
Economics
Experts
Notebooks
Wetlab/Drylab
Measurement
Protocols
About Us
Team Bios
Acknowledgements
Attributions
Medals
Externals
Collaborations
Interlab Study
@MIT_iGEM
Wetlab
Week 1:
Laboratory orientation- learned safety code, where things are in the lab, started work on HPLC
Completed Shuttle Vector design for C. hutchinsonii
Ordered Erythromycin G-Block
Designed G-Block for C. hutchinsonii origin of replication
Discovered that there were too many “A”s at the end of the Origin sequence so G-Block was impossible
Designed primers for Erythromycin gene from iGEM parts registry with MoClo ends C-D to be put into L0 vector
Designed primers for OriC with MoClo ends E-F to be put into L0 Vector
Week 2:
Received OriC and Ery primers
Ran Erythromycin gene PCR, successful
Ran OriC PCR, unsuccessful (x3)
Received Chloramphenicol backbone,LacZ fragment and primers with E and F fusion sites from Boston University for creation of DVC_EF backbone
Ran PCR of LacZ_EF fragment, successful
Ran ligation/transformed, all white so unsuccessful
Went to NEGEM
Week 3:
Ordered new OriC primers with lower TM
Received new primers
Ran OriC PCR, unsuccessful (x2)
Ran colony PCR using C. hutchinsonii with new primers, unsuccessful
Ran ethanol precipitation on C. hutchinsonii to extract genomic DNA
Ran OriC PCR with extracted genomic DNA, unsuccessful
Digested/Ligated/Transformed DVC backbone with LacZ_EF fragment, unsuccessful
Troubleshooted DVC_EF, ran controls, found that the XGal was not working in the chloramphenicol plates.
Week 4:
Ordered new OriC and Ery primers (Ery had BsaI sites instead of BbsI)
Re-ran ethanol precipitation, extracted genomic DNA
Ran OriC PCR, successful
Digested/Ligated/transformed DVC_EF, picked blue colonies, successful
Week 5:
Golden-gated OriC into DVC_EF (x3), eventually successful
Golden-gated OriC into DVK_EF, successful
Designed preliminary circuit, did not use old shuttle vector design. New circuit exhibited use of the LuxR/LuxI Quorum Sensing system
C. hutchinsonii produces Lux AHL constitutively. E. Coli expresses suicide gene (RelE) when E. coli concentration is high and relies on signal from C. hutchinsonii to express an antidote gene (RelB). Due to the strong RBS, Coli requires only a low concentration of the LuxR AHL to rescue itself. A strong LuxR signal (suggesting an abundance of C. hutchinsonii) will cause the LasR AHL to be sent from E. Coli, activating the C. hutchinsonii suicide gene (also RelE). Possible states and outcomes: E high/C low - Plas (and RelE) effectively constitutively active in E. E won’t get enough antitoxin to save self, C won’t kill self. E decreases, C increases. C high/E low - C sends Lux AHL to E, promoting E to save self. C stays high, E increases. E high/C high - Both signals active. C sends Lux AHL to E, promoting E to save self. E sends Las AHL to C. E increases slightly, C decreases slightly. C low/E low - Neither signal active. If signals are negligible, similar growth to a naive co-culture.
Week 6:
Notes: -won’t work. RBS strengths shouldn’t be used for time sensitive things/changing states, only for adjusting steady states/equilibria -simpler one-way communication -relies on E. coli utilizing material from dead C. hutchinsonii #cannibalism? -relies on behavior dictated by metabolic link (as determined from naive co-culture) -need to design more MoClo sites to fit all 5 biodiesel + LuxI + CamR + GFP on one plasmid -need to check this with a “real” adult/advisor who is not an undergraduate   requires TetR to repress RelB more strongly than RelB represses RelE. don’t use different RBS strengths on them. Could be hard to tweak. deals with 3 situations: E Coli nonexistent/ very low: C hutchinsonii lives and grows E Coli medium/high and less E Coli than C hutchinsonii: C hutchinsonii dies (hopefully E Coli will eat the C hutchinsonii and grow more quickly) E Coli medium/high, more E Coli than C hutchinsonii: C hutchinsonii lives and grows Long term behaviour: C hutchinsonii will try to outgrow E Coli, but will be reigned back in by its need for AHLs produced by E Coli. We can tune the ratio using the strengths of the components in the Lux and RelE/B pathway Settled on final circuit Ordered LuxR/LuxI primers
Week 7:
Made list of specific parts we would need to test all failure modes of the project Received LuxR/LuxI primers Ran PCR (x2), eventually successful Golden-gated LuxR, LuxI, pLux (promoter) into DVC_CD backbone, LuxR/LuxI unsuccessful, pLux successful (x2)
Week 8:
Ordered new primers for backbones DVC_AC, DVC_AG, DVC_AH, DVK_AG, and DVK_AH for different sets of tests for the final circuit. Golden-gated LuxR/LuxI into L0s, eventually successful (x3)
Week 9:
Golden-gated LuxR/LuxI into DVK L1 vectors to build transcriptional units, unsuccessful Golden-gated GFP/RFP TUs with OriC, unsuccessful
Week 10:
Started having problems with Liquid Cultures not growing ran test to see if it was a problem with the antibiotic. It was not. Tried making DVK and DVC backbones again, unsuccessful (x3)
