<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/Modeling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenu"><DIV id="navmenubox"><UL id="nav" class="col-md-10"><LI><A href="https://2015.igem.org/Team:MIT">Home</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Description">Project</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Coculture">The Co-Culture</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Chutch">C. Hutchinsonii</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/BiodieselProduction">Biodiesel Production</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Parts">Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Circuit">Circuit</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Modeling">Modeling</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Promoter">Promoter Characterization</A></LI></UL><LI><A href="https://2015.igem.org/Team:MIT/Practices"> Human Practices</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/CBP">Consolidated Bioprocessing</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Economics">Economics</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Experts">Experts</A></LI><LI><A href="https://2015.igem.org/Team:MIT/WetlabDrylab">Notebooks</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/WetlabDrylab">Wetlab/Drylab</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Measurement">Measurement</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Team">About Us</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Team">Team Bios</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Acknowledgements">Acknowledgements</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Collaborations"> Externals</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/InterlabStudy">Interlab Study</A></LI><DIV class="col-md-2"><DIV id="logos"><DIV id="twittericon" class="sidelogos"><A class="twitter-timeline" href="https://twitter.com/MIT_iGEM" data-widget-id="608663252705374209">@MIT_iGEM</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="contents" class="subtitle"><A href="https://2015.igem.org/Team:MIT/ModelingDFBA">Dynamic Flux Balance Analysis</A><A href="https://2015.igem.org/Team:MIT/ModelingCHutch">C. Hutchinsonii Model Development</A><A href="https://2015.igem.org/Team:MIT/ModelingEColi">E. Coli Model Development</A><A href="https://2015.igem.org/Team:MIT/ModelingCoCulture">Coculture Simulations and Conclusions</A></DIV><DIV id="spacer" style="display:block-inline"><DIV id="title">
			Modeling Overview
		</DIV><DIV class="subtitle">
			Introduction to Metabolic Community Modeling
		</DIV><DIV class="text" align="left">
			Models greatly aid in designing synthetic consortia by allowing researchers to study microbial interactions and predict how they influence the dynamics of the co-culture under different conditions. This enables researchers to engineer community members, composition, interactions, and environment to make the consortia have more desirable characteristics such as efficiency of production or stability (Mahadevana and Henson 2012, Biggs et al 2015).
			
			Early modeling efforts used an unstructured and lumped approach, in which all cellular components were lumped in the total biomass concentration and growth depended on a single limiting substrate. Two important advancements greatly improved these models: (1) the development of a cellular/community objective function that is maximized using linear programming and (2) the advent of genome-sequencing technologies which allowed the construction of genome-scale metabolic models. This improved approach came to be known as constraint-based modeling. A type of constraint-based modeling is flux balance analysis (FBA), which uses optimization to solve for the steady-state fluxes of metabolites, given the constraints from the whole-genome scale models. FBA is limited because it assumes that the intracellular and extracellular states are time invariant and interactions between microbes are at steady state. To overcome this limitation, dynamic flux balance analysis (dFBA) was developed. dFBA uses, in addition to genome-scale metabolic models of the species, dynamic extracellular mass balances on important metabolites and uptake/secretion kinetics for each substrate/product. Thus, dFBA accounts for time-varying microbial interactions, and allows for the prediction of microbial population, substrate utilization, and product formation dynamics (Bernstein and Carlson 2012). The main advantage of using dFBA, as opposed to other dynamic cell modeling approaches, is that very little additional information besides the whole-genome scale metabolic models is required; these whole-genome metabolic models are becoming increasingly available (Jayaraman and Hahn 2009, Methods in Bioenegineering). The main challenges of dFBA are determining the substrate uptake kinetics, which must be done experimentally, and numerically solving the dFBA problem ((Jayaraman and Hahn 2009, Methods in Bioenegineering).
		</DIV><DIV class="subtitle">
			Overview of Our Approach
		</DIV><DIV class="text" align="left">
			Here, we apply dFBA to model our co-culture of C. hutchinsonii and E. coli. The goal of our models is to predict how the bacterial populations and metabolite concentrations in our co-culture change over time under different conditions. Our models aid in designing interactions between the two bacteria to stabilize the co-culture, in characterizing the metabolic interactions between the two bacteria, and in optimizing conditions for biofuel production. We implemented our models by obtaining and modifying existing whole-genome scale models of E. coli and C. hutchinsonii, curating and estimating kinetic parameters, and simulating both pure and co-cultures using dFBA software.
		</DIV><DIV class="subtitle">
			Overview of Implementation
		</DIV><DIV class="text" align="left">
			To modify the whole-genome scale metabolic models of E. coli and C. hutchinsonii, we used the Constraints Based Reconstruction and Analysis (COBRA) toolbox in MATLAB (Schellenberger et al 2011). We obtained the iJO1366 model of E. coli online (Orth et al. 2011), and modified it by adding the genes/reactions for biodiesel production, changing the exchange reaction bounds to reflect our culture conditions, knocking out genes to optimize biodiesel production, and adding genes/reactions to account for our synthetic communication network. Where possible, we used reported values of uptake and inhibition kinetic parameters, but we also had to create new parameters to match other existing data or known behavior.
			
			Getting a valid whole-genome scale model for C. hutchinsonii was much more challenging because it is relatively poorly characterized. We investigated several whole genome-scale models for C. hutchinsonii and found that many contained gaps, repeated metabolites, or inaccurate exchange reaction bounds. We decided to use the modelSEED whole-genome scale model (Henry et al 2010) because, although it is less detailed, it contains no repeated metabolites. We modified it by fixing exchange reaction bounds, adding hypothesized reactions for lignocellulose degradation, and adding genes/reactions to account for our synthetic communication network. Since there are no reported kinetic parameters for C. hutchinsonii, we created parameters to match existing data and known behavior.
			
			To simulate monocultures and co-cultures using dFBA, we use a MATLAB-based package called DFBAlab, that was developed by the Process Systems Engineering Laboratory at MIT (Gomez et al 2014), with the linear programming solver Gurobi (gurobi 2015). DFBAlab performs fast and reliable simulation of complex dynamic multispecies cultures using techniques (differential-algebraic equation (DAE) system and lexicographic optimization) that ensure feasibility and unique exchange fluxes.
		</DIV></DIV></DIV></DIV></BODY></HTML>