<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_basic_part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/basic part</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenu"><DIV id="navmenubox"><UL id="nav" class="col-md-10"><LI><A href="https://2015.igem.org/Team:MIT">Home</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Description">Project</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Coculture">The Co-Culture</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Chutch">C. Hutchinsonii</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/BiodieselProduction">Biodiesel Production</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Parts">Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Circuit">Circuit</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Modeling">Modeling</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Promoter">Promoter Characterization</A></LI></UL><LI><A href="https://2015.igem.org/Team:MIT/Practices"> Human Practices</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/CBP">Consolidated Bioprocessing</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Economics">Economics</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Experts">Experts</A></LI><LI><A href="https://2015.igem.org/Team:MIT/WetlabDrylab">Notebooks</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/WetlabDrylab">Wetlab/Drylab</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Measurement">Measurement</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Team">About Us</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Team">Team Bios</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Acknowledgements">Acknowledgements</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Collaborations"> Externals</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/InterlabStudy">Interlab Study</A></LI><DIV class="col-md-2"><DIV id="logos"><DIV id="twittericon" class="sidelogos"><A class="twitter-timeline" href="https://twitter.com/MIT_iGEM" data-widget-id="608663252705374209">@MIT_iGEM</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="spacer"><DIV id="title"><I>Cytophaga hutchinsonii</I> Origin of Replication
		</DIV><DIV class="subtitle">oriC (BBa_K1705010)</DIV><DIV class="text" align="center">
An origin of replication (ori) is necessary for a plasmid to be maintained in a cell line through multiple rounds of reproduction.  Unlike E. coli; which has a range of ori’s characterized for a variety of copy numbers and purposes, the only reasonably well known ori for <I>C. hutchinsonii</I> is its chromosomal origin of replication.  Given the limited existing knowledge, we decided to use the chromosomal ori as the basis of our part.  Following the protocol described in “Development of replicative oriC plasmids and their versatile use in genetic manipulation of <I>Cytophaga hutchinsonii</I>” by Xu et. al1 we PCR amplified the chromosomal origin of replication (oriC) from the <I>C. hutchinsonii</I> genome.  We decided to make the oriC a modular part (as opposed to part of a plasmid backbone) so that it could easily be inserted into existing plasmids in our MoClo library to make them useable in <I>C. hutchinsonii</I>  To do this, we added 5’ extensions to the PCR primers which included BbsI recognition sites and special fusion sites designated E and F, making the part MoClo compatible.  The extensions allowed us to insert the PCR product into a Level 0 destination vector for use in the modular cloning system.  The resulting part is compatible with MoClo assembly standard.  Unfortunately, the ori contains a cut site making the part RFC10 incompatible.  We planned site-directed mutagenesis to remove this cut site but ran out of time; completing this SDM would be a good future improvement to the part.
	This part’s function was validated by transforming the Level 0 vector in both a chloramphenicol and kanamycin resistance backbone into <I>C. hutchinsonii</I> and growing the transformants on selective media.  See “Transforming <I>C. hutchinsonii</I>” for more details on this experiment.
		</DIV></DIV></DIV></DIV></BODY></HTML>