<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/Parts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenu"><DIV id="navmenubox"><UL id="nav" class="col-md-10"><LI><A href="https://2015.igem.org/Team:MIT">Home</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Description">Project</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Coculture">The Co-Culture</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Chutch">C. Hutchinsonii</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/BiodieselProduction">Biodiesel Production</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Parts">Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Circuit">Circuit</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Modeling">Modeling</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Promoter">Promoter Characterization</A></LI></UL><LI><A href="https://2015.igem.org/Team:MIT/Practices"> Human Practices</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/CBP">Consolidated Bioprocessing</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Economics">Economics</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Experts">Experts</A></LI><LI><A href="https://2015.igem.org/Team:MIT/WetlabDrylab">Notebooks</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/WetlabDrylab">Wetlab/Drylab</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Measurement">Measurement</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Team">About Us</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Team">Team Bios</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Acknowledgements">Acknowledgements</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Collaborations"> Externals</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/InterlabStudy">Interlab Study</A></LI><DIV class="col-md-2"><DIV id="logos"><DIV id="twittericon" class="sidelogos"><A class="twitter-timeline" href="https://twitter.com/MIT_iGEM" data-widget-id="608663252705374209">@MIT_iGEM</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="contents" class="subtitle"><A href="https://2015.igem.org/Team:MIT/basic_part">Basic Part</A></DIV><DIV id="spacer"><DIV id="title">
			Parts
		</DIV><DIV class="text" align="center">
Biodiesel Parts
We wanted to engineer production of some useful compound into E. coli to demonstrate the capability of a co-culture for consolidated bioprocessing. Biodiesel was a prime target because its production in E. coli has been shown in the past, and it is industrially relevant. 
We based our pathway for production of biodiesel (fatty acid ethyl esters) on that in Steen et al. The genes we planned to use are listed in the table below. The Keasling lab, co-authors on the Steen paper, graciously sent us an E. coli strain with fadE knocked out. fadE is involved in degradation of fatty acids from Acyl-CoA, so this knockout produces more Acyl-CoA which can be used to make FAEEs. 

Gene
Source
Function
Registry Identifier
pyruvate decarboxylase (pdc)
Zymomonas mobilis
pyruvate → acetaldehyde
N/A
alcohol dehydrogenase B (adhB)
Zymomonas mobilis
acetaldehyde → ethanol
BBa_K1705000
thioesterase (‘tesA)
E. coli
Acyl-ACP → FFA
BBa_K1705002
acyl-CoA synthetase (fadD)
E. coli
FFA→ Acyl-CoA
BBa_K1705003
acyl transferase (atfA)
Acinetobacter sp. ADP1
ethanol + Acyl-CoA → FAEE
BBa_K1705001
→ = reaction catalyzed by enzyme; ACP = acyl carrier protein; FFA = free fatty acid; FAEE = fatty acid ethyl ester

However, we were not able to successfully clone pdc into a transcriptional unit and were thus unable to make a plasmid containing all of the genes necessary for biodiesel production. For each of the other genes, we did succeed in making both basic parts plasmids and transcriptional units. Because we did not have all of the parts together, though, we were unable to assay their functionality; we did not possess the machinery to measure the intermediate metabolites. Our submitted parts have been sequence verified, however, and we hope that they may be of use to a team in the future pursuing further work in this area.
Quorum-sensing Parts
To create communication between the two bacteria in our co-culture, we decided to use the Lux system. Because Lux has been well-characterized and works in many organisms, we proposed it would function in <I>C. hutchinsonii</I> as well. Quorum-sensing was also ideal because we wanted the amount of the signal to be proportional to the population for controlling the ratios between the two strains.
We were unable to get far enough in our experiments to verify functionality of quorum-sensing in <I>C. hutchinsonii</I>, but we 
L0 DVC pLux_AB
L0 DVC luxR_CD (bound by COC6HSL)
L0 DVC luxI_CD (produces COC6HSL)
L0 DVC luxI LVA_CD (produces COC6HSL)
Toxin/Antitoxin Parts
In order to control the population ratios of <I>E. coli</I> and <I>Cytophaga hutchinsonii</I>, our circuit used a toxin/antitoxin system composed of relE and relB. Because these two proteins have been shown to work across many organisms, we hypothesized that they would function in <I>C. hutchinsonii</I> as well. 
<I>E. coli</I> is already reliant upon <I>C. hutchinsonii</I> for the sugars it uses as its carbon source. Having <I>C. hutchinsonii</I> produce a toxin, for which it can only produce the antitoxin in the presence of <I>E. coli</I>, established a link between the populations in the other direction. We planned to fine-tune the population ratios by using different RBS strengths, but we were unable to reach the experimental phase of this part of the experiment. We are submitting our basic parts for use by other teams needing similar communication pathways.
The coding sequences for these two parts were codon optimized for expression in <I>C. hutchinsonii</I> using <A href="”www.jcat.de”">Jcat</A>. The sequences were then fine-tuned by hand to make them more easily synthesizable (e.g. by adjusting the GC content of certain regions) using the codon frequency tables at <A href="“http://www.kazusa.or.jp/codon/cgi-bin/showcodon.cgi?species=269798&amp;aa=15&amp;style=N”">Kazusa</A>.

Gene
Source
Function
Registry Identifier
relB
E. coli
Produces antitoxin
BBa_K1705005
relE
E. coli
Produces toxin
BBa_K1705004

<I>C. hutchinsonii</I> part design:
	In order to make <I>C. hutchinsonii</I> more engineerable, we had to develop some foundational parts such as an origin of replication, promoter and ribosome binding site.
oriC (BBa_K1705010):
	An origin of replication (ori) is necessary for a plasmid to be maintained in a cell line through multiple rounds of reproduction.  Unlike E. coli; which has a range of ori’s characterized for a variety of copy numbers and purposes, the only reasonably well known ori for <I>C. hutchinsonii</I> is its chromosomal origin of replication.  Given the limited existing knowledge, we decided to use the chromosomal ori as the basis of our part.  Following the protocol described in “Development of replicative oriC plasmids and their versatile use in genetic manipulation of <I>Cytophaga hutchinsonii</I>” by Xu et. al1 we PCR amplified the chromosomal origin of replication (oriC) from the <I>C. hutchinsonii</I> genome.  We decided to make the oriC a modular part (as opposed to part of a plasmid backbone) so that it could easily be inserted into existing plasmids in our MoClo library to make them useable in <I>C. hutchinsonii</I>  To do this, we added 5’ extensions to the PCR primers which included BbsI recognition sites and special fusion sites designated E and F, making the part MoClo compatible.  The extensions allowed us to insert the PCR product into a Level 0 destination vector for use in the modular cloning system.  The resulting part is compatible with MoClo assembly standard.  Unfortunately, the ori contains a cut site making the part RFC10 incompatible.  We planned site-directed mutagenesis to remove this cut site but ran out of time; completing this SDM would be a good future improvement to the part.
	This part’s function was validated by transforming the Level 0 vector in both a chloramphenicol and kanamycin resistance backbone into <I>C. hutchinsonii</I> and growing the transformants on selective media.  See “Transforming <I>C. hutchinsonii</I>” for more details on this experiment.

Native Promoter and Ribosome Binding Site (BBa_K1705012 and BBa_K1705011):
	Several papers which discuss structural motifs of promoters in the phylum bacteroidetes3 suggest that they are distinct enough that the more common and well characterized E. coli based promoters may have little to no activity in bacteroidetes.  Based on this, it seemed worthwhile to investigate transcriptional/translational initiation units native to the <I>C. hutchinsonii</I> genome.  The Xu et. al.1 paper mentioned using a PCR product from the region upstream of the CHU_1284 gene to initiate transcription and translation of heterologous genes.  As a positive control, we repeated the PCR described in the paper with modifications similar to the oriC part to make the PCR product MoClo compatible.  The PCR product described in the paper was roughly 800 bp long and included bits of other genes according to an annotated copy of the genome so, although it was useful as a positive control, it was not ideal when it comes to specifying individual functional units.  In hopes of better characterization, we attempted to determine which sequences within this fragment were the actual promoter and ribosome binding site.  
	For the promoter, we began by feeding the sequence of the PCR product through BPROM - a bacterial promoter recognition tool2.  BPROM returned many possible sequences, three of which had a probability greater than 95% of being a promoter.  Given a selection with such high probability, we chose to focus on those three.  One sequence was ruled out because it was included in a region annotated as another gene and was very far from the start of the gene this promoter was meant to control.  The other two were compared against the consensus sequence for bacteroidetes promoter - a -33 TTTG and -7 TANNTTTG sequence - and the most likely candidate was selected based on similarity to that sequence.  The selected sequence was also the closer of the two to the gene which gives a little more credibility to it being the correct sequence.
	We ran into a bit of luck when it came to identifying a native ribosome binding site.  The paper which we pulled the PCR product from had annotated a small 8 bp sequence as the Shine-Dalgarno sequence.  The SD sequence is the essential part of the RBS to which the 16S rRNA binds.  Given this annotation, we thought this would be a good place to start in terms of testing native RBSs.  Also, based on past observations that ribosome binding sites are typically very AG rich, we decided to include the A and G on either side of the annotation in the final part.
	Prior to attempting to construct physical parts for either sequence, we added in BbsI recognition sites on both the 5’ and 3’ ends as well as predetermined fusion sites to allow the parts to be used in the Modular Cloning system.  We ordered both the forward and reverse compliment sequence for each part from IDT as oligonucleotides which we then annealed together and inserted into a Level 0 vector using a golden gate reaction.

Steen et al. Nature, 463:559-562 (2010). 
1Xu et al, Applied Genetics and Molecular Biotechnology, 93:697–705 (2012)
2http://linux1.softberry.com/berry.phtml?topic=bprom&amp;group=programs&amp;subgroup=gfindb
3Chen et al, Journal of Bacteriology, 189:5108-5118 (2007)

		</DIV></DIV></DIV></DIV></BODY></HTML>