<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_WetlabDrylab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/WetlabDrylab</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenu"><DIV id="navmenubox"><UL id="nav" class="col-md-10"><LI><A href="https://2015.igem.org/Team:MIT">Home</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Description">Project</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Coculture">The Co-Culture</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Chutch">C. Hutchinsonii</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/BiodieselProduction">Biodiesel Production</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Parts">Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Circuit">Circuit</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Modeling">Modeling</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Promoter">Promoter Characterization</A></LI></UL><LI><A href="https://2015.igem.org/Team:MIT/Practices"> Human Practices</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/CBP">Consolidated Bioprocessing</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Economics">Economics</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Experts">Experts</A></LI><LI><A href="https://2015.igem.org/Team:MIT/WetlabDrylab">Notebooks</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/WetlabDrylab">Wetlab/Drylab</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Measurement">Measurement</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Team">About Us</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Team">Team Bios</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Acknowledgements">Acknowledgements</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:MIT/Collaborations"> Externals</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:MIT/InterlabStudy">Interlab Study</A></LI><DIV class="col-md-2"><DIV id="logos"><DIV id="twittericon" class="sidelogos"><A class="twitter-timeline" href="https://twitter.com/MIT_iGEM" data-widget-id="608663252705374209">@MIT_iGEM</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="spacer"><DIV id="title">
			Wetlab
		</DIV><DIV class="subtitle">Week 1:
		</DIV><DIV class="text" align="left"><UL align="left"><LI>Laboratory orientation- learned safety code, where things are in the lab, started work on HPLC
</LI><LI>Completed Shuttle Vector design for C. hutchinsonii
</LI><LI>Ordered Erythromycin G-Block
</LI><LI>Designed G-Block for C. hutchinsonii origin of replication
</LI><LI>Discovered that there were too many “A”s at the end of the Origin sequence so G-Block was impossible
</LI><LI>Designed primers for Erythromycin gene from iGEM parts registry with MoClo ends C-D to be put into L0 vector
</LI><LI>Designed primers for OriC with MoClo ends E-F to be put into L0 Vector
</LI></UL></DIV><DIV class="subtitle">Week 2:
		</DIV><DIV class="text" align="left"><UL align="left"><LI>Received OriC and Ery primers
</LI><LI>Ran Erythromycin gene PCR, successful
</LI><LI>Ran OriC PCR, unsuccessful (x3)
</LI><LI>Received Chloramphenicol backbone,LacZ fragment and primers with E and F fusion sites from Boston University for creation of DVC_EF backbone
</LI><LI>Ran PCR of LacZ_EF fragment, successful
</LI><LI>Ran ligation/transformed, all white so unsuccessful
</LI><LI>Went to NEGEM
</LI></UL></DIV><DIV class="subtitle">Week 3:</DIV><DIV class="text" align="left"><UL align="left"><LI>Ordered new OriC primers with lower TM
</LI><LI>Received new primers
</LI><LI>Ran OriC PCR, unsuccessful (x2)
</LI><LI>Ran colony PCR using C. hutchinsonii with new primers, unsuccessful
</LI><LI>Ran ethanol precipitation on C. hutchinsonii to extract genomic DNA
</LI><LI>Ran OriC PCR with extracted genomic DNA, unsuccessful
</LI><LI>Digested/Ligated/Transformed DVC backbone with LacZ_EF fragment, unsuccessful
</LI><LI>Troubleshooted DVC_EF, ran controls, found that the XGal was not working in the chloramphenicol plates.
</LI></UL></DIV><DIV class="subtitle">Week 4:</DIV><DIV class="text" align="left"><UL align="left"><LI>Ordered new OriC and Ery primers (Ery had BsaI sites instead of BbsI)
</LI><LI>Re-ran ethanol precipitation, extracted genomic DNA
</LI><LI>Ran OriC PCR, successful
</LI><LI>Digested/Ligated/transformed DVC_EF, picked blue colonies, successful
</LI></UL></DIV><DIV class="subtitle">Week 5:</DIV><DIV class="text" align="left"><UL align="left"><LI>Golden-gated OriC into DVC_EF (x3), eventually successful
</LI><LI>Golden-gated OriC into DVK_EF, successful
</LI><LI>Designed preliminary circuit, did not use old shuttle vector design. New circuit exhibited use of the LuxR/LuxI Quorum Sensing system
</LI><LI>C. hutchinsonii produces Lux AHL constitutively. E. Coli expresses suicide gene (RelE) when E. coli concentration is high and relies on signal from C. hutchinsonii to express an antidote gene (RelB). Due to the strong RBS, Coli requires only a low concentration of the LuxR AHL to rescue itself. A strong LuxR signal (suggesting an abundance of C. hutchinsonii) will cause the LasR AHL to be sent from E. Coli, activating the C. hutchinsonii suicide gene (also RelE).
Possible states and outcomes:
E high/C low - Plas (and RelE) effectively constitutively active in E. E won’t get enough antitoxin to save self, C won’t kill self. E decreases, C increases.
C high/E low - C sends Lux AHL to E, promoting E to save self. C stays high, E increases.
E high/C high - Both signals active. C sends Lux AHL to E, promoting E to save self. E sends Las AHL to C. E increases slightly, C decreases slightly.
C low/E low - Neither signal active. If signals are negligible, similar growth to a naive co-culture.


		<DIV class="subtitle">Week 6:</DIV><DIV class="text" align="center">

Notes:
-won’t work. RBS strengths shouldn’t be used for time sensitive things/changing states, only for adjusting steady states/equilibria
-simpler one-way communication
-relies on E. coli utilizing material from dead C. hutchinsonii #cannibalism?
-relies on behavior dictated by metabolic link (as determined from naive co-culture)
-need to design more MoClo sites to fit all 5 biodiesel + LuxI + CamR + GFP on one plasmid
-need to check this with a “real” adult/advisor who is not an undergraduate


requires TetR to repress RelB more strongly than RelB represses RelE. don’t use different RBS strengths on them. Could be hard to tweak.
deals with 3 situations:
E Coli nonexistent/ very low: C hutchinsonii lives and grows
E Coli medium/high and less E Coli than C hutchinsonii: C hutchinsonii dies (hopefully E Coli will eat the C hutchinsonii and grow more quickly)
E Coli medium/high, more E Coli than C hutchinsonii: C hutchinsonii lives and grows
Long term behaviour: C hutchinsonii will try to outgrow E Coli, but will be reigned back in by its need for AHLs produced by E Coli. We can tune the ratio using the strengths of the components in the Lux and RelE/B pathway
Settled on final circuit
Ordered LuxR/LuxI primers

		</DIV><DIV class="subtitle">Week 7:</DIV><DIV class="text" align="center">
Made list of specific parts we would need to test all failure modes of the project
Received LuxR/LuxI primers
Ran PCR (x2), eventually successful
Golden-gated LuxR, LuxI, pLux (promoter) into DVC_CD backbone, LuxR/LuxI unsuccessful, pLux successful (x2)

		</DIV><DIV class="subtitle">Week 8:</DIV><DIV class="text" align="center">
Ordered new primers for backbones DVC_AC, DVC_AG, DVC_AH, DVK_AG, and DVK_AH for different sets of tests for the final circuit.
Golden-gated LuxR/LuxI into L0s, eventually successful (x3)

		</DIV><DIV class="subtitle">Week 9:</DIV><DIV class="text" align="center">
Golden-gated LuxR/LuxI into DVK L1 vectors to build transcriptional units, unsuccessful
Golden-gated GFP/RFP TUs with OriC, unsuccessful

		</DIV><DIV class="subtitle">Week 10:</DIV><DIV class="text" align="center">
Started having problems with Liquid Cultures not growing ran test to see if it was a problem with the antibiotic. It was not.
Tried making DVK and DVC backbones again, unsuccessful (x3)

		</DIV></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>