<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ACIBADEM_ISTANBUL_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ACIBADEM ISTANBUL/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>PARTS</TITLE></P><DIV class="top"><DIV class="menu"><UL><LI><A href="https://2018.igem.org/Team:ACIBADEM_ISTANBUL"><DIV class="isim">Home</DIV></A></LI><LI><A href="https://2018.igem.org/Team:ACIBADEM_ISTANBUL/Parts"><DIV class="isim">Project</DIV></A></LI><LI><A href="https://2018.igem.org/Team:ACIBADEM_ISTANBUL/Team"><DIV class="isim">Team</DIV></A></LI><LI><A href="https://2018.igem.org/Team:ACIBADEM_ISTANBUL/Lab"><DIV class="isim">Lab</DIV></A></LI><LI><A href="https://2018.igem.org/Team:ACIBADEM_ISTANBUL/Improve"><DIV class="isim">Parts</DIV></A></LI><LI><A href="https://2018.igem.org/Team:ACIBADEM_ISTANBUL/Human_Practices"><DIV class="isim">Human Practice</DIV></A></LI><LI><A href="https://2018.igem.org/Team:ACIBADEM_ISTANBUL/Judging"><DIV class="isim">Judging Form</DIV></A></LI><LI><A href="https://2018.igem.org/Team:ACIBADEM_ISTANBUL/Attributions"><DIV class="isim">Attributions</DIV></A></LI></UL></DIV><DIV class="hepsi"><DIV class="partscontent"><H2 style="text-align: center">Table of Contents</H2><SECTION class="sect" style="margin-left: 20px"><A href="#Why">1. Why was it designed?</A><A href="#Usage">2. Usage</A><A href="#primer">3. Primers</A><A href="#original">4. Original Part</A><A href="#our">5. Our (Improved) Part</A><A href="#hara">6. Characterisation</A><A href="#what">7. What experiments did we do?</A></SECTION></DIV><DIV class="icerikler"><DIV id="Why"><H3>Why was it Designed?</H3><SECTION>LTNF 2.0 was designed to be an improved version of the original LTNF peptide in almost every way possible. LTNF has already proven to characterize anti-venom activity. Our objective was to design a more effective, more stable and safer version of LTNF using an utterly simple circularization process.
With LTNF 2.0, we finally have a viable peptide-based anti-venom that can finally replace the ancient and morbidly flawed immunoglobulin based anti-venom.
</SECTION></DIV><DIV id="Usage"><H3>Usage</H3><SECTION>With LTNF 2.0, we finally have a viable peptide based anti-venom that can challenge the traditional, immunological based approach that has remained unchanged for over a century.
LTNF 2.0 can be used an affordable, readily available, easy to manufacture and safe treatment modality in snake endemic regions against envenomation.   
</SECTION></DIV><DIV id="primer"><H3>Primers</H3><SECTION><P>We teamed up with sentegen, a leading Turkish synthetic biology company, to design and synthesize the primers required for the Biobrick part registration.</P><DIV><TABLE class="tableee" width="100%" border="0" cellpadding="5"><TBODY><TR><TD style="color: #7787D8" align="center"><STRONG style="color: #7787D8">LTNF 2.0</STRONG></TD><TD style="color: #7787D8" align="center"><STRONG style="color: #7787D8">Quality Control</STRONG></TD></TR><TR><TD><STRONG>Forward Primer</STRONG></TD><TD><EM>(C_LTNF_C F_EcoRI)</EM>gcgcgcGAATTCATGTGTGGCGGTGGAGGC</TD><TD><EM>(BBa_G00100)</EM>tgccacctgacgtctaagaa</TD></TR><TR><TD><STRONG>Reverse Primer</STRONG></TD><TD><EM>(C _LTNF_C_R_PstI)</EM>gcgcgcCTGCAGTCAACAAGGCGTACTTGG</TD><TD><EM>(BBa_G00101)</EM>attaccgcctttgagtgagc</TD></TR></TBODY></TABLE></DIV></SECTION></DIV><DIV id="original"><H3>Original Parts</H3><SECTION><A href="http://parts.igem.org/Part:BBa_K1915001">http://parts.igem.org/Part:BBa_K1915001</A></SECTION><SECTION>This part contains the first 10 amino acids of the Lethal Toxin Neutralizing Factor, a protein found in the serum of the Opossum (Didelphis virginiana).
Previous studies have shown that only the first 10 amino acids are required to capture the anti-venom activity of LTNF protein.<STRONG>DNA sequence</STRONG><SPAN style="color: #578069">CTG AAA GCG ATG GAT CCG ACC CCG CCG CTG</SPAN><STRONG>Amino acid sequence</STRONG><SPAN style="color: #5B7A98">L K A M D P T P P L</SPAN></SECTION></DIV><DIV id="our"><H3>Our (Improved) Part</H3><SECTION><A href="http://parts.igem.org/Part:BBa_K2815001">http://parts.igem.org/Part:BBa_K2815001</A></SECTION><SECTION> This is LTNF 2.0, an improved version of the original part in almost every way. <STRONG> DNA sequence</STRONG>
The DNA sequence was optimized for use in Pichia pastoris by using IDT’s Codon Optimization tool.<SPAN style="color: #578069">TGT GGC GGT GGA GGC TTG AAA GCA ATG GAC CCT ACC CCC CCC CTG TGG ATA AAA ACG GAG TCA CCA AGT ACG CCT TGT</SPAN><STRONG> Amino acid sequence</STRONG><SPAN style="color: #5B7A98">C GGGG LKAMDPTPPLWIKTESPSTP C</SPAN><P><EM> It contains:</EM></P><STRONG> Cysteine</STRONG> Two Cysteine amino acids have been added as the first (1) and last (26) amino acids in the part. The two Cysteine amino acids form a disulphide bridge, thereby connecting the ends of the polypeptide chain together resulting in a more circularized structure, this process is called, “Circularization.”
Circularization causes an increase in entropy, resulting in an increase in stability. More stable peptides have fever toxic effects and longer half-lives compared to non-circularized ones.
Circularization can also increase the bioactivity and thus efficacy of the peptide, resulting in greater potency.
Spacer
Early modeling studies indicated that placing two Cysteine amino acids right next to the LTNF (Active site) sequence caused a great deal of strain, reducing its c-score as well as its activity. Four Glycine amino acids were found to greatly reduce the stress found between the first cysteine amino acid and the LTNF 10 active site, whilst also avoiding interfering with its biochemical properties, given its chemically inert and simplistic nature.<STRONG> LTNF 10</STRONG>
 The original Biobrick part that displays anti-venom activity via the inhibition of metalloproteinases.<STRONG>Extension</STRONG>
 Adding a Cystiene bond to the end of the 10 amino acid LTNF sequence resulted in stress to the end part of the active site. As a way to combat this, we began adding the natural continuation of the LTNF 10 amino acids bit by bit, the highest c-score occurred when exactly an additional 10 amino acids (extensions) were added to the LTNF 10, striking a balance between the least stress and highest entropy levels. 
<STRONG> Where did we obtain it?</STRONG>
LTNF is naturally found in the serum of Opossums (Didelphis virginiana). LTNF 2.0 was ordered as a peptide from Peptide 2.0 and our plasmid was designed and manufactured by Genescript, using the pPICZalphaA plasmid.
				<STRONG>Biosafety:</STRONG>
Biosafety concerns were addressed in the mandatory safety form, a copy of which can be found here.<EM> To summarise:</EM>
Pichia pastoris is a yeast strain that displays a Biosafety Level of 1.
LTNF 2.0 is even safer than the original LTNF peptide, which already displays a Biosafety level of 1.
<STRONG> Debugging</STRONG>
We encountered one slight problem with the LTNF 2.0 that was delivered by Peptide 2.0, that being a large amount of electrostatic activity causing the lyopholized peptides to stick to our weighing spatula.
This resulted in us having to wash off the particles stuck to the weighing spatula back into their containers, lyophilzing it (a process that takes 24 hours) just to be able to weigh them for our experiments.
Sean Fisher from Peptide 2.0’s Customer Service stated that the electrostatic activity is most likely due to the peptides interaction with the plastic containers they were sent with.
As a solution, they have agreed to aliquot our future peptide orders into several vials in order to avoid this problem.</SECTION></DIV><DIV id="hara"><H3>Characterisation</H3><SECTION><STRONG> Degradation rate</STRONG> The original LTNF has a half-life of 0.5 hours.
The half-life of LTNF 2.0 has a half-life of 14.1 hours.
LTNF 2.0 is 28.2 times more stable than the original LTNF.<STRONG> Expression rate</STRONG>
For the predicted value, the maximum population density, specific growth rate, protein concentration, and minimum lag phase duration were 15.7 mg/ml, 3.4 h–1, 0.78 mg/ml, and 4.2 h, and the actual values were 14.3 ± 3.5 mg/ml, 3.6 ± 0.6 h–1, 0.72 ± 0.2 mg/ml, and 4.4 ± 1.6 h, respectively.<STRONG> Strain selection</STRONG>
LTNF 2.0 was designed with industrialization in mind, and has been optimized for fermentation via the pPICZaplhaA plasmid for its synthesis in the Pichia pastoris strain of yeast. 
Pichia pastoris was our yeast strain of choice for the following reasons:</SECTION><SECTION style="margin-left: 50px"><STRONG>1)</STRONG> It being a eukaryote allows for protein processing, post-translational modifications to be implemented, a feature that will come in handy when designing LTNF 3.0.<STRONG> 2)</STRONG> Can be used for large scale production, making it an ideal candidate for use in the industrialization of LTNF 2.0.<STRONG> 3)</STRONG> Can be used for high-yield, high productivity protein processing.<STRONG> 4)</STRONG> Produces proteins that are not contaminated with endotoxin or viruses.<STRONG> 5)</STRONG> Pichia pastoris has already been approved for therapeutic use by the biggest regulatory agencies. <STRONG> 6)</STRONG> Pichia pastoris can be used in edibles, meaning that it can be utilized to make a consumable, prophylactic anti-venom( “Calmette Loops” idea in Human Practice) for snake handlers and people living in snake-endemic regions.</SECTION></DIV><DIV id="what"><H3>What experiments did we do?</H3><SECTION> HPLC Peptide stability 
LTNF hemotoxic activity<STRONG> Controls
HPLC Peptide stability:</STRONG>
Our control group was the original, 10 Amino acid LTNF sequence.<STRONG> LTNF hemotoxic</STRONG>
Our control group was the original, 10 Amino acid LTNF sequence.</SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>