"
Page
Discussion
View source
History
teams
Log in
 
Team:ZJU-China/Perspective
From 2014.igem.org
Home
Team
Team information
Official Profile
Attributions
Project
Introduction
Background
λ-red
Bistable Switch
Gene Socket
Work Processing
Circuit Design
Solution
Modeling
Results
Perspective
GS-BOX
Parts
Lab
Notebook
Protocol
Lab Log
Safety
Achievement
Human Practice
Communication
iGEM
Team Sponsor
Contact us
Team Forum(BBS):
ZJU-China 2014 Team Forum on www.zjubiolab.zju.edu.cn
Post Address:
Biolab Center Room 413, ZJU Zijin'gang Campus,
Yuhang Tang Road No.866, Hangzhou, Zhejiang
Powered by Media Wiki. Zhejiang University.
Email Address:
zjuchina2014 @ 163.com
1 Most important: Our GeneSocket relieve lots of work of scientists.
As shown before, via the methods of lambda-red homologous recombination and new-designed bistable switch, we perfectly achieved our goal to make constructing vectors on chromosomes easier. If we can change the chromosomes as we planned, just through some simple steps, would it sound marvelous?
1.1 The prominent advantages of GeneSocket.
Our GeneSocket have a lot of prominent advantages and it is superb and unique. The advantages can be divided into three parts, CEO.
1.1.1 Convenience
First of all, as you can see, our GeneSocket provide large convenience for adding specific group of genes into specific vectors. It is just like plugging into the power sockets, which is what we do in everyday life, and can be so easy. For example, we have a blueprint in the future, that given a group of genes, maybe 4 or 5, and given metabolic pathways, which decide the logistic relationship among these proteins coded by these genes (even though these genes code promoters and terminators). And our super GeneSocket can handle it right away, with little difficulty or effort. What we do is that we just need to standardize every gene, adding standardized ends to them to change them into parts, and put them into cells one by one. Also because of using standardized ends, we try to create a database and software that can help us standardize the given genes into available parts that can be used in our GeneSocket ( Click here to get GS-BOX! ). We’re really glad to invite you to try our software and give us suggestions. What’s more, the workflow of GeneSocket will be given on the next section.
1.1.2 Efficiency
Secondly, our GeneSocket have high accuracy and high efficiency for constructing a vector. The high speed depends on the high efficiency of the homologous recombination we’ve chosen. Also, the high accuracy results from high accuracy of this homologous recombination. We chose lambda-red dependent homologous recombination as our tools to build GeneSocket, and it is one of the most efficient and mature ways to insert genes in prokaryotes. As we came up with this idea, we thought about the process that we could inset one gene by one day. Maybe, as the efficiency of lambda red could be high, we can insert genes faster than we expected. It can totally improve the efficiency and save time of scientific researches in the future.
1.1.3 Originality
Thirdly, we have an exquisite and original design without resistance screening. We add green and red light (sounds like traffic light) into this system to detect and verify whether the given gene has inserted into the specific spot. In this part, we use a new but great bistable switch to achieve this signal system. Our novel bistable switch is totally different from the traditional bistable switch we’ve seen for many times. It’s more stable and accurate. Through this signal system, do it step by step, we can confirm that every gene has been integrated successfully through every change between red and green light. However, that doesn’t mean that we can just use the fluorescence system. Our two reporters can be various, such as resistance (When the bacterium of interest count a small part of the whole bacterium, resistance can be a good screening way.), blue-white plaque selection and so on. We can choose the adequate reporter with our research. Also, compared to the traditional restriction enzyme digestion and ligation ways, our GeneSocket provide a chance to make a no gap linkage and sometimes it helps a lot.
What’s more, compared to the traditional lambda-red homologous recombination dependent gene knockout method, our novel homologous recombination based GeneSocket can be said that it excels its predecessor.
Period
Traditional lambda-red gene knockout method
GeneSocket
Preparation
Gene of interest are going to be inserted.
Insertion site should be found, a series of plasmids are used to acquire resistance gene with FRT or LoxP site.
All design and preparation steps are designed by GS-BOX.
Recombination
Insertion mode: Gene is linked to resistance gene, and two recombination sites are flanked on each side.
With standard helper parts BBa_K1433013, no resistance gene shall be used in recombination. Helper parts are also shorter than resistance gene to spare more space for GOI.
Plasmid like pKD46 (heat sensitive) carry Lambda red gene to allow recombination happened.
In GeneSocket, and Support device carry Lambda red gene.
Resistance gene retrieve
pKD46 should be discarded after this step, and a new plasmid like pCP20 carrying recombinase is transformed into cell to retrieve resistance gene.
Since no resistance gene are used in insertion fragment, there is no need to consider gene retrieving. All things can be done in one plasmid no repeatedly selection shall be done!
Scar can’t be avoid via this method of the recombinase site.
Scar can be avoided if necessary, by careful design of the recombination site.
1.2 The workflow of using GeneSocket product. (Fantastic guide)
                 Figure. 1
A prepared cell with the socket in it.
Find parts in registry. Design flanking homologous ends by GS-BOX.
Add sites through overlap PCR.
Make competent cells (Arabinose induction) & do electroporation.
Select through "traffic light" system.
Get into next cycle & insert next sequence. (Arabinose induction)
Get more instructions in Working Process and Solution !!!!!
2 Can be used in different species.
Our GeneSocket idea provides a wide application in all kinds of creatures. Except for prokaryotic cells, it can also outreached to eukaryotic cells. We’ve learned another ways of homologous recombination depending on CRISPR, and it is really possible to build GeneSocket in eukaryotic cells instead of lambda red recombination.
2.1 Prokaryotic cells
As shown before, our GeneSocket can be perfectly used in prokaryotic cells via lambda-red homologous recombination and tyrosine recombinase bistable switch methods. The workflow and software are available in the former section.
2.2 Eukaryotic cells
Obviously, it is excited to make our socket idea apply into eukaryotic cells. There is a hot spot in biomedical field and bioengineering field called mammalian cell engineering. Mammalian cells are the most complicated group of creatures’ cells in the world and their complexity can be hundreds of folded of prokaryotic cells. However, Mammalian cell engineering could have a profound impact on treatment of disease and pharmaceutical industry. For example, production of monoclonal antibodies, interferon and other proteins has played a necessary role in pharmaceutical field, with protein drugs comprising almost 50% of total drug revenues and a higher efficiency of approval than chemical drugs. A few of these proteins are made in bacteria or yeast, but most are produced from mammalian cells because these cells mediate natural mammalian glycosylation, are better at catalyzing the folding of many proteins, and do not produce inflammatory contaminants. (e.g., bacterial cell wall material).
However, out GeneSocket idea can solve this problem in the future. Recently, new tools have emerged that should allow specific integration at desired sites in the genome of mammalian cells. For example, methods based on zinc-finger, TALE, and CRISPR fusions to nucleases can be used to generate double-strand breaks at specific sites in the genome (1). If we use these new technologies into our GeneSocket idea, we can also make another successful product for eukaryotic cells, especially mammalian cells(2,3).
3 MAGE tuning
The breadth of genomic diversity found among organisms in nature allows populations to adapt to diverse environments (4,5). However, genomic diversity is difficult to generate in the laboratory and new phenotypes do not easily arise on practical timescales3. Although in vitro and directed evolution methods (6,7,8,9,10,11) have created genetic variants with usefully altered phenotypes, these methods are limited to laborious and serial manipulation of single genes and are not used for parallel and continuous directed evolution of gene networks or genomes. Here, we describe multiplex automated genome engineering (MAGE) for large-scale programming and evolution of cells. MAGE simultaneously targets many locations on the chromosome for modification in a single cell or across a population of cells, thus producing combinatorial genomic diversity. Because the process is cyclical and scalable, we can construct prototype devices that automate the MAGE technology to facilitate rapid and continuous generation of a diverse set of genetic changes (mismatches, insertions, deletions). (12,13)
Also, the MAGE technology comes from lambda-red recombination. We can combine MAGE and GeneSocket together to tune up the new constructed circuits. (on chromosomes or on plasmids). What’s more, MAGE can do insertions and deletions to multi-location at one time. By using this feature, we can adjust our GeneSocket with little changes to the most adequate state.
References
J. C. Way, J. J. Collins, J. D. Keasling, P. A. Silver, Integrating Biological Redesign: Where Synthetic Biology Came From and Where It Needs to Go. Cell 157, 151 (Mar 27, 2014).
T. Gaj, C. A. Gersbach, C. F. Barbas, 3rd, ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering. Trends in biotechnology 31, 397 (Jul, 2013).
F. A. Ran et al., Genome engineering using the CRISPR-Cas9 system. Nature protocols 8, 2281 (Nov, 2013).
Venter, J. C. et al. Environmental genome shotgun sequencing of the Sargasso Sea. Science 304, 66–74 (2004).
Tringe, S. G. et al. Comparative metagenomics of microbial communities. Science 308, 554–557 (2005).
Elena, S. F. & Lenski, R. E. Evolution experiments with microorganisms: the dynamics and genetic bases of adaptation. Nature Rev. Genet. 4, 457–469 (2003).
Ellington, A. D. & Szostak, J. W. In vitro selection of RNA molecules that bind specific ligands. Nature 346, 818–822 (1990).
Crameri, A., Raillard, S.-A., Bermudez, E., Stemmer, W. P. & C. DNA shuffling of a family of genes from diverse species accelerates directed evolution. Nature 391, 288–291 (1998).
Joo, H., Lin, Z. & Arnold, F. H. Laboratory evolution of peroxide-mediated cytochrome P450 hydroxylation. Nature 399, 670–673 (1999).
Zhang, Y. X. et al. Genome shuffling leads to rapid phenotypic improvement in bacteria. Nature 415, 644–646 (2002).
Pfleger, B. F., Pitera, D. J., Smolke, C. D. & Keasling, J. D. Combinatorial engineering of intergenic regions in operons tunes expression of multiple genes. Nature Biotechnol. 24, 1027–1032 (2006).
Cadwell, R. C. & Joyce, G. F. Randomization of genes by PCR mutagenesis. PCR Methods Appl. 2, 28–33 (1992).
H. H. Wang et al., Programming cells by multiplex genome engineering and accelerated evolution. Nature 460, 894 (Aug 13, 2009)
Previous: Results
Next: GS-BOX
Retrieved from " http://2014.igem.org/Team:ZJU-China/Perspective "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
