"
Page
Discussion
View source
History
teams
Log in
 
Team:ZJU-China/Results
From 2014.igem.org
Home
Team
Team information
Official Profile
Attributions
Project
Introduction
Background
λ-red
Bistable Switch
Gene Socket
Work Processing
Circuit Design
Solution
Modeling
Results
Perspective
GS-BOX
Parts
Lab
Notebook
Protocol
Lab Log
Safety
Achievement
Human Practice
Communication
iGEM
Team Sponsor
Contact us
Team Forum(BBS):
ZJU-China 2014 Team Forum on www.zjubiolab.zju.edu.cn
Post Address:
Biolab Center Room 413, ZJU Zijin'gang Campus,
Yuhang Tang Road No.866, Hangzhou, Zhejiang
Powered by Media Wiki. Zhejiang University.
Email Address:
zjuchina2014 @ 163.com
Contents
1 Circuit Construction with Diverse Methods
2 Validation of Chromosome Recombination with λ-Red Technique
2.1 Growth curves under L-arabinose gradients induction
2.2 Choose Homologous Sites
2.3 Recombination towards HL/HR site pair
3 SINGLE Insertion: Socket Feasibility Testing
4 Downstream expression system
5 AttB/P Flanking Region Inversion and Different RBS Modulation
6 Xis-Mediated attL/R Inversion Assay
Circuit Construction with Diverse Methods
We developed a bacteria platform capable of efficient and theoretically infinite gene insertion. As to demonstrate the feasibility and functionality of our system, we designed 11 different circuits located on chromosome and plasmids respectively to cooperate and implement the brilliant function of "super insertion".
Although many of these DNA elements constituting circuits can be found in the iGEM official registry, DNA element assembly remains a big challenge. In spite of iGEM official recommending assembly method, 3A assembly, the circuit construction still seems quite inconvenient and time-consuming because 3A assembly needs a lot of time to proliferate the ligation fragments into cells and leaves a scar between two ligation fragments.
Figure 1 - Multiple fragment assembly with seamless assembly method. These methods depend on bacteria proliferation. Firstly, add two kinds of homology sequences to fragments, that is, add 15~20bp plasmid homology sequences to fragment A and C, and add 9bp linear DNA homology sequences to each fragments. Construct a plasmid, transform, proliferate and serve as a template to amplify the assembly fragment (A+B+C).
Figure 2 - schematic illustration of the protocol of seamless cloning. Add two homology sites (15~20bp) of the linear plasmid backbone to each end of any fragments by PCR. Then the fragment and the plasmid can form a circular DNA without scar.
Figure 3 - Two linear fragments assembly mediated by seamless cloning. Two rounds of PCR are needed. The first round aims at introduce mutual homology sites of both fragments: 3' end of fragment A with the 9bp 5' end of fragment B; 5' end of fragment B with 9bp 3' end of fragment A. The second round of PCR is followed by incubating into seamless cloning reaction system and lead to a large number of assembly fragment (A+B).
In consideration of efficiency and convenience, we mainly used a no-scar assembly method. Different from the famous assembly method, Gibson assembly, our method for assembling the whole circuit is no-scar recombination, also known as seamless cloning or seamless assembly. The method does not depend on restriction enzyme digestion and ligase activity, and thus it is not restricted to DNA fragment restriction enzyme sites introduction. Based on homologous recombination, certain recombinases function in the in vitro system. What needs to do is to add homology sequences of certain length of the end of the second fragment to that of the first one. Then the recombinase recognizes the homology sites and links all the modified fragments together.
The seamless cloning or assembly method is quite an efficient way to construct a complex circuit. Different from Gibson assembly, the assembly method relies on plasmid backbones and bacteria cloning. Exactly speaking, the recombinase-based assembly method, in fact, is divided into two kinds, each of which consists of different components. The first one is called seamless cloning which is only applicable for inserting fragments to linear plasmid backbone (Fig.2). As Fig.2 shows, seamless cloning requests adding two homology sequences (15-20bp) of the plasmid backbone to each ends of the inserting fragment.
Figure 4 - PCR product after applying seamless cloning to two fragments (60bp, 900bp). PCR with each fragment primer and obtain the combining fragments (960bp).
Figure 5. Multiple fragments assembly PCR results. Three different fragments are involved with respective length (1300bp, 60bp, 1500bp).
Figure 6. Two fragments assembly using over-lap PCR. The assembly fragment of 2100bp is combined with 1200bp and 900bp fragment.
After optimizing the reaction system, we developed a new protocol to use the assembly method in a different way (Fig. 3). That is, we use seamless cloning assembly to ligase two linear fragments by following PCR (Fig. 4). As an example, after adding 9bp of fragment B (60bp) 5' end to 3' end of fragment A (900bp) by PCR, adding 9bp of the 3' end of fragment A to 5' end of fragment B in the same way, and incubating in the seamless cloning reaction system, fragment A + B can be got by PCR using 5'end primer of A and 3' end primer of B. As the figure shows, fragment A+B (960 bp) gets successfully proliferated.
When confronting multiple fragment assembly, we use seamless assembly to integrate these fragments to a plasmid backbone (Fig.1). Proliferate the plasmid in bacteria cells and use it as a template to amplify the targeting sequence by PCR (Fig.5). As an example, after PCR modification of three fragments ( A,1300bp,B,60bp,C,1.5bp ) and incubation in seamless assembly reaction liquid, these three fragments can be inserted into the plasmid in a designed direction (A、B、C). Transform, proliferate bacteria cells, PCR and then we get the assembly fragment (about 2.9kb).
Additional approach is over-lap PCR. In situation where assembly fragment cannot be got via PCR due to some unknown reasons, we would like to choose over-lap PCR method. For instance (Fig.6), Frag A (900bp) and Frag B (1200bp) are two fragments which needs to be assemblied together. After over-lap PCR applied, the linking fragments (about 2100bp) are amplified and located on specific position through gel electrophoresis.
Validation of Chromosome Recombination with λ-Red Technique
Growth curves under L-arabinose gradients induction
In order to test if the basic genome-editing part---λ-Red system works, we designed several different knock-in site pairs on E.coli chromosome and utilized antibiotic resistance genes to target them.
Firstly, from a large number of paper collection, we summarized a λ-Red protocol which says that recombination bacteria cell preparation requires L-arabinose induction, and L-arabinose appears to inhibit cell growth due to cytotoxicity of λ-Red proteins. Therefore, as to learn about cell growth circumstances under L-arabinose induction, and test if λ-Red proteins are expressed on certain levels, we made a growth curve of gradient arabinose induced cell growth as follows (Fig.7). From the curve, we can easily find that with the increase of arabinose induced concentration, cells grow slower and slower, implying that the inhibition effect got strengthened.
Figure 7.The growth curves under L-arabinose gradients induction. Inhibition effect got strengthened with the increase of arabinose induced concentration.
Choose Homologous Sites
Then we sought to find a socket-insertion sites on E.coli chromosome because our “socket.coli” needs a target site to be explored as a knock-in platform, which means all of our chromosome circuits should be inserted into such sites. Confronted with as long as 4.6Mbp E.coli genome, several criteria of our homologous site pairs are supposed to be considered. Firstly ,it is no less than 40bp and more than 70bp, in that too short a site is not sufficient to implementλ-Red-mediated recombination, and too long is not efficient to add these homology sites to each ends of targeting fragments. Secondly, since the targeting fragment is modified via PCR, the secondary structure of both site pairs should not be too awful and most desirably meet the criteria of designing PCR primers. In addition, both sites are not on essential genes. Last but not the least, the length of the region flanked by those two sites should be as suited as possible to insert 1~3 kb targeting fragments, for all of our targeting fragments vary from 1kb to 3kb.
Corresponding to the criteria mentioned before, we chose two different homologous site pairs, both of which lie on the lac operon gene, which does not affect the viability of bacteria cells at all. As to deal with targeting fragments of different length, two site pairs are chosen to be the socket target site. The two pairs are named respectively as HL/HR and LacHL/LacHR. Of the two, HL/HR is selected as a target for antibiotic resistant gene knock-in validation, and LacHL/LacHR is mainly used to knock in the chromosome circuit part.
Recombination towards HL/HR site pair
Through recombination towards HL/HR site pair, we demonstrate the basic knock-in function of λ-Red system. The targeting fragments include two kinds of antibiotic resistance genes, kanamycin-resistant gene and tetracycline-resistant gene. Both antibiotic resistant genes have added HL/HR sites by two rounds of PCR.
The positive recombinational cells grow in the corresponding antibiotic LB plate followed by electrically mediated targeting fragment introduction. Using HL/HR lateral detective primer pairs (Fig.8), negative control gets a fragment of the length of 1527bp; on the contrary,if targeting fragment recombines towards the chromosome successfully, a band as long as 1050 bp will appear. And from the electrophoresis gel image, we succeeded to perform genome editing, or exactly, gene knock-in, in E.coli!
Figure 8.Kanamycin-resistant gene mediated knock-in confirmation. Lane 1~4 represents different positive colonies. Bands of 1050bp appearing shows that recombination occurs. Lane 5 and 6 are negative controls to show the original length (1527bp) between HL/HR.
SINGLE Insertion: Socket Feasibility Testing
Posterior to verification of the function of λ-Red system, what we aim to do is to prove that our socket site can get easily knocked-in. Since the recombinational bacterial, Socket. Coli, was not prepared, socket insertion can not be strictly attested on chromosome. Therefore a surrogate assay was performed to show if our socket circuit can be well knocked-in.
Figure 9. Validation of recombination in Single plasmid. Lane 1 represents the whole Single circuit; Lane 2 represents PCR product with primer A and B. The band of 1000bp prove the recombination occurrence and the band of 300bp is on behalf of the double terminator. Lane is a negative control which only owns double terminators.
For imitating the situation where chromosome recombination occurs, we constructed the designed chromosome socket circuit into a low copy number plasmid, PSB4C5, also named "SINGLE plasmid" by us. To insert fragments into the socket circuit, we co-transform "SINGLE" with λ-Red helper plasmid PKD46 and induce the bacteria cells with arabinose; then introduce kanamycin resistant gene segments added by "A" site and "B" sites in each ends. After recombination, positive colonies are obtained and verified by PCR (Fig. 9). Contrasting with chromosome recombination outcomes, recombination of plasmids seems to occur easily and have higher recombination efficiency due to the higher copy number of plasmids relative to chromosome. In addition, PCR confirmation outcome displays that the recombination is also incomplete possibly because of the same reason.
Downstream expression system
In our original design, if recombination happens, the next step is to express the downstream gene turned off by "double terminator" before. Out of the whole design of our project are two kinds of downstream protein expression: GFP & Int.
In terms of GFP, we played the recombination experiments mentioned before to testify if the green fluorescence is observed. Unfortunately, although we got the positive outcome of the kanamycin-resistant gene recombination, green fluorescence cannot be seen.
As for Int, we also devised a testing experiment to detect the downstream expression of Int after recombination, where three distinct plasmids got involved. These plasmids are "SOCKET", "SET" and "PKD46". As mentioned before, when SOCKET and SET are co-transformed, the bacteria shows green fluorescence; but with successful recombination of SOCKET, downstream Int got turned on, which functions as inverting the promoter towards RFP gene and express RFP gene.
Figure 10. SOCKET insertion by kanamycin-resistant gene. Lane 1~6 are PCR products of positive bacteria picked up on the kanamycin plate. 1000bp fragment serves as verification of recombination; 300bp bands represent double terminators which means recombination is incomplete.
On account of deficiency of the chromosome recombination Socket.coli, we also use a low copy number plasmid PSB4C5 to simulate single copy number situation of chromosome. For the substitution of chromosome for plasmid, we should introduce the chl+ plasmid PSB4C5 to support the SOCKET device that should have been placed into chromosome. As can be seen, without the chromosome recombinational bacteria, we have to devise a rather complex experiment to validate the downstream expression of Int by bringing in SET.
Specifically, the control group is easy to carry out by co-transforming SOCKET and SET; nevertheless, the experimental group needs several rounds of electrically transformation and competent cells preparation. In other words, the first step is to co-transform SOCKET and PKD46 into E.coli and prepare arabinose-induced competent cells. Next, kanamycin-resistant gene is transformed and triggers recombination. With the set of negative control and PCR affirmation (Fig. 10), kanamycin-resistant gene inserted bacteria are sure to be gained. Lastly, the positive colonies are picked up to make competent cells and be transformed by SET. Although PKD46 and SET share the same antibiotic-resistant gene, ampicillin R, temperature sensitivity of PKD46 is used to inhibit the PKD46 proliferation in 45℃ and therefore cells carrying SET get huge growth advantage under LB with ampicillin. In spite of no growth of control group, we still did not get expected fluorescence outcome.
AttB/P Flanking Region Inversion and Different RBS Modulation
The bistable switch plays a core role in our whole device, linking and orchestrating the chromosome circuit and plasmid circuit together. An efficient and convenient Socket bacteria throws out high standard requirements for stability and efficiency of the bistable switch. Only when the bistable switch reaches a highly controllable and reliably level can the Socket bacteria realize time-saving and infinite rounds of gene knock-in. Out of the consideration, we selected Bxb1 system rather than the traditional pre-transcriptional leveled switch.
Figure 11 - Plate colony fluorescence stimulated by ultraviolet light. As negative controls, INT-6N and INT-B0031 show no fluorescence, and SET shows nearly all the green fluorescence. Co-transformation of INT-6N and SET gets almost all the red fluorescent colonies; however transformation with INT-B0031 exhibits partial inversion of fluorescence.
Figure 12 - Fluorescence of bacteria cells involved in the attB/P inversion assay. GFP and RFP are both positive control; SET, INT-6N and INT-B0031 serve as negative control; co-transformation of Int-B0031,SET, and INT-6N, SET are experimental groups which displays distinct inversion rate.
The Bxb1 system brackets two important reombinases: Int & Xis. From functionality perspective, the switch owns two opposing inversion behaviors, one of which is attB/P flanking promoter inversion, and the other is attL/R flanking promoter inversion.
We built an attB/P inversion assay to demonstrate the inversion of the crucial flanking element, the promoter, J23110. During Int expression, Int inverts the fluorescence output from green to red. Initially, we set up a B0031-invovled Int circuit, and observed partially inversion of the promoter (Fig.11,12). However, incomplete inversion of the promoter is not preferable for its insufficient efficiency of inversion. So different strength of RBS are considered to be incorporated to enhance the inversion efficiency (Fig. 13,14).
Figure 13 - Fluorescence results of attB/P inversion assay under fluorescent microscope. GFP and RFP are positive controls; 6N, B0031 and SET are negative controls; cotransformation groups certify the inversion activity of INT and exhibit different efficiency with different RBS.
Figure 14 - Fluorescence results of attB/P inversion assay under fluorescent microscope. GFP and RFP are positive controls; 6N, B0031 and SET are negative controls; cotransformation groups certify the inversion activity of INT and exhibit different efficiency with different RBS.
Fig. 15. Fluorescence results of attL/R inversion assay. INT-6N and RESET are negative controls. Experimental group is dealt with co-transformation of RESET and INT-6N and following 50Mm arabinose induction. Then some cells’ promoter inverts from red fluorescence to green fluorescence. The arrow points towards one of fluorescence inversion cells.
For this purpose, another RBS of higher strengeth, called 6N, is introduced into the circuit. With the increase of RBS strength, inversion rate got obvious promotion, approximately up to 100% (Fig. 13,14). Compared with two different RBS fluorescent images, we can attain expected results and know the different inversion rate of two different RBS-involved inversion. To achieve highly-efficient inversion, there’s no doubt to employ 6N RBS circuit.
We applied a second method to verify the inversion of promoters (Fig. 15), although fluorescence results already provides with solid proof of the high efficiency of attB/P inversion. The gel image exhibits that SET is not a promoter directionality pure plasmid, but a plasmid in which the promoter points towards GFP gene plays dominant roles. Again, it manifests even Int-6N cannot get to 100% inversion rate.
In a word, we provide solid verification of Int-mediated inversion, and modulate RBS strength to get nearly 100% inversion rate!
Xis-Mediated attL/R Inversion Assay
AttB/P flanking region inversion only counts for half of the reversible switch; Xis-mediated inversion is the other important part of the switch. Only when the two parts work, these bistable switch can be easily and efficiently controlled to turn on and off downstream genes.
Figure 16. Fluorescence results of attL/R inversion assay. INT-6N and RESET are negative controls. Experimental group is dealt with co-transformation of RESET and INT-6N and following 50Mm arabinose induction. Then some cells’ promoter inverts from red fluorescence to green fluorescence. The arrow points towards one of fluorescence inversion cells.
In case of the cytotoxicity of Xis and system chaos caused by constitutive expression of Xis, we put Xis CDS downstream of a L-arabinosed promoter, PBad. We constructed another plasmid called RESET and co-transform it with Int-6N to attest the inversion after Xis expresses.
Under the Arabinose concentration of 50mM, partial inversion from RFP to GFP was observed (Fig. 16). Although fluorescence of some cells clearly inverts  from red to green, it is absolutely not enough to realize efficient inversion, as the inversion rate is quite low compared to the rate of INT inversion. Unfortunately, we don’t have sufficient time to tune the Xis expression relative to Int expression as what we do in the RBS modulation experiment to optimize our bistable switch, but with the result of the Xis-mediated inversion, we've already certified the stability and usability of our whole bistable switch!
Previous: Modeling
Next: Perspective
Retrieved from " http://2014.igem.org/Team:ZJU-China/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
