<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Kent_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Kent/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="contentContainer"><TITLE>iGEM Kent 2015</TITLE><HEADER id="top-header"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid" width:=""><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:Kent">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Team </A></LI><LI><A href="https://2015.igem.org/Team:Kent/Team"> Meet the team</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2015.igem.org/Team:Kent/Description"> Description</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Experiments"> Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Results"> Results</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Safety"> Safety</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Achievements"> Achievements</A></LI><LI><A href="https://igem.org/2015_Judging_Form?id=1739"> Judging</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Parts</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Basic_Part"> Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Composite_Part"> Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Part_Collection"> Part Collection</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Policy and Practices</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Practices">Policy and Practices</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Collaborations"> Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Meetups"> Meet ups</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Modeling"> Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Software"> Software</A></LI><LI><A href="https://2015.igem.org/Team:Kent/Notebook"> Notebook</A></LI></DIV></DIV></NAV></HEADER><P>&quot;
</P><DIV style="margin-left:7.5%; margin-right:32.5%;"><H1 align="center"> Project Description</H1></DIV><DIV style="float: right; margin-right: 2.5%; width: 20%;" class="sidecontents"><DIV class="navbar navbar-default " data-spy="affix" style="background-color:#FFF"><A href="#c1">Overview</A><A href="#c2">Amyloid</A><UL><A href="#c3">csgA signal sequence</A><A href="#c4">Sup35-NM</A><A href="#c5">Cytochrome <I>b<SUB>562</SUB></I></A></UL><A href="#c6">Future perspectives</A><A href="#c7">Policy and Practices</A><A href="#ref">References</A></DIV></DIV><DIV style="margin-left:7.5%; margin-right:32.5%;padding-right:5px;"><H2>Overview</H2><P align="justify">Our project aims to produce self-assembling conductive amyloid nanowire by harnessing an endogenous amyloid export system in E.coli. This could be used to replace current nanowire technology that relies on chemical synthesis and non-renewable resources. In our project, we produced a fusion protein capable of producing conductive amyloid nanowire, a BioBrick encoding the csgA signal sequence and Sup35-NM that is capable of forming amyloid, and also improved a Sup35NM BioBrick from the Valencia 2010 iGEM team. We have developed a model which emulates the system of self assembling Sup35-NM monomers through the use of the Monte Carlo Brownian dynamics method. Finally we also evaluated the environmental and ethical impacts of our project by contacting local as well as international Politicians, and by conducting a survey to engage the public.</P><H2>Amyloid</H2><P align="justify">In the past amyloid has been surrounded by a negative connotation due to association with diseases such as Alzheimer’s, type II diabetes, and Parkinson’s. In recent years however, increasing number of functional amyloid have been discovered. Organisms are able to harness functional amyloid in a beneficial manner in order to carry out biological functions, for example biofilm formation in E.coli, regulation of the STOP codon in S.cerevisiae, and coordinating the chemistry of melanin biosynthesis in mammals<SUP><A href="#r1"> [1] </A></SUP>. Amyloid fibres possess desirable properties as a material to work with because they show mechanical strength comparable to steel and stiffness comparable to silk<SUP><A href="#r2"> [2] </A></SUP>, thus making them easy to manipulate but difficult to break down.</P><H3>csgA signal sequence</H3><P align="justify">An example of functional amyloid is the curli export system in E. coli that produce curli fibres which are involved in adhesion to surfaces, cell aggregation, and biofilm formation<SUP><A href="#r1"> [3] </A><A href="#r4"> [4]</A></SUP>. The bipartite csgA signal sequence is a chain of amino acids that targets our protein to the Sec protein export pathway followed by endogenous curli export pathway<SUP><A href="#r5"> [5] </A></SUP> via interaction with the chaperone CsgG. This allows our protein to be exported from the E.coli cell and form a high concentration of monomers in the extracellular medium.</P><H3>Sup35-NM</H3><P align="justify">The protein that we have chosen to use for our nanowire self-assembly is Sup35-NM, derived from the Sup35-NM protein found in yeast. Amyloid formation occurs due to stacking interactions of the N domain<SUP><A href="#r6"> [6] </A><A href="#r7"> [7] </A></SUP> and each individual Sup35-NM is sufficiently small and closely stacked to allow the flow of electrons. The amyloid has desirable characteristics for nanostructures, such as heat resistance at both high and low temperatures ranging from 98℃ to -80℃<SUP><A href="#r8"> [8] </A></SUP>, as well as high mechanical and chemical stability. We have produced two BioBricks that contain Sup35-NM; these include Envirowire (BBa_K1739003), and a BioBrick with the csgA signal sequence so that the Sup35-NM is exported into the extracellular space (BBa_K1739002) 

With our Sup35-NM BioBick constructs, we have improved a previously designed BioBrick (Part:BBa_K401001) from the Valencia 2010 iGEM team that encoded the Sup35 protein from Saccharomyces cerevisiae. The previously designed BioBrick contained two illegal cut within the coding region that  not compatible with most of the bioBrick standards, including RFC10. Our improved BioBrick has been sequence optimised in order to remove cut sites, producing a part compatible with the iGEM part submission standards. Validation of this part used a diagnostic Congo Red plate that demonstrated the presence of amyloid by formation of red colonies.</P><H3>Cytochrome <I>b<SUB>562</SUB></I></H3><P align="justify">To make our amyloid conductive we have selected the electron carrier cytochrome b562 to link to Sup35-NM as a fusion protein. Our protein follows the Sec export pathway before entering the curli export system. This means that our protein will be exported in an unfolded conformation. Type b cytochromes bind heme non-covalently via the two ligands Methionine-7 and Histidine-106<SUP><A href="#r10"> [10] </A></SUP>, thus allowing exogenous addition of heme. To facilitate the flow of electrons to consecutive cytochromes, the heme groups must be within 20Å of each other. Due to the small size and close interaction of Sup35-NM monomers, we predict that the distance between heme groups will range from 4Å to 15Å. 

Work has been carried out harnessing amyloid produced in vitro by a derivative of Sup35-NM with engineered cysteine residues to bind Nanogold particles. This work found that the distance between the Nanogold particles was too large to be conductive and thus a bridging technique was required to reduce of the 3-5nm gap, consequently forcing the diameter of the amyloid to be increased from 9-11nm to 80-200nm<SUP><A href="#r9"> [9] </A></SUP>. Baldwin created a fusion protein of SH3 and cytochrome b562 in vitro, demonstrating the potential of self-assembling polypeptide fusion proteins in future applications<SUP><A href="#r9"> [11] </A></SUP>.

Here we have produced a fusion protein that contains the amyloid forming protein Sup35-NM and cytochrome b562. Functional validation has been achieved using atomic force microscopy and conductivity testing.</P><H2>Future perspectives</H2><P align="justify">We believe that our product could be used either to produce nanowires that can be purified from the E.coli cells for electronic devises or could be modified into a fuel cell that produces electricity and nanowire in the future.

The amyloid nanowire produced from our BioBrick in the VS45 strain of E.coli produces a high concentration of amyloid nanowires that are free from the cell and could be purified for use in nano-circuitry. Some of the advantages of using amyloid nanowires include high mechanical strength, thermal, and chemical stability, as well as being environmentally friendly by greatly reducing carbon emissions from production. Complex circuit diagrams can be produces from Sup35-NM amyloid fibres using techniques such as lithography, growth in flows or magnetic field gradients, alignment by electrical fields, active patterning with optical tweezers, dielectrophoresis and 3D patterning using hydrogels or microfluidic channels<SUP><A href="#r7"> [9] </A></SUP>. This will allow the generation of miniature circuit diagrams that could pave the way to advancements in nanoscale communications. The high thermal stability is a favourable characteristic because circuit boards can run up to a temperature of 70°C and silver nanowires with a diameter of 5nm disintegrate into individual nanoclusters at a temperature of 27°C<SUP><A href="#r10 "> [10] </A></SUP>. 

Another branch that this project could take would be to engineer a bacterial cell that transports some electrons from the electron transport chain (ETC) into the amyloid-nanowires. This would need to be done using a strain other than VS45 so that the amyloid fibres would attach to the outer surface of the bacterial cell. As a consequence this would create a product that generates its own electricity and transports it along nanowires produced by the same cell; resulting in a living, self-assembling, energy producing circuit board that could be used in a range of consumer products.</P><H2> Policy and Practices </H2><P align="justify"> Amyloid is well known to be associated with pathology, for example Aβ peptide in Alzheimer’s disease. Therefore, when designing the biobricks, we had several constraints to consider, namely the safety of our future product, and its public reception. We designed our policy and practice approach to integrate our engagement with policy-making and public opinion on the use of such biomaterials so that we could incorporate this into the design of our biobricks. To tackle these constraints, we designed our project to be safe by using the protein sup35-NM, which comes from S. cerevisiae (baker’s yeast) to form our amyloid-based nanowires. Our engagement with local and international politicians, and the general public indicated favourable outlook towards our design. This innovative approach to <A href="https://2015.igem.org/Team:Kent/Practices"> policy and practice</A> gave us insight into the lack of communication between science and policy and that this needs to be addressed.</P><A class="anchor" id="top" name="ref"><H3> References </H3></A>[1] Fowler, D., Koulov, A., Balch, W. and Kelly, J. (2007). Functional amyloid – from bacteria to humans. Trends in Biochemical Sciences, 32(5), pp.217-224.
[2] Smith, J., Knowles, T., Dobson, C., MacPhee, C. and Welland, M. (2006). Characterization of the nanoscale properties of individual amyloid fibrils. Proceedings of the National Academy of Sciences, 103(43), pp.15806-15811.
[3]Goyal, P., Krasteva, P., Van Gerven, N., Gubellini, F., Van den Broeck, I., Troupiotis-Tsaïlaki, A., Jonckheere, W., Péhau-Arnaudet, G., Pinkner, J., Chapman, M., Hultgren, S., Howorka, S., Fronzes, R. and Remaut, H. (2014). Structural and mechanistic insights into the bacterial amyloid secretion channel CsgG. Nature, 516(7530), pp.250-253.
p[4]Chapman, M., Robinson, L., Pinkner, J., Roth, R., Heuser, J., Hammar, M., Normark, S. and Hultgren, S. (2002). Role of Escherichia coli Curli Operons in Directing Amyloid Fiber Formation. Science, 295(5556), pp.851-855.
[5] Sivanathan, V. and Hochschild, A. (2012). Generating extracellular amyloid aggregates using E. coli cells. Genes &amp; Development, 26(23), pp.2659-2667
[6] Sivanathan, V. and Hochschild, A. (2013). A bacterial export system for generating extracellular amyloid aggregates. Nat Protoc, 8(7), pp.1381-1390.
[7] Frederick, K., Debelouchina, G., Kayatekin, C., Dorminy, T., Jacavone, A., Griffin, R. and Lindquist, S. (2014). Distinct Prion Strains Are Defined by Amyloid Core Structure and Chaperone Binding Site Dynamics. Chemistry &amp; Biology, 21(2), pp.295-305.
[8] Glover, J., Kowal, A., Schirmer, E., Patino, M., Liu, J. and Lindquist, S. (1997). Self-Seeded Fibers Formed by Sup35, the Protein Determinant of [PSI+], a Heritable Prion-like Factor of S. cerevisiae. Cell, 89(5), pp.811-819.
[9] Scheibel, T., Parthasarathy, R., Sawicki, G., Lin, X., Jaeger, H. and Lindquist, S. (2003). Conducting nanowires built by controlled self-assembly of amyloid fibers and selective metal deposition. Proceedings of the National Academy of Sciences, 100(8), pp.4527-4532.
[10] Xavier, A., Czerwinski, E., Bethge, P. and Mathews, F. (1978). Identification of the haem ligands of cytochrome b562 by X-ray and NMR methods. Nature, 275(5677), pp.245-247.
[11] Baldwin, A., Bader, R., Christodoulou, J., MacPhee, C., Dobson, C. and Barker, P. (2006). Cytochrome Display on Amyloid Fibrils. J. Am. Chem. Soc., 128(7), pp.2162-2163.
[12] Gordon, E., Karabulin, A., Matyushenko, V. and Khodos, I. (2015). Experimental Study of Thermal Stability of Thin Nanowires. J. Phys. Chem. A, 119(11), pp.2490-2501.
</DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>