Team:SYSU-Software/Validation
SYSU-SOFTWARE IGEM 2015
HOME
PROJECT
DESCRIPTION
DESIGN
FEATURES
TECHNIQUES
VALIDATION & TESTING
TUTORIALS
FUTURE WORK
MODELING
OVERVIEW
EXAMPLE
ATTACHMENT
HUMAN PRACTICES
OVERVIEW
SCIENTIFIC & SOCIETAL THINKING
PROJECT ASSESSMENT
PUBLIC ENGAGEMENT
COLLABORATIONS
SAFETY
ATTRIBUTIONS
INTERLAB
MEDALS
TEAM
MEMBERS
INSTRUCTORS & ADVISORS
NOTEBOOK
Wet-lab Validation, Dry-lab Testing & User Studies
Wet-lab Validation
Overview
After we finished the software, we decided to carry out an alpha test which involves the main functions of our software. Therefore we designed a circuit and build it up in a vector with the help of our software. Expression data are measured and can be used to validate and calibrate the software. Here we show the whole detailed process of the experiment.
Motivation
We would like to build a circuit that turns yellow under exposure of sunlight. With this circuit, bacteria can warn people of strong sunlight and help prevent skin cancer. Therefore we picked a UV sensor and a toggle switch that gives the bacteria two states of different color. If the circuit works, the expected result would be green in normal condition, and yellow under sunlight.
Design our circuit
Search devices for our design
Edition of our circuit
The final design and our plasmids
Plasmid 1
Plasmid 2
Plasmid 3
Manage our experiments
Plan our experiment
We used our software to plan our experiment and scheduled our time very well.
Result
Confirmation
We built applicable constructs using standard biobrick assembly and we introduced the three constructs into Escherichia coli strain DH5α cells. Photo.1, photo.2 shows the confirmation result of the constructs by conducting PCR. Further confirmations were finished by sequencing.
photo 1 20150827 ciy1-4 RBS+UVR8-tetR-Ter1-2 RBS+tetG 02 ciy
photo 2 qie p-r-t rbs
Fluorescence and OD600 data
Simulation
The source of simulation result was from the models in our software. You can read the detail description and analysis in the modeling page
Experimental data
After the sequencing confirmation, we collected the data every hour for ten hours and the fluorescence and OD600 of the liquid cultures were measured using a plate reader. For we did not use the sunlight sensor as our input source, we used tetracycline which inhibits TetR repression at the beginning. Chart.1, Chart.2 and Figure.1 Show the fluorescence and OD600 data. We can notice that the tendency of chart.1 and chart.2 is similar and fit the tendency of a normal E.coli.
Discussion
Why we did not use the sunlight sensor as our input source anymore?
Before further experiment, we needed to ensure all the parts that we would use were in order. Therefore, we conducted a PCR for all the parts that we used after the propagation of the plasmid which are in 2015 distribution kit by transforming them into E.coli. Besides, we sent out samples of all plasmids for the sequencing. Afterwards, we performed a sequence alignment between the sequencing result and the sequence which is provided by the registry. However, we only found one third of the sequence matched the official sequence in the UVR8-TetR sequencing result. Therefore, we decided to change an input source.
How did our circuit work without the sunlight sensor?
We decided to use the toggle switch by using tetracycline as an input. In the toggle switch, tetR protein represses promote tet, and tetracycline inhibits this repression so that promoter tet works. With CI protein inhibiting pcI and tetR protein being unable to inhibit promoter tet, the toggle switch is switched to YFP state. As our data above shows, yellow fluorescence goes up.
Doesn't tetracycline kill your bacteria?
To prevent this, the promoter tet-RBS-CI-RBS-YFP-terminator circuit is constructed in tetracycline resistance plasmid backbone.
How to explain the difference between simulation and experiment data?
We can see fluctuation in the data of the 2nd, 3rd and 4th hour. This may be caused by random error of the plate reader. The data curve went more steadily from 5th to 8th hour than the simulate one. That's because we only used one tube of culture for measurement. Every time we took out 200μ culture for measurement, some of the bacteria were lost, and the growth of the bacterial was also paused for a moment due to the low temperature. This influence is especially obvious in the first few hours, but the two curves would finally be closer to each other. From 8th to 10th hour, the data curve rose much more rapidly because this is the exponential growth phase of E.coli. In this period, the lost of bacteria caused by measurement can be neglected. As there were fewer bacterium in the earlier period, the resources and space made it possible for bacterium to grow more rapidly.
Dry-lab Testing
We did some tests, including installability test and running test, on Windows and MacOS X, to supplement the installability test on Linux that have been kindly done by judges.
MacOS X
1. Environment:                       MacBook Air (13-inch, Early 2015) with OS X Yosemite (Version 10.10.4)                       Safari (Version 8.0.7 (10600.7.12))
2. Procedure:
The compression package is about 142 MB on the disk, and when the package is uncompressed, all the files are 415 MB on the disk
Then begins the setup process. Double click the file “setup.command”, and the terminal opens. The software automatically configure the environment needed to run CORE.
After about 4 minutes (depends on the Internet), the process successfully completed (if the connection to internet.
Then double click the file “runserver.command”, and after a few seconds, the process completed.
Then open Safari, and navigate to “localhost:8888”. Now comes the CORE. Run smoothly.
User Studies
As good practices in software engineering, user studies can help us do some research on how users interact with our project, to what extent has our software met the needs of users, and what can be improved to better meet the needs of users.
Due to limited time period for software development, we could only do user studies covering only a limited number of people. It is a great honor for us to invite Yu Zhou (team leader of 2015 NJU-China), Jiyong Ma (bioinformatics graduates in Ren’s lab), Jinyu Li (team member of 2013 and 2014 SYSU-Software, and now PhD at Yale University), and Haoquan Zhao (team member of 2013 SYSU-Software, and now Master at Harvard University) to help us find out the user experience of CORE.
Our user studies invited them to try out our software (with the help of the easy user guide or some guidance given by us). Then give some feedbacks on the three modules (i.e., CORE Bank, CORE Design, and Co-development) of CORE.
The reasons why we chose these four persons to do our user studies are: first, we should choose participants that can cover different background, say, wet-lab background, to ensure that we can study how CORE could meet the needs of individuals of different background, not only from wet-lab; second, time is limited and we can only do user studies of a limited number of people who are also familiar with us or at the nearby.
Yu Zhou, team leader of 2015 NJU-China
Background Information: 2015 NJU-China is doing a project on devising therapies that treat drug addiction. Yu Zhou is a graduate student in  the team.
Some Feedbacks
“We’re doing the project on si-RNA; however, in CORE Design there are no parts on si-RNA. To make CORE more universal, is it necessary to add si-RNA as biological parts so that the database of CORE is more complete?”
(In fact, it is impossible for CORE to contain all the biological parts in the nature, whether they are in Registry or not. However, CORE can add new parts and define the type of your new parts, so it is possible for users of CORE to extend the parts in the database of CORE.)
“Co-development is good! We can help each other on this platform.”
“For CORE Bank, it should consider the situations in which teams’ projects contain no genetic circuits. Say, how can our project be stored on this repository?”
(It is a good suggestion, and we should work harder on these suggestions)
Inspirations
For team that does not take genetic circuit design as their project, what they feel most useful in CORE might be the Co-development module. It is a platform designed for not only collaborative construction of genetic design but also question and answer. If they have any problem in their project, they can feel free to ask questions and get answers.
Although most of the teams’ projects are devoted to genetic circuit design, we should take more into considerations; for example, not all teams devote their project to circuit design. CORE Bank is aimed specifically at hosting the genetic design of teams; Maybe, there should be a repository for other kinds of project?
Jiyong Ma, a graduate student of bioinformatics in Ren’s lab
Background Information: Jiyong Ma is a graduate student of bioinformatics.
Some Feedbacks
“In CORE Design the safety level of a circuit can be seen, it’s considerate. However, when searching for a genetic circuit, the results should be able to be filtered by safety level.”
(It is a good suggestion!)
“CORE Bank should provide the chance for users to edit the information on the genetic design. However, I like CORE Bank, and it just like GitHub!”
(It is also a good suggestion! However, just like the wiki of one team should not be edited by other team, the record of one genetic design should be original. In CORE Bank, the record of one genetic circuit can be retrieved by other users, and after modifications and improvements, the users can upload another new record.)
“Q & A (in Co-development) can have different topics for users to discuss. Anyway, it’s good to provide a platform for communication in the community.”
Inspirations
For a dry-lab practitioner (a bioinformatician), Ma is more interested in the “Hub”. The idea of adopting the features of “GitHub” and applying them to Synbio is a try. To stand the test of time, the “Hub” should make collaboration and open design a routine to synthetic biologists and iGEMers. Ma is a student in bioinformatics and seldom constructs genetic circuits in vitro (wet-lab practices) or in silico, but CORE can to some extent help him with his research: maybe he could find some research topics in the Co-development that he can use his knowledge of bioinformatics to solve, and a collaboration result!
Jinyu Li, PhD student at Yale
Background Information: Jinyu Li is a PhD student at Yale University. She is a biomedical student and also a wet-lab practitioner.
Some Feedbacks
“I like the protocol management tool! The plasmid is beautiful! What a surprised that it can directly assist the whole process of the experiment. But some more details: make an alarming sound when the time is up?”
(Interesting, but…)
“Q & A. I know how to use it!”
(But it is not only for Q & A.)
“CORE Bank. I can more easily find a genetic circuit.”
Inspirations
For a wet-lab practitioner, he or she would find CORE Design most useful. CORE Design provide a more powerful protocol management and experiment scheduling functions. In addition, he or she might think that a Q & A platform is helpful when they encounter troubles in the experiment. In general, wet-lab practitioners deal with less complex circuits, and they might prefer to see the Co-development module as a platform for Q & A. Well. It is fine to ask for help in the community via this platform, and get, give and share your answers with anyone else in the community.
Haoquan Zhao, Master student at Harvard
Background Information: Haoquan Zhao is a Master student at Harvard University. He is now a student on biostatistics.
Some Feedbacks
“Standardized information storage is a good idea. But more data are needed.”
(We try hard to include more data in the database.)
“Crowdsourcing is a fashionable concept, and it might have an impact. Do you think of using crowdsourcing to do some data analysis in CORE?”
(Good idea!)
“CORE Design. The plas
mid is beautiful.”
Inspirations
As a student on biostatistics, Zhao tends to focus on how data are managed and organized. Biological data are growing in quantity and in quality. CORE provides tools that can help navigate data and information in Registry, say, looking for the sequence of a part, or find the relevant information on that part. CORE Bank and Co-development, collectively called the “Synbio Hub”, is a standardized repositories of genetic design. This Hub provides a new method to organize the genetic design of iGEM teams and synthetic biologists alike.
Email: sysusoftware@126.com
Address: 135# Xingang Rd(W.), Sun Yat-sen University, Guangzhou, China
