<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Johns_Hopkins_Research"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Johns_Hopkins/Research">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Johns_Hopkins/Research&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Research&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Research&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Johns_Hopkins/Research" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Johns Hopkins/Research</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="book"><FONT class="ir"><H2><SPAN class="mw-headline" id="Yeast_Sex_Detector"> Yeast Sex Detector </SPAN></H2><P>As our inaugural project, we set out to simplify one of the somewhat belaboring tasks in the field of molecular biology: S. cerevisiae mating type elucidation. Baker’s yeast, S. cerevisiae, has become an invaluable eukaryotic model for molecular biology research for many reasons. One, it utilizes many proteins and biological pathways homologous to those in humans. Consequently, investigating these molecules and systems in yeast has provided unparalleled insight into the genetics of humans.  Two, because yeast is unicellular and therefore grows fast, it can be studied more readily than many other organisms, especially higher eukaryotes.  Another useful characteristic of yeast is its ability to exist in populations of different ploidy, either diploid or haploid. The process in which ploidy arises is governed by the yeast mating pathway, which is well studied.
</P><P> A haploid yeast cell is either mating type ‘a’ (MATa) or mating type ‘α’ (MATα). In the elucidation of biochemical and genetic processes in yeast, many times it is necessary to initiate sporulation of diploid yeast cells. After sporulation occurs, there are four haploid cells; two MATa and two MAT α. To continue analysis, differentiating between the haploid cells is often crucial, and the necessary assay to do this can take 2 to 3 days. We propose to cut this time to a matter of seconds by constructing a plasmid with fluorescent markers that preferentially activate depending on the mating type. If the cell is MATa, one fluorescent protein is produced. If the cell is MATα, another fluorescent protein is produced. Simply shining a UV lamp over the cells will reveal the mating type, allowing for the cells to be easily separated.</P><H3><SPAN class="mw-headline" id="Theory"> Theory </SPAN></H3><P>Depending on the specific regulatory factors being produced, a cell will exhibit a given sex, either MATa or MAT α. (Herskowitz 750) The MATa-associated-products are controlled by α1 and α2 transcriptional regulators.  The α1 factor promotes the transcription of MATα-associated-proteins and the α2 factor inhibits the production of MATa-associated-proteins.
The MATa cells produce no MATα-associated-proteins due to a lack of α1 to initiate the production of those MATα-associated-proteins. (Sprague 959) It produces only MATa-associated-proteins, partially due to the lack of inhibition by the absense of α2. The a1 factor does not influence the production of MATa-associated-proteins, even though it is present in MATa cells.
The a/α diploid cells have both a1 factor and α2 factor. These two factors physically work together to inhibit certain genes that confers on the diploid cell the ability to only produce diploid-associated-proteins.
The design of our construct would be such that a MATα cell would produce only one fluorescent protein, and in the case of the prototype diagram of our construct, green fluorescent protein (GFP). A MATa cell would produce only RFP. As such we would place a MATα specific promoter in front of the GFP gene, and a MATa specific promoter in front of the RFP gene.
Time permitting we also hope to engage a several sub-projects. First would be to utilize a third fluorescent protein that activates only in diploids. Furthermore, we hope to add additional control to the system by allowing the genetist the option of completely turning off the fluorescent proteins, so that they do not interfere with his/her own fluorescent experiments.

A third sub-project would be to address the life span for an expressed fluorescent protein. A particular protein, in general, is governed by the N-terminus rule pathway. This pathway involves the amino acid sequence associated with the N-terminus of the selected protein. Depending on the final amino acids of the sequence, the protein will not be labeled for degradation for a given amount of time (Varshavksy 15). This terminal sequence will thus determine the half-life of the protein, or basically the time in which the protein is functionally present in the cell. So by manipulating this sequence we will be able to influence how long the desired protein will exist. The control of the half-life of the fluorescence proteins used in the sex detector construct is of crucial importance.  If the proteins remain functional for too long, it may not be possible to accurately determine if the haploid cells have fused back into diploids. Also, if the protein's lifespan is so short that the signal is very weak, than the detector is impractical. By optimizing this sequence we will be able to engineer the perfect half-life of the fluorescent proteins involved in the detector, and thus increase its efficacy. A fourth sub-project, similar to this issue of half-life control, is the control of specific strains of yeast called HO strains. These strains contain an endonuclease that allows them to alter genomic expression of mating regulatory factors after dividing in the haploid state (allows them to switch mating types). The problem that exists in this case is that, after changing mating type, the heterogeneous population of a and α cells can now mate, thus reducing haploid numbers. We propose to manipulate these regulatory factors, mentioned before, to control the mating, and thus the ploidy, of a given population of HO strains. The most efficient way to do this would be to actually prevent the haploid cell from sensing each other, by altering their ability to produce the functional mating pheromones that allow then to sense the reciprocate sex. </P><H3><SPAN class="mw-headline" id="Goals"> Goals </SPAN></H3>The expected outcome of the main project and the four sub-projects listed above is a genetic construct that would allow for efficient sex and ploidy detection of several yeast strains. However, the main goal is to develop a strong and robust team of budding yeast geneticists (pun intended), and provide them with the experience to make the first Johns Hopkins University iGEM team a productive and respected addition to the competition.  This would not only be beneficial to the students hoping to learn from the experience, but it would also be beneficial to the JHU community as a whole by securing a place in the groundbreaking field of synthetic biology, on an international scale.<DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Johns_Hopkins/Research">http://2008.igem.org/Team:Johns_Hopkins/Research</A>&quot;</DIV></FONT></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Johns_Hopkins/Research" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Johns_Hopkins/Research" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Research&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Research&amp;oldid=25935" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>