"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins/Project
From 2008.igem.org
Home
The Team
The Project
Protocols/Papers
Parts Submitted to the Registry
Modeling
Notebook
Project
Sex Detector
As our inaugural project, we set out to simplify one of the somewhat belaboring tasks in the field of molecular biology: S. cerevisiae mating type elucidation. Baker’s yeast, S. cerevisiae, has become an invaluable eukaryotic model for molecular biology research for many reasons. One of the main reasons is its utilization of proteins homologous to those found within humans, as well as many of the same biochemical pathways. Investigating the biochemical systems in yeast has provided insight on various genetic diseases found in humans.  Also, because they are unicellular and therefore grow fast, they can be studied more readily than many other cell lines, especially those of higher eukaryotes.  Another useful characteristic of yeast is its ability to exist in populations of different ploidy, either diploid or haploid. The process in which ploidy arises is governed by the yeast mating pathway, and is well studied. A haploid yeast cell is either mating type ‘a’ (MATa) or mating type ‘α’(MAT α). In the elucidation of biochemical and genetic processes in yeast, many times it is crucial to initiate sporulation of diploid yeast cells. After sporulation occurs, there are four haploid cells; two MATa and two MAT α. To continue analysis, usually differentiation between these cells is crucial, and this process can take from 2 to 3 days. We propose to cut this time by creating a plasmid containing fluorescent proteins that would allow for visual determination of S. cerevisiae mating type.  To do so we would utilize the existing regulator proteins control of the expression of specific fluorescent proteins
Method
Depending on the specific regulators being produced, a cell will exhibit a given sex, either MATa or MAT α. (Herskowitz 750) The MATa associated products are controlled by, α1 and α2 regulators.  The α1 regulator promotes the transcription of MAT α associated products and the α2 regulator inhibits the production of MATa associated factors.  In absence of MAT α regulators, the MATa cells produce only MATa associated factors, without MAT α associated factors due to a lack of inhibition by α2, and a lack of α1 to initiate the production of MAT α associated factors. (Sprague 959) The a1 regulator that is produce by MATa cells is only utilized when in combination with a2 to inhibit specific regions, associated with a MATa/ α diploids.  The design of the construct would be such that a MAT α cell would produce only one fluorescent protein, and in the case of the prototype diagram of our construct, green fluorescent protein (GFP). A MATa cell would produce both fluorescent proteins, GFP and red fluorescent protein (RFP), thus yielding a different color. Also a diploid cell, would be detectable as well due to lack of color, due to regions before each fluorescent protein open reading frame that are inhibited by the diploid a1-α2 combination factor. In addition to these control regions, before both fluorescent protein open reading frames there would also be a promoter region, foreign to yeast, most likely from bacteria. This would allow the investigator to be in control in the production of these sex-dependent fluorescent proteins, such that if protein fluorescence is used to study another aspect of yeast physiology, the sex dependent florescence would be inactivated. By utilizing the specificity in genome control between the sexes of the haploid yeast cells, along with a specific region of control in diploid cells, the creation of a reliable, novel, and practical detector would be achievable.
Experimentation
Another issue associated with this project would be the life span for a fluorescent protein we express in the plasmid. A particular protein, in general, is governed by the N-terminus rule pathway. This pathway involves the amino acid sequence associated with the N-terminal of the selected protein. Depending on the final amino acids of the sequence, the protein will not be labeled for degradation for a given amount of time (Varshavksy 15). This ending sequence will thus determine the half-life of the protein, or basically the time in which the protein is functionally present in the cell. So by manipulating this sequence we will be able to determine for how long the desired protein will exist. The control of the half-life of the fluorescence proteins used in the sex detector construct is a crucial issue.  If the proteins remain functional for too long, it may not be possible to accurately determine if they have fused to form a diploid. Also, if the proteins lifespan is so short that the signal is very weak, than the detector is less practical. By optimizing this sequence we will be able to engineer the perfect half-life of the fluorescent proteins involved in the detector, and thus increase its efficacy.
Similar to this issue of control is the issue of a control of specific strains of yeast called HO strains. These strains contain an endonuclease that allows them to alter genomic expression of mating regulatory factors after dividing in the haploid state. The problem that exists in this case is that, after changing mating type, the heterogeneous population of a and α cells can now mate, thus reducing haploid numbers. We also propose to manipulate these regulatory factors, mentioned before, to control the mating, and thus the ploidy, of a given population of HO strains. The most efficient way to do this would be to actually prevent the haploid cell from sensing each other, by altering their ability to produce the functional mating pheromones that allow then to sense the reciprocate sex.
The expected outcome of the three sub-projects listed above is a genetic construct that would allow for the efficient sex and ploidy detection of several yeast strains. The outcome of this work, however, would allow the first Johns Hopkins University iGEM team to compete in the 2008 iGEM competition.  This would not only be beneficial to the JHU iGEM team, but also be beneficial to the JHU community as a whole by securing a place in the groundbreaking field of synthetic biology, on an international scale.
Retrieved from " http://2008.igem.org/Team:Johns_Hopkins/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
