<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_LMU-TUM_Munich_Proteins skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:LMU-TUM Munich/Proteins</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="row"><DIV class="col-sm-3 col-xs-8 fixed-sm z-1"><DIV class="sidebar-nav"><DIV class="navbar" role="navigation"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".sidebar-navbar-collapse"><SPAN class="sr-only">Toggle menu</SPAN></BUTTON><SPAN class="visible-xs navbar-brand">Menu</SPAN></DIV><DIV class="navbar-collapse collapse sidebar-navbar-collapse"><UL id="main-menu" class="nav navbar-nav proteins"><LI class="menu_item home"><A href="https://2016.igem.org/Team:LMU-TUM_Munich"><SPAN class="visible-xs-block">Home</SPAN></A></LI><LI class="dropdown menu_item team"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI class="members"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Team">Members</A></LI><LI class="collaborations"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Collaborations">Collaborations</A></LI><LI class="sponsors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Sponsors">Sponsors</A></LI><LI class="press"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Press">Press</A></LI><LI class="gllry"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Gallery">Gallery</A></LI></UL><LI class="dropdown menu_item project"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Project </A></LI><LI class="description"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Description">Description</A></LI><LI class="design"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Design">Design</A></LI><LI class="proteins"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proteins">Proteins</A></LI><LI class="receptors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Receptors">Receptors</A></LI><LI class="linker"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Linkerchemistry">Linker Chemistry</A></LI><LI class="poly"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proof">Polymerization</A></LI><LI class="bioprinting"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Demonstrate">Bioprinting</A></LI><LI class="hypoxia"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hypoxia">Hypoxia</A></LI><LI class="medical"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Medical">Medical Application</A></LI><LI class="interlab"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/InterLab">InterLab</A></LI><LI class="software"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Software">Software</A></LI><LI class="dropdown menu_item notebook"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Notebook </A></LI><LI class="labjournal"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Labjournal">Labjournal</A></LI><LI class="materials"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Materials">Materials</A></LI><LI class="methods"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Methods">Methods</A></LI><LI class="howtos"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Howtos">How Tos</A></LI><LI class="dropdown menu_item parts"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Parts </A></LI><LI class="parts-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Parts">Parts</A></LI><LI class="basic-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Basic_Part">Basic Parts</A></LI><LI class="composite-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Composite_Part">Composite Parts</A></LI><LI class="part-collection"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Part_Collection">Part Collection</A></LI><LI class="menu_item hardware"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hardware">Hardware</A></LI><LI class="menu_item model"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Model">Modeling</A></LI><LI class="menu_item safety"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Safety">Safety</A></LI><LI class="dropdown menu_item human_practices dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Human Practices </A></LI><LI class="human_practices-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Human_Practices">Human Practices</A></LI><LI class="silver"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Silver">Silver</A></LI><LI class="gold"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Gold">Gold</A></LI><LI class="integrated-practices"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Integrated_Practices">Integrated Practices</A></LI><LI class="engagement"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Engagement">Engagement</A></LI><LI class="dropdown menu_item entrepreneur dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Entrepreneurship</A></LI><LI class="entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Entrepreneurship">Entrepreneurship</A></LI><LI class="supporting-entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Supporting_Entrepreneurship">Supporting Entrepreneurship</A></LI><LI class="menu_item attributions"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV class="col-sm-9 col-sm-offset-3"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Interaction_of_biotin_and_streptavidin_leads_to_a_pseudopolymerization_of_cells"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Interaction of biotin and streptavidin leads to a pseudopolymerization of cells</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Streptavidin_and_Biotin:_The_strongest_non-covalent_affinity_in_nature"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Streptavidin and Biotin: The strongest non-covalent affinity in nature</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Streptavidin_variants_and_homologs"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Streptavidin variants and homologs</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Traptavidin:_increased_mechanical_strength_of_biotin_binding"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Traptavidin: increased mechanical strength of biotin binding</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Strep-Tactin:_a_Strep-tag_binding_protein"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Strep-Tactin: a <I>Strep</I>-tag binding protein</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Enhanced_Monomeric_Avidin_.28eMA:_engineered_for_applications_like_molecular_labelling"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Enhanced Monomeric Avidin (eMA: engineered for applications like molecular labelling</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Expression_and_purification_of_streptavidin"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Expression and purification of streptavidin</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Streptavidin_Fermentation"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Streptavidin Fermentation</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Biotinylated_PAS-lysine:_A_flexible_linker"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Biotinylated PAS-lysine: A flexible linker</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#eGFP_and_BSA:_alternative_linker_molecules"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">eGFP and BSA: alternative linker molecules</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Further_characterisation_of_streptavidin_variants"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Further characterisation of streptavidin variants</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Overview_of_purified_proteins"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Overview of purified proteins</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Quality_control_using_ESI-TOF_mass_spectrometry"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Quality control using ESI-TOF mass spectrometry</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Quality_control_using_SDS-PAGE_and_isoelectric_focusing"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">Quality control using SDS-PAGE and isoelectric focusing</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Affinity_determination_using_fluorescence_titration"><SPAN class="tocnumber">8.4</SPAN><SPAN class="toctext">Affinity determination using fluorescence titration</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Analysis_of_secondary_structure_and_thermal_stability"><SPAN class="tocnumber">8.5</SPAN><SPAN class="toctext">Analysis of secondary structure and thermal stability</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#References"><SPAN class="tocnumber">8.6</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Interaction_of_biotin_and_streptavidin_leads_to_a_pseudopolymerization_of_cells">Interaction of biotin and streptavidin leads to a pseudopolymerization of cells</SPAN></H1><DIV class="white-box"><DIV class="imagelink float-right"><A href="/wiki/images/4/45/Muc16_Sticker_proteins_001.png" class="internal" title="Muc16 Sticker proteins 001.png">Media:Muc16_Sticker_proteins_001.png</A></DIV><P>For cross-linking cells upon printing using our biotINK-approach, we utilize the extraordinary strength of the biotin-streptavidin interaction. The cells express a receptor which allows them to present a biotin molecule or a streptavidin variant on their cell surface (see <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Localization">Receptor</A>).
The tetrameric streptavidin protein in the printing solution is able to cross-link the biotinylated cells and thus make them form a <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Demonstrate">stable network upon printing</A>. In order to adjust intercellular distances, a biotinylated linker peptide containing regularly spaced biotinylated lysine residues is used.</P></DIV><H1><SPAN class="mw-headline" id="Streptavidin_and_Biotin:_The_strongest_non-covalent_affinity_in_nature">Streptavidin and Biotin: The strongest non-covalent affinity in nature</SPAN></H1><DIV class="imagelink float-right"><A href="/wiki/images/d/dc/Muc16_SA1.png" class="internal" title="Muc16 SA1.png">Media:Muc16_SA1.png</A><DIV class="caption"><B>Figure 1:</B> Crystal structure of a  Streptavidin monomer binding a biotin molecule at molecular resolution.</DIV></DIV><DIV class="white-box"><P>Streptavidin is a tetrameric protein with a molecular weight of 52.8 kDa<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP> and was originally isolated from <I>Streptomyces avidinii</I>. Each subunit is able to bind a single biotin molecule. This specific, non-covalent bond with a unique femtomolar dissociation constant (K<SUB>D</SUB> = 10<SUP>-15</SUP> M<SUP>-1</SUP>) is one of the strongest known biological interactions.<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP> Antibodies, in comparison, have dissociation constants in the range of 10<SUP>-7</SUP> – 10<SUP>-11</SUP> M<SUP>-1</SUP> - being less affine by a factor of a million.
</P><P>One subunit of Streptavidin is organized as an eight stranded antiparallel beta sheet of coiled polypeptide chains which form a hydrogen bonded barrel with extended hydrogen loops<SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP> (Figure 1). 
The biotin binding pocket primarily consists of aromatic and polar side chains, which interact with the hetero atoms of biotin.
Due to the high specificity and affinity of the interaction, Streptavidin is often used for protein-purification via affinity chromatography and as a detection and coupling reagent in daily lab routine.<SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP></P><DIV class="imagelink float-left"><A href="/wiki/images/0/0f/Muc16_ProteinFigure_002.png" class="internal" title="Muc16 ProteinFigure 002.png">Media:Muc16_ProteinFigure_002.png</A><B>Figure 2:</B> Overview of biotin: Streptavidin interaction.</DIV><P>The rapid and very strong linkage between the two binding partners results from the creation of multiple hydrogen bonds as well as Van der Waals interactions upon binding. Comparing the Apo-state of Streptavidin and the Streptavidin-biotin complex hereby shows several structural differences of the binding pocket.</P><P>The ordering of a flexible surface polypeptide loop results in burying the biotin molecule in its pocket. Without biotin, the L3/4 loop is disordered and does not give clear electron density, while it closes upon biotin binding.<SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP>
Important for the high energetic barrier required for dissociation are biotin-tryptophan contacts in the binding pocket. There are four significant tryptophan residues involved in the binding site. Three of those (Trp79, Trp92 and Trp108) are lined up in one section. Trp120 binds the biotin molecule from an adjacent subunit and displays a much larger influence in binding free energy.<SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP>
Moreover the hydrogen bonding network and Van der Waals forces show great influence in binding free energy. Compared to similar hydrogen bonding donors and acceptors of other protein-ligand systems the hydrogen bonding of Streptavidin to the ureido oxygen of biotin is remarkably high. 
</P></DIV><H1><SPAN class="mw-headline" id="Streptavidin_variants_and_homologs">Streptavidin variants and homologs</SPAN></H1><DIV class="white-box"><P>Several streptavidin variants were expressed, purified and tested in order to determine how their specific properties influence the printing processes. An overview of the different physical properties of the Streptavidin variants we used are shown below. 
</P><TABLE class="wikitable table table-bordered"><CAPTION><I><B>Table 1:</B></I> Properties of various Avidin variants.
</CAPTION><TBODY><TR><TH>protein
</TH><TH>amino acids (without Met)
</TH><TH>molecular weight monomer [Da]
</TH><TH>molecular weight tetramer [Da]
</TH><TH>theoretical isoelectric proint (pI)
</TH><TH>extinction coefficient (monomer, ε<SUB>280</SUB>) [M<SUP>-1</SUP> cm<SUP>-1</SUP>]
</TH><TH>T<SUB>m</SUB> [K]
</TH></TR><TR><TD>Streptavidin wt
</TD><TD>126
</TD><TD>13349.55
</TD><TD>52801.36
</TD><TD>6.09
</TD><TD>41940
</TD><TD>361.78
</TD></TR><TR><TD>Traptavidin
</TD><TD>126
</TD><TD>13278.43
</TD><TD>52516.88
</TD><TD>5.14
</TD><TD>41940
</TD><TD> 360.59
</TD></TR><TR><TD>Streptactin
</TD><TD>126
</TD><TD>13390.65
</TD><TD>52516.88
</TD><TD>8.32
</TD><TD>41940
</TD><TD>350.86
</TD></TR><TR><TD>enhanced monomeric Avidin
</TD><TD>138
</TD><TD>15369.51
</TD><TD>/
</TD><TD>5.91
</TD><TD>35075
</TD><TD>320.6
</TD></TR></TBODY></TABLE></DIV><DIV class="imagelink float-right"><A href="/wiki/images/8/83/Muc16_ProteinOverview_001.png" class="internal" title="Muc16 ProteinOverview 001.png">Media:Muc16_ProteinOverview_001.png</A><B>Figure 3:</B> Overview of the proteins used for bioprinting.</DIV><H2><SPAN class="mw-headline" id="Traptavidin:_increased_mechanical_strength_of_biotin_binding">Traptavidin: increased mechanical strength of biotin binding</SPAN></H2><DIV class="white-box"><P>Being a streptavidin derivate with two point mutations that increase its affinity (S52G, R53D), traptavidin shows a 10-times lower dissociation constant compared to the wildtype (K<SUB>D</SUB>=~10<SUP>-16</SUP> M). Moreover, the tetramer has an improved thermostability and thus remains more functional at higher temperatures (T<SUB>M</SUB>~10 °C higher)<SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP>. 
In contrast to streptavidin, the Traptavidin L3/4 loop (residues 45-50) does not change its conformation upon binding biotin. The binding pocket is already closed and lacks flexibility. The loss of a structural change is thought to lower the entropic cost of binding and therefore decreases the k<SUB>off</SUB>-value as well as the dissociation constant.
</P></DIV><H2><SPAN class="mw-headline" id="Strep-Tactin:_a_Strep-tag_binding_protein">Strep-Tactin: a <I>Strep</I>-tag binding protein</SPAN></H2><DIV class="white-box"><P>The <I>Strep</I>-tag is a small peptide composed of eight amino acids and can be used as affinity tag by genetic fusion to the N- or C-terminus of a recombinant protein. Using the <I>Strep</I>-tag's interaction with &quot;Strep&quot;-Tactin for affinity chromatography enables a simple and rapid one step-purification and provides great yields in protein purification. The <I>strep</I>-tagged protein binds Strep-Tactin, which is immobilized on the column matrix. After removing host proteins by washing, the target protein can be eluted with desthiobiotin, which binds the Strep-Tactin several orders of magnitude stronger. Over the years both, the <I>Strep</I>-tag and the streptavidin were optimized  resulting in <I>Strep</I>-tag II (amino acid sequence: WSHPQFEK) affinity tag with enhanced binding to Strep-Tactin. <SUP id="cite_ref-8" class="reference"><A href="#cite_note-8">[8]</A></SUP></P></DIV><H2><SPAN class="mw-headline" id="Enhanced_Monomeric_Avidin_.28eMA:_engineered_for_applications_like_molecular_labelling">Enhanced Monomeric Avidin (eMA: engineered for applications like molecular labelling</SPAN></H2><DIV class="white-box"><P>Due to the monovalent nature of eMA, unwanted cross-linking of biotin conjugats can not occur. 
Since streptavidin binding pockets depend on one residue from a adjacent subunit, binding affinity of a single subunit is decreased (K<SUB>D</SUB> = 10<SUP>-7</SUP> to 10<SUP>-9</SUP> M<SUP>-1</SUP>). eMA consists of a monomerized Rhizavidin dimer, which contains a disulfid bond in the binding site restraining the protein and forming a rigid binding pocket<SUP id="cite_ref-9" class="reference"><A href="#cite_note-9">[9]</A></SUP>.
</P></DIV><H1><SPAN class="mw-headline" id="Expression_and_purification_of_streptavidin">Expression and purification of streptavidin</SPAN></H1><DIV class="imagelink float-right"><A href="/wiki/images/0/08/Muc16_shakingflask.jpg" class="internal" title="Muc16 shakingflask.jpg">Media:Muc16_shakingflask.jpg</A><DIV class="caption"><B>Figure 4:</B> shaking flask production: balancing centrifuge tubes filled with cell culture.</DIV></DIV><DIV class="white-box"><P>&quot;E. coli&quot; BLR(DE3) was transformed with the plasmid pSA1 coding for the minimal streptavidin gene under the control of a T7 promoter. The <I>E.coli</I> BL-R (DE3) strain, a BL21 derivative, encodes the T7-polymerase under the control of a lacUV5 promotor, which can be induced by Isopropyl β-D-1-thiogalactopyranoside (IPTG).
A shaking flask with 2 L LB medium supplemented with ampicillin was inoculated 1:40 with a 50 ml pre-culture grown at 30 °C over night.
After shaking the bacterial culture at 37 °C until its optical density reaches 0.6, protein expression was induced with 1 mM IPTG. Followed by 4 hours shaking at 37 °C cells were harvested by centrifugation for 30 min at 5000 rpm. The cell pellet was resuspended in buffer A (20 mM Tris/HCl - pH 8, 500 mM NaCl). Bacterial cells were lysed using a PANDA homogenizer and inclusion bodies in the cell lysate are pelleted by centrifugation. The pellet fraction was washed with buffer A to remove &quot;E. coli&quot; membrane and cell wall material. Guanidine·HCl (6 M, pH 1,5) was used to solubilize the washed pellet protein.
After solubilization the solution was centrifuged to remove remaining aggregates. Refolding of the solubilized proteins is initiated by the removal of the denaturant. The rapid dilution of the solubilized streptavidin was achieved by dripping the protein solution into large volume of refolding buffer (30 ml PBS for 1 ml protein solution). 
After incubation at 4 °C over night, the solution was centrifuged to remove aggregates.
The Streptavidin purification was proceeded by fractionated ammonium sulfate precipitation. 
At 40 % saturation (1.75 M), streptavidin remains soluble and precipitated proteins were removed by centrifugation. In the next step streptavidin was precipitated by increasing the ammonium sulfate saturation up to 70 % (3.37 M).</P><DIV class="imagelink float-right"><A href="/wiki/images/2/25/Muc16_Aekta.png" class="internal" title="Muc16 Aekta.png">Media:Muc16_Aekta.png</A><DIV class="caption"><B>Figure 5:</B> ÄKTA protein purification system with a Sephacryl S300 16 60 column.</DIV></DIV><P>After centrifugation, the pellet was resuspended in a 50 % saturated ammonium sulfate solution (2.25 M). After following centrifugation the pellet is resuspended in 1x PBS buffer.
The protein was dialysed against 20 mM Tris/HCl buffer  (pH 8.0) without salt and loaded on a Resource Q column for ion exchange chromatography. Proteins were eluted by an increasing salt gradient (hier Abb. Chromatograhie!).
Purified streptavidin was analysed by SDS–PAGE and Gelfiltration chromatography conducted at room temperature using a Sephacryl S-200 HR column. 
After sterile filtration, the protein concentration was determined by UV/VIS spectroscopy.<SUP id="cite_ref-10" class="reference"><A href="#cite_note-10">[10]</A></SUP>
The streptavidin variants StrepTactin and TrapAvidin were produced as cytoplasmic inclusion bodies, solubilized, refolded, purified by fractionated ammonium sulfate precipitation and analysed by SDS–PAGE according to wild-type streptavidin. Enhanced monomeric Avidin was expressed in &quot;E. coli&quot; BL21 and purified via the His-tag using IMAC chromatography.
</P><DIV class="imagelink float-right"><A href="/wiki/images/0/00/MUC16_SA_Production.jpg" class="internal" title="MUC16 SA Production.jpg">Media:MUC16 SA Production.jpg</A><DIV class="caption"><B>Figure 6:</B> Overview Streptavidin expression and isolation.</DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Streptavidin_Fermentation">Streptavidin Fermentation</SPAN></H1><DIV class="white-box"><P><VIDEO width="450" style="float:right; margin-left: 15px; margin-top : 15px" autoplay="" muted="" loop="" controls=""><SOURCE src="https://static.igem.org/mediawiki/2016/0/08/Fermentation.mp4">
Your browser does not support this video player.
</SOURCE></VIDEO></P><P>Fermentation is utilized for upscaling protein production under strictly defined conditions. A high cell density cultivation method was used to produce recombinant streptavidin in E. coli. Glucose was fed for 12 h after a over night batch phase to achieve a specific growth rate of 0.16 h<SUP>−1</SUP> until cells reached an OD<SUB>550</SUB> of 30. Culture was then induced with 1 mM IPTG and cells were further grown for 12 h to reach an OD<SUB>550</SUB> of 60. The recombinant streptavidin was produced as cytoplasmic inclusion bodies, solubilized, refolded, purified by fractionated ammonium sulfate precipitation and analysed by SDS–PAGE as described above. After refolding and purification, streptavidin was greater than 98% pure, homogeneous and functionally active.</P><DIV class="imagelink float-right"><A href="/wiki/images/f/f6/Muc16_fermentationoverview.png" class="internal" title="Muc16 fermentationoverview.png">Media:Muc16_fermentationoverview.png</A><DIV class="caption"><B>Figure 7:</B> Procedure of fermentation.</DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Biotinylated_PAS-lysine:_A_flexible_linker">Biotinylated PAS-lysine: A flexible linker</SPAN></H1><DIV class="white-box"><P>PAS-lysine is an artificial polypeptide of 10 peptide cassettes composed of the three small, chemically stable amino acids Pro, Ala, and Ser (PAS) <SUP id="cite_ref-11" class="reference"><A href="#cite_note-11">[11]</A></SUP> ,which are interspersed by Lysine residues to allow site-specific biotinylation with Biotin NHS-Ester. PAS-Lysine is expressed at high yields in <I>Escherichia coli</I> as a Small Ubiquitin-like Modifier (SUMO) fusion protein, which is easily purified via His-tag and allows isolation of the PAS-Lysine after SUMO protease cleavage. 
</P><DIV class="imagelink float-right"><A href="/wiki/images/5/53/Muc16_PAS_Production_beschriftet.png" class="internal" title="Muc16 PAS Production beschriftet.png">Media:Muc16_PAS_Production_beschriftet.png</A><DIV class="caption"><B>Figure 8:</B> Expression and purification of PAS-lysine.</DIV></DIV><P>A 50 ml preculture with <I>E.coli</I> BL R (DE3) carrying the pSUMO plasmid is poured into a shaking flask with 2 L LB medium. After the bacterial suspension reaches an optical density of 0.5 the expression of PAS-lysine is induced by adding IPTG (1 mM). 
The cells are harvested (5000 rpm, 30 minutes) after five hours shaking at 37 °C and resuspended in 20 mM Tris/HCl buffer with 500 mM salt, pH 8. Homogenization is conducted by French press or PANDA. SUMO PAS-lysine is very good soluble, after centrifugation at 11500 rpm for 30 minutes it is located in the supernatant, the pellet can be discarded. 
SUMO PAS-lysine has the property to remain native even at high temperatures, where most protein denaturate. Heating the solution up to 70 °C for 20 minutes and centrifugation afterwards removes nearly all other proteins. 
The solution is dialysed against IMAC buffer (50 mM sodium dihydrogen phosphate and 500 mM salt, ph 7.5) and loaded on a chromatography column packed with nickel. After washing away impurities, the addition of imidazole results in elution of the SUMO PAS-lysine. 
The SUMO domain has no further use and is cleaved off by the SUMO-protease ULP. Another IMAC separates the PAS-lysine from SUMO domain, which binds specifically to the column. PAS lysine can be identified in the flow through via UV absorption. Because PAS-lysine does not contain aromatic residues, its absorption at 280 nm is very low, while the absorption at 225 nm (amid bondage) is average.</P>
The proper fractions are pooled and the solution is dialysed against 20 mM Tris/HCl buffer with 500 mM salt, pH 8. The linker molecule can now be biotinylated in a molar excess of 10 or 20. After incubation over night, dialysis removes biotin molecules which did not bind the PAS-lysine linker.</DIV><H1><SPAN class="mw-headline" id="eGFP_and_BSA:_alternative_linker_molecules">eGFP and BSA: alternative linker molecules</SPAN></H1><DIV class="white-box"><P>Serum albumin is the most abundant plasma protein in mammals. It is a transporter molecule for a diverse range of metabolites, drugs, nutrients, metals and other molecules and shows remarkable ligand binding capacity. <SUP id="cite_ref-12" class="reference"><A href="#cite_note-12">[12]</A></SUP> Bovine serum albumin, originally isolated from a cow, is often used as a concentration standard for protein assays, it is therefore in stock of many laboratories. Since it has 60 lysine residues within its 607 amino acids long sequence, the soluble BSA protein works well as a linker molecule.
In recent years eGFP has emerged as a powerful reporter molecule for monitoring gene expression, protein localization and protein-protein interaction. <SUP id="cite_ref-13" class="reference"><A href="#cite_note-13">[13]</A></SUP>. Due to its emission of green light, eGFP helped improving the printing and polymerization process by enabling the opportunity to see if polymerization of the sample was successful immediately. Thus, only promissing samples were analysed under a microscope. 
As a globular protein with 20 lysine residues in its sequence, eGFP can be used as a chemically biotinylated linker molecule. 
</P><DIV class="imagelink float-right"><A href="/wiki/images/c/ce/Muc16_eGFP_production.png" class="internal" title="Muc16 eGFP production.png">Media:Muc16_eGFP_production.png</A><DIV class="caption"><B>Figure 9:</B> Expression and purification of eGFP.</DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Further_characterisation_of_streptavidin_variants">Further characterisation of streptavidin variants</SPAN></H1><H2><SPAN class="mw-headline" id="Overview_of_purified_proteins">Overview of purified proteins</SPAN></H2><DIV class="white-box"><P>Streptavidin and its variants were produced by cytoplasmic expression in E. coli in inclusion bodies. After stabilization and refolding of the functional tetrameric and monomeric variants, purification was carried out by ammonium sulfate precipitation, Ion Exchange Chromatography and size exclusion chromatography. Samples of each required step were analysed by SDS-PAGE.
Finally each of the successfully purified variants were characterized. Mass spectrometry was used to confirm the expected size, circular dichroism spectroscopy (cd) to determine folding states, fluorescence titration and surface plasmon resonance for determining binding affinities.
</P></DIV><H2><SPAN class="mw-headline" id="Quality_control_using_ESI-TOF_mass_spectrometry">Quality control using ESI-TOF mass spectrometry</SPAN></H2><DIV class="white-box"><P>The molecular mass of the purified proteins were measured by ESI-MS on an maXis Q-TOF instrument (Bruker, Bremen, Germany) operated in positive ion mode after dialysis against 10 mM ammonium acetate (pH 6.6). Samples were supplemented with 20% (v/v) acetonitrile (LCMS grade; Sigma-Aldrich, Steinheim, Germany) and 0.1% (v/v) formic acid (LC-MS grade; Sigma-Aldrich). Raw data were analyzed and deconvoluted with Compass Data Analysis software (version 4.0). The calculated masses of all proteins were confirmed mass spectrometry data. 
</P></DIV><H2><SPAN class="mw-headline" id="Quality_control_using_SDS-PAGE_and_isoelectric_focusing">Quality control using SDS-PAGE and isoelectric focusing</SPAN></H2><DIV class="white-box"><DIV class="imagelink float-right"><A href="/wiki/images/d/d4/Muc16_results_gels.png" class="internal" title="Muc16 results gels.png">Media:Muc16_results_gels.png</A><DIV class="caption"><B>Figure 12:</B> SDS PAGE and isoelectric focussing of Streptavidin variants.</DIV></DIV><P>Following purification, 12 μl of protein were incubated with  SDS-Loading dye at room temperature. One set of reactions were then denatured at 95 °C for 10 min. and subjected to 13% SDS-PAGE and stained with Coomassie Blue to determine. Previous work on the electrophoretic mobility of streptavidin has shown that the protein maintains its tetrameric native structure (∼60 kDa) and activity in the presence of SDS-PAGE sample buffers, and requires significant heating to break this interaction.
</P></DIV><H2><SPAN class="mw-headline" id="Affinity_determination_using_fluorescence_titration">Affinity determination using fluorescence titration</SPAN></H2><DIV class="white-box"><DIV class="imagelink float-right"><A href="/wiki/images/9/91/Muc16_results_flt.png" class="internal" title="Muc16 results flt.png">Media:Muc16_results_flt.png</A><DIV class="caption"><B>Figure 13:</B> Fluorescence titration curves for various avidin variants.</DIV></DIV><P>Fluorescence titration is a method that utilizes the change of intrinsic protein fluorescence upon ligand binding in order to quantify binding properties. This way, the binding affinity - as characterized by the KD value - can be determined. 
For the analysis of protein-ligand or protein-protein interactions, one binding partner is therefore titrated against a constant concentration of the other binding partner (streptavidin or a variant).
The path length of the cuvettes used for absorption and emission studies hereby was 1 cm. For fluorescence titrations, stock solutions of streptavidin or its variants were prepared (1 μM) in 50 mM Tris–HCl, 100 mM NaCl , pH 8.0. Working solutions of biotin (100 µM) were prepared from a stock solution in the same buffer. Fluorescence titration was performed with a Perkin-Elmer LS 50 instrument. Wavelengths for excitation and emission were set to 295 and 350 nm, respectively, with slide widths of 5 nm. A 2 ml volume of a 1 µM Streptavidin solution was pipetted into the cuvette. Then, 2 µl aliquots of biotin solution were added and incubated for 1 minute. Fluorescence measurements were performed after each incubation by using 5 sec integration time settings. The data was fitted by non-linear least-squares regression.
Thereby, the binding affinities of wild-type streptavidin and its variants towards biotin were quantified via fluorescence titration.
Although traptavidin should have a 10-fold  lower dissociation constant K<SUB>D</SUB> than Streptavidin due to its optimized binding pocket, the results differ. Both, streptavidin and strep-tactin seem to have a lower K<SUB>D</SUB>than traptavidin. Only the results for eMA came up to expectations. 
</P></DIV><H2><SPAN class="mw-headline" id="Analysis_of_secondary_structure_and_thermal_stability">Analysis of secondary structure and thermal stability</SPAN></H2><DIV class="white-box"><P>For the analysis of protein folding states, circular dichroism spectroscopy was applied, as it allows drawing conclusions about the formation of protein secondary structures. 
To determine the ideal wavelengths for measurement, a UV-CD spectrum was recorded in folded and unfolded protein state. Therefore a 10 µM protein solution (according to monomer), solved in buffer A (20 mM KH2PO4, 50 mM K2SO4; pH 7.5), was measured in a 1 mm cuvette. The spectrum was recorded between 190 and 250 nm at 20 °C. By comparing spectra of folded and unfolded state, the wavelength showing maximum variance was selected. Afterwards a denaturation and renaturation spectrum was recorded for each protein by setting 213 nm wavelength and measuring while heating up the solution from 20 °C to 95 °C. As an exception, enhanced monomeric avidin was measured at 203 nm. For blank measurement only buffer was used.
De- and renaturation curves as well as UV-CD-spectra were measured and can be seen in figure 14. All CD spectra were evaluated by the Spectra Manager Software. Results can be seen in figure 14. Furthermore, the temperature melting temperature Tm was calculated, being defined as the temperature where 50% of the protein are denatured. Herefore, the data for each protein was entered into a GraphPad Prism sheet and calculated respectively. Results can be seen in table 1. Streptavidin and streptactin are appear more stable than eMA, but less stable than traptavidin. 
</P></DIV><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><DIV class="white-box"><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Determined via <A rel="nofollow" class="external free" href="http://www.expasy.org/">http://www.expasy.org/</A></SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Chivers, C. E., Koner, A. L., Lowe, E. D., &amp; Howarth, M. (2011). How the biotin–Streptavidin interaction was made even stronger: investigation via crystallography and a chimaeric tetramer. Biochemical Journal, 435(1), 55-63.</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">Weber, P. C., Ohlendorf, D. H., Wendoloski, J. J., &amp; Salemme, F. R. (1989). Structural origins of high-affinity biotin binding to Streptavidin. Science, 243(4887), 85.</SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text">Schmidt, T. G., &amp; Skerra, A. (2007). The Strep-tag system for one-step purification and high-affinity detection or capturing of proteins. Nature protocols, 2(6), 1528-1535.</SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">↑</A></SPAN><SPAN class="reference-text">Weber, P. C., Ohlendorf, D. H., Wendoloski, J. J., &amp; Salemme, F. R. (1989). Structural origins of high-affinity biotin binding to Streptavidin. Science, 243(4887), 85.</SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">↑</A></SPAN><SPAN class="reference-text">Stayton, P. S., Freitag, S., Klumb, L. A., Chilkoti, A., Chu, V., Penzotti, J. E., ... &amp; Stenkamp, R. E. (1999). Streptavidin–biotin binding energetics. Biomolecular engineering, 16(1), 39-44.</SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text">Chivers, C. E., Crozat, E., Chu, C., Moy, V. T., Sherratt, D. J., &amp; Howarth, M. (2010). A streptavidin variant with slower biotin dissociation and increased mechanostability. Nature methods, 7(5), 391-393.</SPAN></LI><LI id="cite_note-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-8">↑</A></SPAN><SPAN class="reference-text">Schmidt, T. G., &amp; Skerra, A. (2007). The Strep-tag system for one-step purification and high-affinity detection or capturing of proteins. Nature protocols, 2(6), 1528-1535.</SPAN></LI><LI id="cite_note-9"><SPAN class="mw-cite-backlink"><A href="#cite_ref-9">↑</A></SPAN><SPAN class="reference-text">Lee, J. M., Kim, J. A., Yen, T. C., Lee, I. H., Ahn, B., Lee, Y., ... &amp; Jung, Y. (2016). A Rhizavidin Monomer with Nearly Multimeric Avidin‐Like Binding Stability Against Biotin Conjugates. Angewandte Chemie International Edition, 55(10), 3393-3397.</SPAN></LI><LI id="cite_note-10"><SPAN class="mw-cite-backlink"><A href="#cite_ref-10">↑</A></SPAN><SPAN class="reference-text">Skerra, A., Gebauer, M., Schönfeld, D., Schmelz, E. (2007). Skript zu &quot;Fortgeschrittenenpraktikum Proteinchemie&quot;</SPAN></LI><LI id="cite_note-11"><SPAN class="mw-cite-backlink"><A href="#cite_ref-11">↑</A></SPAN><SPAN class="reference-text">Schlapschy, M., Binder, U., Börger, C., Theobald, I., Wachinger, K., Kisling, S., ... &amp; Skerra, A. (2013). PASylation: a biological alternative to PEGylation for extending the plasma half-life of pharmaceutically active proteins. Protein Engineering Design and Selection, 26(8), 489-501.</SPAN></LI><LI id="cite_note-12"><SPAN class="mw-cite-backlink"><A href="#cite_ref-12">↑</A></SPAN><SPAN class="reference-text">Majorek, K. A., Porebski, P. J., Dayal, A., Zimmerman, M. D., Jablonska, K., Stewart, A. J., ... &amp; Minor, W. (2012). Structural and immunologic characterization of bovine, horse, and rabbit serum albumins. Molecular immunology, 52(3), 174-182.</SPAN></LI><LI id="cite_note-13"><SPAN class="mw-cite-backlink"><A href="#cite_ref-13">↑</A></SPAN><SPAN class="reference-text">Cinelli, R. A., Ferrari, A., Pellegrini, V., Tyagi, M., Giacca, M., &amp; Beltram, F. (2000). The Enhanced Green Fluorescent Protein as a Tool for the Analysis of Protein Dynamics and Localization: Local Fluorescence Study at the Single‐molecule Level. Photochemistry and photobiology, 71(6), 771-776.</SPAN></LI></OL></DIV></DIV></DIV><DIV class="row"><FOOTER class="col-sm-9 col-sm-offset-3 noprint page-footer clearfix white-box"><A class="btt visible-xs-block" href="#top"><SPAN class="sr-only">Back to top</SPAN></A><DIV class="row"><DIV class="col-sm-4 social"><H2>Follow us on:</H2></DIV><DIV class="col-sm-4 universities"><H2>LMU &amp; TUM Munich</H2><P>United team from Munich's universities</P></DIV><DIV class="col-sm-4 address"><H2>Contact us:</H2><H3>Address</H3><P>iGEM Team TU-Munich
                     Emil-Erlenmeyer-Forum 5
                     85354 Freising, Germany</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>