<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_LMU-TUM_Munich_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:LMU-TUM Munich/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="row"><DIV class="col-sm-3 col-xs-8 fixed-sm z-1"><DIV class="sidebar-nav"><DIV class="navbar" role="navigation"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".sidebar-navbar-collapse"><SPAN class="sr-only">Toggle menu</SPAN></BUTTON><SPAN class="visible-xs navbar-brand">Menu</SPAN></DIV><DIV class="navbar-collapse collapse sidebar-navbar-collapse"><UL id="main-menu" class="nav navbar-nav project"><LI class="menu_item home"><A href="https://2016.igem.org/Team:LMU-TUM_Munich"><SPAN class="visible-xs-block">Home</SPAN></A></LI><LI class="dropdown menu_item team"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI class="members"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Team">Members</A></LI><LI class="collaborations"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Collaborations">Collaborations</A></LI><LI class="sponsors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Sponsors">Sponsors</A></LI><LI class="press"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Press">Press</A></LI><LI class="gllry"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Gallery">Gallery</A></LI></UL><LI class="dropdown menu_item project"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Project </A></LI><LI class="description"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Description">Description</A></LI><LI class="design"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Design">Design</A></LI><LI class="proteins"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proteins">Proteins</A></LI><LI class="receptors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Receptors">Receptors</A></LI><LI class="linker"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Linkerchemistry">Linker Chemistry</A></LI><LI class="poly"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proof">Polymerization</A></LI><LI class="bioprinting"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Demonstrate">Bioprinting</A></LI><LI class="hypoxia"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hypoxia">Hypoxia</A></LI><LI class="medical"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Medical">Medical Application</A></LI><LI class="interlab"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/InterLab">InterLab</A></LI><LI class="software"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Software">Software</A></LI><LI class="dropdown menu_item notebook"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Notebook </A></LI><LI class="labjournal"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Labjournal">Labjournal</A></LI><LI class="materials"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Materials">Materials</A></LI><LI class="methods"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Methods">Methods</A></LI><LI class="howtos"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Howtos">How Tos</A></LI><LI class="dropdown menu_item parts"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Parts </A></LI><LI class="parts-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Parts">Parts</A></LI><LI class="basic-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Basic_Part">Basic Parts</A></LI><LI class="composite-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Composite_Part">Composite Parts</A></LI><LI class="part-collection"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Part_Collection">Part Collection</A></LI><LI class="menu_item hardware"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hardware">Hardware</A></LI><LI class="menu_item model"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Model">Modeling</A></LI><LI class="menu_item safety"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Safety">Safety</A></LI><LI class="dropdown menu_item human_practices dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Human Practices </A></LI><LI class="human_practices-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Human_Practices">Human Practices</A></LI><LI class="silver"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Silver">Silver</A></LI><LI class="gold"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Gold">Gold</A></LI><LI class="integrated-practices"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Integrated_Practices">Integrated Practices</A></LI><LI class="engagement"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Engagement">Engagement</A></LI><LI class="dropdown menu_item entrepreneur dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Entrepreneurship</A></LI><LI class="entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Entrepreneurship">Entrepreneurship</A></LI><LI class="supporting-entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Supporting_Entrepreneurship">Supporting Entrepreneurship</A></LI><LI class="menu_item attributions"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV class="col-sm-9 col-sm-offset-3"><H1><SPAN class="mw-headline" id="The_sub-projects_of_biotINK_and_their_achievements">The sub-projects of biotINK and their achievements</SPAN></H1><DIV class="white-box"><H2><SPAN class="mw-headline" id="BioBrick_Part_Design"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Design">BioBrick Part Design</A></SPAN></H2><P>As does every sophisticated SynBio project, we designed a plethora of BioBrick constructs and devices in the course of our project. Most prominent among these are the modular transmembrane proteins we conceptualized, since they are integral to the functionality of our bioprinting system. Basing our workflow on the standard RFC[10] and RFC[25] assembly standards, we assembled the same intracellular and transmembrane domains, namely mRuby3 and the EGFR transmembrane domain, with different extracellular domains, among them single-chain avidin, monomeric avidin, and NanoLuc fused with an N-terminal Avi-tag for <I>in vivo</I> biotinylation by the co-expressed biotin ligase BirA. We coupled these constructs with a trafficking signal for membrane integration. Including further elements to stabilize these receptors offers advantages over previously assembled BioBricks, and planning our constructs with multiple options for detection provided us ample opportunity to extensively characterize the outcome. Apart from these eukaryotic designs, we drafted an autotransporter construct for bacterial surface display.
</P><H2><SPAN class="mw-headline" id="Production.2C_purification_.26_conjugation_of_recombinant_proteins"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Proteins">Production, purification &amp; conjugation of recombinant proteins</A></SPAN></H2><P>As the second constituent of our two-component polymerization system next to the membrane proteins, we required large amounts of protein, either biotin-binding or bearing chemically introduced biotin depending on the receptor in question. For biotin-binding proteins, we produced and purified the avidin variants wt streptavidin; traptavidin,a variant with higher affinity toward biotin and enhanced thermal stability; streptactin, a variant that binds an oligopeptide sequence termed strep-tag; and enhanced monomeric avidin, which does not assemble into tetramers and is thus monovalent. wt Streptavidin was produced both in the dimensions of shaker flasks and a fermenter, and purified according to an established protocol. As a scaffold for chemical biotinylation, we produced and purified PAS-lysine and GFP. We extensively characterized all proteins as well as the biotinylated forms of the latter group plus BSA by ESI-TOF, SDS-PAGE, and IEF; moreover, we determined the affinity of streptavidin variants <I>via</I> fluorescence titration and their thermal stability using CD spectra. We produced all target proteins in good purity and successfully biotinylated PAS-lysine, GFP, and BSA.
</P><H2><SPAN class="mw-headline" id="A_fully_functional_cross-kingdom_membrane_localization_toolbox"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Receptors">A fully functional cross-kingdom membrane localization toolbox</A></SPAN></H2><P>Based on our modular membrane protein design, we evaluated the functionality of different signal peptide in shuttling the protein into the membrane. Apart from taking the BioBrick EGFR signal peptide, we designed two further signal peptides with increased translational efficiency, incorporating items such as the Kozak sequence. These three signal peptides were analyzed <I>in silico</I> as well as experimentally using a luciferase secretion assay. The BM40 signal, one of our self-designed constructs, was most efficient and thus incorporated into our other devices. Additionally, we incorporated three reporter modules (mRuby3, A3C5 epitope tag and strep-tag II) into the receptors. The finished constructs were used to generate semi-stablly transfected and stable cell lines. Protein localization was confirmed <I>via</I> fluorescence and high-resolution fluorescence microscopy, including characterization of a trafficking signal for increased membrane localization, and the recombinant protein was quantified by FACS and Western blot. mRNA expression levels were detected by RT-q-PCR. For visualization, aggregation of functionalized cells with Strep beads was subjected to SEM. The bacterial autotransporter was characterized and found to successfully shuttle small protein domains to the extracellular surface.
</P><H2><SPAN class="mw-headline" id="Chemical_synthesis_of_amino-reactive_biotinylation_compounds"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Linkerchemistry">Chemical synthesis of amino-reactive biotinylation compounds</A></SPAN></H2><P>Apart from the standard biotin-NHS, several other derivatives spacing biotin and NHS apart by an amino acid residue were synthesized. These derivatives were used in the chemical biotinylation of BSA in order to assess their relative stability toward cleavage by the plasma protein biotinidase. The assay principle is based on a colorimetric reaction and was developed by ourselves. Compared to biotinylation <I>via</I> regular biotin-NHS, the derivative incorporating a spacer aminocaproic acid is cleaved somewhat faster, allowing to dissolve a cell mesh more quickly, and the spacer valine resulted in a drastically slowed hydrolysis rate. 
</P><H2><SPAN class="mw-headline" id="Proof_of_principle_for_the_avidin-biotin_.22polymerization.22"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Proof">Proof of principle for the avidin-biotin &quot;polymerization&quot;</A></SPAN></H2><P>We chose the avidin-biotin system because of their high affinity and rapid association. Our successful polymerization tests showed that cell density, concentration of biotinylated BSA in the bioink, and streptavidin concentration in the reservoir are the three major factors that impact performance. Thus, more BSA entailed higher structural integrity. We successfully printed defined objects with our technology and developed an optimal ratio of constituents. Aggregation was visually confirmed in the micromanipulator.
</P><H2><SPAN class="mw-headline" id="First_bioprinting_results_using_the_biotINK_bioprinter"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Demonstrate">First bioprinting results using the biotINK bioprinter</A></SPAN></H2><P>We successfully produced first prints of defined spatial structure in our bioprinter and show that our technology compares favorably with existing Inkjet and microextrusion technologies in bioprinting, especially in terms of cell viability.
</P><H2><SPAN class="mw-headline" id="A_vascularization_inducer_based_on_a_hypoxia-sensitive_promoter"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Hypoxia">A vascularization inducer based on a hypoxia-sensitive promoter</A></SPAN></H2><P>We designed a hypoxia-driven promoter, which we tested for its functionality under low oxygen as well as normal oxygen conditions. Although the promoter does not increase gene expressions when cells are exposed to hypoxia, it remains remarkably steady - a feature that most promoters do not have.
</P><H2><SPAN class="mw-headline" id="Safety_evaluation_of_our_project_and_the_kill-switch"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Safety">Safety evaluation of our project and the kill-switch</A></SPAN></H2><P>Sequencing results showed a point mutation in both the part by the iGEM team of Freiburg 2010 and the Slovenian part of 2012. The mutation led to a PstI restriction site, that complicated cloning according to iGEM standards. After a site-directed mutagenesis (Quickchange) the restriction site was removed and the sequence hence repaired. 
Quantitative analysis by mRNA measuring of Poly-A-sequence addition showed 4.6-fold higher mRNA stability in comparison to the non-Poly-A-construct. 
Kill-Switch induction turned out to be difficult when using the prodrug famciclovir. Although HEK293T cells are known to produce aldehyde oxidase <SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP> and xanthine oxidase <SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP> the conversion of famciclovir into its active form penciclovir seems not to proceed effectively enough in HEK293T cells.
</P><H2><SPAN class="mw-headline" id="InterLab_study:_Determining_reliability_of_measurements"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/InterLab"><B>InterLab study</B>: Determining reliability of measurements</A></SPAN></H2><P>We participated in this year's interlab study and measured five GFP expression plasmids in 8 different <I>E. coli</I> strains. In addition of measuring according to the standard protocol, we were able to improve upon the protocol in order to make measurements more comparable and allow the analysis of more strains at once.
</P><H2><SPAN class="mw-headline" id="Entrepreneurship"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Entrepreneurship">Entrepreneurship</A></SPAN></H2><P>For the entrepeneurial part of our project, we firstly created a business-plan for a possible commercial cause. Secondly, we took a look at enabling factors for startups resulting from iGEM via a survey, and analyzed what makes iGEM projects become a startup.
</P><H2><SPAN class="mw-headline" id="Hardware"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Hardware">Hardware</A></SPAN></H2><P>We were able to refunctionalize a conventional 3D-printer by replacing the plastic printing head with a syringe pump. For that purpose, we were able to print most of the new parts of the bioprinter via the 3D-printer itself. The low costs of rebuilding the printer allows the design to be adapted easily by other labs as well, thus allowing other groups to easily adapt our technique.</P><H2><SPAN class="mw-headline" id="Software"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Software">Software</A></SPAN></H2><P>Since the Ultimaker 2+ standard software is not versatile enough to allow for all requirements of bioprinting, we modified the software code in order to make the printing system more adaptable and perfectly suited to the needs of printing live cells. A number of parameters can now be specifically addressed and are tailored to bioprinting.
</P><H2><SPAN class="mw-headline" id="Measurement"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Measurement">Measurement</A></SPAN></H2><P>
The majority of our measurements focussed on the characterization of procaryotic and eucaryotic membrane proteins and includes various techniques. We could confirm the functionality of four biobrick devices and identify mRNA stability as a crucial factor for high surface presentation of membrane proteins.
</P><H2><SPAN class="mw-headline" id="Model"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Model">Model</A></SPAN></H2><P>In our modelling project we have developed a mathematical model for the polymerization of multivalent proteins and cells.
</P></DIV></DIV></DIV><DIV class="row"><FOOTER class="col-sm-9 col-sm-offset-3 noprint page-footer clearfix white-box"><A class="btt visible-xs-block" href="#top"><SPAN class="sr-only">Back to top</SPAN></A><DIV class="row"><DIV class="col-sm-4 social"><H2>Follow us on:</H2></DIV><DIV class="col-sm-4 universities"><H2>LMU &amp; TUM Munich</H2><P>United team from Munich's universities</P></DIV><DIV class="col-sm-4 address"><H2>Contact us:</H2><H3>Address</H3><P>iGEM Team TU-Munich
                     Emil-Erlenmeyer-Forum 5
                     85354 Freising, Germany</P></DIV></DIV></FOOTER></DIV></DIV><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text"> Moriwaki, Y., Yamamoto, T., Takahashi, S., Tsutsumi, Z., &amp; Hada, T. (2001). Widespread cellular distribution of aldehyde oxidase in human tissues found by immunohistochemistry staining. Histology and histopathology, 16(3), 745-753. </SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text"> Czupryna, J., &amp; Tsourkas, A. (2012). Xanthine oxidase‐generated hydrogen peroxide is a consequence, not a mediator of cell death. FEBS Journal, 279(5), 844-855. </SPAN></LI></OL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>