<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_LMU-TUM_Munich_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:LMU-TUM Munich/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="row"><DIV class="col-sm-3 col-xs-8 fixed-sm z-1"><DIV class="sidebar-nav"><DIV class="navbar" role="navigation"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".sidebar-navbar-collapse"><SPAN class="sr-only">Toggle menu</SPAN></BUTTON><SPAN class="visible-xs navbar-brand">Menu</SPAN></DIV><DIV class="navbar-collapse collapse sidebar-navbar-collapse"><UL id="main-menu" class="nav navbar-nav design"><LI class="menu_item home"><A href="https://2016.igem.org/Team:LMU-TUM_Munich"><SPAN class="visible-xs-block">Home</SPAN></A></LI><LI class="dropdown menu_item team"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI class="members"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Team">Members</A></LI><LI class="collaborations"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Collaborations">Collaborations</A></LI><LI class="sponsors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Sponsors">Sponsors</A></LI><LI class="press"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Press">Press</A></LI><LI class="gllry"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Gallery">Gallery</A></LI></UL><LI class="dropdown menu_item project"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Project </A></LI><LI class="description"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Description">Description</A></LI><LI class="design"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Design">Design</A></LI><LI class="proteins"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proteins">Proteins</A></LI><LI class="receptors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Receptors">Receptors</A></LI><LI class="linker"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Linkerchemistry">Linker Chemistry</A></LI><LI class="poly"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proof">Polymerization</A></LI><LI class="bioprinting"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Demonstrate">Bioprinting</A></LI><LI class="hypoxia"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hypoxia">Hypoxia</A></LI><LI class="medical"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Medical">Medical Application</A></LI><LI class="interlab"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/InterLab">InterLab</A></LI><LI class="software"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Software">Software</A></LI><LI class="dropdown menu_item notebook"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Notebook </A></LI><LI class="labjournal"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Labjournal">Labjournal</A></LI><LI class="materials"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Materials">Materials</A></LI><LI class="methods"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Methods">Methods</A></LI><LI class="howtos"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Howtos">How Tos</A></LI><LI class="dropdown menu_item parts"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Parts </A></LI><LI class="parts-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Parts">Parts</A></LI><LI class="basic-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Basic_Part">Basic Parts</A></LI><LI class="composite-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Composite_Part">Composite Parts</A></LI><LI class="part-collection"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Part_Collection">Part Collection</A></LI><LI class="menu_item hardware"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hardware">Hardware</A></LI><LI class="menu_item model"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Model">Modeling</A></LI><LI class="menu_item safety"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Safety">Safety</A></LI><LI class="dropdown menu_item human_practices dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Human Practices </A></LI><LI class="human_practices-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Human_Practices">Human Practices</A></LI><LI class="silver"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Silver">Silver</A></LI><LI class="gold"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Gold">Gold</A></LI><LI class="integrated-practices"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Integrated_Practices">Integrated Practices</A></LI><LI class="engagement"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Engagement">Engagement</A></LI><LI class="dropdown menu_item entrepreneur dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Entrepreneurship</A></LI><LI class="entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Entrepreneurship">Entrepreneurship</A></LI><LI class="supporting-entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Supporting_Entrepreneurship">Supporting Entrepreneurship</A></LI><LI class="menu_item attributions"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV class="col-sm-9 col-sm-offset-3"><DIV class="imagelink float-right"><A href="/wiki/images/a/ab/Muc16_Sticker_Design_001.png" class="internal" title="Muc16 Sticker Design 001.png">Media:Muc16_Sticker_Design_001.png</A></DIV><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Receptor_design_highlights"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Receptor design highlights</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Human_cell_surface_receptors"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Human cell surface receptors</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#The_incredible_journey:_signal_peptides_and_protein_targeting"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">The incredible journey: signal peptides and protein targeting</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Elements_for_stability_and_detection"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Elements for stability and detection</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#EGFR_transmembrane_domain"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">EGFR transmembrane domain</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#Elements_for_detection"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Elements for detection</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Design_of_an_autotransporter_construct_for_bacterial_surface_display"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Design of an autotransporter construct for bacterial surface display</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Receptor_design_highlights">Receptor design highlights</SPAN></H1><DIV class="white-box"><P><I>Think before you ink</I> may be a popular slogan wielded by tattoo adversaries, but it has also been perfectly applicable to creating our synthetic biology approach of bioprinting. In the following, we explore the contemplations that contributed to the design of the membrane proteins (also referred to as 'receptors') that allow us to embed eukaryotic and bacterial cells into a <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Proof">stable, three-dimensional and user-definable matrix</A> immediately upon printing. 
</P></DIV><H1><SPAN class="mw-headline" id="Human_cell_surface_receptors">Human cell surface receptors</SPAN></H1><DIV class="white-box"><DIV class="imagelink float-right"><A href="/wiki/images/b/be/Muc16_receptor_schematic_002.png" class="internal" title="Muc16 receptor schematic 002.png">Media:Muc16 receptor schematic_002.png</A><DIV class="caption"><B>Figure 1</B>: A) Receptor constructs in a schematic representation of functional genetic elements. B) Schematic representation of a receptor presenting an extracellular single-chain avidin that is endogenously biotinylated by a coexpressed biotin ligase (BirA) and thus presents biotin groups at the cell surface.</DIV></DIV><P>
For this purpose, we built <B>two kinds of receptor constructs</B>, both on their own able – when expressed in mammalian cells – to cross-link cells for our approach of bioprinting:
</P><P>1) <B>A receptor presenting an extracellular biotin acceptor peptide</B> that is endogenously biotinylated by a coexpressed biotin ligase (BirA), thus presenting biotin molecules on the cellular surface and allowing it to interact with streptavidin in the printing solution.
</P><P>2) <B>Receptors presenting an extracellular monomeric or single-chain avidin variant</B>.
</P><P>In presenting avidin on the cell surface, cells are able to interact with a co-injected biotinylated linker peptide. With this linker connected to several cell surface avidins as well as streptavidin in the printing solution, a protein matrix is created that embeds cells and allows precise positioning of cells in a three-dimensional manner.
</P><P>Our chosen avidin derivates by design allow functional fusion of the otherwise tetrameric avidin molecule with our receptor. Two different variants were used: <I>enhanced monomeric avidin</I>, a single-subunit avidin that is nevertheless able to bind biotin, and <I>single-chain avidin</I>, which resembles the naturally occurring avidin tetramer but consisting of a single polypeptide chain.
</P><P>All receptors mediate the same function: By interacting with streptavidin in the printing reservoir – either directly by presenting biotin (biotin acceptor peptide) or indirectly via binding to a biotinylated linker (single-chain avidin variants), cells are cross-linked due to polyvalent binding of streptavidin molecules in the printing solution to several cellular biotin groups at once.
</P></DIV><H2><SPAN class="mw-headline" id="The_incredible_journey:_signal_peptides_and_protein_targeting">The incredible journey: signal peptides and protein targeting</SPAN></H2><DIV class="white-box"><P>Essential to a protein's function are not only its activity or binding properties, but also its presence at the right time in the right place – the latter in our case being the cell surface. One ubiquitous element for protein targeting is the signal peptide, which enables translocation of proteins to the ER, from where their journey continues to their place of function. Proteins containing a signal peptide include transmembrane proteins, secreted proteins, proteins of the ER itself, proteins of the Golgi apparatus and several more.<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP>  For our project, signal peptides constituted a crucial part of many constructs, used to target transmembrane proteins to the cell surface (receptors) and proteins to the ER (BirA), and to secrete proteins into the surrounding medium (luciferase assays to quantify expression levels).
</P><DIV class="imagelink float-right"><A href="/wiki/images/5/5f/Muc16_SP_schematic.jpeg" class="internal" title="Muc16 SP schematic.jpeg">Media:Muc16 SP schematic.jpeg</A><DIV class="caption"><B>Figure 2:</B> Cotranslational translocation of a membrane protein into the ER in a schematic depiction showing the nascent, unfolded peptide chain containing signal peptide (<I>start-transfer sequence</I>) and <I>stop-transfer sequence</I> as well as the ER membrane.<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP></DIV></DIV><P>The hydrophobic signal peptide (also called <I>start-transfer sequence</I>) usually consists of less than 30 amino acids <SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP> and is recognized by the signal recognition particle upon translation at the rough ER, mediating cotranslational translocation of the nascent peptide chain into the ER.<SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP></P><P>During this project, three different signal peptide constructs were tested in order to determine that resulting in the highest expression levels:
</P><UL><LI> The <B>EGFR signal peptide</B> was taken from the iGEM parts registry (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K157001:Design">BBa_K157001</A>) and combined with a BioBrick containing the CMV promoter sequence (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K747096">BBa_K747096</A>) via the RFC10 cloning standard.</LI></UL><UL><LI> The <B>BM40 and Igκ signal peptides</B> were designed by us and synthesized with regard to adding additional spacing and functional elements to the 5'-UTR of the constructs, and combined with a BioBrick containing the CMV promoter sequence (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K747096">BBa_K747096</A>) via the RFC10 cloning standard. </LI></UL><P>
The EGFR signal peptide taken from the parts registry is constructed such that the signal peptide ORF immediately follows the RFC10 cloning scar after the CMV promoter, thus resulting in a very short 5’ untranslated region (UTR) of transcribed mRNA. The combination of the CMV promoter with the synthesized BM40 and Igκ signal peptides allows for a considerably longer 5’-UTR – and can thus be made to contain a full Kozak consensus sequence by design. The Kozak sequence is recognized by the ribosome as a translational start site; if this element is missing or deviates from the consensus sequence, this may considerably decrease translation efficiency.<SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP> For the EGFR signal peptide construct, a Kozak sequence is not present, as it would have to precede the start codon ATG, i.e. the position occupied by the RFC10 cloning scar.
Since both the distance from the promoter to the open reading frame and the Kozak consensus sequence are considered crucial parameters for expression levels<SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP>, BM40 and Igκ signal peptide constructs are likely to result in increased expression levels compared to proteins containing the EGFR signal peptide from the parts registry.
</P></DIV><DIV class="white-box"><P>In order to predict whether our designed signal peptides would be functional, we used the <A rel="nofollow" class="external text" href="http://www.cbs.dtu.dk/services/SignalP">SignalP</A> server, which allows to make theoretical predictions about signal peptide functionality and cleavage.<SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP></P><DIV class="imagelink float-right"><A href="/wiki/images/3/3d/Muc16_SignalPeptideprediction_002.png" class="internal" title="Muc16 SignalPeptideprediction 002.png">Media:Muc16 SignalPeptideprediction_002.png</A><DIV class="caption"><B>Figure 3:</B> A) Schematic depiction of a protein-coding mRNA containing a 5'- and 3'-UTR, the first of which is an important factor in translation efficiency. B) Theoretical prediction of signal peptides within the receptor construct containing the EGFR, Igκ and BM40 signal peptide via the SignalP 4.1 server. Plotted probabilities represent the raw cleavage site “C-score”, the signal peptide “S-score” and the combined cleavage site “Y-score”. C) ClustalW multiple sequence alignment of the BM40, EGFR and Igκ signal peptide construct, depicting a gap for the EGFR signal peptide construct compared to the newly designed ones and thus indicating a shorter length of the 5'-UTR . For Igκ and BM40 signal peptide constructs, a T7 promoter spacer as well as a Kozak sequence increase the distance between TATA box and protein-coding region.</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Elements_for_stability_and_detection">Elements for stability and detection</SPAN></H2><DIV class="white-box"><P>Apart from the functional parts that mediate the (strept)avidin-biotin interaction, other elements in the receptor were designed to make sure it is optimally translocated to the membrane, as stable as possible, and easily detectable. 
</P></DIV><H3><SPAN class="mw-headline" id="EGFR_transmembrane_domain">EGFR transmembrane domain</SPAN></H3><DIV class="white-box"><P>For anchoring in the membrane, a type I membrane protein with defined localization of N- and C-terminus out- and inside of the cell was used. The N-terminal transmembrane α-helix of human EGFR (UniProt P00533, amino acids 622-653) was herefore chosen (see below the topology prediction of EGFR via the <A rel="nofollow" class="external text" href="http://www.cbs.dtu.dk/services/TMHMM/">TMHMM 2.0 server for the prediction of transmembrane helices</A><SUP id="cite_ref-8" class="reference"><A href="#cite_note-8">[8]</A></SUP>). A stop-transfer sequence consisting of charged amino acids - being characteristic for type I membrane proteins - was added at the C-terminus, and the sequence was furthermore flanked by a (GGGGC)<SUB>2</SUB>-linker at the N- and C-terminus, respectively. The addition of a stop-transfer sequence as in the naturally occurring EGFR sequence, as well as the addition of flexible linkers, makes our EGFR-TMD an improvement over the BioBrick <A rel="nofollow" class="external text" href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K157002">BBa_K157002</A>, which does not possess either of these elements.
</P><DIV class="imagelink float-right"><A href="/wiki/images/0/05/Receptor_TMD_002.png" class="internal" title="Receptor TMD 002.png">Media:Receptor TMD 002.png</A><B>Figure 4:</B> A) Multiple sequence alignment of human EGFR (see <A rel="nofollow" class="external text" href="http://www.uniprot.org/uniprot/P00533">UniProt P00533</A>), the EGFR transmembrane domain BioBrick from the registry (<A rel="nofollow" class="external text" href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K157002">BBa_K157002</A> and the extended EGFR transmembrane designed by us, containing a charged stop-transfer domain as well as linker elements at the C- and N-termini. B) Transmembrane domain prediction by the <A rel="nofollow" class="external text" href="http://www.cbs.dtu.dk/services/TMHMM/">TMHMM 2.0 server</A>, indicating positioning of N- and C-termini out- and inside of the cell, respectively, as well as predicting functionality of transmembrane regions. C) Schematic depiction of the C-terminal stop-transfer sequence of the receptor.</DIV></DIV><H2><SPAN class="mw-headline" id="Elements_for_detection">Elements for detection</SPAN></H2><DIV class="white-box"><P>In order to confirm the correct expression and localization of the receptor, the receptor contains <B>three functional elements for its detection</B> for both the N- and the C-terminal receptor domains: 
</P><UL><LI> The intracellularly located C-terminal <B>red fluorescent protein mRuby 3</B> for detection of the receptor via fluorescence microscopy while also providing a stable, folded domain at the C-terminus,</LI><LI> an <B>extracellular epitope domain</B> near the N-terminus for immunochemical detection via A3C5-antibody fragments as well as FACS, and </LI><LI> an <B>intracellular <I>Strep</I>-tag II</B> at the N-terminus for the detection and purification via immunochemical methods.</LI></UL><P>Further elements that were included into the expression construct are the CMV promoter for overall high expression levels of the receptor, resulting in a high-avidity-interaction. At the 3'-end, the construct contains the polyadenylation signal of human growth hormone (hGH) for functional polyadenylation of the transcribed mRNA. 
</P></DIV><H1><SPAN class="mw-headline" id="Design_of_an_autotransporter_construct_for_bacterial_surface_display">Design of an autotransporter construct for bacterial surface display</SPAN></H1><DIV class="white-box"><P>Besides the design of biotinylated and biotin-binding receptors for eukaryotic cells, we also wanted to apply our <SPAN style="color:#009440">biot</SPAN><SPAN style="color:#3070b3">INK</SPAN> approach for prokaryotic cells. For this purpose, we designed an autotransporter device that is able to present biotinylated or biotin-binding protein domains on the surface of <I>E. coli</I>. Constructs for bacterial surface display are hereby already well known in the field of protein engineering of therapeutic proteins, such as antibodies. There, they are used to screen libraries of different antibodies concerning their affinity towards a given therapeutic target. 
</P><P>We based our design on the autotransporter EspP<SUP id="cite_ref-9" class="reference"><A href="#cite_note-9">[9]</A></SUP>.
</P><DIV class="imagelink float-right"><A href="/wiki/images/e/e5/Muc16_Autotransporterdesign_001.png" class="internal" title="Muc16 Autotransporterdesign 001.png">Media:Muc16 Autotransporterdesign_001.png</A><B>Figure 5</B></DIV><P>The autotransporter system itself is established and functional (see Fig. A<SUP id="cite_ref-10" class="reference"><A href="#cite_note-10">[10]</A></SUP>), and here we &quot;bricked&quot; it down into RFC[10] and RFC[25] BioBricks. At first, we tested different inducible bacterial promoter systems (e.g. &quot;TetR repressed GFP&quot;; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K577893">BBa_K577893</A>) that are available in the parts registry, but when we saw that two of the available systems did not work in our own hands, we decided to built the system on our own. We used the Tet-Repressor generator that was available on the distribution plates (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I739001">BBa_I739001</A>) as we believe that it is very important for the standardization in SynBio to use available parts. Downstream of this repressor generator we designed and fused a BioBrick (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170141">BBa_K2170141</A>) that encodes a double Tet-Operator (Tet-Repressor binding site together with a bacterial promoter and an signal peptide of the outer membrane protein A (OmpA<SUP id="cite_ref-11" class="reference"><A href="#cite_note-11">[11]</A></SUP>) from <I>E. coli</I> that has a 3' RFC[25] restriction site, allowing the protein fusion of proteins that are secreted into the bacterial periplasm when expressed with this BioBrick. As a protein fusion, we then assembled a protein cargo domain that is generally exchangeable for any protein that is so small in size that it can be transported through the outer membrane by the EspP autotransporter. In our project we used the same extracellular domains as for the eukaryotic receptors: enhanced monomeric Avidin (eMA; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170204">BBa_K2170204</A>), single-chain Avidin (scAvidin; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170205">BBa_K2170205</A>) and a composite part composed of a N-terminal Biotinylation acceptor peptide (BAP) and a C-terminal NanoLuciferase (together shown as BAP; ; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170118">BBa_K2170118</A>, see Fig. B). Our design features an A3C5-epitope tag further downstream that can be recognized by a high-affinity Fab-fragment and is generally used to detect and quantify the surface display of the cargo domain<SUP id="cite_ref-12" class="reference"><A href="#cite_note-12">[12]</A></SUP>. Further downstream of this affinity tag we fused the BioBrick for the EspP autotransporter from <I>E. coli</I><SUP id="cite_ref-13" class="reference"><A href="#cite_note-13">[13]</A></SUP><SUP id="cite_ref-14" class="reference"><A href="#cite_note-14">[14]</A></SUP> which is then followed by a bacterial terminator. For the termination we chose again  a well-working terminator from the distribution plate (&quot;double terminator&quot;; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0010">BBa_B0010</A>-<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0012">BBa_B0012</A>). But this was just the DNA-part. What is expected to happen on a protein level? The construct constantly produces Tet-repressor that binds to the Tet-Operator and repressed the promoter activity of the autotransporter gene. As soon as the inducer anhydrotetracycline (aTc) is added to the culture, it binds to the Tet-repressor that can't bind anymore to the Tet-operator and thus the expression of the autotransporter can be regulated. Although the TetR-system is known to be a tight promoter system there is always a certain background expression with most promoters. If the protein expression of the autotransporter is induced using aTc the autotransporter is transcribed and translated. Due to the bacterial signal peptide, the protein is secreted into the bacterial periplasm (the space between the two bacterial membranes of gram-negative bacteria such as <I>E. coli</I>). When present in the bacteria periplasm the autotransporter diffused to the outer membrane and inserts into the membrane. After the integration into the outer membrane, the protein cargo is transported through the beta-barrel of the autotransporter and is presented on the bacterial surface. We are sure that this BioBrick will be a valuable contribution to the Parts Registry as it allows future teams to display small protein domains on the surface of gram-negative bacteria, such as <I>E. coli</I>.
</P></DIV><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><DIV class="white-box"><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Rapoport, T. A. (2007). Protein translocation across the eukaryotic endoplasmic reticulum and bacterial plasma membranes. Nature, 450(7170), 663-669.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Taken from Alberts, B., Bray, D., Hopkin, K., Johnson, A., Lewis, J., Raff, M., ... &amp; Walter, P. (2010). Essential cell biology. Garland Science.</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">Walter, P., Ibrahimi, I., &amp; Blobel, G. U. N. T. E. R. (1981). Translocation of proteins across the endoplasmic reticulum. I. Signal recognition protein (SRP) binds to in-vitro-assembled polysomes synthesizing secretory protein. The Journal of Cell Biology, 91(2), 545-550.</SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text">Keenan, Robert J., et al. &quot;The signal recognition particle.&quot; Annual review of biochemistry 70.1 (2001): 755-775.</SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">↑</A></SPAN><SPAN class="reference-text">Kozak, M. (1989). The scanning model for translation: an update. The Journal of cell biology, 108(2), 229-241.</SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">↑</A></SPAN><SPAN class="reference-text">Kozak, M. (1987). An analysis of 5'-noncoding sequences from 699 vertebrate messenger RNAs. Nucleic acids research, 15(20), 8125-8148.</SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text">Petersen, T. N., Brunak, S., von Heijne, G., &amp; Nielsen, H. (2011). SignalP 4.0: discriminating signal peptides from transmembrane regions. Nature methods, 8(10), 785-786.</SPAN></LI><LI id="cite_note-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-8">↑</A></SPAN><SPAN class="reference-text">Sonnhammer, E. L., Von Heijne, G., &amp; Krogh, A. (1998, July). A hidden Markov model for predicting transmembrane helices in protein sequences. In Ismb (Vol. 6, pp. 175-182).</SPAN></LI><LI id="cite_note-9"><SPAN class="mw-cite-backlink"><A href="#cite_ref-9">↑</A></SPAN><SPAN class="reference-text">Binder, U., Matschiner, G., Theobald, I., &amp; Skerra, A. (2010). High-throughput sorting of an Anticalin library via EspP-mediated functional display on the Escherichia coli cell surface. Journal of molecular biology, 400(4), 783-802.</SPAN></LI><LI id="cite_note-10"><SPAN class="mw-cite-backlink"><A href="#cite_ref-10">↑</A></SPAN><SPAN class="reference-text">Binder, U., Matschiner, G., Theobald, I., &amp; Skerra, A. (2010). High-throughput sorting of an Anticalin library via EspP-mediated functional display on the Escherichia coli cell surface. Journal of molecular biology, 400(4), 783-802.</SPAN></LI><LI id="cite_note-11"><SPAN class="mw-cite-backlink"><A href="#cite_ref-11">↑</A></SPAN><SPAN class="reference-text">Ghrayeb, J., Kimura, H., Takahara, M., Hsiung, H., Masui, Y., &amp; Inouye, M. (1984). Secretion cloning vectors in Escherichia coli. The EMBO journal, 3(10), 2437.</SPAN></LI><LI id="cite_note-12"><SPAN class="mw-cite-backlink"><A href="#cite_ref-12">↑</A></SPAN><SPAN class="reference-text">Costa, J., Grabenhorst, E., Nimtz, M., &amp; Conradt, H. S. (1997). Stable expression of the Golgi form and secretory variants of human fucosyltransferase III from BHK-21 cells Purification and characterization of an engineered truncated form from the culture medium. Journal of Biological Chemistry, 272(17), 11613-11621.</SPAN></LI><LI id="cite_note-13"><SPAN class="mw-cite-backlink"><A href="#cite_ref-13">↑</A></SPAN><SPAN class="reference-text">Barnard, T. J., Dautin, N., Lukacik, P., Bernstein, H. D., &amp; Buchanan, S. K. (2007). Autotransporter structure reveals intra-barrel cleavage followed by conformational changes. Nature structural &amp; molecular biology, 14(12), 1214-1220.</SPAN></LI><LI id="cite_note-14"><SPAN class="mw-cite-backlink"><A href="#cite_ref-14">↑</A></SPAN><SPAN class="reference-text">Skillman, K. M., Barnard, T. J., Peterson, J. H., Ghirlando, R., &amp; Bernstein, H. D. (2005). Efficient secretion of a folded protein domain by a monomeric bacterial autotransporter. Molecular microbiology, 58(4), 945-958.</SPAN></LI></OL></DIV></DIV></DIV><DIV class="row"><FOOTER class="col-sm-9 col-sm-offset-3 noprint page-footer clearfix white-box"><A class="btt visible-xs-block" href="#top"><SPAN class="sr-only">Back to top</SPAN></A><DIV class="row"><DIV class="col-sm-4 social"><H2>Follow us on:</H2></DIV><DIV class="col-sm-4 universities"><H2>LMU &amp; TUM Munich</H2><P>United team from Munich's universities</P></DIV><DIV class="col-sm-4 address"><H2>Contact us:</H2><H3>Address</H3><P>iGEM Team TU-Munich
                     Emil-Erlenmeyer-Forum 5
                     85354 Freising, Germany</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>