<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_LMU-TUM_Munich_Receptors skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:LMU-TUM Munich/Receptors</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="row"><DIV class="col-sm-3 col-xs-8 fixed-sm z-1"><DIV class="sidebar-nav"><DIV class="navbar" role="navigation"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".sidebar-navbar-collapse"><SPAN class="sr-only">Toggle menu</SPAN></BUTTON><SPAN class="visible-xs navbar-brand">Menu</SPAN></DIV><DIV class="navbar-collapse collapse sidebar-navbar-collapse"><UL id="main-menu" class="nav navbar-nav receptors"><LI class="menu_item home"><A href="https://2016.igem.org/Team:LMU-TUM_Munich"><SPAN class="visible-xs-block">Home</SPAN></A></LI><LI class="dropdown menu_item team"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI class="members"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Team">Members</A></LI><LI class="collaborations"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Collaborations">Collaborations</A></LI><LI class="sponsors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Sponsors">Sponsors</A></LI><LI class="press"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Press">Press</A></LI><LI class="gllry"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Gallery">Gallery</A></LI></UL><LI class="dropdown menu_item project"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Project </A></LI><LI class="description"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Description">Description</A></LI><LI class="design"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Design">Design</A></LI><LI class="proteins"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proteins">Proteins</A></LI><LI class="receptors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Receptors">Receptors</A></LI><LI class="linker"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Linkerchemistry">Linker Chemistry</A></LI><LI class="poly"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proof">Polymerization</A></LI><LI class="bioprinting"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Demonstrate">Bioprinting</A></LI><LI class="hypoxia"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hypoxia">Hypoxia</A></LI><LI class="medical"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Medical">Medical Application</A></LI><LI class="interlab"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/InterLab">InterLab</A></LI><LI class="software"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Software">Software</A></LI><LI class="dropdown menu_item notebook"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Notebook </A></LI><LI class="labjournal"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Labjournal">Labjournal</A></LI><LI class="materials"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Materials">Materials</A></LI><LI class="methods"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Methods">Methods</A></LI><LI class="howtos"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Howtos">How Tos</A></LI><LI class="dropdown menu_item parts"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Parts </A></LI><LI class="parts-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Parts">Parts</A></LI><LI class="basic-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Basic_Part">Basic Parts</A></LI><LI class="composite-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Composite_Part">Composite Parts</A></LI><LI class="part-collection"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Part_Collection">Part Collection</A></LI><LI class="menu_item hardware"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hardware">Hardware</A></LI><LI class="menu_item model"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Model">Modeling</A></LI><LI class="menu_item safety"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Safety">Safety</A></LI><LI class="dropdown menu_item human_practices dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Human Practices </A></LI><LI class="human_practices-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Human_Practices">Human Practices</A></LI><LI class="silver"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Silver">Silver</A></LI><LI class="gold"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Gold">Gold</A></LI><LI class="integrated-practices"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Integrated_Practices">Integrated Practices</A></LI><LI class="engagement"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Engagement">Engagement</A></LI><LI class="dropdown menu_item entrepreneur dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Entrepreneurship</A></LI><LI class="entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Entrepreneurship">Entrepreneurship</A></LI><LI class="supporting-entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Supporting_Entrepreneurship">Supporting Entrepreneurship</A></LI><LI class="menu_item attributions"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV class="col-sm-9 col-sm-offset-3"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Functionalized_biotinylated_and_biotin-binding_membrane_proteins"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Functionalized biotinylated and biotin-binding membrane proteins</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Prediction_of_signal_peptide_functionality"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Prediction of signal peptide functionality</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Results_of_signal_peptide_functional_analysis"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Results of signal peptide functional analysis</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Bioinformatic_signal_peptide_analysis_via_the_SignalP_server"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Bioinformatic signal peptide analysis via the SignalP server</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Quantification_of_signal-peptide_mediated_translocation_via_a_secretion_luciferase_assay"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Quantification of signal-peptide mediated translocation via a secretion luciferase assay</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Discussion:_The_choice_of_signal_peptide_was_nailed_down"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Discussion: The choice of signal peptide was nailed down</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Other_receptor_elements:_stability_and_detection"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Other receptor elements: stability and detection</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#EGFR_transmembrane_domain"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">EGFR transmembrane domain</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-9"><A href="#Reporter_proteins_were_included_into_the_design_to_facilitate_later_characterization"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Reporter proteins were included into the design to facilitate later characterization</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#A3C5_epitope_tag"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">A3C5 epitope tag</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#mRuby_3"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">mRuby 3</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Strep-tag_II"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext"><I>Strep</I>-tag II</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Nanoluciferase"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Nanoluciferase</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#CMV_promoter"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">CMV promoter</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#BioBrick-compatible_vectors_for_mammalian_cells:_transient_transfection_.26_stable_integration"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">BioBrick-compatible vectors for mammalian cells: transient transfection &amp; stable integration</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Semi-stable_transfection_of_the_receptors_in_human_cells"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Semi-stable transfection of the receptors in human cells</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#An_everlasting_bond:_creating_a_stable_cell_line"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">An everlasting bond: creating  a stable cell line</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Subcellular_protein_localization_using_fluorescence_microscopy"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Subcellular protein localization using fluorescence microscopy</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#High_resolution_confocal_fluorescence_microscopy"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">High resolution confocal fluorescence microscopy</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Quantification_of_functionalized_membrane_protein_expression_using_flow_cytometry"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Quantification of functionalized membrane protein expression using flow cytometry</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Immunochemical_detecton_of_functionalized_membrane_protein_using_Western_Blot"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Immunochemical detecton of functionalized membrane protein using Western Blot</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Quantification_of_mRNA_expression_levels_by_RT-q-PCR"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Quantification of mRNA expression levels by RT-q-PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#High_resolution_imaging_using_scanning_electron_microscopy_.28SEM.29"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">High resolution imaging using scanning electron microscopy (SEM)</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Design_of_an_autotransporter_construct_for_bacterial_surface_display"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Design of an autotransporter construct for bacterial surface display</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Results:_Testing_the_bacterial_autotransporter_system"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Results: Testing the bacterial autotransporter system</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Discussion:_The_autotransporter_is_working_and_can_present_small_protein_cargo_on_the_bacterial_surface"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">Discussion: The autotransporter is working and can present small protein cargo on the bacterial surface</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#References"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Functionalized_biotinylated_and_biotin-binding_membrane_proteins">Functionalized biotinylated and biotin-binding membrane proteins</SPAN></H1><DIV class="white-box"><DIV class="imagelink float-right"><A href="/wiki/images/b/be/Muc16_Sticker_Receptors_003.png" class="internal" title="Muc16 Sticker Receptors 003.png">Media:Muc16_Sticker_Receptors_003.png</A></DIV><P>Our membrane proteins (also referred to as 'receptors') enable <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Proof">the polymerization of cells</A> into three-dimensional cellular matrices during bioprinting. Therefore, our receptors present biotin (or biotin-binding proteins) on their surfaces, and - being co-injected with a <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Linkerchemistry">biotinylated</A> linker peptide into streptavidin solution - immediately 'glue' to each other by the means of synthetic biology, allowing printing of defined three-dimensional cellular structures.
</P><P>The following extracellular domains were constructed and tested:
1) <B>A biotinylated receptor containing an extracellular biotin acceptor peptide</B> that is endogenously biotinylated by a coexpressed biotin ligase (BirA) and thus presents biotin groups at the cell surface.
</P><P>The biotin acceptor peptide (BAP) is a 15 amino acid peptide sequence originating from <I>E. coli</I>. <SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP> Being biotinylated by the biotin ligase BirA at a lysine residue within the recognition sequence, it mediates the functionality of the receptor by presenting biotin, allowing the interaction of the cell surface with streptavidin in the reservoir solution. 
The biotin ligase BirA is therefore encoded by the same vector as the receptor, with an internal ribosome entry site (IRES) allowing translation of two open reading frames (ORFs) from a single bicistronic mRNA. BirA is targeted to the ER via an Igκ signal peptide and trapped via an ER retention signal (C-terminal KDEL-sequence), allowing it to efficiently biotinylate translocating proteins possessing the recognition sequence.
</P><DIV class="imagelink float-right"><A href="/wiki/images/b/be/Muc16_receptor_schematic_002.png" class="internal" title="Muc16 receptor schematic 002.png">Media:Muc16 receptor schematic_002.png</A><DIV class="caption"><B>Figure 1:</B> A) Schematic depiction of the modular structure of the created genetic receptor constructs. B) Schematic depiction of the modular structure of the membrane receptors for cross-linking of cells via biotin-streptavidin interactions.</DIV></DIV><P>2) <B>Receptors presenting an extracellular avidin variant</B>.
</P><P>These avidin derivates, by design, allow the functional fusion of the otherwise tetrameric avidin molecule with our receptor. Two different variants were used here: The so-called <I>enhanced monomeric avidin</I>, a single subunit avidin that is able to bind biotin as a monomer, and a &quot;single-chain avidin&quot;, which resembles the naturally occuring avidin tetramer, but has the subunits being connected via polypeptide linkers. By linking the C-terminus with the N-terminus of the adjacent monomer, the whole tetravalent avidin has only one N- and one C-terminus and can thus be fused to a type-I membrane protein for surface presentation.<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP></P><P>Both receptor types generally serve the same purpose: By interacting with the complementary component in the printing reservoir, cells are being cross-linked due to the polyvalent binding of streptavidin molecules to polyvalent biotinylated proteins.
</P><P>The <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Design">general, modularized design</A> of our membrane proteins, which we designed and created for both eukaryotic cells (such as HEK cells) and prokaryotic cells (such as <I>E. coli</I>) is in both cases based on the same principle (see Fig. 1). The expression of the membrane protein is - for the prokaryotic as well as for the eukaryotic construct - controlled by a constitutive promoter and secretion into the endoplasmic reticulum or the periplasmic space is enabled by an appropriate signal peptide. The signal peptide is followed by the functional extracellular domain (biotinylated or with biotin-binding properties) via RFC [25], further followed by an antibody binding site that can be used in experiments to quantify the surface presentation of the synthetic membrane protein. The membrane insertion in eukaryotes is enabled by using a simple single-span transmembrane domain, while in eukaroytes a bacterial autotransporter is being used that integrates into the outer membrane of gram-negative bacteria. Via this cross-kingdom compatible modular design, we were not only able to successfully generate cells for our inventive bioprinting approach, but also enable future teams to present protein domains of their respective interest on the surface of both bacterial and mammalian cells for their own application, allowing them to make use of our optimized transmembrane domains and signal sequences.
</P></DIV><H2><SPAN class="mw-headline" id="Prediction_of_signal_peptide_functionality">Prediction of signal peptide functionality</SPAN></H2><DIV class="white-box"><P>For determining a signal peptide suitable for our construct, we tested a total of three signal peptides - one taken from the parts registry, and two designed by ourselves.
</P><UL><LI> The <B>EGFR signal peptide</B> was taken from the iGEM parts registry (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K157001:Design">BBa_K157001</A>) and combined with a BioBrick encoding the CMV promoter sequence (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K747096">BBa_K747096</A>) via the RFC10 cloning standard.</LI></UL><UL><LI> The <B>BM40 and Igκ signal peptides</B> were designed by us, providing additional spacing as well as a Kozak sequence to the 5'-UTR of the constructs and combined with a BioBrick encoding the CMV promoter sequence (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K747096">BBa_K747096</A>) via the RFC10 cloning standard. </LI></UL><P>
As shown below, the EGFR signal peptide taken from the parts registry is constructed in a way that the signal peptide ORF immediately follows the RFC10 cloning scar after the CMV promoter, thus resulting in a very short 5’ untranslated region (UTR) of the transcribed mRNA. The combination of the CMV promoter with the synthesized BM40 and Igκ signal peptides allows for a considerably longer 5’ UTR of the resulting mRNA - additionally allowing them to contain a full Kozak consensus sequence by design. The Kozak sequence is recognized by the ribosome as a translational start site; this element missing or deviating from the consensus sequence may considerably decrease translation efficiency.<SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP> For the EGFR signal peptide construct, a Kozak sequence is not present, as it would have to preceed the start codon ATG - a position which is occupied by the RFC10 cloning scar.
Since both, the distance from the promoter to the open reading frame as well as the Kozak consensus sequence are considered crucial parameters for expression levels<SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP>, BM40 and Igκ signal peptide constructs are likely to resut in increased expression levels compared to proteins containing the EGFR signal peptide from the parts registry.
</P><P>As the functionality of the signal peptides had to be determined before including one of these into the final receptor constructs, the three options (Igκ, BM40, EGFR) were tested via bioinformatic tools as well as by conducting a secretion assay of luciferase constructs.
</P></DIV><H2><SPAN class="mw-headline" id="Results_of_signal_peptide_functional_analysis">Results of signal peptide functional analysis</SPAN></H2><H3><SPAN class="mw-headline" id="Bioinformatic_signal_peptide_analysis_via_the_SignalP_server">Bioinformatic signal peptide analysis via the SignalP server</SPAN></H3><DIV class="white-box"><P>The <A rel="nofollow" class="external text" href="http://www.cbs.dtu.dk/services/SignalP">SignalP</A> server, being able to discriminate between the hydrophobic signal peptide sequence and the hydrophobic transmembrane domain<SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP>, can determine whether a sequence possesses the biophysical requirements for functioning as a signal peptide. Furthermore, it is able to predict the probable cleavage site of the signal peptide after its translocation into the ER. For all constructs, signal peptide functionality is predicted, as well as a potential cleavage site. The complete translated amino acid sequences of the respective receptor constructs were used as input. The algorithm for eukaryotes with default D-cutoff value was chosen.
</P><DIV class="imagelink float-right"><A href="/wiki/images/3/3d/Muc16_SignalPeptideprediction_002.png" class="internal" title="Muc16 SignalPeptideprediction 002.png">Media:Muc16 SignalPeptideprediction_002.png</A><DIV class="caption"><B>Figure 2</B>: A) Schematic depiction of a protein-coding mRNA containing a 5'- and 3'-UTR, the first of which being an important factor in translation efficiency. B) Identification of signal peptide functionality and cleavage site within the receptor construct containing the EGFR, Igκ and BM40 signal peptide via the SignalP 4.1 server. Plotted probabilities represent the raw cleavage site “C-score”, the signal peptide “S-score” and the combined cleavage site “Y-score”. C) ClustalW multiple  sequence alignment of the BM40, EGFR and Igκ signal peptide construct, depicting a gap for the EGFR-signal peptide construct compared to the newly designed ones, indicating smaller length of the 5'-UTR . For the Igκ and BM40 signal peptide constructs, a T7 promoter spacer as well as a Kozak sequence increase the distance between TATA-box and protein-coding region.</DIV></DIV><P>
For all three signal peptides, the high S-score indicates high signal peptide functionality. For the Igκ signal peptide, a cleavage site between amino acid 20 and 21 within the signal peptide is predicted. For the BM40 signal peptide, a cleavage site between amino acid 17 and 18 is predicted, and for the EGFR signal peptide, a cleavage site between amino acid 24 and 25 is predicted.
</P></DIV><H3><SPAN class="mw-headline" id="Quantification_of_signal-peptide_mediated_translocation_via_a_secretion_luciferase_assay">Quantification of signal-peptide mediated translocation via a secretion luciferase assay</SPAN></H3><DIV class="white-box"><P>For the quantification of signal peptide functionality via a luciferase assay, three constructs were created and tested - each containing one of three signal peptides (the EGFR signal peptide, the Igκ signal peptide or the BM40 signal peptide) and a nanoluciferase as well as a CMV promoter, a <I>Strep</I>-tag II for immunochemical detection and the hGH polyadenylation signal sequence. Not containing a transmembrane domain, the nanoluciferase fusion protein is being translocated into the ER and then secreted into the medium. Using a luciferase assay, one can quantifiy the amount of luminescence - and thus, proportionally, the amount of secreted luciferase - by measuring the conversion of luciferin into visible light and integrating it over a timespan of 5 s. Therefore, medium samples were taken every 12 h after transfection of cells and measured via the Promega NanoGlo® luciferase assay system according to the manufacturer's instructions. </P><DIV class="imagelink float-right"><A href="/wiki/images/0/03/Rsz_receptor_signalpeptideresults_003.png" class="internal" title="Rsz receptor signalpeptideresults 003.png">Media:Rsz receptor signalpeptideresults 003.png</A><DIV class="caption"><B>Figure 3:</B> A) Schematic depiction of the genetic constructs used for signal peptide testing via a secretion luciferase assay. B) Results of the luciferase assay, showing luminescence in relative luminescence units (RLU) as a function of time for the previously described three different signal peptide constructs as well as a control construct containing no signal peptide. C) Detection of secreted luciferases in the medium via a Western Blot, using an anti-<I>Strep</I>-tag II antibody as well as an alkaline phosphatase-coupled secondary antibody for detection.</DIV></DIV><H3><SPAN class="mw-headline" id="Discussion:_The_choice_of_signal_peptide_was_nailed_down">Discussion: The choice of signal peptide was nailed down</SPAN></H3><P>As would be expected, the EGFR signal peptide construct showed the lowest level of secretion. Although one might think this could also be attributed to a lower functionality of the signal peptide sequence itself after being translated, the lower secretion level is more likely to stem from lower translation initiation levels. Almost completely missing a 5’ UTR as well as a Kozak sequence, slower translation initiation (being a rate-limiting step) is likely to have lowered the level of translated protein in total. The constructs containing the Igκ and the BM40 signal peptide, on the other hand, contain a longer 5’ UTR as well as the Kozak consensus sequence, and thus result in higher secretion levels. Since the BM40 signal peptide hereby showed the highest secretion levels, it was thus incorporated into the receptor constructs.
</P></DIV><H3><SPAN class="mw-headline" id="Other_receptor_elements:_stability_and_detection">Other receptor elements: stability and detection</SPAN></H3><DIV class="white-box"><P>Apart from the functional parts that mediate the (strept)avidin-biotin interaction, other elements in the receptor were designed to make sure it is optimally translocated to the membrane, as stable as possible, and easily detectable. 
</P></DIV><H3><SPAN class="mw-headline" id="EGFR_transmembrane_domain">EGFR transmembrane domain</SPAN></H3><DIV class="white-box"><P>For anchoring in the membrane, we decided to use a type-I membrane protein, which has a single membrane span with defined localization of N- and C-terminus out- and inside of the cell, respectively. As the Epidermal Growth Factor Receptor (EGFR) has already been successfully used in the iGEM project of Freiburg 2008, we decided to base our project on the same transmembrane domain. The N-terminal transmembrane α-helix of human EGFR (<A rel="nofollow" class="external text" href="http://www.uniprot.org/uniprot/P00533">UniProt P00533</A>, amino acids 622-653) was herefore chosen (see below the topology prediction of EGFR via the <A rel="nofollow" class="external text" href="http://www.cbs.dtu.dk/services/TMHMM/">TMHMM 2.0 server for the prediction of transmembrane helices</A><SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP>). A stop-transfer sequence, that is also present in the endogeneous EGFR, consisting of clustered, positively charged amino acids - being characteristic for type I membrane proteins - was added at the C-terminus, and the sequence was furthermore flanked by a (GGGGC)<SUB>2</SUB>-linker at the N- and C-terminus, respectively. The addition of a stop-transfer sequence as in the naturally occuring EGFR sequence, as well as the addition of flexible linkers, makes our EGFR-TMD an improvement over the BioBrick <A rel="nofollow" class="external text" href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K157002">BBa_K157002</A>, which does not possess either of these elements.
</P><DIV class="imagelink float-right"><A href="/wiki/images/0/05/Receptor_TMD_002.png" class="internal" title="Receptor TMD 002.png">Media:Receptor TMD 002.png</A><DIV class="caption"><B>Figure 4:</B> A) Multiple sequence alignment of human EGFR (see <A rel="nofollow" class="external text" href="http://www.uniprot.org/uniprot/P00533">UniProt P00533</A>), the EGFR transmembrane domain BioBrick from the registry (<A rel="nofollow" class="external text" href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K157002">BBa_K157002</A> and the extended EGFR transmembrane designed by us, containing a charged stop-transfer domain as well as linker elements at the C- and N-termini. B) Transmembrane domain prediction by the <A rel="nofollow" class="external text" href="http://www.cbs.dtu.dk/services/TMHMM/">TMHMM 2.0 server</A>, indicating positioning of N- and C-termini out- and inside of the cell, respectively, as well as predicting functionality of transmembrane regions. C) Schematic depiction of the C-terminal stop-transfer sequence of the receptor.</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Reporter_proteins_were_included_into_the_design_to_facilitate_later_characterization">Reporter proteins were included into the design to facilitate later characterization</SPAN></H2><DIV class="white-box"><P>Moreover, the receptor contains <B>three functional elements for its detection</B>: The intracellularly located red fluorescent protein mRuby 3 for detection of the receptor via fluorescence microscopy, an extracellular epitope domain for immunochemical detection via A3C5-antibody fragments and an intracellular <I>Strep</I>-tag II for the detection and purification via immunochemical methods.
</P><P>The vector furthermore contains the poly-adenylation signal of human growth hormone (hGH) for functional polyadenylation of the transcribed mRNA. A description for the respective functional elements is given in the following sections.
</P></DIV><H3><SPAN class="mw-headline" id="A3C5_epitope_tag">A3C5 epitope tag</SPAN></H3><DIV class="white-box"><P>Antibodies have various areas of application in life science, including protein detection, pulldown experiments and even immunotherapy. Having originally been discovered in 1995 during an <I>in vitro</I> screening for antibodies against cytomegaloviral proteins<SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP>, the A3C5 antibody is an established molecular tool for specific recognition of proteins. By tagging cell surface proteins with an epitope specifically recognized by A3C5 (being a peptide sequence of 11 amino acids), one is easily able to detect tagged proteins via immunofluorescence microscopy, FACS or one may purify them via pulldown experiments. Through the latter, one may also screen for <I>in vivo</I> interaction partners of the tagged protein.
</P></DIV><H3><SPAN class="mw-headline" id="mRuby_3">mRuby 3</SPAN></H3><DIV class="white-box"><P>Having originally been engineered as a monomeric form of the red fluorescent protein eqFP611, mRuby variants are some of the brightest red fluorescent proteins available. <SUP id="cite_ref-8" class="reference"><A href="#cite_note-8">[8]</A></SUP> With an excitation maximum at a wavelength of 558 nm and an emission wavelength maximum of 605 nm, the resulting stokes shift of 57 nm makes mRuby a good choice for fluorescent microscopy imaging and, thus, allows the visualization of the receptor in its cellular environment by being fused to the C-terminus of the transmembrane domain. mRuby is especially powerful for the fusion with receptors - not only due to its small size and high stability, but also due to the fact that, in comparison with eGFP, it appears up to 10 times brighter in membrane enviroments. The mRuby variant used here is mRuby 3, having been engineered for even more improved brightness and photostability.<SUP id="cite_ref-9" class="reference"><A href="#cite_note-9">[9]</A></SUP></P></DIV><H3><SPAN class="mw-headline" id="Strep-tag_II"><I>Strep</I>-tag II</SPAN></H3><DIV class="white-box"><P>The <I>Strep</I>-tag II is an eight amino acid peptide sequence (WSHPQFEK) that specifically interacts with streptavidin and can thus be used for easy one-step purification of the receptor via biochemical and immunochemical methods.<SUP id="cite_ref-10" class="reference"><A href="#cite_note-10">[10]</A></SUP></P></DIV><H3><SPAN class="mw-headline" id="Nanoluciferase">Nanoluciferase</SPAN></H3><DIV class="white-box"><P>The concept of bioluminescence has a great meaning for several invertebrate species, allowing communication with other individuals, signaling receptiveness or deterring predators.<SUP id="cite_ref-11" class="reference"><A href="#cite_note-11">[11]</A></SUP> The most common enzyme responsible for the creation of bioluminescence are the luciferases (from the latin words ’lux’ and ’ferre’, meaning ’light-carrier’). 
These enzymes, among others found in fireflies and deep-sea shrimp, commonly consume a substrate called luciferin as well as energy in the form of ATP or reduction equivalents in order to create light-emission through oxidation. The underlying mechanism hereby relies on the creation of an instable, highly excited intermediate under the consumption of energy. This high-energy intermediate then spontaneously falls back into a state of lower energy, emitting the energetical difference as visible light. While mainly being used as a folded, stabilizing element at the N-terminus for the construct containing the biotin acceptor peptide, the nanoluciferase could also be used for advanced binding studies via luciferase assays.<SUP id="cite_ref-12" class="reference"><A href="#cite_note-12">[12]</A></SUP></P><DIV class="imagelink float-right"><A href="/wiki/images/4/48/Muc16_Nanoluc_Mech.png" class="internal" title="Muc16 Nanoluc Mech.png">Media:Muc16 Nanoluc Mech.png</A><DIV class="caption"><B>Figure 5:</B> Reaction scheme of the reaction catalyzed by the NanoLuc nanoluciferase. Taken from the [<A rel="nofollow" class="external free" href="https://www.promega.com/-/media/files/resources/protocols/technical-manuals/101/nanoglo-luciferase-assay-system-protocol.pdf">https://www.promega.com/-/media/files/resources/protocols/technical-manuals/101/nanoglo-luciferase-assay-system-protocol.pdf</A> NanoGlo Assay Kit manual.</DIV></DIV><P>Luciferases have also found their way into biotechnological applications.<SUP id="cite_ref-13" class="reference"><A href="#cite_note-13">[13]</A></SUP> The simplistic concept of creating visible (and thus easily measurable) light makes luciferases ideal reporters for the expression of secretable proteins via a corresponding assay. This project also made use of a luciferase for the determination of expression levels of the BAP surface receptor while providing a stable and folded domain at the N-terminus, as well as having been used for the choice of signal peptide. The monomeric luciferase used in this project, the so-called ’NanoLuc™'<SUP id="cite_ref-14" class="reference"><A href="#cite_note-14">[14]</A></SUP> (due to terms of simplicity referred to as ’nanoluciferase’ in the following), can be fused to other proteins in order to make their expression visible and quantifiable. This engineered luciferase emits a steady, easily detectable glow when exposed to its substrate, while being only 19 kDa large and being brighter, more specific and steadier than other luciferases commonly found in nature. Using luciferases for the quantification of expression levels offers several advantages over fluorescent proteins, including higher sensitivity and smaller size.
</P></DIV><H3><SPAN class="mw-headline" id="CMV_promoter">CMV promoter</SPAN></H3><DIV class="white-box"><P>Last but not least, high expression levels of the receptor, so that a high avidity for the interaction may be reached. The CMV promoter (taken from the parts registry, <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K74709">BBa_K74709</A>) therefore enables constitutively high expression of the receptor - and, in the case of the BAP-construct, the biotin ligase BirA.
</P></DIV><H1><SPAN class="mw-headline" id="BioBrick-compatible_vectors_for_mammalian_cells:_transient_transfection_.26_stable_integration">BioBrick-compatible vectors for mammalian cells: transient transfection &amp; stable integration</SPAN></H1><DIV class="white-box"><P>We wanted to generate semi-stable transfected cells for initial testing and stably integrated cells for final experiments and the later bioprinting experiments. As there was the need for such BioBrick-compatible vectors, we adapted the pDSG vector system (semi-stable, transient: <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170300">BBa_K2170300</A>) and the pcDNA5 vector system (stable integration using a co-transfected rekombinase: <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170301">BBa_K2170301</A>).
</P></DIV><H2><SPAN class="mw-headline" id="Semi-stable_transfection_of_the_receptors_in_human_cells">Semi-stable transfection of the receptors in human cells</SPAN></H2><DIV class="white-box"><P>For the first trials of testing our constructs in human cells, we chose the MEXi™ cell line for semi-stable transfection. 
MEXi™ is a suspension HEK293E cell line that encodes the viral protein EBNA1 in it's genome. EBNA1 is a protein derived from Eppstein-Barr-Virus that mediates the viruses' replication initiation, gene regulation and maintenance of its DNA in the host cell. The second part of the MEXi™ system is made up by the pDSG-IBA vector, which carries the specific origin of replication that is recognized by EBNA1. EBNA1 binding to this origins results in the recruitment of the replication apparatus as well as inducing the maintenance of the inserted DNA in the host's nucleus, leading to episomal maintenance and co-replication of the plasmid during every division cycle.<SUP id="cite_ref-15" class="reference"><A href="#cite_note-15">[15]</A></SUP></P><P>Thus, our receptor constructs were cloned into the pDSG-iGEM vector and used for transfection of MEXi cells using the PEI transfection method (see methods). Polyethylene imin (PEI) as a cationic polymer forms complexes with the negatively charged DNA. These complexes are taken up by the cell via endocytosis and carried to the nucleus. <SUP id="cite_ref-16" class="reference"><A href="#cite_note-16">[16]</A></SUP></P><DIV class="imagelink float-right"><A href="/wiki/images/b/bd/MUC16_pDSG_Rezeptoren.png" class="internal" title="MUC16 pDSG Rezeptoren.png">Media:MUC16_pDSG_Rezeptoren.png</A><DIV class="caption"><B>Figure 6:</B> Schematical depiction of the pDSG-Iba plasmids, which were used for semi-stable transfection of HEK293E MEXi™ cells.</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="An_everlasting_bond:_creating_a_stable_cell_line">An everlasting bond: creating  a stable cell line</SPAN></H2><DIV class="white-box"><DIV class="imagelink float-right"><A href="/wiki/images/d/d7/MUC16_stable_integration.png" class="internal" title="MUC16 stable integration.png">Media:MUC16_stable_integration.png</A><DIV class="caption"><B>Figure 7:</B> Schematical overview over the workflow undergone using the Flp-FRT system for the generation of cell lines for the stable expression of the membrane proteins.</DIV></DIV><P>Printing tissues - or even only parts of organs - requires billions and bilions of cells. An average liver, for example, weighs about three pounds and hosts ~10<SUP>8</SUP> cells per gram tissue. <SUP id="cite_ref-17" class="reference"><A href="#cite_note-17">[17]</A></SUP><SUP id="cite_ref-18" class="reference"><A href="#cite_note-18">[18]</A></SUP> 
Since transiently and even semi-stably transfected cells do not propagate the transfected DNA indefinitely, repeated transfection of cells would be unwise due to the high consumption of time and ressources. Therefore, we created three stable cell lines that chromosomally encode the respective receptor constructs.
For this purpose we used the Flp™-System by invitrogen. The three compound system consists of HEK-TRex™-293 cells that harbour one chromosomal Flp recombination target (FRT) site, a Flp recombinase expression plasmid and the plasmid encoding the receptor as well as another FRT site for stable integration.<SUP id="cite_ref-19" class="reference"><A href="#cite_note-19">[19]</A></SUP> The detailed principle of the integration is described in the figure below.

Cells that succesfully integrated the plasmid into their genome will lose their resistance for Zeocin™, but become resistant to hygromycin B (hygB) as the plasmid encodes the hygB-phosphotransferase, a kinase inactivating hygB by phophorylation.<SUP id="cite_ref-20" class="reference"><A href="#cite_note-20">[20]</A></SUP>
Thus, successfully transfected cells can be easily selected via hygromycin B. To ensure that the resistence does not come from non-integrated plasmid, this process takes several weeks and divison cycles.
</P><DIV class="imagelink float-right"><A href="/wiki/images/3/3b/MUC16_pcDNA5_Rezeptoren.png" class="internal" title="MUC16 pcDNA5 Rezeptoren.png">Media:MUC16 pcDNA5 Rezeptoren.png</A><DIV class="caption"><B>Fig.8</B> Schematical depiction of the pcDNA5 vectors (invitrogen) used for the generation of cell lines for the stable expression of our designed biotin-presenting or -binding membrane proteins.</DIV></DIV><DIV class="imagelink float-right"><A href="/wiki/images/2/29/MUC16_GenomPCR.png" class="internal" title="MUC16 GenomPCR.png">Media:MUC16 GenomPCR.png</A></DIV><P>The succesful integration of the pcDNA plasmids was verified by genome-PCR (see methods). Therefore, we designed three primers: One that binds to the SV40 promotor and part of the following FRT site, one that binds part the hygromycin restistance gene and another one for the <I>lac</I>Z-Zeocin sequence. In the event of stable integration, the size of the amplicons will be around 550 bp, according to the distance between the SV40 and the hygromycine resistance gene. Wheareas in case of no integration the amplicon size will be around 400 since this corresponds to the distance between the SV40 and the <I>lac</I>Z-Zeocin sequences. 
</P><P>The results are shown in figure on the right. The lower band visible in the samples 2-5 leads us to the conslusion that, despite hygromycine selection, the cells are not yet a monoculture - something we also observed in the flow cytometry analysis further discussed below. On the other hand, a high portion of cells do have the construct integrated into their genome. 
</P><DIV class="imagelink float-left"><A href="/wiki/images/0/06/MUC16_GenomPCR_scheme.png" class="internal" title="MUC16 GenomPCR scheme.png">Media:MUC16_GenomPCR_scheme.png</A><DIV class="caption"><B>Figure 9:</B> Genome PCR workflow for the verification of the stable, genomic integration of the desired protein sequence.</DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Subcellular_protein_localization_using_fluorescence_microscopy">Subcellular protein localization using fluorescence microscopy</SPAN></H1><DIV class="white-box"><DIV class="imagelink float-right"><A href="/wiki/images/9/90/MUC16_Microscopy_scAvidin_001.png" class="internal" title="MUC16 Microscopy scAvidin 001.png">Media:MUC16 Microscopy scAvidin 001.png</A><DIV class="caption"><B>Figure 10:</B>  Fluorescence microscopy images of HEK293 cells during the process of stable integration and subsequent selection. Red fluorescence stems from the C-terminal mRuby3 of the receptor construct.</DIV></DIV><P>
For the traceablitiy of receptor expression, we supplied all of our receptor constructs with means for detection via fluorescence (mRuby, epitope tags) and/or luciferase assays.
mRuby is hereby located on the C-terminus at the cytosolic end of our transmembrane receptors. Cells that express and fold the protein properly should thus give a signal of 605 nm (<I>red</I>) when exited at around 558 nm.<SUP id="cite_ref-21" class="reference"><A href="#cite_note-21">[21]</A></SUP> We tested our contructs three diffrent cell types: HEK293T adherent cells that were stably transfected, HEK293E suspension cells that were semi-stably transfected and N2a cells, a mouse neuroblastoma cell line, that we transiently transfected. 
</P><P>As you can see on the images the signal developed over time. In the beginning, before selection, the protein was visible in little dots and therefore seemed to be localized more in subcellular organelles than in the membrane. 
During selection via hygromycin B, most of the cells died, leaving some resistant cells behind which over time grew to bigger foci consisting of stably transfected cells only (see Figure 10). For the BAP-receptor, hardly any fluorescence was detectable - neither in the semi-stable nor in the stable transfection approach. in the latter case, cells still proliferated in the presence of hygromycin B and formed foci, which would confirm successful chromosomal intergration (as already shown via PCR), hinting at some problem on the level of transcription, translation or protein maturation. 
</P><P>After two weeks of selection on hygromycin B, the foci were harvested and expanded. The cells seemed to recover quickly, and moreover the red dots nearly disappeared leaving only the membrane of the scA-receptor cells illuminated by mRuby. 
</P></DIV><H1><SPAN class="mw-headline" id="High_resolution_confocal_fluorescence_microscopy">High resolution confocal fluorescence microscopy</SPAN></H1><DIV class="white-box"><P>In order to obtain high resolution confocal fluorescence images from our receptor constructs, Neuro-2a (N2a) mouse neuroblastoma cells were transfected with the respective plasmids according to standard protocols. Shortly before the end of the project, Christoph came up with the idea to include a trafficking signal to the C-terminus of the receptor proteins that is described to increase the surface presentation of receptors by sorting them intracellularly towards the cell membrane, the sequence being KSRITSEGEYIPLDQIDINV GG FCYENEV* (consisting of the trafficking signal, a two amino acid linker, an ER export signal and the stop codon, DNA sequence aagagcaggatcaccagcgaggg cgagtacatccccctggaccagatcgacatcaacgtg ggcgg cttctgctacgagaacgaggtgtaa)<SUP id="cite_ref-22" class="reference"><A href="#cite_note-22">[22]</A></SUP>. The sequence was inserted into the previously assembled receptor BioBrick using PCR-based cloning. After the transfection of the receptor plasmids, cells were stained with the following dyes for confocal fluorescence microscopy: Hoechst dye for the nuclear staining at 405 nm; Alexa488-bioting to stain biotin binding proteins; mRuby3 which is a part of the receptor and is expected to show a membrane localization and Dy634-antiA3C5-F<SUB>ab</SUB>. Staining was followed by three washing steps with PBS. Subsequently, cells were mounted on a confocal fluorescence microscope (Leica). The results for each fluorescence channel are given in figure 11.
</P><DIV class="imagelink float-right"><A href="/wiki/images/e/e1/MUC16_confocal_001.png" class="internal" title="MUC16 confocal 001.png">Media:MUC16 confocal 001.png</A><DIV class="caption"><B>Figure 11:</B> Confocal flurorescence microscopy images, including stains for the nucleus, biotin-binding proteins, intrinsic mRuby fluorescence and stains of the A3C5 epitope tag.</DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Quantification_of_functionalized_membrane_protein_expression_using_flow_cytometry">Quantification of functionalized membrane protein expression using flow cytometry</SPAN></H1><DIV class="imagelink float-right"><A href="/wiki/images/d/d8/MUC16_FACS_TRex_001.png" class="internal" title="MUC16 FACS TRex 001.png">Media:MUC16 FACS TRex 001.png</A><DIV class="caption"><B>Figure 11:</B> Flow cytometry data of stably integrated cells. Intrinsic mRuby fluorescence was measured as well as binding of a A3C5-tag specific Fab and of Biotin to single chain Avidin and enhanced monomeric Avidin.</DIV></DIV><DIV class="white-box"><P>To confirm proper presentation of the functional domains of the receptors on the cell surface, and in order to quantify the yield of stably transfected cells we used flow cytometry analysis. 
The results presented in figure 11 hereby show a positive signal in the RFP channel for cells expressing the scAvidin and the eMA receptor. As already observed in fluorescence microscopy, mRuby-fluorescence could not be detected for the BAP-receptor cells. Still - surprisingly - about 40% were positive for staining of the extracellular A3C5 epitope tag, allowing the conclusion that this epitope must be functionally presented on the cell. Furthermore, a luciferase assay utilizing the extracellularly presented nanoluciferase confirmed correct topology of the luciferase of at least a part of the receptor (data not shown). These results may suggest a problem during folding of the mRuby beta-barrel - leading to a loss of fluorescence despite of correct localization and topology of the receptor on the cell surface. 
For scA and eMA cells Dy634 signal could furthermore be detected, confirming correct localization and topology. 
Another stain was the eGFP coupled with biotin giving a positive signal for scA and eMA to which the biotin binds. For obvious reasons, cells expressing the BAP receptor - and thus presenting biotin - did not show any signal.
</P></DIV><H1><SPAN class="mw-headline" id="Immunochemical_detecton_of_functionalized_membrane_protein_using_Western_Blot">Immunochemical detecton of functionalized membrane protein using Western Blot</SPAN></H1><DIV class="white-box"><DIV class="imagelink float-right"><A href="/wiki/images/6/6e/Muc16_Receptor_WB_001.png" class="internal" title="Muc16 Receptor WB 001.png">Media:Muc16 Receptor WB 001.png</A><DIV class="caption"><B>Figure 12:</B> Western Blotting results, using a primary anti-<I>Strep</I>-tag antibody and a secondary HRP-coupled antibody.</DIV></DIV><P>Another way to verify the succesful transfection and protein expression is given by the two epitope tags (A3C5 and <I>Strep</I>-tag II) that are included in the receptor design, which allow detection via Western Blot analysis.
After transfecting mammalian cells with the receptor constructs and selecting them for 2 weeks on hygromycin, we lysed the cells and checked for the presence of <I>Strep</I>- and A3C5-tag via western blot using primary antibodies (or a F<SUB>ab</SUB> for A3C5) against the respective tags and secondary alkaline phosphatase-coupled antibodies (see methods). As the figure shows, the <I>Strep</I>-tag wasn't detectable via Western blotting - however, the A3C5-tag-WB shows significant singular bands at the size of our proteins in the respective samples.
This data corresponds with the flow cytometry results again showing that the BAP receptor is also present in the non-fluorescent cells. 
</P><TABLE><CAPTION> Properties of membrane proteins
</CAPTION><TBODY><TR><TD>Receptor
</TD><TD>BioBrick
</TD><TD>Amino acids
</TD><TD>Molecular mass [Da]
</TD></TR><TR><TD>BAP-Receptor
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170000">BBa_K2170000</A></TD><TD>507
</TD><TD>55 440
</TD></TR><TR><TD>eMA-Receptor
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170001">BBa_K2170001</A></TD><TD>448
</TD><TD>48 618
</TD></TR><TR><TD>scAvidin-Receptor
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170002">BBa_K2170002</A></TD><TD>877
</TD><TD>95 028
</TD></TR></TBODY></TABLE></DIV><H1><SPAN class="mw-headline" id="Quantification_of_mRNA_expression_levels_by_RT-q-PCR">Quantification of mRNA expression levels by RT-q-PCR</SPAN></H1><DIV class="white-box"><P>The characterization of our transfected mammalian cells showed that the two receptors containing an avidin variant are correctly expressed and translocated to the cell membrane. The biotin-presenting receptor containing the biotin-acceptor peptide, though, showed a very low fluorescence signal. As the success of the genomic integration was validated via PCR, the conclusion was made that a problem must have occurred downstream of the integration. Two likely possibilities came to mind - either a problem on the level of mRNA stability, or a problem on the level of protein folding and/or stability. In order to understand where the source of this problem was, we took a closer look at the expression levels of the different receptors by analyzing mRNA levels in comparison to each other and to the housekeeping gene (HKG) GAPDH. 
</P><P>For a better comparibility, we used the same amount of cells for each receptor and not only reviewed cells with stably integrated but also cells with the episomal stable receptor constucts. After extraction of the total mRNA from the cells, the quality and integrity was measured by a Bioanalyzer device. After making sure that all samples showed an <B>RNA integrity number (RIN) above 9</B>, indicating purity of the total mRNA, the total mRNA was converted to cDNA by a reverse transcriptase, which in a further step was then used in quantitative PCR to quantify the amount of a specific mRNA produced by the cells. For this reason two primer pairs were designed. One of them was designed to bind the exact same region in all of the three different receptor mRNAs, amplifying the same part of the receptor, thus allowing comparison. In addition, a second pair was designed to bind only to the biotin-presenting receptor, amplifying a part that is unique to it. Via this method, we wanted to observe if only one part of the mRNA is unstable or if the general mRNA level of this receptor is decreased.
</P><P>For the evaluation of the raw data we used the <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Methods#.CE.94.CE.94CP_method">ΔΔC<SUB>P</SUB> method</A><SUP id="cite_ref-23" class="reference"><A href="#cite_note-23">[23]</A></SUP> in which the crossing points (C<SUB>P</SUB> values) of a gene of interest and of a reference gene are compared and normalized with a constantly expressed HKG, to get an idea of the relative amounts of the mRNA of interest.
</P><P>In the following you can see the results of our experiments. We compared the expression levels of T-REx cells which were transfected with our different receptor constructs and with non-transfected (NT) T-REx cells.
</P><DIV class="imagelink float-right"><A href="/wiki/images/6/67/Muc16_Primer_qPCR_002.png" class="internal" title="Muc16 Primer qPCR 002.png">Media:Muc16_Primer_qPCR_002.png</A><DIV class="caption"><B>Figure 12:</B> Workflow and results of the RT-qPCR for the quantification of mRNA levels for cells expressing the eMA, scAvidin and BAP receptors.</DIV></DIV><P>As can be seen the ΔΔC<SUB>P</SUB> values of the mRNA of the eMA and scAvidin constructs are with a <B>1,2 x  10<SUP>6</SUP> fold increase</B> for scAvidin and <B>1.5 x 10<SUP>3</SUP> fold increase</B> for eMA, significantly higher than those of the BAP construct. The analysis of the different primer pairs in the two regions of the BAP construct also showed similar values and thus confirms the reliability of the measurement. A difference in C<SUB>P</SUB> value is tantamount to relative mRNA differences between cells.
</P><P>Thus, we can conclude that the assumption that the missing RFP signal in cells expressing the BAP construct is caused by a <B>reduced mRNA stability</B>, is indeed correct.
</P></DIV><H1><SPAN class="mw-headline" id="High_resolution_imaging_using_scanning_electron_microscopy_.28SEM.29">High resolution imaging using scanning electron microscopy (SEM)</SPAN></H1><DIV class="white-box"><DIV class="imagelink float-right"><A href="/wiki/images/d/de/Muc16_Receptor_SEM_002.png" class="internal" title="Muc16 Receptor SEM 002.png">Media:Muc16 Receptor_SEM_002.png</A><DIV class="caption"><B>Figure 13:</B> Scanning electron images of cells expressing the BAP receptor construct, being interconnected by an artificial extra-cellular matrix of streptavidin-coated beads.</DIV></DIV><P>As we wanted to obtain high resolution images from the transgenic cells transfected with our modular membrane proteins we contacted one of the experts on electron microscopy in Munich and luckily Prof. em. Dr. Wanner (LMU Munich) agreed to support our project by contributing scanning electron microscopic (SEM) imaging. Even better suitable for our purpose Prof. Wanner has developed a method in which he can correlate the fluorescence signal of a sample with the high resolution images obtained in SEM. As this kind of sample preparation, sample processing and imaging takes a while, we had to start 2 weeks before wiki freeze and at that point we did not have enough stably integrated T-REx 2cells, yet. Thus we used transient transfected, semi-stable MEXi HEK 293T cells that were pulled down with magnetic beads in order to increase the number of transgenic cells in the preparation and to show the specific binding of the beads to the cells. For this immunoprecipitation we incubated scAvidin-receptor and eMA-receptor cells with biotinylated magnetic beads and on the other side cells transfected with the BAP-receptor together with streptavidin-coupled magnetic beads. After the immunoprecipitation the cells were washed twice with PBS and it was visible that the portion of transgenic (red fluorescent) cells was largely increased after this procedure. The depicted SEM image shows two BAP-receptor transfected MEXi cells which have a cluster of magnetic streptavidin-conjugated magnetic beads at their cellular interface. At the border of the cells the superimposed fluorescence image of these cells shows red fluorescence located at the periphery of the cells indicating the presence of mRuby3 fluorescent protein at the cellular membrane. The green fluorescence is caused by biotinylated eGFP that was used for other preparations on the same microscopic slide and came into contact with these cells when the suspension cells were sedimented on the microscopic slide by centrifugation. Although for a reliable conclusion one would have to evaluate several such images this figure already gives a valuable evidence that the BAP-receptor is present on the cells (although perhaps at lower levels) and can bind to strepavidin that was conjugated to the beads.
</P></DIV><H1><SPAN class="mw-headline" id="Design_of_an_autotransporter_construct_for_bacterial_surface_display">Design of an autotransporter construct for bacterial surface display</SPAN></H1><DIV class="white-box"><P>Besides the design of biotinylated and biotin-binding receptors for eukaryotic cells, we also applied the biotINK approach for prokaryotic cells. For this purpose, we designed an autotransporter device that is able to present biotinylated or biotin-binding protein domains on the surface of <I>E. coli</I>. Constructs for bacterial surface display are hereby already well known in the field of protein engineering of therapeutic proteins, such as antibodies. There, they are used to screen libraries of different antibodies concerning their affinity towards a given therapeutic target. 
</P><P>We based our design on the autotransporter EspP<SUP id="cite_ref-24" class="reference"><A href="#cite_note-24">[24]</A></SUP>.
</P><DIV class="imagelink float-right"><A href="/wiki/images/e/e5/Muc16_Autotransporterdesign_001.png" class="internal" title="Muc16 Autotransporterdesign 001.png">Media:Muc16 Autotransporterdesign_001.png</A><DIV class="caption"><B>Figure 14:</B> Design of EspP autotransporter-based cell wall proteins presenting biotin or biotin-binding proteins for gram-negative bacteria.</DIV></DIV><P>The autotransporter system itself is established and functional (see Fig. A<SUP id="cite_ref-25" class="reference"><A href="#cite_note-25">[25]</A></SUP>), and here we &quot;bricked&quot; it down into RFC[10] and RFC[25] BioBricks. At first we tested different inducible bacterial promoter systems (e.g. &quot;TetR repressed GFP&quot;; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K577893">BBa_K577893</A>) that are available in the parts registry, but when we saw that two of the available systems did not work in our own hands, we decided to built the system on our own. We used the Tet-Repressor generator that was available on the distribution plates (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I739001">BBa_I739001</A>) as we believe that it is very important for the standardization in SynBio to use available parts. Downstream of this repressor generator we designed and fused a BioBrick (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170141">BBa_K2170141</A>) that encodes a double Tet-Operator (Tet-Repressor binding site together with a bacterial promoter and an signal peptide of the outer membrane protein A (OmpA<SUP id="cite_ref-26" class="reference"><A href="#cite_note-26">[26]</A></SUP>) from <I>E. coli</I> that has a 3' RFC[25] restriction site, allowing the protein fusion of proteins that are secreted into the bacterial periplasm when expressed with this BioBrick. As a protein fusion, we then assembled a protein cargo domain that is generally exchangeable for any protein that is so small in size that it can be transported through the outer membrane by the EspP autotransporter. In our project we used the same extracellular domains as for the eukaryotic receptors: enhanced monomeric Avidin (eMA; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170204">BBa_K2170204</A>), single-chain Avidin (scAvidin; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170205">BBa_K2170205</A>) and a composite part composed of a N-terminal Biotinylation acceptor peptide (BAP) and a C-terminal NanoLuciferase (together shown as BAP; ; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170118">BBa_K2170118</A>, see Fig. B). Further downstream our design features a A3C5-epitope tag that can be recognized by a high affinity Fab-fragment and is generally used to detect and quantify the surface display of the cargo domain<SUP id="cite_ref-27" class="reference"><A href="#cite_note-27">[27]</A></SUP>. Further downstream of this affinity tag we fused the BioBrick for the EspP autotransporter from <I>E. coli</I><SUP id="cite_ref-28" class="reference"><A href="#cite_note-28">[28]</A></SUP><SUP id="cite_ref-29" class="reference"><A href="#cite_note-29">[29]</A></SUP> which is then followed by a bacterial terminator. For the termination we again chose a well working terminator from the distribution plate (&quot;double terminator&quot;; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0010">BBa_B0010</A>-<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0012">BBa_B0012</A>). But this was just the DNA-part. What is expected to happens on a protein level? The construct constantly produces Tet-repressor that binds to thet Tet-Operator and repressed the promoter activity of the autotransporter gene. As soon as the inducer anhydrotetracycline (aTc) is added to the culture, it binds to the Tet-repressor that can't bind any more to the Tet-operator and thus the expression of the autotransporter can be regulated. Although the TetR-system is known to be a tight promoter system there is always a certain background expression with most promoters. If the protein expression of the autotransporter is induced using aTc the autotransporter is transcribed and translated. Due to the bacterial signal peptide the protein is secreted into the bacterial periplasm (the space between the two bacterial membranes of gram-negative bacteria such as <I>E. coli</I>). When present in the bacteria periplasm the autotransporter diffused to the outer membrane and inserts into the membrane. After the integration into the outer membrane the protein cargo is transportet through the beta-barrel of the autotransporter and is pre sented on the bacterial surface. We are sure that this BioBrick will be a valuable contribution to the Parts Registry as it allows future teams to display small protein domains on the surface of gram-negative bacteria, such as <I>E. coli</I>.
</P></DIV><H2><SPAN class="mw-headline" id="Results:_Testing_the_bacterial_autotransporter_system">Results: Testing the bacterial autotransporter system</SPAN></H2><DIV class="white-box"><DIV class="imagelink float-right"><A href="/wiki/images/9/9f/Muc16_Autotransporter_001.png" class="internal" title="Muc16 Autotransporter 001.png">Media:Muc16 Autotransporter_001.png</A><DIV class="caption"><B>Figure 15:</B> Quantification of luciferase activity in different cellular localizations.</DIV></DIV><P>For the testing of our autotransporter constructs we transformed all three autotransporter constructs together with a secreted NanoLuc luciferase, which served as a control, into <I>E. coli</I> JK321<SUP id="cite_ref-30" class="reference"><A href="#cite_note-30">[30]</A></SUP>. Transformed cells were grown in LB-medium supplemented with chloramphenicol (pSB1C3-vector) at 30°C and 180 RPM. In the morning a stationary over night culture was used to inoculate 50 mL fresh medium with 1 mL of culture. The cultures were grown to OD<SUB>550</SUB> = 0.5 and then the induction of the protein production was started using aTc: &quot;non&quot; = non-induced; &quot;low&quot; = 25 ng/mL aTc; &quot;high&quot; = 200 ng/mL. After induction the cultures were grown for further 5 h at the same concitions and cells were afterwards stored at 4°C until they were used for analysis or bioprinting.
After the cell harvest the optical density (OD<SUB>550</SUB>) was determined again as it is known that a high expression of EspP makes the outer membrane of <I>E. coli</I> permeable and this drastically affects cellular viability. For the scAvidin-autotransporter and the secreted BAP-NanoLuc autotransporter at high inducer concentration show clearly reduced viability as they did not show any growth after induction at OD<SUB>550</SUB> = 0.5 (see Fig. A). In order to analyze the functionality of the whole design and these new BioBricks we subjected each of the twelve cultures to three different treatments and determined subsequently the amount of NanoLuc luciferase using the manufacturers substrate kit (Thanks to our sponsor Promega ;).
At first the total NanoLuc luciferase content was determined in samples that were just washed once with PBS and contained the whole bacterial cell (see Fig. B), bacterial cells that were incubated with PPA-buffer (periplasmic extraction buffer; 500 mM sucrose, 100 mM Tris/HCl pH 8.0, 1 mM EDTA) that extracts the proteins present in the periplasm (see Fig. C) and finally the culture supernatant of the cultivated bacterial cells (see Fig. D). The total NanoLuc activity is only present in bacteria that are transformed with a plasmid encoding the luciferase and there is a clear dependency of inducer concentration a detected luciferase activity. In the samples where a periplasmic extract was prepared the luciferase activity is highest in the samples of the secreted luciferase as the EspP-fused BAP-NanoLuc is attached to the cells and only proteins from lysed cells are present in the periplasmic extract. In the case of the samples where the supernatant of the cultures was analyzed the luciferase activity is present in both &quot;high&quot; induced samples.</P><P>As the luciferase activity only gave an indication for the subcellular localization of the cargo-autotransporter fusion protein, we also performed flow cytometry measurements. For this purpose the same cells were stained with app. 10 µM A3C5-Fab that was coupled to the fluorescent dye Dy634-NHS and 100 µM of the chemical compound biotin-atto488. Cells were incubated for 2 h at 4°C, washed twice with PBS and analyzed using a FACSAria II flow cytometer (BD bioscience, voltages for fluorescence channels were all at 750 V).
</P><DIV class="imagelink float-right"><A href="/wiki/images/8/8b/Muc16_Autotransporter_FACS_001.png" class="internal" title="Muc16 Autotransporter FACS 001.png">Media:Muc16 Autotransporter_FACS_001.png</A><DIV class="caption"><B>Figure 16:</B> Quantification of biotin binding and surface-localization of A3C5 tags using FACS measurements.</DIV></DIV><P>For the channels FITC (biotin-atto488) and Dy634 (A3C5-F<SUB>ab</SUB>-Dy634, detected using a HeNe
laser (633 nm) and a 660/20 band pass ﬁlter) the fluorescence signal was measured and mean values are given. The cells showed a homogeneious population in the FSC/SSC plot. The A3C5-F<SUB>ab</SUB> staining detected high signals up til a mean of over 4000 for the eMA-autotransporter dependent on the inducer concentration. For the same bacterial cells the binding of biotin-Atto488 could be confirmed as well. The single chain Avidin (scAvidin) cells showed absolutely no A3C5 signal but a low Biotin-Atto488 background signal that could be caused by biotin-importers that transport biotin into the cells or the slow diffusion into the periplasm of <I>E. coli</I>. In contrast the BAP-Receptor also shows a high surface presentation as seen in the A3C5 signal of nearly 4000 for the &quot;high&quot; induced sample. The elevated biotin-Atto488 is unexpected but could be explained by the weakened and partly permeable out membrane that does not stop the small-molecule dye from entering the periplasm.
</P></DIV><H2><SPAN class="mw-headline" id="Discussion:_The_autotransporter_is_working_and_can_present_small_protein_cargo_on_the_bacterial_surface">Discussion: The autotransporter is working and can present small protein cargo on the bacterial surface</SPAN></H2><DIV class="white-box"><P>The performed experiment confirm the functionality of the device including the tetracyclin repressed promoter, the secretion into the bacterial periplasm as well as the presentation of the protein cargon on the bacterial outer membrane. In agreement with the expectation the 62 kDa big scAvidin with its extremely strong fold is too big in size for the autotransporter and can not be presented on the bacterial surface. In contrast the smaller BAP-NanoLuc (19 kDa for the NanoLuc) or the eMA (15 kDa) both can be efficiently presented on the surface. The enzymatic biotinylation of the Biotin Acceptor Peptide (BAP) using recombinant BirA biotin ligase is well established but could not be show before the wiki freeze. In Summary the whole device works great but there seems to be a limitation concerning the size of proteindomains that can be presented with the EspP autotransporter. We are curious to see what future iGEM teams will use the EspP Autotransporter in the future.
</P></DIV><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><DIV class="white-box"><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Chen, I., Howarth, M., Lin, W., &amp; Ting, A. Y. (2005). Site-specific labeling of cell surface proteins with biophysical probes using biotin ligase. Nature methods, 2(2), 99-104.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Sigmund et al., upcoming manuscript</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">Kozak, M. (1989). The scanning model for translation: an update. The Journal of cell biology, 108(2), 229-241.</SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text">Kozak, M. (1987). An analysis of 5'-noncoding sequences from 699 vertebrate messenger RNAs. Nucleic acids research, 15(20), 8125-8148.</SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">↑</A></SPAN><SPAN class="reference-text">Petersen, T. N., Brunak, S., von Heijne, G., &amp; Nielsen, H. (2011). SignalP 4.0: discriminating signal peptides from transmembrane regions. Nature methods, 8(10), 785-786.</SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">↑</A></SPAN><SPAN class="reference-text">Sonnhammer, E. L., Von Heijne, G., &amp; Krogh, A. (1998, July). A hidden Markov model for predicting transmembrane helices in protein sequences. In Ismb (Vol. 6, pp. 175-182).</SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text">Alexander, H., Harpprecht, J., Podzuweit, H. G., Rautenberg, P., &amp; Müller-Ruchholtz, W. (1994). Human monoclonal antibodies recognize early and late viral proteins of human cytomegalovirus. Human Antibodies, 5(1-2), 81-90.</SPAN></LI><LI id="cite_note-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-8">↑</A></SPAN><SPAN class="reference-text">Kredel, S., Oswald, F., Nienhaus, K., Deuschle, K., Röcker, C., Wolff, M., ... &amp; Wiedenmann, J. (2009). mRuby, a bright monomeric red fluorescent protein for labeling of subcellular structures. PloS one, 4(2), e4391.</SPAN></LI><LI id="cite_note-9"><SPAN class="mw-cite-backlink"><A href="#cite_ref-9">↑</A></SPAN><SPAN class="reference-text">Bajar, B. T., Wang, E. S., Lam, A. J., Kim, B. B., Jacobs, C. L., Howe, E. S., ... &amp; Chu, J. (2016). Improving brightness and photostability of green and red fluorescent proteins for live cell imaging and FRET reporting. Scientific reports, 6.</SPAN></LI><LI id="cite_note-10"><SPAN class="mw-cite-backlink"><A href="#cite_ref-10">↑</A></SPAN><SPAN class="reference-text">Schmidt, T. G., &amp; Skerra, A. (2007). The <I>Strep</I>-tag system for one-step purification and high-affinity detection or capturing of proteins. Nature protocols, 2(6), 1528-1535.</SPAN></LI><LI id="cite_note-11"><SPAN class="mw-cite-backlink"><A href="#cite_ref-11">↑</A></SPAN><SPAN class="reference-text">Case, J. F. (2004). Flight studies on photic communication by the firefly Photinus pyralis. Integrative and comparative biology, 44(3), 250-258.</SPAN></LI><LI id="cite_note-12"><SPAN class="mw-cite-backlink"><A href="#cite_ref-12">↑</A></SPAN><SPAN class="reference-text">Zhang, L., Song, G., Xu, T., Wu, Q. P., Shao, X. X., Liu, Y. L., ... &amp; Guo, Z. Y. (2013). A novel ultrasensitive bioluminescent receptor-binding assay of INSL3 through chemical conjugation with nanoluciferase. Biochimie, 95(12), 2454-2459.</SPAN></LI><LI id="cite_note-13"><SPAN class="mw-cite-backlink"><A href="#cite_ref-13">↑</A></SPAN><SPAN class="reference-text">Gould, S. J., &amp; Subramani, S. (1988). Firefly luciferase as a tool in molecular and cell biology. Analytical biochemistry, 175(1), 5-13.</SPAN></LI><LI id="cite_note-14"><SPAN class="mw-cite-backlink"><A href="#cite_ref-14">↑</A></SPAN><SPAN class="reference-text">Hall, M. P., Unch, J., Binkowski, B. F., Valley, M. P., Butler, B. L., Wood, M. G., ... &amp; Robers, M. B. (2012). Engineered luciferase reporter from a deep sea shrimp utilizing a novel imidazopyrazinone substrate. ACS chemical biology, 7(11), 1848-1857.</SPAN></LI><LI id="cite_note-15"><SPAN class="mw-cite-backlink"><A href="#cite_ref-15">↑</A></SPAN><SPAN class="reference-text"> Nanbo, A., Sugden, A., &amp; Sugden, B. (2007). The coupling of synthesis and partitioning of EBV's plasmid replicon is revealed in live cells. The EMBO journal, 26(19), 4252-4262.</SPAN></LI><LI id="cite_note-16"><SPAN class="mw-cite-backlink"><A href="#cite_ref-16">↑</A></SPAN><SPAN class="reference-text"> Boussif, O., Lezoualc'h, F., Zanta, M. A., Mergny, M. D., Scherman, D., Demeneix, B., &amp; Behr, J. P. (1995). A versatile vector for gene and oligonucleotide transfer into cells in culture and in vivo: polyethylenimine.Proceedings of the National Academy of Sciences, 92(16), 7297-7301.</SPAN></LI><LI id="cite_note-17"><SPAN class="mw-cite-backlink"><A href="#cite_ref-17">↑</A></SPAN><SPAN class="reference-text"> Molina, D. K., &amp; DiMaio, V. J. (2012). Normal organ weights in men: part II—the brain, lungs, liver, spleen, and kidneys. The American journal of forensic medicine and pathology, 33(4), 368-372.</SPAN></LI><LI id="cite_note-18"><SPAN class="mw-cite-backlink"><A href="#cite_ref-18">↑</A></SPAN><SPAN class="reference-text"> Wilson, Z. E., Rostami‐Hodjegan, A., Burn, J. L., Tooley, A., Boyle, J., Ellis, S. W., &amp; Tucker, G. T. (2003). Inter‐individual variability in levels of human microsomal protein and hepatocellularity per gram of liver. British journal of clinical pharmacology, 56(4), 433-440.</SPAN></LI><LI id="cite_note-19"><SPAN class="mw-cite-backlink"><A href="#cite_ref-19">↑</A></SPAN><SPAN class="reference-text"><A rel="nofollow" class="external free" href="https://www.thermofisher.com/de/de/home/references/protocols/proteins-expression-isolation-and-analysis/protein-expression-protocol/flp-in-system-for-generating-constitutive-expression-cell-lines.html#prot4">https://www.thermofisher.com/de/de/home/references/protocols/proteins-expression-isolation-and-analysis/protein-expression-protocol/flp-in-system-for-generating-constitutive-expression-cell-lines.html#prot4</A></SPAN></LI><LI id="cite_note-20"><SPAN class="mw-cite-backlink"><A href="#cite_ref-20">↑</A></SPAN><SPAN class="reference-text"> Blochlinger, K. A. R. E. N., &amp; Diggelmann, H. E. I. D. I. (1984). Hygromycin B phosphotransferase as a selectable marker for DNA transfer experiments with higher eucaryotic cells. Molecular and cellular biology, 4(12), 2929-2931.</SPAN></LI><LI id="cite_note-21"><SPAN class="mw-cite-backlink"><A href="#cite_ref-21">↑</A></SPAN><SPAN class="reference-text">Kredel, S., Oswald, F., Nienhaus, K., Deuschle, K., Röcker, C., Wolff, M., ... &amp; Wiedenmann, J. (2009). mRuby, a bright monomeric red fluorescent protein for labeling of subcellular structures. PloS one, 4(2), e4391.</SPAN></LI><LI id="cite_note-22"><SPAN class="mw-cite-backlink"><A href="#cite_ref-22">↑</A></SPAN><SPAN class="reference-text">Gradinaru, Viviana, et al. &quot;Molecular and cellular approaches for diversifying and extending optogenetics.&quot; Cell 141.1 (2010): 154-165.</SPAN></LI><LI id="cite_note-23"><SPAN class="mw-cite-backlink"><A href="#cite_ref-23">↑</A></SPAN><SPAN class="reference-text">Livak, Kenneth J., and Thomas D. Schmittgen. &quot;Analysis of relative gene expression data using real-time quantitative PCR and the 2− ΔΔCT method.&quot; methods 25.4 (2001): 402-408.</SPAN></LI><LI id="cite_note-24"><SPAN class="mw-cite-backlink"><A href="#cite_ref-24">↑</A></SPAN><SPAN class="reference-text">Binder, U., Matschiner, G., Theobald, I., &amp; Skerra, A. (2010). High-throughput sorting of an Anticalin library via EspP-mediated functional display on the Escherichia coli cell surface. Journal of molecular biology, 400(4), 783-802.</SPAN></LI><LI id="cite_note-25"><SPAN class="mw-cite-backlink"><A href="#cite_ref-25">↑</A></SPAN><SPAN class="reference-text">Binder, U., Matschiner, G., Theobald, I., &amp; Skerra, A. (2010). High-throughput sorting of an Anticalin library via EspP-mediated functional display on the Escherichia coli cell surface. Journal of molecular biology, 400(4), 783-802.</SPAN></LI><LI id="cite_note-26"><SPAN class="mw-cite-backlink"><A href="#cite_ref-26">↑</A></SPAN><SPAN class="reference-text">Ghrayeb, J., Kimura, H., Takahara, M., Hsiung, H., Masui, Y., &amp; Inouye, M. (1984). Secretion cloning vectors in Escherichia coli. The EMBO journal, 3(10), 2437.</SPAN></LI><LI id="cite_note-27"><SPAN class="mw-cite-backlink"><A href="#cite_ref-27">↑</A></SPAN><SPAN class="reference-text">Costa, J., Grabenhorst, E., Nimtz, M., &amp; Conradt, H. S. (1997). Stable expression of the Golgi form and secretory variants of human fucosyltransferase III from BHK-21 cells Purification and characterization of an engineered truncated form from the culture medium. Journal of Biological Chemistry, 272(17), 11613-11621.</SPAN></LI><LI id="cite_note-28"><SPAN class="mw-cite-backlink"><A href="#cite_ref-28">↑</A></SPAN><SPAN class="reference-text">Barnard, T. J., Dautin, N., Lukacik, P., Bernstein, H. D., &amp; Buchanan, S. K. (2007). Autotransporter structure reveals intra-barrel cleavage followed by conformational changes. Nature structural &amp; molecular biology, 14(12), 1214-1220.</SPAN></LI><LI id="cite_note-29"><SPAN class="mw-cite-backlink"><A href="#cite_ref-29">↑</A></SPAN><SPAN class="reference-text">Skillman, K. M., Barnard, T. J., Peterson, J. H., Ghirlando, R., &amp; Bernstein, H. D. (2005). Efficient secretion of a folded protein domain by a monomeric bacterial autotransporter. Molecular microbiology, 58(4), 945-958.</SPAN></LI><LI id="cite_note-30"><SPAN class="mw-cite-backlink"><A href="#cite_ref-30">↑</A></SPAN><SPAN class="reference-text">Jose, J., Krämer, J., Klauser, T., Pohlner, J., &amp; Meyer, T. F. (1996). Absence of periplasmic DsbA oxidoreductase facilitates export of cysteine-containing passenger proteins to the Escherichia coli cell surface via the Iga β autotransporter pathway. Gene, 178(1), 107-110.</SPAN></LI></OL></DIV></DIV></DIV><DIV class="row"><FOOTER class="col-sm-9 col-sm-offset-3 noprint page-footer clearfix white-box"><A class="btt visible-xs-block" href="#top"><SPAN class="sr-only">Back to top</SPAN></A><DIV class="row"><DIV class="col-sm-4 social"><H2>Follow us on:</H2></DIV><DIV class="col-sm-4 universities"><H2>LMU &amp; TUM Munich</H2><P>United team from Munich's universities</P></DIV><DIV class="col-sm-4 address"><H2>Contact us:</H2><H3>Address</H3><P>iGEM Team TU-Munich
                     Emil-Erlenmeyer-Forum 5
                     85354 Freising, Germany</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>