<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_LMU-TUM_Munich_Methods skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:LMU-TUM Munich/Methods</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="row"><DIV class="col-sm-3 col-xs-8 fixed-sm z-1"><DIV class="sidebar-nav"><DIV class="navbar" role="navigation"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".sidebar-navbar-collapse"><SPAN class="sr-only">Toggle menu</SPAN></BUTTON><SPAN class="visible-xs navbar-brand">Menu</SPAN></DIV><DIV class="navbar-collapse collapse sidebar-navbar-collapse"><UL id="main-menu" class="nav navbar-nav methods"><LI class="menu_item home"><A href="https://2016.igem.org/Team:LMU-TUM_Munich"><SPAN class="visible-xs-block">Home</SPAN></A></LI><LI class="dropdown menu_item team"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI class="members"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Team">Members</A></LI><LI class="collaborations"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Collaborations">Collaborations</A></LI><LI class="sponsors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Sponsors">Sponsors</A></LI><LI class="press"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Press">Press</A></LI><LI class="gllry"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Gallery">Gallery</A></LI></UL><LI class="dropdown menu_item project"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Project </A></LI><LI class="description"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Description">Description</A></LI><LI class="design"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Design">Design</A></LI><LI class="proteins"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proteins">Proteins</A></LI><LI class="receptors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Receptors">Receptors</A></LI><LI class="linker"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Linkerchemistry">Linker Chemistry</A></LI><LI class="poly"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proof">Polymerization</A></LI><LI class="bioprinting"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Demonstrate">Bioprinting</A></LI><LI class="hypoxia"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hypoxia">Hypoxia</A></LI><LI class="medical"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Medical">Medical Application</A></LI><LI class="interlab"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/InterLab">InterLab</A></LI><LI class="software"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Software">Software</A></LI><LI class="dropdown menu_item notebook"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Notebook </A></LI><LI class="labjournal"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Labjournal">Labjournal</A></LI><LI class="materials"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Materials">Materials</A></LI><LI class="methods"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Methods">Methods</A></LI><LI class="howtos"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Howtos">How Tos</A></LI><LI class="dropdown menu_item parts"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Parts </A></LI><LI class="parts-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Parts">Parts</A></LI><LI class="basic-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Basic_Part">Basic Parts</A></LI><LI class="composite-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Composite_Part">Composite Parts</A></LI><LI class="part-collection"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Part_Collection">Part Collection</A></LI><LI class="menu_item hardware"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hardware">Hardware</A></LI><LI class="menu_item model"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Model">Modeling</A></LI><LI class="menu_item safety"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Safety">Safety</A></LI><LI class="dropdown menu_item human_practices dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Human Practices </A></LI><LI class="human_practices-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Human_Practices">Human Practices</A></LI><LI class="silver"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Silver">Silver</A></LI><LI class="gold"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Gold">Gold</A></LI><LI class="integrated-practices"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Integrated_Practices">Integrated Practices</A></LI><LI class="engagement"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Engagement">Engagement</A></LI><LI class="dropdown menu_item entrepreneur dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Entrepreneurship</A></LI><LI class="entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Entrepreneurship">Entrepreneurship</A></LI><LI class="supporting-entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Supporting_Entrepreneurship">Supporting Entrepreneurship</A></LI><LI class="menu_item attributions"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV class="col-sm-9 col-sm-offset-3"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Basic_techniques_in_molecular_biology"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Basic techniques in molecular biology</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Preparation_of_plasmid_DNA"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Preparation of plasmid DNA</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Determination_of_DNA_concentrations"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Determination of DNA concentrations</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#DNA_digestion_and_ligation"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">DNA digestion and ligation</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Agarose_gel_electrophoresis"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Agarose gel electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Polymerase_chain_reaction"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Polymerase chain reaction</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Oligonucleotide_annealing"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Oligonucleotide annealing</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Western_blot"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Western blot</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Enzyme-linked_immunosorbent_assay"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Enzyme-linked immunosorbent assay</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Preparation_of_chemically_competent_E._coli_cells"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Preparation of chemically competent <I>E. coli</I> cells</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-11"><A href="#Basic_techniques_in_protein_biochemistry"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Basic techniques in protein biochemistry</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Determination_of_protein_concentration"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Determination of protein concentration</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Ion_exchange_chromatography_.28IEC.29"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Ion exchange chromatography (IEC)</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Size_exclusion_chromatography_.28SEC.29"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Size exclusion chromatography (SEC)</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Immobilized_metal_ion_affinity_chromatography_.28IMAC.29"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Immobilized metal ion affinity chromatography (IMAC)</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Discontinous_SDS_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Discontinous SDS polyacrylamide gel electrophoresis (SDS-PAGE)</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Protein_production"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Protein production</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Production_of_streptavidin_and_its_variants"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Production of streptavidin and its variants</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Fermentation"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Fermentation</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#PAS-lysine"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">PAS-lysine</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Human_cell_culture"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Human cell culture</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Seeding_and_passaging_of_HEK293T_cells"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Seeding and passaging of HEK293T cells</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Transfection_of_mammalian_cells_using_PEI_and_CaCl2"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Transfection of mammalian cells using PEI and CaCl<SUB>2</SUB></SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Luciferase_assay_for_the_quantification_of_protein_expression"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Luciferase assay for the quantification of protein expression</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Quantitative_reverse_transcription_Polymerase_chain_reaction_.28RT-qPCR.29"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Quantitative reverse transcription Polymerase chain reaction (RT-qPCR)</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#.CE.94.CE.94CP_method"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">ΔΔC<SUB>P</SUB> method</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#Biophysical_Methodology"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Biophysical Methodology</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Mass_spectrometry"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Mass spectrometry</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Fluorescence_titration"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Fluorescence titration</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Circular_dichroism_spectroscopy"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Circular dichroism spectroscopy</SPAN></A></LI><LI class="toclevel-1 tocsection-31"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Basic_techniques_in_molecular_biology">Basic techniques in molecular biology</SPAN></H2><H3><SPAN class="mw-headline" id="Preparation_of_plasmid_DNA">Preparation of plasmid DNA</SPAN></H3><DIV class="white-box"><P>For transformation of <I>E. coli</I> XL-1 blue chemically competent cells, cells were thawed on ice for 5 mins. 50-100 ng plasmid DNA (or ~200 ng ligation product) were added, and cells were further incubated on ice for 20-30 mins. Afterwards, a heatshock was applied at 42°C for 45 secs. Cells were then further incubated on ice for 2 mins, and 950 μl of LB medium were added. Cells were then shaken at 200 rpm for at least 1 h at 37°C, and plated on LB-agar plates containing the appropriate antibiotic. The next day, single clones were picked and used to inoculate a liquid culture of 5 ml LB medium. After being incubated overnight shaking (200 rpm) at 37°C, cells were spun down (6000 g, 5 mins), and plasmid DNA was extracted using the Qiagen QIAprep Spin Miniprep Kit. If ligation products were used for transformation, analytical digestion of DNA was performed to confirm the correct incorporation of the desired fragment. Therefore, DNA was digested with ~0.5 U of suitable restriction enyzmes for 1 h at 37°C and analyzed via analytical agarose gel electrophoresis. Additionally, plasmids were sequenced using the Eurofins Genomics sequencing service.
</P></DIV><H3><SPAN class="mw-headline" id="Determination_of_DNA_concentrations">Determination of DNA concentrations</SPAN></H3><DIV class="white-box"><P>DNA concentrations were determined spectrophotometrically using a Nanodrop device. From the measured absorption at 260 nm, DNA concentrations were determined, an absorption of 1 being assumed equal to a concentration 50 ng/μl DNA. Furthermore, wavelengths at 280 nm and 230 nm were measured to confirm sample purity without contamination of proteins (280 nm) or carbohydrates, phenol or EDTA (230 nm). Ratios of 260/280 nm being in the range between 1.6 and 2.0, as well as the ratios of 260/230 nm being between 2.0 and 2.2 generally indicate sample purity.
</P></DIV><H3><SPAN class="mw-headline" id="DNA_digestion_and_ligation">DNA digestion and ligation</SPAN></H3><DIV class="white-box"><P>For cloning, DNA was digested using restriction endonucleases. The amount of used enzyme hereby depended on the amount of to be digested DNA, whereas ~1 unit was used per μg of DNA. Buffers and DNA concentrations were used according to the manufacturer's suggestion. Digestions were incubated at 37°C. If deemed necessary, 5’-ends of vector fragments were dephosphorylated for 30 min at 37 °C directly by addition of alkaline phosphatase (Fast-AP) to the restriction mixture at 37°C, incubation for 10 mins and inactivation of the enzyme for 5 mins at 75°C.  
</P><P>Ligations were conducted using the appropriate DNA fragments while adding 400 units of ligase in the appropriate buffer. Vector and insert fragments were generally used at a 1:3 ratio, while the amount of vector fragment DNA was varied between 50-100 ng.
</P></DIV><H3><SPAN class="mw-headline" id="Agarose_gel_electrophoresis">Agarose gel electrophoresis</SPAN></H3><DIV class="white-box"><P>For separation of DNA fragments, agarose gel electrophoresis was used. For fragments with a size of more than 500 bp, 1% (m/V) agarose gels were used. For smaller fragments, agarose concentration was increased as deemed necessary, until concentrations of up to 1.8% (m/V) agarose. For the production of gels, the appropriate amount of agarose was solved in 50 ml TAE buffer by heating the suspension. For staining, ethidium bromide was added into the mixture. Gels were run at 100 V for analytical and at 80 V for preparative use (200 mA respectively). For preparative gels, DNA fragments were excised and extracted from the gel using the Qiagen Gel Extraction Kit according to the manufacturer's protocol.
</P></DIV><H3><SPAN class="mw-headline" id="Polymerase_chain_reaction">Polymerase chain reaction</SPAN></H3><DIV class="white-box"><P>For the amplification of DNA fragments, polymerase chain reaction was conducted. Hereby, 50 µl PCR mixture contained 50-500 ng template DNA, 100 pmol of each primer, and 10 nmol of each desoxynucleotide (dATP, dCTP, dGTP, dTTP).  Additionally, the mixture contained 1 U Q5 high-fidelity polymerase and Q5 polymerase buffer, and was filled up with ddH<SUB>2</SUB>O to the final volume. For the PCR, double-stranded fragments were initially denaturated at 98 °C for 2 min and subsequently amplified in 25-30 repetitive cycles, each comprising three steps: (1) Denaturation of the double-stranded DNA (98 °C, 30 s), (2) annealing of the primer to the single-stranded template for 30 s at 50-65 °C (depending on the melting temperature of the primer-template hybrid), and (3) extension of the primed DNA template (72 °C, 60 s per kb fragment length). After that, a final elongation step (2 min, 72 °C) was included to ensure that remaining single-stranded template molecules are fully elongated. 
</P><P>Then, the PCR product was separated by agarose gel electrophoresis to check for its correct size and isolated from the gel as described or directly isolated from the PCR reaction mixture, respectively using the Qiagen Gel extraction and PCR purification kit.
</P></DIV><H3><SPAN class="mw-headline" id="Oligonucleotide_annealing">Oligonucleotide annealing</SPAN></H3><DIV class="white-box"><P>2 µl of a 100 μM solution of the phosphorylated oligonucleotide in 200 µL ddH<SUB>2</SUB>O were heated up to 100 °C. Then, the inset was removed and the solution was allowed to cool down.
</P></DIV><H3><SPAN class="mw-headline" id="Western_blot">Western blot</SPAN></H3><DIV class="white-box"><P>After an SDS-Page with a prestained protein marker, the proteins were blotted on a PDVF membrane using a 50 mA current for 1:30 h. The membrane was afterwards blocked in 3% BSA solution overnight. All washing steps were done using PBS-T (0.1% Tween20 + 1% BSA)  
Proteins were detected via a primary antibody against the <I>Strep</I>-tag (<I>Strep</I>-mAb-classic) and a secondary goat-anti-mouse-F<SUB>c</SUB> antibody conjugated with alkaline phospatase. Additionally, we used a human F<SUB>ab</SUB> against the A3C3-tag and a goat-anti-human alkaline phosphatase-coupled secondary antibody. The Substrate solution consisted of 5-Brom-4-chlor-3-indolylphosphate and nitro blue tetrazolium diluted in AP-buffer (100 mM Tris HCl, pH 8,8, 100 mM NaCl, 5 mM MgCl<SUB>2</SUB>).
</P></DIV><H3><SPAN class="mw-headline" id="Enzyme-linked_immunosorbent_assay">Enzyme-linked immunosorbent assay</SPAN></H3><DIV class="white-box"><P>The anti-PDGF ELISA was done using the abcam 'ab184860 - PDGF BB Human SimpleStep ELISA® Kit'. In short, this kit provides a 96-well plate that is already coated with anti-PDGF antibodies, which are then coupled with a so-called <I>capture</I> and a <I>detection</I> antibody. For measurements, 20 μl of the medium taken at the respective measurements were diluted with 80 μl sample dilution buffer. 
</P><P>Wells were then washed three times with washing buffer, detection substrate was added and the mixture incubated for 1 h at RT at 400 rpm. Afterwards, stop-buffer was added and an end-point absorption measurement at 450 nm was made.
</P></DIV><H3><SPAN class="mw-headline" id="Preparation_of_chemically_competent_E._coli_cells">Preparation of chemically competent <I>E. coli</I> cells</SPAN></H3><DIV class="white-box"><P>A frozen glycerol stock of the desired strain of <I>E. coli</I> cells was streaked out on a LB plate under sterile conditions and incubated overnight at 37°C. A 4 ml LB medium liquid preculture was then inoculated and incubated overnight at 37°C shaking. 500 μl starter culture were used to inoculate a 50 ml LB medium liquid culture. When reaching an OD<SUB>600</SUB> of 0.45-0.5, the culture was immediately chilled on ice for 10 mins, and harvested by centrifugation at 3000g for 10 mins at 4°C. The supernatant was cast away, and the pellet resuspended in 40 ml of ice cold 100 mM MgCl<SUB>2</SUB>. Cells were spun down again at 3000 g for 10 mins at 4°C, and the pellet resuspended in 20 ml of ice cold 50 mM CaCl<SUB>2</SUB>. This suspension was kept on ice for 30 mins, and harvested by centrifugation at 3000 g for 10 mis at 4°C. The pellet was resuspended in 2 ml ice cold 50 mM CaCl<SUB>2</SUB> and 15% glycerol. The resulting bacterial suspension was sampled in 1.5 ml microcentrifuge tubes, frozen in liquid nitrogen and stored at -80°C until further use.
</P></DIV><H2><SPAN class="mw-headline" id="Basic_techniques_in_protein_biochemistry">Basic techniques in protein biochemistry</SPAN></H2><H3><SPAN class="mw-headline" id="Determination_of_protein_concentration">Determination of protein concentration</SPAN></H3><DIV class="white-box"><P>Protein concentrations were determined via UV/VIS spectrometry. According to Lambert-Beer’s law, the measured absorption at 280 nm (0.1 &lt; A<SUB>280</SUB>&lt; 1) was normalised by the extinction coefficient of the protein, which can be predicted by ProtParam based on its sequence. Considering the given dilution and the length d of the cuvette, a protein concentration can be predicted by c = A<SUB>280</SUB>/(ε*d)
</P></DIV><H3><SPAN class="mw-headline" id="Ion_exchange_chromatography_.28IEC.29">Ion exchange chromatography (IEC)</SPAN></H3><DIV class="white-box"><P>Ion exchange chromatography is based on the separation of proteins with different electrostatic properties (pI values) by binding them on a charged matrix and their displacement and elution via a salt gradient.
Positively charged proteins (pH &lt; pI) can be separated by cation exchange chromatography using a Resource S column (Sulfonate group). Negatively charged proteins (pH &gt; pI) in contrast, are purified with a Resource Q column (Quaternary ammonium).
The used columns have a volume of 6 ml.
</P></DIV><H3><SPAN class="mw-headline" id="Size_exclusion_chromatography_.28SEC.29">Size exclusion chromatography (SEC)</SPAN></H3><DIV class="white-box"><P>Size exclusion chromatography is a chromatographic method based on different diffusion times of proteins through a gel matrix in dependence on the hydrodynamic radius of the target protein; heavier proteins generally eluting earlier due to higher diffusion speeds through the gel matrix.
For size exclusion chromatography of the streptavidin variants, 0.5 ml of concentrate was loaded on a Sephacryl S300 16 60 column. Resulting peak fractions were respectively loaded onto an SDS-gel. Possible Streptavidin aggragates and monomers were removed, fractions of the tetramer were pooled.
</P></DIV><H3><SPAN class="mw-headline" id="Immobilized_metal_ion_affinity_chromatography_.28IMAC.29">Immobilized metal ion affinity chromatography (IMAC)</SPAN></H3><DIV class="white-box"><P>If a target protein is expressed as a SUMO/TEV peptide fusion construct with His<SUB>6</SUB>-tag, the recombinantly expressed protein could be purified via IMAC. IMAC makes use of the interaction of imidazole side chains of histidine residues with immobilised nickel ions. Using this method, the target protein can be eluated by addition of imidazole, which competitively binds to the column and thus displaces the target protein in the column, causing it to eluate. 
</P><P>After harvesting cells the target protein was purified using a stripped and equilibrated HiTrap IMAC HP column (6 ml volume). Therefore, the column was loaded with the sample and washed with IMAC buffer (20 mM NaH<SUB>2</SUB>PO<SUB>4</SUB>, 500 mM NaCl, pH 7.5) until absorption at 280 nm hit zero. For elution, a -linear/step- imidazole gradient was set. The resulting fractions were analysed by SDS-polyacrylamide gel electrophoresis and were pooled for further purification, if their purity was deemed sufficient. In order to remove imidazole, dialysis was conducted overnight, using 5 l dialysis buffer.
</P></DIV><H3><SPAN class="mw-headline" id="Discontinous_SDS_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29">Discontinous SDS polyacrylamide gel electrophoresis (SDS-PAGE)</SPAN></H3><DIV class="white-box"><P>For analysis of recombinant protein expression, discontinuous SDS-polyacrylamide gel electrophoresis was conducted. Therefore, samples were mixed with Laemmli buffer (finalc concentration 1x) and incubated at 95°C for 5 min. For electrophoresis, currents of 60 mA were used, while voltage was initially set to 90 V and raised to 130 V after 10 mins. For staining, the gel was incubated with Coomassie brilliant blue R250 in H2O:MeOH:HAc (5:4:1) for 30 minutes.
</P></DIV><H2><SPAN class="mw-headline" id="Protein_production">Protein production</SPAN></H2><H3><SPAN class="mw-headline" id="Production_of_streptavidin_and_its_variants">Production of streptavidin and its variants</SPAN></H3><DIV class="white-box"><P>The minimal Streptavidin wildtype gene is cloned on a pSA plasmid, with its transcription being under control of a T7-promoter. The <I>E.coli</I> BL-R (DE3) strain encodes the T7-polymerase under the control of a lacUV5 promotor, which can be induced by Isopropyl β-D-1-thiogalactopyranoside (IPTG).
A shaking flask with 2 L LB medium supplemented with ampicillin is inoculated 1:40 with a 50 ml preculture grown at 30 °C overnight.
The bacterial culture was shaken at 37 °C until its optical density reached 0.6 and protein production was induced with 1 mM IPTG. After 4 hours, cells were harvested by centrifugation (5000 rpm for 30 min). The cell pellet was resuspended in a 20 mM Tris/HCl buffer (500 mM NaCl, pH 8) and was homogenized via a French press or a GEA PANDA homogenizer. The cell lysate was centrifuged at 11,500 rpm for 30 min to separate inclusion bodies in the pellet from the soluble proteins of the supernatant.
The inclusion bodies consist of aggregated and misfolded protein, and were denaturated in an appropriate amount of 6 M guanidinium chloride and centrifuged at 15000 rpm for 15 minutes. The aggregations in the pellet was discarded and the supernatant was added slowly into a high volume of PBS (30 ml PBS for 1 ml protein solution) and incubated overnight. Thus removing guanidium chloride, proteins are refolded again. The solution was then centrifuged again (11,500 rpm for 30 min) to remove aggregates.
Streptavidin purification was then proceeded by ammonium sulfate precipitation. After each precipitation step the solution is incubated for a few hours and centrifuged at 11,500 rpm for 30 minutes afterwards.
</P><P>The first precipitation step includes 40% saturation (1.75 M). Streptavidin remains soluble, precipitants are discarded. After centrifugation, the ammonium sulfate saturation is increased up to 70 % (3.37 M), with streptavidin being no longer soluble and precipitates.
The pellet was then resuspended in a 50% saturated ammonium sulfate solution (2.25 M), streptavidin again precipitates. After centrifugation the pellet is resuspended in 1x PBS buffer. As a result the solution contains only those proteins which precipitate between 40 % and 50 % ammonium sulfate saturation. One last centrifugation at 15000 rpm for 30 minutes removes insoluble impurities.
The solution was then dialysed against 20 mM Tris/HCl buffer without salt (pH 8). Streptavidin has an isoelectric point of 6.09, therefore it is negatively charged in its buffer and a positively charged ResQ column is used for ion exchange chromatography. An increasing salt concentration of the buffer causes fractional elution of protein.
Streptavidin monomers, tetramers and aggregates can be identified by gel filtration with a sephacryl column. After sterile filtration, a UV/VIS spectrum of the solution was measured in order to determine its concentration.
The production of Traptavidin and Strep-Tactin was performed analogously. Enhanced monomeric Avidin was isolated via IMAC. 
</P></DIV><H3><SPAN class="mw-headline" id="Fermentation">Fermentation</SPAN></H3><DIV class="white-box"><P><I><B>Table 1: Composition of 1 L feed </B></I></P><TABLE><TBODY><TR><TD>glucose
</TD><TD>714 ml
</TD></TR><TR><TD>HCDF salt
</TD><TD>100 ml
</TD></TR><TR><TD>MgSO<SUB>4</SUB></TD><TD>100 ml
</TD></TR><TR><TD>thiamine
</TD><TD>1 ml
</TD></TR><TR><TD>micronutritients
</TD><TD>0.5 ml
</TD></TR><TR><TD>antibiotics (1:1000)
</TD><TD>1 ml
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>83.5 ml
</TD></TR></TBODY></TABLE><P><I><B>Table 2: HCDF medium (for 1L)</B></I></P><TABLE><TBODY><TR><TD>glucose
</TD><TD>14.25 ml
</TD></TR><TR><TD>HCDF salt
</TD><TD>100 ml
</TD></TR><TR><TD>MgSO<SUB>4</SUB></TD><TD>10 ml
</TD></TR><TR><TD>thiamine
</TD><TD>1 ml
</TD></TR><TR><TD>micronutritients
</TD><TD>1 ml
</TD></TR><TR><TD>antibiotics (1:1000)
</TD><TD>1 ml
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>872.75 ml
</TD></TR></TBODY></TABLE><P>
Water and HCDF salt need to be autoclaved separately.
day 1: prepre-culture
Inocculation of 5 ml LB medium (antibiotic 1:1000)and shaking over night at 37°C
day 2: preculture
Inocculation of 1L medium (e.g. M9 ) and incubation of culture over night at 37°C (ideal: 24h)
day 3: Inocculation of fermenter
Calibration of fermenter: pO2 (0-100 %); flow rate (time for default volume). After centrifugation of preculture at 5000 rpm for 10 minutes, the pellet is resuspended in HCDF medium and poured into the fermenter. Following settings remein constant during the over night: p<SUB>O2</SUB>= 60 %; Temp= 30°C, pH~7
day 4: Fermentation
A negative glucose test shows that all glucose of the medium is fully metabolized by bacteria. The exponential feed can be started. Protein expression is induced by IPTG.
day 5: harvesting streptavidin
The cell culture is harvested by centrifugation (30 min at 6000 rpm) and is ready for purification steps. 
</P></DIV><H3><SPAN class="mw-headline" id="PAS-lysine">PAS-lysine</SPAN></H3><DIV class="white-box"><P>A 50 ml preculture with E.coli BL R (DE3) carrying the pSUMO plasmid is poured into a shaking flask with 2 L LB medium. After the bacterial suspension reaches an optical density of 0.5 the production of PAS-lysine is induced by adding IPTG (1 mM).
The cells are harvested (5000 rpm, 30 minutes) after five hours shaking at 37 °C and resuspended in 20 mM Tris/HCl buffer with 500 mM salt, pH 8. Homogenization is conducted by French press or PANDA. SUMO PAS-lysine is very good soluble, after centrifugation at 11500 rpm for 30 minutes it is located in the supernatant, the pellet can be discarded.
SUMO PAS-lysine has the property to remain native even at high temperatures, where most protein denaturate. Heating the solution up to 70 °C for 20 minutes and centrifugation afterwards removes nearly all other proteins.
The solution is dialysed against IMAC buffer (50 mM sodium dihydrogen phosphate and 500 mM salt, ph 7.5) and loaded on a chromatography column packed with nickel. After washing away impurities, the addition of imidazole results in elution of the SUMO PAS-lysine.
The SUMO domain has no further use and is cleaved off by the SUMO-protease ULP. Another IMAC chromatography separates the PAS-lysine from SUMO domain, which binds specifically to the column. PAS lysine can be identified in the flow through via UV absorption. Because PAS-lysine does not contain aromatic residues, its absorption at 280 nm is very low, while the absorption at 225 nm (amid bondage) is average.
The proper fractions are pooled and the solution is dialysed against 20 mM Tris/HCl buffer with 500 mM salt, pH 8. The linker molecule can now be biotinylated in a molar excess of 10 or 20. After incubation over night, dialysis removes biotin molecules which did not bind the PAS-lysine linker.
</P></DIV><H2><SPAN class="mw-headline" id="Human_cell_culture">Human cell culture</SPAN></H2><H3><SPAN class="mw-headline" id="Seeding_and_passaging_of_HEK293T_cells">Seeding and passaging of HEK293T cells</SPAN></H3><DIV class="white-box"><P>Frozen HEK 293T cells, having been stored in liquid nitrogen, were quickly thawed in a 37°C water bath. Cells were then transferred into a tube containing pre-warmed Dulbecco’s Modified Eagle Medium medium (DMEM, including 10% fetal calf serum, 100 U/ml penicillin and 100 μg/ml streptomycin). After having been spun down (300 g, 5 mins), the supernatant was discarded and the pellet resuspended in DMEM. The cell suspension was then transferred into culturing flasks and incubated at 37°C and 5% CO<SUB>2</SUB> until further use.
When reaching confluency, medium was removed from culture flasks and cells were washed twice with PBS. Cells were then treated with trypsine solution (0.05% in 1x PBS) and incubated at 37°C and 5% CO2 for 5 mins. After transferring cells into a 50 ml tube and adding 45 ml of medium, cells were counted and passaged into new culture vessels as deemed suitable.
</P></DIV><H3><SPAN class="mw-headline" id="Transfection_of_mammalian_cells_using_PEI_and_CaCl2">Transfection of mammalian cells using PEI and CaCl<SUB>2</SUB></SPAN></H3><DIV class="white-box"><P>We tried three driffrent ways of transfecting the cells: PEI, calcium phosphate and the commercial XTremeGene transfection solution by Roche GmbH. Polyeethylene imine is a cationic polymer that complexes DNA and transports it into the cell via endocytosis. Cells were splitted one day before transfection to reach a 50% confluency the next day in case of HEK293T and a cell number of 2x10<SUP>2</SUP>/ml for HEK293E. PEI and DNA where mixed in a 3:1 ratio diluted and incubated in ddH<SUB>2</SUB>O for 30 min. 4h prior to transfection media was changed to the cells to boost their metabolism.Just before transfection media was again chenaged to serum and antibiotic free media. In case of HEK293E (MEXi™) cells we switched from culture to transfection media and in case of HEK293T we switched to OptiMEM, which are both free from sera but supplemented with recombinant essential substrates for the cells. The transfection solution was added to the cells in a dropwise manner using ~5µg per 10cm dish (5x10<SUP>6</SUP> cells on day of seeding). For stable transfection the plasmids pOG44 (Flp recombinase) and pcDNA (target gene) were cotransfected using resulting in a total DNA amount of 10µg. 
For calcium phosphate transfection 31µl CaCl<SUB>2</SUB> diluted in 200µl and combined with the DNA were added to 250µl HBS solution containing disodium phosphate). Ca<SUP>2</SUP> and the phosphate preicipitate complexing the DNA. After 30 min of incubation at RT the solution is added to the cells which were prepared as above.
The XTremeGene transfection solution was handled in the same manner as PEI. We achieved high transfection rates with all methods.
</P></DIV><H3><SPAN class="mw-headline" id="Luciferase_assay_for_the_quantification_of_protein_expression">Luciferase assay for the quantification of protein expression</SPAN></H3><DIV class="white-box"><P>For the quantification of expression of either the receptor or of nanoluciferase-containing constructs (e.g. for the choice of signal peptides and quantification of hypoxia-dependent promoter activity), luciferase assays were conducted.
</P><P>Therefore, 4x10<SUP>4</SUP> cells were seeded into 6-well plates in 4 ml DMEM medium until reaching the desired confluency. For transfection, 100 μl OptiMEM were mixed with 3 μg DNA and 12 μl Turbofect, vortexed for 10 s and incubated at room temperature for 20 mins. After removing 1 ml of supernatant, the transfection mixture was added drop-wise and the plate was incubated at 37°C and 8.5% CO<SUB>2</SUB> overnight. The next day, medium was removed and DMEM medium (10% FCS, Pen/Strep) was added.
</P><P>For the luciferase assay, 50 μl of carefully removed medium were mixed with 50 μl luciferase assay buffer/sustrate mix from the NanoGlo® Luciferase Assay Kit, and incubated for 10 mins at RT. Measurements were performed using luminescence settings at 460 nm using a platereader.
</P></DIV><H3><SPAN class="mw-headline" id="Quantitative_reverse_transcription_Polymerase_chain_reaction_.28RT-qPCR.29">Quantitative reverse transcription Polymerase chain reaction (RT-qPCR)</SPAN></H3><DIV class="white-box"><P>For the analysis of the gene expression of the different receptors on mRNA-level, RT-qPCR was performed.
</P><P>Thus 1x10<SUP>6</SUP> and 1x10<SUP>5</SUP> cells were collected and spun down. The medium was discarded. QIAzol was added to the cell pellet and mRNA extraction was performed according to manufacturer's protocol (miRNeasy micro kit, QIAGEN). Afterwards mRNA concentration was measured by nanodrop. 
In addition the total mRNA was analysed for purity and integrity with an Bioanalyzer (Agilent Technologies). The measurement with a RNA Nano Chip was performed according to manufacturers protocol (Agilent technologies).<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP>. To prepare the mRNA for quantitative analysis 750 ng of each sample mRNA were used for reverse transcription (RT).
</P><P>The mastermix for the RT reaction was mixed as following:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Mastermix</B></TD><TD><B>1x</B></TD></TR><TR><TD>random hexamer Primer
</TD><TD>1 μl
</TD></TR><TR><TD>dNTPs
</TD><TD>1.5 μl
</TD></TR><TR><TD>RT
</TD><TD>1 μl
</TD></TR><TR><TD>5x buffer
</TD><TD>5 μl
</TD></TR><TR><TD>RNAse free water
</TD><TD>3.5 μl
</TD></TR><TR><TD>Template (750&amp;n;g
</TD><TD>10 μl
</TD></TR><TR><TD><B>SUM</B></TD><TD>22 μl
</TD></TR></TBODY></TABLE><P>Three reverse transcriptase negative (Neg. RT) samples as well as two controles without template mRNA were added as negative controls. The RT was performed according to <A rel="nofollow" class="external text" href="https://www.promega.de/resources/protocols/product-information-sheets/g/mmlv-reverse-transcriptase-protocol/">manufacturer's protocol</A> (M-MLV RT, Promega).
</P><P>The generated cDNA was then used for the quantitative PCR. The analysis was conducted on a Biorad CFX 1000 with 348 block using the SsoFast EvaGreen PCR mix.
</P></DIV><H3><SPAN class="mw-headline" id=".CE.94.CE.94CP_method">ΔΔC<SUB>P</SUB> method</SPAN></H3><DIV class="white-box"><P>The ΔΔC<SUB>P</SUB> method <SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP> is a method for relative quantification of expression level based on the comparison of C<SUB>P</SUB> values for the specific mRNA of cells with the gene of interest (GOI) to those without the GOI in combination with the C<SUB>P</SUB> values of a house keeping gene.
</P><P>To explain the method you can take a look at the sample calculation below:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>'
</TD><TD><B>NC</B></TD><TD>'
</TD><TD><B>1</B></TD><TD>'
</TD><TD><B>2</B></TD><TD>'
</TD></TR><TR><TD>sample
</TD><TD>A
</TD><TD>B
</TD><TD>C
</TD><TD>D
</TD><TD>E
</TD><TD>F
</TD></TR><TR><TD>C<SUB>P</SUB> value GOI
</TD><TD>33
</TD><TD>35
</TD><TD>32
</TD><TD>34
</TD><TD>14
</TD><TD>16
</TD></TR><TR><TD>C<SUB>P</SUB> value HKG
</TD><TD>22
</TD><TD>20
</TD><TD>19
</TD><TD>23
</TD><TD>21
</TD><TD>23
</TD></TR><TR><TD>Mean C<SUB>P</SUB> value GOI
</TD><TD>34
</TD><TD>/
</TD><TD>33
</TD><TD>/
</TD><TD>15
</TD><TD>/
</TD></TR><TR><TD>Mean C<SUB>P</SUB> value HKG
</TD><TD>21
</TD><TD>/
</TD><TD>21
</TD><TD>/
</TD><TD>22
</TD><TD>/
</TD></TR><TR><TD>ΔC<SUB>P</SUB> value GOI
</TD><TD>13
</TD><TD>/
</TD><TD>12
</TD><TD>/
</TD><TD>(-)7
</TD><TD>/
</TD></TR><TR><TD>ΔΔC<SUB>P</SUB> value GOI
</TD><TD>/
</TD><TD>/
</TD><TD>(-)1
</TD><TD>/
</TD><TD>(-)20
</TD></TR><TR><TD>2<SUP>-ΔΔC<SUB>P</SUB></SUP> value GOI
</TD><TD>/
</TD><TD>/
</TD><TD>2
</TD><TD>/
</TD><TD>1048576,00
</TD><TD>/
</TD></TR><TR><TD><B>relative gene expression</B></TD><TD>/
</TD><TD>/
</TD><TD><B>low</B></TD><TD>/
</TD><TD><B>high</B></TD><TD>/
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Biophysical_Methodology">Biophysical Methodology</SPAN></H2><H3><SPAN class="mw-headline" id="Mass_spectrometry">Mass spectrometry</SPAN></H3><DIV class="white-box"><P>For confirmation of the molecular weight of proteins, electron spray ionisation-quadrupole time of flight was used. Protein masses were determined as a result from the mass/charge ratio (m/z). Herefore, 50-100 μl of a 0.1 mg/ml protein solution were dialyzed with 10 mM ammonium acetate (pH 6.6) overnight. For measurements, a mixture of 140 μl 10 mM ammonium acetate (pH 6.6), 40 μl acetonitrile, 20 μl of the dialized protein solution and 1 μl formic acid was injected.
</P></DIV><H3><SPAN class="mw-headline" id="Fluorescence_titration">Fluorescence titration</SPAN></H3><DIV class="white-box"><P>Fluorescence titration is a method that utilizes the change of intrinsic protein fluorescence upon ligand binding (as mainly mediated by aromatic amino acid side chains) in order to quantify binding properties. This way, the binding affinity (as characterized by the K<SUB>D</SUB> value) can be determined. 
</P><P>For the measurement, 2 ml of continuously stirred protein solution (1 μM according to the respectively monomer; in 50 mM Tris/HCl, 100 mM NaCl; pH 8) were titrated by the continuous addition of 100 μM D-biotin solution in the same buffer. For the measurements, temperatures of 20°C, an excitation wavelength of 295 nm and emission wavelengths of 350 nm were used (slid width of 5 mm, respectively) and the signal integrated over 5 s. The titrant was hereby added in 2 μl steps until a final titrant concentration of 2 μM.
For blank measurement only buffer was used.
</P></DIV><H3><SPAN class="mw-headline" id="Circular_dichroism_spectroscopy">Circular dichroism spectroscopy</SPAN></H3><DIV class="white-box"><P>For the analysis of protein folding states, circular dichroism spectroscopy was used, as it allows drawing conclusions about the formation of protein secondary structures.
</P><P>To determine the ideal wavelength for measurement a UV-CD spectrum was recorded for folded and unfolded state of each protein. Therefore each protein solution (10 μM according to the respectively monomer; in 20 mM KH<SUB>2</SUB>PO<SUB>4</SUB>, 50 mM K<SUB>2</SUB>SO<SUB>4</SUB>; pH7.5) were measureed in a cuvette with a 1 mm thickness. The spectrum was recorded between 190 and 250 nm at 20 °C. By comparing this two spectra the wavelength were both spectra show the maximum of variance were selected.
Now a denaturation and renaturation spectrum were recorded for each protein at 213 nm between 20 and 95 °C. As an exception, enhanced monomeric avidin was measured at 203 nm.
For blank measurement only buffer were used.
</P></DIV><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><DIV class="white-box"><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Schroeder, Andreas, et al. &quot;The RIN: an RNA integrity number for assigning integrity values to RNA measurements.&quot; BMC molecular biology 7.1 (2006): 1.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Livak, Kenneth J., and Thomas D. Schmittgen. &quot;Analysis of relative gene expression data using real-time quantitative PCR and the 2− ΔΔCT method.&quot; methods 25.4 (2001): 402-408.</SPAN></LI></OL></DIV> 
</DIV></DIV><DIV class="row"><FOOTER class="col-sm-9 col-sm-offset-3 noprint page-footer clearfix white-box"><A class="btt visible-xs-block" href="#top"><SPAN class="sr-only">Back to top</SPAN></A><DIV class="row"><DIV class="col-sm-4 social"><H2>Follow us on:</H2></DIV><DIV class="col-sm-4 universities"><H2>LMU &amp; TUM Munich</H2><P>United team from Munich's universities</P></DIV><DIV class="col-sm-4 address"><H2>Contact us:</H2><H3>Address</H3><P>iGEM Team TU-Munich
                     Emil-Erlenmeyer-Forum 5
                     85354 Freising, Germany</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>