<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_LMU-TUM_Munich_Labjournal skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:LMU-TUM Munich/Labjournal</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="row"><DIV class="col-sm-3 col-xs-8 fixed-sm z-1"><DIV class="sidebar-nav"><DIV class="navbar" role="navigation"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".sidebar-navbar-collapse"><SPAN class="sr-only">Toggle menu</SPAN></BUTTON><SPAN class="visible-xs navbar-brand">Menu</SPAN></DIV><DIV class="navbar-collapse collapse sidebar-navbar-collapse"><UL id="main-menu" class="nav navbar-nav labjournal"><LI class="menu_item home"><A href="https://2016.igem.org/Team:LMU-TUM_Munich"><SPAN class="visible-xs-block">Home</SPAN></A></LI><LI class="dropdown menu_item team"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI class="members"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Team">Members</A></LI><LI class="collaborations"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Collaborations">Collaborations</A></LI><LI class="sponsors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Sponsors">Sponsors</A></LI><LI class="press"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Press">Press</A></LI><LI class="gllry"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Gallery">Gallery</A></LI></UL><LI class="dropdown menu_item project"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Project </A></LI><LI class="description"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Description">Description</A></LI><LI class="design"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Design">Design</A></LI><LI class="proteins"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proteins">Proteins</A></LI><LI class="receptors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Receptors">Receptors</A></LI><LI class="linker"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Linkerchemistry">Linker Chemistry</A></LI><LI class="poly"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proof">Polymerization</A></LI><LI class="bioprinting"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Demonstrate">Bioprinting</A></LI><LI class="hypoxia"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hypoxia">Hypoxia</A></LI><LI class="medical"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Medical">Medical Application</A></LI><LI class="interlab"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/InterLab">InterLab</A></LI><LI class="software"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Software">Software</A></LI><LI class="dropdown menu_item notebook"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Notebook </A></LI><LI class="labjournal"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Labjournal">Labjournal</A></LI><LI class="materials"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Materials">Materials</A></LI><LI class="methods"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Methods">Methods</A></LI><LI class="howtos"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Howtos">How Tos</A></LI><LI class="dropdown menu_item parts"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Parts </A></LI><LI class="parts-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Parts">Parts</A></LI><LI class="basic-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Basic_Part">Basic Parts</A></LI><LI class="composite-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Composite_Part">Composite Parts</A></LI><LI class="part-collection"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Part_Collection">Part Collection</A></LI><LI class="menu_item hardware"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hardware">Hardware</A></LI><LI class="menu_item model"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Model">Modeling</A></LI><LI class="menu_item safety"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Safety">Safety</A></LI><LI class="dropdown menu_item human_practices dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Human Practices </A></LI><LI class="human_practices-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Human_Practices">Human Practices</A></LI><LI class="silver"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Silver">Silver</A></LI><LI class="gold"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Gold">Gold</A></LI><LI class="integrated-practices"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Integrated_Practices">Integrated Practices</A></LI><LI class="engagement"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Engagement">Engagement</A></LI><LI class="dropdown menu_item entrepreneur dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Entrepreneurship</A></LI><LI class="entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Entrepreneurship">Entrepreneurship</A></LI><LI class="supporting-entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Supporting_Entrepreneurship">Supporting Entrepreneurship</A></LI><LI class="menu_item attributions"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV class="col-sm-8 col-sm-offset-3"><DIV id="wikicontent-container"><DIV id="wikicontent"><H2><SPAN class="mw-headline" id="Labjournal">Labjournal</SPAN></H2><FORM id="labselect"><FIELDSET class="ui-widget ui-widget-content ui-corner-left"><B>Display:</B><B style="color: rgb(115, 208, 255);">General</B><B style="color: rgb(166, 126, 166);">Streptavidin</B><B style="color: rgb(116, 183, 112);">Linkers</B><B style="color: rgb(255, 122, 97);">Receptor</B><B style="color: rgb(0, 32, 96);">Optogenetics</B><B style="color: rgb(160, 0, 39);">Polymerization</B><B style="color: rgb(219, 48, 130);">Hypoxia</B><B style="color: rgb(255, 255, 0);">Kill-Switch</B><A href="#" id="ExAll">Expand All ...</A><A href="#" id="ColAll">Collapse All ...</A><B>Jump to:</B><A href="#Week_0">Week 0</A> sample<A href="#Week_1">Week 1</A> May 16–22<A href="#Week_2">Week 2</A> May 23–29<A href="#Week_3">Week 3</A> May 30–June 05<A href="#Week_4">Week 4</A> June 06–12<A href="#Week_5">Week 5</A> June 13–19<A href="#Week_6">Week 6</A> June 20–26<A href="#Week_7">Week 7</A> June 27–July 03<A href="#Week_8">Week 8</A> July 04–10<A href="#Week_9">Week 9</A> July 11–17<A href="#Week_10">Week 10</A> July 18–24<A href="#Week_11">Week 11</A> July 25–31<A href="#Week_12">Week 12</A> Aug 01–07<A href="#Week_13">Week 13</A> Aug 08–14<A href="#Week_14">Week 14</A> Aug 15–21<A href="#Week_15">Week 15</A> Aug 22–28<A href="#Week_16">Week 16</A> Aug 29–Sep 04<A href="#Week_17">Week 17</A> Sep 05–11</FIELDSET></FORM><DIV id="ladder" class="ui-widget ui-widget-content ui-corner-right"><B>1 kbp GeneRuler:</B><B>100 bp GeneRuler:</B><B>PageRuler Plus:</B></DIV><H1><SPAN class="mw-headline" id="Samples">Samples</SPAN></H1><DIV class="week" id="Week_0"><DIV class="linkers"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_Phytochrome_B_.282.E2.80.93908_N-terminal_amino_acids.29_.28BBa_K801031.2C_RFC25.2C_pSB1C3.29">Transformation of <I>E. coli</I> XL1 blue with Phytochrome B (2–908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3)</SPAN></H3><P><B>Investigator: Jeff, Rosario</B></P><P><B>Aim of the experiment:</B> Transformation of Phytochrome B for protein fusion
</P><P><B>Procedure:</B></P><UL><LI> CaCl<SUB>2</SUB>-competent <I>E. coli</I> XL1-Blue cells were taken from the stock in –80 °C freezer and gently thawed on ice</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min heat shock at 37 °C</LI></UL><UL><LI> Added 1 ml LB medium to each tube</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13,000 rpm and the supernatant was dicarded</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_pTUM100_with_pGAL.2C_pTEF1.2C_pTEF2.2C_pADH_and_RFC25_compatible_RFP_generator">Miniprep of pTUM100 with pGAL, pTEF1, pTEF2, pADH and RFC25 compatible RFP generator</SPAN></H3><P><B>Investigator: Jeff, Rosario</B></P><P><B>Aim of the experiment:</B> Miniprep of pTUM100 with pGAL, pTEF1, pTEF2, pADH and RFC25 compatible RFP generator
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_RFP-Generator_.28RFC25.2C_pSB1C3.29">Sequencing of RFP-Generator (RFC25, pSB1C3)</SPAN></H3><P><B>Investigator: Jeff, Rosario</B></P><P><B>Aim of the experiment:</B> Sequencing of RFP-Generator (RFC25, pSB1C3)
</P><P><B>Procedure:</B></P><P>Sequencing batch were prepared after manufacturer's protocol. (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer)
</P></DIV><DIV class="optogenetics"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_with_Phytochrome_B_.282-908_N-terminal_amino_acids.29_.28BBa_K801031.2C_RFC25.2C_pSB1C3.29">Picking of of <I>E. coli</I> XL1 blue with Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3)</SPAN></H3><P><B>Investigator: Jeff, Rosario, Florian</B></P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue with Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> pSB1C3 plasmid with BBa_K801031 (PhyB 2 - 908 aa, RFC25): Colonies were picked from chloramphenicol plates.</LI></UL><UL><LI> Picked pipette tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contain 4 mL of LB-medium + 4 µL chloramphenicol(1000x).</LI></UL><UL><LI> 4 colonies were picked.</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_RFP-generator_.28RFC25.2C_pSB1C3.2C_P4_.26_P5.29">Analytical digestion and gelelectrophoresis of RFP-generator (RFC25, pSB1C3, P4 &amp; P5)</SPAN></H3><P><B>Investigator: Jeff, Rosario, Florian</B></P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of RFP-generator (RFC25, pSB1C3, P4 &amp; P5).
</P><P><B>Procedure:</B></P><UL><LI> Batch for analytical digestion for P4 with NgoMIV+AgeI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P4
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion for P5 with NgoMIV+AgeI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P5
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.</LI></UL><P><B>Results:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder DNA ladder
</TD><TD><B>P4</B></TD><TD><B>P5</B></TD></TR><TR><TD><B>Mutation successful</B></TD><TD><B>Mutation successful!</B></TD></TR></TBODY></TABLE><UL><LI> Parts are compliant and do not contain RFC25 forbidden restriction sites.</LI></UL><P><A href="/File:TUM13_20130423_RFP_Generator_RFC25_AgeI_NgoMIV.png" title="File:TUM13 20130423 RFP Generator RFC25 AgeI NgoMIV.png">500px</A></P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_pTUM_vectors_with_pGAL.2C_pADH.2C_pTEF1.2C_pTEF2">Sequencing of pTUM vectors with pGAL, pADH, pTEF1, pTEF2</SPAN></H3><P><B>Investigator: Jeff, Rosario, Florian</B></P><P><B>Aim of the experiment:</B> Sequencing of pTUM vectors with pGAL, pADH, pTEF1, pTEF2
</P><P><B>Procedure:</B></P><P>Sequencing batch were prepared after manufacturer's protocol. (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different vectors we sequenced received the following barcodes:
</P><P>- ADH in pTUM100:   FR01002265
</P><P>- TEF1 in pTUM100:  FR01002266
</P><P>- TEF2 in pTUM100:  FR01002266
</P><P>- GAL in pTUM100:   FR01002268
</P><P>
Sequencing of TEF2 in pTUM100 was not interpretable. The other sequences were consistent with the sequences in the parts registry. 
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_Phytochrome_B_.282-908_N-terminal_amino_acids.29_.28BBa_K801031.2C_RFC25.2C_pSB1C3.29">Miniprep of Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3)</SPAN></H3><P><B>Investigator: Jeff, Florian</B></P><P><B>Aim of the experiment:</B> Miniprep of Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)</LI></UL></DIV><DIV class="linkers"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_Phytochrome_B_.282-908_N-terminal_amino_acids.29_.28BBa_K801031.2C_RFC25.2C_pSB1C3.29.2C_P7_-_P10">Analytical digestion and gelelectrophoresis of Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3), P7 - P10</SPAN></H3><P><B>Investigator: Jeff, Florian</B></P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3), P7 - P10.
</P><P><B>Procedure:</B></P><UL><LI>Batch for analytical digestion for P7 with NgoMIV+AgeI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P7
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion for P8 with NgoMIV+AgeI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P8
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion for P9 with NgoMIV+AgeI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P9
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion for P10 with NgoMIV+AgeI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P10
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.</LI></UL><P><B>Results:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder DNA ladder
</TD><TD><B>P7</B></TD><TD><B>P8</B></TD><TD><B>P9</B></TD><TD><B>P10</B></TD></TR><TR><TD><B>Part is correct</B></TD><TD><B>Part is correct</B></TD><TD><B>Part is correct</B></TD><TD><B>Part is correct</B></TD></TR></TBODY></TABLE><P><A href="/File:TUM13_20130424_PhytochromeB_RFC25_AgeI_NgoMIV.png" title="File:TUM13 20130424 PhytochromeB RFC25 AgeI NgoMIV.png">500px</A></P></DIV><DIV class="linkers"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with">Transformation of <I>E. coli</I> XL1 blue with</SPAN></H3><P><B>Investigator: </B></P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> µl of DNA was added to 100 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of  µl LB-medium to each tube.</LI></UL><UL><LI> The cell suspension was plated on ampicillin plates (inclusive rescue plate) and incubated over night at 37 °C in the cell-culture shaker.</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_1.2C_May_16th_-_May_22nd">Week 1, May 16th - May 22nd</SPAN></H1><DIV class="week" id="Week_1"><H2><SPAN class="mw-headline" id="Monday.2C_May_16th"><B>Monday, May 16th</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Streptavidin_Plasmid_Check">Streptavidin Plasmid Check</SPAN></H3><P><B>Investigator: JB, LK, JH</B></P><P><B>Aim of the experiment:</B> Verify cloning
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep following manufacturer's protocol (QIAprep MiniPrep, Qiagen) (4 clones each of pSA1, pSAm1 in pASK75) 			</LI><LI> Analytic digestion with 0.25 <I>µ</I>L XbaI, 0.25 <I>µ</I>L HindIII (HF), 1 <I>µ</I>L SmartCut Buffer, 5 <I>µ</I>L plasmid DNA, 3.5 <I>µ</I>l H<SUB>2</SUB>O</LI><LI> 5 <I>µ</I>L analyzed in gel electrophoresis (1% agarose)</LI></UL><P><B>Results:</B> Successful cloning verified. Stored at –20 °C
</P><UL><LI> Lane 1: 5 <I>µ</I>L Thermo Fisher, 1 kb ladder</LI></UL><UL><LI> Lanes 2–9: 5 <I>µ</I>L digestions of P6–P13, band of SA (mut1) at about 300 bp, band of digested plasmid at about 3,000 bp</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Streptavidin_Expression:_Transform_BL21">Streptavidin Expression: Transform BL21</SPAN></H3><P><B>Investigator: JB, JH</B></P><P><B>Aim of the experiment:</B> Express pSA1 and pSAm1 in <I>E. coli</I> BL21
</P><P><B>Procedure:</B></P><UL><LI> Transformation into competent <I>E. coli</I> BL21 according to protocol of P6 and P10 </LI></UL><P><B>Results:</B> Plates (LB Amp) in incubator (37 °C) for further processing 
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_May_17th"><B>Tuesday, May 17th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="SDS_Gel_Analysis">SDS Gel Analysis</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Analyze collagen 1/2, eGFP, fraction 30 of egg precipitation <I>via</I> SDS gel
</P><P><B>Procedure:</B></P><UL><LI> Mixed 80 <I>µ</I>L samples with 20 <I>µ</I>L SDS buffer</LI></UL><UL><LI> Heated at 95 °C for 10 min</LI></UL><UL><LI> 1 d staining, 1 d unstaining</LI></UL><P><B>Results:</B> Successful cloning verified. Stored at –20 °C
</P><UL><LI> Lane 1: 8 <I>µ</I>L marker (Thermo Fisher #26610)</LI></UL><UL><LI> Lane 2: Fraction 30 (IEC), 3 <I>µ</I>L, band at 35 kDa, avidin expected at 16 kDa</LI></UL><UL><LI> Lane 3: eGFP, 12 <I>µ</I>L, band at 27 kDa, eGFP expected at 27 kDa, many impurities</LI></UL><UL><LI> Lane 4: Collagen 1, 12 <I>µ</I>L, no sharp band</LI></UL><UL><LI> Lane 5: Collagen 1, 12 <I>µ</I>L, no sharp band</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="MiniPreps_pSb1C3.E2.80.93AviTag.2C_.E2.80.93A3C5_and_pASK75.E2.80.93.28SA1.29.2C_.E2.80.93.28SAm1.29">MiniPreps pSb1C3–AviTag, –A3C5 and pASK75–(SA1), –(SAm1)</SPAN></H3><P><B>Investigator: CR, CG</B></P><P><B>Aim of the experiment:</B> Verify cloning
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep was performed following manufacturer's protocol (QIAprep MiniPrep, Qiagen)</LI></UL><UL><LI> Analytic digestion with 0.25 <I>µ</I>L XbaI, 0.25 <I>µ</I>L HindIII (HF) for pASK plasmids and 0.25 <I>µ</I>L NgomIV, 0.25 <I>µ</I>L AgeI (HF) for pSb1C3 plasmids, 1 <I>µ</I>L SmartCut Buffer, 5 <I>µ</I>L plasmid DNA, 3.5 <I>µ</I>L H<SUB>2</SUB>O</LI></UL><UL><LI> 5 <I>µ</I>L on 1% agarose gel electrophoresis</LI></UL><P><B>Results:</B> Successful cloning verified for pASK plasmids. Repeat pSb1C3 plasmids. Stored at –20 °C
</P><UL><LI> Lane 1: 5 <I>µ</I>L Thermo Fisher, 1 kb Ladder</LI></UL><UL><LI> Lane 2: 5 <I>µ</I>L digestion of pSb1C3–AviTag</LI></UL><UL><LI> Lane 3: 5 <I>µ</I>L digestion of pSb1C3–A3C5</LI></UL><UL><LI> Lane 5: 5 <I>µ</I>L digestion of pASK75(SA1), EB elution</LI></UL><UL><LI> Lane 6: 5 <I>µ</I>L digestion of pASK75(SAmut1), EB elution</LI></UL><UL><LI> Lane 7: 5 <I>µ</I>L digestion of pASK75(SAmut1), H<SUB>2</SUB>O elution</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inoculation_of_Pre-Culture_with_BL21_.28pASK75_.28SA1.29.29_in_LB_Medium">Inoculation of Pre-Culture with BL21 (pASK75 (SA1)) in LB Medium</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Pre-culture for streptavidin expression in TB medium
</P><P><B>Procedure:</B></P><UL><LI> Added 50 <I>µ</I>L ampicillin to 50 mL LB medium </LI></UL><UL><LI> Picked colonies from BL21 (pASK75 (SA1))</LI></UL><UL><LI> Inoculated LB medium</LI></UL><UL><LI> Incubated at 30 °C overnight</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_May_18th"><B>Wednesday, May 18th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Repeat_Analytical_Gel_of_pSb1C3.E2.80.93AviTag.2C_.E2.80.93A3C5">Repeat Analytical Gel of pSb1C3–AviTag, –A3C5</SPAN></H3><P><B>Investigator: CG, CR</B></P><P><B>Aim of the experiment:</B> Verify cloning
</P><P><B>Procedure:</B></P><UL><LI> Analytic digestion with 0.25 <I>µ</I>L NgoMIV, 0.25 <I>µ</I>L AgeI (HF) for pSb1C3 plasmids, 1 <I>µ</I>L SmartCut Buffer, 8.5 <I>µ</I>L plasmid DNA		</LI><LI> 10 <I>µ</I>L analyzed in gel electrophoresis (2% agarose)</LI></UL><P><B>Results:</B></P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inoculation_of_BL21_.28pASK75_.28SA1.29.29_Culture_in_2_L_TB_Medium_and_Induction_of_Streptavidin_Production_by_Addition_of_Tetracycline">Inoculation of BL21 (pASK75 (SA1)) Culture in 2 L TB Medium and Induction of Streptavidin Production by Addition of Tetracycline</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Produce streptavidin
</P><P><B>Procedure:</B></P><UL><LI> Ampicillin (2 mL) was added to the medium (1:1000)	</LI><LI> The pre-culture (50 mL) was poured into the medium</LI><LI> Culture incubated at 37 °C and 140 rpm until OD<SUB>550</SUB> reached 0.5</LI><LI> To induce streptavidin expression, anhydro-tetracycline (200 <I>µ</I>L) was added to the culture (1:10,000)</LI><LI> The culture was incubated at 37 °C and 140 rpm for 4 hours</LI></UL><P><B>Results:</B> Streptavidin expression by BL21
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Expression_and_Harvest_of_Streptavidin_.28pASK75_.28SA1.29.29_in_BL21_in_TB_Medium">Expression and Harvest of Streptavidin (pASK75 (SA1)) in BL21 in TB Medium</SPAN></H3><P><B>Investigator: CR, JB, JH </B></P><P><B>Aim of the experiment:</B> Recombinant expression and purification of streptavidin
</P><P><B>Procedure:</B></P><UL><LI> After expression, cultures were transferred into centrifuge tubes and spun down in the centrifuge (4 °C, 5000 rpm, 20 min, F4X1L rotor) </LI><LI> The supernatant was discarded, and the pellet was transferred into a beaker of sufficient size and resuspended in Tris Buffer B cooled to 4 °C (50 mM Tris/HCl pH = 8.0, 1 mM EDTA)</LI><LI> The solution was homogenized in the PANDA (ask supervisor)</LI><LI> The resulting lysate was transferred into centrifuge tubes and spun down (4 °C, 18,000 rpm, 10 min, XX34-rotor). The supernatant was discarded, and the pellet was resuspended in 6 M Gua-HCl (pH = 1.5) at 4 °C overnight.</LI></UL></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Dialysis_of_eGFP">Dialysis of eGFP</SPAN></H3><P><B>Investigator: NA, JH, CR </B></P><P><B>Aim of the experiment:</B> Purify eGFP
</P><P><B>Procedure:</B></P><UL><LI> eGFP was thawed on ice</LI><LI> eGFP was then poured into a dialysis hose (cut-off 14 kDa)</LI><LI> The hose was then placed in ice-cold Tris/HCl 20 mM pH 8.0</LI><LI> Dialyzed at 4 °C overnight (XX34 rotor). The supernatant was discarded, and the pellet was </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="MiniPrep_of_quickchanged_pNGAL146-A2_EspP">MiniPrep of quickchanged pNGAL146-A2 EspP</SPAN></H3><P><B>Investigator: NA </B></P><P><B>Aim of the experiment:</B> Extraction of pNGAL146-A2 plasmid from XL1 blue
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep was performed after manufacturer's protocol (QIAprep MiniPrep, Qiagen)</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Sequencing_of_P14_.28Avi-Tag.29.2C_P15_.28A3C5.29_.26_P19_.28Quick_change_.28QC.29_EspP_in_pNGAL-A2.29">Sequencing of P14 (Avi-Tag), P15 (A3C5) &amp; P19 (Quick change (QC) EspP in pNGAL-A2)</SPAN></H3><P><B>Investigator: CR, NA </B></P><P><B>Aim of the experiment:</B> Sequencing of P14 (Avi-Tag), P15 (A3C5) &amp; P19 (Quick change (QC) EspP in pNGAL-A2) 
</P><P><B>Procedure:</B></P><UL><LI> Sequencing batches were prepared after manufacturer's protocol (15 µL plasmid DNA (50-100 µM) and 2 µL sequencing primer)</LI><LI> The different plasmids we prepared received the following barcodes:</LI><LI> P14        : FR11326653</LI><LI> P15        : FR11326655</LI><LI> P19 (K4): FR11326654</LI><LI> P16 (K1): FR11326652</LI><LI> P17 (K2): FR11326651</LI><LI> P18 (K3): FR11326650</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Digestion_of_P16_.2C_P17.2C_P18_.26_P19_.28all_QC_EspP_in_pNGAL-A2.29_with_AgeI_.26_HindIII_.2B_analytical_gel">Digestion of P16 , P17, P18 &amp; P19 (all QC EspP in pNGAL-A2) with AgeI &amp; HindIII + analytical gel</SPAN></H3><P><B>Investigator: NA, JH, CR </B></P><P><B>Aim of the experiment:</B> Verification of success of quickchange
</P><P><B>Procedure:</B></P><UL><LI> analytic digestion with: 0,25 µl HindIII (HF), 0,25 µl AgeI (HF), 1 µl SmartCut Buffer, 500 ng plasmid-DNA, fill up with ddH2O (Vtotal= 10µL)</LI><LI> 10 µl on 2% agarose gel for electrophoresis </LI></UL><P><B>Results:</B> No signal at 600 bp --&gt; quickchange seems to be successful (waiting for sequencing)
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_May_19th"><B>Thursday, May 19th</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Re-Sequencing_of_P19">Re-Sequencing of P19</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Re-Sequencing of P19
</P><P><B>Procedure:</B>Sequencing batches were prepared after manufacturer's protocol (15 µL plasmid DNA (50-100 µM) and 2 µL sequencing primer)
</P><P>The different plasmids we prepared received the following barcodes:
</P><UL><LI>P19 (K4): FR11326649</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_A3C5_and_Avi-Tag_into_pSB1C3">Cloning of A3C5 and Avi-Tag into pSB1C3</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Re-Trafo of pSB1C3 RFP for later on: digestion, dephosphorylation and cloning
</P><P><B>Procedure:</B> transformation according to protocol of P4 E. Coli XL1 
</P><P><B>Result:</B> plates (LB Cam) in incubator for further processing (37 °C)
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Streptavidin_refolding">Streptavidin refolding</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Refolding of denaturated Streptavidin
</P><P><B>Procedure:</B> After the pellet had almost completely dissolved in 6M GdmCl, the solution was spun down (4°C, 20 mins, 18,000 rpm). The supernatant was transferred carefully into a falcon tube and the pellet was cast away. Via a hydraulic pump (flow rate: 2x10 ml/min) the lysate was transferred Into 5L PBS 1x. Afterwards the pump was cleaned with technical isopropanol and ELGA water. The solution was stirred overnight at 4°C for refolding.
</P></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Biotinylation_of_BSA">Biotinylation of BSA</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Biotinylation of BSA
</P><P><B>Procedure:</B> A 100 µM (=6.8 mg/ml) solution of BSA (Albumin fraction V, pH=7, in the fridge in the central lab) was created (V=10 ml). 220 µl of a 100 mM Biotin stock were added. The mixture was stored overnight Iin the fridge (4°C).
</P><P><B>Result:</B> Hopefully biotinylated BSA mixture in the fridge (4°C).
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_May_20th"><B>Friday, May 20th</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Qualitative_analysis_of_streptavidin_expression">Qualitative analysis of streptavidin expression</SPAN></H3><P><B>Investigator: CG </B></P><P><B>Aim of the experiment:</B> SDS gel analysis of recombinant strepatividin expression
</P><P><B>Procedure:</B>  mixing of 80 µl sample with 20 µl SDS buffer and heating at 95°C for 10 min. 1 h staining, 1 night unstaining
</P><P><B>Result:</B></P><UL><LI> 1. Lane: 8 µl Marker (Thermo Fisher #26610)</LI><LI> 2. Lane: 12 µl culture aliquot, before induction</LI><LI> 3. Lane: 12 µl culture aliquot, after induction</LI><LI> 4. Lane: 3 µl culture aliquot of the lysed pelet, Strepatvidin expected at about 16 kDa</LI><LI> 5. Lane: 3 µl culture aliquot of the supernatend after lysis, no Strepatvidin expected at about 16 kDa</LI><LI> 6. Lane: 1.5 µl culture aliquot of the lysed pelet, Strepatvidin expected at about 16 kDa</LI><LI> 7. Lane: 1.5 µl culture aliquot of the supernatend after lysis, no Streptavidin expected at about 16 kDa</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_A3C5_and_Avi-Tag_into_pSB1C3_2">Cloning of A3C5 and Avi-Tag into pSB1C3</SPAN></H3><P><B>Investigator: LK</B></P><P><B>Aim of the experiment:</B> Inoculation of pre-culture with E. coli XL1 (pSb1C3 -RFP) in LB-Chloramphenicol-medium
</P><P><B>Procedure:</B>  
Picking of colonies for E. coli XL1 (pSb1C3 -RFP)
Inoculate in 5 ml LB-Chloramphenicol-medium
Incubate at 37°C over night at 200 rpm
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_Biobricks_in_XL1-blue">Transformation of Biobricks in XL1-blue</SPAN></H3><P><B>Investigator: NA, JH</B></P><P><B>Aim of the experiment:</B> Transformation
</P><P><B>Procedure:</B>  
-10 µl dd H2O in well of interest (standard distribution kit) 
-1 µl Plasmid (out of well) to cells
-Transformation according to the SOP 
Used bricks:
K577893, B0015, R0040, B0032, I14033, K747096
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Ammonium_sulfate_precipitation_of_streptavidin">Ammonium sulfate precipitation of streptavidin</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Reduction of the protein solution volume and precipitation of streptavidin
</P><P><B>Procedure:</B> The 5 L protein solution was spun down (20 mins, 5,000 rpm) and the supernatant transferred into a beaker. In order to lower the volume of the solution for ammonium sulfate precipitation, the solution was first filtered via a membrane crossflow pump (membrane: Sartocon 0.45 µm, thick membrane).
</P></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Dialysis_of_biotinylated_BSA">Dialysis of biotinylated BSA</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> purification of biotinylated BSA
</P><P><B>Procedure:</B> dialysis against Tris/HCl 20 mM, pH 8, 10 mM NaCl over night; 
cut off : 14 kDa
</P></DIV></DIV><H1><SPAN class="mw-headline" id="Week_2_.28May_23rd_-_May_29th.29">Week 2 (May 23rd - May 29th)</SPAN></H1><DIV class="week" id="Week_2"><H2><SPAN class="mw-headline" id="Monday.2C_May_23th"><B>Monday, May 23th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_A3C5_and_Avi-Tag_into_pSB1C3_3">Cloning of A3C5 and Avi-Tag into pSB1C3</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Cloning of A3C5 and Avi-Tag into pSB1C3
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep was performed after manufacturer's protocol (QIAprep MiniPrep, Qiagen)</LI><LI> pSB1C3 was digested with AgeI and NgOMIV (10 µL plasmid + 1.5 µL AgeI + 1.5 µL NgoMIV + 3 µL CutSmart + 14 µL ddH2O; incubation for 1h, 37°C)</LI><LI> FastAP was added and the reaction was incubated for another 2h at 37°C</LI><LI> The digestion was purified by gelelctrophoresis (1%, 75V, 1h)</LI><LI> Plasmid backbone was cut from the gel</LI></UL><P><B>Results:</B></P><P>Signal 1: linearized pSB1C3 (successfull digestion)
</P><P>Signal 2: supercoiled pSB1C3 (digestion not successfull)
</P><P>Signal 3: RFP-generator
</P><P>--&gt; next time gel should run longer (just if you need the back bone)
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Filtration_of_Streptavidin_and_precipitation_with_Ammonium-sulfate">Filtration of Streptavidin and precipitation with Ammonium-sulfate</SPAN></H3><P><B>Investigator: MP, CR</B></P><P><B>Aim of the experiment:</B> Concentration of Streptavidin
</P><P><B>Procedure:</B> 
The solution was filtered via a membrane crossflow pump (membrane: Sartocon 0.45 µm, thick membrane).
The final volum was 0.5L
The solution was precipitated by addition of Ammonium-sulfate (2 steps: 1. 40% Ammonium-sulfate; 2. 70% Ammonium-sulfate)
After the first addition of ammonium-sulfate the solution was stirred (1h, 4°C)
Afterwards the solution was centrifuged (10.000 rpm; 30 min) and the supernatant was used for the second precipitation step
The solution was stirred again (over night, 4°C)
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_pre-culture_with_BioBricks_in_pSB1C3_in_LB-Cam">Inoculation of pre-culture with BioBricks in pSB1C3 in LB-Cam</SPAN></H3><P><B>Investigator: JH, JL</B></P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B>  
Add 50 µL chloramphenicol in 50 mL LB-medium
Picking colonies from K577893; B0015; B0032 (in pSB1C3) *
Inoculate 4 mL medium (CAM)
Incubate at 37°C over night
</P><UL><LI>Following cultures didn´t grow and were therefore not inoculated:</LI></UL><P>R0040; K747096; I14033
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Retransformation_of_Biobricks_in_XL1-blue">Retransformation of Biobricks in XL1-blue</SPAN></H3><P><B>Investigator: JH, JL, EF</B></P><P><B>Aim of the experiment:</B> Transformation
</P><P><B>Procedure:</B>  
10 µl dd H2O in well of interest (standard distribution kit) 
1 µl Plasmid (out of well) to cells
Transformation according to the SOP 
Used bricks:
K577893, B0015, R0040, B0032, I14033, K747096
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_May_24th"><B>Tuesday, May 24th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_pre-culture_with_BioBricks_in_pSB1C3_in_LB-Cam_2">Inoculation of pre-culture with BioBricks in pSB1C3 in LB-Cam</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B></P><P><B>Procedure: </B> 
Add 50 µL chloramphenicol in 50 mL LB-medium 
Picking colonies from K577893; B0015; B0032 (in pSB1C3) *
Inoculate 4 mL medium (CAM)
Incubate at 37°C over night
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_A3C5_and_Avi-Tag_into_pSB1C3_4">Cloning of A3C5 and Avi-Tag into pSB1C3</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Gelextraction of Signal 1 from pSB1C3-digestion
</P><P><B>Procedure:</B>   
Gelextraction was performed after manufacturer's protocol (Gelextractionkit, Qiagen)
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Ammonium_sulfate_precipitation">Ammonium sulfate precipitation</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Precipitation the 70% saturated ammonium sulfate solution
</P><P><B>Procedure:</B>   The ammonium sulfate solution, which actually was a suspension, was transferred into centrifuge tubes and spun down (10,000 rpm, 45 mins). As it turns out, unfortunately too much ammonium sulfate was used for the precipitation (wrong table). The precipitate was brought back into solution (20 mM  Tris/HCl, pH 8.0) and loaded onto an SDS gel for analysis, along with other samples (flowthrough of the filtration from the day before). 
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="MiniPrep_of_BioBricks_in_pSB1C3_.28inoculated_on_May_23th.29">MiniPrep of BioBricks in pSB1C3 (inoculated on May 23th)</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Aim of the experiment:</B> Extraction of BioBricks B0032, B0015 and K577893 in pSB1C3 from XL1 blue
</P><P><B>Procedure:</B>  
MiniPrep was performed after manufacturer's protocol (QIAprep MiniPrep, Qiagen)
concentrations measured (in ng/µL):
(B0032) 360.2; (B0015) 254.9; (K577893) 99.2
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P23_.28weak_RBS.29.2C_P24_.28double_terminator.29_.26_P24_.28Tet_repressed_GFP.29">Sequencing of P23 (weak RBS), P24 (double terminator) &amp; P24 (Tet repressed GFP)</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Aim of the experiment:</B> Sequencing of P23, P24 &amp; P25 (x2)
</P><P><B>Procedure:</B>  
Sequencing batches were prepared after manufacturer's protocol (15 µL plasmid DNA (50-100 µM) and 2 µL sequencing primer)
</P><P>The different plasmids we prepared received the following barcodes:
</P><UL><LI> P23  (VF2) : FR11326647</LI><LI> P24  (VF2) : FR11326646</LI><LI> P25  (VF2) : FR11326645</LI><LI> P25  (VR) : FR11326644</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_May_25th"><B>Wednesday, May 25th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_pre-culture_with_tet-repressed_GFP_.28P25.29_in_LB-Cam">Inoculation of pre-culture with tet-repressed GFP (P25) in LB-Cam</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Procedure:</B> 
Add 50 µL chloramphenicol to 50 mL LB-medium 
Picking colonies from plate (BL21)
Incubate over night at 37°C
=&gt; dismissed, because plasmid sequence was inaccurate
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="MiniPrep_of_K747096.2C_I14033.2C_R0040.2C_B0015.2C_K577893_and_B0032">MiniPrep of K747096, I14033, R0040, B0015,  K577893 and B0032</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment: MiniPrep of BioBricks</B></P><P><B>Procedure:</B>  
MiniPrep was performed after manufacturer's protocol (QIAprep MiniPrep, Qiagen)
Concentrations were measured (in ng/µL):
</P><UL><LI> K747096: 	286.2</LI><LI> I14033: 	246.1</LI><LI> R0040: 	243.5</LI><LI> B0015: 	244.1</LI><LI> K577893: 	440.9</LI><LI> B0032: 	259.7</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_of_P19_.28QC_EspP.29_with_O3_and_O4">PCR of P19 (QC EspP) with O3 and O4</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Amplification of quickchanged EspP
</P><P><B>Procedure:</B></P><UL><LI>reaction was performed after manufacturer's protocol (PCR Using Q5® High-Fidelity DNA Polymerase, NEB)</LI><LI>274,4 pg Plasmid were used for amplification (1 µL)</LI><LI>PCR program (standard):</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature [°C]</B></TD><TD><B>Time [s]</B></TD></TR><TR><TD>initial Denaturation</TD><TD>98</TD><TD>30
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>72</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>final Extension</TD><TD>72</TD><TD>5
</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>ꝏ
</TD></TR></TBODY></TABLE><UL><LI> PCR product was purified accoding to manufacturer's protocol (PCR puification kit, Qiagen)</LI><LI> DNA was eluted 40 µL EB buffer</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Digestion_of_PCR_of_EspP_.28P19_with_O3_and_O4.29">Digestion of PCR of EspP (P19 with O3 and O4)</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Digestion of EspP with AgeI and NgoMIV to ligate it in pSB1C3 later on
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Volume</B></TD><TD><B>Reagent</B></TD></TR><TR><TD>38.5 µL</TD><TD>PCR product P19 with O3 and O4
</TD></TR><TR><TD>1 µL</TD><TD>AgeI
</TD></TR><TR><TD>1 µL</TD><TD>NgoMIV
</TD></TR><TR><TD>5 µL</TD><TD>CutSmart buffer
</TD></TR><TR><TD>4.5 µL</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>50 µL</TD><TD>TOTAL
</TD></TR></TBODY></TABLE><UL><LI> Incubation for 2 h at 37 °C </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Sequencing_of_P26_.28CMV_promotor.29.2C_P27_.28p_cat_promotor.29.2C_P28_.28Tet_repressible_promotor.29.2C_P30_.28Tet_repressed_GFP.29">Sequencing of P26 (CMV promotor), P27 (p cat promotor), P28 (Tet repressible promotor), P30 (Tet repressed GFP)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B>
Sequencing of P26, P27, P28 &amp; P30 (x2)
</P><P><B>Procedure:</B>
Sequencing batches were prepared after manufacturer's protocol 
(15 μL plasmid DNA (50-100 μM) and 2 μL sequencing primer)
The different plasmids we prepared received the following barcodes: 
P26 (VF2) : FR11326643
P27 (VF2) : FR11326642
P28 (VF2) : FR11326641
P30 (VF2) : FR11326640
P30 (VR): FR11326639
</P></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Ion_exchange_chromatography_of_eGFP">Ion exchange chromatography of eGFP</SPAN></H3><P><B>Investigator: NA, JB,JL</B></P><P><B>Aim of the experiment:</B> purification of eGFP
</P><P><B>Procedure:</B></P><UL><LI> use Äkta-Purifier (Q-column for eGFP (quarternary ammonium as stationary phase)). Ask Andy before use!</LI><LI> Pump A: running buffer (20 mM Tris/HCl pH 8); pump B: elution buffer (20 mM Tris/HCl pH 8 and  1,000 mM (1 M) NaCl</LI><LI> basic pumpwash before start</LI><LI> injection in loop</LI><LI> gradient of ion strength on column. </LI></UL><P><B>Results:</B>
Elution of eGFP at ~200 mM NaCl.
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_May_26th"><B>Thursday, May 26th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_cultures_with_XL-1_F11_and_F12_clones">Inoculation of cultures with XL-1 F11 and F12 clones</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B>
Inoculation of a 5 ml culture with a clone of the previously transformed F11 (Avi-Tag) F12 (Antibody-binding site). 
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_ligation_F14_.2BF13">Cloning of ligation F14 +F13</SPAN></H3><P><B>Investigator: MP, EF, JH</B></P><P><B>Aim of the experiment:</B> new biobrick EspP in pSB1C3
</P><P><B>Procedure:</B>  
F14 was purified by gelelctrophoresis (1%, 90 V, 40 min)
gelextraction was performed after manufacturer's protocol (Gelextractionkit, Qiagen)
ligation of F14 (EspP) with F13 (dephos. PSB1C3)
transformation according to protocol in competent E.coli XL1 blue
</P><P><B>Results:</B></P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_Biobrick_B0034_in_XL1-blue">Transformation of Biobrick B0034 in XL1-blue</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Aim of the experiment:</B> Transformation
</P><P><B>Procedure:</B></P><UL><LI> 10 µl ddH2O in well of interest (standard distribution kit, plate 4, well 1N) </LI><LI> 1 µl Plasmid (out of well) to cells</LI><LI> Transformation according to the SOP (incl. rescue)</LI></UL></DIV><DIV class="general"><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_Biobricks_K577895_and_K577894_in_XL1-blue">Transformation of Biobricks K577895 and K577894 in XL1-blue</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Transformation
</P><P><B>Procedure:</B></P><UL><LI> 10 µl ddH2O in well of interest (standard distribution kit, plate 1, wells 14B / 12P) </LI><LI> 1 µl Plasmid (out of well) to cells</LI><LI> Transformation according to the SOP (incl. rescue)</LI></UL></DIV><H3><SPAN class="mw-headline" id="SDS-PAGE_of_eGFP-fractions_after_IEC">SDS-PAGE of  eGFP-fractions after IEC</SPAN></H3><P><B>Investigator: NA, CG</B></P><P><B>Procedure:</B>  mixing of 80 µl sample with 20 µl SDS buffer and heating at 95°C for 10 min. 1 h staining, 1 night unstaining
Concentrations were measured via nanodrop (A280, extinction coefficient: 55000 M−1cm−1)
</P><P><B>Result:</B> concentrations in mg/ml
1. Lane: 8 µl Marker 
2. Lane: 10 µl of fraction 10; c= 0
3. Lane: 10 µl of fraction 11; c= 0,15
4. Lane: 10 µl of fraction 12; c= 0,77
5. Lane: 10 µl of fraction 13; c= 0,29
6. Lane: 10 µl of fraction 14; c= 0,4
7. Lane: 10 µl of fraction 15; c= 0,144
8. Lane: 10 µl of fraction 16; c= 0,085
</P><P>eGFP is clearly detectable at about 27 kDa. The most pure fractions 12 and 13 were pooled for later biotinylation.
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_May_27th"><B>Friday, May 27th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_pre-cultures_with_B0034_.28strong_RBS.29.2C_K577894_.28Tet_repressed_YFP.29.2C_KK577895_.28Tet_repressed_RFP.29.2C_F13.2BF14">Inoculation of pre-cultures with B0034 (strong RBS), K577894 (Tet repressed YFP), KK577895 (Tet repressed RFP), F13+F14</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Procedure:</B></P><UL><LI> Add 50 µL ampicillin(B0034)/chloramphenicol(K5777894, K577895, F13+F14) to 50 mL LB-medium </LI><LI> Picking colonies from plate (XL1 blue; all rescue)</LI><LI> Incubate over night at 37°C</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inoculation_of_BL21_.28pASK75_.28SA1.29.29_culture_in_2_L_TB-Medium_and_induction_of_streptavidin_production_by_addition_of_tetracycline">Inoculation of BL21 (pASK75 (SA1)) culture in 2 L TB-Medium and induction of streptavidin production by addition of tetracycline</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Production of streptavidin
</P><P><B>Procedure:</B></P><UL><LI> Ampicillin (2 mL) was added to the Medium (1:1000)</LI><LI> The pre-culture (50 mL) was poured into the Medium</LI><LI> Culture was incubated at 37°C and 140 rpm until OD550 reached 0.5</LI><LI> To induce streptavidin expression anhydro-tetrazycline (200 µL) was added to the culture (1:10000)</LI><LI> The culture was incubated at 37°C and 140 rpm for 4 hours</LI></UL><P><B>Result:</B>
Streptavidin expression by BL21
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Re-sequencing_of_P30_.28Tet_repressed_GFP_in_pSB1C3.29">Re-sequencing of P30 (Tet repressed GFP in pSB1C3)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B>
Purification and sequencing of P30
</P><P><B>Procedure:</B></P><UL><LI> P30 was heated up to 100 °C and centrifugated afterwards to remove proteins from Promotor (if there are some), the supernatant is sequenced</LI><LI> Sequencing batches were prepared after manufacturer's protocol </LI></UL><UL><LI>(15 μL plasmid DNA (50-100 μM) and 2 μL sequencing primer)</LI><LI>The different plasmids we prepared received the following barcodes: 
</LI><LI>P30 (VF2) : FR11326638</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Expression_and_harvest_of_Streptavidin_.28pASK75_.28SA1.29.29_in_BL21_in_TB-medium">Expression and harvest of Streptavidin (pASK75 (SA1)) in BL21 in TB-medium</SPAN></H3><P><B>Investigator: JH, NA </B></P><P><B>Aim of the experiment:</B> Recombinant expression and purification of Streptavidin
</P><P><B>Procedure:</B></P><UL><LI> After expression, cultures were transferred into centrifuge tubes and spun down in the centrifuge (4°C, 5000 rpm, 20 mins, F 4X1L rotor) </LI><LI> The supernatant was cast away and the pellet was stored at –20 °C</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_A3C5_and_Avi-Tag_into_pSB1C3_5">Cloning of A3C5 and Avi-Tag into pSB1C3</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> MiniPrep of cloned pSB1C3-A3C5 and –AviTag Plasmids
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep of inoculated precultures was performed according to manufacturer’s protocol (QIAprep MiniPrep, Qiagen)</LI><LI> Concentrations were measured (in ng/µL):</LI></UL><P>Colony 1 of pSB1C3- AviTag P32: 	120
Colony 2 of pSB1C3- AviTag P33: 	110
Colony 1 of pSB1C3-A3C5 P34: 		126
Colony 2 of pSB1C3-A3C5 P35: 		188
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_A3C5_and_Avi-Tag_into_pSB1C3_6">Cloning of A3C5 and Avi-Tag into pSB1C3</SPAN></H3><P><B>Investigator: NA, JH, CG</B></P><P><B>Aim of the experiment:</B> Initial vector digestion was performed with NgoMiV and AgeI, producing complementary overhangs. Religated pSB1C3 (due to incomplete vector dephosphorylation) loses the AgeI recognition sequence. Therefore plasmids without insert should run in the supercoiled position at about 1.5 kb, plasmids with inserts  should run in the linearised position at 3 kb. Single enzyme digestion was chosen, because the insert is too small to be solved on a gel.
</P><P><B>Procedure:</B></P><UL><LI> analytic digestion: 3 µl of plasmid DNA P32 to P35 (about 300 ng) were digested with 0.3 µl AgeI-HF in 1 µl CutSmart Buffer and 5.7 µl H2O. The reaction was incubated for 2 h at 37°C.</LI><LI> 5 µl digestion mix were loaded on a 1% agarose gel for electrophoresis, 1 kb ladder</LI></UL><P><B>Result:</B></P><UL><LI>P32 to P35 might carry the insert because they all appear in the linearized position at 3 kb</LI><LI>Therefore plasmid sequencing should confirm the result.</LI></UL><P>Lane 1: 1 kb ladder
Lane 3: P32
Lane 4: P33
Lane 5: P34
Lane 6: P35
</P></DIV></DIV><H1><SPAN class="mw-headline" id="Week_3_.28May_30th_-_June_5th.29">Week 3 (May 30th - June 5th)</SPAN></H1><DIV class="week" id="Week_3"><H2><SPAN class="mw-headline" id="Monday.2C_May_30th"><B>Monday, May 30th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_A3C5_and_Avi-Tag_into_pSB1C3_7">Cloning of A3C5 and Avi-Tag into pSB1C3</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> 
Sequencing of P32 and P35
</P><P><B>Procedure:</B></P><UL><LI>Sequencing batches were prepared according to manufacturers protocol (Mix2Seq Kit, Eurofins Genomics): 15 μL plasmid DNA (50-100 μM) and 2 μL sequencing primer (10 µM)</LI></UL><UL><LI>The plasmids received the following barcodes: 
</LI><LI>P32 (VF2) : FR11326637</LI><LI>P35 (VF2) : FR11326636</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Religation_of_F13_.28digest_of_pSB1C3_with_AgeI_and_NgoMIV.29_and_transformation_in_XL1_blue">Religation of F13 (digest of pSB1C3 with AgeI and NgoMIV) and transformation in XL1 blue</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> 
Re-ligate digested and dephosphorylated pSB1C3 and transformation in XL1blue
</P><P><B>Procedure:</B></P><UL><LI> Plasmid was ligated with the T4 DNA Ligase following the manufacturer’s protocol (NEB)</LI><LI> 52 ng Vector DNA were used</LI><LI> The Ligation mix was incubated for 30 min at RT</LI><LI> 7 μL Ligation mixed were used for the transformation of 50 μL XL1 blue.</LI><LI> Transformation was performed following the SOP</LI></UL><P><B>Result:</B></P><UL><LI>5 clones on plate</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_pre-cultures_with_B0034_.28strong_RBS.29.2C_K577894_.28TET_repressed_YFP.29.2C_KK577895_.28Tet_repressed_RFP.29.2C_F13.2BF14_2">Inoculation of pre-cultures with B0034 (strong RBS), K577894 (TET repressed YFP), KK577895 (Tet repressed RFP), F13+F14</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Procedure:</B></P><UL><LI> Add 50 µL ampicillin(B0034)/chloramphenicol(K5777894, K577895, F13+F14) to 50 mL LB-medium </LI><LI> Picking colonies from plate (XL1 blue; all rescue)</LI><LI> Incubate over night at 37°C</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_May_31th"><B>Tuesday, May 31th</B></SPAN></H2><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Dialysis_of_eGFP_2">Dialysis of eGFP</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Changing the buffer of eGFP for biotinylation (from TRIS to PBS)
</P><P><B>Procedure:</B></P><UL><LI> eGFP was poured in a dialysis hose (cut-off 14 kDa)</LI><LI> The hose was then placed in ice cold PBS pH 7.4</LI><LI> The dialysis took place at 4°C overnight</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Expression_and_harvest_of_Streptavidin_.28pASK75_.28SA1.29.29_in_BL21_in_TB-medium_2">Expression and harvest of Streptavidin (pASK75 (SA1)) in BL21 in TB-medium</SPAN></H3><P><B>Investigator: JB, NA</B></P><P><B>Aim of the experiment:</B> Recombinant expression and purification of Streptavidin
</P><P><B>Procedure:</B></P><UL><LI> The pellet was transferred into a beaker of sufficient size and resuspended in fridge-cooled Tris Buffer B (50 mM Tris/HCl (pH = 8.0), 1 mM EDTA)</LI><LI> The solution was homogenized in the PANDA (see SOP)</LI><LI> The resulting lysate was transferred into centrifuge tubes and spun down (4°C, 18,000 rpm, 30 mins, SS34-rotor). The supernatant was cast away and the pellet was resuspended in 6M Gua-HCl (pH = 1.5) at 4°C overnight.</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_P30_.28Tet_repressed_GFP.29">Transformation of P30 (Tet repressed GFP)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> expression of tet-repressed GFP in E. Coli BL21
</P><P><B>Procedure:</B> transformation in competent E. Coli BL21 according to protocol 
</P><P><B>Result:</B> plates (LB Cam) in incubator for further processing (37 °C)
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_A3C5_and_Avi-Tag_into_pSB1C3_8">Cloning of A3C5 and Avi-Tag into pSB1C3</SPAN></H3><P><B>Investigator: CR, JH</B></P><P><B>Aim of the experiment:</B> 
Preculture of pSB1C3 (F13+F14) was red
See if RFP-generator is still inside the pSB1C3
See if ligation of EspP was successful
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep of P39 was performed after manufacturer's protocol (QIAprep MiniPrep, Qiagen)</LI><LI> pSB1C3 was digested with AgeI and NgOMIV (1.5 µL P39 + 1.5 µL AgeI + 1.5 µL NgoMIV + 3 µL CutSmart + 22.5 µL ddH2O; incubation for 2h, 37°C)</LI><LI> FastAP was added and the reaction was incubated for another 10min at 37°C</LI><LI> FastAP was inactivated by heating the mix to 75°C for 10min.</LI><LI> The digestion was purified by gelelctrophoresis (1%, 75V, 1h)</LI></UL><P><B>Results:</B></P><P>Unknown signals.
Repeat transformation, digestion, dephosphorylation and ligation of pSB1C3
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Re-transformation_of_P4_.28pSB1C3_.28RFP-generator.29.29_in_XL1_blue">Re-transformation of P4 (pSB1C3 (RFP-generator)) in XL1 blue</SPAN></H3><P><B>Investigator: JH, CR</B></P><P><B>Aim of the experiment:</B> Transformation
</P><P><B>Procedure:</B></P><UL><LI> 2 μL P4 were used for transformation of XL1 blue </LI><LI> Transformation was performed according to SOP (incl. rescue plate)</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Competent_E._coli_XL1_cells">Competent E. coli XL1 cells</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Generation of competent E.coli XL1 cells
</P><P><B>Procedure:</B></P><UL><LI> 5 ml LB w.o. antibiotics (sterile)</LI><LI> Inoculation of one colony of E.coli XL1 cells</LI><LI> Incubate overnight at 37°C</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_June_1th"><B>Wednesday, June 1th</B></SPAN></H2><DIV class="polymerization"><H3><SPAN class="mw-headline" id="eGFP_Biotinylation">eGFP Biotinylation</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Biotinylation of dialysed eGFP past IEC
</P><P><B>Procedure:</B></P><UL><LI> The average concentration of the pooled factions 12 and 13 should be about 0.5 mg/ml (=19 µM)</LI><LI> 10 ml of a 100 mM Biotin stock-solution was prepared (340 mg)</LI><LI> 3.5 ml of eGFP were biotinylated with 12 µl 100 µM Biotin-solution (20x molar excess)</LI><LI> Incubation at 4 °C overnight</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="pSB1C3_vector_backbone_preperation">pSB1C3 vector backbone preperation</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Inoculation of two colonies pSB1C3-RFP Generator
</P><P><B>Procedure:</B></P><UL><LI> 2x 5 ml LB+Cam medium (1:1000)</LI><LI> Picking colonies from plate (XL1 blue, rescue)</LI><LI> Incubate overnight at 37°C</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Generation_of_GFP_expressing_E._coli_cells">Generation of GFP expressing E. coli cells</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Inoculation of BL21 transformed cell with P30
</P><P><B>Procedure:</B></P><UL><LI> 50 ml LB+Cam medium (1:1000)</LI><LI> Picking colonies from plate (BL21, rescue)</LI><LI> Incubate over night at 37°C</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Competent_E._coli_XL1_cells_2">Competent E. coli XL1 cells</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Generation of competent E. coli XL1 cells
</P><P><B>Procedure:</B></P><UL><LI> Inoculated 500 µl preculture in 50 ml LB media w.o. antibiotics</LI><LI> Incubated and shaken at 37°C until the OD550 reached 0.44</LI><LI> Transferred into a falcon tube and incubated on ice for 10 mins</LI><LI> Spun down at 3000 g for 10 mins at 4°C, supernatant cast away</LI><LI> Pellet resuspended in 40 ml fridge-cooled MgCl2 (100 mM), once again spun down at 4°C for 10 mins at 3000 g </LI><LI> Supernatant cast away, pellet resuspended in 20 ml cold CaCl2 (50 mM)</LI><LI> Incubation on ice for 30 mins</LI><LI> Centrifugation repeated, supernatant cast away</LI><LI> Pellet resuspended in 2 ml of CaCl2 (50 mM) and glycerol (15%)-solution, aliquoted (50 µl each), frozen in liquid nitrogen and stored at -80°C for further use</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_Pcat-RBS-construct_in_pSB1A2">Cloning of Pcat-RBS-construct in pSB1A2</SPAN></H3><P><B>Investigator: CG, JH, CR</B></P><P><B>Aim of the experiment:</B> Pcat (I14033) was cut out of P27 and ligated into P38 for generating the Pcat-RBS (B0034) construct.
</P><P><B>Procedure:</B></P><UL><LI> P27 was digested with EcoRI and SpeI (4 µL P27 + 1.5 µL EcoRI + 1.5 µL SpeI + 3 µL CutSmart + 20 µL ddH2O; incubation for 2h, 37°C)</LI><LI> P38 was digested with EcoRI and XbaI (10 µL P27 + 1.5 µL EcoRI + 1.5 µL XbaI + 3 µL CutSmart + 14 µL ddH2O; incubation for 2h, 37°C)</LI><LI> Preperative gel electrophoresis was performed</LI><LI> For P27 the signal at about 50 bp was cut out and gel extracted according to manufacturer's protocol (Gelextractionkit, Qiagen): 5.9 ng/µl</LI><LI> For P30 the signal at about 2.1 kb was cut out and gel extracted according to manufacturer's protocol (Gelextractionkit, Qiagen): 8.9 ng/µl</LI><LI> Ligation according to SOP</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Purification_of_streptavidin">Purification of streptavidin</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Refolding denaturized streptavidin
</P><P><B>Procedure:</B></P><UL><LI> the GdmCl-solved streptavidin was spun down using an SS-34 rotor (~60 min, 18,000 rpm, 4°C)</LI><LI> The supernatant containing the solved and denaturized protein was transferred into a falcon tube, the pellet cast away</LI><LI> The protein solution was slowly dripped into 4 l of cooled 1x PBS (in the 4°C room) using a hydraulic pump and stirred overnight</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_June_2th"><B>Tuesday, June 2th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_pSB1C3_for_further_use">Preparation of pSB1C3 for further use</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Digestion of pSB1C3 to remove RFP-generator
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep was performed after manufacturer's protocol (QIAprep MiniPrep, Qiagen)</LI><LI> P40 was digested with AgeI and NgOMIV (2.5 µL P40 + 1.5 µL AgeI + 1.5 µL NgoMIV + 3 µL CutSmart + 21.5 µL ddH2O)</LI><LI> Incubation over night at 37°C</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Generation_of_GFP_expressing_E._coli_cells_2">Generation of GFP expressing E. coli cells</SPAN></H3><P><B>Investigator: CG, JH</B></P><P><B>Aim of the experiment:</B> Various inductor concentrations 
</P><P><B>Procedure:</B></P><UL><LI> 12x 5 ml LB+Cam media were inoculated with 50 µl preculture</LI><LI> when cultures reached OD550 of 0.5, they were induced with tetrazycline-anhydride of various concentrations (0 ng/µl, 20 ng/µl, 40 ng/µl etc., 200 ng/µl)</LI><LI> t=0, =30, =90, =150, =210, =270 a 100 µl aliquot of each culture was stored at 4 °C in the refrigerator</LI></UL></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Dialysis_of_biotinylated_eGFP">Dialysis of biotinylated eGFP</SPAN></H3><P><B>Investigator: JH, JB</B></P><P><B>Aim of the experiment:</B> Purification of biotinylated eGFP
</P><P><B>Procedure:</B></P><UL><LI> eGFP was poured in a dialysis hose (cut-off 14 kDa)</LI><LI> The hose was then placed in 2 L ice cold Tris/HCl 20 mM pH 8.0</LI><LI> The dialysis took place at 4°C over night</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Ligation_of_F17.1_.28p.28cat.29.29_and_F18_.28stron_RBS.29_and_transformation_in_XL1_blue">Ligation of F17.1 (p(cat)) and F18 (stron RBS) and transformation in XL1 blue</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Aim of the experiment:</B>
Ligation and transformation in XL1blue
</P><P><B>Procedure:</B></P><UL><LI> Plasmid was ligated with the T4 DNA Ligase following the manufacturer’s protocol (NEB)</LI><LI> 9 µL F18 (B0034 in pSB1A2; RBS) and 1 µL of F17.1 (I14033; P(cat)) were used [AmpR]</LI><LI> The Ligation mix was incubated for at least 2h at RT</LI><LI> 7 μL Ligation mixed were used for the transformation of 50 μL XL1 blue</LI><LI> Transformation was performed following the SOP [AmpR], incl. rescue plate</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_June_3th"><B>Friday, June 3th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_F8_.28A3C5.29.2C_F9_.28Avi-Tag.29_and_F14_.28EspP.29_into_F20">Cloning of F8 (A3C5), F9 (Avi-Tag) and F14 (EspP) into F20</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Cloning of A3C5, Avi-Tag and into pSB1C3
</P><P><B>Procedure:</B></P><UL><LI> FastAP was added and the digestion of F20 and was incubated for another 15 min at 37°C</LI><LI> The digestion was purified by gelelctrophoresis (1%, 75V, 1h)</LI><LI> Plasmid backbone (signal 1) was cut from the gel </LI><LI> Gelextraction was performed after manufacturer's protocol (Gelextractionkit, Qiagen)</LI></UL><P><B>Results:</B></P><P>Signal 1: F20
</P><P>[F20]= 13,8 ng/μL
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Generation_of_GFP_expressing_E._coli_cells_3">Generation of GFP expressing E. coli cells</SPAN></H3><P><B>Investigator: CG, NA</B></P><P><B>Aim of the experiment:</B> SDS PAGE for verification of successful GFP expression 
</P><P><B>Procedure:</B></P><UL><LI> SDS electophoresis according to SOP</LI><LI> Lane  1: #12, 90 min</LI><LI> Lane  2: #12, 150 min</LI><LI> Lane  3: #12, 210 min</LI><LI> Lane  4: #12, 270 min</LI><LI> Lane 5: 8 µl </LI></UL><P><B>Results:</B></P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_colonies_from_F19_.28pcat-strong_RBS.29">Inoculation of colonies from F19 (pcat-strong RBS)</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> three colonies were picked from yesterdays transformation
</P><P><B>Procedure:</B></P><UL><LI> 3x 5 ml LB+Amp media</LI><LI> Each culture was inoculated with one colony</LI><LI> Incubation at 37°C overnight</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_F8_.28A3C5.29.2C_F9_.28Avi-Tag.29_and_F14_.28EspP.29_into_F20_.28pSB1C3.29">Cloning of F8 (A3C5), F9 (Avi-Tag) and F14 (EspP) into F20 (pSB1C3)</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Ligation of F21, F22 and F23 into F20 and transformation
</P><P><B>Procedure:</B></P><UL><LI> Ligation was performed according to the SOP with: 0.7 µl F8 (1:10) / 9.3 µl F20, 0.7 µl F9 (1:10) / 9.3 µl F20, 9.1 µl F14 / 0.9 µl F20</LI><LI> Transformation was performed according to the SOP on LB Cam-Agar</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Precipitation_of_Streptavidin">Precipitation of Streptavidin</SPAN></H3><P><B>Investigator: NA, JB, LK</B></P><P><B>Aim of the Experiment:</B> Purification of Streptavidin
</P><P><B>Procedure:</B></P><UL><LI> Slowly add (NH4+)2SO42- until the concentration reaches 40% (23,1 g/100mL) while stirring </LI><LI> Let it stir over night to ensure complete precipitation</LI><LI> Centrifuge at 8500 rpm at 4°C for 15min (rotor: SLA1500), dismiss the pellet and use the supernatant for further processing</LI><LI> Slowly add (NH4+)2SO42- until the concentration reaches 70% (19,1 g/100mL) while stirring </LI><LI> Let it stir over night (slowly to avoid foaming)</LI><LI> Centrifuge at 8500 rpm at 4°C for 15min (rotor: SLA1500), dismiss the supernatant and use the pellet for further processing</LI><LI> Resuspend the protein pellet in a minimum volume of TRIS/HCl; pH 8; 20mM (NO SALT) (6 mL per pellet), the pellets where then united and titrated with Buffer until the solution became clear. Total amount of buffer: 40mL </LI><LI> Transfer to 50 mL Falcon Tubes and centrifuge at full speed for 15 min. Filter the supernatant through 45nm sterile filter and keep the pellet (rescue). </LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_June_4th"><B>Saturday, June 4th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_colonies_from_F21_.28pSB1C3_.28A3C5.29.29.2C_F22_.28pSB1C3_.28Avi-Tag.29.29.2C_F23_.28pSB1C3_.28EspP.29.29">Inoculation of colonies from F21 (pSB1C3 (A3C5)), F22 (pSB1C3 (Avi-Tag)), F23 (pSB1C3 (EspP))</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> three colonies were picked from each transformation
</P><P><B>Procedure:</B></P><UL><LI> 3x 5 ml LB+Cam media for each fragment</LI><LI> Each culture was inoculated with one colony</LI><LI> Incubation at 37°C overnight</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="MiniPrep_of_F19_.28pSB1A2_.28pcat-strong_RBS.29.29">MiniPrep of F19 (pSB1A2 (pcat-strong RBS))</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Extraction of Ligation: F17+F18 (starke RBS+P(cat))  
</P><P><B>Procedure:</B>   
MiniPrep was performed after manufacturer's protocol (QIAprep MiniPrep, Qiagen)
Concentrations: K1= 66,2 ng/µl; K2= 60,2 ng/µl; K3= 104,4 ng/µl
</P></DIV></DIV><H1><SPAN class="mw-headline" id="Week_4_.28June_6th_-_June_12th.29">Week 4 (June 6th - June 12th)</SPAN></H1><DIV class="week" id="Week_4"><H2><SPAN class="mw-headline" id="Monday.2C_June_6th"><B>Monday, June 6th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_TetR_from_p28_into_p31_.28pSB1C3.28weak_RBS.29.29">Cloning of TetR from p28 into p31 (pSB1C3(weak RBS))</SPAN></H3><P><B>Investigator: LK, JH, NA</B></P><P><B>Aim of the experiment:</B> Cloning TetR in front of weak RBS in pSB1C3 
</P><P><B>Procedure:</B></P><UL><LI> Digestion of p28 with EcoR1 and Spe1 (20µl DNA, 2,5 µl each enzyme, 5µl CutSmart buffer, 20µl ddH2O) and p31 (4µl DNA, 1µl EcoRI / XbaI, 2µl, 12µl DNA) -&gt; Incubation at 37°C for 2h. </LI><LI> Electrophoresis: 1% Agar, 15 min , 70 V (oben p31 und p41, unten p28)</LI></UL><UL><LI> Gelextraction  concentrations: p28 (F25) = 2,3 ng/µl</LI></UL><P>p31 (F24) = 7,8 ng/µl
</P><UL><LI> Ligation with 6,3 µl vector (F24) and 1,7 µl Insert (F25) + 1µl Buffer + 1µl Ligase  over night, 16°C </LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="SDS-PAGE_of_Streptavidin">SDS-PAGE of  Streptavidin</SPAN></H3><P><B>Investigator: NA, JH, LK</B></P><P><B>Procedure:</B>  mixing of 80 µl sample with 20 µl SDS buffer and heating at 95°C for 10 min. 1 h staining, 1 night unstaining. Concentration of streptavidin was measured via UV/VIS spectroscopy. 
</P><P>c = 23,5 mg/ml 
</P><P><B>Result:</B></P><UL><LI> 1. Lane: 8 µl Marker </LI><LI> 2. Lane: 5 µl of Streptavidin_1:10 </LI><LI> 3. Lane: 5 µl of Streptavidin_1:50</LI><LI> 4. Lane: 5 µl of Pellet_1:500</LI><LI> 5. Lane: 5 µl of after 40% precipitation_1:10</LI><LI> 6. Lane: 5 µl of supernatant after 70% precipitation and centrifugation </LI></UL><P>Successful Streptavidin production verified.
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Sequencing_of_P41_.28pSB1C3_.28strongRBS-p.28cat.29.29.29">Sequencing of P41 (pSB1C3 (strongRBS-p(cat)))</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Sequencing of P41
</P><P><B>Procedure:</B>  
Sequencing batches were prepared after manufacturer's protocol (15 µL plasmid DNA (100 ng/ µl) and 2 µL sequencing primer (VF2))
</P><P>The plasmids prepared received the following barcode:
</P><UL><LI> P41        : FR11326633</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_June_7th"><B>Tuesday, June 7th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_P30_.28Tet_repressed_GFP.29.2C_P36_.28Tet_repressed_RFP.29.2C_P37_.28Tet_repressed_YFP.29">Transformation of P30 (Tet repressed GFP), P36 (Tet repressed RFP), P37 (Tet repressed YFP)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Transformation of RFP, GFP and YFP in BL21
</P><P><B>Procedure:</B>
transformation in competent E. Coli BL21 according to protocol + rescue
</P><P><B>Result:</B> 
plates (LB Cam) in incubator for further processing (37 °C)
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="MiniPrep_of_P42_to_P50.2C_analytic_digestion.2C_sequencing">MiniPrep of P42 to P50, analytic digestion, sequencing</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Verification of successful cloning
</P><P><B>Procedure:</B>   
MiniPrep was performed according to manufacturer's protocol (QIAprep MiniPrep, Qiagen)
</P><UL><LI> Concentrations (ng/µl each) for P42: 260, P43: 247, P44: 181, P45: 118, P46: 261, P47: 135, P48: 148, P49: 240, P50: 299</LI><LI> Analytic digestion: 0.3 µl AgeI, 2 µl CutSmart, 6.7 µl H2O for P42, P43, P44, P45, P46, P49, P50 and 0.3 µl EcoRI, 0.3 µl PstI, 2 µl CutSmart, 6.4 µl H2O for 2 h at 37°C</LI><LI> 1% agarose gel electrophoresis</LI><LI> Sequencing: P42= FR11326632 (VF2), P45 =FR11326631 (VF2)</LI></UL><P><B>Results:</B></P><UL><LI> Lane 1: 5 µl 1 kb Ladder</LI><LI> Lane 2-10: Digestion of P42 to P50. P47 and P48 are carrying the insert. All other plasmids are religated and lost their AgeI site, hence they run in the ccc position.</LI></UL></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Dialysis_of_not_biotinylated_eGFP">Dialysis of not biotinylated eGFP</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Purification of eGFP for mass determination
</P><P><B>Procedure:</B></P><UL><LI> eGFP was poured in a dialysis tubes</LI><LI> The tubes were then placed in ice cold ammonium acetate</LI><LI> The dialysis took place at 4°C over night</LI></UL></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Polymerisation_attempts">Polymerisation attempts</SPAN></H3><P><B>Investigator: JH, CR, CG, JB</B></P><P><B>Aim of the experiment:</B> Identification of optimal polymerisation conditions for biotinylated eGFP and BSA in a Streptavidin reservoir
</P><P><B>Procedure:</B></P><UL><LI> variate of compositions for both solutions was examined to modify density properties during “printing”</LI><LI> all compositions can be found in excel sheet</LI></UL><P><B>Results:</B></P><UL><LI> no or little successful polymerization at Streptavidin concentrations of 4 mg/mL and below; immense density incompatibilities</LI><LI> sucessfull polymerization at Streptavidin concentrations of 10 mg/mL and BSA solution with approx. 15% glycerol</LI><LI> no success with biotinylated eGFP (too few Biotin molecules?!)</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_June_8th"><B>Wednesday, June 8th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Sequencing_of_P36_.28Tet_repressed_RFP.29_and_P37_.28Tet_repressed_YFP.29">Sequencing of P36 (Tet repressed RFP) and P37 (Tet repressed YFP)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Sequencing of P36 and P37 in both directions
</P><P><B>Procedure:</B>  
Sequencing batches were prepared after manufacturer's protocol (15 µL plasmid DNA (50-100 µM) and 2 µL sequencing primer)
</P><P>The different plasmids we prepared received the following barcodes:
</P><UL><LI> P36 (VF2): FR11326630</LI><LI> P36 (VR): FR11326629</LI><LI> P37 (VF2): FR11326628</LI><LI> P37 (VR): FR11326627</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Dialysis_of_Streptavidin">Dialysis of Streptavidin</SPAN></H3><P><B>Investigator: NA, JH</B></P><P><B>Aim of the experiment:</B> Purification of Streptavidin (possible ammonium sulfate contamination after precipitation)
</P><P><B>Procedure:</B></P><UL><LI> Streptavidin was poured in a dialysis hose (cut-off 14 kDa)</LI><LI> The hose was then placed in 3 L ice cold Tris/HCl 20 mM pH 8.0 (no salt)</LI><LI> The dialysis took place at 4°C over night</LI></UL></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Polymerisation_attempts_2">Polymerisation attempts</SPAN></H3><P><B>Investigator: JH, JB</B></P><P><B>Aim of the experiment:</B> Identification of optimal polymerisation conditions for biotinylated BSA in a Streptavidin reservoir
</P><P><B>Procedure:</B></P><UL><LI> variate of compositions for both solutions was examined to modify density properties during “printing” (Streptavidin at 10 mg/mL or higher)</LI><LI> all compositions can be found in excel sheet </LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_June_9th"><B>Thursday, June 9th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Generation_of_GFP.2C_YFP.2C_RFP_expressing_E._coli_cells">Generation of GFP, YFP, RFP expressing E. coli cells</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Various inductor concentrations, various fluorescent proteins were expressed in E coli cells
</P><P><B>Procedure:</B></P><UL><LI> 3x 5 ml LB+Cam media were inoculated with 50 µl preculture of GFP (P30), RFP (P36), YFP (P37)</LI><LI> when cultures reached OD550 of 0.5, they were induced with anhydro-tetrazycline of various concentrations (0 ng/µl, 100 ng/ml, 200 ng/ml)</LI><LI> 50 ng/ml = 12,5 µl of 1:100 2 mg/ml stock solution, 200 ng/ml = 50 µl of 1:100 2 mg/ml stock solution</LI><LI> t=0, =4 h and stored on ice</LI></UL><P><B>Resuts:</B> no fluorescent could be detected. Therefore an overnight induction at 37°C with 1 µM/ml inductor concentration was performed with the RFP plasmid.
</P></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Biotinylation_of_GFP_expressing_E._coli_cells">Biotinylation of GFP expressing E. coli cells</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Biotinylation of P30 carrying E. coli cells
</P><P><B>Procedure:</B></P><UL><LI> 850 µl P30 preculture from Thursday 2nd were centrifuged and the supernatend was discarded</LI><LI> Pellet was resuspended in 800 µl PBS-buffer</LI><LI> 100 µl 100 mM Biotin-NHS was added</LI><LI> incubation for 4 h at room temperature</LI><LI> Experimenting with some glycerol and streptavidin concentrations. A more structured approach will be done the next day.</LI></UL><P><B>Results:</B> A shape thin “sausage” could be detected. Therefore a bigger E. coli culture was biotinylated at 4°C overnight for experimenting with the next day.
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_TetR_promotor_and_weak_RBS_in_pSB1C3">Cloning of TetR promotor and weak RBS in pSB1C3</SPAN></H3><P><B>Investigator: CR, JB</B></P><P><B>Aim of the experiment:</B> MiniPrep and NanoDrop of F26 K1, K2 and K3 
Sequencing of P51 
</P><P><B>Procedure:</B>  
MiniPrep was performed according to manufacturer's protocol (QIAprep MiniPrep, Qiagen) 
Concentrations were as following: 
</P><UL><LI> P51: 163,3 </LI><LI> P52: 151,7 </LI><LI> P53: 140,5 </LI></UL><P>P51 was sequenced:
</P><UL><LI> 1. with VF2: FR11326626</LI><LI> 2. with VR: FR11326625</LI></UL></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Biotinylation_of_eGFP">Biotinylation of eGFP</SPAN></H3><P><B>Investigator: VL, JB</B></P><P><B>Aim of the experiment:</B> Biotinylation of eGFP with 200x biotin-NHS molar excess over eGFP (15x excess over surficial K residues). NB: previous trial unsuccessful. Excess of biotin was likely insufficient to ensure efficient biotinylation, esp. since biotin-NHS slowly hydrolyzes in the aqueous environment.
</P><P><B>Procedure:</B>  1 ml of eGFP was biotinylated with an 200x excess of biotin in DMF (100 uM; 38 uL), 5.5 h at rt. The solution was dialysed versus 20 mM Tris/HCl pH 8 overnight at 4 °C.
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_of_gene_syntheses">PCR of gene syntheses</SPAN></H3><P><B>Investigator: LK</B></P><P><B>Aim of the experiment:</B> Amplification od iGEM_1_SA-m1 (F27), iGEM_1_mRuby_EGFR (F28) and iGEM_2_IRES-BirA (F29)
</P><P><B>Procedure:</B></P><UL><LI> Add 100µl of ddH2O to the gene sythesis samples and incubate for 20 min at 50°C</LI><LI> Dilute samples to a final concentration of max. 1ng/µl (1:10) with water.</LI><LI> Approach: </LI></UL><P>1µl DNA (diluted) 
</P><P>5µl buffer 
</P><P>0,5µl dNTPs
</P><P>1,25µl forward Primer VF2
</P><P>1,25µl reverse Primer VR
</P><P>0,25µl Q5 Polymerase 
</P><P>Fill up with water up to 25µl
</P><UL><LI> Make sure to pipette everything on ice and add the polymerase at the end</LI><LI> Setup: (calculated with NEB calc. <A rel="nofollow" class="external free" href="http://tmcalculator.neb.com/#!/">http://tmcalculator.neb.com/#!/</A>) </LI></UL><P>98°C_1min initiation step
</P><P>98°C_10sec
</P><P>66°C_30sec   &gt;  step 2-4: 35 repeats
</P><P>72°C_30sec
</P><P>72°C_2min final step 
</P><P>4°_hold
</P><UL><LI> The samples were then prepared with the PCR Product Purification Kit </LI></UL></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Polymerisation_experiments_for_biotinylated_eGFP_and_BSA_with_Streptavidin">Polymerisation experiments for biotinylated eGFP and BSA with Streptavidin</SPAN></H3><P><B>Investigator: JH, CR, JB</B></P><P><B>Aim of the experiment:</B> Determination of ideal concentrations for both protein interaction partners as well as buffer viscosity (glycerol content etc.) for polymerisation in a test tube
</P><P><B>Procedure:</B></P><UL><LI> Different concentrations of proteins and glycerol were tested to support stable polymerisation without protein aggregates rising to the surface or sinking to quickly </LI><LI> The experiments done are not going to be further documented as the streptavidin used for the experiments was previously determined in a wrong fashion and was actually considerably lower than expected (4x lower). Streptavidin was, due to that circumstance, concentrated to be able to support experiments with at least 5 mg/ml concentrations</LI><LI> First experiments with concentrated streptavidin seemed promising, though a complete series of experiments with varying parameters should be conducted in the future</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P29_.28pSB1C3_.28double_terminator.29.2C_P36_.28pSB1C3_.28Tet_repressed_RFP.29.29_and_P37_.28pSB1C3_.28Tet_repressed_RFP.29.29">Sequencing of P29 (pSB1C3 (double terminator), P36 (pSB1C3 (Tet repressed RFP)) and P37 (pSB1C3 (Tet repressed RFP))</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Sequencing of P29, P36 and P37 
</P><P><B>Procedure:</B>  According to the manufacturer's protocol.
</P><UL><LI> P29: FR11326624 (VF2)</LI><LI> P36: FR11326623 (VF2)</LI><LI> P37: FR11326621 (VR), FR11326622 (VF2)</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_.28pre.29cultures_for_a_pSB1C3_midiprep">Inoculation of (pre)cultures for a pSB1C3 midiprep</SPAN></H3><P><B>Investigator: JB, VM</B></P><P><B>Aim of the experiment:</B> Massively amplifying the pSB1C3 vector (which will be needed for building up new biobricks) by midiprepping the P40 plasmid (RFP-generator in pSB1C3). Similar to minipreps, but on a greater scale.
<B>Procedure:</B>  3 ml precultures were inoculated with a clone from the plate previously used for minipreps. After the precultures had grown for 4 hours (due to shortage of time), 2 x 50 ml cultures were inoculated and shaken overnight at 37°C. The next day, cultures were put into the fridge for storage until Monday. 
</P></DIV></DIV><H1><SPAN class="mw-headline" id="Week_5_.28June_13th_-_June_19th.29">Week 5 (June 13th - June 19th)</SPAN></H1><DIV class="week" id="Week_5"><H2><SPAN class="mw-headline" id="Monday.2C_June_13th"><B>Monday, June 13th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Digestion_of_gene_synthesis_iGEM1_and_iGEM2">Digestion of gene synthesis iGEM1 and iGEM2</SPAN></H3><P><B>Investigator: LK, JH</B></P><P><B>Aim of the experiment:</B> Digest F28 (iGEM1) and F29 (iGEM2) in order to ligate them into P40, which was equally digested (to get rid of the rfp-generator and have pSB1C3 as backbone)
</P><P><B>Procedure:</B></P><UL><LI> 30µl of F28 were digested for 3h with SpeI and NgoMIV, and 30 µl of F29 were digested with XbaI and PstI</LI><LI> 2 x 20 µl where digested with the equivalent enzymes for 3h </LI><LI> After electrophoresis the bands were cut out and purified</LI></UL><P><B>Results:</B> Results show that PCR of F28, F29 might not have worked. P40 was extracted from gel (F30 and F31). PCR of gene synthesis was repeated (50µl!) 
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Midiprep_of_P40">Midiprep of P40</SPAN></H3><P><B>Investigator: LK</B></P><P><B>Aim of the experiment:</B> Purification of P40 from 
</P><P><B>Procedure:</B></P><UL><LI> See protocol of Midiprep kit </LI><LI> 1xuse-centrifugation-columns broke in centrifuge during last step, DNA rescue was tried by resuspending rest of the solution in the tube in EB buffer and redo the last steps (isoprop and ethanol precipitation) </LI><LI> Airdry &quot;pellet&quot;(no pellet visible) for 15 min under fume hood and resuspend in 100µl EB Buffer </LI><LI> Measure concentration with nanodrop</LI></UL></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Preparation_of_Collagen_.2B_Biotinylation">Preparation of Collagen + Biotinylation</SPAN></H3><P><B>Investigator: VL, JL</B></P><P><B>Aim of the experiment:</B> prepare collagen solution + biotinylate small sample 
</P><P><B>Procedure:</B></P><UL><LI> Dissolve 5 g collagen in 50 mL Tris/HCl pH 8 20 mM w/o salt --&gt; 770 uM solution (100 mg/mL)</LI><LI> Sterile filter</LI><LI> Biotinylation: 980 uL Tris buffer + 20 uL collagen solution + 30 uL biotin-NHS (100 mM in DMF, 200x molar excess over collagen monomers)</LI><LI> Reaction time: </LI><LI> Dialyze 4 °C Tris/HCl pH 8 20 mM w/o salt o/n</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Transformation_of_P9_.28pASK75.29">Transformation of P9 (pASK75)</SPAN></H3><P><B>Investigator: JL, VL</B></P><P><B>Aim of the experiment:</B> transform streptavidin expression vector P9 (clone 4) into E. coli BL21
</P><P><B>Procedure:</B></P><UL><LI> Standard (see booklet): 45 s at 42 °C; incubate 1h, 37°C; plate: medium w. chloramphenicol (+ rescue plate) Incubate o/n 37°C.</LI><LI> Wrong antibiotic plate, repeated the next day of the experiment:<B> Massively amplifying the pSB1C3 vector (which will be needed for building up new biobricks) by midiprepping the P40 plasmid (RFP-generator in pSB1C3). Similar to minipreps, but on a greater scale.</B></LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_June_14th"><B>Tuesday, June 14th</B></SPAN></H2><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Mass_spectrometry_of_biotinylated_eGFP">Mass spectrometry of biotinylated eGFP</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> identify biotinylation degree of eGFP
</P><P><B>Procedure:</B></P><PRE>[generic_display_report] (i could not upload this file, sorry.)
</PRE><P><B>Results:</B> about 4-5 biotins per protein. To verify the biotinylation degree the unbiotinylated eGFP will be measured.
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_F32_.28PCR_iGEM_1_mRuby-EGFR.29_and_F33_.28PCR_iGEM_2_IRES-BirA.29_in_pSB1C3">Cloning of F32 (PCR iGEM_1 mRuby-EGFR) and F33 (PCR iGEM_2 IRES-BirA) in pSB1C3</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Purification of PCR products
</P><P><B>Procedure:</B></P><UL><LI> Volume of PCR reaction: 50 µL</LI><LI> PCR products were purified by manufacturers protocol (QIAquick PCR purification kit, Qiagen)</LI><LI> Concentration measured by Nanodrop:</LI><LI> F32: 87,5 ng/µL</LI><LI> F33: 68,3 ng/µL</LI><LI> Test if PCR worked:</LI><LI> Samples were applied on a 2% agarose gel and was run at 80 V for 40 min.</LI></UL><P><B>Results:</B></P><UL><LI> PCR of F28 worked --&gt; F32</LI><LI> PCR of F29 did not work --&gt; wrong primers were used for amplification --&gt; F33 not right (Re-Do PCR with appropriate primers</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_F32_.28PCR_iGEM_1_mRuby-EGFR.29_and_F33_.28PCR_iGEM_2_IRES-BirA.29_in_pSB1C3_2">Cloning of F32 (PCR iGEM_1 mRuby-EGFR) and F33 (PCR iGEM_2 IRES-BirA) in pSB1C3</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Digestion of F32 (SpeI, NgoMIV) and F33 (XbaI, PstI)
</P><P><B>Procedure:</B></P><UL><LI> Digestion F32: 27 µl of F32 (2.4 µg), 5 µl CutSmart Buffer, 1.5 µl SpeI, 2 µl NgoMIV, 14.5 µl H2O</LI><LI> Digestion F33: 27 µl of F32 (1.8 µg), 5 µl CutSmart Buffer, 1.5 µl XbaI, 1.5 µl PstI, 15 µl H2O</LI><LI> Incubation for 3 h at 37°C</LI><LI> Preperative gel:</LI></UL><UL><LI> Gelextraction of F32 upper band and F32 lower band was executed by manufacturers protocol (Qiagen)</LI><LI> F32 upper band (mRuby) was named F34 (=10 g/yl)</LI><LI> F32 lower band (eGFR) was named F35 (=6 ng/yl)</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_F32_.28PCR_iGEM_1_mRuby-EGFR.29_and_F33_.28PCR_iGEM_2_IRES-BirA.29_in_pSB1C3_3">Cloning of F32 (PCR iGEM_1 mRuby-EGFR) and F33 (PCR iGEM_2 IRES-BirA) in pSB1C3</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Ligation of F34 (mRuby) and F35 (eGFR) in F30 (pSB1C3); Tranformation in XL1blue
</P><P><B>Procedure:</B></P><UL><LI> Ligation mixtures were prepared as following:</LI><LI> F34: 8.0 μL F34, 2 μL F30, 2 μL Ligase buffer, 7 μL ddH2O, 1 μL Ligase) </LI><LI> F35: 7.6 μL F35, 2.4 μL F30, 2 μL Ligase buffer, 7 μL ddH2O, 1 μL Ligase) </LI><LI> Ligation was incubated for 30 min at RT</LI></UL><PRE>[something is missing here] 
</PRE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Re-Sequencing_of_P37_.28Tet_repressed_YFP.29">Re-Sequencing of P37 (Tet repressed YFP)</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B></P><P>Purification and sequencing of P30
</P><P><B>Procedure:</B></P><UL><LI> P30 was heated up to 100 °C and centrifugated afterwards to remove proteins from Promotor (if there are some), the supernatant is sequenced</LI><LI> Sequencing batches were prepared after manufacturer's protocol </LI><LI> (15 μL plasmid DNA (50-100 μM) and 2 μL sequencing primer)</LI><LI> The plasmid we prepared received the following barcode: P37 (VF2) : FR11326620</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Streptavidin_expression">Streptavidin expression</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B>
Transformation of P9 (pASK74 (SA1)) in BL21
</P><P><B>Procedure:</B></P><UL><LI> Transformation was executed following SOP</LI><LI> Transformed cells were plated on LB medium (Amp) (50 µL and rescue (50 µL)</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Dialysis_of_Streptavidin_and_biotinylated_BSA">Dialysis of Streptavidin and biotinylated BSA</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Streptavidin and bitinylated BSA are both in Tris/HCl without salt. SInce we will perform later experiments in salty solutions the dialysis was performed against 137 mM NaCl (Saltconcetration of PBS)
</P><P><B>Procedure:</B></P><UL><LI> 3 L Tris/HCl NaCl solution was prepared</LI><LI> Dialysis was prepared according to SOP</LI><LI> Cut off Dialysis hose: 14 kDa</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_.28pre.29cultures_of_P40_for_a_pSB1C3_midiprep_.5Brepeat.5D">Inoculation of (pre)cultures of P40 for a pSB1C3 midiprep [repeat]</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Aim of the experiment:</B> Massively amplifying the pSB1C3 vector by midiprepping the P40 plasmid (RFP-generator in pSB1C3)
</P><P><B>Procedure:</B>  four 4 ml precultures were inoculated with a clone from the plate previously used for minipreps. After the precultures had grown for 6 hours, 50 ml cultures were inoculated and shaken overnight at 37°C.
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Subcloning_of_P41_.28strong_RBS-p.28cat.29.29_Insert_into_pSB1C3">Subcloning of P41 (strong RBS-p(cat)) Insert into pSB1C3</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Aim of the experiment:</B> Moving P41 (pSB1A2; Pcat+weakRBS) to pSB1C3 for biobrick submission
</P><P><B>Procedure:</B></P><UL><LI> Digestion of P41 with XbaI and PstI (10 µl DNA, 1,5 µl each enzyme, 3 µl CutSmart buffer x10, 14 µl ddH2O) -&gt; Incubation at 37°C for 2h. </LI><LI> Electrophoresis: 2% Agar, 40 min , 90 V (1kb ladder, empty, P41, 100b ladder)</LI><LI> GelExtraction</LI></UL><P><B>Resuts:</B> no visible signal of insert at approx. 100b. Area was cut blind for GelEx. No DNA in the sample according to NanoDrop measurement. Repeated the next day.
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_colonies_from_NanoLuc">Inoculation of colonies from NanoLuc</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Procedure:</B></P><UL><LI> 2x 4 ml LB+Cam media</LI><LI> Each culture was inoculated with one colony</LI><LI>Incubation at 37°C overnight</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_June_15th"><B>Wednesday, June 15th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Subcloning_of_P41_.28strong_RBS-p.28cat.29.29_Insert_into_pSB1C3_2">Subcloning of P41 (strong RBS-p(cat)) Insert into pSB1C3</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Moving P41 (pSB1A2; Pcat+weakRBS) to pSB1C3 for biobrick submission. No gel extraction was performed because the fragement is too small to visualize on the gel.
</P><P><B>Procedure:</B></P><UL><LI> Digestion of P41 with XbaI and PstI (10 µl DNA, 1,5 µl each enzyme, 3 µl CutSmart buffer x10, 1 µl FastAP, 13 µl ddH2O) -&gt; Incubation at 37°C for 2 h, heat inactivation at 80°C for 20 min. </LI></UL><P>fragement is to small to visualize on agel.
</P><UL><LI> Subcloning into F31: 3.8 μL F38, 6.2 μL F31, 2 μL Ligase buffer, 7 μL ddH2O, 1 μL Ligase according to the SOP</LI><LI> Transformation into E. coli XL1 according to the SOP</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_pSB1C3.28Nanoluc.29">Miniprep of pSB1C3(Nanoluc)</SPAN></H3><P><B>Investigator: LK</B></P><P><B>Aim of the experiment:</B> Preperation of a BioBrick with NanoLuciferase in pSB1C3
</P><P><B>Procedure:</B></P><UL><LI>The Miniprep was done according to the protocol. </LI><LI>The two tubes where pooled. </LI><LI>The new Plasmid is found under P54 and was sent to sequencing. </LI></UL></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Polymerisation">Polymerisation</SPAN></H3><P><B>Investigator: EU</B></P><P><B>Aim of the experiment:</B> Determine optimal polymerisation conditions for BSA and Collagen
</P><P><B>Procedure:</B> The experiment was performed in a 96 Well plate:
</P><P><B>Results:</B> maximum concentration of Streptavidin and biotinylated Protein works best. Collagen does not work. Glycerolconcentrations do not inhibit interaction. Reaction might be to slow at given concentrations to be effective in printing process.
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="MidiPrep_of_pSB1C3_with_RFP_generator_.28P4.29">MidiPrep of pSB1C3 with RFP generator (P4)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> purification of pSB1C3 with RFP generator for further processing (P56)
</P><P><B>Procedure:</B></P><UL><LI>MidiPrep was performed according to manufacturer's protocol (QIAprep MidiPrep, Qiagen)</LI></UL><P><B>Results:</B>   
Concentration very low (12ng/μL), but high volume (500 μL)
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_June_16th"><B>Thursday, June 16th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="MiniPrep_of_F36_.28Ligation_mRuby_in_pSB1C3.29_K1_.26_K2.2C_F37_.28Ligation_EGFR_in_pSB1C3.29.2C_I13453_.28pBad_promotor.29_K1_.26_K2.2C_K577881_.28pC.2BAraC.2BpBad.2BGFPc.29_K1_.26_K2">MiniPrep of F36 (Ligation mRuby in pSB1C3) K1 &amp; K2, F37 (Ligation EGFR in pSB1C3), I13453 (pBad promotor) K1 &amp; K2, K577881 (pC+AraC+pBad+GFPc) K1 &amp; K2</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> isolate Plasmids
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep was performed according to manufacturer's protocol (QIAprep MiniPrep, Qiagen)</LI><LI> Plasmids were named as following: [concentration]</LI></UL><P>F36 K1: P57 [212,9 ng/μL]
</P><P>F36 K2: P58 [181,3 ng/μL]
</P><P>F37: P59 [195,5 ng/μL]
</P><P>K577881 K1: P60 [557,2 ng/μL]
</P><P>K577881 K2: P61 [791,9 ng/μL]
</P><P>I13453 K1: P62 [130 ng/μL]
</P><P>I13453 K2: P63 [460 ng/μL]
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P29_.28double_terminator.29.2C_P36_.28Tet_repressed_RFP.29_and_P37_.28Tet_repressed_YFP.29">Sequencing of P29 (double terminator), P36 (Tet repressed RFP) and P37 (Tet repressed YFP)</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Sequencing of P56, P57, P59, P61 and P62 
</P><P><B>Procedure:</B>  According to the manufacturer's protocol.
</P><UL><LI> P56: FR11326618 (VF2)</LI><LI> P57: FR11326617 (VF2)</LI><LI> P59: FR11326616 (VF2)</LI><LI> P61: FR11326613 (VR), FR11326615 (VF2)</LI><LI> P63: FR11326614 (VF2)</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="MidiPrep_P4_.28pSB1C3.29">MidiPrep P4 (pSB1C3)</SPAN></H3><P><B>Investigator: VL</B></P><P><B>Aim of the experiment:</B> isolate Plasmids
</P><P><B>Procedure:</B></P><UL><LI> MidiPrep was performed according to manufacturer's protocol (QIAprep MidiPrep, Qiagen). Centrifugation steps at 20,000 g performed at 12,500 rpm in special centrifugation tubes. Since only one column was left, both prepped cell pellets were applied to one column. Flow-through and wash was collected.</LI><LI> Conc:</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Amplification_of_BioBricks_.28overnight_cultures_for_minipreps.29">Amplification of BioBricks (overnight cultures for minipreps)</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Inoculation of 5 ml overnight cultures for minipreps. The following biobricks were used:
</P><UL><LI> BBa_K157012</LI><LI> BBa_K243004</LI><LI> BBa_K243006         </LI><LI> BBa_K157001</LI><LI> BBa_K243005</LI><LI> BBa_I712009</LI><LI> BBa_K404108</LI><LI> BBa_K782061</LI><LI> BBa_K1371012</LI><LI> BBa_K782060</LI></UL><P><B>Procedure:</B></P><UL><LI> Three hours of incubation of the tubes containing the bacterial agar at 37°C (Note: 12 hours are actually recommende)</LI><LI> Inoculation of two times 5 ml preculture containing the appropriate antibiotic per clone. For KanRAmpR (reistance against both ampicillin and kanamycin) it was only selected for ampicillin.</LI><LI> Incubation shaking overnight at 37°C</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_Amplification_of_genesynthesis_2_.28F29.3B_IRES-BirA.29">PCR Amplification of genesynthesis 2 (F29; IRES-BirA)</SPAN></H3><P><B>Investigator: LK</B></P><P><B>Aim of the experiment:</B> Amplification of Genesynthesis_2 (IHRES_BirA_StrepTag) 
</P><P><B>Procedure:</B> A 50µl approach was made: 
</P><UL><LI> 33,5 µl Water</LI><LI> 2µl DNA (F29, diluted 1:10)</LI><LI> 2,5µl of each Primer (O11 and O12 c:10µM) </LI><LI> 0,5µl dNTPmix</LI><LI> 5µl Reaction buffer</LI><LI> 0,5µl Q5-Polymerase</LI><LI> The Annealing Temp. was set an 58°C </LI><LI> Next step is to run an analytic gel (band should be visible at 1,5 kB) </LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_June_17th"><B>Friday, June 17th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_genesynthesis_2_.28IRES-BirA.29_into_pSB1C3">Cloning of genesynthesis 2 (IRES-BirA) into pSB1C3</SPAN></H3><P><B>Investigator: CG, NA, JH</B></P><P><B>Aim of the experiment:</B> Purification oft he PCR reaction, gel verification, restriction digestion (F29), ligation with F31 and transformation  
</P><P><B>Procedure:</B></P><UL><LI> Yesterdays PCR was purified according to Qiagen PCR purification kit</LI><LI> An anlaytic gel electrophoresis was performed on a 1% gel with 4 µl purified PCR product</LI><LI> Restriction digest: 26 µl purified PCR product, 1.5 µl XbaI, 1.5 µl PstI, 5 µl NEB SmartCut, 16 µl H2O. Incubation for 2 h at 37°C.</LI><LI> Gel extraction</LI><LI> Ligation: F31</LI><LI> Transformation</LI></UL><P><B>Results:</B> Gel electrophoresis after PCR purification exhibit a thin signal at 1.5 kb. Next time 1:30 min for elongation time might increase the signal.
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_new_iGEM_BioBricks">Miniprep new iGEM BioBricks</SPAN></H3><P><B>Investigator: VL</B></P><P><B>Aim of the experiment:</B> Purify plasmid DNA from transformed cells
</P><P><B>Procedure:</B></P><UL><LI> Miniprep according to manual (pellet centrifuged 5000 rpm, 10 min, 4 °C)</LI><LI> Elution with 50 uL EB Buffer</LI><LI> Concentrations (all ng/uL):</LI></UL><P>K782061 (a) 172.9, (b) 53.0
</P><P>K243006 (a) 241.1, (b) 58.6
</P><P>K1371012 (a) 138.0, (b) 43.0
</P><P>K157001 (a) 184.3, (b) 186.4
</P><P>K782060 (a) 339.0, (b) 254.0
</P><P>K243005 (a) 62.3, (b) 176.2
</P><P>K243004 (a) 122.3, (b) 110.3
</P><P>I711009 (a) 185.2, (b) 174.7
</P><P>K157012 (a) 181.2, (b) 195.3
</P><P>K404108 (a) 116.4, (b) 142.5
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="MidiPrep_of_pSB1C3_with_RFP_generator_.28P4.29_2">MidiPrep of pSB1C3 with RFP generator (P4)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> purification of pSB1C3 with RFP-Reportercasette for further processing (P74)
</P><P><B>Procedure:</B>   
MidiPrep was performed according to manufacturer's protocol (QIAprep MidiPrep, Qiagen)
</P><P><B>Results:</B></P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_IRES-BirA_.28F33-.3EF40.29_into_digested_F31_.28pSB1C3.29">Cloning of IRES-BirA (F33-&gt;F40) into digested F31 (pSB1C3)</SPAN></H3><P><B>Investigator: JH, LK</B></P><P><B>Aim of the experiment:</B> Ligation of IRES-BirA (F29-&gt;F33-&gt;F40) into F31 (digested P40 by XbaI and PstI = pSB1C3 without insert), gelelectrophoresis and tranformation in XL1blue
</P><P><B>Procedure:</B></P><UL><LI> F33 (PCR product of F29/gen synthesis 2) was digested with XbaI and PstI</LI><LI> Gelextraction of band at 1500 bp was executed by manufacturers protocol (Qiagen)</LI><LI> intermediate result: F40 with 6.6 ng/µL</LI><LI> Ligation mixtures were prepared as following:</LI><LI> 9.6 µL F40, 0.4 µL F31, 2 μL Ligase buffer, 7 μL ddH2O, 1 μL Ligase</LI><LI> Transformation: (Saturday) </LI><LI> Transformation of Ligation (F29+F31) into XL-1 blue was done according to the SOP. The plate was incubated until Sunday at 37°C. </LI><LI> 3mL-cultures in LB+ 30µg/ml cam where inoculated on Sunday </LI><LI> Mini-prep according to manufacturer's protocol was done on Monday.</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR.2C_purification_and_digestion_of_P19_.28EspP.29_and_F28_.28mRuby_EGFR.29_for_BioBrick_creation">PCR, purification and digestion of P19 (EspP) and F28 (mRuby EGFR) for BioBrick creation</SPAN></H3><P><B>Investigator: JB, LK</B></P><P><B>Aim of the experiment:</B> Amplification of EspP from P19 and mRuby/EGFR (F28) for creation of Biobricks
</P><P><B>Procedure (for both fragments):</B></P><UL><LI> PCR according to the SOP (primers for P19: O13 and O03, primers for F28: VR and VF2)</LI><LI> PCR purification via the purification kit according to the manufacturer's protocol (concentrations: P19(EspP):119,5 ng/µl F28(mRuby):56,2ng/µl</LI><LI> Analysis on an 1% agarose gel using 5 µl purified PCR product</LI><LI> Restriction digestion with NgoMIV/SpeI (for both cases) </LI><LI> 40µl DNA</LI><LI> 1,5µl of each Enzyme</LI><LI> 2µl ddH2O</LI><LI> 5µl CutSmart Buffer </LI><LI> The tubes where incubated at 37°C for ~2 days.</LI></UL><P><B>Results:</B> 
Analytical gel electrophoresis after PCR purification shows thick bands at roughly 1000 bp 
</P><P><B>Still to do:</B></P><UL><LI> Gel extraction of both signals for F28 and one signal for EspP-PCR</LI><LI> Ligation with appropriate vector (e.g. F30)</LI><LI> Transformation</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Minipreps_of_F39_.28strong_RBS-pcat_in_pSB1C3.29">Minipreps of F39 (strong RBS-pcat in pSB1C3)</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Amplification of F39 (ligation, pCat with strong RBS)
</P><P><B>Procedure:</B></P><UL><LI> Miniprep according to the manufacturer's protocol</LI><LI> Resulting plasmid: P76</LI><LI> Concentration: 52 ng/µl</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Minipreps_of_pSB1C3_with_RFP_generator_.28P4.29">Minipreps of pSB1C3 with RFP generator (P4)</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Aim of the experiment:</B> purification of pSB1C3 with RFP-Reportercasette for further processing (P75)</P><P><B>Procedure:</B></P><UL><LI> Minipreps according to the manufacturer's protocol (with 8* 6 mL culture)</LI><LI> Resulting plasmid: P75</LI><LI> Concentration: 142.2 ng/µl [~200 µL]</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_colony_from_NanoLuc_.5BP54.2C_Repeat.5D">Inoculation of colony from NanoLuc [P54, Repeat]</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Procedure:</B></P><UL><LI> 5 ml LB+Cam media</LI><LI> culture was inoculated with one colony</LI><LI> Incubation at 37°C overnight</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_June_18th"><B>Saturday, June 18th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_P54_.28NanoLuc.29">Miniprep of P54 (NanoLuc)</SPAN></H3><P><B>Investigator: LK</B></P><P><B>Aim of the experiment:</B> Purification of P54 (NanoLuc)
</P><P><B>Procedure:</B></P><UL><LI> Miniprep according to the manufacturer's protocol</LI><LI> Resulting plasmid: pSB1C3 mit NanoLuc (P54, von )</LI><LI> Concentration: 497,9 ng/µl</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Hybridisation_of_O19_and_O20_.28StrepTag.29">Hybridisation of O19 and O20 (StrepTag)</SPAN></H3><P><B>Investigator: LK</B></P><P><B>Aim of the experiment:</B> Hybridisation of the Strep-Oligos
</P><P><B>Procedure:</B>
2µl of each Oligo where diluted in 16µl ddH2O and incubated for 10 min. at 95°. Cooling down at RT for at least half an hour. Stored in fridge 
</P></DIV></DIV><H1><SPAN class="mw-headline" id="Week_6_.28June_20th_-_June_26th.29">Week 6 (June 20th - June 26th)</SPAN></H1><DIV class="week" id="Week_6"><H2><SPAN class="mw-headline" id="Monday.2C_June_20th"><B>Monday, June 20th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_F42_.28IRES-BirA_in_pSB1C3.29">Miniprep of F42 (IRES-BirA in pSB1C3)</SPAN></H3><P><B>Investigator: LK</B></P><P><B>Aim of the experiment:</B> Purification of IRES-BirA Plasmids
</P><P><B>Procedure:</B></P><UL><LI> Miniprep according to the manufacturer's protocol</LI><LI> Resulting plasmid: pSB1C3 with IRES-BirA</LI></UL><P>Next Steps: 
</P><UL><LI> analytical digestion with EcoRI+PstI and gel (expected bands at 1500 and 2000 Bp)</LI><LI> Sequencing </LI></UL><P><B>Results:</B>
Concentration: 583,5 ng/µl
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_and_Purification_of_BirA">PCR and Purification of BirA</SPAN></H3><P><B>Investigator: LK, JB</B></P><P><B>Aim of the experiment:</B> Amplification of BirA-construct 
</P><P><B>Procedure:</B></P><UL><LI> PCR according to the SOP (primers: O17 and O18)</LI><LI> PCR purification via the purification kit according to the manufacturer's protocol </LI><LI> Analysis on an 1% agarose gel using 5 µl purified PCR product</LI></UL><P><B>Results:</B></P><UL><LI> Concentration: 185,8 ng/µl --&gt; F41 </LI><LI> Analytical gel shows no significant band at 960 Bp --&gt;Repeat</LI></UL><P>Next Steps: 
</P><UL><LI> Repetition of PCR and analytical gel! </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Gel_and_Gelextraction_of_P19_.28QC_EspP.29_and_F28_.28EspP_and_mRuby.29">Gel and Gelextraction of P19 (QC EspP) and F28 (EspP and mRuby)</SPAN></H3><P><B>Investigator: LK</B></P><P><B>Aim of the experiment:</B> Purification of both constructs for Ligation  
</P><P><B>Procedure:</B></P><UL><LI> Preperative gel according to SOP</LI><LI> The Purification was done according to the manufacturer's protocol. The EGFR gel-fragment was eluted in 50µl and the mRuby-fragment in 30µl (because of thin band) </LI></UL><P><B>Results:</B></P><UL><LI> One thick band just before 1000Bp and one lighter band just before 750 Bp. This fits to the lengths of EspP and mRuby. </LI><LI> Concentrations: EspP(now F43): 49,5ng/µl   mRuby(now F44): 9,9ng/µl</LI></UL><P>Next Steps: 
</P><UL><LI> Ligation into pSB1C3</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_F42_.28pSB1C3.28IRES-BirA.29.29.2C_P54_.28pSB1C3.28NanoLuc.29.29_.2C_P74_.28pSB1C3.28RFP_generator.29.29_and_P76_.28pSB1C3.28strong_RBS-pCat.29.29">Sequencing of F42 (pSB1C3(IRES-BirA)), P54 (pSB1C3(NanoLuc)) , P74 (pSB1C3(RFP generator)) and P76 (pSB1C3(strong RBS-pCat))</SPAN></H3><P><B>Investigator: JL</B></P><P><B>Aim of the experiment:</B> Sequencing of F42, P54, P74 and P76 in forward direction.
</P><P><B>Procedure:</B>  
Sequencing batches were prepared after manufacturer's protocol (15 µL plasmid DNA (50-100 µM) and 2 µL sequencing primer)
</P><P>The different plasmids we prepared received the following barcodes:
</P><UL><LI> F42 (VF2): FR11326601</LI><LI> P54 (VF2): FR11326600</LI><LI> P74 (VF2): FR11326599</LI><LI> P76 (VF2): FR11326598</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="PCR_and_Purification_of_genesynthesis_6_.28iGEM6.29_Streptavidin.2FTraptactin">PCR and Purification of genesynthesis 6 (iGEM6) Streptavidin/Traptactin</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Amplification of gen synthesis 6
</P><P><B>Procedure:</B>
PCR according to the SOP (primers: VF2 and VR)
</P><P>Next Steps: 
</P><UL><LI> PCR purification via the purification kit according to the manufacturer's protocol </LI><LI> Analysis on an 1% agarose gel using 5 µl purified PCR product</LI><LI> PCR purification via the purification kit according to the manufacturer's protocol </LI><LI> Analysis on an 1% agarose gel using 5 µl purified PCR product</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_and_Purification_of_BirA_.28F41.29">PCR and Purification of BirA (F41)</SPAN></H3><P><B>Investigator: NA </B></P><P><B>Aim of the experiment:</B> Amplification of BirA (F29)
</P><P><B>Procedure:</B>
PCR according to the SOP (primers: O17 and O18)
</P><P>Next Steps: 
</P><UL><LI> PCR purification via the purification kit according to the manufacturer's protocol </LI><LI> Analysis on an 1% agarose gel using 5 µl purified PCR product</LI><LI> PCR purification via the purification kit according to the manufacturer's protocol </LI><LI> Analysis on an 1% agarose gel using 5 µl purified PCR product</LI></UL><P>--&gt; repeat with gene-synthesis 2 (F29 may not be the gene-synthesis 2)
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Ligations_of_StrepTag.2C_EspP_and_mRuby3_with_F30_.28digested_pSB1C3.29">Ligations of StrepTag, EspP and mRuby3 with F30 (digested pSB1C3)</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Aim of the experiment:</B> Ligations of StrepTag (O19+O20), EspP (F43) and mRuby3 (F44) into F30 (NgoMIV and SpeI; pSB1C3 without insert)
</P><P><B>Procedure:</B></P><UL><LI> Ligation mixtures were prepared as following:</LI><LI> O19+O20: 1.7 (O19+O20), 8.3 μL F30, 2 μL Ligase buffer, 7 μL ddH2O, 1 μL Ligase</LI><LI> F43: 	4.6 μL F43, 5.4 µL F30, 2 μL F30, 2 μL Ligase buffer, 7 μL ddH2O, 1 μL Ligase </LI><LI> F44: 	8.1 μL F44, 1.9 μL F30, 2 μL Ligase buffer, 7 μL ddH2O, 1 μL Ligase </LI><LI> Ligation was incubated overnight at 16 °C</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Digestion_of_GSY5_.28Streptactin.3B_F27.29">Digestion of GSY5 (Streptactin; F27)</SPAN></H3><P><B>Investigator: LK</B></P><P><B>Aim of the experiment:</B> Restricting GSY5 into StrepActin fragment (Streptavidin mutant) and Leptin fragment (Leptin is for Volker)    
</P><P><B>Procedure:</B></P><UL><LI> Restriction mix:  </LI></UL><P>32µl DNA (F27)  
</P><P>10µl ddH2O 
</P><P>1µl of each enzyme (SpeI, HindIII, SapI) 
</P><P>5µl cut smart buffer  
</P><P>Next Steps:  
</P><UL><LI> Preperative gel, 1,5% Agarose, cut out and purify bands at 80 Bp and 425 Bp. (80 is leptin for volker and 425 is StreptActin for further cloning into pASK)  </LI></UL><P><B>Results:</B></P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_June_21st"><B>Tuesday, June 21st</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Plating_of_BioBricks_that_didn.27t_survive_sequencing">Plating of BioBricks that didn't survive sequencing</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Instead of directly inoculating cultures with the transformed cells from the bacterial agar that was sent by the HQ, the bacterial agar was now used for overnight plating on agar plates containing the appropriate antibiotic resistance. 
</P><P><B>Procedure:</B> Two methods were used for each BioBrick: For the first method, an inoculation loop was used for plating directly. For the second methods, the inoculation loop was used to inoculate a 50 µl LB culture that 
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_various_plasmids">Sequencing of various plasmids</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Sequencing of various plasmids with either VF2 or VR
</P><P><B>Procedure:</B>  
Refer to the “Inventarliste” for detailed informations on sequencing codes and primers. Sequencing batches were prepared according to manufacturer's protocol (15 µL plasmid DNA (50-100 µM) and 2 µL sequencing primer)
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_of_gene_synthesises_.28iGEM_1.29_.28mRuby_EGFR.29">PCR of gene synthesises (iGEM_1) (mRuby_EGFR)</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> iGEM_1_mRuby_EGFR (F28) for preparing the EGFR signal (at about 100 bp)
</P><P><B>Procedure:</B>
Approach: 
</P><UL><LI> 1µl DNA (diluted, 1:10) </LI><LI> 25 µl 2x MasterMix </LI><LI> 0,5µl dNTPs</LI><LI> 2,5 µl forward Primer VF2</LI><LI>2,5µl reverse Primer VR</LI><LI> Fill up with water up to 50 µl</LI></UL><P>Setup: (calculated with NEB calc. <A rel="nofollow" class="external free" href="http://tmcalculator.neb.com/#!/">http://tmcalculator.neb.com/#!/</A>) 
</P><UL><LI> 98°C_1min initiation step</LI><LI> 98°C_10sec</LI><LI> 66°C_30sec   &gt;  step 2-4: 35 repeats</LI><LI> 72°C_30sec</LI><LI> 72°C_2min final step </LI><LI> 4°_hold</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_Ligations_of_F50_.28StrepTag.29.2C_F51_.28EspP.29_and_F52_.28mRuby3.29_in_pSB1C3">Transformation of Ligations of F50 (StrepTag), F51 (EspP) and F52 (mRuby3) in pSB1C3</SPAN></H3><P><B>Investigator: JH</B></P><P><B>Aim of the experiment:</B> Transformation of StrepTag (F50), EspP (F51) and mRuby3 (F52) in XL1 blue
</P><P><B>Procedure:</B> 
transformation in competent E. Coli XL1 blue according to protocol + rescue 
(repeat due to incubation with LB+Cam ! )
</P><P><B>Result:</B> 
plates (LB Cam) in incubator for further processing (37 °C)
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Digestion_of_P9_.28pASK75.29.2C_P74_.28pSB1C3.29.2C_P26_.28CMV_promotor.29_and_F47_.28BirA.29">Digestion of P9 (pASK75), P74 (pSB1C3), P26 (CMV promotor) and F47 (BirA)</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of the experiment:</B> Digestion of P9 and P74 to gain the vectors pASK75 and pSB1C3, respectively, as well as digestion of F47 (PCR fragment of BirA) and P26 (CMV promoter) for the ultimate creation of biobricks
</P><P><B>Procedure:</B> P74 was digested in a total volume of 30 µl, while the others were digested in a total volume of 50 µl. Enzymes used were: 
</P><UL><LI> P74: XbaI and PstI</LI><LI> P9: HindIII and XbaI</LI><LI> F47: Xba I and PstI (didn’t work)</LI><LI> P26: SpeI and PstI </LI></UL><P>The digestions were incubated at 37°C for 40 minutes and were thereafter inactivated at 80°C for 5 mins. 
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_cultures_for_minipreps_for_P76_.28pCat_.2B_strong_RBS.29">Inoculation of cultures for minipreps for P76 (pCat + strong RBS)</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Procedure:</B> Inoculation of 2x5 ml cultures with P76 clones. Incubation at 37°C overnight.
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_June_22nd"><B>Wednesday, June 22nd</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Minipreps_of_P79_.28pCat.2BRBS.29">Minipreps of P79 (pCat+RBS)</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Procedure:</B> According to the manufacturer's protocol. 
</P><P><B>Results:</B> Concentration 212 ng/µl. Sent to sequencing
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_ligation_F56_.28pASK75.28Streptactin.29.29.2C_P60_.28pSB1C3_.28pC.2BAraC.2BpBad.2BGFPc.29.29_.2C_P70_.28pSB1C3_.28short_linker.29.29">Transformation of ligation F56 (pASK75(Streptactin)), P60 (pSB1C3 (pC+AraC+pBad+GFPc)) , P70 (pSB1C3 (short linker))</SPAN></H3><P><B>Investigator: CG</B></P><P><I>Aim of the experiment:'</I> Transformation of pASK75-Streptactin (F56) into E. <I>coli</I> XL1 blue, K577881 (P60) into BL21 and K243004 (P70) in XL1
</P><P><B>Procedure:</B> 
transformation in competent E. Coli XL1 blue according to protocol
</P><P><B>Result:</B> 
plates (LB + appropriate antibiotic) in incubator for further processing (37 °C)
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_colonies_from_K157001_.28EGFR_signal_peptide.29.2C_EspP_.28F50.29.2C_mRuby_.28F51.29.2C_Strep-Tag_.28F52.29">Inoculation of colonies from K157001 (EGFR signal peptide), EspP (F50), mRuby (F51), Strep-Tag (F52)</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Procedure:</B></P><UL><LI> 2x 5 ml LB+Cam (Amp for K157001)  media</LI><LI> Each culture was inoculated with one colony, for ligations 2 colonies each</LI><LI> Incubation at 37°C overnight</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Plateing_of_K243004_.28short_linker.29.2C_K243005_.28middle_linker.29_from_stab_cultures">Plateing of K243004 (short linker), K243005 (middle linker) from stab cultures</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Procedure:</B>  
For both biobricks: stabing into the agar with the inocluation loop and resuspending in 50 µl LB media. Plating on Amp-agar.
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Digestion_of_P64_.28EGFR_sgnal_peptide.29_and_P71_.28CD4_signal_peptide.29_for_cloning_into_F57_.28pSB1C3_.28CMV_promotor.29.29">Digestion of P64 (EGFR sgnal peptide) and P71 (CD4 signal peptide) for cloning into F57 (pSB1C3 (CMV promotor))</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Digestion of P67 and P71 for cloning the resulting fragments into the digested F57 fragement.
</P><P><B>Procedure:</B> both plasmids were digested in 50 µl, 3 µg for each plasmid DNA. 1 µl of XbaI and PstI, 5 µl Buffer, 20 µl DNA, 23 µl H2O. The digestions were incubated at 37°C overnight.
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_agarose_gels_for_separation_of_digestion_of_P64_.28EGFR_sgnal_peptide.29_and_P71_.28CD4_signal_peptide.29">Preparative agarose gels for separation of digestion of P64 (EGFR sgnal peptide) and P71 (CD4 signal peptide)</SPAN></H3><P><B>Investigators: NA, JB</B></P><P><B>Aim of the experiment:</B> Separation of digestion fragments of both digested PCR fragments as well as vectors
</P><P><B>Procedure:</B></P><UL><LI> P74: XbaI and PstI -&gt; Fragment corresponding to the pSB1C3 vector isolated</LI><LI> P9: HindIII and XbaI -&gt; Fragment corresponding to the pASK75 vector isolated</LI><LI> F47: Xba I and PstI (didn’t work) –&gt; only one fragment at 3 kbp present (weird since that was a gene synthesis pcr) </LI><LI> P26: SpeI and PstI  -&gt; Fragment corresponding to the cmv-promoter isolated</LI></UL><UL><LI> Gelextraction of P9, P74 and P26 was executed by manufacturers protocol (Qiagen)</LI></UL><P><B>Results:</B></P><UL><LI> F55: pASK75 (2,9 ng/µl)</LI><LI> F56: pSB1C3 (23,6 ng/µl)</LI><LI> F57: CMV-Promotor (5,9 ng/µl)</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_of_GSY2_.28IRES-BirA.29_with_O17.2F18">PCR of GSY2 (IRES-BirA) with O17/18</SPAN></H3><P><B>Investigators: LK</B></P><P><B>Aim of the experiment:</B> Amplification of BirA, using new dilution of GSY2 this time, because last 2 PCRs showed confusing results (bands at 2500) 
</P><P><B>Procedure:</B></P><UL><LI> PCR was performed  according SOP </LI><LI> Primers: O17 and O18</LI><LI> Annealing at 68°C   </LI></UL><P>Next Steps:
</P><UL><LI> PCR-Purification</LI><LI> Analytical Gel </LI><LI> Ligation into pSB1C3</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="QuickChange-PCR_of_pASK75_with_O21_and_O22">QuickChange-PCR of pASK75 with O21 and O22</SPAN></H3><P><B>Investigators: LK</B></P><P><B>Aim of the experiment:</B> Deleting NdeI from pASK (P03)       
</P><P><B>Procedure:</B></P><UL><LI> QC-PCR was performed according to SOP </LI><LI> Primers: O-021 and O-022</LI><LI> Annealing at 68°C</LI><LI> Elongation time: 3:30 min </LI></UL><P>Next Steps:
</P><UL><LI> Digestion with DpnI at 37°C for 1h                   </LI><LI> Purification?</LI><LI> Preperative Gel (band at 960), purification</LI><LI> Ligation into pSB1C3</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Verification_of_PCR_and_Digestion_of_F28_.28gene_synthesis_1.29">Verification of PCR and Digestion of F28 (gene synthesis 1)</SPAN></H3><P><B>Investigator: NA, LK</B></P><P><B>Aim of the experiment:</B> Verification of successful PCR via gel electrophoresis and preparation of EGFR 
</P><P><B>Procedure:</B></P><UL><LI> Gel electrophoresis with 1 kb-ladder</LI><LI> Digestion: 42µl of PCR Purification, 1,5 µl of each enzyme (NgoMIV and SpeI), 5µl of cut smart buffer. </LI><LI> Incubation at 37°C over night </LI></UL><P><B>Results:</B></P><P>Next steps:   
</P><UL><LI> Preparative gel and gel extraction</LI><LI> Ligation into pSB1C3</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_June_23th"><B>Thursday, June 23th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_E._coli_Xl1-Blue_transformed_with_ligation_product_P80.2F81_.28mRuby3_K1.2F2.29.2C_P82.2F83_.28EspP_K1.2F2.29.2C_P84.2F85_.28StrepTag_K1.2F2.29_and_Trafo_of_K157001">Miniprep of E. coli Xl1-Blue transformed with ligation product P80/81 (mRuby3 K1/2), P82/83 (EspP K1/2), P84/85 (StrepTag K1/2) and Trafo of K157001</SPAN></H3><P><B>Investigator: Jan, Julian</B></P><P><B>Aim of the experiment:</B> Miniprep of E. coli Xl1-Blue transformed with ligation product F50(K1,2), F51(K1,2), F52(K1,2) and Trafo of K157001
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)</LI><LI> Concentrations:</LI></UL><TABLE class="wikitable" cellspacing="0" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>c [ng/µl]</B></TD></TR><TR><TD>P80
</TD><TD>432,7
</TD></TR><TR><TD>P81
</TD><TD>294,8
</TD></TR><TR><TD>P82
</TD><TD>450,5
</TD></TR><TR><TD>P83
</TD><TD>479,0
</TD></TR><TR><TD>P84
</TD><TD>108,0
</TD></TR><TR><TD>P85
</TD><TD>356,0
</TD></TR><TR><TD>P86
</TD><TD>47,2
</TD></TR></TBODY></TABLE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_of_gene_synthesises">PCR of gene synthesises</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> iGEM_1_mRuby_EGFR (F28) for preparing the EGFR signal (at about 100 bp)
</P><P><B>Procedure:</B>  
Approach: 
</P><UL><LI> 1µl DNA (diluted, 1:10) </LI><LI> 25 µl 2x MasterMix </LI><LI> 0,5µl dNTPs</LI><LI> 2,5 µl forward Primer VF2</LI><LI> 2,5µl reverse Primer VR</LI><LI> Fill up with water up to 50 µl</LI></UL><P>Setup: (calculated with NEB calc. <A rel="nofollow" class="external free" href="http://tmcalculator.neb.com/#!/">http://tmcalculator.neb.com/#!/</A>) 
</P><UL><LI> 98°C_1min initiation step</LI><LI> 98°C_10sec</LI><LI> 66°C_30sec   &gt;  step 2-4: 35 repeats</LI><LI> 72°C_30sec</LI><LI> 72°C_2min final step </LI><LI> 4°_hold</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="QuickChange-PCR_of_pASK_with_O21_and_O22">QuickChange-PCR of pASK with O21 and O22</SPAN></H3><P><B>Investigators: CR</B></P><P><B>Aim of the experiment:</B> Digestion of PCR product with DpnI to digest methylated DNA       
</P><P><B>Procedure:</B></P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_purification_of_BirA_and_mRuby.2FEGFR">PCR purification of BirA and mRuby/EGFR</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> purification of PCR
</P><P><B>Procedure:</B> 
PCR purification via the purification kit according to the manufacturer's protocol
The samples were renamed:
</P><UL><LI> BirA: F59</LI><LI> mRuby/EGFP: F60</LI></UL><P>The samples were analysed on a 1% agarose gel. Samples were prepared after SOP.
</P><P><B>Results:</B></P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_P60_.28K577881.3B_pC.2BAraC.2BpBad.2BGFPc_K1.29_and_QC_P3_.28pASK75.28SA1.29.29">Transformation of P60 (K577881; pC+AraC+pBad+GFPc K1) and QC P3 (pASK75(SA1))</SPAN></H3><P><B>Investigator: EF, JB</B></P><P><B>Aim of the experiment:</B> Transformation of Q CP3 in XL1 blue; transformation of p60  in BL21
</P><P><B>Procedure:</B> 
transformation in competent E. Coli BL21 and XL1blue according to protocol
</P><P>1µl p60 in BL21
7 µl Q CP3 in XL1 blue
</P><P><B>Result:</B> plates (+ rescue plates) incubating
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_CD4-_and_EGFR-signal_peptide_into_pSB1C3-CMV">Cloning of CD4- and EGFR-signal peptide into pSB1C3-CMV</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Digestion of P71 and P69
<B>Procedure:</B></P><UL><LI> Digestions were prepared as following:</LI><LI> 20 µl CD4 (P71); 1 µl XbaI; 1 µl PstI; 5 µl CutSmart; 23 µl ddH2O</LI><LI> 7 µl pSB1C3-CMV; P26; 1 µl PstI; 1 µl SpeI; 2 µl CutSmart;  µl ddH2O</LI><LI> 20 µl EGFR-signal peptide (P67);1 µl XbaI; 1 µl PstI; 5 µl CutSmart; 23 µl ddH2O</LI><LI> Digestions were incubated for 3 h at 37 °C</LI><LI> The Samples CD4- and EGFR-signal peptide were applied to a 2% and the pSB1C3-CMV to a 1% agarose Gel</LI></UL><P><B>Results:</B></P><P>--&gt; passt
</P><P>--&gt; passt
</P><P>Next steps:
</P><UL><LI> Ligation of </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_EGFR_.28F62.29_.2F_CD4_.28F61.29_signale_peptide_into_pSB1C3-CMV_.28F63.29">Cloning of EGFR (F62) / CD4 (F61) signale peptide into pSB1C3-CMV (F63)</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Gelextraction and ligation of EGFR- and CD4-signal peptide into pSB1C3 and transformation
</P><P><B>Procedure:</B></P><UL><LI> Gel extraction according to manufacturers protocol (Qiagen): F61 (CD4) 2 ng/µl, F62 (EGFR) 0.6 ng/ µl, F63 (CMV): 45 ng/ µl</LI><LI> Ligation: 1 µl T4 Ligase, 2 µl Ligase buffer, 7 µl H20, 1.8 µl vector and 8.2 µl EGFR insert or 4.3 µl vector and 5.7 µl CD4 insert</LI><LI> Transformation according to SOP</LI><LI> Plating on LB-Cam agar</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_June_24th"><B>Friday, June 24th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P83_.28EspP_K2.29_and_P83_.28StrepTag_K2.29">Analytical digestion and gelelectrophoresis of P83 (EspP K2) and P83 (StrepTag K2)</SPAN></H3><P><B>Investigator: Julian, Niklas, Luisa</B></P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P82/83 (EspP K1/2) and P84/85 (StrepTag K1/2).
</P><P><B>Procedure:</B></P><UL><LI> Batch for analytical digestion for P82-P85 with EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.5/1.0 µl
</TD><TD>Plasmid DNA (-/P84)
</TD></TR><TR><TD>1 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF(10 U/µl)
</TD></TR><TR><TD>8/7.5 µl
</TD><TD>ddH2O (-/P84)
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><A href="/File:Muc16_P82-85_EcoR.png" title="File:Muc16 P82-85 EcoR.png">500px</A></P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Sequencing_of_P80.28_mRuby3_K1.29.2C_P83_.28EspP_K2.29_and_P85_.28StrepTag_K2.29">Sequencing of P80( mRuby3 K1), P83 (EspP K2) and P85 (StrepTag K2)</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of the experiment:</B> Sequencing of P80(mRuby3 K1), P83 (EspP K2) and P85 (StrepTag K2)
</P><P><B>Procedure:</B></P><P>Sequencing batch were prepared after manufacturer's protocol. (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer). Sequencing primer VF2 was used
</P><P>The different vectors we sequenced received the following barcodes:
</P><UL><LI> mRuby3 in pSB1C3 (P80):   FR11326590</LI></UL><UL><LI> EspP in pSB1C3 (P83):  FR11326588</LI></UL><UL><LI> Streptag in pSB1C3 (P85):  FR11326587</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_F64_.28quickchanged_P3.28pSAm1.29.29">Transformation of <I>E. coli</I> XL1 blue with F64 (quickchanged P3(pSAm1))</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 1 µl of DNA was added to 50 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube.</LI></UL><UL><LI> The cell suspension was plated on ampicillin plates (inclusive rescue plate) and incubated over night at 37 °C.</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Gelextraction_of_F67.28BirA.29.2C_F68.28mRuby.29.2C_F69.28EGFR-TMD.29.2C_F70.28pSB1C3.29_and_F71.28pSB1C3.29">Gelextraction of F67(BirA), F68(mRuby), F69(EGFR-TMD), F70(pSB1C3) and F71(pSB1C3)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of the experiment:</B> Gelextraction of F67(BirA(Digest. F59 [EcoRI; SpeI])), F68(mRuby(Digest. F60 [NgoMIV; SpeI])), F69(EGFR-TMD(Digest. F60 [NgoMIV; SpeI]), F70(pSB1C3(digest. P74 [NgoMIV; SpeI]) and F71(pSB1C3(digest. P74 [EcoRI; SpeI])
</P><P><B>Procedure:</B></P><P>Gelextraction was performed by manufacturers protocol (Qiagen).
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_F59_.28BirA.29_and_F60_.28EGFR-TMD.29_into_pSB1C3">Cloning of F59 (BirA) and F60 (EGFR-TMD) into pSB1C3</SPAN></H3><P><B>Investigator: CR</B></P><P><B>Aim of the experiment:</B> Generate biobricks with EGFR-TMD and BirA
</P><P><B>Procedure:</B></P><UL><LI> Digestions were prepared as following:</LI><LI> 1. xµl F59 (BirA); 1 µl EcoRI; 1 µl SpeI; 3 µl CutSmart; 23 µl ddH2O</LI><LI> 2. 2 µl P74 (pSB1C3 (RFP-generator); 1 µl NgoMIV; 1 µl SpeI; 3 µl CutSmart; 23 µl ddH2O</LI><LI> 3.  xµl F60 (mRuby/EGFR); 1 µl NgoMIV; 1 µl SpeI; 3 µl CutSmart; 23 µl ddH2O</LI><LI> 4. 2 µl P74 (pSB1C3 (RFP-generator), 1 µl EcoRI; 1 µl SpeI; 3 µl CutSmart; 23 µl ddH2O</LI><LI> Digestion was incubated for 3 h at 37 °C</LI></UL><P>To Do:
</P><UL><LI> Gel (already prepared)</LI><LI> Gelextraction</LI><LI> Ligation</LI><LI> Trafo</LI></UL><P><B>Results:</B></P><P>1. band: P74(NgoMiV, SpeI) 2. band: P74 (EcoRI, SpeI) 
</P><P>1. band: F59 	2. band: F60 
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_EGFR-TMD-pSB1C3_and_BirA-pSB1C3_ligations">Transformation of EGFR-TMD-pSB1C3 and BirA-pSB1C3 ligations</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Procedure:</B> According to protocol. Plates were incubated overnight at 37°C.
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Chemical_biotinylation_of_BSA">Chemical biotinylation of BSA</SPAN></H3><P><B>Investigator: MT, JB</B></P><P><B>Procedure:</B> 
BSA was chemically biotinylated with a 20x, 40x and 60x molar excess in two different buffers:100 mM Bicin (pH 8.85) and 100 mM borate (pH 8.85) with 50 mM NaCl respectively. 
</P><P>-&gt; A total of six different tubes. 
</P><P>After one hour incubation at RT, the samples were put in the freezer
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_liquid_cultures_.28F65_and_F66.29">Inoculation of liquid cultures (F65 and F66)</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Procedure:</B></P><UL><LI> Clones were picked for inoculation of 5 ml liquid cultures containing the appropriate antibiotic</LI><LI> Incubation overnight at 37°C/shaking</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_June_25th"><B>Saturday, June 25th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_E._coli_Xl1-Blue_transformed_with_P60_.28mRuby.2FEGFR.29.2CF58_.28Ligation_pASK75_.2B_Streptactin.29.2C_F65_.28CMV_.2B_CD4.29.2C_F66_.28CMV_.2B_EGFR.29.2C_P70_.28Short_Linker.29">Miniprep of E. coli Xl1-Blue transformed with P60 (mRuby/EGFR),F58 (Ligation pASK75 + Streptactin), F65 (CMV + CD4), F66 (CMV + EGFR), P70 (Short Linker)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)</LI><LI> Concentrations:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>c [ng/µl]</B></TD></TR><TR><TD>P87
</TD><TD>81
</TD></TR><TR><TD>P88
</TD><TD>34,5
</TD></TR><TR><TD>P89
</TD><TD>86,3
</TD></TR><TR><TD>P90
</TD><TD>108,5
</TD></TR><TR><TD>P91
</TD><TD>417,4
</TD></TR></TBODY></TABLE></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_F64_.28quickchanged_P3.2C_pASK75.28SA1.29.29_for_verification_of_success_of_quickchange">Analytical digestion and gelelectrophoresis of F64 (quickchanged P3, pASK75(SA1)) for verification of success of quickchange</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> Analytical digestion with NdeI and gelelectrophoresis. If quickchange worked there should be a band at about 3200 bp (only one restriction site left)</LI></UL><UL><LI> Incubation over night at room temperature.</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.</LI></UL><P><B>Results:</B></P><P>Just a band showing a few bp (Primer), there is no plasmid band -&gt; Quickchange did not work
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_colonies_from_Ligation_of_F69_.2B_F70_.28EGFR-TMD_in_pSB1C3.29_and_F44_.2B_F33_.28mRuby_in_pSB1C3.29">Inoculation of colonies from Ligation of F69 + F70 (EGFR-TMD in pSB1C3) and F44 + F33 (mRuby in pSB1C3)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> 6x 4 ml LB+Cam media </LI></UL><UL><LI> Each culture was inoculated with one colony</LI></UL><UL><LI> Incubation at 37°C overnight</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_June_26th"><B>Sunday, June 26th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_E._coli_Xl1-Blue_transformed_with_ligation_product_F44_.2B_F30_.28mRuby3_in_pSB1C3.29.2C_F66_.2B_F70_.28CMV.2BEGFR_in_pSB1C3.29">Miniprep of E. coli Xl1-Blue transformed with ligation product F44 + F30 (mRuby3 in pSB1C3), F66 + F70 (CMV+EGFR in pSB1C3)</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim of the experiment:</B> Extracting F44 + F30 (mRuby3 in pSB1C3), F66 + F70 (CMV+EGFR in pSB1C3) from E.coli XL-1-blue 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)</LI><LI> Concentrations:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>c [ng/µl]</B></TD></TR><TR><TD>P92
</TD><TD>162,9
</TD></TR><TR><TD>P93
</TD><TD>447,9
</TD></TR></TBODY></TABLE></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Repetition_of_Quick-Change_PCR_of_P3_.28pASK_.2B_SAm1.29">Repetition of Quick-Change PCR of P3 (pASK + SAm1)</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Procedure:</B></P><UL><LI> The QC-PCR was performed according the SOP.</LI></UL><UL><LI> Reaction Mix:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1,25 µl 
</TD><TD>Primer O21
</TD></TR><TR><TD>1,25 µl 
</TD><TD>Primer O22
</TD></TR><TR><TD>1 µl 
</TD><TD> dNTP-mix
</TD></TR><TR><TD> 5 µl 
</TD><TD>Pfu-Ultra-II reaction buffer
</TD></TR><TR><TD>1 µl 
</TD><TD> template DNA (1:10 dilution of p3)
</TD></TR><TR><TD> 0,5 µl
</TD><TD>Pfu-Ultra-II Polymerase
</TD></TR><TR><TD>40,5 µl 
</TD><TD> ddH2O
</TD></TR></TBODY></TABLE><UL><LI> digestion of PCR-Product with DpnI for 1h at 37°C</LI></UL><UL><LI> now labeled P94</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_P94_.28quickchanged_P3.28pSAm1.29.29">Transformation of <I>E. coli</I> XL1 blue with P94 (quickchanged P3(pSAm1))</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 10 µl of DNA was added to 100 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube.</LI></UL><UL><LI> The cell suspension was plated on ampicillin plates (inclusive rescue plate) for pASK (F72) and on chloramphenicol plates for P92 and P93 and incubated over night at 37 °C in the incubator.</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P92_.28mRuby.29.2C_P93_.28CMV_.2B_EGFR.29_and_P94_.28pASK75.29">Analytical digestion and gelelectrophoresis of P92 (mRuby), P93 (CMV + EGFR) and P94 (pASK75)</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P92 (mRuby), P93 (CMV + EGFR) and P94 (pASK75).
</P><P><B>Procedure:</B></P><UL><LI> Batch for analytical digestion for P92 and P93 with EcoRI-HF and PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>1 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF(10 U/µl) and PstI-HF (10 U/µl) for P92/93, NdeI (10 U/µl) for P94
</TD></TR><TR><TD>7.5/7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI> 1. band (P92): no mRuby at 700bp visible, only empty vector </LI><LI> 2. band (P93): empty vector and EGFR --&gt; perfect</LI><LI> 3. band (P94): no signal at all --&gt; repetition of QC-PCR</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_7_.28June_27th_-_July_3rd.29">Week 7 (June 27th - July 3rd)</SPAN></H1><DIV class="week" id="Week_7"><H2><SPAN class="mw-headline" id="Monday.2C_June_27th"><B>Monday, June 27th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Sequencing_of_P67_.28EGFR-Signalpeptid.29">Sequencing of P67 (EGFR-Signalpeptid)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><P>Sequencing batch was prepared after manufacturer's protocol. (15 µl of plasmid DNA (100 ng) and 2 µl sequencing primer (VF2))
</P><P>FR11326586
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Repetition_of_Quick-Change_PCR_of_P3_.28pASK_.2B_SAm1.29_2">Repetition of Quick-Change PCR of P3 (pASK + SAm1)</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Procedure:</B></P><UL><LI> The QC-PCR was performed according the SOP.</LI></UL><UL><LI> Reaction Mix:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1,25 µl 
</TD><TD>Primer O21
</TD></TR><TR><TD>1,25 µl 
</TD><TD>Primer O22
</TD></TR><TR><TD>1 µl 
</TD><TD> dNTP-mix
</TD></TR><TR><TD> 5 µl 
</TD><TD>Pfu-Ultra-II reaction buffer
</TD></TR><TR><TD>1 µl 
</TD><TD> template DNA (1:10 dilution of p3)
</TD></TR><TR><TD> 0,5 µl
</TD><TD>Pfu-Ultra-II Polymerase
</TD></TR><TR><TD>40,5 µl 
</TD><TD> ddH2O
</TD></TR></TBODY></TABLE><UL><LI> Digestion of PCR-Product with DpnI for 1h at 37°C.</LI></UL><UL><LI> Transformation of 10µl into component E.coli XL-1-blue, according to SOP (1h incubation at 37°C necessary despite AmpR). </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_of_Genesynthesis_3_and_4">PCR of Genesynthesis 3 and 4</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim of Experiment: Amplification of Genesynthesis 3 (contains BAP and IGKappa) and 4 (contains A3C5-tag and BM40)</B></P><P><B>Procedure:</B></P><UL><LI> The PCR was performed according the SOP.</LI></UL><UL><LI> Reaction Mix:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD> 2,5 µl 
</TD><TD>Primer VF2
</TD></TR><TR><TD> 2,5 µl 
</TD><TD>Primer VR2
</TD></TR><TR><TD> 1 µl 
</TD><TD> dNTP-mix
</TD></TR><TR><TD> 10 µl 
</TD><TD> Q5 Polymerase reaction buffer
</TD></TR><TR><TD>1 µl 
</TD><TD> template DNA (1:10 dilution of p3)
</TD></TR><TR><TD> 0,5 µl
</TD><TD> Q5-Polymerase
</TD></TR><TR><TD> 18 µl 
</TD><TD> ddH2O
</TD></TR></TBODY></TABLE><UL><LI>Setup: iGEM_standard (Promega-cycler)</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>temperature</B></TD><TD><B>time</B></TD></TR><TR><TD> 98°C 
</TD><TD> 2min
</TD></TR><TR><TD> 98°C 
</TD><TD> 10sec
</TD></TR><TR><TD> 66°C
</TD><TD> 30sec
</TD></TR><TR><TD> 72°C
</TD><TD> 30sec
</TD></TR><TR><TD> 72°C 
</TD><TD> 2min
</TD></TR><TR><TD> 4°C
</TD><TD> hold
</TD></TR></TBODY></TABLE><UL><LI> the batches were then purified using the Quiagen PCR-Purification Kit. </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P88_.28pASK75.28Steptactin.29.29.2C_P89_.28pSB1C3.28CMV.2BCD4.29_and_P90_.28pSB1C3.28CMV.2BEGFR-Signal_peptide.29">Analytical digestion and gelelectrophoresis of P88 (pASK75(Steptactin)), P89 (pSB1C3(CMV+CD4) and P90 (pSB1C3(CMV+EGFR-Signal peptide)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of experiment: Analytical digestion and gelelectrophoresis of P88 (pASK75 + Streptactin, former F58), P89 (CMV + CD4, former F65) and P90 (CMV + EGFR-signal-peptide, former F66)</B></P><P><B>Procedure:</B></P><UL><LI> Batches for analytical digestions:</LI></UL><P>P88: EcoRI
</P><P>P89: EcoRI and PstI
</P><P>P90: EcoRI
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5,8/2,3/1,8  µl
</TD><TD>Plasmid DNA (P88/P89/P90)
</TD></TR><TR><TD>1 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF(10 U/µl)/ PstI
</TD></TR><TR><TD>required amount for total volume of 10 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Ligation_of_F67_.28BirA.29_and_F71_.28vector_fragment_pSB1C3.29.2C_Transformation_of_E..C2.A0coli_XL1_blue_afterwards">Ligation of F67 (BirA) and F71 (vector fragment pSB1C3), Transformation of <I>E. coli</I> XL1 blue afterwards</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of the experiment:</B> Ligation of F67 (BirA) and F71 (empty pSB1C3), Transformation of <I>E. coli</I> XL1 blue afterwards.
</P><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,4 µl
</TD><TD>Vektor
</TD></TR><TR><TD>7,6 µl
</TD><TD>Insert
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were taken out of stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 7 µl of DNA was added to 100 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of 750 µl LB-medium to each tube.</LI></UL><UL><LI> Incubation for 1 hour at 37 °C</LI></UL><UL><LI> The cell suspension was plated on Cam-plates (inclusive rescue plate) and incubated over night at 37 °C in the cell-culture shaker.</LI></UL><UL><LI> next step: analytic digestion of transformation was successful</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Digestion_of_PCR_on_genesynthesis_3_and_4.2C_and_pSB1C3">Digestion of PCR on genesynthesis 3 and 4, and pSB1C3</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim of experiment: Division of Leptin, IGKappa, A3C5, BM40 and BAP using SapI, HindIII, XbaI, AgeI for both batches.</B></P><P><B>Procedure:</B></P><UL><LI> Batches for analytical digestions:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl each
</TD><TD>enzyme (SapI, HindIII, XbaI, AgeI)
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>41 µl
</TD><TD>DNA (purified PCR-products of GSY3 and 4)
</TD></TR></TBODY></TABLE><UL><LI> Additionally 10µg of the vector P74 was digested with XbaI and AgeI in 100µl batch (2µl of each enzyme, 10µl of Cut-Smart buffer). Digestion was performed over night and purified via gelelectrophoresis and gelextraction according to the manufacturer's protocoll. --&gt; Now labeled F80. </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P80_.2C_P78_and_P85">Preparative digestion and gelelectrophoresis of P80 , P78 and P85</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of experiment: Preparative digestion and gelelectrophoresis of P80 (mRuby3), P78 (NanoLuc) and P85 (Strep-Tag)</B></P><P><B>Procedure:</B></P><UL><LI> Batches for analytical digestions:</LI></UL><UL><LI>P80 and P78: EcoRI and AgeI</LI></UL><UL><LI>P85: EcoRI and NgoMIV</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10  µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>32  µl
</TD><TD>ddH2O
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>each enzyme(10 U/µl)/ PstI
</TD></TR><TR><TD>50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Ligation_of_F75_.28mRuby.29_and_F76_.28NanoLuc.29_into_F74_.28pSB1C3_with_Strep-Tag.29_and_Transformation_into_E..C2.A0coli_XL1_blue">Ligation of F75 (mRuby) and F76 (NanoLuc) into F74 (pSB1C3 with Strep-Tag) and Transformation into <I>E. coli</I> XL1 blue</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim of the experiment:</B> Ligation of F75 (mRuby) and F76 (NanoLuc) into F74 (pSB1C3+Strep-Tag), Transformation of <I>E. coli</I> XL1 blue afterwards.
</P><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4,3µl(for F75), 8,2µl (for F76)
</TD><TD>Vector
</TD></TR><TR><TD>12,7µl (F75), 8,8 (F76)
</TD><TD>Insert
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Ligation was incubated at RT for 1,5h.</LI></UL><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were taken out of stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 7 µl of DNA was added to 100 µl of competent cells and gently mixed.</LI></UL><UL><LI> 15 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube.</LI></UL><UL><LI> Incubation for 1 hour at 37 °C</LI></UL><UL><LI> The cell suspension was plated on Cam-plates (inclusive rescue plate) and incubated over night at 37 °C in the incubator.</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Chemical_biotinylation_of_BSA_2">Chemical biotinylation of BSA</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> BSA was chemically biotinylated with a 20x and 40x molar excess:</LI></UL><UL><LI> 10 ml of 100 mM borate buffer with 50 mM NaCl (pH 8.85)</LI></UL><UL><LI> dissolve BSA (10 mg/ml)</LI></UL><UL><LI> Add biotin-NHS-ester: 20,5 mg for 40x molar excess </LI></UL><UL><LI> reaction over night</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_June_28th"><B>Tuesday, June 28th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P83_.28EspP.29_and_P29_.28DoubleTerminator.29">Preparative digestion and gelelectrophoresis of P83 (EspP) and P29 (DoubleTerminator)</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of experiment: Preparative digestion and gelelectrophoresis of P83 (EspP) and P29 (DoubleTerminator)</B></P><P><B>Procedure:</B></P><UL><LI> Batches for analytical digestions:</LI></UL><UL><LI>P83: EcoRI and SpeI</LI></UL><UL><LI>P29: EcoRI and XbaI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6.5/13  µl
</TD><TD>Plasmid DNA (P83/P29)
</TD></TR><TR><TD>35.5/29  µl
</TD><TD>ddH2O (P83/P29)
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>each enzyme
</TD></TR><TR><TD>50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Preparative gelelectrophoresis was performed at 80 V for 80 min</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Gelelectrophoresis_of_F73_.28pSB1C3_digst._EcoRI.2C_AgeI.29">Gelelectrophoresis of F73 (pSB1C3 digst. EcoRI, AgeI)</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of experiment: gelelectrophoresis of P80 (mRuby3), P78 (NanoLuc) and P85 (Strep-Tag)</B></P><P><B>Procedure:</B>
Preparative gelelectrophoresis was performed at 100 V for 45 min using a pocket spanning over the full width of the 1% agarose gel
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_June_29th"><B>Wednesday, June 29th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_F83_.28EspP-Fragment_in_Double_Terminator_.2B_pSB1C3.29.2C_F84_.28GSY3_.2B_pSB1C3.29_F85_.28GSY4_.2B_pSB1C3.29">Transformation of F83 (EspP-Fragment in Double Terminator + pSB1C3), F84 (GSY3 + pSB1C3) F85 (GSY4 + pSB1C3)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 7 µl of DNA was added to 50 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube.</LI></UL><UL><LI> The cell suspensions were plated on Cam-plates (inclusive rescue plate) and incubated over night at 37 °C.</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_June_30th"><B>Thursday, June 30th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_new_E..C2.A0coli_XL1-blue_.28P75.29.2C_additional_plating_of_untransformated_XL1-blue">Transformation of new <I>E. coli</I> XL1-blue (P75), additional plating of untransformated XL1-blue</SPAN></H3><P><B>Aim of experiment: Quality assurance</B></P><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> the new competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 1 µl of DNA was added to 50 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube.</LI></UL><UL><LI> The cell suspension was plated on Cam-plates (inclusive rescue plate) and incubated over night at 37 °C.</LI></UL><UL><LI> additional plating of untransformated XL1-blue on Tet-, Cam-, Amp- and Kan-plates</LI></UL><P><B>Result:</B></P><UL><LI> After 12 hous of incubation there were some (~20) colonies on the 50µl-plate transformed with Cam-resistence-plasmids. The transformation has been succesful, though cells grow slow. </LI><LI> There were no colonies on the kan-, amp- and cam- plates with the untransformed E.Coli. Hence we can assume that there are no other plasmids with resistences  against kan, amp or cam in our batch of competent cells.</LI></UL><P>There are cells on the tet-plate. This was expected.
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_June_31th"><B>Friday, June 31th</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Transformation_of_P9_.28pASK_with_Streptavidin_WT.29_and_QC-PCR_product_into_new_E..C2.A0coli_XL1-blue">Transformation of P9 (pASK with Streptavidin WT) and QC-PCR product into new <I>E. coli</I> XL1-blue</SPAN></H3><P><B>Aim of experiment: Tranforming pASK without NdeI-cutting point into E.coli (two batches from diffrent approches were tested). And Transforming pASK_SA1 into E.coli XL-1-blue.</B></P><P><B>Investigator: Luisa</B></P><P><B>Procedure:</B></P><UL><LI> the new competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 10 µl of DNA was added to 50 µl of competent cells and gently mixed.</LI></UL><UL><LI> 15 min incubation on ice</LI></UL><UL><LI> 30 sec. heat shock at 42 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube. 1h hour incubation at 37°C. </LI></UL><UL><LI> The cells were plated out on Amp-plates (inclusive rescue plate) and incubated over the weekend at 30 °C.</LI></UL></DIV><H1><SPAN class="mw-headline" id="Week_8_.28July_04.E2.80.9310.29">Week 8 (July 04–10)</SPAN></H1><DIV class="week" id="Week_8"><H2><SPAN class="mw-headline" id="Monday.2C_July_04"><B>Monday, July 04</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Repetition_of_the_Transformation_of_QC-PCR_product_into_new_E..C2.A0coli_XL1-blue">Repetition of the Transformation of QC-PCR product into new <I>E. coli</I> XL1-blue</SPAN></H3><P><B>Aim of experiment: Tranforming pASK without NdeI-cutting point into E.coli (two batches from diffrent approches were tested).</B></P><P><B>Investigator: Luisa</B></P><P><B>Procedure:</B></P><UL><LI> the new competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 10 µl of DNA was added to 50 µl of competent cells and gently mixed.</LI></UL><UL><LI> 15 min incubation on ice</LI></UL><UL><LI> 30 sec. heat shock at 42 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube. 1h hour incubation at 37°C. </LI></UL><UL><LI> The cells were plated out on Amp-plates (inclusive rescue plate) and incubated over the weekend at 30 °C.</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Transformation_of_P9_.28Streptavidin_in_pASK75.29_into_E..C2.A0coli_BL-21">Transformation of P9 (Streptavidin in pASK75) into <I>E. coli</I> BL-21</SPAN></H3><P><B>Aim of experiment: Tranforming pASK with Streptavidin into E.coli BL-21 for Expression.</B></P><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> the new competent <I>E. coli</I> BL-21 cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 1 µl of DNA was added to 50 µl of competent cells and gently mixed.</LI></UL><UL><LI> 15 min incubation on ice</LI></UL><UL><LI> 5&amp;nbspmin. heat shock at 37 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube. </LI></UL><UL><LI> The cells were plated out on Amp-plates (inclusive rescue plate) and incubated over the weekend at 30 °C</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_F94_.28hGH_polyadenylation_sequence_and_Nanoluc_with_StrepTag.29">Transformation of F94 (hGH polyadenylation sequence and Nanoluc with StrepTag)</SPAN></H3><P><B>Investigator: Javier</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 7 µl of DNA was added to 50 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 45 sec. heat shock at 42 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium.</LI></UL><UL><LI> The cell suspensions were plated on Cam-plates (inclusive rescue plate) and incubated over night at 37 °C.</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_E._coli_Xl1-Blue_transformed_with_ligation_product_P108_and_P113_.28EspP-Fragment_in_Double_Terminator.2C_F81.2BF82.29.2C_P109_and_P110_.28GSY3.2B_F73_vector_AgeI.2BXbaI_.29.2C_P111_and_P112_.28GSY4.2B_F73_vector_AgeI.2BXbaI.29">Miniprep of E. coli Xl1-Blue transformed with ligation product  P108 and P113 (EspP-Fragment in Double Terminator, F81+F82), P109 and P110 (GSY3+ F73 vector AgeI+XbaI ), P111 and P112 (GSY4+ F73 vector AgeI+XbaI)</SPAN></H3><P><B>Investigator: Javier</B></P><P><B>Aim of the experiment:</B> Extracting P108 and P113 (EspP-Fragment in Double Terminator, F81+F82), P109 and P110 (GSY3+ F73 vector AgeI+XbaI ), P111 and P112 (GSY4+ F73 vector AgeI+XbaI) from E.coli XL-1-blue 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)</LI><LI> Concentrations:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>c [ng/µl]</B></TD></TR><TR><TD>P108
</TD><TD>116,2
</TD></TR><TR><TD>P109
</TD><TD>86,6
</TD></TR><TR><TD>P110
</TD><TD>136,8
</TD></TR><TR><TD>P111
</TD><TD>144,8
</TD></TR><TR><TD>P112
</TD><TD>163,2
</TD></TR><TR><TD>P113
</TD><TD>123,3
</TD></TR></TBODY></TABLE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P74_.28RFP-Gen..29">Preparative digestion and gelelectrophoresis of P74 (RFP-Gen.)</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of experiment: Preparative digestion and gelelectrophoresis of P74 (RFP-Gen.) </B></P><P><B>Procedure:</B></P><UL><LI> Batch for analytical digestion:</LI></UL><UL><LI>P74: EcoRI and SpeI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>39.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>each enzyme
</TD></TR><TR><TD>50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Preparative gelelectrophoresis was performed at 90 V for 45 min</LI></UL><P><B>Result: </B>pSB1C3 (EcoRI und SpeI) F95
</P><P><A href="/File:Muc16_F95_0407.jpeg" title="File:Muc16 F95 0407.jpeg">File:Muc16 F95 0407.jpeg</A></P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_F96_.28BirA_in_pSB1C3.29">Cloning of F96 (BirA in pSB1C3)</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 10 µl of DNA was added to 50 µl of competent cells and gently mixed.</LI></UL><UL><LI> 5 min incubation on ice</LI></UL><UL><LI> 30 s heat shock at 37 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium</LI></UL><UL><LI> Incubation at 37 °C</LI></UL><UL><LI> The cell suspension was plated on Cam-plates (inclusive rescue plate) and incubated over night at 37 °C</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_05th"><B>Tuesday, July 05th</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Preparing_of_200_ml_TB_medium_for_inoculation_of_2_L_medium_on_Thursday">Preparing of 200 ml TB medium for inoculation of 2 L medium on Thursday</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> 2.4 g Tryptone 4.8 g yeast extract</LI></UL><UL><LI> 800 µl Glycerine</LI></UL><UL><LI> adjust to 180 ml with ELGA-water</LI></UL><UL><LI> seperately: 3.29 g H<SUB>2</SUB>KPO<SUB>4</SUB> + 0,46 g K<SUB>2</SUB>HPO<SUB>4</SUB> with ELGA water (Volume = 20 ml) </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Digestion_of_F49_.28BirA.29_with_EcoRI_and_SpeI_and_cloning_in_F95_.28pSB1C3.29">Digestion of F49 (BirA) with EcoRI and SpeI and cloning in F95 (pSB1C3)</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Procedure:</B></P><UL><LI>Digestion of F49 with EcoRI and SpeI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA (F49)
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF(10 U/µl)
</TD></TR><TR><TD>1  µl
</TD><TD>SpeI(10 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Digestion was incubated for 1.5 h at 37°C</LI><LI>Digestion was applied to a 1% agarose-gel</LI><LI>Gel ran at 80V for 35 min</LI></UL><P><A href="/File:Muc16_digestF49_0507.jpeg" title="File:Muc16 digestF49 0507.jpeg">File:Muc16 digestF49 0507.jpeg</A></P><UL><LI>BirA had a length of 1012 bp</LI><LI>Upper probe was cut from the gel</LI><LI>BirA was purified with gelextraction which was performed by manufacturer's protocol</LI><LI>Concentration was measured
</LI><LI>[F97]= 12.4</LI></UL><LI>Ligation of F97 into pSB1C3 (F95)</LI><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.2 µl
</TD><TD>Vector: pSB1C3 (F95)
</TD></TR><TR><TD>8.8 µl
</TD><TD>Insert: BirA (F97)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 DNA Ligase buffer
</TD></TR><TR><TD>fill up to 20 µl
</TD><TD>dd H<SUB>2</SUB>O
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase
</TD></TR><TR><TD>=21 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Ligation was incubated at RT for 30 min</LI><LI>Transformation:
</LI><LI>Transformation was performed after SOP</LI><LI>7 µl were used for transformation in E. <I>coli</I> XL1blue</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Sequencing_of_P108_-_P112">Sequencing of P108 - P112</SPAN></H3><P><B>Aim of experiment: Sequencing </B></P><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> sequencing batches after protocol with following barcodes:</LI></UL><UL><LI> P108 (EspP-Fragment in Double Terminator (F83)) FR11326572 (VF2) </LI></UL><UL><LI> P109 (GSY3+ F73 Leervektor AgeI+XbaI (F84)) FR11326576 (VF2) </LI></UL><UL><LI> P110 (GSY3+ F73 Leervektor AgeI+XbaI (F84)) FR11326575 (VF2) </LI></UL><UL><LI> P111 (GSY4+ F73 Leervektor AgeI+XbaI (F84)) FR11326574 (VF2) </LI></UL><UL><LI> P112 (GSY4+ F73 Leervektor AgeI+XbaI (F84)) FR11326573 (VF2) </LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="NEB_Turbo_CaCl2-competent_cells">NEB Turbo CaCl<SUB>2</SUB>-competent cells</SPAN></H3><P><B>Investigator: Vivien</B></P><P><B>Procedure:</B></P><UL><LI> Inoculate 50 mL LB with 0.5 mL overnight culture of NEB Turbo cells</LI></UL><UL><LI> Incubate at 37 °C for 7 h; final OD<SUB>500</SUB>: 0.357</LI></UL><UL><LI> Very little growth in the course of the last hour; OD<SUB>500</SUB> 0.45 required</LI></UL><UL><LI> Experiment discontinued. Inoculated 3 further colonies in 5 mL LB for o/n growth (37 °C). Repeat experiment on Wed, July 6</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_P114_.28NanoLuc_.2B_StrepTag_.2B_Poly-A_in_pSB1C3.28F94.29.29">Miniprep P114 (NanoLuc + StrepTag + Poly-A in pSB1C3(F94))</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI>  Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P114_and_P115">Analytical digestion and gelelectrophoresis of P114 and P115</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of the experiment:</B></P><UL><LI> Analytical digestion and gelelectrophoresis of P114 and P115 (NanoLuc + StrepTag + Poly-A in pSB1C3 K1 and K2)</LI></UL><P><B>Procedure:</B></P><UL><LI> Batch for analytical digestion for P114 and P115 with EcoRI+PstI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.75 µl
</TD><TD>Plasmid DNA P114/P115
</TD></TR><TR><TD>1 µl
</TD><TD>cut smart (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI
</TD></TR><TR><TD>6,25 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 45 min at 37 °C.</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.</LI></UL><P><B>Results:</B></P><P><A href="/File:Muc16_digestP114_P115_0507.jpeg" title="File:Muc16 digestP114 P115 0507.jpeg">File:Muc16 digestP114 P115 0507.jpeg</A></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder DNA ladder
</TD><TD><B>P114</B></TD><TD><B>P115</B></TD></TR><TR><TD><B>X</B></TD><TD><B>X</B></TD></TR></TBODY></TABLE></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inoculation_of_Quick_change_of_P88_.28pASK75_.28Streptactin.29.29_preculture">Inoculation of Quick change of P88 (pASK75 (Streptactin)) preculture</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Procedure:</B></P><UL><LI> four colonies were picked from plate</LI><LI> Cells were inoculated in 5 mL LB medium (ampicillin 1:1000)</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P108_and_P113_.28both_EspP_.2B_double_terminator_.28F83.29.29_with_EoRI_and_PstI">Analytical digestion of P108 and P113 (both EspP + double terminator (F83)) with EoRI and PstI</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Procedure:</B></P><UL><LI>Digestion of F49 with EcoRI and SpeI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P108/P113)
</TD></TR><TR><TD>1 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF(10 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI(10 U/µl)
</TD></TR><TR><TD>7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Digestion was incubated for 1.5 h at 37°C</LI><LI>Digestion was applied to a 1% agarose-gel</LI><LI>Gel ran at 80V for 35 min</LI></UL><P><A href="/File:Muc16_analyt.digest.P108_P113_0507.jpeg" title="File:Muc16 analyt.digest.P108 P113 0507.jpeg">File:Muc16 analyt.digest.P108 P113 0507.jpeg</A></P><UL><LI>P108 and P113 showed the right bands</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_July_06th"><B>Wednesday, July 06th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Sequencing_of_P108_.28EspP-Fragment_in_Double_Terminator_.28F83.29.29">Sequencing of P108 (EspP-Fragment in Double Terminator (F83))</SPAN></H3><P><B>Investigator: &gt;Niklas</B></P><P><B>Procedure:</B></P><UL><LI> sequencing batches after protocol with following barcodes:</LI></UL><UL><LI> P108 (EspP-Fragment in Double Terminator (F83)) FR11326572 (VR) </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Digestion_of_P75_.28RFP-Generator.29_with_HincII_.2B_dephosphorylation">Digestion of P75 (RFP-Generator) with HincII + dephosphorylation</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of experiment:</B> Blunt-Ligation of BirA into pSB1C3
</P><P><B>Procedure:</B></P><UL><LI>Digestion of P75 (RFP-Generator) with HincII + dephosphorylation</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>Plasmid DNA (P75)
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>HincII
</TD></TR><TR><TD>1 µl
</TD><TD>Fast AP (for dephosphorylation)
</TD></TR><TR><TD>14.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Digestion was incubated for over night at 37°C</LI></UL><UL><LI>next steps: PCR of GSY2 with O17 and O18 + phosphorylation, Ligation in digested pSB1C3 afterwards</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_A3C5_Tag_into_the_MiddleLinker-Plasmid">Cloning of A3C5 Tag into the MiddleLinker-Plasmid</SPAN></H3><P><B>Investigator: Christoph</B></P><P><B>Aim of experiment:</B> Cloning of A3C5 Tag into the MiddleLinker-Plasmid
</P><P><B>Procedure:</B></P><UL><LI>Digestion of Middle_Linker Plasmid (P69) with AgeI and PstI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.3 µl
</TD><TD>Plasmid DNA (69)
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI
</TD></TR><TR><TD>1 µl
</TD><TD>PstI
</TD></TR><TR><TD>1 µl
</TD><TD>Fast AP (for dephosphorylation)
</TD></TR><TR><TD>14.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.3 µl
</TD><TD>Plasmid DNA (P112)
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgomIV
</TD></TR><TR><TD>1 µl
</TD><TD>PstI
</TD></TR><TR><TD>14.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Digestion was incubated for over night at 37°C, heat inactivation at 80°C for 20 min.</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>digested Middle-Linker (F99)
</TD></TR><TR><TD>8 µl
</TD><TD>digested A3C5 (F100)
</TD></TR><TR><TD>2 µl
</TD><TD>10x Ligase Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Transformation according to SOP in E. coli XL1 blue</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_with_Streptactin_in_quickchanged_pASK75.2C_F84_.28digested_GSY3.29.2C_F85_.28digested_GSY4.29">Picking of of <I>E. coli</I> XL1 blue with Streptactin in quickchanged pASK75, F84 (digested GSY3), F85 (digested GSY4)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> Streptactin in quickchanged pASK75: Colonies were picked from Amp-plates. (10x)</LI></UL><UL><LI> F84/F85: Colonies were picked from Cam-plates. (8x each)</LI></UL><UL><LI> Picked pipette tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contains 4 mL of LB-medium + 4 µL of the proper antibiotic.</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Quick_Change_of_mRuby_Plasmids">Quick Change of mRuby Plasmids</SPAN></H3><P><B>Investigator: Christoph</B></P><P><B>Aim of experiment:</B> Correction of wrong amino acid within the mRuby sequence
</P><P><B>Procedure:</B></P><P>QC according to manufacturers protocol
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 ng (= 0,3 µl 1:10)
</TD><TD>Plasmid DNA (80)
</TD></TR><TR><TD>5 µl
</TD><TD>10x Pfu Ultra II Buffer
</TD></TR><TR><TD>1.25 µl
</TD><TD>O23
</TD></TR><TR><TD>1.25 µl
</TD><TD>O24
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II
</TD></TR><TR><TD>1 µl
</TD><TD>dNTPs
</TD></TR><TR><TD>40.7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 ng (= 0,3 µl 1:10)
</TD><TD>Plasmid DNA (100)
</TD></TR><TR><TD>5 µl
</TD><TD>10x Pfu Ultra II Buffer
</TD></TR><TR><TD>1.25 µl
</TD><TD>O23
</TD></TR><TR><TD>1.25 µl
</TD><TD>O24
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II
</TD></TR><TR><TD>1 µl
</TD><TD>dNTPs
</TD></TR><TR><TD>40.7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>PCR: 2 min. 98°C initial denaturation, 10 sec. 98°C, 30 sec. Annealing st 60 °C, 5 min 68°C, 35 Cycles</LI></UL><UL><LI>1 h at 37°C DpnI Digestion</LI></UL><UL><LI>Transformation according to SOP in E. coli XL1 blue, QC on mRuby-Strep (F103) was also transformed in E. coli Turbo cells for further investigating the growth rate</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Miniprep_of_Quick_Change_of_pASK75-Streptactin_.2B_analytical_digestion_with_Nde1">Miniprep of Quick Change of pASK75-Streptactin + analytical digestion with Nde1</SPAN></H3><P><B>Investigator: Jan</B></P><P><B>Procedure:</B></P><UL><LI>Miniprep using the kit according to the manufacturer's protocol.</LI></UL><UL><LI>Although cultures seemed to have grown properly, very low and unreliable DNA concentrations were obtained via Nanodrop measurement and no fragment at all could be detected via analytical agarose gel electrophoresis after 90 minutes digestion with Nde1</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Creation_of_competent_NEB_Turbo_cells">Creation of competent NEB Turbo cells</SPAN></H3><P><B>Investigator: Jan</B></P><P><B>Procedure:</B></P><UL><LI> 250 µl of precultures were used to inoculate 50 ml LB-medium</LI><LI> After an OD=0.45 was reached, cells were put on ice and made competent according to the manufacturer's protocol</LI><LI> 50 µl aliquots of competent cells were frozen in liquid nitrogen and stored at -80°C until further use</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_a_BL21_pBAD-GFP_.28P60.29_preculture">Inoculation of a BL21 pBAD-GFP (P60) preculture</SPAN></H3><P><B>Investigator: Jan</B></P><P><B>Procedure:</B></P><UL><LI> A single clone was picked in order to inoculate 5 ml LB medium (+Cam) </LI><LI> Culture was grown overnight at 37°C</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inoculation_of_a_BL21_preculture_for_Streptavidin_production_.28P8.29">Inoculation of a BL21 preculture for Streptavidin production (P8)</SPAN></H3><P><B>Investigator: Jan</B></P><P><B>Procedure:</B></P><UL><LI> A clone was picked for the inoculation of 5 ml LB medium</LI><LI> After 6-8 hours of incubation, the 5 ml preculture was used for the inoculation of a 50 ml LB preculture</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_July_07"><B>Thursday, July 07</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_Amplification_of_birA_.28GSY_2.29">PCR Amplification of <I>birA</I> (GSY 2)</SPAN></H3><P><B>Investigator: Vivien</B></P><P><B>Aim:</B> Amplify <I>birA</I> gene
</P><P><B>Procedure:</B></P><UL><LI> Prepared PCR mix (25 μL reaction volume): NEB Q5 High-Fidelity 2× Master Mix (12.5 μL), ddH<SUB>2</SUB>O (9.5 μL), F29 (1 μL), O17 and O18 (1 μL each)</LI></UL><UL><LI> Cycling protocol: start 1.5 min, 98 °C; 35 cycles 30 s, 98 °C – 45 s, 72 °C – 30 s, 72 °C; final elongation 5 min, 72 °C; end 4 °C</LI></UL><UL><LI> Purified <I>via</I> Quiagen PCR Purification kit following manufacturer's instructions. DNA eluted from column by adding 50 μL EB buffer and heating at 37 °C for 5 min prior to centrifugation.</LI></UL><P><B>Results:</B> DNA concentration: 82.7 ng/μL. Stored as F102 at –20 °C
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Induction_of_Streptavidin">Induction of Streptavidin</SPAN></H3><P><B>Investigator: Vivien</B></P><P><B>Aim:</B> Induce streptavidin production 
</P><P><B>Procedure:</B></P><UL><LI> Inoculated 2 L TB medium with 50 mL o/n pre-culture and added 2 mL ampicillin (10 mg/mL in 50% EtOH)</LI><LI> Let grow (37 °C, 200 rpm) until OD<SUB>500</SUB>=0.622 (<I>c.</I> 2.5 h)</LI><LI> Induced with anhydrotetracycline (1:10,000 i.e. 200 μL of 2 mg/mL stock)</LI><LI> Let grow (37 °C, 200 rpm) for 4 h</LI></UL><P><B>Results:</B> Induced culture for harvesting (see below)
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Production_of_eGFP">Production of eGFP</SPAN></H3><P><B>Investigator: Clemens, Vivien, Christoph</B></P><P><B>Aim:</B> Produce eGFP
</P><P><B>Procedure:</B></P><UL><LI> Inoculated 50 mL LB+Cam medium with 0.5 mL o/n pre-culture</LI></UL><UL><LI> Let grow (37 °C, 200 rpm) until OD<SUB>500</SUB>=0.58</LI></UL><UL><LI> Induced with 0.1 mM arabinose (i.e. 100 μL of 50 mM stock)</LI></UL><UL><LI> After 7 h of induction the cells were stored in the fridge overnight for maturation of the GFP-chromophore</LI></UL><P><B>Results:</B></P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Harvesting_of_Streptavidin">Harvesting of Streptavidin</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of the experiment:</B> Recombinant expression and purification of streptavidin
</P><P><B>Procedure:</B></P><UL><LI> After expression, cultures were transferred into centrifuge tubes and spun down in the centrifuge (4 °C, 5000 rpm, 20 min, F4X1L rotor) </LI><LI> The supernatant was discarded, and the pellet was transferred into a Falcon and frozen</LI><LI> next steps: resuspending and homogenizing in the PANDA </LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_08"><B>Friday, July 08</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_Digest_of_mRuby3_Constructs">Analytical Digest of mRuby3 Constructs</SPAN></H3><P><B>Aim of experiment: Verify integration of mRuby3 constructs in F103 and F104</B></P><P><B>Investigator: Vivien</B></P><P><B>Procedure:</B></P><UL><LI> Mixed 16 μL F103 (or 8 μL F104 plus 8 μL ddH<SUB>2</SUB>O) with 2 μL 10× CutSmart buffer and 1 μL each of EcoRI-HF and PstI-HF (20 μL total volume)</LI></UL><UL><LI> Incubated for 45 min at 37 °C</LI></UL><UL><LI> Subjected to gel electrophoresis (1% agarose, 80 V, 120 min)</LI></UL><P><B>Results:</B> No band around 700 bp as expected for mRuby3 constructs
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Picking_clones_and_inoculating_cultures_for_F101_and_F103">Picking clones and inoculating cultures for F101 and F103</SPAN></H3><P><B>Investigator: Jan</B></P><P><B>Procedure:</B></P><UL><LI> All clones available (only two for F101, one for F103) were picked and used to inoculate 5 ml of LB medium containing the appropriate antibiotic</LI><LI> Cultures were incubated at 37°C overnight</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Solubilization.2C_lysis_and_unfolding_of_streptavidin">Solubilization, lysis and unfolding of streptavidin</SPAN></H3><P><B>Investigator: Jan</B></P><P><B>Procedure:</B></P><UL><LI> According to protocol -&gt; lysis of pellet (19.2 g) in 58 ml lysis buffer (50 mM Tris/HCl pH 8, 1 mM EDTA)</LI><LI> As only two liters of medium were used, the Panda couldn't be utilized for cell lysis -&gt; French Press was used</LI><LI> Cells were additionally submitted to ultrasonic treatment</LI><LI> Centrifugation of cells, casting away the supernatant and resolubilizing the pellet in 5 ml GdmCl (6M, pH 1.5)</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Sequencing_of_P88">Sequencing of P88</SPAN></H3><P><B>Aim of experiment: Verification of QC  </B></P><P><B>Investigator: Julian</B></P><P><B>Procedure:</B></P><P>Sequencing batch was prepared after manufacturer's protocol. (15 µl of plasmid DNA (100 ng) and 2 µl sequencing primer (PR1))
</P><P>FR11326565
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_P75_.28RFP-Generator.29.2C_F101_.28middle_linker.2B_A3C5.29.2C_P125_.28dig._GSY4.29_into_E..C2.A0coli_XL1-blue_and_P75_in_NEB_Turbo_E..C2.A0coli">Transformation of P75 (RFP-Generator), F101 (middle linker+ A3C5), P125 (dig. GSY4) into <I>E. coli</I> XL1-blue and P75 in NEB Turbo <I>E. coli</I></SPAN></H3><P><B>Investigator: Niklas, Vivien</B></P><P><B>Procedure:</B></P><UL><LI> the competent <I>E. coli</I> cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 1 µl of DNA (6 µl for Ligation F101) was added to 50 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5&amp;nbspmin. heat shock at 37 °C</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube. </LI></UL><UL><LI> The cells were plated out on Cam-plates/Amp-plate for F101 (inclusive rescue plate) and incubated over the weekend at 30 °C</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="QuickChange-PCR_of_P80_.28O23_and_O24.29_and_P100_.28O19_and_O20.29">QuickChange-PCR of P80 (O23 and O24) and P100 (O19 and O20)</SPAN></H3><P><B>Investigators: Niklas</B></P><P><B>Aim of the experiment:</B> Changing of amino acid       
</P><P><B>Procedure:</B></P><UL><LI> the batches were split, each batch contained only one primer, the batches were pooled after 15 cycles (1 µl Polymerase is added) </LI><LI> Annealing at 55°C</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.25µl
</TD><TD>O23/O24 and O19/O20
</TD></TR><TR><TD>0,5 µl
</TD><TD>dNTP-mix
</TD></TR><TR><TD>1 µl
</TD><TD>1:100 P80 / P100
</TD></TR><TR><TD>2.5 µl
</TD><TD>Pfu Ultra II-buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II Polymerase
</TD></TR><TR><TD>19.25 µl
</TD><TD>dd H<SUB>2</SUB>O
</TD></TR><TR><TD>4 x 25 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preperative_gelelektrophoresis_of_F102_.28PCR_of_BirA.29_.2B_phosphorylation_and_ligation_into_pSB1C3">Preperative gelelektrophoresis of F102 (PCR of BirA) + phosphorylation and ligation into pSB1C3</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim of experiment:</B> Blunt-Ligation of BirA into pSB1C3
</P><P><B>Procedure:</B></P><UL><LI>The whole PCR-product was applied to a 1% agorose gel. After runnig the gel two lines where visible: one at 1000Bp and one at ~500Bp.</LI></UL><UL><LI>The 1000 Bp band was cut out and purified with the Gelextraction kit according to manufacturer's protocoll. Concentration: 41,5ng/µl. </LI></UL><UL><LI>The DNA was phosphorylated as follows: </LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>DNA (~1pmol)
</TD></TR><TR><TD>2 µl
</TD><TD>Ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 Polynukleotide-Kinase (for phosphorylation)
</TD></TR><TR><TD>2 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>The batch was incubated for 20 min at 37°C and then the enzyme was inactivated for 10 min at 75°C. </LI></UL><UL><LI>The phosphorylated DNA was then ligated into prepared (HincII-digested) pSB1C3 as follows: </LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,3 µl
</TD><TD>Vector
</TD></TR><TR><TD>7,7 µl
</TD><TD>Insert (BirA)
</TD></TR><TR><TD>2 µl
</TD><TD>Ligase Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>next steps: Transformation into E.coli Xl-1-blue.</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Digestion_of_P78.2C_P123_.28for_BAP-Nanoluc-construct.29.2CP122_.28IgKappa_for_CMV-IgKappa-construct.29.2C_P89.2C_P114_.28for_CMV-EGFR-Nanoluc-StrepTag-PolyA-construct.29">Digestion of P78, P123 (for BAP-Nanoluc-construct),P122 (IgKappa for CMV-IgKappa-construct), P89, P114 (for CMV-EGFR-Nanoluc-StrepTag-PolyA-construct)</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim of experiment:</B> Getting of BAP-Nanoluc-construct, CMV-IgKappa-construct and CMV-EGFR-Nanoluc-StrepTag-PolyA-construct.
</P><P><B>Procedure:</B></P><UL><LI> DNA was digested according to the following table: </LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µg (for P78, P123 and P122) and 1,5 µg (for P89, P114)
</TD><TD>DNA
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>1,5 µl of NgoMIV and PstI
</TD><TD>for P78
</TD></TR><TR><TD>1,5 µl of AgeI and PstI
</TD><TD>for P123
</TD></TR><TR><TD>1,5 µl of XbaI and PstI
</TD><TD>for P122
</TD></TR><TR><TD>1,5 µl of AgeI and PstI
</TD><TD>for P189
</TD></TR><TR><TD>1,5 µl of NgoMIV and PstI
</TD><TD>for P123
</TD></TR><TR><TD>up to 50µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Results:</B> Concentrations: P78 now F106: 2,7 ng/µl, P89 now F107: 14,4 ng/µl. Gelextractions of P114, P122 and P123 didn't contain DNA. (To few DNA available) 
</P><UL><LI>next steps: Retransformation of verified clones (P114, P89) into Xl-1-blue for amplification of plasmid. </LI></UL><P>Repetition of digestion of P122 and P123 with more DNA (10µg!), then Gel with 100Bp ladder!
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Repetition_of_digestion_of_P123_.28BAP_for_BAP-Nanoluc-construct.29_and_P122_.28IgKappa_for_CMV-IgKappa-construct.29">Repetition of digestion of  P123 (BAP for BAP-Nanoluc-construct) and P122 (IgKappa for CMV-IgKappa-construct)</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim of experiment:</B> Getting of BAP-Nanoluc-construct, CMV-IgKappa-construct 
</P><P><B>Procedure:</B></P><UL><LI> DNA was digested according to the following table: </LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µg → 20µl
</TD><TD>DNA
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>1,5 µl of AgeI and PstI
</TD><TD>for P123
</TD></TR><TR><TD>1,5 µl of XbaI and PstI
</TD><TD>for P122
</TD></TR><TR><TD>up to 50µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Incubation at 37°C over night. </LI></UL><UL><LI>next steps: Gel with 100Bp ladder and Gelex. Then Ligation with prepared vectors from former digestion (F106 an F63) </LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_9.2C_July_11.E2.80.9317">Week 9, July 11–17</SPAN></H1><DIV class="week" id="Week_9"><H2><SPAN class="mw-headline" id="Monday.2C_July_11"><B>Monday, July 11</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="MiniPreps_of_pSb1C3-Middle_Linker.2BA3C5.2C_-BM40.2C_-CMV-EGFR_Retrafo.2C_Streptactin_in_QC_pASK75">MiniPreps of pSb1C3-Middle Linker+A3C5, -BM40, -CMV-EGFR Retrafo, Streptactin in QC pASK75</SPAN></H3><P><B>Investigator: Christoph</B></P><P><B>Aim of the experiment:</B> Plasmid preparation, analytic digestion, sequencing
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep was performed according to manufacturer's protocol (QIAprep MiniPrep, Qiagen)</LI></UL><UL><LI> Analytic digestion with 0.25 <I>µ</I>L EcoRI, 0.25 <I>µ</I>L PstI (HF) for pSB1C3 plasmids and 0.5 µl NdeI for pASK75-Streptactin, 1 <I>µ</I>L SmartCut Buffer, 3 <I>µ</I>L plasmid DNA, 5.5 <I>µ</I>L H<SUB>2</SUB>O</LI></UL><UL><LI> 5 <I>µ</I>L on 1% agarose gel electrophoresis</LI></UL><UL><LI> plasmid assignment was tricky because of confusion during the weekend Minipreps</LI></UL><P><B>Results:</B></P><UL><LI>  pASK75-Streptactin clone 1 and 2 were not visible</LI></UL><UL><LI>  verified Plasmid names are available after sequencing</LI></UL><UL><LI> Lane 1: 5 µl 1 kb Ladder</LI></UL><UL><LI> Lane 2: 5 <I>µ</I>L Plasmid F101 K1 (seems to be pSB1C3-RFP-Generator)</LI></UL><UL><LI> Lane 3: 5 <I>µ</I>L Plasmid F101 K2 (seems not to be pSB1C3-RFP-Generator)</LI></UL><UL><LI> Lane 4: 5 µl 100 bp Ladder</LI></UL><UL><LI> Lane 5: 5 <I>µ</I>L P125 K1 (seems not to be pSB1C3-RFP-Generator)</LI></UL><UL><LI> Lane 6: 5 <I>µ</I>L P125 K2 (seems to be pSB1C3-RFP-Generator)</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_NanoLuc_.2B_StrepTag_.2B_Poly-A_.28F112.29_and_cloning_behind_CMV-EGFR_.28F107.29">Preperative digestion of NanoLuc + StrepTag + Poly-A (F112) and cloning behind CMV-EGFR (F107)</SPAN></H3><P><B>Investigator: Christoph, Niklas</B></P><P><B>Aim of the experiment:</B> preperative digestion for cloning of the construct behind of CMV-EGFR
</P><P><B>Procedure:</B></P><UL><LI> 15 µl P115 (NanoLuc + StrepTag + Poly-A), 1 µl NgoMiV, 1 µl PstI, 3 µl CutSmart buffer, 10 µl H2O</LI></UL><UL><LI> Incubation for 2 h at 37°C</LI></UL><UL><LI> 1% gel electrophoresis </LI></UL><P>Photo
</P><UL><LI> gel extraction according to SOP </LI></UL><UL><LI> Ligation of 2 µl F107 (AgeI, PstI) + 8 µl F112 (NgomIV, PstI) with 2 µl  Ligase buffer, 7 µl ddH<SUB>2</SUB>O, 1 µl T4-Ligase for 40 minutes</LI></UL><UL><LI> CaCl<SUB>2</SUB>-competent <I>E. coli</I> XL1-Blue cells were taken from the stock in –80 °C freezer and gently thawed on ice</LI></UL><UL><LI> 7 µl of DNA was added to 100 µl of competent cells and gently mixed</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min heat shock at 37 °C</LI></UL><UL><LI> Added 1 ml LB medium to each tube</LI></UL><UL><LI> Incubation for 55 min at 37 °C in the 180 rpm cell-culture shaker</LI></UL><UL><LI> 60 µl of the cell suspension was plated on one chloramphenicol plate</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13,000 rpm and the supernatant was dicarded</LI></UL><UL><LI> The pellet was resuspended in 60 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate (=rescue plate).</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_Nanoluc_behind_BAP">Cloning of Nanoluc behind BAP</SPAN></H3><P><B>Investigator: Christoph, Niklas</B></P><P><B>Aim of the experiment:</B> Cloning of Nanoluc behind BAP based (testing fast cloning strategy)
</P><P><B>Procedure:</B></P><UL><LI> 5 µl P78 (Nanoluc), 0.5 µl NgoMiV, 0.5 µl PstI, 1 µl CutSmart buffer, 2.5 µl H2O, 0.5 SacI = F108</LI><LI> 4 µl P123 (pSB1C3-BAP), 0.5 µl AgeI, 0.5 µl PstI, 1 µl CutSmart buffer, 3 µl H2O,  1 µl FastAP = F109</LI></UL><UL><LI> Incubation for 2 h at 37°C, 15 min heat inactivation</LI></UL><UL><LI> Ligation: 2 µl F108 (Nanoluc) + 8 µl F109 (BAP), 2 µl buffer, 1 µl Ligase, 7 µl H2O</LI></UL><UL><LI>Transformation:</LI></UL><UL><LI> CaCl<SUB>2</SUB>-competent <I>E. coli</I> XL1-Blue cells were taken from the stock in –80 °C freezer and gently thawed on ice</LI></UL><UL><LI> 7 µl of DNA was added to 100 µl of competent cells and gently mixed</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min heat shock at 37 °C</LI></UL><UL><LI> Added 1 ml LB medium to each tube</LI></UL><UL><LI> Incubation for 55 min at 37 °C in the 180 rpm cell-culture shaker</LI></UL><UL><LI> 60 µl of the cell suspension was plated on one chloramphenicol plate</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13,000 rpm and the supernatant was dicarded</LI></UL><UL><LI> The pellet was resuspended in 60 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate (=rescue plate).</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Gel_extraction_of_IgKappa_.28F111.29_and_cloning_behind_CMV_in_pSB1C3_.28F63.29">Gel extraction of IgKappa (F111) and cloning behind CMV in pSB1C3 (F63)</SPAN></H3><P><B>Investigator: Christoph, Niklas</B></P><P><B>Aim of the experiment:</B> gel extraction after digestion for cloning of the construct behind of CMV
</P><P><B>Procedure:</B></P><UL><LI> 1% gel electrophoresis </LI></UL><P>Photo
</P><UL><LI> gel extraction according to SOP </LI></UL><UL><LI> Ligation of 6.1 µl F111 ( XbaI, PstI) + 3.9 µl F63 (SpeI, PstI) with 2 µl  Ligase buffer, 7 µl ddH<SUB>2</SUB>O, 1 µl T4-Ligase for 40 minutes</LI></UL><UL><LI> CaCl<SUB>2</SUB>-competent <I>E. coli</I> XL1-Blue cells were taken from the stock in –80 °C freezer and gently thawed on ice</LI></UL><UL><LI> 7 µl of DNA was added to 100 µl of competent cells and gently mixed</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min heat shock at 37 °C</LI></UL><UL><LI> Added 1 ml LB medium to each tube</LI></UL><UL><LI> Incubation for 55 min at 37 °C in the 180 rpm cell-culture shaker</LI></UL><UL><LI> 60 µl of the cell suspension was plated on one chloramphenicol plate</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13,000 rpm and the supernatant was dicarded</LI></UL><UL><LI> The pellet was resuspended in 60 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate (=rescue plate).</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Retrafo_of_P123_.28digest._GSY3-BAP.29">Retrafo of P123 (digest. GSY3-BAP)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> CaCl<SUB>2</SUB>-competent <I>E. coli</I> XL1-Blue cells were taken from the stock in –80 °C freezer and gently thawed on ice</LI></UL><UL><LI> 1 µl of DNA was added to 100 µl of competent cells and gently mixed</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min heat shock at 37 °C</LI></UL><UL><LI> Added 1 ml LB medium to each tube</LI></UL><UL><LI> Incubation for 55 min at 37 °C in the 180 rpm cell-culture shaker</LI></UL><UL><LI> 60 µl of the cell suspension was plated on one chloramphenicol plate</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13,000 rpm and the supernatant was dicarded</LI></UL><UL><LI> The pellet was resuspended in 60 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate (=rescue plate).</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Digestion_of_quickchanged_mRuby_.28without_StrepTag.29_and_transformation_of_E..C2.A0coli_XL1-Blue_afterwards">Digestion of quickchanged mRuby (without StrepTag) and transformation of <I>E. coli</I> XL1-Blue afterwards</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> the PCR-product is digested by 1 µl DpnI (cuts methylated DNA)</LI></UL><UL><LI> CaCl<SUB>2</SUB>-competent <I>E. coli</I> XL1-Blue cells were taken from the stock in –80 °C freezer and gently thawed on ice</LI></UL><UL><LI> 7 µl of DNA was added to 100 µl of competent cells and gently mixed</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min heat shock at 37 °C</LI></UL><UL><LI> Added 1 ml LB medium to each tube</LI></UL><UL><LI> Incubation for 55 min at 37 °C in the 180 rpm cell-culture shaker</LI></UL><UL><LI> 60 µl of the cell suspension was plated on one chloramphenicol plate</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13,000 rpm and the supernatant was dicarded</LI></UL><UL><LI> The pellet was resuspended in 60 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate (=rescue plate).</LI></UL><UL><LI> next steps: 4 Minipreps, test digestion, Sequencing</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="QuickChange-PCR_of_P100_.28O23_and_O23.29">QuickChange-PCR of P100 (O23 and O23)</SPAN></H3><P><B>Investigators: Julian, Niklas</B></P><P><B>Aim of the experiment:</B> Changing of amino acid in mRuby3      
</P><P><B>Procedure:</B></P><UL><LI> the batches were split, each batch contained only one primer, the batches were pooled after 15 cycles (1 µl Polymerase is added) </LI><LI> Annealing at 55°C</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.25µl
</TD><TD>O23/O24
</TD></TR><TR><TD>0,5 µl
</TD><TD>dNTP-mix
</TD></TR><TR><TD>1 µl
</TD><TD>1:100 P100
</TD></TR><TR><TD>2.5 µl
</TD><TD>Pfu Ultra II-buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II Polymerase
</TD></TR><TR><TD>19.25 µl
</TD><TD>dd H<SUB>2</SUB>O
</TD></TR><TR><TD>4 x 25 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Repeated_digestion_of_P75_.28RFP-Generator.29_with_HincII_.2B_dephosphorylation">Repeated digestion of P75 (RFP-Generator) with HincII + dephosphorylation</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of experiment:</B> Blunt-Ligation of BirA into pSB1C3
</P><P><B>Procedure:</B></P><UL><LI>Digestion of P75 (RFP-Generator) with HincII + dephosphorylation</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>Plasmid DNA (P75)
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>HincII
</TD></TR><TR><TD>1 µl
</TD><TD>Fast AP (for dephosphorylation)
</TD></TR><TR><TD>11 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Digestion was incubated for 2 hours at 37°C</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Ligation_of_phosphorylated_F102_.28PCR_of_BirA.29_into_F113_.28RFP-Generator_P75_digest._HincII.29">Ligation of phosphorylated F102 (PCR of BirA) into F113 (RFP-Generator P75 digest. HincII)</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of experiment:</B> Blunt-Ligation of BirA into pSB1C3
</P><P><B>Procedure:</B></P><UL><LI>The phosphorylated DNA (prepared on July 8th) was ligated into F113 (HincII-digested RFP-Generator) as follows: </LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4,5 µl
</TD><TD>Vector F113
</TD></TR><TR><TD>5,7 µl
</TD><TD>Insert (BirA)
</TD></TR><TR><TD>2 µl
</TD><TD>Ligase Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>next steps: Transformation into E.coli NEB Turbo.</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_12">Tuesday, July 12</SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_BirA.E2.80.93RFP_generator_ligation_.28F114.29_and_Re-Transformation_of_P114">Transformation of BirA–RFP generator ligation (F114) and Re-Transformation of P114</SPAN></H3><P><B>Investigator: Vivien</B></P><P><B>Aim:</B> Transform NEB Turbo cells
</P><P><B>Procedure:</B></P><UL><LI> Followed standard transformation protocol (SOP) using 1 μL and 10 μL of P114 and F114, respectively</LI></UL><UL><LI> Incubated at 37 °C for 1:10 h</LI></UL><UL><LI> Plated on LB Cam (50 μL + rescue)</LI></UL><UL><LI> Incubated at 37 °C (start 12:30)</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Arabinose_Induction_Experiment:_pBAD.E2.80.93GFP">Arabinose Induction Experiment: pBAD–GFP</SPAN></H3><P><B>Investigator: Vivien</B></P><P><B>Aim:</B> Test different induction concentrations of arabinose
</P><P><B>Procedure:</B></P><UL><LI> Of 30 °C o/n culture, filled 5 mL each into four 10 mL culture tubes</LI></UL><UL><LI> Induced with 0, 0.1, 1, and 10 mM final conc. of arabinose</LI></UL><UL><LI> Took 0.5 mL sample each and stored at 4 °C</LI></UL><UL><LI> Incubated 5 mL cultures at 30 °C for 4 h</LI></UL><UL><LI> Took another 0.5 mL sample each and stored at 4 °C</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Picking_Colonies_of_F101_.28P69_Middle_Linker_.E2.80.93_P112_A3C5.29">Picking Colonies of F101 (P69 Middle Linker – P112 A3C5)</SPAN></H3><P><B>Investigator: Vivien</B></P><P><B>Aim:</B> Pick transformed colonies
</P><P><B>Procedure:</B></P><UL><LI> Picked 3 colonies off plate with transformed cells</LI></UL><UL><LI> Let grow at 37 °C in LB Amp (start 11:50 am)</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Picking_Colonies_of_F123_.28BAP.29">Picking Colonies of F123 (BAP)</SPAN></H3><P><B>Investigator: Vivien</B></P><P><B>Aim:</B> Pick transformed colonies
</P><P><B>Procedure:</B></P><UL><LI> Picked 2 colonies off plate with transformed cells</LI></UL><UL><LI> Let grow at 37 °C in LB Cam (start 17:00)</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Verification_of_succesful_Quickchange_of_P141_.28pASK75_with_Streptactin.29">Verification of succesful Quickchange of P141 (pASK75 with Streptactin)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> the Plasmid (5 µl) is digested by 0.5 µl NdeI (Addiition of 1 µl cut smart and 3.5 µl ddH2O)</LI></UL><UL><LI> since the quickchange should eliminate the second Nde-restriction site, there should be one band on the gel</LI></UL><P><B>result:</B></P><UL><LI> there is no band on the gel (the experiment was repeated multiple times showing the same results)</LI></UL><UL><LI><B>therefore new primers are designed and ordered</B></LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Refolding_of_Streptavidin">Refolding of Streptavidin</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> the sample (pelett dissolved in 6M GdmCl)was spun down (4°C, 20 mins, 18,000 rpm)</LI></UL><UL><LI> the supernatant was transferred carefully into a falcon tube and the pellet was cast away</LI></UL><UL><LI> the supernatant was added dropwise to 1 X PBS (500 ml) and stirred over night</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Colony_PCR_of_F110_.28BAP.2BNanoluc.29">Colony PCR of F110 (BAP+Nanoluc)</SPAN></H3><P><B>Investigator: Vivien, Niklas</B></P><P><B>Aim:</B> Amplification 
</P><P><B>Procedure:</B></P><UL><LI> samples were prepared with 5 µL Q5-Mastermix, 1 µl VF2-, 1 µL VR-Primer, 2 µL ddH20</LI></UL><UL><LI> four clones were picked and added to the samples respectively</LI></UL><UL><LI> PCR- cycles according to SOP</LI></UL><UL><LI> Lane 1: 1 kb Ladder, Lane 2-5: clones 1-4, 10 µl each</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P37_and_P26">Preparative digestion and gelelectrophoresis of P37 and P26</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of experiment: Preparative digestion and gelelectrophoresis of P137 (BM40)and P26 (CMV)</B></P><P><B>Procedure:</B></P><UL><LI> Batches for analytical digestions:</LI></UL><UL><LI>P137: EcoRI and XbaI</LI></UL><UL><LI>P26: EcoRI and SpeI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>23/7  µl
</TD><TD>Plasmid DNA P137/P26
</TD></TR><TR><TD>19/35  µl
</TD><TD>ddH2O P137/P26
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>each enzyme(10 U/µl)
</TD></TR><TR><TD>50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Preparative gelelectrophoresis was performed at 90 V for 60 min</LI></UL><UL><LI>The whole PCR-product was applied to a 1% agorose gel. After runnig the gel two lines where visible: one at 1000Bp and one at ~500Bp.</LI></UL><UL><LI>The 600/2100 bp band (P26/P137) was cut out and purified with the Gelextraction kit according to manufacturer's protocoll. Concentration: 20.4/33.1 ng/µl. (F115/F116)</LI></UL><P><A href="/File:Muc16_P137-P26_1207.jpeg" title="File:Muc16 P137-P26 1207.jpeg">File:Muc16 P137-P26 1207.jpeg</A></P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Ligation_of_F115_into_F116">Ligation of F115 into F116</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of experiment:</B> Ligation of F115 (CMV) into F116 (BM40 in pSB1C3)
</P><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4,3  µl
</TD><TD>Vector F116 (BM40 in pSB1C3)
</TD></TR><TR><TD>5,7 µl
</TD><TD>Insert F115(CMV)
</TD></TR><TR><TD>2 µl
</TD><TD>Ligase Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>next steps: Transformation into E.coli NEB Turbo.</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_F117_.28CMV_in_pSB1C3_with_BM40.29.2C_F118_.28QC_P100.29.2C_F119_.28QC_P80.29_and_P26_.28CMV.3B_Retrafo.29">Transformation of F117 (CMV in pSB1C3 with BM40), F118 (QC P100), F119 (QC P80) and P26 (CMV; Retrafo)</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of the experiment:</B> Transformation of CMV in pSB1C3 with BM40 (F117), QC P100 (F118), QC P80 (F119) and P26 in NEB Turbos(F117)/XL1 blue
</P><P><B>Procedure:</B> 
transformation in competent E. coli NEB Turbo (F117)/XL1 blue (rest) according to protocol + rescue 
</P><P><B>Result:</B> 
plates (LB Cam) in incubator for further processing (37 °C)
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_July_13">Wednesday, July 13</SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="QuickChange-PCR_of_P80_.28O23_and_O24.29">QuickChange-PCR of P80 (O23 and O24)</SPAN></H3><P><B>Investigators: Christoph</B></P><P><B>Aim of the experiment:</B> Changing of amino acid in mRuby3      
</P><P><B>Procedure:</B></P><UL><LI> the batches were split, each batch contained only one primer, the batches were pooled after 15 cycles (1 µl Polymerase is added) </LI><LI> Annealing at 55°C</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.25µl
</TD><TD>O23/O24
</TD></TR><TR><TD>0,5 µl
</TD><TD>dNTP-mix
</TD></TR><TR><TD>1 µl
</TD><TD>1:100 P80
</TD></TR><TR><TD>2.5 µl
</TD><TD>Pfu Ultra II-buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II Polymerase
</TD></TR><TR><TD>19.25 µl
</TD><TD>dd H<SUB>2</SUB>O
</TD></TR><TR><TD>4 x 25 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>Cycling was performed according to SOP.
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="MiniPrep_of_Middle_Linker-A3C5_construct">MiniPrep of Middle Linker-A3C5 construct</SPAN></H3><P><B>Investigator: Christoph </B></P><P><B>Aim of the experiment:</B> MiniPrep two colonies of Middle Linker-A3C5 construct
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep was performed according to manufacturer's protocol (QIAprep MiniPrep, Qiagen)</LI><LI> Both plasmids were sent to sequencing without analytical digestion because of the short lenght</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_F120_.28CMV-IgKappa.29_and_F121_.28CMV.2BEGFR_.2F_Nanoluc.2BStrepTag.2BPolyA.29">Cloning of F120 (CMV-IgKappa) and F121 (CMV+EGFR / Nanoluc+StrepTag+PolyA)</SPAN></H3><P><B>Investigator: Christoph</B></P><P><B>Aim of the experiment:</B> Ligations of F120 (CMV-IgKappa) and F121 (CMV+EGFR / Nanoluc+StrepTag+PolyA) and transformation
</P><P><B>Procedure:</B></P><UL><LI> Ligation mixtures were prepared as following:</LI><LI> F120: 	5 μL F63, 5 µL F111, 2 μL Ligase buffer, 7 μL ddH2O, 1 μL Ligase </LI><LI> F121: 	3 μL F107, 7 μL F112, 2 μL Ligase buffer, 7 μL ddH2O, 1 μL Ligase </LI><LI> Ligation was incubated for 30 min. at room temperature</LI><LI> transformation was performed according to SOP (with 100 µl cells and 20 µl ligation mix) and plating in LB+Cam agar</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_July_14">Thursday, July 14</SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="MiniPreps_of_Overnight_Cultures">MiniPreps of Overnight Cultures</SPAN></H3><P><B>Investigator: Vivien</B></P><P><B>Aim:</B> Prep plasmid DNA of P26, F110, F117, F114, P114, and P118
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep performed according to manufacturer's instructions with Qiagen kit</LI></UL><UL><LI> Prior to eluting, 50 μL EB buffer was applied to the membrane, and the columns were heated at 37 °C for 7 min</LI></UL><P><B>Results:</B> Plasmid DNA solutions of the following concentrations: P26 (a: 55.1 ng/μL; b: 48.1 ng/μL); F110 (K2: 150.1 ng/μL; K3: 173.0 ng/μL); F117 (a: 257.7 ng/μL; b: 159.0 ng/μL; CMV/BM40: 133.2 ng/μL); F114 (83.1 ng/μL); P114 (a: 26.1 ng/μL; 170.7 ng/μL); P118 (<B>F118?</B>) (a: 248.0 ng/μL; 53.2 ng/μL; 241.3 ng/μL). F110 (K2: FR12904474; K3: FR12904473) and F117 (a: FR12904477; b: FR12904476; CMV/BM40: FR12904475) were sent for sequencing with VF2.
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="DpnI_Digest_of_mRuby3_QC_and_Transformation">DpnI Digest of mRuby3 QC and Transformation</SPAN></H3><P><B>Investigator: Vivien</B></P><P><B>Aim:</B> Digest original-sequence plasmid and transform obtained pure quick-changed plasmid
</P><P><B>Procedure:</B></P><UL><LI> QC was digested with DpnI (40 μL QC, 5 μL 10× CutSmart buffer, 3.5 μL ddH<SUB>2</SUB>O, 1.5 μL DpnI = 50 μL) at 37 °C for 2 h</LI></UL><UL><LI> Digest (10 μL) was directly transformed into Turbo cells following SOP (including 1 h incubation time prior to plating due to chloramphenicol resistance)</LI></UL><UL><LI> Plated 50 μL on LB Cam + rescue plate</LI></UL><UL><LI> Incubated at 37 °C</LI></UL><P><B>Result:</B> two plates; remaining digest stored at 4 °C in styrofoam tube rack
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_15"><B>Friday, July 15</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_mRuby_Quick_change.2C_F120_.28CMV.2BIgKappa.29_and_F121_.28CMV.2BEGFR.2FNanoluc.2BStrepTag.2BPolyA.29_preculture_and_miniprep">Inoculation of mRuby Quick change, F120 (CMV+IgKappa) and F121 (CMV+EGFR/Nanoluc+StrepTag+PolyA) preculture and miniprep</SPAN></H3><P><B>Investigator: Niklas, Clemens</B></P><P><B>Procedure:</B></P><UL><LI> 6 colonies were picked from plate</LI><LI> Cells were inoculated in 5 mL LB medium (Cam 1:1000)</LI><LI> Fragments were minipreped following manufacturer's protocol (QiaPrep, Qiagen)</LI><LI> Fragments were renamed receiving a P number</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Quickchange_of_P88_.28pASK75.28Streptactin.29.29_with_O-029_and_O-030">Quickchange of P88 (pASK75(Streptactin)) with O-029 and O-030</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim:</B> Quickchange pASK75(Streptactin) to erase one NdeI restriction site
</P><P><B>Procedure:</B></P><UL><LI> For quickchange two seperate batches were prepared:</LI><LI> 1. batch: 0.7 µL O-029, 0.5 µL dNTP mix, 2.5 µL buffer, 0.5 µL Pfu ultra polymerase, 1.5 µL P88</LI><LI> 2. batch: 0.7 µL O-030, 0.5 µL dNTP mix, 2.5 µL buffer, 0.5 µL Pfu ultra polymerase, 1.5 µL P88</LI><LI> first stage of quickchange PCR was performed as following:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature [°C]</B></TD><TD><B>Time [s]</B></TD></TR><TR><TD>initial Denaturation</TD><TD>98</TD><TD>30
</TD></TR><TR><TD rowspan="3">15 cycles</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>55</TD><TD>30
</TD></TR><TR><TD>68</TD><TD>5
</TD></TR><TR><TD>final Extension</TD><TD>68</TD><TD>5
</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>ꝏ
</TD></TR></TBODY></TABLE><UL><LI> PCR batches were mixed and 0.5 µL pfu ultra polymerase was added</LI><LI> Second stage of quickchange PCR was performed as in the following:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature [°C]</B></TD><TD><B>Time [s]</B></TD></TR><TR><TD>initial Denaturation</TD><TD>98</TD><TD>30
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>55</TD><TD>30
</TD></TR><TR><TD>68</TD><TD>5
</TD></TR><TR><TD>final Extension</TD><TD>68</TD><TD>5
</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>ꝏ
</TD></TR></TBODY></TABLE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_P79_.28Nanoluc.29_into_P147_.28pSB1C3.28BAP.29.29">Cloning of P79 (Nanoluc) into P147 (pSB1C3(BAP))</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim:</B></P><P><B>Procedure:</B></P><UL><LI> 7 µL P79 were digested with NgOMIV and PstI</LI><LI> 10 µL P147 were digested with AgeI and PstI</LI><LI> Digestion products were run on a 1% agarose gel</LI><LI> Both fragments could be cut from the gel</LI><LI> Fragments were purified by gelectraction following manufacturer's protocol (Qiagen)</LI></UL><UL><LI>Fragments were renamed:
</LI><LI> digest P78= F123</LI><LI> digest P147= F124</LI></UL><LI> 9,5 µL F124 and 0.5 µL F123 were ligated following manufacturers protocol (T4 DNA ligase, Thermo scientific)</LI><LI> NEB Turbo were transformed with 7 µL of F125 (ligation F123+F124) following SOP</LI><LI> Cells were plated on CAM LB medium and placed in the incubator at 37°C</LI></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_P145_.28Middle_Linker_.2BA3C5.29_and_P146_.28Middle_Linker_.2BA3C5.29_into_F70_.28Vector_fragment_pSB1C3.29">Cloning of P145 (Middle Linker +A3C5) and P146 (Middle Linker +A3C5) into F70 (Vector fragment pSB1C3)</SPAN></H3><P><B>Investigator: Niklas, Clemens</B></P><P><B>Procedure:</B></P><UL><LI>Ligation:
</LI><LI> 2,5 µL P145 and 1 µL P146 were pooled and digested with NgOMIV and SpeI for 1h at 37°C.</LI><LI> Full digestion was ligated into 1 µL F70</LI></UL><UL><LI> CaCl<SUB>2</SUB>-competent <I>E. coli</I> XL1-Blue cells were taken from the stock in –80 °C freezer and gently thawed on ice</LI></UL><UL><LI> 7 µl of DNA was added to 100 µl of competent cells and gently mixed</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min heat shock at 37 °C</LI></UL><UL><LI> Add 1 ml LB medium to each tube</LI></UL><UL><LI> Incubation for 55 min at 37 °C in the 180 rpm cell-culture shaker</LI></UL><UL><LI> 60 µl of the cell suspension was plated on one chloramphenicol plate</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13,000 rpm and the supernatant was dicarded</LI></UL><UL><LI> The pellet was resuspended in 60 µl of LB-medium and this concentrated cell suspension was plated again on a chlorampenicol plate (=rescue plate).</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Test_digestion_of_P164_.28BirA_blunt_end_ligation.29_.28after_miniprep.29_and_P161.2C_P162.2C_and_P163_.28all_QC_mRuby.29">Test digestion of P164 (BirA blunt end ligation) (after miniprep) and P161, P162, and P163 (all QC mRuby)</SPAN></H3><P><B>Investigator: Niklas, Clemens</B></P><P><B>Aim:</B> Test digestion to see which plasmids contain the right insert
</P><P><B>Procedure:</B></P><UL><LI> 150 ng of plasmid DNA were digested with EcoRI and PstI following SOP</LI><LI> The whole batch was applied to a 1% gel to check for cut inserts.</LI></UL><P><B>Results:</B></P><P>FOTO
</P><UL><LI> Blunt end ligation seems to not have worked (P164)</LI><LI> P161, P162 and P163 all still carry the insert</LI></UL><P>Sequenced: P161= FR12904472 (NA)
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Harvesting_of_Streptavidin_2">Harvesting of Streptavidin</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of the experiment:</B> Recombinant expression and purification of streptavidin
</P><P><B>Procedure:</B></P><UL><LI> After precipitation in 70 % ammonium sulfate the soluton was spun down in the centrifuge (4°C, 11000 rpm, 40 mins)</LI><LI> The supernatant was discarded, and the pellet was dissolved in 2.2 M ammonium sulfate (18 mL)</LI><LI> the sample was spun down in the centrifuge (4°C, 15000 rpm, 30 mins)</LI><LI> the pellet was dissolved in 1X PBS(16 mL)</LI><LI> next steps: last centrifugation and dialysis against Tris/Hcl (see plan on board)</LI></UL></DIV><H1><SPAN class="mw-headline" id="Week_10.2C_July_18.E2.80.9324">Week 10, July 18–24</SPAN></H1><DIV class="week" id="Week_10"><H2><SPAN class="mw-headline" id="Monday.2C_July_18"><B>Monday, July 18</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_precultures_for_miniprep_of_F125_.28pSB1C3.28BAP-NanoLuc.29.29_and_F122_.28pSB1C3.28Middlelinker-A3C5.29.29">Inoculation of precultures for miniprep of F125 (pSB1C3(BAP-NanoLuc)) and F122 (pSB1C3(Middlelinker-A3C5))</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment: </B></P><UL><LI> Miniprep, analytical digestion, analytical gel, sequencing</LI></UL><P><B>Procedure:</B></P><UL><LI> Three colonies were picked from each plate (F125, F122)</LI><LI> Inoculation was performed according to SOP</LI><LI> Cells grew for 5h
</LI><LI> Cells onlygrew in cultures with F125 (In cultures with F122 there was no pelet) </LI></UL><LI> Miniprep was performed according to manufactrer's protocol</LI><LI> Samples were renamed:
</LI><LI> F125 K1: P168</LI><LI> F125 K2: P169</LI><LI> F125 K3: P170</LI><UL><LI> Analytical digestion:
</LI><LI> Each plasmid was digested with EcoRI and PstI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>Plasmid DNA P168/P169/P170
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI(10 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Results:</B></P><P>Foto:
</P><UL><LI>Lane 1: 1kb ladder</LI><LI>Lane 2: P168</LI><LI>Lane 3: P169</LI><LI>Lane 4: P170 --&gt; right signals (2000 bp, ~600 bp(BAP-NanoLuc)</LI></UL><UL><LI>Send for sequencing:
</LI><LI>P170: FR12904468 (VF2)</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P149_.2C_P150_.28both_pSB1C3.28CMV-IgKappa.29.29.2C_P151.2C_P152_.28both_pSB1C3.28CMV-EGFR-NanoLuc-StrepTag-polyA.29.29.2C_P153.2C_P154_.28both_QC_pSB1C3.28mRuby3.29.29">Analytical digestion of P149 , P150 (both pSB1C3(CMV-IgKappa)), P151, P152 (both pSB1C3(CMV-EGFR-NanoLuc-StrepTag-polyA)), P153, P154 (both QC pSB1C3(mRuby3))</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment: </B> 
Analytical digeston to find out if the cloning was successful
</P><P><B>Procedure:</B></P><UL><LI> measure concentrations of Plasmids (see inventary list)</LI></UL><UL><LI> Analytical digestion:
</LI><LI> Each plasmid was digested with EcoRI and PstI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>Plasmid DNA P149/ P150/ P151/ P152/ P153/ P154
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI(10 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Results:</B></P><P>Foto:
</P><UL><LI>Lane 1: 1kb ladder</LI><LI>Lane 2: P149 --&gt; right signals (2000bp; 730bp)</LI><LI>Lane 3: P150 --&gt; right signals (2000bp; 730bp)</LI><LI>Lane 4: P151</LI><LI>Lane 5: P152 --&gt; right signals (2000bp; 1626bp)</LI><LI>Lane 6: P153 --&gt; very weak signal</LI><LI>Lane 7: P154 --&gt; right signals (2000bp; 700bp)</LI></UL><UL><LI>Send for sequencing:
</LI><LI> P150: FR12904471 (VF2)</LI><LI> P152: FR12904470 (VF2)</LI><LI>P154: FR12904469 (VF2)</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Transformation_of_QC_of_P88_.28pASK75.28Streptactin.29.29">Transformation of QC of P88 (pASK75(Streptactin))</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment: </B></P><UL><LI> See if quickchange has worked</LI></UL><P><B>Procedure:</B></P><UL><LI> NEB Turbo cells were transformed with 10 µl of QC P88 according to SOP</LI><LI> Cells were plated on LB medium (Amp) and incubated at 37°C</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Digestion_of_P165_.28pSB1C3.28CMV-BM40.29_and_cloning_of_F112_.28Nanoluc-StrepTag-polyA.29_into_P165">Digestion of P165 (pSB1C3(CMV-BM40) and cloning of F112 (Nanoluc-StrepTag-polyA) into P165</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment: </B></P><UL><LI> Create biobrick: CMV-BM40-Nanoluc-StrepTag-polyA </LI></UL><P><B>Procedure:</B></P><UL><LI> Preparative digestion batch was prepared as in the following:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>Plasmid DNA P156
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI(20 U/µl)
</TD></TR><TR><TD>10 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestion was seperated by gelelectrophoresis</LI></UL><P><B>Results:</B></P><P>Foto:
</P><UL><LI> Lane 1: 1kb ladder</LI><LI> Lane 2: F126 (digest P165) --&gt; right length</LI></UL><P><B>Procedure:</B></P><UL><LI> Fragment was cut from the gel and purified by gelextraction which was performed by manufacturer's protocol (Qiagen)</LI></UL><UL><LI> F126 was ligated with F112</LI></UL><P>Ligation batch: 
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>Vector DNA F126
</TD></TR><TR><TD>18 µl
</TD><TD>Insert DNA F112
</TD></TR><TR><TD>3 µl
</TD><TD>T4 ligation buffer (10x)
</TD></TR><TR><TD>6 µl
</TD><TD>T4 ligase
</TD></TR><TR><TD> µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was incubated at 37°C for 30 min</LI><LI> NEB Tubros were transformed with F127 (lig. F112+F126)</LI><LI> Cells were plated on LB medium (Cam) </LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_19"><B>Tuesday, July 19</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_Precultures_of_F127_.28pSB1C3.28CMV-BM40-Nanoluc-StrepTag-polyA.29.29">Inoculation of Precultures of F127 (pSB1C3(CMV-BM40-Nanoluc-StrepTag-polyA))</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim:</B> Prepare cells for miniprep of plasmids
</P><P><B>Procedure:</B></P><UL><LI> six colonies were picked and inoculated in LB medium (CAM)</LI><LI> Plates were incubated on a shaker at 37°C</LI></UL><P><B>Results:</B></P><UL><LI> cellgrowth ?</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Reamplifiying_Streptactin_from_GSY6">Reamplifiying Streptactin from GSY6</SPAN></H3><P><B>Investigator: Christoph</B></P><P><B>Aim:</B> Reamplifiying Streptactin from GSY6 for further processing in QC reaction (delete NdeI site from pASK)
</P><P><B>Procedure:</B></P><UL><LI> PCR reaction was prepared according to SOP</LI><LI> Cycling steps were taken from the QC protocol because no further cycler was free</LI></UL><P><B>Results:</B></P><UL><LI> will be available after DpnI digestion and transformation</LI></UL></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="QuickChange-PCR_of_pASK75_with_O29_and_O30">QuickChange-PCR of pASK75 with O29 and O30</SPAN></H3><P><B>Investigator: Christoph</B></P><P><B>Aim:</B> Deleting NdeI from pASK (P10)       
</P><P><B>Procedure:</B></P><UL><LI> QC-PCR was performed according to SOP </LI><LI> Primers: O-029 and O-030</LI><LI> Annealing at 55°C</LI><LI> Elongation time: 5 min </LI><LI> 2-step protocol was applied</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_E._coli_Xl1-Blue_P171.2C_P172_and_P173_.28pSB1C3.28CMV-BM40-Nanoluc-StrepTag-polyA.29.29">Miniprep of E. coli Xl1-Blue P171, P172 and P173 (pSB1C3(CMV-BM40-Nanoluc-StrepTag-polyA))</SPAN></H3><P><B>Investigator: Javier</B></P><P><B>Aim of the experiment:</B> Extracting P171, P172 and P173 (CMV-BM40-Nanoluc-StrepTag-polyA) from E.coli XL-1-blue 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)</LI><LI> Concentrations:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>c [ng/µl]</B></TD></TR><TR><TD>P171
</TD><TD>75,4
</TD></TR><TR><TD>P172
</TD><TD>108,2
</TD></TR><TR><TD>P173
</TD><TD>66,4
</TD></TR></TBODY></TABLE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_F136_.28A3C5.2BMiddle-Linker.29.2C_F137_.28EGFR.2BmRuby-Strep.29.2C_F138_.28mRuby3-Strep.2BPolyA.29_and_P161_.28QC_mRuby-Strep.29">Transformation of F136 (A3C5+Middle-Linker), F137 (EGFR+mRuby-Strep), F138 (mRuby3-Strep+PolyA) and P161 (QC mRuby-Strep)</SPAN></H3><P><B>Investigator: Javier</B></P><P><B>Aim of the experiment:</B> Transformation of A3C5+Middle-Linker (F136), EGFR+mRuby-Strep (F137), mRuby3-Strep+PolyA (F138) and QC mRuby-Strep (P161) XL1 blue
</P><P><B>Procedure:</B> 
transformation in competent E. coli XL1 blue according to protocol + rescue 
</P><P><B>Result:</B> 
plates (LB Cam) in incubator for further processing (37 °C)
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Colony_PCR_of_F127_.28CMV-BM40-Nanoluc-StrepTag-polyA.29">Colony PCR of F127 (CMV-BM40-Nanoluc-StrepTag-polyA)</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim:</B> See if cloning was successfull
</P><P><B>Procedure:</B></P><UL><LI> samples wwere prepared with 5 µL Q5-Mastermix, 1 µl VF2-, 1 µL VR-Primer, 2 µL ddH20</LI></UL><UL><LI> six clones were picked and added to the samples respectively</LI></UL><UL><LI> PCR- cycles according to SOP (Annealing-temp: 66°C; Ampflification time: 1.5min)</LI></UL><P><B>Results:</B></P><UL><LI> Lane 1: 1 kb Ladder</LI><LI> Lane 2: F127 K1</LI><LI> Lane 3: F127 K2</LI><LI> Lane 4: F127 K3</LI><LI> Lane 5: F127 K4</LI><LI> Lane 6: F127 K5</LI><LI> Lane 7: F127 K6</LI></UL><P><A href="/File:Muc16_ColonyPCR_F117_1907.jpeg" title="File:Muc16 ColonyPCR F117 1907.jpeg">500px</A></P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P161_.28pSB1C3.28mRuby3-StrepTag.29.29.2C_P73_.28pSB1C3.28.28PolyA.29.29.2C_P96_.28.28pSB1C3.28EGFR-TMD.29.29.2C_P69_.28pSB1C3.28Middle-linker.29.29_and_P112_.28pSB1C3.28A3C5.29.29">Preparative digestion of P161 (pSB1C3(mRuby3-StrepTag)), P73 (pSB1C3((PolyA)), P96 ((pSB1C3(EGFR-TMD)), P69 (pSB1C3(Middle-linker)) and P112 (pSB1C3(A3C5))</SPAN></H3><P><B>Investigator: Clemens, Christoph, Javier</B></P><P><B>Aim:</B> Create biobricks
</P><P><B>Procedure:</B></P><UL><LI> Preparative digestion of P161 and P73</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl/ 20 µl
</TD><TD>Plasmid DNA P161/ P73
</TD></TR><TR><TD>3 µl/ 3 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>1 µl/ 1 µl
</TD><TD>EcoRI/ EcoRI
</TD></TR><TR><TD>1 µl/ 1 µl
</TD><TD>SpeI/ XbaI
</TD></TR><TR><TD>10 µl/ 5 µl
</TD><TD>ddH2O/ ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Preparative digestion of P96 and P161</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl/ 15 µl
</TD><TD>Plasmid DNA P96/ P161
</TD></TR><TR><TD>3 µl/ 3 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>1 µl/ 1 µl
</TD><TD>PstI/ PstI
</TD></TR><TR><TD>1 µl/ 1 µl
</TD><TD>AgeI/ NgOMIV
</TD></TR><TR><TD>15 µl/ 10 µl
</TD><TD>ddH2O/ ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Preparative digestion of P69 and P112</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl/ 15 µl
</TD><TD>Plasmid DNA P69/ P112
</TD></TR><TR><TD>3 µl/ 3 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>1 µl/ 1 µl
</TD><TD>EcoRI/ EcoRI
</TD></TR><TR><TD>1 µl/ 1 µl
</TD><TD>AgeI/ NgOMIV
</TD></TR><TR><TD>5 µl/ 10 µl
</TD><TD>ddH2O/ ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestions were incubated at 37°C for 2h</LI><LI> Samples were applied to a 1% agarose gel (40 min; 90V)</LI><LI> Samples were cut from the gel afterwards:</LI><LI> Renamed:
</LI><LI> digest P69: F128</LI><LI> digest P112: F129</LI><LI> digest P161: F130</LI><LI> digest P96: F131</LI><LI> digest P161: F132</LI><LI> digest P73: F133</LI></UL><P><B>Results:</B></P><P>Fotos
</P><P><B>Procedure:</B></P><UL><LI> Gelextraction was performed after manufacturers protocol (Qiagen)</LI><LI> After gelextraction concentrations were measured:</LI></UL><UL><LI> Ligation:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>xx µl
</TD><TD>Vektor F129
</TD></TR><TR><TD>xx µl
</TD><TD>Insert F128
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>x µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>xx µl
</TD><TD>Vektor F131
</TD></TR><TR><TD>7xx µl
</TD><TD>Insert F130
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>x µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>xx µl
</TD><TD>Vektor F133
</TD></TR><TR><TD>7xx µl
</TD><TD>Insert F132
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>x µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed according to SOP</LI><LI> Xl1 blue cells were transformed with the Ligations according to SOP</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_July_20"><B>Wednesday, July 20</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P171-P173_.26_QC_of_P10">Analytical digestion and gelelectrophoresis of P171-P173 &amp; QC of P10</SPAN></H3><P><B>Investigator: JB, MT</B></P><P><B>Aim of experiment:</B>  Check if everything is fine.
</P><P><B>Procedure:</B></P><UL><LI> Digest P171-P173 with XbaI + SpeI and QC of P10 with NedI and incubate 1 h at 37 °C</LI><LI> Run 1 % agarose-gel at least 40 min at 100 V</LI></UL><P><B>Results:</B></P><P>Lane1: 1Kb ladder; Lane2: P171; Lane3: P172; Lane4: P173; Lane5: QC of P10
</P><P>=&gt; P171 was sent for sequencing: FR12904466
=&gt; NdeI test digestion shows two signals although there should have been just one
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_GSY_6.3B_F134.3B_F135_for_ligation">Preparation of GSY 6; F134; F135 for ligation</SPAN></H3><P><B>Investigator: JB, MT </B></P><P><B>Aim of experiment:</B> Prepare GSY6 for Ligation.
</P><P><B>Procedure:</B></P><UL><LI> Digestion of GSY6 with SpeI + HindIII</LI><LI> Preparative 1 % Agarose gel 100 V for at least 40 min.</LI><LI> Gel extraction of GSY6, F134, F135 by Quiagen kit.</LI></UL><P><B>Results:</B></P><UL><LI> GSY6 -&gt; F139 : 2.5 ng/µL</LI><LI> F134 : 4 ng/µL </LI><LI> F135 : 10.2 ng/µL</LI></UL><P>Preparative agarose gel GSY6:
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_.26_transformation_of_F134_.2B_F63.3B_F135_.2B_F139">Ligation &amp; transformation of F134 + F63; F135 + F139</SPAN></H3><P><B>Investigator: JB, MT</B></P><P><B>Aim of experiment:</B>  Ligation &amp; transformation of F134 + F63; F135 + F139
</P><P><B>Procedure:</B></P><P>Ligation following SOP(13.02.2013)
</P><UL><LI> F134 + F63 -&gt; F140</LI><LI> F135 + F139 -&gt; F141</LI></UL><P>transformation following SOP(13.02.2013)
</P><UL><LI> F140 on CAM</LI><LI> F141 on AMP</LI></UL><P><B>Results:</B></P><PRE>  We will see tomorrow, please update this section.
</PRE></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_July_21"><B>Thursday, July 21</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_F136_.28K1_.26_K2.29_.28pSB1C3.28Middle_linker-A3C5.29.29.2C_F137_.28K1_.26_K2.29_.28pSB1C3.28EGFR_TMD-mRuby3-StepTag.29.29.2C_F138_.28K1_.26_K2.29_.28pSB1C3.28mRuby3-StrepTag-polyA.29_and_retrafo_of_P161_.28pSB1C3.28mRuby3-StrepTag.29.29">Miniprep of F136 (K1 &amp; K2) (pSB1C3(Middle linker-A3C5)), F137 (K1 &amp; K2) (pSB1C3(EGFR TMD-mRuby3-StepTag)), F138 (K1 &amp; K2) (pSB1C3(mRuby3-StrepTag-polyA) and retrafo of P161 (pSB1C3(mRuby3-StrepTag))</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of experiment:</B>  Get plasmids for analytical digestion and sequencing
</P><P><B>Procedure:</B></P><UL><LI> Minipreps were performed according to manufacturers protocol (Qiagen, QiaPrep)</LI><LI> Samples were renamed afterwards:
</LI><LI> F136 K1: P174</LI><LI> F136 K2: P175</LI><LI> F137 K1: P176</LI><LI> F137 K2: P177</LI><LI> F138 K1: P178</LI><LI> F138 K2: P179</LI><LI> retrafo of P161: P180 </LI></UL><UL><LI> Analytical digestion:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI
</TD></TR><TR><TD>7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was renamed: F143</LI><LI> Digestion products were seperated via gelelectrophoresis</LI></UL><P><B>Results:</B></P><P>Foto:
</P><UL><LI> All plasmids seemed to carry the appropriate insert</LI></UL><P>Sequencing:
</P><UL><LI> P175: FR12904465 (VF2)</LI><LI> P177: FR12904464 (VF2)</LI><LI> P178: FR12904463 (VF2)</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P116_.28pSB1C3.28IgKappa.29.29">Preparative digestion of P116 (pSB1C3(IgKappa))</SPAN></H3><P><B>Investigator: Clemens, Christoph</B></P><P><B>Aim of experiment:</B>  cloning of CMV in front of IgKappa signal peptide
</P><P><B>Procedure:</B></P><UL><LI> preparative digestion batch</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>Plasmid DNA P116
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestion was applied to a 1% agarose gel (45 min; 90V)</LI><LI> Sample was cut from the gel and gelextraction was performed according to manufacturers protocol</LI></UL><UL><LI> Ligation of Fx and F115 (digest CMV promotor):</LI></UL><UL><LI> Ligation:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3.9 µl
</TD><TD>Vektor F142
</TD></TR><TR><TD>6.1 µl
</TD><TD>Insert F115
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>20 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Transformation of XL1blue was performed according to SOP</LI><LI> Cells were plated on LB medium (CAM) and incubated over night at 37°C</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_22"><B>Friday, July 22</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_pASK_T7_RBS_LCN2_His">Analytical digestion of pASK_T7 RBS_LCN2_His</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim of the experiment: </B> 
Analytical digestion to find out whether there is more than one NdeI restriction site. 
</P><P><B>Procedure:</B></P><UL><LI> Analytical digestion:
</LI><LI>The plasmid was digested with NdeI and HindIII</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>NdeI
</TD></TR><TR><TD>0.5 µl
</TD><TD>HindIII
</TD></TR><TR><TD>5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI> Two bands were visible what means that there is only one NdeI restriction site. </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_Precultures_of_F143_.28pSB1C3.28CMV.2BIgKappa.29.29">Inoculation of Precultures of F143 (pSB1C3(CMV+IgKappa))</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim:</B> Prepare cells for miniprep of plasmids
</P><P><B>Procedure:</B></P><UL><LI> two colonies were picked and inoculated in LB medium (CAM)</LI><LI> Plates were incubated on a shaker at 37°C over night. </LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Harvesting_of_Streptavidin_3">Harvesting of Streptavidin</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of the experiment:</B> Recombinant expression and purification of streptavidin
</P><P><B>Procedure:</B></P><UL><LI> the solution was spun down in the centrifuge (4°C, 15000 rpm, 20 mins) to remove last contamination</LI><LI> The pellet was discarded and the supernantant was dialysed against 20 mM Tris/HCl (ph 8) witt 50 mM NaCl</LI><LI> last step to do: steril filtration</LI></UL><UL><LI> the concentration was determined via UV/VIS: the absorption at 280 nm is 0.469 (1:100 dilution), this equals a concentration of 62 mg/mL (about 16 mL) (mass 54 kDa; Absorbtion coefficient 40455 1/M*cm</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Sequencing_of_P174.28Middle_Linker_.2B_A3C5.29">Sequencing of P174(Middle Linker + A3C5)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> Sequencing batch were prepared after manufacturer's protocol. (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer)</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_P178_.28mRubyStrepPolyA.29_.2F_P180_.28mRubyStrep.29_behind_EGFR-TMD_in_pSB1C3_.28P96.29">Cloning of P178 (mRubyStrepPolyA) / P180 (mRubyStrep) behind EGFR-TMD in pSB1C3 (P96)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Procedure:</B></P><UL><LI> Preparative digestion:</LI><LI> The Inserts were digested with PstI and NgoMIV</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>7.7 / 8.2 µl (about 3µg)
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>8.3 / 7.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> both fragments were ligated with F131 (digest P96 [AgeI; PstI])</LI></UL><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were taken out of stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 7 µl of DNA was added to 100 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of 750 µl LB-medium to each tube.</LI></UL><UL><LI> Incubation for 1 hour at 37 °C</LI></UL><UL><LI> The cell suspension was plated on Cam-plates (inclusive rescue plate) and incubated over night at 37 °C in the cell-culture shaker.</LI></UL><UL><LI> Results: Cloning didn't work. No colonies on plate. Repeat.</LI></UL></DIV><DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_Middle_Linker_before_A3C5_in_pSB1C3_.28Repeat.29">Cloning of Middle Linker before A3C5 in pSB1C3 (Repeat)</SPAN></H3><P><B>Investigator: Julian, Niklas</B></P><P><B>Procedure:</B></P><UL><LI> Preparative digestion:
</LI><LI>The Middle Linker (P69) was digested with EcoRI and AgeI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>30 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>5 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>1.5 µl
</TD><TD>AgeI
</TD></TR><TR><TD>12 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> the fragment was ligated with F129(digest P112 [EcoRI; NgOMIV])</LI></UL><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were taken out of stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 7 µl of DNA was added to 100 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of 750 µl LB-medium to each tube.</LI></UL><UL><LI> Incubation for 1 hour at 37 °C</LI></UL><UL><LI> The cell suspension was plated on Cam-plates (inclusive rescue plate) and incubated over night at 37 °C in the cell-culture shaker.</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_11.2C_July_25th_-_July_31st">Week 11, July 25th - July 31st</SPAN></H1><DIV class="week" id="Week_11"><H2><SPAN class="mw-headline" id="Monday.2C_July_25"><B>Monday, July 25</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_Precultures_of_F143_.28pSB1C3.28CMV.2BIgKappa.29.29.2C_F141new_.28pASK.28Streptactin.29.29.2C_F151_.28pSB1C3.28middleLinker.2BA3C5.29.2C_pASK75_T7-RBS-LCN2-His.2C_F148_.28pSB1C3.28mRubyStrepPolyA.2B_CMV-EGFR_Signal-Pept.29">Inoculation of Precultures of F143 (pSB1C3(CMV+IgKappa)), F141new (pASK(Streptactin)), F151 (pSB1C3(middleLinker+A3C5), pASK75_T7-RBS-LCN2-His, F148 (pSB1C3(mRubyStrepPolyA+ CMV-EGFR Signal-Pept)</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim:</B> Prepare cells for miniprep of plasmids
</P><P><B>Procedure:</B></P><UL><LI> Three colonies of each were picked and inoculated in LB medium (CAM) respectively (AMP) fpr pASK. </LI><LI> Plates were incubated on a shaker at 37°C for 6 hours. </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="MiniPreps_and_analytical_digestion_of_F143_cultures">MiniPreps and analytical digestion of F143 cultures</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim:</B> Prep plasmid DNA of F143 (pSB1C3(CMV + IgKappa)) and checking wheather ligation was successful. 
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep performed according to manufacturer's instructions with Qiagen kit</LI></UL><UL><LI> Prior to eluting, 30 μL EB buffer was applied to the membrane since the pellet was quite small, and the columns were heated at 37 °C for 7 min</LI></UL><P><B>Results:</B></P><UL><LI> Concentrations: K1 = 230,4ng/µl, K2 = 162,4 ng/µl</LI></UL><P><B>Analytical Digestion:</B></P><UL><LI> An 20µl batch was prepared for analytical digestion for each of the two clones:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoI
</TD></TR><TR><TD>1 µl
</TD><TD>PstI
</TD></TR><TR><TD>6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI> Two bands were visible at 750 Bp which is the size of IgKappa and CMV together. Cloning was successful. Sequence still has to be ckecked though. </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_P178_.28mRubyStrepPolyA.29_.2F_P180_.28mRubyStrep.29_behind_EGFR-TMD_in_pSB1C3_.28P96.29._Repetition.">Cloning of P178 (mRubyStrepPolyA) / P180 (mRubyStrep) behind EGFR-TMD in pSB1C3 (P96). Repetition.</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim:</B> See title.  
</P><P><B>Procedure:</B></P><UL><LI>Digestion of P178 and P180: 7µg of DNA were digested. </LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>18 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>5 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>2,5 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>2,5 µl
</TD><TD>PstI
</TD></TR><TR><TD>22 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestion at 37°C for 2,5h. </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_NanoLuc-Strep-PolyA_.28F112.29_behind_CMV-IgKappa_in_pSB1C3_.28P181.29.">Cloning of NanoLuc-Strep-PolyA (F112) behind CMV-IgKappa in pSB1C3 (P181).</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim:</B> See title.  
</P><P><B>Procedure:</B></P><UL><LI>Digestion of P181:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>5 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>2,5 µl
</TD><TD>AgeI (20u/µl)
</TD></TR><TR><TD>2,5 µl
</TD><TD>PstI (20u/µl)
</TD></TR><TR><TD>30 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestion at 37°C for 2,5h. </LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Digestion_and_gel-extraction_of_P178.3B_P180">Digestion and gel-extraction of P178; P180</SPAN></H3><P><B>Investigator: MT</B></P><P><B>Aim of experiment:</B> Preparation of P178 and P180 for ligation.
</P><P><B>Procedure:</B></P><UL><LI> Digest P178; P180 with NgoMIV + PstI incubate  1 h at 37 °C</LI><LI> Run 1 % agarose-gel at least 40 min at 100 V</LI><LI> Gel extraction by Quiagen kit.</LI></UL><P><B>Results:</B></P><UL><LI> P178 [NgoMIV +PstI digest] -&gt; F152: 17 ng/µL</LI><LI> P180 [NgoMIV +PstI digest] -&gt; F153: 8.1 ng/µL</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_.26_transformation_of_F131_.2B_F152.3B_F131_.2B_F153">Ligation &amp; transformation of F131 + F152; F131 + F153</SPAN></H3><P><B>Investigator: MT</B></P><P><B>Aim of experiment:</B> Ligation &amp; transformation of F131 + F152; F131 + F153
</P><P><B>Procedure:</B></P><P>Ligation following SOP(13.02.2013) 
</P><UL><LI> F131 + F152 -&gt; F154</LI><LI> F131 + F153 -&gt; F155</LI></UL><P>Transformation following SOP(13.02.2013) 
</P><UL><LI> F154 E. coli. turbo CAM</LI><LI> F155 E. coli. turbo CAM</LI></UL><P><B>Results:</B></P><PRE>  We will see tomorrow, please update this section.
</PRE></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_July_27th"><B>Wednesday, July 27th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Minipreps_of_EGFR-TMD_-_mRuby_-_Strep_-_PolyA_.28F131.2BF152.29_and_EGFR-TMD_-_mRuby_-_Strep_.28F131.2BF153.29">Minipreps of EGFR-TMD - mRuby - Strep - PolyA (F131+F152) and EGFR-TMD - mRuby - Strep (F131+F153)</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Results:</B>
Procedure according to protocol (spin kit). Resulting plasmids:
</P><P>EGFR-TMD - mRuby - Strep - PolyA (F131+F152) -&gt; P196, P197
EGFR-TMD - mRuby - Strep (F131+F153) -&gt; P198, P199
</P><P>Concentrations: 
P196: 228 ng/µl
P197: 212 ng/µl
P198: 197 ng/µl
P199: 322 ng/µl
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_July_28th"><B>Thursday, July 28th</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inoculation_of_Precultures_of_F141new_.28pASK.28Streptactin.29.29">Inoculation of Precultures of F141new (pASK(Streptactin))</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim:</B> Prepare cells for miniprep of plasmids
</P><P><B>Procedure:</B></P><UL><LI> Three colonies of each were picked and inoculated in LB medium (CAM) respectively (AMP) fpr pASK. </LI><LI> Plates were incubated on a shaker at 37°C for 6 hours. </LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="PCR_of_GSY6"><B>PCR of GSY6</B></SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim:</B> Amplification of Streptavidin mutant and Traptactin for further cloning into pASK. 
</P><P><B>Procedure:</B></P><UL><LI>reaction was performed after SOP (PCR Using Q5® High-Fidelity DNA Polymerase, NEB)</LI><LI> 2ng Plasmid were used for amplification (1 µL)</LI><LI> Primers: VR and VF2</LI></UL><UL><LI>PCR program (standard):</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature [°C]</B></TD><TD><B>Time [s]</B></TD></TR><TR><TD>initial Denaturation</TD><TD>98</TD><TD>30
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>66</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>90
</TD></TR><TR><TD>final Extension</TD><TD>72</TD><TD>5
</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>ꝏ
</TD></TR></TBODY></TABLE><UL><LI> PCR product was purified accoding to manufacturer's protocol (PCR puification kit, Qiagen)</LI><LI> DNA was eluted 40 µL EB buffer</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Digestion_of_pASK">Digestion of pASK</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim:</B> Preparing pASK for ligation with Streptavidin mutant and Traptactin.
</P><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>5 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>1,5 µl
</TD><TD>NdeI
</TD></TR><TR><TD>1,5 µl
</TD><TD>HindIII
</TD></TR><TR><TD>22 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Cloning_of_Traptactin_and_Streptavidin_mutant_into_pASK_.28P191.29.">Cloning of Traptactin and Streptavidin mutant into pASK (P191).</SPAN></H3><P><B>Investigator: Luisa</B></P><P><B>Aim</B>: Preperation of the plasmids for Streptavidin and Traptactin expression in E.coli. 
</P><P><B>Procedure</B>: 
</P><UL><LI> Cutting out the crucial bands from the gel containing the PCR-product and the digested plasmid (P190) and purifiing them with the Gelextraction Kit according to the protocoll. <B>First Gel failed: Very small band with wrong length...--&gt; Repetition of digestion with P190.</B></LI></UL><UL><LI> The PCR product of GSY6 was extracted from the gel (Band at ~800 Bp) with the Gelextraction Kit and then digested with NdeI and HindIII to get the fragments Streptavidin m1 and Traptactin. Inactivation was performed at 80°C for 10 min. </LI></UL><UL><LI> Concentrations: 33,9ng/µl (GSY6), </LI></UL><UL><LI> Ligation... </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_NanoLuc-Strep-PolyA_.28F112.29_behind_CMV-IgKappa_in_pSB1C3_.28P182.29._Repetition.">Cloning of NanoLuc-Strep-PolyA (F112) behind CMV-IgKappa in pSB1C3 (P182). Repetition.</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim:</B> See title.  
</P><P><B>Procedure:</B></P><UL><LI>Digestion of P182:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>1,5 µl
</TD><TD>AgeI (20u/µl)
</TD></TR><TR><TD>1,5 µl
</TD><TD>PstI (20u/µl)
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestion at 37°C for 2.5h. </LI><LI> Take care: P182 is not sequenced</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Retrafos_of_P178_.28mRuby-Strep-PolyA.29_and_P181_.28CMV_.2B_IgKappa.29">Retrafos of P178 (mRuby-Strep-PolyA) and P181 (CMV + IgKappa)</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Results:</B> 
Transformation according to protocol, plates were incubated at 37°C overnight. 
</P></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Dialysis_of_eGFP_3">Dialysis of eGFP</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of the experiment:</B> Preparation of eGFP for following biotinylation
</P><P><B>Procedure:</B></P><UL><LI> eGFP was poured in a dialysis hose (cut-off 14 kDa)</LI><LI> The hose was then placed in 4 L ice cold borate buffer 50 mM pH 8.85 (no salt)</LI><LI> The dialysis took place at 4°C over night</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Sequencing_of_P160_.28NanoLuc.2BStrepTag.2BPoly-A.29">Sequencing of P160 (NanoLuc+StrepTag+Poly-A)</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Procedure:</B></P><UL><LI> Sequencing batch were prepared after manufacturer's protocol. (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer)</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_29th"><B>Friday, July 29th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Minipreps_of_EGFR-TMD_-_mRuby_-_Strep_-_PolyA_.28F131.2BF152.29_and_EGFR-TMD_-_mRuby_-_Strep_.28F131.2BF153.29_2">Minipreps of EGFR-TMD - mRuby - Strep - PolyA (F131+F152) and EGFR-TMD - mRuby - Strep (F131+F153)</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Procedure:</B></P><P>Procedure according to protocol (spin kit). Resulting plasmids:
</P><P>CMV+EGFR-SigPep+mRuby3+StrepT+PolyA (F131+F152) -&gt; P200, P201
pASK_T7 RBS_LCN2_His (Vorlage für gewünschte Inserts) -&gt; P202, P203
</P><P><B>Results:</B></P><P>Concentrations: 
P196: 385.3 ng/µl
P197: 462.6 ng/µl
P198:  84.8 ng/µl
P199:  96.0 ng/µl
</P><UL><LI>Send for sequencing:
</LI><LI>P201: FR12904454 (VF2)</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P202_and_P203_.28pASK_T7_RBS_LCN2_His.29">Analytical digestion of P202 and P203 (pASK_T7 RBS_LCN2_His)</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of the experiment: </B> 
Controll whether there is more than one NdeI restriction site. 
</P><P><B>Procedure:</B></P><UL><LI> Analytical digestion:
</LI><LI>The plasmid was digested with NdeI and HindIII</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>1 µl
</TD><TD>CutSmart buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>NdeI
</TD></TR><TR><TD>0.5 µl
</TD><TD>HindIII
</TD></TR><TR><TD>6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI> Two bands were visible what means that there is only one NdeI restriction site. </LI></UL></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Chemical_biotinylation_of_BSA_and_eGFP">Chemical biotinylation of BSA and eGFP</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Procedure:</B></P><P>1 g BSA was chemically biotinylated with a 40x molar excess of biotin-NHS-ester:
</P><UL><LI> 100 ml of 50 mM borate buffer with 50 mM NaCl pH 8.85</LI></UL><UL><LI> dissolve BSA (10 mg/ml)</LI></UL><UL><LI> add adequate amount of biotin-NHS-ester: 205 mg</LI></UL><UL><LI> reaction for approx. 45 minutes at RT after full dissolving of biotin-NHS-ester</LI></UL><UL><LI> store at 4 °C</LI></UL><P>eGFP was chemically biotinylated with a 16x (error in calculations) molar excess of biotin-NHS-ester:
</P><UL><LI> eGFP in 75 ml of 50 mM borate buffer without NaCl pH 8.85</LI></UL><UL><LI> 1to10 dilution: extinction(280) = 0.371</LI></UL><UL><LI> add adequate amount of biotin-NHS-ester: 69 mg</LI></UL><UL><LI> reaction for approx. 45 minutes at RT after full dissolving of biotin-NHS-ester</LI></UL><UL><LI> store at 4 °C</LI></UL><P><B>Next Steps</B></P><UL><LI> add another 104 mg of biotin-NHS-ester incl. reaction at RT tp reach desired 40X molar excess</LI><LI> Purification via column chromatography</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_12.2C_August_1st_-_August_7th">Week 12, August 1st - August 7th</SPAN></H1><DIV class="week" id="Week_12"><H2><SPAN class="mw-headline" id="Monday.2C_August_01"><B>Monday, August 01</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Cloning_of_Streptavidin.2FStreptactin.2FTraptactin_from_GSY5_and_GSY6_into_pASK75_.2B_Transformation">Cloning of Streptavidin/Streptactin/Traptactin from GSY5 and GSY6 into pASK75 + Transformation</SPAN></H3><P><B>Investigator: JB</B></P><UL><LI>PCR products from GSY5 and GSY6 were, after digestion overnight with Nde1, loaded on an 1.8% agarose gel. The fragment bands at roughly 350 bps were extracted.</LI></UL><UL><LI>Digested pASK75 (P202 and P203 dig. with Nde1) were loaded onto an 1% agarose gel, the vector fragments at 3800 bp isolated and purified. </LI></UL><UL><LI>Vector fragment plus Streptavidin fragments were ligated and transformed into XL-1 blue</LI></UL><P>-&gt; F159+F160, F159+F161
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_EGFR-TMD_-_mRuby_-_Strep_with_IRES-BirA">Cloning of EGFR-TMD - mRuby - Strep with IRES-BirA</SPAN></H3><P><B>Investigator: JB</B></P><UL><LI> 2 µg of P77 (EcoRI, XbaI) and P199 (EcoRI, SpeI) were digested for 2 hours at 37°C</LI><LI> Digestions were separated on a 1% agarose gel, fragments corresponding to the vector (P77) and to the to be cloned insert (P199) were isolated and purified</LI><LI> Ligation of inserts for 30 mins at RT</LI><LI> Transformation of ligation into XL1-blue</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Retrafos_of_cell_culture_constructs_for_signal_peptide_testing_for_midipreps">Retrafos of cell culture constructs for signal peptide testing for midipreps</SPAN></H3><P><B>Investigator: JB</B></P><UL><LI> Retransformation of all constructs in XL1-blue</LI><LI> Constructs concerned are: P152, P171, P205 (Nanoluc-constructs) and P201, P208 and P211 (mRuby-constructs)</LI><LI> Transformation according to protocol, plates incubated at 37°C overnight</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_August_02"><B>Tuesday, August 02</B></SPAN></H2><H2><SPAN class="mw-headline" id="Wednesday.2C_August_03"><B>Wednesday, August 03</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Verification_of_cloning_P210_.28CMV-IgKappa-Nanoluc-StrepTag-PolyA.29">Verification of cloning P210 (CMV-IgKappa-Nanoluc-StrepTag-PolyA)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of the experiment:</B> Verification of successful cloning after sequencing showed that the last part of the construct might be missing
</P><P><B>Procedure:</B></P><UL><LI> digestion with AgeI and PstI (sample was digested with EcoRI and PstI for 10 minutes, than inactivated an ligated)</LI></UL><P><B>Results:</B></P><UL><LI> the gel shows a band at about 3500 bp (pSB1C3 + full insert)</LI><LI> gel shows additionally a band below 750 bp (CMV+ IgKappa)</LI><LI> the sample contains the full insert as well as the small insert (CMV+IgKappa without Nanoluc-StrepTag-PolyA)</LI><LI> cloning not succesful</LI><LI> the construct was inocculated from plate again</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_Ligations_F159_.2B_F160_and_F159_.2B_F161_.28pASK_.2B_streptavidin_from_GSY_5_and_6.29">Transformation of <I>E. coli</I> XL1 blue with Ligations F159 + F160 and F159 + F161 (pASK + streptavidin from GSY 5 and 6)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 7 µl of DNA was added to 100 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube.</LI></UL><UL><LI> The cell suspension was plated on ampicillin plates (inclusive rescue plate) and incubated over night at 37 °C in the incubator.</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_04"><B>Thursday, August 04</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="MidiPrep_P152_and_P171_.28pSB1C3.29">MidiPrep P152 and P171 (pSB1C3)</SPAN></H3><P><B>Investigator: EF</B></P><P><B>Aim of the experiment:</B> isolate Plasmids
</P><P><B>Procedure:</B></P><UL><LI> MidiPrep was performed according to manufacturer's protocol (QIAprep MidiPrep, Qiagen). After washing the DNA with 70% Ethanol the pellet was left in special centrifugation tubes (labeled 152+QF; 171+QF) in order to air-dry. EB buffer for storage was later added by CG.</LI><LI> Conc: 1000 ng/ul</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_BAP_Nanoluc_.28P206.29_before_A3C5_.28P213.29">Cloning of BAP Nanoluc (P206) before A3C5 (P213)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Procedure:</B></P><UL><LI> Preparative digestion:</LI></UL><P>P213 (A3C5): EcoRI and NgoMIV -&gt; F170
</P><P>P206 (BAP Nanoluc): EcoRI and AgeI -&gt; F171
</P><P>Both fragments were ligated after gelextraction. 
</P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were taken out of stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 7 µl of Ligation was added to 100 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of 750 µl LB-medium to each tube.</LI></UL><UL><LI> Incubation for 1 hour at 37 °C</LI></UL><UL><LI> The cell suspension was plated on Cam-plates (inclusive rescue plate) and incubated over night at 37 °C in the cell-culture shaker.</LI></UL><P><B>results:</B>  
Gelextraction:
</P><P>the plates showed no colonies. repeat
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_05"><B>Friday, August 05</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Verification_of_successful_cloning_of_all_Streptavidin_variants_into_pASK75">Verification of successful cloning of all Streptavidin variants into pASK75</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Minipreps von pASK75 Streptavidin variants (codon optimzied Streptavidin, Streptactin, Traptactin)
</P><P><B>Procedure:</B></P><UL><LI> MiniPrep was performed according to the manufacturer's protocol (QIAprep MiniPrep, Qiagen) for 3 clones of GSY5 and 6 clones of GSY6</LI></UL><UL><LI> Analytic digestion with 0.25 <I>µ</I>L NdeI, 0.25 <I>µ</I>L HindII, 1 <I>µ</I>L SmartCut Buffer, 200 ng plasmid DNA,  H<SUB>2</SUB>O to 20 µl</LI></UL><UL><LI> 10 <I>µ</I>L on 1% agarose gel electrophoresis</LI></UL><P><B>Results:</B> Successful cloning was verified for all plasmids by analytical digestion. Expected and observed fragment size for the vector was 3.1 kb and about 350 bp for the inserts. 4 plasmids were sent for sequencing (1x GSY5, 3x GSY6). Alle sequences were error-free. Therefore all 3 Streptavidin variants are available now.
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of:_Streptavidin_variants.2C_PAS-Plasmid.2C_Interlab-Plasmids.2C_Ligation_of_EGFR-TMD-mRuby-Stre-IRES-BirA_and_Ligation_of_BAP-Nanoluc-A3C5">Transformation of: Streptavidin variants, PAS-Plasmid, Interlab-Plasmids, Ligation of EGFR-TMD-mRuby-Stre-IRES-BirA and Ligation of BAP-Nanoluc-A3C5</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Transformation of  Streptavidin variants, PAS-Plasmid, Interlab-Plasmids into E. coli BL21 and Ligation of EGFR-TMD-mRuby-Stre-IRES-BirA, Ligation of BAP-Nanoluc-A3C5 in E. coli XL1 blue
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 7 µl of Ligations mixes were added to 50 µl of competent cells and gently mixed, 1 µl for ccc Plasmids</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 1 min. heat shock at 42 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube, and incubation for 1 h at 37°C</LI></UL><UL><LI> The cell suspension was plated on ampicillin plates (inclusive rescue plate) / CamR- plates and incubated for 7 h at 37 °C in the programmed incubator.</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Ligation_of_EGFR-TMD-mRuby-Stre-IRES-BirA_and_Ligation_of_BAP-Nanoluc-A3C5">Ligation of EGFR-TMD-mRuby-Stre-IRES-BirA and Ligation of BAP-Nanoluc-A3C5</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Re-Ligation of EGFR-TMD-mRuby-Stre-IRES-BirA, Re-Ligation of BAP-Nanoluc-A3C5
</P><P><B>Procedure:</B></P><P>The corresponding Fragments were mixed with 2 µl Ligase-buffer, 1 µl T4-Ligase and filled with water up to 20 µl
</P><UL><LI> 7.3 µl F163 + 2.7 µl F162 (=F173)</LI></UL><UL><LI> 3.7 µl F170  + 6.3 µl F171 (=F174)</LI></UL><P>The Ligation mixes were incubated for 1 h at room temperature.
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="SDS_Page_of_Streptavidin_production">SDS Page of Streptavidin production</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Verification of successful Streptavidin production and preperation for the data package
</P><P><B>Procedure:</B>
Streptavidin samples from the production process were further processed with SDS buffer. Some samples had very few volume, therefore the overall volume was reduced. For informations on the samples content please refer to NA. The low volume might be a problem due to evaporation issues.
</P><UL><LI> Sample 1: the tube was empty</LI><LI> Sample 2: 1 µl sample was mixed with 7 µl water and 2 µl SDS buffer</LI><LI> Sample 3: 80 µl sample was mixed with 20 µl SDS buffer</LI><LI> Sample 4: 80 µl sample was mixed with 20 µl SDS buffer</LI><LI> Sample 5: 20 µl sample was mixed with 5 µl SDS buffer</LI><LI> Sample 6: 80 µl sample was mixed with 20 µl SDS buffer</LI><LI> Sample 7: 80 µl sample was mixed with 20 µl SDS buffer</LI><LI> Sample 8: 80 µl sample was mixed with 20 µl SDS buffer</LI><LI> Sample 9: 80 µl sample was mixed with 20 µl SDS buffer</LI></UL><P>10 µl of each sample were loaded on the pre-mixed SDS gel.
</P><P>Gel 1: 8 µl Ladder, samples 1-6
</P><P>Gel 2: 8 µl Ladder, samples 7-9, 5 µl of samples 7-9
</P><P>The gel ran at 160 V for about 1 1/2 h. Afterwards it was incubated for 30 min. in staining solution, 10 min. in destaining solution 1 and over the weekend in destaining solution 2.
</P><P><B>Results:</B></P></DIV></DIV><H1><SPAN class="mw-headline" id="Week_13.2C_August_8th_-_August_14th">Week 13, August 8th - August 14th</SPAN></H1><DIV class="week" id="Week_13"><H2><SPAN class="mw-headline" id="Monday.2C_August_08"><B>Monday, August 08</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR.2C_Purification_and_digestion_of_BirA">PCR, Purification and digestion of BirA</SPAN></H3><P><B>Investigator: LK </B></P><P><B>Aim of the experiment:</B> Amplification of BirA 
</P><P><B>Procedure:</B></P><UL><LI> PCR according to the SOP (primers: O17 and O18, Annealing at 62°C) </LI><LI> PCR purification via the purification kit according to the manufacturer's protocol </LI><LI> preparative digestion with XbaI and PstI over night in a 50µl batch</LI></UL><P><B>Results:'</B></P><UL><LI> Concentration after PCR-purification: 116ng/µl </LI><LI> Gel: One thick band at ~1000Bp --&gt; BirA and one at ~2100Bp --&gt; vector, we're cut out and purified via the Gelextraction kit according to the manufacturer's protocoll. </LI><LI> Concentrations: BirA (F186) : 62,5 ng/µl, pSB1C3 (F187): 32,5 ng/µl </LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Determination_of_Streptavidin_concentration">Determination of Streptavidin concentration</SPAN></H3><P><B>Investigator: NA </B></P><P><B>Procedure:</B></P><UL><LI> OD280 measurement with quartz cuvette </LI></UL><P><B>result:</B> the 1:100 dilution showed an OD of 0,469
</P><P>Calculation with extinction coefficient (expasy tool) of 41940 M <SUP>-1</SUP> * cm <SUP>-1</SUP> and 13.3 kDA as molar mass of streptavidin monomer provides concentration below:
</P><P>14.87 mg/mL
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inocculation_of_Streptavidin_.28wild_type.2C_P235.29_in_M9_medium_for_fermentation">Inocculation of Streptavidin (wild type, P235) in M9 medium for fermentation</SPAN></H3><P><B>Investigator: NA </B></P><P><B>Procedure:</B></P><UL><LI> M9 medium with Amp (1:1000) is inocculated with preculture (~50 mL) of P235 </LI></UL><UL><LI> the culture is incubated in the shaker over night at 37°C</LI></UL></DIV><DIV class="receptor"><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_purification">PCR purification</SPAN></H3><P><B>Investigator: EF </B></P><P><B>Aim of the experiment:</B> Purificate bands for blunt ligation in F113
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Seeding_of_HEK293T_cells_in_cell_culture">Seeding of HEK293T cells in cell culture</SPAN></H3><P><B>Investigator: JB</B></P><P><B>Aim of experiment:</B></P><P>Thawing and seeeding of HEK293T cells for experiments later this week
</P><P><B>Procedure:</B></P><UL><LI> Tube with cells was thawed quickly in a 37°C waterbath</LI></UL><UL><LI> After bringing most of the cell suspension in a fluid state, the suspension was transferred into a 50 ml Falcon tube and suspended in 50 ml of RPMI1640 medium (with 10% CFS and Pen/Strep added). </LI></UL><UL><LI> Cells were spun down for 2 mins at 2,300 rcf and the pellet resuspended in 20 ml of medium in order to remove residual DSMO from the cells (DSMO having been used for freezing the cells).</LI></UL><UL><LI> The resuspended cells were transferred into a culture flask and incubated overnight at 37°C/5% CO².</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_IgKappa_signale_peptide.2C_PAS-Lysin_and_further_constructs_for_the_receptor">Cloning of IgKappa signale peptide, PAS-Lysin and further constructs for the receptor</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of experiment:</B></P><P>Cloning of CMV-IgKappa+Nanoluc-Strep-PolyA, BAP-Nanoluc+A3C5, IRES-BirA + polyA and Sumo-PAS-Lysin into pASK75
</P><P><B>Procedure:</B></P><P>Digestion of 3 µg of each plasmid in 30 µl mix:
</P><UL><LI> 20 µl P205 (CMV-IgKappa) with EcoRI, AgeI -&gt; Gel 1, left lane, expected fragment at about 600 bp -&gt; F175</LI><LI> 15 µl P160 (Nanoluc-Strep-PolyA) with EcoRI, NgoMiV -&gt; Gel 1, right lane, expected fragment at about 3000 bp -&gt; F176</LI><LI> 20 µl P212 (A3C5) with EcoRI, NgoMiV -&gt; Gel 2, left lane, expected fragment at about 2.200 bp -&gt; F177</LI><LI> 10 µl P206 (BAP-Nanoluc) with EcoRI, AgeI -&gt; Gel 2, right lane, expected fragment at about 500 bp -&gt; F178</LI><LI> 10 µl P199 (EGFR-TMD-mRuby-Strep) with EcoRI, SpeI -&gt; Gel 3, left lane, expected fragment at about 1.500 bp -&gt; F179</LI><LI> 20 µl P236 (PAS-Lysin-Sumo) with NdeI, HindIII -&gt; Gel 3, right lane, expected fragment at about 1.000 bp -&gt; F180</LI><LI> 10 µl P77 (IRES-BirA-KDEL) with EcoRI, SpeI -&gt; Gel 4, left lane, expected fragment at about 1000 bp -&gt; F181</LI></UL><P>Ligation in 20 µl mix (s. above fragment names):
</P><UL><LI> F182: 2.9 µl F176 + 7.1 µl F175</LI><LI> F183: 5.2 F177 + 4.8. F178</LI><LI> F184: 7.0 µl F159 + 3.0 µl F180</LI><LI> F185: 2.9 µl F133 + 7.1 µl F181</LI></UL><P>Transformation according to SOP.
</P><P><B>Results:</B></P><P>For all ligations colonies grew the next day.
</P><P>Gel 1:
</P><P>Gel 2:
</P><P>Gel 3:
</P><P>Gel 4:
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_August_09"><B>Tuesday, August 09</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_Biobricks_in_NEB_Turbo">Transformation of Biobricks in NEB Turbo</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of the experiment:</B> Acquisition of pBAD/AraC and Lox site
</P><P><B>Procedure:</B></P><UL><LI> resuspend DNA with 10 µl ddH2O in well of interest (standard distribution kit) </LI><LI> incubate for 5 min</LI><LI> transformation of 1 µl plasmid according to SOP in NEB Turbo</LI></UL><P>Used bricks:
BBa_I0500 and BBa_J61046
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_and_induction_of_AraC.2BpBAD_controlled_eGFP_production_in_E.coli_BL-21">Inoculation and induction of AraC+pBAD controlled eGFP production in E.coli BL-21</SPAN></H3><P><B>Investigator: LK </B></P><P><B>Aim of the experiment:</B> Confirmation of arabinose dependend activation of the pBAD-promotor in AraC-Operon. 
</P><P><B>Procedure:</B></P><UL><LI> E.coli Bl-21 with P61 (PC+AraC+pBAD+eGFP in pSB1C3) we're picked and a 4ml preculture (LB+cam) was inoculated and incubated for 6h. </LI><LI> According to the following table 50ml cultures (LB+cam) we're incoculated with 500µl of the preculture and incubated over night at 37°C. </LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Arabinose concentration:</B></TD><TD>negative control
</TD><TD>100mM
</TD><TD>10mM
</TD><TD>1mM
</TD><TD>100µM
</TD><TD>10µM
</TD></TR></TBODY></TABLE><P><B>Results:</B></P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_F186_.28BirA.29_into_F187_.28pSB1C3.29">Cloning of F186 (BirA) into F187 (pSB1C3)</SPAN></H3><P><B>Investigator: LK </B></P><P><B>Aim of the experiment:</B> Getting the BirA for the prokaryotic receptor. 
</P><P><B>Procedure:</B></P><UL><LI> 1,7µl of F187 and 6,3µl of F186 we're ligated for 1h at RT accoridng to the SOP. --&gt; F189</LI><LI> The ligation-batch was transformed into NEB-Turbo cells and plated out on cam-plates. Incubation at 37° o/n. </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Inoculation_of_precultures_from_Ligations_F182-F185">Inoculation of precultures from Ligations F182-F185</SPAN></H3><P><B>Investigator: LK </B></P><P><B>Aim of the experiment:</B> Getting plamids of CMV-IgKappa+Nanoluc-Strep-PolyA (F182), BAP-Nanoluc+A3C5 (F183),Sumo-PAS-Lysin (F184) and IRES-BirA + polyA (F185)
</P><P><B>Procedure:</B></P><UL><LI> Inoculation of 4mL cultures (LB cam resp. amp (F184)). Incubation at 37°C o/n. </LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inocculation_of_fermenter">Inocculation of fermenter</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> streptavidin production
</P><P><B>Procedure:</B></P><UL><LI> centrifuge preculture (P235 in M9 medium)</LI><LI> resuspend pellet in HCDF medium and inocculate fermenter (HCDF medium)</LI><LI> pO<SUB>2</SUB>= 60 %; Temp= 30°C, pH~7</LI><LI> growth over night</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="SDS_Page_of_Streptavidin_production_2">SDS Page of Streptavidin production</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Verification of successful Streptavidin production and preperation for the data package
</P><P><B>Procedure:</B>
Streptavidin samples from the production process were further processed with SDS buffer. Some samples had very few volume, therefore the overall volume was reduced.
</P><UL><LI> 1: 8 µl Ladder</LI><LI> 2: 10 µl after induction (1 µl sample was mixed with 7 µl water and 2 µl SDS buffer)</LI><LI> 3: 5 µl supernantant after denaturation (20 µl sample was mixed with 20 µl H<SUB>2</SUB>O and 10 µl SDS buffer)</LI><LI> 4: 5 µl before precipitation(80 µl sample was mixed with 20 µl SDS buffer)</LI><LI> 5: 10 µl supernantant after 40% precipitation (80 µl sample was mixed with 20 µl SDS buffer)</LI><LI> 6: 10 µl supernantant after 70% precipitation (80 µl sample was mixed with 20 µl SDS buffer)</LI><LI> 7: 10 µl after dissolving in 2.2M (NH<SUB>4</SUB>)2SO<SUB>4</SUB> (80 µl sample was mixed with 20 µl SDS buffer)</LI></UL><P>
The gel ran at 145 V
</P><P><B>Results:</B></P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_F192_and_P238">Transformation of F192 and P238</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.</LI></UL><UL><LI> 7 µl of DNA was added to 50 µl of competent cells and gently mixed.</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min. heat shock at 37 °C</LI></UL><UL><LI> Adding of 950 µl LB-medium to each tube.</LI></UL><UL><LI> The cell suspensions were plated on Cam-plates (inclusive rescue plate) and incubated over night at 37 °C.</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_August_10"><B>Wednesday, August 10</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Minipreps_and_analytical_digestion_of_P237_-_P243">Minipreps and analytical digestion of P237 - P243</SPAN></H3><P><B>Investigator:</B> LK 
</P><P><B>Aim of the experiment:</B> Isolating and verifiing plamids of CMV-IgKappa+Nanoluc-Strep-PolyA (F182), BAP-Nanoluc+A3C5 (F183),Sumo-PAS-Lysin (F184) and IRES-BirA + polyA (F185)
</P><P><B>Procedure:</B></P><UL><LI> Minipreps were performed according to the manufacturer's protocol. </LI><LI> ~200- 500ng of ach Plasmid (accept P241) were digested with EcoI and PstI for 45 min. and a gel-elctrophoresis (1% Agarose) of the digestion-batches was performed.</LI></UL><P><B>Results:</B></P><PRE>* P237: expected band for CMV-IgKappa+Nanoluc-Strep-PolyA at ~1850 Bp , but unexpected band at ~1100. Discard sample? 
* P238: expected band for CMV-IgKappa+Nanoluc-Strep-PolyA at ~1850 Bp 
* P239 and P240: expected band at ~650 Bp 
* P242 and P243: bands at ~2000 Bp. Digested vector and Fragment are not distinguishable --&gt; sequencing 
</PRE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_constructs_for_the_eukaryotic_and_prokaryotic_receptor">Cloning constructs for the eukaryotic and prokaryotic receptor</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of experiment:</B></P><P>Cloning of EGFR-TMD-mRuby-Strep-IRES-BirA-KDEL-PolyA, BAP-Nanoluc-A3C5-EspP-2xTerm
</P><P><B>Procedure:</B></P><P>Digestion of 3 µg of each plasmid in 30 µl mix:
</P><UL><LI> 6 µl P242 (IRES-BirA-KDEL-PolyA) with EcoRI, XbaI -&gt; Gel 1, left lane, expected fragment at about 4000 bp -&gt; F190</LI><LI> 20 µl P108 (EspP-Termin.) with EcoRI, NgoMiV -&gt; Gel 1, right lane, expected fragment at about 3000 bp -&gt; F191</LI></UL><P>Ligation in 20 µl mix (s. fragment names above):
</P><UL><LI> F192: 4 µl F190 + 6 µl F179</LI></UL><P>
Transformation according to SOP.
</P><P><B>Results:</B></P><P>
Gel 1:
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_P22.3B_P67.3B_P122.3B_P137.3B_P160">Transformation of P22; P67; P122; P137; P160</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><P>Amplification of P22; P67; P122; P137; P160 
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD><B>Plasmid</B></TD><TD><B>Description</B></TD></TR><TR><TD>P22
</TD><TD>K157001
</TD><TD>RFP-Gen.(Retrafo P4)
</TD></TR><TR><TD>P67
</TD><TD>pSB1C3
</TD><TD>EGFR signal peptide (transloc. pore)
</TD></TR><TR><TD>P122
</TD><TD>pSB1C3
</TD><TD>Ig Kappa (digest. GSY3)
</TD></TR><TR><TD>P137
</TD><TD>pSB1C3
</TD><TD>T7/BM40 aus GSY
</TD></TR><TR><TD>P160
</TD><TD>pSB1C3
</TD><TD>NanoLuc + StrepTag + Poly-A
</TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> Transformation following SOP(13.02.2013) </LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Fermentation">Fermentation</SPAN></H3><P><B>Investigator:</B> NA
</P><P><B>Aim of the experiment:</B> Streptavidin production
</P><P><B>Procedure:</B></P><UL><LI> exponential feed (glucose, MgSO<SUB>4</SUB>, Thiamin, Amp, HCDF salt, micronutrients) for ~13 h (about 3-4 L)</LI><LI> Induction of streptavidin expression (anhydro tetracyclin) at OD= 111.2</LI><LI> production over night</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_11"><B>Thursday, August 11</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Minipreps_and_analytical_digestion_of_BBa_I0500_.28AraC.2B_pBAD.29.2C_BBa_J61046_.28LoxP.29_and_F189_.28BirA.29">Minipreps and analytical digestion of BBa_I0500 (AraC+ pBAD), BBa_J61046 (LoxP) and F189 (BirA)</SPAN></H3><P><B>Investigator:</B> LK
</P><P><B>Aim of the experiment:</B> Isolating and verifiing the plasmids. 
</P><P><B>Procedure:</B></P><UL><LI> The Preperation was performed according to the manufacturer's protocol. </LI><LI> Each plasmid was digested with EcoRI and PstI and a gel-elctrophoresis (1% Agarose) of the digestion-batches was performed.</LI></UL><P><B>Results:</B></P><UL><LI> BBa_I0500 (AraC+ pBAD) = P246 and P247: Band at ~1250; Cloning was succesful</LI><LI> BBa_J61046 (LoxP) = P244 and P245: Fragment is too small (34Bp), not visible on gel. </LI><LI> F189 (BirA) = P248: Band at ~ 1000Bp; Cloning was succesful </LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Cloning_of_Streptavidin-variants_into_pASK111">Cloning of Streptavidin-variants into pASK111</SPAN></H3><P><B>Investigator: JB, LK, CG</B></P><P><B>Aim of experiment:</B></P><P>Cloning of Streptactin, Traptavidin, Streptavidin into CamR pASK111
</P><P><B>Procedure:</B></P><P>Digestion of about 2.5 µg of each plasmid in 30 µl mix:
</P><UL><LI> 25 µl P233 (Streptavidin) with XbaI, HindIII  -&gt; Gel 1, left lane, expected fragment at about 400 bp -&gt; F195</LI><LI> 25 µl P231 (Traptavidin) with XbaI, HindIII  -&gt; Gel 1, right lane, expected fragment at about 400 bp -&gt; F196</LI><LI> 15 µl P241 (PAS-Lysin) with XbaI, HindIII  -&gt; Gel 2, left lane, expected fragment at about 400 bp -&gt; -</LI><LI> 25 µl P229 (Streptactin) with XbaI, HindIII  -&gt; Gel 2, left right, expected fragment at about 400 bp -&gt; F197</LI><LI> 15 µl P249 (pASK111) with XbaI, HindIII -&gt; Gel 3, left lane, expected fragment at about 3000 bp -&gt; F198</LI></UL><P>Ligation in 20 µl mix overnight (s. fragment names above):
</P><UL><LI> F199: 3.7 µl F195 + 6.3 µl F198</LI><LI> F200: 5.4 µl F196 + 4.6 µl F198</LI><LI> F201: 4.1 µl F197+ 5.9 µl F198</LI></UL><P>
Transformation according to SOP.
</P><P><B>Results:</B></P><P>
Gel 1:
</P><P>Gel 2:
</P><P>Gel 3:
</P></DIV><DIV class="polymerization"><H3><SPAN class="mw-headline" id="Testpolymerization:_BSA-coating">Testpolymerization: BSA-coating</SPAN></H3><P><B>Investigator: EU, JH</B></P><P><B>Aim of experiment:</B></P><P>Coating of 6-well plate with BSA according to a modified ELISA-protocol. 
</P><P><B>Procedure:</B></P><P>Bicarbonate/carbonate coating buffer (100mM):
30,3 mg Na2CO3
60 mg NaHCO3
10 ml distilled water
</P><P>&quot;Blocking&quot; Buffer:
1% BSA in PBS
</P><P>Each well was covered with 0,5 ml BSA buffer and 0,5 ml bicarbonate buffer. After overnight incubation at RT, solution in wells was aspirated one by one and streptavidin was added. After a short incubation, the streptavidin(&amp;Tris/HCl) solution was aspirated and Biotin solution was added (by needle). 
</P><P>1. well: 200 µl Streptavidin (11,2 mg/ml) + 200 µl Biotin, 2. well: 400 µl Streptavidin and 400 µl Tris/HCl + 200 µl Biotin, 3. well: 500 µl Streptavidin and 500 µl Tris/HCl + 300 µl Biotin, 4. well: 500 µl Streptavidin + 10 µl Biotin (this time with 2,5 µl pipette)
</P><P><B>Results:</B></P><P>Overall polymerization was observed again, although it seemed more like an aggregation than a polymerization.
No coating was observed except in the 4. well. Biotin-injection speed may be the critical point.
</P><P>Poly-D-Lysine coating will follow as next step.
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_P64.3B_P73">Transformation of P64; P73</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><P>Amplification of P22; P67; P122; P137; P160 
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD><B>Plasmid</B></TD><TD><B>Description</B></TD></TR><TR><TD>P64
</TD><TD>pSB1C3
</TD><TD>VEGF-p2a-PDGF
</TD></TR><TR><TD>P73
</TD><TD>pSB1C3
</TD><TD>Poly-A
</TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> Transformation following SOP(13.02.2013) </LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Over_night_liquid_culture_of_P22.3B_P67.3B_P122.3B_P137.3B_P160">Over night liquid culture of P22; P67; P122; P137; P160</SPAN></H3><P><B>Investigator: EF, MP </B></P><P><B>Aim of the experiment:</B></P><P>Getting enough cellmass for plasmid preparation
</P><P><B>Procedure:</B></P><UL><LI> Inoculation of 5 mL LB-medium (containing antibiotica) with colonies of foregone transformation</LI><LI> Incubate over night at 37 °C in shaker</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="harvesting_streptavidin">harvesting streptavidin</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> streptavidin production
</P><P><B>Procedure:</B></P><UL><LI> OD= 174</LI><LI> centrifuge culture</LI><LI> freeze pellet</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="SDS_PAGE_Streptavidin_fermentation">SDS PAGE Streptavidin fermentation</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Verification of successfull production
</P><P><B>Procedure:</B></P><UL><LI> according to protocol</LI><LI> 8 µl Ladder; before induction; after induction</LI><LI> both samples: 3 µl culture + 77 µl TB + 20 µl SDS buffer; 12 µl on SDS gel</LI></UL><P><B>Results:</B></P><UL><LI>The fermentation was not successful, streptavidin band (unter 15 kDa marker) is just a bit thicker. The reason might be the ampR of the transformed plasmid. Due to the high cell concentration beta lactamase (secreted in periplasma) gets in the medium and hydrolyses amp. As a result cells lose plasmid because there is no selective pressure without an antibiotic in medium.</LI></UL><UL><LI> next step: cloning pf streptavidin mutants into pASK111 (camR)</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P238.2C_P245.2C_P248">Sequencing of P238, P245, P248</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Procedure:</B></P><UL><LI> Sequencing batch were prepared after manufacturer's protocol. (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer) - all samples: VF2</LI><LI> barcodes in stock list</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_12"><B>Friday, August 12</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_of_Gensythesis_.28GSY.29_7_and_8">PCR of Gensythesis (GSY) 7 and 8</SPAN></H3><P><B>Investigator: MT</B></P><P><B>Aim of the experiment:</B></P><P>Amplification of GSY 7 and 8
</P><P><B>Procedure:</B></P><UL><LI>PCR Using Q5® High-Fidelity DNA Polymerase (M0491) according to online protocol</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>25 µL reaction</B></TD><TD><B>Final concentration</B></TD></TR><TR><TD>5X Q5 Reaction Buffer
</TD><TD>5 µL
</TD><TD>1 X
</TD></TR><TR><TD>10 mM dNTPs
</TD><TD>0.5 µL
</TD><TD>200 μM
</TD></TR><TR><TD>10 μM Forward Primer (D20)
</TD><TD>1.25 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>10 μM Reverse Primer (VR)
</TD><TD>1.25 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>Template DNA
</TD><TD>1 µL
</TD><TD>10 ng
</TD></TR><TR><TD>Q5 High-Fidelity DNA Polymerase
</TD><TD>0.25 μl
</TD><TD>0.02 U/μl
</TD></TR><TR><TD>Nuclease-Free Water
</TD><TD>to 25 μl
</TD></TR></TBODY></TABLE><P>Thermocycling Conditions for a Routine PCR:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>STEP</B></TD><TD><B>TEMP</B></TD><TD><B>TIME</B></TD></TR><TR><TD>Initial Denaturation
</TD><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98 °C</TD><TD>5 - 10 sec
</TD></TR><TR><TD>61 °C
</TD><TD>30 sec
</TD></TR><TR><TD>72 °C
</TD><TD>60 sec
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>120 sec
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD></TR></TBODY></TABLE><P>Note: Use of the NEB T m Calculator is highly recommended.
</P><P><B>Results:</B></P><P>After running on 1 %agarose gel only the 2kb bands belonging to GSY8 could be distinguished. These were extracted from the gel. 
</P><P>
-&gt; Repeat PCR for GSY7
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Gelextraction_of_GSY8">Gelextraction of GSY8</SPAN></H3><P><B>Investigator: EF</B></P><P><B>Aim of the experiment:</B> Gelextraction of 2kb band of PCR belonging to GSY8
<B>Procedure:</B></P><P>Gelextraction was performed by manufacturers protocol (SLG).
</P><P><B>Concentration:</B></P><P>7,7 ng/ul
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Plasmid_preparation_of_liquid_culture_of_P22.3B_P67.3B_P122.3B_P137.3B_P160">Plasmid preparation of liquid culture of P22; P67; P122; P137; P160</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Purification of P22; P67; P122; P137; P160  
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD><B>Plasmid</B></TD><TD><B>Description</B></TD></TR><TR><TD>P22
</TD><TD>K157001
</TD><TD>RFP-Gen.(Retrafo P4)
</TD></TR><TR><TD>P67
</TD><TD>pSB1C3
</TD><TD>EGFR signal peptide (transloc. pore)
</TD></TR><TR><TD>P122
</TD><TD>pSB1C3
</TD><TD>Ig Kappa (digest. GSY3)
</TD></TR><TR><TD>P137
</TD><TD>pSB1C3
</TD><TD>T7/BM40 aus GSY
</TD></TR><TR><TD>P160
</TD><TD>pSB1C3
</TD><TD>NanoLuc + StrepTag + Poly-A
</TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> According to Mini-prep kit, SLG</LI></UL><P><B>Results:</B></P><UL><LI> P22 -&gt;P1001 103.6 ng/µL</LI><LI> P67 -&gt;P1004 92.9 µL</LI><LI> P122 -&gt; P1000 80.8 ng/µL</LI><LI> P137 -&gt; P1003 87.9 µL</LI><LI> P160 -&gt; P1002 100 ng/µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Dephosphorylation_of_F113.5BHincII_digested.5D">Dephosphorylation of F113[HincII digested]</SPAN></H3><P><B>Investigator: MP, EF </B></P><P><B>Aim of the experiment:</B></P><P>Dephosphorylation of F113[HincII digested] (RFP-Reportercassette )
</P><P><B>Procedure:</B>
According to NEB online Protocol for Dephosphorylation of 5 ́-ends of
DNA using Antarctic Phosphatase (M0289):
</P><UL><LI> 12 µL F113 (85.1 ng/µL)</LI><LI> 2 µL Antarctic Phosphatase Rxn Buffer (10X)</LI><LI> 1 µL Antarctic Phosphatase (5000 u/mL)</LI><LI> 5 µL ddH2O</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Midiprep_P238_.28CMV-IgKappa-Nanoluc-Strep-polyA.29">Midiprep P238 (CMV-IgKappa-Nanoluc-Strep-polyA)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Procedure:</B></P><UL><LI> MidiPrep was performed according to manufacturer's protocol (QIAprep MidiPrep, Qiagen)</LI></UL><P><B>Results:</B></P><P>P253 - concentration: 2134 ng/µl
</P></DIV><DIV class="hypoxia"><PRE>=== Digestion, Gel extraction and ligation of F1000 (P122), F1001 (P67), F1002 (P160) and F1003 (P137) ===
</PRE><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><P>Preparation of plasmids
</P><P><B>Procedure:</B></P><UL><LI> 33 µL DNA</LI><LI> 1 µL enzyme 1</LI><LI> 1 µL enzyme 2</LI><LI> 5 µL cut smart buffer</LI><LI> 10 µL ddH20</LI></UL><P>Enzymes used: 
</P><UL><LI> F1000: AgeI + PstI</LI><LI> F1001: AgeI + PstI</LI><LI> F1002: NgoMiT + PstI</LI><LI> F1003: AgeI + PstI</LI></UL><P>Gelextraction:
</P><UL><LI> Using gelextraction kit, Quiagen</LI></UL><P>Ligation
</P><UL><LI> 2µl F1000 + 6µl F1002 </LI><LI> 2µl F1001 + 6µl F1002</LI><LI> 2µl F1003 + 6µl F1002</LI></UL><P>+ 1µl T4 Ligase and 1 l T4 ligase buffer for each ligation
</P><P><B>Results:'</B></P><P>Left to right F1002(2x), F1001(2x), F1000(2x) and F1003(1x)
</P><UL><LI> F1000 30 ng/µL </LI><LI> F1001 23,8 ng/µL</LI><LI> F1002 9,4 ng/µL </LI><LI> F1003 23,1 ng/µL</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_August_13"><B>Saturday, August 13</B></SPAN></H2><DIV class="linkers"><H3><SPAN class="mw-headline" id="Miniprep_of_pASK111_.28empty.29_and_diegestion">Miniprep of pASK111 (empty) and diegestion</SPAN></H3><P><B>Investigator:</B> LK
</P><P><B>Aim of the experiment:</B> Amplification of pASK111 and digestion for further ligation with SUMO-PAS-Lysin. 
</P><P><B>Procedure:</B></P><UL><LI> The Miniprep was performed according to the manufacturer's protocol.</LI><LI> The whole DNA was diegested with XbaI and HindIII. Over night at 37°C. </LI><LI> Gelelectrophoresis was performed and showed a significant band at ~2700.</LI></UL><P><B>Comment</B></P><UL><LI> Gel-ectraction has yet to be done . </LI><LI> equally digested insert from P236 was visible on gel. --&gt; no ligation possible.</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_F1006.3B_F1007.3B_F1008">Transformation of F1006; F1007; F1008</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><P>Amplification of F1006; F1007; F1008
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD><B>Ligation</B></TD><TD><B>Plasmid</B></TD><TD><B>Description</B></TD></TR><TR><TD>F1006
</TD><TD>F1000 + F1002
</TD><TD>pSB1C3
</TD><TD>Ig Kappa  + NanoLuc + StrepTag + Poly-A
</TD></TR><TR><TD>F1007
</TD><TD>F1003 + F1002
</TD><TD>pSB1C3
</TD><TD>T7/BM40 + NanoLuc + StrepTag + Poly-A
</TD></TR><TR><TD>F1008
</TD><TD>F1001 + F1002
</TD><TD>pSB1C3
</TD><TD>EGFR + NanoLuc + StrepTag + Poly-A
</TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> Transformation following SOP(13.02.2013) </LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_and_gelextraction_of_P64.3B_P73">Digestion and gelextraction of P64; P73</SPAN></H3><P><B>Investigator: JB, MT </B></P><P><B>Aim of the experiment:</B></P><P>Preparation for ligation
</P><P><B>Procedure:</B></P><UL><LI> 10 µL DNA</LI><LI> 1 µL enzyme 1</LI><LI> 1 µL enzyme 2</LI><LI> 2 µL buffer</LI><LI> 6 µL ddH20</LI></UL><P>Enzymes used: 
</P><UL><LI> P64: EcoRI + SpeI</LI><LI> P73: EcoRI + XbaI</LI></UL><P>Gelextraction:
</P><UL><LI> Using gelextraction kit, Quiagen</LI></UL><P><B>Results:'</B></P><P>Left: P64 Right: P73
</P><UL><LI> P64 -&gt; F1011 14,2 ng/µL </LI><LI> P73 -&gt; F1012 17,6 ng/µL</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_August_14"><B>Sunday, August 14</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Miniprep_of_P199-P201_and_analytical_digestion">Miniprep of P199-P201 and analytical digestion</SPAN></H3><P><B>Investigator:</B> LK 
</P><P><B>Aim of the experiment:</B> Isolation of &quot;Streptavidin-variants&quot; in pASK111 for preinoculation tomorrow. 
</P><P><B>Procedure:</B></P><UL><LI> The Miniprep was performed according to the manufacturer's protocol.</LI><LI> ~200ng where digested with XbaI and HindIII. </LI></UL><P><B>Results:</B></P><UL><LI> All samples showed significant bands at ~400 Bp. </LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Transformation_of_P199-201_into_E.coli_BL-21">Transformation of P199-201 into E.coli BL-21</SPAN></H3><P><B>Investigator:</B> LK 
</P><P><B>Aim of the experiment:</B> Introducing Strep-variants in pASK111 into BL-21 for further fermentation. 
</P><P><B>Procedure:</B></P><UL><LI> The cells were transformed according to the SOP with 1µl of DNA and after 1h of incubation in LB without antibiotics plated out on Cam-plates. </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_P113-P213_and_analytical_digestion">Miniprep of P113-P213 and analytical digestion</SPAN></H3><P><B>Investigator:</B> LK 
<B>Aim of the experiment:</B> Isolation of &quot;Streptavidin-variants&quot; in pASK111 for preinoculation tomorrow. 
</P><P><B>Procedure:</B></P><UL><LI> The Miniprep was performed according to the manufacturer's protocol.</LI><LI> ~200 ng were digested with EcoRI and PstI.</LI></UL><P><B>Results:</B></P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_of_Gensynthesis_.28GSY.29_7.2C_8">PCR of Gensynthesis (GSY) 7, 8</SPAN></H3><P><B>Investigator: MT</B></P><P><B>Aim of the experiment:</B></P><P>Amplification of GSY7 
</P><P><B>Procedure:</B></P><UL><LI> PCR Using Q5® High-Fidelity DNA Polymerase (M0491)</LI><LI> Different amounts of template DNA were used</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>50 µL reaction</B></TD><TD><B>Final concentration</B></TD></TR><TR><TD>5X Q5 Reaction Buffer
</TD><TD>10 µL
</TD><TD>1 X
</TD></TR><TR><TD>10 mM dNTPs
</TD><TD>1 µL
</TD><TD>200 μM
</TD></TR><TR><TD>10 μM Forward Primer (D20)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>10 μM Reverse Primer (VR)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>Template DNA
</TD><TD>1 µL
</TD><TD>1 - 10 ng
</TD></TR><TR><TD>Q5 High-Fidelity DNA Polymerase
</TD><TD>0.5 μl
</TD><TD>0.02 U/μl
</TD></TR><TR><TD>Nuclease-Free Water
</TD><TD>to 50 μl
</TD></TR></TBODY></TABLE><P>Thermocycling Conditions for a Routine PCR:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>STEP</B></TD><TD><B>TEMP</B></TD><TD><B>TIME</B></TD></TR><TR><TD>Initial Denaturation
</TD><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD rowspan="3">40 cycles</TD><TD>98 °C</TD><TD>5 - 10 sec
</TD></TR><TR><TD>58 °C
</TD><TD>30 sec
</TD></TR><TR><TD>72 °C
</TD><TD>60 sec
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>120 sec
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI>PCR GSY8 successful. </LI></UL><P>-&gt; Do it again for GSY 7!
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Over_night_liquid_culture_of_F1006.3B_F1007.3B_F1008">Over night liquid culture of F1006; F1007; F1008</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><P>Getting enough cellmass for plasmid preparation
</P><P><B>Procedure:</B></P><UL><LI> Inoculation of 5 mL LB-medium (containing antibiotica) with colonies of foregone transformation</LI><LI> Incubate over night at 37 °C in shaker</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Gelextaction_of_F1004">Gelextaction of F1004</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI> Using gelextraction kit, Quiagen</LI></UL><P><B>Results:'</B></P><UL><LI> long F1004 -&gt; F1013 6,4 ng/µL</LI><LI> short F1004 -&gt; F1014 1,1 ng/µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation_F1012_.2B_F1011">Ligation F1012 + F1011</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><P>Ligation F1012 + F1011
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD><B>Plasmind</B></TD><TD><B>Description</B></TD></TR><TR><TD>F1011
</TD><TD>VEGF-p2a-PDGF
</TD></TR><TR><TD>F1012
</TD><TD>pSB1C3
</TD><TD>Poly-A
</TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> 3 µL F1012</LI><LI> 6 µL F1011</LI><LI> 2 µL buffer</LI><LI> 1 µL T4 Ligase</LI><LI> 8 µL ddH2O</LI></UL><P><B>Results:</B></P><UL><LI> F1015</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_F1015">Transformation of F1015</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><P>Amplification of F1015
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD><B>Ligation</B></TD><TD><B>Plasmid</B></TD><TD><B>Description</B></TD></TR><TR><TD>F1015
</TD><TD>F1011 + F1012
</TD><TD>pSB1C3
</TD><TD>VEGF-p2a-PDGF + poly-A
</TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> Transformation following SOP(13.02.2013) </LI></UL><P><B>Results:</B></P><UL><LI> No colonies. Do it again.</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_14.2C_Aug_15th_-_Aug_21st">Week 14, Aug 15th - Aug 21st</SPAN></H1><DIV class="week" id="Week_14"><H2><SPAN class="mw-headline" id="Monday.2C_Aug_15th"><B>Monday, Aug 15th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P251_and_P252">Analytical digestion of P251 and P252</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment:</B> Verify cloning
</P><P><B>Procedure:</B></P><UL><LI> 1µL Plasmid was digested with 1µL of EcoRI and PstI</LI><LI> Digestions were incubated for 1h</LI><LI> Samples were seperated by gelelectrophoresis on a 1% agarose gel </LI></UL><P><B>Results:</B></P><UL><LI> Cloning seems to have worked</LI></UL><P><A href="/File:Muc16_P251_P252.png" title="File:Muc16 P251 P252.png">500px</A></P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Plasmidpreparation_of_liquid_culture_of_F1006.3B_F1007.3B_F1008">Plasmidpreparation of liquid culture of F1006; F1007; F1008</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Purification of F1006; F1007; F1008
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD><B>Plasmid</B></TD><TD><B>Description</B></TD></TR><TR><TD>F1006
</TD><TD>pSB1C3
</TD><TD>Ig Kappa  + NanoLuc + StrepTag + Poly-A
</TD></TR><TR><TD>F1007
</TD><TD>pSB1C3
</TD><TD>T7/BM40 + NanoLuc + StrepTag + Poly-A
</TD></TR><TR><TD>F1008
</TD><TD>pSB1C3
</TD><TD>EGFR + NanoLuc + StrepTag + Poly-A
</TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> According to Mini-prep. kit, Quiagen</LI></UL><P><B>Results:</B></P><UL><LI> F1006 -&gt; F1005 83.2 ng/µL</LI><LI> F1007 -&gt;  F1006 91.8 ng/µL</LI><LI> F1008 -&gt; F1007 367.1 ng/µL</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Cloning_of_Nanoluc-Strep_in_front_of_double_terminator_and_PAS-Lysine_into_pASK111">Cloning of Nanoluc-Strep in front of double terminator and PAS-Lysine into pASK111</SPAN></H3><P><B>Investigator: LK, CG</B></P><P><B>Aim of experiment:</B></P><P>Cloning of Nanoluc-Strep in front of double terminator and PAS-Lysine into pASK111
</P><P><B>Procedure:</B></P><P>Plasmid preperation according to SOP of P264, P265
</P><P>Digestion of about 3 µg of each plasmid in 30 µl mix:
</P><UL><LI> 6 µl P29 (double teminator) with EcoRI, XbaI  -&gt; Gel 1, left lane, expected fragment at about 3000 bp -&gt; F207</LI><LI> 15 µl P98 (Nanoluc-Strep) with EcoRI, SpeI  -&gt; Gel 1, right lane, expected fragment at about 500 bp -&gt; F208</LI><LI> 15 µl P264 (PAS-Lysin) with XbaI, HindIII  -&gt; Gel 2, left lane, expected fragment at about 1000 bp -&gt; F210</LI></UL><P>Ligation in 20 µl mix for 30 min. at room temperature (s. fragment names above):
</P><UL><LI> F209: 3.7 µl F207 + 6.3 µl F208</LI><LI> F211: 1.1 µl F210 + 8.9 µl F206</LI></UL><P>
Transformation according to SOP.
</P><P><B>Results:</B></P><P>
Gel 1:
</P><P>Gel 2:
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_Aug_16th"><B>Tuesday, Aug 16th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Over_night_liquid_culture_of_P64.3B_P73">Over night liquid culture of P64; P73</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Getting enough cellmass for plasmid preparation
</P><P><B>Procedure:</B></P><UL><LI> Inoculation of 5 mL LB-medium (containing antibiotica) with colonies of foregone transformation</LI><LI> Incubate over night at 37 °C in shaker</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Gelextraction_of_PCR_GSY8.28Saturday.2C_August_13.29">Gelextraction of PCR GSY8(Saturday, August 13)</SPAN></H3><P><B>Investigator: EF </B></P><P><B>Aim of the experiment:</B></P><P>Purification of PCR GSY8(Sunday, August 14)
</P><P><B>Procedure:</B></P><UL><LI> Using gelextraction kit, SLG</LI></UL><P><B>Results:</B></P><UL><LI> F1016 3,4 ng/µL</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Seeding_of_HEK293T_cells_for_transfection">Seeding of HEK293T cells for transfection</SPAN></H3><P><B>Investigator: LK, CG </B></P><P><B>Aim of the experiment:</B></P><P>3x4 petri dishes with HEK293T cells for performing the Nanoluc assay in order to validate the signale peptides
</P><P><B>Procedure:</B></P><UL><LI> Washing each T75 flask with 10 ml PBS</LI><LI> Centrifugation for 10 min. at 1.8 rcf</LI><LI> Resuspending the cell pellet in 10 ml medium</LI><LI> Counting the cells (33x10^6 cells in 10 ml)</LI><LI> Plating 0.5x10^6 cells onto petri dishes in 10 ml medium an 1x10^6 onto a T75 flask</LI><LI> Incubation for 2 days</LI></UL><P><B>Results:</B></P><UL><LI> ready to transfect cells in petri dishes</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Gelextraction_of_PCR_GSY8.28Saturday.2C_August_13.29_2">Gelextraction of PCR GSY8(Saturday, August 13)</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Purification of PCR GSY8(Sunday, August 14)
</P><P><B>Procedure:</B></P><UL><LI> Using gelextraction kit, SLG</LI></UL><P><B>Results:</B></P><UL><LI> F1016 3,4 ng/µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_F1015_2">Transformation of F1015</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><P>Amplification of F1015
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD><B>Ligation</B></TD><TD><B>Plasmid</B></TD><TD><B>Description</B></TD></TR><TR><TD>F1015
</TD><TD>F1011 + F1012
</TD><TD>pSB1C3
</TD><TD>VEGF-p2a-PDGF + poly-A
</TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> Transformation following SOP(13.02.2013) </LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_of_Gensynthesis_.28GSY.29_7">PCR of Gensynthesis (GSY) 7</SPAN></H3><P><B>Investigator: EF</B></P><P><B>Aim of the experiment:</B></P><P>Amplification of GSY7 
</P><P><B>Procedure:</B></P><UL><LI> PCR Using Q5® High-Fidelity DNA Polymerase (M0491)</LI><LI> Reactions with and without GC Enhancer were performed</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>50 µL reaction x 6</B></TD><TD><B>Final concentration</B></TD></TR><TR><TD>5X Q5 Reaction Buffer
</TD><TD>10 µL
</TD><TD>1 X
</TD></TR><TR><TD>10 mM dNTPs
</TD><TD>1 µL
</TD><TD>200 μM
</TD></TR><TR><TD>10 μM Forward Primer (D20)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>10 μM Reverse Primer (VR)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>Template DNA
</TD><TD>1 µL
</TD><TD>1 ng
</TD></TR><TR><TD>Q5 High-Fidelity DNA Polymerase
</TD><TD>0.5 μl
</TD><TD>0.02 U/μl
</TD></TR><TR><TD>5X Q5 High GC Enhancer (optional)
</TD><TD>5 µL
</TD><TD>1X
</TD></TR><TR><TD>Nuclease-Free Water
</TD><TD>to 50 μl
</TD></TR></TBODY></TABLE><P>Thermocycling Conditions for a Routine PCR:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>STEP</B></TD><TD><B>TEMP</B></TD><TD><B>TIME</B></TD></TR><TR><TD>Initial Denaturation
</TD><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD rowspan="3">40 cycles</TD><TD>98 °C</TD><TD>5 - 10 sec
</TD></TR><TR><TD>58 °C
</TD><TD>30 sec
</TD></TR><TR><TD>72 °C
</TD><TD>60 sec
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>120 sec
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI>negativ. Two 1% agarose gels were performed to prove the PCR product. Both were negativ.</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Precipitation_of_eGFP">Precipitation of eGFP</SPAN></H3><P><B>Investigator:</B> LK 
</P><P><B>Aim of experiment:</B> Determination of (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB> -concentration needed to precipitate eGFP. 
</P><P><B>Procedure:</B></P><UL><LI> 50mL of eGFP solution (from fermentation) where gradually precipitated with rising concentrations (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB> (step size: 400mM) </LI><LI> All steps where performed at 4°C. </LI><LI> Pellets where solved in 10 mL 1x PBS. </LI></UL><P><B>Results:</B></P><P><I>Extinction:</I></P><UL><LI> Maximum Absorption at 488nm for eGFP. Absorption coefficient(eGFP,488): 53000 mol/Lcm.</LI></UL><TABLE cellspacing="1" border="1"><TBODY><TR><TD><B>mM (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB></B></TD><TD><B>Dilution</B></TD><TD><B>Extinction</B></TD><TD><B>Concentration eGFP [µM]</B></TD></TR><TR><TD>400
</TD><TD>1:10
</TD><TD>0,197
</TD><TD>37,0
</TD></TR><TR><TD>800
</TD><TD>1:10
</TD><TD>0,084
</TD><TD>15,8
</TD></TR><TR><TD>1200
</TD><TD>1:10
</TD><TD>0,457
</TD><TD>86,2
</TD></TR><TR><TD>1600
</TD><TD>1:100
</TD><TD>0,16
</TD><TD>301,8
</TD></TR><TR><TD>2000
</TD><TD>1:100
</TD><TD>0,375
</TD><TD>707,5
</TD></TR><TR><TD>2400
</TD><TD>1:10
</TD><TD>0,505
</TD><TD>95,3
</TD></TR><TR><TD>2800
</TD><TD>1:10
</TD><TD>0,08
</TD><TD>15,0
</TD></TR></TBODY></TABLE></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inocculation_of_P258_E.coli_Bl-21_in_M9_medium_for_fermentation">Inocculation of P258_E.coli_Bl-21 in M9 medium for fermentation</SPAN></H3><P><B>Investigator: LK </B></P><P><B>Procedure:</B></P><UL><LI> M9 medium with Cam (1:1000) was inocculated with preculture (~50 mL) of P258 (pASK111+Streptavidin) </LI></UL><UL><LI> the culture was incubated in the shaker over night at 37°C</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Phosphorylation_of_F1005.3B_F1016">Phosphorylation of F1005; F1016</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Phosphorylation of F1005; F1016
</P><P><B>Procedure:</B></P><UL><LI> 39 µL DNA</LI><LI> 5 µL 10x T4 PNK Bufer</LI><LI> 10 mM ATP</LI><LI> T4 PNK</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Phosphorylation_of_O31.3B_O32">Phosphorylation of O31; O32</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Phosphorylation of O31; O32
</P><P><B>Procedure:</B></P><UL><LI> 0.5 µL Oligonucleotides (10µM)</LI><LI> 2 µL 10x T4 PNK Bufer</LI><LI> 2 µL 10 mM ATP</LI><LI> 1 µL T4 PNK</LI><LI> 14.5 µL H2O</LI><LI> 95 °C; 3 min</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Hybridization_of_phosphorylated_O31.3B_O32">Hybridization of phosphorylated O31; O32</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Hybridization of phosphorylated oligonucleotides: O31; O32
</P><P><B>Procedure:</B></P><UL><LI> 2 µL O31</LI><LI> 2 µL O32</LI><LI> 200 µL ddH20</LI><LI> 37 °C 20 min</LI><LI>75 °C 10 min</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_August_17"><B>Wednesday, August 17</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Gelelectrophoresis_of_GSY7">Gelelectrophoresis of GSY7</SPAN></H3><P><B>Investigator: EF</B></P><P><B>Aim of experiment: gelelectrophoresis of the remaining product of GSY7 PCR (from 16.8.16)</B></P><P><B>Procedure:</B>
Agarose gelelectrophoresis was performed using an increased amount of SYBR Green
</P><P><B>Results</B>
There was no PCR product identified. 
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Plasmidpreparation_of_liquid_culture_of_P64.3B_P73">Plasmidpreparation of liquid culture of P64; P73</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Purification of P64; P73
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD><B>Plasmid</B></TD><TD><B>Description</B></TD></TR><TR><TD>P64
</TD><TD>pSB1C3
</TD><TD>VEGF-p2a-PDGF
</TD></TR><TR><TD>P73
</TD><TD>pSB1C3
</TD><TD>Poly-A
</TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> According to Mini-prep. kit, SLG</LI></UL><P><B>Results:'</B></P><UL><LI> P64 134.4 ng/µL</LI><LI> P73 149.4 ng/µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Over_night_liquid_culture_of_F1015">Over night liquid culture of F1015</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Getting enough cellmass for plasmid preparation
</P><P><B>Procedure:</B></P><UL><LI> Inoculation of 5 mL LB-medium (containing antibiotica) with colonies of foregone transformation</LI><LI> Incubate over night at 37 °C in shaker</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_of_scAvidin"><B>PCR of scAvidin</B></SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim:</B> Extraction of single chain Avidin from vector for further cloning 
</P><P><B>Procedure:</B></P><UL><LI> reaction was performed after SOP (PCR Using Q5® High-Fidelity DNA Polymerase, NEB)</LI><LI> 2 ng Plasmid were used for amplification (1 µL)</LI><LI> Primers: O-27 and O-28</LI></UL><UL><LI>PCR program (standard):</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature [°C]</B></TD><TD><B>Time [s]</B></TD></TR><TR><TD>initial denaturation</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>64</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>300
</TD></TR><TR><TD>final extension</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>ꝏ
</TD></TR></TBODY></TABLE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Quickchange_of_P206_and_P239_to_correct_BAP">Quickchange of P206 and P239 to correct BAP</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim:</B> Quickchange P206 (BAP-NanoLuc) and P239 (BAP-NanoLuc-A3C5) to correct BAP
</P><P><B>Procedure:</B></P><UL><LI> reaction was performed after SOP (PCR usingPfuUltra II Fusion HS DNA Polymerase, Agilent)</LI><LI> 10 ng Plasmid were used for amplification (2.5 µL)</LI><LI> Primers: O-33 and O-34</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature [°C]</B></TD><TD><B>Time [s]</B></TD></TR><TR><TD>initial denaturation</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>55</TD><TD>30
</TD></TR><TR><TD>68</TD><TD>300
</TD></TR><TR><TD>final extension</TD><TD>68</TD><TD>360
</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>ꝏ
</TD></TR></TBODY></TABLE></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inocculation_of_fermenter_2">Inocculation of fermenter</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> streptavidin production
</P><P><B>Procedure:</B></P><UL><LI> centrifuge preculture (P258 in M9 medium)</LI><LI> resuspend pellet in HCDF medium of fermenter and inocculate </LI><LI> pO<SUB>2</SUB>= 60 %; Temp= 37°C, pH~7</LI><LI> growth over night</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_Aug_18th"><B>Thursday, Aug 18th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_of_gene_synthesises_8_with_Q5_polymerase">PCR of gene synthesises 8 with Q5 polymerase</SPAN></H3><P><B>Investigator: EF</B></P><P><B>Aim of the experiment:</B> Amplification of GSY7 with optimized Tm following negativ PCR reactions in order to amplify GSY7.
<B>Procedure:</B>
Approach: 
</P><UL><LI> 2,5 µl forward Primer VF2</LI><LI>2,5µl reverse Primer VR</LI><LI> 1µl dNTPs</LI><LI> 1µl DNA (1ng) </LI><LI> 10µl 5x Q5-Puffer</LI><LI> 0,5µl Q5-Polymerase</LI><LI> Fill up with water up to 50 µl</LI></UL><P>Setup: (calculated with NEB calc. <A rel="nofollow" class="external free" href="http://tmcalculator.neb.com/#!/">http://tmcalculator.neb.com/#!/</A>) 
</P><UL><LI> 98°C_2min initiation step</LI><LI> 98°C_10sec</LI><LI> 62°C_30sec   &gt;  step 2-4: 35 repeats</LI><LI> 72°C_1min</LI><LI> 72°C_5min final step </LI><LI> 4°_hold</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_of_gene_synthesises_8_with_Pfu_Ultra_II_polymerase">PCR of gene synthesises 8 with Pfu Ultra II polymerase</SPAN></H3><P><B>Investigator: EF</B></P><P><B>Aim of the experiment:</B> Amplification of GSY7 with Pfu Ultra II polymerase following negativ PCR reactions in order to amplify GSY7.
<B>Procedure:</B>
Approach: 
</P><UL><LI> 2,5 µl forward Primer VF2</LI><LI>2,5µl reverse Primer VR</LI><LI> 1µl dNTPs</LI><LI> 1µl DNA (1ng) </LI><LI> 10µl Pfu Ultra II-Puffer</LI><LI> 0,5µl Q5-Polymerase</LI><LI> Fill up with water up to 50 µl</LI></UL><P>Setup: (calculated with NEB calc. <A rel="nofollow" class="external free" href="http://tmcalculator.neb.com/#!/">http://tmcalculator.neb.com/#!/</A>) 
</P><UL><LI> 98°C_2min initiation step</LI><LI> 98°C_10sec</LI><LI> 56°C_30sec   &gt;  step 2-4: 35 repeats</LI><LI> 72°C_2min</LI><LI> 72°C_5min final step </LI><LI> 4°_hold</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Fermentation_2">Fermentation</SPAN></H3><P><B>Investigator:</B> NA
</P><P><B>Aim of the experiment:</B> Streptavidin production
</P><P><B>Procedure:</B></P><UL><LI> exponential feed (Cam) for ~13 h (about 3-4 L); start at OD= 11.66</LI><LI> Induction of streptavidin expression (anhydro tetracyclin) at OD= 119</LI><LI> change of settings: pO2= 40 %; Temp= 30°C, pH~7</LI><LI> production over night</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Streptavidin_production_in_shaking_flask">Streptavidin production in shaking flask</SPAN></H3><P><B>Investigator:</B> NA
</P><P><B>Aim of the experiment:</B> Streptavidin production
</P><P><B>Procedure:</B></P><UL><LI> 2 L LB medium (Cam 1:1000) was inocculated with ~10 ml M9 preculture (P258 in BL21)</LI><LI> induction with anhydro tetracycline (200µl, 2 mg/ml) at OD= 3,32</LI><LI> culture was incubated at 37 °C and 140 rpm for 4 hours </LI><LI> after expression, cultures were transferred into centrifuge tubes and spun down in the centrifuge (4 °C, 5000 rpm, 20 min, F4X1L rotor)</LI><LI> the supernatant was discarded, and the pellet was frozen away</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Transformation_of_quickchanged_P239_and_206_.28BAP_Nanoluc_A3C5_and_BAP_Nanoluc.29">Transformation of quickchanged P239 and 206 (BAP_Nanoluc_A3C5 and BAP_Nanoluc)</SPAN></H3><P><B>Investigator: LK</B></P><P><B>Procedure:</B></P><UL><LI> After DpnI-digestion 10µl of each plasmid was transformed into E.coli XL-1-blue. </LI><LI> Cells where plated out an Cam-plates after 1h of incubation without antibiotics. </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="MiniPrep_of_F209_to_P266_and_P267.2C_analytic_digestion">MiniPrep of F209 to P266 and P267, analytic digestion</SPAN></H3><P><B>Investigator: JL</B></P><P><B>Aim of the experiment:</B> Verification of successful cloning.
</P><P><B>Procedure:</B>   
MiniPrep was performed according to manufacturer's protocol (QIAprep MiniPrep, Qiagen)
</P><UL><LI> Concentrations:</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>c [ng/µl]</B></TD></TR><TR><TD>P266
</TD><TD>198
</TD></TR><TR><TD>P267
</TD><TD>200,8
</TD></TR></TBODY></TABLE><UL><LI> Analytic digestion: 0.5 µl PstI, 0,5 µl EcoRI, 1 µl CutSmart, 6 µl ddH2O for P266 and P267 for 2 h at 37°C.</LI><LI> 1% agarose gel electrophoresis.</LI></UL><P><B>Results:</B></P><UL><LI> Lane 1: 10 µl 1 kb Ladder.</LI><LI> Lane 2: Digest of F209 to P266.</LI><LI> Lane 3: Digest of F209 to P267. </LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Phosphorylation_of_O31.3B_O32_2">Phosphorylation of O31; O32</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Phosphorylation of O31; O32
</P><P><B>Procedure:</B></P><UL><LI> 5 µL Oligonucleotides (µM)</LI><LI> 2 µL 10x T4 PNK Bufer</LI><LI> 2 µL 10 mM ATP</LI><LI> 1 µL T4 PNK</LI><LI> 10 µL H2O</LI><LI> 95 °C 3min.</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Hybridization_of_phosphorylated_O31.3B_O32_2">Hybridization of phosphorylated O31; O32</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Hybridization of phosphorylated oligonucleotides: O31; O32
</P><P><B>Procedure:</B></P><UL><LI> 2 µL O31</LI><LI> 2 µL O32</LI><LI> 200 µL ddH20</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Plasmidpreparation_of_liquid_culture_of_F1015">Plasmidpreparation of liquid culture of F1015</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Purification of F1015
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD><B>Plasmid</B></TD><TD><B>Description</B></TD></TR><TR><TD>F1015
</TD><TD>pSB1C3
</TD><TD>VEGF-p2a-PDGF + Poly-A
</TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> According to Mini-prep. kit, SLG</LI></UL><P><B>Results:'</B></P><UL><LI> F1015 -&gt; P1008 256.6 ng/µL</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_Aug_19th"><B>Friday, Aug 19th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Gelelectrophoresis_of_GSY7_2">Gelelectrophoresis of GSY7</SPAN></H3><P><B>Investigator: EF</B></P><P><B>Aim of experiment: gel electrophoresis of repeated GSY7 PCR (from 18.8.16)</B></P><P><B>Procedure:</B>
Agarose gelelectrophoresis was performed at 90V
</P><P><B>Results</B>
There was no PCR product identified. 
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="harvesting_streptavidin_2">harvesting streptavidin</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> streptavidin production
</P><P><B>Procedure:</B></P><UL><LI> OD= 181</LI><LI> centrifuge culture for 1 h at 4800 rpm</LI><LI> freeze pellet (2033 g)</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="SDS_PAGE_Streptavidin_fermentation_2">SDS PAGE Streptavidin fermentation</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> Verification of successfull production
</P><P><B>Procedure:</B></P><UL><LI> according to protocol</LI><LI> 8 µl Ladder; before induction; after induction; sample of shaking flask production after induction</LI><LI> all samples: 0.5/OD µl culture filled up with EB to 80µl + 20 µl SDS buffer; 10 µl on SDS gel</LI></UL><P><B>Results:</B></P><P>Clear bands of acetyl transferase (chloramphenicol resistance) in fermenter samples. Therefore cells kept plasmid. The streptavidin bands before and after induction do not differ. The fermentation was not succesful. 
The shaking flask sample however shows a brighter band for streptavidin.
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_F211_.28pASK111.28PAS_Lys.29.29.2C_I739001_.28Biobrick.29_and_F206_.28pASK111_vectorfragment.29">Miniprep of F211 (pASK111(PAS Lys)), I739001 (Biobrick) and F206 (pASK111 vectorfragment)</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment:</B> Miniprep of plasmids
<B>Procedure:</B></P><UL><LI> Miniprep was performed by manufacturers protocol (Qiaprep, Qiagen)</LI><LI> Plasmids were named:
</LI><LI> F211 a: P268</LI><LI> F211 b: P269</LI><LI> I739001 a: P270</LI><LI> I739001 b: P271</LI><LI> F206 a: P272</LI><LI> F206 b: P273</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_P260_in_P261_and_F215_in_P74">Cloning of P260 in P261 and F215 in P74</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment:</B> Digestion, ligation and transformation 
<B>Procedure:</B></P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transfection_of_HEK_cells_with_P253.2C_P225_and_P226">Transfection of HEK cells with P253, P225 and P226</SPAN></H3><P><B>Investigator:</B> LK
</P><P><B>Aim of the experiment:</B> Transfecting HEK cells with vectors encoding 3 diffrent types of signal peptides for secretion.
</P><P><B>Procedure:</B></P><UL><LI> HEK293 where cultured and seeded out on 12 cellculture dishes. (~5x10<SUP>5</SUP> cells per dish) </LI><LI> Transfection was performed on day 3 after seeding (60-70% confluency) </LI><LI> 10µg DNA of each signalpeptide mixed with 30µg PEI (polyenthylenimine) in ddH2O where incubated for 15 min and added to the cells, which where previously washed with PBS and covered (4mL) with optiMEM media. (Pen/Strep;No serum!) </LI><LI> After 4h of incubation at 37°C, 5% CO<SUB>2</SUB> the media was changed to RPMI (Pen/Strep; 5% FCS) again. </LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1_blue_in_quickchanged_pSB1C3_in_P239_and_P206.2C_Retrafo_of_P261_and_P213">Picking of <I>E. coli</I> XL1 blue in quickchanged pSB1C3 in P239 and P206, Retrafo of P261 and P213</SPAN></H3><P><B>Investigator: JL</B></P><P><B>Procedure:</B></P><UL><LI> Quickchanged pSB1C3 in P239: Colonies were picked from Cam-plates.</LI></UL><UL><LI> Retrafo of P261 and P213: Colonies were picked from Cam-plates.</LI></UL><UL><LI> Picked pipette tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contains 4 mL of LB-medium + 4 µL of the proper antibiotic.</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Repeat_of_PCR_of_scAvidin"><B>Repeat of PCR of scAvidin</B></SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim:</B> Extraction of single chain Avidin from vector for further cloning 
</P><P><B>Procedure:</B></P><UL><LI> reaction was performed after SOP (PCR Using Q5® High-Fidelity 2X Master Mix, NEB)</LI><LI> 2 ng Plasmid were used for amplification</LI><LI> Primers: O-27 and O-28</LI><LI> annealing temperature was lowered to 60 °C</LI></UL><UL><LI>PCR program (standard):</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature [°C]</B></TD><TD><B>Time [s]</B></TD></TR><TR><TD>initial denaturation</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>60</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>150
</TD></TR><TR><TD>final extension</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>ꝏ
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Repeat_of_PCR_of_Gene_Synthesis_9"><B>Repeat of PCR of Gene Synthesis 9</B></SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim:</B> Amplification of gensynthesis 9 for further cloning 
</P><P><B>Procedure:</B></P><UL><LI> reaction was performed after SOP (PCR Using Q5® High-Fidelity 2X Master Mix, NEB)</LI><LI> 2 ng Plasmid were used for amplification</LI><LI> Primers: VF2 and VR</LI></UL><UL><LI>PCR program (standard):</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature [°C]</B></TD><TD><B>Time [s]</B></TD></TR><TR><TD>initial denaturation</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>61</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>final extension</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>ꝏ
</TD></TR></TBODY></TABLE></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P8_.28Streptavidin_wt.29">Preparative digestion of P8 (Streptavidin wt)</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of experiment:</B>  preparation for cloning of SA1 into pASK111
</P><P><B>Procedure:</B></P><UL><LI> preparative digestion batch</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>40 µl
</TD><TD>Plasmid DNA P8
</TD></TR><TR><TD>5 µl
</TD><TD>Cutsmart (10x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>HindIII
</TD></TR><TR><TD>1.5 µl
</TD><TD>XbaI
</TD></TR><TR><TD>2 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>duration: over night</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_Aug_20th"><B>Saturday, Aug 20th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_and_analytical_digestion_of_P274-P277">Miniprep and analytical digestion of P274-P277</SPAN></H3><P><B>Investigator:</B> LK
</P><P><B>Aim of the experiment:</B> Isolating and verifiing A3C5+EspP_ddT ligation, &quot;quickchanged&quot; BAP-constructs and amplification of A3C5 in pSB1C3. 
</P><P><B>Procedure:</B></P><UL><LI> Minipreps where performed according to manufacturer's protocol. </LI><LI> Analytical digestion with EcoRI and PstI of ~ 200-250ng of each plasmid was performed for 30 min at 37°C. </LI></UL><P><B>Results:</B></P></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_Aug_21st"><B>Sunday, Aug 21st</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_GSY9_and_scAvidin">Cloning of GSY9 and scAvidin</SPAN></H3><P><B>Investigaor:</B> LK, JL
</P><P><B>Procedure:</B></P><UL><LI> The PCR of scAvdin was digested over night with SpeI and NgoMIV at 37°C. </LI><LI> The PCR of GSY9 was divided: one half was digested with XbaI and AgeI (-&gt; F224) and the other half was digested with XbaI and HindIII (-&gt;F225). </LI><LI> Additionally P74 (pSB1C3+RFP-Reporter) was digested with XbaI and AgeI (-&gt; F227) to provide a plasmid for the ligation with F224. </LI></UL><UL><LI> The digestions where plotted on a gel, and bands at 1700Bp for scAvidin, 222Bp for TetO-OmpA, 490Bp for eMA and 2100Bp for pSB1C3 where cut out and purified with the Promega Wizard SV Gel Kit according to the manufactuerers protocol. </LI></UL><UL><LI> F224 (TetO-OmpA) was then ligated with F227 (pSB1C3), F225 (enhanced monomeric Avidin) was ligated with F206 (pASK111) and F226 (scAvidin) was ligated with F218 (pSB1C3). Ligations where incubated over night at 16°C. </LI></UL><UL><LI> Transformation was performed according to the SOP on Monday morning. </LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_15.2C_Aug_22nd_-_Aug_28th">Week 15, Aug 22nd - Aug 28th</SPAN></H1><DIV class="week" id="Week_15"><H2><SPAN class="mw-headline" id="Monday.2C_Aug_22nd"><B>Monday, Aug 22nd</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P268.2C_P276.2C_P277_and_P279">Sequencing of P268, P276, P277 and P279</SPAN></H3><P><B>Investigator: Julian</B></P><P><B>Aim of the experiment:</B> Sequencing of P268 (PAS lysine), P276 (QC BAP-NanoLuc-A3C5), P277 (QC BAP-NanoLuc) and P279 (A3C5-EspP-DoubleTerminator)
</P><P><B>Procedure:</B></P><P>Sequencing batch were prepared after manufacturer's protocol. (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer)
</P><UL><LI> P268 (D20) FR12904429</LI><LI> P276 (VF2) FR12904428</LI><LI> P277 (VF2) FR12904427</LI><LI> P279 (VF2) FR12904426</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P278_.28pSB1C3.28A3C5-EspP-doubleterminator.29_.26_P279">Analytical digestion of P278 (pSB1C3(A3C5-EspP-doubleterminator) &amp; P279</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment:</B> cofirm success of cloning
</P><P><B>Procedure:</B></P><UL><LI> Batch for analytical digestion for P278/P279 with PstI-HF and EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl/ 1 µl
</TD><TD>Plasmid DNA P278/P279
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>5 µl/ 6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI> P279 showed the expected signal at 1100 bp</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Ligations_of_F1013_.2B_gsy7.3B_.2Bgsy8">Ligations of F1013 + gsy7; +gsy8</SPAN></H3><P><B>Investigator: EF</B></P><P><B>Aim of the experiment:</B> cloning for further ligation steps
<B>Procedure:</B></P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_Aug_23rd"><B>Tuesday, Aug 23rd</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Inoculation_of_Precultures_of_F228.2C_F229_and_F230">Inoculation of Precultures of F228, F229 and F230</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment:</B> Prepare cells for Miniprep
</P><P><B>Procedure:</B></P><UL><LI> Two colonies were picked from each plate</LI><LI> LB CAM was inoculated with F228 and F230</LI><LI> LB AMP was inoculated with F229</LI><LI> Precultures were incubated for 7 hours at 37°C</LI></UL><P><B>Results:</B></P><UL><LI> Each preculture showed cell growth</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_F228.2C_F229_and_F30">Miniprep of F228, F229 and F30</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment:</B> Obtain plasmids
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed according to manufacturers protocol (QiaPrep, QIAGEN)</LI></UL><UL><LI> Plasmids were renamed:
</LI><LI> F228 1: P281</LI><LI> F228 2: P282</LI><LI> F229 1: P283</LI><LI> F229 2: P284</LI><LI> F230 1: P285</LI><LI> F230 2: P286</LI></UL><P><B>Results:</B></P><UL><LI> Concetrations in &quot;Inventarliste&quot;</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_IRES-construct_.28F1019.29_by_PmeI.2BNotI">Digestion of IRES-construct (F1019) by PmeI+NotI</SPAN></H3><P><B>Investigator: EU</B></P><P><B>Aim of the experiment:</B> Prepare construct for ligation later on
</P><P><B>Procedure:</B></P><P>5 µl DNA
38µl ddH2O
1µl of each enzyme
5µl cut smart buffer
</P><P>incubation at 37°C for 1 h
Gel electrophoresis and gel extraction
</P><P><B>Results:</B></P><P>Fragment was expected at about 4 kb. This could be shown comparing with 1kb DNA ladder. 
</P></DIV></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_Aug_24th"><B>Wednesday, Aug 24th</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="analytical_digestion_of_P281.2C_P282.2C_P283.2C_P284.2C_P285_and_P286">analytical digestion of P281, P282, P283, P284, P285 and P286</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment:</B> Verify success of cloning
</P><P><B>Procedure:</B></P><UL><LI> Batch for analytical digestion of P281/P282 with PstI-HF and EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl/ 1 µl
</TD><TD>Plasmid DNA P281/P282
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>6 µl/ 6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion of P283/P284 with XbaI and HindIII-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl/ 1 µl
</TD><TD>Plasmid DNA P283/P284
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>1 µl
</TD><TD>HindIII-HF
</TD></TR><TR><TD>6 µl/ 6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion of P285/P286 with PstI-HF and EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl/ 1 µl
</TD><TD>Plasmid DNA P285/P286
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>6 µl/ 6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestions were incubated at 37° for 1h</LI></UL><P><B>Results:</B></P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="PCR_of_Genesynthesis_9_with_VF2_and_VR">PCR of Genesynthesis 9 with VF2 and VR</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment:</B> Amplification of GSY9
</P><P><B>Procedure:</B></P><UL><LI> reaction was performed after SOP (Q5 Hotstart, NEB)</LI><LI> 2 ng Plasmid were used for amplification</LI><LI> Primers: VF2 and VR</LI><LI> Over all volume= 50 µl</LI></UL><UL><LI>PCR program (standard):</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature [°C]</B></TD><TD><B>Time [s]</B></TD></TR><TR><TD>initial denaturation</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>66</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>30
</TD></TR><TR><TD>final extension</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>ꝏ
</TD></TR></TBODY></TABLE><UL><LI> PCR purification was performed according to manufacturers protocol (Wizard SV Gel and PCR clean-up kit, Promega)</LI></UL><P><B>Results:</B></P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_of_linear_pSB1C3">PCR of linear pSB1C3</SPAN></H3><P><B>Investigator: MT</B></P><P><B>Aim of the experiment:</B></P><P>Amplification of linear pSB1C3 and introduction of restriction site: blunt at 5' and NotI at 3'
</P><P><B>Procedure:</B></P><UL><LI> PCR Using Q5® High-Fidelity DNA Polymerase (M0491) according to online protocol</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>50 µL reaction</B></TD><TD><B>Final concentration</B></TD></TR><TR><TD>Q5® High-Fidelity 2X Master Mix
</TD><TD>25 µL
</TD><TD>1 X
</TD></TR><TR><TD>10 μM Forward Primer (O-038)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>10 μM Reverse Primer (O-039)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>Template DNA
</TD><TD>1 µL
</TD><TD>1 ng
</TD></TR><TR><TD>Nuclease-Free Water
</TD><TD>to 50 μl
</TD></TR></TBODY></TABLE><P>Thermocycling Conditions for a Routine PCR:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>STEP</B></TD><TD><B>TEMP</B></TD><TD><B>TIME</B></TD></TR><TR><TD>Initial Denaturation
</TD><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98 °C</TD><TD>10 sec
</TD></TR><TR><TD>68 °C
</TD><TD>30 sec
</TD></TR><TR><TD>72 °C
</TD><TD>60 sec
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>120 sec
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P74.2C_P165.2C_P196.2C_P273.2C_P274.2C_P275.2C_P285_and_PCR_of_GSY9">Preparative digestion of P74, P165, P196, P273, P274, P275, P285 and PCR of GSY9</SPAN></H3><P><B>Investigator: Clemens</B></P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P74 with XbaI and AgeI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>Plasmid DNA P74
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>23 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P74 with NgOMIV and AgeI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>Plasmid DNA P74
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgOMIV
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>23 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P165 with PstI-HF and AgeI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12 µl
</TD><TD>Plasmid DNA P165
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P196 with NgOMIV and PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>Plasmid DNA P196
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgOMIV
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>10 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P273 with XbaI and HindIII-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P273
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>1 µl
</TD><TD>HindIII-HF
</TD></TR><TR><TD>5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P274 with AgeI-HF and PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>Plasmid DNA P274
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P275 with NgOMIV and PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>8 µl
</TD><TD>Plasmid DNA P275
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgOMIV
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P285 with NgOMIV and PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12 µl
</TD><TD>Plasmid DNA P285
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgOMIV
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of PCR of GS9 with NgOMIV, AgeI-HF and XbaI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>Plasmid DNA PCR GSY9
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgOMIV
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=31 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of PCR of GS9 with HindIII-HF and XbaI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>Plasmid DNA PCR GSY9
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>1 µl
</TD><TD>HindIII-HF
</TD></TR><TR><TD>0 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestions were incubated at 37° over night</LI><LI> Samples were applied on a 1% agarose gel</LI><LI> Samples were seperated for 45 min at 80V</LI></UL><P><B>Results:</B></P><UL><LI> All Digests showed the expected lengths</LI><LI> Fragments were cut from the gel and purified according to manufacturers protocol (Wizard SV Gel and PCR clean-up system, Promega)</LI></UL></DIV><DIV class="kill-switch"><H3><SPAN class="mw-headline" id="Transformation_of_P296_.28pSB1C3_mGMK-Tk30.2C_BBa_K782063.29_in_xl1-blue">Transformation of P296 (pSB1C3 mGMK-Tk30, BBa_K782063) in xl1-blue</SPAN></H3><P><B>Investigator: MT</B></P><P><B>Aim of the experiment:</B> Transformation for Poly-A-tail adding
</P><P><B>Procedure:</B>
transformation in competent xl1-blue according to protocol + rescue
</P><P><B>Result:</B> 
plates (LB Cam) in incubator for further processing (37 °C) showed after 24 h colonies
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id=".28Re.29ligations_of_F1013_.28pSB1C3.29_and_F1016_.28phosphoryliert.29_.2B_F1013_and_F1005_in_pSB1C3">(Re)ligations of F1013 (pSB1C3) and F1016 (phosphoryliert) + F1013 and F1005 in pSB1C3</SPAN></H3><P><B>Investigator: EF</B></P><P><B>Aim of the experiment:</B> Ligations of PCR Produkt from GSY8 in pSB1C3 vector for  later transformation and cloning 
</P><P><B>Procedure:</B></P><UL><LI> Ligation mixtures were prepared as following; DNA amounts were calculated using the ligation calculator <A rel="nofollow" class="external free" href="http://www.insilico.uni-duesseldorf.de/Lig_Input.html">http://www.insilico.uni-duesseldorf.de/Lig_Input.html</A> :</LI><LI> F1013+F1016: 1.6 μL F1013, 5 μL F1016, 2μL T4-DNA-Ligase-Buffer, 1 μL T4-DNA-Ligase, 10.4 μL ddH2O</LI><LI> F1013+F1005: 1.6 μL F1013, 2.2 μL F1005, 2μL T4-DNA-Ligase-Buffer, 1 μL T4-DNA-Ligase, 13.2 ddH2O</LI><LI> Ligation was incubated for 1.5 hours at 16 °C</LI></UL></DIV><DIV class="Hypoxia"><H2><SPAN class="mw-headline" id="Thursday.2C_Aug_25th"><B>Thursday, Aug 25th</B></SPAN></H2></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Phosphorylation_and_hybridization_of_O35.2C_O36.2C_O37">Phosphorylation and hybridization of O35, O36, O37</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><P>Phosphorylation and hybridization of O35, O36, O37
</P><P><B>Procedure:</B>
Phosphorylation
</P><UL><LI> 0,5 µL Oligonucleotides</LI><LI> 2 µL 10x T4 PNK Bufer</LI><LI> 2 µL T4 Ligase Buffer</LI><LI> 1 µL T4 PNK</LI><LI> 14,5 µL H2O</LI></UL><P>incubate at 37°C for 20 min and inactivate T4PNK (for 10 min at 75°C)
</P><P>
Hybridization
</P><UL><LI> 2 µL O31</LI><LI> 2 µL O32</LI><LI> in 200 µL ddH20</LI></UL><P>incubate at 95°C for 3 min, let cool down slowly
</P><P><B>Results:</B></P></DIV><DIV class="kill-switch"><H3><SPAN class="mw-headline" id="Inoculation_of_P296.28Kill-Switch.29">Inoculation of P296(Kill-Switch)</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B> production of P296
</P><P><B>Procedure:</B>
2x5ml LB-CAM media, each inoculated by one colony (not by rescue colonies)
</P><P><B>Results:</B></P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P74_.2B_HincII">Digestion of P74 + HincII</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of P74 + HincII</LI></UL><P><B>Procedure:</B></P><UL><LI> 3 µL DNA (1431 ng/µL)</LI><LI> 1 µL HincII  </LI><LI> 14 µL ddH2O </LI><LI> 2 µL CutSmart buffer</LI><LI> incubation for 2h at 37 °C</LI></UL><P><B>Results:'</B></P><UL><LI> P74 -&gt; P1021 111,2 ng/µL </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Ligation_of_F233.2BF238.2C_F236.2BF234.2C_F233.2BF237.2C_F232.2BF240.2C_F231.2BF239.2C_F235.2BF241_and_F248.2BF249">Ligation of F233+F238, F236+F234, F233+F237, F232+F240, F231+F239, F235+F241 and F248+F249</SPAN></H3><P><B>Investigator: </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Ligation and transformation of XL1 blue</LI></UL><P><B>Procedure:</B></P><UL><LI>Batch for Ligation of F233 and F238</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1,7 µl
</TD><TD>Vector F233
</TD></TR><TR><TD>8,3 µl
</TD><TD>Insert F238
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>H₂O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Batch for Ligation of F236 and F234</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1,1 µl
</TD><TD>Vector F236
</TD></TR><TR><TD>8,9 µl
</TD><TD>Insert F234
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>H₂O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Batch for Ligation of F233 and F237</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3,1 µl
</TD><TD>Vector F233
</TD></TR><TR><TD>6,7 µl
</TD><TD>Insert F237
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>H₂O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Batch for Ligation of F232 and F240</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4,8 µl
</TD><TD>Vector F232
</TD></TR><TR><TD>5,2 µl
</TD><TD>Insert F240
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>H₂O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Batch for Ligation of F231 and F239</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5,6 µl
</TD><TD>Vector F231
</TD></TR><TR><TD>4,4 µl
</TD><TD>Insert F239
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>H₂O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Batch for Ligation of F235 and F241</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6,5 µl
</TD><TD>Vector F235
</TD></TR><TR><TD>3,5 µl
</TD><TD>Insert F241
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>H₂O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Batch for Ligation of F228 and F249</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,7 µl
</TD><TD>Vector F248
</TD></TR><TR><TD>7,3 µl
</TD><TD>Insert F249
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>H₂O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligations were performed according to SOP</LI><LI> XL1blue were transformed with the resulting fragments</LI></UL><P><B>Results:'</B></P><P>Cellgrowth yes/no?
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P1007_.2B_EcoRI_.2B_SpeI">Digestion of P1007 + EcoRI + SpeI</SPAN></H3><P><B>Investigator: EF </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of P1007</LI></UL><P><B>Procedure:</B></P><UL><LI> 10 µL DNA (3.6 ug)</LI><LI> 0.25 µL EcoRI  </LI><LI> 0.25 µL SpeI </LI><LI> 34.5 µL ddH2O </LI><LI> 5 µL CutSmart Buffer NEB</LI><LI> incubation for 1h at 37 °C</LI></UL><P><B>Results:</B></P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_digestion.2C_gel_electrophoresis_and_gel_extraction_of_P288_.28CMV-Promotor.29_and_P289_.28NanoLuc_.2B_Streptag_.2B_Poly-A.29">Preparative digestion, gel electrophoresis and gel extraction of P288 (CMV-Promotor) and P289 (NanoLuc + Streptag + Poly-A)</SPAN></H3><P><B>Investigator: JL</B></P><P><B>Aim of experiment: Preparative digestion, gel electrophoresis and gel extraction of P288 (CMV-Promotor) and P289 (NanoLuc + Streptag + Poly-A)</B></P><P><B>Procedure:</B></P><UL><LI> Batches for analytical digestions:</LI></UL><UL><LI>P288: PstI and SpeI.</LI></UL><UL><LI>P29: PstI and XbaI.</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6  µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>19  µl
</TD><TD>ddH2O
</TD></TR><TR><TD>3 µl
</TD><TD>CutSmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>Each enzyme
</TD></TR><TR><TD>30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Preparative gel electrophoresis was performed at 90 V for 60 min.</LI><LI>Preperative gel according to SOP.</LI><LI>The Purification was done according to the manufacturer's protocol. </LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P293.2C_P294.2C_P295_.28eMA_pASK111.29">Analytical digestion and gelelectrophoresis of P293, P294, P295 (eMA pASK111)</SPAN></H3><P><B>Investigator: JL</B></P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P293, P294, P295 (eMA pASK111).
</P><P><B>Procedure:</B></P><UL><LI> Batch for analytical digestion for P293, P294, P295.</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2/1/4 µl
</TD><TD>Plasmid DNA P293/P294/P295
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.5 µl
</TD><TD>HindIII (20 U/µl)
</TD></TR><TR><TD>6/8/4 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=12 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 120 min at 37 °C.</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 30 min.</LI></UL><P><B>Results:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder DNA ladder
</TD><TD><B>P293</B></TD><TD><B>P294</B></TD><TD><B>P294</B></TD></TR></TBODY></TABLE><P><A href="/File:TUM13_20130423_RFP_Generator_RFC25_AgeI_NgoMIV.png" title="File:TUM13 20130423 RFP Generator RFC25 AgeI NgoMIV.png">500px</A></P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_Aug_26"><B>Friday, Aug 26</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P74_.2B_HincII_2">Digestion of P74 + HincII</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of P74 + HincII</LI></UL><P><B>Procedure:</B></P><UL><LI> 3 µL DNA (1431 ng/µL)</LI><LI> 1 µL HincII  </LI><LI> 41 µL ddH2O </LI><LI> 5 µL CutSmart buffer</LI><LI> * 2h; 37 °C</LI></UL><P><B>Results:</B></P><P>generuler; pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2; P74[HincII]
</P><UL><LI> F1024 49.3 ng\µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2">Transformation of pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Amplification of Vector</LI></UL><P><B>Procedure:</B></P><UL><LI> Transformation following SOP(13.02.2013) </LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2">Digestion of pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2 + PmeI &amp; NotI</LI></UL><P><B>Procedure:</B></P><UL><LI> 7 µL DNA (77 ng/µL)</LI><LI> 0.25 µL PmeI </LI><LI> 0.25 µL NotI</LI><LI> 34.5 µL ddH2O </LI><LI> 5 µL CutSmart buffer</LI><LI> 2h; 37 °C</LI></UL><P><B>Results:</B></P><P>generuler; pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2; P74[HincII] 
</P><UL><LI> F1023 7.6 ng\µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Gelextraction_PCR_pSB1C3">Gelextraction PCR pSB1C3</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Purification of PCR linear pSB1C3 </LI></UL><P><B>Procedure:</B></P><UL><LI> According to Gelextraction kit; Quiagen</LI></UL><P><B>Results:</B></P><P>generuler; PCR gsy8; PCR linear pSB1C3
</P><UL><LI> F1025 9.2 ng\µL</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P249.2C_P268_and_P269_.28each_by_XbaI.26HindIII.29">Analytical digestion of P249, P268 and P269 (each by XbaI&amp;HindIII)</SPAN></H3><P><B>Investigator: EU</B></P><P><B>Aim of the experiment:</B> Analyze construct for eventual preparative digestion
</P><P><B>Procedure:</B></P><P>5 µl DNA (except for P249 --&gt; 13 µl)
12µl ddH2O (except for P249 --&gt; 4 µl)
1µl of each enzyme
2µl cut smart buffer
</P><P>incubation at 37°C for 2 h
Gel electrophoresis and gel extraction
</P><P><B>Results:</B></P><P>In each digestion only one fragment (the vector)  was expected at about 4 kb. This could be shown comparing with 1kb DNA ladder. 
</P></DIV><DIV class="linkers"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P272_and_P273_.28each_by_XbaI.26HindIII.29">Preparative digestion of P272 and P273 (each by XbaI&amp;HindIII)</SPAN></H3><P><B>Investigator: EU</B></P><P><B>Aim of the experiment:</B> Prepare construct for ligation later on
</P><P><B>Procedure:</B></P><P>5 µl DNA
36µl ddH2O
2µl of each enzyme
5µl cut smart buffer
</P><P>incubation at 37°C for 2 h
Gel electrophoresis and gel extraction by Gel extraction kit (Promega) according  to manufacturer’s protocol
</P><P><B>Results:</B></P><P>Fragment was expected at about 4 kb. This could be shown comparing with 1kb DNA ladder. 
</P></DIV><DIV class="kill-switch"><H3><SPAN class="mw-headline" id="Miniprep_of_P296">Miniprep of P296</SPAN></H3><P><B>Investigator: EU</B></P><P><B>Aim of the experiment:</B> Obtain plasmids
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed according to manufacturers protocol (QiaPrep, QIAGEN)</LI></UL><UL><LI> Plasmid was not renamed.
</LI></UL><P><B>Results:</B></P><UL><LI> Concetrations in &quot;Inventarliste&quot;</LI></UL></DIV><DIV class="kill-switch"><H3><SPAN class="mw-headline" id="Sequencing_of_P296">Sequencing of P296</SPAN></H3><P><B>Investigator: EU</B></P><P><B>Aim of the experiment:</B> Sequencing of P296
</P><P><B>Procedure:</B></P><UL><LI>Sequencing batch was prepared after manufacturer's protocol:</LI></UL><UL><LI>7 µl of plasmid DNA (end concentration 50 - 100ng) in 8µl H2O</LI><LI>2 µl sequencing primer (one batch with VF2 and another with VR)</LI></UL><UL><LI>The different vectors we sequenced received the following barcodes:</LI></UL><UL><LI> P296 with VF2: FR</LI><LI> P296 with VR: FR</LI></UL><P><B>Results:</B></P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P1007.2C_P1008_with_EcoRI_HF.2C_SpeI_HF_.2B_analytical_gel">Digestion of P1007, P1008 with EcoRI HF, SpeI HF + analytical gel</SPAN></H3><P><B>Investigator: EF </B></P><P><B>Aim of the experiment:</B> Isolation of pSB1C3 plasmid from p1007)
<B>Procedure:</B></P><UL><LI> p1007 digested with: </LI></UL><P>0,25 µl EcoRI (HF) 
0,25 µl SpeI (HF), 
5 µl CutSmart Buffer 
3 µg plasmid-DNA (P1007)
fill up with ddH2O (Vtotal= 50µL)
</P><UL><LI> 2x 25 µl on 1% agarose gel for electrophoresis (2log DNA-ladder as standard)</LI></UL><P><B>Results:</B> Despite optimised enzyme quantity the insert seems to not have been cut out of the plasmid. One single band at 3kb --&gt; repeat reaction; check cutting sites with genious
(*)has p1007 been sequenzed before(?). However the band was extracted from the geled and freezed labeled as &quot;8.1&quot; 
</P><UL><LI> p1008 digested with: </LI></UL><P>0,25 µl EcoRI (HF) 
0,25 µl SpeI (HF), 
5 µl CutSmart Buffer 
3 µg plasmid-DNA (P1008)
fill up with ddH2O (Vtotal= 50µL)
</P><UL><LI> 2x 25 µl on 1% agarose gel for electrophoresis (2log DNA-ladder as standard)</LI></UL><P><B>Results:</B> Insert was isolated by gel extraction following GelExtraction Kit instructions.
</P><P><A href="/File:P1007,_P1008_26.8.16_EF_black.scn" title="File:P1007, P1008 26.8.16 EF black.scn">500px</A></P></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_Aug_27"><B>Saturday, Aug 27</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_GSY8">PCR GSY8</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Amplification of GSY 8 </LI></UL><P><B>Procedure:</B></P><UL><LI> PCR Using Q5® High-Fidelity DNA Polymerase (M0491) according to online protocol</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>50 µL reaction</B></TD><TD><B>Final concentration</B></TD></TR><TR><TD>Q5® High-Fidelity 2X Master Mix
</TD><TD>25 µL
</TD><TD>1 X
</TD></TR><TR><TD>10 μM Forward Primer (VR)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>10 μM Reverse Primer (D20)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>Template DNA
</TD><TD>1 µL
</TD><TD>10 ng
</TD></TR><TR><TD>Nuclease-Free Water
</TD><TD>to 50 μl
</TD></TR></TBODY></TABLE><P>Thermocycling Conditions for a Routine PCR:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>STEP</B></TD><TD><B>TEMP</B></TD><TD><B>TIME</B></TD></TR><TR><TD>Initial Denaturation
</TD><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98 °C</TD><TD>10 sec
</TD></TR><TR><TD>61 °C
</TD><TD>30 sec
</TD></TR><TR><TD>72 °C
</TD><TD>60 sec
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>120 sec
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI> negativ two times :(</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Mung_Bean_nuclease_.28MBN.29_digestion_of_F1023">Mung Bean nuclease (MBN) digestion of F1023</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Get rid of overhangs. Preparation of DNA for blunt ligation</LI></UL><P><B>Procedure:</B></P><UL><LI> 18 µL DNA (144 ng)</LI><LI> 2 µL MBN buffer</LI><LI> 0.5 µL MBN</LI><LI> 30 min; 30 °C</LI><LI> purification by PCR purification kit, Promega</LI></UL><P><B>Results:</B></P><UL><LI> F1026 24 ng\µL; must be wrong, because this is more than digested. Endvolume 20 µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Dephosphorylation_of_F1024">Dephosphorylation of F1024</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Preparation of Vekor for blunt end Ligation </LI></UL><P><B>Procedure:</B></P><UL><LI> 13 µL DNA (~50 ng\µL)</LI><LI> 5 µL Antarctic phosphatase (AP) </LI><LI> 2 µL AP buffer</LI><LI> 30 min ; 37 °C</LI><LI> heat inactivation: 2 min; 80 °C</LI></UL><P><B>Results:</B></P><UL><LI> F1029</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F1025_.2BF1022_.3D_F1027">Ligation: F1025 +F1022 = F1027</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F1025 +F1022</LI></UL><P><B>Procedure:</B></P><UL><LI> 1 µL F1025 (10 ng\µL)</LI><LI> 10 µL F1022 (5 ng\µL)</LI><LI> 2 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 6 µL ddH2O</LI><LI> 60 min;  30°C</LI></UL><P><B>Results:</B></P><UL><LI> F1025 +F1022 = F1027</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F1025_.2BF1026_.3D_F1028">Ligation: F1025 +F1026 = F1028</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F1025 +F1026</LI></UL><P><B>Procedure:</B></P><UL><LI> 1 µL F1025 (10 ng\µL)</LI><LI> 20 µL F1026 (24 ng\µL)</LI><LI> 3 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 4 µL ddH2O</LI><LI> 60 min; 37 °C</LI></UL><P><B>Results:</B></P><UL><LI> F1025 +F1026 = F1028</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Phosphorylation_of_GSY7.3B_GSY8">Phosphorylation of GSY7; GSY8</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Phosphorylation of GSY7; GSY8</LI></UL><P><B>Procedure:</B></P><UL><LI> 5 µL DNA</LI><LI> 2 µL T4 Ligase buffer </LI><LI> 1 µL T4 PNK</LI><LI> 12 µL ddH20</LI><LI> 30 min; 37 °C</LI><LI> heat inactivation: 20 min; 65 °C</LI></UL><P><B>Results:</B></P><UL><LI> F1030 (GSY7)</LI><LI> F1031 (GSY8)</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Over_night_liquid_culture_of_pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2">Over night liquid culture of pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Getting enough cellmass for plasmid preparation</LI></UL><P><B>Procedure:</B></P><UL><LI> Inoculation of 5 mL LB-medium (containing antibiotica) with colonies of foregone transformation</LI><LI> Incubate over night at 37 °C in shaker</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F1029_.2B_F1030_.3D_F1032.3B_F1029_.2B_F1031_.3D_F1033">Ligation: F1029 + F1030 = F1032; F1029 + F1031 = F1033</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Ligation: F1029 + F1030 = F1032; F1029 + F1031 = F1033</LI></UL><P><B>Procedure:</B></P><UL><LI> 5 µL F1029 ( ~ 30 ng) {ERROR!}</LI><LI> 10 µL F1030/F1031 (~ 25 ng)</LI><LI> 2 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 2 µL ddH2O</LI><LI> 60 min; 37 °C</LI></UL><P><B>Results:</B></P><UL><LI> F1029 + F1030 = F1032 </LI><LI> F1029 + F1031 = F1033</LI><LI>Both negativ, due to miscalculation</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_F1027.3B_F1028.3B_F1032.3B_F1033">Transformation of F1027; F1028; F1032; F1033</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Amplification of Vector</LI></UL><P><B>Procedure:</B></P><UL><LI> Transformation following SOP(13.02.2013) </LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_Aug_28"><B>Sunday, Aug 28</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_GSY8_2">PCR GSY8</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Amplification of GSY 8 </LI></UL><P><B>Procedure:</B></P><UL><LI> PCR Using Q5® High-Fidelity DNA Polymerase (M0491) according to online protocol</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>50 µL reaction</B></TD><TD><B>Final concentration</B></TD></TR><TR><TD>Q5® High-Fidelity 2X Master Mix
</TD><TD>25 µL
</TD><TD>1 X
</TD></TR><TR><TD>10 μM Forward Primer (D20)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>10 μM Reverse Primer (VR)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>Template DNA
</TD><TD>1 µL
</TD><TD>10 ng
</TD></TR><TR><TD>Nuclease-Free Water
</TD><TD>to 50 μl
</TD></TR></TBODY></TABLE><P>Thermocycling Conditions for a Routine PCR:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>STEP</B></TD><TD><B>TEMP</B></TD><TD><B>TIME</B></TD></TR><TR><TD>Initial Denaturation
</TD><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98 °C</TD><TD>10 sec
</TD></TR><TR><TD>58 °C
</TD><TD>30 sec
</TD></TR><TR><TD>72 °C
</TD><TD>60 sec
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>120 sec
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><P><B>Results:</B></P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Over_night_liquid_culture_of_F1032.3B_F1028">Over night liquid culture of F1032; F1028</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Getting enough cellmass for plasmid preparation</LI></UL><P><B>Procedure:</B></P><UL><LI> Inoculation of 5 mL LB-medium (containing antibiotica) with colonies of foregone transformation</LI><LI> Incubate over night at 37 °C in shaker</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Plasmidpreparation_of_liquid_culture_of_pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2">Plasmidpreparation of liquid culture of pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2</SPAN></H3><P><B>Investigator: MP </B></P><P><B>Aim of the experiment:</B></P><P>Purification of pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2
</P><P><B>Procedure:</B></P><UL><LI> According to Mini-prep. kit, SLG</LI></UL><P><B>Results:'</B></P><UL><LI> P1009 608.6 ng/µL</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_16.2C_Aug_29nd_-_Sep_4th">Week 16, Aug 29nd - Sep 4th</SPAN></H1><DIV class="week" id="Week_16"><H2><SPAN class="mw-headline" id="Monday.2C_Aug_29nd"><B>Monday, Aug 29nd</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Plasmid_preparation_of_liquid_culture_of_F1032.28clon_1.29.3B_F1032.28clon_2.29">Plasmid preparation of liquid culture of F1032(clon 1); F1032(clon 2)</SPAN></H3><P><B>Investigator: JL </B></P><P><B>Aim of the experiment:</B></P><P>Purification of F1032(clon 1); F1032(clon 2)
</P><P><B>Procedure:</B></P><UL><LI> According to Mini-prep. kit, Quiagen</LI></UL><P><B>Results:</B></P><UL><LI> P 1010  448.4 ng/µL</LI><LI> P 1011 344.6 ng/µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P1010.3B_P1011">Analytical digestion of P1010; P1011</SPAN></H3><P><B>Investigator: JL </B></P><P><B>Aim of the experiment:</B></P><UL><LI> verify constructs: P1010; P1011</LI></UL><P><B>Procedure:</B></P><UL><LI> 2 µL DNA</LI><LI> 0.5 µL EcoRI</LI><LI> 0.5 µL PstI</LI><LI> 1 µL CutSmart buffer</LI><LI> 6 µL ddH20</LI></UL><P><B>Results:</B></P><UL><LI> 1kb ladder; P1010; P1011</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_and_purification_of_GSY7_new_and_GSY8">PCR and purification of GSY7_new and GSY8</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Amplification of GSY 7_new and GSY 8 </LI></UL><P><B>Procedure:</B></P><P>
Same procedure for both GSYs:
</P><UL><LI> PCR Using Q5® High-Fidelity DNA Polymerase (M0491) according to online protocol</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>50 µL reaction</B></TD><TD><B>Final concentration</B></TD></TR><TR><TD>Q5® High-Fidelity 2X Master Mix
</TD><TD>25 µL
</TD><TD>1 X
</TD></TR><TR><TD>10 μM Forward Primer (VR)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>10 μM Reverse Primer (D20)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>Template DNA
</TD><TD>1 µL
</TD><TD>10 ng
</TD></TR><TR><TD>Nuclease-Free Water
</TD><TD>to 50 μl
</TD></TR></TBODY></TABLE><P>Thermocycling Conditions for a Routine PCR:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>STEP</B></TD><TD><B>TEMP</B></TD><TD><B>TIME</B></TD></TR><TR><TD>Initial Denaturation
</TD><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98 °C</TD><TD>10 sec
</TD></TR><TR><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD>61 °C
</TD><TD>60 sec
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>120 sec
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><P>Purification according to Promega PCR purification kit
</P><P><B>Results:</B>
GSY7: 9,4 ng/µl
GSY8: 1,5 ng/µl, probably contaminated with GSY7_new
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Phosphorylation_of_GSY7_new_and_GSY8">Phosphorylation of GSY7_new and GSY8</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Phosphorylation of GSY7_new; GSY8</LI></UL><P><B>Procedure:</B></P><UL><LI> 5 µL DNA</LI><LI> 2 µL T4 Ligase buffer </LI><LI> 1 µL T4 PNK</LI><LI> 12 µL ddH20</LI><LI> 30 min; 37 °C</LI><LI> heat inactivation: 10 min; 75 °C</LI></UL><P><B>Results:</B></P><UL><LI> F1034 (GSY7_new)</LI><LI> F1035 (GSY8)</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F1034_.2B_F1029_.3D_F1036.3B_F1035_.2B_F1029_.3D_F1037">Ligation: F1034 + F1029 = F1036; F1035 + F1029 = F1037</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Ligation: F1034 + F1029 = F1036; F1035 + F1029 = F1037</LI></UL><P><B>Procedure:</B></P><UL><LI> 5 µL F1029 ( ~ 10 ng)</LI><LI> 10 µL F1034/F1035 (~ 25 ng)</LI><LI> 2 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 2 µL ddH2O</LI><LI> 60 min; 37 °C</LI></UL><P><B>Results:</B></P><UL><LI> F1029 + F1034 = F1036 </LI><LI> F1029 + F1035 = F1037</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P297.2C_P299.2C_P279.2C_P285.2C_P285.2C_P277.2C_P312_.26_P251">Preparative digestion of P297, P299, P279, P285, P285, P277, P312 &amp; P251</SPAN></H3><P><B>Investigator:  </B> Clemens
</P><P><B>Aim of the experiment:</B></P><P>Preparation for cloning in pSB1C3/pASK111
</P><P><B>Procedure:</B></P><UL><LI> * Batch for preparative digestion of P297 with AgeI-HF &amp; PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>Plasmid DNA P297
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI HF
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>220 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P299 with NgOMIV and PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>Plasmid DNA P299
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgOMIV
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>19 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P279 with NgOMIV &amp; PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P279
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgOMIV
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> * Batch for preparative digestion of P285 with AgeI-HF &amp; PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>Plasmid DNA P297
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI HF
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>10 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> * Batch for preparative digestion of P277 with AgeI-HF &amp; PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>Plasmid DNA P277
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI HF
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>19 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> * Batch for preparative digestion of P312 with AgeI-HF &amp; PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>Plasmid DNA P312
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI HF
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>20 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P251 with NgOMIV &amp; PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>Plasmid DNA P251
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgOMIV
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>10 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestions were incubated for 3h at 37°C</LI><LI> Fragments were cut from the gel and purified by gelectraction (promega)</LI></UL><P><B>Results:</B></P><UL><LI> Fragments were renamed (see Inventarliste) and concentrations were measured</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Ligation_of_F254.2B_F255.2C_F256.2BF257.2C_F256.2BF258_.26_F260.2BF312">Ligation of F254+ F255, F256+F257, F256+F258 &amp; F260+F312</SPAN></H3><P><B>Investigator: </B> Clemens
</P><P><B>Aim of the experiment:</B></P><UL><LI> Ligation</LI></UL><P><B>Procedure:</B></P><UL><LI>Batch for Ligation of F254 and F255</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>Vector F254
</TD></TR><TR><TD>7 µl
</TD><TD>Insert F255
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>H₂O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Batch for Ligation of F257 and F256</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.9 µl
</TD><TD>Vector F257
</TD></TR><TR><TD>8.1 µl
</TD><TD>Insert F256
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>H₂O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Batch for Ligation of F258 and F256</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.3 µl
</TD><TD>Vector F258
</TD></TR><TR><TD>8.7 µl
</TD><TD>Insert F256
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>H₂O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Batch for Ligation of F312 and F260</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.2 µl
</TD><TD>Vector F312
</TD></TR><TR><TD>8.8 µl
</TD><TD>Insert F260
</TD></TR><TR><TD>2 µl
</TD><TD>10X DNA-Ligase-buffer
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>7 µl
</TD><TD>H₂O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligations were performed according to SOP</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_pASK111.2C_P268_.28SUMO-PAS.29.2C_P6.2C_and_P259_.28both_SA.29">Analytical digestion and gelelectrophoresis of pASK111, P268 (SUMO-PAS), P6, and P259 (both SA)</SPAN></H3><P><B>Investigator: Niklas</B></P><P><B>Aim of the experiment: verification before cloning for fermentation</B></P><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>P268/ P259/ P6/ pASK111</B></TD></TR><TR><TD>5,8 µl/ 17 µl/ 8,33 µl/ 14,2 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2 µl
</TD><TD>cut smart (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.5 µl
</TD><TD>HindIII
</TD></TR><TR><TD>11,2 µl/ 0 µl/ 8,67 µl/ 2,8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.</LI></UL><P><B>Results:</B></P><UL><LI> on of the enzymes was contaminated</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_P310_in_BL_21.3B_P6.2C_P259.2C_P268.2C_pASK111.2C_F1034.2BF1028_in_xl1-blue">Transformation of P310 in BL 21; P6, P259, P268, pASK111, F1034+F1028 in xl1-blue</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Procedure:</B></P><UL><LI> CaCl<SUB>2</SUB>-competent <I>E. coli</I> cells were taken from the stock in –80 °C freezer and gently thawed on ice</LI></UL><UL><LI> 1 µl (retrafo)/ 7 µl (ligation) of DNA was added to 50 µl of competent cells and gently mixed</LI></UL><UL><LI> 30 min incubation on ice</LI></UL><UL><LI> 5 min heat shock at 37 °C</LI></UL><UL><LI> Added 0,9 ml LB medium to each tube</LI></UL><UL><LI> Incubation for 60 min at 37 °C in the 180 rpm cell-culture shaker</LI></UL><UL><LI> 50 µl of the cell suspension was plated on one plate</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13,000 rpm and the supernatant was dicarded</LI></UL><UL><LI> The pellet was resuspended in 50 µl of LB-medium and this concentrated cell suspension was plated on a  plate.</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_Aug_30rd"><B>Tuesday, Aug 30rd</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_on_GSY7_and_GSY8">PCR on GSY7 and GSY8</SPAN></H3><P><B>Investigator: CG </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Amplification of GSY 7 and GSY 8 </LI></UL><P><B>Procedure:</B></P><P>
Same procedure for both GSYs:
</P><UL><LI> PCR Using Q5® High-Fidelity DNA Polymerase (M0491) according to online protocol</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>50 µL reaction</B></TD><TD><B>Final concentration</B></TD></TR><TR><TD>Q5® High-Fidelity 2X Master Mix
</TD><TD>25 µL
</TD><TD>1 X
</TD></TR><TR><TD>10 μM Forward Primer (VR)
</TD><TD>1,5 μl
</TD><TD>0.25 μM
</TD></TR><TR><TD>10 μM Reverse Primer (D20)
</TD><TD>1,5 μl
</TD><TD>0.25 μM
</TD></TR><TR><TD>Template DNA
</TD><TD>2 µL
</TD><TD>10 ng
</TD></TR><TR><TD>Nuclease-Free Water
</TD><TD>to 50 μl
</TD></TR></TBODY></TABLE><P>Thermocycling Conditions for a Routine PCR:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>STEP</B></TD><TD><B>TEMP</B></TD><TD><B>TIME</B></TD></TR><TR><TD>Initial Denaturation
</TD><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD rowspan="3">45 cycles</TD><TD>98 °C</TD><TD>10 sec
</TD></TR><TR><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD>60 °C
</TD><TD>90 sec
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>120 sec
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI> GSY8 negative: smear at the upper end of the gel.</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P270_.26_P308">Preparative digestion of P270 &amp; P308</SPAN></H3><P><B>Investigator:  </B> Clemens
</P><P><B>Aim of the experiment:</B></P><P>Preparation for cloning in pSB1C3
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P270 with EcoRI-HF &amp; Spe-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>Plasmid DNA P270
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>0 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P308 with EcoRI-HF and XbaI</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>Plasmid DNA P308
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestions were incubated for 3h at 37°C</LI><LI> Fragments were cut from the gel and purified by gelectraction (promega)</LI></UL><P><B>Results:</B></P><UL><LI> Fragments were renamed (see Inventarliste) and concentrations were measured</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2_.28PmeI.2BNotI.29_in_pSB1C3">Cloning of pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2 (PmeI+NotI) in pSB1C3</SPAN></H3><P><B>Investigator:  CG</B></P><P><B>Aim of the experiment:</B></P><P>Cloning of pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2 (PmeI+NotI) in pSB1C3
</P><P><B>Procedure:</B></P><UL><LI> 17 µl pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2 (PmeI+NotI) digestion with 1 µl Mung Bean Nuclease and 2 µl buffer for 30 min. at 30°C to receive blunt ends</LI><LI> Ligation of  8.7 µl pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2 (PmeI+NotI) and 1.3 µl pSB1C3</LI><LI> Overnight incubation at 16 °C</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Retransformation_of_P258">Retransformation of P258</SPAN></H3><P><B>Investigator: CG </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Amplification of Vector</LI></UL><P><B>Procedure:</B></P><UL><LI> Transformation following SOP(13.02.2013) </LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Digestion_of_P1005.2C_P1006.2C_P1007_with_XbaI.2C_PstI_HF_.2B_analytical_gel">Digestion of P1005, P1006, P1007 with XbaI, PstI HF + analytical gel</SPAN></H3><P><B>Investigator: EF </B></P><P><B>Aim of the experiment:</B> Isolation of insert for further construction of hypoxia system
<B>Procedure:</B></P><UL><LI> p1005, p1006, p1007 digested with: </LI></UL><P>0,25 µl EcoRI (HF) 
0,25 µl SpeI (HF), 
5 µl CutSmart Buffer 
3 µg plasmid-DNA (Pp1005, p1006, p1007)
fill up with ddH2O (Vtotal= 50µL)
</P><UL><LI> 30 µl on 1% agarose gel for electrophoresis (2log DNA-ladder as standard)</LI></UL><UL><LI> Digestions were incubated for 1h at 37°C</LI></UL><P><B>Results:</B> Insert and vector were clearly separated. All cut vector remained at the expected position of 2kb. Inserts were isolated following extraction kit protocol . F number and concentrations were included into the Inventarliste. Remain of 30µl (with dye) frozen at -20 C -&gt; Labeled as 5.1; 6,1; 7.2  
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P1007_with_EcoRI_HF.2C_SpeI_HF_.2B_analytical_gel">Digestion of  P1007 with EcoRI HF, SpeI HF + analytical gel</SPAN></H3><P><B>Investigator: EF </B></P><P><B>Aim of the experiment:</B> Isolation of vector. Verification of digestion performed on 29th, Friday
<B>Procedure:</B></P><UL><LI>p1007 digested with: </LI></UL><P>0,25 µl EcoRI (HF) 
0,25 µl SpeI (HF), 
5 µl CutSmart Buffer 
3 µg plasmid-DNA (p1007)
fill up with ddH2O (Vtotal= 50µL)
</P><UL><LI> Digestion was incubated for 1h at 37°C</LI></UL><UL><LI> 30 µl on 1% agarose gel for electrophoresis (2log DNA-ladder as standard)</LI></UL><P><B>Results:</B> Again plasmid wasn´t cut. One single band at 3kb. Further procedur: sequence plasmid (?) repeat ligation to generate p1007. Isolate pSB1C3 from another plasmid.
</P><P>[File to be uploaded]
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F1001_and_F1002">Ligation of F1001 and F1002</SPAN></H3><P><B>Investigator: EF </B></P><P><B>Aim of the experiment:</B> generate p1007 due to gel results (since 28th, Thursday)
<B>Procedure:</B>  Ligation was performed as follows: 6,3 µl F1001, 5,3 µl F1002, 2 µl T4-DNA-Ligase Buffer, 1 µl T4-DNA-Ligase, filled up with ddH20 to 20 µl *3:1 vector-insert ratio 
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation_of_F248_and_1015">Ligation of F248 and 1015</SPAN></H3><P><B>Investigator: EF </B>
'<I>Aim of the experiment:</I> hypoxia
<B>Procedure:</B> Ligation was performed as follows: 1 µl F248, 5 µl F1015, 2 µl T4-DNA-Ligase Buffer, 1 µl T4-DNA-Ligase, filled up with ddH20 to 20 µl *Approximate 3:1 vector insert ratio  
</P><P><B>Results</B>: Plates incubating over night at 37C 
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P259_.28SA_from_GSY6.3B_P303_and_P304_.28eMA.29">Sequencing of P259 (SA from GSY6; P303 and P304 (eMA)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Procedure:</B></P><UL><LI> Sequencing batch were prepared after manufacturer's protocol. (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer) - all samples: VF2</LI><LI> barcodes in stock list</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_with_P6.2C_pASK111.2C_P268">Picking of of <I>E. coli</I> XL1 blue with P6, pASK111, P268</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Procedure:</B></P><UL><LI> 2 colonies were picked.</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight</LI></UL></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inocculation_of_preculture_.28P310_eMA-HisTag.29">Inocculation of preculture (P310 eMA-HisTag)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Procedure:</B></P><UL><LI> 2 colonies were picked (20 mL LB with Cam)</LI></UL><UL><LI> the preculture was transferred in a cell culture shaker at 37 °C and were incubated overnight</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="harvesting_SUMO_Protease">harvesting SUMO Protease</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> SUMO Protease production
</P><P><B>Procedure:</B></P><UL><LI> 6 x 2 L cultures</LI><LI> centrifuge cultures for 30 min at 6000 rpm</LI><LI> freeze pellet (120.5 g)</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_Aug_31st"><B>Wednesday, Aug 31st</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Over_night_liquid_cultures_of_P1007_and_P1012">Over night liquid cultures of P1007 and P1012</SPAN></H3><P><B>Investigator: EF </B></P><P><B>Aim of the experiment:</B></P><P>Obtaining enough cellmass for plasmid preparation. Liquid cultures of Turbo and XL1 blue cells.
</P><P><B>Procedure:</B></P><UL><LI> Inoculation of 5 mL LB-medium with colonies of foregone transformation</LI><LI> Incubate over night at 37 °C in shaker</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Phosphorylation_of_F1036">Phosphorylation of F1036</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Phosphorylation of F1036</LI></UL><P><B>Procedure:</B></P><UL><LI> 36 µL DNA (*350 ng)</LI><LI> 4 µL T4 ligase buffer</LI><LI> 1 µL PNK</LI><LI> 30 min; 37 °C </LI><LI> heat deactivation: 20 min; 65 °C </LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F1029_.2B_F1037_.3D_F1038">Ligation: F1029 + F1037 = F1038</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><P>Ligation: F1029 + F1037 = F1038
</P><P><B>Procedure:</B></P><UL><LI> 0.5 µL F1029 (30 ng\µL)</LI><LI> 10 µL F1037 (9 ng\µL)</LI><LI> 2 µL  T4 ligase buffer </LI><LI> 1 µL T4 ligase</LI><LI> 7 µL ddH2O</LI><LI> over night ; 16 °C</LI></UL><P><B>Results:</B></P><UL><LI> F1029 + F1037 = F1038</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_various_plasmids.2C_analytical_digestion.2C_sequencing">Miniprep of various plasmids, analytical digestion, sequencing</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Obtain plasmids for: eMA, PAS-Lys-Sumo, pASK111, Streptavidin (non-codon-optimized)
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed according to manufacturers protocol (QiaPrep, QIAGEN)</LI><LI> Digestion of about 250 ng for each eMA plasmid with EcoRI and PstI</LI></UL><P><B>Results:</B></P><UL><LI> Names and concetrations in &quot;Inventarliste&quot;</LI><LI> Sequencing results</LI><LI> Gel: </LI></UL><P>1. GSY8, 2. - 5.: P319-P322 with EcoRI / PstI
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Cloning_of_Streptavidin_.28non-codon-optimized.29_into_pASK111">Cloning of Streptavidin (non-codon-optimized) into pASK111</SPAN></H3><P><B>Investigator: JL, NA, CG</B></P><P><B>Aim of the experiment:</B> Obtain Streptavidin (non-codon-optimized) in expression vector
</P><P><B>Procedure:</B></P><UL><LI> Preparative digestion of 42 µl P6 (Streptavidin) and 42 µl P315, 5 µl CutSmart and 1.5 µl XbaI/HindIII</LI><LI> Gel extraction</LI><LI> Ligation: 5.9 µl pASK111 + 4.1 µl SA + 2 µl ligase buffer + 7 µl ddH2o + 1 µl T4 ligase</LI><LI> Transformation according to SOP</LI></UL><P><B>Results:</B></P><UL><LI> Gel: </LI></UL><P>1. Streptavidin (non-codon-optimized), 2. pASK111
</P></DIV><DIV class="kill-switch"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P296_and_P73">Preparative digestion of P296 and P73</SPAN></H3><P><B>Investigator: EU</B></P><P><B>Aim of the experiment:</B> Prepare construct for ligation later on
</P><P><B>Procedure:</B></P><P>5 µl DNA
36µl ddH2O
2µl of each enzyme (P296 --&gt; EcoRI&amp;SpeI, P73 --&gt; EcoRI&amp;XbaI)
5µl cut smart buffer
</P><P>incubation at 37°C for 3 h
Gel electrophoresis and gel extraction by Gel extraction kit (Promega) according  to manufacturer’s protocol
</P><P><B>Results:</B></P><P>Fragments were expected at about 1 kb (P296) and 2,5 kb (P73). This could be shown comparing with 1kb DNA ladder. 
</P></DIV><DIV class="kill-switch"><H3><SPAN class="mw-headline" id="Ligation:_F269_.2B_F270_.3D_F271">Ligation: F269 + F270 = F271</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Ligation: F269 + F270 = F271</LI></UL><P><B>Procedure:</B></P><UL><LI> 5 µL F269 ( ~  ng)</LI><LI> 10 µL F270 (~  ng)</LI><LI> 2 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 2 µL ddH2O</LI><LI> ONT; 16 °C</LI></UL><P><B>Results:</B></P><UL><LI> F269 + F270 = F271 </LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_and_purification_of_GSY7_new_and_GSY8_2">PCR and purification of GSY7_new and GSY8</SPAN></H3><P><B>Investigator: CG, EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Amplification of GSY 7_new and GSY 8 </LI></UL><P><B>Procedure:</B></P><P>
Same procedure for both GSYs:
</P><UL><LI> PCR using PfuUltra II Fusion HotStart DNA Polymerase  according to manufacturer’s online protocol</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>10 µL reaction</B></TD><TD><B>Final concentration</B></TD></TR><TR><TD>PfuUltra II Hotstart PCR Master Mix
</TD><TD>25 µL
</TD><TD>1 X
</TD></TR><TR><TD>10 μM Forward Primer (O-038)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>10 μM Reverse Primer (O-039)
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>Template DNA
</TD><TD>1 µL
</TD><TD>10 ng
</TD></TR><TR><TD>Nuclease-Free Water
</TD><TD>to 50 μl
</TD></TR></TBODY></TABLE><P>Thermocycling Conditions for a Routine PCR:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>STEP</B></TD><TD><B>TEMP</B></TD><TD><B>TIME</B></TD></TR><TR><TD>Initial Denaturation
</TD><TD>98 °C
</TD><TD>2 min
</TD></TR><TR><TD rowspan="3">40 cycles</TD><TD>98 °C</TD><TD>10 sec
</TD></TR><TR><TD>98 °C
</TD><TD>20 sec
</TD></TR><TR><TD>45-64,7 °C (gradient)
</TD><TD>20 sec
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>30 sec
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><P>Purification according to Promega PCR purification kit
</P><P><B>Results:</B>
no amplification recognizable
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="harvesting_SUMO_Protease_2">harvesting SUMO Protease</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> SUMO Protease production
</P><P><B>Procedure:</B></P><UL><LI> resuspend pellet in PBS + 10 mM EDTA</LI><LI> homogenize cells with PANDA</LI><LI> centrifuge cultures for 30 min at 8000 rpm (protein in supernantant)</LI><LI> dialysis against IMAC buffer</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Inocculation_and_induction_of_eMA-HisTag_.28P310.29">Inocculation and induction of eMA-HisTag (P310)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> eMA-HisTag production
</P><P><B>Procedure:</B></P><UL><LI> 2 L LB shaking flask</LI></UL><UL><LI> the cell culture was incubated at 37 °C </LI><LI> induction with 200 µl anhydro tetracycliine (2mg/ml)  at OD= 0,55</LI><LI> production over night</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_Sep_1st"><B>Thursday, Sep 1st</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Miniprep_of_F261.2C_F262.2C_F263.2C_F264.2C_I739001.2C_P305_.26_P258_.2Canalytical_digestion_and_sequencing">Miniprep of F261, F262, F263, F264, I739001, P305 &amp; P258 ,analytical digestion and sequencing</SPAN></H3><P><B>Investigator: Clemens </B></P><P><B>Aim of the experiment:</B> 
Confirm sucessful cloning
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed according to manufacturer's protocol (QIAprep, Qiagen)</LI><LI> Analytic digestion was performed to check if the cloning was successful</LI></UL><UL><LI> Batch for analytical digestion of P323/P324 with PstI-HF and EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl/ 0.5 µl
</TD><TD>Plasmid DNA P323/P324
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF (10 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>7.5 µl/ 7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion of P325/P326 with PstI-HF and EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl/ 0.5 µl
</TD><TD>Plasmid DNA P326/P325
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF (10 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>7.5 µl/ 7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion of P327/P328 with PstI-HF and EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl/ 0.5 µl
</TD><TD>Plasmid DNA P328/P327
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF (10 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>7.5 µl/ 7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion of P329/P330 with PstI-HF and EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.5 µl/ 0.5 µl
</TD><TD>Plasmid DNA P329/P330
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF (10 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>7.5 µl/ 7.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion of P331/P332 with PstI-HF and EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.5 µl/ 0.5 µl
</TD><TD>Plasmid DNA P331/P332
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF (10 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>7.5 µl/ 7.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion of P333/P334/P322 with AgeI-HF and EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.5 µl/ 0.5 µl/ 0.5 µl
</TD><TD>Plasmid DNA P333/P334/P322
</TD></TR><TR><TD>1 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF (10 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>7.5 µl/ 7.5 µl/ 7.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestions were incubated for 1.5h at 37°C</LI><LI> Samples were run on an 1% agarose gel for 45 min at 90V</LI></UL><P><B>Results:</B></P><UL><LI> Except for P329, P330 and P322 all fragments were of the expected size</LI><LI> Cloning of the inserts of P29 and P30 was started again.</LI></UL><P>For confirmation of the Sequence the following plasmids were send for sequencing:
</P><UL><LI> P323 (VF2 &amp; VR)</LI><LI> P331 (VF2)</LI><LI> P333 (VF2)</LI><LI> P328 (VF2)</LI><LI> P326 (VF2)</LI></UL></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P333.2C_P308.2C_P331_and_P301">Preparative digestion of P333, P308, P331 and P301</SPAN></H3><P><B>Investigator: Clemens </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Cloning of new constructs</LI></UL><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P333 with NgOMIV and PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>Plasmid DNA P333
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>NgOMIV
</TD></TR><TR><TD>19 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P333 with AgeI-HF and PstI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>Plasmid DNA P333
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>19 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P308 with XbaI and EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>Plasmid DNA P308
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>10 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P333 with SpeI-HF and EcoRI-HF</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>8 µl
</TD><TD>Plasmid DNA P331
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P301 with PstI-HF and NgOMIV</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>Plasmid DNA P301
</TD></TR><TR><TD>3 µl
</TD><TD>Cutsmart buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>NgOMIV
</TD></TR><TR><TD>19 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=30 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestions were incubated for 3h at 37°C</LI><LI> Samples were applied on a 1% agarose gel and run at 90V for 45min</LI></UL><P><B>Results:</B></P><UL><LI> Digestions of P308, P331 and P301 showed the expected fragment sizes.</LI><LI> These fragments were cut from the gel and purified by gelextraction (Promega)</LI><LI> Samples were renamed and given Fragmentnumbers</LI></UL><UL><LI> The Digestions of P333 showed fragments of unexpected length. The Ligation of eMA in pSB1C3 (previous step) failed. This ligation has to be repeated.</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_and_purification_of_GSY7_new_and_GSY8_3">PCR and purification of GSY7_new and GSY8</SPAN></H3><P><B>Investigator: MP, EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Amplification of GSY 7_new and GSY 8 </LI></UL><P><B>Procedure:</B></P><UL><LI>PCR Using Q5® High-Fidelity DNA Polymerase (M0491) according to online protocol</LI><LI> each GSY was amplified two times; once with Q5 GC-Enhancer and once without</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>25 µL reaction</B></TD><TD><B>Final concentration</B></TD></TR><TR><TD>5X Q5 Reaction Buffer
</TD><TD>5 µL
</TD><TD>1 X
</TD></TR><TR><TD>10 mM dNTPs
</TD><TD>0.5 µL
</TD><TD>200 μM
</TD></TR><TR><TD>10 μM Forward Primer (D20)
</TD><TD>1.25 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>10 μM Reverse Primer (VR)
</TD><TD>1.25 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>Template DNA
</TD><TD>1 µL
</TD><TD>10 ng
</TD></TR><TR><TD>Q5 High-Fidelity DNA Polymerase
</TD><TD>0.25 μl
</TD><TD>0.02 U/μl
</TD></TR><TR><TD>Nuclease-Free Water
</TD><TD>to 25 μl (minus 5µl for GC-Enhancer respectively)
</TD></TR></TBODY></TABLE><P>Thermocycling Conditions for a Routine PCR:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>STEP</B></TD><TD><B>TEMP</B></TD><TD><B>TIME</B></TD></TR><TR><TD>Initial Denaturation
</TD><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98 °C</TD><TD>5 - 10 sec
</TD></TR><TR><TD>61 °C
</TD><TD>30 sec
</TD></TR><TR><TD>72 °C
</TD><TD>60 sec
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>120 sec
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD></TR></TBODY></TABLE><P>Note: Use of the NEB T m Calculator is highly recommended.
</P><P>Purification according to Promega PCR purification kit
</P><P><B>Results:</B>
no amplification GSY8 recognizable, GSY7did work
</P></DIV><DIV class="kill-switch"><H3><SPAN class="mw-headline" id="Transformation_of_F271">Transformation of F271</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Transformation of F271 for further application</LI></UL><P><B>Procedure:</B></P><UL><LI> 3 µL DNA were added to blr cells</LI><LI> incubation on ice for 20 min</LI><LI> heatshock in 42°C for 40s</LI><LI> incubation on ice for 4 min</LI><LI> 700µl LB-medium were added</LI><LI> incubation for 1h in 37°C shaker</LI><LI> centrifugation for 3 min</LI></UL><P><B>Results:</B></P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_F1038">Transformation of F1038</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Transformation of F1038 for further application</LI></UL><P><B>Procedure:</B></P><UL><LI> 3 µL DNA were added to blr cells</LI><LI> incubation on ice for 20 min</LI><LI> heatshock in 42°C for 40s</LI><LI> incubation on ice for 4 min</LI><LI> 700µl LB-medium were added</LI><LI> incubation for 1h in 37°C shaker</LI><LI> centrifugation for 3 min</LI></UL><P><B>Results:</B></P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="harvesting_of_eMA-HisTag">harvesting of eMA-HisTag</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> eMA-HisTag production
</P><P><B>Procedure:</B></P><UL><LI> centrifugate at 6000 rpm for 30 min (OD= 6.15)</LI><LI> 11,1 g pellet</LI><LI> SDS-PAGE after protocol </LI><LI> resuspend pellet in 20 mM Tris/HCl + 50 mM NaCl</LI><LI> homogenize cells with PANDA</LI><LI> centrifuge 30 min at 18000 rpm </LI><LI> denaturate inclusion bodies in 16 ml Gnd-Chl (pH 1.5) over night</LI></UL><P><B>results:</B></P><P>marker, eMA before induction, eMA after induction, ULP after induction (SUMO protease)
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_Sep_2nd"><B>Friday, Sep 2nd</B></SPAN></H2><DIV class="receptor"><H3><SPAN class="mw-headline" id="Ligation_of_F274.2BF275.2C_F276.2BF260_and_F279.2BF240">Ligation of F274+F275, F276+F260 and F279+F240</SPAN></H3><P><B>Investigator: </B> Clemens
</P><P><B>Aim of the experiment:</B></P><UL><LI> Cloning of new constructs</LI></UL><P><B>Procedure:</B></P><P><B>Results:</B></P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="harvesting_of_eMA-HisTag_2">harvesting of eMA-HisTag</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> eMA-HisTag production
</P><P><B>Procedure:</B></P><UL><LI> add 4 ml Gnd-Chl because pellet did not dissolve over night, continue shaking for few hours</LI><LI> centrifugate for 15 min at 15000 rpm (protein in supernatant)</LI><LI> refolding in 1.5 L cold PBS over night </LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="purification_of_SUMO_protease_using_.C3.84KTA_purifier">purification of SUMO protease using ÄKTA purifier</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> eMA-HisTag production
</P><P><B>Procedure:</B></P><UL><LI> centrifugation</LI><LI> destroying DNA using ultrasonic for 10 min</LI><LI> sterilfiltration</LI><LI> purification due to HisTag affinity of protein to nickel in column (buffer: 20 mM Tris/HCl + 500 mM NaCl</LI><LI> over night</LI></UL><P><B>Procedure:</B></P><UL><LI> purification stopped because of pressure alarm, was restarted an showed no SUMO protease in the sample</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Cloning_of_extended_MCS_into_pDSG_and_pcDNA5_vector">Cloning of extended MCS into pDSG and pcDNA5 vector</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Extending the MCS of pDSG and pcDNA5 for cloning of BioBricks
</P><P><B>Procedure:</B></P><UL><LI> Digestion of 5 µl P291 (pDSG) and 15 µl P339 (pcDNA) with SphI and NruI-HF for pDSG and MluI-HF for pcDNA5</LI><LI> Gelextraction of both signals at about 4 kb</LI><LI> Oligo-hybridzation of O35/O36 and O36/O37 wo prior phosphorylation to obtain a total concentration of 17 ng/µl</LI><LI> Ligation of oligo-hybridzation and vector-fragements according to calculations in the Inventarliste and overnight incubation at 16°C</LI></UL><P><B>Procedure:</B></P><UL><LI> expected signals after digestion: pDSG with 4 kb and 1.7 kb (lane 1), pcDNA with 4 kb and 1 kb (lane 2)</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_2x_pcDNA5_vector_and_2x_Streptavidin_.28nco.29_in_pASK111">Miniprep of 2x pcDNA5 vector and 2x Streptavidin (nco) in pASK111</SPAN></H3><P><B>Investigator: CG</B></P><P><B>Aim of the experiment:</B> Obtain plasmids for pcDNA5 vector and 2x Streptavidin (nco) in pASK111
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed according to manufacturers protocol (QiaPrep, QIAGEN)</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_Sep_3nd"><B>Saturday, Sep 3nd</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_F285.3B_F286">Transformation of F285; F286</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><P>Amplification of F285; F286
</P><P><B>Procedure:</B></P><UL><LI> Transformation following SOP(13.02.2013) </LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P1009_.2B_PmeI.26NotI">Digestion of P1009 + PmeI&amp;NotI</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of P1009 + PmeI&amp;NotI </LI></UL><P><B>Procedure:</B></P><UL><LI> 33 µL DNA (~600 ng/µL)</LI><LI> 2 µL PmeI</LI><LI> 1 µL NotI  </LI><LI> 9 µL ddH2O </LI><LI> 5 µL CutSmart buffer</LI><LI> incubation for 1h at 37 °C</LI></UL><P><B>Results:</B></P><UL><LI> F1039  80 ng/µL </LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_Sep_4th"><B>Sunday, Sep 4th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Phosphorylation_of_F1040_.28PCR_product_GSY7.29">Phosphorylation of F1040 (PCR product GSY7)</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Phosphorylation of F1040 (PCR product GSY7) </LI></UL><P><B>Procedure:</B></P><UL><LI> 15 µL F1040 (20 ng/µL)</LI><LI> 5 µL T4 Ligase buffer </LI><LI> 1 µL T4 PNK</LI><LI> 29 µL ddH20</LI><LI> 30 min; 37 °C</LI><LI> heat inactivation: 20 min; 65 °C</LI></UL><P><B>Results:</B></P><UL><LI> F1041 (PCR product GSY7)</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Mung_Bean_nuclease_.28MBN.29_digestion_of_F1039_.28pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2_.5BPmeI.2BNotI.5D.29">Mung Bean nuclease (MBN) digestion of F1039 (pAAV-CWS_MmZip14_EMCV-IRES_mTagBFP2 [PmeI+NotI])</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Get rid of overhangs. Preparation of DNA for blunt ligation</LI></UL><P><B>Procedure:</B></P><UL><LI> 18 µL F1039 ( 80 ng/µL)</LI><LI> 2 µL  10x CutSmart buffer</LI><LI> 0.25 µL MBN (10 u/µL)</LI><LI> 60 min; 30 °C</LI><LI> purification by PCR purification kit, Promega</LI></UL><P><B>Results:</B></P><UL><LI> F1042 30 ng\µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F1042_.2BF1025_.3D_F1043">Ligation: F1042 +F1025 = F1043</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F1042 +F1025</LI></UL><P><B>Procedure:</B></P><UL><LI> 5.5 µL F1025 (9.2 ng\µL)</LI><LI> 8.2 µL F1042 (35 ng\µL)</LI><LI> 2 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 3.5 µL ddH2O</LI><LI> over night; 16 °C</LI></UL><P><B>Results:</B></P><UL><LI> F1042 +F1025 = F1043</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F1041_.2BF1029_.3D_F1044">Ligation: F1041 +F1029 = F1044</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F1041 +F1029</LI></UL><P><B>Procedure:</B></P><UL><LI> 1.6 µL F1029 (30 ng\µL)</LI><LI> 23 µL F1041 (6 ng\µL)</LI><LI> 3 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 1.3 µL ddH2O</LI><LI> over night; 16 °C</LI></UL><P><B>Results:</B></P><UL><LI> F1041 +F1029 = F1044</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Cloning_of_SUMO-PAS_in_pASK75">Cloning of SUMO-PAS in pASK75</SPAN></H3><P><B>Investigator: JL, EF </B></P><P><B>Aim of the experiment:</B> 
Ligation of F158 and &quot;P310&quot; (JL) (SUMO+PAS-Lysin in pASK75):
</P><UL><LI>5,6 µL</LI><LI>2,4 µL</LI><LI>2 µL</LI><LI>1 µL T4-DNA-Ligase</LI><LI>fill up with ddH2O to 20 µL</LI></UL><P>Transformation in NEB Turbo cells of 10 µL ligation
</P></DIV><H1><SPAN class="mw-headline" id="Week_17.2C_Sep_5th_-_Sep_11th">Week 17, Sep 5th - Sep 11th</SPAN></H1><DIV class="week" id="Week_17"><H2><SPAN class="mw-headline" id="Monday.2C_Sep_5th"><B>Monday, Sep 5th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="harvesting_of_eMA-HisTag_3">harvesting of eMA-HisTag</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> eMA-HisTag production
</P><P><B>Procedure:</B></P><UL><LI> centrifugate after refolding at 11500 rpm for 30 min </LI></UL><P><B>results:</B></P><P>marker, eMA before induction, after induction, after denaturation and refolding 
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Inocculation_of_P341_.28E.coli_Bl-21.3B_50_ml_LB.29_in_M9_medium_for_fermentation">Inocculation of P341 (E.coli Bl-21; 50 ml LB) in M9 medium for fermentation</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Procedure:</B></P><UL><LI> M9 medium with Cam (1:1000) was inocculated with preculture (~50 mL) of P258 (pASK111+Streptavidin) </LI></UL><UL><LI> the culture was incubated in the shaker over night at 37°C</LI></UL><P><B>results:</B></P><UL><LI> a 50 ml culture with Ecoli Bl21 was induced</LI></UL><P>the SDS PAGE shows no streptavidin expression after induction 
</P><UL><LI> next fermentation in pSA (not pASK111)</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_Sep_6th"><B>Tuesday, Sep 6th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Phosphorylation_of_F1005.2C_F1016">Phosphorylation of F1005, F1016</SPAN></H3><P><B>Investigator: EF </B></P><P><B>Aim of the experiment:</B>  to be completed
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation_of_F1045.2C_F1046">Ligation of F1045, F1046</SPAN></H3><P><B>Investigator: EF </B></P><P><B>Aim of the experiment:</B>  to be completed
</P><P>Ligation of F1005, F10016 (GelEx GSY8) in F1013
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_Sep_7th"><B>Wednesday, Sep 7th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="harvesting_of_eMA-HisTag_4">harvesting of eMA-HisTag</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> eMA-HisTag production
</P><P><B>Procedure:</B></P><UL><LI> precipitation with ammonium sulfate gradient (10% to 90%)</LI><LI> the percentage was increased in 10% steps, after each addition of ammonium sulfate the solution was centrifugated at 8500 rpm for 20 min</LI><LI> SDS-PAGE: samples of the supernatants</LI></UL><P><B>results:</B></P><P>The pellets of the precipitations with 50%, 60% and 70% were pooled and dialysed against IMAC buffer
Next step: Äkta
</P></DIV><DIV class="receptor"><H3><SPAN class="mw-headline" id="MidiPrep_of_pSB1C3_with_RFP_generator_.28P4.29_3">MidiPrep of pSB1C3 with RFP generator (P4)</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> purification of CMV Nanoluc StrepTag PolyA (P312)
</P><P><B>Procedure:</B>   
MidiPrep was performed according to manufacturer's protocol (QIAprep MidiPrep, Qiagen)
</P><P><B>Results:</B></P><P>3268 ng/µl in 80 µl
</P></DIV><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Inocculation_of_fermenter_3">Inocculation of fermenter</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> streptavidin production
</P><P><B>Procedure:</B></P><UL><LI> inocculation with 50 ml preculture (P20 in LB medium) </LI><LI> pO<SUB>2</SUB>= 60 %; Temp= 37°C, pH~7</LI><LI> growth over night</LI><LI> OD next morning: 0.4</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_Sep_8th"><B>Thursday, Sep 8th</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="Fermentation_3">Fermentation</SPAN></H3><P><B>Investigator:</B> NA
</P><P><B>Aim of the experiment:</B> Streptavidin production
</P><P><B>Procedure:</B></P><UL><LI> exponential feed (Amp) for ~13 h (about 3-4 L); start at OD= 7.5 (glucose test negative)</LI><LI> Induction of streptavidin expression (IPTG) at OD= 30</LI><LI> production over night</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_Sep_9th"><B>Friday, Sep 9th</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="harvesting_streptavidin_3">harvesting streptavidin</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> streptavidin production
</P><P><B>Procedure:</B></P><UL><LI> OD= 41.4</LI><LI> centrifuge culture for 1 h at 4800 rpm</LI><LI> resuspend pellet (275 g)in 20mM Tris HCl + 500 mM NaCl(pH 8)</LI><LI> homogenize with PANDA</LI><LI> the resulting lysate was transferred into centrifuge tubes and spun down (4°C, 9000 rpm, 30 mins). The supernatant was cast away and the pellet was resuspended in 6M Gua-HCl (pH = 1.5) at 4°C overnight.</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_Sep_10th"><B>Saturday, Sep 10th</B></SPAN></H2><H2><SPAN class="mw-headline" id="Sunday.2C_Sep_11th"><B>Sunday, Sep 11th</B></SPAN></H2><DIV class="streptavidin"><H3><SPAN class="mw-headline" id="harvesting_streptavidin_4">harvesting streptavidin</SPAN></H3><P><B>Investigator: NA</B></P><P><B>Aim of the experiment:</B> streptavidin production
</P><P><B>Procedure:</B></P><UL><LI> the denaturated protein solution was refolded in 15 L 1x PBS (added drop wise) over night</LI><LI> the solution was centrifugated at 9500 rpm for 20 minutes, the pellet was stored in the freezer, the supernatant will be concentrated via cross flow and precipitated with ammonium sulfate afterwards</LI></UL><P><B>results:</B></P></DIV><H1><SPAN class="mw-headline" id="Week_18.2C_Sep_12th_-_Sep_18th">Week 18, Sep 12th - Sep 18th</SPAN></H1><DIV class="week" id="Week_18"><H2><SPAN class="mw-headline" id="Tuesday.2C_Sep_13th"><B>Tuesday, Sep 13th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P74">Digestion of P74</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of P74</LI></UL><P><B>Procedure:</B></P><P>F1055:
</P><UL><LI> 2 µL DNA (1431 ng/µL)</LI><LI> 2 µL CutSmart buffer</LI><LI> 0.25 µL HincII (10 u/µL) </LI><LI> 16 µL ddH2O </LI><LI> incubation for 1h at 37 °C</LI></UL><P>F1056:
</P><UL><LI> 2 µL DNA (1431 ng/µL)</LI><LI> 2 µL CutSmart buffer</LI><LI> 0.25 µL HincII (10 u/µL)</LI><LI> 0.25 µL AgeI (20 u/µL)  </LI><LI> 15.5 µL ddH2O </LI><LI> incubation for 1h at 37 °C</LI></UL><P>F1057:
</P><UL><LI> 2 µL DNA (1431 ng/µL)</LI><LI> 2 µL CutSmart buffer</LI><LI> 0.25 µL HincII (10 u/µL)</LI><LI> 0.25 µL NgoMIV  </LI><LI> 15.5 µL ddH2O </LI><LI> incubation for 1h at 37 °C</LI></UL><P><B>Results:</B></P><P>1Kb ruler, F1055[HincII], F105[HincII&amp;AgeI]6, F1057[HincII&amp;NgoMIV], P1017[EcoRI&amp;SpeI]
</P><UL><LI> F1055 49.5 ng/µL</LI><LI> F1056 37.0 ng/µL</LI><LI> F1057 29.7 ng/µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P1017">Analytical digestion of P1017</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Analytical digestion of P1017</LI></UL><P><B>Procedure:</B></P><UL><LI> 2 µL DNA (312 ng/µL)</LI><LI> 2 µL CutSmart buffer</LI><LI> 0.25 µL SpeI</LI><LI> 0.25 µL EcoRI</LI><LI> 15.5 µL ddH2O </LI><LI> incubation for 1h at 37 °C</LI></UL><P><B>Results:</B></P><P>1Kb ruler, F1055[HincII], F105[HincII&amp;AgeI]6, F1057[HincII&amp;NgoMIV], P1017[EcoRI&amp;SpeI]
</P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="PCR_of_GSY10">PCR of GSY10</SPAN></H3><P><B>Investigator: MT</B></P><P><B>Aim of the experiment:</B></P><P>Amplification of GSY10
</P><P><B>Procedure:</B></P><UL><LI> PCR Using Q5® High-Fidelity DNA Polymerase (M0491) according to online protocol</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>50 µL reaction</B></TD><TD><B>Final concentration</B></TD></TR><TR><TD>Q5® High-Fidelity 2X Master Mix
</TD><TD>25 µL
</TD><TD>1 X
</TD></TR><TR><TD>10 μM VR
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>10 μM VF2
</TD><TD>2,5 μl
</TD><TD>0.5 μM
</TD></TR><TR><TD>Template DNA
</TD><TD>0.1 µL
</TD><TD>1 ng
</TD></TR><TR><TD>Nuclease-Free Water
</TD><TD>to 50 μl
</TD></TR></TBODY></TABLE><P>Thermocycling Conditions for a Routine PCR:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>STEP</B></TD><TD><B>TEMP</B></TD><TD><B>TIME</B></TD></TR><TR><TD>Initial Denaturation
</TD><TD>98 °C
</TD><TD>30 sec
</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>98 °C</TD><TD>10 sec
</TD></TR><TR><TD>62 °C
</TD><TD>30 sec
</TD></TR><TR><TD>72 °C
</TD><TD>60 sec
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>120 sec
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Dephosphorylation_of_F1055">Dephosphorylation of F1055</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Preparation of Vekor for blunt end Ligation </LI></UL><P><B>Procedure:</B></P><UL><LI> 15 µL F1055 (~50 ng\µL)</LI><LI> 1 µL FastAP (1 u/µL) </LI><LI> 2 µL  10x FastAP buffer</LI><LI> 10 min ; 37 °C</LI><LI> heat inactivation: 5 min; 75 °C</LI></UL><P><B>Results:</B></P><UL><LI> F348</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P1017">Digestion of P1017</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of P1017</LI></UL><P><B>Procedure:</B></P><UL><LI> 10 µL DNA (312 ng/µL)</LI><LI> 5 µL CutSmart buffer</LI><LI> 0.25 µL SpeI </LI><LI> 0.25 µL PstI</LI><LI> 34.5 µL ddH2O </LI><LI> incubation for 1h at 37 °C</LI></UL><P><B>Results:</B></P><P>1Kb ruler, P1017[SpeI&amp;PstI], P1017[SpeI&amp;PstI], F362: PCR-GSY7[AgeI], PCR-GSY7, PCR-GSY8
</P><UL><LI> F350</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Over_night_liquid_culture_of_P1009">Over night liquid culture of P1009</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Getting enough cellmass for plasmid preparation</LI></UL><P><B>Procedure:</B></P><UL><LI> Inoculation of 5 mL LB-medium (containing antibiotica) with colonies of foregone transformation</LI><LI> Incubate over night at 37 °C in shaker</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_Sep_14th"><B>Wednesday, Sep 14th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Gelextraction_of_F357.5BNgoMIV.5D.3B_P1006.5BXbaI.26PstI.5D">Gelextraction of F357[NgoMIV]; P1006[XbaI&amp;PstI]</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Gelextaction of F357[NgoMIV]; P1006[XbaI&amp;PstI]</LI></UL><P><B>Procedure:</B></P><UL><LI> Using gelextraction kit, Promega</LI></UL><P><B>Results:</B></P><P>1kb ladder, F357[NgoMIV]; P1006[XbaI&amp;PstI]
</P><UL><LI> F357[NgoMIV] -&gt; F364 41 ng/µL</LI><LI> P1006[XbaI&amp;PstI] -&gt; F365 9.1 ng/µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F1057_.2B_F364_.3D_F366">Ligation: F1057 + F364 = F366</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F1057 + F364</LI></UL><P><B>Procedure:</B></P><UL><LI> 1,6 µL F1057 (29.7 ng\µL)</LI><LI> 2.1 µL F364 (41 ng\µL)</LI><LI> 3 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 22.3 µL ddH2O</LI><LI> 30 min; RT</LI></UL><P><B>Results:</B></P><UL><LI> F1057 + F364 = F366</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F350_.2B_F365_.3D_F367">Ligation: F350 + F365 = F367</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F350 + F365</LI></UL><P><B>Procedure:</B></P><UL><LI> 2,6 µL F350 (19 ng\µL)</LI><LI> 2.1 µL F365 (9.1 ng\µL)</LI><LI> 3 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 15.4 µL ddH2O</LI><LI> 30 min;  RT</LI></UL><P><B>Results:</B></P><UL><LI> F350 + F365 = F367</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P1008">Digestion of P1008</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of P1008</LI></UL><P><B>Procedure:</B></P><UL><LI> 10 µL P1008 (256.5 ng/µL)</LI><LI> 5 µL CutSmart buffer</LI><LI> 0.25 µL XbaI </LI><LI> 0.25 µL PstI</LI><LI> 34.5 µL ddH2O </LI><LI> incubation for 1h at 37 °C</LI></UL><P><B>Results:</B></P><P>1Kb ruler, ... P1008[XbaI&amp;PstI]
</P><UL><LI> P1008 -&gt; F368 ~1 ng/µL</LI></UL><UL><LI> Maybe do it again, fragments do not have the right size, apparently.</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_F366.3B_F367">Transformation of F366; F367</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Amplification of Vector</LI></UL><P><B>Procedure:</B></P><UL><LI> Transformation following SOP(13.02.2013) </LI></UL></DIV><DIV class="kill-switch"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P296_and_F269">Analytical digestion of P296 and F269</SPAN></H3><P><B>Investigator: EU</B></P><P><B>Aim of the experiment:</B> Analyze construct for eventual preparative digestion
</P><P><B>Procedure:</B></P><P>2 µl DNA 
15µl ddH2O
1µl of PstI
2µl cut smart buffer
</P><P>incubation at 37°C over night
Gel electrophoresis and gel extraction
</P><P><B>Results:</B></P></DIV><DIV class="kill-switch"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P313.2C_P296_and_P73">Preparative digestion of P313, P296 and P73</SPAN></H3><P><B>Investigator: EU</B></P><P><B>Aim of the experiment:</B> Prepare construct for ligation later on
</P><P><B>Procedure:</B></P><P>10 µl (P73)/ 15 µl (P296)/ 20 µl (P313) DNA
2µl of each enzyme (P296&amp;P313 --&gt; EcoRI&amp;SpeI, P73 --&gt; EcoRI&amp;XbaI)
5µl cut smart buffer
ad 50 µl ddH2O
</P><P>incubation at 37°C over night
Gel electrophoresis and gel extraction by Gel extraction kit (Promega) according  to manufacturer’s protocol
</P><P><B>Results:</B></P><UL><LI> P296 [EcoRI&amp;SpeI] --&gt; F378</LI><LI> P313 [EcoRI&amp;SpeI] --&gt; F383</LI><LI> P73   [EcoRI&amp;XbaI] --&gt; F379</LI></UL></DIV><DIV class="kill-switch"><H3><SPAN class="mw-headline" id="Ligation:_F269_.2B_F270_.3D_F380.3B_F269_.2B_F92_.3D_F381.3B_F269_.2B_F133_.3D_F382">Ligation: F269 + F270 = F380; F269 + F92 = F381; F269 + F133 = F382</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Ligation: </LI></UL><UL><LI> F269 + F270 = F380 </LI><LI> F269 + F92 = F381</LI><LI> F269 + F133 = F382</LI></UL><P><B>Procedure:</B></P><UL><LI> 3 µL F269 (6,3 ng)</LI><LI> 0,5 µL F270 (5,6ng)/ 0,25 µl F92 (6,75 ng)/ 0,25 µl F133 (13 ng)</LI><LI> 0,5 µL buffer </LI><LI> 0,5 µL T4 ligase</LI><LI> 0,5/0,75 µL ddH2O</LI><LI> ONT; 16 °C</LI></UL><P><B>Results:</B></P><UL><LI> F269 + F270 = F380 </LI><LI> F269 + F92 = F381</LI><LI> F269 + F133 = F382</LI></UL></DIV><DIV class="kill-switch"><H3><SPAN class="mw-headline" id="Sequencing_of_P313">Sequencing of P313</SPAN></H3><P><B>Investigator: EU</B></P><P><B>Aim of the experiment:</B> Sequencing of P313
</P><P><B>Procedure:</B></P><UL><LI>Sequencing batch was prepared after manufacturer's protocol:</LI></UL><UL><LI>6 µl of plasmid DNA (end concentration 58,6 ng) in 9µl H2O</LI><LI>2 µl sequencing primer (VF2)</LI></UL><UL><LI>The different vectors we sequenced received the following barcodes:</LI></UL><UL><LI> P313 with VF2: FR12904387</LI></UL><P><B>Results:</B></P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_Sep_15th"><B>Thursday, Sep 15th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P1008_2">Digestion of P1008</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of P1008</LI></UL><P><B>Procedure:</B></P><UL><LI> 10 µL P1008 (256.5 ng/µL)</LI><LI> 5 µL CutSmart buffer</LI><LI> 0.25 µL XbaI </LI><LI> 0.25 µL PstI</LI><LI> 34.5 µL ddH2O </LI><LI> incubation for over night at 37 °C</LI></UL><P><B>Results:</B></P><P>P1008[XbaI&amp;PstI], P1008[XbaI&amp;PstI], 1Kb ruler
</P><UL><LI> P1008[XbaI&amp;PstI] -&gt; F375  32 ng/µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F63_.2B_F375_.3D_F377">Ligation: F63 + F375 = F377</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F63 + F375</LI></UL><P><B>Procedure:</B></P><UL><LI> 2.2 µL F63 (45 ng\µL)</LI><LI> 6.3 µL F375 (32 ng\µL)</LI><LI> 3 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 17.5 µL ddH2O</LI><LI> 30 min; RT</LI></UL><P><B>Results:</B></P><UL><LI> F63 + F375 = F377</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Fill_in_of_overhangs_of_F386_with_Large_Klenow_fragment">Fill in of overhangs of F386 with Large Klenow fragment</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> 3' overhang removal and fill-in of 3' recessed (5' overhang) ends using DNA Polymerase I, Large (Klenow) Fragment</LI><LI> short: create blunt ends, needed for ligation</LI></UL><P><B>Procedure:</B></P><UL><LI> 17.09 µL F386 (25.9 µL)</LI><LI> 0.66 µL 10 mM dNTP Mix </LI><LI> 2 µL T4 Ligase buffer</LI><LI> 0.25 µL Klenow Fragment</LI><LI> 15 min; RT</LI><LI> heat inactivation: 20 min; 75 °C</LI></UL><P><B>Results:</B></P><UL><LI> F387</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Incubation_of_F386_with_Q5_polymerase">Incubation of F386 with Q5 polymerase</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Create blunt ends, needed for ligation</LI></UL><P><B>Procedure:</B></P><UL><LI> 5 µL 2x Q5 Mastermix</LI><LI> 5 µL F386 (25.9 ng/µL) </LI><LI> 15 min; 72 °C</LI></UL><P><B>Results:</B></P><UL><LI> F388</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F374_.2B_F386_.3D_F390">Ligation: F374 + F386 = F390</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F374 + F386</LI></UL><P><B>Procedure:</B></P><UL><LI> 1 µL F374 (25.9 ng\µL)</LI><LI> 17.5 µL F387 (~ 22 ng\µL)</LI><LI> 2 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 30 min; RT</LI></UL><P><B>Results:</B></P><UL><LI> F374 + F386 = F390</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F374_.2B_F388_.3D_F391">Ligation: F374 + F388 = F391</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F374 + F388 = F391</LI></UL><P><B>Procedure:</B></P><UL><LI> 0.3 µL F374 (25.9 ng\µL)</LI><LI> 10 µL F388 (~ 13 ng\µL)</LI><LI> 2 µL T4 ligase buffer </LI><LI> 1 µL T4 ligase</LI><LI> 6.7 µL ddH2O</LI><LI> 30 min; RT</LI></UL><P><B>Results:</B></P><UL><LI> F374 + F388 = F391</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_F390.2C_F391.2C_F376_and_F377_.28INSULIN.29">Transformation of F390, F391, F376 and F377 (INSULIN)</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Transformation of F390, F391, F376 and F377 for further application</LI></UL><P><B>Procedure:</B></P><UL><LI> 3 µL DNA were added to xl1-cells</LI><LI> incubation on ice for 20 min</LI><LI> heatshock in 42°C for 40s</LI><LI> incubation on ice for 4 min</LI><LI> 700µl LB-medium were added</LI><LI> incubation for 1h in 37°C shaker</LI><LI> centrifugation for 3 min</LI></UL><P><B>Results:</B></P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_F389_.28zk-vector.29">Transformation of F389 (zk-vector)</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Transformation of F389 for further application</LI></UL><P><B>Procedure:</B></P><UL><LI> 3 µL DNA were added to xl1-cells</LI><LI> incubation on ice for 20 min</LI><LI> heatshock in 42°C for 40s</LI><LI> incubation on ice for 4 min</LI><LI> 700µl LB-medium were added</LI><LI> incubation for 1h in 37°C shaker</LI><LI> centrifugation for 3 min</LI></UL><P><B>Results:</B></P></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Transformation_of_F366">Transformation of F366</SPAN></H3><P><B>Investigator: EU </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Transformation of F366 for further application</LI></UL><P><B>Procedure:</B></P><UL><LI> 3 µL DNA were added to xl1-cells</LI><LI> incubation on ice for 20 min</LI><LI> heatshock in 42°C for 40s</LI><LI> incubation on ice for 4 min</LI><LI> 700µl LB-medium were added</LI><LI> incubation for 1h in 37°C shaker</LI><LI> centrifugation for 3 min</LI></UL><P><B>Results:</B></P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_Sep_16th"><B>Friday, Sep 16th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P430">Digestion of P430</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of P430</LI></UL><P><B>Procedure:</B></P><UL><LI> 10 µL 430 (~150 ng/µL)</LI><LI> 2 µL CutSmart buffer</LI><LI> 0.25 µL PacI</LI><LI> 0.25 µL AgeI</LI><LI> 7.5 µL ddH2O </LI><LI> 1h; 37 °C</LI></UL><P><B>Results:</B></P><P>1kb ruler; P430[PacI&amp;AgeI], P424[SpeI&amp;PstI], P425[SpeI&amp;PstI]
</P><UL><LI> P430[PacI&amp;AgeI] -&gt; F400 2.1 ng/µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P424">Digestion of P424</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of P424</LI></UL><P><B>Procedure:</B></P><UL><LI> 17 µL 424 (60.9 ng/µL)</LI><LI> 2 µL CutSmart buffer</LI><LI> 0.25 µL SpeI</LI><LI> 0.25 µL PstI </LI><LI> 1h; 37 °C</LI></UL><P><B>Results:</B></P><P>1kb ruler; P430[PacI&amp;AgeI], P424[SpeI&amp;PstI], P425[SpeI&amp;PstI]
</P><UL><LI> P424[SpeI&amp;PstI] -&gt; F401 15.7ng/µL</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P425">Digestion of P425</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of P425</LI></UL><P><B>Procedure:</B></P><UL><LI> 10 µL 425 (~222 ng/µL)</LI><LI> 2 µL CutSmart buffer</LI><LI> 0.25 µL SpeI</LI><LI> 0.25 µL PstI </LI><LI> 7.5 µL ddH2O</LI><LI> 1h; 37 °C</LI></UL><P><B>Results:</B></P><P>1kb ruler; P430[PacI&amp;AgeI], P424[SpeI&amp;PstI], P425[SpeI&amp;PstI]
</P><UL><LI> P425[SpeI&amp;PstI] -&gt; F402 53 ng/µL</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_19.2C_Sep_19th_-_Sep_25th">Week 19, Sep 19th - Sep 25th</SPAN></H1><DIV class="week" id="Week_19"><H2><SPAN class="mw-headline" id="Tuesday.2C_Sep_20th"><B>Tuesday, Sep 20th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Digestion_of_P444.3B_P445.3B_P446">Digestion of P444; P445; P446</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI> Digestion of  P444; P445; P446</LI></UL><P><B>Procedure:</B></P><P>
P444:
</P><UL><LI> 8 µL P444 (377.7 ng/µL)</LI><LI> 2 µL CutSmart buffer</LI><LI> 0.25 µL SpeI</LI><LI> 0.25 µL PstI</LI><LI> 9.5 µL ddH2O </LI><LI> 1h; 37 °C</LI></UL><P>P445:
</P><UL><LI> 14 µL P445 (223.7 ng/µL)</LI><LI> 2 µL CutSmart buffer</LI><LI> 0.25 µL SpeI</LI><LI> 0.25 µL PstI</LI><LI> 3.5 µL ddH2O </LI><LI> 1h; 37 °C</LI></UL><P>P446:
</P><UL><LI> 17.5 µL P446 (144 ng/µL)</LI><LI> 2 µL CutSmart buffer</LI><LI> 0.25 µL SpeI</LI><LI> 0.25 µL PstI </LI><LI> 1h; 37 °C</LI></UL><P><B>Results:</B></P><P>1kb ruler P451[PacI&amp;MluI], P451[PacI&amp;MluI], P444[SpeI&amp;PstI], P446[SpeI&amp;PstI]
</P><P>-&gt; I did P444 again, because there was marker in the line.
</P><P>1kb ruler, P444[SpeI&amp;PstI], P445[SpeI&amp;PstI]
</P><UL><LI> P444[SpeI&amp;PstI]  -&gt; F420  18 ng/µL</LI><LI> P445[SpeI&amp;PstI]  -&gt; F418  24.2 ng/µL</LI><LI> P446[SpeI&amp;PstI]  -&gt; F421  7.4 ng/µL</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_Sep_21th"><B>Wednesday, Sep 21th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation_.7BF438_.2B_F435.7D_.2B_F433_or_F334">Ligation {F438 + F435} + F433 or F334</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation {F438 + F435} + F433 or F334</LI></UL><P><B>Procedure:</B></P><P>F438 + F435:
</P><UL><LI> 10 µL F438 (40.2 ng/µL)</LI><LI> 11 µL F435 (30.5)  </LI><LI> 2 µL T4 Ligase buffer</LI><LI> 1 µL T4 Ligase </LI><LI> 6 µL ddH2O</LI></UL><P>{F438 + F435} + F433:
</P><UL><LI> 0.5 µL F433 (48.2 ng/µL)</LI><LI> 4.5 µL {F438 + F435} (~24.4 ng/µL) </LI><LI> 2 µL T4 Ligase buffer</LI><LI> 1 µL T4 Ligase </LI><LI> 12 µL ddH2O</LI></UL><P>{F438 + F435} + F434:
</P><UL><LI> 0.5 µL F434 (109 ng/µL)</LI><LI> 4.5 µL {F438 + F435} (~24.4 ng/µL) </LI><LI> 2 µL T4 Ligase buffer</LI><LI> 1 µL T4 Ligase </LI><LI> 11.5 µL ddH2O</LI></UL><P><B>Results:</B></P><UL><LI> F438 + F435 = F439</LI><LI> {F438 + F435} + F434 =  F440</LI><LI> {F438 + F435} + F433 = F441</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_Sep_25th"><B>Sunday, Sep 25th</B></SPAN></H2><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F457_.2B_F463_.3D_F464">Ligation: F457 + F463 = F464</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F457 + F463</LI></UL><P><B>Procedure:</B></P><UL><LI> 2.7 µL F457 (29.3 ng\µL)</LI><LI> 4.3 µL F463 (22.1 ng\µL)</LI><LI> 2 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 30 min;  RT</LI></UL><P><B>Results:</B></P><UL><LI> F457 + F463 = F464</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F458_.2B_F463_.3D_F465">Ligation: F458 + F463 = F465</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F458 + F463</LI></UL><P><B>Procedure:</B></P><UL><LI> 3.8 µL F458 (15.5 ng\µL)</LI><LI> 3.2 µL F463 (22.1 ng\µL)</LI><LI> 2 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> 30 min;  RT</LI></UL><P><B>Results:</B></P><UL><LI> F458 + F463 = F465</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F434_.2B_F464_.3D_F466">Ligation: F434 + F464 = F466</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F434 + F464</LI></UL><P><B>Procedure:</B></P><UL><LI> 0.5 µL F434 (109.1 ng\µL)</LI><LI> 6.5 µL F464 (17.4 ng\µL)</LI><LI> 2 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> over night; 16 °C</LI></UL><P><B>Results:</B></P><UL><LI> F434 + F464 = F466</LI></UL></DIV><DIV class="hypoxia"><H3><SPAN class="mw-headline" id="Ligation:_F434_.2B_F465_.3D_F467">Ligation: F434 + F465 = F467</SPAN></H3><P><B>Investigator: MT </B></P><P><B>Aim of the experiment:</B></P><UL><LI>Ligation: F434 + F465</LI></UL><P><B>Procedure:</B></P><UL><LI> 0.4 µL F434 (109.1 ng\µL)</LI><LI> 6.6 µL F465 (12.96 ng\µL)</LI><LI> 2 µL buffer </LI><LI> 1 µL T4 ligase</LI><LI> over night; 16 °C</LI></UL><P><B>Results:</B></P><UL><LI> F434 + F465 = F467</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_20.2C_Sep_26th_-_Aug_2th">Week 20, Sep 26th - Aug 2th</SPAN></H1></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>