Team:LMU-TUM Munich/Design
Toggle menu
Menu
Home
Team
Members
Collaborations
Sponsors
Press
Gallery
Project
Description
Design
Proteins
Receptors
Linker Chemistry
Polymerization
Bioprinting
Hypoxia
Medical Application
InterLab
Software
Notebook
Labjournal
Materials
Methods
How Tos
Parts
Parts
Basic Parts
Composite Parts
Part Collection
Hardware
Modeling
Safety
Human Practices
Human Practices
Silver
Gold
Integrated Practices
Engagement
Entrepreneurship
Entrepreneurship
Supporting Entrepreneurship
Attributions
Media:Muc16_Sticker_Design_001.png
Contents
1 Receptor design highlights
2 Human cell surface receptors
2.1 The incredible journey: signal peptides and protein targeting
2.2 Elements for stability and detection
2.2.1 EGFR transmembrane domain
2.3 Elements for detection
3 Design of an autotransporter construct for bacterial surface display
4 References
Receptor design highlights
Think before you ink may be a popular slogan wielded by tattoo adversaries, but it has also been perfectly applicable to creating our synthetic biology approach of bioprinting. In the following, we explore the contemplations that contributed to the design of the membrane proteins (also referred to as 'receptors') that allow us to embed eukaryotic and bacterial cells into a stable, three-dimensional and user-definable matrix immediately upon printing.
Human cell surface receptors
Media:Muc16 receptor schematic_002.png
Figure 1: A) Receptor constructs in a schematic representation of functional genetic elements. B) Schematic representation of a receptor presenting an extracellular single-chain avidin that is endogenously biotinylated by a coexpressed biotin ligase (BirA) and thus presents biotin groups at the cell surface.
For this purpose, we built two kinds of receptor constructs, both on their own able – when expressed in mammalian cells – to cross-link cells for our approach of bioprinting:
1) A receptor presenting an extracellular biotin acceptor peptide that is endogenously biotinylated by a coexpressed biotin ligase (BirA), thus presenting biotin molecules on the cellular surface and allowing it to interact with streptavidin in the printing solution.
2) Receptors presenting an extracellular monomeric or single-chain avidin variant.
In presenting avidin on the cell surface, cells are able to interact with a co-injected biotinylated linker peptide. With this linker connected to several cell surface avidins as well as streptavidin in the printing solution, a protein matrix is created that embeds cells and allows precise positioning of cells in a three-dimensional manner.
Our chosen avidin derivates by design allow functional fusion of the otherwise tetrameric avidin molecule with our receptor. Two different variants were used: enhanced monomeric avidin, a single-subunit avidin that is nevertheless able to bind biotin, and single-chain avidin, which resembles the naturally occurring avidin tetramer but consisting of a single polypeptide chain.
All receptors mediate the same function: By interacting with streptavidin in the printing reservoir – either directly by presenting biotin (biotin acceptor peptide) or indirectly via binding to a biotinylated linker (single-chain avidin variants), cells are cross-linked due to polyvalent binding of streptavidin molecules in the printing solution to several cellular biotin groups at once.
The incredible journey: signal peptides and protein targeting
Essential to a protein's function are not only its activity or binding properties, but also its presence at the right time in the right place – the latter in our case being the cell surface. One ubiquitous element for protein targeting is the signal peptide, which enables translocation of proteins to the ER, from where their journey continues to their place of function. Proteins containing a signal peptide include transmembrane proteins, secreted proteins, proteins of the ER itself, proteins of the Golgi apparatus and several more. [1] For our project, signal peptides constituted a crucial part of many constructs, used to target transmembrane proteins to the cell surface (receptors) and proteins to the ER (BirA), and to secrete proteins into the surrounding medium (luciferase assays to quantify expression levels).
Media:Muc16 SP schematic.jpeg
Figure 2: Cotranslational translocation of a membrane protein into the ER in a schematic depiction showing the nascent, unfolded peptide chain containing signal peptide (start-transfer sequence) and stop-transfer sequence as well as the ER membrane. [2]
The hydrophobic signal peptide (also called start-transfer sequence) usually consists of less than 30 amino acids [3] and is recognized by the signal recognition particle upon translation at the rough ER, mediating cotranslational translocation of the nascent peptide chain into the ER. [4]
During this project, three different signal peptide constructs were tested in order to determine that resulting in the highest expression levels:
The EGFR signal peptide was taken from the iGEM parts registry ( BBa_K157001 ) and combined with a BioBrick containing the CMV promoter sequence ( BBa_K747096 ) via the RFC10 cloning standard.
The BM40 and Igκ signal peptides were designed by us and synthesized with regard to adding additional spacing and functional elements to the 5'-UTR of the constructs, and combined with a BioBrick containing the CMV promoter sequence ( BBa_K747096 ) via the RFC10 cloning standard.
The EGFR signal peptide taken from the parts registry is constructed such that the signal peptide ORF immediately follows the RFC10 cloning scar after the CMV promoter, thus resulting in a very short 5’ untranslated region (UTR) of transcribed mRNA. The combination of the CMV promoter with the synthesized BM40 and Igκ signal peptides allows for a considerably longer 5’-UTR – and can thus be made to contain a full Kozak consensus sequence by design. The Kozak sequence is recognized by the ribosome as a translational start site; if this element is missing or deviates from the consensus sequence, this may considerably decrease translation efficiency. [5] For the EGFR signal peptide construct, a Kozak sequence is not present, as it would have to precede the start codon ATG, i.e. the position occupied by the RFC10 cloning scar. Since both the distance from the promoter to the open reading frame and the Kozak consensus sequence are considered crucial parameters for expression levels [6] , BM40 and Igκ signal peptide constructs are likely to result in increased expression levels compared to proteins containing the EGFR signal peptide from the parts registry.
In order to predict whether our designed signal peptides would be functional, we used the SignalP server, which allows to make theoretical predictions about signal peptide functionality and cleavage. [7]
Media:Muc16 SignalPeptideprediction_002.png
Figure 3: A) Schematic depiction of a protein-coding mRNA containing a 5'- and 3'-UTR, the first of which is an important factor in translation efficiency. B) Theoretical prediction of signal peptides within the receptor construct containing the EGFR, Igκ and BM40 signal peptide via the SignalP 4.1 server. Plotted probabilities represent the raw cleavage site “C-score”, the signal peptide “S-score” and the combined cleavage site “Y-score”. C) ClustalW multiple sequence alignment of the BM40, EGFR and Igκ signal peptide construct, depicting a gap for the EGFR signal peptide construct compared to the newly designed ones and thus indicating a shorter length of the 5'-UTR . For Igκ and BM40 signal peptide constructs, a T7 promoter spacer as well as a Kozak sequence increase the distance between TATA box and protein-coding region.
Elements for stability and detection
Apart from the functional parts that mediate the (strept)avidin-biotin interaction, other elements in the receptor were designed to make sure it is optimally translocated to the membrane, as stable as possible, and easily detectable.
EGFR transmembrane domain
For anchoring in the membrane, a type I membrane protein with defined localization of N- and C-terminus out- and inside of the cell was used. The N-terminal transmembrane α-helix of human EGFR (UniProt P00533, amino acids 622-653) was herefore chosen (see below the topology prediction of EGFR via the TMHMM 2.0 server for the prediction of transmembrane helices [8] ). A stop-transfer sequence consisting of charged amino acids - being characteristic for type I membrane proteins - was added at the C-terminus, and the sequence was furthermore flanked by a (GGGGC)2-linker at the N- and C-terminus, respectively. The addition of a stop-transfer sequence as in the naturally occurring EGFR sequence, as well as the addition of flexible linkers, makes our EGFR-TMD an improvement over the BioBrick BBa_K157002 , which does not possess either of these elements.
Media:Receptor TMD 002.png
Figure 4: A) Multiple sequence alignment of human EGFR (see UniProt P00533 ), the EGFR transmembrane domain BioBrick from the registry ( BBa_K157002 and the extended EGFR transmembrane designed by us, containing a charged stop-transfer domain as well as linker elements at the C- and N-termini. B) Transmembrane domain prediction by the TMHMM 2.0 server , indicating positioning of N- and C-termini out- and inside of the cell, respectively, as well as predicting functionality of transmembrane regions. C) Schematic depiction of the C-terminal stop-transfer sequence of the receptor.
Elements for detection
In order to confirm the correct expression and localization of the receptor, the receptor contains three functional elements for its detection for both the N- and the C-terminal receptor domains:
The intracellularly located C-terminal red fluorescent protein mRuby 3 for detection of the receptor via fluorescence microscopy while also providing a stable, folded domain at the C-terminus,
an extracellular epitope domain near the N-terminus for immunochemical detection via A3C5-antibody fragments as well as FACS, and
an intracellular Strep-tag II at the N-terminus for the detection and purification via immunochemical methods.
Further elements that were included into the expression construct are the CMV promoter for overall high expression levels of the receptor, resulting in a high-avidity-interaction. At the 3'-end, the construct contains the polyadenylation signal of human growth hormone (hGH) for functional polyadenylation of the transcribed mRNA.
Design of an autotransporter construct for bacterial surface display
Besides the design of biotinylated and biotin-binding receptors for eukaryotic cells, we also wanted to apply our biotINK approach for prokaryotic cells. For this purpose, we designed an autotransporter device that is able to present biotinylated or biotin-binding protein domains on the surface of E. coli. Constructs for bacterial surface display are hereby already well known in the field of protein engineering of therapeutic proteins, such as antibodies. There, they are used to screen libraries of different antibodies concerning their affinity towards a given therapeutic target.
We based our design on the autotransporter EspP [9] .
Media:Muc16 Autotransporterdesign_001.png
Figure 5
The autotransporter system itself is established and functional (see Fig. A [10] ), and here we "bricked" it down into RFC[10] and RFC[25] BioBricks. At first, we tested different inducible bacterial promoter systems (e.g. "TetR repressed GFP"; BBa_K577893 ) that are available in the parts registry, but when we saw that two of the available systems did not work in our own hands, we decided to built the system on our own. We used the Tet-Repressor generator that was available on the distribution plates ( BBa_I739001 ) as we believe that it is very important for the standardization in SynBio to use available parts. Downstream of this repressor generator we designed and fused a BioBrick ( BBa_K2170141 ) that encodes a double Tet-Operator (Tet-Repressor binding site together with a bacterial promoter and an signal peptide of the outer membrane protein A (OmpA [11] ) from E. coli that has a 3' RFC[25] restriction site, allowing the protein fusion of proteins that are secreted into the bacterial periplasm when expressed with this BioBrick. As a protein fusion, we then assembled a protein cargo domain that is generally exchangeable for any protein that is so small in size that it can be transported through the outer membrane by the EspP autotransporter. In our project we used the same extracellular domains as for the eukaryotic receptors: enhanced monomeric Avidin (eMA; BBa_K2170204 ), single-chain Avidin (scAvidin; BBa_K2170205 ) and a composite part composed of a N-terminal Biotinylation acceptor peptide (BAP) and a C-terminal NanoLuciferase (together shown as BAP; ; BBa_K2170118 , see Fig. B). Our design features an A3C5-epitope tag further downstream that can be recognized by a high-affinity Fab-fragment and is generally used to detect and quantify the surface display of the cargo domain [12] . Further downstream of this affinity tag we fused the BioBrick for the EspP autotransporter from E. coli [13] [14] which is then followed by a bacterial terminator. For the termination we chose again  a well-working terminator from the distribution plate ("double terminator"; BBa_B0010 - BBa_B0012 ).
But this was just the DNA-part. What is expected to happen on a protein level? The construct constantly produces Tet-repressor that binds to the Tet-Operator and repressed the promoter activity of the autotransporter gene. As soon as the inducer anhydrotetracycline (aTc) is added to the culture, it binds to the Tet-repressor that can't bind anymore to the Tet-operator and thus the expression of the autotransporter can be regulated. Although the TetR-system is known to be a tight promoter system there is always a certain background expression with most promoters. If the protein expression of the autotransporter is induced using aTc the autotransporter is transcribed and translated. Due to the bacterial signal peptide, the protein is secreted into the bacterial periplasm (the space between the two bacterial membranes of gram-negative bacteria such as E. coli). When present in the bacteria periplasm the autotransporter diffused to the outer membrane and inserts into the membrane. After the integration into the outer membrane, the protein cargo is transported through the beta-barrel of the autotransporter and is presented on the bacterial surface. We are sure that this BioBrick will be a valuable contribution to the Parts Registry as it allows future teams to display small protein domains on the surface of gram-negative bacteria, such as E. coli.
References
↑ Rapoport, T. A. (2007). Protein translocation across the eukaryotic endoplasmic reticulum and bacterial plasma membranes. Nature, 450(7170), 663-669.
↑ Taken from Alberts, B., Bray, D., Hopkin, K., Johnson, A., Lewis, J., Raff, M., ... & Walter, P. (2010). Essential cell biology. Garland Science.
↑ Walter, P., Ibrahimi, I., & Blobel, G. U. N. T. E. R. (1981). Translocation of proteins across the endoplasmic reticulum. I. Signal recognition protein (SRP) binds to in-vitro-assembled polysomes synthesizing secretory protein. The Journal of Cell Biology, 91(2), 545-550.
↑ Keenan, Robert J., et al. "The signal recognition particle." Annual review of biochemistry 70.1 (2001): 755-775.
↑ Kozak, M. (1989). The scanning model for translation: an update. The Journal of cell biology, 108(2), 229-241.
↑ Kozak, M. (1987). An analysis of 5'-noncoding sequences from 699 vertebrate messenger RNAs. Nucleic acids research, 15(20), 8125-8148.
↑ Petersen, T. N., Brunak, S., von Heijne, G., & Nielsen, H. (2011). SignalP 4.0: discriminating signal peptides from transmembrane regions. Nature methods, 8(10), 785-786.
↑ Sonnhammer, E. L., Von Heijne, G., & Krogh, A. (1998, July). A hidden Markov model for predicting transmembrane helices in protein sequences. In Ismb (Vol. 6, pp. 175-182).
↑ Binder, U., Matschiner, G., Theobald, I., & Skerra, A. (2010). High-throughput sorting of an Anticalin library via EspP-mediated functional display on the Escherichia coli cell surface. Journal of molecular biology, 400(4), 783-802.
↑ Binder, U., Matschiner, G., Theobald, I., & Skerra, A. (2010). High-throughput sorting of an Anticalin library via EspP-mediated functional display on the Escherichia coli cell surface. Journal of molecular biology, 400(4), 783-802.
↑ Ghrayeb, J., Kimura, H., Takahara, M., Hsiung, H., Masui, Y., & Inouye, M. (1984). Secretion cloning vectors in Escherichia coli. The EMBO journal, 3(10), 2437.
↑ Costa, J., Grabenhorst, E., Nimtz, M., & Conradt, H. S. (1997). Stable expression of the Golgi form and secretory variants of human fucosyltransferase III from BHK-21 cells Purification and characterization of an engineered truncated form from the culture medium. Journal of Biological Chemistry, 272(17), 11613-11621.
↑ Barnard, T. J., Dautin, N., Lukacik, P., Bernstein, H. D., & Buchanan, S. K. (2007). Autotransporter structure reveals intra-barrel cleavage followed by conformational changes. Nature structural & molecular biology, 14(12), 1214-1220.
↑ Skillman, K. M., Barnard, T. J., Peterson, J. H., Ghirlando, R., & Bernstein, H. D. (2005). Efficient secretion of a folded protein domain by a monomeric bacterial autotransporter. Molecular microbiology, 58(4), 945-958.
Back to top
Follow us on:
LMU & TUM Munich
United team from Munich's universities
Contact us:
Address
iGEM Team TU-Munich
Emil-Erlenmeyer-Forum 5
85354 Freising, Germany
