<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_LMU-TUM_Munich_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:LMU-TUM Munich/Measurement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="row"><DIV class="col-sm-3 col-xs-8 fixed-sm z-1"><DIV class="sidebar-nav"><DIV class="navbar" role="navigation"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".sidebar-navbar-collapse"><SPAN class="sr-only">Toggle menu</SPAN></BUTTON><SPAN class="visible-xs navbar-brand">Menu</SPAN></DIV><DIV class="navbar-collapse collapse sidebar-navbar-collapse"><UL id="main-menu" class="nav navbar-nav measurement"><LI class="menu_item home"><A href="https://2016.igem.org/Team:LMU-TUM_Munich"><SPAN class="visible-xs-block">Home</SPAN></A></LI><LI class="dropdown menu_item team"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI class="members"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Team">Members</A></LI><LI class="collaborations"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Collaborations">Collaborations</A></LI><LI class="sponsors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Sponsors">Sponsors</A></LI><LI class="press"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Press">Press</A></LI><LI class="gllry"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Gallery">Gallery</A></LI></UL><LI class="dropdown menu_item project"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Project </A></LI><LI class="description"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Description">Description</A></LI><LI class="design"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Design">Design</A></LI><LI class="proteins"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proteins">Proteins</A></LI><LI class="receptors"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Receptors">Receptors</A></LI><LI class="linker"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Linkerchemistry">Linker Chemistry</A></LI><LI class="poly"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Proof">Polymerization</A></LI><LI class="bioprinting"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Demonstrate">Bioprinting</A></LI><LI class="hypoxia"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hypoxia">Hypoxia</A></LI><LI class="medical"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Medical">Medical Application</A></LI><LI class="interlab"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/InterLab">InterLab</A></LI><LI class="software"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Software">Software</A></LI><LI class="dropdown menu_item notebook"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Notebook </A></LI><LI class="labjournal"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Labjournal">Labjournal</A></LI><LI class="materials"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Materials">Materials</A></LI><LI class="methods"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Methods">Methods</A></LI><LI class="howtos"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Howtos">How Tos</A></LI><LI class="dropdown menu_item parts"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Parts </A></LI><LI class="parts-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Parts">Parts</A></LI><LI class="basic-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Basic_Part">Basic Parts</A></LI><LI class="composite-parts"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Composite_Part">Composite Parts</A></LI><LI class="part-collection"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Part_Collection">Part Collection</A></LI><LI class="menu_item hardware"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Hardware">Hardware</A></LI><LI class="menu_item model"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Model">Modeling</A></LI><LI class="menu_item safety"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Safety">Safety</A></LI><LI class="dropdown menu_item human_practices dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Human Practices </A></LI><LI class="human_practices-sub"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Human_Practices">Human Practices</A></LI><LI class="silver"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Silver">Silver</A></LI><LI class="gold"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/HP/Gold">Gold</A></LI><LI class="integrated-practices"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Integrated_Practices">Integrated Practices</A></LI><LI class="engagement"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Engagement">Engagement</A></LI><LI class="dropdown menu_item entrepreneur dropup"><A href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Entrepreneurship</A></LI><LI class="entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Entrepreneurship">Entrepreneurship</A></LI><LI class="supporting-entrepreneurship"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Supporting_Entrepreneurship">Supporting Entrepreneurship</A></LI><LI class="menu_item attributions"><A href="https://2016.igem.org/Team:LMU-TUM_Munich/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV class="col-sm-9 col-sm-offset-3"><H1><SPAN class="mw-headline" id="Eucaryotic_receptor_constructs">Eucaryotic receptor constructs</SPAN></H1><P>This is a page that is required for the evaluation of the best measurement category. Here we present the measurements of our membrane protein toolbox that allows surface-display of functional protein domains on prokaryotic as well as eukaryotic cells. If you want to read the whole story including the design of our modular receptors, please go to our <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Localizatio">Receptor page</A>. 
</P><H1><SPAN class="mw-headline" id="Subcellular_protein_localization_using_fluorescence_microscopy">Subcellular protein localization using fluorescence microscopy</SPAN></H1><DIV class="white-box"><P>For better traceabilty we supplied all of our receptor constructs with reporter genes like mRuby and/or Nanoluciferase.
mRuby is located on the C-terminus respectively the cytosolic end of our transmembrane receptors right before the <I>Strep</I>-tag. Cells that express and fold the protein properly should give a signal of 605 nm (red) when exited at around 558 nm (rfp channel).<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP> We tested our contructs three diffrent cell types: HEK293T adherent cells that were stably transfected, HEK293E suspension cells that were semi-stably transfected and N2a cells, a mouse neuroblastoma cell line, that we transiently transfected. 
</P><P>As you can see on the images the signal developed over time. In the beginning, before selection, the protein was visible in little dots and therefor seemed to be localized more in subcellular oragnelles than in the membrane. 
During selection on hygromycin B resistency most of the cells died, leaving some resistant cells behind, which over time grew to bigger foci consisting of stably transfected cells only (See Fig. B). For the BAP-receptor there was hardly to no fluorescence detectable neither in semi-stable nor in the stable transfection approach. But in latter case the cells still grew on hygromycin B and formed foci, which leads us to the conclusion that chromosomal intergration was susccesful (as already shown via PCR) hence there must be some problem in later steps of transcription, translation or protein maturation. 
After 2 weeks of selection on hygromycin B the foci were harvested and expanded. The cells seemed to recover fast and moreover the red dots nearly disappeared leaving only the membrane of the scA-receptor cells illuminated by mRuby. 
</P></DIV><H1><SPAN class="mw-headline" id="High_resolution_confocal_fluorescence_microscopy">High resolution confocal fluorescence microscopy</SPAN></H1><DIV class="white-box"><P>In order to obtain high resolution confocal fluorescence images from our receptor constructs we transfected the Neuro-2a (N2a) mouse neuroblastoma cells line with the respective plasmids. Shortly before the end of the project Christoph came up with the idea to include a trafficking signal to the C-terminus of the receptor proteins that is described to increase the surface presentation of receptors by sorting them intracellularly towards the cell membrane. The sequence of this trafficking signal is DFFdDFFJDFJDKJFKFDJFD (aagagcaggatcaccagcgagggcgagtacatccccctgg accagatcgacatcaacgtgGGCGGCttctgctacgagaacgaggtgtaa)<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP> and we have inserted it into the assembled BioBrick device using PCR cloning. After the transfection of the receptor plasmids cells were stained with the following reagents: Hoechst dye for the nuclear staining at 405 nm; Alexa488-bioting to stain biotin binding proteins; mRuby3 which is a part of the receptor and is expected to show a membrane localization and Dy634-antiA3C5-F<SUB>ab</SUB>. The staining was followed by three washing steps with PBS. Following this staining procedure the cells were mounted on a confocal fluorescence microscope (Leica). The results for each fluorescence channel are given in the following figure (see Fig.Â ???).
</P></DIV><H1><SPAN class="mw-headline" id="Quantification_of_functionalized_membrane_proteins_using_flow_cytometry">Quantification of functionalized membrane proteins using flow cytometry</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Fig.11</B> Flow cytometry data of stably integrated cells. Intrinsic mRuby fluorescence was measured as well as binding of a A3C5-tag specific Fab and of Biotin to single chain Avidin and enhanced monomeric Avidin.</DIV></DIV></DIV><DIV class="white-box"><P>To make sure the receptor is properly presented on the surface of the cells and to quantify the yield of stably transfected cells we used flow cytometry analysis. 
The results presented in Fig. 11 show a positive signal in the rfp channel for the scAvidin and the eMA receptor bearing cells. As already observed in microscopy mRuby-fluorescence could not be detected for the BAP-receptor cells. But surprisingly about 40% where positive for Dy634, meaning that the A3C5 tag, against which the fluorophor labeled Fab is dierectet, must be present on the cell surface. This results suggest that there might be a problem about the folding of the mRuby beta-barrel leading to a loss of fluorescent activity. 
For scA and eMA cells Dy634 signal has been positive albeit not as high as expected. 
Another stain was the eGFP coupled with biotin giving a positive signal for scA and eMA to which the biotin binds. For BAP cells there was no signal since this receptor, is biotinylated by the BirA and doesn't cooperatively bind the Biotin. Moreover the biotin snd the dye are coupled via the C-terminal tail of the biotin leaving no contact point for the BirA to ligate the eGFP-biotin to the receptor's acceptor peptide.
</P></DIV><H1><SPAN class="mw-headline" id="Immunochemical_detecton_of_functionalized_membrane_protein_using_Western_Blot">Immunochemical detecton of functionalized membrane protein using Western Blot</SPAN></H1><DIV class="white-box"><PRE>Another way to verify the succesful transfection and protein expression is given by the two tags (A3C5 and <I>Strep</I>-tag II) that are included in the receptor design, which allow detection via Western Blot analysis.
</PRE><P>
After transfecting mammalian cells with the receptor constructs and selecting them for 2 weeks on hygromycin, we lysed the cells and checked for the presence of <I>Strep</I>- and A3C5-tag via western blot using primary antibodies (or a F<SUB>ab</SUB> for A3C5) against the respective tags und secondary alkaline phosphatase coupled antibodies (see methods). As the figure shows, the <I>Strep</I>-tag wasn't detectable via WB - however, the A3C5-tag-WB shows significant singular bands at the size of our proteins in the respective samples.
This data corresponds with the flow cytometry results again showing that the BAP receptor is also present in the non-fluorescent cells. 
</P><TABLE><CAPTION> Properties of membrane proteins
</CAPTION><TBODY><TR><TD>Receptor
</TD><TD>BioBrick
</TD><TD>Amino acids
</TD><TD>Moelcular mass [Da]
</TD></TR><TR><TD>BAP-Receptor
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170000">BBa_K2170000</A></TD><TD>507
</TD><TD>55 440
</TD></TR><TR><TD>eMA-Receptor
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170001">BBa_K2170001</A></TD><TD>448
</TD><TD>48 618
</TD></TR><TR><TD>scAvidin-Receptor
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170002">BBa_K2170002</A></TD><TD>877
</TD><TD>95 028
</TD></TR></TBODY></TABLE></DIV><H1><SPAN class="mw-headline" id="Quantification_of_mRNA_expression_levels_by_RT-q-PCR">Quantification of mRNA expression levels by RT-q-PCR</SPAN></H1><DIV class="white-box"><P>The characterization of our transfected mammalian cells showed that the two receptors containing an avidin variant were expressed and translocated to the outer cell membrane. The biotin-presenting receptor containing the biotin-acceptor peptide, though, showed a very low fluorescence signal. Therefore, the success of the stable integration was tested by PCR of the genomic integration, which lead to the conclusion that the construct was transfected and integrated sucessfully, hinting at a problem downstream of the integration. In order to understand where the source of this problem was, we took a closer look at the expression levels of the different receptors by analyzing mRNA levels in comparison to each other and to the housekeeping gene (HKG) GAPDH. 
</P><P>For a better comparibility, we used the same amount of cells for each receptor and not only reviewed cells with stably integrated but also cells with the episomal stable receptor constucts. After extraction of the total mRNA from the cells the quality and integrity was measured by Bioanalyzer. All samples showed a <B>RNA integrity number (RIN) above 9</B> which means that the total mRNA is pure and does not have any contamination.
During the first step of RT-qPCR the total mRNA in converted to cDNA by a reverse transcriptase, which is then used in the actual quantitative PCR to quantify the amount of a specific mRNA was produced by the cells. For this reason two primer pairs were designed. One of them was designed to bind the exact same region in all of the different receptor mRNAs and amplify the same part of the receptor. Contrary to that the second pair was supposed to bind only in the biotin presenting receptor and amplify a part that is unique to this receptor. By this method we wanted to observe if only one part of the mRNA is unstable or if the general mRNA level of this receptor is decreased.
</P><P>For the evaluation of the raw data we used the <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:LMU-TUM_Munich/Methods#.CE.94.CE.94CP_method">ÎÎC<SUB>P</SUB> method</A><SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP> in which the crossing points (C<SUB>P</SUB> values) of a gene of interest and of a reference gene are compared and normalized with a constantly expressed HKG, to get an idea of the relative amounts of the mRNA of interest.
</P><P>In the following you can see the results of our experiments. We compared the expression levels of T-REx cells which were transfected with our different receptor constructs and with non-transfected (NT) T-REx cells.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Fig.x</B></DIV></DIV></DIV></DIV><P>As can be seen the ÎÎC<SUB>P</SUB> values of the mRNA of the eMA and scAvidin constructs are with a <B>1,2Â xÂ Â 10<SUP>6</SUP>Â foldÂ increase</B> for scAvidin and <B>1.5 x 10<SUP>3</SUP> fold increase</B> for eMA, significantly higher than those of the BAP construct. The analysis of the different primer pairs in the two regions of the BAP construct also showed similar values and thus confirms the reliability of the measurement. A difference in C<SUB>P</SUB> value is tantamount to relative mRNA differences between cells.
</P><P>Together with the result that the integration of the constuct into the genome was successful we can conclude that our first assumption, that the missing RFP signal in cells, which were transfected with the BAP construct, is caused by a <B>reduced mRNA stability</B>, is correct.
</P></DIV><H1><SPAN class="mw-headline" id="High_resolution_imaging_using_scanning_electron_microscopy_.28SEM.29">High resolution imaging using scanning electron microscopy (SEM)</SPAN></H1><DIV class="white-box"><P>As we wanted to obtain high resolution images from the transgenic cells transfected with our modular membrane proteins we contacted one of the experts on electron microscopy in Munich and luckily Prof. em. Dr. Wanner (LMU Munich) agreed to support our project by contributing scanning electron microscopic (SEM) imaging. Even better suitable for our purpose Prof. Wanner has developed a method in which he can correlate the fluorescence signal of a sample with the high resolution images obtained in SEM. As this kind of sample preparation, sample processing and imaging takes a while, we had to start 2 weeks before wiki freeze and at that point we did not have enough stably integrated T-REx 2cells, yet. Thus we used transient transfected, semi-stable MEXi HEK 293T cells that were pulled down with magnetic beads in order to increase the number of transgenic cells in the preparation and to show the specific binding of the beads to the cells. For this immunoprecipitation we incubated scAvidin-receptor and eMA-receptor cells with biotinylated magnetic beads and on the other side cells transfected with the BAP-receptor together with streptavidin-coupled magnetic beads. After the immunoprecipitation the cells were washed twice with PBS and it was visible that the portion of transgenic (red fluorescent) cells was largely increased after this procedure. The depicted SEM image shows two BAP-receptor transfected MEXi cells which have a cluster of magnetic streptavidin-conjugated magnetic beads at their cellular interface. At the border of the cells the superimposed fluorescence image of these cells shows red fluorescence located at the periphery of the cells indicating the presence of mRuby3 fluorescent protein at the cellular membrane. The green fluorescence is caused by biotinylated eGFP that was used for other preparations on the same microscopic slide and came into contact with these cells when the suspension cells were sedimented on the microscopic slide by centrifugation. Although for a reliable conclusion one would have to evaluate several such images this figure already gives a valuable evidence that the BAP-receptor is present on the cells (although perhaps at lower levels) and can bind to strepavidin that was conjugated to the beads.
</P></DIV><H1><SPAN class="mw-headline" id="Procaryotic_receptor_constructs">Procaryotic receptor constructs</SPAN></H1><H1><SPAN class="mw-headline" id="Design_of_an_autotransporter_construct_for_bacterial_surface_display">Design of an autotransporter construct for bacterial surface display</SPAN></H1><DIV class="white-box"><P>Besides the design of biotinylated and biotin-binding receptors for eukaryotic cells, we also wanted to apply our biotINK approach for prokaryotic cells. For this purpose, we designed an autotransporter device that is able to present biotinylated or biotin-binding protein domains on the surface of <I>E. coli</I>. Constructs for bacterial surface display are hereby already well known in the field of protein engineering of therapeutic proteins, such as antibodies. There, they are used to screen libraries of different antibodies concerning their affinity towards a given therapeutic target. 
</P><P>We based our design on the autotransporter EspP<SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP>.
</P><P>The autotransporter system itself is established and functional (see Fig. A<SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP>), and here we &quot;bricked&quot; it down into RFC[10] and RFC[25] BioBricks. At first we tested different inducible bacterial promoter systems (e.g. &quot;TetR repressed GFP&quot;; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K577893">BBa_K577893</A>) that are available in the parts registry, but when we saw that two of the available systems did not work in our own hands, we decided to built the system on our own. We used the Tet-Repressor generator that was available on the distribution plates (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I739001">BBa_I739001</A>) as we believe that it is very important for the standardization in SynBio to use available parts. Downstream of this repressor generator we designed and fused a BioBrick (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170141">BBa_K2170141</A>) that encodes a double Tet-Operator (Tet-Repressor binding site together with a bacterial promoter and an signal peptide of the outer membrane protein A (OmpA<SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP>) from <I>E. coli</I> that has a 3' RFC[25] restriction site, allowing the protein fusion of proteins that are secreted into the bacterial periplasm when expressed with this BioBrick. As a protein fusion, we then assembled a protein cargo domain that is generally exchangeable for any protein that is so small in size that it can be transported through the outer membrane by the EspP autotransporter. In our project we used the same extracellular domains as for the eukaryotic receptors: enhanced monomeric Avidin (eMA; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170204">BBa_K2170204</A>), single-chain Avidin (scAvidin; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170205">BBa_K2170205</A>) and a composite part composed of a N-terminal Biotinylation acceptor peptide (BAP) and a C-terminal NanoLuciferase (together shown as BAP;Â ; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2170118">BBa_K2170118</A>, see Fig. B). Further downstream our design features a A3C5-epitope tag that can be recognized by a high affinity Fab-fragment and is generally used to detect and quantify the surface display of the cargo domain<SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP>. Further downstream of this affinity tag we fused the BioBrick for the EspP autotransporter from <I>E. coli</I><SUP id="cite_ref-8" class="reference"><A href="#cite_note-8">[8]</A></SUP><SUP id="cite_ref-9" class="reference"><A href="#cite_note-9">[9]</A></SUP> which is then followed by a bacterial terminator. For the termination we again chose a well working terminator from the distribution plate (&quot;double terminator&quot;; <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0010">BBa_B0010</A>-<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0012">BBa_B0012</A>). But this was just the DNA-part. What is expected to happens on a protein level? The construct constantly produces Tet-repressor that binds to thet Tet-Operator and repressed the promoter activity of the autotransporter gene. As soon as the inducer anhydrotetracycline (aTc) is added to the culture, it binds to the Tet-repressor that can't bind any more to the Tet-operator and thus the expression of the autotransporter can be regulated. Although the TetR-system is known to be a tight promoter system there is always a certain background expression with most promoters. If the protein expression of the autotransporter is induced using aTc the autotransporter is transcribed and translated. Due to the bacterial signal peptide the protein is secreted into the bacterial periplasm (the space between the two bacterial membranes of gram-negative bacteria such as <I>E. coli</I>). When present in the bacteria periplasm the autotransporter diffused to the outer membrane and inserts into the membrane. After the integration into the outer membrane the protein cargo is transportet through the beta-barrel of the autotransporter and is pre sented on the bacterial surface. We are sure that this BioBrick will be a valuable contribution to the Parts Registry as it allows future teams to display small protein domains on the surface of gram-negative bacteria, such as <I>E. coli</I>.
</P></DIV><H2><SPAN class="mw-headline" id="Results:_Testing_the_bacterial_autotransporter_system">Results: Testing the bacterial autotransporter system</SPAN></H2><DIV class="white-box"><P>For the testing of our autotransporter constructs we transformed all three autotransporter constructs together with a secreted NanoLuc luciferase, which served as a control, into <I>E. coli</I> JK321<SUP id="cite_ref-10" class="reference"><A href="#cite_note-10">[10]</A></SUP>. Transformed cells were grown in LB-medium supplemented with chloramphenicol (pSB1C3-vector) at 30Â°C and 180 RPM. In the morning a stationary over night culture was used to inoculate 50 mL fresh medium with 1 mL of culture. The cultures were grown to OD<SUB>550</SUB> = 0.5 and then the induction of the protein production was started using aTc: &quot;non&quot; = non-induced; &quot;low&quot; = 25 ng/mL aTc; &quot;high&quot; = 200 ng/mL. After induction the cultures were grown for further 5 h at the same concitions and cells were afterwards stored at 4Â°C until they were used for analysis or bioprinting.
After the cell harvest the optical density (OD<SUB>550</SUB>) was determined again as it is known that a high expression of EspP makes the outer membrane of <I>E. coli</I> permeable and this drastically affects cellular viability. For the scAvidin-autotransporter and the secreted BAP-NanoLuc autotransporter at high inducer concentration show clearly reduced viability as they did not show any growth after induction at OD<SUB>550</SUB> = 0.5 (see Fig. A). In order to analyze the functionality of the whole design and these new BioBricks we subjected each of the twelve cultures to three different treatments and determined subsequently the amount of NanoLuc luciferase using the manufacturers substrate kit (Thanks to our sponsor PromegaÂ ;).
At first the total NanoLuc luciferase content was determined in samples that were just washed once with PBS and contained the whole bacterial cell (see Fig. B), bacterial cells that were incubated with PPA-buffer (periplasmic extraction buffer; 500 mM sucrose, 100 mM Tris/HCl pH 8.0, 1 mM EDTA) that extracts the proteins present in the periplasm (see Fig. C) and finally the culture supernatant of the cultivated bacterial cells (see Fig. D). The total NanoLuc activity is only present in bacteria that are transformed with a plasmid encoding the luciferase and there is a clear dependency of inducer concentration a detected luciferase activity. In the samples where a periplasmic extract was prepared the luciferase activity is highest in the samples of the secreted luciferase as the EspP-fused BAP-NanoLuc is attached to the cells and only proteins from lysed cells are present in the periplasmic extract. In the case of the samples where the supernatant of the cultures was analyzed the luciferase activity is present in both &quot;high&quot; induced samples.</P><P>As the luciferase activity only gave an indication for the subcellular localization of the cargo-autotransporter fusion protein, we also performed flow cytometry measurements. For this purpose the same cells were stained with app. 10 ÂµM A3C5-Fab that was coupled to the fluorescent dye Dy634-NHS and 100 ÂµM of the chemical compound biotin-atto488. Cells were incubated for 2 h at 4Â°C, washed twice with PBS and analyzed using a FACSAria II flow cytometer (BD bioscience, voltages for fluorescence channels were all at 750 V).
</P><P>For the channels FITC (biotin-atto488) and Dy634 (A3C5-F<SUB>ab</SUB>-Dy634, detected using a HeNe
laser (633 nm) and a 660/20 band pass ï¬lter) the fluorescence signal was measured and mean values are given. The cells showed a homogeneious population in the FSC/SSC plot. The A3C5-F<SUB>ab</SUB> staining detected high signals up til a mean of over 4000 for the eMA-autotransporter dependent on the inducer concentration. For the same bacterial cells the binding of biotin-Atto488 could be confirmed as well. The single chain Avidin (scAvidin) cells showed absolutely no A3C5 signal but a low Biotin-Atto488 background signal that could be caused by biotin-importers that transport biotin into the cells or the slow diffusion into the periplasm of <I>E. coli</I>. In contrast the BAP-Receptor also shows a high surface presentation as seen in the A3C5 signal of nearly 4000 for the &quot;high&quot; induced sample. The elevated biotin-Atto488 is unexpected but could be explained by the weakened and partly permeable out membrane that does not stop the small-molecule dye from entering the periplasm.
</P></DIV><H2><SPAN class="mw-headline" id="Discussion:_The_autotransporter_is_working_and_can_present_small_protein_cargos_on_the_bacterial_surface">Discussion: The autotransporter is working and can present small protein cargos on the bacterial surface</SPAN></H2><DIV class="white-box"><P>The performed experiment confirm the functionality of the device including the tetracyclin repressed promoter, the secretion into the bacterial periplasm as well as the presentation of the protein cargon on the bacterial outer membrane. In agreement with the expectation the 62 kDa big scAvidin with its extremely strong fold is too big in size for the autotransporter and can not be presented on the bacterial surface. In contrast the smaller BAP-NanoLuc (19 kDa for the NanoLuc) or the eMA (15 kDa) both can be efficiently presented on the surface. The enzymatic biotinylation of the Biotin Acceptor Peptide (BAP) using recombinant BirA biotin ligase is well established but could not be show before the wiki freeze. In Summary the whole device works great but there seems to be a limitation concerning the size of proteindomains that can be presented with the EspP autotransporter. We are curious to see what future iGEM teams will use the EspP Autotransporter in the future.
</P></DIV><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><DIV class="white-box"><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">â</A></SPAN><SPAN class="reference-text">Kredel, S., Oswald, F., Nienhaus, K., Deuschle, K., RÃ¶cker, C., Wolff, M., ... &amp; Wiedenmann, J. (2009). mRuby, a bright monomeric red fluorescent protein for labeling of subcellular structures. PloS one, 4(2), e4391.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">â</A></SPAN><SPAN class="reference-text">Gradinaru, Viviana, et al. &quot;Molecular and cellular approaches for diversifying and extending optogenetics.&quot; Cell 141.1 (2010): 154-165.</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">â</A></SPAN><SPAN class="reference-text">Livak, Kenneth J., and Thomas D. Schmittgen. &quot;Analysis of relative gene expression data using real-time quantitative PCR and the 2â ÎÎCT method.&quot; methods 25.4 (2001): 402-408.</SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">â</A></SPAN><SPAN class="reference-text">Binder, U., Matschiner, G., Theobald, I., &amp; Skerra, A. (2010). High-throughput sorting of an Anticalin library via EspP-mediated functional display on the Escherichia coli cell surface. Journal of molecular biology, 400(4), 783-802.</SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">â</A></SPAN><SPAN class="reference-text">Binder, U., Matschiner, G., Theobald, I., &amp; Skerra, A. (2010). High-throughput sorting of an Anticalin library via EspP-mediated functional display on the Escherichia coli cell surface. Journal of molecular biology, 400(4), 783-802.</SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">â</A></SPAN><SPAN class="reference-text">Ghrayeb, J., Kimura, H., Takahara, M., Hsiung, H., Masui, Y., &amp; Inouye, M. (1984). Secretion cloning vectors in Escherichia coli. The EMBO journal, 3(10), 2437.</SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">â</A></SPAN><SPAN class="reference-text">Costa, J., Grabenhorst, E., Nimtz, M., &amp; Conradt, H. S. (1997). Stable expression of the Golgi form and secretory variants of human fucosyltransferase III from BHK-21 cells Purification and characterization of an engineered truncated form from the culture medium. Journal of Biological Chemistry, 272(17), 11613-11621.</SPAN></LI><LI id="cite_note-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-8">â</A></SPAN><SPAN class="reference-text">Barnard, T. J., Dautin, N., Lukacik, P., Bernstein, H. D., &amp; Buchanan, S. K. (2007). Autotransporter structure reveals intra-barrel cleavage followed by conformational changes. Nature structural &amp; molecular biology, 14(12), 1214-1220.</SPAN></LI><LI id="cite_note-9"><SPAN class="mw-cite-backlink"><A href="#cite_ref-9">â</A></SPAN><SPAN class="reference-text">Skillman, K. M., Barnard, T. J., Peterson, J. H., Ghirlando, R., &amp; Bernstein, H. D. (2005). Efficient secretion of a folded protein domain by a monomeric bacterial autotransporter. Molecular microbiology, 58(4), 945-958.</SPAN></LI><LI id="cite_note-10"><SPAN class="mw-cite-backlink"><A href="#cite_ref-10">â</A></SPAN><SPAN class="reference-text">Jose, J., KrÃ¤mer, J., Klauser, T., Pohlner, J., &amp; Meyer, T. F. (1996). Absence of periplasmic DsbA oxidoreductase facilitates export of cysteine-containing passenger proteins to the Escherichia coli cell surface via the Iga Î² autotransporter pathway. Gene, 178(1), 107-110.</SPAN></LI></OL></DIV></DIV></DIV><DIV class="row"><FOOTER class="col-sm-9 col-sm-offset-3 noprint page-footer clearfix white-box"><A class="btt visible-xs-block" href="#top"><SPAN class="sr-only">Back to top</SPAN></A><DIV class="row"><DIV class="col-sm-4 social"><H2>Follow us on:</H2></DIV><DIV class="col-sm-4 universities"><H2>LMU &amp; TUM Munich</H2><P>United team from Munich's universities</P></DIV><DIV class="col-sm-4 address"><H2>Contact us:</H2><H3>Address</H3><P>iGEM Team TU-Munich
                     Emil-Erlenmeyer-Forum 5
                     85354 Freising, Germany</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>