Team:Thessaloniki/Model/dCas9
ig
dcas9 - sgRNA Model
Figure 1: Schematic representation of the system
After the study of TALE system we moved on to the study of the model, which                     uses as repressor the complex dcas9-sgRNA, whose schematic representation                     is shown in Figure 1.
This system, consists of three subsystems.The main subsystem, is the one                     that replaces the system of TALE and has the same functionality. The only                     difference is that now the repressor consists of the complex dcas9-sgRNA                     and for that reason, instead of the transcription and translation of the                     repressor, we just have a binding of the two molecules. The other two                     subsystems which were studied are about the creation of those two                     molecules. The dcas subsystem works with extra repression for its                     production, which is controlled via Doxycycline.In contrast, the sgRNA                     subsystem operates by activation through Rhamnose-RhaS complex which is                     also adjustable.
For each one of those subsystems, we have studied separately their                     characteristics and the parameters which affects more the output of every                     subsystem. In order to achieve that, we used the knowledge that we gained                     from studying the TALE system. Along the way, we realized possible problems                     and gave solutions to them, which we then shared with Wet lab, so as to                     integrate them in system design.
dcas9 subsystem
We started by studying the dcas subsystem. The dcas is repressed by the                     tetR, because in large quantities can be toxic for the cell. Also, by                     adding doses of Doxycycline, which in turn creates complex with tetR, we                     managed to decrease the repression that happens in dcas to the desired                     level.
As shown in Figure 1, the dcas9 and the tetR are on a different plasmid                     than the rest of the system. This means, that the change in copy number                     that we examine does not affect the plasmids which are responsible for the                     production of the dcas9 and the tetR. After the completion of the modeling                     process, an analysis was held to consider other options that Wet lab                     proposed, regarding the different expression sites of dcas.
For the mathematical representation of the system, we took into                     consideration the following assumptions:
Inducer dynamic Doxycycline: We assume that as soon as                             Doxycycline                             is injected into the solution, is stabilised in a steady state                             inside the cell and we do not take into account influx and outflux                             rates.The concentration of Dox inside the cell is set to 8ng/ml,                             that is 2.4 mM. When the complex [TetR: Dox] is degraded with                             the same rate as tetR, we assume that Dox can be free inside the                             cell again.
tetR dimerization, binding to Doxycycline and binding rate to                                 tetO2 operator sites: A tetR dimer is sufficient for                             specific binding to the operator sequence. However, a tetR monomer                             is unable to bind and exert any repressive effect. Thus, we assume                             that the repressor dimer binds to the operator (denoted as species                             O) with a one-to-one stoichiometry and also assume that a dimer                             binds one to one with Doxycycline. These assumptions are valid, as                             monomers forming a dimer and dimer forming a complex with                             Doxycycline are very fast interactions and do not affect other                             simulation components. As a consequence, we halved the translation                             rate of tetR and the concentration of Doxycycline that was injected                             into the solution.
We expressed the Hill equation as mass action kinetics for the two                             different states as we did in TALE system.
The subsystem is described by the following kinetic reactions and                     differential equations, which we used to implement the simulation in                     Simbiology, and also in python.
Parameters
Biological meaning
cn
psb3k3 plasmid
at
transcription rate of mtetR
bt
translation rate of tetR
dmt
degradation rate of mtetR
dt
degradation of tetR
kdt
binding rate of tetR to Doxycycline
kdt-
unbinding rate of tetR to Doxycycline
kon2
binding of tetR to promoter site
koff2
unbindng rate of tetR to promoter site
Tmax
maximum transcription rate of dcas9
Tmin
minimum transcription rate of dcas9
bd
translation rate of dcas9
dmd
degradation of mdcas9
dd
degradation rate of dcas9
n
cooperativity tetR to pTet promoter
Parameter estimation
For parameter values, first estimates were made through bibliography and                     methods which were used also for TALE, such as RBS Calculator.
The parameters for which it was deemed necessary to do further analysis, as                     bibliographic references proved to be quite uncertain, are the binding and                     unbinding rates between doxycycline and tetR and also those of tetR with                     DNA, in the two position of pTet promoter. So the sensitivity analysis, was                     applied to these parameters.
Figure 2: Sensitivity analysis on uncertain model parameters. Sample                         from each parameter: 20.000. Parameter sets: 120.000
As can be seen in Figure 2, the kon2 between tetR and DNA shows                     approximately 0.6 first order index and taking into account also higher                     order interactions, this is the most influential parameter of the system.                     The toxicity of dcas is an important limiting factor which we should                     consider.Therefore, its concentration must be under control via the tetR                     repression and the Doxycycline dose.
Taking into consideration all the above, we decided to examine the system                     behaviour with different sets of parameters. Specifically, we used all the                     parameters that was used in sensitivity analysis and we created 5000                     different sets. The values of the kinetics ranged from 0.001 to 10.
Figure 3: Scatter plot from the analysis under different parameter                         settings.
The data from the analysis (Figure 3), indicate that the mean value of                     dcas9 concentration is approximately 0.826 nM. This result appears to be                     problematic, as a concentration this low, would set an upper bound on the                     final repressor and by extension, sfGFP could not be independent from the                     copy number, in any case.
While trying to locate the problem, we noticed that, in order for the                     system to acquire a reasonable dcas concentration, we had to change its                     dissociation constant, .In Figure 3, kd’s                     maximum value is ,but scanning this parameter for a bigger range                     we found that if we                     increase it to ,we will finally get satisfying                     results.
Figure 4: dcas concentration over time. Effects of tetR repression for                         different kd.
After we shared this result with Wet lab, combined with the fact that the                     kd for which our system works, disagrees with those found in bibliography                     [4][5], we decided to make an experiment to validate the functionality of                     the dcas subsystem. The results of the dcas9 toxicity measurement,                     indicate that dcas is produced, in very good level, without causing any                     problem for the cell growth.
The reason that the concentration of dcas in the model depends that much by                     tetR’s ability to bind to DNA, we believe is due either to wrong                     configuration of tetR or to the modeling of the Doxycycline dose in the                     system. Having received the feedback from the experiment, we wanted to                     correct the model so as to bring the right result. We decided, to not                     change the modeling of the dose, but to adjust the concentration of dcas,                     by changing the aforementioned kd as seen in Fig.4. We ended up, that the                     value of the constant must be equal with as its                     concentration is greatly                     increased for other, bibliography based, values [2].
sgRNA subsystem
The sgRNA is produced by the promoter Pbad, only when he is activated by                     the complex Rhamnose-RhaS. In contrast, with the repressed function of                     dcas, the sgRNA is activated by that complex and that is induced by                     Rhamnose, which we add as a dose to the system.
As shown in Fig.1, the production of protein RhaS is done by the genome of                     the cell and therefore the concentration is independent from changes in its                     copy number of sfGFP. From that fact, we can understand that the complex                     which it creates with Rhamnose has this upper limit in each experiment for                     different copy number. The exact amount of complex and by extension the                     activation for the production of sgRNA, will directly depend from the                     Rhamnose that will be provided to the system.
For the mathematical representation of the system, considered the following                     assumptions:
Inducer dynamic Rhamnose: Stabilization of Rhamnose as soon                             as it’s injected to the solution. As Rhamnose degradation rate is                             quite lower than this of rhaS, we assume that the degradation rate                             of the complex is the same as rhaS degrade. When rhaS degrades the                             free Rhamnose molecule is added to total Rhamnose concentration.
RhaS dimer and binding rate to operator site of PrhaBAD:Like                             in tetR, we assume that Rhas is produced as a dimer because of it’s                             fast dimer binding. Because of its fast binding with L-Rhamnose we                             also assume, one to one stoichiometry. In the presence of                             L-rhamnose,                             its concentration is 0.02 g/ml or 1.2 mM,                             Expression from the rhaBAD promoter is log linear with n=1 [7] with                             respect to inducer concentration.
Again, Hill equation was expressed as mass action kinetics                             equations.
The following kinetic reactions and differential equations were used to                     describe the system mathematically:
Parameters
Biological meaning
ah
transcription rate of mrhaS
bh
translation rate of rhaS
dmh
degradation rate of mrhaS
dh
degradation of rhaS
khr
binding rate of Rhas to Rhamnose
khr-
unbindng rate of Rhas to Rhamnose
kon1
binding of [rhaS:Rham] to promoter site
koff1
unbindng rate of[rhaS:Rham] to promoter site
hmax
maximum transcritpion rate of sgRNA
hmin
minimum transcription rate of sgRNA
ds
degradation of sgRNA
n
cooperativity of [rhaS:Rham] to PBAD promoter
Parameter estimation
After using a bibliographic approach (see bibliography based estimation) to                     find the parameter values of the subsystem sgRNA, we ended up in the                     following uncertain parameters:
Binding and unbinding rates of Rhamnose with RhaS.
Binding and unbinding rates of Rhamnose:RhaS complex with Pbad                             promoter.
Degradation rate of RhaS
We applied sensitivity analysis for these parameters, using the same rate                     ranges as dcas9 subsystem and the degradation variation is 0.001 to 1. The                     results are shown in Figure 5 where you can see that the most influential                     parameter is clearly the binding rate to Pbad promoter.
Figure 5: Sensitivity analysis on uncertain model parameters. Sample                         from each parameter: 20.000. Parameter sets: 120.000.
Binding and unbinding rates of RhaS to Rhamnose do not have any effect at                     all and we assume this is due to the very big concentration of Rhamnose in                     the system. (set to 1.2 mM) Τhe other two parameters have relatively high                     total order indices because of their interaction with kon (same                     differential equation).
In order to get more insights regarding the system behaviour, we applied a                     robustness analysis, shown in Figure 6. The result appeared as expected                     with sgRNA concentration increasing for higher copy number. The standard                     deviations show us that for little copy number the variation of the                     concentration is trivial but for higher copy numbers the variation                     increases accordingly. However, we can clearly tell that for most parameter                     sets the linearity of the system cannot be ruined.
Figure 6: Robustness analysis for the sgRNA concentration for copy                         number variations. The blue line is the mean sgRNA concentration of all                         parametric sets calculated for different copy number. The error bars                         represent the standard deviation of the mean, that is how much the                         sgRNA concentration can vary.
Main system
The main system consists of the formation of the repressor complex and the                     production of sfGFP, which is being repressed by the complex. The complex                     binds to DNA slowly, as dcas9 has to find the PAM sequence.The dissociation                     is also very slow and approaches the dillution rate of the cell. The                     disruption percentage of RNAP, arises from sgRNA’s complementarity with DNA                     and the concentration of polymerase. We also assumed the degradation of the                     repressor complex, being the same rate as dcas9 degradation.
Parameters
Biological meaning
c
inducible copy number ( low,medium,high)
kds
binding rate of dcas9:sgRNA
kds-
unbinding rate of dcas9:sgRNA
kon3
binding of [dcas9:sgRNA] to promoter site
koff3
unbinding of [dcas9:sgRNA] to promoter site
aGmax
maximum transcription rate of sfGFP
aGmin
minimum transcription rate of sfGFP
bG
translation of sgGFP
dmG
degradation of msfGFP
dG
degradation of sfGFP
Parameter estimation
As previously done with the subsystems we studied bibliography to find out                     about system parameters and their ranges. After that, we applied                     sensitivity analysis for the association and dissociation constants.                     Interestingly, the results showed that the repressor’s kdson and kdsoff                     does not affect the system output. After changing some values of these                     parameters we found that this is true and the system appeared to be very                     robust regarding the repressor concentration. Due to the strong repression                     that it does, combined with its very small degradation rate, the repressor                     is binding to DNA and almost never dissociates, thus making small                     repression concentrations sufficient for the system to work. Specifically,                     using different values we were able to get exactly the same output error                     with many combinations of kdson and kdsoff.
Figure 7: First and Total order Sensitivity indices for Main system’s                         association and dissociation constants
Figure 8: Heatmap Representation of Second order Sensitivity indices.                         The bar in the right showcases the color-value match.
The second thing we noticed that reinforces the previous statement is that                     the second order indices have a peak for the interaction of kon3 with                     kdsoff. That makes sense, because in the case of decreasing the repression                     strength, the system needs more repressor to operate the same.
Knowing from fact that the dcas9-sgRNA repression is very strong [1][2][3],                     we fixed the parameters accordingly (see Bibliography based parameter                     estimation) and finished model characterization. However, the final model                     produced an Error of approximately 22, which we will discuss in the next                     section.
dcas9-sgRNA Repressor analysis
In the previous section, we draw the conclusion from the results of                     sensitivity analysis, that the system is not sensitive regarding the                     formation of repressor, thus we can tell as an extension of this that the                     system is robust concerning the concentration of dcas and sgRNA.
On the other hand, we found out that the system appears to be generally                     unusable with an error of 22, that was hard to change. Scanning the most                     influential parameter, kon3, as evidenced by Figure 7, we were able to                     change the repressive power of the repressor complex and produce sfGFP                     concentration of different magnitudes, though the system Error (as                     described on the system evaluation section of TALE) did not show to be                     affected.
This could potentially mean, that the system would fail, even if we didn’t                     have set the binding affinities of the system to their right values with                     our characterization. That is, because even when trying to change the error                     by scanning different parameters and combinations of them, we weren’t able                     to correct the situation.
In order to find the root of this problem we had to think backwards. While                     plotting different component concentrations during simulations, we                     realised, that the repressor concentration was the same for different                     plasmid copy numbers and then we presented the problem mathematically,                     proving, that a constant repressor amount can not make the system’s output                     independent from the copy number. Below we present the proof of this                     statement.
Our goal is to minimize the error, between sfGFP expression for                                 low (cmin) and high (cmax) copy numbers, find the conditions                                 under which, the system is stabilized and to prove that this                                 cannot be achieved with a constant repressor concentration over                                 increasing copy number. For the gene of interest and the repressor                                 we use the annotatuon G and R respectively.
In our case, the gene expression of hill equation, has                                 cooperativity equal to one and the ratio of unbinding and                                 binding rate is equal to kD (dissociation constant) expressed                                 as
(1)
with
where is the maximum trascription rate, is                                 the translation rate and the degradation rate of G and also is                                 the degradation rate of mG.
kD is the concentration of the repressor, in which half                                 repression of G is achieved and we can assume, that in the                                 steady state for every copy number, , thus (1) becomes
(2)
For simplification purposes let's consider that we have                                 constitutive promoters for the expression of dcas9 and sgRNA.                                 Dcas9 is expressed constantly by a medium value copy number,                                 cn, while sgRNA's copy number increases in the interval [cmin,                                 cmax], as it is located on the same plasmid as sfGFP is. Then,                                 the analytical solution on the steady state for sgRNA and dcas9                                 concentration is
with cn = constant and
The dcas9 and sgRNA binding has one-to-one stoichiometry, then                                 for each copy number when the reversible reaction reaches in                                 equilibrium, the relationship between the reactants and the                                 product R is
(3)
If for then                                 dcas9 acts as the limiting reagent for the formation of the                                 complex R, which by extension has a maximum concentration of                                 acn. Specifically its concentration is given below
with Notice                                 that R is independent of the copy number c.
We can now replace R in (2) and get
Likewise if sgRNA is the limiting reagent, that is for Replacing the corresponding R to (2) we end up with the                                 following relationship
proving that for R, which is scaling proportionally to copy                                 number increments, expression of G is independent of the copy                                 number variation.
Figure 9: On the y axis the concentrations of dcas9, sgRNA and by                         extension, the repressor’s are presented. On the x axis we have copy                         number variation. The concentration concerns the steady state of the                         system for every copy number.
From the graphical representation, we can easily tell, that sgRNA indeed                     surpasses the level of dcas concentration at copy number 50. Thus, after                     that -let’s name it- breaking point, the repressor is constant for the                     remaining copy numbers, with dcas9 acting as the limiting reagent. It is                     obvious that, we had to change the repressor amount so as to be increased                     for increasing copy number. This could be achieved by either increasing                     dcas9 concentration or decreasing sgRNA’s one. We chose the latter option,                     because, as we concluded in dcas subsystem section, the amount of dcas is                     already enough and if increased, due to its toxicity it could be a problem                     for the cell’s growth.
In Fig. 10 we present the fixation of copy number variation. For a kd of 10                     raised to the power of 4 we can see that dcas9 becomes the restriction                     reagent for high copy numbers. It is important to mention, that changing                     the binding affinity is difficult as it requires to change also the                     promoter and different promoters have discrete amount of affinity. This can                     be solved by changing the Rhamnose dose for small changes needed. Though                     the amount of Rhamnose should be small to make a difference because this time the limiting reagent is RhaS and its concentration is very small.                     We were able to make the aforementioned kd value stay under dcas9                     concentration barrier, by providing Rhamnose near the value of 150 nM or                     lower.
Figure 10: Changing the binding affinity of the Rhamnose:RhaS complex                         with the promoter of sgRNA.
Finally, as we found that the concentration of Repressor is not a problem                     for the system (main system section), we concluded, that we can have                     different magnitudes of sfGFP with very small errors, by optimizing sgRNA                     concentration.
dcas9 Expression Site
In this section we are going to examine the different options we had for                     the expression site of dcas9 in order to inform Wet lab, which is the                     better option for our system. The three options that the Wet lab was                     thinking were:
dcas to be expressed on the same plasmid with sgRNA and sfGFP
Double plasmid solution where dcas is expressed in the same plasmid                             as tetR.
dcas to be expressed directly from the genome.
The second option is the one that we examined until now. For all three                     options we set tetR’s copy number as 15.
Figure 11: Dcas9 Concentration over different copy numbers for all three options
Figure 12: sfGFP concentration over different copy numbers for all three options
As we expected, the results show, that when dcas is expressed from the                     genome, sfGFP is greatly increased as copy number is increased, because it                     sets a very low upper limit for the repressor concentration, which is                     reached very soon even for low copy numbers.
When dcas9 is expressed within the same plasmid as sfGFP, there is no                     limiting reagent in the system and both sgRNA and dcas are increasing.                     The final sfGFP production and system error is exactly the same with the                     final option, which is also our main system. This can be interpreted as in                     the main system we concluded that the system is robust, regarding the                     repressor binding and concentration, due to the final strong repression,                     slow unbinding and also slow degradation rate of the repressor. However, in                     the case of double plasmid, we are able to monitor the expression of dcas,                     in contrast with the single plasmid were dcas is constantly increasing.
Taking the case that we do not use tetR for dcas repression, the only                     difference is that the expression of dcas would be even higher.
Also we thought of having tetR and dcas both expressed in the same plasmid                     as sfGFP, but the results were not good. Actually dcas9 concentration was                     decreasing while copy number were increased. This meant that for high copy                     numbers, tetR concentration scales fast, thus the repression on dcas is                     stronger.
Bibliography based parameter estimation
All the values for the parameters of this model are presented in the                     following table. The parameters for which the bibliography is estimated are                     the ones that were uncertain and where estimated within each subsystem                     section of parameter estimation.
Using the same methods as we used in TALE system found or calculated the                     transcription, translation and degradation rates of the proteins [8][9].
The minimum transcription rate of repressed promoters was assumed 0.0001                     for all proteins, because.
For all mRNAs, we found that degradation rate is approximately 0.2 for                     E.coli cells [10]. Regarding protein degradation rates, they all have been                     found through different studies with the exception of rhaS, whose parameter                     has been fixed, because it did not create much variance to sgRNA                     expression, as shown in Fig. 5. The fixated value was small based on other                     protein degradation rates.
Product
Sequence length
Transcription rate nt/min
Translation rate(1)/min
Translation rate(2)/min
translation rate /min
tetR
624
4.33
5.76
0.13
1.82
dcas9
4107
0.66-0.73
0.87
0.42
0.56
rhaS
837
3.22
4.3
0.02
1.3
sfGFP
714
3.78
05.04
03.05
3.65
Parameters
Biological meaning
Value
Units
Bibliography
cn
medium psb3k3 plasmid
15
no dimension
-
at
transcription rate of mtetR
4.33
1/min
[9]
bt
translation rate of tetR
1.819
1/min
[8][9]
dmt
degradation rate of mtetR
0.2
1/min
[10]
dt
degradation of tetR
0.017
1/min
[5]
kdt
binding rate of tetR to Doxycycline
126
1/nM*min
[6]
kdt-
unbinding rate of tetR to Doxycycline
0.12
1/min
[6]
kon2
binding of tetR to promoter site
0.0001
1/nM*min
estimated
koff2
unbindng rate of tetR to promoter site
10000
1/min
estimated
Tmax
maximum transcription rate of dcas9
0.73
1/min
[9]
Tmin
minimum transcription rate of dcas9
0.0001
1/min
-
bd
translation rate of dcas9
0.55
1/min
[8][9]
dmd
degradation of mdcas9
0.2
1/min
[10]
dd
degradation rate of dcas9
0.005
1/min
[3]
n
cooperativity tetR:DNA
2
no dimension
[4][5]
ah
transcription rate of mrhaS
3.22
nM/min
[9]
bh
translation rate of rhaS
1.304
1/min
[8][9]
dmh
degradation rate of mrhaS
0.2
1/min
[10]
dh
degradation of rhaS
0.06
1/min
estimated
khr
binding rate of Rhas to Rhamnose
100
1/min*nM
estimated
khr-
unbindng rate of Rhas to Rhamnose
100
1/min
estimated
kon1
binding rate of [rhaS:Rham] to DNA site
0.001
1/min*nM
estimated
koff1
unbindng rate of [rhaS:Rham] to DNA site
10
1/min
estimated
hmax
maximum transcritpion rate of sgRNA
5
nM/min
[3]
hmin
minimum transcription rate of sgRNA
0.0001
1/min
-
ds
degradation of sgRNA
0.18
nM/min
[3]
n
cooperativity [rhaS:Rham]:DNA inducible copy number (                                     low,medium,high)
1
no dimension
[7]
c
binding rate of [dcas9:sgRNA]
5-100
no dimension
-
kds
unbinding rate of [dcas9:sgRNA]
10
1/nM*min
[3]
kds-
binding of transcriptional repressor [dcas9:sgRNA] to                                     promoter PG site
0.001
1/min
[3]
kon3
unbinding of transcriptional repressor
0.693
1/nM*min
[3]
koff3
[dcas9:sgRNA]to promoter site
0.018
1/min
[3][2]
aGmax
maximum transcription rate of sfGFP
3.78
1/min
[9]
aGmin
minimum transcription rate of sfGFP
0.0001
1/min
-
bG
translation of sgGFP
3.65
1/min
[8][9]
dmG
degradation of msfGFP
0.2
1/min
[10]
dG
degradation of sfGFP
0.0193
1/min
[11]
Discussion
Our analysis aims at giving a better understanding of the system behaviour.                     We achieved that by building the subsystems of the model and combining                     them, in an effort to characterize the model as best as possible. Methods                     such as sensitivity and robustness analysis helped us to analyze and see                     connections between parameters and focus on the more important aspects of                     the system. By using those, we were also able to detect problems that the                     system had.                     Finally, we gave feedback for the system design by providing the better                     option regarding the dcas9 expression site. Also, by making the system                     sgRNA optimized, after the analysis of the repressor that we made, we                     provided helpful insights regarding the relative concentrations of sgRNA                     and dcas9 that made the system work properly.
[1] Nielsen, A. and Voigt, C. (2014). Multi-input CRISPR/Cas genetic                         circuits that interface host regulatory networks. Molecular Systems                         Biology, 10(11), pp.763-763.
[2] Zhang, S. and Voigt, C. (2018). Engineered dCas9 with reduced                         toxicity in bacteria: implications for genetic circuit design. Nucleic                         Acids Research.
[3] Clamons, S. and Murray, R. (2017). Modeling Dynamic Transcriptional                         Circuits with CRISPRi.
[4] Bintu, L., Buchler, N., Garcia, H., Gerland, U., Hwa, T., Kondev,                         J., Kuhlman, T. and Phillips, R. (2005). Transcriptional regulation by                         the numbers: applications. Current Opinion in Genetics & Development,                         15(2), pp.125-135.
[5] Mondaini, R. (2012). Biomat 2011 - International Symposium on                         Mathematical and Computational Biology. Hackensack: World Scientific                         Publishing Company, Incorporated.
[6] Kedracka-Krok, S., Gorecki, A., Bonarek, P. and Wasylewski, Z.                         (2005). Kinetic and Thermodynamic Studies of Tet Repressor−Tetracycline                         Interaction†. Biochemistry, 44(3), pp.1037-1046.
[7] Kelly, C., Liu, Z., Yoshihara, A., Jenkinson, S., Wormald, M.,                         Otero, J., Estévez, A., Kato, A., Marqvorsen, M., Fleet, G., Estévez,                         R., Izumori, K. and Heap, J. (2016). Synthetic Chemical Inducers and                         Genetic Decoupling Enable Orthogonal Control of the rhaBAD Promoter.                         ACS Synthetic Biology, 5(10), pp.1136-1145.
[8] Espah Borujeni, A., Channarasappa, A. and Salis, H. (2013).                         Translation rate is controlled by coupled trade-offs between site                         accessibility, selective RNA unfolding and sliding at upstream standby                         sites. Nucleic Acids Research, 42(4), pp.2646-2659.
[9] Bionumbers.hms.harvard.edu. (2018). Transcription and translation                         rates - Bacteria Escherichia coli - BNID 109043. [online] Available at: Here
[10]Bionumbers.hms.harvard.edu. (2018). Typical mRNA degradation time -                         Bacteria Escherichia coli - BNID 108598. [online] Available at: Here
[11]2016.igem.org. (2018). Team:ETH Zurich/Parameters - 2016.igem.org.                         [online] Available at: https://2016.igem.org/Team:ETH_Zurich/Parameters
