<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Thessaloniki_Model_Tale skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Thessaloniki/Model/Tale</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="q-app" style="margin-top: 16px"><IGEM-LAYOUT><Q-PAGE class="text-justify q-pa-lg"><DIV class="row justify-center q-mb-md" :class="rM('text-dark', 'text-light')"><DIV class="col-xs-12 col-sm-10 col-md-9 col-lg-8"><DIV class="flex flex-center"><H3>TALE</H3></DIV><P>
                    The first system studied utilizes the TALE repressor and specifically the
                    TALEsp1 or TALEsp2 sequence.
                </P><P>
                    The kinetic reactions that make up the system are described by mass action
                    kinetics for the transcription, translation and degradation of molecules,
                    and by Hill kinetics described as mass action kinetics in two possible
                    states for binding and dissociating the TALE repressor at the binding site
                    of promoter P.
                </P><P>
                    The TALE gene is replicated at a constant rate from each plasmid c and then
                    translated into the TALE protein. This protein then binds to the promoter
                    of sfGFP (G), whose gene is on the same plasmid as TALE. It is associated
                    with dissociation constant kD = koff / kon and cooperativity n and produces
                    the final output of the gene of interest, which in our case is sfGFP.
                    This system has been studied and proved experimentally [1]. Our work on
                    this model aims to better study its parameters and their characterization,
                    as well as to reply to wet lab questions about the system and its possible
                    extensions.

                </P><P>
                    We started by describing the kinetic reactions that the model works with:
                    </P><P>
                    As the model follows the IFFL's network motif, what we expect is the
                    stabilization of sfGFP at a specific level, regardless of changes in the
                    number of plasmids (copy number, c). The analytical solution of the
                    differential equations resulting from the above kinetic reactions at steady
                    state, associates the concentration of sfGFP protein with the copy number
                    through the following ratio:
                </P><P>
                    where n is the cooperativity between the TALE repressor and the operator
                    site of DNA. So, easily, we can conclude that in order to isolate the
                    output from the plurality of plasmids, n should be equal to 1.
                    After studying and approximating the system parameters, which are described
                    in detail in the section of parameter estimations, we ended up with the
                    final featured model, which we used to get results and insights for the
                    system.
                </P><P>
                    Specifically, we run the simulation of differential equations for a copy
                    number of 5 to 100. For every copy number, the system was simulated for
                    1000 min and we noticed, that the steady state was reached after
                    approximately 400 min.
                </P><DIV class="text-center"><SMALL class="text-grey-6">
                        Figure 1: sfGFP concentration for copy numbers 5-100 for the system
                        TALEsp1.
                    </SMALL></DIV><H4>System evaluation</H4><P>
                    From figure 2, we can conclude that the system responds to our
                    expectations, but it seems necessary to use a metric to evaluate its
                    performance. So to interpret the performance, we will measure the error by
                    dividing the result of the difference between final (copy number = 100) and
                    initial (copy number = 5) sfGFP, with the initial sfGFP.
                </P><P>
                    The E of the system in figure 1 was proven experimentally [1] to be equal
                    to 0.52. This information was used in the robustness analysis section of
                    the parameter estimation.
                </P><H4>Cooperativity comparisons</H4><P>
                    The binding of TALE to the DNA is done through Hill's kinetics as mentioned
                    above and cooperativity n shows how strong the repress is.
                </P><DIV class="text-center"><SMALL class="text-grey-6">Figure 2: The gene of interest transcription
                        rate expressed as the maximum transcription rate multiplied by the
                        repressive Hill Function, relative to the TALE repressor concentration
                        for n = 1, n = 0.5 and n = 2. Maximum production refers to a single
                        plasmid.</SMALL></DIV><P>
                    We noticed that as we increase n, the steepness of the curve increases.
                    Then we wanted to prove via the model, that n must be 1, in order for the
                    system to function correctly. It is worth noting that the values ​​of all
                    the parameters for this experiment will not be used. Although TALE has been
                    found to have n = 1, different cooperativity may refer to different kinds
                    of repressor and their association with different promoters. The assumption
                    is that different cooperativities n will be due to a change of the promoter
                    and thus the parameters that will change are kon koff and the maximum
                    transcription rate of sfGFP, aGmax.
                </P><P>
                    We got the results from 3 robustness analyses, for n = 1, n = 2, and n =
                    0.5, using 10,000 different sets of parameters in each. The parameters of
                    the model to be varied are the kon,koff of the TALE-DNA binding and maximum
                    translation rate aG. The range obtained was based on what was used in the
                    sensitivity analysis and is 0.001 to 10. Through this procedure we
                    calculated the average sfGFP production for different copy numbers.
                    Finally, we calculated the Standard Error for each copy number with 95%
                    confidence intervals as shown in Figure 3.
                </P><DIV class="text-center"><SMALL class="text-grey-6">
                        Figure 3: Mean sfGFP concentration with Standard Error, for copy number
                        5-100 where <B>Α)</B> n=0.5, <B>B)</B> n=1 and <B>C)</B> n=2.
                    </SMALL></DIV><P>
                    We can easily notice, that if n is not 1 the system does not stabilize. We
                    also calculated the Error for every parameter set.
                </P><DIV class="flex flex-center"><DIV style="overflow-x:auto;"><TABLE class="simple-table text-center"><TBODY><TR><TD><B>Cooperativity coefficient, n </B></TD><TD><B>Mean Error of parameter sets</B></TD></TR><TR><TD>0.5 </TD><TD>4.03</TD></TR><TR><TD>1</TD><TD>0.39</TD></TR><TR><TD>2</TD><TD>11.35</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="text-center"><SMALL class="text-grey-6">Table 1: The mean error of all parameter sets
                        for different cooperativity coefficient.</SMALL></DIV><H4>Parameter Estimation</H4><H5>Sensitivity analysis</H5><P>
                    After the initial modeling of chemical reactions of the system, and having
                    set a range of values ​​for each kind of parameter, based on the type of
                    biological component and the nature of the reaction, we proceeded to
                    sensitivity analysis for all parameters. Our goal is to characterize the
                    parameters and apply this approach from a simple model to a more complex
                    one such as the dcas-sgRNA system we studied later on.
                </P><P>
                    Initially studying the bibliography and previous iGEM teams, we were able
                    to gather several values ​​for the rates of transcription, translation and
                    degradation of the components, and we set the range in which each parameter
                    ranges. Next, we created the samples within the defined range using
                    Saltelli's sampling scheme from SALib Library [3]. In particular, we used
                    for each variable N = 40,000 samples translated into 880,000 parameter
                    sets, for the calculation of first and second order Sobol indices through
                    the relation: sets=N(2D +2), where D is the number of parameters. At first,
                    we had a smaller sample size and we noticed that several sensitivity
                    indices values ​​were negative, or with a confidence interval greater than
                    50% of their value, so we increased sample size to get more accurate
                    values, which we judged based on how small their confidence interval is.
                </P><P>
                    Figure 4, shows First and Total order sensitivity indices from the first
                    sensitivity analysis we applied. This analysis contains all the parameters
                    of the system, and we used it as the initial criterion for the significance
                    of each parameter. The ranges of parameters, that were defined after our
                    study, are listed in the workflow sensitivity analysis section. Second
                    order indices of this analysis were not quoted, because their values were
                    so small that they make their investigation unnecessary.
                </P><DIV class="text-center"><SMALL class="text-grey-6">Figure 4: Sensitivity analysis for all the model
                        parameters. First and total order sensitivity indices is displayed for
                        every parameter, as well as their confidence intervals.</SMALL></DIV><P>
                    Thus, we see that First-order indices have too low values, which we
                    expected because of the complexity of the model. This shows, that each
                    parameter alone does not affect the result. However, summarizing first and
                    higher order interactions in Total-order indices, we observe the overall
                    sensitivity of the model in changing each parameter. Parameter values
                    ​​with a small Total-order index can be considered as constant at any value
                    within the range given to them without affecting the output variance. As
                    can be seen, the variables that most affect output, with Total-order
                    indices above 0.2 is the degradation of sfGFP, dG followed by the binding
                    rate kon, aT, bT and bG. The high influence of degradation on sfGFP is due
                    to the direct relationship to output of about 0.8 with confidence interval
                    (0.17).
                </P><H4>Bibliography based estimation</H4><Q-LIST><Q-COLLAPSIBLE icon="notes" label="Transcription, translation and degradation rates"><DIV><H5>Transcription and translation rates</H5><P>
                                Transcription and translation rates of TALE and sfGFP were
                                calculated by using the average transcription rate of 45 nt / s
                                and an average translation rate of 15 aa / s, respectively [7].
                                Also, the RBS Calculator from Salis Lab [7] was used and the
                                translation rates of mRNA in the system were also estimated. To
                                obtain the mRNA translation rate from each sequence, we had to
                                calculate the ratio  as described by
                                [6], based on total Gibbs
                                free energy between the ribosome and mRNA. In order to find the
                                translation rate of sfGFP, we found its bibliographic rate and
                                based on it and the corresponding sequence, in relationship
                                with our’s, starting at another codon the fold-change was
                                calculated between them and hence the rate.
                            </P><P>
                                In order to approximate the translation rate, we put weight in
                                both the RBS calculator results and the bibliography, which
                                provided the average translation rate of sequences with only
                                criterion, it’s length. In order to include both translation
                                rates from the two approaches, we used a weight function with a
                                weight factor for each.
                            </P><DIV class="flex flex-center"><DIV style="overflow-x:auto;"><TABLE class="simple-table text-center"><TBODY><TR><TD><B>Product</B></TD><TD><B>Sequence length</B></TD><TD><B>Transcription rate /min</B></TD><TD><B>Translation rate(1)/min </B></TD><TD><B>Translation rate(2)/min</B></TD><TD><B>Translation rate/min</B></TD></TR><TR><TD><B>TALEsp1</B></TD><TD>2600</TD><TD>01.03</TD><TD>1.38</TD><TD>0.03</TD><TD>0.44</TD></TR><TR><TD><B>TALEsp2</B></TD><TD>2600</TD><TD>01.03</TD><TD>1.38</TD><TD>0.18</TD><TD>0.55</TD></TR><TR><TD><B>sfGFP</B></TD><TD>714</TD><TD>3.78</TD><TD>05.04</TD><TD>03.05</TD><TD>3.65</TD></TR></TBODY></TABLE></DIV><DIV class="text-center"><SMALL class="text-grey-6">Table 2: The translation rates take
                                    two values that co-depend in different estimations.
                                    Translation rate(1) is calculated by average translation
                                    rate and Translation rate (2) is estimated via RBS
                                    calculator. The last column show the value of weight
                                    average function.</SMALL></DIV></DIV><H5>Degradation rates</H5><P>
                                The degradation of the protein’s mRNA, was found to be an
                                average of 0.2 [8] in E. coli cells. The degradation of TALE
                                was evaluated through the range of reasonable values from
                                protein degradation and as shown in figure 4, it does not
                                affect the end result.
                            </P></DIV></Q-COLLAPSIBLE></Q-LIST><DIV class="flex flex-center"><DIV style="overflow-x:auto;"><TABLE class="simple-table text-center"><TBODY><TR><TD><B>Parameters</B></TD><TD><B>Biological Meaning</B></TD><TD><B>Value</B></TD><TD><B>Units</B></TD><TD><B>Bibliography</B></TD></TR><TR><TD><B>aT</B></TD><TD>Transcription of TALE</TD><TD>1.03</TD><TD>nM/min</TD><TD>[7],</TD></TR><TR><TD><B>bΤ</B></TD><TD>Translation of TALEsp2,sp1</TD><TD>0.55,0.44</TD><TD>1/min</TD><TD>[6],[7]</TD></TR><TR><TD><B>dmT</B></TD><TD>Degradation of mRNA TALE</TD><TD>0.2</TD><TD>1/min</TD><TD>[8]</TD></TR><TR><TD><B>dT</B></TD><TD>Degradation of TALE</TD><TD>0.047</TD><TD>1/min</TD><TD>estimated</TD></TR><TR><TD><B>n</B></TD><TD>Cooperativity</TD><TD>1</TD><TD>dimentionless</TD><TD>[1]</TD></TR><TR><TD><B>kon</B></TD><TD>Binding rate to Promoter</TD><TD>100</TD><TD>1/nMmin</TD><TD>estimated</TD></TR><TR><TD><B>koff</B></TD><TD>Unbinding rate to Promoter</TD><TD>0.1</TD><TD>1/min</TD><TD>estimated</TD></TR><TR><TD><B>aG</B></TD><TD>maximum production rate</TD><TD>3.78</TD><TD>nM/min</TD><TD>[6]</TD></TR><TR><TD><B>bG</B></TD><TD>Translation of sfGFP</TD><TD>3.65</TD><TD>1/min</TD><TD>[6],[7]</TD></TR><TR><TD><B>dmG</B></TD><TD>Degradation of mRsfGFP</TD><TD>0.2</TD><TD>1/min</TD><TD>[8]</TD></TR><TR><TD><B>dG</B></TD><TD>Degradation of sfGFP</TD><TD>0.019</TD><TD>1/min</TD><TD>[9]</TD></TR></TBODY></TABLE></DIV></DIV><P>
                    The kon, koff and degradation values of Tale, dT were not found
                    bibliographically, and after replacing the constant values of the other
                    parameters in the model, we applied another Sensitivity to these three
                    values.
                </P><DIV class="text-center"><SMALL class="text-grey-6">
                        Figure 5: Sensitivity analysis for model parameters kon, koff and dT.
                        All the other parameters are fixed based on bibliography. First and
                        total order sensitivity indices is displayed for every parameter, as
                        well as their confidence intervals.
                    </SMALL></DIV><P>
                    As shown, the binding of the repressor to the promoter is a critical
                    parameter for the final stabilization of sfGFP. Under these circumstances,
                    it is necessary to estimate kD (= koff / kon). kD can be evaluated
                    experimentally as the point of the curve that the repressor concentration
                    leads to the expression of sfGFP in half of its repression. However, the
                    kon and koff reaction rates are not available and can not be found from
                    experimental data. Thus, different combinations of kon, koff can lead to
                    the same kD, which can cause a different time of arrival of the system to a
                    steady state. This does not affect our analysis as we do not study the
                    speed of the system.
                </P><P>
                    For these parameters, second order interactions appear to have interesting
                    results.
                </P><DIV class="text-center"><SMALL class="text-grey-6">
                        Figure 6: Heatmap representation of Second-order sensitivity indices
                        for the same analysis as Figure 5.
                    </SMALL></DIV><P>
                    We see that a major part of the Total influence of dT to the final output
                    is coming from the value of the kon. This is understandable as the two
                    parameters are in the same reaction and are therefore dependent.
                    After the process of sensitivity analysis and parameter definition, the
                    question we had, was whether we should set a value for dT. Finally, we have
                    left it variable for the moment and its definition was made after the
                    Robustness analysis.

                </P><H5>Robustness analysis</H5><P>
                    At this point three parameters made the result of the model variable, kon,
                    koff and dT. Wishing to approach the behavior of the model under different
                    values of these parameters, we applied a Robustness analysis by keeping the
                    average of the values of sfGFP per copy number for 10,000 parameter sets.
                    The Robustness analysis process is explained in the workflow.
                </P><DIV class="text-center"><SMALL class="text-grey-6">
                        Figure 7: Mean sfGFP production for TALE system under different
                        parameter settings.
                    </SMALL></DIV><P>
                    As stated at the beginning, our main goal was to characterize all the
                    parameters of the model. In order to do this, we combined the data we
                    obtained from robustness analysis with the proven TALE experimental error
                    [1]. This error is <B>0.52</B> for <B>TALEsp1</B> and <B>0.07</B> for <B>TALEsp2</B>.
                </P><DIV class="text-center"><SMALL class="text-grey-6">
                        Figure 8: System Error presented as a histogram with bin size=0.1
                    </SMALL></DIV><P>
                    The system seems to be very robust as the error is very low for almost all
                    parameter values. To characterize our last parameters all we needed to do
                    was, find the kon, koff and dT set with the aforementioned experimental
                    errors and also with the corresponding transcription rate. Though, in order
                    for the characterization to be valid we needed more data about sfGFP
                    concentration from experiments. Using only the system error we could find
                    many sets that were producing it. For different parameter sets, sfGFP
                    expression was different too. With an extra scan for bigger values of kd,
                    we found the same errors for very big amount of expressed sfGFP (Fig. 1).
                    Finally, we decided to keep the small kd because of the strong repression
                    that TALE does.
                </P><DIV class="flex flex-center"><DIV style="overflow-x:auto;"><TABLE class="simple-table text-center"><TBODY><TR><TD><B>Repressor</B></TD><TD><B>kon</B></TD><TD><B>koff</B></TD><TD><B>dT</B></TD></TR><TR><TD>TALEsp1</TD><TD>9.85</TD><TD>2.19</TD><TD>0.047 1/min</TD></TR><TR><TD>TALEsp2</TD><TD>1.86</TD><TD>3.85</TD><TD>0.047 1/min</TD></TR></TBODY></TABLE></DIV></DIV><H4 id="test">TALE tuning with IPTG-LacI</H4><P>
                    For the tuning of the final concentration of sfGFP, the system was studied
                    when LacI was placed after the sfGFP promoter on the same plasmid,
                    repressed by TALE. Our system is stabilized for specific binding
                    characteristics of the TALE with the sfGFP promoter as discussed above.
                </P><P>
                    When LacI binds to an operator site of the promoter of sfGFP, we have gene
                    expression even if a repressor is bound to the promoter too. The ideal case
                    with the incorporation of LacI would be to regulate the final amount of
                    sfGFP in each copy number at a small deviation around it ‘s final
                    concentration .
                </P><P>
                    Although, when LacI binds to the promoter, it appears to reduce the
                    possibility of TALE to also bind, due to steric interactions between the
                    two, thus making the system’s behaviour undesirable.
                </P><P>
                    Τhe competition mechanism between TALE and LacI on the binding sites of
                    sfGFP’s promoter P can be described as the following diagram:
                </P><P>
                    We will analyze the mechanism, in which, two ligands occupy different sites
                    of the promoter and assume that both interact due to electrostatic
                    interaction. Because their binding sites are close enough, we assume that
                    due to the size of LacI, Repressor TALE will display weak DNA binding when
                    it is to bind in the presence of LacI, increasing the dissociation constant
                    KT of TALE. This means that the binding affinity of the enzyme will be
                    reduced, but can be readjusted in the presence of IPTG in the system by
                    reacting with the free LacI reducing the likelihood of it, binding to the
                    promoter. The dissociation constant for binding to the free promoter site
                    is KT for TALE binding and KL for LacI binding. Similarly, when Tale is
                    bound to the promoter, the affinity of LacI with the corresponding operator
                    site will be reduced. The new dissociation constants will be k times bigger
                    and we assume that k is the same for both ligands since interference
                    between them will cause the same difficulty for each.
                </P><P>
                    The expression of the hill equation, as described for TALE, is transformed
                    as, the maximum transcription rate function for sfGFP, multiplied by the
                    competitive equation, in the presence of LacI and includes the
                    concentrations of the TALE repressor, LacI inhibitor and dissociation
                    constant KT, KL, kKT, kKL. Despite the fact, that LacI does not interfere
                    with the same operator site on the promoter, the interaction has a
                    competitive nature, because of the steric interactions. As a result, TALE
                    affinity to the DNA decreases.
                </P><P>
                    Τhe production rate of sfGFP expressed as
                </P><P>
                    where kKT &gt;&gt; KT because of reduced affinity.
                </P><P>
                    For the differential equations that were added to the model:
                    <UL><LI>
                            We did not include IPTG as although in high concentrations it can
                            reduce the concentration of LacI and smooth it out, it cannot
                            neutralize this phenomenon to some degree. We have studied the case
                            with only LacI regulating sfGFP expression.

                        </LI><LI>
                            Repressor TALE also represses the production of LacI with a similar
                            dissociation constant and cooperativity characteristics with the
                            sfGFP promoter.
                        </LI><LI>
                            The rates for LacI were found bibliographically and added to the
                            TALE model.
                        </LI></UL></P><H4>Parameters</H4><P>
                    TALE rates are characterized above as well as dissociation constant KT = kd
                    is equal to 106 nM.
                </P><DIV class="flex flex-center"><DIV style="overflow-x:auto;"><TABLE class="simple-table text-center"><TBODY><TR><TD><B>Parameters</B></TD><TD><B>Biological Meaning</B></TD><TD><B>Value</B></TD><TD><B>Units</B></TD><TD><B>Bibliography</B></TD></TR><TR><TD><B>aL</B></TD><TD>
                                Transcription of LacI
                            </TD><TD>                         0.23
                            </TD><TD>
                                nM/min
                            </TD><TD>
                                [11]
                            </TD></TR><TR><TD><B>bL</B></TD><TD>
                                Translation of Laci
                            </TD><TD>
                                0.128
                            </TD><TD>
                                1/min
                            </TD><TD>
                                [12]
                            </TD></TR><TR><TD><B>dmL</B></TD><TD>
                                Degradation of mRNA LacI
                            </TD><TD>
                                0.1824
                            </TD><TD>
                                1/min
                            </TD><TD>
                                [12]
                            </TD></TR><TR><TD><B>dL</B></TD><TD>
                                Degradation of LacI
                            </TD><TD>
                                0.0231
                            </TD><TD>
                                1/min
                            </TD><TD>
                                [13]
                            </TD></TR><TR><TD><B>KL </B></TD><TD>
                                dissociation constant LacI to Promoter
                            </TD><TD>
                                800
                            </TD><TD>
                                nM
                            </TD><TD>
                                [13]
                            </TD></TR><TR><TD><B>k</B></TD><TD>
                                constant
                            </TD><TD>
                                100
                            </TD><TD>
                                -
                            </TD><TD>
                                Est.
                            </TD></TR></TBODY></TABLE></DIV></DIV><P>
                    From Figure 9, the final values of the sfGFP concentration for each copy
                    number in the steady state, in the presence of LacI, are constantly
                    increasing and the repressor cannot hold the same fold change for different
                    copy number. Also, the deviation from the balancing of sfGFP shown in the
                    TALE system is two orders of magnitude larger and the control of its
                    quantity with LacI cannot satisfy the desired adjustment levels around the
                    steady state.
                </P><DIV class="text-center"><SMALL class="text-grey-6">
                        Figure 9: The blue line shows the sfGFP expression under regulation of
                        LacI tuning, while the red one shows the sfGFP without the regulation
                        effect of LacI.
                    </SMALL></DIV><P>
                    For a low copy number over time, it is observed that although the amount of
                    TALE produced is the same, it cannot repress with the same power as its
                    absence, since the affinity is kx times larger than natural binding.
                </P><DIV class="text-center"><SMALL class="text-grey-6">
                        Figure 10: The sfGFP expression over time for a constant, low copy
                        number. The red line shows the TALE system, while the blue one presents
                        the LacI regulated sfGFP. The system reaches a steady state and remains
                        stable at a low concentration of sfGFP.
                    </SMALL></DIV><DIV class="row justify-center q-mb-md q-mt-md" :class="rM('text-dark', 'text-light')"><SMALL class="row q-mb-sm">
                        [1] Segall-Shapiro, T., Sontag, E. and Voigt, C. (2018). Engineered
                        promoters enable constant gene expression at any copy number in
                        bacteria. Nature Biotechnology, 36(4), pp.352-358.
                    </SMALL><SMALL class="row q-mb-sm">
                        [2] Adler, M., Mayo, A. and Alon, U. (2014). Logarithmic and Power Law
                        Input-Output Relations in Sensory Systems with Fold-Change Detection.
                        PLoS Computational Biology, 10(8), p.e1003781.
 </SMALL><SMALL class="row q-mb-sm">
                        [3] Herman, J. and Usher, W. (2017) SALib: An open-source Python library
                        for sensitivity analysis. Journal of Open Source Software, 2(9).
                        doi:10.21105/joss.00097
                    </SMALL><SMALL class="row q-mb-sm">
                        [4] Zhang, X., Trame, M., Lesko, L. and Schmidt, S. (2015). Sobol
                        Sensitivity Analysis: A Tool to Guide the Development and Evaluation of
                        Systems Pharmacology Models. CPT: Pharmacometrics &amp; Systems
                        Pharmacology, 4(2), pp.69-79.
                    </SMALL><SMALL class="row q-mb-sm">
                        [5] Saltelli, A., Annoni, P., Azzini, I., Campolongo, F., Ratto, M. and
                        Tarantola, S. (2010). Variance based sensitivity analysis of model
                        output. Design and estimator for the total sensitivity index. Computer
                        Physics Communications, 181(2), pp.259-270.
                    </SMALL><SMALL class="row q-mb-sm">
                        [6] Espah Borujeni, A., Channarasappa, A. and Salis, H. (2013).
                        Translation rate is controlled by coupled trade-offs between site
                        accessibility, selective RNA unfolding and sliding at upstream standby
                        sites. Nucleic Acids Research, 42(4), pp.2646-2659.
                    </SMALL><SMALL class="row q-mb-sm">
                        [7] Bionumbers.hms.harvard.edu. (2018). Transcription and translation
                        rates - Bacteria Escherichia coli - BNID 109043. [online] Available at:
                        <A href="http://bionumbers.hms.harvard.edu/bionumber.aspx?id=109043&amp;ver=3&amp;trm=average%20transcription%20%20and%20translation&amp;org">Here</A></SMALL><SMALL class="row q-mb-sm">
                        [8]Bionumbers.hms.harvard.edu. (2018). Typical mRNA degradation time -
                        Bacteria Escherichia coli - BNID 108598. [online] Available at:
                        <A href="http://bionumbers.hms.harvard.edu/bionumber.aspx?id=108598&amp;ver=0&amp;trm=rna%20degradation&amp;org">Here</A></SMALL><SMALL class="row q-mb-sm">
                        [9]2016.igem.org. (2018). Team:ETH Zurich/Parameters - 2016.igem.org.
                        [online] Available at: <A href="https://2016.igem.org/Team:ETH_Zurich/Parameters">Here</A></SMALL><SMALL class="row q-mb-sm">
                        [10]Hu, C., Takahashi, M., Zhang, Y. and Lucks, J. (2018). Engineering
                        a Functional Small RNA Negative Autoregulation Network with
                        Model-Guided Design. ACS Synthetic Biology, 7(6), pp.1507-1518.
                    </SMALL><SMALL class="row q-mb-sm">
                        [11] Stamatakis, M. and Mantzaris, N. (2009). Comparison of
                        Deterministic and Stochastic Models of the lac Operon Genetic Network.
                        Biophysical Journal, 96(3), pp.887-906.

                    </SMALL><SMALL class="row q-mb-sm">
                        [12] Semsey, S., Jauffred, L., Csiszovszki, Z., Erdossy, J., Steger,
                        V., Hansen, S. and Krishna, S. (2013). The effect of LacI
                        autoregulation on the performance of the lactose utilization system in
                        Escherichia coli. Nucleic Acids Research, 41(13), pp.6381-6390.
                    </SMALL><SMALL class="row q-mb-sm">
                        [13] 2015.igem.org. (2018). Team:ETH Zurich/Modeling- 2015.igem.org.
                        [online] Available at: <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Here</A></SMALL></DIV></DIV></DIV></Q-PAGE></IGEM-LAYOUT></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>