<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Thessaloniki_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Thessaloniki/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="q-app" style="margin-top: 16px"><IGEM-LAYOUT><Q-PAGE class="text-justify"><Q-PARALLAX src="https://static.igem.org/mediawiki/2018/0/04/T--Thessaloniki--experiments-2-vector.jpg"><H1 slot="loading">Project Description</H1><H1>Project Description</H1></Q-PARALLAX><DIV class="row justify-center gutter-md" :class="rM('text-dark', 'text-light')"><DIV class="col-xs-12 col-sm-11 col-md-10 col-lg-8"><H4>1.Motivation</H4><P>
                    Since the very first days of our brainstorming, our primary objective was
                    to create something that could have an immediate impact on the scientific
                    community. Among other things, the unpredictability that characterizes
                    biological systems immediately drew our attention. Even though Synthetic
                    Biology has come a long way in designing orthogonal and predictable
                    systems, there are still many interfering factors, obstructing consistency
                    and functionality.
                    One such factor is gene/plasmid copy number. For that reason we decided to
                    create a tool that enhances stability and robustness of synthetic
                    biological systems. We achieve that by engineering stabilized promoters
                    which produce the same output regardless of their biological context in
                    order to render synthetic biological systems more predictable and reliable.

                </P></DIV></DIV><DIV class="row justify-center gutter-md" :class="rM('text-dark', 'text-light')"><DIV class="col-xs-12 col-sm-11 col-md-10 col-lg-8"><H4>2.Why is Copy Number independence useful?</H4><P>
                    Despite the fact that copy number is, usually, regarded as being constant,
                    many factors related with the internal and external environment of the cell
                    contribute to copy number variation both over time and between different
                    cells. [2]
                </P><P>
                    Gene Expression is characterised by a positive correlation with the plasmid
                    copy number.
                    Thus, variations in copy number lead to variations in gene expression,
                    affecting the function of the system. Particularly, in complex systems that
                    require high precision, such as metabolic pathways and multi-input logic
                    gates, accumulation of perturbations caused by variations in their
                    individual parts greatly affect the functionality of said system.
                </P><P>
                    Regulating the expression strength is usually achieved via the replacement
                    of regulatory elements such as RBSs and Promoters. Still, this does not
                    ensure a predictable behaviour over different genetic contexts, therefore
                    re-tuning is mandatory to maintain the system’s proper function.
                </P></DIV></DIV><DIV class="row justify-center q-mt-lg" :class="rM('text-dark', 'text-light')"><DIV class="col-xs-12 col-sm-11 col-md-10 col-lg-8"><H4>3.GALENE</H4><P>
                    This year we built upon the work of Shapiro et. al., [1] implementing an
                    incoherent FeedForward Loop network motif that consists of a regulatory
                    element which compensates for changes in copy number, thus stabilizing the
                    promoter. We provide Galene to the iGEM Community, a genetic toolbox that
                    enhances the stability of biological systems by disassociating
                    transcription from gene copy number and provides translational control of
                    protein production.
                </P><P><B>Stabilized promoters</B> are promoter sequences that achieve expression
                    independent of gene/plasmid copy number. They possess great potential for
                    usage as a synthetic biology tool with many applications, ranging from part
                    characterization to metabolic pathway and genetic circuitry fine-tuning [].
                    Balancing gene expression in synthetic biological systems is a key
                    parameter for their proper function. By achieving gene/plasmid copy number
                    independence, such systems become more robust, predictable and less prone
                    to perturbations arising from both their internal and external environment.
                </P><P>
                    Through <A href="">model driven feedback</A>, we accomplish promoter
                    stabilization
                    using 2 distinct systems, each with its own advantages: <B><U>TAL Effectors</U>
                        and
                        <U>CRISPR interference</U></B>.
                    We have, also, <U>designed</U> and <U>simulated</U> a 3rd stabilization
                    system using
                    <B>Attenuator-interacting sRNA regulators</B>.
                </P><P>
                    Wanting to expand on the stabilized promoters system, we designed a
                    theophylline Riboswitch that would allow, on top of the stabilization of a
                    promoter, its induction and activation to the desired expression level.
                    <UL><LI>inducibility</LI></UL></P></DIV></DIV><DIV class="row justify-center q-mt-lg" :class="rM('text-dark', 'text-light')"><DIV class="col-xs-12 col-sm-11 col-md-10 col-lg-8"><H4>4.Possible Applications of Stabilized Promoters</H4><H5>Genome Insertion</H5><P>
                    The output of genetic circuits inserted into the genome is affected both by
                    the number of inserted copies and their distance from the Origin of
                    Replication (ORI). [3] Stabilized promoter systems can be transferred to
                    genome, maintaining the circuit’s level of expression, independent of the
                    position of the inserted gene relative with the origin of replication.
                </P><H5>Medicine</H5><P>
                    Precise control of genetic systems is of utmost importance when they are
                    applied for therapeutic purposes. [4] Stabilized promoters can be used to
                    introduce predictability and increase specificity in contemporary
                    therapeutic methods, such as gene and cancer therapies, which are based
                    upon the implementation of sophisticated synthetic biological circuits.
                </P><H5>Toxic Protein Production</H5><P>
                    Production of highly toxicity proteins requires tight control of gene
                    expression, so that sufficient expression can be achieved before host cells
                    become dysfunctional. [5] Stabilized promoters can be used for sufficient
                    expression before toxicity in the host cells comes into play.
                </P><H5>Metabolic Engineering</H5><P>
                    The enzymes participating in metabolic pathways must be expressed in
                    certain stoichiometric ratios, as variability in their expression could
                    disrupt the system’s proper function. [6] Stabilized promoters can be
                    utilized as regulatory tools for the expression of genes participating in
                    metabolic pathways, increasing the system’s robustness and performance.
                </P><H5>Large Scale Production</H5><P>
                    Genetic Systems used for industrial production require high fidelity to
                    cope with the varying conditions inside a bioreactor [7] that might
                    influence plasmid’s copy number. On those lines, stabilizing the expression
                    of genes crucial in the production process, has the potential to increase
                    the yield. Copy Number variation can influence the expression of
                    <B>fermentation</B>-related genes in organisms used for industrial
                    production,
                    such as Saccharomyces cerevisiae. [8] Stabilized promoters can be a
                    valuable tool for the control of genes related the process of fermentation
                    and, therefore, improve the quality of the final product.
                </P></DIV></DIV><DIV class="row justify-center" :class="rM('text-dark', 'text-light')"><DIV class="col-md-9"><H6>REFERENCES</H6><DIV class="row justify-center q-mb-md q-mt-md" :class="rM('text-dark', 'text-light')"><SMALL class="row q-mb-sm">
                        [1] Segall-Shapiro, T. H., Sontag, E. D., &amp; Voigt, C. A. (2018).
                        Engineered promoters enable constant gene expression at any copy number
                        in bacteria. Nature Biotechnology. <A href="https://doi.org/10.1038/nbt.4111">https://doi.org/10.1038/nbt.4111</A></SMALL><SMALL class="row q-mb-sm">
                        [2] Wong Ng, J., Chatenay, D., Robert, J., &amp; Poirier, M. G. (2010).
                        Plasmid copy number noise in monoclonal populations of bacteria.
                        Physical Review E - Statistical, Nonlinear, and Soft Matter Physics.
                        <A href="https://doi.org/10.1103/PhysRevE.81.011909">https://doi.org/10.1103/PhysRevE.81.011909</A></SMALL><SMALL class="row q-mb-sm">
                        [3] Block, D. H. S., Hussein, R., Liang, L. W., &amp; Lim, H. N. (2012).
                        Regulatory consequences of gene translocation in bacteria. Nucleic
                        Acids Research. <A href="https://doi.org/10.1093/nar/gks694">https://doi.org/10.1093/nar/gks694</A><A href="https://www.sciencedirect.com/science/article/pii/S221112471300315X?via%3Dihub">sciencedirect</A></SMALL><SMALL class="row q-mb-sm">
                        [4] Arkin, A. P. (2013). A wise consistency: Engineering biology for
                        conformity, reliability, predictability. Current Opinion in Chemical
                        Biology. <A href="https://doi.org/10.1016/j.cbpa.2013.09.012">https://doi.org/10.1016/j.cbpa.2013.09.012</A></SMALL><SMALL class="row q-mb-sm">
                        [5] Saida, F., Uzan, M., Odaert, B., &amp; Bontems, F. (2006). Expression
                        of Highly Toxic Genes in E. coli: Special Strategies and Genetic Tools.
                        Current Protein and Peptide Science.
                        <A href="https://doi.org/10.2174/138920306775474095">https://doi.org/10.2174/138920306775474095</A></SMALL><SMALL class="row q-mb-sm">
                        [6] Keasling, J. D. (2012). Synthetic biology and the development of
                        tools for metabolic engineering. Metabolic Engineering.
                        <A href="https://doi.org/10.1016/j.ymben.2012.01.004">https://doi.org/10.1016/j.ymben.2012.01.004</A></SMALL><SMALL class="row q-mb-sm">
                        [7] Moser, F., Broers, N. J., Hartmans, S., Tamsir, A., Kerkman, R.,
                        Roubos, J. A., … Voigt, C. A. (2012). Genetic circuit performance under
                        conditions relevant for industrial bioreactors. ACS Synthetic Biology.
                        <A href="https://doi.org/10.1021/sb3000832">https://doi.org/10.1021/sb3000832</A></SMALL><SMALL class="row q-mb-sm">
                        [8] Steenwyk, J., &amp; Rokas, A. (2017). Extensive Copy Number Variation
                        in Fermentation-Related Genes Among Saccharomyces cerevisiae Wine
                        Strains. G3 (Bethesda, Md.). <A href="https://doi.org/10.1534/g3.117.040105">https://doi.org/10.1534/g3.117.040105</A></SMALL></DIV></DIV></DIV></Q-PAGE></IGEM-LAYOUT></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>