Team:Thessaloniki/Experiments
Protocols
Protocols
Frozen Competent Cells
Materials
Plate of cells to be made competent
LB Broth
CaCl2 solution 0,1M
100% glycerol
Ice
Method
Inoculate 1 colony from a freshly grown plate into 50ml broth
Incubate cells overnight at 37°C in a shaking incubator                                 (200-250 rpm)
Transfer cells into 500 ml LB Broth in a 500ml E. flask
Incubate at 250-300 rpm and 37°C for approx. 2,5 hours (or OD = 0.5). Centrifuge at 5000 rpm and 4°C for 15 min
Remove supernatant and resuspend pelleted cells in 250 ml 0,1 M                                 CaCl2 solution
Incubate on ice for 20 min
Centrifuge at 5000 rpm for 15 min
Remove supernatant and resuspend pelleted cells in 43 ml 0,1 M                                 CaCl2 solution
Incubate on ice for 2-12 hours
Note: Maximum Incubation at 24 hours. Efficiency of                                     Transformation is gradually reduced afterwards
Add 7ml 100% glycerol, mix well
Aliquot cells into prechilled 15ml tubes
Quick freeze in liquid nitrogen
Store (-80°C)
Fresh Competent Cells
Materials
Plate of cells to be made competent
LB Broth
CaCl2 solution 0,1M
Ice
Method
Inoculate 1 colony from a freshly grown plate into 50ml LB                                 Broth in a 500ml E. Flask
Incubate at 250-300 rpm and 37°C for approx. 3 hours (or                                 until OD600 is about 0,375)
Note: Transformation Effeciency will decrease if                                     cells are allowed to grow to a density greater than 0,4 OD
Centrifuge 3000 rpm and 4°C for 7 minutes. No brake.
Remove supernatant and drain the tubes by keeping them inverted                                 on an absorbent paper for 1 minute
Resuspend pelleted cells in 10ml ice-cold CaCl2 solution by                                 pipetting gently up and down. Make sure that cells are fully                                 resuspended. Keep on ice for 5 min
Centrifuge 3000 rpm and 4°C for 7 minutes. No brake.
Remove supernatant and drain the tubes by keeping them inverted                                 on an absorbent paper for 1 minute
Resuspend pelleted cells in 2ml ice-cold CaCl2 solution by                                 pipetting gently up and down. Make sure that cells are fully                                 resuspended. Keep on ice.
Note: Increased transformation efficiency next 24h,                                     decrease after the 24 hour mark.
Chemical Transformation
Materials
Chemically competent cells
DNA to be transformed
LB agar with antibiotics
Ice
Method
Thaw 100 μl of competent cells on ice.
Note: Thawing at room temp decreases efficiency
Add DNA:
For transformation of ligated products add 8μl of the                                         ligation reaction mix
For transformation of known plasmids add approx. 10-100                                         μg DNA
Incubate cells on ice for 30min.
Note: Cutting this incub time short slightly                                     decreases transformation efficiency.
Heat shock cells at 42°C for 90 sec.
Note: Optimal heat shock duration depends on the                                     strain and the method used to make it competent.
Incubate on ice for 5 minutes
Heat shock cells at 42°C for 90 sec
Incubate on ice for 5 minutes
Add 1ml of SOC or LB Broth.
Note: Using SOC increases transformation efficiency.
Incubate cells for 1 hour at 37°C in a shaking incubator                                 (200-250rpm).
Note: Incubation without shaking will decrease                                     transformation efficiency
Note: If using LB Broth, longer incubation up to 2                                     hours can increase transformation Efficiency
Centrifuge at 8000rpm for 3 minutes
Remove 920μl of supernatant
Resuspend pelleted cells in the remaining LB broth or SOC.
Plate resuspended cells onto LB agar plates containing the                                 appropriate antibiotic.
Let the plates dry in the incubator for approx. 30min.
Flip upside down and incubate for 16-18 hours.
Growing Overnight Cultures
Materials
5-10 ml LB broth
5-10 μl antibiotic
culture tubes
Method
Overnight cultures are prepared under sterile conditions using a                             Bunsen burner
Add 5-10 ml ofLB media into a culture tubes.
Add 5-10 μl of appropriate antibiotic into the broth.
Using a loop, pick a single colony and inoculate the culture by                                 dipping the loop into the LB broth.
Incubate overnight at 37°C shaking at 200-250 rpm.
LB Broth
Materials
20 g LB broth (powder)
1 Litre Purified Water
Method
Add 20g LB broth to 1 litre purified water
Autoclave
LB Agar
Materials
35 g LB Agar (powder)
1 Litre Purified Water
Method
Add 37g LB Agar to 1 litre purified water
Autoclave
SOC Medium
We followed the Quiagen                                 protocol
Materials
0.5% Yeast Extract
2% Tryptone
10 mM NaCl
2.5 mM KCl
10 mM MgCl2
10 mM MgSO4
20 mM Glucose*
Notes
Add Glucose after autoclaving the solution with the remaining                                 ingredients, and letting it cool down. Sterilize the final                                 solution by passing it through a 0.2 µm filter.
SOC medium can be stored at room temperature and is stable for                                 several years.
Glycerol Stocks
Materials
500µl glycerol (50%)
500µl overnight culture in LB
Method
Add 500µl glycerol (80%) to 1.5ml eppendorf tube
Add 500µl overnight culture in LB
Store at -80°C
PCR From Plasmid DNA Template
We followed the NEB PCR with hiFi Q5 master mix Protocol for our PCR reactions.
Materials
2x Q5 High-Fidelity Master Mix
10μΜ Forward Primer
10μΜ Reverse Primer
Template DNA
Nuclease Free Water
For a 20 µL reaction:
Q5 High-Fidelity 2X Master Mix
10 μl
10 µM Forward Primer
1 μl
10 µM Reverse Primer
1 μl
Template DNA
variable (1-5 ng)
Nuclease-Free Water
to 20 μl
Note: Reactions are set on ice
Thermocycling
Initial Denaturation
95°C
2 minutes
25-35 cycles
95°C (Denaturation)
0.5-1 minute
42-65°C (Annealing)
0.5-1 minute
72°C (Extension)
1 min/kb
Final Extension
72°C
5 minutes
Hold
4–10°C
Indefinite
Note: We used the NEB                                 ™ Calculator to determine the annealing temperature.
Colony PCR
We followed the Promega GoTaq® DNA Polymeras Protocol for our Colony PCR reactions.
Materials
5X Green or Colorless GoTaq® Reaction Buffer1
GoTaq® DNA Polymerase
PCR Nucleotide Mix
Forward Primer
Reverse Primer
Template DNA
Nuclease Free Water
For a 50 μl PCR Reaction:
5X Green or Colorless GoTaq® Reaction Buffer1
10 μl
PCR Nucleotide Mix, 10mM each
1 μl
Forward Primer
1 μl
Reverse Primer
1 μl
GoTaq® DNA Polymerase (5u/µl)
0,25 μl
template DNA
variable (1-5 ng)
Nuclease-Free Water
to 50 μl
Method
We performed 10μl Colony PCR Reactions
Thermocycling
Initial Denaturation
98°C
30 seconds
25-30 cycles
98°C (Denaturation)
10 seconds
50–72°C (Annealing)
30 seconds
72°C (Extension)
30 seconds/kb
Final Extension
72°C
2 minutes
Hold
4–10°C
Indefinite
Notes: Perform PCR using your standard parameters
Reactions are set on ice
Reactions are placed in a thermal cycler that has been                                 preheated to 95°C
The thermal cycling protocol has an initial denaturation step where samples are heated at 95°C for 2 minutes to ensure that                                 the target
Agarose Gel Electrophoresis
Equipment
Casting tray
Well combs
Voltage source
Gel box
UV light source
Reagents
TAE
Agarose
Ethidium bromide (stock concentration of 10 mg/mL)
Method
For a standard 1% agarose gel
Measure 1 g of agarose and mix agarose powder with 100 mL 1xTAE                                 in a microwavable flask
Microwave for 1-3 min until the agarose is completely dissolved                                 and let agarose solution cool down to about 50 °C
Add ethidium bromide (EtBr) to a final concentration of                                 approximately 0.2-0.5 μg/mL. EtBr binds to the DNA and allows                                 you to visualize the DNA under ultraviolet (UV) light
Pour the agarose into a gel tray with the well comb in place
Place newly poured gel at 4 °C for 10-15 mins OR let sit at                                 room temperature for 20-30 mins, until it has completely solidified
Add loading buffer to each of your DNA samples
Once solidified, place the agarose gel into the gel box                                 (electrophoresis unit). Fill gel box with 1xTAE (or TBE) until                                 the gel is covered
Carefully load a molecular weight ladder into the first lane of                                 the gel and your samples into the additional wells of the gel
Run the gel at 80-150 V until the dye line is approximately                                 75-80% of the way down the gel. A typical run time is about 40                                 minutes-1.5 hours, depending on the gel concentration and                                 voltage
Turn OFF power, disconnect the electrodes from the power                                 source, and then carefully remove the gel from the gel box
Using any device that has UV light, visualize your DNA                                 fragments. The fragments of DNA are usually referred to as                                 ‘bands’ due to their appearance on the gel
Restriction Digestion
Restriction Enzyme: NEB                                     enzyme finder used to determine the appropriate                                 restriction enzymes
Reaction buffer: NEB                                     double digest finder used to determine an appropriate                                 buffer and protocol for the restriction enzymes used
Ligation
We followed the NEB Ligation Protocol with T4 DNA Ligase.
Materials
Microcentrifuge Tubes
Ice
T4 DNA Ligase Buffer (10x)
T4 DNA Ligase
Vector DNA
Insert DNA
Nuclease Free Water
Method
Set up the following reaction in a microcentrifuge tube on ice
Component
20 μl Reaction
T4 DNA Ligase Buffer (10x)
2 μl
T4 DNA Ligase
1 μl
Vector DNA
50 ng
Insert DNA
ng calculated using the NeBioCalculator
Nuclease Free Water
To 20 μl
For cohesive (sticky) ends, incubate at 16°C overnight or room                                 temperature for 10 minutes. For blunt ends or single base                                 overhangs, incubate at 16°C overnight or room temperature for 2                                 hours
Heat inactivate at 65°C for 10 minutes
Chill on ice and transform 8μl of the reaction into 100 μl                                 competent cells
Notes:
T4 DNA Ligase should be added last
Ligation using a molar ratio of 1:3 vector to insert                                     increases the reaction’s efficiency
The T4 DNA Ligase Buffer should be thawed and resuspended                                     at room temperature
Golden Gate Assembly
We followed the Golden Gate Assembly Protocol for Using NEB Golden Gate Assembly Mix
Materials
Destination Plasmid
Insert/Inserts
NEB Golden Gate Buffer (10x)
NEB Golden Gate Assembly Mix
Nuclease-free H2O
Method
Set up a 20 μl reaction according to the following table
Reagent
Assembly Reaction
Destination Plasmid 75ng/μl
1 μl
Insert:
If Precloned
If in amplicon Form
75-100 ng each plasmid                                             2:1 molar ratio insert:vector NEB Golden Gate Buffer (10x) 2 μl NEB Golden Gate Assembly Mix 1 μl Nuclease-free H2O to 20 μl
Assembly Protocol
Number Of Inserts
Assembly Protocol
1-4 inserts
37°C for 60 minutes → 55°C for 5 minutes
5-10 inserts
(37°C for 1 minute → 16°C for 1 minute)x30 → 55°C for 5                                     minutes
For our plasmid isolation we used the Macherey Νagel NucleoSpin®                             Plasmid isolation Kit, unless otherwise stated.                             For PCR Purification and Gel Extraction we used the Macherey Nagel                             NucleoSpin® Gel and PCR Clean-up Kit, unless otherwise stated.
dCas9 Toxicity Measurements
All measurements shown regarding Abs600 were taken using plate                             reader.
Day 1
Inoculate one single colony of DH5a E.coli cells containing the                             dcas9 cassette into 1 ml LB+Kanamycin in
2‐ml eppendorf tubes
and                             grown at 37 °C, 250 rpm overnight in a MultitronPro shaker                             incubator.
Day 2
The overnight cultures were diluted 1:360 into 1ml LB+ kanamycin in 2‐ml eppendorf tubes and doxycycline was added to the following                             final concentration: 0, 0.2, 0.4, 0.6, 0.8, 1.2, 4, 8.The cultures                             were grown at 37 °C, 250 rpm in a MultitronPro shaker incubator. After 6 hours measurements for Abs600 were performed using plate                             reader
Flow Cytometry Sample Preparation
We followed the protocol from Adam Mayer et al [1].
All measurements shown were taken by cytometry of cells in mid-log                             growth except when noted.
Day1
Glycerol stocks of strains containing the plasmids of interest were                             streaked on LB + 1.5% Agar plates and grown overnight at 37 °C
Day2
Single colonies were inoculated into 1 ml LB + antibiotics in 2‐ml                                 eppendorf tubes and grown at 37 °C, 250 rpm overnight in a MultitronPro shaker incubator
Day3
The overnight growths were diluted 1:200 into 1 ml LB + antibiotics                             in 2‐ml eppendorf tubes and grown at 37 °C, 250 rpm.
After 2 hours the growths were diluted 1:500 into prewarmed LB +                             antibiotics + inducer where necessary in 2‐ml eppendorf tubes and grown at 37 °C, 250 rpm for 5 hours.
After growth, 20 μl of culture sample was diluted into 180 μl PBS +                             200 μg/ml kanamycin to inhibit translation. The samples were stored                             at 4°C for 1 hour and then measurements were performed using the                             CyFlow Cube8 Sysmex Partec Flow Cytometer.
[1] Meyer, A. J., Segall-Shapiro, T. H., & Voigt, C. A. (2018).                             Marionette: E. coli containing 12 highly-optimized small molecule sensors. BioRxiv. Retrieved from                             http://biorxiv.org/content/early/2018/04/10/285866.abstract
Plate Reader Sample Preparation for our Interlaboratory                             Collaboration
We followed the iGEM 2018 InterLab Study Protocol
Primers
Primers
Primers Name
Sequence
Usage
VR
attaccgcctttgagtgagc
Amplification of IDT gBlocks
Colony PCR
VF2
tgccacctgacgtctaagaa
Amplification of IDT gBlocks
Colony PCR
E Vector
belongs to the Metabrick Platform
Vector Amplification for insertion of  BsaI sites in the Prefix and Suffix
P Vector
belongs to the Metabrick Platform
Vector Amplification for insertion of  BsaI sites in the Prefix and Suffix
E Primer
belongs to the Metabrick Platform
Part Amplification for insertion of BsaI sites in the Prefix
P Primer
belongs to the Metabrick Platform
Part Amplification for insertion of BsaI sites in the Suffix
X Primer
belongs to the Metabrick Platform
Part Amplification for insertion of BsaI sites in the Suffix
S Primer
belongs to the Metabrick Platform
Part Amplification for insertion of BsaI sites in the Prefix
P.Rem 1
aGGTCTCcgcagcaccctgctgcccttg
Removal of PstI illegal Site from the Addgene Plasmids pTHSSe_59 & pTHSSe_60 (Insertion Point 50)
P.Rem 4
aGGTCTCcgtgtctcgacataccaaattgagtcatgg
Removal of PstI illegal Site from the Addgene Plasmids pTHSSe_59 & pTHSSe_60 (Insertion Point 50)
P.Rem 2
GGTCTCgctgcgataaggctacgatgtgg
Removal of PstI illegal Site from the Addgene Plasmids pTHSSe_59 & pTHSSe_60  (Insertion Point 802)
P.Rem 3
GGTCTCgacacggactgctgcaactctttcgtag
Removal of PstI illegal Site from the Addgene Plasmids pTHSSe_59 & pTHSSe_60 (Insertion Point 802)
In 1
AGGTCTCAGGAActtcagccaaaaaacttaagacc
Amplification of Test Device 1 and Test Device 2 from pTHSSe_59 & pTHSSe_60, used for our Interlaboratory Study
In 2
AGGTCTCAACTgacatagcgtataaacgcaga
Amplification of Test Device 1 and Test Device 2 from pTHSSe_59 & pTHSSe_60, used for our Interlaboratory Study
Ri 1
AGGTCTCATTCCTGGCTGAAACCTATTATGACG
Amplification of pTHSSe_59 & pTHSSe_60 vectors for our TALEsp1 and TALEsp2 stabilised Riboswitch Constructs.
Ri 2
AGGTCTCACAGTCGCTATGTCTTCTACTAGTAGC
Amplification of pTHSSe_59 & pTHSSe_60 vectors for our TALEsp1 and TALEsp2 stabilised Riboswitch Constructs.
DC1
AGGTCTCAGGAAgctttactccaccgttgg
Amplification of pAN-PTet-dCas9 plasmid to make the insert RFC10 compatible
DC2
AGGTCTCAACTGcctcagataaaatatttgctcatgagc
Amplification of pAN-PTet-dCas9 plasmid to make the insert RFC10 compatible
Vi1
AGGTCTCATTCCGCTCCGTGTCTTCCTC
Amplification of pTHSSe_59 & pTHSSe_60 vectors for insertion of a BsaI site in the prefix
AG1
AGGTCTCAATGTCCAGATTAGATAAAAGTAAAGTG
Amplification of the ANDgate Plasmid to replace the J23114 Promoter with a Talesp1 psp1w1 promoter for the TetR expression
AG2
AGGTCTCATTCCGGCTGAACTCTAGAAGC
Amplification of the ANDgate Plasmid to replace the J23114 Promoter with a Talesp1 psp1w1 promoter for the TetR expression
YG1
aggtctcaggaaGAAGACACGGAGCTCG
Amplification of pTHSSe_59 psp1w1  to replace the J23114 Promoter with a Talesp1 psp1w1 promoter for the TetR expression
YG2 (30)
aggtctcaACATATGGTATTTCTCCTCtttAAtttaaaca
Amplification of pTHSSe_59 to replace the J23114 Promoter with a Talesp1 psp1w1 promoter for the TetR expression
YG2 (32)
aggtctcaACATCTAGTACTTTCCTGTGTGA
Amplification of pTHSSe_59 to replace the J23114 Promoter with a Talesp1 psp1w1 promoter for the TetR expression
IA1
aggtctcactttTTAAACAAAATTATTTGTAGAGGCTGTTTCGtc
Amplification of pTHSSe_59 psp1w1 vector  for insertion of  BsaI sites in the Prefix
IA2
aggtctcaTACTAGAGCCAGGCATCAAATAAAACG
Amplification of pTHSSe_59 psp1w1 vector  for insertion of  BsaI sites in the Suffix
Amplification of pSB1C3 and pTHSSe_59 for the Insertion of constitutively expressed sgRNA cassette.
IG1
aggtctcatcaaGGCTGTAAGAATCCTATAGGTTGAG
Amplification of pSB1C3 and pTHSSe_59 for the Insertion of constitutively expressed sgRNA cassette.
GG1
aggtctcattgacagctagctcagtcctGTTTTAGAG
Amplification of pRha sgRNA cassette in order to replace pRha with psp1w1
GG2
aggtctcaagtataTTATTTGTAGAGCTCATCCATGCCATG
Amplification of pRha sgRNA cassette in order to replace pRha with psp1w1
GK1
aggtctctaaagaggagaaatactagATGAGCAAAG
Amplification of B0034-sfGFP for the incorporation of BsaI restriction sites.
GK2
aggtctcaagtaTTATTTGTAGAGCTCATCCATGC
Amplification of B0034-sfGFP for the incorporation of BsaI restriction sites.
Clonning Strategy
Cloning Strategy
TALE stabilized Promoters
TALEsp1 / TALEsp2 - Stabilized Promoters
TALEsp1-Pupsp1 stabilized promoter /TALEsp2-Pupsp2 stabilized                                 promoter
TALEsp1 / TALEsp2 - Stabilized Promoters
Steps
PCR amplification of psb1c3 vector with a set of                                     standardized primers.
These primers belong to the Metabrick Platform [1] and                                         incorporate BsaI restriction sites to the                                         biobrick                                         prefix and suffix.
PCR amplification of pTHSSe_59 and pTHSSe_60 plasmid with E,P standardized primers.
These primers belong to the Metabrick platform and                                         incorporate BsaI restriction sites to the                                         plasmid’s                                         biobrick prefix and suffix.
Golden Gate assembly of the PCR amplified products.
The amplified products after restriction digestion with                                         bsaI restriction enzyme, acquire complementary sticky                                         ends                                         and can be ligated together forming the TALEsp1-Pupsp1-pSB1C3, TALEsp2-Pupsp2-pSB1C3 plasmids.
Restriction digestion with EcoRI and PstI,                                     enzymes that are unique to prefix and suffix.
Since the assembled Talesp1-Pupsp1 / Talesp2-Pupsp2 constructs are RFC[10] compatible, they can be                                         easily assembled with other vectors. This can be                                         achieved                                         by cutting the Talesp1-Pupsp1-pSB1C3 vector and                                         the                                         receiver vectors with two biobrick enzymes one of which                                         having it’s restriction site to prefix and the other to                                         suffix.
Ligation between the digested pieces.
Psp1w1 promoter characterization device
The Psp1w1 promoter characterization device emerged from the                                 assembly of three individual parts: Psp1w1, sfGFP                                     (without                                     starting codon) and amplified pSB1C3 vector.
Steps
PCR amplification of pSB1C3 vector with a set of                                     standardized primers.
Design and synthesis of the Psp1w1 and sfGFP                                         (without                                         starting codon) with the appropriate flanking                                     regions.
Golden Gate assembly between Psp1w1, sfGFP                                         (without starting codon) and the amplified                                         pSB1C3 vector.
This reaction forms the final plasmid Psp1w1-pSB1C3 containing the Psp1w1 promoter characterization device.
Talesp1-Psp1w1 stabilized promoter
The following procedure has been followed in order to replace                                 the Pupsp1 stabilized promoter with the Psp1w1 stabilized                                 promoter .The expression levels of the sfGFP driven by the                                 Psp1w1 promoter are lower than the sfGFP expression levels                                 driven by the Pupsp1 promoter.
Steps
PCR amplification of Psp1w1-pSB1C3 plasmid with In1, In2 primers.
These primers amplify the Psp1w1 promoter                                         characterization device and incorporate BsaI sites flanking the amplified device.
PCR amplification of TALEsp1-pSB1C3 vector with Ri1, Ri2 primers.
These primers amplify the TALEsp1 cassette together with                                         the plasmid backbone and incorporate BsaI restriction                                         sites at the external regions of the amplified                                         sequence.
Restriction digestion with BsaI restriction enzyme of                                         the previously amplified sequences creates sticky                                         complementary ends and thus they can be assembled                                         together.
Restriction digestion with BsaI restriction enzyme of the                                     previously amplified sequences creates sticky complementary                                     ends and thus they can be assembled together.
This reaction forms the final plasmid TALEsp1-Psp1w1-pSB1C3 plasmid containing the Psp1w1 TALE stabilized promoter.
Riboswitches
The final plasmids for the characterization of the theophylline                                 riboswitches consist of three individual parts: A(Pupsp1 promoter fused with riboswitch), B(sfGFP marker)                                     and C. Each part’s sequence is flanked by BsaI recognition sites and                                 after digestion with BsaI restriction enzyme, part A has sticky                                 ends with part B and part C, while part B has sticky ends with                                 part A and part C.
Steps
PCR amplification of psb1c3 vector with a set of                                     standardized primers.
These primers amplify the plasmid backbone without the                                         insert, incorporate BsaI sites at prefix and suffix,                                         forming the amplified pSB1C3 vector (part C).
Design and synthesis of the part A, part B fragments with the appropriate flanking regions.
Golden Gate assembly between part A, part B and the amplified pSB1C3 Vector (part C). This reaction                                     forms the                                     final plasmids Ribo12.1-pSB1C3 and RiboT27-pSB1C3 containing the riboswitch characterization device.
TALEsp1 stabilized promoters with translational control.
Steps
PCR amplification of RiboT27-pSB1C3 plasmid with In1, In2 primers.
These primers amplify the riboswitch characterization                                         device and incorporate BsaI sites at the                                         external                                         regions of the amplified device.
PCR amplification of pTHSSe_59 plasmid with Ri1, Ri2 primers.
These primers amplify the TALEsp1 cassette together                                         with the plasmid backbone and incorporate BsaI restriction sites flanking the amplified sequence.
Restriction digestion with BsaI restriction                                         enzyme of the previously amplified sequences, creates                                         sticky complementary ends and thus they can be                                         assembled together.
Golden Gate assembly reaction between the amplified                                     products from step 1 and step 2.
This reaction forms the TALEsp1-Pupsp1-RT27-pSB1C3 plasmid containing the theophylline inducible TALE                                         stabilized promoter.
CRISPRi system
Constructs used to test dcas9/sgrna repression.
pRha-sgRNA Cassette
pRha-sgRNA Cassette test construct
Steps
PCR amplification of pSB1C3 vector and pTHSSe_59 plasmid with a set of standardized primers.
These primers belong to the Metabrick                                         Platform,                                         amplify the plasmid backbones without the                                         insert and incorporate BsaI sites at prefix and                                         suffix,                                         forming the amplified pSB1C3 and amplified                                             pTHSSe_59 vectors.
Design and synthesis of part 1 (sgRNA construct) and                                     part 2 (sfGFP construct).
Each part’s sequence is flanked by BsaI recognition                                         sites and after digestion with BsaI restriction enzyme,                                         part 1 has sticky ends with part 2 and the amplified                                             pSB1C3 or amplified pTHSSe_59 vector,                                         while part 2 has                                         sticky ends with part 1 and the amplified vectors.
Golden gate Assembly between part 1, part2 and one of the amplified pSB1C3 or amplified pTHSSe_59 vector.
Depending on the amplified vector we chose, this                                         reaction leads to the formation of the sgRNA-sfGFP-pSB1C3 or the sgRNA-sfGFP-pTHSSe_59 plasmid.
We transformed DH5a E.coli competent cells with sgRNA-sfGFP-pTHSSe_59 vector along with dCas9-pSB3K3 vector in order to test the response function for the                                         repression of the J23104 promoter by the                                         sgRNA/dCas9                                         complex. This procedure is thoroughly described at the                                         characterization page.
CRISPRi stabilized promoter
After testing the response function of the CRISPRi system we                                 replaced the rhamnose inducible promoter that is located                                 upstream of the sgRNA cassette with the constitutive Psp1w1                                 promoter.
Steps
Digestion with biobrick enzymes of the Psp1w1 promoter                                     characterization device and ligation with digested pTHSSe_59 vector.
Since the Psp1w1 promoter characterization device is                                         compatible with Biobrick RFC[10], it can be                                         easily                                         assembled with other RFC[10] compatible vectors.                                         This                                         requires digestion of the characterization device and                                         the receiver vector with two enzymes having their                                         restriction sites on the prefix and suffix, followed by                                         ligation.
These reactions forms the final plasmid psp1w1-pTHSSe_59 containing the Psp1w1 promoter characterization device.
PCR amplification of the psp1w1-pTHSSe_59 plasmid                                     with IG1, IA2 primers.
These primers amplify the psp1w1 promoter sequence                                         together with the pTHSSe_59 plasmid backbone and                                         incorporate BsaI restriction sites flanking the                                         amplified sequence.
PCR amplification of the sgRNA-sfGFP-1c3 plasmid                                     with GG1, GG2 primers.
These primers amplify the sgrna cassette along with the                                         stabilized J23104 promoter cassette and                                         incorporate BsaI restriction sites at the external regions                                         of the                                         amplified sequence.
Golden gate assembly of the amplified products.
After digestion with BsaI restriction enzyme,the                                         amplified sequences acquire complementary sticky ends                                         and thus they can be assembled together forming the Psp1w1-CRISPRi-stabilized promoter-pTHSSe_59 vector.
Restriction digestion with two enzymes that are unique to                                     prefix and suffix.
Since the assembled CRISPRi-stabilized-Psp1w1 construct                                         is compatible with Biobrick RFC[10], it can be                                         easily                                         assembled with other vectors. This can be achieved by                                         cutting the Psp1w1-sgRNA-stabilized                                             promoter-pTHSSe_59 vector and the receiver vectors with two enzymes having                                         their restriction sites on the prefix and                                         suffix,followed by ligation.
dCas9-tetR expression cassette
The dcas9-tetR expression cassette is located at the                                 pAN-Ptet-dcas9 plasmid. This plasmid was obtained from Addgene                                 and is not compatible with RFC[10] biobrick standard,                                 since it                                 lacks prefix and suffix. Furthermore there is one recognition site of the EcoRI enzyme in the coding sequence of the                                 dCas9.                                 Taking all the above into consideration, we decided to modify                                 this plasmid in order to make it compatible with the RFC[10] biobrick standard.
Steps
PCR amplification of TALEsp1-pSB1C3 plasmid with Vi1, Ri2 primers.
These primers amplify pSB1C3 along with prefix, suffix                                         and incorporate BsaI restriction sites and the                                         external regions of the amplified sequence.
PCR amplification of pAN-Ptet-dCas9 plasmid with DC1, DC2 primers.
These primers amplify the dCas9-tetR-lacI expression cassette and add BsaI restriction                                         sites flanking the amplified sequence.
Golden gate assembly of the amplified products.
After digestion with BsaI restriction enzyme,the                                         amplified sequences acquire complementary sticky ends                                         and thus they can be assemble together forming the pAN-Ptet-dCas9-prefsuf plasmid.
PCR amplification of pAN-Ptet-dcas-prefsuf plasmid with dcm1 and dcm2 primers and sequential Golden Gate                                     assembly.
These primers amplify the dcas9-tetR-lacI expression                                         cassette along with the vector backbone and incorporate BsaI sites at the amplified sequence. After                                         digestion                                         with BsaI, followed by ligation this sequence                                         can be                                         circularized forming the initial plasmid with a single                                         mutation at the EcorI recognition site.
AND GATE
AND Gate-pSB4K5 & AND Gate-pSB3K3
The following procedure has been followed in order to                                 incorporate the AND Gate circuit at pSB4K5 and pSB3K3 plasmids.                                 Then, we evaluated the circuit’s behaviour at these plasmids of                                 different copy number.
Steps
Design and synthesis of the part A, part B fragments with the appropriate flanking regions.
PCR amplification of pSB1C3 vector with a set of                                     standardized primers.
These primers belong to the Metabrick Platform and                                         incorporate BsaI restriction sites to the                                         biobrick prefix and suffix.
Golden Gate assembly reaction between part A,part                                             B and                                         the amplified pSB1C3 vector. This reaction forms                                         the                                         final plasmids AND Gate-pSB1C3 containing the                                         AND gate                                         circuit.
Restriction digestion with EcoRI and SpeI (enzymes unique to prefix and suffix)
Since the assembled AND gate construct is compatible                                         with Biobrick RFC[10], it can be easily                                         assembled with                                         other vectors. This can be achieved by cutting the AND                                             Gate-pSB1C3 plasmid and the receiver vectors                                         with two                                         enzymes having their restriction sites on the prefix                                         and suffix. Following this approach we digested and                                         assembled the AND gate circuit with pSB4K5 and pSB3K3 vector, constructing the AND Gate-pSB4K5 and AND                                             Gate-pSB3K3 plasmids.
[1] Damalas, S. (2017, June 9). The MetaBrick platform for DNA manipulation                     and standardization. Bridging Synthetic Biology standards for optimized                     interoperability. https://doi.org/10.17605/OSF.IO/8DHMU
