<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Thessaloniki_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Thessaloniki/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="q-app" style="margin-top: 16px"><IGEM-LAYOUT><Q-PAGE class="text-justify q-pa-lg"><DIV class="flex flex-center q-mb-md" :class="rM('text-dark', 'text-light')"><DIV class="col-xs-12 col-sm-10 col-md-9 col-lg-5"><P>
                    The goal of the model is to study the Incoherent Feed Forward Loop (IFFL)
                    as a network type in three systems that use different components critical
                    to the proper network operation. This function aims at balancing the final
                    product to a specific steady state, regardless of the input disruption. We
                    then responded to questions raised by the wet lab, thus helping towards the
                    right course of thought during experimental planning, as well as a better
                    understanding of the systems and the different choices that emerged. We
                    applied a sensitivity analysis to classify system parameters according to
                    their significance, and finally, we explored the robustness of our system
                    predictions by indentifying parameter sets and evaluating their cellular
                    behavior.
                </P><H3>How IFFL works</H3><P>
                    The stability of a system in which IFFL is applied is represented by the
                    staying of its output at a constant level in the equilibrium state for each
                    change of input. In this type of network, the number of plasmids is the
                    input, which is variable and the final protein is the output of the system
                    that must remain constant at each change of input.
                </P></DIV></DIV><DIV class="row justify-center q-mb-md" :class="rM('text-dark', 'text-light')"><DIV class="col-md-4 col-sm-12"><CIRCLE-CONTENT title="Workflow"><TEMPLATE slot="modal-content"><DIV class="row justify-center q-pa-md"><DIV class="col-xs-12 col-sm-10 col-md-9 col-lg-8"><H3>Dry Lab Workflow</H3><P>
                                    As Dry Lab our work consists of:
                                    <UL><LI>Accurately describe the mathematical representation
                                            of
                                            the biological system.</LI><LI>Create simulations that predict the behaviour of
                                            the
                                            system.</LI><LI>
                                            Characterize system parameters by:
                                            </LI><LI>working with previous studies.</LI><LI>analyzing and prioritizing the effect that
                                                    parameters have on the system output.</LI><LI>analyzing the robustness of the system and
                                                    make
                                                    observations.</LI></UL></P><P>
                                    All of the above are going to be useful for your project,
                                    as
                                    you need to have a model, that can estimate the in vivo
                                    system
                                    behaviour of the system, in order to be able to make and
                                    test
                                    hypotheses that you and the Wet Lab poses.
                                </P><P><B>In this section</B> we our going to <B>share the
                                        knowledge</B>
                                    that we
                                    gained, from our experience working for our project, in
                                    designing and working on biological network motifs. You can
                                    also view this section as <B>brief introduction on all the
                                        things
                                        we did</B>, as we put links as examples in the
                                    workflow,
                                    that take
                                    you to our models.
                                </P><P class="text-center">
                                    In Figure 1, we present a flowchart, that summarizes our
                                    understanding regarding how you should approach the
                                    modelling
                                    of a biological network.
                                    <SMALL class="text-grey-5">Figure 1: Dry Lab workflow summarized in a
                                        flowchart.</SMALL></P><P>
                                    The most important part is that you need to understand
                                    perfectly how the system should work and what each
                                    component of
                                    the system does. This means, that you have to arrange
                                    sessions
                                    with Wet lab where you will discuss about the system and
                                    you
                                    ask as many questions as you have. These sessions will
                                    never
                                    end, but it’s important to arrange as many as you can when
                                    starting off, in order to build a strong foundation on
                                    understanding your system and what it has to offer.
                                </P><P>
                                    After that you will be confident enough to showcase the
                                    interactions happening in the system and create a
                                    mathematical
                                    representation of it. The mathematical representation in
                                    our
                                    case, were the Ordinary Differential Equations of the
                                    system
                                    and the chemical reactions happening between different
                                    components in the cell.
                                </P><P>
                                    Furthermore, for the model to be accurate it has to have
                                    well
                                    characterized parameters. For this, you can actually do
                                    many
                                    things. We combined sensitivity and robustness analysis
                                    alongside with a very extensive bibliography search. Our
                                    methods are described later in this section and you can
                                    find
                                    implementations of them on both TALE and dcas9-sgRNA
                                    models. As
                                    shown in Figure 1, it is important to take into account,
                                    that
                                    the condition that will break the characterization loop is
                                    different for each project and depends on what you want
                                    from
                                    your model and how accurate should it be.
                                </P><P>
                                    And that’s it. After all these things, you have a model
                                    that
                                    tries its best to describe your system behaviour and
                                    estimate
                                    the concentration of the different components. But, we are
                                    not
                                    finished yet. Now you have to arrange new sessions with Wet
                                    lab, show them your system, describe to them what you did
                                    and
                                    answer all the questions they have, and most importantly,
                                    make
                                    decisions about the system design that you could not answer
                                    before. Now you can make any hypothesis for the system and
                                    actually test it with your model. You should though, take
                                    into
                                    consideration, that the model is an estimation of the
                                    reality
                                    and depending on the effort you put in the designing and
                                    characterizing it, you will get better or worse
                                    estimations. As
                                    an example, of how we used our models to help in the system
                                    design, you can take a look at <A href="https://2018.igem.org/Team:Thessaloniki/Model/Tale#test">TALE tuning with IPTG- 
                                    LacI</A>,
                                    <A href="https://2018.igem.org/Team:Thessaloniki/Model/dCas9#repan">dcas9-sgRNA repressor 
                                    analysis</A>, 
                                    and 
                                    <A href="https://2018.igem.org/Team:Thessaloniki/Model/dCas9#express">dcas9 expression 
                                    site</A> 
                                    decision. As a last part of our model, we also tested the
                                    implementation of <A href="https://2018.igem.org/Team:Thessaloniki/Model/iffl">dcas9-sgRNA 
                                    repressor 
                                    analysis</A> IFFL in RNA level. On the other hand, an
                                    example of how we integrated Wet lab’s feedback into our
                                    model
                                    is shown in Dcas9 subsystem.
                                </P><P>
                                    Finally, when the experiments are finished you will have
                                    the
                                    chance to take the experimental data and the model
                                    parameters
                                    into them. This could be as simple as using the data
                                    produced
                                    by robustness analysis and finding the parameter set with
                                    the
                                    minimum distance from the experimental data, as we did in
                                    TALE
                                    robustness analysis, or using methods like linear/non
                                    linear
                                    regression, bayesian estimation etc. The most experimental
                                    data
                                    you have, the better for the model characterization. In our
                                    case, the data we used [1] to fit the model was the system
                                    error and for this reason, we had limited options regarding
                                    what parameters we should fit to achieve the better
                                    results. To
                                    overpass this obstacle, we used the result from TALE
                                    sensitivity analysis that showed as what was the most
                                    influential parameters for the output and fitted them while
                                    fixing the others.
                                </P><P>
                                    Now you have a model, that is fitted to experimental
                                    results
                                    and you can use that model, if you want to continue your
                                    work
                                    in the system. If the experimental results, are not as
                                    expected, after fitting your model to them, you can
                                    optimize
                                    the parameters or use new components and continue by giving
                                    new
                                    feedback to Wet lab. Again for optimizing you can perform a
                                    new
                                    robustness analysis, because with it you can test
                                    extensively
                                    different sets from the parametric space save and analyze
                                    all
                                    the results as you want. Another option that we considered,
                                    is
                                    a gradient descend algorithm but as we didn’t used it in
                                    the
                                    end, we will not discuss further about it. If computational
                                    cost is not a problem for you, we would recommend
                                    robustness
                                    analysis, otherwise gradient descent algorithms or other
                                    Artificial Intelligence algorithms are the way to go.
                                </P><H4>Sensitivity analysis</H4><H5>What is sensitivity analysis?</H5><P>
                                    The study of how uncertainty in the output of a model
                                    (numerical or otherwise) can be apportioned to different
                                    sources of uncertainty in the model input [3].
                                </P><H5>Why we use sensitivity analysis?</H5><P>
                                    When we construct a computational model, there are many
                                    parameters that can not be identified, because it is likely
                                    that a similar experiment has not been carried out. Also,
                                    there is a chance where a similar experiment has been
                                    carried out, but for minor variations of the experimental
                                    arrangements, the changes in each parameter are large. It
                                    is therefore extremely useful to use a mathematical
                                    tool,the sensitivity analysis, with which we can come to
                                    many conclusions that will help us to evolve the model. The
                                    most important of them are[3]:
                                </P><P><B>Factor prioritization:</B> One reason for using
                                    sensitivity analysis is to study the parameters in order to
                                    identify the most important ones. The most important ones,
                                    i.e. those with the highest sensitivity, are the ones that
                                    cause the greatest variation in the final output. By
                                    knowing what these are, we can focus on their precise
                                    measurement to reduce the variance of the final system,
                                    while providing a model that best approaches reality.
                                </P><P><B>Factor Fixing:</B> Having ranked parameters from the
                                    most important to the less important, we can choose a fixed
                                    point for the non-critical parameters that lies within the
                                    originally defined range since it does not contribute much
                                    to the output variance. In this way, we can better observe
                                    the inter-relationships of important parameters in many
                                    system subunits that contain a large set of kinetic
                                    equations.
                                </P><P><B>Variance cutting:</B> To reduce the variance of the
                                    output, we
                                    can apply a series of sensitivity analyses, starting from
                                    all system inputs and investigating each time which
                                    parameters lead to greater sensitivity in output variance.
                                    By removing the parameters, that are the most influential,
                                    as long as they are characterized well for the biological
                                    system we are studying, then the output variance that
                                    remains is the sum of variations caused by the
                                    non-important parameters which is not big. This leads the
                                    system to fewer input parameters with less output variation
                                    and simultaneous reduction of computational cost.
                                </P><P><B>Factor Mapping:</B> To achieve concrete outputs, we need
                                    to
                                    evaluate the relationships between inputs and which
                                    parameter combinations lead to undue variation in output.
                                    There are parameter interactions that for a particular
                                    system cannot achieve system response within strictly set
                                    limits and certain conditions. For example, in a system
                                    that controls the amount of a toxic substance that, if
                                    present in large quantities, can lead to cell destruction,
                                    we need to explore the system's parameters, and find which
                                    combinations lead to the desired results.
                                </P><H4>Global vs Local Sensitivity analysis</H4><P>Sensitivity analysis can be applied in 2 variations:</P><P>
                                    Local sensitivity analysis evaluates changes in model
                                    outputs with respect to variations in single parameter
                                    input [3] [4]. In particular, a small change is made,
                                    accompanied by a calculation of the local sensitivity
                                    index, for each parameter separately. The local sensitivity
                                    index of a parameter p is calculated by the partial output
                                    of the system relative to the parameter for a very small
                                    change of the parameter value.
                                </P><P>
                                    On the contrary, Global sensitivity analysis simultaneously
                                    varies all parameters over the entire parameter space,
                                    which allows to simultaneously evaluate the relative
                                    contributions of each individual parameter as well as the
                                    interactions between parameters to the model output
                                    variance [3]. Because of this, global sensitivity is more
                                    capable of approaching the parameters when non-linear
                                    phenomena occur in the system.
                                </P><P>
                                    The biological systems that we study in synthetic biology
                                    show many non-linear phenomena among their components and
                                    the range of parameter values cannot always be found in
                                    bibliography and is often considered large. For this
                                    reason, it is proposed to explore the importance of the
                                    parameters globally.
                                </P><H4>Sobol sensitivity analysis</H4><P>
                                    Below we describe the procedure followed by the Sobol
                                    sensitivity analysis to calculate:
                                    <UL><LI>First and second order indices referring to the
                                            contribution of a particular parameter to the
                                            output and to the contribution of the interaction
                                            between the pair of parameters respectively.</LI><LI>Total-effect indices to calculate the contribution
                                            of a parameter in combination with its contribution
                                            from its interactions with other parameters.</LI></UL>
                                    The algorithmic analysis process begins by producing a
                                    sample from the input space of each parameter. This is done
                                    by quasi-random sequences that produce samples that are
                                    uniformly distributed in the area with small differences
                                    between them and are often used for monte carlo integration
                                    [3]. Then, for each set of parameters, the output of the
                                    system is calculated. Finally, Sobol indices are calculated
                                    by the procedure described below and the results are
                                    analyzed in first, second and total sensitivity indices
                                    [4].
                                </P><P>For a system, the output of which is characterized by a
                                    function f(x), which is integrable and its inputs are
                                    x(x1,x2,x3). The first move is to decompose into the sums
                                    of the effect, that the various components of the system
                                    have.</P><P>Defined as , is the average of f(x)
                                    for different parameter sets values</P><P>
                                    f(x) Decomposition:
                                    </P><P>
                                    We can generalize the above as follows
                                    </P>
                                    where n is the number of system inputs and i &lt; j,
                                    because fij(xi,xj) = fji(xj,xi).
                                <P>
                                    Then we define the variance D of the output function, of
                                    the total sums and the average of each parameter as a
                                    random variable, that is evenly distributed over an
                                    interval [4] and the result is the sum of the variance
                                    caused by each parameter or a combination of these in each
                                    set of parameters describing the total variance of the
                                    output as the following sum:

                                    </P>
                                    where f(x) is square-integrable, therefore as total sums of
                                    the contribution of all the differences between the set
                                    parameters,
                                    
                                    Equation (3) is analyzed as (1), and shows how much each
                                    variable has contributed, as well as their combined
                                    interactions in the final overall system variance.
                                    
                                    where  is the variance we have
                                    from specific parameters. For example, if we wanted the
                                    first order sensitivity index of variable we would use as a
                                    numerator only .
                                <P>
                                    Computational cost is proportional to the type of problem,
                                    the number of parameters and the non-linear phenomena that
                                    appear in the relationship between the interacting
                                    parameters. For small numbers of parameters like the model
                                    we have performed, variance-based methods are considered
                                    satisfactory and relatively quick. As the order of
                                    sensitivity indices increases, the cost increases
                                    accordingly. As mentioned above, one way to use a
                                    variance-based method like Sobol is to reduce the number of
                                    parameters (variance cutting) by excluding for each
                                    calculation those that do not affect.
                                </P><H3>Robustness analysis</H3><P>
                                    When we start building a model, its structure contains many
                                    uncertainties that are not easy to assess. Such
                                    uncertainties are the mathematical expression of kinetic
                                    equations, stoichiometric coefficients of kinetics, and the
                                    simplifications made from a more complex network to a more
                                    simplified model. In addition, the complexity of biological
                                    systems makes it impossible to fully analyze all the
                                    reactions that occur inside and outside the environment,
                                    among the various components that they consist of.
                                    Robustness analysis is a tool useful for studying the
                                    general behavior of a system, despite the uncertainties
                                    that exist in it.
                                </P><P>
                                    Applying robustness analysis to the mathematical model
                                    under study, we set price ranges for each parameter that we
                                    do not know, setting maximum and minimum values ​​that are
                                    considered realistic within the biological system, which
                                    are estimated from bibliography and experimental data. Then
                                    the model is exhaustively simulated in many sets of
                                    parameters created by sampling. The sampling method used is
                                    the Latin hypercube, which generates random numbers divided
                                    into equal intervals within the range given for each
                                    parameter. Within each interval a value is randomly
                                    selected.
                                </P><P>
                                    We use error bars in order to show the output variation
                                    around the mean value. They may correspond to a large range
                                    of parameter values that are spaced apart to several orders
                                    of magnitude between them. The error bars were calculated
                                    either by standard deviation of all concentrations for each
                                    copy number or by the corresponding standard error.
                                </P><P>
                                    Once, the sensitivity analysis described above, has been
                                    applied to reduce the number of uncertain parameters in the
                                    system, and robustness analysis to output values ​​that are
                                    highly variable, we can evaluate the results and better
                                    study the behavior of the system. In our system we studied
                                    the uncertainty of protein of interest concentrations at a
                                    time when all components of the system are stabilized in
                                    steady-state, relative to the copy number of the plasmids.
                                </P><H3>Implementation</H3><H5>Simbiology Matlab®</H5><P>
                                    We created the biological networks, building-up the
                                    dynamic
                                    systems using the MATLAB®language, specifically Simbiology®
                                    toolbox that allowed as to analyze easily the kinetic
                                    equations
                                    with species, reactions and doses-response with it’s simple
                                    to
                                    use Graphical interface. We chose to solve the ordinary
                                    differential equations with variable-step continuous solver
                                    ode15s . The ode15s is a good choice for stiff problems and
                                    in
                                    our simulations settings we set absolute tolerance 
                                    and
                                    relative tolerance  . Υου can see
                                    our simbiology diagrams
                                    here.
                                </P><H5>Python packages </H5><P>
                                    After the usage of Simbiology for the chemical interaction
                                    modelling we use scipy’s odeint for the construction of our
                                    systems ODE equations. We use python because we wanted to
                                    integrate other packages (e.g for sensitivity analysis)
                                    that help us to extend our analysis, and compare the
                                    results. The interaction between simbiology and python help
                                    us to understand the differences that exist between them
                                    and to practice in both environments.
                                </P><P>
                                    In addition to that, we used Python’s built in
                                    multiprocessing module in order to increase the
                                    computational speed by dividing the tasks of robustness and
                                    sensitivity analysis to many processes (as many as the
                                    cores of each PC).
                                </P><H5>SALib - Sensitivity Analysis Library in Python</H5><P>
                                    This library helped us to incorporate sensitivity analysis
                                    in all systems calculating the effects of model input
                                    (different parameters in each model) regarding variation of
                                    model output (concentration of sfGFP). We specifically used
                                    Sobol Sensitivity analysis that calculates the sensitivity
                                    index generating model inputs using Saltelli’s extension of
                                    the Sobol sequence sampling. This way, we were able to
                                    understand which parameters are the most affected and
                                    discuss with wet lab if and what changes are needed. You
                                    can see all of our simulation code in our <A href="https://github.com/igem-thessaloniki/Our-models">Github page</A>. 
                                </P><H5>Robustness analysis</H5><P>
                                    For robustness analysis, we use python packages and
                                    SALib.sample package to produce the sampling. The sampling
                                    uses the Latin hypercube method (LHS), that generates a
                                    random sample of N points for each uncertain input
                                    parameter. The number of parameter sets is 10.000 unless
                                    otherwise mentioned and the parameter ranges take the same
                                    values as in sensitivity analysis. For each result,
                                    robustness analysis measures the standard deviation, and
                                    the mean for every copy number as well as the error between
                                    input-output and prints these results in a csv.
                                </P><H5>Digital Ocean</H5><P>
                                    Furthermore, in order to increase the computational speed of the analysis as well as running more experiments together, we used Digital Ocean cloud computing platform.
                                </P><H5>Salis calculator RBS</H5><P>
                                        We used RBS calculator from Salis Lab [5] to estimate the mRNA’ s translation rate for all proteins on our systems. The calculations are described in each model’s page. See translation rate estimation.
                                </P><H6>REFERENCES</H6><DIV class="row justify-center q-mb-md q-mt-md" :class="rM('text-dark', 'text-light')"><SMALL class="row q-mb-sm">
                                        [1] Segall-Shapiro, T., Sontag, E. and Voigt, C. (2018). Engineered promoters enable constant gene expression at any copy number in bacteria. Nature Biotechnology, 36(4), pp.352-358.
                                    </SMALL><SMALL class="row q-mb-sm">
                                        [2] Herman, J. and Usher, W. (2017) SALib: An open-source Python library for sensitivity analysis. Journal of Open Source Software, 2(9). doi:10.21105/joss.00097
                                    </SMALL><SMALL class="row q-mb-sm">
                                        [3] Ratto, M., Andres, T., Campolongo, F., Cariboni, J., Gatelli, D., Saisana, M. and Tarantola, S. (2008). Global Sensitivity Analysis. John Wiley &amp; Sons.
                                    </SMALL><SMALL class="row q-mb-sm">
                                        [4] Zhang, X., Trame, M., Lesko, L. and Schmidt, S. (2015). Sobol Sensitivity Analysis: A Tool to Guide the Development and Evaluation of Systems Pharmacology Models. CPT: Pharmacometrics &amp; Systems Pharmacology, 4(2), pp.69-79.
                                    </SMALL><SMALL class="row q-mb-sm">
                                        [5] Espah Borujeni, A., Channarasappa, A. and Salis, H. (2013). Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites. Nucleic Acids Research, 42(4), pp.2646-2659.
                                    </SMALL></DIV></DIV></DIV></TEMPLATE></CIRCLE-CONTENT></DIV></DIV></Q-PAGE></IGEM-LAYOUT></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>