Team:Thessaloniki/Design
Design
Design
Project Design
Overview
Through model and expert driven feedback, we designed systems that                     allow for the stabilization of promoters and on-the-fly control of protein                     expression at desired levels.
Engineering Principles
While designing our systems, we focused on the engineering principles                     governing both the functionality of the iFFL network motif and that of our                     individual parts, as members of a functional whole.
It is integral to maintain the balance between different parameters                     determining the behaviour of the iFFL in order to achieve the stabilization                     of a promoter as desired. We made sure that the systems are orthogonal and do not interface with the native genetic circuitry of the chassis to                     the degree that would cause the systems not to function as expected [1].
Having a pivotal role in the design process of functional parts and                     projects, we made sure to include debugging in our day-to-day problem solving efforts. Both in experiments, part design                     and modelling, constantly cataloguing and revising our progress, along with                     feedback from analyzing our results, helped us pinpoint our mistakes and                     find ways to tackle them. Our debugging efforts were focused on deciphering                     what went wrong with the constructs that did not function as expected.
In order for the parts to be easily integrated and successfully used by                     other iGEM teams, their behaviour has to be predictable and standardized.                     Working toward this, we conducted a series of experiments to characterize                     and tune our systems, while keeping in mind the various future applications                     and uses they might have. Along those lines, we, also, made sure that our                     measurements were expressed in relative units, calibrated with a                     fluorescein curve or calibration beads.
Stabilized Promoters
Stabilized promoters are regulatory genetic elements that achieve gene                     expression independent of gene/plasmid copy number. They can be designed                     with the implementation of a type I incoherent feedforward Loop (iFFL)                     network motif that renders the Gene of Interest (GOI) expression                     independent of the plasmid’s copy number. Copy number positively affects                     GOI expression, while its repressor, being under the same positive                     influence, compensates for this change. If the Hill coefficient that                     characterizes the repression is 1, and thus the repression is perfectly                     non-cooperative, it results in the disassociation of the GOI’s expression                     from the copy number.
Each stabilized promoter is characterized by its strength, the gene                     of interest expression level and error, the relative change in the gene of interest expression as the copy number increases from the lowest to                     the highest copy number measured [2]. As the expression level of the                     repressor increases, the stabilization error                     and the promoter’s strength decrease, leading to weaker but more stable                     expression across different copy numbers.
Reaching the desired expression level can be achieved by two different                     approaches. Either by re-tuning the expression level of the repressor, and                     thus the stabilization level, or by on-the-fly regulation by induction from                     a riboswitch regulatory element.
System Design
TAL Effector Stabilized Promoters
The first system we used was a set of TAL Effector stabilized promoters,                     described by Shapiro et. al. [2], in order to investigate their function                     and potential uses. TAL Effectors bind to specific promoter sequences and                     prevent the recruitment of RNA polymerase in order to create a type I                     incoherent Feedforward Loop (iFFL) network motif that renders the                     expression of the downstream sequences independent of the plasmid’s copy                     number.
We used TALEsp1 (TALE stabilized promoter 1) and TALEsp2 (TALE stabilized                     promoter 2) in order to achieve the desired expression level and Error.                     TALE proteins bind as monomers in specific regions inside the promoter thus                     inhibiting transcription initiation. The repression level is determined by                     the operator’s site in the promoter. Specifically, TALEsp1 has stronger                     expression and higher Error level when compared to TALEsp2, since TALEsp2                     binding site includes a nucleotide from the -10 promoter region.                     To characterize TALE stabilized promoters we measured their expression                     level in different copy numbers. We, also, conducted an Interlaboratory                     Collaboration in order to investigate if the TALEsp1 and TALEsp2 reduce                     interlaboratory variations.
Inducibility
Wanting to expand on the TAL Effector stabilized promoters system, we                     designed a tool that would allow, on top of the stabilization of a                     promoter, its induction and activation to the desired expression level.
Ιnitially, we considered the implementation of the LacI repressor that                     represses transcription by binding to a specific DNA sequence. This system                     is well characterized and is widely used for transcription regulation.
Since we wanted complete control over the expression of the stabilized                     promoter we intended to introduce a LacOsym binding site near the promoter                     region, besides where the TAL Effector recognized and interacted with the                     promoter sequence. During our time designing the system we took note of                     possible steric effects between the TAL Effector and the LacI                     repressor while bound to the Promoter Sequence.
In order to conceptualize and better understand the implications of possible                     steric effects on the stabilization of the promoter, a model of ODEs                     describing the primary processes of gene expression and the effects of such interactions was set up and simulated on SimBiology.
In order to conceptualize and better understand the implications of                     possible steric effects on the stabilization of the promoter, a model of                     ODEs describing the primary processes of gene expression and the effects of                     such interactions was set up and simulated on SimBiology.The results                     pointed towards a loss of the non-cooperative nature of the TAL Effector                     repression, depending on the IPTG induction level.                     At that point, during an enlightening conversation with Thomas H.                     Segall-Shapiro , we noted our concern about the induction via the LacI                     repressor and the issues it might cause. He confirmed our concerns and                     encouraged us to use another type of regulator.
Since interference with the regulatory effects of TALE repressors was                     obstructive to their intended function, as predicted by the model, we chose                     to abstain from implementing transcriptional regulators, and thus moved to                     translational regulatory elements, like the Riboswitch, instead.
We chose the theophylline riboswitch, since, in contrast to the adenine                     riboswitch, which we also considered, is completely orthogonal and thus                     provides greater predictability in the outcome of the induction. In order                     to allow for more precise control of the expression levels a riboswitch                     with high dynamic range was required .
We came in contact with Prof. Howard                     Salis to ask for advice about the riboswitch design. He noted that the post                     and pre-aptamer regions where important, and different RNA sequences might                     have varying effects on the behaviour of the riboswitch due to possible                     changes in the secondary structure of the aptamer. He advised us to use a                     fusion protein as a marker that would maintain the first 99 ribonucleotides                     (33 amino-acids) of the reporter used in the study, luciferase, if we                     wanted to use a different one like sfGFP, and avoid aberrant aptamer                     secondary structures that would, potentially, deteriorate its                     functionality.
We implemented two different riboswitches, BBa_K2839006 and Theo-27 [3]                     in the TALEsp1 system. As a reporter, we chose sfGFP fused with the first                     99                     nucleotides of luciferase, due to the fact that it uses luciferase as the                     reporter. The fusion protein can be, also, used with the 12.1 riboswitch                     [4], because it does not affect the aptamer region. We selected sfGFP over                     other available marker since its maturation and fluorescence is unaffected                     by fusion partner misfolding [5].
CRISPRi
While the TALE-stabilized promoters function with great precision [2], in                     order to stabilize a large number of promoters, (e.g. in a complex                     metabolic pathway or genetic circuit) one would need either more than one                     TAL Effector proteins or to redesign the promoter, changing the TALE                     binding site to effectively stabilize promoters with the desired                     strength-error trade-off and to the required expression level. This would                     increase the metabolic burden imposed on the cell by the expression of                     multiple proteins at high levels. This sets a limit to the usage of                     stabilized promoters.
To tackle this issue we decided to use CRISPRi to repress and                     stabilize promoters. It utilizes the ability of the catalytically dead                     variant of Cas9, dCas9, to bind to specific sequences, in order to prevent                     transcription initiation by blocking RNAP recruitment [6]. CRISPRi, being a                     relatively new addition to the arsenal of gene regulation factors,                     possesses great potential in controlling gene expression due to dCas9’s                     strong affinity and specificity to the target sequence, determined solely                     by its complementarity with the 20 first nucleotides of the single guide                     RNA (sgRNA) sequence [7]. The only limitation is the need for the presence                     of a protospacer adjacent motif (PAM) for the dCas9 enzyme to recognize                     [8]. Afterwards, it interrogates the upstream sequence for complementarity                     with the sgRNA. The aforementioned complementarity determines the binding                     affinity of the sgRNA-dCas9 repressor complex to the determined sequence                     [7].
As the output promoter, we used an Anderson promoter, since the Anderson                     library is well characterized and all promoters include a 5’-NGG-3’ PAM                     sequence just upstream of the -10 site. This allows for easy design of the                     sgRNA [8] and ensures tight repression [9][10].
Choosing where each component should be expressed from, we set up a model                     to simulate the system. Initially, we had 3 options for the design of our system, regarding dCas9 expression site: the plasmid with the sfGFP marker,                     the genome, or the insertion of a second compatible plasmid. Results showed                     that the expression of both the sgRNA and the dCas9 from a single plasmid                     ( Topology A ), in which the promoter to be                     stabilized resides, broke the                     system and exhibited behaviour similar to repression with a Hill                     coefficient of 2.                     Another option was the genome insertion of dCas9 ( Topology                         B ), which would                     offer increased system stability, but, when simulated in our model,                     expression of dCas9 proved inadequate.
We decided to use a double plasmid system ( Topology                         C ), co-expressing the sgRNA along with sfGFP and expressing                     dCas9 on a separate vector. The plasmid containing the dCas9 expression                     cassette is a low-medium copy number vector (p15A ORI) and dCas9 expression                     rate is controlled by a pTet promoter, therefore its output depends on the                     usage of the external inducer, doxycycline.
In this system, the sfGFP promoter’s stabilization was designed to depend                     only on the expression level of the sgRNA. Therefore, we expressed dCas9 on                     the highest level that creates saturating condition without inhibiting                     growth due to excessive metabolic burden [11]. In order to investigate the                     effect of dCas9 production on the growth rate of the DH5alpha E.coli                     strain, we conducted a growth assay, expressing dCas9 on different levels,                     via induction with Doxycycline.
Regarding the sgRNA design, in order to achieve the required repression                     level, we had 2 options: either constitutively express a sgRNA partially                     complementary to the target sequence or induce the expression of a fully                     complementary sgRNA to its target. We decided to induce the expression of                     sgRNA so that we could create a response function from which we can acquire                     valuable data on the behaviour of the system [1]. Control of sgRNA                     expression was achieved through a L-Rhamnose inducible promoter, pRha [12].                     Full complementarity of sgRNA to its target sequence, in combination with                     dCas9 overexpression and lack of saturation with sgRNA, made it possible to                     control the repression levels by adding L-Rhamnose.
In order to characterize the CRISPRi stabilized promoter we induced the                     expression of the sgRNA driven by the L-Rhamnose inducible promoter at the                     desired lowest copy number (psc101). From the response function we can                     determine the cooperativity of the repression and -choose the expression                     level of the sgRNA that corresponds to the desired strength of the                     stabilized promoter.
After characterizing the response function of the input and output                     promoters, we would have settled on a desired Strength-Error level and                     replace the rhamnose inducible promoter with a constitutive one, thereby                     maintaining the expression stable, independent of the plasmids copy number,                     for all copy numbers without need for addition of more rhamnose to compensate for the increase in the number of RhaS binding sites.                     Unfortunately, we observed minimal induction when measuring the pRha                     mediated sgRNA expression so we hypothesized that this was due to the                     change of a conserved region after the promoter’s +1 site.                     Therefore, we chose BBa….. for the expression of the sgRNA cassette. We                     opted for a strong constitutive promoter in order to achieve a low Error                     value as this may prove valuable when implementing the CRISPRi                     stabilization system in order to stabilize complex biosynthetic pathways.
Stabilized heterologous AND gate
As we wanted not only to characterize stabilized promoters, but also to                     implement them into synthetic circuits that exhibit dysfunctionality over                     different copy numbers, we chose a previously well-characterized                     heterologous AND gate, that, when introduced into E. coli cells at                     different copy number backbones, exhibits contradicting behaviour, with                     fluorescence output seemingly being inversely proportional to the copy                     number.                     The circuit consists of the σ54-dependent hrpL promoter which is a part of                     hrp regulon naturally found at the plant pathogen Pseudomonas syringae.The                     system is active producing the output signal only if both the co-activating                     input proteins hrpR and hrpS are expressed forming the HrpR/Hrps complex                     which binds and induces the output promoter. HrpS is expressed from the                     TetR repressor promoter, PTet, inducible by anhydrotetracycline and                     doxycycline while HrpR, from the LuxR activatable promoter, PLux, induced                     by AHL. Despite being orthogonal to E. coli cells, its exhibits                     contradictory behaviour stemming from irregularities in the TetR induction                     level on different copy numbers. While we would expect expression from a                     low copy number plasmid to yield lower fluorescence relative to that of a                     medium copy number plasmid, this AND gate, behaves in the opposite way. Due                     to higher TetR levels expressed from the medium copy number plasmid, the                     same amount of inducer yields lower expression of HrpS co-activator. This                     leads to a lower activator complex concentration, since it is determined by                     the concentration of the subunit with the lowest expression level.
In order to allow for its stabilization across different copy numbers                     without the need for re-tuning, we attempted to stabilize the expression of                     TetR at a level that would allow for the same behaviour at both copy                     numbers investigated.
Unfortunately, the And Gate didn’t function at all.
Attenuator
In search for a more elegant approach to the stabilized promoters, we                     turned to RNA-based regulation, that does not require the introduction of                     transgenic proteins that might cause burden-related issues due to                     overexpression and has a shorter response time. We designed a sequence                     containing many individually acting units, as part of the same transcript.                     Firstly, two small RNA (sRNA) separated by a hammerhead ribozyme are                     transcribed upstream of an attenuator sequence that is itself flanked by                     hammerhead ribozymes. The sRNAs interact with the attenuator sequence, halting transcription, by forming a terminator loop. Downstream of the                     attenuator is the RBS along with the coding sequence. This way, the sRNAs                     are constitutively expressed and, by causing premature transcription                     termination, repress the transcription of the sequences downstream of the                     attenuator. Through modeling, we decided to use two copies of the sRNA                     instead of one since they provide tighter repression and thus lower                     stabilization error. (graphs)                     In addition to the gene/copy number independant expression, this design                     achieves promoter-independent expression. That is to say, a relatively                     stable expression over a wide range of promoters with different output                     levels. This is achieved by the inclusion of the sRNA, acting as the                     repressor, in the same transcript as the coding sequence. Thus, all events                     occurring before the transcription of the repressor, including the                     frequency at which the RNA Polymerase (RNAP) is recruited (ή: the rate of                     RNAP recruitment), are part of the primary input of the iFFL and rendered                     irrelevant to the expression level achieved.
This design is expected to greatly reduce variations arising from                     transcription related events, thus diminishing promoter-caused internal                     noise.
Due to time constraints, we did not manage to submit and characterize this                     part. It’s model-predicted behavior can be found here .
Experimental design
Colonies from transformation of ligated products (Golden Gate assembly, 3A assembly) were sampled and colony PCR was performed with VR and VF2 primers                     to determine the size of the insert and thus its validity. For more                     information on the methods and cloning strategy we used, visit the protocols or parts page respectively.
In order to study and test the predicted function of our systems we                     designed experiments measuring fluorescence intensity of a reporter gene.                     The reporter we used was the superfolder Green Fluorescent Protein (sfGFP).                     We measured single-cell fluorescence in a CyFlow(R) Cube 8 Flow cytometer                     at FL1 (ex. 488). The majority of the single cell fluorescence was measured                     during mid log phase of growth, as described in the protocol section of our Notebook page.
Riboswitch behavior was characterized by induction at different inducer                     levels independently and under the stabilization of the TALEsp1 repressor BBa_K2839000 .
All measurements were performed in biological replicates (n=3), unless                     otherwise stated.                     For the characterization of the CRISPRi stabilized promoters we set up                     fluorescence intensity measurements of the output promoter under the                     control of an inducible input promoter at different levels of induction.                     Characterization of the input promoters by measuring the reporter                     expression at different concentrations of the inducer and corresponding the                     inducer concentration used for the output promoter characterization with the input promoter expression, results in the response function of the NOT                     Gate [1].
From the response function we can gather information about the                     experimentally-derived cooperativity of our system, a strong predictor of                     its proper function. By creating a response function we obtained insight                     into the behavior of the system at different repressor expression levels.                     This allows for the replacement of the inducible input promoter with a                     constitutive one at a desired expression level, thus tuning the stabilized                     promoter and allowing for the choice of a desired strength level and                     subsequent strength-error stade-off.                     To determine the optimal expression level of dCas9 from the psB3K3 vector,                     we performed a growth rate analysis of different dCas9 expression levels.                     From the results we determined a inducer concentration, corresponding to                     the highest dCas9 expression level that does not greatly hinder the growth                     rate of the bacteria. Subsequent measurements that included the induction                     of dCas9 were performed at that concentration of Doxycycline
[1] Brophy, J. A. N., & Voigt, C. A. (2014). Principles of genetic circuit                     design. Nature Methods. https://doi.org/10.1038/nmeth.2926
[2] Segall-Shapiro, T. H., Sontag, E. D., & Voigt, C. A. (2018). Engineered                     promoters enable constant gene expression at any copy number in bacteria.                     Nature Biotechnology. https://doi.org/10.1038/nbt.4111
[3] Espah Borujeni, A., Mishler, D. M., Wang, J., Huso, W., & Salis, H. M.                     (2016). Automated physics-based design of synthetic riboswitches from                     diverse RNA aptamers. Nucleic Acids Research, 44(1), 1–13. https://doi.org/10.1093/nar/gkv1289
[4] Lynch, S. A., & Gallivan, J. P. (2009). A flow cytometry-based screen                     for synthetic riboswitches. Nucleic Acids Research, 37(1), 184–192. https://doi.org/10.1093/nar/gkn924
[5] Pédelacq, J. D., Cabantous, S., Tran, T., Terwilliger, T. C., & Waldo,                     G. S. (2006). Engineering and characterization of a superfolder green                     fluorescent protein. Nature Biotechnology. https://doi.org/10.1038/nbt1172
[6] Qi, L. S., Larson, M. H., Gilbert, L. A., Doudna, J. A., Weissman, J.                     S., Arkin, A. P., & Lim, W. A. (2013). Repurposing CRISPR as an RNA-guided                     platform for sequence-specific control of gene expression. Cell, 152(5),                     1173–1183. https://doi.org/10.1016/j.cell.2013.02.022
[7] Sternberg, S. H., Redding, S., Jinek, M., Greene, E. C., & Doudna, J. A. (2014). DNA interrogation by the CRISPR RNA-guided endonuclease Cas9.                     Nature, 507(7490), 62–67. https://doi.org/10.1038/nature13011
[8] Larson, M. H., Gilbert, L. A., Wang, X., Lim, W. A., Weissman, J. S., & Qi, L. S. (2013). CRISPR interference (CRISPRi) for sequence-specific                     control of gene expression. Nature Protocols, 8(11), 2180–2196. https://doi.org/10.1038/nprot.2013.132
[9] Bikard, D., Jiang, W., Samai, P., Hochschild, A., Zhang, F., & Marraffini, L. A. (2013). Programmable repression and activation of                     bacterial gene expression using an engineered CRISPR-Cas system. Nucleic                     Acids Research, 41(15), 7429–7437. https://doi.org/10.1093/nar/gkt520
[10] Vigouroux, A., Oldewurtel, E., Cui, L., Bikard, D., & van Teeffelen, S.                     (2018). Tuning dCas9’s ability to block transcription enables robust,                     noiseless knockdown of bacterial genes. Molecular Systems Biology, 14(3),                     e7899. https://doi.org/10.15252/MSB.20177899
[11] Nielsen, A. A., & Voigt, C. A. (2014). Multi-input CRISPR/Cas genetic                     circuits that interface host regulatory networks. Molecular Systems                     Biology, 10(11), 763–763. https://doi.org/10.15252/msb.20145735
[12] Kelly, C. L., Liu, Z., Yoshihara, A., Jenkinson, S. F., Wormald, M. R.,                     Otero, J., … Heap, J. T. (2016). Synthetic Chemical Inducers and Genetic                     Decoupling Enable Orthogonal Control of the rhaBAD Promoter. ACS Synthetic                     Biology, 5(10), 1136–1145. https://doi.org/10.1021/acssynbio.6b00030
