<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Thessaloniki_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Thessaloniki/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="q-app" style="margin-top: 16px"><IGEM-LAYOUT><Q-PAGE><Q-PARALLAX src="https://static.igem.org/mediawiki/2018/thumb/e/eb/T--Thessaloniki--lab--photo-3.jpeg/800px-T--Thessaloniki--lab--photo-3.jpeg"><H1 slot="loading">Protocols</H1><H1>Protocols</H1></Q-PARALLAX><DIV class="flex flex-center q-mb-md q-mt-md" :class="rM('text-dark', 'text-light')"><Q-LIST separator="" class="col-xs-12 col-sm-11 col-md-10 col-lg-8 q-pa-md" striped="" highlight="" no-border=""><Q-COLLAPSIBLE label="Frozen Competent Cells" :dark="nightMode" color="red"><DIV><H3>Frozen Competent Cells</H3><H4>Materials</H4><UL><LI>Plate of cells to be made competent</LI><LI>LB Broth</LI><LI>CaCl2 solution 0,1M</LI><LI>100% glycerol</LI><LI>Ice</LI></UL><H4>Method</H4><OL><LI>Inoculate 1 colony from a freshly grown plate into 50ml broth</LI><LI>Incubate cells <U>overnight</U> at 37°C in a shaking incubator
                                (200-250 rpm)</LI><LI>Transfer cells into 500 ml LB Broth in a 500ml E. flask</LI><LI>Incubate at 250-300 rpm and 37°C for approx. <U>2,5 hours</U>
                                (or OD = 0.5). Centrifuge at 5000 rpm and 4°C for <U>15 min</U></LI><LI>Remove supernatant and resuspend pelleted cells in 250 ml 0,1 M
                                CaCl2 solution</LI><LI>Incubate on ice for <U>20 min</U></LI><LI>Centrifuge at 5000 rpm for <U>15 min</U></LI><LI>Remove supernatant and resuspend pelleted cells in 43 ml 0,1 M
                                CaCl2 solution</LI><LI>
                                Incubate on ice for <U>2-12 hours</U><SPAN class="text-light-blue"><B>Note:</B> Maximum Incubation at 24 hours. Efficiency of
                                    Transformation is gradually reduced afterwards
                                </SPAN></LI><LI>Add 7ml 100% glycerol, mix well</LI><LI>Aliquot cells into prechilled 15ml tubes</LI><LI>Quick freeze in liquid nitrogen</LI><LI>Store (-80°C)</LI></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Fresh Competent Cells" :dark="nightMode"><DIV><H3>Fresh Competent Cells</H3><H4>Materials</H4><UL><LI>Plate of cells to be made competent</LI><LI>LB Broth</LI><LI>CaCl2 solution 0,1M</LI><LI>Ice</LI></UL><H4>Method</H4><OL><LI>Inoculate 1 colony from a freshly grown plate into 50ml LB
                                Broth in a 500ml E. Flask</LI><LI>
                                Incubate at 250-300 rpm and 37°C for approx. <U>3 hours</U> (or
                                until OD600 is about 0,375)
                                <SPAN class="text-light-blue"><B>Note:</B> Transformation Effeciency will decrease if
                                    cells are allowed to grow to a density greater than 0,4 OD
                                </SPAN></LI><LI>Centrifuge 3000 rpm and 4°C for <U>7 minutes</U>. No brake.</LI><LI>Remove supernatant and drain the tubes by keeping them inverted
                                on an absorbent paper for <U>1 minute</U></LI><LI>Resuspend pelleted cells in 10ml ice-cold CaCl2 solution by
                                pipetting gently up and down. Make sure that cells are fully
                                resuspended. Keep on ice for <U>5 min</U></LI><LI>Centrifuge 3000 rpm and 4°C for <U>7 minutes</U>. No brake.</LI><LI>Remove supernatant and drain the tubes by keeping them inverted
                                on an absorbent paper for <U>1 minute</U></LI><LI>
                                Resuspend pelleted cells in 2ml ice-cold CaCl2 solution by
                                pipetting gently up and down. Make sure that cells are fully
                                resuspended. Keep on ice.
                                <SPAN class="text-light-blue"><B>Note:</B> Increased transformation efficiency next 24h,
                                    decrease after the 24 hour mark.
                                </SPAN></LI></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Chemical Transformation" :dark="nightMode"><DIV><H3>Chemical Transformation</H3><H4>Materials</H4><UL><LI>Chemically competent cells</LI><LI>DNA to be transformed</LI><LI>LB agar with antibiotics</LI><LI>Ice</LI></UL><H4>Method</H4><OL><LI>
                                Thaw 100 μl of competent cells on ice.
                                <SPAN class="text-light-blue"><B>Note:</B> Thawing at room temp decreases efficiency
                                </SPAN></LI><LI>
                                Add DNA:
                                </LI><LI>For transformation of ligated products add 8μl of the
                                        ligation reaction mix</LI><LI>For transformation of known plasmids add approx. 10-100
                                        μg DNA</LI></OL><LI>
                                Incubate cells on ice for <U>30min</U>.
                                <SPAN class="text-light-blue"><B>Note:</B> Cutting this incub time short slightly
                                    decreases transformation efficiency.
                                </SPAN></LI><LI>
                                Heat shock cells at 42°C for <U>90 sec</U>.
                                <SPAN class="text-light-blue"><B>Note:</B> Optimal heat shock duration depends on the
                                    strain and the method used to make it competent.
                                </SPAN></LI><LI>Incubate on ice for <U>5 minutes</U></LI><LI>Heat shock cells at 42°C for <U>90 sec</U></LI><LI>Incubate on ice for <U>5 minutes</U></LI><LI>                              Add 1ml of SOC or LB Broth.
                                <SPAN class="text-light-blue"><B>Note:</B> Using SOC increases transformation efficiency.
                                </SPAN></LI><LI>
                                Incubate cells for <U>1 hour</U> at 37°C in a shaking incubator
                                (200-250rpm).
                                <SPAN class="text-light-blue"><B>Note:</B> Incubation without shaking will decrease
                                    transformation efficiency
                                </SPAN><SPAN class="text-light-blue"><B>Note:</B> If using LB Broth, longer incubation up to 2
                                    hours can increase transformation Efficiency
                                </SPAN></LI><LI>Centrifuge at 8000rpm for <U>3 minutes</U></LI><LI>Remove 920μl of supernatant</LI><LI>Resuspend pelleted cells in the remaining LB broth or SOC.</LI><LI>Plate resuspended cells onto LB agar plates containing the
                                appropriate antibiotic.</LI><LI>Let the plates dry in the incubator for approx. <U>30min</U>.</LI><LI>Flip upside down and incubate for <U>16-18 hours</U>.</LI></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Growing Overnight Cultures" :dark="nightMode"><DIV><H3>Growing Overnight Cultures</H3><H4>Materials</H4><UL><LI>5-10 ml LB broth</LI><LI>5-10 μl antibiotic</LI><LI>culture tubes</LI></UL><H4>Method</H4><P>Overnight cultures are prepared under sterile conditions using a
                            Bunsen burner</P><OL><LI>Add 5-10 ml ofLB media into a culture tubes.</LI><LI>Add 5-10 μl of appropriate antibiotic into the broth.</LI><LI>Using a loop, pick a single colony and inoculate the culture by
                                dipping the loop into the LB broth.</LI><LI>Incubate overnight at 37°C shaking at 200-250 rpm.</LI></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Preparation of LB Broth, Agar, SOC and Glycerol Stocks" :dark="nightMode"><DIV><H3>LB Broth</H3><H4>Materials</H4><UL><LI>20 g LB broth (powder)</LI><LI>1 Litre Purified Water</LI></UL><H4>Method</H4><OL><LI>Add 20g LB broth to 1 litre purified water</LI><LI>Autoclave</LI></OL><H3>LB Agar</H3><H4>Materials</H4><UL><LI>35 g LB Agar (powder)</LI><LI>1 Litre Purified Water</LI></UL><H4>Method</H4><OL><LI>Add 37g LB Agar to 1 litre purified water</LI><LI>Autoclave</LI></OL><H3>SOC Medium</H3><P>We followed the <A href="https://www.qiagen.com/se/resources/faq?id=d3be05d0-ec02-4ced-aabe-dcd2a1061920&amp;lang=en&amp;Print=1">Quiagen
                                protocol</A></P><H4>Materials</H4><UL><LI>0.5% Yeast Extract</LI><LI>2% Tryptone</LI><LI>10 mM NaCl</LI><LI>2.5 mM KCl</LI><LI>10 mM MgCl2</LI><LI>10 mM MgSO4</LI><LI>20 mM Glucose*</LI></UL><H4 class="text-light-blue">Notes</H4><UL class="text-light-blue"><LI>Add Glucose after autoclaving the solution with the remaining
                                ingredients, and letting it cool down. Sterilize the final
                                solution by passing it through a 0.2 µm filter.</LI><LI>SOC medium can be stored at room temperature and is stable for
                                several years.</LI></UL><H3>Glycerol Stocks</H3><H4>Materials</H4><UL><LI>500µl glycerol (50%)</LI><LI>500µl overnight culture in LB</LI></UL><H4>Method</H4><OL><LI>Add 500µl glycerol (80%) to 1.5ml eppendorf tube</LI><LI>Add 500µl overnight culture in LB</LI><LI>Store at -80°C</LI></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="PCR From Plasmid DNA Template" :dark="nightMode"><DIV><H3>PCR From Plasmid DNA Template</H3><P>We followed the NEB PCR with hiFi Q5 master mix <A href="https://international.neb.com/protocols/2012/12/07/protocol-for-q5-high-fidelity-2x-master-mix-m0492">Protocol</A>
                            for our PCR reactions.</P><H4>Materials</H4><UL><LI>2x Q5 High-Fidelity Master Mix</LI><LI>10μΜ Forward Primer</LI><LI>10μΜ Reverse Primer</LI><LI>Template DNA</LI><LI>Nuclease Free Water</LI></UL><H5>For a 20 µL reaction:</H5><TABLE class="text-center simple-table"><TBODY><TR><TD>Q5 High-Fidelity 2X Master Mix</TD><TD>10 μl</TD></TR><TR><TD>10 µM Forward Primer</TD><TD>1 μl</TD></TR><TR><TD>10 µM Reverse Primer</TD><TD>1 μl</TD></TR><TR><TD>Template DNA</TD><TD>variable (1-5 ng)</TD></TR><TR><TD>Nuclease-Free Water</TD><TD>to 20 μl</TD></TR></TBODY></TABLE><SPAN class="text-light-blue"><B>Note: </B>Reactions are set on ice
                        </SPAN><H4>Thermocycling</H4><TABLE class="text-center simple-table"><TBODY><TR><TD>Initial Denaturation</TD><TD>95°C</TD><TD>2 minutes</TD></TR><TR><TD rowspan="3">25-35 cycles</TD><TD>95°C (Denaturation)</TD><TD>0.5-1 minute</TD></TR><TR><TD>42-65°C (Annealing)</TD><TD>0.5-1 minute</TD></TR><TR><TD>72°C (Extension)</TD><TD>1 min/kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>5 minutes</TD></TR><TR><TD>Hold</TD><TD>4–10°C</TD><TD>Indefinite</TD></TR></TBODY></TABLE><SPAN class="text-light-blue"><B>Note: </B>We used the <A href="https://tmcalculator.neb.com/#!/main">NEB
                                ™ Calculator</A> to determine the annealing temperature.
                        </SPAN></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Colony PCR" :dark="nightMode"><DIV><H3>Colony PCR</H3><P>
                            We followed the Promega GoTaq® DNA Polymeras <A href="https://worldwide.promega.com/-/media/files/resources/protocols/product-information-sheets/g/gotaq-dna-polymerase-m300.pdf?la=en">Protocol</A>
                            for our Colony PCR reactions.
                        </P><H4>Materials</H4><UL><LI>5X Green or Colorless GoTaq® Reaction Buffer1</LI><LI>GoTaq® DNA Polymerase </LI><LI>PCR Nucleotide Mix</LI><LI>Forward Primer</LI><LI>Reverse Primer</LI><LI>Template DNA</LI><LI>Nuclease Free Water</LI></UL><H5>For a 50 μl PCR Reaction:</H5><TABLE class="text-center simple-table"><TBODY><TR><TD>5X Green or Colorless GoTaq® Reaction Buffer1 </TD><TD>10 μl</TD></TR><TR><TD>PCR Nucleotide Mix, 10mM each</TD><TD>1 μl</TD></TR><TR><TD>Forward Primer</TD><TD>1 μl</TD></TR><TR><TD>Reverse Primer</TD><TD>1 μl</TD></TR><TR><TD>GoTaq® DNA Polymerase (5u/µl)</TD><TD>0,25 μl</TD></TR><TR><TD>template DNA</TD><TD>variable (1-5 ng)</TD></TR><TR><TD>Nuclease-Free Water</TD><TD>to 50 μl</TD></TR></TBODY></TABLE><H4>Method</H4><P>
                            We performed 10μl Colony PCR Reactions
                        </P><H4>Thermocycling</H4><TABLE class="text-center simple-table"><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR><TD rowspan="3">25-30 cycles</TD><TD>98°C (Denaturation)</TD><TD>10 seconds</TD></TR><TR><TD>50–72°C (Annealing)</TD><TD>30 seconds</TD></TR><TR><TD>72°C (Extension)</TD><TD>30 seconds/kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR><TD>Hold</TD><TD>4–10°C</TD><TD>Indefinite</TD></TR></TBODY></TABLE></DIV><P class="text-light-blue"><B>Notes:</B> Perform PCR using your standard parameters
                        <UL class="text-light-blue"><LI>Reactions are set on ice</LI><LI>Reactions are placed in a thermal cycler that has been
                                preheated to 95°C</LI><LI>The thermal cycling protocol has an initial denaturation step
                                where samples are heated at 95°C for 2 minutes to ensure that
                                the target</LI></UL></P></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Agarose Gel Electrophoresis" :dark="nightMode"><DIV><H3>Agarose Gel Electrophoresis</H3><H4>Equipment</H4><UL><LI>Casting tray</LI><LI>Well combs</LI><LI>Voltage source</LI><LI>Gel box</LI><LI>UV light source</LI></UL><H4>Reagents</H4><UL><LI>TAE</LI><LI>Agarose</LI><LI>Ethidium bromide (stock concentration of 10 mg/mL)</LI></UL><H4>Method</H4><H5>For a standard 1% agarose gel</H5><OL><LI>Measure 1 g of agarose and mix agarose powder with 100 mL 1xTAE
                                in a microwavable flask</LI><LI>Microwave for 1-3 min until the agarose is completely dissolved
                                and let agarose solution cool down to about 50 °C</LI><LI>Add ethidium bromide (EtBr) to a final concentration of
                                approximately 0.2-0.5 μg/mL. EtBr binds to the DNA and allows
                                you to visualize the DNA under ultraviolet (UV) light</LI><LI>Pour the agarose into a gel tray with the well comb in place</LI><LI>Place newly poured gel at 4 °C for 10-15 mins OR let sit at
                                room temperature for 20-30 mins, until it has completely
                                solidified</LI><LI>Add loading buffer to each of your DNA samples</LI><LI>Once solidified, place the agarose gel into the gel box
                                (electrophoresis unit). Fill gel box with 1xTAE (or TBE) until
                                the gel is covered</LI><LI>Carefully load a molecular weight ladder into the first lane of
                                the gel and your samples into the additional wells of the gel</LI><LI>Run the gel at 80-150 V until the dye line is approximately
                                75-80% of the way down the gel. A typical run time is about 40
                                minutes-1.5 hours, depending on the gel concentration and
                                voltage</LI><LI>Turn OFF power, disconnect the electrodes from the power
                                source, and then carefully remove the gel from the gel box</LI><LI>Using any device that has UV light, visualize your DNA
                                fragments. The fragments of DNA are usually referred to as
                                ‘bands’ due to their appearance on the gel</LI></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Restriction Digestion" :dark="nightMode"><DIV><H3>Restriction Digestion</H3><UL><LI><B>Restriction Enzyme:</B><A href="https://enzymefinder.neb.com/#!#nebheader">NEB
                                    enzyme finder</A> used to determine the appropriate
                                restriction enzymes</LI><LI><B>Reaction buffer:</B><A href="https://nebcloner.neb.com/#!/redigest">NEB
                                    double digest finder</A> used to determine an appropriate
                                buffer and protocol for the restriction enzymes used</LI></UL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Ligation" :dark="nightMode"><DIV><H3>Ligation</H3><P>We followed the NEB Ligation <A href="https://international.neb.com/Protocols/0001/01/01/dna-ligation-with-t4-dna-ligase-m0202">Protocol</A>
                            with T4 DNA Ligase.</P><H4>Materials</H4><UL><LI>Microcentrifuge Tubes</LI><LI>Ice</LI><LI>T4 DNA Ligase Buffer (10x)</LI><LI>T4 DNA Ligase</LI><LI>Vector DNA</LI><LI>Insert DNA</LI><LI>Nuclease Free Water</LI></UL><H4>Method</H4><OL><LI>
                                Set up the following reaction in a microcentrifuge tube on ice
                                <TABLE class="text-center simple-table"><TBODY><TR><TD><B>Component</B></TD><TD><B>20 μl Reaction</B></TD></TR><TR><TD>T4 DNA Ligase Buffer (10x)</TD><TD>2 μl</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 μl</TD></TR><TR><TD>Vector DNA</TD><TD>50 ng</TD></TR><TR><TD>Insert DNA</TD><TD>ng calculated using the <A href="http://nebiocalculator.neb.com/#!/ligation">NeBioCalculator</A></TD></TR><TR><TD>Nuclease Free Water</TD><TD>To 20 μl</TD></TR></TBODY></TABLE></LI><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room
                                temperature for 10 minutes. For blunt ends or single base
                                overhangs, incubate at 16°C overnight or room temperature for 2
                                hours</LI><LI>Heat inactivate at 65°C for 10 minutes</LI><LI>Chill on ice and transform 8μl of the reaction into 100 μl
                                competent cells</LI><SPAN class="text-light-blue"><B>Notes:</B></SPAN><UL><LI>T4 DNA Ligase should be added last</LI><LI>Ligation using a molar ratio of 1:3 vector to insert
                                    increases the reaction’s efficiency</LI><LI>The T4 DNA Ligase Buffer should be thawed and resuspended
                                    at room temperature</LI></UL></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Golden Gate Assembly" :dark="nightMode"><DIV><H3>Golden Gate Assembly</H3><P>
                            We followed the Golden Gate Assembly <A href="https://international.neb.com/protocols/2015/03/04/golden-gate-assembly-protocol-for-using-neb-golden-gate-assembly-mix-e1600">Protocol</A>
                            for Using NEB Golden Gate Assembly Mix
                        </P><H4>Materials</H4><UL><LI>Destination Plasmid</LI><LI>Insert/Inserts</LI><LI>NEB Golden Gate Buffer (10x)</LI><LI>NEB Golden Gate Assembly Mix</LI><LI>Nuclease-free H2O</LI></UL><H4>Method</H4><OL><LI>
                                Set up a 20 μl reaction according to the following table
                                <TABLE class="text-center simple-table"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Assembly Reaction</B></TD></TR><TR><TD>Destination Plasmid 75ng/μl</TD><TD>1 μl</TD></TR><TR><TD>
                                            Insert:
                                            </TD></TR></TBODY></TABLE></LI><LI>If Precloned</LI><LI>If in amplicon Form</LI>
                                            75-100 ng each plasmid
                                            2:1 molar ratio insert:vector
                                        NEB Golden Gate Buffer (10x)2 μlNEB Golden Gate Assembly Mix1 μlNuclease-free H2Oto 20 μl</OL><H4>Assembly Protocol</H4><TABLE class="text-center simple-table"><TBODY><TR><TD><B>Number Of Inserts</B></TD><TD><B>Assembly Protocol</B></TD></TR><TR><TD>1-4 inserts</TD><TD>37°C for 60 minutes → 55°C for 5 minutes</TD></TR><TR><TD>5-10 inserts</TD><TD>(37°C for 1 minute → 16°C for 1 minute)x30 → 55°C for 5
                                    minutes</TD></TR></TBODY></TABLE></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Minipreps, PCR Purification, Gel Extraction" :dark="nightMode"><DIV><P>For our plasmid isolation we used the Macherey Νagel NucleoSpin®
                            Plasmid isolation Kit, unless otherwise stated.
                            For PCR Purification and Gel Extraction we used the Macherey Nagel
                            NucleoSpin® Gel and PCR Clean-up Kit, unless otherwise stated.</P></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="dCas9 Toxicity Measurements" :dark="nightMode"><DIV><H3>dCas9 Toxicity Measurements</H3><P>All measurements shown regarding Abs600 were taken using plate
                            reader.</P><H5 class="text-cyan">Day 1</H5><P>
                            Inoculate one single colony of DH5a E.coli cells containing the
                            dcas9 cassette into 1 ml LB+Kanamycin in </P><P>2‐ml eppendorf tubes</P>
                            and
                            grown at 37 °C, <B>250 rpm</B> overnight in a MultitronPro shaker
                            incubator.
                        <H5 class="text-cyan">Day 2</H5><P>
                            The overnight cultures were diluted <B>1:360</B> into 1ml LB+ kanamycin in
                            <B>2‐ml eppendorf</B> tubes and doxycycline was added to the following
                            final concentration: <B>0, 0.2, 0.4, 0.6, 0.8, 1.2, 4, 8</B>.The cultures
                            were grown at 37 °C, <B>250 rpm</B> in a MultitronPro shaker incubator.
                            <B>After 6 hours</B> measurements for Abs600 were performed using plate
                            reader
                        </P></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Flow Cytometry Sample Preparation" :dark="nightMode"><DIV><H3>Flow Cytometry Sample Preparation</H3><B>We followed the protocol from Adam Mayer et al [1].</B><P>All measurements shown were taken by cytometry of cells in mid-log
                            growth except when noted.</P><H5>Day1</H5><P>Glycerol stocks of strains containing the plasmids of interest were
                            streaked on LB + 1.5% Agar plates and grown overnight at 37 °C</P><H5>Day2</H5><P>Single colonies were inoculated into 1 ml LB + antibiotics in <B>2‐ml
                                eppendorf tubes</B> and grown at 37 °C, <B>250 rpm</B>
                            overnight in a MultitronPro shaker incubator</P><H5>Day3</H5><P>
                            The overnight growths were diluted 1:200 into 1 ml LB + antibiotics
                            in <B>2‐ml eppendorf tubes</B> and grown at 37 °C, <B>250 rpm</B>.
                            
                            After 2 hours the growths were diluted 1:500 into prewarmed LB +
                            antibiotics + inducer where necessary in <B>2‐ml eppendorf tubes</B>
                            and grown at 37 °C, <B>250 rpm</B> for 5 hours.
                            
                            After growth, 20 μl of culture sample was diluted into 180 μl PBS +
                            200 μg/ml kanamycin to inhibit translation. The samples were stored
                            at 4°C for 1 hour and then measurements were performed using the
                            CyFlow Cube8 Sysmex Partec Flow Cytometer.
                        </P><SMALL>[1] Meyer, A. J., Segall-Shapiro, T. H., &amp; Voigt, C. A. (2018).
                            Marionette: E. coli containing 12 highly-optimized small molecule
                            sensors. BioRxiv. Retrieved from
                            http://biorxiv.org/content/early/2018/04/10/285866.abstract</SMALL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Plate Reader Sample Preparation for our Interlaboratory Collaboration" :dark="nightMode"><DIV><H3>Plate Reader Sample Preparation for our Interlaboratory
                            Collaboration</H3><P>We followed the iGEM 2018 InterLab Study <A href="https://static.igem.org/mediawiki/2018/0/09/2018_InterLab_Plate_Reader_Protocol.pdf">Protocol</A></P></DIV></Q-COLLAPSIBLE></Q-LIST></DIV><Q-PARALLAX src="https://static.igem.org/mediawiki/2018/thumb/e/eb/T--Thessaloniki--lab--photo-3.jpeg/800px-T--Thessaloniki--lab--photo-3.jpeg"><H1 slot="loading">Primers</H1><H1>Primers</H1></Q-PARALLAX><DIV class="row justify-center"><DIV class="col-xs-12 col-sm-11 col-md-10 col-lg-8 q-pa-md" :class="rM('text-dark', 'text-light')"><DIV style="overflow-x:auto;"><TABLE class="simple-table text-center"><TBODY><TR class="bg-light text-black"><TD class="q-pa-md"><B>Primers Name</B></TD><TD><B>Sequence</B></TD><TD><B>Usage</B></TD></TR><TR><TD>VR</TD><TD style="text-transform: uppercase;">attaccgcctttgagtgagc</TD><TD><UL><LI>Amplification of IDT gBlocks</LI><LI>Colony PCR</LI></UL></TD></TR><TR><TD>VF2</TD><TD style="text-transform: uppercase;">tgccacctgacgtctaagaa</TD><TD><UL><LI>Amplification of IDT gBlocks</LI><LI>Colony PCR</LI></UL></TD></TR><TR><TD>E Vector</TD><TD style="text-transform: uppercase;">belongs to the Metabrick Platform</TD><TD><UL><LI>Vector Amplification for insertion of  BsaI sites in the Prefix and Suffix</LI></UL></TD></TR><TR><TD>P Vector</TD><TD style="text-transform: uppercase;">belongs to the Metabrick Platform</TD><TD><UL><LI>Vector Amplification for insertion of  BsaI sites in the Prefix and Suffix</LI></UL></TD></TR><TR><TD>E Primer</TD><TD style="text-transform: uppercase;">belongs to the Metabrick Platform</TD><TD><UL><LI>Part Amplification for insertion of BsaI sites in the Prefix</LI></UL></TD></TR><TR><TD>P Primer</TD><TD style="text-transform: uppercase;">belongs to the Metabrick Platform</TD><TD><UL><LI>Part Amplification for insertion of BsaI sites in the Suffix</LI></UL></TD></TR><TR><TD>X Primer</TD><TD style="text-transform: uppercase;">belongs to the Metabrick Platform</TD><TD><UL><LI>Part Amplification for insertion of BsaI sites in the Suffix</LI></UL></TD></TR><TR><TD>S Primer</TD><TD style="text-transform: uppercase;">belongs to the Metabrick Platform</TD><TD><UL><LI>Part Amplification for insertion of BsaI sites in the Prefix</LI></UL></TD></TR><TR><TD>P.Rem 1</TD><TD style="text-transform: uppercase;">aGGTCTCcgcagcaccctgctgcccttg</TD><TD><UL><LI>Removal of PstI illegal Site from the Addgene Plasmids pTHSSe_59 &amp; pTHSSe_60 (Insertion Point 50)</LI></UL></TD></TR><TR><TD>P.Rem 4</TD><TD style="text-transform: uppercase;">aGGTCTCcgtgtctcgacataccaaattgagtcatgg</TD><TD><UL><LI>Removal of PstI illegal Site from the Addgene Plasmids pTHSSe_59 &amp; pTHSSe_60 (Insertion Point 50)</LI></UL></TD></TR><TR><TD>P.Rem 2</TD><TD style="text-transform: uppercase;">GGTCTCgctgcgataaggctacgatgtgg</TD><TD><UL><LI>Removal of PstI illegal Site from the Addgene Plasmids pTHSSe_59 &amp; pTHSSe_60  (Insertion Point 802)</LI></UL></TD></TR><TR><TD>P.Rem 3</TD><TD style="text-transform: uppercase;">GGTCTCgacacggactgctgcaactctttcgtag</TD><TD><UL><LI>Removal of PstI illegal Site from the Addgene Plasmids pTHSSe_59 &amp; pTHSSe_60 (Insertion Point 802)</LI></UL></TD></TR><TR><TD>In 1</TD><TD style="text-transform: uppercase;">AGGTCTCAGGAActtcagccaaaaaacttaagacc</TD><TD><UL><LI>Amplification of Test Device 1 and Test Device 2 from pTHSSe_59 &amp; pTHSSe_60, used for our Interlaboratory Study</LI></UL></TD></TR><TR><TD>In 2</TD><TD style="text-transform: uppercase;">AGGTCTCAACTgacatagcgtataaacgcaga</TD><TD><UL><LI>Amplification of Test Device 1 and Test Device 2 from pTHSSe_59 &amp; pTHSSe_60, used for our Interlaboratory Study</LI></UL></TD></TR><TR><TD>Ri 1</TD><TD style="text-transform: uppercase;">AGGTCTCATTCCTGGCTGAAACCTATTATGACG</TD><TD><UL><LI>Amplification of pTHSSe_59 &amp; pTHSSe_60 vectors for our TALEsp1 and TALEsp2 stabilised Riboswitch Constructs.</LI></UL></TD></TR><TR><TD>Ri 2</TD><TD style="text-transform: uppercase;">AGGTCTCACAGTCGCTATGTCTTCTACTAGTAGC</TD><TD><UL><LI>Amplification of pTHSSe_59 &amp; pTHSSe_60 vectors for our TALEsp1 and TALEsp2 stabilised Riboswitch Constructs.</LI></UL></TD></TR><TR><TD>DC1</TD><TD style="text-transform: uppercase;">AGGTCTCAGGAAgctttactccaccgttgg</TD><TD><UL><LI>Amplification of pAN-PTet-dCas9 plasmid to make the insert RFC10 compatible</LI></UL></TD></TR><TR><TD>DC2</TD><TD style="text-transform: uppercase;">AGGTCTCAACTGcctcagataaaatatttgctcatgagc</TD><TD><UL><LI>Amplification of pAN-PTet-dCas9 plasmid to make the insert RFC10 compatible</LI></UL></TD></TR><TR><TD>Vi1</TD><TD style="text-transform: uppercase;">AGGTCTCATTCCGCTCCGTGTCTTCCTC</TD><TD><UL><LI>Amplification of pTHSSe_59 &amp; pTHSSe_60 vectors for insertion of a BsaI site in the prefix</LI></UL></TD></TR><TR><TD>AG1</TD><TD style="text-transform: uppercase;">AGGTCTCAATGTCCAGATTAGATAAAAGTAAAGTG</TD><TD><UL><LI>Amplification of the ANDgate Plasmid to replace the J23114 Promoter with a Talesp1 psp1w1 promoter for the TetR expression</LI></UL></TD></TR><TR><TD>AG2</TD><TD style="text-transform: uppercase;">AGGTCTCATTCCGGCTGAACTCTAGAAGC</TD><TD><UL><LI>Amplification of the ANDgate Plasmid to replace the J23114 Promoter with a Talesp1 psp1w1 promoter for the TetR expression</LI></UL></TD></TR><TR><TD>YG1</TD><TD style="text-transform: uppercase;">aggtctcaggaaGAAGACACGGAGCTCG</TD><TD><UL><LI>Amplification of pTHSSe_59 psp1w1  to replace the J23114 Promoter with a Talesp1 psp1w1 promoter for the TetR expression</LI></UL></TD></TR><TR><TD>YG2 (30)</TD><TD style="text-transform: uppercase;">aggtctcaACATATGGTATTTCTCCTCtttAAtttaaaca</TD><TD><UL><LI>Amplification of pTHSSe_59 to replace the J23114 Promoter with a Talesp1 psp1w1 promoter for the TetR expression</LI></UL></TD></TR><TR><TD>YG2 (32)</TD><TD style="text-transform: uppercase;">aggtctcaACATCTAGTACTTTCCTGTGTGA</TD><TD><UL><LI>Amplification of pTHSSe_59 to replace the J23114 Promoter with a Talesp1 psp1w1 promoter for the TetR expression</LI></UL></TD></TR><TR><TD>IA1</TD><TD style="text-transform: uppercase;">aggtctcactttTTAAACAAAATTATTTGTAGAGGCTGTTTCGtc</TD><TD><UL><LI>Amplification of pTHSSe_59 psp1w1 vector  for insertion of  BsaI sites in the Prefix</LI></UL></TD></TR><TR><TD>IA2</TD><TD style="text-transform: uppercase;">aggtctcaTACTAGAGCCAGGCATCAAATAAAACG</TD><TD><UL><LI>Amplification of pTHSSe_59 psp1w1 vector  for insertion of  BsaI sites in the Suffix</LI><LI>Amplification of pSB1C3 and pTHSSe_59 for the Insertion of constitutively expressed sgRNA cassette.</LI></UL></TD></TR><TR><TD>IG1</TD><TD style="text-transform: uppercase;">aggtctcatcaaGGCTGTAAGAATCCTATAGGTTGAG</TD><TD><UL><LI>Amplification of pSB1C3 and pTHSSe_59 for the Insertion of constitutively expressed sgRNA cassette.</LI></UL></TD></TR><TR><TD>GG1</TD><TD style="text-transform: uppercase;">aggtctcattgacagctagctcagtcctGTTTTAGAG</TD><TD><UL><LI>Amplification of pRha sgRNA cassette in order to replace pRha with psp1w1</LI></UL></TD></TR><TR><TD>GG2</TD><TD style="text-transform: uppercase;">aggtctcaagtataTTATTTGTAGAGCTCATCCATGCCATG</TD><TD><UL><LI>Amplification of pRha sgRNA cassette in order to replace pRha with psp1w1</LI></UL></TD></TR><TR><TD>GK1</TD><TD style="text-transform: uppercase;">aggtctctaaagaggagaaatactagATGAGCAAAG</TD><TD><UL><LI>Amplification of B0034-sfGFP for the incorporation of BsaI restriction sites.</LI></UL></TD></TR><TR><TD>GK2</TD><TD style="text-transform: uppercase;">aggtctcaagtaTTATTTGTAGAGCTCATCCATGC</TD><TD><UL><LI>Amplification of B0034-sfGFP for the incorporation of BsaI restriction sites.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></DIV><Q-PARALLAX src="https://static.igem.org/mediawiki/2018/thumb/e/eb/T--Thessaloniki--lab--photo-3.jpeg/800px-T--Thessaloniki--lab--photo-3.jpeg"><H1 slot="loading">Clonning Strategy</H1><H1>Cloning Strategy</H1></Q-PARALLAX><DIV class="row justify-center q-mb-md q-mt-md" :class="rM('text-dark', 'text-light')"><DIV class="col-xs-12 col-sm-11 col-md-10 col-lg-8 q-pa-md"><H4>TALE stabilized Promoters</H4><Q-LIST separator="" striped="" highlight="" no-border=""><Q-COLLAPSIBLE label="TALEsp1 / TALEsp2 - Stabilized Promoters" :dark="nightMode" color="red"><DIV><H4>TALEsp1 / TALEsp2 - Stabilized Promoters</H4><P>
                                TALEsp1-Pupsp1 stabilized promoter /TALEsp2-Pupsp2 stabilized
                                promoter
                                
                                TALEsp1 / TALEsp2 - Stabilized Promoters
                            </P><H5>Steps</H5><OL><LI>
                                    PCR amplification of psb1c3 vector with a set of
                                    standardized primers.
                                    <P>
                                        These primers belong to the Metabrick Platform [1] and
                                        incorporate <B>BsaI</B> restriction sites to the
                                        biobrick
                                        prefix and suffix.
                                    </P></LI><LI>
                                    PCR amplification of <B>pTHSSe_59</B> and <B>pTHSSe_60</B>
                                    plasmid with E,P standardized primers.

                                    <P>
                                        These primers belong to the Metabrick platform and
                                        incorporate <B>BsaI</B> restriction sites to the
                                        plasmid’s
                                        biobrick prefix and suffix.
                                    </P></LI><LI>
                                    Golden Gate assembly of the PCR amplified products.
                                    <P>
                                        The amplified products after restriction digestion with
                                        bsaI restriction enzyme, acquire complementary sticky
                                        ends
                                        and can be ligated together forming the <B>TALEsp1-Pupsp1-pSB1C3</B>,
                                        <B>TALEsp2-Pupsp2-pSB1C3</B> plasmids.
                                    </P></LI><LI>
                                    Restriction digestion with <B>EcoRI</B> and <B>PstI</B>,
                                    enzymes that are unique to prefix and suffix.
                                    <P>
                                        Since the assembled <B>Talesp1-Pupsp1</B> / <B>Talesp2-Pupsp2</B>
                                        constructs are <B>RFC[10] compatible</B>, they can be
                                        easily assembled with other vectors. This can be
                                        achieved
                                        by cutting the <B>Talesp1-Pupsp1-pSB1C3</B> vector and
                                        the
                                        receiver vectors with two biobrick enzymes one of which
                                        having it’s restriction site to prefix and the other to
                                        suffix.
                                    </P></LI><LI>
                                    Ligation between the digested pieces.
                                </LI></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Psp1w1 promoter characterization device" :dark="nightMode" color="red"><DIV><H4>Psp1w1 promoter characterization device</H4><P>
                                The Psp1w1 promoter characterization device emerged from the
                                assembly of three individual parts: <B>Psp1w1</B>, <B>sfGFP
                                    (without
                                    starting codon)</B> and <B>amplified pSB1C3</B> vector.
                            </P><H5>Steps</H5><OL><LI>
                                    PCR amplification of <B>pSB1C3</B> vector with a set of
                                    standardized primers.
                                </LI><LI>
                                    Design and synthesis of the <B>Psp1w1</B> and <B>sfGFP
                                        (without
                                        starting codon)</B> with the appropriate flanking
                                    regions.
                                </LI><LI>
                                    Golden Gate assembly between <B>Psp1w1</B>, <B>sfGFP
                                        (without starting codon)</B> and the <B>amplified
                                        pSB1C3</B>
                                    vector.
                                    
                                    This reaction forms the final plasmid <B>Psp1w1-pSB1C3</B>
                                    containing the Psp1w1 promoter characterization device.
                                </LI></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Talesp1-Psp1w1 stabilized promoter" :dark="nightMode" color="red"><DIV><H4>Talesp1-Psp1w1 stabilized promoter</H4><P>The following procedure has been followed in order to replace
                                the Pupsp1 stabilized promoter with the Psp1w1 stabilized
                                promoter .The expression levels of the sfGFP driven by the
                                Psp1w1 promoter are lower than the sfGFP expression levels
                                driven by the Pupsp1 promoter.</P><H5>Steps</H5><OL><LI>
                                    PCR amplification of <B>Psp1w1-pSB1C3</B> plasmid with <B>In1</B>,
                                    <B>In2</B> primers.
                                    <P>These primers amplify the Psp1w1 promoter
                                        characterization device and incorporate <B>BsaI</B>
                                        sites flanking the amplified device.</P></LI><LI>                      PCR amplification of <B>TALEsp1-pSB1C3</B> vector with <B>Ri1</B>,
                                    <B>Ri2</B> primers.
                                    <P>These primers amplify the TALEsp1 cassette together with
                                        the plasmid backbone and incorporate BsaI restriction
                                        sites at the external regions of the amplified
                                        sequence.</P><P>Restriction digestion with BsaI restriction enzyme of
                                        the previously amplified sequences creates sticky
                                        complementary ends and thus they can be assembled
                                        together.</P></LI><LI>
                                    Restriction digestion with BsaI restriction enzyme of the
                                    previously amplified sequences creates sticky complementary
                                    ends and thus they can be assembled together.
                                    <P>This reaction forms the final plasmid <B>TALEsp1-Psp1w1-pSB1C3</B>
                                        plasmid containing the Psp1w1 TALE stabilized promoter.</P></LI></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="Riboswitches" :dark="nightMode" color="red"><DIV><H4>Riboswitches</H4><P>
                                The final plasmids for the characterization of the theophylline
                                riboswitches consist of three individual parts:
                                <B>A(Pupsp1 promoter fused with riboswitch), B(sfGFP marker)
                                    and C.</B>
                                Each part’s sequence is flanked by BsaI recognition sites and
                                after digestion with BsaI restriction enzyme, part A has sticky
                                ends with part B and part C, while part B has sticky ends with
                                part A and part C.
                            </P><H5>Steps</H5><OL><LI>
                                    PCR amplification of <B>psb1c3 vector</B> with a set of
                                    standardized primers.
                                    <P>
                                        These primers amplify the plasmid backbone without the
                                        insert, incorporate BsaI sites at prefix and suffix,
                                        forming the <B>amplified pSB1C3 vector (part C)</B>.
                                    </P></LI><LI>
                                    Design and synthesis of the <B>part A</B>, <B>part B</B>
                                    fragments with the appropriate flanking regions.
                                </LI><LI>
                                    Golden Gate assembly between <B>part A</B>, <B>part B</B>
                                    and the
                                    <B>amplified pSB1C3 Vector</B> (part C). This reaction
                                    forms the
                                    final plasmids <B>Ribo12.1-pSB1C3</B> and <B>RiboT27-pSB1C3</B>
                                    containing the riboswitch characterization device.
                                </LI></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="TALEsp1 stabilized promoters with translational control" :dark="nightMode" color="red"><DIV><H4>TALEsp1 stabilized promoters with translational control.</H4><H5>Steps</H5><OL><LI>
                                    PCR amplification of <B>RiboT27-pSB1C3</B> plasmid with <B>In1</B>,
                                    <B>In2</B>
                                    primers.
                                    <P>
                                        These primers amplify the riboswitch characterization
                                        device and incorporate <B>BsaI</B> sites at the
                                        external
                                        regions of the amplified device.
                                    </P></LI><LI>
                                    PCR amplification of <B>pTHSSe_59</B> plasmid with <B>Ri1</B>,
                                    <B>Ri2</B> primers.
                                    <P>
                                        These primers amplify the TALEsp1 cassette together
                                        with the plasmid backbone and incorporate <B>BsaI</B>
                                        restriction sites flanking the amplified sequence.
                                    </P><P>
                                        Restriction digestion with <B>BsaI</B> restriction
                                        enzyme of the previously amplified sequences, creates
                                        sticky complementary ends and thus they can be
                                        assembled together.
                                    </P></LI><LI>
                                    Golden Gate assembly reaction between the amplified
                                    products from step 1 and step 2.
 <P>
                                        This reaction forms the <B>TALEsp1-Pupsp1-RT27-pSB1C3</B>
                                        plasmid containing the theophylline inducible TALE
                                        stabilized promoter.
                                    </P></LI></OL></DIV></Q-COLLAPSIBLE></Q-LIST><H4>CRISPRi system</H4><P>Constructs used to test dcas9/sgrna repression.</P><Q-LIST separator="" striped="" highlight="" no-border=""><Q-COLLAPSIBLE label="pRha-sgRNA Cassette" :dark="nightMode" color="red"><DIV><H4>pRha-sgRNA Cassette</H4><P>
                                pRha-sgRNA Cassette test construct
                            </P><H5>Steps</H5><OL><LI>
                                    PCR amplification of <B>pSB1C3</B> vector and <B>pTHSSe_59</B>
                                    plasmid with a set of standardized primers.
                                    <P>
                                        These primers belong to the Metabrick
                                        Platform,
                                        amplify the plasmid backbones without the
                                        insert and incorporate <B>BsaI</B> sites at prefix and
                                        suffix,
                                        forming the <B>amplified pSB1C3</B> and <B>amplified
                                            pTHSSe_59</B>
                                        vectors.
                                    </P></LI><LI>
                                    Design and synthesis of part 1 <B>(sgRNA construct)</B> and
                                    part 2 <B>(sfGFP construct)</B>.
                                    <P>
                                        Each part’s sequence is flanked by BsaI recognition
                                        sites and after digestion with BsaI restriction enzyme,
                                        part 1 has sticky ends with part 2 and the <B>amplified
                                            pSB1C3</B> or <B>amplified pTHSSe_59</B> vector,
                                        while part 2 has
                                        sticky ends with part 1 and the amplified vectors.
                                    </P></LI><LI>
                                    Golden gate Assembly between part 1, part2 and one of the
                                    <B>amplified pSB1C3</B> or <B>amplified pTHSSe_59</B>
                                    vector.
                                    <P>

                                        Depending on the amplified vector we chose, this
                                        reaction leads to the formation of the
                                        <B>sgRNA-sfGFP-pSB1C3</B> or the <B>sgRNA-sfGFP-pTHSSe_59</B>
                                        plasmid.
                                    </P><P>
                                        We transformed DH5a E.coli competent cells with
                                        <B>sgRNA-sfGFP-pTHSSe_59</B> vector along with <B>dCas9-pSB3K3</B>
                                        vector in order to test the response function for the
                                        repression of the <B>J23104</B> promoter by the
                                        sgRNA/dCas9
                                        complex. This procedure is thoroughly described at the
                                        characterization page.
        </P></LI></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="CRISPRi stabilized promoter" :dark="nightMode" color="red"><DIV><H4>CRISPRi stabilized promoter</H4><P>
                                After testing the response function of the CRISPRi system we
                                replaced the rhamnose inducible promoter that is located
                                upstream of the sgRNA cassette with the constitutive Psp1w1
                                promoter.
                            </P><H5>Steps</H5><OL><LI>
                                    Digestion with biobrick enzymes of the Psp1w1 promoter
                                    characterization device and ligation with digested <B>pTHSSe_59</B>
                                    vector.
                                    <P>
                                        Since the Psp1w1 promoter characterization device is
                                        compatible with Biobrick <B>RFC[10]</B>, it can be
                                        easily
                                        assembled with other <B>RFC[10]</B> compatible vectors.
                                        This
                                        requires digestion of the characterization device and
                                        the receiver vector with two enzymes having their
                                        restriction sites on the prefix and suffix, followed by
                                        ligation.
                                    </P><P>
                                        These reactions forms the final plasmid <B>psp1w1-pTHSSe_59</B>                       containing the Psp1w1 promoter characterization device.
                                    </P></LI><LI>
                                    PCR amplification of the <B>psp1w1-pTHSSe_59</B> plasmid
                                    with <B>IG1</B>, <B>IA2</B> primers.
                                    <P>
                                        These primers amplify the psp1w1 promoter sequence
                                        together with the pTHSSe_59 plasmid backbone and
                                        incorporate <B>BsaI</B> restriction sites flanking the
                                        amplified sequence.
                                    </P></LI><LI>
                                    PCR amplification of the <B>sgRNA-sfGFP-1c3</B> plasmid
                                    with <B>GG1</B>, <B>GG2</B> primers.
                                    <P>
                                        These primers amplify the sgrna cassette along with the
                                        stabilized <B>J23104</B> promoter cassette and
                                        incorporate
                                        <B>BsaI</B> restriction sites at the external regions
                                        of the
                                        amplified sequence.
                                    </P></LI><LI>
                                    Golden gate assembly of the amplified products.
                                    <P>
                                        After digestion with <B>BsaI</B> restriction enzyme,the
                                        amplified sequences acquire complementary sticky ends
                                        and thus they can be assembled together forming the
                                 <B>Psp1w1-CRISPRi-stabilized promoter-pTHSSe_59</B>
                                        vector.
                                    </P></LI><LI>
                                    Restriction digestion with two enzymes that are unique to
                                    prefix and suffix.
                                    <P>
                                        Since the assembled <B>CRISPRi-stabilized-Psp1w1</B>
                                        construct
                                        is compatible with Biobrick <B>RFC[10]</B>, it can be
                                        easily
                                        assembled with other vectors. This can be achieved by
                                        cutting the <B>Psp1w1-sgRNA-stabilized
                                            promoter-pTHSSe_59</B>
                                        vector and the receiver vectors with two enzymes having
                                        their restriction sites on the prefix and
                                        suffix,followed by ligation.
                                    </P></LI></OL></DIV></Q-COLLAPSIBLE><Q-COLLAPSIBLE label="dCas9-tetR expression cassette" :dark="nightMode" color="red"><DIV><H4>dCas9-tetR expression cassette</H4><P>
                                The dcas9-tetR expression cassette is located at the
                                pAN-Ptet-dcas9 plasmid. This plasmid was obtained from Addgene
                                and is not compatible with <B>RFC[10]</B> biobrick standard,
                                since it
                                lacks prefix and suffix. Furthermore there is one recognition
                                site of the <B>EcoRI</B> enzyme in the coding sequence of the
                                dCas9.
                                Taking all the above into consideration, we decided to modify
                                this plasmid in order to make it compatible with the <B>RFC[10]</B>
                                biobrick standard.
                            </P><H5>Steps</H5><OL><LI>
                                    PCR amplification of <B>TALEsp1-pSB1C3</B> plasmid with <B>Vi1</B>,
                                    <B>Ri2</B> primers.
                                    <P>
                                        These primers amplify pSB1C3 along with prefix, suffix
                                        and incorporate <B>BsaI</B> restriction sites and the
                                        external regions of the amplified sequence.
                                    </P></LI><LI>
                                    PCR amplification of <B>pAN-Ptet-dCas9</B> plasmid with <B>DC1</B>,
                                    <B>DC2</B> primers.
                                    <P>
                                        These primers amplify the <B>dCas9-tetR-lacI</B>
                                        expression cassette and add <B>BsaI</B> restriction
                                        sites flanking the amplified sequence.
                                    </P></LI><LI>
                                    Golden gate assembly of the amplified products.
                                    <P>
                                        After digestion with BsaI restriction enzyme,the
                                        amplified sequences acquire complementary sticky ends
                                        and thus they can be assemble together forming the
                                        <B>pAN-Ptet-dCas9-prefsuf</B> plasmid.
                                    </P></LI><LI>
                                    PCR amplification of pAN-Ptet-dcas-prefsuf plasmid with <B>dcm1</B>
                                    and <B>dcm2</B> primers and sequential Golden Gate
                                    assembly.
                                    <P>
                                        These primers amplify the <B>dcas9-tetR-lacI</B>
                                        expression
                                        cassette along with the vector backbone and incorporate
                                        <B>BsaI</B> sites at the amplified sequence. After
                                        digestion
                                        with <B>BsaI</B>, followed by ligation this sequence
                                        can be
                                        circularized forming the initial plasmid with a single
                                        mutation at the EcorI recognition site.
                                    </P></LI></OL></DIV></Q-COLLAPSIBLE></Q-LIST><H4>AND GATE</H4><Q-LIST separator="" striped="" highlight="" no-border=""><Q-COLLAPSIBLE label="AND Gate-pSB4K5 &amp; AND Gate-pSB3K3" :dark="nightMode" color="red"><DIV><H4>AND Gate-pSB4K5 &amp; AND Gate-pSB3K3</H4><P>
                                The following procedure has been followed in order to
                                incorporate the AND Gate circuit at <B>pSB4K5</B> and <B>pSB3K3</B>
                                plasmids.
                                Then, we evaluated the circuit’s behaviour at these plasmids of
                                different copy number.
                            </P><H5>Steps</H5><OL><LI><P>Design and synthesis of the <B>part A</B>, <B>part B</B>
                                        fragments with the appropriate flanking regions.</P></LI><LI>
                                    PCR amplification of <B>pSB1C3</B> vector with a set of
                                    standardized primers.
                                    <P>
                                        These primers belong to the Metabrick Platform and
                                        incorporate <B>BsaI</B> restriction sites to the
                                        biobrick prefix and suffix.
                                    </P></LI><LI><P>
                                        Golden Gate assembly reaction between <B>part A</B>,<B>part
                                            B</B> and
                                        the amplified <B>pSB1C3</B> vector. This reaction forms
                                        the
                                        final plasmids <B>AND Gate-pSB1C3</B> containing the
                                        AND gate
                                        circuit.
                                    </P></LI><LI>
                                    Restriction digestion with <B>EcoRI</B> and <B>SpeI</B>
                                    (enzymes unique to prefix and suffix)
                                    <P>
                                        Since the assembled AND gate construct is compatible
                                        with <B>Biobrick RFC[10]</B>, it can be easily
                                        assembled with
                                        other vectors. This can be achieved by cutting the <B>AND
                                            Gate-pSB1C3</B> plasmid and the receiver vectors
                                        with two
                                        enzymes having their restriction sites on the prefix
                                        and suffix. Following this approach we digested and
                                        assembled the AND gate circuit with <B>pSB4K5</B> and
                                        <B>pSB3K3</B>
                                        vector, constructing the <B>AND Gate-pSB4K5</B> and <B>AND
                                            Gate-pSB3K3 plasmids</B>.
                                    </P></LI></OL></DIV></Q-COLLAPSIBLE></Q-LIST></DIV></DIV><DIV class="row justify-center q-mb-md q-mt-md" :class="rM('text-dark', 'text-light')"><DIV class="col-xs-12 col-sm-11 col-md-10 col-lg-8 q-pa-md"><SMALL class="row">
                    [1] Damalas, S. (2017, June 9). The MetaBrick platform for DNA manipulation
                    and standardization. Bridging Synthetic Biology standards for optimized
                    interoperability. <A href="https://doi.org/10.17605/OSF.IO/8DHMU">https://doi.org/10.17605/OSF.IO/8DHMU</A></SMALL></DIV></DIV></Q-PAGE></IGEM-LAYOUT></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>