<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NAU-CHINA_Experiment skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NAU-CHINA/Experiment</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><UL id="gn-menu" class="gn-menu-main"><LI><A href="https://2016.igem.org/Team:NAU-CHINA"><SPAN> HOME</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Participator"> TEAM</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Description"> PROJECT</A></LI><UL><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Experiment">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Demonstrate">Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Model">Model </A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Notebook">Notebook </A></LI></UL><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Part_Collection"> PARTS</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Safety"> SAFETY</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Attributions"> ATTRIBUTION</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Human_Practices"> HP</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Conclusion">CONCLUSION</A></LI></UL></DIV></DIV><NAV><DIV class="nav" id="main" role="main"><UL class="menu"><LI><A href="https://2016.igem.org/Team:NAU-CHINA">HOME</A></LI><LI><A>TEAM</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Participator">Participator</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Collaborations">Collaboration</A></LI></UL><LI><A>PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Experiment">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Demonstrate">Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Model">Model </A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Notebook">Notebook </A></LI><LI><A>PARTS</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Part_List">Part List</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Part_Collection">Collection</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Safety">SAFETY</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Attributions">ATTRIBUTION</A></LI><LI><A>HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Conclusion">CONCLUSION</A></LI></DIV></NAV><DIV id="maintext"><DIV><H1>Experiment</H1><H3>Fig. Degradation of  3-phenoxybenzoate, 3-hydroxybenzoate, Catechol</H3><UL><LI>Degradation of  3-phenoxybenzoate <B style="color:blue;"><A href="https://2016.igem.org/Team:NAU-CHINA/Experiment/3PBA"> for more details...</A></B></LI><LI>Degradation of  3-hydroxybenzoate <B style="color: blue;"><A href="https://2016.igem.org/Team:NAU-CHINA/Experiment/3HBA"> for more details...</A></B></LI><LI>Degradation of  Catechol <B style="color: blue;"><A href="https://2016.igem.org/Team:NAU-CHINA/Experiment/CAT"> for more details...</A></B></LI><LI>Promoter activity assay
		 			</LI><LI>copA promoter <B style="color:blue;"><A href="https://2016.igem.org/Team:NAU-CHINA/Experiment/COPA"> for more details...</A></B></LI><LI>mhbpt <B style="color:blue;"><A href="https://2016.igem.org/Team:NAU-CHINA/Experiment/MHBPT"> for more details...</A></B></LI></UL><H1>Protocol(Click the list for details.)</H1><UL><LI id="LI_HPLC"><B style="color:blue;">HPLC</B></LI><DIV id="HPLC"><H2>Material:</H2><H2>Instrument:</H2><P>DIONEX UltiMate 3000 UHPLC , Buchner funnel , Vacuum suction machine  ,Organic membrane  , 1000ml cylinder , 100ml cylinder ,  ultrasonic oscillator</P><H2>Reagent:</H2><P>Methanol(chromatographic pure)   acetonitrile(chromatographic pure)   acetic acid(chromatographic pure)   Ultra pure water</P><H2>Steps for preparing mobile phase:</H2><P>Step one: Pour 1000ml of acetonitrile into a 1000ml screw mouth bottle. Pouring 990ml of water and 10ml of acetic acid into a 1000ml screw mouth bottle.</P><P>Step two:  Oscillating bottle up and down.</P><P>Step three: Use 50ml of mobile phase to rinse the buchner funnel. (Organic waste liquid is poured into the waste liquid recovery barrel)</P><P>Step four: Use buchner funnel and organic membrane to filter the mobile phase.</P><P>Step five: Use ultrasonic oscillator to oscillat the bottle to remove the gas in the mobile phase.</P><P>Step six: Store the mobile phase at room temperature for using.</P><H2>Steps for extracting sample:</H2><P>Step one: Open the instrument before opening the computer.</P><P>Step two: Loosen instrument valve.</P><P>Step three: Use 50% of acetonitrile rinsing the chromatographic column for about 30 minutes until there is no impurity to interfere baseline.</P><P>Step four: Close the valve.</P><P>Step five: Monitore baseline.</P><P>Step six: Create method file and sequence file.</P><P>Step seven: Start sampling when the baseline is flat.(When testing 3-pba, we use light of 240nm and 280nm. The current speed is 0.8 ml/min.)</P><P>Step eight: Rinse the column with 100% acetonitrile for at least 40 minutes.</P><P>Step nine: Turn off the device in reverse order when the samples have all be detected.</P><P>Step ten: Cleaning waste liquid.</P></DIV><LI id="LI_OSD"><B style="color: blue">One –step site –directed mutagenesis</B></LI><DIV id="OSD"><P><B>Introduction:</B> Site-directed mutagenesis is a molecular biology method that is used to make specific and intentional changes to the DNA sequence of a gene and any gene products.</P><P>The limitation of restriction sites in cassette mutagenesis may be overcome using polymerase chain reaction with oligonucleotide &quot;primers&quot;.</P><P>Gene mhbR:</P><H2>Diagrammatic presentation:</H2><H2>Procedure of One-step site-directed mutagenesis:</H2><P>First RCR:</P><P>1.DNA ladder M(238bp) acquirement:</P><P>PCR reaction system (50uL):</P><TABLE><TBODY><TR><TH>Reagents Name:</TH><TH>Dosage:</TH></TR><TR><TD>ddH<SUB>2</SUB>0</TD><TD>17uL</TD></TR><TR><TD>Primer M_ F</TD><TD>3uL</TD></TR><TR><TD>Primer Tubian_R</TD><TD>3uL</TD></TR><TR><TD>Template(mhbR M+P)</TD><TD>2uL</TD></TR><TR><TD>Premix Taq (Ex Taq Version 2.0, from Takara)</TD><TD>25uL</TD></TR></TBODY></TABLE><P>PCR program:(cycle 30)</P><TABLE><TBODY><TR><TD>95°C</TD><TD>5min</TD></TR><TR><TD>98°C</TD><TD>10sec</TD></TR><TR><TD>61°C</TD><TD>20sec</TD></TR><TR><TD>72°C</TD><TD>30sec</TD></TR><TR><TD>72°C</TD><TD>30sec</TD></TR><TR><TD>16°C</TD><TD>5min</TD></TR></TBODY></TABLE><P>Agarose gel electrophoresis: 1xTAE, 1%,120V,150mA.</P><P>Gel Extraction (Gel Extraction Kit from OMEGA bio-tec)</P><P>The DNA product is the ladder we want.</P></DIV><LI id="LI_TF"><B style="color: blue;">Transformation</B></LI><DIV id="TF"><H2>Procedure:</H2><OL><LI>Take competent cells (E.coli DH5α or BL21) from -70°C refrigerator and put it on ice. （set negative control by using chemically competent E.coli cells without plasmids）</LI><LI>When the competent cells dissolve (about 10min), add 10 uL DNA Ligation product or 2 uL plasmid per tube, standing for 30 minutes.</LI><LI>Heat shock at 42℃ for exactly 90 seconds.</LI><LI>Put the 1.5 mL tubes back on ice for 5 minutes. </LI><LI>Add 500 uL LB fluid medium without antibiotics into the 1.5 mL tubes and then culture in the shaker incubator at 37℃ for 1h 20 minutes.</LI><LI>Extract 100-200 uL bacteria liquid, spread it on LB medium with relevant antibiotic.</LI><LI>Place plates upside down and incubate at 37℃ overnight</LI></OL></DIV><LI id="LI_TGLS"><B style="color: blue;">The gene ligation system</B></LI><DIV id="TGLS"><TABLE><TBODY><TR><TH>Components (10uL)</TH><TH>Volume(uL)</TH></TR><TR><TD>Solution I</TD><TD>5</TD></TR><TR><TD>Plasmid Skeleton</TD><TD>0.5</TD></TR><TR><TD>Insert Gene</TD><TD>4.5</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Components (10uL)</TH><TH>Volume(uL)</TH></TR><TR><TD>T4 DNA ligase</TD><TD>1</TD></TR><TR><TD>10×T4 DNA Ligase Buffer</TD><TD>1</TD></TR><TR><TD>Plasmid Skeleton</TD><TD>1.5</TD></TR><TR><TD>Insert Gene</TD><TD>6.5</TD></TR></TBODY></TABLE></DIV><LI id="LI_PM"><B style="color: blue;">Reagent and medium</B></LI><DIV id="PM"><OL><LI>LB medium:10g/L tryptone, 5g/L yeast extract, 10g/L NaCl</LI><LI>LB agar plate:LB medium with 2% (w/v) agar.</LI><LI>Kanamycin:25ug/mL</LI><LI>Ampicillin:50ug/ml</LI><LI>Chloramphenicol：25ug/ml （dissolved in ethanol ）</LI><P>High copy plasmids allow 35 ug/ml concentration. Low-copy plasmids like bacterial artificial chromosomes allow 12.5 ug/ml. Stock solutions can be made at 35 mg/ml in ethanol, kept at -20℃.</P><LI>Ethanol for sterilization:75%</LI><LI>Glycerol for storing strain:50%</LI><LI>pH7.4 PBS (phosphate buffer saline): NaCl 8.00g/L、KCl 8.00 g/L、Na<SUB>2</SUB>HPO<SUB>4</SUB> 3.58g/L、KH<SUB>2</SUB>PO<SUB>4</SUB> 0.244g/L</LI><P>Use sterile filter membrane (pore size=0.22 um) to filtrate bacteria.</P><LI>TAE</LI><P>Tris-base: 242 g</P><P>Acetate (100% acetic acid): 57.1 ml</P><P>EDTA: 100 ml 0.5M sodium EDTA</P><P>Add dH<SUB>2</SUB>O up to one litre.</P><P>To make 1x TAE from 50X TAE stock, dilute 20ml of stock into 980 ml of deionised water.</P><P>TAE is a commonly used buffer for making and running DNA agarose gels.</P><LI>SOB Medium. Used in growing bacteria for preparing chemically competent cells Ingredients</LI><P>0.5% (w/v) yeast extract</P><P>2% (w/v) tryptone</P><P>10 mM NaCl</P><P>2.5 mM KCl</P><P>20 mM MgSO<SUB>4</SUB></P><P>Per liter:</P><P>5 g yeast extract</P><P>20 g tryptone</P><P>0.584 g NaCl</P><P>0.186 g KCl</P><P>2.4 g MgSO<SUB>4</SUB></P></OL></DIV><LI id="LI_PE"><B style="color: blue;">Plasmid extraction</B></LI><DIV id="PE"><H2>Procedure:</H2><OL><LI>Add 200 uL buffer CBS to Adsorption Column (column in the collection tube) and centrifuge at 12,000 rpm for 1 minute, discard the filtrate and reuse collection tube.</LI><LI>Take 5-15mL bacterial solution into a centrifuge tube, centrifuge at 8,000 rpm for 1 min and remove supernatant.</LI><LI>Add 250 uL SolutionⅠin centrifuge tube, using the pipet or vortex oscillator to suspend the cells.</LI><LI>Add 250 uL Solution II in centrifuge tube and gently flip upside down for 4-6 times to make sure the germ is full cracked. </LI><LI>Add 350 uL Solution III, gently flip upside down for 8-9 times to mix until white, flocculent precipitate appears and centrifuge at 12,000rpm for 10 minutes.</LI><LI>Add the supernatant to the adsorption column in step 1, centrifuge at 12,000rpm for l minute, discard the filtrate and reuse collection tube.</LI><LI>Add 500 uL W1 solution to the adsorption column, centrifuge at 12,000rpm for l minute, discard the filtrate and reuse collection tube.</LI><LI>Add 500 uL wash solution to the adsorption column, centrifuge at 12,000rpm for l minute, discard the filtrate and reuse collection tube.</LI><LI>Repeat step 8</LI><LI>Centrifuge the empty adsorption column at 12,000rpm for l minute to dry the column matrix. (Residual ethanol may impact downstream application).</LI><LI>Transfer the adsorption column into a clean 1.5 mL centrifuge tube, add 50 -100uL Elution buffer to the center of the column membrane, let sit at room temperature for 2 minutes and centrifuge at 12,000rpm for l minute, collect the plasmid solution in the centrifuge tube.</LI><LI>Store the plasmid at -20 ° C.</LI><P>10-12 mL of Escherichia coli culture to be miniprepped</P></OL></DIV><LI id="LI_PPP"><B style="color: blue;">PCR product purification</B></LI><DIV id="PPP"><H2>Procedure:</H2><OL><LI>Transfer the PCR product from the PCR tube to a clean 1.5 mL microcentrifuge tube. (Note: The volume of the PCR product should be 50-100 uL. If it is less than 50 uL, add ddH2O until reach to 50 uL )</LI><LI>Add five-fold volume of Binding Solution, turn it upside down to mix them.</LI><LI>Insert a GenClean Column to a Collection Tube.</LI><LI>Transfer the mixture to the GenClean Column, let sit at room temperature for 2 minutes.</LI><LI>Centrifuge at 8,000 rpm for 1 minute.</LI><LI>Transfer the filtrate to the GenClean Column and repeat steps 4-5.</LI><LI>Discard the filtrate and reuse the collection tube.</LI><LI>Add 500 uL Wash Solution, centrifuge at 12,000 rpm for 1 minute.</LI><LI>Discard the filtrate and reuse the collection tube.</LI><LI>Repeat steps 8-9.</LI><LI>Centrifuge the empty GenClean Column at 12,000 rpm for 1 minute to dry the column matrix. (To further dry it, let sit at room temperature for a few minutes.)</LI><LI>Transfer the GenClean Column to a clean 1.5 mL microcentrifuge tube.</LI><LI>Add 30-50 uL Elution Buffer directly to the center of the column membrane.</LI><LI>Let sit at room temperature for 2minutes.</LI><LI>Centrifuge at 12,000 rpm for 1 minute.</LI><LI>Store DNA at -20℃ or use it for follow-up experiments immediately.</LI></OL></DIV><LI id="LI_PRS"><B style="color: blue;">PCR reaction system</B></LI><DIV id="PRS"><TABLE><TBODY><TR><TH>Components (50uL)</TH><TH>Volume(uL)</TH></TR><TR><TD>PrimeSTAR Max</TD><TD>25</TD></TR><TR><TD>Primer-F(10uM)</TD><TD>2</TD></TR><TR><TD>Primer-R(10uM)</TD><TD>2</TD></TR><TR><TD>Template</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>20</TD></TR></TBODY></TABLE><OL><LI>Mix solution well.</LI><LI>Place tube in PCR thermocycler. Set thermocycler program:</LI><P>Inititial denaturation: 5-10min at 95℃;</P><P>Loop (29 cycles), Denaturation: 10s at 95℃,Annealing: 5s at 60℃,Elongation: 5s/1kb at 72℃;</P><P>Final elongation: 50s at 72℃;</P><P>Store: 16℃.(not for too long).</P><P>*We use 5ul of the PCR product for electrophoresis and 45ul for purification (details see PCR Product Purification).</P></OL></DIV><LI id="LI_CP"><B style="color: blue;">Culture preservation</B></LI><DIV id="CP"><H2>Introduction :</H2><P>Bacterias can be stored for months and years if they are stored at -80℃ and in high percentage of glycerol</P><H2>Materials:</H2><OL><LI>50% glycerol solution</LI><LI>Day/overnight culture</LI><LI>1.5mL centrifuge tube</LI></OL><H2>Method:</H2><OL><LI>Pick a single colony off a plate and let grow in an appropriate selectable liquid medium (5ml) overnight.</LI><LI>Add 400 uL 50% glycerol into a 1.5mL centrifuge tube.</LI><LI>Add 600 uL bacteria liquid .</LI><LI>Gently vortex to ensure that the culture and glycerol are well-mixed.</LI><LI>Store in a -80℃ freezer.</LI></OL></DIV><LI id="LI_GE"><B style="color: blue;">GEL extraction</B></LI><DIV id="GE"><H2>Steps:</H2><OL><LI>Perform agarose gel electrophoresis to fractionate DNA fragments. Use fresh TAE as running buffer.</LI><LI>When adequate separation of bands has occurred, carefully excise the DNA fragment of interest using a wide, clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.</LI><LI>Determine the appropriate volume of the gel slice by weighting it in a clean 1.5 mL microcentrifuge tube. Assuming a density of 1 g/mL, the volume of gel is derived as follows: a gel slice of mass 0.3 g will have a volume of 0.3 mL.</LI><LI>Add 1 volume Binding Buffer (XP2).</LI><LI>Incubate at 50-60℃ for 7 minutes or until the gel has completely melted. Vortex or shake the tube every 2-3 minutes.</LI><LI>Insert a HiBind DNA Mini Column in a 2 mL Collection Tube.</LI><LI>Add no more than 700uL DNA/agarose solution from step 5 to the HiBind○R DNA Mini Column.</LI><LI>Centrifuge at 10,000×g for 1 minute at room temperature.</LI><LI>Discard the filtrate and reuse collection tube.</LI><LI>Repeat step 7-9 until all of the sample has been transferred to the column.</LI><LI>Add 300uL Binding Buffer (XP2).</LI><LI>Centrifuge at maximum speed (≥13,000×g) for 1 minute at room temperature.</LI><LI>Discard the filtrate and reuse collection tube.</LI><LI>Add 700uL SPW Wash Buffer.</LI><P>(Note: SPW Wash Buffer must be diluted with 100% ethanol prior to use.)</P><LI>Centrifuge at maximum speed (≥13,000×g) for 1 minute at room temperature.</LI><LI>Discard the filtrate and reuse collection tube.</LI><LI>Repeat steps 14-16 for a second SPW Wash Buffer wash step.</LI><LI>Centrifuge the empty HiBind○R DNA Mini Column for 2 minutes at maximum speed to dry the column matrix. To further dry it, let sit at room temperature for a few minutes.</LI><LI>Transfer the HiBind○R DNA Mini Column to a clean 1.5 mL microcentrifuge tube.</LI><LI>Add 15-30 uL Elution Buffer or deionized water directly to the center of the column membrane.</LI><LI>Let sit at room temperature for 2 minutes.</LI><LI>Centrifuge at maximum speed for 1 minute.</LI><LI>Store DNA at -20℃ or use it for ligation immediately.</LI></OL></DIV><LI id="LI_TDEDS"><B style="color: blue;">The double enzyme digestion system</B></LI><DIV id="TDEDS"><TABLE><TBODY><TR><TH>Components (50uL)</TH><TH>Volume(uL)</TH></TR><TR><TD>10 x buffer</TD><TD>5</TD></TR><TR><TD>Enzyme Ⅰ</TD><TD>1</TD></TR><TR><TD>Enzyme Ⅱ</TD><TD>1</TD></TR><TR><TD>DNA</TD><TD>1-2ug</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>Add to 50uL</TD></TR></TBODY></TABLE><H2>Universal buffer of Double Digestion:</H2><TABLE><TBODY><TR><TH>Enzyme</TH><TH>EcoR I</TH><TH>Xba I</TH><TH>Spe I</TH><TH>Pst I</TH></TR><TR><TH>EcoR I</TH><TD>1*H</TD><TD>1*M</TD><TD>1*H</TD><TD>1*H</TD></TR><TR><TH>Xba I</TH><TD>1*M</TD><TD>1*M+BSA</TD><TD>1*M</TD><TD>1*M</TD></TR><TR><TH>Spe I</TH><TD>1*H</TD><TD>1*M</TD><TD>1*M</TD><TD>1*H</TD></TR><TR><TH>Pst I</TH><TD>1*H</TD><TD>1*M</TD><TD>1*H</TD><TD>1*H</TD></TR></TBODY></TABLE><P>1 mg 1,000bp DNA =1.52 pmol </P></DIV><LI id="LI_DAGE"><B style="color: blue">DNA agarose gel electrophoresis</B></LI><DIV id="DAGE"><H2>Step:</H2><OL><LI>Seal the two ends of the gel-forming model with medical tape properly, position it horizontally, posit the selected comb properly.</LI><LI>Weigh 0.5g agarose for DNA electrophoresis, put it in a 250 mL Erlenmeyer flask, add 50 mL 1xTAE buffer and mix, then put the Erlenmeyer flask in the oven and heat to boiling until the agarose is completely dissolved.</LI><LI>Turn off the microwave, remove the fluid to model, let sit at room temperature until gel completely solidified, it takes about 30 minutes. Remove the tape, pull out the comb, put the plastic sheet into the electrophoresis tank.</LI><LI>Add 1xTAE buffer into the electrophoresis tank above the gel surface around high 2mm. </LI><LI>Using pipette to transfer samples. Add mixture of 5 uL DNA sample and 2ul 10x loading  buffer to spotting holes. </LI><LI>Power on, and adjust the voltage to 120 volts. After 15-20 minutes of electrophoresis, remove the gel plate, staining in ethidium bromide solution for 10min, then observe the result under UV light.</LI></OL></DIV><LI id="LI_PCC"><B style="color: blue">Preparation of chemically competent E.coli cells</B></LI><DIV id="PCC"><H2>Steps:</H2><OL><LI>Inoculate 2ml LB broth with an aliquot (about 50ul) of the desired E.coli from the -80℃ freezer stock of cells.</LI><LI>Incubate for 2h at 37℃.</LI><LI>Add the 2ml seed culture to 250ml LB broth and grow at 37℃, shaking (about 200rpm) until OD600 of 0.3-0.4 (about 5 hours).</LI><LI>Pre-cool the 50ml polypropylene tube, 80 EP tubes, CaCl<SUB>2</SUB>-glycerine (0.1mol/L CaCl<SUB>2</SUB>) and CaCl<SUB>2</SUB>- MgCl<SUB>2</SUB> (80mmol/L MgCl<SUB>2</SUB>, 20mmol/L CaCl<SUB>2</SUB>). Set the centrifuge and prepare the ice tray.</LI><LI>Transfer the bacteria into the 50ml polypropylene tube. Place it on ice for 10 minutes.</LI><LI>Centrifuge at 4℃, 4100rpm for 10 minutes.</LI><LI>Discard supernatant, then place the tube upside down to make sure trace liquid medium runs out.</LI><LI>Add 30ml of pre-cooled CaCl<SUB>2</SUB>- MgCl<SUB>2</SUB> per 50ml of initial liquid medium to resuspend bacteria cell pellet.</LI><LI>Centrifuge at 4℃, 4100rpm for 10 minutes.</LI><LI>Discard supernatant then place the tube upside down to make sure trace liquid medium runs out.</LI><LI>Add 2ml of pre-cooled CaCl<SUB>2</SUB> per 50ml of initial liquid medium to resuspend bacteria cell pellet.</LI><LI>Transfer to EP tubes (50ul every tube) and store at -80℃.</LI></OL><H2>Preparation Method of Competent Cell of Ecoli.DH5α:</H2><P>Introduction : Competent cells are those that possess more easily altered cell walls that DNA can be passed through easily. Therefore , these cells readily incorporate foreign DNA. One example of a competent cell is E. coli.</P><P>Main reagent:</P><OL><LI>LB solid culture medium /100mL</LI><P>*0.5g  yeast extract</P><P>*1.0g  tryptone</P><P>*1.0g  NaCl</P><P>*2.0g  agar</P><P>*Dissolve in nanopure water and autoclave to sterilize with High-Pressure Steam Sterilization Pot.</P><LI>LB liquid culture medium /L:</LI><P>*5g   yeast extract</P><P>*10g  tryptone</P><P>*10g  NaCl</P><P>*Dissolve in nanopure water and autoclave to sterilize with High-Pressure Steam Sterilization Pot.</P><LI> 0.1mol/L CaCl<SUB>2</SUB> solution:</LI><P>*CaCl<SUB>2</SUB>(1.11g/100mL ) or CaCl<SUB>2</SUB>.2H2O(1.47g/100mL)</P><P>*Dissolve in nanopure water and adjust to final volume. Sterilize by filtration with 0.45um filter membrane and store at 4°C</P><LI>50% glycerin /100mL:</LI><P>*50mL glycerin dissolves in 50mL nanopure water and autoclave to sterilize with High-Pressure Steam Sterilization Pot.</P></OL><H2>Experimental procedure(normal calcium method):</H2><OL><LI>Culture cells (DH5a) through plate streaking on LB agar plate at 37°C for 12 hours.</LI><LI>Pick up single colony and culture in 5ml LB liquid medium(in 30mL test tube) with a vigorous shaking（180rpm/min） at 37°C overnight.</LI><LI>Take 0.5mL bacterial fluid to 100mL LB(in 250mL conical flask) and cultures in shaker incubator at 37°C with a speed of 180rpm/min to OD600=0.35~0.4 (normally it takes 1.5-2 hours) .</LI><LI>Packing those bacterial fluid in two 50mL centrifuge tubes and place them on ice for 10 min.</LI><LI>Precipitate the cells by spining at 4,000rpm for 10 min at 4°C with refrigerated centrifuge.</LI><LI>Discard the supernatant and add 10mL CaCl<SUB>2</SUB> (0.1mol/L) to resuspend cells gently. Place them on ice for 30 minutes.( 0.1mol/L CaCl<SUB>2</SUB> solution needs to be precooled at 4°C).</LI><LI>Centrifuge at 4,000rpm for 10 minutes at 4°C and discard the supernatant ./li&gt;
			</LI><LI>Gently resuspend the cells with 1.2mL CaCl<SUB>2</SUB> solution and 0.8 mL glycerin(50%).</LI><LI>Aliquote the cells to 100uL at 1.5mL centrifuge tube(these tubes should be precooled on ice or liquid nitrogen) and place them into liquid nitrogen immediately. Then they can be used for transformation or stored at -70°C.</LI></OL></DIV><LI id="LI_CPCR"><B style="color: blue">Colony PCR</B></LI><DIV id="CPCR"><P>Colony PCR can be used after a transformation to screen colonies with desired plasmid. Primers are used to generate a PCR product of known size. Thus, any colony that gives rise to an amplification product of the expected size is likely to contain the correct DNA sequence. Choose a forward primer from plasmid and a reverse primer in the insert fragment separately.</P><P>10ul:</P><TABLE><TBODY><TR><TH>Components (10uL)</TH><TH>Volume(uL)</TH></TR><TR><TD>Taq Mix(2×)</TD><TD>5</TD></TR><TR><TD>Primer-F(10uM)</TD><TD>0.5</TD></TR><TR><TD>Primer-R(10uM)</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>4</TD></TR></TBODY></TABLE></DIV><LI id="LI_IBA"><B style="color: blue">In-fusion biobrick assembly</B></LI><DIV id="IBA"><P>Introduction: This method describes an alternative assembly method that allows for BioBricks to be assembled via fusion of PCR products. One PCR-amplified BioBrick has homology on each end with the second PCR-amplified BioBrick (vector amplified with the BioBrick) to allow for the fragments to be fused together in the In-Fusion reaction. This method can be adapted to fuse more than two BioBricks together and four fragments have been successfully fused. The advantages of In-Fusion BioBrick assembly over standard assembly are that it is faster, does not require restriction digestions or ligations or DNA extraction from a gel.</P><P>Step one: Select a base vector and identify the insertion site. Linearize the vector by restriction enzyme digestion</P><P>Step two: Design PCR primers for your gene of interest with 15 bp extensions (5’) that are complementary to the ends of the linearized vector.</P><P>Step three: Amplify your gene of interest through PCR</P><P>Step four: Purify your PCR product</P><P>Step five: Set up your In-Fusion cloning reaction: (10ul)</P><P>2 ul of 5X In-Fusion HD Enzyme Premix</P><P>X ul of Linearized Vector</P><P>X ul of Insert</P><P>X ul of dH<SUB>2</SUB>O to a Total Reaction Volume of 10 ul. Mix well.</P><P>Step six: Incubate the reaction for 15 min at 50°C, then place on ice.</P><P>Step seven: Transform competent cells with 2.5 ul of the reaction mixture.</P></DIV><LI id="LI_FM"><B style="color: blue">Fluorescence measurement</B></LI><DIV id="FM"><P>The cells were grown according to the following protocol before assaying their fluorescence.</P><P>Step one: Cells were inoculated from single colonies on LB agar plates and grown overnight in 5ml LB in test tubes at 37 °C with shaking (180rpm). </P><P>Step two: Overnight cultures were diluted 200-fold into in 40ml LB in 100ml triangle bottle, then were incubated at 37 °C in constant temperature breeding shaker with shaking (180rpm) for 3 h.</P><P>Step three: The cultures were diluted 1000-fold and loaded into fresh LB liquid culture medium containing inducer in 40ml LB in 100ml triangle bottle.</P><P>Step four: Collect sample at intervals</P><P>A sample of each culture was transferred to a new 2ml centrifuge tubes containing PBS and 2 mg/ml kanamycin to stop protein expression. The fluorescence distribution of each sample was measured using flow cytometer (BD Accuri C6) .Each distribution contains at least 50,000 events.</P></DIV><LI id="LI_OP"><B style="color: blue">Overlapping PCR</B></LI><DIV id="OP"><P>The cells were grown according to the following protocol before assaying their fluorescence.</P><P>Step one: Cells were inoculated from single colonies on LB agar plates and grown overnight in 5ml LB in test tubes at 37 °C with shaking (180rpm). </P><P>Step two: Overnight cultures were diluted 200-fold into in 40ml LB in 100ml triangle bottle, then were incubated at 37 °C in constant temperature breeding shaker with shaking (180rpm) for 3 h.</P><P>Step three: The cultures were diluted 1000-fold and loaded into fresh LB liquid culture medium containing inducer in 40ml LB in 100ml triangle bottle.</P><P>Step four: Collect sample at intervals</P><P>A sample of each culture was transferred to a new 2ml centrifuge tubes containing PBS and 2 mg/ml kanamycin to stop protein expression. The fluorescence distribution of each sample was measured using flow cytometer (BD Accuri C6) .Each distribution contains at least 50,000 events.</P></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>