<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NAU-CHINA_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NAU-CHINA/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><UL id="gn-menu" class="gn-menu-main"><LI><A href="https://2016.igem.org/Team:NAU-CHINA"><SPAN> HOME</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Participator"> TEAM</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Description"> PROJECT</A></LI><UL class="submenu"><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Experiment">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Demonstrate">Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Model">Model </A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Notebook">Notebook </A></LI></UL><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Part_Collection"> PARTS</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Safety"> SAFETY</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Attributions"> ATTRIBUTION</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Human_Practices"> HP</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Conclusion">CONCLUSION</A></LI></UL></DIV></DIV><NAV><DIV class="nav" id="main" role="main"><UL class="menu"><LI><A href="https://2016.igem.org/Team:NAU-CHINA">HOME</A></LI><LI><A>TEAM</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Participator">Participator</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Collaborations">Collaboration</A></LI></UL><LI><A>PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Experiment">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Demonstrate">Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Model">Model </A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Notebook">Notebook </A></LI><LI><A>PARTS</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Part_List">Part List</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Part_Collection">Collection</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Safety">SAFETY</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Attributions">ATTRIBUTION</A></LI><LI><A>HUMAN PRACTICE</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:NAU-CHINA/Conclusion">CONCLUSION</A></LI></DIV></NAV><DIV id="maintext"><H1>Description</H1><P><B style="color: red;">Insecticide pyrethroids</B> are widely used in the agricultural production to control the insect pest. The <B style="color: red;">main metabolite</B> in the degradation of insecticide pyrethroids is <B style="color: red;">3-phenoxybenzoate (3-PBA)</B> which is is typically detected as an important environmental contaminant<SUP>[1]</SUP>. This time we would like to <B style="color: red;">degrade 3-PBA thoroughly</B> using the concept of Synthetic Biology and microbial biodegradation techniques. Also, we are eager to take the first step to make our project into reality. Microorganism was shown to acquire the ability to degrade various toxic compounds during their evolution. However, the efficiency of removal of these compounds by bacteria is actually low for they evolve on the basis of environmental fitness rather than degradation efficiency. In our project, we found Sphingobiumwenxiniae JZ-1<SUP>T</SUP>, which is resistant to 3-phenoxybenzoate acid and can utilize it as the sole carbon and energy sources for growth. However, according to our preliminary experiment , Sphingobiumwenxiniae JZ-1<SUP>T</SUP> is easy to lose the ability of degradation and cannot grow in LB medium containing high concentration of 3-PBA. In the lab, we aim to realize the biodegradation of 3-PBA using recombinant plasmid. The genes pbaC-pbaA1A2B-c23o-mhbDHIM, encoding enzymes of the 3-phenoxybenzoate degradation pathway were successfully built finally. To make the engineered bacteria work, we <B style="color: red;">improved the previous parts copA and characterized Anderson Promoter.</B></P><H1>Outline</H1><P>Click the list for details.</P><UL><LI id="LI_BG" class="dianlezhankai"><B style="color: blue;">Background - Why we need to degrade 3-PBA thoroughly?(Click for details)</B></LI><DIV id="BG"><TABLE><TBODY><TR><TD><UL>Our analysis:
							<LI>Be harmful to the environment and human</LI><LI>Drosophila toxicological experiment</LI><LI>Cost-benefit analysis for engineered bacteria</LI></UL><H2>Be harmful to the environment and human</H2><P>The main metabolite in the degradation of insecticide pyrethroids is 3-phenoxybenzoate (3-PBA).Because of the wide use of pyrethroids and the stability of the diarylether compound itself, 3-PBA is typically detected as an important environmental contaminant. As a metabolite, 3-PBA is easy to be ignored by people. It's time to arouse people's attention on 3-PBA and pyrethroids because of the existed truth reported by paper and media. Through searching a large number of papers and analyzing data, we found that the harms of 3-PBA are as follows:</P><OL><LI>Long natural degradation half-life</LI><P style="padding-left: 0px;font-family:'Comic Sans MS','Arial Black';color: rgb(100,100,100);">The half life period of 3-PBA is about 180 days , longer than most of pyrethroids, making it difficult to be degraded in the environment.</P><LI>Hindering the mineralization of pyrethroids，which cut off the biodegradation pathway that pesticide is transformed into non-toxic (poisonousless) small molecule.</LI><LI>Making secondary pollution to farm</LI><LI>Widespread occurrence in water, sediment, and soil</LI><LI>3-PBA is suspected of having reproductive toxicity</LI><LI>3-PBA is an important material in chemical industry</LI></OL><H2>Drosophila toxicological experiment</H2><P>To analyze the toxicity of the 3-PBA, we used fruit fly to test. We tested 3-PBA's influence on reproductive capacity of fruit fly, and found that 3-PBA can affect the normal secreting of sex hormone.</P><P>We counted the number of the new imagoes hatched by flies that were fed in medium with 3-PBA before.</P></TD></TR><TR><TD>A</TD><TD>B</TD></TR><TR><TD colspan="2"><B> Fig.1. A.B.The results of Drosophila toxicological experiment.</B></TD></TR></TBODY></TABLE><P><B><A href="https://2016.igem.org/Team:NAU-CHINA/Drosophila">for more details...</A></B></P><H2>Cost-benefit analysis for engineered bacteria</H2><P>Degradation comes at a price. Thorough biodegradation of 3-PBA involves a series of enzymatic reactions. Protein synthesis and enzyme reactions needs materials, ATP and NADH. Catechol , an intermediate metabolite of 3-PBA, is toxic to the Escherichia coli. 3-PBA can be degraded and absorbed entering the TCA circle to produce energy. Based on these facts we can get a formula Net proceeds(N)=Benefit(B)-Cost(C).</P><DIV class="imgbox"><P><B> Fig.2.The sketch map of cost-benefit analysis.</B></P></DIV></DIV><LI>Method - How to degrade 3-PBA thoroughly?</LI><UL><LI id="LI_GCD" class="dianlezhankai"><B style="color: blue;">Gene circuit design(Click for details)</B></LI><DIV id="GCD"><TABLE><TBODY><TR><TD><H2>Simplest genetic circuit ---our small step to degradation</H2><H3>Introduction</H3><P>We used constitutive promoter (BBa_J23101) to transcribe downstream genes including gene cluster pbaA1A2B and C23O. Constitutive promoter (BBa_J23100)was also used to transcribe gene pbaC. Gene cluster pbaA1A2B and gene pbaC encode the 3-phenoxybenzoate 1',2'-dioxygenase,which is an angular dioxygenase catalyzes the hydroxylation in the 1'and 2' positions of the benzene moiety of 3-phenoxybenzoate(3-PBA)<SUP>[4]</SUP>, yielding 3-hydroxybenzoate(3-HBA)<SUP>[5]</SUP> and catechol. 3-HBA is further transformed by 3-hydroxybenzoate 6-hydroxylase, gentisate 1,2-dioxygenase, glutathione (GSH)-dependent maleylpyruvateisomerase and fumarylpyruvate hydrolase coding by gene cluster mhbDHIM. MhbM, MhbD, MhbI and MhbH convert 3-HBA to pyruvate and fumarate. Meanwhile, catechol is detoxified by catechol 2,3-dioygenase coding by gene C23O. The genetic circuit is shown in <B>Fig1</B>. and the whole metabolic pathway is shown in <B>Fig2.</B>.</P><DIV class="imgbox"><P><B>Fig1. Our device is designed to degrade 3-phenoxybenzoatecompletely</B></P></DIV><DIV class="imgbox"><P><B>Fig2.The passway of biodegradation of 3-pba in our engineering bacteria</B></P></DIV></TD></TR></TBODY></TABLE></DIV><LI id="LI_EPA" class="dianlezhankai"><B style="color: blue;">Experimental test &amp; previous parts improvement &amp; analysis(Click for details)</B></LI><DIV id="EPA"><TABLE><TBODY><TR><TD><H1>Previous parts improvement</H1><H2>Overview</H2><P>CopA, the principal copper effluxATPase in Escherichia coli, is induced by elevated copper in the medium<SUP>[1]</SUP>. CopA promoter is active in the presence of copper ion.We intended to character copA promoter independently. Therefore , we utilized RiboJ which was placed between promoter and protein coding sequence to eliminate the interference of two different parts. Output ( fluorescence) depended only on the activity of copA promoter when be induced, and not the sequence at the part junction. RiboJ can reliably maintain relative promoter strengths.</P><H2>First experiment</H2><P>We test copA promoter in BL21 (DE3),DH5α. By measuring fluorescence intensity in cells by flow cytometer,we got data to analyze sensitivity and specificity of copA promoter.</P><H3>Results</H3><P>In our experiment, copA promoter was induced by different concentration of copper ion (37.5umol/L, 50umol/L, 62.5umol/L, 75umol/L) . That fluorescence intensity in cell increase firstly and decreasewith small oscillations(Fig.3A,B). At 4-5th hour fluorescence intensity in cell increases dramatically. Dose response curves was fitted to twice induction within 9 hours. CopA promoter has relative leaky basal expression by comparing the negative control’s output and basal leakage of copA promoter in E. coli expression systems(Fig.4). In comparison of two graphs A, B, we can obviously find that the degradation of protein is much faster in DH5α than that in BL21 (DE3)，because BL21 (DE3) has a deficiency of protease. In the group of 0μmol/L Cu2+, the fluorescence shows a trend of falling firstly then rising(Fig.3C). Actually, the fluorescence which produced by the leakage of copA will not change. The change quantity comes from the different growth periods of the E.coli. We added 40ul bacterial fluid into new medium with inductionto start measuring. So bacteria will go through a period of growing from growth period to maturation period, so as to the change of the fluorescence.Maturation period is great period for the expression of protein. </P><DIV class="imgbox"><P><B>Fig.3.A.Changes in fluorescenceintensity induced by different concentration of copper ion in E.coli BL21 (DE3). B.Changes in fluorescence intensityinduced by differentconcentration of copper ion in E.coli DH5α. C .Changes in fluorescenceintensity in E.coli without induction comparing with two strains. D .Changes in fluorescenceintensity in E.coli which do not contain copA promoter comparing with two strains.</B></P></DIV><DIV class="imgbox"><P><B>Fig.4.CopA promoter has leaky basal expression</B></P></DIV><H2>Second experiment</H2><P>We placed insulator RiboJ between copA promoter and RBS(Fig.5).</P><DIV class="imgbox"><P><B>Fig.5.Two device used to detect copper in solution, the upper device has riboJ and the under one has no riboJ.</B></P></DIV><H3>Result</H3><P>We use 50μmol/L copper ion to induce copA promoter.A device without RiboJ has an unstable Fluorescent quantity. At fourth hour, the fluorescence intensity in cells rose sharply. By contrast,a device with RiboJ response to copper ion and express GFP gradually. (Fig 6A) In addition,a device without RiboJ has high leakage with fluctuation. However, a device with RiboJ has low and stable leakage. (Fig 6B)</P><DIV class="imgbox"><P><B>Fig .6 A.Changes in fluorescenceintensity induced by copper ion in E.coli over time.  B.leakage of copA promoter over time.</B></P></DIV><H2>Improvement</H2><P>We concluded that RiboJ helps reduce the leakage of copA promoter greatly. After adding copper ions, the expression of green-fluorescent protein increased steadily. So, copA promoter with RiboJ can balance the expression of target protein in Escherichia coli.</P><H2>Reference</H2><P>[1] Outten FW, Outten CE, Hale J, O'Halloran TV. Transcriptional activation of an Escherichia coli copper efflux regulon by the chromosomal MerR homologue, cueR. Journal of Biological Chemistry 2000;275:31024-9.</P></TD></TR></TBODY></TABLE></DIV><LI id="LI_GCR" class="dianlezhankai"><B style="color: blue;">Gene circuit redesign(Click for details)</B></LI><DIV id="GCR"><TABLE><TBODY><TR><TD><H2>Precision gene expression</H2><P>The ability to regulate gene expression is of central importance for the adaptabilityof living organisms to the changes in their external and internal environment. Our group want to tune the gene expression level to achieve energy saving and highly effective degradation.In our multienzyme pathway, differentconstruction methods were involved to generate genetic circuitincluding different promoters, RBSs, CDSs, terminators. And we also change gene orders and use different plasmid backbones to make different. </P><H2>Gene order</H2><P>Gene orders is the permutation of genome arrangement. Changing gene order can adjusting the activities and relative ratios of these enzymes at the same time. Adjusting the activities of enzymes simultaneously is the moststraightforward way to optimize. Therefore, we rearrange rearranging the mhb gene order within the mhb operon, letting protein expression following the order of enzymatic reaction Fig1. It demonstratedthat genes positioned closer to the promoter were more highlyexpressed than genes positioned further away from the promoter.In addition, the gene expression levels were positively correlatedto the protein expression levels. It’s predictive that toxic intermediate metabolite will reduce.</P><DIV class="imgbox"><P><B>Fig1.Proposed pathway for 3-hydroxybenzoate catabolism in strain M5a1, together with the catabolic reactions catalyzed bymhbgene productsin vivo.</B></P></DIV><H2>Codon optimization</H2><P>In fact, the codons in a gene may be true bottlenecks, especially in cases where foreign genes are expressed in a host in which the usage of codons in highly expressed genes does not resemble the usage of codons in the species from which the foreign gene originates.In such cases, it has been shown that substitution of rare codons in the introduced gene may increase the yield dramatically. In addition, replacement of rare codons might decrease the chance of misincorporation and protect the protein from premature turnover .We use Codon Optimization Tool (http://sg.idtdna.com/CodonOpt) which is announced that calculates a codon-optimized sequence of any gene based on knowledge of highly expressed genes of a host and created by IDT. We use full optimization----optimization of every codon of a coding sequence in an attempt to increase translational efficiency.</P><H2>Terminators</H2><P>Our circuits have many transcription units, each of which needs strong termination to avoid recombination. These terminators must be sequence diverse to avoid recombination. We use there different terminatorsB0015, B0010, B0014 which are available from parts.org.</P><H2>Insulators</H2><P>There is a direct interference of one part type on another .The strength of RBS is influenced by the promoter. Insulator parts have been developed to diminish the effect of genetic context.RiboJ is the sequence for a ribozyme studied in Lou et. al 2012.We used this sequence between the promoter and ribosome sequence which serves as an insulator to generalize the transfer function of a circuit regardless of promoter.</P><H2>Riboswitch</H2><P>Riboswitch is used to regulate gene expression in response to specific small-molecule ligands. The aptamer domain selectively binds a target metabolite, resulting in a conformational change which is communicated to an adjacent expression platform. We use riboswitchM6”which is shown in Fig2 as regulatory element.RNA-based gene regulatory controlcould offer advantages over conventional protein-based strategies.Cells exhibit inresponse to PPDA.PPDA is short for Pyrimido[4,5-d]pyrimidine-2,4-diamineshown in Fig3.</P><DIV class="imgbox"><P><B>Fig2.Riboswitch M6” secondary structure (come from RNAfold Webserver)</B></P></DIV><DIV class="imgbox"><P><B>Fig3. Pyrimido[4,5-d]pyrimidine-2,4-diamine</B></P></DIV><H2>Ribosome binding sites</H2><P>The RBS calculator has been used to predict protein expression in various bacteria or to design RBS sequences for a desired expression level.</P><P>Do you have any experience about this tool? </P><H2>Principle</H2><P>The RBS is a part that is relatively simple to achieve different expression level.The RBS calculator provides a computational framework for designing RBS sequences of a given strength based on a thermodynamic model of translation initiation. Prokaryotic ribosome-binding site (RBS) element that initiates translation for one coding sequence might not function at all with another coding sequence, so we couldn’t just choose RBS which from parts.org or still use native RBSs.Translation initiation (and hence protein expression) is thus tunedby choosing an RBS sequence with the desired interaction energy.</P><H2>In-silico Design of RBS</H2><P>The sequences upstream gene mhbD, mhbH, mhbI, mhbMwere regarded as native RBSs, and their strengths were evaluated using RBS Calculator (reverse engineering mode).Their strengths indicated by translation initiation rate (T.I.R.) were 32146.26 (RBS mhbD), 50417.64 (RBS mhbH), and 249.03 (RBS mhbI), 1039.46 (RBS mhbM).Specific T.I.R.s for each of the four genes were designed using RBSCalculator (forward engineering mode).Their strengths indicated by translation initiation rate (T.I.R.) were 20607.43 (RBS mhbM), 9166.64 (RBS mhbD), and 2017.02 (RBS mhbI), 6637.54 (RBS mhbH).We define specific T.I.R.s for each gene by their enzyme activity measured in vitro.The unit of enzyme activity was defined as the amount required for the disappearance of 1 mol of substrate per minuteat room temperature. Specific activities are expressed as units per milligram of protein.The specific activities of mhbD, mhbH, mhbI, mhbM coding proteins were indicated as 0.262 U/mg , 0.695 U/mg, 4.99 U/mg, 1.45 U/mg respectively. These proteins’ molecular weight were calculated through the tool of Protein Molecular Weight Calculator. They were 43.89KD(mhbM), 38.97KD(mhbD), 24.1 KD(mhbI), 25.74KD(mhbH). In biology, the molecular weight of protein is defined as kDa ,1kDa=1000g/mol. We calculated with the combination of enzyme specific activity and the molecular weight of protein to define the mole ratio of the four proteins in vivo. Then, we set the translation initiation rates of mhbI as 2000 au. Based on the mole ratio,the translation rates of mhbM, mhbD, mhbH are 20000au, 8950au, 6444au respectively.</P><H2>Promoter</H2><P>One of the paramount goals of synthetic biology is to have the ability to tune transcriptional networks to targeted levels of expression at will.As a step in that direction, our team useinducible promoters which are a very powerful tool in genetic engineering .The expression of genes which can be turned on at certain stages operably links to them. In our original genetic, we choose copA promoter which is induced by elevated copper in the medium. Considering toxicity of copper ion, we replace it withLacI regulated promoter Plac .In the presence of LacI protein and CAP protein, this promoter inhibits transcription. LacI can be inhibited by IPTG. This promoter wasrepressed by LacI protein and inducted by IPTG.In addition ,we use a promoter which can be induced by 3-hydroxybenzoate with assistant of protein mhbR and mhbT.</P><P>In our genetic circuit ,we also useAndersonpromoter BBa_J23100 whose measure strength is 1.0.In order to make contribution to character Anderson promoter in different way , we used TASSEL.5 software to analyze the inter-dependent relationship between each mutation and the strength of the promoter.</P><H2>3-PBA Sensing</H2><P>As the first dedicated phase of gene expression, transcription serves as one method by which cells mobilize a cellular response to an environmental perturbation. As such, the genes to be expressed, promoters, transcription factors and other parts of the transcription machinery all serve as potential engineering components for transcriptional biosensors. We want to establish a biosensor for our system.</P><H2>Design Principle</H2><DIV class="imgbox"><P><B>Fig.5.In Sphingobium wenxiniaeJZ-1TPbaR binds specifically to the 29-bp motif （AATAGAAAGTCTGCCGTACGGCTATTTTT） in thepbaA1A2Bpromoter area and that the palindromic sequence (GCCGTACGGC)within the motif is essential for PbaR binding. PbaR is a transcriptional activator which regulates expression downstream gene.</B></P></DIV><H2>Our device oftranscriptional biosensing</H2><P>Sphingobium wenxiniaeJZ-1T possesses a good base level of resistance to 3-phenoxybenzoate (3-PBA).Here the engineered Sphingobium wenxiniaeJZ-1T needs to be able to sense 3-PBA and activate expression of green fluorescent protein accordingly.</P><DIV class="imgbox"><P><B>Fig.6.The device containing our 3-pba inducible promoter and a reporter gene.</B></P></DIV><H2>Redesign of genetic circuit</H2><DIV class="imgbox"><P><B>Fig.4.We redesigned our genetic circuit based on semi-rational design. The Mhbpt promoter is a promoterwith low leakage and 3-HBA induction. The expression of gene cluster mhbDHIM is regulated by Placpromoter.To achieve greater regulatory control, we additionallyplaced the Riboswitch M6” between Placpromoterand mhbDHIM gene cluster, creating an expression system that combines  transcriptional-translational control over gene expression .</B></P></DIV></TD></TR></TBODY></TABLE></DIV><LI id="LI_APC" class="dianlezhankai"><B style="color: blue;">Anderson Promoter Characterization(Click for details)</B></LI><DIV id="APC"><TABLE><TBODY><TR><TD><P>The Anderson promoters are a collection of variable strength constitutive promoters for use in E. coli and other prokaryotes. The collection is known to cover a range of activities.They were created from a consensus sequence (J23119) by Chris Anderson. NAU-CHINA Team used TASSEL.5 software to analyze the inter-dependent relationship between each mutation and the strength of the promoter.The association analysis was initially applied to locate QTLs(quantitative trait locus)in analyzing human genome and plant genome, because it can use linkage disequilibrium to identifythe phenotypic variation caused by different allelesand developing functional markers.</P><P>We combine the sequence of individual promoters and the relative strengths of these promoters and use the method described below .The P-Value is calculated using the Generalized linear model (GLM).The result is showed on a Manhattan plot Fig1.  Manhattan plot is a type of scatter plot, usually used to display data with a large number of data-points - many of non-zero amplitude, and with a distribution of higher-magnitude values, for instance in genome-wide association studies (GWAS).</P><DIV class="imgbox"><P><B>Fig1. In GWAS Manhattan plots, each site of the sequence of the promoter are displayed along the X-axis and theinitiation site is“0”, with the negative logarithm of the association P-value for each single nucleotide polymorphism (SNP) displayed on the Y-axis, meaning that each dot on the Manhattan plot signifies a SNP. It shows the correlation degree between the each site of the promoter and the strength of the promoter through analyzing a series of Anderson promoters. </B></P></DIV><H2>Method</H2><P><B>Step one:</B> Set J23119 as a reference sequence.</P><P><B>Step two:</B> List all of the mutations and correspond them with the each site.</P><P><B>Step three:</B> Store the file as a genotype information.</P><P><B>Step four:</B> List all of these promoters,strength and correspond them with the each promoter.</P><P><B>Step five:</B> Store the file as a phenotype file.</P><P><B>Step six:</B> Load the genotype file and the phenotype file in the TASSEL5.0.</P><P><B>Step seven:</B> Intersect the two files.</P><P><B>Step eight:</B> Use GLM to analyze the consolidated data.</P><P><B>Step nine:</B> Drawing the Manhattan Plot based on the consolidated data.</P><H2>Results</H2><P>According to the Manhattan plots, we concluded that the higher the P-Value is, the more important the site is. We can think that the site has conservative base .If the conservative site（influential base ）gets mutated, the strength of promoter will decreased. From Fig 2，mutations in site &quot;24&quot;pose great influence on the strength of promoter .</P></TD></TR></TBODY></TABLE></DIV></UL><LI>Practice - How to make our project into reality?</LI><UL><LI>Social investigation (Human Practice)</LI><LI>Physical unit (Applied design)</LI><LI>Concept &amp; achievement sharing (Human Practice)</LI></UL></UL><P><B>Reference</B></P><P>[1] Cheng MG, Chen K, Guo SH, Huang X, He J, Li SP, et al. PbaR, an IclR Family Transcriptional Activator for the Regulation of the 3-Phenoxybenzoate 1 ',2 '-Dioxygenase Gene Cluster in Sphingobium wenxiniae JZ-1(T). Appl Environ Microb 2015;81:8084-92.</P><P>[2] Duan XQ, Zheng JW, Zhang J, Hang BJ, Jian HE, Shun-Peng LI. Characteristics of a 3-Phenoxybenzoic Acid Degrading-Dacterium and the Construction of a Engineering Bacterium. Environmental Science 2011;32:240-6.</P><P>[3] Deng W, Liu S, Yao K. [Microbial degradation of 3-phenoxybenzoic acid--A review]. Wei sheng wu xue bao = Acta microbiologica Sinica 2015;55.</P><P>[4] Wang C, Chen Q, Wang R, Shi C, Yan X, He J, et al. A Novel Angular Dioxygenase Gene Cluster Encoding 3-Phenoxybenzoate 1′,2′-Dioxygenase in Sphingobium wenxiniae JZ-1. Applied &amp; Environmental Microbiology 2014;80:3811-8.</P><P>[5] Lin LX, Hong L, Zhou NY. MhbR, a LysR-type regulator involved in 3-hydroxybenzoate catabolism via gentisate in Klebsiella pneumoniae M5a1. Microbiological Research 2010;165:66-74.</P><P>[6]Hiroe A, Tsuge K, Nomura C T, et al. Rearrangement of gene order in the phaCAB operon leads to effective production of ultrahigh-molecular-weight poly [(R)-3-hydroxybutyrate] in genetically engineered Escherichia coli[J]. Applied and environmental microbiology, 2012, 78(9): 3177-3184.</P><P>[7] Fuglsang A. Codon optimizer: a freeware tool for codon optimization[J]. Protein expression and purification, 2003, 31(2): 247-249.</P><P>[8] Xu Y, Gao X, Wang S H, et al. MhbT is a specific transporter for 3-hydroxybenzoate uptake by Gram-negative bacteria[J]. Applied and environmental microbiology, 2012, 78(17): 6113-6120.</P><P>[9] Robinson C J, Vincent H A, Wu M C, et al. Modular riboswitch toolsets for synthetic genetic control in diverse bacterial species[J]. Journal of the American Chemical Society, 2014, 136(30): 10615-10624.</P><P>[10] Brewster R C, Jones D L, Phillips R. Tuning promoter strength through RNA polymerase binding site design in Escherichia coli[J]. PLoS Comput Biol, 2012, 8(12): e1002811.</P><P>[11] Lou C, Stanton B, Chen Y J, et al. Ribozyme-based insulator parts buffer synthetic circuits from genetic context[J]. Nature biotechnology, 2012, 30(11): 1137-1142.</P><P>[12] Outten F W, Outten C E, Hale J, et al. Transcriptional Activation of an Escherichia coliCopper Efflux Regulon by the Chromosomal MerR Homologue, CueR[J]. Journal of Biological Chemistry, 2000, 275(40): 31024-31029.</P><P>[13]Morra R, Shankar J, Robinson C J, et al. Dual transcriptional-translational cascade permits cellular level tuneable expression control[J]. Nucleic acids research, 2015: gkv912.</P><P>[14] Khalil A S, Collins J J. Synthetic biology: applications come of age[J]. Nature Reviews Genetics, 2010, 11(5): 367-379.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>