<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UESTC_Software_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UESTC Software/Design</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!doctype html&gt;<TITLE>Design</TITLE></P><NAV class="dou_divide_nav"><DIV class="dou_divide_nav_wrap"><UL><LI><A href="https://2015.igem.org/Team:UESTC_Software">HOME</A></LI><LI class="choose"><A class="" href="javascript:void(0)">PROJECT<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Validation">Validation</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Results">Results</A></LI></UL><LI><A href="javascript:void(0)">DOCUMENTS<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/User_guide">User Guide</A></LI><LI style=""><A href="https://2015.igem.org/Team:UESTC_Software/Documentation">API</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Practices">HUMANPRACTICE</A></LI><LI><A href="javascript:void(0)">REQUIREMENTS<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Collaborations">Collaborations</A></LI><LI><A href="javascript:void(0)">TEAM<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Attributions">Attributions</A></LI></DIV></NAV><DIV class="cd_half_section" id="c_h_s_desi"><H1>Design</H1><P>Photo From © PEXELS</P></DIV><DIV class="safetyDiv design_div"><DIV class="QandA" style="text-align:center"><H3 class="design_h3" style="text-align:left">1. A reliable source of data</H3><P>To deal with the data problem, we derived data of experimentally-determined essential genes from CEG database<SUP>[1]</SUP>.  CEG(Cluster of Essential Gene) is a database which contains clusters orthologous essential genes. A notable feature of CEG is it deposits essential genes in orthologous groups rather than as single gene. There are seven tables in CEG database, and we used five of them selectively according to demand. Including ceg_core, which contains the information of each essential gene; ceg_base, which contains information of each gene cluster. And other three tables are reference, annotation and nucleic acid sequences.</P><P>We know upgrading is a key influence factor of the Product Life Cycle, and this is also our key consideration. The data of MCCAP will be constantly improving with updating of CEG, and reference species of it has increased from 16 to 29.</P></DIV><DIV class="QandA"><H3 class="design_h3">2. Determining the minimal gene set</H3><P>The first and most important step of building a minimal artificial cell is determining the minimal gene set (minimal genome). Our method of solving this key task is the half retaining strategy. It is quite simple but effective. The implementer of the half-retaining strategy, MCCAP, achieves good results.</P><P style="font-size:1.4rem;"><STRONG>However, what exactly is the half retaining strategy?  </STRONG></P><P>Our half retaining strategy is basically one kind of Comparative Genomics Approach but it is further improved by starting with essential gene data determined by inactivation experiments. Firstly, we classify genes with the same function in different species into one category, named gene cluster. Secondly, we calculate the size of each gene cluster. Size means the number of species which contain the genes in this cluster. Finally, we put genes, whose size is no less than the half number (HN) together as the minimal gene set. For example, when there are 15 species, HN will be 8 according to our definition. So, if the size of a gene cluster is equal or more than 8, it will be added to the minimal gene set. In practice, we optimize the retaining ratio from the nominal half number to obtain more stable result.</P><P style="font-size:1.4rem;"><STRONG>And what is difference between the half retaining strategy and traditional method?</STRONG></P><P>Contrast with the traditional comparative genomics method, it has two advantages as follow:</P><P>1.	Traditional method is using comparative genomic approach to find ubiquitous genes from two completely sequenced small bacterial genomes. As we know in introduction, it will lose some genes when essential cellular function is performed by unrelated proteins that show no sequence similarities to each other in different organisms. However, half retaining strategy can get a stable result of minimal gene set, for it is based on plenty of reference species and an ideal retaining ratio.</P><P>2.	Half retaining strategy uses experimentally-determined essential genes which are specific to organisms. Differ from traditional method, which is based on the whole genome, Half retaining strategy has lower false positive results.</P><P>MCCAP implements the half retaining strategy perfectly. Differ from traditional method, which provides a certain result of minimal gene set, MCCAP believe that minimal gene set is not unique, and it will be difference with different research fields. MCCAP provides a reference species selector interface to users, so that you can get a specific minimal gene set by your purposeful selection.</P><P>Most important information of minimal gene set to users is functions of genes. So in the results interface, MCCAP will give the function descriptions of gene firstly. MCCAP have also divided minimal gene set to four categories by their functions reference the standard of COG<SUP>[2]</SUP>. In fact, there are 25 function categories of COG. But, it is too complicated for user’s browsing experience. So MCCAP reorganizes them into four categories. These four categories are Transport and metabolism, Transcription and translation, General function prediction only and Others. </P><P>In addition to base information, MCCAP will also provide users with information related to KO(KEGG Orthology), EC number of enzymes, COG(Clusters of Orthologous Group) and so on. It is convenient for researchers to do further study.</P></DIV><DIV class="QandA"><H3 class="design_h3">3. Generating gene network diagram</H3><P>In addition to determining the minimal gene set, there is still a crucial consideration is to know gene metabolism and regulation in building a minimal artificial cell.  In order to solve this problem, we combine KEGG Pathway and KEGG Module with the minimal gene set to generate gene network diagram. KEGG Pathway is a collection of manually drawn pathway maps representing our knowledge on the molecular interaction and reaction networks. KEGG Module is a collection of manually defined functional units<SUP>[3]</SUP>. </P><P>Because metabolism is the foundation by which organisms sustain their lives, we divided both pathway and module into two kinds. One’s prefix is Meta, which means gene network diagram generated by only those genes related to metabolism. The other one’s prefix is Entire, which means gene network diagram generated by the whole minimal gene set.</P><P>For the functions of fetching information from KEGG, we reused the project of two previous iGEM teams.  USTC_Software (https://2014.igem.org/Team:USTC-Software) and Illinois-Tools (https://2009.igem.org/Team:Illinois-Tools). </P></DIV><DIV class="QandA"><H3 class="design_h3">4. Processing gene to standard parts</H3><P>For researchers aiming to bring their design into reality, MCCAP provi3des information related to standard parts and biobrick assembly standard. So that users can decide which certain gene to use and how to process this gene into a standard part. Currently, biobrick assembly standard supported by MCCAP contains RFC[10], RFC[12], RFC[21], RFC[23] and RFC[25].</P><P>Besides, considering the data, there may be some genes which have been processed into standard parts. In order to avoid repeating the wheel, then we tried to find out them from protein coding sequences in registry of standard biological parts. If a gene from minimal gene set has already been processed into a biobrick, MCCAP will provide relevant information and we call it biobrick content. For this kind of gene is really seldom, we also list all of them in a table by considering the user’s convenience.</P></DIV><DIV class="des_QandA QandA wenx_div"><H3 class="des_h3">Reference:</H3><H3><P><B>[1] Ye YN, Hua ZG, Huang J, Rao N, Guo FB. CEG: a database of essential gene clusters. BMC Genomics. 2013, 14:769. (From instructor’s group)</B></P><P style="font-weight:100;">[2] Tatusov RL, Galperin MY, Natale DA, Koonin EV. The COG database: a tool for genome-scale analysis of protein functions and evolution. Nucleic Acids Res. 2000, 28(1):33-6.</P><P style="font-weight:100;">[3] Kanehisa M, Goto S, Sato Y, Kawashima M, Furumichi M, Tanabe M. Data, information, knowledge and principle: back to metabolism in KEGG. Nucleic Acids Res. 2014, 42(Database issue):199-205.</P><FOOTER><DIV class="footer_define"><DIV class="footer_logo"><P>@2015 Designed &amp; Developed by <SPAN class="green_em">UESTC_SOFTWARE</SPAN> iGEM</P></DIV></DIV></FOOTER></H3></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>