<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UESTC_Software_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UESTC Software/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Description</TITLE></P><NAV class="dou_divide_nav"><DIV class="dou_divide_nav_wrap"><UL><LI><A href="https://2015.igem.org/Team:UESTC_Software">HOME</A></LI><LI class="choose"><A class="" href="javascript:void(0)">PROJECT<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Validation">Validation</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Results">Results</A></LI></UL><LI><A href="javascript:void(0)">DOCUMENTS<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/User_guide">User Guide</A></LI><LI style=""><A href="https://2015.igem.org/Team:UESTC_Software/Documentation">API</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Practices">HUMANPRACTICE</A></LI><LI><A href="javascript:void(0)">REQUIREMENTS<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Collaborations">Collaborations</A></LI><LI><A href="javascript:void(0)">TEAM<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Attributions">Attributions</A></LI></DIV></NAV><DIV class="cd_half_section" id="c_h_s_desc"><H1>Description</H1><P>Photo From © PEXELS</P></DIV><DIV class="back_to_top"><I class="iconfont"></I></DIV><DIV class="safetyDiv des_div"><DIV class="des_QandA QandA"><H3 class="des_h3">Background </H3><P>It is well-known that <B>microbial cell factories</B> are able to produce many valuable products such as biofuels, new drugs for treatment of diseases and industry enzymes. In order to <B>improve the efficiency of cell factories</B>, we use synthetic biology to <B>design the minimal cell</B><SUP>[1]</SUP>. The lower left figure is extracted from the web page<SUP>[2]</SUP>.</P><P>It is widely accepted that building a minimal artificial cell serves as a <B>critical role</B> in synthetic biology. A synthetic minimal cell will supply a <B>proper chassis</B> to integrate functional synthetic parts, devices and systems with functions which can meet different purposes and requirements<SUP>[3]</SUP>. More and more researchers have taken the design and construction of a functional minimal genome into great consideration when assembly such a minimal artificial cell or chassis, since all the cell functions can be dates back to genome. For <B>the minimal genome</B>, it represents the infrastructure of a minimal cell and could be taken as the <B>foundation for synthetic biology</B><SUP>[4]</SUP>.It is well-known that minimal genome can be used for chassis cell while applying to the <B>industrial manufacture</B> and synthetic biology.The top right figure is designed by us.</P><P>The works for evaluating the size and content of minimal genome have been carried out through comparative and experimental approaches. Mushegian and Koonin<SUP>[5]</SUP> pioneered the work of determining a minimal gene set by <B>comparative genomic approach</B>. They derived such a repertoire for the first minimal gene set with 256 conserved genes based on the first two completely sequenced small bacterial genomes, Mycoplasma genitalium and Haemophilus influenza, which belong to two ancient bacterial lineages, i.e., Gram-positive and Gram-negative bacteria, respectively. Later,   based on comprehensively comparing the published data from comparative genomics and experiments, <B>Gil and his colleagues</B><SUP>[6]</SUP> determined the core of a minimal bacterial gene set with 206 genes, and reconstructed a minimal metabolic machinery necessary to sustain life. In practice, the comparative-genomic approach to minimal gene set derivation is based on the key evolutionary notion of orthology, and the resulting sets of genes should approximate those of ancestral life-forms <SUP>[7]</SUP>. </P><P>It is predictable that <B>with the increasing number of chromosomes from different organisms, the number of orthologous genes would descend</B>. Analysis of about 100 genomes shows that only 63 genes are ubiquitous. That may be caused by the condition, when the genes which perform the same function in two species are non-orthologous, the essential gene will be missed<SUP>[8]</SUP>. However, the researchers had to adopt the <B>“ubiquitous” strategy</B> just because at that time only a few bacterial genomes had been completely sequenced.</P></DIV><DIV class="des_QandA QandA"><H3 class="des_h3">Overview</H3><P>In recent years, dozens of bacteria have obtained genome scale essentiality data with experimental validations and they can provide us the feasibility to develop <B>a novel strategy of minimal gene set extraction</B> from the experimental essentiality data. Following the comparative genomic approach, we developed with our instructor’s group the <B>“half-retaining”</B>, by comparing the experimentally-determined essential genes among various reference species. This strategy<SUP>[9]</SUP> retains the essential genes persisting in more than half of the reference genomes to obtain conserved essential genes. <B>Conventional methods</B><SUP>[5]</SUP> which retain genes persistent in all reference species as a minimal gene set suffer from the disadvantage that the number of orthologous genes would drop off with the increase of the number of chromosomes from different organisms. In contrast, <B>we overcome this issue in principle and results in a robust gene set</B>, only if the reference species are randomly selected and cover all the primary phylogenetic lineages. In addition, our <B>source data is derived from the initial set of experimentally-determined essential genes</B>, which ensures our results are not redundant. Hence, our results represent that the size and content of minimal gene set are stable and convergent. </P></DIV><DIV class="des_QandA QandA"><H3 class="des_h3">Virtues</H3><P>To construct an updated minimal gene set of bacteria, we renovate systematically the conventional process in the following two points. Firstly, it starts from the experimentally-determined essential genes and thus <B>the result is more reliable and accurate</B>. Secondly, we developed a new strategy called half-retaining to identify essential genes conserved among over half of the reference species, which made the <B>obtained minimal gene set more robust</B>.</P></DIV><DIV class="des_QandA QandA"><H3 class="des_h3">Innovation </H3><P>For innovation, our strategy, half-retaining, has blazed a new trail in three aspects. Based on the increasingly accumulated number of essential gene experimentally determined by genome scale gene in-activation way, we together with instructors’ groups proposed our strategy, half-retaining, to solve this problem<SUP>[9]</SUP>. Primarily, we <B>combined comparative genomics approach with data (essential genes) from experiments (gene inactivation)</B> instead of using theoretical method alone to construct a more complete minimal gene set than previous. In the second place, the minimal gene set that we defined would be confirmed <B>according to users' requirements and purpose</B>, rather than the only gene set in the previous. At last, the ubiquitous genes of minimal gene set determined by Koonin suffers from the flaw that enlarging the number of reference species would reduce the amount of genes retained in the minimal gene set. However, our half-retaining strategy <B>overcomes this problem and results in a robust gene set</B>. The number of selected genes is tending towards stability with the increase reference species by our strategy. </P></DIV><DIV class="des_QandA QandA wenx_div"><H3 class="des_h3">Reference</H3><P>[1] Foley, PL and Shuler, ML. Considerations for the design and construction of a synthetic platform cell for biotechnological applications. Biotechnol Bioeng, 2010, 105(1), 26-36.</P><P>[2] http://teacherpages.hallco.org/webpages/trennspies/index.cfm?subpage=1247337.</P><P>[3] Juhas M. On the road to synthetic life: the minimal cell and genome-scale engineering. Crit Rev Biotechnol, 2015, July 7. [Epub head of print]</P><P>[4] Juhas M, Eberl L, Church GM. Essential genes as antimicrobial targets and cornerstones of synthetic biology. Trends Biotechnol, 2012, 30(11)601-7.</P><P>[5] Mushegian AR, Koonin EV. A minimal gene set for cellular life derived by comparison of complete bacterial genomes. Proc Natl Acad Sci USA, 1996, 93(19):10268-73. </P><P>[6] Gil R, Silva FJ, Pereto J, Moya A. Determination of the core of a minimal bacterial gene set. Microbiol Mol Biol Rev, 2004, 68(3): 518-37.</P><P>[7] Koonin EV. Comparative genomics, minimal gene sets and the last universal common ancestor. Nat Rev Microbiol, 2003, 1(2):127-36.</P><P>[8] Zhang LY, Chang SH, Wang J. How to make a minimal genome for synthetic minimal cell. Protein Cell. 2010, 1(5):427-34</P><P><B>[9] Ye YN, Ma BG, Dong C, Zhang H, Chen LL and Guo FB. A novel bacterial minimal gene set and neo-construction of a general minimal metabolic network.  Scientific Reports, under review.(From instructors' group)</B></P></DIV></DIV><FOOTER><DIV class="footer_define"><DIV class="footer_logo"><P>@2015 Designed &amp; Developed by <SPAN class="green_em">UESTC_SOFTWARE</SPAN> iGEM</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>