<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UESTC_Software_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UESTC Software/Modeling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Modeling</TITLE></P><NAV class="dou_divide_nav"><DIV class="dou_divide_nav_wrap"><UL><LI><A href="https://2015.igem.org/Team:UESTC_Software">HOME</A></LI><LI class="choose"><A class="" href="javascript:void(0)">PROJECT<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Validation">Validation</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Results">Results</A></LI></UL><LI><A href="javascript:void(0)">DOCUMENTS<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/User_guide">User Guide</A></LI><LI style=""><A href="https://2015.igem.org/Team:UESTC_Software/Documentation">API</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Practices">HUMANPRACTICE</A></LI><LI><A href="javascript:void(0)">REQUIREMENTS<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Collaborations">Collaborations</A></LI><LI><A href="javascript:void(0)">TEAM<I class="iconfont"></I></A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:UESTC_Software/Attributions">Attributions</A></LI></DIV></NAV><DIV class="cd_half_section" id="c_h_s_modeling"><H1>Modeling</H1><P>Photo From © PEXELS</P></DIV><DIV class="back_to_top"><I class="iconfont"></I></DIV><DIV class="modeling_div"><SECTION><H3>Overview</H3><P>Modeling is one of the most important parts in our project. In the modeling, the data we used mainly from the CEG (Cluster of Essential Genes)<SUP>[1]</SUP> data base.In the <B>CEG</B>, genes’ category  were based on their corresponding <B>KEGG</B>(Kyoto Encyclopedia of Genes and Genomes) Orthology (KO) and <B>COG</B><SUP>[2]</SUP>(Clusters of Orthologous Group) category and function descriptions.</P><P><B>Ubiquity-retaining strategy</B><SUP>[3]</SUP> was first put up by Koonin .E V, we used half-retaining strategy to cover the shortage. Actually we use the 42% as the ratio not the half. Our modeling is to find it and prove it logically.</P><P>In the modeling, we use the knowledge of statistic. Find the relation of size of minimal gene set with number of organisms and HNR(Half Number Ratio). HNR is a ratio value that we use to replace the “half” in our strategy.  Then we construct the 3d modeling for that.</P><P>Besides, to get the <B>ideal value</B> of HNR, we use the  <B>multivariable linear regression</B><SUP>[4]</SUP> to get the fitting polynomial for the relation of OE rate with the HNR. From the polynomial, we find the appropriate <B>HNR</B> for our strategy, which has the highest OE rate and many other advantages compared with other HNR.Finally, we test our software’s <B>calculating time</B>:  start as we input organisms until we get the output result.</P><P>We divide our modeling into 3 steps.</P></SECTION><SECTION><H3>Step 1</H3><P>Firstly, we randomly choose 4 organisms from the data base(totally 29 organisms) as the experiment no.1 and rechoose  5 organism as experiment no.2 until we have 28 organisms in experiment no.25(We choose 10 times for one experiment, then get the <B>average of every data of it</B>) .We observed the trend of the size of minimal gene set from experiment no.1~25, and noticed a decrease trend when using the <B>ubiquity-retaining strategy</B>(Koonin) while a stabilization trend when using  the <B>half-retaining strategy</B>. Line chart as follow.</P><P>Obviously, the result of the half-retaining strategy seems more steady and more significant with the number of organisms’ increasing than the ubiquity-retaining strategy.</P><P>Secondly, to find the relationship between the <B>HNR</B>(Half Number Ratio) and the <B>size of minimal gene set</B>. We divide 100% into <B>100 groups</B> (1%~100%). Now we have 100 groups and 25 experiments. Then we respectively get the experiment no.1~25’s size of minimal gene set in every group. And finally get line And finally get line chart like follow two figures.(Cause there are too much data that we only list the 1/10,2/10~10/10’s result in Size of gene set – Number of organisms chart):</P><P>Our ideal size of minimal gene set is about 150~300. The conclusion is that the HNRs between interval <B>[30/100~51/100]</B> ’s size of minimal gene set are more close to the ideal amount. </P><P>Finally,to get a more particular knowledge of changing trend. We made a diagram to illustrate the <B>relation</B> of minimal gene set with number of organisms and HNR.</P><P>We use the <B>least square method</B> to get the <B>fitting polynomial</B>For every lines in the Size of gene set – Number of organisms chart(100 polynomials here). We choose the <B>3 order</B> polynomial. To get the 4 coefficients for the polynomial, we use the same method to get the <B>relation of coefficient with the HNR</B>. The 4 coefficients’  fitting polynomials are as follows:</P><P>The polynomial for the relation of the minimal gene set with number of organisms and HNR is as follows:</P><P>Now we have the polynomial to connect the relation of minimal gene set with number of organisms and HNR. The diagram is as follows:</P><P>From diagram above, we found an area that have a stable gene number when organisms number changed. It means use these half number ratio can get a stable result of minimal gene set. But to find the most stable HNR for the “half” retaining strategy, let’s move to step 2.</P></SECTION><SECTION><H3>Step 2</H3><P>From the Step 1, we divided 100% into <B>100 groups</B> (1%~100%). And we also have 25 experiments. Besides, before doing the experiments, we should define a new variable to measure whether the strategy is stable and effective. We call it <B>OE rate</B>( Overlapping Effective Rate). It’s calculated by this formula:</P><P><I>cUGS: the number of current experiment’s universal-gene-set-result</I></P><P><I>UGS29:the number of 29-organism-experiment’s universal-gene-set-result</I></P><P>This formula is to reduce the influence of  the Gene’s amount. More OE rate means the strategy has a more sufficient result.</P><P>Now we have 100 groups and 25 experiments. Then we respectively calculate experiment no.1~25’s <B>average of OE rate</B> in every group. And finally get this line chart like following figure.</P><P>Obviously, we could find there are several groups has an ideal result, and also some unexpected results.</P><P>Next, we calculate experiment no.1~25’s <B>variance</B> in every group. Line chart as follow:</P><P>From above figure, comparing the variances we could easily find the groups between 26/100~61/100’s variances are less than others which means strategy in this interval  is more steady. It’s fortunate that we nearly get the same conclusion with the best interval mentioned in step 1.</P><P>Fortunately, we got a pretty results. We use  the <B>least square method</B> to get the <B>fitting polynomial</B>. The new curve graph like this:</P><P>The polynomial is as follow:</P><P>After comparing the  order of polynomial，we choose the 4 order polynomial. The  <B>correlation coefficient</B> is 0.9703. According to the list. The polynomial is 97% to be accepted.</P><P>Next we calculate the <B>maximization</B> point as our ideal HNR (42%). That is how we define the ratio value of the retaining strategy.</P></SECTION><SECTION><DIV class="des_QandA QandA wenx_div"><H3 class="des_h3">Reference</H3><P><B>[1] Ye YN, Hua ZG, Huang J, Rao N, Guo FB. CEG: a database of essential gene clusters. BMC Genomics, 2013, 14:769. (From instructor’s group)</B></P><P>[2] Tatusov RL, Galperin MY, Natale DA, Koonin EV. The COG database: a tool for genome-scale analysis of protein functions and evolution. Nucleic Acids Res. 2000, 28(1):33-6.</P><P>[3] Mushegian AR, Koonin EV. A minimal gene set for cellular life derived by comparison of complete bacterial genomes. Proc Natl Acad Sci U S A. 1996, 93(19):10268-73.</P><P>[4] Cho MJ, Martinez WL, Martinez / Cho, Statistics in MATLAB: A Primer, CRC Press, Inc. 2014.</P></DIV></SECTION></DIV><FOOTER><DIV class="footer_define"><DIV class="footer_logo"><P>@2015 Designed &amp; Developed by <SPAN class="green_em">UESTC_SOFTWARE</SPAN> iGEM</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>