"
Page
Discussion
View source
History
teams
Log in
 
Team:St Andrews/project/RBS
From 2010.igem.org
University of St Andrews iGEM 2010
Welcome!
University of St Andrews iGEM 2010
Our first year at iGEM!
Home
Modelling
CqsA and Bistable Switch
RBS
test
Parts
Human Practices
Safety
Igem 2010 Home
Contents
1Motivation
2Ribosome Binding Site
3How we can measure and regulate protein expression
4Experiment and RBS Characterizing
5Lab work
6The result of our experiment:
7In conclusion
Motivation
One of the major problems that we have observed from the past teams it was not having fine control over protein expression. Since the Ribosome Binding Site sequence play a key role on rate of protein expression, here we face this problem by using a bioinformatic tool able to design the RBS sequence for a specific coding sequence with the wanted translation rate.
Ribosome Binding Site
The Ribosome Binding Site (RBS) is a sequence on the messenger RNA to which the ribosome can bind and initiate protein translation. In bacteria, the initiation of the translation requires a RBS sequence and a start codon. Ribosome binding sites can control the translation initiation rate and thereby protein expression [1]. Regulating the level of protein expression is required to connect genetic circuits [1] and control the energy flux through a metabolic pathway [2].
The RBS affects the translation rate of an open reading frame (ORF) in two main ways [3]. First, the rate at which ribosomes are attached to the mRNA and initiate translation is dependent on the RBS sequence. Second, the RBS influences the level of protein synthesis by modifying mRNA stability. The stability of the mRNA has impact on the steady state level of mRNA; for example, two mRNAs are being produced at the same rate but the stable one, the one with longer life, would produce more protein than the less stable one. Since the primary sequence and the secondary structure of an RBS (for example, the RBS could introduce a RNase site) can affect the stability of the mRNA, the RBS can affect the amount of mRNA translated and thus, the level of protein expression.
Translation in Bacteria involves four steps: initiation, elongation, termination and ribosome turnover [4]. Translation initiation is the rate-limiting step. Its rate is determined by molecular interactions, such as hybridization of the 16S rRNA to the RBS sequence, the binding site of tRNA-fMET to the start codon, the distance between the 16S rRNA binding site and the start codon (spacing) and the presence of RNA secondary structures that occlude either the 16S rRNA binding site or the standby site [1]. A thermodynamic model takes into account the strength of molecular interaction between an mRNA and the 30S ribosome complex to predict the translation initiation rate [1]. The model is based on two separated reversible transition states. The initial state is the folded mRNA and the free 30S complex and the final state is the assembled 30S complex on an mRNA. The difference in Gibbs free energy between these two states (ΔGtot) relies on the mRNA sequence surrounding a start codon (AUG or GUG). ΔGtot is more negative when there is an attractive interaction between ribosome and mRNA and ΔGtot is more positive when there is exclusive secondary structure [1].
How we can measure and regulate protein expression
The RBS Calculator is a piece of software programmed to design synthetic ribosome binding sites, facilitating a rational control of protein expression. The software has two parts: forward engineering and reverse engineering. Forward engineering incorporates a thermodynamic model into an optimizing algorithm for designing a ribosome binding site sequence which is predicted to drive protein translation at a particular rate. Reverse engineering predicts the relative translation rate of an existing RBS sequence with regard to upstream protein coding sequence.
RBS Calculator
Experiment and RBS Characterizing
We decided to design RBS sequences with different translation rate for GFPmut3b as it is the most popular fluorescent protein by using the RBS calculator. Therefore two RBS sequence was designed by the Forward engineering program with the specific rate. In addition the rate of one of the Anderson RBS catalog was measured by the Reverse engineering program.  We designed an experiment to measure the rate of protein expression by using these three RBS. The next step was ordering a sequence and then following the lab work for ligating these RBS with a promoter and GFP protein. After a successful ligation, the rate of protein expression of each biobrick was measured by flow cytometry and the negative control was I13401.
The designed RBS are:
<partinfo>BBa_K356105 SpecifiedComponents</partinfo><partinfo>BBa_K356106 SpecifiedComponents</partinfo>
The designed devices are:
BBa_k356101 <partinfo>BBa_K356101 SpecifiedComponents</partinfo> BBa_k356102 <partinfo>BBa_K356102 SpecifiedComponents</partinfo> BBa_k356103 <partinfo>BBa_K356103 SpecifiedComponents</partinfo>
Lab work
We fallow pretty much Endy protocols for everything.
Lab procedure includes:
Annealing primers,
primers contain promoter, RBS sequence and they desinged to have sticky ends
procedure:
Adding this materials:
8 μL of each of the concentrated primers
4 μL of salt solution (10 mM NaCl)
20 μL of water
and Anneal the primers by heating them at least 5°C above their melting point and cooling them down slowly in stages using a Thermocycler.
Digestion of I13401 with EcorI and XbaI
Adding this materials:
Water: 35 μL
XbaI: 1μL
EcorI:1μL
BSA: 0.5μL
DNA: 7.4 μL
Buffer 2 NEB: 2.5μL
And then icubet at 37°C for 2 hours and at 80°C for 20 minutes.
Ligation of primers into the I13401
Adding this materials:
Primers: 1 μL
plasmid: 7.4 μL
T4 Ligase: 0.5 μL
T4 Buffer: 1.0 μL
Transformation to E.coli
use 50 μL of the competent cells and 7.4 μL of DNA they mixed in ice for 30 minutes and heat shock at 42°C for 45 seconds and tehy puted back to ice for 2 minutes. then using soc soultion to bring the final volume to 0.5 ml and tehy incubted for one hour at 37°C and shaking at 225 rpm and tehy were plated on LB Agar with ampicillin.
Make overnight culture
5 ml of Lb  5μL of Ampicillin from the stock solution
Miniprep them and Digestion with EcorI and PstI and doing Gel purification
QUIAGEN protocl for Miniprep and Gel purification were used.
Degistion materials:
Water: 6μL
PstI: 1μL
EcorI:1μL
DNA: 10μL
Buffer D: 2μL
And then icubet at 37°C for 2 hours and at 80°C for 20 minutes.
Digestion of pSB1C3 with EcorI and PstI and Gel purification
Water: 6μL
PstI: 1μL
EcorI:1μL
DNA: 10μL
Buffer D: 2μL
And then icubet at 37°C for 2 hours and at 80°C for 20 minutes.
Ligation of the whole biobrick to the pSB1C3
Composed part: 15 μL
plasmid: 1 μL
T4 Ligase: 0.5 μL
Let the 10 μL solution sit at 22.5°C for 30 mins an then denature the ligase at 65°C for 10min.
Transformation of Ligated DNA and measuring the cells number and flourescence by Flow cytometry was the last procedure.
The result of our experiment:
The red lines represent the cells without RBS and the green line represent the cells with interested RBS.
BBa_K356101
BBa_K356102
BBa_K356103
In conclusion
The average fluorescent which produced by K356101 is the highest amount. Therefore, K356101 has a stronger Ribosome Binding Site than K356102 and K356103 as it was predicted by the RBS calcultor. Based on the result from Flow cytometry, the biobrick K356106 is around 6 times stronger than J61135 and 20 times stronger than K356105. Thus, we are confident to say that the RBS calculator prediction was correct.
Reference
1. Voigt, C., Mirsky, E., Salis, H. Automated desing of sysnthetic ribosome binding sites to control protein expression. Nature Biotechnology ,946-950 (2009)
2. Dueber, J.E. et al. Synthetic protein scaffolds provide modular control over metabolic flux. Nature. Biotechnolgy. 27, 753–759 (2009)
3. Bernstein, JA., Khodursky, AB., Lin, PH., Lin-Chao, S., Cohsen, SN., Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays.
Retrieved from " http://2010.igem.org/Team:St_Andrews/project/RBS "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
