<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_St_Andrews_project_laboratory_notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:St_Andrews/project/laboratory/notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:St_Andrews/project/laboratory/notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:St_Andrews/project/laboratory/notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:St_Andrews/project/laboratory/notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:St_Andrews/project/laboratory/notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:St Andrews/project/laboratory/notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="slideshow"><DIV id="slideshow-inner"><DIV class="slideshow-text"><H3>University of St Andrews iGEM 2010</H3><P>Welcome!</P></DIV></DIV><DIV id="slideshow-inner"><DIV class="slideshow-text"><H3>University of St Andrews iGEM 2010</H3><P>Our first year at iGEM!</P></DIV></DIV></DIV></DIV><DIV class="menu"><UL class="dropdown dropdown-horizontal"><LI><A href="https://2010.igem.org/Team:St_Andrews">Home     </A></LI><LI><A href="https://2010.igem.org/Team:St_Andrews/project/modelling">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:St_Andrews/project/laboratory">CqsA and Bistable Switch</A></LI><LI><A href="https://2010.igem.org/Team:St_Andrews/project/RBS">RBS</A></LI><UL><LI>test</LI></UL><LI><A href="https://2010.igem.org/Team:St_Andrews/project/parts">Parts</A></LI><LI><A href="https://2010.igem.org/Team:St_Andrews/project/ethics">Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:St_Andrews/project/safety">Safety</A></LI><LI><A href="https://igem.org/Main_Page"> Igem 2010 Home </A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Protocols"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Week_by_week_progress"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week by week progress</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Week_28th_June"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Week 28th June</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Week_5th_July"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Week 5th July</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Week_12th_July"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Week 12th July</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Week_19th_July"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Week 19th July</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Week_23rd_July"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Week 23rd July</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Week_30th_July"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Week 30th July</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Week_2nd_August"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Week 2nd August</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Week_9th_August"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Week 9th August</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Week_16th_August"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">Week 16th August</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Week_23rd_August"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">Week 23rd August</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Week_30th_August"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">Week 30th August</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Week_6th_September"><SPAN class="tocnumber">2.12</SPAN><SPAN class="toctext">Week 6th September</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Week_13th_September"><SPAN class="tocnumber">2.13</SPAN><SPAN class="toctext">Week 13th September</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H2><P>We found our notebook was being clogged by protocol notes. We therefore moved all of our protocols to our <A href="https://2010.igem.org/wiki/index.php?title=Team:St_Andrews/project/laboratory/protocols" class="external text" rel="nofollow">protocols section.</A></P><H2><SPAN class="mw-headline" id="Week_by_week_progress"> Week by week progress </SPAN></H2><H3><SPAN class="mw-headline" id="Week_28th_June"> Week 28th June </SPAN></H3><P>Chose roles within the team
</P><P>Initial lab preparations including making agar, LB broth, antibiotic stock solutions, copied out protocols.
</P><P>We decided to run test transformations on parts that we would not be using (Part BBa_K09899), as a transformation failure for a critical part would be a problem
</P><P>First transformation was carried on the 30th of June. This first step was to set the tone for the rest of the project. It failed. Part BBa_K09899. A possible cause of this may be the fact that there was no LB in the agar, this was due to a recent change away from ready mixed LB agar powder (read: we forgot to put nutrients in our media – facepalm).
</P><H3><SPAN class="mw-headline" id="Week_5th_July"> Week 5th July </SPAN></H3><P>Transformed BBa_C0077 resulting in another failure, although this time we made up LB agar properly.
</P><P>Possible cause – competent cells being used were over three years old. We decided to streak out some of the 2007 competent cells, along with glycerol stocks of regular DH5-alpha and a tube of competent cells that had been kept in the -20 freezer. The old competent cells turned out to be still alive and grew just as vigorously as the regular DH5-alpha and predictably the ones from the -20 were dead. We decided to make up our own competent cells while trying to transform more unuseful parts
</P><P>Appeared to have successful transformation on the 8th of July. This was a transform of J01003 using the 2007 competent cells. Miniprep and nanodrop readings confirmed presence of plasmid DNA (47.5ng/ul).
</P><P>We were trained on the safe use of ethidium bromide and the techniques of gel electrophoresis, in addition to the real blast from the past that was figuring out the imaging system software which was made in 1993, on similarly aged hardware.
</P><P>Following an analytical scale restriction digest of our plasmid DNA using EcoRI and PstI, gel electrophoresis showed plasmid and insert bands for each different amount of DNA we used in the digest. Success!
</P><H3><SPAN class="mw-headline" id="Week_12th_July"> Week 12th July </SPAN></H3><P>We began trying to transform parts that would be required for the bi-stable switch. Dave had a bad feeling about this.
</P><P>These parts were BBa_F1610, BBa_C0062 and BBa_R0062
</P><P>Transformation appeared to be successful (colonies observed) but this later proved to be incorrect.
</P><P>After setting up an overnight culture in antibiotic broth, we miniprepped each part to obtain plasmid DNA. Nanodrop readings varied wildly (this was later discovered to be because the Nanodrop is a difficult beast. It requires scrubbing thoroughly with ethanol and reinitialising before every reading for it to even think of giving you an accurate result, Otherwise the DNA concentration will appear to drop by around 20% with each reading taken)
</P><P>A restriction digest with EcoRI and PstI followed by gel electrophoresis showed…. “bugger all DNA”. This phrase became commonplace in the following weeks.
</P><H3><SPAN class="mw-headline" id="Week_19th_July"> Week 19th July </SPAN></H3><P>In this week it was discovered that we had been growing mutants, the transformation was not successful and our “competent” cells were in fact, not at all competent. We then found that the calcium chloride concentration used to make our competent cells was incorrect due to a calculation error (biologists can’t handle moles!) so we decided to make up the solution again and make more competent cells. After growing our culture overnight, we diluted 500ul into 50ml LB broth and grew for 2 hours. The protocol suggested measuring the absorbance with the Nanodrop to determine whether or not the cells were in the exponential phase of growth, which we did.
</P><P>The problem was, the nanodrop gave us a concentration tenfold lower than what we expected. On the second reading, we saw a negative absorbance reading. This was strange. We thought perhaps the cells were dying, but then the ever cynical Dave actually read the Nanodrop manual (unprecedented in this department I think) and found that the protocol gave absorbance values for a 10mm pathlength (the Nanodrop measures a 1mm path).
</P><P>Learned how to use the SORVALL RC6 Plus SS34 centrifuge (the washing machine sized monster upstairs). Made more competent cells.
</P><H3><SPAN class="mw-headline" id="Week_23rd_July"> Week 23rd July </SPAN></H3><P>Ordered parts we needed as stab cultures from the registry.
</P><P>Made new TBE buffer because an unnamed member of the team disposed of the last lot.
</P><P>We began having success using competent cells made with the addition of beta mercapto ethanol. This success was short lived, it also turns out that beta mercapto ethanol is toxic, highly pungent and must be diluted to a stock solution in a fume hood.
</P><H3><SPAN class="mw-headline" id="Week_30th_July"> Week 30th July </SPAN></H3><P>Began experimenting with isopropanol miniprep extraction procedure.
This gave massive DNA yields, but it appears that the separation of plasmid DNA from genomic DNA was poor.
</P><H3><SPAN class="mw-headline" id="Week_2nd_August"> Week 2nd August </SPAN></H3><P>Cultured B0015, E0240, R0062, C0062, B0032 and R0063 from agar stabs.
</P><P>We compared DNA extraction using isopropanol to extraction using quiagen columns and determined that the isopropanol extracts much more DNA but it as not all plasmid DNA.
</P><P>A gel was run which showed plasmid but no part.
</P><P>Began using NEB high efficiency turbo K12 competent E. coli. In general these made life much easier because they allow paarts to be transformed and miniprepped on the same day. They were also much more competent than the cells that we were using before.
</P><H3><SPAN class="mw-headline" id="Week_9th_August"> Week 9th August </SPAN></H3><P>Carried out a restriction digest on the results from the isopropanol extraction of E0240, B0032, C0062, R0063, R0062.
</P><P>
Transformed C0061 into new turbo cells, also transformed PUC19 control plasmid.
</P><P>Ran a 1.5% agarose gel of the restriction digests. Even though the isopropanol extracted DNA gave very high nanodrop readings, the bands on the gel weren't any brighter than usual. Insert bands of the correct size were also not observable. We concluded that the nanodrop could become wildly inaccurate at high DNA concentrations, and the isopropanol extraction technique increased the amount of genomic DNA contamination.  Decided it was best to verify insert size using PCR with VF2 and VR primers.
</P><P>Ran another gel, this time with regular miniprep products. Observed some faint bands at the correct insert sizes, but not for all parts. Found that the registry quality control gels also had very faint bands for small parts. Decided to trust the registry and go ahead with assembly.
</P><P>Colonies from C0061 transformation observed in ring shape, but did not extend to the edge (so unrelated to antibiotic plating) Attributed to cell plating technique. Decided to modify our spreading style, making the &quot;star&quot; in the centre first. Single colonies used to innoculate three 10ml tubes of broth.
</P><P>Performed miniprep of C0061 transformed previously. &quot;Gunk&quot; observed in culture tubes. Possibly biofilm associated with overgrowth of our Turbo E. coli. Ran restriction digests and gel on this part. Perfectly sized, bright insert bands observed for two of the three cultures taken.
</P><P>This was the week of the Open golf tournament so the university was shut down for part of the week while the golfers and their fans brought the town to a standstill.
</P><H3><SPAN class="mw-headline" id="Week_16th_August"> Week 16th August </SPAN></H3><P>Began biobrick assembly using NEB assembly kit (3A method).
</P><P>Transformed 5 µl of the ligation into Turbo E. coli.
</P><P>Primers; f,r,VR and VF2 arrived on the 17th of August allowing PCR based assembly to begin. The primers were all resuspended to make stock solutions.
</P><P>On the 18th  PCR amplification was carried out with VF2 and VR primers on B0032
</P><P>A miniprep was carried on the O/N cultures from the 16/8/10 assembly. 
</P><P>On the 19th 10mM stock solutions of dNTP stock solutions were made.
</P><P>PCR amplification of R0063, R0062 and B0015 was carried out. the resulting DNA was of very low concentration. 
</P><P>We ran a gel on digests of the biobrick composites plus some individual parts, the gel was not positive.
</P><P>We received a stern warning from Olivia that we should be more complete when completing our lab notebooks, a mental note of this information was made.
</P><P>We digested B0015 from the PCR product. We also digested R0063, R0062, B0015, C0062, C0061, E00240 and confirmed our results on a gel. There was a cock up with the promoter digestions so they were repeated. We then began ligating the different parts of our bistable switch together.
</P><H3><SPAN class="mw-headline" id="Week_23rd_August"> Week 23rd August </SPAN></H3><P>Transformed our ligations, made an overnight, colony PCR (Drew Endy protocol)
</P><P>
PCR purification, gel - gave a negative result.
</P><P>We had not been getting the DNA yields that we would have liked using the column miniprep technique so this week we experimented with ethanol precipitation using the following protocol
</P><P>
This protocol led to massively increased DNA yield however it appeared that the majority of it was not plasmid.
</P><H3><SPAN class="mw-headline" id="Week_30th_August"> Week 30th August </SPAN></H3><P>We inoculated LB using a C0261 colony and allowed it to grow. we followed this with a miniprep. 
</P><P>This week our CqsA gene arrived from Mr Gene. We transformed some cells with it and then did a miniprep. We then digested the miniprep using the standard NEB protocol. This was followed by ligating the CqsA into the standard biobrick plasmid, pSB1C3 
We also did some colony pcr and ran more gels.
</P><H3><SPAN class="mw-headline" id="Week_6th_September"> Week 6th September </SPAN></H3><P>Began by preparing CqsA for sequencing with VR primer. Clontech's In Fusion enzyme arrived so began In-Fusion PCR assembly by trying to optimise the PCR reaction. The upstream part gave a specific band exactly where we needed it but the downstream part in the destination plasmid gave many different products. We attributed this to the extension time being too short. We forgot for a moment that we were expecting an amplicon of &gt;3kb. Increased extension time and observed one bright specific band at just over 3kb. 
</P><H3><SPAN class="mw-headline" id="Week_13th_September"> Week 13th September </SPAN></H3><P>Attempted In Fusion reaction, after analysis on gel it appeared the DNA had vanished. This was perplexing. Re checked the pH of our TE buffer to make sure it was not destroying our DNA.  Retried In Fusion reaction. After another gel with no DNA decided we had no more time to spare. We then concentrated on verifying CqsA. Fatemeh began constructing her ribosome binding site test parts.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:St_Andrews/project/laboratory/notebook">http://2010.igem.org/Team:St_Andrews/project/laboratory/notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:St_Andrews/project/laboratory/notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:St_Andrews/project/laboratory/notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:St_Andrews/project/laboratory/notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:St_Andrews/project/laboratory/notebook&amp;oldid=190583" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>