<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Guanajuato_Mx_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Guanajuato Mx/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Guanajuato Mx</TITLE></P><DIV class="navbar"><DIV class="navbar-inner"><DIV class="container"><UL class="nav nav-collapse pull-right"><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx" class="dropbtn"> Team</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx">Our team</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Collaborations</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Attributions">Attributions</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Project</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/Project">Project Description</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Design">Design</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Experiments/MolecularLab">Experiments</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/ResultsAndDiscussions/MolecularLab">Results and Discussions</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/References">References</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Parts</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/BasicParts">Basic parts</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/CompositeParts">Composite parts</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Part Collection</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Model</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx">Link 1</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Link 2</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Human Practices</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/HP/Silver">Our Human Practices</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Integrated_Practices">Integrated Practices</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Policy</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Safety</A></DIV></LI></UL></DIV></DIV></DIV><DIV class="container profile"><DIV class="btn-group btn-group-justified"><DIV class="btn-group"><A type="button" class="btn btn-theme btn-primary" style="color: #fff;" href="https://2016.igem.org/Team:Guanajuato_Mx/Notebook/MolecularLab">MOLECULAR LAB</A></DIV><DIV class="btn-group"><A type="button" class="btn btn-theme btn-primary" style="color: #fff;" href="https://2016.igem.org/Team:Guanajuato_Mx/Notebook/PatchLab">PATCH LAB</A></DIV><DIV class="btn-group"><A type="button" class="btn btn-theme btn-primary" style="color: #fff;" href="https://2016.igem.org/Team:Guanajuato_Mx/Notebook/Protocols">PROTOCOLS</A></DIV></DIV><B><P style="font-size:3em">NOTEBOOK <SPAN style="font-size:.5em"> BY MONTH</SPAN></P></B><CENTER><B><P style="font-size:2em">PATCH LAB</P></B></CENTER><DIV class="showback"><H3 class="label-theme03">From June 6th to July 6th</H3><H4>We made different experiments to get the optimal conditions for our patch, such as agar concentration, trials with the nitrocellulose membrane and consistency of the polymer PVA.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 6th</H3><H4>180 mL of Nutrient broth was prepared with 0.9 g of peptone and 0.54g of meat extract. For solid medium, 15 mL of nutrient broth and 0.75 g of agar was necessary. To prove semi-solid medium, we trial different concentration, 4,6,8,10 and 12 (g/L) with 0.1,0.15,0.2, 0.25 and 0.3 g of agar respectively and all of them with 25 mL of nutrient broth.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 8th</H3><H4>70 mL of Nutrient broth was prepared with  0.35 g of peptone and 0.21g meat extract. Three new different concentrations of semi-solid medium was prepared, 2.5, 3 and 3.5 g/L each with 10 mL of broth and 0.07, 0.06 and 0.05 g of agar respectively.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 9th</H3><H4>In order to know which concentration of semi-solid medium was the one with better motility, we inoculated by puncture <I>E.coli</I>  at the center of the  tube, where the medium was placed. (All the concentrations were inoculated).</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 15th</H3><H4>We checked the results of motility and the chosen one was between 3 and 3.5 g/L.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 17th</H3><H4>We choose two different agar concentration: 3 and 3.5 g/L. 50 mL of nutrient broth with  0.25 g of peptone and 0.15 g of meat extract; 25 mL of semi-solid medium was prepared.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 18th</H3><H4>The prepared media were placed at the center of petri dishes, it was tested with 500, 700 and 800 microliters.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 20th</H3><H4>The petri dishes were incubated by adding a filter paper and a membrane.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 21st</H3><H4>We decided to repeat the experiment.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 22nd</H3><H4>All petri dishes were incubated at room temperature, starting at 5:00 p.m.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 1st</H3><H4>Membranes were in contact with the medium for 24 hours. Then, these were placed in liquid medium at 10:45 a.m.</H4><H4>About the second experiment, samples: M2, M3 and C3 showed the filter paper out of place. About the first experiment, the filter paper adhered to the tube wall was observed.</H4><H4>At 10:50 a.m., the tubes were vortexed and incubated at 37ºC. We observe the boxes with large filter papers.plated.</H4><H4>At 11:00 a.m., the plating of the second experiment began and transfer to liquid medium.</H4><H4>At 12:12 p.m., the positive control was plated and transferred to liquid medium.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 2nd</H3><H4>Results from the previous experiment were observed. Nutrient broth where filter paper pieces from experiment 1 were put revealed no turbidity except for sample 3 (its initial Petri Dish even showed more colonies than inoculated at the beginning). From experiment 2 no bacterial growth was observed. Positive controls showed turbidity and therefore, bacterial growth. All membranes from both experiments were transferred to tubes containing nutrient broth and were incubated at 37ºC for 48 h.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 4th</H3><H4>Tubes with nutrient broth were checked once more. Same results than those reported on July 2nd. Membranes from all experiments incubated in nutrient broth showed turbidity. Membranes from controls showed no bacterial growth except for Control 1 from Experiment 2.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 5th</H3><H4>PVA was dissolved in distilled water (1g per 10 mL), heated at 70ºC (constant stirring) for 25 and 40 min.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 6th</H3><H4>Both protocols for polymer (PVA) preparation were carried out. For more information related to these experiments, see Methodology and Results.</H4></DIV><CENTER><B><P style="font-size:2em">MOLECULAR LAB</P></B></CENTER><DIV class="showback"><H3 class="label-theme03">From June 14th to 23rd</H3><H4>We performed the inhibitory activity of the bacteriocin Nisin and Lysozyme.  According to:<UL><LI>
          Inhibitory assays for <I>S. aureus</I>, <I>E. coli</I>, <I>P. aeruginosa</I>, <I>E. faecalis</I> and <I>L. lactis</I> were performed. We followed the protocol according to Barboza-Corona <I>et al.</I> (2007). For Nisin, eight dilutions 1:1 were made with 50 mM citric acid with 50% ethanol, which were placed on the wells (100 μL) and the highest concentration was 250 u/ml. As negative control was used 50 mM citric acid only. For Lysozyme, a 1 mg/ml stock was used and this was the highest concentration. Also eight dilutions were made with distilled water and no negative control was performed.
        </LI><LI>
          The growing media for each bacteria was Tryptic Soy Broth (TSB, 0.7 g) plus 6 g of agar for 100 mL.
        </LI></UL></H4></DIV><DIV class="showback"><H3 class="label-theme03">June 15th</H3><H4>We checked the construction for LasR gene to see which enzymes were required for the cutting and we designed the oligos for this gene.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 16th</H3><H4>We did the dilutions for Nisin and Lysozyme and we performed all the protocol for the in vitro inhibitory activity.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 17th</H3><H4>We finished the last part of the protocol for the inhibitory activity and we designed the oligos for the RhIR promoter. The enzymes required were EcoRI and BamHI.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 20th</H3><H4>We did the inhibitory activity again because of contamination. We did new dilutions of Nisin without ethanol.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 22nd</H3><H4>We performed the protocol for biofilm formation of P. aeruginosa, according to Toodle (2011).</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 23rd</H3><H4>We performed the Lysozyme extraction from human blood using the protocol for RNA Extraction with TRIzol®  according to Invitrogen.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 27th</H3><H4>We did new dilutions with Nisin now diluted in distilled water because it gave us better results (data not shown). We also runned the gel for checking the quality of the RNA extracted. The gel was 1.2% Formaldehyde Agarose, ran at 1 hour at 90V, we do not observed anything because the time was too much.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 28th</H3><H4>We did a new electrophoresis gel was 1.2% Formaldehyde Agarose, ran 20 minutes at 90V, we observed only a one little band because the buffer was very strong.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 29th</H3><H4>We performed the Lysozyme extraction from human blood using the protocol for RNA Extraction with TRIzol®  according to Invitrogen. We did a electrophoresis gel was 1.2% Formaldehyde Agarose, ran 5 minutes at 90V, 15 minutes at 60V, 30 minutes at 49V. The samples were degraded.</H4></DIV><DIV class="showback"><H3 class="label-theme03">June 30th</H3><H4>We performed the amplification by a PCR reaction of the signal peptide ChiA from <I>B. thuringensis</I>, the promotor RhIR from <I>P. aeruginosa</I> and Alginate lyase from <I>B. thuringensis</I>.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 1st</H3><H4>We did another PCR reaction for the signal peptide (SP) for us to have a duplicate and it was the only successful reaction. We performed the Lysozyme extraction from human blood using the protocol for RNA Extraction with TRIzol®  according to Invitrogen.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 4th</H3><H4>We did a denaturant electrophoresis gel to RNA of human lysozyme samples. We do not observed anything because the gel had very much Bromure Ethidium.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 5th</H3><H4>We did the transformation of the pCOLD vector with (TOP10 Cells), Following next procedure:<OL><LI>
          Add into the electroporation cell 100 μL of  E. coli TOP10 + 1μL  of the ligation product.
        </LI><LI>
          Perform the electroporation at 2.5 kV.
        </LI><LI>
          Add immediately 1 mL of liquid LB without antibiotic into the cell after electroporation.
        </LI><LI>
          Resuspend by gently mix
        </LI><LI>
          Incubate for 1 hour at 37°C
        </LI><LI>
          Cultivate 100 μL of the bacteria in a Petri dish.
        </LI><LI>
          Centrifuge 1 min at 9,000 rpm.
        </LI><LI>
          Pour the supernatant and leave approximately 100 μL. Resuspend by pipetting.
        </LI><LI>
          Cultivate another Petri dish with this concentrated cell culture.
        </LI></OL>
      We did a electrophoresis gel was 1% Formaldehyde Agarose and prepared the electrophoresis camera with NaOH 0.1N for 1 hour, the water and buffer are prepared with DEPC. We load two samples of RNA of human lysozyme. We observed three bands, one for DNA and other two for RNA 16S &amp; 32S.
    </H4></DIV><DIV class="showback"><H3 class="label-theme03">July 6th</H3><H4>We performed the DNA extraction of the pCOLD according to Jena Bioscience, and we also performed the purification of SP by the PCR Purification Kit by the same company. We keep the ligation of pCOLD + SP overnight.</H4><H4>We made treatment for the ARN with DNAse and EDTA, after made a RT PCR with iScript protocol by Invitrogen. We also runned the gel for checking the quality of the DNA amplified. We do not observed anything.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 7th</H3><H4>We keep the ligation reaction on the thermoshaker until it reaches an approximate volume of 5 μL. We performed the dialysis procedure for this ligation DNA. We did the transformation of E. coli in order to introduce pCOLD +SP. We repeat the RT PCR and used a temperature gradient at 50° C to 60°C.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 8th</H3><H4>We performed the amplification of pCOLD + SP by PCR reactions and then checked it in an electrophoresis gel.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 11th</H3><H4>Since we did not obtain a good result of the amplification we repeated the entire protocol of ligation starting with the digestion of SP with BamHI and KpnI. We did the ligation with the same quantities of components. We used a Isolation of mononuclear cells by density gradient separation with Ficoll-hypaque for isolate human leukocytes, but we do not had a good results.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 12th</H3><H4>Finally we try to continue work with the older samples and repeat the RT PCR with iScript, but do not obtained bands in electrophoresis gel, and decided discard the Lysozyme and try with other compound.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 13th</H3><H4>The grown colonies were used for growing them into liquid LB medium in Falcon tubes. In total 24 colonies were grown. The tubes were kept overnight. In addition,  we prepared the solutions required for the protocol, Isolation of DNAp for use in sequencing. We inoculated 5 strains that possibly possess the alginate lyase gene, <I>B. thuringiensis 4Q7, B. thuringiensis HD73, B. thuringiensis HD1, B. thuringiensis Cry-B, B. Cereus</I>.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 14th</H3><H4>We transferred 1-1.5 μL of the medium in the Falcon tubes into Eppendorf tubes for making the extraction of DNAp. we extracted DNA for the cultures possibly possess the alginate lyase gene using a boiled extraction protocol. After we make a PCR for isolate alginate lyase gene, ran the samples at electrophoresis gel and we observed positive bands for <I>B. thuringiensis 4Q7</I> and <I>B.cereus</I>.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 15th</H3><H4>We retried the digestion of sp+pCOLD with BamHI y Kpn1.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 18th</H3><H4>We purified alginate lyase  and pCOLD with the Kit PCR Purification of Jenna Bioscience, considering five volumes of binding buffer.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 19th</H3><H4>We transformed TOP10 cells with alginate lyase DNA. following the standard protocol.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 20th</H3><H4>We proceeded the digestion of pCOLD+sp, and ligation of pCOLD+Alginate lyase.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 21st</H3><H4>We did the PCR of DNa from pCOLD+sp, this day we confirmed 4 colonies, with the protocol TAQ THUR. Then the product was purified.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 22nd</H3><H4>We confirm a one positive colony with the digestion with PstI y BamHI. In addition we quantified the product of sp. The same day we put the triple ligation of sp + alginate lyase and pCOLD.</H4></DIV><DIV class="showback"><H3 class="label-theme03">July 25th</H3><H4>We proceeded to transformed the ligation, perhaps until 29 th, we tried the ligation with non positive result.In addition we produced TOP10 cells  with the respectivily protocol.</H4></DIV><DIV class="showback"><H3 class="label-theme03">August</H3><H4>This month we paused the development of molecular assays, only a  few days preparing material to complete the experiments.</H4></DIV><DIV class="showback"><H3 class="label-theme03">September</H3><H4>We tried the triple ligation, and additional work for modelling and project.</H4></DIV><DIV class="showback"><H3 class="label-theme03">October</H3><H4>The first two weeks of the month, we worked on the submission of our parts.</H4></DIV></DIV><DIV class="footer"><DIV class="container"><P class="pull-left" style="font-size:.8em;"><A href="https://www.facebook.com/IGEM-Guanajuato-Mx-2016-1169541999738587/?fref=ts"> iGEM Guanajuato Mx</A></P><P class="pull-right" style="font-size:.8em;"><A href="mailto:igemgto@gmail.com"> igemgto@gmail.com</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>