<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Guanajuato_Mx_ResultsAndDiscussions_MolecularLab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Guanajuato Mx/ResultsAndDiscussions/MolecularLab</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Guanajuato Mx</TITLE></P><DIV class="navbar"><DIV class="navbar-inner"><DIV class="container"><UL class="nav nav-collapse pull-right"><LI class="dropdown"><A class="dropbtn"> Team</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/Team">Our team</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Collaborations">Collaborations</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Attributions">Attributions</A></DIV></LI><LI class="dropdown"><A> Project</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/Project">Project Description</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Design">Design</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Experiments/MolecularLab">Experiments</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/ResultsAndDiscussions/MolecularLab">Results and Discussions</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Notebook/MolecularLab">Notebook</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/References">References</A></DIV></LI><LI class="dropdown"><A> Parts</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/BasicParts">Basic parts</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/CompositeParts">Composite parts</A></DIV></LI><LI class="dropdown"><A> Model</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/TheoreticalFit">Theoretical Fit</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Difussion">Difussion through the nitrocellulose membrane</A></DIV></LI><LI class="dropdown"><A> Human Practices</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/HP/Silver">Our Human Practices</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Integrated_Practices">Integrated Practices</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Policy">Policy &amp; Safety</A></DIV></LI></UL></DIV></DIV></DIV><DIV class="container profile"><DIV class="btn-group btn-group-justified"><DIV class="btn-group"><A type="button" class="btn btn-theme btn-primary" style="color: #fff;" href="https://2016.igem.org/Team:Guanajuato_Mx/ResultsAndDiscussions/MolecularLab">MOLECULAR LAB</A></DIV><DIV class="btn-group"><A type="button" class="btn btn-theme btn-primary" style="color: #fff;" href="https://2016.igem.org/Team:Guanajuato_Mx/ResultsAndDiscussions/PatchLab">PATCH LAB</A></DIV><DIV class="btn-group"><A type="button" class="btn btn-theme btn-primary" style="color: #fff;" href="https://2016.igem.org/Team:Guanajuato_Mx/ResultsAndDiscussions/Modelling">MODELLING</A></DIV></DIV><B><P style="font-size:3em">RESULTS AND DISCUSSIONS</P></B><H1>Molecular Lab</H1><H2 style="font-size:1.3em">Biosensor design</H2><H4>    Our biosensors using as chassis <I>Escherichia coli</I> K12 and <I>Lactobacillus lactis</I> Nis+ are based on two modules, which show differences to the works previously reported (Hwan <I>et al.</I>, 2013; Saeidi <I>et al.</I>, 2011).  The first module has the genetic elements for the production of the protein LasR. This biomolecule is the sensor protein which binds to the homoserine lactone and activates the second module. This module has two devices, the alginate lyase gene and an antimicrobial peptide microcin or nisin.The use of microcin or lisin depends on whether <I>E. coli</I> or <I>L. lactis</I> are used as chassis, respectively (Figures 1 and  2, section PROJECT-&gt;DESIGN). The alginate lyase will destroy the <I>Pseudomonas aeruginosa</I> biofilm mainly made of alginate, and the bacteriocins or antimicrobial peptides will kill the pathogenic bacteria (Tielen <I>et al.</I>, 2013; Saeidi <I>et al.</I>, 2011). We selected the microcin gene to be expressed in <I>E. coli</I>, as this bacteriocin is produced by a probiotic <I>E. coli</I> G3/10 (Zschüttig <I>et al.</I>, 2012) and also because it was available in the Registry of Standard Biological Parts  (Bba_K565004). In addition, although nisin, a bacteriocin produced by <I>L. lactis</I> has been reported in some studies that does not have effect on Gram negative bacteria, e.g. <I>P. aeruginosa</I>, Giacommetti <I>et al.</I> (1996) found that a high concentration of this antimicrobial, has an effect on <I>P. aeruginosa</I> viability. We decided to test the possibility that increasing the concentration of nisin in <I>L. lactis</I> Nis+, might have an effect on <I>P. aeruginosa</I>. It is important to indicate that although we might have positive results, the use of nisin in a biopatch should be analyzed carefully, mainly because <A href="http://www.fda.gov/downloads/Food/IngredientsPackagingLabeling/GRAS/NoticeInventory/ucm266587.pdf" target="_blank">nisin is accepted by the FDA</A> to be used as food preservative. Additionally, we decided to include in our design the signal peptide sequences of the enzyme ChiA74 (spchiA74) and USP45, to secrete the alginate lyase and the microcin in <I>E. coli</I> and the alginate lyase in <I>L. lactis</I>, respectively. Although the spchiA74 is obtained from an enzyme produced by <I>Bacillus thuringiensis</I>, it is useful to secrete ChiA74 in <I>E. coli </I>(Barboza-Corona <I>et al.</I>, 2014). In addition, it has been previously reported that the USP45 allows the translocation of proteins in <I>L. lactis</I> (<A href="https://2014.igem.org/Team:Groningen" target="_blank">https://2014.igem.org/Team:Groningen</A>).</H4><H2 style="font-size:1.3em">Biosensor construction</H2><H4>    Unfortunately, we did not received our Biobricks kit, so we decided to ask for support from the Biosint_Mexico team of the “Tecnológico de Monterrey Campus Querétaro”, who provided us with the Biobricks Bba_K1365997, BBa_K649000, Bba_K565004, and Bba_K1365000, for the <I>lasR</I> gene, lasI promoter, microcin and nisin genes, respectively. Because our modules harbor genetic elements not available in the Biobricks collection, i.e. the chiA74p (or BtI-BtIIp), the alginate lyase gene and spchiA74, our efforts are focused on the amplification, cloning and test the functionality of those devices in <I>E. coli</I> previous to the module construction.</H4><H2 style="font-size:1.3em">Cloning of <I>chiA</I>74 and BtI-BtII promoters (chiA74p, BtI-BtIIp)</H2><H4>    Promoters <I>chiA</I>74p and BtI-BtIIp were amplified obtaining amplicons of ~600 bp and 400 bp, respectively (Figure 1A). Both promoters were cloned into the plasmid pHT3101, which is a shuttle vector with two replication origins, functional in both <I>E. coli</I> and <I>B. thuringiensis</I> (Lereclus <I>et al.</I>). In figure 1B (lane 3)  and 1C (lane 4) we show the release of the <I>chiA</I>74p (~0.6 kbp ) and BtI-BtII (~ 0.4 kbp) from the pHT3101 (~7.0 kbp) using EcoRI/PstI. Upstream to each promoter the gfp gene is cloned, for example when pHT3101-BtI-BtIIp-gfp is digested with PstI, a fragment of ~ 0.73 kb, corresponding to the gfp gene, is released (Figure 1D, lane 2). We are in in process of cloning both promoters into the pSB1C3 vector and also of confirming their functionality to regulate the expression of the green fluorescent protein in <I>E. coli</I>.</H4><H4><B>Figure 1. Cloning of the <I>chiA</I>74p into the pHT3101 vector.</B> (A) Lane M, 1 Kb plus DNA ladder; PCR amplification of <I>chiA</I>74p (lane 1, ~ 0.6 kbp), BtI-BtIIp (lane 2, ~ 0.4 kbp). Arrows indicate the position of the amplicons. (B) Lane 1, BtI-BtIIp; lanes 2 and 3, pHT3101-chiA74p digested with EcoRI/PstI. Asterisk and arrow show the position of the pHT3101 (~ 7 kbp) and chiA74p (~ 0.6 kbp), respectively. (C) Lane 1, 1 Kb plus DNA ladder; lanes 2, 3 pHT3101 digested with PstI; lane 4, digestion of pHT3101-BtI-BtII with EcoRI/PstI releasing a fragment of ~ 0.4 kbp corresponding to BtI-BtIIp. (D) Lane 1, 1 Kb plus DNA ladder; lane 2, pHT3101-BtI-BtIIp-gfp digested with PstI. Arrow shows the position of the gfp (~ 0.73 kb).
</H4><H2 style="font-size:1.3em">Amplification of Alginate lyase gene</H2><H4>    When we designed the biosensor, our first option to attack the alginate of the <I>P. aeruginosa</I> biofilms was the lysozyme (Rajaraman <I>et al.</I>, 2016).  At the beginning we designed primers to amplify the lysozyme from leukocytes, we obtained RNA and then synthesized cDNA for using as template to obtain the lysozyme gene. Nevertheless, we were not able to amplify the lysozyme gene, so we decided to look for other options. Alginate is hydrolyzed by alginate lyase (Farrell and Tipton, 2012), one of the main component of <I>P. aeruginosa</I> biofilm (Tielen <I>et al.</I>, 2013), but this  gene was not found in the registry. We analyzed the genome of one strain of <I>B. thuringiensis</I> subsp. <I>israelensis</I> AM65-52 (GenBank accession number: CP013275.1) and found a putative alginate lyase gene, so we designed primers to amplify this gene. It is important to indicate that there is no report of the cloning and characterization of an alginate lyase in <I>B. thuringiensis</I>. In this regard, we amplified for the first time a putative alginate lyase genes from a strain of <I>B. thuringiensis</I>, obtaining an amplicon of between 1.5  and 2.0 kpb in <I>B. thuringiensis </I>4Q7 and <I>B. cereus</I> (Figure 2). We decided to continue working with amplicon of 4Q7 and it was ligated into the pCold to generate the pCold-algy and was transformed in <I>E. coli</I>. Also we sequenced the <I>algy</I> gene at Langebio, México. As pCold allows that alginate lyase be 6X His-tagged, we are in process of purification with Ni column to study the effect on alginate and P. aeruginosa biofilms.</H4><H4><B>Figure 2. Amplification of the alginate lyase gene from <I>B. thuringiensis</I> 4Q7 and <I>B. cereus</I>.</B>Lane 1, DNA ladder (invitrogen); lane 2, total DNA from <I>B. thuringiensis</I> strain 1,lane 3; <I>B. thuringiensis</I> 4Q7; lane 4,  <I>B. thuringiensis</I> strain 3; lane 5, <I>B. thuringiensis</I> strain 4; lane 6, <I>B. cereus</I>; lane 7,  <I>B. thuringiensis</I> strain 5. Arrows indicate the position of the amplicons.</H4><H2 style="font-size:1.3em">Amplification of Signal Peptide sequence of ChiA74 and cloning into the pCold</H2><H4>    Analysis <I>in silico</I> of the putative alginate lyase of <I>B. thuringiensis</I>  subsp. <I>israelensis</I> AM65-52 (GenBank accession number: CP013275.1) indicate that this gene does not have a signal peptide (<A href="http://www.cbs.dtu.dk/services/SignalP/" target="_blank">http://www.cbs.dtu.dk/services/SignalP/</A>), for this reason we decided to control the secretion of these genes with the signal peptide sequence of ChiA74. The spchiA74 was amplified from the vector pEHchiA74 reported in Barboza-Corona <I>et al.</I> (2009), digested with KpnI/PstI and then ligated into the pCold-algy previously digested with  BamHI/Kpn (Figure 3). This construct is transformed in <I>E. coli</I> to generate <I>E. coli/pCold-spchiA74-algy</I>. This signal peptide will allow the secretion of alginate lyase in <I>E. coli</I>. We are in process to check the effectiveness of the spchiA74 to secrete the alginate lyase comparing the enzymatic activity of  <I>E. coli/pCold-algy</I> versus  <I>E. coli/pCold-spchiA74-algy</I>.</H4><H4><B>Figure 3. Amplifications of the signal peptide sequence (spchiA74) and generation of  pCold-spchiA74-algy.</B> Lanes 1 and 5, DNA ladder (invitrogen); lane 2 and 3, spchiA74 after PCR of pEHchiA74; lane 6, digestion of pCold-spchiA74-algy with  BamHI/PstI releasing the pCold and spchiA74; lane 7 and 8, pCold digested with  BamHI/PstI.
</H4><H2 style="font-size:1.3em"><I>P. aeruginosa</I> biofilm formation</H2><H4>    In order to check the effectiveness of the alginate lyase synthesized by <I>E. coli</I>, we standardized the biofilm formation  according with  O´Toole (2011), which is stained in blue (Figure 4). Once we check the effectiveness of the alginate lyase, we  will be able to test the effect on the P. aeruginosa biofilms.</H4><H4><B>Figure 4. Formation of the biofilm of <I>P. aeruginosa</I>.</B> Biofilm is stained in blue.</H4><H2 style="font-size:1.3em">Inhibitory effect of nisin</H2><H4>    We tested the effect of commercial nisin on pathogenic bacteria. We found inhibitory activity against <I>Enterococcus faecalis</I>, <I>Staphylococcus aureus</I>. This is not a surprise since nisin has effect on Gram positive bacteria. However, when we tested the activity against <I>P. aeruginosa</I> we did not  find  inhibition under the concentrations of the bacteriocin used in the assay (Figure 5). <I>P. aeruginosa</I> is a Gram negative bacterium and nisin has little effect unless high concentrations of the bacteriocin are used (i.e. 32 mg/L) (Giacommetti <I>et al.</I>, 1996) or it is used in combination with other molecules. It will be necessary to test higher nisin concentrations (more than 25 U) alone or in combination with alginate lyase against <I>P. aeruginosa</I> biofilm to be sure if the biosensor based on <I>L. lactis</I> overexpressing nisin might  be useful to detect and kill the <I>P. aeruginosa</I> or it will be necessary to change nisin by another bacteriocin, for example microcin. We did not find nisin activity against <I>E. coli</I> (data not shown).</H4><H4><B>Figure 5. Inhibitory effect of nisin against different pathogenic bacteria.</B>Well 1, 25 U; well 2, 12.5 U; well 3, 6.125 U; well  4, 3.1 U;well  5, 1.5U; well 6, 0.8 U.</H4><H2 style="font-size:1.3em">Conclusions</H2><H4>     Because our modules harbor genetic elements not available in the Biobricks collection, i.e. the chiA74p (or BtI-BtIIp), the alginate lyase gene and spchiA74, our efforts are focused on the amplification, cloning and test the functionality of those devices in E. coli previous to the module construction. The last, with the fact that we never received our Biobrick kits, delay our work. Alternatively,  we do not have previous reports about the cloning and expression of an alginate lyase from B. thuringiensis, in this regard this is the first time that this kind of gene is amplified from B. thuringiensis, the most important bioinsecticide worldwide. Additionally, alginate lyase genes have not been used previously in biosensors used  to destroy the P. aeruginosa biofilms, indicating that we are using a novel strategy to attack this pathogenic bacterium. Recently, we sent to sequence the alginate lyase gene cloned in this work to confirm and compare its sequence with other orthologous genes. Our data with nisin assays indicate that this bacteriocin has little or no inhibitory effect on P. aeruginosa at concentration used in this work. We need to test higher concentrations (higher than 25 U) of this antimicrobial peptide alone or in combination with alginate lyase against P. aeruginosa. If we did not have satisfactory results, it would be necessary to substitute nisin in the second module of our biosensor that uses L. lactis as chassis,  by other bacteriocin whose effectiveness against P. aeruginosa has been already confirmed previously, e.g. microcin S.</H4></DIV><DIV class="footer"><DIV class="container"><P class="pull-left" style="font-size:.8em;"><A href="https://www.facebook.com/IGEM-Guanajuato-Mx-2016-1169541999738587/?fref=ts"> iGEM Guanajuato Mx</A></P><P class="pull-right" style="font-size:.8em;"><A href="mailto:igemgto@gmail.com"> igemgto@gmail.com</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>