<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Guanajuato_Mx_Experiments_MolceularLab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Guanajuato Mx/Experiments/MolceularLab</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Guanajuato Mx</TITLE></P><DIV class="navbar"><DIV class="navbar-inner"><DIV class="container"><UL class="nav nav-collapse pull-right"><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx" class="dropbtn"> Team</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx">Our team</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Collaborations</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Attributions">Attributions</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Project</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/Project">Project Description</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Design">Design</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Experiments">Experiments</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/ResultsAndDiscussions">Results and Discussions</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/References">References</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Parts</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/BasicParts">Basic parts</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Composite parts</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Part Collection</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Model</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx">Link 1</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Link 2</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Human Practices</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/HP/Silver">Our Human Practices</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Integrated_Practices">Integrated Practices</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Policy</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Safety</A></DIV></LI></UL></DIV></DIV></DIV><DIV class="container profile"><B><P style="font-size:3em">EXPERIMENTS</P></B><H1>Molecular Lab</H1><H2 style="font-size:1.3em">Amplification and cloning of <I>chiA74p</I> and BtI-BtII promoters (<I>chiA74p</I>, BtI-BtIIp)</H2><H4>    The <I>chiA74p</I> is amplified from the pEHchiA74 (Castañeda-Ramírez et al., 2013) with the primers pChiA -1/FW-EcoRI (5´-CGGAATTCCTTTAATATATCTTTTTGTAGTTCCA-3) and pChiA -2/Rv-PstI (5´-  CAAAACTGCAGTTCTCCTTTCAAAATAAAAGATA-3´). Primers BtI-II-1/FW-EcoRI (5´- CGGAATTCTTACAATTCAAGATGAATTGCAGG-3´) and BtI-II/Rv-PstI (5´-CAAAACTGCAGAAGTTACCTCCATCTCTTTTATTAAG-3´) are used to amplify BtI-BtIIp from the <I>cry1Ac</I> gene from <I>Bacillus thuringiensis</I> HD73. Amplification is carried out as follows: 94°C for 2 min, followed by 30 cycles of denaturation at 94°C for 30 s, annealing at 55°C for 1 min, extension at 72°C for 40 s min and final extension of 72°C for 5 min. Amplicons are visualized in agarose gel, purified with the Kit PCR Purification” (Jena Bioscience) and then cloned into the pSB1C3 or pHT3101 (Castañeda-Ramírez et al. 2013) vectors previously digested with EcoRI/PstI to obtain pSB1C3-chiA74p and pSB1C3-BtI-BtIIp, or pHT3101-chiA74p and pHT3101-BtI-BtIIp.  Constructs are transformed into <I>Escherichia coli</I> TOP10.</H4><H4>    In order to test the functionality of the two promoters in <I>E. coli</I>, the green fluorescent (GFP) protein is cloned downstream of each promoter. This gene is amplified from the plasmid pGLO (Bio-Rad, Hercules CA, USA) (Barboza-Corona et al., 2014) with the oligonucleotides GFP/Fw-Pst-1 (5´CAAAACTGCAGATGGCTAGCAAAGGAGAAGAACTTT-3´) and GFP (Rv-Pst-2 (5´-CAAAACTGCAGTTATTTGTAGAGCTCATCCATGCC-3´) using the same conditions as above, but with an extension of 1 min.  Amplicon is digested with PstI and ligated into recombinant plasmids to generate pSB1C3-chiA74p-gfp and pSB1C3-BtI-BtIIp-gfp (or pHT3101-chiA74p-gfp and pHT3101-BtI-BtIIp-gfp. All recombinant plasmids are expressed in <I>E. coli</I> Top10. Recombinant bacteria are grown in LB plus ampicillin, and sonicated (30 Hz using a 20 kHz) in an ultrasonic processor (Sonic and Materials, Inc., Newtown, CT 06470-1614 USA). Intracellular proteins are obtained and fluorescence determined in a BioTek fluorometer (Synergi NTX, Biotek, Winooski, VT, USA).</H4><H2 style="font-size:1.3em">Amplification and cloning of alginate lyase gene and the signal peptide sequence of <I>ChiA74</I></H2><H4>    The alginate lyase gene (<I>alyg</I>) was amplified from <I>B. thuringiensis</I> 4Q7 using the oligonucleotides AlgLyaKpnI/Fw (5´-GGGGTACCATGACTCTTACTTTAATGGAGGGAATAG-3´), AlgLyaPstI/Rv(5´-AAAACTGCAGCGGCCGCTACTAGTATTTTCTTAGCTATCGTTACAATTAGTTTC-3´) that allow the amplification of alginate lyase from the start codon and the putative transcriptional terminator. Oligonucleotides were designed based on the genome sequence reported for <I>B. thuringiensis</I> subsp. <I>israelensis</I> AM65-52 (GenBank accession number: CP013275.1). Genomic DNA was obtained as previously described (Barboza-Corona et al., 2007). Amplification was carried out with an initial denaturation at 95°C for 5 min, followed by 39 cycles of denaturation at 95°C for 30 s, annealing at 55°C for 1 min, an extension at 72°C for 1.5 min and final extension of 72°C for 5 min. The amplicon was visualized in agarose gel, purified with the Kit PCR Purification” (Jena Bioscience) and then cloned into the pColdI (Takara BioInc, Otsu Shiga Japan), previously digested with PstI and KpnI. The construct was transformed into <I>E. coli</I> TOP10 at 2.5 KV using an <I>E. coli</I> pulser (BioRad). Recombinant plasmids were obtained from <I>E. coli</I> using the method described in Pospiech and Neumann (1995).</H4><H4>    Alternatively, the chiA74 signal peptide sequence (chiA74sp) was amplified from pEHchiA74 (Castañeda-Ramírez et al., 2013) using the primers SPChi-BamHI/Fw-5´-CGGGATCCATGGCTATGAGGTCTCAAAAATTCACACTGC-3´and SPChi-KpnI/Rv- 5´-GGGGTACCTGCTAATGCGAGATTTGGAGTAATAAAATTTGTGAG-3´ with the following conditions: 95°C for 5 min of initial denaturation, followed by 20 cycles of denaturation at 95°C for 30 s, annealing at 55°C for 1 min, an extension at 72°C for 20 s and final extension of 72°C for 5 min. Amplification was confirmed in agarose gel stained with ethidium bromide. Both the chiA74 signal peptide sequence and the alginate lyase gene are digested with BamHI/KpnI and KpnI/PstI, respectively, and ligated into the pColdI previously digested with BamHI/PstI, to obtain the construct pCold-chiA74sp-algy). This construct is transformed in E. coli to generate <I>E. coli/pCold-chiA74sp-algy</I>.</H4><H2 style="font-size:1.3em">Characterization of the alginate lyase device in engineered <I>E. coli- pCold-alyg</I> an formation of the  <I>Pseudomonas aeruginosa</I> biofilm</H2><H4>    <I>E. coli- pCold-alyg</I> was grown at 16 °C to induce the synthesis of alginate lyase. Cells were sonicated at amplitude of 30 Hz using a 20 kHz ultrasonic processor (Sonic and Materials, Inc., Newtown, CT 06470-1614 USA). The extract was centrifuged 30 min at 13000 g and the supernatant  was passed through a HiTrap Ni affinity column (GE Healthcare Bio-Sciences AB, Upsala Sweden) according to Casados-Vázquez et al. (2015).  The purified protein was analyzed by SDS-PAGE and Bradford assay. Purified protein was tested directly on biofilms of <I>P.aeruginosa</I> according with Saedi <I>et al.</I> (2011).</H4><H4>    In order to produce a biofilm of <I>P. aeruginosa</I>, the bacterium was grown overnight in LB media, a sample was diluted at 1:100 into fresh M63 minimal medium supplemented with magnesium sulfate, glucose and casamino acids. 100 mL of the dilution were added per well in a 96 well plate and then incubated for 24 at 37°C. In order to check the biofilm formation it was stained with 0.1 % solution of crystal violet in water according to Toodle (2011). In order to determine the effect of the alginate lyase secreted by the engineered E. coli/pCold-chiA74sp-<I>algy</I>, bacteria was co-cultured with <I>P. aeruginosa</I> and the effect on the biofilm was tested with a protocol described in Saedi <I>et al.</I> (2011).</H4><H2 style="font-size:1.3em">Inhibitory activity of the bacteriocin nisin</H2><H4>    <I>P. aeruginosa</I>, <I>Enterococcus faecalis</I>, <I>Staphylococcus aureus</I>, <I>E. coli</I> and <I>Lactococcus lactis</I> were grown overnight in BHI, TSB, LB and MRS media, respectively, at 37 °C, 180 rpm. Then 50 μL of each bacteria were transferred to 5 mL of new fresh medium,  and cultivated again during 2 h at the same conditions. Then 140 μL of each bacterium were added into 20 mL of soft agar (0.7 g TSB with 6 g agar), poured into the Petri dishes and were left to solidify. Once solid, wells of approximately 5 mm of diameter were made, and the bacteria were incubated at 37°C for 2 hours, and 100 μL of nisin dilutions (2.5 U) were added. Samples were incubated at 4°C for 12 h to allow the nisin diffusion and then at 37°C for 24 hours. For the nisin dilutions, nisin powder was diluted in 50 mM citric acid with 50% ethanol.</H4></DIV><DIV class="footer"><DIV class="container"><P class="pull-left" style="font-size:.8em;"><A href="https://www.facebook.com/IGEM-Guanajuato-Mx-2016-1169541999738587/?fref=ts"> iGEM Guanajuato Mx</A></P><P class="pull-right" style="font-size:.8em;"><A href="mailto:igemgto@gmail.com"> igemgto@gmail.com</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>