<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Guanajuato_Mx_ResultsAndDiscussions skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Guanajuato Mx/ResultsAndDiscussions</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Guanajuato Mx</TITLE></P><DIV class="navbar"><DIV class="navbar-inner"><DIV class="container"><UL class="nav nav-collapse pull-right"><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx" class="dropbtn"> Team</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx">Our team</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Collaborations</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Attributions">Attributions</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Project</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/Project">Project Description</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Design">Design</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Experiments/MolecularLab">Experiments</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/ResultsAndDiscussions">Results and Discussions</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/References">References</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Parts</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/BasicParts">Basic parts</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/CompositeParts">Composite parts</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Part Collection</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Model</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx">Link 1</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Link 2</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Guanajuato_Mx"> Human Practices</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Guanajuato_Mx/HP/Silver">Our Human Practices</A><A href="https://2016.igem.org/Team:Guanajuato_Mx/Integrated_Practices">Integrated Practices</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Policy</A><A href="https://2016.igem.org/Team:Guanajuato_Mx">Safety</A></DIV></LI></UL></DIV></DIV></DIV><DIV class="container profile"><DIV class="btn-group btn-group-justified"><DIV class="btn-group"><A type="button" class="btn btn-theme btn-primary" style="color: #fff;" href="https://2016.igem.org/Team:Guanajuato_Mx/ResultsAndDiscussions/MolecularLab">MOLECULAR LAB</A></DIV><DIV class="btn-group"><A type="button" class="btn btn-theme btn-primary" style="color: #fff;" href="https://2016.igem.org/Team:Guanajuato_Mx/ResultsAndDiscussions/PatchLab">PATCH LAB</A></DIV><DIV class="btn-group"><A type="button" class="btn btn-theme btn-primary" style="color: #fff;" href="https://2016.igem.org/Team:Guanajuato_Mx/ResultsAndDiscussions/Modelling">MODELLING</A></DIV></DIV><B><P style="font-size:3em">RESULTS AND DISCUSSIONS</P></B><H1>Molecular Lab</H1><H2 style="font-size:1.3em">Biosensor design</H2><H4>    Our biosensors using as chassis <I>Escherichia coli</I> K12 and <I>Lactobacillus lactis</I> Nis+ are based on two modules, which show differences to the works previously reported (Hwan <I>et al.</I>, 2013; Saeidi <I>et al.</I>, 2011).  The first module has the genetic elements for the production of the protein LasR. This biomolecule is the sensor protein which binds to the homoserine lactone and activates the second module. This module has two devices, the alginate lyase gene and an antimicrobial peptide microcin or nisin.The use of microcin or lisin depends on whether <I>E. coli</I> or <I>L. lactis</I> are used as chassis, respectively (Figures 1 and  2, section PROJECT-&gt;DESIGN). The alginate lyase will destroy the <I>Pseudomonas aeruginosa</I> biofilm mainly made of alginate, and the bacteriocins or antimicrobial peptides will kill the pathogenic bacteria (Tielen <I>et al.</I>, 2013; Saeidi <I>et al.</I>, 2011). We selected the microcin gene to be expressed in <I>E. coli</I>, as this bacteriocin is produced by a probiotic <I>E. coli</I> G3/10 (Zschüttig <I>et al.</I>, 2012) and also because it was available in the Registry of Standard Biological Parts  (Bba_K565004). In addition, although nisin, a bacteriocin produced by <I>L. lactis</I> has been reported in some studies that does not have effect on Gram negative bacteria, e.g. <I>P. aeruginosa</I>, Giacommetti <I>et al.</I> (1996) found that a high concentration of this antimicrobial, has an effect on <I>P. aeruginosa</I> viability. We decided to test the possibility that increasing the concentration of nisin in <I>L. lactis</I> Nis+, might have an effect on <I>P. aeruginosa</I>. It is important to indicate that although we might have positive results, the use of nisin in a biopatch should be analyzed carefully, mainly because <A href="http://www.fda.gov/downloads/Food/IngredientsPackagingLabeling/GRAS/NoticeInventory/ucm266587.pdf" target="_blank">nisin is accepted by the FDA</A> to be used as food preservative. Additionally, we decided to include in our design the signal peptide sequences of the enzyme ChiA74 (spchiA74) and USP45, to secrete the alginate lyase and the microcin in <I>E. coli</I> and the alginate lyase in <I>L. lactis</I>, respectively. Although the spchiA74 is obtained from an enzyme produced by <I>Bacillus thuringiensis</I>, it is useful to secrete ChiA74 in <I>E. coli </I>(Barboza-Corona <I>et al.</I>, 2014). In addition, it has been previously reported that the USP45 allows the translocation of proteins in <I>L. lactis</I> (<A href="https://2014.igem.org/Team:Groningen" target="_blank">https://2014.igem.org/Team:Groningen</A>).</H4><H2 style="font-size:1.3em">Biosensor construction</H2><H4>    Unfortunately, we did not received our Biobricks kit, so we decided to ask for support from the Biosint_Mexico team of the “Tecnológico de Monterrey Campus Querétaro”, who provided us with the Biobricks Bba_K1365997, BBa_K649000, Bba_K565004, and Bba_K1365000, for the <I>lasR</I> gene, lasI promoter, microcin and nisin genes, respectively. Because our modules harbor genetic elements not available in the Biobricks collection, i.e. the chiA74p (or BtI-BtIIp), the alginate lyase gene and spchiA74, our efforts are focused on the amplification, cloning and test the functionality of those devices in <I>E. coli</I> previous to the module construction.</H4><H2 style="font-size:1.3em">Cloning of <I>chiA</I>74 and BtI-BtII promoters (chiA74p, BtI-BtIIp)</H2><H4>    Promoters <I>chiA</I>74p and BtI-BtIIp were amplified obtaining amplicons of ~600 bp and 400 bp, respectively (Figure 1A). Both promoters were cloned into the plasmid pHT3101, which is a shuttle vector with two replication origins, functional in both <I>E. coli</I> and <I>B. thuringiensis</I> (Lereclus <I>et al.</I>). In figure 1B (lane 3)  and 1C (lane 4) we show the release of the <I>chiA</I>74p (~0.6 kbp ) and BtI-BtII (~ 0.4 kbp) from the pHT3101 (~7.0 kbp) using EcoRI/PstI. Upstream to each promoter the gfp gene is cloned, for example when pHT3101-BtI-BtIIp-gfp is digested with PstI, a fragment of ~ 0.73 kb, corresponding to the gfp gene, is released (Figure 1D, lane 2). We are in in process of cloning both promoters into the pSB1C3 vector and also of confirming their functionality to regulate the expression of the green fluorescent protein in <I>E. coli</I>.</H4><H2 style="font-size:1.3em">Amplification of Alginate lyase gene</H2><H4>    When we designed the biosensor, our first option to attack the alginate of the <I>P. aeruginosa</I> biofilms was the lysozyme (Rajaraman <I>et al.</I>, 2016).  At the beginning we designed primers to amplify the lysozyme from leukocytes, we obtained RNA and then synthesized cDNA for using as template to obtain the lysozyme gene. Nevertheless, we were not able to amplify the lysozyme gene, so we decided to look for other options. Alginate is hydrolyzed by alginate lyase (Farrell and Tipton, 2012), one of the main component of <I>P. aeruginosa</I> biofilm (Tielen <I>et al.</I>, 2013), but this  gene was not found in the registry. We analyzed the genome of one strain of <I>B. thuringiensis</I> subsp. <I>israelensis</I> AM65-52 (GenBank accession number: CP013275.1) and found a putative alginate lyase gene, so we designed primers to amplify this gene. It is important to indicate that there is no report of the cloning and characterization of an alginate lyase in <I>B. thuringiensis</I>. In this regard, we amplified for the first time a putative alginate lyase genes from a strain of <I>B. thuringiensis</I>, obtaining an amplicon of between 1.5  and 2.0 kpb in <I>B. thuringiensis </I>4Q7 and <I>B. cereus</I> (Figure 2). We decided to continue working with amplicon of 4Q7 and it was ligated into the pCold to generate the pCold-algy and was transformed in <I>E. coli</I>. Also we sequenced the <I>algy</I> gene at Langebio, México. As pCold allows that alginate lyase be 6X His-tagged, we are in process of purification with Ni column to study the effect on alginate and P. aeruginosa biofilms.</H4><H2 style="font-size:1.3em">Amplification of Signal Peptide sequence of ChiA74 and cloning into the pCold</H2><H4>    Analysis <I>in silico</I> of the putative alginate lyase of <I>B. thuringiensis</I>  subsp. <I>israelensis</I> AM65-52 (GenBank accession number: CP013275.1) indicate that this gene does not have a signal peptide (<A href="http://www.cbs.dtu.dk/services/SignalP/" target="_blank">http://www.cbs.dtu.dk/services/SignalP/</A>), for this reason we decided to control the secretion of these genes with the signal peptide sequence of ChiA74. The spchiA74 was amplified from the vector pEHchiA74 reported in Barboza-Corona <I>et al.</I> (2009), digested with KpnI/PstI and then ligated into the pCold-algy previously digested with  BamHI/Kpn (Figure 3). This construct is transformed in <I>E. coli</I> to generate <I>E. coli/pCold-spchiA74-algy</I>. This signal peptide will allow the secretion of alginate lyase in <I>E. coli</I>. We are in process to check the effectiveness of the spchiA74 to secrete the alginate lyase comparing the enzymatic activity of  <I>E. coli/pCold-algy</I> versus  <I>E. coli/pCold-spchiA74-algy</I>.</H4><H2 style="font-size:1.3em"><I>P. aeruginosa</I> biofilm formation</H2><H4>    In order to check the effectiveness of the alginate lyase synthesized by <I>E. coli</I>, we standardized the biofilm formation  according with  O´Toole (2011), which is stained in blue (Figure 4). Once we check the effectiveness of the alginate lyase, we  will be able to test the effect on the P. aeruginosa biofilms.</H4><H2 style="font-size:1.3em">Inhibitory effect of nisin</H2><H4>    We tested the effect of commercial nisin on pathogenic bacteria. We found inhibitory activity against <I>Enterococcus faecalis</I>, <I>Staphylococcus aureus</I>. This is not a surprise since nisin has effect on Gram positive bacteria. However, when we tested the activity against <I>P. aeruginosa</I> we did not  find  inhibition under the concentrations of the bacteriocin used in the assay (Figure 5). <I>P. aeruginosa</I> is a Gram negative bacterium and nisin has little effect unless high concentrations of the bacteriocin are used (i.e. 32 mg/L) (Giacommetti <I>et al.</I>, 1996) or it is used in combination with other molecules. It will be necessary to test higher nisin concentrations (more than 25 U) alone or in combination with alginate lyase against <I>P. aeruginosa</I> biofilm to be sure if the biosensor based on <I>L. lactis</I> overexpressing nisin might  be useful to detect and kill the <I>P. aeruginosa</I> or it will be necessary to change nisin by another bacteriocin, for example microcin. We did not find nisin activity against <I>E. coli</I> (data not shown).</H4><H1>Patch Lab</H1><H2 style="font-size:1.3em">PVA preparation techniques</H2><H4>    As previously mentioned on the methodology section, our patch consists of a multilayer matrix composed of: (i) Poly(vinyl alcohol), which serves as a protection mechanism of the complete system and also allows in/out oxygen transfer. (ii) petrolatum, which is a chemical that prevents the polymer to be directly in contact with the culture medium and also avoids the matrix  to be dissolved. (iii) Nutritive Agar + PVA, the former provides nutrients and medium for the microorganism, while the latter is added as a way to preserve the medium. (iv)Nitrocellulose membrane (0.22 μm) allows the diffusion of bacteriocin and alginate lyase, but avoids the spreading of the bacteria towards the injury.  (v) Autoadherible polymer, which holds the system hermetically. Poly(vinyl alcohol) (PVA) is a water-soluble synthetic polymer produced via hydrolysis of polyvinyl acetate to remove acetate groups. PVA film is a biomaterial that shows properties like inertness and stability, and it is considered safe and biocompatible in medicine and pharmacy (Muppalaneni and Omidian, 2013). In order to obtain the PVA film with suitable characteristics for our application; two experiments of polymerization were performed. In experiment 1, the polymer was dissolved in water; while in the experiment 2, HCl was used as a catalyst for the reaction of polymerization. In experiment 1 (no addition of HCl), all samples were able to form an elastic film, when drops of water were added the film tended to soften and with an excessive amount of water, the film became softener. In experiment 2 (wherein HCl was used as a catalyst), sample with only 100 μL of HCl appeared to be harder than samples from experiment 1 but it displayed more air bubbles within its structure. With 200 μL of HCl, the film showed less softening compared to those from the experiment 1. Even though the addition of HCl conferred waterproofness to the film, air bubbles were a major issue, so it was decided that the use of this acid was unnecessary. Therefore, for future assays, the mixture was heated at the conditions of time and temperature of the experiment 1 (60°C for 40 minutes), and afterwards it was poured all over an aluminium foil and kept at room temperature for 24 hours. With this procedure a firm and promising film was obtained to be used in the biopatch. On previous studies where the stability of PVA films were tested it was shown that the use of additives such as HCl affects the stability of the film altering its properties in the presence of different chemical compounds such as ammonium bisulfate (NH<SUB>4</SUB>HSO<SUB>4</SUB>), NaOH and formaldehyde (Koo <I>et al.</I>, 2011), which supports our choice to work with the conditions of the experiment 1.</H4><H2 style="font-size:1.3em">Selection of agar medium concentration</H2><H4>    The culture medium is an essential part for the product that we are trying to develop because it is going to provide the nutrients demanded by the microorganism; furthermore it is a matrix where our sensor and machinery will be placed. We observed cell motility of E. coli and appearance of agar concentrations of 2.5, 3, 3.5, 4 and 4.5 g/L. All of them presented good cell motility but the one that seemed to be more consistent was 3.5 g/L; 4 and 4.5 g/L had a solid consistency instead of a semisolid (data not shown) and lower concentrations looked more liquid (Table 1).</H4><H2 style="font-size:1.3em">Selection of PVA medium concentration</H2><H4>    For greater preservation of the culture medium which will contain the genetically modified bacteria, we used a combination of culture medium (optimal agar concentration of 3.5 g/L obtained in this work) and various concentrations of the polymer PVA (5, 10 and 15%, v/v), from the PVA obtained under the conditions determined in this work as ideal. The preservation of the culture medium was measured based on the height that it presented. According to the medium heights obtained, it was decided not to use polymer in the medium due to the low decrease of height in the negative control after ten days compared to those where certain amounts of PVA were added. Also, we observed that in the last day of the experiment (12th day) the heights of the medium with 5% of PVA and the negative control (no PVA added) were similar (Figure 6).</H4><H2 style="font-size:1.3em">Membrane permeability for microorganisms</H2><H4>    In order to prove that microorganisms are not able to pass through the nitrocellulose membrane which has a pore diameter of 0.22 µm, we simulated the biopatch being in contact with skin. To do so we inoculated nutritive agar medium with E. coli and then covered it with the membrane and two filter paper pieces in order to simulate skin. From experiment 1, samples did not show growth both in the tubes containing the filter paper pieces (Figure 7a) and in the Petri dishes where filter paper 1 was put. The only exception was sample 3 (Table 2) which came out to be contaminated due possibly to a deficient manipulation technique during the procedure, even in the initial Petri dish more colonies than inoculated appeared. From experiment 2, exactly as for experiment 1, all samples did not show bacterial growth with the exception of one, S1, which also displayed more colonies than inoculated at the beginning (Table 2). As expected, for the positive controls, where no membrane was added, tubes containing its filter paper pieces (1 and 2) revealed bacterial growth (turbid broth) as well as the Petri dish where filter paper 1 was put. All negative controls had no growth at all since no microorganism was inoculated (Figure 7b).</H4><H4>    These experiments validate that the nitrocellulose membrane with a pore diameter of 0.22 μm does not allow bacteria to move from the biopatch towards the burn injury because the size of <I>E. coli</I> is approximately 1 to 3 micrometers (Reshes <I>et al.</I>, 2008).</H4><H1>Modelling</H1><OL><LI>
      Our first model will study the behavior of how <I>P. aeuroginosa</I> grows in natural conditions similar to those in the skin environment and it will also study how our sensing system in  <I>E. coli</I> or <I>L.lactis</I> previously transformed with the genetic circuit grows. It will then analyze how <I>P. aeuroginosa</I> dies with the effect of an antimicrobial protein (lysozyme or alginate lyase protein).
    </LI><LI>
      Our second model will describe the concentration profile on the nitrocellulose membrane under the analysis of a single flux of homoserine lactone and antimicrobial agent (lysozyme or alginate lyase protein), on the other hand the analysis of a non-equimolar opposite diffusion of the two fluxes under study.
    </LI></OL><H2 style="font-size:1.3em">Conclusions</H2></DIV><DIV class="footer"><DIV class="container"><P class="pull-left" style="font-size:.8em;"><A href="https://www.facebook.com/IGEM-Guanajuato-Mx-2016-1169541999738587/?fref=ts"> iGEM Guanajuato Mx</A></P><P class="pull-right" style="font-size:.8em;"><A href="mailto:igemgto@gmail.com"> igemgto@gmail.com</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>