Team:UNC-Asheville/Description
About Us
The Project
Description and Results
Basic Parts
Composite Parts
Lab Notebook
Lab Book
En Silico
Model
InterLab Measurement Assay
Human Practices
Human Practices
Community Outreach
Collaborations
The Team
Attributions
iGEM UNC-Asheville Community Project Description
The Project
Fixing Our Backyard.
Degredation of TCE with novel metabolic pathway with soluble methane monooxyengenase and haloalkanoic dehalogenase
Trichloroethylene (TCE) is volatile organic solvent that has been used in dry-cleaning fluid and as an additive in paint removers, adhesives and spot removers.1 According to the EPA and the International Agency for Research Cancer (IARC), there is definitive evidence of TCE causing kidney cancer, and limited evidence for its correlation with liver cancer and non-Hodgkin lymphoma. Acute exposure to TCE of between 17-85 ppm can cause central nervous system depression with symptoms including ataxia, short-term memory loss and long term oculomotor dysfunction.2 According to the Agency for toxic substance and Disease Registry (ATSDR), TCE has been found in at least 1,045 of the 1,699 National Priorities List sites identified by the EPA.3
Trichloroethylene was initially developed by the Imperial Chemical Industries in Britain as an anesthetic to replace chloroform, but was soon replaced due to patients developing cardiac arrhythmias and hepatotoxicity. TCE found some use as a dry cleaning solvent, but was replaced by tetrachloroethylene in the 1950s. The compound found its greatest use as an degreaser for metal parts. In the United States, TCE was used heavily by both military agencies and NASA as way to clean engine fuel systems, including the F1 rocket engine which was used to in the Saturn V rockets of the 1960s and 70s.4
In 2001 the EPA proposed stricter regulation of TCE, setting off an interdepartmental argument with the Department of Defense, Department of Energy and NASA. After some political maneuvering, the EPA withdrew its risk assessment. Since then, the US military has sharply reduced its use of TCE and total US consumption of the solvent was ~100 tons as of 2006.5 The state of North Carolina is home to seven trichloroethylene spill sites, including Camp Lejeune US Marine Base - which some consider the largest contamination site in the country - and the CTS Superfund site in Asheville. This year, the UNC-Asheville iGEM team will focus on the biodegradation of the TCE found in the CTS Superfund site in south Asheville.
The pollution at the CTS site was only discovered after a family, who lived next door, noticed an oily substance in their drinking well. In 1999 a member of that family contracted brain cancer, and shortly thereafter another was diagnosed with skin cancer and two thyroid tumors.6 Initially, this was thought to be genetic, but one year later their granddaughter, who also lived on the property was diagnosed with a brain tumor. The maximum level of safe human consumption for TCE is 5 ppb. The EPA tested TCE levels of 830,000 ppb underneath the CTS site, and up to 34,000 ppb in nearby streams.7
The EAWAG Biocatalysis/Biodegradation Database was started by the University of Minnesota started in 1995 and includes information on 1200 compounds, 800 enzymes, 1500 reactions and 500 microorganism entries. 8 This database consists of a biochemical periodic table and rule-based Pathway Prediction System. Here it was discovered that several potential pathways exist for the degradation of TCE. Environmental biologists have long proposed methanophiles as potential biodegraders. Relying on methane as a carbon source requires a strong oxidizing bond which can break the C-H bond of methane and convert it to methanol. These proteins, called methane monooxygenases, are some of the strongest oxidizing proteins on the planet. When placed under copper limited environments, many methanophiles produce an alternative to the usual particulate methane monooxygenase (pMMO) to produce a soluble version (sMMO).9  Other proteins that are able to degrade TCE include toluene 2- monooxygenase, toluene dioxygenase, and trichloroethene reductive dehalogenase. Further background research revealed sMMO to be the fastest degrader of TCE and given that this protein had already been characterized as a biobrick by the Braunschweig 2014 team, we decided to focus on pathways that started with sMMO.
sMMO has been shown to degrade TCE both chloral hydrate, and an intermediate epoxide, which is short lived at room temperature. Our team decided to attempt the pathway which goes through the epoxide, and ends with glyoxylate. Glyoxylate is part of a native secondary anabolic pathway of E.coli which theoretically could mean that colonies expressing our metabolic pathway would route TCE into the glyoxylate cycle. Our synthetic pathway is linear and because of this, all of the chlorinated intermediates would eventually be fed into the native metabolism and turn into native metabolites. Additionally, the epoxide intermediate spontaneously degrades into four compounds CO2, formate, glyoxylate and dichloroacetic acid. The first three of these are these harmless and/or feed directly into E.coli metabolism. Dichloroacetic acid meanwhile, can be degraded by a protein known as haloalkanoic acid dehalogenase (dhlB-2), isolated from Xanthobacter autotrophicus strain GJ10. The DNA sequence for this protein was isolated and converted into a biobrick by the 2014 Sydney Australia iGEM team. While this team was unable to fully assemble their metabolic pathway intended to degrade dichloroethane, they did manage to characterize dhlB-2, and showed that it degraded both chloroacetate and 1,2-DCA. Other papers have shown that dhlB-2 can degrade dichloroacetic acid (DCAA) and we predicted that a functional dhlB-2 which could degrade other halogenated compounds would degrade DCAA.10
The last protein we needed to make our full pathway complete was GroEl/Es. The Braunschweig team described difficulties of getting sMMO to fold properly in JM109 E.coli, and used the Takara Chaperone protein folding kit to that end. Although this protein is naturally encoded by E.coli, it was demonstrated by the Braunschweig 2014 iGEM team that using the GroEl/Es plasmid under the inducible pBAD promoter allowed for the six subunits of sMMO to assemble properly. Rather than spend the money on the kit, our team decided to make GroEl/Es a Biobrick, with the intention to eventually make it a composite part with pBAD.
Our project now formalized, our team now turned to the task of convincing our university to fund our research project. UNC-Asheville is a small liberal arts university and this project is the first ever iGEM team. The most similar thing that had been done at our school was a First Robotics team, which had strong professor support and was extremely well funded. Our team was told three days before the registration fee deadline that no money was forthcoming and that we would have to fundraise our own registration fee using a website called Givecampus. Although we were frustrated from being explicitly told not to fundraise on our own and then having to do so last minute, we managed to raise our registration fee with two days to spare, with the help of a large $2000 donation secured from the NC Biotech Center.
As the school year drew to a close we began asking departments for a spare room to use, and were eventually directed to the physical chemistry lab in our chemistry department. When we moved into the room in May, we had $84 leftover from our Givecampus page and equipment which had been previously purchased out of pocket for DIY bio projects at home - a few pipettes, a small centrifuge, and a hacked together cell incubator fondly named Gorgeous George. Fortunately, our advisor allowed us to borrow some of his disposables and reagents until money we had raised from our university campus commission unlocked - $2099 which did not unlock until July 1st.
In May we designed our parts to be ordered from IDT, a kickback without which our project would have been very hard to complete. We decided to remain as faithful as possible to sMMO while splitting it into 2 parts to adhere to IDT’s 3kB gBlock limit. Some minor edits were made to non-coding regions due to the IDT website disallowing sequences with large homology. dhLB was printed exactly as it was listed on the Australian iGEM website. The sequence for GroEl/Es was pulled from the Takara website and BLASTed to deduce what species it came from. It was identical to E.coli GroEl/Es, so our team added a Biobrick prefix and suffix, edited out the restriction sites that were not compatible to the format and sent it off to be printed. While our sequences were being printed, the team worked on other possible revenue streams, favors and getting loaned equipment. It was unanimously decided that the pressure cooker we had purchased from eBay could be dropped in favor of the school’s autoclave.
For our first attempt at assembly, we decided to attempt ligation into the psB1A3 backbone provided with ampicillin resistance. It was concerning that this vector was not only linearized, but provided at the low concentration of 25 ng/uL. Approximately half of our IDT DNA was wasted attempting both a single insert digest/ligation and the double insert ligation protocol provided on the iGEM website. Out of concern for running out of DNA, PCR primers were designed which could amplify these parts. The standard oligos for the biobrick prefix and suffix were unable to amplify either sMMO1 or GroEl/Es successfully.
After many, many, many rounds of attempted PCR and ligation attempts, and additional attempts to PCR up the backbone, our team decided to assemble in a puc19 backbone (We foolishly did not consider the possibility of amplifying existing Biobricks and restricting out the necessary chloramphenicol backbone). When this failed, primers were designed which would conceivably amplify the IDT parts while giving them sufficient overlaps for Gibson assembly to work. No one in our school had ever done Gibson assembly before, so most of our advice came from the internet, at least until we went to the iGEM meetup in Virginia, which was extraordinarily useful. After two rounds of attempted PCR, a decision was made to simply re-order the parts with Gibson overlaps. Several attempts at Gibson assembly were made, but for a reason we never figured out, all that came back on diagnostic gels were the chloramphenicol backbones, or bands that were clearly not our parts. One last ditch effort was made to create biobricks by ordering dhlB under inducible lac promoters from Genscript, already in puc19 via a grant. With a few weeks to spare, we finally got our first Biobrick in the correct backbone, and collective nervous sighs of relief could be heard echoing off the halls of Zeis. In the eleventh hour a hail mary ligation produced bands that appeared to smmo1 and sMMO2 as well as a definitive, proving we had GroEl/Es in a backbone. While we had no time to sequence these, we did send them off to iGEM HQ (Godspeed you beautiful bricks…).
Results
Initially our team had planned to measure our project on mass spectrometry. However due to being unable to assemble the full pathway, equipment being offline for ¾ of summer and lack of correct columns, our team decided to scrap this in favor of a simple colorimetric assay. In the 11th hour, our team developed and tested an assay which measures free chloride form the competitive reaction of Fe2+ and Hg2+ for 2,4,6-Tris(2-pyridyl)- s-triazine(TPTZ). In this assay, mercury ions preferentially bind to TPTZ, producing a colorless solution. Upon the addition of chloride, mercury and chloride form HgCl2 precipitating out of solution, and Fe2+ binds to TPTZ forming a purple solution which can be measured by UV absorbance. Our team had to develop a homebrew version of this assay because at the cost of $250, the assays sold by sigma were too expensive. A patent for the assay was found online and emulated, though this recipe was eventually tweaked as it was discovered not to work as intended. Eventually a solution system was discovered which would could colorimetrically measured free chloride linearly over the range of 0.2mM - 2mM of free chloride. Our experimental results did not fall in this linear range, however qualitative data was collecting showing that our parts containing dhlB under inducible Lac expression did indeed produce more free chloride than either A) dummy plasmids containing GFP + IPTG and B) dhlB without IPTG. Although these results are extremely preliminary they are extraordinarily exciting for our team.
Chloride concentration is determined by a competition reaction between Hg2+ and Fe2+ for 2,4,6-Tris(2-pyridyl)- s-triazine(TPTZ). The preferred Hg-TPTZ complex exhibits no color. In the presence of chloride, Hg2+ forms HgCl2, which precipitates, allowing TPTZ to complex with Fe2+. The Fe-TPTZ complex results in a colorimetric (620 nm) product proportional to the chloride present.11
Links:
(2011, March 3) Toxic Substances Portal. Centers for Disease Control and Prevention. Centers for Disease Control and Prevention.
Environmental Health and Medicine Education. Centers for Disease Control and Prevention. Centers for Disease Control and Prevention.
Toxic Substances Portal - Trichloroethylene (TCE). Centers for Disease Control and Prevention. Centers for Disease Control and Prevention.
Zorba, P. The Use of Trichloroethylene at NASA’s SSFL Sites . NASA. NASA.
Ralph Vartabedian | Times Staff Writer. (2006, March 29) How Environmentalists Lost the Battle Over TCE. Los Angeles Times. Los Angeles Times.
Group, S. B. Buried Secrets: Investigation into the CTS Superfund Site. WLO
CTS of Asheville Superfund Site. Pollution Report Profile - EPA OSC Response.
Gao, J., Ellis, L. B. M., and Wackett, L. P. (2009) The University of Minnesota Biocatalysis/Biodegradation
Database: improving public access. Nucleic Acids Research 38.
Speitel, G. E., Thompson, R. C., and Weissman, D. (1993) Biodegradation kinetics of Methylosinus trichosporium OB3b at low concentrations of chloroform in the presence and absence of enzyme competition by methane. Water Research 27, 15–24.
Ploeg, J. V. D., Hall, G. V., and Janssen, D. B. (1991) Characterization of the haloacid dehalogenase from Xanthobacter autotrophicus GJ10 and sequencing of the dhlB gene. Journal of Bacteriology 173, 7925–7933.
Stephens, T. W. (1983, July 12) Assay method and reagent for the determination of chloride - American Monitor Corporation. FPO IP Research & Communities.
