"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Rutgers
From 2014.igem.org
Project
Team
Notebook
Safety
The Next Step in DNA Synthesis
De novo DNA Synthesis is extremely important to the world of Synthetic Biology, but the low efficiency of today's DNA Synthesis technology limits us.
We can only synthesize about 150 bases at a time, so new genes have to be stitched together from smaller strands, which adds a lot to the time and cost required. This needs to change.
A Primer on DNA Synthesis
Currently we use organic (anhydrous) solvents and reactive amidites to synthesize all custom DNA sequences, whether for antibody/enzyme engineering, multi-gene pathway building, or even genome construction . The process looks something like this:
This is one cycle in the process of DNA Synthesis. A solution of blocked nucleotides (whichever A, C, G, or T comes next in the sequence) is added to a group of immobilized DNA strands, and a single nucleotide is incorporated at the end of each strand. The trouble with this highly unnatural method is that each of the chemicals depicted
(trichloroacetic acid, 2-benzylthiotetrazole, acetonitrile)
causes some harm to the chemical structure of DNA, and this will limit the yield in a way that gets exponentially worse when building longer and longer pieces of DNA.
The process pictured above typically causes enough side-reactions to chemically ruin 1.5% of the existing DNA strands in each and every cycle (leaving 98.5% of them intact and with the correct sequence). This concept is called coupling efficiency , and it's the reason we can only synthesize up to 130 -150 bases at a time (or even 200 bases if you're willing to pay a whole lot extra).
Our Idea: Use Enzymes
We believe that both cost and efficiency could be vastly improved by using enzymes for de novo DNA Synthesis. Polymerases have evolved alongside nucleic acids for billions of years and developed beautiful two-metal-ion machinery that makes phosphate diester formation a piece of cake. In theory, this machinery could be applied to de novo DNA Synthesis to eliminate side-reactions and significantly simplify the process overall. A more natural, aqueous synthesis environment would lend itself to higher efficiency, too.
This process simplification could be beneficial on two counts: less complexity in required machinery would reduce capital costs, and less material requirements would reduce operational costs.
This is the enzymatic process that our team has envisioned. The enzyme is a template-independent polymerase called Terminal deoxynucleotidyl Transferase (TdT). It favors single-stranded 3' ends when it adds whatever dNTP's are available. Using the same principles as today's synthesis strategies, it could serve to catalyze the coupling step in each cycle. Upon inspection of this polymerase's binding pocket, it is apparent that even large blocking groups could fit snuggly without hindering the catalytic machinery, but we decided to test a small, simple blocking group for this project to begin with: the acetyl group.
We found that the acetyl group undergoes a bit of background hydrolysis, especially at high pH, so it wouldn't be the ideal blocking group. It does, however, perform well enough to make benchmarking possible so that we can establish a proof of principle for this strategy. We ran many assays, outlined below.
Accomplishments
These assays serve to elucidate the functionality of TdT in the conditions of our envisioned DNA Synthesis strategy. We used short, single-stranded primers (each composed of 15 thymidine residues) with 5' fluorescent tags (5'-FAM), and visualized the results on 22% polyacrylamide nondenaturing gels. The images are oriented with the loading wells above, so that the shortest oligos (which travel the fastest) end up furthest down in each lane. Each assay has a Control lane containing only the 15-length primer for reference. All lanes are labeled with neon-green text superimposed over the strange gray blotches of loading dye.
Tested the extension of single-stranded primers with TdT
The lanes are labeled with how many minutes the primers were incubated with TdT and dTTP.
This assay established that TdT will add ordinary (not blocked) thymidine triphosphate to the ends of the single-stranded primers. The 30-minute result has the longest oligos (having traveled the least distance), but it appears more condensed because it had less distance to effectively resolve the differing oligo lengths. In later assays we let the gels run as far as possible to avoid the bunching-up of long oligos.
Based on data in the literature, we estimate that a few hundred bases were added to each oligo (on average) under these reaction conditions (detailed in the Notebook->Protocols section). A DNA ladder would unfortunately be pretty expensive to create for these short, single-stranded, fluorescent primers, so we only interpret the relative activity of the enzyme between different lanes. This assay method doesn't allow us to quantify activity.
Tested the effect of pH on acetylated thymidine incorporation
The lanes are labeled with a number representing the pH that the reaction was run in. Different MOPS buffers were used for the different pH's, except the last lane, which used NEB's Tris buffer (pH 7.9). Each reaction was carried out for 10 minutes with primers, TdT, and acetylated thymidine monomers (specific conditions can be found in the Notebook->Protocols section). The last lane used unmodified thymidine triphosphate, as a reference.
This assay demonstrates that TdT adds less 3'-acetylated ("blocked") nucleotides at a lower pH. Aside from lane "7.0" (the anomaly), there is a clean step-ladder effect where longer oligos are produced at higher pH. There is still a significant spread of differing oligo lengths in the pH 6.5 lane, which indicates that enough hydrolysis occurs (of the acetyl blocking group) to allow for multiple additions. This conflicts with the findings of the later assays (below) that show no such spread of oligo lengths after reaction at pH 6.5.
In other (unlisted) assays, we established that TdT gets progressively less active at pH's lower than 7.9. The "dTTP+NEB Buffer" lane shows the activity of TdT with unmodified nucleotides at pH 7.9. Comparing the activity in this lane with the others shows that acetylated nucleotides cause some truncation throughout the distribution of oligo lengths. Another (unlisted) assay shows that the difference in TdT's activity between NEB's Tris buffer and a homemade MOPS buffer (both at pH 7.9) was indiscernible.
Tested the extent of hydrolysis of incorporated acetylated thymidines
The lanes are labeled according to their respective pH of reaction. Labels ending in "a" were carried out with acetylated thymidine for 10 minutes, before splitting the reaction volume in half and halting the reaction in the first half. The second half continues to react, with more unmodified thymidine added to the mixture. The two lanes labeled "ccc" were similar reactions, except that they had unmodified thymidine in the first reaction ("a"). These "ccc" reactions were carried out at a pH of 6.5 to show how active the enzyme is at that pH (with unmodified nucleotides).
This assay attempted to show that 3'-blocked nucleotides can effectively halt multiple additions, because of the terminal acetyl group. There is clearly a large amount of blocking-group-hydrolysis in the pH 7.9 reactions, leading to multiple additions when the extra unmodified nucleotide was added. The hydrolysis is limited enough at a pH of 6.5 that no significant amount of multiple additions occurs, even after adding extra unmodified nucleotide. These results are promising.
The second "7.9a" lane in the bottom image is mislabeled, it should read "7.9b".
Parts
Name
Type
Decription
Length
BBa_K1556000
coding
Mouse TdT
1590
This part was submitted late, and is not eligible for medals and awards.
Students
Kenny Kostenbader
Chem Eng
Scott Lazaro
Cell Bio & Neurosci
Wilson Wong
Mol Bio
Jay Patel
Chem Eng
Faculty
Sagar Khare
P.I.
Andrew Laudisi
Lab Manager
Attributions
Dr Jones is a professor in the Rutgers Chemistry Department who researches modified nucleotides. He gave valuable feedback on our initial project ideas, and suggested a way to create (and characterize) 3'-acetylated thymidine triphosphate in our lab. We attempted this synthesis (involving pyridine and acetic anhydride) and got promising results (via LC/MS), but then we found out that TriLink (below) could simply synthesize and purify it for us for free.
Arun Nayar was on last year's Rutgers iGEM team, so he helped train us with various lab protocols.
All of the pictured results on the Project page represent assays that were designed jointly by the students and Dr Khare, and carried out completely by the undergraduate team members in the Khare lab. Additional help was provided by other Rutgers students as well, namely: Diego Barreto, Wesley Okwemba, Neil Patel, Harsh Patel, and Samantha Ashley. The fluorescent gels were imaged using the "BioRad Gel Doc" machine in the Kalodimos lab (next door). Kenny did the web design and coding.
Trilink Biotech supported the project by custom-synthesizing acetylated thymidine triphosphate, and supplying it free of charge. Thanks TiLink!
NEB Inc supported the project by supplying a Terminal Transferase enzyme kit and a dNTP set, both free of charge. Thanks NEB!
Gen9 supported the project with a monetary donation. Thanks Gen9!
Protocols
Click to download a copy of each protocol that we followed and/or wrote this summer:
Testing the extent of hydrolysis of incorporated acetylated thymidines
Testing the effect of pH on acetylated thymidine incorporation
Chemical Transformation of Competent E. coli cells
Growing Cell Cultures Overnight and Creating a Glycerol Stock
NEB's Gibson Assembly protocol
Project phases
In May and June we brainstormed and investigated ways to create acetylated thymidine in lab. After much much research into the matter, we decided it would require far to many resources to synthesize acetylated nucleoside monomers (without phosphates present) followed by difficult and expensive triphosphorylation (the stepwise installation of the triphosphate moiety). It would be far too expensive. Instead, we tried Dr Jones' suggestion of mixing acetic anhydride with thymidine triphosphate in pyridine, and this seemed to work (there was a peak at the right molecular weight on our LC/MS run :). This phase of the project ended when we discovered that TriLink could custom-synthesize acetylated thymidine triphosphate, and they would do it for free :)
In July we expressed TdT in E. coli using the trusty pET-29b+ vector. Expression level was low (but still something!), so we were relieved when we found out that NEB would be happy to supply us with free a free TdT enzyme kit.
In August we ordered fluorescent primers to begin assaying with. It took the full month to test different methods for every step of the designed assay (especially fluorescence visualization) until we finally settled on the protocol(s) that we've uploaded (above)
September and October saw the bulk of our assaying. We had, however, neglected to move the mouse TdT gene from our pET-29b+ plasmid over to the standard iGEM pSB1C3 plasmid until early October. Our first Gibson Assembly did not work well (transformation yielded no colonies), so we ended up submitting our part late. That was a bummer.
All lab members underwent Biosafety Level 2 training with the Rutgers Environmental Health & Safety department.
We utilized only one organism: Escherichia coli K-12 DH5a
Public health and safety would be under little conceivable risk in the event of an accidental release of our biobrick part into the wild (in the form DNA or a transformed colony of cells), because the TdT enzyme expression is very low, and the enzyme is only capable of adding deoxynucleoside triphosphates to the 3' ends of DNA strands. Given that very little TdT would be produced, and that free 3' DNA ends are rare in living systems, we believe that the risk posed by our biobrick part is minimal.
Appropriate precautions were taken when casting and using the Polyacrylamide Electrophoresis gels for our various assays. Used gels were disposed of properly in acrylamide waste bins.
Our team's safety form can be found here .
Retrieved from " http://2014.igem.org/Team:Rutgers "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
