<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Macquarie_Australia_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Macquarie_Australia/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Macquarie_Australia/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Macquarie_Australia/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Macquarie Australia/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><UL><LI><A href="/Team:Macquarie_Australia">home</A></LI><LI><A href="/Team:Macquarie_Australia/Team">Team</A></LI><LI><A href="/Team:Macquarie_Australia/Project">project</A></LI><LI><A href="/Team:Macquarie_Australia/parts">Data Page</A></LI><LI><A href="/Team:Macquarie_Australia/Characterisation">Results/Characterisation</A></LI><LI><A href="/Team:Macquarie_Australia/Notebook">Lab Notes</A></LI><LI><A href="/Team:Macquarie_Australia/Safety"> Safety </A></LI><LI><A href="/Team:Macquarie_Australia/Attributions">Sponsors</A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Our_project"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Our project</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Background"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#References"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-4"><A href="#Aims"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Aims</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#The_Experiments"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">The Experiments</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Primer_Design"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Primer Design</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#BioBrick_Construction_Method"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">BioBrick Construction Method</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Alternative_Assembly_Method"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Alternative Assembly Method</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Our_project"><B>Our project</B></SPAN></H1><H2><SPAN class="mw-headline" id="Background"><B>Background</B></SPAN></H2><P>Phytochromes were first discovered in plants and were found to be associated with photo-perception - indeed, they are classed as photoreceptors. They are now known to exist in all plant species as well as some species of fungi and bacteria [2,5]. Plants use phytochromes as ambient light sensors involved for morphological development such as the germination of seeds, flower generation, responses to sunlight and position relative to shade [1,5,6,7,8,9]. Phytochrome proteins act as light sensors by absorbing red/far red light, however some phytochromes can detect light between the UV-A to far-red regions.  </P><P>Bacteriophytochromes act as photoreceptors for bacteria, mediating sensory responses to their light environment [4,6]. Bacteriophytochromes can also act as protein kinases triggered by light. These are able to modify or regulate various components in the cell such as protein activity and the synthesis of the light harvesting complex (chromophore) via the promotion of transcription factors. By doing so, they create what can be thought of as a “light switch”. The light switch bacteriophytochrome that is utilised by <I>Agrobacterium</I> and <I>Deinococcus</I> has the domain structure: PAS-GAF-PHY-HisKinase, and is able to cause a shift in conformation of the chromophore complex according to the wavelength absorbed. Essentially, what this means is that two different chromophores from two different phytochromes can activate two separate pathways due to their slight differences in light absorbance affinity. </P><P>For a light switch to be self-sufficient and fully functional two main components are required: 1) A heme oxygenase enzyme and 2) the bacteriophytochrome coupled with ribosome binding sites. The heme oxygenase is required for the degradation of the porphyrin heme (which is naturally occuring in <I>E.coli</I>) to form the linear tetrapyrrole biliverdin [10]. Biliverdin is then able to bind to the GAF domain of the bacteriophytochrome, forming the chromophore complex. A colour is thus produced according to the ratio of red to far-red light absorbed.</P><P>Phytochromes exist in a default off state (Pr) where the protein is biologically inactive [1]. In this state, the phytochrome absorbs red light and converts to the far red state (Pfr). A phytochrome in the far red state is considered to be in the active form, which absorbs far-red light. When in the Pr form, the phytochrome expresses a blue colour. If the balance of red/far red light absorbed shifts to the far red side, then the phytochrome is able to isomerise to a green colour [2,3,4]. Essentially, the phytochrome acts as an oscillating switch - hit it with red light and the chromophore emits a green colour, hit it with far red light and the chromophore emits a blue colour. This holds enormous potential for differential expression of two different kinase pathways (remember the phytochrome structure) while at the same time providing a clear detection mechanism.</P><P>The bacteriophytochrome vector will be transformed into <I>E. coli</I> competent BL21 (DE3) cells, which contains the T7 RNA polyermase for the expression of T7 promoter regulated operons. As <I>E. coli</I> do not contain a native heme oxygenase gene, it must be incorporated into the final assembled operon construct along with a T7 promoter and the bacteriophytochrome genes from <I>Deinococcus</I> and <I>Agrobacterium</I> for the &quot;light switch&quot; to be self-assembled and functional. </P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P>
1.	Park, C. M., Kim, J. I., Yang, S. S., Kang, J. G., Kang, J. H., Shim, J. Y., Chung, Y. H., Park, Y. M. &amp; Song, P. S. (2000). A second photochromic bacteriophytochrome from Synechocystis sp. PCC 6803: spectral analysis and down-regulation by light. Biochemistry 39, 10840-7.
2.	Karniol, B. &amp; Vierstra, R. D. (2003). The pair of bacteriophytochromes from Agrobacterium tumefaciens are histidine kinases with opposing photobiological properties. Proceedings of the National Academy of Sciences of the United States of America 100, 2807-12.
3.	Tarutina, M., Ryjenkov, D. A. &amp; Gomelsky, M. (2006). An unorthodox bacteriophytochrome from Rhodobacter sphaeroides involved in turnover of the second messenger c-di-GMP. The Journal of biological chemistry 281, 34751-8.
4.	Giraud, E., Fardoux, J., Fourrier, N., Hannibal, L., Genty, B., Bouyer, P., Dreyfus, B. &amp; Vermeglio, A. (2002). Bacteriophytochrome controls photosystem synthesis in anoxygenic bacteria. Nature 417, 202-5.
5.	Davis, S. J., Vener, A. V. &amp; Vierstra, R. D. (1999). Bacteriophytochromes: phytochrome-like photoreceptors from nonphotosynthetic eubacteria. Science 286, 2517-20.
6.	Wilde, A., Fiedler, B. &amp; Borner, T. (2002). The cyanobacterial phytochrome Cph2 inhibits phototaxis towards blue light. Molecular microbiology 44, 981-8.
7.	Rockwell, N. C., Su, Y. S. &amp; Lagarias, J. C. (2006). Phytochrome structure and signaling mechanisms. Annual review of plant biology 57, 837-58.
8.	Halliday, K. J. (2007). Photoreceptors and Associated Signaling I: Phytochromes. In Encyclopedia of Plant and Crop Science, pp. 881-884. 0 vols. Taylor &amp; Francis.
9.	Wagner, J. R., Zhang, J., Brunzelle, J. S., Vierstra, R. D. &amp; Forest, K. T. (2007). High resolution structure of Deinococcus bacteriophytochrome yields new insights into phytochrome architecture and evolution. The Journal of biological chemistry 282, 12298-309.
10.	Grangeiro, N. M., Aguiar, J. A., Chaves, H. V., Silva, A. A., Lima, V., Benevides, N. M., Brito, G. A., da Graca, J. R. &amp; Bezerra, M. M. (2011). Heme oxygenase/carbon monoxide-biliverdin pathway may be involved in the antinociceptive activity of etoricoxib, a selective COX-2 inhibitor. Pharmacological reports : PR 63, 112-9.</P><H2><SPAN class="mw-headline" id="Aims"><B>Aims</B></SPAN></H2><P>The objective in this project is to build and characterise a biological light switch in <I>E. coli</I>. This will involve construction of heme-oxygenase and bacteriophytochrome BioBrick parts. In 2010 the Macquarie Team cloned bacteriophytochrome from two sources. They showed that when one was expressed, it was functionally assembled when incubated with exogenous biliverdin and able to elicit a colour change when excited with far-red light. However, the part created is not directly usable as a BioBrick as it contains an internal <I>Eco</I>RI site (<I>Deinococcus radiodurans</I> phytochrome) and 2 <I>Pst</I>I sites (<I>Agrobacterium tumefaciens</I> phytochrome). As biliverdin is not native to <I>E. coli</I>, the addition of heme oxygenase is required for the synthesis of bilivedin, enabling the self-assembly of the light switch. The 2010 Macquarie Team had managed to combine a ribosome binding site (RBS) to all 3 proteins of interest.
</P><P>
Hence, the aims of the team this year are to complete the light switch construction:

1. Assemble 3 fully functional BioBricks which are functionally expressed in <I>E.coli</I>.

2. Remove the restriction sites from the bacteriophytochrome which are incompatible with BioBrick assembly.

3. Assemble an operon consisting of the heme-oxygenase and bacteriophytochrome BioBrick parts.

4. Optimise the gene expression from the operon such that the bacteriophytochrome light switch works without requiring the addition of biliverdin. </P><H1><SPAN class="mw-headline" id="The_Experiments"><B>The Experiments</B></SPAN></H1><H2><SPAN class="mw-headline" id="Primer_Design"><B>Primer Design</B></SPAN></H2><DL><DD><TABLE border="1" cellpadding="5" cellspacing="0"><TBODY><TR><TH>Primer
</TH><TH>Sequence
</TH><TH>Melting temp (°C)
</TH></TR><TR><TD>BB-FWD
</TD><TD>5'- <B>GAA TTC</B> GCG GCC GCT <B>TCT AGA</B> -3'
</TD><TD align="center"> 59.7
</TD></TR><TR><TD>BB-REV
</TD><TD>5'- <B>CTG CAG</B> CGG CCG CT<B>A CTA GT</B>A -3'
</TD><TD align="center"> 61
</TD></TR><TR><TD>T7-BB-FWD
</TD><TD>5'- <B>GAA TTC</B> GCG GCC GCT <B>TCT AGA</B> TCT CGA TCC CGC G -3'
</TD><TD align="center"> 69.4
</TD></TR><TR><TD>T7-BB-REV
</TD><TD>5'- <B>CTG CAG</B> CGG CCG CT<B>A CTA GT</B>A GAG GGG AAT TGT TAT -3'
</TD><TD align="center"> 66.5
</TD></TR><TR><TD>HO1-BB-FWD
</TD><TD>5'- GCG GCC GCT <B>TCT AGA</B> CTT TAA GAA GGA GAT ATA C -3'
</TD><TD align="center"> 61.9
</TD></TR><TR><TD>HO1-BB-REV
</TD><TD>5'- GCG GCC GCT <B>ACT AGT</B> ACT TTC GGG CTT TGT TAG C -3'
</TD><TD align="center"> 67
</TD></TR><TR><TD>DR-BB-FWD
</TD><TD>5'- TCG CGG CCG CT<B>T CTA GA</B>A GGA GGG CTG CTA TGA GC -3'
</TD><TD align="center"> 71
</TD></TR><TR><TD>DR-BB-REV
</TD><TD>5'- GCG GCC GCT <B>ACT AGT</B> ATC AGG CAT CGG CGG CTC CCG G -3'
</TD><TD align="center"> 68.7
</TD></TR><TR><TD>DR-M-FWD
</TD><TD>5'- TTG CTG ATT CTG GA<I>G</I> TTC GAG CCG ACG GAG -3'
</TD><TD align="center"> 63.5
</TD></TR><TR><TD>DR-M-REV
</TD><TD>5'- CTC CGT CGG CTC GAA <I>C</I>TC CAG AAT CAG CAA -3'
</TD><TD align="center"> 61.7
</TD></TR><TR><TD>AT-BB-FWD
</TD><TD>5'- CGC GGC CGC T<B>TC TAG A</B>GA TTA GGA GGG CTG CTA TGA G -3'
</TD><TD align="center"> 69.1
</TD></TR><TR><TD>AT-BB-REV
</TD><TD>5'- GCG GCC GCT <B>ACT AGT</B> AGA TTT CAG GCA ATT TTT TCC -3'
</TD><TD align="center"> 64.3
</TD></TR><TR><TD>AT-M1-FWD
</TD><TD>5'- GTC CCT GCA TTA TCT <I>T</I>CA GAT GAT ATC AGA -3'
</TD><TD align="center"> 53.8
</TD></TR><TR><TD>AT-M1-REV
</TD><TD>5'- TCT GAT ATC ATC TG<I>A</I> AGA TAA TGC AGG GAC -3'
</TD><TD align="center"> 55.7
</TD></TR><TR><TD>AT-M2-FWD
</TD><TD>5'- GTG TTT CAG AGG CT<I>T</I> CAG CGG GTG GAG GA -3'
</TD><TD align="center"> 63.9
</TD></TR><TR><TD>AT-M2-REV
</TD><TD>5'- TCC TCC ACC CGC TG<I>A</I> AGC CTC TGA AAC AC -3'
</TD><TD align="center"> 64.5
</TD></TR><TR><TD colspan="3"> BB - BioBrick compatible, M - Mutagenic primer (<I>Italics</I> - base mutation), <B>Bold</B> - Restriction sites
</TD></TR></TBODY></TABLE></DD></DL><P>Primers were designed such that the amplicon would consist of at least the <I>Xba</I>I site or <I>Spe</I>I site and be restricted by the length of the primers. As such, an <I>Eco</I>RI or <I>Pst</I>I restriction digest would not work for our BioBrick construction, resulting in the <I>Xba</I>I and <I>Spe</I>I digestion for the ligation. This creates a problem - the inserted coding sequence would be incorrectly orientated - <I>Spe</I>I ligating with <I>Xba</I>I instead. To solve this problem, we've decided to screen at least 10 colonies that grow on the antibiotic resistant plates, and perform an <I>Xba</I>I digest to screen for inserts that are in the right orientation. Colony PCR was trialled, but due to the lack of difference between the right and wrong orientation of inserts (only 2 single base difference), the screening method was abandoned (see <A href="https://2011.igem.org/Team:Macquarie_Australia/Notebook#26.2F8.2F11" class="external text" rel="nofollow">notebook</A>).
</P><H2><SPAN class="mw-headline" id="BioBrick_Construction_Method"><B>BioBrick Construction Method</B></SPAN></H2><P>1 - Qiagen Gel extraction kit, 2 - Sigma PCR clean up kit, 3 - Qiagen Spin Miniprep kit
</P><P>Along with the PCR clean up of the pre-ligation products, the cut plasmid backbone was separated on a gel, and gel purified to remove any red fluorescent protein gene products.
</P><H2><SPAN class="mw-headline" id="Alternative_Assembly_Method"><B>Alternative Assembly Method</B></SPAN></H2><P>In contrast to the assembly protocol as proposed by iGEM, we've adopted a slightly different protocol. This can be applied when the antibiotic backbone of one of the parts remains the same (such as preparation for submission), while inserting the desired part. The screening of successful assembly can either be by protein expression (eg. T7 promoter part + GFP = fluoresce), PCR using BioBrick primers, or by plasmid extraction (successful assembly would have a larger band size). To enhance the efficiency of the assembly, alkaline phophatase treatment should be performed on the plasmid with the selection resistance to minimise re-ligation.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Macquarie_Australia/Project">http://2011.igem.org/Team:Macquarie_Australia/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Macquarie_Australia/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Macquarie_Australia/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Macquarie_Australia/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Macquarie_Australia/Project&amp;oldid=223380" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>