<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Montpellier_Toolbox skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Montpellier/Toolbox</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menus"><NAV><A href="https://2018.igem.org/Team:Montpellier">Home</A></NAV><NAV><A>Project</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Description">Description</A><A href="https://2018.igem.org/Team:Montpellier/Antisperm/antibodies">Antisperm antibodies</A><A href="https://2018.igem.org/Team:Montpellier/Peptides">Peptides</A><A href="https://2018.igem.org/Team:Montpellier/L_jensenii"><I>Lactobacillus jensenii</I></A><A href="https://2018.igem.org/Team:Montpellier/Results">Conclusions</A><A href="https://2018.igem.org/Team:Montpellier/Perspectives">Perspectives</A></DIV></NAV><NAV><A>DryLab</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Model">Modeling</A><A href="https://2018.igem.org/Team:Montpellier/Tracking/Software">Tracking Software</A></DIV></NAV><NAV><A>WetLab</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Protocols">Protocols</A><A href="https://2018.igem.org/Team:Montpellier/Toolbox">Toolbox</A><A href="https://2018.igem.org/Team:Montpellier/WetLab_ASA">Antisperm Antibodies</A><A href="https://2018.igem.org/Team:Montpellier/WetLab_Peptides">Peptides</A><A href="https://2018.igem.org/Team:Montpellier/Demonstrate">Sperm Motility Assay</A><A href="https://2018.igem.org/Team:Montpellier/InterLab">InterLab</A></DIV></NAV><NAV><A>Human Practices</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Human_Practices">Integrated human practices</A><A href="https://2018.igem.org/Team:Montpellier/Public_Engagement"> Education &amp; Public Engagement</A><A href="https://2018.igem.org/Team:Montpellier/Ethics">Ethics</A></DIV></NAV><NAV><A>Parts</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Parts">Parts</A></DIV></NAV><NAV><A>Team</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Team">Members</A><A href="https://2018.igem.org/Team:Montpellier/Attributions">Attributions</A><A href="https://2018.igem.org/Team:Montpellier/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Montpellier/Sponsors">Sponsors</A></DIV></NAV><NAV><A>Achievements</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Award">Award</A><A href="https://2018.igem.org/Team:Montpellier/Medals">Medals</A></DIV></NAV></DIV><P>This toolbox contains all kind of information that can be useful for working with <I>Lactobacillus jensenii</I>. First, we provide culture conditions that we used and were proven to be working. Second, it contains a transformation protocol that we optimized and found to be the most efficient of all those we tested. Finally, the toolbox contains all DNA sequences (vectors and biobricks parts including <I>L. jensenii</I> specific promoters) that we used in <I>L. jensenii</I> for our project. We hope that this toolbox will facilitate the use of <I>L. jensenii</I> by other iGEM and research teams. We plan to extend the toolbox in the future by providing the community with a physical distribution of well-characterized vectors and regulatory elements.</P><H2>How to grow <I>Lactobacillus jensenii</I>?</H2><H3>Strain and medium</H3><P>For our project we used the following strain and culture medium :</P><H4>Bacteria</H4><P><I>Lactobacillus jensenii</I> Gasser et al. (ATCC® 25258™)</P><H4>Culture</H4><P>BD Difco™ <I>Lactobacilli</I> MRS Broth</P><P>We used this broth for both our liquid and solid culture media. The liquid medium was used to do the solid medium by adding agar to it.  We tested different concentrations of Agar to measure which one was the best.</P><DIV class="contener_plates"><DIV class="image_plate"><P>5g/L</P></DIV><DIV class="image_plate"><P>7g/L</P></DIV><DIV class="image_plate"><P>10g/L</P></DIV><DIV class="image_plate"><P>12g/L</P></DIV><DIV class="image_plate"><P>15g/L</P></DIV></DIV><FIGCAPTION><SPAN class="underline">Figure 1:</SPAN> The different concentrations of Agar used for solid MRS culture medium and <I>L. jensenii</I> grown on the plates for 24 hours.</FIGCAPTION><P>The plates with 5g/L and 7 g/L of Agar were not showing any colonies. For the three others there were more colonies for 12g/L. From this experiment we decided to use 12g/L of Agar for our experiments.</P><H3>Cell culture</H3><H4>Growth on solid medium</H4><P><I>L. jensenii</I> is a facultative anaerobic bacterium, it does not absolutely need CO<SUB>2</SUB> to survive but grows way better in anaerobic conditions. For our project we did not have a CO<SUB>2</SUB> incubator available for growing bacteria. We thus decided to grow our bacteria using an “old school” microbiology method. The goal of the method is to get rid of the Oxygen present in the environment of the bacteria. To do so, we put the plates in a hermetic container and placed a burning candle inside it before closing the lid. The candle consumes all the oxygen and turns it into in CO<SUB>2</SUB>, until all O<SUB>2</SUB> is consumed and the candle extinguish.</P><P>For our culture we used two kinds of plates and thus two kinds of containers.</P><P>As <I>L. jensenii</I> grows in small colonies we usually used small plates (60mm diameter) and put them in a jar (Figure 2) as follows:</P><FIGURE><FIGCAPTION><SPAN class="underline">Figure 2:</SPAN> Small plates (60mm diameter) are placed in an hermetically closed jar with a burning candle.  Using this method we were able to produce the anaerobic environment required for <I>L. Jensenii</I> growth.</FIGCAPTION></FIGURE><P>For transformant selection, we needed more colonies so we used normal culture plates (100mm diameter). We put them in an airtight-closing bowl with three candles (Figure 3).</P><FIGURE><FIGCAPTION><SPAN class="underline">Figure 3:</SPAN> 100mm plates incubated in a airtight-closing bowl with candles.</FIGCAPTION></FIGURE><H4>Growth in liquid medium</H4><FIGURE><FIGCAPTION><SPAN class="underline">Figure 4:</SPAN> Growth curve of <I>Lactobacillus jensenii</I> determined by OD<SUB>600</SUB> measurements.</FIGCAPTION></FIGURE><P>The stationary phase is reached at 8 hours of culture (Figure 4).
The culture takes generally 24 hours in liquid and solid media and can take up to 48 hours to see colonies on plates. It is much slower than <I>E. coli</I> and this can pose be a constraint for research on this strain.</P><H2>How to transform <I>L. jensenii</I>?</H2><P>To transform <I>L. jensenii</I>, we found several protocols. All of these protocols were specific to <I>Lactobacillus plantarum</I> or <I>Lactobacilli</I> in general, but not to <I>Lactobacillus jensenii</I>.
We selected different protocols that used or not magnesium in the electroporation buffer. Indeed, our electroporation machine could not stand magnesium, as it can causes electric arcs and damage the machine. We had to find another electroporator at the University of Montpellier. These electroporator had advanced settings such as the resistance (Figure 5).</P><FIGURE><FIGCAPTION><SPAN class="underline">Figure 5:</SPAN> Electroporater from the University of Montpellier we used.</FIGCAPTION></FIGURE><P>The three protocols that we chose were one from <A class="lien" href="https://openwetware.org/wiki/Lactobacillus_transformation_(Berthier_1996)" target="_blank">Berthier</A> in 1996, one from <A class="lien" href="" target="_blank">Speer</A> in 2012, and the last one from Chatel <A class="lien" href="#references">[1]</A>.</P><P>We tried to transform our bacteria several times with these three different protocols. Finally, we decided to focus on Berthier 1996 as we had better results. A reason why Speer 2012 did not work well was the addition of glycine to grow bacteria. It weaken the bacterial wall too much, and inhibited the grow. We did not focus on Chatel because the use of magnesium in the electroporation buffer gave us bad time constants, that means that bacteria were not well transformed.</P><P>Once we chose the most promising protocol, we had to optimize it. We tried different conditions for the following parameters:
<UL><LI><STRONG>Amount of DNA per reaction:</STRONG> 10ng, 100ng, 1000ng, 5000ng (of pLEM415) ⇒ did not change anything. We decided to add about 1000ng of plasmid per reaction.</LI><LI><STRONG>Voltage:</STRONG> 9kV/cm, 10kV/cm, 12.5kV/cm ⇒ we obtained better time constant (closer to 11ms to 13ms as explained in Berthier 1996) with 12.5kV/cm for the electroporation.</LI><LI><STRONG>Resistance (ohm Ω):</STRONG> 300Ω, 500Ω, 600Ω ⇒ to reach a time constant between 11ms and 13ms, we decided to follow the protocol and to use 600Ω.</LI><LI><STRONG>Antibiotic concentration for transformant selection:</STRONG> erythromycin 0.5µg/mL, 5µg/mL ⇒ we chose 0.5µg/mL as it was enough to select the transformants. 0.5µg/mL can kill non-resistant <I>L. jensenii</I>.</LI><LI><STRONG>Volume of the reaction:</STRONG> 50µL ⇒ we used 0.1cm cuvettes, so this volume was enough to electroporate correctly <I>L. jensenii</I>.</LI></UL></P><P>You can find the exact protocol for transforming <I>L. jensenii</I> on our <A href="https://2018.igem.org/Team:Montpellier/Protocols target=" _blank="">Protocol page</A>.</P><P>We designed a procedure to verify every problem that might happen during transformation. It is a table to fill with the number of colonies we have in plates after transforming or not and grow with or without antibiotics (Table 1). For these experiments, we inserted pLEM415 into <I>L. jensenii</I>. It has an erythromycin resistance gene.</P><TABLE><TBODY><TR><TH>No electroporation</TH><TH>Electroporation</TH></TR><TR><TD><STRONG>No antibiotics</STRONG></TD><TD>Number of bacteria in one reaction (50µL)</TD><TD>Electroporation conditions, survival rate of competent bacteria</TD></TR><TR><TD><STRONG>Antibiotics</STRONG></TD><TD>Negative control, check for contaminants</TD><TD>Transformation efficiency</TD></TR></TBODY></TABLE><FIGCAPTION><SPAN class="underline">Table 1:</SPAN> Analysis of the transformation results.</FIGCAPTION><UL><LI><STRONG>No electroporation/No antibiotics:</STRONG> We prepared competent cells that were not electroporated and plated on MRS Agar without antibiotics. We did serial dilutions to determine how many bacteria we had in a 50µL reaction.</LI><LI><STRONG>No electroporation/Antibiotics:</STRONG> We prepared this control where competent cells are plated on MRS Agar with 0.5µg/mL of erythromycin to check the eventual contamination in reactions. This is the negative control for the transformation, and it was done each time we transformed <I>L. jensenii</I>.</LI><LI><STRONG>Electroporation/No antibiotics:</STRONG> This condition was to set our electroporation conditions. Transformed competent cells are plated on MRS Agar without antibiotics. It gives us the survival rate of competent bacteria after electroporation as there is no selection. We determined a survival rate of 10% with our optimized protocol.</LI><LI><STRONG>Electroporation/Antibiotics:</STRONG> This condition is the final step. It permits us to determine the transformation efficiency. Transformed bacteria are plated on MRS Agar with 0.5µg/mL of erythromycin. Only the transformants that inserted the plasmid will survive.</LI></UL><P>We obtained positive results validated by the correspondent controls. We succeeded to transform <I>L. jensenii</I> at least once, but the experiment still lack a good reproducibility as we failed to achieve the transformation other times after this. The protocol still needs to be optimized to work each time for further use.</P><P>To go further, analysis such as colony PCR is required. Indeed, this should confirm if we succeed to transform <I>L. jensenii</I> with our plasmid or not.</P><H2><I>L. jensenii</I> plasmid</H2><P>The choice of the plasmid has been complicated because the strain is little used in synthetic biology. Thus, it has been difficult to find plasmids &quot;free&quot; of intellectual property. Indeed, the only existing plasmids (pOSELp23 and pOSEL144) were created by the company Osel Inc (USA). Unfortunately, we could not obtain the constructs from the company.</P><P>We therefore had to find a vector responding to the following specifications:
<OL><LI>An <I>E. coli</I> specific origin of replication.</LI><LI>An antibiotic  resistance gene for selection in <I>E. coli</I> ⇒ These parts are required to support cloning and propagation of our vectors into <I>E. coli</I> before transforming them into  <I>L. jensenii</I>.</LI><LI>A <I>L. jensenii</I> specific origin of replication.</LI><LI>An antibiotic  resistance gene for selection in <I>L. jensenii</I> ⇒ These parts will allow propagation and maintenance of the plasmids in <I>L. jensenii</I>.</LI></OL></P><P>We chose not to create a vector from scratch but to start by trying to reuse different already ones existing. Our choice turned toward specific vector with different origins of replications operating into <I>Lactobacillus</I> species available on the Addgene website. While none of them was previously used for <I>L. jensenii</I>, we hoped that at least  one of these vectors could be functional in <I>L. jensenii</I>:
<UL><LI><A class="lien" href="https://www.addgene.org/71312/" target="_blank">pTRKH2</A></LI><LI><A class="lien" href="https://www.addgene.org/27168/" target="_blank">pTRKH3-slpGFP</A></LI><LI><A class="lien" href="https://www.addgene.org/99842/" target="_blank">pLEM415-ldhL-mRFP1</A></LI><LI><A class="lien" href="https://www.addgene.org/71313/" target="_blank">pTRK669</A></LI><LI><A class="lien" href="https://www.addgene.org/71803/" target="_blank">pTRK892</A></LI></UL></P><P>We were able to transform pLEM415. If we had more time, we would have tested all the plasmids in <I>L. jensenii</I>. Some of them might be functional.</P><H3>Testing and choosing a vector</H3><P>pLEM415 is the most documented so it the easiest to use for our manipulations. This construct was used to transform <I>Lactobacillus casei</I>, <I>Lactobacillus delbrueckii</I> and <I>Lactobacillus acidophilus</I><A class="lien" href="#references">[1]</A>. Because of time constraints, we were not able to test the other plasmids.</P><P>Shuttle vector pLEM415 was obtained by ligating PstI-digested pLEM5 with PstI-linearized <I>Escherichia coli</I> vector pBII <A class="lien" href="#references">[2]</A>. pLEM5 contain the putative replication protein gene (<A class="lien" href="https://www.ncbi.nlm.nih.gov/protein/1230607" target="_blank">rep</A>). Moreover, the plasmid contain the minus origin of replication (ori-), the plus origin of replication (ori+) (CTaTCTTtATCTTGATACTTA) from <I>Lactobacillus fermentum</I>.</P><P>To compare, all of the OSEL vectors contain the repA and erm genes are from a naturally occurring <I>L. reuteri</I> plasmid pLEM7 <A class="lien" href="#references">[3]</A>. pLEM7 was generated by inserting of multiple cloning site into pLEM5 <A class="lien" href="#references">[2]</A>.</P><P>To conclude, OSEL vectors and pLEM415 have the same origin vector. While OSEL work with <I>L. jensenii</I>, we could suppose that our vector work as well in our strain.</P><P>In recent OSEL research, they create a modular shuttle vector pOSEL175, a modified version of pOSEL144 that can be integrated into the minor capsid gene of <I>L. jensenii</I> through homologous recombination <A class="lien" href="#references">[4]</A>.</P><H3>Antibiotic resistances</H3><P>We started by checking the natural resistances of <I>L. jensenii</I> to different antibiotics to confirm the relevance of using erythromycin and to know the other L. jensenii sensitivity to other antibiotics. We confirmed that L. jensenii is naturally resistant to Kanamycin, Neomycin, Spectinomycin, and Carbenicillin.</P><P>We tested the resistance to eight different antibiotics in a liquid MRS medium: Kanamycin, Carbenicillin (Carb), Spectinomycin (Spec), Chloramphenicol (CAM), Neomycin, Streptomycin, Erythromycin (ERM), Tetramycin.</P><P>We measured the OD<SUB>600</SUB> of <I>L. jensenii</I> cultures and diluted them to have cultures of 0.1 OD<SUB>600</SUB> in 5mL of MRS with the conditions presented below (Table 2). The cultures were observed the next day, after 24h of growth.</P><TABLE><TBODY><TR><TH>Concentration (µg/mL)</TH><TH>Kanamycin</TH><TH>Carb</TH><TH>Spec</TH><TH>CAM</TH><TH>Neomycin</TH><TH>Streptomycin</TH><TH>ERM</TH><TH>Tetramycin</TH></TR><TR><TD>25</TD><TD>+</TD><TD>+</TD><TD>+</TD><TD>-</TD><TD>-</TD><TD>+</TD><TD>+</TD><TD>-</TD></TR><TR><TD>50</TD><TD>+</TD><TD>+</TD><TD>+</TD><TD>-</TD><TD>-</TD><TD>+</TD><TD>+</TD><TD>-</TD></TR><TR><TD>75</TD><TD>+</TD><TD>+</TD><TD>+</TD><TD>-</TD><TD>-</TD><TD>+</TD><TD>+</TD><TD>-</TD></TR><TR><TD>100</TD><TD>+</TD><TD>+</TD><TD>+</TD><TD>-</TD><TD>-</TD><TD>+</TD><TD>+</TD><TD>-</TD></TR><TR><TD>125</TD><TD>+</TD><TD>+</TD><TD>+</TD><TD>-</TD><TD>-</TD><TD>+</TD><TD>+</TD><TD>-</TD></TR></TBODY></TABLE><FIGCAPTION><SPAN class="underline">Table 2:</SPAN> Antibiotic resistances of <I>L. jensenii</I>. +: colonies in plates. -: no colonies in plates.</FIGCAPTION><P>The vector that we use has an erythromycin resistance, so we prepared plates with MRS Agar and carbenicillin to control.</P><P><STRONG>Experiments</STRONG></P><FIGURE><FIGCAPTION><SPAN class="underline">Figure 6:</SPAN> Scheme of the experiment conducted for the antibiotic resistance test.</FIGCAPTION></FIGURE><P>We were able to demonstrate (Figure 6) that <I>Lactobacillus jensenii</I> is sensitive from 0.4mg/mL of erythromycin.</P><H3>Reporter</H3><P>To test the putative promoters and the different signal peptides, we used a reporter gene that encodes the protein mCherry (RFP). The mCherry red fluorescence protein excited at wavelengths 44 longer than 600 nm and is photostable.</P><P>We chose RFP as a reporter because it is more adapted in acid media and we saw it in several studies with intestinal <I>Lactobacilli</I><A class="lien" href="#references">[8]</A> but also in vaginal <I>Lactobacillus</I><A class="lien" href="#references">[4]</A>.</P><H3>Promoter studies</H3><P>We aimed at developing a library of differents promoters that work in <I>L. jensenii</I>.  We first retrieved known <I>L. jensenii</I> promoters from the literature. We also generated a bioinformatic tool using MEME to find pre-gene sequences in bacterial genomes. Wwe then chose the most relevant ones to design and clone the constructs for testing their strength. For more information see our modeling page. Moreover, as we wanted to use <I>B. subtilis</I> a surrogate Gram positive chassis, we designed a set of construct operating into this bacterium.</P><P>In total, we selected 3 Promoters from Osel papers <A class="lien" href="#references">[1]</A>, 10 <I>in silico</I> predicted  sequences, and 4 promoters for <I>B. subtilis</I><A class="lien" href="#references">[5]</A> and we synthesized these promoter sequences. We use the already described RpsU promoter used in our constructs that are supposed to be coding the proteins for our project.</P><P>These 17 promoter candidates were placed upstream RFP gene coupled with a strong RBS to quantify their strength by analysing the fluorescence intensity of the cells.</P><P>Several sequences were cloned into <I>E. coli</I>. Unfortunately their strength couldn’t be analyzed on time on <I>L. jensenii</I>. Indeed, putative promoter were designed for <I>Lactobacillus</I> and the transformation on this strain was difficult so we did not have time to characterize the strength of this promoter (Table 3). However, we tried to study the strength of the promoter in <I>E. coli</I> by flow cytometry (Figure 7).</P><H4>Putative promoters:</H4><UL><LI><STRONG>Primer NC</STRONG></LI><LI><STRONG>Primerprom3:</STRONG> aaaactagttctttttttatgcaaaaaattagcaaatagcttgcattagtgctaatttattagtatattattcatgttagcacctgagagttatgagtgctaattaagaggtcgagaat</LI><LI><STRONG>Primerprom43:</STRONG> tgttactgttaagtaatttagtctttctaaataaaaaggacaatctacgttatgtagattgtctttttttttgcttatttttgttataatcttactagaa</LI><LI><STRONG>Primerprom43-1SUV:</STRONG> agacacataagattaacttttttaggcataaaactagcaattaagaaaggttttgagttatattaataattattacaataaaattaaaaggggatttaat</LI><LI><STRONG>Primerprom188SUV:</STRONG> agacacataagattaacttttttaggcataaaactagcaattaagaaaggttttgagttatattaataattattacaataaaattaaaaggggatttaat</LI><LI><STRONG>Primerprom241-1SUV:</STRONG> aagttttaaaacaagcggatttagtggttattgatactagcgagttagacctagcaaaaatcattgcaatatttaatagtaagaaagtaggttaatgcaa</LI><LI><STRONG>Primerprom259-1JV:</STRONG> aatgtagaagccaaacttgaggaacttgataaacagcgggatgcagggttattaccacttaaaaagcaaaataaatttgtaaaaagtaaggagaaaaata</LI><LI><STRONG>Primerprom267-1SUV:</STRONG> tgccatatatgattccacagcgttttgtatatcgtgaagatatgccaatttcacaaaatggaaaagttgacattaaggctgtaattaaggaggtaaacgc</LI><LI><STRONG>Primerprom378-1JV:</STRONG> tttaagttcaaccattttaagcgaagcaggtaatgctcttgaagaattcaaggtggctgcacctgaagatgatgatgatgaaatataaggaggaattttt</LI><LI><STRONG>Primerprom403JV:</STRONG> aaagaaagcttaatagttttacaacttcctttaaccttttacttaaaataggaattaaactggttcgatgatataattttaagaaaaggaggagccgaat</LI><LI><STRONG>Primerprom500JV:</STRONG> taatacgcgtggtgctagttaaatcgttttagttaataaaaaagtccaatcttgttagactcgacttgtaaatagtaataatagaaaagagtctgatctg</LI></UL><H4><I>B. subtilis</I> promoters</H4><UL><LI><STRONG>PrimerpromBS-Ppgi-R0:</STRONG> cctttcttcttgacttgatttcacagataagttcatataaagtgaaagatgattaactaataaggaggacaaac</LI><LI><STRONG>PrimerpromBS-PrelA-R0:</STRONG> acttctgctctttacatctttcgtttttttcttgataataaactacaatagattaactaataaggaggacaaac</LI><LI><STRONG>PrimerpromBS-Pveg-R0:</STRONG> aattttgtcaaaataattttattgacaacgtcttattaacgttgataccggttaaattttatttgacaaaaatgggctcgtgttgtacaataaatgtggtggaaaggaggtgatcgac</LI><LI><STRONG>PrimerpromBS-PymdA-R0:</STRONG> gttaagatggcaagcttgacaagtatttccgacacatttacaatgaagttgattaactaataaggaggacaaac</LI></UL><H4><I>L. jensenii</I> promoters</H4><UL><LI><STRONG>PrimerpromPWM:</STRONG> gtgctaagctcttgatttgccaaagtatttttctctagttaaaatagcaggggttcattatagaaactttattatcagaatttatagaagctggtgaaaaaatggattacgagagtatcttaaatcaattagttgatggggaattaacagaatataagatagaagctaaggatgcttttgcttttcaacaggctctacgagcttttggaaaaaggacttatatcaagggacgtgcccttagaggtggtgcaattatctacactgcttcaaatagtaatgaataaggttgaaaacatttacattattgttgtattaaacttataaacatgttactcttaatatcgtacgggtattttaa</LI><LI><STRONG>PrimerpromRpsU:</STRONG> ggtaccttctaaaagaactattgacgtataaggacctttttggctataatttaactatattgttatggcagtactgccaagaaatg</LI><LI><STRONG>PrimerpromptsH:</STRONG> tgcaaatctgttgatactgcatcttgaagctcgtcgggacttttttcgtaaaaaaaagatggtctcataaggaggaattgatg</LI></UL><TABLE><TBODY><TR><TH>Part</TH><TH>Designed</TH><TH>Cloned</TH><TH>Characterized</TH></TR><TR><TD>Primer NC</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>Primerprom3</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>Primerprom43</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>Primerprom43-1SUV</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>Primerprom188SUV</TD><TD>YES</TD><TD>YES</TD><TD>YES</TD></TR><TR><TD>Primerprom241-1SUV</TD><TD>YES</TD><TD>YES</TD><TD>YES</TD></TR><TR><TD>Primerprom259-1JV</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>Primerprom267-1SUV</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>Primerprom378-1JV</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>Primerprom403JV</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>Primerprom500JV</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>PrimerpromBS-Ppgi-R0</TD><TD>YES</TD><TD>YES</TD><TD>YES</TD></TR><TR><TD>PrimerpromBS-PrelA-R0</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>PrimerpromBS-Pveg-R0</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>PrimerpromBS-PymdA-R0</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>PrimerpromPWM</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>PrimerpromRpsU</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR><TR><TD>PrimerpromptsH</TD><TD>YES</TD><TD>YES</TD><TD>NO</TD></TR></TBODY></TABLE><FIGCAPTION><SPAN class="underline">Table 3:</SPAN> Advances state for each sequence.</FIGCAPTION><P>We were able to characterize PrimerpromBS-Ppgi-R0 in <I>E. coli</I> by flow cytometry (Figure 7). The cells produced RFP with the promoter which is a witness for gene expression.</P><FIGURE><FIGCAPTION><SPAN class="underline">Figure 7:</SPAN> Flow cytometry analysis of R0 RFP fluorescence. A: In grey: negative control (NC). In pink: R0. B: relative fluorescence produced by R0.</FIGCAPTION></FIGURE><H3>Secretion tags</H3><P>We quickly encountered a problem in the project. We need to express spermicidal molecules in the supernatant so that they can act directly on the spermatozoa.</P><P>In the case of peptides, we have not added an excretion system because proteins already undergo many post-translational rearrangements. We did not want to burden the process with the addition of a secretion system.</P><P>In the case of antibodies, we looked for different secretory peptides associated with gram + bacteria:
<UL><LI>The first that we chose is CbsA. This signal peptide (SP) is used for secretion of antibodies in <I>L. jensenii</I><A class="lien" href="#references">[4]</A><A class="lien" href="#references">[6]</A>. The SP is synthesized on the N-terminus domain. Downstream the CbsA sequence are added 4 amino acids (APVT) at the N-terminus domain of the scFv. These 4 amino acids are similar to a native signal peptidase cleavage site of this protein. The addition of the APVT sequence shown to improve the secretion of a full-length protein in <I>L. jensenii</I>.</LI><LI>YncM, Epr, YjfA: These SPs are coming from a screening of SPs of <I>Bacillus subtilis</I>. As they can be used to secrete proteins in <I>B. subtilis</I> which is a gram positive bacteria <A class="lien" href="#references">[7]</A>, we decided to try to use them to secrete our stuff in <I>L. jensenii</I> that is also a gram positive bacteria. We used the same template for the construction. These SPs are on the N-terminus domain, and followed by the APVT sequence on their N-terminus domain.</LI></UL></P><P>Furthermore, we merged the SP with RFP to study secretion.</P><SECTION class="references" id="references"><TABLE class="references_table"><TBODY><TR><TH class="references_title" colspan="2">References</TH></TR><TR><TD class="references_left">[1]</TD><TD class="references_right">Bao, S., Zhu, L., Zhuang, Q., Wang, L., Xu, P. X., Itoh, K., ... &amp; Lin, J. (2013). Distribution dynamics of recombinant <I>Lactobacillus</I> in the gastrointestinal tract of neonatal rats. <I>PloS one</I>, 8(3), e60007.</TD></TR><TR><TD class="references_left">[2]</TD><TD class="references_right">Fons, M., Hégé, T., Ladiré, M., Raibaud, P., Ducluzeau, R., &amp; Maguin, E. (1997). Isolation and Characterization of a Plasmid fromLactobacillus fermentumConferring Erythromycin Resistance. <I>Plasmid</I>, 37(3), 199-203.</TD></TR><TR><TD class="references_left">[3]</TD><TD class="references_right">Chang, T. L. Y., Chang, C. H., Simpson, D. A., Xu, Q., Martin, P. K., Lagenaur, L. A., ... &amp; Lewicki, J. A. (2003). Inhibition of HIV infectivity by a natural human isolate of <I>Lactobacillus jensenii</I> engineered to express functional two-domain CD4. <I>Proceedings of the National Academy of Sciences</I>, 100(20), 11672-11677.</TD></TR><TR><TD class="references_left">[4]</TD><TD class="references_right">Marcobal, A., Liu, X., Zhang, W., Dimitrov, A. S., Jia, L., Lee, P. P., ... &amp; Lagenaur, L. A. (2016). Expression of human immunodeficiency virus type 1 neutralizing antibody fragments using human vaginal <I>Lactobacillus</I>. <I>AIDS research and human retroviruses</I>, 32(10-11), 964-971.</TD></TR><TR><TD class="references_left">[5]</TD><TD class="references_right">Guiziou, S., Sauveplane, V., Chang, H. J., Clerté, C., Declerck, N., Jules, M., &amp; Bonnet, J. (2016). A part toolbox to tune genetic expression in <I>Bacillus subtilis</I>. <I>Nucleic acids research</I>, 44(15), 7495-7508.</TD></TR><TR><TD class="references_left">[6]</TD><TD class="references_right">Xiaowen Liu, Laurel A. Lagenaur, David A. Simpson, Kirsten P. Essenmacher, Courtney L. Frazier-Parker, Yang Liu, Daniel Tsai, Srinivas S. Rao, Dean H. Hamer, Thomas P. Parks, Peter P. Lee and Qiang Xu. (2006). Engineered Vaginal <I>Lactobacillus</I> Strain for Mucosal Delivery of the Human Immunodeficiency Virus Inhibitor Cyanovirin-N. <I>Antimicrob Agents Chemother</I> Vol. 50, No. 10, p. 3250–3259</TD></TR><TR><TD class="references_left">[7]</TD><TD class="references_right">Brockmeier, U., Caspers, M., Freudl, R., Jockwer, A., Noll, T., &amp; Eggert, T. (2006). Systematic screening of all signal peptides from <I>Bacillus subtilis</I>: a powerful strategy in optimizing heterologous protein secretion in Gram-positive bacteria. <I>Journal of molecular biology</I>, 362(3), 393-402.</TD></TR><TR><TD class="references_bottom_left">[8]</TD><TD class="references_bottom_right">van Zyl, W. F., Deane, S. M., &amp; Dicks, L. M. (2015). Use of the mCherry fluorescent protein to study intestinal colonization by Enterococcus mundtii ST4SA and Lactobacillus plantarum 423 in mice. <I>Applied and environmental microbiology, 81</I>(17), 5993-6002.</TD></TR></TBODY></TABLE></SECTION><DIV class="footer_sponsors"><DIV class="contact_us"><H3> Sponsors </H3></DIV><DIV class="follow_us"><H3> Follow us on </H3></DIV><DIV class="contact_us"><H3> Contact us </H3><DIV class="contact_us_icons"><DIV class="deux_colonnes"><DIV class="colonne_2_70"><DIV><P class="p_footer"> igem.assomtp@gmail.com </P></DIV><DIV><P class="p_footer"> Centre de Biochimie Structurale, 29 Rue de Navacelles, 34090 Montpellier, France </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>