<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Montpellier_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Montpellier/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menus"><NAV><A href="https://2018.igem.org/Team:Montpellier">Home</A></NAV><NAV><A>Project</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Description">Description</A><A href="https://2018.igem.org/Team:Montpellier/Antisperm/antibodies">Antisperm antibodies</A><A href="https://2018.igem.org/Team:Montpellier/Peptides">Peptides</A><A href="https://2018.igem.org/Team:Montpellier/L_jensenii"><I>Lactobacillus jensenii</I></A><A href="https://2018.igem.org/Team:Montpellier/Results">Conclusions</A><A href="https://2018.igem.org/Team:Montpellier/Perspectives">Perspectives</A></DIV></NAV><NAV><A>DryLab</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Model">Modeling</A><A href="https://2018.igem.org/Team:Montpellier/Tracking/Software">Tracking Software</A></DIV></NAV><NAV><A>WetLab</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Protocols">Protocols</A><A href="https://2018.igem.org/Team:Montpellier/Toolbox">Toolbox</A><A href="https://2018.igem.org/Team:Montpellier/WetLab_ASA">Antisperm Antibodies</A><A href="https://2018.igem.org/Team:Montpellier/WetLab_Peptides">Peptides</A><A href="https://2018.igem.org/Team:Montpellier/Demonstrate">Sperm Motility Assay</A><A href="https://2018.igem.org/Team:Montpellier/InterLab">InterLab</A></DIV></NAV><NAV><A>Human Practices</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Human_Practices">Integrated human practices</A><A href="https://2018.igem.org/Team:Montpellier/Public_Engagement"> Education &amp; Public Engagement</A><A href="https://2018.igem.org/Team:Montpellier/Ethics">Ethics</A></DIV></NAV><NAV><A>Parts</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Parts">Parts</A></DIV></NAV><NAV><A>Team</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Team">Members</A><A href="https://2018.igem.org/Team:Montpellier/Attributions">Attributions</A><A href="https://2018.igem.org/Team:Montpellier/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Montpellier/Sponsors">Sponsors</A></DIV></NAV><NAV><A>Achievements</A><DIV><A href="https://2018.igem.org/Team:Montpellier/Award">Award</A><A href="https://2018.igem.org/Team:Montpellier/Medals">Medals</A></DIV></NAV></DIV><P><TITLE>Protocols</TITLE></P><P class="protocols_title"><I>Lactobacillus jensenii</I> protocols</P><BUTTON class="accordion"><I>L. jensenii</I> : electrocompetent cells and transformation</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN> for 10 tubes of 50µL of competent cells</P><UL><LI>Glycerol stock or fresh culture of <I>L. jensenii</I></LI><LI>MRS agar plates with appropriate antibiotic</LI><LI>100mL of liquid MRS</LI><LI>25mL of liquid MRS 80mM MgCl<SUB>2</SUB> (for recovery)</LI><LI>25mL of <SPAN class="red bold">cold</SPAN> wash solution (10mM MgCl<SUB>2</SUB>)</LI><LI>25mL of <SPAN class="red bold">cold</SPAN> electroporation buffer (0.5M Sucrose, 10% glycerol)</LI><LI><SPAN class="red bold">Cold</SPAN> 1.5mL Eppendorf tubes</LI><LI><SPAN class="red bold">Cold</SPAN> electroporation cuvettes</LI><LI>Culture tubes for recovery</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN> preparation of competent cells</P><OL><LI>Grow cells overnight (ON) in MRS at 37°C</LI><LI>Use ON culture to inoculate 100mL MRS</LI><LI>Incubate at 37°C until OD<SUB>600</SUB> = 0.4 to 0.6 (about 3 hours)</LI><LI>Split the 100mL culture in two 50mL falcon tubes</LI><LI>Centrifuge for 5min at 3000g at 4°C</LI><LI><SPAN class="red bold">Starting from here, do everything on ice</SPAN></LI><LI>Wash 2x with <SPAN class="red bold">cold</SPAN> Wash Solution (WS) :</LI><UL><LI>resuspend the pellet in each tube with 5mL of WS</LI><LI>centrifuge for 5min at 3000g at 4°C</LI><LI>repeat</LI></UL><LI>Wash 1x with <SPAN class="red bold">cold</SPAN> Electroporation Buffer (EB) : resuspend the pellet in each tube with 5mL of EB</LI><LI>Centrifuge for 5min at 3000g at 4°C</LI><LI>Resuspend each pellet in 5mL of EB</LI><LI>Concentrate in EB :</LI><UL><LI>centrifuge for 5min at 3000g at 4°C</LI><LI>resuspend each pellet in 250µL and transfer these to 1.5mL Eppendorf tubes</LI><LI>make 50µL aliquots</LI></UL><LI>Add about 1µg of plasmid DNA</LI><LI>Keep aliquots up to 2 hours</LI></OL><P><SPAN class="protocols_subpart">Protocol :</SPAN> electroporation of competent cells</P><OL><LI>Fill the cuvettes with the samples without making bubbles</LI><LI>Electroporate at 12.5kV/cm, 25µF, 600𝛀 (time constants for this protocol are between 11ms and 13ms)</LI><LI>Recover cells by adding 1mL of 80mM MgCl<SUB>2</SUB> MRS</LI><LI>Incubate <SPAN class="red bold">ON</SPAN> at 37°C</LI><LI>Plate to select</LI></OL></DIV><P class="protocols_title"><I>Bacillus subtilis</I> protocols</P><BUTTON class="accordion"><I>B. subtilis</I> : chemocompetent cells</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN> for 22 tubes of <I>B. subtilis</I> competent cells</P><UL><LI>Glycerol stock of <I>B. subtilis</I></LI><LI>1x LB or LB + ATB plate</LI><LI>2x 50 or 100mL shakeflask</LI><LI>Medium A</LI><LI>Medium B</LI><LI>1.5mL tubes</LI><LI>1mL, 10mL and 25mL pipettes</LI><LI>Incubator equipped with shaker and set at 37°C</LI><LI>Spectrophotometer cuvettes</LI><LI>Spectrophotometer for OD<SUB>650</SUB>nm determinations</LI></UL><P><SPAN class="protocols_subpart">Culture media to prepare :</SPAN></P><UL><LI>10X Medium A base</LI><UL><LI>Yeast extract : 10g</LI><LI>Casamino acids : 2g</LI><LI>Distilled water : fill to 900mL</LI><LI>Autoclave, then add :</LI><LI>50% glucose, filter-sterilized : 100mL</LI><LI>Aliquot by 10mL</LI><LI>Stock at RT</LI></UL><LI>10X <I>Bacillus subtilis</I> salts</LI><UL><LI>(NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB> 20g</LI><LI>K<SUB>2</SUB>HPO<SUB>4</SUB> 3H<SUB>2</SUB>O 183g</LI><LI>KH<SUB>2</SUB>PO<SUB>4</SUB> 60g</LI><LI>Na<SUP>+</SUP> citrate 10g</LI><LI>MgSO<SUB>4</SUB> 7H<SUB>2</SUB>O 2g</LI><LI>Distilled water : fill to 1000mL</LI><LI>Aliquot by 9mL</LI><LI>Stock at RT</LI></UL><LI>50mM CaCl<SUB>2</SUB> 2H<SUB>2</SUB>O</LI><UL><LI>Filtered 0.2µL</LI><LI>Stock at RT in 50mL tubes</LI></UL><LI>250mM MgCl<SUB>2</SUB> 6H<SUB>2</SUB>O</LI><UL><LI>Filtered 0.2µL</LI><LI>Stock at RT in 50mL tubes</LI></UL><LI>Medium A (200mL)</LI><UL><LI>Sterile H<SUB>2</SUB>O 162mL</LI><LI>10X Medium A base 20mL</LI><LI>Bacillus salts 18mL</LI><LI>Mix</LI><LI>Filter at 0.2µm (filtration unit)</LI><LI>Stock at RT</LI></UL><LI>Medium B (100mL)</LI><UL><LI>Medium A 100mL</LI><LI>50mM CaCl<SUB>2</SUB> 2H<SUB>2</SUB>O</LI><LI>250mM MgCl<SUB>2</SUB> 6H<SUB>2</SUB>O</LI><LI>Mix</LI><LI>Filter at 0.2µm (filtration unit)</LI><LI> Stock at RT</LI></UL></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><UL><P><LI>Day 1 :</LI></P><OL><LI>Streak the strain on a petri dish from the glycerol stock on a LB Agar plate or LB + antibiotic if the train contains a resistance gene</LI><LI>Incubate overnight (18h) at 37°C</LI></OL><P><LI>Day 2 :</LI></P><OL><LI>In a flask, inoculate 20mL of medium A with few colonies, until 0.2 &lt; OD<SUB>65</SUB>nm &lt; 0.25 (approximately one line of colony, resuspend well before to measure the OD)</LI><LI>Incubate at 37°C with vigorous aeration and read the OD every 30 minutes for monitor</LI><LI>When the OD not increase anymore, incubate the culture 90 minutes more at 37°C</LI><LI>Dilute 10 times of this culture into some Medium B in a new flask (1mL of culture in 9mL of medium B)</LI><LI>Incubate at 37°C with vigorous aeration for 90minutes</LI><LI>Add 50% glycerol to the culture in a 1:10<SUP>th</SUP> ration (1mL of glycerol in 10mL of culture)</LI><LI>Aliquot per 500µL in tubes of 1.5mL</LI><LI>Flash freeze tubes in liquid nitrogen</LI><LI>Stock tubes at -80°C</LI></OL></UL></DIV><BUTTON class="accordion"><I>B. subtilis</I> : transformation of chemocompetent cells</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>Competent cells</LI><LI>Plasmid</LI><LI>Water bath at 37°C</LI><LI>Incubator at 37°C with shaking</LI><LI>Centrifuge</LI><LI>LB medium</LI><LI>LB + ATB plate</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Defrost 2 tubes of competent <I>B. subtilis</I> cells (500µL/tube) in the water bath at 37°C</LI><LI>Add 500ng of plasmid in one tube (mix by inverting 2 or 3 times). Nothing to add in the other tube (there is the negative control)</LI><LI>Place the tubes <SPAN class="red">30 minutes</SPAN> in the incubator at 37°C with shaking</LI><LI>Add 200µL of LB per tube and incubate again for 30 min or more at 37°C with shaking</LI><LI>Centrifuge 2min at 4000 rpm at room temperarture</LI><LI>Remove 600μL of supernatant</LI><LI>Resuspend the pellet and plate on LB + ATB (for plating, we use glass beads, alternatively you can use a pipette rake)</LI><LI>Incubate inverted plates ON at 37°C</LI></OL></DIV><BUTTON class="accordion">Colony PCR for <I>B. subtilis</I></BUTTON><DIV class="panel"><OL><LI>Dilute each colony in 100µL sterile ddH2O</LI><LI>Prepare the PCR Mix :<TABLE><TBODY><TR><TH>1 reaction</TH><TH>5 reactions</TH></TR><TR><TD>Buffer B</TD><TD>5µL</TD><TD>25µL</TD></TR><TR><TD>Enhancer</TD><TD>5µL</TD><TD>25µL</TD></TR><TR><TD>Primer 1 (Sp 27 rev)</TD><TD>0.75µL</TD><TD>3.75µL</TD></TR><TR><TD>Primer 2 (P280)</TD><TD>0.75µL</TD><TD>3.75µL</TD></TR><TR><TD>dNTP</TD><TD>0.5µL</TD><TD>2.5µL</TD></TR><TR><TD>Kapa Polymerase</TD><TD>0.1µL</TD><TD>0.5µL</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>12µL</TD><TD>60µL</TD></TR></TBODY></TABLE></LI><LI>In PCR tube : 1µL resuspended colony + 25µL PCR Mix</LI><LI>PCR Cycle : Kapa<TABLE><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Initial denaturation</TD><TD>95°C</TD><TD>5min</TD></TR><TR><TD>95°C</TD><TD>30s</TD></TR><TR><TD>30 Cycles</TD><TD>63°Ctemperature depend of primers</TD><TD>30s</TD></TR><TR><TD>72°C</TD><TD>1min/2kbtime according to the fragment size</TD></TR><TR><TD>72°C</TD><TD>1min</TD></TR><TR><TD>12°C</TD><TD>∞</TD></TR></TBODY></TABLE></LI><LI>Verify by gel the colony PCR : 0.8% agarose in TBE + 3µL Sybr Safe DNA2.55µL PCR product + blue and 2µL ladder 1kb, 100V 30min (for a single gel)</LI></OL></DIV><P class="protocols_title"><I>Escherichia coli</I> protocols</P><BUTTON class="accordion"><I>E. coli</I> : electrocompetent cells and transformation</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Steps :</SPAN></P><OL><LI>Streak the E.coli from glycerol stock.</LI><LI>Pre-culture.</LI><LI>Culture and preparation of electrocompetent cells.</LI><LI>Transformation test.</LI><LI>Calculate the efficiency.</LI></OL><P><SPAN class="protocols_subpart">Materials :</SPAN> to prepare 2L of culture</P><UL><LI><I>E. coli</I> NEB10β (NEB, C3020K), TG1. Stored at -80°C</LI><LI>pUC19 DNA (NEB N3041A): 50pg/µL in ddH2O. Stored at -20°C</LI><LI>Ampicillin (Sigma): 100 mg/mL stock solution in ddH2O, filter-sterilized. Stored at -20°C</LI><LI>Cam, Kana, Spec</LI><LI>SOC medium</LI><LI>2L LB medium</LI><LI>3L Pure H2O, autoclaved</LI><LI>2L of 10% (v/v) Glycerol in H2O, autoclaved (Measure 100mL of glycerol 99% in a measuring tube, add it in a 1L glass bottle, rinse the measuring tube with miliQ water, add the water in the same bottle, adjust the volume in the bottle to 1L with miliQ water)</LI><LI>2 x 250mL Measuring tubes autoclaved</LI><LI>1 x 500mL Measuring tubes autoclaved</LI><LI>4 x 500 ml centrifuge bottles (NALGENE)</LI><LI>PCR tubes or 1.5mL tubes</LI><LI>1mL syringes for Eppendorf pipette</LI><LI>10mL, 5mL and 2mL pipettes</LI><LI>1 x 15mL tube</LI><LI>High speed centrifuges (Sorvall)</LI><LI>Shaking incubator at 37°C</LI><LI>Electroporator (Bio-rad)</LI><LI>Electroporation cuvettes (Bio-rad, 0.1 or 0.2 cm)</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><UL><P><LI>Day 1 :</LI></P><OL><LI>Streak the E.coli from glycerol stock (-80°C) onto a LB plate without antibiotic. Incubate the plate at 37°C overnight. The plate can be storage at 4°C in one week.</LI></OL><P><LI>Day 2 :</LI></P><OL><LI>Inoculate a single E.coli colony in 30 ml of LB medium. Vortex. Grow overnight at 250 rpm and 37°C. The amount of overnight culture depends on how many liters of competent cell you are going to prepare. For 1L culture you need 4mL of overnight culture.</LI><LI>Do a <SPAN class="red">negative control of contamination</SPAN> with 4 antibiotics (Kam, Carb, Kana, Spec) : 2mL of preculture + 2μL Antibiotic (4 different tubes). Vortex. Grow overnight at 250 rpm and 37°C.</LI><LI>Prepare 2YT medium or LB medium (500mL in 2L flask). Keep them at RT°C.</LI><LI>Prepare the autoclaved 3L pure H2O, 2L 10% glycerol, 3 Measuring tubes, 4 centrifuge bottles, PCR tubes or autoclaved 1.5mL tubes, PCR racks, 10mL 5mL and 2mL pipettes, few 1mL syringes for Eppendorf pipette and 1 x 15mL tube. Afterward, keep them at 4°C.</LI><LI>Book the 37°C incubator for tomorrow (4 x 2L from 9 to 13 for example).</LI><LI>Book the centrifuge for tomorrow (Rotor F9 / 2500g / 4°C / during 3h).</LI></OL><P><LI>Day 3 :</LI></P><P><SPAN class="protocols_subpart">Cells preparation :</SPAN></P><OL><LI>Check the negatives controls.</LI><LI>Transfer 4 mL of overnight culture into 2-liter flask containing 1L of LB medium (1:250) without antibiotic. Homogenize.</LI><LI>Do a <SPAN class="red">negative control of contamination</SPAN> with 4 antibiotics (Kam, Carb, Kana, Spec): 2mL of culture + 2μL Antibiotic (4 different tubes). Vortex. Grow overnight at 250 rpm and 37°C.</LI><LI>For the culture in 4 x 2-liter flask (500mL/flask) with a sterilized 500mL measuring tube.</LI><LI>Incubate at 250 rpm and 37°C until the OD<SUB>600</SUB> is about 0.5 ~ 0.6 (~4h) (To measure the OD, do the blanc with 1mL LB medium).
		  <SPAN class="red">Note : For NEB10β, it takes about 4 hours to reach OD 0.52. You should check the OD of your culture after 3 hours and 30 minutes incubation time.</SPAN></LI><LI>After OD is reached, pour the culture into 4 pre-chilled 500 ml centrifuge bottles.</LI><LI>Balance the weight of the bottles with the culture and chill it on ice for 30 min.</LI><LI>Keep flasks as waste.
		    <SPAN class="red">Note : From here on, everything should be kept on ice, in the cold room and done as rapidly as possible.</SPAN></LI><LI>Cool down the centrifuge at 4°C.</LI><LI>Spin at 2500g for 20 min at 4°C.</LI><LI>Decant the sup in a “Flask –waste”. <SPAN class="red">Be careful, always dry the pellet the most of possible and don’t touch the flask with the bottle.</SPAN></LI><LI>Use a sterilized 250mL measuring tube. Re-suspend cell pellet <SPAN class="red">by hand and quickly, don’t use vortex</SPAN> in 250mL/bottle of pre-chilled pure H2O. You can pool it in 2 bottles (2 x 500mL) equilibrate and then spin at 2300g for 10 min at 4°C.</LI><LI>Decant the sup in a “Flask – waste”.</LI><LI>Use the same sterilized 250mL measuring tube and re-suspend cell pellet as step 12, in 250mL/bottle of pre-chilled pure H2O, and then spin at 2300g for 10 min at 4°C.</LI><LI>Decant the sup in a “Flask – waste”.</LI><LI>Use a sterilized 250mL measuring tube and re-suspend cell pellet as step 12, in 250 mL/bottle of pre-chilled 10% glycerol, and then spin at 2300g for 10 min at 4°C.
		      <SPAN class="red">Note : Take care when decanting because the cells lose adherence in 10% glycerol. Perform this right after centrifugation can reduce the lost.</SPAN></LI><LI>Repeat step 16 for two times with 100mL/bottle of pre-chilled 10% glycerol.</LI><LI>18.Carefully decant the sup and re-suspend cells in the remaining volume (about 1 ~ 2 mL) with <I>E.coli</I> electrocompetent cells preparation and transformation with 2mL pipette. (If it’s not possible, add 500μL glycerol 10% and re-suspend the pellet.) Transfer it in the 15mL cold tube.
		        <SPAN class="red">Note : This is best done by gently swirling rather than vortex. Or use plastic drop for gently flushing.</SPAN></LI><LI>Measure the OD<SUB>600</SUB> of a 0, 5:100 dilution of the cell suspension. 5μL culture + 995μL H20, mix, mesure. The spectrophotometer can measure only until OD<SUB>600</SUB> = 1.0 (If the measure is too high, dilute it of a 1:2 dilution).
		          <SPAN class="red">1.0 OD<SUB>600</SUB> = ~2.5 x 10<SUP>8</SUP> cells/mL. For High efficiency we need OD<SUB>600</SUB> = 150–200.</SPAN></LI><LI>Aliquot immediately by 80μL into 1.5mL tubes, drop into a bath of liquid nitrogen, and transfer to -80°C for storage.
		            <SPAN class="red">For Gibson assembly we need OD<SUB>600</SUB> = 80.</SPAN></LI><LI>Do a <SPAN class="red">negative control of contamination</SPAN> with 4 antibiotics (Kam, Carb, Kana, Spec): Add 8mL LB medium in the 15mL Tubes with few μL of re-suspended pellet, Pour in 4 tubes (2mL/tube) + 2μL Antibiotic / tube. Vortex. Grow overnight at 250 rpm and 37°C.</LI></OL><P><LI>Day 3 or 4 :</LI></P><P><SPAN class="protocols_subpart">Cells transformation by electroporation and efficiency test :</SPAN></P><OL><LI>Pre-warm the SOC medium at 37°C. Pre-warm the plate contain selection antibiotics in 37°C for 1 hour.</LI><LI>Pre-chilled cuvettes (Bio-rad, 0.1 or 0.2 cm) on ice for 5 minutes before adding the cell/DNA mixture.</LI><LI>Thaw the electrocompetent cells on ice. Add a) 100 pg of pUC19 control plasmid (2μL at 50pg/L) for efficiency test (it’s possible to do it with 1μL just take it in consideration when you do the efficiency calculation), or b) 5 μL of your DNA sample for experiments, or d) 1μL of Gibson Assembly product in 40 μl of the thawed competent cells, mix by gently pipetting, and transfer to the pre-chilled cuvette. Stand for 1 min.</LI><LI>Perform electroporation by Bio-Rad GenePluser, Ec1 for 0.1 cm cuvette (brown) and Ec2 for 0.2 cm cuvette(green). (EC1, Bacteria = 1.8Kv and 6ms for Blanc or without cell). Flush the cuvette <SPAN class="red">immediately</SPAN> with 1mL 37°C pre-warmed SOC medium and transfer to a 12mL culture tube. Recover for 1h at 37°C, 200 rpm shaking.</LI><LI>Wash cuvettes with ethanol and after water two times.</LI><LI><SPAN class="protocols_subpart">For efficiency test :</SPAN>
		              Perform serial dilution. Dilute 10μL of the culture in 990μL LB, then plate 100μL on LB Ampi plate (= 10<SUP>-3</SUP>).
		              Dilute 10μL of <SUP>-3</SUP> in 990μL LB, then plate 100μL on LB Ampi plate (= 10<SUP>-5</SUP>)
		              (pUC19 = Ampi resistant)<SPAN class="protocols_subpart">In experiment :</SPAN>
		              Plate 100μL on a plate. Centrifuge the culture, throw 800μL, and plate the pellet resuspend in the 100μL SN.</LI><LI>Incubate Over Night at 37°C.</LI></OL><P><LI>Day 4 :</LI></P><OL><LI>Calculate the competency in colonies.
		                Ex : On the 10<SUP>-3</SUP> plate we have 27 colonies = 2,7.10<SUP>4</SUP> UFC/mL (in our tube after electroporation)
		                2μL pUC19 = 100pg = 2,7.10<SUP>4</SUP> UFC
		                100ng = 2,7.10<SUP>7</SUP> UFC
		                Efficiency 2,7.10<SUP>8</SUP> UFC/μg pUC19</LI></OL><P><SPAN class="protocols_subpart">Trouble shooting :</SPAN></P><OL><P><LI>Low colony number of pUC19 transformed plate :</LI></P><OL><LI>Check the DNA quality and quantity. The OD 260:230 should be higher than 1.6 at least.</LI><LI>Check the preparation procedure of competent cells. Especially the cell concentration. The OD<SUB>600</SUB> of 100-fold dilution of competent cell should higher than 1 at least.</LI><LI>Check the cuvette. Rusted or additional water on its surface ?</LI><LI>Check the voltage and time constant. The typical time constant is 4.8 to 5.1. If the time constant is lower than 4.5 ms, check your DNA preparation procedure. You should elute your DNA by pre-warmed H2O.</LI><LI>Your sample volume should be less than 5λ in 50λ transformation reaction to prevent the dilution of competent cell concentration.</LI></OL></OL></UL></DIV><BUTTON class="accordion"><I>E. coli</I> : chemocompetent cells and transformation</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials : for chemocompetent cells</SPAN></P><UL><LI>LB medium : For NEB 10 Beta = LB without antibiotic, for DH5 Alpha Z1 = LB Spectinomycin 50, for NEB Turbo = LB without antibiotic</LI><LI>TSS medium : To make 50 mL : 5g PEG 8000, 1.5 mL 1M MgCl2 (or 0.30g MgCl2*6H20), 2.5 mL DMSO and LB to 50 mL. Filter sterilized (0.22 μm filter).</LI><LI>LB plates</LI><LI>1.5mL tubes</LI><LI>50mL falcons</LI><LI>Ice</LI><LI>500mL flask for culture</LI><LI>Liquid nitrogen</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><UL><P><LI>Day - 1 :</LI></P><OL><LI>Streak an LB plate (with ATB if needed) with the <I>E. coli</I> strain</LI></OL><P><LI>Day 0 :</LI></P><OL><LI>Inoculate 5mL of LB (with ATB if needed) with the strain from the fresh streak plate</LI><LI>Incubate overnight at 37°C</LI></OL><P><LI>Day 1 :</LI></P><OL><LI>Dilute the overnight culture into 50mL of LB without antibiotic at 1/500 (200μL in 100mL)</LI><LI>Incubated at 37°C until OD<SUB>650</SUB> reach 0.2-0.3 (3-4h)</LI><LI>Place 1.5mL tubes, racks, 10mL pipettes at -20°C</LI><LI><SPAN class="red">DO EVERYTHING ON ICE</SPAN></LI><LI>Incubate the culture on ice for 10min in 50mL falcon tubes</LI><LI>Cool down the centrifuge to 4°C</LI><LI>Centrifuge the culture at 3000rpm 4°C for 10min</LI><LI>Remove the supernatant</LI><LI>Resuspend cells in 10% volume of TSS buffer</LI><LI>Aliquot cells in 100uL in 1.5mL tubes with multdistrib pipette</LI><LI>Force them with liquid nitrogen</LI><LI>Store at -80°C</LI></OL><P>To test them : use pUC19 as positive control and do not forget negative control.</P><OL><LI>Thaw gently 3 tubes of cells on ice</LI><LI>Add 1μL pUC19 in 1 tube, add nothing in the others (negative control) (keep cells on ice)</LI><LI>Incubate 30min on ice</LI><LI>Heat-shock cells at 42°C during 45s (in water bath)</LI><LI>Put back on ice after heat-shock</LI><LI>Add 900 μL pre-warmed SOC (37°C) (rich medium)</LI><LI>Incubate cells at 37°C with agitation during at least 30min</LI><LI>Centrifuge cells at 4000rpm during 2min, remove 800μL of supernatant</LI><LI>Plate the rest from pUC19 positive control on LB Carb plate</LI><LI>Plate 100μL from negative control respectively on LB Chloramphenicol, LB Kanamycin, LB Carbenicillin and LB Spectinomycin plates</LI><LI>Incubate at 37°C overnight</LI></OL><P>Efficiency should be around 10<SUP>7</SUP> colonies/μg pUC19</P></UL><P><SPAN class="protocols_subpart">Materials : for transformation of chemocompetent cells</SPAN></P><UL><LI>Chemical competent cells</LI><LI>Ice</LI><LI>Clean DNA to transform</LI><LI>SOC</LI><LI>Selective agar plates</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Thaw gently cells on ice</LI><LI>Add DNA (keep cells on ice)</LI><LI>Incubate 30min on ice</LI><LI>Heat-shock cells at 42°C during 45s (in water bath)</LI><LI>Put back on ice after heat-shock</LI><LI>Add 900 μL pre-warmed SOC (37°C) (rich medium)</LI><LI>Incubate cells at 37°C with agitation during at least 30min</LI><LI>Plate 100 μL of transformation in selective agar plate or centrifuge cells at 4000rpm during 1min, remove 800μL of supernatant and plate the rest</LI><LI>Incubate at 37°C overnight</LI></OL></DIV><P class="protocols_title" id="bww"> Sperm analysis protocols</P><BUTTON class="accordion">General experimental procedure</BUTTON><DIV class="panel"><P>For each experience the spermatozoa were diluted to a 1/100 concentration. This dilution allows us to individually track spermatozoa in the video and avoid crowding whereas still having a significant number of spermatozoa. A total volume of 20µL of the sample is put on the slide. This volume was calculated to have a height of sample bigger that spermatozoa heads while reducing the proportion of cells in different focal planes</P><P>The microscope used was an <I>Olympus IX71</I> with magnificence 20x ; 1000µs of exposure time. The camera used for the recording was a <I>Phantom MIRO 320 vision research</I> with 768x430 of resolution and 60 FPS. For each sample, 5 videos of 10 seconds were recorded.</P></DIV><BUTTON class="accordion">BWW Buffer preparation</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN> for 100mL</P><UL><LI>NaCl 94.7mM</LI><LI>KCl 4.8mM</LI><LI>CaCl<SUB>2</SUB> 1.71mM</LI><LI>KH<SUB>2</SUB>PO<SUB>4</SUB> 1.2mM</LI><LI>MgSO<SUB>4</SUB> 7H<SUB>2</SUB>O 12mM</LI><LI>NaHCO<SUB>3</SUB> 25mM</LI><LI>Glucose 5.6mM</LI><LI>HEPES 20mM</LI><LI>Carbenicillin 1mL of 50mg/mL stock</LI><LI>Streptomycin 1mL of 100mg/mL stock</LI><LI>BSA 6g</LI><LI>ddH<SUB>2</SUB>O</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Use a 200mL bottle with 50mL of ddH<SUB>2</SUB>O</LI><LI>Weigh and dissolve all powders in the water</LI><LI>Add the antibiotics</LI><LI>Fill with ddH<SUB>2</SUB>O to 100mL</LI><LI>Store at 4°C</LI></OL></DIV><BUTTON class="accordion">Nisin elution preparation</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>Nisin 2.5% from Sigma</LI><LI>HCl 1M</LI><LI>NaCl powder</LI><LI>ddH<SUB>2</SUB>O</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Use 50mL tube with 10mL ddH<SUB>2</SUB>O</LI><LI>Weigh and add 187,5 mg of NaCl in the ddH<SUB>2</SUB>O</LI><LI>Add 0.5mL HCl 1M</LI><LI>Fill to 25mL with ddH<SUB>2</SUB>O</LI><LI>Aliquot to 2.5mL in 10mL tubes</LI><LI>Store at 4°C</LI><LI>When you need to use nisin, add 100mg of nisin powder to 2.5mL eluent to have 400µg/mL nisin</LI></OL></DIV><P class="protocols_title"> Cloning by Gibson Assembly</P><BUTTON class="accordion">Gibson Assembly cloning</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Steps :</SPAN></P><OL><LI>Design</LI><LI>PCR Q5</LI><LI>DpnI digestion</LI><LI>PCR Clean-up</LI><LI>Gibson Assembly</LI><LI>Transformation</LI><LI>Verifications</LI><LI>Storage</LI></OL><P><SPAN class="protocols_subpart">Materials and protocols :</SPAN></P><OL><SPAN class="bold"><LI>PCR Q5 :</LI></SPAN><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>Q5® Hot Start High-Fidelity DNA polymerase (NEB)</LI><LI>Primers 20μM</LI><LI>Template adjust to 1ng/μL</LI><LI>Ultrapure Water</LI><LI>PCR tubes</LI><LI>PCR machine</LI><LI>Agar gel material</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Depending on the final quantity of fragment needed and PCR yield, you can perform either a PCR with 20μL, 40μL or several times 40μL. Mix 8μL Water + 10μL Q5 + 1μL Template + 0.5μL each Primer.<TABLE><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Initial denaturation</TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD>98°C</TD><TD>10s</TD></TR><TR><TD>30 Cycles</TD><TD>50–72°Ctemperature depend of primers</TD><TD>20s</TD></TR><TR><TD>72°C</TD><TD>30s/kbtime according to the fragment size</TD></TR></TBODY></TABLE><P>Use of the NEB T<SUB>m</SUB> Calculator is highly recommended.</P></LI><LI>Prepare a 0.8% agar gel for sample larger than 1kb and 1.5% for sample smaller than 1kb</LI><LI>Load 2.5μL of the PCR reaction with 0.5μL of loading dye in the gel and a 1kb ladder or 100bp ladder according to the size of your expected fragment</LI><LI>Image the gel</LI></OL><SPAN class="bold"><LI>DpnI digestion :</LI></SPAN><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>DpnI (NEB)</LI><LI>PCR machine</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Add 1μL DpnI in 17.5μL tube from Q5</LI><LI>Mix well by pipetting up and down so that you solution is homogenous and glycerol is not in the bottom of the tube</LI><LI>1h at 37°C</LI><LI>10min at 80°C and hold at 12°C</LI></OL><SPAN class="bold"><LI>PCR Clean-up :</LI></SPAN><P><SPAN class="protocols_subpart">Material :</SPAN></P><UL><LI>Biosentec PCR Clean-up Kit</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>biosentec protocol</LI><LI>Measure the DNA concentration with Nanodrop</LI></OL><SPAN class="bold"><LI>Gibson Assembly :</LI></SPAN><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>2X Gibson Assembly Mix (see Gibson Assembly Mix protocol)</LI><LI>Insert fragment(s) from PCR or ordered</LI><LI>Vector fragment (after PCR, DpnI and clean-up)</LI><LI>Ultrapure water</LI><LI>Thermo-cycler at 50°C</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Calculate the volume of insert and vector to mix in the Gibson Assembly reaction. For 2 fragment assembly (one insert and one vector), the optimum is to mix 100ng of vector with 3 times more insert in mole, and for more than 2 fragment assembly, 100ng of vector with same quantity of each insert in mole. The total volume of insert(s) and vector have to be 10μL or less, if the calculated total volume is higher, the quantity of vector can be reduced up to 50ng and the volume of insert calculated accordingly.</LI><LI>Mix the vector and insert(s) fragments according to previously calculated proportions with 10μL of the Gibson Assembly Mix and adjust the total volume of the reaction to 20μL with water. As negative control of assembly, mix the vector alone in the same proportion than previously with 10μL of Gibson Assembly Mix and adjust the total volume of the reaction to 20μL with water.</LI><LI>Place the reactions at 50°C during one hour</LI></OL><P><SPAN class="red">Note : For a better conservation of your Gibson Assembly reaction at room temperature you can incubate your reaction 10min at 80°C.</SPAN></P><SPAN class="bold"><LI>Transformation <SPAN class="red">(not for <I>L. jensenii</I>)</SPAN> :</LI></SPAN><SPAN class="bold"><LI>Colony PCR :</LI></SPAN><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>2X One-taq quick load master mix (NEB)</LI><LI>Primers at 20μM</LI><LI>Colonies</LI><LI>PCR tubes</LI><LI>PCR machine</LI><LI>Agar gel material</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>For each cloning, perform two colony PCR from two different colonies</LI><LI>For each colony PCR, pick one colony and re-suspend it in 10μL of sterile water (in PCR tube)</LI><LI>Pre-mix the One-Taq master mix, primers and water for the corresponding number of reaction, such as for one reaction : 5µL of master mix, 0.25μL of each primer, 3.5μL of water</LI><LI>Keep the colony re-suspended in water at 4°C, to use afterward to inoculate the culture for plasmid extraction</LI><LI>Mix 9µL of the pre-mix with 1L of the re-suspended colony</LI><LI>Place the tube in the PCR machine with the following PCR cycle :
		                        </LI><LI>95°C 5min</LI><LI>95°C 20s</LI><LI>Temperature dependent on primers – 30s</LI><LI>68°C – 1min/kb</LI><LI>Cycle 30 Times the 3 last steps</LI><LI>68°C 5min</LI><LI>Hold at 12°C</LI></OL><LI>Prepare a 0.8% agar gel for sample larger than 1kb and 1.5% for sample smaller than 1kb</LI><LI>Load directly 5μL of the PCR reaction in the gel and a 1kb ladder or 100bp ladder according to the size of your expected fragment</LI><LI>Image the gel</LI></OL><SPAN class="bold"><LI>Plasmid extraction :</LI></SPAN><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>LB with appropriate antibiotic</LI><LI>Falcon tubes</LI><LI>Incubator</LI><LI>Plasmid extraction kit from Qiagen</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>For the colony PCR with the corresponding fragment size, mix 5μL of the re-suspended colony in water in 5mL of LB for high copy plasmid or 10mL for low copy plasmid in a 50mL falcon</LI><LI>Place the culture at 37°C with agitation overnight</LI><LI>From the overnight culture, perform a strick of each culture in a petri dish with the appropriate antibiotic for further glycerol stock (Plate at 37°C overnight, and stored at 4°C)</LI><LI>Centrifuge the culture (5min at 5000rpm) and perform plasmid extraction according to protocol from Qiagen kit</LI></OL><P><SPAN class="red">Note : For low copy plasmid, double re-suspension, lysis and neutralization volume and elute in 30μL.</SPAN></P><SPAN class="bold"><LI>Sequencing and glycerol stock :</LI></SPAN><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>GATC barcodes</LI><LI>Tubes 1.5mL</LI><LI>Primers</LI><LI>dd water</LI><LI>50% glycerol</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><UL><LI>Send the extracted plasmid DNA to sequencing with the appropriate primers to verify the full cloned sequence, follow GATC procedure (5μL DNA at 30 to 100ng/μL + 5μL of one Primer at 5μM)</LI><LI>For correct sequence, inoculate 2mL of LB with appropriate antibiotic with a colony from the corresponding streak</LI><LI>Place the culture at 37°C with agitation during 6 hours, until the culture is trouble</LI><LI>Mix 1.2mL of culture with 400μL of 50% glycerol (15% glycerol)</LI><LI>Annotate the tube (Name of construction / Strain / Antibiotic / Book ref / Date / Your name)</LI><LI>Place the glycerol stock at -80°C</LI></UL></DIV><P class="protocols_title"> Media, buffers and glycerol stock</P><BUTTON class="accordion">Agar plate preparation</BUTTON><DIV class="panel"><OL><LI>Warm agar medium on microwave until agar have been dissolved (be careful it is hot! And open the bottle a little bit before to warm it). Warm time : 4 min / 200mL Bottle ; 6min + 5min + 3min / 700mL Bottle</LI><LI>Wait for the medium to be colder (around 50°C, when you can take it without burning yourself).</LI><LI>Add the antibiotic</LI><LI>Mix without making bubbles</LI><LI>Make plates</LI><LI>Let dry</LI><LI>Store them upside down, in a plastic bag, at 4°C. Identify the bag : Medium name, antibiotic name, your name, date</LI></OL></DIV><BUTTON class="accordion">MRS Medium preparation</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>Difco™ Lactobacilli MRS Broth powder</LI><LI>Agar</LI><LI>MiliQ H<SUB>2</SUB>O</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Weigh 13.7g of Difco™ Lactobacilli MRS Broth powder</LI><LI>Weigh 2.5g of Agar if you want to make agar plates</LI><LI>Add 250mL of miliQ H<SUB>2</SUB>O in a 500mL bottle</LI><LI>Add powders to the bottle</LI><LI>Mix well until everything is disolved</LI><LI>Put it to autoclave</LI><LI>Store it at RT</LI></OL></DIV><BUTTON class="accordion">SOC Medium preparation</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>SOB medium powder</LI><LI>ddH2O</LI><LI>50mL graduated cylinder</LI><LI>Ten 100mL glass bottles</LI><LI>Glucose powder</LI><LI>Syringes</LI><LI>0.2µm filters</LI><LI>50mL sterile tube</LI><LI>5mL sterile pipettes</LI><LI>1.5mL sterile centrifuge tubes</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Prepare SOB medium :</LI><P><OL><LI>Dissolve 14g SOB medium into 500mL (final volume) ddH2O (28g/L final concentration).</LI><LI>Aliquot the solution with the graduated cylinder: 50mL in each 100mL glass bottle.</LI><LI>Autoclave them the same day and cool down.</LI></OL></P><LI>Prepare 2M glucose solution :</LI><P><OL><LI>Dissolve 18 g glucose into 50 ml (final volume) ddH2O and filter-sterilize into a sterile 50 mL tube under the hood PSM.</LI><LI>Aliquot 0.5mL per microcentrifuge tube under the hood PSM.</LI><LI>Store aliquots at -20°C.</LI></OL></P><LI>Prepare SOC medium :</LI><P><OL><LI>Add 0.5mL glucose solution into 50mL SOB medium before use.</LI><LI>Divide into 10mL aliquots with SOC medium to avoid contamination.</LI></OL></P></OL></DIV><BUTTON class="accordion">Antibiotics preparation</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>Antibiotic powders (usually in the common fridge in the flow cytometer room)</LI><LI>ddH2O</LI><LI>100% ethanol solution (under the chemical hood)</LI><LI>Syringes</LI><LI>0.2μm filters</LI><LI>50mL tubes</LI><LI>1.5mL sterile centrifuge tubes</LI><LI>10mL sterile pipettes</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Weight the powder.</LI><LI>Add ~50% of desired liquid to a 50mL tube.</LI><LI>Add powder to liquid in tube and mix to dissolve.</LI><LI>Adjust the volume to the desired final value.</LI><LI>Filter the solution, <SPAN class="red">ONLY if the antibiotic is diluted in water</SPAN>, under the hood PSM.</LI><LI>Aliquot the solution under the hood PSM by 1mL in 1.5mL tubes using a 10mL pipette.</LI><LI>Store aliquots at -20°C.</LI></OL><P><TABLE><TBODY><TR><TH>Antibiotic</TH><TH>Stock concentration</TH><TH>Weight (x g in x mL)</TH><TH>Solution for dilution</TH><TH>Working concentration</TH></TR><TR><TD>Ampicillin</TD><TD>100mg/mL</TD><TD>5g/50mL</TD><TD>H<SUB>2</SUB>O</TD><TD>100µg/mL</TD></TR><TR><TD>Carbenillin</TD><TD>50mg/mL</TD><TD>2.5g/50mL</TD><TD>H<SUB>2</SUB>O</TD><TD>50µg/mL</TD></TR><TR><TD>Kanamycin</TD><TD>50mg/mL</TD><TD>2.5g/50mL</TD><TD>H<SUB>2</SUB>O</TD><TD>50µg/mL</TD></TR><TR><TD>Spectinomycin</TD><TD>100mg/mL</TD><TD>1g/10mL</TD><TD>H<SUB>2</SUB>O</TD><TD>100µg/mL</TD></TR><TR><TD>Chloramphenicol</TD><TD>25mg/mL</TD><TD>-</TD><TD>100% ethanol</TD><TD>25µg/mL</TD></TR><TR><TD>Neomycin</TD><TD>15mg/mL</TD><TD>750mg/50mL</TD><TD>H<SUB>2</SUB>O</TD><TD>-</TD></TR><TR><TD>Streptomycin</TD><TD>100mg/mL</TD><TD>-</TD><TD>H<SUB>2</SUB>O</TD><TD>-</TD></TR><TR><TD>Erythromycin</TD><TD>0.5mg/mL</TD><TD>0.025g/50mL</TD><TD>100% ethanol</TD><TD>-</TD></TR><TR><TD>Tetracyclin</TD><TD>10mg/mL</TD><TD>-</TD><TD>-</TD><TD>10µg/mL</TD></TR></TBODY></TABLE></P></DIV><BUTTON class="accordion">Glycerol stock preparation</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Principle :</SPAN></P><P>Freezing is an efficient way of storing bacteria. Glycerol allows to reduce the harmful effect of ice crystals
		                      of bacteria which can damage cells by dehydration caused by a localized increase in salt concentration leading to denaturation of proteins. Additionally, ice crystals can also puncture cellular membranes. Glycerol as a cryoprotectant depresses the freezing point of bacterial cells, enhancing supercooling. It does so by forming strong hydrogen bonds with water molecules, competing with water-water hydrogen bonding. This disrupts the crystal lattice formation of ice unless the temperature is significantly lowered. The final concentration have to be no more than 20%.</P><P><SPAN class="protocols_subpart">Materials :</SPAN> for cryogenic tubes</P><UL><LI>Cryogenic tube with screw cap volume 2mL</LI><LI>Overnight culture or 6h culture of bacteria in LB with antibiotic for selection</LI><LI>Sterile 50% glycerol</LI><LI>Cryo-box 100 positions</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Label a cryogenic tube with sticker label : construction name, short name, plasmid, antibiotic resistance, strain, book reference, date and initials ...</LI><LI>In sterile conditions mix in a cryogenic tube : 300μL of 50% glycerol + 1200μL of culture.</LI><LI>Homogenize by inverting.</LI><LI>Store the tube at -80°C in a labeled plastic cryo-box 100 positions.</LI></OL></DIV><BUTTON class="accordion">TSS Buffer preparation</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>PEG 8000</LI><LI>MgCl2*6H20</LI><LI>DMSO</LI><LI>LB medium</LI><LI>Syringes</LI><LI>0.2μm filters</LI><LI>15mL sterile tube</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN> to make 250mL TSS Buffer</P><OL><LI>Mix : 25g PEG 8000 + 1.5g MgCl2*6H20 in 12.5mL DMSO</LI><LI>Add LB to 250 mL.</LI><LI>Filter sterilize (0.22 μm filter) under the hood PSM.</LI><LI>Aliquot in 15mL sterile tubes.</LI></OL></DIV><BUTTON class="accordion">Gibson Assembly Mix preparation</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>1.5mL microtubes</LI><LI>PCR tubes</LI><LI>To prepare the 5X Isothermal solution (ISO 5X) :</LI><UL><LI>Tris-HCl pH 7.5 solution 1M</LI><LI>PEG-8000</LI><LI>MgCl<SUB>2</SUB> solution 1M</LI><LI>DTT 1M</LI><LI>dNTP Mix 10mM</LI><LI>NAD 100mM</LI><LI>ddH<SUB>2</SUB>O</LI></UL><LI>To prepare the Gibson Assembly Mix 2X solution :</LI><UL><LI>ISO 5X</LI><LI>T5 exonucelase (10 U/µL)</LI><LI>Taq DNA ligase (40 U/µL)</LI><LI>Phusion DNA polymerase (2 U/µL)</LI><LI>ddH<SUB>2</SUB>O</LI></UL></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><UL><LI>5X isothermal reaction buffer : preparation for 10mL final solution</LI><TABLE><TBODY><TR><TH>10mL final</TH><TH>Final concentration</TH></TR><TR><TD>Tris-HCl pH 7.5</TD><TD>5mL</TD><TD>500mM</TD></TR><TR><TD>PEG-8000</TD><TD>2.5g</TD><TD>50%</TD></TR><TR><TD colspan="3">Vortex</TD></TR><TR><TD>MgCl<SUB>2</SUB></TD><TD>500μL</TD><TD>50mM</TD></TR><TR><TD>DTT</TD><TD>500μL</TD><TD>50mM</TD></TR><TR><TD>dNTP Mix</TD><TD>1mL</TD><TD>1mM</TD></TR><TR><TD>NAD</TD><TD>500μL</TD><TD>5mM</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>Qsp 10mL</TD><TD>-</TD></TR></TBODY></TABLE><P>The 5X isothermal reaction buffer has to be aliquot in 1mL, in 1.5mL tubes.</P><LI>2X Gibson Assembly mix</LI><TABLE><TBODY><TR><TH>x2</TH><TH>x2</TH></TR><TR><TD>Final volume (µL)</TD><TD>1000</TD><TD>1500</TD></TR><TR><TD>ISO 5X (μL)</TD><TD>300.8</TD><TD>451.2</TD></TR><TR><TD>T5 exonucelase (10 U/μL)</TD><TD>0.6</TD><TD>0.9*</TD></TR><TR><TD>Taq DNA ligase (40 U/μL)</TD><TD>150.4</TD><TD>225.6</TD></TR><TR><TD>Phusion DNA polymerase (2 U/μL)</TD><TD>18.8</TD><TD>28.2</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>529.4</TD><TD>794.1</TD></TR></TBODY></TABLE><P>The Gibson Assembly 2X mix has to be aliquot in 10μl in PCR tubes on a rack. Put the rack at -20°C. When it’s freeze put all tubes in a “tips” box.</P><P>* : Or do a dilution 1/10 of the solution 10 U/μL and add 9μL for 1500μL final.</P></UL></DIV><BUTTON class="accordion">Twist DNA resuspension</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>Centrifuge</LI><LI>10mM Tris-HCl pH 8.0</LI><LI>PRC tubes to aliquot the DNA resuspended</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Upon receipt, briefly centrifuge the tube or plate and add without mixing nuclease-free 10mM Tris-HCl pH 8.0 to the desired concentration : in 96-well plate add 10μL buffer. We do not recommend resuspension in water</LI><LI>Wait for 3 to 4 hours at RT</LI><LI>Resuspend by pipetting and briefly centrifuge if needed</LI><LI>Prepare aliquots of the stock dilution and separate working aliquots to limit chances of contamination and to reduce the number of freeze/thaw cycles. Use working aliquots as soon as possible after preparation and minimize exposure to high temperatures. (Keep the mother plate in case you need it after)</LI><LI>For long-term storage (up to one year), freeze DNA at -20°C. For even longer storage, freeze DNA at -80°C</LI></OL></DIV><P class="protocols_title"> Cleaning procedures</P><BUTTON class="accordion">Cleaning procedure for beads</BUTTON><DIV class="panel"><P>Used beads should be placed in recipient with bleach before washing.</P><OL><LI>Put the beads (without the bleach) in a bucket.</LI><LI>Add new bleach and water to cover all of the beads, wait 30min for full decontamination.</LI><LI>Remove the bleach, place the beads in a clean bucket and add water. Mix beads and water to be sure that all beads have been in contact with the water.</LI><LI>Re-iterate the previous step.</LI><LI>Perform twice step 3 with deionized water instead of water.</LI><LI>Perform once step 3 with ethanol 70%.</LI><LI>Remove ethanol.</LI><LI>Clean your bench.</LI><LI>Dry beads using paper (place paper on your bench, smoothly put beads on it and use paper to dry them).</LI><LI>Place beads into small bottles (you can use the paper as a funnel).</LI><LI>Autoclave the bottles (you can just add autoclave tap and place them on kitchen table).</LI></OL></DIV><BUTTON class="accordion">Cleaning procedure for electroporation cuvettes</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Materials :</SPAN></P><UL><LI>Dirty Gene Pulser / MicroPulser Electroporation cuvettes</LI><LI>2 500mL beakers</LI><LI>1L of 70% ethanol</LI></UL><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Arrange the cuvettes side by side in two beakers identified A and B.</LI><LI>Fill cuvettes with 70% ethanol.</LI><LI>Add other lying cuvettes and remove the caps on top.</LI><LI>Fill the beakers up to 500mL with ethanol. Warning : all cuvettes must be filled.</LI><LI>Cover the beakers of double aluminum foil.</LI><LI>Sonicate the beakers alternatively 15min A, 15min B, 15min A, 15min B, in the sonication bath.</LI><LI>Cool on the bench overnight.</LI><LI>Empty the ethanol from the beaker by opening minimally the aluminum foil (to avoid contamination).</LI><LI>Make 5 to 8 holes in the aluminum of each beakers with a syringe needle to create a ventilation.</LI><LI>Dry the cuvettes in their beakers in an oven at 65°C (2 days or more).</LI><LI>When the tanks and caps are dry, close them under sterile condition (flame or under the hood).</LI></OL></DIV><BUTTON class="accordion">Cleaning procedure for 96 wells plates</BUTTON><DIV class="panel"><P><SPAN class="protocols_subpart">Clean recommandation :</SPAN></P><P>96 wells plate should be clean just after usage (at least in the 48h). Afterward cells tend to precipitate and it starts to be trickier to well clean them. If 96 well plates are store for a long time before cleaning, it is important to carefully control the plate after cleaning. Do not hesitate to throw plate which are not perfectly clean (and white).</P><P><SPAN class="protocols_subpart">Protocol :</SPAN></P><OL><LI>Throw the liquid culture from the plate in a bucket and add bleach to it. After 30min, throw the liquid to the sink and clean the bucket and the sink.</LI><LI>Place the plate to a bucket and add bleach mix to water to completely fill the plate. Wait 30min for complete decontamination of the plate. (do not wait several days, it obstructs the sink and the plate will turn yellow with time).</LI><LI>Remove the bleach.</LI><LI>Clean the plate with water, repeat this step twice. You can use the tap to add water specifically to each well. Remove as much as possible liquid from the plate between two rounds of cleaning by manually shaking the plate.</LI><LI>Clean the plate with deionized water, repeat this step twice.</LI><LI>Clean the plate with ethanol, repeat this step twice.</LI><LI>Remove liquid from the plate and let it dry on paper in the reverse orientation during at least one day (depending the external temperature and humidity percentage).</LI><LI>Control the quality of the cleaning, if the plate is not perfectly clean: throw it, if it is clean, place it on the shelf with it friends.</LI></OL></DIV><DIV class="footer_sponsors"><DIV class="contact_us"><H3> Sponsors </H3></DIV><DIV class="follow_us"><H3> Follow us on </H3></DIV><DIV class="contact_us"><H3> Contact us </H3><DIV class="contact_us_icons"><DIV class="deux_colonnes"><DIV class="colonne_2_70"><DIV><P class="p_footer"> igem.assomtp@gmail.com </P></DIV><DIV><P class="p_footer"> Centre de Biochimie Structurale, 29 Rue de Navacelles, 34090 Montpellier, France </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>