"
Page
Discussion
View source
History
teams
Log in
 
Team:Austin Texas/photocage
From 2014.igem.org
UT Austin
HOME
PROJECTS
HUMAN PRACTICES
OFFICIAL REQUIREMENTS
Expanded Genetic Code
Measurement Kit
Photocaged
T7 RNA Polymerase
Interlab
Study
Caffeinated coli
@SXSW Create
Team
Parts
Safety
Medals
Photocaged T7 RNA Polymerase
Contents
1 Introduction
2 Background
3 Experimental Methods
4 Results
5 Discussion
6 References
Introduction
Figure 1. The ONBY ncAA used for our photocaging project.  When exposed to 365 nm light, the ONB group is released, resulting in a normal tyrosine amino acid.
To demonstrate an application of adding a noncanonical amino acid to the genetic code, we recreated a light-activatable T7 RNA polymerase (RNAP) for the spatio-temporal control of protein expression (Chou et al. 2010). The light-activatable T7 RNAP was created by mutating a tyrosine codon at position 639 of a domain crucial for the polymerization of RNA during transcription.  Y639 was mutated to an amber codon, allowing us to incorporate a ncAA at this position.  We used ortho-nitrobenzyl tyrosine (ONBY), which is a "photocaged" ncAA (Figure 1).  Thus, if our synthetase/tRNA pair works, position 639 should contain ONBY in place of tyrosine.
When the photocaged ONBY is incorporated at position 639, RNAP activity is inhibited.  The polymerase can become activated only upon decaging of the ONB group from the ONBY.  This is accomplished by exposing the cells to 365 nm light.  When exposed to 365 nm light, the ONB group is released, resulting in a normal tyrosine amino acid at position 639.  T7 RNAP was selected because of its orthogonal nature, which allows us to selectively induce the expression of specific genes that are preceded by the T7 RNAP promoter.  Because T7 promoters are not natively found in E. coli, a gene downstream of a T7 promoter may be exclusively expressed through the introduction of 365 nm light.
Background
Tyrosine residue 639 (Y639) was specifically targeted because it lies on a crucial position on the O-helix domain of T7 RNAP and has been proven to be essential for polymerization (Chou et al. 2010). The Y639 residue in the O-helix is responsible for two major roles in the elongation stage of DNA polymerization. First, this tyrosine residue discriminates between deoxyribose and ribose substrates using the Tyrosyl-OH (Temiakov et al. 2004). Second, Y639 is responsible for moving newly synthesized RNA out of the catalytic site and preparing for the next NTP to be inserted (Achenbach 2004). These functions of the O-helix were shown to be essential through mutational analysis (Osumidavis 1994). Introducing a bulky group such as ONBY in place of tyrosine renders the enzyme nonfunctional in several ways. First, the native tyrosine-OH is not there anymore to coordinate Mg2+, which plays an essential role in discriminating between deoxyribose and ribose substrates. Additionally, because of the sterics of the ONBY molecule itself, it blocks incoming nucleotides from entering the active site. Because the loss of this tyrosine residue in the active site leads to a non-functional polymerase, Y639 proved to be a good candidate for incorporating a photocaged amino acid (Chou et al. 2010).
Figure 2. The caged T7 RNAP is decaged via exposure to 365 nm light. Figure reproduced from Chou et al. 2010.
Incorporation of ONBY at position 639 of T7 RNAP halts activity because of the bulky nature of ONBY (Chou et al. 2010).  This ONB side group effectively renders T7 RNAP inactive.  However, the bulky ONB group is able to be removed through irradiation with 365 nm light. The wavelength of light used to decage the amino acid proved to be another advantage of this system because 365 nm light is not toxic to the cell (Chou et al 2010). Once the ONB group is removed, a normal tyrosine residue is left in its place, restoring T7 RNA polymerase activity (Figure 2).
In order to incorporate the ncAA into amberless E.coli (which is described here ), a Methanocaldoccus jannaschii tyrosyl-tRNA synthetase/tRNA pair was previously mutated to selectively charge and incorporate ONBY. Six residues (Tyr 32, Leu 65, Phe 108, Gln 109, Asp 158, and Leu 162) on the original synthetase were randomized and the library was selected for its ability to charge ONBY while discriminating against other canonical amino acids.  The resulting mutant ONBY synthetase contained five mutations (Deiters et al. 2006). The following are the residues that were mutated on the synthetase: Tyr32→Gly32, Leu65→Gly65, Phe108→Glu108, Asp158→Ser158, and Leu162→Glu162. The Asp158→Ser158 and Tyr32→Gly32 mutations are believed to result in the loss of hydrogen bonds with the natural substrate, which would disfavor binding to tyrosine. Additionally, the Tyr 32→Gly 32 and Leu 65→Gly 65 mutations are believed to increase the size of the substrate-binding pocket to accommodate for the size of the bulky o-nitrobenzyl group (Deiters et al. 2006).
Experimental Methods
In order to create an in vivo light-activated GFP expression system, two plasmids were constructed and transformed into amberless E. coli cells. The first plasmid contains the tRNA and synthetase pair, which is necessary to incorporate ONBY into the amber stop codon of the T7 RNAP. The second plasmid contains the coding sequence for T7 RNAP, which has a mutation on the Tyrosine 639 residue, and a GFP coding sequence bound to an upstream T7 promoter. The T7 RNAP coding sequence was taken from the BioBrick part BBa_K145001 and mutated to construct our desired plasmid. Once these components were assembled by Gibson Assembly, the two plasmids were transformed via electroporation into aliquots of amberless E. coli.
For this experiment, there were also other necessary control strains to test alongside the experimental strain. These controls included a T7-GFP construct with no amber stop codon in the O-helix (to serve as a positive control for expression with T7 polymerase), sfGFP amberless E. coli (to observe the expression of GFP by native polymerase), amberless E. coli (to serve as a cell background control), and LB supplemented with ncAA (to serve as a media background control).
All of these strains were grown overnight in LB and appropriate antibiotics. The next morning, three 100 microliters samples of each of the strains were inoculated into three different conditions. The conditions are as follows:
(-)IPTG, (-)ONBY
(+)IPTG, (+)ONBY
(+)IPTG, (+) ONBY
Each of the conditions were in sterile test tubes and contained 1 mL of media and appropriate antibiotics. These cultures were allowed to grow for 2 hours. After 2 hours of growth, IPTG was added to all (+)IPTG condition tubes. The test tubes were allowed to grow another 2-4 hours, or until the OD600 reached at least 0.3. At this point, five 100 microliter samples of each of the strains were transferred into a 96-well plate. Each of the five samples represents a time interval of light exposure.
Next, the 96-well plate was covered in foil such that only the first column (which only contained samples that are to be exposed to 365 nm light for 30 minutes) was exposed. The handheld UV light was held 5 cm over the 96-well plate and the first column was exposed to the 365 nm light for 15 minutes. After 15 minutes of exposure, the foil was removed to reveal the second column of cultures. Then, after another 10 minutes of exposure, the foil was removed to exposed the third column. After another 4 minutes, the fourth column was exposed. Finally, after the final 1 minute of exposure, the plate was re-covered with foil. After all of the sample were irradiated, an initial measurement of GFP expression was taken by the fluorometer. The purpose of the initial reading was to see what the initial effects of irradiation on the cultures were.
Once the initial readings were recorded, the 100 microliters samples in the black 96 well plate were transferred into a deep 96-well plate, covered, and allowed to grow overnight at 37°C and 225 RPM.
The cultures were allowed 16 hours of growth so that the decaged T7 RNA polymerase would have time to polymerize mRNA transcripts of the GFP, which is downstream of a T7 promoter. After 16 hours of growth, cultures were transferred back to a black 96-well plate and the fluorescence was measured.
Results
Figure 3. GFP fluorescence observed after exposure to 365 nm light for the time indicated.  Althought there is a fair amount of fluorescence without 365 nm light, the amount of fluorescence is directly proportional to the amount of time that the culture was exposed to 365 nm light.
To demonstrate the functionality of the light-activatable T7 RNAP, a GFP reporter was placed under the control of a T7 reporter.  Cells with this plasmid were grown in culture with ONBY for 4-6 hours.  These cells were then exposed to varying amounts of 365 nm light, ranging from 0 to 30 minutes.  During this exposure, the ONB functional group at position ONBY639 should be released, resulting in de-caging of Y639.
After this exposure, the cells were grown for an additional 16 hours, allowing the newly decaged T7 RNAP to transcribe the GFP reporter gene, which ultimately results in fluorescence.  As can be seen in Figure 3, while the fluorescence background at time zero is not as low as we would like, there is a clear and dramatic increase in fluorescence that is directly dependent on the amount of time the culture was exposed to 365 nm light.  This result is consistent with previous results using this system (Chou et al. 2010).
We also tested the spatial activation of T7 RNAP activity by placing a cutout above a petri dish and shining 365 nm light through it (Figure 4). Although cells were present uniformly across the entire surface, we only observed GFP expression in the areas exposed to light, as expected.
Discussion
Figure 4. Photocage culture plated onto ONBY solid media and exposed to 365 nm light under a UT Longhorn stencil for 15 minutes.  Although not yet perfected, the system is indeed light-activated and shows promise.
The data from our photocage project provides strong evidence that we have, in fact, replicated the light-activated protein expression system that Chou et al. had constructed. From the data, we are able to observe that GFP expression has a positive correlation with the amount of time that it is exposed to 365 nm light. Although the difference in fluorescence between 0 minutes and 1 minute appears to be negligible, there was a clear change in fluorescence at the 5 minute, 15 minute, and 30 minute time intervals.
Although the increases in fluorescence are clearly present, there are still a few problems regarding the system. For one, the level of background expression of the uninduced T7 RNAP is relatively high. The uninduced caged T7 RNAP has roughly 25 times more expression compared to the negative control. The relatively high level of fluorescence may be explained by the efficient nature of T7 RNAP. Even if a few polymerases were decaged by spurious light during the initial making of the culture or during transfers, the decaged T7 RNAP would have 16 hours to transcribe the RNA the GFP gene.
A potential solution to reduce the background expression of GFP would be to mutate a second amber stop codon into the coding sequence of T7 RNA polymerase. Adding a second stop codon would theoretically reduce the number of T7 RNA polymerases that are being transcribed. If a second amber stop codon were introduced into the coding sequence of T7 RNAP, it would be beneficial for it to be at another potentially vital site in the enzyme such as Y427.  This is important if the incorporated ONBY at 639 is being decaged at times other than the exposure to 365 nm light.  Another possible location of the second amber stop codon would be near the start of the sequence, such as at Y116. Placing an amber stop codon at the beginning of the sequence would prematurely halt the translation in the absence of ONBY, and this would be important if misincorporation is the cause of the high background fluorescence. Either of these solutions could potentially reduce the background.
A third potential solution would be to redesign the entire plasmid containing the T7 RNAP to reduce the amount of T7 RNAP that is made by the cell.  This could be through a lower copy number plasmid or choosing a different promoter sequence for the purpose of slowing down gene expression.
The recreation of this light-activatible gene expression system provides a potential foundation for future iGEM projects that wish to use light to control gene expression.  By replacing the GFP reporter with coding sequences for other proteins,  we will be able to explore novel applications of light-activated protein expression. Photodecaging of ONBY can also be used to activate other types of proteins that include important tyrosine residues.
References
Achenbach, J. C., Chiuman, W., Cruz, R. P. G., Li, Y. (2004), DNAzymes: from creation in vitro to application in vivo.Curr. Pharm. Biotechnol. 5:321 –336.
Chou, C., Young, D. D. and Deiters, A. (2010), Photocaged T7 RNA polymerase for the light activation of transcription and gene function in pro- and eukaryotic cells. ChemBioChem 11:972–977.
Deiters, A., Groff, D., Ryu, Y., Xie, J. and Schultz, P. G. (2006), A genetically encoded photocaged tyrosine. Angew. Chem. 118:2794–2797.
Osumidavis, P. A., Sreerama, N., Volkin, D. B., Middaugh, C. R., Woody, R. W., Woody, A. Y. M. (1994), Photocaged T7 RNA polymerase for the light activation of transcription and gene function in pro- and eukaryotic cells. J. Mol. Biol. 237:5–19.
Temiakov, D., Patlan, V., Anikin, M., McAllister, W. T., Yokoyama, S., Vassylyev, D. G.. (2004), Structural basis for substrate selection by T7 RNA polymerase. Cell 116:381–391.
Retrieved from " http://2014.igem.org/Team:Austin_Texas/photocage "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
