<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Austin_Texas_kit"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Austin_Texas/kit">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Austin_Texas/kit&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Austin_Texas/kit&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Austin_Texas/kit&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Austin_Texas/kit" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Austin Texas/kit</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TABLE id="menu" width="100%" cellspacing="0"><TBODY><TR height="20px"><TD rowspan="3" align="center" onmouseover="this.bgColor='#000000'" onmouseout="this.bgColor='#CC6600'" bgcolor="#CC6600"><A href="https://2014.igem.org/Team:Austin_Texas" style="text-decoration:none;color:#1C140D"><FONT color="#FFFFFF">
UT AustinHOME </FONT></A></TD><TD colspan="3" align="center" bgcolor="#666666"><FONT color="#FFFFFF">PROJECTS</FONT></TD><TD colspan="1" align="center" bgcolor="#666666"><FONT color="#FFFFFF">HUMAN PRACTICES</FONT></TD><TD colspan="2" align="center" bgcolor="#666666"><FONT color="#FFFFFF">OFFICIAL REQUIREMENTS</FONT></TD></TR><TR height="20px"><TD rowspan="2" align="center" onmouseover="this.bgColor='#000000'" onmouseout="this.bgColor='#CC6600'" bgcolor="#CC6600"><A href="https://2014.igem.org/Team:Austin_Texas/kit" style="text-decoration:none;color:#1C140D"><FONT color="#FFFFFF">Expanded Genetic CodeMeasurement Kit</FONT></A></TD><TD rowspan="2" align="center" onmouseover="this.bgColor='#000000'" onmouseout="this.bgColor='#CC6600'" bgcolor="#CC6600"><A href="https://2014.igem.org/Team:Austin_Texas/photocage" style="text-decoration:none;color:#1C140D"><FONT color="#FFFFFF">PhotocagedT7 RNA Polymerase</FONT></A></TD><TD rowspan="2" align="center" onmouseover="this.bgColor='#000000'" onmouseout="this.bgColor='#CC6600'" bgcolor="#CC6600"><A href="https://2014.igem.org/Team:Austin_Texas/interlab_study" style="text-decoration:none;color:#1C140D"><FONT color="#FFFFFF">InterlabStudy</FONT></A></TD><TD rowspan="2" align="center" onmouseover="this.bgColor='#000000'" onmouseout="this.bgColor='#CC6600'" bgcolor="#CC6600"><A href="https://2014.igem.org/Team:Austin_Texas/human_practices" style="text-decoration:none;color:#1C140D"><FONT color="#FFFFFF">Caffeinated coli@SXSW Create</FONT></A></TD><TD align="center" onmouseover="this.bgColor='#000000'" onmouseout="this.bgColor='#CC6600'" bgcolor="#CC6600"><A href="https://2014.igem.org/Team:Austin_Texas/team" style="text-decoration:none;color:#1C140D"><FONT color="#FFFFFF">Team</FONT></A></TD><TD align="center" onmouseover="this.bgColor='#000000'" onmouseout="this.bgColor='#CC6600'" bgcolor="#CC6600"><A href="https://2014.igem.org/Team:Austin_Texas/parts" style="text-decoration:none;color:#1C140D"><FONT color="#FFFFFF">Parts</FONT></A></TD></TR><TR height="20px"><TD align="center" onmouseover="this.bgColor='#000000'" onmouseout="this.bgColor='#CC6600'" bgcolor="#CC6600"><A href="https://2014.igem.org/Team:Austin_Texas/safety" style="text-decoration:none;color:#1C140D"><FONT color="#FFFFFF">Safety</FONT></A></TD><TD align="center" onmouseover="this.bgColor='#000000'" onmouseout="this.bgColor='#CC6600'" bgcolor="#CC6600"><A href="https://2014.igem.org/Team:Austin_Texas/medal_requirements" style="text-decoration:none;color:#1C140D"><FONT color="#FFFFFF">Medals</FONT></A></TD></TR><TR height="34px"><TD colspan="9" align="center" bgcolor="#000000"><FONT color="#FFFFFF" size="+2">Expanded Genetic Code Measurement Kit</FONT></TD></TR></TBODY></TABLE></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Kit_Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Kit Introduction</SPAN></A></LI><LI class="toclevel-2"><A href="#Motivation"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Motivation</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-1"><A href="#Experimental_Design_and_Method"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Experimental Design and Method</SPAN></A></LI><LI class="toclevel-2"><A href="#Plasmids"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Plasmids</SPAN></A></LI><LI class="toclevel-2"><A href="#Using_The_Kit"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Using The Kit</SPAN></A></LI><LI class="toclevel-2"><A href="#Experimental_Preparation"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Experimental Preparation</SPAN></A></LI><LI class="toclevel-1"><A href="#Results_and_Data"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results and Data</SPAN></A></LI><LI class="toclevel-2"><A href="#Fidelity_of_Incorporation"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Fidelity of Incorporation</SPAN></A></LI><LI class="toclevel-2"><A href="#Synthetase_Efficiency"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Synthetase Efficiency</SPAN></A></LI><LI class="toclevel-2"><A href="#Incorporation_Value"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Incorporation Value</SPAN></A></LI><LI class="toclevel-1"><A href="#Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1"><A href="#Conclusion"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1"><A href="#ncAA_Table"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">ncAA Table</SPAN></A></LI><LI class="toclevel-1"><A href="#ncAA_Synthetase_Table"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">ncAA Synthetase Table</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><DIV align="justify"><H1><SPAN class="mw-headline" id="Kit_Introduction">Kit Introduction</SPAN></H1><P>In recent years the ability to expand the genetic code has been made possible by re-coding the amber stop codon, UAG, via the use of modified tRNA synthetase/tRNA pairs.  These synthetase/tRNA pairs act together to charge a modified tRNA with a non-canonical amino acid (ncAA), an amino acid that is not one of the 20 amino acids normally encoded by a codon.   While the library of ncAA synthetase/tRNA pairs continues to grow, the properties of each pairing have yet to be systematically characterized using a standardized methodology.
</P><P>There are certain desired properties of a synthetase/tRNA pair, such as high fidelity and high efficiency, that must be characterized. Without knowing these properties, it is impossible to know how effectively a synthetase/tRNA pair will work and the resulting products will remain vague unless specifically analyzed with each use. The 2014 University of Texas at Austin iGEM Team has developed a standardized method that allows for efficient qualitative and quantitative in vivo characterization of ncAA tRNA synthetase/tRNA pairs. This Expanded Genetic Code Measurement Kit is portable, easy-to-use, and can be quickly used with any synthetase/tRNA pair.
</P><H2><SPAN class="mw-headline" id="Motivation">Motivation</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1.</B> The process of charging a ncAA, followed by its incorporation during translation.  To do this successfully, a novel tRNA synthetase must be evolved that specifically recognizes the ncAA.</DIV></DIV></DIV><P>In order to recode UAG, a synthetase must be mutated to effectively &quot;charge&quot; a ncAA onto the corresponding tRNA. Various methods of directed evolution are typically used to modify a synthetase such that it can interact with and charge a specific ncAA (Liu et al. 2010). The ncAA synthetases available have different levels of reported efficiency, and they are not always well characterized.  Many of the ncAA systems are not widely (re)used. They are published in single, short articles lacking full documentation, and our own experiences of working with a number of ncAAs indicate that not all ncAA synthetases are created equal. Thus, we created a standard kit designed to characterize the properties of any ncAA synthetase/tRNA pair. <B>Our goal is to produce a measurement kit that is cheap, reproducible, easy to use, AND easily portable (i.e. you don't need a lot of advanced equipment).</B></P><H1><SPAN class="mw-headline" id="Background">Background</SPAN></H1><P>The genetic code is composed of 64 nucleotide triplets (codons) that code for 20 highly conserved amino acids that are essential to all organisms on Earth. While the genetic code is specific, it is also degenerate, meaning that more than one codon can encode for the incorporation of a specific amino acid.  For example, there are six serine codons and three stop codons (called amber, ochre, and opal).  By recoding one of the redundant codons, the recoded codon can signal for the incorporation of a non-canonical amino acid (ncAA) rather than the codon's original usage. Of the three stop codons, the amber codon is the least abundant and thus, the easiest and most efficient to recode.
</P><P>The Schultz lab was the first to expand the genetic code using a unique synthetase/tRNA pair. This synthetase/tRNA pair originated from <I>M. jannaschii</I> tyrosine RS (Mj-TyrRS) and allowed the cell to incorporate <I>o</I>-methyl-<SMALL>L</SMALL>-tyrosine at the amber codon (Wang et al. 2001). Since then, numerous ncAA synthetase/tRNA pairs have been generated. A total of seven ncAAs were used in addition to tyrosine for our project.  <A href="https://2014.igem.org/Team:Austin_Texas/kit#ncAA_Table" class="external text" rel="nofollow">The full list of amino acids used in this study can be found here</A>.
</P><P>It is important to note that complications can arise when the genetic code is recoded. In a normal bacterium, release factor RF1 is responsible for terminating translation when the ribosome reaches an amber stop codon. To expand the potential of ncAAs, it is vital to avoid termination at amber codons.  Seeing this need, the Church and Isaacs groups engineered a strain of <I>E. coli</I> that had all of the amber codons removed from the genome, which allowed them to knock out the RF1 gene (Isaacs et al. 2011). The resulting strain, called &quot;amberless&quot; <I>E. coli</I>, has NO AMBER CODONS.  Thus, to better characterize the synthetase/tRNA pairs, we have employed amberless <I>E. coli</I> as a chassis organism.  The advantage of this system is that since there are no amber codons in the genome, we cannot accidentally interfere with any cellular processes.  Additionally, since RF1 is knocked out, there is no competition between RF1 and the novel synthetase/tRNA pair.  These factors allow us to better assess the fidelity and efficiency of ncAA tRNA synthetase/tRNA pairs.
</P><H1><SPAN class="mw-headline" id="Experimental_Design_and_Method">Experimental Design and Method</SPAN></H1><H2><SPAN class="mw-headline" id="Plasmids">Plasmids</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:572px;"><DIV class="thumbcaption"><B>Figure 2.</B>  pStG contains the specific ncAA synthetase/tRNA pair being tested. pFRYC and pFRY are the ncAA kit reporter plasmids.  pFRYC is the control plasmid that yields GFP and RFP expression regardless of synthetase/tRNA pair or the presence of ncAA. pFRY is a nearly identical plasmid with a single difference, the linker between the GFP and RFP sequences contains an amber codon (star) in place of the tyrosine codon found in pFRYC. Thus, the level of GFP expression for pFRY is directly dependent on the efficiency and fidelity of the synthetase/tRNA pair being tested.</DIV></DIV></DIV><P>The kit consists of a three plasmid system: pStG, pFRYC, and pFRY (<B>Figure 2</B>).
</P><UL><LI><B>pStG</B> contains the ncAA Synthetase/tRNA pair (St) to be tested as well as a Gentamicin (G) resistance gene.
</LI><LI><B>pFRYC</B> is the control plasmid and contains an IPTG-induced reporter system and a kanamycin resistance gene.  The reporter system is composed of RFP and sfGFP fused via a linker sequence between the two.
</LI><LI><B>pFRY</B> is the experimental reporter plasmid and is nearly identical to pFRYC with the exception that its linker sequence contains an amber codon (star in <B>Figure 2</B>) in the middle of the linker, whereas pFRYC contains a tyrosine codon in the same location.
</LI></UL><H2><SPAN class="mw-headline" id="Using_The_Kit">Using The Kit</SPAN></H2><P>To use the kit properly, each culture must have two of the three plasmids from above.  There must always be a pStG plasmid as well as either pFRYC or pFRY.  However, additional controls can be conducted using only one of the three plasmids.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 3.</B> Schematic demonstrating the gene expression of the kit plasmids under different growth conditions in the presence of IPTG. A) pFRYC (+/-) pStG, (+/-) ncAA. B) pFRY  (-) pStG, (+/-) ncAA.  C) pFRY  (+) pStG, (-) ncAA.  D) pFRY  (+) pStG, (+) ncAA. The above results represent the &quot;perfect&quot; tRNA synthetase/tRNA pair: one that has 100% fidelity and 100% efficiency.  Actual synthetase/tRNA pairs will occasionally misincorporate an incorrect amino acid, and these pairs will not always be perfectly efficient at conducting every step in the process of reading through an amber codon.</DIV></DIV></DIV><UL><LI><B> A)</B> In a cell containing pStG and pFRYC, the ribosome will translate the RFP reporter, linker, and sfGFP, producing red and green fluorescent proteins that result in visible yellow fluorescence (<B>Figure 3A</B>).  This happens because the reporter contains no amber codon, and thus does not require a ncAA synthetase/tRNA pair.
</LI></UL><UL><LI><B> B)</B> If a cell contained only pFRY, without pStG, then the ribosome will translate the RFP and terminate at the amber codon on the linker producing a red fluorescence (<B>Figure 3B</B>).  This occurs because without the pStG, the amber codon is not recoded to allow for ncAA incorporation.
</LI></UL><UL><LI><B>C)</B> In a cell containing pStG and pFRY, there are multiple possible outcomes, depending on what is present in the culture.  In the absence of ncAA, the ribosome will translate the RFP and terminate at the amber stop codon on the linker producing a red fluorescence (<B>Figure 3C</B>).  This occurs because while the codon has been recoded, there is no ncAA and without ncAA there can be no incorporation at the amber codon.
</LI></UL><UL><LI><B> D)</B> In a cell containing pStG and pFRY, there are multiple possible outcomes.  In the PRESENCE of ncAA, the ribosome will translate the RFP, then it incorporates the ncAA at the amber codon in the linker, and finally it proceeds to translate the downstream sfGFP reporter, producing a visible yellow fluorescence (<B>Figure 3D</B>).  This occurs because the synthetase found in pStG can covalently attach the ncAA to the tRNA gene present in pStG.  This &quot;charged tRNA&quot; is then present during translation, allowing incorporation of the ncAA into the reporter protein.
</LI></UL><P>
However, the above scenarios assume that the synthetase/tRNA pair functions perfectly.  Some of these systems are non-ideal because they have:
</P><UL><LI><B>LOW FIDELITY:</B> Actual synthetase/tRNA pairs sometimes have <B>low fidelity</B>, leading to the incorporation of an incorrect amino acid.  This &quot;misincorporation&quot; means that such a synthetase/tRNA pair would not give results that mimic <B>Figure 3C</B> in the absence of ncAA.  Rather, in the absence of ncAA, such a pair would give results similar to <B>Figure 3D</B>, full expression of RFP and sfGFP.  High rates of misincorporation are equivalent to <B>low fidelity</B>, and tRNA synthetases with low fidelity are not ideal because they give a mixture of protein products with the normal and new amino acids at the recoded position.
</LI></UL><UL><LI><B>LOW EFFICIENCY:</B> Actual synthetase/tRNA pairs can also sometimes have <B>low efficiency</B>.  As these are artificially selected pairs, sometimes they do not function as well as natural synthetase/tRNA pairs.  In such a case, the amount of sfGFP translated in <B>Figure 3D</B> might be significantly lower than expected.  <B>low efficiency</B> is also not ideal as it may limit levels of protein expression.
</LI></UL><H2><SPAN class="mw-headline" id="Experimental_Preparation">Experimental Preparation</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:222px;"><DIV class="thumbcaption">Protection of light-sensitive ncAAs using a foil-wrapped incubator.</DIV></DIV></DIV><P>Each ncAA synthetase/tRNA pair to be tested was cloned into pStG and transformed into pFRYC amberless <I>E. coli</I> and pFRY amberless <I>E. coli</I>. Other necessary control strains include RFP amberless <I>E. coli</I> (RFP control), sfGFP amberless <I>E. coli</I> (GFP control), amberless <I>E. coli</I> (cell background control), and LB media supplemented with ncAA (media background control). An overnight culture of each strain was grown in LB with the appropriate antibiotics at 37ºC and 225rpm. 10 mL of media with the appropriate antibiotics was inoculated with 100 µL of overnight culture and allowed to grow in the same conditions until the culture density was ~0.2-0.3 OD<SUB>600</SUB>, or ~3 hours. The 10 mL culture was split between 4 different sterile test-tubes, 2 mL of culture per tube. The conditions of the four test tubes were as follows: 
</P><UL><LI>-IPTG,-ncAA
</LI><LI>-IPTG,+ncAA 
</LI><LI>+IPTG, -ncAA
</LI><LI>+IPTG, +ncAA
</LI></UL><P>If the cultures did not have IPTG or ncAA, an equal volume of sterile deionized water was added in order to keep the volumes between cultures constant. Once the water, the IPTG, and the ncAA were added appropriately, the cultures were allowed to grow to ~0.5 OD<SUB>600</SUB>. 70 µL of each culture condition and control culture were added to a separate well in clear-bottom black 96-well plate for fluorescence and OD<SUB>600</SUB> readings using a plate reader. 
</P><P>We accounted for several ncAA-specific concerns. These included: light-sensitivity, oxidation, and interference with fluorescence readings. Certain ncAAs were protected from light due to their light-sensitive molecular structures such as <A href="https://2014.igem.org/Team:Austin_Texas/kit#ncAA_Table" class="external text" rel="nofollow">ONBY and AzF</A>. These ncAAs were prepared in a dark room and wrapped in foil. All cultures were grown in a foil-wrapped incubator for consistency. Some ncAAs are more prone to oxidation, specifically <SMALL>L</SMALL>-DOPA. When oxidized, the solution turns black. To prevent oxidation, each ncAA was prepared the day of the test for consistency. By this method, the oxidation of <SMALL>L</SMALL>-DOPA did not occur quickly enough to have an effect on the data. Most ncAA solutions were transparent once prepared with the exception of 3-nitro-<SMALL>L</SMALL>-tyrosine. The yellow-orange tint of 3-nitro-<SMALL>L</SMALL>-tyrosine solution was accounted for by measuring the fluorescence of 1mM 3-nitro-<SMALL>L</SMALL>-tyrosine in media and subtracting any possible background fluorescence from culture fluorescence grown in 1mM 3-nitro-<SMALL>L</SMALL>-tyrosine.
</P><P>
RFP fluorescence was measured using: excitation - 550 nm, emission - 675 nm
</P><P>GFP fluorescence was measured using: excitation - 480 nm, emission - 525 nm
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Preparation for the ncAA Kit Test</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">A picture showing culture with or without IPTG.  Without IPTG, there is no induction of GFP (left).  With IPTG, strong fluorescence is easily seen (right).</DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Results_and_Data">Results and Data</SPAN></H1><P>The full list of ncAAs and their tRNA synthetase/tRNA pairs can be found in the <A href="https://2014.igem.org/Team:Austin_Texas/kit#ncAA_Table" class="external text" rel="nofollow">tables at the bottom of this page</A>.  These table include a plethora of information relevant for performing the experiments with each amino acid as well as DNA sequencing data for each synthetase used.
</P><H2><SPAN class="mw-headline" id="Fidelity_of_Incorporation">Fidelity of Incorporation</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 4.</B>  Graph showing the level of GFP fluorescence relative to RFP fluorescence for each condition.  Each pStG plasmid is referred to based on the tRNA synthetase/tRNA pair present in the specific plasmid.  Each of these plasmids was then paired with either pFRY or pFRYC and grown in the presence or absence of a specific ncAA.  For example, the &quot;3-AminoY-FRYC&quot; and the &quot;3-AminoY-FRY&quot; samples both contain the 3-AminoY synthetase/tRNA pair and both samples were grown in the absence or presence of the ncAA &quot;3-AminoY&quot;.   Data are presented as the average of three independent cultures.  Error bars denote standard deviation.</DIV></DIV></DIV><P>We wanted to test whether the ncAA synthetase/tRNA pairs would incorporate anything besides their amino acid at the amber stop codon (UAG), and that they would in fact incorporate their specific ncAA if it was present. This is essentially the fidelity of each ncAA sythetase/tRNA pair.
</P><P>Fidelity was measured by comparing the production of GFP in cultures containing pStG/pFRY with or without ncAA. In the absence of a ncAA only RFP should be translated, as translation is expected to halt between RFP and GFP at the amber codon on the linker sequence in pFRY. Alternatively, if the corresponding ncAA for pStG is present or if the synthetase/tRNA pair has low fidelity and can misincorporate a different amino acid, translation should continue through the UAG. In this case, RFP and GFP should both be translated. We also tested pStG/pFRYC strains in (+/-) ncAA conditions as a control for what effect the ncAA has on the fluorescence or growth of the cells. These strains should express RFP and GFP in all conditions since pFRYC does not have an amber codon in the linker (<B>Figure 2</B>).  However, if the presence of ncAA affects cell growth or fluorescence activity, we will need these controls to determine the extent of the effect.
</P><P>To determine the change in GFP fluorescence when the ncAA was present, we first had to calculate how much GFP was expressed relative to the RFP, which would give an upper estimate of how much GFP could theoretically be expressed. We first divided both the GFP and RFP levels by the OD<SUB>600</SUB> of the culture in order to get the per cell fluorescence levels. We then normalized the GFP fluorescence for one culture to its RFP fluorescence so that we could compare the GFP fluorescence levels between cultures. The normalized GFP values were then compared between cultures grown in the presence of ncAA and cultures grown in the absence of ncAA, which would indicate how the level of GFP fluorescence changes when the ncAA is present.
</P><P>When these values were graphed (<B>Figure 4</B>), some synthetase/tRNA pairs such as 4-azido-<SMALL>L</SMALL>-phenylalanine (AzF), 3-nitro-<SMALL>L</SMALL>-tyrosine, 3-iodo-<SMALL>L</SMALL>-tyrosine, and <I>o</I>-(2-nitrobenzyl)-<SMALL>L</SMALL>-tyrosine (ONBY) resulted in higher GFP fluorescence in the presence of ncAA than in the absence of ncAA, which suggests that those synthetases only incorporated an amino acid if their specific amino acid was present, meaning that they have a high fidelity. However, the other synthetase/tRNA pairs (3-amino-<SMALL>L</SMALL>-tyrosine, <SMALL>L</SMALL>-DOPA, and 4-cyano-<SMALL>L</SMALL>-phenylalanine (CNF)) did not show a difference in GFP fluorescence normalized to RFP fluorescence dependent on the presence of ncAA, indicating that these pairs incorporated other amino acids at the amber codon when their specific ncAA was not present (and perhaps even when it was), and thus have a low fidelity.
</P><H2><SPAN class="mw-headline" id="Synthetase_Efficiency">Synthetase Efficiency</SPAN></H2><P>Another measure of quality for these ncAA synthetase/tRNA pairs is how efficiently they charge their ncAA, ultimately resulting in incorporation of the ncAA in our reporter protein. An inefficient synthetase/tRNA pair will yield incorporation of their ncAA only a fraction of the time, even when their ncAA is present. Our system can be used to measure this level of efficiency, though it does not indicate why a particular synthetase is efficient or inefficient. By comparing the level of GFP fluorescence to the normalized level of RFP fluorescence when the amino acid is present, we can see how efficient the synthetase is. In essence, if the normalized fluorescence of GFP relative to RFP is close to 100% (if the GFP is expressed roughly 100% of the time that RFP is expressed), then the synthetase is very efficient. On the other side, if say the normalized fluorescence of GFP relative to RFP is closer to 10%, then the synthetase would not be very efficient, because even when the ncAA was there, it only incorporated it at the amber codon about 10% of the time.
</P><P>For our results (<B>Figure 4</B>), two synthetase/tRNA pairs stood out as relatively inefficient: 3-nitro-<SMALL>L</SMALL>-tyrosine and ONBY. Both of these synthetase/tRNA pairs showed a significantly smaller normalized GFP to RFP fluorescence when the ncAA was present with the pFRY construct. While both synthetase/tRNA pairs show a significant increase in normalized GFP to RFP fluorescence when the amino acid was present compared to when it was absent, which indicates a high fidelity, the actual GFP fluorescence relative to the RFP fluorescence was only around 50% for 3-nitro-<SMALL>L</SMALL>-tyrosine and 20% for ONBY. These results suggest that these synthetase/tRNA pairs do not always efficiently incorporate their ncAA at an amber stop codon.  It is important to note that this could partly result from ncAA-related toxicity, and such an effect was repeatedly observed for ONBY.
</P><H2><SPAN class="mw-headline" id="Incorporation_Value">Incorporation Value</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 5.</B> Incorporation values for each synthetase/tRNA pair.  The dashed line equals an incorporation value of 1.  Values greater than 1 when using pFRY indicate high efficiency and fidelity of ncAA incorporation.  The GFP:RFP relative fluorescence values in the presence and absence of ncAA (see <B>Figure 4</B>) were used to calculate incorporation values.  Values were determined by dividing the GFP:RFP relative fluorescence in the presence of ncAA by the GFP:RFP relative fluorescence in the absence of ncAA. Data are presented as the average of three independent cultures. Error bars denote standard deviation.</DIV></DIV></DIV><P>We wanted a way to better summarize the output of the kit with a single measurement value. We decided to calculate an overall <B>incorporation value</B> as the GFP:RFP relative fluorescence value in the presence of the ncAA divided by the GFP:RFP relative fluorescence value when the ncAA is not included in the culture. Thus, the incorporation value is a single number that measures the relative efficiency and fidelity of a ncAA tRNA synthetase/tRNA pair.  A value of 1 indicates no significant dependence of incorporation on the presence of ncAA in the culture.  The higher the value, the higher the fidelity and/or efficiency of the synthetase/tRNA pair.  It is important to note that ncAA-related effects on growth and translation can also reduce the apparent incorporation value of a synthetase/tRNA pair below 1.
</P><P><B>The synthetase/tRNA pairs that showed the highest incorporation values were AzF, 3-nitrotyrosine, ONBY, and 3-iodotyrosine.</B> Among the ncAAs tested, ONBY consistently slowed the growth rate of the culture significantly (as seen by OD<SUB>600</SUB> readings) suggesting possible toxicity to the cell.  However, ncAA incorporation remained relatively high, resulting in a good incorporation value.  <B>Synthetase/tRNA pairs that showed the lowest levels of ncAA incorporation included 3-amino-<SMALL>L</SMALL>-tyrosine, <SMALL>L</SMALL>-DOPA, and CNF.</B></P><H1><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H1><P>Our ncAA measurement kit was able to successfully compare the fidelity and efficiency of seven different synthetase/tRNA pairs. We were able to confidently say that one synthetase/tRNA pair had a higher fidelity than another.  <B>We saw clear evidence that three of our tRNA synthetase/tRNA pairs seemed incapable of discriminating between their ncAA and the 20 canonical amino acids present in media: 3-amino-<SMALL>L</SMALL>-tyrosine, <SMALL>L</SMALL>-DOPA, and CNF.</B>  We were able to test triplicate cultures of seven different ncAAs and tyrosine (eight total synthetase/tRNA pairs) in one day, generating results that immediately told us which ncAAs worked best in our system both in terms of fidelity and efficiency of incorporation.  This required no advanced training or access to equipment besides a shaker and a fluorometer.
</P><P>While the measurements may not be perfectly accurate due to the nature of this test, the measurement kit is designed more around ease of use, cost-efficiency, and portability. The ncAA measurement kit can be used to quickly test a synthetase/tRNA pair's quality with minimal effort, as opposed to other more accurate but more intensive or expensive measures such as mass spectrometry. While mass spectrometry can give much more definitive information about a synthetase/tRNA pair, there are many different factors that could make it unreasonable for undergraduate research or high-throughput use. It is a technique that requires a fair amount of skill to do properly, and can be very costly as well.  Additionally, the sample needs to be purified from cell culture and prepped for mass spectrometry.
</P><P>For the ncAA measurement kit, all that is necessary is to request from us the kit plasmids (pFRYC and PFRY in separate strains of amberless <I>E. coli</I>), which we are immediately making available to anyone who requests the Expanded Genetic Code Measurement Kit.  Once you receive the frozen cultures (or plasmids), transform the synthetase/tRNA pair that you wish to test into the two different cell strains, and run the experiment. Thus, if you receive the measurement kit on Monday, you could have your synthetase/tRNA pairs characterized by Friday.  The kit can also be used to test how one or more synthetase/tRNA pairs work in different cells or under different conditions.  However, in this case, you may need to retransform the kit plasmids into the different strain.
</P><P>We are optimistic that this measurement kit can help speed up the process of developing a new synthetase/tRNA pairs, as well as characterizing large groups of synthetase/tRNA pairs, as we now have a way to quickly, cheaply, and easily do large scale characterizations. For example, we could now systematically test how changing the expression levels of a synthetase and its tRNA alters the efficiency and fidelity of ncAA incorporation.
</P><P>Due to the nature of the Expanded Genetic Code Measurement Kit, <I>in vivo</I> influences will affect certain data and results. This knowledge is very useful by providing a broader and biological perspective of cellular health. In general, the presence of ncAAs slowed the growth of amberless <I>E. coli</I>, sometimes quite significantly. This knowledge, as seen through the GFP:RFP ratio, can be very informative when assessing whether to use a ncAA tRNA synthetase/tRNA pair.
</P><H1><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H1><P>This year the UT Austin iGEM Team has chosen to take part in the <A href="https://2014.igem.org/Tracks/Measurement" class="external text" rel="nofollow">Measurement Track</A> of the iGEM competition.  We have presented here the Expanded Genetic Code Measurement Kit for the purpose of quickly characterizing novel ncAA tRNA synthetase/tRNA pairs, which meet the <A href="https://2014.igem.org/Team:Austin_Texas/medal_requirements" class="external text" rel="nofollow">medal criteria</A>.  Prior to this measurement kit there was not a widely available and easily accessible kit for assessing ncAA tRNA synthetase/tRNA pairs.  Although numerous labs have used mass spectrometry, such a technique is not always cheap not widely available.  Additionally, we have begun the process of standardization for ncAA synthetase/tRNA pairs by using our kit to simultaneously characterize seven different ncAA synthetase/tRNA pairs as well as a tyrosine RS. <B>This method of testing was designed to be easy-to-use, portable, quicker, and more available than other methods of characterization and analysis.   We are optimistic that this is a first step in opening the doorway to more iGEM teams and labs using these new chemical building blocks in their projects.</B></P><P>Documentation, including sequence information, of the tRNA synthetase/tRNA pairs used can be found <A href="https://2014.igem.org/Team:Austin_Texas/kit#ncAA_Synthetase_Table" class="external text" rel="nofollow">below</A>.
</P><P>Information regarding the ncAAs used can also be found <A href="https://2014.igem.org/Team:Austin_Texas/kit#ncAA_Table" class="external text" rel="nofollow">below</A>.
</P></DIV><H1><SPAN class="mw-headline" id="ncAA_Table">ncAA Table</SPAN></H1><TABLE class="wikitable"><TBODY><TR><TH> ncAA
</TH><TH> Molecular Structure
</TH><TH> Molecular Weight (g)
</TH><TH> Soluble in
</TH><TH> Other Notes
</TH></TR><TR><TD><SMALL>L</SMALL>-tyrosine (canonical)
</TD><TD> 181.19
</TD><TD> Soluble in water Heat to 70°C and vortex to dissolve Must stay warm to remain in solution
</TD><TD> Stock Concentration: 10mM Concentration in Culture: 1mM
</TD></TR><TR><TD> 3-amino-<SMALL>L</SMALL>-tyrosine (AminoY)
</TD><TD> 287.14
</TD><TD> Soluble in water Heat to 70°C and vortex to dissolve
</TD><TD> Stock Concentration: 10mM Concentration in Culture: 1mM
</TD></TR><TR><TD> 3-nitro-<SMALL>L</SMALL>-tyrosine (NitroY)
</TD><TD> 226.2
</TD><TD> Soluble in water Heat to 70°C and vortex to dissolve
</TD><TD> Stock Concentration: 10mM Concentration in Culture: 1mM
</TD></TR><TR><TD> 3-iodo-<SMALL>L</SMALL>-tyrosine (IodoY)
</TD><TD> 307.09
</TD><TD> Soluble in water Heat to 70°C and vortex to dissolve
</TD><TD> Stock Concentration: 10mM (.03g in 10mL H2O) Concentration in Culture: 1mM
</TD></TR><TR><TD> 3,4-dihydroxy-<SMALL>L</SMALL>-phenylalanine (<SMALL>L</SMALL>-DOPA)
</TD><TD> 197.1879
</TD><TD> Soluble in water Heat to 70°C and vortex to dissolve
</TD><TD> Stock Concentration: 10mM Concentration in Culture: 1mM
</TD></TR><TR><TD><I>o</I>-(2-nitrobenzyl)-<SMALL>L</SMALL>-tyrosine (ONBY)
</TD><TD> 316.308
</TD><TD> Soluble in 50% DMSO in Water Heat up to 70°C and add NaOH to dissolve
</TD><TD> Stock Concentration: 50mM Concentration in Culture: 1mM Light Sensitive
</TD></TR><TR><TD> 4-azido-<SMALL>L</SMALL>-phenylalanine (AzF)
</TD><TD> 206.204
</TD><TD> Soluble in 10% DMSO in Water Requires heating and overnight shaking to dissolve
</TD><TD> Stock Concentration: &lt;10mM (.02g in 10mL) Concentration in Culture: 1mM Light Sensitive
</TD></TR><TR><TD> 4-cyano-<SMALL>L</SMALL>-phenylalanine (CNF)
</TD><TD> 190.2
</TD><TD> Soluble in water Heat to 70°C and vortex to dissolve
</TD><TD> Stock Concentration: 10mM Concentration in Culture: 1mM
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="ncAA_Synthetase_Table">ncAA Synthetase Table</SPAN></H1><TABLE class="wikitable"><TBODY><TR><TH> Name of Synthetase
</TH><TH> Parent Synthetase
</TH><TH> Synthetase Mutations
</TH><TH> tRNA
</TH><TH> Synthetase Reference
</TH><TH> Sequence confirmed?
</TH><TH> Biobrick?
</TH><TH> Incorporation Value
</TH></TR><TR><TD> tyrosine RS
</TD><TD><I>M. jannaschii</I> tyrosine RS (Mj-TyrRS)
</TD><TD> Wild Type
</TD><TD> Mj-tRNA Tyr (CUA)
</TD><TD> Wang et al. 2001.
</TD><TD><A href="https://2014.igem.org/Team:Austin_Texas/kit/Sequences/TYR_RS" class="external text" rel="nofollow">Yes</A></TD><TD> No
</TD><TD> 1
</TD></TR><TR><TD> AminoY RS
</TD><TD> Mj-TyrRS
</TD><TD> Y32Q, L65E, F108G, Q109L, D158S, L162Y
</TD><TD> Mj-tRNA Tyr (CUA)
</TD><TD> Seyedsayamdost et al. 2007.
</TD><TD><A href="https://2014.igem.org/Team:Austin_Texas/kit/Sequences/3-aminoY_RS" class="external text" rel="nofollow">Yes</A></TD><TD> No
</TD><TD> 1
</TD></TR><TR><TD> NitroY RS
</TD><TD> Mj-TyrRS
</TD><TD> Y32R, H70L, Q155M, D158G, I159L, L162H
</TD><TD> Mj-tRNA Tyr (CUA)
</TD><TD> Neumann et al. 2008.
</TD><TD><A href="https://2014.igem.org/Team:Austin_Texas/kit/Sequences/3-nitroY_RS" class="external text" rel="nofollow">Yes</A></TD><TD> No
</TD><TD> 6
</TD></TR><TR><TD> IodoY RS
</TD><TD> Mj-TyrRS
</TD><TD> H70A, D158T, I159S, D286Y
</TD><TD> Mj-tRNA Tyr (CUA)
</TD><TD> Sakamoto et al. 2009.
</TD><TD><A href="https://2014.igem.org/Team:Austin_Texas/kit/Sequences/3-iodoY_RS" class="external text" rel="nofollow">Yes</A></TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1416001" class="external text" rel="nofollow">BBa_K1416001</A></TD><TD> 5
</TD></TR><TR><TD><SMALL>L</SMALL>-DOPA RS
</TD><TD> Mj-TyrRS
</TD><TD> Y32L, A67S, H70N, A167Q
</TD><TD> Mj-tRNA Tyr (CUA)
</TD><TD> Alfonta et al. 2003.
</TD><TD><A href="https://2014.igem.org/Team:Austin_Texas/kit/Sequences/L-DOPA_RS" class="external text" rel="nofollow">Yes</A></TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1178000" class="external text" rel="nofollow">BBa_K1178000</A>, 2013
</TD><TD> 1
</TD></TR><TR><TD> ONBY RS
</TD><TD> Mj-TyrRS
</TD><TD> Y32G, L65G, F108E, D158S, L162E
</TD><TD> Mj-tRNA Tyr (CUA)
</TD><TD> Deiters et al. 2006.
</TD><TD><A href="https://2014.igem.org/Team:Austin_Texas/kit/Sequences/ONBY_RS" class="external text" rel="nofollow">Yes</A></TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1416000" class="external text" rel="nofollow">BBa_K1416000</A></TD><TD> 6
</TD></TR><TR><TD> AzF RS
</TD><TD> Mj-TyrRS
</TD><TD> Y32T, E107N, D158P, I159L, L162Q?
</TD><TD> Mj-tRNA Tyr (CUA)
</TD><TD> Chin et al. 2002.
</TD><TD><A href="https://2014.igem.org/Team:Austin_Texas/kit/Sequences/AzF_RS" class="external text" rel="nofollow">Yes</A></TD><TD> No
</TD><TD> 9
</TD></TR><TR><TD> CNF RS
</TD><TD> Mj-TyrRS
</TD><TD> Y32L, L65V, F108W, Q109M, D158G, I159A
</TD><TD> Mj-tRNA Tyr (CUA)
</TD><TD> Schultz et al. 2006.
</TD><TD><A href="https://2014.igem.org/Team:Austin_Texas/kit/Sequences/CNF_RS" class="external text" rel="nofollow">Yes</A></TD><TD> No
</TD><TD> 1
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><UL><LI> Alfonta, L., Zhang, Z., Uryu, S., Loo, J. A., Schultz, P. G. (2003) Site-specific incorporation of a redox-active amino acid into proteins. <I>J. Am. Chem. Soc.</I><B>125</B>:14662-14663.
</LI><LI> Chin, J. W., Santoro, S. W., Martin, A. B., King, D. S., Wang, L., Schultz, P. G. (2002) Addition of p-azido-<SMALL>L</SMALL>-phenylalanine to the genetic code of <I>Escherichia coli</I>. <I>J. Am. Chem. Soc.</I><B>1294</B>:9026-9027.
</LI><LI> Deiters, A., Groff, D., Ryu, Y., Xie, J., Schultz, P. G. (2006) A genetically encoded photocaged tyrosine. <I>Angew. Chem. Int. Ed. Engl.</I><B>45</B>:2728-2731.
</LI><LI>Isaacs, F. J., Carr, P. A., Wang, H. H., Lajoie, M. J., Sterling, B., Kraal, L., Tolonen, A.C., Gianoulis, T.A., Goodman, D.B., Reppas, N.B., Emig, C.J., Bang, D., Hwang, S.J., Jewett, M. C., Jacobson, J. M., Church, G.M. (2011) Precise manipulation of chromosomes in vivo enables genome-wide codon replacement. <I>Science</I><B>333</B>:348-353
</LI><LI>Liu, C. C., Schultz, P. G. (2010) Adding new chemistries to the genetic code. <I>Annu. Rev. Biochem.</I><B>79</B>:413-44.
</LI><LI> Neumann, H., Hazen, J. L., Weinstein, J., Mehl, R. A., Chin, J. W. (2008) Genetically encoding protein oxidative damage. <I>J. Am. Chem. Soc.</I><B>130</B>: 4028-4033.
</LI><LI> Sakamoto, K., Murayama, K., Oki, K., Iraha, F., Kato-Murayama, M., Takahashi, M., Ohtake, K., Kobayashi, T., Kuramitsu, S., Shirouzu, M., Yokoyama, S. (2009) Genetic encoding of 3-iodo-<SMALL>L</SMALL>-tyrosine in <I>Escherichia coli</I> for single-wavelength anomalous dispersion phasing in protein crystallography. <I>Structure</I>. <B>17</B>: 335-344.
</LI><LI> Schultz, K. C., Supekova, L., Ryu, Y., Xie, J., Perera, R., Schultz, P. G. (2006) A genetically encoded infrared probe. <I>J. Am. Chem. Soc.</I><B>129</B>:13984-13985. 
</LI><LI> Seyedsayamdost, M. R., Xie, J., Chan, C. T. Y., Schultz, P. G., Stubbe, J. (2007) Site-specific insertion of 3-aminotyrosine into subunit α-2 of <I>E. coli</I> ribonucleotide reductase: Direct evidence for involvement of Y<SUB>730</SUB> and Y<SUB>731</SUB> in radical propagation. <I>J. Am. Chem. Soc.</I><B>129</B>: 15060–15071.
</LI><LI> Wang, L., Brock, A., Herberich, B., Schultz, P. G. (2001) Expanding the genetic code of <I>Escherichia coli</I>. <I>Science</I><B>292</B>: 498–500.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Austin_Texas/kit">http://2014.igem.org/Team:Austin_Texas/kit</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Austin_Texas/kit" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Austin_Texas/kit" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Austin_Texas/kit&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Austin_Texas/kit&amp;oldid=406315" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>