"
Page
Discussion
View source
History
teams
Log in
 
Team:Austin Texas/interlab study
From 2014.igem.org
UT Austin
HOME
PROJECTS
HUMAN PRACTICES
OFFICIAL REQUIREMENTS
Expanded Genetic Code
Measurement Kit
Photocaged
T7 RNA Polymerase
Interlab
Study
Caffeinated coli
@SXSW Create
Team
Parts
Safety
Medals
Interlab Measurement Study
Contents
1 Interlab Study
2 Devices Measured
3 Protocols
3.1 Sample Preparation
3.2 Measurement Protocol
4 Sequencing Data
5 Microplate Reader Data and Discussion
5.1 Discussion of Unexpected Results
Interlab Study
The interlab study is a new feature of iGEM this year.  As part of the measurement track, we participated in the interlab study as described on the Interlab Study page .  The purpose of this study was to measure the fluorescence of three different devices, two of which we constructed in lab from BioBrick parts.  The third device was simply a plasmid included in the 2014 parts kit.  Below, we detail our Interlab Study experience and discuss what our findings could mean.
Devices Measured
Figure 1. Triplicate liquid cultures of each construct.
1. The first device is BBa_I20260 , which is a composite BioBrick part that contains:
A promoter, BBa_J23101 , a downstream gene ( BBa_E0240 ), and a backbone ( pSB1C3 ).
The downstream gene is also a composite of a number of BioBrick parts including an RBS - B0032 , a GFP coding sequence - E0040 , and a transcriptional terminator - B0015 .
2. The second device was cloned in lab, as described in the interlab study procedure.
The device is composed of BBa_J23101 and BBa_E0240 , which are the same parts as in the first device.
However, the backbone of this device is different, pSB3K3 .
3. The third device was also cloned in lab, as described in the interlab study procedure.
The device is composed of BBa_J23115 and BBa_E0240 , which is the same downstream gene as in devices 1 and 2.  However, the promoter sequence of device 3 is different.
This device is in pSB1C3 , the same backbone as the first device.
Protocols
Sample Preparation
First, inoculate triplicate 10 mL cultures of LB with frozen stocks of the three devices (in TOP10 cells; I20260, J23101+E0240, J23115+E0240), a cell background control (TOP10 cells), and an LB only control in a 50 mL Erlenmeyer flask. Grow for 16-18 hours at 37°C, shaking at 300rpm.
The next morning, add 10µL of each overnight culture to 10 mL of LB with three replicates of each (15 total flasks) and grow 16-18 hours at 37°C, shaking at 300rpm.  Triplicate cultures were continued.
After 16-18 hours, add 80 µL of each culture to the wells of a clear-bottomed black 96-well plate.
Measurement Protocol
The fluorescence of each sample was measured using a 96-well plate as follows:
The 96-well plate was inserted into an Infinite 200 PRO Microplate Reader
Then, using the Tecan i-control the proper settings were selected and recorded.
The Settings were: Excitation at 480 nm (9 nm width) and Emission at 525 nm (20 nm width) with optimal gain.
Finally, the plate reader measured the fluorescence and the absorbance (OD600) for each well.
Sequencing Data
After sequencing our constructs, we aligned them to the reference part sequences. While constructs 1 and 2 were consistent with the references, we were surprised to see two point mutations in construct 3. This seemed highly unlikely to have occurred by chance. As such, we consulted the parts registry and found that the sequence analysis of the spring 2014 plate shows two point mutations consistent with our sequence reads.
Sequence of construct BBa_I20260
Sequence of construct BBa_J23101+BBa_E0240
Sequence of construct BBa_J23115+BBa_E0240
Compilation of all three sequences in .ab1 format with quality metadata
Microplate Reader Data and Discussion
Figure 2. Relative fluorescence data for the three parts measured.
Data was collected using an Infinite 200 PRO Microplate Reader and a 96-well black plate with 80 µl of culture per well.  As shown in Figure 2, we used triplicate cultures and took averages of each set to represent the measured relative fluorescence of the part. Our representation of the data subtracts the fluorescence of the media background from the GFP signal, and then is divided by the OD600 of the cell culture with the OD600 of the media subtracted, or (GFP−LBbkgd)/(OD600−OD600LBbkgd).
Despite the genetic similarities in the devices—all three contained the same coding sequence, two devices had the same backbone, and two devices had the same promoter sequence—there were stark differences in the amount of fluorescence produced by each of the devices.
Discussion of Unexpected Results
While we expected the device with a strong promoter and the highest copy number plasmid to have the highest fluorescence, we did not observe this to be the case. Instead, we saw that the two constructs with the same promoter and coding region in both the medium copy number plasmid pSB3K3 and the high copy number plasmid pSB1C3 actually yielded the strongest fluorescence signal in the medium copy number plasmid. It is possible that the high copy number plasmid pSB1C3 had a negative effect on overall fluorescence—perhaps it became toxic or slowed cell growth.  However, it is also possible that we may have swapped cultures or mislabeled an initial eppendorf or culture tube.
If this is not the case, then we would consider the possibility that the medium copy plasmid pSB3K3, while lower copy number than pSB1C3, may be a better expression platform for this protein. One can also compare the relative qualitative fluorescence of the cultures visually (Figure 1) by looking at the top two rows of flasks: the top row being BBa_I20260 in pSB1C3 and the middle row being the same construct (BBa_J23101+E0240) in pSB3K3.
In comparing constructs #1 and #3 (with the same coding sequence and plasmid backbone but different promoters), it should be noted that the sequence of promoter J23115 (device #3) differed by two bases from the reference sequence in the registry. While the sequence we used is consistent with the Spring 2014 Kit Distribution sequence analysis performed by the BioBricks Foundation, it is likely that this mutated promoter may have a different activity than the reference sequence. We observed that J23101 (device #1) shows a several-fold stronger signal than our mutated promoter J23115. While the measured relative fluorescence data of the reference sequences generally agree with our data, the mutated J23115 shows slightly less activity compared to the reference data. The relative qualitative fluorescence of the cultures can be compared visually (Figure 1) by looking at the top and bottom rows: the top row using promoter J23101 and the bottom row using mutated promoter J23115, both in pSB1C3.
Retrieved from " http://2014.igem.org/Team:Austin_Texas/interlab_study "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
