"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg bioware/Project/invivo
From 2009.igem.org
Artisteer
Home
The Team
Overview
Portraits
The Project
Summary
Subprojects
Highlights
Human Practice
Ethics
Safety
Notebook
Parts
Basic Parts
Composite Parts
Collaboration
Modeling
 in vivo assay
Fluorescence microscope image of RV308 cells, electroporated with fluorescein labeled oligonucleotides
Introduction
In vivo use of programmable restriction enzymes can provide the opportunity of genome engineering. Here we develop and test strategies for the application of our programmable restriction endonuclease.
In our strategy we cotransformed Fok_i and Fok_a constructs fused to binding domains into a certain bacteria strain leading to the basal expression of the Fok heterodimers. Transformation of single-stranded bacteriophage M13 DNA hybridized with modified oligonucleotides enables the Fok heterodimers to bind the DNA. Inside the cells the phage DNA should be cut by the dimerized and thereby activated Fok_a should result in a decreased progeny of phages within the bacterial culture in comparison to control assays. Two ways of detection were envisioned. A qPCR should reveal successful cutting of the phage DNA by the lack of amplified phage DNA. In addition an adjacent phage titer determination via a blue white screening ressembling phage assay by plating the cells to IPTG/Xgal plates should lead to a decreased number of plaques in comparison to the control assays.
in vivo scheme
Methods
In detail two heterodimeric Fok constructs His-Dig-Split-Fok_a (BBa_K243036) and His-FluA-Split-Fok_i (BBa_K243010) are encoded by two different plasmids, plasmid pJS419 with a chloramphenicol and pEx with an ampicillin resistance. This enables a positive selection of the bacteria strain XL1 blue cotransformed with both plasmids. Subsequent to the preparation of electrocompetent cotransformed XL1 blue, the M13 ssDNA hybridized with the modified oligonucleotides were inserted into these bacteria by electroporation at 1.7 kV. The infected cells were incubated at 37°C for 0.5 h. At different dilutions the phages were mixed with 180μl ER2738 cells with an OD600 of about 0.5. Then they were mixed with 3 ml of 42°C warm top agar and plated on IPTG/X-Gal plates.
XL1 Blue as well as ER2738 harbour a F-plasmid coding for the omega-fragment of beta-galactosidase. The M13 phages in turn encodes the lacking alpha-fragment. Consequently, alpha-complementation can be detected via a screening for blue plaques. After incubation overnight at 37°C the infected cells were inspected for blue plaques formation because of the enzymatic cleavage of X-Gal and production of the blue dye 5,5'-dibromine-4,4'-dichloride-indigo and plaques can be seen where the phages lysed the XL1 blue cells. In comparison with plated bacteria only transformed with M13 ssDNA, M13 ssDNA with other oligos not modified or infected with entire phages directly the number of pfu (plaque forming units) should be decreased.
Another possible approach is a qPCR after the insertion and incubation of the phage DNA hybridized with a modified oligo or pure ssDNA into the XL1 blue. This quantitative real time polymerase chain reaction allows to amplify and simultaneously quantify a certain sequence of the phage DNA. The additional feature of the qPCR is the quantification of the accumulating DNA happens in real time after each amplification step. The phage DNA would be extracted by boiling preparation, a method faster than conventional alkaline lysis minipreps leaving the DNA somewhat dirtier, but adequate for qPCR. After the qPCR with appropriate primers, the DNA can be quantified with a fluorescent dye, intercalating in the double-stranded DNA. The expected result would be a lower amount of double-stranded phage DNA for our XL1 blue transformed with DNA and oligo in comparison to the control assay.
Results and Discussion
Our attempt to plate cotransformated XL1 Blue on IPTG/X-Gal plates showed several blue plaques for the M13 DNA without hybridized oligonucleotides. In comparison the M13 DNA hybridized with the modified oligonucleotide showed no blue plaques, demonstrating that the binding of the basally expressed heterodimeric Fok constructs triggered by the labeled guide oligonucleotides and the following cleavage of the phage DNA was successful and the progeny of the phages and thus alpha/complementation decreased.
M13 DNA without (left) and with (right) hybridized modified oligonucleotide, no plaques formation in presence of modified oligonucleotide
The qPCR assay is still in progress. Expected results have still to be verified.
 
contact:  freigem09@googlemail.com
Web Template created with Artisteer.
Retrieved from " http://2009.igem.org/Team:Freiburg_bioware/Project/invivo "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
