<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_bioware_Notebook_May"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Freiburg_bioware/Notebook/May">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_bioware/Notebook/May&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_bioware/Notebook/May&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_bioware/Notebook/May&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_bioware/Notebook/May" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg bioware/Notebook/May</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P>&lt;!DOCTYPE html PUBLIC &quot;-//W3C//DTD XHTML 1.0 Transitional//EN&quot; &quot;<A href="http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd" class="external free" rel="nofollow">http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd</A>&quot;&gt;<TITLE>FREiGEM</TITLE></P><DIV id="art-main"><DIV class="art-Sheet"><DIV class="art-Sheet-body"><DIV class="art-nav"><UL class="art-menu"><LI><A class="active" href="https://2009.igem.org/Team:Freiburg_bioware"><SPAN class="t">Home</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Team"><SPAN class="t">The Team</SPAN></A></LI><LI><A href="a href=" https:="">Overview</A></LI><LI><A href="a href=" https:="">Photos</A></LI></UL><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Project"><SPAN class="t">The
Project</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Project#Summary">Summary</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Project#Subprojects">Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Project#Highlights">Highlights</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Human_Practice"><SPAN class="t">Human
Practice</SPAN></A></LI><LI><A href="#">D'oh! Something's missing here.</A></LI><LI><A href="#">1</A></LI><LI><A href="#">2</A></LI><LI><A href="#">3</A></LI><LI><A href="#">4</A></LI><LI><A href="#">5</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Notebook"><SPAN class="t">Notebook</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Collaboration"><SPAN class="t">Collaboration</SPAN></A></LI></DIV><DIV class="art-contentLayout"><DIV class="art-content"><DIV class="art-Post"><DIV class="art-Post-body"><DIV class="art-Post-inner"><DIV class="art-PostMetadataHeader"><H2 style="border-bottom: none;" class="art-PostHeaderIcon-wrapper">  May
</H2></DIV><DIV class="art-PostContent"><H3>02.05 13-20.30o'clock, Rüdiger</H3>
datamining and investigation about DNA und FOKI. Paper Miller07,
Wah1998.
3d analysis and pictures taken from pymol.
Alignment of 2BamHI and 2FOKI to get DNA in proper position. However it
is still not perfekt, 
but gives a reasonable insight into distances we need for the linker
and where we can position the oligo
<DIV style="text-align: left;"><TABLE style="text-align: left; width: 633px; height: 76px;" border="0" cellpadding="0" cellspacing="2"><TBODY><TR><TD>
Figure 1: this is the truncated catalytically active fokI(dimer) bound
to DNA. Pink residues are the catalytic ones.</TD><TD>
Figure 2: Like this could be the completed construct. You see the
FOKIdomains as in fig.1 combined with 2streptavidin proteins that bind
to biotin.</TD></TR><TR><TD>
Scheme 3: cut through from the side to make dna accessibility visible.
If we add a linker from below(in this picture it is the top) the
indicated base would be a good place.</TD><TD>
Scheme 4: cut through from below to make dna accessibility visible. If
we add a linker from below the indicated base would be a good place.</TD></TR></TBODY></TABLE></DIV><H3>05th may, 11-13o'clock Rüdiger</H3>
Sponsoringmeeting with Timo, Anika, Rüdiger
to lazy to translate already taken notes....
Uni-interne zeitschriften zuspammen
-nachwuchsförderung
-lernen in der praxis
-kooperatives lernen
-gruppenmanagement
firmenzeitschriften finden und zuspammen
Sponsoren tour
fernsehen??!
radio
Mentorenprogramm
to do:
argumentenliste (schleimig)
-ziele
-selbstständig arbeiten
-eigenes projekt von planung bis zur vollendung, sowas gibt es sonst
nirgends. Einblick in zukunft unseres 
Forscherlebens.
neues universelles werkzeug für synthetische biologen: soll
laboralltag revolutionieren.
2Projekte: 
Entwicklung eines universellen Schneidewerkzeuges: Man könnte
es mit einem Schraubenzieher vergleichen, 
auf den man alle arten von Bits aufsetzen kann. Jede Art von Schraube
ist mit einem einzigen ,statt, 
wie bisher mit hunderten verschiedenen Schraubenziehern herausdrehbar.
Es soll neue opensource Software für iGEM entwickelt werden,
das die komplete Arbeit mit den BiobrickTM 
Standard Parts in einer einzigen Anwendung zusammenführt. Dies
beinhaltet das Suchen und runterladen der 
Sequenzen, sowie die Zusammenstellung und Bearbeitung, bis hin zur
Veröffentlichung der Sequenzen in Form eines
neuen Biobrick Parts.
Man kann sich das wie Lego basteln vorstellen: Beispielsweise man
möchte ein Haus aus Lego bauen. 
Dazu sucht man sich alle benötigten Steine zusammen. Aber
bevor man anfängt wild drauf los zu basteln, 
zieht man das Programm zu Rate und spielt alle Varianten am Computer
vorher durch. Das Programm zeigt einem 
dabei mittels verständlicher Grafiken, was machbar und
sinnvoll ist und druckt am Ende eine fertige Bauanleitung aus.
Und da das Programm der Open-source Idee verpflichtet ist, kommen alle
Anleitung für alle zur freien
Verfügung ins WWW.
-programm<H3> 05th may, 17o'clock Rüdiger</H3>
17o'clock, Meeting the Captain(Igloi) with Dieter and Rüdiger.
Captain tells us about how to link PNA with Protein via Ni/Co and His
tag.
same could be applied to link ologo and protein.
dna needs to hybridize with around 30 bases oligo to properly build up
double helix.
<H3>07th may, 14-18o'clock Rüdiger</H3>
Dicussing FokI Sequence and further strategy with Laura, Gerrit Dieter
and Kristian.
Companys to buy Genes from:
-Geneart
-Mr.gene(low budget version without support)
-itelichon or sth like that
-DNA 2.0 (expensive but can demand adequate features of vector) 
Glenn research Bioscience offers modified nucleotides for our oligos
Vektor we should create/order now:
*must not contain IGEM restriction sites
*1FokI, active cleavage sitewe need to check literature about that. 
Possible experiment with original cys containg fokI: 
add iodoacetamide to it and look if it gets acetylated and if it still
cleaves.
*1FokI without cleavage Aminoacids
*replacing cys by ser (first 
Possible Tags (will be investigated later in more detail):
*Fluoreszin &amp; Digoxigenin ? coupled to oligo
*Lipocalin can bind to these specufically
*does the Captain have Ni/Co for Oligo synthesis?
*or maybe Bannwarth ? ?
Program for Vector handling: Gentle<H3>08th may, 17.15-18.30o'clock Caro</H3>
Dicussing RAG1/RAG2 DNA cleavage mechanism with Caro, Manu, Gerrit,
Hannes, Kristian, Cristoph.
Question discussed:
* Which kind and how many domains have RAG1/2?
* What are the cleavage mechanims and the interaction of theese domains?
*What is the essential part of theese proteins and what can be
eliminated?<B>Next meeting for RAG1/RAG2 and Recommbineering group: Monday,
11.may, 17.30 at kristians lab</B><H3>08th may, 13:30-19:30o'clock Laura</H3>
Planning the design for two vectors including one different
FokI-heterodimer each with Rüdiger, Dieter and Laura
Procedures for both vectors:
*extract coding region of Fok from the restriction-modification genes
of the chromosomal DNA of Flavobacterium okeanokoites
[http://www.ncbi.nlm.nih.gov/nuccore/148723?ordinalpos=1&amp;itool=EntrezSystem2.PEntrez.Sequence.Sequence_ResultsPanel.Sequence_RVDocSum
F.okeanokoites fokIR and fokIM genes]
*delete the first 1158 nucleotides/386 aa (recognition domain) 
*switch Cystein 541/463-465 to Ser (TGT-&gt;TCT)
Modifications of the single vectors to introduce heterodimeric
modifications according to
[[Media:Miller_J,_Rebar_E_Nature_biotech_2007_25_778.pdf‎]]:
Modifications of the first vector (catalytic active heterodimer) 
-heterodimeric aminio acids
*switch Glutamate 490/310-312 to Lysin (GAA-&gt;AAA)
*switch isoleucin 538/454-456 to Lysin (ATC-&gt;AAA)
Modifications of the second vector (catalytic inactive heterodimer) 
-heterodimeric amino acids
*switch Glutamin 486/298-300 to Glutamate (CAA-&gt;GAA)
*switch Isoleucin 499/337-339 to Leucin (ATC-&gt;CTG)
-catalytic amino acids
*switch Aspartate 450/190-192 to Alanin (GAC-&gt;GCG)
*switch Aspartate 467/243-245 to Alanin (GAT-&gt;GCG)
Annotations:
*The notation e.g. for Cystein 541/463-465 means the amino acid 541 in
literature which correspond to the codons 463-465 in our vector. 
*For exchanging the amino acids we used the Codon usage table in E.coli
from Hénaut and Danchin.
[http://www.faculty.ucr.edu/~mmaduro/codonusage/codontable.htm E.coli
Codon Usage]<B>Next meeting: Monday, 11th may, 16pm at kristians lab</B><H3>11th may, 17.30-19.30o'clock Max</H3>
Attendees: Hannes, Caro, Manuel, Gerrit, Christoph, Max<B>RAG:</B> * <B>Discussion about different domains.</B>
**If in RAG1 R855 and K856 are exchanged there is no hairpin but still
nicking.
* RAG1: - cleavage active sites: D600,D708,E962 provides metalbinding
function and ion -induced hydroxy-cleavage.
*RAG2: -The core is characterised by a six repeats of a kelch motif(50
residues each) 
- each repead forms a four-stranded twisted antiparallel beta-sheet(six
bladed propeller). This structure takes part in protein protein binding.
- T490 phosphorylation site
- aa 352-410 acidic portion
-aa 420-480 is a Cys-His rich reagion similar to homodomains of zink
fingers 
-R73A, K118A/K119A Hairpin-opening 
- RAG2 binds RAG1 and DNA 
* <B>What can be taken away from the protein so that it still
works?</B>
Large parts can be deleted without loosing recombination activity. The
full length of mouse RAG1 is 1040 amino acids. Aminoacids 384-1008 are
required.
Mouse RAG2 has about 527 aa. Only the first 383(387)are required. In
presence of Mn2+ RAG1/2 efficiently cuts an RSS(recombination signal
sequence) in a DNA fragment to yield blunt 5´ phosphorylated
signal
ends and hairpin coding ends that retain the full coding sequence. *
RAG may be quite difficult to be made suitable for our purpose.
* We decided to look for another suiting enzyme just in case RAG turns
out as not realizable.
<B> Next meeting: Thursday, 05/14/09, 12.30 @ Zoo-Café</B><H3>11th may, 16:00-20:30o'clock Laura</H3>
Planning the design for two vectors with Rüdiger, Dieter and
Laura
Execution of the planned modifications with the program Gentle
To avoid restriction sites of the restriction enzymes used by iGEM:
*switch Isoleucin 109-111 to Isoleucin (ATT-&gt;ATC)
*switch Alanine 520-522 to Alanine (GCC-&gt;GCG)
sequences of the completely edited active FOKI heterodimer:
-amino
acid sequence:
KSELEEKKSELRHKLKYVPHEYIELIEIARNSTQDRILEMKVMEFFMKVYGYRGKHLGGSRKPDGAIYTVGSPIDYGVIVDTKAYSGGYNLPIGQADEMQRYVKENQTRNKHINPNEWWKVYPSSVTEFKFLFVSGHFKGNYKAQLTRLNHKTNSNGAVLSVEELLIGGEMIKAGTLTLEEVRRKFNNGEINF
-nucleotide sequence:
AAAAGTGAACTGGAGGAGAAGAAATCTGAACTTCGTCATAAATTGAAATATGTGCCTCATGAATATATTGAATTAATTGAAATTGCCAGAAATTCCACTCAGGATAGAATCCTTGAAATGAAGGTAATGGAATTTTTTATGAAAGTTTATGGATATAGAGGTAAACATTTGGGTGGATCAAGGAAACCGGACGGAGCAATTTATACTGTCGGATCTCCTATTGATTACGGTGTGATCGTGGATACTAAAGCTTATAGCGGAGGTTATAATCTGCCAATTGGCCAAGCAGATGAAATGCAACGATATGTCAAAGAAAATCAAACACGAAACAAACATATCAACCCTAATGAATGGTGGAAAGTCTATCCATCTTCTGTAACGGAATTTAAGTTTTTATTTGTGAGTGGTCACTTTAAAGGAAACTACAAAGCTCAGCTTACACGATTAAATCATAAAACTAATTCTAATGGAGCTGTTCTTAGTGTAGAAGAGCTTTTAATTGGTGGAGAAATGATTAAAGCGGGCACATTAACCTTAGAGGAAGTGAGACGGAAATTTAATAACGGCGAGATAAACTTT
sequences ot the completely edited inactive FOKI heterodimer:
-amino acid sequence:
KSELEEKKSELRHKLKYVPHEYIELIEIARNSTQDRILEMKVMEFFMKVYGYRGKHLGGSRKPDGAIYTVGSPIDYGVIVDTKAYSGGYNLPIGQADEMQRYVKENQTRNKHINPNEWWKVYPSSVTEFKFLFVSGHFKGNYKAQLTRLNHKTNSNGAVLSVEELLIGGEMIKAGTLTLEEVRRKFNNGEINF
-nucleotide sequence:
AAAAGTGAACTGGAGGAGAAGAAATCTGAACTTCGTCATAAATTGAAATATGTGCCTCATGAATATATTGAATTAATTGAAATTGCCAGAAATTCCACTCAGGATAGAATCCTTGAAATGAAGGTAATGGAATTTTTTATGAAAGTTTATGGATATAGAGGTAAACATTTGGGTGGATCAAGGAAACCGGCGGGAGCAATTTATACTGTCGGATCTCCTATTGATTACGGTGTGATCGTGGCGACTAAAGCTTATAGCGGAGGTTATAATCTGCCAATTGGCCAAGCAGATGAAATGGAACGATATGTCGAAGAAAATCAAACACGAAACAAACATCTGAACCCTAATGAATGGTGGAAAGTCTATCCATCTTCTGTAACGGAATTTAAGTTTTTATTTGTGAGTGGTCACTTTAAAGGAAACTACAAAGCTCAGCTTACACGATTAAATCATATCACTAATTCTAATGGAGCTGTTCTTAGTGTAGAAGAGCTTTTAATTGGTGGAGAAATGATTAAAGCGGGCACATTAACCTTAGAGGAAGTGAGACGGAAATTTAATAACGGCGAGATAAACTTT
Introduction of prefix and suffix according to the Fusion Protein
(Freiburg)Biobrick assembly standard
[[Image:Fusion part.JPG|none|thumb|400x400px]]

Ask for an offer for our two vectors from &quot;Biomatik Corporation&quot; <H3>14th May 12:30-15:00 o'clock, Hannes</H3>
Attendees: Hannes, Caro, Gerrit<B>RAG</B>
- preparation of RAG-presentation for the meeting on friday 
- discussion of new ideas (AloI)<H3>15th may 2009, 13:00- 16:00 o'clock Anika</H3>
Attendees: Max, Sascha, Timo, Hannes, Dieter, Rüdiger, Laura,
Julia, Caro, Paul, Gerrit, Manuel, Kristian, Anika

Presentation of different subprojects
- <B>FOKI</B>
genes are ordered, each vector costs approx. 344 USD, we ordered two
vectors for the beginning 
- <B>homepage</B>
basis is written in html so that everybody can contribute to complete
the page, everything in english, except the 'Vereinssatzung'in german
authors of the subitems: 
Team: Caro
Project: Rüdiger
Publicity: Manu
iGEM Contest: Anika, Timo
iGEM e.V.: Laura
-&gt; send everything to Sascha
- <B>software</B>
still looking for ideas, Java seems to be a good idea, program should
run without an internt connnection, a Plug-In based, we will need some
biologist testing this program
- <B>sponsoring/marketing</B>
discussion about different possibilities to find donators
- <B>RAG1+2</B>
group presented their plans, for the moment it seems that FOKI has the
better prospect<B>next meeting: Wednesday, 20th May 2009</B><H3>20th may 2009, 11:00 o'clock ct. Anika</H3>
Attendees: Imi, Laura, Caro, Christoph, Rüdiger, Sascha,
Kristian, Anika
Topics: <B>1. Vereinsbelangen</B>
Problem: If we are going to be an 'allgemeinnütziger Verein'
it is
prohibited to make advertising for our sponsors. On our website as well
as on our T- shirts.
Caro and Laura will go to the tax and legal advice of the u-Asta.<B>2. Sponsoring</B>
We are going to start posting the letters to the companies beginning of
next week. There will be an extra meeting for the sponsoring group
which is not arranged yet.<B>3. Plasmidconstruction</B>
The ordered things arrivedwhich means that the laboratory work can
start. The FokI group is looking for people who have time to work in
the lab for cloning.
One more thing: the website is going to put online on Sunday this week.
<H3>25th may 2009, 10:00- 17:00 o'clock Laura and Caro</H3>
Attendees: Max, Sascha, Hannes, Gerrit, Anika, Christoph, Imi, Sonja,
Caro, Laura

Introduction into the lab work:
Hier die Einführung auf deutsch (muss ja nicht ins richtige
Wiki übernommen werden):
Unser Laborraum: 
*UV-Gerät auf Kühlschrank zum Reinigen der Pipetten
bei Handhabung mit elektrokompetenten Zellen
* fertig gegossene SDS-Gele in Kühlschrank hinter unserer
bench (evtl. selber gießen) 
* Kopierkarte auf Annas Platz hinter unserem Laborplatz
*Zellkulturschrank (neben Zellkultur): Zellkulturflaschen zum
Sterilfiltrieren (für Puffer etc.) und große Kolben
Messraum (EKTA-Raum?): Viva spins (Zentrifugeneinsätze bei
Proteinaufzentrifugation) im oberen Schrank; Spitzen (verschiedene
Größen) im unteren Schrank
PCR/Gelraum (von der Tür zw. PRC-Raum und unserem Labor
gesehen): 
* Ethidiumbromidverschmutzter Bereich auf der bench 
* Ethidiumbromid und normaler Abfall auf der bench getrennt; auch beim
Ausgang an der zweiten Tür größere
Abfalltonnen für EthBr (Achtung:
keine Handschuhe!) und Glasbruchabfall
* Proteinpuffer und Comassiefärbelösungen in Regal
über Tonnen
* eigenes Fach im -80°C Kühlschrank oben; Fach
für Klone und Proteine im unteren Teil 3. Spalte, 1. Zeile 
* Waage hinten rechts in der Ecke
* nichtexplosive und nichtgiftige Chemikalien im Schrank auf der
rechten Seite
* brennbare und explosive Substanzen im gelben Schrank an hinterer Wand
* häufig benutzte Chemikalien im Regal rechts daneben
* im Abzug bestimmte Abfälle wie für Silber etc. 
* gefährliche Chemikalien in Metallschränken unter
dem Abzug
* für pH-Einstellungen pH-Meter unter Abzug (in Liste daneben
mit
Datum, Name und Kalibrierwert eintragen; Elektrode wird in 3M
KCl-Lösung gelagert) 
* Zentrifugen in der Mitte des Raumes;
Zentrifugengefäße im Schrank darunter
* Laufpuffer für Gele im Schrank oberhalb der bench
Verschiedene Gänge von Kristians Labor aus gesehen: 
Links: 
* Kopierraum
Geradeaus: 
* noch links vor Laboreingang: 37°C Raum (großen
Schüttler immer wieder anstellen!) 
*Platten zum Inkubieren ins Regal; Blotting-Papier und Schneidemaschine
vor 37°C Raum
* im Labor: im Zentrifugenraum Eismaschine und große
Zentrifugen;
UV-Lichtraum zum Geldokumentieren und -schneiden weiter hinten,
UV-Lichtkamera rechts von Raum nicht benutzen
Rechts: 
*Schrank rechts mit Handschuhen, Eppis, Pipettenspitze (bei Gebrauch
auf Liste bei Müller AG Strich) 
* Geräteraum links mit temperierbaren Shakern (Alternative zu
37°C
Raum; bei Gebrauch mit post-it über Dauer der Benutzung
informieren) 
* Biorad-Geldock auch zum Gel fotografieren (aber EthBr frei) 
Spülküche (noch links vor Laboreingang): 
* alle mögliche Glaswaren (Messbecher, Glaskolben etc) 
* beim Platten Gießen Flasche mit Wasserbad beschriften;
Flasche wird
dann ins Wasserbad gestellt und kann nach zwei Stunden abgeholt werden
*Gießraum: UV-Licht vorm Betreten ausschalten!; Petrischalen
vorhanden;
an der Tür verbrauchte Materialien in die Liste eintragen;
nach
Verlassen UV-Licht wieder anschalten
*Spezialbecken für Restagar neben Waschbecken im hinteren
Bereich (mit Heißwasser auspülen und
wegschütten) 
* daneben Feinwaagen 
* Wagen für Spitzen, Eppis etc. und Wagen für
Flüssigkeitsabfall zum Totautoklavieren
* Millipore Wasser-Anlage im mittleren Bereich: Process, ca. 1 min
warten bis über mind. 18,1 MOhm, mit oberem Drehschalter
Wasser
einlassen
Goßes Labor: 
* SDS-Gel Scanner; oberhalb befinden sich multipler SDS
Plattengießer (mighty small), Spritzfilter und Tris-Puffer
*Eppis, Spitzen, PCR tubes und DNA Präparations Kits etc. in
großem Schrank vor Waschbecken; Falcons obendrauf
* Rührfische nebenWaschbecken, Zylinder unter Waschbecken
*neben Waschbecken auch Gefäßsammelstelle
für Gefäße und
Flüssigkeitsabfall (falls zu voll oder
Gefäße zu groß, selbst Sachen in
die Spülküche bringen) 
* zum Sequenzieren an Christina wenden (über GATC Service;
Eppis iun organgenen Boxen, im EG in den Briefkasten) 
* im Kühlschrank gekühlte Feststoffe wie Antibiotika
*Autoklaviertape, Alufolie etc. auf dem Kühlschrank
Further Topics: 
* First steps: Preparation of mediums, agarplates,....
* Work-Plan of the week:
Tuesday: Competent cells (Sascha, Christophe)
Wednesday: Inoculation (Imi)
Thursday: VectorPrep (Laura, Caro) and Transformation (Hannes)
Friday:
Next Week: Individual workings(Proteinexpression) of the Argonaute team
and all people that want to help
--&gt; For this please write a message to christoph
Next Meeting: Monday 8th june, 11:00 in the Seminar room
<H3>26th may 2009, 09:00- 15:00 o'clock; Christoph</H3>
Attendees: Max, Gerrit, Christoph, Sarah

We made about 200 aliquods of competent cells from the strands RV 308
and X blue (not so sure about which strands we exactly took... I will
lock it up in the labbock!). Those can be used by all groups.
<H3>27th may 2009, 12:00- 15:00 o'clock; Hannes</H3>
Attendees: Imi, Hannes<B>Transformation</B> was done with XL1 blue and the two
vectors (exact
name?--&gt; pET SUMO/ pET28a). We made a control plate with XL1
(without the plasmid) as well. Plates are grown over night at
37°C.
Inoculation in LB+antibiotic has to be done tomorrow (Thursday)(growth
not more than 16h) and the plasmid prep on Friday.
<H3>28th may 2009, 11:00- 12:30 o'clock; Caro, Laura</H3>
*check of the three plates: Control is negative, plates with vectors
are grown very well.
*discussion of the expression vector of Raik Grünberg. We
emailed further questions concerning the vector.<H3>28th may 2009, 18:30 - 19:00 o'clock; Hannes</H3>
*Inoculation of
2x3ml LB+Kan with a single colony of the two transformed strains
(XL1+Tt and XL1+Aa). 12-16 hours growth over night.<H3>29th may 2009, 10:30- 12:30 o'clock; Max</H3>
Attendees: Caro, Gerrit, Hannes, Max
Plasmid purification of Tt and Aa with QIAprep Spin Miniprep Kit </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="art-Footer"><DIV class="art-Footer-inner"><DIV class="art-Footer-text"><P>contact: <A href="mailto:freigem09@googlemail.com">freigem09@googlemail.com</A></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Freiburg_bioware/Notebook/May">http://2009.igem.org/Team:Freiburg_bioware/Notebook/May</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_bioware/Notebook/May" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_bioware/Notebook/May" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_bioware/Notebook/May&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_bioware/Notebook/May&amp;oldid=117885" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>