<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_bioware_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Freiburg_bioware/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_bioware/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_bioware/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg bioware/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>FREiGEM</TITLE></P><DIV id="art-main"><DIV class="art-Sheet"><DIV class="art-Sheet-body"><DIV class="art-nav"><UL class="art-menu"><LI><A href="https://2009.igem.org/Team:Freiburg_bioware"><SPAN class="t">Home</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Team"><SPAN class="t">The Team</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Team/Portraits">Portraits</A></LI></UL><LI><A class="active" href="https://2009.igem.org/Team:Freiburg_bioware/Project"><SPAN class="t">The
Project</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Project#Summary">Summary</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Project#Highlights">Highlights</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Human_Practice"><SPAN class="t">Human
Practice</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Human_Practice/Ethics">Ethics</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Human_Practice/Safety">Safety</A></LI><LI><A href="#"><SPAN class="t">Notebook</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/cloning1"><SPAN class="t">Parts</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/cloning1">Basic
Parts</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/cloning">Composite
Parts</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Collaboration"><SPAN class="t">Collaboration</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Modeling"><SPAN class="t">Modeling</SPAN></A></LI></DIV><DIV class="art-contentLayout"><DIV class="art-content"><DIV class="art-Post"><DIV class="art-Post-body"><DIV class="art-Post-inner"><DIV class="art-PostMetadataHeader"><H2 class="art-PostHeaderIcon-wrapper">  Project Report</H2></DIV><DIV class="art-PostContent"><DIV style="text-align: left;"><TABLE style="text-align: left; margin-left: 0px; margin-right: 0px; width: 202px; height: 304px;" border="1" cellpadding="2" cellspacing="0"><TBODY bgcolor="#cddbe5"><TR><TD> <SPAN style="font-weight: bold; text-decoration: underline;">Contents</SPAN><OL><LI><A href="#Summary">Summary</A></LI><LI><A href="#Highlights">Highlights</A></LI><OL style="list-style-type: upper-roman;"><LI><A href="#Modelling">Modelling</A></LI><LI><A href="#3D-Modelling">3D-Modelling</A></LI><LI><A href="#Fok_a">In vitro assays</A></LI><LI><A href="#Fok_Monomer">Fok-Monomer</A></LI><LI><A href="#Purification">Protein
Expression and Purification</A></LI><LI><A href="#In_vivo_Expression">In vivo
assays</A></LI><LI><A href="#AGO">AGO</A></LI><LI><A href="#FOS">FOS</A></LI><LI><A href="#Cloning_strategy">Cloning
Strategy</A></LI></OL><LI><A href="#Literature_">Literature </A></LI></OL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="art-Post"><DIV class="art-Post-body"><DIV class="art-Post-inner"><DIV class="art-PostMetadataHeader"><H2 class="art-PostHeaderIcon-wrapper"> Summary</H2></DIV><DIV class="art-PostContent">

The goal of providing an universal restriction enzyme was approached
with two design strategies. The first strategy evolves around novel
protein fusion constructs combining the cleavage domain of the Type IIs
restriction enzyme FokI with hapten binding anticalins, which are then
guided to their target sites by a modified oligonucleotide. The second
approach aimed at converting the Argonaute proteins from thermophilic
organisms from an RNase to an DNase activity while accepting a DNA
guide oligonucleotide. For the Fok-based strategy, several variations
were tested. In both projects important milestones were reached. Most
importantly, we demonstrated guided cleavage by one of our Fok-based
fusion constructs and phage display of an Argonaute protein.
In the following we introduce these two projects and
then list the highlights of our work with links to detailed
descriptions of each project part. 
Both strategies rely on locating the universal restriction enzyme at
the cleavage site with adapter or guide oligonucleotides. This is in
contrast to previous designs which either use chemical linkage of an
oligonucleotide to a nuclease or genetic fusion with a DNA binding
domain. Previous fusion protein approaches have the conceptual
disadvantage that for each target sequence a special protein has to be
designed, expressed, purified, and stored. We only need to produce one
protein. The oligonucleotide planning is aided by readily available PCR
primer design programs and there are virtually no limits to define the
cleavage site. Modified oligonucleotides can be ordered from many
suppliers at low cost with short delivery times and easily can be
stored long term. In in-vitro applications hybridization of the
oligonucleotide is easily achieved also to double strand by heating and
cooling as it is well known from PCR procedures. In case of the
Argonaute proteins from thermophiles we can assume that they survive
several cycles of heat dissociation and annealing. In case of the Fok
we plan to introduce thermostability by directed evolution. In
addition, other groups are working with oligonucleotides forming triple
helices and their general hybridization with any sequence or with
peptide nucleotide acids which provide higher stability and sneak in
existing double helices. These technologies are compatible with our
approach.
In both of our universal restriction enzyme strategies we do not cut
the double strand but rather nick the stand opposite to our guide
oligonucleotide. Thus, our guide oligonucleotide only needs to be added
only in catalytic quantities. The nicking feature is already present in
the Argonaute proteins. In the case of the universal Fok-based enzyme
we use a heterodimer design combined with cleavage inactivating
mutations for one monomer. 
<STRONG>Details of the universal restriction enzyme based on Fok-Anticalin fusions</STRONG>
The restriction endonuclease FokI from Flavobacterium okeanokoites is a
well studied protein. It consists of two domains, a DNA recognition
domain and a DNA cleavage domain. Upon recognition of one target site
and dimerization it cleaves the DNA nine bases apart from the
recognition site. Several groups reported experiments to uncouple the
cleavage and restriction domains of FokI and created a novel
site-specific endonucleases by linking the cleavage domain to zinc
finger proteins (Miller et al. 2007). 
For our project we combined two previous research results and generated
a Fok cleavage heterodimer comprising an enzymatically active and
inactive monomer. For the catalytic active Fok partner, named Fok_a, as
well as for the catalytic inactive Fok partner, Fok_i, the association
interface was mutated to disfavor homodimerization and promote
heterodimerization. In Fok_i additional amino acid exchanges led to the
inactivation.
The two heterodimeric partners were fused to anticalins binding
different adapter molecules. Fok_a is genetically fused to a
digoxigenin-binding anticalin (DigA) and Fok_i to a fluorescein-binding
anticalin (FluA). The adapter molecules digoxigenin and fluorescein are
common modifications linked to oligonucleotides thus mediating the
binding to the DNA site of interest. Two modifications allow for a
better spatial control of the cleavage site. On the target site Fok_i
and Fok_a constructs are brought into close proximity and can form a
heterodimer. The inactive Fok domain will serve as an activator of the
active Fok domain, which cuts one strand of the DNA. Our structural
‘3D’ models, indicate that Fok domains can be
positioned in such a way that Fok_a will cleave the target DNA, and
Fok_i would be directed towards the modified oligonucleotide. Different
linkers were designed and fused between cleavage domain and the
anticalin binding moieties to test for the optimal distance. In
addition we generated a monomeric Fok fusion construct, to enable phage
display and te4st a further setting requiring only one binding domain
and hapten. Furthermore, we made a Fok cleavage domain fusion with a
coiled coil based DNA binding domain, because we can combine these with
existing light switchable inhibitors, which prevent DNA binding. As a
result we will obtaina light switchable restriction enzyme.
<STRONG>Milestones</STRONG>
To reach our goal within the short given time frame we started several
subprojects in parallel. Our subprojects listed here are defined along
these projects. 
Designing of the constructs was aided by extensive model building and
analysis of the spatial orientation of the different proteins and
oligonucleotides used. All designed and constructed parts feature full
BioBrick compatibility and in addition allow for the construction of
fusion proteins based on the RFC 25 (Freiburg) cloning standard.
Cloning of the respective parts was followed by expression purification
and analysis. Despite previous literature data our active Fok construct
was toxic to cells when expressed and we tested
periplasmic expression with export of the nascent polypeptide
chain before folding.

In our experiments we addressed the following questions:•    Structural Model building•    Design of protein fusion parts•    Cloning of anticalin Fok fusions•    Cloning of a monomeric Fok construct and of a Jun/Fos directed Fok construct•    Expression and purification of constructs•   <EM>In vitro</EM> assays•   <EM>In vivo</EM> assays•    Phage Display of an Ago protein•    Modeling of assembly and cleavage with differential equations•    An international survey of laymen on synthetic biology
In short, we successfully worked on all aspects. Experiments to validate our approach in more compelx settings are ongoing. 
The labs of Kristian Müller and Katja Arndt provided all technology and support for the project.<TABLE style="text-align: left; width: 100%;" border="0" cellpadding="2" cellspacing="2"><TBODY><TR><TD>
      Schematic Model of the universal restriction enzymes based on
      FokI and anticalins.
      </TD><TD>
      Structural model of the universal restriction enzymes based on FokI and anticalins.
      </TD></TR><TR><TD>
       Model of the catalytic cycle; hybridization - cleavage - temperature
       promoted release.
      </TD><TD>
       Structure of an Ago protein, demonstrating guide oligonucleotide
       mediated binding.
      </TD></TR></TBODY></TABLE><DIV class="art-Post"><DIV class="art-Post-body"><DIV class="art-Post-inner"><DIV class="art-PostMetadataHeader"><H2 class="art-PostHeaderIcon-wrapper"> Highlights</H2></DIV><H4>Modeling of the Enzyme Kinetics</H4><TABLE style="text-align: justify; width: 100%"><TBODY><TR><TD>For our modeling analyses we constructed various sets of differential equations describing
     protein-protein and protein-DNA interactions and the final cleavage. <A href="https://2009.igem.org/Team:Freiburg_bioware/Modeling">Read more...</A></TD></TR></TBODY></TABLE><H4>Modeling of the Enzyme Structure</H4><TABLE style="text-align: justify; width: 100%"><TBODY><TR><TD>
      Structural modeling was an initial step towards
       our planned universal restriction enzyme. Using molecular display software we arranged published
       crystal structures of FokI, anticalins and DNA. The arrangement was guided by superimposition with
       further structures of enzyme bound DNA. The modeling defined spatial requirements for linker lengths
       and positioning of modifications within oligonucleotides.
       <A href="https://2009.igem.org/Team:Freiburg_bioware/Project/3d-modeling">Read more...</A></TD></TR></TBODY></TABLE><H4><I>In vitro</I> assays</H4><TABLE style="text-align: justify; width: 100%"><TBODY><TR><TD>
      After the cloning, expression and the purification of
      the Fok constructs we conducted several assays in order to to analyze the activity of the enzyme. To
      establish the
      assay and as a reference for activity we used wild type FokI. Binding of the modified 
      nucleotides and enzymatic activity were tested with the Fok_i / Fok_a construct.
      <A href="https://2009.igem.org/Team:Freiburg_bioware/Project/invitro">Read more...</A></TD></TR></TBODY></TABLE><H4><I>In vivo</I> Assays</H4><TABLE style="text-align: justify; width: 100%"><TBODY><TR><TD>
      We demonstrated that modified guide oligonucleotides can be transfected in E. coli.
       Importantly, we showed that a M13 DNA hybridized with a guide oligonucleotide does
       not produce phages when transfectd in cells expressing Fok fusion proteins whereas 
       the control M13 DNA does.
      
      <A href="https://2009.igem.org/Team:Freiburg_bioware/Project/invivo">Read more...</A></TD></TR></TBODY></TABLE><H4>Protein Expression and Purification</H4><TABLE style="text-align: justify; width: 100%"><TBODY><TR><TD>
      Proteins were recombinantly expressed in E. coli strains BL21 or RV308. Purification was achieved by 
      affinity chromatography utilizing the His-tag, Strep-tag, or the GST fusion. If needed, two affinity
      purifications due to two available tags were combined or a size exclusion chromatogrpahy was added. 
      <A href="https://2009.igem.org/Team:Freiburg_bioware/Project/purification">Read more...</A></TD></TR></TBODY></TABLE><H4>AGO</H4><TABLE style="text-align: justify; width: 100%"><TBODY><TR><TD>
      The Argonaute proteins represent our second approach to generate universal restriction enzymes.
      These RNases provide already important features such as recognition of guide oligonucleotides   and thermostability. 
      We expressed and purified an AGO protein and showed residual DNase activtiy. Consequently, we started
      phage display of error prone PCR diversified AGO in order to convert the RNase in a DNase. We 
      demonstratd AGO display, successfully completed two rounds of phage display and identified interesting mutants.
      <A href="https://2009.igem.org/Team:Freiburg_bioware/Project/AGO">Read more...</A></TD></TR></TBODY></TABLE><H4>Ethics</H4><TABLE style="text-align: justify; width: 100%"><TBODY><TR><TD>
     We conducted an international survey on benefit and risk perception of Synthetic Biology by laymen, which was translated in 10 languages.
     <A href="https://2009.igem.org/Team:Freiburg_bioware/Human_Practice/Ethics">Read more...</A></TD></TR></TBODY></TABLE><H4>Fok Monomer</H4><TABLE style="text-align: justify; width: 100%"><TBODY><TR><TD>
      To ease production, purification, handling and analysis we cloned a monomeric Fok variant.
      In addition, this variant is better suited for phage display, which will allow us selection for
      improved properties. This protein comprises four functional subunits interspaced by two short and
      one long linker (anticalin-Fok_i-long_linker-Fok_a-anticalin).
      <A href="https://2009.igem.org/Team:Freiburg_bioware/Project/fokmonomer">Read more...</A></TD></TR></TBODY></TABLE><H4>Alternative way of binding: Jun/Fos</H4><TABLE style="text-align: justify; width: 100%"><TBODY><TR><TD> As an alternative way of binding of Fok to DNA we cloned the DNA 
      binding domain (bZIP) of the activator protein-1 (AP-1). As light switching of coiled coils 
      is established in our host lab, this will give us a light switchable restriction enzyme. 
     <A href="https://2009.igem.org/Team:Freiburg_bioware/Project/FOS">Read more...</A></TD></TR></TBODY></TABLE><H4>Cloning strategy</H4><TABLE style="text-align: justify; width: 100%"><TBODY><TR><TD>
     Here we give an overview of the cloning strategies we used for creating a universal restriction endonuclease. 
     Fusion proteins were generated according to the Freiburg Assembly standard RFC 25.
     <A href="https://2009.igem.org/wiki/index.php?title=Team:Freiburg_bioware/Project/clonestrat">Read more...</A></TD></TR></TBODY></TABLE><P>contact: <A href="mailto:freigem09@googlemail.com">freigem09@googlemail.com</A>; 
<A href="mailto:freigem09@googlemail.com">kristian@biologie.uni-freiburg.de</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Freiburg_bioware/Project">http://2009.igem.org/Team:Freiburg_bioware/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_bioware/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_bioware/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project&amp;oldid=169539" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>