<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_bioware_Project_3d-modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Freiburg_bioware/Project/3d-modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_bioware/Project/3d-modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project/3d-modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project/3d-modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_bioware/Project/3d-modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg bioware/Project/3d-modeling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>FREiGEM</TITLE></P><DIV id="art-main"><DIV class="art-Sheet"><DIV class="art-Sheet-body"><DIV class="art-nav"><UL class="art-menu"><LI><A href="https://2009.igem.org/Team:Freiburg_bioware"><SPAN class="t">Home</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Team"><SPAN class="t">The Team</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Team/Portraits">Portraits</A></LI></UL><LI><A class="active" href="https://2009.igem.org/Team:Freiburg_bioware/Project"><SPAN class="t">The
Project</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Project#Summary">Summary</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Project#Highlights">Highlights</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Human_Practice"><SPAN class="t">Human
Practice</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Human_Practice/Ethics">Ethics</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Human_Practice/Safety">Safety</A></LI><LI><A href="#"><SPAN class="t">Notebook</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/cloning1"><SPAN class="t">Parts</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/cloning1">Basic
Parts</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/cloning">Composite
Parts</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Collaboration"><SPAN class="t">Collaboration</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Modeling"><SPAN class="t">Modeling</SPAN></A></LI></DIV><DIV class="art-contentLayout"><DIV class="art-content"><DIV class="art-Post"><DIV class="art-Post-body"><DIV class="art-Post-inner"><DIV class="art-PostMetadataHeader"><H2 class="art-PostHeaderIcon-wrapper"> 
3D-Modeling</H2></DIV><DIV class="art-PostContent"><DIV style="text-align: center;"><DIV style="text-align: left;"><P class="MsoNormal"><B style=""><SPAN lang="EN-US">Structure
Based Design of a Programmable Restriction Enzyme</SPAN></B></P><P class="MsoNormal"><SPAN lang="EN-US"><O:P> </O:P></SPAN></P><P class="MsoNormal"><SPAN lang="EN-US"><O:P> </O:P></SPAN></P><P class="MsoNormal"><B style=""><SPAN lang="EN-US">Introduction</SPAN></B></P><P class="MsoNormal"><SPAN lang="EN-US"><O:P> </O:P></SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">To
construct the universal restriction
enzyme we had in mind, it was necessary to understand the 3dimensional
structure
of enzymes which are able to cut DNA.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">Prerequisite
for our <SPAN style=""> </SPAN>restriction
enzyme design was a DNA cutting
site separated from its DNA recognition site. Many enzymes have both in
the
same spot and therefore it is very difficult to modify the recognition
pattern
without destroying the enzyme´s catalytic activity. However,
there is one
enzyme that perfectly matched our needs, namely FokI (Fig.1).</SPAN></P><P class="MsoNormal"><B style=""><SPAN lang="EN-US"><O:P> </O:P></SPAN></B></P><P class="MsoNormal"><B style=""><SPAN lang="EN-US"><O:P> </O:P></SPAN></B></P><P class="MsoNormal"><B style=""><SPAN lang="EN-US">3d
model of the cleavage domains</SPAN></B></P><P class="MsoNormal"><SPAN lang="EN-US"><O:P> </O:P></SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">The
idea was to modify FokI in a way that enables
us to make the protein bind single strand DNA at a user defined site
and
consequently to cut DNA precisely. So we searched for all structural
data
available. The crystallized FokI-homodimer (pdbID: 2FOK), published by
Wah and
colleagues (Wah et al 1998), gave us the initial clue to start in
silico
modeling. We used the program PyMOL (DeLano W.L. 2002)</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">FokI
functions as a dimer, however, only
one of the identical proteins binds to DNA with its DNA binding domain.
Since
there is no use for a specific DNA binding domain in a universal
restriction
enzyme, we decided to use both cleavage domains separately and to
delete the
binding domains. The two cleavage domains from amino acid 387 to 579
remained
(Fig.1). They appeared to be hydrophilic on the surface, so nothingneeded to be modified,
except cystein 541 which was
replaced by a serine. Cysteins might interferewhen
the protein is expressed in the periplasm of e.coli. The catalytically
active
amino acids (Fig.2 green) each conjugate two Ca<SUP>2+</SUP>
ions to brake DNA
backbone. </SPAN><SPAN style="">Mechanism reviewed
by </SPAN><SPAN style="">Pingoud 2005.</SPAN></P><TABLE style="text-align: left; width: 900px;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD><DIV style="text-align: center;"><SMALL><B style=""><SPAN lang="EN-US">Fig1.
FokI-Dimer: </SPAN></B><SPAN lang="EN-US">Grey:
DNA binding domains,(amino acids
1 – 386); Red and Green: catalytic domains (amino acids 387
to 579.)</SPAN></SMALL></DIV></TD></TR></TBODY></TABLE><P class="MsoNormal"><SPAN class="itatext"><B style=""><SPAN lang="EN-US">Creating the
heterodimer Interface</SPAN></B></SPAN></P><P class="MsoNormal"><SPAN class="itatext"><SPAN lang="EN-US"><O:P> </O:P></SPAN></SPAN></P><P class="MsoNormal"><SPAN class="itatext"><SPAN lang="EN-US">To avoid unspecific cleavage
of DNA by unbound catalytic FokI domains during recombinant expression
of the
protein, we were looking for a control feature. In theory this was easy
to
manage by switching the homodimer into a heterodimer. In a Fok
heterodimer,
each active part only matches with its complementary counterpart. We
reasoned
that separate expression of each monomer in different E. coli cultures
would
avoid formation of the active FokI cleavage domains and thus diminish
off site
celavage. The heterodimer interface was designed according to work
published by
Miller (</SPAN></SPAN><SPAN style="" lang="EN-US">Miller
et al. 2007)</SPAN><SPAN class="itatext"><SPAN lang="EN-US">. In the </SPAN></SPAN><SPAN lang="EN-US">catalytically active heterodimer we switched
glutamate 490 to lysin
(Base Triplet GAA-&gt;AAA) and isoleucin 538 to lysin (Base Triplet
ATC-&gt;AAA).
In the catalytically inactive heterodimer we switched glutamin 486 to
glutamate
(Base Triplet CAA-&gt;GAA) and isoleucin 499to leucin (Base Triplet
ATC-&gt;CTG).</SPAN></P><P class="MsoNormal"><SPAN class="itatext"><SPAN lang="EN-US"><O:P> </O:P></SPAN></SPAN></P><P class="MsoNormal"><SPAN class="itatext"><SPAN lang="EN-US"><O:P> </O:P></SPAN></SPAN></P><P class="MsoNormal"><SPAN class="itatext"><SPAN lang="EN-US">Another advantage of
creating two distinct parts is that we can exactly define which one
will cut
the ssDNA and which one will only have a stabilizing function. The
stabilizing one
will be located above the complementary strand of the DNA that should
not be
cut. Therefore we exchanged its catalytically active amino acids into
neutral
ones based on the work of Wah </SPAN></SPAN><SPAN lang="EN-US">(Wah 1997).We
replaced amino acids aspartate 450 with alanin (Base Triplet
GAC-&gt;GCG) and
aspartate 467 with alanin (GAT-&gt;GCG) (Fig.2).</SPAN></P><TABLE style="text-align: left; width: 900px;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR align="center"><TD><P class="MsoNormal"><SMALL><SPAN class="itatext"><B style=""><SPAN lang="EN-US">Fig.2 two FokI cleavage domains (amino acid 387
to 579). </SPAN></B><SPAN lang="EN-US">Red:Catalytically active
FokI cleavage domain; Green: Catalytically inactive FokI cleavage
domain; Pink:
catalytically active aminoacids; Yellow: </SPAN></SPAN><SPAN lang="EN-US">glutamate
490 and isoleucin 538; Blue: glutamin 486 and isoleucin 499.</SPAN></SMALL></P></TD></TR></TBODY></TABLE><P class="MsoNormal"><SPAN class="itatext"><B style=""><SPAN lang="EN-US">3D model of the DNA
</SPAN></B></SPAN></P><P class="MsoNormal"><SPAN class="itatext"><SPAN lang="EN-US"><O:P> </O:P></SPAN></SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">The
next step was to determine optimal positioning
of the DNA relative to the protein to mediate desired cleavage. We
superimposed
structures as described by Wah 1998 and aligned the structure of BamHI
(pdbID:
2BAM) (<SPAN class="listtext">Viadiu et al 1998) </SPAN>with
our two domains. In
fact, only the parts of the protein that are responsible for cleavage
are
similar. We picked four matching amino acids that were in close
proximity to
the catalytically active amino acids for the structural alignment. In
BamHI these
are Asp-94 and Val-108 and in FokI these are Asp-450 and Val-164. Each
aminoacid was selected two times, since the structures are dimers.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">After
alignment we kept the Ca<SUP>2+</SUP>
ions and the DNA that were crystallized together with both BamHI
domains and
deleted the rest of the BamHI file from the model.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">The
DNA part was too short to work with it,
so we created a 50Bp long DNA with the DNA origami software SARSE
(Anderson et
al.) and replaced the BamHI DNA with this 50bp DNA by aligning the
phosphates
of the DNA backbone (Fig.4 DNA cyan, Oligonucleotide magenta). </SPAN></P><P class="MsoNormal"><B style=""><SPAN lang="EN-US"><O:P> </O:P></SPAN></B></P><P class="MsoNormal"><B style=""><SPAN lang="EN-US"><O:P> </O:P></SPAN></B></P><P class="MsoNormal"><B style=""><SPAN lang="EN-US">Determination
of length and tag positions of the oligonucleotides </SPAN></B></P><P class="MsoNormal"><SPAN lang="EN-US"><O:P> </O:P></SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">The
next task was to connect the cleavage
domains with a construction that directs them exactly to that area of
the ssDNA
one wishes to cut.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">As
outlined in the general introduction we had
the following plan: the user defines oligonucleotides complementary to
the area
of the cutting site. These oligonucleotides must be marked with a
fluorescein
and a digoxigenin in a predefined distance from the position that is to
be cut.
These tags are linked to the oligonucleotide by C6 linkers (having the
length
of 6 carbons) (Fig.4 yellow). They will be recognized by specially
designed lipocalins
that are attached to the cleavage domain. Using the chemical modeling
software
ACD/ChemSketch (http://www.acdlabs.com/download/chemsketch) we
generated a 3D
model of the C6 linker and manually fitted in the correct position in
the
model.</SPAN></P><TABLE style="text-align: left; width: 900px;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR align="center"><TD><P class="MsoNormal"><SMALL><B style=""><SPAN lang="EN-US">Fig.3
Oligonucleotide scheme.</SPAN></B><SPAN lang="EN-US">
The picture shows
symbolically the ssDNA in blue and its complementary strand that will
be the oligonucleotides
in red. The DNA sequence is a random example. The bars indicate
templates for
oligonucleotides with several options for their position relative to
the
cutting site. Pink: the position where the universal restriction enzyme
based
on FokI will cut (on the right side of the marked base); Yellow left:
position of
the fluorescein tagged base; Yellow right: position of the digoxigenin
tagged
base; </SPAN></SMALL></P><P class="MsoNormal"><SMALL><SPAN lang="EN-US">Upper bar: <SPAN style=""> </SPAN>dioligo
AB 16 bases with 5' fluorescein tag,;
diologo A 16 bases with 5' tag digoxigenin, the planned cut site is
between base
8/9 from the 5' end; </SPAN></SMALL></P><P class="MsoNormal"><SMALL><SPAN lang="EN-US">Middle bar: monooligo 30 bases with
3’
fluorescein tag and internal digoxigenin modification 14 bases from 5',
the
planned cut site ist between base 21/22 from its 5' end;</SPAN></SMALL></P><P class="MsoNormal"><SMALL><SPAN lang="EN-US">Lower bar: dioligoAB; dioligo B with
internal digoxigenin modification15 bases from the 5'end, the planned
cut is between
base 22/23 from 5' end, 30bp long;</SPAN></SMALL></P></TD></TR></TBODY></TABLE><P class="MsoNormal"><SPAN lang="EN-US">Visual
inspection revealed that the FokI
cleavage domains in binding position occupy two opposing minor grooves
of a
helical DNA. It seemed reasonable to arrange the lipocalins flanking
the
cleavage domains on the outside, right above the neighboring major
grooves
(Fig.5). By this arrangement the shortest possible length of the
oligonucleotides
and the position of the tags were defined. From the cutting site
towards the 5´
end of the ssDNA we determined a distance of about 30nm which can be
spanned by
e.g. 11 amino acids. This means the complementary oligonucleotide
(Fig.3
dioligo AB) has to have its fluorescein tag on its 5´
position, the oligonucleotide
however can have any length in the <SPAN style=""> </SPAN>3´
direction
(here we chose 16bp). The second oligonucleotide (Fig.3 dioligo A) with
a
digoxigenin tag is located next to the first, further towards the
3´ end of the
ssDNA. Its tag must be located 6 bases further towards the 3´
end of the ssDNA.
Since it is much cheaper to buy oligonucleotides tagged on 3´
or 5´ end we
designed the second oligonucleotide with 16 bases and its tag also on
the 5´
end (Fig.3 and 4).</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">In
total the hybridized DNA should be
around 30bp long to ensure its folding into an helical state at room
temperature.</SPAN></P><TABLE style="text-align: left; width: 900px;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD><DIV style="text-align: center;"><SMALL><B style=""><SPAN lang="EN-US">Fig.4
DNA, Oligonucleotides and FokI domains</SPAN></B><SPAN lang="EN-US">. Cyan: ssDNA;
Magenta: two 16 bases long oligonucleotide; Yellow: C6 linkers, left
with
fluorescein, right with digoxigenin; Green: the FokI inactive domain;
Red: the
FokI active cleavage domain; Pink: 3 catalytically active aminoacids;
Grey: two
calcium ions. </SPAN></SMALL><SPAN style=""><SMALL>Not
shown are
lipocalins.</SMALL></SPAN></DIV></TD></TR></TBODY></TABLE><P class="MsoNormal"><SPAN lang="EN-US">We
also designed further oligonucleotide
variants. One includes both tags (Fig.3 monooligo) and another one
(Fig.3
dioligo B) can be combined with diologo AB., However in contrast to
dioligo A its
tag is internal. The advantages of a single oligonucleotide is that the
kinetics of the cleavage reaction would be improved, since it is one
part less
that has to find its position to initiate the reaction. The advantage
of an
extended second oligonucleotide is that it can work under higher
temperatures,
what also speeds up the reaction. </SPAN></P><P class="MsoNormal"><SPAN lang="EN-US"><O:P> </O:P></SPAN></P><P class="MsoNormal"><SPAN lang="EN-US"><O:P> </O:P></SPAN></P><P class="MsoNormal"><B style=""><SPAN lang="EN-US">Lipocalins
as connectors between oligonucleotides and FokI cleavage domains</SPAN></B></P><P class="MsoNormal"><SPAN lang="EN-US"><O:P> </O:P></SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">Lipocalins,
also called Anticalins, are 20 kDa
small proteins that can be designed to bind specific substances. We
used an
anticalin against digoxigenin (pdbID: 1NOS; Fig.5 orange) (Korndorfer
2003a) and
another against fluorescein (pdbID: 1LKE; Fig.5 blue) (Korndorfer
2003b).</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">They
were loaded into the model and fitted
manually above the major DNA grooves next to the FokI cleavage domains
(Fig. 4).
C-termini of the lipocalins show towards the N-termini of the cleavage
domains.
To connect each lipocalin with each cleavage domain we designed several
linkers
(see parts). These are not shown in the model. The linkers need to be
long
enough to allow easy binding of the FokI cleavage domains, but must be
tight
enough to ensure exact cutting. The fluorescein binding lipocalin needs
to be
connected to the inactive FokI cleavage domain, the digoxigenin binding
one to
the active domain. </SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">These
lipocalins were readily available from
the <ST1:PLACE w:st="on">Freiburg</ST1:PLACE> iGEM
team 2008.</SPAN></P><TABLE style="text-align: left; width: 900px; height: 38px;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR align="center"><TD><SMALL><B style=""><SPAN style="font-size: 12pt; font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" lang="EN-US">Fig.5 complete universal restriction enzyme</SPAN></B><SPAN style="font-size: 12pt; font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" lang="EN-US">. Cyan: ssDNA; Magenta: two 16 bases long
oligonucleotide;
Yellow: C6 linkers, left with fluorescein, right with digoxigenin;
Green: the
FokI inactive domain; Red: the FokI active cleavage domain; Pink: 3
catalytically active aminoacids; Grey: two Calcium ions; <ST1:CITY w:st="on"><ST1:PLACE w:st="on">Orange</ST1:PLACE></ST1:CITY>:
digoxigenin binding lipocalin; Blue:
Fluorescein A binding lipocalin</SPAN></SMALL></TD></TR></TBODY></TABLE><P class="MsoNormal"><B style=""><SPAN lang="EN-US">References</SPAN></B></P><P class="MsoNormal"><SPAN style="" lang="EN-US"><O:P> </O:P></SPAN></P><P class="MsoNormal"><SPAN style="" lang="IT">Andersen
<EM>DNA
Origami</EM> program SARSE </SPAN><CITE><SPAN lang="EN-US">www.<SPAN style="">sarse</SPAN>.org/index.php/<SPAN style="">DNA</SPAN>_<SPAN style="">origami</SPAN></SPAN></CITE></P><P class="MsoNormal"><SPAN style="" lang="IT"><O:P> </O:P></SPAN></P><P class="MsoNormal"><SPAN class="listtext"><SPAN lang="EN-US">Korndorfer IP,
Schlehuber S, Skerra A 2000a </SPAN></SPAN><SPAN lang="EN-US"><A href="http://www.ncbi.nlm.nih.gov/pubmed/12823976"><SPAN style="color: windowtext; text-decoration: none;">Structural
mechanism of specific ligand recognition by a lipocalin tailored for
the
complexation of digoxigenin</SPAN></A><SPAN class="itatext">J. Mol. Biol. v330,
p.385-396</SPAN></SPAN></P><P class="MsoNormal"><SPAN class="itatext"><SPAN lang="EN-US"><O:P> </O:P></SPAN></SPAN></P><P class="MsoNormal"><SPAN class="listtext"><SPAN lang="EN-US">Korndorfer IP, Beste
G, Skerra A 2000b </SPAN></SPAN><SPAN lang="EN-US"><A href="http://www.ncbi.nlm.nih.gov/pubmed/12945055"><SPAN style="color: windowtext; text-decoration: none;">Crystallographic
analysis of an &quot;anticalin&quot; with tailored specificity for fluorescein
reveals high structural plasticity of the lipocalin loop region</SPAN></A><SPAN class="itatext">Proteins v53, p.121-129</SPAN></SPAN></P><P class="MsoNormal"><SPAN lang="EN-US"><O:P> </O:P></SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">Miller,
J.C., Holmes, M.C., Wang, J.,
Guschin, D.Y., Lee, Y.L., Rupniewski, I., Beausejour, C.M., Waite, A.J<SPAN class="msoins0">., Wang, N.S., Kim, K.A., Gregory, P.D.,
Pabo, C.O., &amp; Rebar,
E.J. (2007) An improved zinc-finger nuclease architecture for highly
specific
genome editing, <SPAN style="">Nat Biotechnol</SPAN>
25, 778-785</SPAN></SPAN></P><P class="MsoNormal"><SPAN class="msoins0"><SPAN lang="EN-US"><O:P> </O:P></SPAN></SPAN></P><P class="MsoNormal"><SPAN style="" lang="EN-US">A.
Pingoud, M. Fuxreiterb,V. Pingoud and W.Wende Type II restriction
endonucleases: structure and mechanism, Review, Cell. Mol. Life Sci. 62
(2005)
685–707</SPAN></P><P class="MsoNormal"><B><SPAN style="" lang="EN-US"><O:P> </O:P></SPAN></B></P><PRE><ST1:CITY w:st="on"><ST1:PLACE w:st="on"><SPAN style=""><SPAN style="font-size: 12pt; font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" lang="EN-US">DeLano</SPAN></SPAN></ST1:PLACE></ST1:CITY><SPAN style=""><SPAN style="font-size: 12pt; font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" lang="EN-US">, W.L. <SPAN style="">The PyMOL Molecular Graphics System</SPAN> (2002) on World Wide Web <SPAN style="">http://www.pymol.org</SPAN></SPAN></SPAN></PRE><P class="MsoNormal"><B><SPAN style="" lang="EN-US"><O:P> </O:P></SPAN></B></P><P class="MsoNormal"><SPAN class="listtext"><SPAN lang="EN-US">Viadiu H, <ST1:PLACE w:st="on"><ST1:CITY w:st="on">Aggarwal</ST1:CITY><ST1:STATE w:st="on">AK</ST1:STATE></ST1:PLACE></SPAN></SPAN><SPAN lang="EN-US"><A href="http://www.ncbi.nlm.nih.gov/pubmed/9783752"><SPAN style="color: windowtext; text-decoration: none;">The role
of metals in catalysis by the restriction endonuclease BamHI</SPAN></A><SPAN class="itatext">Nat. Struct. Biol. v5, p.910-916 </SPAN></SPAN><SPAN style="" lang="EN-US">(1998) </SPAN></P><P class="MsoNormal"><B><SPAN style="" lang="EN-US"><O:P> </O:P></SPAN></B></P><P class="MsoNormal"><SPAN class="listtext"><SPAN lang="EN-US">Wah DA, Bitinaite J,
Schildkraut I, Aggarwal AK</SPAN></SPAN><SPAN lang="EN-US"><A href="http://www.ncbi.nlm.nih.gov/pubmed/9724743"><SPAN style="color: windowtext; text-decoration: none;">Structure
of FokI has implications for DNA cleavage</SPAN></A><SPAN class="itatext">Proc.
Natl. Acad. Sci. U. S. A. v95, p.10564-10569 </SPAN></SPAN><SPAN style="" lang="EN-US">(1998)</SPAN></P><P class="MsoNormal"><SPAN style=""><O:P> </O:P></SPAN></P><P class="MsoNormal"><SPAN style="">David
A. Wah, Joel A. Hirsch*, Lydia F. Dorner†, Ira
Schildkraut†&amp; Aneel K.
Aggarwal*; Structure of the multimodular endonuclease <SPAN style="">Fok</SPAN>I
bound to DNA, NATURE |VOL 388 | 3 JULY 1997</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="art-Footer"><DIV class="art-Footer-inner"><DIV class="art-Footer-text"><P>contact: <A href="mailto:freigem09@googlemail.com">freigem09@googlemail.com</A></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Freiburg_bioware/Project/3d-modeling">http://2009.igem.org/Team:Freiburg_bioware/Project/3d-modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_bioware/Project/3d-modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_bioware/Project/3d-modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project/3d-modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project/3d-modeling&amp;oldid=168272" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>