<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_bioware_Project_fokmonomer"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Freiburg_bioware/Project/fokmonomer">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_bioware/Project/fokmonomer&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project/fokmonomer&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project/fokmonomer&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_bioware/Project/fokmonomer" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg bioware/Project/fokmonomer</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>FREiGEM</TITLE></P><DIV id="art-main"><DIV class="art-Sheet"><DIV class="art-Sheet-body"><DIV class="art-nav"><UL class="art-menu"><LI><A href="https://2009.igem.org/Team:Freiburg_bioware"><SPAN class="t">Home</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Team"><SPAN class="t">The Team</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Team">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Team/Portraits">Portraits</A></LI></UL><LI><A class="active" href="https://2009.igem.org/Team:Freiburg_bioware/Project"><SPAN class="t">The
Project</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Project#Summary">Summary</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Project#Subprojects">Subprojects</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Project#Highlights">Highlights</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Human_Practice"><SPAN class="t">Human
Practice</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Human_Practice/Ethics">Ethics</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Human_Practice/Safety">Safety</A></LI><LI><A href="#"><SPAN class="t">Notebook</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/cloning1"><SPAN class="t">Parts</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/cloning1">Basic
Parts</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/cloning">Composite
Parts</A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Collaboration"><SPAN class="t">Collaboration</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Freiburg_bioware/Modeling"><SPAN class="t">Modeling</SPAN></A></LI></DIV><DIV class="art-contentLayout"><DIV class="art-content"><DIV class="art-Post"><DIV class="art-Post-body"><DIV class="art-Post-inner"><DIV class="art-PostMetadataHeader"><H2 class="art-PostHeaderIcon-wrapper"> 
Fok Monomer</H2></DIV><DIV class="art-PostContent"><DIV style="text-align: left;"><P class="MsoNormal" style="text-align: justify;"><B style=""><U>Introduction</U></B></P><P class="MsoNormal" style="text-align: justify;"><O:P> </O:P></P></DIV><P style="text-align: justify;"><SPAN style="" lang="EN-GB">In order to create a universal
restriction
enzyme, we followed several ideas and models. The first and most
promising idea
we had was to create a monomeric restriction enzyme as it represents the optimal and
easiest
model for a universal restriction enzyme</SPAN></P><P class="MsoNormal" style="text-align: justify;"><SPAN style="" lang="EN-GB">If we are able to employ a
monomeric enzyme,
this protein would have a couple of advantages: Most importantly we
would no longer
need two separate oligonucleotides to achieve specific binding and
cleavage of
the target DNA. Also, only one protein has to be purified –
thus saving time and
money. A scientific advantage is the option to optimize the monomer by
phage
display. Using this technology, we would have the chance to create a
thermostable, specific, universal restriction enzyme whose DNA binding
specificity
is simply created by a single oligonucleotide.</SPAN></P><P class="MsoNormal" style="text-align: justify;"><SPAN style="" lang="EN-GB">Our goal was first, to create
a Fok-monomer that
is able to cut DNA without a primary dimerization step, and second, to
show
that our heterodimeric interface design works properly. To reach this,
we had
to clone all the required parts one after another, resulting in a huge
fusion
protein.</SPAN></P><P class="MsoNormal" style="text-align: justify;"><B style=""><U><SPAN style="" lang="EN-GB">Results</SPAN></U></B></P><P class="MsoNormal" style="text-align: justify;"><U><SPAN style="" lang="EN-GB">3D modeling and design goals</SPAN></U></P><TABLE style="text-align: left; width: 208px; height: 225px;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD><SPAN style="font-size: 10pt;" lang="EN-GB">Fig. 1:
      <ST1:PLACE w:st="on"><ST2:SN w:st="on">Scheme of monomoeric Fok:</ST2:SN><ST2:SN w:st="on">I.</ST2:SN></ST1:PLACE>
lipocalin,<SPAN style=""> </SPAN>II.
Foka/Foki (heterodimers), III.
Linker,<SPAN style=""> </SPAN>IV. His-Tag</SPAN></TD></TR></TBODY></TABLE><P class="MsoNormal" style="text-align: justify;"><SPAN style="" lang="EN-US">The necessary parts in the
model of a
monomeric restriction enzyme are: Tag-binding protein (lipocalin) - Fok
heterodimer1 -
linker - Fok heterodimer2 (Fig. 1). </SPAN></P><P class="MsoNormal" style="text-align: justify;"><SPAN style="font-size: 10pt;" lang="EN-US"><O:P> </O:P></SPAN><SPAN style="font-size: 10pt;" lang="EN-US"><O:P><TABLE style="text-align: left; width: 705px; height: 50px;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="text-align: center;">Tag</TD><TD style="text-align: center;">binding protein</TD><TD style="text-align: center;">linker</TD><TD style="text-align: center;">Fok heterodimer</TD></TR><TR><TD style="text-align: left;" valign="top">
      HisTag (BBa_K157011)
      StrepTag(BBa_K157012)
      </TD><TD style="text-align: left;" valign="top">
      FluA(BBa_K157004)
      DigA(BBa_K243003)
      Fos(BBa_K243027)
      </TD><TD style="text-align: left;" valign="top">
      GSAT-Linker(BBa_K243029)
      SEGLinker(BBa_K243030)
      </TD><TD style="text-align: left;" valign="top">
      FokA (K243000)
      Foki (K243001)
      </TD></TR></TBODY></TABLE></O:P></SPAN><SPAN style="font-size: 9pt;" lang="EN-US">Tab. 1: List of
parts usable for
Fok Monomer</SPAN></P><P class="MsoNormal" style="text-align: justify;"><SPAN style="" lang="EN-US">In order to purify this fusion
protein, we employed an N-terminal tag. The binding protein (a lipocalin-derived binding protein referred to as anticalin in analogy to antibodies) mediates 
between the
FokI protein and the tagged oligonucleotide</SPAN><SPAN style="" lang="EN-GB">. Both parts of the Fok heterodimers are
associated with a long flexible linker allowing
them to establish the cutting site conformation. This artificial
restriction
nuclease domain is the functional part of the monomer and catalyzes the
DNA
cleavage.</SPAN></P><P class="MsoNormal" style="text-align: justify;"><SPAN style="" lang="EN-GB">Before
we started in the wet lab, we thoroughly planned all
the individual steps. To figure out the best way to create the Fok
monomer, we first designed it <I style="">in
silico.</I></SPAN></P><TABLE style="text-align: left; width: 196px; height: 225px;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD><SPAN style="font-size: 10pt;" lang="EN-GB">Fig. 2:
3D model of Fok Monomer</SPAN></TD></TR></TBODY></TABLE><P class="MsoNormal" style="text-align: justify;"><SPAN style="" lang="EN-GB">The model (Fig. 2) shows that we
have to use a
linker which is about 70 Angstrom in length to
span the required
distance between the two Fok parts. Therefore, we decided to order two
complementary oligonucleotides encoding a 36 amino acid long glycine-serine
linker.</SPAN></P><P class="MsoNormal" style="text-align: justify;"><SPAN style="display:block; width:100%; overflow:scroll;" lang="EN-GB"><O:P> </O:P>
5'-CTAGATGGCCGGCGGTTCTGGTGGTGGTTCTGGCGGTGGTTCTGGAGGTAGTTCTGGCGGTGGATCTGGAGGCGGTTCTGGGTCAGGATCTGGTG<B style=""><SPAN style="color: lime;">GA</SPAN></B>GGTTCTGGCTCTGGGAATCAGA-3'         3'-TACCGGCCGCCAAGACCACCACCAAGACCGCCACCAAGACCTCCATCAAGACCGCCACCTAGACCTCCGCCAAGACCCAGTCCTAGACCAC<B style=""><SPAN style="color: lime;">TA</SPAN></B>CCAAGACCGAGACCCTTAGTCTGGCC-5</SPAN></P><P class="MsoNormal" style="text-align: justify;"><SPAN style="" lang="EN-GB"><SPAN style="text-decoration: underline;">Cloning</SPAN></SPAN></P><P class="MsoNormal" style="text-align: justify;"><SPAN style="" lang="EN-GB">However, at this stage the first
problem
appeared. These are the oligonucleotides, each
120 nt long, which we ordered from Mr.Gene. Unfortunately, we made a mistake when
ordering the
complementary oligonucleotide. Accidentally we introduced two mismatches
(compare
colored bases in the sequence). But somehow we managed to dimerize the
two
oligonucleotides. After dimerization in the thermocycler
and
cloning into a pMA (BBa_K243031) vector (XbaI/AgeI) some mutations
appeared in
the 36GSLinker gene caused by the mismatches. As there was no
frame
shift we decided to carry on. 
First, we picked the parts we were going to
use. We decided to use one we already completed in our earlier
experiments: His
– FluA –<ST1:PLACE w:st="on"><ST1:CITY w:st="on">Split</ST1:CITY></ST1:PLACE> -
Foki (BBa_K243010) had all the functions we needed. These are (i) a purification tag, 
(ii) a Fok domain and (iii) the anticalin that binds the
tagged
oligonucleotide (Fig. 1). Cloning the linker (no part) behind these parts was
the next
step. We followed the assembly standard 25 and opened the vector with AgeI
and
PstI and cut the inserted with NgoMIV and PstI.</SPAN></P><P class="MsoNormal"><SPAN style="font-size: 10pt;" lang="EN-GB">Fig. 3: Gel
picture of test digest</SPAN></P>
However, we ran into some more problems. After cloning Foka
(BBa_K243000) after the other parts, again assembly standard 25, we
performed test restriction digests cutting out the insert to confirm its size. These digests showed that the
plasmid
became even smaller as it would be without an insertion (pEX: ~ 4,5kbp
and after
test digestion ~ 2,2kbp). The insert should have a size of about 1.9 kbp but was found to be only about 900 bp (Fig. 3). <SPAN style=""> </SPAN>The sequencing did not work either.<P class="MsoNormal" style="text-align: justify;"><SPAN style="" lang="EN-GB">Most likely, the active Fok did
cut the plasmid
and promoted deletion of the Fok gene, which gave such mutated cells a
significant
growth advantage. After we encountered these problems, we decided to order the 36GS-linker (GSAT
Linker:
BBa_K243029) as gene synthesis.</SPAN></P><P class="MsoNormal" style="text-align: justify;"><SPAN style="" lang="EN-GB"><O:P> </O:P>5´-GAGCTCGAATTCGCGGCCGCTTCTAGATGGCCGGCGGTGGTTCTGCCGGTGGCTCCGGTTCTGGCTCCAGCGGTGGCAGCTCTGGTGCGTCCGGCACGGG
TACTGCGGGTGGCACTGGCAGCGGTTCCGGTACTGGCTCTGGCACCGGTAATACTAGTAGCGGCCGCTGCAGGGTACC-3´</SPAN></P><P class="MsoNormal" style="text-align: justify;"><SPAN style="" lang="EN-GB">This
time the order was well planned but, unfortunately, it
took over 4 weeks for the synthesis and delivery, and the genes arrived too
late. Once
again we have to make all the cloning steps. As of mid October 2009, the
project
is still in progress.</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="art-Footer"><DIV class="art-Footer-inner"><DIV class="art-Footer-text"><P>contact: <A href="mailto:freigem09@googlemail.com">freigem09@googlemail.com</A></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Freiburg_bioware/Project/fokmonomer">http://2009.igem.org/Team:Freiburg_bioware/Project/fokmonomer</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_bioware/Project/fokmonomer" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_bioware/Project/fokmonomer" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project/fokmonomer&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_bioware/Project/fokmonomer&amp;oldid=165082" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>