"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg bioware/Project/AGO
From 2009.igem.org
FREiGEM
Home
The Team
Overview
Portraits
The Project
Summary
Highlights
Human Practice
Ethics
Safety
Notebook
Parts
Basic Parts
Composite Parts
Collaboration
Modeling
 
Argonaute Proteins
The argonaute proteins represent one of our side projects in creating a universal programmable endonuclease.
Abstract:
The proteins of the argonaute family are the key players in RNA interference, where they cut and degrade mRNA. Therefore getting loaded with a siRNA or miRNA that is the inverse complement of the target sequence is a necessary step. The RNA-induced silencing complex, or RISC as the complex of the siRNA and the argonaute protein is often referred to, has been shown to cut the target mRNA exactly after the 10th base of the guide si-/miRNA. There seem to be no extensive RNA-specific interactions, so in theory, the RISC complex should be able to cut ssDNA as well. Then, using two inverse complement guide oligonucleotides combined with a cycle of heat denaturation to make the DNA duplex accessible to the RISC should allow its use as an endonuclease, whose cutting site has been determined by the oligos complementary to the target sequence.
Introduction:
The RISC (RNA-induced silencing complex) is one of the most important key players in post transcriptional gene regulation. The DICER complex processes dsRNA or short intramolecular double-stranded regions of pre-miRNA by cutting it down to 20-25 nt fragments. 21 nt long ssRNA fragments, then called siRNA, can be loaded into argonaute proteins, where it serves as a guide oligonucleotide.  This complex is then referred to as the RISC. The RISC is then ready to cut any RNA that has complementary sequence to the loaded guide oligonucleotid.
Cleavage of the target RNA has been shown to occur exclusively after the 10th base of the guide oligonucleotide counted from its 5’ end, ensuring both sequence as well as cleavage position specificity.
As there is no evidence of extensive interactions with the 2’-OH of the target RNA ribose part of the target, as well as the one of the guide oligonucleotide, it should be possible to cut ssDNA as well. It has indeed been shown, that a ssDNA 21mer (5’-phosphorylated) is even more efficient in acting as the guide than the naturally occurring siRNA.  Combination of two complementary 21mer guide oligonucleotides with a heat denaturation step converting the target DNA into the single stranded form should in theory enable the argonaute protein to catalyze a dsDNA cleavage event at any chosen site.
Results and Discussion:
We were very thankful to receive two vectors containing argonaute protein genes, namely the ones of the thermophilic bacteria Aquifex aeolicus and Thermus thermophilus, from Stefan Juranek of the Howard Hughes Medical Institute Laboratory of RNA Molecular Biology Rockefeller University. After expression and purification of the proteins, we first tried to cut ssDNA produced by M13 bacteriophages, but had little success - probably due to the extensive secondary structure of the DNA. We spared to do some RNA cleavage assays, because we would have had to establish all the arrangements for handling RNA first, which would have exceeded the capacities we were able to provide for this side project. Next we tried to generate ssDNA using a modified thermocycler protocol, where we used just one primer and a digestion product as a dsDNA template to ensure the production of ssDNA of a determined length. The product was visualized on an agarose gel and purified prior to the assay.
In parallel we were able to establish a phage display system, where the Aquifex aeolicus argonaute protein was displayed on M13 phages. Therefore we created a library of the argonaute protein gene containing multiple point mutations introduced by error prone PCR, that were inserted into a phagmid vector.
We hope to increase the argonaute protein’s affinity towards ssDNA as well as the ssDNA cleavage catalysis, by selecting phages bearing argonaute proteins, with that features out of the library.
Methods:
Purification of the protein was achieved by Ni NTA column and size exclusion chromatography (see “protein expression and purification”).
 ssDNA was produced by prepearation of genomic ssDNA of M13 as well as by a modified thermocycler protocol, similar to pcr but with just one primer and a digestion product as template to ensure constant length of the cycle product.
The cleavage test assay was performed analogous to the RNA cutting assays performed by Wang et al in “Structure of an argonaute silencing Complex with a seed-containing guide DNA and target RNA duplex”, Nature 456, 921-926 (18 December 2008) : “Recombinant T. thermophilus Ago (1 mM final concentration) was incubated with a reaction mixture containing 10mM HEPES-KOH, pH7.5, 100mM NaCl, 5mM MnCl2, and 0.5 mM guide strand for 30min at 55 °C in a final volume of 15 ml…””…RNA substrate at a final concentration of 0.1 mM was added. The incubation was continued for 30min at either 55 or 70 °C. The reaction was stopped by addition of 185 ml proteinase K solution (1mg/ml proteinase K, 20mM HEPES-KOH, pH7.5, 1.5mM EDTA, 100mM NaCl,1.5mM CaCl2 and 1.5% SDS), and incubated at 55 °C for 10min…” Products were analyzed on polyacrylamide gel.
The argonaute protein genes were transformed in a library by error prone PCR using Taq-polymerase at high salinity. The product was inserted into a phagmid vector that generated a fusionprotein containing the argonaute protein, the N-terminal part of the gene-3-proteine as well a signal sequence to export the protein into the periplasm.
Display was controlled using the same phagmid with the wt Aquifex aeolicus argonaute protein was fused with an additional FLAG-tag. An ELISA assay was carried out using surface coupled anti-FLAG antibodies to bind the argonaute presenting phages. The phages then were detected using anti-M13 antibodies that were coupled to horseradish peroxidase. Peroxidase activity was detected using ABTS and absorbance measurement at 405 nm.
Panning is being performed by incubating the argonaute presenting phages with a 21mer just like in the cleavage assays at the moment. A complemantary biotinylated target oligonucleotid is coupled to streptavidin on an immunotube. Phages not binding tho ssDNA oligonucleotid will be discarded during the washing steps. Those able to bind will be eluded with a buffer containing the cation neccasary for the cleavage activity, so argonaute proteins able to cut ssDNA should be eluded here. The other proteins, just able to bind, but not to cut ssDNA should be eluded by a buffer containing DNaseI (by cutting the ssDNA link to the immunotibe).
contact:  freigem09@googlemail.com
Retrieved from " http://2009.igem.org/Team:Freiburg_bioware/Project/AGO "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
