"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg bioware/Project/fokmonomer
From 2009.igem.org
FREiGEM
Home
The Team
Overview
Portraits
The Project
Summary
Subprojects
Highlights
Human Practice
Ethics
Safety
Notebook
Parts
Basic Parts
Composite Parts
Collaboration
Modeling
 
Fok Monomer
Introduction
 
In order to create a universal restriction enzyme, we followed several ideas and models. The first and most promising idea we had was to create a monomeric restriction enzyme as it represents the optimal and easiest model for a universal restriction enzyme
If we are able to employ a monomeric enzyme, this protein would have a couple of advantages: Most importantly we would no longer need two separate oligonucleotides to achieve specific binding and cleavage of the target DNA. Also, only one protein has to be purified – thus saving time and money. A scientific advantage is the option to optimize the monomer by phage display. Using this technology, we would have the chance to create a thermostable, specific, universal restriction enzyme whose DNA binding specificity is simply created by a single oligonucleotide.
Our goal was first, to create a Fok-monomer that is able to cut DNA without a primary dimerization step, and second, to show that our heterodimeric interface design works properly. To reach this, we had to clone all the required parts one after another, resulting in a huge fusion protein.
Results
3D modeling and design goals
Fig. 1:
Scheme of monomoeric Fok:
I.
lipocalin,  II. Foka/Foki (heterodimers), III. Linker,  IV. His-Tag
The necessary parts in the model of a monomeric restriction enzyme are: Tag-binding protein (lipocalin) - Fok heterodimer1 - linker - Fok heterodimer2 (Fig. 1).
 
Tag
binding protein
linker
Fok heterodimer
HisTag (BBa_K157011)
StrepTag(BBa_K157012)
FluA(BBa_K157004)
DigA(BBa_K243003)
Fos(BBa_K243027)
GSAT-Linker(BBa_K243029)
SEGLinker(BBa_K243030)
FokA (K243000)
Foki (K243001)
Tab. 1: List of parts usable for Fok Monomer
In order to purify this fusion protein, we employed an N-terminal tag. The binding protein (a lipocalin-derived binding protein referred to as anticalin in analogy to antibodies) mediates  between the FokI protein and the tagged oligonucleotide. Both parts of the Fok heterodimers are associated with a long flexible linker allowing them to establish the cutting site conformation. This artificial restriction nuclease domain is the functional part of the monomer and catalyzes the DNA cleavage.
Before we started in the wet lab, we thoroughly planned all the individual steps. To figure out the best way to create the Fok monomer, we first designed it in silico.
Fig. 2: 3D model of Fok Monomer
The model (Fig. 2) shows that we have to use a linker which is about 70 Angstrom in length to span the required distance between the two Fok parts. Therefore, we decided to order two complementary oligonucleotides encoding a 36 amino acid long glycine-serine linker.
 
5'-CTAGATGGCCGGCGGTTCTGGTGGTGGTTCTGGCGGTGGTTCTGGAGGTAGTTCTGGCGGTGGATCTGGAGGCGGTTCTGGGTCAGGATCTGGTGGAGGTTCTGGCTCTGGGAATCAGA-3'          3'-TACCGGCCGCCAAGACCACCACCAAGACCGCCACCAAGACCTCCATCAAGACCGCCACCTAGACCTCCGCCAAGACCCAGTCCTAGACCACTACCAAGACCGAGACCCTTAGTCTGGCC-5
Cloning
However, at this stage the first problem appeared.
These are the oligonucleotides, each 120 nt long, which we ordered from Mr.Gene. Unfortunately, we made a mistake when ordering the complementary oligonucleotide. Accidentally we introduced two mismatches (compare colored bases in the sequence). But somehow we managed to dimerize the two oligonucleotides.
After dimerization in the thermocycler and cloning into a pMA (BBa_K243031) vector (XbaI/AgeI) some mutations appeared in the 36GSLinker gene caused by the mismatches. As there was no frame shift we decided to carry on.
First, we picked the parts we were going to use. We decided to use one we already completed in our earlier experiments: His – FluA –
Split
- Foki (BBa_K243010) had all the functions we needed. These are (i) a purification tag,  (ii) a Fok domain and (iii) the anticalin that binds the tagged oligonucleotide (Fig. 1). Cloning the linker (no part) behind these parts was the next step. We followed the assembly standard 25 and opened the vector with AgeI and PstI and cut the inserted with NgoMIV and PstI.
Fig. 3: Gel picture of test digest
However, we ran into some more problems. After cloning Foka (BBa_K243000) after the other parts, again assembly standard 25, we performed test restriction digests cutting out the insert to confirm its size. These digests showed that the plasmid became even smaller as it would be without an insertion (pEX: ~ 4,5kbp and after test digestion ~ 2,2kbp). The insert should have a size of about 1.9 kbp but was found to be only about 900 bp (Fig. 3).  The sequencing did not work either.
Most likely, the active Fok did cut the plasmid and promoted deletion of the Fok gene, which gave such mutated cells a significant growth advantage. After we encountered these problems, we decided to order the 36GS-linker (GSAT Linker: BBa_K243029) as gene synthesis.
 
5´-GAGCTCGAATTCGCGGCCGCTTCTAGATGGCCGGCGGTGGTTCTGCCGGT
GGCTCCGGTTCTGGCTCCAGCGGTGGCAGCTCTGGTGCGTCCGGCACGGG
TACTGCGGGTGGCACTGGCAGCGGTTCCGGTACTGGCTCTGGCACCGGTA
ATACTAGTAGCGGCCGCTGCAGGGTACC-3´
This time the order was well planned but, unfortunately, it took over 4 weeks for the synthesis and delivery, and the genes arrived too late. Once again we have to make all the cloning steps. As of mid October 2009, the project is still in progress.
contact:  freigem09@googlemail.com
Retrieved from " http://2009.igem.org/Team:Freiburg_bioware/Project/fokmonomer "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
