"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg bioware/Project/3d-modeling
From 2009.igem.org
FREiGEM
Home
The Team
Overview
Portraits
The Project
Summary
Highlights
Human Practice
Ethics
Safety
Notebook
Parts
Basic Parts
Composite Parts
Collaboration
Modeling
 
3D-Modeling
Structure Based Design of a Programmable Restriction Enzyme
 
 
Introduction
 
To construct the universal restriction enzyme we had in mind, it was necessary to understand the 3dimensional structure of enzymes which are able to cut DNA.
Prerequisite for our  restriction enzyme design was a DNA cutting site separated from its DNA recognition site. Many enzymes have both in the same spot and therefore it is very difficult to modify the recognition pattern without destroying the enzyme´s catalytic activity. However, there is one enzyme that perfectly matched our needs, namely FokI (Fig.1).
 
 
3d model of the cleavage domains
 
The idea was to modify FokI in a way that enables us to make the protein bind single strand DNA at a user defined site and consequently to cut DNA precisely. So we searched for all structural data available. The crystallized FokI-homodimer (pdbID: 2FOK), published by Wah and colleagues (Wah et al 1998), gave us the initial clue to start in silico modeling. We used the program PyMOL (DeLano W.L. 2002)
FokI functions as a dimer, however, only one of the identical proteins binds to DNA with its DNA binding domain. Since there is no use for a specific DNA binding domain in a universal restriction enzyme, we decided to use both cleavage domains separately and to delete the binding domains. The two cleavage domains from amino acid 387 to 579 remained (Fig.1). They appeared to be hydrophilic on the surface, so nothing needed to be modified, except cystein 541 which was replaced by a serine. Cysteins might interfere when the protein is expressed in the periplasm of e.coli. The catalytically active amino acids (Fig.2 green) each conjugate two Ca2+ ions to brake DNA backbone. Mechanism reviewed by Pingoud 2005.
Fig1. FokI-Dimer: Grey: DNA binding domains,(amino acids 1 – 386); Red and Green: catalytic domains (amino acids 387 to 579.)
Creating the heterodimer Interface
 
To avoid unspecific cleavage of DNA by unbound catalytic FokI domains during recombinant expression of the protein, we were looking for a control feature. In theory this was easy to manage by switching the homodimer into a heterodimer. In a Fok heterodimer, each active part only matches with its complementary counterpart. We reasoned that separate expression of each monomer in different E. coli cultures would avoid formation of the active FokI cleavage domains and thus diminish off site celavage. The heterodimer interface was designed according to work published by Miller (Miller et al. 2007). In the catalytically active heterodimer we switched glutamate 490 to lysin (Base Triplet GAA->AAA) and isoleucin 538 to lysin (Base Triplet ATC->AAA). In the catalytically inactive heterodimer we switched glutamin 486 to glutamate (Base Triplet CAA->GAA) and isoleucin 499to leucin (Base Triplet ATC->CTG).
 
 
Another advantage of creating two distinct parts is that we can exactly define which one will cut the ssDNA and which one will only have a stabilizing function. The stabilizing one will be located above the complementary strand of the DNA that should not be cut. Therefore we exchanged its catalytically active amino acids into neutral ones based on the work of Wah (Wah 1997).We replaced amino acids aspartate 450 with alanin (Base Triplet GAC->GCG) and aspartate 467 with alanin (GAT->GCG) (Fig.2).
Fig.2 two FokI cleavage domains (amino acid 387 to 579). Red: Catalytically active FokI cleavage domain; Green: Catalytically inactive FokI cleavage domain; Pink: catalytically active aminoacids; Yellow: glutamate 490 and isoleucin 538; Blue: glutamin 486 and isoleucin 499.
3D model of the DNA
 
The next step was to determine optimal positioning of the DNA relative to the protein to mediate desired cleavage. We superimposed structures as described by Wah 1998 and aligned the structure of BamHI (pdbID: 2BAM) (Viadiu et al 1998) with our two domains. In fact, only the parts of the protein that are responsible for cleavage are similar. We picked four matching amino acids that were in close proximity to the catalytically active amino acids for the structural alignment. In BamHI these are Asp-94 and Val-108 and in FokI these are Asp-450 and Val-164. Each aminoacid was selected two times, since the structures are dimers.
After alignment we kept the Ca2+ ions and the DNA that were crystallized together with both BamHI domains and deleted the rest of the BamHI file from the model.
The DNA part was too short to work with it, so we created a 50Bp long DNA with the DNA origami software SARSE (Anderson et al.) and replaced the BamHI DNA with this 50bp DNA by aligning the phosphates of the DNA backbone (Fig.4 DNA cyan, Oligonucleotide magenta).
 
 
Determination of length and tag positions of the oligonucleotides
 
The next task was to connect the cleavage domains with a construction that directs them exactly to that area of the ssDNA one wishes to cut.
As outlined in the general introduction we had the following plan: the user defines oligonucleotides complementary to the area of the cutting site. These oligonucleotides must be marked with a fluorescein and a digoxigenin in a predefined distance from the position that is to be cut. These tags are linked to the oligonucleotide by C6 linkers (having the length of 6 carbons) (Fig.4 yellow). They will be recognized by specially designed lipocalins that are attached to the cleavage domain. Using the chemical modeling software ACD/ChemSketch (http://www.acdlabs.com/download/chemsketch) we generated a 3D model of the C6 linker and manually fitted in the correct position in the model.
Fig.3 Oligonucleotide scheme. The picture shows symbolically the ssDNA in blue and its complementary strand that will be the oligonucleotides in red. The DNA sequence is a random example. The bars indicate templates for oligonucleotides with several options for their position relative to the cutting site. Pink: the position where the universal restriction enzyme based on FokI will cut (on the right side of the marked base); Yellow left: position of the fluorescein tagged base; Yellow right: position of the digoxigenin tagged base;
Upper bar:  dioligo AB 16 bases with 5' fluorescein tag,; diologo A 16 bases with 5' tag digoxigenin, the planned cut site is between base 8/9 from the 5' end;
Middle bar: monooligo 30 bases with 3’ fluorescein tag and internal digoxigenin modification 14 bases from 5', the planned cut site ist between base 21/22 from its 5' end;
Lower bar: dioligoAB; dioligo B with internal digoxigenin modification15 bases from the 5'end, the planned cut is between base 22/23 from 5' end, 30bp long;
Visual inspection revealed that the FokI cleavage domains in binding position occupy two opposing minor grooves of a helical DNA. It seemed reasonable to arrange the lipocalins flanking the cleavage domains on the outside, right above the neighboring major grooves (Fig.5). By this arrangement the shortest possible length of the oligonucleotides and the position of the tags were defined. From the cutting site towards the 5´ end of the ssDNA we determined a distance of about 30nm which can be spanned by e.g. 11 amino acids. This means the complementary oligonucleotide (Fig.3 dioligo AB) has to have its fluorescein tag on its 5´ position, the oligonucleotide however can have any length in the  3´ direction (here we chose 16bp). The second oligonucleotide (Fig.3 dioligo A) with a digoxigenin tag is located next to the first, further towards the 3´ end of the ssDNA. Its tag must be located 6 bases further towards the 3´ end of the ssDNA. Since it is much cheaper to buy oligonucleotides tagged on 3´ or 5´ end we designed the second oligonucleotide with 16 bases and its tag also on the 5´ end (Fig.3 and 4).
In total the hybridized DNA should be around 30bp long to ensure its folding into an helical state at room temperature.
Fig.4 DNA, Oligonucleotides and FokI domains. Cyan: ssDNA; Magenta: two 16 bases long oligonucleotide; Yellow: C6 linkers, left with fluorescein, right with digoxigenin; Green: the FokI inactive domain; Red: the FokI active cleavage domain; Pink: 3 catalytically active aminoacids; Grey: two calcium ions.
Not shown are lipocalins.
We also designed further oligonucleotide variants. One includes both tags (Fig.3 monooligo) and another one (Fig.3 dioligo B) can be combined with diologo AB., However in contrast to dioligo A its tag is internal. The advantages of a single oligonucleotide is that the kinetics of the cleavage reaction would be improved, since it is one part less that has to find its position to initiate the reaction. The advantage of an extended second oligonucleotide is that it can work under higher temperatures, what also speeds up the reaction.
 
 
Lipocalins as connectors between oligonucleotides and FokI cleavage domains
 
Lipocalins, also called Anticalins, are 20 kDa small proteins that can be designed to bind specific substances. We used an anticalin against digoxigenin (pdbID: 1NOS; Fig.5 orange) (Korndorfer 2003a) and another against fluorescein (pdbID: 1LKE; Fig.5 blue) (Korndorfer 2003b).
They were loaded into the model and fitted manually above the major DNA grooves next to the FokI cleavage domains (Fig. 4). C-termini of the lipocalins show towards the N-termini of the cleavage domains. To connect each lipocalin with each cleavage domain we designed several linkers (see parts). These are not shown in the model. The linkers need to be long enough to allow easy binding of the FokI cleavage domains, but must be tight enough to ensure exact cutting. The fluorescein binding lipocalin needs to be connected to the inactive FokI cleavage domain, the digoxigenin binding one to the active domain.
These lipocalins were readily available from the
Freiburg
iGEM team 2008.
Fig.5 complete universal restriction enzyme. Cyan: ssDNA; Magenta: two 16 bases long oligonucleotide; Yellow: C6 linkers, left with fluorescein, right with digoxigenin; Green: the FokI inactive domain; Red: the FokI active cleavage domain; Pink: 3 catalytically active aminoacids; Grey: two Calcium ions;
Orange
: digoxigenin binding lipocalin; Blue: Fluorescein A binding lipocalin
References
 
Andersen DNA Origami program SARSE
www.sarse.org/index.php/DNA_origami
 
Korndorfer IP, Schlehuber S, Skerra A 2000a Structural mechanism of specific ligand recognition by a lipocalin tailored for the complexation of digoxigenin J. Mol. Biol. v330, p.385-396
 
Korndorfer IP, Beste G, Skerra A 2000b Crystallographic analysis of an "anticalin" with tailored specificity for fluorescein reveals high structural plasticity of the lipocalin loop region Proteins v53, p.121-129
 
Miller, J.C., Holmes, M.C., Wang, J., Guschin, D.Y., Lee, Y.L., Rupniewski, I., Beausejour, C.M., Waite, A.J., Wang, N.S., Kim, K.A., Gregory, P.D., Pabo, C.O., & Rebar, E.J. (2007) An improved zinc-finger nuclease architecture for highly specific genome editing, Nat Biotechnol 25, 778-785
 
A. Pingoud, M. Fuxreiterb,V. Pingoud and W.Wende Type II restriction endonucleases: structure and mechanism, Review, Cell. Mol. Life Sci. 62 (2005) 685–707
 
DeLano
, W.L. The PyMOL Molecular Graphics System (2002) on World Wide Web http://www.pymol.org
 
Viadiu H,
Aggarwal
AK
The role of metals in catalysis by the restriction endonuclease BamHI Nat. Struct. Biol. v5, p.910-916 (1998)
 
Wah DA, Bitinaite J, Schildkraut I, Aggarwal AK Structure of FokI has implications for DNA cleavage Proc. Natl. Acad. Sci. U. S. A. v95, p.10564-10569 (1998)
 
David A. Wah, Joel A. Hirsch*, Lydia F. Dorner†, Ira Schildkraut† & Aneel K. Aggarwal*; Structure of the multimodular endonuclease FokI bound to DNA, NATURE |VOL 388 | 3 JULY 1997
contact:  freigem09@googlemail.com
Retrieved from " http://2009.igem.org/Team:Freiburg_bioware/Project/3d-modeling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
