<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Arizona_State_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Arizona State/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="/Team:Arizona_State">ASU</A></DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="/Team:Arizona_State/Team">About</A></LI><LI><A href="/Team:Arizona_State/Collaborations">Collaborations</A></LI></UL><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Arizona_State/Description">Description</A></LI><LI><A href="/Team:Arizona_State/Experiments"> Protocols</A></LI><LI><A href="/Team:Arizona_State/Results"> Results </A></LI><LI><A href="/Team:Arizona_State/Notebook"> Notebook </A></LI><LI><A href="/Team:Arizona_State/Safety"> Safety </A></LI><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="/Team:Arizona_State/Parts">Parts and Design</A></LI><LI><A href="/Team:Arizona_State/Basic_Part">Basic Parts </A></LI><LI><A href="/Team:Arizona_State/Composite_Part"> Composite Parts </A></LI><LI><A href="/Team:Arizona_State/Part_Collection"> Part Collection </A></LI><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="/Team:Arizona_State/Human_Practices"> Human Practices </A></LI><LI><A href="/Team:Arizona_State/HP/Silver"> Practices:Silver </A></LI><LI><A href="/Team:Arizona_State/HP/Gold"> Practices:Gold </A></LI><LI><A href="/Team:Arizona_State/Integrated_Practices"> Integrated Practices </A></LI><LI><A href="/Team:Arizona_State/WhitePaper"> White Paper </A></LI><LI><A href="/Team:Arizona_State/Attributions"> Attributions </A></LI><LI><A href="/Team:Arizona_State/Awards"> Achievements</A></LI></DIV></DIV><DIV class="container"><H2>The Problem</H2><UL><LI>There are only around 4 widely-used N-acyl homoserine lactone (AHL) quorum sensing systems, while there are over 100 different systems found in nature</LI><LI>AHL molecules are not completely system-specific, as crosstalk can occur between systems-this causes problems with developing functional genetic circuits</LI><LI>AHL quorum sensing is used to regulate factors such as virulence and biofilm formation, potentially activating pathogenic bacteria</LI></UL><H2>Brief Summary</H2><P> Quorum sensing is the ability of bacteria to scan their surroundings and detect concentrations of their own population. A species of bacteria will produce an inducer protein, which generates chemical signals in the form of N-acyl homo-serine lactones (AHLs). When the AHL reaches a high concentration, the bacterial cells will respond by collectively activating a set of genes. In nature, these AHLs are able to drastically influence the growth behavior of bacterial cells, activating biofilm formation, bioluminescence, virulence, motility, etc. </P><P> The objective of our project is to design and test a variety of quorum sensing networks. We have developed a flexible testing platform in which the QS system is separated into two components designated the “Sender” and the “Receiver”. The AHL synthase is expressed in the Sender cell, while the inducible promoter and regulator are carried by a Receiver cell. When the Sender produces a signal, the HSL, it diffuses across cell membranes and activates the Receiver. In our current system, Receivers will express green fluorescent protein (GFP) in response to induction by Senders from different bacterial species. Ideally, the designed systems would have low amounts of interference and form a functional genetic circuit. Our team has built 10 senders and their respective receivers. </P><P> Our iGEM team is investigating the diverse applications that fit with our quorum sensing project. Some of the sub-projects include: investigating the Aub strain, which originates from unidentified soil bacterium; writing a white paper on addressing potential safety concerns with AHLs, a visual optical density test on AHL production by Senders, a more comprehensive characterization of AHLs produced by our Senders using gas chromatography, a direct comparison induction test on Senders using the well-characterized part F2620,  and develop comprehensive safety procedures for the handling of AHLs. The white paper and safe disposal protocol are several of our human practices projects that the ASU team investigated during the course of the project. </P><H2>MOTIVATION</H2><P> N-acyl homoserine lactone (AHL) quorum sensing (QS) is a type of communication that allows bacteria to monitor their population density for the purpose of controlling various group activities such as virulence or luminescence. Synthetic biologists have taken advantage of the simplicity of the QS system to incorporate signal-processing pathways into genetic circuits. This project aims to determine pathways with minimal overlap (“crosstalk”) and engineer QS modules as a flexible tool for building layered genetic circuits. Crosstalk occurs when when a single regulator is activated by multiple varieties of HSL molecules, impeding successful operation of complex genetic circuitry that uses multiple quorum sensing pathways. Homologous AHL networks have been identified in over 100 species of bacteria, but only four have been used in synthetic systems reported to date. Our goal is to expand the QS toolbox and enable the implementation of higher-order, complex genetic circuitry in synthetic biology.</P><P>In addition to expanding the QS toolbox, our team felt that an in-depth investigation on the safety of AHLs would be valuable to future quorum sensing research. From the interviews and questionnaires that we conducted to industry experts, it was clear that AHLs were not a focal point in their safety reviews. However, from literature reviews, we found that there are many opportunistic pathogens that utilize AHL to activate virulence. A brief list is shown below:</P><P>Because there are potential pathogens that could become activated, proper safety measures in the development and production of AHLs were necessary. In our Human Practices, we gathered information about the dangers of AHLs from experts and literature, developed a safe disposal plan, tested the safe disposal plan as an extension of our F2620 induction tests, and wrote a report on suggestions for the safety of AHLs. In addition, we added a safety section to our parts pages that we submitted to the registry. </P></DIV><DIV class="container"><H2>N-Acyl Homoserine Lactones</H2><P>AHL quorum sensing has a myriad of different systems. A total of 10 systems were investigated in this project </P><TABLE style="width:100%"><TBODY><TR><TH>AHL System</TH><TH>Bacteria of Origin</TH><TH>AHL Name</TH><TH>3D-Model</TH></TR><TR><TD>Aub</TD><TD>Unknown</TD><TD>N-(2-oxooxolan-3-yl)dodecanamide</TD></TR><TR><TD>Bja</TD><TD>Bradyrhizobium japonicum</TD><TD>3-methyl-N-[(3S)-2-oxooxolan-3-yl]butanamide</TD></TR><TR><TD>Bra</TD><TD>Paraburkholderia kururiensis</TD><TD>(3S)-3-[(2-oxo-3-phenylpropyl)amino]oxolan-2-one</TD></TR><TR><TD>Cer</TD><TD>Rhodobacter sphaeroides</TD><TD>(Z)-3-hydroxy-N-[(3S)-2-oxooxolan-3-yl]tetradec-7-enamide</TD></TR><TR><TD>Esa</TD><TD>Erwinia stewartii</TD><TD>3-oxo-N-[(3S)-2-oxooxolan-3-yl]hexanamide</TD></TR><TR><TD>Las</TD><TD>Pseudomonas aeruginosa</TD><TD>3-oxo-N-(2-oxooxolan-3-yl)dodecanamide</TD></TR><TR><TD>Lux</TD><TD>Vibrio fischeri</TD><TD>3-oxo-N-(2-oxooxolan-3-yl)hexanamide</TD></TR><TR><TD>Rhl</TD><TD>Rhizobium leguminosarum</TD><TD>N-(2-oxooxolan-3-yl)butanamide</TD></TR><TR><TD>Rpa</TD><TD>Rhodopseudomonas palustris</TD><TD>(S)-α-amino-γ-butyrolactone</TD></TR><TR><TD>Sin</TD><TD>Sinorhizobium meliloti</TD><TD>N-[(3S)-2-oxooxolan-3-yl]octanamide*</TD></TR></TBODY></TABLE><P>*Sin system produces 6 different variants of AHL. The 3D structures of all the Sin compounds can be found <A href="https://www.dropbox.com/sh/cfo744xq81t6iu4/AAAAG8yiFkHfnUojBZE7bWKla?dl=0">here</A>.</P><P>AHLs share the same basic structure, with a lactone ring, an N-acyl and ketone group. The defining R group lies in the acyl tail, which is the primary determinant in its transcription factor binding affinity. The graphic below demonstrates the categorization of the AHLs produced by the 10 studied systems</P><H2>F2620 Inductions</H2><P>The ASU team would like this experiment to be considered for the Gold Medal Requirement of improving the characterization of a previously existing BioBrick part. All 10 systems were studied in an induction test. The part BBa_F2620 (designed by Barry Canton from MIT) was used to induce production in the Lux AHL system and test induction in any other AHL systems. By studying interactions between the 10 constructed Senders and F2620, we were able to analyze the systems for potential orthogonality. The resulting part collection allows direct comparison in AHL induction between multiple systems. The Part Numbers for these Senders are 
<A href="http://parts.igem.org/Part:BBa_K2033000">BBa_K2033000</A>, 
<A href="http://parts.igem.org/Part:BBa_K2033002">BBa_K2033002</A>,
<A href="http://parts.igem.org/Part:BBa_K2033004">BBa_K2033004</A>,
<A href="http://parts.igem.org/Part:BBa_K2033006">BBa_K2033006</A>, 
<A href="http://parts.igem.org/Part:BBa_K2033008">BBa_K2033008</A>, which correspond to the Aub, Bja, Bra, Cer, and Sin systems.  We believe that this Part Collection is a valuable addition to the registry, because while small in size, it contributes 5 Senders to the registry which contains around 6 Senders. In addition, this will add information about these new systems to a well-characterized and widely-used part in F2620. Not only will this determine interactions between systems, but also, provide characterization information for any future quorum sensing research involving those systems.</P><P>We also created Senders for the Esa, Las, Lux, Rhl, and Rpa systems, which already exist in the registry. These correspond to the following parts:</P><UL><LI>EsaI-<A href="http://parts.igem.org/Part:BBa_K1670004">BBa_K1670004</A></LI><LI>LasI-<A href="http://parts.igem.org/Part:BBa_C0178">Bba_C0178</A></LI><LI>LuxI-<A href="http://parts.igem.org/Part:BBa_C0161">BBa_C0161</A></LI><LI>RhlI-<A href="http://parts.igem.org/Part:BBa_C0170">BBa_C0170</A></LI><LI>RpaI-<A href="http://parts.igem.org/Part:BBa_K1421006">BBa_K1421006</A></LI><P>We added the induction data that we collected for these systems to their respective parts pages, which provides information on their compatibility with F2620.</P><DIV class="container"><H2>REFERENCES</H2><OL><LI>Davis, René Michele, Ryan Yue Muller, and Karmella Ann Haynes. &quot;Can the Natural Diversity of Quorum-Sensing Advance Synthetic Biology?&quot; Frontiers in Bioengineering and Biotechnology 30th ser. 3 (2015)</LI><LI>Borchardt, S.A., Allain, E.J., Michels, J.J. “Reaction of Acylated Homoserine Lactone Bacterial Signaling Molecules with Oxidized Halogen Antimicrobials” Applied and Environmental Microbiology. 67.7. (2001); 3174-3179. </LI><LI>Canton, Barry, Anna Labno, and Drew Endy. &quot;Refinement and Standardization of Synthetic Biological Parts and Devices.&quot; Nature Biotechnology 26.7 (2008): 787-93.</LI><LI>Scott, Spencer R., and Jeff Hasty. &quot;Quorum Sensing Communication Modules for Microbial Consortia.&quot; ACS Synthetic Biology 5.9 (2016): 969-77.</LI><LI>Miller, Melissa B., and Bassler, Bonnie L. “Quorum Sensing in Bacteria.” Microbiology. 55. (2001); 165-99</LI><LI>Tait, Karen, Zoe Hutchison, Fabiano L. Thompson, and Colin B. Munn. &quot;Quorum Sensing Signal Production and Inhibition by Coral-associated Vibrios.&quot; Environmental Microbiology Reports 2.1 (2010): 145-50.</LI><LI>Grant, Paul, Dalchau, Neil, Brown, James R. “Orthogonal intercellular signaling for programmed spatial behavior.” Molecular Systems Biology 12.1 (2016):849. </LI><LI>Steindler, Laura, Bertani, Iris, De Sordi, Luisa. &quot;LasI/R and RhlI/R Quorum Sensing in a Strain of Pseudomonas aeruginosa Beneficial to Plants.&quot; Applied and Environmental Microbiology. 75.15: 5131–5140. </LI><LI>Lindemann A. &quot;Isovaleryl-homoserine lactone, an unusual branched-chain quorum-sensing signal from the soybean symbiont Bradyrhizobium japonicum.&quot; PubMed. 108.40 (2011):16765-70.</LI><LI>Ahlgren NA. &quot;Aryl-homoserine lactone quorum sensing in stem-nodulating photosynthetic bradyrhizobia.&quot; PubMed. 108.17 (2011):7183-8..</LI><LI>Hoang, Hanh H. &quot;The LuxR Homolog ExpR, in Combination with the Sin Quorum Sensing System, Plays a Central Role in Sinorhizobium meliloti Gene Expression.&quot; Journal of Bacteriology. 186.16 (2004): 5460-5472. </LI><LI>Marketon, Melanie M. &quot;Characterization of the Sinorhizobium meliloti sinR/sinI Locus and the Production of Novel N-Acyl Homoserine Lactones.&quot; Journal of Bacteriology. 184.20 (2002):  5686-5695.</LI><LI>Case, Rebecca J., Labbate, Maurizio, Kjelleberg, Staffan. &quot;AHL-driven quorum-sensing circuits: their frequency and function among the Proteobacteria.&quot; The ISME Journal. 2 (2008): 345–349.</LI><LI>Winstanley, Craig, Fothergill, Joanne L.  The role of quorum sensing in chronic cystic fibrosis Pseudomonas aeruginosa infections.&quot; FEMS Microbiology Letters. (2009) 1-9. </LI><LI>Watson, William T. &quot;Structural Basis and Specificity of Acyl-Homoserine Lactone Signal Production in Bacterial Quorum Sensing.&quot; Molecular Cell. 9.1 (2002)  685–694.</LI></OL></DIV><FOOTER><DIV class="container center"><P class="copyright">
     © 2016 ASU iGEM </P></DIV></FOOTER></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>