<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Arizona_State_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Arizona State/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="/Team:Arizona_State">ASU</A></DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="/Team:Arizona_State/Team">About</A></LI><LI><A href="/Team:Arizona_State/Collaborations">Collaborations</A></LI></UL><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Arizona_State/Description">Description</A></LI><LI><A href="/Team:Arizona_State/Experiments"> Protocols</A></LI><LI><A href="/Team:Arizona_State/Results"> Results </A></LI><LI><A href="/Team:Arizona_State/Notebook"> Notebook </A></LI><LI><A href="/Team:Arizona_State/Safety"> Safety </A></LI><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="/Team:Arizona_State/Parts">Parts and Design</A></LI><LI><A href="/Team:Arizona_State/Basic_Part">Basic Parts </A></LI><LI><A href="/Team:Arizona_State/Composite_Part"> Composite Parts </A></LI><LI><A href="/Team:Arizona_State/Part_Collection"> Part Collection </A></LI><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="/Team:Arizona_State/Human_Practices"> Human Practices </A></LI><LI><A href="/Team:Arizona_State/HP/Silver"> Practices:Silver </A></LI><LI><A href="/Team:Arizona_State/HP/Gold"> Practices:Gold </A></LI><LI><A href="/Team:Arizona_State/Integrated_Practices"> Integrated Practices </A></LI><LI><A href="/Team:Arizona_State/WhitePaper"> White Paper </A></LI><LI><A href="/Team:Arizona_State/Attributions"> Attributions </A></LI><LI><A href="/Team:Arizona_State/Awards"> Achievements</A></LI></DIV></DIV><DIV class="container"><H1> Protocols</H1><H3>[0] Recipes</H3><H4>1x TAE</H4><P>1mM EDTA, 20mM acetic acid, 40mM tris</P><H4>LB Broth(Lennox)</H4><P>10g Caesin tryptone, 5g yeast extract, 5g NaCl, 1mL NaOH</P><H4>LB Plates(Lennox)</H4><P>15g Bacto-agar, 10g Caesin tryptone, 5g yeast extract, 5g NaCl, 1mL NaOH</P><H4>Gene Ruler 1kb+ DNA Ladder</H4><P>100uL Gene Ruler plus, 50uL 20x loading dye, 850uL dH2O</P><H4>Bromoblue 20x loading dye</H4><P>12mL 100% glycerol, 250mg bromophenol blue, 250mg xylene cyanol</P></DIV><DIV class="container"><H3> [1A] Polymerase Chain Reaction </H3><P>PCR is used to rapidly amplify certain segments of DNA that are selected via designed primers. This protocol was used to accomplish a variety of goals, from extraction of parts from a cloning vector to the addition of restriction cut sites at the ends of parts. </P><OL><LI>Defrost template, 2x Master Mix and primers</LI><LI>Determine desired total working volume (~25uL)</LI><LI>Label reaction tubes</LI><LI>Create below reaction mix</LI><LI>Place mixture in thermal cycler at settings shown below</LI></OL><TABLE style="width:100%"><TBODY><TR><TH>Part</TH><TH>Concentration</TH><TH>Volume for 25uL reaction</TH></TR><TR><TD>2x Master Mix</TD><TD>2x</TD><TD>12.5uL</TD></TR><TR><TD>dsDNA Template</TD><TD>-</TD><TD>1uL</TD></TR><TR><TD>Forward primer</TD><TD>-</TD><TD>1uL</TD></TR><TR><TD>Reverse primer</TD><TD>-</TD><TD>1uL</TD></TR><TR><TD>DI H2O</TD><TD>-</TD><TD>9.5uL</TD></TR></TBODY></TABLE><H4>Sample Reaction</H4><P>This is an example protocol used in the thermal cycler, durations and temperatures subject to change</P><TABLE style="width:100%"><TBODY><TR><TH>Step Name</TH><TH># of Cycles</TH><TH>Temperature(℃)</TH><TH>Duration(min)</TH></TR><TR><TD>Initial</TD><TD>1</TD><TD>98</TD><TD>3</TD></TR><TR><TD>Denaturation</TD><TD>30</TD><TD>98</TD><TD>0.5</TD></TR><TR><TD>Annealing</TD><TD>30</TD><TD>56</TD><TD>0.5</TD></TR><TR><TD>Elongation</TD><TD>30</TD><TD>72</TD><TD>0.5</TD></TR><TR><TD>Final Elongation</TD><TD>1</TD><TD>72</TD><TD>10</TD></TR><TR><TD>Storage</TD><TD>1</TD><TD>4</TD><TD>∞</TD></TR></TBODY></TABLE></DIV><DIV class="container"><H3>[1B] PCR Cleanup </H3><P>The Qiagen Qiaquick PCR cleanup kit was used to purify the thermal cycler product and remove remaining polymerases and nucleotides.</P><OL><LI>Gather the solutions from the PCR cleanup kit</LI><LI>Mix 5 times volume of Buffer PB with PCR reaction tube</LI><LI>Add 10uL 3M sodium acetate</LI><LI>Spin down the mixture in a QIAquick filter column at max speed for 60s. Discard flow-through and replace column into tube </LI><LI>Add 750uL of Buffer PE to the mixture and spin down at max speed for 60s to wash. Discard flow-through and replace column</LI><LI>Centrifuge the tube one more time to remove any remaining wash buffer</LI><LI>Place column in clean 1.5mL plastic tube</LI><LI>Add 30uL of Buffer EB to the column and let stand for 1min. Centrifuge at max speed for 1 min. </LI></OL></DIV><DIV class="container"><H3>[2A] DNA Restriction Digest</H3><OL><LI>Gather DNA sample, digestion buffer, and restriction enzymes from cold storage</LI><LI>Label tubes</LI><LI>Create the below reaction mixture </LI><LI>Place mixture in 37°C heat block for 10 minutes</LI></OL><TABLE style="width:100%"><TBODY><TR><TH>Component</TH><TH>Volume for 25uL reaction</TH></TR><TR><TD>DNA Sample</TD><TD>15uL</TD></TR><TR><TD>10x FastDigest Digestion Buffer</TD><TD>2.5uL</TD></TR><TR><TD>Restriction Enzyme 1</TD><TD>1uL</TD></TR><TR><TD>Restriction Enzyme 2</TD><TD>1uL</TD></TR><TR><TD>DI H2O</TD><TD>5.5uL</TD></TR></TBODY></TABLE></DIV><DIV class="container"><H3>[2B] Gel Verification/Extraction</H3><OL><LI>Make 1% gel by adding .6g of agarose to 60mL of 1x TAE buffer</LI><LI>Swirl in glass flask and microwave for 40s</LI><LI>Remove from microwave and swirl again. Microwave again for 40s</LI><LI>Let cool until safe to touch. Add 6uL of SYBR safe red stain</LI><LI>Pour mixture into gel mold with comb set up</LI><LI>Fill gel electrophoresis apparatus with 1x TAE </LI><LI>Remove comb from gel and submerge gel in apparatus</LI><LI>Add 4uL of 1kb+ blue ladder to the first well</LI><LI>Add samples mixed with loading dye to other wells</LI><LI>Connect negative/positive ends to apparatus. Run at 110V</LI><LI>Run for 30min-1hr. Remove and take to UV imager</LI><LI>Photograph under UV light. If doing gel extraction, use scalpel to cut out blocks of gel containing desired bands</LI><LI>Gel purify using Qiagen Gel Purification kit (the protocol can be found <A href="http://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma/Bulletin/na1111bul.pdf">here</A>)</LI><LI>(If extracting)Use Nanodrop Spectrophotometer to measure DNA concentration</LI></OL></DIV><DIV class="container"><H3>[2C] Ligation</H3><OL><LI>Retrieve digested samples(insert and vector DNA), ligase, and ligation buffer from cold storage</LI><LI>Determine a desired ratio of insert:vector molecules in reaction (2:1, or higher for higher success)</LI><LI>Set up the below reaction</LI><LI>Create a negative control with vector DNA but no insert DNA</LI><LI>Mix via flicking the tube. Incubate at room temperature for 10 min</LI></OL><TABLE style="width:100%"><TBODY><TR><TH>Component</TH><TH>Volume for 10uL reaction</TH></TR><TR><TD>Insert DNA</TD><TD>-</TD></TR><TR><TD>Vector DNA</TD><TD>-</TD></TR><TR><TD>2x Rapid Ligation Buffer</TD><TD>5uL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1uL</TD></TR><TR><TD>DI H2O</TD><TD>-</TD></TR></TBODY></TABLE></DIV><DIV class="container"><H3>[2D] Transformation(for ligated samples)</H3><OL><LI>Retrieve ligation sample, and enough plates for all samples and a control</LI><LI>Warm up plates in 37°C</LI><LI>Retrieve competent cells from freezer and store in ice</LI><LI>Mix total ligation mixture with 50uL of cells in a tube. Mix via pipetting</LI><LI>Incubate on ice for 10 minutes</LI><LI>Place tubes in 42°C heat block for 45 seconds to heat shock, then put them back on ice for 1 minute</LI><LI>Add 750uL SOC media to tubes</LI><LI>Tape tubes to 37°C shaking incubator for 45 minutes to recover</LI><LI>Remove from incubator and pellet in centrifuge. Discard the supernatant</LI><LI>Add 100uL of LB broth with antibiotic that matches selected plates</LI><LI>Pipette total mixture onto plates with antibiotic. Spread with glass beads</LI><LI>Incubate overnight at 37°C</LI></OL></DIV><DIV class="container"><H3>[3] Miniprep</H3><P>The miniprep kit that was used was the Sigma-Aldrich GenElute Plasmid Miniprep Kit. The used protocol can be found 
<A href="https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma/Bulletin/pln70bul.pdf">here</A></P></DIV><DIV class="container"><H3>[4] Site-Directed Mutagenesis PCR</H3><OL><LI>Both primers (25-45bp) must contain the desired mutation and anneal to the same sequence on opposite strands of the plasmid</LI><LI>Melting temperature must be calculated from the equation: Tm = 81.5 + 0.41(%GC)-675/N, where N=primer length (bp) and %GC is the GC percentage in the primer (~40%). </LI><LI>Create reaction mixture below, plasmid should be in excess (~125ng)</LI><LI>Run reaction in thermal cycler with settings below</LI><LI>Run gel verification</LI><LI>Digest samples with DpnI to remove any residual nucleic acids</LI><LI>Run the long transformation protocol with heat shock on LB+Amp plates</LI></OL><TABLE style="width:100%"><TBODY><TR><TH>50ng Reaction</TH><TH>Volume(uL)</TH><TH>Negative Ctrl</TH><TH>Volume(uL)</TH><TH>Positive Control</TH><TH>Volume(uL)</TH></TR><TR><TD>5X phusion buffer</TD><TD>10</TD><TD>5X phusion buffer</TD><TD>10</TD><TD>5X phusion buffer</TD><TD>10</TD></TR><TR><TD>Template</TD><TD>10</TD><TD>Template</TD><TD>0</TD><TD>pWhitescript 4.5kb control</TD><TD>5</TD></TR><TR><TD>Primer 1</TD><TD>1</TD><TD>Primer 1</TD><TD>1</TD><TD>Primer 1</TD><TD>1</TD></TR><TR><TD>Primer 2</TD><TD>1</TD><TD>Primer 2</TD><TD>1</TD><TD>Primer 2</TD><TD>1</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>dNTPs</TD><TD>1</TD><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>ddH2O</TD><TD>26</TD><TD>ddH2O</TD><TD>36</TD><TD>ddH2O</TD><TD>34</TD></TR><TR><TD>Phusion pol.</TD><TD>1</TD><TD>Phusion pol.</TD><TD>1</TD><TD>Phusion pol.</TD><TD>1</TD></TR><TABLE style="width:100%"><TBODY><TR><TH>Step Name</TH><TH># of Cycles</TH><TH>Temperature(℃)</TH><TH>Duration(min)</TH></TR><TR><TD>Initial</TD><TD>1</TD><TD>98</TD><TD>3</TD></TR><TR><TD>Denaturation</TD><TD>35</TD><TD>98</TD><TD>1/6</TD></TR><TR><TD>Annealing</TD><TD>35</TD><TD>55</TD><TD>1</TD></TR><TR><TD>Elongation</TD><TD>35</TD><TD>72</TD><TD>2</TD></TR><TR><TD>Final Elongation</TD><TD>1</TD><TD>72</TD><TD>10</TD></TR><TR><TD>Storage</TD><TD>1</TD><TD>4</TD><TD>∞</TD></TR></TBODY></TABLE><DIV class="container"><H3>[5A] Liquid-Liquid Extraction</H3><OL><LI>Grow 5ml of sender in 10mL culture tube</LI><LI>Remove 1mL of culture media</LI><LI>Centrifuge at 8500g for 10min</LI><LI>Add 5uL of 10uM d9 C5HSL to 1mL of acidified (0.01% acetic acid) ethyl-acetate</LI><LI>Add the 1mL of Ethyl Acetate + HSL to sender media </LI><LI>Vortex for 1 minute </LI><LI>Remove the organic phase</LI><LI>Repeat the extraction procedure, without the addition of the HSL (just ethyl acetate)</LI><LI>The ethyl acetate extracts were evaporated to dryness using rotary evaporation</LI><LI>Samples were stored at 4 degrees Celsius</LI></OL><H3>[5B] High-Performance Liquid Chromatography (HPLC)</H3><P> After the liquid-liquid extraction, the AHL sample was purified using HPLC. The HPLC device that was used was located at the BioSurface Facility at ASU's Biodesign Institute, which has the model HP1100 Preperative/Analytical HPLC. The protocol for this machine can be found <A href="https://www.agilent.com/cs/library/usermanuals/Public/G1380-90000.pdf">here </A>. </P><H3>[5C] Gas Chromatography</H3><P>BL21 E. coli expressing Sender plasmids were grown in 6mLs of LB+amp overnight. After 12 hours, HSLs were extracted three times with 1mL of ethyl acetate. BL21 E. coli expressing the Modular Sender vector (no HSL synthase) was used as a negative control. Samples were analyzed by GC-FID.</P></DIV><DIV class="container"><H3>[6] Induction Optical Density Test</H3><OL><LI>Grow Sender cultures in liquid LB broth</LI><LI>Add any necessary activators to liquid cultures</LI><LI>Obtain a 96-well plate with compatible settings to the plate reader</LI><LI>Pipette 180uL of LB broth w/ ampicillin into each of the desired wells</LI><LI>Add 20uL of sample to their designated wells. Add samples in triplicate</LI><LI>Set plate reader to read over an 8 hour period with settings at 600nm and 580-610nm</LI><LI>Conduct 8-hour overnight read</LI></OL><H3>[7] AHL Disposal Experiment</H3><OL><LI> Grow the receivers (F2620, GFP+, and Neg. Receiver) overnight in 4 mL of LB Amp</LI><LI> Grow the senders (LuxI, LasI) overnight in 4 mL of LB Amp</LI><LI>Spin the senders down at 5,500 RPM for 10 minutes </LI><LI>Filter the media through a 35 um filter</LI><LI>Remove 1 mL of the sender media for the control induction</LI><LI>Add bleach to the sender media</LI><LI>At 10, 20 and 30 minutes, remove 1 mL of the media</LI><LI>Add an equal volume of ethyl acetate</LI><LI>Shake for 30 seconds</LI><LI>Wait 2 minutes, then extract the organic phase
</LI><LI>Plate on a 96 well plate as shown in the table below (test different system by replacing LuxI)</LI><LI>Note: By removing steps 6-10, this is the protocol to conduct F2620 induction tests.</LI></OL><CENTER>Bleach Test Setup</CENTER></DIV><FOOTER><DIV class="container center"><P class="copyright">
     © 2016 ASU iGEM </P></DIV></FOOTER></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>