Team:Arizona State/Notebook/Jiaqi
ASU
Team
About
Collaborations
Project
Description
Protocols
Results
Notebook
Safety
Parts
Parts and Design
Basic Parts
Composite Parts
Part Collection
Practices
Human Practices
Practices:Silver
Practices:Gold
Integrated Practices
White Paper
Attributions
Achievements
Week of 7/05/2016 · Benchling
Week of 7/05/2016
Made with Benchling
Project: Jiaqi's iGEM Stuff
Authors: Jiaqi Wu
Dates: 2016-07-05 to 2016-07-07
Tuesday, 7/5
●
Digest pLas, pCer with EcoRI and SpeI. PCR purify.
A
B
1
Reagent
Volume (uL)
2
DNA
3
10x FD Buffer
2
4
EcoRI
1
5
SpeI
1
6
Water
Table1
●
Digest CerR_MRV with BbsI and treat with phosphatase. Heat deactivate.
●
Week of 7/05/2016
Made with Benchling
Project: Jiaqi's iGEM Stuff
Authors: Jiaqi Wu
Dates: 2016-07-05 to 2016-07-07
Wednesday, 7/6
●
Ligate pLas + LasR_MRV and pCer + CerR_MRV at a 3:1 ratio
Week of 7/05/2016
Made with Benchling
Project: Jiaqi's iGEM Stuff
Authors: Jiaqi Wu
Dates: 2016-07-05 to 2016-07-07
Thursday, 7/7
See OWW for rest of the week's protocols:
Week of 7/18/2016
Made with Benchling
Project: Jiaqi's iGEM Stuff
Authors: Jiaqi Wu
Dates: 2016-07-18 to 2016-07-20
Monday, 7/18
●
Long transform CerR and Bta3R ligation reactions from the week prior. I realized I've been using BL21's, so I switched to DH5A-T's.
●
Retry the Taq PCR reaction with adjusted parameters (based on lab meeting suggestions)
○
I used 50C–54C for pLas, pBta3, and pCer
○
64–66C for pExp (last gel showed an extra band around 300bp...René suggested to increase specificity)
Week of 7/18/2016
Made with Benchling
Project: Jiaqi's iGEM Stuff
Authors: Jiaqi Wu
Dates: 2016-07-18 to 2016-07-20
Tuesday, 7/19
●
I got a lot more colonies with the long transform + DH5A-T's!
●
The Cer plate had almost no colonies on the vector control and a plateful of CerR colonies. Super exciting
●
The Bta3R plate, as predicted, had a lot of colonies on both plates.
●
I picked 3 Cer colonies and 5 Bta3R colonies for liquid cultures. I will miniprep tomorrow.
●
Made a gel for the PCR reactions. Was unable to image due to technical difficulties—asked Tim to run for me tomorrow ^.^
Week of 7/18/2016
Made with Benchling
Project: Jiaqi's iGEM Stuff
Authors: Jiaqi Wu
Dates: 2016-07-18 to 2016-07-20
Wednesday, 7/20
●
I plan to miniprep + digest verify CerR and Bta3R.
●
I plan to follow up with the PCR reactions (i.e., good to digest?) once I can image the gel.
●
LasR sequencing results came back... the 3 colonies on the Las plate were faulty :( did not have the insert.
○
Had a light bulb moment: Ryan uses Eco and Xba primers for pLas...I've been digesting with EcoRI and SpeI for all promoters. They leave the same overhang, but because I am not cutting with the correct enzyme, maybe it's not leaving an overhang at all? I will try a digestion with XbaI instead of SpeI.
Inductions (from Cass)
Made with Benchling
Project: Jiaqi's iGEM Stuff
Authors: Jiaqi Wu
Date: 2016-09-08
Thursday, 9/8
Purpose: Inductions for Aub; might try this protocol on LuxR and BjaR as well, since their gBlocks are here already. Copied from Cass's AubR induction, with some changes.
Items that need attention are highlighted in yellow.
Methods:
Induction
1.
Grow 5mL overnight cultures in LB+Amp of AubI, AubR, neg receiver, negative sender, and GFP positive.
a.
Where can I find all the control colonies? In freezer?
2.
Create 1μM synthetic N-Dodecanoyl-DL-homoserine lactone in LB+Amp and store at 4C. (2uL of 5mM HSL into 10mL LB+Amp)
a.
Use [ ] for LuxR
b.
Use [ ] for BjaR
c.
Are these still in freezer, and what are their concentrations?
3.
Incoluate tubes with the following: --wait, huh? what tubes, and when? before or after overnight?
a.
4mL LB+Amp with 400uL AubR
b.
4mL LB+Amp with 400uL neg receiver
c.
4ml LB+Amp with 400uL GFP positive
4.
The next day, pellet and filter neg sender sample and AubI sample.
5.
Grow these cultures shaking for ~5hrs at 37C. Check OD to make sure receiver culture is at at least 0.6, then induce. Could take upwards of 8 hours. (AubR at 5-> 0.2OD....not good enough yet, but going to run any way because controls were both at 0.7)
6.
Set up a plate as follows:
a.
100uL AubR + 50uL neg send + 50uL fresh X3
b.
100uL AubR + 50uL HSL + 50uL neg sender X3
c.
100uL AubR + 50uL AubI + 50uL fresh X3
d.
100uL neg receiver + 50uL neg send + 50uL fresh X3
e.
100uL neg receiver + 50uL HSL + 50uL neg sender X3
f.
100uL neg receiver + 50uL AubI + 50uL fresh X3
g.
100uL GFP pos + 50uL neg send + 50uL fresh X3
h.
100uL GFP pos + 50uL HSL + 50uL neg sender X3
i.
100uL GFP pos + 50uL AubI + 50uL fresh X3
j.
200uL LB+Amp X3
-Run this plate on the overnight GFP and OD plate reader protocol. Set GFP gain to automatic. Re-set time to 8 hours.
-Plate stopped at X hours and samples taken from each well group into 500uL of PBS (30uL from each sample type except for j, combined)
-Run these tubes a-i on flow, gating at 10,000 cells
When we run these overnight...what do we do about people who need to use the plate reader?
Does anyone need stuff to do? I could use some help with cloning, and definitely help with inductions...
Maybe we can coordinate times to be in lab (since we have classes) and inductions are more time sensitive.
If we need to take something out of the shaker and none of us can make it, then we ask Cass/René?
Primer Designs (09/2016)
Made with Benchling
Project: Jiaqi's iGEM Stuff
Authors: Jiaqi Wu
Date: 2016-09-12
Monday, 9/12
Today, I created primers for standardized biobrick parts. This allows us to integrate our parts into pSB1C3. The forward primers add a XbaI site upstream of the gene of interest, and the reverse primers add SpeI, NotI, and PstI.
(Note: Only XbaI is added in the front to minimze the primer sequence length. We assume that the backbone is BioBrick compatible, meaning it has EcoRI and NotI in the front already.
I used The Haynes Lab protocol for designing the primers: https://benchling.com/hayneslab/f/KvLBmezO-dna-assembly/prt-ePZz1rq2-making-standardized-biobrick-parts/edit
●
AubR gBlocks
○
FWD:  5' CCTTTCTAGAccgtcacctggcggttccgc 3’
■
10 overhang + 20 matching
○
REV: 5' AAGGCTGCAGCGGCCGCTACTAGTttacaccacattgggcaaac 3’
■
24 overhang + 20 matching
●
AubR transcription factor
○
FWD: 5’ CCTTTCTAGAatgtcgcactcggtgggcaa 3’
■
10 overhang + 20 matching
○
REV: 5’ AAGGCTGCAGCGGCCGCTACTAGTttacaccacattgggcaaac 3’
■
24 overhang + 20 matching
■
This is the same as the gBlock reverse primer
●
AubI synthase (Jimmy + Rob)
○
●
BjaR gBlock
●
LuxR gBlock
Plasmid Transformations (09/2016)
Made with Benchling
Project: Jiaqi's iGEM Stuff
Authors: Jiaqi Wu
Dates: 2016-09-13 to 2016-09-19
Tuesday, 9/13
Since we are anticipating a lot of cloning, we will need a lot of backbone vectors. I will be fast-transforming the Modular Receiver Vector and pSB1C3 into DH5ɑTs. **MRV has Ampicillin resistance, while pSB1C3 has CamR.
Both processes will follow this protocol:
●
Fast transform into DH5ɑ-Ts
●
Next day: grow liquid cultures (6mL LB+Amp or 6mL LB + CamR)
●
Next day: mini-prep and digest verify
○
For MRV—digest with BbsI and XbaI to see two bands at 1kb and 2.1kb
○
For pSB1C3—digest with XbaI and PstI
●
Label and store in -20°C!
Use 1000x amp for LB. Ex: 500uL of AMP in 500mL LB
Plasmid Transformations (09/2016)
Made with Benchling
Project: Jiaqi's iGEM Stuff
Authors: Jiaqi Wu
Dates: 2016-09-13 to 2016-09-19
Monday, 9/19
Prepping iGEM DNA Distrbution Plasmids for transformation:
1.
Punch a hole in the foil of the desired well with a pipette tip
2.
Pipette 10 uL dH20 into the well; pipette up and down a few times to mix
a.
The color of the solution will be red because of the dried dye
3.
Fast transform 1uL of the plasmid
4.
I used two plasmids:
a.
DNA Kit Plate 1: 1A ( Bba_K314110 )
b.
DNA Kit Plate 2: 13H ( Bba_J202012 )
I will transform two plasmids and 1 negative control plasmid into CamR plates that Brady has prepped the week before.
© 2016 ASU iGEM
