<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCL_London_Research_Supercoiliology_Model"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCL_London/Research/Supercoiliology/Model">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCL_London/Research/Supercoiliology/Model&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL_London/Research/Supercoiliology/Model&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL_London/Research/Supercoiliology/Model&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCL_London/Research/Supercoiliology/Model" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCL London/Research/Supercoiliology/Model</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="ecoili"><DIV id="submenu" class="sm-research-supercoiliology"><UL><LI class="li-150"><A href="/Team:UCL_London/Research/Supercoiliology" title="Team:UCL London/Research/Supercoiliology">Concept</A></LI><LI class="li-150"><A href="/Team:UCL_London/Research/Supercoiliology/Theory" title="Team:UCL London/Research/Supercoiliology/Theory">Theory</A></LI><LI class="li-150"><STRONG class="selflink">Model</STRONG></LI><LI class="li-150"><A href="/Team:UCL_London/Research/Supercoiliology/Experiments" title="Team:UCL London/Research/Supercoiliology/Experiments">Experiments</A></LI><LI class="li-150"><A href="/Team:UCL_London/Research/Supercoiliology/Results" title="Team:UCL London/Research/Supercoiliology/Results">Results</A></LI><LI class="li-150"><A href="https://static.igem.org/mediawiki/2011/d/d8/Lab_Diary_for_Supercoiliology.pdf" class="external text" rel="nofollow">Lab Diary</A></LI></UL></DIV><DIV id="content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Modeling_Gyrase_Overexpression"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Modeling Gyrase Overexpression</SPAN></A></LI><LI class="toclevel-2"><A href="#For_a_regular_Top10_E.coli_cell"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">For a regular Top10 E.coli cell</SPAN></A></LI><LI class="toclevel-2"><A href="#Supercoiling_in_an_unmodified_Top10_E.coli_cell"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Supercoiling in an unmodified Top10 E.coli cell</SPAN></A></LI><LI class="toclevel-2"><A href="#For_a_Top10_E.coli_cell_with_the_E.coili_plasmid"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">For a Top10 E.coli cell with the E.coili plasmid</SPAN></A></LI><LI class="toclevel-2"><A href="#Supercoiling_in_a_modified_Top10_cell_with_an_E.coili_plasmid"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Supercoiling in a modified Top10 cell with an E.coili plasmid</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Modeling_Gyrase_Overexpression">Modeling Gyrase Overexpression</SPAN></H1><P>The animation shows the functional E. coli DNA gyrase enzyme which consists of 2 GyrA subunits and 2 GyrB subunits. GyrB is the ATP consuming subunits and is yellow in colour in the animation. The blue coloured domains are the GyrA subunits whereby DNA breakage and resealing occurs, while the cyan colour domains are the C-Terminal domains of GyrA and are involved in the wrapping of DNA for allowing the correct progression of a full catalytic cycle in order to introduce negative supercoiling into the plasmid DNA molecule.
</P>We developed a mathematical model to demonstrate the production of gyrase in a typical Top10 E.coli cell over the course of the cell´s doubling time. Additionally, we modeled the change in the expression of gyrase caused by the insertion of the gyrase expression cassette we designed and how that influenced the level of supercoiled plasmids in the cell over time.
<P>In order to model the expression of gyrase and the way it´s inhibited by the level of supercoiled plasmid in the cell, we used the equations found from P. R. Jensen et al. (1999)<SUP><A href="/Team:UCL_London/Bibliography#supercoiliology-5" title="Team:UCL London/Bibliography">[5]</A></SUP> to demonstrate the feedback mechanism of supercoiled plasmid by DNA gyrase and Topoisomerase 1 and the data on typical supercoiled DNA linking numbers from Barth, Dederich and Dedon (2009)<SUP><A href="/Team:UCL_London/Bibliography#supercoiliology-6" title="Team:UCL London/Bibliography">[6]</A></SUP>.
</P><H2><SPAN class="mw-headline" id="For_a_regular_Top10_E.coli_cell">For a regular Top10 E.coli cell</SPAN></H2><P>Assumptions:
1.1.	There are 300 plasmids in a typical Top10 E.coli cell that have a length of 4000 nucleotide base pairs each on average, and 50 of which code for DNA gyrase.
1.2.	All the plasmids that need to be expressed in the parent cell are present at t=0 of the cell´s life.
1.3.	The plasmids coding for gyrase produce as much gyrase as the number of copies there are in within the cell for that particular plasmid. 
1.4.	All plasmids within this Top10 cell have the same copy number.
1.5.	Both the subunits for gyrase A and gyrase B are expressed each plasmid coding for gyrase
</P><P>E.coli cells have a doubling time of 40 minutes.
</P><P>We made use of the transcription and translation rates of the plasmid coding for gyrase in order to ascertain the time taken for gyrase production in the cell. 
</P><P>1.6	Assume that transcription and translation of the gyrase encoding plasmids occur simultaneously and there are no limiting factors in the cell for these processes. That is to say that the ribosomes, transfer RNA (tRNA) and any enzymes required in the RNA forming complex are not limiting, allowing optimal, constant, translational and transcriptional rates.
</P><P>1.7	Assume that within the cell the optimal conditions for growth are being met and maintained within the cell throughout its life cycle. 
</P><P>1.8	The overexpression of gyrase enzyme won´t slow down E.coli cell growth resulting in cell death.
</P><P>For E.coli cells the rates of transcription and translation are 50 nucleotides/ second and 20 amino acids/ second (which corresponds to 60 nucleotides/ second) respectively.  Since both of these reactions occur simultaneously, we must acknowledge that transcription is the rate determining step.
</P><P>The first molecule of gyrase would be produced after 1.3 minutes.  Assuming that all the plasmids coding for gyrase are fully expressed, at the end of the cell life (at t=40) there are meant to be 50 gyrase molecules present in the cell. Assuming no limiting factors during protein production, all 50 molecules of gyrase should be expressed simultaneously at t= 1.3 minutes. The rate of gyrase degradation also has to be accounted for in order to ascertain the total amount of gyrase that will be present in the cell. According to Reece and Maxwell (1991), gyrase has a half-life of 60 hours at 23 degrees Celsius at a dissociation constant of 0.1 to 0.5 nM. Due to the lack of experimental data which indicate the half life of the gyrase in vivo, we will assume that these are the conditions faced by the Top10 E.coli cell being modelled.
</P><P>Using the following equations derived for determining half life we were able to ascertain the amount of gyrase molecules left in the cell by the time the parent cell was ready to divide into two daughter cells. It was found that amongst the total number of gyrase molecules in the cell, not a single one is fully degraded within the timespan of 40 minutes. This is to say that there is a steady increase in the amount of gyrase over time due to the transcription and translation of the gyrase encoding plasmids, until the cell has expressed the maximum amount of gyrase after which gyrase production levels off. Evidence of this is demonstrated graphically below:
</P><P>Knowing the maximum amount of gyrase present in the cell is beneficial as it allows us to determine the exact time taken to produce adequate levels of supercoiling within the cell.
</P><H2><SPAN class="mw-headline" id="Supercoiling_in_an_unmodified_Top10_E.coli_cell">Supercoiling in an unmodified Top10 E.coli cell</SPAN></H2><P>For DNA molecules present within the cell, it is commonplace that one strand is always wound around the other forming the DNA double helix. In order for a DNA molecule to be supercoiled, it must form a closed loop with its ends fixed. Once its ends are covalently fixed together, it is in its relaxed topological state. This state can be described by the following equations<SUP><A href="/Team:UCL_London/Bibliography#supercoiliology-7" title="Team:UCL London/Bibliography">[7]</A></SUP>:
</P><P><I>Lk = Tw+Wr</I></P><P>Lk=Linking number (Number of double helical turns in the original linear molecule)
Tw=The helical twist (Represents the way in which strands of DNA coil around each other about the axis of the DNA helix
Wr= superhelical turns  (Measure of the contortion of the helix axis in space)
</P><P>In a relaxed DNA plasmid, Lk is known as Lk(0) and is equal to Tw while Wr=0. Tw= N/h, h is the number of base pairs per turn of the helix and N is the number of base pairs in the DNA. 
</P><P>So for our plasmids Lk(0)= 381, and Wr=0. 
The specific linking difference (σ) is number of turns added or removed relative to the total number of turns in the relaxed plasmid and thus showing the level of supercoiling.
</P><P><I>σ = (Lk-Lk(0))/Lk(0)</I></P><P>The typical specific linking difference for prokaryotic cells is σ=-0.06 <SUP><A href="/Team:UCL_London/Bibliography#supercoiliology-7" title="Team:UCL London/Bibliography">[7]</A></SUP>so the linking number for the E.coli Top10 plasmids is 358.14. The turnover number for gyrase B is 1 coil/ gyrase/ second, that is a linking number of differences of ±2 per gyrase (which requires 2 molecules of ATP)/second.
The supercoiling of plasmid DNA (pDNA) to the standard amount within a prokaryotic cell requires a linking number deficit of 22.86 ≈ 23 per pDNA. 
</P><P>It takes 1.15 minutes in order to supercoil the entire plasmid content of the parent E.coli cell.
</P><H2><SPAN class="mw-headline" id="For_a_Top10_E.coli_cell_with_the_E.coili_plasmid">For a Top10 E.coli cell with the E.coili plasmid</SPAN></H2><P>Assumptions
2.1	The E.coili plasmid is a high copy number plasmid (ca 300 plasmids) and has a length of 5000 nucleotide base pairs.
2.2	The only difference between a normal Top10 E.coli cell and the Top10 cell that overexpresses gyrase is that the latter has the E.coili plasmid.
2.3	The E.coili plasmid will only express DNA gyrase.
2.4	The plasmids coding for gyrase produce as much gyrase as the number of copies there are in within the cell for that particular plasmid. This is because that the IPTG promoter which has been ligated into the E.coili plasmid does not subject to the natural chromosomal (regulatory mechanism explained in Supercoiliology Theory section).
2.5	All plasmids within this Top10 cell, exluding the E.coili plasmid have the same copy number.
</P><P>Assumptions 1.1, 1.2, 1.5, 1.6, 1.7 and 1.8 still hold true.
</P><P>The transcriptional and translational rates within the E.coli cell, as well as the growth conditions within the cell are to be held constant.
</P><P>Therefore, the first gyrase molecule is produced after 1.3 minutes as well. Assuming no limiting factors during protein production, all 350 molecules of gyrase should be expressed simultaneously at t= 1.3 minutes. After 40 minutes, using the same calculations for half life as shown above, we get the gyrase degradation rate for this cell. It is found that after 40 minutes 347 out of a total of 350 gyrase molecules are surviving in the cell. Evidence of this is demonstrated graphically below:
</P><H2><SPAN class="mw-headline" id="Supercoiling_in_a_modified_Top10_cell_with_an_E.coili_plasmid">Supercoiling in a modified Top10 cell with an E.coili plasmid</SPAN></H2><P>The typical specific linking difference for prokaryotic cells is σ=-0.06 so the linking number for the E.coli Top10 plasmids is 358.14. While the linking number for the high copy number plasmids encoding for gyrase is 447.44. The turnover number for gyrase B is 1 coil/ gyrase/ second, that is a linking number of differences of ±2 per gyrase (which requires 2 molecules of ATP)/second.
</P><P>The supercoiling of plasmid DNA (pDNA) to the standard amount within a prokaryotic cell requires a linking number deficit of 28.66 ≈ 29 per E.coili plasmids and 22.86 ≈ 23 per pDNA.
</P><P>It takes 0.37 minutes in order to supercoil the entire plasmid content of the parent E.coli cell. Even though there are a greater number of plasmids (which have more nucleotide base pairs), the increase in gyrase expressed in the cell allowed this larger amount of parental E.coli cell plasmid to be supercoiled more quickly than it was when the E.coili plasmid wasn´t being expressed. This demonstrates that the addition of our E.coili plasmid allows an overall increase in the speed of plasmid supercoiling to get a uniform amount of sc plasmid throughout the parental Top10 E.coli cell.
</P><P>The following is a diagrammatic representation of the negative feedback mechanism for the production of gyrase within the cell according to P. R. Jensen et al. (1999). We were not able to create a mathematical model for this system since we lacked the kinetic data for DNA topoisomerase 1 as well as the fluctuating ATP/ ADP ratio according to set boundaries within the cell.
</P><P>The upper level of this diagram represents the synthesis of gyrase, while the lower level represents the breakdown of gyrase and topoisomerase 1.
</P></DIV><DIV id="footer"><A href="/Team:UCL_London/Team" title="Team:UCL London/Team">Team</A> | <A href="/Team:UCL_London/Data" title="Team:UCL London/Data">Data</A> | <A href="/Team:UCL_London/Safety" title="Team:UCL London/Safety">Safety</A> | <A href="/Team:UCL_London/Attributions" title="Team:UCL London/Attributions">Attributions</A> | <A href="/Team:UCL_London/Partners" title="Team:UCL London/Partners">Partners</A> | <A href="/Team:UCL_London/Bibliography" title="Team:UCL London/Bibliography">Bibliography</A> | <A href="/Team:UCL_London/Protocols" title="Team:UCL London/Protocols">Protocols</A> | <A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2011&amp;group=UCL_London" target="_blank">Registry of Parts</A> | <A href="http://www.ucl.ac.uk/biochemeng/index.html" target="_blank">UCL Biochemical Engineering</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:UCL_London/Research/Supercoiliology/Model">http://2011.igem.org/Team:UCL_London/Research/Supercoiliology/Model</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCL_London/Research/Supercoiliology/Model" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCL_London/Research/Supercoiliology/Model" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCL_London/Research/Supercoiliology/Model&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCL_London/Research/Supercoiliology/Model&amp;oldid=168259" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>