Team:UCLouvain/Protocols
iGEM UCLouvain Team
iGEM UCLouvain Team
Home
our project
overview
auxotrophs
comR/S
badge
Results
Auxotrophs
Com R/S
human practices
safety
in the lab
interlab
Lab Notebook
protocols
Parts
about us
team
attributions
our university
our events
aknowledgement
judging form
In the Lab
Protocols
DNA Gel electrophoresis
Materials
Agarose Powder
TAE buffer
Gel mould
Gel Tank
DNA ladder
DNA loading dye
Procedure
Prepare 0.8% agarose (w/v) solution in TAE buffer.
Heat until the agarose dissolve.
Pour into the gel mould and set a comb.
Wait for the gel to solidify.
Remove the comb and transfer into the gel tank. Fill with TAE buffer.
Fill the wells with DNA ladder and your samples mixed with corresponding amount of loading dye.
Run for 20-30 minutes at 120 V.
T4 Ligation
Materials
PCR tubes
T4 DNA Ligase (NEB)
T4 DNA Ligase buffer 10x
ddH20
Insert DNA
Procedure
Calculate insert to vector ratio with NEB calculator. Most common is 3:1.
Add the vector plasmid, the insert DNA, 2 µL of buffer, 1 µL of T4 ligase and enough ddH20 to reach 20 µL.
Gently mix by pipeting up and down.
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes. For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours.
Heat inactivate at 65°C for 10 minutes.
Selection of the sgRNA and modification of pTarget
Introduction
This protocol cover the identification of a suitable N20 sequence and the modification of the pTarget plasmid. Protocol from S. Worms.
Procedure
1) Selection of the N20 sequence
Use Benchling's CRISPR tool, click on "Design and analyse guide"
Using the default settings (note: Doench et al.'s optimized scoring algorithm doesn't work for circular sequences such as bacterial chromosomes)
Select the target region and press (+)
Select the N20 with the higher On-target score.
2) Modification of pTarget
pTarget is currently available in two version, pTargetF with spectinomycin resistance and pTargetTet_pyrF and its derivatives with a tetracyclin resistance. They should behave identically.
Design a forward oligo to use for a QuickChange-like insertion of the N20 sequence. The sequence of the primer should be the following:
agctagctcagtcctaggtataatactagtNNNNNNNNNNNNNNNNNNNNgttttagagctagaaatagcaagttaaaa
where the Ns stands for your N20 sequence.
Amplify the pTarget backbone of your choice using the designed forward primer and the appropriate reverse primer (actagtattatacctaggactgagctagctg) using Q5 polymerase and the following conditions:
Temperature (°C)
Time
98
30 s
98
15 s
35 x
65
30 s
72
75 s
72
120 s
4
inf.
Media
Materials
NaCl
Tryptone
Yeast Extract
Na2HPO4
KH2PO4
NH4Cl
Glucose
Tyrosine
ddH20
CaSO5
MgSO4
Agar
Heat-resistant glass bottles
Procedure M9 medium
Make 10xM9 salts.
Mix the following in 500 mL H20 and sterilize by autoclave.
Mass (g)
Na2HPO4
64 (Na2HPO4.7H2O)
KH2PO4
15
NaCl
2.5
NH4Cl
5
Make 1M CaCl2 stock solution. The CaCl2 is sterilized separately to avoid formation of CaSO4 precipitate. Dissolve 5.592 g CaC2 to 50 mL H2O and sterilize by filtration or autoclaving.
Make 1M MgSO4 stock solution.
Make 20% glucose solution. Dissolve 10g of glucose in 50 mL H2O and sterilize by filtration. Do not autoclave. Dissolve 6.018 g of MgSO4 in 50 mL H2O and sterilize by filtration or autoclaving.
Make 5x tyrosine solution. Dissolve 0.25 g of tyrosine in 1L H2O and sterilize by filtration.
Sterilize water by autoclaving.
For 500 mL.
M9
M9 + Tyr
M9 Agar
M9 Agar + Tyr
10x M9 salts
50
50
50
50
Agar (g)
0
0
7.5
7.5
H2O
445
345
445
345
Autoclave
No
No
Yes
Yes
M1 MgSO4
1
1
1
1
M1 CaSO4
0,1
0,01
0,01
0,1
20% glucose
5
5
5
5
Tyrosine
0
100
0
100
Procedure LB medium
Mix the following.
Volume (1L)
LB
LB agar
Yeast Extract (g)
8
8
Tryptone (g)
5
5
NaCl (g)
5
5
Agar (g)
0
14
Add ddH2O to the desired volume.
Sterilize by autoclaving.
Genome modification of E. coli using recombineering and CRISPR/Cas9
Introduction
This protocol is adapted from Jiang et al., 2015. It assumes a version of pTargetTet targeting the correct locus of the E. coli genome has already been created, as well as a donor DNA that include overlap regions and the gene to be inserted. For information on the design and creation of the sgRNA-producing pTarget, see Selection of the sgRNA and modification of pTarget.
Materials
E. coli strain containing the pCas plasmid.
pTargetTet targeting the locus to be modified.
Donor DNA: gene of interest flanked by two 500-bp overlaps, or an ssDNA for deletion.
LB, Kanamycin, Tetracyclin.
Arabinose 10%.
Chilled sterile ddH2O.
Sterile PEP tube.
IPTG 0,1 M.
Procedure
pCas has a thermo-sensitive origin of replication. All steps until pCas curing should be incubated at 30°C.
Day 1
Start a 5 ml pre-culture of TOP10 pCas in LB-Kanamycin. Incubate it overnight at 30°C without shaking.
Day 2
Competent cells
Start a culture for transformation by inoculating 40 ml of LB-Kan with 400 µl of pre-culture. Incubate in a 250 ml flask at 30°C until an OD600 between 0,4 and 0,6 is attained.
Add 600 µl of 10% arabinose to the culture to cause the induction of the λ-Red genes. Incubate for 15 minutes at 30°C.
Chill the culture in a water-ice bath. Transfer to a 50 ml centrifuge tube and centrifuge at 4600 x g for 7 minutes in a centrifuge cooled to 4°C.
Add 30 ml of ice-cold water to the pellet. Swirl gently to resuspend the cells. Centrifuge again as described in step 5.
Resuspend the pellet in 1 ml of cold ddH2O. Transfer it using a 1000 µl tip to a pre-chilled sterile EP tube. Centrifuge down at 10.000 x g for 4 seconds in a chilled tabletop centrifuge. (Note, due to speed-up time, doing a quickspin up to 10.000 x g and then letting it slow down is usually good enough) Carefully pipette out the supernatant.
Repeat step 7.
Resuspend cells in 200 µl of chilled ddH2O. Keep on ice until ready to transform.
Transformation
Using a 1-mm transformation culture, electroporate 100 ng of pTargetTet plasmid and 400 ng of linear donor DNA in 50 µl of competent cells (Note: when using an ssDNA oligo instead of dsDNA linear donor DNA, transform 100 ng of the plasmid with 100 ng of oligo). Recover the cells at 30°C for 1 hour in SOC medium then plate on LB-Tet-Kan plates. Incubate the plate at 30°C overnight. (Note: it often takes up to 48 hours for colony to show up).
Screen transformant by colony PCR to identify correct clones.
Plasmid curing
To cure the plasmids, resuspend a colony in 2 ml of LB-Kan to which 10 µl of IPTG 0,1 M are added (Final concentration: 0,5 mM) and the culture is grown overnight at 30 °C. In the morning, the colonies are plated on LB plates and incubated at 37°C O/N to cure them of the pCas plasmid.
RFP fluorescence test
Materials
400 W LEO / S-400-94-CR light
Five 12-wells plates
M9, LB
Tyrosine, ortho-nitrobenzyl tyrosine, L-arabinose
Precultures procedure
Inoculate 10 mL of liquid LB and incubate at 37°C overnight with appropriate antibiotic
Bl21(DE3), TyrA- (not transformed), TyrA- + RFP
Fluorescence test procedure
Pellet precultures at 3000 xg for 10 minutes at 4°C
Wash 3x with 5 mL liquid M9 to remove tyrosine present in medium
When OD600 ~ 0.6, concentrate 10X
Add the following in each well of a 12-wells plate (final volume 3 mL, completed withliquid M9). Make three replicas.
Tyr + Ara (BL21)
Tyr (BL21)
ONB-Tyr + Ara (BL21)
ONB-Tyr (BL21)
Tyr + Ara (TyrA-)
Tyr (TyrA-)
ONB-Tyr + Ara (TyrA-)
ONB-Tyr (TyrA-)
Tyr + Ara (TyrA-/RFP)
Tyr (TyrA-/RFP)
ONB-Tyr + Ara (TyrA-/RFP)
ONB-Tyr (TyrA-/RFP)
Concentrations
Tyrosine: 0.05 mg/mL
ONB-Tyr: 0.084 mg/mL
L-arabinose: 0.1% (w/v)
KAN30, CM34
Glucuronidase expression test with ComRS (E. coli)
Materials
Na2HPO4
KCl
MgSO4
ComS
Na2CO3
para-nitrophenyl glucuronide
FastPrep machine
Procedure
ComS induction
Dilute preculture to OD600 = 0.1
Incubate 2h at 37°C
Induce for 2h at 37°C with 8 µM of ComS
Gus reaction test
Pellet cells at 13 000 RPM
Resuspend in Gus tampon (60 mM Na2HPO4, 40 mM NaH2PO4, 10 mM KCl, 1 mM MgSO4, pH 7)
Lyse with FastPrep for cycles of 30 sec each (see FastPrep protocol)
Recover supernatant after centrifugation
Optional: mesure total concentration of protein with Bradford test
Add 4 µL of substrat (12.5 mM para-nitrophenyl glucuronide) per 100 µL of supernatant and incubate 10 minutes at 37°C
Stop reaction by addition of a 1 M Na2CO3 solution (150 µL per 100 µL of supernatant)
Mesure absorbance at 405 nm
Note: the specific activity for Gus is defined in arbitrary units as the concentration of para-nitrophénol (calculated with a standard curve) produced per minute and per mg of total protein.
PCR from DNA template or colony
Materials
Phusion HF DNA Polymerase (NEB) or GoTaq Polymerase (Promega)
5x Phusion HF buffer or GoTaq Flexi Green
Template DNA
ddH20
PCR tubes
Primers (forward and reverse, 10 mM)
Thermocycler
dNTP solution (5 mM each)
25 mM MgCl2 solution
Procedure
PCR reaction mix
In PCR tubes, add the following:
A
B
1
Component
Volume (µL)
2
Buffer
10
3
Primer FWD
2.5
4
Primer REV
2.5
5
MgCl2
2 (only Go Taq)
6
dNTPs
1
7
Polymerase
0.5
8
DNA Template
10-200 ng
9
ddH20
up to 50 µL
Mix with up-and-downs
Spin gently to collect the liquid at the bottom
Thermocycler
In a PCR machine run the following program.
Note: use NEB's Tm Calculator to estimate the annealing temperature.
A
B
C
D
1
Step
Temperature
Time
2
Initial Denaturation
98°C
5 minutes
3
Denaturation
98°C
30 seconds
25-35 cycles
4
Annealing
See note
30 seconds
5
Elongation
72 °C
1 minute/kb (GoTaq)
6
1 minute/kb (GoTaq)
7
Final Elongation
72 °C
7 minutes
8
Soak
4°C
∞
Load DNA on gel or purify for downstream applications.
Colony PCR
Same protocol, pick a colony to add in the mix instead of DNA.
Plate out colonies on LB agar for safekeeping
Gibson Assembly Protocol
Introduction
This is adapted from the protocol for Gibson Assembly using the Gibson Assembly® Cloning Kit (NEB).
Materials
Gibson Assembly Mastermix (NEB)
DNA Polymerases (for generating PCR products)
Recommended: Q5® High-Fidelity DNA Polymerase, Q5 Hot Start High-Fidelity DNA Polymerase, or Q5 Hot Start High-Fidelity 2X Master Mix
ddH20
LB plates with appropriate antibiotic
Procedure
Set up the following reaction on ice:
Reaction volumes:
NEB recommends a total of 0.02–0.5 pmols of DNA fragments when 1 or 2 fragments are being assembled into a vector and 0.2–1.0 pmoles of DNA fragments when 4–6 fragments are being assembled. Efficiency of assembly decreases as the number or length of fragments increases. To calculate the number of pmols of each fragment for optimal assembly, based on fragment length and weight, we recommend using NEB's online tool, NEBioCalculator .
A
B
C
1
2-3 Fragments Assembly
4-6 Fragments Assembly
2
Concentration Range of DNA fragments
0.2 - 0.5 pmols*
0.2 - 1.0 pmols*
3
Total Volume of Fragments (μl)
Depends on DNA concentration
Depends on DNA concentration
4
Gibson Assembly Master Mix (2x) (μl)
10
10
5
ddH20 (μl)
Completing to reach 20
Completing to reach 20
6
Total Volume (μl) **
20
20
*Optimized cloning efficiency is 50–100 ng of vectors with 2–3 fold of excess inserts. Use 5 times more of inserts if size is less than 200 bps. Total volume of unpurified PCR fragments in Gibson Assembly reaction should not exceed 20%.
** If greater numbers of fragments are assembled, additional Gibson Assembly Master Mix may be required.
Incubate samples in a thermocycler at 50°C for 15 minutes when 2 or 3 fragments are being assembled or 60 minutes when 4-6 fragments are being assembled.
Note: Extended incubation up to 60 minutes may help to improve assembly efficiency in some cases
Store samples on ice or at –20°C for subsequent transformation.
FastPrep cell lysis
Introduction
This is the protocol for cell lysis using the FastPrep technique. Protocol received from Nadia El Bakkali Taheri.
Materials
Glass beads (Glasperlen 0,17-0,18 mm, suspended in 10 ml PBS pH 7.5).
PBS pH 7.5
FastPrep machine
Procedure
Centrifuge 2 ml of cell culture at 4000 rpm and 4°C for 10 minutes.
Discard the supernatant and resuspend the cells in 500 µl of PBS.
With a cut tip, add 500 µl of the beads in PBS. Keep agitating to keep the beads in suspension.
Lyse using the FastPrep machine for 60 sec.
Centrifuge 10 minutes at maximum speed.
Recover the supernatant that include the lysate.
you think we're cool? so are our sponsors!
