<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_University_of_Sheffield_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:University_of_Sheffield_/Project">Page               </A></LI><LI><A href="/Talk:Team:University_of_Sheffield_/Project">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:University_of_Sheffield_/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:University_of_Sheffield_/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:University_of_Sheffield_/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:University of Sheffield /Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="color:#888888;background-color:##888888;" cellpadding="5" cellspacing="2" border="2" width="85%" align="center"><TBODY><TR><TH align="center"><A href="/Team:University_of_Sheffield" title="Team:University of Sheffield">Introduction</A></TH><TH align="center"><STRONG class="selflink">Our project</STRONG></TH><TH align="center"><A href="/Team:University_of_Sheffield_/Modelling" title="Team:University of Sheffield /Modelling">Modelling</A></TH><TH align="center"><A href="/Team:University_of_Sheffield_/Wet_Lab" title="Team:University of Sheffield /Wet Lab">Wet Lab</A></TH><TH align="center"><A href="/Team:University_of_Sheffield_/Lab_Books" title="Team:University of Sheffield /Lab Books"> Our team</A></TH><TH align="center"><A href="/Team:University_of_Sheffield_/Timetable" title="Team:University of Sheffield /Timetable"> Timetable</A></TH><TH align="center"><A href="/Team:University_of_Sheffield_/Misc" title="Team:University of Sheffield /Misc"> Miscellaneous</A></TH></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Fusion_Receptor"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Fusion Receptor</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Overview"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Protein_analysis"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Protein analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Final_remarks"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Final remarks</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#The_Native_E.coli_Pathway"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">The Native E.coli Pathway</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Downstream_the_Fusion_Kinase"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Downstream the Fusion Kinase</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Fluorescent_Reporter"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Fluorescent Reporter</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Introduction_2"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Re-usable_testing_kit"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Re-usable testing kit</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#pgaABCD_operon_control"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">pgaABCD operon control</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Target_of_insertion"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Target of insertion</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Cholera_Autoinducers"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Cholera Autoinducers</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#The_Microchip_Theory"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">The Microchip Theory</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#How.3F"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">How?</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Any_takers.3F"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Any takers?</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Parts_and_BioBricks"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Parts and BioBricks</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Acknowledgements"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Acknowledgements</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#References"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H1><H1><SPAN class="mw-headline" id="Fusion_Receptor">Fusion Receptor</SPAN></H1><H3><SPAN class="mw-headline" id="Overview">Overview</SPAN></H3><P>We hypothesise that expression of receptor protein specific for Vibrio cholerae signalling molecules called autoinducers would confer sensing property on Escherichia coli cell. Because presence of autoinducers in water is associated with cholera pathogens expression of the receptor protein would transform an E.coli into a biosensor. The most straightforward, which does not mean the easiest, way of doing so could be expression of whole signaling pathway in the modified organism. However such an approach leads to a few problems. Firstly, many proteins of downstream signal transduction pathway spanning foreign receptor and DNA have be to introduced into the cell. For example in V.cholera's signalling pathway multiple proteins and small regulatory RNAs are involved. To give response for external stimulus all of them have to be successfully expressed and keep they functionality at satisfactory level. In practice this can be very hard as foreign proteins often tend to form inclusion bodies or their functionality is lowered or lost in a new intracellular conditions they are exposed to. In addition introduction of a whole new pathway can lead to unforeseen consequences for the host cell.
</P><P>Another method of making bacteria sensitive for external stimulus could be creating fusion receptor protein. In every receptor two elements can be distinguished – receiver and transmitter parts. If two receptors are related closely enough and if they have similar topology fusion of receiver element from one protein with transmitter from the other should give rise to receptor that retains its functionality (Zhu, 2008). Transmitter domain would pass then the signal downstream through a pathway native for modified organism. Such an approach could bring together sensitivity and efficiency of a receptor on one hand  and reliability and good level of understanding of signal transduction pathway native to modified organism on the other.
</P><H3><SPAN class="mw-headline" id="Protein_analysis">Protein analysis</SPAN></H3><P>We want to use this approach in constructing a biosensor monitoring cholera contamination of water. Fusion receptor would consist of two parts, a signal receiver domain from CqsS, protein which is native to V. cholera and binds to CAI-1 molecules (autoinducers synthesised by V.cholerae) and signal conveyer domain of BarA which is a protein present on the surface of E.coli cell  and that passes signal downstream to DNA. 
</P><P>Our choice of BarA protein was driven by similarity of those receptors. Both of them are multisubunit molecules of similar topology and use the same method of signal transduction – they belong to Two Component Regulatory System (TCRS). Such a system includes two cooperating proteins a histidine kinase located in the membrane and its partner response regulator.  Histidine kinase detects signal from the environment and autophosphorylates itself at the His residue at its cytoplasmic surface. The phosphate group is then passed onto aspartate residue on a response regulator that interacts directly with the DNA or relays signal further on. 
</P><P>Both CqsS and BarA are histidine kinases. Their analysis reveals that they contain sequences that belongs to the same superfamilies -  HATPase_c and REC. 3D structures of those proteins haven not been established yet however multisequence alignment and comparison with proteins and protein's domains belonging to the same families revealed conserved structural patterns and probable similar topology. Their function and mechanism of action is most likely conserved. From there we deduced that we should combine N-terminal domain of CqsS protein that is responsible for recognition of autoinducers and autophosphorylation and C-terminal domain of BarA that accepts phosphate and relays it through aspartate downstream.
</P><H3><SPAN class="mw-headline" id="Final_remarks">Final remarks</SPAN></H3><P>Big advantage is that we know exactly how the E.coli's native pathway we are employing works. Introduction of reporter system (e.g. GFP) within the site of DNA positively regulated by BarA should show if the stimulus we are sensing for is present in the environment. Shortly if E.coli shines water is contaminated by cholera.
This could be general approach used
</P><P>Few questions have to be addressed. Firstly, what is the concentration of autoinducers in water that corresponds to amount of pathogens recognised as contamination and if this amount high enough to trigger the response in the receptor. Second question asks how fusion would decrease efficiency of the receptor. Answering these questions is crucial to assess if the machine based on fusion receptor would be of any practical use.
</P><P>We think that fusing two domains from two different but similar receptors to build a new one could be a general approach to embrace during making biosensors that are based on  a foreign receptor.
</P><H1><SPAN class="mw-headline" id="The_Native_E.coli_Pathway">The Native E.coli Pathway</SPAN></H1><H2><SPAN class="mw-headline" id="Downstream_the_Fusion_Kinase">Downstream the Fusion Kinase</SPAN></H2><P>The sensing event triggers the phosphorilation of the CqsS-BarA and stimulates the transfer of phosphate group downstream the signaling cascade. Naturally, E.coli passes the phosphate group from BarA to its response regulator UvrY. This particular protein then binds to DNA triggering the transcription of small regulatory RNA called CsrB - Carbon Storage Regulator B. The most complicated part of the pathway is that CsrB inhibits CsrA. CsrA in turn inhibits a whole set of genes depicted in the table on the left ( plus represents activation, minus - inhibition). 
</P><P>One of the genes that CsrA inhibits is responsible for biofilm adhesion protein formation  - PGA. This particular operon was our target for reporter molecule insertion. More details about <I>pgaABCD</I> operon can be found below. 
</P><P>Reference for the images: <A href="/Team:University_of_Sheffield_/References#2:_Cerca.2C_N._and_Jefferson.2C_K._K" title="Team:University of Sheffield /References"> [2]</A></P><H1><SPAN class="mw-headline" id="Fluorescent_Reporter">Fluorescent Reporter</SPAN></H1><H3><SPAN class="mw-headline" id="Introduction_2">Introduction</SPAN></H3><P>The initial plan was to introduce a Green Fluorescent reporter molecule under one of the genes controlled by small regulatory RNA called CsrA. So as not to reduce the viability of the cell as well as to avoid any possible “noise” in the engineered system, the gene choice required several attempts. 
</P><H3><SPAN class="mw-headline" id="Re-usable_testing_kit">Re-usable testing kit</SPAN></H3><P>The plan to have a micro – chip for sensing biological water contamination ideally would include a possibility of reusing it. Thus, the aim of our project was to use GFP with an LVA tag (AANDENYALAA). This would mean that the GFP would have an amino acid tag targeted by  housekeeping proteases and would be degraded within 45 minutes once there is no more stimulation for gene expression. This in turn would mean that the E.coli system sensing V.cholerae presence in water would stop fluorescing once the concentration of CAI-1 molecules decreased up to tolerable level.<A href="/Team:University_of_Sheffield_/References#2:_Cerca.2C_N._and_Jefferson.2C_K._K" title="Team:University of Sheffield /References"> [2]</A></P><P>The idea to include degradable GFP in our project was taken from the Andersen, J.B et al. (1998) New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria. Applied and Environmental Microbiology.
</P><H3><SPAN class="mw-headline" id="pgaABCD_operon_control"><I>pgaABCD</I> operon control</SPAN></H3><P>This particular four - gene locus in E.coli is responsible for synthesis of the biofilm adhesion protein called PGA. As this is a virulence factor, i.e. non-essential for growth in supplemented media, a possible insertion of GFP and possible disruption the operon would not cause lethal effects to our cells. 
</P><P>As far as it is known, the <I>pgaABCD</I> genes are subject to post-translational inhibition by the small regulatory RNA CsrA as well as activation by the cation-responsive regulatory protein NhaR.  The first fact is to our advantage. In theory, once the binding of the CAI-1 to fusion kinase occurred, the CsrA inhibition would be releaved and the GFP would be expressed . Also, according to research it has been proven that the <I>csrA </I> deletion mutant had an impressive 3000 fold increase compared to the wild type. 
</P><P>Possible “noise” in the system
</P><P>As mentioned before, one of the factors to be taken into account prior choosing the gene within which the GFP would be inserted is the interference or “noise” in the system. This possibility might have caused background fluorescence or GFP expression independent from the sensing event of our interest.   
</P><P>The reasons underlying this “noise” are listed below: 
</P><UL><LI>High salt concentrations (NhaR dependent)
</LI><LI>Alkaline pH (NhaR dependent)
</LI><LI>Ethanol ( NhaR dependent)
</LI><LI>Glucose presence in the media (Independent from both CsrA and NhaR)
</LI></UL><P>The noise in the system appears only if the cells are grown at extreme conditions. However, if the growth media contains only 1% glucose, 1 % ethanol, 1 % NaCl the levels of the pgaABCD transcript should not be elevated by any other factors except CsrA.  On the other hand, if additional NaCl or glucose are added to the medium it might cause significant increase in the <I>pgaABCD</I> mRNA appearance [2]
</P><H3><SPAN class="mw-headline" id="Target_of_insertion">Target of insertion</SPAN></H3><P>As the <I>pgaABCD</I> is a four-gene locus another question that arose was the actual place within the operon where the insertion could take place. Another obstacle was that genes <I>b, c</I> and<I> d</I> overlap with each other. 
</P><P>One would theoretically suggest insertion under the promoter sequence, before the actual gene<I> a</I> . However, the very high inhibition with CsrA is actually due to 6 binding sites, one of which overlaps with gene <I>a</I> formyl-methionine codon.   
</P><P>Thus, the only possible way of inserting GFP, with minimal disruption of the whole operon, is at the end of gene <I>a</I> sequence just before the termination codon. This would give a high level of control by CsrA, as it is just after the CsrA regulatory binding sites, and would hopefully prevent extensive damage to genes <I>b,c</I> and <I>d</I> expression.
</P><H1><SPAN class="mw-headline" id="Cholera_Autoinducers">Cholera Autoinducers</SPAN></H1><P>Cholera autoinducer (CAI) is a signaling molecule. It is unusual from most pathogen autoinducers as it actually inhibits virulence factor production at high concentration. This is because of the nature of the disease <I>V.Cholerae</I> causes. The inhibition of virulence factors allows the bacteria to be flushed from the host body into the environment.
</P><P>
CAI is used naturally only by <I>V.Cholerae</I>. This makes it very useful in our bacterial sensor, because unlike other autoinducers, such as AI-2, which are used by many bacteria, this allows us to know bacteria species detected.
</P><P>CAI is a short ketone with a hydroxyl group. The image below is the different CAI molecules (3 isomer pairs)
</P><H1><SPAN class="mw-headline" id="The_Microchip_Theory">The Microchip Theory</SPAN></H1><P>Our idea originated with the concept of a microchip that would test water for pathogen concentration. Whilst one summer wouldn’t allow us to actually build it, we still had an impression as to what it would look like. 
</P><H3><SPAN class="mw-headline" id="How.3F">How?</SPAN></H3><P>The target was for a cheap, reusable, quick and non-technical piece of equipment. At first, the microchip really was micro, and would require a microscope to look at, but in areas of natural disaster or extreme poverty this wasn’t viable. So we planned and hand-held size chip, with a number of ‘wells’ or ‘chambers’ built into it. Each chamber would contain a different pathogen-sensing E.coli colony. This presented other problems into how to keep the E.coli alive – could they survive if attached physically to the chamber? Or would they have to be frozen, or suspended in media? Frozen seemed the most likely, but not ideal, but proper design could be implemented later.
Then the idea was to take a sample of water, possibly contaminated, and place it on the chip. The contaminants would interact with the specially designed E.coli, and those activated by contaminants would produce a distinctive glow visible to the naked eye. The chamber that glows corresponds to an unhealthy level of a particular pathogen in the water sample.
</P><P>The reusable aspect came from the idea that disposable equipment is environmentally unfriendly. However, it could be cheaper than a reusable option, so if out plan were to be properly carried out, both chips should be developed to provide options. To make the chip reusable however, LVA tags are attached to the GFP in the DNA code, which attracts a housekeeping protease to cleave the GFP more rapidly than if left to degrade in the cell alone. A short burst of glow would be visible in cell, which would disappear by the time the chip dries out/the water passes through it.
</P><H3><SPAN class="mw-headline" id="Any_takers.3F">Any takers?</SPAN></H3><P>This approach requires absolutely no technical knowledge, and could be ideal for its purposes. A future iGEM project anyone?
</P><H1><SPAN class="mw-headline" id="Parts_and_BioBricks">Parts and BioBricks</SPAN></H1><P>After various attempts to use the Biobrick booklet for transformation procedures needed for implied project ideas, we came to conclusion that the <B>booklet</B> shipped was actually <B>faulty</B>. Tests then proved this to be true. Thus the required BioBrick vector for submission of our designed parts was impossible to use from the booklet we were provided. Unfortunately, we ascertained this too late to receive variance approval. Which means we have BioBricks built but no way to send/enter them.
</P><P>
The BioBricks we made, however, were:
</P><UL><LI>1.) CsrA, with promoter
</LI><LI>2.) CsrA, without promoter
</LI><LI>3.) CsrB, with promoter
</LI><LI>4.) CsrB, without promoter
</LI><LI>5.) UvrY, with promoter
</LI><LI>6.) UvrY, without promoter
</LI><LI>7.) CsqS
</LI><LI>8.) CqsA
</LI><LI>9.) BarA
</LI><LI>10.) Red recombinase system!
</LI><LI>11.) pKD46 Ori site - temperature sensitive
</LI><LI>12.) FLP recombinase
</LI></UL><H1><SPAN class="mw-headline" id="Acknowledgements">Acknowledgements</SPAN></H1><P>We would like to thank to everyone who helped us on our way. That is to<B> Prof. Visakan Kadirkamanathan</B> whose excitement about our 
venture made us believe that it is possible. <B>Prof. Philip Wright</B> for providing us with a fine lab space in the Sheffield Bioincubator. <B>Dr Catherine Biggs</B> for supporting us with her knowledge and advice.<B> Prof. David Hornby</B> and his lab for being constant source of practical help. <B>Esther Karunakaran</B> was playing role of our advisor. She was a patient tutor that guided us through our project. <B> Prof. Jeff Green</B> and<B> Prof David Rice</B> for an advice regarding theoretical aspects of our project. <B>Dr Graham Stafford</B>, <B>Bassler Lab</B> and <B>Ojar Melfors</B> for providing us biological material for our experiments. All members of<B> ChELSI group</B>, especially <B>Joy</B>, for their constant advice on every day basis. <B>Dave Wengraf</B> who knows all legal aspect of science helped us with paper work and got us VAT exemptions. <B>Prof. Robert Pool</B> and <B>Prof. Dominic Shellard</B> for providing generous contribution toward our travel to iGEM Jamboree. <B>IDT DNA</B> for their sponsorship. <B>IChemE</B>, <B>BioFusion</B> and University of Sheffield <B>Union of Students</B> for supporting us financially.  Without you this project would be much harder, if not impossible to carry out!
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><UL><LI>1.) Andersen, J.B et al. (<B>1998</B>) New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria. <I> Applied and Environmental Microbiology</I></LI></UL><UL><LI>2.) Cerca, N. and Jefferson, K. K. (<B>2008</B>) Effect of Growth Conditions On Poly-N-acetylglucosamine expression and Biofilm Formation in Escherichia coli
</LI></UL><UL><LI>3.) Goller, C. et al. ( <B>2006</B>) The Cation-Responsive Protein NhaR of <I>Excherichia coli</I> Activates <I>pgaABCD</I> Transcription, Required for Production of the Biofilm Adhesin Poly-Beta-1, 6 - N - D - Glucosamine. <I>Journal of Bacteriology</I></LI></UL><UL><LI>4.)Datsenko &amp; Wanner, (<B>2000</B>), One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products
</LI></UL><UL><LI>5.) Higgins, Bassler et al, (<B>2007</B>), The major <I>Vibrio cholerae</I> autoinducer and its role in virulence factor production
</LI></UL><UL><LI>6.) Hammer &amp; Bassler, (<B>2007</B>), Regulatory small RNAs circumvent the conventional quorum sensing pathway in pandemic <I>Vibrio cholerae</I></LI></UL><UL><LI>7.) Jun Zhu, Melissa B. Miller, et al, (<B>2001</B>), Quorum-sensing regulators control virulence gene expression in <I>Vibrio cholerae</I></LI></UL><UL><LI>8.)Tomenius, Pernestig et al, (<B>2005</B>), Genetic and functional characterization of the E.coli BarA-UvrY Two-componant system
</LI></UL><UL><LI>9.) Suzuki et al, (<B>2002</B>), Regulatory Circuitry of thr CsrA/CrsB and BarA/UvrY systems of E.coli
</LI></UL><UL><LI>10.) Sahu, Acharya et al, (<B>2003</B>), The bacterial adaptive response gene, barA, encodes a novel conserved istidine kinase regulatory switch for adaptation and modulation of metabolism in E.coli
</LI></UL><UL><LI>11) Sabnis, A. N. et al., (<B>1995</B>), Pleiotropic Regulation of Central Carbohydrate Metabolism in Escherichia coli via the Gene <I>csrA</I></LI></UL><UL><LI>12) Ytoh, J. et al., (<B>2006</B>), Roles of pgaABCD Genes in Synthesis, Modification, and Export of the Escherichia coli Biofilm Adhesin Poly-beta-1,6-N-Acetyl-D-Glucosamine
</LI></UL><UL><LI>13) Yang., H. et al. , (<B>1995</B>), Coordinate Genetic Regulation of Glycogen Catabolism and Biosynthesis in scherichia coli via the CsrA Gene Product
</LI></UL><UL><LI>14) Romeo, T. (<B>1998</B>), Global Regulation by the small RNA-binding protein CsrA and non-coding RNA molecule CsrB
</LI></UL><UL><LI>15) Wang, X. et al. (<B>2005</B>), CsrA post-transcriptionally represses pgaABCD,responsible for synthesis of a iofilm polysaccharide adhesin of <I>Escherichia coli</I></LI></UL><UL><LI>16) Romeo, T. et al. (<B>1995</B>), Identification and Molecular Characterization of <I>csrA</I>, a Pleiotropic Gene from Escherichia coli That Affects Glycogen Biosynthesis, Gluconeogenesis, Cell Size, and Surface Properties
</LI></UL><UL><LI>17) Pernestig, A. K. et al. (<B>2002</B>), The Escherichia coli BarA-UvrY Two-Component System Is Needed for Efficient Switching between Glycolytic and Gluconeogenic Carbon Sources
</LI></UL><UL><LI>18) Pernestig, A. K. et al. (<B>2000</B>), Identification of UvrY as the Cognate Response Regulator for the BarA Sensor Kinase in <I> Escherichia coli</I></LI></UL><UL><LI>19) Mondragon, V. et al. (<B>2006</B>), pH-Dependent Activation of the BarA-UvrY Two-Component System in <I>Escherichia coli</I></LI></UL><UL><LI>20) Rogov, V. V. et al. (<B>2006</B>), A New Structural Domain in the <I>Escherichia coli</I> RcsC Hybrid Sensor Kinase onnects Histidine Kinase and Phosphoreceiver Domains
</LI></UL><UL><LI>21) Tomenius, H. et al. ( <B>2006</B>), The <I>Escherichia coli</I> BarA-UvrY two-component system is a virulence determinant in the urinary tract
</LI></UL><UL><LI>22) Zhu, Y. and Inouye, M. (<B>2004</B>), The HAMP Linker in Histidine Kinase Dimeric Receptors Is Critical for Symmetric Transmembrane Signal Transduction
</LI></UL><UL><LI>23) Zhu, y. and Inouye, M. (<B>2008</B>), Analysis of the Role of the EnvZ Linker Region in Signal Transduction Using a Chimeric Tar/EnvZ Receptor Protein, Tez1
</LI></UL><UL><LI>24) Walters, M. and Sperandio, V. (<B>2006</B>), Quorum sensing in <I>Escherichia coli</I> and <I>Salmonella</I></LI></UL><UL><LI>25) Gonzales, J. E. and Keshavan, N. D. (<B>2006</B>), Messing with Bacterial Quorum Sensing
</LI></UL><UL><LI>26) Haseltine, E. L. and Arnold, F. H. (<B>2007</B>), Implications of Rewiring Bacterial Quorum Sensing
</LI></UL><UL><LI>27) Alur, J. et al. (<B>2002</B>), Modeling and Analyzing Biomolecular Networks
</LI></UL><UL><LI>28) Dunlap, P. V. (<B>1999</B>), Quorum Regulation of Luminescence in <I>Vibrio fischeri</I></LI></UL><UL><LI>29) Sveningsen, S. L. et al. (<B>2008</B>), A negative feedback loop involving small RNAs accelerates <I>Vibrio cholerae’s</I> transition out of quorum-sensing mode
</LI></UL><UL><LI>30) De Silva, R. S. et al. (<B>2007</B>), Crystal Structure of the <I>Vibrio cholerae</I> Quorum-Sensing Regulatory Protein HapR
</LI></UL><UL><LI>31) Lenz, D. H. et al. (<B>2005</B>), CsrA and three redundant small RNAs regulate quorum sensing in <I>Vibrio cholerae</I></LI></UL><UL><LI>32) Heidelberg, J. F. et al. (<B>2000</B>), DNA sequence of both chromosomes of the cholera pathogen <I>Vibrio cholerae</I></LI></UL><UL><LI>33) Neidtich, M. B. et al. (<B>2006</B>), Ligand-Induced Asymmetry in Histidine Sensor Kinase Complex Regulates Quorum Sensing
</LI></UL><UL><LI>34) Matson, J. S. et al. (<B>2007</B>), Regulatory Networks Controlling <I>Vibrio cholerae</I> Virulence Gene Expression
</LI></UL><UL><LI>35) Hammer, B. K. and Bassler, B. L. (<B>2003</B>), Quorum sensing controls biofilm formation in <I>Vibrio cholerae</I></LI></UL><UL><LI>36) Hammer, B. K. and Bassler, B. L. (<B>2007</B>), Regulatory small RNAs circumvent the conventional quorum sensing pathway in pandemic <I>Vibrio cholerae</I></LI></UL><UL><LI>37) Lin, W. et al. (<B>2005</B>), Requirements for <I>Vibrio cholerae</I> HapR Binding and Transcriptional Repression at the <I>hapR</I> Promoter Are Distinct from Those at the aphA Promoter
</LI></UL><UL><LI>38) Lenz, D. H. et al. (<B>2004</B>), The Small RNA Chaperone Hfq and Multiple Small RNAs Control Quorum Sensing in <I>Vibrio harveyi</I> and <I>Vibrio cholerae</I></LI></UL><UL><LI>40) Hoch, J. A. and Varughese, K. J. (<B>2001</B>), Keeping Signals Straight in Phosphorelay Signal Transduction
</LI></UL><UL><LI>41) McLean, R. J. C. et al. (<B>2004</B>), A simple screening protocol for the identification of quorum signal antagonists
</LI></UL><UL><LI>42) McClean, K. H. et al. (<B>1997</B>), Quorum sensing and <I>Chromobacterium violaceum</I> : exploitation of violacein
</LI></UL><P>production and inhibition for the detection of N-acylhomoserine lactones
</P><UL><LI>43) Martinko, J. M., Madigan, M. T. (<B>2006</B>) Brock Microbiology of Microorganisms. Pearson Benjamin Cummings, San Fransisco.
</LI></UL><UL><LI>43) Vendeville, A. et al. (<B>2005</B>), Making‘SENSE’ Of Metabolism:Autoinducer!2, LuxS and Pathogenic Bacteria
</LI></UL><UL><LI>44) Water, C. M. and Bassler, B. L. (<B>2005</B>), Quorum Sensing: Cell-to-Cell Communication in Bacteria
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:University_of_Sheffield_/Project">http://2008.igem.org/Team:University_of_Sheffield_/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:University_of_Sheffield_/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:University_of_Sheffield_/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:University_of_Sheffield_/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:University_of_Sheffield_/Project&amp;oldid=84084" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>