Team:EPFL/Notebook-2
iGEM EPFL 2018
iGEM EPFL 2018
HOME
Project
Description
Take a closer look at CAPOEIRA
Design
Learn more about how we designed CAPOEIRA
Software
Explore the Bioinformatic Pipeline implemented in CAPOEIRA
Results
Discover the Results of our experiments
Demonstration
Demonstration of how our Therapeutic System works
Interlab
Learn more about our participation in the Interlab Study
Parts
Discover the Parts that make up CAPOEIRA
Improve
Explore our Novel and Improvements Parts
Measurement
Learn about measurement approaches we used for Part Characterization
Team
Team Members
Learn more about our Team Members
Collaborations
Discover the interactions we had with other iGEM Teams during CAPOEIRA
Human Practices
Integrated Human Practices
Discover how we integrated Human Practices into the Purpose, Design and Implementation of CAPOEIRA
Ethics
Explore the ethical considerations of CAPOEIRA
Applied Design
Explore how our therapeutic system attempts to meet real world medical need
Education and Public Engagement
Discover our Education and Public Engagement work with the public and scientific communities
European and Eurasian Meetup
Learn about our interactions with the iGEM Community during the Meetups
Lab
Safety
Entrepreneurship
Acknowledgements
Attributions
Sponsors
Partners
Awards
Index
Notebook-Vaccine part
This page collects the notebooks of the Vaccine part of our project. The experiments are sorted by week
Friday, 20/07/18
INSERT PCR WITH TAQ POLYMERASE
Aim: The Aim of this experience is to produce the hexahistidine insert with a PCR using TAQ.
Note:The primers we use from now on were diluted 10 times by mixing 10μlof the original primers with 90μl of water
PCR mix:
Green taq master mix (2X)
10μl
HexaHisForwardInsert
1μl
HexaHisReverseInsert
1μl
nuclease free water
8μl
total
20μl
PCR Cycle
Green taq master mix (2X)
10μl
HexaHisForwardInsert
1μl
HexaHisReverseInsert
1μl
nuclease free water
8μl
total
20μl
INSERT PCR WITH PHUSION POLYMERASE
Aim: The Aim of this experience is to produce the hexahistidine insert with a PCR using Phusion.
PCR mix in μl
Forward Primer
1μl
Reverse Primer
1μl
dNTPs (10mM)
0.4μl
Phusion Buffer
4μl
Phusion polymerase
0.2μl
H2O
13.4μl
Total
20μl
PCR cycle
STEP
Temp
Duration
Initial Denaturation
98
30s
Denaturation
98
10s
30 cycles
Annealing
72
30s
Extension
72
30s
Final Extension
72
1min
Hold
4
BACKBONE PCR
Aim: The Aim here is to amplify the backbone plasmid of encapsulin with a PCR.
PCR mix
Template 1ng/μl
1μl
Forward Primer 10mM
1μl
Reverse Primer 10mM
1μl
dNTPs (10mM)
0.4μl
Phusion Buffer
4μl
Phusion polymerase
0.2μl
H2O
12.6μl
Total
20μl
PCR cycle
STEP
Temp
Duration
Initial Denaturation
98
30s
Denaturation
98
8s
30 cycles
Annealing
63
25s
Extension
72
165s
Final Extension
72
10min
Hold
4-10
Results:
PCR gel electrophoresis of the insert PCR with Taq and Phusion
The PCR product were run on a 2% agarose gel, at 100V for 30 minutes.
The ladder is the o'generuler low range.
The first lane contains the PCR product of the insert with Taq, the 2nd lane contains the PCR product of the insert with Phusion. The 3rd lane is the forward insert primer and the last lane the reverse insert primer. 2μl of the Phusion PCR                 product and the primers were loaded with 2μl of loading dye. 4μl of the Taq PCR product, as well as the ladder, were loaded on the gel.
We can see a signal for the inserts that seem to be the right size as they are both next to the 100bp band of the ladder, the expected size for the insert is of 90 bp. It looks like Taq does work better than Phusion because the signal is                 stronger for Taq. It is also noticeable that the insert migrated less far than the primers which is what we expected, the primers were only 54 and 60 bp long.
PCR gel electrophoresis of the insert PCR with Taq and Phusion
Discussion
The PCR product for the backbone was run on a 1% agarose gel for 45 minutes at 100V.
2μl of PCR product were loaded with 2μ of thermo scientific 6X loading dye in the two lanes next to the generuler express DNA ladder. There are faint signals at around 100bp, the primers are supposed to be 28 and 30 bp long, So it's hard                 to tell what these can be. There is no trace of the backbone DNA, template, or PCR product.
Monday, 23/07/18
Since the last PCR didn't work we will try to change the temperature, by doing a gradient PCR with several temperature values
BackBone PCR
PCR mix
Template 1ng/μl
1μl
Forward Primer 10mM
1.25μl
Reverse Primer 10mM
1.25μl
dNTPs (10mM)
0.5μl
Q5 Buffer
5μl
Q5 polymerase
0.25μl
H2O
15.75μl
Total
25μl
PCR Cycle
STEP
Duration
Temp
Temp
Temp
Temp
Temp
Initial Denaturation
30s
98
98
98
98
98
Denaturation
8s
30 cycles
98
98
98
98
98
Annealing
25s
62
64
66
68
70
Extension
165s
72
72
72
72
72
Final Extension
2min
72
72
72
72
72
Hold
4
4
4
4
4
Results
The PCR product for the backbone was run on a 1% agarose gel for 45 minutes at 100V.
2.5µl of PCR product were loaded onto the gel with 2.5µl of thermo scientific 6X loading dye. 5µl of the o'gene ruler express ladder were loaded onto the gel.
As we can see there are no signal for the 5500 bp backbone we were expecting, which means that the PCR didn't work for any of the temperatures.
we can see signals below the 100bp mark of the ladder, these are probably the primers.
Tuesday, 24/07
We tried another PCR for the backbone but this time using a different template for the backbone. We had two different template.
Template #1, comes from an aliquot of the backbone template at 844 ng/µl. It is labeled with encap sample 1 in the freezer.
Template #2 comes from an aliquot of the backbone template at 350.8 ng/µl. It is labeled with encap sample 2 in the freezer.
Both samples were diluted to a final concentration of 1ng/µl
We recreated the following PCR mix, 3 times for each DNA template. for a total of 6 pcr mixes.
Template 1ng/μl
1μl
Forward Primer 10mM
1.25μl
Reverse Primer 10mM
1.25μl
dNTPs (10mM)
0.5μl
Q5 Buffer
5μl
Q5 polymerase
0.25μl
H2O
15.75μl
Total
25μl
and ran the following gradient cycle for both sets of 3 PCR mix
STEP
Duration
Temp
Temp
Temp
Initial Denaturation
30s
98
98
98
Denaturation
10s
30 cycles
98
98
98
Annealing
30s
64
66
68
Extension
165s
72
72
72
Final Extension
2min
72
72
72
Hold
4
4
4
Results:
We can see that the PCR worked at least once for each template. Surprisingly, the only time the PCR worked for template#1 was with an annealing temperature of 68°C while the calculated annealing temperature for the primers was 68°C. For                 template#2 the PCR worked for 66°C but we can also see a faint signal for the PCR with an annealing temperature of 68°C. However, it is clear that the PCR worked better with an annealing temperature of 66°C.
Conclusion
If you want to do a PCR amplification of the backbone, use an annealing temperature of 68°C if you use the template diluted from the 844ng/µl aliquot (encap sample1) and use an annealing temperature of 66°C if you want to use the template                 diluted from the 350.8ng/µl aliquot (encap sample2).
Notebook week 3 (23/07/18)
FRIDAY, 7/27/2018
Inoculating cultures, and miniprep
Aim: The aim of this experience is to grow liquid cell cultures from our transformed cells so we can use them to purify plasmid DNA later, and do glycerol stocks
Protocol: Inoculating cultures
We grew 2 different liquid cultures with 2 colonies from our transformed bacteria plate, as well as a negative control with only LB medium and ampicilin but no cells. They were incubated overnight at 37°C and shaked at 225 rpm.
Results:
We can see that the 2 cell cultures turned white overnight while the negative control stayed translucid indicating that no contamination occurred and that cells successfully grew overnight.
SATURDAY, 7/28/2018
Isolation of plasmid DNA
Goals:
To purify plasmid DNA from E.coli cells
To quantify the amount of purified plasmid
To prepare a glycerol stock of bacteria for long term storage
1.Plasmid Purification
Protocol: Plasmid miniprep
We performed 2 different MiniPrep reactions from our 2 different cell cultures, and obtained two different samples of purified plasmid
Results: Measurements of the purified plasmids with the nanodrop lite yielded the following results
Measurements were done with the nanodrop lite, using 1μl of nuclease-free water as blank, and 1μl of purified plasmid for the measurement.
Purified plasmids from cell culture #1:
Concentration:14.6 ng/μl
A260/280:1.72
Purified plamids from cell culture #2:
Concentration:16.1 ng/μl
A260/280:1.84
The concentration of purified plasmids are very low for both samples. Probably too low for being useful. We should perform another plasmid purification with more cell culture. Here we only purified 600μl from the cell culture. Next time                 we can try purifying a larger volume.
Notebook week 4 (30/07/18)
Monday, 30/07/18
Today we did another cell culture, like last time but with 4ml so we would take 500μl to make a glycerol stock of the cells and use the rest to purify the plasmids
Protocol: Inoculating cultures
We did 2 liquid cell cultures with 2 different colonies from our plate. As well as a negative control
Tuesday, 31/07/18
1.Preparation of glycerol stocks
Aim:Make glycerol stocks of the cell cultures that grew overnight
Protocol: Glycerol stock preparation
We did a glycerol stock of both cell cultures in 2 different tubes, that were placed in the -80°C freezer
2.Isolation of plasmid DNA from a larger volume of cell culture
Aim:To try the purification of the plasmid DNA again, but with a larger volume of cell culture
Protocol Plamid miniprep
This time, we centrifuged 3ml of cell culture, discarded the supernatant and resuspended the cell pellet in 600ul of water to perform the miniprep
We finally purified 3ml from each cell cultures that were grown last night
Results: Measurements of the purified plasmids with the nanodrop lite yielded the following results
Measurements were done with the nanodrop lite, using 1μl of nuclease-free water as blank, and 1μl of purified plasmid for the measurement
Purified plasmids from cell culture #1:
Concentration: 71,1 ng/μl
A260/280: 1.72
Purified plasmids from cell culture #2
Concentration: 167.5 ng/μl
A260/280: 1.51
The concentrations are better than last time, However the A260/280 ratios are a bit low, especially for the second sample. This might lead to problems because of impurities in the samples.
Notebook week 5 (06/08/18)
MONDAY, 8/6/2018
Competent cell transformation: pMC_sfGFP-BsaI_noT7 plasmid into DH10β competent E.coli cells
pMC_sfGFP-BsaI_noT7 plasmid contains the sfGFP sequence which we will insert into our modified encapsulin plasmid in order to check take up by DCs The cell transfection will allow us to produce new plasmids, but first we have to check if                 the plasmid is still usable as it was stored quite long at room temperature.
We used the competent cell transformation protocol that we slightly adapted to our needs. Here are the modifications we made:
Amounts in µl
Transformation
Transformation control
Competent cells (in tube)
50
50
plasmids
5
-
Nuclease free water
-
1
Heat shock the cells at 42 °C for up to 45s (we let them 45s). Immediately transfer the tube back on ice for minimum 2 min (we made 3 min)
We did not add cell medium to the mixture because it is not needed when working with cells rendered resistant to Ampicilin
Ampicilin directly kills cells which are not resistant to it --> not the case of other antibiotics the outgrowth step serves to prevent growth of other types of bacteria that could then destroy the ampicilin on the plate and then there is a risk that not resistant bacteria grow
We spread 50 uL each time on LB+ Amp plates using spreader we made ourselves and sterilised
TUESDAY, 8/7/2018
Results of the transformation of DH10β competent E.coli cells with pMC_sfGFP-BsaI_noT7 plasmid
Negative control (cells that were not transfected with the plasmid). As expected, no colonies have grown
Here were plated the transfected cells. No culture have grown showing that the plasmids should have been degraded when staying too long at room temperature.
Inoculation of cultures from glycerol stocks of cells (DH5α or DH10β) containing HexaHistidine-encap plasmids
Cultures were inoculated following this protocol
We had glycerol stock from 2 different colonies so we made 2 different tubes as well as a negative control in which we just put LB medium and Ampicilin.
We used 1000x Ampicilin and as we need 1000x less Ampicilin than medium we put 3uL of Ampicilin in each tube.
Purification of the plasmids of the cultures from glycerol stocks of cells (DH5α or DH10β) containing HexaHistidine-encap plasmids
The plasmids were purified using the Pure yieldTM plasmid miniprep kit from promega following the corresponding protocol.
The resulting of the purification from the two different colonies are as follows:
colony 1:
260/280: 1,75
concentration:50,4 ng/uL
colony 2:
260/2801,79
concentration:41,2 ng/uL
Test of the efficiency of DH-10 β competent E.coli cells
puC19 Plasmid, pMC65, pMC116 were heat shocked into three different competent cell tubes and plated on Amp plates. The plates were incubated overnight at 37 degrees
THURSDAY, 8/9/2018
Result of the test of the efficiency of DH-10 β competent E.coli cells
No colonies were found on the plates, it may be that the cells used are not competent.
Competent cell transformation using DH5α competent E.coli cells
The competent cells were given to us by the LBNC lab and their efficiency was already assessed.
We transformed the cells with the 3 following plasmids:
pMC_sfGFP-BsaI_noT7
(contains sfGFP) --> we called it p65
pSIREN_U6-Cpf1sgRNA
(contains RFP) --> we called it p116
puc19 (a standard control plasmid)
We followed this protocol (https://benchling.com/lbnc/f/OOhQKnto-encapsulin/prt-b5aeAD4Y-competent-cell- transformation/edit) to transfrom the cells and added a transformation control (50uL of competent cells with 5uL of nuclease free water).
FRIDAY, 8/10/2018
Results of the cell transformation using DH5α competent E.coli cells
No colonies were found on any of the plates. As the cells have already shown efficiency in other experiments, this could mean that the plasmids that stayed for weeks are room temperature (p116 and p65) were all degraded and that the puc19                 whose expiration date is 2017 is not viable anymore.
Competent cell transformation using DH5α competent E.coli cells
The competent cells were given to us by the LBNC and their efficiency was already assessed.
We transformed the cells with the 3 following plasmids:
pMC_sfGFP-BsaI_noT7
(contains sfGFP) --> we called it p65
(this one was given to us by the LBNC and was stored in good conditions)
plasmid carring mCherry from iGEM distribution kit
plasmid carring Cyan Fluorescent Protein from iGEM distribution kit
plasmid carring YGFP from iGEM distribution kit
We followed this protocol (https://benchling.com/lbnc/f/OOhQKnto-encapsulin/prt-b5aeAD4Y-competent-cell- transformation/edit) but as we were not well organized we were not able to start directly when we got the cells from the LBNC and we                 let them thaw a bit on ice and then refroze them in the -20 °C freezer before finally thaw them on ice and use them. Moreover we were using 1 plasmid conferring Ampicilin resistance and 2 conferring chlorampenicol resistance but we did                 the outgrowth step for the 3, although we should not have done it for the 1 conferring ampicilin resistance. Finally, the quantity of DNA we used to transform the cells with the plasmids from iGEM's distribution kit was very low; indeed                 iGEM suggests to dilute the dried DNA from the distribution kit in 10uL water and then do the transformation with 1uL which corresponds to 200-300pg of DNA.
SATURDAY, 8/11/2018
Result of the competent cell transformation using DH5α competent E.coli cells
No colonies were found.
Notebook week 6 (13/08/18)
Cells we used:DH5α competent E.coli cells (some from Ivan and some from Michael)
Plasmids we used:
pMC_sfGFP-BsaI_noT7 (https://benchling.com/lbnc/f/2wg8QdVJ-introduction/seq-Uj4WhTuX-pmc_sfgfp-bsai_not7/edit)
puc19
CFP carrying plasmid (http://parts.igem.org/Part:BBa_K404319)
mCherry carrying plasmid (http://parts.igem.org/Part:BBa_J06504)
SYFP2 containing plasmid (http://parts.igem.org/Part:BBa_K864100)
Plasmid
pMC_sfGFP-BsaI_noT7
puc19
CFP carrying plasmid
mCherry carrying plasmid
SYFP2 containing plasmid
Resistance
Ampicilin
Ampicilin
Chloramphenicol
Chloramphenicol
Chloramphenicol
Vector
-
-
pSB1C3
pSB1C3
pSB1C3
We followed iGEM's single tube transformation protocol (http://parts.igem.org/Help:Protocols/Transformation) but we put only 30uL of cells in each tube and used LB medium instead of SOC.
We did the mistake to also put medium to the ampicilin resistant bacteria, they were put in the incubator for ~1min before she realized it. Then we took them, put them quickly on ice (which was a bad idea), put them in the microcentrifuge                 and centrifuge them for 3min at 6,8g. We removed the supernatant and vortex them a bit and then plate the resulting 160uL.
Result of the competent cell transformation
We had colonies only on the plate where we put the cells transfected with pMC_sfGFP-BsaI_noT7 (https://benchling.com/lbnc/f/2wg8QdVJ-introduction/seq-Uj4WhTuX-pmc_sfgfp-bsai_not7/edit).
Pouring LB Agar plates
For 200mL of LB Agar mixture (~10 plates) --> 3g Agar, 5g of LB
! to cover the erlenmeyer after the autoclave!
Competent cell transfection with Laurine + postivie control
Notebooks under PDF
Notebook Week 6
Notebook Week 7
Notebook Week 8
Notebook Week 9
Notebook Week 10
Notebook Week 11
Notebook Week 12
Notebook Week 13
Notebook Week 14
iGEM2018@groupes.epfl.ch
IGEM EPFL 2018
Twitter Facebook
