Team:EPFL/Protocols
iGEM EPFL 2018
iGEM EPFL 2018
HOME
Project
Description
Take a closer look at CAPOEIRA
Design
Learn more about how we designed CAPOEIRA
Software
Explore the Bioinformatic Pipeline implemented in CAPOEIRA
Results
Discover the Results of our experiments
Demonstration
Demonstration of how our Therapeutic System works
Interlab
Learn more about our participation in the Interlab Study
Parts
Discover the Parts that make up CAPOEIRA
Improve
Explore our Novel and Improvements Parts
Measurement
Learn about measurement approaches we used for Part Characterization
Team
Team Members
Learn more about our Team Members
Collaborations
Discover the interactions we had with other iGEM Teams during CAPOEIRA
Human Practices
Integrated Human Practices
Discover how we integrated Human Practices into the Purpose, Design and Implementation of CAPOEIRA
Ethics
Explore the ethical considerations of CAPOEIRA
Applied Design
Explore how our therapeutic system attempts to meet real world medical need
Education and Public Engagement
Discover our Education and Public Engagement work with the public and scientific communities
European and Eurasian Meetup
Learn about our interactions with the iGEM Community during the Meetups
Lab
Safety
Entrepreneurship
Acknowledgements
Attributions
Sponsors
Partners
Awards
Protocols
This page collects the different protocols used in our project. They are sorted in alphabetical order.
Agarose gel electrophoresis
Introduction
A protocol on how to prepare and run a standard agarose gel electrophoresis at a certain percentage of agarose. The percentage of agarose to use can be estimated according to the sequence length (bp)1.
Materials
10X TAE buffer
SYBR Safe (10 000X)
Distilled water
DNA samples (PCR,..)
Parafilm
Gel Loading Dye, Purple (6X) (NEB)
GeneRuler 1 kb Plus DNA Ladder (ladder should be adapted to the sequence length)
Procedure
Preparation of the gel
Prepare 60 ml of (1-2)% Agarose in 1X TAE buffer.
Dilute the 10X concentrated TAE buffer by adding 6 ml to 54 ml of water in a column.
Weight the desired amount of agarose (1% := 600 mg) and put it in an Erlenmeyer. Add the buffer.
Melt the agarose in a microwave oven (no aluminium or Parafilm) for around 2 minutes (until all the agarose is dissolved).
Add 6 µl SYBR safe into the Erlenmeyer.
Set up the gel-casting mold into the frame (rubber ends against the walls). Place the combs onto the frame at the top of the mold.
Pour the agarose solution into the gel-cast (don't overfill) and wait until the gel solidifies before loading your samples (between 1/2 and 1 hour).
IMPORTANT: Place the gel-cast so that the holes are directed towards the negative terminal (black wire).
Cover the gel with 1X TAE buffer (~250 ml) and CAREFULLY remove the combs.
Sample loading
On a parafilm, mix 8 μl of your DNA samples with the amount of loading dye needed for a total reaction volume of 12 µl, then load your samples and 5 µl of DNA ladder separately on different wells.
Run the gel at 100 Volts for 30-40 minutes.
Remove the gel from the chamber and take a photography at the UV transilluminator.
References
Promega- What percentage agarose is needed to sufficiently resolve my DNA sample?
Antibody Staining for FACS  - Protocol
Antibody Staining for FACS  - Protocol
Introduction
Fluorescence Activated Cell Sorting (FACS) is a technique that can separate different cell population or types by using the light scattering and fluoresce characteristics of each individual cell. In this case we want to be able to differentiate the cells that have been activated during our incubation with our vaccine and antigen, and see if there is a population of cell that is presenting the antigen on the surface protein of the cell (MHC I). In order to perform this task we florescence label an antibody against our standard OVA epitope, and after flasking the culture only the cells with the antigen-antibody complex on their surface will present high fluoresce. In order to perform this protocol the cells have to be previously culture and incubated with the encapsulin.
Materials
5 x 105 cells per test - Primary Dendritic Cells extracted from Bone Marrow
FACS buffer: Cold PBS with 0.2% BSA
DAPI working solution: 5 μM DAPI in FACS buffer (diluted from 10X DAPI stock solution)
FACS block buffer: 0.3 μL anti-CD16/32 antibody solution in 20 μL FACS buffer (1:100 dilution; 0.3 μL antibody per test; 20 μL FACS block buffer per test)
FACS antibody cocktail: fluorescent dye-labeled antibodies were prepared according the FACS antibody inventory (X.X μL antibody per test; 20 μL FACS antibody cocktail per test)
Procedure
For 1.5 mL Eppendorf tube:
Wash cells with FACS buffer, centrifuge down by 3000 rcf, 2 min, discard the supernatant by vacuum pump (try to discard all the supernatant, assume the total volume of remaining liquid and cells is 10 μL). Completely resuspend the cell pellet by vortex or pipetting.
Add 20 μL block buffer (total volume: 30 μL). Mix by vortex. Incubate for 15 min in 4 ℃.
Without wash, directly add 20 μL FACS antibody cocktail (total volume: 50 μL). Mix by vortex. Incubate for 20 min in 4 ℃.
Add 1 mL FACS buffer to dilute the mixture, centrifuge down the antibodies-labeled cells by 3000 rcf, 2 min, discard the supernatant by vacuum pump. Completely resuspend the cell pellet by vortex or pipetting.
Add 200 μL FACS buffer or DAPI working solution, filter the cell suspension by 200 nylon mesh or 70 μm cell strainer into a new tube.
For U bottom 96 well plate:
Wash cells with FACS buffer, centrifuge down by 1800 rpm, 5 min, discard the supernatant by throwing once (try to discard all the supernatant, assume the total volume of remaining liquid and cells is 10 μL). Completely resuspend the cell pellet by vortex or pipetting.
Add 20 μL block buffer (total volume: 30 μL). Mix by vortex. Incubate for 15 min in 4 ℃.
Without wash, directly add 20 μL FACS antibody cocktail (total volume: 50 μL). Mix by vortex. Incubate for 20 min in 4 ℃.
Add 200 μL FACS buffer to dilute the mixture, centrifuge down the antibodies-labeled cells by 1800 rpm, 5 min, discard the supernatant by throwing once. Completely resuspend the cell pellet by vortex or pipetting.
Add 200 μL FACS buffer or DAPI working solution, filter the cell suspension by 200 nylon mesh into a new U bottom 96 well plate.
Data collecting by Attune NxT. Acquisition speed: 200 μL /min. Acquisition volume: 100 μL.
Buffer A
Introduction
Buffer A is an essential component of cell-free expression reactions.
Materials
Deionized water
Tris base
Magnesium glutamate (L-Glutamic acid hemimagnesium salt tetrahydrate)
Potassium glutamate (L-Glutamic acid potassium salt monohydrate)
DTT (1,4-Dithiothreitol)
Procedure
Preparation of 1M stock solutions of tris base magnesium glutamate, potassium glutamate
Weight 12.11 g of tris buffer (121.14 g/mol), add it to 100 ml of deionized water and adjust to pH 8.2 with acetic acid.
Weight 38.86 g of magnesium glutamate (388.61 g/mol), add it to 100 ml of deionized water.
Weight 20.32 g of potassium glutamate (203.23 g/mol), add it to 100 ml of deionized water.
Preparation of 1L of buffer A
Add 10 ml of 1 M tris acetate (final concentration is 10 mM).
Add 14 ml of 1 M magnesium glutamate (final concentration is 14 mM).
Add 60 ml of 1 M potassium glutamate (final concentration is 60 mM).
If necessary, add 2 ml of 1 M DTT (final concentration is 2 mM) in order to have buffer A + DTT and store at -20°C.
Complete with deionized water to 1L.
Store at room temperature.
Cell free reaction (standard)
Introduction
In order to test the efficiency of the different components of our cell-free reactions, we perform microplate reader cell-free expression reactions with GFP DNA template. We use the fluorescence emitted by the GFP as a way to measure protein expression. We used this kind of experiments as an internal reference to assess in a relative manner the quality of our lysates and energy solutions.
It is important to test only one component of the cell-free reaction at a time.
We will perform 10µl reactions with 4 repeats of each reaction type. It is crucial to do a negative control per reaction type (we also suggest to do 4 repeats for each negative control).
Materials
Lysate
GFP DNA circular template (10ng/µl of reaction)
Eppendorf tubes
Micro-centrifuge
Mircoplate reader Victor X3 from PerkinElmer
384 Nunc plate from ThermoFisher
Chillout Liquid Wax from BioRad
Procedure
Microplate reader settings
Start preheating the machine to 29°C
Configure the machine for GFP measurement.
Set the parameters as follows: 300 measurement, 90s and quick shaking between each measurement (this corresponds to around 7.5 h expression)
Master mixes preparation
We prepare 2 master mixes: Master Mix 1 (MM1) containing lysate and buffer A, and Master Mix 2 (MM2) containing energy solution, water and DNA (or just energy solution and water for the negative controls).
We calculated the volumes needed based on the following 10µl cell-free expression reaction:
2.5µl Lysate
2.5µl Energy Solution (ES)
2.5µl Buffer A
100ng of DNA
Nuclease free water (complete for a total volume of 10µl)
For each reaction type, we suggest to prepare enough master mixes for 4,5 reactions because of pipetting errors.
Lysate
Buffer A
11.25µl
11.25µl
ES
DNA
Nuclease free water
11.25µl
450ng of DNA (volume is plasmid stock solution concentration dependant)
complete to a total volume of 45µl
ES
Nuclease free water
11.25µl
11.25µl
Put 5µl of MM1 in one corner of each well
Put 5µl of either MM2+ or MM2- in the opposite corner of each well.
Centrifuge the plate for a short time (30-60 seconds) at 4000 rpm to remove any bubbles andcollect the solutions at the bottom of the wells.
Add 35 μl of wax to each well.
Load the plate and start the program.
Competent cell transformation (with Ampicilin)
Introduction
This protocol shows how to transfer plasmid DNA into competent cells.
Materials
Competent cells
Control plasmid
ligation mix
LB-Ampicilin plates
Heating Block
Bunsen burner
Ethanol 96%
Procedure
Add components according to the following table to three tubes of competent cells
Amounts in μl
Transfection mix
Vector control
Competent cells (In tube)
50μl
50μl
Plasmid DNA
5μl
-
Vector
-
5μl
Incubate on ice for 30 min.
Heat shock the cells up to 45 sec. at 42°C. Immediatly transfer the tube back on ice for 5 min.
Spread 50μl.
Incubate the plates overnight at 37°C to select for transformants.
crRNA Transcription using T7 RNA Polymerase (Promega)
Introduction
In this part we're going to transcribe the CRISPR RNA (crRNA) required for the CRISPR-Cas12a assay using the isothermal T7 RNA polymerase. This enzyme will only transcribe DNA downstream of a double-stranded T7                                             promoter (in the 5' to 3' direction), thus we had to fuse the single-stranded DNA (ssDNA) coding sequence (CDS, flanked on its 3' end with the 3' to 5' strand of T7 promoter) with a primer (T7 primer, complementary 5' to 3' )                                             in order to constitute the promoter and obtain efficient transcription. We initially followed the Annealing oligonucleotides protocol in order to anneal the T7 primer to the ssDNA which will constitute our DNA template. This is                                             based on Promega's protocol: "Synthesis of Nonlabeled RNA" [1]
Materials
Transcription Optimized 5X Buffer
DTT, 100mM
Recombinant RNasin® Ribonuclease inhibitor
rATP, rGTP, rUTP, rCTP
DNA template* (annealed ssDNA + primer) in water ( orTE buffer at 2–5μg)
T7 RNA polymerase (Phage RNA polymerase)
Nuclease-Free Water
Procedure
Make the rNTP mix as following
Products
Concentration
rATP
2.5 mM
rGTP
2.5 mM
rUTP
2.5 mM
rCTP
2.5 mM
In nuclease free water
In a microcentrifuge tube, add the following reagents at room temperature in the order listed
Materials
quantity
Transcription Optimized 5X Buffer
20μl
DTT, 100mM
10μl
Recombinant RNasin® Ribonuclease Inhibitor
100 units
rNTP mix
20μl
DNA template, linearized (in water or TE buffer at 2–5μg, i.e. ~100 µM concentrated)*
2μl
Phage RNA polymerase
40 units
Nuclease-Free Water to final volume of
100μl
Incubate for 2 hours at 37°C.
Purify the sample following the RNA purification protocol.
References
Promega: "Synthesis of Nonlabeled RNA" protocol, https://www.promega.com/-/media/files/resources/protocols/product-information-sheets/n/t7-rna-polymerase-protocol.pdf
DPNI plasmid digestion
Introduction
DpnI cleaves only when it's recognition site is methylated. Useful for removing cell-derived plasmid template from PCR samples.
Materials
DPNI
Enzyme buffer (Might work with the one used for the PCR)
PCR product
Procedure
Digest mix
PCR product
50μl
DPNI
1μl
Incubation
Incubate for one hour at 37°C
DPNI heat inactivation
incubate at 80°C for 20 minutes
Energy solution (for cell-free expression) READY
Introduction
Buffer A is an essential component of cell-free expression reactions.
Materials
Procedure
Make amino acids stock solution
In one tube weigh each amino acid one by one using a small parafilm paper.
Add all the amino acids into the same tube apart from the tyrosine that should be added in a separate tube by itself.
To the first tube (amino acid – tyrosine ) add 500 ul of water in total: small volumes of water should be added periodically before adding the 500 ul total. After each addition check for the pH, it should be around 5-6. We can also add small volumes of KOH 1% (100 ul max) to dissolve the amino acids in the solution.
To the tyrosine only tube add 400 ul of water and 10 ul of KOH. Add those volume periodically while vortexing to help the amino acid dissolve in solution. The pH should be around 9.
Do not mix the two tubes together at this stage of the preparation to prevent precipitation.
Making energy solution
After preparing the amino acids, prepare the rest of the components by weighing each time the component in a tube and adding the amount of water listed in the table below. Vortex well so the component dissolves.
After weighing all the components, mix the amino acids and the rest of the solutions together, according to the table.
Aliquot into small 25ul tube and flash freeze in liquid nitrogen.
Store in a -80°C.
Fluorophore-Quencher reporter Cas12a assay
Introduction
This assay's purpose is to detect a specific DNA sequence (the activator) using the CRISPR/Cas12a system. Cas12a's feature is to cleave any ssDNA in the sample once it has found its target. We used this to our                                               advantage and used a single-stranded fluorophore-quencher reporter (DNaseAlert) to be able to quantify our sequence of interest using a plate reader. This protocol is based on and optimized from the LbCas12a collateral                                               detection protocol ("Fluorophore quencher (FQ)-labeled reporter assays")[1]
Materials
EnGen® Lba Cas12a (Cpf1) 1 μM
Purified crRNA from DNA sequence 1μM
10X Binding buffer (200 mM Tris-HCl, pH 7.5, 1 M KCl, 50 mM MgCl2, 50% glycerol, 500 μg/ml heparin, 1mM DTT)
DNaseAlert™ (IDT) 1 µM
Nuclease-Free Water
6 of small pcr tubes (0.2 ml) for your master mix and samples
DNase I enzyme (Zymo)
384 well Plate
Multi Well Plate Sealing Films
Procedure
Preparation of Cas12a master mix
LbCas12a
concentration
gRNA
concentration
Activator
concentraion
DNaseAlert
concentration
Buffer
Dnase I
FQ1
62.5 nM
75 nM
Variable
160 nM
1X Binding
-
Negative control
62.5 nM
75 nM
-
160 nM
1X Binding
-
Blank
-
-
-
-
1X Binding
-
Positive control
-
-
-
160 nM
1X Binding
2.4 μl
Add the following materials except the DNaseAlert and activator into a 0.2 ml tubes. Incubate at 37°C for 30 min, then add the rest of the components.
Components
FQ1
Negative
control
Blank
Positive
Control
10X Binding Buffer
6.6
6.6
6.6
6.6
Cas12 [1µM]
3.75
3.75
-
-
crRNA [1µM]
4.5
4.5
-
-
Nuclease Free Water
29.6
35.6
53.4
41.4
Incubation 30 minutes at 37°C
DNase Alert
9.6
9.6
-
9.6
Activator
6
-
-
-
DNase I
-
-
-
2.4
Final volume (µL)
60
60
60
60
Prepare the Optical Plate
Load 24 µl of each corresponding tubes into a 384 opti plate (in duplicate) in the following order:
Negative
FQ1
blank
Positive control
Negative
FQ2
blank
Positive control
Stick an adhesive film on the top of the plate.
Put it in the plate reader. Set up the device: at 535 nm excitation and 590 nm emission, 37℃ and take the measurements every 20 seconds for 180 repeats.
Plot a graph of the fluorescence as a function of time (minutes), taking the average of the fluorescence obtained for each well.
References
[1]Chen, J. S., Ma, E., Harrington, L. B., Da Costa, M., Tian, X., Palefsky, J. M., & Doudna, J. A. (2018). CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity. Science, 360(6387), 436–439.
Gel extraction
Introduction
In order to test the efficiency of the different components of our cell-free reactions, we perform microplate reader cell-free expression reactions with GFP DNA template. We use the fluorescence emitted by the GFP as a way to measure protein expression. We used this kind of experiments as an internal reference to assess in a relative manner the quality of our lysates and energy solutions.
It is important to test only one component of the cell-free reaction at a time.
We will perform 10µl reactions with 4 repeats of each reaction type. It is crucial to do a negative control per reaction type (we also suggest to do 4 repeats for each negative control).
Materials
Lysate
GFP DNA circular template (10ng/µl of reaction)
Eppendorf tubes
Micro-centrifuge
Mircoplate reader Victor X3 from PerkinElmer
384 Nunc plate from ThermoFisher
Chillout Liquid Wax from BioRad
Procedure
Microplate reader settings
Start preheating the machine to 29°C
Configure the machine for GFP measurement.
Set the parameters as follows: 300 measurement, 90s and quick shaking between each measurement (this corresponds to around 7.5 h expression)
Master mixes preparation
We prepare 2 master mixes: Master Mix 1 (MM1) containing lysate and buffer A, and Master Mix 2 (MM2) containing energy solution, water and DNA (or just energy solution and water for the negative controls).
We calculated the volumes needed based on the following 10µl cell-free expression reaction:
2.5µl Lysate
2.5µl Energy Solution (ES)
2.5µl Buffer A
100ng of DNA
Nuclease free water (complete for a total volume of 10µl)
For each reaction type, we suggest to prepare enough master mixes for 4,5 reactions because of pipetting errors.
Lysate
Buffer A
11.25µl
11.25µl
ES
DNA
Nuclease free water
11.25µl
450ng of DNA (volume is plasmid stock solution concentration dependant)
complete to a total volume of 45µl
ES
Nuclease free water
11.25µl
11.25µl
Put 5µl of MM1 in one corner of each well
Put 5µl of either MM2+ or MM2- in the opposite corner of each well.
Centrifuge the plate for a short time (30-60 seconds) at 4000 rpm to remove any bubbles andcollect the solutions at the bottom of the wells.
Add 35 μl of wax to each well.
Load the plate and start the program.
Glycerol stock preparation
Introduction
This is how to make glycerol stocks of bacteria cell cultures that are suitable for long time storage
Materials
Liquid cell culture
Glycerol
1.5 ml tube
Procedure
After you have bacteria growth in your liquid culture, add 500μl of overnight culture to 500μl of 50% glycerol in the 1.5ml tube and gently mix
Freeze the glycerol stock tube at -80°C. The stock is now available for years as long as its kept at -80°C.
To remove bacteria from the glycerol stock, open the tube and use a sterile tip to scrape some of the frozen bacteria.
gRNA purification (ZYMO Research RNA Clean & Concentrator™-5 Kit)
Introduction
Purification of newly transcribed crRNA (T7 RNA polymerase; Promega1), following the ZYMO Research RNA purification kit (RNA Clean & Concentrator™-5) protocol [2].
Materials
Purification (RNA Clean & Concentrator™-5; ZYMO Research)
RNA Binding Buffer
RNA Prep Buffer
RNA Wash Buffer
DNase I
DNA Digestion Buffer
DNase/RNase Free Water
Zymo Spin IC Columns
Collection Tubes
RNase-free Microfuge Tubes (1.5 mL) Not provided with the kit.
Procedure
DNase I treatment (Before clean-up)
IMPORTANT: Prior to use, reconstitute the lyophilized DNase I as indicated on the vial. Store frozen aliquots.
For each sample to be treated, prepare DNase I reaction mix in an RNase-free tube (not provided). Mix well by gentle inversion, once the following components added (volume of components to add to the RNA sample can be                                                       readjusted according to the sample's volume)
Product
Volume
RNA sample (≤10 μg) volume adjusted with water or TE buffer
40 μl
DNase I
5 μl
DNA Digestion Buffer
5 μl
Total volume
50 μl
Incubate at room temperature (20-30ºC) for 15 minutes.
Buffer preparaiton
Before starting, add 48 ml 100% ethanol (52ml 95% ethanol) to the 12 ml RNA Wash Buffer concentrate(R1013, R1015) or 96 ml 100% ethanol (104ml of 95% ethanol) to the 24 ml RNA Wash Buffer concentrate (R1014, R1016).
Wash
All centrifugation steps should be performed at 10,000 –16,000 x g. RNA species ≥17 nt will be recovered.
Add 2 volumes RNA Binding Buffer to each sample and mix. Example: Mix 100 μl buffer and 50 μl sample.
Add an equal volume of ethanol (95-100%) and mix. Example: Add 150 μl ethanol.
Transfer the sample to the Zymo-Spin™IC Column in a Collection Tube and centrifuge for 30 seconds. Discard the flow-through.
Add 400 μl RNA Prep Buffer to the column and centrifuge for 30 seconds. Discard the flow-through.
Add 700μl RNA Wash Buffer to the column and centrifuge for 30 seconds. Discard the flow-through.
Add 400 μl RNA Wash Buffer to the column and centrifuge for 2 minutes to ensure complete removal of the wash buffer. Transfer the column carefully into an RNase-free tube (not provided in the kit).
Elution
Add 15μl DNase/RNase-Free Water directly to the column matrix and centrifuge for 30 seconds. (Alternatively, for highly concentratedRNA use ≥ 6μl elution).The eluted RNA can be used immediately or stored at -70°C.
References
Promega: "Synthesis of Nonlabeled RNA" protocol, https://www.promega.com/-/media/files/resources/protocols/product-information-sheets/n/t7-rna-polymerase-protocol.pdf
ZYMO Research: RNA Clean & Concentrator™-5, Instruction Manual, https://www.zymoresearch.eu/media/amasty/amfile/attach/_R1013_R1014_R1015_R1016_RNA_Clean_Concentrator-5_ver2.2.1.pdf
Inoculating cultures
Introduction
This protocol explains how to inoculate cultures to grow bacterial clones.
Materials
LB ampicillin plates from our transformation
LB ampicillin medium
14ml sterile round tubes with dual position snap cap
sterile tips
shaker at 37°C
Procedure
Pick a colony from the ligation plate using a sterile tip
Shake the tip into a bacterial culture tube containing 3ml of LB/Amp medium so the colony mixes with the medium
Close tubes (loose position for sterile aerobic culturing)
Put your tubes onto a shaker at 37°C and incubate overnight with agitation at 225 rpm
Lysate preparation
Introduction
Lysate is an essential component of cell-free expression reactions.  This protocol is taken from the iGEM EPFL 2017 wiki.
Materials
Lysate
Autoclave
Autoclaved LB medium
Autoclaved tips and dishes Flame
Buffer A
Buffer A + DTT
Tubes for bacterial culture
Spectrophotometer
Plastic cuvettes
50 ml Falcon tubes
Liquid nitrogen
Big and small centrifuge Sonicator
IPTG (0.1 M: 0.2383 g of IPTG which Mw = 238.3 g/mol, and fill up to 10 mL with deionized water)
Procedure
Preparation of Materials (For 2 x 200 ml of bacterial culture)
Prepare minimum 420 ml of LB medium and autoclave for 20 min at 121ºC
Autoclave tips and dishes, which will be used for bacteria growth
Prepare minimum 120 ml buffer A (- DTT)
Prepare minimum 3 ml buffer A + DTT
Bacteria growth
When working with bacteria, always work next to the flame to avoid contaminations and wear glasses when working with liquid nitrogen.
At 17h approximatively, add 5 ml of LB medium and antibiotics according to resistance’s cells to a tube and inoculate with a small amount of desired bacteria from the glycerol stock (pick by tip or inoculating loop). Put the top of the bottle of the LB and the cap under the flame to avoid contamination of LB medium (contaminate very easily). (Note: No antibiotics is used when preparing BL21 DE3 lysates)
Grow the culture overnight at 37°C, 200rpm, pay attention to keep the lid loose to allow for air exchange.
At 9h approximately, measure absorbance at 600 nm with 10X dilution of the culture (900 μl of LB + 100 μl of bacteria culture) in a plastic cuvette, OD600 should be around 4.
Add 1 ml overnight culture to 200 ml of LB medium and correct antibiotics in a 500 ml Erlenmeyer flask.
Incubate the culture for 2 hours at 37°C, 200 rpm (pay attention to the rpm setting – this speed is optimal for the growth).
Induce if necessary: 400 μl of 100 mM IPTG for T7 polymerase inducing; 5 ml of 10% arabinose for gamS inducing (under pBAD promoter).
Incubate again the culture for 2 hours at 37°C, 20 0rpm and measure absorbance, OD600 should be around 1.5 – 2.
Centrifugation and cleaning
Let the big centrifuge cool down to 4ºC (takes around 10 min).
Weigh one 50 ml falcon tube for one culture (clearly label the tube as well the cap).
Separate the grown culture to four 50 ml falcon tubes and spin at 4000 rpm at 4°C for 20 min.
Keep the bacteria on ice as much as possible.
Discard the supernatant and add 10 ml of buffer A to the bacteria pellet, re-suspend the bacteria by pipetting or by vortexing and transfer all four parts to one tube, spin at 4000 rpm at 4°C for 10 min.
Discard the supernatant and re-suspend the pellet in 10 ml of buffer A, spin at 4000 rpm at 4°C for 10 min.
Re-do the washing step 6 one more times.
Discard the supernatant (remove as much as possible).
Weigh the pellet (wet mass).
Wear protective glasses (danger of tube explosion) and flash freeze with liquid nitrogen, store at –80°C.
Sonication
Re-suspended the thawed bacteria based on their mass in buffer A by vortexing (1ml of buffer A + DTT per grams of thawed bacteria).
Keep the bacteria or lysate on ice as much as possible.
Aliquot 1 ml of re-suspended bacteria to a new 2 ml Eppendorf tube.
Place the tube in an ice bath (ice mixed with water) and place the sonicated tip inside the tube so it is immersed as much as possible in the liquid without touching the surface of the tube.
Sonicate with 50% amplitude and pulsing 10 s:10 s (energy : pause) until you reach 400 J (around 1 min and 24 s).
Separation
Let the small centrifuge cool down to 4ºC (takes around 20 min).
Centrifuge the lysed bacteria at 12000 rpm and 4°C for 10 min.
Transfer the supernatant (top layer) to a new tube. To prevent any transfer of bacteria debris, do not take all the lysate.
Place the lysate to an incubator at 37 °C and 200 rpm for 1h30 min, this run off reaction will lead to degradation of the rest of the DNA.
Centrifuge the run off reaction at 12000 rpm at 4°C for 10 min.
Transfer the supernatant to a new tube. As before, to prevent any transfer of bacteria debris, do not take all the lysate.
Aliquot the lysate to 0.5 ml microfuge tubes according to your needs (around 25 μl is recommended) and keep 1 μl of the lysate for the Bradford assay.
Flash freeze the aliquots in liquid nitrogen and store at -80°C (wear glasses).
Bradford assay
Perform the Bradford assay to determine the protein concentration: dilute 1 μl of the lysate in 99 μl of buffer A
Mix 5 μl of the diluted lysate with 250 μl of the Bradford reagent.
Let the reaction incubate for 5 min
Measure the absorbance using the Nanodrop device, protein concentration should be between 400 and 800 μg/ml
Microplate reader cell-free reaction (circular DNA template)
Introduction
In order to test the efficiency of the different components of our cell-free reactions, we perform microplate reader cell-free expression reactions with GFP DNA template. We use the fluorescence emitted by the GFP as a way to measure protein expression. We used this kind of experiments as an internal reference to assess in a relative manner the quality of our lysates and energy solutions.
It is important to test only one component of the cell-free reaction at a time.
We will perform 10µl reactions with 4 repeats of each reaction type. It is crucial to do a negative control per reaction type (we also suggest to do 4 repeats for each negative control).
Materials
Lysate
Energy solution
Buffer A
GFP DNA circular template (10ng/µl of reaction)
Eppendorf tubes
Micro-centrifuge
Mircoplate reader Victor X3 from PerkinElmer
384 Nunc plate from ThermoFisher
Chillout Liquid Wax from BioRad
Procedure
Microplate reader settings
Start preheating the machine to 29°C
Configure the machine for GFP measurement.
Set the parameters as follows: 300 measurement, 90s and quick shaking between each measurement (this corresponds to around 7.5 h expression)
Master mixes preparation
We prepare 2 master mixes: Master Mix 1 (MM1) containing lysate and buffer A, and Master Mix 2 (MM2) containing energy solution, water and DNA (or just energy solution and water for the negative controls).
We calculated the volumes needed based on the following 10µl cell-free expression reaction:
2.5µl Lysate
2.5µl Energy Solution (ES)
2.5µl Buffer A
100ng of DNA
Nuclease free water (complete for a total volume of 10µl)
For each reaction type, we suggest to prepare enough master mixes for 4,5 reactions because of pipetting errors.
Lysate
Buffer A
11.25µl
11.25µl
ES
DNA
Nuclease free water
11.25µl
450ng of DNA (volume is plasmid stock solution concentration dependant)
complete to a total volume of 45µl
ES
Nuclease free water
11.25µl
11.25µl
Put 5µl of MM1 in one corner of each well
Put 5µl of either MM2+ or MM2- in the opposite corner of each well.
Centrifuge the plate for a short time (30-60 seconds) at 4000 rpm to remove any bubbles andcollect the solutions at the bottom of the wells.
Add 35 μl of wax to each well.
Load the plate and start the program.
Oligomer Phosphorylation
Introduction
This protocol is used to phosphorylate the 5' ends of inserts used in a subsequent Golden Gate ligation reaction
Materials
Forward Oligo 100 μM
Reverse Oligo 100 μM
T4 DNA Ligase Buffer 10X
PNK
NFW
NaCl 2M aqueous solution
Procedure
In a PCR tube mix the following (total volume 29 μL):
3 μL Forward Oligo 100 μM
3 μL Reverse Oligo 100 μM
3 μL T4 DNA Ligase Buffer 10X
2 μL PNK
18 μL water
Incubate the mixture for 2 hours at 37C
Heat inactivate PNK at 65C for 20 minutes
Add 1 μL of 2 M NaCl aqueous solution
Heat to 98C for 2 minutes then slowly ramp down to room temperature and hold at 4C when finished
Oligonucleotides/gBlocks Resuspension and Storage (IDT)
Introduction
Recommendations on how to resuspend and store oligos or/and gBlocks gene fragments (IDT) once shipped. Check IDT's page: "My oligos have arrived: Now what?"1 for more details.
Materials
Oligos or gBlocks gene fragments
Nuclease-free water/TE buffer
Procedure
Briefly centrifuge the tubes before opening them
the oligos should be resuspended in TE buffer (10 mM Tris, 0.1 mM EDTA, pH 8.0). Nuclease-free water (pH 7.0) may be used alternatively. However, use of HPLC- or molecular biology–grade water is preferable. CAUTION:                                                           Nuclease-free water will not modulate pH over time as will TE buffer.
Standard recommendation: Resuspend oligos to a 100 µM stock concentration, The volume of TE buffer required to achieve a 100 µM stock is easily determined by multiplying the number of nanomoles (nmol) listed for a                                                           particular oligo by a factor of 10, and then resuspending the dry DNA in that same number of microliters of TE buffer. For example, if the oligo specification sheet states that 20.3 nmol of oligo were delivered, add 203 µL TE                                                           buffer to obtain a 100 µM stock solution. This stock solution can be diluted as needed into appropriate working solutions.
For oligos that are harder to resuspend, and for which one might observe residual precipitate present following resuspension, the oligo should be heated at 55°C for 1–5 minutes, vortexed thoroughly, and then briefly                                                           centrifuged.
Storage
Store your resuspended oligonucleotides at -20°C (stable for at least 24 months when either dried down, or resuspended in TE buffer or nuclease-free water).
References
[1] Integrated DNA Technologies (IDT): "My oligos have arrived: Now what?", https://eu.idtdna.com/pages/education/decoded/article/my-oligos-have-arrived-now-what-; "Tips for resuspending and diluting your oligonucleotides",                                                           https://eu.idtdna.com/pages/education/decoded/article/tips-for-resuspending-and-diluting-your-oligonucleotides
PCR amplification in Plasma
Introduction
This is a protocol on how to amplify DNA fragments in plasma. Since we are going to target specific DNA fragments with our CRSPR-cas12a assay following this PCR we will add ourselves the oligos in our sample. If                                                           you want to amplify DNA that is already contained in your sample skip the additon of DNA in plasma. This protocol is based on the one found in PCR protocols by Professor Kenji Abe1.
Materials
DNA template
PCR primers
dNTPs (10 mM)
Nuclease-Free water
5X Phusion HF buffer
Phusion DNA polymerase
Human Blood Plasma
10x Phosphate-buffered Saline (PBS)
Thermal cycler
Procedure
Plasma samples will be dilutated 1:5 in 1x PBS but as not to dilute the plasma too much we will proceed by adding the DNA template in PBS in the following way
Make a mastermix of 1 part plasma (4μl) in 3 part PBS diluted as following: 1.6μl 10X PBS in 10.4μl NFW
We want the PBS to be 1X in 4 part of the sample (16μl) but one part is used to put the templated. Once we add the last part the PBS will become 1X.
Put 8μl of the master mix in two different tubes and make the two following samples:
Components
Sample A
Negative
control
Template
1µl
-
Nuclease free water
1μl
2µl
Total
2μl
2µl
The diluted plasma sample are heated for 3 min at 95℃ then cooled rapidly on ice for 3 to 5 min.
Make the following MasterMix
Components
A
Negative
control
MasterMix 2.5x
Final concentration
NFW
28.5
28.5
71.25
-
5X Phusion HF buffer
10
10
25
1X
10 mM dNTPs
1
1
2.5
200 µM
Forward primer (10 μM)
2.5
2.5
6.25
0.5 µM
Reverse primer (10μM)
2.5
2.5
6.25
0.5 µM
From Master mix
44.5
44.5
111.25
-
Sample A
-
5
-
-
Negative control
5
-
-
Variable
Phusion DNA Polymerase
0.5
0.5
-
1.0 units/50 µl PCR
Total volume
50
50
-
-
Gently mix the reaction. Transfer PCR tubes from ice to a PCR machine with the block preheated to 98 °C and begin thermocycling:
Temperature
Time
Number of cycles
98 °C
30 sec (Initial Denaturation)
1
98 °C
10 sec (denaturation)
30 cycles
Variable
30 sec (primer annealing)
30 cycles
72 °C
30 sec (extension)
30 cycles
72 °C
10 min (Final Extension)
1
4 °C
infinite
References
1 Abe, Kenji. « Direct PCR from Serum ». PCR Protocols, edited by John M. S. Bartlett and David Stirling, Humana Press, 2003, p. 161‑66. Springer Link, doi:10.1385/1-59259-384-4:161
PCR using Phusion® High-Fidelity DNA Polymerase
Introduction
A standard PCR protocol using Phusion high fidelity DNA polymerase. Mainly used for amplifying the junction/point mutated fragments we aim to detect using CRISPR-Cas12a. The protocol is based on NEB PCR Protocol                                                             for Phusion® High-Fidelity DNA Polymerase (M0530)1.
Materials
DNA template
Forward primer
Reverse primer
dNTPs, 10 mM
Nuclease-Free water
5X Phusion high fidelity (HF) buffer
Phusion DNA polymerase (ThermoFisher, 2 U/µl)
Thermal cycler
Procedure
The thermal cycler (PCR machine) should be programmed as following. IMPORTANT: Annealing temperature may vary depending on the primer's melting temperature (Tm).
Step
Temperature
Time
Cycles
Initial denaturation
98°C
30 sec
-
Denaturation
98°C
10 sec
35
Primer annealing
45-72°C (depending on primers' Tm)
30 sec
35
Extension
72°C
30 sec
35
Final extension
72°C
10 min
-
-
4°C
-
-
For one sample, add the following amounts of products. If working with several samples, a master mix containing the elements that do not change between each sample (water, buffer, dNTPs or/and primers/DNA) can be prepared.                                                               However, ALWAYS add the enzyme in the end.
Component
Volume µl
Final concentration
Nuclease-free water
Up to 50 µl
-
5X Phusion HF buffer
10
1X
10 mM dNTPs
1
200 µM
10 µM Forward primer
2.5
0.5 µM
10 µM Reverse primer
2.5
0.5 µM
Template DNA
variable
Up to 250 ng
Phusion DNA Polymerase
0.5
1.0 units/50 µl PCR
Total volume
50
-
Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary. Transfer PCR tubes from ice to a PCR machine (programmed previously) and begin thermocycling.
Control your samples by doing an agarose gel electrophoresis following the Agarose gel electrophoresis protocol.
References
1. New England BioLabs (NEB). PCR Protocol for Phusion® High-Fidelity DNA Polymerase (M0530): https://international.neb.com/Protocols/0001/01/01/pcr-protocol-m0530
Preparation of dumbbell probes
Introduction
The goal is to prepare dumbbell probes in order to amplify miRNAs by Rolling Circle Amplification (RCA).
Materials
2 μL DNA template
1μL T4 polynucleotide kinase
1 μL ( 100 U/μL ) T4 ligase
4 μL T4 DNA ligase reaction buffer (x10) (2μL for the phosphorylation and 2μL for the ligation)
400 mM Tris-HCl, 100 mM MgCl2, 100 mM Dithiothreitol, 5 mM ATP, pH 7.8 at 25 °C
22 μL DEPC-treated H2O (15 μL for phosphorylation and 7 for ligation)
Exonuclease I (20 U/μL) and Exonuclease III (100 U/μL)
Procedure
Phosphorylation of the oligos
In the IDT tubes, put the amount of water to get 100μM of DNA probe [you take the number of moles N and suspend in a volume of 10*N μl]
In a tube, put 2 μL oligos, 15 μL of water, the T4 ligase buffer (2 μl), and finally 1 μL of the kinase.
Incubate the mixture at 37°C for 1 hour.
Heat at 60°C for 20 min to inactivate the enzyme.
The buffer has to be new ( less than 1 year) and we should avoid repeated freeze-thaw cycle with it.
Ligation of the probes
Add in a reaction tube 10 μl (because at the end of phoshorylation is 10 μM and not 100 μM) of DNA template, the T4 ligase (1 μl), the reaction buffer (2 μl) and 7 μl DEPC-treated water.
Put the tube at 16°C for 2 hours to process the ligation.
Then heat at 65°C for 10 min to terminate the reaction.
Add the exonucleases (1μl each - total volume of 22 μl) and incubate the reaction mixture at 37°C for 2 hours.
Then the enzymes are denatured by heating at 80°C for 20 min.
The ligation can be controlled by electrophoresis on agarose gel (1.5%).
Real-time fluorescence measurement of RCA
Introduction
The goal is to make sure that the RCA worked fine for different concentration of probes and miRNAs. The initial assessement was done with 0.5% agarose gel but the results were difficult to interpret. We are using 25x                                                                 SYBR Green I, an intercalating                                                                 agent for dsDNAs that is also fluorescent (Excitation wavelength is 494 nm, emission wavelength is 521nm).
Materials
1 μL prepared probes
2.5μL phi29 DNA polymerase reaction buffer (x10)
500 mM Tris-HCl, pH 7.5 at 25°C, 100 mM MgCl2, 100 mM (NH4)2SO4, 40 mM Dithiothreitol
0.25 μL BSA (20 mg/mL)
6 μL dNTPs(10 mM for each)
2.5 μL of target miRNA solution
11.75 μL DEPC-treated H2O
0.5 μL phi29 DNA polymerase (10 U/μL)
0.5 μl 25x SYBR Green I
Procedure
Fluorescence measurement using SYBR Green I
The SYBR Green I that we purchased is optimised for Gel and so it was very concentrated (x10000) so we have to dilute it to 25x with the polymerase buffer.
Put all the components in a tube
Inject 24 (25) μl in a 96-wells plate and put it in the plate reader.
The reaction should be at 37°C. The florescence is measured every 2 min during 180 min under excitation and emission wavelengths of 495(497) and 515(520) nm, respectively.
Rolling Circle Amplification
Introduction
The goal here is to amplify miRNAs by RCA (Rolling circle amplification). The dumbbell probes are designed in order to get a complementary region with specific miRNAs. The miRNAs bind to this region and the probes                                                                       become circular and                                                                       the amplification can begin. We finally obtain a concatemer (long continuous DNA molecule that contains multiple copies of the same DNA sequence linked in series).
Materials
1 μL prepared probes
2.5μL phi29 DNA polymerase reaction buffer (x10)
500 mM Tris-HCl, pH 7.5 at 25°C, 100 mM MgCl2, 100 mM (NH4)2SO4, 40 mM Dithiothreitol
0.25 μL BSA (20 mg/mL)
6 μL dNTPs (10 mM for each)
2.5 μL of target miRNA solution
12.25 μL DEPC-treated H2O
0.5 μL phi29 DNA polymerase (10 U/μL)
2 μL SYBR I SYBR I (x10)
Procedure
Amplification of the miRNAs
Add all the component, except the last one in a 25μL mixture tube.
Incubate the mixture at 37°C for 2h
Heat it at 65°C for 10 min to stop the reaction.
The mixture can be analysed by using electrophoresis or fluorescence analysis.
SDS-PAGE for protein electrophoresis
SDS-PAGE for protein electrophoresis
Introduction
PAGE (polyacrylamide gel electrophoresis) is a technique allowing to separate charged molecule according to their molecular masses. SDS-PAGE (sodium dodecyl sulfate–polyacrylamide gel electrophoresis) is a variant                                                                       of PAGE allowing to                                                                       separate protein molecules according to their molecular masses. SDS (sodium dodecylsulphate) is a negatively charged molecule which will bind to proteins as they are heat denatured and confers them a charge nearly proportional to                                                                       their                                                                       length (and hence their mass). It therefore allows to separate proteins according to their molecular masses.
Materials
Laemmli buffer (SDS-PAGE loading buffer, contains SDS and DTT)
Commercial polyacrylamide gels in glass plates
PAGE machine
Running buffer (1X TGS buffer)
Long pipet tips
Procedure
Sample preparation
Mix 10µl of Laemmli with 5µl of protein sample and 5µl nuclease free water in a tube. Take care to mix well the sample tube before by flicking it gently several times.
Incubate at 100°C for 15min in a dry heating block compatible with the tube used (e.g. for PCR tubes use the ThermoCycler). Do not forget to use lids heating at 105°C to avoid condensation.
The samples can now be stored at 4°C until they are loaded on the gel.
PAGE-machine preparation
Verify that you have all the machine's components (tank, lid with power cables, electrode assembly, cell buffer dam, casting frames and stands) as well as a new polyacrylamide gel in glass plates.
Rince all the components with distilled water.
Open the gel and remove the plastic lid that protects the bottom of the gel.
Open the casting frame and put the gel inside. Close it again.
Test waterproofness by puting new running buffer inside (here it is important to use new running buffer!). Do it above the tank (in case it is not water proof).
Put the casting frame in the tank. Take care of the electrodes' colors! The black wire should be above the black electrode, same for the read one.
Fill the part of the tank outside of the casting stand with running buffer (you can put already used running buffer here.) You should fill it until the black line drawn on the tank (the 2 gels line if you run with only 1                                                                         casting stand, the 4 gel one if you run with 2 casting stands.)
Close the tank with the lid with powe cables.
Loading the gel
Use long tips! Take care to pipet really slow in order to have the right volume. Long tips are very thin and tend not to fill completely.
Load 10µl of sample or 5µl of ladder per well.
Go to the glass plate in front of you and try to put your tip a bit in the well. Once you think you are in, try to go back and forth. If your land in the middle of the buffer, it means that you were not really in the well!                                                                         You should only be able to move between the 2 glass plates in which your gel is. Once your in, dive a bit deeper in the well and load the content of your pipet.
Tip: put one ladder and one negative control per gel!
PAGE run
Set the machine on 30min and 120V. You can make sure the machine is running by looking below the gel. You should see bubbles forming.
If you do not see any bubbles, this means that the electric circuit is not closed. This could be due to the oversight to remove the plastic lid at the bottom of the gel or to the fact that there is not enough buffer in the                                                                         tank or between the 2 gels. Sometimes it is necessary to put more buffer than the level indicated on the tank.
After 5min of run, set the voltage on 200V.
Gel wash
Take the casting stand out and put the now used running buffer in a bottle. You can reuse it multiple times.
Put the running buffer of the tank in the same bottle.
Remove the gel of the casting frame and open it carefully using the specialized metal instrument (or just a metallic spatule if you do not have one).
Put the gel with distilled water in a cylinder bowl that you can close. In order to transfer the gel from the glass plate to the bowl without breaking it, you can flip
Pour distilled water in order to fill the bowl up to 4cm.
Mix a bit and throw the water away. You can use your fingers to prevent the gel to fall in the sink.
Pour distilled water in the bowl and put it on the shaker for 10min.
Coomassie staining
Put on the shaker for 1h.
Wash twice with distilled water and recover the gel with distilled water. Let on the shaker overnight.
Gel visualisation
Your gel is now ready to be visualized. To take it out of the bowl, it is better that it is not at the bottom of the bowl. You can pour some water at one border of the bowl in order to make it float a bit and then slip your                                                                         hand (with clean gloves!) underneath. Then put the gel in a plastic sleeve.
In order to visualize the gel, you can use a scanner (gives the best quality), or the white light mode of an UV transilluminator.
Standard heat purification for proteins
Standard heat purification for proteins
Introduction
When the protein of interest is heat stable, the heat purification method is a straightforward way to get rid of the majority of non-desired proteins of a sample.
Materials
Protein samples to purify
Eppendorf tubes
Heating block compatible to the tubes used
Ice
Microcentrifuge
Procedure
Denaturation
Heat at 70ºC for 20 min.
Put on ice for 15 min.
Centrifugation
Centrifuge at 12000 g for 10 min.
The protein of interest is now mainly located in the supernatant of the solution.
Vaccine Presentation to Immature Dendritic Cells
Vaccine Presentation to Immature Dendritic Cells
Introduction
This protocol uses murine dendritic cells extracted from bone marrow for vaccine representation.
Materials
Immature murine bone marrow dendritic cells
RPMI medium
RPMI (with 2 ng/ml of GD-CSF)
24 well plate
Conical Tubes
Procedure
Cell Counting and Resuspention
Cells are supplied in a plate. There are usually around 6 million cells in 10 ml of RPMI in a plate.
Collect the cells and transfet them into a 50 ml conical tube.
Resuspend into a new concentration of 1 million cells per ml pf RPMI after centrifugation and cell counting.
Cell Plating
Add 1 ml of RPMI ( with 2 ng/ml GF-CSF) into each well of a 24 well plate
Pipette 0.5 ml of the cell stock (at 1 million cell/ml) to each well
Add to each well the different condition (Negative Control, Vaccine,...). The volume should range between 50 and 100 µl.
Even out the mixture by gently shaking the well.
Incubate for 18-24 hrs at 37 ºC.
The cells are now ready for Antibody Staining for FACS
3D culture of dendritic cells in Matrigel
3D culture of dendritic cells in Matrigel
Introduction
This protocol is based on Corning’s protocol: Corning® Matrigel® Basement Membrane Matrix for 3D Culture In Vitro.
Materials
Immature murine Bone Marrow Dendritic Cells C57BL/6
Corning Matrigel matrix (Corning Cat. No. 356234)
RPMI medium (with GM-CSF at 2ng/ml)
PBS
Ibidi µ-Dish 35 mm, low
Expressed and purified fluorescent Encapsulin
Procedure
Thaw Matrigel matrix overnight by submerging the vial in a 4°C refrigerator before use. Once Matrigel matrix is thawed, swirl vial to ensure the material is dispersed.
Resuspend the immature dendritic cells to a concentration of 3.81*10^5 cells/ml in RPMI (with 2 ng/ml).
Add 100 µl of Matrigel to 250 µl of the dendritic cell suspension, and mix well.
Plate onto the ibidi µ-Dish 35 mm, low.
Incubate at 37 ºC for 20 min.
Bring the dish outside for imaging.
Present the fluorescent encapsulin to the dendritic cell by adding 50 µl of the expressed and purified fluorescent encapsulin.
Put back to the 37ºC incubator.
One hour post presentation, bring the dish outside for imaging, then put the dish back  in the incubator.
Four hours post presentation, bring the dish out for imaging, then put the dish back into the 37 ºC incubator.
Six hours post presentation, bring the dish out for washing. Wash twice by tilting the dish and pipetting 1 ml of PBS onto the higher side of the dish, so that the PBS runs through the Matrigel. Collect the wash on the other side of the dish.
Do imaging post wash.
Note that the microscope used with this protocol is NanoLive’s precision Tomographic Microscope. This technology allowed us to make 3D images and 2D fluorescence images of the cells.
iGEM2018@groupes.epfl.ch
IGEM EPFL 2018
Twitter Facebook
