<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM EPFL 2018</TITLE></P><DIV class="container"><A class="navbar-brand mr-lg-5" href="https://2018.igem.org/Team:EPFL"><STRONG>iGEM</STRONG> EPFL 2018</A><DIV class="collapse navbar-collapse" id="navbar_main_collapse"><UL class="navbar-nav align-items-lg-center ml-lg-auto"><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:EPFL">HOME</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Description" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Description</H2><P class="mb-0">Take a closer look at CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Design" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Design</H2><P class="mb-0">Learn more about how we designed CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Software" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Software</H2><P class="mb-0">Explore the Bioinformatic Pipeline implemented in CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Results" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Results</H2><P class="mb-0">Discover the Results of our experiments</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Demonstrate" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Demonstration</H2><P class="mb-0">Demonstration of how our Therapeutic System works</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/InterLab" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Interlab</H2><P class="mb-0">Learn more about our participation in the Interlab Study</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Parts" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Parts</H2><P class="mb-0">Discover the Parts that make up CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Improve" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Improve</H2><P class="mb-0">Explore our Novel and Improvements Parts</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Measurement" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Measurement</H2><P class="mb-0">Learn about measurement approaches we used for Part Characterization</P></DIV></DIV></A></DIV></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Team" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Team Members</H2><P class="mb-0">Learn more about our Team Members</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Collaborations" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Collaborations</H2><P class="mb-0">Discover the interactions we had with other iGEM Teams during CAPOEIRA</P></DIV></DIV></A></DIV></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Human_Practices" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Integrated Human Practices</H2><P class="mb-0">Discover how we integrated Human Practices into the Purpose, Design and Implementation of CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Ethics" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Ethics</H2><P class="mb-0">Explore the ethical considerations of CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Applied_Design" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Applied Design</H2><P class="mb-0">Explore how our therapeutic system attempts to meet real world medical need</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Public_Engagement" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Education and Public Engagement</H2><P class="mb-0">Discover our Education and Public Engagement work with the public and scientific communities</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Meetups" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">European and Eurasian Meetup</H2><P class="mb-0">Learn about our interactions with the iGEM Community during the Meetups</P></DIV></DIV></A></DIV></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:EPFL/Notebook#">Lab</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:EPFL/Safety">Safety</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:EPFL/Entrepreneurship">Entrepreneurship</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Acknowledgements</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Attributions" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Attributions</H2></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Sponsors" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Sponsors</H2></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Partners" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Partners</H2></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Awards" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Awards</H2></DIV></DIV></A></DIV></DIV></LI></UL></DIV></DIV><BUTTON onclick="topFunctionAlt()" id="myBtn" title="Go to top">Index</BUTTON><MAIN><SECTION><DIV class="container pt-lg-lg"><DIV class="row justify-content-center lead"><DIV class="col-md-12"><H1 class="text-center">Design</H1><UL class="nav nav-pills nav-fill flex-column flex-sm-row" id="myTab" role="tablist"><LI class="nav-item"><A class="nav-link mb-sm-3 active" id="Vaccine-tab" data-toggle="tab" href="#Detection" role="tab" aria-controls="home" aria-selected="true">Vaccine</A></LI><LI class="nav-item"><A class="nav-link mb-sm-3" id="FollowUp-tab" data-toggle="tab" href="#FollowUp" role="tab" aria-controls="contact" aria-selected="false">Follow Up</A></LI></UL><DIV class="tab-content" id="myTabContent"><DIV class="tab-pane fade show active" id="Detection" role="tabpanel" aria-labelledby="home-tab"><SECTION class="slice"><DIV class="container"><DIV class="row justify-content-center lead"><DIV class="col-lg-3"><DIV class="card"><DIV class="card-header"><SPAN class="h5">Index</SPAN></DIV><DIV class="list-group list-group-flush"><A href="#IntroVaccine" data-scroll-to="" data-scroll-to-offset="50" class="list-group-item list-group-item-action d-flex justify-content-between"><DIV><SPAN>Preface</SPAN></DIV></A><A href="#EncapsulinDelivery" data-scroll-to="" data-scroll-to-offset="50" class="list-group-item list-group-item-action d-flex justify-content-between"><DIV><SPAN>Encapsulin Antigen Delivery</SPAN></DIV></A><A href="#EncapsParagraph" data-scroll-to="" data-scroll-to-offset="50" class="list-group-item list-group-item-action d-flex justify-content-between"><DIV><SPAN>Encapsulin</SPAN></DIV></A><A href="#OurVaccine" data-scroll-to="" data-scroll-to-offset="50" class="list-group-item list-group-item-action d-flex justify-content-between"><DIV><SPAN>Our Vaccine Design</SPAN></DIV></A></DIV></DIV></DIV><DIV class="col-lg-9"><H1 id="IntroVaccine">Preface</H1><H3>How Neoantigen-based Cancer Immunotherapy Works</H3><P class="lead"><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 1.</B> Fundamental theory behind neoantigen based cancer Immunotherapy.</FIGCAPTION></FIGURE></CENTER></P><P class="lead">Cells become cancerous because of changes in their genetic makeup. These same changes can result in proteins that are differentially expressed on the cancerous cells but not human cells. These are called <I>neoantigens</I>, and refer to new cancer antigens that can signal the immune system to attack the cancer and eliminate it.</P><P class="lead">A patient is diagnosed with a cancer tumor. A biopsy of the tumor and a biopsy of healthy tissue are acquired to perform whole exome sequencing on both biopsies. A bioinformatic tool (such as CAPOEIRA’s <A href="https://2018.igem.org/Team:EPFL/Software"><SPAN style="color:blue">Ginga</SPAN></A>) processes the whole exome sequences of both the healthy and tumor biopsies used to identify neoantigens.
							A specific neoantigen that is differentially expressed on tumor cells and not healthy cells is supplied to the patient through a vaccine formulation. Dendritic cells of the patient uptake the neoantigen from the vaccine formulation. Alongside the neoantigen, the vaccine formulation supplies an adjuvant that activates the dendritic cell to uptake foreign material, and perceive them as danger signals.</P><P class="lead">The dendritic cell then processes the neoantigen and cross-presents it on MHC-1 complexes on its surface, where naïve CD8+ T cells can recognize it. Once the naïve CD8+ cells recognize the neoantigen, they mature into cytotoxic CD8+ T cells that specifically attack cells expressing this neoantigen; in this case, the tumor cells.</P></DIV><DIV class="col-lg-12"><H3>Rising Importance of Cancer Vaccination</H3><P class="lead">The immunogenicity of neoantigens leading to T-cell activation has long been demonstrated in patients (<A href="#Wolfel1995"><SPAN style="color:blue">Wolfel <I>et al.</I>, 1995</SPAN></A>). In fact, preclinical and clinical data has already shown that neoantigen specific cytotoxic T lymphocytes (CTLs) constitute the most potent T-cell populations for tumor rejection (<A href="#Wolfel1995"><SPAN style="color:blue">Wolfel <I>et al.</I>, 1995</SPAN></A>; <A href="#Matsushita2012"><SPAN style="color:blue">Matsushita <I>et al.</I>, 2012</SPAN></A>).
							Still, the natural production of neoantigen-specific CTLs by a patient’s immune system is scarce because of low clonal frequency and ineffective presentation of neoantigens (<A href="#Alexandrov2013"><SPAN style="color:blue">Alexandrov <I>et al.</I>, 2013</SPAN></A>; <A href="#Zhu2017"><SPAN style="color:blue">Zhu <I>et al.</I>, 2017</SPAN></A>). Therefore, cancer vaccines or adjuvant cancer therapies (ACT) are crucial to potentiate immunity against neoantigens for cancer treatment. Hence, a large number of strategies have been progressed for the creation, formulation and delivery of various cancer vaccines; for example, whole tumor cell lysate, nucleotide (mRNA/ DNA), protein or peptide-based vaccines, dendritic cell (DC) based vaccines, viral vectors and biomaterial-assisted vaccines.</P><P class="lead">However, it remains challenging to develop a universal and effective delivery strategy to target neoantigen-based vaccines to professional antigen-presenting cells (APCs) for eliciting robust and potent T-cell responses against cancer.</P><P class="lead">In general, parenterally injected soluble antigens or adjuvants rapidly spread into the systemic circulation making them ineffective due to their small molecular sizes, poor targeting, and rapid draining in lymph nodes (LNs). This ultimately results in a limited immune response (<A href="#Liu2014"><SPAN style="color:blue">Liu <I>et al.</I>, 2014</SPAN></A>; <A href="#Fifis2004"><SPAN style="color:blue">Fifis <I>et al.</I>, 2004</SPAN></A>).</P><P class="lead">In addition, even if such soluble tumor neoantigens are acquired by DCs, they would be trapped in endolysosomal compartments and digested into peptides, which are subsequently loaded almost entirely onto MHC class II molecules for presentation to CD4+ helper T-cells solely. However, for achieving an effective immune response, the therapeutic cancer vaccine is expected to elicit robust cytotoxic CD8+ T-cell responses, which is essential for tumor cell destruction (<A href="#Janssen2005"><SPAN style="color:blue">Janssen <I>et al.</I>, 2005</SPAN></A>).</P><P class="lead">Thus, it is also key for cancer vaccines to enable cytosolic delivery of neoantigens for a successful activation of cytotoxic T-cell mediated immunity. Effectively, having a platform for neoantigen delivery is favourable for vaccine delivery as it protects antigen and adjuvant molecules from degradation and clearing, enhances lymphoid organ targeting, and modulates APCs’ functions for better presentation (<A href="#Amigorena2010"><SPAN style="color:blue">Amigorena <I>et al.</I>, 2010</SPAN></A>).</P></DIV><DIV class="col-lg-12"><H1 id="EncapsulinDelivery">Encapsulin Antigen Delivery</H1><P class="lead">In 2016, an article was published by Sebyung Kang and colleagues describing the employment of the protein cage nanoparticles, Encapsulin (Encap), as neoantigenic peptide nanocarriers by genetically incorporating the OT-1 peptide of ovalbumin (OVA) protein (used as vaccine for B16-OVA melanoma tumor model) to three different positions of the Encap subunit (<A href="#Choi2016"><SPAN style="color:blue">Choi <I>et al.</I>, 2016</SPAN></A>). This article motivated us to look further into Encapsulin as a strong candidate for the vaccine platform.</P><P class="lead">In the mentioned study (<A href="#Choi2016"><SPAN style="color:blue">Choi <I>et al.</I>, 2016</SPAN></A>), DCs that were pulsed with constructs of OT1-Encap-C (C-terminal fusion with OT-1 peptide) induced OT-1-specific CD8+ T cell proliferation both in vivo and in vitro. This indicates Encapsulin ability to enhance the uptake of the OT-1 peptides by dendritic cells and the subsequent presentation of these peptides to DC8+ T cells. </P><P class="lead">OT1-Encap-C presentation to DCs was also able to induce the differentiation of functional effector CD8+ T cells in murine spleen. Finally, OT-1-Encap subcutaneous vaccinations in B16-OVA melanoma tumor bearing mice effectively activated OT-1 peptide specific cytotoxic CD8+ T cells before or even after tumor generation, resulting in significant suppression of tumor growth in prophylactic as well as therapeutic treatments. </P><P class="lead">Encapsulin was thus chosen as the platform for  CAPOEIRA’s vaccine system, for multiple reasons:
									<OL><LI>Encapsulin was shown to have an effective activation of dendritic and T cells in vitro and in vivo</LI><LI>Encapsulin allows for the easy conjugation of libraries of neoantigen, as this can be realized through genetic ligation of the neoantigen oligonucleotide sequences to the C-terminus of Encapsulin</LI><LI>Encapsulin, along with the neoantigens, can be expressed in a rapid and straightforward manner using the cell free expression system</LI><LI>Such expression systems might help in reducing the cost of generating libraries of peptides by other technologies such as solid-phase peptide synthesis</LI></OL></P><H1 id="EncapsParagraph">Encapsulin</H1><P class="lead">
							  Encapsulin (Figure 2) is a protein cage nanoparticle found in the thermophilic bacteria <I>Thermotoga maritima</I>.
Its crystal structure has been recently solved, and was published in a paper in 2008 (<A href="#Sutter2008"><SPAN style="color:blue">Sutter <I>et al.</I>, 2008</SPAN></A>). The Encapsulin multimer is assembled from 60 identical 31 kDa monomers having a thin and icosahedral T=1 symmetric cage structure, with interior and exterior diameters of 20 and 24 nm, respectively. The multimer automatically assembles from the monomers once expressed, as it leads to a lower energy state. The C-terminus is outward pointing, allowing for easy conjugation of peptides after the C-terminus (<A href="#Moon2014"><SPAN style="color:blue">Moon <I>et al.</I>, 2014</SPAN></A>).</P><P class="lead">The Encapsulin monomer was modified by inserting a Hexahistidine linker (GGGGGGHHHHHHGGGGG) between residues 43 and 44 of the WT Encapsulin (<A href="#Moon2014"><SPAN style="color:blue">Moon <I>et al.</I>, 2014</SPAN></A>). This was shown to convey exceptional heat stability and better hydrodynamic properties for the Encapsulin multimer. These properties are crucial to obtain a simpler and more efficient purification of the Encapsulin protein.</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 2.</B><I>Left:</I> Scientific Rendition of Encapsulin monomer and Bioassembly based on the pdb-3DKT (VMD). <I>Right:</I> Cartoon representation of Encapsulin in its monomeric and multimeric form along with neoantigens and hexahistidine loops.</FIGCAPTION></FIGURE></CENTER><H1 id="OurVaccine">Vaccine Design Project</H1><P class="lead">
								The vaccine design process aimed at establishing a platform that receives a library of neoantigens from Ginga, and outputs a library of vaccines that incorporate these neoantigens on the surface of Encapsulin (Figure 3).</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 3.</B> Overview of vaccine design</FIGCAPTION></FIGURE></CENTER><DIV id="VaccineDesignCard"><DIV class="card"><A data-toggle="collapse" href="#GenIncorportation"><DIV class="card-header"><H3 class="card-link">
										Genetic Incorporation of Neoantigen
								  </H3></DIV></A><DIV id="GenIncorportation" class="collapse"><DIV class="card-body"><P class="lead">A major requirement of a neoantigen vaccine is allowing for the facile and secure introduction of neoantigen libraries onto the scaffold/carrier. Using Encapsulin, one accessible method for such a conjugation would be the genetic ligation of the neoantigen oligonucleotide sequence to the C-terminus of Encapsulin, as depicted in Figure 4.</P><P class="lead">After acquiring the raw Encapsulin sequence from the LBNC lab at EPFL (<A href="#CassidyAmstutz2016"><SPAN style="color:blue">Cassidy-Amstutz <I>et al.</I>, 2016</SPAN></A>; Addgene Catalogue # 86405), we genetically introduced a HexaHistidine linker between Amino Acids 43 &amp; 44 to create HexaHistidine Encapsulin, which was reported to have higher heat resistance and better hydrodynamic properties (<A href="#Moon2014"><SPAN style="color:blue">Moon <I>et al.</I>, 2014</SPAN></A>). This modification was done using a Golden Gate assembly with BsaI as a type IIS restriction enzyme. The insert was assembled from two synthesized oligos (60 bp each which partially anneal) with BsaI cut sites. The insert was converted to dsDNA using PCR. The Original Encapsulin plasmid was amplified using primers incorporating BsaI cut sites and the insert was incorporated using Golden Gate.</P><P class="lead">To obtain a rapid, efficient, and reliable incorporation of neoantigens onto the HexaHistidine Encapsulin platform, we designed the plasmid HexaHistidine Encapsulin-CBsaI (Figure 5) (<A href="http://parts.igem.org/Part:BBa_K2686005"><SPAN style="color:blue">Registry Part BBa_K2686005</SPAN></A>). Starting from the HexaHistidine Encapsulin plasmid, we introduce at the C-terminus an sfGFP CDS under its native promoter flanked by two BsaI cut sites.</P><P class="lead">The BsaI cut sites would allow for the rapid, scarless introduction of oligonucleotides encoding for the neoantigens using Golden Gate Assembly (Figures 5 &amp; 6). These neoantigens would be fused to the C-terminus of Encapsulin, and displayed on its outer surface. Such a system allows for a reliable, but fast expression of libraries of encapsulin-neoantigens.</P><P class="lead">The insert in between the two BsaI cut sites, consisting of sfGFP with a native promoter and terminator, allows for checking the success of the insertion of the neoantigen after transformation of cells with the Golden Gate product (green colonies do not contain the desired peptide insert, but the original plasmid instead). This cloning strategy was useful in the initial characterization of the system and production of the encapsulin fused with OT-1 peptide. For high-throughput production of encapsulin-neoantigen constructs, different strategies avoiding <I>in vivo</I> could be envisioned. </P></DIV><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 4.</B> Genetic Incorporation of Neoantigen Libraries onto our vaccine platform, Encapsulin.</FIGCAPTION></FIGURE></CENTER><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 5.</B> CAPOEIRA’s designed plasmid HHEncap_BSaI (Part BBa_K2686005) for neoantigen incorporation following Encapsulin C-terminus.</FIGCAPTION></FIGURE></CENTER><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 6.</B> Golden Gate Assembly of the Vaccine System.</FIGCAPTION></FIGURE></CENTER></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#CFEEncap"><DIV class="card-header"><H3 class="card-link">
										 Cell free expression of Encapsulin
								  </H3></DIV></A><DIV id="CFEEncap" class="collapse"><DIV class="card-body"><P class="lead">We exploited the fact that Encapsulin is made of protein exclusively, and thus, can be fully expressed as a recombinant protein in a bacterial expression system. However, accelerating the pace of the vaccine production requires a new approach for the rapid expression of proteins encoded on plasmid/linear DNA constructs. Current standard bacterial expression systems require days due to cloning and in-vivo transformations. </P><P class="lead">This is why CAPOEIRA uses a cell free expression approach, which preserves the protein production capability and regulatory mechanisms of <I>E. coli</I>. Cell-free systems (Figure 7) use all of the inner workings of a cell without having the constricting boundary of the cell wall and thus the precondition of keeping cells alive (<A href="#Rollin2013"><SPAN style="color:blue">Rollin <I>et al.</I>, 2013</SPAN></A>). This allows speeding the design-expression process. When preparing the cell-free systems, all genomic DNA and membranes are eliminated, resulting in a solution containing all of the cells proteins without the limiting factors of a living cell.</P><P class="lead">The cell free expression has 2 advantages in for CAPOEIRA:</P><OL><LI>Faster expression of proteins from DNA constructs (8 to 10 hours of expression), allowing for fast and easy expression of libraries of proteins</LI><LI>Faster &amp; Easier purification of protein products from cell free expression reactions compared to purification from cells</LI></OL><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 7.</B> Cell Free Expression of the Vaccine</FIGCAPTION></FIGURE></CENTER></DIV></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#HPEncap"><DIV class="card-header"><H3 class="card-link">
										 Heat Purification of Encapsulin
								  </H3></DIV></A><DIV id="HPEncap" class="collapse"><DIV class="card-body"><P class="lead">The combination of a protein with high heat resistance further improved after Histag modification, along with a cell free expression system allows for an efficient one-step heat purification of our vaccine product. In short, after the expression of the vaccine construct using the cell free expression system (which takes around 10 hours), heat purification of the sample goes as follows (Figure 8):</P><OL><LI>Heating at 70 ºC for 20 min</LI><LI>Putting on ice for 15 min</LI><LI>Centrifugation at 12,000 xg for 10 min</LI><LI>Separation of the supernatant (containing the purified vaccine construct) from the pellet</LI></OL><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 8.</B> CAPOEIRA’s heat purification approach for the expressed vaccine</FIGCAPTION></FIGURE></CENTER><P class="lead">This simple heat purification step allows for an exceptional purity of CAPOEIRA’s vaccine system in less than an hour. After the heat purification step, the obtained purity might be very close to a final formulation for vaccine delivery.</P></DIV></DIV></DIV></DIV><ARTICLE><H2><I><U>References</U></I></H2><UL><LI id="Alexandrov2013">Alexandrov, Ludmil B., et al. &quot;Signatures of mutational processes in human cancer.&quot; <I>Nature</I>, 500.7463 (2013): 415.</LI><LI id="Amigorena2010">Amigorena, Sebastian, and Ariel Savina. &quot;Intracellular mechanisms of antigen cross presentation in dendritic cells.&quot; <I>Current opinion in immunology</I>, 22.1 (2010): 109-117.</LI><LI id="CassidyAmstutz2016">Cassidy-Amstutz, Caleb, et al. &quot;Identification of a minimal peptide tag for in vivo and in vitro loading of encapsulin.&quot; <I>Biochemistry</I>, 55.24 (2016): 3461-3468.</LI><LI id="Choi2016">Choi, Bongseo, et al. &quot;Effective delivery of antigen–encapsulin nanoparticle fusions to dendritic cells leads to antigen-specific cytotoxic T cell activation and tumor rejection.&quot; <I>ACS nano</I>, 10.8 (2016): 7339-7350.</LI><LI id="Fifis2004">Fifis, Theodora, et al. &quot;Size-dependent immunogenicity: therapeutic and protective properties of nano-vaccines against tumors.&quot; <I>The Journal of Immunology</I>, 173.5 (2004): 3148-3154.</LI><LI id="Janssen2005">Janssen, Edith M., et al. &quot;CD4+ T-cell help controls CD8+ T-cell memory via TRAIL-mediated activation-induced cell death.&quot; <I>Nature</I>, 434.7029 (2005): 88.</LI><LI id="Liu2014">Liu, Haipeng, et al. &quot;Structure-based programming of lymph-node targeting in molecular vaccines.&quot; <I>Nature</I>, 507.7493 (2014): 519.</LI><LI id="Matsushita2012">Matsushita, Hirokazu, et al. &quot;Cancer exome analysis reveals a T-cell-dependent mechanism of cancer immunoediting.&quot; <I>Nature</I>, 482.7385 (2012): 400.</LI><LI id="Moon2014">Moon, Hyojin, et al. &quot;Developing genetically engineered encapsulin protein cage nanoparticles as a targeted delivery nanoplatform.&quot; <I>Biomacromolecules</I>, 15.10 (2014): 3794-3801.</LI><LI id="Rollin2013">Rollin, Joseph A., Tsz Kin Tam, and Y-H. Percival Zhang. &quot;New biotechnology paradigm: cell-free biosystems for biomanufacturing.&quot; <I>Green chemistry</I>, 15.7 (2013): 1708-1719.</LI><LI id="Sutter2008">Sutter, Markus, et al. &quot;Structural basis of enzyme encapsulation into a bacterial nanocompartment.&quot; <I>Nature structural &amp; molecular biology</I>, 15.9 (2008): 939.</LI><LI id="Wolfel1995">Wolfel, Thomas, et al. &quot;A p16INK4a-insensitive CDK4 mutant targeted by cytolytic T lymphocytes in a human melanoma.&quot; <I>Science</I>, 269.5228 (1995): 1281-1284.</LI><LI id="Zhu2017">Zhu, Guizhi, et al. &quot;Efficient nanovaccine delivery in cancer immunotherapy.&quot; <I>ACS nano</I>, 11.3 (2017): 2387-2392.</LI></UL></ARTICLE></DIV></DIV></DIV></SECTION></DIV><DIV class="tab-pane fade" id="FollowUp" role="tabpanel" aria-labelledby="contact-tab"><SECTION class="slice"><DIV class="container"><DIV class="row justify-content-center lead"><DIV class="col-lg-9"><H1 id="IntroFollowup">Introduction</H1><P class="lead">Through our interviews with health specialists and oncology experts (more information in <A href="https://2018.igem.org/Team:EPFL/Human_Practices"><SPAN style="color:blue">Integrated human practices</SPAN></A>) we assessed the necessity to have a non-invasive treatment companion to determine our vaccine efficacy. Here, we want to provide a proof-of-concept that would allow us to monitor the patient’s response by using the same set of identified neoantigens used for our vaccine.
We also believe that it is important to be able to detect relapses in early melanoma stages, as the survival rates for patients dramatically drop to 20% in stage IV compared to 99% survival rate in stage I and II (<A href="#Siegel2018"><SPAN style="color:blue">Siegel <I>et al.</I>, 2018</SPAN></A>). </P><P class="lead">To answer these needs, we envision a new generation of diagnostic tools by which a liquid peripheral blood draw could give an accurate prognosis regarding the elimination of the tumor cells and, by targeting specific biomarkers, be a good predictor of relapse. This requires a detection system that is both highly sensitive and specific since single base pair polymorphisms, barely detectable in the blood, can lead to tumorigenesis.</P><P class="lead">Our idea is to develop a Cas12a detection system coupled to an amplification step. This detection system is rapid, sensitive and specific enough to reliably detect these biomarkers.</P></DIV><DIV class="col-lg-3"><DIV class="card"><DIV class="card-header"><SPAN class="h5">Index</SPAN></DIV><DIV class="list-group list-group-flush"><A href="#IntroFollowup" data-scroll-to="" data-scroll-to-offset="50" class="list-group-item list-group-item-action d-flex justify-content-between"><DIV><SPAN>Introduction</SPAN></DIV></A><A href="#Biomarkers" data-scroll-to="" data-scroll-to-offset="50" class="list-group-item list-group-item-action d-flex justify-content-between"><DIV><SPAN>Biomarkers and Liquid Biopsies</SPAN></DIV></A><A href="#Cas12a" data-scroll-to="" data-scroll-to-offset="50" class="list-group-item list-group-item-action d-flex justify-content-between"><DIV><SPAN>Cas12a</SPAN></DIV></A><A href="#SamplePreparation" data-scroll-to="" data-scroll-to-offset="50" class="list-group-item list-group-item-action d-flex justify-content-between"><DIV><SPAN>Sample preparation</SPAN></DIV></A><A href="#Amplification" data-scroll-to="" data-scroll-to-offset="50" class="list-group-item list-group-item-action d-flex justify-content-between"><DIV><SPAN>Amplification</SPAN></DIV></A><A href="#Cas12aAssay" data-scroll-to="" data-scroll-to-offset="50" class="list-group-item list-group-item-action d-flex justify-content-between"><DIV><SPAN>Our detection scheme</SPAN></DIV></A></DIV></DIV></DIV><DIV class="col-lg-12"><H1 id="Biomarkers">Biomarkers and Liquid Biopsies</H1><P class="lead">Recently, several studies have shown that non-invasive <B>liquid biopsy</B> methods are a promising way to detect cancer relapse and monitor tumor regression (<A href="#Heitzer2017"><SPAN style="color:blue">Heitzer <I>et al.</I>, 2017</SPAN></A>). Liquid biopsies represent a fast, reliable and easy way to obtain samples compared to the invasive nature of solid biopsies which are generally time-consuming, difficult to perform frequently and not without some risks to the patient.</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 1.</B> ctDNA and miRNA in blood</FIGCAPTION></FIGURE></CENTER><UL class="nav nav-tabs nav-fill flex-column flex-sm-row" id="myTabAmplification" role="tablist"><LI class="nav-item"><A class="nav-link mb-sm-3 active" id="ctDNAAmpl-tab" data-toggle="tab" href="#ctDNA1" role="tab" aria-controls="home" aria-selected="true">ctDNA</A></LI><LI class="nav-item"><A class="nav-link mb-sm-3" id="miRNAAmpl-tab" data-toggle="tab" href="#miRNA1" role="tab" aria-controls="contact" aria-selected="false">miRNA</A></LI></UL><DIV class="tab-content" id="ctDNAampl"><DIV class="tab-pane fade show active" id="ctDNA1" role="tabpanel" aria-labelledby="home-tab"><H3>ctDNA - A look at the tumor DNA</H3><P class="lead">Circulating free DNA (cfDNA) is a common term that refers to all the DNA fragments that are present in the blood. This fragmented DNA is thought to originate from apoptotic cells (<A href="#Harris2016"><SPAN style="color:blue">Harris <I>et al.</I>, 2016</SPAN></A>). In cancer patients the proportion of cfDNAs from necrotic tumor cells - known as “<B>circulating tumor DNA</B>” (<B>ctDNA</B>) -  represents a large part of the circulating DNA.
											These short DNA fragments of size ranging from 100bp to 200bp - with a peak at 145bp (<A href="#Underhill2016"><SPAN style="color:blue">Underhill <I>et al.</I>, 2016</SPAN></A>) - contain virtually all the possible genetic defects that can be found in the original tumor cell population, including somatic point mutations and translocations (<A href="#Harris2016"><SPAN style="color:blue">Harris <I>et al.</I>, 2016</SPAN></A>; <A href="#Calapre2017"><SPAN style="color:blue">Calapre <I>et al.</I>, 2017</SPAN></A>). Moreover, literature has shown that levels of ctDNA in the blood are correlated with progression or remission of disease in several cancers, including melanoma (<A href="#Gray2015"><SPAN style="color:blue">Gray <I>et al.</I>, 2015</SPAN></A>; <A href="#Girotti2016"><SPAN style="color:blue">Girotti <I>et al.</I>, 2016</SPAN></A>; <A href="#Tsao2015"><SPAN style="color:blue">Tsao <I>et al.</I>, 2015</SPAN></A>; <A href="#Calapre2017"><SPAN style="color:blue">Calapre <I>et al.</I>, 2017</SPAN></A>).</P><P class="lead">Our goal using ctDNA as biomarkers is to come up with a personalized follow-up, and the personalized touch comes back again from our implemented <A href="https://2018.igem.org/Team:EPFL/Software"><SPAN style="color:blue">bioinformatic software</SPAN></A>: Ginga. Indeed, Ginga takes as an input the genetic sequence of the tumor, to generate not only a list of neoantigens that will form the basis of our <A href="#Detection"><SPAN style="color:blue">vaccine</SPAN></A>, but also a library of another molecular alteration specific to the tumor, namely chromosomal rearrangements, that we will target for relapse detection.</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 2.</B> Overview of the release of ctDNA in the blood by necrotic cancer cells. These are, along with some miRNAs, promising biomarkers present in the blood.</FIGCAPTION></FIGURE></CENTER><DIV id="ctDNABiom"><DIV class="card"><A data-toggle="collapse" href="#Biom1"><DIV class="card-header"><H3 class="card-link">
									 Vaccine monitoring through point mutation
								  </H3></DIV></A><DIV id="Biom1" class="collapse" data-parent="#ctDNABiom"><DIV class="card-body"><P class="lead">Our goal here is to detect the point-mutated sequences that code for the neoantigens we have selected for our vaccine. More precisely, we seek to quantify the presence of these sequences in the bloodstream through ctDNA. This gives us the opportunity to monitor tumor remission directly by studying the patient’s blood.</P></DIV></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Biom2"><DIV class="card-header"><H3 class="card-link">
									Cancer relapse detection through chromosomal rearrangement
								  </H3></DIV></A><DIV id="Biom2" class="collapse" data-parent="#ctDNABiom"><DIV class="card-body"><P class="lead">As part of any cancer therapy, there is always a need to be vigilant against any recurrence, since it can occur at any time: indeed, although the targeted cell populations have been eliminated, other cells may have survived and resurface after some time. To address this problem, we want to detect the sequences of the individualized junctions identified using our bioinformatic pipeline directly in the blood, using our CRISPR-Cas12a based assay. The detection of such sequences will alert the patient of a potential relapse and the need for a closer follow-up, which can have a lead time of up to 11 months in detecting relapses over clinical established methods in some types of cancers, according to <A href="#Olsson2015"><SPAN style="color:blue">Olsson <I>et al.</I>, 2015</SPAN></A>.</P></DIV></DIV></DIV></DIV></DIV><DIV class="tab-pane fade" id="miRNA1" role="tabpanel" aria-labelledby="contact-tab"><H3>Cancer relapse detection through miRNA</H3><P class="lead"><B>MicroRNAs</B> (<B>miRNAs</B>) are short (18-24 nt) non-coding RNA molecules which act as post-transcriptional regulators of gene expression. Over the years, miRNAs have been proved to play a critical role in a variety of different diseases, including cancer (<A href="#Larrea"><SPAN style="color:blue">Larrea <I>et al.</I>, 2016</SPAN></A>). Moreover, miRNAs are remarkably stable in human plasma (<A href="#Mitchell"><SPAN style="color:blue">Mitchell <I>et al.</I>, 2008</SPAN></A>), and several miRNAs circulating in the blood have recently been shown to be dysregulated (either over- or under-expressed) in patients with certain cancers, including melanoma, with respect to healthy subjects (<A href="#Mirzaei"><SPAN style="color:blue">Mirzaei <I>et al.</I>, 2016</SPAN></A>). For these reasons, miRNAs have been proposed as potential prognostic and diagnostic biomarkers for melanoma, which makes them suitable candidates for the follow-up part of our project as well.</P><P class="lead">Previous iGEM teams (e.g. NUDT China 2016 team) have shown promising results with Rolling Circle Amplification of miRNAs by means of dumbbell-shaped probes (details in “Amplification”). Our aim is to investigate whether is possible to combine this dumbbell probe design with a Cas12a system to achieve a sensitive and specific detection assay.</P></DIV><H1 id="Cas12a">Cas12a</H1><DIV id="Cas12"><P class="lead">To answer the need for a fast and robust detection method we chose to work with the newly characterized <B>Cas12a</B> (<B>Cpf1</B>) protein. </P><P class="lead">CRISPR-Cas (clustered regularly interspaced short palindromic repeats–CRISPR-associated) systems are originally inspired by an antiviral defense mechanism used by prokaryotes which work by recognizing and cleaving the foreign DNA/RNA. They have, in the recent years, widely been used as a gene editing tool for their ability to find and cut at a specific site allowing the insertion of a desired sequence. This target sequence is what we call the <I>activator</I>.</P><P class="lead">In the case of Cas12a this activator is composed of two different strands: the target strand (TS) and the non-target strand (NTS). The NTS requires a T-rich protospacer adjacent motif (PAM) sequence whereas the TS contains the sequence we want to detect. CRISPR scans all PAM sequences in the genome and compares its loaded <I>guide RNA</I> (<I>gRNA</I>) with all possible adjacent target sequences. When Cas12a finds its target, it undergoes a conformational change and cleaves the activator: its double stranded DNA (dsDNA) target.
							It is also worth mentioning that Cas12a proteins retains the capacity to recognize and cleave ssDNA without any PAM sequence.</P><P class="lead">As a result of its conformational change upon target recognition, Cas12a unleashes a non-specific endonuclease activity (i.e. <I>collateral cleavage</I>) virtually against any single stranded DNA (ssDNA). Each activated Cas12a protein can cleave huge numbers of ssDNA molecules, and this is what makes this system so suitable for detection, as it greatly amplifies the signal. As explained more in detail in “Fluorescent readout”, by coupling this  property to a single-stranded FQ reporter, we can hugely increase even very small signals, which means higher sensitivity for this system.</P><P class="lead">In our assays we worked with the purified Lba-Cas12a (type V-A CRISPR) extracted from <I>Lachnospiraceae bacterium ND2006</I> and provided by <A href="#NebCas12a"><SPAN style="color:blue">New England BioLabs</SPAN></A>.
							</P><DIV class="card"><A data-toggle="collapse" href="#crRNAdes"><DIV class="card-header"><H3 class="card-link">
										Design of the gRNA
								  </H3></DIV></A><DIV id="crRNAdes" class="collapse"><DIV class="card-body"><P class="lead">The gRNA must contain a 17 to 24bp complementary sequence to the dsDNA of interest. For activating Cas12a and further collateral cleavage, it is crucial that the activator incorporates a T-rich PAM sequence, TTTN, 5’ of the target sequence. Once the protein has recognized the PAM sequence and the gRNA has bound the complementary sequence, the staggered cut will occur around 18 bases 3′ of the PAM and leaves 5′ overhanging ends (<A href="#Zetsche2017"><SPAN style="color:blue">Zetsche <I>et al.</I>, 2017</SPAN></A>).</P><P class="lead">Our gRNAs were transcribed using T7 polymerase starting from a ssDNA with the coding sequence downstream of a T7 promoter.
An appropriate design of the gRNA-coding ssDNA consists of three separate parts in the following order:</P><UL><LI><B>T7 promoter</B> (5’-<I>ctTAATACGACTCACTATAgg</I>-3’): This is needed for the transcription and the sequence will not appear in the final gRNA. To increase the polymerase efficiency, it is recommended to add 1, 2 or 3 G’s right after the promoter (<A href="#sgRNASynth"><SPAN style="color:blue">New England BioLabs</SPAN></A>) as well as adding CT upstream of it (<A href="#Baklanov1996"><SPAN style="color:blue">Baklanov <I>et al.</I>, 1996</SPAN></A>)</LI><LI><B>Scaffold</B> (5’-<I>TAATTTCTACTAAGTGTAGAT</I>-3’): This sequence can change according to the Cas12a species - the one shown here is specific for LBa Cas12a (<A href="#Zetsche2017"><SPAN style="color:blue">Zetsche <I>et al.</I>, 2017</SPAN></A>)</LI><LI><B>Spacer</B>: It is the gRNA sequence that is complementary to the activator sequence (TS). For the ctDNA group we chose to use shorter guide sequences (17 bp rather than 20) for detecting both single base polymorphism and chromosomal rearrangements, based on the work done by <A href="#Li2018"><SPAN style="color:blue">Li <I>et al.</I>, 2018</SPAN></A>, where they proved that shorter guide sequences yielded higher cleavage specificity</LI></UL><P class="lead">The T7 polymerase needs a double stranded region to bind to. It is thus necessary to order a primer for this region. The rest of the sequence can stay single stranded for a lower cost.</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 3.</B> Recognition of the target sequence (activator), via complementary binding of the gRNA</FIGCAPTION></FIGURE></CENTER></DIV></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Readout"><DIV class="card-header"><H3 class="card-link">
										Fluorescent readout
								  </H3></DIV></A><DIV id="Readout" class="collapse"><DIV class="card-body"><P class="lead">Following <A href="#Chen2018"><SPAN style="color:blue">Chen <I>et al.</I>, 2018</SPAN></A>, we designed a Cas12a detection assay based on the cleavage of DNaseAlert (IDT), which are fluorescence-quenched oligonucleotide probes that emit a fluorescent signal after DNAse degradation: when DNases are present, the linkage between the fluorophore and its quencher is cleaved, which leads to the emission of a bright signal upon excitation at 535-556 nm (<A href="#DNaseAlertIDT"><SPAN style="color:blue">Integrated DNA Technologies</SPAN></A>).</P><P class="lead">By exploiting indiscriminate cleavage of the Cas12a protein that is triggered upon target recognition, we were able to obtain a fluorescent reading following the cleavage of our reporter molecules. This allows for a rapid and sensitive detection of the dsDNA activator.</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 4.</B> Cas12a assay principles: Activation of Cas12a unleashing the proteins endonuclease activity against ssDNA (here a Fluorophore-Quencher reporter).</FIGCAPTION></FIGURE></CENTER></DIV></DIV></DIV></DIV><H1 id="SamplePreparation">Sample preparation</H1><P class="lead">A simple blood draw is necessary for both our treatment companion and relapse detection.</P><P class="lead">
								The analysis of our biomarkers is done directly in the plasma, without the need to isolate them, sparing us precious time, costs and unnecessary contamination that can occur during nucleic acid extraction (<A href="#Abe2003"><SPAN style="color:blue">Abe, 2003</SPAN></A>). The first step for our sample preparation is the isolation of plasma from whole blood. As part of our experiments on ctDNA, we used commercially ordered human plasma for both practical and ethical reasons. The next step is to treat it with PBS then heat it at 95°C for 3 minutes to precipitate proteins.
							</P><P class="lead">Sample preparation for miRNA can theoretically be achieved in a similar way: <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A> showed that is possible to perform amplification of miRNA directly in serum samples pre-diluted in DEPC-treated water and boiled at 95 °C for 10 minutes. We expect that a similar protocol might be applied also to plasma for miRNA, as measurements of miRNA between plasma and serum have been found to be highly correlated (<A href="#Mitchell"><SPAN style="color:blue">Mitchell <I>et al.</I>, 2008</SPAN></A>).</P><P class="lead">Amplification of each biomarker is done afterwards, in order to have enough copies to be able to perform the Cas12a assay effectively.</P><H1 id="Amplification">Amplification</H1><P class="lead"><UL class="nav nav-tabs nav-fill flex-column flex-sm-row" id="myTabAmplification" role="tablist"><LI class="nav-item"><A class="nav-link mb-sm-3 active" id="ctDNAAmpl-tab" data-toggle="tab" href="#ctDNA" role="tab" aria-controls="home" aria-selected="true">ctDNA</A></LI><LI class="nav-item"><A class="nav-link mb-sm-3" id="miRNAAmpl-tab" data-toggle="tab" href="#miRNA" role="tab" aria-controls="contact" aria-selected="false">miRNA</A></LI></UL></P><DIV class="tab-content" id="ctDNAampl"><DIV class="tab-pane fade show active" id="ctDNA" role="tabpanel" aria-labelledby="home-tab"><P class="lead">Due to the very low concentration of ctDNA in blood it is necessary to amplify the target prior to Cas12a detection assay. We chose PCR as it is a common practice in most laboratories.
										It is important to note that it is possible to replace this method with an isothermal amplification, like LAMP or RPA, to get this assay closer to point of care.</P><P class="lead">One of the limitation of a Cas12a is the need for a PAM sequence near the target we want to detect. Following <A href="#Li2018"><SPAN style="color:blue">Li <I>et al.</I>, 2018</SPAN></A> and to overcome this limitation, we designed primers that would add the PAM sequence by introducing synthetic mutations. This enables us to virtually target any desired sequence regardless of existence of a T-rich PAM sequence near the target.</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 5.</B> Amplification of the target fragment and introduction of the PAM sequence synthetically.</FIGCAPTION></FIGURE></CENTER></DIV><DIV class="tab-pane fade" id="miRNA" role="tabpanel" aria-labelledby="contact-tab"><P class="lead">Although miRNAs are potentially very valid candidates as biomarkers, they are associated with some hurdles (particularly low abundance) which are not completely overcome by currently existing detection methods (<A href="#Miao"><SPAN style="color:blue">Miao <I>et al.</I>, 2015</SPAN></A>).   </P><P class="lead">Among different recent amplification techniques, <B>Rolling Circle Amplification</B> has been proved to be one of the most suitable, thanks to its robustness, simplicity, specificity and high sensitivity (<A href="#Cheng"><SPAN style="color:blue">Cheng <I>et al.</I>, 2009</SPAN></A>). Rolling-Circle Amplification (RCA) is an isothermal amplification (contrarily for instance to Polymerase Chain Reaction) where miRNA (or another short RNA or DNA sequence) is amplified by means of a circular DNA template (i.e. a <I>probe</I>) and a special DNA (or RNA) polymerase: the miRNA acts as a primer, with the RCA product (i.e. the <I>amplicon</I>) consisting in a concatemer containing tens to hundreds of tandem repeats that are complementary to the probe (<A href="#Ali"><SPAN style="color:blue">Ali <I>et al.</I>, 2014</SPAN></A>).</P><P class="lead">Toehold-initiated Rolling Circle Amplification (tiRCA), in particular, employs phi-29 DNA polymerase and is based on structure-switchable <B>dumbbell-shaped probes</B> (<A href="#Deng"><SPAN style="color:blue">Deng <I>et al.</I>, 2014</SPAN></A>): upon hybridization with the specific target miRNA, one of the two strands of the double-stranded region of the probe is displaced, resulting in an &quot;activated&quot; circular form of the probe with triggers the start of the RCA reaction. The complete mechanism of RCA is shown in Figure 6:</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 6.</B> Schematic representation of the tiRCA reaction. miRNA is represented in <SPAN style="color:Magenta">magenta</SPAN>, the dumbbell-shaped probe is shown in <SPAN style="color:DarkCyan">light blue</SPAN> and the amplicon in <SPAN style="color:green">green</SPAN>.</FIGCAPTION></FIGURE></CENTER><P class="lead">Although it is the probe - and not directly the miRNA - to be amplified, RCA allows to significantly increase the concentration of the miRNA sequence in solution: indeed, since a large portion of the probe is complementary to the miRNA, the amplicon of the probe will incorporate several copies of the original miRNA. This can theoretically be exploited to increase the sensitivity of an assay for quantification of miRNA. As later explained, while our Amplification step was mostly inspired by <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A>, we explored a new, ambitious Detection step after RCA based on Cas12a (and not on Cas9 and split reporter proteins). This implied designing new probes with specific characteristics for Cas12a, as explained in the following sections.</P><DIV id="miRNADesignAmpl"><DIV class="card"><A data-toggle="collapse" href="#Probes"><DIV class="card-header"><H3 class="card-link">
									 Our probes
								  </H3></DIV></A><DIV id="Probes" class="collapse"><DIV class="card-body"><P class="lead">The first miRNA we decided to target is let-7a-5p: this miRNA is not among the ones found to be relevant as melanoma biomarkers (as instead are other miRNAs of the let-7 family) (<A href="#Larrea"><SPAN style="color:blue">Larrea <I>et al.</I>, 2016</SPAN></A>; <A href="#Mirzaei"><SPAN style="color:blue">Mirzaei <I>et al.</I>, 2016</SPAN></A>); nonetheless, we thought it might
                            be the best option to start from it as a proof of concept, because it was already well characterized for Rolling Circle Amplification (RCA) by <A href="#Deng"><SPAN style="color:blue">Deng <I>et al.</I>, 2014</SPAN></A> and <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A></P><P class="lead"><A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A>, as well as our colleagues from the related 2016 iGEM team of NUDT China, had designed their probes in order for the amplicons to be recognized by a CRISPR-Cas 9 system. Since our project deals instead with CRISPR-Cas
                            12a, despite the miRNA sequence being the same, we therefore had to modify the sequences of our probes accordingly. More specifically, we had to adapt the PAM sequence (placed on the amplicon of the probe) in order to match
                            our Cas protein (we worked with LbCpf1): while the requirement for Cas9 was NGG on the 3' of the amplicon, in our case we needed to have TTTN on the 5'. More details on the design are described in the section &quot;Detailed design&quot;.</P><P class="lead">We wanted to test different designs of probes: some were conceived to have the PAM at the beginning of the larger loop of the amplicon (as in the probes from NUDT China), but we also investigated the case where the PAM was placed
                            on the double-stranded part (the stem) instead; the sequence on the uncostrained large loop was also changed among the probes. </P><P class="lead">We ordered 10 different probes; the sequence and related notes are described in the Table below. </P><CENTER><TABLE style="undefined;table-layout: fixed; width: 911px"><TBODY><TR><TH>Name</TH><TH>Sequence (5'-&gt;3')</TH><TH>Description</TH></TR><TR><TD class="super_script">Probe 1</TD><TD class="super_script">pACAACCTACTACCTCAAACGTAGGTTGTATAGTTTAAAGGGAGTCGGCGGAACTAT</TD><TD class="super_script">Probe designed by our team for Cas 12a. PAM on the large loop of the amplicon. </TD></TR><TR><TD class="super_script">Probe 2</TD><TD class="super_script">pACCTCATTGTATAGCCCCCCCCTGAGGTAGTAGGTTGCCCAACTATACAACCTACT</TD><TD class="super_script">Probe from <A href="#Deng"><SPAN style="color:blue">Deng <I>et al.</I>, 2014</SPAN></A> and <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A> (respectively referred to as &quot;SP-let-7a&quot; and &quot;let-7a probe 1&quot;), designed for Cas9. Used as a control for the efficiency of the amplification.</TD></TR><TR><TD class="super_script">Probe 3</TD><TD class="super_script">pACCTCACCCCCCCCCCCCCCCCTGAGGTAGTAGGTTGCCCAACTATACAACCTACT</TD><TD class="super_script">Probe from <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A> (&quot;let-7a probe 2&quot;), designed for Cas9. Used as a control for the efficiency of the amplification.</TD></TR><TR><TD class="super_script">Probe 4</TD><TD class="super_script">pACCTCAAAAAAAAAAAAAACCCTGAGGTAGTAGGTTGCCCAACTATACAACCTACT</TD><TD class="super_script">Probe from <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A> (&quot;let-7a probe 3&quot;), designed for Cas9. Used as a control for the efficiency of the amplification.</TD></TR><TR><TD class="super_script">Probe 5</TD><TD class="super_script">pACCTCATTTTTTTTTTTTTCCCTGAGGTAGTAGGTTGCCCAACTATACAACCTACT</TD><TD class="super_script">Probe from <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A> (&quot;let-7a probe 4&quot;), designed for Cas9. Used as a control for the efficiency of the amplification.</TD></TR><TR><TD class="super_script">Probe 6</TD><TD class="super_script">pACAACCTACTACCTCAAACGTAGGTTGTATAGTTTAAAGGGGGGGGGGCGAACTAT</TD><TD class="super_script">Probe designed by our team for Cas 12a. PAM on the large loop of the amplicon. Large loop with repetitive sequence of Gs.</TD></TR><TR><TD class="super_script">Probe 7</TD><TD class="super_script">pACAACCTACTACCTCAAACGTAGGTTGTATAGTTTAAAGGGAGTGGTTTAAACTAT</TD><TD class="super_script">Probe designed by our team for Cas 12a. PAM on the stem. Large loop made of 8 bases.</TD></TR><TR><TD class="super_script">Probe 8</TD><TD class="super_script">pACAACCTACTACCTCAAACGTAGGTTGTATAGTTTAAAGGGAGTCGGCGGTTTAAACTAT</TD><TD class="super_script">Probe designed by our team for Cas 12a. PAM on the stem. Large loop made of 12 bases.</TD></TR><TR><TD class="super_script">Probe 9</TD><TD class="super_script">pACAACCTACTACCTCAAACGTAGGTTGTATAGTTTAAAGGGGGGGGGGGGGGCGTTTAAACTAT</TD><TD class="super_script">Probe designed by our team for Cas 12a. PAM on the stem. Large loop made of 16 bases.</TD></TR><TR><TD class="super_script">Probe 10</TD><TD class="super_script">pACAACCTACTACCTCAAACGTAGGTTGTAGAGTTTAAAGGGAGTCGGCGGAACTCT</TD><TD class="super_script">Probe designed by our team for Cas 12a. PAM on the large loop of the amplicon. Single base mismatch on the stem with respect to the target miRNA sequence.</TD></TR></TBODY></TABLE></CENTER><P class="lead">Note: The sequences of the probes include a phosphate group at the 5' end (in order to ligate the probes). We nonetheless always ordered the oligonucleotides without the phosphate (because the cost was significantly lower) and
                            then performed <A href="#">phosphorylation</A> by means of T4 Polynucleotide Kinase prior to ligation. </P><P class="lead">For each probe we ran an analysis of the secondary structure by means of available servers online (<A href="#NUPACK"><SPAN style="color:blue">NUPACK</SPAN></A>, <A href="Mfold"><SPAN style="color:blue">MFold</SPAN></A>): in all cases the structure of the probe, of its amplicon and of the series of 4-5 copies of the amplicon
                            were tested in order to check the absence of unwanted secondary structures. We also used <A href="RNAstructure"><SPAN style="color:blue">RNAstructure</SPAN></A> DuplexFold to test the secondary structure of the dimer probe/miRNA: we were not able to find a more suitable tool for
                            the analysis of the duplex; nonetheless we believe that this server, despite its limitations with respect to our analysis (no possibility of having a circular probe, no possibility to have a DNA/RNA dimer), was enough to show
                            qualitatively the interaction between our probe and let-7a.</P></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#SYBRParagraph"><DIV class="card-header"><H3 class="card-link">
									 SYBR Green I
								  </H3></DIV></A><DIV id="SYBRParagraph" class="collapse"><DIV class="card-body"><P class="lead">Two main alternatives are suitable in order to test the efficacy of Rolling Circle Amplification (<A href="#Deng"><SPAN style="color:blue">Deng <I>et al.</I>, 2014</SPAN></A>; <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A>). First of all, the amplicons can be tested by means of an agarose gel to verify the size; nonetheless, this method shows some limitations because of the large size of the amplicons. </P><P class="lead">A more valid alternative is instead to perform a real-time fluorescence measurement by means of SYBR Green I.</P><P class="lead">SYBR green I is an intercalating dye that preferentially binds to minor grooves of double-stranded (dsDNA) (<A href="#Zipper"><SPAN style="color:blue">Zipper <I>et al.</I>, 2004</SPAN></A>). It has also been shown to bind to single-stranded DNA (ssDNA) and RNA (for which instead SYBR Green II is a more suitable option (<A href="#SYBRG"><SPAN style="color:blue">Sigma-Aldrich</SPAN></A>)), but with a significantly lower performance (<A href="#Vitzthum"><SPAN style="color:blue">Vitzthum <I>et al.</I>, 1999</SPAN></A>). </P><P class="lead">When complexed with nucleid acid, SYBR Green I absorbs blue light (maximum excitation wavelength is 497 nm) and emits green light (emission peak at 520 nm) (<A href="#SYBRGI"><SPAN style="color:blue">Sigma-Aldrich</SPAN></A>), which makes it suitable for quantification - by means of a plate reader - of the DNA amplicons (i.e. the reverse complement of the probes) from our Rolling Circle Amplification (RCA). </P><P class="lead">Indeed, since we verified in all cases the absence of unwanted secondary structures (more details in <A href="#DetailedDesign"><SPAN style="color:blue">Detailed Design</SPAN></A>), the stems in the probes and in the amplicons are the only double-stranded targets to which SYBR Green I can preferentially bind: this allows to observe the increase over time in the size of the amplicon during RCA.</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 7.</B> &quot;Scheme for miRNA detection by TIRCA in vitro&quot; [Reproduced from <A href="#Deng"><SPAN style="color:blue">Deng <I>et al.</I>, 2014</SPAN></A> (Figure 1A)].</FIGCAPTION></FIGURE></CENTER></DIV></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#collapseOne"><DIV class="card-header"><H3 class="card-link">
										Detailed design
								  </H3></DIV></A><DIV id="collapseOne" class="collapse"><DIV class="card-body"><P class="lead">This section is more specifically dedicated to the reasonings behind the sequences of our probes.</P><DIV class="card"><A data-toggle="collapse" href="#ProbeAnalysis"><DIV class="card-header"><H4 class="card-link">
													Analysis of given probes
												</H4></DIV></A><DIV id="ProbeAnalysis" class="collapse"><DIV class="card-body"><P class="lead">We started our design from the analysis of one probe from <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A>, namely &quot;let-7a probe 1&quot; (Probe 2 for us). The sequence was the following one:</P><P class="lead">5’-p<U><SPAN style="color:orange">ACCTCA</SPAN></U><I>TTGTATAGCCCCCCCC</I><SPAN style="color:green">TGAGGTAGTAGGTTG</SPAN><I>CCC<U>AACTATA</U></I><U><SPAN style="color:orange">CAACCTACT</SPAN></U>-3’</P><P class="lead">where:</P><UL><LI>the regions in <I>italic</I> are those belonging to the loops of the hairpin</LI><LI>the regions in <SPAN style="color:orange">orange</SPAN> and <SPAN style="green">green</SPAN> are those belonging to the stem of the hairpin (and which are complementary with each other)</LI><LI>the <U>underlined</U> region is the one complementary to the miRNA (let-7a-5p: <B>UGAGGUAGUAGGUUGUAUAGUU</B>)</LI></UL><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 8.</B> Secondary structure of &quot;let-7a probe 1&quot; (Probe 2 for us). dG=-10.40.</FIGCAPTION></FIGURE></CENTER><P class="lead">Such probe consists of a double-stranded stem part, a 10 bases-long loop (which from now on we will refer to as &quot;small loop&quot; - on the right in the figure above) and a 16 bases-long loop (&quot;large loop&quot; - on the left). As we can
                            observe, the toehold region of the probe (i.e. the part on the small loop where the miRNA binds) is 7 bases long, in accordance with <A href="#Deng"><SPAN style="color:blue">Deng <I>et al.</I>, 2014</SPAN></A>, who proved it to be the optimal length to achieve both sensitivity and specificity.</P><P class="lead">The amplicon of such probe is therefore*: </P><H5><SPAN class="thicker">5'-<SPAN style="color:orange">AGTAG<MARK style="color:orange">G</MARK>TTG</SPAN><I>TA<MARK>T</MARK>AGT</I></SPAN><SPAN class="lead"><I>TGGG</I><SPAN class="lead" style="color:green">CAACCTACTACCTCA</SPAN><I><SPAN class="lead" style="color:red">GGG</SPAN>GGGGGCTATACAA</I></SPAN><SPAN class="thicker" style="color:orange">TGAGGT</SPAN>-3’</H5><P class="lead">where:</P><P class="lead">the sequence in <B>bold</B> is the one which is complementary to the gRNA (except for two mismatches, which are <MARK>highlighted</MARK>) and the region in <SPAN style="color:red">red</SPAN> is the PAM sequence (in this case single stranded). </P><P class="lead">We emphasize here that the PAM sequence is on a single-stranded part of the amplicon (the one complementary to the large loop of the probe): therefore, such single-stranded PAM can only be present on the amplicon, and not on the probe itself (as would have been instead if the PAM was on a double stranded part).</P><P class="lead">The gRNA sequence (as indicated by <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A>) is:</P><P class="lead">5’-<I>ACU<MARK>G</MARK>UA</I>CAA<MARK>A</MARK>CUACU<SPAN style="color:red">|</SPAN>ACCUCA(GUUUUAGAGCUAGAAAUAGCAAGUUAAAAUAAGGCUAGUCCG) -3’</P><P class="lead">with the scaffold region indicated in parentheses. The region out of the brackets is the spacer, binding to the amplicon, and the sequence in <I>italic</I> corresponds in particular to the part of the spacer binding on the loop of the
                            amplicon (with the rest of the spacer binding to the stem). The sign <SPAN style="color:red">|</SPAN> indicates the position where the gRNA binds to the point on the amplicon where each new &quot;copy&quot; of the amplicon is considered to start (i.e. the point where
                            the 3' of a &quot;subunit&quot; of the amplicon and the 5' of the successive subunit are linked together).</P><P class="lead">More specifically, we can notice that in this design the spacer coincides with the reverse complement of let-7a, with the exception of the two mismatches and of a missing A at the beginning. The template of the gRNA for Cas9
                            would therefore be:</P><P class="lead">5'-[reverse complement of miRNA]-[scaffold]-3'</P><P class="lead">The expected interaction between amplicon and gRNA is outlined in Figure 9:</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 9.</B><I>On the left:</I> Generic interaction between a target and a gRNA for Cas9 [Reproduced from <A href="#Xie"><SPAN style="color:blue">Xie and Yang, 2013</SPAN></A> (Figure 1A)]. <I>On the right:</I> Predicted interaction of one subunit of the amplicon of Probe 2 with the gRNA.</FIGCAPTION></FIGURE></CENTER><P class="lead">We can observe how the PAM sequence (in red in the figure) is located at the very beginning of the large loop in the amplicon, whereas the gRNA binds to the whole stem part and partially to the small loop.</P><H6>*Here and after, when referring to the &quot;amplicon sequence&quot;, we only show one single copy of the reverse transcript of the probe. The actual amplicon, by definition of Rolling Circle Amplification, is of course made instead of
                            sequential copies of this &quot;unitary&quot; sequence.</H6></DIV></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#ComparisonCas"><DIV class="card-header"><H4 class="card-link">
													Comparison Cas9/Cas12a
												</H4></DIV></A><DIV id="ComparisonCas" class="collapse"><DIV class="card-body"><P class="lead">We then tried to design our own probes for Cas 12a, working backwards from the gRNA.</P><P class="lead">Contrarily to Cas 9, for which the PAM must be on the 3' side of the target, for Cas12a the PAM must be on the 5’ side of the target instead. This implies that the scaffold part of the gRNA must be on the 5’ side (instead of the 3’) as well (Figure 10).</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 10.</B> &quot;Schematic representation of Lba Cas12a nuclease sequence recognition and DNA cleavage&quot;. [Reproduced from <A href="#NebCas12a"><SPAN style="color:blue">New England BioLabs</SPAN></A>].</FIGCAPTION></FIGURE></CENTER><P class="lead">Below is shown a direct comparison of the interaction between target amplicon and gRNA for Cas 9 and Cas 12a.</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 11.</B> Comparison of the interaction between target amplicon and gRNA for Cas 9 (<I>on the left</I>) and Cas 12a (<I>on the right</I>).</FIGCAPTION></FIGURE></CENTER><P class="lead">We therefore conclude that the template for our guide RNA for Cas 12a should be:</P><P class="lead">5’-(UAAUUUCUACUAAGUGUAGAU)NNNNNN<SPAN style="color:red">|</SPAN>NNNNNNNNN<I>NNNNNNN</I>-3’  [<B><I>gRNA template</I></B>] </P><P class="lead">where the sequence in parentheses indicates the scaffold of the gRNA for LbCas12a. The sequence out of the brackets is the spacer, binding to the amplicon, and in particular the sequence in <I>italic</I> corresponds to the part binding on the loop of the amplicon.</P><P class="lead">The spacer is therefore 22 bases long (as let-7a-5p), 15 of which bind to the stem part of the amplicon and the remaining 7 bind to the small loop of the amplicon. Note that the gRNA for Cas9 from <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A> was instead 21 bases long (15 and 6): we decided to add one more base at the end to completely match the length of the miRNA.
												</P><P class="lead">We can notice that also in this design the spacer has to coincide with the reverse complement of let-7a (as for Cas 9) . The template of the gRNA for Cas12a would therefore be: </P><P class="lead">5'-[scaffold]-[reverse complement of miRNA]-3'</P></DIV></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Sequences"><DIV class="card-header"><H4 class="card-link">
													Probes for Cas12a
												</H4></DIV></A><DIV id="Sequences" class="collapse"><DIV class="card-body"><P class="lead">From the template above we can therefore conclude that the gRNA for our Cas 12a system, designed as the one for Cas 9 from <A href="#Deng"><SPAN style="color:blue">Deng <I>et al.</I>, 2014</SPAN></A>, has to be: </P><P class="lead">5’-(UAAUUUCUACUAAGUGUAGAU)AACUAU<SPAN style="color:red">|</SPAN>ACAACCUAC<I>UACCUCA</I>-3’   [<B><I>gRNA sequence</I></B>]</P><P class="lead">From the specifications for the probe above (10 bases small loop, 16 bases large loop) and from the gRNA sequence, the template amplicon therefore needs to have the following structure:</P><H5>5’-<SPAN class="thicker"><SPAN style="color:orange">ATAGTT</SPAN></SPAN><SPAN class="lead"><I>NNNNNNNNNNNN<SPAN style="color:red">TTTN</SPAN></I><SPAN style="color:green">AACTATACAACCTAC</SPAN><I>NNN</I></SPAN><SPAN class="thicker">TGAGGTA<SPAN style="color:orange">GTAGGTTGT</SPAN></SPAN><SPAN class="lead">-3’  [<B><I>amplicon template</I></B>]</SPAN></H5><P class="lead">We then proceeded to define the bases for the Ns, aiming not to have unwanted minor secondary structures (e.g. smaller loops) in the loops. This was done mostly by considering pairing principles, e.g. avoiding non-Watson-Crick interaction (e.g. T-G) which might be thermodynamically favoured or trying not to have complementary bases with more than 1 base in between (which might lead to hairpin loops). In all cases, the minimum free energy structure (MFE) was plotted by means of the available software (NUPACK, Mfold), both for the amplicon and the probe - i.e. its reverse complement-, to check that the intended dumbbell shape was indeed achieved.</P><P class="lead">This lead us to the sequence of Probe 1 and Probe 6 (Probes from 2 to 5 were the probes for Cas 9 from <A href="#Deng"><SPAN style="color:blue">Deng <I>et al.</I>, 2014</SPAN></A> and <A href="#Qiu"><SPAN style="color:blue">Qiu <I>et al.</I>, 2018</SPAN></A>).</P><P class="lead">-----</P><P class="lead">We also wanted to test the case of probes with the amplicon having the PAM sequence not on the large loop, but on the stem instead (i.e. a double-stranded PAM, as usually required in Cas systems, and not single-stranded). We considered in this case three different alternatives:</P><OL><LI><P class="lead">Changing 4 bases in the large loop in order for them to be complementary to the PAM sequence, without adding more bases. This leads to a 19 bases-long stem, a 10 bases-long &quot;small&quot; loop and a 8 bases-long &quot;large&quot; loop. The template sequence of the amplicon is the following one:</P><H5>5’-<SPAN class="thicker"><SPAN style="color:orange">ATAGTT</SPAN></SPAN><SPAN class="lead"><SPAN style="color:red">N'AAA</SPAN><I>NNNNNNNN</I><SPAN style="color:red">TTTN</SPAN><SPAN style="color:green">AACTATACAACCTAC</SPAN><I>NNN</I></SPAN><SPAN class="thicker">TGAGGTA<SPAN style="color:orange">GTAGGTTGT</SPAN></SPAN><SPAN class="lead">-3’  (with N' being the base complementary to the N in the PAM)</SPAN></H5></LI><LI><P class="lead">Inserting 4 more bases complementary to the PAM on one end of the large loop (after ATAGTT), without changing any base. This leads to a 19 bases-long stem, a 10 bases-long small loop and a 12 bases-long large loop. The template sequence of the amplicon is the following one:</P><H5>5’-<SPAN class="thicker"><SPAN style="color:orange">ATAGTT</SPAN></SPAN><SPAN class="lead"><SPAN style="color:red">N'AAA</SPAN><I>NNNNNNNNNNNN</I><SPAN style="color:red">TTTN</SPAN><SPAN style="color:green">AACTATACAACCTAC</SPAN><I>NNN</I></SPAN><SPAN class="thicker">TGAGGTA<SPAN style="color:orange">GTAGGTTGT</SPAN></SPAN><SPAN class="lead">-3’</SPAN></H5></LI><LI><P class="lead">Inserting 4 more bases complementary to the PAM on one end of the large loop (after ATAGTT) and 4 more bases at the other end of the large loop (before the PAM sequence), in order to keep the original length of the large loop (16 bases). This leads to a 19 bases-long stem, a 10 bases-long small loop and a 16 bases-long large loop. The template sequence of the amplicon is the following one:</P><H5>5’-<SPAN class="thicker"><SPAN style="color:orange">ATAGTT</SPAN></SPAN><SPAN class="lead"><SPAN style="color:red">N'AAA</SPAN><I>NNNNNNNNNNNNNNNN</I><SPAN style="color:red">TTTN</SPAN><SPAN style="color:green">AACTATACAACCTAC</SPAN><I>NNN</I></SPAN><SPAN class="thicker">TGAGGTA<SPAN style="color:orange">GTAGGTTGT</SPAN></SPAN><SPAN class="lead">-3’</SPAN></H5></LI></OL><P class="lead">These three alternatives led, respectively, to Probe 7, Probe 8 and Probe 9.</P><P class="lead">-----</P><P class="lead">Finally, Probe 10 was designed in a way to have a mismatched base in the stem with respect to the let-7a sequence (<MARK>highlighted</MARK> in both strands below):</P><P class="lead">5'-p<U><SPAN style="color:orange">ACAACCTAC</SPAN><I>TACCTCA</I></U><I>AAC</I><SPAN style="color:green">GTAGGTTGTA<MARK><SPAN style="color:green">G</SPAN></MARK>AGTT</SPAN><I>TAAAGGGAGTCGGCGG</I><U><SPAN style="color:orange">AACT<MARK><SPAN style="color:orange">C</SPAN></MARK>T</SPAN></U>-3'</P></DIV></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#gRNADes"><DIV class="card-header"><H4 class="card-link">
													Design of gRNAs after new hypothesis
												</H4></DIV></A><DIV id="gRNADes" class="collapse"><DIV class="card-body"><P class="lead">Halfway through our project (see <A href="https://2018.igem.org/Team:EPFL/Notebook-Detection"><SPAN style="color:blue">Notebook</SPAN></A> for more details), after starting testing our amplicons with Cas12a and the fluorescent reporter (DNase Alert), we realized that the probe itself (more specifically the product of RCA in the absence of miRNA, i.e. with no amplicon) was triggering the Cas system causing a very high fluorescence signal, comparable to the signal obtained for the samples with miRNA (i.e. with probe+amplicon).</P><P class="lead">We hypothesized that this was due to the fact the our Cas12a was working PAM-independently (more details in &quot;Promiscuous Cas12a activation: probes as a target&quot; in <A href="https://2018.igem.org/Team:EPFL/Results"><SPAN style="color:blue">Results</SPAN></A>). More specifically, our gRNA was meant to target the whole stem (and in addition 7 bases in the small loop) of the amplicon; since the stem is double-stranded, the target sequence for the gRNA is also present in the probe (in the opposite strand).</P><P class="lead">This would not have been a problem if the Cas had been working, as expected, PAM-dependently, because the PAM is only contained in the amplicon, not in the probe. Nonetheless, if the Cas does not need the PAM sequence, but simple recognizes a target from the sequence of the gRNA, then also the probe itself is recognized as a target. Moreover, since the concentration of the probe in the RCA reaction is higher than the expected concentration of amplicon, the signal from the probe behaves as noise, overcoming the signal of interest (i.e. from the amplicon).</P><P class="lead">We therefore designed a new guide RNA with the aim of targeting only the amplicon and not the probe. Our idea was to have the gRNA binding not on the stem, but on the large loop of the amplicon instead. Since the loops of the amplicon are single-stranded (and not double-stranded as the stem) this should allow the gRNA to target only the amplicon and not the probe, being the target sequence contained only in the amplicon and not in its reverse-complement: more specifically, we decided to design a guide RNA perfectly complementary to the large loop of the amplicon of Probe 1; in this way Probe 1, having on the contrary exactly the same sequence as the gRNA, should have never been targeted by this new gRNA.</P><P class="lead">As from the template gRNA above (5'-[scaffold]-[reverse complement of miRNA]-3'), the spacer was therefore modified to bind (with perfect match) to the large loop of the amplicon of probe 1. </P><P class="lead">Two different designs were tested, one  - referred to elsewhere as &quot;S_1&quot; - binding to the whole large loop and to the first 4 bases after the large loop (for a total of a 20 bases-long spacer), and one - &quot;L_1&quot; elsewhere - binding only to the large loop (16 bases-long spacer).  The complete sequences are the following ones:</P><P class="lead">5'-(UAAUUUCUACUAAGUGUAGAU)UAAAGGGAGUCGGCGGAACU-3' [<B>gRNA sequence - S_1</B>]</P><P class="lead">5'-(UAAUUUCUACUAAGUGUAGAU)UAAAGGGAGUCGGCGG-3' [<B>gRNA sequence - L_1</B>]</P><P class="lead">The comparison between the mode of action of the previous, original gRNA and the &quot;new&quot; ones is better explained in Figure 12:</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 12.</B> Comparison of the interaction between the gRNA and the amplicon for the three different gRNAs we investigated</FIGCAPTION></FIGURE></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><H1 id="Cas12aAssay">Our detection scheme</H1><P class="lead">We envision a  follow-up based on repeated liquid biopsies in order to track the sequences that have been identified using our bioinformatic software, amplified by either PCR, isothermal amplification or RCA, and finally detected directly in the plasma using our Cas12a based system.</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 13.</B> Representation of our detection scheme: from a single drop of blood we collect the plasma in which reside our biomarkers, ctDNA and miRNAs. Depending on the follow-up assay (vaccine monitoring or relapse detection), we will amplify specific target sequences that we will detect afterwards using our Cas12a assay.</FIGCAPTION></FIGURE></CENTER><P class="lead">In the following example, the patient receives our treatment based on a cocktail of neoantigens presented on the surface of the encapsulin. The target population decreases with time, which suggests a response to our immunotherapy-based vaccine for a certain period of time but, due to the emergence of resistance or the survival of another cell population, the patient relapses. Chromosomal rearrangements and miRNAs are then the object of our detection, and would suggest in this particular case a potential relapse. It is then strongly recommended for the patient to carry out a clinical test (biopsy, imaging, endoscopy) for  confirmation.</P><CENTER><FIGURE><FIGCAPTION class="mt-3 text-muted"><B>Figure 14.</B> Example of the use of biomarkers as a means of prognosis on the health of a patient with melanoma. In this case, the patient receives our vaccine as a treatment, and we assume that the treatment worked. This would be marked by a decrease in the concentration of ctDNA characteristic of the neoantigens targeted by our vaccine, ideally until their complete eradication. The condition of the patient stabilizes for a certain amount of time but it still ends up in relapse, which is nonetheless promptly marked by an increase of chromosomal rearrangements ctDNA fragments in the blood.</FIGCAPTION></FIGURE></CENTER><ARTICLE><H2><I><U>References</U></I></H2><UL><LI id="Abe2003">Abe, Kenji. &quot;Direct PCR from Serum.&quot; <I>PCR Protocols</I>. Humana Press, 2003. 161-166.</LI><LI id="Ali">Ali, M. Monsur, et al. &quot;Rolling circle amplification: a versatile tool for chemical biology, materials science and medicine.&quot; <I>Chemical Society Reviews</I>, 43.10 (2014): 3324-3341.</LI><LI id="Baklanov1996">Baklanov, Michail M., Larisa N. Golikova, and Enrst G. Malygin. &quot;Effect on DNA transcription of nucleotide sequences upstream to T7 promoter.&quot; <I>Nucleic acids research</I>, 24.18 (1996): 3659-3660.</LI><LI id="Calapre2017">Calapre, Leslie, et al. &quot;Circulating tumor DNA (ctDNA) as a liquid biopsy for melanoma.&quot; <I>Cancer letters</I>, 404 (2017): 62-69.</LI><LI id="Chen2018">Chen, Janice S., et al. &quot;CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity.&quot; <I>Science</I>, 360.6387 (2018): 436-439.</LI><LI id="Cheng">Cheng, Yongqiang, et al. &quot;Highly sensitive determination of microRNA using target-primed and branched rolling-circle amplification.&quot; <I>Angewandte Chemie International Edition</I>, 48.18 (2009): 3268-3272.</LI><LI id="Deng"> Deng, Ruijie, et al. &quot;Toehold-initiated rolling circle amplification for visualizing individual microRNAs in situ in single cells.&quot; <I>Angewandte Chemie</I>, 126.9 (2014): 2421-2425.</LI><LI id="DNaseAlertIDT">&quot;DNaseAlert™&quot; - Integrated DNA Technologies website. URL: https://eu.idtdna.com/site/order/stock/index/alert (Accessed 16/10/2018).</LI><LI id="NebCas12a">&quot;EnGen Lba Cas12a (Cpf1)&quot; - New England BioLabs website. URL: https://international.neb.com/products/m0653-engen-lba-cas12a-cpf1#Product%20Information_Notes (Accessed 24/09/2018)</LI><LI id="Girotti2016">Girotti, Maria Romina, et al. &quot;Application of sequencing, liquid biopsies, and patient-derived xenografts for personalized medicine in melanoma.&quot; <I>Cancer discovery</I>, 6.3 (2016): 286-299.</LI><LI id="Gootenberg2017">Gootenberg, Jonathan S., et al. &quot;Nucleic acid detection with CRISPR-Cas13a/C2c2.&quot; <I>Science</I>, (2017): eaam9321.</LI><LI id="Gray2015">Gray, Elin S., et al. &quot;Circulating tumor DNA to monitor treatment response and detect acquired resistance in patients with metastatic melanoma.&quot; <I>Oncotarget</I>, 6.39 (2015): 42008.</LI><LI id="Harris2016">Harris, Faye R., et al. &quot;Quantification of somatic chromosomal rearrangements in circulating cell-free DNA from ovarian cancers.&quot; <I>Scientific reports</I>, 6 (2016): 29831.</LI><LI id="Heitzer2017">Heitzer, Ellen, et al. &quot;The potential of liquid biopsies for the early detection of cancer.&quot; <I>NPJ precision oncology</I>, 1.1 (2017): 36.</LI><LI id="Larrea">Larrea, Erika, et al. &quot;New concepts in cancer biomarkers: circulating miRNAs in liquid biopsies.&quot; <I>International journal of molecular sciences</I>, 17.5 (2016): 627.</LI><LI id="Li2018">Li, Shi-Yuan, et al. &quot;CRISPR-Cas12a-assisted nucleic acid detection.&quot; <I>Cell discovery</I>, 4.1 (2018): 20.</LI><LI id="Miao">Miao, Peng, et al. &quot;Ultrasensitive detection of microRNA through rolling circle amplification on a DNA tetrahedron decorated electrode.&quot; <I>Bioconjugate chemistry</I>, 26.3 (2015): 602-607.</LI><LI id="Mirzaei">Mirzaei, Hamed, et al. &quot;MicroRNAs as potential diagnostic and prognostic biomarkers in melanoma.&quot; <I>European journal of cancer</I>, 53 (2016): 25-32.</LI><LI id="Mitchell">Mitchell, Patrick S., et al. &quot;Circulating microRNAs as stable blood-based markers for cancer detection.&quot; <I>Proceedings of the National Academy of Sciences</I>, 105.30 (2008): 10513-10518.</LI><LI id="Olsson2015">Olsson, E. et al. Serial monitoring of circulating tumor DNA in patients with primary breast cancer for detection of occult metastatic disease. <I>EMBO Mol Med</I>, 7, 1034–1047 (2015).</LI><LI id="Qiu">Qiu, Xin-Yuan, et al. &quot;Highly Effective and Low-Cost MicroRNA Detection with CRISPR-Cas9.&quot; <I>ACS synthetic biology</I>, 7.3 (2018): 807-813.</LI><LI id="RNAstructure">Reuter, Jessica S., and David H. Mathews. &quot;RNAstructure: software for RNA secondary structure prediction and analysis.&quot; <I>BMC bioinformatics</I>, 11.1 (2010): 129.</LI><LI id="sgRNASynth">&quot;sgRNA Synthesis Using the HiScribe™ Quick T7 High Yield RNA Synthesis Kit&quot; - New England BioLabs website. URL:https://international.neb.com/protocols/2015/11/24/sgrna-synthesis-using-the-hiscribe-quick-t7-high-yield-rna-synthesis-kit-neb-e2050 (Accessed 14/10/2018)</LI><LI id="Siegel2018">Siegel, R. L., Miller, K. D. and Jemal, A. &quot;Cancer statistics, 2018.&quot; <I>CA: A Cancer Journal for Clinicians</I>, (2018) 68: 7-30.</LI><LI id="SYBRGI"> &quot;SYBR Green I nucleic acid gel stain&quot; - Sigma-Aldrich. Datasheet. URL: https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma-Aldrich/Datasheet/s9430dat.pdf (Accessed 11/10/2018) </LI><LI id="SYBRG"> &quot;SYBR Green II RNA Gel Stain&quot; - Sigma-Aldrich. Datasheet. URL: https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma/Datasheet/2/s9305dat.pdf (Accessed 11/10/2018)</LI><LI id="Tsao2015">Tsao, Simon Chang-Hao, et al. &quot;Monitoring response to therapy in melanoma by quantifying circulating tumour DNA with droplet digital PCR for BRAF and NRAS mutations.&quot; <I>Scientific reports</I>, 5 (2015): 11198.</LI><LI id="Underhill2016">Underhill, Hunter R., et al. &quot;Fragment length of circulating tumor DNA.&quot; <I>PLoS genetics</I>, 12.7 (2016): e1006162.</LI><LI id="Vitzthum">Vitzthum, Frank, et al. &quot;A quantitative fluorescence-based microplate assay for the determination of double-stranded DNA using SYBR Green I and a standard ultraviolet transilluminator gel imaging system.&quot; <I>Analytical biochemistry</I>, 276.1 (1999): 59-64.</LI><LI id="Xie">Xie, Kabin, and Yinong Yang. &quot;RNA-guided genome editing in plants using a CRISPR–Cas system.&quot; <I>Molecular plant</I>, 6.6 (2013): 1975-1983.</LI><LI id="NUPACK">Zadeh, Joseph N., et al. &quot;NUPACK: analysis and design of nucleic acid systems.&quot; <I>Journal of computational chemistry</I>, 32.1 (2011): 170-173.</LI><LI id="Zetsche2017">Zetsche, Bernd, et al. &quot;Multiplex gene editing by CRISPR–Cpf1 using a single crRNA array.&quot; <I>Nature biotechnology</I>, 35.1 (2017): 31. </LI><LI id="Zipper">Zipper, Hubert, et al. &quot;Investigations on DNA intercalation and surface binding by SYBR Green I, its structure determination and methodological implications.&quot; Nucleic acids research, 32.12 (2004): e103-e103</LI><LI id="Mfold">Zuker, Michael. &quot;Mfold web server for nucleic acid folding and hybridization prediction.&quot; <I>Nucleic acids research</I>, 31.13 (2003): 3406-3415.</LI></UL></ARTICLE></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></SECTION></MAIN></DIV></DIV></DIV><DIV class="container"><DIV class="align-items-center"><DIV class="text-center"><DIV><SPAN class="pl-3 d-inline-block">iGEM2018@groupes.epfl.ch</SPAN></DIV><SPAN class="d-block mt-4">IGEM EPFL 2018</SPAN><A href="https://twitter.com/EPFL_iGEM" data-fancybox="" class="btn btn-twitter btn-icon-label btn-circle"><SPAN class="btn-inner--text">Twitter</SPAN></A><A href="https://www.facebook.com/igemepfl/?ref=br_rs" data-fancybox="" class="btn btn-facebook btn-icon-label btn-circle"><SPAN class="btn-inner--text">Facebook</SPAN></A></DIV></DIV></DIV></DIV></DIV></BODY></HTML>