Team:EPFL/Notebook-Detection-ctDNA
iGEM EPFL 2018
iGEM EPFL 2018
HOME
Project
Description
Take a closer look at CAPOEIRA
Design
Learn more about how we designed CAPOEIRA
Software
Explore the Bioinformatic Pipeline implemented in CAPOEIRA
Results
Discover the Results of our experiments
Demonstration
Demonstration of how our Therapeutic System works
Interlab
Learn more about our participation in the Interlab Study
Parts
Discover the Parts that make up CAPOEIRA
Improve
Explore our Novel and Improvements Parts
Measurement
Learn about measurement approaches we used for Part Characterization
Team
Team Members
Learn more about our Team Members
Collaborations
Discover the interactions we had with other iGEM Teams during CAPOEIRA
Human Practices
Integrated Human Practices
Discover how we integrated Human Practices into the Purpose, Design and Implementation of CAPOEIRA
Ethics
Explore the ethical considerations of CAPOEIRA
Applied Design
Explore how our therapeutic system attempts to meet real world medical need
Education and Public Engagement
Discover our Education and Public Engagement work with the public and scientific communities
European and Eurasian Meetup
Learn about our interactions with the iGEM Community during the Meetups
Lab
Safety
Entrepreneurship
Acknowledgements
Attributions
Sponsors
Partners
Awards
Top
Follow Up Notebook
FQ reporter assays Notebook
Chromosomal rearrangements Notebook
Point mutations Notebook
FQ reporter assays Notebook
Sunday,01/07/2018
This set of experiments are based on our protocol: Fluorophore-Quencher reporter Cas12a assay.
ts aim is to detect a specific dsDNA sequence (the Activator) with the CRSIPR/Cas12a system. If said sequence is found in the sample, Cas12a will unleash single-stranded DNase acivity and thus cleave every ssDNA found in the sample.                       We're using this mechanism to get a fluorescent read out using DNaseAlert (ssDNA reporters).
Our template is made of a non target strand (NTS) and a target strand (TS) (target 1_NTS /target 1_TS cf. supplementary materials of Chen, J. S. et al.)
NTS (non targeting strand):GCTTGTGGCCGTTTACGTCGCCGTCCAGCTCGACCAGGATGGGCACCACCCCGGC
TS (targeting strand):GCCGGGGTGGTGCCCATCCTGGTCGAGCTGGACGGCGACGTAAACGGCCACAAGC
The TS is the strand which will be recognized by the cas12a and thus is complementary to the crRNA.
We annealed our NTS and TS strand according to our annealing protocol: this is are activator/target.
We transcribed our crRNA using our transcription protocol from the following sequence:
GGTCGAGCTGGACGGCGACGATCTACACTTAGTAGAAATTACCTATAGTGAGTCGTATTAAG
Monday 30/07/18
Testing Cas12a complex + DNaseAlert
Simple experience to test whether the CRISPR/cas12a system works (i.e. if the crRNA was correctly designed and did form a complex with the cas12a which should eventually cleave the ssDNA reporter (DNaseAlert).
To get optimal results we've mixed the protocols found in the Supplementary materials of the paper stated above and the protocol given with the lba cas12a from NEB
Protocol:
Final sample (20μl) : Cas12a (50nM), crRNA (62nM), activator (1nM), DNaseAlert (50nM), NEBuffer 10x (1x)
Add 2μl of nuclease free water
Add 2μl of NEBuffer
Add 1μl of Cas12a (1μM)
Add 1μl of crRNA (1.37 μM)
Incubate 10 minutes at room temperatures
Add 4μl of activator (5nM 4.a)
Incubate 10 minutes at 37°C
Add 10μl of DNaseAlert (single use tube diluted in 40μl)
Let incubate in 37°C
Look at the fluorescence at desired times (here we checked the fluorescence after ~10minutes)
Negative control (done at the same time as the sample)
Add 3μl of Nuclease free water
Add 2μl of NEBuffer
Add 1μl of crRNA
Incubate for 10 minutes at room temperature (with other sample)
Add 4 μl of activator
Incubate for 10 minutes at 37°C (with other sample)
Add 10μl of DNaseAlert (single use tube diluted in 40μl)
Look at the fluorescence at desired times (here we checked the fluorescence after ~10minutes)
Results
Analysis/Discussion
As it can be seen on the picture there are some fluorescence from the sample with the activator and none from the negative control (without cas12a). This suggest that our CRISPR/cas12a system activates correctly the DNaseAlert. BUT the negative control should have been the same as the other sample without the activator to see if cas12a truly only gains its nuclease activity when the activator is present in the sample.
Tuseday 31/07/18
Trial 1 (50 nM Cas12a, 62.5 nM crRNA, 50 nM DNaseAlert, NEBuffer 2.1)
This experiment contained 50 nM of DNaseAlert and negative control contained only DNaseAlert (in addition to buffer) Incubation step (30min/37°C) at the following concentrations: Lbcas12a 250nM / crRNA 320nM. We made different samples                       with different activator concentrations : 1nM-1μM.
Results:
Analysis:
Controls seem to work and the enzyme seems to be activated, since we can measure fluorescence in all samples. Also, those with higher concentrations of template DNA exhibit more fluorescence, except for the 100 nM sample which has                       a fluorescence higher than the 1 μM sample.
Discussion
Overall, controls seem to work, samples with higher concentrations of template DNA exhibit more fluorescence, except for the 100 nM sample which has a fluorescence higher than the 1μM sample. Regarding this, we could have inverted                       the samples. Another explanation would be that for higher concentration of activator, the enzyme might be cleaving that instead of DNaseAlert reporter. The negative control should be cas12a with crRNA without any template. We'll                       modify this for the next experiment.
Thursday 02/08/2018
Trial 2 (50 nM Cas12a, 62.5 nM crRNA, 100 nM DNaseAlert, NEBuffer)
We did this experiment using Nebuffer 2.1 protocol and added a negative control sample containing this time Cas12a enzyme. Incubation step (30min/37°C) at the following concentrations: Lbcas12a 250nM / crRNA 320nM. We made different                       samples with different activator concentrations : 1nM-1μM.
Results:
Analysis:
We can see the increase in the level of fluorescence signal (again 1 μM is below 100 nM activator in terms of fluorescence). Moreover, the negative control level is mostly above the ones with activators.
Discussion
It seems that Cas12a could be activated even without binding to the activator.. Or maybe we put some DNA by mistake. The pattern of fluorescence as a function of concentration is still not what we would expect (more fluorescence for                       concentrated samples). We’re planning on using Binding buffer for next trial.
Friday 03/08/2018
Trial 3 (50 nM Cas12a, 62.5 nM crRNA, 200 nM DNaseAlert)
We did this experiment as usual with the following changes: 200 nM DNaseAlert (concentration has been doubled), and again negative control containing the Cas12a enzyme. The experiment was performed using binding buffer (10X, 20mM Tris-HCl,                       pH7.5, 100mM KCL, 5mM MgCl2, 1mMDTT, 5% gylcerol, 50 ug/ml heparin) used in the paper.
Results:
Analysis
We can see the increase in the level of fluorescence signal (again 1 μM is below 100 nM activator in terms of fluorescence). This time the negative control has the lowest fluorescence. The DNaseAlert concentration was not doubled in                       the positive control by mistake.
Discussion
We might be comparing fluorescence signals which are quite low. Maybe we should once again increase the DNaseAlert concentration.
Saturday 04/08/18
Trial 4 (50 nM Cas12a, 62.5 nM crRNA, 200 nM DNaseAlert)
We followed the protocol with the same modifications as in the 3rd trial (200 nM DNase alert). This time, we also repeated the dilutions in order to obtain activator at concentrations of 1μM, 100nM, 10nM and 1nM (i.e. we tested samples                       of 100nM, 10nM, 1nM and 0.1nM concentrations). However we forgot to add the buffer after the incubation step.
Results
Analysis
It is clear that we couldn't activate the enzyme this time, since the fluorescence of all our samples is ranging near the negative control’s one at all time. However, we got an increase of fluorescence regarding our positive control                       compared to other trials (nearly two times more), which is consistent with the fact that we doubled the amount of DNaseAlert substrate, and indicates that we could have forgotten to do so in trial 3.
Discussion
Overall, we can conclude that this trial is a complete failure. This big difference in terms of fluorescent signal with the 3rd trial (same concentration of DNaseAlert substrate) was either due to the fact that we forgot to add the                       binding buffer in an adequate amount, or maybe because our crRNA has degraded in the meantime
Saturday, 11/08/18
Trial 5 (50 nM Cas12a, 62.5 nM crRNA, 160 nM DNaseAlert)
Another trial following the same protocol using Binding buffer protocol.
Results
Analysis/Discussion
The results are not good at all. Again, the negative control sample exhibits an abnormally high fluorescence, 100 nM sample is at the same level of fluorescence than the 1 μM
Trial 6 (75 nM LbCas12a, 90 nM crRNA, 160 nM DNaseAlert)
Another trial following the same protocol using Binding buffer, where we increased the concentration of LbCas12a to 75 nM and crRNA to 90 nM, compared to the previous trial.
Results
Analysis
First, the signal of 1 μM of activator was below the others, but it increased slightly above the 100 nM, which we were expecting at the end. Also the negative control is increasing which is maybe due to contamination of DNase I enzyme.
Discussion
This one is has shown the best results so far. At the end of the assay, the fluorescence signal reached the expected pattern.
mONDAY, 13/08/18
Trial 7
In this trial, we wanted to evaluate whether the increase in the concentration of Cas12 can give us higher level of signal. We performed this trial with two different concentrations of the enzyme, following the same protocol as usual                       (with Binding buffer).
Results
(200 nM LbCas12a, 120 nM crRNA, 160 nM DNaseAlert)
(100 nM LbCas12a, 62.5 crRNA, 160 nM DNaseAlert)
Analysis
We can observe an increase in negative controls overall.
Discussion
The problem could be due to contamination of samples with pipettes (because we used the same pipette for taking Dnase I for positive control). We're planning on repeating another trial using new pipettes tips and cleaning well our                       pipettes. Also, one of our TAs highlighted that Cas13a is very unstable at low salt, which means it might bind things that it is not meant to. It could be interesting to investigate the right concentration of buffer to use, since                       diluting it too much could lead to a very low sodium concentration which makes the enzyme promiscuous. The promiscuous binding could explain why our negative control actually cleaves.
Trial 8 (200 nM LbCas12a, 120 nM crRNA, 160 nM DNaseAlert)
We're still trying to evaluate whether the increase in the concentration of Cas12a can give us higher level of signal. We performed this trial with two different concentrations of the enzyme. This time we cleaned our pipettes carefully,                       used new sterile pipette tips, and made sure for every other aspect so that we wouldn't get any contamination.
Results
Error bars were drawn just for 1 μM, for more clarity.
Analysis/Discussion
This time the rise in sample fluorescence was more satisfying (the 1 uM almost reached the same level of activation as previous positive controls). However, we still have a very sharp rise for the negative control (with Cas12+crRNA),                       which made this trial again unsuccessful. However, the negative control without Cas12 was not activated at all, and even the level of signal decreased somehow.
Tuesday, 14/08/18
Trial 9 (200 nM LbCas12a, 120 nM crRNA, 180 nM DNaseAlert)
We Tried different concentration of salt to see if it influences the way the Cas12a system works. To do this we increased the                   binding buffer concentration as well as the NEBuffer 2.1 for another tube. We wanted to see which of the buffer worked better.
Results
Activator as template
Negative control as template
Analysis
The results are not good at all. Increasing the concentration of one or the other of the two buffers results in an even lower                   activation of the enzyme. Indeed, increasing the concentration of binding buffer reduces both signals in negative control and                   sample with activators. Also Cas12a doesn't get fully activated neither in 1X NEBuffer, nor in 5X, which suggests that using binding                   buffer in the future could be more relevant.
Discussion
We're not sure that using 5x of each buffer is the most efficient way of increasing the salt concentration within the assay, but it                   doesn't seem to fix our problem anyways, since increasing the concentration of buffer reduces both signals in negative control and                   sample with activators. However, we got convinced that using binding buffer is a better option, since it has shown a better                   activation of the Cas12a enzyme. We decided to use it for all the other trials to come.
Wednesday, 15/08/18
Trial 10 (200 nM LbCas12a, 120 nM crRNA, 180 nM DNaseAlert)
Compared with the previous trial, we have once again diluted our activators. In addition, we have transcribed and purified our crRNA again and this time we made sure that we did every step in the right way. We wanted to try different                       kinds of negative in order to check whether we had contamination in our of our tubes which could explain our high negative control.
Results
Analysis
The negative control sample which contains crRNA without Cas12a, does not produce a signal, which suggests that our purified crRNA is free of DNases.
Discussion
There is no Dnase contamination in our crRNA solution, nor in our Cas12 solution or in our DnaseAlert substrate. The activation of cas12a in our sample seems to be due to something other than contamination...
Friday, 17/08/18
Trial 11
In this experiment with tried different concentration of crRNA while keeping Cas12a concentration constant (50nM cas12a vs. 62.5nM/50nM/36.25nM/25nM of crRNA). We did the pre-incubation at 37°C for around 15minutes.
Results
(50 nM LbCas12a, 36.25 nM crRNA, 200 nM DNaseAlert)
(50 nM LbCas12a, 50 nM crRNA, 200 nM DNaseAlert)
(50 nM LbCas12a, 62.5 nM crRNA, 200 nM DNaseAlert)
(50 nM LbCas12a, 25 nM crRNA, 160 nM DNaseAlert)
Analysis
Once again the negative controls do not work and the fluorescent signal produced was above all the others which still makes no sense since the enzyme should not be active in that sample (no target in the solution).
Discussion
We were thinking of doing the experiments as in the NEB protocol given with the cas12a (with NEBuffer + 10min preincubation at room temperature and a ratio of 10:10:1 of cas12a/crRNA/activator). Also thinking of ordering the Ascas12a                       from IDT.
Sunday, 19/08/18
Trial 12
We repeated once again the assay, this time according to the Neb protocol given with LbCas12a
10 minute preincubation at 25°C
same concentration of crRNA as cas12a
A
B
N
P
NEBuffer
5
5
5
5
Cas12a (1um)
1.5
1.5
1.5
-
crRNA(1um)
1.5
1.5
1.5
-
activator
5 [tube 2: 1uM]
5 [tube 3: 0.1uM]
-
-
DNaseAlert
5
5
5
5
DNase I
-
-
-
-
NFW
32
32
37
39.8
Total
50
50
50
50
Results
Analysis
This method does not seem to resolve our negative control problem in which there is still a mysterious activation of the enzyme that is once again cleaving the ssDNA reporters without activation.
Discussion
We're going to stick to the other protocols based on our reference paper for the next trials...
Monday, 20/08/18
Trial 13
We're trying to test some combinations of different concentrations of LbCas12a and crRNA (while keeping the concentration of activator constant : 100 nM), to see which ones work the best for our system. Also testing whether not incubating                       the samples could lead to better results. This time, we didn’t first create a Master Mix with 4X concentration of Cas12+crRNA (200 nM of Cas12 + 250 nM of crRNA) as in the paper, where we needed to dilute them to 1X (50 nM of Cas12                       + 62.5 nM of crRNA). Instead, we directly put the right amount in order to obtain the final concentration of Cas12+crRNA in each sample separately, and performed the assay with or w/o pre-incubation of the Cas12a/crRNA complex (for                       assembling).
Results
FQ1 (50 nM LbCas12a, 62.5 nM crRNA, 160 nM DNaseAlert), with Pre-incubation
FQ2 (75 nM LbCas12a, 62.5 nM crRNA, 160 nM DNaseAlert), with pre-incubation.
FQ3 (75 nM LbCas12a, 90 nM crRNA, 160 nM DNaseAlert), with pre-incubation.
FQ4 (75 nM LbCas12a, 90 nM crRNA, 160 nM DNaseAlert), without pre-incubation.
Analysis
We were able to get good activation for each of our samples (FQ1, FQ2, FQ3 and FQ4) with a relatively low rate of activation in the negative control for each except for the second one. Surprisingly, the experiment without pre-incubation                       gave up the best yield in terms of fluorescence.
Apart from that, we can also see through the error bars that we have a lot of variations between our two measured duplicates (not drawn for negative and positive controls), and from this, the only sample to consider for success would                       be the FQ4 one (without pre-incubation).
Discussion
We finally concluded that the problem was related to the Master mix step: the cas12a and crRNA are incubated at too high of a concentration. Also, in the samples without pre-incubation we were able to get a very good result which could                       mean that pre-incubation made Cas12a not as activated as it should be.
Besides, we're getting too much variation for each sample. Should we use more replica for each sample (right now we are using two replicas) ? or is it from the device ? Another possible sources of errors would be either pipetting or                       bubbles formation...
Tuesday, 21/08/18
Trial 14
Following the same hypothesis of the 13th trial, we're still looking for the ideal concentrations of LbCas12a and crRNA to use in our assay. Four Samples (100 nM of dsDNA activator) where either left to pre-incubate or not, and contained                       different amounts of the enzyme and crRNA. No master mix was prepared. We also did the pipetting more carefully and tried our best to avoid bubbles.
Results
FQ1 (50 nM LbCas12a, 62.5 nM crRNA, 160 nM DNaseAlert), with pre-incubation
FQ2 (75 nM LbCas12a, 62.5 nM crRNA, 160 nM DNaseAlert), with pre-incubation
FQ3 (75 nM LbCas12a, 90 nM crRNA, 160 nM DNaseAlert), with pre-incubation
FQ4 (75 nM LbCas12a, 90 nM crRNA, 160 nM DNaseAlert), without pre-incubation
Analysis
We can see the effect of pre-incubation here: without pre-incubation the slope is lower than with pre-incubation for the same concentrations of both Cas12a and crRNA (FQ3 vs FQ4). Also, we can see that the 75 nM of Cas12a and 90 nM                       of crRNA are the most efficient concentrations here (regardless of the incubation) since these samples (FQ3 and FQ4) gave out the highest signal.
Discussion
It looks like we're going in the right direction for figuring out the optimal concentrations to use in our assay.
Friday, 24/08/18
Trial 16
We did this experiment with the same hypothesis as in the previous two trials. We ran different assays with different concentrations of Cas12a and crRNA. All samples were incubated (for Cas12a assembling).
FQ6 (62.5 nM LbCas12a, 75 nM crRNA, 160 nM DNaseAlert), with pre-incubation
FQ2 (75 nM LbCas12a, 62.5 nM crRNA, 160 nM DNaseAlert), with pre-incubation
FQ3 (75 nM LbCas12a, 90 nM crRNA, 160 nM DNaseAlert), with pre-incubation
Analysis
Overall, this Trial was a success, and by looking at the slopes, we can say that the optimal concentrations of Cas12a and crRNA for our system were obtained for FQ6 sample (62.5 nM LbCas12a and 75 nM crRNA), since we got the sharpest                       slope, i.e. faster activation of Cas, for our sample of interest, which surpassed the positive control.
Discussion
We finished our optimization of the concentrations after obtaining a very nice result with our FQ6 sample, and an activation which reaches its maximum after approximatively 80 minutes. For the next assays, we're planning on working                       with 62.5 nM of LbCas12a and 75 nM crRNA while maybe trying to change the concentration of dsDNA activator to see how it affects the signal.
Notebook Chromosomal rearrangements
Wednesday, 05/09/18
PCR amplification of Bcr-Abl junction
In the PCR amplification of Bcr-Abl junction series of experiments, we want to amplify the Bcr-Abl junction in order to detect it further with LbCas12a. We're going to start from extremely low concentrations of template (1 pM                       and/or 10 fM) and try to amplify them with different primers (three forward primers and two reverse primers), each containing an inserted PAM sequence. There are two main aims:
Overcome the fact that there might exist no suitable PAM sequence (TTTN for Cas12a) nearby the SNP site or junction, for the Cas12a recognition assay.
Prove that these fragments can be specifically detected in very small quantities by combining the sensitivity of PCR amplification method and the specificity of CRISPR-Cas system, this in order to get as close as possible to the                         concentrations present in the blood, and so be able to establish a proof of concept about the possibility of such detection in blood plasma.
Please reffer to the PCR using Phusion® High-Fidelity DNA Polymerase protocol.
Approach: We ordered the sequence corresponding to the junction as gBlocks from Integrated DNA Technology (IDT), as well as several base pairs of the two normal genes (Bcr and Abl) (Fig.2, 3) that extend on both sides                       of the junction. Primer oligos were ordered from IDT (c.f. Fig.1):
Forward primers (from top to bottom, Fig.1 below):
F_primer_chromo_symu_01 (F1)
F_primer_chromo_symu_04 (F4)
F_primer_chromo_symu_03 (F3)
Reverse primers (from left to right, Fig.1 below):
R_primer_chromo_symu_02 (R2)
R_primer_chromo_symu_01 (R1)
Resuspension of the gBlocks/Oligos was done as following, according to this protocol:
Oligonucleotides/gBlocks Resuspension and Storage (IDT)
gBlocks/Oligos
Amount delivered (nmol)
Desired concentration
Amount of water to ass (ul)
Bcr-Abl template
0.002077
100 nM
20.8
Abl
0.002492
100 nM
24.9
Bcr
0.002060
100 nM
20.6
F_primer_Chromo_symu_01
75.1
100 µM
751
F_primer_chromo_symu_03
74.4
100 µM
744
F_primer_chromo_symu_04
66.6
100 µM
666
R_primer_chromo_symu_02
73.1
100 µM
731
Template DNA solutions (dilutions) were prepared as below:
Bcr-Abl (BA-)
Bcr (BC-)
Abl (AB-)
Dilution (label)
Final concentration
Amount of template to take (ul)
Nuclease free water (ul)
Dilution fold
BA-1
1 nM
2 (from resuspended 100 µM Bcr-Abl stock)
198
1:100 dilution
BA-2
10 pM
2 (from BA-1 dilution 1)
198
1:100 dilution
BA-3
1 pM
10 (from BA-2 dilution)
90
1:10 dilution
BA-4
100 fM
10 (from BA-3 dilution)
90
1:10 dilution
BA-5
10 fM
10 (from BA-4 dilution)
90
1:10 dilution
BA-6
1 fM
10 (from BA-5 dilution)
90
1:10 dilution
BC-1
1  nM
2 (from resuspended Bcr 100 nM stock)
198
1:100
BC-2
10 pM
2 (from BC-1 dilution)
198
1:100
BC-3
1 pM
10 (from BC-2 dilution)
90
1:10
BC-4
100 fM
10 (from BC-3 dilution)
90
1:10
BC-5
10 fM
10 (from BC-4 dilution)
90
1:10
BC-6
1 fM
10 (from BC-5 dilution)
90
1:10
AB-1
1 nM
2 (from resuspended Abl 100 nM stock)
198
1:100
AB-2
10 pM
2 (from AB-1)
198
1:100
AB-3
1 pM
10 (from AB-2)
90
1:10
AB-4
100 fM
10 (from AB-3)
90
1:10
AB-5
10 fM
10 (from AB-4)
90
1:10
AB-6
1 fM
10 (from AB-5)
90
1:10
Then, the 10 µM dilutions of the primers (for PCR use) were prepared:
Primers (100um stock solution)
Volume to take (ul)
Amount of water to add (ul)
Total volume (ul)
Tube label
F_primer_Chromo_symu_01
10
90
100
F1
F_primer_chromo_symu_03
10
90
100
F3
F_primer_chromo_symu_04
10
90
100
F4
R_primer_chromo_symu_01
10
90
100
R1
R_primer_chromo_symu_02
10
90
100
R2
Figure 1. Overview of the junction and primer sequences.
Figure 2. Bcr sequence ordered. Shown in green is the piece of the gene involved in the chromosomal rearrangement ("bcr gene" annotated fragment in fig.1). Also shown in light brown the annealing position of the forward primer one                         (F1) on this gene.
Figure 3. Abl sequence ordered. Shown in blue the piece of the gene involved in the chromosomal rearrangement ("abl gene" annotated fragment in fig.1). Also shown in light brown and grey the annealing positions of the reverse primer                         one (R1) and two (R2) on this gene, respectively
Hypothesis:
Bcr-Abl DNA sequence amplification should result in an amplicon of 105-109 bp when using R2 primer and either of the forward ones, or 142-146 bp if using R1 primer.
Amplification of Bcr and Abl fragments is linear (i.e. only forward primer annealing on Bcr template, reverse on Abl) rather than exponential, meaning that we shouldn't be able to observe it on the gel, given the low initial concentration                         (1 pM or 10 fM). Also, amplification of Bcr fragment should result in an amplicon of ~239 bp (F1 primer), while for Abl, the amplicon obtained with R1 is of 316 bp length, and 279 for R2 (see Fig.4):
Figure 4-a. Overview of the Bcr amplicon with primer F1.
Figure 4-b. Overview of the Abl amplicon with respectively primer R1 and R2 (from right to left).
Cas12a detection assays
In this series of experiments, we want to be able to detect specifically the presence of the junction (previously amplified by PCR), using Cas12a system. Recognition of the activator by the crRNA unleashes indiscriminate single-stranded                       DNA (ssDNA) cleavage activity by Cas12a that will completely degrades our ssDNA reporter molecules (DNaseAlert, IDT), which results in a higher fluorescent signal. For the CRISPR RNA design (crRNA), we chose to use a shorter guide                       sequence (17 bp rather than 20), designed to recognize the sequence right after the PAM introduced by primer F1, F3 but not F4. First, the choice of using a shorter crRNA is based on the work done by Li S, Cheng Q, Wang J et al.1,                       where they proved that point mutations within a large region (1st–16th bases) resulted in more than 2-fold difference in fluorescence signals for both 16-nt and 17-nt crRNA guide sequences.
We followed the Fluorophore-Quencher reporter Cas12a assay . crRNAs oligo fragments were annealed with a T7 primer and transcribed then purified following respectively the crRNA Transcription using T7 RNA Polymerase (Promega) and crRNA purification (ZYMO Research RNA Clean & Concentrator™-5 Kit) protocols.
Hypothesis:
Cas12a enzyme should be activated by two of the amplified templates (Bcr and Bcr-Abl junction); however, we should detect a signal that is 2-fold less (at least) intense when using the separate gene (Bcr) as activator, since it's                         target strand (TS, not shown here for clarity) present several mismatches with the used crRNA, which is identical to the non-target strand (NTS) (see Fig.5).
Also, Abl amplified fragment should not activate the Cas12a system, since it's amplified only with one reverse primer that does not contain the PAM sequence.
Moreover, fragments amplified with the forward primer four (F4) should not activate the Cas neither, since the PAM sequence is shifted from a base.
Figure 5-a. Alignment of the target region (NTS) in the ordered DNA sequence of Bcr-Abl (top) and the crRNA guide sequence (spacer) used for detection (bottom). The guide sequence is the same as the NTS, there is no mismatches.
Figure 5-b. Alignment of the CRISPR-Cas12a target region on the Bcr ordered fragment (NTS, sequence on top) and the crRNA guide sequence (spacer) used for detection (bottom). The guide sequence is not the same as the NTS (4 different                         base pairs), i.e. four mismatches with the corresponding target strand (TS) are expected.
Figure 5-c. Alignment of the CRISPR-Cas12a target region on the Abl ordered fragment (NTS, sequence on top) and the crRNA guide sequence (spacer) used for detection (bottom). The guide sequence is not the same as the NTS (3 different                         base pairs), i.e. three mismatches with the corresponding target strand (TS) are expected.
PCR amplification of Bcr-Abl junction-Trial 1
In this version, we're performing the assay with one forward primer only (F_primer_Chromo_symu_01), the second reverse primer (R_primer_chromo_symu_02)and different annealing temperatures for the PCR reactions, i.e. three                       duplicates of the two samples (A-A1), in order to select the right temperature to use for the amplification, since it was difficult to estimate the melting temperature of the primers due to the mismatches with the template sequence.
59 °C : 1st duplicate.
64 °C: 2nd duplicate.
69°C: 3rd duplicate.
Amounts (1ul)
A (1pM)
A1 (10fM)
A (10pM)
A1 (10fM)
A (1 pM)
A1 (10 fM)
(-) PCR control
Master Mix (7.5 x)
Final concentrations
Nuclease-free water
28.5
28.5
28.5
28.5
28.5
28.5
28.5
213.75
-
5X Phusion HF buffer
10
10
10
10
10
10
10
75
1X
10 mM dNTPs
1
1
1
1
1
1
1
7.5
200 µM
From Master mix
39.5
39.5
39.5
39.5
39.5
39.5
39.5
296.25
-
10 µM F1-10
2.5
2.5
2.5
2.5
2.5
2.5
2.5
-
0.5 µM
10 µM R2-10
2.5
2.5
2.5
2.5
2.5
2.5
2.5
-
0.5 µM
100 fM Template (BA-4)
-
5
-
5
-
5
-
-
10 fM
10 pM Template (BA-2)
5
-
5
-
5
-
-
-
1 pM
Nuclease-free water
-
-
-
-
-
-
5
-
-
Phusion DNA Polymerase
0.5
0.5
0.5
0.5
0.5
0.5
0.5
-
1.0 units/50 µl PCR
Total volume
50
50
50
50
50
50
50
-
-
The thermocycler was programmed as described below. IMPORTANT: three blocs were used for proceding with the samples simultaneously, since we have three sets with different forward primers.
Temperature
Time
98 °C
30 sec (Initial Denaturation)
98 °C
10 sec (denaturation)
35 cycles
59 °C : 1st duplicate.        64 °C: 2nd duplicate.                         69°C:                           3rd duplicate
30 sec (primer annealing)
35 cycles
72 °C
30 sec (extension)
35 cycles
72 °C
10 min (Final Extension)
4 °C
As controls, we prepared a PCR negative control sample in which we only add water and no template (D), and two others (N and N1) that contain the initial concentrations of template without amplification (in order to be sure that we                       can't detect these without amplification). Gel was load as following:
Lane
1
2
3
4
5
6
7
8
Sample
Generuler 1kb Plus DNA Ladder
A (59°C)
A1 (59°C)
A (64°C)
A1 (64°C)
A (69°C)
A1 (69°C)
D
µl
5
12
12
12
12
12
12
12
Results
Analysis/Discussion
Overall, an amplification was obtained for all annealing temperatures tested, and we were able to detect the expected amplicon of 105 bp. Moreover, we couldn't observe any band in the negative control's lane, which means                       that our samples were free from any DNA contamination. We can however argue that we got a better amplification when the annealing step was performed under 69°C.
We can state that the most efficient amplification was achieved when we annealed our primers (forward F1 and reverse R2) at 69°C. Since it was the one found using the Neb calculator (that doesn't take into account mismatches though),                       we decided to trust it for our further amplifications.
References
1.Li, S., Cheng, Q., Wang, J., Zhao, G. & Wang, J. CRISPR-Cas12a-assisted nucleic acid detection. 5,1–8
Thursday, 06/09/18
PCR amplification of Bcr-Abl junction-Trial 2.
In this version, we performed the assay with our three different forward primers (TTTN PAM sequence introduced with two mutations near a thymine, different lengths, GC content, annealing temperatures) and R_primer_chromo_symu_02 (R2)                       reverse primer, to see which one works the best (in terms of amplification). The annealing temperatures for each set of primers (one reverse primer used), was calculated using NEB Tm calculator. As controls, we prepared a PCR negative                       control sample in which we only add water and no template, and two others that contain the initial concentrations of template without amplification (in order to be sure that we can't detect these without amplification).
Read notes of 05/09 for approach and hypothesis for the general "PCR amplification of Bcr-Abl junction" assay.
PCR samples:
Amounts (ul)
A (1pM template)
A1 (10 pM template)
B (1 pM template)
B1 (10 fM template)
C (1 pM template)
C1 (10 fM template)
D: (-) PCR control
Master Mix
Final concentrations
Nuclease-free water
28.5
28.5
28.5
28.5
28.5
28.5
28.5
270.75
-
5X Phusion HF buffer
10
10
10
10
10
10
10
95
1X
10 mM dNTPs
1
1
1
1
1
1
1
9.5
200 µM
From Master mix
39.5
39.5
39.5
39.5
39.5
39.5
39.5
375.25
-
10 µM F1-10
2.5
2.5
-
-
-
-
2.5
-
0.5 µM
10 µM F3-10
-
-
2.5
2.5
-
-
-
-
0.5 µM
10 µM F4-10
-
-
-
-
2.5
2.5
-
-
0.5 µM
10 µM R2-10
2.5
2.5
2.5
2.5
2.5
2.5
2.5
-
0.5 µM
Nuclease-free water
-
-
-
-
-
-
5
-
-
100 fM Bcr-Abl Template (BA-4)
-
5
-
5
-
5
-
-
10 fM
10 pM Bcr-Abl Template (BA-2)
5
-
5
-
5
-
-
-
1 pM
Phusion DNA Polymerase
0.5
0.5
0.5
0.5
0.5
0.5
0.5
-
1.0 units/50 µl PCR
Total volume
50
50
50
50
50
50
50
-
-
The thermocycler was programmed as described below. IMPORTANT: three blocs were used for proceding with the samples simultaneously, since we have three sets with different forward primers.
98 °C
30 sec (Initial Denaturation)
98 °C
10 sec (denaturation)
35 cycles
A-A1-D (F1/R2): 69°C.                            B-B1 (F3/R2): 72°C.                               C-C1                             (F4/R2): 72°C.
30 sec (primer annealing)
35 cycles
72 °C
30 sec (extension)
35 cycles
72 °C
10 min (Final Extension)
4 °C
Results:
Analysis
We could observe a unique band of roughly 100 bp for each tested sample, which would in principle say that the amplification worked well, since we're expecting amplicons of 105 bp (F1 and F4 primers) and 109 bp (F3). However, we got                       the same band in our negative control sample (water, no DNA template), in which we sould not observe anything, and this suggests a possible DNA contamination of our PCR reaction samples. We could also notice the absence of any band                       for the 1 pM/10 fm (without amplification) samples, which is coherent with the fact that we can't detect such low concentrations of DNA without amplification.
Discussion
Can we really conclude as to the success of this pcr knowing that our negative control did not work ? Anyway, we're going to do the same PCR again in order to be sure.
Friday, 07/09
PCR amplification of Bcr-Abl junction-Trial 3.
Same approach as in the previous trial. This time, we prepared samples containing the Bcr and Abl genes as templates (1 pM), to compare the efficiency of amplification and assess the linear amplification results (see "hypothesis", point two). We amplified them using F_primer_chromo_symu_01 (F1) primer and R_primer_chromo_symu_02 (R2).
Read notes of 05/09 for approach and hypothesis for the general "PCR amplification of Bcr-Abl junction" assay.
Results
Analysis
Same results as trial 2 unfortunately, we got again this band in our negative control sample. Also, we can observe the exact same band in the samples containing the Abl and Bcr coding sequences, respectively. This suggests that we                       got an exponential amplification for these rather than linear one. This was not expected, since only one of the primers used is complementary to each sequence (respectively F1 with Bcr and R2 with Abl). Also, the amplification of                       these two sequences should've resulted in amplicons with respectively 239 and 279 bp (c.f. the hypothesis of the experiment, listed on the 05/09), while in this scheme it seems that the same template was amplified...
Discussion
Since it's the second time that we obtain this strange result for the negative control, and the band is the exact replicate of the one obtained in all the other samples, this excludes the hypothesis of a random DNA contamination. We                       can state that either we put (again) one of the DNA templates (Bcr-Abl junction most probably) in the control by mistake, or one of the 10 µM primer solution is contaminated with DNA template. As for the Bcr and Abl samples, this                       same band doesn't make sense at all...
Sunday, 09/09/18
Another trial with the same hypothesis as the previous one (amplification done with reverse primer two i.e. R2 for all samples). Still trying to figure out the origin of the band we got in our negative control sample, and to obtain                       the right amplification for each DNA template (i.e. three different amplicons).
Read notes of 05/09 for approach and hypothesis for the general "PCR amplification of Bcr-Abl junction" assay.
Results
Analysis
We still have our strange contamination for the negative control sample. Same pattern of bands.
Discussion
It looks like we're obtaining the same amplicon in each sample, while two of them contain different DNA templates. This could suggest a contamination in one of our 10 µM primer stocks that we've been using so far, perhaps the forward                       primer one (F1), used to amplify different DNA templates, or the reverse primer two (R2), the only reverse primer used so far.
Monday, 10/09/18
Cas12a detection assay-Trial 1
We performed our first Cas12a assay for detecting the junction, using the PCR products obtained during "PCR amplification of Bcr-Abl junction-Trial 4", following the hypothesis elaborated on the 05/09.
Results
Cas12a assay using different amplicons (as activators) obtained from the amplification of the Bcr-Abl junction with different forward primers (and R2 as reverse primer).
Cas12a assay performed with F1-R2 amplicons (activators) obtained from both the junction and Bcr-Abl genes.
Analysis
Several observations can be made:
We can clearly see that the primer pair F1-R2 gives the best results (Bcr-Abl junction amplification), followed by F3-R2 and F4-R2. Indeed, the F1-R2 amplicon obtained from the amplification of 1 pM junction part significantly activates                         the Cas12a enzyme and generates a signal ~1.5 times as strong as the one obtained using Bcr or Abl separate genes. Regarding Bcr, this fits more or less with our hypothesis, however Abl gene shouldn't activate the Cas at all.
Several amplicons of the junction part (F1-R2, F3-R2 both for a concentration of 10 fM and 1 pM before amplification, see first graph) yield almost the same signal than Bcr and Abl normal genes. This was not expected, due to the                         point mutations present in the target strand recognized by the crRNA, and that should have resulted in much less activation of the Cas12a system, with no activation with Abl.
Activation with F4-R2 amplicon as a target, which shouldn’t be recognized by our crRNA.
Discussion
Desired results obtained just with the F1-R2 primers amplification of the junction: ~1.5 folds decrease in signal when using Bcr or Abl as activators in comparison. We have to figure out why our Cas12a enzyme is being activated with                       F4 amplicons and Abl..
Tuesday, 11/09/18
PCR for negative controls without any DNA
We wanted to understand why our negative control showed some amplification. The aim is to see whether some of our samples are contaminated. For this, we created six PCR (-) control samples, in which we only added water and different                       combination of primers (see below). Afterwards, we ran a gel with two extra samples: a lane containing Phusion enzyme alone, and one with loading dye, in order to verify that these samples are indeed free of contaminations.
Read notes of 05/09 for approach and hypothesis for the general "PCR amplification of Bcr-Abl junction" assay.
Results
Analysis
It is hard to really know what do the bands actually represent but one of our suspicion is that our reverse primer 2 stock (R_primer_chromo_symu_02) is contaminated by one of our template (probably the Bcr-Abl junction since we got                       the desired band of ~ 105 bp) and that the bands when the reverse primer 1 was used somehow represent the primers (hetero-dimers ?)
Discussion
Since we performed the dilutions of our primer stocks again (resuspended at 100 µM) in order to obtain the 10 µM primer solutions that we can use for our PCR, we are convinced that the reverse primer 2 (R2) stock was somehow contaminated                       with Bcr-Abl junction.
Wednesday, 12/09/18
PCR amplification of Bcr-Abl junction-Trial 5
Our hypothesis was that our reverse primer 2 stock solution (R_primer_chromo_symu_02-R2) is contaminated. In this trial we tried amplification with our reverse primer 1 (R_primer_chromo_symu_01-R1) to see if it would differ from the                       amplicon produced using the contaminated one (R2). We used 1 pM DNA template for each sample (either Bcr-Abl, Bcr or Abl alone), and two negative controls (no DNA) containing different combinations of primers.
Read notes of 05/09 for approach and hypothesis for the general "PCR amplification of Bcr-Abl junction" assay.
Results
Analysis
The amplification seem to have worked using R1 (~145 bp).
Discussion
We can quite confidently say that our R2 reverse primer was contaminated. We'll perform a Cas12a assay with these PCR products.
Cas12a detection assay-Trial 2
We performed another Cas12a assay, using the PCR products obtained from the PCR above ("PCR amplification of Bcr-Abl junction-Trial 5").
Read notes of 05/09 for approach and hypothesis for the general "Cas12a detection assays" set of experiments.
Reults:
Analysis/Discussion
Obviously this trial did not work. What could have happened:
Problem with Cas12a : crRNA might have been degraded
Amplification of the wrong template
Since the positive control (DNase I sample) is working fine, we can be confident that the DNaseAlert wasn't degraded.
Thursday, 13/09/18
PCR amplification of Bcr-Abl junction-Trial 6
We performed one more time the PCR on our templates (1 pM concentration before amplification), this time using only forward primer 1 (F1) and one of the two reverse primers (R1 or R2). Negative controls included F1 primer and either                       one of the reverse primers, amplified with water as template. As for the gel, we also ran different samples containing F1 forward primer, R1 and R2 reverse primers (diluted at 50 µM, without amplification), as well as two samples                       containing annealed primers F1-R1 and F2-R2 (50 µM, without amplification), in order to figure out whether the bands we were obtaining so far were due to dimers.
Read notes of 05/09 for approach and hypothesis for the general "PCR amplification of Bcr-Abl junction" assay.
Results
Analysis
The junction (Bcr-Abl sample) was not amplified by the reverse primer 1, but amplified successfully with the second reverse primer. Is it some manipulation error ?
Bcr seems to be amplified with both combinations of primers (F1-R1 and F1-R2), However, the size still do not make sense (size between 75 and 200 bp rather than the 239 expected). Abl was not amplified with F1-R1, compared to the                         discrete band obtained when F1-R2 were used. According to our hypothesis (c.f. wednesday 05/09), we shouldn't be able to observe the PCR products of these two samples on a gel due to the lack of sensitivity of the SYBR Safe nucleic                         acid stain, given that the concentration of the amplification products is expected to be really low because of the linear amplification (rather than exponential) that is occuring in these samples (~35 pM since we started from 1                         pM of DNA concentration).. The bands obtained for Abl could be due to contamination (i.e. amplification with Bcr-Abl contaminated primer R2); however, for Bcr F1-R1, it doesn't really make sense.
As for the control sample in which we added water and F1-R2 primers, the amplification resulted in an amplicon of the same size as we would expect if we had amplified the Bcr-Abl junction, thus suggesting that our reverse primer                         2 is indeed contaminated. Also, adding this primer without amplifying it resulted in a completely different band, suggesting that the amplification is essential in order to obtain a quantity such that we can observe Bcr-Abl fragments                         on gel.
Discussion
the only deduction that can be drawn from this gel is that our concentrated 100 µM stock of the reverse primer 2 (R2) is contaminated. From now on, we decided to work with the first reverse primer (R1).
Friday, 14/09
Cas12a detection assay-Trial 3
We performed another Cas12a assay, using the PCR products obtained from the PCR above ("PCR amplification of Bcr-Abl junction-Trial 6").
Read notes of 05/09 for approach and hypothesis for the general "Cas12a detection assays" set of experiments.
Results
Analysis
Bcr-Abl F1-R1 amplicon (amplified part of the junction containing the PAM sequence) is up to four times less activating the Cas12a enzyme compared to the other gene templates used (Bcr and Abl amplicons, respectively), which is the                         contrary of what we're expecting to see. Moreover, we do not expect Abl to be an activator for the Cas12a (no PAM sequence).
Bcr-Abl F1-R2 induces a much higher signal but still way less than Bcr and Abl separate genes.
Both PCR negative control samples ("Water_F1-R1/F1-R2") are surprinsigly activating the Cas12a enzyme, resulting in a fluorescent signal which is clearly unexpected, since these samples contain no DNA template before their amplification...
Bcr-Abl without amplification (no PAM sequence) sample (dashed purple line) did not activate the enzyme, which confirms that the absence of the PAM sequence is clearly preventing the Cas12a from cleaving the dsDNA template (Bcr-Abl                         fragment), thus preventing the trans-cleavage of DNaseAlert reporter ssDNA fragments.
Discussion
Overall, this trial is a disaster. We did not reach the corresponding levels of activation that we're expecting for the different dsDNA activators (c.f. the hypothesis in "Cas12a detection assays", drawn on the 05/09). More strangely                       is that our dsDNA free samples are somehow activating the Cas12a enzyme. Is our R1 primer also contaminated ? or is it F1 unstead ?? How about Abl being an activator ? Anyway, we're going to perform another assay to figure out.
Saturday, 15/09/18
PCR amplification of Bcr-Abl junction-Trial 7
We performed one more time the PCR on our templates (1 pM concentration before amplification), this time using either forward primer 1 (F1) or forward primer 4 (F4), and reverse primer 1 (R1). Negative control included F1 primer and                       R1 reverse primer in combination with water.
Here, we also wanted to test the specificity of the PAM-Cas12a interaction. For this, we amplified both our Bcr-Abl junction and Bcr gene with forward primer 4 (F4), with the hypothesis being that the resulting amplified products shouldn't                       be recognized by Cas12a, since the crRNA used is not complementary to the sequence right after the PAM sequence added by this primer. Also for this trial, 30 amplification cycles were ran instead of 35 used until now for the PCR reaction with PHUSION polymerase (Thermo Fisher scientific).
Read notes of 05/09 for approach and hypothesis for the general "PCR amplification of Bcr-Abl junction" assay.
Results
Sunday, 16/09/18
Cas12a detection assay-Trial 4
We performed another Cas12a assay, using the PCR products obtained from the experiment "PCR amplification of Bcr-Abl junction-Trial 7" done during the previous day. Read notes of 05/09 for approach and hypothesis for the general "Cas12a                       detection assays" set of experiments
Results
Analysis
Roughly the same observations as in the previous Cas12a detection assay.
Activation of Cas12a with the Bcr-Abl F4-R1 amplicon. This is indeed in disagreement with our hypothesis.
High signal obtained for the PCR negative control sample ("Water_F1-R1", dashed blue line).
Discussion
We're starting to think that the dsDNA contamination that we're facing so far may be due to primer F1. The presence of some Bcr-Abl template DNA in this primer would explain why we're getting such an activation with our PCR products                       containing Bcr or Abl, as well as the negative control result. We decided to order F1 (F_primer_Chromo_symu_01) as well as R2 (R_primer_Chromo_symu_02) again from Integrated DNA Technologies (IDT).
Thursday, 20/09/18
Cas12a detection assay-Trial 5
Aim:
For this trial we wanted to check wether the primers somehow activated our cas12a by doing hetero dimers. In this way we could eliminate the possibility of dimers and prioritize the hypothesis of contamination.
The protocol we followed is : FQ reporter assay V.5 -For primers [20.09.18]
Read notes of 05/09 for approach and hypothesis for the general "Cas12a detection assays" set of experiments
Results:
Discussion:
We can see that the annealed primers as well as the individual primers do not activate cas12a. The samples were not amplified which mean that our primers can be contaminated.
Friday, 21/09/18
PCR amplification of Bcr-Abl junction with Plasma - Trial 1
Aim:
To see whether our usual PCR worked correctly when done in plasma.
Results:
Wednesday, 26/09/18
PCR amplification of Bcr-Abl junction (20 cycles)-Trial 8
Aim:
Amplification of the Bcr with the newly received F1 and R2 primers as our old one seemed to have been contaminated.
Results:
Discussion:
Seems to work! Only the Bcr-Abl has been amplified as should be. But contamination (on the comb? in the ladder?)
Thursday, 27/09/18
Aim:
To try the gel again with the previous pcr products (20 cycles) as the last gel was contaminated
Results:
Discussion:
The amplification seems to have been successful! Only the bcr-abl sequence seem to have been amplify. The bands are quite faint so we will reproduce the same PCR with more cycles.
Cas12a detection assay-Trial 5 (from PCR products trial 9)
Discussion
The results are not as expected: the negative control N of the cas12a assay (Cas12a with crRNA and DNaseAlert) is highly fluorescent. The sample in which we would like to detect the highest level of fluorescence, Bcr-Abl with F1-R2,                       is quite low compared to samples which should barely activate the enzyme (Bcr F1-R2).
Friday, 28/09/18
PCR amplification of Bcr-Abl junction (30 cycles)-Trial 10
Results:
Discussion:
As we can see the results differ a lot from the previous gel (20 cycles). There seem to be some contamination althought the R1 and F1 primers are new. To make sure it is contamination that happened between this gel and the last we                       will reproduce those same pcr.
Saturday, 29/09/18
PCR amplification of Bcr-Abl junction (20 cycles)-Trial 11
Aim:
To remake the same pcr as previous to check for contamination
Results:
Discussion:
The gel is the same as the first 20 cycles gel. This suggest that there was no contamination in the F1, R2 primers or nuclease free water between the previous 20 cycles gel and the 30 cycles gel. We want to see the 30 cycles amplification                       again.
Cas12a detection assay-Trial 6 (from PCR products trial 11)
Discussion:
The negative control is not as high as it previously was but still too high. We think there might be some problem with the new Cas12a protein we received. The Bcr-Abl F1-R2 is the sample which activates cas the most which is what we                       want! But the Abl F1-R2 sample shows the highest fluorescence which is not a good thing since abl sequence is not only not complementary to the crRNA but also doesn't contain the PAM sequence.
Sunday, 30/09/18
PCR amplification of Bcr-Abl junction (30 cycles)-Trial 12
Aim:
To remake the 30 cycles pcr in order to see how sensitive and specific our amplifcation is.
Results:
Discussion:
Again there are some bands in the negative controls as well as in the bcr and abl lanes. Next time we will either put a lower concentration to start with or/and put background DNA (from salmon sperm DNA). Hopefully the negative controls                       will be empty again.
Cas12a detection assay-Trial 7 (from PCR products trial 12)
Discussion:
The negative control is once again way too high. We will try to do it in plasma next time as we think that maybe having background DNA will make the amplification more specific.
Monday, 01/10/18
PCR amplification of Bcr-Abl junction with Plasma - Trial 13
We wanted to try amplificaiton in Plasma again as we believe that having some sorts of DNA background could help our amplification specificity. We chose to make a 30 cycles reactions to have clear bands.
Results:
Cas12a detection assay in plasma-Trial 1 (from plasma PCR products trial 13)
Results:
Analysis:
The activation is overall very small. In this assay we used the remaining of an older cas12a protein. Maybe the concentration of the protein wasn't high enough. Another possibility is that the amplification was not successful.
Wednesday, 03/10/18
PCR amplification of Bcr-Abl junction with Plasma (30 cycles)- Trial 14
As last cas12a assay revealed no activation we decided to try again with the same samples. We removed the primer F4 as we don't find it necessary.
The bands look very faint for a 30cycles amplification.
Cas12a detection assay in plasma-Trial 2 (from plasma PCR products trial 14)
Hooray! This assay is sucessful!! Bcr-Abl is activated as it should be. The other samples show small to no activation at all.
Thursday, 04/10/18
PCR amplification of Bcr-Abl junction with Plasma - Trial 15
Here we wanted to try different kind of Bcr-Abl concentration to see how the cas12a activation would change.
Cas12a detection assay in plasma-Trial 3 (from plasma PCR products trial 15)
Friday, 05/10/18
PCR amplification of Bcr-Abl junction with Plasma - Trial 16 (25 cycles)
We decided to change our approach and to look at the difference in activation of a sample with healthy cfDNA (the background: Bcr and Abl) compared to a sample which contains both the background and the mutated cfDNA (Bcr-Abl). This                       is indeed closer to the reality as a patient with cancer will have both ctDNA and cfDNA corresponding to the healthy cells.
There is very faint bands on the gel which is maybe due to the fact that we only did around 25 cycles.
Cas12a detection assay in plasma-Trial 4 (from plasma PCR products trial 16)
Sadly the results are not as expected.. The background which contains no Bcr-Abl activated the Cas12a targeting Bcr-Abl more than the sample that actually has template.
Tuesday, 09/10/18
PCR amplification of Bcr-Abl junction with Plasma - Trial 17 (30 cycles)
Today we wanted to reproduce the same kind of experience as yesterday but with two different concentrations as we've previously seen that template concentrations can sometimes affect cas12a.
Cas12a detection assay in plasma-Trial 5 (from plasma PCR products trial 17)
Samples 1 :
b1 1pM
BAb 10fM
Samples 2:
b2 100fM
BAb2 1fM
Again, a very high negative control from plasma. It is not surprising as there was a band in the gel above for the negative control. Except this we can see that the samples containing the targeted Bcr-Abl junction is in both case more                       activated than the ones without (the background). This is a good thing. Next time we'll try the same again with possibly different concentrations and hopefully we'll get a sucessful negative control.
Wednesday 10/10/18
PCR amplification of Bcr-Abl junction with Plasma - Trial 18 (30 cycles)
We tried to do the amplification one last time under the same conditions as last time. In addition we tried another concentration.
We can see that the samples containing the Bcr-Abl segment are brighter and larger than the one without. The problem is that even the negative control has a band: the same band which are present in every background only sample.
Cas12a detection assay in plasma-Trial 6(from plasma PCR products trial 18)
Notebook (24/09-11/10) Point mutation detection
Notebook (24/09-30/09)
Monday, 24/09/18
Point mutation discrimination proof of concept via detection of BRAF V600E point mutated DNA fragment
Aim:
Here, we want to establish that we can destinguish between single point mutations using our optimized Cas12a/crRNA complex. For that we chose as a template a particular sequence that has been highly correlated with melanomagenesis1,                           i.e. the BRAF V600E mutation that constitute 90% of all the BRAF V600 mutations in melanoma patients (50% were found to carry these activating BRAF mutations that result in an unregulated BRAF serine/threonine protein kinase),                           as well as the original coding sequence for the normal BRAF protein (Fig.1). We choose to order 276 bp sequences that harbors the mutated (resp. non-mutated) nucleotide (target 1, yellow triangle), as well as another point                           mutation introduced synthetically (target 2, pink triangle), for a sequence-independent detection of any point mutation proof of concept (since our BRAF V600E point mutation happened to be found right after a PAM sequence).
For that, we want to couple Cas12a specificity (c.f. Cas12a detection assay for point mutations experiments ) with PCR sensitivity (c.f. PCR amplification of BRAF V600E mutated DNA fragment) in order to detect                               really small quantities of DNA templates.
PCR amplification of BRAF V600E mutated DNA fragment
We ordered the sequences as gBlocks from Integrated DNA Technology (IDT). Primer oligos were also ordered from IDT (Fig.1-a):
Primers (from left to right, Fig.1-a below):
F1_BRAF_Mutated (F1)
R1_BRAF_Mutated (R1)
F2_BRAF_Mutated (F2)
R2_BRAF_Mutated (R2)
Fig.1-a. Overview of the sequence containing the BRAF V600E point mutation and the primers. The particular mutation is pointed with a pink triangle, while the introduced mutation is indicated in yellow. Primers from top to                               bottom: F1_BRAF_Mutated (F1), R1_BRAF_Mutated (R1), F2_BRAF_Mutated (F2), and R2_BRAF_Mutated (R2)
Fig.1-b. Overview of the non mutated sequence of the BRAF protein. Indicated respectively with a triangle (from left to right) the non mutated nucleotides. Target regions shown in different colors.
Resuspension of the gBlocks/Oligos was done as following, according to the Oligonucleotides/gBlocks Resuspension and Storage (IDT) protocol
Oligos
Delivered amount (nmol)
Desired concentration
Amount of water to add (ul)
BRAF V600E
0.002935
100 nM
29.4
BRAF original
0.002935
100 nM
29.4
F1_BRAF_Mutated
24.4
100 µM
244
F2_BRAF_Mutated
18.4
100 µM
184
R1_BRAF_Mutated
30.1
100 µM
301
R2_BRAF_Mutated
20.4
100 µM
204
Template DNA solutions (dilutions) were prepared as below:
BRAF V600E mutated fragment (BM-)
BRAF original (BO-)
Dilution label
Final concentration
Amount of template to take (ul)
Nuclease-free water to add (ul)
Dilution fold
BO-1
1 nM
2 (from resuspended BRAF original 100 nM stock)
198
1:100
BO-2
10 pM
2 (from BO-1 dilution)
198
1:100
BO-3
1 pM
10 (from BO-2 dilution)
90
1:10
BO-4
100 fM
10 (from BO-3 dilution)
90
1:10
BO-5
10 fM
10 (from BO-4 dilution)
90
1:10
BO-6
1 fM
10 (from BO-5 dilution)
90
1:10
Dilution label
Final concentration
Amount of template to take (ul)
Nuclease free water to add (ul)
Dilution fold
BM-1
1 nM
2 (from resuspended BRAF mutated 100 nM stock)
198
1:100
BM-2
10 pM
2 (from BM-1)
198
1:100
BM-3
1 pM
10 (from BM-2)
90
1:10
BM-4
100 fM
10 (from BM-3)
90
1:10
BM-5
10 fM
10 (from BM-4)
90
1:10
BM-6
1 fM
10 (from BM-5)
90
1:10
10 µM dilutions of the primers:
Primers (100 uM stock solution)
Volume to take (ul)
Amount of water to add (ul)
Total volume (ul)
Tube label
F1_BRAF_Mutated
10
90
100
F1
F2_BRAF_Mutated
10
90
100
F2
R1_BRAF_Mutated
10
90
100
R1
R2_BRAF_Mutated
10
90
100
R2
Experiments were done following the PCR for phusion protocol . For experiments in plasma, we followed the PCR amplification in plasma protocol .
Hypothesis
Sequences amplification should result in an amplicon of 64 bp when using F1-R1 primers, and 98 bp if amplified with F2-R2.
Cas12a detection assay for point mutations
In this series of experiments, we want to be able to specifically distinguish the mutated sequence from the non-mutated one (previously amplified by PCR), using our optimized Cas12a/crRNA system. As told several times, recognition                             of the activator by the crRNA unleashes indiscriminate single-stranded DNA (ssDNA) cleavage activity by Cas12a that will completely degrades our ssDNA reporter molecules (DNaseAlert, IDT), which results in a higher fluorescent                             signal. For the CRISPR RNA design (crRNA), we chose to use shorter guide sequences (17 bp rather than 20), based on the work done by Li S, Cheng Q, Wang J et al.2, where they proved that point mutations within a large region                             (1st–16th bases) resulted in more than 2-fold difference in fluorescence signals for both 16-nt and 17-nt crRNA guide sequences. We ordered four crRNAs in total, one complementary to the region of each point-mutated/original                             nucleotide (target 1 and 2, see Fig.1 above, M stand for mutated fragment while O is for original. ex: M1 is the crRNA complementary to target 1 of the BRAF V600E mutated fragment):
crRNA BRAF V600E-target 1 (M1)
crRNA BRAF V600E-target 2 (M2)
crRNA BRAF Original-target 1 (O1)
crRNA BRAF Original-target 2 (O2)
We followed the Fluorophore-Quencher reporter Cas12a assay . crRNAs oligo fragments were annealed with a T7 primer and transcribed then purified following respectively the crRNA Transcription using T7 RNA Polymerase                               (Promega) . and crRNA purification (ZYMO Research RNA Clean & Concentrator™-5 Kit) protocols
Hypothesis
Expected fluorescence signal for each combination of crRNAs and template . O1 and M1 refer both to crRNAs targeting the first region (target 1) on both mutated/non mutated amplified (F1-R1) DNA templates (resp. O2 and M2).
Amplified target/crRNA
crRNA BRAF V600E (M1)
crRNA BRAF V600E (M2)
crRNA BRAF Original (01)
crRNA BRAF Original (02)
BRAF
High fluorescence expected.
fluorescence
Low fluorescence
Nearly no fluorescence
BRAF
Nearly no fluorescence
High fluorescence expected.
fluorescence
Low fluorescence
BRAF
Low fluorescence
Nearly no fluorescence
High fluorescence expected.
Nearly no fluorescence
BRAF
Nearly no fluorescence
Low fluorescence
Nearly no fluorescence
High fluorescence expected.
PCR amplification of BRAF V600E mutated DNA fragment-Trial 1
We amplified both regions (target 1 and 2) on the two templates we have (with BRAF V600E and BRAF original DNA fragment). Samples E and F were used as controls (water with either set of the primers). PCR was done as following:
Amount (µl)
A
B
C
D
E (-) PCR control
F (-) PCR control
Master Mix (7x)
Final concentration
Nuclease-free water
28.5
28.5
28.5
28.5
28.5
28.5
199.5
-
5X Phusion HF buffer
10
10
10
10
10
10
70
1X
10 mM dNTPs
1
1
1
1
1
1
7
200 µM
From Master mix
39.5
39.5
39.5
39.5
39.5
39.5
276.5
-
10 µM F1
2.5
-
2.5
-
2.5
-
-
0.5 µM
10 µM F2
-
2.5
-
2.5
-
2.5
-
0.5 µM
10 µM R1
2.5
-
2.5
-
2.5
-
-
0.5 µM
10 µM R2
-
2.5
-
2.5
-
2.5
-
0.5 µM
Nuclease-free water
-
-
-
-
5
5
-
-
BO-2 (10 pM BRAF-original template)
5
5
-
-
-
-
-
1 pM
BM-2 (10 pM BRAF-mutated template)
-
-
5
5
-
-
-
1 pM
Phusion DNA Polymerase
0.5
0.5
0.5
0.5
0.5
0.5
-
1.0 units/50 µl PCR
Total volume
50
50
50
50
50
50
-
-
The thermocycler should be programmed as described below. IMPORTANT: three blocs should be used for proceding with the samples simultaneously, since we have three sets with different forward primers (i.e. different annealing                                 temperatures).
Temperature
Time
Cycles
98 °C
30 sec (Initial Denaturation)
98 °C
10 sec (denaturation)
30
A-C-E (F1-R1):                                       65°C.                                                   B-D-F                                       (F2-R2): 62°C.
30 sec (primer annealing)
30
72 °C
30 sec (extension)
30
72 °C
10 min (Final Extension)
4 °C
A gel electrophoresis was then performed to check for the results (Agarose gel electrophoresis protocol)
Results
Analysis
Correct amplification. Negatives worked.
Cas12a detection assay for point mutations-Trial 1
We performed a Cas12a assay using previously amplified fragments:
M1-O(1): Original BRAF fragment (O) where we amplified only target 1 (1) and performed Cas12a assay with M1 crRNA.
M1-M(1): Mutated BRAF V600E fragment where we amplified only target 1 (M(1)) and performed Cas12a assay with M1 crRNA.
M1 (-): Negative control for the assay with no template and M1 crRNA. (resp. M2 (-)).
M1-W(1): water (W) amplified with primers F1 and R1 used for target 1, before performing detection assay using M1 crRNA (resp. M2-W(2)).
Component (µl)
M1-0(1)
M1-M(1)
M1(-)
M2-0(2)
M2-M(2)
M2 (-)
M1-W(1)
M2-W(2)
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
1 µM crRNA (M1 or M2)
4.5
4.5
4.5
4.5
4.5
4.5
4.5
4.5
water (nuclease free)
30
30
36
30
30
36
30
30
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
PCR products (6 µl)
C
A
-
D
B
-
E
F
DNase I
-
-
-
-
-
-
-
-
Final volume
60
60
60
60
60
60
60
60
Results
Notebook (01/10-07/010)
MONDAY, 01/10
PCR amplification of BRAF V600E mutated DNA fragment-Trial 2
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment.
We amplified both regions (target 1 and 2) on the two templates we have (with BRAF V600E and BRAF original DNA fragment). Samples E and F were used as controls (water with either set of the primers). PCR was done as on the 24/09.
Cas12a detection assay for point mutations-Trial 2
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment. We performed a Cas12a assay using previously amplified fragments (except the negative controls)
M1-O(1): Original BRAF fragment (O) where we amplified only target 1 (1) and performed Cas12a assay with M1 crRNA.
M1-M(1): Mutated BRAF V600E fragment where we amplified only target 1 (M(1)) and performed Cas12a assay with M1 crRNA.
M1 (-): Negative control for the assay with no template and M1 crRNA. (resp. M2 (-)).
Component (µl)
M1-M(1)
M1-0(1)
M1 (-)
M2-M(2)
M2-0(2)
M2 (-)
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3.75
1 µM crRNA (4.5 µl)
M1
M1
M1
M2
M2
M2
water (nuclease free)
35
35
36
35
35
36
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
PCR products (6 µl)
C
A
-
D
B
-
DNase I
-
-
-
-
-
-
Final volume
60
60
60
60
60
60
Results
Analysis
The assay didn't work since we can see that the original fragment is activating the Cas assembled with the mutated crRNA (which is complementary to the mutated fragment).
discussion
We're going to repeat the assay with the new Cas12a enzyme that we've just receive.
TUESDAY, 02/10
PCR amplification of BRAF V600E mutated DNA fragment-Trial 3
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment. We amplified both regions (target 1 and 2) on the two templates we have (with BRAF V600E and BRAF original DNA fragment). Samples E and F were used as       controls (water with either set of the primers). PCR was done as on monday 01/10 (Trial 2).
Results
Analysis/Discussion
It seems that the amplification didn't work...
Cas12a detection assay for point mutations-Trial 3
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment. We performed a Cas12a assay again using amplified fragments obtained from PCR amplification of BRAF V600E mutated DNA fragment-Trial 3 obtained on monday       (negative controls used this time). Also we used the new LbCas12a (NEB) that we received.
M1-O(1): Original BRAF fragment (O) where we amplified only target 1 (1) and performed Cas12a assay with M1 crRNA(resp.M2-O(2)).
M1-M(1): Mutated BRAF V600E fragment where we amplified only target 1 (M(1)) and performed Cas12a assay with M1 crRNA (resp.M2-M(2).
M1 (-): Negative control for the assay with no template and M1 crRNA. (resp. M2 (-)).
M1-W(1): water (W) amplified with primers F1 and R1 used for target 1, before performing detection assay using M1 crRNA (resp. M2-W(2)).
Component
M1-M(1)
M1-O(1)
M1(-)
M2-M(2)
M2-O(2)
M2 (-)
M1-W(1)
M2-W(2)
W
(+) control
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
-
-
1 µM crRNA (4.5 µl)
M1
M1
M1
M2
M2
M2
M1
M2
-
-
water (nuclease free)
35
35
36
35
35
36
35
35
53.4
41.4
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
PCR products (6 µl)
C
A
-
D
B
-
E
F
F
-
DNase I
-
-
-
-
-
-
-
-
-
2.4
Final volume
60
60
60
60
60
60
60
60
60
60
Results
Analysis
We got very exciting results actually. As can be seen, we got activation where we wanted (mutated region 1 with crRNA M1, mutated region 2 with crRNA M2) and negative signal for non-mutated fragments.
Discussion
Now that we proved that we can distinguish between a mutated strand and a non mutated one via the specificity of the RNA guided DNA binding of our Cas12a assay, we're ready to work on plasma for more realistic conditions.
WEDNESDAY, 03/10
PCR amplification of BRAF V600E mutated DNA fragment in plasma-Trial 1
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment.
The aim is to Prepare our plasma samples in which we dilute the activator (mutated or non mutated fragment) and amplify them directly in plasma. We       will amplify only target 2 (BRAF V600E mutation position), in different samples with different concentrations of template (samples used in red, then diluted 100 folds in the final PCR reaction mix).
Template DNA solutions (BM- and BO-) were prepared as on the 24/09. Pre-treatment of Plasma Samples
1. Dilute 40 µl of plasma in 120 µl of PBS (1:4 final dilution of plasma in PBS) in a microtube (PLASMA+PBS).
2. DNA dilutions in plasma were made as following:
Component
A
B
C
D
E
F
PLASMA+PBS
18
18
18
18
18
18
BM-1
2
-
-
-
-
-
BM-3
-
2
-
-
-
-
BM-6
-
-
2
-
-
-
BO-1
-
-
-
2
-
-
BO-3
-
-
-
-
2
-
BO-6
-
-
-
-
-
2
Total
20
20
20
20
20
20
1. The diluted serum samples were heated for 3 min at 95°C then cooled rapidly on ice for 3 to 5 min.
2. We prepared the following PCR reaction mix :
Components
A (Mutated 10 pM)
B (Mutated 10 fM)
C(Mutated 10 aM)
D (Original 10 pM)
E (Original 10 fM)
F(Original 10 aM)
G (-) PCR control
Master Mix (8x)
Final concentration)
Water
28.5
28.5
28.5
28.5
28.5
28.5
28.5
228
-
5X Phusion HF buffer
10
10
10
10
10
10
10
80
1X
10 mM dNTPs
1
1
1
1
1
1
1
8
200 µM
10 µM F2
2.5
2.5
2.5
2.5
2.5
2.5
2.5
20
10 µM R2
2.5
2.5
2.5
2.5
2.5
2.5
2.5
20
From Master mix
44.5
44.5
44.5
44.5
44.5
44.5
44.5
356
-
PLASMA+PBS
-
-
-
-
-
-
5
-
-
BM-1 (P)
5
-
-
-
-
-
-
-
BM-3 (P)
-
5
-
-
-
-
-
-
BM-6 (P)
-
-
5
-
-
-
-
BO-1 (P)
-
-
-
5
-
-
-
-
BO-3 (P)
-
-
-
-
5
-
-
-
BO-6 (P)
-
-
-
-
-
5
-
Phusion polymerase
0.5
0.5
0.5
0.5
0.5
0.5
0.5
-
1.0 units/50 µl PCR
Total volume
50
50
50
50
50
50
50
-
-
Thermocycling was done with 35 cycles, and primers were annealed at 62°C (F2-R2)
Results (right part of the gel only)
Primers F2-R2 were actually used to amplify target 2. Negative control contained treated plasma without addition of activator
Analysis/discussion
We obtained the correct size of amplicon, with the desired expression intensity (10 aM < 10 fM < 10 pM).
Limit of detection of PCR obtained for ~10 aM and below
Negative control (plasma without activator) yielded the corrected amplicon, which suggests that it contained somehow our desired sequence prior to amplification, or could be that we contaminated         it with activator...
Cas12a detection assay for point mutations in plasma-Trial 1
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment. We used as DNA templates the PCR reaction mixes from "PCR amplification of BRAF V600E mutated DNA fragment in                                           plasma-Trial 1"
Now that we have demonstrated that we can recognize a mutated strand of an original strand (still to be confirmed in plasma), we seek to establish a correlation between the concentration of       activator (mutated or not) and the degree of activation of Cas12a (fluorescent signal), and perhaps have an idea bout the limit of detection...
FA: CRISPR-Cas12a assay used with M2 (resp O2 for FA') crRNA on 10 pM mutated sample amplified (target 2) by PCR.
FB: CRISPR-Cas12a assay used with M2 (resp O2 for FB') crRNA on 10 fM mutated sample amplified (target 2) by PCR.
FD: CRISPR-Cas12a assay used with M2(resp. O2 for FD') crRNA on 10 pM of original template amplified (target 2) by PCR.
FE: CRISPR-Cas12a assay used with M2(resp. O2 for FD') crRNA on 10 fM of original template amplified (target 2) by PCR.
FG: CRISPR-Cas12a assay used with M2 (resp. O2 for FG') crRNA on negative control of PCR (plasma+PBS w/o DNA) by PCR.
N1/N2: (-) control for Cas experiment (w/o activator) with either M2 or O2 crRNA.
Expectations:
No or little activation (no activator)
Low or no fluorescent signal
High fluorescent signal
Components
FA (10 pM)
FA' (10pM)
FB (10 fM)
FB' (10 fM)
FD(10 pM)
FD'(10 pM)
FE (10fM)
FE' (10fM)
FG (-)
FG'(-)
N!
N2
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
1 µM crRNA (4.5 µl)
M2
O2
M2
O2
M2
O2
M2
O2
M2
O2
O2
O2
water (nuclease free)
30
30
30
30
30
30
30
30
30
30
36
36
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
PCR products (6 µl)
A
A
B
B
D
D
E
E
G
G
-
-
DNase I
-
-
-
-
-
-
-
-
-
-
-
-
Final volume
60
60
60
60
60
60
60
60
60
60
60
60
Results
Analysis
10 pM mutated sample wasn't activated with O2 crRNA, which proves that mismatch greatly reduces the activity of Cas12a. 10 pM original fragment targeted with O2 crRNA resulted in a good fluorescent signal as well, in contrary to when it was targeted with       M2 crRNA. 10 fM mutated fragment targeted with M2 crRNA exhibits good activation as well, and did not when targeted with O2 crRNA.
Same with 10 pM original sample: activation when targeted with O2 crRNA, no activation with M2.
However, the difference of fluorescent signal wasn't that high when the original fragment was tested at 10 fM in the solution.
Discussion
We need to investigate the strange reason behind the lack of specificity of Cas12a when tested on the 10 fM original fragment.
THURSDAY, 04/10
PCR amplification of BRAF V600E mutated DNA fragment in plasma-Trial2
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment.
The aim is to Prepare our plasma samples in which we dilute the activator (mutated or non mutated fragment) and amplify them directly in plasma. We       will amplify only target 2 (BRAF V600E mutation position), in different samples with different concentrations of template (samples used in red, then diluted 100 folds in the final PCR reaction mix).
Pre-treatment of Plasma Samples
1. We diluted We diluted 10µl of plasma in 30µl (4 µl 10X PBS + 26 µl water) of 1X PBS. This is the Mastermix (MM).
2. We then took 8µl of MM that we added to each of the following samples:
Component
B0b(Mutated+bg)
BRb(Original +bg)
b0 (bg for BRb)
bM(bg for B0b
N
MM
8
8
8
8
8
Nuclease-free water
-
-
1
1
2
BM-4 (100fM)
1
-
-
-
-
BM-2 (10pM)
-
1
1
-
-
BO-2 (10pM)
1
-
-
1
-
BO-4 (100 fM)
-
1
-
-
-
Total
2
2
2
2
2
3. The diluted serum samples were heated for 3 min at 95℃ then cooled rapidly on ice for 3 to 5 min.
Amplification
Components
A (1fM mutated, 0.1 pM bg)
B (1 fM original, 0.1 pM bg)
C (0.1 pM original)
D (0.1 pM mutated)
N (-) PCR control
Master Mix (6x)
Final concentration
Water
28.5
28.5
28.5
28.5
28.5
171
-
5X Phusion HF buffer
10
10
10
10
10
60
1X
10 mM dNTPs
1
1
1
1
1
6
200 µM
10 µM F2
2.5
2.5
2.5
2.5
2.5
15
0.5 µM
10 µM R2
2.5
2.5
2.5
2.5
2.5
15
0.5 µM
From Master mix
44.5
44.5
44.5
44.5
44.5
-
-
BOb
5
-
-
-
-
-
BRb
-
5
-
-
-
-
bO
-
-
5
-
-
-
bM
-
-
-
5
-
-
N
-
-
-
-
5
-
Phusion DNA Polymerase
0.5
0.5
0.5
0.5
0.5
-
1.0 units/50 µl PCR
Total volume
50
50
50
50
50
-
-
Thermocycling was done at 25 cycles, 62°C annealing.
FRIDAY, 05/10
PCR amplification of BRAF V600E mutated DNA fragment in plasma-Trial 3
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment.
The aim is to Prepare our plasma samples in which we dilute the activator (mutated or non mutated fragment) and amplify them directly in plasma.         We will amplify only target 2 (BRAF V600E mutation position), in different samples with different concentrations of template (samples used in red, then diluted 100 folds in the final PCR reaction mix).
Same experiment as yesterday.
SATURDAY, 06/10
Cas12a detection assay for point mutations in plasma-Trial 2
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment.
Cas12a experiment done with PCR fragments obtained yesterday (05/10).
Here, we wanted to test whether we could still detect small concentrations of the mutated fragment out of the background (bg)         (i.e. non-mutated BRAF fragment and other DNA present in plasma). We tested with 1 fM of the fragment of interest + 0.1 pM of background (bg) which would be the other DNA that we do not want to detect. Also, in order to prove that our detection system         is specific enough, we did other samples where we put for instance the original DNA and performed the assay with the crRNA complementary to the mutated strand, hoping that the mismatch between the crRNA and activator won't activate (or at least not         that much) the Cas12a enzyme. We also prepared control samples containing the treating plasma (with PBS) that do not contain any activator, so we're expecting to have no fluorescent signal there.
Expectations
No or little activation (no activator)
Low or no fluorescent signal
High fluorescent signal
Component
A (1jM mutated, 0.1 pM bg)
B(1fM original, 0.1 M bg)
D (0.1 pM original)
C(0.1 pM mutated)
Plasma + PBS w/o activator
Plasma + PBS w/o activator
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3.75
1 µM crRNA (4.5 µl)
M2
O2
M2
O2
O2
M2
water (nuclease free)
30
30
30
30
30
30
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
PCR products (6 µl)
A
B
D
C
N
N
DNase I
-
-
-
-
-
-
Final volume
60
60
60
60
60
60
Results
Positive control contains DNase I as endonuclease unstead of the Cas12a/crRNA complex. bg stands for background, which is original template in excess if mutated fragment used, mutated if original used.
Analysis/Discussion
This trial is a complete disaster. The background is much more activated than the template we're targeting. We're going to do the dilutions again for next time, to ensure we get rid of any trace of contamination.
Notebook (08/10-14/010)
MONDAY, 08/10
PCR amplification of BRAF V600E mutated DNA fragment in plasma-Trial 4
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment.
The aim is to Prepare our plasma samples in which we dilute the activator (mutated or non mutated fragment) and amplify them directly in plasma. We amplified target 2 (BRAF V600E mutation position), in different samples with different concentrations of template (samples used in red, then diluted 100 folds in the final PCR reaction mix). IMPORTANT: This time we reduced the primers' concentration in the reaction mix (250 nM unstead of 500 nM) in order to see if we could increase the specificity of the reaction.
Template DNA solutions (100X) were prepared again (c.f 24/09), as well as primer dilutions.
Pre-treatement of Plasma Samples
1. Here we wanted to have a final concentration of 1 part plasma + 4 part PBS (5x dilutions). For that we have first made a master-mix of 3 part PBS : 1 part Plasma and added later the                           final part of PBS with the activator in it.
2. We diluted 10 µl of plasma in 30µl (4 µl 10X PBS + 26 µl water) of 1X PBS. This is the Mastermix (MM).
3. We then took 8µl of MM that we added to each of the following samples:
Component
B0b (Mutated+bg)
BRb (Original + bg)
b0 (bg for BRb)
bM (bg for B0b)
N
MM
8
8
8
8
8
Nuclease-free water
-
-
1
1
2
BM-4 (100fM)
1
-
-
-
-
BM-2 (10pM)
-
1
1
-
-
BO-2 (10pM)
1
-
-
1
-
BO-4 (100 fM)
-
1
-
-
-
Total
10
10
10
10
10
4. The diluted serum samples were heated for 3 min at 95°C/b> then cooled rapidly on ice for 3 to 5 min.
Amplification
Component
A (1fM mutated, 0.1 pM bg)
B (1 fM original, 0.1 pM bg)
C(0.1 pM mutated )
D (0.1 pM original
N (-) PCR control
Master Mix (5.6 x)
Final concentration
Water
31
31
31
31
31
173.6
-
5X Phusion HF buffer
10
10
10
10
10
56
1X
10 mM dNTPs
1
1
1
1
1
5.6
200 µM
10 µM F2
1.25
1.25
1.25
1.25
1.25
7
0.25 µM
10 µM R2
1.25
1.25
1.25
1.25
1.25
7
0.25 µM
From Master mix
44.5
44.5
44.5
44.5
44.5
-
-
BOb
5
-
-
-
-
-
BRb
-
5
-
-
-
-
bO
-
-
5
-
-
-
bM
-
-
-
5
-
-
N
-
-
-
-
5
-
Phusion DNA Polymerase
0.5
0.5
0.5
0.5
0.5
-
1.0 units/50 µl PCR
Total volume
50
50
50
50
50
-
-
Thermocycling was done at 30 cycles, 61°C annealing.
PCR amplification of BRAF V600E mutated DNA fragment in plasma-Trial 5
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment.
We want to titrate in plasma the concentration of both mutated and non-mutated BRAF sequence using the Cas assay. We will amplify                             only target 2 (BRAF V600E mutation position), starting from different concentrations of template (samples used in red, then diluted 100 folds in the final PCR reaction mix).
IMPORTANT: This time we reduced the primers' concentration in the reaction mix (250 nM unstead of 500 nM) in order to see if we could increase the specificity of the reaction.
1. We prepared two other dilutions.
BRAF mutated (BM-)
BRAF original (BO-)
BO-5
10 fM
10 (from BO-4)
90
1:10
BO-6
1 fM
10 (from BO-5)
90
1:10
BM-5
10 fM
10 (from BM-4)
90
1:10
BM-6
1 fM
10 (from BM-5)
90
1:10
Pre-treatment of Plasma Samples
1. We diluted 17,5 µl of plasma in 52,5 µl (5,25 µl 10X PBS + 47,25 µl water) of 1X PBS. This is the Mastermix (MM).
2. We then took 8 µl of MM that we added to each of the following samples (5X dilution of plasma in the                             end):
Component
BM-1 (P)
BM-3 (P)
BM-6 (P)
BO-1 (P)
BO-3 (P)
BO-6 (P)
N(P)
MM
8
8
8
8
8
8
8
Nuclease-free water
-
-
-
-
-
-
2
BM-1
2
-
-
-
-
-
-
BM-3
-
2
-
-
-
-
-
BM-6
-
-
2
-
-
-
-
BO-1
-
-
-
2
-
-
-
BO-3
-
-
-
-
2
-
-
BO-6
-
-
-
-
-
2
-
Total
10
10
10
10
10
10
10
1. The diluted serum samples were heated for 3 min at 95°C then cooled rapidly on ice for 3 to 5 min.
2. We prepared the following PCR reaction mix:
Components
A (Mutated 10 pM)
B (Mutated 10 fM)
C (Mutated 10 aM)
D (Original 10 pM)
E (Original 10 fM)
F (Original 10 aM)
G (-) PCR control
Master Mix (7.7x)
Final concentration
Water
33.5
33.5
33.5
33.5
33.5
33.5
33.5
257.95
-
5X Phusion HF buffer
10
10
10
10
10
10
10
77
1X
10 mM dNTPs
1
1
1
1
1
1
1
7.7
200 µM
10 µM F2
1.25
1.25
1.25
1.25
1.25
1.25
1.25
9.63
0.25 µM
10 µM R2
1.25
1.25
1.25
1.25
1.25
1.25
1.25
9.63
0.25 µM
From Master mix
47
47
47
47
47
47
47
-
-
N (P)
-
-
-
-
-
-
2.5
-
-
BM-1 (P)
2.5
-
-
-
-
-
-
-
BM-3 (P)
-
2.5
-
-
-
-
-
-
BM-6 (P)
-
-
2.5
-
-
-
-
-
BO-1 (P)
-
-
-
2.5
-
-
-
-
BO-3 (P)
-
-
-
-
2.5
-
-
-
BO-6 (P)
-
-
-
-
-
2.5
-
Phusion polymerase
0.5
0.5
0.5
0.5
0.5
0.5
0.5
-
1.0 units/50 µl PCR
Total volume
50
50
50
50
50
50
50
-
-
Thermocycling was done at 30 cycles, 61°C annealing.
Results for trial 4 and 5
We loaded both samples of this experiment and the one before (trial 4) on the same gel, following the Agarose gel electrophoresis protocol:
Lane
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
Sample
Generuler 1kb plus DNA ladder
A (1 fM mutated, 0.1 pM bg)
B (1 fM original, 0.1 pM bg)
C (0.1 pM mutated)
D (0.1 pM original)
N1 (Plasma + PBS w/o activator)
N2 (water w/o plasma or activator)
-
Generuler 1kb plus DNA ladder
A (Mutated 10 pM)
B (Mutated 10 fM)
C (Mutated 10 aM)
D (Original 10 pM)
E (Original 10 fM)
F (Original 10 aM)
G (plasma + PBS w/o activator)
Generuler 1kb DNA ladder
loading dye + DNA (µl)
5
12
12
12
12
12
12
-
5
12
12
12
12
12
12
12
5
Analysis
The expected amplicon's length (98 bp) was roughly reached for most of the samples.
However, some samples seemed to have migrated farther.
Exactly same band for the negative (plasma + PBS w/o activator), which could suggest                             that our plasma originally contains the same sequence that we amplified among others.. This is not a contamination since our PCR negative control (water and pcr reagents, w/o plasma) is clear.
Discussion
Could it be that our plasma contains the exactly same sequence that we're targeting ??
TUESDAY, 09/10
Cas12a detection assay for point mutations in plasma-Trial 3
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment. We used as DNA templates the PCR reaction mixes from "PCR amplification of BRAF V600E mutated DNA fragment                                                       in plasma-Trial 4"
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment. We used as DNA templates the PCR reaction mixes from "PCR amplification of BRAF V600E mutated DNA fragment in plasma-Trial 4
Expectations:
No or little activation (no activator)
Low or no fluorescent signal
High fluorescent signal
Component
A(1 fM mutated, 0.1 pM bg)
B (1 fM original, 0.1 pM bg)
D (0.1 pM original )
C (0.1 pM mutated )
N1 (Plasma + PBS w/o activator )
N2 (Plasma + PBS w/o activator)
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3.75
1 µM crRNA (4.5 µl)
M2
O2
M2
O2
O2
M2
water (nuclease free)
30
30
30
30
30
30
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
PCR products (6 µl)
A
B
D
C
N
N
DNase I
-
-
-
-
-
-
Final volume
60
60
60
60
60
60
Cas12a detection assay for point mutations in plasma-Trial 4
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment. We used as DNA templates the PCR reaction mixes from "PCR amplification of BRAF V600E mutated DNA fragment in plasma-Trial 5 Now that we have demonstrated that we can recognize a mutated strand of an original strand (still to be confirmed in plasma), we seek to establish a correlation between the concentration of activator (mutated or not) and the degree of activation of Cas12a                               (fluorescent signal), and perhaps have an idea bout the limit of detection...
Expectations:
No or little activation (no activator)
Low or no fluorescent signal
High fluorescent signal
Experiement set-up (please refer to the standard protocol for more informations):
Components
FA (Mutated 10 pM)
FA' (10 pM Mutated)
FB (Mutated 10 fM)
FB' (Mutated 10 fM)
FD(Original 10 pM)
FD' (Original 10 pM)
FE (Original 10 fM)
FE' (Original 10 fM)
FG((-) PCR control)
FG' ((-) PCR control)
N1: (-) control for Cas12a
N2: (-) control for Cas12a
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
1 µM crRNA (4.5 µl)
MT
OA
MT
OA
MT
OA
MT
OA
MT
OA
OA
MT
water (nuclease free)
30
30
30
30
30
30
30
30
30
30
36
36
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
PCR products (6 µl)
A
A
B
B
D
D
E
E
G
G
-
-
DNase I
-
-
-
-
-
-
-
-
-
-
-
-
Final volume
60
60
60
60
60
60
60
60
60
60
60
60
Results/Analysis
Trial 3
Here we showed that we could detect a very small concentration of mutated fragment among a healthy fragment present in excess, directly in plasma. As can be seen in the second graph, the Cas12a/M2-crRNA complex (crRNA complementary                               to the mutated piece of target 2, mutated fragment) was barely activated with the background here (0.1 pM original fragment), resulting in more than 2-fold difference in fluorescent signal between the sample containing the target                               sequence (full line) and the control (dashed line).
However, no difference was observed when we targeted the original fragment out of the mutated (1st graph), which was not expected.
Slight activation of the sample containing                               plasma.
Trial 4
- Mutated fragment
- Original fragment
Again, the specificity of the RNA guiding DNA binding was demonstrated (1st graph). Nearly no activation was observed when we targeted the original fragment with the crRNA designed to recognize the mutated part, and the activation                                 was excellent (more than 4-folds difference in fluorescent signal) when the mutated fragment (no mismatch with crRNA) was used at activator.
We could barely distinguish a difference in signal between the sample containing                                 the original fragment targeted with its complementary crRNA, and the one where we used the mutated crRNA. Overall, the activation is really low.
Discussion
Good results. The non-receptivity of the Cas12a/crRNA (O2) towards the original fragment is yet to be figured. A problem with the crRNA maybe ?
add the rest of the stuff for titration-Trial 4, + negatives, what's N ????                                 (09/10/18 Trial 16).
-> Overall, the experimental results matched the expectations for the mutated fragment, but not for the original one.
-> Plasma activated slightly the Cas12a, which is in accordance with the gel obtained on the 08/10                                 which shows that our plasma contained the same sequence in very small amount (among other cfDNA fragments) that was amplified by PCR to a detectable concentration by gel.
WEDNESDAY, 10/10
PCR amplification of BRAF V600E mutated DNA fragment in plasma-Trial 6
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment.
We want to titrate in plasma the concentration of both mutated and non-mutated BRAF sequence using the Cas assay. In this                                   trial, we amplified only target 1 (synthetically inserted mutation position), starting from different concentrations of template (samples used in red, then diluted 100 folds in the final PCR reaction mix).
IMPORTANT: Still using decreased primers' concentrations in the reaction mix (250 nM unstead of 500 nM) in order to see if we could increase the specificity of the reaction.
1. We prepared mutated template DNA dilutions again:
A
B
C
D
E
BM-1
1 nM
2 (from resuspended BRAF mutated 100 nM stock)
198
1:100
BM-2
10 pM
2 (from BM-1)
198
1:100
BM-3
1 pM
10 (from BM-2)
90
1:10
BM-4
100 fM
10 (from BM-3)
90
1:10
BM-5
10 fM
10 (from BM-4)
90
1:10
BM-6
1 fM
10 (from BM-5)
90
1:10
BM-7
100 aM
10 (from BM-6)
90
1:10
Pre-treatment of Plasma Samples
1. We diluted 16 µl of plasma in 48 µl (6.4 µl 10X PBS + 41.6 µl water) of 1X PBS. This is the Mastermix (MM).
2. We then took 8 µl of MM that we added to each of the following samples (5X dilution of plasma in the                                   end):
COmponent
Stock (P)
BM-1 (P)
BM-2 (P)
BM-4 (P)
BM-6 (P)
BM-7 (P)
N (P)
MM
8
8
8
8
8
8
8
Nuclease-free water
-
-
-
-
-
-
2
Mutated stock (100 nM)
2
-
-
-
-
-
-
BM-1 (1 nM)
-
2
-
-
-
-
-
BM-2 (10 pM)
-
-
2
-
-
-
-
BM-4 (100 fM)
-
-
-
2
-
-
-
BM-6 (1 fM)
-
-
-
-
2
-
-
BM-7 (100 aM)
2
-
-
-
-
2
-
Total
10
10
10
10
10
10
10
1. The diluted serum samples were heated for 3 min at 95°C then cooled rapidly on ice for 3 to 5 min.
2. We prepared the following PCR reaction mix:
Components
A (Mutated 1 nM)
B (Mutated 10 pM)
C (Mutated 100 fM)
D (Mutated 1 fM)
E (Mutated 10 aM)
F (Mutated 1 aM)
G (-) PCR control
Master Mix (7.7 x)
Final concentration
Water
33.5
33.5
33.5
33.5
33.5
33.5
33.5
257.95
-
5X Phusion HF buffer
10
10
10
10
10
10
10
77
1X
10 mM dNTPs
1
1
1
1
1
1
1
7.7
200 µM
10 µM F1
1.25
1.25
1.25
1.25
1.25
1.25
1.25
9.63
0.25 µM
10 µM R1
1.25
1.25
1.25
1.25
1.25
1.25
1.25
9.63
0.25 µM
From Master mix
47
47
47
47
47
47
47
-
-
N (P)
-
-
-
-
-
-
2.5
-
-
Stock (P)
2.5
-
-
-
-
-
-
-
BM-1 (P)
-
2.5
-
-
-
-
-
-
BM-2 (P)
-
-
2.5
-
-
-
-
-
BM-4 (P)
-
-
-
2.5
-
-
-
-
BM-6 (P)
-
-
-
-
2.5
-
-
-
BM-7 (P)
-
-
-
-
-
2.5
-
Phusion polymerase
0.5
0.5
0.5
0.5
0.5
0.5
0.5
-
1.0 units/50 µl PCR
Total volume
50
50
50
50
50
50
50
-
-
Thermocycling was done at 30 cycles, 61°C annealing.
Results
PCR amplification of BRAF V600E mutated DNA fragment in plasma-Trial 7
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment.
We want to titrate in plasma the concentration of both mutated and non-mutated BRAF sequence using the Cas assay. In this                                   trial, we amplified only target 1 (synthetically inserted mutation position), starting from different concentrations of template (samples used in red, then diluted 100 folds in the final PCR reaction mix). IMPORTANT: Still using decreased primers' concentrations in the reaction mix (250 nM unstead of 500 nM).
Template DNA solutions (100X) used from yesterday preparations for mutated fragment (BM-), and done prepared again for original one (BO-). 10 µM dilutions of the primers were also prepared again.
BO-1
1 nM
2 (from resuspended BRAF original 100 nM stock)
198
1:100
BO-1.2
100 pM
10 (from BO-1)
90
1:10
BO-2
10 pM
2 (from BO-1)
198
1:100
BO-3
1 pM
10 (from BO-2)
90
1:10
BO-4
100 fM
10 (from BO-3)
90
1:10
BO-5
10 fM
10 (from BO-4)
90
1:10
BO-6
1 fM
10 (from BO-5)
90
1:10
Pre-treatment of Plasma Samples
1. We diluted 16 µl of plasma in 48 µl (6.4 µl 10X PBS + 41.6 µl water) of 1X PBS. This is the Mastermix (MM).
2. We then took 8 µl of MM that we added to each of the following samples (5X dilution of plasma in                                     the end):
Component
BMb 1
BMb 2
BMb 3
bM 1
bM 2
bM 3
N
MM
8
8
8
8
8
8
8
Nuclease-free water
-
-
-
1
1
1
2
BM-4 (100fM)
1
-
-
-
-
-
-
BO-2 (10pM)
1
-
-
1
-
-
-
BM-3 (1 pM)
-
1
-
-
-
BO-1.2 (100 pM)
-
1
-
-
1
-
-
BM-2 (10pM)
-
-
1
-
-
-
-
BO-1 (1 nM)
-
-
1
-
-
1
-
Total
10
10
10
10
10
10
10
The diluted serum samples were heated for 3 min at 95℃ then cooled rapidly on ice for 3 to 5 min.
Amplification
Component
A (1fM mutated, 0.1 pM bg)
B (10fM mutated, 1 pM bg)
C( 100fM mutated, 10 pM bg)
D(0.1 pM bg)
E (1 pM bg)
F(10 pM bg)
N (-) PCR control
Master Mix (7.6x)
Final concentration
Water
31
31
31
31
31
31
31
235.6
-
5X Phusion HF buffer
10
10
10
10
10
10
10
76
1X
10 mM dNTPs
1
1
1
1
1
1
1
7.6
200 µM
10 µM F1
1.25
1.25
1.25
1.25
1.25
1.25
1.25
9.5
0.25 µM
10 µM R1
1.25
1.25
1.25
1.25
1.25
1.25
1.25
9.5
0.25 µM
From Master mix
44.5
44.5
44.5
44.5
44.5
44.5
44.5
-
-
BMb 1
5
-
-
-
-
-
-
-
BMb 2
-
5
-
-
-
-
-
-
BMb 3
-
-
5
-
-
-
-
-
bM 1
-
-
-
5
-
-
-
-
bM 2
-
-
-
-
5
-
-
-
bM 3
-
-
-
-
-
5
-
-
N
-
-
-
-
-
-
5
-
Phusion DNA Polymerase
0.5
0.5
0.5
0.5
0.5
0.5
0.5
-
1.0 units/50 µl PCR
Total volume
50
50
50
50
50
50
50
-
-
PCR done with 30 cycles, 61°C annealing.
Results
Analysis/discussion
Expected amplicon obtained. Clear negative control.
Cas12a detection assay for point mutations in plasma-Trial 5
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment. We used as DNA templates the PCR reaction mixes from "PCR amplification of BRAF V600E mutated DNA fragment in                                                     plasma-Trial 6"
Now that we have demonstrated that we can recognize a mutated strand of an original strand in plasma, we seek to establish a correlation between the concentration of mutated activator                                     and the degree of activation of Cas12a (fluorescent signal). For that, we tried different concentrations and loaded our Cas12a protein with M1 crRNA.
Experiment set-up (please refer to the standard protocol for more informations):
Components
FA(Mutated 1 nM)
FB (Mutated 10 pM)
FC (Mutated 100 fM)
FD (Mutatate 1 fM)
FE (Mutated 10 aM)
FF (Mutated 1 aM)
FG ( (-) PCR control )
N1
N2
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
1 µM crRNA (4.5 µl)
M1
M1
M1
M1
M1
M1
M1
M1
M1
water (nuclease free)
30
30
30
30
30
30
30
36
30
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
PCR products (6 µl)
A
B
C
D
E
F
G
-
BCR-ABL
DNase I
-
-
-
-
-
-
-
-
-
Final volume
60
60
60
60
60
60
60
60
60
Results
Analysis
Overall, we can see that that the fluorescent signal drops with the concentration of target DNA. This was demonstrated to be true except for 100 fM and 1 fM concentrations. Also it seems that we loose sensitivity below the                                     fM range (we could not distinguish between 10 and 1 aM, barely between 1 and 100 fM), since the level of these signals is positioned at the threshold of the negative control.
Discussion
Maybe we inverted between the sample containing 1 fM and the one containing 100 fM..
It could be that we've reached somehow the threshold for concentration below which we can't distinguish a random DNA sequence from                                     the targeted one.
Cas12a detection assay for point mutations in plasma-Trial 6
Please reffer to notes taken on the 24/09 for general aim and hypothesis of this experiment. We used as DNA templates the PCR reaction mixes from "PCR amplification of BRAF V600E mutated DNA fragment in                                                     plasma-Trial 7"
Here, we wanted to test whether we could still detect small concentrations of the mutated fragment out of the background (bg) (i.e. non-mutated BRAF fragment and other DNA present                                     in plasma). We tested with 1 fM, 10 fM and 100 fM of the fragment of interest + 0.1 pM, 1 pM, 10 pM of background (bg) which would be the original DNA that we do not want to detect. We also prepared control samples containing                                     the treating plasma (with PBS) that do not contain any activator, so we're expecting to have no (or very little) fluorescent signal there.
Expectations:
No or little activation (no activator)
Low or no fluorescent signal
High fluorescent signal
Component
A (1 fM mutated, 0.1 pM original)
B (10 fM mutated, 1pM original )
C (100 fM mutated, 10 pM original )
D (0.1 pM original )
E (1 pM original)
F (10 pM original )
N1 (Plasma +PBS w/o activator )
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3.75
3.75
1 µM crRNA (4.5 µl)
M1
M1
M1
M1
M1
M1
M1
water (nuclease free)
30
30
30
30
30
30
30
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
PCR products (6 µl)
A
B
C
D
E
F
N1
DNase I
-
-
-
-
-
-
-
Final volume
60
60
60
60
60
60
60
Results
analysis
The fact that we got activation only in the samples containing the sequence of interest (mutated one, targeted by our Cas12a/crRNA-M1 system) demonstrated once again the specificity of our optimized detecting method, that                                     can recognize the right sequence even in the presence of 100 times more DNA background that differ only by a single bp.
This specific signal is correlated with the concentration of targeted DNA: We can see that the                                     more we have mutated DNA in our samples, the more we get fluorescence.
Negative control worked as well (lowest signal), and we see that even at the highest concentration of original DNA (bg), the signal only slightly                                     exceeds that of the negative, remaining very low overall.
However, the signal difference between 1 fM and 10 fM is very low, while it falls sharply between 100 fM and 10 fM (10 folds concentration difference for both).
Discussion
We could demonstrate the specificity of our detection tool in the presence of two DNA fragments that differ only by one base pair. However, it seems that at a certain concentration of target DNA, the difference in fluorescent                                     signal is no longer very significant...
iGEM2018@groupes.epfl.ch
IGEM EPFL 2018
Twitter Facebook
