<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Notebook-2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Notebook-2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM EPFL 2018</TITLE></P><DIV class="container"><A class="navbar-brand mr-lg-5" href="https://2018.igem.org/Team:EPFL"><STRONG>iGEM</STRONG> EPFL 2018</A><DIV class="collapse navbar-collapse" id="navbar_main_collapse"><UL class="navbar-nav align-items-lg-center ml-lg-auto"><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:EPFL">HOME</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Description" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Description</H2><P class="mb-0">Take a closer look at CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Design" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Design</H2><P class="mb-0">Learn more about how we designed CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Software" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Software</H2><P class="mb-0">Explore the Bioinformatic Pipeline implemented in CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Results" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Results</H2><P class="mb-0">Discover the Results of our experiments</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Demonstrate" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Demonstration</H2><P class="mb-0">Demonstration of how our Therapeutic System works</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/InterLab" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Interlab</H2><P class="mb-0">Learn more about our participation in the Interlab Study</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Parts" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Parts</H2><P class="mb-0">Discover the Parts that make up CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Improve" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Improve</H2><P class="mb-0">Explore our Novel and Improvements Parts</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Measurement" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Measurement</H2><P class="mb-0">Learn about measurement approaches we used for Part Characterization</P></DIV></DIV></A></DIV></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Team" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Team Members</H2><P class="mb-0">Learn more about our Team Members</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Collaborations" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Collaborations</H2><P class="mb-0">Discover the interactions we had with other iGEM Teams during CAPOEIRA</P></DIV></DIV></A></DIV></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Human_Practices" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Integrated Human Practices</H2><P class="mb-0">Discover how we integrated Human Practices into the Purpose, Design and Implementation of CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Ethics" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Ethics</H2><P class="mb-0">Explore the ethical considerations of CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Applied_Design" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Applied Design</H2><P class="mb-0">Explore how our therapeutic system attempts to meet real world medical need</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Public_Engagement" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Education and Public Engagement</H2><P class="mb-0">Discover our Education and Public Engagement work with the public and scientific communities</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Meetups" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">European and Eurasian Meetup</H2><P class="mb-0">Learn about our interactions with the iGEM Community during the Meetups</P></DIV></DIV></A></DIV></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:EPFL/Notebook#">Lab</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:EPFL/Safety">Safety</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:EPFL/Entrepreneurship">Entrepreneurship</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Acknowledgements</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Attributions" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Attributions</H2></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Sponsors" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Sponsors</H2></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Partners" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Partners</H2></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Awards" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Awards</H2></DIV></DIV></A></DIV></DIV></LI></UL></DIV></DIV><BUTTON onclick="topFunction()" id="myBtn" title="Go to top">Index</BUTTON><SECTION><DIV class="container pt-lg-lg"><DIV class="row justify-content-center"><DIV class="col-md-12"><H1 class="lh-150 mb-3">Notebook-Vaccine part</H1><P class="lead text mb-0">This page collects the notebooks of the Vaccine part of our project. The experiments are sorted by week</P></DIV></DIV></DIV></SECTION></DIV><MAIN><SECTION class="slice"><DIV class="container"><DIV class="row justify-content-center lead"><DIV class="col-lg-12"><ARTICLE><H6>Friday, 20/07/18</H6><H3><U>INSERT PCR WITH TAQ POLYMERASE</U></H3><P class="lead">Aim: The Aim of this experience is to produce the hexahistidine insert with a PCR using TAQ.</P><P class="lead">Note:The primers we use from now on were diluted 10 times by mixing 10μlof the original primers with 90μl of water</P><OL><LI>PCR mix:</LI><TABLE><TBODY><TR><TH>Green taq master mix (2X)</TH><TH>10μl</TH></TR><TR><TD>HexaHisForwardInsert</TD><TD>1μl</TD></TR><TR><TD>HexaHisReverseInsert</TD><TD>1μl</TD></TR><TR><TD>nuclease free water</TD><TD>8μl</TD></TR><TR><TD>total</TD><TD>20μl</TD></TR></TBODY></TABLE><LI>PCR Cycle</LI><TABLE><TBODY><TR><TH>Green taq master mix (2X)</TH><TH>10μl</TH></TR><TR><TD>HexaHisForwardInsert</TD><TD>1μl</TD></TR><TR><TD>HexaHisReverseInsert</TD><TD>1μl</TD></TR><TR><TD>nuclease free water</TD><TD>8μl</TD></TR><TR><TD>total</TD><TD>20μl</TD></TR></TBODY></TABLE></OL><H3><U>INSERT PCR WITH PHUSION POLYMERASE</U></H3><P class="lead">Aim: The Aim of this experience is to produce the hexahistidine insert with a PCR using Phusion.</P><OL><LI>PCR mix in μl</LI><TABLE><TBODY><TR><TH><SPAN style="font-weight:bold">Forward Primer</SPAN></TH><TH>1μl</TH></TR><TR><TD>Reverse Primer</TD><TD>1μl</TD></TR><TR><TD>dNTPs (10mM)</TD><TD>0.4μl</TD></TR><TR><TD>Phusion Buffer</TD><TD>4μl</TD></TR><TR><TD>Phusion polymerase</TD><TD>0.2μl</TD></TR><TR><TD>H2O</TD><TD>13.4μl</TD></TR><TR><TD>Total</TD><TD>20μl</TD></TR></TBODY></TABLE><LI>PCR cycle</LI><TABLE><TBODY><TR><TH>STEP</TH><TH>Temp</TH><TH>Duration</TH></TR><TR><TD>Initial Denaturation </TD><TD>98</TD><TD>30s</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>10s</TD><TD rowspan="3"><SPAN style="font-weight:bold">30 cycles</SPAN></TD></TR><TR><TD>Annealing</TD><TD>72</TD><TD>30s</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>30s</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>1min</TD></TR><TR><TD>Hold</TD><TD>4</TD></TR></TBODY></TABLE></OL><H3><U>BACKBONE PCR</U></H3><P class="lead">Aim: The Aim here is to amplify the backbone plasmid of encapsulin with a PCR.</P><OL><LI>PCR mix</LI><TABLE><TBODY><TR><TH>Template 1ng/μl</TH><TH>1μl</TH></TR><TR><TD>Forward Primer 10mM</TD><TD>1μl</TD></TR><TR><TD>Reverse Primer 10mM</TD><TD>1μl</TD></TR><TR><TD>dNTPs (10mM)</TD><TD>0.4μl</TD></TR><TR><TD>Phusion Buffer</TD><TD>4μl</TD></TR><TR><TD>Phusion polymerase</TD><TD>0.2μl</TD></TR><TR><TD>H2O</TD><TD>12.6μl</TD></TR><TR><TD>Total</TD><TD>20μl</TD></TR></TBODY></TABLE><LI>PCR cycle</LI><TABLE><TBODY><TR><TH>STEP</TH><TH>Temp</TH><TH>Duration</TH></TR><TR><TD>Initial Denaturation</TD><TD>98</TD><TD>30s</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>8s</TD><TD>30 cycles</TD></TR><TR><TD>Annealing</TD><TD>63</TD><TD>25s</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>165s</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>10min</TD></TR><TR><TD>Hold</TD><TD>4-10</TD></TR></TBODY></TABLE></OL><H4 class="text-mutedf">Results:</H4><H5> PCR gel electrophoresis of the insert PCR with Taq and Phusion</H5><P class="lead">The PCR product were run on a 2% agarose gel, at 100V for 30 minutes.</P><P class="lead">The ladder is the o'generuler low range.</P><P class="lead">The first lane contains the PCR product of the insert with Taq, the 2nd lane contains the PCR product of the insert with Phusion. The 3rd lane is the forward insert primer and the last lane the reverse insert primer. 2μl of the Phusion PCR
                product and the primers were loaded with 2μl of loading dye. 4μl of the Taq PCR product, as well as the ladder, were loaded on the gel.</P><P class="lead">We can see a signal for the inserts that seem to be the right size as they are both next to the 100bp band of the ladder, the expected size for the insert is of 90 bp. It looks like Taq does work better than Phusion because the signal is
                stronger for Taq. It is also noticeable that the insert migrated less far than the primers which is what we expected, the primers were only 54 and 60 bp long. </P><H5>PCR gel electrophoresis of the insert PCR with Taq and Phusion</H5><H4 class="text-muted">Discussion</H4><P class="lead">The PCR product for the backbone was run on a 1% agarose gel for 45 minutes at 100V.</P><P class="lead">2μl of PCR product were loaded with 2μ of thermo scientific 6X loading dye in the two lanes next to the generuler express DNA ladder. There are faint signals at around 100bp, the primers are supposed to be 28 and 30 bp long, So it's hard
                to tell what these can be. There is no trace of the backbone DNA, template, or PCR product.</P><H6>Monday, 23/07/18</H6><P class="lead">Since the last PCR didn't work we will try to change the temperature, by doing a gradient PCR with several temperature values</P><H4>BackBone PCR</H4><OL><LI>PCR mix</LI><TABLE><TBODY><TR><TH>Template 1ng/μl</TH><TH>1μl</TH></TR><TR><TD>Forward Primer 10mM</TD><TD>1.25μl</TD></TR><TR><TD>Reverse Primer 10mM</TD><TD>1.25μl</TD></TR><TR><TD>dNTPs (10mM)</TD><TD>0.5μl</TD></TR><TR><TD>Q5 Buffer</TD><TD>5μl</TD></TR><TR><TD>Q5 polymerase</TD><TD>0.25μl</TD></TR><TR><TD>H2O</TD><TD>15.75μl</TD></TR><TR><TD>Total</TD><TD>25μl</TD></TR></TBODY></TABLE><LI>PCR Cycle</LI><TABLE><TBODY><TR><TH>STEP</TH><TH>Duration</TH><TH>Temp</TH><TH>Temp</TH><TH>Temp</TH><TH>Temp</TH><TH>Temp</TH></TR><TR><TD>Initial Denaturation</TD><TD>30s</TD><TD>98</TD><TD>98</TD><TD>98</TD><TD>98</TD><TD>98</TD></TR><TR><TD>Denaturation</TD><TD>8s</TD><TD>30 cycles</TD><TD>98</TD><TD>98</TD><TD>98</TD><TD>98</TD><TD>98</TD></TR><TR><TD>Annealing</TD><TD>25s</TD><TD>62</TD><TD>64</TD><TD>66</TD><TD>68</TD><TD>70</TD></TR><TR><TD>Extension</TD><TD>165s</TD><TD>72</TD><TD>72</TD><TD>72</TD><TD>72</TD><TD>72</TD></TR><TR><TD>Final Extension</TD><TD>2min</TD><TD>72</TD><TD>72</TD><TD>72</TD><TD>72</TD><TD>72</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD></TR></TBODY></TABLE></OL><H4 class="text-muted">Results</H4><P class="lead">The PCR product for the backbone was run on a 1% agarose gel for 45 minutes at 100V.</P><P class="lead">2.5µl of PCR product were loaded onto the gel with 2.5µl of thermo scientific 6X loading dye. 5µl of the o'gene ruler express ladder were loaded onto the gel.</P><P class="lead">As we can see there are no signal for the 5500 bp backbone we were expecting, which means that the PCR didn't work for any of the temperatures.</P><P class="lead">we can see signals below the 100bp mark of the ladder, these are probably the primers.</P><H6>Tuesday, 24/07</H6><P class="lead">We tried another PCR for the backbone but this time using a different template for the backbone. We had two different template. </P><UL><LI>Template #1, comes from an aliquot of the backbone template at 844 ng/µl. It is labeled with encap sample 1 in the freezer.</LI><LI>Template #2 comes from an aliquot of the backbone template at 350.8 ng/µl. It is labeled with encap sample 2 in the freezer.</LI></UL><P class="lead">Both samples were diluted to a final concentration of 1ng/µl</P><P class="lead">We recreated the following PCR mix, 3 times for each DNA template. for a total of 6 pcr mixes.</P><TABLE><TBODY><TR><TH>Template 1ng/μl</TH><TH>1μl</TH></TR><TR><TD>Forward Primer 10mM</TD><TD>1.25μl</TD></TR><TR><TD>Reverse Primer 10mM</TD><TD>1.25μl</TD></TR><TR><TD>dNTPs (10mM)</TD><TD>0.5μl</TD></TR><TR><TD>Q5 Buffer</TD><TD>5μl</TD></TR><TR><TD>Q5 polymerase</TD><TD>0.25μl</TD></TR><TR><TD>H2O</TD><TD>15.75μl</TD></TR><TR><TD>Total</TD><TD>25μl</TD></TR></TBODY></TABLE><P class="lead">and ran the following gradient cycle for both sets of 3 PCR mix</P><TABLE><TBODY><TR><TH>STEP</TH><TH>Duration</TH><TH>Temp</TH><TH>Temp</TH><TH>Temp</TH></TR><TR><TD>Initial Denaturation</TD><TD>30s</TD><TD>98</TD><TD>98</TD><TD>98</TD></TR><TR><TD>Denaturation</TD><TD>10s</TD><TD>30 cycles</TD><TD>98</TD><TD>98</TD><TD>98</TD></TR><TR><TD>Annealing</TD><TD>30s</TD><TD>64</TD><TD>66</TD><TD>68</TD></TR><TR><TD>Extension</TD><TD>165s</TD><TD>72</TD><TD>72</TD><TD>72</TD></TR><TR><TD>Final Extension</TD><TD>2min</TD><TD>72</TD><TD>72</TD><TD>72</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>4</TD><TD>4</TD></TR></TBODY></TABLE><H4 class="text-muted">Results:</H4><P class="lead">We can see that the PCR worked at least once for each template. Surprisingly, the only time the PCR worked for template#1 was with an annealing temperature of 68°C while the calculated annealing temperature for the primers was 68°C. For
                template#2 the PCR worked for 66°C but we can also see a faint signal for the PCR with an annealing temperature of 68°C. However, it is clear that the PCR worked better with an annealing temperature of 66°C.</P><H4 class="text-muted">Conclusion</H4><P class="lead">If you want to do a PCR amplification of the backbone, use an annealing temperature of 68°C if you use the template diluted from the 844ng/µl aliquot (encap sample1) and use an annealing temperature of 66°C if you want to use the template
                diluted from the 350.8ng/µl aliquot (encap sample2).</P><H2 id="week_3">Notebook week 3 (23/07/18)</H2><H6>FRIDAY, 7/27/2018</H6><H4><U>Inoculating cultures, and miniprep</U></H4><P class="lead"><STRONG>Aim:</STRONG> The aim of this experience is to grow liquid cell cultures from our transformed cells so we can use them to purify plasmid DNA later, and do glycerol stocks</P><P class="lead"><STRONG>Protocol:</STRONG><A href="https://2018.igem.org/Team:EPFL/Protocols">Inoculating cultures</A></P><P class="lead">We grew 2 different liquid cultures with 2 colonies from our transformed bacteria plate, as well as a negative control with only LB medium and ampicilin but no cells. They were incubated overnight at 37°C and shaked at 225 rpm.</P><H4>Results:</H4><FIGURE class="figure"><FIGCAPTION class="mt-3 text-muted">We can see that the 2 cell cultures turned white overnight while the negative control stayed translucid indicating that no contamination occurred and that cells successfully grew overnight.</FIGCAPTION></FIGURE><H6>SATURDAY, 7/28/2018</H6><H4><U>Isolation of plasmid DNA</U></H4><H5>Goals:</H5><UL><LI>To purify plasmid DNA from E.coli cells</LI><LI>To quantify the amount of purified plasmid</LI><LI>To prepare a glycerol stock of bacteria for long term storage</LI></UL><H4>1.Plasmid Purification</H4><P class="lead"><STRONG>Protocol:</STRONG><A href="#">Plasmid miniprep</A></P><P class="lead">We performed 2 different MiniPrep reactions from our 2 different cell cultures, and obtained two different samples of purified plasmid
              </P><P class="lead"><STRONG>Results: Measurements of the purified plasmids with the nanodrop lite yielded the following results</STRONG></P><P class="lead">Measurements were done with the nanodrop lite, using 1μl of nuclease-free water as blank, and 1μl of purified plasmid for the measurement.
              </P><P class="lead">Purified plasmids from cell culture #1:</P><UL><LI>Concentration:14.6 ng/μl</LI><LI>A260/280:1.72</LI></UL><P class="lead">Purified plamids from cell culture #2:</P><UL><LI>Concentration:16.1 ng/μl</LI><LI>A260/280:1.84</LI></UL><P class="lead">The concentration of purified plasmids are very low for both samples. Probably too low for being useful. We should perform another plasmid purification with more cell culture. Here we only purified 600μl from the cell culture. Next time
                we can try purifying a larger volume.</P><H2 id="week_4">Notebook week 4 (30/07/18)</H2><H6>Monday, 30/07/18</H6><P class="lead">Today we did another cell culture, like last time but with 4ml so we would take 500μl to make a glycerol stock of the cells and use the rest to purify the plasmids</P><P class="lead"><STRONG>Protocol:</STRONG><A href="https://2018.igem.org/Team:EPFL/Protocols">Inoculating cultures</A></P><UL><LI>We did 2 liquid cell cultures with 2 different colonies from our plate. As well as a negative control</LI></UL><H6>Tuesday, 31/07/18</H6><H4>1.Preparation of glycerol stocks</H4><P class="lead"><STRONG>Aim:</STRONG>Make glycerol stocks of the cell cultures that grew overnight</P><P class="lead"><STRONG>Protocol:</STRONG><A href="https://2018.igem.org/Team:EPFL/Protocols">Glycerol stock preparation</A></P><P class="lead">We did a glycerol stock of both cell cultures in 2 different tubes, that were placed in the -80°C freezer </P><H4>2.Isolation of plasmid DNA from a larger volume of cell culture</H4><P class="lead"><STRONG>Aim:To try the purification of the plasmid DNA again, but with a larger volume of cell culture</STRONG></P><P class="lead"><STRONG>Protocol</STRONG><A href="#">Plamid miniprep</A></P><P class="lead">This time, we centrifuged 3ml of cell culture, discarded the supernatant and resuspended the cell pellet in 600ul of water to perform the miniprep</P><P class="lead">We finally purified 3ml from each cell cultures that were grown last night</P><P class="lead"><STRONG>Results: Measurements of the purified plasmids with the nanodrop lite yielded the following results</STRONG></P><P class="lead">Measurements were done with the nanodrop lite, using 1μl of nuclease-free water as blank, and 1μl of purified plasmid for the measurement
              </P><H5>Purified plasmids from cell culture #1:</H5><UL><LI>Concentration: 71,1 ng/μl</LI><LI>A260/280: 1.72</LI></UL><H5>Purified plasmids from cell culture #2</H5><UL><LI>Concentration: 167.5 ng/μl</LI><LI>A260/280: 1.51</LI></UL><P class="lead">The concentrations are better than last time, However the A260/280 ratios are a bit low, especially for the second sample. This might lead to problems because of impurities in the samples.</P><H2 id="week_5">Notebook week 5 (06/08/18)</H2><H6>MONDAY, 8/6/2018</H6><H4><U>Competent cell transformation: pMC_sfGFP-BsaI_noT7 plasmid into DH10β
competent E.coli cells</U></H4><P class="lead">pMC_sfGFP-BsaI_noT7 plasmid contains the sfGFP sequence which we will insert into our modified encapsulin plasmid in order to check take up by DCs The cell transfection will allow us to produce new plasmids, but first we have to check if
                the plasmid is still usable as it was stored quite long at room temperature.</P><P class="lead">We used the competent cell transformation protocol that we slightly adapted to our needs. Here are the modifications we made:</P><OL><TABLE><TBODY><TR><TH><SPAN style="font-weight:bold">Amounts in µl</SPAN></TH><TH>Transformation</TH><TH>Transformation control</TH></TR><TR><TD>Competent cells (in tube)</TD><TD>50</TD><TD>50</TD></TR><TR><TD>plasmids</TD><TD>5</TD><TD>-</TD></TR><TR><TD>Nuclease free water</TD><TD>-</TD><TD>1</TD></TR></TBODY></TABLE><LI>Heat shock the cells at 42 °C for up to 45s (we let them 45s). Immediately transfer the tube back on ice for minimum 2 min (we made 3 min) </LI><LI>We did not add cell medium to the mixture because it is not needed when working with cells rendered resistant to Ampicilin</LI><I>Ampicilin directly kills cells which are not resistant to it --&gt; not the case of other antibiotics
the outgrowth step serves to prevent growth of other types of bacteria that could then destroy the ampicilin on the plate and then
there is a risk that not resistant bacteria grow</I><LI>We spread 50 uL each time on LB+ Amp plates using spreader we made ourselves and sterilised</LI></OL><H6>TUESDAY, 8/7/2018</H6><H4><U>Results of the transformation of DH10β competent E.coli cells with
pMC_sfGFP-BsaI_noT7 plasmid</U></H4><FIGURE class="figure"><FIGCAPTION class="mt-3 text-muted">Negative control (cells that were not transfected with the plasmid). As expected, no colonies have grown</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="mt-3 text-muted">Here were plated the transfected cells. No culture have grown showing that the plasmids should have been degraded when staying too long at room temperature.</FIGCAPTION></FIGURE><H4><U>Inoculation of cultures from glycerol stocks of cells (DH5α or DH10β)
containing HexaHistidine-encap plasmids</U></H4><P class="lead">Cultures were inoculated following this protocol</P><P class="lead">We had glycerol stock from 2 different colonies so we made 2 different tubes as well as a negative control in which we just put LB medium and Ampicilin.</P><P class="lead">We used 1000x Ampicilin and as we need 1000x less Ampicilin than medium we put 3uL of Ampicilin in each tube.</P><H4><U>Purification of the plasmids of the cultures from glycerol stocks of cells (DH5α
or DH10β) containing HexaHistidine-encap plasmids</U></H4><P class="lead">The plasmids were purified using the Pure yield<SUP>TM</SUP> plasmid miniprep kit from promega following the corresponding protocol.</P><P class="lead">The resulting of the purification from the two different colonies are as follows:</P><UL><LI><U>colony 1:</U></LI><P class="lead"><STRONG>260/280</STRONG>: 1,75 <STRONG>concentration:</STRONG>50,4 ng/uL</P><LI><U>colony 2:</U></LI><P class="lead"><STRONG>260/280</STRONG>1,79 <STRONG>concentration:</STRONG>41,2 ng/uL</P></UL><H4><U>Test of the efficiency of DH-10 β competent E.coli cells</U></H4><P class="lead">puC19 Plasmid, pMC65, pMC116 were heat shocked into three different competent cell tubes and plated on Amp plates. The plates were incubated overnight at 37 degrees</P><H6>THURSDAY, 8/9/2018</H6><H4><U>Result of the test of the efficiency of DH-10 β competent E.coli cells</U></H4><P class="lead">No colonies were found on the plates, it may be that the cells used are not competent.</P><H4><U>Competent cell transformation using DH5α competent E.coli cells</U></H4><P class="lead">The competent cells were given to us by the LBNC lab and their efficiency was already assessed.</P><P class="lead">We transformed the cells with the 3 following plasmids:</P><UL><LI>pMC_sfGFP-BsaI_noT7</LI><P class="lead">(contains sfGFP) --&gt; we called it p65</P><LI>pSIREN_U6-Cpf1sgRNA</LI><P class="lead">(contains RFP) --&gt; we called it p116</P><LI>puc19 (a standard control plasmid)</LI></UL><P class="lead">We followed this protocol (https://benchling.com/lbnc/f/OOhQKnto-encapsulin/prt-b5aeAD4Y-competent-cell- transformation/edit) to transfrom the cells and added a transformation control (50uL of competent cells with 5uL of nuclease free water).
              </P><H6>FRIDAY, 8/10/2018</H6><H4><U>Results of the cell transformation using DH5α competent E.coli cells</U></H4><P class="lead">No colonies were found on any of the plates. As the cells have already shown efficiency in other experiments, this could mean that the plasmids that stayed for weeks are room temperature (p116 and p65) were all degraded and that the puc19
                whose expiration date is 2017 is not viable anymore.</P><H4><U>Competent cell transformation using DH5α competent E.coli cells</U></H4><P class="lead">The competent cells were given to us by the LBNC and their efficiency was already assessed.</P><P class="lead">We transformed the cells with the 3 following plasmids:</P><UL><LI>pMC_sfGFP-BsaI_noT7</LI><P class="lead">(contains sfGFP) --&gt; we called it p65 (this one was given to us by the LBNC and was stored in good conditions)</P><LI>plasmid carring mCherry from iGEM distribution kit</LI><LI>plasmid carring Cyan Fluorescent Protein from iGEM distribution kit</LI><LI>plasmid carring YGFP from iGEM distribution kit</LI></UL><P class="lead">We followed this protocol (https://benchling.com/lbnc/f/OOhQKnto-encapsulin/prt-b5aeAD4Y-competent-cell- transformation/edit) but as we were not well organized we were not able to start directly when we got the cells from the LBNC and we
                let them thaw a bit on ice and then refroze them in the -20 °C freezer before finally thaw them on ice and use them. Moreover we were using 1 plasmid conferring Ampicilin resistance and 2 conferring chlorampenicol resistance but we did
                the outgrowth step for the 3, although we should not have done it for the 1 conferring ampicilin resistance. Finally, the quantity of DNA we used to transform the cells with the plasmids from iGEM's distribution kit was very low; indeed
                iGEM suggests to dilute the dried DNA from the distribution kit in 10uL water and then do the transformation with 1uL which corresponds to 200-300pg of DNA.</P><H6>SATURDAY, 8/11/2018</H6><H4><U>Result of the competent cell transformation using DH5α competent E.coli cells</U></H4><P class="lead">No colonies were found.</P><H2 id="week_6">Notebook week 6 (13/08/18)</H2><P class="lead"><STRONG>Cells we used:</STRONG>DH5α competent E.coli cells (some from Ivan and some from Michael)</P><P class="lead">Plasmids we used:</P><UL><LI>pMC_sfGFP-BsaI_noT7 (https://benchling.com/lbnc/f/2wg8QdVJ-introduction/seq-Uj4WhTuX-pmc_sfgfp-bsai_not7/edit)</LI><LI>puc19</LI><LI>CFP carrying plasmid (http://parts.igem.org/Part:BBa_K404319)</LI><LI>mCherry carrying plasmid (http://parts.igem.org/Part:BBa_J06504)</LI><LI>SYFP2 containing plasmid (http://parts.igem.org/Part:BBa_K864100)</LI></UL><TABLE><TBODY><TR><TH>Plasmid</TH><TH>pMC_sfGFP-BsaI_noT7</TH><TH>puc19</TH><TH>CFP carrying plasmid</TH><TH>mCherry carrying plasmid</TH><TH>SYFP2 containing plasmid</TH></TR><TR><TD>Resistance</TD><TD>Ampicilin</TD><TD>Ampicilin</TD><TD>Chloramphenicol</TD><TD>Chloramphenicol</TD><TD>Chloramphenicol</TD></TR><TR><TD>Vector</TD><TD>-</TD><TD>-</TD><TD>pSB1C3</TD><TD>pSB1C3</TD><TD>pSB1C3</TD></TR></TBODY></TABLE><P class="lead">We followed iGEM's single tube transformation protocol (http://parts.igem.org/Help:Protocols/Transformation) but we put only 30uL of cells in each tube and used LB medium instead of SOC.</P><P class="lead">We did the mistake to also put medium to the ampicilin resistant bacteria, they were put in the incubator for ~1min before she realized it. Then we took them, put them quickly on ice (which was a bad idea), put them in the microcentrifuge
                and centrifuge them for 3min at 6,8g. We removed the supernatant and vortex them a bit and then plate the resulting 160uL.</P><H4><U>Result of the competent cell transformation</U></H4><P class="lead">We had colonies only on the plate where we put the cells transfected with pMC_sfGFP-BsaI_noT7 (https://benchling.com/lbnc/f/2wg8QdVJ-introduction/seq-Uj4WhTuX-pmc_sfgfp-bsai_not7/edit).
              </P><H4><U>Pouring LB Agar plates</U></H4><P class="lead">For 200mL of LB Agar mixture (~10 plates) --&gt; 3g Agar, 5g of LB <STRONG>! to cover the erlenmeyer after the autoclave!</STRONG></P><P class="lead">Competent cell transfection with Laurine + postivie control</P><H1>Notebooks under PDF</H1><LI><A href="https://static.igem.org/mediawiki/2018/9/90/T--EPFL--Notebook_week_6_%2813_08_18%29_·_Benchling.pdf" target="_blank"><SPAN style="color:blue">Notebook Week 6</SPAN></A></LI><LI><A href="https://static.igem.org/mediawiki/2018/2/2d/T--EPFL--Notebook_week_7_%2820_08_18%29_·_Benchling.pdf" target="_blank"><SPAN style="color:blue">Notebook Week 7</SPAN></A></LI><LI><A href="https://static.igem.org/mediawiki/2018/d/d2/T--EPFL--Notebook_week_8_%2827_08_18%29_·_Benchling.pdf" target="_blank"><SPAN style="color:blue">Notebook Week 8</SPAN></A></LI><LI><A href="https://static.igem.org/mediawiki/2018/3/33/T--EPFL--Notebook_week_9_%2803_09_18%29_·_Benchling.pdf" target="_blank"><SPAN style="color:blue">Notebook Week 9</SPAN></A></LI><LI><A href="https://static.igem.org/mediawiki/2018/b/b3/T--EPFL--Notebook_week_10_%2810_09_18%29_·_Benchling.pdf" target="_blank"><SPAN style="color:blue">Notebook Week 10</SPAN></A></LI><LI><A href="https://static.igem.org/mediawiki/2018/f/f7/T--EPFL--Notebook_week_11_%2818_09_18%29_·_Benchling.pdf" target="_blank"><SPAN style="color:blue">Notebook Week 11</SPAN></A></LI><LI><A href="https://static.igem.org/mediawiki/2018/3/3e/T--EPFL--Notebook_week_12_%2824_09_18%29_·_Benchling.pdf" target="_blank"><SPAN style="color:blue">Notebook Week 12</SPAN></A></LI><LI><A href="https://static.igem.org/mediawiki/2018/8/85/T--EPFL--Notebook_week_13_%2830_09_18%29_·_Benchling.pdf" target="_blank"><SPAN style="color:blue">Notebook Week 13</SPAN></A></LI><LI><A href="https://static.igem.org/mediawiki/2018/9/99/T--EPFL--Notebook_Week_14_·_Benchling.pdf" target="_blank"><SPAN style="color:blue">Notebook Week 14</SPAN></A></LI></ARTICLE></DIV></DIV></DIV></SECTION></MAIN><DIV class="container"><DIV class="align-items-center"><DIV class="text-center"><DIV><SPAN class="pl-3 d-inline-block">iGEM2018@groupes.epfl.ch</SPAN></DIV><SPAN class="d-block mt-4">IGEM EPFL 2018</SPAN><A href="https://twitter.com/EPFL_iGEM" data-fancybox="" class="btn btn-twitter btn-icon-label btn-circle"><SPAN class="btn-inner--text">Twitter</SPAN></A><A href="https://www.facebook.com/igemepfl/?ref=br_rs" data-fancybox="" class="btn btn-facebook btn-icon-label btn-circle"><SPAN class="btn-inner--text">Facebook</SPAN></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>