<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM EPFL 2018</TITLE></P><DIV class="container"><A class="navbar-brand mr-lg-5" href="https://2018.igem.org/Team:EPFL"><STRONG>iGEM</STRONG> EPFL 2018</A><DIV class="collapse navbar-collapse" id="navbar_main_collapse"><UL class="navbar-nav align-items-lg-center ml-lg-auto"><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:EPFL">HOME</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Description" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Description</H2><P class="mb-0">Take a closer look at CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Design" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Design</H2><P class="mb-0">Learn more about how we designed CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Software" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Software</H2><P class="mb-0">Explore the Bioinformatic Pipeline implemented in CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Results" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Results</H2><P class="mb-0">Discover the Results of our experiments</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Demonstrate" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Demonstration</H2><P class="mb-0">Demonstration of how our Therapeutic System works</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/InterLab" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Interlab</H2><P class="mb-0">Learn more about our participation in the Interlab Study</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Parts" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Parts</H2><P class="mb-0">Discover the Parts that make up CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Improve" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Improve</H2><P class="mb-0">Explore our Novel and Improvements Parts</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Measurement" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Measurement</H2><P class="mb-0">Learn about measurement approaches we used for Part Characterization</P></DIV></DIV></A></DIV></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Team" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Team Members</H2><P class="mb-0">Learn more about our Team Members</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Collaborations" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Collaborations</H2><P class="mb-0">Discover the interactions we had with other iGEM Teams during CAPOEIRA</P></DIV></DIV></A></DIV></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Human_Practices" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Integrated Human Practices</H2><P class="mb-0">Discover how we integrated Human Practices into the Purpose, Design and Implementation of CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Ethics" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Ethics</H2><P class="mb-0">Explore the ethical considerations of CAPOEIRA</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Applied_Design" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Applied Design</H2><P class="mb-0">Explore how our therapeutic system attempts to meet real world medical need</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Public_Engagement" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Education and Public Engagement</H2><P class="mb-0">Discover our Education and Public Engagement work with the public and scientific communities</P></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Meetups" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">European and Eurasian Meetup</H2><P class="mb-0">Learn about our interactions with the iGEM Community during the Meetups</P></DIV></DIV></A></DIV></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:EPFL/Notebook#">Lab</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:EPFL/Safety">Safety</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:EPFL/Entrepreneurship">Entrepreneurship</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Acknowledgements</A><DIV class="dropdown-menu dropdown-menu-lg py-0"><DIV class="list-group list-group-flush"><A href="https://2018.igem.org/Team:EPFL/Attributions" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Attributions</H2></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Sponsors" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Sponsors</H2></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Partners" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Partners</H2></DIV></DIV></A><A href="https://2018.igem.org/Team:EPFL/Awards" class="list-group-item list-group-item-action"><DIV class="media d-flex align-items-center"><DIV class="media-body ml-3"><H2 class="heading h6 mb-1">Awards</H2></DIV></DIV></A></DIV></DIV></LI></UL></DIV></DIV><MAIN><SECTION class="slice slice-lg bg-gradient-primary" data-separator="rounded-left" data-separator-orientation="bottom"><DIV class="container pt-lg-md"><DIV class="row"><DIV class="col-lg-8"><H2 class="display-4 text-white">Protocols</H2></DIV></DIV></DIV></SECTION><SECTION><DIV class="container pt-lg-lg"><DIV class="row justify-content-center lead"><DIV class="col-md-12"><ARTICLE><P class="lead">This page collects the different protocols used in our project. They are sorted in alphabetical order.</P><DIV id="protocols"><DIV class="card"><A data-toggle="collapse" href="#agarose_gel"><DIV class="card-header"><H3 class="card-link">						 Agarose gel electrophoresis
                      								 </H3></DIV></A><DIV id="agarose_gel" class="collapse" data-parent="#protocol"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">A protocol on how to prepare and run a standard agarose gel electrophoresis at a certain percentage of agarose. The percentage of agarose to use can be estimated according to the sequence length (bp)1. </P><H4 class="text-muted">Materials</H4><UL><LI>10X TAE buffer</LI><LI>SYBR Safe (10 000X)</LI><LI>Distilled water</LI><LI>DNA samples (PCR,..)</LI><LI>Parafilm</LI><LI>Gel Loading Dye, Purple (6X) (NEB)</LI><LI>GeneRuler 1 kb Plus DNA Ladder (ladder should be adapted to the sequence length)</LI></UL><H4 class="text-muted">Procedure</H4><OL><H5> Preparation of the gel</H5><LI>Prepare 60 ml of (1-2)% Agarose in 1X TAE buffer. </LI><UL><LI>Dilute the 10X concentrated TAE buffer by adding 6 ml to 54 ml of water in a column.</LI><LI>Weight the desired amount of agarose (1% := 600 mg) and put it in an Erlenmeyer. Add the buffer.</LI><LI>Melt the agarose in a microwave oven (no aluminium or Parafilm) for around 2 minutes (until all the agarose is dissolved). </LI></UL><LI>Add 6 µl SYBR safe into the Erlenmeyer.</LI><LI>Set up the gel-casting mold into the frame (rubber ends against the walls). Place the combs onto the frame at the top of the mold.</LI><LI>Pour the agarose solution into the gel-cast (don't overfill) and wait until the gel solidifies before loading your samples (between 1/2 and 1 hour).</LI><LI><B>IMPORTANT:</B> Place the gel-cast so that the holes are directed towards the negative terminal (black wire).</LI><LI>Cover the gel with 1X TAE buffer (~250 ml) and CAREFULLY remove the combs.</LI><H5>Sample loading</H5><LI>On a parafilm, mix 8 μl of your DNA samples with the amount of loading dye needed for a total reaction volume of 12 µl, then load your samples and 5 µl of DNA ladder separately on different wells.</LI><LI>Run the gel at 100 Volts for 30-40 minutes.</LI><LI>Remove the gel from the chamber and take a photography at the UV transilluminator.</LI></OL><H5>References</H5><P>Promega-<A href="https://ch.promega.com/resources/pubhub/enotes/what-percentage-agarose-is-needed-to-sufficiently-resolve-my-dna-sample/">What percentage agarose is needed to sufficiently resolve my DNA sample?</A></P></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#AbStain"><DIV class="card-header"><H3 class="card-link">
                                                                    Antibody Staining for FACS  - Protocol
                                                                    </H3></DIV></A><DIV id="AbStain" class="collapse"><DIV class="card-body"><H2 id="heat"><U>Antibody Staining for FACS  - Protocol</U></H2><H4 class="text-muted">Introduction</H4><P class="lead">Fluorescence Activated Cell Sorting (FACS) is a technique that can separate different cell population or types by using the light scattering and fluoresce characteristics of each individual cell. In this case we want to be able to differentiate the cells that have been activated during our incubation with our vaccine and antigen, and see if there is a population of cell that is presenting the antigen on the surface protein of the cell (MHC I). In order to perform this task we florescence label an antibody against our standard OVA epitope, and after flasking the culture only the cells with the antigen-antibody complex on their surface will present high fluoresce. In order to perform this protocol the cells have to be previously culture and incubated with the encapsulin. </P><H4 class="text-muted">Materials</H4><UL><LI>5 x 105 cells per test - Primary Dendritic Cells extracted from Bone Marrow</LI><LI>FACS buffer: Cold PBS with 0.2% BSA</LI><LI>DAPI working solution: 5 μM DAPI in FACS buffer (diluted from 10X DAPI stock solution)</LI><LI>FACS block buffer: 0.3 μL anti-CD16/32 antibody solution in 20 μL FACS buffer (1:100 dilution; 0.3 μL antibody per test; 20 μL FACS block buffer per test)</LI><LI>FACS antibody cocktail: fluorescent dye-labeled antibodies were prepared according the FACS antibody inventory (X.X μL antibody per test; 20 μL FACS antibody cocktail per test)</LI></UL><H4 class="text-muted">Procedure </H4><H5> For 1.5 mL Eppendorf tube:</H5><OL><LI>Wash cells with FACS buffer, centrifuge down by 3000 rcf, 2 min, discard the supernatant by vacuum pump (try to discard all the supernatant, assume the total volume of remaining liquid and cells is 10 μL). Completely resuspend the cell pellet by vortex or pipetting.</LI><LI>Add 20 μL block buffer (total volume: 30 μL). Mix by vortex. Incubate for 15 min in 4 ℃.</LI><LI>Without wash, directly add 20 μL FACS antibody cocktail (total volume: 50 μL). Mix by vortex. Incubate for 20 min in 4 ℃.</LI><LI>Add 1 mL FACS buffer to dilute the mixture, centrifuge down the antibodies-labeled cells by 3000 rcf, 2 min, discard the supernatant by vacuum pump. Completely resuspend the cell pellet by vortex or pipetting.</LI><LI>Add 200 μL FACS buffer or DAPI working solution, filter the cell suspension by 200 nylon mesh or 70 μm cell strainer into a new tube.</LI></OL><H5>For U bottom 96 well plate:</H5><OL><LI>Wash cells with FACS buffer, centrifuge down by 1800 rpm, 5 min, discard the supernatant by throwing once (try to discard all the supernatant, assume the total volume of remaining liquid and cells is 10 μL). Completely resuspend the cell pellet by vortex or pipetting.</LI><LI>Add 20 μL block buffer (total volume: 30 μL). Mix by vortex. Incubate for 15 min in 4 ℃.</LI><LI>Without wash, directly add 20 μL FACS antibody cocktail (total volume: 50 μL). Mix by vortex. Incubate for 20 min in 4 ℃.</LI><LI>Add 200 μL FACS buffer to dilute the mixture, centrifuge down the antibodies-labeled cells by 1800 rpm, 5 min, discard the supernatant by throwing once. Completely resuspend the cell pellet by vortex or pipetting.</LI><LI>Add 200 μL FACS buffer or DAPI working solution, filter the cell suspension by 200 nylon mesh into a new U bottom 96 well plate.</LI><LI>Data collecting by Attune NxT. Acquisition speed: 200 μL /min. Acquisition volume: 100 μL.</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Buffer"><DIV class="card-header"><H3 class="card-link">
                                    Buffer A
                                      </H3></DIV></A><DIV id="Buffer" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">Buffer A is an essential component of cell-free expression reactions. </P><H4 class="text-muted">Materials</H4><UL><LI>Deionized water</LI><LI>Tris base</LI><LI>Magnesium glutamate (L-Glutamic acid hemimagnesium salt tetrahydrate)</LI><LI>Potassium glutamate (L-Glutamic acid potassium salt monohydrate)</LI><LI>DTT (1,4-Dithiothreitol)</LI></UL><H4 class="text-muted">Procedure</H4><H4>Preparation of 1M stock solutions of tris base magnesium glutamate, potassium glutamate</H4><OL><LI>Weight 12.11 g of tris buffer (121.14 g/mol), add it to 100 ml of deionized water and adjust to pH 8.2 with acetic acid.</LI><LI>Weight 38.86 g of magnesium glutamate (388.61 g/mol), add it to 100 ml of deionized water.</LI><LI>Weight 20.32 g of potassium glutamate (203.23 g/mol), add it to 100 ml of deionized water.</LI></OL><H4>Preparation of 1L of buffer A</H4><OL start="4"><LI>Add 10 ml of 1 M tris acetate (final concentration is 10 mM).</LI><LI>Add 14 ml of 1 M magnesium glutamate (final concentration is 14 mM).</LI><LI>Add 60 ml of 1 M potassium glutamate (final concentration is 60 mM).</LI><LI>If necessary, add 2 ml of 1 M DTT (final concentration is 2 mM) in order to have buffer A + DTT and store at -20°C.</LI><LI>Complete with deionized water to 1L.</LI><LI>Store at room temperature.</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Cellfree"><DIV class="card-header"><H3 class="card-link">
                                      Cell free reaction (standard)
                                        </H3></DIV></A><DIV id="Cellfree" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">In order to test the efficiency of the different components of our cell-free reactions, we perform microplate reader cell-free expression reactions with GFP DNA template. We use the fluorescence emitted by the GFP as a way to measure protein expression. We used this kind of experiments as an internal reference to assess in a relative manner the quality of our lysates and energy solutions.
                                          It is important to test only one component of the cell-free reaction at a time. We will perform 10µl reactions with 4 repeats of each reaction type. It is crucial to do a negative control per reaction type (we also suggest to do 4 repeats for each negative control). </P><H4 class="text-muted">Materials</H4><UL><LI>Lysate</LI><LI>GFP DNA circular template (10ng/µl of reaction)</LI><LI>Eppendorf tubes</LI><LI>Micro-centrifuge</LI><LI>Mircoplate reader Victor X3 from PerkinElmer</LI><LI>384 Nunc plate from ThermoFisher</LI><LI>Chillout Liquid Wax from BioRad</LI></UL><H4 class="text-muted">Procedure</H4><H4>Microplate reader settings</H4><OL><LI>Start preheating the machine to 29°C </LI><LI> Configure the machine for GFP measurement.
                                            </LI><LI>Set the parameters as follows: 300 measurement, 90s and quick shaking between each measurement (this corresponds to around 7.5 h expression) </LI></OL><H4>Master mixes preparation</H4><P class="lead">
                                          We prepare 2 master mixes: Master Mix 1 (MM1) containing lysate and buffer A, and Master Mix 2 (MM2) containing energy solution, water and DNA (or just energy solution and water for the negative controls).
                                          
                                          We calculated the volumes needed based on the following 10µl cell-free expression reaction:
                                        </P><UL><LI> 2.5µl Lysate</LI><LI>2.5µl Energy Solution (ES)</LI><LI>2.5µl Buffer A</LI><LI>100ng of DNA</LI><LI> Nuclease free water (complete for a total volume of 10µl)</LI></UL><P class="lead">For each reaction type, we suggest to prepare enough master mixes for 4,5 reactions because of pipetting errors.</P><CENTER><TABLE><TBODY><TR><TH>Lysate</TH><TH>Buffer A</TH></TR><TR><TD>11.25µl</TD><TD>11.25µl</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR><TH>ES</TH><TH>DNA</TH><TH>Nuclease free water</TH></TR><TR><TD>11.25µl</TD><TD>450ng of DNA (volume is plasmid stock solution concentration dependant)</TD><TD>complete to a total volume of 45µl</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR><TH>ES</TH><TH>Nuclease free water</TH></TR><TR><TD>11.25µl</TD><TD>11.25µl</TD></TR></TBODY></TABLE></CENTER><OL start="4"><LI>Put 5µl of MM1 in one corner of each well</LI><LI>Put 5µl of either MM2+ or MM2- in the opposite corner of each well.</LI><LI>Centrifuge the plate for a short time (30-60 seconds) at 4000 rpm to remove any bubbles andcollect the solutions at the bottom of the wells.</LI><LI>Add 35 μl of wax to each well.</LI><LI>Load the plate and start the program.</LI><DIV class="card"><A data-toggle="collapse" href="#transformation"><DIV class="card-header"><H3 class="card-link">
                                       Competent cell transformation (with Ampicilin)
                                      </H3></DIV></A><DIV id="transformation" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">This protocol shows how to transfer plasmid DNA into competent cells.</P><H4 class="text-muted">Materials</H4><UL><LI>Competent cells</LI><LI>Control plasmid</LI><LI>ligation mix</LI><LI>LB-Ampicilin plates</LI><LI>Heating Block</LI><LI>Bunsen burner</LI><LI>Ethanol 96%</LI></UL><H4 class="text-muted">Procedure</H4><OL><LI>Add components according to the following table to three tubes of competent cells</LI><CENTER><TABLE><TBODY><TR><TH>Amounts in μl</TH><TH>Transfection mix</TH><TH>Vector control</TH></TR><TR><TD>Competent cells (In tube)</TD><TD>50μl</TD><TD>50μl</TD></TR><TR><TD>Plasmid DNA</TD><TD>5μl</TD><TD>-</TD></TR><TR><TD>Vector</TD><TD>-</TD><TD>5μl</TD></TR></TBODY></TABLE></CENTER><LI>Incubate on ice for 30 min.</LI><LI>Heat shock the cells up to 45 sec. at 42°C. Immediatly transfer the tube back on ice for 5 min.</LI><LI>Spread 50μl.</LI><LI>Incubate the plates overnight at 37°C to select for transformants.</LI></OL></DIV></DIV></DIV><DIV><DIV class="card"><A data-toggle="collapse" href="#crRNAtranscription"><DIV class="card-header"><H3 class="card-link">
                                         crRNA Transcription using T7 RNA Polymerase (Promega)
                                        </H3></DIV></A><DIV id="crRNAtranscription" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">In this part we're going to transcribe the CRISPR RNA (crRNA) required for the CRISPR-Cas12a assay using the isothermal T7 RNA polymerase. This enzyme will only transcribe DNA downstream of a double-stranded T7
                                            promoter (in the 5' to 3' direction), thus we had to fuse the single-stranded DNA (ssDNA) coding sequence (CDS, flanked on its 3' end with the 3' to 5' strand of T7 promoter) with a primer (T7 primer, complementary 5' to 3' )
                                            in order to constitute the promoter and obtain efficient transcription. We initially followed the Annealing oligonucleotides protocol in order to anneal the T7 primer to the ssDNA which will constitute our DNA template. This is
                                            based on Promega's protocol: &quot;Synthesis of Nonlabeled RNA&quot; [1] </P><H4 class="text-muted">Materials</H4><UL><LI>Transcription Optimized 5X Buffer</LI><LI>DTT, 100mM</LI><LI>Recombinant RNasin® Ribonuclease inhibitor</LI><LI>rATP, rGTP, rUTP, rCTP</LI><LI>DNA template* (annealed ssDNA + primer) in water ( orTE buffer at 2–5μg)</LI><LI>T7 RNA polymerase (Phage RNA polymerase)</LI><LI>Nuclease-Free Water</LI></UL><H4 class="text-muted"> Procedure</H4><OL><LI>Make the rNTP mix as following</LI><CENTER><TABLE class="tg"><TBODY><TR><TH class="tg-0lax">Products</TH><TH class="tg-0lax">Concentration</TH></TR><TR><TD class="tg-0lax">rATP</TD><TD class="tg-0lax">2.5 mM</TD></TR><TR><TD class="tg-0lax">rGTP</TD><TD class="tg-0lax">2.5 mM</TD></TR><TR><TD class="tg-0lax">rUTP</TD><TD class="tg-0lax">2.5 mM</TD></TR><TR><TD class="tg-0lax">rCTP</TD><TD class="tg-0lax">2.5 mM</TD></TR><TR><TD class="tg-0lax">In nuclease free water</TD></TR></TBODY></TABLE></CENTER><LI>In a microcentrifuge tube, add the following reagents at room temperature in the order listed</LI><CENTER><TABLE class="tg"><TBODY><TR><TH class="tg-0lax">Materials</TH><TH class="tg-0lax">quantity</TH></TR><TR><TD class="tg-0lax">Transcription Optimized 5X Buffer</TD><TD class="tg-0lax">20μl</TD></TR><TR><TD class="tg-0lax">DTT, 100mM</TD><TD class="tg-0lax">10μl</TD></TR><TR><TD class="tg-0lax">Recombinant RNasin® Ribonuclease Inhibitor</TD><TD class="tg-0lax">100 units</TD></TR><TR><TD class="tg-0lax">rNTP mix</TD><TD class="tg-0lax">20μl</TD></TR><TR><TD class="tg-0lax">DNA template, linearized (in water or TE buffer at 2–5μg, i.e. ~100 µM concentrated)*</TD><TD class="tg-0lax">2μl</TD></TR><TR><TD class="tg-0lax">Phage RNA polymerase</TD><TD class="tg-0lax">40 units</TD></TR><TR><TD class="tg-0lax">Nuclease-Free Water to final volume of</TD><TD class="tg-0lax">100μl</TD></TR></TBODY></TABLE></CENTER><LI>Incubate for 2 hours at 37°C.</LI><LI>Purify the sample following the RNA purification protocol.</LI></OL><H4 class="text-muted"> References</H4><OL><LI> Promega: &quot;Synthesis of Nonlabeled RNA&quot; protocol, https://www.promega.com/-/media/files/resources/protocols/product-information-sheets/n/t7-rna-polymerase-protocol.pdf</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#DPNI"><DIV class="card-header"><H3 class="card-link">
                                         DPNI plasmid digestion
                                        </H3></DIV></A><DIV id="DPNI" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">DpnI cleaves only when it's recognition site is methylated. Useful for removing cell-derived plasmid template from PCR samples.</P><H4 class="text-muted">Materials</H4><UL><LI>DPNI</LI><LI>Enzyme buffer (Might work with the one used for the PCR)</LI><LI>PCR product</LI></UL><H4 class="text-muted">Procedure</H4><UL><H5>Digest mix</H5><CENTER><TABLE><TBODY><TR><TH>PCR product</TH><TH>50μl</TH></TR><TR><TD>DPNI</TD><TD>1μl</TD></TR></TBODY></TABLE></CENTER><H5>Incubation</H5><P class="lead">Incubate for one hour at 37°C</P><H5>DPNI heat inactivation</H5><P class="lead">incubate at 80°C for 20 minutes</P></UL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Energy"><DIV class="card-header"><H3 class="card-link">
                                        Energy solution (for cell-free expression) READY
                                          </H3></DIV></A><DIV id="Energy" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">Buffer A is an essential component of cell-free expression reactions. </P><H4 class="text-muted">Materials</H4><H4 class="text-muted">Procedure</H4><H4>Make amino acids stock solution</H4><OL><LI>In one tube weigh each amino acid one by one using a small parafilm paper.</LI><LI>Add all the amino acids into the same tube apart from the tyrosine that should be added in a separate tube by itself.</LI><LI>To the first tube (amino acid – tyrosine ) add 500 ul of water in total: small volumes of water should be added periodically before adding the 500 ul total. After each addition check for the pH, it should be around 5-6. We can also add small volumes of KOH 1% (100 ul max) to dissolve the amino acids in the solution.</LI><LI>To the tyrosine only tube add 400 ul of water and 10 ul of KOH. Add those volume periodically while vortexing to help the amino acid dissolve in solution. The pH should be around 9.</LI><LI>Do not mix the two tubes together at this stage of the preparation to prevent precipitation.</LI></OL><H4>Making energy solution</H4><OL start="6"><LI>After preparing the amino acids, prepare the rest of the components by weighing each time the component in a tube and adding the amount of water listed in the table below. Vortex well so the component dissolves.</LI><LI>After weighing all the components, mix the amino acids and the rest of the solutions together, according to the table.</LI><LI>Aliquot into small 25ul tube and flash freeze in liquid nitrogen.</LI><LI>Store in a -80°C.</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Cas12a-Assay"><DIV class="card-header"><H3 class="card-link">                Fluorophore-Quencher reporter Cas12a assay
                                          </H3></DIV></A><DIV id="Cas12a-Assay" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">This assay's purpose is to detect a specific DNA sequence (the activator) using the CRISPR/Cas12a system. Cas12a's feature is to cleave any ssDNA in the sample once it has found its target. We used this to our
                                              advantage and used a single-stranded fluorophore-quencher reporter (DNaseAlert) to be able to quantify our sequence of interest using a plate reader. This protocol is based on and optimized from the LbCas12a collateral
                                              detection protocol (&quot;Fluorophore quencher (FQ)-labeled reporter assays&quot;)[1] </P><H4 class="text-muted">Materials</H4><UL><LI>EnGen® Lba Cas12a (Cpf1) 1 μM</LI><LI>Purified crRNA from DNA sequence 1μM</LI><LI>10X Binding buffer (200 mM Tris-HCl, pH 7.5, 1 M KCl, 50 mM MgCl2, 50% glycerol, 500 μg/ml heparin, 1mM DTT)</LI><LI>DNaseAlert™ (IDT) 1 µM </LI><LI>Nuclease-Free Water</LI><LI>6 of small pcr tubes (0.2 ml) for your master mix and samples</LI><LI>DNase I enzyme (Zymo)</LI><LI>384 well Plate</LI><LI>Multi Well Plate Sealing Films</LI></UL><H4 class="text-muted">Procedure</H4><OL><LI> Preparation of Cas12a master mix </LI><CENTER><TABLE class="tg"><TBODY><TR><TH class="tg-0lax">LbCas12a concentration</TH><TH class="tg-0lax">gRNA concentration</TH><TH class="tg-0lax">Activator concentraion</TH><TH class="tg-0lax">DNaseAlertconcentration</TH><TH class="tg-0lax">Buffer</TH><TH class="tg-0lax">Dnase I</TH></TR><TR><TD class="tg-0lax">FQ1</TD><TD class="tg-0lax">62.5 nM</TD><TD class="tg-0lax">75 nM</TD><TD class="tg-0lax">Variable</TD><TD class="tg-0lax">160 nM</TD><TD class="tg-0lax">1X Binding</TD><TD class="tg-0lax">-</TD></TR><TR><TD class="tg-0lax">Negative control</TD><TD class="tg-0lax">62.5 nM</TD><TD class="tg-0lax">75 nM</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">160 nM</TD><TD class="tg-0lax">1X Binding</TD><TD class="tg-0lax">-</TD></TR><TR><TD class="tg-0lax">Blank</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">1X Binding</TD><TD class="tg-0lax">-</TD></TR><TR><TD class="tg-0lax">Positive control</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">160 nM</TD><TD class="tg-0lax">1X Binding</TD><TD class="tg-0lax">2.4 μl</TD></TR></TBODY></TABLE></CENTER><UL><LI>Add the following materials except the DNaseAlert and activator into a 0.2 ml tubes. Incubate at 37°C for 30 min, then add the rest of the components.</LI><CENTER><TABLE class="tg"><TBODY><TR><TH class="tg-0lax">Components</TH><TH class="tg-0lax">FQ1</TH><TH class="tg-0lax">Negativecontrol</TH><TH class="tg-0lax">Blank</TH><TH class="tg-0lax">PositiveControl</TH></TR><TR><TD class="tg-0lax">10X Binding Buffer</TD><TD class="tg-0lax">6.6</TD><TD class="tg-0lax">6.6</TD><TD class="tg-0lax">6.6</TD><TD class="tg-0lax">6.6</TD></TR><TR><TD class="tg-0lax">Cas12 [1µM]</TD><TD class="tg-0lax">3.75</TD><TD class="tg-0lax">3.75</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD></TR><TR><TD class="tg-0lax">crRNA [1µM]</TD><TD class="tg-0lax">4.5</TD><TD class="tg-0lax">4.5</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD></TR><TR><TD class="tg-0lax">Nuclease Free Water</TD><TD class="tg-0lax">29.6</TD><TD class="tg-0lax">35.6</TD><TD class="tg-0lax">53.4</TD><TD class="tg-0lax">41.4</TD></TR><TR><TD class="tg-z0zd" colspan="5">Incubation 30 minutes at 37°C</TD></TR><TR><TD class="tg-0lax">DNase Alert</TD><TD class="tg-0lax">9.6</TD><TD class="tg-0lax">9.6</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">9.6</TD></TR><TR><TD class="tg-0lax">Activator</TD><TD class="tg-0lax">6</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD></TR><TR><TD class="tg-0lax">DNase I</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">2.4</TD></TR><TR><TD class="tg-0lax">Final volume (µL)</TD><TD class="tg-0lax">60</TD><TD class="tg-0lax">60</TD><TD class="tg-0lax">60</TD><TD class="tg-0lax">60</TD></TR></TBODY></TABLE></CENTER></UL><LI>Prepare the Optical Plate</LI><UL><LI>Load 24 µl of each corresponding tubes into a 384 opti plate (in duplicate) in the following order:</LI><CENTER><TABLE><TBODY><TR><TH>Negative</TH><TH>FQ1</TH><TH>blank</TH><TH>Positive control</TH></TR><TR><TD>Negative</TD><TD>FQ2</TD><TD>blank</TD><TD>Positive control</TD></TR></TBODY></TABLE></CENTER><LI>Stick an adhesive film on the top of the plate.</LI></UL><LI>Put it in the plate reader. Set up the device: at 535 nm excitation and 590 nm emission, 37℃ and take the measurements every 20 seconds for 180 repeats.</LI><LI>Plot a graph of the fluorescence as a function of time (minutes), taking the average of the fluorescence obtained for each well.</LI></OL><H4 class="text-muted">References</H4><UL><LI>[1]Chen, J. S., Ma, E., Harrington, L. B., Da Costa, M., Tian, X., Palefsky, J. M., &amp; Doudna, J. A. (2018). CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity. Science, 360(6387), 436–439.
                                              </LI></UL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Gel"><DIV class="card-header"><H3 class="card-link">
                                          Gel extraction
                                            </H3></DIV></A><DIV id="Gel" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">In order to test the efficiency of the different components of our cell-free reactions, we perform microplate reader cell-free expression reactions with GFP DNA template. We use the fluorescence emitted by the GFP as a way to measure protein expression. We used this kind of experiments as an internal reference to assess in a relative manner the quality of our lysates and energy solutions.
                                              It is important to test only one component of the cell-free reaction at a time. We will perform 10µl reactions with 4 repeats of each reaction type. It is crucial to do a negative control per reaction type (we also suggest to do 4 repeats for each negative control). </P><H4 class="text-muted">Materials</H4><UL><LI>Lysate</LI><LI>GFP DNA circular template (10ng/µl of reaction)</LI><LI>Eppendorf tubes</LI><LI>Micro-centrifuge</LI><LI>Mircoplate reader Victor X3 from PerkinElmer</LI><LI>384 Nunc plate from ThermoFisher</LI><LI>Chillout Liquid Wax from BioRad</LI></UL><H4 class="text-muted">Procedure</H4><H4>Microplate reader settings</H4><OL><LI>Start preheating the machine to 29°C </LI><LI> Configure the machine for GFP measurement.
                                                </LI><LI>Set the parameters as follows: 300 measurement, 90s and quick shaking between each measurement (this corresponds to around 7.5 h expression) </LI></OL><H4>Master mixes preparation</H4><P class="lead">
                                              We prepare 2 master mixes: Master Mix 1 (MM1) containing lysate and buffer A, and Master Mix 2 (MM2) containing energy solution, water and DNA (or just energy solution and water for the negative controls).
         
                                              We calculated the volumes needed based on the following 10µl cell-free expression reaction:
                                            </P><UL><LI> 2.5µl Lysate</LI><LI>2.5µl Energy Solution (ES)</LI><LI>2.5µl Buffer A</LI><LI>100ng of DNA</LI><LI> Nuclease free water (complete for a total volume of 10µl)</LI></UL><P class="lead">For each reaction type, we suggest to prepare enough master mixes for 4,5 reactions because of pipetting errors.</P><CENTER><TABLE><TBODY><TR><TH>Lysate</TH><TH>Buffer A</TH></TR><TR><TD>11.25µl</TD><TD>11.25µl</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR><TH>ES</TH><TH>DNA</TH><TH>Nuclease free water</TH></TR><TR><TD>11.25µl</TD><TD>450ng of DNA (volume is plasmid stock solution concentration dependant)</TD><TD>complete to a total volume of 45µl</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR><TH>ES</TH><TH>Nuclease free water</TH></TR><TR><TD>11.25µl</TD><TD>11.25µl</TD></TR></TBODY></TABLE></CENTER><OL start="4"><LI>Put 5µl of MM1 in one corner of each well</LI><LI>Put 5µl of either MM2+ or MM2- in the opposite corner of each well.</LI><LI>Centrifuge the plate for a short time (30-60 seconds) at 4000 rpm to remove any bubbles andcollect the solutions at the bottom of the wells.</LI><LI>Add 35 μl of wax to each well.</LI><LI>Load the plate and start the program.</LI><DIV class="card"><A data-toggle="collapse" href="#Glycerol"><DIV class="card-header"><H3 class="card-link">
                                              Glycerol stock preparation
                                            </H3></DIV></A><DIV id="Glycerol" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">This is how to make glycerol stocks of bacteria cell cultures that are suitable for long time storage</P><H4 class="text-muted">Materials</H4><UL><LI>Liquid cell culture</LI><LI>Glycerol</LI><LI>1.5 ml tube</LI></UL><H4 class="text-muted">Procedure</H4><OL><LI>After you have bacteria growth in your liquid culture, add 500μl of overnight culture to 500μl of 50% glycerol in the 1.5ml tube and gently mix</LI><LI>Freeze the glycerol stock tube at -80°C. The stock is now available for years as long as its kept at -80°C.</LI><LI>To remove bacteria from the glycerol stock, open the tube and use a sterile tip to scrape some of the frozen bacteria.
                                                </LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#gRNA purification"><DIV class="card-header"><H3 class="card-link">
                                              gRNA purification (ZYMO Research RNA Clean &amp; Concentrator™-5 Kit)
                                              </H3></DIV></A><DIV id="gRNA purification" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">Purification of newly transcribed crRNA (T7 RNA polymerase; Promega1), following the ZYMO Research RNA purification kit (RNA Clean &amp; Concentrator™-5) protocol [2].</P><H4 class="text-muted">Materials</H4><H5> Purification (RNA Clean &amp; Concentrator™-5; ZYMO Research) </H5><U1><LI>RNA Binding Buffer</LI><LI>RNA Prep Buffer</LI><LI>RNA Wash Buffer</LI><LI>DNase I</LI><LI>DNA Digestion Buffer</LI><LI>DNase/RNase Free Water</LI><LI>Zymo Spin IC Columns</LI><LI>Collection Tubes</LI><LI>RNase-free Microfuge Tubes (1.5 mL) Not provided with the kit.</LI><H4 class="text-muted">Procedure </H4><H5>DNase I treatment (Before clean-up)</H5><OL><LI><B>IMPORTANT</B>: Prior to use, reconstitute the lyophilized DNase I as indicated on the vial. Store frozen aliquots. </LI><LI>For each sample to be treated, prepare DNase I reaction mix in an RNase-free tube (not provided). Mix well by gentle inversion, once the following components added (volume of components to add to the RNA sample can be
                                                      readjusted according to the sample's volume)</LI><CENTER><TABLE class="tg"><TBODY><TR><TH class="tg-0lax">Product</TH><TH class="tg-0lax">Volume </TH></TR><TR><TD class="tg-0lax">RNA sample (≤10 μg) volume adjusted with water or TE buffer</TD><TD class="tg-0lax">40 μl</TD></TR><TR><TD class="tg-0lax">DNase I</TD><TD class="tg-0lax">5 μl</TD></TR><TR><TD class="tg-0lax">DNA Digestion Buffer</TD><TD class="tg-0lax">5 μl</TD></TR><TR><TD class="tg-0lax">Total volume</TD><TD class="tg-0lax">50 μl</TD></TR></TBODY></TABLE></CENTER><LI>Incubate at room temperature (20-30ºC) for 15 minutes.</LI></OL><H5> Buffer preparaiton</H5><OL><LI>Before starting, add 48 ml 100% ethanol (52ml 95% ethanol) to the 12 ml RNA Wash Buffer concentrate(R1013, R1015) or 96 ml 100% ethanol (104ml of 95% ethanol) to the 24 ml RNA Wash Buffer concentrate (R1014, R1016).</LI></OL><H5>Wash</H5><P>All centrifugation steps should be performed at <U>10,000 –16,000 x g. </U>RNA species ≥17 nt will be recovered. </P><OL><LI>Add 2 volumes RNA Binding Buffer to each sample and mix. Example: Mix 100 μl buffer and 50 μl sample.</LI><LI>Add an equal volume of ethanol (95-100%) and mix. Example: Add 150 μl ethanol.</LI><LI>Transfer the sample to the Zymo-Spin™IC Column in a Collection Tube and centrifuge for 30 seconds. Discard the flow-through.</LI><LI>Add 400 μl RNA Prep Buffer to the column and centrifuge for 30 seconds. Discard the flow-through.</LI><LI>Add 700μl RNA Wash Buffer to the column and centrifuge for 30 seconds. Discard the flow-through.</LI><LI>Add 400 μl RNA Wash Buffer to the column and centrifuge for 2 minutes to ensure complete removal of the wash buffer. Transfer the column carefully into an RNase-free tube (not provided in the kit).</LI></OL><H5>Elution</H5><OL>
                                                    Add 15μl DNase/RNase-Free Water directly to the column matrix and centrifuge for 30 seconds. (Alternatively, for highly concentratedRNA use ≥ 6μl elution).The eluted RNA can be used immediately or stored at -70°C.
                                                  </OL><H4 class="text-muted">References </H4><OL><LI>Promega: &quot;Synthesis of Nonlabeled RNA&quot; protocol, https://www.promega.com/-/media/files/resources/protocols/product-information-sheets/n/t7-rna-polymerase-protocol.pdf</LI><LI>ZYMO Research: RNA Clean &amp; Concentrator™-5, Instruction Manual, https://www.zymoresearch.eu/media/amasty/amfile/attach/_R1013_R1014_R1015_R1016_RNA_Clean_Concentrator-5_ver2.2.1.pdf</LI></OL></U1></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#inoculating"><DIV class="card-header"><H3 class="card-link">
                                                Inoculating cultures
                                                </H3></DIV></A><DIV id="inoculating" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">This protocol explains how to inoculate cultures to grow bacterial clones.</P><H4 class="text-muted">Materials</H4><UL><LI>LB ampicillin plates from our transformation</LI><LI>LB ampicillin medium</LI><LI>14ml sterile round tubes with dual position snap cap</LI><LI>sterile tips</LI><LI>shaker at 37°C</LI></UL><H4 class="text-muted">Procedure</H4><OL><LI>Pick a colony from the ligation plate using a sterile tip</LI><LI> Shake the tip into a bacterial culture tube containing 3ml of LB/Amp medium so the colony mixes with the medium
                                                    </LI><LI>Close tubes (loose position for sterile aerobic culturing)</LI><LI>Put your tubes onto a shaker at 37°C and incubate overnight with agitation at 225 rpm</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Lysate_preparation"><DIV class="card-header"><H3 class="card-link">           Lysate preparation
                                                  </H3></DIV></A><DIV id="Lysate_preparation" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead"> Lysate is an essential component of cell-free expression reactions. 
This protocol is taken from the iGEM EPFL 2017 wiki. </P><H4 class="text-muted">Materials</H4><UL><LI>Lysate</LI><LI> Autoclave</LI><LI>Autoclaved LB medium</LI><LI>Autoclaved tips and dishes Flame</LI><LI>Buffer A</LI><LI>Buffer A + DTT</LI><LI>Tubes for bacterial culture</LI><LI>Spectrophotometer</LI><LI>Plastic cuvettes</LI><LI>50 ml Falcon tubes</LI><LI>Liquid nitrogen</LI><LI>Big and small centrifuge Sonicator</LI><LI>IPTG (0.1 M: 0.2383 g of IPTG which Mw = 238.3 g/mol, and fill up to 10 mL with deionized water)</LI></UL><H4 class="text-muted">Procedure</H4><H4>Preparation of Materials (For 2 x 200 ml of bacterial culture)</H4><OL><LI>Prepare minimum 420 ml of LB medium and autoclave for 20 min at 121ºC </LI><LI> Autoclave tips and dishes, which will be used for bacteria growth</LI><LI> Prepare minimum 120 ml buffer A (- DTT)</LI><LI> Prepare minimum 3 ml buffer A + DTT</LI></OL><H4>Bacteria growth</H4><P class="lead">
                                                    When working with bacteria, always work next to the flame to avoid contaminations and wear glasses when working with liquid nitrogen.
                                                    
                                                  </P><OL><LI> At 17h approximatively, add 5 ml of LB medium and antibiotics according to resistance’s cells to a tube and inoculate with a small amount of desired bacteria from the glycerol stock (pick by tip or inoculating loop). Put the top of the bottle of the LB and the cap under the flame to avoid contamination of LB medium (contaminate very easily). (Note: No antibiotics is used when preparing BL21 DE3 lysates)</LI><LI> Grow the culture overnight at 37°C, 200rpm, pay attention to keep the lid loose to allow for air exchange.</LI><LI> At 9h approximately, measure absorbance at 600 nm with 10X dilution of the culture (900 μl of LB + 100 μl of bacteria culture) in a plastic cuvette, OD600 should be around 4.</LI><LI> Add 1 ml overnight culture to 200 ml of LB medium and correct antibiotics in a 500 ml Erlenmeyer flask.</LI><LI> Incubate the culture for 2 hours at 37°C, 200 rpm (pay attention to the rpm setting – this speed is optimal for the growth).</LI><LI> Induce if necessary: 400 μl of 100 mM IPTG for T7 polymerase inducing; 5 ml of 10% arabinose for gamS inducing (under pBAD promoter).</LI><LI> Incubate again the culture for 2 hours at 37°C, 20 0rpm and measure absorbance, OD600 should be around 1.5 – 2.</LI></OL><H4>Centrifugation and cleaning</H4><OL><LI> Let the big centrifuge cool down to 4ºC (takes around 10 min).</LI><LI> Weigh one 50 ml falcon tube for one culture (clearly label the tube as well the cap).</LI><LI> Separate the grown culture to four 50 ml falcon tubes and spin at 4000 rpm at 4°C for 20 min.</LI><LI> Keep the bacteria on ice as much as possible.</LI><LI> Discard the supernatant and add 10 ml of buffer A to the bacteria pellet, re-suspend the bacteria by pipetting or by vortexing and transfer all four parts to one tube, spin at 4000 rpm at 4°C for 10 min.</LI><LI> Discard the supernatant and re-suspend the pellet in 10 ml of buffer A, spin at 4000 rpm at 4°C for 10 min.</LI><LI> Re-do the washing step 6 one more times.</LI><LI> Discard the supernatant (remove as much as possible).</LI><LI> Weigh the pellet (wet mass).</LI><LI> Wear protective glasses (danger of tube explosion) and flash freeze with liquid nitrogen, store at –80°C.</LI></OL><H4>Sonication</H4><OL><LI> Re-suspended the thawed bacteria based on their mass in buffer A by vortexing (1ml of buffer A + DTT per grams of thawed bacteria).</LI><LI> Keep the bacteria or lysate on ice as much as possible.</LI><LI> Aliquot 1 ml of re-suspended bacteria to a new 2 ml Eppendorf tube.</LI><LI> Place the tube in an ice bath (ice mixed with water) and place the sonicated tip inside the tube so it is immersed as much as possible in the liquid without touching the surface of the tube.</LI><LI> Sonicate with 50% amplitude and pulsing 10 s:10 s (energy : pause) until you reach 400 J (around 1 min and 24 s).</LI></OL><H4>Separation</H4><OL><LI> Let the small centrifuge cool down to 4ºC (takes around 20 min).</LI><LI>Centrifuge the lysed bacteria at 12000 rpm and 4°C for 10 min. </LI><LI> Transfer the supernatant (top layer) to a new tube. To prevent any transfer of bacteria debris, do not take all the lysate.</LI><LI> Place the lysate to an incubator at 37 °C and 200 rpm for 1h30 min, this run off reaction will lead to degradation of the rest of the DNA.</LI><LI> Centrifuge the run off reaction at 12000 rpm at 4°C for 10 min.</LI><LI> Transfer the supernatant to a new tube. As before, to prevent any transfer of bacteria debris, do not take all the lysate.</LI><LI> Aliquot the lysate to 0.5 ml microfuge tubes according to your needs (around 25 μl is recommended) and keep 1 μl of the lysate for the Bradford assay.</LI><LI> Flash freeze the aliquots in liquid nitrogen and store at -80°C (wear glasses).</LI></OL><H4>Bradford assay</H4><OL><LI> Perform the Bradford assay to determine the protein concentration: dilute 1 μl of the lysate in 99 μl of buffer A</LI><LI> Mix 5 μl of the diluted lysate with 250 μl of the Bradford reagent.</LI><LI> Let the reaction incubate for 5 min</LI><LI> Measure the absorbance using the Nanodrop device, protein concentration should be between 400 and 800 μg/ml</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Microplate"><DIV class="card-header"><H3 class="card-link">
                                                  Microplate reader cell-free reaction (circular DNA template) 
                                                  </H3></DIV></A><DIV id="Microplate" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">In order to test the efficiency of the different components of our cell-free reactions, we perform microplate reader cell-free expression reactions with GFP DNA template. We use the fluorescence emitted by the GFP as a way to measure protein expression. We used this kind of experiments as an internal reference to assess in a relative manner the quality of our lysates and energy solutions.
                                                    It is important to test only one component of the cell-free reaction at a time. We will perform 10µl reactions with 4 repeats of each reaction type. It is crucial to do a negative control per reaction type (we also suggest to do 4 repeats for each negative control). </P><H4 class="text-muted">Materials</H4><UL><LI>Lysate</LI><LI><A data-toggle="collapse" href="#Energy"><SPAN style="color:blue">Energy solution</SPAN></A></LI><LI><A data-toggle="collapse" href="#Buffer"><SPAN style="color:blue">Buffer A</SPAN></A></LI><LI>GFP DNA circular template (10ng/µl of reaction)</LI><LI>Eppendorf tubes</LI><LI>Micro-centrifuge</LI><LI>Mircoplate reader Victor X3 from PerkinElmer</LI><LI>384 Nunc plate from ThermoFisher</LI><LI>Chillout Liquid Wax from BioRad</LI></UL><H4 class="text-muted">Procedure</H4><H4>Microplate reader settings</H4><OL><LI>Start preheating the machine to 29°C </LI><LI> Configure the machine for GFP measurement.
                                                      </LI><LI>Set the parameters as follows: 300 measurement, 90s and quick shaking between each measurement (this corresponds to around 7.5 h expression) </LI></OL><H4>Master mixes preparation</H4><P class="lead">
                                                    We prepare 2 master mixes: Master Mix 1 (MM1) containing lysate and buffer A, and Master Mix 2 (MM2) containing energy solution, water and DNA (or just energy solution and water for the negative controls).
                                                    
                                                    We calculated the volumes needed based on the following 10µl cell-free expression reaction:
                                                  </P><UL><LI> 2.5µl Lysate</LI><LI>2.5µl Energy Solution (ES)</LI><LI>2.5µl Buffer A</LI><LI>100ng of DNA</LI><LI> Nuclease free water (complete for a total volume of 10µl)</LI></UL><P class="lead">For each reaction type, we suggest to prepare enough master mixes for 4,5 reactions because of pipetting errors.</P><CENTER><TABLE><TBODY><TR><TH>Lysate</TH><TH>Buffer A</TH></TR><TR><TD>11.25µl</TD><TD>11.25µl</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR><TH>ES</TH><TH>DNA</TH><TH>Nuclease free water</TH></TR><TR><TD>11.25µl</TD><TD>450ng of DNA (volume is plasmid stock solution concentration dependant)</TD><TD>complete to a total volume of 45µl</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR><TH>ES</TH><TH>Nuclease free water</TH></TR><TR><TD>11.25µl</TD><TD>11.25µl</TD></TR></TBODY></TABLE></CENTER><OL start="4"><LI>Put 5µl of MM1 in one corner of each well</LI><LI>Put 5µl of either MM2+ or MM2- in the opposite corner of each well.</LI><LI>Centrifuge the plate for a short time (30-60 seconds) at 4000 rpm to remove any bubbles andcollect the solutions at the bottom of the wells.</LI><LI>Add 35 μl of wax to each well.</LI><LI>Load the plate and start the program.</LI><DIV class="card"><A data-toggle="collapse" href="#oligo"><DIV class="card-header"><H3 class="card-link">
                                                Oligomer Phosphorylation
                                                  </H3></DIV></A><DIV id="oligo" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">This protocol is used to phosphorylate the 5' ends of inserts used in a subsequent Golden Gate ligation reaction</P><H4 class="text-muted">Materials</H4><UL><LI>Forward Oligo 100 μM</LI><LI>Reverse Oligo 100 μM</LI><LI>T4 DNA Ligase Buffer 10X</LI><LI>PNK</LI><LI>NFW</LI><LI>NaCl 2M aqueous solution</LI></UL><H4 class="text-muted">Procedure</H4><OL><LI>In a PCR tube mix the following (total volume 29 μL):</LI><UL><LI>3 μL Forward Oligo 100 μM</LI><LI>3 μL Reverse Oligo 100 μM</LI><LI>3 μL T4 DNA Ligase Buffer 10X</LI><LI>2 μL PNK</LI><LI>18 μL water</LI></UL><LI>Incubate the mixture for 2 hours at 37C</LI><LI>Heat inactivate PNK at 65C for 20 minutes</LI><LI>Add 1 μL of 2 M NaCl aqueous solution</LI><LI>Heat to 98C for 2 minutes then slowly ramp down to room temperature and hold at 4C when finished</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Resuspension"><DIV class="card-header"><H3 class="card-link">
                                                      Oligonucleotides/gBlocks Resuspension and Storage (IDT)
                                                    </H3></DIV></A><DIV id="Resuspension" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">Recommendations on how to resuspend and store oligos or/and gBlocks gene fragments (IDT) once shipped. Check IDT's page: &quot;My oligos have arrived: Now what?&quot;1 for more details. </P><H4 class="text-muted">Materials</H4><UL><LI>Oligos or gBlocks gene fragments</LI><LI>Nuclease-free water/TE buffer </LI></UL><H4 class="text-muted">Procedure</H4><OL><LI>Briefly centrifuge the tubes before opening them</LI><LI>the oligos should be resuspended in TE buffer (10 mM Tris, 0.1 mM EDTA, pH 8.0). Nuclease-free water (pH 7.0) may be used alternatively. However, use of HPLC- or molecular biology–grade water is preferable. <B>CAUTION</B>:
                                                          Nuclease-free water will not modulate pH over time as will TE buffer.</LI><LI>Standard recommendation: Resuspend oligos to a 100 µM stock concentration, The volume of TE buffer required to achieve a 100 µM stock is easily determined by multiplying the number of nanomoles (nmol) listed for a
                                                          particular oligo by a factor of 10, and then resuspending the dry DNA in that same number of microliters of TE buffer. For example, if the oligo specification sheet states that 20.3 nmol of oligo were delivered, add 203 µL TE
                                                          buffer to obtain a 100 µM stock solution. This stock solution can be diluted as needed into appropriate working solutions.</LI><LI>For oligos that are harder to resuspend, and for which one might observe residual precipitate present following resuspension, the oligo should be heated at 55°C for 1–5 minutes, vortexed thoroughly, and then briefly
                                                          centrifuged. </LI></OL><H5>Storage</H5><UL><LI>Store your resuspended oligonucleotides at -20°C (stable for at least 24 months when either dried down, or resuspended in TE buffer or nuclease-free water).</LI></UL><H4 class="text-muted">References</H4><UL><LI>[1] Integrated DNA Technologies (IDT): &quot;My oligos have arrived: Now what?&quot;, https://eu.idtdna.com/pages/education/decoded/article/my-oligos-have-arrived-now-what-; &quot;Tips for resuspending and diluting your oligonucleotides&quot;,
                                                          https://eu.idtdna.com/pages/education/decoded/article/tips-for-resuspending-and-diluting-your-oligonucleotides</LI></UL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#Plasma PCR"><DIV class="card-header"><H3 class="card-link">
                                                      PCR amplification in Plasma
                                                      </H3></DIV></A><DIV id="Plasma PCR" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">This is a protocol on how to amplify DNA fragments in plasma. Since we are going to target specific DNA fragments with our CRSPR-cas12a assay following this PCR we will add ourselves the oligos in our sample. If
                                                          you want to amplify DNA that is already contained in your sample skip the additon of DNA in plasma. This protocol is based on the one found in PCR protocols by Professor Kenji Abe1. </P><H4 class="text-muted">Materials</H4><UL><LI>DNA template</LI><LI>PCR primers </LI><LI>dNTPs (10 mM)</LI><LI>Nuclease-Free water</LI><LI>5X Phusion HF buffer</LI><LI>Phusion DNA polymerase</LI><LI>Human Blood Plasma</LI><LI>10x Phosphate-buffered Saline (PBS)</LI><LI>Thermal cycler</LI></UL><H4 class="text-muted">Procedure</H4><OL><LI>Plasma samples will be dilutated 1:5 in 1x PBS but as not to dilute the plasma too much we will proceed by adding the DNA template in PBS in the following way</LI><LI>Make a mastermix of 1 part plasma (4μl) in 3 part PBS diluted as following: 1.6μl 10X PBS in 10.4μl NFW</LI><P>We want the PBS to be 1X in 4 part of the sample (16μl) but one part is used to put the templated. Once we add the last part the PBS will become 1X.</P><LI>Put 8μl of the master mix in two different tubes and make the two following samples:</LI><CENTER><TABLE class="tg"><TBODY><TR><TH class="tg-0lax">Components</TH><TH class="tg-0lax">Sample A</TH><TH class="tg-0lax">Negative control </TH></TR><TR><TD class="tg-0lax">Template</TD><TD class="tg-0lax">1µl</TD><TD class="tg-0lax">-</TD></TR><TR><TD class="tg-0lax">Nuclease free water</TD><TD class="tg-0lax">1μl</TD><TD class="tg-0lax">2µl</TD></TR><TR><TD class="tg-0lax">Total</TD><TD class="tg-0lax">2μl</TD><TD class="tg-0lax">2µl</TD></TR></TBODY></TABLE></CENTER><LI>The diluted plasma sample are heated for 3 min at 95℃ then cooled rapidly on ice for 3 to 5 min.</LI><LI>Make the following MasterMix</LI><CENTER><TABLE class="tg"><TBODY><TR><TH class="tg-fymr">Components</TH><TH class="tg-fymr">A</TH><TH class="tg-fymr">Negative control</TH><TH class="tg-fymr">MasterMix 2.5x</TH><TH class="tg-fymr">Final concentration</TH></TR><TR><TD class="tg-fymr">NFW</TD><TD class="tg-0pky">28.5</TD><TD class="tg-0pky">28.5</TD><TD class="tg-0pky">71.25</TD><TD class="tg-0pky">-</TD></TR><TR><TD class="tg-fymr">5X Phusion HF buffer</TD><TD class="tg-0pky">10</TD><TD class="tg-0pky">10</TD><TD class="tg-0pky">25</TD><TD class="tg-0pky">1X</TD></TR><TR><TD class="tg-fymr">10 mM dNTPs</TD><TD class="tg-0pky">1</TD><TD class="tg-0pky">1</TD><TD class="tg-0pky">2.5</TD><TD class="tg-0pky">200 µM</TD></TR><TR><TD class="tg-fymr">Forward primer (10 μM)</TD><TD class="tg-0pky">2.5</TD><TD class="tg-0pky">2.5</TD><TD class="tg-0pky">6.25</TD><TD class="tg-0pky">0.5 µM</TD></TR><TR><TD class="tg-fymr">Reverse primer (10μM)</TD><TD class="tg-0pky">2.5</TD><TD class="tg-0pky">2.5</TD><TD class="tg-0pky">6.25</TD><TD class="tg-0pky">0.5 µM</TD></TR><TR><TD class="tg-fymr">From Master mix</TD><TD class="tg-0pky">44.5</TD><TD class="tg-0pky">44.5</TD><TD class="tg-0pky">111.25</TD><TD class="tg-0pky">-</TD></TR><TR><TD class="tg-fymr">Sample A</TD><TD class="tg-0pky">-</TD><TD class="tg-0pky">5</TD><TD class="tg-0pky">-</TD><TD class="tg-0pky">-</TD></TR><TR><TD class="tg-fymr">Negative control</TD><TD class="tg-0pky">5</TD><TD class="tg-0pky">-</TD><TD class="tg-0pky">-</TD><TD class="tg-0pky">Variable</TD></TR><TR><TD class="tg-fymr">Phusion DNA Polymerase</TD><TD class="tg-0pky">0.5</TD><TD class="tg-0pky">0.5</TD><TD class="tg-0pky">-</TD><TD class="tg-0pky">1.0 units/50 µl PCR</TD></TR><TR><TD class="tg-fymr">Total volume</TD><TD class="tg-fymr">50</TD><TD class="tg-fymr">50</TD><TD class="tg-0pky">-</TD><TD class="tg-0pky">-</TD></TR></TBODY></TABLE></CENTER><LI>Gently mix the reaction. Transfer PCR tubes from ice to a PCR machine with the block preheated to 98 °C and begin thermocycling:</LI><CENTER><TABLE class="tg"><TBODY><TR><TH class="tg-1wig">Temperature</TH><TH class="tg-1wig">Time</TH><TH class="tg-1wig">Number of cycles</TH></TR><TR><TD class="tg-0lax">98 °C</TD><TD class="tg-0lax">30 sec (Initial Denaturation)</TD><TD class="tg-0lax">1</TD></TR><TR><TD class="tg-0lax">98 °C</TD><TD class="tg-0lax">10 sec (denaturation)</TD><TD class="tg-0lax">30 cycles</TD></TR><TR><TD class="tg-0lax">Variable</TD><TD class="tg-0lax">30 sec (primer annealing)</TD><TD class="tg-0lax">30 cycles</TD></TR><TR><TD class="tg-0lax">72 °C</TD><TD class="tg-0lax">30 sec (extension)</TD><TD class="tg-0lax">30 cycles</TD></TR><TR><TD class="tg-0lax">72 °C</TD><TD class="tg-0lax">10 min (Final Extension)</TD><TD class="tg-0lax">1</TD></TR><TR><TD class="tg-0lax">4 °C</TD><TD class="tg-0lax">infinite</TD></TR></TBODY></TABLE></CENTER></OL><H4 class="text-muted">References</H4><OL><LI>1 Abe, Kenji. « Direct PCR from Serum ». PCR Protocols, edited by John M. S. Bartlett and David Stirling, Humana Press, 2003, p. 161‑66. Springer Link, doi:10.1385/1-59259-384-4:161</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#PCR Phusion"><DIV class="card-header"><H3 class="card-link">
                                                        PCR using Phusion® High-Fidelity DNA Polymerase
                                                        </H3></DIV></A><DIV id="PCR Phusion" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead"> A standard PCR protocol using Phusion high fidelity DNA polymerase. Mainly used for amplifying the junction/point mutated fragments we aim to detect using CRISPR-Cas12a. The protocol is based on NEB PCR Protocol
                                                            for Phusion® High-Fidelity DNA Polymerase (M0530)1.</P><H4 class="text-muted">Materials</H4><UL><LI>DNA template </LI><LI>Forward primer</LI><LI>Reverse primer</LI><LI>dNTPs, 10 mM</LI><LI>Nuclease-Free water</LI><LI>5X Phusion high fidelity (HF) buffer</LI><LI>Phusion DNA polymerase (ThermoFisher, 2 U/µl)</LI><LI>Thermal cycler</LI></UL><H4 class="text-muted">Procedure</H4><OL><LI>The thermal cycler (PCR machine) should be programmed as following. <U>IMPORTANT</U>: Annealing temperature may vary depending on the primer's melting temperature (Tm).</LI><CENTER><TABLE class="tg"><TBODY><TR><TH class="tg-1wig">Step</TH><TH class="tg-1wig">Temperature</TH><TH class="tg-1wig">Time</TH><TH class="tg-1wig">Cycles</TH></TR><TR><TD class="tg-0lax">Initial denaturation</TD><TD class="tg-0lax">98°C</TD><TD class="tg-0lax">30 sec</TD><TD class="tg-0lax">-</TD></TR><TR><TD class="tg-0lax">Denaturation</TD><TD class="tg-0lax">98°C</TD><TD class="tg-0lax">10 sec</TD><TD class="tg-0lax">35</TD></TR><TR><TD class="tg-0lax">Primer annealing</TD><TD class="tg-0lax">45-72°C (depending on primers' Tm)</TD><TD class="tg-0lax">30 sec</TD><TD class="tg-0lax">35</TD></TR><TR><TD class="tg-0lax">Extension</TD><TD class="tg-0lax">72°C</TD><TD class="tg-0lax">30 sec</TD><TD class="tg-0lax">35</TD></TR><TR><TD class="tg-0lax">Final extension</TD><TD class="tg-0lax">72°C</TD><TD class="tg-0lax">10 min</TD><TD class="tg-0lax">-</TD></TR><TR><TD class="tg-0lax">-</TD><TD class="tg-0lax">4°C</TD><TD class="tg-0lax">-</TD><TD class="tg-0lax">-</TD></TR></TBODY></TABLE></CENTER><LI>For one sample, add the following amounts of products. If working with several samples, a master mix containing the elements that do not change between each sample (water, buffer, dNTPs or/and primers/DNA) can be prepared.
                                                              However, ALWAYS add the enzyme in the end.</LI><CENTER><TABLE class="tg"><TBODY><TR><TH class="tg-1wig">Component</TH><TH class="tg-1wig">Volume µl</TH><TH class="tg-1wig">Final concentration</TH></TR><TR><TD class="tg-1wig">Nuclease-free water</TD><TD class="tg-0lax">Up to 50 µl</TD><TD class="tg-0lax">-</TD></TR><TR><TD class="tg-1wig">5X Phusion HF buffer</TD><TD class="tg-0lax">10</TD><TD class="tg-0lax">1X</TD></TR><TR><TD class="tg-1wig">10 mM dNTPs</TD><TD class="tg-0lax">1</TD><TD class="tg-0lax">200 µM</TD></TR><TR><TD class="tg-1wig">10 µM Forward primer</TD><TD class="tg-0lax">2.5</TD><TD class="tg-0lax">0.5 µM</TD></TR><TR><TD class="tg-1wig">10 µM Reverse primer</TD><TD class="tg-0lax">2.5</TD><TD class="tg-0lax">0.5 µM</TD></TR><TR><TD class="tg-1wig">Template DNA</TD><TD class="tg-0lax">variable</TD><TD class="tg-0lax">Up to 250 ng</TD></TR><TR><TD class="tg-1wig">Phusion DNA Polymerase</TD><TD class="tg-0lax">0.5</TD><TD class="tg-0lax">1.0 units/50 µl PCR</TD></TR><TR><TD class="tg-1wig">Total volume</TD><TD class="tg-1wig">50</TD><TD class="tg-1wig">-</TD></TR></TBODY></TABLE></CENTER><LI>Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary. Transfer PCR tubes from ice to a PCR machine (programmed previously) and begin thermocycling.</LI><LI>Control your samples by doing an agarose gel electrophoresis following the Agarose gel electrophoresis protocol.</LI></OL><H4 class="text-muted">References</H4><UL><LI>1. New England BioLabs (NEB). PCR Protocol for Phusion® High-Fidelity DNA Polymerase (M0530): https://international.neb.com/Protocols/0001/01/01/pcr-protocol-m0530</LI></UL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#ProbesPreparation"><DIV class="card-header"><H3 class="card-link">
                                                          Preparation of dumbbell probes
                                                          </H3></DIV></A><DIV id="ProbesPreparation" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">The goal is to prepare dumbbell probes in order to amplify miRNAs by Rolling Circle Amplification (RCA).</P><H4 class="text-muted">Materials</H4><UL><LI>2 μL DNA template</LI><LI>1μL <A href="https://www.neb.com/products/m0201-t4-polynucleotide-kinase#Other%20Tools%20&amp;%20Resources" target="_blank">T4 polynucleotide kinase</A></LI><LI>1 μL ( 100 U/μL ) <A href="https://www.neb.com/products/m0202-t4-dna-ligase#Product%20Information" target="_blank">T4 ligase</A></LI><LI>4 μL T4 DNA ligase reaction buffer (x10) (2μL for the phosphorylation and 2μL for the ligation)
                                                                </LI><LI> 400 mM Tris-HCl, 100 mM MgCl2, 100 mM Dithiothreitol, 5 mM ATP, pH 7.8 at 25 °C</LI></UL><LI> 22 μL DEPC-treated H2O (15 μL for phosphorylation and 7 for ligation)</LI><LI>Exonuclease I (20 U/μL) and <A href="https://www.neb.com/products/m0206-exonuclease-iii-e-coli#Product%20Information" target="_blank">Exonuclease III (100 U/μL)</A></LI><H4 class="text-muted">Procedure</H4><OL><H4>Phosphorylation of the oligos</H4><LI>In the IDT tubes, put the amount of water to get 100μM of DNA probe [you take the number of moles N and suspend in a volume of 10*N μl]</LI><LI>In a tube, put 2 μL oligos, 15 μL of water, the T4 ligase buffer (2 μl), and finally 1 μL of the kinase.</LI><LI>Incubate the mixture at 37°C for 1 hour.</LI><LI>Heat at 60°C for 20 min to inactivate the enzyme. <P class="lead"><STRONG>The buffer has to be new ( less than 1 year) and we should avoid repeated freeze-thaw cycle with it.</STRONG></P></LI><H4 class="text-muted">Ligation of the probes</H4><LI>Add in a reaction tube 10 μl (because at the end of phoshorylation is 10 μM and not 100 μM) of DNA template, the T4 ligase (1 μl), the reaction buffer (2 μl) and 7 μl DEPC-treated water.</LI><LI>Put the tube at 16°C for 2 hours to process the ligation.</LI><LI>Then heat at 65°C for 10 min to terminate the reaction.</LI><LI>Add the exonucleases (1μl each - total volume of 22 μl) and incubate the reaction mixture at 37°C for 2 hours.</LI><LI>Then the enzymes are denatured by heating at 80°C for 20 min.</LI><LI>The ligation can be controlled by electrophoresis on agarose gel (1.5%).</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#RT-RCA"><DIV class="card-header"><H3 class="card-link">
                                                            Real-time fluorescence measurement of RCA
                                                            </H3></DIV></A><DIV id="RT-RCA" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">The goal is to make sure that the RCA worked fine for different concentration of probes and miRNAs. The initial assessement was done with 0.5% agarose gel but the results were difficult to interpret. We are using 25x
                                                                SYBR Green I, an intercalating
                                                                agent for dsDNAs that is also fluorescent (Excitation wavelength is 494 nm, emission wavelength is 521nm).</P><H4 class="text-muted">Materials</H4><UL><LI>1 μL prepared probes</LI><LI>2.5μL phi29 DNA polymerase reaction buffer (x10)</LI><P class="lead">500 mM Tris-HCl, pH 7.5 at 25°C, 100 mM MgCl2, 100 mM (NH4)2SO4, 40 mM Dithiothreitol</P><LI><A href="https://www.neb.com/products/b9000-bsa-molecular-biology-grade#Product%20Information">0.25 μL BSA (20 mg/mL)</A></LI><LI><A href="https://www.neb.com/products/n0447-deoxynucleotide-dntp-solution-mix#Product%20Information">6 μL dNTPs(10 mM for each)</A></LI><LI>2.5 μL of target miRNA solution</LI><LI>11.75 μL DEPC-treated H2O</LI><LI><A href="https://www.neb.com/products/m0269-phi29-dna-polymerase#Product%20Information">0.5 μL phi29 DNA polymerase (10 U/μL)</A></LI><LI>0.5 μl 25x SYBR Green I </LI></UL><H4 class="text-muted">Procedure</H4><P class="lead"><STRONG>Fluorescence measurement using SYBR Green I</STRONG></P><P class="lead"><OL><LI>The SYBR Green I that we purchased is optimised for Gel and so it was very concentrated (x10000) so we have to dilute it to 25x with the polymerase buffer.</LI><LI>Put all the components in a tube</LI><LI>Inject 24 (25) μl in a 96-wells plate and put it in the plate reader.</LI><LI>The reaction should be at 37°C. The florescence is measured every 2 min during 180 min under excitation and emission wavelengths of 495(497) and 515(520) nm, respectively.</LI></OL></P></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#RCA"><DIV class="card-header"><H3 class="card-link">
                                                                  Rolling Circle Amplification
                                                                  </H3></DIV></A><DIV id="RCA" class="collapse"><DIV class="card-body"><H4 class="text-muted">Introduction</H4><P class="lead">The goal here is to amplify miRNAs by RCA (Rolling circle amplification). The dumbbell probes are designed in order to get a complementary region with specific miRNAs. The miRNAs bind to this region and the probes
                                                                      become circular and
                                                                      the amplification can begin. We finally obtain a concatemer (long continuous DNA molecule that contains multiple copies of the same DNA sequence linked in series).</P><H4 class="text-muted">Materials</H4><UL><LI>1 μL prepared probes</LI><LI>2.5μL phi29 DNA polymerase reaction buffer (x10)</LI><UL><LI> 500 mM Tris-HCl, pH 7.5 at 25°C, 100 mM MgCl2, 100 mM (NH4)2SO4, 40 mM Dithiothreitol</LI></UL><LI>0.25 μL <A href="https://www.neb.com/products/b9000-bsa-molecular-biology-grade#Product%20Information" target="_blank">BSA</A> (20 mg/mL)</LI><LI>6 μL <A href="https://www.neb.com/products/n0447-deoxynucleotide-dntp-solution-mix#Product%20Information" target="_blank">dNTPs</A>(10 mM for each)</LI><LI>2.5 μL of target miRNA solution</LI><LI>12.25 μL DEPC-treated H2O</LI><LI>0.5 μL <A href="https://www.neb.com/products/m0269-phi29-dna-polymerase#Product%20Information" target="_blank">phi29 DNA polymerase </A>(10 U/μL)</LI><LI>2 μL <A href="https://www.thermofisher.com/order/catalog/product/S7585" target="_blank">SYBR I</A>SYBR I (x10)</LI></UL><H4 class="text-muted">Procedure</H4><OL><H4>Amplification of the miRNAs</H4><LI>Add all the component, except the last one in a 25μL mixture tube.</LI><LI>Incubate the mixture at 37°C for 2h</LI><LI>Heat it at 65°C for 10 min to stop the reaction.</LI><LI>The mixture can be analysed by using electrophoresis or fluorescence analysis.</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#SDSPage"><DIV class="card-header"><H3 class="card-link">
                                                                    SDS-PAGE for protein electrophoresis
                                                                    </H3></DIV></A><DIV id="SDSPage" class="collapse"><DIV class="card-body"><H2 id="SDSPage">SDS-PAGE for protein electrophoresis</H2><H4 class="text-muted">Introduction</H4><P class="lead">PAGE (polyacrylamide gel electrophoresis) is a technique allowing to separate charged molecule according to their molecular masses. SDS-PAGE (sodium dodecyl sulfate–polyacrylamide gel electrophoresis) is a variant
                                                                      of PAGE allowing to
                                                                      separate protein molecules according to their molecular masses. SDS (sodium dodecylsulphate) is a negatively charged molecule which will bind to proteins as they are heat denatured and confers them a charge nearly proportional to
                                                                      their
                                                                      length (and hence their mass). It therefore allows to separate proteins according to their molecular masses.</P><H4 class="text-muted">Materials</H4><UL><LI>Laemmli buffer (SDS-PAGE loading buffer, contains SDS and DTT)</LI><LI>Commercial polyacrylamide gels in glass plates</LI><LI>PAGE machine</LI><LI>Running buffer (1X TGS buffer) </LI><LI>Long pipet tips </LI></UL><H4 class="text-muted">Procedure</H4><OL><H5>Sample preparation</H5><LI>Mix 10µl of Laemmli with 5µl of protein sample and 5µl nuclease free water in a tube. Take care to mix well the sample tube before by flicking it gently several times.</LI><LI>Incubate at 100°C for 15min in a dry heating block compatible with the tube used (e.g. for PCR tubes use the ThermoCycler). Do not forget to use lids heating at 105°C to avoid condensation.</LI><LI>The samples can now be stored at 4°C until they are loaded on the gel.</LI><H5>PAGE-machine preparation</H5><LI>Verify that you have all the machine's components (tank, lid with power cables, electrode assembly, cell buffer dam, casting frames and stands) as well as a new polyacrylamide gel in glass plates.</LI><LI>Rince all the components with distilled water.</LI><LI>Open the gel and remove the plastic lid that protects the bottom of the gel.</LI><LI>Open the casting frame and put the gel inside. Close it again.</LI><LI>Test waterproofness by puting new running buffer inside (here it is important to use new running buffer!). Do it above the tank (in case it is not water proof).</LI><LI>Put the casting frame in the tank. Take care of the electrodes' colors! The black wire should be above the black electrode, same for the read one.</LI><LI>Fill the part of the tank outside of the casting stand with running buffer (you can put already used running buffer here.) You should fill it until the black line drawn on the tank (the 2 gels line if you run with only 1
                                                                        casting stand, the 4 gel one if you run with 2 casting stands.)</LI><LI>Close the tank with the lid with powe cables.</LI><H5>Loading the gel</H5><LI>Use long tips! Take care to pipet really slow in order to have the right volume. Long tips are very thin and tend not to fill completely.</LI><LI>Load 10µl of sample or 5µl of ladder per well.</LI><LI>Go to the glass plate in front of you and try to put your tip a bit in the well. Once you think you are in, try to go back and forth. If your land in the middle of the buffer, it means that you were not really in the well!
                                                                        You should only be able to move between the 2 glass plates in which your gel is. Once your in, dive a bit deeper in the well and load the content of your pipet.</LI><LI>Tip: put one ladder and one negative control per gel!</LI><H5>PAGE run</H5><LI>Set the machine on 30min and 120V. You can make sure the machine is running by looking below the gel. You should see bubbles forming.</LI><I>If you do not see any bubbles, this means that the electric circuit is not closed. This could be due to the oversight to remove the plastic lid at the bottom of the gel or to the fact that there is not enough buffer in the
                                                                        tank or between the 2 gels. Sometimes it is necessary to put more buffer than the level indicated on the tank.</I><LI>After 5min of run, set the voltage on 200V. </LI><H5>Gel wash</H5><LI>Take the casting stand out and put the now used running buffer in a bottle. You can reuse it multiple times.</LI><LI>Put the running buffer of the tank in the same bottle.</LI><LI>Remove the gel of the casting frame and open it carefully using the specialized metal instrument (or just a metallic spatule if you do not have one).</LI><LI>Put the gel with distilled water in a cylinder bowl that you can close. In order to transfer the gel from the glass plate to the bowl without breaking it, you can flip</LI><LI>Pour distilled water in order to fill the bowl up to 4cm.</LI><LI>Mix a bit and throw the water away. You can use your fingers to prevent the gel to fall in the sink.</LI><LI>Pour distilled water in the bowl and put it on the shaker for 10min.</LI><H5>Coomassie staining</H5><LI>Put on the shaker for 1h.</LI><LI>Wash twice with distilled water and recover the gel with distilled water. Let on the shaker overnight.</LI><H5>Gel visualisation</H5><LI>Your gel is now ready to be visualized. To take it out of the bowl, it is better that it is not at the bottom of the bowl. You can pour some water at one border of the bowl in order to make it float a bit and then slip your
                                                                        hand (with clean gloves!) underneath. Then put the gel in a plastic sleeve.</LI><LI>In order to visualize the gel, you can use a scanner (gives the best quality), or the white light mode of an UV transilluminator.</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#heat"><DIV class="card-header"><H3 class="card-link">
                                                                    Standard heat purification for proteins
                                                                    </H3></DIV></A><DIV id="heat" class="collapse"><DIV class="card-body"><H2 id="heat"><U>Standard heat purification for proteins</U></H2><H4 class="text-muted">Introduction</H4><P class="lead">When the protein of interest is heat stable, the heat purification method is a straightforward way to get rid of the majority of non-desired proteins of a sample.</P><H4 class="text-muted">Materials</H4><UL><LI>Protein samples to purify</LI><LI>Eppendorf tubes</LI><LI>Heating block compatible to the tubes used</LI><LI>Ice</LI><LI>Microcentrifuge</LI></UL><H4 class="text-muted">Procedure </H4><UL><LI>Denaturation
                                                                          </LI><LI>Heat at 70ºC for 20 min.</LI><LI>Put on ice for 15 min.</LI><LI>Centrifugation
                                                                          </LI><LI>Centrifuge at 12000 g for 10 min.</LI><I>The protein of interest is now mainly located in the supernatant of the solution.</I></UL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#DCpresentation"><DIV class="card-header"><H3 class="card-link">
                                                                    Vaccine Presentation to Immature Dendritic Cells
                                                                    </H3></DIV></A><DIV id="DCpresentation" class="collapse"><DIV class="card-body"><H2 id="heat"><U>Vaccine Presentation to Immature Dendritic Cells</U></H2><H4 class="text-muted">Introduction</H4><P class="lead">This protocol uses murine dendritic cells extracted from bone marrow for vaccine representation.</P><H4 class="text-muted">Materials</H4><UL><LI>Immature murine bone marrow dendritic cells</LI><LI>RPMI medium </LI><LI>RPMI (with 2 ng/ml of GD-CSF)</LI><LI>24 well plate</LI><LI>Conical Tubes</LI></UL><H4 class="text-muted">Procedure </H4><H5>Cell Counting and Resuspention</H5><OL><LI>Cells are supplied in a plate. There are usually around 6 million cells in 10 ml of RPMI in a plate.</LI><LI>Collect the cells and transfet them into a 50 ml conical tube.</LI><LI>Resuspend into a new concentration of 1 million cells per ml pf RPMI after centrifugation and cell counting.</LI></OL><H5>Cell Plating</H5><OL><LI>Add 1 ml of RPMI ( with 2 ng/ml GF-CSF) into each well of a 24 well plate</LI><LI>Pipette 0.5 ml of the cell stock (at 1 million cell/ml) to each well</LI><LI>Add to each well the different condition (Negative Control, Vaccine,...). The volume should range between 50 and 100 µl.</LI><LI>Even out the mixture by gently shaking the well.</LI><LI>Incubate for 18-24 hrs at 37 ºC.</LI><LI>The cells are now ready for Antibody Staining for FACS</LI></OL></DIV></DIV><DIV class="card"><A data-toggle="collapse" href="#3D"><DIV class="card-header"><H3 class="card-link">
                                                                    3D culture of dendritic cells in Matrigel

                                                            </H3></DIV></A><DIV id="3D" class="collapse"><DIV class="card-body"><H2 id="heat"><U>3D culture of dendritic cells in Matrigel</U></H2><H4 class="text-muted">Introduction</H4><P class="lead">This protocol is based on Corning’s protocol: Corning® Matrigel® Basement Membrane Matrix for 3D Culture In Vitro.
</P><H4 class="text-muted">Materials</H4><UL><LI>Immature murine Bone Marrow Dendritic Cells C57BL/6 </LI><LI>Corning Matrigel matrix (Corning Cat. No. 356234)</LI><LI>RPMI medium (with GM-CSF at 2ng/ml)</LI><LI>PBS</LI><LI>Ibidi µ-Dish 35 mm, low</LI><LI>Expressed and purified fluorescent Encapsulin</LI></UL><H4 class="text-muted">Procedure </H4><OL><LI>Thaw Matrigel matrix overnight by submerging the vial in a 4°C refrigerator before use. Once Matrigel matrix is thawed, swirl vial to ensure the material is dispersed.</LI><LI>Resuspend the immature dendritic cells to a concentration of 3.81*10^5 cells/ml in RPMI (with 2 ng/ml).</LI><LI>Add 100 µl of Matrigel to 250 µl of the dendritic cell suspension, and mix well.</LI><LI>Plate onto the ibidi µ-Dish 35 mm, low.</LI><LI>Incubate at 37 ºC for 20 min.</LI><LI>Bring the dish outside for imaging.</LI><LI>Present the fluorescent encapsulin to the dendritic cell by adding 50 µl of the expressed and purified fluorescent encapsulin.</LI><LI>Put back to the 37ºC incubator.</LI><LI>One hour post presentation, bring the dish outside for imaging, then put the dish back  in the incubator.</LI><LI>Four hours post presentation, bring the dish out for imaging, then put the dish back into the 37 ºC incubator.</LI><LI>Six hours post presentation, bring the dish out for washing. Wash twice by tilting the dish and pipetting 1 ml of PBS onto the higher side of the dish, so that the PBS runs through the Matrigel. Collect the wash on the other side of the dish.</LI><LI>Do imaging post wash.</LI></OL>
                                                                        Note that the microscope used with this protocol is NanoLive’s precision Tomographic Microscope. This technology allowed us to make 3D images and 2D fluorescence images of the cells.
                                                                         
                                                          </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></ARTICLE></DIV></DIV></DIV></SECTION></MAIN><DIV class="container"><DIV class="align-items-center"><DIV class="text-center"><DIV><SPAN class="pl-3 d-inline-block">iGEM2018@groupes.epfl.ch</SPAN></DIV><SPAN class="d-block mt-4">IGEM EPFL 2018</SPAN><A href="https://twitter.com/EPFL_iGEM" data-fancybox="" class="btn btn-twitter btn-icon-label btn-circle"><SPAN class="btn-inner--text">Twitter</SPAN></A><A href="https://www.facebook.com/igemepfl/?ref=br_rs" data-fancybox="" class="btn btn-facebook btn-icon-label btn-circle"><SPAN class="btn-inner--text">Facebook</SPAN></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>