Team:EPFL/Notebook-Detection
iGEM EPFL 2018
iGEM EPFL 2018
HOME
Project
Description
Take a closer look at CAPOEIRA
Design
Learn more about how we designed CAPOEIRA
Software
Explore the Bioinformatic Pipeline implemented in CAPOEIRA
Results
Discover the Results of our experiments
Demonstration
Demonstration of how our Therapeutic System works
Interlab
Learn more about our participation in the Interlab Study
Parts
Discover the Parts that make up CAPOEIRA
Improve
Explore our Novel and Improvements Parts
Measurement
Learn about measurement approaches we used for Part Characterization
Team
Team Members
Learn more about our Team Members
Collaborations
Discover the interactions we had with other iGEM Teams during CAPOEIRA
Human Practices
Integrated Human Practices
Discover how we integrated Human Practices into the Purpose, Design and Implementation of CAPOEIRA
Ethics
Explore the ethical considerations of CAPOEIRA
Applied Design
Explore how our therapeutic system attempts to meet real world medical need
Education and Public Engagement
Discover our Education and Public Engagement work with the public and scientific communities
European and Eurasian Meetup
Learn about our interactions with the iGEM Community during the Meetups
Lab
Safety
Entrepreneurship
Acknowledgements
Attributions
Sponsors
Partners
Awards
Index
Notebook - Follow-up: miRNA
This page collects the notebooks of the Follow-up part of our project specifically dedicated to miRNA. The experiments are sorted according to the three main subparts of this section, and then by week (inside each             subpart).
Probe preparation and Rolling Circle Amplification
SYBR Green I assay
Cas12a assay
Probe preparation and Rolling Circle Amplification
This part comprises all the steps from the phosporylation of the linearized probes, to the ligation and the Rolling Circle Amplification (RCA). More information is available on the protocols of probe preparation and RCA .
Index
Week 3 (23/07/18 - 29/07/18)
Week 4 (30/07/18 - 05/08/18)
Week 6 (13/08/18 - 19/08/18)
Week 8 (27/08/18 - 02/09/18)
References
Week 3 (23/07/18 - 29/07/18)
THURSDAY, 26/07/2018
Phosphorylation and ligation of Probe 2
Aim:
The aim is to prepare the probes by phosphorylating and ligating of the oligos we received from IDT. We first start to test probe 2, that we know from literature is correctly amplified, to verify that the protocol is               working.               Then, we plan to repeat both steps also for the other probes.
Description:
The oligos we ordered arrived dried, so we needed to resuspend them to 100 μM by using DEPC-treated water.
We first decided to make the phosphorylation and the ligation with probe 2, because among the ones proposed by Qiu et al. , it was proved to be the most efficient               one with respect to amplification.
The probes were ordered from IDT as non-phosphorylated oligos, because they were significantly less expensive than 5'-phosphorylated oligos with the same sequence. As a consequence, in order to achieve the ligation, we               will always have to first phosphorylate our oligonucleotides with T4 Polynucleotide Kinase (PNK).
The sequence of the probe 2 is: 5'-ACCTCATTGTATAGCCCCCCCCTGAGGTAGTAGGTTGCCCAACTATACAACCTACT-3'.
During this step, we used (and we will have to use for every phosphorylation) T4 ligase buffer - that contains ATP - instead of PNK buffer, because the nucleotide is required to transfer a phosphate group to the 5' end of               the oligo. We point out that the phosphorylation step cannot be verified e.g. by means of an agarose gel               because of course the phosphorylated and non-phosphorylated oligos actually have the same length.
The ligation was done with T4 DNA ligase, and all the non-ligated probes were digested by using Exonuclease I and Exonuclease III, two 3'->5' exonucleases. The complete protocol for the preparation of the probes is shown here .
We decided to put the exonucleases without any buffer, as suggested in the protocol from Qiu et al. . Nonetheless, we were not sure whether the activity of the               proteins was sufficient to cleave all the non-ligated oligos. Therefore, we decided to assess our sample on a 1.5% agarose gel (100 V, 30 min) together with a negative control, as described in the table below:
Sample
T4 ligase
Exonucleases
Ligated probe 2
+
+
Negative control
-
-
If a band at 56bp (size of the probe) is also shown in the lane for the negative control it means that the Exonucleases did not work fine and that the protocol needs to be modified.
The agarose gel also provides information on the phosphorylation: indeed, if no any band is shown for our sample either, it means the oligo was not ligated and that the phosphorylation did not succeed.
Results:
Agarose gel for the ligation of the probe 2.
One band was shown for the sample where we add both ligase and exonucleases. We cannot tell the exact size of this band because the shortest band of our ladder was 100bp long. Nonetheless, from this result we can be               confident that the band is indeed the ligated probe 2.
Moreover, for the negative control, no band was shown in the gel, meaning that all the non-ligated oligos were actually digested by the enzymes.
Discussion
We can see that 1μl of Exonuclease I and III cleave all the non-ligated probes (100μM) in the second sample. As a consequence, we can conclude that the amount of nucleases we put is sufficient.
Also, the results confirm that the phosphorylation and the ligation worked fine.
SATURDAY, 28/07/18
Rolling Circle Amplification of probe 2 - Trial 1
Aim:
The aim is to amplify the probe 2 by using different concentration of "miRNA" : 10μM, 1μM, 100nM, 10nM. The amplification is then checked by using an agarose gel.
Description:
As of today, we have not received the RNA oligonucleotide for let-7a yet. Nonetheless, we also ordered a DNA oligo with same sequence as let-7a (but with "thymine" instead of "uracil"): for the first part of our project,               we will work with this DNA oligo instead of the actual RNA sequence. The idea is to check whether the amplification also works with DNA, which could be highly convenient as a proof of concept because of the high prices of RNA               oligonucleotides (10-fold more expensive               than a DNA oligonucleotide with the same sequence).
Moreover, our idea is also validated from literature, since Rauzan et al. have shown the thermodynamics of the DNA/DNA duplex formation is similar to RNA/DNA               duplex.
The DNA "miRNA" oligonucleotide (later also referred to as "fake miRNA") was first of all resuspended to 100 μM by adding Ultra-pure water. We then performed serial dilutions from the DNA solution stock to 10 μM, 1μM,               100nM and 10nM.
After, we performed Rolling Circle Amplification of Probe 2 as from the protocol, with 10 nM fake miRNA. The reaction product was then tested by means of a 0.7% agarose gel (100V, 40 min)
Results:
Rolling Circle Amplification of probe 2, with 1μM of the DNA oligo for let-7a.
Discussion:
The results are not so clear but it appears that there is a band in the well, due to the size of the amplicons. We will perform the same experiment again so as to be sure of the results.
SUNDAY, 29/07/18
Rolling Circle Amplification of probe 2 - Trial 2
Description:
We performed the Rolling Circle Amplification again for different concentrations of fake miRNA (DNA) solution, namely 10nM, 100nM, 1μM and 10μM.
Results:
Rolling Circle Amplification of probe 2, with different concentrations of the DNA oligo for let-7a.
The results might be not straightforward to interpret. Indeed, as a product of the RCA reaction we expect to have a very large amplicon (roughly 70 kb long): because of its big size, such amplicon can therefore migrate                 very slowly in the gel and remain very close to the wells. This is indeed what we mostly can observe here, with very thin bands near the wells. In addition, we also notice smears for the samples with higher concentration                 of miRNA.
Discussion:
Again, due to the high size of the amplicon, agarose gels might not be the most suitable assay to definitively verify that the amplification worked fine for the different concentration of probes, even if our gel seems                 to show                 consistent bands very close to the wells, as expected.
Moreover, our gel is sufficiently comparable to gels shown by other groups working on Rolling Circle Amplification, as the one presented below:
Agarose gel analysis of RCA product, indicated by the arrow. Reproduced from Takahashi et al. (Figure 4C).
Week 4 (30/07/18 - 05/08/18)
MONDAY, 30/07/18
Preparation of probes from 3 to 10
Aim:
The aim is to prepare the other probes, from 3 to 10, now that we know that the phosphorylation and the ligation work. The preparation of probe 1 will be done in the next days.
Description:
After following the protocols for ligation and phosphorylation, we tested the different probes by means of a 1.5% agarose gel (100V, 30 min). The negative control was not done here.
Results:
Agarose gel after ligation for probes from 3 to 10
As expected we can observe that there is a band for all the probes. For probe 3, the band has a very low intensity. Also, the band for probes 6 and 9 seems to be more smeared and to extend up to a higher size (>100bp).
Discussion:
In general, we were able to show the all the probes from 3 to 10 could successfully be ligated. The low intensity of probe 3 means that the ligation yielded a low amount of properly-ligated probes. For what concerns               instead probes 6 and 9, both of them have a G-rich loop which would explain the unexpected size: the smear might indeed be due to the formation of duplexes of misligated probes. We will therefore not use probes 6 and 9               for the Cas12a assay.
TUESDAY, 31/07/18
Rolling Circle Amplification of probes 3 and 4
Today we proceeded to amplify the probes 3 and 4 and to test whether the RCA reaction could work well enough with different concentrations of miRNA (DNA) : 1μM, 100nM, 10nM, 1nM. We ran a 0.7% agarose gel (at 120 V, 45               min), but unfortunately we broke the gel prior to put it in the UV transilluminator. As a consequence, we need to make the electrophoresis again tomorrow, to be sure that the amplification actually worked.               By putting a sufficient amount of RCA product in the wells of the gel we actually expect to see a smear for a very wide range of sizes, because the size of the amplicon should depend on the time when the duplex phi29-probe is formed               (if we do not assume that all the duplexes are formed at the same time).
WEDNESDAY, 01/08/18
Rolling Circle Amplification of probes 3 and 4
Aim:
The aim is to perform the RCA of the other probes, in order to be sure that the results we got for probe 2 can be replicated also with other probes. We first amplified probes 3 and 4.
Description:
As usual, we followed the protocol for RCA, then verified the products by means of a 0.5% agarose gel (100V, 1h). We tested two different ladders (2-log purple ladder - highest band at 1000kb, on the right in the image -,               and 100bp ladder - left in the image), because the first ladder was expired and so we were not sure if it could be               used.
Results:
Rolling Circle amplification of the probe 3 and 4.
It seems again that there are some bands very close to the well for both of the probes and for the different concentration of let-7a (DNA).
Discussion:
We can now be rather confident the RCA works for these probes. A more accurate way to analyze the RCA products would be to use the intercalating dye SYBR Green I; nonetheless, this reagent is very expensive and therefore               we do not plan to buy it for the moment.
THURSDAY, 02/08/18
Rolling Circle Amplification of probes 5 to 10
Aim:
The aim is to make the amplification of the other probes, this time probes from 5 to 10.
Description:
The assessment of the experiment has also been done with a 0.5% gel. As our electrophoresis device is small we could only use the small comb that produces only ten wells. As a consequence, we had to reduce the number of               different let-7a DNA               concentrations and we could not represent the results for all the probes in one gel.
Results:
Agarose gel for the RCA products for different concentrations of let-7a DNA, with probes 5-6-7-8 (on the left) and probes 9 and 10 (on the right).
Discussion:
Also in this case the results suggest that the amplification for probes 5 to 9 was successful. Surprisingly, a band was also shown for probe 10, which was not expected, since probe 10 was designed to incorporate a               mismatch with respect to the               miRNA sequence; this implies that a non-complete match between the probe and the miRNA is sufficient to trigger amplification. This might be a positive finding (implying the need of less stringent constraints on the probe design), but               also needs carefully considerations: the               probes need indeed to be sensitive enough to bind only to the target miRNA and not also to other miRNAs with similar sequences.
Week 6 (13/08/18 - 19/08/18)
TUESDAY, 14/08/18
Rolling Circle Amplification of probe 1
Aim:
We want to check if the amplification works also with probe 1, which is, among the probes we designed ourselves, the one which we believe might lead to the best result.
Description:
We did the phosphorylation and the ligation of probe 1 but we decided not to make another gel because we already validated our protocol multiple times.
It has to be noted that it is the last time we will use let-7a DNA. Indeed, we finally received our RNA oligonucleotide and we also want to test if our RCA protocol works also with actual RNA.
The gel analysis was performed with a 0.5 % agarose at 120V for 75min.
Results:
Agarose gel for the RCA product of probe 1
Discussion:
The image is very unclear, probably also because of a mistake in the UV exposure of the gel. Furthermore, we wanted to run the gel for a longer time to see whether the amplicons could travel in the gel further from the               wells. This nonetheless almost resulted in the removal of the ladder from the gel (we can only see the larger bands), but did not have a positive effect on the amplicons.
Week 8 (27/08/18 - 02/09/18)
TUESDAY, 28/08/18
Rolling Circle Amplification using miRNA :
Aim:
The goal is to make the amplification using the actual RNA let-7a oligonucleotide instead of its DNA copy.
Description:
Now that we received let-7a under RNA form, we can start to work with it and to make sure that the amplification also works with actual RNA. We decided to assess the amplification using probe 1, as it is the probe that               we currently               use for Cas12a assay.
The gel is as usual a 0.5% agarose gel at 100V for 60min.
As we use RNA now, we have to be careful and to work in a RNase Free Zone.
Results:
Agarose gel of the RCA products of probe 1 with different concentrations of miRNA
We were able to see bands in the wells, but as usual the results were not very clear: in particular, we were able to see faded bands close to the well for all the four concentrations, even if they were brighter for the               two highest concentrations.
Discussion:
Even if the results are not very accurate, we can, by comparing this gel with the ones from our previous experiments, qualitatively infer that the amplification works with miRNA as well. Once more, this confirms to us               that agarose gels are not the most suitable assay to test whether RCA works, and that we might indeed need to buy SYBR Green I to have more quantitative information on the RCA products.
WEDNESDAY, 29/08/18
Today we first of all proceeded to aliquote the different dilutions of the let-7a: the 100 μL tubes with each of the four meaningful concentrations (1 μM, 100 nM, 10 nM and 1 nM) were divided into 3 smaller tubes of 20               μL, with the remaining               40 μL in the original tubes. All the tubes were stored back at -65°C, except for 1 small 1 μM one which was partly directly used for experiment of today.
Rolling Circle Amplification with different incubation times.
Aim:
The aim was to have further confirmation of the amplification being working, in addition to the gels from the previous experiments. We want to prepare different samples that have the same probe and the same amount of RNA,               but we will vary the incubation               time (i.e. the step of the protocol at 37°C, before the denaturation step at 65°C), from 5 min to 2h.
Description:
In this experiment we wanted to repeat the amplification with probe 1, but with different incubation times, in order to check on the gel whether shifted bands appeared (to have a further proof of the RCA being working).               We therefore               tested RCA for the 1 μM let-7a in 4 different tubes, incubating those latter respectively for 2 hours, 1 hour, 25 minutes and 5 minutes (just pausing and resuming the ThermoCycler to insert the samples), with of course also the final               10 min 65°C denaturation               step. We then ran a gel of the obtained amplicons with the standard parameters (0.7%, 120 V, 40 minutes).
Results:
Agarose gel of RCA products (probe 1, 1 μM let-7a) with different incubation times
We can see smears for all the lanes, from roughly 50bp (not precise because our ladder is adapted to high sizes of DNA) to a very high size (that we cannot determine with our ladder) for 1h and 2h incubation times.
We can also see a large band at 25 min (the size cannot be determined accurately because of the exponential behaviour of the ladder) and bright bands in the wells for 1h and 2h, as well as other bands of various sizes.
Discussion:
First of all, we can say that the amplification is initiated pretty quickly because we can see a smear already at 5 min. Also, in this lane, there is a pretty bright band at 2kb. The rate of the phi29 polymerase is 2280               nt/min at 30°C (the rate               at 37°C is not provided by NEB but we can assume that it is around 2000 nt/min): therefore, we can conclude that the binding of miRNA to the probe and the binding of the polymerase to the miRNA-probe complex occur within very little               time from each other.
These results give further proof that the amplification actually worked properly.
Conclusion for this part
For the moment, the different assays that we performed allow us to qualitatively infer that our protocol for Rolling Circle Amplification actually works with our probes. Nonetheless, real-time measurements of SYBR Green I               fluorescence activity might be needed in order to quantitatively verify the RCA products.
References
Qiu, Xin-Yuan, et al. "Highly Effective and Low-Cost MicroRNA Detection with CRISPR-Cas9." ACS synthetic biology, 7.3 (2018):                     807-813.
Rauzan, Brittany, et al. "Kinetics and thermodynamics of DNA, RNA, and hybrid duplex formation." Biochemistry, 52.5 (2013): 765-772.
Takahashi, Hirokazu, et al. "RNase H-assisted RNA-primed rolling circle amplification for targeted RNA sequence detection." Scientific                       reports, 8.1 (2018): 7770.
SYBR Green I assay
This part deals with the analysis of the RCA reaction products by means of the real-time measurements of the fluorescent signal from SYBR Green I. The detailed protocol of the real-time fluorescence measurement of RCA is             described here. The notebook starts at Week 9 because we only started to work on this assay at the beginning of September.
Index
Week 10 (10/09/18 - 16/09/18)
Week 12 (10/09/18 - 16/09/18)
References
Week 10 (10/09/18 - 16/09/18)
MONDAY, 10/09/2018
Planning of the experiment
We should receive SYBR Green today so we plan to make the experiment this afternoon. In the first experiment, we want to see if the amplification of probe 1 is working, for a certain concentration of miRNA.(and probe?):
The positive control is the amplicon from Reza's group. We know that their molecules are dsDNAs.
The negative control should be the probe 1 without miRNA. As the probe is supposed to have a double stranded part, we should see a small signal compared to the amplicon that has a lot of stem part. We also really need to dilute the               concentration of the probe because it can lead to a saturation, just like with our cas12a assays.
TUESDAY, 11/09/2018
First trial
Aim:
We want to performed a real-time RCA with probe 1 directly in the plate reader.
Description:
We tested 5 samples:
1) 5 µM probe and 10 nM miRNA;
2) 5 µM probe and 1 nM miRNA;
3) 5 µM probe and no miRNA [negative control];
4) 1 µM probe and 10 nM miRNA;
5) 1 µM probe and no miRNA [negative               control].
We also considered a positive control, consisting in a sample (tube "E") from the ctDNA group which we were sure contained a high amount of double-stranded DNA: this sample was added in the same quantity as the probe,               i.e. we put 1 µL of it inside the the 25.5 µL reaction. The protocol is described here:
As a note, we erroneously did not consider any blank for this experiment; moreover, due to some trouble in setting up the experiment, the measurements were started around 10 minutes after the preparation of the plate.
Results:
Discussion:
Overall it seems we obtained the expected order among the different samples, even though the errors are rather high to confirm this. Nonetheless there is a significant difference between the samples with miRNA and those               without, which seems to confirm that the amplification is actually occurring.
We can also observe two behaviours which we wouldn't expect:
- The signal seems to decrease after reaching a peak, whereas we would expect a continuous rise corresponding to an increasing concentration of amplicon. Some of the hypotheses might include bleaching or exhaustion of Sybr Green I.
- We would expect a clear increase of the signal from a "start" level, which we don't clearly see here. As a note we had some trouble in setting the machine, which delayed the start of the measurement by around 10 minutes. This might               anyway be not sufficient to explain this behaviour.
WEDNESDAY, 12/09/2018
Amplification with different probes and with different concentrations of miRNA
Aim:
The goal is to investigate both the amplification and the amplification with different concentrations of miRNA.
Description:
The protocol is described here:
Results:
Discussion:
As we can clearly observe, the order of magnitude of the results has completely changed. Two parameters were changed compared to Trial 1: first, the plate format had been erroneously set to "ProxiPlate 384" in the first               Trial, and was now correctly changed to "OptiPlate 384; secondly, the Lamp Energy was set to 0, whereas it should have been instead around 14000.
Week 12 (24/09/18 - 30/09/18)
TUESDAY, 25/09/2018
Repetition of real-time RCA
Aim:
The aim is to repeat a real time RCA after repeating every sample from the beginning (from the phosphorylation of the probes, taking care of also re-doing the dilution of let-7a - and of (fake) let-7d).
Description:
The table with all the samples is the following one:
p
RCA 10 n
RCA 1nM
RCA 100pM
RCA 10pM
RCA 1 pM
RCA 100 fM
Control RCA (no probe)
RCA 1 nM (let 7d)
N*
W
P
NF water
12.25
12.25
12.25
12.25
12.25
12.25
14.75
12.25
15.25
15.75
14.25
dNTPs
6
6
6
6
6
6
6
6
6
6
6
Phi-29 buffer
2.5
2.5
2.5
2.5
2.5
2.5
2.5
2.5
3
3
3
Probe
1
1
1
1
1
1
1
1
-
-
-
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
/
/
/
Double-stranded product for positive (10 uM)
-
-
-
-
-
-
-
-
-
-
1
miRNA
2.5
2.5
2.5
2.5
2.5
2.5
-
2.5
-
-
-
(let7a, 10nM)
(let7a, 1nM)
(let7a, 100pM)
(let7a, 10pM)
(let7a, 1pM)
(let7a, 100fM)
-
(let7d, 1nM)
/
/
/
BSA
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
SYBR Green I
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
-
0.5
Phi-29 polymerase
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
-
-
-
Final volume
25.5
25.5
25.5
25.5
25.5
25.5
25.5
25.5
25
25
25
Two replicas of 12 uL each were used for each sample.
Results:
Discussion:
The RCA works fine, at least the 140 first minutes. We get the right order in the concentrations of let-7a. With this test, it seems that the detection limit is 100 pM let-7a.
SATURDAY, 29/09/2018
Real-time RCA with different concentration of probes
Aim:
The goal is to perform a new real-time RCA, titrating the probe from 100nM to 100pM
Description:
The idea is to understand whether there is a concentration of probe sufficiently high to trigger RCA but also sufficiently low to not cause a significant activation in the Cas assay.
p
1a
1b
1c
2a
2b
2c
3a
3b
3c
4a
4b
4c
N*
W
P
NF water
11.75
11.75
14.25
11.75
11.75
14.25
11.75
11.75
14.25
11.75
11.75
14.25
15.25
15.75
14.25
dNTPs
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
Phi-29 buffer
2.5
2.5
2.5
2.5
2.5
2.5
2.5
2.5
2.5
2.5
2.5
2.5
3
3
3
Probe
1
1
1
1
1
1
1
1
1
1
1
1
-
-
-
(Probe 1, 100nM)
(Probe 1, 100nM)
(Probe 1, 100nM)
(Probe 1, 10nM)
(Probe 1, 10nM)
(Probe 1, 10nM)
(Probe 1, 1nM)
(Probe 1, 1nM)
(Probe 1, 1nM)
(Probe 1, 100pM)
(Probe 1, 100pM)
(Probe 1, 100pM)
/
/
/
Double-stranded product for positive (10 uM)
-
-
-
-
-
-
-
-
-
-
-
-
-
-
1
miRNA
2.5
2.5
-
2.5
2.5
-
2.5
2.5
-
2.5
2.5
-
-
-
-
(let7a, 10nM)
(let7a, 100pM)
-
(let7a, 10nM)
(let7a, 100pM)
-
(let7a, 10nM)
(let7a, 100pM)
-
(let7a, 10nM)
(let7a, 100pM)
-
/
/
/
BSA
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
0.25
SYBR Green I
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
-
0.5
Phi-29 polymerase
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
-
-
-
Final volume
25
25
25
25
25
25
25
25
25
25
25
25
25
25
25
Results:
Discussion:
We can see that,for a concentration of 10 nM let-7a, decreasing the concentration of the probes induces a reduction of  the signal. This indicates that the RCA is less efficient, and even not efficient at all for concentrations below 10 nM.
References
Zipper, Hubert, et al. "Investigations on DNA intercalation and surface binding by SYBR Green I, its structure determination and methodological implications."Nucleic acids research, 32.12 (2004):                     e103-e103.
"SYBR Green II RNA Gel Stain" - Sigma-Aldrich. Datasheet.
Vitzthum, Frank, et al. "A quantitative fluorescence-based microplate assay for the determination of double-stranded DNA using SYBR Green I and a standard ultraviolet transilluminator gel imaging system." Analytical biochemistry , 276.1 (1999): 59-64.
"SYBR Green I nucleic acid gel stain" - Sigma-Aldrich. Datasheet.
Cas12a assay
This part describes the experiments we performed with our Cas12a system to detect the concentration of the amplicons. The detailed protocol of the Cas12a detection assay is described here. The notebook starts at Week 6             because from Week 3 to Week 5 we aimed to optimize the amplification step.
Index
Week 6 (13/08/18 - 19/08/18)
Week 7 (20/08/18 - 26/08/18)
Week 8 (27/08/18 - 02/09/18)
Week 9 (03/09/18 - 09/09/18)
Week 10 (10/09/18 - 16/09/18)
Week 11 (17/09/18 - 23/09/18)
Week 12 (24/09/18 - 30/09/18)
Week 13 (01/10/18 - 07/10/18)
Week 6 (13/08/18 - 19/08/18)
WEDNESDAY, 15/08/18
Transcription of the sgRNA
Today, we received the template that we should use for transcription of crRNA. This crRNA binds to the stem part of the amplicon. We know that the target sequence is also found in the stem part of the probe, but we               believe that cas12a will recognise this part as double-stranded. As a consequence, it will require the presence of the PAM sequence which is only in the amplicon. If the protein acts as we expect, we should have an activation of cas12a               with the probes only.
Aim:
The aim is to transcribe the crRNA targeting our amplicons by using T7 polymerase. Then, we plan to purify the molecules using Zymo-RNA Clean & Concentrator.
Description:
We start the transcription of the sgRNA targeting the amplicons, using the T7 polymerase. Unfortunately, we have to note some procedural mistakes we made, which hopefully should not affect the results. First and more               importantly, the annealing was not performed in the RNAase free-zone, so contaminations might have involved the "sgRNA for miRNA" and the "T7 (2G)" tube. The other two mistakes are related to the resuspension of the "sgRNA for miRNA":               the Ultrapure nuclease-free water was used instead of the Nuclease-Free Duplex Buffer, and the sample was suspended to 100 μM instead of 200 μM (as done with "T7 (2G)"). To rectify this mistakes we proceeded as follows: the 100 μL mix               of the two oligo strands, which was supposed to be composed of 50 μL of 200 μM sgRNA and 50 μL of 200 μM T7 primer - for a final concentration of 100 μM -, was instead composed of 50 μL of 100 μM sgRNA, 25 μL of 200 μM T7 primer and 25               μL of Duplex Buffer, therefore keeping the requested final concentration.               Before annealing, the mix was further diluted to 10 μM by adding 400 μL Duplex Buffer (total volume of 500 μL) [the reason will be explained after].               This mix was then annealed at 94 °C for 2 minutes, gradually cooled and then stored in ice, as from the protocol.
For the actual transcription, we wanted to have 5 μg of annealed DNA template: usually 2 μL are required in the protocol, but we decided to put 20 μL instead to "dilute" the faulty nuclease-free water used to suspend the               sgRNA [the important thing is the mass of DNA template, not the volume of its solution]. The calculations we made are the following:               MW=MW_primer+MW_sgRNA=(6.39+18.93)*103 g/mol= 25.32*103 g/mol
=> Required moles n: n=m/MW=5 μg/ (25.32*103 g mol-1) = 0.197 nmol               => Required concentration to put 20 μL: 0.197 nmol/20μL ≈ 10 μM [Hence the 10 μM we used before].               The final quantities we used for transcription are therefore the following:
Components
Volume
Transcription Optimized 5x buffer
20μl
DTT
10μl
RNasin
2.5μl
rNTP mix
20μl
DNA template(10μM)
20μl
T7 polymerase
2.5μl
Nuclease free water
25.5μl
We then incubate the resulting 100 μL mix at 37°C for roughly 3-3.5 hours. We did the rest of the protocol (changing quantities accordingly: 50 μL->125 μL). Nanodrop results: 65 ng/μL. The obtained RNA was kept overnight               at -20°C, then stored at -60 °C.
Results:
The yield is not very big so we will try to remake the transcription from scratch tomorrow.
THURSDAY, 16/08/18
Repetition of experiments from Yesterday
Aim:
The aim is to remake the transcription of the crRNA in order to get a better yield at the end. Yesterday we made mistakes that we will try not to make again.
Description:
We first took care to be in a RNase Free Zone, in order to avoid contamination.               The only change we made in the protocol was to use 5 μg of 50 μM (corresponding to about 4 μL) DNA template, instead of 10 μM (20 μL); in order to do so we had to start again from the annealing, which was performed in a total volume of               100 μL :
Components
Volume
100 μM template
50μl
200 μM T7 primer
25μl
Duplex buffer
25μl
Then, we did the transcription using the following components :
Components
Volume
Transcription Optimized 5x buffer
20μl
DTT
10μl
RNasin
2.5μl
rNTP mix
20μl
DNA template(50μM)
4μl
T7 polymerase
2μl
Nuclease free water
41.5μl
Incubation was done at 37 °C for 4 hours.
Results:
Indeed the results from the nanodrop are better: 285.3 ng/μL.
FRIDAY, 17/08/18
First cas12a assay (Trial 0)
Aim:
The goal is to test for the first time cas12a with different amplicons from probe 1, 8 and 10.
Description:
Yesterday the experiment with Cas12a for the ctDNA group did not work again. Nonetheless today we are going to try the Cas12a assay for the miRNA with 3 probes (3 amplicons): the hypothesis is that for the ctDNA group               either the crRNA design might have a problem or that the problem is in the target DNA (since based on their protocol they added 5 fold more of NTS to TS for annealing their activators).               We had the chances to test 3 amplicons, we opted for those from probes 1 (with PAM on the loop), 8 (with PAM on the stem) and 10 (with PAM on the loop and mismatch). We needed to dilute our purified crRNA to 1 μM, knowing that the               concentration (from the nanodrop) was 285.3 ng/μL= 285300 ng/ml. The MW of the sgRNA (spacer+scaffold) was calculated from its sequence and was found to be equal to 13470.9 Da [this step might be checked better], which leads to a               concentration of roughly 21 μM. The crRNA had been diluted as from the protocol for purification into 15 μL NFW and 1 μL was used for the nanodrop (->14μL left). In order to dilute 14 μL of 21 μM solution to 1 μM the total volume needs               to be 294μL -> We added 280 μL NFW to the crRNA solution.
The quantity of miRNA used to make the amplification is 1μM.
We followed the protocol carefully and we prepared a mastermix :
Components
Volume
Concentration
10X Binding buffer/Neb buffer
6 μl
1X
1 µM LbCas12a
12.5 μl
250 nM
1 µM crRNA solution
16 μl
320 nM
Nuclease-free water
16.5 μl
          -
Final volume
50 μl
          -
Unfortunately the protocol was missing the addition of 0.8 μL DNAase I to the positive control tube (P). We realized right after starting the plate reader, so we added 0.4 μL DNAase I to each of the two wells for P but,               from the preliminary feedbacks we got from the machine while it was still running, this did not seem to work. We therefore expect to have a bad positive control.
Results:
Cas assay for different probes
Note to the graph: Probe 1 = Probe 1, Probe 2 = Probe 8, Probe 3 = Probe 10
Discussion:
As we expected, the positive control did not work. As a consequence, it is difficult to say something about the results we got. The signals are pretty weak for all the samples and according to the error bars, the               difference between probe 10 and probe 1 are not really significant, which is actually a problem because we should not have amplification with the probe 10 (because of the mismatch).
Week 7 (20/08/18 - 26/08/18)
MONDAY, 20/08/18
Cas12a assay without mastermix (Trial 1)
Aim:
Last time, we did the cas12a assay and we prepared a mastermix. The aim is to remake another cas12a assay without making it.
Description:
The quantity of miRNA used to make the amplification is, as before, 1μM.
The different samples that we made are the following :
Components (μl)
A1-6
A2-6
A1-2.4
A2-2.4
A8-6
N1-6
N8-6
N1-2.4
W
P
10X Binding buffer
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3
3
3
3
3
3
3
-
-
-
1 µM crRNA
3.75
3.75
3.75
3.75
4.5
3.75
4.5
-
-
-
NF water
31.1
31.1
34.7
34.7
30.3
37.1
36.3
43.8
53.4
42.8
1 µM DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
Amplicon
6 (amplicon 1)
6 (amplicon 2)
2.4 (amplicon 1)
2.4 (amplicon 2)
6 (amplicon 8)
-
-
-
-
-
DNase I
-
-
-
-
-
-
-
-
-
1
Final volume (µl)
60
60
60
60
60
60
60
60
60
60
Results:
It has to be understood that, here, the labels refer to the probe that we used to make the amplification. We consider here that it is the amplicons that are targeted.
Discussion:
The error bars are still very high which make the difference between amplicon 1 and amplicon 2 not significant (at equal volume). More surprising, we do have a signal for the amplicon 2. Yet, the probe 2 should not be               targeted by cas12a because it is specific to cas9. If the probe has the complementary sequence with the crRNA, it should not activate the endonuclease because of the absence of the PAM sequence (TTTN in 5' end). Indeed, the crRNA was               designed to target the stem part of the amplicons. As a consequence, it should be a double-stranded part and so the presence of the PAM sequence is mandatory.               In order to make sure that cas12a only target the amplicons, we decide to make the same experiment, but with another negative control (it is just the probe and no miRNA during the amplification).
WEDNESDAY, 22/08/18
Effect of the preincubation and activation of cas12a by the probe (Trial 2)
Aim:
The first aim is here to check the effect of preincubation of the solution cas12a/crRNA/binding buffer/water and also to check if we have an activation of the probe itself, which could be very prejudiciable because it               would mean that we cannot differenciate between a patient that have no miRNA in his blood, and someone that have some.
Description:
First of all, we did two assay : one with the 30 min pre-incubation, and one with no incubation.               We also took the probe 1 only, in water (and not in phi29 buffer !) and also the probe 2 that we amplified.
Component (µl)
A1-pre(+)
A2-pre(+)
N+
A1-pre(-)
A2-pre(-)
Negative control
N*
W
P
10X Binding buffer
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3
3
3
3
3
3
-
-
-
1 µM crRNA
3.75
3.75
3.75
3.75
4.5
4.5
-
-
-
NF water
31.1
31.1
31.1
31.1
31.1
36.3
43.8
53.4
42.8
Pre-incubation
+
+
+
-
-
1 µM DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
Amplicon
2.4 (amplicon 1)
2.4 (amplicon 2)
-
2.4 (amplicon 1)
2.4 ( amplicon 2)
-
-
-
-
Probes
-
-
2.4 (probe 1)
-
-
-
-
-
-
DNase I
-
-
-
-
-
-
-
-
1
Final volume (µl)
60
60
60
60
60
60
60
60
60
Results:
Discussion:
The difference between preincubation and no preincubation is not significant . Indeed, for both the amplicon 1 and 2, the signal is similar.
Concerning the activation of probe 1, there is a non-null signal which suppose that there is an activation of the probe itself. Furthermore, there is an activation of amplicon 2 as well (like for the previous experiment). This suggests               that there is a PAM-independent activation of cas12a. It can happen when the target DNA is single-stranded.
This is actually not good at all. But, we decide to modify the crRNA so that it cannot target the probe itself. The issue is that this puts a constrains for our probes. Indeed, the current crRNA is complementary to the stem part of the               amplicon (as well as the one of the probe), and we designed the probes so that the stem part is the same (the stem part is a part of our miRNA, let7a). Now that we need to change the sequence of the crRNA, we have to make it               complementary to one of the two loops of the amplicon. Yet, these parts are different between the probes that we created.
We decide to make the crRNA that has the same sequence as the big loop of the probe 1. Thus, right now we can only work with the probe 1.
Another interesting thing is that the signal of the probe is not so high. This is maybe due to the fact that we can have a higher signal due to the buffer. This is also something that we have to check later.
FRIDAY, 24/08/18
Cas12a assay with probe 8(Trial 3)
Aim:
The goal is to check if the probe 8 works better than probe 1, and if the probe alone can activate Cas12a protein.
Description:
Components (µl)
A1µ-2.4
A1µ-0.5
A100n/6
A10n-6
N+
N*
W
P
10X Binding buffer
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
-
-
-
1 µM crRNA
4.5
4.5
4.5
4.5
4.5
-
-
-
NF water
33.15
35.05
29.55
29.55
33.15
43.8
53.4
42.8
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
Amplicon 8
2.4 (1 µM)
0.5 (1 µM)
6 (100 nM)
6 (10 nM)
-
-
-
-
Probe 8
-
-
-
-
2.4
-
-
-
DNaseI
-
-
-
-
-
-
-
1 .2
Final volume (µl)
60
60
60
60
60
60
60
60
Results:
Discussion:
Here again, the probe itself is activating the endonuclease. Nevertheless, it seems that the activation is not as important as for probe 1
Week 8 (27/08/18 - 02/09/18)
WEDNESDAY, 29/08/18
Cas12a Assay with different concentrations of miRNA (Trial 4)
Aim:
The aim is to test if we can compare the differences of signal for different concentrations of miRNA used for the amplification.
Description
We first ran some amplification, using probe 1 and different concentrations of let7-a : 1 μM, 100nM, 10 nM, and 1nM. We then did the cas12a assay using the following components :
Components(µl)
A1
A10
A100
A1000
N+
W
P
10X Binding
6.6 µl
6.6 µl
6.6 µl
6.6 µl
6.6 µl
6.6 µl
6.6 µl
Cas12
3.75µl
3.75µl
3.75µl
3.75µl
3.75 µl
-
-
1 µM crRNA
4.5µl
4.5µl
4.5µl
4.5µl
4.5µl
-
-
NF water
33.35
33.35
33.35
33.35
33.35
53.4
42.8
DNase Alert
9.6
9.6
9.6
9.6
9.6
-
9.6
Amplicon
2.2 µl (1 nM)
2.2 µl (10 nM)
2.2 µl (100 nM)
2.2 µl (1 µM)
-
-
-
Probe
-
-
-
-
2.2 µl
-
-
DNase I
-
-
-
-
-
-
1 .2µl
Final volume
60
60
60
60
60
60
60
Results:
Discussion:
It has to be noted that the different error bars are very big ( they are not shown here for visibility reasons), especially for 1 nM (roughly around 2000 AU).               As a consequence, it is very easy to say that the differences are not significant. In this situation, we cannot distinguish between 1nM of miRNA and 1 µM. This is very prejudiciable because our goal is actually to quantify the               concentration of miRNa in blood samples.               There are two possible explanations : the first one is the fact that the amplification of the probe did not work fine. This is pretty unlikely, because we saw that the amplification worked fine, thanks to the agarose gels.               The second possible explanation would be that the probe is also targeted, as it appeared in the previous experiments, and so the signal is actually saturated (the concentration of probes during the amplification is 5 µM).               The negative + is the probe alone in water. Here also, it seems that the amplification buffer play a role in the activation of cas12a or in the cleavage of DNaseAlert.
Week 9 (03/09/18 - 09/09/18)
TUESDAY, 04/09/18
Effect of the different concentrations of miRNA during amplification (Trial 5)
Aim:
Here again the aim is to dilute miRNA samples so as to get different concentrations of miRNA. Today we have decided to make the RCA with lower concentrations. That's why we diluted the stock solution to 10 nM, 1 nM, 100               pM, 10 pM and 1 pM.
Discussion:
Today we proceeded to test with the standard Cas assay the RCA products produced yesterday with low miRNA concentrations (10 nM, 1 nM, 100 pM, 10 pM and 1 pM). The protocol is explained below. No significant change was               applied to the protocol. We only produced the negative controls with the probes in a slightly different way: instead of just diluting 2.5 μL probe in 22.5 μL DEPC-treated water, we aimed at keeping also the buffer concentration the               same as the test samples with the amplicon; therefore, the dilution was 2.5 μL probe, 2.5 μL phi29 buffer, 20 μL DEPC-treated water.               We also considered one "control" (C) sample from the 10 nM RCA product, but investigating the concentrations of Cas12/crRNA used in the first trial (50 nM/ 62.5 nM) and not the ones used in the last trials (62.5 nM/ 75 nM).
Components(µl)
A10n
A1n
A100p
A10p
A1p
A1n'
N+
N+'
N*
W
P
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3
3.75
3
-
-
-
1 µM crRNA
4.5
4.5
4.5
4.5
4.5
3.75
4.5
3.75
-
-
-
NF water
33.35
33.35
33.35
33.35
33.35
34.85
33.35
34.85
43.8
53.4
42.6
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
Amplicon
2.2 (10 nM)
2.2 (1 nM)
2.2 (100 pM)
2.2 (10 pM)
2.2 (1 pM)
2.2 (1 nM)
-
-
-
-
-
Probe
-
-
-
-
-
-
2.2
2.2
-
-
-
DNase I
-
-
-
-
-
-
-
-
-
1.2 µl
Final volume
60
60
60
60
60
60
60
60
60
60
60
Results:
Discussion:
We can see that even with much lower concentrations of miRNA used during the amplification, the signal is still saturating. This is again consistent with the hypothesis that the probe is finally targeted by the cas12a. Indeed, miRNA               acts in the reaction as the limiting reagent. Therefore, at the end of the RCA, there are many probes remaining in the solution, and so cas12a can also get triggered by them, inducing the cleavage of DNaseALert and so the apparition of               a signal.
Another interesting remark to make thanks to the two graphs, is the fact that we have now a saturated signal with our new negative +, compared to the previous ones, where the signal was around 2000 AU. This might explain the role of               the amplification buffer in the cleavage of DNaseAlert or in the activation of the endonuclease.               The effect of the buffer will be studied later.
THURSDAY, 06/09/17
Negative Control Titration (1)
Aim:
The aim is to titrate the negative control for the probes 1 and 8, in order to be sure that Cas12a actually target the probes and if we can manage to reduce the signal
Description
We decided to use probe 1 and 8 :
Components (μl )
A
N1-1
N1-2
N1-3
N1-4
N8-1
N8-2
N*
W
P
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3.75
3.75
-
-
-
1 µM crRNA
4.5
4.5
4.5
4.5
4.5
4.5
4.5
-
-
-
NF water
33.35
33.35
34.05
34.55
35.05
33.35
35.05
43.8
53.4
42.6
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
Amplicon
2.2 (10 nM)
-
-
-
-
-
-
-
-
-
Probe
-
2.2
1.5
1
0.5
2.2 (probe 8)
0.5 (probe 8)
-
-
-
DNase I
-
-
-
-
-
-
-
-
-
1.2 µl
Final volume
60
60
60
60
60
60
60
60
60
60
Results:
Negation Control Titration for probe 1 and 8
Discussion:
In general, by decreasing the concentration of probes 1 and 8, we manage to reduction a little bit the signal. Furthermore, it seems that this reduction is easier with probe 8.
FRIDAY, 07/09/18
Negative Control Titration (2)
Aim:
The aim is to remake the titration by using much diluted solutions of probe 1 and 8 : 100 nM, 10 nM, 1 nM, 100 pM, 10 pM.
Description:
We are at this point pretty much convinced that the probe is completely saturating the Cas assay: indeed the concentration of probe (1 µL) we put in the RCA reaction is 5 µM, with the miRNA concentration being much                 lower (but whatever the concentration of the miRNA is the concentration of the probe is enough to saturate the Cas/crRNA complex).
We therefore proceeded to first of all make dilutions of the ligated probe 1: we started from the estimated concentration of 5 µM (since we put 2 out of 20 µL of 100 µM probe in the phosporylation reaction-> 10 µM at the end; then we                 put 10 µL out of 20 µL of 10 µM phosphorylated probe-> 5 µM); we then diluted serially to 1 µM, 100 nM, 10 nM, 1 nM, 100 pM, 10 pM and 1 pM.
Then, we aimed to prepare again the "fake" RCA products which we want to use in our negative + for the Cas assay: that is, we prepared the 25 µL RCA reaction by putting inside 1 µL of such probes, 14.75 µL DEPC-treated water, 0.25                 BSA, 3 µL phi-29 buffer and 6 µL dNTPs => So no miRNA of course (exchanged with water) and not even polymerase this time (not to waste such enzyme). We prepared these RCA products with 5 different concentrations of the 1 µL probe:                 100 nM, 10 nM, 1 nM, 100 pM, 10 pM.Our hypothesis is that we this huge dilutions we should be able not to saturate the Cas/crRNA system.
In today's protocol we also wanted to directly perform RCA with much lower concentrations of probe, trying to see if they were sufficient to trigger amplification in presence of miRNA: we therefore tested two samples, one with 100 nM                 probe and 10 nM miRNA, and the other one with 10 nM probe and 10 nM miRNA.
The components are shown in the following table :
Components (μl)
N-100nM
N-10nM
N-1 nM
N-100pM
N-10pM
A10-10
A10-100
N*
W
P
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
3.75
3.75
3.75
3.75
3.75
3.75
3.75
-
-
-
1 µM crRNA
4.5
4.5
4.5
4.5
4.5
4.5
4.5
-
-
-
NF water
33.35
33.35
33.35
33.35
33.35
33.35
33.35
43.8
53.4
42.6
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
Amplicon
-
-
-
-
-
2.2 (10 nM miRNA, 10 nM probe)
2.2 (10 nM miRNA, 100 nM probe)
-
-
-
Probe
2.2 (100 nM probe)
2.2 (10 nM probe)
2.2 (1 nM probe)
2.2 (100 pM probe)
2.2 (10 pM probe)
-
-
-
-
-
DNase I
-
-
-
-
-
-
-
-
-
1.2 µl
Final volume
60
60
60
60
60
60
60
60
60
60
Results:
Negation Control Titration for probe 1 and 8
Discussion:
By higly titrating the probe 1, the signal is significantly reduce for all the samples. On one hand, we manage to reduce the noise causing by the probe itself. On the other hand, the samples with different concentration have the same signal as the negative control. As a consequence, it seems that with low concentration of probe 1, the RCA is not achieved.
Week 10 (10/09/18 - 16/09/18)
MONDAY, 10/09/18
Negative Control Titration (3)
Aim:
The aim is to make a titration of probe 1 from 5µM to 100nM, in order to fine an optimal concentration that doesn't highly activate the endonuclease, but that activate it with 10nM let-7a.
Description:
In order to do so, we proceeded to also repeat the dilution of probes suspecting that there might have been an error in the dilution made of Friday (we had a tube also for 5 µM dilution  - which should have been the starting concentration -  so actually all the previous concentrations might have been diluted 5-10 times more than expected).
Results:
Discussion:
The results shows that whatever the concentrtion of probe 1, the negative is always above the amplicon. This might be because, at this concentration of miRNA, the effective concentration of amplicon is very high, and as there are part of it that are single-stranded, the endonuclease will also collaterally cleave it, at the expense of DNaseALert.
The initial concentration of probe that we put was 5µM. Now, we will make a Cas assay with 1µM.
TUESDAY, 11/09/18
Transcription of the old crRNA
Today we first did the trancription of our old crRNA, targeting the stem part. The protocol was followed as on thursday 16/08. The protocol has been clearly remodified to be adapted to our transcription.
Unfortunately someone used the Thermal Cycler and so I could only incubate at 37°C for 2h40 (Thomas was also incubating 3h but then he makes a increase of the temperature to 80°C for 5 min).
The purification was done and the Nanodrop measurements was done. The concentration of crRNA is 68.4ng/ul (~5uM) and the A260/A280 ratio is 2.24. The ratio is a bit above the acceptable range (2-2.2) but we considered that the               difference is not significant and that our sample is pure enough. We then dilute our sample to get a concentration of 1uM ( we added 56 uM).
THURSDAY, 13/09/18
Cas12a assay with 1μM probe and different miRNA concentrations (Trial 10)
Aim:
The aim is to make the Cas12a assay with amplicons produced by adding 1μM of probe 1 and different concentrations of miRNa : 10nM,1nM,100pM,10p and 1pM.
Description:
According to the results we obtained during the previous experiments, the signal of the negative control for 1μM of probe is lower than the signal with 5μM probe (at least at the beginning of the measurements). That is               why we decide to perform another assay with this new concentration, in order to test if we can differenciate the various concentrations of miRNA.
components (μl)
N*
A1-10n
A1-1n
A1-100p
A1-10p
A1-1p
A1-0
W
P
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
-
3.75
3.75
3.75
3.75
3.75
3.75
-
-
1 µM crRNA
-
4.5
4.5
4.5
4.5
4.5
4.5
-
-
NF water
43.8
33.35
33.35
33.35
33.35
33.35
33.35
53.4
42.6
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
Amplicon
-
2.2 (10 nM)
2.2 (1 nM)
2.2 (100 pM)
2.2 (10 pM)
2.2 (1 pM)
-
-
-
Probe
-
-
-
-
-
-
2.2
-
-
DNase I
-
-
-
-
-
-
-
-
1.2 µl
Final volume
60
60
60
60
60
60
60
60
60
Results:
Discussion:
Here, the results show that with 1µM probe 1, the signals are not saturated from the beginning. Nevertheless, in this experiment, we don't have the right order in the concentrations of miRNA.
FRIDAY, 14/09/18
Competition between the amplicon and DNaseALert (Trial 11)
Aim:
The aim is to test the hypothesis that the single-stranded parts of the amplicon represent a competitor for the DNAaseAlert, resulting in a lower fluorescence level in the presence of a higher concentration of miRNA                 (and then of amplicon).
Description:
we introduced a ssDNA not related at all with our part (more specifically a primer from the ctDNA group, not containing at all the sequence for let7-a): such a ssDNA was diluted from its stock concentration of 100 µM                 to lower dilutions (10 µM, 1 µM); then, in each case, 2.2 µL were added inside the 60 µL reaction for the Cas assay in addition to the usual 2.2 uL of the "fake" RCA product (probe with no miRNA). The additional volume of 2.2 µL was                 not carefully optimized and might be too high, considering that the concentration of the probe is instead referred to the concentration before the RCA.
Component(µL)
N*
100 µM
10 µM
1 µM
Control - probe
Control - amplicon
W
P
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
-
3.75
3.75
3.75
3.75
3.75
-
-
1 µM crRNA
-
4.5
4.5
4.5
4.5
4.5
-
-
NF water
43.8
31.15
31.15
31.15
33.35
33.35
53.4
42.6
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
Amplicon
-
-
-
-
-
2.2 (10 nM miRNA, 1 µM probe before RCA)
-
-
Probe
-
2.2 (no miRNA, 1 µM probe before RCA)
2.2 (no miRNA, 1 µM probe before RCA)
2.2 (no miRNA, 1 µM probe before RCA)
2.2 (no miRNA, 1 µM probe before RCA)
-
-
-
ssDNA
-
2.2 (100 µM)
2.2 (10 µM)
2.2 (1 µM)
-
-
-
-
DNase I
-
-
-
-
-
-
-
1.2 µl
Final volume
60
60
60
60
60
60
60
60
Results:
Discussion:
The addition of additional single-stranded DNAs doesn't significantly modify the signal. We can conclude that the hypothetical collateral effect of the amplicon is negligeable.
Week 11 (17/09/18 - 23/09/18)
TUESDAY, 18/09/18
Transcriptions of the new crRNAs (Trial 12)
Aim:
The aim is to anneal, transcribe and purify the new crRNAs.
Description:
Today we received the new crRNA :
S_1 : same sequence as the loop of the probe + a part of the stem
L_1 : only the same sequence as the loop of the probe.
Those new crRNAs are not complementary to the probe anymore so it shouldn't target it, which would lead to a reduction of the number of false positives. We now have three different crRNA, if we include the old one.               The transcription of those three crRNA were done as usual, with annealing between the IDT oligo and the T7 promoter, and the transcription using phage polymerase.               The purification were done by following Zymo RNA concentrator and clean protocol and the concentration of each RNA was calculated using Nanodrop. Please note that at this point we made a mistake because we added the elution buffer (               Nuclease Free Water) twice in S_1. But it is not really a problem because we take it into consideration during the dilution step (to 1uM).               Finally, we also needed to dilute the samples to 1uM. The volume of water required for each crRNA are written below :
We needed to dilute our purified old crRNA to 1 μM, knowing that the concentration (from the nanodrop) was 139.8 ng/μL= 139800 ng/ml. The MW of this sgRNA (spacer+scaffold) was calculated from its sequence and                 was found                 to be equal to 13470.9 Da, which leads to a concentration of roughly 10.4 μM. The crRNA had been diluted as from the protocol for purification into 15 μL NFW and 1 μL was used for the nanodrop (->14μL left). In order to dilute 14 μL                 of 10.4 μM solution to 1 μM the total volume needs to be 145.6μL -> We added 131.6 μL NFW to the old crRNA solution.
We needed to dilute our purified S_1 crRNA to 1 μM, knowing that the concentration (from the nanodrop) was 76.2 ng/μL= 76200 ng/ml. The MW of the sgRNA (spacer+scaffold) was calculated from its sequence and was found to be                 equal to                 13294.90 Da, which leads to a concentration of roughly 5.7μM. The crRNA had been diluted for purification into 30 μL NFW and 1 μL was used for the nanodrop (->29μL left). In order to dilute 29 μL of 5.7 μM solution to 1 μM the total                 volume needs to be 165.3μL -> We added 136.3 μL NFW to the S_1 crRNA solution.
We needed to dilute our purified L_1 crRNA to 1 μM, knowing that the concentration (from the nanodrop) was 220 ng/μL= 220000 ng/ml. The MW of the sgRNA (spacer+scaffold) was calculated from its sequence and was found to be                 equal to                 12025.14 Da, which leads to a concentration of roughly 18.3 μM. The crRNA had been diluted as from the protocol for purification into 15 μL NFW and 1 μL was used for the nanodrop (->14μL left). In order to dilute 14 μL of 18.3 μM                 solution to 1 μM the total volume needs to be 256.2 μL -> We added 242.2 μL NFW to the L_1 crRNA solution.
The calculation of the Molecular Weigh was done by molbiotools here .
WEDNESDAY, 19/09/18
Cas12a assay with new crRNAs
Aim:
The aim is to test the new crRNAs. The new crRNAs are not complementary to the probe anymore so we should have no signal for the negative control + (with only the probe 1).
Description:
First of all, we want to compare the signal of the two new crRNA (L_1 and S_1) with the old one by performing the Cas12a assay with 1nM amplicon and with the negative control. Then, we want to see if the probe only is                 still able to activate the endonuclease
The other goal is to see if, this time, we are able to differenciate the concentrations of miRNA. In order to perform it, we choose L_1 and perform the Cas12a assay with the following concentrations of let-7a :                 1nM, 10pM, 1pM.
Components (μl)
N*
L1-10p
L1-1p
L1-1n
L1-0
S1-1n
OLD-1n
S1-0
OLD-0
W
P
10X Binding
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12
-
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
-
-
1 µM crRNA
-
4.5 (L_1)
4.5 (L_1)
4.5 (L_1)
4.5 (L_1)
4.5 (S_1)
4.5 (S_1)
4.5 (S_1)
4.5 (old)
-
-
NF water
43.8
33.35
33.35
33.35
33.35
33.35
33.35
33.35
33.35
53.4
42.6
DNase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
Amplicon
-
2.2 (10 pM)
2.2 (1 pM)
2.2 (1 nM)
-
2.2 (1 nM)
2.2 (1 nM)
-
-
-
-
Probe
-
-
-
-
2.2
-
-
2.2
2.2
-
-
DNase I
-
-
-
-
-
-
-
-
-
-
1.2
Final volume
60
60
60
60
60
60
60
60
60
60
60
Results:
Discussion:
First of all, the new crRNAs activate less the endonuclease. Nevertheless, we are still not able to get the right order in the concentrations of miRNA.
Week 12 (24/09/18 - 30/09/18)
WEDNESDAY, 26/09/18
Cas12a assay with L_1 (Trial 13)
Aim:
The aim is to remake the Cas12a assay with L_1 crRNA.
Description:
Components (μl)
RCA 1nM
RCA 10pM
RCA 1pM
RCA 100fM
Control RCA (no probe)
RCA 1nM (let 7d)
N*
W
P
NF water
33.35
33.35
33.35
33.35
33.35
33.35
43.8
53.4
42.6
10x Binding Buffer
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12a
3.75
3.75
3.75
3.75
3.75
3.75
-
-
-
crRNA (L_1)
4.5
4.5
4.5
4.5
4.5
4.5
-
-
-
RCA product
2.2
2.2
2.2
2.2
2.2
2.2
-
-
-
(Probe 1, 1 μM)
(Probe 1, 1μM)
(Probe 1, 1μM)
(Probe 1, 1μM)
(Probe 1, 1μM)
(Probe 1, 1μM)
/
/
/
(let7a, 1nM)
(let7a, 10pM)
(let7a, 1pM)
(let7a, 100fM)
-
(let7d, 1nM)
/
/
/
DNAase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
DNAase I
-
-
-
-
-
-
-
-
1.2
Final volume
60
60
60
60
60
60
60
60
60
Results:
Discussion:
Here again, the Cas assay didn't work fine.
FRIDAY, 28/09/18
Effect of the RCA buffer to the Cas12a assay (1)
Aim:
The aim is to analyze the effect of the buffer to the endonuclease. Indeed, according to the previous experiments, it seems that the buffer might play a role in self-activation of the probe.
Description:
We want to test the effect of the different components. As a consequence, we decide to make multiple samples, with each time, one components removed. The goal is to identify component(s) that can be responsible of                   the self-activation of the probe
0a
0b
1a
1b
2
3
4
5
6
DEPC-tw
12.25
14.75
17.25
17.75
20.75
15.75
15
21
24
buffer
2.5
2.5
2.5
2.5
2.5
3
dNTPs
6
6
6
6
6
6
probe
1
1
1
1
1
1
1
1
BSA
0.25
0.25
0.25
0.25
0.25
0.25
miRNA
2.5
phi29
0.5
0.5
0.5
0.5
0.5
0.5
We perform the RCA today and we plan to make the Cas12a assay tomorrow
SATURDAY, 29/09
Effect of the RCA buffer to the Cas12a assay (2)
Aim:
The aim is to perform the Cas12a assay with the RCA product we made yesterday.
Description:
We also added again a proper negative control, which had not been considered in the previous Cas assay.
The different samples are the following :
Components ( μl)
0a
0b
1
1b
2
3
4
5
6
Negative control
N*
W
P
NF water
33.35
33.35
33.35
33.35
33.35
33.35
33.35
33.35
33.35
35.55
43.8
53.4
42.6
10x Binding Buffer
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12a
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
-
-
-
crRNA (L_1)
4.5
4.5
4.5
4.5
4.5
4.5
4.5
4.5
4.5
4.5
-
-
-
RCA product
2.2
2.2
2.2
2.2
2.2
2.2
2.2
2.2
2.2
-
-
-
-
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
no probe
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
/
/
/
/
(let7a, 1nM)
no miRNA
no miRNA, no phi-29 buffer
no miRNA, no phi-29 buffer, no phi-29
no miRNA, no dNTPs
no miRNA
no miRNA, no BSA
only probe, DEPC-tw and phi29-buffer
only probe and DEPC-tw
/
/
/
/
DNAase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
DNAase I
-
-
-
-
-
-
-
-
-
-
-
-
1.2
Final volume
60
60
60
60
60
60
60
60
60
60
60
60
60
Results:
Discussion:
The results are surprising because even the removal of BSA (only 0.25 µl) is able to reduce the signal to 0.
Week 13 (01/10/18 - 07/10/18)
MONDAY, 01/10/18
Effect of the RCA buffer to the Cas assay (Trial 15)
Aim:
The aim is to repeat the experiments that we did on Friday and on Saturday, in the same day, in order to prevent potential degradation of the amplicons
Description:
The sample were done in large tubes.
0a
0b
0c
1
2
3
4
5
6
DEPC-tw
12.25
14.75
14.75
17.25
20.75
15.75
15
21
24
buffer
2.5
3
2.6
2.5
2.5
2.5
3
dNTPs
6
6
6
6
6
6
probe
1
1
1
1
1
1
1
1
BSA
0.25
0.25
0.25
0.25
0.25
0.25
miRNA
2.5
phi29
0.5
0.5
0.5
0.5
0.5
0.5
We then performed a new extensive Cas assay:
Components(μl)
Negative
Negative 1
Negative 2
Negative 3
0A
0B
0C
1
2
3
4
5
6
R1
R2
N*
W
P
NF water
35.55
41.6
37.85
37.10
33.35
33.35
33.35
33.35
33.35
33.35
33.35
33.35
33.35
-
-
43.8
53.4
42.6
10x Binding Buffer
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
-
-
6.6
6.6
6.6
Cas12a
3.75
-
3.75
-
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
3.75
-
-
-
crRNA (L_1)
4.5
-
-
4.5
4.5
4.5
4.5
4.5
4.5
4.5
4.5
4.5
4.5
4.5
4.5
-
-
-
RCA product
-
2.2
(probe 1)
2.2
(probe 1)
2.2
(probe 1)
2.2
(0a)
2.2
(0b)
2.2
(0c)
2.2
(1)
2.2
(2)
2.2
(3)
2.2
(4)
2.2
(5)
2.2
(6)
-
-
-
-
-
DNAase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
DNAase I
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
1.2
DEPC-tw
-
-
-
-
-
-
-
-
-
-
-
-
-
11.55
17.55
-
-
-
phi-29 buffer
-
-
-
-
-
-
-
-
-
-
-
-
-
6
6
-
-
-
dNTPs
-
-
-
-
-
-
-
-
-
-
-
-
-
14.4
14.4
-
-
-
probe 1 (1 uM)
-
-
-
-
-
-
-
-
-
-
-
-
-
2.4
2.4
-
-
-
BSA
-
-
-
-
-
-
-
-
-
-
-
-
-
0.6
0.6
-
-
-
miRNA (let-7a)
-
-
-
-
-
-
-
-
-
-
-
-
-
6
-
-
-
-
phi-29
-
-
-
-
-
-
-
-
-
-
-
-
-
1.2
1.2
-
-
-
Final volume
60
57.8
57.8
57.8
57.8
57.8
57.8
57.8
57.8
57.8
57.8
57.8
57.8
60
60
60
60
60
Discussion:
First of all, coupling both RCA and Cass assay was unsuccessful. Though, it appears that we finally successfully get a very good result in the last graph. Indeed, we managed to get a high signal for 1nM amplicon and a very small signal for the probe only.
Those results are very promising.
,THURSDAY,06/10/18
Trial 16
Aim:
We want to test again the Cas assay with different concentrations of miRNAs. We took newly-diluted probes, small PCR tubes and "fresh" RCA samples to make it.
Description:
Components(μl)
N*
Negative control
0
1nM
100pM
10nM
1pM
100fM
W
P
NF water
43.8
35.55
33.35
33.35
33.35
33.35
33.35
33.35
53.4
42.6
10x Binding Buffer
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
6.6
Cas12a
-
3.75
3.75
3.75
3.75
3.75
3.75
3.75
-
-
crRNA (L_1)
-
4.5
4.5
4.5
4.5
4.5
4.5
4.5
-
-
RCA product
-
-
2.2
2.2
2.2
2.2
2.2
2.2
-
-
-
-
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
(Probe 1, 1uM)
-
-
-
-
no miRNA
1nM miRNA
100pM miRNA
10 pM miRNA
1pM miRNA
100 fM miRNA
-
-
DNAase Alert
9.6
9.6
9.6
9.6
9.6
9.6
9.6
9.6
-
9.6
DNAase I
-
-
-
-
-
-
-
-
-
1.2
Final volume
60
60
60
60
60
60
60
60
60
60
Results:
Discussion:
Unfortunately, the assay didn't work well.
iGEM2018@groupes.epfl.ch
IGEM EPFL 2018
Twitter Facebook
