<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Notebook_Safety_And_Protocols_Comments"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Notebook/Safety_And_Protocols/Comments">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Notebook/Safety_And_Protocols/Comments&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Safety_And_Protocols/Comments&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Safety_And_Protocols/Comments&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Notebook/Safety_And_Protocols/Comments" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Notebook/Safety And Protocols/Comments</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV class="menuBar"><UL id="nav"><LI><A href="https://2010.igem.org/Team:Calgary" onmouseover="mopen('m1')" onmouseout="mclosetime()">
				Home</A></LI><LI><A href="https://2010.igem.org/Team:Calgary/Team" onmouseover="mopen('m2')" onmouseout="mclosetime()">
				Team</A><DIV id="m2" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://igem.org/Team.cgi?year=2010&amp;team_name=Calgary">Official Team Profile</A><A href="https://2010.igem.org/Team:Calgary/Team/Biographies">Biographies</A><A href="https://2010.igem.org/Team:Calgary/Team/Facilitators">Facilitators</A><A href="https://2010.igem.org/Team:Calgary/Team/About_The_University">About the University</A><A href="https://2010.igem.org/Team:Calgary/Team/Acknowledgements">Acknowledgements</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Project" onmouseover="mopen('m3')" onmouseout="mclosetime()">
				Project</A><DIV id="m3" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2010.igem.org/Team:Calgary/Project">Overview</A><A href="https://2010.igem.org/Team:Calgary/Project/Transcription">Transcription/Translation</A><A href="https://2010.igem.org/Team:Calgary/Project/misfolding_overview">Protein Misfolding</A><A href="https://2010.igem.org/Team:Calgary/Project/Controls">Testing Our System</A><A href="https://2010.igem.org/Team:Calgary/Project/Achievements">Achievements</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Parts/Parts_Submitted" onmouseover="mopen('m4')" onmouseout="mclosetime()">
				Parts</A><DIV id="m4" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2010.igem.org/Team:Calgary/Parts/Parts_Submitted">Parts Submitted</A><A href="https://2010.igem.org/Team:Calgary/Parts/Characterization">Characterization</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Notebook" onmouseover="mopen('m5')" onmouseout="mclosetime()">
				Notebook</A><DIV id="m5" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2010.igem.org/Team:Calgary/Notebook/Calendar">Calendar</A><A href="https://2010.igem.org/Team:Calgary/Notebook/Future_Directions">Future Directions</A><A href="https://2010.igem.org/Team:Calgary/Notebook/Safety_And_Protocols">Protocols</A><A href="https://2010.igem.org/Team:Calgary/Notebook/Safety">Safety</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Modelling" onmouseover="mopen('m6')" onmouseout="mclosetime()">
				Modelling</A><DIV id="m6" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2010.igem.org/Team:Calgary/Modelling/MATLAB">MATLAB</A><A href="https://2010.igem.org/Team:Calgary/Modelling/Maya">Animation</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Community" onmouseover="mopen('m7')" onmouseout="mclosetime()">
				Community</A><DIV id="m7" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2010.igem.org/Team:Calgary/Community/Ethics">Ethics/Human Practices</A><A href="https://2010.igem.org/Team:Calgary/Community/Gallery">Gallery</A><A href="https://2010.igem.org/Team:Calgary/Community/Conferences">Conferences</A><A href="https://2010.igem.org/Team:Calgary/Community/Suffield">DRDC Suffield</A><A href="https://2010.igem.org/Team:Calgary/Community/Podcasts">Podcasts</A><A href="https://2010.igem.org/Team:Calgary/Community/Blog">Visit Our Blog!</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A class="menuend" href="https://2010.igem.org/Team:Calgary/Extras">Extras</A></LI></UL></DIV><H1>Comments on the Protocols</H1><A href="https://2010.igem.org/Team:Calgary/Notebook/Safety_And_Protocols">Safety and Protocols Main Page</A><H2><I>Taq</I> Polymerase Chain Reaction</H2><H3>Theory</H3><P>PCR amplifies a portion of DNA, as bounded by specific primers. The process mimics the replication of DNA in a regular cell.</P><OL><LI>First, the two strands of the template DNA are separated (<B>denatured</B>) at 95°C, which is a high enough temperature to break the hydrogen bonds between the base pairs.</LI><LI>Next comes the <B>annealing</B> of the primers. The annealing temperature is extremely specific to the primer being used. There are many programs out there that can help you determine the annealing temperature of the primer such as BLAST primer and GCK.</LI><LI>Then, at 72°C, the <I>Taq</I> polymerase <B>extends</B> the annealed primer to create a copy of the DNA template. The process repeats. Note: you can use Pfu or other polymerases with exonuclease activity in order to ensure that there are no mutations are added.</LI></OL><P><B>Primers</B> are short pieces of DNA (approximately 20-30 bases long) that are complementary to the base strand. Since polymerase can only attach an incoming 5' nucleotide to an existing 3' end, a starting point for the polymerase to &quot;attach to&quot; has to exist. This is the end of the primer. As a result, the portion bound between the two primers is amplified.</P><P><I>Taq</I> polymerase is harvested from bacteria that live in hot springs, and it is used because it is functional at relatively high temperatures. Its optimal functioning temperature is at 72°C. Otherwise, the denaturation temperatures (at 95°C) will destroy the polymerase and cause the PCR to fail. MgCl<SUB>2</SUB> is a salt that allows <I>Taq</I> polymerase to function.</P><P>Because polymerase can only go so fast, you should allow approximately 1 minute for every 1000 base pairs to be amplified, to ensure that the strand is fully copied before the cycle repeats. Step 5 is the <B>final extension</B> time, which ensures that all strands are extended completely. Then the PCR product can be held as long as you need it to be held at 4°C.</P><H3>Tips</H3><UL><LI>For the Master Mix, you should add water in two portions (half added before all other reagents and half added after all other reagents).</LI><LI>Make sure you spin down the mixture with centrifuge or microfuge before you add Taq</LI></UL><A href="javascript:window.close();">Close Window</A><H2>Preparation of Competent cells</H2><A href="javascript:window.close();">Close Window</A><H2>Bacterial Transformation</H2><H3>Theory</H3><P>Transformation is the process by which bacteria can take up foreign DNA. For this process to occur, competent cells are necessary. Transformation allows bacteria to express traits/genotypes that are not native to the strain for example: antibiotic resistance, GFP etc.</P><U>Heat Shock method</U><UL><LI> In this method the competent cells are thawed and heat shocked for 5 minutes. Compentent cells are frozen with pores in their cell walls/membranes. Icing the cells for 30 minutes allows the plasmid to adhere to the cell wall/membrane. Heat shocking allows the plasmid to get in by enlarging the pores and the cooling the cells right after allows the pores to become smaller. The SOC treatment allows the bacteria to activate itself and &quot;heal its woulds&quot;. Although, it is believed that heat shock works this way it is not proven in the literature.
</LI></UL><U>Electroporation </U><UL><LI>

Electroporation essentially uses the same principals but engages in a differnt method of inserting the DNA. Elctroporation applies an electric field across the cells which creates pores in the cell wall/membrane. This allows the plasmid to be taken in by the cells. These cells are then recovered and allowed to grow.
</LI></UL><H3>Tips</H3><P>You may leave the cells shaking for longer than an hour but not overnight. Leaving the cells in the shaker for about 2-4 hours allows the cells to grow and recover well. This gives high transformation efficieny with lots of colonies.</P><A href="javascript:window.close();">Close Window</A><H2>Restriction Digest</H2><H3>Theory</H3><P>

Restriciton digest entails cutting DNA fragments with restriction enzymes. The restriction enzymes recognize DNA fragments at specific site. There are 6 cutters which recognize a series of six bases and 8 cutters that recognize 8 bp fragments. These fragments have to palendromic in order for the restriciton enzymes to digest and cut. There are restriction enzymes that are blunt cutters which do not leave overhangs. There are also enzymes that leave overhangs either 3' or 5'.

</P><A href="javascript:window.close();">Close Window</A><H2>Ligation</H2><P>Quick ligase is expensive, you can use quick ligase buffer with regular T4 ligase and ligate at room temperature for 15-20 minutes and it will do the same trick!</P><A href="javascript:window.close();">Close Window</A><H2>Miniprep Plasmid Preparation (using GenElute)</H2><H3>Theory</H3><P>Miniprepping allows one to extract plasmid DNA from bacteria. The principle behind this includes alkaline lysis followed by a silica column adhesion and elusion using water/ respective buffer.

For example the QIAgen kit works like

Principle

The QIAprep miniprep procedure is based on alkaline lysis of bacterial cells followed
by adsorption of DNA onto silica in the presence of high salt (1). The unique silica
membrane used in QIAprep Miniprep Kits completely replaces glass or silica slurries
for plasmid minipreps.</P><P>The procedure consists of three basic steps:</P><P>Firstly, the cells are cultured overnight and spun down and resuspended to a concentrated volume. The resuspension solution contains  RNAase which degrades all the RNA to minimize the contamination and possible interaction of RNA with the plasmidsSubsequently this concentrated volume of the cells are lysed using an alkaline lysis buffer. This breaks open the cell wall and membrane to spill out the inner composition of the cell. Following the lysis, the cell is neutralized using an acidic solution which also serves the purpose of creating a high salt environment with ions. Following this step, the cells are transfered to a silica column which binds DNA along with some endonucleases but allows everything else to pass through. A final wash step is done to ensure that the endonuclease is washed off leaving mostly the DNA. However, this can have some RNA contamination left in it. Then the column is washed with 50-100 µL and speced to measure the concentration.</P><H3>Tips</H3><UL><LI>When you are spec-ing your DNA, make sure you use the elution water/ buffer as your blank. </LI><LI> When spinning your DNA after lysing and neutralization you can spin for 15-20 minutes. This allows the debris to stick to the side of the tube allowing transferring the supernatant with the plasmid much easier and contaminant free.</LI></UL><A href="javascript:window.close();">Close Window</A><H2>Agarose Gel Electrophoresis</H2><H3>Theory</H3><P>
Gel electrophoresis is done in order to separate DNA fragments according to size by applying charge to it. Since DNA is negatively charged, applying negative current to it repels the DNA towards teh positive terminal (bottom of the gel). The bigger fragments move slowly through the gel whereas the smaller fragments move faster. As a result of this, the bigger fragments are higher up in the gel (close to the negative terminal) and the smaller fragments are lower down in the gel (close to the positive terminal)
</P><H3>Tips</H3><P>If you are looking for a big band higher up in the ladder, run the gel at a higher voltage for a shorter period of time. Ethidium bromide wash if necessary.

You don't always have to prepare the 20 μL of solution, you can add appropriate amount of DNA usually 3 μL and then add 2 μL dye and just run teh entire mixture in the gel. 

Make sure you are careful and there are no bubbles in the well, or in your pipette tip. That would move the DNA out of the gel.</P><A href="javascript:window.close();">Close Window</A><H2>Making of Agarose Gels</H2><H3>Tips</H3><UL><LI> When looking for large bands, make gels that are higher than one percent and run at a higher voltage, do the opposite for smaller sized bands.</LI></UL><A href="javascript:window.close();">Close Window</A><H2>Rehydration of Registry DNA</H2><A href="javascript:window.close();">Close Window</A><H2>Construction Technique</H2><H3>Theory</H3><P> The standard construction technique that is widely used in iGEM is the Biobrick cloning method. Each of the standard parts have a prefix which consist of EcoRI, NotI and XbaI and a suffix that consists of SpeI, Not I and PstI. This method entails cutting with respective restriction enzymes so that the two parts have a scar that is unrecognizable by any restriction enzyme, that is : non palendromic. This allows the two biobrick parts to be put together making a device. This device has the same prefix and suffic that each of the original parts had.The restoration of the prefix and suffix sequences allows more parts to be cloned in similarly.

</P><A href="javascript:window.close();">Close Window</A><H2>Overnight Cultures</H2><H3>Tips</H3><UL><LI>Make a control tube with your respective antibiotic. Do not put a colony in there and see if there is any growth. This will ensure that your antibiotics are working.</LI></UL><A href="javascript:window.close();">Close Window</A><H2>Glycerol Stock Preparation</H2><A href="javascript:window.close();">Close Window</A><H2>LB Agar Plate Preparation</H2><H3>Tips</H3><UL><LI>Tape up plate sleeves if planning to reuse the sleeves</LI></UL><A href="javascript:window.close();">Close Window</A><H2>QIAprep Spin Miniprep Kit</H2><H3>Tips</H3><UL><LI>Drying for 2 minutes instead of 1 minute can ensure that all of the wash solution is removed</LI><LI>When doing the aliquot step, put the pipette tip directly on the filter and eject in the center of the filter</LI><LI>Wait 15 minutes after adding the water onto the filter in the aliquot step</LI></UL><A href="javascript:window.close();">Close Window</A><H2>Gel Extraction</H2><H3>Theory</H3><P>This method allows selection for specific bands. Gel extracting allows disregarding non-specific bands and product that might have been a part of a procedure such as PCR, Restriction digest etc. 

This ensures selection of the right product for cloning experiments so that the product obtained is the product that is needed.</P><H3>Tips</H3><UL><LI>Shorten melting time and alternate between 30 seconds in heat plate and 30 seconds in vortex and look at solution in light to ensure gel is melted.</LI><LI>Use clear 1.5 tubes so that its possible to observe the colour change caused Binding Buffer(XP2) when the pH is too high or too low</LI><LI>Dry step after the addition of Binding Buffer(XP2) could help separate the SPW buffer and Binding Buffer(XP2) stages.</LI><LI>Shorten the length of spin and increase the rpm for the addition of binding buffer and SPW steps and increase to 2 minutes for drying stages</LI><LI>Ensure that the spin column is dry before transferring into the clean 1.5 mL tube</LI><LI> You may use isopropanol to dilute the QG solution after the gel is melted. This increases the yield of DNA </LI></UL><A href="javascript:window.close();">Close Window</A><H2>PCR Purification</H2><H3>Theory</H3><P> PCR purification is done in order to remove exces debris from the PCR tubes such as enzymes, salts,RNA and primers. These side products can interfere with using the DNA later on. for example: salts that are present in the 10x DNA buffer can interfere with a future restriction enzyme digest that might be done. It can also interfere with transformation and reduce transformation efficieny.
</P><A href="javascript:window.close();">Close Window</A><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Calgary/Notebook/Safety_And_Protocols/Comments">http://2010.igem.org/Team:Calgary/Notebook/Safety_And_Protocols/Comments</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Notebook/Safety_And_Protocols/Comments" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Notebook/Safety_And_Protocols/Comments" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Safety_And_Protocols/Comments&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Safety_And_Protocols/Comments&amp;oldid=164611" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>