<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Project_Transcription"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Project/Transcription">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Project/Transcription&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/Transcription&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/Transcription&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Project/Transcription" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Project/Transcription</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV class="menuBar"><UL id="nav"><LI><A href="https://2010.igem.org/Team:Calgary" onmouseover="mopen('m1')" onmouseout="mclosetime()">
				Home</A></LI><LI><A href="https://2010.igem.org/Team:Calgary/Team" onmouseover="mopen('m2')" onmouseout="mclosetime()">
				Team</A><DIV id="m2" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://igem.org/Team.cgi?year=2010&amp;team_name=Calgary">Official Team Profile</A><A href="https://2010.igem.org/Team:Calgary/Team/Biographies">Biographies</A><A href="https://2010.igem.org/Team:Calgary/Team/Facilitators">Facilitators</A><A href="https://2010.igem.org/Team:Calgary/Team/About_The_University">About the University</A><A href="https://2010.igem.org/Team:Calgary/Team/Acknowledgements">Acknowledgements</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Project" onmouseover="mopen('m3')" onmouseout="mclosetime()">
				Project</A><DIV id="m3" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2010.igem.org/Team:Calgary/Project">Overview</A><A href="https://2010.igem.org/Team:Calgary/Project/Transcription">Transcription/Translation</A><A href="https://2010.igem.org/Team:Calgary/Project/misfolding_overview">Protein Misfolding</A><A href="https://2010.igem.org/Team:Calgary/Project/Controls">Testing Our System</A><A href="https://2010.igem.org/Team:Calgary/Project/Achievements">Achievements</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Parts/Parts_Submitted" onmouseover="mopen('m4')" onmouseout="mclosetime()">
				Parts</A><DIV id="m4" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2010.igem.org/Team:Calgary/Parts/Parts_Submitted">Parts Submitted</A><A href="https://2010.igem.org/Team:Calgary/Parts/Characterization">Characterization</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Notebook" onmouseover="mopen('m5')" onmouseout="mclosetime()">
				Notebook</A><DIV id="m5" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2010.igem.org/Team:Calgary/Notebook/Calendar">Calendar</A><A href="https://2010.igem.org/Team:Calgary/Notebook/Future_Directions">Future Directions</A><A href="https://2010.igem.org/Team:Calgary/Notebook/Safety_And_Protocols">Protocols</A><A href="https://2010.igem.org/Team:Calgary/Notebook/Safety">Safety</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Modelling" onmouseover="mopen('m6')" onmouseout="mclosetime()">
				Modelling</A><DIV id="m6" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2010.igem.org/Team:Calgary/Modelling/MATLAB">MATLAB</A><A href="https://2010.igem.org/Team:Calgary/Modelling/Maya">Animation</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Community" onmouseover="mopen('m7')" onmouseout="mclosetime()">
				Community</A><DIV id="m7" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2010.igem.org/Team:Calgary/Community/Ethics">Ethics/Human Practices</A><A href="https://2010.igem.org/Team:Calgary/Community/Gallery">Gallery</A><A href="https://2010.igem.org/Team:Calgary/Community/Conferences">Conferences</A><A href="https://2010.igem.org/Team:Calgary/Community/Suffield">DRDC Suffield</A><A href="https://2010.igem.org/Team:Calgary/Community/Podcasts">Podcasts</A><A href="https://2010.igem.org/Team:Calgary/Community/Blog">Visit Our Blog!</A></DIV></LI><LI><A href="https://2010.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A class="menuend" href="https://2010.igem.org/Team:Calgary/Extras">Extras</A></LI></UL></DIV><DIV class="container"><DIV class="sidebar"><H1>Project Descriptions</H1><UL><LI><A href="https://2010.igem.org/Team:Calgary/Project/Transcription">Transcription/Translation Reporter Circuit</A></LI><LI><A href="https://2010.igem.org/Team:Calgary/Project/misfolding_overview">Protein Misfolding Reporters</A></LI><LI><A href="https://2010.igem.org/Team:Calgary/Project/IbpAB">Cytoplasmic Stress Detectors</A></LI><LI><A href="https://2010.igem.org/Team:Calgary/Project/CpxP">Periplasmic Stress Detectors</A></LI></UL><LI><A href="https://2010.igem.org/Team:Calgary/Project/Controls">Testing Our System</A></LI><LI><A href="https://2010.igem.org/Team:Calgary/Project/Achievements">Achievements</A></LI></DIV><DIV class="mainbody"><SPAN id="bodytitle"><H1>Transcription/Translation Circuit</H1></SPAN><H2 style="color:#0066CC">Overview</H2><P>
Transcription and translation are essential processes for protein expression. Problems that arise during these processes could lead to improper protein formation. Issues that can occur include shortage in length, folding problems, low or no expression, etc. These issues are accentuated in synthetic biology as foreign genes are implemented into prokaryotes such as <I>Escherichia coli</I>. The transcription translation detector circuit was developed in order to test whether or not a gene of interest is being correctly transcribed and translated.</P><H3>How the Circuit Works</H3><TABLE><TBODY><TR><TD><P>
The gene of interest is fused to a mutant RFP.  Downstream of this is GFP with its own ribosomal binding site.  If transcription is occurring, the transcript would include the gene of interest, RFP as well as GFP.  Because GFP has its own ribosomal biding site, it should be translated if transcription is happening.</P></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><P>
 If the gene of interest is also being translated, then RFP should also be translated because it is fused to the GOI.the RFP was specially selected from Dr. Lewenza’s lab. This RFP (nicknamed sRFP or special red fluorescent protein) can fold in the cytoplasm, periplasm and the cellular membrane.
</P></TD></TR></TBODY></TABLE><H2 style="color:#0066CC">Details</H2><H3>Assumptions:</H3><P>
 For this circuit to work, there are several assumptions that must be made. The first of which is a result of a limitation within the design which is that sRFP will not affect the stability of the protein of interest. Both positives and negatives are not ideal because the circuit functions as an indicator, any assistance could lead to false positives or vice versa. Second, AraC is the right promoter for this circuit. Although there are many benefits for using an arabinose inducible promoter, however evolutionary conditions have established optimal expression in natural promoters.  Third is folding properties in the periplasm and cytoplasm (Lewenza, et al., 2006)  had to be the same such that a sRFP in the cytoplasm will give the same absorbance as one in the periplasm. Fourth would be that the GOI does not contain a “rut” site (Rho utilization site) which would prematurely stop transcription using Rho dependent termination. Fifth would be that <I>E.coli</I> would be the most compatible cell available for protein expression. Much like the second assumption, genes are optimally expressed in its natural host. Transferring these genes into E.coli might decrease the efficiency of protein expression. These are considerations that must be made in order to ensure the success of this circuit towards its utilization within our testing kit. It is definitely more “artificial” compared to the other two mostly because it overrides the necessity for the natural systems within. However if all limitations are accounted for, this could be a very useful tool if coupled with our other systems. </P><P>Helpful tips with understanding the circuit: With the way the circuit is developed, a failure of transcription will lead to a failure of translation. Therefore it is impossible to see only red cells, but possible to see green cells. If a brownish color is expressed (a mixture of red and green), this indicates both transcription and translation. Also if only green cells are noticed, then to definitively test whether or not there is something wrong with translation, a user must employ the other two circuits. Meaning positive in the folding circuits indicates the translation mechanism works. However, due to the design of attaching sRFP with the GOI, the GOI misfolding will affect the stability of sRFP.</P><H3>Problems that can arise during transcription / translation:</H3><P>There are numerous problems that can arise in the transcription and translation, especially when trying to turn <I>E.coli</I> into a factory for foreign proteins. Each category of transcription and translation can be broken down into pre, during, and post. Although some aspects between post-transcription and pre-translation are slightly grey, there are parts of it that are quite clear. For example, the attachment of the 30S subunit from rRNA would be considered pre-translational but not post-transcriptional. This section describes some of the possible transcription/translation issues and the following responses by the system. </P><H4 style="color:#003366">Transcription</H4><P><B>Pre-transcription</B></P><P><EM>Transcription factors</EM></P><P>One of the main ideas of synthetic biology is the expression of proteins from foreign enzymes. For example, GFP comes from Aequorea Victoria (Andersen, et al., 1998) . One of the considerations is whether or not foreign circuits have the corresponding transcriptions within <I>E.coli</I>. If these transcription factors have a profound effect on whether or not transcription can occur (Kleinert, et al., 2003) , then natural promoters might be hindered or lack the necessary transcription factors for expression. Therefore it is necessary to include an arabinose promoter (pBad/araC), a well characterized and working promoter in <I>E.coli</I>. (iGEM registry,2003) </P><P>If the problem of the foreign circuit lies in the promoter, the circuit can be used to detect this simply through inserting the RBS+GOI into the circuit and compare this with inserting the foreign promoter + RBS + GOI. If there is expression without the foreign promoter, and no expression with it, then there could be a repressor bounded to the operon of the circuit. If there is expression in both then a third circuit can be constructed with just the foreign promoter + RBS + GOI without the arabinose promoter. If there is no expression in the third, then the foreign promoter lacks the necessary transcription factors to operate in the host <I>E.coli</I>.</P><P><EM>Promoter strength</EM></P><P>This is not a problem with natural promoters however this is an issue faced by many synthetic biologist when matching a promoter with a GOI. More is not always better, over expression of protein could lead to higher amounts of aggregation and longer folding time.(Brock, 2010)  Choosing the pBAD/araC promoter is beneficial because induction varies with arabinose concentrations. Therefore it is possible to use a 96 well plate with varying levels of arabinose to promote induction at various strengths. A plate reader can then be used to read absorbance levels to find the optimal amount of indicator expressed.</P><P><B>Transcription</B></P><P><EM>Repressor/amount of inducer</EM></P><P>The ratio of inducer to plasmid copy number would be a problem when trying to express a foreign protein in <I>E.coli</I>. Much like issue with transcription factors, the circuit was designed to include an arabinose promoter that way it is possible to control the concentration of the inducer arabinose. In that case there will be no shortage in the concentration of inducer because the promoter is well characterized meaning its induction is known.</P><P><EM>Hair pin loop/rho dependent termination</EM></P><P>The formation of premature hair pin loops and rho utilization sites formed from within the gene are potential methods of premature stops to transcription. Hair pin loops are typically 7 to 20 amino acids long (Lewin, 2007)  and ruts sites are 22-116 base pairs.(Banerjee, et al., 2007)  The more likely of the two when forming an accidental termination site would be a hair pin loop. This relies on the palindrome formation with high concentrations of guanine and cysteine which results in a RNA pulling from the DNA. Our system would detect premature termination of RNA would result in no signal with our GFP signal.</P><P><B>Post-transcription</B></P><P><EM>mRNA shape degradation</EM></P><P>Although transcription occurs, mRNA instability results in the degradation of the mRNA. The circuit would suggest that the GFP report was not expressed. Despite transcription occurring completely, the most logical approach would be to group this under issues with transcription, also because pre-translational steps have not occurred yet.</P><H4 style="color:#003366">Translation</H4><P><B>Pre-translation</B></P><P>No current issues arise from this step.</P><P><B>Translation</B></P><P><EM>Multi codon usage</EM></P><P>When inserting foreign genes into <I>E.coli</I>, the ratios of tRNAs in <I>E.coli</I> in comparison to the foreign source can vary. Shortages in tRNA can lead to problems with rate and accuracy of translation. (Ran and Higgs, 2010)  Kinetics is a factor of rate of protein formation, decreased concentrations of necessary tRNAs results in slower formation of proteins. Based on the research by Drummond and Wilke, lack of accuracy is caused by mistranslation causing higher amount of misfolding.(Drummond and Wilke, 2008)  If the GOI’s multi codon usage disagrees with the host <I>E.coli</I>, there would be aggregation which will inhibit protein expression. The circuit can detect that there are problems with translation because sRFP would be form aggregate bodies with the protein of interest (POI). </P><P><EM>Premature stop codon</EM></P><P>Stop codons inhibit translation. The circuit would indicate the presence of a premature stop codon because the sRFP would not be translated therefore no signal would be present. </P><P><EM>RBS compatibility</EM></P><P>The ribosome binding site allows the attachment of the ribosome. Differences in ribosome strength could change the translation frequencies. This leaves room for protein misfolding. Because of the specificity of the RBS to the expression of the gene, as well as the potential of affecting the triple nucleotide site which could shift the reading frame. The circuit was designed in such a way that the user is capable of attaching their own RBS.</P><P><EM>Copy number</EM></P><P>Copy number refers to the number of plasmids that can exist within on <I>E.coli</I> cell.(iGEM Registry, 2009)  Although this does not change the rate of transcription (polymerase per second, PoPs) like promoter strength, the effects are similar. Increasing the concentration of slower folding proteins could result in aggregation due to exposed hydrophobic segments . (Ran and Higgs, 2010)  The circuit will detect this as an issue with translation as this could affect the protein.</P><P><B>Post-translation</B></P><P><EM>Lack of chaperones</EM></P><P>The lack of essential chaperones could result in protein misfolding. <I>E.coli</I> may not have the necessary chaperones to correct the conformation of the POI. The formation of misfolded protein will cause the aggregation of sRFP, therefore indicating an error in translation.</P><H2 style="color:#0066CC">Design of the Circuit</H2><P>pBad/araC Promoter- This promoter was chosen because it allow for variable strength without replacing the promoter (if the circuit had a promoter library). Because more is not always better, the user can customize optimal levels of promoter strength in protein expression. This part is also highly characterized (iGEM Registry, 2003).</P><P>Multiple Cloning Sites- We are using modified biobrick prefix and suffix. What this means is that these sites are not separating the biobrick parts from the sequences, rather they are located between the arabinose promoter and the sRFP.</P><P>ccdB- A place holder that was added for selection in addition to antibiotic selection. The circuit will contain a suicide ccdB gene as a placeholder for the GOI. If this is not removed, the cell which has this transformed plasmid will die. This will ensure that the only cells present on the plate will only express the genes intended to be there.</P><P>RBS- We have decided not include a RBS within this sequence to allow customizability. Natural RBS are known to indicate optimal PoPs plus issues with this would indicate compatibility problems on the part of the RBS and GOI. This would also eliminate any issues regard reading frame shifts of the RBS to the GOI for those that are biobricking new parts.</P><P>sRFP (special red fluorescent protein)- This is part of the translation portion of the circuit. This indicator was chosen because it can fold in the cytoplasm, periplasm and membranes.(Lewenza, et al., 2006)   One of the limitations of this circuit is that the GOI must be fused to sRFP in order for translation detection to occur. This means additional time on the part of the user to rebiobrick the end portion of the GOI such that the stop codons are removed. Current studies by Lewenza, et al. reveals that RFP can be localized in the cytoplasm as well as the outer membrane.</P><H2 style="color:#0066CC">Reference:</H2><P>1. Andersen, J. B., Sternberg, C., Poulsen, L. K., Bjorn, S. P., Givskov, M., Molin, S., et al. (1998). New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria. Appl. Envir. Microbiol., 64(6), 2240-2246. Retrieved from http://aem.asm.org/cgi/content/abstract/64/6/2240.</P><P>2. Banerjee, S., Chalissery, J., Bandey, I., &amp; Sen, R. (2006). Rho-dependent transcription termination: more questions than answers. Journal of microbiology (Seoul, Korea), 44(1), 11-22. Retrieved from: http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1838574&amp;tool=pmcentrez&amp;rendertype=abstract.</P><P>3. Drummond, D. A., &amp; Wilke, C. O. (2008). Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution. Cell, 134(2), 341-52. doi: 10.1016/j.cell.2008.05.042.</P><P>4. Help:plasmid backbones/features. (2008). Retrieved from http://partsregistry.org/Help:Plasmid_backbones/Features</P><P>5. Kleinert, H., Schwarz, P. M., &amp; Förstermann, U. (n.d.). Regulation of the expression of inducible nitric oxide synthase. Biological chemistry, 384(10-11), 1343-64. doi: 10.1515/BC.2003.152.</P><P>6. Kosuri, S. (2003, December 5). Part:bba_i0500. Retrieved from http://partsregistry.org/Part:BBa_I0500</P><P>7. Lewin, Benjamin (2007). Genes IX. Sudbury, MA: Jones and Bartlett Publishers.</P><P>8. Lewenza, S., Vidal-Ingigliardi, D., &amp; Pugsley, A. P. (2006). Direct visualization of red fluorescent lipoproteins indicates conservation of the membrane sorting rules in the family Enterobacteriaceae. Journal of bacteriology, 188(10), 3516-24. doi: 10.1128/JB.188.10.3516-3524.2006.</P><P>9. Madigan, M.T., Martinko, J.M., Dunlap, P.V., &amp; Clark, D.P. (2008). Brock biology of microorganisms (12th edition). San Francisco, California: Benjamin Cummings.</P><P>10. Ran, W., &amp; Higgs, P. G. (2010). The influence of anticodon-codon interactions and modified bases on codon usage bias in bacteria. Molecular biology and evolution, 27(9), 2129-40. doi: 10.1093/molbev/msq102.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Calgary/Project/Transcription">http://2010.igem.org/Team:Calgary/Project/Transcription</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Project/Transcription" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Project/Transcription" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Project/Transcription&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Project/Transcription&amp;oldid=208535" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>