"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/6 August 2010
From 2010.igem.org
Home
Team
Official Team Profile Biographies Facilitators About the University Acknowledgements
Project
Overview Transcription/Translation Protein Misfolding Testing Our System Achievements
Parts
Parts Submitted Characterization
Notebook
Calendar Future Directions Protocols Safety
Modelling
MATLAB Animation
Community
Ethics/Human Practices Gallery Conferences DRDC Suffield Podcasts Visit Our Blog!
Sponsors
Extras
Notebook
May
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
Friday August 6, 2010
Jeremy
Today I ran my gradient PCR that I left overnight and the gel revealed that the pRFP did not get biobricked. Although this has been accomplished many times, for some reason this time it did not work. We had a meeting with out facilitators who suggested that I should try to increase the biobrick concentration because the bands are extremely faint for a PCR reaction. The thought was that the primers are not annealing as well as expected. The two suggestions were optimizing PCR  or using a TOPO vector. Henry and Emily also suggested vacuum PCR purification to increase the concentration of DNA. Dr. Logan is also inquiring of GC richness in the DNA sample of pRFP. I also made the Notebook and Facilitators Page for the wiki and organized all the files for Patrick to put onto the wiki.
Dev
Colony PCR performed yesterday was unsuccessful. Plates with CpxR+I13504/I13507 construction were found in the fridge from Jeremy's construction in late June. Performed a colony PCR of 17 of these colonies. 8 of these colonies showed the correct size bands (850-900bp) on a 1% agarose gel.
Emily
Today I got the sequencing results back from malE31 C2.  The good news is that it is now biobricked.  Unfortunately it looks like the primer that was sued has actually introduced a single base pair deletion mutation near the end of the gene.  Three amino acids have been added and six other amino acids have been changed.  Fortunately however, there is still a STOP codon in the mutated version.  Because of the presence of this STOP codon amnd the fact that the mutation is near the end of the gene, we have decided to continue by trying to construct this biobricked gene into our system.  We will also however, restart biobricking it with another primer.  This afternoon we had a brief team meeting to see where we are in the project.  After the meeting I transformed my I0500-I13504 and I0500-I13507 constructs into TOP10 E. Coli cells.  I then plated these on A, K and AK plates and left for overnight growth.  Henry will take these plates out of the incubator on Saturday morning.  I also set up a restriction digest of PCR Purified malE and malE31 and PsB1AK3 and psB1AC3 vectors with both combinations of EcoRI/PstI and XbaI/SpeI.  I left these in the 37 water bath.
Himika
Today I kept working on the modelling project just as I was before. Emily got sequencing results back for the MalE31 biobricking which seemed successful other than the fact that it had a mutation at the end of the gene. I will construct using MalE31 and I0500-B0034 on monday and go from there to building the tester gene circuits for our stress promoters.
Patrick
Most of today was spent looking into the mechanism of aggregate body formation, as well as gathering the images for the Team page.
Chris
Today, I did plasmid preparations of the pSB1AK3 and pSB1AC3 from the 2009 Registry distributions. The reason we took these parts was that they contained the ccdB gene which would kill the cells if they did not replace it with the insert. As well, I prepared restriction digests of the pSB1AK3 and pSB1AC3 to insert CpxP purified into. They were digested with combination of XbaI/PstI and XbaI/SpeI. This was done as there is sometimes better growth with one combination of enzymes.
Retrieved from " http://2010.igem.org/Team:Calgary/6_August_2010 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
