"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/3 August 2010
From 2010.igem.org
Home
Team
Official Team Profile Biographies Facilitators About the University Acknowledgements
Project
Overview Transcription/Translation Protein Misfolding Testing Our System Achievements
Parts
Parts Submitted Characterization
Notebook
Calendar Future Directions Protocols Safety
Modelling
MATLAB Animation
Community
Ethics/Human Practices Gallery Conferences DRDC Suffield Podcasts Visit Our Blog!
Sponsors
Extras
Notebook
May
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
Tuesday August 3, 2010
Jeremy
Today I did a restriction digest of the plasmids pSB1AK3 and pSB1AC3 for the plasmid switch of pRFP. This was also constructed together and transformed on AK and AC plates. Also today I designed a wiki main page template.
Raida
Today I set-up and ran a PCR with the forward primers that were designed for MalE and MalE31 with the signal sequence deleted. The PCR conditions are the same as before, with the annealing temperature being 54 degrees celcius for 30 seconds instead of 45 seconds.
I also did a restriction digest to check whether our insert (MalE, MalE31 C1, MalE31C2)had properly ligated to the TOPO vector.
At 4:00 pm, I set up and ran an Agarose Gel Electrophoresis of my PCR products and also of my restriction digest. For the PCR product, I am expecting a band around 1200 bp and for the restriction digest, one at 1400 and the other at 3500.
Today, I also worked on the Ethics paper. I have completed the Introduction to Synthetic Biology and Gene Expression Toolkit and have sent it to another team member to edit and give feedbacks.
Chris
Today, I did plasmid preparations of the CpxP promoter inserted in both Ampicillin-kanamycin and ampicillin-chloramphenicol plasmids that overnight cultures were made of yesterday by Emily. As well, I helped finish Dev's plasmid switch of the CpxR (K135000) promoter into both ampicillin-kanamycin and ampicillin-chloramphenicol plasmids from just ampicillin-resistant plasmids. Finally, I did transformations of the pSB1AK3 and pSB1AC3 from the 2009 Registry parts. These were plated with the ccdB gene embedded (from the Registry standard). I also made negative controls which should not grow of both regular Top10 cells and ccdB resistant cells on ampicillin-kanamycin and ampicillin-chloramphenicol plates.
Himika
Today I made overnight cultures of I0500-B0034 for plasmid prepping and I also did research for the modelling project. I researched about effect of pH and temperature on inclusion body formation so that I can improve the model that already exists in MATLAB.
Emily
Today I set up a PCR with BBK-CP primers of malE and malE31 with the hypothetical XbaI and PstI restriction sites.  This is to verify that we have been able to get this part into the psB1AK3 Biobrick Vector.  If this was successful, we expected to see bands at about 1.4 KB.  We saw that in only one of the lanes, malE31 C2.  Regardless, I made overnight cultures of seven colonies of malE and malE31.  Today I also set up a PCR of malE and malE31 in TOPO Vector with the M13 F/R primers.  This is to verify that we were able to successfully get our gene of interest into the TOPO Vector.  Today I also made restreaks from Glycerol stocks of psB1AK3 and psB1AC3.  I also helped Alex prepare the degP-GFP and degP-RFP constructs for DNA sequencing downstrairs.  I also made overnight cultures of my I0500-I13504 and I0500-I13507 constructs in order to try the arabinose induction as well as harvest plasmid DNA.
Dev
Performed another plasmid switch from scratch of the CpxR promoter. Inserted CpxR into AK and AC plasmids.
Patrick
I searched around today for a space-filling model of the CpxP or CpxR proteins, to little success. Instead, I'm going to try and send the amino acid sequence of CpxP to Rosetta in an attempt to have at least a partially accurate model to base the animation off of.
Retrieved from " http://2010.igem.org/Team:Calgary/3_August_2010 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
