<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ColumbiaU_NYC_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ColumbiaU NYC/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Columbia University iGEM</TITLE></P><DIV role="navigation" class="navbar navbar-default"><DIV class="container"><UL class="nav navbar-nav"><LI class="active"><A href="https://2016.igem.org/Team:ColumbiaU_NYC">Home</A></LI><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/About">About</A></LI><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Team">People</A></LI><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
	      Documentation </A></LI><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Description">Description</A></LI></UL><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Attributions">Acknowledgements</A></LI><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Engagement">Outreach</A></LI></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-3"><DIV id="nav-sidebar-scroll"><UL class="nav nav-sidemenu-scroll hidden-print hidden-sm hidden-xs affix"><LI><A href="#may">May</A></LI><LI><A href="#5-22">5/22-5/28</A></LI><LI><A href="#5-29">5/29-6/4</A></LI></UL><LI><A href="#june">June</A></LI><LI><A href="#6-5">6/5-6/11</A></LI><LI><A href="#6-12">6/12-6/18</A></LI><LI><A href="#6-19">6/19-6/25</A></LI><LI><A href="#6-26">6/26-7/2</A></LI><LI><A href="#july">July</A></LI><LI><A href="#7-3">7/3-7/9</A></LI><LI><A href="#7-10">7/10-7-16</A></LI><LI><A href="#7-17">7/17-7/23</A></LI><LI><A href="#7-24">7/24-7/30</A></LI><LI><A href="#7-24">7/24-7/30</A></LI><LI><A href="#7-31">7/31-8/6</A></LI><LI><A href="#August">August</A></LI><LI><A href="#8-7">8/7-8/13</A></LI><LI><A href="#8-14">8/14-8/20</A></LI><LI><A href="#8-21">8/21-8/27</A></LI><LI><A href="#8-28">8/28-9/3</A></LI><LI><A href="#September">September</A></LI><LI><A href="#9-4">9/4-9/10</A></LI><LI><A href="#9-11">9/11-9/17</A></LI><LI><A href="#9-18">9/18-9/24</A></LI><LI><A href="#9-25">9/25-10/1</A></LI></DIV></DIV><DIV class="col-md-9"><H1 id="may">May</H1><H2 id="5-22">5/22-5/28</H2><P>We solidified the synthetic mosquito repellent
	    idea. We spent the week conducting a literature
	    review to determine what research has been
	    conducted on the rhamnolipids and bacteria
	    strains. We pinpointed a few subtopics that we
	    could pursue such as freeze drying and skin
	    grafting. We divided the research among ourselves
	    and put together a powerpoint containing a summary
	    of our ideas to present to our faculty
	    advisors. After the meeting, we made a
	    comprehensive project timeline schedule and made
	    experimental plans for the following week.
	  </P><H2 id="5-29">5/29-6/4</H2><H4>Molecular Cloning</H4><P>
	    We started molecular cloning with a PCR amplification of
	    the RhlAB operon from the P. aeruginosa P14 genome
	    extract. For the screening assay, we used a superfolded
	    GFP.  With our faculty members, we discussed whether we
	    should clone using EcoRI restriction enzyme cuts or Gibson
	    assembly. Initially, our PCR was unsuccessful because the
	    primers we designed had too few base pairs annealing to
	    the sequence. After ordering the correct primers, we used
	    Gibson assembly to insert the gene into the backbone,
	    which was successful. Additionally, we transformed
	    plasmids without the RhlAB insert into P. putida
	    KT2440. Then, we picked colonies with different GFP
	    intensities to obtain a library of different constitutive
	    promoters as well.
	  </P><H4>Qualitative Analysis of Rhamnolipids</H4><P>
	    Regarding the production of rhamnolipids, we researched a
	    few different tests before deciding to try the CTAB
	    qualitative test with pure rhamnolipids. Furthermore, we
	    decided to try freeze-dry <EM>P. putida</EM> to determine what
	    percent of the bacteria would be viable after extended
	    periods of time. We could use this knowledge to determine
	    shelf-life of the mosquito repellent for commercial
	    applications.
	  </P><H4>Mosquito and Mouse Experiments</H4><P>
	    We reached out to several researchers as well in
	    preparation for future experiments. We contacted
	    Dr. Voshall at Rockefeller University who does mosquito
	    testing.  We want to determine what concentration of
	    mono-rhamnolipids and di-rhamnolipids actively repel
	    mosquitoes. We also reached out to Dr. Christiano who
	    works with mice to discuss whether we can use her
	    laboratory to conduct rhamnolipid experiments using human
	    skin grafted on mice. In preparation for the meeting, we
	    drafted a set of experimental protocols and created a
	    Powerpoint presentation.
	  </P><H4>Community Outreach</H4><P>
	    We planned a community outreach project in which we can
	    administer a survey to individuals to determine if they
	    are highly attracted or unattracted to mosquitoes and then
	    use next generation sequencing to sequence samples of the
	    skin biome to deduce if there are any commonalities.
	  </P><H1 id="june">June</H1><H2 id="6-5">6/5-6/11</H2><H4>Freeze Dry</H4><P>
	    The initial freeze dry experiments were successful but we
	    realized that we need a more quantitative way to analyze
	    the success of the freeze dry. Perhaps, we need to
	    incorporate a survival rate or ratio calculation. We
	    decided that we need to also extend the timeline for the
	    freeze dry experiments to more closely mimic real world
	    situations.
	  </P><H4>Rhamnolipid Quantification</H4><P>
	    We conducted the CTAB experiments with the rhamnolipids
	    and blue halos were present. However, we realized CTAB
	    only works well for higher concentrations than what we
	    need to produce. Furthermore, we would need a more
	    quantitative metric than qualitative one. We brainstormed
	    and figured that we could use an image application to
	    measure the diameter of the halos and correlate that to
	    the concentration of rhamnolipids.
	  </P><H4>Community Outreach</H4><P>
	    We had several meetings this week one of which was with
	    Dr. Shaman, a skin swabbing expert, to discuss our idea
	    for community outreach. Dr. Shaman had collaborated with
	    the Museum of Natural History to swab skin biomes in the
	    skin biome exhibit. He suggested that we avoid
	    collaborating with the museum due to the extensive IRB
	    process and provisions of the board of trustees. He also
	    cautioned our timeline and monetary expenses.
	  </P><H4>Keratinocyte Experiments</H4><P>
	    We met with Dr. Prince who is a skin bacteria specialist
	    to discuss collaboration. We planned to conduct
	    experiments to determine how the bacteria strains and
	    rhamnolipids affect skin cell viability. Dr. Prince liked
	    our idea and put us in contact with a graduate student in
	    her laboratory to gather more information. She agreed to
	    provide her lab space, human skin cells (keratinocyte),
	    cell media, and other reagents to aid in our experiments.
	  </P><H2 id="6-12">6/12-6/18</H2><H4>Molecular Cloning</H4><P>
	    Our Gibson assembly worked for two different promoters; we
	    now have two constructs transformed in E. coli including a
	    strong promoter from Streptococcus pneumoniae and a weak
	    promoter from Propionibacterium acnes. We started to work
	    on the construction of BioBrick parts. We began to build
	    BioBricks for different promoters for the RhlAB operon. We
	    made plans to characterize the promoters in E. coli,
	    P. aeruginosa, and P. putida. This week, we also continued
	    experiments to transform recombinant plasmids into
	    P. putida using electroporation. We contacted Dr. Jiang
	    from the Marraffini Rockefeller University to obtain the
	    S. epidermidis strains, recommended plasmids, and shuttle
	    vector for transformation/conjugation. He also sent us
	    specific protocols to utilize with this strain.
	  </P><H4>Rhamnolipid Quantification</H4><P>
	    We conducted more CTAB experiments in which we made test
	    plates with different concentrations of rhamnolipids to
	    make a standard curve using the diameters of the halos for
	    our quantitative measure. Unfortunately, we determined
	    that the halo diameter varies with the thickness of the
	    CTAB on plates in that thicker plates give bigger halos
	    for the same concentration of rhamnolipids. This
	    characteristic created inconsistency across different CTAB
	    plates. There were no visible halos for low concentrations
	    of rhamnolipids indicating that we might need to calibrate
	    CTAB to detect lower concentrations of rhamnolipids. We
	    also conducted the CTAB test using live E. coli cells with
	    the RhlAB recombinant plasmid, but there were no visible
	    Halos possibly due to low Rhamnolipid concentrations. The
	    overall conclusion is that the CTAB test may be used as a
	    qualitative analysis, but is not reliable as a
	    quantitative method due to condition-dependent halo
	    diameters. We found a different assay called an orcinol
	    assay that may be used in place of the CTAB test.
	  </P><H4>Freeze Dry</H4><P>
	    We focused on counting the exact levels of survival using cell count methods. 
	  </P><H4>Mosquito Experiments</H4><P>
	    Because the lab at Rockefeller University did not respond
	    to emails, we contacted the Fidock lab at Columbia
	    University. The Fidock lab has extensive experience with
	    Malaria experiments and has incubators that they would be
	    willing to allow us to use. We made a meeting for the
	    following week and began to draft experimental plans to
	    discuss with Professor Fidock in person.
	  </P><H2 id="6-19">6/19-6/25</H2><H4>Molecular Cloning</H4><P>
	    We received S. Epidermidis strain RP62A, shuttle strain
	    S. Aureus RN4220, a plasmid pC194 from S.A. RN4220, a
	    plasmid pC221 and pGO1 from S.A. OS2 from Dr. Jiang at
	    Rockefeller. We will start cloning next week once we
	    design the primers.
	  </P><H4>Freeze Dry</H4><P>
	    Experiments were conducted using M9 minimal media or LB as
	    the rehydration media. The lyoprotectants used was either
	    10% Sucrose or 10% Sucrose with LB. Based upon the
	    analysis conducted in the freeze dry research paper,
	    survival rate was calculated. The highest survival rate
	    calculated was 2.25% using LB as the rehydration and the
	    10% Sucrose lyoprotectants.
	  </P><H4>Rhamnolipid Quantification</H4><P>
	    As a way of detecting rhamnolipids, we decided to try
	    thin-layer chromatography (TLC). For different staining
	    methods, we tried orcinol with 50% sulfuric acid, orcinol
	    with 10% sulfuric acid, CAM, and KMnO4. We discovered that
	    orcinol with 10% sulfuric acid gives the best visibility,
	    and the limit of detection was determined to be 0.5
	    mg/mL. We were also able to distinguish between
	    mono-rhamnolipids and di-rhamnolipids through TLC
	    analysis.
	  </P><H4>Mosquito Experiments</H4><P>
	    We met with Professor Fidock who agreed to let us work in
	    his laboratory. He put us in contact with a mosquito
	    seller from NYU and a trained epidemiologist with whom we
	    can discuss the experimental setup. However, the
	    epidemiologist will not be available until August
	    1st. Thus, in the meantime, we will communicate via email
	    regarding the experimental protocol and setup.
	  </P><H2 id="6-26">6/26-7/2</H2><H4>Molecular Cloning</H4><P>
	    We made glycerol stocks of Dr. Jiang's strains and mini
	    prepped plasmids using lysostaphin. We began the process
	    of building recombinant plasmids using Gibson assembly. In
	    addition, we successfully created a BioBrick with a weak
	    promoter that we transformed into P. putida. We are still
	    working on removing illegal cut sites in RhlAB operon. We
	    also began the documentation and characterization process
	    of our parts.
	  </P><H4>Transformation</H4><P>
	    At this point, we have a weak promoter construct working,
	    yet we are still having difficulty transforming a strong
	    promoter in order to produce a greater degree of
	    rhamnolipids. We believe that this may be the result of a
	    metabolic burden. We brainstormed a few possible solutions
	    to this problem such as using conjugation methods or an
	    inducible promoter.
	  </P><H4>Rhamnolipid Quantification</H4><P>
	    This week we tried to increase the sensitivity of the CTAB
	    assay by using less CTAB. We discovered that the detection
	    limit was still too high. It was difficult to tell if the
	    halos were due to rhamnolipids or GFP already present in
	    the cell culture. Regular LB also showed halos as well,
	    reinforcing evidence that this assay should not be used.
	  </P><P>
	    TLC was utilized to quantify rhamnolipids first using a
	    small batch and then a larger batch of P. putida. Using
	    the smaller quantity of P. putida, there was too little
	    rhamnolipid to detect. For the larger batch, cell culture
	    was grown for a longer period of time and purified via
	    liquid-liquid extraction. Still the quantity of
	    rhamnolipid was too small to detect. We decided that we
	    should use supercritical fluid chromatography mass
	    spectroscopy (SFC-MS) to increase sensitivity for
	    measuring rhamnolipid quantity.
	  </P><H4>Freeze Dry</H4><P>
	    We repeated the freeze dry experiments using different
	    storage times: immediate and 3 days. In addition to
	    calculating survival rate, we also calculated the
	    bacterial survival ratio. The paper we were using as a
	    model reported an 80% &quot;survival rate.&quot; In reality, the
	    survival rate they reported was a different metric called
	    bacterial survival ratio. This difference in terminology
	    explained why we thought our experiment was not
	    working. We had a 2.26% survival rate which corresponds to
	    an 82.34 bacterial survival ratio, which is the number the
	    paper reported. Thus, our results are on par with the
	    paper.
	  </P><H4>PhaC Knockout</H4><P>
	    We ran a PCR on the PhaC gene in knockout P. putida from
	    Spain against PCRed PhaC gene of WT P. putida to confirm
	    if a transposon was indeed in the PhaC gene. Lengths of
	    the PhaC gene in both strains were the same, indicating
	    the transposon was not in the PhaC gene.However, a PCR
	    product created from transposon-internal primer sequence
	    provided in the paper existed for the knockout but not WT,
	    indicating that the transposon was somewhere in the
	    knockout.We will do a full sequence of 200 bp upstream of
	    PhaC gene to 200bp downstream PhaC gene in the knockout to
	    get clearer picture of what is going on.
	  </P><H4>Mouse and Mosquito Experiments</H4><P>
	    We began the process of IRB approval and spoke with
	    Dr. Christiano who suggested we speak with Dr. Owens about
	    establishing the IRB. We also registered for lab safety
	    training using mice. We met with a lab member from
	    Dr. Owen's lab, Rong Du, to discuss the IRB in more detail
	    and gave her a copy of our experimental protocols. We also
	    improved our protocol upon discuss with a graduate
	    student, Lekha Nair, from Dr. Christiano;s lab.
	  </P><H1 id="july">July</H1><H2 id="7-3">7/3-7/9</H2><H4>Keratinocyte Experiments</H4><P>
	    We began our first keratinocyte experiments in
	    Dr. Prince’s laboratory. The goal of the experiment was to
	    determine which MOIs of the bacteria strains (Putida,
	    Mutant, Arginosa) were toxic to the cells and to establish
	    whether the bacteria was engulfed by the cells by
	    plating. The experiment took longer than expected as we
	    included two incubation points: 2 hours and 4 hours. We
	    then had to leave the laboratory as there was no longer a
	    qualified scientist available to watch us. Before leaving,
	    we placed the cells on slides in preparation to count them
	    in a different laboratory for later use. However, by the
	    time we got to the other laboratory the cells had died. We
	    learned from this experiment that we should not make the
	    slides too far in advance. We also learned that we should
	    use a legitimate cell viability assay rather than a crude
	    counting method. Finally, we realized that we should split
	    the large experiment into smaller experiments to determine
	    dosage and time point this way we will have enough time to
	    accurately complete both experiments. Then we can use the
	    optimal conditions determined by these experiments in a
	    future culminatory experiment.
	  </P><H4>Rhamnolipid Production Troubleshooting</H4><P>
	    Seeing as it seems that there were no Rhamnolipids being
	    produced by our strain from the MS and TLC data we decided
	    it would be best to break down the process to a step by
	    step level and trouble shoot at each level. To test our
	    hypothesis that there was a metabolic burden, we agree it
	    would be best to see if a high concentration of
	    rhamnolipids is toxic to cell growth and test an inducible
	    promoter. We also thought that there may be necessary
	    reagents to grow such as glucose. So, we decided to
	    replicate the exact conditions of the primary paper from
	    which we were working. We questioned the promoters as
	    well. Should we clone the natural promoter of RhlAB into
	    P. Putida. Furthermore, is the bacterial strain an issue?
	    To answer the lattermost question, we decided to check if
	    transformed E.Coli produces Rhamnolipids along with
	    P. Aeruginosa. To analyze the transcription step we ran a
	    qPCR and to assess the translation step we ran an
	    SDS-Page.
	  </P><P>
	    Based upon the TLC analysis, we made the following
	    conclusions. First, we Confirmed that purification of
	    rhamnolipids via liquid-liquid extraction and TLC analysis
	    work efficiently for cell culture as well. Next, we
	    Confirmed that PAO1 mostly produces
	    di-rhamnolipids. Lastly, both E. Coli and P. putida do not
	    produce enough rhamnolipids for TLC analysis. Comparing
	    the MS peaks from P. Arginosa, pure rhamnolipids, and
	    E.Coli + RhlAB, the peak induced by our construct was not
	    sufficiently pronounced to conclude that rhamnolipids were
	    produced.
	  </P><H4>S. epidermidis Experiments</H4><P>
	    After adding 5-fold inserts instead of the usual 3, we
	    successfully PCRed out all the parts necessary to form the
	    two recombinant plasmids needed to perform Gibson
	    Assembly. Filter conjugation of empty pC221 into
	    S. epidermidis RP62A was successful. We made glycerol
	    stocks of electrocompetent S. Aureus RN4220, OS2/pGO1 and
	    S. Epidermidis RP62A cells. We made plans to electroplate
	    empty vector and construct in the coming week. To ensure
	    that S. epidermidis could survive the secretion of
	    rhamnolipids, we conducted a qualitative survival test by
	    directly plating S. epidermidis with the necessary 1g/L
	    rhamnolipids for Mosquito repellent activity. The
	    resultant colonies appeared more sparse in
	    rhamnolipid-Epidermidis cultures than cultures with Just
	    the bacteria. In order to determine the effect of
	    rhamnolipids on already established colonies, we grew
	    S. Epidermidis on plates overnight, and then added 1g/L
	    rhamnolipids to the already present colonies. A day later,
	    the colonies in the after one day rhamnolipid addition
	    plate were slightly denser than those in the immediate
	    addition of rhamnolipid plate, but less dense than the
	    plates that had absolutely no rhamnolipids present.
	  </P><H4>Freeze Dry</H4><P>
	    We analyzed the difference between survival rate and
	    survival ratio to get a better sense of the difference
	    between the two metrics. We conducted the experiment again
	    with the four day incubation. We planned future
	    experiments in which the incubation time would be extended
	    to about a month.
	  </P><H4>Mouse and Mosquito Experiments</H4><P>
	    Given a sample IRB from Dr. Owens we drafted our own IRB
	    using our protocols and sent it to Dr. Owens for review.
	  </P><H4>Community Outreach</H4><P>
	    Because the next generation sequencing is highly
	    expensive, we decided for our community outreach that this
	    may not be the best idea. Additionally, we would have to
	    undergo training for human subjects experiments and
	    establish a new IRB protocol. We still thought it was
	    beneficial to develop a survey and FAQ about mosquito
	    borne illnesses. We drafted a version of the survey asking
	    questions about frequency of mosquito bites, knowledge of
	    mosquito borne illnesses, and preferred characteristics of
	    mosquito repellent (ex: is scent important when choosing a
	    repellent?). We sent the survey to a cohort of individuals
	    as a pilot study to find any errors in the survey data and
	    gain preliminary results. The power analysis revealed that
	    we need about 250 participants or more to obtain
	    statistically significant results.
	  </P><H2 id="7-10">7/10-7/16</H2><H4>Rhamnolipid Quantification</H4><P>
	    We grew wild type P. Aeruginosa and transformed
	    P. Aeruginosa with L1 + RhlAB, and extracted rhamnolipids
	    from the two strains using liquid-liquid extraction.
	    SFC-MS result shows that mutant P.A. produces much more
	    mono-rhamnolipids, confirming that our construct works
	    well in P. Aeruginosa.  We also tried growing mutant
	    Putida and E. Coli (transformed with L1 + RhlAB and H2 +
	    RhlAB) in a different growth media, LB + glucose.  SFC-MS
	    result shows that all of them make mono-rhamnolipids:
	    E. Coli + H2_RhlAB made the most mono-rhamnolipids, then
	    E. Coli + L1_RhlA, then putida + L1_RhlAB.
	  </P><H4>Transformation of P. putida PhaC knockout</H4><P>
	    We tried transforming the knockout strain with 4 different
	    promoters and RhlAB, but only L1+RhlAB showed GFP positive
	    colonies.  We will sequence it to confirm the operon and
	    quantify the amount of rhamnolipids.
	  </P><H4>Protein Analysis</H4><P>
	    We ran the gel, but it was unclear and blurred because we
	    did not purify the gel. However, there were bands of the
	    right size for RhlA and RhlB. MS showed that rhamnolipids
	    were detected. So, we decided not to spend the time
	    purifying the gel.
	  </P><H4>qPCR</H4><P>
	    Results show that mRNA is being properly synthesized by
	    our construct, although our negative controls (wild-type
	    and empty vector) indicated some unexpected mRNA.
	  </P><H4>Keratinocyte Experiments</H4><P>
	    We obtained our cells, but they were highly confluent
	    during the weekend. We performed experiment testing effect
	    of different bacterial strains (paK, WT PP, PP+L1+Rham,
	    and WT PP with 1g/L mixed rhamnolipid) with different MOIs
	    (10, 100, 1,000, 10,000, 100,000) on keratinocyte
	    viability. Cell survival was testing with the MTS cell
	    viability assay, which measures the level of live cells
	    per sample. Keratinocytes were first plated at 3x10^4 per
	    well into 48 well plates, then weaned off P/S antibiotics
	    on the next day. The following day, the different strains
	    were each added with the different MOIs with 3 trials per
	    situation. Positive control was a &quot;0 MOI&quot; situation with
	    no bacteria added, which should represent maximum cell
	    viability, and two negative controls were pure media and
	    pure media with 100,000 MOI of each bacterial strain,
	    which should give a cell viability of 0. This was repeated
	    at 2 time points: 3 and 24 hours. Data from the 3 hour
	    time point has been collected.
	  </P><H4>Molecular Cloning</H4><P>
	    We tested the Staphylococcus protocols for miniprepping,
	    electroporation, and filter-mating with empty vectors. We
	    also constructed our recombinant pC221+RhlAB+GFP and
	    pC194+RhlAB+GFP plasmids for transformation into the Staph
	    strains. To more closely observe the effect of
	    Rhamnolipids on S. epidermidis, we performed a gradient
	    plate experiment which once again showed sparser colonies
	    for S. epidermidis with higher concentrations of
	    Rhamnolipids. We decided that designing a quantitative
	    experiment with a 96-well plate, instead of just relying
	    on qualitative experimentation was necessary.
	  </P><H2 id="7-17">7/17-7/23</H2><H4>Metabolics Troubleshooting</H4><P>
	    Given the low yield of rhamnolipids from the successfully
	    cloned promoters of our promoter library, alternative
	    promoters were explored to possibly boost production.
	    Experiments were planned to more closely replicate the
	    Wittgen paper by extracting the RhlAB operon with its
	    natural promoter by PCRing out the operon along with 200
	    extra bps upstream.  Cloning this new operon had limited
	    success and rather than repeating experiments it was
	    deemed less relevant.  More priority has been placed on
	    designing/extracting the Xylose inducible promoter given
	    that it might be viable for usage in Staph epidermidis as
	    well.
	  </P><H4>Promoter Characterization</H4><P>
	    Most of the higher strength promoters have been unreliable
	    in terms of transformation and rhamnolipid production.  It
	    is speculated that the issue with this lies in that they
	    are constitutively triggering the RhlA gene to make the
	    rhamnolipid precursor HAA while the RhlB is only active
	    during the stationary phase.  Buildup of HAA might be
	    causing the bacteria to view the operon as impeding
	    survivability and in some way shut it down.  This could be
	    solved using the Xylose inducible promoters so that the
	    buildup of HAA is prevented and high levels of
	    rhamnolipids would be produced upon induction in the
	    stationary phase.
	  </P><H4>Keratinocyte Experiments</H4><P>
	    Keratinocytes were treated with different MOIs of bacteria
	    strains Putida, Arginosa, Mutant, and Putida with
	    Rhamnolipid to determine the relationship between
	    bacterial concentration and cell viability. The cells were
	    treated with gentamicin after 3 or 24 hours and then an
	    MTS assay was utilized to quantify cell viability. The
	    cells were then plated to deduce if bacteria had been
	    drawn in by the cells. The results showed that there was
	    not a significant difference in the bacterial
	    concentrations, which may indicate that the serial
	    dilutions were incorrect. Additionally, there was a high
	    degree of bacterial growth on the plates, which was not
	    expected. Perhaps, the gentamicin did not kill the
	    bacteria on the cells either because the concentration was
	    too low or the the incubation time period was not long
	    enough.
	  </P><P>
	    Another experiment was conducted to determine the IC50 of
	    rhamnolipid by treating the keratinocytes with different
	    concentrations of rhamnolipids for 24 hours. The
	    concentrations selected were based upon the results of
	    several research papers that utilized rhamnolipids with
	    different cell lines. Graphpad software was utilized to
	    model the the data and it was determined that the IC50 is
	    between 40 and 60 ug/ml.
	  </P><H2 id="7-24">7/24-7/30</H2><H4>Molecular Cloning</H4><P>
	    Tired of waiting for the xylose inducible promoter
	    backbone plasmid, we decided to utilize what we have
	    already available in the lab and clone arabinose inducible
	    promoter into our construct. We designed and ordered the
	    primers for the new construct with the arabinose promoter.
	  </P><H4>Rhamnolipid Quantification</H4><P>
	    This week, we wanted to see if saturating LB with glucose
	    helps producing rhamnolipids.  We grew mutant putida with
	    LB + 10g/L of glucose, 30 g/L of glucose, and 50 g/L of
	    glucose and measured the amount of rhamnolipids at
	    different time points. After 24 hours of growth, 50 g/L
	    glucose gave the most amount of rhamnolipids.  After 72
	    hours of growth, all three growth media gave slightly less
	    rhamnolipids.  It is possible that bacteria are eating up
	    rhamnolipids as they run out of carbon sources during the
	    stationary phase. We also created a more accurate standard
	    curve using 95% pure mono-rhamnolipids.
	  </P><H4>Keratinocyte Experiments</H4><P>
	    Based upon our suspicion that the gentamicin was not
	    killing the bacteria, we conducted an experiment in which
	    we manipulated the concentration of gentamicin and the
	    incubation time points using four different strains of
	    bacteria (E.coli, W.T. Putida, Aeruginosa, and Mutant
	    strain). The highest time point and concentration was four
	    hours at 200 ug/mL yet these conditions were not
	    successful on all of the strains.
	  </P><P>
	    We planned repeat the bacteria MOI experiment with
	    different, accurately calculated MOIs knowing the new
	    information about gentamicin incubation time and
	    concentration. Because the 4 hour time point was
	    unsuccessful, we were unsure for how long to incubate the
	    cells in gentamicin before completing the MTS assay. Thus,
	    we decided to leave the cells in gentamicin overnight and
	    concomitantly treat a separate bacterial culture with
	    gentamicin overnight to compare the bacteria growth.
	  </P><P>
	    We encountered another problem with the experiment because
	    our control cells were contaminated by bacteria. We could
	    not bring the bacteria into the cell culture hood so we
	    brought the cells to the bench when we added the bacteria
	    under flame. Even with these precautions there was
	    contamination. After discussion, we decided to continue
	    the experiment by treating with gentamicin overnight,
	    thinking that the gentamicin will kill the contamination
	    in the negative control conditions as well. While this
	    experiment may not yield all the results we wanted
	    (specifically bacteria intake by the cells), we will be
	    able to determine if there is a difference in cell
	    viability given different MOIs.
	  </P><H4>Transformation of S. epidermidis</H4><P>
	    We successfully transformed pC194_RhlAB into the shuttle
	    vector for electroporation, S. Aureus RN4220, however
	    transformation of pC221_RhlAB into the conjugation vector,
	    OS2/pGO1, was unsuccessful. We re-tried electroporating
	    pC221_RhlAB into OS2/pGO1 a couple more times, but we were
	    unsuccessful. We also conducted the 96-well S. EPidermidis
	    Experiment, however, the Rhamnolipid blank began to grow
	    around the 7;12 hr mark, making it necessary for us to
	    disregard the results.
	  </P><H2 id="7-31">7/31-8/6</H2><H4>Metabolics Troubleshooting</H4><P>
	    We have successfully cloned the arabinose inducible RhlAB
	    backbone and transformed into 10-beta E. Coli strain.  We
	    have also transformed it into Putida, which gave us
	    bacterial lawns.  Surprisingly, the lawns were GFP
	    positive, and we learned that Arabinose isn't really
	    inducible in Pseudomonas.  We will still compare its
	    expression level to our original construct.
	  </P><H4>Keratinocyte Experiments</H4><P>
	    We conducted more experiments to determine at what
	    concentration of gentamicin and time point successfully
	    kills the bacterial strains. However, they were
	    unsuccessful again. We spoke to our mentors and decided
	    that a biofilm may be forming that prevents the bacteria
	    from dying. We conducted OD experiments to determine at
	    what point the bacterial strains are in mid exponential
	    phase. The results were inconclusive for all of the
	    bacterial strains except for E.Coli. Thus, we tried to
	    conduct an experiment to assess which MOIs of E.Coli cause
	    cell death but there was an error with our cell culture
	    and the cells were dead in the morning. We now think that
	    the problem with our experiments is that we are using very
	    high bacteria concentrations. We made plans to repeat the
	    experiments with lower concentrations.
	  </P><H4>Molecular Cloning</H4><P>
	    We successfully electroporated pC221_RhlAB into S. Aureus
	    RN4220. We intend to electroporate mini-prepped pGO1
	    helper plasmid into RN4220, and then filter conjugate from
	    here, since we were unable to electroporate pC221_RhlAB
	    directly into OS2 which already contained pGO1. After
	    Millipore filtering our Rhamnolipids solution, we repeated
	    the 96-well S. Epidermidis vs. Rhamnolipids survival
	    test. Our negative control was S. epidermidis with 2uL of
	    34 ng/uL Kanamycin, while our positive control was
	    S. epidermidis with just BHI. After adding a 1/1000th
	    dilution of overnight to a range of two-fold serial
	    dilutions from 2000mg/L to 15.6mg/L. The resultant OD600s
	    showed that growth of S. Epidermidis is staggered by
	    higher concentrations of Rhamnolipids up to125mg/L of
	    Rhamnolipids, although the OD600 Does eventually get to OD
	    1 by the 12th hour. So although production of
	    Rhamnolipids in the high concentrations we need might
	    stagger the growth of S. epidermidis, it will not
	    necessarily kill our cells.
	  </P><H4>IRB Update</H4><P>
	    We have submitted a draft of IRB amendments to IACUC,
	    which we are adding on to Dr. Owens protocol. Since we are
	    changing Dr. Owens’ pain level to D, as a result of the
	    skin-grafting component of our mouse experiments, we are
	    getting a vet approval which will be sent to the IACUC as
	    well this week. We have also been assigned an IACUC
	    consultant, Sierra Fuller, who has been advising us on the
	    level of detail that must be included in the protocol.
	  </P><H4>Marketing</H4><P>
	    We have started thinking about how we can market our
	    mosquito repellent.  Using AutoCad, we will 3D print our
	    marketing bottle!
	  </P><H1 id="august">August</H1><H2 id="8-7">8/7-8/13</H2><H4>Metabolics Troubleshooting</H4><P>
	    It looks like there was a recombining when we transformed
	    the arabinose inducible promoter plasmid into Putida.
	    Even though the colonies were GFP positive, sequencing
	    result gave us a negative result.
	  </P><H4>Keratinocyte Experiments</H4><P>
	    This week the cells died because they were contaminated
	    with bacteria. We plated the cells to confirm our
	    hypothesis. We contacted the laboratory that gave us the
	    cells. The graduate student will not have them available
	    for another few weeks because their cells were also
	    contaminated. Thus, these experiments will be put on
	    hold. However, we decided to try a different assay
	    technique in which we lyse the cells and then measure
	    bacteria presence. This will avoid the complications
	    regarding the concentration and duration of gent
	    application to the cells.
	  </P><H4>Community Outreach</H4><P>
	    We created a map based upon the survey data illustrating
	    what regions of the country experience more mosquito bites
	    than others. We collaborated with the University of
	    Georgia and assisted them on their project. We also
	    contacted BioBus an organization in which scientists teach
	    students about the field while they are on a bus to plan
	    an event with them. However, the scientists will not be
	    back to work until September. We made arrangements to
	    contact them in September. Additionally, we made contact
	    with the Chinese iGEM Team who is putting together a
	    newsletter and asked if we would like to contribute.
	  </P><H2 id="8-14">8/14-8/20</H2><H4>Mosquito Experiments</H4><P>
	    Last Friday, we sourced Aedes Aegypti mosquitoes from the
	    Vosshall lab at Rockefeller University, who kindly gave us
	    two large tupperwares full of larvae and pupae and braved
	    NYC traffic to drive us back to CUMC. The mosquito life
	    cycle consists of 4 stages: eggs, larvae, pupae, and
	    adult. On Saturday, Sunday, Monday, and Tuesday, we sorted
	    out the male mosquitos (cannot suck blood) in both pupal
	    and adult stage. The pupae are sexed by size (females are
	    larger) and adults are sexed by antenna (males have fuzzy
	    antennae while females do not)[insert picture] We spend
	    countless hours in the cold room sorting adult and pupal
	    mosquitoes because the cold prevents mosquitoes from
	    escaping.
	  </P><H4>Keratinocyte Experiments</H4><P>
	    This week we conducted an experiment using smaller
	    bacterial MOIs to determine if the high MOI was the reason
	    that gentamicin was ineffectual. All four bacterial
	    strains were plated at MOIs that do not exceed 10. The
	    following day there was no evidence of bacterial growth!
	    Thus, when the new keratinocytes are received this
	    concentration of gentamicin can be utilized in the
	    experimental procedure.
	  </P><H4>Community Outreach</H4><P>
	    We arranged an event scheduled for October 1st with the
	    organization ThinkSTEAM that promotes science and math
	    education in girls. The program will consist of a
	    presentation introducing systems biology, research
	    overview, and laboratory demonstrations.
	  </P><H2 id="8-21">8/21-8/27</H2><H4>Keratinocyte Experiments</H4><P>
	    This week we obtained the flask from the Prince Lab
	    because they resolved the microplasma problem. I fed the
	    flasks and split them. However, when I came back to the
	    lab after the weekend there were fungi growing in the
	    flasks. I checked the incubators and there is a warning
	    that filters need to be replaced, which may explain the
	    contamination. I notified the laboratory of the
	    contamination issue. I contacted other laboratories in the
	    building to see if we can use incubators while waiting for
	    the incubators to be replaced and one laboratory agreed.
	  </P><H2 id="8-28">8/28-9/3</H2><P>
	    I was able to plate cells this week and conduct an
	    experiment to determine if E.Coli causes cell death. We
	    utilized a new essay from a paper entitled,
	    &quot;Internalization of Pseudomonas aeruginosa Strain PAO1
	    into Epithelial Cells Is Promoted by Interaction of a T6SS
	    Effector with the Microtubule Network.&quot; We utilized the
	    gent concentration specified in the article and lysed the
	    cells prior to plating. The experiments revealed that the
	    bacteria did not cause significant cell death at any
	    concentration. Additionally, when the cells were plated
	    bacterial colonies did not grow, indicating that the
	    bacteria was not internalized by the cells.
	  </P></DIV></DIV></DIV><DIV class="footer"><DIV class="container"><DIV class="row"><DIV class="col-md-4"><P>116th Street and Broadway, New York, NY 10027</P></DIV><DIV class="col-md-4"><P><A href="https://2016.igem.org/Team:ColumbiaU_NYC/Safety">Safety</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>