Team:TU Dresden/Project/Results
Home
Project
Overview
Description
Background
Methods
Results
Parts
Modeling
Conclusions
Team
About Us
Attributions
Gallery
Official Team Site
Contact Us!
Sponsors
Video
Notebook
Protocols
Lab Journal
Plasmid Design
Human Practices
Sustainability
Social Outreach (LNdW)
Education
Team Collaborations
Meetings
Safety
Results
Folding study of target protein
Structure analysis of our targets and their interactions
Investigation of P3 threshold for E. coli resistance
Conversion of BACTH into an iGEM standard and analysis of function
Set up of flow system
Biobrick assembly
Folding study of target protein
Transformation experiments
The plasmid encoding the soluble part of the HER2 fragment was designed as per Biobrick specifications and synthesized by IDT (HER2 codon) (figure 1).
Figure 1 - pIDTSMART-KAN plasmid encoding the soluble part of the HER2 protein coding sequence. Synthesized by IDT, contains kanamycin resistance for selection.
Since the above plasmid is not an expression plasmid, the HER2 fragment had to be subcloned into another plasmid (pET28a), which is a common expression plasmid containing N/C-terminal Histidine tag. The HER2 fragment was amplified from the Biobrick plasmid pIDTSMART-KAN using primers - subcl_fwd and subcl_rev created with and additional restriction site NheI. The cloning was done by conventional method, digestion of both the HER2 amplified fragment and the pET28a vector with NotI and NheI restriction enzymes (figure 2).
Figure 2 - pET28a plasmid linearized with both NotI and NheI. We may not be able to see the cut fragment as it is few bp in length. The PCR fragment after cut with NotI and NheI was gel purified and hence no electrophoresis data is shown.
This subcloned HER2 fragment was ligated into the restriction digested pET28a expression plasmid in such a manner that the protein has N-terminal His tag. Once the fragment is inserted, we sequenced the plasmid. Good correlations was observed between the sequenced plasmid with the original HER2 sequence [data not shown here]. We also confirmed the same with a restriction digestion of the HER2 inserted pET28a (figure 3).
Figure 3 - Restriction digested of modified pET28a + HER2. The 600 bp fragment indicated with a green line shows HER2 has been cleaved out of the pET28a plasmid.
The resultant plasmid was further subcloned into expression vector E. coli BL21 with kanamycin resistance for selection. Plates produced good transformation efficiency (figure 4).
Figure 4 - E. coli BL21 transformed cells containing the modified pET28a + HER2.
The plasmid was re-extracted from the transformed E. coli BL21 and restrict digested with NotI and NheI to reconfirm is the correct plasmid was transformed (figure 5).
Figure 5 - Restriction digestion of pET28a + HER2 obtained from E. coli BL21 transformed cells. Lane 1 shows 1 kb ladder. Lane 2, 3 and 4 show restrict digested modified pET28a + HER2 plasmid. The cut pET28a is clearly visible, whereas the 600 bp HER2 fragment is not clearly visible.
Four colonies were picked from the transformed cultures and grown in 4 L flasks for protein expression experiments.
Affinity chromatography
The cell cultures were collected and lysates were extracted using french press. The resulting cell lysate was subjected to affinity chromatography using Nickel beads to purify the His-tagged HER2 protein. The protein fractions were collected in different eppendorf tubes. Fractions A1-A11 were selected as per the peaks in the chromatogram as shown in figure 6.
Figure 6 - Affinity chromatogram of the protein lysate. The arrows in blue indicate the volume of the fraction collected (A1-A11) for further analysis using size exclusion chromatography.
Size-exclusion chromatography
Further size-exclusion chromatography was performed to separate dimer, oligomers of proteins. Fractions C9-D15, D14-D5, D6-E5 were collected separately as per peaks in the chromatogram as shown in figure 7.
Figure 7 - Size Exclusion chromatogram, there elutes were obtained: elute 1 (blue arrow), elute 2 (red arrow), elute 3 (grey arrow).
The different elutes indicate that there might be oligomers. Hence a SDS and native PAGE experiment was performed to check them.
SDS-PAGE
The elutes of size exclusion chromatography were taken and analysed by using SDS PAGE to re-confirm if the protein extracted correspond to the 22 kDa molecular weight of HER2. Here eluates from initial cell lysates, affinity purified lysates and size exclusion lysates were analysed to check the 22 kDa protein (figure 8a and 8b).
Figure 8A - SDS-PAGE of HER2 elutes in different fractions. Lane 1: marker (Color Plus P7712s); lane 2: cell lysate; lane 3: fraction without IPTG protein; lane 4: IPTG induced cell fraction; lane 5: 1st peak of the affinity chromatogram  (initial wash elute); lane 6: 2nd peak of the affinity chromatogram (cell debris elute); lane 7: HER2 1st elute of the size exclusion chromatography; lane 8: HER2 2nd elute of the size exclusion chromatography; lane 9: HER2 3rd elute of the size exclusion chromatography.
Figure 8B - SDS-PAGE of HER2 elutes in different fractions. Lane 1: marker (Color Plus P7712s); lane 2: cell lysate; lane 3: fraction without IPTG protein; lane 4: IPTG induced cell fraction; lane 5: 1st peak of the affinity chromatogram  (initial wash elute); lane 6: 2nd peak of the affinity chromatogram (cell debris elute); lane 7: HER2 1st elute of the size exclusion chromatography; lane 8: HER2 2nd elute of the size exclusion chromatography; lane 9: HER2 3rd elute of the size exclusion chromatography.
It is evident from the figures 8A and 8B that the 22 kDa protein (HER2) is present in small quantities in all the samples, but it is evidently seen in elute 3 from SEC. It is also visible is that elute 2 has a high banding pattern below the 80 kDa mark. This might correspond to all the 22 kDa HER2 protein aggregating at this point or form oligomers. To analyze the same a native PAGE experiment was done.
Native PAGE
The 3 peaks in the size-exclusion chromatography collected separately were analysed in blue native PAGE, to check if they were oligomers of the protein (figure 9).
Figure 9 - Protein purified eluates separated. Lane 1 shows 1 kb ladder. Lane 2, 3, 4: elute 1 in 1 μM, 4.5 μM, 6 μM, respectively. Lane 5, 6, 7: elute 2 in 1 μM, 4.5 μM, 6 μM, respectively. Lane 8, 9, 10: elute 3 in 1 μM, 4.5 μM, 6 μM, respectively. Here the elute 3 in lanes 8, 9, 10 shows the 22 kDa band.
Secondary structure prediction with circular dichroism (CD)
The protein fractions collected were in separate dialysis bags and dialyzed overnight to minimize the effect of chloride salts making CD measurements. The CD data for fraction 1 showed pure monomeric form of protein. As per table 1 and figure 10, the CD plot of wavelength (nm) vs circular dichorism (mDeg) shows that the protein is largely consists of alpha-helices and a fair amount of random coils.
Table 1 - Percentage of each secondary structure.
Secondary structure
190 - 260 nm
Helix
22.7 %
Antiparallel
27.9 %
Parallel
10.6 %
Beta-turn
19.9 %
Random coil
34.3 %
Total sum
103.0 %
Figure 10 - Secondary structure CD spectra through size-exclusion chromatography. The negative curves represent the characteristics of alpha helical structure.
Here all three eluates show different curve slopes, this might be due to the concentration of the protein and the presence of oligomers.
Based on the SDS data, the elute 3 was selected for further denaturation experiments with CD as it showed a higher concentration of 22 kDa HER2 protein.
Figure 11 - Results obtained from the circular dichroism. CD profile for elute 3 (A). Temperature based CD measurements from from 19.9 °C, 29.3 °C, 39.0 °C, 48.9 °C, 58.7 °C, 68.7 °C, 78.7 °C, and 88 °C (B). Re-folding of HER2 spectral run on CD. Graph shows data at 20.9 °C.
Upon denaturation the alpha helical structure disrupts slowly and forms random coils. It is observed that in the temperature range of 50 - 60 °C there is rapid denaturation.
It is clearly evidenced from figure 11.c that upon re-folding of the denatured protein down to a temperature of 20 °C still does not revive the protein back to its native structure. Hence we can conclude that the denaturation is not reversible as we observed that, the negative curve is similar to the pattern of random coil.
Comparison of CD data with PDB structural data
The original soluble part of HER2 structure was already predicted using NMR and the PDB-ID is 3MZW . We compared our protein's folding pattern with to analyze the correctness.
CLUSTAL O (1.2.1) multiple sequence alignment:
With the comparison with the crystal structure of we found that the aligned region matches to region of receptor L domain and Furin-like domain of the extracellular domain of (figure 12). This region is known to contan beta sheets and alpha helices. It is also observed from the de-convolution data from HER2 CD spectra that there is a fairly large amount of beta sheets and alpha helix.
Figure 12 - Pdb secondary structure data for HER2 (pdb id: 3MZW)corresponding to multiple alignment.
Structure analysis of our targets and their interactions
Structure check of HER2
Before starting a structural analysis on an atomic level the protein structure has to be validated and checked for possible errors which can occcur during assignment of cordinates to the atoms.
For structures resolved by X-ray crystallography the resolution is the most important experimental parameter. The smaller it is the better the exact atom positions can be defined. The HER2 structure 3MZW was obtained with a resolution of 2.9Å. In general this is not a good resolution, but starting from a resolution of 3Å the correct side chains can be mapped to their respective electron clouds.
A second important parameter is the R-factor. It is a measure of the agreement between the crystallographic model and the experimental X-ray diffraction data. In other words, it is a measure of how well the refined structure predicts the observed data. 3MZW, having a R-factor of 0.208, it lies perfectly in the area of typical values of about 20% and also the value of its R-free (0.278) is within the acceptable area of under 30%.
A method to check the stereochemical quality of a protein structure is the validation of the dihedral angles of each amino acid. This is provided by the Ramachandran plot.
The Ramachandran plot shows the phi-psi torsion angles for all residues in the structure (except those at the chain termini). Glycine residues are separately identified by triangles as these are not restricted to the regions of the plot appropriate to the other sidechain types.     The colouring/shading on the plot represents regions with different favorability: the darkest areas (here shown in red) correspond to the "core" regions representing the most favourable combinations of phi-psi values.     Ideally, one would hope to have over 90 % of the residues in these "core" regions. The percentage of residues in the "core" regions is one of the better guides to stereochemical quality.
Interactions of HER2 and its affibody
After definition of the interfaciual atoms, electrostatic interactions in the interface can be defined and visualized as shown in figure 13.
Figure 13 - Electrostatic interactions of HER2 and its affibody ZHER2 shown as dashed yellow lines. Labels indicate the respective atom distances.
A total number of 9 hydrogen bonds were identified between HER2 and its affibody using a distance cutoff of 3.2Å and an angle cutoff of 55 degrees. Those are listed below with their respective distances.
Conservation study of HER2
In order to get an impression about possible variabilities of the HER2 structure a conservation study of HER2 was performed using 11 structures from different organisms (figure 14). The multiple sequence alignment which is required for the calculation can be seen here . Looking at the binding interface of HER2 and its affibody, we can state that the regions where both get into contact are rather conserved.
Figure 14 - HER2 conservation - calculated using 11 HER2 structures from different organisms.
Negative results:
Performing the rather automatic analysis of HER2 conservation by using all available HER2 structures gives a very large amount of structures to compare with.
This results in large alignment gaps and in an overall relatively low conservation without larger shade differences except for single amino acid (the whole Amino Acid Conservation Scores can be found here and the first lines of the color coded alignment can be seen here ). Therefore the the sample is too large for a nice visualization and also the database structures might be biased.
In case of the affibody a conservation analysis could not be performed since it is an artificially engineered molecule.       Therefore, in order to nevertheless get an impression about possible variabilities of the affibody structure an analysis of its cristallographic B-factors was performed.
Visualization of the B-factor for the affibody ZHER2
In crystallography the B-factor, also called temperature factor or "Debye-Waller factor", describes the displacement of an atom from its mean position in a crystal structure. The displacement may be the result of temperature-dependent atomic vibrations or static disorder in a crystal lattice. Static disorder means that some regions of the molecule may adopt different conformations in different copies of the molecule, each molecule's conformation being relatively stable. In the case of our affibody static disorder is not so probable, since it is a very small protein, designed to adopt a stable conformation.
Reflecting the disorder of an atom, the B-factor is therefore an indicator for flexibility caused by thermal motion.
As depicted in the following pictures the affibody has low B-factor values, meaning that it stays in a stable position without any larger fluctuations (indicated by the blue color). Only at the ends of the molecule a slight increase of the B-factor can be stated (figure 15). This is normal and due to thermal motion, since the atoms have less interaction partners there, which can hold them on place.     This stable position of the affibody suggests a high binding affinity at this position.
Affibody ZHER2 surface coloured by b-factor
Affibody ZHER2 structure coloured by b-factor
Affibody ZHER2 structure coloured by b-factor
Figure 15 - The affinity matured 3-helix affibody ZHER2 binding to HER2 (PDB-ID: ). Affibody coloured by B-factor (colour gradient: blue - green - red), HER2 in grey.
Video
The following video shows the structure of the extracellular regions of HER2 with the affinity matured 3-helix affibody ZHER2 (PDB-ID: 3MZW ) and focuses on their interaction, whereas hydrogen bonds are represented as dashed yellow lines and then the complete interacting interface is represented as surface, colored by atom type (N-blue, O-red).
Your browser either does not support HTML5 or cannot handle MediaWiki open video formats. Please consider upgrading your browser, installing the appropriate plugin or switching to another browser.
Investigation of P3 threshold for E. coli resistance
Transformation of E. coli with the construct
To study the resistance that is build up by E. coli as response to P3 production, we constructed a plasmid that carries P3. The expression can be controlled by the inducible lac promoter. At the same time the reporter CFP is co-expressed (figure 16).
Figure 16 - Final construct of pLac, P3, RBS and CFP. The induction of the promoter leads to the expression of P3 and the coexpression of CFP as a reporter.
The plasmid was transformed into the F+, Δ(lacZ) E. coli strain after its construction. The F+ gives the M13 phages the possibility to infect the bacteria. Because of the lacZ deficiency the alpha-complementation can be carried out by the phages and the cells can be used for lacZ/blue-white screening. To have a very simple check for the functionality of the construct, we placed the induced cells on a UV table. A clear blue fluorescence could be seen, but not distinguished from the auto-fluorescence of the LB medium. Thus we performed the more sophisticated analysis of the induced cells with a 3D scan over a wide range of excitation and emission wavelengths.
To get hands on the grade of resistance we used the blue-white screening that gives a ratio of infected (blue) to uninfected cells (white). Before we conducted the measurement in the reactor we tested if the blue-white screening works as assumed in simple reaction tubes (figure 17) and on LB plates (figure 18).
Figure 17 - Left: ER2738 cells induced with X-Gal but without M13mp18. Right: Production of blue dye as a response to the infection with M13mp18. The phage performs the alpha-complementation of the damaged Β-galactosidase. The enzyme can then convert X-Gal into the blue indigo dye.
Figure 18 - The plate was covered with top-agar which contained infected and uninfected cells. A continuous bacteria carpet developed and turned blue at the spots where infected cells grew. The bigger blue spots arose from condensate drops that allowed the phages to spread after the top agar was deposited.
The first cultures in the minimal medium showed a growth in the expected range until an OD600 of roughly 0.4. After transferring the culture into a second flask, the growth decreased, leading to a final OD600 around 1. After transferring parts of the culture into a new flask with fresh medium no growth could be detected, even though the prolin which the strain needed was added. The experiments were repeated several times with the result that the strain was missing more nutrition which was provided by the initial cells coming from the LB-plates. We decided to move the more intense medium missing studies to a later date and try to perform a proof of concept with our strain. Therefore yeast extract was added to the medium and growth could be observed. It was also decided to add the yeast extract to the medium reservoir of the continuous cultivation.
CFP expression
Two samples (one of an induced culture (2 mM IPTG) and a IPTG free culture) were measured using a fluorescence spectrometer. Both samples had an OD of 1. A fluorescence signal peak resulting from a CFP expression was expected in a range from 435 - 440 nm (excitation) and from 470 - 476 nm (emission) in the induced sample. In contrast, a lower fluorescence signal in this wavelength region was expected for the IPTG free sample due to a lower concentration of IPTG. The measured 3D fluorescence spectra are shown in figure 19. As expected, the IPTG free showed a fluorescence signal, which was around 50 to 100 units lower than in the induced culture. This indicates a lower CFP expression in the IPTG free sample. However, the CFP expression in the induced sample was too low to see a clear peak in the fluorescence spectrum. This might be a result of the co-expression of IPTG after the P3 expression. Furthermore, the cell culture could have entered the static phase, which could result in a nutrition limitation and a lower CFP expression.
Figure 19 - 3D fluorescence spectra of an induced E. coli culture sample (left: 2 mM IPTG) and a E. coli culture sample without IPTG (right). Both samples had an OD of 1.
Analysis of the plasmid stability
The plasmid stability was analyzed with the samples for the phage infection. The stability of the plasmid showed itself around 90 % over the whole cultivation the missing 10 % are most likely a result of problems with the transferring of the cultures with the stamp. Therefore the high plasmid stability in the system can be assumed. The antibiotics are degraded by the chloramphenicol acetyltransferase. Thus it is likely that most of the antibiotics are being deactivated during the cultivation leaving the assumption that the plasmid might be stable even without an antibiotic in the medium. To ensure the stability more experiments will be performed with an antibiotic free medium. The minimal medium that will be used also helps to ensure a low infection rate.
Analysis of the phage infection
As a next step the phages were added to the lagoon and cultivated for 2 hours. A sample was taken and plated on the X-Gal plates as a result around 25 % of the colonies showed a color change as a result of the infection. Over the next time the IPTG concentration was increased step-wise. All samples except the one with the 3 mM IPTG concentration showed some infected colonies which leads to the first assumption that, only after a concentration of 3 mM of IPTG, the induction is high enough to stop the infection. But, as all of the induced samples showed a low infection rate, it might be possible that the sample only contained a small amount of phages. Due to a lack of time, only a small part of the planned continuous cultivations could be performed. Therefore, further cultivation is required to determine the phage infection. Furthermore, it should be considered to analyze the expression level achieved with the different IPTG concentrations.
Conversion of BACTH into an iGEM standard and analysis of function
The inserts T25, T18, LZT18 and LZT25 all fit the iGEM Biobrick standard which meant that they had fixed prefix restriction sites and suffix restriction sites. The vectors they came in had kanamycin resistance.
Fusion ligation of T18, LZT18 and T25, LZT25
The plasmids containing T18, LZT18 and T25, LZT25 were restriction digested and gel electrophoresis was performed to purify them (figure 20).
Figure 20 - Gel run with the different plasmids. The second lane shows two bands (restriction digest of T25): the upper band is the plasmid construct and the lower band is the T25 that has been cut out. Similarly the third lane shows two bands of which the lower band is the T18 and the upper is its plasmid construct. Lanes four and five show the restriction digests of the LZT25 and LZT18, respectively. The upper bands in these lanes correspond to the vector containing LZT25 and LZT18 and the lower bands are short sequences of DNA that have been cut out to make the vector linear.
After this gel run, the necessary bands were eluted out and the inserts were ligated with their respective vectors. An electroporation was done to transform E. coli GBO5 with these plasmids and were streaked onto kanamycin resistant plates that gave colonies meaning successful transformation (figure 21).
Figure 21 - Transformed E. coli on kanamycin resistant plates. Colonies were seen in both T18/LZT18 transformed colonies (left) and T25/LZT25 transformed colonies (right).
Ligation of fusion products: T18-LZT18 and T25-LZT25
Selected colonies from the above plates were cultured to extract plasmids. These plasmids were then restriction digested and a gel electrophoresis was carried out to purify them (figure 22).
Figure 22 - Gel run with the restriction digests. The second lane shows restriction digest of T18/LZT18. There is only one band because the other DNA fragment is very small, and the band visible contains the linearized plasmid containing T18/LZT18 which acts as the vector. The third lane shows restriction digest of T25/LZT25 and the lower band contains T25/LZT25 which acts the insert.
After this gel run, the necessary bands were eluted out and the insert was ligated with the vector. An electroporation was done to transform E.coli GBO5 with these plasmids and were streaked onto kanamycin resistant plates that gave colonies meaning successful transformation (figure 23).
Figure 23 - E. coli colonies with the plasmids. The colonies seen present T18-LZT18 and T25-LZT25.
Ligation with lacZ
Selected colonies from the above plates were cultured to extract plasmids. These plasmids were then restriction digested and a gel electrophoresis was carried out to purify them (figure 24).
Figure 24 - Gel run after digesting the plasmids. The second and the fourth lanes show restriction digest of T18/LZT18 + T25/LZT25. The lower band in these lanes corresponds to the T18/LZT18 + T25/LZT25 cassette that acts as the insert and a linearized pLac Biobrick (pSB1C3 backbone) acts as the vector.
After this gel run, the necessary bands were eluted out and the insert was ligated with the vector. An electroporation was done to transform E.coli BTH101 with this plasmid and was streaked onto X-Gal plates. The plate showed several white colonies along with blue colonies which meant that the transformation was not good. This step should be repeated to achieve the hypothesised result (figure 25).
Figure 25 - X-Gal plate with blue colonies. These blue colonies correspond to E. coli BTH101 with the T18/LZT18 + T25/LZT25 plasmid. Although it cannot be properly seen, the number white colonies made the transformation unacceptable.
The vector map of the final product is given below (figure 26).
Figure 26 - Structure of the final product that the blue E. coli BTH101 colonies contain.
Set up of flow system
Continuous stirred-tank cultivation
The continuous cultivation was performed after the initial cultivation. The experience with the minimal media lead to the decision to use the minimal media with yeast extract. The initial batch cultivation was performed overnight and had a final OD600 of 0.43. The next step the continuous cultivation was started. After roughly 9 hours and the resulting 2 volume changes, the OD600 stabilized around 0.18 and reached the continuous state of the cultivation. Then, the plasmid stability and phage infection could be analyzed.
Biobrick assembly
The parts synthesized from IDT were restrict digested with EcoRI and PstI and were run on a gel (figure 27 and 28.)
Figure 27 - Restriction digestion of the biobricks with EcoRI and PstI. Lane 1 shows the 1 kb ladder; lane 2 shows pUCIDT-KAN: gene III fusion; lane 3 shows pUCIDT-KAN: T25; lane 4 shows pIDTSMART-KAN: HER2 codon; lane 5 shows pIDTSMART-KAN: T18 on 0.7 % agarose gel.
Figure 28 - Restriction digestion of the biobricks with EcoRI and PstI. Lane 1 shows the 1 kb ladder; lane 2 is left empty and 3 pIDTSMART-KAN: ZHER2; lane 4 is pIDTSMART-KAN: LZT18; lane 5 pIDTSMART-KAN: LZT25, on 2 % agarose gel.
The digested parts were ligated individually with the iGEM standard vector backbone pSB1C3. Then these parts were transformed into E. coli GB05 and good transformation efficiency was observed.
Two clones of each part transformed plate was selected for sequencing. Also control digests were setup to re-confirm the inserted genes (figure 29 and 30).
Figure 29 - Control digests of final plasmids with the pSB1C3 backbone. Lane 1 shows 1 kb ladder; lane 2 - 5 show BBa_K1781000 , lane 6 - 8 show BBa_K1781001 ; lane 9 - 11 show BBa_K1781003 , lane 12 - 14 show BBa_K1781004 .
Figure 30 - Control digests of final plasmids with the pSB1C3 backbone. Lane 1 shows the 1 kb ladder; lane 2 and 3 belong to BBa_K1781002 ; lane 3 and 4 correspond to BBa_K1781006 ; lane 6 and 7 show BBa_K1781005 .
After confirmation of sequenced DNA with original part sequence, sent dried DNA sample to iGEM  headquarters and registered parts.
To the top!
