<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Dresden_Notebook_LabJournal skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Dresden/Notebook/LabJournal</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV><UL><LI class="cat1"><A href="/Team:TU_Dresden">Home</A></LI><LI class="cat2"><A href="https://2015.igem.org/Team:TU_Dresden/Project">Project</A></LI><LI><A href="/Team:TU_Dresden/Project/Overview">Overview</A></LI><LI><A href="/Team:TU_Dresden/Description">Description</A></LI><LI><A href="/Team:TU_Dresden/Project/Background">Background</A></LI><LI><A href="/Team:TU_Dresden/Project/Methods">Methods</A></LI><LI><A href="/Team:TU_Dresden/Project/Results">Results</A></LI><LI><A href="/Team:TU_Dresden/Project/Parts">Parts</A></LI><LI><A href="/Team:TU_Dresden/Modeling">Modeling</A></LI><LI><A href="/Team:TU_Dresden/Project/Conclusions">Conclusions</A></LI></UL><LI class="cat3"><A href="https://2015.igem.org/Team:TU_Dresden/Team">Team</A></LI><LI><A href="/Team:TU_Dresden/Team/Aboutus">About Us</A></LI><LI><A href="/Team:TU_Dresden/Team/Attributions">Attributions</A></LI><LI><A href="/Team:TU_Dresden/Team/Gallery">Gallery</A></LI><LI><A href="https://igem.org/Team.cgi?id=1781">Official Team Site</A></LI><LI><A href="/Team:TU_Dresden/Team/Contact">Contact Us!</A></LI><LI><A href="/Team:TU_Dresden/Team/Sponsors">Sponsors</A></LI><LI class="cat5"><A href="https://2015.igem.org/Team:TU_Dresden/Video">Video</A></LI><LI class="cat4"><A href="https://2015.igem.org/Team:TU_Dresden/Notebook">Notebook</A></LI><LI><A href="/Team:TU_Dresden/Notebook/Protocols">Protocols</A></LI><LI><A href="/Team:TU_Dresden/Notebook/LabJournal">Lab Journal</A></LI><LI><A href="/Team:TU_Dresden/Notebook/PlasmidDesign">Plasmid Design</A></LI><LI class="cat6"><A href="https://2015.igem.org/Team:TU_Dresden/Practices">Human Practices</A></LI><LI><A href="/Team:TU_Dresden/Practices/Sustainability">Sustainability</A></LI><LI><A href="/Team:TU_Dresden/Practices/Social_Outreach">Social Outreach (LNdW)</A></LI><LI><A href="/Team:TU_Dresden/Practices/Education">Education</A></LI><LI><A href="/Team:TU_Dresden/Collaborations">Team Collaborations</A></LI><LI><A href="/Team:TU_Dresden/Practices/Meetings">Meetings</A></LI><LI class="cat7"><A href="/TU_Dresden/Safety">Safety</A></LI></NAV><DIV id="TueContent"><H1>Lab Journal</H1><P style="margin-left:-11px;font-size:1.3em;color: #045FB4 !important;  font-weight: bold; text-shadow: 0px 0px 0px #000000 !important;        };"><A href="#" onclick="ddaccordion.collapseall('technology'); return false" style="
    color: #045FB4 !important; font-weight: bold; text-shadow: 0px 0px 0px #000000 !important;  margin: 12px ; font-size: 18px !important;  text-decoration: none;  };
">[Collapse all]</A>  | <A href="#" onclick="ddaccordion.expandall('technology'); return false" style="
    color: #045FB4 !important; font-weight: bold; text-shadow: 0px 0px 0px #000000 !important;  margin: 12px !important;  font-size: 18px !important;  text-decoration: none;  };
">[Expand all]</A></P><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">February</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Computational tool analysis for protein-protein interactions.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Contacted Dr. Pisabarro for help in Bioinformatics. Analysis performed using EMBL-EBI tools.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Antibody-antigen Analysis to find out suitable X and Y proteins to carry out the study.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Literature search for appropriate proteins for proof of principle.</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">March</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Creation of group ID.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Meeting with the God of recombineering Dr. Jun Fu to help design the plasmids.</LI><LI style="margin-bottom: 10px;line-height:1.8;">CRTD meeting with group leaders to present our idea and for potential funding.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Planning for Lange Nacht der Wissenschaften (LNdW).</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">April</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Planning for iGEM presentation in our Cellular Machines lecture.</LI><LI style="margin-bottom: 10px;line-height:1.8;">M13 Assay, plaque assay literature search and protocol design.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Appointment with Biosaxony for funding.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Material transfer agreement initiated with Prof. David. R. Liu, Department of Chemistry and Chemical Biology, Harvard University.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Affibody and HER2 chosen as the proteins of interest for the study.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Team Registration on iGEM.</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">May</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Finances come through for the trip and the visas needed to go to Boston.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Requested iDT for information on plasmid synthesis.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Survey questions from iGEM HQ filled out.</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">June</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">All plasmids designed (initial draft).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Created models, posters, and flyers for LNdW.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Meetings with Prof. Mascher (Department of Microbiology, TU Dresden), an iGEM veteran from TU Munich.</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">July</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Biobricks and new plasmid designs (<A style="text-decoration:none;" href="https://2015.igem.org/Team:TU_Dresden/Notebook/PlasmidDesign"><FONT color="#045FB4">Parts</FONT></A>).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Core team and subprojects designated.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Order of newly designed plasmids from IDT (<A style="text-decoration:none;" href="https://2015.igem.org/Team:TU_Dresden/Project/Parts"><FONT color="#045FB4">Plasmid Design</FONT></A>).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Biobrick kit arrives from iGEM HQ.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Wiki created.</LI><LI style="margin-bottom: 10px;line-height:1.8;"><I>E. coli</I> strains arrived from Macher’s Lab.</LI><LI style="margin-bottom: 10px;line-height:1.8;">IDT shipment received and lab space designated by Dr. Groß.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Contact with Dr. Micheal Schlierf contacted for protein folding experiments.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Registration for the iGEM Giant Jamboree.</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">Week 1 (20-26 July)</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><H4>Investigation of P3 threshold for <I>E. coli</I> resistance</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Plasmids were received (T18, ZHER2, LZT18, LZT25 and HER2 codon) and diluted. Also two pSB1C3 plasmids from iGEM containing RBS (<A style="text-decoration:none;" href="http://parts.igem.org/Part:BBa_B0030"><FONT color="#045FB4">BBa_E0020</FONT></A>) and CFP (<A style="text-decoration:none;" href="http://parts.igem.org/Part:BBa_E0020"><FONT color="#045FB4">BBa_E0020</FONT></A>) were diluted.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Preparation of <I>E. coli</I> GB05 and set cultures overnight.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Electroporation of the cells with the plasmids.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transfected cells were streaked out on chloramphenicol (Cm) and kanamycin (kan) plates.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Colonies were picked and mixed with Cm and kan. These were stored in the fridge.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Streaking out colonies from DHM1, BTH101 and K12 plates.</LI></UL><H4>Wiki</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Our wiki starts taking shape!</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">Week 2 (27 July-2 August)</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><H4>Investigation of P3 threshold for <I>E. coli</I> resistance</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">A MiniPrep was done with the seven cultures with the plasmids. A glycerol stock of each culture was saved.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The concentration of the plasmids was measured using the nanodrop.</LI><LI style="margin-bottom: 10px;line-height:1.8;">A digestion of the plasmids was done and a gel was run.</LI></UL><H4>Structure analysis of our targets and their interactions</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">The necessary data and sequences were collected.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Simulations were run and the results were obtained.</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">Week 3 (3-9 August)</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><H4>Folding study of target protein</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">pET28a vector preparation: digestion and dephosphorylation. Afterwards a gel was run and the DNA was extracted.
</LI><LI style="margin-bottom: 10px;line-height:1.8;">A PCR was set to create HER2 with NheI and NotI restriction sites. A digestion was done and the product was run on a gel and purified. The concentration was measured with the nanodrop. This process was repeated some days later since the concentration was too low.</LI></UL><H4>Investigation of P3 threshold for <I>E. coli</I> resistance</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">3 cultures for GB05 and K12 were set overnight.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation of <I>T25</I> and <I>gene III</I> plasmids and promoter (pLac) in <I>E. coli</I>: <I>gene III</I> in GB05 and K12; <I>T25</I> in GB05; promoter  in K12 and GB05.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformed colonies were plated on kan plates (gIII and T25) and Cm (pLac). 
A MiniPrep was done with the overnight cultures of pLac, T25 and gIII. The concentration of plasmid was measured with the nanodrop.
</LI><LI style="margin-bottom: 10px;line-height:1.8;">New back-up cultures were set overnight.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Cloning of P3 and RBS+CFP in pLac, followed by dephosphorylation and gel running and purification. The concentration was measured with the nanodrop.</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">Week 4 (10-16 August)</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><H4>Folding study of target protein</H4><UL style="padding: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Restriction digestion of HER2 and pET28a. The products were run on a gel and afterwards the weight was measured.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The gel extraction for HER2. Agarose gel electrophoresis used to confirm the size of the products. </LI><LI style="margin-bottom: 10px;line-height:1.8;">Ligation of overnight cultures (pET28a with HER2) using the vector insert calculation with the molar ratio formula.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Setup of overnight cultures of <I> E. coli</I> GB05 (3) and <I> E. coli</I> K12 (6) without antibiotics along with control in duplicates.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The transformation of the GB05 and K12 cells through electroporation with tthe following constructs was performed: water control, HER2 + pET28a (2 samples).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Plating of cells onkanamycin and the counting colonies the next day.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation was successful only in GB05 cells and K12 showed no growth. Subculturing transformed cells in 1 mL and 30 mL tubes in the incubator at 37 °C.</LI><LI style="margin-bottom: 10px;line-height:1.8;">MiniPrep of pET28a-HER2.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Restriction digestion with NotI HF and NheI HF was carried out and an garose gel electrophoresis to confirm the restriction digestion. Due to voltage changes the results on the gel could not be properly analysed and was repeated later in the week.</LI></UL><H4>Investigation of P3 threshold for <I>E. coli</I> resistance</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Ligation of pLac + P3 and RBS + CFP: first set up the cultures for transformation, and the transform the electrocompetent cells: pLac in K12, pLac+P3 in K12 and GB05, and RBS+CFP in GB05. Plate the transformed cells and incubate them overnight.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Plasmid MiniPrep from RBS+CFP in GB05 and pLac+P3 in GB05. The concentration of the plasmids was measured with the nanodrop. The products were then digested, dephosphorylated and loaded on a gel.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Plasmid prep of pLac + P3 and RBS + CFP was performed from overnight cultures. </LI><LI style="margin-bottom: 10px;line-height:1.8;">The gel purification performed and concentration measured using the nanodrop.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Ligation of linearized plasmids using T4 ligase.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation of pLac + P3 + RBS + CFP via electro-competent cells was carried out GB05 and K12 strains were used. They were plated and incubated overnight.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Very small colonies were observed on all the plates. Re-plating cultures and setting up of overnight cultures.</LI><LI style="margin-bottom: 10px;line-height:1.8;">pLac, pLac + P3, pLAC + P3 + RBS + CFP in K12 were then used for the continuous culture in our very own bioreactor to check for cyanofluorescence.</LI></UL><H4>Conversion of BACTH into an iGEM standard and analysis of function</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Restriction digestion for the biobrick constructs and BACTH buildup: ZHER2, HER2, LZT25, LZT18, T18, T25 and P3. </LI><LI style="margin-bottom: 10px;line-height:1.8;">The gel extraction for LZT18, LZT25, and T25 vector without AgeI PstI fragment. Agarose gel electrophoresis used to confirm the size of the products. </LI><LI style="margin-bottom: 10px;line-height:1.8;">3 of the samples were not extracted properly and had to be repeated.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Due to the failure in gel extraction, a plasmid prep of the LZT25, LZT18, and T18 was performed.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Following the iGEM website, the linearized plasmid pSB1C3 was digested with PSTI and EcoRI and DpnI to destroy the template DNA. (15ng/μl using nanodrop).</LI><LI style="margin-bottom: 10px;line-height:1.8;">BACTH concentrations were measured.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Ligation of the fragments in the required mannerigation of the fragments in the required manner.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The ligations of the BACTH were &quot;de-salted&quot; on a nitrocellulose membrane placed on water.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation of the 7 ligations and one negative control of water (through electroporation) into <I>E. coli</I> GB05 overnight cultures.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Plating of the BACTH ligations onto plates containing kanamycin.</LI></UL><H4>Biobrick assembly</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">The Biobricks and BacTH concentrations were measured and Ligation of fragments in the required manner.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The ligations of the Biobricks and the BacTH were &quot;de-salted&quot; on a nitrocellulose membrane placed on water.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation of the 7 ligations and one negative control of water (through electroporation) into <I>E. coli</I> GB05 overnight cultures.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Plating of the Biobricks ligations on agarose plates containing chloramphenicol.</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">Week 5 (17-23 August)</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><H4>Biobrick assembly</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Biobrick LZT25: restriction digest and ligation.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Biobricks P3, HER2, ZHER2, T25, T18, LZT18: transfected into <I>E. coli</I> GB05, 4 colonies per Biobrick transformation were picket, MiniPreps were performed the following day with control digests using EcoRI and PstI. Two positive clones per Biobrick were sent for sequencing (except for HER2, only one positive clone).</LI></UL><H4>Folding study of target protein</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">The restriction digest of pET28a with Her2 was repeated and run on a 2 % agarose gel. Concentrations were low.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Na-Ac concentration method used to concentrate the plasmid DNA and duplicates pooled together and concentration determined. Unfortunately, not enough DNA sample for sequencing.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The experiment was repeated later in the week and was successful the second time.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Sent for sequencing.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Glycerol stocks of the pET28a-HER2 clones were made.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Electroporation of <I>E. coli</I> BL21 strains with pET28a-HER2.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Setting up of overnight cultures of the transformed cells were in 6 X 5 mL tubes.</LI></UL><H4>Investigation of P3 threshold for <I>E. coli</I> resistance</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">The transformation of M13mp18 in K12 was performed.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Setting up of 4 x 1 mL overnight cultures of K12 ER2738 in LB with 3 μg/ml tetracycline.</LI><LI style="margin-bottom: 10px;line-height:1.8;">A phage assay on the K12 which were transfected with M13. Grown with X-Gal and induced with iPTG.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation for pLac-P3 in ER2783 was performed.</LI></UL><H4>Conversion of BACTH into an iGEM standard and analysis of function</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">All ligations showed colonies except pSB1C3-T18 and cultured again overnight.</LI><LI style="margin-bottom: 10px;line-height:1.8;">MiniPreps followed by the same procedure and plasmid DNA concentration determined.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Unsatisfactory results and could be due to low concentrations obtained.</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">Week 6 (24-30 August)</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><H4>Folding study of target protein</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Plasmid prep of cultures pET28a+HER2 in <I>E. coli</I> BL21.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Concentration measured.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Restriction digestion by NotI HF and NheI HF.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Overnight cultures set up for protein purification.</LI><LI style="margin-bottom: 10px;line-height:1.8;">3 clones selected.</LI></UL><H4>Investigation of P3 threshold for <I>E. coli</I> resistance</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Preparation of the LB, LB + Cm and LB + Cm + X-Gal plates for the analysis.</LI></UL><H4>Set up of flow system</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Preparation LB-medium and minimal medium for initial cultivation.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Initial cultivation to optimize composition of the minimal medium + Cm.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Inoculation of a fresh flask with minimal medium + Cm from initial cultivation.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Inoculation of the 1 L bioreactor from the second flask.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Continuous cultivation needs further optimization of the medium.</LI></UL><H4>Biobrick assembly</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Biobricks P3, HER2, ZHER2, T25, T18, LZT18: Re-transformations to have enough concentration for iGEM submission.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Used the CLC Workbench 6 program to compare received sequence analysis with expected sequences. The sequencing verified all positive clones.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transfection of Biobrick LZT25, followed by MiniPrep of 4 colonies. The control digest with EcoRI and PstI found all 4 colonies having the correct fragment.</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">Week 7 (31 August-6 September)</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><H4>Investigation of P3 threshold for <I>E. coli</I> resistance</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Additional cultures are cultivated in LB + Cm for the CFP expression analysis.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Induced and uninduced cultures are analyzed with the fluorescence spectrometer.</LI></UL><H4>Conversion of BACTH into an iGEM standard and analysis of function</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Linearized pLac vector + T18/LZT18 and concentration measured using nanodrop.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Restriction digestion using S + P of the linearized pLac vector.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Overnight digest of T18/LZT18 with X + P.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Overnight cultures for colonies T25/LZT25 followed by a MiniPrep.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Digested and the T25/LZT25 (vector) and T18/LZT18 (insert) concentration determined.</LI><LI style="margin-bottom: 10px;line-height:1.8;">De-phosphorylation followed by ligation of the vector and insert.</LI></UL><H4>Set up of flow system</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Addition of yeast extract to the medium.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Cultivation using optimized minimal medium.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Inoculating next flask with minimal medium.</LI></UL><H4>Biobrick assembly</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Follow up sequence analysis for the LZT25 Biobrick verified the two colonies that were sent.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Re-transformations for both clones for LZT25 to have enough concentration for iGEM</LI><LI style="margin-bottom: 10px;line-height:1.8;">Measuring concentrations for iGEM samples.</LI></UL><H4>Wiki</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">The wiki work intensifies: text is added and the format is defined.</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">Week 8 (7-13 September)</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><H4>Investigation of P3 threshold for <I>E. coli</I> resistance</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Start of the initial 1 L batch cultivation with the new medium.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Start of the continuous cultivation using a feed pump.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The lagoon is infected with phages.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The inducing of the lagoon is started by adding IPTG in a continuous manner.</LI><LI style="margin-bottom: 10px;line-height:1.8;">For each IPTG concentration a sample is taken and analyzed for the plasmid stability and phage infection.</LI><LI style="margin-bottom: 10px;line-height:1.8;">A second expression analysis of the CFP is performed with the fluorescence spectrometer.</LI></UL><H4>Wiki</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Wiki freeze is coming...</LI></UL></DIV><DIV class="technology" style="font-family: 'Open Sans', Arial, sans-serif;line-height:1.8;">Week 9 (14-20 September)</DIV><DIV class="thelanguage" style="font-family: 'Open Sans', Arial, sans-serif;"><H4>Wiki</H4><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;">Wiki freeze is here!</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>