Team:TU Dresden/Project/Background
Home
Project
Overview
Description
Background
Methods
Results
Parts
Modeling
Conclusions
Team
About Us
Attributions
Gallery
Official Team Site
Contact Us!
Sponsors
Video
Notebook
Protocols
Lab Journal
Plasmid Design
Human Practices
Sustainability
Social Outreach (LNdW)
Education
Team Collaborations
Meetings
Safety
Background
Phage-assisted continuous evolution (PACE)
Phage assisted continuous evolution or PACE is a system designed for the continuous, directed evolution of biomolecules. The principle is that Escherichia coli flow through a lagoon in which the bacteriphage M13 are present and viable.  Important to note is that the flow rate of the E. coli is adjusted so that it is faster than their doubling time but not of that of the M13. Another important aspect of this setup is the deletion of gene P3 on M13, which is necessary for infection and proliferation. Instead the gene P3 of the M13 is encoded on the E. coli plasmid under the control of an upstream activating sequence (UAS). To undergo further infection cycles, the initial infectious generation of transgenic phage must activate the UAS by binding their activating domain (AD) to the binding domain (BD). This requires favorable protein-protein interactions between an X provided by the M13 and Y provided by the UAS of the E. coli. As a result, and some additional help from a mutagenesis plasmid, M13 evolves this interaction between X and Y in order to stay in the lagoon. A stronger interaction creates a selection advantage and will be favored over a weaker interaction (figure 1).
Figure 1 - Representation of our system. Here the phage M13 is found inside the lagoon and does not present the information necessary to produce P3 in its genome. This gene is encoded in the genome of the E. coli that will flow through the lagoon.
The bacterial two-hybrid (BACTH) system
Adenylate cyclase is an enzyme from B. pertussis, the agent of whooping cough, and acts as a toxin that binds to calmodulin in eukaryotes elevating the level of cAMP (cyclic adenosine monophosphate) in the organism. This is a large protein with 1,706 amino acids, however its catalytic activity resides in the first 400 amino acids. The catalytic domain is divided further into two sub-domains: catalytic site (25 kDa: residues 1-224) and the calmodulin binding site (18 kDa: residues 225-399). Interestingly, when T18 and T25 domains are expressed separately, no cAMP is produced. But when two interacting proteins of interest, either cystolic or membrane, are fused to T18 and T25, the domains are brought into close proximity and adenylate cyclase activity is restored.
In a BACTH setup, the gene coding for endogenous adenylate cyclase is deleted on an E. coli strain (cya-). The cya- strain is then transformed with plasmids containing the T25 and T18 hybrids. A positive interaction between two proteins of interest, an X and a Y, restores adenylate cyclase activity of T25 and T18. Newly synthesized cAMP will then interact with the catabolite activator protein (CAP). The cAMP/CAP complex binds to promoters and regulates transcription of several genes (figure 2).
Figure 2 - BATCH system adapted to our needs. It can be seen that only when the two proteins X and Y interact, the two subunits of the adenylate cyclase will be brought together to produce cAMP, which will allow the expression of P3.
This can be combined with the PACE system as the UAS promoter of P3 in E. coli. The expression of P3 would be regulated by the interaction of any genes of interest from the T18/T25 fusion and evolve over time in the lagoon.  This is an enhanced version of PACE system because the proteins of interest are not limited to those that only bind DNA. Quantitative assays can directly measure the level of cAMP synthesized and give insight into how the protein interactions are developing over time in the lagoon. This is a high unprecedented throughput assay, creating a very time efficient continuous system untarnished by human intervention steps, unlike that of phage display libraries. BACTH in combination with PACE is a simple and rapid tool, utilizing simple organisms to create strong and complex protein interactions.
Affibodies
Affibodies are small molecules that represent a new group of affinity proteins. They are derived from the immunoglobulin-binding B-domain region of Staphylococcal protein A. The B-domain of the protein A was mutated at multiple bases in order to enhance the stability of the protein. The new structure, which was termed the Z-domain, is made up of 58 amino acids. The mutated structure has a higher affinity when binding to other proteins.
These molecules are more thermally and chemically stable than antibodies and have a larger impact due to their smaller size. The main applications for affibodies are imaging, therapy and biotechnology. They are deemed the most promising tracer group, due to their rapid bio-distribution and tissue penetration levels, as well as helping with the detection of proteins in immunoprecipitation assays. Importantly, affibodies are able to identify different epitopes on proteins, so they are also able to be used as a mean to observe the effectiveness of monoclonal antibody therapies. Finally, in our project we are going to focus on the affibody ZHER2 that is able to bind to the human epidermal growth factor 2 (HER2) as our control protein-protein interaction for SPACE-P (figure 3).
Figure 3 - The structural representation of HER2 when binding to the HER2 affibody as well as the pertuzumab and trastuzumab monoclonal antibodies. Picture obtained from ref. [2].
References
Salehi, E., Farajnia, S., Parivar, K., Baradaran, B., Majidi, J., Omidi, Y., Saeedi, N. (2010). Recombinant expression and purification of L2 domain of human epidermal growth factor receptor. African Journal of Biotechnology, 9(33), 5292-5296.
Eigenbrot, C., Ultsch, M., Dubnovitsky, A., Abrahmsén, L., Härd, T. (2010). Structural basis for high-affinity HER2 receptor binding by an engineered protein. Proceedings of the National Academy of Sciences, 107(34), 15039-15044.
Löfblom, J., Feldwisch, J., Tomachev, V., Carlsson, J.,Ståh, S., Frejd, F. Y.  (2010). Affibody molecules: engineered proteins for therapeutic, diagnostic and biotechnological applications. FEBS Letters, 584(12), 2670-2680.
Battesti, A., Bouveret, E. (2012). The bacterial two-hybrid system based on adenylate cyclase reconstitution in Escherichia coli. Methods, 58(4), 325-334.
Carlson, J. C., Badran, A. H., Guggiana-Nilo, D. A., Liu, D. R. (2014). Negative selection and stringency modulation in phage-assisted continuous evolution. Natural Chemical Biology, 10(3), 216-222.
Dickinson, B. C., Leconte, A. M., Allen, B., Esvelt, K. M., Liu, D. R. (2013). Experimental interrogation of the path dependence and stochasticty of protein evolution using phage-assisted continuous evolution. Proceedings of the National Academy of Sciences, 110(22), 9007-9012.
Leconte, A. M., Dickinson, B. C., Yang, D. D., Chen, I. A., Allen, B., Liu, D. R. (2013). A population-based experimental model for protein evolution: effects of mutation rate and selection stringency on evolutionary outcomes. Biochemistry, 52(8), 1490-1499.
Esvelt, K. M., Carlson, J. C., Liu, D. R. (2013). A system for the continuous directed evolution of biomolecules. Nature, 472(7344), 499-503.
Rockberg, J., Schwenk, J. M., Uhlén, M. (2009). Discovery of epitopes for targeting the human epidermal growth factor receptor 2 (HER2) with antibodies. Molecular Oncology, 3(3), 238-247.
To the top!
