<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Dresden_Project_Methods skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Dresden/Project/Methods</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV><UL><LI class="cat1"><A href="/Team:TU_Dresden">Home</A></LI><LI class="cat2"><A href="https://2015.igem.org/Team:TU_Dresden/Project">Project</A></LI><LI><A href="/Team:TU_Dresden/Project/Overview">Overview</A></LI><LI><A href="/Team:TU_Dresden/Description">Description</A></LI><LI><A href="/Team:TU_Dresden/Project/Background">Background</A></LI><LI><A href="/Team:TU_Dresden/Project/Methods">Methods</A></LI><LI><A href="/Team:TU_Dresden/Project/Results">Results</A></LI><LI><A href="/Team:TU_Dresden/Project/Parts">Parts</A></LI><LI><A href="/Team:TU_Dresden/Modeling">Modeling</A></LI><LI><A href="/Team:TU_Dresden/Project/Conclusions">Conclusions</A></LI></UL><LI class="cat3"><A href="https://2015.igem.org/Team:TU_Dresden/Team">Team</A></LI><LI><A href="/Team:TU_Dresden/Team/Aboutus">About Us</A></LI><LI><A href="/Team:TU_Dresden/Team/Attributions">Attributions</A></LI><LI><A href="/Team:TU_Dresden/Team/Gallery">Gallery</A></LI><LI><A href="https://igem.org/Team.cgi?id=1781">Official Team Site</A></LI><LI><A href="/Team:TU_Dresden/Team/Contact">Contact Us!</A></LI><LI><A href="/Team:TU_Dresden/Team/Sponsors">Sponsors</A></LI><LI class="cat5"><A href="https://2015.igem.org/Team:TU_Dresden/Video">Video</A></LI><LI class="cat4"><A href="https://2015.igem.org/Team:TU_Dresden/Notebook">Notebook</A></LI><LI><A href="/Team:TU_Dresden/Notebook/Protocols">Protocols</A></LI><LI><A href="/Team:TU_Dresden/Notebook/LabJournal">Lab Journal</A></LI><LI><A href="/Team:TU_Dresden/Notebook/PlasmidDesign">Plasmid Design</A></LI><LI class="cat6"><A href="https://2015.igem.org/Team:TU_Dresden/Practices">Human Practices</A></LI><LI><A href="/Team:TU_Dresden/Practices/Sustainability">Sustainability</A></LI><LI><A href="/Team:TU_Dresden/Practices/Social_Outreach">Social Outreach (LNdW)</A></LI><LI><A href="/Team:TU_Dresden/Practices/Education">Education</A></LI><LI><A href="/Team:TU_Dresden/Collaborations">Team Collaborations</A></LI><LI><A href="/Team:TU_Dresden/Practices/Meetings">Meetings</A></LI><LI class="cat7"><A href="/TU_Dresden/Safety">Safety</A></LI></NAV><DIV id="TueContent"><H1>Methods</H1><P style="line-height:1.8;">Here you can find a short summary of the methods used in each subproject:</P><UL><LI style="margin-bottom: 10px;line-height:1.8;"><A style="text-decoration:none;" href="https://2015.igem.org/Team:TU_Dresden/Project/Methods#Folding"><FONT color="#045FB4">Folding study of target protein</FONT></A></LI><LI style="margin-bottom: 10px;line-height:1.8;"><A style="text-decoration:none;" href="https://2015.igem.org/Team:TU_Dresden/Project/Methods#Structure"><FONT color="#045FB4">Structure analysis of our targets and their interactions</FONT></A></LI><LI style="margin-bottom: 10px;line-height:1.8;"><A style="text-decoration:none;" href="https://2015.igem.org/Team:TU_Dresden/Project/Methods#Resistance"><FONT color="#045FB4">Investigation of P3 threshold for <I>E. coli</I> resistance</FONT></A></LI><LI style="margin-bottom: 10px;line-height:1.8;"><A style="text-decoration:none;" href="https://2015.igem.org/Team:TU_Dresden/Project/Methods#BACTH"><FONT color="#045FB4">Conversion of BACTH into an iGEM standard and analysis of function</FONT></A></LI><LI style="margin-bottom: 10px;line-height:1.8;"><A style="text-decoration:none;" href="https://2015.igem.org/Team:TU_Dresden/Project/Methods#Setup"><FONT color="#045FB4">Set up of flow system</FONT></A></LI><LI style="margin-bottom: 10px;line-height:1.8;"><A style="text-decoration:none;" href="https://2015.igem.org/Team:TU_Dresden/Project/Methods#biobrick"><FONT color="#045FB4">Biobrick assembly</FONT></A></LI></UL><H2 id="Folding">Folding study of target protein</H2><OL><LI style="margin-bottom: 10px;line-height:1.8;">Designing of primers (primer section) for adding flanking regions containing restriction sites for adding HER2 to His-tagged pET28. </LI><LI style="margin-bottom: 10px;line-height:1.8;">Digestion of pET28a plasmid and PCR product with enzymes NheI-HF and NotI-HF simultaneously. </LI><LI style="margin-bottom: 10px;line-height:1.8;">Gel electrophoresis on 0.7 % agarose gel to confirm digestion.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Gel extraction of required pET28a backbone and PCR product digests. </LI><LI style="margin-bottom: 10px;line-height:1.8;">Ligation of digest products in molar ratio of 1 part of pET28a to 2.5 parts of HER2 with T4 ligase. Determination of concentration using nanodrop. </LI><LI style="margin-bottom: 10px;line-height:1.8;">Electroporation of ligated plasmid (pET28a + HER2) into Escherichia coli GB05 and streak onto kanamycin plates for selection.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Plates checked for colonies, calculate overall transformation efficiency.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Setting up of overnight cultures from selected colonies.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Plasmid preparation from overnight cultures to extract the pET28a-HER2 plasmid, then restriction digestion with enzymes NheI-HF and NotI-HF and run on a 0.7 % gel to confirm proper ligation. </LI><LI style="margin-bottom: 10px;line-height:1.8;">Sequencing of purified plasmids. </LI><LI style="margin-bottom: 10px;line-height:1.8;">After confirmation of plasmid pET28a-HER2 sequence, electroporation of plasmids into E. coli BL21 (expression vector) for protein expression studies. </LI><LI style="margin-bottom: 10px;line-height:1.8;">Plasmid preparation, restriction digestion and gel electrophoresis in tandem. Cells plated on kanamycin selection media.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Confirmation of electroporation efficiency. </LI><LI style="margin-bottom: 10px;line-height:1.8;">Set up 4 x 1 L cultures for protein extraction experiments. </LI><LI style="margin-bottom: 10px;line-height:1.8;">Extraction of His-tagged protein using affinity chromatography with Ni beads.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Size exclusion chromatography experiments using purified His-tagged protein elute from affinity chromatography.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Peforming dialysis on purified elutes to ready sample for circular dichroism experiments.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Amicon ultracentrifugation done to concentrate purified elutes from size exclusion chromatography. </LI><LI style="margin-bottom: 10px;line-height:1.8;">Performing circular dichroism on the elutes and check for deconvolution data for percentage of each secondary structure from elutes collected.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Comparing deconvolution results with PDB structural data for 3MZW and doing multiple sequence alignment to find percentage of correlation of secondary structures between expressed protein and crystallized whole protein (HER2). </LI><LI style="margin-bottom: 10px;line-height:1.8;">Analyzing protein expression in elutes obtained from size exclusion chromatography with Blue Native PAGE and SDS-PAGE.</LI></OL><H2 id="Structure">Structure analysis of our targets and their interactions</H2><P style="line-height:1.8">For the following analysis. All molecule pictures and the structure video were made using the PyMOL [2] suite. Further information on PyMOL can be found <A style="text-decoration:none;" href="https://www.pymol.org/"><FONT color="#045FB4">here.</FONT></A></P><P style="line-height:1.8">For structure analysis the protein data base structure with the PDB ID: 3MZW was used [1].</P><UL style="padding-left: 0.6cm;"><LI style="margin-bottom: 10px;line-height:1.8;"><B>Structure check of HER2</B>: The structure of HER2 with the PDB ID: 3MZW was validated with the use of the program <A style="text-decoration:none;" href="http://www.ebi.ac.uk/thornton-srv/software/PROCHECK/"><FONT color="#045FB4">PROCHECK</FONT></A> [3, 4].
    </LI><LI style="margin-top: 10px;line-height:1.8;"><B>Calculation of intefacial residues of HER2 and its bound affibody</B>: In order to define the interfacial residues of HER2 and its affibody the python script <A style="text-decoration:none;" href="https://static.igem.org/mediawiki/2015/4/4e/InterfaceResidues.txt"><FONT color="#045FB4">InterfaceResidues.py</FONT></A> was used within PyMOL.
    </LI><LI style="margin-top: 10px;line-height:1.8;"><B>Calculation of electrostatic interactions in the interface</B>: The electrostatic interactions in the interface were first defined in PyMOL using the preselected interface atoms, atom types and maximum distances. For explicit calculation of hydrogen bonds the Python script <A style="text-decoration:none;" href="https://static.igem.org/mediawiki/2015/a/a1/Space-p_list_hbonds.txt"><FONT color="#045FB4">list_hbonds.py</FONT></A> was used in PyMOL used with a distance cutoff of 3.2 Angsrom and an angle cutoff of 55 degrees. The script automatically adds a requirement that atoms in the selection must be either of the elements N or O.
    </LI><LI style="margin-top: 10px;line-height:1.8;"><B>Conservation study of HER2</B>: A search for similar sequences of the sequence of HER2 (<A style="text-decoration:none;" href="https://static.igem.org/mediawiki/2015/8/8a/3MZW_HER2-receptor.txt"><FONT color="#045FB4">3MZW_HER2-receptor.fasta</FONT></A>) was performed using the Basic Local Alignment Search Tool (<A style="text-decoration:none;" href="http://blast.ncbi.nlm.nih.gov/Blast.cgi"><FONT color="#045FB4">BLAST</FONT></A>) [5, 6] followed by a manual selection of 11 sequences from different organisms. Then a multiple sequence alignment was performed using <A style="text-decoration:none;" href="http://www.ebi.ac.uk/Tools/msa/clustalot"><FONT color="#045FB4">CLUSTAL 2.1</FONT></A> [7]. The obtained alignment could then be used for conservation calculation on the <A style="text-decoration:none;" href="http://consurf.tau.ac.il/"><FONT color="#045FB4">ConSurf Server</FONT></A> [8, 9, 10, 11].
      <P style="line-height:1.8;">In order to obtain a structure color coded by conservation the new (changed) PDB file obtained from the conservation calculation was loaded into PyMOL together with the Python script <A style="text-decoration:none;" href="https://static.igem.org/mediawiki/2015/7/7f/Consurf_HER2.txt"><FONT color="#045FB4">consurf_HER2.py</FONT></A>.</P></LI><LI style="margin-top: 10px;line-height:1.8;"><B>Visualization of the B-factor for the affibody ZHER2</B>: For obtaining a structure color coded by B-factor the Python script <A style="text-decoration:none;" href="https://static.igem.org/mediawiki/2015/e/e6/Color_b.txt"><FONT color="#045FB4">color_b.py</FONT></A> was used in PyMOL.
    </LI></UL><H2 id="Resistance">Investigation of P3 threshold for <I>E. coli</I> resistance</H2><H4>Transformation of <I>E. coli</I> with the construct</H4><OL><LI style="margin-bottom: 10px;line-height:1.8;">Resuspension of synthesized plasmids T25, T18, ZHER2, LZT18, LZT25, HER2 and P3.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Resuspension of iGEM plasmids containing RBS (<A style="text-decoration:none;" href="http://parts.igem.org/Part:BBa_B0030"><FONT color="#045FB4">BBa_E0020</FONT></A>), CFP (<A style="text-decoration:none;" href="http://parts.igem.org/Part:BBa_E0020"><FONT color="#045FB4">BBa_E0020</FONT></A>), and pLac (<A style="text-decoration:none;" href="http://parts.igem.org/Part:BBa_K611025"><FONT color="#045FB4">BBa_K611025</FONT></A>).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation of the 10 plasmids into <I>E. coli</I> GB05.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Plating of transfected cells on Cm and Kan plates for synthesized and iGEM plasmids respectively.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Plasmid preparation for the 10 plasmids.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Nanodrop measurement of plasmid prep.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Analytical digest of plasmid prep with NotI.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Digest of P3 and CFP with XbaI and PstI-HF → lin P3, lin CFP.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Digest of pLac and RBS with SpeI and PstI-HF → lin pLac, lin RBS.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Dephosphorylation of pLac and RBS.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Gel-purification of lin P3, lin CFP, lin pLac and lin RBS.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Nanodrop measurement of lin P3, lin CFP, lin pLac and lin RBS.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Ligation of pLac with P3 → (pLac + P3) and RBS with CFP → (RBS + CFP).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation of (pLac + P3) and (RBS + CFP) into <I>E. coli</I> GB05.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Plasmid preparation of (pLac + P3) and (RBS + CFP).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Nanodrop measurement of (pLac + P3) and (RBS + CFP).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Digest of (pLac + P3) with SpeI and Pst-HF → lin (pLac + P3).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Digest of (RBS + CFP) with XbaI and PstI-HF → lin (RBS + CFP).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Dephosphorylation of lin (pLac + P3).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Gel-purification of lin (RBS + CFP) and lin (pLac + P3).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Nanodrop measurement of lin (RBS + CFP) and lin (pLac + P3).</LI><LI style="margin-bottom: 10px;line-height:1.8;">Ligation of lin (RBS + CFP) with lin (pLac + P3) → final construct.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation of final construct into <I>E. coli</I> ER2738.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Medium
</LI><LI style="margin-bottom: 10px;line-height:1.8;">Lysogeny broth:</LI><UL style="padding-left:1.2cm;"><LI style="margin-bottom: 10px;line-height:1.8;">10 g L<SUP>-1</SUP> peptone</LI><LI style="margin-bottom: 10px;line-height:1.8;">5 g L<SUP>-1</SUP> yeast extract</LI><LI style="margin-bottom: 10px;line-height:1.8;">10 g <SUP>-1</SUP> sodium chloride</LI></UL><P style="margin-bottom: 10px;line-height:1.8;padding-left:1.2cm;"><B>Optional:</B></P><UL style="padding-left:1.2cm;"><LI style="margin-bottom: 10px;line-height:1.8;">15 g L<SUP>-1</SUP> agar-agar</LI><LI style="margin-bottom: 10px;line-height:1.8;">0.025 g L<SUP>-1</SUP> chloramphenicol</LI><LI style="margin-bottom: 10px;line-height:1.8;">0.04 g L<SUP>-1</SUP> X-Gal</LI></UL><LI style="margin-bottom: 10px;line-height:1.8;">Measurement of the expression of P3 in <I>E. coli</I> co-expressed with CFP.</LI></OL><H4>Analysis of the plasmid stability</H4><OL><LI style="margin-bottom: 10px;line-height:1.8;">The plasmid stability was analyzed by using two LB agar plates with and without antibiotics.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The samples were diluted and plated overnight.</LI></OL><H4>Analysis of the phage infection</H4><OL><LI style="margin-bottom: 10px;line-height:1.8;">A 10<SUP>-5</SUP> dilution from the plasmid stability was used to analyze whether phages are washed away during the cultivation.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The samples were plated on a plate with chloramphenicol and X-Gal.</LI></OL><H2 id="BACTH" style="line-height:1.8;">Conversion of BACTH into an iGEM standard and analysis of function</H2><H4>Fusion ligation of T18, LZT18 and T25, LZT25</H4><OL><LI style="margin-bottom: 10px;line-height:1.8;">Restriction digestion of T18 and T25 using the enzymes EcoRI and AgeI .</LI><LI style="margin-bottom: 10px;line-height:1.8;">Restriction digestion of LZT18 and LZT25 using the enzymes EcoRI and NgoMI.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Purification of restriction digests using gel electrophoresis.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Ligation of T18 with LZT18 vector and T25 with LZT25 vector respectively.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation into <I>E. coli</I> GB05 strain and plate in agar with kanamycin.</LI></OL><H4>Ligation of fusion products, T18-LZT18 and T25-LZT25</H4><OL><LI style="margin-bottom: 10px;line-height:1.8;">Restriction digestion of T25-LZT25 fusion product with EcoRI HF and SpeI HF and T18-LZT18 fusion product with EcoRI HF and XbaI.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Purification of restriction digests using gel electrophoresis.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Ligation of T25-LZT25 cassette with T18-LZT18 vector. For simplicity reasons let us call this T18 T25 product.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation into <I>E. coli</I> GB05 strain and plate agar with kanamycin.</LI></OL><H4>Ligation with lacZ</H4><OL><LI style="margin-bottom: 10px;line-height:1.8;">Restriction digestion of pLac using SpeI HF and PstI HF.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Restriction digestion of T18-T25 product using XbaI and PstI HF.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Purification of restriction digests using gel electrophoresis.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Ligation of pLac vector with T18-T25 insert.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transformation into <I>E. coli</I> BTH101 strain and plate on X-Gal plates.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Observe blue colonies, which contain successful ligation product.</LI></OL><H2 id="Setup">Set up of flow system</H2><P style="line-height:1.8">For this subproject, two devices were used, which are summarized in the following table:</P><CENTER><TABLE cellpadding="10" align="center"><TBODY><TR><TH align="center">Device</TH><TH align="center">Manufacturer</TH><TH align="center">Type series</TH></TR><TR><TD align="center">Fluorescence spectrometer</TD><TD align="center">Hitach</TD><TD align="center">F-4500</TD></TR><TR><TD align="center">Bioreactor</TD><TD align="center">Applikon</TD><TD align="center">1 L</TD></TR></TBODY></TABLE></CENTER><H4>Continuous stirred-tank cultivation</H4><OL><LI style="margin-bottom: 10px;line-height:1.8;">The bioreactor (1 L Applikon) was filled with minimal medium with an OD<SUB>600</SUB> of 0.04.</LI><LI style="margin-bottom: 10px;line-height:1.8;">After the inoculation the medium was stirred and aerated medium (synthetic air).</LI><LI style="margin-bottom: 10px;line-height:1.8;">The culture was grown until a OD<SUB>600</SUB> of 0.2 was reached.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The continuous cultivation was started by activating the feed pump to supply the reactor with fresh medium (figure 1 and 2).</LI><LI style="margin-bottom: 10px;line-height:1.8;">The medium is continuously removed and pumped to the lagoon.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Next isopropyl β-D-1-thiogalactopyranoside (IPTG) was added to the feed entering the lagoon.</LI><LI style="margin-bottom: 10px;line-height:1.8;">After 3 hours a sample from the lagoon's outlet was taken and analyzed.</LI><LI style="margin-bottom: 10px;line-height:1.8;">The IPTG concentration was increased every 3 hours after the sample was taken.</LI></OL><FIGURE align="center"><DIV style="font-size: 13px">Figure 1 - Schematic setup of the continue flow system. The medium is pumped from the reservoir into the bioreactor aerated with 5 NL/h. The culture is pumped into the lagoon and IPTG is added to the flow with the syringe pump.</DIV></FIGURE><FIGURE align="center"><DIV style="font-size: 13px">Figure 2 - Real setup used for the experiments. </DIV></FIGURE><H2 id="biobrick">Biobrick assembly</H2><OL><LI style="margin-bottom: 10px;line-height:1.8;">Restriction digestion of plasmids parts (P3, HER2, T18, T25, ZHER2, LZT18, and LZT25) from IDT (<A style="text-decoration:none;" href="https://2015.igem.org/Team:TU_Dresden/Project/Parts"><FONT color="#045FB4">Parts</FONT></A>) using EcoRI and PstI and pSB1C3 iGEM Biobrick plasmid backbone.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Running gel electrophoresis of digested parts (P3, HER2, T18, and T25) on 0.7 % gel and ZHER2, LZT18, and LZT25 on 2 % agarose gel.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Gel purification of parts. </LI><LI style="margin-bottom: 10px;line-height:1.8;">Ligation of parts into digested pSB1C3 backbone.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Transforming <I>E. coli</I> GB05 with ligated plasmids.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Checking for transformation efficiency and subculturing of clones.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Plasmid preparation from transformed clones.</LI><LI style="margin-bottom: 10px;line-height:1.8;">Sequencing of plasmids.</LI></OL><H2>References</H2><P style="line-height:1.8"><OL><LI id="#bib01" style="margin-bottom: 10px;line-height:1.8;">
      Eigenbrot, C., Ultsch, M., Dubnovitsky, A., Abrahmsén, L., Härd, T. (2010). Structural basis for high-affinity HER2 receptor binding by an engineered protein. <I>Proceedings of the National Academy of Sciences</I>, 107(34), 15039-15044.
    </LI><LI id="#bib02" style="margin-bottom: 10px;line-height:1.8;">
      The PyMOL Molecular Graphics System, Version 1.7.4 Schrödinger, LLC.
    </LI><LI id="#bib03" style="margin-bottom: 10px;line-height:1.8;">
      Laskowski, R. A., MacArthur, M. W., Moss, D. S., Thornton, J. M. (1993). PROCHECK - a program to check the stereochemical quality of protein structures. <I>Journal of Applied Crystallography</I>, 26, 283-291.
    </LI><LI id="#bib04" style="margin-bottom: 10px;line-height:1.8;">
      Laskowski, R. A., Rullmannn, J. A., MacArthur, M. W., Kaptein, R., Thornton, J. M. (1996). AQUA and PROCHECK-NMR: programs for checking the quality of protein structures solved by NMR. <I>Journal of Biomolecular NMR</I>, 8, 477-486.
    </LI><LI id="#bib05" style="margin-bottom: 10px;line-height:1.8;">
      Goujon, M., McWilliam, H., Li, W., Valentin, F., Squizzato, S., Paern, J., Lopez, R. (2010). A new bioinformatics analysis tools framework at EMBL-EBI. <I>Nucleic acids research</I>, 38 Suppl: W695-9 DOI:10.1093/nar/gkq313.
    </LI><LI id="#bib06" style="margin-bottom: 10px;line-height:1.8;">
      Altschul, S.F., Gish, W., Miller, W., Myers, E. W., Lipman, D.J. (1990). Basic local alignment search tool. <I>Journal of Molecular Biology</I>, 215, 403-410.
    </LI><LI id="#bib07" style="margin-bottom: 10px;line-height:1.8;">
      Sievers, F., Wilm, A., Dineen, D. G., Gibson, T. J., Karplus, K., Li, W., Lopez, R., McWilliam, H., Remmert, M., Söding, J., Thompson, J. D., Higgins, D. G. (2011). Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. <I>Molecular Systems Biology</I>, 7 Article number: 539 doi:10.1038/msb.2011.75.
    </LI><LI id="#bib08" style="margin-bottom: 10px;line-height:1.8;">
      Celniker, G., Nimrod, G., Ashkenazy, H., Glaser, F., Martz, E., Mayrose, I., Pupko, T., Ben-Tal, N. (2013). ConSurf: using evolutionary data to raise testable hypotheses about protein function. <I>Israel Journal of Chemistry</I>, doi: 10.1002/ijch.201200096
    </LI><LI id="#bib09" style="margin-bottom: 10px;line-height:1.8;">
      Ashkenazy, H., Erez, E., Martz, E., Pupko, T., Ben-Tal, N. (2010). ConSurf 2010: calculating evolutionary conservation in sequence and structure of proteins and nucleic acids. <I>Nucleic Acids Research</I>, doi: 10.1093/nar/gkq399.
    </LI><LI id="#bib10" style="margin-bottom: 10px;line-height:1.8;">
      Landau, M., Mayrose, I., Rosenberg, Y., Glaser, F., Martz, E., Pupko, T., Ben-Tal, N. (2005). ConSurf 2005: the projection of evolutionary conservation scores of residues on protein structures. <I>Nucleic Acids Research</I>, 33:W299-W302.
    </LI><LI id="#bib11" style="margin-bottom: 10px;line-height:1.8;">
      Glaser, F., Pupko, T., Paz, I., Bell, R.E., Bechor, D., Martz, E., Ben-Tal, N. (2003). ConSurf: identification of functional regions in proteins by surface-mapping of phylogenetic information. <I>Bioinformatics</I>, 19, 163-164.
    </LI></OL></P><H3 style="text-align:center;"><A style="text-decoration:none;" href="#top"><FONT color="#045FB4">To the top!</FONT></A></H3></DIV></DIV></DIV></DIV></DIV></BODY></HTML>