Team:TU Dresden/Project/Methods
Home
Project
Overview
Description
Background
Methods
Results
Parts
Modeling
Conclusions
Team
About Us
Attributions
Gallery
Official Team Site
Contact Us!
Sponsors
Video
Notebook
Protocols
Lab Journal
Plasmid Design
Human Practices
Sustainability
Social Outreach (LNdW)
Education
Team Collaborations
Meetings
Safety
Methods
Here you can find a short summary of the methods used in each subproject:
Folding study of target protein
Structure analysis of our targets and their interactions
Investigation of P3 threshold for E. coli resistance
Conversion of BACTH into an iGEM standard and analysis of function
Set up of flow system
Biobrick assembly
Folding study of target protein
Designing of primers (primer section) for adding flanking regions containing restriction sites for adding HER2 to His-tagged pET28.
Digestion of pET28a plasmid and PCR product with enzymes NheI-HF and NotI-HF simultaneously.
Gel electrophoresis on 0.7 % agarose gel to confirm digestion.
Gel extraction of required pET28a backbone and PCR product digests.
Ligation of digest products in molar ratio of 1 part of pET28a to 2.5 parts of HER2 with T4 ligase. Determination of concentration using nanodrop.
Electroporation of ligated plasmid (pET28a + HER2) into Escherichia coli GB05 and streak onto kanamycin plates for selection.
Plates checked for colonies, calculate overall transformation efficiency.
Setting up of overnight cultures from selected colonies.
Plasmid preparation from overnight cultures to extract the pET28a-HER2 plasmid, then restriction digestion with enzymes NheI-HF and NotI-HF and run on a 0.7 % gel to confirm proper ligation.
Sequencing of purified plasmids.
After confirmation of plasmid pET28a-HER2 sequence, electroporation of plasmids into E. coli BL21 (expression vector) for protein expression studies.
Plasmid preparation, restriction digestion and gel electrophoresis in tandem. Cells plated on kanamycin selection media.
Confirmation of electroporation efficiency.
Set up 4 x 1 L cultures for protein extraction experiments.
Extraction of His-tagged protein using affinity chromatography with Ni beads.
Size exclusion chromatography experiments using purified His-tagged protein elute from affinity chromatography.
Peforming dialysis on purified elutes to ready sample for circular dichroism experiments.
Amicon ultracentrifugation done to concentrate purified elutes from size exclusion chromatography.
Performing circular dichroism on the elutes and check for deconvolution data for percentage of each secondary structure from elutes collected.
Comparing deconvolution results with PDB structural data for 3MZW and doing multiple sequence alignment to find percentage of correlation of secondary structures between expressed protein and crystallized whole protein (HER2).
Analyzing protein expression in elutes obtained from size exclusion chromatography with Blue Native PAGE and SDS-PAGE.
Structure analysis of our targets and their interactions
For the following analysis. All molecule pictures and the structure video were made using the PyMOL [2] suite. Further information on PyMOL can be found here.
For structure analysis the protein data base structure with the PDB ID: 3MZW was used [1].
Structure check of HER2: The structure of HER2 with the PDB ID: 3MZW was validated with the use of the program PROCHECK [3, 4].
Calculation of intefacial residues of HER2 and its bound affibody: In order to define the interfacial residues of HER2 and its affibody the python script InterfaceResidues.py was used within PyMOL.
Calculation of electrostatic interactions in the interface: The electrostatic interactions in the interface were first defined in PyMOL using the preselected interface atoms, atom types and maximum distances. For explicit calculation of hydrogen bonds the Python script list_hbonds.py was used in PyMOL used with a distance cutoff of 3.2 Angsrom and an angle cutoff of 55 degrees. The script automatically adds a requirement that atoms in the selection must be either of the elements N or O.
Conservation study of HER2: A search for similar sequences of the sequence of HER2 ( 3MZW_HER2-receptor.fasta ) was performed using the Basic Local Alignment Search Tool ( BLAST ) [5, 6] followed by a manual selection of 11 sequences from different organisms. Then a multiple sequence alignment was performed using CLUSTAL 2.1 [7]. The obtained alignment could then be used for conservation calculation on the ConSurf Server [8, 9, 10, 11].
In order to obtain a structure color coded by conservation the new (changed) PDB file obtained from the conservation calculation was loaded into PyMOL together with the Python script consurf_HER2.py .
Visualization of the B-factor for the affibody ZHER2: For obtaining a structure color coded by B-factor the Python script color_b.py was used in PyMOL.
Investigation of P3 threshold for E. coli resistance
Transformation of E. coli with the construct
Resuspension of synthesized plasmids T25, T18, ZHER2, LZT18, LZT25, HER2 and P3.
Resuspension of iGEM plasmids containing RBS ( BBa_E0020 ), CFP ( BBa_E0020 ), and pLac ( BBa_K611025 ).
Transformation of the 10 plasmids into E. coli GB05.
Plating of transfected cells on Cm and Kan plates for synthesized and iGEM plasmids respectively.
Plasmid preparation for the 10 plasmids.
Nanodrop measurement of plasmid prep.
Analytical digest of plasmid prep with NotI.
Digest of P3 and CFP with XbaI and PstI-HF → lin P3, lin CFP.
Digest of pLac and RBS with SpeI and PstI-HF → lin pLac, lin RBS.
Dephosphorylation of pLac and RBS.
Gel-purification of lin P3, lin CFP, lin pLac and lin RBS.
Nanodrop measurement of lin P3, lin CFP, lin pLac and lin RBS.
Ligation of pLac with P3 → (pLac + P3) and RBS with CFP → (RBS + CFP).
Transformation of (pLac + P3) and (RBS + CFP) into E. coli GB05.
Plasmid preparation of (pLac + P3) and (RBS + CFP).
Nanodrop measurement of (pLac + P3) and (RBS + CFP).
Digest of (pLac + P3) with SpeI and Pst-HF → lin (pLac + P3).
Digest of (RBS + CFP) with XbaI and PstI-HF → lin (RBS + CFP).
Dephosphorylation of lin (pLac + P3).
Gel-purification of lin (RBS + CFP) and lin (pLac + P3).
Nanodrop measurement of lin (RBS + CFP) and lin (pLac + P3).
Ligation of lin (RBS + CFP) with lin (pLac + P3) → final construct.
Transformation of final construct into E. coli ER2738.
Medium
Lysogeny broth:
10 g L-1 peptone
5 g L-1 yeast extract
10 g -1 sodium chloride
Optional:
15 g L-1 agar-agar
0.025 g L-1 chloramphenicol
0.04 g L-1 X-Gal
Measurement of the expression of P3 in E. coli co-expressed with CFP.
Analysis of the plasmid stability
The plasmid stability was analyzed by using two LB agar plates with and without antibiotics.
The samples were diluted and plated overnight.
Analysis of the phage infection
A 10-5 dilution from the plasmid stability was used to analyze whether phages are washed away during the cultivation.
The samples were plated on a plate with chloramphenicol and X-Gal.
Conversion of BACTH into an iGEM standard and analysis of function
Fusion ligation of T18, LZT18 and T25, LZT25
Restriction digestion of T18 and T25 using the enzymes EcoRI and AgeI .
Restriction digestion of LZT18 and LZT25 using the enzymes EcoRI and NgoMI.
Purification of restriction digests using gel electrophoresis.
Ligation of T18 with LZT18 vector and T25 with LZT25 vector respectively.
Transformation into E. coli GB05 strain and plate in agar with kanamycin.
Ligation of fusion products, T18-LZT18 and T25-LZT25
Restriction digestion of T25-LZT25 fusion product with EcoRI HF and SpeI HF and T18-LZT18 fusion product with EcoRI HF and XbaI.
Purification of restriction digests using gel electrophoresis.
Ligation of T25-LZT25 cassette with T18-LZT18 vector. For simplicity reasons let us call this T18 T25 product.
Transformation into E. coli GB05 strain and plate agar with kanamycin.
Ligation with lacZ
Restriction digestion of pLac using SpeI HF and PstI HF.
Restriction digestion of T18-T25 product using XbaI and PstI HF.
Purification of restriction digests using gel electrophoresis.
Ligation of pLac vector with T18-T25 insert.
Transformation into E. coli BTH101 strain and plate on X-Gal plates.
Observe blue colonies, which contain successful ligation product.
Set up of flow system
For this subproject, two devices were used, which are summarized in the following table:
Device
Manufacturer
Type series
Fluorescence spectrometer
Hitach
F-4500
Bioreactor
Applikon
1 L
Continuous stirred-tank cultivation
The bioreactor (1 L Applikon) was filled with minimal medium with an OD600 of 0.04.
After the inoculation the medium was stirred and aerated medium (synthetic air).
The culture was grown until a OD600 of 0.2 was reached.
The continuous cultivation was started by activating the feed pump to supply the reactor with fresh medium (figure 1 and 2).
The medium is continuously removed and pumped to the lagoon.
Next isopropyl β-D-1-thiogalactopyranoside (IPTG) was added to the feed entering the lagoon.
After 3 hours a sample from the lagoon's outlet was taken and analyzed.
The IPTG concentration was increased every 3 hours after the sample was taken.
Figure 1 - Schematic setup of the continue flow system. The medium is pumped from the reservoir into the bioreactor aerated with 5 NL/h. The culture is pumped into the lagoon and IPTG is added to the flow with the syringe pump.
Figure 2 - Real setup used for the experiments.
Biobrick assembly
Restriction digestion of plasmids parts (P3, HER2, T18, T25, ZHER2, LZT18, and LZT25) from IDT ( Parts ) using EcoRI and PstI and pSB1C3 iGEM Biobrick plasmid backbone.
Running gel electrophoresis of digested parts (P3, HER2, T18, and T25) on 0.7 % gel and ZHER2, LZT18, and LZT25 on 2 % agarose gel.
Gel purification of parts.
Ligation of parts into digested pSB1C3 backbone.
Transforming E. coli GB05 with ligated plasmids.
Checking for transformation efficiency and subculturing of clones.
Plasmid preparation from transformed clones.
Sequencing of plasmids.
References
Eigenbrot, C., Ultsch, M., Dubnovitsky, A., Abrahmsén, L., Härd, T. (2010). Structural basis for high-affinity HER2 receptor binding by an engineered protein. Proceedings of the National Academy of Sciences, 107(34), 15039-15044.
The PyMOL Molecular Graphics System, Version 1.7.4 Schrödinger, LLC.
Laskowski, R. A., MacArthur, M. W., Moss, D. S., Thornton, J. M. (1993). PROCHECK - a program to check the stereochemical quality of protein structures. Journal of Applied Crystallography, 26, 283-291.
Laskowski, R. A., Rullmannn, J. A., MacArthur, M. W., Kaptein, R., Thornton, J. M. (1996). AQUA and PROCHECK-NMR: programs for checking the quality of protein structures solved by NMR. Journal of Biomolecular NMR, 8, 477-486.
Goujon, M., McWilliam, H., Li, W., Valentin, F., Squizzato, S., Paern, J., Lopez, R. (2010). A new bioinformatics analysis tools framework at EMBL-EBI. Nucleic acids research, 38 Suppl: W695-9 DOI:10.1093/nar/gkq313.
Altschul, S.F., Gish, W., Miller, W., Myers, E. W., Lipman, D.J. (1990). Basic local alignment search tool. Journal of Molecular Biology, 215, 403-410.
Sievers, F., Wilm, A., Dineen, D. G., Gibson, T. J., Karplus, K., Li, W., Lopez, R., McWilliam, H., Remmert, M., Söding, J., Thompson, J. D., Higgins, D. G. (2011). Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Molecular Systems Biology, 7 Article number: 539 doi:10.1038/msb.2011.75.
Celniker, G., Nimrod, G., Ashkenazy, H., Glaser, F., Martz, E., Mayrose, I., Pupko, T., Ben-Tal, N. (2013). ConSurf: using evolutionary data to raise testable hypotheses about protein function. Israel Journal of Chemistry, doi: 10.1002/ijch.201200096
Ashkenazy, H., Erez, E., Martz, E., Pupko, T., Ben-Tal, N. (2010). ConSurf 2010: calculating evolutionary conservation in sequence and structure of proteins and nucleic acids. Nucleic Acids Research, doi: 10.1093/nar/gkq399.
Landau, M., Mayrose, I., Rosenberg, Y., Glaser, F., Martz, E., Pupko, T., Ben-Tal, N. (2005). ConSurf 2005: the projection of evolutionary conservation scores of residues on protein structures. Nucleic Acids Research, 33:W299-W302.
Glaser, F., Pupko, T., Paz, I., Bell, R.E., Bechor, D., Martz, E., Ben-Tal, N. (2003). ConSurf: identification of functional regions in proteins by surface-mapping of phylogenetic information. Bioinformatics, 19, 163-164.
To the top!
