"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-Germany/Results/vector
From 2012.igem.org
Start
Team
Members
Photos
Bielefeld University
Acknowledgements
Contact
Project
Description
Achievements
Approach
Model
Database
Background
Labjournal
Protocols
Outlook
Results
Summary
Datapage
Laccases
Immobilization
Substrate Analysis
Cellulose Binding Domain
Shuttle Vector
Collaboration
Since Regionals
Summary
Labjournal
Human Practice
Overview
Expert
Politics
Industry
Outreach
Education
Environment
Press
Safety
Judging
Partner
Shuttle vector
Summary
Contents
1 Description of the shuttle vector system
2 Site directed mutagenesis of the shuttle vector
3 Shuttle vector in E. coli
4 Shuttle vector in P. pastoris
5 GFP integrated in shuttle vector
6 Since Regionals: TVEL5 integrated in shuttle vector
Description of the shuttle vector system
The design of a minimal shuttle vector system with defined regions (or better DNA fragments) for expression and secretion of proteins of interest (POI), like laccase, which needed glycolisation is the basic concept. The shuttle vector needs a bacterial part for cloning in bacteria (like E. coli) and an eucaryotic part for genomic integration and selection in yeast (like P. pastoris). An other team will be able to clime in frame their gene of interest via the AarI restriction site. With only one restriction ligation cloning step the shuttle vector will be ready to use and integrate in the eucaryote P. pastoris.
The shuttle vector consists of the plasmid pSB1C3, 5' UTR of alcohol oxidase 1 gene (aox1) containing the aox1 promoter region, Kozak sequence, mating factor alpha 1 (MFalpha1), AarI restriction site, aox1 terminator, his4 gene and 3' UTR of aox1 gene. Cloning and plasmid replication in E. coli are able via the pSB1C3 part.  The gene of interest (like laccase) can be included in frame with MFalpha1 via AarI restriction site. With the N-terminal MFalpha1 the POI could be secreted in the media. Genomic integation of MFalpha1-taged POI is able via the 5' UTR and 3' UTR of the aox1 gene. This allows a double cross over and the genomic integration without any bacterial proportion of DNA which could be a decisive point for industrial application. The complementation of histidine auxotrophie via his4 gene was chosen instead of a zeocine resistance. This selection strategy is chosen because we want to avoid the application of antibiotics.
Gibson assembly was used for building the shuttle vector (see the figure below) and the fragments with 5' overlap were amplified via PCR. In addition, the fragments were designed as basic BioBrick parts for different applications by the community. The origin of the DNA sequence for design of the shuttle vector and the source of DNA for PCR is listed in the table below.
Elements of the shuttle vector and their origin
Element
BioBrick
Origin of DNA sequence of design
Origin of DNA sequence of PCR
pSB1C3
pSB1C3
pSB1C3
pSB1C3
5'UTR of aox1
K863200
plasmid pPICZalphaA (Invitrogen)
P. pastoris wild type X-33
Kozak sequence
J63003
BBa_J63003
integrated in primer sequence
MFalpha1
K863206
plasmid pPICZalphaA (Invitrogen)
plasmid pPICZalphaA (Invitrogen)
aox1 terminater
K863203
plasmid pPIC9K
P. pastoris wild type X-33
his4
K863202
plasmid pPIC9K
P. pastoris wild type X-33
3'UTR of aox1
K863201
plasmid pPIC9K
P. pastoris wild type X-33
Figure 1: Plasmid map for shuttle vector pSB1C3::BBa_K863204.
Site directed mutagenesis of the shuttle vector
To remove the illegal XbaI restriction site in the shuttle vector, site directed mutagenesis was done. On the selection medium plattes some clones grow. Colony PCR was done for some of them, but in all tried cases the results are negative.
Shuttle vector in E. coli
The developed shuttle vector could be transformed in E. coli and analyzed by restriction with the enzymes PvuII and HindIII . The restriction pattern is positive. A sequencing was also done.
Shuttle vector in P. pastoris
After linearization by restriction with EcoRI and SpeI the shuttle vector could be transformed in competent P. pastoris cells. Transformants grow on selective media plates because of complementation of histidine auxotrophy. For differentiation between the M+ and Ms genotype a PCR with the primer 5AOX-Phenotype-FW and TT-Phenotype-RV have to be done. The M+ phenotype means that a single cross over recombination took place and the cultivation with methanol have no effect on the growth of the cells. The Ms phenotype means that a double cross over recombination took place and the gene aox1 was removed from the genome. During cultivation and induction with methanol the cells grow slow, because now only the gene aox2 exist in the genome.
GFP integrated in shuttle vector
Cloning of green fluorescent protein (GFP) via the AarI restriction site into the shuttle vector was done to test the system. A restriction analysis show that the construction were successful. The site directed recombination failed and no clones could be found.
Since Regionals: TVEL5 integrated in shuttle vector
The laccase TVEL5 was cloned via the AarI restriction site into the shuttle vector ( BBa_K863207 ). Restriction analysis and sequencing confirmed that success. Furthermore the linearized construct could be transformed into the yeast P. pastoris. Positive clones with single cross overs were identified by PCR on genomic DNA with the primer 5AOX-Phenotype-FW and TT-Phenotype-RV as shown in Figure 1 and 2. Positive clones with single cross over resulted in a 2.2 kb DNA band, the aox1 gene, and a 1.9 kb DNA band, the TVEL5 laccase DNA band. As control the shuttle vector ( BBa_K863204 ) itself was used. The expected DNA size is 437 bp.
Figure 1: BBa_K863204 was used as a control for PCR with the primer 5AOX-Phenotype-FW and TT-Phenotype-RV. Expected DNA band: 437 bp.
Figure 2: PCR product of the gDNA from the culture 1, 2 and 3 (C1, C2, C3) with the primer 5AOX-Phenotype-FW and TT-Phenotype-RV. Expected DNA bands: 1.9 kb (tvel5) and 2.2 kb (aox1).
The construct was integrated into the genome with a single cross over. This results in a M+ phenotype. Furthermore we could show that the production and secretion of the active laccase TVEL5 after induction with methanol worked. The clones were cultivated in minimal medium and induced with 1% (v/v) methanol twice a day for 2 days. The supernatant of the culture was examined to active laccases by testing the enzyme activity. For this purpose the supernatant was re-buffered into H2O with HiTrap Desalting Columns and incubated with 0.4 mM CuCl2. After 2 hours of incubation time 140 µL of the re-buffered supernatant were applied for the activity assay with 40 µL Britton Robinson buffer (pH 5) and measured at 25 °C. These setting were chosen because of the experiences with the TVEL0 laccase. To detect activity of the TVEL5 laccase 9 mM ABTS were used. Figure 3 shows a distinct activity of each cultivation (C1, C2, C3) in comparison to the control, which was treated like the other samples. The control, which was also treated with CuCl2 also represented the negative control to determine the effect of CuCl2 oxidizing ABTS. In Figure 4 the same experimental setup was used in a total volume of 1 mL. The oxidation of ABTS by TVEL5 laccase is visible to the naked eye in C1, C2 and C3 in comparison to the control. C2 shows the highest potential in oxidizing ABTS in the activity measurements and also in the experiments with a higher volume. This may be due to a higher TVEL5 concentration in the supernatant of this cultivation.
Figure 3:Activity assay of TVEL5 using the re-buffered, CuCl2 incubated supernatant of C1, C2 and C3 with 140 µL each. 40 µL Britton Robinson buffer (pH 5) and 9 mM ABTS were added. Oxidized ABTS was detected by measuring the change in OD at 420 nm at 25 °C for 8 hours.
Figure 4:Activity test of cultivation (culture 1, 2, 3) supernatant of BBa_K863207 . The control is the supernatant of P. pastoris GS115. The supernatants were re-buffered and CuCl2 incubated. Britton Robinson buffer and 9 mM ABTS were applied. Picture was taken after 8 hours incubation at 25 °C.
55px
About our Wiki
Fifteen students from different fields of study from the Bielefeld University aspired to complete a project called TOXIC COMPOUNDS IN NATURAL WATER - A CASE FOR LACCASE. In his Wiki we document our achievements and our approach.
Contact Us
Universität Bielefeld
Universitätsstraße 25
D-33615 Bielefeld
info(at)igem-bielefeld.de
Navigate our Wiki
Start
Team
Project
Results
Since Regionals
Human Practice
Safety
Judging
Partner
Retrieved from " http://2012.igem.org/Team:Bielefeld-Germany/Results/vector "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
