<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Amsterdam_Labjournal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Amsterdam/Labjournal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Amsterdam/Labjournal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Amsterdam/Labjournal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Amsterdam/Labjournal&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Amsterdam/Labjournal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Amsterdam/Labjournal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="navigation"><DIV style=" z-index:10000;"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany">Start</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Team</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Bielefeld-University">Bielefeld University</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Project</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Achievments">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Database">Database</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Outlook">Outlook</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Results</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Datapage">Datapage</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Laccases">Laccases</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/immo">Immobilization</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/substrate">Substrate Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/cbc">Cellulose Binding Domain</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/vector">Shuttle Vector</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/london">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Since Regionals</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Summary</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Expert">Expert</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Politics">Politics</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Sewage_treatment_plant">Industry</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human Practices/Outreach">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/StudentAcademy">Education</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Environment">Environment</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV></DIV><DIV id="page-title"><SPAN id="page-title-text">
                Labjournal
            </SPAN></DIV><DIV id="tab"><UL style="list-style-type:none"><LI><A href="#1"><STRONG>Week 24</STRONG></A></LI><LI><A href="#2"><STRONG>Week 25</STRONG></A></LI><LI><A href="#3"><STRONG>Week 26</STRONG></A></LI></UL></DIV><DIV id="labsite"><DIV id="anzeige"><H1>Week 24 (10/08 - 10/14/12)</H1><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_24_.2810.2F08_-_10.2F14.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 24 (10/08 - 10/14/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_October_08th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday October 08th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_October_09th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday October 09th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_October_10th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday October 10th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_October_11th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday October 11th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_October_12th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday October 12th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_October_13th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday October 13th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_October_14th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday October 14th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Monday_October_08th">Monday October 08th</SPAN></H3><P>All:
Day off in Amsterdam!!!
</P><H3><SPAN class="mw-headline" id="Tuesday_October_09th">Tuesday October 09th</SPAN></H3><H3><SPAN class="mw-headline" id="Wednesday_October_10th">Wednesday October 10th</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B></LI><LI> Ligation of pSB1C3::BBa_K863202 construct in KRX cells was done.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Gradient-PCRs of CBDcex_Freiburg; CBDclos_Freiburg; GFP_Freiburg and ecol_Freiburg (47 °C to 67°C)
</LI><LI> Product:
</LI><LI> CBDcex_Freiburg at all temperatures
</LI><LI> CBDclos_Freiburg at all temperatures
</LI><LI> GFP_Freiburg at all temperatures
</LI><LI> ecol_Freiburg at 53 °C to 61°C (best at 57 to 59 °C)
</LI><LI> pooled fractions for gel clean up
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Ordered Primers for the SDM of the illegal XbaI restriction site in the shuttle vector
</LI><LI><B>Team Cultivation and Purification:</B></LI><LI>Made precultures of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>for 19L fermentation (500mL preculture)
</LI><H3><SPAN class="mw-headline" id="Thursday_October_11th">Thursday October 11th</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Clean-Up of the Gel-pieces (from PCR-products)
</LI><LI> Restriction of PCR-products (3h):
</LI><LI> GFP_Freiburg:
</LI><LI> XbaI+PstI Tango-Buffer for Ligation 
</LI><LI> XbaI+AgeI Tango 4xAgeI for Assembly
</LI></UL><LI> CBDclos/cex:
</LI><LI> NgoMIV+PstI P.4+BSA 2xPstI for Assembly
</LI><LI> Assembly and Ligation (1h with T4):
</LI><LI> GFP_Freiburg + CBDcex_Freiburg + J61101
</LI><LI> GFP_Freiburg + CBDclos_Freiburg + J61101
</LI><LI> GFP_Freiburg + J61101
</LI><LI> Transformation in <I>E. coli</I> KRX (3 µL, chemical):
</LI><LI> GFP_Freiburg + CBDcex_Freiburg + J61101
</LI><LI> GFP_Freiburg + CBDclos_Freiburg + J61101
</LI><LI> GFP_Freiburg + J61101
</LI><LI> All Transformations were plated on AMP-select-agar
</LI><UL><LI><B>Team Cultivation and Purification</B></LI><LI> 12 L Cultivation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A></LI><LI>Settings: 
</LI></UL><P>Bioengineering NFL 19L fermenter, autoinduction HSG medium, 60 µg/mL chloramphenicol, 19 L, 37 °C, stirrer on cascade to hold a pO2 of 50 %, airflow: 12 NL/m, 16 hours. HSG medium was choosen to get a high biomass concentration with hope for a higher amount of laccases. 
</P><UL><LI>Made precultures of <I>E. coli</I> Rosetta Gami 2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; (09/09) behind a constitutive promoter. (500mL preculture)
</LI></UL><H3><SPAN class="mw-headline" id="Friday_October_12th">Friday October 12th</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> A lot of colonies on all three dishes (J61101+GFP_Freibug;J61101+GFP_Freibug+CBDcex;J61101+GFP_Freibug+CBDclos)
</LI><LI> no obvious green fluorescence, not even at J61101+GFP
</LI><LI> Maybe the combination of J23100 and J61101 isn't strong enough to express a visible amount of GFP
</LI></UL><LI> Colony-PCR of 16 colonies of every dish
</LI><LI> J61101 + GFP_Freiburg:
</LI><LI> 16/16 positive
</LI><LI> plated two on select-agar
</LI><LI> J61101 + GFP_Freiburg + CBDcex:
</LI><LI> 8/16 positve
</LI><LI> plated three on select-agar
</LI><LI> J61101 + GFP_Freiburg + CBDclos:
</LI><LI> 7/16 positive
</LI><LI> plated three on select-agar
</LI><UL><LI><B>Team Shuttle Vector:</B></LI><LI> The induction with 0.5% (v/v) methanol of P. pastoris GS115 cells with integrated <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> was started.
</LI></UL><UL><LI><B>Team Substrate Analysis</B>: 
</LI><LI> Since we have seen some new peaks after Estradiol treated with Laccases in the LC-MS, we wanted to have more degradation products to do an MS so we used a higher concentradion of Estradiol and Laccase and let them incubate for 66 hours.
</LI></UL><UL><LI><B>Team Cultivation and Purification</B></LI><LI> 12L Cutlivations of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> were harvested and stored at 4°C until purification. 
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 1:</B> Fermentation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K863005&lt;/partinfo&gt; (ECOL) in an Bioengineering NLF 19, scale: 12 L, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> + 60 µg/mL chloramphenicol, 37 °C, pH 7, agitation on cascade to hold pO<SUB>2</SUB> at 50 %, OD<SUB>600</SUB> measured every 30 minutes.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 2:</B> Fermentation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K863000&lt;/partinfo&gt; (BPUL) in an Bioengineering NLF 19, scale: 12 L, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> + 60 µg/mL chloramphenicol, 37 °C, pH 7, agitation on cascade to hold pO<SUB>2</SUB> at 50 %, OD<SUB>600</SUB> measured every 30 minutes.</DIV></DIV></DIV><UL><LI> 12 L Cultivation of <I>E. coli</I> Rosetta Gami 2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; (09/09)
</LI><LI> Settings: 
</LI></UL><P>Bioengineering NFL 19L fermenter, HSG medium, 60 µg/mL chloramphenicol, 300 µg/mL ampicillin, 12 L, 37 °C, stirrer on cascade to hold a pO2 of 50 %, airflow: 12 NL/m, 22-24 hours. HSG medium was chosen to get a high biomass concentration with hope for a higher amount of laccases.
</P><UL><LI><B>Team Substrate Analysis:</B></LI><LI> Dissolved estradiol, estrone and ethinyl estradiol in 50 % (v/v) acetonitrile with a concentration of 200 µg mL<SUP>-1</SUP></LI><LI> Creating  and measuring of negative controls for estradiol, estrone and ethinyl estradiol. Experimental setup: 2 µg substrate, 110 µL BR-buffer in 200 µL reaction volume. Measured at t<SUB>0</SUB> and after 3 hours.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_October_13th">Saturday October 13th</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B></LI><LI> The P. pastoris GS115 cells integrated with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> were also inducted with 0.5% (v/v) methanol (100%).
</LI></UL><UL><LI><B>Team Cultivation and Purification</B></LI><LI> 12L Cutlivations of <I>E. coli</I> Rosetta Gami 2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; (09/09) were harvested and stored at 4°C until purification.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 1:</B> Fermentation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K863005&lt;/partinfo&gt; (ECOL) in an Bioengineering NLF 19, scale: 12 L, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> + 60 µg/mL chloramphenicol, 37 °C, pH 7, agitation on cascade to hold pO<SUB>2</SUB> at 50 %, OD<SUB>600</SUB> measured every 30 minutes.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 2:</B> Fermentation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K863000&lt;/partinfo&gt; (BPUL) in an Bioengineering NLF 19, scale: 12 L, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> + 60 µg/mL chloramphenicol, 37 °C, pH 7, agitation on cascade to hold pO<SUB>2</SUB> at 50 %, OD<SUB>600</SUB> measured every 30 minutes.</DIV></DIV></DIV><UL><LI><B>Team Substrate Analysis:</B></LI><LI> Creating  and measuring of negative controls for estradiol, estrone and ethinyl estradiol. Experimental setup: 2 µg substrate, 0.1 mM ABTS, 110 µL BR-buffer in 200 µL reaction volume. Measured at t<SUB>0</SUB> and after 10 minutes.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_October_14th">Sunday October 14th</SPAN></H3><UL><LI><B>Team Shuttle Vector</B>:
</LI><LI> The P. pastoris GS115 cells integrated with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> were also inducted with 0.5% (v/v) methanol (100%).
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Plasmid isolation of pSB1C3::BBa_K863207 for transformation into P. pastoris GS115 was done. After isolation the plasmid was digested with <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials">NotI</A>, purified with the PCR clean up kit and transformed via <A href="/Team:Bielefeld-Germany/Protocols/molecular_genetics#Transformation_of_yeast_cells" title="Team:Bielefeld-Germany/Protocols/molecular genetics">protocol</A> in P. pastoris GS115.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Tried to isolate all plates, but only got low concentrations (5 ng/µL - 20 ng/µL)
</LI><LI> plated eight different colonies for isolation
</LI></UL><UL><LI><B>Team Substrate Analysis</B>:
</LI><LI> We took sample from the reaction of the 12th October.
</LI></UL></DIV><DIV id="anzeige"><H1>Week 25 (10/15 - 10/21/12)</H1><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_25_.2810.2F15_-_10.2F21.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 25 (10/15 - 10/21/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_October_15th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday October 15th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_October_16th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday October 16th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_October_17th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday October 17th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_October_18th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday October 18th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_October_19th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday October 19th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_October_20th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday October 20th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_October_21st"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday October 21st</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Monday_October_15th">Monday October 15th</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B></LI><LI> There are no colonies of pBS1C3::BBa_K863202 KRX transformantion.
</LI><LI> The P. pastoris GS115 cells integrated with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> were harvested and the supernatant was examined on GFP by fluorescence. But there was no GFP.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Yeah, the yeast cells with integrated <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863207" class="external text" rel="nofollow">BBa_K863207</A> are growing.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Tried to isolate the eight plated, but did get low concentrations again (except one J61101+GFP_Freiburg+CBDcex_Freiburg with conc &gt;100 ng/µL)
</LI><LI> cleaned up some more PCR-products (GFP_Freiburg; CBDcex_Freiburg; CBDclos_Freiburg)
</LI></UL><LI><B>Team Substrate Analysis</B>: We stopped the reaction from the 12th October with Methanol and gave it to Dr. Marcus Persicke to measure the degradation on LC-MS
</LI><UL><LI><B>Team Cultivation and Purification</B></LI><LI> Harvested Cells from 12L Cutlivations of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> as well as 12 L Cultivation of <I>E. coli</I> Rosetta Gami 2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; (09/09) were disrupted via high-pressure homogenizer and filtrated by <A href="http://www.millipore.com/catalogue/module/C7493" class="external text" rel="nofollow">Millipore Pellicon XL 50</A> with first 300 kDa to seperate all celldebris and 10 kDa to concentrate the laccases. The concentrated solutions were purified by the Ni-NTA-column.
</LI></UL><LI><B>Team Substrate Analysis:</B></LI><LI> Creating  and measuring of estradiol and ethinyl estradiol degradation samples for the laccase TVEL0. Experimental setup: 2 µg substrate, 0.1 mM ABTS, 110 µL BR-buffer and 10 µg laccase in 200 µL reaction volume. Measured at t<SUB>0</SUB> and after 10 minutes.
</LI><UL><LI><B>Team Immobilization</B></LI><LI>Started immobilization of TVEL0 to analyze the binding of laccase during the first 10 hours representative for following immobilizations.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_October_16th">Tuesday October 16th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI> Today we received the samples from Team Cultivation &amp; Purification. We are really motivated to measure the activity of our new produced enzymes. But before that we have to prepare the samples. This implied the determination of the protein concentration in each fraction, which we did today. The results are listed in the following table:
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Transformed B0034 into KRX
</LI><LI> Transformed J23100 into KRX
</LI><LI> Plated both on AMP-select-agar
</LI><LI> Plated one more Colony of J61101 + GFP_Freiburg and J61101 + GFP_Freiburg + CBDclos
</LI></UL><UL><LI><B>Team Substrate Analysis:</B> The Massspectrometry data showed two potential products after the Laccase treatment of Estradiol and Ethinyl estradiol but we unfortunatly could not identify them so we decided to do MS-MS on thoose. The products showed that after Laccase treatment both Estradiol and Ethinyl estradiol losses two <B>H</B> atoms which was new for us.
</LI></UL><UL><LI><B>Team Immobilization:</B></LI></UL><P>Taking samples for immobilization time analyzation.
</P><H3><SPAN class="mw-headline" id="Wednesday_October_17th">Wednesday October 17th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI> Before starting our activity assay we re-buffered the fractions we got from Team Cultivation and Purification. It took some time, that's why we had no time for activity analysis yet. But we prepared everything for the next day, including determining the protein concentration of the fractions after they have been re-buffered. This is what we got:
</LI></UL><UL><LI><B>Team Shuttle Vector:</B></LI><LI> Genomic DNA isolation of yeast cells with integrated <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> construct was done with the Kit. For phenotype characterisation an PCR with the primer pair 5AOX-Phenotype-FW and TT-Phenotype-RV was done and the fragment size was determined. The results are: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> was not integrated.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> YPD liquid culture was inoculated with one yeast colony (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863207" class="external text" rel="nofollow">BBa_K863207</A>) and incubated at 30°C and 150 rpm.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Picked colonies of B0034 and J23100 and plated them on AMP selection agar for plasmid isolation
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> SDM-PCR of the Shuttle vector
</LI><LI> Added <I>Dpn</I>I over night
</LI></UL><UL><LI><B>Team Cultivation and Purification:</B></LI><LI> Made SDS-Pages of the cultivations from 10/11 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>) and 10/12 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>, &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt;).
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 1:</B> SDS-Page of purification from the 12 L fermentations from 10/11 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>) and 10/12 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>, &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt;). Purification of the supernatant via His-trap column, step gradient with 5 %, 50 % and 100 % elution buffer.</DIV></DIV></DIV></DIV><UL><LI><B>Team Immobilization:</B></LI></UL><P>Measurement of Protein concentration with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" class="external text" rel="nofollow">Roti-Nanoquant</A> and analyses of binding capacity.
</P><H3><SPAN class="mw-headline" id="Thursday_October_18th">Thursday October 18th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI> Finally the day had come to measure the activity of our own produced laccases. This activity assay should give information about the enzyme content in every fraction. To make the measurements comparable we applied the same protein amount in every sample. Again, we used our <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Analytics#General_setup_of_enzyme_activity_measurements/" class="external text" rel="nofollow">standard activity assay protocol</A>, but this time we used the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Britton-Robinson_Buffer/" class="external text" rel="nofollow">Britton-Robinson Buffer</A> at pH 5. Also we applied 0.1 mM ABTS and measured over night.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of each purified fraction of our new cultivation with ECOL. Samples were re-buffered into H<SUB>2</SUB>O and the protein amount in each fraction has been adjusted. The Measurement was done using the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Analytics#General_setup_of_enzyme_activity_measurements/" class="external text" rel="nofollow">standard activity assay protocol</A> over night.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of each purified fraction of our new cultivation with BPUL. Samples were re-buffered into H<SUB>2</SUB>O and the protein amount in each fraction has been adjusted. The Measurement was done using the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Analytics#General_setup_of_enzyme_activity_measurements/" class="external text" rel="nofollow">standard activity assay protocol</A> over night.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of each purified fraction of our new cultivation with BHAL. Samples were re-buffered into H<SUB>2</SUB>O and the protein amount in each fraction has been adjusted. The Measurement was done using the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Analytics#General_setup_of_enzyme_activity_measurements/" class="external text" rel="nofollow">standard activity assay protocol</A> over night.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of each purified fraction of our new cultivation with TTHL. Samples were re-buffered into H<SUB>2</SUB>O and the protein amount in each fraction has been adjusted. The Measurement was done using the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Analytics#General_setup_of_enzyme_activity_measurements/" class="external text" rel="nofollow">standard activity assay protocol</A> over night.</DIV></DIV></DIV><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Isolated J23100 plasmid
</LI><LI> Digested J23100 with <I>Eco</I>RI and <I>Pst</I>I
</LI><LI> Digested pSB1C3-Backbone with <I>Eco</I>RI and <I>Pst</I>I
</LI><LI> Clean up of pSB1C3-BB and J23100 + RFP - Insert via Gel
</LI><LI> Ligation of J23100 and pSB1C3-BB
</LI><LI> Isolated B0034 plasmid
</LI><LI> Digested B0034 with <I>Spe</I>I and <I>Pst</I>I
</LI><LI> Digested GFP_Freiburg-PCR-product with <I>Xba</I>I and <I>Pst</I>I
</LI><LI> Digested an other fraction of the GFP_Freiburg-PCR-product with <I>Xba</I>I and <I>Age</I>I
</LI><LI> Assembled (Ligation):
</LI><LI> B0034 with GFP_Freiburg
</LI><LI> B0034 with GFP_Freiburg and CBDcex_Freiburg
</LI><LI> B0034 with GFP_Freiburg and CBDclos_Freiburg
</LI></UL><LI> Transformed these three into KRX
</LI><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Bands of 19C-PRC-product in Gel at 7 kbp (which is correct)
</LI><LI> Clean-Up of 19C-PRC-product
</LI><LI> Transformation of the 19C-PRC-product in XL1 Blue
</LI></UL><UL><LI><B>Team Immobilization:</B></LI><LI>Discussed new method of activity measurements for immobilized laccases on beads with less lacasse solution needed. Started first general tests with photometer.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_October_19th">Friday October 19th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI> After getting our interesting activity measurement results, we had to figure out which fraction is going to be used in the following experiments and how much laccase it contains besides other proteins. We agreed, that the most active fraction contains 90 % laccase, which is commonly used in the literature. Since we applied the same protein amount of each fraction for the activity measurements, we made sure that the results really correspond to the amount of laccase. These are the most active fractions we have chosen with their contained laccase amount:
</LI><LI> ECOL: Fraction 50% 2 with a laccase concentration of 63,9 µg mL<SUP>-1</SUP>.
</LI><LI> BPUL: Fraction 50% 2 with a laccase concentration of 25,1 µg mL<SUP>-1</SUP>.
</LI><LI> BHAL: Fraction  5% 3 with a laccase concentration of 10,9 µg mL<SUP>-1</SUP>.
</LI><LI> TTHL: The best fraction of our TTHL laccase was fraction 50% 1 with a laccase concentration of 4,03 µg mL<SUP>-1</SUP>. In regard to Team Immobilization and Team Substrate Analysis we had to reach a higher amount of TTHL laccase. So we thought of combining two fractions. We chose the second best fraction of TTHL, which is 5% 3. To calculate the amount of contained laccase we had to compare the activity and reconsidered the slope. Since we stated the fraction with its highest activity as 90 % contained laccase, we calculated the amount of laccase regarding to that one. The following figure shows the slopes we got out of the activity from fraction 50% 1 and fraction 5% 3 (which is the second best). Accordingly to this, fraction 5% 3 contains 68,9% of laccase, which is 4,8 µg mL<SUP>-1</SUP>. In total, the composed fraction contains 4,4 µg mL<SUP>-1</SUP>.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Slope of the two most active fractions of purified TTHL laccase. By comparing them, the amount of contained laccase in the fractions had been calculated.</DIV></DIV></DIV></DIV><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Colony-PCR of B0034 + GFP_Freiburg; B0034 + GFP_Freiburg + CBDcex; B0034 + GFP_Freiburg + CBDcex_Freiburg; B0034 + GFP_Freiburg + CBDclos_Freiburg.
</LI><LI> All negative
</LI></UL><LI> Restriction of Ecol_Freiburg with <I>Xba</I>I, <I>Age</I>I and <I>Dpn</I>I
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><UL><LI><B>Team Substrate Analysis:</B></LI><LI> Creating  and measuring of estradiol and ethinyl estradiol degradation samples for the laccases TVEL0 and BPUL. The experimental setup changed. The laccase amount was adjusted to the highest possible amount of the lowest concentrated laccase. TTHL has a concentration of 4,03 µg mL<SUP>-1</SUP> and sets the maximal laccase amount to 0.6 µg per 200 µL reaction volume. To compensate these worse conditions, the substrate amount was lowered to 1 µg and the temperature was changed to 30°C.
</LI><LI> Test of estradiol and ethinyl estradiol degradation done by TVEL0 and BPUL  without ABTS. Experimental setup: 1 µg substrate, 0.1 mM ABTS, 110 µL BR-buffer and 0.6 µg laccase in 200 µL reaction volume. Measured at t<SUB>0</SUB> and after 3 hours.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_October_20th">Saturday October 20th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Negativ control for substrate saturation analysis to investigate in the effect of CuCl<SUB>2</SUB> oxidizing ABTS. Instead of laccase in H<SUB>2</SUB>O, only H<SUB>2</SUB>O mixed with 0.4 mM CuCl<SUB>2</SUB> was applied.</DIV></DIV></DIV> During the Jamboree in Amsterdam we had the oppurtunity to discuss our activity results with a lot of people. One remark was about the calculation of the enzyme specific units. As you all remember, we measured the activty with 0.1 mM ABTS, but to get comparable results with the literature, we had to measure each protein with other concentrations of ABTS. The concentration of ABTS should be saturated for each laccase. So we started with ECOL and BPUL and applied different ABTS concentrations starting from 0.1 mM as usual: 0.1 mM, 0.5 mM, 1 mM and 5 mM. We applied 616 ng Laccase in each sample. Our results showed, that especially ECOL does not reach its substrate saturation with 5 mM ABTS. Our plans are now to check these concentrations of ABTS for BPUL and TTHL and also higher concentrations of ABTS for all laccases. For that we should reduce the enzyme amount to be able to detect the change in OD 420 properly.
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of our new ECOL laccase with different ABTS concentration to find the substrate saturation of ECOL. 616 ng ECOL laccase (incubated with 0.4 mM CuCl<SUB>2</SUB>) and 20 % Britton-Robinson buffer (pH 5) were applied.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of our new BPUL laccase with different ABTS concentration to find the substrate saturation of BPUL. 616 ng BPUL laccase (incubated with 0.4 mM CuCl<SUB>2</SUB>) and 20 % Britton-Robinson buffer (pH 5) were applied.</DIV></DIV></DIV><UL><LI><B>Team Substrate Analysis:</B></LI><LI> TVEL0 and BPUL were able to degrade 100 % of estradiol and 100 % of ethinyl estradiol. To get comparable results, the reaction temperature was lowered to 20 °C again, to have a closer look, where differences are apparent. Again TVEL0 and BPUL degraded 100 % of the substrates. But before the laccase stocks deplete, first ECOL, BHAL and TTHL should be tested.
</LI><LI> Did substrate degradation analysis with ECOL without ABTS. Experimental setup: 1 µg substrate, 0.1 mM ABTS, 110 µL BR-buffer and 0.6 µg laccase in 200 µL reaction volume. Measured at t<SUB>-0</SUB> and after 3 hours.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_October_21st">Sunday October 21st</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B></LI></UL><P>Genomic isolation was done with the <A href="http://www.promega.com/resources/protocols/technical-manuals/0/wizard-genomic-dna-purification-kit-protocol/" class="external text" rel="nofollow">Promega Wizard genomic DNA purification system kit</A> of our approximately 50 yeast clones with integrated (a) GFP and (b) tvel5. 
</P><UL><LI><B>Team Activity Tests:</B></LI><LI> Today was BHAL's and TTHL's turn. The experiment setup was the same as described the day before. But again we could not make sure, that 5 mM ABTS is the substrate saturation for our tested laccases. We are now sure to go higher in ABTS concentrations.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of our new BHAL laccase with different ABTS concentration to find the substrate saturation of BHAL. 616 ng ECOL laccase (incubated with 0.4 mM CuCl<SUB>2</SUB> and 20 % Britton-Robinson buffer pH 5) were applied.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of our new TTHL laccase with different ABTS concentration to find the substrate saturation of TTHL. 616 ng TTHL laccase (incubated with 0.4 mM CuCl<SUB>2</SUB> and 20 % Britton-Robinson buffer pH 5) were applied.</DIV></DIV></DIV><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Colonies PCRs of B0034 + GFP_Freiburg; B0034 + GFP_Freiburg + CBDcex_Freiburg; B0034 + GFP_Freiburg + CBDclos_Freiburg
</LI><LI> All negativ
</LI></UL><LI> Transformed once again (with an mix of old an new GFP_Freiburg)
</LI><LI> Isolated plated colonies of transformed colonies (two GFP; two GFP+Cex; two GFP+Clos 
</LI><LI> Restriction analysis showed only bands at 2K bp (pSB1C3+BBa_B0034 vector without insert)
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Isolated the plasmids of the six SDM-dishes
</LI><LI> Over-night digestion with <I>Not</I>I 
</LI><LI><B>Team Substrate Analysis:</B></LI><LI> ECOL, BHAL and TTHL were tested with the same conditions and all of them degraded the substrates much slower than TVEL0 and BPUL.
</LI><LI><B> Team Immobilization</B></LI><LI>Started immobilization of new laccases ECOL, BPUL, BHAL, TTHL and TVEL0.
</LI></DIV><DIV id="anzeige"><H1>Week 26 (10/22 - 10/28/12)</H1><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_26_.2810.2F22_-_10.2F28.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 26 (10/22 - 10/28/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_October_22nd"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday October 22nd</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_October_23rd"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday October 23rd</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_October_24th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday October 24th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_October_25th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday October 25th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_October_26th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday October 26th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_October_27th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday October 27th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_October_28th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday October 28th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Monday_October_22nd">Monday October 22nd</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B></LI></UL><P>The identification of positive yeast clones and determination of the phenotype (M<SUP>+</SUP> or M<SUP>S</SUP>) was done by PCR with the primers 5AOX-Phenotype-FW and TT-Phenotype-RV. The result of the PCR was: no GFP integrated clones and three tvel5 integrated (1.9 kb and 2.2 kb) clones. The cassette, including <I>tvel5</I> and <I>his4</I>, was recombinated by a single cross over, because the 2.2 kb PCR product of the <I>aox1</I> gene has been formed. <A href="/Team:Bielefeld-Germany/Protocols/Materials#Minimal_methanol_media_.28MM.29" title="Team:Bielefeld-Germany/Protocols/Materials"> Minimal methanol medium</A> was inoculated with these three positive clones with the M<SUP>+</SUP> phenotype.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863204" class="external text" rel="nofollow">BBa_K863204</A> was used as a controll for PCR with the primer 5AOX-Phenotype-FW and TT-Phenotype-RV. Expected lane: 437 bp.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">PCR product of the gDNA from the culture 1, 2 and 3 (C1, C2, C3) with the primer 5AOX-Phenotype-FW and TT-Phenotype-RV. Expected lane: 1.9 kb (<I>tvel5</I>) and 2.2 kb (<I>aox1</I>).</DIV></DIV></DIV><UL><LI><B>Team Activity Tests:</B></LI><LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Negative control for the experimental setup with different pHs of the Britton-Robinson buffer. Effect of 5 mM ABTS was determined.</DIV></DIV></DIV>Today was all about finding a comfy environment for our laccases. We thought of determining the right pH before going on with substrate saturation experiments since we could not be sure, that the activity was influenced by the pH. We chose a range from pH 4 to pH 9, applied 616 ng laccase and 5 mM ABTS. Our results revealed the following results:
</LI><LI> optimal pH of ECOL: pH 5
</LI><LI> optimal pH of BPUL: pH 4 and pH 5
</LI><LI> optimal pH of BHAL: pH 5
</LI><LI> optimal pH of TTHL: pH 5
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">BHAL activity measured under different pH conditions (pH 4 to pH 9) to determine the pH optimum. 616 ng Laccase and 5 mM ABTS were used.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">TTHL activity measured under different pH conditions (pH 4 to pH 9) to determine the pH optimum. 616 ng Laccase and 5 mM ABTS were used.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">ECOL activity measured under different pH conditions (pH 4 to pH 9) to determine the pH optimum. 616 ng Laccase and 5 mM ABTS were used.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">BPUL activity measured under different pH conditions (pH 4 to pH 9) to determine the pH optimum. 616 ng Laccase and 5 mM ABTS were used.</DIV></DIV></DIV><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Sequencing of &lt;partinfo&gt;BBa_K863120&lt;/partinfo&gt; in &lt;partinfo&gt;BBa_J61101&lt;/partinfo&gt; showed a deletion within the ORF of the GFP; this seemed to be the reason that the colonies are not fluorescent.
</LI><LI> Gradient-PCR on &lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt; with GFP_Freiburg Pre- and Suffix Primers
</LI><LI> Worked fine. Clean up and digestion with <I>Xba</I>I and <I>Pst</I>I
</LI></UL><LI> Gradient-PCR on &lt;partinfo&gt;BBa_K863103&lt;/partinfo&gt; with CBDcex_Freiburg and GFP_Freiburg_compl Primers
</LI><LI> One of 12 temperatures worked. Clean up and digestion with <I>Xba</I>I and <I>Pst</I>I
</LI><LI> Gradient-PCR on &lt;partinfo&gt;BBa_K863113&lt;/partinfo&gt; with CBDclos_Freiburg and GFP_Freiburg_compl Primers
</LI><LI> did not get product at all
</LI><LI> Ligation of the PCR products and GFP_Freiburg and CBDcex_Freiburg-GFP_Freiburg, respectively
</LI><LI> Transformation of both ligations
</LI><LI> Cells plated on selection agar
</LI><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Stopped Digestion of the shuttle vector, but gel went wrong and the products were lost
</LI></UL><UL><LI><B>Team Substrate Analysis:</B></LI><LI> BHAL and TTHL were tested with the same conditions and all of them degraded the substrates much slower than TVEL0 and BPUL.
</LI></UL><UL><LI><B> Team Immobilization:</B></LI><LI> Measured of protein concentration in supernatants of immobilized ECOL, BPUL, BHAL, TTHL and TVEL0 with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" class="external text" rel="nofollow">Roti-Nanoquant</A>. Analysed binding capacity of the different laccases.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_October_23rd">Tuesday October 23rd</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B> The induction of the yeast cells with 1% (v/v) 100% methanol was done at morning and in the evening.
</LI></UL><UL><LI><B>Team Activity Tests:</B></LI><LI> In order to find the substrate saturation we continued to measure our laccase activities with higher concentrations of ABTS. To have a slower reaction and also try not to waste the enzymes we halfed the amount of used laccases from 616 ng to 308 ng of each laccase. The following substrate saturations were determined:
</LI><LI> ECOL: 9 mM ABTS
</LI><LI> BPUL: 5 mM ABTS
</LI><LI> BHAL: 7 mM ABTS
</LI><LI> TTHL: 7 mM ABTS
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Colony PCR of B0034 + GFP_Freiburg
</LI><LI> 14/24 positive
</LI><LI> Plated three on selection agar for plasmid isolation
</LI></UL><LI> Colony PCR of B0034 + CBDcex-GFP_Freiburg
</LI><LI> 16/24 positive
</LI><LI> Plated three on selection agar for plasmid isolation
</LI><LI><B>Team Immobilization</B></LI><LI>Started measurement of enzymatic activity of the different immobilized laccases with the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Immobilization#Activity_measurement_of_beads" class="external text" rel="nofollow">Photometer</A>.
</LI><H3><SPAN class="mw-headline" id="Wednesday_October_24th">Wednesday October 24th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI> pH measurements were repeated under substrate saturation with a pH range from pH 4 to pH 9.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B> In the morning and evening the induction of yeast cells with 1% (v/v) methanol was done.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Isolated Plasmids containing B0034 + GFP_Freiburg &amp;</LI><LI> Digestion of pooled plasmids with <I>Xba</I>I and <I>Pst</I>I
</LI></UL><LI> Isolated Plasmids containing B0034 + CBDcex-GFP_Freiburg
</LI><LI> Digestion of pooled plasmids with <I>Xba</I>I and <I>Pst</I>I
</LI><LI> Digestion of J23100 with <I>Spe</I>I and <I>Pst</I>I
</LI><LI> Clean up of inserts (B0034 + GFP_Freiburg &amp; B0034 + CBDcex-GFP_Freiburg) and Backbone (J23100) via gel
</LI><LI> Ligation of Backbone and the inserts
</LI><LI> Transformation and plated on selection agar
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Digestion showed that all isolated colonies still have the illegal <I>Xba</I>I restriction side
</LI><LI><B>Team Substrate Analysis:</B></LI><LI> Repeating of the degradation experiments without ABTS for TVEL0, BPUL, ECOL, BHAL and TTHL with a degradation time of five hours to increase the differences between the laccases.
</LI><H3><SPAN class="mw-headline" id="Thursday_October_25th">Thursday October 25th</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Induction of the yeast cells with 1% (v/v) methanol was done. After 4 h the cells were harvested by centrifugation at 3000xg 4°C for 5 min. 
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> All colonies did not show any obvious green glow
</LI></UL><UL><LI><B>Team Substrate Anaylsis:</B></LI><LI>The MS-MS data arrived from Marcus Persicke. To ensure our measurements and discuss what happens after the Laccase treatment and on the ESI we decided to ask an organic chemistrer so we went to Prof. Dr. Dietmar Kuck whos expert on this field. Since he had less time to discuss we have only speculativ testifies.
</LI></UL><UL><LI><B>Team Cultivation and Purification:</B></LI></UL><P>Last offline analysis of the the two cultivations of <I>E.coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> with Carobon Source Detection with HPLC.
</P><UL><LI><B>Team Substrate Analysis:</B></LI><LI> Repeating of the degradation experiments with ABTS for ECOL, BHAL and TTHL to replace previous experimental errors.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_October_25th_2">Thursday October 25th</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Induction of the yeast cells with 1% (v/v) methanol was done. After 4 h the cells were harvested by centrifugation at 3000xg 4°C for 5 min. 
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> All colonies did not show any obvious green glow
</LI></UL><UL><LI><B>Team Substrate Anaylsis:</B></LI><LI>The MS-MS data arrived from Marcus Persicke. To ensure our measurements and discuss what happens after the Laccase treatment and on the ESI we decided to ask an organic chemistrer so we went to Prof. Dr. Dietmar Kuck whos expert on this field. Since he had less time to discuss we have only speculativ testifies.
</LI></UL><UL><LI><B>Team Activity Tests:</B></LI><LI> Activity assay of all 4 laccases and TVEL0 were done at 10 °C.
</LI><LI> TVEL5 was incubated in copper and after adjusting buffer composition of the supernatant the activity test was done. The result is shown in the following figure:
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Activity test of cultivation (culture 1, 2, 3) supernatant of <A href="http://partsregistry.org/Part:BBa_K863207" class="external text" rel="nofollow">BBa_K863207</A>. The control is the supernatant of <I>P. pastoris</I> GS115.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Friday_October_26th">Friday October 26th</SPAN></H3><UL><LI><B>Wiki day</B></LI><LI><B>Team Cultivation and Purification</B></LI></UL><P>Final offline analysis of the <I>E.coli</I> Rosetta Gami 2 cultivations with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; (09/09).
</P><H3><SPAN class="mw-headline" id="Saturday_October_27th">Saturday October 27th</SPAN></H3><H3><SPAN class="mw-headline" id="Sunday_October_28th">Sunday October 28th</SPAN></H3></DIV></DIV><TABLE cellspacing="10"><TBODY><TR><TD><A href="/File:Promega_logo_180x109.gif" title="File:Promega logo 180x109.gif">55px</A></TD></TR></TBODY></TABLE><DIV id="content-footer-box"><DIV class="content-footer-box-left"><DIV class="about"><P>About our Wiki</P><P>Fifteen students from different fields of study from the Bielefeld University aspired to complete a project called TOXIC COMPOUNDS IN NATURAL WATER - A CASE FOR LACCASE. In his Wiki we document our achievements and our approach. </P></DIV><DIV class="contact"><P>Contact Us</P><P>Universität Bielefeld
Universitätsstraße 25
D-33615 Bielefeld<A href="mailto:info@igem-bielefeld.de">info(at)igem-bielefeld.de</A></P></DIV></DIV><DIV class="content-footer-box-right"><DIV class="navigate"><P>Navigate our Wiki</P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany">Start</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Team</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Project</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Results</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Since Regionals</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Human Practice</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Labjournal">http://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Labjournal</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Amsterdam/Labjournal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Amsterdam/Labjournal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Amsterdam/Labjournal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Amsterdam/Labjournal&amp;oldid=296298" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>