<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Labjournal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Labjournal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Labjournal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Labjournal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Labjournal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="navigation"><DIV style=" z-index:10000;"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany">Start</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Team</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Bielefeld-University">Bielefeld University</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Project</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Achievments">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Database">Database</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Outlook">Outlook</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Results</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Datapage">Datapage</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Laccases">Laccases</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/immo">Immobilization</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/substrate">Substrate Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/cbc">Cellulose Binding Domain</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/vector">Shuttle Vector</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/london">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Since Regionals</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Summary</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Expert">Expert</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Politics">Politics</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Sewage_treatment_plant">Industry</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human Practices/Outreach">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/StudentAcademy">Education</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Environment">Environment</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV></DIV><DIV id="page-title"><SPAN id="page-title-text">
                Labjournal
            </SPAN></DIV><DIV id="navlab" class="tabs"><DIV class="scroller"><UL style="list-style-type:none"><LI><A href="#1"><STRONG>Prologue</STRONG></A></LI><LI><A href="#2"><STRONG>Week 1</STRONG></A></LI><LI><A href="#3"><STRONG>Week 2</STRONG></A></LI><LI><A href="#4"><STRONG>Week 3</STRONG></A></LI><LI><A href="#5"><STRONG>Week 4</STRONG></A></LI><LI><A href="#6"><STRONG>Week 5</STRONG></A></LI><LI><A href="#7"><STRONG>Week 6</STRONG></A></LI><LI><A href="#8"><STRONG>Week 7</STRONG></A></LI><LI><A href="#9"><STRONG>Week 8</STRONG></A></LI><LI><A href="#10"><STRONG>Week 9</STRONG></A></LI><LI><A href="#11"><STRONG>Week 10</STRONG></A></LI><LI><A href="#12"><STRONG>Week 11</STRONG></A></LI><LI><A href="#13"><STRONG>Week 12</STRONG></A></LI><LI><A href="#14"><STRONG>Week 13</STRONG></A></LI><LI><A href="#15"><STRONG>Week 14</STRONG></A></LI><LI><A href="#16"><STRONG>Week 15</STRONG></A></LI><LI><A href="#17"><STRONG>Week 16</STRONG></A></LI><LI><A href="#18"><STRONG>Week 17</STRONG></A></LI><LI><A href="#19"><STRONG>Week 18</STRONG></A></LI><LI><A href="#20"><STRONG>Week 19</STRONG></A></LI><LI><A href="#21"><STRONG>Week 20</STRONG></A></LI><LI><A href="#22"><STRONG>Week 21</STRONG></A></LI><LI><A href="#23"><STRONG>Week 22</STRONG></A></LI><LI><A href="#24"><STRONG>Week 23</STRONG></A></LI><LI><A href="#25"><STRONG>Week 24</STRONG></A></LI><LI><A href="#26"><STRONG>Week 25</STRONG></A></LI><LI><A href="#27"><STRONG>Week 26</STRONG></A></LI></UL></DIV></DIV><DIV id="labsite"><DIV id="anzeige"><H1>Prologue</H1><H2><SPAN class="mw-headline" id="Starting_the_team">Starting the team</SPAN></H2><P>Beginning in january and february members of the former iGEM team from Bielefeld started seminars to inform interested students about synthetic biology, iGEM and the past Bielefeld projects. In March the final 2012 iGEM Bielefeld team was formed of 15 students and weekly meetings began. Our team was established and it was time to find a suitable project.
</P><H2><SPAN class="mw-headline" id="Find_a_project">Find a project</SPAN></H2><P>The first weekly meeting were more like big group brainstorming and we discussed idea, which in some cases were totally different from each other. Everyone had to inform about ideas of others so that, in the end, we all could discuss together.
</P><P>First project ideas were:
</P><UL><LI> the detection of multiresistent pathogens
</LI><LI> communication between bacteria and fungi using quorum sensing
</LI><LI> a bacterial hand warmer
</LI><LI> a possibility to detect and destroy mold fungus
</LI><LI> something about spontaneous combustion of hay bale
</LI><LI> an enzyme dispenser
</LI></UL><P>For most of the ideas little information was available. For example spontaneous combustion of hay bales is probably a combination of the metabolisms of different microorganisms and fungus. After some reports in media and press about the environmental effects of steroid hormones, we decided to go for hormones. From the beginning our aim was not to detect but to degrade hormones. We found several possible ways for degradation as there are the hydrolysis of estradiol-derivates with sufatases and glucoronidases. But we thought the best way to degrade steroid hormones would be with the use of laccases. Laccases have the ability to radicalize aromatic rings and can therefore be used to degrade or polymerize a broad range of substances, such as steroid hormones, special insecticides, polycyclic aromatic carbohydrates and aromatic acids. In nature laccases are often used for degradation or polymerisation of lignin or pigments.
</P><H2><SPAN class="mw-headline" id="Molding_together_to_a_team">Molding together to a team</SPAN></H2><P>After we found our project idea we decided to have a get-to-know-weekend with some presentations about iGEM, important methods and ideas for human practices. We also held presentations about other possible iGEM projects to extend our horizon, as there were: <I>e.g.</I> RNA aptamers  and magnetotactic bacteria.
But the most important part of this weekend was the growing as a team. We realized that we all had one summer to work together, have fun together and most important to stand up together as a team.
</P><H2><SPAN class="mw-headline" id="Find_the_right_one_for_the_right_job">Find the right one for the right job</SPAN></H2><P>Now it was time to organize the work and find a suitable task for everyone. In a developing team a lot of different jobs have to be done, <I>e.g.</I>:
</P><UL><LI> finding sponsors
</LI><LI> communication with the public
</LI><LI> human practices
</LI><LI> wiki- and homepage-design
</LI><LI> modelling
</LI><LI> a forum for exchange of information
</LI><LI> a joker, who entertains the team and lifts the mood
</LI></UL><P>And finally lab work began, feel free to follow us in our weekly labjournal and have a look how our labwork, results and of course problems and their solutions, evolved.

		</P></DIV><DIV id="anzeige"><H1>Summary of Week 1</H1><H2><SPAN class="mw-headline" id="Week_1_.2804.2F30_-_05.2F06.2F12.29">Week 1 (04/30 - 05/06/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_1_.2804.2F30_-_05.2F06.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 1 (04/30 - 05/06/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_April_30th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday April 30th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_May_1th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday May 1th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_May_2th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday May 2th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_May_3th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday May 3th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_May_4th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday May 4th</SPAN></A></LI></UL><UL><LI> Start of our WET LAB time.
</LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar"> Weekly Seminar </SPAN></H3><UL><LI> Do we want to order strains of <I>Trametes versicolor</I> and <I>Trametes villosa</I>?
</LI><LI> Gathering information about signal sequences in yeast
</LI><LI> Decision to create a database, so that we can easily number and inscribe our lab results
</LI><LI> Decision to arrange a summer school for pupils in their last year before the final exams
</LI><LI> Discussion about how to meet a member of the german <I><A href="http://www.bundestag.de/" class="external text" rel="nofollow">Bundestag</A></I> (the german parliament)
</LI></UL><H3><SPAN class="mw-headline" id="Monday_April_30th"> Monday April 30th </SPAN></H3><UL><LI><B>Team Student Academy:</B></LI></UL><DL><DD><UL><LI>We got the chance to organize one part of the first school academy “synthetic biology/ biotechnology” at the CeBiTec of University Bielefeld by arranging experiments for the pupils and by presenting us and the iGEM competition. For the experimental part our general idea was to give them an understanding of principle methods in biotechnology / synthetic biology by using fluorescent proteins. We planned the following experiments:
</LI><LI> Plasmid isolation of RFP/GFP from a liquid culture.
</LI><LI> Transformation of a plasmid mixture consisting of two different fluorescent proteins (e.g. RFP and GFP) and different antibiotic resistances into <I>E. coli</I> KRX. It will be plated out on LB agar plates without antibiotics and on plates containing one of the two antibiotics, which are present on the plasmids. This way we can demonstrate the effect of antibiotics as selective pressure. 
</LI></UL></DD></DL><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI></UL><P align="justify"><UL><LI>Before our lab time started we sent requests for different plasmids with the desired laccase genes to working groups, which have already worked with laccases we are interested in. Sadly just one working group responded to us. We got answer for a vector with the laccase-ORF <A href="http://www.ncbi.nlm.nih.gov/protein/194015788" class="external text" rel="nofollow">CotA</A> from <I>Bacillus pumilus ATCC7061</I> and an ampicillin resistance from the Swiss Federal Laboratories for Materials Science and Technology, Laboratory for Biomaterials in Switzerland. They promised to send us the plasmid pBpL6. <A href="http://www.biomedcentral.com/1472-6750/11/9" class="external text" rel="nofollow">More information...</A> For an uniformly labeling we will further call this laccase BPUL. 
</LI><LI> In a <A href="http://www.ncbi.nlm.nih.gov/pubmed/21790191" class="external text" rel="nofollow">publication</A> we found a research group who worked with the laccase <A href="http://www.ncbi.nlm.nih.gov/protein/21230052" class="external text" rel="nofollow">CopA</A> from <I>Xanthomonas pv. campestris</I> ATCC33913. Luckily the sequence of this laccase is the same in <I>Xanthomonas campestris pv. campestris B100</I> which we got from a research group at our university. We will call this laccase XCCL in our wiki from now on. 
</LI><LI>We found a <A href="http://www.springerlink.com/content/x33205rp257397kr/" class="external text" rel="nofollow">publication</A> which described the laccase <A href="http://www.ncbi.nlm.nih.gov/protein/85674340" class="external text" rel="nofollow">CueO</A> from <I>E. coli W3110</I>. After blasting this laccase we found out that <I>E. coli</I> BL21(DE3) has this laccase, too. We decided to isolate the laccase from <I>E. coli BL21(DE3)</I> because this strain is available in our lab. 
</LI><LI> We generated new competent <I>E.coli</I> KRX cells.
</LI><LI> For the extraction of genomic DNA we cultivated <I>Xanthomonas campestris B100</I> and <I>E. coli</I> BL21(DE3). The bacterial strains we got from a working group at our University. After cultivation we isolated the genomic DNA. The DNA was needed as template for PCRs to purify the wanted laccase ORFs. 
</LI><LI> Primer design for the isolation of laccases from genomic DNA of <I>Xanthomonas campestris</I> B100 and <I>E. coli</I> BL21(DE3) and from plasmid pBpL6 <I>Bacillus pumilus</I> ATCC7061.
</LI><LI> In a team meeting earlier in the project we decided that we want to express the laccases under control of an inducible T7 promoter so we can choose the time point of induction. Additionally we decided that we want to use a c-terminal His-tag. Therefore the forward primers were designed with T7 promoter, RBS and the first 20 bases of the wanted gene. The reverse primers were designed with the last 20 bases of the wanted gene without the stop codon, a His-Tag and two stop codons. <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols#Primers" class="external text" rel="nofollow">Primers</A>: Xcc_LAC_FW_T7, Xcc_LAC_RV_HIS, E.coli_LAC_FW_T7, E.coli_LAC_RV_HIS, B.pumi_LAC_FW_T7 and B. pumi_LAC_RV_HIS
</LI></UL></P><H3><SPAN class="mw-headline" id="Tuesday_May_1th"> Tuesday May 1th </SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI> Searching for two plasmids with different fluorescent proteins and antibiotic resistance in parts registry. Found <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A>, a Plasmid with RFP and chloramphenicol resistance (but lacI and CAP sensitive), <A href="http://partsregistry.org/Part:BBa_J23100" class="external text" rel="nofollow">BBa_J23100</A>, a plasmid with RFP and ampicillin resistance and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I13522" class="external text" rel="nofollow">BBa_I13522</A>, a Plasmid with GFP and ampicillin resistance in Kit Plate 2011.
</LI></UL><UL><LI><B>Team Database:</B></LI><LI> We decided to create a database, so that we can easily number and inscribe our lab results and that everybody has the chance to find out witch results the other groups has so far. 
</LI><LI> Starting to look for possibilities to design such a database.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_May_2th"> Wednesday May 2th </SPAN></H3><UL><LI><B>Team Activity Test</B>: Good morning everybody and welcome to the labjournal of Team Activity Tests. Today we started our work with some literature research about enzyme activity tests, laccases and its substrates. So today was filled with online research, reading papers and collecting information about the laccases our team decided to use.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_May_3th"> Thursday May 3th </SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>:
</LI><LI> After the vector with the laccase gene bpul from <I>Bacillus pumilus</I> arrived, we transformed it into the competent <I>E. coli</I> KRX to have a larger amount of vector. The protocol we used was as followed: 
</LI><LI> The electroporation setup: U = 2,5 kV, C = 25 µF and R = 400 Ω
</LI><LI> Since we did not know the efficient of our competent KRX we used two different <I>E.coli</I> volumes for the transformation, 50 µL and 100 µL. We gave 50 µL 10% glycerol to the reaction tubes with 1 µL of the vector DNA (<I>Bacillus pumilus</I>). After the transformation we plated them into ampicillin plates.
</LI></UL><LI> PCR with the <I>Xanthomonas campestris</I> B100 and <I>E. coli</I> BL21(DE3) genomic DNA to isolate the laccases. Therefore we used the primers Xcc_LAC_FW_T7, Xcc_LAC_RV_HIS, E.coli_LAC_FW_T7 and E.coli_LAC_RV_HIS which are listed under Materials.
</LI><H3><SPAN class="mw-headline" id="Friday_May_4th"> Friday May 4th </SPAN></H3><P><B>Team Cloning of Bacterial Laccases</B>: We did Colony PCR on the transformed the <I>Bacillus pumilus</I> CotA plasmid. Unfortunately the control with colony PCR didn't work. So we just picked some colonies for plasmid isolation in the hope that on the AMP plate were no false positives colonies.
</P></DIV><DIV id="anzeige"><H1>Summary of Week 2</H1><H2><SPAN class="mw-headline" id="Week_2_.2805.2F07_-_05.2F13.2F12.29">Week 2 (05/07 - 05/13/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_2_.2805.2F07_-_05.2F13.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 2 (05/07 - 05/13/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_May_7th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday May 7th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_May_8th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday May 8th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_May_9th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday May 9th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_May_10th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday May 10th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_May_11th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday May 11th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_2"> Weekly Seminar </SPAN></H3><UL><LI> Found our first sponsors: <A href="http://corporate.evonik.com/en/Pages/default.aspx" class="external text" rel="nofollow">Evonik</A>, <A href="http://www.biocircle.com/en-ca/" class="external text" rel="nofollow">BioCircle</A> and <A href="http://www.merckgroup.com/en/index.html" class="external text" rel="nofollow">Merck</A>, now treaties have to be created and signed.
</LI><LI> Julia V. is working on the database.
</LI><LI> Decision to organize a waffle sale to fill up our petty cash.
</LI><LI> Gabi and Isabel are designing a poster for the waffle sale.
</LI><LI> For our human practices we wanted to find a sociology student, willing to think about bioethics, but did not succeed yet.
</LI><LI> Our video is nearly done, it is cut and only needs be underlain with music.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_May_7th"> Monday May 7th </SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI>First transformation of <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I13522" class="external text" rel="nofollow">BBa_I13522</A> and plating on selective agar. Result: We got little colonies. There weren’t any green colonies and only some pale red fluorescent colonies.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> More PCRs of laccase genes xccl from <I>Xanthomonas campestris pv. campestris</I> B100 and ecol from <I>E. coli</I> BL21(DE3) with the isolated genomic DNA as template and Xcc_LAC_FW_T7 / Xcc_LAC_RV_HIS and E.coli_LAC_FW_T7 / E.coli_LAC_RV_HIS primer pairs.
</LI><LI> Since we wanted to screen and characterize laccases from different bacteria we had to order the bacterial strains which weren't available at Bielefeld University from <A href="http://www.dsmz.de/|" class="external text" rel="nofollow"><I>DSMZ</I></A>. Below is a list of the ordered strains and the laccases we want to isolate from these strains.
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/protein/46197298" class="external text" rel="nofollow">Laccase</A> from <A href="http://www.dsmz.de/catalogues/details/culture/DSM-7039.html?tx_dsmzresources_pi5%5BreturnPid%5D=304|" class="external text" rel="nofollow"><I>Thermus thermophilus HB27</I></A> (look <A href="http://www.ncbi.nlm.nih.gov/pubmed/15999224" class="external text" rel="nofollow">here</A> for a publication to this laccase)
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/protein/10174701" class="external text" rel="nofollow">BH2082</A> from <A href="http://www.dsmz.de/catalogues/details/culture/DSM-18197.html?tx_dsmzresources_pi5%5BreturnPid%5D=304|" class="external text" rel="nofollow"><I>Bacillus halodurans C-125</I></A> (look <A href="http://www.ncbi.nlm.nih.gov/pubmed/15293032" class="external text" rel="nofollow">here</A> for a publication to this laccase)
</LI><LI> We ordered <A href="http://www.dsmz.de/catalogues/details/culture/DSM-40069.html?tx_dsmzresources_pi5%5BreturnPid%5D=304|" class="external text" rel="nofollow"><I>S. lavendulae sp. lavendulae</I> ATCC 14158</A>. Originally we wanted the strain <I>Streptomyces lavendulae</I> REN-7 but this strain isn't available at DSMZ. So we now hope that the laccase  gene <A href="http://www.ncbi.nlm.nih.gov/nuccore/23491745" class="external text" rel="nofollow">STSL</A> from <I>Streptomyces lavendulae</I> REN-7 is similar to that from <I>S. lavendulae sp. lavendulae ATCC 14158</I> because there's no DNA sequence for the laccase from this strain available. <A href="http://www.ncbi.nlm.nih.gov/pubmed/14586105" class="external text" rel="nofollow">publication</A></LI><LI> We wanted the laccase <A href="http://www.ncbi.nlm.nih.gov/protein/182434812" class="external text" rel="nofollow">EpoA</A> from <I>Streptomyces griseus</I> IFO 13350 (for the publication look <A href="http://jb.oxfordjournals.org/content/133/5/671.full.pdf" class="external text" rel="nofollow">here</A>. This strain was not available so we ordered <A href="http://www.dsmz.de/catalogues/details/culture/DSM-40236.html?tx_dsmzresources_pi5%5BreturnPid%5D=304|" class="external text" rel="nofollow"><I>Streptomyces griseus</I> ATCC 10137</A>. Unfortunately for this strain are no blast results after blasting the <A href="http://www.ncbi.nlm.nih.gov/protein/182434812" class="external text" rel="nofollow">laccase</A> from <I>Streptomyces griseus</I> IFO 13350 against database. So we decided to make primers for the laccase sequence from <I>Streptomyces griseus</I> IFO 13350 in the hope that the sequences are similar enough to get a PCR product.
</LI></UL><LI><B>Team Modeling</B>: 
</LI><LI>Looking for suitable software and enzymkinetics to model the degradation of our substrates with the different laccases. Finding the Michaelis-Menten kinetics and matlab.
</LI><H3><SPAN class="mw-headline" id="Tuesday_May_8th"> Tuesday May 8th </SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI> Repetition of the transformation didn’t change the result. We made a liquid culture of <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A>, but it did not fluoresce. Searching for mistakes and alternatives. Maybe competent cells are not that good and in case of RFP the lacI sensitivity could be the problem.
</LI></UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> After some empty agarose gels we finally isolated  the laccase gene bpul from <I>Bacillus pumilus</I> ATCC7061 as PCR product with the desired overhanging ends. As template we used the plasmid we got from the Swiss working group.
</LI><H3><SPAN class="mw-headline" id="Wednesday_May_9th"> Wednesday May 9th </SPAN></H3><UL><LI><B>Team Activity Test</B>: From the information we collected during our literature research we created a protocol for our first experiments. We decided to check the activity via a photometer. The one we may use here at the Cebitec is a Tecan Microplate reader. Check  <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Tecan_Infinite_Microplate_Reader" class="external text" rel="nofollow">protocols</A> for further information. If oxidized by laccase, ABTS can me measured at 420 nm. Also we found out that sodium acetate buffer (100 mM / pH 5) would give an optimal environment to our enzyme. So let´s have a look at our protocol:
</LI><LI><SPAN style="text-decoration: underline;">Initial laccase activity test:</SPAN></LI><LI>100 mM sodium acetate buffer, pH 5.0
</LI><LI>5 mM ABTS
</LI><LI>8 U laccase
</LI><LI>ad 200 µL deionized H<SUB>2</SUB>0
</LI></UL><LI>Also we talked about further characterization after accomplishing the first experiments and confirming that the used concentrations are a good choice. We are planning to buy and characterize the laccase from <I>T.versicolor</I> (TVEL0), to have a comparison to our future recombinant laccases. That laccase we are going to analyze in sodium acetate buffers that are adjusted to pH 1, 3, 5, 7 and 9. Further we are going to analyze the effect of different temperatures on the enzymes activity. For that we will first do some more research on the temperatures of the waste water in clarification plants here in Germany. Also we found out that an addition of copper does enhance the laccases activity, so we are going to do some measurements with copper concentrations from 0.1 mM to 0.5 mM in each sample. This seems like some great experiments for the start, so next we are going to order what we need to do the measurements.
</LI><UL><LI><B>Team  Database</B></LI><LI> Finding a first initial design
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 1: The first initial design of the database.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Thursday_May_10th"> Thursday May 10th </SPAN></H3><UL><LI><B>Team Student Academy</B></LI><LI> Testing the competent cells by transformation of pUC19. The transformation did not work that good, so that we produced new ones.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases</B></LI><LI> We got the ordered strains from <A href="http://www.dsmz.de/" class="external text" rel="nofollow">DSMZ</A>. So we did PCR on <I>Thermus thermophilus</I> genomic DNA. First we dissolved some of the lyophilized powder in water and for opening the cells we boiled them for a few minutes. The primers we used were T.thermo_LAC_FW_T7 and T.thermo_LAC_RV_HIS to get the laccase with the same overhangs described in <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">Monday April 30th</A>. Finally with additional DMSO and GC-buffer we had a product of the GC-rich laccase.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_May_11th"> Friday May 11th </SPAN></H3><UL><LI><B>Team Activity Tests</B>: For some pre test and characterization for our future laccase activity standard we ordered <A href="http://www.sigmaaldrich.com/catalog/product/sigma/53739?lang=de&amp;region=DE" class="external text" rel="nofollow">laccase</A> from <I>Trametes versicolor</I>. As well we had to order a substrate that the laccase could use to demonstrate its abilities. According to the literature <A href="http://www.sigmaaldrich.com/catalog/product/sigma/a1888?lang=de&amp;region=DE" class="external text" rel="nofollow">ABTS</A> is a well working substrate to characterize oxidizing enzym activity. So we ordered.
</LI></UL><UL><LI><B>Team Database</B>: Conceptual Design: In this phase of the design we try to find relations between the different tables and build a entity-relationship-model. This model helps to visualize the entities (our different tables) together with their attributes (the entries belonging to a table) and the relationship between these entities. The next figure shows a very easy E-R-Model.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 2: E-R-Model of our database.</DIV></DIV></DIV></DIV></DIV><DIV id="anzeige"><H1>Summary of Week 3</H1><H2><SPAN class="mw-headline" id="Week_3_.2805.2F14_-_05.2F20.2F12.29">Week 3 (05/14 - 05/20/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_3_.2805.2F14_-_05.2F20.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 3 (05/14 - 05/20/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_May_14th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday May 14th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_May_15th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday May 15th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_May_16th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday May 16th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_May_17th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday May 17th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_May_18th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday May 18th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_May_19th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday May 19th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_May_20th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday May 20th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_3"> Weekly Seminar </SPAN></H3><UL><LI> first lab service: Robert
</LI><LI> our GFP, which we wanted to use for the summer school for pupils, does not work.
</LI><LI> first competent cells have to be made: Julia S. and Robert.
</LI><LI> Decision to buy a commercial laccase to establish the analytics and the enzyme activity tests.
</LI><LI> our scientific exposé has to be translated into english: Malak
</LI><LI> Last planning for our waver sell.
</LI><LI> Julia S. is creating a vector for <I>Pichia pastoris</I> and is now searching for usable sequences.
</LI><LI> The <A href="http://www.bmbf.de/en/index.php" class="external text" rel="nofollow">BMBF</A> invites all german iGEM teams to Berlin to attend at the Biotechnologie2020+ strategy process.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_May_14th">Monday May 14th</SPAN></H3><H3><SPAN class="mw-headline" id="Tuesday_May_15th">Tuesday May 15th</SPAN></H3><UL><LI><B>Team Database</B>: 
</LI><LI>Creating the logically design for the database. For example for our table 'Eppi' we opt for the following logic:
</LI></UL><TABLE><TBODY><TR><TD>eppi ( </TD><TD> ID INT(11),
</TD></TR><TR><TD>DataID INT(11),
</TD></TR><TR><TD>Caption TEXT,
</TD></TR><TR><TD>Content TEXT,
</TD></TR><TR><TD>Experiment VARCHAR(255),
</TD></TR><TR><TD>Where2 VARCHAR(255) ,
</TD></TR><TR><TD>Location TEXT,
</TD></TR><TR><TD>Coments TEXT,
</TD></TR><TR><TD>Empty ENUM ('N', 'J'),
</TD></TR><TR><TD>Create_User INT(11),
</TD></TR><TR><TD>Creat_Date DATETIME,
</TD></TR><TR><TD>Edit_Date DATETIME,
</TD></TR><TR><TD>Delete ENUM ('N', 'J')
</TD></TR><TR><TD>Delete_User INT(11),
</TD></TR><TR><TD>Delete_Date DATETIME )
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Wednesday_May_16th">Wednesday May 16th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>: 
</LI></UL><DL><DD><UL><LI>For cloning our laccases we need pSB1C3 backbone. Therefore we we transformed <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A> (pSB1C3 with RFP) in competent KRX cells.
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="Thursday_May_17th">Thursday May 17th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>: Plasmid isolation of &lt;partinfo&gt;BBa_J04450&lt;/partinfo&gt;.
</LI></UL><UL><LI><B>Team Modeling</B>: 
</LI><LI>Meeting Mrs. Lutter, a mathematics prof. of our course of studies and looking for our first model of a metabolic pathway, finding out, that we don't need such a complex model. Start thinking that we want and what we need.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_May_18th">Friday May 18th</SPAN></H3><UL><LI><B>Team Activity Tests</B>: Our <A href="http://www.sigmaaldrich.com/catalog/product/sigma/53739?lang=de&amp;region=DE" class="external text" rel="nofollow"><I>T.versicolor</I> laccase</A> and the <A href="http://www.sigmaaldrich.com/catalog/product/sigma/53739?lang=de&amp;region=DE" class="external text" rel="nofollow">ABTS</A> arrived! We couldn´t wait to start, so we set up the stock solutions we will need, such as sodium acetat buffer (pH 5), 10 mM ABTS and deluted laccase.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_May_19th">Saturday May 19th</SPAN></H3><H3><SPAN class="mw-headline" id="Sunday_May_20th">Sunday May 20th</SPAN></H3></DIV><DIV id="anzeige"><H1>Summary of Week 4</H1><H2><SPAN class="mw-headline" id="Week_4_.2805.2F21_-_05.2F27.2F12.29">Week 4 (05/21 - 05/27/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_4_.2805.2F21_-_05.2F27.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 4 (05/21 - 05/27/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_May_21st"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday May 21st</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_May_22nd"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday May 22nd</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_May_23rd"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday May 23rd</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_May_24th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday May 24th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_May_25th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday May 25th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_May_26th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday May 26th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_May_27th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday May 27th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_4"> Weekly Seminar </SPAN></H3><UL><LI> Lab service: Isabel
</LI><LI> We try to establish a collaboration with the iGEM team from <A href="https://2012.igem.org/Team:SDU-Denmark" class="external text" rel="nofollow">SDU-Denmark</A>.
</LI><LI> Got our distribution kits
</LI><LI> First successful cloning and cultivations.
</LI><LI> Who wants to be a summer school teacher?
</LI><LI> We will not travel to the ACHEMA because only local teams are invited.
</LI><LI> Do we want to participate in the Biolympics? (It's a sports party with fun organized by the <A href="http://bts-ev.de/" class="external text" rel="nofollow">bts</A>).
</LI></UL><H3><SPAN class="mw-headline" id="Monday_May_21st">Monday May 21st</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>:
</LI><LI> We wanted to clone our  laccase PCR products xccl and ecol in pSB1C3 backbone. Therefore we did some restriction digests on the PCR products and the vector <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A>.
</LI></UL><UL><LI><B>Team Modeling</B>: Our aims for modeling:
</LI><LI> model the expression of the laccases in the organisms.
</LI><LI> model the activity of our enzymes.
</LI><LI> model the interesting parts of a clarification plant (the part witch are interesting for our cleaner.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_May_22nd">Tuesday May 22nd</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>: 
</LI></UL><DL><DD><UL><LI>Ligation  of the digested PCR products in pSB1C3 backbone and transformation in KRX electro-competent cells.
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="Wednesday_May_23rd">Wednesday May 23rd</SPAN></H3><UL><LI><B> Team Student Academy:</B></LI><LI> Repetition of the transformation of <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I13522" class="external text" rel="nofollow">BBa_I13522</A> with new competent <I>E. coli</I> KRX cells. Got intense red fluorescing colonies but no green fluorescing colonies. Made a backup of <I>E. coli</I> KRX <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A>. 
</LI><LI> Asking for other plasmids containing GFP at the working groups of our University.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases</B>: 
</LI></UL><P align="justify"><UL><LI> After there were no colonies on our pSB1C3 + xccl(T7)_His (<I>Xanthomonas campestris</I>) transformation plate we did the transformation with the same ligation preparation again. The other ligation with pSB1C3 + ecol(T7)_His (<I>E. coli</I>) showed colonies so we started colony PCRs to find positive colonies. Sadly the colony PCRs showed no products but the problem was that we just had the long overhang primers (E.coli_LAC_FW_T7 and E.coli_LAC_RV_HIS ). Therefore we ordered the F and R <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols#Primers" class="external text" rel="nofollow">primers</A>.
</LI></UL></P><H3><SPAN class="mw-headline" id="Thursday_May_24th">Thursday May 24th</SPAN></H3><P align="justify"><UL><LI><B>Team Student Academy:</B></LI><LI> Made a liquid culture of <I>E. coli</I> KRX with <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A> at 30 °C. There was no fluorescence. 
</LI><LI> Transformation of <A href="http://partsregistry.org/Part:BBa_J23100" class="external text" rel="nofollow">BBa_J23100</A> into <I>E. coli</I> KRX. Also got intense red fluorescing colonies.
</LI></UL></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig. 1: Oxidation of ABTS by TVEL0 reporting the activity of TVEL0 (n=4). Measurement setup see text.</DIV></DIV></DIV><UL><LI><B>Team Activity Tests:</B></LI><LI>On our schedule today was testing our bought laccases (from now on called <A href="http://www.sigmaaldrich.com/catalog/product/sigma/53739?lang=de&amp;region=DE" class="external text" rel="nofollow">TVEL0</A>) and improving our protocol in case it won't work. We got familiar with the <A href="http://www.greinerbioone.com/UserFiles/File/PDFCatalogue/e_2_6_96_Well_Microplates.pdf" class="external text" rel="nofollow">flat bottom microplates</A> and got a briefing into using the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Tecan_Infinite_Microplate_Reader" class="external text" rel="nofollow">Tecan</A> correctly. We started with the following setup: 100 mM sodium acetate (pH 5), 5 mM ABTS, 8 U TVEL0 laccase, ad 200 µL with deionized water. Then we measured the absorption at 420 nm every 15 seconds over a time period of 2 minutes. Immediately our samples turned dark blue but unfortunately the changeover was out of range to be detected by Tecan. With this information we needed to reduce the laccase concentration to get measurable results. We have chosen to try another measurement with 0.1 U TVEL0 laccase and it worked! The result was a nice saturation curve but it reached a too high, because not good measurable, optimum within 1 minute (Fig. 1). To avoid the loss of important data in the beginning of the reaction and to reduce the saturation OD we decided to slow everything down by using less ABTS.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_May_25th">Friday May 25th</SPAN></H3><P align="justify"><UL><LI><B>Team Cloning of Bacterial Laccases</B>:
</LI></UL><DL><DD><UL><LI> We had to do the PCR on <I>T. thermo</I> laccase again because after the purification of the last PCR product the DNA amount was very low.
</LI></UL></DD></DL><UL><LI><B>Team Student Academy:</B></LI><LI> Made a liquid culture of <I>E. coli</I> KRX with <A href="http://partsregistry.org/Part:BBa_J23100" class="external text" rel="nofollow">BBa_J23100</A> at 30 °C. Result: There was no fluorescence.
</LI></UL><LI><B>Team Database:</B></LI><LI> Yeah we have a login!! 
</LI></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 2: Screen shot of our Login!!</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Saturday_May_26th">Saturday May 26th</SPAN></H3><H3><SPAN class="mw-headline" id="Sunday_May_27th">Sunday May 27th</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig. 3: Oxidation of 0.01 mM ABTS by TVEL0 reporting the activity of TVEL0 (n=24).</DIV></DIV></DIV><UL><LI><B>Team Activity Test:</B></LI><LI><P align="justify">Remember our first activity measurements? The activity was awesome and reached quickly its saturation. So today we decided to have a lazy Sunday and reduce absorption maximum and reaction speed, too. For this we used 0.1 mM ABTS instead of 5 mM ABTS and we got good results. The saturation reached its maximum at a OD<SUB>420</SUB> of ~1.6 after roundabout 2 minutes (see Fig. 2). Taking together we halved the maximal OD<SUB>420</SUB> and doubled the reaction time until maximum is reached with using 0.1 mM ABTS instead of 5 mM ABTS.</P></LI></UL></DIV><DIV id="anzeige"><H1>Summary of Week 5</H1><H2><SPAN class="mw-headline" id="Week_5_.2805.2F28_-_06.2F03.2F12.29">Week 5 (05/28 - 06/03/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_5_.2805.2F28_-_06.2F03.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 5 (05/28 - 06/03/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_May_28th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday May 28th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_May_29th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday May 29th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_May_30th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday May 30th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_May_31st"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday May 31st</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_June_1st"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday June 1st</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_June_2nd"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday June 2nd</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_June_3rd"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday June 3rd</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_5">Weekly Seminar</SPAN></H3><UL><LI> We found our next sponsor: <A href="http://www.analytik-jena.de/en/" class="external text" rel="nofollow">Analytik Jena AG</A>.
</LI><LI> all sub-teams have to present their results and problems next monday.
</LI><LI> Things to do in the lab:Site directed mutagenisis, cultivation, purification.
</LI><LI> Kevin will bring a white board for better communication in the lab.
</LI><LI> Julia presented the first screenshots of her database.
</LI><LI> Derya will be responsible for sequencing from now on.
</LI><LI> Collaborations: Nadine will contact <A href="https://2012.igem.org/Team:SDU-Denmark" class="external text" rel="nofollow">SDU-Denmark</A> and <A href="https://2012.igem.org/Team:TU_Darmstadt" class="external text" rel="nofollow">Darmstadt</A>.
</LI><LI><A href="http://www.cas.uni-muenchen.de/veranstaltungen/tag_synth_bio_2012/index.html" class="external text" rel="nofollow">CAS conference for Synthetic Biology</A> (23th - 25th of july) in Munich, Miriam will read up on travel and acccommodation expenses
</LI><LI><A href="http://www.biotechnologie2020plus.de/" class="external text" rel="nofollow">strategy process &quot;Biotechnologie2020+&quot;</A> in Berlin, Gabi, Timo and Robert will represent the team
</LI><LI> press release: last changes are in progress
</LI></UL><H3><SPAN class="mw-headline" id="Monday_May_28th">Monday May 28th</SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI> Made liquid cultures of <I>E. coli</I> KRX with <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A> and with <A href="http://partsregistry.org/Part:BBa_J23100" class="external text" rel="nofollow">BBa_J23100</A>. Result: Intense red fluorescence. We made a glycerol stock  and a plasmid isolation
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Digest of ecol(T7)_HIS, xccl(T7)_His and bpul(T7)_His and <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A> (pSB1C3 + RFP) with the same restriction enzymes, ligation of the digested products and transformation in <I>E. coli</I> KRX cells.
</LI><LI> Since we have less bpul(T7)_His and tthl(T7)_His DNA, we set an PCR from the remaining PCR approach.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_May_29th">Tuesday May 29th</SPAN></H3><P align="justify"><UL><LI><B>Team Student Academy:</B></LI><LI><I>E. coli</I> Mach1 with pMTE cp46 His received from the working group “Fermentation Engineering”, University Bielefeld. Plasmid contains genes for GFP and kanamycine resistance. We plated it and made a liquid culture at 37°C. Result: There was an intense fluorescence. We made a glycerol stock  and a plasmid isolation.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> We did colony PCRs on the transformations from the day before. We got product for every transformation approach so we plated the positive colonies on new plates to make plasmid isolation. So hopefully in some days we have the plasmids with the <I>E. coli</I> laccase, the <I>Xanthomonas campestris</I> laccase and the <I>B. pumilus</I> laccase with the inducible T7 promoter and a His-tag.
</LI></UL></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 1:Saturation curves of TVEL0 activity with 0.1 mM and 0.05 mM ABTS (n=24).</DIV></DIV></DIV><UL><LI><B>Team Activity Tests:</B></LI><LI>After establishing our recipe for activity measurements we were curious about different ABTS concentrations and wanted to make sure we took the right one for our approach. With this in mind we did our activity measurement with 0.1 U <A href="http://www.sigmaaldrich.com/catalog/product/sigma/53739?lang=de&amp;region=DE" class="external text" rel="nofollow">TVEL0 laccase</A>, 100 mM sodium acetate buffer (pH 5), ad 200 µl H<SUB>2</SUB>O dest., but with two new ABTS concentrations, namely 0.2 mM and 0.05 mM ABTS (see Fig. 1). It turned out that using 0.05 mM ABTS leads to a low maximum of saturation but reaches it quickly. With 0.2 mM ABTS the opposite occurs: the activity curve is saturated at a OD<SUB>420</SUB> of ~2.7 but it needs more time to reach its maximum. Knowing this we are happy using 0.1 mM ABTS because it is saturating slowly and the maximum is not too high. 0.1 mM ABTS is therefore established!</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_May_30th">Wednesday May 30th</SPAN></H3><P align="justify"><UL><LI><B>Team Cloning Bacterial Laccases:</B></LI></UL><DL><DD><UL><LI> After plasmid isolation we digested our plasmids with <I>Not</I>I to see if the colony PCR was correct and our laccases are in the backbone. The agarose gel showed that for all of the different plasmids we had at least one plasmid with two DNA bands on the correct height in agarose gel.
</LI></UL></DD></DL></P><H3><SPAN class="mw-headline" id="Thursday_May_31st">Thursday May 31st</SPAN></H3><UL><LI><B>Team Cloning of Bacterial laccase</B>:
</LI><LI> We sent the isolated pSB1C3 plasmids with  xccl(T7)_His, Bpul(T7)_HIS and Ecol(T7)_His for sequencing.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_June_1st">Friday June 1st</SPAN></H3><UL><LI><B>Team Activity Test</B>: 
</LI><LI> Today we prepared for our next measurements by setting up the sodium acetate buffer in different pHs. We choose to test the activity of TVEL0 in a pH-Gradient of 1,2,5,7 and 9.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_June_2nd">Saturday June 2nd</SPAN></H3><H3><SPAN class="mw-headline" id="Sunday_June_3rd">Sunday June 3rd</SPAN></H3><P align="justify"><UL><LI><B>Team Cloning of Bacterial Laccases</B>: 
</LI><LI> PCRs of genomic DNA on bhal from <I>B. halodurans C-125</I> we ordered from <A href="http://www.dsmz.de/" class="external text" rel="nofollow">DSMZ</A> before. We handled the cells in the same way we did with <I>T. thermophilus</I> before and soluted the lyophlized cells in water and boiled them before PCR. After PCR we cleaned up the product with gel electrophoresis and <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. However the DNA amount was so low that we had to do the PCRs again.
</LI></UL></P><DIV id="anzeige"><H1>Summary of Week 6</H1><H2><SPAN class="mw-headline" id="Week_6_.2806.2F04_-_06.2F10.2F12.29">Week 6 (06/04 - 06/10/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_6_.2806.2F04_-_06.2F10.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 6 (06/04 - 06/10/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_June_4th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday June 4th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_June_5th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday June 5th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_June_6th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday June 6th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_June_7th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday June 7th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_June_8th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday June 8th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_June_9th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday June 9th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_June_10th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday June 10th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_6"> Weekly Seminar </SPAN></H3><UL><LI> As part of the sponsoring with <A href="http://www.merckgroup.com/en/index.html" class="external text" rel="nofollow">Merck</A> we will present our project in Darmstadt. Kevin, Nadine and Gabi travel to Darmstadt on 21th of june.
</LI></UL><P>The team will gather on 14th, 15th and 18th of june to practice the presentation.
</P><UL><LI>  Participants at the <A href="http://www.cas.uni-muenchen.de/veranstaltungen/tag_synth_bio_2012/index.html" class="external text" rel="nofollow">CAS conference for Synthetic Biology</A>: Hakan, Derya, Miriam, Julia S., Gabi, Malak, Timo, Robert, Isabel, Nils.
</LI><LI> Robert is responsible for ordering the following BioBricks from the Partsregistry:
</LI></UL><P>&lt;partinfo&gt;K500000&lt;/partinfo&gt;&lt;partinfo&gt;K500001&lt;/partinfo&gt;&lt;partinfo&gt;K500002&lt;/partinfo&gt;&lt;partinfo&gt;K500003&lt;/partinfo&gt;&lt;partinfo&gt;K392014&lt;/partinfo&gt;</P><UL><LI> The treaty with our sponsor <A href="http://www.biocircle.com/en-ca/" class="external text" rel="nofollow">BioCircle</A> is signed.
</LI><LI> All teams presented their lab results.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_June_4th">Monday June 4th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>: 
</LI></UL><DL><DD><UL><LI> The Bacteria <I>S. griseus</I> and <I>S. lavendulae</I> has been delivered so we can start with PCRs. We set the first PCR with them as followed:
</LI><LI><B>PCR table</B></LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TH> Material </TH><TH> Volume
</TH></TR><TR><TD> Buffer (10x Phusion) </TD><TD> 10µL
</TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5µL
</TD></TR><TR><TD> dNTPs </TD><TD> 1µL
</TD></TR><TR><TD> Primer Mix </TD><TD> 1µL
</TD></TR><TR><TD> Template DNA </TD><TD> 1µL
</TD></TR><TR><TD> DMSO </TD><TD> 1,5µL
</TD></TR><TR><TD> Water </TD><TD> 35µL
</TD></TR></TBODY></TABLE><UL><LI><B> PCR program</B></LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Temperature </TH><TH> Time
</TH></TR><TR><TD> 1) 98°C </TD><TD> 7 mins
</TD></TR><TR><TD> 2) 98°C </TD><TD> 20 sec
</TD></TR><TR><TD> 3) 55°C </TD><TD> 20 sec
</TD></TR><TR><TD> 4) 72°C </TD><TD> 1 min
</TD></TR><TR><TD> 5) 72°C </TD><TD> 3 min
</TD></TR><TR><TD> 6) 12°C </TD></TR></TBODY></TABLE><P>Cycle between step 2 and 4 35 times.
</P><UL><LI><B>Team Fungal and Plant Laccases: </B></LI><LI>Primerdesign for isolating a laccase from <I>Arababidopsis thaliana</I> cDNA. Since we want to express the laccase in E.coli we designed the primers like before for the bacterial laccases with T7 promotor and HIS-tag.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_June_5th">Tuesday June 5th</SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI> Transformation of a plasmid mixture of either pMTE cp46 His and <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A> or pMTE cp46 His and <A href="http://partsregistry.org/Part:BBa_J23100" class="external text" rel="nofollow">BBa_J23100</A>. We plated both on LB agar without antibiotics and with Kanamycin. The first one was also plated on LB agar with ampicillin and the second on LB agar with chloramphenicol. Result: Works as expected. <A href="http://partsregistry.org/Part:BBa_J23100" class="external text" rel="nofollow">BBa_J23100</A> has a more intense fluorescence and was chosen for the experiment.
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Figure 1:Activity measurements of the bought laccase from <I>T.versiolor</I> analyzed through the OD<SUB>420</SUB> of oxidized ABTS in sodium acetate buffer at different pHs depending on time. Values are calculated by taking the average and standard deviation out of 4 measurements (n=4).</DIV></DIV></DIV><UL><LI><B>Team Cloning of Bacterial Laccases</B>: 
</LI><LI> The sequencing results for isolated plasmids xccl(T7)_His, bpul(T7)_His and ecol(T7)_His came. The results showed that only the xccl(T7)_His was ok – our first finished biobrick *yeha*. We proudly name it &lt;partinfo&gt;BBa_K863015&lt;/partinfo&gt;. The sequence of 'ecol(T7)_His showed that there are missing 4 bases in the promoter region  and the bpul(T7)_His sequence showed a mutation which leads to another amino acid in protein sequence.
</LI><LI> Again we did PCRs on <I>T. thermophilus</I> laccase and <I>B. halodurans</I> laccase with B.halo_FW_T7 / B.halo_FW_HIS and T.thermo_LAC_FW_T7 / T.thermo_LAC_RV_HIS primers and purified the product,this time with enough material for a restriction.
</LI></UL><UL><LI><B>Team Activity Tests</B>: 
</LI><LI> After testing the <I>T. versicolor</I> laccase under conditions that are optimal (pH 5, 25°C ) according to the literature we now started further characterization under different pHs. We analyzed the laccases behavior when working in 100 mM sodium acetate buffer at pH 1, 3, 7 an 9. Result: We agree with the literature that pH 5 seems to make the laccase happy. Since not all waste waters (especially those here in Germany) are not as warm as 25°C we now wonder what our laccase might do when exposed to lower temperaturers. Stay tuned.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_June_6th">Wednesday June 6th</SPAN></H3><UL><LI><B>Team Wiki:</B> Yay for Team Wiki´s first entry. Our first steps with the iGEM Bielefeld 2012 Wiki contain thinking about contents, layouts, programming and responsibilities. Our first rules are: 
</LI><LI> we are programming static pages in HTML and all the other pages (those that will be updated by all team members) in wiki code.
</LI><LI> we created all pages and will fill them up with some nice and beautiful content constantly from now on.
</LI><LI> Our temporary banner contains our outstanding logo and a DNA but we will set up a new layout soon.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases:</B> Digest of tthl(T7)_His and bhal(T7)_His PCR products and ligation in pSB1C3 backbone. After that we transformed the plasmids in competent <I>E. coli</I> KRX cells.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_June_7th">Thursday June 7th</SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI> Repeating of Transformation of 06/05 to verify the function. It is reproducible :)
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases</B>: Colony PCRs of the transformed colonies from yesterday's transformation showed some positive PCR bands.
</LI></UL><UL><LI><B>Team Modeling</B>: becoming acquainted with matlab while reading the manual
</LI></UL><H3><SPAN class="mw-headline" id="Friday_June_8th">Friday June 8th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>We plated colonies for plasmid isolations on new plates and made a control restriction with <I>Not</I>I. The electrophoretic separation showed gel bands in the right height for the Tthl(T7)_His and with bhal(T7)_His.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_June_9th">Saturday June 9th</SPAN></H3><H3><SPAN class="mw-headline" id="Sunday_June_10th">Sunday June 10th</SPAN></H3><DIV id="anzeige"><H1>Summary of Week 7</H1><H2><SPAN class="mw-headline" id="Week_7_.2806.2F11_-_06.2F17.2F12.29">Week 7 (06/11 - 06/17/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_7_.2806.2F11_-_06.2F17.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 7 (06/11 - 06/17/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_June_11th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday June 11th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_June_12th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday June 12th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_June_13th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday June 13th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_June_14th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday June 14th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_June_15th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday June 15th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_June_16th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday June 16th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_June_17th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday June 17th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_7"> Weekly Seminar </SPAN></H3><UL><LI> Agatha and Saskia will perform a mRNA isolation from <I>Arabidopsis thaliana</I>.
</LI><LI> Everything is in progress. Check.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_June_11th">Monday June 11th</SPAN></H3><P align="justify"><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Prepared plasmids for sequencing. We sent another isolated plasmid with'ecol(T7)_His. Also tthl(t7)_His,  bahl(T7)_His and  bpul(T7)_His plasmids were ready for sequencing.
</LI></UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> We looked for the sequences of the laccase of <I>Trametes versicolor</I> in different databases. By searching the specific sequences we realized, that we have to pay special attention to the strains the sequnces came from. During our search we cannot find any published strain in the DMSZ or other straincollections. So we decided to write e-mails to different working groups all over the world to get the published strains.
</LI><LI><B>Team Immobilization:</B></LI><LI>Today, we took our first step: we started searching for publications which describe different methods for the immobilization of laccases, especially those from <I>Trametes versicolor</I> in order to figure out the best method that suits our laccases. Some of the papers are listed below: 
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/22398306" class="external text" rel="nofollow">Recent developments and applications of immobilized laccase.</A></LI><LI><A href="http://www.hindawi.com/journals/er/2011/725172/" class="external text" rel="nofollow">Enzyme-Catalyzed Oxidation of 17β-Estradiol Using Immobilized Laccase from Trametes versicolor</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/17917725" class="external text" rel="nofollow">Reactive blue 19 decolouration by laccase immobilized on silica beads</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/16988786" class="external text" rel="nofollow">Alginate/carbon composite beads for laccase and glucose oxidase encapsulation: application in biofuel cell technology.</A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S1381117709002045" class="external text" rel="nofollow">A disposable Laccase–Tyrosinase based biosensor for amperometric detection of phenolic compounds in must and wine</A></LI></P><H3><SPAN class="mw-headline" id="Tuesday_June_12th">Tuesday June 12th</SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI> The whole experiment was tested by another team member to plan the course.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_June_13th">Wednesday June 13th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>: Since the GC amount of the <I>S. griseus</I> and <I>S. lavendulae</I> laccases are high we used betain to solve the PCR problem. Addition of betain did not change anything on the results, we still didn't got our laccase DNA.
</LI></UL><UL><LI><B>Team Modeling</B>: Programming our first differential equation and finding the ODE15s function witch solves these equations.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_June_14th">Thursday June 14th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>: 
</LI><LI> Because our PCRs have not worked well we thought it may depends on the primer annealing temperature so we did gradient PCR with the same conditions as before (<A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week6" class="external text" rel="nofollow">PCR June 4th</A>). But this also showed no result. Because we made Coloyn PCRs from the arrived DSMZ reaction tubes our next idea was to cultivate the bacteria in media and isolate genomic DNA.
</LI></UL><UL><LI><B>Team Activity Tests</B>: 
</LI><LI> Since our Tecan microplate reader is not able to actively cool down to 4 °C we got the chance to meet the photometer Carry. Check &quot;protocols&quot; for further information about her. We used the same set up with 100 mM natrium acetate buffer, 0,1 U <I>T. versicolor</I> laccase and 0,1 mM ABTS as before but now measured at 4°C.  Our team is planning to visit a municipal sewage plant for getting some insights into the water conditions there, so we will for sure test other temperatures after having more information. Let´s hope the water there is a little warmer since laccase does not seem to be totally satisfied at 4°C. I would not either.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_June_15th">Friday June 15th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>: 
</LI><LI> Sequencing of the pSB1C3 plasmid with bhal(T7)_His was ok. In conflict to our reference sequence there was a point mutation in the DNA sequence but this mutation doesn’t lead to another amino acid. So..next BioBrick (&lt;partinfo&gt;BBa_K863020&lt;/partinfo&gt;) is ready to use!
</LI><LI> The sequenced plasmid bpul(T7)_His showed again the same mutation in the laccase ORF compared to the reference sequence. We concluded that probably the PCR amplification caused the point mutation. So we did the digest  of bpul(T7)_His PCR products from a new PCR, ligated it in pSB1C3 backbone and transformed it in competent KRX cells. Additionally we did the digest tthl(T7)_His and the ligation in pSB1C3 backbone again.
</LI></UL><LI><B>Team Fungal and Plant Laccases</B>:
</LI><LI> The whole week we were looking for different sources of the laccase sequences and the corresponding strain, we find an interessting study of the Federal Environmental Agency (Umwelt-Bundes-Amt Germany). In this study they analysed the oppurtunities to synthesis different chemical compounds with the aid of different laccase of <I>Tramtetes versicolor</I>. In this document we found a working group from the Institute of Biochemistry, Dept. of Biotechnology &amp; Enzyme Catalysis at Ernst-Moritz-Arndt-University in Greifswald (Germany) who have isolated four sequences of different laccases of <I>Trametes versicolor</I> and the sequence of one <I>Pycnoporus cinnabarinus</I>. We have send a request for the plasmids containing cDNA sequences of five different laccases from <I>Trametes versicolor</I> and <I>Pycnoporus cinnabarinus</I> .
</LI><UL><LI><B>Team Database</B>: We finished our first tabel, you can add a new entry, delete it and edit it. 
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Figure 1: Screen shot of our Plates table.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Saturday_June_16th">Saturday June 16th</SPAN></H3><H3><SPAN class="mw-headline" id="Sunday_June_17th">Sunday June 17th</SPAN></H3><DIV id="anzeige"><H1>Summary of Week 8</H1><H2><SPAN class="mw-headline" id="Week_8_.2806.2F18_-_06.2F24.2F12.29">Week 8 (06/18 - 06/24/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_8_.2806.2F18_-_06.2F24.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 8 (06/18 - 06/24/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_June_18th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday June 18th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_June_19th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday June 19th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_June_20th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday June 20th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_June_21st"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday June 21st</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_June_22nd"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday June 22nd</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_June_23rd"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday June 23rd</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_June_24th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday June 24th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_8"> Weekly Seminar </SPAN></H3><UL><LI> Julia V. presents the database to have an easier overview over the labwork, the biobricks and as a digital lab diary.
</LI><LI> the last changes have been implemented into the press release, it is about time to let the world know about our project.
</LI><LI> Everyone please keep the iGEM deadlines in mind.
</LI><LI> Our advisor Timo presents: iGEM 101: Introduction to the wiki.
</LI><LI> Julia S is presenting her progesses in designing the <I>Pichia pastoris</I> shuttle vector.
</LI><LI> Modelling: Julia V. is thinking about how expression rates and promoter activity can be implemented into the model. Sebastian tries to model a sewage treatment plant.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_June_18th">Monday June 18th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>We started Colony PCRs on the colonies from June 15th  transformation and picked positive colonies to plate them for plasmid isolation. Sadly we just had   positive clones for tthl(T7)_His and not for bpul(T7)_His.
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Reviewed all Assembly-Standards and made a list of all illegal restriction-sites:
</LI><LI><I>Eco</I>RI, <I>Not</I>I, <I>Pst</I>I, <I>Spe</I>I, <I>Xba</I>I (Silver), <I>Age</I>I, <I>Ngo</I>MIV (Freiburg), <I>Bam</I>HI, <I>Bgl</I>II, <I>Xho</I>I (Berkeley)
</LI><LI> Decided to not care for restriction-sites of the Berkeley-assembly (even if it is a great assembly for protein-fusion), because the used Vector (pSB1C3) already has two <I>Xho</I>I-restriction-sites
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases</B></LI><LI> We have not received any reply to our e-mail. Team meeting to discuss the further way forward. We decided to send a second wave of e-mails to the different working groups, try to call the working groups(if possible) and to look for  strains of the corresponding and published sequences in additional straincollections.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_June_19th">Tuesday June 19th</SPAN></H3><UL><LI><B>Team Wiki</B>: 
</LI><LI> While using the lab journal more frequently there came up some questions. 
</LI><LI> How detailed do we plan to write our lab journal entries?
</LI><LI> Do we want to write in keywords or explain everything in full sentences?
</LI><LI> Do we want to note every little detail about every successful or unsuccessful experiment or just the main important aspect?
</LI><LI> We discussed, sighted some former iGEM team wikis and decided:
</LI><LI> each team is responsible for their own lab journal entries
</LI><LI> we divide our lab journal in weeks <SPAN style="text-decoration: underline;">and</SPAN> days to prevent it from looking too chaotic.
</LI><LI> the texts are supposed to state which team is writing, which experiment has been done and what the main aspects were. Also we will write about successful experiments, as well as problems and solutions we came up with. If possible links to protocols with further information shall be created.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_June_20th">Wednesday June 20th</SPAN></H3><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Imported sequences of most of the used bacterial Laccases into Clonemanager and analysed their restriction-sites:
</LI><LI> bhal has no illegal restriction-sites
</LI><LI> ecol has one <I>Ngo</I>MIV-restriction-site
</LI><LI> bpul has one <I>Xba</I>I-Restriction-site and one mutation two <I>Age</I>I- and two <I>Ngo</I>MIV-restriction-sites (Decided to delete the Freiburg-restriction-sites would take too much time)
</LI><LI> tthl has one <I>Pst</I>I-restriction-site and two <I>Ngo</I>MIV-restriction-sites (Decided to delete the Freiburg-restriction-sites would take too much time)
</LI><LI> xccl has two <I>Pst</I>I, one <I>Age</I>I and seven <I>Ngo</I>MIV-restriction-sites (Decided not to change the <I>Ngo</I>MIV-sites, since to mutate seven would take too much time)
</LI></UL><UL><LI><B>Team Modeling:</B></LI><LI> Finding out, that the &quot;normal&quot; Michaelis-Menten kinetic isn't the right kinetic to model our situation, because therefor you need a high and steady state concentration of the substrates. We have low concentrations and not really study state. We found a transformed equation.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_June_21st">Thursday June 21st</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> Plasmid isolation and control digest with <I>Not</I>I on tthl(T7)_His and luckily this time the bands were where they should be. Again and again we did transformation of ligation with bpul(T7)_His laccase in pSB1C3 backbone..all fingers are crossed that this time we have colonies with the correct plasmid.
</LI></UL><UL><LI><B>Team Activity Tests</B> and <B>Team Immobilization:</B></LI><LI> After all this characterizing we feel so much closer to our <I>T. versicolor</I> laccase that its about time to make some activity test under immobilized conditions. So now we are cooperating with Team Immobilization. We have thought about many ways how to immobilize the laccase and decided to give Silica Beads the first try. Check the Immobilization Team´s protocol for further information. Our main problem was how to measure the samples with all those beads in it. Tecan will probably be confused and give us some false values due to the beads that are disturbing its laser. So we need a way to get the beads out (and thus also stop the reaction) at a very precise point of time. Centrifugation wasn´t an option because it would simply take too long and not stop the reaction exactly in the second we want. While checking the internet for solutions we found Multi-Well Membrane-Bottom Filter Plates. Those are supposed to work in a similar way then our regular plates which we used for the Tecan but furthermore those plates contain a membrane that sieve the liquids through the filter when centrifugated. Thus the beads are separated and the ABTS-Buffer solution can me analyzed at 420 nm for oxidized ABTS. The plates will need a while before they arrive here at the CeBiTec, so we decided to first find out what the optimal amount of beads is and whether the beads might also bind ABTS (see lab journal Team Immobilization).
</LI></UL><UL><LI><B>Team Immobilization:</B></LI><LI>So after a lot of reading and discussion, we decided to try immobilization using beads. Since silica beads were already available in our lab (from last year’s team), we decided to give them a try. The first challenge was to find out a convenient ratio of beads/laccase. According to the protocol of last year’s team, the ratio 1:1000 was used (1000mg beads/ 1 mg protein). Therefore we decided to try the ratios 1:500, 1:1000 and 1:1500. We prepared different buffers: <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Immobilization" class="external text" rel="nofollow">HBSS buffer</A>, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Immobilization" class="external text" rel="nofollow">Recrystallization buffer</A> both of which were used with silica beads; in addition to <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Immobilization" class="external text" rel="nofollow">Britton-Robinson Buffer</A>, which was mentioned in publications as the best buffer for laccase immobilization. Laccases from TVEL0 were incubated with silica beads and different buffers at room temperature for 4hours on a rotator. After that, we collected the supernatants and delivered them to the team “Activity test” and waited for the good news.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_June_22nd">Friday June 22nd</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Because our PCR didn't work on the boiled lyophilized cells we used <A href="http://www.carlroth.com/catalogue/catalogue.do;jsessionid=2E5F0AF60BF1F28909D8475CF0053386?favOid=00000000000180e500070023&amp;act=showBookmark&amp;lang=de-de&amp;market=DE" class="external text" rel="nofollow">CASO Medium</A> for cultivation of <I>S. griseus</I> and <I>S. lavendulae</I>.
</LI></UL><LI><B>Team Immobilization:</B></LI><LI>Unfortunately, the activity test results weren’t promising. According to publications, immobilization via covalent binding is the most widely used method. Silica dioxide bead offer only an adsorption of laccases. Therefore, we decided to order CPC-(controlled pore carrier) silica beads, to which laccases covalently bind, especially that we found some papers with protocols and activity tests proving their efficiency.
</LI><H3><SPAN class="mw-headline" id="Saturday_June_23rd">Saturday June 23rd</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> The cultured <I>S. griseus</I> and <I>S. lavendulae</I> bacterials has been centrifuged at 13.000 rpm for 5 minutes. After this step we ribolyzed the pellet in 1 ml TE-Puffer and set a PCR reaction after. But we still haven't had any results.
</LI><LI> Colony PCRs on the transformations with plasmid with bpul(T7)_His and plating positive colonies.
</LI></UL><UL><LI><B>Team Database:</B></LI><LI> In the last wekk we finished the tabels Eppi, User, Sequencing and BioBrick. 
</LI><LI> We have a problem with the search function, we use the false SQL statement and allready deleted entries are shown. 
</LI><LI> In the next week we have to write a new statement.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_June_24th">Sunday June 24th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>:
</LI><LI> We isolated plasmids and did control digests with <I>Not</I>I. We finally had a positive restriction digest for bupl(T7)_His. So we prepared this plasmids and the plasmid tthl(T7)_His which we isolated some days before for sequencing.
</LI></UL><DIV id="anzeige"><H1>Summary of Week 9</H1><H2><SPAN class="mw-headline" id="Week_9_.2806.2F25_-_07.2F01.2F12.29">Week 9 (06/25 - 07/01/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_9_.2806.2F25_-_07.2F01.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 9 (06/25 - 07/01/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_June_25th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday June 25th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_June_26th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday June 26th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_June_27th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday June 27th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_June_28th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday June 28th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_June_29th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday June 29th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_June_30th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday June 30th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_July_1st"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday July 1st</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_9"> Weekly Seminar </SPAN></H3><UL><LI> evaluation of the presentation, we did in Darmstadt as part of our sponsoring through <A href="http://www.merckgroup.com/en/index.html" class="external text" rel="nofollow">Merck</A></LI><LI> Merck is strongly interested in our results, because the company has problems with lignin in the waste water (and laccases can be used for degradation of lignin, too).
</LI><LI> We were advised to use silica-beads, pretreated with carbodiimide, to immoblize our enzymes
</LI><LI> We decided to visit at least one sewage treatment plant as part of our Human Practices and we will try to integrate the treatment plants into our modelling.
</LI><LI> Everything is prepared for our summer school. Kevin and Gabi will present the experiment and 
</LI></UL><P>Nadine, Kevin, Gabi, Mo, Miriam, Derya and Isabel will advise the pupils.
</P><UL><LI> Gabi, Timo and Robert went to the <A href="/Team:Bielefeld-Germany/Human_Practices/StrategicProcess" title="Team:Bielefeld-Germany/Human Practices/StrategicProcess">strategy process</A><A href="http://www.biotechnologie2020plus.de/BIO2020/Navigation/DE/root,did=152728.html" class="external text" rel="nofollow">&quot;Biotechnologie2020+&quot;</A> in Berlin.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_June_25th">Monday June 25th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI></UL><DL><DD><UL><LI> Retried the DNA isolation from <I>S. griseus</I> and <I>S. lavendulae</I> without any success.
</LI></UL></DD></DL><UL><LI><B>Team Fungal and Plant Laccases</B>:
</LI><LI> Phonecall with the Leader of the working group of the <A href="http://biotech.uni-greifswald.de/" class="external text" rel="nofollow">University Greifswald Prof. Dr. U. Bornscheuer</A>. We explained our project and asked, if we can get the sequnces of the <I>Trametes versicolor</I> laccases. We got the commitment for getting the laccase sequences and plasmids containing the sequences (four laccases of <I>Trametes versicolor</I> and one of <I>Pycnoporus cinnabarinus</I>).
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_June_26th">Tuesday June 26th</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases: </B></LI><LI> The thing about plants is that they have to grow. Fortunately we got 6 beautiful 4 weeks-old wildtype plants from Patrick Treffon from the Institute of Plant Physiology and Biochemistry at Bielefeld University. With the help of the <A href="http://bar.utoronto.ca/efp/cgi-bin/efpWeb.cgi" class="external text" rel="nofollow">efp-Browser</A> we found out that the <A href="http://www.ncbi.nlm.nih.gov/protein/AAM77221.1" class="external text" rel="nofollow">laccase</A> in <I>A. thaliana</I> is only expressed in the developing seeds. So we now have to wait for the siliques to develop.
</LI></UL><UL><LI><B>Team Shuttle Vector:</B></LI><LI> Prepare the <A href="/Team:Bielefeld-Germany/Protocols/Materials#YPD_media" title="Team:Bielefeld-Germany/Protocols/Materials">YPD Media</A> for cultivation of the yeast strains <I>Pichia patoris</I> X-33 (wild type) and GS115 (Invitrogen). Both organisms are provided from the <A href="http://www.techfak.uni-bielefeld.de/ags/fermtech/" class="external text" rel="nofollow">chair of Fermentation Engineering</A> (D5) from Dr. Thomas Hug.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_June_27th">Wednesday June 27th</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Activity measurement of TVELO in sodium acetate buffer (pH5) and Briton Robinson buffer (pH 5). Measurements were taken via OD<SUB>420</SUB> of oxidized ABTS.</DIV></DIV></DIV><UL><LI><B>Team Shuttle Vector:</B></LI><LI> Cultivation of <I>Pichia pastoris</I> X-33 and GS115 in YPD media for isolation of the genomic DNA.
</LI></UL><UL><LI><B>Team Activity Tests</B>: 
</LI><LI> We like our new cooperation with Team Immobilization. The thing is, that they don´t like our buffer. Sodium acetate (pH 5) seems perfect for activity tests but apparently not for immobilization. What they prefer is a Britton-Robinson buffer (pH 5). To find out whether there is a difference between the two buffers that causes different activity habits of our laccase TVEL0, we setup comparable measurements with the two buffers and TVEL0. We concluded that the laccase in sodium acetate buffer shows a slower saturation but all in all both laccase samples reach the same maximum so that it is ok for us to use both buffer systems.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_June_28th">Thursday June 28th</SPAN></H3><UL><LI><B>Team Wiki</B>: 
</LI><LI> Today we browsed our wiki and were not very impressed: it's a lonesome place. So we started to think of how we could blow a little more life into it. For this, texts should appear soon on our wiki. To manage this bunch of work, we divided the subtopics of our wiki and appointed them to group members. Now everyone has a topic which he is responsible for. And that includes writing the texts, uploading pictures and keeping the represented information updated. Before anybody had the chance to disappear behind his/her notebook being busy editing his own page, we had to establish our <SPAN style="text-decoration: underline;"><B>wiki rules</B></SPAN>:
</LI><LI>Use the standardized formatting as presented in our <A href="https://2012.igem.org/Team:Bielefeld-Germany/Example" class="external text" rel="nofollow">example page</A>.
</LI><LI>Try to edit your text without using HTML code as far as possible and use wiki code instead. Useful advices when using wiki code are represented on our <A href="https://2012.igem.org/Team:Bielefeld-Germany/Example" class="external text" rel="nofollow">example page</A>, too.
</LI><LI>If you want to change anything that does not belong to your scope of duties, ask kindly the person of charge and make sure he/she is fine with it.
</LI><LI>Make your text more understandable by using images and charts. But remember: you are only allowed to upload pictures if you own them or if they are published without licenses.
</LI></UL><UL><LI><B>Team Shuttle Vector:</B></LI><LI> Isolation of genomic DNA from <I>Pichia pastoris</I> GS115 (Invitrogen) and the wild type X-33 with the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Genetics" class="external text" rel="nofollow">Wizward genomic DNA purification system Kit</A> (Promega).
</LI><LI><A href="/Team:Bielefeld-Germany/Protocols/molecular_genetics#Phusion_PCR" title="Team:Bielefeld-Germany/Protocols/molecular genetics">Phusion PCR</A> was done with the following primers to check that we created the right primers.
</LI></UL><TABLE class="wikitable" style="text-align: center; width: 300px: height: 300px;" border="0}"><TBODY><TR><TH> Used primers
</TH><TH> Template
</TH><TH> successful amplification
</TH></TR><TR><TD>5'AOX_FW and 5'AOX_RV
</TD><TD>pPICzA
</TD><TD>yes
</TD></TR><TR><TD>3'AOX_FW and 3'AOX_RV
</TD><TD>pPICzA
</TD><TD>no
</TD></TR><TR><TD>pAOX1_FW and pAOX1_RV
</TD><TD>pPICzA
</TD><TD>yes
</TD></TR><TR><TD>tAOX1_FW and tAOX1_RV
</TD><TD>X-33
</TD><TD>yes
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Friday_June_29th">Friday June 29th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>: 
</LI><LI> Sequencing results showed that even with a new PCR product the same mutation occurs in bpul(T7)_His so it is probably already present on the plasmid which was sent to us. So we decided to use this plasmid. We give him the name &lt;partinfo&gt;BBa_K863000&lt;/partinfo&gt;. Tth(T7)_His showed a positive sequencing result, too. So we have &lt;partinfo&gt;BBa_K863010&lt;/partinfo&gt; ready for use!
</LI></UL><UL><LI><B>Team Shuttle Vector</B>:
</LI><LI>Primer design was done for the shuttle vector. The following primers create fragments with a 5' overlap. Were ordered the primers: pSB1C3-5aox1-f, pSB1C3-5aox1-r, 5aox1-mfalpha1-f, 5aox1-mfalpha1-r, mfalpha1-aarI-taox1-f, mfalpha1-aarI-taox1-r, taox1-phis4-f, taox1-phis4-r, phis4-kozak-his4-f, phis4-kozak-his4-r, his4-3aox1-f, his4-3aox1-r, 3aox1-pSB1C3-f , 3aox1-pSB1C3-r.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_June_30th">Saturday June 30th</SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI> We prepared a script for pupils containing background information and a protocol and wrote an abstract for the school academy program.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_July_1st">Sunday July 1st</SPAN></H3><DIV id="anzeige"><H1>Summary of Week 10</H1><H2><SPAN class="mw-headline" id="Week_10_.2807.2F02_-_07.2F08.2F12.29">Week 10 (07/02 - 07/08/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_10_.2807.2F02_-_07.2F08.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 10 (07/02 - 07/08/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_July_2nd"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday July 2nd</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_July_3rd"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday July 3rd</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_July_4th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday July 4th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_July_5th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday July 5th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_July_6th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday July 6th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_July_7th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday July 7th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_July_8th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday July 8th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_10"> Weekly Seminar </SPAN></H3><UL><LI> Summer school: we will organize a barbecue and drinks, therefore some stuff has to be bought.
</LI><LI> We now have contact to the first sewage treatment plant.
</LI><LI> Malak will translate our abstract into english for the <A href="http://www.cas.uni-muenchen.de/veranstaltungen/tag_synth_bio_2012/index.html" class="external text" rel="nofollow">CAS conference</A> in munich.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_July_2nd">Monday July 2nd</SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI> During the whole week the two presentations for school academy were prepared and the barbecue for the pupils was organized. In the lab the last preparations were made.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Sending a request for  plasmids containing cDNA sequences of five different laccases from <I>Trametes versicolor</I> and <I>Pycnoporus cinnabarinus</I> from the Institute of Biochemistry, Dept. of Biotechnology &amp; Enzyme Catalysis at Ernst-Moritz-Arndt-University in Greifswald.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_July_3rd">Tuesday July 3rd</SPAN></H3><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Decided how to insert silent mutations to get rid of the restriction-sites with an eye on the codon-usage of host- and scource-oragnism, using <A href="http://www.kazusa.or.jp/codon/" class="external text" rel="nofollow">the Kazusa “Codon Usage Database”</A></LI><LI> ecol’s illegal <I>Ngo</I>MIV will be deleted by changing ggG to ggA (Glycin) at 2307
</LI><LI> tthl’s illegal <I>Pst</I>I will be deleted by changing caG to caA (Glutamine) at 2796
</LI><LI> bpul’s illegal <I>Xba</I>I will be deleted by changing  ctA to ctT (Leucin) at 2883
</LI><LI> bpul’s mutation will be deleted by changing Gag (Glutamat) to Aag (Lysin) at 2317
</LI><LI> xccl’s first illegal <I>Pst</I>I will be deleted by changing ctG to ctC (Valine) at 2247
</LI><LI> xccl’s second illegal <I>Pst</I>I will be deleted by changing ctG to ctC (Valine) at 3633
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_July_4th">Wednesday July 4th</SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI> Final meeting with all instructors of the student academy to talk about all details.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> After the failed attempts to isolate the laccase genes from <I>Streptomyces lavendulae</I> and <I>Streptomyces griseus</I> we assumed that maybe the sequences from this laccases and the laccase sequences we designed the primers for were to different to get a product <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week2#Monday_May_7th" class="external text" rel="nofollow">(look here for more details)</A>. So we used the sequences of the laccase genes we tried to isolate from the <I>Streptomycetes</I> strains and blasted them against an intern database from our university. The results showed that we could use genomic DNA from <I>Streptomyces sp. tuebingen</I>, <I>S.roseochromogenes</I> and <I>Streptomyces sp. goettingen</I>, from which we can get already isolated genomic DNA. Therefore we designed primers for the isolation of the predicted laccase genes from these three strains.
</LI></UL><UL><LI><B>Team Shuttle Vector:</B></LI><LI><A href="/Team:Bielefeld-Germany/Protocols/molecular_genetics#Phusion_PCR" title="Team:Bielefeld-Germany/Protocols/molecular genetics">Phusion PCR</A> was done with the following primers to see if the genomic DNA can used as template for the part 3OAX1. For creating the mating factor alpha 1 <A href="http://partsregistry.org/Part:BBa_K863206" class="external text" rel="nofollow">K863206</A> the plasmid pPICzA was used, because we did not have the wildtype strain of <I>Saccharomyces cereviceae</I>. For protein secretion we used the mating factor alpha 1 from <I>Sacharomyces sereviceae</I> because it secrete protein in the outer media better than the secretion factor from <I>P. pastoris</I> (LITERATURANGABE)
</LI></UL><TABLE class="wikitable" style="text-align: center; width: 300px: height: 300px;" border="0}"><TBODY><TR><TH> Used primers
</TH><TH> Template
</TH><TH> Successful amplification
</TH><TH> For BioBrick part
</TH></TR><TR><TD>3'AOX_FW and 3'AOX_RV
</TD><TD>X-33
</TD><TD>yes
</TD><TD><A href="http://partsregistry.org/Part:BBa_K863201" class="external text" rel="nofollow">K863201</A></TD></TR><TR><TD>3'AOX_FW and 3'AOX_RV
</TD><TD>GS115
</TD><TD>yes
</TD><TD><A href="http://partsregistry.org/Part:BBa_K863201" class="external text" rel="nofollow">K863201</A></TD></TR><TR><TD>Ko_alpha_FW and Ko_alpha_RV
</TD><TD>pPICzA
</TD><TD>yes
</TD><TD><A href="http://partsregistry.org/Part:BBa_K863206" class="external text" rel="nofollow">K863206</A></TD></TR><TR><TD>tAOX1_FW and tAOX1_RV
</TD><TD>X-33
</TD><TD>yes
</TD><TD><A href="http://partsregistry.org/Part:BBa_K863203" class="external text" rel="nofollow">K863203</A></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Thursday_July_5th">Thursday July 5th</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B></LI><LI> Some fragments for Gibson Assembly with overlapping DNA sequences were amplified via Phusion-PCR. The fragment 5'AOX1 was amplified with the primer pair pSB1C3-5aox1-f and pSB1C3-5aox1-r. The fragment MF-alpha1 was amplified with the primer pair 5aox1-mfalpha1-f and 5aox1-mfalpha1-r. The fragment 3'AOX1 was amplified with the primer pair his4-3aox1-f and his4-3aox1-r. The fragment pSB1C3 was amplified with the primer pair 3aox1-pSB1C3-f and 3aox1-pSB1C3-r.
</LI></UL><UL><LI><B>Team Shuttle Vector:</B></LI><LI><A href="/Team:Bielefeld-Germany/Protocols/molecular_genetics#Phusion_PCR" title="Team:Bielefeld-Germany/Protocols/molecular genetics">Phusion PCR</A> was done with the following primers to create fragments with 5' overlap for the Gibson assembly. Also a gel purification was done.
</LI></UL><TABLE class="wikitable" style="text-align: center; width: 300px: height: 300px;" border="0}"><TBODY><TR><TH> Fragment
</TH><TH> Used primers
</TH><TH> Template
</TH><TH> Successful amplification
</TH></TR><TR><TD>5AOX1
</TD><TD>pSB1C3-5aox1-f and pSB1C3-5aox1-r
</TD><TD>X-33
</TD><TD>yes
</TD></TR><TR><TD>Kozak + MFalpha1
</TD><TD>5aox1-mfalpha1-f and 5aox1-mfalpha1-r
</TD><TD>X-33
</TD><TD>yes
</TD></TR><TR><TD>3AOX1
</TD><TD>his4-3aox1-f and his4-3aox1-r
</TD><TD>X-33
</TD><TD>yes
</TD></TR><TR><TD>pSB1C3
</TD><TD>3aox1-pSB1C3-f and 3aox1-pSB1C3-r
</TD><TD>RFP::pSB1C3
</TD><TD>yes
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Friday_July_6th">Friday July 6th</SPAN></H3><UL><LI><B>Team Database</B>:
</LI><LI> FINISHED!! We finished our database. The search statement is now correct. And here it is:
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 1: Finally our saerch statement.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Saturday_July_7th">Saturday July 7th</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases</B>:
</LI><LI>We received an e-mail from the working group leader Prof. Dr. Bornscheuer (Institute of Biochemistry, Dept. of Biotechnology &amp; Enzyme Catalysis at Ernst-Moritz-Arndt-University in Greifswald (Germany)). He send us a confirmation that the plasmids with the cDNA of four <I>Trametes versicolor</I>-laccases and one laccase of  <I>Pycnoporus cinnabarinus</I> were sended to the iGEM Team Bielefeld.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_July_8th">Sunday July 8th</SPAN></H3><DIV id="anzeige"><H1>Summary of Week 11</H1><H2><SPAN class="mw-headline" id="Week_11_.2807.2F09_-_07.2F15.2F12.29">Week 11 (07/09 - 07/15/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_11_.2807.2F09_-_07.2F15.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 11 (07/09 - 07/15/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_July_9th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday July 9th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_July_10th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday July 10th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_July_11th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday July 11th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_July_12th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday July 12th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_July_13th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday July 13th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_July_14th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday July 14th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_July_15th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday July 15th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Monday_July_9th">Monday July 9th</SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI> This week the Student Academy took place. Today we held a presentation about the background of our experiments and answered all the questions the pupils had. Furthermore two of us participated at the an unconstrained meeting between the instructors and the pupils in the evening.
</LI></UL><UL><LI><B>Team Shuttle Vector:</B></LI><LI><A href="/Team:Bielefeld-Germany/Protocols/molecular_genetics#Phusion_PCR" title="Team:Bielefeld-Germany/Protocols/molecular genetics">Phusion PCR</A> was done with the following primers to create fragments with 5' overlap for Gibson assembly. Also a gel purification was done.
</LI></UL><TABLE class="wikitable" style="text-align: center; width: 300px: height: 300px;" border="0}"><TBODY><TR><TH> Fragment
</TH><TH> Used primers
</TH><TH> Template
</TH><TH> Successful amplification
</TH></TR><TR><TD><I>his4</I> promoter
</TD><TD>taox1-phis4-f and taox1-phis4-r
</TD><TD>X-33
</TD><TD>no
</TD></TR><TR><TD>Kozak + <I>his4</I></TD><TD>phis4-kozak-his4-f and phis4-kozak-his4-r
</TD><TD>X-33
</TD><TD>no
</TD></TR></TBODY></TABLE><UL><LI> Because no PCR fragments could be detected. The same PCR was running with 10 µL of 5M Betain in a sample of 50 µL and provide to the results, that no PCR fragment for the fragment pHIS4 could. But the fragment Kozak + <I>his4</I> with an 5' overlap could be detected.
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B> Council with Katharina Thiedig, who did the Site Directed Mutagenesis for the last years iGEM-Team-Bielefeld, about how to use the “QuikChange Primer Design”-program and the SDM-protocol
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Talked to Dominik Cholewa from the fermentation group of Bielfeld University about cellulose binding domains and got confirmed that they have some on plasmid and want to share them with us.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_July_10th">Tuesday July 10th</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B></LI></UL><P>Surprisingly, why the PCR for the fragment <I>his4</I> promoter does not raise the theoretical fragment with X33 as template, we check the DNA sequence we used for primer design. For design the primer taox1-phis4-f and taox1-phis4-r we used the DNA sequence from the BioBrick <A href="http://partsregistry.org/Part:BBa_K563000" class="external text" rel="nofollow">BBa_K563000</A>. But this part was designed on DNA from <I>Sacharomyces cereviceae</I>. So we thought that we can not use this DNA sequence in <I>Pichia pastoris</I>. Our concerns were that the promoter site could not be detected by the transcriptional regulatory machinery. We tried to identified the promoter region upstream the <I>his4</I> gene on the plasmid pPIC9K from Invitrogen. Some blasts later, we saw that the <I>his4</I> ORF from this plasmid is virtually the same as the CDS in <A href="http://www.ncbi.nlm.nih.gov/nuccore/537483" class="external text" rel="nofollow">NCBI</A>. Therefore, to create primers we used the DNA sequence from the plasmid pPIC9K and used the genomic DNA from the wild type <I>P. pastoris</I> X-33 or PCR amplification. Thus the primers taox-his4-f, taox-his4-r and his4-3aox1-f02 were new designed and ordered.
</P><UL><LI><B>Team Site Directed Mutagenesis:</B> Generated primer-sequences with Agilent Technologies “QuikChange Primer Design” for bpul, xccl, ecol and tthl and named the primers by their source-organism and place of mutation flanked on the left side by the original base and on the right by the mutated base.
</LI><LI>ecol-g2307a
</LI><LI>tthl-g2796a
</LI><LI>bpul-a2883t
</LI><LI>bpul-g2317t
</LI><LI>xccl-g2247c
</LI><LI>xccl-g3633c
</LI></UL><LI><B>Team Cellulose Binding Domain:</B> Searched NCBI for Cellulose, Chitin &amp; Keratin binding motifs in accessible organisms; found two Chitin-binding-domains in Bacillus halodurans
</LI><LI><B>Team Student Academy:</B></LI><LI> The experiments were performed by one half of the pupils in groups of 2-3. Afterwards we held a presentation about the iGEM competition, the project of the last two teams from Bielefeld and about our project. In the evening we had a barbecue with the pupils and the iGEM team with a lot of interesting discussions.
</LI><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> We got the sequences for five different fungal laccases from the requested plasmids from <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week10#Monday_July_2nd" class="external text" rel="nofollow">Monday 2nd</A> and they wanted to send us plasmids which contain the cDNA sequences of five different laccases from <I>Trametes versicolor</I> and <I>Pycnoporus cinnabarinus</I>.
</LI><LI> tvel5 from <I>Trametes versicolor</I></LI><LI> tvel10 from <I>Trametes versicolor</I></LI><LI> tvel13 from <I>Trametes versicolor</I></LI><LI> tvel20 from <I>Trametes versicolor</I></LI><LI> pcil35 from <I>Pycnoporus cinnabarinus</I></LI></UL><LI>We designed primer pairs with prefix and suffix overhanging ends for cloning in pSB1C3 and a primer pair for cloning in shuttle vector.
</LI><LI>The first primer pairs were designed with standard prefix and suffix sequence and 20 bases complementary to the start and end of the ORF sequences. 
</LI><LI> Additional primer pairs were designed with <I>Aar</I>I restriction site and Kozak consensus sequence before the first 20 bases from the start of the ORF (forward primers). The reverse primers were designed with the last 20 bases of the laccase genes and a terminator overhanging end and a <I>Aar</I>I restriction site.
</LI><H3><SPAN class="mw-headline" id="Wednesday_July_11th">Wednesday July 11th</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B> Looked the sequence of <A href="http://www.ncbi.nlm.nih.gov/nuccore/M73817" class="external text" rel="nofollow">Clostridium cellulovorans cellulose binding protein gene (cbp A)</A> up and made a Clonemanager-file. We used a protein-BLAST of the translated protein-sequence to find the location of the cellulose binding domain within the protein: We found it should be from base 103 to base 378 of the open reading frame.
</LI></UL><UL><LI><B>Team Student Academy:</B> The experiments were performed by the second half of the pupils and the groups from yesterday analyzed their plates.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases:</B> We made glycerin cultures of the finished BioBricks.
</LI></UL><UL><LI><B>Team Shuttle Vector:</B> The gel purified fragments 3AOX1 (<A href="http://partsregistry.org/Part:BBa_K863200" class="external text" rel="nofollow">K863200</A>), MFalpha1 (<A href="http://partsregistry.org/Part:BBa_K863206" class="external text" rel="nofollow">K863206</A>), pAOX1, tAOX1 (<A href="http://partsregistry.org/Part:BBa_K863203" class="external text" rel="nofollow">K863203</A>) and the plasmid pSB1C3 were double digested with <I>EcoR</I>I and <I>Spe</I>I as in the follow. Mix 10 µL of PCR amplification and pSB1C3, respectively to 18 µL bidest. water, add2 µL of 10x buffer O, 0,5 µL of each enzyme. After incubation for 1 h at 37 °C and inactivation at 80 °C for 20 min the plasmid was dephosphorylised with Shrimp Alkaline Phosphatase (SAP) by adding 1 µL SAP to the samples. The ligation was done with an equimolar ratio of plasmid to insert of 1:3 and at presence of 2 µL 10x ligase buffer and 1 µL of T4 ligase enzyme. After incubation at  room temperature for 1 h, the cells were transformed in <I>E. coli</I> KRX cells. But the transformation was not successful.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_July_12th">Thursday July 12th</SPAN></H3><UL><LI><B>Team Shuttle vector</B> PCR for more amplification product of 3AOX1 for the part shuttle vector (<A href="http://partsregistry.org/Part:BBa_K863204" class="external text" rel="nofollow">K863204</A>) and 5AOX1 for the part <A href="http://partsregistry.org/Part:BBa_K863200" class="external text" rel="nofollow">K863200</A> was done as described before.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_July_13th">Friday July 13th</SPAN></H3><UL><LI><B>Team Student Academy:</B></LI><LI> The groups from Wednesday analyzed their plates. Together with the pupils we made a final analysis of the results, discussed about all problems and questions and helped with the preparation of a presentation, they had to hold in front of all instructors and participants.
</LI></UL><UL><LI><B>Team Shuttle Vector&gt;</B> Phusion PCR of the fragment tAOX1 that contains upstream an bidirectional double <I>Aar</I>I restriction site was done with the wild type X-33 and GS115 as DNA template. The positive control was as ever positive, but no other amplicons.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases</B>:
</LI><LI> We got new <I>Streptomyces</I> DNA from or supervisor C. Rueckert. He did an alignment with our <I>S. griseus</I> and <I>S. lavendulae</I> sequences against a local database for Streptomyces and identified <I>S. rosechromogenes</I>, <I>S. tuebingen</I> and <I>S.goettingen</I>. Those laccase genes showed simularity to our laccase genes. Since he had isolated chromosomal DNA we were able to work with them. We set the PCR with the three <I>Streptomyces</I>. On this PCR <I>S. tuebingen</I> DNA could be amplified, but it wasn't specific for that what we expected. No products could be amplified on the other two.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_July_14th">Saturday July 14th</SPAN></H3><H3><SPAN class="mw-headline" id="Sunday_July_15th">Sunday July 15th</SPAN></H3><DIV id="anzeige"><H1>Summary of Week 12</H1><H2><SPAN class="mw-headline" id="Week_12_.2807.2F16_-_07.2F22.2F12.29">Week 12 (07/16 - 07/22/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_12_.2807.2F16_-_07.2F22.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 12 (07/16 - 07/22/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_July_16th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday July 16th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_July_17th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday July 17th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_July_18th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday July 18th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_July_19th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday July 19th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_July_20th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday July 20th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_July_21st"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday July 21st</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_July_22nd"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday July 22nd</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_11"> Weekly Seminar </SPAN></H3><UL><LI> For our sponsoring we sent all the treaties, to all our sponsors so far.
</LI><LI> Julia V. will organize a guided tour to a sewage treatment plant.
</LI><LI> The summer school was a great success and all the pupils appreciated our organization and the experiments.
</LI><LI> Julia V. will be responsible for the wiki design from now on.
</LI><LI> A big question appeared: How can we put our database on the wiki?
</LI><LI> We will take new team photos next week.
</LI><LI> For the safety/security everyone has to read the safety regulations regarding the organisms and chemicals.
</LI><LI> Next week there will be a short presentation about all used organisms and their laccases
</LI><LI> Agatha and Saskia will create templates for our webpages.
</LI><LI> Nadine will gather information about places to stay in Amsterdam and how to get there.
</LI><LI> Everyone has a passport?
</LI><LI> Gabi will present the judging criteria next week.
</LI><LI> Julia is looking for a BioBrick, which we could use and improve.
</LI><LI> Isabel created a poster for the conference in Munich, everyone has to review it.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_July_16th">Monday July 16th</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI>After we forgot to delete the signal peptide sequences, which are present in the fungal laccases we designed new forward primers for the laccases tvel5, tvel10, tvel13, tvel20 and pcil35 with the overhanging ends for cloning in our shuttle vector. <I>Trametes</I> laccases have a signal sequence after the start codon. This signal peptide we now delete by taking the first 20 bases after this sequence in our FW primers.
</LI><LI>Our plants had a great time during the last weeks in the climate chamber. So today it was time for them to donate their seeds for RNA isolation, cDNA synthesis and a PCR (check <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/molecular_genetics#Arabidopsis_thaliana:_Total_RNA_Isolation" class="external text" rel="nofollow">protocols</A>). We ran an additional sample with actin primers as a positive control. However both samples did not show any bands. Maybe the high salt concentration in our sample was responsible or the laccase concentration in the 1:10 diluted cDNA was too low. We will do some washing and try again.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> The two plasmids (p570 and p714) with the cellulose binding domains of <A href="http://www.ncbi.nlm.nih.gov/nucleotide/327179207?report=genbank&amp;log$=nucltop&amp;blast_rank=3&amp;RID=152ZCN0E01N" class="external text" rel="nofollow"><I>Cellulomonas fimi</I> ATCC 484 exoglucanase gene</A> and the <A href="http://www.ncbi.nlm.nih.gov/nuccore/M73817" class="external text" rel="nofollow">cellulose binding protein gene (cbp A)</A> from the fermentation group of Bielefeld University arrived.
</LI><LI><B>Team Cultivation &amp; Purification</B>: 
</LI><LI> We searched for some information for the best cultivation conditions in the internet. We found an interesting report of the <A href="http://www.dbu.de/OPAC/ab/DBU-Abschlussbericht-AZ-13191.pdf" class="external text" rel="nofollow">Deutsche Bundesstiftung Umwelt (DBU)</A> containing some interesting facts about different laccases as for example that the bacterial laccases are toxic to the bacterias, so that the production could be better under oxygen limitation and reduced temperature. Based on this article we decided to test flask with and without baffles and different temperatures.
</LI><LI> We prepared the basic media and solutions we will need in the lab.
</LI><LI> Note: All following BioBricks are cloned into pSB1C3 and therefore cultivated with 20 µg/mL chloramphenicol unless otherwise specified! Cultivations of <I>E. coli</I> KRX without plasmid will be performed without antibiotics.
</LI><H3><SPAN class="mw-headline" id="Tuesday_July_17th">Tuesday July 17th</SPAN></H3><UL><LI><B>Team Site Directed Mutagenesis:</B> Made Clone Manager files for the <I>Trametes versicolor</I> laccase plasmids we got send and analyzed them:
</LI><LI>tvel5: 
</LI><LI>one silent mutation (at 154 bp) and one amino acid alternating mutation at 227 bp (G for A, replacing D with N), a third was claimed to be at 1559 but sequencing showed it wasn’t. 
</LI><LI>No illegal restriction-site for the Silver-assembly, but three Freiburg-assembly restriction-sites: two <I>Ngo</I>MIV- and one <I>Age</I>I-restriction-sites 
</LI></UL><LI>tvel10: 
</LI><LI>eight silent mutations 171, 444, 1020, 1173, 1239, 1443, 1485 &amp; 1503 bp
</LI><LI>Two amino acid alternating mutations (1048 bp C for T, replacing F with L and 1078 bp A for G, replacing D with N)
</LI><LI>Two <I>Pst</I>I-restriction-sites (One in the signaling-sequence at 7 bp and one at 1160) and one <I>Spe</I>I-restriction-site at 241 bp
</LI><LI>One Freiburg-assembly restriction-site (<I>Age</I>I at 912 bp)
</LI><LI>tvel13: 
</LI><LI>56 silent mutations
</LI><LI>Three amino acid alternating mutations and one whole codon is missing at the very end
</LI><LI>One <I>Eco</I>RI- and one <I>Pst</I>I-restriction-site
</LI><LI>Two Freiburg-assembly restriction-sites (two <I>Ngo</I>MIV-sites) 
</LI><LI>tvel20: 
</LI><LI>about 32 silent mutations, three amino acid alternating mutations and four Freiburg-assembly restriction-sites (one <I>AgeI</I>, three <I>Ngo</I>MIV)
</LI><LI>pcil35: 
</LI><LI>no illegal restriction-site
</LI><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> We prepared our first precultures of <I>E. coli</I> KRX as negative control and <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> and <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A>.
</LI><LI> Note: We will cultivate <I>E. coli</I> KRX with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A> until we will get the laccase ORF with T7 promotor and His tag in the same pSB1C3 vector as the other BioBricks. To cultivations of <I>E. coli</I> KRX with pBpL6 we always will add 100 µg/mL ampicillin.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_July_18th">Wednesday July 18th</SPAN></H3><UL><LI><B>Team Site Directed Mutagenesis:</B> Made Primer-Mixes for the bacterial laccases. Set Pre-culture of XL1 blue. Got everything ready for Lab work.
</LI><LI><B>Team Cloning of Bacterial Laccases</B>: 
</LI></UL><DL><DD><UL><LI> We diluted our chromosomal DNA to a concentration of 20 ng/µL, since the volume we got was to low for doing many PCRs and did again a PCR reaction. This time <I>S. goettingen</I> laccase DNA could be identified but not <I>S. tuebingen</I>. The reaction conditions were the same so we were surprised because <I>S. tuebingen</I> didn`t work.
</LI></UL></DD></DL><UL><LI><B>Team Fungal and Plant Laccases</B>: Some of the ordered parts from the Parts Registry arrived and we plated the biobricks <A href="http://partsregistry.org/Part:BBa_K500000" class="external text" rel="nofollow">BBa_K500000</A>, <A href="http://partsregistry.org/Part:BBa_K500001" class="external text" rel="nofollow">BBa_K500001</A>, <A href="http://partsregistry.org/Part:BBa_K500002" class="external text" rel="nofollow">BBa_K500002</A>, <A href="http://partsregistry.org/Part:BBa_K500003" class="external text" rel="nofollow">BBa_K500003</A> and <A href="http://partsregistry.org/Part:BBa_K392014" class="external text" rel="nofollow">BBa_K392014</A>. Above all we are interested in <A href="http://partsregistry.org/Part:BBa_K500002" class="external text" rel="nofollow">BBa_K500002</A> because it’s a codon optimized laccase from <I>Trametes versicolor</I> and we want to use this laccase in our <I>P. pastoris</I> shuttle vector and characterize it.
</LI><LI><B>Team Cultivation &amp; Purification</B>: 
</LI><LI> Today we performed our first flask cultivation. We cultivated <I>E. coli</I> KRX with <A href="http://partsregistry.org/Part:BBa_K500005" class="external text" rel="nofollow">BBa_K500005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> and as negative control we used <I>E. coli</I> KRX. 
</LI><LI> Settings: We used 300 mL flasks without baffles, final volume: 60 mL, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>, 30/37 °C, durance: 24 hours 
</LI></UL><LI> Found out that we had a mixed culture of <I>E. coli</I> KRX with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A>, because it grew on chloramphenicol, but has only an ampicilline resistance. So we could not use this culture.
</LI><H3><SPAN class="mw-headline" id="Thursday_July_19th">Thursday July 19th</SPAN></H3><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Made electrocompetent XL1 blue cells. Made <I>pfu</I>-PCR of the bpul-plasmid with bpul-a2883t primer-mix, the ecol-plasmid with ecol-g2307a primer-mix, the xccl-plasmid with the xccl-g2247c primer-mix and tthl-plasmid with the tthl-g2796a primer-mix. Digested the template with <I>Dpn</I>I.
</LI><LI> Generated Primers for silent mutations of tvel10 illegal restriction-sites:
</LI><LI>the illegal <I>Spe</I>I will be deleted by changing acT  to ggA (Threonine) at 243 bp of the know sequence
</LI><LI>the illegal <I>Pst</I>I will be deleted by changing ccT to ccA (Proline) at 1161 bp of the known sequence
</LI><LI>one illegal restriction site in the signaling-sequence can not be mutated, since it is to close to the beginning of the known sequence (7 bp) and also it is not essencial, because the gene will be used without the signaling sequence
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Gathered some information about carbohydrate binding domain X2 (which is a more common domain in the organisms we handle than a cellulose Binding Domain).
</LI><LI>Used NCBI Nucleotid BLAST on &lt;partinfo&gt;BBa_K392014&lt;/partinfo&gt; (the Cellulose-binding motif from <I>C. josui</I> Xyn10A gene) and it 100% fits to 1526 bp to 2608bp of AB041993.1 (<I>Clostridium josui</I> xynA gene for xylanase A).
</LI><LI>Used NCBI Protein-BLAST on AB041993.1 and found one cellulose binding domain and one carbohydrate binding domain within the protein.:
</LI><LI>pfam02018: CBM_4_9 (Carbohydrate binding domain) from AS 193 to AS 332 - corresponds to 1088 bp - 1507 bp in the gene and is not in the coding sequence of &lt;partinfo&gt;BBa_K392014&lt;/partinfo&gt;.
</LI><LI>cd09619: CBM9_like_4 (DOMON-like type 9 carbohydrate binding module) from AS 716 to AS 887 - corresponds to 2657 bp to 3172 bp in the gene and is also not in coding sequence of &lt;partinfo&gt;K392014&lt;/partinfo&gt;.
</LI><LI>Given sequence of &lt;partinfo&gt;BBa_K392014&lt;/partinfo&gt; (1589bp to 2590bp) does corresponds to the sequence of the Glycosyl hydrolase 1589 bp  to 2590 bp and is not one of the binding motifs. The whole review can be read at the <A href="http://partsregistry.org/Part:BBa_K392014:Experience" class="external text" rel="nofollow">Experience-page of BBa_K392014</A>.
</LI><LI><B>Team Cloning of Bacterial Laccases:</B></LI><DL><DD><UL><LI><I>S. goettingen</I> and <I>S. tuebingen</I> DNA were cleaned up and we done an enzymatic digestion to ligate it into the pSB1C3 vector. We did the digestion with EcoRI and SpeI.
</LI></UL></DD></DL><UL><LI><B>Team Fungal and Plant Laccases:</B> Colonies from plated BioBricks from 18.07 were spread on nutrient agar for plasmid isolation.
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> The cultures of 07/18 were centrifugated, cells were disrupted via sonication in the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#special_buffers_for_resuspending_the_cell_pellet" class="external text" rel="nofollow">special buffer</A> for each laccase and after another centrifugation the supernatant was given to the activity test team.
</LI><LI> Made precultures analogous to them on 07/17.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_July_20th">Friday July 20th</SPAN></H3><UL><LI><B>Team Modeling:</B> We need a contact to a clarification plant to get information about clarification plant itself and perhaps to proof our cleaner with real probes. therefor  we are calling Mr. Bülter form the clarification plant Schloß Holte (near Bielefeld) and he invited us to present our project.
</LI><LI><B>Team Shuttle vector:</B> Gel purification of the fragments Kozak-MFalpha1 <A href="http://partsregistry.org/Part:BBa_K863206" class="external text" rel="nofollow">K863206</A> and 3'AOX1 <A href="http://partsregistry.org/Part:BBa_K863201" class="external text" rel="nofollow">K863201</A> was done.
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Transformation of XL1 Blue with the PCR products (see day before)
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Used Protein-BLAST on the translated sequence of the <I>Clostridium cellulovorans</I> cellulose binding protein, the <I>Bacillus halodurans</I> strain Cochin chitinase GU481106.1 and chitin-binding protein [<I>Bacillus halodurans C-125</I>]  BAB05022.1, to get more information about the predicted sequences of Carbohydrate binding domains.
</LI><LI>Looked up possible linkers in the Parts Registry:
</LI><LI>2 aa GS linker: <A href="http://partsregistry.org/Part:BBa_J18920" class="external text" rel="nofollow">BBa_J18920</A> [GS] 
</LI><LI>10 aa [GS]x linker: &lt;partinfo&gt;BBa_J18922&lt;/partinfo&gt; [GSGSGSGSGS]
</LI><LI>15 aa flexible glycine-serine protein domain linker; Freiburg standard 1 Star: &lt;partinfo&gt;BBa_K157013&lt;/partinfo&gt; [GGGGSGGGGSGGGGS]
</LI><LI>10 aa flexible protein domain linker 1 Star: &lt;partinfo&gt;BBa_K105012&lt;/partinfo&gt;: [GENLYFQSGG]
</LI><LI>Made a few possible primers for <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos</A> with T7 and RBS (&lt;partinfo&gt;BBa_K525998&lt;/partinfo&gt;), a Freiburg-Suffix and possible linkers 
</LI><LI>Wrote an E-Mail to Jun.Prof Thorsten Seidel (FRET-Expert of Bielefeld University) asking him about the linkers they use for fusing GFP for FRET.
</LI><LI><B>Team Fungal and Plant Laccases:</B> Isolating the plasmids <A href="http://partsregistry.org/Part:BBa_K500000" class="external text" rel="nofollow">BBa_K500000</A>, <A href="http://partsregistry.org/Part:BBa_K500001" class="external text" rel="nofollow">BBa_K500001</A>, <A href="http://partsregistry.org/Part:BBa_K500002" class="external text" rel="nofollow">BBa_K500002</A>, <A href="http://partsregistry.org/Part:BBa_K500003" class="external text" rel="nofollow">BBa_K500003</A> and <A href="http://partsregistry.org/Part:BBa_K392014" class="external text" rel="nofollow">BBa_K392014</A>. To be sure that in every plasmid contains the correct part we made a control restriction with <I>Not</I>I. It showed that all parts are on the correct hight in agarose gel.
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Another flask cultivation was made analogous to the one on 07/18 but at 23 °C.
</LI><H3><SPAN class="mw-headline" id="Saturday_July_21st">Saturday July 21st</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Made primers for <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> (using <A href="http://partsregistry.org/Part:BBa_I13522" class="external text" rel="nofollow">BBa_I13522</A> as template)adding a C-termial His6tag for easy extraction
</LI><LI>Jun.Prof Thorsten Seidel answered to the mail of yesterday and said that GFPs are easy to fuse and that they use linkers with four AS:
</LI><LI>ccg gtc gcc acc upstream of the GFP
</LI><LI>gaa agc ggc cgc downstream of the GFP; Which is will work fine, even with a proline in it's sequence.
</LI></UL><LI>We decided to use four linking AS also and chose to add a C-terminal Glycine-Serine-Linker (<A href="http://partsregistry.org/Part:BBa_J18920" class="external text" rel="nofollow">BBa_J18920</A>) to the cellulose binding domain, which would be four linking AS with the Freiburg-scar (Threonine-Glycine).
</LI><LI><B>Team Cloning of Bacterial Laccases</B>:
</LI><DL><DD><UL><LI> Since our PCRs didn't work and the <I>Streptomyces</I> laccases have more then 70% GC content we changed the dNTP concentration to 10 mM per base. Our gelelectrophoresis showed us that again <I>S. goettingen</I> has a product in correct length so we did again a PCR with same conditions. The rest of this product was cutted from an agarose gel and set ligation with the pSB1C3 vector and transformation. Because the isolated DNA concentration was to low the transformation showed no positive results.
</LI></UL></DD></DL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Culture from 07/20 was harvested and the cells were disrupted via sonication.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_July_22nd">Sunday July 22nd</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B></LI><LI>To get more amplicon from the part 5AOX1 <A href="http://partsregistry.org/Part:BBa_K863200" class="external text" rel="nofollow">K863200</A> an other Phusion PCR was done and the PCR product gel purified.
</LI><LI>To get the tAOX1 fragment for the shuttle vector, we run a PCR with the <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> KOD</A> DNA polymerase as descriped in the <A href="/Team:Bielefeld-Germany/Protocols/molecular_genetics#KOD_DNA_polymerase_reaction" title="Team:Bielefeld-Germany/Protocols/molecular genetics"> protocol</A>.
</LI><LI>While debugging and finding an solution for PCR fragment for the part tAOX1, test the annealing temperature of 60 °C that was proposed by the <A href="http://www.neb.com/nebecomm/tech_reference/tmcalc/default.asp#.UGNW-K-zZ5w" class="external text" rel="nofollow">Tm calculator from NEB</A>. Therefore, we run a temperature gradient from 50 °C until 67 °C with the usual protocol for Phusion DNA polymerase. But no specific product could be detected.
</LI><LI>Because there were also many problems to get a PCR fragment of the HIS4 fragment for the shuttle vector we run a temperature gradient from 55 °C until 60 °C with the usual Phusion DNA polymerase protocol. From 50 °C until 65 °C we could detect the right fragment. The highest yield was at 60 °C.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>We decided to use the exact sequence of <A href="http://partsregistry.org/Part:BBa_J18920" class="external text" rel="nofollow">BBa_J18920</A> (GGCAGC) of the linker in the partsregistry.
</LI><LI>Checked the CBDs we had access to for illegal restriction sites and found none, not even <I>Ngo</I>MIV or <I>Age</I>I. So we decided to use a Freiburg-Assembly to build the CBD-GFP fusion protein.
</LI></UL><DIV id="anzeige"><H1>Summary of Week 13</H1><H2><SPAN class="mw-headline" id="Week_13_.2807.2F23_-_07.2F29.2F12.29">Week 13 (07/23 - 07/29/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_13_.2807.2F23_-_07.2F29.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 13 (07/23 - 07/29/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_July_23rd"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday July 23rd</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_July_24th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday July 24th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_July_25th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday July 25th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_July_26th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday July 26th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_July_27th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday July 27th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_July_28th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday July 28th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_July_29th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday July 29th</SPAN></A></LI></UL><UL><LI> From 07/23 - 07/25/12, some of our team members participated at the CAS conference in Munich.
</LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_12"> Weekly Seminar </SPAN></H3><UL><LI> we will attend at the Biolympics it Münster, hosted by the <A href="http://bts-ev.de/" class="external text" rel="nofollow">bts</A> just to have some fun.
</LI></UL><UL><LI> We decided to write down every result in the labjournal, so that problems and their solution can be understood and reconstructed
</LI></UL><H3><SPAN class="mw-headline" id="Monday_July_23rd">Monday July 23rd</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Activity of ECOL, BHAL, BPUL and XCCL, measured in the supernatant of cells cultivated at 30°C via oxidized ABTS depending on time. <I>E.coli KRX</I> without plasmid functiones as a negative control. Values are calculated by taking the average out of 4 measurements (n=4).</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Activity of ECOL, BHAL, BPUL and XCCL, measured in the supernatant of cells cultivated at 37°C via oxidized ABTS depending on time. <I>E.coli KRX</I> without plasmid functiones as a negative control. Values are calculated by taking the average out of 4 measurements (n=4).</DIV></DIV></DIV><UL><LI><B>Team Activity Tests:</B> Today was the moment of truth. We received the first recombinant laccases from the cultivation team. We started with testing the supernatant from the cultivated cells to check whether they secrete laccase. We added 0,1 mM ABTS in each well that was filled with supernatant from E.coli KRX that contained the plasmid to produce ECOL, BPUL, BHAL, XCCL or <I>E. coli KRX</I> (negative control) and measured as usual. There was no change in the OD so that we assume that no secretion takes place.
</LI></UL><UL><LI><B>Team Shuttle vector:</B></LI><LI>Electroporation of RFP::pSB1C3 was done to get more plasmid DNA, but the wrong competent cells and selective plates were choosen.
</LI><LI>Phusion PCR of the tAOX1 part for the shuttle vector was done without betaine addition with the genomic DNA of <I>P. pastoris</I> GS115 and the wild type X-33. But only a small unspecific product was visible.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases:</B> The transformation from the 21st July showed no results. The plates were unfortunately empty. So we did an another PCR. This time the Annealing Temp. was set to 56°C.
</LI><LI><B>Team Fungal and Plant Laccases:</B> Successful PCRs of laccase gene Tv5 from <I>Trametes versicolor</I> with plasmid DNA from Uni Greifswald as template.
</LI><LI><B>Team Site Directed Mutagenesis:</B> Transfered 4 colonies of each transformation to an new dish for plasmid-isolation.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_July_24th">Tuesday July 24th</SPAN></H3><UL><LI><B> Team Shuttle Vector:</B> To get the fragment tAOX1 for the shuttle vector the Phusion PCR conditions were changed. A two-step PCR was done with the following conditions: After initial denaturation at 98 °C for 6 min, the first 10 cycles were run at: denaturation at 98 °C for 15 s, primer annealing at 55 °C for 30 s, elongation at 72 °C for 30 s. The following cycles were run at: denaturation at 98 °C for 15 s, primer annealing at 67 °C for 30 s, elongation at 72 °C for 30 s. The final elongation run at 72 °C for 5 min. But the PCR ghost was not satisfied and the right amplicons could not be detected. Just some unspecific small fragments could be detected in the geleletrophoresis.
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B> Plasmid-isolation of all 16 different colony-dishes
</LI></UL><UL><LI><B>Team Fungal and Plant Laccase</B>: Since our cDNA is now ready to go and also our primers already arrived we started a PCR. Check <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/molecular_genetics#PCR_for_A.thaliana_laccase_ampflification" class="external text" rel="nofollow">protocols</A> for more information about the exact setup. Unfortunately no bands could be seen via gel electrophoresis. We blamed the high salt concentration and decided that a some cleaning via ethanol precipitation might be a solution to our problem.
</LI></UL><UL><LI><B>Team Activity Tests</B>: After testing the supernatant of the cultivations yesterday we got to the more exciting part today. We tested the cell extract for laccase activity by filling each well with 140 µL cell extract, 40 µL buffer, 18 µL H<SUB>2</SUB>O and added 2 µL ABTS. Unfortunately no activity was seen. We felt like we could not take the laziness of our laccases. We thought and discussed what the reason could be for them to be inactive.
</LI></UL><CENTER><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fig. 1: Check for active <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> laccases in the cell extract of the last cultivation at 23°C, 30°C and 37°C As well compared</DIV></DIV></DIV></DIV><TABLE style="background-color:#eeeeeee;"><TBODY><TR><TD style="background-color:#eeeeeee;"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fig. 2: Check for active <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> laccases in the cell extract of the last cultivation at 23°C, 30°C and 37°C (n=4).</DIV></DIV></DIV></DIV></TD><TD style="background-color:#eeeeeee;"><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fig. 3: Check for active <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863020" class="external text" rel="nofollow">BHAL</A> laccases in the cell extract of the last cultivation at 23°C, 30°C and 37°C (n=4).</DIV></DIV></DIV></TD><TD style="background-color:#eeeeeee;"><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fig. 4: Check for active <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863015" class="external text" rel="nofollow">XCCL</A> laccases in the cell extract of the last cultivation at 23°C, 30°C and 37°C (n=4).</DIV></DIV></DIV></TD><TD style="background-color:#eeeeeee;"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fig. 5: Cell extract of the last cultivation of <I>E. coli KRX</I> without plasmid at 23°C, 30°C and 37°C as a negative control (n=4).</DIV></DIV></DIV></TD></TR></TBODY></TABLE></CENTER><UL><LI> Is there something wrong with the transformed construct, maybe a mutation?
</LI><LI> Was the laccase synthesized but is inactive for some reason?
</LI><LI> Was the laccase produced but not in a high enough amount so that we can´t detect its activity in the first place?
</LI></UL><P>Another thought was that there might be copper missing. Since laccase is a copper-enzyme it might be necessary to add copper to the medium whenever it is produced in such high amounts. It´s late, so we will do that tomorrow.
</P><H3><SPAN class="mw-headline" id="Wednesday_July_25th">Wednesday July 25th</SPAN></H3><UL><LI><B>Team Site Directed Mutagenesis:</B> Test-digestion (with the enzym of the mutated restriction-site) of the Plasmids from the 16 different colonies revealed that two bpul, one ecol and two tthl mutants have the right bands in the gel and seem be mutated correctly. Xccl-colonies are all unmutated (no Band at 3636).
</LI><LI><I>pfu</I>-PCRs with the two correct bpul-plasmids as template and the bpul-g2317t primer-mix
</LI><LI>two new <I>pfu</I>-PCRs with the xccl-plasmid one with the g2247c primer-mix and the other with the xccl-g3633c primer-mix
</LI><LI>Prepared all positive colones for sequencing
</LI></UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> We made precultures of <I>E. coli</I> KRX without plasmid and with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> , <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A> as well as with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A>.
</LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Cell extracts from cultivations were incubated with 2 mM copper sulfate for 2 h and measured for activity via OD<SUB>420</SUB>.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Cell extracts from cultivations were incubated with 4 mM copper sulfate for 2 h and measured for activity via OD<SUB>420</SUB>.</DIV></DIV></DIV><UL><LI><B>Team Activity Test</B>:  Today is copper day! First we divided each sample (the ones from yesterday) and added 2mM or 4mM copper sulfate to the halfs, respectively. We incubated our samples for 2h and hoped that the concentration gradient would cause the laccases to exchange the ion they have integrated instead against copper. We measured their activity again after those 2h but there was no change seen. Team Cultivation plans to add copper from the very beginning of cultivation next time.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_July_26th">Thursday July 26th</SPAN></H3><UL><LI><B>Team Modeling</B> and <B>Team Sponsoring</B>: meeting Mr. Bülter from the clarification plant of Schloß Holte and finding a new cooperation partner. He wants to give us the information we need to equate our model and design our cleaner. On top he wants to ask if the clarification plant can sponsor our project.
</LI><LI><B>Team Site Directed Mutagenesis:</B> Went on with the PCR products of xccl and bpul, transformed them into XL1 Blue and plated them on select-agar
</LI><LI><I>pfu</I>-PCRs with the tvel10-plasmid, one with the tvel10-243 primer-mix and one with the tvel10-1161 primer-mix. Both PCRs showed correct bands for the PCR-product in a test-agarose-gel-electrophoresis. Digested template with <I>Dpn</I>I.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B> We did PCR on <I>Trametes versicolor</I> laccase tvel5  and <I>Pycnoporus cinnabarinus</I> pcil35 with Tv_lac10.P.FW / Tv_lac10.S.RV and Pc_lac35.P.FW / Pc_lac35.S.RV primers. Additionally we want the laccases with overhanging ends for cloning in our shuttle vector for expression in <I>Pichia pastoris</I>. Therefore we used Pc_lac35_FW_oS / Pc_lac35_RV and Tv_lac5_FW_oS / Tv_lac5_RV <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Primers" class="external text" rel="nofollow">primer</A> pairs.
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Glycerine cultures were made of all given BioBricks, to use them for precultures.
</LI><LI> Made new precultures analogous to 07/25, as well as a preculture of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>.
</LI><LI> Found out, that it could be important to add CuCl<SUB>2</SUB> to the medium, because the copper is important for the active center of the laccases and to improve their stability.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_July_27th">Friday July 27th</SPAN></H3><UL><LI><B>Team Site Directed Mutagenesis:</B> plated four colonies of each transformation-dish (xccl &amp; bpul) on a petri dish for plasmid-isolation. Purified the digested PCR-products of tvel10.
</LI><LI><B>Team Cellulose Binding Domain:</B></LI><LI>We made the decision to only use cellolose binding domains (CBDs) and not carbohydrate binding domains of any kind to keep them comparable. This means we won't use the binding domains of <I>Bacillus halodurans</I>.
</LI><LI>Redesigned Primers for <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos</A> - added the bases TA in between RBS and ATG since the RBS can be made stronger by adding more adenines in the sequence upstream of the RBS.
</LI><LI>Made similar primers for <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex</A> (the CBD of the <I>Cellulomonas fimi</I> exoglucanase we got for the fermentation-group of Bielefeld University)
</LI><LI>Checked the Wiki of the iGEM 2012 Osaka University-Team about their &lt;partinfo&gt;BBa_K392014&lt;/partinfo&gt; BioBrick which should be a binding motif, but is the glycosyl hydrolase domain, for their intentions. We guess they really messed it up. It would have been nice to have a CBD from the partsregistry.
</LI></UL><LI><B>Team Fungal and Plant Laccases:</B> The PCR on tvel5 laccase was positive after the first trial for both primer pairs. For the tvel35 laccase the PCR didn’t work.
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> We cultivated <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A>,the new BioBrick <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A> as well as <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A>. As negative control we used <I>E. coli</I> KRX.
</LI><LI> Settings: 300 mL/ 1 L flasks without/without baffles, final volume: 60 mL/200 mL, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>, 0.25 mM CuCl<SUB>2</SUB>, 30 °C
</LI><LI> We made glycerine cultures of homologous culture of <I>E. coli</I> KRX with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A>.
</LI><H3><SPAN class="mw-headline" id="Saturday_July_28th">Saturday July 28th</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Activity Test of different Laccases cultivated with 0.25 mM CuCl. Cultivation has taken place using different methods. 'os' stands for a cultivation without chicanery, 'ms' for a cultivation with chicanery. Numbers (60, 200) indicate the volume of cultivation. No oxidation of ABTS trough our produced laccases could be measured.</DIV></DIV></DIV><UL><LI><B>Team Shuttle Vector:</B></LI><LI>Because the first Gibson assembly failed, more fragments with a higher concentration are necessary. All parts were amplificate by the same protocols as described.
</LI><LI>Also the Part HIS4 <A href="http://partsregistry.org/Part:BBa_K863202" class="external text" rel="nofollow">K863202</A> was amplified as described.
</LI><LI>And the Part tAOX1 <A href="http://partsregistry.org/Part:BBa_K863203" class="external text" rel="nofollow">K863203</A> was amplified by the same conditions as the fragment tAOX1 for the shuttle vector.
</LI></UL><UL><LI><B>Team Activity Tests:</B> See, we are back already! Today we received different samples from <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863022" class="external text" rel="nofollow">BHAL</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">XCCL</A> and <I>E. coli</I> KRX (negative control - cells without plasmid). The cells were all cultivated at 30°C and supplied with copper. We tested each one of them but none of them showed laccase activity.
</LI><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Purification of <I>Trametes versicolor</I> tvel5 PCR product.
</LI><LI> With the cDNA samples we got out of the siliques of <I>A. thaliana</I> we did two PCRs. After waiting a hour for the PCR to stop, we started a gel, hoping for really thick bands. But nonetheless there were non. Also the PCR with actin primers didn't give a result. A possible reason for this could have been contaminations like salts in our samples. Before translating the mRNA into cDNA we measured the following properties: 260/280: 1.93 and 260/230: 1.28. This indicates a high salt concentration which could be interfering during the PCR. For the next PCR we want to wash our cDNA and hope for better results with a clean and pretty sample.
</LI></UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Cells of cultivation 07/27 were disrupted via sonication and given to the activity test team.
</LI><H3><SPAN class="mw-headline" id="Sunday_July_29th">Sunday July 29th</SPAN></H3><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Preparation of all the stuff needed for SDS-Pages.
</LI><LI> Made new precultures of <I>E. coli</I> KRX without plasmid and with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A>,<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, as well as <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A> and our new BioBrick <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>.
</LI></UL><DIV id="anzeige"><H1>Summary of Week 14</H1><H2><SPAN class="mw-headline" id="Week_14_.2807.2F30_-_08.2F05.2F12.29">Week 14 (07/30 - 08/05/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_14_.2807.2F30_-_08.2F05.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 14 (07/30 - 08/05/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_July_30th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday July 30th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_July_31st"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday July 31st</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_August_1st"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday August 1st</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_August_2nd"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday August 2nd</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_August_3rd"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday August 3rd</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_August_4th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday August 4th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_August_5th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday August 5th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_13"> Weekly Seminar </SPAN></H3><UL><LI> From now on, there will be two weekly meetings: monday, 12.15 and thursday 9.00.
</LI><LI> Nadine sent all of us a poll, answers will be used on the wiki.
</LI><LI> For picture uploads we will use the name: Bielefeld2012_x.
</LI><LI> Which sewage treatment plant uses activated carbon in the third stage of treatment?
</LI><LI> On 25th of august, all german teams will host the <A href="/Team:Bielefeld-Germany/StreetScience" title="Team:Bielefeld-Germany/StreetScience">SynBioDay</A>. In Bielefeld we will cooperate with <A href="http://www.bielefeld-marketing.de/de/index.html" class="external text" rel="nofollow">Bielefeld Marketing</A> and to raise public interest we will call it StreetScience. As part of our collaboration we will invite the iGEM team from <A href="https://2012.igem.org/Team:SDU-Denmark" class="external text" rel="nofollow">SDU-Denmark</A> to help us with the organization and the realization of this common day.
</LI><LI> On StreetScience we will have a little experiment about extracting fruit DNA with an easy method.
</LI><LI> Do we want to travel to Amsterdam by bus or train?
</LI><LI> Do we want a ship as our accommodation in Amsterdam?
</LI><LI> Gabi has presented the judging criteria including the bronze, silver and gold criteria as well as the special prices.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_July_30th">Monday July 30th</SPAN></H3><UL><LI><B>Cloning of Bacterial Laccases</B>:
</LI><LI> The PCR products from the 23th July was digested with EcoRI and SpeI same as the pSB1C3 vector. Then the ligation was set and transformed into competent <I>E. coli KRX</I> cells. 
</LI><LI> Since we haven't got any more PCR products we set a PCR. Only <I>S. goettingen</I> worked. 
</LI></UL><LI><B>Team Fungal and Plant Laccases:</B> Digestion of tvel5 PCR product with prefix and suffix ends for cloning in pSB1C3 and digest of laccase with the overhangs for cloning in shuttle vector. Additionally we did the PCR on tvel35 with both primer pairs again. This time we lowered the annealing temperatures and got products with both primer pairs.
</LI><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Redesigned the primer-sequences another time giving the CBDs a few additional amino acids from the by Protein-BLAST predicted domain. <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex</A>: 4 AS N-terminal, 2 C-terminal. <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos</A>: 2 N-terminal (starting with a natural ATG) and 2 AS C-terminal.
</LI><LI>Prepared  &lt;partinfo&gt;BBa_K392014&lt;/partinfo&gt; for Sequencing
</LI><UL><LI><B>Team Site Directed Mutagenesis:</B> Plasmid-isolations of all the bpul- and xccl-mutants.
</LI><LI><B>Team Cultivation &amp; Purification:</B><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 1:</B> Growth kinetics of cultivation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> and <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A>. Parameters: <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>, 0.25 mM CuCl<SUB>2</SUB>, 28 °C.</DIV></DIV></DIV></LI><LI> We made the SDS-Pages for the cultivation from 07/27, but they did not seem to be promising.
</LI><LI> We started another cultivation of <I>E.coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> and <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A>.
</LI><LI> Settings: 300 mL flasks without baffles, final volume: 60 mL, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>, 0.25 mM CuCl<SUB>2</SUB>, 28 °C.
</LI><LI> A growth kinetics was recorded every 45 minutes.
</LI></UL><LI> We decided that we also need a positive control for the next cultivations, to see if our autoinduction medium works. We chose <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>. Furthermore we will make cultivations in which we manually induce the expression after 4 hours.
</LI><LI> Made a preculture of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> and <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A> as well as with <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>. We used <I>E. coli</I> KRX as negative control.
</LI><H3><SPAN class="mw-headline" id="Tuesday_July_31st">Tuesday July 31st</SPAN></H3><UL><LI><B> Team Shuttle Vector</B></LI><LI> To get the tAOX1 fragment for the shuttle vector, we run a primer gradient from 0.1 µM to 1 µM in 0.1 µM steps with a primer annealing time of 15 s and 20 s and the usual Phusion PCR protocol. No fragments could be detected.
</LI><LI> A DNA template gradient from 10 ng to 250 ng in 50 ng steps was choosen. But no fragment could be detected. Also the primer annealing temperature was varied in 15 s and 20 s. But only small unspecific fragments with from 100 bp to 150 bp could be detected.
</LI></UL><LI><B>Team Cloning of Bacterial Laccases</B>:
</LI><LI> After our laccases are not produced or not detectable we decided to try to express the laccases with a constitutive promoter. Therefore we searched for appropropriate promoters and found different promoters from the <A href="http://partsregistry.org/Ribosome_Binding_Sites/Prokaryotic/Constitutive/Anderson" class="external text" rel="nofollow">Anderson promoter family</A>. We picked three different promoters with different promoter strengths. We chose the parts <A href="http://partsregistry.org/wiki/index.php/Part:BBa_J23103" class="external text" rel="nofollow">BBa_J23103</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_J23103" class="external text" rel="nofollow">BBa_J23110</A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_J23103" class="external text" rel="nofollow">BBa_J23117</A> and for all three promoters the RBS <A href="http://partsregistry.org/wiki/index.php/Part:BBa_J23103" class="external text" rel="nofollow">BBa_B0034</A>. The primers were designed that the FW and the RV primers anneal together to a short oligonucleotide with overhanging restriction site ends for <I>Eco</I>RI and <I>Spe</I>I. So we don’t have to cut the annealed primers, because the sites should appear with correct annealing of FW and RV primers. The goal is to clone the different promoters, the PCR products with the different laccase genes with His-Tag in pSB1C3 in one ligation step. Furthermore we want to exclude the possibility that the His-tag is the reason for no activity, so we also want to clone the gene sequences without a His-tag under control of a constitutive promoter in pSB1C3. 
</LI><LI>Additionally we want to produce the constructs with a new T7 promoter. After our laccases were not expressed we now think that maybe the RBS &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;, which we changed in our primers from originally 5' aAagaggagaaa 3' to 5' aGagaggagaaa 3' is not or poorly recognized from the ribosomes in the cells.
</LI><LI>We have a primer pair from last year with the T7 promoter sequence with the RBS BBa_B0034. The primers can be annealed to an oligonucleotide. After boiling the primers and cooling down there should be an oligonucleotide with an <I>Eco</I>RI and a <I>Pst</I>I restriction site. We now want to assemble the promoter the laccase genes and the pSB1C3 vector in one step. The pSB1C3 backbone was digested with <I>Eco</I>RI and <I>Pst</I>I, the laccases with <I>Xba</I>I and <I>Pst</I>I, and the promoter has the restriction sites <I>Spe</I>I and <I>Eco</I>RI (we have to digest the promoter with <I>Spe</I>I, because the original restriction site is <I>Pst</I>I).
</LI><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI>Purification of tvel35 PCR product and digestion for cloning in pSB1C3 backbone.
</LI><LI> Ligation of tvel5 laccase with pSB1C3 backbone.
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B> Plated four colonies per dish of tvel-t243g and tvel-t1161a for plasmid-isolation.
</LI><LI>Test-restriction of the xccl-mutants showed that no colony was mutated correctly
</LI><LI>The second site directed mutagenesis of bpul-g2317t could not be rated by test-digestion, since the mutation at 2317 is not because of a illegal restriction site, but a amino acid alternating mutation.
</LI><LI>Made four bpul-colonies (with both mutations) ready for sequencing
</LI><LI>plated a four more colonies of xccl (both SDM-sites) for plasmid-isolation
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Made SDS-Pages of cultivation from 07/30. <DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 2:</B> SDS-PAGES of the lysate, the supernatant, and with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Production#Solubilization_of_inclusion_bodies" class="external text" rel="nofollow">urea treated pellet</A> (cultivation 07/30) of <I>E.coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> or <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A>. 60 mL in 300 mL flasks without baffles with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>, 0.25 mM CuCl<SUB>2</SUB>, 28 °C</DIV></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 3:</B> Growth kinetics of cultivation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> and <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A>. Parameters: <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction</A> or manual induction, 0.25 mM CuCl<SUB>2</SUB>, 30 °C.</DIV></DIV></DIV></LI><LI> Cultivation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> and <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A> as well as with <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>. We used <I>E. coli</I> KRX as negative control.
</LI><LI> general settings: 300 mL flasks without baffles, final volume: 60 mL, 30 °C
</LI><LI> special settings: 
</LI></UL><DL><DD>a) <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> and <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A> were cultivated in <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#LB_medium" class="external text" rel="nofollow">LB media</A> and manually induced after 4 hours with 0.1 % rhamnose.
</DD><DD>b) <I>E. coli</I> KRX without plasmid was cultivated with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#LB_medium" class="external text" rel="nofollow">LB medium</A> and induced after 4 hours with 0.1 % rhamnose.
</DD><DD>c) <I>E. coli</I> KRX with <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A> was cultivated equally to b)
</DD><DD>d) <I>E. coli</I> KRX without plasmid was cultivated with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> and 20 µg/mL chloramphenicol as well as 100 µg/mL ampicillin as a control.
</DD></DL><H3><SPAN class="mw-headline" id="Wednesday_August_1st">Wednesday August 1st</SPAN></H3><UL><LI><B>Team Site Directed Mutagenesis:</B> Plasmid-isolation of tvel- and the xccl-colony-dishes.
</LI><LI>Test-restriction with showed correct bands for two tvel and one xccl-colony.
</LI><LI>Made the correct xccl-plasmid ready for sequencing.
</LI></UL><LI><B>Team Cloning of Bacterial Laccases</B>:
</LI><LI>With the new T7 promoter we started new assemblies. We first want to try the new promoter with ecol. So wedigested our laccase PCR products from ecol with and without HIS tag for suffix insertion with <I>Xba</I>I and <I>Pst</I>I, boiled and cooled down the joined T7 primer pairs and digested pSB1C3 backbone with <I>Eco</I>RI and <I>Pst</I>I. If the plan works, the parts anneal to our final plasmid. We did the ligation and transformation of the approach into competent <I>E. coli</I> KRX cells.
</LI><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Harvesting and cell disruption via sonication of yesterday's cultivation.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_August_2nd">Thursday August 2nd</SPAN></H3><UL><LI><B>Team Wiki:</B> This morning we met for taking a new picture of our group and individual pictures of everyone. Check out our beautiful team members <A href="https://2012.igem.org/Team:Bielefeld-Germany/Students" class="external text" rel="nofollow">here</A>.
</LI><LI><B>Team Cloning of Bacterial Laccases</B>:
</LI></UL><DL><DD><UL><LI> There were just red colonies on the plates from the transformation the day before. So we didn't do colony PCR and have to do the assembly again.
</LI><LI> Our Laccases genes from <I>S. goettingen</I>, <I>S. tuebingen</I> and <I>S. roseochromogenes</I> are around 2kb. The NEB-Phusion has a range of 1kb/30sec so we thought if we set the elongation time higher we might have products for <I>S. tuebingen</I>,<I>S. roseochromogene</I> so we did PCR. But the results showed that it doesn't depend on the elongation time from the Phusion because we haven't had any products.
</LI></UL></DD></DL><UL><LI><B>Team Fungal and Plant Laccases:</B> Ligation of tvel5 and pcil35 in pSB1C3.
</LI><LI><B>Team Site Directed Mutagenesis:</B><I>pfu</I>-PCRs; two with the positive tvel-t243g-plasmids as templates and the tvel-t1161a primer-mix and another <I>pfu</I>-PCR with the positve xccl-g2247c-plasmid and the xccl-g3633c primer-mix
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Made SDS-Pages of cultivation from 07/31.
</LI></UL><CENTER><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 4:</B> SDS-PAGES of the lysate, the supernatant, and with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Production#Solubilization_of_inclusion_bodies" class="external text" rel="nofollow">urea treated pellet</A> (cultivation 07/31) of <I>E.coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> and <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A> as well as with <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A> (positive control) and <I>E.coli</I> KRX (negative control). 60 mL in 300 mL flasks without baffles, 30 °C. mI = manual induction, -I = without induction, auto-I = <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">auto-induction</A>.</DIV></DIV></DIV></DIV></CENTER><H3><SPAN class="mw-headline" id="Friday_August_3rd">Friday August 3rd</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>: 
</LI></UL><DL><DD><UL><LI> Digest of ecol PCR products with and without HIS-Tag and ligation with pT7 in pSB1C3. Transformation of this ligations and transformation of <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A> for Team Cultivation. 
</LI><LI> We did an annealing temp. gradient PCR with the Streptomyces bacteria. But expect '<I>S.goettingen</I> neighter <I>S. tuebingen</I> not <I>S. rosechromogenes</I> did not work.
</LI></UL></DD></DL><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 5:</B> Oxidizing activity of Laccases produced on 07/30 measured with ABTS. 'I(-)' and I(+) indicate no or Inducion which took place, respectively. A increase in OD<SUB>420</SUB> couldn't be seen. (n=4)</DIV></DIV></DIV><UL><LI><B>Team Fungal and Plant Laccases:</B> Transformation of tvel5 and tvel35 in pSB1C3 backbone.
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI><I>Dpn</I>I-digestion of the <I>pfu</I>-PCR-products (from yesterday). Transformed tevl- and xccl-double-mutants into XL1 Blue and plated them on selection-agar.
</LI><LI>Sequencing results:
</LI><LI>Both tthl-g2796a-mutations were successful
</LI><LI>ecol-g2307a has additional mutations
</LI></UL><LI><B>Team Cellulose Binding Domain:</B> Transformed the plasmids p714 (with the CDBcex domain) and p570 (with the CBDclos domain) we got from the fermentation group of Bielefeld University in KRX an plated them on kanamycin-selection-agar.
</LI><LI><B>Team Activity Tests</B>: Our lovely Team Cultivation started new cultivations on 07/30 and we got the products. More precisely we got the supernatant of the lysis Team Cultivation did before. We tested 198 µL of each sample with 2 µL ABTS for the activity reaction, but sadly we couldn't detect anything (see Fig. 5).
</LI><H3><SPAN class="mw-headline" id="Saturday_August_4th">Saturday August 4th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>:
</LI></UL><DL><DD><UL><LI> We did the transformation of the ligations from August 1st again.
</LI></UL></DD></DL><UL><LI><B>Team Site Directed Mutagenesis:</B> Plated four double-mutant-colonies from each transformation (both tvel-mutants and the xccl-mutant on selection-agar for plasmid isolation.
</LI><LI><B>Team Cellulose Binding Domain:</B></LI><LI>The transformation of p570 brought up only a few colonies, so we took a few and plated them again on a selection-agar-dish.
</LI><LI>Plasmid-isolation of p714
</LI></UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> We prepared precultures of <I>E.coli</I> KRX without plasmid and <I>E.coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A> or <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A> as positive control.
</LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 6:</B> Check for active laccases in supernatants from lysed samples of the last cultivation (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005/" class="external text" rel="nofollow">ECOL</A>,<A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A>,<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020/" class="external text" rel="nofollow">BHAL</A>,<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010/" class="external text" rel="nofollow">TTHL</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015/" class="external text" rel="nofollow">XCCL</A>) (n=4).</DIV></DIV></DIV><UL><LI><B>Team Activity Tests:</B> Another cultivation ended and we are happy to measure the supernatants from the 31st September cultivated samples. We used 158 µL of the supernatant with 40 µL and added 2 µL of 20 mM ABTS to measure some activity (Fig. 6). Nevertheless we couldn't see any increase in absorbency at 420 nm after 5 minutes, indicating, that we have no active laccase in the supernatants.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_August_5th">Sunday August 5th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases</B>:
</LI><LI> The assemblies don't work by now in one step by now, we always get just red colonies from the original RFP plasmid. So we started to clone one part after another in pSb1C3 backbone. For this reason we want to ligate the ecol gene without any promoter in pSB1C3 backbone and in the next step do a prefix insertion with the promoter fragment. Therefore we did the restriction of ecol and ecol_HIS PCR products, the ligation in pSB1C3 and the transformation. 
</LI></UL><LI><B>Team Site Directed Mutagenesis:</B> Plasmid-isolation of the xccl- and tvel-double-mutants.<DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Growth kinetics of <I>E. coli</I> KRX with &lt;partinfo&gt;K863000&lt;/partinfo&gt;</DIV></DIV></DIV></LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Growth kinetics of <I>E. coli</I> KRX with &lt;partinfo&gt;K863005&lt;/partinfo&gt;</DIV></DIV></DIV><UL><LI>Test-restriction showed two positive plasmids on the xccl-g2247c and xccl-g3633c-mutagensis and no positve tvel.
</LI><LI>plated two colonies of the ecol-g2307a-mutagenesis and plated them on select-agar for plasmid-isolation.
</LI><LI>Prepared the two positive double mutants of the xccl-plasmid for sequencing.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Transformated BBa_I13522 in KRX and plating it on AMP-selection-agar.
</LI><LI>Plasmid-isolation of p570
</LI><LI>PCR of <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex</A> (417 bp) and <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos</A> (369 bp) followed by test-gel-electrophoresis showed bands of the correct size and PCR-Clean-up
</LI><LI>Restriction of PCR-products and pSB1C3_RFP with <I>Xba</I>I and <I>Pst</I>I.
</LI><UL><LI><B>Team Cultivation &amp; Purification:</B></LI></UL><UL><LI> We made another flask cultivation of <I>E. coli</I> KRX without plasmid and with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week1#Monday_April_30th" class="external text" rel="nofollow">pBpL6</A> and <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A> as positive control.
</LI><LI> general settings: 250 mL flasks without baffles, final volume: 50 mL, LB medium
</LI><LI> additional settings:
</LI></UL><DL><DD> a) 2 flasks of each culture were inducted with 0.1 % rhamnose after 4 hours of cultivation
</DD><DD> b) 2 flasks of each culture were not induced.
</DD></DL><UL><LI> We recorded the growth kinetics once per hour.
</LI></UL><DIV id="anzeige"><H1>Summary of Week 15</H1><H2><SPAN class="mw-headline" id="Week_15_.2808.2F06_-_08.2F12.2F12.29">Week 15 (08/06 - 08/12/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_15_.2808.2F06_-_08.2F12.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 15 (08/06 - 08/12/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_August_6th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday August 6th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_August_7th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday August 7th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_August_8th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday August 8th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_August_9th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday August 9th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_August_10th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday August 10th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_August_11th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday August 11th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_August_12th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday August 12th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_14"> Weekly Seminar </SPAN></H3><UL><LI> We did a safety and security training today.
</LI><LI> We agreed to write in wiki code in our wiki, because not everyone is capable of writing in html.
</LI><LI> Next week Hakan will present his problems with getting the PCR to work
</LI></UL><H3><SPAN class="mw-headline" id="Monday_August_6th">Monday August 6th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> No positive colonies after transformation of our assemblies from August 1st but we realized that the primers we used for making the promoter parts can’t ligate with our backbone because the primers are dephosphorylated and the plasmid backbone is dephosphorylated, too. Much effort in a mission which can't work but at least we know now why it doesn't work.
</LI></UL><DL><DD><UL><LI> Picking positive colonies from transformation of ecol and ecol_HIS in pSB1C3 for plasmid isolation.
</LI></UL></DD></DL><UL><LI><B>Team Fungal and Plant Laccases:</B> Plating positive colonies from cloning of tvel5 in pSb1C3 backbone.
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Plasmid-isolation of two more ecol-g2307a colonies.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Plated one colony of &lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt; on selection-agar.
</LI><LI>Dephosphorylation of pSB1C3 Backbone.
</LI><LI>Ligation of <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> with the pSB1C3-Backbone and <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A> with the pSB1C3-Backbone.
</LI><LI>Transformed both in KRX and plated on CM-selection-agar.
</LI><UL><LI><B>Team Immobilization:</B></LI><LI>CPC beads have just arrived! We couldn’t wait any longer to start our new experiments. Since the amount of beads used and the incubation period varied from one paper to another (see Literature), we decided to find out ourselves the most convenient conditions for our laccases. We started with different bead concentrations: 0.01, 0.02, 0.04, 0.06, and 0.08 mg/mL. The beads were first immersed in 2.5% glutaraldehyde and incubated for 2 hours under light vacuum in order to allow as much beads’ surface area to be coated with aldehyde groups, which crosslink the laccases to the beads. After that, the beads were washed 3 times with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols#Briton_Robinson_Buffer" class="external text" rel="nofollow">Britton-Robinson Buffer</A> and then immersed in 1mg/mL laccases from TVEL0 and placed on a rotator at 4°C for 18h and 36h respectively. For further information see <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Immobilization" class="external text" rel="nofollow">Protocol</A>.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_August_7th">Tuesday August 7th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI></UL><DL><DD><UL><LI> Plasmid isolation and control restriction of ecol and ecol_HIS in pSB1C3 showed correct fragment sizes in agarose gel. So we did a digest for prefix insertion of the new T7 promoter.
</LI></UL></DD></DL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Digestion of the two ecol-g2307a-mutants showed that one has lost the restriction-site. Prepared that one for sequencing.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Plasmid-isolation of &lt;partinfo&gt;Bba_I13522&lt;/partinfo&gt;.
</LI><LI>PCR with &lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt; as template and GFP_Freiburg and GFP_His6_compl-primers and clean-up through agarose-gel.
</LI><LI>Restriktion of Linear Backbone, <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> and <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> with <I>Eco</I>RI and <I>Pst</I>I.
</LI><LI>Ligation of Linear Backbone with <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> and with <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>, respectively.
</LI><LI>Transformation of all that stuff and plating on selection-agar.
</LI><LI><B>Team Immobilization:</B></LI><LI>The 18h samples (see above: Monday August 6th/Team Immobilization) were ready for further analysis. The supernatants were gathered to be tested for laccase activity. The beads were then washed with Britton-Robinson Buffer (link), then with 0.5M NaCl solution in order to wash away all noncovalently bound laccases and again twice with the buffer. Subsequently, the beads were immersed in 2.5 mg/mL glycine for 18h at 4°C.
</LI><H3><SPAN class="mw-headline" id="Wednesday_August_8th">Wednesday August 8th</SPAN></H3><UL><LI><B> Team Shuttle Vector:</B> Some more conditions were varied to get the fragment tAOX1 for the shuttle vector:
</LI><LI> An other annealing temperature gradient was run with 2% DMSO, 4% DMSO included samples and samples mixed with 5x GC buffer instead of 5x HF buffer. The right products were detected under the following conditions:
</LI><LI> In the sample with 2% DMSO and an annealing temperature of 40.9 °C.
</LI><LI> In the sample with 4% DMSO and an annealing temperature of 40.3 °C, 57.1 °C, 59.9 °C.
</LI><LI> In the sample with 5x GC buffer and an annealing temperature of 50.7 °C, 54.0 °C, 59.9 °C.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> We dephosphorylated the digested the plasmids from day before and phosphorylated the promoter parts. After that we ligated the two parts and transformated the products into KRX electrocompetent cells.
</LI><LI><B>Team Fungal and Plant Laccases:</B> Plasmid isolation of tvel5 laccase in pSB1C3 backbone.
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Made PCRs on tvel-t143g-mutants with Tv_lac10.P.FW and Tv_lac10.S.RV primers, with products of 1.6 kbp when there should be about 4.0 kbp.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Picked one not-red colony of <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)</A>, <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos</A> and <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> and plated it on seletion-agar
</LI><LI>Restriktion of <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>-PCR-product with <I>Age</I>I and <I>Eco</I>RI with <I>Dpn</I>I
</LI><LI>Restriktion of <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>-PCR-product with <I>Ngo</I>MIV and <I>Pst</I>I and <I>Dpn</I>I
</LI><LI>Freiburg-Assembly with <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> and <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> on the linearized pSB1C3 Backbone followed by plating on selection-agar.
</LI><LI><B>Team Activity Tests</B>: Today was an exciting day for us. We got samples from Team Immobilization for activity measurements. They were trying to analyze which concentration of beads is a good choice for immobilizing the laccases. The exciting part was to check all the solvents they are using for possible reaction with ABTS during measurements. Check Team Immobilization´s Labjournal and <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols" class="external text" rel="nofollow">protocols</A> for the output and further information.
</LI><LI><B>Team Immobilization:</B></LI><LI>The 18h samples (see above: Monday August 6th/Team Immobilization and Tuesday August 7th/Team Immobilization) were washed with Britton-Robinson Buffer (link), then with 0.5M NaCl solution and again twice with the buffer.  The 36h samples were treated similarly as the 18h samples (see above: Tuesday August 7th/Team Immobilization). The supernatants were delivered to the “Activity Test” team to measure the activity of the non-bound laccases. Based on the results, we would be able to figure out the most convenient bead concentration and incubation period. After comparing the results, we found out that the activity of the supernatant after 36h was less than after 18h, which indicated that 36h incubation is more convenient. On the other hand, the activity also decreased upon increasing the beads’ concentration without reaching a saturation level (see Fig.). Therefore, we decided to carry out another experiment with higher beads concentration.
</LI><H3><SPAN class="mw-headline" id="Thursday_August_9th">Thursday August 9th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1:</B> Agarosegel from the Streptomyces gradient PCR. 1-12: <I>S. goettingen</I> in Sequence with following Annealing temp. 54.9°C, 55.1°C, 55.8°C, 56.8°C, 58.1°C, 59.5°C, 61.0°C, 62.4°C, 63.7°C, 64.8°C, 65.6°C, 66°C. 13-22: <I>S. tuebingen</I> in Sequence with following Annealing temp. 54.9°C, 55.1°C, 55.8°C, 56.8°C, 58.1°C, 59.5°C, 61.0°C, 62.4°C, 63.7°C, 64.8°C</DIV></DIV></DIV></LI><LI> We sent ecol and ecol_His (both in pSB1C3) for sequencing.
</LI><LI> We did again a gradient PCR for the Streptomyces but this time without <I>S. tuebingen</I> because the DNA we got was empty. The results can be seen on Figure 1.
</LI></UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI>Again: Ligation of tvel35 in pSB1C3 backbone.
</LI><LI> Plasmid isolation of tvel5 + pSb1C3 and control digest with <I>Not</I>I. For one of the two plasmids the digest looks good. We sent this plasmid for sequencing.
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Sequencing results arrived:
</LI><LI>One bpul-plasmid is positive on both mutations
</LI><LI>All Xccl-mutants are negative (there even was a part of 900 bp gone missing!)
</LI><LI>plated six more colonies of the xccl-g3633c for plasmid-isolation
</LI><LI>PCR with the original tvel10-plasmid and the Prefix/Suffix-primers, showed no product at all.
</LI><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Plasmid-isolation of <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>, <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> and <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>.
</LI><LI>Transformated the assembly again with more of volume of the assembly-CBDcex-GFP_His-mix.
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> We got <I>E. coli</I> KRX with our own BioBrick <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> in pSB1C3 like the rest.
</LI><LI> We discussed if we did not produce anything because of the toxicity of our protein, which may reduce the plasmid stability. We searched for maximal used concentration of chloramphenicol and found out that concentrations up to 170 µg/mL were used. Therefore we decided to start a new flask cultivation with concentrations of chloramphenicol varying in 5 steps from 20 µg/mL to 170 µg/mL. Today we made the precultures of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> and <I>E. coli</I> KRX as negative control.
</LI><H3><SPAN class="mw-headline" id="Friday_August_10th">Friday August 10th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI></UL><DL><DD><UL><LI> Again we did the digest of our new T7 promoter part and the ligation in pSB1C3 backbone with ecol PCR products with and without HIS tag. After that we transformed the ligations in pSB1C3. Additionally we did the same with promoter J23110 instead of T7 promoter.
</LI><LI> We did PCR on <A href="http://partsregistry.org/Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A> with the primers B.halo_FW and B.halo_RV for cloning the gene in pSB1C3 backbone without promoter and HIS tag.
</LI><LI> We ligated the digested pSB1C3 plasmids with ecol and ecol_HIS with the new pT7 promoter and pSB1C3 backbone and transformed the approaches in KRX.
</LI></UL></DD></DL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Plasmid-isolation xccl-g3633c-colonies and test-digestion showed no positive mutated plasmids.
</LI><LI>New <I>pfu</I>-PCR on the xccl-plasmid with xccl-g3633c primer-mix
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Picked <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A> assembly colony and plated it on CM-selection-agar for plasmid-isolation.<DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Cultivation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K863005&lt;/partinfo&gt; for different concentrations of chloramphenicol (20 to 170 µg mL <SUP>-1</SUP>. OD<SUB>600</SUB> was measured.</DIV></DIV></DIV></LI><LI>Restricted <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>with <I>Eco</I>RI and <I>Age</I>I
</LI><LI>Digested insert did not appear in the 
</LI><LI>Colony-PCR of <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>.
</LI><UL><LI>Picked and plated several colonies of the <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>-assembly and plated the on CM-Agar.
</LI></UL><LI><B>Team Cultivation &amp; Purification:</B></LI><UL><LI>Flask cultivation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863005</A> to test various concentrations of chloramphenicol. We used <I>E. coli</I> KRX as negative control.
</LI><LI>Settings: 100 mL flasks without baffles, final volume: 10 mL, autoinduction medium with 20/ 57.5/ 95/ 132.5/ 170 µg/mL chloramphenicol, 37 °C, 120 rpm, double determination.
</LI></UL><LI> Note: Found out, that we did not use the maximal possible concentration of chloramphenicol. We decided to use a higher concentration than 20 µg/mL and chose 60 µg/mL for further cultivations. This is a compromise between benefits and costs.
</LI><H3><SPAN class="mw-headline" id="Saturday_August_11th">Saturday August 11th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI></UL><DL><DD><UL><LI> Colony PCRs showed positive colonies for ecol(J23110) so we plated this colonies on new plates.
</LI><LI> We did the PCRs of the laccase genes ecol, bpul, bhal and tthl again. We used the …_FW / …_RV primers and the …_FW / …_RV_HIS primers of the different genes. Digestion of this PCR products and ligation with pT7 or promoter J23110 and the pSB1C3 plasmid backbone.
</LI></UL></DD></DL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Agarose-gel-electrophoresis of xccl-PCR (yesterday) showed bands at 3.2 kbp and over 12 kbp but not at 4 kbp as it should be.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>A few of the plated  pSB1C3+CDBcex+GFP-assembly colonies have grown
</LI><LI>Colony-PCR <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>-colonies showed no positive clones.
</LI><LI><B>Team Activity Tests</B>: Today we received samples of <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> from Team Cultivation that were cultivated in different chloramphenicol concentrations. Very exciting! We measured as usual at OD<SUB>420</SUB> and analyzed the supernatant (Figure 1) and the cell extract (see Figure 2). Unfortunately no activity was seen for any of the samples. We feel sorry.
</LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Figure 1:</B> Activity measurement of the supernatant of the<A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> cultivation. Cells were cultivated in different chloramphenicol concentrations. Measurements were done via oxidized ABTS and at OD<SUB>420</SUB>.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Figure 2:</B> Activity measurement of the cell extract of the<A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> cultivation. Cells were cultivated in different chloramphenicol concentrations. Measurements were done via oxidized ABTS and at OD<SUB>420</SUB>.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Sunday_August_12th">Sunday August 12th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> PCR cleanup of the PCR products from the day before.
</LI><LI>We did control digest and got bands for ecol with J23110 promoter and with the new pT7. So we sent this plasmids for sequencing.
</LI></UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI><I>pfu</I>-PCR with xccl-plasmid and xccl-g2247c primers resulted in no product.
</LI><LI>PCR with pre- and suffix primers for tvel10 with original Topo-plasmid as template with increasing temperature at each step (0.1°C) resulted in no product.
</LI><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Colony-PCR for <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>, <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A> and <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>.
</LI><LI>resulted in two positve <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>-colonies.
</LI><LI>Did another colony-pcr on <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> and <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A> that brought up nine positive clones of <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>.
</LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Figure 2:</B> Comparison of different lysis methods.Potential laccase activity of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> was measured after lysis via OD<SUB>420</SUB>.</DIV></DIV></DIV><UL><LI><B>Team Activity Tests</B>: Today was another day for finding the mistake. Team Cultivation used different methods the for lysis of <I>E.coli KRX</I> cell extract to check whether our <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> laccases and thus their activity might me damaged during lysis. Compared were an enzymatic lysis via lysozyme and a chemical lysis via B.Per/DNase, also both ways were combined in the third sample We are sorry to tell that no activity was seen. 
</LI></UL><DIV id="anzeige"><H1>Summary of Week 16</H1><H2><SPAN class="mw-headline" id="Week_16_.2808.2F13_-_08.2F19.2F12.29">Week 16 (08/13 - 08/19/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_16_.2808.2F13_-_08.2F19.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 16 (08/13 - 08/19/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_August_13th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday August 13th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_August_14th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday August 14th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_August_15th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday August 15th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_August_16th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday August 16th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_August_17th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday August 17th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_August_18th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday August 18th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_August_19th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday August 19th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_15"> Weekly Seminar </SPAN></H3><UL><LI> As part of our collaboration the iGEM team from <A href="https://2012.igem.org/Team:SDU-Denmark" class="external text" rel="nofollow">SDU-Denmark</A> will arrive on saturday (25th of august).
</LI><LI> Our StreetScience booth will have 16 m<SUP>2</SUP>.
</LI><LI> We already ordered all info-material we want to provide at StreetScience.
</LI><LI> Just in case we are going to qualify for the world finals in Boston, everyone needs a visa. Especially our team members without a european nationality have to inform about visa terms.
</LI><LI> Derya is responsible for planning a new, project-driven video.
</LI><LI> As deadlines come closer we have to think about a team name.
</LI><LI> He have decided to rent a bus to travel to Amsterdam.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_August_13th">Monday August 13th</SPAN></H3><UL><LI><B> Team Cloning of Bacterial Laccases</B>: 
</LI><LI> After cleaning up the gradient PCR from the 09th August we did an digestion with the <I>S. goettingen</I> DNA for cloning it into the pSB1C3. The pSB1C3 was additionaly treated with Shrimp-alkaline-phosphotase. After this we set the ligation of insert DNA and vector DNA. 
</LI></UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Gradient-PCR of tvel10 55 to 66°C with DMSO (12 Steps) resulted in a lot of product at 55°C and 63°C to 66°C; a little product at 56°-59° and 61°-62°C and no product at 60°C (59°C was the temperature Clonemanager predicted and I used before). Merged the products and cleaned them up.
</LI><LI>Digested the product with <I>Eco</I>RI, <I>Pst</I>I and <I>Dpn</I>I.
</LI><LI><B>Team Cellulose Binding Domain:</B></LI><LI>We did another PCR on the &lt;partinfo&gt;I13522&lt;/partinfo&gt; with the <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>-primers with positive result.
</LI><LI>Plasmid-isolation of two positive clones of <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A> followed by digestion with <I>Not</I>I that confirmed that the plasmid is <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>. Prepared both for sequencing.
</LI><LI>Plated three positve <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>-colonies.
</LI><LI>Restriction of the <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>-PCR-product with <I>Eco</I>RI and <I>Pst</I>I to bring it into the pSB1C3-backbone.
</LI><LI>Also: Restriction of the <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>-PCR-product with <I>Ngo</I>MIV and <I>Pst</I>I for an assembly.
</LI><LI>Restriktion of the <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>-PCR-product with <I>Age</I>I and <I>Eco</I>RI for the assembly.
</LI><LI>All digestions were done with <I>Dpn</I>I to get rid of the templates.
</LI><LI>Ligation of <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> with the pSB1C3-backbone.
</LI><LI>Ligation of <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7) and GFP_His</A> with the pSB1C3-backbone.
</LI><LI>Transformed all ligations into KRX.
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Today we discussed that we may get activity if we start to purify our laccases. So from now we will purify our products before measuring them. We hope this will bring promising results. Therefore we searched for a useful method.
</LI><H3><SPAN class="mw-headline" id="Tuesday_August_14th">Tuesday August 14th</SPAN></H3><UL><LI><B> Team Cloning of Bacterial Laccases:</B> We finally had positive bacteria for the <I>S. goettingen</I>. So we decide to do a colony PCR. But the result has shown that there were no insert DNA but we didn't want to give it up so we transfered some colonies to a new plate an let them grown over night again.
</LI><LI><B>Team Fungal and Plant Laccase</B>: Today our cDNA took a bath in ethanol and got cleaned. We are pretty sure our next PCR will be a lot more successful. Check protocols for further information about the cDNA washing via <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Genetics#Ethanol_precipitation_to_clean_DNA" class="external text" rel="nofollow">ethanol precipitation</A>.
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Made preculture of <I>E. coli</I> KRX without plasmid and with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> as well as <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>.
</LI></UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Ligation of tvel (PCR-product) and pSB1C3 and transformation into KRX
</LI><LI>Gradient-PCRs (55°C bis 72 °C) with xccl-plasmid using xccl-g3633c and xccl-g2247c primer-mixes, respectively, resulted in no product of the right size.
</LI><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Sequencing results showed, that the insertion of <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> wasn't successful at all, <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> had a lot of mutations and <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A> had one silent mutation.
</LI><LI>On the <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>-transformation-dish did not grow any colony. Did a new transformation with a little more ligation-product and plated the cells on selection-agar.
</LI><LI>Plasmid-isolation of the three positive <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>-plasmids and digestion showed not the correct fragments. Plated some more positive colonies on selection-agar for plasmid-isolation.
</LI><LI>The <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>-dish had only nine colonies and a colony-PCR showed that none of them had the right insert.
</LI><LI><B>Team Activity Tests</B>: 
</LI><LI> Again our skills in measuring laccase activity were in demand today. We had a job incoming from Team Substrate Analytic. Since they need to dissolve their substrates in methanol or acetonitrile, they were curious about laccase activity related to these solvents. And they infected us with their curiosity! So we started measurements using 0.1 U TVEL0 Laccase, 40 µL sodium acetate buffer and different amounts of MeOH or acetonitrile respectively, ad 200 µL H<SUB>2</SUB>O. Our tested range went from 2 µL to 18 µL. The results show that MeOH and acetonitril have an impact on the ability of TVEL0 to oxidize ABTS. Using 16 µL MeOH (which means 8% of the amount of the sample contained MeOH) led to a different saturation curve as usual. The saturation couldn't be reached any more (see Fig. 1). Acetonitrile had a noticeable impact when using 5% (14 µL) of it in relation to the sample (see Fig. 2). But in the end both solvents couldn't stop TVEL0 laccase from being active. We are happy to tell Team Substrate Analytics that they can proceed as planned and dissolve the substrates in the tested solvents. 
</LI><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:432px;"><DIV class="thumbcaption"><B>Figure 1:</B> TVEL0 activity measured using 0.1 mM ABTS at 25°C over a time period of 5 minutes in presence of different MeOH concentrations (n=4)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:432px;"><DIV class="thumbcaption"><B>Figure 2:</B> TVEL0 activity measured using 0.1 mM ABTS at 25°C over a time period of 5 minutes in presence of different acetonitrile concentrations (n=4)</DIV></DIV></DIV></DIV><UL><LI><B>Team Substrate Analytic</B>:
</LI><LI> Our first attempt do detect diverse estrogens with the clairity HPLC. Dominic helped us to set the HPLC method based on the paper „Enzyme-Catalyzed Oxidation of 17β-Estradiol Using Immobilized Laccase from Trametes versicolor”.
</LI><LI> We prepared dilution series from three different estrogens and tryed to mesure them. Sadly all substrates left the column with the same retention time as the solvent.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_August_15th">Wednesday August 15th</SPAN></H3><UL><LI><B>Team Wiki</B>: We set up some more wiki rules. Today´s rules were about citations. We agreed about some standards that will for sure help make the wiki a little prettier. Have a look:
</LI><LI><SPAN style="text-decoration: underline;">Paper</SPAN>: Exampleman M <I>et al</I>. (2002). The example paper. The example journal (Volume): pages.
</LI><LI><SPAN style="text-decoration: underline;">Website</SPAN>: Name of website, URL, Index, date site visited
</LI><LI><SPAN style="text-decoration: underline;">Book</SPAN>: Examplewomen M <I>et al</I>. (1999). The example book. The example publisher (edition).
</LI></UL><P>Next will be a standard for charts.
</P><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> Plasmid isolation from the plate of the 14th August. This was done with the Analytic Jena Kit.
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Colony-PCR of tvel10-colonies resulted in small bands. Quickly explained: tvel10 still had illegal <I>Pst</I>I-restrictions-sites. Digestion of tvel10 PCR-product with <I>Not</I>I as well as digestion of RFP-pSB1C3 with <I>Not</I>I
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Plasmid-isolation of the three positive <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>-colonies, but digestion showed no positive result, again.
</LI><LI>Colony-PCR of <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> and <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A> did not show any positive result, ether.
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Flask cutivation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>,<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A>. We used <I>E. coli</I> KRX negative control as well as <I>E. coli</I> KRX with <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A> as positive control. 
</LI><LI>Settings: 1 L flasks without baffles, final volume: 250 mL, autoinduction medium supplemented with 60 µg/mL chloramphenicol, 37 °C, 120 rpm, single determination
</LI><LI><B>Team Activty Tests:</B> Today we got another mission from Team Cultivation. They gave us a huge amount of laccase samples for measurements. The samples contained as well general laccase samples for activity tests but also the flow through from the purification process to check if any laccase is lost during this process. The results show no enzyme activity in any case.
</LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Output of enzyme activity measurements via oxidized ABTS and OD<SUB>420</SUB>. Measured is the flow through of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863015" class="external text" rel="nofollow">XCCL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863020" class="external text" rel="nofollow">BHAL</A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A>. <I>E. coli KRX</I> (no plasmid) was used as negativ control and cells containing a plasmid with ligase was used for the control of induction. The elution buffer was as well tested as a negative control.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Output of enzyme activity measurements via oxidized ABTS and OD<SUB>420</SUB>. Measured is the activity of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863015" class="external text" rel="nofollow">XCCL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863020" class="external text" rel="nofollow">BHAL</A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A>. <I>E. coli KRX</I> (no plasmid) was used as negativ control and cells containing a plasmid with ligase was used for the control of induction. The binding buffer was as well tested as a negative control.</DIV></DIV></DIV><UL><LI><B>Team Substrate Analytic:</B></LI><LI> Dominik tried to regenerate the column. After that we measured new dilution series from all three estrogens. Once again all substrates left the column together with the solvent. That means the column is defect.
</LI><LI> We ordered new C18-columns from Knauer and waited for their arriving. A Eurosphere and a Bluespehere column.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_August_16th">Thursday August 16th</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B> For the Gibson assembly we need more higher concentrated fragments. In fact, Phusion PCR was done for the fragments 5AOX1, 3AOX1, Kozak-MFalpha1 and pSB1C3 at following conditions with 35 cycles. PCR fragments were used as DNA template.
</LI></UL><TABLE class="wikitable" style="text-align: center; width: 300px: height: 300px;" border="0}"><TBODY><TR><TH> Step
</TH><TH> Condition
</TH></TR><TR><TD>Initial denaturation
</TD><TD>98 °C, 30 s
</TD></TR><TR><TD>Denaturation
</TD><TD>98 °C, 10 s
</TD></TR><TR><TD>Primer annealing
</TD><TD>67 °C, 20 s
</TD></TR><TR><TD>Elongation
</TD><TD>72 °C, 90 s
</TD></TR><TR><TD>Final Elongation
</TD><TD>72 °C, 5 min
</TD></TR></TBODY></TABLE><P>Phusion PCR was also done for the HIS4 fragment at following conditions with 35 cycles and also PCR product as template.
</P><TABLE class="wikitable" style="text-align: center; width: 300px: height: 300px;" border="0}"><TBODY><TR><TH> Step
</TH><TH> Condition
</TH></TR><TR><TD>Initial denaturation
</TD><TD>98 °C, 30 s
</TD></TR><TR><TD>Denaturation
</TD><TD>98 °C, 10 s
</TD></TR><TR><TD>Primer annealing
</TD><TD>60 °C, 20 s
</TD></TR><TR><TD>Elongation
</TD><TD>72 °C, 90 s
</TD></TR><TR><TD>Final Elongation
</TD><TD>72 °C, 5 min
</TD></TR></TBODY></TABLE><P>Phusion PCR was also done for the tAOX1 fragment at following conditions with 35 cycles and also PCR product as template. For us it can not be recognized at which annealing temperature the primer binds best. That is why we choose an annealing temperature gradient more. This time from 50 °C until 60 °C. The mixture of the sample is the same as last time.
</P><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Clean-up of tvel10 and pSB1C3 (both cut with <I>Not</I>I); Dephosphorilation of pSB1C3 with SAP and ligation. Transformed into KRX and plated on select-agar.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Colony-PCR of <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>-clones with no result, even the positive-control was negative, so we plated three colonies for a test-digestion instead.
</LI><LI>Digestion of <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A> with <I>Age</I>I and <I>Pst</I>I and digestion of the <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>-PCR-product with <I>Ngo</I>MIV and <I>Pst</I>I over-night.
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Today we harvested the cells from the cultivation on 08/15, disrupted them in <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Buffer_for_Ni-NTA-HisTag_affinity_chromatography" class="external text" rel="nofollow">equilibration buffer</A> via sonication, centrifugated and purified the supernatant via HisTrap column. The purified samples in 500 mM Imidazol were given to the activity test team.
</LI><H3><SPAN class="mw-headline" id="Friday_August_17th">Friday August 17th</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Stopped digestion of <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A> and <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>. Dephosphorylated pSB1C3+<A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>.
</LI><LI>Plasmid-isolation of the three <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>-dishes.
</LI><LI>The sequencing of the two <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>-plasmids showed the same result as the first: one silent mutation. We decided to carry on with this plasmid.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_August_18th">Saturday August 18th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial laccases</B>: 
</LI></UL><DL><DD><UL><LI> Today we performed a ligation from the digestions we have done before. The ligation included of course the T7 promotor, the backbone pSB1C3 and the individual laccase genes. Ligation was performed with T4 ligase for 20 minutes at room temperature and stopped through a five minute incubation at 70 °C.
</LI></UL></DD></DL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Ligated <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A> and <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>, transform into KRX and plated on CM-selection-Agar.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_August_19th">Sunday August 19th</SPAN></H3><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Colony-PCR of tvel10-colonies with three positive result.
</LI><LI>Plated those three on select-agar for plasmid-isolation.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Colony-PCR of colonies from the <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A>-trafo-dish with only negative results.
</LI><DIV id="anzeige"><H1>Summary of Week 17</H1><H2><SPAN class="mw-headline" id="Week_17_.2808.2F20_-_08.2F26.2F12.29">Week 17 (08/20 - 08/26/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_17_.2808.2F20_-_08.2F26.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 17 (08/20 - 08/26/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_August_20th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday August 20th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_August_21st"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday August 21st</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_August_22nd"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday August 22nd</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_August_23rd"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday August 23rd</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_August_24th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday August 24th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_August_25th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday August 25th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_August_26th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday August 26th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_16"> Weekly Seminar</SPAN></H3><UL><LI> Our denish guests from <A href="https://2012.igem.org/Team:SDU-Denmark" class="external text" rel="nofollow">SDU-Denmark</A> will stay from 25th to 27th of august.
</LI><LI> Program will be as follows: On saturday together we will present our teams on StreetScience and saturday evening it is time for Eurodance with music from David Hasselhoff. On sunday we will have breakfast together and after we will have a scientific symposium with discussing our projects, problems and trouble shooting. On sunday evening we will show our denish guests the cozy inner part of Bielefeld.
</LI><LI> StreetScience: we can't rent televisions from Bielefeld Marketing, but Derya could organize two other big televisions. We will have a light microscope and our dress code will be to wear a black t-shirt.
</LI><LI> Agatha and Saskia will bring <I>Volvox</I>, Nadine will bring <I>Xanthomonas</I> and Robert <I>Corynebaterium</I>. All strains are harmless and wild-type strains.
</LI><LI> Mo organizes some games for kids and Agatha&amp;Saskia will tinker prices.
</LI><LI> Start of preparation is 9.30
</LI></UL><H3><SPAN class="mw-headline" id="Monday_August_20th">Monday August 20th</SPAN></H3><UL><LI><B>Team  Shuttle Vector:</B> A Gibson assembly was done as described in the <A href="/Team:Bielefeld-Germany/Protocols/molecular_genetics#Gibson_assembly" title="Team:Bielefeld-Germany/Protocols/molecular genetics"> protocol</A>, but only 10 ng could be used, because the concentration of the fragments was to low. The sample was transformed in <I>E. coli</I> KRX via electroporation and plated out on selective LB Cm plates.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Digestion of pSB1C3 plasmid backbone, tthl_HIS and bpul_HIS for doing the assembly with the new T7 promoter and J23110 again.
</LI><LI> We did PCRs on bpul_HIS, bhal_HIS and ecol_HIS again.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Biggg Colony-PCR of the <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A>-ligation with the result, that all seem to carry <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>. The ligation didn't work at all...
</LI><LI><B>Team Cultivation &amp; Purification</B>: 
</LI><LI>Today we decided to finish our lean period of non-active laccases. It was time for a transcript analysis to have a look if our laccases are transcribed into RNA and if there is a lot of it or if its gets degraded just in time. We took our cells, centrifuged and shock freezed them in liquid nitrogen to stop every metabolism in the cell and freeze the status of the RNA. The frozen cell pellets were stored at -80°C for the upcoming tests. As well we designed and ordered primers.
</LI><UL><LI><B>Team Immobilization</B></LI><LI>Today we set up a new immobilization experiment following the same procedure as in week 15 (see week 15 Monday August 6th-Wednesday August 8th /Team Immobilization) but with different bead concentrations: 0.06, 0.08, 0.1, 0.12, 0.14 and 0.16 mg/mL. Based on the results of the first experiment, the samples were incubated with TVEL0 for 36h.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_August_21st">Tuesday August 21st</SPAN></H3><UL><LI><B> Team Shuttle Vector:</B></LI><LI>Design of primers for the Part <I>his4</I><A href="http://partsregistry.org/Part:BBa_K863202" class="external text" rel="nofollow">K863202</A>.
</LI><LI>Colonies from yesterday were plates out and Colony-PCR was done with GoTaq and the F and R primer to amplificate the part <A href="http://partsregistry.org/Part:BBa_K863204" class="external text" rel="nofollow">K863204</A>.
</LI><LI>From positive clones the plasmid was isolated and restriction analyses with <I>Not</I>I was done. No one of the clones has passed the restriction analyse.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> The isolated plasmid from the 15th August was prepared and given up for sequencing. 
</LI><LI> We designed primers for q-PCR for Team Activity Tests (details, look above). The PCRs with this primer pairs should result in about 200 bases long fragments.
</LI><LI> We realized that the primer pairs we anneal for the promoter parts had about 2000 ng/µl if diluted 1:10 from originally 100 pmol. Maybe our used amounts on promoter for the assemblies was a LITTLE to high. So now we try a dilution of 1:1000 with about 2 ng/µl.
</LI><LI> PCR products of different laccases were purificated and digested for suffix insertion.
</LI><LI> Ligation of the laccase genes ecol, ecol_HIS and bpul and bpul_HIS, the 1:000 diluted (0,1 pm/µl) promoters (pT7 and P110) in pSB1C3 vector.
</LI><LI>Ligation of the digested ecol_pSB1C3 and ecol_HIS_pSB1C3 plasmids with the 1:000 diluted (0,1 pm/µl) promoters (pT7 and P110).
</LI><LI>Ligation of T7 promoter and J23110 promoter in pSB1C3 backbone.
</LI></UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> We designed primers for the laccase &lt;partinfo&gt;BBa_K500002&lt;/partinfo&gt; for cloning in <I>P. pastoris</I> shuttle vector.
</LI><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Tested restriction enzymes.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_August_22nd">Wednesday August 22nd</SPAN></H3><UL><LI><B>Street Science:</B></LI></UL><DL><DD><UL><LI>Went to Dr. Joe Max Risse today and asked him about the mircoorganisms of the Fermentationgroup. He recommended <I>Euglena gracilis</I>, since it is without risk, big, colorful, and very healthy under the microscop.
</LI><LI>I asked for the bacteria they have and he told me I could check if their <I>Kocuria rosea</I> is a wild type.
</LI><LI>In the DMSZ catalog there are three strains of <I>Kocuria rosea</I> and all have been assessed to be of riskgroup 1, which is the safest group of bacteria and means it is unlikely that it will infect humans.
</LI><LI>I also asked for <I>Penicillium chrysogenum</I> but the one they use is a GVO.
</LI></UL></DD></DL><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> Colony PCRs: some prefix insertions of promoters in pSB1C3 with ecol and ecol_HIS showed positive bands, so we picked the colonies for plasmid isolation. The problem with the prefix insertion of the promoters in pSB1C3 with the ecol gene is that we can’t easily prove that the promoters were ligated in the backbone because with control restriction there would be an about 50 bp longer fragment which is difficult to see even in 3 % agarose gel. For this reason we designed FW primers (J23103_K_FW and J23110/117_K_FW and T7_K_FW) for our colony PCR, which can just bind, if the new promoter sequence is included in the plasmid. Until all colonies from the promoter J23110 ligation were red, we had some white colonies for the pT7 promoter in pSB1C3 backbone. So we picked a positive colony and plated it for plasmid isolation.
</LI><LI> We plated colonies on new plates, which weren't red from the transformation of pSB1C3 + pT7. We can't do colony PCR on this plasmids, because the resulting parts would be smaller than 100 bps.
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 1:</B> SDS-PAGE of purification from cultivation (08/15) of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>,<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A>, <I>E. coli</I> KRX (negative control) and <I>E. coli</I> KRX with <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A> (positive control). 250 mL in 1 L flasks without baffles, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> with 60 µg mL<SUP>-1</SUP> chloramphenicol at 37 °C for 12 hours. Purification of the supernatant via <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Production#Syringe_method" class="external text" rel="nofollow">syringe method</A>. The arrow shows ECOL in lane 10 and 11 (elution 1 and 2).</DIV></DIV></DIV></LI></UL><DL><DD><UL><LI>Ordered new primers for Xantomonas Campestris SDM, since there is still no positive colony and even gradient-PCR gave the wrong product
</LI></UL></DD></DL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Test-restriction of all isolated <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>- and <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>-plasmids with <I>Not</I>I with no positive result.
</LI><LI>Restarted from the beginning and made a gradient-PCR for <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> and <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> to get some more product.
</LI></UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Made SDS-PAGES of cultivation from 08/15.
</LI><H3><SPAN class="mw-headline" id="Thursday_August_23rd">Thursday August 23rd</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Made plasmid isolation of pSB1C3 + pT7 and sent plasmids for sequencing. 
</LI></UL><LI><B>Team Fungal Laccases:</B> We did PCRs on tvel35 plasmid with primer pairs Pc_lac35.P.FW / Pc_lac35.S.RV for trying ligation in pSB1C3 again and with  Pc_lac35_FW_oS / Pc_lac35_RV for cloning in shuttle vector.
</LI><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Clean-up of <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>- and <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>-PCR-product (pooled good ones and bad ones, respectively).
</LI><LI>Restriktion of the <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>-PCR-product with <I>Eco</I>RI and <I>Pst</I>I for ligation with the backbone and <I>Eco</I>RI and <I>Age</I>I for a assembly.
</LI><LI>Restriktion of the <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>-PCR-product with <I>Eco</I>RI and <I>Pst</I>I for ligation with the backbone and <I>Ngo</I>MIV and <I>Pst</I>I.
</LI><H3><SPAN class="mw-headline" id="Friday_August_24th">Friday August 24th</SPAN></H3><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Plasmid-isolation and digestion of tvel10 colonies with <I>Not</I>I showed that two of the plasmids have an insert of the right size.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Stopped restrictions and clean-up them up. Dephosphorilated the linearized backbone and ligated it to <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>, <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> and assembled <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>.
</LI><LI><A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>-cells plated to get some more plasmid.
</LI><H3><SPAN class="mw-headline" id="Saturday_August_25th">Saturday August 25th</SPAN></H3><P>STREET SCIENCE
Visit of Danish guys till 08/27
</P><H3><SPAN class="mw-headline" id="Sunday_August_26th">Sunday August 26th</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Made-up the idea to have a constitutive promoter (like &lt;partinfo&gt;BBa_J61101&lt;/partinfo&gt;) in front of the cellulose binding domain and the GFP. With a Freiburg-Prefix in front of the CBDs we could even easily switch directions of CBD and GFP. 
</LI><LI>Transformed &lt;partinfo&gt;BBa_J61101&lt;/partinfo&gt; from the iGEM-plate into KRX and plated it on AMP-selection-dish.
</LI></UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Made precultures of <I>E. coli</I> KRX without plasmid (negative control) and with plasmids containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A>. As positive control we used <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710 (Ligase A)</A>.
</LI><LI><B>Team Activity Tests:</B></LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Enzyme activity of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>, measured after incubation with 0.4 mM CuCl<SUB>2</SUB> and measured at OD<SUB>420</SUB> via oxidized ABTS.</DIV></DIV></DIV><UL><LI> What an awesome day! We were sitting here with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Tecan_Infinite_Microplate_Reader" class="external text" rel="nofollow">Tecan</A> preparing to do our usual measurements as suddenly super <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> appears. It came over from Team Cultivation and surprised all of us with its activity (see Figure 1).  This time we incubated the enzymes for 2 hours in 0.4 mM CuCl<SUB>2</SUB> before measurements. Also we measured longer to make sure to notice every single activity of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>. Stay tuned!
</LI></UL><DIV id="anzeige"><H1>Summary of Week 18</H1><H2><SPAN class="mw-headline" id="Week_18_.2808.2F27_-_09.2F02.2F12.29">Week 18 (08/27 - 09/02/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_18_.2808.2F27_-_09.2F02.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 18 (08/27 - 09/02/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_August_27th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday August 27th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_August_28th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday August 28th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_August_29th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday August 29th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_August_30th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday August 30th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_August_31th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday August 31th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_September_1st"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday September 1st</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_September_2nd"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday September 2nd</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_17"> Weekly Seminar</SPAN></H3><P><I>Summary of the weekend with our denish guests</I></P><UL><LI> We will try to collaborate in modelling and Julia V tries to help the iGEM team from SDU-Denmark to build up the wiki.
</LI></UL><P><I>Summary of StreetScience</I></P><UL><LI> Our commonday SynBioDay - StreetScience in Bielefeld was success, although it was a little chaotic in the beginning.
</LI><LI> Derya will evaluate our feedback poll.
</LI></UL><P><I>Deadlines</I></P><UL><LI> Mo is responsible for our registration to the jamboree.
</LI><LI> Miriam is responsible for our safety page. She has nearly finished the texts so everyone has to review the page.
</LI><LI> Gabi and Robert will write the abstract.
</LI></UL><UL><LI> Next week René Röspel, member of the german parliament, will visit us for a discussion about iGEM and Synthetic Biology.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_August_27th">Monday August 27th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Digest of our new plasmid pSB1C3 with pT7 promoter (hopefully, cause we have to wait for sequencing results). Dephosphorylation of digested vector and purification over agarose gel. After that, the backbone was ligated with ecol_HIS, tthl_HIS, bpul_HIS and bhal_HIS inserts. 
</LI><LI>Again ligation of J231110 in pSB1C3 backbone.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Colony-PCR <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>, <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> and <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A> with no positive result at all.
</LI><LI>Restriction of pSB1C3+<A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A> with <I>Pst</I>I and <I>Age</I>I followed by dephosphorylation.
</LI><LI>Restriction of pSB1C3+RFP with <I>Xba</I>I and <I>Pst</I>I, followed by dephosphorylation.
</LI><LI>Restriction of <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>-PCR-product with <I>Xba</I>I and <I>Pst</I>I
</LI><LI>Restriction of <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A> and <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>-PCR-products with <I>Xba</I>I and <I>Age</I>I
</LI><LI>Transformed the <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A>-ligation into KRX and plated cell on selection-agar. 
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Digestion of two tvel10-plasmids with <I>Bam</I>HI and <I>Pst</I>I to define direction of the insert. One is positive, one negative.
</LI><LI>Made the positive tvel10-plasmid ready for sequencing.
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> We exchanged the buffer of the purified ECOL from 08/14, which showed a promising band in the SDS-Page.
</LI><LI> Made a flask cultivation of  <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A> with positive (<A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>) and negative control (without plasmid).
</LI><LI>Settings: 1 L flask without baffles, autoinduction medium, final volume: 250 mL, 60 µg/mL chloramphenicol, 37 °C, 120 rpm, durance: 12 hours, single determination
</LI><LI> Cells from today's cultivation were disrupted via sonication and laccase was purified by using the HisTrap column.
</LI><LI> Made precultures of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>. As negative control we used <I>E. coli</I> KRX and as positive control we used <I>E.coli</I> KRX with <A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>.
</LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Figure 1:</B> Comparison of TVEL0 laccase stored in a refrigerator or in a freezer. Activity measurement was done by measuring oxidized ABTS via optical density 420.</DIV></DIV></DIV><UL><LI><B>Team Activity Tests</B>: 
</LI><LI> There have been some discussions about the storage of our laccases. It was said that the freezing of laccase (once deluted) might cause increased enzymatic activity. To find out whether storing the enzymes in the refrigerator or in the freezer is a better choice we compared to different samples from TVEL0. One was stored cooled and one was frozen and then measured as usual. As shown in figure 1 the frozen laccases showed an increased saturation curve and also reached the saturation slower. This seems like a clear hint to better store our enzymes in the refrigerator.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_August_28th">Tuesday August 28th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Ligation of pSB1C3 with promoters J23117, J23110 and J23103. The promoters (which are again primer pairs we  annealed) were diluted 1:1000 (0,1 pmol) before and boiled with 98°C and slowly cooled down for ligation of the forward and the reverse primers. 
</LI><LI>Colony PCR of ligated products with T7 promoter and insert. The PCR showed for the tthl_HIS and for ecol a band in agarose gel on the correct height. So we picked this colonies and plated them to isolate the plasmids.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Gel-clean up of the digested pSB1C3-backbone.
</LI><LI>Colony-PCR of <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> and <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A> with four positive colonies of <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>.
</LI><LI>Plasmid-isolation of one <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A> followed by restriction with no positive result.
</LI><LI>Plated another <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>-colony for plasmid-isolation.
</LI><LI>Plated small dots of 25 colonies of the <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A>-transformation to look if they are red.
</LI><LI>Plasmid isolation of &lt;partinfo&gt;J61101&lt;/partinfo&gt; and digestion with <I>SpeI</I> and <I>Pst</I>I
</LI><LI><B>Team Fungal and Plant Laccases:</B></LI><LI>Digest of tvel35 PCR product for cloning in pSB1C3.
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> The band appearing in the SDS-PAGE of the cultivation of the 08/14 was analysed via MALDI-TOF and we got a positive result: we got our laccase!
</LI><LI> We made a SDS-PAGE from yesterday's cultivation and got the same band for cultivation of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, so we reproduced our result:) It seems, that the purification and the higher temperature had the essential influence on the production. 
</LI><LI> Started a new flask cultivation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A> with positive (<A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>) and negative control (<I>E. coli</I> KRX without plasmid). 
</LI><LI>Settings: 300 mL flasks without baffles, final volume: 60 mL, autoinduction medium, 0,35 mM CuCl<SUB>2</SUB>, 37 °C, 120 rpm, durance: 12 hours
</LI><LI> Cells from today's cultivation were disrupted via sonication and laccase was purified by using the HisTrap column. This time the column was better cleaned by using the twofold volume of 500 mM imidazol.
</LI><LI><B>Team Activity Tests:</B></LI><LI> Today we received even more samples from Team Cultivation. The interesting fact about those samples is that the cells that produced those laccases have been cultivated with CuCl<SUB>2</SUB>. We compared different measurements setup to see what the influences of different environments on the laccase can cause regarding enzyme activity. Figure 1 shows the activity of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863015" class="external text" rel="nofollow">XCCL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863020" class="external text" rel="nofollow">BHAL</A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> in H<SUB>2</SUB>O and with previous incubation of 0.4 mM CuCl<SUB>2</SUB>. Figure 2 shows the laccase activity in H<SUB>2</SUB>O without previous CuCl<SUB>2</SUB> incubation. In both cases the enzyme activity is comparable. This let´s us conclude that their is no distinct difference between adding CuCl<SUB>2</SUB> during cultivation of before activity measurements. Figure 3 and 4 show the results of the same laccase that have not been rebuffered in H<SUB>2</SUB>O but measured in an imidazole buffer that is used during purification. Only the <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> laccase shows activity in the imidazole buffer. 
</LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Figure 1:</B> Activity measurements via oxidized ABTS and OD<SUB>420</SUB> of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863015" class="external text" rel="nofollow">XCCL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863020" class="external text" rel="nofollow">BHAL</A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> in H<SUB>2</SUB>O. The enzymes were incubated in 0.4 mM  CuCl<SUB>2</SUB> before measurements.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Figure 2:</B> Activity measurements via oxidized ABTS and OD<SUB>420</SUB> of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863015" class="external text" rel="nofollow">XCCL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863020" class="external text" rel="nofollow">BHAL</A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> in H<SUB>2</SUB>O. The enzymes were not incubated in CuCl<SUB>2</SUB> before measurements.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Figure 3:</B> Activity measurements via oxidized ABTS and OD<SUB>420</SUB> of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863015" class="external text" rel="nofollow">XCCL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863020" class="external text" rel="nofollow">BHAL</A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> in imidazole. The enzymes were incubated in 0.4 mM  CuCl<SUB>2</SUB> before measurements.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Figure 4:</B> Activity measurements via oxidized ABTS and OD<SUB>420</SUB> of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863015" class="external text" rel="nofollow">XCCL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863020" class="external text" rel="nofollow">BHAL</A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> in imidazole. The enzymes were not incubated in CuCl<SUB>2</SUB> before measurements.</DIV></DIV></DIV><P>We sadly agree that rebuffering is necessary to ensure that our laccases are happy and active.
</P><UL><LI><B>Team Substrate Analytic:</B></LI><LI> Dominik installed the new column into the HPLC.
</LI><LI> We tryed again to mesure the three estrogens,but we recived the same result as with the old column.
</LI><LI> We lowerd the oven temperature to 30 °C. With this settings we could determin the retention times 5.8 minutes for Estradiol, 4.7 minutes for Estrone and 5.2 minutes for Ethinyl estradiol.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_August_29th">Wednesday August 29th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Restriction of ecol_HIS, bpul_HIS, tthl_HIS and bhal_HIS PCR products estimated for the hundredth time.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Plasmid-isolation of two plated <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A>- and the <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>-coloniy and test-digestion of the plasmid with <I>Not</I>I showed fragments of the correct size.
</LI><LI>Colony-PCR of <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A>-colonies brought up no positive colonies.
</LI><LI>To have a comparison to our His-tagged-GFP-fusion-protein we decided to use an exact copy of the His-tagged-GFP. Therefore we made primers to add the His-tag the &lt;partinfo&gt;I13522&lt;/partinfo&gt;.
</LI><LI>Made gradient-pcr with &lt;partinfo&gt;I13522&lt;/partinfo&gt; as template and the GFP_FW and GFP_His_compl primers. the most correct product was at 62°C.
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>New Primers for xccl arrived.
</LI><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Made SDS-PAGES of yesterday's cultivation, but it was too pale, so it had to be repeated
</LI><LI> Made SDS-PAGES-repeat of yesterday's cultivation. 
</LI></UL><CENTER><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 2:</B> SDS-PAGE of of purification from cultivation (08/28) of  <I>E.coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A> with positive (<A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>) and negative control (without plasmid). 250 mL in 1 L flasks without baffles, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> with 60 µg mL<SUP>-1</SUP> chloramphenicol at 37 °C for 12 hours. Purification of the supernatant via <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Production#Syringe_method" class="external text" rel="nofollow">syringe method</A>. The arrow shows ECOL (53 kDa).</DIV></DIV></DIV></DIV></CENTER><UL><LI> Installation of the two 3 L fermenters used for practical courses in our lab. Had to search for all of the components with the help of members of fermentation group and test electrodes.
</LI><LI> Made precultures of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A> as well as of positive (<A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>) and negative control (<I>E. coli</I> KRX without plasmid).
</LI></UL><LI><B>Team Substrate Analytic</B>: 
</LI><LI>Marcus Persicke has detected the Estrone and Estradiol limit.
</LI><LI>Dominic installed the arived columns from Knauer into the HPLC. 
</LI><LI>We prepared samples in the same way as on the 14te August.
</LI><LI>But again, despite the new column all samples had the same retention time, as the solvent. So we reduced the oven tempreture to 30°C. With this settings we´ve got the 5min as retention time for E1 5,8min as retention time for E2 and 5,5 as retention time for EE.
</LI><H3><SPAN class="mw-headline" id="Thursday_August_30th">Thursday August 30th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Ligation of the digested PCR products from the day before in BBa_J61101_pSB1A2 plasmid backbone which we transformed from distribution plate before.
</LI><LI> Ligation of J23110 and J23103 (annealed primers) in pSB1C3 backbone.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Digest of tvel35 again and ligation in pSB1C3.
</LI><LI> PCR of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K500002" class="external text" rel="nofollow">BBa_K500002</A> with K500002_FW and K500002_RV primers.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Pooled the six good and the six other PCR-fractions with producd of <A href="http://partsregistry.org/Part:BBa_K863122" class="external text" rel="nofollow">const.GFP_His</A>-gradient-PCR.
</LI><LI>Clean-Up of the PCR-products
</LI><LI>Start of Restriktion GFP-His6tag with EcoRI, PstI and DpnI in O-Buffer .
</LI><LI>Colony-PCRs of <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> - All negativ.
</LI><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Started another flask cultivation, the repetition of the cultivation on monday in a smaller scale but with a double determination. Cultivation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A> with positive (<A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>) and negative control (<I>E. coli</I> KRX without plasmid). 
</LI><LI>Settings: 300 mL flasks without baffles, autoinduction medium, final volume: 60 mL, 37 °C, 120 rpm, 12 hours
</LI></UL><LI> Continuing Installation of the two 3 L fermenters.
</LI><LI> Starting the first fermentation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A></LI><LI>Settings: fermenter: Braun, autoinduction medium, final volume: 3 L, 37 °C, stirrer on cascade to hold a pO<SUB>2</SUB> of 50 %, airflow: 2 NL/m,  12 hours. We had some problems with the controllation of settings.
</LI><LI> Made precultures of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A> as well as of positive (<A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>) and negative control (<I>E. coli</I> KRX without plasmid).
</LI><LI><B>Team Substrate Analytic:</B></LI><LI> 100 µg ml<SUP>-1</SUP> and 10 µg ml<SUP>-1</SUP> Estradiol is to high concentrated, to integrate the peak area. Concentrations around 1 µg ml<SUP>-1</SUP> seems to be ideally for our HPLC.
</LI><H3><SPAN class="mw-headline" id="Friday_August_31th">Friday August 31th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> Colony-PCRs of the ligations with J23100 + J61101 + ecol_HIS, bpul_HIS, tthl_HIS and bhal_HIS. Positive colonies for for ecol_HIS and tthl_HIS and bpul_HIS were plated on LB + AMP.
</LI><LI> We isolated plasmids with pSB1C3 + promoter J23117 from a former ligation and transformation.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Purification of PCR product <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K500002" class="external text" rel="nofollow">BBa_K500002</A>.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Ligation of <A href="http://partsregistry.org/Part:BBa_K863122" class="external text" rel="nofollow">const.GFP_His</A> with a pSB1C3 and transformation of KRX with the product of the ligation.
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>sequencing of tvel10-plasmid showed a mutation at base 1573, altering the second last amino acid from threonine to asparagine
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Cells of the yesterday's flask cultivation and a sample of equal size of <I>E. coli</I> KRX fermentation with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> were disrupted via sonication and ECOL was purified by using the HisTrap column. The purificated sample of the fermentation did not show any activity, so we decided not to purify the whole sample.
</LI><LI> First fermentation of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A></LI><LI>Settings: fermenter: Braun Biostat B, autoinduction medium, final volume: 3 L, 37 °C, stirrer on cascade to hold a pO<SUB>2</SUB> of 50 %, airflow: 2 NL/m,  12 hours.
</LI><LI> A growth kinetic was determined by measuring the OD<SUB>600</SUB> every 30 minutes.
</LI></UL><UL><LI><B>Team Activity Tests:</B></LI><LI> Yay, a new job for us! Again we receives samples from all our currently available laccase for some activity test. This times the cells were not cultivated with CuCl<SUB>2</SUB>. However we incubated half of the samples with 0.4 mM CuCl<SUB>2</SUB> and measured as usual. As seen in Figure 1 and 2 there is activity for both samples of <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> if incubated with CuCl<SUB>2</SUB> or not before measurements.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Figure 1:</B> Activity measurements of two samples each of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863015" class="external text" rel="nofollow">XCCL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863020" class="external text" rel="nofollow">BHAL</A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A>. KRX (cells without plasmid) and ligA (cells with a ligase A plasmid for the control of induction) have been used as a negative control. Enzymes have been incubated with 0.4 mM CuCl<SUB>2</SUB> for 2h.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Figure 2:</B> Activity measurements of two samples each of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863015" class="external text" rel="nofollow">XCCL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863020" class="external text" rel="nofollow">BHAL</A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A>. KRX (cells without plasmid) and ligA (cells with a ligase A plasmid for the control of induction) have been used as a negative control.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Saturday_September_1st">Saturday September 1st</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI></UL><DL><DD><UL><LI> Plasmid isolations on the plated colonies and control restriction with <I>Not</I>I. Control digest showed for <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">J23100 + J61101+ tthl_HIS</A>, J23100 + J61101 + ecol_HIS positive bands. So we sent these plasmids for sequencing.
</LI></UL></DD></DL><UL><LI><B>Team Fungal and Plant Laccases:</B> Plating two colonies from tvel35+pSB1C3 transformation on LB +CM agar.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Got green colonies on <A href="http://partsregistry.org/Part:BBa_K863122" class="external text" rel="nofollow">const.GFP_His</A>-dish. Picked three and plated them on CM-select-agar.
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Harvesting and centrifugation culture of fermentation on 08/31 (<I>E.coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>). Store pellet at 4 °C.
</LI></UL><UL><LI><B>Team Substrate Analytic:</B></LI><LI> Dominik instructed us, that we could operate the HPLC on our own. We also learnd how to clean the HPLC vials.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_September_2nd">Sunday September 2nd</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Control digest of tvel35 in pSB1C3 was positive. 
</LI><LI>Plasmid isolation from pSB1C3 + J23110, pSB1C3 + J23103 and J23100 + J61101+ bpul_HIS. Control digest showed correct bands for J23100 + J61101+ bpul_HIS, pSB1C3 + J23103 and pSB1C3 + J23110. 
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>unsuccessful Colony-PCR of <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A>.
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Gradient-PCR of tvel10-plasmid with tvel10-t243g primer-mix. 
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Made precultures of <I>E. coli</I> KRX containing either <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863121" class="external text" rel="nofollow">BBa_K863121</A>.
</LI></UL><UL><LI><B>Team Activity Tests:</B></LI><LI> This week we had Team Modeling over and they told us about their concerns. To continue modeling they wanted to have a look at the activity of our laccase from <I>T. versicolor</I> but with different ABTS concentrations. Especially the were interested in the first time points after adding ABTS. This should give them enough information to calculate the enzyme activity. We didn't want to wait, so we started immediately with our standard activity test. Our tested ABTS concentrations were: 0.5µl, 1µl, 2µl, 4µl and 8µl. We got nice activity curves but also noticed, that the activity saturated quickly and therefore the initial activity of our laccase can not be measured accurately. Of course Team Modeling got our data just in time, but we also want to start new activity tests with half of the amount of laccase. So we are still trying to keep our lovely Team Modeling satisfied.
</LI></UL><DIV id="anzeige"><H1>Summary of Week 19</H1><H2><SPAN class="mw-headline" id="Week_19_.2809.2F03_-_09.2F09.2F12.29">Week 19 (09/03 - 09/09/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_19_.2809.2F03_-_09.2F09.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 19 (09/03 - 09/09/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_September_3rd"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday September 3rd</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_September_4th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday September 4th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_September_5th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday September 5th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_September_6th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday September 6th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_September_7th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday September 7th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_September_8th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday September 8th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_September_9th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday September 9th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_18"> Weekly Seminar</SPAN></H3><UL><LI> We added a band for our sponsors on every wiki-page.
</LI><LI> On thursday we will meet with René Röspel, member of the german parliament.
</LI><LI> We are invited to the annual reception of the University Bielefeld on the 5th of october. Well that won't work for us.
</LI><LI> We still are working on a design for our team t-shirt.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_September_3rd">Monday September 3rd</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> We sent different plasmids for sequencing. 
</LI><LI> We got another <I>E.coli</I> strain (Rosetta-gami 2) from our supervisor Dr. Christian Rückert. The cells carry a chloramphenicol-resistant plasmid, pRARE2, which supplies tRNAs for seven rare codons, AUA, AGG, AGA, CUA, CCC, GGA and CGG under the control of their native promoter. We want to use this strain for the expression of TTHL and BHAL hopeful that some rare codons are the reason that the laccases can’t be expressed by now. So we need the genes under control of a constitutive promoter because this strain has no T7 polymerase and we can’t use our inducible constructs. Further the plasmids need to have AMP resistance, because otherwise we can't select on positive transformed cells. Hopefully the plasmids, we sent for sequencing are ok, so we can use them.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> A autoinduced culture of KRX with the <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>-plasmid does not seem to produce the protein (no green fluorescence).
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Clean-up of tvel10-t243g-PCR
</LI><LI><I>pfu</I>-gradient-PCR of xccl-plasmid with xccl-primers with no product.
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Made one half of the SDS-PAGES of the flask cultivation from 08/30.<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 1:</B> SDS-PAGES of purification from cultivation (08/28, smaller scale but with a double determination) of <I>E.coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A> with positive (<A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>) and negative control (<I>E.coli</I> KRX without plasmid). 60 mL in 300 mL flasks without baffles with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>, 60 µg mL<SUP>-1</SUP> chloramphenicol at 37 °C for 12 hours. Purification of the supernatant via <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Production#Syringe_method" class="external text" rel="nofollow">syringe method</A>. The arrow shows ECOL (53 kDa)</DIV></DIV></DIV></DIV></LI><LI> Work on culture of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> (08/31) was continued and we performed the cell disruption by high-pressure homogeniser in 3 cycles. Then it was purificated by using a Ni NTA column. The product was given to the activity team for analysis.
</LI><LI> Fermentation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863121" class="external text" rel="nofollow">BBa_K863121</A> for the cellulose binding team. 
</LI><LI>Settings: fermenter: Braun, autoinduction medium, 60 µg/mL chloramphenicol, 3 L, 37 °C, stirrer on cascade to hold a pO<SUB>2</SUB> of 50 %, airflow: 2 NL/m. This time we cultivated only for 9 hours and then decreased the temperature at 20 °C for a better folding of the protein for 1 hour. We had some problems with cascade at the end of the fermentation. 
</LI></UL><LI> Cells of today's fermentation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863121" class="external text" rel="nofollow">BBa_K863121</A> were disrupted via high-pressure homogeniser and purified by using the Ni NTA column. The flow-through fluoresced very strong, so that not that much of the protein had bound to the column. Conclusion: The column had to be reloaded.
</LI><LI> Fermentation of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>.
</LI><LI>Settings: fermenter: Braun, autoinduction medium, 60 µg/mL chloramphenicol, 3 L, 37 °C, stirrer on cascade to hold a pO<SUB>2</SUB> of 50 %, airflow: 2 NL/m, 9 hours. We only cultivated 9 hours, because we saw that a great amount of our cells already died. 
</LI><LI> Made precultures of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> and one containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863113" class="external text" rel="nofollow">BBa_K863113</A>.
</LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 2:</B> Activity in oxidizing ABTS of our produced <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> laccase depending on time. Values are calculated by taking the average out of 4 measurements (n=4).</DIV></DIV></DIV><UL><LI><B>Team Activity Tests:</B></LI><LI>Today our hard-working Team Cultivation offered us some <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> laccases. After fermentation and Ni NTA purification the laccase was still hanging around in imidazole buffer, so we rebuffered it into nanopure water. We incubated the sample with copper and considered to measure the protein concentration before running the activity tests. Using the protocol for determining the protein concentration via Bradford (standards were 1 mg/ml, 1.5 mg/ml and 2 mg/ml BSA) we got a final <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> laccase concentration of 0.22 mg/ml. With this information we designed the following activity test different as usual. As a positive control we took 140 µl of our bought laccase TVEL0 with a concentration of 0,021 mg/ml. Accordingly to this concentration we applied 14 µl of the <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> laccase to get approximately the same amount of laccase into each well. Additionally we tested 140 µl of the <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> laccase to make sure that we will definitely see any activity. After measuring the samples the whole night we got our results in the morning. It turned out, that our store bought laccase TVEL0 was rapidly active as seen before in our activity tests. The <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> laccase was also active. Using 140 µl of it led to the maximal state of oxidation of ABTS after 45 minutes. With the reduced amount of 14 µl of laccase this optimum was reached after 90 minutes. For our following activity tests we want to include the protein concentration again.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_September_4th">Tuesday September 4th</SPAN></H3><UL><LI> We signed in for our tracks today! Our first choice is the 'Environment' track since it is obvious that we want to clean up drinking water and therefore help our environment. The second choice is the 'New Application' track. We are hoping our filter system can be applied in wastewater treatment plants and establishes a new area of water purification.
</LI></UL><UL><LI><B>Team Shuttle Vector:</B></LI><LI>PCR Clean up was done for tAOX1 Part <A href="http://partsregistry.org/Part:BBa_K863203" class="external text" rel="nofollow">K863203</A>.
</LI><LI>PCR of the fragment of tAOX1 for the shuttle vector was done with addition of betain (10 µL) and DMSO (1 µL) refered to 25 µL sample volume.
</LI></UL><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> Digest of pSB1C3 + J23110 and pSB1C3+ J23103 for suffix insertion of the laccase genes. 
</LI><LI> Digest of the different laccase ORFs for cloning in pSB1C3 backbone without any regulatory elements. Ligation of tthl, bhal, ecol and bpul in pSB1C3.
</LI><LI> Transformation of ligations from day before (J23100+J61101+pSB1A2+bhal_His).
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> plated six colonies of tvel-t243g for plasmid-isolation.
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Second half of the SDS-PAGES of the flask cultivation from 08/30 was made.<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 3:</B> SDS-PAGES of purification from cultivation (08/28, smaller scale but with a double determination) of <I>E.coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863010" class="external text" rel="nofollow">BBa_K863010</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863015" class="external text" rel="nofollow">BBa_K863015</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863020" class="external text" rel="nofollow">BBa_K863020</A> with positive (<A href="http://partsregistry.org/Part:BBa_K525710" class="external text" rel="nofollow">BBa_K525710</A>) and negative control (<I>E.coli</I> KRX without plasmid). 60 mL in 300 mL flasks without baffles with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>, 60 µg mL<SUP>-1</SUP> chloramphenicol at 37 °C for 12 hours. Purification of the supernatant via <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Production#Syringe_method" class="external text" rel="nofollow">syringe method</A>.</DIV></DIV></DIV></DIV></LI><LI> After reloading we repeated the purification of BPUL produced by fermentation on 08/31. Therefore the flowthrough and the washing fraction were used. The different fractions should be tested by the activity team.
</LI><LI> A new fermentation of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> was started 
</LI><LI>Settings: fermenter: Infors, autoinduction medium, 60 µg/mL chloramphenicol, 3 L, 37 °C, stirrer on cascade to hold a pO<SUB>2</SUB> of 50 %, airflow: 2 NL/m, 9 hours. We shorten the time, because the effect should be similar to the production of BPUL.
</LI></UL><LI> The fermentation of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> and the one containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> were harvested and after centrifugation stored at 4 °C, because our high-pressure homogeniser is out of work.
</LI><LI> Made a flask cultivation for the cellulose binding team: cultivation of <I>E. coli</I> KRX containing the &lt;partinfo&gt;BBa_K863103&lt;/partinfo&gt;-plasmid to express a cellulose binding domain (CBD) fused to a His-tagged GFP.
</LI><LI>Settings: 1 L flask with baffles, autoinduction medium, 60 µg/mL chloramphenicol, final volume: 250 mL, 37 °C, 120 rpm, 9 hours. We made a three-fold determination, but did not get fluorescing cultures. The problem is to purify this product, because it might be impossible to remove the protein from the column. If it binds via CBD and not via His-tag.
</LI><LI> Made precultures of <I>E. coli</I> KRX containing plasmids with laccases from ECOL from <I>E. coli</I> BL21 (DE3), <I>B. pumilus</I> DSM27 and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A> behind the constitutive promoter J23100 as well as one containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A></LI><UL><LI><B>Team Substrate Analytic:</B></LI><LI> Today we let TVEL0 degrade Estradiol and Estrone. We prepared seven 2 mL Eppendorf tubes with: 425 µL BR-buffer, 50 µL TVEL0 and 25 µL Estradiol. To stop the reaction we added 1,5 mL MeOH. The Estradiol concentration should be 2 µg ml <SUP>-1</SUP>. We stopped at houre 0, 1, 2, 3, 4, 5, and over night, and mesured the Estradiol concentration with the HPLC and the LCMS.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_September_5th">Wednesday September 5th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Transformation of tthl, bhal, ecol and bpul in pSB1C3.
</LI></UL><LI><B>Team Cultivation &amp; Purification:</B><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 4:</B> SDS-PAGE of purification from our first fermentation of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> (08/31).  3 L fermenter (Infors), 37 °C, pO<SUB>2</SUB> of 50 % for 12 hours. Purification of the supernatant via <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Production#Syringe_method" class="external text" rel="nofollow">syringe method</A>. The arrow shows BPUL (58.6 kDa) in lane 5-8 (elution 1 and 2).</DIV></DIV></DIV></LI><LI> Performed the cell disruption of yesterday's flask cultivation to produce CBD-GFP+His fusion protein via sonication and made a SDS-Page of it. The problem is to purify this product, because it might be impossible to remove the protein from the column. If it binds via CBD and not via His-tag.
</LI><LI> Made a flask cultivation of <I>E. coli</I> KRX containing new BioBricks: Laccase ECOL from <I>E. coli</I> BL21 (DE3), BPUL from <I>B. pumilus</I>DSM27 and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A> behind the constitutive promoter J23100. 
</LI><LI>Settings: 100 mL flask without baffles, autoinduction medium, 60 µg/mL chloramphenicol, final volume: 60 mL, 37 °C, 120 rpm, 12 hours, double determination. 
</LI><LI> We repeated the purification of His-tagged GFP by using the flow-through and the washing fraction, but it did not work. Then we got to know that it could not work, because the sequencing showed us that the GFP was not tagged to His.
</LI><LI> Made a SDS-Page of the fractions of purified <I>B. pumilus</I> laccase produced at 08/31.
</LI><LI> Starting of another fermentation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>.
</LI><LI>Settings: fermenter: Braun, autoinduction medium, 60 µg/mL chloramphenicol, 3 L, 37 °C, stirrer on cascade to hold a pO2 of 50 %, airflow: 2 NL/m, 6 hours. Found out that we got a maximal optical density after 4 hours, after which the cells start to die. Therefore we harvested the cells after 6 hours.
</LI><LI>Aim: Getting a growth kinetics. 
</LI><LI> Made preculture of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A></LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 5:</B> TVEL0 oxidizing different concentrations of ABTS over a time period of 5 minutes (n=4).</DIV></DIV></DIV><UL><LI><B>Team Activity Tests:</B></LI><LI>Today we had to deliver some data for Team Modeling. To calculate enzyme kinetics of our laccases the initial reaction speed has to be determined. With our usual approach using 0.1 U TVEL0 Laccase we couldn't accurately measure the first minute of reaction with ABTS because everything went to fast. So we cut down the volume of TVEL0 laccase from 140 µL to 35 µL (0.025 U) and measured the oxidizing potential of TVEL0 with different ABTS concentrations. The results are shown in Fig. 5. There you go, Team Modeling! Make the best out of it!
</LI></UL><UL><LI><B>Team Substrate Analytic</B>: 
</LI><LI> The degradations from the 4th September were stopped with methanol. The samples were evaporated for six hours and diluted in 50 % - 50 % acetonitril - water. Then we used the C18E-columns for purifying the samples. The measuring with LC-MS was done from Marcus Persicke.
</LI><LI> It´s time to regenerate the column. Dominik showed us the protokol.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_September_6th">Thursday September 6th</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 6:</B> SDS-PAGE of purification from our first fermentation of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> (08/31). 3 L fermenter (Infors), 37 °C, pO<SUB>2</SUB> of 50 % for 12 hours. Purification of the supernatant via Talon column. The arrow shows BPUL (58.6 kDa) in lane 4 and 5 (fraction 7 and 8).</DIV></DIV></DIV><UL><LI><B> Team Cloning of Bacterial Laccases:</B></LI><LI> Colony PCR on <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863022" class="external text" rel="nofollow">J23100 + J61101 + bhal_HIS</A> showed four positive colonies which were plated on LB with CM + AMP.
</LI><LI> Plating colonies from colony PCR on bpul+ pSB1C3 and tthl + pSB1C3 on new plates.
</LI><LI> We still have to wait for the sequencing results but control restrictions suggests that we have tthl_HIS, bhal_HIS, ecol_HIS and bpul_HIS in a vector with a constitutive promoter. So we transformed the plasmids in <I>E.coli</I> Rosetta Gami 2. The cells carry a chloramphenicol-resistant plasmid, pRARE2, which supplies tRNAs for seven rare codons, AUA, AGG, AGA, CUA, CCC, GGA and CGG under the control of their native promoter.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> gradient-PCRs to generate <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A> and <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos_Freiburg</A> on the orignial plasmids with the  CBDcex_Freiburg + CBDclos_Freiburg and 2AS_Freiburg-Reverseprimers. Only the <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A>-product could be generated (68,5°C is best).
</LI><LI> two colonies of <A href="http://partsregistry.org/Part:BBa_K863122" class="external text" rel="nofollow">const.GFP_His</A> plated on selection-agar for plasmid-isolation.
</LI></UL><UL><LI><B> Team Cultivation &amp; Purification:</B></LI><LI> Made a SDS-PAGE of cultivation from 08/31.
</LI><LI> Fermentation of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> to get a growth kinetics 
</LI><LI>Settings: fermenter: Infors, autoinduction medium, 60 µg/mL chloramphenicol, 3 L, 37 °C, stirrer on cascade to hold a pO2 of 50 %, airflow: 2 NL/m.
</LI></UL><LI> Preparation of the bigger fermenter (7 L) for the first time
</LI><LI> Performing the cell disruption via sonification and the purification via HisTrap column supported by Äkta of the flask cultivation from 09/05.
</LI><LI> We got to know that the working group of physical chemistry has another homogeniser, that we could use. We performed the cell disruption via high-pressure homogeniser of fermentations from 09/03, 04 and 05.
</LI><LI> Made preculture of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A></LI><LI><B> Team Substrate Analytic:</B></LI><LI> The fluorescent lamp is broken. So it was not possible to mesure today.
</LI><H3><SPAN class="mw-headline" id="Friday_September_7th">Friday September 7th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI> Plasmid isolation of different plated colonies and control digest. The restriction showed positive results for the plasmids for <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863001" class="external text" rel="nofollow">bpul+pSB1C3</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863011" class="external text" rel="nofollow">tthl + pSB1C3</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863022" class="external text" rel="nofollow">bhal_His + J23110 + RBS + pSB1C3</A>. So we sent the plasmids for sequencing.
</LI><LI> Transformation of &lt;partinfo&gt;BBa_K863012&lt;/partinfo&gt; and  &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; in <I>E. coli</I> Rosetta-Gami 2.
</LI></UL><UL><LI><B>Team Shuttle Vector:</B></LI><LI>Designed and ordered sequencing primers for the shuttle vector. We set a primer every 700 to 750 bases.
</LI><LI>Now all fragments are gel purified and the second Gibson assembly could be started. The fragments were mixed equimolar as described below with 15 µL of the Gibson master mix:
</LI></UL><TABLE class="wikitable" style="text-align: center; width: 300px: height: 300px;" border="0}"><TBODY><TR><TH> Fragment
</TH><TH> Quantity [ng]
</TH><TH> Volume [µL]
</TH></TR><TR><TD>5AOX1
</TD><TD>26.5
</TD><TD>0.44
</TD></TR><TR><TD>MFalpha1
</TD><TD>8.3
</TD><TD>0.11
</TD></TR><TR><TD>3AOX1
</TD><TD>19.4
</TD><TD>0.44
</TD></TR><TR><TD>HIS4
</TD><TD>73.0
</TD><TD>2.16
</TD></TR><TR><TD>pSB1C3
</TD><TD>55.44
</TD><TD>1.54
</TD></TR><TR><TD>tAOX1
</TD><TD>8.9
</TD><TD>0,11
</TD></TR></TBODY></TABLE><P>After incubation for 1 h at 50 °C, 2 µL were transformed in <I>E. coli</I> KRX cells by electroporation. And plated on selective LB Cm plates.
</P><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Today we performed lots of purifications. This time we used a Ni-NTA column, because it seemed to work better than the TALON column. Therefore it was reloaded newly before starting the purifications. The purified samples were BPUL produced on 09/03 and 09/05 as well as ECOL produced on 09/04. The results were not that promising, so maybe we harvested the cells too early. They start dying more or less when we expect them to produce our (toxic) protein. So we think we had to let some of them die, so that the rest will produce our protein. The next fermentations should endure at least 12 hours.
</LI><LI> Fermentation of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> within the Bioengineering NFL22 fermenter.
</LI><LI>Settings: fermenter: Biostat NFL22 (7 L), autoinduction medium, 60 µg/mL chloramphenicol, final volume: 6 L, 37 °C, stirrer increased 2 % if the pO<SUB>2</SUB> got below 30 %, airflow: 5 NL/m, 12 hours.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_September_8th">Saturday September 8th</SPAN></H3><UL><LI><B>Team Shuttle vector:</B></LI><LI>Colonies were picked and Colony PCR was done. Some positive clones could be identifed and were plates out for plasmid isolation.
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Harvesting and centrifugation of fermentation of 09/07. Pellet stored at 4 °C.
</LI><LI> Made precultures of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> and of <I>E. coli</I> Rosetta-Gami 2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A> and laccase from <I>B.halodurans</I>&lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt;.
</LI></UL><UL><LI><B>Team Activity Tests:</B> Today we managed to rebuffer the <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> samples given by Team Cultivation. We got fraction 3 to 7 and 15 to 19 of the purification, because they have shown a nice peak. After bringing the <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> laccases into water, we incubated with 0.5 mM CuCl<SUB>2</SUB> and measured the protein concentration via Bradford. The offered fractions contained the following protein concentrations after rebuffering: 0.0087 mg/mL (fr. 3), 0.0110 fr. 4), 0.0632 (fr. 5), 0.1672 (fr. 6), 0.1229 (fr. 7), 0.0060 (fr. 15), 0.0034 (fr. 16), 0.0326 (fr. 18), 0.0225 (fr. 19). The protein amount in fraction 17 couldn't be measured but we didn't leave it behind. We adjusted the protein concentrations to the concentration of TVEL0, but did also measurements using 140 µL of each sample, because the protein concentrations do not only represent gained laccases. The results of our plain activity measurement of the samples are demonstrated in Fig. 7 and Fig. 8 (positive and negative control shown in Fig. 3):
</LI></UL><CENTER><DIV class="thumb tleft"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption">Fig. 7: <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> activity of fractions 3 to 7 at 25°C with 0.01 mM ABTS (n=4).</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption">Fig. 8: <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> activity of fractions 15 to 19 at 25°C with 0.01 mM ABTS (n=4).</DIV></DIV></DIV></CENTER><P>Fractions 6 and 7 show the highest activity leading to the assumption, that faction 7 contains as much protein as fraction 6 and that this amount is almost represented through <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> laccase.
</P><CENTER><TABLE><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig. 9: Positive control represented through the activity of TVEL0. Negativ control with 0.5 mM CuCl in H<SUB>2</SUB>O (n=4).</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig. 10: Comparison of the <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> activity depending on CuCl incubation (n=4).</DIV></DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig. 11: Effects on <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> by rebuffering from imidazole buffer in H<SUB>2</SUB>O (n=4).</DIV></DIV></DIV></TD></TR></TBODY></TABLE></CENTER><P>Additionally we were interested in the impact on copper incubation. We have chosen fr. 18 and compared the activity of a copper incubated and a not copper incubated sample. With a incubation time of 2 hours with 0.5 mM CuCl<SUB>2</SUB> we can double the activity of the <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> laccase (see Fig. 10). Another interesting topic was the impact of imidazole buffer on the <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> activity. For that we tested fr. 7 and fr. 18 in imidazole buffer and rebuffered in H<SUB>2</SUB>O. The results we got were unambiguously: <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000/" class="external text" rel="nofollow">BPUL</A> in imidazole buffer didn't show much activity. Unfortunately we can't omit the rebuffering.
</P><H3><SPAN class="mw-headline" id="Sunday_September_9th">Sunday September 9th</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B></LI><LI> Plasmid from the positive clones was isolated.
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Fermentation of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> (fermenter: Infors) or containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> (fermenter: Braun) to get a growth kinetics.
</LI><LI> Settings: autoinduction medium, 60 µg/mL chloramphenicol, final volume: 3 L, 37 °C, stirrer on cascade to hold a pO<SUB>2</SUB> of 50 %, airflow: 2 NL/m, 12 hours.
</LI></UL><LI> Harvesting and centrifugation of today's 3 L fermentations.
</LI><LI> Fermentation of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A></LI><LI> Settings: fermenter: Bioengineering NFL22 (7 L), autoinduction medium, final volume: 6 L, 37 °C, pH 7 stirrer increased 2 % if the pO<SUB>2</SUB> got below 30 %, airflow: 5 NL/m, 12 hours.
</LI><LI> Flask cultivation of <I>E. coli</I> Rosetta-Gami 2 cells containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A> and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt;. 
</LI><LI> Settings: flask without baffles, LB medium, 200 mL, 60 µg/mL chloramphenicol and 300 µg/mL ampicillin, 37 °C, 120 rpm. We cultivated for 48 hours and stored the culture then overnight at 4 °C.
</LI><LI><B>Team Activity Tests:</B></LI><LI> Today Team Cultivation handed us some exciting samples that were not our well known and active <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> or <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A>. In the next days we are going to work with <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>! Our first samples were the wash and the flowthrough from purification that are supposed to be analyzed for a possible enzyme loss during this process. We checked and are happy to announce that there was only very little activity in those 12h measurements.
</LI><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><B>Figure 1:</B> Activity measurements of wash and flowthrough samples of the <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A> purification. Measurements were done via OD<SUB>420</SUB> and oxidized ABTS.</DIV></DIV></DIV></DIV><DIV id="anzeige"><H1>Summary of Week 20</H1><H2><SPAN class="mw-headline" id="Week_20_.2809.2F10_-_09.2F16.2F12.29">Week 20 (09/10 - 09/16/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_20_.2809.2F10_-_09.2F16.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 20 (09/10 - 09/16/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_September_10th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday September 10th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_September_11th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday September 11th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_September_12th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday September 12th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_September_13th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday September 13th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_September_14th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday September 14th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_September_15th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday September 15th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_September_16th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday September 16th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_19"> Weekly Seminar</SPAN></H3><UL><LI> We are discussing our lab results. Some experiments have to be repeated, but we are on a good way.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_September_10th">Monday September 10th</SPAN></H3><UL><LI><B> Team Shuttle Vector:</B></LI><LI>The PCR amplicons from the Parts: 3AOX1, tAOX1, 5AOX1, MFalpha1 and HIS4 were double digested with <I>EcoR</I>I and <I>Pst</I>I in buffer O over night at 37 °C.
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Digestion of the six tvel10-plasmids. Three were unmutated and other three hadn't lost the illegal <I>Spe</I>I-restriction-site, but their second fragment was of a smaller size.
</LI><LI>plated three additional tvel-t243g-colonies.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Sequencing results arrived: One <A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> is completely right!	
</LI><LI> Nanodropping plasmids of isolated <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A> showed that the cells did not have a plasmid at all (selection-agar did not work)
</LI><LI> Gradient-PCR with <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A> as template and <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos_Freiburg</A>-primers did work just fine (best temperature 61,7°C); Digestion with <I>Xba</I>I and <I>Age</I>I.
</LI><LI>Gel-Clean-up of <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A> and <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A></LI><LI> Digestion of <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A> with <I>Xba</I>I+<I>Age</I>I.
</LI><LI> Digestion of <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> with <I>Ngo</I>MIV and <I>Pst</I>I.
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1:</B> SDS-PAGE of purification from fermentation (09/07) of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, 6 L in NFL22, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>, 37 °C,  pO<SUB>2</SUB> 30 % for 12 hours. Purification of the supernatant via Ni-NTA column. The arrow shows ECOL (53 kDa).</DIV></DIV></DIV><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Cell disruption via high-pressure homogenizer and purification via Ni-NTA column were performed for the following samples: 3 L fermentation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> (09/09), 6 L fermentation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> (09/07) and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> (09/09).
</LI><LI> We made SDS-PAGES of purification of ECOL.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_September_11th">Tuesday September 11th</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B></LI><LI>Digest of shuttle shuttle vector with <I>Pvu</I>II and <I>Hind</I>III as control. Agarose gel looks good.
</LI><LI>The parts (HIS4, 3AOX1, tAOX1, 5AOX1, MFalpha1) were ligated in pSB1C3 and transformed in <I>E. coli</I> KRX cells.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI>PCR on tvel5 laccase for cloning in shuttle vector. Digest of shuttle vector and digest of tvel35 with <I>Aar</I>I enzyme, ligation and transformation in <I>E. coli</I> KRX cells.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Assembly of &lt;partinfo&gt;BBa_K863104&lt;/partinfo&gt;  and &lt;partinfo&gt;BBa_K863114&lt;/partinfo&gt;:
</LI><LI> Assembled <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos_Freiburg</A> and <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A> with <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> and &lt;partinfo&gt;J61101&lt;/partinfo&gt; and plated the ligation on AMP-selection-agar (because of the pSB1A2-backbone).
</LI></UL><LI> Restriction of <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos_Freiburg</A> and <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A> with <I>Eco</I>RI and <I>Pst</I>I.
</LI><LI> Ligation of CBDcex_Freiburg  and CBDclos_Freiburg with the pSB1C3-backbone and transformated and plated on selection-agar
</LI><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Made precultures of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> as well as of <I>E. coli</I> KRX.
</LI></UL><UL><LI><B> Team Activity Tests:</B> Today we got to meet <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> again. Team Cultivation gave us 8 fractions (Fraction 2-10) for activity measurements. We measured as usual and are happy to report that almost every fraction showed great OD<SUB>42</SUB> values after rebuffering in H<SUB>2</SUB>O, adjusting the protein concentration via a Bradford assay, incubation with 0.4 mM CuCl<SUB>2</SUB>and measuring with 0.1 mM ABTS, chek our Figure 1!
</LI></UL><P>[[File:Bielefeld2012 ColiFr2-10 11 09.jpg|thumb|center|<B>Figure 1:</B> Activity measurements of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> fraction 2-10 after incubation with 0.4 mM CuCl<SUB>2</SUB>. Measurements were done via oxidized ABTS and at OD<SUB>420</SUB>.]
</P><H3><SPAN class="mw-headline" id="Wednesday_September_12th">Wednesday September 12th</SPAN></H3><UL><LI><B> Team Shuttle Vector:</B></LI><LI>Colony PCR was done with the clones and picked on new plates.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Ligation of shuttle vector with tvel5 (digested with <I>Aar</I>I).
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI>Plasmid-isolation of the three tvel10-plasmids and digestion with <I>Spe</I>I showed two unmutated plasmids and one with the same wrong restriction-fragments as Monday. There must be a systematical error. <I>pfu</I>-PCR should be done again.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> There are only few colonies on all selection-agar-dishes, but none is obviously green fluorescing, even with UV-light emission could not be stimulated.
</LI><LI> Plated colonies of <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A> and <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos_Freiburg</A> to see if they are red or not.
</LI><LI> Colony-PCR of <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A> and <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos_Freiburg</A> showed eight positive colonies to the &lt;partinfo&gt;K863104&lt;/partinfo&gt;-insert. Plated the positive colonies for plasmid-isolation.
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification</B></LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 2:</B> SDS-PAGE of purification from fermentation (09/09) of <I>E. coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>, 3 L in Infors, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>, 37 °C,  pO<SUB>2</SUB> 30 % for 12 hours. Purification of the supernatant via Ni-NTA column.</DIV></DIV></DIV><UL><LI> Cell disruption via sonication and purification via Ni-NTA column were performed for the following samples: 200 mL cultivation of <I>E. coli</I> Rosetta Gami 2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; (09/09) behind a constitutive promoter.
</LI><LI> Made SDS-PAGE of the fermentation from 09/09.
</LI></UL><LI><B>Team Activity Tests:</B> It´s <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> time! We got some new and nice fractions that we tested immediately after rebuffering in H<SUB>2</SUB>O, determining and adjusting the protein concentration via Bradford and incubating with 0.4 mM CuCl<SUB>2</SUB>. The samples we got were large tubes so that this measurement was decisive for further characterization of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>. Fraction 7 was our favorite (it convinced us with its high protein concentration and great enzyme activity) so that we chose that one to be our candidate for all future measurements with <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> (see Figure 3)
</LI><LI> Additionally we went on with the measurements of <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>. The laccase was rebuffered, adjusted and incubated as well so that we can now tell that there are some fractions that do show activity! Check out Figure 4.
</LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Activity measurements of different fractions of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>. Measurements were done via oxidized ABTS and OD<SUB>420</SUB>.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Activity measurements of different fractions of <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A>. Measurements were done via oxidized ABTS and OD<SUB>420</SUB>.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Thursday_September_13th">Thursday September 13th</SPAN></H3><UL><LI><B>Team Fungal Laccases and Team Shuttle Vector:</B></LI><LI> Bad day for us. We realized that our primers we used for amplification of the laccase genes for cloning in the shuttle vector don't have overhanging ends which were needed for an optimal activity of the enzyme <I>Aar</I>I. So we had to order new primers with some bases prior to the recognition sites. But after there is little time left until wiki freeze and the new primers need some time until they are here we started a last trial for cloning this 'wrong' PCR products in the shuttle vector. The idea was to phosphorylate and ligate the ends of the PCR product from tvel5 together. Then the problem with the missing bases at the end would be solved and <I>Aar</I>I can digest the end of the PCR product. We digested the shuttle vector with <I>Aar</I>I and dephosphorylated it. After ligation we transformed the approaches in <I>E. coli</I> KRX cells.
</LI><LI> The ligation of the parts in pSB1C3 was not successful. So the Parts and the plasmid have to be digested with <I>EcoR</I>I and <I>Spe</I>I restriction enzymes.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Designed a lot of primers to cope with the expression problem. E.g. inserting a long S3N10-Linker via blunt-end-cloning between the CBD and the GFP, also primers to get rid of the His-tag on the GFP and a last set to easily change the order of CBD and GFP (via assembly but with no linker in between).
</LI><LI> Plasmid-isolation of &lt;partinfo&gt;BBa_K863104&lt;/partinfo&gt;-transformation clones
</LI><LI> Colony-PCR of <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A> and <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos_Freiburg</A> colonies. All <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos_Freiburg</A>-colonies are positive and half of the <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A>-colonies.
</LI><LI> Colony-PCR of CBDclos_F.+GFP with no positive result
</LI><LI> Colony-PCR of <A href="http://partsregistry.org/Part:BBa_K863122" class="external text" rel="nofollow">const.GFP_His</A>: 2 positive (one fluorescend); Plated both positive and one additional fluorescend.
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 3:</B> SDS-PAGE of purification from cultivation (09/09) of <I>E. coli</I> Rosetta-Gami 2 cells containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A> and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt;. 200 mL in flasks without baffles, LB medium, 60 µg/mL chloramphenicol and 300 µg/mL ampicillin, 37 °C for 48 hours. Purification of the supernatant via HisTrap column. The red arrow shows TTHL in lane 10-12 (elution 1-3).</DIV></DIV></DIV><UL><LI> SDS-Pages of the flask cultivation from 09/09 ( <I>E. coli</I> Rosetta Gami 2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt;)
</LI><LI> Fermentation of <I>E. coli</I> KRX without plasmid (fermenter: Infors) and with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> (fermenter: Braun Biostat)
</LI><LI> Settings: fermenter: Infors/Braun Biostat, final volume: 3 L, autoinduction medium, 60 µg/mL chloramphenicol added, 37 °C, stirrer on cascade to hold a pO<SUB>2</SUB> of 50 %, airflow: 2 NL/m, durance: 12 h.
</LI></UL><LI> Made preculture of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863103" class="external text" rel="nofollow">BBa_K863103</A> (CBD-GFP-His).
</LI><LI> Made preculture of <I>P. pastoris</I> GS115 
</LI><LI> Fermentation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>.
</LI><LI> Settings: fermenter: Bioengineering NFL22(7 L), final volume: 6 L, autoinduction medium with 60 µg/mL chloramphenicol added, 37 °C, stirrer increased 2 % if the pO<SUB>2</SUB> got below 30 %, airflow: 5 NL/m, 12 hours.
</LI><UL><LI><B>Team Activity Test:</B> After having a good night of sleep we are now ready to start the characterization of our newly available <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>. First step was a characterization regarding the pH optimum. We applied 0.03 mg/mL of protein in each sample and incubated half of the samples with 0.4 mM CuCl<SUB>2</SUB> as usual. We chose to investigate pHs of 1, 3, 5, 7 and 9 for their influence on <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> activity.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Activity measurements of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> regarding 100 mM sodium acetate buffer of different pHs. The enzymes were incubated with 0.4 mM CuCl<SUB>2</SUB> before measurements. Activity was measured via OD<SUB>420</SUB> and oxidized ABTS.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Activity measurements of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> regarding 100 mM sodium acetate buffer of different pHs without incubation with 0.4 mM CuCl<SUB>2</SUB> before measurements. Activity was measured via OD<SUB>420</SUB> and oxidized ABTS.</DIV></DIV></DIV><UL><LI><B>Team Substrate Analytic:</B> We did an Anthracen and Ethinylestradiol degradation to analyze them in LC-MS.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_September_14th">Friday September 14th</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 4:</B> SDS-PAGE of purification from fermentation from (09/13) of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, 6 L in NFL22, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>, 37 °C, pO<SUB>2</SUB> 50 % for 12 hours. Purification of the supernatant via Ni-NTA column. The red arrow shows BPUL</DIV></DIV></DIV><UL><LI><B>Team Fungal Laccases and Team Shuttle Vector:</B></LI><LI> Waiting for colonies from our transformation the day before.
</LI><LI> Ligate the new restricted parts in pSB1C3 and transformed them in KRX.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Isolated three glowing dishes of KRX with the <A href="http://partsregistry.org/Part:BBa_K863122" class="external text" rel="nofollow">const.GFP_His</A>-plasmid, three with the <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos_Freiburg</A>-plasmid and <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A>-plasmid.
</LI></UL><LI><B>Team Cultivation &amp; Purification: </B></LI><LI> Cell disruption of fermentation 09/13 via high-pressure homogenizer and purification via Ni-NTA column. Made SDS-Pages of purificated fractions.
</LI><LI> Repeat the preculture of <I>P. pastoris</I> GS115, because of using the wrong media.
</LI><LI> Made preculure of <I>E. coli</I> Rosetta Gami 2 with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>.
</LI><UL><LI><B>Team Substrate Analytic:</B></LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Anthracen control</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Ethinylestradiol control</DIV></DIV></DIV></DIV><P>We analyze Anthracen and Ethinyl estradiol from the 13th September. As you can see in the following figures Anthracen distingrates and Ethinyl estradiol on the other hand was stable. The corelation between the first peak and the other two peaks on Ethinyl estradiol may a pipetting mistake since the other two peaks are nearly simular. Distingration on Anthracen, a polycyclic aromatic hydrocarbon, were tested in different media and we determine that it depends on the Britton-Puffer (<A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary" class="external text" rel="nofollow">read more</A>)
</P><H3><SPAN class="mw-headline" id="Saturday_September_15th">Saturday September 15th</SPAN></H3><UL><LI><B>Team Cloning of Bacterial Laccases:</B></LI><LI>Digest of  &lt;partinfo&gt;BBa_J04450&lt;/partinfo&gt; to have pSB1C3 backbone for the last clonings.
</LI><LI><B>Team Fungal Laccases and Team Shuttle Vector:</B></LI><LI> We had some colonies on our LB + CM plates from our transformation of shuttle vector + tvel5. So we did Colony PCR to see if the gene was inserted. But sadly all colony PCRs are negative.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> KRX culture of <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)-GFP_His</A> seems to have a green glow.
</LI><LI> Isolation of four pellets of <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)-GFP_His</A> for us and SDU Denmark.
</LI><LI><A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A> digested with <I>Spe</I>I and <I>Age</I>I and deposphorylated.
</LI><LI><A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A>-PCR-product (gel-clean) digested with <I>Spe</I>I and <I>Ngo</I>MIV.
</LI><LI> Ligated <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A> with <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> and plated on select-Agar.
</LI><LI> Restriction-analysis showed that all <A href="http://partsregistry.org/Part:BBa_K863122" class="external text" rel="nofollow">const.GFP_His</A>-plasmids are correct, as are the three <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos</A> and two of the <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex</A></LI><LI> Collected data to make a protocol for a Cellulose binding assay:
</LI><LI> Avicel: about 11,4 mg protein (CBD) binds to 1 g Avicel (0,14 mg/12,3 mg)
</LI><LI> Duration of incubation for CBD to bind to Avicel:  about 30 minutes
</LI><LI> Washing and Lysis-buffer: 50mM Tris-HCl (pH 8.0)
</LI><LI> If needed: Elution with 80% ethylen-glycol (EG) or 1/5 Pellet to 4/5 EG (100%) of the overall volume.
</LI><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Made competent <I>P. pastoris</I> GS115 cells.
</LI><LI> Fermentation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A></LI><LI> Settings: fermenter: Bioengineering NFL22 (7 L), final volume: 6 L, LB-medium with 60 µg/mL chloramphenicol and 300 µg/mL ampicillin added, 37 °C, stirrer increased 2 % if the pO<SUB>2</SUB> got below 30 %, airflow: 5 NL/m. Problem: stirrer cascade did not work at the beginning.
</LI><LI> Harvest and centrifugation of cultivation &amp; fermentations 09/14. Store pellet at 4 °C.
</LI><LI><B>Team Activity Test:</B></LI><LI> Today we got the chance to work with Team Immobilization again. The samples they gave us were supernatants from different immobilization approaches. The team tested diverse bead concentrations with even higher amounts of beads then before. Check their labjournal for further information about the output of the measurements. Besides the supernatants we also received beads with immobilized laccases TEVL0 and <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>. So finally we got to use the filter plates that helped us to carry out the activity measurements and then stop the reaction at a precice point of time via centrifugation. Team Immobilization gave us enough samples for 7 measurements after different reactions times. We chose to stop the reaction after 2, 4, 8, 16, 32 and 64 minutes, respectively, to check the behavior of the immobilized laccase in relation to the time. Check their lab journal for the very exciting results!
</LI><H3><SPAN class="mw-headline" id="Sunday_September_16th">Sunday September 16th</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>Colony-PCR of 15 colonies from the <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A> transformation-plate (biotaq - Armin Nestat's recipe) with the result of a lot of positive clones.
</LI><LI>Prepared the sequencing of three <A href="http://partsregistry.org/Part:BBa_K863122" class="external text" rel="nofollow">const.GFP_His</A>, three <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos_Freiburg</A> and <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex_Freiburg</A></LI><LI> Lysis of <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A> isn't glowing anymore, or never was.
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Harvest and centrifugation of fermentation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>. Store pellet at 4 °C.
</LI><LI> Cell disruption via sonication and purification of cultivation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863103" class="external text" rel="nofollow">BBa_K863103</A> via Ni-NTA column. 
</LI></UL><DIV id="anzeige"><H1>Summary of Week 21</H1><H2><SPAN class="mw-headline" id="Week_21_.2809.2F17_-_09.2F23.2F12.29">Week 21 (09/17 - 09/23/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_21_.2809.2F17_-_09.2F23.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 21 (09/17 - 09/23/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_September_17th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday September 17th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_September_18th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday September 18th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_September_19th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday September 19th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_September_20th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday September 20th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_September_21st"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday September 21st</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_September_22nd"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday September 22nd</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_September_23rd"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday September 23rd</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_20"> Weekly Seminar</SPAN></H3><UL><LI> Everyone has to change his/her information on the team-site and we are trying to fill in the polls of the other iGEM teams.
</LI><LI> We are discussing our lab results, same procedure as every week.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_September_17th">Monday September 17th</SPAN></H3><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> Cell disruption of fermentation of <I>E. coli</I> Rosetta-Gami 2 with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A> via high-pressure homogenization and purification via Ni-NTA column.
</LI></UL><LI><B>Team Activity Tests:</B></LI><LI> Another day full of measurements is over! Today we measured some more samples from Team Immobilization: check their labjournal. As well we went on with characterizing our laccases. <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> was analyzed regarding its behavior in different concentrations of MeOH and acetonitrile. The activity was measured with concentrations of 2-16 µL, respectively. Since ABTS is changing it´s emission spectrum when being in contact with methanol we also detected the OD<SUB>530</SUB>. As expected ECOL is happier when only little MeOH is in its environment. We did the same measurements with acetonitrile and got the same result: the fewer acetonitrile, the better for ECOL. Again, we will hand over these results to Team Substrate Analytics.
</LI><CENTER><TABLE><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Figure 1: Activity of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> measured via oxidized ABTS at OD<SUB>420</SUB> in relation to a MeOH gradient.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Figure 2: Activity of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> measured via oxidized ABTS at OD<SUB>530</SUB> in relation to a MeOH gradient.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Figure 3: Activity of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> measured via oxidized ABTS at OD<SUB>420</SUB> in relation to a acetonitrile gradient.</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Figure 4: Activity of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">TTHL</A> measured via oxidized ABTS at OD<SUB>420</SUB>, showing activity in each fraction after prufication.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Figure 5: Activity of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> measured via oxidized ABTS at OD<SUB>530</SUB>in relation to a acetonitrile gradient.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Figure 6: Activity of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863022" class="external text" rel="nofollow">BHAL</A> measured via oxidized ABTS at OD<SUB>530</SUB>, showing activity in each fraction after prufication.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE></CENTER><UL><LI> Also another measurement session started today. Team Cultivation lend their purified laccases from <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">TTHL</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863022" class="external text" rel="nofollow">BHAL</A> produced with constitutive promoters to us. SO we rebuffered them and checked the activity with our standard protocol. And the results were good! In both cases we could detect activity in each fraction that Team Cultivation gave us (see Fig. 1 and Fig. 2). We are happy to have more laccases, but we are not sure, if we can pay enough attention to characterize them. We are trying our best, but <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> come first.
</LI></UL><UL><LI><B>Team Immobilization</B></LI><LI>Started new immobilization in this case of ECOL, BPUL and TVEL0. Incubation time of laccase solution: 18 hours.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_September_18th">Tuesday September 18th</SPAN></H3><UL><LI><B>Team Shuttle vector:</B></LI></UL><P>The shuttle vector was linearized with <I>EcoR</I>I and <I>Spe</I>I and was transformed in competened <I>P. pastoris</I> cells.
</P><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Primer arrived:
</LI><LI> Gradient-PCR with the <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>-plasmid and S3N10-CBDcex_RV and S3N10-GFP_FW primer-mix. the right product appeared only at 66,5°C; Lower: wrong product; Higher: no product
</LI><LI> Following PCR at 66,5°C(50µL) had the wrong product (main product at 2 kbp); could be because of a different template concentration or not exact temperature.
</LI><LI> Alternativly took the 20 µL-tube with right product and added 0,5 µL <I>Dpn</I>I for over-night digestion.
</LI></UL><LI> Gradient-PCR with <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>-template and a CBDcex_Freiburg- + GFP_Freiburg_compl-primer-mix to amplify the whole fusion-protein and get rid of the His-tag.
</LI><LI> Gradient-PCR with <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A>-template and a CBDclos_Freiburg- + GFP_Freiburg_compl-primer-mix to amplify the whole fusion-protein and get rid of the His-tag.
</LI><LI><B>Team Activity Tests:</B></LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fig. 1: Measurements of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> in a CuCl gradient of 0,1 to 0,7 mM. The activity was measured via oxidized ABTS at OD<SUB>420</SUB>, (n=4).</DIV></DIV></DIV><UL><LI> Today we measured the activity of our <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> laccase in a CuCl gradient to check whether another concentration than the one we usually incubated the laccases with might be more optimal. We chose a gradient from 0,1 mM to 0,7 mM (with an increment of 0,1) and measured the incubated samples as usual in 100 mM sodium acetate buffer, 0,1 mM ABTS (ad 200 µL H<SUB>2</SUB>O), respectively. Fig. 1 shows that the chosen concentration do influence ECOL, 0,4 mM seems like the best choice for incubation before the activity measurments. Thus the spectrum of OD<SUB>420</SUB> maxima is not immensely wide so that <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> seems to be satisfied with any CuCl concentration. 
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fig. 2: Activity measurements of <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A> via OD<SUB>420</SUB>.</DIV></DIV></DIV><UL><LI> Additionally we measured the activity of <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863010" class="external text" rel="nofollow">TTHL</A> without constitutive promoter at 25°C and in sodium acetate buffer (pH 5). TTHL finally also shows some activity, our favorite fractions of Team Cultivations output are number 1 and 2. Unfortunately TTHL couldn´t be indentified correctly via <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Analytics#MALDI" class="external text" rel="nofollow">MALDI</A> yet so that we will have to wait with further characterizing until we get the go through the MALDI results.
</LI><LI> And another big suprise was the activity test of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A>. After Team cultivation fermented it in a 6 L fermentation and purified our lovely enzyme, we did the standardized activity measurement protocol overnight. And have a look at our great results in fig. 6! We clearly have a good activity of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> reaching its saturation after half an hour. A fast reaction like this is comparable to the store bought laccase TVEL0. We are happy and will rebuffer as much as we can from fraction 4 which seems to have most of the Protein. Also we are going to determine the protein concentration. This is going to be awesome! <DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fig. 2: Activity measurements of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> via OD<SUB>420</SUB>. Different fractions from Team Cultivations purification were analyzed and they all showed activity.</DIV></DIV></DIV></LI></UL><LI><B>Team Substrate Anayltics</B>: 
</LI><LI> We analyze Ethin estradiol and estradiol degradation with the Spectrofluorophotometer. The measure showed that the degradation product are detectable but the results weren't clear. We decide to do this measurement again to have better results so a new degradation of the two substrates were set.
</LI><UL><LI><B>Team Immobilization</B></LI><LI>Got on with immobilization of ECOL, BPUL and TVEL0. Incubation time of laccase solution: 18 hours. Measurement of protein concentration in supernatant with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" class="external text" rel="nofollow">Roti-Nanoquant</A>.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_September_19th">Wednesday September 19th</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Stopped over-night-<I>Dpn</I>I-digestion of S3N10-clos-GFP_his PCR-product and cleaned up in gel. Ligated the PCR-product with itself via Blunt-end-ligation and transformed it into KRX. Plated on Select-Agar. 
</LI><LI> Gradient-PCRs of the <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A>-plasmid and a <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A>-plasmid (unsequenced) with the CBDcex_Freiburg- and GFP_Freiburg-compl- and the CBDclos_Freiburg- and GFP_Freiburg_compl primers, respectively. The CBDcex-setup showed no correct product, but the CBDclos-setup had the right bands at temperatures from 57°C to 64°C; Merged the correct fractions and cleaned them up through the gel.
</LI><LI>Digested the cleaned-up product with <I>Spe</I>I and <I>Xba</I>I and <I>Dpn</I>I.
</LI><LI>Ligated &lt;partinfo&gt;J61101&lt;/partinfo&gt; and <A href="http://partsregistry.org/Part:BBa_K863111" class="external text" rel="nofollow">CBDclos_Freiburg</A> with <A href="http://partsregistry.org/Part:BBa_K863120" class="external text" rel="nofollow">GFP_Freiburg</A> and transformed it into KRX.
</LI></UL><LI><B>Team Activity Test</B>:
</LI><LI> What a day! The deadline is coming closer and we still had (!) a lot of measurements to do, so today was chosen to be a day full of important measurements. First we measured the activity of the TVEL0 and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> in a CuCl gradient. As well we examined what our <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> laccase does when its supposed to work in different pHs. <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> was additionally characterized regarding its behavior in different concentrations of MeOH and acetonitrile. This output is of course most interesting for Team Substrate Analytics. <DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Comparison of activity of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> in sodium acetate buffer and Britton-Robinson buffer, measured at OD<SUB>420</SUB> respectively.</DIV></DIV></DIV>Since we already compared the behaviour of TVEL0 in the environment of sodium acetate buffer (pH 5) and Britton-Robinson buffer (pH 5), respectively, we also compared <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> and <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> in those buffers. ECOL is not impressed by the two different buffer systems and thus not influenced in it´s activity.
</LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> laccase activity measured via ABTS and OD<SUB>420</SUB> under usage of different pHs of sodium actate buffer. The results show a high activity at pH 5.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Negative control for pH measurements using ABTS. ABTS is oxidized through havy metals like Cu, which we are using for laccase activity tests. At different tested pHs ABTS gets oxidized but not very fast allowing us to leave it outer consideration.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> laccase activity measured with different concentration of CuCl showing that an incubation with 0.6 mM CuCl is optimal for the laccae activity, but higher CuCl concentrations seem to interfere in the activity.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Negative control for analyzing the oxidization of ABTS by different CuCl concentrations. An effect on the oxidization of ABTS is clearly distinguishable, but between the applied concentrations there is no difference in OD<SUB>420</SUB>.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Characterization of the activity of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> with sodium acetate buffer and Briton-Robinson Buffer at pH 5. Using the Briton-Robinson Buffer the saturation of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> activity is reached slower but stays constant. With sodium acetate buffer the maximum is reached considerably earlier.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> activity under different concentrations of acetonitrile. Acetonitrile has a small impact on the activity but indicates a decrease in activity the more actetonitrile is used.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> activity under different concentrations of MeOH. MeOH seems to have no impact on the activity using this small amounts of it.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Thursday_September_20th">Thursday September 20th</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases :</B></LI><LI> Our primers for GFP, tvel35, tvel5 and &lt;partinfo&gt;BBa_K500002&lt;/partinfo&gt; arrived. We did PCR on GFP with the new primers for cloning in our shuttle vector.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI>No Colonies on the S3N10-<A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A>-transformations-dish with the blunt-end ligation; reason: forgotten to phosphorylate the PCR-product.
</LI><LI> Two colonies on the CBDclos+GFP-dish with the constitutive <A href="http://partsregistry.org/Part:BBa_J61101" class="external text" rel="nofollow">J23100 + J61101</A> promoter. But none of them is glowing. And them shouldn't ... used the wrong selection-agar (since J61101 carries pSB1A2-backbone with AMP-resistence.
</LI><LI><B>Team Activity Tests:</B></LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Temperatur measurements of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> at 10°C and 25°C. Measurements were done via optical density 420 nm.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Temperatur measurements of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> at 10°C and 25°C. Measurements were done via optical density 420 nm.</DIV></DIV></DIV><UL><LI> Today we started another exciting measurement. Today was all about temperatures. Through our cooperation with the clarification plant we found out that the waster water here in Germany has a minimal temperature of 10°C throughout the year. We used this value together with our previous measurements at 25°C to complete the temperature gradient of our measurements. Eventhough <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A>  shows an increased activity at 10°C, it does reach a comparable OD value as the samples measured at 25°C. The increased activity is expressed by achieving the maximal OD<SUB>420</SUB> slower than the laccase measured at 25°C. This seems like good news for Team Modeling. On the other hand <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> shows nearly the same great activity at 10°C as at 25°C. Of course the reaction is a bit slower, but nevertheless the saturation is reached after 3 hours.
</LI></UL><LI><B>Team Substrate Analytics</B>: 
</LI><LI> Degradation measure from Ethinyl estradiol and Estradiol with the Spectrofluorophotometer. Degradation was from the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal/week21#Tuesday_September_18th" class="external text" rel="nofollow">18th September</A></LI><H3><SPAN class="mw-headline" id="Friday_September_21st">Friday September 21st</SPAN></H3><UL><LI><B>Team Shuttle vector:</B></LI></UL><P>P. pastoris colonies are seen and are picked into liquid culture for genomic DNA isolation and genotype characterisation.
</P><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI>Digest of  &lt;partinfo&gt;BBa_K863204&lt;/partinfo&gt; and GFP PCR product with <I>Aar</I>I and ligation. Transformation in <I>E. coli</I> KRX cells.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Transformated &lt;partinfo&gt;J61101&lt;/partinfo&gt; with CBDclos+GFP again and plated on the right selection-agar this time!
</LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> activity with different concentration of ABTS.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> activity with different concentration of ABTS.</DIV></DIV></DIV><UL><LI><B>Team Activity Tests:</B></LI><LI> Another part of the characterization of our <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K863000" class="external text" rel="nofollow">BPUL</A> laccase was to measure their activity when exposed to different concentrations of ABTS. We choose an exponential gradient of 2 µL to 16 µL and measured with the usual laccase concentration of 0,003 mg/mL and 100 mM sodium acetate buffer (ad 200 µL H<SUB>2</SUB>O). Unfortunately the measurements that contained 16 µL ABTS could not be measured completely because the OD<SUB>420</SUB> exceeded the maximal value that <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Tecan_Infinite_Microplate_Reader" class="external text" rel="nofollow">Tecan</A> was able to measure. The results of <A href="http://partsregistry.org/Part:BBa_K863005" class="external text" rel="nofollow">ECOL</A> (see. Fig. 1) show as well an exponential increase in the OD<SUB>420</SUB> correlated to the ABTS concentrations, respectively. The negative control was chosen to contain the maximum of 16 µL ABTS in buffer and water that was incubated with CuCl<SUB>2</SUB>.
</LI></UL><UL><LI><B>Team Immobilization</B></LI><LI> Measurement of activity of beads supernatant and original laccase solution.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_September_22nd">Saturday September 22nd</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> We did colony PCR on the transformed cells from day before. The results show that almost all colonies have the insert GFP. So we picked three of the positive colonies and plated each on five  LB + CM plates, because ee need much vector for the transformation in <I>Pichia pastoris</I>.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Once again two colonies on the agar-dish: can not say if they have a green glow...
</LI><LI> picked the colonies and them put into AMP-LB-Medium.
</LI><LI> Also made a flask with 10 mL LB-Medium and cells with the <A href="http://partsregistry.org/Part:BBa_K863104" class="external text" rel="nofollow">CBDcex(J61101)+GFP_His</A>-plasmid
</LI><UL><LI><B>Team Cultivation &amp; Purification:</B></LI><LI> For collaboration with <A href="https://2012.igem.org/Team:University_College_London" class="external text" rel="nofollow">UCL London</A>: Made precultures of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K729006&lt;/partinfo&gt;, with &lt;partinfo&gt;BBa_K863005&lt;/partinfo&gt; and without plasmid.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_September_23rd">Sunday September 23rd</SPAN></H3><UL><LI><B>Team shuttle vector:</B></LI></UL><P>genomic DNA could be isolated and have to be analyzed with primers: 5AOX-Genotyp-FW and TT-Genotyp-RV.
</P><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> All cultures did not show any sign of GFP-glow under uv-light.
</LI><LI> Gradient-PCR of &lt;partinfo&gt;BBa_K863104&lt;/partinfo&gt; with S3N10_Cex_compl- and S3N10_GFP-primermix to generate a 13 amino acid-linker between the cellulose binding domain and the GFP.
</LI><LI> Transformated <A href="http://partsregistry.org/Part:BBa_K863103" class="external text" rel="nofollow">CBDcex(T7)+GFP_His</A> and <A href="http://partsregistry.org/Part:BBa_K863113" class="external text" rel="nofollow">CBDclos(T7)+GFP_His</A> into BL21, to test if the problem is in the expression-system.
</LI></UL><UL><LI><B>Team Cultivation &amp; Purification:</B></LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 1:</B> Growth kinetics of flask cultivation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K729006&lt;/partinfo&gt; and without plasmid as negative control using <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#LB_medium" class="external text" rel="nofollow">LB-medium</A> and of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K863005&lt;/partinfo&gt; using <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>. Further parameters: final volume of 60 mL, 60 µg/mL chloramphenicol, 37 °C, durance: 12 hours.</DIV></DIV></DIV><UL><LI> For collaboration with <A href="https://2012.igem.org/Team:University_College_London" class="external text" rel="nofollow">UCL London</A>: Cultivation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K729006&lt;/partinfo&gt; and <I>E. coli</I> KRX without plasmid as negative control
</LI><LI> Settings: 250 mL flasks without baffles, final volume: 60 mL, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#LB_medium" class="external text" rel="nofollow">LB-medium</A>, 60 µg/mL chloramphenicol, 37 °C, durance: 12 hours.
</LI><LI> Settings: 1 L flask without baffles, final volume: 200 mL, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#LB_medium" class="external text" rel="nofollow">LB-medium</A>, 60 µg/mL chloramphenicol, 37 °C, durance: 12 hours.
</LI><LI> We determined the OD<SUB>600</SUB> for a growth kinetics (see figure 1).
</LI></UL><LI> Cultivation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K863005&lt;/partinfo&gt; for comparison.
</LI><LI> Settings: 250 mL flasks without baffles, final volume: 60 mL, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A>, 60 µg/mL chloramphenicol, 37 °C, durance: 12 hours.
</LI><LI> Made SDS-Page of the cultivation from 09/15. 
</LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 2:</B> SDS-PAGE of purification from our first fermentation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A> (09/15). 6 L in fermenter NFL22, 37 °C, pO<SUB>2</SUB> of 30 % for 12 hours., LB-medium with 60 µg/mL chloramphenicol and 300 µg/mL ampicillin. Purification of the supernatant via His-trap column. The red arrow shows TTHL in lane 9-11 (elution 1-3).</DIV></DIV></DIV><UL><LI><B>Team Substrate Analytic:</B> Since our last degradation with Estradiol didn't work because the Estradiol tubes were wrong we did an Estradiol degradation. We could find degradation products but not identify thoose products. Besides this we anayzed the sewage water we got from the Obere-Lutter and Schloß-Holte sewage water.
</LI></UL><DIV id="anzeige"><H1>Summary of Week 22</H1><H2><SPAN class="mw-headline" id="Week_22_.2809.2F24_-_09.2F30.2F12.29">Week 22 (09/24 - 09/30/12)</SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_22_.2809.2F24_-_09.2F30.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 22 (09/24 - 09/30/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_September_24th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday September 24th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_September_25th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday September 25th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_September_26th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday September 26th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_September_27th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday September 27th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_September_28st"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday September 28st</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_September_29nd"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday September 29nd</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_September_30rd"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday September 30rd</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Weekly_Seminar_21"> Weekly Seminar</SPAN></H3><UL><LI> First plan for the team t-shirt: a pacman eating estrogen.
</LI><LI> We now gather every day 9 o'clock in the morning to work on the wiki.
</LI><LI> Julia V is implementing the last design changes to the wiki.
</LI></UL><H3><SPAN class="mw-headline" id="Monday_September_24th">Monday September 24th</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> There was no obvious glow in the IPTG induced cultures. Todays time is only enough to harvest. The cultures were centrifuged at top speed and the supernatant was discarded. Cell-pellets were stored at -20 °C for further measurements.
</LI><LI> Phosphorylation of the CDBcex-S3N10-GFP-PCR-product, followed by ligation, transformation in <I>E. coli</I> KRX and plating on select-agar.
</LI></UL><LI><B>Team Cultivation &amp; Purification:</B><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1:</B> SDS-PAGE of the lysate supernatant from cultivation (09/23) of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K729006&lt;/partinfo&gt; ,<I>E. coli</I> KRX without plasmid (negative control) and with &lt;partinfo&gt;BBa_K863005&lt;/partinfo&gt; (positive control). 60 mL in 250 mL flasks without baffles,<A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#LB_medium" class="external text" rel="nofollow">LB-medium</A> or <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> for &lt;partinfo&gt;BBa_K863005&lt;/partinfo&gt; , 60 µg/mL chloramphenicol, 37 °C for 12 hours.</DIV></DIV></DIV></LI><LI> For collaboration with <A href="https://2012.igem.org/Team:University_College_London" class="external text" rel="nofollow">UCL London</A>: Made celldisruption via sonication and SDS-Page from lysates. The bands were analysed by MALDI-TOF.
</LI><UL><LI><B>Team Substrate Analytic</B>: The sewage water has shown no substrates of our interests. We only had 1L of the sewage plant water, so it may depend on the low concentration that we couldn't detect anything. Estradiol degradation on the other hand has shown some peaks, probably from degradation products.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_September_25th">Tuesday September 25th</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> No colonies on the select agar plate. Could have some reasons: Phosphroylation could be in-successful; expanding the time, or phosphorylating the primers before doing the PCR could be done. Also could be added to the mix since it has a <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC210195/" class="external text" rel="nofollow">positive effect on the phosporylation efficiency</A>.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_September_26th">Wednesday September 26th</SPAN></H3><UL><LI> Wiki Wiki Wiki
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_September_27th">Thursday September 27th</SPAN></H3><UL><LI> Wiki
</LI><LI> Sleep sleep sleep
</LI></UL><H3><SPAN class="mw-headline" id="Friday_September_28th">Friday September 28th</SPAN></H3><H3><SPAN class="mw-headline" id="Saturday_September_29th">Saturday September 29th</SPAN></H3><H3><SPAN class="mw-headline" id="Sunday_September_30th">Sunday September 30th</SPAN></H3><DIV id="anzeige"><H1>Summary of Week 23</H1><H2><SPAN class="mw-headline" id="Week_23_.2810.2F01_-_10.2F07.2F12.29">Week 23 (10/01 - 10/07/12)</SPAN></H2><H3><SPAN class="mw-headline" id="Weekly_Seminar_22"> Weekly Seminar</SPAN></H3><H3><SPAN class="mw-headline" id="Monday_October_01st">Monday October 01st</SPAN></H3><H3><SPAN class="mw-headline" id="Tuesday_October_02nd">Tuesday October 02nd</SPAN></H3><H3><SPAN class="mw-headline" id="Wednesday_October_03rd">Wednesday October 03rd</SPAN></H3><H3><SPAN class="mw-headline" id="Thursday_October_04th">Thursday October 04th</SPAN></H3><UL><LI> All:
</LI><LI> Giving a presentation for our supporting working groups.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_October_05th">Friday October 05th</SPAN></H3><UL><LI> All:
</LI><LI> Leaving Bielefeld with destination Amsterdam at 6 a.m.
</LI><LI> Check in at the hotel at 11 a.m.
</LI><LI> Practice session at 18:30 at VU Amsterdam University
</LI><LI> Meeting of young Minds
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_October_06th">Saturday October 06th</SPAN></H3><UL><LI> All:
</LI><LI> Introduction at VU University Amsterdam at 9 a.m.
</LI><LI> Presentation at 12:30 a.m.
</LI><LI> Poster session at 18 p.m.
</LI><LI> iGEM Party!!!
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_October_07th">Sunday October 07th</SPAN></H3><UL><LI> All:
</LI><LI> Award Ceremony
</LI><LI> We're shipping up to Boston
</LI></UL></DIV><DIV id="anzeige"><H1>Summary of Week 24 </H1><H2><SPAN class="mw-headline" id="Week_24_.2810.2F08_-_10.2F14.2F12"> Week 24 (10/08 - 10/14/12 </SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_24_.2810.2F08_-_10.2F14.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 24 (10/08 - 10/14/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_October_08th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday October 08th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_October_09th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday October 09th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_October_10th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday October 10th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_October_11th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday October 11th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_October_12th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday October 12th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_October_13th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday October 13th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_October_14th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday October 14th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Monday_October_08th">Monday October 08th</SPAN></H3><P>All:
Day off in Amsterdam!!!
</P><H3><SPAN class="mw-headline" id="Tuesday_October_09th">Tuesday October 09th</SPAN></H3><H3><SPAN class="mw-headline" id="Wednesday_October_10th">Wednesday October 10th</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B></LI><LI> Ligation of pSB1C3::BBa_K863202 construct in KRX cells was done.
</LI></UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Gradient-PCRs of CBDcex_Freiburg; CBDclos_Freiburg; GFP_Freiburg and ecol_Freiburg (47 °C to 67°C)
</LI><LI> Product:
</LI><LI> CBDcex_Freiburg at all temperatures
</LI><LI> CBDclos_Freiburg at all temperatures
</LI><LI> GFP_Freiburg at all temperatures
</LI><LI> ecol_Freiburg at 53 °C to 61°C (best at 57 to 59 °C)
</LI><LI> pooled fractions for gel clean up
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Ordered Primers for the SDM of the illegal XbaI restriction site in the shuttle vector
</LI><LI><B>Team Cultivation and Purification:</B></LI><LI>Made precultures of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>for 19L fermentation (500mL preculture)
</LI><H3><SPAN class="mw-headline" id="Thursday_October_11th">Thursday October 11th</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Clean-Up of the Gel-pieces (from PCR-products)
</LI><LI> Restriction of PCR-products (3h):
</LI><LI> GFP_Freiburg:
</LI><LI> XbaI+PstI Tango-Buffer for Ligation 
</LI><LI> XbaI+AgeI Tango 4xAgeI for Assembly
</LI></UL><LI> CBDclos/cex:
</LI><LI> NgoMIV+PstI P.4+BSA 2xPstI for Assembly
</LI><LI> Assembly and Ligation (1h with T4):
</LI><LI> GFP_Freiburg + CBDcex_Freiburg + J61101
</LI><LI> GFP_Freiburg + CBDclos_Freiburg + J61101
</LI><LI> GFP_Freiburg + J61101
</LI><LI> Transformation in <I>E. coli</I> KRX (3 µL, chemical):
</LI><LI> GFP_Freiburg + CBDcex_Freiburg + J61101
</LI><LI> GFP_Freiburg + CBDclos_Freiburg + J61101
</LI><LI> GFP_Freiburg + J61101
</LI><LI> All Transformations were plated on AMP-select-agar
</LI><UL><LI><B>Team Cultivation and Purification</B></LI><LI> 12 L Cultivation of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A></LI><LI>Settings: 
</LI></UL><P>Bioengineering NFL 19L fermenter, autoinduction HSG medium, 60 µg/mL chloramphenicol, 19 L, 37 °C, stirrer on cascade to hold a pO2 of 50 %, airflow: 12 NL/m, 16 hours. HSG medium was choosen to get a high biomass concentration with hope for a higher amount of laccases. 
</P><UL><LI>Made precultures of <I>E. coli</I> Rosetta Gami 2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; (09/09) behind a constitutive promoter. (500mL preculture)
</LI></UL><H3><SPAN class="mw-headline" id="Friday_October_12th">Friday October 12th</SPAN></H3><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> A lot of colonies on all three dishes (J61101+GFP_Freibug;J61101+GFP_Freibug+CBDcex;J61101+GFP_Freibug+CBDclos)
</LI><LI> no obvious green fluorescence, not even at J61101+GFP
</LI><LI> Maybe the combination of J23100 and J61101 isn't strong enough to express a visible amount of GFP
</LI></UL><LI> Colony-PCR of 16 colonies of every dish
</LI><LI> J61101 + GFP_Freiburg:
</LI><LI> 16/16 positive
</LI><LI> plated two on select-agar
</LI><LI> J61101 + GFP_Freiburg + CBDcex:
</LI><LI> 8/16 positve
</LI><LI> plated three on select-agar
</LI><LI> J61101 + GFP_Freiburg + CBDclos:
</LI><LI> 7/16 positive
</LI><LI> plated three on select-agar
</LI><UL><LI><B>Team Shuttle Vector:</B></LI><LI> The induction with 0.5% (v/v) methanol of P. pastoris GS115 cells with integrated <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> was started.
</LI></UL><UL><LI><B>Team Substrate Analysis</B>: 
</LI><LI> Since we have seen some new peaks after Estradiol treated with Laccases in the LC-MS, we wanted to have more degradation products to do an MS so we used a higher concentradion of Estradiol and Laccase and let them incubate for 66 hours.
</LI></UL><UL><LI><B>Team Cultivation and Purification</B></LI><LI> 12L Cutlivations of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> were harvested and stored at 4°C until purification. 
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 1:</B> Fermentation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K863005&lt;/partinfo&gt; (ECOL) in an Bioengineering NLF 19, scale: 12 L, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> + 60 µg/mL chloramphenicol, 37 °C, pH 7, agitation on cascade to hold pO<SUB>2</SUB> at 50 %, OD<SUB>600</SUB> measured every 30 minutes.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 2:</B> Fermentation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K863000&lt;/partinfo&gt; (BPUL) in an Bioengineering NLF 19, scale: 12 L, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> + 60 µg/mL chloramphenicol, 37 °C, pH 7, agitation on cascade to hold pO<SUB>2</SUB> at 50 %, OD<SUB>600</SUB> measured every 30 minutes.</DIV></DIV></DIV><UL><LI> 12 L Cultivation of <I>E. coli</I> Rosetta Gami 2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; (09/09)
</LI><LI> Settings: 
</LI></UL><P>Bioengineering NFL 19L fermenter, HSG medium, 60 µg/mL chloramphenicol, 300 µg/mL ampicillin, 12 L, 37 °C, stirrer on cascade to hold a pO2 of 50 %, airflow: 12 NL/m, 22-24 hours. HSG medium was chosen to get a high biomass concentration with hope for a higher amount of laccases.
</P><UL><LI><B>Team Substrate Analysis:</B></LI><LI> Dissolved estradiol, estrone and ethinyl estradiol in 50 % (v/v) acetonitrile with a concentration of 200 µg mL<SUP>-1</SUP></LI><LI> Creating  and measuring of negative controls for estradiol, estrone and ethinyl estradiol. Experimental setup: 2 µg substrate, 110 µL BR-buffer in 200 µL reaction volume. Measured at t<SUB>0</SUB> and after 3 hours.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_October_13th">Saturday October 13th</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B></LI><LI> The P. pastoris GS115 cells integrated with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> were also inducted with 0.5% (v/v) methanol (100%).
</LI></UL><UL><LI><B>Team Cultivation and Purification</B></LI><LI> 12L Cutlivations of <I>E. coli</I> Rosetta Gami 2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; (09/09) were harvested and stored at 4°C until purification.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 1:</B> Fermentation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K863005&lt;/partinfo&gt; (ECOL) in an Bioengineering NLF 19, scale: 12 L, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> + 60 µg/mL chloramphenicol, 37 °C, pH 7, agitation on cascade to hold pO<SUB>2</SUB> at 50 %, OD<SUB>600</SUB> measured every 30 minutes.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 2:</B> Fermentation of <I>E. coli</I> KRX with &lt;partinfo&gt;BBa_K863000&lt;/partinfo&gt; (BPUL) in an Bioengineering NLF 19, scale: 12 L, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium" class="external text" rel="nofollow">autoinduction medium</A> + 60 µg/mL chloramphenicol, 37 °C, pH 7, agitation on cascade to hold pO<SUB>2</SUB> at 50 %, OD<SUB>600</SUB> measured every 30 minutes.</DIV></DIV></DIV><UL><LI><B>Team Substrate Analysis:</B></LI><LI> Creating  and measuring of negative controls for estradiol, estrone and ethinyl estradiol. Experimental setup: 2 µg substrate, 0.1 mM ABTS, 110 µL BR-buffer in 200 µL reaction volume. Measured at t<SUB>0</SUB> and after 10 minutes.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_October_14th">Sunday October 14th</SPAN></H3><UL><LI><B>Team Shuttle Vector</B>:
</LI><LI> The P. pastoris GS115 cells integrated with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> were also inducted with 0.5% (v/v) methanol (100%).
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Plasmid isolation of pSB1C3::BBa_K863207 for transformation into P. pastoris GS115 was done. After isolation the plasmid was digested with <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials">NotI</A>, purified with the PCR clean up kit and transformed via <A href="/Team:Bielefeld-Germany/Protocols/molecular_genetics#Transformation_of_yeast_cells" title="Team:Bielefeld-Germany/Protocols/molecular genetics">protocol</A> in P. pastoris GS115.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Tried to isolate all plates, but only got low concentrations (5 ng/µL - 20 ng/µL)
</LI><LI> plated eight different colonies for isolation
</LI></UL><UL><LI><B>Team Substrate Analysis</B>:
</LI><LI> We took sample from the reaction of the 12th October.
</LI></UL><DIV id="anzeige"><H1>Summary of Week 25 </H1><H2><SPAN class="mw-headline" id="Week_25_.2810.2F15_-_10.2F21.2F12.29">  Week 25 (10/15 - 10/21/12) </SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_25_.2810.2F15_-_10.2F21.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 25 (10/15 - 10/21/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_October_15th"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday October 15th</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_October_16th"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday October 16th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_October_17th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday October 17th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_October_18th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday October 18th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_October_19th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday October 19th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_October_20th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday October 20th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_October_21st"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday October 21st</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Monday_October_15th">Monday October 15th</SPAN></H3><UL><LI><B>Team Shuttle Vector:</B></LI><LI> There are no colonies of pBS1C3::BBa_K863202 KRX transformantion.
</LI><LI> The P. pastoris GS115 cells integrated with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> were harvested and the supernatant was examined on GFP by fluorescence. But there was no GFP.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Yeah, the yeast cells with integrated <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863207" class="external text" rel="nofollow">BBa_K863207</A> are growing.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Tried to isolate the eight plated, but did get low concentrations again (except one J61101+GFP_Freiburg+CBDcex_Freiburg with conc &gt;100 ng/µL)
</LI><LI> cleaned up some more PCR-products (GFP_Freiburg; CBDcex_Freiburg; CBDclos_Freiburg)
</LI></UL><LI><B>Team Substrate Analysis</B>: We stopped the reaction from the 12th October with Methanol and gave it to Dr. Marcus Persicke to measure the degradation on LC-MS
</LI><UL><LI><B>Team Cultivation and Purification</B></LI><LI> Harvested Cells from 12L Cutlivations of <I>E. coli</I> KRX with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> as well as 12 L Cultivation of <I>E. coli</I> Rosetta Gami 2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; (09/09) were disrupted via high-pressure homogenizer and filtrated by <A href="http://www.millipore.com/catalogue/module/C7493" class="external text" rel="nofollow">Millipore Pellicon XL 50</A> with first 300 kDa to seperate all celldebris and 10 kDa to concentrate the laccases. The concentrated solutions were purified by the Ni-NTA-column.
</LI></UL><LI><B>Team Substrate Analysis:</B></LI><LI> Creating  and measuring of estradiol and ethinyl estradiol degradation samples for the laccase TVEL0. Experimental setup: 2 µg substrate, 0.1 mM ABTS, 110 µL BR-buffer and 10 µg laccase in 200 µL reaction volume. Measured at t<SUB>0</SUB> and after 10 minutes.
</LI><UL><LI><B>Team Immobilization</B></LI><LI>Started immobilization of TVEL0 to analyze the binding of laccase during the first 10 hours representative for following immobilizations.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_October_16th">Tuesday October 16th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI> Today we received the samples from Team Cultivation &amp; Purification. We are really motivated to measure the activity of our new produced enzymes. But before that we have to prepare the samples. This implied the determination of the protein concentration in each fraction, which we did today. The results are listed in the following table:
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Transformed B0034 into KRX
</LI><LI> Transformed J23100 into KRX
</LI><LI> Plated both on AMP-select-agar
</LI><LI> Plated one more Colony of J61101 + GFP_Freiburg and J61101 + GFP_Freiburg + CBDclos
</LI></UL><UL><LI><B>Team Substrate Analysis:</B> The Massspectrometry data showed two potential products after the Laccase treatment of Estradiol and Ethinyl estradiol but we unfortunatly could not identify them so we decided to do MS-MS on thoose. The products showed that after Laccase treatment both Estradiol and Ethinyl estradiol losses two <B>H</B> atoms which was new for us.
</LI></UL><UL><LI><B>Team Immobilization:</B></LI></UL><P>Taking samples for immobilization time analyzation.
</P><H3><SPAN class="mw-headline" id="Wednesday_October_17th">Wednesday October 17th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI> Before starting our activity assay we re-buffered the fractions we got from Team Cultivation and Purification. It took some time, that's why we had no time for activity analysis yet. But we prepared everything for the next day, including determining the protein concentration of the fractions after they have been re-buffered. This is what we got:
</LI></UL><UL><LI><B>Team Shuttle Vector:</B></LI><LI> Genomic DNA isolation of yeast cells with integrated <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> construct was done with the Kit. For phenotype characterisation an PCR with the primer pair 5AOX-Phenotype-FW and TT-Phenotype-RV was done and the fragment size was determined. The results are: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863205" class="external text" rel="nofollow">BBa_K863205</A> was not integrated.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> YPD liquid culture was inoculated with one yeast colony (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863207" class="external text" rel="nofollow">BBa_K863207</A>) and incubated at 30°C and 150 rpm.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Picked colonies of B0034 and J23100 and plated them on AMP selection agar for plasmid isolation
</LI></UL><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> SDM-PCR of the Shuttle vector
</LI><LI> Added <I>Dpn</I>I over night
</LI></UL><UL><LI><B>Team Cultivation and Purification:</B></LI><LI> Made SDS-Pages of the cultivations from 10/11 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>) and 10/12 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>, &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt;).
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 1:</B> SDS-Page of purification from the 12 L fermentations from 10/11 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A>) and 10/12 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>, &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt;). Purification of the supernatant via His-trap column, step gradient with 5 %, 50 % and 100 % elution buffer.</DIV></DIV></DIV></DIV><UL><LI><B>Team Immobilization:</B></LI></UL><P>Measurement of Protein concentration with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" class="external text" rel="nofollow">Roti-Nanoquant</A> and analyses of binding capacity.
</P><H3><SPAN class="mw-headline" id="Thursday_October_18th">Thursday October 18th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI> Finally the day had come to measure the activity of our own produced laccases. This activity assay should give information about the enzyme content in every fraction. To make the measurements comparable we applied the same protein amount in every sample. Again, we used our <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Analytics#General_setup_of_enzyme_activity_measurements/" class="external text" rel="nofollow">standard activity assay protocol</A>, but this time we used the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Britton-Robinson_Buffer/" class="external text" rel="nofollow">Britton-Robinson Buffer</A> at pH 5. Also we applied 0.1 mM ABTS and measured over night.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of each purified fraction of our new cultivation with ECOL. Samples were re-buffered into H<SUB>2</SUB>O and the protein amount in each fraction has been adjusted. The Measurement was done using the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Analytics#General_setup_of_enzyme_activity_measurements/" class="external text" rel="nofollow">standard activity assay protocol</A> over night.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of each purified fraction of our new cultivation with BPUL. Samples were re-buffered into H<SUB>2</SUB>O and the protein amount in each fraction has been adjusted. The Measurement was done using the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Analytics#General_setup_of_enzyme_activity_measurements/" class="external text" rel="nofollow">standard activity assay protocol</A> over night.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of each purified fraction of our new cultivation with BHAL. Samples were re-buffered into H<SUB>2</SUB>O and the protein amount in each fraction has been adjusted. The Measurement was done using the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Analytics#General_setup_of_enzyme_activity_measurements/" class="external text" rel="nofollow">standard activity assay protocol</A> over night.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of each purified fraction of our new cultivation with TTHL. Samples were re-buffered into H<SUB>2</SUB>O and the protein amount in each fraction has been adjusted. The Measurement was done using the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Analytics#General_setup_of_enzyme_activity_measurements/" class="external text" rel="nofollow">standard activity assay protocol</A> over night.</DIV></DIV></DIV><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Isolated J23100 plasmid
</LI><LI> Digested J23100 with <I>Eco</I>RI and <I>Pst</I>I
</LI><LI> Digested pSB1C3-Backbone with <I>Eco</I>RI and <I>Pst</I>I
</LI><LI> Clean up of pSB1C3-BB and J23100 + RFP - Insert via Gel
</LI><LI> Ligation of J23100 and pSB1C3-BB
</LI><LI> Isolated B0034 plasmid
</LI><LI> Digested B0034 with <I>Spe</I>I and <I>Pst</I>I
</LI><LI> Digested GFP_Freiburg-PCR-product with <I>Xba</I>I and <I>Pst</I>I
</LI><LI> Digested an other fraction of the GFP_Freiburg-PCR-product with <I>Xba</I>I and <I>Age</I>I
</LI><LI> Assembled (Ligation):
</LI><LI> B0034 with GFP_Freiburg
</LI><LI> B0034 with GFP_Freiburg and CBDcex_Freiburg
</LI><LI> B0034 with GFP_Freiburg and CBDclos_Freiburg
</LI></UL><LI> Transformed these three into KRX
</LI><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Bands of 19C-PRC-product in Gel at 7 kbp (which is correct)
</LI><LI> Clean-Up of 19C-PRC-product
</LI><LI> Transformation of the 19C-PRC-product in XL1 Blue
</LI></UL><UL><LI><B>Team Immobilization:</B></LI><LI>Discussed new method of activity measurements for immobilized laccases on beads with less lacasse solution needed. Started first general tests with photometer.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_October_19th">Friday October 19th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI> After getting our interesting activity measurement results, we had to figure out which fraction is going to be used in the following experiments and how much laccase it contains besides other proteins. We agreed, that the most active fraction contains 90 % laccase, which is commonly used in the literature. Since we applied the same protein amount of each fraction for the activity measurements, we made sure that the results really correspond to the amount of laccase. These are the most active fractions we have chosen with their contained laccase amount:
</LI><LI> ECOL: Fraction 50% 2 with a laccase concentration of 63,9 µg mL<SUP>-1</SUP>.
</LI><LI> BPUL: Fraction 50% 2 with a laccase concentration of 25,1 µg mL<SUP>-1</SUP>.
</LI><LI> BHAL: Fraction  5% 3 with a laccase concentration of 10,9 µg mL<SUP>-1</SUP>.
</LI><LI> TTHL: The best fraction of our TTHL laccase was fraction 50% 1 with a laccase concentration of 4,03 µg mL<SUP>-1</SUP>. In regard to Team Immobilization and Team Substrate Analysis we had to reach a higher amount of TTHL laccase. So we thought of combining two fractions. We chose the second best fraction of TTHL, which is 5% 3. To calculate the amount of contained laccase we had to compare the activity and reconsidered the slope. Since we stated the fraction with its highest activity as 90 % contained laccase, we calculated the amount of laccase regarding to that one. The following figure shows the slopes we got out of the activity from fraction 50% 1 and fraction 5% 3 (which is the second best). Accordingly to this, fraction 5% 3 contains 68,9% of laccase, which is 4,8 µg mL<SUP>-1</SUP>. In total, the composed fraction contains 4,4 µg mL<SUP>-1</SUP>.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Slope of the two most active fractions of purified TTHL laccase. By comparing them, the amount of contained laccase in the fractions had been calculated.</DIV></DIV></DIV></DIV><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Colony-PCR of B0034 + GFP_Freiburg; B0034 + GFP_Freiburg + CBDcex; B0034 + GFP_Freiburg + CBDcex_Freiburg; B0034 + GFP_Freiburg + CBDclos_Freiburg.
</LI><LI> All negative
</LI></UL><LI> Restriction of Ecol_Freiburg with <I>Xba</I>I, <I>Age</I>I and <I>Dpn</I>I
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><UL><LI><B>Team Substrate Analysis:</B></LI><LI> Creating  and measuring of estradiol and ethinyl estradiol degradation samples for the laccases TVEL0 and BPUL. The experimental setup changed. The laccase amount was adjusted to the highest possible amount of the lowest concentrated laccase. TTHL has a concentration of 4,03 µg mL<SUP>-1</SUP> and sets the maximal laccase amount to 0.6 µg per 200 µL reaction volume. To compensate these worse conditions, the substrate amount was lowered to 1 µg and the temperature was changed to 30°C.
</LI><LI> Test of estradiol and ethinyl estradiol degradation done by TVEL0 and BPUL  without ABTS. Experimental setup: 1 µg substrate, 0.1 mM ABTS, 110 µL BR-buffer and 0.6 µg laccase in 200 µL reaction volume. Measured at t<SUB>0</SUB> and after 3 hours.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_October_20th">Saturday October 20th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Negativ control for substrate saturation analysis to investigate in the effect of CuCl<SUB>2</SUB> oxidizing ABTS. Instead of laccase in H<SUB>2</SUB>O, only H<SUB>2</SUB>O mixed with 0.4 mM CuCl<SUB>2</SUB> was applied.</DIV></DIV></DIV> During the Jamboree in Amsterdam we had the oppurtunity to discuss our activity results with a lot of people. One remark was about the calculation of the enzyme specific units. As you all remember, we measured the activty with 0.1 mM ABTS, but to get comparable results with the literature, we had to measure each protein with other concentrations of ABTS. The concentration of ABTS should be saturated for each laccase. So we started with ECOL and BPUL and applied different ABTS concentrations starting from 0.1 mM as usual: 0.1 mM, 0.5 mM, 1 mM and 5 mM. We applied 616 ng Laccase in each sample. Our results showed, that especially ECOL does not reach its substrate saturation with 5 mM ABTS. Our plans are now to check these concentrations of ABTS for BPUL and TTHL and also higher concentrations of ABTS for all laccases. For that we should reduce the enzyme amount to be able to detect the change in OD 420 properly.
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of our new ECOL laccase with different ABTS concentration to find the substrate saturation of ECOL. 616 ng ECOL laccase (incubated with 0.4 mM CuCl<SUB>2</SUB>) and 20 % Britton-Robinson buffer (pH 5) were applied.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of our new BPUL laccase with different ABTS concentration to find the substrate saturation of BPUL. 616 ng BPUL laccase (incubated with 0.4 mM CuCl<SUB>2</SUB>) and 20 % Britton-Robinson buffer (pH 5) were applied.</DIV></DIV></DIV><UL><LI><B>Team Substrate Analysis:</B></LI><LI> TVEL0 and BPUL were able to degrade 100 % of estradiol and 100 % of ethinyl estradiol. To get comparable results, the reaction temperature was lowered to 20 °C again, to have a closer look, where differences are apparent. Again TVEL0 and BPUL degraded 100 % of the substrates. But before the laccase stocks deplete, first ECOL, BHAL and TTHL should be tested.
</LI><LI> Did substrate degradation analysis with ECOL without ABTS. Experimental setup: 1 µg substrate, 0.1 mM ABTS, 110 µL BR-buffer and 0.6 µg laccase in 200 µL reaction volume. Measured at t<SUB>-0</SUB> and after 3 hours.
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_October_21st">Sunday October 21st</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B></LI></UL><P>Genomic isolation was done with the <A href="http://www.promega.com/resources/protocols/technical-manuals/0/wizard-genomic-dna-purification-kit-protocol/" class="external text" rel="nofollow">Promega Wizard genomic DNA purification system kit</A> of our approximately 50 yeast clones with integrated (a) GFP and (b) tvel5. 
</P><UL><LI><B>Team Activity Tests:</B></LI><LI> Today was BHAL's and TTHL's turn. The experiment setup was the same as described the day before. But again we could not make sure, that 5 mM ABTS is the substrate saturation for our tested laccases. We are now sure to go higher in ABTS concentrations.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of our new BHAL laccase with different ABTS concentration to find the substrate saturation of BHAL. 616 ng ECOL laccase (incubated with 0.4 mM CuCl<SUB>2</SUB> and 20 % Britton-Robinson buffer pH 5) were applied.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Activity assay of our new TTHL laccase with different ABTS concentration to find the substrate saturation of TTHL. 616 ng TTHL laccase (incubated with 0.4 mM CuCl<SUB>2</SUB> and 20 % Britton-Robinson buffer pH 5) were applied.</DIV></DIV></DIV><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Colonies PCRs of B0034 + GFP_Freiburg; B0034 + GFP_Freiburg + CBDcex_Freiburg; B0034 + GFP_Freiburg + CBDclos_Freiburg
</LI><LI> All negativ
</LI></UL><LI> Transformed once again (with an mix of old an new GFP_Freiburg)
</LI><LI> Isolated plated colonies of transformed colonies (two GFP; two GFP+Cex; two GFP+Clos 
</LI><LI> Restriction analysis showed only bands at 2K bp (pSB1C3+BBa_B0034 vector without insert)
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Isolated the plasmids of the six SDM-dishes
</LI><LI> Over-night digestion with <I>Not</I>I 
</LI><LI><B>Team Substrate Analysis:</B></LI><LI> ECOL, BHAL and TTHL were tested with the same conditions and all of them degraded the substrates much slower than TVEL0 and BPUL.
</LI><LI><B> Team Immobilization</B></LI><LI>Started immobilization of new laccases ECOL, BPUL, BHAL, TTHL and TVEL0.
</LI><DIV id="anzeige"><H1>Summary of Week 26 </H1><H2><SPAN class="mw-headline" id="Week_26_.2810.2F22_-_10.2F28.2F12.29"> Week 26 (10/22 - 10/28/12) </SPAN></H2><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_26_.2810.2F22_-_10.2F28.2F12.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 26 (10/22 - 10/28/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Monday_October_22nd"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Monday October 22nd</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Tuesday_October_23rd"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Tuesday October 23rd</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Wednesday_October_24th"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Wednesday October 24th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thursday_October_25th"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Thursday October 25th</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Friday_October_26th"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Friday October 26th</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Saturday_October_27th"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Saturday October 27th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sunday_October_28th"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sunday October 28th</SPAN></A></LI></UL><H3><SPAN class="mw-headline" id="Monday_October_22nd">Monday October 22nd</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B></LI></UL><P>The identification of positive yeast clones and determination of the phenotype (M<SUP>+</SUP> or M<SUP>S</SUP>) was done by PCR with the primers 5AOX-Phenotype-FW and TT-Phenotype-RV. The result of the PCR was: no GFP integrated clones and three tvel5 integrated (1.9 kb and 2.2 kb) clones. The cassette, including <I>tvel5</I> and <I>his4</I>, was recombinated by a single cross over, because the 2.2 kb PCR product of the <I>aox1</I> gene has been formed. <A href="/Team:Bielefeld-Germany/Protocols/Materials#Minimal_methanol_media_.28MM.29" title="Team:Bielefeld-Germany/Protocols/Materials"> Minimal methanol medium</A> was inoculated with these three positive clones with the M<SUP>+</SUP> phenotype.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863204" class="external text" rel="nofollow">BBa_K863204</A> was used as a controll for PCR with the primer 5AOX-Phenotype-FW and TT-Phenotype-RV. Expected lane: 437 bp.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">PCR product of the gDNA from the culture 1, 2 and 3 (C1, C2, C3) with the primer 5AOX-Phenotype-FW and TT-Phenotype-RV. Expected lane: 1.9 kb (<I>tvel5</I>) and 2.2 kb (<I>aox1</I>).</DIV></DIV></DIV><UL><LI><B>Team Activity Tests:</B></LI><LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Negative control for the experimental setup with different pHs of the Britton-Robinson buffer. Effect of 5 mM ABTS was determined.</DIV></DIV></DIV>Today was all about finding a comfy environment for our laccases. We thought of determining the right pH before going on with substrate saturation experiments since we could not be sure, that the activity was influenced by the pH. We chose a range from pH 4 to pH 9, applied 616 ng laccase and 5 mM ABTS. Our results revealed the following results:
</LI><LI> optimal pH of ECOL: pH 5
</LI><LI> optimal pH of BPUL: pH 4 and pH 5
</LI><LI> optimal pH of BHAL: pH 5
</LI><LI> optimal pH of TTHL: pH 5
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">BHAL activity measured under different pH conditions (pH 4 to pH 9) to determine the pH optimum. 616 ng Laccase and 5 mM ABTS were used.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">TTHL activity measured under different pH conditions (pH 4 to pH 9) to determine the pH optimum. 616 ng Laccase and 5 mM ABTS were used.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">ECOL activity measured under different pH conditions (pH 4 to pH 9) to determine the pH optimum. 616 ng Laccase and 5 mM ABTS were used.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">BPUL activity measured under different pH conditions (pH 4 to pH 9) to determine the pH optimum. 616 ng Laccase and 5 mM ABTS were used.</DIV></DIV></DIV><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Sequencing of &lt;partinfo&gt;BBa_K863120&lt;/partinfo&gt; in &lt;partinfo&gt;BBa_J61101&lt;/partinfo&gt; showed a deletion within the ORF of the GFP; this seemed to be the reason that the colonies are not fluorescent.
</LI><LI> Gradient-PCR on &lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt; with GFP_Freiburg Pre- and Suffix Primers
</LI><LI> Worked fine. Clean up and digestion with <I>Xba</I>I and <I>Pst</I>I
</LI></UL><LI> Gradient-PCR on &lt;partinfo&gt;BBa_K863103&lt;/partinfo&gt; with CBDcex_Freiburg and GFP_Freiburg_compl Primers
</LI><LI> One of 12 temperatures worked. Clean up and digestion with <I>Xba</I>I and <I>Pst</I>I
</LI><LI> Gradient-PCR on &lt;partinfo&gt;BBa_K863113&lt;/partinfo&gt; with CBDclos_Freiburg and GFP_Freiburg_compl Primers
</LI><LI> did not get product at all
</LI><LI> Ligation of the PCR products and GFP_Freiburg and CBDcex_Freiburg-GFP_Freiburg, respectively
</LI><LI> Transformation of both ligations
</LI><LI> Cells plated on selection agar
</LI><UL><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Stopped Digestion of the shuttle vector, but gel went wrong and the products were lost
</LI></UL><UL><LI><B>Team Substrate Analysis:</B></LI><LI> BHAL and TTHL were tested with the same conditions and all of them degraded the substrates much slower than TVEL0 and BPUL.
</LI></UL><UL><LI><B> Team Immobilization:</B></LI><LI> Measured of protein concentration in supernatants of immobilized ECOL, BPUL, BHAL, TTHL and TVEL0 with <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" class="external text" rel="nofollow">Roti-Nanoquant</A>. Analysed binding capacity of the different laccases.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_October_23rd">Tuesday October 23rd</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B> The induction of the yeast cells with 1% (v/v) 100% methanol was done at morning and in the evening.
</LI></UL><UL><LI><B>Team Activity Tests:</B></LI><LI> In order to find the substrate saturation we continued to measure our laccase activities with higher concentrations of ABTS. To have a slower reaction and also try not to waste the enzymes we halfed the amount of used laccases from 616 ng to 308 ng of each laccase. The following substrate saturations were determined:
</LI><LI> ECOL: 9 mM ABTS
</LI><LI> BPUL: 5 mM ABTS
</LI><LI> BHAL: 7 mM ABTS
</LI><LI> TTHL: 7 mM ABTS
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Colony PCR of B0034 + GFP_Freiburg
</LI><LI> 14/24 positive
</LI><LI> Plated three on selection agar for plasmid isolation
</LI></UL><LI> Colony PCR of B0034 + CBDcex-GFP_Freiburg
</LI><LI> 16/24 positive
</LI><LI> Plated three on selection agar for plasmid isolation
</LI><LI><B>Team Immobilization</B></LI><LI>Started measurement of enzymatic activity of the different immobilized laccases with the <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Immobilization#Activity_measurement_of_beads" class="external text" rel="nofollow">Photometer</A>.
</LI><H3><SPAN class="mw-headline" id="Wednesday_October_24th">Wednesday October 24th</SPAN></H3><UL><LI><B>Team Activity Tests:</B></LI><LI> pH measurements were repeated under substrate saturation with a pH range from pH 4 to pH 9.
</LI></UL><UL><LI><B>Team Fungal and Plant Laccases:</B> In the morning and evening the induction of yeast cells with 1% (v/v) methanol was done.
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> Isolated Plasmids containing B0034 + GFP_Freiburg &amp;</LI><LI> Digestion of pooled plasmids with <I>Xba</I>I and <I>Pst</I>I
</LI></UL><LI> Isolated Plasmids containing B0034 + CBDcex-GFP_Freiburg
</LI><LI> Digestion of pooled plasmids with <I>Xba</I>I and <I>Pst</I>I
</LI><LI> Digestion of J23100 with <I>Spe</I>I and <I>Pst</I>I
</LI><LI> Clean up of inserts (B0034 + GFP_Freiburg &amp; B0034 + CBDcex-GFP_Freiburg) and Backbone (J23100) via gel
</LI><LI> Ligation of Backbone and the inserts
</LI><LI> Transformation and plated on selection agar
</LI><LI><B>Team Site Directed Mutagenesis:</B></LI><LI> Digestion showed that all isolated colonies still have the illegal <I>Xba</I>I restriction side
</LI><LI><B>Team Substrate Analysis:</B></LI><LI> Repeating of the degradation experiments without ABTS for TVEL0, BPUL, ECOL, BHAL and TTHL with a degradation time of five hours to increase the differences between the laccases.
</LI><H3><SPAN class="mw-headline" id="Thursday_October_25th">Thursday October 25th</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Induction of the yeast cells with 1% (v/v) methanol was done. After 4 h the cells were harvested by centrifugation at 3000xg 4°C for 5 min. 
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> All colonies did not show any obvious green glow
</LI></UL><UL><LI><B>Team Substrate Anaylsis:</B></LI><LI>The MS-MS data arrived from Marcus Persicke. To ensure our measurements and discuss what happens after the Laccase treatment and on the ESI we decided to ask an organic chemistrer so we went to Prof. Dr. Dietmar Kuck whos expert on this field. Since he had less time to discuss we have only speculativ testifies.
</LI></UL><UL><LI><B>Team Cultivation and Purification:</B></LI></UL><P>Last offline analysis of the the two cultivations of <I>E.coli</I> KRX containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863000" class="external text" rel="nofollow">BBa_K863000</A>and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863005" class="external text" rel="nofollow">BBa_K863005</A> with Carobon Source Detection with HPLC.
</P><UL><LI><B>Team Substrate Analysis:</B></LI><LI> Repeating of the degradation experiments with ABTS for ECOL, BHAL and TTHL to replace previous experimental errors.
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_October_25th_2">Thursday October 25th</SPAN></H3><UL><LI><B>Team Fungal and Plant Laccases:</B></LI><LI> Induction of the yeast cells with 1% (v/v) methanol was done. After 4 h the cells were harvested by centrifugation at 3000xg 4°C for 5 min. 
</LI></UL><UL><LI><B>Team Cellulose Binding Domain:</B></LI><LI> All colonies did not show any obvious green glow
</LI></UL><UL><LI><B>Team Substrate Anaylsis:</B></LI><LI>The MS-MS data arrived from Marcus Persicke. To ensure our measurements and discuss what happens after the Laccase treatment and on the ESI we decided to ask an organic chemistrer so we went to Prof. Dr. Dietmar Kuck whos expert on this field. Since he had less time to discuss we have only speculativ testifies.
</LI></UL><UL><LI><B>Team Activity Tests:</B></LI><LI> Activity assay of all 4 laccases and TVEL0 were done at 10 °C.
</LI><LI> TVEL5 was incubated in copper and after adjusting buffer composition of the supernatant the activity test was done. The result is shown in the following figure:
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Activity test of cultivation (culture 1, 2, 3) supernatant of <A href="http://partsregistry.org/Part:BBa_K863207" class="external text" rel="nofollow">BBa_K863207</A>. The control is the supernatant of <I>P. pastoris</I> GS115.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Friday_October_26th">Friday October 26th</SPAN></H3><UL><LI><B>Wiki day</B></LI><LI><B>Team Cultivation and Purification</B></LI></UL><P>Final offline analysis of the <I>E.coli</I> Rosetta Gami 2 cultivations with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863012" class="external text" rel="nofollow">BBa_K863012</A>(09/09) and &lt;partinfo&gt;BBa_K863022&lt;/partinfo&gt; (09/09).
</P><H3><SPAN class="mw-headline" id="Saturday_October_27th">Saturday October 27th</SPAN></H3><H3><SPAN class="mw-headline" id="Sunday_October_28th">Sunday October 28th</SPAN></H3><DIV id="navlabunten" class="tabs"><DIV class="scroller"><UL style="list-style-type:none"><LI><A href="#1"><STRONG>Prologue</STRONG></A></LI><LI><A href="#2"><STRONG>Week 1</STRONG></A></LI><LI><A href="#3"><STRONG>Week 2</STRONG></A></LI><LI><A href="#4"><STRONG>Week 3</STRONG></A></LI><LI><A href="#5"><STRONG>Week 4</STRONG></A></LI><LI><A href="#6"><STRONG>Week 5</STRONG></A></LI><LI><A href="#7"><STRONG>Week 6</STRONG></A></LI><LI><A href="#8"><STRONG>Week 7</STRONG></A></LI><LI><A href="#9"><STRONG>Week 8</STRONG></A></LI><LI><A href="#10"><STRONG>Week 9</STRONG></A></LI><LI><A href="#11"><STRONG>Week 10</STRONG></A></LI><LI><A href="#12"><STRONG>Week 11</STRONG></A></LI><LI><A href="#13"><STRONG>Week 12</STRONG></A></LI><LI><A href="#14"><STRONG>Week 13</STRONG></A></LI><LI><A href="#15"><STRONG>Week 14</STRONG></A></LI><LI><A href="#16"><STRONG>Week 15</STRONG></A></LI><LI><A href="#17"><STRONG>Week 16</STRONG></A></LI><LI><A href="#18"><STRONG>Week 17</STRONG></A></LI><LI><A href="#19"><STRONG>Week 18</STRONG></A></LI><LI><A href="#20"><STRONG>Week 19</STRONG></A></LI><LI><A href="#21"><STRONG>Week 20</STRONG></A></LI><LI><A href="#22"><STRONG>Week 21</STRONG></A></LI><LI><A href="#23"><STRONG>Week 22</STRONG></A></LI><LI><A href="#24"><STRONG>Week 23</STRONG></A></LI><LI><A href="#25"><STRONG>Week 24</STRONG></A></LI><LI><A href="#26"><STRONG>Week 25</STRONG></A></LI><LI><A href="#27"><STRONG>Week 26</STRONG></A></LI></UL></DIV></DIV><TABLE cellspacing="10"><TBODY><TR><TD><A href="/File:Promega_logo_180x109.gif" title="File:Promega logo 180x109.gif">55px</A></TD></TR></TBODY></TABLE><DIV id="content-footer-box"><DIV class="content-footer-box-left"><DIV class="about"><P>About our Wiki</P><P>Fifteen students from different fields of study from the Bielefeld University aspired to complete a project called TOXIC COMPOUNDS IN NATURAL WATER - A CASE FOR LACCASE. In his Wiki we document our achievements and our approach. </P></DIV><DIV class="contact"><P>Contact Us</P><P>Universität Bielefeld
Universitätsstraße 25
D-33615 Bielefeld<A href="mailto:info@igem-bielefeld.de">info(at)igem-bielefeld.de</A></P></DIV></DIV><DIV class="content-footer-box-right"><DIV class="navigate"><P>Navigate our Wiki</P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany">Start</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Team</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Project</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Results</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Since Regionals</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Human Practice</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Bielefeld-Germany/Labjournal">http://2012.igem.org/Team:Bielefeld-Germany/Labjournal</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Labjournal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Labjournal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal&amp;oldid=298544" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>