<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Results_CBD"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Results/CBD">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Results/CBD&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/CBD&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/CBD&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Results/CBD" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Results/CBD</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Cellulose_Binding_Domain_-_Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Cellulose Binding Domain - Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#The_Binding_Assay"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">The Binding Assay</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Cloning_of_the_Cellulose_Binding_Domains"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Cloning of the Cellulose Binding Domains</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Literature"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Literature</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Cellulose_Binding_Domain_-_Introduction">Cellulose Binding Domain - Introduction</SPAN></H2><DIV style="text-align:justify;"><P>In the field of cheap protein-extraction cellulose binding domains (CBD) have made themselves a name. A lot of publications have been made, concerning a cheap strategy to capture a protein from the cell-lysate with a CBD-tag. Also enhanced segregation with CBD-tagged proteins have been observed. Here the idea is different, we want to take advantage of the binding capacity of binding domains not only for purification reasons (it is still a benefit), but also as an immobilizing-protocol for our laccases.
</P><P>To make a purification and immobilization-tag out of a protein domain, there are a lot of decisions and characterizations you have to get through.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 1:Graphical sequence-alignment of &lt;partinfo&gt;K392014&lt;/partinfo&gt; and the <A href="http://www.ncbi.nlm.nih.gov/nuccore/AB041993" class="external text" rel="nofollow">Xyn10A gene of C.josui</A></DIV></DIV></DIV><P>Starting with the choice of the binding domain, the first limitation is accessibility. Our first place to look was, of course the <A href="http://partsregistry.org" class="external text" rel="nofollow">Partsregistry</A>. We found a promising Cellulose binding motif of the <I>C. josui</I> Xyn10A gene (&lt;partinfo&gt;BBa_K392014&lt;/partinfo&gt;) there and ordered it right from the spot for our project. After some research later concerning the sequence of that BioBrick it turned out that the part is not the CBD of the Xylanase as it should be, but the glycosyl hydrolase domain of the protein (Figure 1). This result made the part useless for our project (<A href="http://partsregistry.org/Part:BBa_K392014:Experience" class="external text" rel="nofollow">complete review</A>) and it was the only binding domain in the <A href="http://partsregistry.org" class="external text" rel="nofollow">Partsregistry</A> that fitted to our project.
</P><P>So we started to search the accessible organisms we had via NCBI for binding-domains, -proteins and -motifs and asked in work-groups if they could help us out. The results of our database research were only two chitin/carbohydate binding modules within the <I>Bacillus halodurans</I> genome (we ordered that stain for it's laccase &lt;partinfo&gt;BBa_K863020&lt;/partinfo&gt;). One is in the <A href="http://www.ncbi.nlm.nih.gov/nucleotide/289656506?report=genbank&amp;log$=nuclalign&amp;blast_rank=1&amp;RID=0JPT9WMS01N" class="external text" rel="nofollow">Cochin chitinase gene</A> and the other in a <A href="http://www.ncbi.nlm.nih.gov/protein/BAB05022.1" class="external text" rel="nofollow">chitin binding protein</A>.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:212px;"><DIV class="thumbcaption">Figure 2: Plasmid with CBDcex</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:232px;"><DIV class="thumbcaption">Figure 3: Plasmid with CBDclos</DIV></DIV></DIV><P>Meanwhile the Fermentation group of our university offered use two plasmids (p570 &amp; p671), containing two different cellulose binding domains. The Cellulose binding domain of the <A href="http://www.ncbi.nlm.nih.gov/nucleotide/327179207?report=genbank&amp;log$=nucltop&amp;blast_rank=3&amp;RID=152ZCN0E01N" class="external text" rel="nofollow"><I>Cellulomonas fimi</I> ATCC 484 exoglucanase gene</A> (CBDcex) and the Cellulose binding domain of <A href="http://www.ncbi.nlm.nih.gov/nuccore/M73817" class="external text" rel="nofollow"><I>Clostridium cellulovorans</I> cellulose binding protein gene (cbp A)</A> (CBDclos). At that point we decided to use these two domains. Staying within the cellulose binding domain-family and leave other protein domains like carbohydrate binding domains aside will keep the results comparable. Like changing to a different binding material would change the binding capacities of both domains in the same way. Also we would stay within bacterial CBD and didn't have to spend time thinking about post-translational modification and glycosylation.
</P><P>To get to know more about these two domains, their properties and their proteins we consulted NCBI and used the <A href="http://blast.ncbi.nlm.nih.gov/" class="external text" rel="nofollow">BLAST</A>-tool to identify the cellulose binding domains and ExPASy-tools for further measurements.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:132px;"><DIV class="thumbcaption">Figure 5: Crystal structure of the CBM_2-family</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 6: Protein-BLAST of <A href="http://www.ncbi.nlm.nih.gov/nucleotide/327179207?report=genbank&amp;log$=nucltop&amp;blast_rank=3&amp;RID=152ZCN0E01N" class="external text" rel="nofollow"><I>Cellulomonas fimi</I> ATCC 484 exoglucanase gene</A></DIV></DIV></DIV><P>The CBD of the Cellulomonas fimi ATCC 484 exoglucanase gene (Figure 6) is a 100 amino acid long domain, close to the C-terminal ending of the protein with a theoretical pI of 8.07 and a molecular weight of 10.3 kDa. It is classified to be stable and belongs to the Cellulose Binding Modul family 2 (pfam00553/cl02709). This means two tryptophan residues are involved in cellulose binding, this type of CBD is only found in bacteria. Also a CBM49 Carbohydrate binding domain is found within the protein domain, where <A href="http://www.ncbi.nlm.nih.gov/pubmed/17322304?dopt=Abstract" class="external text" rel="nofollow">binding studies</A> have shown, that it binds to crystalline cellulose, which could be a possible target for immobilization.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 7: Protein-BLAST of <A href="http://www.ncbi.nlm.nih.gov/nuccore/M73817" class="external text" rel="nofollow"><I>Clostridium cellulovorans</I> cellulose binding protein gene (cbp A)</A></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:132px;"><DIV class="thumbcaption">Figure 8: Crystal structure of the CBM_3-family</DIV></DIV></DIV><P>The CBD of the <A href="http://www.ncbi.nlm.nih.gov/nuccore/M73817" class="external text" rel="nofollow"><I>Clostridium cellulovorans</I> cellulose binding protein gene (cbp A)</A> on the other hand is a N-terminal domain with 92 amino acids, theoretical pI of 4.56 and ist also classified as stable. It belongs to the Cellulose Binding Modul family 3 (pfam00942/cl03026) and is part of a very large cellulose binding protein with four other carbohydrate binding moduls and a lot of docking interfaces for the proteins in its amino acid sequence (Figure 7).
</P><H2><SPAN class="mw-headline" id="The_Binding_Assay">The Binding Assay</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Aequorea victoria green fluorescent protein in action</DIV></DIV></DIV><P>To measure the capacity and strength of the bonding between the cellulose binding domains and different types of cellulose many different assays have been made. One of the simplest and most often used is the fusion of the CBD to a reporter-protein, especially <A href="http://www.ncbi.nlm.nih.gov/pubmed/22305911" class="external text" rel="nofollow">green or red fluorescent protein (GFP/RFP)</A> is very common. The place of the CBD is measured through the fluorescence of the fused GFP and quantification can easily been done.
</P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/18573384" class="external text" rel="nofollow">Protocol</A>:
</P><UL><LI> Harvest the <I>E coli</I>-cells producing the fusion-protein of CBD and GFP and centifuge 10 minutes at top speed.
</LI><LI> Re-suspend the cell-pellet in 50 mM Tris-HCl-Buffer (pH 8.0).
</LI><LI> Break down cells via sonication.
</LI><LI> Centrifuge at top speed for 20 minutes to get rid of the cell-debris. 
</LI><LI> Take the supernatant and measure the emission at 511 nm (excitation at 501 nm) (&lt;partinfo&gt;E0040&lt;/partinfo&gt;)
</LI><LI> Mix a definite volume of lysate with a definite volume or mass of e.g. crystalline cellulose (CC) or reactivated amorphous cellulose (RAC)
</LI><LI> Wait 15 (RAC) to 30 (CC) minutes 
</LI><LI> Take supernatant and measure the emission at 511 nm again. 
</LI><LI> The difference between the first an the second measurement is the relative quantity of what has bound to the cellulose.
</LI></UL><H2><SPAN class="mw-headline" id="Cloning_of_the_Cellulose_Binding_Domains">Cloning of the Cellulose Binding Domains</SPAN></H2><P>The cloning of the CBDs should fit to the cloning of our laccases, so we designed the BioBricks with a T7-promoter and the B0034 RBS to have a similar method of cultivation. After we investigated the restriction-sites it showed, that at least for the characterization of the CBDs a quick in-frame assembly of the CBDs and a GFP would be possible, because nether the CBDs nor the GFP (&lt;partinfo&gt;I13522&lt;/partinfo&gt;) of the partsregistry inherits a <I>Age</I>I- or <I>Ngo</I>MIV-site, which makes Freiburg-assembly possible. To do so, we designed <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Primers" class="external text" rel="nofollow">primers</A> for the constructs &lt;partinfo&gt;BBa_K863101&lt;/partinfo&gt; (CBDcex(T7)), carrying the CBDcex domain from the <I>C. fimi</I> exoglucanase and &lt;partinfo&gt;BBa_K863111&lt;/partinfo&gt; (CBDclos(T7)), carrying the CBDclos domain from the <I>C.cellulovorans</I> binding protein. The protein-BLAST of the two CBDs gave a exact picture of which bases are count to the binding domains and which are not. To be sure not to disturb the folding anyway we kept 6 to 12 bases up and downstream of the domains as conserved sequences. Even if the <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex</A>is a N-terminal domain we chose to make the domains N-terminal, so the BioBrick already carries all regulatory parts and the linked protein can easily be exchanged. This also would be nice for other people using this part.
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> CBDcex_T7RBS</TD><TD> 80 </TD><TD> TGAATTCGCGGCCGCTTCTAGAGTAATACGACTCACTATAGGGAAAGAGGAGAAATAATGGGTCCGGCCGGGTGCCAGGT
</TD></TR><TR><TD> CBDcex_2AS-Link_compl </TD><TD> 56 </TD><TD> CTGCAGCGGCCGCTACTAGTATTAACCGGTGCTGCCGCCGACCGTGCAGGGCGTGC
</TD></TR><TR><TD> CBDclos_T7RBS </TD><TD> 73 </TD><TD> CCGCTTCTAGAGTAATACGACTCACTATAGGGAAAGAGGAGAAATAATGTCAGTTGAATTTTACAACTCTAAC
</TD></TR><TR><TD> CBDclos_2ASlink_compl</TD><TD> 63 </TD><TD> CTGCAGCGGCCGCTACTAGTATTAACCGGTGCTGCCTGCAAATCCAAATTCAACATATGTATC
</TD></TR></TBODY></TABLE></CENTER><P>The listed complementary primers added, besides the Freiburg-suffix, a two amino acid Glycine-Serine-linker to the end of the CBDs. This is a very short linker, but as GFP-experts and <A href="http://www.ncbi.nlm.nih.gov/pubmed/17394253" class="external text" rel="nofollow">publications</A> described GFP and CBDs are very stable proteins and should cope with a very short linker. The benefit of a short linker is that protease activity is kept minimal.
</P><P>The GFP &lt;partinfo&gt;K863121&lt;/partinfo&gt; we wanted to use for the assay was an alternated version of the &lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt;. We made <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Primers" class="external text" rel="nofollow">primers</A> that added a Freiburg-pre- and suffix to the GFP coding sequence and a His-tag to the C-terminus to get it purified for the measurements. This part &lt;partinfo&gt;BBa_K863121&lt;/partinfo&gt; (GFP_His) should be easily added to the CBDs and assemble to the fusion-proteins &lt;partinfo&gt;BBa_K863103&lt;/partinfo&gt; CBDcex(T7)+GFP_His] and &lt;partinfo&gt;BBa_K863113&lt;/partinfo&gt; CBDclos(T7)+GFP_His]).
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> GFP_Frei </TD><TD> 54 </TD><TD> TACGGAATTCGCGGCCGCTTCTAGATGGCCGGCATGCGTAAAGGAGAAGAACTT
</TD></TR><TR><TD> GFP_His6_compl </TD><TD> 74 </TD><TD> CTGCAGCGGCCGCTACTAGTATTAACCGGTGTGATGGTGATGGTGATGTTTGTATAGTTCATCCATGCCATGTG
</TD></TR></TBODY></TABLE></CENTER><P>Since we added a His-tag to the end of the GFP, we had to make a alternated version of the &lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt; to compare binding of GFP with and without CBD. Therefore we made a forward primer, to amplify the whole &lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt; and used the GFP_His_compl-primer to add the His-tag to the C-terminus.
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> GFP_FW_SV </TD><TD> 39 </TD><TD> acgtcacctgcgtgtagctCGTAAAGGAGAAGAACTTTT
</TD></TR></TBODY></TABLE></CENTER><P>Due to bad cleavage efficiency at the <I>Pst</I>I restiction-site in nearly all PCR-products the cloning of the CBDs (<A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> and <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>)and especially the insertion of <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> to the CBDs took a lot more time as expected. This was because, while designing the primers, we missed to add additional base pairs from the cleavage-site to the termini to increase the efficiency to standard. When we got aware of the problem, cloning got a lot quicker and more successful.
</P><P>As the project went on and the T7-constructs didn't seem to work one suspected reason was that a stop codon (TAA) that accidentally resulted between the RBS and ATG could be the reason for that. To solve the problem we started to change to a constitutive promoter (&lt;partinfo&gt;BBa_J61101&lt;/partinfo&gt;) using the Freiburg-assembly. Therefore Freiburg forward primers for the CBDs were made.
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> CBDcex_Freiburg-Prefix</TD><TD> 54 </TD><TD> GCTAGAATTCGCGGCCGCTTCTAGATGGCCGGCGGTCCGGCCGGGTGCCAGGTG
</TD></TR><TR><TD> CBDclos_Freiburg-Prefix </TD><TD> 57 </TD><TD> GCTAGAATTCGCGGCCGCTTCTAGATGGCCGGCTCATCAATGTCAGTTGAATTTTAC
</TD></TR></TBODY></TABLE></CENTER><P>The second advantage of these primers were, that the order of the fusion proteins could easily been changed, when a Freiburg suffix-primer for the GFP would be available, so we ordered that.
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> GFP_Freiburg_compl </TD><TD> 61 </TD><TD> ACGTCTGCAGCGGCCGCTACTAGTATTAACCGGTTTTGTATAGTTCATCCATGCCATGTGT
</TD></TR></TBODY></TABLE></CENTER><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">PCR-product of BBa_K863104 with S3N10-primers</DIV></DIV></DIV><P>Sadly the primers arrived just a few days before wiki freeze and we had no time to test that. The switching to the constitutive promoter had no obvious effect (no green colonies or culture). Changing the expression strain of <I>E. coli</I> BL21 didn't do no positive effect ether. Which led us to the conclusion, that besides further information the problem has to be the space in between the two proteins and the CBD and GFP must hamper each other from folding correctly. To test and solve this a very long linker with three serines followed by ten asparagines should be assembled in the already existing parts via a blunt end cloning. This also is right at hand, just wasn't successful in the time which was left. 
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> S3N10_Cex_compl </TD><TD> 40 </TD><TD> TTGTTGTTGTTCGAGCTCGAGCCGACCGTGCAGGGCGTGC
</TD></TR><TR><TD> S3N10_Clos_compl </TD><TD> 40 </TD><TD> TTGTTGTTGTTCGAGCTCGAGCTGCCGCCGACCGTGCAGG
</TD></TR><TR><TD> S3N10_GFP </TD><TD> 40 </TD><TD> CAATAACAATAACAACAACCGTAAAGGAGAAGAACTTTTC
</TD></TR></TBODY></TABLE></CENTER><H2><SPAN class="mw-headline" id="Literature"> Literature </SPAN></H2><P>Biotechnol Bioeng. 2007 Oct 15;98(3):599-610.
Strategy for selecting and characterizing linker peptides for CBM9-tagged fusion proteins expressed in Escherichia coli.
Kavoosi M, Creagh AL, Kilburn DG, Haynes CA.
J Biol Chem. 2007 Apr 20;282(16):12066-74. Epub 2007 Feb 23.
A tomato endo-beta-1,4-glucanase, SlCel9C1, represents a distinct subclass with a new family of carbohydrate binding modules (CBM49).
Urbanowicz BR, Catalá C, Irwin D, Wilson DB, Ripoll DR, Rose JK.
Protein Expr Purif. 2012 Apr;82(2):290-6. Epub 2012 Jan 28.
Cellulose affinity purification of fusion proteins tagged with fungal family 1 cellulose-binding domain.
Sugimoto N, Igarashi K, Samejima M.
Anal Chim Acta. 2008 Jul 28;621(2):193-9. Epub 2008 May 27.
Bioseparation of recombinant cellulose-binding module-proteins by affinity adsorption on an ultra-high-capacity cellulosic adsorbent.
</P>
Hong J, Ye X, Wang Y, Zhang YH.</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Bielefeld-Germany/Results/CBD">http://2012.igem.org/Team:Bielefeld-Germany/Results/CBD</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Results/CBD" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Results/CBD" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/CBD&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/CBD&amp;oldid=170337" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>