<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Results_cbc"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Results/cbc">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Results/cbc&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/cbc&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/cbc&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Results/cbc" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Results/cbc</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="navigation"><DIV style=" z-index:10000;"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany">Start</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Team</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Bielefeld-University">Bielefeld University</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Project</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Achievments">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Database">Database</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Outlook">Outlook</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Results</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Datapage">Datapage</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Laccases">Laccases</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/immo">Immobilization</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/substrate">Substrate Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/cbc">Cellulose Binding Domain</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/vector">Shuttle Vector</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/london">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Since Regionals</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Summary</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Expert">Expert</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Politics">Politics</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Sewage_treatment_plant">Industry</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human Practices/Outreach">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/StudentAcademy">Education</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Environment">Environment</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV></DIV><DIV id="page-title"><SPAN id="page-title-text">
               Cellulose Binding Domain 
            </SPAN></DIV><DIV id="grey_bg"><H1>Summary</H1><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#The_Binding_Assay"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">The Binding Assay</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Cloning_of_the_Cellulose_Binding_Domains"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Cloning of the Cellulose Binding Domains</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Since_Regionals"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Since Regionals</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Literature"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Literature</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><DIV style="text-align:justify;"><P>In the field of cheap protein-extraction, cellulose binding domains (CBD) have made themselves a name. A lot of publications are available providing capture of a protein from the cell-lysate with a CBD-tag as a cheap purification strategy. Also enhanced segregation with CBD-tagged proteins have been observed. In this project the idea is different, the binding of our self-produced proteins to cellulose via a protein-domain-tag is taking advantage of the binding capacity of binding domains not only for purification reasons (it is still a great benefit), but also as an immobilization procedure for our laccases for later application.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 1:</B> Graphical sequence-alignment of &lt;partinfo&gt;K392014&lt;/partinfo&gt; and the <A href="http://www.ncbi.nlm.nih.gov/nuccore/AB041993" class="external text" rel="nofollow"><I>xyn10A</I> gene of <I>C.josui</I></A></DIV></DIV></DIV><P>To make a purification and immobilization-tag out of a protein domain, there are a lot of decisions and characterizations to do.
</P><P>Starting with the choice of the binding domain, the first limitation is accessibility. The first place to look at is of course the <A href="http://partsregistry.org" class="external text" rel="nofollow">Partsregistry</A>. A promising Cellulose binding motif of the <I>C. josui</I><I>xyn10A</I> gene (&lt;partinfo&gt;BBa_K392014&lt;/partinfo&gt;) was found and ordered for the project right from the spot. After some research concerning the sequence of that BioBrick, it turned out that the part is not the CBD of the Xylanase as it should be, but the glycosyl hydrolase domain of the protein (Figure 1). This result made the part useless for the project (<A href="http://partsregistry.org/Part:BBa_K392014:Experience" class="external text" rel="nofollow">complete review</A>) and it was the only binding domain in the <A href="http://partsregistry.org" class="external text" rel="nofollow">Partsregistry</A> that fitted to the project.
</P><P>Accessible organisms were searched via NCBI for binding-domains, -proteins and -motifs and work groups were asked if they could help out. The results of the database research were only two chitin/carbohydrate binding modules within the <I>Bacillus halodurans</I> genome (that strain was ordered for its laccase &lt;partinfo&gt;BBa_K863020&lt;/partinfo&gt;). One is in the <A href="http://www.ncbi.nlm.nih.gov/nucleotide/289656506?report=genbank&amp;log$=nuclalign&amp;blast_rank=1&amp;RID=0JPT9WMS01N" class="external text" rel="nofollow">Cochin chitinase gene</A> and the other in a <A href="http://www.ncbi.nlm.nih.gov/protein/BAB05022.1" class="external text" rel="nofollow">chitin binding protein</A>.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 2:</B> Plasmid-map of p714 with CBDcex; Origin: Fermentation group of Bielefeld University</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:222px;"><DIV class="thumbcaption"><B>Figure 3:</B> Plasmid-map of p570 with CBDclos; Origin: Fermentation group of Bielefeld University</DIV></DIV></DIV><P>Meanwhile the Fermentation group of our university offered the use of two plasmids (p570 &amp; p671), containing cellulose binding domains. The cellulose binding domain of the <A href="http://www.ncbi.nlm.nih.gov/nucleotide/327179207?report=genbank&amp;log$=nucltop&amp;blast_rank=3&amp;RID=152ZCN0E01N" class="external text" rel="nofollow"><I>Cellulomonas fimi</I> ATCC 484 exoglucanase gene</A> (CBDcex) and the cellulose binding domain of <A href="http://www.ncbi.nlm.nih.gov/nuccore/M73817" class="external text" rel="nofollow"><I>Clostridium cellulovorans</I> cellulose binding protein gene (<I>cbpA</I>)</A> (CBDclos). The decision was made to use these two domains. Staying within the cellulose binding domain-family and leave other protein domains like carbohydrate binding domains aside would keep the results comparable. For example, changing to a different binding material would change the binding capacities of both domains in the same way. Also both are bacterial CBDs and no post-translational modification and glycosylation had to be dealt with.
</P><P>To get to know more about these two domains, their properties and their proteins, the NCBI databases were consulted. <A href="http://blast.ncbi.nlm.nih.gov/" class="external text" rel="nofollow">BLAST</A> was used to identify the cellulose binding domains and ExPASy-tools were used for further analyses.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><B>Figure 5:</B> Predicted structure of the CBM_2-family made with Cn3D</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 6:</B> Protein-BLAST of <A href="http://www.ncbi.nlm.nih.gov/nucleotide/327179207?report=genbank&amp;log$=nucltop&amp;blast_rank=3&amp;RID=152ZCN0E01N" class="external text" rel="nofollow"><I>Cellulomonas fimi</I> ATCC 484 exoglucanase</A></DIV></DIV></DIV><P>The CBD of the <I>Cellulomonas fimi</I> ATCC 484 exoglucanase gene (Figure 6) is a 100 amino acid long domain, close to the C-terminal ending of the protein with a theoretical pI of 8.07 and a molecular weight of 10.3 kDa. It is classified to be stable and belongs to the Cellulose Binding Modul family 2 (pfam00553/cl02709; Figure 5). Two tryptophane residues are involved in cellulose binding in this family which is only found in bacteria. Also, a CBM49 carbohydrate binding domain is found within the protein domain, where <A href="http://www.ncbi.nlm.nih.gov/pubmed/17322304?dopt=Abstract" class="external text" rel="nofollow">binding studies</A> have shown, that it binds to crystalline cellulose, which could be a possible target for immobilization.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 7:</B> Protein-BLAST of <A href="http://www.ncbi.nlm.nih.gov/nuccore/M73817" class="external text" rel="nofollow"><I>Clostridium cellulovorans</I> cellulose binding protein (Cbp A)</A></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><B>Figure 8:</B> Predicted structure of the CBM_3-family made with Cn3D</DIV></DIV></DIV><P>The CBD of the <A href="http://www.ncbi.nlm.nih.gov/nuccore/M73817" class="external text" rel="nofollow"><I>Clostridium cellulovorans</I> cellulose binding protein gene (<I>cbpA</I>)</A> on the other hand is a N-terminal domain with 92 amino acids, theoretical pI of 4.56 and is also classified as stable. It belongs to the Cellulose Binding Module family 3 (pfam00942/cl03026; Figure 8) and is part of a very large cellulose binding protein with four other carbohydrate binding modules and a lot of docking interfaces for the proteins in its amino acid sequence (Figure 7).
</P><H2><SPAN class="mw-headline" id="The_Binding_Assay">The Binding Assay</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 9:</B><I>Aequorea victoria</I> green fluorescent protein in action</DIV></DIV></DIV><P>To measure the capacity and strength of the bonding between the cellulose binding domains and different types of cellulose many different assays have been made. One of the simplest and most often used is the fusion of the CBD to a reporter-protein, especially <A href="http://www.ncbi.nlm.nih.gov/pubmed/22305911a" class="external text" rel="nofollow">green or red fluorescent protein (GFP/RFP)</A> is very common. The place of the CBD is measured through the fluorescence of the fused GFP and quantification can easily been done.
</P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/18573384" class="external text" rel="nofollow">Protocol</A>:
</P><UL><LI> Harvest the <I>E coli</I>-cells producing the fusion-protein of CBD and GFP and centifuge 10 minutes at top speed.
</LI><LI> Re-suspend the cell-pellet in 50 mM Tris-HCl-Buffer (pH 8.0).
</LI><LI> Break down cells via sonication.
</LI><LI> Centrifuge at top speed for 20 minutes to get rid of the cell-debris. 
</LI><LI> Take the supernatant and measure the emission at 511 nm (excitation at 501 nm) (&lt;partinfo&gt;E0040&lt;/partinfo&gt;)
</LI><LI> Mix a definite volume of lysate with a definite volume or mass of e.g. crystalline cellulose (CC) or reactivated amorphous cellulose (RAC)
</LI><LI> Wait 15 (RAC) to 30 (CC) minutes 
</LI><LI> Take supernatant and measure the emission at 511 nm again. 
</LI><LI> The difference between the first an the second measurement is the relative quantity of what has bound to the cellulose.
</LI></UL><H2><SPAN class="mw-headline" id="Cloning_of_the_Cellulose_Binding_Domains">Cloning of the Cellulose Binding Domains</SPAN></H2><P>The cloning of the CBDs should fit to the cloning of our laccases, so the BioBricks were designed with a T7-promoter and the B0034 RBS to have a similar method of cultivation. After investigating the restriction-sites it was found, that at least for the characterization of the CBDs a quick in-frame assembly of the CBDs and a GFP would be possible, because neither the CBDs nor the GFP (&lt;partinfo&gt;I13522&lt;/partinfo&gt;) of the Partsregistry inherits a <I>Age</I>I- or <I>Ngo</I>MIV-site, which makes Freiburg-assembly possible. To do so, <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Primers" class="external text" rel="nofollow">primers</A> for the constructs &lt;partinfo&gt;BBa_K863101&lt;/partinfo&gt; (CBDcex(T7)), carrying the CBDcex domain from the <I>C. fimi</I> exoglucanase and &lt;partinfo&gt;BBa_K863111&lt;/partinfo&gt; (CBDclos(T7)), carrying the CBDclos domain from the <I>C.cellulovorans</I> binding protein were designed. The protein-BLAST of the two CBDs gave an exact picture of which bases belong to the binding domains and which do not. To be sure not to disturb the folding anyway 6 to 12 bases up and downstream of the domains as conserved sequences were kept. Even if the <A href="http://partsregistry.org/Part:BBa_K863101" class="external text" rel="nofollow">CBDcex</A>is an C-terminal domain both domains were made N-terminal, so the BioBricks can carry all regulatory parts and the linked protein can easily be exchanged. This would also be nice for other people using this part.
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> CBDcex_T7RBS</TD><TD> 80 </TD><TD> TGAATTCGCGGCCGCTTCTAGAGTAATACGACTCACTATAGGGAAAGAGGAGAAATAATGGGTCCGGCCGGGTGCCAGGT
</TD></TR><TR><TD> CBDcex_2AS-Link_compl </TD><TD> 56 </TD><TD> CTGCAGCGGCCGCTACTAGTATTAACCGGTGCTGCCGCCGACCGTGCAGGGCGTGC
</TD></TR><TR><TD> CBDclos_T7RBS </TD><TD> 73 </TD><TD> CCGCTTCTAGAGTAATACGACTCACTATAGGGAAAGAGGAGAAATAATGTCAGTTGAATTTTACAACTCTAAC
</TD></TR><TR><TD> CBDclos_2ASlink_compl</TD><TD> 63 </TD><TD> CTGCAGCGGCCGCTACTAGTATTAACCGGTGCTGCCTGCAAATCCAAATTCAACATATGTATC
</TD></TR></TBODY></TABLE></CENTER><P>The listed complementary primers added, besides the Freiburg-suffix, a two amino acid glycine-serine-linker to the end of the CBDs. This is a very short linker, but as GFP-experts and <A href="http://www.ncbi.nlm.nih.gov/pubmed/17394253" class="external text" rel="nofollow">publications</A> described GFP and CBDs are very stable proteins and should cope with a very short linker. The benefit of a short linker is that protease activity is kept minimal.
</P><P>The GFP &lt;partinfo&gt;K863121&lt;/partinfo&gt; used in the assay was an alternated version of the &lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt;. The <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Primers" class="external text" rel="nofollow">primers</A> that were made added a Freiburg-pre- and suffix to the GFP coding sequence and a His-tag to the C-terminus to get it purified for the measurements. This part &lt;partinfo&gt;BBa_K863121&lt;/partinfo&gt; (GFP_His) should be easily added to the CBDs and assembled to the fusion-proteins &lt;partinfo&gt;BBa_K863103&lt;/partinfo&gt; CBDcex(T7)+GFP_His and &lt;partinfo&gt;BBa_K863113&lt;/partinfo&gt; CBDclos(T7)+GFP_His.
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> GFP_Frei </TD><TD> 54 </TD><TD> TACGGAATTCGCGGCCGCTTCTAGATGGCCGGCATGCGTAAAGGAGAAGAACTT
</TD></TR><TR><TD> GFP_His6_compl </TD><TD> 74 </TD><TD> CTGCAGCGGCCGCTACTAGTATTAACCGGTGTGATGGTGATGGTGATGTTTGTATAGTTCATCCATGCCATGTG
</TD></TR></TBODY></TABLE></CENTER><P>Since a His-tag was added to the end of the GFP, an alternate version of the &lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt; had to be made to compare binding of GFP with and without CBD. Therefore, a forward primer was made, to amplify the whole &lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt; with the GFP_His_compl-primer to add the His-tag to the C-terminus.
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> GFP_FW_SV </TD><TD> 39 </TD><TD> acgtcacctgcgtgtagctCGTAAAGGAGAAGAACTTTT
</TD></TR></TBODY></TABLE></CENTER><P>Due to bad cleavage efficiency at the <I>Pst</I>I restriction-site in nearly all PCR-products the cloning of the CBDs (<A href="http://partsregistry.org/Part:BBa_K863102" class="external text" rel="nofollow">CBDcex(T7)</A> and <A href="http://partsregistry.org/Part:BBa_K863112" class="external text" rel="nofollow">CBDclos(T7)</A>) and especially the insertion of <A href="http://partsregistry.org/Part:BBa_K863121" class="external text" rel="nofollow">GFP_His</A> to the CBDs took a lot more time as expected. This was due to the fact that no additional bases were added to the 5' termini of the designed primers, thereby greatly reducing cleavage efficiency. When the problem was discovered and protocols adjusted or primers extended, cloning got a lot quicker and more successful.
</P><P>As the project went on the T7-constructs did not seem to work. One suspected reason was that a stop codon (TAA) that was accidentally introduced between the RBS and ATG could be the reason. To solve the problem a change to a constitutive promoter (&lt;partinfo&gt;BBa_J61101&lt;/partinfo&gt;) was made using the Freiburg-assembly. Therefore Freiburg forward primers for the CBDs were made.
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> CBDcex_Freiburg-Prefix</TD><TD> 54 </TD><TD> GCTAGAATTCGCGGCCGCTTCTAGATGGCCGGCGGTCCGGCCGGGTGCCAGGTG
</TD></TR><TR><TD> CBDclos_Freiburg-Prefix </TD><TD> 57 </TD><TD> GCTAGAATTCGCGGCCGCTTCTAGATGGCCGGCTCATCAATGTCAGTTGAATTTTAC
</TD></TR></TBODY></TABLE></CENTER><P>The primers arrived just a few days before wiki freeze (Europe). Switching to the constitutive promoter had no obvious effect (no green colonies or culture), neither had changing the expression strain from <I>E. coli</I> KRX to <I>E. coli</I> BL21 for the T7-construct.
</P><H2><SPAN class="mw-headline" id="Since_Regionals">Since Regionals</SPAN></H2>
After Amsterdam two approaches were made. One changing the order of the fused proteins and one altering the linker between CBD and the GFP. Both started simultaneously. <DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 10:</B> PCR-product of BBa_K863104 with S3N10-primers</DIV></DIV></DIV><P>One reason for the problems could be the too short space in between the two proteins, resulting in CBD and GFP hampering each other from folding correctly. To test and solve this, a very long linker with three serines followed by ten asparagines should be assembled in the already existing parts via a blunt end cloning.
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> S3N10_Cex_compl </TD><TD> 40 </TD><TD> TTGTTGTTGTTCGAGCTCGAGCCGACCGTGCAGGGCGTGC
</TD></TR><TR><TD> S3N10_Clos_compl </TD><TD> 40 </TD><TD> TTGTTGTTGTTCGAGCTCGAGCTGCCGCCGACCGTGCAGG
</TD></TR><TR><TD> S3N10_GFP </TD><TD> 40 </TD><TD> CAATAACAATAACAACAACCGTAAAGGAGAAGAACTTTTC
</TD></TR></TBODY></TABLE></CENTER><P>While the PCR worked properly, ligation of the linearized new plasmid was not successful and no transformed colonies could be found in two attempts. A reason for this could be insufficient phosphorylation of the PCR product.
Because already Freiburg pre- and suffix primers for the CBDs were at hand the order of the fusion proteins could easily been changed, when a Freiburg suffix primer for the GFP would be available, so it was ordered.
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TD> GFP_Freiburg_compl </TD><TD> 61 </TD><TD> ACGTCTGCAGCGGCCGCTACTAGTATTAACCGGTTTTGTATAGTTCATCCATGCCATGTGT
</TD></TR></TBODY></TABLE></CENTER><P>When changing the fusing proteins to GFP N-terminal and CBD C-terminal an additional single GFP was cloned in a plasmid behind a J61101 RBS to see the expression of the J23100/J61101 promoter/RBS and to have a corresponding protein for the assay as a negative control. The result was the same as before, no green colonies. The colony PCRs and digestions showed the right bands, but no construct and even the single GFP was expressed correctly. Sequencing later showed that a deletion of one of the first few bases of the ORF caused a frame shift. At this point in time it seemed as if the promoter/RBS did not work properly.
So in spite of continuing working as planed on the linker between GFP and the CBDs, the efforts now focused on the usage of promoter and RBS. In the last week of lab work it was tried to assemble GFP with a CBD behind the strong B0034 RBS, followed by cutting this piece out with <I>Xba</I>I and <I>Pst</I>I and ligate this insert via a standard suffix insertion with the J23100 promoter. After some unsuccessful attempts <I>Age</I>I ran empty and could not be ordered in time. A last quick-shot with a PCR product of an old (but correct) assembly (CBDcex-GFP) and a single GFP_Freiburg went well in adding it to the RBS but did not show green glowing colonies when fused to the promoter.
</P><H2><SPAN class="mw-headline" id="Literature"> Literature </SPAN></H2><P>Kavoosi <I>et al.</I> (2007) Strategy for selecting and characterizing linker peptides for CBM9-tagged fusion proteins expressed in Escherichia coli. <I>Biotechnol Bioeng.</I> Oct 15;98(3):599-610.
</P><P>Urbanowicz <I>et al.</I> (2007) A tomato endo-beta-1,4-glucanase, SlCel9C1, represents a distinct subclass with a new family of carbohydrate binding modules (CBM49). <I>J Biol Chem.</I> Apr 20;282(16):12066-74. Epub 2007 Feb 23.
</P><P>Sugimoto <I>et al.</I> (2012) Cellulose affinity purification of fusion proteins tagged with fungal family 1 cellulose-binding domain. <I>Protein Expr Purif.</I> Apr;82(2):290-6. Epub 2012 Jan 28.
</P><P>Hong <I>et al.</I> (2008) Bioseparation of recombinant cellulose-binding module-proteins by affinity adsorption on an ultra-high-capacity cellulosic adsorbent. <I>Anal Chim Acta.</I> Jul 28;621(2):193-9. Epub 2008 May 27.
</P></DIV></DIV><TABLE cellspacing="10"><TBODY><TR><TD><A href="/File:Promega_logo_180x109.gif" title="File:Promega logo 180x109.gif">55px</A></TD></TR></TBODY></TABLE><DIV id="content-footer-box"><DIV class="content-footer-box-left"><DIV class="about"><P>About our Wiki</P><P>Fifteen students from different fields of study from the Bielefeld University aspired to complete a project called TOXIC COMPOUNDS IN NATURAL WATER - A CASE FOR LACCASE. In his Wiki we document our achievements and our approach. </P></DIV><DIV class="contact"><P>Contact Us</P><P>UniversitÃ¤t Bielefeld
UniversitÃ¤tsstraÃe 25
D-33615 Bielefeld<A href="mailto:info@igem-bielefeld.de">info(at)igem-bielefeld.de</A></P></DIV></DIV><DIV class="content-footer-box-right"><DIV class="navigate"><P>Navigate our Wiki</P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany">Start</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Team</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Project</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Results</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Since Regionals</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Human Practice</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Bielefeld-Germany/Results/cbc">http://2012.igem.org/Team:Bielefeld-Germany/Results/cbc</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Results/cbc" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Results/cbc" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/cbc&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/cbc&amp;oldid=298200" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>