<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Protocols_molecular_genetics"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Protocols/molecular_genetics">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Protocols/molecular_genetics&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/molecular_genetics&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/molecular_genetics&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Protocols/molecular_genetics" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Protocols/molecular genetics</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="navigation"><DIV style=" z-index:10000;"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany">Start</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Team</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Bielefeld-University">Bielefeld University</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Project</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Achievments">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Database">Database</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Outlook">Outlook</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Results</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Datapage">Datapage</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Laccases">Laccases</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/immo">Immobilization</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/substrate">Substrate Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/cbc">Cellulose Binding Domain</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/vector">Shuttle Vector</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/london">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Since Regionals</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Summary</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Expert">Expert</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Politics">Politics</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Sewage_treatment_plant">Industry</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human Practices/Outreach">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/StudentAcademy">Education</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Environment">Environment</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV></DIV><DIV id="page-title"><SPAN id="page-title-text">
                Molecular
            </SPAN></DIV><DIV id="grey_bg"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Bacterial_specific_methods"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Bacterial specific methods</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Generating_electrocompetent_cells"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Generating electrocompetent cells</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Transformation_via_electroporation"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Transformation via electroporation</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Transformation_of_Single_Step_.28KRX.29_Competent_Cells_by_Promega"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Transformation of Single Step (KRX) Competent Cells by Promega</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Standard_BioBrick_Assembly"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Standard BioBrick Assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Suffix_Insertion"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Suffix Insertion</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Prefix_Insertion"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Prefix Insertion</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Variations"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">Variations</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-9"><A href="#Standard_Freiburg_BioBrick_Assembly"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Standard Freiburg BioBrick Assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Suffix_Insertion_2"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">Suffix Insertion</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Prefix_Insertion_2"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">Prefix Insertion</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Variations_2"><SPAN class="tocnumber">1.5.3</SPAN><SPAN class="toctext">Variations</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Restriction_analysis"><SPAN class="tocnumber">1.5.4</SPAN><SPAN class="toctext">Restriction analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Colony_PCR"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Phusion_PCR"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Phusion PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Site_Directed_Mutagenesis"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Site Directed Mutagenesis</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Plants_specific_methods"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Plants specific methods</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Arabidopsis_thaliana:_Growth_Conditions_and_Plant_Material"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Arabidopsis thaliana: Growth Conditions and Plant Material</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Arabidopsis_thaliana:_Total_RNA_Isolation"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Arabidopsis thaliana: Total RNA Isolation</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Arabidopsis_thaliana:_cDNA_Synthesis"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Arabidopsis thaliana: cDNA Synthesis</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#PCR_for_A.thaliana_Laccase_Ampflification"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">PCR for A.thaliana Laccase Ampflification</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Yeast_Specific_Methods"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Yeast Specific Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Pichia_pastoris:_Growth_Conditions"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Pichia pastoris: Growth Conditions</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Genomic_DNA_Isolation"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Genomic DNA Isolation</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Generating_competent_yeast_cells"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Generating competent yeast cells</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Transformation_of_yeast_cells"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Transformation of yeast cells</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#General_methods"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">General methods</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Ethanol_precipitation_to_clean_DNA"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Ethanol precipitation to clean DNA</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Gibson_assembly"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Gibson assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Preparation_of_DNA_molecules_for_in_vitro_recombination"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">Preparation of DNA molecules for in vitro recombination</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#In_vitro_recombination"><SPAN class="tocnumber">4.2.2</SPAN><SPAN class="toctext">In vitro recombination</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#KOD_DNA_polymerase_reaction"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">KOD DNA polymerase reaction</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Bacterial_specific_methods"> Bacterial specific methods </SPAN></H1><H2><SPAN class="mw-headline" id="Generating_electrocompetent_cells"> Generating electrocompetent cells </SPAN></H2><P>(<I>E.coli</I>-<A href="http://www.promega.com/products/cloning-and-dna-markers/cloning-tools-and-competent-cells/bacterial-strains-and-competent-cells/single-step-_krx_-competent-cells/" class="external text" rel="nofollow"><I>KRX</I></A>, 
<A href="http://www.genomics.agilent.com/files/Manual/200249.pdf" class="external text" rel="nofollow"><I>XLI Blue</I></A> and
<A href="http://www.merckmillipore.com/is-bin/INTERSHOP.enfinity/WFS/Merck-DE-Site/de_DE/-/EUR/ViewPDF-Print.pdf;sid=MURSuevozHZ2ubuzCYY-7kMotsdASYFPCWpM71hFGa-SnWVmHaQxE-3WJh_fBJar5MJCpegxzHx7vmIhk8DOvz7AELBL_Sc4At7qmFPx-WwuKfv0Mp2vgcLv?RenderPageType=ProductDetail&amp;CatalogCategoryID=&amp;ProductUUID=6t.b.s1OalsAAAEY0BwK0D3I&amp;PortalCatalogUUID=tUmb.s1O2d0AAAEXcutI1u8e" class="external text" rel="nofollow"><I>Rosetta gami</I></A> )
</P><P>Materials:
</P><UL><LI>550 mL <A href="https://2012.igem.org/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/Materials#LB_medium" class="external text" rel="nofollow">LB-Medium</A></LI><LI>1 L cooled bidest. H<SUB>2</SUB>O (4 Â°C)
</LI><LI>150 mL cooled 10Â % glycerine (4 Â°C)
</LI><LI>10 pre-cooled 50 mL Falcons (-18Â°C)
</LI></UL><P>Protocol:
</P><UL><LI>Inoculate 2x3 mL <A href="https://2012.igem.org/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/Materials#LB_medium" class="external text" rel="nofollow">LB</A> with bacterial stock; incubate over night at 37 Â°C and 140 rpm
</LI><LI>Inoculate 2x250 mL <A href="https://2012.igem.org/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/Materials#LB_medium" class="external text" rel="nofollow">LB</A> with the over night cultures in 1-litre-flask (with baffles) at 37 Â°C and 140 rpm
</LI><LI>Incubate until OD<SUB>600</SUB> 0.4 - 0.6
</LI><LI>Cool the culture 15-30 minutes on ice
</LI></UL><P>Important: keep your cells at 2-4 Â°C during the following steps
</P><UL><LI>Divide the cultures into cooled 50 mL Falcons and centrifugate for 15 minutes (4000 rpm, 4 Â°C) IMPORTANT: slowly accelerate and deccelerate 
</LI><LI>Discard  supernatant
</LI><LI>Resuspend  cell pellet in 5 mL cooled bidest H<SUB>2</SUB>O (and don't get frustrated while doing it, keep shaking gently)
</LI><LI>Pool two suspensions each, add bidest H<SUB>2</SUB>O up to 50 mL and centrifugate again (see centrifugation above)
</LI><LI>Discard supernatant (Keep in mind: keep your cells at 2-4 Â°C)
</LI><LI>Resuspend pellet in 5 mL cooled bidest H<SUB>2</SUB>O
</LI><LI>Add bidest H<SUB>2</SUB>O up to 50 mL and centrifugate again (see centrifugation above)
</LI><LI>Discard supernatant (Keep in mind: keep your cells at 2-4 Â°C)
</LI><LI>Resuspend pellet in 5 mL cooled 10Â % glycerine
</LI><LI>Transfer suspensions in two 50 mL Falcons and centrifugate again (see centrifugation above)
</LI><LI>Discard supernatant
</LI><LI>Add volume of 10Â % cooled glycerine (2-4Â°C) that is approximately equal to the volume of the pellet and resuspend
</LI><LI>Divide cells in 100 Î¼L aliquots and freeze in liquid N<SUB>2</SUB> immediately
</LI><LI>Store at -80 Â°C
</LI></UL><H2><SPAN class="mw-headline" id="Transformation_via_electroporation"> Transformation via electroporation </SPAN></H2><UL><LI> Thaw 50 ÂµL competent <I>E.coli</I> cells on ice, dilute with icecold 50 ÂµL glycerine (10Â %) if necessary
</LI><LI> Add 0.5-2 ÂµL plasmid to 50 Âµl electrocompetent cells
</LI><LI> Store cells on ice for 1 minute 
</LI><LI> Electroporate at U = 2.5 kV, C = 25 ÂµF, R = 400 Î 
</LI><LI> Transfer transformation reaction to 450 ÂµL SOC-Medium and shake 1 h at 37 Â°C 
</LI><LI> Centrifuge 2 min at 800 rpm and plate on selective LB-Medium
</LI></UL><H2><SPAN class="mw-headline" id="Transformation_of_Single_Step_.28KRX.29_Competent_Cells_by_Promega">Transformation of Single Step (KRX) Competent Cells by Promega</SPAN></H2><P>Using <A href="http://www.promega.com/~/media/Files/Resources/Protocols/Technical%20Bulletins/101/Single%20Step%20Competent%20Cells%20Protocol.ashx" class="external text" rel="nofollow">protocol <I>E. coli</I> KRX single step competent cells by Promega</A></P><UL><LI> Remove Single Step (KRX) Competent Cells from â70 Â°C, and place on ice for 5 minutes or until just thawed.
</LI><LI> Add 1â50 ng of DNA (in a volume not greater than 5 Î¼L) to the Single Step (KRX) Competent Cells. Move the pipette tip through the cells while dispensing. Quickly flick the tube several times. Do not vortex!
</LI><LI> Immediately return the tubes to ice for 5â30 minutes
</LI><LI> Heat-shock cells for 15â20 seconds in a water bath at exactly 42 Â°C. Do not shake.
</LI><LI> Immediately place the tubes on ice for 2 minutes.
</LI><LI> Add 450 Î¼L of room-temperature <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#SOC_medium" class="external text" rel="nofollow">SOC-Medium</A> to each transformation reaction, and incubate for 60 minutes at 37 Â°C with shaking (approximately 225 rpm). For best transformation efficiency, lay the tubes on their sides and tape them to the platform.
</LI><LI> For each transformation reaction, we recommend plating 100 Î¼L of undiluted cells and 1:10 and 1:100 cell dilutions on antibiotic plates. Incubate the antibiotic plates at 37 Â°C over night.
</LI></UL><H2><SPAN class="mw-headline" id="Standard_BioBrick_Assembly">Standard BioBrick Assembly</SPAN></H2><P>modified from <A href="http://openwetware.org/wiki/Silver:_BB_Strategy" class="external text" rel="nofollow">Silver lab</A>: 
This assembly method can be used for BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, called vector. So you have to differentiate between a prefix and a suffix insertion. 
</P><H3><SPAN class="mw-headline" id="Suffix_Insertion"> Suffix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 ÂµL volume, 1 ÂµL 10x Tango buffer, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Xba</I>I</A>, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Pst</I>I</A>. Digest for 2 h to 4 h at 37 Â°C, afterwards inactivation for 20 min at 80 Â°C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI><LI> Digestion of vector about 700 ng DNA / 10 ÂµL volume, 1 ÂµL 10x orange buffer, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Spe</I>I</A>, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Pst</I>I</A>. Digest for 2 h to 4 h at 37 Â°C, afterwards inactivation for 20 min at 80 Â°C. Add 1 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SAP (shrimp alcaline phosphatase)</A> and 1.2 ÂµL 10 x SAP buffer, incubate for 1 h at 37 Â°C. Clean up the vector via gel electrophoresis and <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. 
</LI><LI> Ligation: after digestion and clean-up: 50 - 100 ng of vector, 3 - 10 fold molar access of insert, 20 ÂµL ligation volume, 2 ÂµL T4-Ligase-Buffer, 1 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 Â°C. Then: store at -20 Â°C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Prefix_Insertion"> Prefix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 ÂµL volume, 1 ÂµL 10x BamHI buffer, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Eco</I>RI</A>, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Spe</I>I</A>. Digest for 2 h at 37 Â°C, afterwards inactivation for 20 min at 80 Â°C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI><LI> Digestion of vector about 700 ng DNA / 10 ÂµL volume, 1 ÂµL 10 x Tango buffer, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Eco</I>RI</A>, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Xba</I>I</A>. Digest for 2h at 37 Â°C, afterwards inactivation for 20 min at 80 Â°C. Add 1 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SAP (shrimp alkaline phosphatase)</A> and 1.2 ÂµL 10 x SAP buffer, incubate for 1 h at 37 Â°C. Clean up the vector via gel electrophoresis and <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. 
</LI><LI> Ligation: after digestion and clean-up: 50 - 100 ng of vector, 3 - 10 fold molar access of insert, 20 ÂµL ligation volume, 2 ÂµL T4-Ligase-Buffer, 1 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 Â°C. Then: store at -20 Â°C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Variations"> Variations </SPAN></H3><UL><LI> A digestion over night is possible. If you digest over night use only 0.1 ÂµL restriction enzyme. 
</LI><LI> It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up.
</LI></UL><H2><SPAN class="mw-headline" id="Standard_Freiburg_BioBrick_Assembly"> Standard Freiburg BioBrick Assembly </SPAN></H2><P>Modified from <A href="http://openwetware.org/wiki/Silver:_BB_Strategy" class="external text" rel="nofollow">Silver lab</A> and <A href="http://partsregistry.org/Assembly_standard_25" class="external text" rel="nofollow">Assembly standard 25</A>:
</P><P>This assembly method can be used for fusion protein assemblies with BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, which is called vector and needs to be available in the BioBrick <A href="http://partsregistry.org/Assembly_standard_25" class="external text" rel="nofollow">Assembly standard 25</A>. You have to differentiate between a prefix and a suffix insertion.
</P><H3><SPAN class="mw-headline" id="Suffix_Insertion_2"> Suffix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 ÂµL volume, 1 ÂµL 10x NEB buffer 4 + 0.1 ÂµL 100x BSA, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Ngo</I>MIV (NEB)</A>, 1 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Pst</I>I</A>. Digest for 2 h at 37 Â°C, afterwards inactivation for 20 min at 80 Â°C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI><LI> Digestion of vector about 700 ng DNA / 10 ÂµL volume, 1 ÂµL 10x orange buffer, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Age</I>I</A>, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Pst</I>I</A>. Digest for 2 h at 37 Â°C, afterwards inactivation for 20 min at 80 Â°C. Add 1 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SAP (shrimp alkaline phosphatase)</A> and 1.2 ÂµL 10x SAP buffer, incubate for 1 h at 37 Â°C. Clean up the vector via gel electrophoresis and<A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. 
</LI><LI> Ligation: after digestion and clean-up: 50 - 100 ng of vector, 3 - 10 fold molar access of insert, 20 ÂµL ligation volume, 2 ÂµL T4-Ligase-Buffer, 1 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 Â°C. Then: store at -20 Â°C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Prefix_Insertion_2"> Prefix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 ÂµL volume, 1 ÂµL 10x orange buffer, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Eco</I>RI</A>, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Age</I>I</A>. Digest for 2 h to 4 h at 37 Â°C, afterwards inactivation for 20 min at 80 Â°C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI><LI> Digestion of vector about 700 ng DNA / 10 ÂµL volume, 1 ÂµL 10 x NEB buffer 4, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Eco</I>RI</A>, 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Ngo</I>MIV (NEB)</A>. Digest for 2 h to 4 h at 37 Â°C, afterwards inactivation for 20 min at 80 Â°C. Add 1 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SAP (shrimp alcaline phosphatase)</A> and 1.2 ÂµL 10 x SAP buffer, incubate for 1 h at 37 Â°C. Clean up the vector with a <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. 
</LI><LI> Ligation: after digestion and clean-up: 50 - 100 ng of vector, 3 - 10 fold molar access of insert, 20 ÂµL ligation volume, 2 ÂµL T4-Ligase-Buffer, 1 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 Â°C. Then: store at -20 Â°C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Variations_2"> Variations </SPAN></H3><UL><LI> A digestion over night is possible. If you digest over night use only 0.1 ÂµL restriction enzyme. 
</LI><LI> It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up. 
</LI></UL><H3><SPAN class="mw-headline" id="Restriction_analysis"> Restriction analysis </SPAN></H3><UL><LI> Digest BioBrick of interest: about 400 ng DNA / 10 ÂµL volume, 1 ÂµL 10x orange buffer, 0.5 ÂµL <I>Not</I>I or <I>Pst</I>I. Digest for 2 h at 37 Â°C. <I>Not</I>I is used to determine the length of the BioBrick and the plasmid backbone, <I>Pst</I>I is used to determine the length of the BioBrick in the plasmid backbone.
</LI><LI> Gel electrophoresis: add 2 ÂµL loading buffer to every digestion mix, apply about 100 - 200 ng DNA / pocket in gel. Don't forget to apply the uncut BioBrick as well. A good agarose concentration for BioBricks between 0.2 and 3 kb is 1.5Â %. The smaller your BioBrick of interest is the higher the agarose concentration should be and vice versa. The gel electrophoresis is made with TAE-buffer. Be sure that you melt your agarose gel in the same buffer you use for the electrophoresis later. 
</LI></UL><H2><SPAN class="mw-headline" id="Colony_PCR"> Colony PCR </SPAN></H2><P>Colony PCR was done with <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> GoTaq DNA-polymerase (Promega)</A> and <A href="/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/Meterials&amp;action=edit&amp;redlink=1" class="new" title="Team:Bielefeld-Germany/Protocols/Meterials (page does not exist)"> BIOTAQ DNA-polymerase (Bioline)</A>.
</P><UL><LI> Pick one colony with a sterile tip and put it in one PCR tube with mastermix.
</LI><LI> The reaction mix and PCR program is described below in the table.
</LI></UL><P><B>PCR reaction mix</B></P><TABLE class="wikitable" style="text-align: center; width: 300px: height: 300px;" border="0}"><TBODY><TR><TH> GoTaq DNA-polymerase
</TH><TH> BIOTAQ DNA-polymerase
</TH></TR><TR><TD>10x NH4 buffer
</TD><TD>1.5 Âµl
</TD><TD>0 Âµl
</TD></TR><TR><TD>5x green or colorless GoTaq reaction buffer
</TD><TD>0 Âµl
</TD><TD>5 Âµl
</TD></TR><TR><TD>MgCl2 (50 mM)
</TD><TD>0.6 Âµl
</TD><TD>0.5 Âµl
</TD></TR><TR><TD>dNTPs (10 mM)
</TD><TD>0.3 Âµl
</TD><TD>0.5 Âµl
</TD></TR><TR><TD>Primermix FW and RV (10 mM)
</TD><TD>0.75 Âµl
</TD><TD>0.5 Âµl
</TD></TR><TR><TD>DNA-Polymerase
</TD><TD>0.4 Âµl
</TD><TD>0.125 Âµl
</TD></TR><TR><TD>bidest ddH2O
</TD><TD>11.45 Âµl
</TD><TD>18.375 Âµl
</TD></TR><TR><TD>total volume
</TD><TD>15 Âµl
</TD><TD>25 Âµl
</TD></TR></TBODY></TABLE><P><B>PCR cycle program</B></P><TABLE class="wikitable" style="text-align: center; width: 300px: height: 300px;" border="0}"><TBODY><TR><TH> GoTaq DNA-polymerase
</TH><TH> BIOTAQ DNA-polymerase
</TH></TR><TR><TD>Initiale denaturation
</TD><TD>3 min, 94 Â°C
</TD><TD>8 min, 95 Â°C
</TD></TR><TR><TD>35 cycles of
</TD></TR><TR><TD>denaturation
</TD><TD>30 s, 94 Â°C
</TD><TD>30 s, 95 Â°C
</TD></TR><TR><TD>primer annealing
</TD><TD>30 s, 55 Â°C
</TD><TD>30 s, 58 Â°C
</TD></TR><TR><TD>elongation
</TD><TD>1 min / 1 kb template, 72 Â°C
</TD><TD>30 s / 1 kb template, 72 Â°C
</TD></TR><TR><TD>final elongation
</TD><TD>5 min, 72 Â°C
</TD><TD>5 min, 72 Â°C
</TD></TR></TBODY></TABLE><UL><LI> Gel electrophoresis: check the fragment size
</LI><LI> Plate the correct colonies
</LI></UL><H2><SPAN class="mw-headline" id="Phusion_PCR"> Phusion PCR </SPAN></H2><P>Phusion PCR was done with <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> Phusion HF DNA-polymerase (Finnzymes)</A>.
</P><UL><LI> One reaction mix contains: 
</LI><LI> 10 ÂµL 5x Phusion buffer
</LI><LI> 1 ÂµL dNTPs (10 mM)
</LI><LI> 0.5 ÂµL pro primer (0.5 ÂµM)
</LI><LI> 250 ng DNA
</LI><LI> 0.5 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> Phusion HF DNA-polymerase (Finnzymes)</A></LI><LI> add ddH<SUB>2</SUB>O to 50 ÂµL
</LI></UL><LI> PCR program: 
</LI><LI> Initiale denaturation: 3 min, 98 Â°C
</LI><LI> 35 cycles of: 
</LI><LI> 10 s, 98 Â°C
</LI><LI> 20 s, annealing temperature depends on primer pair
</LI><LI> 30 s / 1 kb template, 72 Â°C
</LI><LI> Finale elongation: 5 min, 72 Â°C
</LI><LI> Gel electrophoresis: check the fragment size
</LI><H2><SPAN class="mw-headline" id="Site_Directed_Mutagenesis"> Site Directed Mutagenesis </SPAN></H2><UL><LI>Design Primers using the QuickChange Primer Design tool
</LI><LI>Run mutagenesis PCR
</LI><LI>One reaction mix contains:
</LI><LI>5 ÂµL 10x buffer
</LI><LI>0.4 ÂµL Pfu polymerase
</LI><LI>1 ÂµL dNTPs
</LI><LI>39.6 ÂµL H<SUB>2</SUB>O
</LI><LI>1 ÂµL DMSO
</LI><LI>2 ÂµL primer mix
</LI><LI>1 ÂµL template 
</LI></UL><LI>PCR program:
</LI><LI>Start: 2 min, 94 Â°C
</LI><LI>16 cycles of:
</LI><LI>45 s, 95 Â°C
</LI><LI>30 s, 55 Â°C
</LI><LI>2 min/kb, 73 Â°C 
</LI><LI>Finish: 5 min, 72 Â°C 
</LI><LI>Control the amplified PCR-fragments by gel electrophoresis
</LI><LI>Digest the template plasmid by adding 1 ÂµL of DpnI and incubate for 2-3 h
</LI><LI>Purify the PCR product with a PCR clean-up kit
</LI><LI>Transform 3 ÂµL of the purified PCR product into electrocompetent E. coli XL1 blue cells
</LI><LI>Screen the transformants using restiction digest and sequencing
</LI><H1><SPAN class="mw-headline" id="Plants_specific_methods"> Plants specific methods </SPAN></H1><H2><SPAN class="mw-headline" id="Arabidopsis_thaliana:_Growth_Conditions_and_Plant_Material"><I>Arabidopsis thaliana</I>: Growth Conditions and Plant Material</SPAN></H2><P>Six weeks old <I>A. thaliana</I> plants, ecotype Columbia 0 (wildtype), have been gratefully offered by Patrick Treffon and Thorsten Seidel. They have been cultivated under normal day conditions (14 hours light [100 Âµmol â quanta m<SUP>-2</SUP>s<SUP>-1</SUP>] at 21Â°C, 10 hours darkness at 18Â°C). For induction of the formation of siliques the plants were shifted into long day conditions (16 hours light [100 Âµmol â quanta m<SUP>-2</SUP>s<SUP>-1</SUP>] at 21Â°C, 18 hours darkness at 18Â°C). After two weeks in long day conditions the plants have developed 2 cm long siliques. The siliques were harvested and frozen in liquid nitrogen for further use.
</P><H2><SPAN class="mw-headline" id="Arabidopsis_thaliana:_Total_RNA_Isolation"><I>Arabidopsis thaliana</I>: Total RNA Isolation</SPAN></H2><P>The frozen plant material has to be grinded in a precooled mortar in liquid nitrogen. About 120 mg of pulverized plant material are transfered into a precooled 2 ml Eppendorf tube and kept frozen until the following steps:
</P><UL><LI>Add 0.5 ml lysis buffer and immediately homogenize through rough shaking.
</LI><LI>Add 0.5 ml of saturated phenol and mix strongly.
</LI><LI>Add 0.5 ml of chloroform isoamyl alcohol (24:1) and vortex again at high speed for at least 30 seconds.
</LI><LI>Centrifugate for 5 min at 13,000 rpm.
</LI><LI>The lower phase contains now lipids and lipophilic compounds. The upper phase contains nucleic acids (~ 550 Âµl) and has to be carefully transferred into a new 2 ml Eppendorf tube. This tube has to be filled with 0.5 ml saturated phenol and 0.5 ml chloroform isoamyl alcohol (24:1). Mix immediately.
</LI><LI>Centrifugate at 13,000 rpm for 3 minutes.
</LI><LI>Prepare a new 2 ml Eppendorf tube with 1 ml of chloroform isoamyl alcohol (24:1). Transfer the upper aqueous phase (~ 540 Âµl) containing the protein purified nucelic acids into the new tube and vortex strongly.
</LI><LI>Centrifugate at 13,000 rpm for 3 minutes.
</LI><LI>Prepare a new 1.5 ml Eppendorf tube with 0.5 ml of pure isopropanol. For the last time transfer the upper phase (~ 400 Âµl) into the new tube and mix gently.
</LI><LI>Incubate the mixture over night at -20Â°C. The nucleic acids will precipitate.
</LI><LI>Centrifugate the samples at 13,000 rpm for 15 minutes at 4Â°C.
</LI><LI>Discard the supernatant and resuspend the pellet in 375 Âµl sterile H<SUB>2</SUB>O.
</LI><LI>Add 125 Âµl 8 M lithium chloride and incubate for 2 hours on ice at 4Â°C. At this point most of the RNA is going to be precipitated.
</LI><LI>Centrifugate at 13,000 rpm at 4Â°C and discard the supernatant.
</LI><LI>Wash the pellet with 100 Âµl 70% (v/v) ethanol and discard it after centrifugation.
</LI><LI>Dry the pellet at room temperature.
</LI><LI>Dissolve the pellet in sterile H<SUB>2</SUB>O (~ 25 Âµl, depending on the size of the pellet).
</LI><LI>Check the quantity and quality of the RNA with a Nanodrop spectrophotometer before starting with a cDNA synthesis.
</LI></UL><H2><SPAN class="mw-headline" id="Arabidopsis_thaliana:_cDNA_Synthesis"><I>Arabidopsis thaliana</I>: cDNA Synthesis</SPAN></H2><P>After a successful total RNA isolation the RNA has to be translated in cDNA through RT-PCR:
</P><UL><LI>Take 3 Âµg/Âµl of total RNA and add sterile H<SUB>2</SUB> to 8 Âµl.
</LI></UL><P>Additionally add 
</P><TABLE border="0" rules="cols" align="center"><TBODY><TR><TD align="center"> 1.1 mM </TD><TD align="center"> Oligo-d(T)-Primer </TD></TR><TR><TD align="center"> 0.83 mM </TD><TD align="center"> dNTPs </TD></TR><TR><TD align="center"> 3.5 Âµl </TD><TD align="center"> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE><UL><LI>Vortex and centrifugate shortly.
</LI><LI>Incubate the samples for 10 minutes at 70Â°C.
</LI><LI>Immediately transfer the samples into ice water for 5 minutes.
</LI><LI>After cooling the samples centrifugate shortly.
</LI><LI>To start the synthesis add
</LI></UL><TABLE border="0" rules="cols" align="center"><TBODY><TR><TD align="center"> 6 Âµl </TD><TD align="center"> 5xMMLV-Puffer </TD></TR><TR><TD align="center"> 4.5 Âµl </TD><TD align="center"> H<SUB>2</SUB>O </TD></TR><TR><TD align="center"> 1 Âµl </TD><TD align="center"> MMLV-reverse Transkriptase [200 U/Âµl] </TD></TR><TR><TD align="center"> 0.5 Âµl </TD><TD align="center"> RNasin RNase-Inhibitor [40 U/Âµl] </TD></TR></TBODY></TABLE><UL><LI>Mix the samples and centrifugate shortly.
</LI><LI>Incubate for 1 hour at 42 Â°C to translate the RNA into cDNA.
</LI><LI>Transfer the samples to 70 Â°C for 15 minutes to stop the reaction.
</LI><LI>The new synthesized cDNA can be used for PCR after diluting 1:10 with water. Store the cDNA at -20 Â°C.
</LI></UL><H2><SPAN class="mw-headline" id="PCR_for_A.thaliana_Laccase_Ampflification">PCR for <I>A.thaliana</I> Laccase Ampflification</SPAN></H2><P>The PCR setup was determined thorugh primer length and GC content.
The samples contained the following reagents:
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> concentration
</TH></TR><TR><TD> buffer </TD><TD> 10 ÂµL
</TD></TR><TR><TD><A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Primers" class="external text" rel="nofollow">Primer</A> forward </TD><TD> 1 ÂµL
</TD></TR><TR><TD><A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Primers" class="external text" rel="nofollow">Primer</A> reverse </TD><TD> 1 ÂµL
</TD></TR><TR><TD> dNTPÂ´s </TD><TD> 1 ÂµL
</TD></TR><TR><TD> Template </TD><TD> 3 ÂµL
</TD></TR><TR><TD> Phusion polymerase </TD><TD> 1 ÂµL
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> ad 50 ÂµL
</TD></TR></TBODY></TABLE><P>Due to the length of the primers cycles with two different temperatures have been used:
</P><TABLE class="wikitable"><TBODY><TR><TH> temperature </TH><TH> duration
</TH></TR><TR><TD>98Â°C </TD><TD> 30 sec
</TD></TR><TR><TD> 46Â°C /74Â°C </TD><TD> 10 sec
</TD></TR><TR><TD> 47Â°C/75Â°C </TD><TD> 20 sec
</TD></TR><TR><TD> 72Â°C </TD><TD> 1 min
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Yeast_Specific_Methods"> Yeast Specific Methods </SPAN></H1><H2><SPAN class="mw-headline" id="Pichia_pastoris:_Growth_Conditions"><I>Pichia pastoris: Growth Conditions </I></SPAN></H2><P>The cultivation conditions of the yeast <I>Pichia pastoris</I> are: 30 Â°C at 130 rpm.
</P><H2><SPAN class="mw-headline" id="Genomic_DNA_Isolation"> Genomic DNA Isolation </SPAN></H2><P>The complete genome isolation was done with the <A href="http://www.promega.com/resources/protocols/technical-manuals/0/wizard-genomic-dna-purification-kit-protocol/" class="external text" rel="nofollow">Promega Wizard genomic DNA purification system kit</A>.
</P><DIV style="text-align:justify;"><UL><LI>Pellet 10 mL of over-night liquid culture grown in YPD broth in a 1.5 mL tube by centrifugation at 14,000 x g for 2 minutes.
</LI><LI>Remove the supernatant.
</LI><LI>Resuspend the cells in 90 Î¼L of 50 mM EDTA.
</LI><LI>Add 10 Î¼L of 1000u lyticase and pipet 4 times to mix.
</LI><LI>Incubate the sample at 37Â°C for 60 minutes to digest the cell wall.
</LI><LI>Centrifuge the sample at 14,000 Ã g for 2 minutes and then remove the supernatant.
</LI><LI>Add 300 Î¼l of Nuclei Lysis Solution to the cell pellet and pipet to mix.
</LI><LI>Add 100 Î¼l of Protein Precipitation Solution and vortex at high speed for 20 seconds.
</LI><LI>Let the sample sit on ice for 5 minutes.
</LI><LI>Centrifuge at 14,000 Ã g for 3 minutes.
</LI><LI>Transfer the supernatant containing the DNA to a clean 1.5 ml tube containing 300 Î¼l of room temperature isopropanol.
</LI><LI>Gently mix by inversion until the DNA is visible.
</LI><LI>Centrifuge at 14,000 Ã g for 2 minutes.
</LI><LI>Carefully decant the supernatant and drain the tube on clean absorbent paper.
</LI><LI>Add 300 Î¼l of room temperature 70% ethanol and invert the tube several times to wash the DNA pellet.
</LI><LI>Centrifuge at 14,000 Ã g for 2 minutes.
</LI><LI>Drain the tube on clean absorbent paper and allow the pellet to air-dry for 15 minutes.
</LI><LI>Add 50 Î¼l of DNA Rehydration Solution.
</LI><LI>Add 1.5Î¼l of RNase Solution to the purified DNA sample. Vortex the sample for 1 second and incubate at 37Â°C for 15 minutes.
</LI><LI>Rehydrate the DNA by incubating at 65Â°C for 1 hour. Periodically mix the solution by gently tapping the tube.
</LI><LI>Store the DNA at 2â8Â°C.
</LI></UL><H2><SPAN class="mw-headline" id="Generating_competent_yeast_cells"> Generating competent yeast cells </SPAN></H2><P>Condensed protocol based on <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2504082/" class="external text" rel="nofollow">Lin-Cereghino <I>et al.</I> (2005)</A> and describes an innovative and quick method to generate competent yeast cells.
</P><P>Materials:
</P><UL><LI> 250 mL <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#YPD_media" class="external text" rel="nofollow">YPD medium</A></LI><LI> 50 mL ice-cooled <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#BEDS_buffer" class="external text" rel="nofollow">BEDS</A></LI><LI> 5 mL ice-cooled <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Dithiothreitol-Solution_.28DTT.29" class="external text" rel="nofollow">dithiothreitol (DTT)-Solution</A></LI><LI>150 mL cooled 10Â % glycerine
</LI><LI>10 pre-cooled 50 mL Falcons (-18Â°C)
</LI></UL><P>Protocol:
</P><UL><LI> Cultivate a  overnight culture of the yeast cells in 50-mL <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#YPD_media" class="external text" rel="nofollow">YPD medium</A>  at 30Â°C (120 rpm).
</LI><LI> Dilute the overnight culture to an OD<SUB>600</SUB> of 0.15â0.20 in a volume of 50 mL YPD in a flask large enough to provide good aeration.
</LI></UL><UL><LI> Incubate 250 mL cells to desired OD<SUB>600</SUB> of 0.8-0.9.
</LI><LI> Centrifuge cells for 5 min at room temperature and 500g.
</LI><LI> Resuspend in 9 mL ice-cooled (2-4Â°C))<A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#BEDS_buffer" class="external text" rel="nofollow">BEDS</A> and 1 mL ice-cooled 1.0 M <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Dithiothreitol-Solution_.28DTT.29" class="external text" rel="nofollow">dithiothreitol (DTT)-Solution</A>.
</LI><LI> Incubate for 5 min with gently shaking at 100 rpm at room temperature.
</LI><LI> Centrifuge cells for 5 min at room temperature and 500g.
</LI><LI> Resuspend cells in 1 mL <A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols/Materials#BEDS_buffer" class="external text" rel="nofollow">BEDS</A>.
</LI><LI> Divide the resuspended cells in 150 Î¼L aliquots (now the cells are ready to use).
</LI><LI> Freeze cells slowly  at -80Â°C (Don't freeze in liquid N<SUB>2</SUB>!).
</LI><LI> Place in -80Â°C freezer until needed.
</LI></UL><H2><SPAN class="mw-headline" id="Transformation_of_yeast_cells"> Transformation of yeast cells </SPAN></H2><P>Condensed protocol based on <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2504082/" class="external text" rel="nofollow">Lin-Cereghino <I>et al.</I> (2005)</A> and describes an innovative and quick method to transform DNA into yeast cells.
</P><UL><LI> Thaw <A href="/Team:Bielefeld-Germany/Protocols/molecular_genetics#Generating_competent_yeast_cells" title="Team:Bielefeld-Germany/Protocols/molecular genetics">competent yeast cells</A> on ice.
</LI><LI> Add 4 ÂµL (approximately 500 ng) linearized plasmid DNA to 40 ÂµL of competent cells in an eppi.
</LI><LI> Incubate for 2 min on ice.
</LI><LI> Fill the mixture in cold electroporation cuvette.
</LI><LI> Electroporate the samples using the following parameters: charging voltage = 1500 V, resistance = 200 Î©, capacitance = 25 ÂµF.
</LI><LI> Add rapidly 1 mL cold 1.0 M sorbitol.
</LI><LI> Incubate in a 30 Â°C shaker for 2 hours.
</LI><LI> Plate on selective media (<A href="/Team:Bielefeld-Germany/Protocols/Materials#MD_agar_plates_for_selection_of_recombinant_His.2B_clones" title="Team:Bielefeld-Germany/Protocols/Materials">MD agar plates for selection of recombinant His+ clones</A>) for histidine auxotrophic <A href="http://www.biocompare.com/21435-Pichia-pastoris-Strains/83258-GS115/" class="external text" rel="nofollow"><I>P. pastoris</I> GS115 strain</A>.
</LI></UL><H1><SPAN class="mw-headline" id="General_methods">General methods</SPAN></H1><H2><SPAN class="mw-headline" id="Ethanol_precipitation_to_clean_DNA">Ethanol precipitation to clean DNA</SPAN></H2><P>To get rid of distracting salts the DNA has to be cleaned. For this we used the following protocol:
</P><UL><LI>If the volume of the sample containing the DNA is less than 200 Âµl bring the volume up to 200 Âµl.
</LI><LI>Add 1/10th volume of 3M sodium acetate and mix.
</LI><LI>Now add 2 volumes of -20Â°C cold 100% ethanol and vortex for 10 seconds.
</LI><LI>The sample can now be placed in a -20Â°C freezer overnight or incubated for 30 minutes at -80Â°C.
</LI><LI>Centrifugate for 10 minutes at 4Â°C.
</LI><LI>Discard the supernatant containing the ethanol.
</LI><LI>Wash the pellet with 500 Âµl 4Â°C cold 70% ethanol by rolling the sample gently.
</LI><LI>Discard the supernatant.
</LI><LI>Let the pellet dry at room temperature or speedvac the pellet.
</LI><LI>Resuspend the Pellet in water (amount is depending on the size of the pellet).
</LI></UL><H2><SPAN class="mw-headline" id="Gibson_assembly"> Gibson assembly </SPAN></H2><P>Modified from <A href="http://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html" class="external text" rel="nofollow">Gibson <I>et al.</I> (2009)</A></P><P>This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5â exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.
</P><H3><SPAN class="mw-headline" id="Preparation_of_DNA_molecules_for_in_vitro_recombination">Preparation of DNA molecules for <I>in vitro</I> recombination</SPAN></H3><UL><LI> Generate the complementary sequence overlaps by PCR using the <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> Phusion DNA-polymerase</A>. If necessary add 5 M Betain, 4Â % (v/v) DMSO or use 5x GC Phusion buffer in the reaction mix by reducing the amount of H<SUB>2</SUB>O to decrease the number of false PCR products.
</LI><LI> Identify the PCR products of interest by gel electrophoresis with known DNA standards.
</LI><LI> Extract the PCR products from the gel by cutting out the DNA fragments and clean them up by using a commercial <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" class="external text" rel="nofollow">clean up kit</A>.
</LI></UL><H3><SPAN class="mw-headline" id="In_vitro_recombination"><I>In vitro</I> recombination</SPAN></H3><UL><LI> The assembly mixture contains:
</LI><LI> 320 ÂµL <A href="/Team:Bielefeld-Germany/Protocols/Materials#5x_isothermal_reaction_buffer_for_Gibson_assembly" title="Team:Bielefeld-Germany/Protocols/Materials"> 5x isothermal reaction buffer</A></LI><LI> 0.64 ÂµL of 10 U mL<SUP>-1</SUP><A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T5 exonuclease</A> (for DNA molecules overlapping by greater than 150 bp add 3.2 ÂµL of 10 U ml<SUP>â1</SUP><A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T5 exonuclease</A>)
</LI><LI> 20 ÂµL of 2 U mL<SUP>-1</SUP><A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> Phusion HF DNA polymerase</A></LI><LI> 160 ÂµL of 40 U mL<SUP>-1</SUP><A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>taq</I> DNA ligase</A></LI><LI> add ddH<SUB>2</SUB>O water up to a final volume of 1.2 mL
</LI><LI> aliquote 15 ÂµL of the reagent-enzyme mix and store it at â20 ËC
</LI></UL><LI> Thaw 15 ÂµL assembly mixture aliquot and keep it on ice until use.
</LI><LI> Add 5 ÂµL of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each DNA fragment).
</LI><LI> Incubate the resulting mixture at 50 ËC for 60 min.
</LI><LI> Transformation (heatshock or <A href="/Team:Bielefeld-Germany/Protocols/molecular_genetics#Transformation_via_electroporation" title="Team:Bielefeld-Germany/Protocols/molecular genetics"> electroporation</A>) without cleaning up the assembly product.
</LI><H2><SPAN class="mw-headline" id="KOD_DNA_polymerase_reaction"> KOD DNA polymerase reaction </SPAN></H2><P>The PCR was run with the following pipetting scheme and conditions:
</P><P><B>Pipetting scheme</B></P><P>Mix the following reagents on ice up to a sample volume of 50 ÂµL:
</P><UL><LI> 5 ÂµL of 10x KOD buffer
</LI><LI> 3 ÂµL of MgSO<SUB>4</SUB> (25 mM)
</LI><LI> 1 ÂµL of Primermix (50 mM)
</LI><LI> 2 ÂµL of DMSO (100Â %)
</LI><LI> 1 ÂµL of dNTPs (10 mM)
</LI><LI> 1 ÂµL of template DNA (200 ng/50 ÂµL sample)
</LI><LI> 1 ÂµL KOD DNA polymerase
</LI><LI> 36 ÂµL bidest. water
</LI></UL><P><B> PCR protocol</B></P><TABLE class="wikitable" style="text-align: center; width: 300px: height: 300px;" border="0}"><TBODY><TR><TH> Step
</TH><TH> Condition
</TH></TR><TR><TD>Initial denaturation
</TD><TD>95 Â°C, 2 min
</TD></TR><TR><TD>Denaturation
</TD><TD>95 Â°C, 15 s
</TD></TR><TR><TD>Primer annealing
</TD><TD>65 Â°C, 30 s
</TD></TR><TR><TD>Elongation
</TD><TD>70 Â°C, 15 s
</TD></TR><TR><TD>Final Elongation
</TD><TD>is not necessary
</TD></TR></TBODY></TABLE></DIV></DIV><TABLE cellspacing="10"><TBODY><TR><TD><A href="/File:Promega_logo_180x109.gif" title="File:Promega logo 180x109.gif">55px</A></TD></TR></TBODY></TABLE><DIV id="content-footer-box"><DIV class="content-footer-box-left"><DIV class="about"><P>About our Wiki</P><P>Fifteen students from different fields of study from the Bielefeld University aspired to complete a project called TOXIC COMPOUNDS IN NATURAL WATER - A CASE FOR LACCASE. In his Wiki we document our achievements and our approach. </P></DIV><DIV class="contact"><P>Contact Us</P><P>UniversitÃ¤t Bielefeld
UniversitÃ¤tsstraÃe 25
D-33615 Bielefeld<A href="mailto:info@igem-bielefeld.de">info(at)igem-bielefeld.de</A></P></DIV></DIV><DIV class="content-footer-box-right"><DIV class="navigate"><P>Navigate our Wiki</P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany">Start</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Team</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Project</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Results</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Since Regionals</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Human Practice</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Bielefeld-Germany/Protocols/molecular_genetics">http://2012.igem.org/Team:Bielefeld-Germany/Protocols/molecular_genetics</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Protocols/molecular_genetics" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Protocols/molecular_genetics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/molecular_genetics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/molecular_genetics&amp;oldid=231646" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>