"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-Germany/Protocols/molecular genetics
From 2012.igem.org
Start
Team
Members
Photos
Bielefeld University
Acknowledgements
Contact
Project
Description
Achievements
Approach
Model
Database
Background
Labjournal
Protocols
Outlook
Results
Summary
Datapage
Laccases
Immobilization
Substrate Analysis
Cellulose Binding Domain
Shuttle Vector
Collaboration
Since Regionals
Summary
Labjournal
Human Practice
Overview
Expert
Politics
Industry
Outreach
Education
Environment
Press
Safety
Judging
Partner
Molecular
Contents
1 Bacterial specific methods
1.1 Generating electrocompetent cells
1.2 Transformation via electroporation
1.3 Transformation of Single Step (KRX) Competent Cells by Promega
1.4 Standard BioBrick Assembly
1.4.1 Suffix Insertion
1.4.2 Prefix Insertion
1.4.3 Variations
1.5 Standard Freiburg BioBrick Assembly
1.5.1 Suffix Insertion
1.5.2 Prefix Insertion
1.5.3 Variations
1.5.4 Restriction analysis
1.6 Colony PCR
1.7 Phusion PCR
1.8 Site Directed Mutagenesis
2 Plants specific methods
2.1 Arabidopsis thaliana: Growth Conditions and Plant Material
2.2 Arabidopsis thaliana: Total RNA Isolation
2.3 Arabidopsis thaliana: cDNA Synthesis
2.4 PCR for A.thaliana Laccase Ampflification
3 Yeast Specific Methods
3.1 Pichia pastoris: Growth Conditions
3.2 Genomic DNA Isolation
3.3 Generating competent yeast cells
3.4 Transformation of yeast cells
4 General methods
4.1 Ethanol precipitation to clean DNA
4.2 Gibson assembly
4.2.1 Preparation of DNA molecules for in vitro recombination
4.2.2 In vitro recombination
4.3 KOD DNA polymerase reaction
Bacterial specific methods
Generating electrocompetent cells
(E.coli- KRX , XLI Blue and Rosetta gami )
Materials:
550 mL LB-Medium
1 L cooled bidest. H2O (4 °C)
150 mL cooled 10 % glycerine (4 °C)
10 pre-cooled 50 mL Falcons (-18°C)
Protocol:
Inoculate 2x3 mL LB with bacterial stock; incubate over night at 37 °C and 140 rpm
Inoculate 2x250 mL LB with the over night cultures in 1-litre-flask (with baffles) at 37 °C and 140 rpm
Incubate until OD600 0.4 - 0.6
Cool the culture 15-30 minutes on ice
Important: keep your cells at 2-4 °C during the following steps
Divide the cultures into cooled 50 mL Falcons and centrifugate for 15 minutes (4000 rpm, 4 °C) IMPORTANT: slowly accelerate and deccelerate
Discard  supernatant
Resuspend  cell pellet in 5 mL cooled bidest H2O (and don't get frustrated while doing it, keep shaking gently)
Pool two suspensions each, add bidest H2O up to 50 mL and centrifugate again (see centrifugation above)
Discard supernatant (Keep in mind: keep your cells at 2-4 °C)
Resuspend pellet in 5 mL cooled bidest H2O
Add bidest H2O up to 50 mL and centrifugate again (see centrifugation above)
Discard supernatant (Keep in mind: keep your cells at 2-4 °C)
Resuspend pellet in 5 mL cooled 10 % glycerine
Transfer suspensions in two 50 mL Falcons and centrifugate again (see centrifugation above)
Discard supernatant
Add volume of 10 % cooled glycerine (2-4°C) that is approximately equal to the volume of the pellet and resuspend
Divide cells in 100 μL aliquots and freeze in liquid N2 immediately
Store at -80 °C
Transformation via electroporation
Thaw 50 µL competent E.coli cells on ice, dilute with icecold 50 µL glycerine (10 %) if necessary
Add 0.5-2 µL plasmid to 50 µl electrocompetent cells
Store cells on ice for 1 minute
Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ώ
Transfer transformation reaction to 450 µL SOC-Medium and shake 1 h at 37 °C
Centrifuge 2 min at 800 rpm and plate on selective LB-Medium
Transformation of Single Step (KRX) Competent Cells by Promega
Using protocol E. coli KRX single step competent cells by Promega
Remove Single Step (KRX) Competent Cells from –70 °C, and place on ice for 5 minutes or until just thawed.
Add 1–50 ng of DNA (in a volume not greater than 5 μL) to the Single Step (KRX) Competent Cells. Move the pipette tip through the cells while dispensing. Quickly flick the tube several times. Do not vortex!
Immediately return the tubes to ice for 5–30 minutes
Heat-shock cells for 15–20 seconds in a water bath at exactly 42 °C. Do not shake.
Immediately place the tubes on ice for 2 minutes.
Add 450 μL of room-temperature SOC-Medium to each transformation reaction, and incubate for 60 minutes at 37 °C with shaking (approximately 225 rpm). For best transformation efficiency, lay the tubes on their sides and tape them to the platform.
For each transformation reaction, we recommend plating 100 μL of undiluted cells and 1:10 and 1:100 cell dilutions on antibiotic plates. Incubate the antibiotic plates at 37 °C over night.
Standard BioBrick Assembly
modified from Silver lab :  This assembly method can be used for BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, called vector. So you have to differentiate between a prefix and a suffix insertion.
Suffix Insertion
Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x Tango buffer, 0.5 µL XbaI , 0.5 µL PstI . Digest for 2 h to 4 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.
Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL SpeI , 0.5 µL PstI . Digest for 2 h to 4 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL SAP (shrimp alcaline phosphatase) and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector via gel electrophoresis and PCR clean-up kit .
Ligation: after digestion and clean-up: 50 - 100 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL T4-Ligase . Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
Prefix Insertion
Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x BamHI buffer, 0.5 µL EcoRI , 0.5 µL SpeI . Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.
Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10 x Tango buffer, 0.5 µL EcoRI , 0.5 µL XbaI . Digest for 2h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL SAP (shrimp alkaline phosphatase) and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector via gel electrophoresis and PCR clean-up kit .
Ligation: after digestion and clean-up: 50 - 100 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL T4-Ligase . Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
Variations
A digestion over night is possible. If you digest over night use only 0.1 µL restriction enzyme.
It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with PCR clean-up kit . This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up.
Standard Freiburg BioBrick Assembly
Modified from Silver lab and Assembly standard 25 :
This assembly method can be used for fusion protein assemblies with BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, which is called vector and needs to be available in the BioBrick Assembly standard 25 . You have to differentiate between a prefix and a suffix insertion.
Suffix Insertion
Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x NEB buffer 4 + 0.1 µL 100x BSA, 0.5 µL NgoMIV (NEB) , 1 µL PstI . Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.
Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL AgeI , 0.5 µL PstI . Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL SAP (shrimp alkaline phosphatase) and 1.2 µL 10x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector via gel electrophoresis and PCR clean-up kit .
Ligation: after digestion and clean-up: 50 - 100 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL T4-Ligase . Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
Prefix Insertion
Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL EcoRI , 0.5 µL AgeI . Digest for 2 h to 4 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.
Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10 x NEB buffer 4, 0.5 µL EcoRI , 0.5 µL NgoMIV (NEB) . Digest for 2 h to 4 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL SAP (shrimp alcaline phosphatase) and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a PCR clean-up kit .
Ligation: after digestion and clean-up: 50 - 100 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL T4-Ligase . Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
Variations
A digestion over night is possible. If you digest over night use only 0.1 µL restriction enzyme.
It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with PCR clean-up kit . This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up.
Restriction analysis
Digest BioBrick of interest: about 400 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL NotI or PstI. Digest for 2 h at 37 °C. NotI is used to determine the length of the BioBrick and the plasmid backbone, PstI is used to determine the length of the BioBrick in the plasmid backbone.
Gel electrophoresis: add 2 µL loading buffer to every digestion mix, apply about 100 - 200 ng DNA / pocket in gel. Don't forget to apply the uncut BioBrick as well. A good agarose concentration for BioBricks between 0.2 and 3 kb is 1.5 %. The smaller your BioBrick of interest is the higher the agarose concentration should be and vice versa. The gel electrophoresis is made with TAE-buffer. Be sure that you melt your agarose gel in the same buffer you use for the electrophoresis later.
Colony PCR
Colony PCR was done with GoTaq DNA-polymerase (Promega) and BIOTAQ DNA-polymerase (Bioline) .
Pick one colony with a sterile tip and put it in one PCR tube with mastermix.
The reaction mix and PCR program is described below in the table.
PCR reaction mix
GoTaq DNA-polymerase
BIOTAQ DNA-polymerase
10x NH4 buffer
1.5 µl
0 µl
5x green or colorless GoTaq reaction buffer
0 µl
5 µl
MgCl2 (50 mM)
0.6 µl
0.5 µl
dNTPs (10 mM)
0.3 µl
0.5 µl
Primermix FW and RV (10 mM)
0.75 µl
0.5 µl
DNA-Polymerase
0.4 µl
0.125 µl
bidest ddH2O
11.45 µl
18.375 µl
total volume
15 µl
25 µl
PCR cycle program
GoTaq DNA-polymerase
BIOTAQ DNA-polymerase
Initiale denaturation
3 min, 94 °C
8 min, 95 °C
35 cycles of
denaturation
30 s, 94 °C
30 s, 95 °C
primer annealing
30 s, 55 °C
30 s, 58 °C
elongation
1 min / 1 kb template, 72 °C
30 s / 1 kb template, 72 °C
final elongation
5 min, 72 °C
5 min, 72 °C
Gel electrophoresis: check the fragment size
Plate the correct colonies
Phusion PCR
Phusion PCR was done with Phusion HF DNA-polymerase (Finnzymes) .
One reaction mix contains:
10 µL 5x Phusion buffer
1 µL dNTPs (10 mM)
0.5 µL pro primer (0.5 µM)
250 ng DNA
0.5 µL Phusion HF DNA-polymerase (Finnzymes)
add ddH2O to 50 µL
PCR program:
Initiale denaturation: 3 min, 98 °C
35 cycles of:
10 s, 98 °C
20 s, annealing temperature depends on primer pair
30 s / 1 kb template, 72 °C
Finale elongation: 5 min, 72 °C
Gel electrophoresis: check the fragment size
Site Directed Mutagenesis
Design Primers using the QuickChange Primer Design tool
Run mutagenesis PCR
One reaction mix contains:
5 µL 10x buffer
0.4 µL Pfu polymerase
1 µL dNTPs
39.6 µL H2O
1 µL DMSO
2 µL primer mix
1 µL template
PCR program:
Start: 2 min, 94 °C
16 cycles of:
45 s, 95 °C
30 s, 55 °C
2 min/kb, 73 °C
Finish: 5 min, 72 °C
Control the amplified PCR-fragments by gel electrophoresis
Digest the template plasmid by adding 1 µL of DpnI and incubate for 2-3 h
Purify the PCR product with a PCR clean-up kit
Transform 3 µL of the purified PCR product into electrocompetent E. coli XL1 blue cells
Screen the transformants using restiction digest and sequencing
Plants specific methods
Arabidopsis thaliana: Growth Conditions and Plant Material
Six weeks old A. thaliana plants, ecotype Columbia 0 (wildtype), have been gratefully offered by Patrick Treffon and Thorsten Seidel. They have been cultivated under normal day conditions (14 hours light [100 µmol ⁄ quanta m-2s-1] at 21°C, 10 hours darkness at 18°C). For induction of the formation of siliques the plants were shifted into long day conditions (16 hours light [100 µmol ⁄ quanta m-2s-1] at 21°C, 18 hours darkness at 18°C). After two weeks in long day conditions the plants have developed 2 cm long siliques. The siliques were harvested and frozen in liquid nitrogen for further use.
Arabidopsis thaliana: Total RNA Isolation
The frozen plant material has to be grinded in a precooled mortar in liquid nitrogen. About 120 mg of pulverized plant material are transfered into a precooled 2 ml Eppendorf tube and kept frozen until the following steps:
Add 0.5 ml lysis buffer and immediately homogenize through rough shaking.
Add 0.5 ml of saturated phenol and mix strongly.
Add 0.5 ml of chloroform isoamyl alcohol (24:1) and vortex again at high speed for at least 30 seconds.
Centrifugate for 5 min at 13,000 rpm.
The lower phase contains now lipids and lipophilic compounds. The upper phase contains nucleic acids (~ 550 µl) and has to be carefully transferred into a new 2 ml Eppendorf tube. This tube has to be filled with 0.5 ml saturated phenol and 0.5 ml chloroform isoamyl alcohol (24:1). Mix immediately.
Centrifugate at 13,000 rpm for 3 minutes.
Prepare a new 2 ml Eppendorf tube with 1 ml of chloroform isoamyl alcohol (24:1). Transfer the upper aqueous phase (~ 540 µl) containing the protein purified nucelic acids into the new tube and vortex strongly.
Centrifugate at 13,000 rpm for 3 minutes.
Prepare a new 1.5 ml Eppendorf tube with 0.5 ml of pure isopropanol. For the last time transfer the upper phase (~ 400 µl) into the new tube and mix gently.
Incubate the mixture over night at -20°C. The nucleic acids will precipitate.
Centrifugate the samples at 13,000 rpm for 15 minutes at 4°C.
Discard the supernatant and resuspend the pellet in 375 µl sterile H2O.
Add 125 µl 8 M lithium chloride and incubate for 2 hours on ice at 4°C. At this point most of the RNA is going to be precipitated.
Centrifugate at 13,000 rpm at 4°C and discard the supernatant.
Wash the pellet with 100 µl 70% (v/v) ethanol and discard it after centrifugation.
Dry the pellet at room temperature.
Dissolve the pellet in sterile H2O (~ 25 µl, depending on the size of the pellet).
Check the quantity and quality of the RNA with a Nanodrop spectrophotometer before starting with a cDNA synthesis.
Arabidopsis thaliana: cDNA Synthesis
After a successful total RNA isolation the RNA has to be translated in cDNA through RT-PCR:
Take 3 µg/µl of total RNA and add sterile H2 to 8 µl.
Additionally add
1.1 mM
Oligo-d(T)-Primer
0.83 mM
dNTPs
3.5 µl
H2O
Vortex and centrifugate shortly.
Incubate the samples for 10 minutes at 70°C.
Immediately transfer the samples into ice water for 5 minutes.
After cooling the samples centrifugate shortly.
To start the synthesis add
6 µl
5xMMLV-Puffer
4.5 µl
H2O
1 µl
MMLV-reverse Transkriptase [200 U/µl]
0.5 µl
RNasin RNase-Inhibitor [40 U/µl]
Mix the samples and centrifugate shortly.
Incubate for 1 hour at 42 °C to translate the RNA into cDNA.
Transfer the samples to 70 °C for 15 minutes to stop the reaction.
The new synthesized cDNA can be used for PCR after diluting 1:10 with water. Store the cDNA at -20 °C.
PCR for A.thaliana Laccase Ampflification
The PCR setup was determined thorugh primer length and GC content. The samples contained the following reagents:
component
concentration
buffer
10 µL
Primer forward
1 µL
Primer reverse
1 µL
dNTP´s
1 µL
Template
3 µL
Phusion polymerase
1 µL
H2O
ad 50 µL
Due to the length of the primers cycles with two different temperatures have been used:
temperature
duration
98°C
30 sec
46°C /74°C
10 sec
47°C/75°C
20 sec
72°C
1 min
Yeast Specific Methods
Pichia pastoris: Growth Conditions
The cultivation conditions of the yeast Pichia pastoris are: 30 °C at 130 rpm.
Genomic DNA Isolation
The complete genome isolation was done with the Promega Wizard genomic DNA purification system kit .
Pellet 10 mL of over-night liquid culture grown in YPD broth in a 1.5 mL tube by centrifugation at 14,000 x g for 2 minutes.
Remove the supernatant.
Resuspend the cells in 90 μL of 50 mM EDTA.
Add 10 μL of 1000u lyticase and pipet 4 times to mix.
Incubate the sample at 37°C for 60 minutes to digest the cell wall.
Centrifuge the sample at 14,000 × g for 2 minutes and then remove the supernatant.
Add 300 μl of Nuclei Lysis Solution to the cell pellet and pipet to mix.
Add 100 μl of Protein Precipitation Solution and vortex at high speed for 20 seconds.
Let the sample sit on ice for 5 minutes.
Centrifuge at 14,000 × g for 3 minutes.
Transfer the supernatant containing the DNA to a clean 1.5 ml tube containing 300 μl of room temperature isopropanol.
Gently mix by inversion until the DNA is visible.
Centrifuge at 14,000 × g for 2 minutes.
Carefully decant the supernatant and drain the tube on clean absorbent paper.
Add 300 μl of room temperature 70% ethanol and invert the tube several times to wash the DNA pellet.
Centrifuge at 14,000 × g for 2 minutes.
Drain the tube on clean absorbent paper and allow the pellet to air-dry for 15 minutes.
Add 50 μl of DNA Rehydration Solution.
Add 1.5μl of RNase Solution to the purified DNA sample. Vortex the sample for 1 second and incubate at 37°C for 15 minutes.
Rehydrate the DNA by incubating at 65°C for 1 hour. Periodically mix the solution by gently tapping the tube.
Store the DNA at 2–8°C.
Generating competent yeast cells
Condensed protocol based on Lin-Cereghino et al. (2005) and describes an innovative and quick method to generate competent yeast cells.
Materials:
250 mL YPD medium
50 mL ice-cooled BEDS
5 mL ice-cooled dithiothreitol (DTT)-Solution
150 mL cooled 10 % glycerine
10 pre-cooled 50 mL Falcons (-18°C)
Protocol:
Cultivate a  overnight culture of the yeast cells in 50-mL YPD medium at 30°C (120 rpm).
Dilute the overnight culture to an OD600 of 0.15–0.20 in a volume of 50 mL YPD in a flask large enough to provide good aeration.
Incubate 250 mL cells to desired OD600 of 0.8-0.9.
Centrifuge cells for 5 min at room temperature and 500g.
Resuspend in 9 mL ice-cooled (2-4°C)) BEDS and 1 mL ice-cooled 1.0 M dithiothreitol (DTT)-Solution .
Incubate for 5 min with gently shaking at 100 rpm at room temperature.
Centrifuge cells for 5 min at room temperature and 500g.
Resuspend cells in 1 mL BEDS .
Divide the resuspended cells in 150 μL aliquots (now the cells are ready to use).
Freeze cells slowly  at -80°C (Don't freeze in liquid N2!).
Place in -80°C freezer until needed.
Transformation of yeast cells
Condensed protocol based on Lin-Cereghino et al. (2005) and describes an innovative and quick method to transform DNA into yeast cells.
Thaw competent yeast cells on ice.
Add 4 µL (approximately 500 ng) linearized plasmid DNA to 40 µL of competent cells in an eppi.
Incubate for 2 min on ice.
Fill the mixture in cold electroporation cuvette.
Electroporate the samples using the following parameters: charging voltage = 1500 V, resistance = 200 Ω, capacitance = 25 µF.
Add rapidly 1 mL cold 1.0 M sorbitol.
Incubate in a 30 °C shaker for 2 hours.
Plate on selective media ( MD agar plates for selection of recombinant His+ clones ) for histidine auxotrophic P. pastoris GS115 strain .
General methods
Ethanol precipitation to clean DNA
To get rid of distracting salts the DNA has to be cleaned. For this we used the following protocol:
If the volume of the sample containing the DNA is less than 200 µl bring the volume up to 200 µl.
Add 1/10th volume of 3M sodium acetate and mix.
Now add 2 volumes of -20°C cold 100% ethanol and vortex for 10 seconds.
The sample can now be placed in a -20°C freezer overnight or incubated for 30 minutes at -80°C.
Centrifugate for 10 minutes at 4°C.
Discard the supernatant containing the ethanol.
Wash the pellet with 500 µl 4°C cold 70% ethanol by rolling the sample gently.
Discard the supernatant.
Let the pellet dry at room temperature or speedvac the pellet.
Resuspend the Pellet in water (amount is depending on the size of the pellet).
Gibson assembly
Modified from Gibson et al. (2009)
This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5‘ exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.
Preparation of DNA molecules for in vitro recombination
Generate the complementary sequence overlaps by PCR using the Phusion DNA-polymerase . If necessary add 5 M Betain, 4 % (v/v) DMSO or use 5x GC Phusion buffer in the reaction mix by reducing the amount of H2O to decrease the number of false PCR products.
Identify the PCR products of interest by gel electrophoresis with known DNA standards.
Extract the PCR products from the gel by cutting out the DNA fragments and clean them up by using a commercial clean up kit .
In vitro recombination
The assembly mixture contains:
320 µL 5x isothermal reaction buffer
0.64 µL of 10 U mL-1 T5 exonuclease (for DNA molecules overlapping by greater than 150 bp add 3.2 µL of 10 U ml–1 T5 exonuclease )
20 µL of 2 U mL-1 Phusion HF DNA polymerase
160 µL of 40 U mL-1 taq DNA ligase
add ddH2O water up to a final volume of 1.2 mL
aliquote 15 µL of the reagent-enzyme mix and store it at –20 ˚C
Thaw 15 µL assembly mixture aliquot and keep it on ice until use.
Add 5 µL of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each DNA fragment).
Incubate the resulting mixture at 50 ˚C for 60 min.
Transformation (heatshock or electroporation ) without cleaning up the assembly product.
KOD DNA polymerase reaction
The PCR was run with the following pipetting scheme and conditions:
Pipetting scheme
Mix the following reagents on ice up to a sample volume of 50 µL:
5 µL of 10x KOD buffer
3 µL of MgSO4 (25 mM)
1 µL of Primermix (50 mM)
2 µL of DMSO (100 %)
1 µL of dNTPs (10 mM)
1 µL of template DNA (200 ng/50 µL sample)
1 µL KOD DNA polymerase
36 µL bidest. water
PCR protocol
Step
Condition
Initial denaturation
95 °C, 2 min
Denaturation
95 °C, 15 s
Primer annealing
65 °C, 30 s
Elongation
70 °C, 15 s
Final Elongation
is not necessary
55px
About our Wiki
Fifteen students from different fields of study from the Bielefeld University aspired to complete a project called TOXIC COMPOUNDS IN NATURAL WATER - A CASE FOR LACCASE. In his Wiki we document our achievements and our approach.
Contact Us
Universität Bielefeld
Universitätsstraße 25
D-33615 Bielefeld
info(at)igem-bielefeld.de
Navigate our Wiki
Start
Team
Project
Results
Since Regionals
Human Practice
Safety
Judging
Partner
Retrieved from " http://2012.igem.org/Team:Bielefeld-Germany/Protocols/molecular_genetics "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
