<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Results_vector"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Results/vector">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Results/vector&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/vector&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/vector&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Results/vector" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Results/vector</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="navigation"><DIV style=" z-index:10000;"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany">Start</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Team</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Bielefeld-University">Bielefeld University</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Project</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Achievments">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Database">Database</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Outlook">Outlook</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Results</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Datapage">Datapage</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Laccases">Laccases</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/immo">Immobilization</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/substrate">Substrate Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/cbc">Cellulose Binding Domain</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/vector">Shuttle Vector</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/london">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Since Regionals</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Summary</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Expert">Expert</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Politics">Politics</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Sewage_treatment_plant">Industry</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human Practices/Outreach">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/StudentAcademy">Education</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Environment">Environment</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></LI><LI><A href="https://2012.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV></DIV><DIV id="page-title"><SPAN id="page-title-text">
                 Shuttle vector
            </SPAN></DIV><DIV id="grey_bg"><H1>Summary</H1><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Description_of_the_shuttle_vector_system"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Description of the shuttle vector system</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Site_directed_mutagenesis_of_the_shuttle_vector"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Site directed mutagenesis of the shuttle vector</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Shuttle_vector_in_E._coli"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Shuttle vector in E. coli</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Shuttle_vector_in_P._pastoris"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Shuttle vector in P. pastoris</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#GFP_integrated_in_shuttle_vector"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">GFP integrated in shuttle vector</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Since_Regionals:_TVEL5_integrated_in_shuttle_vector"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Since Regionals: TVEL5 integrated in shuttle vector</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Description_of_the_shuttle_vector_system"> Description of the shuttle vector system </SPAN></H2><P>The design of a minimal shuttle vector system with defined regions (or better DNA fragments) for expression and secretion of proteins of interest (POI), like laccase, which needed glycolisation is the basic concept. The shuttle vector needs a bacterial part for cloning in bacteria (like <I>E. coli</I>) and an eucaryotic part for genomic integration and selection in yeast (like <I>P. pastoris</I>). An other team will be able to clime in frame their gene of interest via the <I>Aar</I>I restriction site. With only one restriction ligation cloning step the shuttle vector will be ready to use and integrate in the eucaryote <I>P. pastoris</I>.
</P><P>The shuttle vector consists of the plasmid pSB1C3, 5' UTR of alcohol oxidase 1 gene (<I>aox1</I>) containing the <I>aox1</I> promoter region, Kozak sequence, mating factor alpha 1 (MFalpha1), <I>Aar</I>I restriction site, <I>aox1</I> terminator, <I>his4</I> gene and 3' UTR of <I>aox1</I> gene. Cloning and plasmid replication in <I>E. coli</I> are able via the pSB1C3 part.  The gene of interest (like laccase) can be included in frame with MFalpha1 via AarI restriction site. With the N-terminal MFalpha1 the POI could be secreted in the media. Genomic integation of MFalpha1-taged POI is able via the 5' UTR and 3' UTR of the <I>aox1</I> gene. This allows a double cross over and the genomic integration without any bacterial proportion of DNA which could be a decisive point for industrial application. The complementation of histidine auxotrophie via <I>his4</I> gene was chosen instead of a zeocine resistance. This selection strategy is chosen because we want to avoid the application of antibiotics.
</P><P>Gibson assembly was used for building the shuttle vector (see the figure below) and the fragments with 5' overlap were amplified via PCR. In addition, the fragments were designed as basic BioBrick parts for different applications by the community. The origin of the DNA sequence for design of the shuttle vector and the source of DNA for PCR is listed in the table below.
</P><P><B>Elements of the shuttle vector and their origin</B></P><TABLE class="wikitable" style="text-align: center; width: 300px: height: 300px;" border="0}"><TBODY><TR><TH> Element
</TH><TH> BioBrick
</TH><TH> Origin of DNA sequence of design
</TH><TH> Origin of DNA sequence of PCR
</TH></TR><TR><TD><A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A></TD><TD><A href="http://partsregistry.org/wiki/index.php/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A></TD><TD><A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A></TD><TD><A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K863200" class="external text" rel="nofollow">5'UTR of <I>aox1</I></A></TD><TD><A href="http://partsregistry.org/Part:BBa_K863200" class="external text" rel="nofollow">K863200</A></TD><TD><A href="http://products.invitrogen.com/ivgn/product/V19520" class="external text" rel="nofollow">plasmid pPICZalphaA (Invitrogen)</A></TD><TD><I>P. pastoris</I> wild type X-33
</TD></TR><TR><TD>Kozak sequence
</TD><TD><A href="http://partsregistry.org/Part:BBa_J63003" class="external text" rel="nofollow">J63003</A></TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J63003" class="external text" rel="nofollow">BBa_J63003</A></TD><TD>integrated in primer sequence
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K863203" class="external text" rel="nofollow">MFalpha1</A></TD><TD><A href="http://partsregistry.org/Part:BBa_K863206" class="external text" rel="nofollow">K863206</A></TD><TD><A href="http://products.invitrogen.com/ivgn/product/V19520" class="external text" rel="nofollow">plasmid pPICZalphaA (Invitrogen)</A></TD><TD><A href="http://products.invitrogen.com/ivgn/product/V19520" class="external text" rel="nofollow">plasmid pPICZalphaA (Invitrogen)</A></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K863203" class="external text" rel="nofollow"><I>aox1</I> terminater</A></TD><TD><A href="http://partsregistry.org/Part:BBa_K863203" class="external text" rel="nofollow">K863203</A></TD><TD><A href="https://products.invitrogen.com/ivgn/product/V17520" class="external text" rel="nofollow">plasmid pPIC9K</A></TD><TD><I>P. pastoris</I> wild type X-33
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K863202" class="external text" rel="nofollow"><I>his4</I></A></TD><TD><A href="http://partsregistry.org/Part:BBa_K863202" class="external text" rel="nofollow">K863202</A></TD><TD><A href="https://products.invitrogen.com/ivgn/product/V17520" class="external text" rel="nofollow">plasmid pPIC9K</A></TD><TD><I>P. pastoris</I> wild type X-33
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K863201" class="external text" rel="nofollow">3'UTR of <I>aox1</I></A></TD><TD><A href="http://partsregistry.org/Part:BBa_K863201" class="external text" rel="nofollow">K863201</A></TD><TD><A href="https://products.invitrogen.com/ivgn/product/V17520" class="external text" rel="nofollow">plasmid pPIC9K</A></TD><TD><I>P. pastoris</I> wild type X-33
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 1:</B> Plasmid map for shuttle vector pSB1C3::BBa_K863204.</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Site_directed_mutagenesis_of_the_shuttle_vector"> Site directed mutagenesis of the shuttle vector </SPAN></H2><P>To remove the illegal <I>Xba</I>I restriction site in the shuttle vector, site directed mutagenesis was done. On the selection medium plattes some clones grow. Colony PCR was done for some of them, but in all tried cases the results are negative.
</P><H2><SPAN class="mw-headline" id="Shuttle_vector_in_E._coli"> Shuttle vector in <I>E. coli</I></SPAN></H2><P>The developed shuttle vector could be transformed in <I>E. coli</I> and analyzed by restriction with the enzymes <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Pvu</I>II</A> and <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Hind</I>III</A>. The restriction pattern is positive. A <A href="http://partsregistry.org/cgi/partsdb/dna.cgi?part_name=BBa%20K863204" class="external text" rel="nofollow">sequencing</A> was also done.
</P><H2><SPAN class="mw-headline" id="Shuttle_vector_in_P._pastoris"> Shuttle vector in <I>P. pastoris</I></SPAN></H2><P>After linearization by restriction with <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>EcoR</I>I</A> and <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"><I>Spe</I>I</A> the shuttle vector could be transformed in competent <I>P. pastoris</I> cells. Transformants grow on selective media plates because of complementation of histidine auxotrophy. For differentiation between the M<SUP>+</SUP> and M<SUP>s</SUP> genotype a PCR with the primer <A href="/Team:Bielefeld-Germany/Protocols/Materials#Primers" title="Team:Bielefeld-Germany/Protocols/Materials"> 5AOX-Phenotype-FW</A> and <A href="/Team:Bielefeld-Germany/Protocols/Materials#Primers" title="Team:Bielefeld-Germany/Protocols/Materials"> TT-Phenotype-RV</A> have to be done. The M<SUP>+</SUP> phenotype means that a single cross over recombination took place and the cultivation with methanol have no effect on the growth of the cells. The M<SUP>s</SUP> phenotype means that a double cross over recombination took place and the gene <I>aox1</I> was removed from the genome. During cultivation and induction with methanol the cells grow slow, because now only the gene <I>aox2</I> exist in the genome.
</P><H2><SPAN class="mw-headline" id="GFP_integrated_in_shuttle_vector"> GFP integrated in shuttle vector </SPAN></H2><P>Cloning of green fluorescent protein (GFP) via the <I>Aar</I>I restriction site into the shuttle vector was done to test the system. A restriction analysis show that the construction were successful. The site directed recombination failed and no clones could be found.
</P><H2><SPAN class="mw-headline" id="Since_Regionals:_TVEL5_integrated_in_shuttle_vector"> Since Regionals: TVEL5 integrated in shuttle vector </SPAN></H2><P>The laccase TVEL5 was cloned via the <I>Aar</I>I restriction site into the shuttle vector (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863207" class="external text" rel="nofollow">BBa_K863207</A>). Restriction analysis and sequencing confirmed that success. Furthermore the linearized construct could be transformed into the yeast <I>P. pastoris</I>. Positive clones with single cross overs were identified by PCR on genomic DNA with the primer 5AOX-Phenotype-FW and TT-Phenotype-RV as shown in Figure 1 and 2. Positive clones with single cross over resulted in a 2.2 kb DNA band, the <I>aox1</I> gene, and a 1.9 kb DNA band, the TVEL5 laccase DNA band. As control the shuttle vector (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863204" class="external text" rel="nofollow">BBa_K863204</A>) itself was used. The expected DNA size is 437 bp. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:222px;"><DIV class="thumbcaption"><B>Figure 1:</B><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K863204" class="external text" rel="nofollow">BBa_K863204</A> was used as a control for PCR with the primer 5AOX-Phenotype-FW and TT-Phenotype-RV. Expected DNA band: 437 bp.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 2:</B> PCR product of the gDNA from the culture 1, 2 and 3 (C1, C2, C3) with the primer 5AOX-Phenotype-FW and TT-Phenotype-RV. Expected DNA bands: 1.9 kb (<I>tvel5</I>) and 2.2 kb (<I>aox1</I>).</DIV></DIV></DIV><P>The construct was integrated into the genome with a single cross over. This results in a M<SUP>+</SUP> phenotype. Furthermore we could show that the production and secretion of the active laccase TVEL5 after induction with methanol worked.
The clones were cultivated in minimal medium and induced with 1% (v/v) methanol twice a day for 2 days. The supernatant of the culture was examined to active laccases by testing the enzyme activity. For this purpose the supernatant was re-buffered into H<SUB>2</SUB>O with HiTrap Desalting Columns and incubated with 0.4 mM CuCl<SUB>2</SUB>. After 2 hours of incubation time 140 µL of the re-buffered supernatant were applied for the activity assay with 40 µL Britton Robinson buffer (pH 5) and measured at 25 °C. These setting were chosen because of the experiences with the TVEL0 laccase. To detect activity of the TVEL5 laccase 9 mM ABTS were used. Figure 3 shows a distinct activity of each cultivation (C1, C2, C3) in comparison to the control, which was treated like the other samples. The control, which was also treated with CuCl<SUB>2</SUB> also represented the negative control to determine the effect of CuCl<SUB>2</SUB> oxidizing ABTS. In Figure 4 the same experimental setup was used in a total volume of 1 mL. The oxidation of ABTS by TVEL5 laccase is visible to the naked eye in C1, C2 and C3 in comparison to the control. C2 shows the highest potential in oxidizing ABTS in the activity measurements and also in the experiments with a higher volume. This may be due to a higher TVEL5 concentration in the supernatant of this cultivation.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption"><B>Figure 3:</B>Activity assay of TVEL5 using the re-buffered, CuCl<SUB>2</SUB> incubated supernatant of C1, C2 and C3 with 140 µL each. 40 µL Britton Robinson buffer (pH 5) and 9 mM ABTS were added. Oxidized ABTS was detected by measuring the change in OD at 420 nm at 25 °C for 8 hours.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 4:</B>Activity test of cultivation (culture 1, 2, 3) supernatant of <A href="http://partsregistry.org/Part:BBa_K863207" class="external text" rel="nofollow">BBa_K863207</A>. The control is the supernatant of <I>P. pastoris</I> GS115. The supernatants were re-buffered and CuCl<SUB>2</SUB> incubated. Britton Robinson buffer and 9 mM ABTS were applied. Picture was taken after 8 hours incubation at 25 °C.</DIV></DIV></DIV></DIV><TABLE cellspacing="10"><TBODY><TR><TD><A href="/File:Promega_logo_180x109.gif" title="File:Promega logo 180x109.gif">55px</A></TD></TR></TBODY></TABLE><DIV id="content-footer-box"><DIV class="content-footer-box-left"><DIV class="about"><P>About our Wiki</P><P>Fifteen students from different fields of study from the Bielefeld University aspired to complete a project called TOXIC COMPOUNDS IN NATURAL WATER - A CASE FOR LACCASE. In his Wiki we document our achievements and our approach. </P></DIV><DIV class="contact"><P>Contact Us</P><P>Universität Bielefeld
Universitätsstraße 25
D-33615 Bielefeld<A href="mailto:info@igem-bielefeld.de">info(at)igem-bielefeld.de</A></P></DIV></DIV><DIV class="content-footer-box-right"><DIV class="navigate"><P>Navigate our Wiki</P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany">Start</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Members">Team</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Project/Description">Project</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/Summary">Results</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Amsterdam/Results">Since Regionals</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Human Practice</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></P><P><A href="https://2012.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Bielefeld-Germany/Results/vector">http://2012.igem.org/Team:Bielefeld-Germany/Results/vector</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Results/vector" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Results/vector" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/vector&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/vector&amp;oldid=296608" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>