"
Page
Discussion
View source
History
teams
Log in
 
Team:XMU-China/Project
From 2014.igem.org
Project
Project
Background
Design
Modelling
Results
Application
Interlab
Future
P&P
HP
Newsletter
Communication
Acknowledge
Judging
Notebook
Lab notes
Calendar
Team
Members
Gallery
Safety
Project
Background
Design
Modelling
Results
Application
Future
P&P
Human Practice
Newsletter
Communication
Acknowledge
Judging
Notebook
Lab notes
Calendar
Team
Members
Gallery
Safety
 
Bacterial chemotaxis, which is universal in E.coli, is defined as bacteria cells migration in response to a chemical stimulus. The natural E.coli chemotaxis has limited receptor proteins which can bind to only six kinds of amino acid. Nevertheless, the reprogrammed chemotaxis named pseudotaxis makes the bacteria able to respond to molecules, whose receptor proteins do not exist in classical E.coli, such as IPTG and L-arabinose, etc.
 
                                                                                 
Figure 1 Chemotaxis   mechanism of E.coli. The direction   of rotation of the flagellar motor is controlled by the protein CheY. If the   CheY is phosphorylated (CheY-P), it can bind to the flagellar motor protein   FliM, causing the cell to tumble. While CheY is not phosphorylated, the   flagellar motor rotates counterclockwise (CCW). [1]
 
E.coli have several flagella per cell (4–10 typically), which can rotate in two ways: counterclockwise (CCW) and clockwise (CW). [2] The former aligns the flagella into a single rotating bundle, causing the bacterium to swim in line, while the later breaks the flagella bundle apart such that each flagellum points in a different direction, causing the bacterium to tumble. The motility is determined by the phosphorylation state of CheY protein governed by CheZ protein. In the presence of CheZ protein, CheY-P is dephosphorylated and produce CheY, and the flagellar motor rotates CCW resulting in swimming. In the absence of CheZ, CheY is phosphorylated to CheY-P which binds to the flagellar switch protein FliM resulting in tumbling (Figure 1). [1] Therefore, we are able to control the bacterial motility by knocking out the CheZ gene of the wild-type then transfecting circuit containing CheZ gene into a CheZ knockout (ΔCheZ) strain. Besides, we introduce aptamers responding to a mass of specific molecules which can be applied to regulate gene expression, in our project, CheZ (Figure 2).
 
Figure 2  Mechanism of how aptamers controls the   translation of CheZ protein. In the absence of target molecules (theophylline   as an example). The mRNA’s ribosome binding site is paired, which inhibits   the translation of CheZ protein. In the absence of CheZ, CheY-P will remain   phosphorylated and the cells tumble in place. While in the presence of   theophylline, the mRNA’s ribosome binding site will expose and the CheZ can   be expressed, allowing the cells to run and tumble. [1]
 
Characterizing the circuit we constructed, we combine mathematical modeling with experiments, using modeling to guide experiments and to explain experimental phenomena. .
 
And we can characterize the efficiency of RBS and promoter via migration distance positively associated with the expression strength of CheZ.
 
, Mathematic is the simplest and clearest language, whose value to the development of human civilization is now widely recognized because its extensive application of science, society and daily life. However, the mathematical laws in life sciences is still unclear and even in chaos. Luckily, synthetic biology can overcome these shortcomings on some level. Based on this, we design a gene circuit and expect mathematical regularities to realize the regulation and control of life activities. We hope our work can inspire people's interests to combine mathematic with synthetic biology.
 
References
[1] Topp, Shana, and Justin P. Gallivan. "Guiding bacteria with small molecules and RNA." Journal of the American Chemical Society 129.21 (2007): 6807-6811.
[2] http://en.wikipedia.org/wiki/Chemotaxis
 
 
Overview
What means would you like to use to get a mathematical pattern? Draw one with compass and ruler, or type a function in a drawing software? Well, E.coli can help us to achieve our goals. We make the first attempt at introducing pseudotaxis of bacteria to form patterns in shape of conic section (such as ellipse and hyperbola). Firstly, let’s make precise mathematical definition on ellipse, hyperbola and parabola.
 
What are ellipse, hyperbola and parabola?
 
In mathematics, ellipse is a curve on a plane surrounding two focal points such that a straight line drawn from one of the focal points to any point on the curve and then back to the other focal point has the same length for every point on the curve (Figure 1).
                                                                                 
Figure 1 Schematic of   ellipse. Point F1, F2 are the two focal points and Point A is on the ellipse   curve. The sum of the distance AF1 and AF2 is equal to the constant k:   AF1+AF2=k.
And Hyperbola is a conic consisting of those points whose distances to some point, called a focus, and some line, called a directrix, are in a fixed ratio (>1), called the eccentricity. (Figure 2B)
Parabola is a conic whose eccentricity is equal to 1 (Figure 2C).
   
A
B
Figure 2A, Schematic of   hyperbola. Point F1, F2 are the two focal points and Point A is on one of the   hyperbola branches. The absolute value of the difference of the distance AF1,   AF2 is equal to the constant k: |AF1-AF2|=k. 2B, one branch of hyperbola can be defined by directrix and   eccentricity. Point A is a spot on the curve, point F is the focus. AB is the   distance between A and directrix. The eccentricity e equal to TF/AB, and   e>1.
 
Figure 2C Schematic of   Parabola. Point A is a spot on the curve, point F is the focus. AB is the   distance between A and directrix. The eccentricity e equals to AF/AB, and   e=1.
 
 
Hypothesis
 
We find that the focal points, the constant and the ratio are the key of conic section. Therefore, we can easily get any eclipse by presetting reasonable focal points with the acceptable constant, as well as parabola or hyperbola by a fixed ratio. Based on these, we put forward the following hypothesis:
If we spot stimulus on semi-solid plate, it will spread from the spotting center out to the periphery. In the spreading process, the concentration is negative correlation to the distance from center. And the concentration gradient of the stimulus will be formed and maintained for a long period.
There is a threshold ratio of the concentrations of inducer and repressor. This means that more repressor will cause more repression, hence more inducer is needed to relieve the repression, and vice versa.
 
Design of circuit
 
Our circuit consists of two parts, of which one is named C (constraint), the other is named M (motile). (Figure 4):
 
We build our circuit in E.coli CL-1 which lacks gene lacI and CheZ (ΔlacI, ΔCheZ). At the absence of CheZ, CL-1 adopts non-motile phenotype.
Without any exogenous stimulus, E.coli will produce background amount of AraC to repress pBAD in limit degree. Even when L-arabinose isn’t involved in, promoter pBAD has expression leakage, so that part C will produces protein LacI which can bind to the operon of promoter pLac and thus repress its transcription. Because L-arabinose could induce pBAD, within certain concentration range, more L-arabinose involved in means that part C could produce more LacI to repress the expression of pLac. Because of its ability to constrain chemotaxis, this part is named C (abbreviate from constraint).
When IPTG is involved in, it can relieve the repression from LacI, therefore protein CheZ is produced to make our engineering bacteria (CL-1) regain motile ability. Within certain L-Arabinose concentration range which means certain constraint condition, more cheZ is produced with more IPTG involved in leading to stronger motile ability. Because of its ability to make CL-1 motile, this part is named M (abbreviate from motile).
Figure 4 Part C produces LacI to repress the   expression of part M. Part M could produce CheZ to make CL-1 regain motile ability.
 
Characterization of circuit
We sequenced the circuit above and characterized it in E.coli CL-1. As CL-1 lacks LacI gene, promoter pLac won’t be repressed by background amount of LacI protein.
We apply gradient test to find out which influence would be made on reprogrammed chemotaxis under the following parameters: the concentration of chloramphenicol, IPTG and L-arabinose.
At first, we need to find out at which chloramphenicol concentration we could get the best chemotaxis performance. We try gradient concentration of chloramphenicol at semi-solid medium culture as Table 1 show. We find that the chemotaxis performance doesn’t have overt linear relationship to chloramphenicol. Interestingly, 50μg/ml of chloramphenicol gives CL-1 the best chemotaxis. So we applied that to our following characterization.
 
Characterization of circuit
We sequence the circuit above and characterize it in E.coli CL-1. As CL-1 lacks LacI gene, promoter pLac won’t be repressed by background amount of LacI protein.
 
We design gradient tests of the concentration of chloramphenicol, IPTG and L-arabinose to find out their influence on reprogrammed chemotaxis. To begin with, we test for the best chloramphenicol concentration. We test gradient concentration of chloramphenicol at semi-solid medium culture as Table 1 show. We find that the activity of chemotaxis doesn’t have overt linear relationship to chloramphenicol. Interestingly, 50μg/ml of chloramphenicol gives CL-1 the best chemotaxis. So we apply that to our following characterization.
 
 
Table 1 Curve of chemotaxis diameter under gradient concentration of   Cm (chloramphenicol).
 
As promoter pBAD leads to a certain level of expression leakage of LacI, CL-1 has the worst chemotaxis. We added IPTG at gradient concentration and got the results (Table 2). We find that the chemotaxis performance keeps increasing when the concentration of IPTG increases from 0 and 0.02μM and gets the best performance with the IPTG range from 0.02 to 0.025μM. We apply 0.025μM IPTG for our following characterization.
 
Table 2 Curve of chemotaxis diameter over time under gradient   concentration of IPTG.
 
As more L-arabinose added in, the expression from promoter pBAD will be stronger which leads to more LacI produced resulting in the inhibition to chemotaxis. As our expectation, chemotaxis performance keeps going down as the concentration of L-arabinose increases (Table 3). We find that 0.2% of L-arabinose has the best inhibitory effect on chemotaxis with 0.025μM of IPTG added in.
 
Table 3 Curve   of chemotaxis diameter under gradient concentration of L-arabinose.
 
 
Eclipse
 
If we spot IPTG on the center of the semi-solid culture medium, concentration gradient will be formed as Figure 3A. Larger circle represents lower concentration with a lower number labeled.
A
B
Figure 3 Schematic of concentration   gradient. A, single point   spreading model. Larger number represents higher concentration while smaller   number represents lower concentration. B,   double points spreading model. Letter A~J represent equal concentration   points on the ellipse with two labeled number added up to 6. Red curve   represents the ellipse with two focal points on the IPTG spots.
If we spot two IPTG simultaneously, two spots’ concentration gradient will interact with each other to form the ellipse boundary as image in Figure 3B.
 
If we spot E.coli CL-1 with circuit above transformed in at the center between two IPTG spots on semi-solid culture plate as the image in Figure 5. The concentration of L-Arabinose added in the culture medium determines the degree of constraint from part C. Each red ellipse curve represents equal IPTG concentration, and there are a series of such ellipses from inside to outside represent IPTG concentrations from highest to lowest. One of the ellipses is the critical line indicating that the constraint from part C can just be relieved by the certain concentration of IPTG. Initially, as the concentration of IPTG is enough to relieve the constraint, bacteria can swim from the center out to periphery. When the bacteria swim out of the critical line, the concentration of IPTG can’t relieve the constraint, so the bacteria adopt non-motile phenotype. On the contrary, when the bacteria are inside of the critical line, they adopt motile phenotype. When the bacteria swim from inside to outside, motile bacteria become non-motile, so the bacteria will aggregate outside the critical line while the bacteria density inside the critical line will decline. Thus, an ellipse boundary is formed.
Figure 5 Schematic of   Critical Line model for ellipse. Critical line represents the IPTG   concentration which can just relieve the repression from part C. So that CL-1 is motile inside the ellipse while non-motile outside the   ellipse.
 
 
Parabola and hyperbola
 
We got the optimum concentrations of IPTG and L-arabinose as inducer and repressor through preliminary experiments. Because the concentration of stimulus will decrease during spreading, so does their effect, we use IPTG and L-arabinose of which concentrations are a little bit higher than the optimum values for our experiments.
 
We draw a straight line with L-arabinose on the semi-solid culture medium, and a spot with the mixture of IPTG and CL-1 on one side of the line. In the area around the spot, the induction of IPTG is stronger than the repression of L-arabinose, CheZ is expressed and the bacteria adopt motile phenotype. However, when they approach the line where the repressor have a greater effect on the motility, they will lost their motile phenotype and stop.
 
According to the hypothesis 2, on the threshold ratio of the concentration of inducer and repressor, their effects are offset and a critical line is formed. Distances of the points on the critical line to the IPTG spot (focus), and the L-arabinose line (directrix) are in a fixed ratio (eccentricity). If the ratio is equal to 1, the critical line is a parabola. If the ratio is larger than 1, it is a branch of a hyperbola.
 
Figure 6 Schematic of   Critical Line model for parabola and hyperbola. CL-1 becomes no-motile on the   left side of the critical line while motile on the other side. The   eccentricity e equals to PF/PB. If e=1, we define the critical line as   parabola. If e>1, we define that as one branch of hyperbola.
 
Other function curves
 
After explorations, we found a new way to spot bacteria and stimulus and got a special function curves.
We draw two spots on the semi-solid culture medium, one with L-arabinose (Figure 7A) and the other with the mixture of IPTG and CL-1. Similarly, on the threshold ratio of the concentration of inducer and repressor, their effects are offset and a critical line is formed. Distances of the points on the critical line to the spot A and the spot B are in a fixed ratio (Figure 7B). Actually, as the critical line is quiet similar to hyperbola, we name it quasi-hyperbola.
A
B
Figure 7A Schematic of   quasi-hyperbola formation on semi-solid medium culture. 7B Actual experiment result is shown.
 
Other interesting ways to spot bacteria and stimulus are waiting to be discovered, and the idea can be extended to other function curves and patterns.
 
Thousands of years ago in China, people began to preserve food by curing them which was recorded in Qimin Yaoshu around 540 AD (Figure 1). Curing is any of various food preservation and flavoring processes of foods such as meat, fish and vegetables, by the addition of a combination of salt, nitrates, nitrite or sugar and it is one of the oldest methods of preserving food. Table salt is the primary ingredient used in food curing. Removal of water and addition of salt to meat creates a solute-rich environment where osmotic pressure draws water out of microorganisms, slowing down their growth. Doing this requires a concentration of salt of nearly 20%. It has already been proved that 5% concentration of NaCl could inhibit the growth of E.coli. However, utilizing hyperosmotic pressure to kill E.coli
haven’t been fully explored in synthetic biology. This year, our team have put efforts on this topic and developed a system that will contribute to biosafety.
 
Figure 1 The production process of curing food.
 
 
Figure 2 The schematic of osmotic-taxis design.
E.coli makes use of the EnvZ/OmpR system to mediate signal transduction in response to environmental osmolarity changes. EnvZ, a histidine kinase, undergoes trans-autophosphorylation, then the high-energy phosphoryl group is subsequently transferred to OmpR, a response regulator.
In our system, we involved OmpR-controlled promoter (PompC) in (Figure 2). The expression strength of PompC is depending upon the medium osmolarity. When medium osmolarity is increasing, the EnvZ will phosphorylate more OmpR into phosphorylated OmpR (OmpR-P), resulting in stronger expression strength of PompR. In our circuitry design, CheZ is upstream regulated by PompR.
 
We use semi-solid medium culture with gradient concentration of sucrose to characterize the device (BBa_K1412008). And we assume that the motile ability is proportional to the moving radius. In the plot (Figure 3), when no sucrose added in, the motile ability is the weakest. The motile ability keeps growing as the concentration of sucrose increases from 0 to 4%. Then the motile ability goes down slightly as the sucrose concentration increased from 4% to 10%, but the ability is still stronger than that at concentration 0. We can draw a conclusion that our device is working as expectation, the motile ability goes down (4%~10%) because of the inhibition from hyperosmotic pressure.
Figure 3 The plot of moving radius versus sucrose concentration. The four curves were measured after 10h, 11h, 12h and 16.5h respectively.
 
Based on the characterization, we spotted hyperosmotic pressure spot and reprogrammed CL-1 spot on semi-solid medium culture as Figure 4 shows. The concentration will decrease with the increase of the distance away from hyperosmotic pressure spot. As the osmotic pressure is proportional to the medium concentration. The moving tendency of reprogrammed CL-1 will orient to the hyperosmotic pressure spot. Even at the inhibiting osmotic pressure, the motile ability is still stronger than that without any inducer. So reprogrammed CL-1 may even swim towards the high-osmotic site and die. The killing mechanism is just like the black hole. When the bacteria move into the “event horizon” where the osmotic pressure reaches to the critical value named the killing osmotic pressure, the bacteria can’t go out of the border
and be killed finally.
 
Figure 4 Schematic of killing bacteria by black hole.
 
It’s very cheap and accessible to get the source (such as NaCl and sucrose) to create hyperosmotic pressure, while antibiotics is expensive and have a bad effect on environmental microbiology because of drug resistance. However, the source to hyperosmotic pressure is environmentally friendly and won’t generate the risk of drug resistance. If our black hole system could be fully developed, it will reduce the access barriers to microbiology research especially for the scientists from poor countries.
 
The sources (such as NaCl and sucrose) to create hyperosmotic pressure are cheap, accessible and environmentally friendly, while antibiotics is expensive and have a bad effect on environmental microbiology because of drug resistance. If our black hole system could be fully developed, barriers to microbiology research will be removed especially for the scientists from poor countries.
XMU-China 2013 has tried to construct oscillation system by standard biobricks. The synchronized oscillation system used in that study (Figure 1A) is based on the quorum sensing machineries in Vibrio fischeri and Bacillus thurigensis. Three identical luxI promoters are in charge of luxI (from V. fischeri), aiiA (from B.thurigensis) and gfp genes separately. The LuxI synthase generates an acyl-homoserine-lactone (AHL), which can spread across the cell membrane and mediate intercellular coupling. AHL then binds to LuxR produced intracellularly, and the LuxR-AHL complex would activate the luxI promoter. AiiA catalyzes the degradation of AHL as the negative feedback in the circuit. Therefore, both the activator AHL and the repressor AiiA of the network are activated by the luxI promoter simultaneously.
 
A
B
Figure 1 A
B Two oscillation cycles were observed within 500 minutes.
 
Based on above principle, one published paper has already realized synchronized oscillations under microfluidic device. However, XMU-China 2013 can’t get synchronized oscillation on microfluidics, and that will be discussed later. Through calculating fluorescence on 96-microwell plate every 15 minutes, they got two oscillation cycles within 500 minutes (Figure 1B).
 
Based on that, we construct our circuit by replacing GFP with CheZ (Figure 2). As the expression strength of CheZ is oscillatory fluctuating, the motile ability will change periodically. Bacteria will have the strongest motile ability at wave crest while even be non-motile at wave trough. Thus, the periodical change of motile ability leads to bacteria density distributing unevenly. When the bacteria are at non-motile period, they will aggregate together leading to the formation of growth-ring-like patterns which could be distinguished by naked eyes.
 
Many trees in temperate zones make one growth ring each year, with the newest adjacent to the bark. We can tell a tree’s age by counting the number of growth rings. Analogously, bacteria rings could also be formed by gene oscillator. Multiply the period by the quantity of bacteria rings, we can tell how much time has passed.
Figure 2
 
In the project of iGEM13 XMU-China, they can’t get expected oscillation. However, this year iGEM14 XMU-China further investigate the reason of abnormal oscillation. We further review SDS-PAGE analysis to confirm the circuit at protein level. The SDS-PAGE data is shown in Figure 1. Based on that, we make a reasonable assumption that the unexpected behavior of the LuxR Promoter leads to the misfolding proteins hence the abnormal oscillation.
 
Figure 1 SDS-PAGE analysis of E.coli K strain (DH5α). (a) Lane 1-2: supernatant and pellet of original DH5α; Lane 3-4: supernatant and pellet of strain with single plasmid A1 (BBa_K1036003); Lane 5-6: supernatant and pellet of strain with both plasmids A1 (BBa_K1036003) and B (BBa_K1036000). The red arrows indicate the misfolding GFP-LVA protein (27.6 kDa) in the precipitation. (b) Lane 1-2: supernatant and pellet of original BL21; Lane 3-4: supernatant and pellet of strain with single plasmid A1 (BBa_K1036003); Lane 5-6: supernatant and pellet of strain with both plasmids A1 (BBa_K1036003) and B (BBa_K1036000). The blue arrows indicate LuxR (27.5 kDa), GFP-LVA (27.6 kDa) and AiiA-LVA (28.7 kDa) in the supernatant. The orange arrows indicate LuxI-LVA (22.4 kDa) in the supernatant. (The marker of b was not in right position, however, the proteins were confirmed by MALDI-TOF-TOF .)
As the SDS-PAGE shows, a large amount of GFP-LVA and LuxI-LVA appear in pellet where misfolding proteins often exist. Both proteins directly affect the oscillation result. And it is critical to find out the reason for misfolding proteins. iGEM14 XMU-China make the following assumption:
   
The 2012 published paper reveals an unexpected behavior of Lux pR (BBa_R0062). In the absence of autoinducer 3OC6 (AHL), LuxR binds to Plux (Lux pR) and activates backwards transcription (Figure 2).
 
Figure 2 Relative RFP fluorescence for a control construct designed to measure backwards transcription from Lux pR. Addition of LuxR and 3OC690 (AHL) as indicated. Error bars in all panels are one standard deviation.
 
The imperfect simplification of setting lux pL and Lux pR in the same direction:
From the original design by Jeff Hasty, we can see that Lux pR and Lux pL are set in opposite directions (Figure 3A). In the absence of AHL, LuxR could activate backward transcription of Lux pR leading to more expression of LuxR which is critical to meet the oscillation conditions. However, present literature don’t consider the backwards transcription which have effect on quorum sensing oscillation.
 
A. Original Design
B. iGEM13 XMU-China Design
Figure 3 A. Top row is the original design by Jeff Hasty. B. Bottom row is the simplified design which sets lux pL and lux pR in the same direction.
In the simplified design (Figure 3B), when LuxR activates the backward transcription, RNA polymerase will be blocked by the terminators B0015. So that this simplification doesn’t follow the original design. Actually, the reverse terminated efficiency of B0015 is 0.295(CC) which may lead to leakage transcription. However, the correct sequence of GFP-LAA can’t be transcribed during the backwards transcription, even if the minus-strand of GFP-LAA could be transcribed, the sequence of the RNA is not the right codons of GFP-LAA. hence incorrect amino acid sequences may be translated, resulting in misfolding GFP expression which is just as the SDS-PAGE shows (Figure 1). Misfolding GFP may also be produced by bi-directional transcription.
 
Because of the imperfect simplified design doesn’t follow the original function completely, the abnormal oscillation is justifiable. The misfolding protein is a evidence to support our assumption.
 
iGEM14 XMU-China involved sequence comparison to investigate the difference between the original and the registry parts. We find that the original Lux pR has 20bp overlapping sequence with original Lux pR. There is a restriction enzyme cutting site (EcoR I) at the 56bp of original Lux pR (Figure 4).
Figure 4 Schematic of original P system.
Parts registry truncate the original Lux pR at 56bp to get the 55bp Lux pR (BBa_R0062). On the contrary, Lux pL (BBa_R0063) is longer the original Lux pL, and at the end of BBa_R0063 is initial part of 41bp LuxR (BBa_C0062). Thus new problems arise—Is the modification of original P system reasonable? Does the modification result in the unexpected backward transcription?
 
Quorum sensing system is so widely used in the synthetic biology, we think it’s remarkable to make it clear. We highlight the abnormal phenomenon of QS oscillation which may be caused by imperfect simplification for the very first time. We hope that more effort would be made to figure out the interaction between QS oscillation parts.
 
Retrieved from " http://2014.igem.org/Team:XMU-China/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
