"
Page
Discussion
View source
History
teams
Log in
 
Team:XMU-China/Notebook Protocol
From 2014.igem.org
side_bar
Project
Background
Conic Curve
Application
P System
Modelling
Future Work
Interlab
Notebook
Protocol
Lab notes
Calendar
Judging
Parts
Achievements
Team
Members
Attribution
P&P
Overview
Newsletter
Communication
Popularization
Assistance
Acknowledgements
Safety
Protocols
General Considerations
E. coli Strains
E. coli strain DH5α, was offered by XMU-China iGEM 2013.
E. coli strain CL-1, whose gene of cheZ had been knocked out, was offered generously by iGEMer JianDong Huang and Lizhong Liu, from Department of Biochemistry, The University of Hong Kong.
Antibiotic
Antibiotic
Stock Concentration (mg/mL)
Final Concentration (μg/mL)
Solvent
Chloramphenicol
50
50
Absolute ethyl alcohol
Ampicillin
50
100
ddH2O
Kanamycin
50
50
ddH2O
Tetracycline
50
50
Absolute ethyl alcohol
 All the antibiotics above require filtration to sterilize.
 Stock in -20°C
LB Liquid Medium
Ingredients
Percent (g/mL)
Yeast Extract
0.5%
Tryptone
1%
NaCl
1%
LB Solid Medium
Ingredients
Percent (g/mL)
Yeast Extract
0.5%
Tryptone
1%
NaCl
1%
Agar
1.5%
LB Semi-Solid Medium
Ingredients
Percent (g/mL)
Yeast Extract
0.5%
Tryptone
1%
NaCl
1%
Agar
0.25%
Preparation of Competent Cell (TaKaRa Competent Cell Preparation Kit)
 Activate the E. coli strain on LB-plate from glycerol stock under the condition of 37°C for 12 hours.
 Pre-culture a single colony in 5 mL LB liquid medium under the condition of 37°C, 200 rpm for 12 hours.
 Add 200 μL pre-culture bacteria into 20 mL LB liquid medium. Culture under the condition of 37°C, 200 rpm for 2 hours.
 Place on ice for 30 min.
 Subpackage the 20 mL culture medium into 1.5 mL sterile centrifuge tube, each centrifuge tube containing 1 mL.
 Centrifuge at 1500×g for 5 min at 4°C, then discard supernatant.
 Re-suspend each centrifuge tube gently with 100 μL Solution A, which has been precooled in ice.
 Centrifuge at 1500×g for 5 min at 4°C, then discard supernatant.
 Re-suspend each centrifuge tube gently with 100 μL Solution B, which has been precooled in ice.
 Stock in -20°C
Transformation
 Add 1 μL purified plasmid or 10 μL ligation system into 100 μL fresh competent cells, which is contained in 1.5 mL centrifuge tube. Then mix gently.
 Leave on ice for 30 minutes.
 Heat shock at 42°C for 90 seconds.
 Leave on ice for 10 minutes immediately.
 Add 400 μL fresh LB medium into the centrifuge tube.
 Incubate for 2 hours under the condition of 37°C, 200 rpm.
Note: If the resistance of vector is ampicillin, incubation time is shortened to 1 hour.
 Spread 100 μL culture medium on a LB solid medium plate, which contains appropriate antibiotics.
 Incubate overnight at 37°C
Confirmation
1. Plasmid Extraction (Omega Plasmid Mini Kit)
 Add 1 mL overnight suspension culture into a 1.5 mL sterile centrifuge tube. Centrifuge at 10000 x g for 1 min, then discard supernatant.
 Repeat the step above several times, until the amount of bacteria is appropriate.
 Re-suspend cells gently with 250 μL Solution I/RNase A.
 Add 250 μL Solution II into each centrifuge tube. Mix gently by inverting 4-6 times to obtain cleared lysate.
 Add 350 μL Solution III into each centrifuge tube. Mix well to form protein precipitate.
 Centrifuge at ≥13000 x g for 10 min.
 Transfer cleared lysate into a HiBind Mini Column placed in a 2 mL collection tube.
 Centrifuge at 10000 x g for 1 min. Discard liquid.
 Wash the HiBind Mini Column with 500 μL HB Buffer. Centrifuge at 10000 x g for 1 min. Discard liquid.
 Using the same collection tube, wash the HiBind Mini Column with 700 Μl DNA Wash Buffer, which is diluted with ethanol. Centrifuge at 10000 x g for 1 min. Discard liquid.
 Wash the HiBind Mini Column a second time with an additional 700 μL DNA Wash Buffer. Centrifuge at 10000 x g for 1 min. Discard liquid.
 Centrifuge the empty HiBind Mini Column at ≥ 13000 x g for 2 min to spin-dry.
 Place the HiBind Mini Column into a sterile 1.5 mL centrifuge tube.
 Add 30 μL sterile ultrapure water directly into the HiBind Mini Column. Sit at room temperature for 2 minutes. Centrifuge at maximum speed (≥ 13000 x g) for 1 min to elute DNA.
2. Digestion for Confirmation
Reaction condition: 37°C for 2 hours.
Single Digestion
Double Digestion
Plasmid
5 μL
Plasmid
5 μL
EcoR I
0.5 μL
Xba I
0.5 μL
Pst I
0.5 μL
10×H Buffer
1 μL
10×M Buffer
1 μL
ddH2O
3.5 μL
ddH2O
3 μL
Total Volume
10 μL
Total Volume
10 μL
3. DNA gel electrophoresis (take 20 mL as example)
 Dilute 50×TAE to 1×TAE with ddH2O.
 Measure 20 mL 1×TAE buffer.
 Pour 1×TAE buffer into an erlenmeyer flask.
 Weigh 0.2g agarose to make 1% DNA gel.
 Pour agarose into the erlenmeyer flask.
 Make agarose melt by microwave, lasting for 105 seconds until melt completely.
 Cool for 5 minutes.
 Assemble gel pouring apparatus by inserting gate into slots.
 Pour agarose gel into gel tray.
 Bring a piece of silver paper on the top of the gel pouring apparatus.
 Wait for 30-40 minutes to solidify the DNA agarose gel.
Ligation
1. Digestion for Ligation
Reaction condition: 37°C for 2.5 hours.
Insert(A) + Vector(B)
Vector (A) + Insert (B)
Insert(A)
30 μL
Vector(B)
30 μL
Insert(B)
30 μL
Vector(A)
30 μL
EcoR I
3 μL
EcoR I
4.5 μL
Xba I
3 μL
Spe I
4.5 μL
Spe I
3 μL
Xba I
4.5 μL
Pst I
3 μL
Pst I
4.5 μL
10×H Buffer
4 μL
10×M Buffer
4 μL
10×M Buffer
4 μL
10×H Buffer
4 μL
ddH2O
0 μL
ddH2O
7 μL
ddH2O
0 μL
ddH2O
7 μL
Total Volume
40 μL
Total Volume
50 μL
Total Volume
40 μL
Total Volume
50 μL
2. Gel Extraction of DNA (Omega Gel Extraction Kit)
 Perform DNA gel electrophoresis to fractionate DNA fragments.
 When adequate separation of bands has occurred, excise the DNA fragments of interest carefully.
 Weigh in sterile 1.5 mL centrifuge tube.
 Add corresponding volume of Binding Buffer (XP2) into the centrifuge tube.
 Heat the DNA/gel mixture at 55-65°C for 7 min, until the gel melt completely.
 Put a HiBind DNA Mini Column in a 2 mL collection tube. Add the DNA/gel solution to the HiBind DNA Mini Column.
 Centrifuge at 10000 x g for 1 min at room temperature. Discard the flow-through liquid.
 Add 300 μL Binding Buffer (XP2) into the centrifuge tube. Centrifuge at 10000 x g for 1 minute at room temperature. Then discard the flow-through liquid.
 Add 700 μL SPW Wash Buffer (diluted with absolute ethanol). Centrifuge at 10000 x g for 1 minute at room temperature. Then discard the flow-through liquid.
 Repeat the previous step.
 Centrifuge the empty HiBind Mini Column at ≥ 13000 x g for 2 min to spin-dry.
 Place the HiBind Mini Column into a sterile 1.5 mL centrifuge tube.
 Add 30 μL Elution Buffer directly into the HiBind Mini Column. Sit at room temperature for 2 minutes. Centrifuge at maximum speed (≥ 13000 x g) for 1 min to elute DNA.
3. Ligation
 Reaction condition: 16°C for 5-12 hours.
 Measure the concentration of DNA fragments and vector which are going to be ligated.
 Calculate the volume of insert fragments and vector which should be added, based on the formula:
 Add DNA fragments, vector, ligation buffer and T4 Ligase together in an EP tube as the table below.
Insert Fragments
V1 μL
Vector
V2 μL
10×Ligation Buffer
1 μL
T4 Ligase
1 μL
Add ddH2O until the total volume equals 10.0 μL
Note: Mixing the ligation system well by pipetting up and down gently is necessary.
General PCR (Polymerase Chain Reaction)
PCR System
Template
1 μL
Forward Primer
2 μL
Reverse Primer
2 μL
dNTP Mixture
2 μL
10×PCR Buffer
5 μL
rTaq (TakaRa)
1 μL
ddH2O
37 μL
Total Volume
50 μL
PCR Condition
1
94°C
5 min
2
94°C
30 s
30 loops
3
Tm°C
30 s
4
72°C
1 min/kb
5
72°C
10 min
6
15°C
5 min
Conic curve formation
Medium: M63 [1] semi-solid medium preparation
M63 semi-solid medium/100 mL
Ingredients
Quantities
KH2PO4
1.36 g
KOH
0.42 g
(NH4)2SO4
0.2 g
MgSO4
0.012 g
Agar( gel strength>750 g/cm2)
0.25 g
H2O 1
96 mL
Glycerol
0.2 mL
FeSO4 2
10.84*10-4 g
D-glucose
0.4 g
Asp 3
6.6*10-6 g
Met, Leu, His, Thr (0.015 g/mL) 4
1 mL
Note：
1. Add 0.25 g Agar into a erlenmeyer flask first, then pour into 96 mL H2O, shaking up, continue to add glycerol using pipette.
2. Add 10 μL 0.1 g/mL FeSO4 solution into the medium.
3. Add 10 μL 66 mg/mL Asp solution into the medium.
4. Met, Leu, His, Thr could be premixed for convenience, and the amino acid should be added by injector through 0.22 μm filter membrane. Pay attention that this process must be carried out after high temperature sterilization of the M63 semi-solid medium.
I. Mix-doting bacteria, inductor and inhibitor on M63 semi-solid medium plate [2]
1. Pour 20 ml M63 semi-solid medium on a plate, leave the plate air-dried for 90 min in laminar flow clean bench.
2. Dot 15 μL inductor (IPTG) or inhibitor (L-Arabinose) on the plate through pipette, take care when insert the tip into the medium. Leave the plate air-dried for 20 min in laminar flow clean bench.
3. Dot 3 μL culture medium at the same position of the inductor (IPTG) or inhibitor (L-Arabinose), leave the plate air-dried for 5 min in laminar flow clean bench.
4. Seal the plate.
II. Lineation of inductor and inhibitor on the M63 semi-solid medium plate
1. Pour 20 mL M63 semi-solid medium on a plate, leave the plate air-dried for 90 min in laminar flow clean bench.
2. Inhale 15 μL inductor (IPTG) or inhibitor (L-Arabinose), doting along the line pre-pianted carefully, bear in mind don’t destroy the semi-solid medium when moving along the line.
3. Leave the plate air-dried for 90 min in laminar flow clean bench.
4. Repeat the process mix-doting bacteria, inductor and inhibitor on M63 semi-solid medium plate.
Characterization the activity of promoter and efficiency of RBS by chemotaxis
culture
1. Add 50 μL bacterium solution (pLac-RBS (1.0)-CheZ-TT, pLac-RBS (0.3)-CheZ-TT, pLac-RBS (0.01)-CheZ-TT,) into 5 mL fresh LB liquid medium, in which chloromycetin concentration is 50 μg/ml.
2. Culture overnight under the condition of 37°C, 200 rpm.
3. Add 50 μL bacterium solution from the LB liquid medium above to another 5 mL fresh LB liquid medium, in which chloromycetin concentration is 50 μg/mL.
4. Culture under the condition of 37°C, 200 rpm for 3 hours. Stock it in 4°C.
5. Prepare M63 semi-solid medium as above.
6. Draw three dots on a M63 semi-solid medium plate first, then inoculate the three kinds of bacterium solution at the three dots, respectively. The volume of the inoculated bacterium solution is 3 μL.
7. Culture in incubator at 37°C.
Measurement
1. Use a ruler to measure the radius of colony from the bottom of the semi-solid medium plate. The initial colony radius is recorded as R1, and the radiuses measured at 12h, 24h, 30h, 36h, 42h... as R2, R3, R4, R5, R6, R7…
2. Record time and diameter in a table.
3. Process data with excel.
Black hole
Plasmid Construction
All DNA parts applied in these experiments came from the MIT Registry of Standard Biological Parts. The new BioBrick part, PompC-cheZ (BBa_K1412010) was formed by ligating the pre-existing parts, PompC (BBa_R0082) and cheZ (BBa_K629003). The E.Z.N.A.TM Plasmid Mini Kit I (200) was used according to the manufacturer’s instructions to extract plasmid DNA. BioBrick parts were digested with restriction enzymes, EcoRI, Xba I, Spe I, and Pst I to produce ‘sticky ends’ that were used to combine BioBrick parts. Digested DNA was purified using 1% agarose gel electrophoresis followed by gel purification with the E.Z.N.A.TM Gel Extraction Kit (200). BioBrick parts with matching ‘sticky ends’ were ligated using T4 ligase produced by Takara Biotechnology (Dalian) Co., Ltd. (Takara Dalian). Plasmid DNA was transformed into DH5α
first. Then we began a new circulation consisting of extracting plasmids, digestion and DNA gel electrophoresis to confirm the success of the ligation. Finally, the construct was sequenced for verification.
Construction of E. coli strain CL-1
The new biobrick part, PompC-cheZ (BBa_K1412010) on pSB1C3 was introduced into E. coli strain CL-1, whose gene of cheZ had been knocked out.
I. Gradient Plate Experiments (semi-solid medium, Cm 50μg/ml)
Medium: Medium A [3] (Improved)
Ingredients
Percent (g/mL)
Yeast Extract
0.26%
Peptone
0.27%
K2HPO4
0.37%
KH2PO4
0.13%
NaCl
0.05%
Glycerol
0.2%
Sucrose concentration gradient: 0~20%, the concentration gradient was 2%.
Mid-log-phase cell suspensions preparation:
1. Overnight culture of the required strain grow at 37 °C in Medium A (Improved) containing appropriate chloramphenicol.
2. Dilute the overnight culture at the ratio of 1:100 in fresh culture (Medium A), then incubate the culture under the condition of 37°C, 200 rpm, until its OD590 reaches 0.2.
3. Stock in 4°C.
The method of preparing gradient semi-solid medium (0.25% agar)
1. Prepare 100% sucrose solution.
2. Prepare the medium with tripled ingredients amount.
3. Put the 100% sucrose solution, the prepared medium and plenty of deionized water into the autoclave for 121°C, 20 minutes.
4. After sterilization, put the 100% sucrose solution, the prepared medium and the deionized water into a drying oven for around 10 minutes.
5. Take 5 mL tripled Medium A (Improved) into a 25 mL erlenmeyer flask, then add appropriate volume of 100% sucrose solution into the erlenmeyer flask, while the shortage will be complemented by deionized water to make the total volume to be 15 mL.
6. Add 15 μL chloramphenicol into the erlenmeyer flask.
7. Pour the mixture into a Petri dish (85 mm dia.) as quickly as possible.
8. Leave the plate air-dried for 90 min in laminar flow clean bench.
9. After solidification of the media, cell suspensions from mid-log-phase cultures (3 μL) were inoculated in the center of the plate.
10. Leave the plate dried in air for 15 min in laminar flow clean bench.
11. Leave the plate incubated at 37°C. During the period of time, measure the diameters of the colony and take them down.
II. Plates Crossed Experiments (semi-solid medium, Cm 50μg/ml)
Medium: half Medium A (Improved)
Ingredients
Percent (g/mL)
Yeast Extract
0.13%
Peptone
0.135%
K2HPO4
0.195%
KH2PO4
0.065%
NaCl
0.025%
Glycerol
0.1%
The concentration of sucrose: 0%, 10%
The method of preparing liquid medium containing different concentrations of sucrose:
1. Prepare the medium with corresponding ingredients amount in the table above.
2. Add corresponding amount of solid sucrose into the liquid medium.
3. Wait for sterilization.
Mid-log-phase cell suspensions preparation:
1. Overnight culture of the required strain grow at 37°C in Medium A (Improved) containing appropriate chloramphenicol.
2. Dilute the overnight culture at the ratio of 1:100 in fresh culture (half Medium A), then incubate the culture under the condition of 37°C, 200 rpm, until its OD590 reaches 0.2.
3. Stock in 4°C.
The method of preparing semi-solid medium with a cross (0.25% agar)
1. Prepare the medium with corresponding ingredients amount in the table above.
2. Put the medium containing agar, the liquid medium containing sucrose and plenty of deionized water into the autoclave for 121°C, 20 minutes.
3. After sterilization, put the mediums and the deionized water into a drying oven for around 10 minutes.
4. Take 15 mL of the medium containing agar into a 25 mL erlenmeyer flask.
5. Add 15 μL chloramphenicol into the erlenmeyer flask.
6. Pour the mixture into a Petri dish (85 mm dia.) as quickly as possible.
7. Leave the plate air-dried for 90 min in laminar flow clean bench.
8. After solidification of the media, draw a cross composed of two straight lines on the plate, by the way of Intensive RBI (1 μL/mm). One straight line is dotted with water, the other is dotted with liquid medium containing 10% sucrose. The length of the lines is 6 centimeters.
9. Leave the plate dried in air for 15 min in laminar flow clean bench.
10. Cell suspensions from mid-log-phase cultures (3 μL) is inoculated at the cross point.
11. Leave the plate dried in air for 15 min in laminar flow clean bench.
12. Leave the plate incubated at 37°C. During the period of time, take pictures of the colony.
Osicllation timer
Medium: Luria–Bertani (LB) medium [4]
Luria–Bertani (LB) medium/100 mL
Ingredients
Quantities
Yeast Extract
0.25 g
Bacto Tryptone
0.5 g
NaCl
0.5 g
Glycerol 1
0.5 g
0.1 M HEPES 2
3-4 mL
NaOH 3
Until pH 8.0
Note :
1. Glycerol is used as the carbon source.
2. 3-4 mL 0.1 M HEPES is added after rich defined medium is utilized, serving for minimizing the effects of pH.
3. NaOH is added until PH of the medium reaches 8.0.
Experiments
1. Pour 10 mL of the medium above containing 0.25% agar (Difco, Bacto agar) into a Petridish to prepare semi-solid medium.
2. Leave the plate air-dried for 90 min in laminar flow clean bench at room temperature.
3. Spot medium culture in half thickness in the centern of the plate.
4. Cultivate at 37°C.
Aptamer key-lock
PCR System
Template
1 μL
Forward Primer
1 μL
Reverse Primer
1 μL
dNTP Mixture
4 μL
10×PCR Buffer
5 μL
rTaq (TakaRa)
1 μL
ddH2O
37 μL
Total Volume
50 μL
The source and formation of the idea
Designation
Sequence
theophylline Aptamer1
GGU GAU ACC AGC AUC GUC UUG AUG CCCUUG GCA GCA CC
theophylline Aptamer_RBS_AUG1
GGU GAU ACC AGC AUC GUC UUG AUG CC CUU GGC AGC ACC CC G CUG CAA GAC AAC AAG AUG
crRNA
CGU CUU GAU UCC UUC UUU
RBS
AGG AGG AAA
taRNA
GCA GAA CUA AGG AAG AAA
Sequence
Designation
Sequence( 5’-3’)
Length
Lock Templates
TCCCTATCAGTGATAGAGATTGACATCCCTATCAGTGATAGAGATACTGAGCACGAATTT
CTTTCTTCCTTAG TTCTGCCTAAGGAGGAAA
91 bp
Forward Primer
GTTTCTTCGAATTCGCGGCCGCTTCTAGAGTCCCTATCAGTGA
50 bp
Reverse primer
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTTCCTCCTTAGGCAGAACT
49 bp
Aptamer key Templates
TCCCTATCAGTGATAGAGATTGACATCCCTATCAGTGATAGAGATACTGAGCACGAATTTCGGTGAT
ACCAGCATCGTCTTGATGCCCTTGGCAGCACCCTGCGTAAAGAAGGAATCAAGACG
123 bp
Forward primer
GTTTCTTCGAATTCGCGGCCGCTTCTAGAGTCCCTATCAGTGATAGAGATTGA
53 bp
Reverse primer
GTTTCTTCCTGCAGCGGCCGCTACTAGTACGTCTTGATTCCTTCTTTACGCA
52 bp
New RBS Templates
TCCCTATCAGTGATAGAGATTGACATCCCTATCAGTGATAGAGATACTGAGCACTAC
TAGAGAGGAGGAAA
71 bp
Forward primer
GTTTCTTCGAATTCGCGGCCGCTTCTAGAGTCCCTATCAGTGA
50 bp
Reverse primer
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTTCCTCCTCTCTAGTAGTG
49 bp
PCR Conditions
Lock
1
94°C
5 min
2
94°C
30 s
30 loops
3
55°C
30 s
4
72°C
30 s
5
72°C
10 min
6
15°C
5 min
Aptamer Key
1
94°C
5 min
2
94°C
30 s
30 loops
3
52°C
30 s
4
72°C
30 s
5
72°C
10 min
6
15°C
5 min
New RBS
1
94°C
5 min
2
94°C
30 s
30 loops
3
55°C
30 s
4
72°C
30 s
5
72°C
10 min
6
15°C
5 min
References
[1]Adler J. A method for measuring chemotaxis and use of the method to determine optimum conditions for chemotaxis by Escherichia coli[J]. Journal of General Microbiology, 1973, 74(1): 77-91.
[2] S T, JP. G. Guiding bacteria with small molecules and RNA[J]. Journal of the American Chemical Society, 2007, (21):6807-6811.
[3] Kawaji, H., T. Mizuno, and S. Mizushima. "Influence of molecular size and osmolarity of sugars and dextrans on the synthesis of outer membrane proteins O-8 and O-9 of Escherichia coli K-12." Journal of bacteriology 140.3 (1979): 843-847.
[4] Liu C, Fu X, Liu L, et al. Sequential establishment of stripe patterns in an expanding cell population[J]. Science, 2011, 334(6053): 238-241.
[5] Topp S, Gallivan J P. Guiding bacteria with small molecules and RNA[J]. Journal of the American Chemical Society, 2007, 129(21): 6807-6811.
 
Protocol
Lab notes
Calendar
Retrieved from " http://2014.igem.org/Team:XMU-China/Notebook_Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
