<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_XMU-China_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:XMU-China/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:XMU-China/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU-China/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU-China/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:XMU-China/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:XMU-China/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Project</TITLE></P><DIV id="warp_Project"><DIV id="warp_navbar"><DIV id="nav_bar"><UL id="nav_bar_main"><LI id="navbar_main_01" class="navbar_main"><A href="#" class="nav_bar_full" id="nav_bar_full_01">Project</A></LI><LI><A href="#Project_background">Background</A></LI><LI><A href="#Project_Circuits_Design">Design</A></LI><LI><A href="#Project_">Modelling</A></LI><LI><A href="#">Results</A></LI><LI><A href="#">Application</A></LI><LI><A href="#">Interlab</A></LI><LI><A href="#">Future</A></LI></UL><LI id="navbar_main_02" class="navbar_main"><A href="https://2014.igem.org/Team:XMU-China/p&amp;p" class="nav_bar_full" id="nav_bar_full_02">P&amp;P</A></LI><LI><A href="#">HP</A></LI><LI><A href="#">Newsletter</A></LI><LI><A href="#">Communication</A></LI><LI><A href="#">Acknowledge</A></LI><LI id="navbar_main_03" class="navbar_main"><A href="https://2014.igem.org/Team:XMU-China/judging" class="nav_bar_full" id="nav_bar_full_03">Judging</A></LI><LI id="navbar_main_04" class="navbar_main"><A href="https://2014.igem.org/Team:XMU-China/notebook" class="nav_bar_full" id="nav_bar_full_04">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/notebook#labNote">Lab notes</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/notebook#caption">Calendar</A></LI><LI id="navbar_main_05" class="navbar_main"><A href="https://2014.igem.org/Team:XMU-China/team" class="nav_bar_full" id="nav_bar_full_05">Team</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/team#teamMember">Members</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/team#ourTeam">Gallery</A></LI><LI id="navbar_main_06" class="navbar_main"><A href="https://2014.igem.org/Team:XMU-China/safety" class="nav_bar_full" id="nav_bar_full_06">Safety</A></LI></DIV></DIV><DIV id="side_bar"><UL id="side_bar_main_1"><LI class="sidebar_main"><H3><A href="#">Project</A></H3></LI><LI><A href="#">Background</A></LI><LI><A href="#">Design</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="#">Results</A></LI><LI><A href="#">Application</A></LI><LI><A href="#">Future</A></LI></UL><LI class="sidebar_main"><H3><A href="https://2014.igem.org/Team:XMU-China/p&amp;p">P&amp;P</A></H3></LI><LI><A href="#">Human Practice</A></LI><LI><A href="#">Newsletter</A></LI><LI><A href="#">Communication</A></LI><LI><A href="#">Acknowledge</A></LI><LI class="sidebar_main" id="side_bar_judging"><H3><A href="https://2014.igem.org/Team:XMU-China/judging">Judging</A></H3></LI><UL id="side_bar_main_2"><LI class="sidebar_main"><H3><A href="https://2014.igem.org/Team:XMU-China/notebook">Notebook</A></H3></LI><LI><A href="#">Lab notes</A></LI><LI><A href="#">Calendar</A></LI></UL><LI class="sidebar_main"><H3><A href="https://2014.igem.org/Team:XMU-China/team">Team</A></H3></LI><LI><A href="#">Members</A></LI><LI><A href="#">Gallery</A></LI><LI class="sidebar_main" id="side_bar_safety"><H3><A href="https://2014.igem.org/Team:XMU-China/safety">Safety</A></H3></LI></DIV><DIV id="Project_background"><DIV id="Project_background_wd"><P style="line-height: 150%;"><SPAN style="line-height: 150%; font-size: 27px;"><SPAN> </SPAN></SPAN></P><P style="text-align: justify;"><SPAN>Bacterial chemotaxis, which is universal in <EM>E.coli</EM>, is defined as bacteria cells migration in response to a chemical stimulus. The natural <EM>E.coli </EM>chemotaxis has limited receptor proteins which can bind to only six kinds of amino acid. Nevertheless, the reprogrammed chemotaxis named pseudotaxis makes the bacteria able to respond to molecules, whose receptor proteins do not exist in classical <EM>E.coli</EM>, such as IPTG and L-arabinose, etc. </SPAN></P><P><SPAN> </SPAN></P><TABLE cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR class="firstRow"><TD width="553" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><SPAN>                                         </SPAN></P></TD></TR><TR><TD width="553" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><SPAN style="font-size: 12px;"><STRONG><SPAN>Figure </SPAN></STRONG><STRONG><SPAN>1</SPAN></STRONG><SPAN>Chemotaxis   mechanism of<EM> E.coli. </EM>The direction   of rotation of the flagellar motor is controlled by the protein CheY. If the   CheY is phosphorylated (CheY-P), it can bind to the flagellar motor protein   FliM, causing the cell to tumble. While CheY is not phosphorylated, the   flagellar motor rotates counterclockwise (CCW). [1]</SPAN></SPAN></P></TD></TR></TBODY></TABLE><P><SPAN style="color: red;"><SPAN> </SPAN></SPAN></P><P style="text-align: justify;"><SPAN><EM>E.coli</EM> have several flagella per cell (4–10 typically), which can rotate in two ways: counterclockwise (CCW) and clockwise (CW). [2] The former aligns the flagella into a single rotating bundle, causing the bacterium to swim in line, while the later breaks the flagella bundle apart such that each flagellum points in a different direction, causing the bacterium to tumble. The motility is determined by the phosphorylation state of CheY protein governed by CheZ protein. In the presence of CheZ protein, CheY-P is dephosphorylated and produce CheY, and the flagellar motor rotates CCW resulting in swimming. In the absence of CheZ, CheY is phosphorylated to CheY-P which binds to the flagellar switch protein FliM resulting in tumbling (Figure 1). [1] Therefore, we are able to control the bacterial motility by knocking out the <EM>CheZ</EM> gene of the wild-type then transfecting circuit containing <EM>CheZ</EM> gene into a <EM>CheZ</EM> knockout (Δ<EM>CheZ</EM>) strain. Besides, we introduce aptamers responding to a mass of specific molecules which can be applied to regulate gene expression, in our project,<EM> CheZ </EM><STRONG>(Figure 2)</STRONG>. </SPAN></P><P><SPAN style="font-size: 19px;"><SPAN> </SPAN></SPAN></P><TABLE cellspacing="0" cellpadding="0" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD width="553" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><SPAN style="font-size: 12px;"><STRONG><SPAN>Figure </SPAN></STRONG><STRONG><SPAN>2</SPAN></STRONG><SPAN> Mechanism of how aptamers controls the   translation of CheZ protein. In the absence of target molecules (theophylline   as an example). The mRNA’s ribosome binding site is paired, which inhibits   the translation of CheZ protein. In the absence of CheZ, CheY-P will remain   phosphorylated and the cells tumble in place. While in the presence of   theophylline, the mRNA’s ribosome binding site will expose and the CheZ can   be expressed, allowing the cells to run and tumble. [1]</SPAN></SPAN></P></TD></TR></TBODY></TABLE><P><SPAN style="font-size: 19px;"><SPAN> </SPAN></SPAN></P><P style="text-align: justify;"><SPAN>Characterizing the circuit we constructed, we combine mathematical modeling with experiments, using modeling to guide experiments and to explain experimental phenomena. .</SPAN></P><P style="text-align: justify;"><SPAN> </SPAN></P><P style="text-align: justify;"><SPAN>And we can characterize the efficiency of RBS and promoter via migration distance positively associated with the expression strength of <EM>CheZ</EM>.</SPAN></P><P><SPAN style="font-size: 19px;"><SPAN> </SPAN></SPAN></P><P style="text-align: justify;"><SPAN>, Mathematic is the simplest and clearest language, whose value to the development of human civilization is now widely recognized because its extensive application of science, society and daily life. However, the mathematical laws in life sciences is still unclear and even in chaos. Luckily, synthetic biology can overcome these shortcomings on some level. Based on this, we design a gene circuit and expect mathematical regularities to realize the regulation and control of life activities. We hope our work can inspire people's interests to combine mathematic with synthetic biology.</SPAN></P><P><SPAN style="font-size: 19px;"><SPAN> </SPAN></SPAN></P><P><STRONG><SPAN style="font-family: 宋体; font-size: 19px;">References</SPAN></STRONG></P><P style="text-align: justify;"><SPAN>[1] Topp, Shana, and Justin P. Gallivan. &quot;Guiding bacteria with small molecules and RNA.&quot; Journal of the American Chemical Society 129.21 (2007): 6807-6811.</SPAN></P><P style="text-align: justify;"><SPAN>[2] http://en.wikipedia.org/wiki/Chemotaxis</SPAN></P><P><SPAN style="font-size: 19px;"><SPAN> </SPAN></SPAN></P><P> </P></DIV></DIV><DIV id="Project_Circuits_Design"><DIV id="Project_Circuits_Design_wd"><P><STRONG><SPAN style="color: rgb(0, 112, 192); font-family:;" new="" times="">Overview</SPAN></STRONG></P><P><SPAN style="color: rgb(0, 112, 192); font-family:;" new="" times="">What means would you like to use to get a mathematical pattern? Draw one with compass and ruler, or type a function in a drawing software? Well, <EM>E.coli</EM> can help us to achieve our goals. We make the first attempt at introducing pseudotaxis of bacteria to form patterns in shape of conic section (such as ellipse and hyperbola).</SPAN><SPAN>Firstly, let’s make precise mathematical definition on ellipse, hyperbola and parabola.</SPAN></P><P><SPAN> </SPAN></P><P><STRONG><SPAN>What are ellipse, hyperbola and parabola?</SPAN></STRONG></P><P><SPAN> </SPAN></P><P><SPAN>In mathematics, ellipse is a curve on a plane surrounding two focal points such that a straight line drawn from one of the focal points to any point on the curve and then back to the other focal point has the same length for every point on the curve (<STRONG>Figure 1</STRONG>).</SPAN></P><TABLE cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR class="firstRow"><TD width="321" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><SPAN>                                         </SPAN></P></TD></TR><TR><TD width="321" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><STRONG><SPAN>Figure 1 </SPAN></STRONG><SPAN>Schematic of   ellipse. Point F1, F2 are the two focal points and Point A is on the ellipse   curve. The sum of the distance AF1 and AF2 is equal to the constant k:   AF1+AF2=k.</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN>And Hyperbola is a conic consisting of those points whose distances to some point, called a focus, and some line, called a directrix, are in a fixed ratio (&gt;1), called the eccentricity. (<STRONG>Figure 2B</STRONG>)</SPAN></P><P><SPAN>Parabola is a conic whose eccentricity is equal to 1 (<STRONG>Figure 2C</STRONG>).</SPAN></P><P><SPAN> </SPAN><SPAN> </SPAN><SPAN> </SPAN></P><TABLE cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR class="firstRow"><TD width="302" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><STRONG><SPAN>A</SPAN></STRONG></P></TD><TD width="251" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><STRONG><SPAN>B</SPAN></STRONG></P></TD></TR><TR><TD width="553" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;" colspan="2"><P><STRONG><SPAN>Figure 2A</SPAN></STRONG><SPAN>, Schematic of   hyperbola. Point F1, F2 are the two focal points and Point A is on one of the   hyperbola branches. The absolute value of the difference of the distance AF1,   AF2 is equal to the constant k: |AF1-AF2|=k. <STRONG>2B</STRONG>, one branch of hyperbola can be defined by directrix and   eccentricity. Point A is a spot on the curve, point F is the focus. AB is the   distance between A and directrix. The eccentricity e equal to TF/AB, and   e&gt;1.</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN> </SPAN></P><TABLE cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD width="198" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><STRONG><SPAN>Figure 2C</SPAN></STRONG><SPAN> Schematic of   Parabola. Point A is a spot on the curve, point F is the focus. AB is the   distance between A and directrix. The eccentricity e equals to AF/AB, and   e=1.</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN> </SPAN></P><P><SPAN> </SPAN></P><P><STRONG><SPAN>Hypothesis </SPAN></STRONG></P><P><SPAN> </SPAN></P><P><SPAN>We find that the focal points, the constant and the ratio are the key of conic section. Therefore, we can easily get any eclipse by presetting reasonable focal points with the acceptable constant, as well as parabola or hyperbola by a fixed ratio. Based on these, we put forward the following hypothesis:</SPAN></P><OL class=" list-paddingleft-2" style="list-style-type: decimal;"><LI><P style="color: rgb(0, 0, 0); font-size: 14px; font-style: normal; font-weight: normal; margin-top: 0px; margin-bottom: 0px;"><SPAN>If we spot stimulus on semi-solid plate, it will spread from the spotting center out to the periphery. In the spreading process, the concentration is negative correlation to the distance from center. And the concentration gradient of the stimulus will be formed and maintained for a long period.</SPAN></P></LI><LI><P style="color: rgb(0, 0, 0); font-family:;"><SPAN>There is a threshold ratio of the concentrations of inducer and repressor. This means that more repressor will cause more repression, hence more inducer is needed to relieve the repression, and vice versa.</SPAN></P></LI></OL><P><SPAN> </SPAN></P><P><STRONG><SPAN>Design of circuit</SPAN></STRONG></P><P><SPAN> </SPAN></P><P><SPAN>Our circuit consists of two parts, of which one is named C (constraint), the other is named M (motile). (<STRONG>Figure 4</STRONG>):</SPAN></P><P><SPAN> </SPAN></P><OL class=" list-paddingleft-2" style="list-style-type: decimal;"><LI><P style="color: rgb(0, 0, 0); font-size: 14px; font-style: normal; font-weight: normal; margin-top: 0px; margin-bottom: 0px;"><SPAN>We build our circuit in <EM>E.coli</EM><EM>CL-1</EM> which lacks gene <EM>lacI</EM> and <EM>CheZ </EM>(</SPAN><SPAN>Δ</SPAN><SPAN>lacI, </SPAN><SPAN>Δ</SPAN><SPAN>CheZ). At the absence of CheZ,<EM> CL-1</EM> adopts non-motile phenotype.</SPAN></P></LI><LI><P style="color: rgb(0, 0, 0); font-family:;"><SPAN>Without any exogenous stimulus, <EM>E.coli</EM> will produce background amount of AraC to repress pBAD in limit degree. Even when L-arabinose isn’t involved in, promoter pBAD has expression leakage, so that part <STRONG>C </STRONG>will produces protein LacI which can bind to the operon of promoter pLac and thus repress its transcription. Because L-arabinose could induce pBAD, within certain concentration range, more L-arabinose involved in means that part <STRONG>C</STRONG> could produce more LacI to repress the expression of pLac. Because of its ability to constrain chemotaxis, this part is named C (abbreviate from constraint).</SPAN></P></LI><LI><P style="color: rgb(0, 0, 0); font-family:;"><SPAN>When IPTG is involved in, it can relieve the repression from LacI, therefore protein CheZ is produced to make our engineering bacteria (<EM>CL-1</EM>) regain motile ability. Within certain L-Arabinose concentration range which means certain constraint condition, more cheZ is produced with more IPTG involved in leading to stronger motile ability. Because of its ability to make <EM>CL-1</EM> motile, this part is named <STRONG>M</STRONG> (abbreviate from motile).</SPAN></P></LI></OL><TABLE cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD width="321" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P style="margin: 0px 0px 0px 28px; text-align: left;"><STRONG><SPAN>Figure 4 </SPAN></STRONG><SPAN>Part<STRONG> C </STRONG>produces LacI to repress the   expression of part <STRONG>M</STRONG>. Part <STRONG>M</STRONG> could produce CheZ to make <EM>CL-1 </EM>regain motile ability.</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN> </SPAN></P><P><STRONG><SPAN style="color: rgb(0, 176, 80);"><SPAN style="font-family: Calibri;">Characterization of circuit</SPAN></SPAN></STRONG></P><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times="">We sequenced the circuit above and characterized it in <EM>E.coli CL-1</EM>. As <EM>CL-1</EM> lacks <EM>LacI </EM>gene, promoter pLac won’t be repressed by background amount of LacI protein. </SPAN></P><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times="">We apply gradient test to find out which influence would be made on reprogrammed chemotaxis under the following parameters: the concentration of chloramphenicol, IPTG and L-arabinose.</SPAN></P><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times="">At first, we need to find out at which chloramphenicol concentration we could get the best chemotaxis performance. We try gradient concentration of chloramphenicol at semi-solid medium culture as <STRONG>Table 1</STRONG> show. We find that the chemotaxis performance doesn’t have overt linear relationship to chloramphenicol. Interestingly, 50μg/ml of chloramphenicol gives <EM>CL-1</EM> the best chemotaxis. So we applied that to our following characterization.</SPAN></P><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times=""> </SPAN></P><P><STRONG><SPAN style="color: rgb(0, 112, 192);"><SPAN style="font-family: Calibri;">Characterization of circuit</SPAN></SPAN></STRONG></P><P><SPAN style="color: rgb(0, 112, 192); font-family:;" new="" times="">We sequence the circuit above and characterize it in <EM>E.coli CL-1</EM>. As <EM>CL-1</EM> lacks <EM>LacI </EM>gene, promoter pLac won’t be repressed by background amount of LacI protein. </SPAN></P><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times=""> </SPAN></P><P><SPAN style="color: rgb(0, 112, 192); font-family:;" new="" times="">We design gradient tests of the concentration of chloramphenicol, IPTG and L-arabinose to find out their influence on reprogrammed chemotaxis. To begin with, we test for the best chloramphenicol concentration. We test gradient concentration of chloramphenicol at semi-solid medium culture as <STRONG>Table 1</STRONG> show. We find that the </SPAN><SPAN style="color: red; font-family:;" new="" times="">activity</SPAN><SPAN style="color: rgb(0, 112, 192); font-family:;" new="" times=""> of chemotaxis doesn’t have overt linear relationship to chloramphenicol. Interestingly, 50μg/ml of chloramphenicol gives <EM>CL-1</EM> the best chemotaxis. So we apply that to our following characterization.</SPAN></P><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times=""> </SPAN></P><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times=""> </SPAN></P><TABLE cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD width="538" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><STRONG><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times="">Table 1</SPAN></STRONG><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times=""> Curve of chemotaxis diameter under gradient concentration of   Cm (chloramphenicol).</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times=""> </SPAN></P><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times="">As promoter pBAD leads to a certain level of expression leakage of LacI, <EM>CL-1</EM> has the worst chemotaxis. We added IPTG at gradient concentration and got the results (<STRONG>Table 2</STRONG>). We find that the chemotaxis performance keeps increasing when the concentration of IPTG increases from 0 and 0.02μM and gets the best performance with the IPTG range from 0.02 to 0.025μM. We apply 0.025μM IPTG for our following characterization.</SPAN></P><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times=""> </SPAN></P><TABLE cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD width="553" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times="">Table 2 Curve of chemotaxis diameter over time under gradient   concentration of IPTG. </SPAN></P></TD></TR></TBODY></TABLE><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times=""> </SPAN></P><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times="">As more L-arabinose added in, the expression from promoter pBAD will be stronger which leads to more LacI produced resulting in the inhibition to chemotaxis. As our expectation, chemotaxis performance keeps going down as the concentration of L-arabinose increases (<STRONG>Table 3</STRONG>). We find that 0.2% of L-arabinose has the best inhibitory effect on chemotaxis with 0.025μM of IPTG added in.</SPAN></P><P><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times=""> </SPAN></P><TABLE cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD width="553" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><STRONG><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times="">Table 3</SPAN></STRONG><SPAN style="color: rgb(0, 176, 80); font-family:;" new="" times=""> Curve   of chemotaxis diameter under gradient concentration of L-arabinose.</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN> </SPAN></P><P><SPAN> </SPAN></P><P><STRONG><SPAN>Eclipse</SPAN></STRONG></P><P><SPAN> </SPAN></P><P><SPAN>If we spot IPTG on the center of the semi-solid culture medium, concentration gradient will be formed as <STRONG>Figure</STRONG><STRONG>3A</STRONG>. Larger circle represents lower concentration with a lower number labeled.</SPAN></P><TABLE width="567" cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR class="firstRow" style="height: 279px;"><TD width="222" height="279" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><STRONG><SPAN>A</SPAN></STRONG></P></TD><TD width="344" height="279" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><STRONG><SPAN>B</SPAN></STRONG></P></TD></TR><TR style="height: 21px;"><TD width="567" height="21" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;" colspan="2"><P><STRONG><SPAN>Figure</SPAN></STRONG><SPAN><STRONG>3</STRONG> Schematic of concentration   gradient. <STRONG>A,</STRONG> single point   spreading model. Larger number represents higher concentration while smaller   number represents lower concentration. <STRONG>B</STRONG>,   double points spreading model. Letter A~J represent equal concentration   points on the ellipse with two labeled number added up to 6. Red curve   represents the ellipse with two focal points on the IPTG spots.</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN>If we spot two IPTG simultaneously, two spots’ concentration gradient will interact with each other to form the ellipse boundary as image in <STRONG>Figure 3B</STRONG>. </SPAN></P><P><SPAN> </SPAN></P><P><SPAN>If we spot <EM>E.coli</EM><EM>CL-1</EM> with circuit above transformed in at the center between two IPTG spots on semi-solid culture plate as the image in <STRONG>Figure 5</STRONG>. The concentration of L-Arabinose added in the culture medium determines the degree of constraint from part <STRONG>C</STRONG>. Each red ellipse curve represents equal IPTG concentration, and there are a series of such ellipses from inside to outside represent IPTG concentrations from highest to lowest. One of the ellipses is the <STRONG>critical line</STRONG> indicating that the constraint from part <STRONG>C</STRONG> can just be relieved by the certain concentration of IPTG. Initially, as the concentration of IPTG is enough to relieve the constraint, bacteria can swim from the center out to periphery. When the bacteria swim out of the <STRONG>critical line</STRONG>, the concentration of IPTG can’t relieve the constraint, so the bacteria adopt non-motile phenotype. On the contrary, when the bacteria are inside of the critical line, they adopt motile phenotype. When the bacteria swim from inside to outside, motile bacteria become non-motile, so the bacteria will aggregate outside the <STRONG>critical line</STRONG> while</SPAN><SPAN> the b</SPAN><SPAN>acteria density inside the<STRONG> critical line</STRONG> will decline. Thus, an ellipse boundary is formed.</SPAN></P><TABLE cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD width="425" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><STRONG><SPAN>Figure 5</SPAN></STRONG><SPAN> Schematic of   Critical Line model for ellipse. Critical line represents the IPTG   concentration which can just relieve the repression from part <STRONG>C</STRONG>. So that <EM>CL-1</EM> is motile inside the ellipse while non-motile outside the   ellipse.</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN> </SPAN></P><P><SPAN> </SPAN></P><P><STRONG><SPAN>Parabola and hyperbola</SPAN></STRONG></P><P><SPAN> </SPAN></P><P><SPAN>We got the optimum concentrations of IPTG and L-arabinose as inducer and repressor through preliminary experiments. Because the concentration of stimulus will decrease during spreading, so does their effect, we use IPTG and L-arabinose of which concentrations are a little bit higher than the optimum values for our experiments.</SPAN></P><P><SPAN> </SPAN></P><P><SPAN>We draw a straight line with L-arabinose on the semi-solid culture medium, and a spot with the mixture of IPTG and CL-1 on one side of the line. In the area around the spot, the induction of IPTG is stronger than the repression of L-arabinose, <EM>CheZ</EM> is expressed and the bacteria adopt motile phenotype. However, when they approach the line where the repressor have a greater effect on the motility, they will lost their motile phenotype and stop. </SPAN></P><P><SPAN> </SPAN></P><P><SPAN>According to the hypothesis 2, on the threshold ratio of the concentration of inducer and repressor, their effects are offset and a critical line is formed. Distances of the points on the critical line to the IPTG spot (focus), and the L-arabinose line (directrix) are in a fixed ratio (eccentricity). If the ratio is equal to 1, the critical line is a parabola. If the ratio is larger than 1, it is a branch of a hyperbola.</SPAN></P><P><SPAN> </SPAN></P><TABLE cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD width="315" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><STRONG><SPAN>Figure 6 </SPAN></STRONG><SPAN>Schematic of   Critical Line model for parabola and hyperbola. CL-1 becomes no-motile on the   left side of the critical line while motile on the other side. The   eccentricity e equals to PF/PB. If e=1, we define the critical line as   parabola. If e&gt;1, we define that as one branch of hyperbola.</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN> </SPAN></P><P><STRONG><SPAN>Other function curves</SPAN></STRONG></P><P><SPAN> </SPAN></P><P><SPAN>After explorations, we found a new way to spot bacteria and stimulus and got a special function curves.</SPAN></P><P><SPAN>We draw two spots on the semi-solid culture medium, one with L-arabinose (Figure 7A) and the other with the mixture of IPTG and CL-1. Similarly, on the threshold ratio of the concentration of inducer and repressor, their effects are offset and a<STRONG> critical line</STRONG> is formed. Distances of the points on the <STRONG>critical line</STRONG> to the spot A and the spot B are in a fixed ratio (Figure 7B). Actually, as the critical line is quiet similar to hyperbola, we name it quasi-hyperbola.</SPAN></P><TABLE cellspacing="0" cellpadding="0" width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR class="firstRow"><TD width="252" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><SPAN>A</SPAN></P></TD><TD width="296" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;"><P><SPAN>B</SPAN></P></TD></TR><TR><TD width="548" valign="top" style="padding: 0px 7px; border: rgb(0, 0, 0); border-image: none; background-color: transparent;" colspan="2"><P style="margin: 0px 0px 0px 7px;"><STRONG><SPAN>Figure 7A</SPAN></STRONG><SPAN> Schematic of   quasi-hyperbola formation on semi-solid medium culture. <STRONG>7B</STRONG> Actual experiment result is shown.</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN> </SPAN></P><P><SPAN>Other interesting ways to spot bacteria and stimulus are waiting to be discovered, and the idea can be extended to other function curves and patterns.</SPAN></P><P> </P></DIV></DIV><DIV id="Project_Black_hole"><DIV id="Project_Black_hole_wd"><P><SPAN>T</SPAN><SPAN>housands of years ago in C</SPAN><SPAN>hina, people </SPAN><SPAN>began to preserve food by curing them</SPAN><SPAN> which was recorded in</SPAN><SPAN style="font-family: ; font-style: italic;"> Qimin Yaoshu</SPAN><SPAN> around 540 AD</SPAN><SPAN> (</SPAN><SPAN style="font-family: ; font-weight: 700;">Figure 1</SPAN><SPAN>)</SPAN><SPAN>. Curing is any of various food preservation and flavoring processes of foods such as meat, fish and vegetables, by the addition of</SPAN><SPAN> a combination of salt, nitrate</SPAN><SPAN>s</SPAN><SPAN>, nitrite or sugar and it is one of the oldest m</SPAN><SPAN>ethods of preserving food. T</SPAN><SPAN>able salt is the primary ingredient used in food curing. Removal of water and addition of salt to meat creates a solute-rich environment where osmotic pressure draws water out of microorganisms, slowing down their growth. Doing this requires a concentration of salt of nearly 20%. </SPAN><SPAN>It has already been proved that 5% concentration of NaCl could inhibit the growth of </SPAN><SPAN style="font-family: ; font-style: italic;">E.coli</SPAN><SPAN>. However, utilizing </SPAN><SPAN>hyperosmotic pressure</SPAN><SPAN>to kill </SPAN><SPAN style="font-family: ; font-style: italic;">E.coli</SPAN></P><P><SPAN> haven’t been fully explored</SPAN><SPAN> in synthetic biology</SPAN><SPAN>. T</SPAN><SPAN>his year, our team have put efforts on this </SPAN><SPAN>topic </SPAN><SPAN>and develop</SPAN><SPAN>ed</SPAN><SPAN> a system </SPAN><SPAN>that</SPAN><SPAN> will contribute to biosafety.</SPAN></P><P> </P><TABLE width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 553px; vertical-align: top;"><P><SPAN style="font-family: ; font-weight: 700;">Figure 1</SPAN><SPAN> The production process of curing food.</SPAN></P></TD></TR></TBODY></TABLE><P> </P><P> </P><TABLE width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD style="padding: 0px 7px; border: currentColor; width: 553px; vertical-align: top;"><P><SPAN style="font-family: ; font-weight: 700;">Figure 2</SPAN><SPAN> The </SPAN><SPAN>schematic of osmotic-taxis design.</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN style="font-family: ; font-style: italic;">E</SPAN><SPAN style="font-family: ; font-style: italic;">.c</SPAN><SPAN style="font-family: ; font-style: italic;">oli</SPAN><SPAN> make</SPAN><SPAN>s</SPAN><SPAN> use of the EnvZ/OmpR system to mediate signal transduction in response to environmental osmolarity changes. EnvZ, a histidine kinase, undergoes trans-autophosphorylation, then the </SPAN><SPAN>high-</SPAN><SPAN>energy phosphoryl group is subsequently transferred to OmpR, a response regulator</SPAN><SPAN>.</SPAN></P><P><SPAN>In </SPAN><SPAN>our</SPAN><SPAN> system, we </SPAN><SPAN>involved OmpR-controlled promoter</SPAN><SPAN> (PompC)</SPAN><SPAN> in</SPAN><SPAN> (Figure 2)</SPAN><SPAN>. </SPAN><SPAN>The expression strength of PompC</SPAN><SPAN> is </SPAN><SPAN>depending upon the medium osmolarity. </SPAN><SPAN>When medium osmolarity is increasing, the EnvZ will phosphorylate more OmpR into phosphorylated OmpR (OmpR-P), </SPAN><SPAN>resulting</SPAN><SPAN> in stronger expression strength of PompR.</SPAN><SPAN>In our circuitry design, </SPAN><SPAN style="font-family: ; font-style: italic;">CheZ</SPAN><SPAN> is upstream regulated by PompR.</SPAN></P><P> </P><P><SPAN>We use semi-solid medium culture with gradient concentration of sucrose</SPAN><SPAN> to characterize the device (BBa_K1412008)</SPAN><SPAN>.</SPAN><SPAN>And we assume that </SPAN><SPAN>the motile ability is proportional to the moving radius.</SPAN><SPAN> In t</SPAN><SPAN>he plot (Figure 3), when </SPAN><SPAN>no sucrose added in, the motile ability is the weakest. The motile ability keeps growing </SPAN><SPAN>as </SPAN><SPAN>the concentration of sucrose increase</SPAN><SPAN>s </SPAN><SPAN>from 0 to 4%. Then t</SPAN><SPAN>he motile ability goes </SPAN><SPAN>down slightly as the sucrose concentration increased from 4% to 10%, but </SPAN><SPAN>the ability is</SPAN><SPAN> still stronger than that at</SPAN><SPAN> concentration 0. We can draw a </SPAN><SPAN>conclusion that our device is working </SPAN><SPAN>as expectation</SPAN><SPAN>, the motile ability </SPAN><SPAN>goes</SPAN><SPAN> down</SPAN><SPAN> (4%~10%)</SPAN><SPAN> because of the inhibition from hyperosmotic pressure.</SPAN></P><TABLE width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 553px; vertical-align: top;"><P><SPAN style="font-family: ; font-weight: 700;">Figure 3</SPAN><SPAN> The plot of moving radius versus sucrose concentration. The four curves were measured after 10h, 11h, 12h and 16.5h respectively.</SPAN></P></TD></TR></TBODY></TABLE><P> </P><P><SPAN>Based on </SPAN><SPAN>the</SPAN><SPAN>characterization, we </SPAN><SPAN>spotted </SPAN><SPAN>hyperosmotic pressure spot and reprogrammed</SPAN><SPAN style="font-family: ; font-style: italic;"> CL-1</SPAN><SPAN> spot </SPAN><SPAN>on semi-solid medium culture </SPAN><SPAN>as Figure 4 show</SPAN><SPAN>s</SPAN><SPAN>.</SPAN><SPAN> The concentration will decrease</SPAN><SPAN>w</SPAN><SPAN>ith the increase of the distance away from hyperosmotic pressure spot</SPAN><SPAN>. As the osmotic pressure is </SPAN><SPAN>proportional to the medium concentration. The moving tendency of reprogrammed </SPAN><SPAN style="font-family: ; font-style: italic;">CL-1</SPAN><SPAN> will orient to the hyperosmotic pressure spot.</SPAN><SPAN> Even at the inhibiting osmotic pressure, the motile ability is still stronger than that without </SPAN><SPAN>any inducer.</SPAN><SPAN> So reprogrammed </SPAN><SPAN style="font-family: ; font-style: italic;">CL-1 </SPAN><SPAN>may even swim </SPAN><SPAN>t</SPAN><SPAN>owards the high-osmotic site and die</SPAN><SPAN>. The killing mechan</SPAN><SPAN>ism is just like the black hole. W</SPAN><SPAN>hen the bacteria </SPAN><SPAN>move</SPAN><SPAN> into the “event horizon” </SPAN><SPAN>w</SPAN><SPAN>here the osmotic pressure reaches to the critical value named the killing osmotic pressure</SPAN><SPAN>, the bacteria can’t go out of the border</SPAN></P><P><SPAN> and be killed finally.</SPAN></P><P> </P><TABLE width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 378px; vertical-align: top;"><P><SPAN style="font-family: ; font-weight: 700;">Figure </SPAN><SPAN style="font-family: ; font-weight: 700;">4</SPAN><SPAN> Schematic of k</SPAN><SPAN>illing bacteria by black hole.</SPAN></P></TD></TR></TBODY></TABLE><P> </P><P><SPAN>It’s very cheap and </SPAN><SPAN>accessible</SPAN><SPAN> to get the source (such as NaCl and sucrose) to create </SPAN><SPAN>hyperosmotic pressure</SPAN><SPAN>, w</SPAN><SPAN>hile </SPAN><SPAN>antibiotics</SPAN><SPAN> is expensive and have </SPAN><SPAN>a bad</SPAN><SPAN> effect</SPAN><SPAN> on </SPAN><SPAN>environmental microbiology</SPAN><SPAN> because of drug resistance. However, the source to </SPAN><SPAN>hyperosmotic pressure</SPAN><SPAN> is </SPAN><SPAN>environmentally</SPAN><SPAN> friendly and won’t generate the risk of drug resistance. If our black hole system could be fully developed, it will </SPAN><SPAN>reduce the access barriers to microbiology research especially for the scientists from poor countries.</SPAN></P><P> </P><P><SPAN>The source</SPAN><SPAN>s</SPAN><SPAN> (such as NaCl and sucrose) to create hyperosmotic pressure</SPAN><SPAN> are cheap, </SPAN><SPAN>accessible</SPAN><SPAN> and </SPAN><SPAN>environmentally friendly</SPAN><SPAN>, </SPAN><SPAN>while antibiotics is expensive and have a bad effect on environmental microbiology because of drug resistance.</SPAN><SPAN>If our black hole system could be fully developed, barriers to microbiology research </SPAN><SPAN>will be removed </SPAN><SPAN>especially for the scientists from poor countries.</SPAN></P></DIV></DIV><DIV id="Project_Oscillation_Timer"><DIV id="Project_Oscillation_Timer_wd"><P><SPAN>XMU-China 2013 has tried to </SPAN><SPAN>construct </SPAN><SPAN>o</SPAN><SPAN>scillation </SPAN><SPAN>system by standard biobricks. </SPAN><SPAN>The synchronized oscillation syst</SPAN><SPAN>em used in that study (Figure 1</SPAN><SPAN>A</SPAN><SPAN>) is based on the quorum sensing machineries in </SPAN><SPAN style="font-family: ; font-style: italic;">Vibrio fischeri</SPAN><SPAN> and </SPAN><SPAN style="font-family: ; font-style: italic;">Bacillus thurigensis</SPAN><SPAN>. Three identical luxI promoters are in charge of </SPAN><SPAN style="font-family: ; font-style: italic;">luxI</SPAN><SPAN> (from </SPAN><SPAN style="font-family: ; font-style: italic;">V. fischeri</SPAN><SPAN>), </SPAN><SPAN style="font-family: ; font-style: italic;">aiiA</SPAN><SPAN> (from </SPAN><SPAN style="font-family: ; font-style: italic;">B.thurigensis</SPAN><SPAN>) and </SPAN><SPAN style="font-family: ; font-style: italic;">gfp</SPAN><SPAN> genes separately. The LuxI synthase generates an acyl-homoserine-lactone (AHL), which can spread across the cell membrane and mediate intercellular coupling. AHL then binds </SPAN><SPAN>to LuxR produced intracellularly</SPAN><SPAN>, and the LuxR-AHL complex would activate the luxI promoter. AiiA catalyzes the degradation of AHL as the negative feedback in the circuit. Therefore, both the activator AHL and the repressor AiiA of the network are activated by the luxI</SPAN><SPAN>promoter simultaneously.</SPAN></P><P> </P><TABLE width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR class="firstRow"><TD style="padding: 0px 7px; border: 1px rgb(0, 0, 0); border-image: none; width: 302px; vertical-align: top;"><P><SPAN style="font-family: ; font-weight: 700;">A</SPAN></P></TD><TD style="border-width: 1px 1px 1px medium; border-color: rgb(0, 0, 0) rgb(0, 0, 0) rgb(0, 0, 0) currentColor; padding: 0px 7px; width: 340px; vertical-align: top;"><P style="text-align: justify;"><SPAN style="font-weight: 700;">B</SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 642px; vertical-align: top;" colspan="2"><P><SPAN>Figure 1</SPAN><SPAN> A</SPAN></P><P><SPAN>B </SPAN><SPAN>Two</SPAN><SPAN>oscillation cycles were observed within 500 minutes.</SPAN></P></TD></TR></TBODY></TABLE><P> </P><P><SPAN>Based on above principle, one published paper has already realized </SPAN><SPAN>synchronized oscillations</SPAN><SPAN> under </SPAN><SPAN>microfluidic</SPAN><SPAN>device. However</SPAN><SPAN>,</SPAN><SPAN>XMU-China 201</SPAN><SPAN>3 can’</SPAN><SPAN>t get </SPAN><SPAN>synchronized </SPAN><SPAN>oscillation</SPAN><SPAN> on microfluidics</SPAN><SPAN>, and that will be </SPAN><SPAN>discuss</SPAN><SPAN>ed later. </SPAN><SPAN>Through calculating </SPAN><SPAN>fluorescence </SPAN><SPAN>on 96-microwell plate e</SPAN><SPAN>very 15 minutes, they got two oscillation cycles within 500 minutes (</SPAN><SPAN>Figure 1B</SPAN><SPAN>).</SPAN></P><P> </P><P><SPAN>Based on that, we con</SPAN><SPAN>struct our circuit by replacing </SPAN><SPAN style="font-family: ; font-style: italic;">GFP</SPAN><SPAN>with</SPAN><SPAN style="font-family: ; font-style: italic;">CheZ</SPAN><SPAN>(Figure 2)</SPAN><SPAN>.</SPAN><SPAN>As the expression strength of </SPAN><SPAN style="font-family: ; font-style: italic;">CheZ</SPAN><SPAN>is </SPAN><SPAN>oscillatory</SPAN><SPAN>fluctuating</SPAN><SPAN>, </SPAN><SPAN>the motile ability will change </SPAN><SPAN>periodically</SPAN><SPAN>.</SPAN><SPAN> Bacteria will have the strongest motile ability at wave crest while even be non-motile at wave tr</SPAN><SPAN>ough. Thus, the periodical </SPAN><SPAN>change of motile ability leads to bacteria density </SPAN><SPAN>distributing </SPAN><SPAN>unevenly. When the bacteria </SPAN><SPAN>are</SPAN><SPAN> at non-motile period, </SPAN><SPAN>they</SPAN><SPAN> will aggregate together leading to the formation of </SPAN><SPAN>growth</SPAN><SPAN>-ring</SPAN><SPAN>-like</SPAN><SPAN>patterns which could be distinguished by naked eyes.</SPAN></P><P> </P><P><SPAN>Many trees in </SPAN><SPAN>temperate</SPAN><SPAN> zones make one growth ring each year, </SPAN><SPAN>with the newest adjacent to the bark</SPAN><SPAN>. We can tell </SPAN><SPAN>a tree’s </SPAN><SPAN>age b</SPAN><SPAN>y counting the number of growth </SPAN><SPAN>ring</SPAN><SPAN>s</SPAN><SPAN>. A</SPAN><SPAN>nalogously</SPAN><SPAN>, bacteria rings could also be formed by gene </SPAN><SPAN>oscillator</SPAN><SPAN>. </SPAN><SPAN>Multiply</SPAN><SPAN> the </SPAN><SPAN>period</SPAN><SPAN>by</SPAN><SPAN> the </SPAN><SPAN>quantity</SPAN><SPAN> of bacteria rings, we c</SPAN><SPAN>an tell how much time has passed.</SPAN></P><TABLE width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 378px; vertical-align: top;"><P><SPAN>Figure</SPAN><SPAN> 2</SPAN></P></TD></TR></TBODY></TABLE><P> </P></DIV></DIV><DIV id="Project_Psystem"><DIV id="Project_Psystem_wd"><P style="text-align: justify; font-family: ; styleName: Normal;"><SPAN>In</SPAN><SPAN> the project </SPAN><SPAN>of</SPAN><SPAN> iGEM</SPAN><SPAN>13 XMU-China, they</SPAN><SPAN> can’t get expected oscillation. H</SPAN><SPAN>owever</SPAN><SPAN>,</SPAN><SPAN>this year</SPAN><SPAN>iGEM14 XMU-China</SPAN><SPAN> further </SPAN><SPAN>investigate the reason of abnormal oscillation. We further review</SPAN><SPAN> SDS-PAGE analysis to confirm </SPAN><SPAN>the circuit </SPAN><SPAN>at</SPAN><SPAN> protein leve</SPAN><SPAN>l. The SDS-PAGE data is shown in </SPAN><SPAN style="font-family: ; font-weight: 700;">Figure 1</SPAN><SPAN>.</SPAN><SPAN> Based on that, we make a reasonable assumption that the unexpected behavior of the LuxR Promoter lead</SPAN><SPAN>s</SPAN><SPAN> to the misfolding proteins </SPAN><SPAN>hence the</SPAN><SPAN> abnormal oscillation.</SPAN></P><P style="text-align: justify; font-family: ; styleName: Normal;"> </P><TABLE width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR style="height: 21px;"><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 588px; vertical-align: top;"><P><SPAN style="font-family: ; font-weight: 700;">Figure 1</SPAN><SPAN>SDS-PAGE analysis of E.</SPAN><SPAN>coli K strain (</SPAN><SPAN style="font-family: ; font-style: italic;">DH5α</SPAN><SPAN>). </SPAN><SPAN style="font-family: ; font-weight: 700;">(a)</SPAN><SPAN> Lane 1-2: supernatant and pellet of original </SPAN><SPAN style="font-family: ; font-style: italic;">DH5α</SPAN><SPAN>; Lane 3-4: supernatant and pellet of strain with single plasmid A1 (BBa_K1036003); Lane 5-6: supernatant and pellet of strain with both plasmids A1</SPAN><SPAN>(BBa_K1036003) and B</SPAN><SPAN> (BBa_K1036000)</SPAN><SPAN>. The red arrows indicate the </SPAN><SPAN style="font-family: ; font-weight: 700;">misfolding</SPAN><SPAN> GFP-LVA protein (27.6 kDa) in the precipitation. </SPAN><SPAN style="font-family: ; font-weight: 700;">(b)</SPAN><SPAN> Lane 1-2: supernatant and pellet of original </SPAN><SPAN style="font-family: ; font-style: italic;">BL21</SPAN><SPAN>; Lane 3-4: supernatant and pellet of strain with single plasmid A1</SPAN><SPAN> (BBa_K1036003)</SPAN><SPAN>; Lane 5-6: supernatant and pellet of strain with both plasmids A1</SPAN><SPAN> (BBa_K1036003) </SPAN><SPAN>and B</SPAN><SPAN> (BBa_K1036000)</SPAN><SPAN>. The blue arrows indicate LuxR (27.5 kDa), GFP-LVA (27.6 kDa) and AiiA-LVA (28.7 kDa) in the supernatant. The orange arrows indicate LuxI-LVA (22.4 kDa) in the supernatant. (The marker of b was not in right position, however, the proteins were confirmed by MALDI-TOF-TOF .)</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN>As the SDS-PAGE show</SPAN><SPAN>s</SPAN><SPAN>, a large amount of </SPAN><SPAN>GFP-LVA and LuxI-LVA </SPAN><SPAN>appear</SPAN><SPAN> in pellet where misfolding proteins often exist. Both protein</SPAN><SPAN>s</SPAN><SPAN> directl</SPAN><SPAN>y </SPAN><SPAN>a</SPAN><SPAN>ffect the oscillation result. </SPAN><SPAN>And it is critical to find out the reason for misfolding proteins</SPAN><SPAN>. iGEM14 XMU-China make the following assumption:</SPAN></P><P><SPAN>  </SPAN></P><P><SPAN>The 2012 published paper reveals an unexpected behavior of </SPAN><SPAN>L</SPAN><SPAN>ux pR</SPAN><SPAN>(BBa_R0062).</SPAN><SPAN> In the absence of autoinducer 3OC6 (AHL), LuxR binds to Plux</SPAN><SPAN> (Lux pR)</SPAN><SPAN> and activates backwards transcription (</SPAN><SPAN style="font-family: ; font-weight: 700;">Figure 2</SPAN><SPAN>).</SPAN></P><P> </P><TABLE width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR style="height: 62px;"><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 567px; vertical-align: top;"><P><SPAN style="font-family: ; font-weight: 700;">Figure</SPAN><SPAN style="font-family: ; font-weight: 700;"> 2</SPAN><SPAN> Relative RFP fluorescence for a control construct designed to measure backwards transcription from </SPAN><SPAN>Lux pR</SPAN><SPAN>. Addition of LuxR and 3OC6</SPAN><SPAN>90 (AHL)</SPAN><SPAN> as indicated. Error bars in all panels are one standard deviation.</SPAN></P></TD></TR></TBODY></TABLE><P> </P><P style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">The imperfect simplification</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"> of setting </SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">lux pL</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"> and </SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">L</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">ux pR</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"> in the same direction</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">:</SPAN></P><P style="text-align: justify; font-family: ; font-weight: 700; styleName: Normal;"><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">From the original design by</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">Jeff</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">H</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">asty</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">, we can see that Lux pR and Lux pL are </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">set</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> in opposite directions</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> (</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">Figure 3</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">A</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">)</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">.</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">In the absence of AHL</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">, LuxR could activate backward transcription of Lux pR</SPAN><SPAN>leading to more expression of LuxR which is</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> critical to meet the oscillation conditions. However, present literature don’t consider </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">the backwards transcription which </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">have effect</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> on </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">q</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">uorum sensing oscillation.</SPAN></P><P style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"> </P><TABLE width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR class="firstRow"><TD style="padding: 0px 7px; border: 1px rgb(0, 0, 0); width: 553px; vertical-align: top;"><P style="text-align: justify; text-indent: -24px; font-family: ; font-weight: 700; margin-left: 24px; styleName: Normal;"><SPAN style="font-weight: 700;">A.</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">Original </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">D</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">esign</SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 553px; vertical-align: top;"><P style="text-align: justify; text-indent: -24px; font-family: ; font-weight: 700; margin-left: 24px; styleName: Normal;"><SPAN style="font-weight: 700;">B.</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">iGEM13 XMU-China Design</SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 553px; vertical-align: top;"><P style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">Figure 3</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">A.</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">Top row is the original design by</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">Jeff Hasty</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">.</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">B.</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">Bottom row is the simplif</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">ied</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> design which sets lux pL and lux pR in the same direction.</SPAN></P></TD></TR></TBODY></TABLE><P style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">In the simplif</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">ied</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> design (</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">Figure 3B</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">), when LuxR activates the backward transcription, RNA polymerase will be blocked by the terminators B0015. So that thi</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">s simplif</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">ication</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">doesn’t follow the original design. Actually, the</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> reverse </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">terminated </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">efficiency </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">of B0015 is 0.295</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">(CC) which may lead to leakage transcription. </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">However, the correct sequence of GFP-LAA can’t be transcribed during the backwards transcription, even if the minus-strand of GFP-LAA could be transcribed, the sequence of the RNA is not the right codons of GFP-LAA. hence incorrect amino acid sequences may be translated, resulting in misfolding GFP expression which is just as the SDS-PAGE shows (</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">Figure 1</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">). </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">Misfolding GFP may also be produced by </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">bi-directional</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> transcription</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">.</SPAN></P><P style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"> </P><P style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">Because of the </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">imperfect simplified</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> design doesn’t follow the original function completely, the abnormal oscillation is justifiable. The misfolding protein is a evidence to support our assumption.</SPAN></P><P style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"> </P><P style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">iGEM14 XMU-China involved sequence comparison to investigate the difference between the </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">original and </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">the registry parts.</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> We find that</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> the original Lux pR has 20bp overlapping sequence with origin</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">al Lux p</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">R</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">. </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">There is </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">a r</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">estriction e</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">nzyme cutting site (EcoR I)</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> at the 56bp of original Lux pR</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> (</SPAN><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">Figure 4</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">)</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">.</SPAN></P><TABLE width="600px" style="position:relative;margin-left:-300px;left:50%"><TBODY><TR><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 245px; vertical-align: top;"><P style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"><SPAN style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;">Figure 4</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> Schematic of original P system.</SPAN></P></TD></TR></TBODY></TABLE><P><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">Parts registry truncate the original Lux pR at 56bp to get the </SPAN><SPAN>55bp Lux pR</SPAN><SPAN>(BBa_R0062). </SPAN><SPAN>On the contrary, Lux pL</SPAN><SPAN>(BBa_R0063) is longer the original Lux pL, and at the end of BBa_R0063 is </SPAN><SPAN>initial part of </SPAN><SPAN>41bp </SPAN><SPAN>LuxR (BBa_C0062). </SPAN><SPAN>Thus new problems</SPAN><SPAN> arise</SPAN><SPAN>—Is the modification of original P system reasonable? Does the m</SPAN><SPAN>odification result in the unexpected backward transcription?</SPAN></P><P> </P><P style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">Quorum sensing system is so widely used in the synthetic biology, we think it’s remarkable to make it clear. We highlight the abnormal phenomenon of QS oscillation which may</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> be caused by imperfect simplification</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> for the very first time. </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">We hope</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> that more </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">effort</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> would be </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">made</SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;"> to figure out the interaction between </SPAN><SPAN style="font-family: ; font-weight: 400; styleName: Default Paragraph Font;">QS oscillation parts.</SPAN></P><P style="font-family: ; font-weight: 700; styleName: Default Paragraph Font;"> </P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:XMU-China/Project">http://2014.igem.org/Team:XMU-China/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:XMU-China/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:XMU-China/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:XMU-China/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:XMU-China/Project&amp;oldid=231932" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>