<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_XMU-China_Project_Application_Aptamer"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:XMU-China/Project_Application_Aptamer">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:XMU-China/Project_Application_Aptamer&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU-China/Project_Application_Aptamer&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU-China/Project_Application_Aptamer&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:XMU-China/Project_Application_Aptamer" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:XMU-China/Project Application Aptamer</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>side_bar</TITLE></P><DIV id="warp_navbar_main"><DIV id="warp_navbar"><DIV id="nav_bar"><UL id="nav_bar_main"><LI id="navbar_main_01" class="navbar_main"><A href="https://2014.igem.org/Team:XMU-China/Project_Background" class="nav_bar_full" id="nav_bar_full_01">Project</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Project_Background">Background</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Project_ConicCurve">Conic Curve</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Project_Application">Application</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Project_PSystem">P System</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Project_Modelling">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Project_FutureWork">Future Work</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Project_Interlab">Interlab</A></LI></UL><LI id="navbar_main_02" class="navbar_main"><A href="https://2014.igem.org/Team:XMU-China/Notebook_Protocol" class="nav_bar_full" id="nav_bar_full_02">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Notebook_Protocol">Protocol</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Notebook_Labnotes">Lab notes</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Notebook_Calendar">Calendar</A></LI><LI id="navbar_main_03" class="navbar_main"><A href="https://2014.igem.org/Team:XMU-China/Judging_Parts" class="nav_bar_full" id="nav_bar_full_03">Judging</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Judging_Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Judging_Achievements">Achievements</A></LI><LI id="navbar_main_04" class="navbar_main"><A href="https://2014.igem.org/Team:XMU-China/Team_Members" class="nav_bar_full" id="nav_bar_full_04">Team</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Team_Members">Members</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/Team_Attribution">Attribution</A></LI><LI id="navbar_main_05" class="navbar_main"><A href="https://2014.igem.org/Team:XMU-China/PP_Overview" class="nav_bar_full" id="nav_bar_full_05">P&amp;P</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/PP_Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/PP_Newsletter">Newsletter</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/PP_Communication">Communication</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/PP_Popularization">Popularization</A></LI><LI><A href="https://2014.igem.org/Team:XMU-China/PP_Assistance">Assistance</A></LI><LI id="navbar_sub_Acknowledgements"><A href="https://2014.igem.org/Team:XMU-China/PP_Acknowledgements">Acknowledgements</A></LI><LI id="navbar_main_06" class="navbar_main"><A href="https://2014.igem.org/Team:XMU-China/Safety" class="nav_bar_full" id="nav_bar_full_06">Safety</A></LI></DIV></DIV></DIV><DIV id="side_bar"><UL id="side_bar_main_1"><LI class="sidebar_main"><H3><A href="https://2014.igem.org/Team:XMU-China/Project_Background">Background</A></H3></LI><LI class="sidebar_main"><H3><A href="https://2014.igem.org/Team:XMU-China/Project_ConicCurve">Conic Curve</A></H3></LI><LI class="sidebar_main" id="side_bar_judging"><H3><A href="https://2014.igem.org/Team:XMU-China/Project_Application">Application</A></H3></LI></UL><UL id="side_bar_main_2"><LI class="sidebar_main"><H3><A href="https://2014.igem.org/Team:XMU-China/Project_PSystem">P System</A></H3></LI><LI class="sidebar_main"><H3><A href="https://2014.igem.org/Team:XMU-China/Project_Modelling">Modelling</A></H3></LI><LI class="sidebar_main"><H3><A href="https://2014.igem.org/Team:XMU-China/Project_FutureWork">Future Work</A></H3></LI><LI class="sidebar_main" id="side_bar_safety"><H3><A href="https://2014.igem.org/Team:XMU-China/Project_Interlab">Interlab</A></H3></LI></UL></DIV><DIV id="Project_Application_RBSpromoter"><DIV id="Project_Application_RBSpromoter_wd"><P style="font-family: Arial;text-align:center;font-size: 29px;"><SPAN style="font-family: Arial; font-size: 29px; font-weight: 700;">APTAMER</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 21px;"><SPAN style="font-family: Arial; font-size: 21px; font-weight: 700;">Overview</SPAN></P><P style="font-family: Arial; font-size: 21px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">Aptamer is one kind of oligonucleotide fragments that can specifically combine proteins and small molecules selected by systematic evolution of ligands by exponential enrichment (SELEX). Tuerk and Gold</SPAN><SPAN style="font-family: Arial; font-size: 16px;"><SUP>[1]</SUP></SPAN><SPAN style="font-family: Arial; font-size: 16px;"> proposed firstly in 1990 that we could obtain a kind of target substance with high specificity to DNA and RNA via the method of application chemistry to synthesize a large-capacity random oligonucleotides library and exerting the optional press combining with the SELEX after several rounds of selection enrichment. Aptamers were applied worldwide in the field of chemistry, biology and medicine.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">Synthetic biology shows its broad application perspective in the fields of medicine, chemical synthesis, and the production of energy. In recent years, RNA aptamers have been widely used in synthetic biology. We can make full use of the gene circuits to select the aptamers to regulate the circuit expressions. For example, Culler SJ</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;"><SUP>[2]</SUP></SPAN><SPAN style="font-family: Arial; font-size: 16px;"> created a modular platform for regulating gene expression by increasing or decreasing alternative splicing based on protein-binding aptamers in 2010. What’s more, aptamers can regulate the cut of mRNA. It’s just like what Weiquand JE</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[3]</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> did in 2007. He used tetracycline-binding aptamer which can insert close to the 5’splice site (SS) to efficient regulate the splicing of pre-mRNA. Aptamers can also regulate the translation of mRNA. Shana Topp</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[4]</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> used theophylline aptamer to regulate the protein translation. When theophylline is absence, using RNA aptamer to block ribosome bind site (RBS), leading to target genes can’t express. What these scientists did showed that the diversity of how aptamers regulate the gene circuits. And we are considering using aptamers to regulate the gene circuits based on this theory.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 21px;"><SPAN style="font-family: Arial; font-size: 21px; font-weight: 700;">Design</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">James J Collins</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[5]</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> group’s work draw our attention. They use crRNA and taRNA to control </SPAN><SPAN style="font-family: Arial; font-size: 16px;">whether the circuit silence or activate. In 2009</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[6]</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> and 2012</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[7]</SPAN><SPAN style="font-family: Arial; font-size: 16px;">, Friedland A E and Callura J M, et al.</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> expand apta</SPAN><SPAN style="font-family: Arial; font-size: 16px;">mer’s applications. However, one thing to note here is that a promoter must be linked before the corresponding DNA to </SPAN><SPAN style="font-family: Arial; font-size: 16px;">transcribe them. The promoters mentioned in the papers are mostly primary ones in</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> Part Registry</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> such </SPAN><SPAN style="font-family: Arial; font-size: 16px;">as pTet, pLlacO-1 and pBAD, etc. In order to enrich the regulatory system, we </SPAN><SPAN style="font-family: Arial; font-size: 16px;">add an </SPAN><SPAN style="font-family: Arial; font-size: 16px;">apta</SPAN><SPAN style="font-family: Arial; font-size: 16px;">mer to the original circuit</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[5]</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> as a switch. Our new circuit is as following:</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="text-align: center; font-family: Arial; font-size: 16px;"><VIDEO src="https://static.igem.org/mediawiki/2014/7/7f/Xmu_project_appleication_vedio.mp4" width="504px" height="284px" controls="controls">
Sorry, your browser version is outdate.To get a better browsing experience to get a better browsing experience, please upgrade your browser.
</VIDEO></P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 21px; font-weight: 700;">Lock and Key</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">At the absence of stimulus, </SPAN><SPAN style="font-family: Arial; font-size: 16px;">crRNA is paired with RBS and block </SPAN><SPAN style="font-family: Arial; font-size: 16px;">its expression, thus CheZ</SPAN><SPAN style="font-family: Arial; font-size: 16px;">cannot be </SPAN><SPAN style="font-family: Arial; font-size: 16px;">translated </SPAN><SPAN style="font-family: Arial; font-size: 16px;">and the chemotaxis </SPAN><SPAN style="font-family: Arial; font-size: 16px;">of bacteria </SPAN><SPAN style="font-family: Arial; font-size: 16px;">won’t</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> be</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> express</SPAN><SPAN style="font-family: Arial; font-size: 16px;">ed</SPAN><SPAN style="font-family: Arial; font-size: 16px;">.</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> Meanwhile, </SPAN><SPAN style="font-family: Arial; font-size: 16px;">the aptamer </SPAN><SPAN style="font-family: Arial; font-size: 16px;">are locked by </SPAN><SPAN style="font-family: Arial; font-size: 16px;">t</SPAN><SPAN style="font-family: Arial; font-size: 16px;">aRNA and all mRNA are in silence.</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> At the presence of small regulatory molecule like theophylline</SPAN><SPAN style="font-family: Arial; font-size: 16px;">, taRNA as the key is </SPAN><SPAN style="font-family: Arial; font-size: 16px;">aroused</SPAN><SPAN style="font-family: Arial; font-size: 16px;">, so is crRNA. </SPAN><SPAN style="font-family: Arial; font-size: 16px;">Because the </SPAN><SPAN style="font-family: Arial; font-size: 16px;">binding force between taRNA and crRNA is stronger than that between crRNA and RBS, RBS is exposed and CheZ can be transcribed causing that the flagella </SPAN><SPAN style="font-family: Arial; font-size: 16px;">rotates counterclockwise</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> and the cells swim.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 21px; font-weight: 700;">Principles of design</SPAN><SUP>[3][4][5][6][7][8]</SUP></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="text-align: justify; text-indent: -24px; font-family: Arial; font-size: 14px; margin-left: 24px; styleName: Normal;"><SPAN style="font-family: Times New Roman,宋体;">1.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">The binding fore between taRNA and crRNA is stronger than that between crRNA and RBS or that between aptamer and taRNA.</SPAN></P><P style="text-align: justify; text-indent: -24px; font-family: Arial; font-size: 14px; margin-left: 24px; styleName: Normal;"><SPAN style="font-family: Times New Roman,宋体;">2.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">RBS and AUG are separated by about 6 bases.</SPAN></P><P style="text-align: justify; text-indent: -24px; font-family: Arial; font-size: 14px; margin-left: 24px; styleName: Normal;"><SPAN style="font-family: Times New Roman,宋体;">3.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">The binding site of taRNA and aptamer is near the site of theophylline and aptamer</SPAN></P><P style="text-align: justify; text-indent: -24px; font-family: Arial; font-size: 14px; margin-left: 24px; styleName: Normal;"><SPAN style="font-family: Times New Roman,宋体;">4.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">The order of secondary structures energy</SPAN><SPAN style="font-family: Arial; font-size: 16px;">: single aptamer</SPAN><SPAN style="font-family: Arial; font-size: 16px;">&lt; combination of aptamer and taRNA</SPAN><SPAN style="font-family: Arial; font-size: 16px;">&lt; combination of aptamer and theophylline</SPAN></P><P style="text-align: justify; text-indent: -24px; font-family: Arial; font-size: 14px; margin-left: 24px; styleName: Normal;"><SPAN style="font-family: Times New Roman,宋体;">5.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">The conformation of the secondary structure formed by aptamer and taRNA is the same as the one Shana Topp</SPAN><SPAN style="font-family: Arial; font-size: 16px;"><SUP>[4]</SUP></SPAN><SPAN style="font-family: Arial; font-size: 16px;"> put forward. </SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">To</SPAN><SPAN style="font-family: Arial; font-size: 16px;">achieve the functionality of the aptamer and RBS</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[4]</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> (aptamer-RBS1 for short)</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> in </SPAN><SPAN style="font-family: Arial; font-size: 16px; font-weight: 700;">Figure 1</SPAN><SPAN style="font-family: Arial; font-size: 16px;">, the position of RBS should be near the binding site of theophylline and aptamer</SPAN><SPAN style="font-family: Arial; font-size: 16px;">, as principle 3.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"> </P><TABLE style="border: currentColor; border-image: none; margin-right: auto; margin-left: auto; border-collapse: collapse; styleName: Normal Table; border-insideH: 1px none #000000; border-insideV: 1px none #000000; cellpadding: 0px 7px 0px 7px;"><TBODY><TR style="height: 65px;"><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 553px; vertical-align: top;"><P style="text-align: justify; font-size: 14px;"><SPAN style="font-size: 16px; font-weight: 700;">Figure 1</SPAN><SPAN style="font-size: 16px; font-weight: 700;">.</SPAN><SPAN style="font-size: 16px;">The green part and dark blue part are the aptamer sequence. The binding site of theophylline is marked in dark blue. The linker sequence is marked in gray. RBS is in pink, and AUG is in pale pink.</SPAN></P></TD></TR></TBODY></TABLE><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">In order to understand the </SPAN><SPAN style="font-family: Arial; font-size: 16px;">sequence’s meaning, </SPAN><SPAN style="font-family: Arial; font-size: 16px;">Justin P.Gallivan group</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[8]</SPAN><SPAN style="font-family: Arial; font-size: 16px;">set up a bank of 87296 linker sequence</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> and </SPAN><SPAN style="font-family: Arial; font-size: 16px;">transfered the cells to 96-well microtiterplates, and select </SPAN><SPAN style="font-family: Arial; font-size: 16px;">the switch by assaying</SPAN><SPAN style="font-family: Arial; font-size: 16px;">β</SPAN><SPAN style="font-family: Arial; font-size: 16px;">-galactosidase activity</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> (</SPAN><SPAN style="font-family: Arial; font-size: 16px; font-weight: 700;">Figure 2</SPAN><SPAN style="font-family: Arial; font-size: 16px;">)</SPAN><SPAN style="font-family: Arial; font-size: 16px;">.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><TABLE style="border: currentColor; border-image: none; margin-right: auto; margin-left: auto; border-collapse: collapse; styleName: Normal Table; border-insideH: 1px none #000000; border-insideV: 1px none #000000; cellpadding: 0px 7px 0px 7px;"><TBODY><TR><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 553px; vertical-align: top;"><P style="text-align: justify; font-size: 14px;"><SPAN style="font-size: 16px; font-weight: 700;">Figure 2</SPAN><SPAN style="font-size: 16px; font-weight: 700;">.</SPAN><SPAN style="font-size: 16px;">Model for synthetic riboswitch function</SPAN><SPAN style="font-size: 16px;">. </SPAN><SPAN style="font-size: 16px; font-weight: 700;">A.</SPAN><SPAN style="font-size: 16px;">Predicted structures and free</SPAN><SPAN style="font-size: 16px;"> energies of a portion of the 5’ </SPAN><SPAN style="font-size: 16px;">region clone 8.1 determine by mFold. To represent the free energies of the bound structures, the groups add the experimentally determined free energy of theophylline binding (-9.2kcal/mol) to the free energy of the unliganded structure determiner by mFold since the secondary structure of the Mtct-8-4 aptamer does not change upon ligand binding. </SPAN><SPAN style="font-size: 16px; font-weight: 700;">B.</SPAN><SPAN style="font-size: 16px;">Kinetic model for synthetic riboswitch function</SPAN><SPAN style="font-size: 16px;">.</SPAN></P></TD></TR></TBODY></TABLE><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">According to the energy value provided in the paper, we </SPAN><SPAN style="font-family: Arial; font-size: 16px;">work out</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> the energy of single aptamer</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[4]</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> and aptamer-RBS1</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[4]</SPAN><SPAN style="font-family: Arial; font-size: 16px;">by</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> mF</SPAN><SPAN style="font-family: Arial; font-size: 16px;">old.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="text-align: right; font-family: Arial; font-size: 16px;"> </P><TABLE style="border: currentColor; border-image: none; margin-right: auto; margin-left: auto; border-collapse: collapse; styleName: Normal Table; border-insideH: 1px none #000000; border-insideV: 1px none #000000; cellpadding: 0px 7px 0px 7px;"><TBODY><TR style="height: 37px;"><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 553px; vertical-align: top;" colspan="2"><P style="font-size: 16px;"><SPAN style="font-size: 16px; font-weight: 700;">Figure 3</SPAN><SPAN style="font-size: 16px; font-weight: 700;">. </SPAN><SPAN style="font-size: 16px; font-weight: 700;">A.</SPAN><SPAN style="font-size: 16px;"> The energy value of single aptamer is -12.90kcal/mol.</SPAN></P><P style="font-size: 16px; margin-right: 32px;"><SPAN style="font-size: 16px;">    </SPAN><SPAN style="font-size: 16px; font-weight: 700;">B.</SPAN><SPAN style="font-size: 16px;"> The energy value of the aptamer-RBS1 is -19.20kcal/mol.</SPAN></P></TD></TR></TBODY></TABLE><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">Theophylline binding energy</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[8]</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> is about -9.2kcal/mol</SPAN><SPAN style="font-family: Arial; font-size: 16px;">. The ener</SPAN><SPAN style="font-family: Arial; font-size: 16px;">gy of the aptamer we designed should be</SPAN><SPAN style="font-family: Arial; font-size: 16px;">-12.90kcal/mol</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> ~</SPAN><SPAN style="font-family: Arial; font-size: 16px;">-22.1kcal/mol</SPAN><SPAN style="font-family: Arial; font-size: 16px;">, about -19.20kcal/mol better</SPAN><SPAN style="font-family: Arial; font-size: 16px;">, as principle 4</SPAN><SPAN style="font-family: Arial; font-size: 16px;">. </SPAN><SPAN style="font-family: Arial; font-size: 16px;">Because of the limit of time and ability, our secondary structure had better agree with the one mentioned in the paper, as principle 5</SPAN><SPAN style="font-family: Arial; font-size: 16px;">.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">To make the key and the lock match better, </SPAN><SPAN style="font-family: Arial; font-size: 16px;">the design of the lock should be considered. </SPAN><SPAN style="font-family: Arial; font-size: 16px;">The energy of the cross of crRNA and taRNA is ΔG= -37.31kcal/mol calculated by OligoAnalyzer 3.1</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">[7]</SPAN><SPAN style="font-family: Arial; font-size: 16px;">, the energy difference should be as large as possible, as principle 1.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">Principle 2</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> is based </SPAN><SPAN style="font-family: Arial; font-size: 16px;">on that there are usually 6 bases between RBS and AUG </SPAN><SPAN style="font-family: Arial; font-size: 16px;">on iGEM’s</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> website</SPAN><SPAN style="font-family: Arial; font-size: 16px;">.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">According to the principles above, we finally design the following</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> keys and locks:</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><TABLE style="border: currentColor; border-image: none; margin-right: auto; margin-left: auto; border-collapse: collapse; styleName: Normal Table; border-insideH: 1px solid #FFFFFF; border-insideV: 1px solid #FFFFFF; cellpadding: 0px 7px 0px 7px;"><TBODY><TR class="firstRow" style="background: rgb(68, 114, 196); color: rgb(255, 255, 255); font-weight: 700; vertical-align: top; border-top-color: rgb(255, 255, 255); border-right-color: rgb(255, 255, 255); border-left-color: rgb(255, 255, 255); border-top-width: 1px; border-right-width: 1px; border-left-width: 1px; border-top-style: solid; border-right-style: solid; border-left-style: solid; border-insideH: none; border-insideV: none;"><TD style="padding: 0px 7px; border: 1px solid rgb(255, 255, 255); width: 130px; vertical-align: top;"><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">Name</SPAN></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: rgb(255, 255, 255) rgb(255, 255, 255) rgb(255, 255, 255) currentColor; padding: 0px 7px; width: 423px; vertical-align: top;"><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">Sequence</SPAN></P></TD></TR><TR style="background: rgb(180, 198, 231); vertical-align: top;"><TD style="background: rgb(68, 114, 196); border-width: 1px medium 1px 1px; border-style: solid none solid solid; border-color: rgb(255, 255, 255) currentColor rgb(255, 255, 255) rgb(255, 255, 255); padding: 0px 7px; width: 130px; color: rgb(255, 255, 255); font-weight: 700; vertical-align: top;"><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">Lock:crRNA_RBS</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: currentColor rgb(255, 255, 255) rgb(255, 255, 255) currentColor; padding: 0px 7px; width: 423px; vertical-align: top;"><P style="font-family: Arial; font-size: 16px;color:black">TTT CTT CCT TAG TTC TGC CTA AGG AGG AAA</P></TD></TR><TR style="background: rgb(217, 226, 243); vertical-align: top;"><TD style="background: rgb(68, 114, 196); border-width: 1px medium 1px 1px; border-style: solid none solid solid; border-color: rgb(255, 255, 255) currentColor rgb(255, 255, 255) rgb(255, 255, 255); padding: 0px 7px; width: 130px; color: rgb(255, 255, 255); font-weight: 700; vertical-align: top;"><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">Key:aptamer_taRNA</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: currentColor rgb(255, 255, 255) rgb(255, 255, 255) currentColor; padding: 0px 7px; width: 423px; vertical-align: top;"><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;color:black">GGT GAT ACC AGC ATC GTC TTG ATG CCC TTG GCA GCA CCC TGC GTA AA GAA GGA ATC AAG ACG</SPAN></P></TD></TR></TBODY></TABLE><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">The energy values of mRNA secondary structures are calculated by OligoAnalyzer 3.1 and mfold</SPAN><SPAN style="font-family: Arial; font-size: 16px;">.</SPAN></P><TABLE style="border: currentColor; border-image: none; margin-right: auto; margin-left: auto; border-collapse: collapse; styleName: Normal Table; border-insideH: 1px none #000000; border-insideV: 1px none #000000; cellpadding: 0px 7px 0px 7px;"><TBODY><TR style="height: 115px;"><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 553px; vertical-align: top;"><P style="font-size: 16px;"><SPAN style="font-size: 16px; font-weight: 700;">Figure 4.</SPAN><SPAN style="font-size: 16px; font-weight: 700;">A. </SPAN><SPAN style="font-size: 16px;">The energy of the key is -14.20kcal/mol. The energy of the cross of crRNA and taRNA is ΔG= -32.27kcal/mol calculated by OligoAnalyzer 3.1, which agrees with the principle 1, 2. </SPAN></P><P style="text-indent: 4em; font-size: 16px;"><SPAN style="font-size: 16px; font-weight: 700;">B.</SPAN><SPAN style="font-size: 16px;"> The energy of the lock is -19.20kcal/mol, which agrees with the principle 2, 4, 5.</SPAN></P><P style="text-align: left; font-family: Arial; font-size: 16px;"> </P></TD></TR></TBODY></TABLE><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">We began this work a little late and it was late September that we got the primer and the template.</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> According to the requirement of iGEM, we connect all the biobricks to the pSB1C3 backbone and do </SPAN><SPAN style="font-family: Arial; font-size: 16px;">experiment.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 21px;"><SPAN style="font-family: Arial; font-size: 21px; font-weight: 700;">Experiment</SPAN></P><P style="text-align: right; font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px; font-weight: 700;">-- C</SPAN><SPAN style="font-family: Arial; font-size: 16px; font-weight: 700;">onstruction of our genetic dispatcher</SPAN></P><P style="text-align: right; font-family: Arial; font-size: 21px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">Based on the principle of the circuit design, the RBS in lock should be paid </SPAN><SPAN style="font-family: Arial; font-size: 16px;">attention to for it was not shown on the official website of iGEM! If we used the RBS the iGEM offered to us, we found that it was not suitable for the principle 1, 3 and 5 above. So we wanted to design a new circuit of RBS according to the principles offered by iGEM</SPAN><SPAN style="font-family: Arial; font-size: 16px; valign: sup;">9</SPAN><SPAN style="font-family: Arial; font-size: 16px;">. In order to verify the function of this new RBS, we connected RFP with this kind of RBS. We wanted to know whether this new RBS could translate as the former ones via the method of measuring the intensity of the expression of the fluorescence.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">We used promoter tetR in our project. For your convenience, we named the connection system of lock and RFP </SPAN><SPAN style="font-family: Arial; font-size: 16px;">L2R</SPAN><SPAN style="font-family: Arial; font-size: 16px;">, and the</SPAN><SPAN style="font-family: Arial; font-size: 16px;"> connection system of lock and </SPAN><SPAN style="font-family: Arial; font-size: 16px; font-style: italic;">c</SPAN><SPAN style="font-family: Arial; font-size: 16px; font-style: italic;">heZ </SPAN><SPAN style="font-family: Arial; font-size: 16px;">L2Z</SPAN><SPAN style="font-family: Arial; font-size: 16px;">. What’s more, we named the connection system of the new RBS and RFP</SPAN><SPAN style="font-family: Arial; font-size: 16px;">R2R</SPAN><SPAN style="font-family: Arial; font-size: 16px;">, and the connection system of TT and psB1C3 containing TT </SPAN><SPAN style="font-family: Arial; font-size: 16px; font-style: italic;">K2T</SPAN><SPAN style="font-family: Arial; font-size: 16px;">.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">First we did the experiments of PCR. And the result of the agarose gel electrophoresis was shown as the following:</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><TABLE style="border: currentColor; border-image: none; width: 604px; margin-right: auto; margin-left: auto; border-collapse: collapse; styleName: Normal Table; border-insideH: 1px none #000000; border-insideV: 1px none #000000; cellpadding: 0px 7px 0px 7px;"><TBODY><TR class="firstRow" style="height: 292px;"><TD style="border-width: 1px 1px 1px medium; border-color: rgb(0, 0, 0) rgb(0, 0, 0) rgb(0, 0, 0) currentColor; padding: 0px 7px; width: 302px; vertical-align: top;"><P style="text-align: justify;"><SPAN style="font-size: 14px;">1: 100 bp Marker;</SPAN></P><P style="text-align: justify;"><SPAN style="font-size: 14px;">2: Lock ( restricted by double enzymes) ;</SPAN></P><P style="text-align: justify;"><SPAN style="font-size: 14px;">3: The new RBS(constructed by us);</SPAN></P><P style="text-align: justify;"><SPAN style="font-size: 14px;">4: 500 bp Marker.</SPAN></P><P style="text-align: justify;"> </P><P style="text-align: justify;"><SPAN style="font-size: 14px;">Purpose: The gel electrophoresis was prepared for the ligation of Lock and RBS.</SPAN></P><P style="text-align: justify;"><SPAN style="font-size: 14px;">Results/discussion: T</SPAN><SPAN style="font-size: 14px;">he </SPAN><SPAN style="font-size: 14px;">respective length of Lock and the new RBS are 150 bp, 130 bp. From the length we could know that the verification of PCR was correct. </SPAN></P></TD></TR></TBODY></TABLE><P style="text-align: justify;"> </P><P style="text-align: justify;"> </P><TABLE style="border: currentColor; border-image: none; width: 557px; margin-right: auto; margin-left: auto; border-collapse: collapse; styleName: Normal Table; border-insideH: 1px none #000000; border-insideV: 1px none #000000; cellpadding: 0px 7px 0px 7px;"><TBODY><TR class="firstRow" style="height: 321px;"><TD style="border-width: 1px 1px 1px medium; border-color: rgb(0, 0, 0) rgb(0, 0, 0) rgb(0, 0, 0) currentColor; padding: 0px 7px; width: 236px; vertical-align: top;"><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">1: 500</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">bp Marker</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">2: </SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">key </SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">( E S )</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">(182 bp);</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">3: </SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">key </SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">( E S )</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">(182bp)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">4: BBa_B0015;</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">5: BBa_B0014</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">6: DL2000 Marker</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">.</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"> </P><P style="text-align: left; font-family: Minion Pro;"><SPAN style="font-family: Minion Pro; font-size: 14px;">Purpose: </SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">The gel electrophoresis was pr</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">epared for the ligation of Ptet, aptamer, </SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">taRNA. It is called key. </SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">Results/discussion:</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">The length of the key is 187</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">bp theoretically and from line1 and line2 we could see that the ligation was successful.</SPAN></P></TD></TR></TBODY></TABLE><P style="text-align: justify;"> </P><P style="text-align: justify;"> </P><TABLE style="border: currentColor; border-image: none; margin-right: auto; margin-left: auto; border-collapse: collapse; styleName: Normal Table; border-insideH: 1px none #000000; border-insideV: 1px none #000000; cellpadding: 0px 7px 0px 7px;"><TBODY><TR><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 553px; vertical-align: top;"><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">1: DL2000 Marker</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">; </SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">2: R2C(+2)_1</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">3: R2C(+1)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">; </SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">4: L2C(+1)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">5: L2R(-1)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">6: L2C(+1)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">7: L2R(-1)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">8: L2R-L2R(+1)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">9: L2R(-1)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN></P></TD></TR></TBODY></TABLE><TABLE style="border: currentColor; border-image: none; margin-right: auto; margin-left: auto; border-collapse: collapse; styleName: Normal Table; border-insideH: 1px none #000000; border-insideV: 1px none #000000; cellpadding: 0px 7px 0px 7px;"><TBODY><TR class="firstRow" style="height: 267px;"><TD style="padding: 0px 7px; border: 1px rgb(0, 0, 0); width: 553px; vertical-align: top;"><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">10: 500</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">bp Marker</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">11: DL2000 Marker</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">12: Pter_aptamer_taRNA(-1)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">13: Pter_aptamer_taRNA(+1)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">14:Pter_aptamer_taRNA+BBa_B0015</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">15: Pter_aptamer_taRNA(-2)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">16: Pter_aptamer_taRNA(-1)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">17: BBa_K823000</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">18: BBa_J04650</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">19:BBa_K629003+BBa_B0015</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">;</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">20: 500</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">bp Marker</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">.</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"> </P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">Purpose: </SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">The gel electrophoresis was prepared for the ligation of R2C, L2C and L2R. </SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">Results/discussion:</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">We found that the ligation system of L2R was successfully. What’s more, the backbones of RFG and BBa_K629003+BBa_B0015 was also right. Unfortuna</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">tely, we couldn’t get any </SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">band</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">s of R2Z and R2R in the image.</SPAN></P></TD></TR></TBODY></TABLE><P style="text-align: justify;"> </P><TABLE style="border: currentColor; border-image: none; width: 548px; margin-right: auto; margin-left: auto; border-collapse: collapse; styleName: Normal Table; border-insideH: 1px none #000000; border-insideV: 1px none #000000; cellpadding: 0px 7px 0px 7px;"><TBODY><TR class="firstRow" style="height: 337px;"><TD style="border-width: 1px 1px 1px medium; border-color: rgb(0, 0, 0) rgb(0, 0, 0) rgb(0, 0, 0) currentColor; padding: 0px 7px; width: 246px; vertical-align: top;"><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">1: DL2000 Marker</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">；</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">2: R2R(10.12dh 5a)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">；</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">3: R2R(10.12CL-1)</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">；</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">4</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">:</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">500</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">bp Marker</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">；</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"><SPAN style="font-family: Minion Pro; font-size: 14px;">5</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">: DL5000</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">Marker</SPAN><SPAN style="font-family: Minion Pro; font-size: 14px;">.</SPAN></P><P style="text-align: left; font-family: Minion Pro; font-size: 14px;"> </P><P style="text-align: left;"><SPAN style="font-family: Minion Pro; font-size: 14px;">From the image, we know the length of R2R is 973bp theoretically and the length of what we got matched this length. The ligation was successful.</SPAN></P></TD></TR></TBODY></TABLE><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">In order to justify that the new RBS has an undiscovered function, we used the microplate reader to measure the expression of the fluorescent protein. We diluted the bacteria solution into 1.2, 1.5, 2, 3, 6 times than before. When we measured the OD, we found that the value of the 1.2 time-diluted solution and the 1.5 time-diluted phosphate buffered saline solution were abnormal. After reiterating the experiment, we took the 2 time-diluted solution, 3 time-diluted solution and 6 time-diluted solution as samples to measure the value of the fluorescence. The detailed results were showed as the following:</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><TABLE style="border: currentColor; border-image: none; margin-right: auto; margin-left: auto; border-collapse: collapse; styleName: Normal Table; border-insideH: 1px none #000000; border-insideV: 1px none #000000; cellpadding: 0px 7px 0px 7px;"><TBODY><TR><TD style="border-width: medium 1px 1px; border-color: currentColor rgb(0, 0, 0) rgb(0, 0, 0); padding: 0px 7px; width: 553px; vertical-align: top;"><P style="font-size: 16px;"><SPAN style="font-size: 16px; font-weight: 700;">Fi</SPAN><SPAN style="font-size: 16px; font-weight: 700;">gure 5.</SPAN><SPAN style="font-size: 16px;"> BBa_J61002 is the plasmid backbone that contain RFP while ptetR_RBS_( BBa_J61002) is our project that contain new RBS. From the curve trend, we know that our RBS </SPAN><SPAN style="font-size: 16px;">takes effect to translate protein.</SPAN></P></TD></TR></TBODY></TABLE><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">We hoped to justify whether our new RBS would express via the comparison of the expression of the fluorescent proteins in the backbone with pSB1C3 ligated with RFP and the circuit of ptetR_RBS_RFP. We were so delighted to see that the new RBS really expressed.</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">Because of the limited time, we didn’t finish our project. But we will continue it in the future.</SPAN></P><P>Want to see more applications: <A href="https://2014.igem.org/Team:XMU-China/Project_Application_RBSpromoter" target="_blank">Promoter Yardstick</A>,<A href="https://2014.igem.org/Team:XMU-China/Project_Application_BlackHole" target="_blank">Black Hole</A>, <A href="https://2014.igem.org/Team:XMU-China/Project_Application_OscillationTimer" target="_blank">Oscillation timer</A>.</P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 21px;"><SPAN style="font-family: Arial; font-size: 21px; font-weight: 700;">Reference</SPAN><SPAN style="font-family: Arial; font-size: 21px; font-weight: 700;">s</SPAN></P><P style="font-family: Arial; font-size: 16px;"> </P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">1.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">  Tuerk C, Gold L, Systematic evolution of ligands by exponential enrichment: RNA ligands to bacteriophage T4 DNA polymerase [J]. nature, 1990, 249(4968):505-510</SPAN></P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">2.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">  </SPAN><SPAN style="font-family: Arial; font-size: 16px;">Culler S J, Hoff K G, Smolke C D. Reprogramming cellular behavior with RNA controllers responsive to endogenous proteins[J]. Science, 2010, 330(6008): 1251-1255.</SPAN></P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">3.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">  </SPAN><SPAN style="font-family: Arial; font-size: 16px;">Weigand J E, Suess B. Tetracycline aptamer-controlled regulation of pre-mRNA splicing in yeast[J]. Nucleic acids research, 2007, 35(12): 4179-4185.</SPAN></P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">4.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">  </SPAN><SPAN style="font-family: Arial; font-size: 16px;">Topp S, Gallivan J P. Guiding bacteria with small molecules and RNA[J]. Journal of the American Chemical Society, 2007, 129(21): 6807-6811.</SPAN></P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">5.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">  Farren J Isaasc, Daniel J Dwyer, Ding C, Pervouchine DD, Cantor CR, Collins JJ, et al.Engineered riboregulators enable post-translational control of </SPAN><SPAN style="font-family: Arial; font-size: 16px;">gene expression. Nat Biotechnol. 2004 Jul;22(7):841-7. </SPAN></P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">6.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">  Friedland A E, Lu T K, Wang X, et al. Synthetic gene networks that count[J]. science, 2009, 324(5931): 1199-1202.</SPAN></P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">7.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">  Callura J M, Cantor C R, Collins J J. Genetic switchboard for synthetic biology applications[J]. Proceedings of the National Academy of Sciences, 2012, 109(15): 5850-5855.</SPAN></P><P style="font-family: Arial; font-size: 16px;"><SPAN style="font-family: Arial; font-size: 16px;">8.</SPAN><SPAN style="font-family: Arial; font-size: 16px;">  Lynch S A, Desai S K, Sajja H K, et al. A high-throughput screen for synthetic riboswitches reveals mechanistic insights into their function[J]. Chemistry &amp; biology, 2007, 14(2): 173-184.</SPAN></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:XMU-China/Project_Application_Aptamer">http://2014.igem.org/Team:XMU-China/Project_Application_Aptamer</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:XMU-China/Project_Application_Aptamer" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:XMU-China/Project_Application_Aptamer" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:XMU-China/Project_Application_Aptamer&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:XMU-China/Project_Application_Aptamer&amp;oldid=402597" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>