"
Page
Discussion
View source
History
teams
Log in
 
Team:XMU-China/Project Application Aptamer
From 2014.igem.org
side_bar
Project
Background
Conic Curve
Application
P System
Modelling
Future Work
Interlab
Notebook
Protocol
Lab notes
Calendar
Judging
Parts
Achievements
Team
Members
Attribution
P&P
Overview
Newsletter
Communication
Popularization
Assistance
Acknowledgements
Safety
Background
Conic Curve
Application
P System
Modelling
Future Work
Interlab
APTAMER
 
Overview
 
Aptamer is one kind of oligonucleotide fragments that can specifically combine proteins and small molecules selected by systematic evolution of ligands by exponential enrichment (SELEX). Tuerk and Gold[1] proposed firstly in 1990 that we could obtain a kind of target substance with high specificity to DNA and RNA via the method of application chemistry to synthesize a large-capacity random oligonucleotides library and exerting the optional press combining with the SELEX after several rounds of selection enrichment. Aptamers were applied worldwide in the field of chemistry, biology and medicine.
 
Synthetic biology shows its broad application perspective in the fields of medicine, chemical synthesis, and the production of energy. In recent years, RNA aptamers have been widely used in synthetic biology. We can make full use of the gene circuits to select the aptamers to regulate the circuit expressions. For example, Culler SJ[2] created a modular platform for regulating gene expression by increasing or decreasing alternative splicing based on protein-binding aptamers in 2010. What’s more, aptamers can regulate the cut of mRNA. It’s just like what Weiquand JE[3] did in 2007. He used tetracycline-binding aptamer which can insert close to the 5’splice site (SS) to efficient regulate the splicing of pre-mRNA. Aptamers can also regulate the translation of mRNA. Shana Topp[4] used theophylline aptamer to regulate the protein translation. When theophylline is absence, using RNA aptamer to block ribosome bind site (RBS), leading to target genes can’t express. What these scientists did showed that the diversity of how aptamers regulate the gene circuits. And we are considering using aptamers to regulate the gene circuits based on this theory.
 
 
Design
 
James J Collins[5] group’s work draw our attention. They use crRNA and taRNA to control whether the circuit silence or activate. In 2009[6] and 2012[7], Friedland A E and Callura J M, et al. expand aptamer’s applications. However, one thing to note here is that a promoter must be linked before the corresponding DNA to transcribe them. The promoters mentioned in the papers are mostly primary ones in Part Registry such as pTet, pLlacO-1 and pBAD, etc. In order to enrich the regulatory system, we add an aptamer to the original circuit[5] as a switch. Our new circuit is as following:
 
Sorry, your browser version is outdate.To get a better browsing experience to get a better browsing experience, please upgrade your browser.
Lock and Key
 
At the absence of stimulus, crRNA is paired with RBS and block its expression, thus CheZ cannot be translated and the chemotaxis of bacteria won’t be expressed. Meanwhile, the aptamer are locked by taRNA and all mRNA are in silence. At the presence of small regulatory molecule like theophylline, taRNA as the key is aroused, so is crRNA. Because the binding force between taRNA and crRNA is stronger than that between crRNA and RBS, RBS is exposed and CheZ can be transcribed causing that the flagella rotates counterclockwise and the cells swim.
 
Principles of design[3][4][5][6][7][8]
 
1. The binding fore between taRNA and crRNA is stronger than that between crRNA and RBS or that between aptamer and taRNA.
2. RBS and AUG are separated by about 6 bases.
3. The binding site of taRNA and aptamer is near the site of theophylline and aptamer
4. The order of secondary structures energy: single aptamer < combination of aptamer and taRNA < combination of aptamer and theophylline
5. The conformation of the secondary structure formed by aptamer and taRNA is the same as the one Shana Topp[4] put forward.
 
To achieve the functionality of the aptamer and RBS[4] (aptamer-RBS1 for short) in Figure 1, the position of RBS should be near the binding site of theophylline and aptamer, as principle 3.
 
 
Figure 1. The green part and dark blue part are the aptamer sequence. The binding site of theophylline is marked in dark blue. The linker sequence is marked in gray. RBS is in pink, and AUG is in pale pink.
 
In order to understand the sequence’s meaning, Justin P.Gallivan group[8] set up a bank of 87296 linker sequence and transfered the cells to 96-well microtiterplates, and select the switch by assaying β-galactosidase activity (Figure 2).
 
Figure 2. Model for synthetic riboswitch function. A. Predicted structures and free energies of a portion of the 5’ region clone 8.1 determine by mFold. To represent the free energies of the bound structures, the groups add the experimentally determined free energy of theophylline binding (-9.2kcal/mol) to the free energy of the unliganded structure determiner by mFold since the secondary structure of the Mtct-8-4 aptamer does not change upon ligand binding. B. Kinetic model for synthetic riboswitch function.
 
According to the energy value provided in the paper, we work out the energy of single aptamer[4] and aptamer-RBS1[4] by mFold.
 
 
Figure 3. A. The energy value of single aptamer is -12.90kcal/mol.
        B. The energy value of the aptamer-RBS1 is -19.20kcal/mol.
 
Theophylline binding energy[8] is about -9.2kcal/mol. The energy of the aptamer we designed should be -12.90kcal/mol ~ -22.1kcal/mol, about -19.20kcal/mol better, as principle 4. Because of the limit of time and ability, our secondary structure had better agree with the one mentioned in the paper, as principle 5.
 
To make the key and the lock match better, the design of the lock should be considered. The energy of the cross of crRNA and taRNA is ΔG= -37.31kcal/mol calculated by OligoAnalyzer 3.1[7], the energy difference should be as large as possible, as principle 1.
 
Principle 2 is based on that there are usually 6 bases between RBS and AUG on iGEM’s website.
 
According to the principles above, we finally design the following keys and locks:
 
Name
Sequence
Lock:crRNA_RBS
TTT CTT CCT TAG TTC TGC CTA AGG AGG AAA
Key:aptamer_taRNA
GGT GAT ACC AGC ATC GTC TTG ATG CCC TTG GCA GCA CCC TGC GTA AA GAA GGA ATC AAG ACG
 
The energy values of mRNA secondary structures are calculated by OligoAnalyzer 3.1 and mfold.
Figure 4. A. The energy of the key is -14.20kcal/mol. The energy of the cross of crRNA and taRNA is ΔG= -32.27kcal/mol calculated by OligoAnalyzer 3.1, which agrees with the principle 1, 2.
B. The energy of the lock is -19.20kcal/mol, which agrees with the principle 2, 4, 5.
 
 
 
We began this work a little late and it was late September that we got the primer and the template. According to the requirement of iGEM, we connect all the biobricks to the pSB1C3 backbone and do experiment.
 
Experiment
-- Construction of our genetic dispatcher
 
Based on the principle of the circuit design, the RBS in lock should be paid attention to for it was not shown on the official website of iGEM! If we used the RBS the iGEM offered to us, we found that it was not suitable for the principle 1, 3 and 5 above. So we wanted to design a new circuit of RBS according to the principles offered by iGEM9. In order to verify the function of this new RBS, we connected RFP with this kind of RBS. We wanted to know whether this new RBS could translate as the former ones via the method of measuring the intensity of the expression of the fluorescence.
 
We used promoter tetR in our project. For your convenience, we named the connection system of lock and RFP L2R, and the connection system of lock and cheZ L2Z. What’s more, we named the connection system of the new RBS and RFP R2R, and the connection system of TT and psB1C3 containing TT K2T.
 
First we did the experiments of PCR. And the result of the agarose gel electrophoresis was shown as the following:
 
1: 100 bp Marker;
2: Lock ( restricted by double enzymes) ;
3: The new RBS(constructed by us);
4: 500 bp Marker.
 
Purpose: The gel electrophoresis was prepared for the ligation of Lock and RBS.
Results/discussion: The respective length of Lock and the new RBS are 150 bp, 130 bp. From the length we could know that the verification of PCR was correct.
 
 
1: 500 bp Marker;
2: key ( E S )(182 bp);
3: key ( E S )(182bp);
4: BBa_B0015;
5: BBa_B0014;
6: DL2000 Marker.
 
Purpose: The gel electrophoresis was prepared for the ligation of Ptet, aptamer, taRNA. It is called key. Results/discussion: The length of the key is 187 bp theoretically and from line1 and line2 we could see that the ligation was successful.
 
 
1: DL2000 Marker; 2: R2C(+2)_1; 3: R2C(+1);
4: L2C(+1); 5: L2R(-1); 6: L2C(+1);
7: L2R(-1); 8: L2R-L2R(+1); 9: L2R(-1);
10: 500 bp Marker; 11: DL2000 Marker; 12: Pter_aptamer_taRNA(-1);
13: Pter_aptamer_taRNA(+1); 14:Pter_aptamer_taRNA+BBa_B0015;
15: Pter_aptamer_taRNA(-2); 16: Pter_aptamer_taRNA(-1);
17: BBa_K823000; 18: BBa_J04650;
19:BBa_K629003+BBa_B0015; 20: 500 bp Marker.
 
Purpose: The gel electrophoresis was prepared for the ligation of R2C, L2C and L2R.
Results/discussion: We found that the ligation system of L2R was successfully. What’s more, the backbones of RFG and BBa_K629003+BBa_B0015 was also right. Unfortunately, we couldn’t get any bands of R2Z and R2R in the image.
 
1: DL2000 Marker；
2: R2R(10.12dh 5a)；
3: R2R(10.12CL-1)；
4: 500 bp Marker；
5: DL5000 Marker.
 
From the image, we know the length of R2R is 973bp theoretically and the length of what we got matched this length. The ligation was successful.
 
In order to justify that the new RBS has an undiscovered function, we used the microplate reader to measure the expression of the fluorescent protein. We diluted the bacteria solution into 1.2, 1.5, 2, 3, 6 times than before. When we measured the OD, we found that the value of the 1.2 time-diluted solution and the 1.5 time-diluted phosphate buffered saline solution were abnormal. After reiterating the experiment, we took the 2 time-diluted solution, 3 time-diluted solution and 6 time-diluted solution as samples to measure the value of the fluorescence. The detailed results were showed as the following:
 
Figure 5. BBa_J61002 is the plasmid backbone that contain RFP while ptetR_RBS_( BBa_J61002) is our project that contain new RBS. From the curve trend, we know that our RBS takes effect to translate protein.
 
We hoped to justify whether our new RBS would express via the comparison of the expression of the fluorescent proteins in the backbone with pSB1C3 ligated with RFP and the circuit of ptetR_RBS_RFP. We were so delighted to see that the new RBS really expressed.
 
Because of the limited time, we didn’t finish our project. But we will continue it in the future.
Want to see more applications: Promoter Yardstick , Black Hole , Oscillation timer .
 
 
References
 
1.    Tuerk C, Gold L, Systematic evolution of ligands by exponential enrichment: RNA ligands to bacteriophage T4 DNA polymerase [J]. nature, 1990, 249(4968):505-510
2.    Culler S J, Hoff K G, Smolke C D. Reprogramming cellular behavior with RNA controllers responsive to endogenous proteins[J]. Science, 2010, 330(6008): 1251-1255.
3.    Weigand J E, Suess B. Tetracycline aptamer-controlled regulation of pre-mRNA splicing in yeast[J]. Nucleic acids research, 2007, 35(12): 4179-4185.
4.    Topp S, Gallivan J P. Guiding bacteria with small molecules and RNA[J]. Journal of the American Chemical Society, 2007, 129(21): 6807-6811.
5.    Farren J Isaasc, Daniel J Dwyer, Ding C, Pervouchine DD, Cantor CR, Collins JJ, et al.Engineered riboregulators enable post-translational control of gene expression. Nat Biotechnol. 2004 Jul;22(7):841-7.
6.    Friedland A E, Lu T K, Wang X, et al. Synthetic gene networks that count[J]. science, 2009, 324(5931): 1199-1202.
7.    Callura J M, Cantor C R, Collins J J. Genetic switchboard for synthetic biology applications[J]. Proceedings of the National Academy of Sciences, 2012, 109(15): 5850-5855.
8.    Lynch S A, Desai S K, Sajja H K, et al. A high-throughput screen for synthetic riboswitches reveals mechanistic insights into their function[J]. Chemistry & biology, 2007, 14(2): 173-184.
Retrieved from " http://2014.igem.org/Team:XMU-China/Project_Application_Aptamer "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
