<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UT-Dallas_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UT-Dallas/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UT-Dallas/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UT-Dallas/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UT-Dallas/Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UT-Dallas/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UT-Dallas/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P>igem14
</P><H2><SPAN class="mw-headline" id=".3D.3D">==</SPAN></H2><DIV id="content_body"><SECTION id="titlechart"><DIV id="top_bar"><H2><A href="https://2014.igem.org/Team:UT-Dallas">UT Dallas iGEM</A></H2><UL id="navlist"><LI><A href="https://2014.igem.org/Team:UT-Dallas/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:UT-Dallas/Project/history">History</A></LI><LI><A href="https://2014.igem.org/Team:UT-Dallas/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:UT-Dallas/Project/details">Details</A></LI><LI><A href="https://2014.igem.org/Team:UT-Dallas/Project/results">Results</A></LI><LI><A href="https://2014.igem.org/Team:UT-Dallas/Parts">Parts   </A></LI><LI><A href="https://2014.igem.org/Team:UT-Dallas/Notebook">Notebook</A></LI></UL><LI><A href="https://2014.igem.org/Team:UT-Dallas#team">Team</A></LI><LI><A href="https://2014.igem.org/Team:UT-Dallas/Human-Practices">Human Practice</A></LI><LI><A href="https://2014.igem.org/Team:UT-Dallas/Attributions">Acknowledgement</A></LI><LI><A href="https://2014.igem.org/Team:UT-Dallas/Safety">Safety</A></LI></DIV><DIV class="page_content"><H2>MODELING</H2><P style="display:block">
						We utilized the CRISPR/Cas system with gRNA engineered to recognize genes from infectious bacteria using bacterial 
						specific phages delivery system. Specifically, we target Cholera as a proof of principle. To target the Choleras genome, 
						our team focus on (1) cutting efficiency of the gRNA-CRISPR/Cas9 system and (2) the delivering efficiency of the phage 
						delivery system.
					
					
					There are two important features we have to look at:
						<OL type="1"><LI>On the Intracellular level, in a single Cholera cell, the CRISPR/Cas9-gRNA system is transcript 
							and translated. The complex then cleaves the Cholera native genome, leading to cell death. 
							</LI><LI>On the population level, the phagemid delivers the CRISPR/Cas9-gRNA plasmid from the probiotics 
							to Cholera population.
						</LI></OL>Our Mathematical Model and computer simulation provide a great way to describe the functioning and operation
					of our system. Enjoy!
					<B>LIST OF CONTENTS:</B><UL><LI>INTRACELLULAR LEVEL</LI><OL type="1"><LI>Design
							</LI><LI>Model
							</LI><LI>Result
								</LI><LI>Cas9 On-stage
									</LI><LI>Cas9 Off-stage
								</LI></OL><LI>POPULATION LEVEL</LI><OL type="1"><LI>Design
							</LI><LI>Simulation
							</LI><LI>Result
						</LI></OL><LI>CODE</LI></UL></P><H2>CELLULAR LEVEL</H2><P style="display:block">The model focus on simulating our experiment, with the decrease YFP signal observed as a signal of cell death. 
					Each of the components in our system is transcript with a separate promoter. Our further goal with the model
					is to determine the more efficient promoter by varying transcription rate.
					</P><H2>CELLULAR LEVEL DESIGN</H2><P style="display:block"> This part describes the design of the model.
					 We utilize MATLAB and SimBiology to build this intracellular modeling system. 
					<A href="https://2014.igem.org/Team:UT-Dallas/Project/details">Please refer to our project details page for details
					on the components of the system.</A> This is the SimBiology Diagram of our 3 main species of interest. From left to right:
					<UL><LI>The Therapeutic Module with gRNA and Cas9
						</LI><LI>The Reporter Module with YFP
						</LI><LI>The Inducer (Doxycycline and Tet Repressor Protein)
					</LI></UL></P><H2>MODEL</H2><P style="display:block"> 
					A more mathematical view on our system. Depending perspectives, this text can be seen as ‘very extensive’ 
					or ‘very short and preliminary’, but please decide for yourself. Enjoy!
					We utilize MATLAB and SimBiology to build this intracellular modeling system. 
					The equations we use follow the Law of Mass Action, except that the transcription of YFP mRNA inhibited by 
					CRISPR_Complex follows a first order Hill Function. 
					 The Equations that used Law of Mass Action:
					 The Inhibition Equations:
					The Parameters:
					<I><FONT size="2">Parameters α1,2,3,4,7 are parameters taken from the Bleris Lab, the others are educated estimation.</FONT></I></P><H2>MODEL RESULT</H2><P style="display:block"><B>Figure 1. Intracellular Model with large amount of Dox and active CRISPR system from minute zero-th to 1400th</B>(a) Cas9 is continuously being transcript, thus a logarithmic growth of mRNA molecules of Cas9 
						(blue line, mRNA-Cas9) is observed. Counterbalanced by natural degradation rate, the total count of 
						mRNA-Cas9 reaches a stable population and switches to a plateau.
						Cas9 Protein (green line, pro-cas9) total count is keep at zero, due to immediate conversion into CRISPR complex.
						(b) gRNA (blue line) total count grows logarithmically and reaches a plateau due to natural degradation.
						The size of gRNA is 7 times smaller than Cas9, thus it is transcript much faster than Cas9. Thus the CRISPR 
						complex (green line, Cas9-gRNA) is much lower in comparison with the blue line.
						(c) This figure shows the huge starting amount of Dox (red line, Dox) for this simulation 10e11. Dox is not 
						regenerated and only goes down due to natural degradation and binding with TetR protein (green line, pr-tetR) to 
						form Dox-TetR Complex (cyan line, Dox-tetR). All the other amount are too low in relation to Dox, thus harder to 
						observe in the figure. 
						(Go <A ref="https://static.igem.org/mediawiki/2014/0/03/UTDallas_Figure1SCALE_DOWN.jpg"> here </A>for a better look at mRNA of TetR, TetR protein, and Dox-TetR Complex)(d) Yellow Fluorescence Protein (green line, pr-YFP) go through a logarithmic growth, reaching a peak, 
						and then degrades. Total count molecules mRNA of Yellow Fluorescence Protein (blue line, mRNA-YFP) observes a 
						much higher logarithmic growth and then degrades really fast as the transcription is stopped by the YFP-DNA cleaving
						of CRISPR Complex.
					</P></DIV></SECTION><SECTION class="demo"><DIV class="spacer s1"><DIV class="page_content"><H2>CAS9 INACTIVE</H2><P style="display:block"><B>Figure 2. Intracellular Model without Dox and Inactive CRISPR system from minute zero-th to 1400th</B>(a) Cas9 is transcript with a rapid logarithmic growth of total free mRNA molecules of Cas9 
						(blue line, mRNA-Cas9) then quickly declines into logarithmic death phase. Cas9 Protein (green line, 
						pro-cas9) total count is keep at zero as no mRNA is available for translation. 
						(b) gRNA (blue line) total count grows logarithmically and reaches a plateau due to natural 
						degradation. The size of gRNA is 7 times smaller than Cas9, thus it is transcript much faster than
						Cas9. Thus the CRISPR complex (green line, Cas9-gRNA) is much lower in comparison with the blue line. 
						No formation of Cas9 Protein implies no binding of Cas9 and gRNA, implies no formation of CRISPR complex.
						(c) No Dox (red line, Dox) is added in this simulation. Thus TetR protein (green line, pr-tetR) is 
						not binded with dox to form Dox-TetR Complex (cyan line, Dox-tetR), therefore Dox-TetR Complex is always
						zero. TetR protein and mRNA of TetR total count grow logarithmically and reach the plateau phase due to 
						natural degradation.  
						(d) Without the effect of CRISPR Complex, Yellow Fluorescence Protein (green line, pr-YFP) and
						molecules mRNA of Yellow Fluorescence Protein (blue line, mRNA-YFP)  grow logarithmically and reach the 
						plateau phase due to natural degradation.
					</P></DIV></DIV><DIV class="spacer s1"><DIV class="page_content"><H2>MODEL 3</H2><P style="display:block"><B>Figure 3. Intracellular Model with a small amount of Dox and Inactive CRISPR system from minute zero-th to 1400th</B>We are mainly interested in figure 3.(c) which shows the interaction between mRNA of TetR, Protein TetR,
						binding Dox molecules, and Dox-TetR complex.
						Refer to Figure 1 for Figure 3.(a), 3.(b), and 3.(d) interpretation.
						(c) Dox (red line, Dox) goes down from a moderate amount of 900 to zero at time step ~320. TetR protein 
						(green line, pr-tetR) amount from time step 0 to ~320 is zero due to binding with dox to form Dox-TetR Complex 
						(cyan line, Dox-tetR), which increases. 
						From time step ~321 to 1400, Dox is zero, therefore Dox-TetR Complex changes into degradation phase as it is no 
						longer being produced (no Dox to bind with TetR). Free TetR proteins start to increase from time step ~321.
						Without effect on TetR transcription, mRNA of TetR (blue line, mRNA-tetR) total count grow logarithmically and 
						reach the plateau phase due to natural degradation. 
						</P></DIV></DIV><DIV class="spacer s2"><DIV class="page_content"><H2>POPULATION MODEL</H2><P style="display:block">
						This simulation provides a simple and intuitive visualization of the effects of the probiotics onto the pathogenic
						Choleras population.
						Initially, we plan to test on Phagemid M13 and then CTXphi, which is more specific to Cholerae. 
						(Please refer to our experiment design page). Our lab had ordered Cholerae for this experiment. However, 
						the limited time did not allow us to test this. After Wiki Freeze, we still expect to continue the experiment. 
						Nevertheless, this is where this population simulation comes in and shows the potential application of 
						our bio-system.
						</P><H2>POPULATION LEVEL DESIGN</H2><P style="display:block">This part describes the design of the simulation, how we come up with it and its relation to our 
							wet lab experiment.
							Our design starts with a relatively stable probiotics population (green) and a growing pathogenic
							Cholerae population (black). The probiotics should be able to sense the presence of Cholerae with a 
							sensing mechanism. (We discussed with Colombia iGEM Team on collaboration on this subject, as they were 
							working on the sensing specificity for Cholerae. We invite you to visit their wiki and check out their 
							project.)
							Ultimately, we design the probiotics with our genetically modified plasmid with gRNA-Cas9 system 
							to be a stable non-pathogenic colony growing in harmony with the human intestinal flora.
							<UL><LI>[mode 0] In the absence of Cholerae, the probiotics would not release phages.
								</LI><LI>[mode 1] In the presence of Cholerae, the probiotics would release phages with the CRISPR system plasmid.
							</LI></UL>The molecules released by Cholerae induces the production of phages (the build up of phage capsid protein), 
							the packaging of our CRISPR system plasmid, and the release of such phages.Once the phages get in contact with 
							Cholerae cells, the plasmid is released into the Cholerae cytosol. Thus, the cleaving system goes to work and 
							the bad Cholerae die! (To know more on our system mechanism, please visit our project mechanism page.) Once the 
							Cholerae and its presence-sensing-signal died out, the probiotics return back to [mode 0] and stop releasing 
							phages.
							</P></DIV></DIV><DIV class="spacer s2"><DIV class="page_content"><H2>POPULATION LEVEL SIMULATION</H2><P style="display:block">We use GRO - The cell programming language, a shareware developed by <A ref="http://depts.washington.edu/soslab/gro/">
							The Klavins Lab at the University of Washington. </A><UL><LI>Green population := the probiotics (at time 0, population = 36)
									</LI><LI>Black population := the pathogens (V. Cholerae) (throughout the simulation, population = 36)
									</LI><LI>Blue signal := signal emitted by the pathogens, indicating its presence (at time 0, population = 15)
									</LI><LI>Pink signal := the phages emitted by the probiotics, carrying the CRISPR plasmid
								</LI></UL></P><H2>POPULATION LEVEL RESULT</H2><P style="display:block">The delay in signal disappearance indicates the significance of choosing appropriate phages 
						for the delivery system with appropriate speed, effectiveness, specificity, and degradation rate.
						</P></DIV></DIV></SECTION></DIV></DIV><H2>CODE</H2><DIV style="height:600px;width:600px;border:1px solid #ccc;font:16px/26px Georgia, Garamond, Serif;overflow-y:scroll;overflow-x:scroll;"><DIV class="page_content"><H2>POPULATION MODEL</H2><P style="width:100%;">
https://static.igem.org/mediawiki/2014/7/72/IGEM_intracell_wiki_final.m
https://static.igem.org/mediawiki/2014/3/31/GRO-Population-Model.txt
		</P></DIV>

====



<DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UT-Dallas/Modeling">http://2014.igem.org/Team:UT-Dallas/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UT-Dallas/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UT-Dallas/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UT-Dallas/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UT-Dallas/Modeling&amp;oldid=403797" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>