<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Kent_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Kent/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbox"><NAV id="navdiv"><UL><LI><A href="#">Project</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Results">Results</A></LI></UL><LI><A href="#">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Basic_Part">Basic Parts</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Notebook">Logbook</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Improve">Future Ideas</A></LI><LI><A href="#">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Safety">Project Safety</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Team">Meet the Team</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Attributions">Attribution</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Kent/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Kent/HP/Gold_Integrated">Gold</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Kent/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Kent/Collaborations">Collaboration</A></LI></NAV></DIV><DIV id="title"><SPAN>Experiments &amp; Protocols </SPAN></DIV><NAV class="droptext arrows"><HEADER class="hull"><LABEL for="acc-close" class="hull-title">Basic Protocols</LABEL></HEADER><SECTION class="hull"><LABEL class="hull-title" for="cb1">Production of Lysogeny broth (LB)</LABEL><DIV class="hull-content">For 1 litre of LB a mixture of 10g of sodium chloride, 10g peptone, 5g of yeast extract as well as 1
litre of distilled water in a glass bottle. We then used a magnetic spinner to help mix the powders
with the water, we avoided shaking the glass bottle as it would cause froth and waste some of the
LB.

When making the LB we also made another litre batch and added 15g of agar extract to be able to
grow bacteria on plates.</DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb2">Production of SOB medium and magnesium stock</LABEL><DIV class="hull-content">Bringing together 20g of tryptone, 5g of yeast extract, 0.584g of NaCl, 0.186g of KCl and mixing it
with 990ml of millipure water (using the magnetic mixer again) which was then put in to autoclave
to sterilise it, after it was taken out and let for it to cool down to below 60 o C.

10ml of 2M Mg 2+ stock was then added and then brought to 100ml with millipure water, 0.2mm
filter sterilize was then used</DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb3">Production of SOC medium and glucose stock</LABEL><DIV class="hull-content">Once again bring 20g of tryptone, 5g of yeast of extract, 0.584g of NaCl, 0.186g of KCL, and then
bring 970 ml with millipure water and use the magnetic mixer once again, this was also then put in
to autoclave.

10ml of 2M Mg 2+ stock and then bring it to 100ml with milllipure water, filter sterilize it with 0.2m
and then final add 20ml of 1M glucose stock.</DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb4">Production of Glycerol stock</LABEL><DIV class="hull-content">If you wish to store bacteria long term, you will need to create a Glycerol Stock after
inoculating an overnight liquid culture
<UL><LI>Once bacterial growth has been achieved, 500μL of the overnight liquid
culture needs to be added to 500μL of 50% glycerol in a 2mL tube where it
should be gently mixed</LI><LI>The glycerol stock should then be frozen at -80 o C</LI><LI> Successive freeze and thaw cycles will reduce the stocks shelf life</LI></UL></DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb5">Running Agarose Gel</LABEL><DIV class="hull-content">After the cells have been miniprepped and the plasmid put through a restriction digest, the agarose gel can be run.
<UL><LI>Make up some agarose. This is done by taking 0.5g of agarose powder and putting it in a
250ml sterile conical flask, with 50ml of TAE buffer, then microwaving it in small pulses (20
seconds then swirling it around) until it is dissolved. Don’t overheat it or it will evaporate too
much. Make up the evaporated volume to 50ml with distilled water.</LI><LI>Add 1 vial of cybersafe (ask technical services for a tube of it and add all of it)</LI><LI>Line the white sides of the tank with the agarose solution, to seal it and prevent leakage. Use
a p1000 pipette set to 1ml. Let it dry (about 5 mins max)</LI><LI>Then pour all the agarose/sybrsafe solution into the tank and put in the comb. Let it set and
solidify (maximum 30 mins)</LI><LI>When the gel has set, remove the comb from the tank (gently!) and then cover the whole
tank with TAE buffer, so there’s at least half a centimetre of TAE covering the gel.</LI><LI>Now, the samples need to be loaded. Load some DNA markers (ask technical services for a
tube of this and load the whole tube) into well 1( left hand side) and then choose what you
load into wells 2, 3, and 4 etc. (make sure you note what’s in each lane!)</LI><LI>Load all of your digests into the wells 2,3, and 4.</LI><LI>Plug into a power supply and put the cover on. Run for 40 mins to an hour at 80v. The amps
don’t matter.</LI><LI>Once the visible markers have reached the half way point of the tank, turn off the power
supply and drain the TAE buffer form the tank. Remove the gel with a spatula and place in a
UV imaging box. Take an image of the gel under UV light, save it onto a USB stick.</LI></UL></DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb6">Overnights protocol</LABEL><DIV class="hull-content">After a transformation has been run and plates have been streaked and patched, overnight cultures will need to be made:
<UL><LI>Add 10mL of LB broth (not agar) into as many autoclaved conical flasks as
needed</LI><LI>Add 10uL of Chloramphenicol into each conical flask as well</LI><LI>Using a pipette tip, scrape up some of the cell colonies on the agar plates
prepared beforehand and drop it into the conical flask</LI><LI>Cover up the flask using aluminum foil</LI><LI>Incubate the cultures at 37oC and 180 rpm</LI></UL></DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb7">Protocol for transformation/ heat shock</LABEL><DIV class="hull-content">
This requires chemically competent cells to be made beforehand. These must be stored at -80
degrees Celsius:
<UL><LI>First, the competent cells and the plasmid intended for transformation must be thawed on
ice. Additionally, some 1.5ml Eppendorf tubes should be chilled on ice, along with some
pipette tips.</LI><LI>100ul of the chemically competent cells are then pipetted (using the chilled pipette tip) into
a chilled Eppendorf tube.</LI><LI>5ul of DNA is the pipetted into the tube with a chilled tip. This tube is then stored on ice for
30 minutes.</LI><LI>The tube is then placed in a water bath at 42 degrees Celsius for precisely 90 seconds. After
90 seconds is up, the tube is transferred back to ice for 2 minutes.</LI><LI>900ul SOC medium is then added into the tube (with a normal pipette tip, doesn’t need to
be chilled) and then mix very gently by pipetting up and down inside the tube.</LI><LI>The tube is then incubated at 37 degrees Celsius for 45 minutes. The tube should not be
shaken at all at this point.</LI><LI>100ul of the transformation mix is then pipetted into the centre of a plate containing LB agar
and the appropriate antibiotic (for example, for plasmid pSB1C3, Chlorophenicol should be
used, and for pSB1A3, use Ampicillin). Use 1ul of antibiotic for each ml of agar.</LI><LI>In sterile conditions (Bunsen burner, gloves cleaned with IMS (ALLOWED TO DRY)), spread
the bacteria around the plate by keeping the lid as closed as possible and inserting the
spreader then turning the plate around to spread the cells. Then immediately close and
store upside down.</LI><LI>The remained of the transformation mix is then spun at the highest possible speed for 2
minutes. The resulting pellet is then resuspended in 100ul of the existing medium and plated
onto the LB and antibiotic plate.</LI><LI>Incubate in a 37-degree Celsius incubator for 16-18 hours.</LI><LI>Any colonies that result from this should be plated on a patch plate.</LI><LI>This is done by taking a plate of LB agar with the appropriate antibiotic and dividing it up
into sections by drawing a grid on the bottom. These sections are numbered and then using
a sterile pipette tip, the colonies are gently streaked in each section- 1 colony per section.</LI></UL></DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb8">Transformation Guidelines (QuickChange Protocol)</LABEL><DIV class="hull-content">
Store all XL1-Blue supercompetent cells at -80 o C (prevents loss of efficiency) as they are
sensitive to the smallest of temperature variations, even transferring tubes from one freezer
to another will result in loss of efficiency

Storage Conditions:
<UL><LI>XL1-Blue supercompetent cells should be stored at the bottom of a -80 o C freezer</LI><LI>They should be placed at -80 o C directly from dry ice shipping container</LI>
Aliquoting Cells
<LI>Keep the XL1-Blue supercompetent cells on ice at all times</LI><LI>Essential that BD-Falcon polypropylene tubes are places on ice before cells are
thawed</LI><LI>Cells must be aliquoted directly into prechilled tubes (Use of 14-mL BD Falcon Polypropylene Round-Bottom Tubes)</LI><LI>These tubes must be used (BD Biosciences Catalog #352059) for the transformation
protocol</LI><LI>The heat-pulse steps’ duration is critical and has been optimized for the thickness as
well as shape of these tubes</LI>
Length of Heat Pulse
<LI>Optimal efficiencies observed when cells are heat pulsed for 45 seconds</LI><LI>Heat pulsing for at least 45 seconds is recommended, allowing for slight variations
in incubation length</LI><LI>Efficiencies noted to decrease sharply when pulsed for &lt;30 seconds or &gt;45 seconds</LI><LI>This defined window of highest efficiency for the XL1-Blue cells results from heat
pulse in step 3 of transformation protocol</LI>
Preparing Agar Plates for Colour Screening
<LI>To the LB agar, add </LI><LI>80 µg/ml of 5-bromo- 4-chloro- 3-indolyl- β-D-galactopyranoside (X-gal)</LI><LI>20 mM isopropy-1- thio β-D galactopyranoside (IPTG)</LI><LI>Appropriate antibiotic</LI></UL><LI>These are all added to prepare the LB agar plates for blue-white colour screening </LI><LI>Alternatively </LI><LI>100 μl of 10 mM IPTG and 100 μl of 2% X-gal can be spread on LB agar
plates 30 minutes prior to plating transformations</LI><LI>The IPTG must be prepared in sterile dH2O </LI><LI>The X-gal must be prepared in dimethylformamide (DMF) </LI><LI>IPTG and X-gal MUST NOT be mixed before being pipetted onto the plates since the
chemicals may precipitate.</LI></DIV></SECTION></NAV><NAV class="droptext arrows"><HEADER class="hull"><LABEL for="acc-close" class="hull-title">Interlab Protocols</LABEL></HEADER><SECTION class="hull"><LABEL class="hull-title" for="cb10">Calibration of OD 600 Reference Point</LABEL><DIV class="hull-content">
LUDOX-S40 must be used as a single point reference with the aim to attain a
ratiometric conversion factor, which in turn will be used to transform absorbance
data into standard OD 600 measurement.

Standard 1 cm path length spectrophotometer readings are instrument dependent,
while plate readers possess a path length less than 1 cm and are volume dependent.

Therefore, in this situation, there are 2 key objectives:
<UL><LI>Ratiometric conversion to transform Abs 600 measurements into OD 600
measurements</LI><LI> Accounting for instrument differences</LI></UL>
Before starting the protocol, path length correction must be switched off. This is
because scattering increases with longer wavelengths therefore adjustment is
confounded by scattering solutions such as dense cells. However, many plate
readers have automatic path length correction which is based on volume
adjustment that uses ratio of absorbance measurements at 900 + 950 nm.
LUDOX solution is only weakly scattering so will produce low absorbance values

Use same cuvettes, plates and volumes that are going to be used in cell based assays

Materials:
<UL><LI>1 mL 100% LUDOX</LI><LI>H 2 O</LI><LI>96 well plate (black with flat, transparent/clear bottom)</LI></UL>
Method
<UL><LI>100 µl of LUDOX should be added into wells A1, B1, C1 and D2 (or 1mL into
cuvette)</LI><LI>100 µl of H 2 O should be added into wells A2, B2, C2 and D2</LI><LI>Measure absorbance 600nm of all samples in all standard measurement modes in instrument</LI><LI>Record data</LI></UL></DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb11">Production of Fluorescein stock solution</LABEL><DIV class="hull-content"><UL><LI> Spin down the Fluorescein stock tube and ensure the pellet is at the tubes' bottom </LI><LI>Prepare 2x fluorescein stock solution (100 µM)</LI><LI>Resuspend Fluorescein in 1mL 1xPBS</LI><LI>Ensure Fluorescein is properly dissolved
After the resuspension, pipette up and down and examine the solution in
the tip (if particulates are visisble, continue to mix solution until they
disappear)</LI></UL><LI>Dilute the 2x Fluorescein stock solution</LI><LI> With 1xPBS to make 1x fluorescein solution</LI><LI>With resulting concentration of fluorescein stock solution 50 µM
(500 µL of 2x fluorescein in 500 µL 1x PBS to make 1 mL of 50 µM (1x)
fluorescein solution)</LI></DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb12">Fluorescein Fluorescence Standard Curve</LABEL><DIV class="hull-content">
A dilution series of Fluorescein in 4 replicates must be prepared where the
fluorescence is measured in a 96 well plate in standard mode on a plate reader. A
standard curve will be generated of fluorescence of fluorescein concentration. This
will be used to correct cell based readings to an equivalent fluorescein
concentration, which will then be converted into a GFP concentration.

Materials
<UL><LI>Fluorescein</LI><LI>10mL 1xPBS (Phosphate Buffered Saline)</LI><LI>96 well plate (black with flat, transparent/clear bottom)</LI></UL>
Method
Serial dilutions need to be performed across columns 1-11
Column 12 must contain PBS buffer only

The plate will initially be setup fluorescein stock in column 1 and equal volume of1xPBS in columns 2-12
<UL><LI> Add 100 µL of PBS into wells A2-A12, B2-B12, C2-C12 and D2-D12</LI><LI>Add 200 µL of Fluorescein 1x stock solution into A1, B1, C1 and D1</LI><LI>Transfer 100 µL of Fluorescein stock solution from A1 into A2</LI><LI>Mix A2 by pipetting up and down 3x and transfer 100 µL into A3
Repeat the process for A3 into A4, A4 into A5, etc. until A11</LI><LI>Mix A11 by pipetting up and down 3x and transfer 100 µL into liquid waste</LI><LI>Repeat dilution series for rows B, C and D</LI><LI>Measure fluorescence of all samples in all standard measurement modes in
instrument</LI><LI>Record the data</LI></UL>
Measurement notes
<UL><LI>The plates can now be measured in the plate reader</LI><LI>Standard GFP settings must be used (same as those used when measuring the
cells):</LI><LI>Excitation 485nm
</LI><LI>Emission 530/30
</LI><LI>Turn off path length correction</LI></UL><LI>Would be ideal to repeat measurements with different settings
</LI><LI>Generates series of standard curves to choose from</LI><LI>Use number of settings that affect sensitivity (gain and/or slit width)
</LI><LI>Also consider orbital averaging, top/bottom optics</LI></DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb13">Cell Measurement Protocol</LABEL><DIV class="hull-content">
The calibration measurements should be performed before the measurements on the cells are performed. This allows that the measurement process is understood
and that the cell measurements are taken under the same conditions.

Materials
<UL><LI>Competent cells (E.coli strain DH5-alpha)</LI><LI>LB (Luria Bertani) media</LI><LI>Chloramphenicol (stock concentration 25 mg/mL dissolved in EtOH –
working stock 25 ug/mL)</LI><LI>50 mL Falcon tube (covered in foil to block light)</LI><LI>Incubator at 37oC</LI><LI>1.5mL Eppendorf tubes for sample storage</LI><LI>Ice bucket</LI><LI>Pipettes</LI><LI>96 well plate (black with flat, transparent/clear bottom)</LI>************??????
</UL></DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb14">Calcium Chloride Competent Cells</LABEL><DIV class="hull-content">
Prior Preparation
<UL><LI>Autoclave 50mM Calcium Chloride and keep it cold at about 4 o C</LI><LI>For the starter cultures</LI><LI>Add a colony of E.coli DH5cells to 5mL of LB</LI><LI>Incubate at 37 o C overnight</LI></UL>
Method:<UL><LI> Keep cells on ice at all times where possible</LI><LI> To 100mLs of LB, add 100uL of cells from the overnight culture</LI><LI> Let it grow at 37 o C and 250 rpm (until it reaches OD 600 ~0.6-0.8)</LI><LI> Place cells on ice immediately to cool them once the correct OD 600 has been
reached</LI><LI>Centrifuge at max speed for 10 mins and 4 o C</LI><LI>Discard supernatant</LI><LI>Resuspend the pellet in 50% of the original volume with ice-cold 50mM CaCl 2; In a 5omL culture, add 25mL CaCl 2</LI><LI>Allow them to sit on ice for 30 mins</LI><LI>Centrifuge at max speed for 10 mins at 4 o C</LI><LI>Discard the supernatant</LI></UL>
Preparation of Competent Cells for Storage

Materials
<UL><LI>Cell Line</LI><LI>Sterile LB</LI><LI>10mM sterile and chilled Calcium Chloride</LI><LI>Dry ice</LI><LI>Acetone</LI></UL>
Method
<UL><LI>Inoculate the cells (either 1:50 or 1:100) into 50mL of LB</LI><LI>Grow them at 37 o C until OD600 is around 0.4-0.5</LI><LI>Place on ice for 10 minutes while Falcon tubes are pre-chilled</LI><LI>The cells should be harvested at 3000 rpm, 4C for 8 minutes</LI><LI>The pellet then needs to be resuspended in 1mL of 100mM CaCl 2 and 30%
(v/v) glycerol</LI><LI>The resulting solution needs to be aliquoted into chilled Eppendorf tubes
(100uL per tube)</LI><LI>Place each Eppendorf tube into an acetone dry ice bath to snap freeze them</LI><LI>Then store at -80 o C</LI></UL></DIV></SECTION></NAV><NAV class="droptext arrows"><HEADER class="hull"><LABEL for="acc-close" class="hull-title">Complex Protocols</LABEL></HEADER><SECTION class="hull"><LABEL class="hull-title" for="cb15">DNA Miniprep Kit (Qiagen)</LABEL><DIV class="hull-content">
Method: (passive + our)
<UL><LI>2 x 5 mL of our ampicillin resistant bacteria, containing the plasmid of interest and grown
overnight on LB medium, are centrifuged in falcon tubes at 4500 rpm for 6 minutes.</LI><LI>The supernatant is removed and the pelleted bacteria are resuspended in 250μL of P1 buffer
(containing 100 μg/mL RNase A). Thoroughly mix/ Vortex mix of the samples is required to
ensure full resuspension. The samples are transferred into Eppendorf tubes.</LI><LI>250 μL of P2 buffer is added to each sample and gently mixed by inverting the tube ca. 10 times.
This lysis reaction should not exceed 5 minutes.</LI><LI>350 μL of N3 buffer is pipetted to each sample, and gently but thoroughly mixed by inverting the
tube ca. 10 times. The samples are then centrifuged in a table top centrifuge at 13.000 rpm for
10 minutes.</LI><LI>The supernatant contains our plasmid of interest, while the white pellet is cell debris. 800 μl of
the supernatant are pipetted into Qiagen Spin Columns.</LI><LI>The columns are centrifuged for 60 seconds. The plasmids are retained in a silica mesh, while
remaining substances flow through the column into a collection tube.</LI><LI>The column is washed with 500 μL of PB buffer and centrifuged (13.000 rpm for 60 sec) to
remove any remaining nucleases which could interfere with further processing of the plasmids.</LI><LI>750 μL of PE buffer is added to each sample and centrifuged for 60 seconds to remove any
remaining wash buffer. The flow through is discarded and the spin column is placed into a fresh
Eppendorf tube.</LI><LI>To elute the bound plasmid DNA, 50 μL of EB buffer is added to the column. After letting the
samples stand for ca. 2 minutes, each tube is centrifuged at high speed (13.000 rpm) for 60
seconds.</LI><LI>The spin column is discarded, the Eppendorf tubes now contain our desired plasmid DNA.</LI></UL></DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb16">Enzyme Digest Protocol</LABEL><DIV class="hull-content">
A restriction enzyme digestion is usually performed in a volume of 20μL with 0.2-1.5μg of substrate DNA and two-to tenfold excess of enzyme.

If a large volume of DNA or enzyme is used, abnormal results may occur

When pipetting the samples in the different lanes of the gel, the enzyme componentof the tube needs to make up 1μL.

Method:
1. The 5 lanes of the gel are as follows<UL><LI>Marker</LI><LI>Control (with no cutting enzyme)</LI><LI>1μL EcoR1</LI><LI>1μL Pst1</LI><LI>1μL EcoR1 and Pst1</LI>
2. Assemble the following components in a sterile tube:

Note: Different lanes require different tubes to be made up
3. Mix the solution gently by pipetting up and down
4. Close the tube and centrifuge for a few seconds in a microcentrifuge
5. Incubate at the specific enzyme’s optimum temperature (37 o C in this case)for 1-4 hours
6. Add loading buffer to a 1 X final concentration and proceed to the gel analysis</UL></DIV></SECTION><SECTION class="hull"><LABEL class="hull-title" for="cb17">PCR Protocol for Q5 High-Fidelity 2X Master Mix</LABEL><DIV class="hull-content">All reaction components should be assembled on ice then quickly transferred to a thermocycler that’s been preheated to the denaturation temperature (98oC)

Components:
All the components should be mixed prior to use

Method:
<UL><LI>Gently mix the reaction</LI><LI>Collect all the liquid found at the bottom of the tube by a quick spin if needed</LI><LI>Overlay the sample with mineral oil when using a PCR machine that doesn’t have a heated lid</LI><LI>Transfer the PCR tubes to the PCR machine to begin thermocycling</LI></UL>
Thermocycling conditions:

Annealing temperatures shouldn’t exceed 72 o C. You can use the NEB T m Calculator
found on the New England BioLabs website to calculate temperatures needed and
timings.

Guidelines

Template
<UL><LI>A high quality, purified DNA template is preferred as it greatly improves PCR success. Recommended amounts of such a template are shown below for a 50uL reaction:</LI></UL>
Primers
<UL><LI>Oligonucleotide primers should generally be 20-40 nucleotides long while having a GC content of 40-60%</LI><LI>Best results are seen when using each primer at a final concentration of 0.5uM in the reaction</LI></UL>
Mg2+ and additives
<UL><LI>The Q5 High-Fidelity Master Mix contains 2mM Mg++ when used at a 1X concentration, which is optimal for most PCR products</LI></UL>
Deoxynucleotides
<UL><LI>Final concentration of dNTPs is 200uM of each deoxynucleotide in the 1X final concentration</LI><LI>Q5 High-Fidelity DNA Polymerase cannot incorporate dUTP and isn’t recommended for use with uracil-containing primers or templates</LI></UL>
Q5 High-Fidelity DNA Polymerase concentration
<UL><LI>Concentration in the Master Mix has been optimized for best results under a wide conditions range</LI></UL>
Denaturation
<UL><LI>Initial denaturation of 30 seconds occurs at 98oC, which is enough for most amplicons from pure DNA templates.</LI><LI>Though longer denaturation times going up to 3 minutes can be used for templates that require it</LI></UL>
Annealing
<UL><LI>Optimal annealing temperatures for this Master Mix tend to be higher than for other PCR polymerases</LI><LI>Typically 10-30 second annealing steps should be used at 3oC above the Tm of the lower Tm primer</LI><LI>Temperature gradients can also be used to optimize the annealing temperature for each primer pair</LI><LI>For higher Tm primer pairs, two-step cycling without a separate annealing step can be used</LI></UL>
Extension
<UL><LI>Recommended extension temperature is 72oC
</LI><LI>With the recommended time being between 20-30 seconds per kb for complex, genomic samples.</LI></UL><LI>The time can be reduced to 10 seconds per kb for simpler templates (plasmid, E.coli, etc.) or complex templates smaller than 1kb</LI><LI>The extension time can be increased to 40 seconds per kb for cDNA or other long, complex templates if needed</LI><LI>A final extension of 2 minutes at 72oC is recommended</LI>


Cycle Number
<UL><LI>25-35 cycles yield sufficient products generally</LI><LI>For genomic amplicons, 30-35 cycles are advised</LI></UL>
2-step PCR
<UL><LI>Used when primers have annealing temperatures exceeding or are equal to 72oC (≥ 72°C).</LI><LI>This 2-step thermocycling protocol combines annealing and extension into one step</LI></UL>
Amplification of long products
<UL><LI>When amplifying products &gt; 6kb, you can increase the extension time to 40-50 seconds per kb.</LI></UL>
PCR Product
<UL><LI>Products generated using this Master Mix have blunt ends</LI><LI>If clonding is the next step then blunt-end cloning isn’t recommended</LI><LI>If T/A-cloning is to be done, the DNA should be purified prior to A-addition, since the Q5 High-Fidelity DNA Polymerase will degrade any overhangs generated</LI></UL></DIV></SECTION></NAV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>