Team:Kent/Results
Project
Description
Results
Modelling
Parts
Parts
Basic Parts
Composite Parts
Part Collection
Notebook
Logbook
Experiments
Future Ideas
Safety
Project Safety
Hazard Signs
Team
Meet the Team
Contribution
Attribution
Human Practices
Integrated Human Practices
Public Engagement
Interlab
Collaboration
Results
Our plan of creating our biobrick was pursued in two separate ways. First we attempted to clone the GFP with Standard 25 Prefix/Suffix from the biobrick library (BBa_K6480131) as well as the dCas13a we obtained from Addgene (L. buccalis C2C2, addgene #834852). We also ordered the whole gene through Integrated DNA Technologies (IDT)
Figure 1. 1% agarose gel electrophoresis diagnostic restriction digestion results of GFP with Standard 25 Prefix/Suffix (BBa_K648013) in pSB1C3 . Lane 1: DNA Marker; Lane 2: undigested; lane 3; Single digestion with EcoRI; lane 4: single digestion with Pst1; lane 5: double digestion with EcoRI and PstI.
dCAS13a
Figure 2:  1% agarose gel electrophoresis diagnostic restriction digestion results of L. buccalis C2C2, addgene #83485 in p2CT. Lane 1: DNA Marker; Lane 2: undigested; lane 3; Single digestion with EcoRI.
After successfully transforming these plasmids we designed primers for PCR amplification and Gibson Assembly3, adding a flexible linker sequence in between the GFP and the dCAS13a.
Because the dCAS13 gene contains three illegal restriction sites for iGEM submission (two EcoRI and one PstI) we also designed and ordered Quikchange4 PCR primers to remove these illegal sites.
After Gibson Assembly a screening PCR was performed with two screening primers, one 200 bp upstream and one 200bp downstream of the insert. Figure 3 shows a band the size approximately 4666 bp which indicates our insert (4266bp) plus an additional 400bp which account for the two screening primers on the 5’ and the 3’ end.
Figure 3. 1% agarose gel electrophoresis screening PCR results of dCAS13a - GFP in pSB1C3 . Lane 1: DNA Marker; Lane 2: negative control; lane 3; Screening PCR; lane 4: Screening PCR; lane 5: Screening PCR.
In order for our construct to work, it had to be transfected into HEK2935, a mammalian cell line. RNA localisation in bacteria is a contentious and poorly understood field. It does happen but not much is known about how, where and why6. In mammalian cells it is much better understood and poor localization has been linked with disease states7,8. Also because mammalian cells have cellular compartments, signals sequences can be added to the protein. We added nuclear localization sequences on both the N- and the C-terminal of the protein, so that unbound dCAS13a-GFP would localize in the nucleus until it is bound to an mRNA to track.
We transfected our construct (BBa_K2340000) with a guide RNA plasmid (BBa_K2340001 - BBa_K2340011) (1:2 ratio) into HEK293 cells using Lipofectamine™ 2000 in a 24 well plate in media and incubated for 24 hours at 37℃. This was viewed under a brightfield microscope at 100x magnification.
Figure 4. Plate layout for transfection and brightfield microscopy. Well 1: Cas13 + Rab13 (1), well 2: Cas13 + Rab13 (2), well 3: Cas13 + Rab13 (3), well 4: Cas13 + EV, well 5: Rab13 (1) + EV, well 6: Pkp4 (1) + EV, well 7: Cas13 + Pkp4 (1), well 8: Cas13 + Pkp4 (2), well 9: Cas13 + Pkp4 (3), well 10: Inpp1 (1)+ EV, well 11: EV + ꞵ-actin (1), well 12: EV, well 13: Cas13 + Inpp1 (1), well 14: Cas13 + Inpp1 (2), well 15: Cas13 + Inpp1 (3), well 16: Transfection reagent control, well 17: parental, well 18: GFP control, well 19: Cas13 + ꞵ-actin (1), well 20: Cas13 + ꞵ-actin (2), well 21: Cas13 + ꞵ-actin (3). EV: Empty vector. Transfection reagent control: Lipofectamine™ 2000 GFP Control: eGFP in pcDNA3 (Addgene plasmid # 13031)
In Figure 5 we see the results from the brightfield microscopy. We saw no expression of our construct (Fig 5A) but the GFP control did express (Fig. 5B), indicating that there was transfection. The difference may lie in the type of GFP. This version of the construct had a wtGFP, which is natively expressed at lower temperatures (15-20 oC) and folds poorly at 37oC. We aim to redesign the GFP found on our construct to an eGFP (which is mutated to perform better at 37oC) to submit to the registry.
Figure 5. Brightfield microscopy results. A: well 19: Cas13 + ꞵ-actin (1); no expression after 24 hours. After 30 and 48 hours there was still no expression (not shown here) B well 18: GFP Control; GFP expression after 24 hours.
Conclusion
We successfully build one biobrick by fusing together a dCas13a gene with a wtGFP gene from the biobrick library, adding a flexible linker and two nuclear localization sequences to improve stability and reduce background noise. The function of our construct has not been able to proof due to the poor folding of the wtGFP at 37oC and the short time frame. We continued to tweak the design, mainly substituting the wtGFP to an eGFP. Future experiments could test how the protein would react with single a NLS and longer/shorter or rigid  linker sequences.
References
1Rose, J. (2011). GFP with Standard 25 Prefix/Suffix. [WWW Document]. URL http://parts.igem.org/Part:BBa_K648013
2Addgene (2016). p2CT-His-MBP-Lbu_C2c2_R472A_H477A_R1048A_H1053A. [WWW Document]. URL http://www.addgene.org/83485/
3New England Biolabs (2017). Gibson Assembly® Master Mix. [WWW Document]. URL https://www.neb.com/products/e2611-gibson-assembly-master-mix#Product%20Information
4Agilent genomics (2017). Quikchange. [WWW Document]. URL http://www.genomics.agilent.com/en/Site-Directed-Mutagenesis/QuikChange/?cid=AG-PT-175&tabId=AG-PR-1160
5Thomas P., Smart, T.G. (2005). HEK293 cell line: A vehicle for the expression of recombinant proteins. Journal of Pharmacological and Toxicological Methods, 51: 187-200.
6Avraam Buskila, A.., Kannaiah, S. & Amster-Choder, O. (2014). RNA localization in bacteria. RNA Biology, 11(8): 1051-1060.
7Pyke, C., et al. (1992). Localization of Messenger RNA for MT 72,000 and 92,000 Type IV Collagenases in Human Skin Cancers by in Situ Hybridization. Cancer Research, 52: 1336-1341.
8Bashirullah, A., Cooperstock, R.L., Lipshitz, H.D. (1998). RNA localization in development. Annu Rev Biochem, 67: 335-94.
