<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Warsaw_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Warsaw/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Warsaw/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Warsaw/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Warsaw/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Warsaw Team</TITLE></P><DIV id="menu_div"><DIV id="menu"><UL><LI><A href="https://2012.igem.org/Team:Warsaw">HOME</A></LI><LI><A href="https://2012.igem.org/Team:Warsaw/Project#whyThis">PROJECT</A></LI><LI rel="whyThis"><A href="https://2012.igem.org/Team:Warsaw/Project#whyThis">why this?</A></LI><LI rel="steps"><A href="https://2012.igem.org/Team:Warsaw/Project#steps">steps</A></LI><LI rel="measurement"><A href="https://2012.igem.org/Team:Warsaw/Project#measurement">measurement</A></LI></UL><LI><A href="https://2012.igem.org/Team:Warsaw/Wetlab">WETLAB</A></LI><LI><A href="https://2012.igem.org/Team:Warsaw/Parts">PARTS</A></LI><LI><A href="https://2012.igem.org/Team:Warsaw/Safety">SAFETY</A></LI><LI><A href="https://2012.igem.org/Team:Warsaw/Team#team">TEAM</A></LI><LI rel="team"><A href="https://2012.igem.org/Team:Warsaw/Team#team">team</A></LI><LI rel="sponsors"><A href="https://2012.igem.org/Team:Warsaw/Team#sponsors">sponsors</A></LI><LI rel="acknowledgements"><A href="https://2012.igem.org/Team:Warsaw/Team#acknowledgements">acknowledgements</A></LI><LI rel="attribution"><A href="https://2012.igem.org/Team:Warsaw/Team#attribution">attribution</A></LI><LI><A href="https://2012.igem.org/Team:Warsaw/HumanPractices">HUMAN PRACTICE</A></LI></DIV></DIV></DIV><DIV id="left-menu"><UL><LI rel="whyThis"><A href="https://2012.igem.org/Team:Warsaw/Project#whyThis">why this?</A></LI><LI rel="steps"><A href="https://2012.igem.org/Team:Warsaw/Project#steps">steps</A></LI><LI rel="measurement"><A href="https://2012.igem.org/Team:Warsaw/Project#measurement">measurement</A></LI></UL></DIV><DIV id="main-content"><DIV id="whyThis"><SPAN><B>Why this project?</B></SPAN><P><I>Escherichia coli</I>, which is by far the greatest model for iGEM projects, is a gram-negative bacterium. Because of that, the expression of some proteins, which came from gram-positive bacteria, is sometimes hard to achieve in <I>E. coli</I>. Even if we managed to express proteins from gram-positive bacteria in a gram-negative model, the proteins could for example have different localization in the cell and behave differently from how they did in the original cell they came from.</P><P><I>Bacillus subtilis</I> is a great model of gram-positive bacteria; it was also used by some iGEM teams in previous years. We were curious, why only a few teams used that model. When we began searching partsregistry for some BioBricks especially for <I>Bacillus subtilis</I>, we realized that there is only a very limited number of them. It could be the reason why this model isn't so popular, but it is also the reason why we found working with <I>Bacillus subtilis</I> so interesting and at the same time so daring and challenging.</P><P>   Of course, we wanted to make a functional project, but we also thought that it is a great challenge to work with another model than <I>E. coli</I>. We truly believed, that making some new BioBricks for <I>Bacillus subtilis</I>, even such basic ones like promoters and rbs, is important because it would help the iGEM community to work with this great model of gram-positive bacteria in the future.</P><P>Our goal this year was to create a bacteria strain which could carry certain genes into eucaryotic cells. For safety reasons, and also to keep the work more coherent, we divided our task into two separate steps:</P><UL><LI><P>The first one was creating an 'invasive' <I>Bacillus subtilis</I> strain. <I>B. subtilis</I> is a non-pathogenic bacteria living peacefully in the soil. However, there are many bacteria species that have the ability to invade animal as well as human cells. We had an idea to create a plasmid for <I>B. subtilis</I> carrying listerolysin gene as to enable the bacteria to enter the eucaryotic cells just as <I>Listeria monocytogenes</I> from which the gene was taken does. <I>L. monocytogenes</I> is a dangerous pathogen; however, <I>B. subtilis</I> is a safe bacterium. With the lysis device installed in it, <I>B. subtilis</I> cells lyse after entering the eucaryotic cells. Since both of the bacteria are gram-positive, gene expression should undergo without obstructions. The plasmid also carries the GFP coding device which will help us to determine the success of our experiment. After the lysis of <I>B. subtilis</I> cells, GFP will be released and the measurement of its fluorescence will give us the idea of how the experiment worked out.
</P></LI><SMALL><B>The mechanism of invasion of mammalian cells by bacteria leading to delivered gene expression</B>
1. Adhesion
2. Internalization of the bacteria by the cell and subsequent phagocytosis
3. Expression of listeriolysin present on the <I>B. subtilis</I> construct lyses the phagosomal membrane and bacteria escape from the phagosome 
4. Release of the vector due to the installed lysis device in the bacteria
5. Replication and expression of the mammalian vector induced by oriP</SMALL><LI><P>The next step was creating a vector capable of replication and gene expression in mammalian cells. After the lysis of <I>B. subtilis</I> cells, the vector would be released directly into the cytoplasm, and then act just as mammalian DNA molecule. The EBNA1 gene from Epstein-Barr virus is essential for oriP to work properly. The vector would carry RFP, whose fluorescent activity measurement would prove the success of the presence, replication and expression of the vector within the mammalian cells. Due to safety measures, we never placed this plasmid in the bacteria cells. </P></LI><LI><P>The last step was creating a shuttle vector which would be capable of replication and gene expression inside the eucaryotic cells. The vector will carry RFP coding device, and it will be RFP flourescence measurement which will confirm the presence of the vector in  mammalian cells and GFP in bacterial cells. However, due to safety reasons, we did not combine the two steps of our experiment. Each of the systems is tested separately, with all the safety precautions. In the future, when all the safety issues are resolved, these systems might be able to help a great deal in gene therapy. Delivering certain genes right into the malfunctioning cells would be a tremendous help in treatment of complicated diseases.  </P></LI></UL></DIV><DIV id="steps"><B>First step</B><P>    In the first step of our project we designed a few BioBricks adapted for <I>Bacillus subtilis</I>. We decided, that even if some basic parts are available in partsregistry, we would create a few more to expand the amount of BioBricks for this bacterium. Therefore, we designed:</P><UL><LI>     Two new promoters for <I>Bacillus subtilis</I></LI><LI>    Three new RBSes for <I>Bacillus subtilis</I></LI><LI>       One terminator for <I>Bacillus subtilis</I></LI></UL><P>   We believed that these parts will be useful for creating constructs designed for <I> Bacillus subtilis</I> in this year's project of ours and in the future projects based on this bacterium.
We performed all designing work in Clone Manager software, which we found easy and useful.</P><P>    Then, we designed parts especially devoted to our project. Our goal was to create two main constructs:
the first one should allow <I>Bacillus subtilis</I> to enter mammalian cell. To give <I>Bacillus</I> this ability, we decided to use listeriolysin, a protein naturally existing in <I>Listeria monocytogenes</I>. To confirm that <I>Bacillus</I> entered the mammalian cell we used GFP protein, which would give us a fluorescent signal, easy to notice by using fluorescent microscopy.
We designed our construct like this:</P><P>   The listeriolysin gene was taken from the <A href="http://partsregistry.org/Part:BBa_K177026">BioBrick BBa_K177026</A>. Listeriolysin is a protein considered to be the virulence factor of <I>Listeria monocytogenes</I>, because of its pivotal role is the process of pathogenesis of these bacteria. Since both <I>L. monocytogenes</I> and <I>B. subtilis</I> are gram-positive bacteria, we believed that the expression of listeriolysin will be much better in <I> B. subtilis</I> than in the gram-negative <I>E. coli</I>.  </P>

    Our plan was to test all our basic parts in this construct, by creating few parallel versions, using different promoters and RBSes and choose the best version.<P>While doing that, we were also preparing construct which could replicate and be expressed in both bacterial (<I>E. coli</I>) and mammalian cell. We based our plasmid on pSB1A3 plasmid backbone and ligated into it the <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J52034">CMV mammalian promoter</A> (designed and sent to us by Slovenia iGEM Team) with mammalian RBSes (we wanted to test both <A href="http://partsregistry.org/Part:BBa_J63003">J63003</A> and <A href="http://partsregistry.org/Part:BBa_K165002">K165002</A> Kozak sequence) and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J06504">RFP</A> (to confirm expression in mammalian cells). To confirm expression in <I>E. coli</I> cells, we used the construct with <A href="http://partsregistry.org/Part:BBa_I746916">SF-GFP</A></P><P>  Another main construct was devoted for using directly in mammalian cells. It was based on BioBricks from partregistry with our new part containing oriP. To allow the plasmid to replicate in a mammalian cell, we prepared a BioBrick containing oriP and coding sequence of EBNA1 gene (which is essential for oriP to work properly) from Epstein-Barr virus. We got the sequence of oriP from <A href="http://tools.invitrogen.com/content/sfs/manuals/pcep4_man.pdf">pCep4</A> commercial plasmid available from Invitrogen. We had been trying to receive this BioBrick using PCR method.</P><P>    Unfortunately, the original oriP sequence had a restriction site of SpeI enzyme in the middle of the sequence. We decided to amplify this in two separate parts, because in this situation we could modify enzyme site by placing it in the middle of the primer for one of the parts. After amplification, we planned to ligate these two parts into one full BioBrick. Unluckily, we didn't manage to fulfill our plan. When we realized that we fail with preparing this part that way, we tried to get this BioBrick even with this illegal restriction site and just clone it without using SpeI enzyme. After several attempts, we got this BioBrick and ligated it with the construct containing CMV, RBS and RFP. Unfortunately when we received it, we were already running out of time and didn't manage to confirm the sequence of this part and test it in mammalian cells.</P><P>    The design of the constructs looked like this:</P><B>Second step</B><P>
Then we entered the wet lab and started preparing our BioBricks made of DNA, not only pixels on the computer screen ;)
We decided to achieve our promoters, RBS and terminator parts with method called in our laboratory “PCR without template”. It goes like this:<UL><LI>  For each part we designed two starters that covered each other in about 20bp on 3' ends</LI><LI>    Each primer had prefix or suffix (depending on the primer) on its 5' end</LI><LI>    Then we prepared PCR reaction mixture</LI><LI>  We used Taq polymerase MasterMix and primers in different concentrations</LI><LI>    We didn't add any DNA template</LI><LI>    We calculated the annealing temperature by oligocalculator </LI><LI>    For first three cycles of PCR reaction temperature was calculated for primer without overhang</LI><LI>    Then for thirty more cycles temperature of annealing was calculated for the whole long primer with overhang – that is because from this moment there was enough product of the reaction in the reaction mix to become a template for the next cycles</LI><LI>    And that's it!</LI></UL></P><P>Then we cloned our PCR product into pJet plasmid, using special <A href="http://www.fermentas.com/en/products/all/molecular-cloning/kits/k123-clonejet-pcr-cloning">kit for cloning PCR products</A>, and transformed with it <I>E. coli</I> TOP10. After isolation of the plasmid, we confirmed the sequence of received product by DNA sequencing (traditional Sanger sequencing). From this step, our parts were ready to digest and clone with procedures recommended by iGEM :)</P><P>We managed to optimize reaction conditions and received all of ours parts by PCR reaction. Unfortunately, then we faced some problems and weren't as successful as we hoped. We didn't manage to confirm all our products' sequences by DNA sequencing. Finally we received a full BioBrick part:</P><UL><LI><A href="http://partsregistry.org/Part:BBa_K780003">Promoter</A></LI><LI>Two RBS, <A href="http://partsregistry.org/Part:BBa_K780001">BBa_K780001</A> and <A href="http://partsregistry.org/Part:BBa_K780002">BBa_K780002</A></LI><LI><A href="http://partsregistry.org/Part:BBa_K780000">Terminator</A></LI></UL><B>Third step</B><P>    As soon as we received our first full and confirmed BioBrick part, we started preparing our construct for <I>Bacillus subtilis</I>. We began from ligation of this part with GFP, because that would allow us to check if our promoter and RBSes work. We used 3A assembly procedure, which we found on iGEM website. We followed the set pattern:</P>


We knew that we had still a long way ahead to receive our full construct in <I>Bacillus subtilis</I>, and so we decided to start measuring our parts in <I>E. Coli</I> for the start. Measurement in <I>E. Coli</I> is the most common standard of measurement in iGEM community. We predicted that our parts will be working also in <I>E. coli</I>, but with lower efficiency than in <I>Bacillus subtilis</I>.


We successfully measured our RBS37 (BBa_K780002) comparing to RBS Bba_B0034, and we obtained result 10,13%.

We also tried to measure our other parts, but we didn't manage to put together full constructs with them, so the measurement was impossible.<B>Step four</B><P>    	When we had a ready construct consisting of promoter + RBS + GFP we cloned it on <A href="http://partsregistry.org/Part:BBa_I742103">pTG262 vector</A>. This plasmid, which was constructed and sent to us by Edinburgh iGEM Team, is the only autoreplicating plasmid meeting iGEM standards designed for <I>Bacillus subtilis</I>. We managed to successfully clone our insert to the vector, what we confirmed by digesting with EcoRI and PstI and gel electrophoresis of the vector and insert.
In the same time we had been improving our constructs by adding the terminator to them.</P><B>Step five</B><P>Finally, in order to confirm the functionality of our construct, we intended to make series of measurements using fluorometry. We tried to transform two different strains of <I>Bacillus subtilis</I>: 168 and 979 and used two different transforming procedures which you can see <A href="https://static.igem.org/mediawiki/2012/3/32/Transformation_Bsu.pdf">here</A> and <A href=" http://wiki.biol.uw.edu.pl/t/img_auth.php/b/b5/Bacillus_subtilis_competent_cells.pdf">here</A>.</P><P>First of all, we tried to select our transformants on plates with LB agar with chloramphenicol. Unfortunately, after several trials we didn't received desirable colonies. Bacteria that grow on dishes with chloramphenicol do not carry our plasmid. We thought that something could be wrong with the concentration of the antibiotic, so we tried few different ones, but none worked. Then, we tried to select bacteria using neomycin resistance, since pTG626 plasmid also carries a resistance gene for this antibiotic. Unfortunately, it didn’t work either.</P><P> - Bielecki J., Youngman P., Connelly P., Portnoy D., '<I>Bacillus subtilis expressing a haemolysin gene from Listeria monocytogenes can grow in mammalian cells.</I>', Nature, 1990. </P><P> - Stewart C. et all, '<I> Genes and Regulatory Sites of the &quot;Host Takeover Module&quot; in the Terminal Redundancy of Bacillus subtilis Bacteriophage SPO1.'</I>, Virology, 1998.</P></DIV><DIV id="measurement"><B>Measurement</B><B>General info</B>

We designed two new RBSes for <I>Bacillus subtilis</I>. We managed to successfully measure one of them comparing to RBS BBa_B0034. Our measurement was performed in <I>E. coli</I>. Our reasults are presented below.

We also planned to measure our part in <I>Bacillus subtilis</I>, but unfortunately we didn't manage to successfully prepare and transform our measuring construct into that bacteria.<B>Sample preparation</B>

To measure the fluorescence of samples of 3 milliliters of LB with KAN were inoculated with a colony from the plate. Cultures were vigorously shaken in 37°C overnight. The next step was to centrifuge 1,5 milliliters of liquid culture in an 1,5 ml tube. The pellet was resuspended in 200 microliters of RF. Finally, the prepared samples were placed in the fluorimeter plate. We used Corning 96 deep well plates with clear UV transparent bottom. Measurements were carried out using Tecan HS2000 pro fluorimeter with bottom read mode and using active gain optimization function.<CENTER><TABLE style="margin:5px; width:300px; " cellpadding="10" cellspacing="0"><TBODY><TR><TH>Wavelength value</TH><TH> GFP </TH></TR><TR><TD> Excitation [nm]</TD><TD> 488 </TD></TR><TR><TD> Emission [nm]  </TD><TD> 509 </TD></TR></TBODY></TABLE><CENTER><B> Results</B></CENTER></CENTER></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Warsaw/Project">http://2012.igem.org/Team:Warsaw/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Warsaw/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Warsaw/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Warsaw/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Warsaw/Project&amp;oldid=228254" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>