"
Page
Discussion
View source
History
teams
Log in
 
Team:Warsaw/Project
From 2012.igem.org
Warsaw Team
HOME
PROJECT
why this?
steps
measurement
WETLAB
PARTS
SAFETY
TEAM
team
sponsors
acknowledgements
attribution
HUMAN PRACTICE
why this?
steps
measurement
Why this project?
Escherichia coli, which is by far the greatest model for iGEM projects, is a gram-negative bacterium. Because of that, the expression of some proteins, which came from gram-positive bacteria, is sometimes hard to achieve in E. coli. Even if we managed to express proteins from gram-positive bacteria in a gram-negative model, the proteins could for example have different localization in the cell and behave differently from how they did in the original cell they came from.
Bacillus subtilis is a great model of gram-positive bacteria; it was also used by some iGEM teams in previous years. We were curious, why only a few teams used that model. When we began searching partsregistry for some BioBricks especially for Bacillus subtilis, we realized that there is only a very limited number of them. It could be the reason why this model isn't so popular, but it is also the reason why we found working with Bacillus subtilis so interesting and at the same time so daring and challenging.
Of course, we wanted to make a functional project, but we also thought that it is a great challenge to work with another model than E. coli. We truly believed, that making some new BioBricks for Bacillus subtilis, even such basic ones like promoters and rbs, is important because it would help the iGEM community to work with this great model of gram-positive bacteria in the future.
Our goal this year was to create a bacteria strain which could carry certain genes into eucaryotic cells. For safety reasons, and also to keep the work more coherent, we divided our task into two separate steps:
The first one was creating an 'invasive' Bacillus subtilis strain. B. subtilis is a non-pathogenic bacteria living peacefully in the soil. However, there are many bacteria species that have the ability to invade animal as well as human cells. We had an idea to create a plasmid for B. subtilis carrying listerolysin gene as to enable the bacteria to enter the eucaryotic cells just as Listeria monocytogenes from which the gene was taken does. L. monocytogenes is a dangerous pathogen; however, B. subtilis is a safe bacterium. With the lysis device installed in it, B. subtilis cells lyse after entering the eucaryotic cells. Since both of the bacteria are gram-positive, gene expression should undergo without obstructions. The plasmid also carries the GFP coding device which will help us to determine the success of our experiment. After the lysis of B. subtilis cells, GFP will be released and the measurement of its fluorescence will give us the idea of how the experiment worked out.
The mechanism of invasion of mammalian cells by bacteria leading to delivered gene expression
1. Adhesion
2. Internalization of the bacteria by the cell and subsequent phagocytosis
3. Expression of listeriolysin present on the B. subtilis construct lyses the phagosomal membrane and bacteria escape from the phagosome
4. Release of the vector due to the installed lysis device in the bacteria
5. Replication and expression of the mammalian vector induced by oriP
The next step was creating a vector capable of replication and gene expression in mammalian cells. After the lysis of B. subtilis cells, the vector would be released directly into the cytoplasm, and then act just as mammalian DNA molecule. The EBNA1 gene from Epstein-Barr virus is essential for oriP to work properly. The vector would carry RFP, whose fluorescent activity measurement would prove the success of the presence, replication and expression of the vector within the mammalian cells. Due to safety measures, we never placed this plasmid in the bacteria cells.
The last step was creating a shuttle vector which would be capable of replication and gene expression inside the eucaryotic cells. The vector will carry RFP coding device, and it will be RFP flourescence measurement which will confirm the presence of the vector in  mammalian cells and GFP in bacterial cells. However, due to safety reasons, we did not combine the two steps of our experiment. Each of the systems is tested separately, with all the safety precautions. In the future, when all the safety issues are resolved, these systems might be able to help a great deal in gene therapy. Delivering certain genes right into the malfunctioning cells would be a tremendous help in treatment of complicated diseases.
First step
In the first step of our project we designed a few BioBricks adapted for Bacillus subtilis. We decided, that even if some basic parts are available in partsregistry, we would create a few more to expand the amount of BioBricks for this bacterium. Therefore, we designed:
Two new promoters for Bacillus subtilis
Three new RBSes for Bacillus subtilis
One terminator for Bacillus subtilis
We believed that these parts will be useful for creating constructs designed for Bacillus subtilis in this year's project of ours and in the future projects based on this bacterium.
We performed all designing work in Clone Manager software, which we found easy and useful.
Then, we designed parts especially devoted to our project. Our goal was to create two main constructs: the first one should allow Bacillus subtilis to enter mammalian cell. To give Bacillus this ability, we decided to use listeriolysin, a protein naturally existing in Listeria monocytogenes. To confirm that Bacillus entered the mammalian cell we used GFP protein, which would give us a fluorescent signal, easy to notice by using fluorescent microscopy. We designed our construct like this:
The listeriolysin gene was taken from the BioBrick BBa_K177026 . Listeriolysin is a protein considered to be the virulence factor of Listeria monocytogenes, because of its pivotal role is the process of pathogenesis of these bacteria. Since both L. monocytogenes and B. subtilis are gram-positive bacteria, we believed that the expression of listeriolysin will be much better in B. subtilis than in the gram-negative E. coli.
Our plan was to test all our basic parts in this construct, by creating few parallel versions, using different promoters and RBSes and choose the best version.
While doing that, we were also preparing construct which could replicate and be expressed in both bacterial (E. coli) and mammalian cell. We based our plasmid on pSB1A3 plasmid backbone and ligated into it the CMV mammalian promoter (designed and sent to us by Slovenia iGEM Team) with mammalian RBSes (we wanted to test both J63003 and K165002 Kozak sequence) and RFP (to confirm expression in mammalian cells). To confirm expression in E. coli cells, we used the construct with SF-GFP
Another main construct was devoted for using directly in mammalian cells. It was based on BioBricks from partregistry with our new part containing oriP. To allow the plasmid to replicate in a mammalian cell, we prepared a BioBrick containing oriP and coding sequence of EBNA1 gene (which is essential for oriP to work properly) from Epstein-Barr virus. We got the sequence of oriP from pCep4 commercial plasmid available from Invitrogen. We had been trying to receive this BioBrick using PCR method.
Unfortunately, the original oriP sequence had a restriction site of SpeI enzyme in the middle of the sequence. We decided to amplify this in two separate parts, because in this situation we could modify enzyme site by placing it in the middle of the primer for one of the parts. After amplification, we planned to ligate these two parts into one full BioBrick. Unluckily, we didn't manage to fulfill our plan. When we realized that we fail with preparing this part that way, we tried to get this BioBrick even with this illegal restriction site and just clone it without using SpeI enzyme. After several attempts, we got this BioBrick and ligated it with the construct containing CMV, RBS and RFP. Unfortunately when we received it, we were already running out of time and didn't manage to confirm the sequence of this part and test it in mammalian cells.
The design of the constructs looked like this:
Second step
Then we entered the wet lab and started preparing our BioBricks made of DNA, not only pixels on the computer screen ;)
We decided to achieve our promoters, RBS and terminator parts with method called in our laboratory “PCR without template”. It goes like this:
For each part we designed two starters that covered each other in about 20bp on 3' ends
Each primer had prefix or suffix (depending on the primer) on its 5' end
Then we prepared PCR reaction mixture
We used Taq polymerase MasterMix and primers in different concentrations
We didn't add any DNA template
We calculated the annealing temperature by oligocalculator
For first three cycles of PCR reaction temperature was calculated for primer without overhang
Then for thirty more cycles temperature of annealing was calculated for the whole long primer with overhang – that is because from this moment there was enough product of the reaction in the reaction mix to become a template for the next cycles
And that's it!
Then we cloned our PCR product into pJet plasmid, using special kit for cloning PCR products , and transformed with it E. coli TOP10. After isolation of the plasmid, we confirmed the sequence of received product by DNA sequencing (traditional Sanger sequencing). From this step, our parts were ready to digest and clone with procedures recommended by iGEM :)
We managed to optimize reaction conditions and received all of ours parts by PCR reaction. Unfortunately, then we faced some problems and weren't as successful as we hoped. We didn't manage to confirm all our products' sequences by DNA sequencing. Finally we received a full BioBrick part:
Promoter
Two RBS, BBa_K780001 and BBa_K780002
Terminator
Third step
As soon as we received our first full and confirmed BioBrick part, we started preparing our construct for Bacillus subtilis. We began from ligation of this part with GFP, because that would allow us to check if our promoter and RBSes work. We used 3A assembly procedure, which we found on iGEM website. We followed the set pattern:
We knew that we had still a long way ahead to receive our full construct in Bacillus subtilis, and so we decided to start measuring our parts in E. Coli for the start. Measurement in E. Coli is the most common standard of measurement in iGEM community. We predicted that our parts will be working also in E. coli, but with lower efficiency than in Bacillus subtilis.
We successfully measured our RBS37 (BBa_K780002) comparing to RBS Bba_B0034, and we obtained result 10,13%.
We also tried to measure our other parts, but we didn't manage to put together full constructs with them, so the measurement was impossible.
Step four
When we had a ready construct consisting of promoter + RBS + GFP we cloned it on pTG262 vector . This plasmid, which was constructed and sent to us by Edinburgh iGEM Team, is the only autoreplicating plasmid meeting iGEM standards designed for Bacillus subtilis. We managed to successfully clone our insert to the vector, what we confirmed by digesting with EcoRI and PstI and gel electrophoresis of the vector and insert.
In the same time we had been improving our constructs by adding the terminator to them.
Step five
Finally, in order to confirm the functionality of our construct, we intended to make series of measurements using fluorometry. We tried to transform two different strains of Bacillus subtilis: 168 and 979 and used two different transforming procedures which you can see here and here .
First of all, we tried to select our transformants on plates with LB agar with chloramphenicol. Unfortunately, after several trials we didn't received desirable colonies. Bacteria that grow on dishes with chloramphenicol do not carry our plasmid. We thought that something could be wrong with the concentration of the antibiotic, so we tried few different ones, but none worked. Then, we tried to select bacteria using neomycin resistance, since pTG626 plasmid also carries a resistance gene for this antibiotic. Unfortunately, it didn’t work either.
- Bielecki J., Youngman P., Connelly P., Portnoy D., 'Bacillus subtilis expressing a haemolysin gene from Listeria monocytogenes can grow in mammalian cells.', Nature, 1990.
- Stewart C. et all, ' Genes and Regulatory Sites of the "Host Takeover Module" in the Terminal Redundancy of Bacillus subtilis Bacteriophage SPO1.', Virology, 1998.
Measurement
General info
We designed two new RBSes for Bacillus subtilis. We managed to successfully measure one of them comparing to RBS BBa_B0034. Our measurement was performed in E. coli. Our reasults are presented below.
We also planned to measure our part in Bacillus subtilis, but unfortunately we didn't manage to successfully prepare and transform our measuring construct into that bacteria.
Sample preparation
To measure the fluorescence of samples of 3 milliliters of LB with KAN were inoculated with a colony from the plate. Cultures were vigorously shaken in 37°C overnight. The next step was to centrifuge 1,5 milliliters of liquid culture in an 1,5 ml tube. The pellet was resuspended in 200 microliters of RF. Finally, the prepared samples were placed in the fluorimeter plate. We used Corning 96 deep well plates with clear UV transparent bottom. Measurements were carried out using Tecan HS2000 pro fluorimeter with bottom read mode and using active gain optimization function.
Wavelength value
GFP
Excitation [nm]
488
Emission [nm]
509
Results
Retrieved from " http://2012.igem.org/Team:Warsaw/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
