<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Jiangnan_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Jiangnan/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="Jnav"><DIV class="Jnavtitle"><A href="https://2018.igem.org/Team:Jiangnan/Safety">Safety</A></DIV><DIV class="Jnavtitle"><A href="https://2018.igem.org/Team:Jiangnan/Hardware">Hardware</A></DIV><DIV class="Jnavtitle" onmouseover="Jnavshow(this)" onmouseleave="Jnavhide(this)"><A href="https://2018.igem.org/Team:Jiangnan/Team">Team</A><DIV class="Jnavdrag"><UL><LI><A href="https://2018.igem.org/Team:Jiangnan/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Attributions">Attribution</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Collaborations">Collaboration</A></LI></UL></DIV></DIV><DIV class="Jnavtitle" onmouseover="Jnavshow(this)" onmouseleave="Jnavhide(this)"><A href="https://2018.igem.org/Team:Jiangnan/Human_Practices">Human Practice</A><DIV class="Jnavdrag"><UL><LI><A href="https://2018.igem.org/Team:Jiangnan/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Silver">Silver</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Entrepreneurship">Gold</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Entrepreneurship">Entrepreneurship</A></LI></UL></DIV></DIV><DIV class="Jnavtitle"><A href="https://2018.igem.org/Team:Jiangnan/Model">Model</A></DIV><DIV class="Jnavtitle" onmouseover="Jnavshow(this)" onmouseleave="Jnavhide(this)"><A href="https://2018.igem.org/Team:Jiangnan/Notebook">Notebook</A><DIV class="Jnavdrag"><UL><LI><A href="https://2018.igem.org/Team:Jiangnan/Notebook">Lab Book</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Protocol">Protocol</A></LI></UL></DIV></DIV><DIV class="Jnavtitle" onmouseover="Jnavshow(this)" onmouseleave="Jnavhide(this)"><A href="https://2018.igem.org/Team:Jiangnan">Project</A><DIV class="Jnavdrag"><UL><LI><A href="https://2018.igem.org/Team:Jiangnan/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Demonstrate">Demonstration</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Results">Result</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Parts">Part</A></LI></UL></DIV></DIV></DIV><DIV style="width:100%;background-color: #f0ebea"><DIV style="position: relative;"><DIV style="position: absolute;left: 2em; bottom: 4em;"><H4 style="color: white;">Determination of
		Plasma device parameter</H4></DIV></DIV><DIV class="row"><DIV class="col s10 offset-s1"><P><B>An</B> orthogonal L18[3]<SUP>7</SUP> test was designed to explore the effect of different parameter combinations of plasma-activated medium (PAM). Eighteen trials encompassing 7 factors (i.e., [T]treatment time, [A]the well size, [F]helium flow rate, [C]number of cells, [U]output voltage, [D1]distance from the tail of the plasma jet to the surface of the medium, [D2]thickness of medium) and 3 levels were conducted (Table 1 &amp; 2). The frequency was fixed at 8.8KHz.</P></DIV></DIV><DIV class="row"><DIV class="col s10 offset-s1"><H5 class="JredL">Linear model construction</H5><P>Thre linear was constructed using R as equation(1):</P><P>The dependent variable Y is the measurement of virus amplification after Plasma-treated through orthogonal design, and the other 7 factors are independent variables in this equation.
		The full model encompassing all these parameters was constructed by multivariate linear regression. The stepwise removal of each parameter was conducted followed by model feasibility assessment to identify independent parameters without collinearity. </P><H5 class="JredL">Optimal parameter configuration of PAM identified for triple-negative breast cancer cells</H5><P>T9 was selected as the optimal experimental configuration, which corresponds to ‘treatment time’ (‘T’) of 3 min, ‘liquid surface area’ (‘A’) of 4.5 cm<SUP>2</SUP>, ‘thickness of medium’ (‘D2’) of 0.2 cm, ‘number of cells’ (‘C’) of 1.5×105 cells/mL, ‘output voltage’ (‘U’) of 1.1 kV, ‘distance from the tail of the plasma jet to the surface of the medium’ (‘D1’) of 1 cm and ‘helium flow rate’ (‘F’) of 1.5 SLM, respectively.</P><H5 class="JredL">Linear model assessmentOutlier test </H5><P>Outliers were detected according to the student T test of studentized residuals from the outlier test and the Cook's distance from the influence analysis<SPAN class="JredL">T test</SPAN>
		The Bonferroni corrected p value from the T-test of studentized residuals of the built TN linear model was used to identify the outliers of the trials. The studentized residuals <SUP>[3]</SUP> were computed using Equation (2). 
		</P><P>where 'SRESI D<SUB>i</SUB>', 'e<SUB>i</SUB>', 'S<SUB>yx</SUB>', 'n' and 'X<SUB>i</SUB>'' each represents studentized residuals, residual, standard error, sample size and i<SUP>th</SUP> the variable, respectively.<SPAN class="JredL">Influence analysis  </SPAN>
		Cook's D <SUP>[3]</SUP> was used to identify trials with strong influence on the results, as defined in Equation (3). 
		</P><P>where 'h<SUB>i</SUB>', ‘D<SUB>i</SUB>’, ‘n’, ‘k’ each represents the leverage, Cook's D, sample size and the number of variables in the model. If Cook's D is greater than 4 / (n-k-1), it will be recognized as a strong influential trial with statistic significance. </P><H5 class="JredL">Normality test</H5><P>To determine whether the trials are well-modeled by a normal distribution, the quantile-quantile plot (QQ-plot) <SUP>[4]</SUP> of the standardized data against normal distribution was drawn. The trials were considered to follow the normal distribution if they fell close to the 45 degree line in the plot.</P><P>According to the normal QQ plot, the plot comparing residuals and fitted values, the trials fall close to the 45 degree line representing the correlation between standardized residuals and theoretical quantiles, the square root of the standardized residuals are almost randomly distributed across all fitted values. Thus, the optimal TN model satisfies the normality and homoscedasticity assumption of the fitted linear model.</P><H5 class="JredL">Multicollinearity </H5><P>To test whether there is a linear association between the variables in the model and all variables are independent, the variance inflation factor,<SUP>[3]</SUP> denoted as variance inflation factor (VIF) and defined using Equation (5), was used to test the multicollinearity of the model. </P><P>where 's<SUP>2</SUP>'' represents variance.</P><P>Multicollinearity, defined as the situation where one variable can be linearly predicted from the others with substantial degree of accuracy, was assessed using VIF that increases with collinearity. VIF is defined as VIF=1 / (1-s2), where s2 refers to the variance. It is canonically considered to have the multiple collinearity problems if VIF &gt; 4. <SUP>[5]</SUP> The VIF values of ‘T’ ‘A’ ‘D2’ ‘C’ are all close to 1 (i.e., 1.000559, 1.000559, 1.000000 and 1.000000, respectively), suggesting that there is no multiple co-linearity between the four deterministic parameters</P><P>The linear model encompassing the four deterministic parameters is: </P></DIV></DIV><DIV class="row"><DIV class="col s10 offset-s1"><H4 class="Jgreen" style="text-align: right;">High titer</H4><P>We collected a panel of genes responsible for virus multiplication through text mining, retrieved other associated gene by computing correlations using public datasets from the GEO database, and constructed the corresponding network using the fast heuristic algorithm and label propagation algorithm with GENEMANIA. </P><P>In particular, the heuristic algorithm was used for calculating a single composite functional association network from multiple data sources based on linear regression, and a label propagation algorithm was used to predict gene functionalities given the composite functional association network.</P><H5 class="Jgreen">Fast heuristic algorithm</H5><P>Each network data source is represented as a weighted interaction network where each pair of genes is assigned an association weight. The weight is either zero (indicating no interaction) or a positive value (reflecting the strength of interaction). The association of a pair of genes in a gene expression dataset can be assigned as the Pearson correlation coefficient of their expression levels across multiple conditions in an experiment. The more likely that the genes are co-expressed, the higher the weight is, which ranges from -1 to 1.
		Both binary and continuous values can be used for building functional association networks. In the case of binary data, all zeros are replaced with log (1 - β ) and ones replaced with -log(β), where β is the proportion of samples with the given feature being 1. This allows for the emphasis of similarities between genes that share 'uncommon' features. 
		Similarity matrices were constructed for both types of data using the Pearson correlation coefficient to measure pair-wise similarities. </P><H5 class="Jgreen">Label propagation algorithm </H5><P>A variation of the Gaussian field label propagation algorithm was used here to predict the composite network. 
		Label propagation algorithm, like most functionality prediction algorithms, assigns a score to each node in the network, called the 'discriminant value'. This score reflects the computed degree of association that the node has to the seed list defining the given function. This value can be thresholded to enable predictions of a given gene function. 
		WeA positive weight reflecting its usefulness in predicting a given function of interest.is assigned to each functional association network derived from these data sources. with a positive weight reflecting its usefulness in predicting a given function of interest. Once these weights were calculated, the weighted average of the association networks was constructed into a function-specific association network.
		Denote the vector of discriminant values by f, the bias vector by y, and the matrix derived from the association network by W. We can represent an association network over n genes by a symmetric matrix W whose non-zero entries indicate the associations in the network. In particular,(i,j)<SUP>th</SUP> the element of W, W<SUB>ij</SUB>, is the association between genes i and j,W<SUB>ij</SUB> with = 0 indicating no edge between genes i and j. To ensure that all associations are non-negative, any negative associations can be set to zero. There is l labeled genes and u unlabeled genes (n = l + u) for each binary classification task. These labels are used to specify a bias vector y, where y  {+1, k, -1}, to represent that gene i is positive, unlabeled, or negative, respectively. In the label propagation algorithm:</P><P>where n+ and n- are the numbers of positive and negative genes, respectively. The discriminant values were computed by solving the following objective function:</P><P>which ensures that the discriminant values of positive and negative genes remain close to their label bias (first term in the summation) and the discriminant values of the associated genes (genes that have positive W<SUB>ij</SUB>) are not too different from each other (second term in the summation). Equation 1 can be written in matrix notation as:</P><P>where L = D - W is called the graph Laplacian matrix and D = diag(d<SUB>i</SUB>) (D is a diagonal matrix with  = D<SUB>ij</SUB> and D<SUB>ij</SUB> = 0 if i  j) and di =. Since the association matrix is symmetric, L is symmetric and semi-definite positive, and equation 2 is a quadratic optimization problem with a global minimum. In fact, solutions to equation 2 can be obtained by solving a sparse linear system y = (I - L)f.</P></DIV></DIV><DIV class="row" style="margin-bottom:0"><DIV class="col s10 offset-s1"><H4 style="color: #ffab00">Suspension Culture</H4><P>Frist, we sequenced the transcriptome of suspension and adherent cell lines of BHK-21 and CHO-K1, and then aligned their reads to the reference transcripts. The reference of BHK-21 is MesAur1.0 () and that of CHO-K1 is CriGri_1.0 (ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/223/135/GCF_000223135.1_CriGri_1.0/GCF_000223135.1_CriGri_1.0_rna.fna.gz).</P><P>Second, through transcriptome analysis, we eliminated the low-quality genes using R and quantified the expression using FPKM (fragments Per kb per Million), and genes with FPKM &amp;lt 0.5 were removed. Also, a threshold, FDR (false discovery rate) ≤ 0.05 and FC (fold change) ≥ 1, was defined to further filter the remaining genes to find DEGs (different expressed genes), i.e. BHK_DEGs (4916) and CHO_DEGs (3597). </P><P>We analysed genes differentially expressed between suspended and adherent cells to explore genes potentially responsible for the suspension feature of cells. We eventually obtained two sets of DEGs, one from BHK and one from CHO, namely BHK_sus_muts and CHO_sus_muts. </P><P>Further, we obtained the sus_muts gene set by analyzing their SNPs and mutations where genes with potential causal genetic changes were reserved. By intersecting each of the two DEGs set with sus_muts set respectively, we get BHK_keys and CHO_keys. Through taking the intersection of these two sets, we obtained 27 genes. </P><P>After checking and reserving genes with consistent regulatory directions in suspended vs. adherent cells, we obtained 18 genes as our target in the end. The network was constructed using the same algorithm as above as Figure XX↓, where the top gene was selected for genetic modulation in the experiments.</P></DIV><DIV class="col s10 offset-s1"><H4 style="text-align: center;">Reference</H4><P>1. Montojo, J., et al., GeneMANIA: Fast gene network construction and function prediction for Cytoscape. F1000Res, 2014. 3: p. 153.
			2. Team, R.C., R: A Language and Environment for Statistical Computing. 2018</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>