<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Jiangnan_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Jiangnan/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="Jnav"><DIV class="Jnavtitle"><A href="https://2018.igem.org/Team:Jiangnan/Safety">Safety</A></DIV><DIV class="Jnavtitle"><A href="https://2018.igem.org/Team:Jiangnan/Hardware">Hardware</A></DIV><DIV class="Jnavtitle" onmouseover="Jnavshow(this)" onmouseleave="Jnavhide(this)"><A href="https://2018.igem.org/Team:Jiangnan/Team">Team</A><DIV class="Jnavdrag"><UL><LI><A href="https://2018.igem.org/Team:Jiangnan/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Attributions">Attribution</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Collaborations">Collaboration</A></LI></UL></DIV></DIV><DIV class="Jnavtitle" onmouseover="Jnavshow(this)" onmouseleave="Jnavhide(this)"><A href="https://2018.igem.org/Team:Jiangnan/Human_Practices">Human Practice</A><DIV class="Jnavdrag"><UL><LI><A href="https://2018.igem.org/Team:Jiangnan/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Silver">Silver</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Entrepreneurship">Gold</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Entrepreneurship">Entrepreneurship</A></LI></UL></DIV></DIV><DIV class="Jnavtitle"><A href="https://2018.igem.org/Team:Jiangnan/Model">Model</A></DIV><DIV class="Jnavtitle" onmouseover="Jnavshow(this)" onmouseleave="Jnavhide(this)"><A href="https://2018.igem.org/Team:Jiangnan/Notebook">Notebook</A><DIV class="Jnavdrag"><UL><LI><A href="https://2018.igem.org/Team:Jiangnan/Notebook">Lab Book</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Protocol">Protocol</A></LI></UL></DIV></DIV><DIV class="Jnavtitle" onmouseover="Jnavshow(this)" onmouseleave="Jnavhide(this)"><A href="https://2018.igem.org/Team:Jiangnan">Project</A><DIV class="Jnavdrag"><UL><LI><A href="https://2018.igem.org/Team:Jiangnan/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Demonstrate">Demonstration</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Results">Result</A></LI><LI><A href="https://2018.igem.org/Team:Jiangnan/Parts">Part</A></LI></UL></DIV></DIV></DIV><DIV style="width=100%;background-color: #f0ebea"><DIV class="row"><DIV class="col s7 offset-s2 Jshadow"><UL><LI class="Jrightborder Jbottomborder"><DIV style="width: 100%; text-align: center;"><DIV style="width: 100%">week 1</DIV></DIV></LI><LI class="Jrightborder Jbottomborder"><DIV style="width: 100%; text-align: center;"><DIV style="width: 100%">week 2</DIV></DIV></LI><LI class="Jrightborder Jbottomborder"><DIV style="width: 100%; text-align: center;"><DIV style="width: 100%">week 3</DIV></DIV></LI><LI class="Jrightborder Jbottomborder"><DIV style="width: 100%; text-align: center;"><DIV style="width: 100%">week 4</DIV></DIV></LI><LI class="Jrightborder Jbottomborder"><DIV style="width: 100%; text-align: center;"><DIV style="width: 100%">week 5</DIV></DIV></LI><LI class="Jbottomborder"><DIV style="width: 100%; text-align: center;"><DIV style="width: 100%">week 6</DIV></DIV></LI><LI class="Jrightborder"><DIV style="width: 100%; text-align: center;"><DIV style="width: 100%">week 7</DIV></DIV></LI><LI class="Jrightborder"><DIV style="width: 100%; text-align: center;"><DIV style="width: 100%">week 8</DIV></DIV></LI><LI class="Jrightborder"><DIV style="width: 100%; text-align: center;"><DIV style="width: 100%">week 9</DIV></DIV></LI><LI class="Jrightborder"><DIV style="width: 100%; text-align: center;"><DIV style="width: 100%">week 10</DIV></DIV></LI><LI class="Jrightborder"><DIV style="width: 100%; text-align: center;"><DIV style="width: 100%">week 11</DIV></DIV></LI></UL></DIV></DIV><DIV class="row" id="broadweek1"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Broad Spectrum</H5><H4>Week 1</H4><P>Primer design</P></DIV><P>Referring to the canine-derived Nectin 4 mRNA sequence (GenBank accession number: NM_001313853.1) registered in GenBank, we added enzyme cutting site HindIII and Kozak sequence at the 5' end of the initiation codon ATG, and the FLAG-tag, termination codon TGA and the restriction enzyme cutting site Xho I before the termination codon of the 3' end of the coding sequence. </P><DIV class="JFigure"><P><B>Tablet 1.</B> The sequence of primers used in this study.</P></DIV></DIV></DIV><DIV class="row" id="broadweek2"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Broad Spectrum</H5><H4>Week 2</H4><P>Nectin 4 biobrick construction (pcDNA-DogN4-FLAG)</P></DIV><P>We received the Nectin 4 fragment and ligated it into the vector pcDNA3.1(+) to obtain the plasmid pcDNA-DogN4-FLAG.</P><P>1. Prepare the vector</P><DIV class="JFigure"><P><B>Tablet 2.</B> Reaction system.</P></DIV><P>2. Prepare the insertion fragment</P><DIV class="JFigure"><P><B>Tablet 3.</B> Reaction system.</P></DIV><P>3. Obtain the plasmid pcDNA-DogN4-FLAG</P><DIV class="JFigure"><P><B>Tablet 4.</B> Reaction system.</P></DIV><P>We transformed the ligation product into competent E. coli.</P><P>5.	We plated the culture solution on an Amp-resistant LB plate to get monoclonal.</P><P>6.	We conducted colonal PCR verification.</P><P>7.	We isolated the plasmid from E.coli.</P><P>Result: 
		The double digestion results of the restriction enzymes HindIII and Xho I were shown in Fig 1. The 5400 bp band in Lane 1 was the pcDNA3.1(+) plasmid backbone cut by both enzymes. The 1570 bp band was the Nectin 4 fragment that connects the two enzyme cutting sites. The results showed that we have successfully obtained the Nectin 4 biobrick pcDNA-DogN4-FLAG.</P><DIV class="JFigure"><P><B>Fig 1.</B> Identification of recombinant pcDNA-DogN4-FLAG.
M: DL10000 DNA Marker; 1: pcDNA-DogN4-FLAG (digested with HindIII and XbaI)</P></DIV></DIV></DIV><DIV class="row" id="broadweek3"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Broad Spectrum</H5><H4>Week 3</H4><P>Electroporation transfection and calcium phosphate transfection</P></DIV><P>1. Electroporation transfection 
			Day 1 Material preparation: We prepared materials for electrorotation, such as cells (2×106 cells/mL) and plasmid (20 ug) with concentration greater than 1 ug/ul for each sample. 
			Day 2 Transfection: We transfected MDBK cells. After incubation for 12h, we changed cells back to conventional medium.
			Day 3 Medium changing: We removed the supernatant from the 6-well plate and replaced it with fresh medium.
			Day 5 Test: We observed GFP expression in cells using fluorescence microscopy. </P><P>2. Liposomal transfection 
				Day 3 Cell seeding: We seeded the logarithmic growth phase cells in a 6-well plate. 
				Day 4 Transfection: We transfected MDBK cells. After incubation for 6-8 h, we replaced it with complete medium.
				Day 5-6 Cell culture: We changed cell medium when cells grew to 90% confluence.
				Day 7 Test: We observed GFP expression in cells using fluorescence microscopy. </P></DIV></DIV><DIV class="row" id="broadweek4"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Broad Spectrum</H5><H4>Week 4</H4><P>Electroporation transfection and calcium phosphate transfection</P></DIV><P>1. Electroporation transfection 
			Day 1 Material preparation: We prepared materials for electrorotation, such as cells (2×106 cells/mL) and plasmid (20 ug) with concentration greater than 1 ug/ul for each sample. 
			Day 2 Transfection: We transfected MDBK cells. After incubation for 12h, we changed cells back to conventional medium.
			Day 3 Medium changing: We removed the supernatant from the 6-well plate and replaced it with fresh medium.
			Day 5 Test: We observed GFP expression in cells using fluorescence microscopy. </P><P>2. Calcium phosphate transfection 
			Day 3 Material preparation: We seeded the logarithmic growth phase cells in a 6-well plate and cultivated them for 24 h when they reached 50% -70% confluence. We precipitated the plasmid using ethanol and the solution concentration was 500 ng/ul. Moreover, we dried the precipitate in the air.
			Day 4 Transfection: We transfected MDBK cells and placed the 6-well plate in the incubator overnight.
			Day 5-6 Medium changing: We removed the supernatant from the 6-well plate and washed cells twice using D-PBS, and replaced it using fresh medium.
			Day 7 Test: We observed GFP expression in cells using fluorescence microscopy.</P><P>Result:
			The results are shown in Fig 2. The transfection efficiency of the lentivirus transfection method was 70%, which is the highest among all tested approaches. The efficiency was 40% using electroporation transfection, which is lower but adequate. The efficiency of the liposomal transfection method was only 4% and the calcium phosphate transfection method was 1%, which are too low to be accepted. Though lentivirus has the highest efficiency, we used the electroporation approach in the following precedures as the lentivirus method is time-consuming.</P><DIV class="JFigure"><P><B>Fig 2.</B> Transfection efficiency in MDBK cells by different approaches (×100).</P></DIV></DIV></DIV><DIV class="row" id="broadweek56"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Broad Spectrum</H5><H4>Week 5-6</H4><P>Optimum antibiotics screen concentration determination</P></DIV><P>Day 1 Cell seeding: We seeded the logarithmic growth phase cells in 96-well plates for 100 ul per well and cultivated them for 24 h when they reached 50%-70% confluence.
		Day 2 Dosing: We set nine groups and three parallel wells in each group, and added antibiotics G418 to reach different concentration gradients in each group, i.e., 0, 1.0, 1.2, 1.4, 1.6, 1.8, 2.0, 2.2, 2.4 g/L.</P><P>We changed the fluid every three days and added antibiotics G418 at the same time. When cells died after 7-10 days of dosing, the dosage was considered the optimum screen concentration.</P><P>Result:
		As seen from Fig 3, after G418 selection for 7 days, the cell survival rate of the experimental group at a concentration of 2 g/L or more was close to 0%. Thus, 2 g/L of G418 was selected as the screen concentration.</P><DIV class="JFigure"><P><B>Fig 3.</B> The survival rate of MDBK cells screened at different concentrations of G418 after 7 days.</P></DIV></DIV></DIV><DIV class="row" id="broadweek78"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Broad Spectrum</H5><H4>Week 7-8</H4><P>Genetically modified cell strain selection</P></DIV><P>1. Electroporation transfection
		2. Genetically modified cell strain selection 
		After one week of antibiotic screen, we selected monoclonal cells and expanded them using antibiotic G418 (1 g/L) and finally obtained the genetically modified cell strain named MDBK-N4.
		3. Cell cryopreservation
		Cell cryopreservation formulation: 50% cell culture medium, 40% serum, 10% DMSO.</P><P>Results:
		According to these results, we successfully transfected the plasmid pcDNA-DogN4-FLAG into MDBK cells using the electroporation transfection approach, and screened cells using antibiotic G418(2 g/L) to obtain the genetically modified cell strain MDBK-N4. </P></DIV></DIV><DIV class="row" id="broadweek9"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Broad Spectrum</H5><H4>Week 9</H4><P>Nectin 4 mRNA and protein expression detection</P></DIV><P>1. Nectin 4 expression detection at the transcriptional level in genetically modified cell strain using RT-PCR
		We designed the primers of the RT-PCR using the Primer-blast function in the NCBI website and submitted it to Suzhou Jinzhizhi Biotechnology Co. Ltd. for synthesis. The primers used in this test are shown in Table 5. </P><DIV class="JFigure"><P><B>Table 5.</B> Primers in this study.</P></DIV><P>2. PCR amplification products test using agarose gel electrophoresis
		We seeded MDBK cells (negative control) and MDBK-N4 cells on six-well plates and cultivated them until 80%-90% confluence. Then, we extracted the total RNA of the cells using the TIANGEN RNA simple (DP149) Kit. Afterwards, we reversely transcribed RNA (1 ug) to cDNA with Titanium® One-Step RT-PCR Kit and amplified Nectin 4 and GADPH using PCR where reverse transcription PCR primers were used as the primers and cDNA was used as the template. Finally, we tested the PCR amplification products using agarose gel electrophoresis, observed and analyzed it using a gel imager. </P><P>3. Nectin 4 expression detection at the translational level using Western blot 
		We seeded MDBK (negative control) and MDBK-N4 cells in six-well plates and cultivated them until 80%-90% confluence. We lysed cells with lysate RIPA to extract and quantify proteins. We took the protein sample (30 ug) to perform polyacrylamide gel electrophoresis and transferred the desired protein to a PVDF membrane using an electrorotometer. Following that, we washed the membrane three times with TBST and blocked it with TBST containing 5% skim milk for 2 h. After blocking, we washed the membrane three times with TBST and incubated it using a primary antibody at 4°C overnight. After incubation with primary antibody, we washed the membrane three times using TBST and added the secondary antibody to incubate it for 1.5 h at room temperature. Then we washed the membrane three times using TBST. It was observed that Bcl reagent developed color using a chemiluminescence gel imager.</P><P>Result:
		1. RT-PCR results are shown in Fig 4. When the cDNA of MDBK-N4 cells (the 10th generation) was used as a template for PCR, lanes 2 and 4 showed the expression of both Nectin 4 and GAPDH. When the cDNA of MDBK cells was used as a template, lanes 1 and 3 showed only GAPDH expression. This indicated that Nectin 4 was expressed at the transcriptional level in MDBK-N4 cells.</P><DIV class="JFigure"><P><B>Fig 4.</B> Nectin 4 mRNA expression detected in MDBK-N4 cells by RT-PCR.
M: DL500 DNA Marker; 1-2: MDBK cells and MDBK-N4 cells for Nectin 4;3-4: MDBK cells and MDBK-N4 cells for GAPDH</P></DIV><P>2. Western blot results are shown in Fig 5. Compared with the control MDBK cells, MDBK-N4 cells (10th generation) showed a clear band at 56 kDa, indicating that MDBK-N4 cells can stably express NECTIN4 protein.</P><DIV class="JFigure"><P><B>Fig 5.</B> Nectin 4 expression detected in MDBK-N4 cells by Western blot.</P></DIV></DIV></DIV><DIV class="row" id="broadweek10"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Broad Spectrum</H5><H4>Week 10</H4><P>Nectin 4 and viral coat protein expression detection and TfR primer design </P></DIV><P>1.	Nectin 4 sample preparation: We seeded MDBK (negative control) and MDBK-N4 cells on 96-well plates and cultivated them until 50%-60% confluence. Then we observed Nectin 4 expression.
		2.	Virus coat protein sample preparation: When cells grew to 70%-80% confluence, we washed cells once using PBS and inoculated them with CDV(50 ul). After virus adsorption for 1 h, we replaced the medium with DMEM medium containing 2% FBS and cultivated cells until significant lesions appeared. 
		3.	Indirect immunofluorescence.
		4.	Primer design.</P><DIV class="JFigure"><P><B>Tablet 6.</B> The sequence of primers used in this study.</P></DIV><P>Result:
		1. Nectin 4 expression in MDBK-N4 cells 
		Immunofluorescence results are shown in Fig 6. Compared with control MDBK cells (Fig 6A), Nectin 4 (green light) was expressed in MDBK-N4 cells (10th generation) (Fig 6B). Enlarged results are shown in Fig 7. Nectin 4 in MDBK-N4 cells was mainly distributed on cell membrane, encompassing the blue-emitting nuclei (Fig 7B). This indicates that MDBK-N4 cells can stably express and transport Nectin 4 to cell membrane.</P><DIV class="JFigure"><P>Fig 6. Nectin 4 expression detected in MDBK-N4 cells by immunofluorescence (×40). 
			A: MDBK cells; B: MDBK-N4 cells</P></DIV><DIV class="JFigure"><P>Fig 7 Nectin 4 expression detected in MDBK-N4 cells by immunofluorescence (×100).
			A: MDBK cells; B: MDBK-N4 cells</P></DIV><P>2.  Viral infection of MDBK-N4 cells
		The immunofluorescence results are shown in Fig 8. After CDV infection for 3 days, MDBK-N4 cells (the 10th generation) developed obvious pathological changes. Some anoikised, most were drawn, and the virus coat protein (green light) was distributed inside cells. It was demonstrated that Nectin4 protein expressed by MDBK-N4 cells is functional, i.e., it does function as a virus receptor. It was observed that CDV replicated and proliferated in cells to generate progeny virus, which infected the surrounding cells and resulted in cell apoptosis.</P><DIV class="JFigure"><P>Fig 8. Envelope protein of CDV detected in MDBK-N4 cells by fluorescent microscope (×40).
			A: MDBK cells; B: MDBK-N4 cells</P></DIV></DIV></DIV><DIV class="row" id="broadweek11"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Broad Spectrum</H5><H4>Week 11</H4><P>TfR biobrick construction (PLVX-TfR)</P></DIV><P>We received the TfR fragment and ligated it into the vector PLVX to obtain the plasmid PLVX-TfR.</P><P>1. Prepare the vector</P><DIV class="JFigure"><P><B>Tablet 2.</B> Reaction system.</P></DIV><P>2. Prepare the insertion fragment</P><DIV class="JFigure"><P><B>Tablet 3.</B> Reaction system.</P></DIV><P>3. Obtain the plasmid PLVX-TfR</P><DIV class="JFigure"><P><B>Tablet 4.</B> Reaction system.</P></DIV><P>4.	We transformed the ligation product into competent E. coli.
		5.	We plated the culture solution on an Amp-resistant LB plate to get monoclonal.
		6.	We conducted colonal PCR verification.
		7.	We isolated the plasmid from E.coli</P><P>Results 
		The double digestion results of the restriction enzymes EcoR I and Xho I were shown in Fig 9. The 8787 bp band in lane 1 was the PLVX plasmid backbone cut by both enzymes. The 2313 bp band was the TfR fragment connecting the two enzyme cutting sites. The results showed that we have successfully obtained the TfR biobrick PLVX-TfR.</P><DIV class="JFigure"><P>Fig 9. Identification of recombinant PLVX-TfR.
 			M: DL10000 DNA Marker; 1: PLVX-TfR (digested with EcoRI and XbaI)</P></DIV></DIV></DIV><DIV class="row" id="susweek1"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Suspension Culture</H5><H4>Week 1</H4></DIV><P>Day1   BHK-21 and CHO-k1 adherent cells and suspension cells recovery. 
		The adherent cells of BHK-21 and CHO-K1 were cultured in T25 flasks, and the suspension cells of BHK-21 and CHO-K1 were cultured in 50ml centrifuge tubes.                     
		Day3   Passage of BHK-21 and CHO-K1 adherent cells and suspension cells.      
		Day5   Passage of BHK-21 and CHO-K1 adherent cells and suspension cells.
		Day7   BHK-21 adherent cells and CHO-K1 adherent cells were cultured in T25 flasks until 90% confluence. Then we extracted RNA from cells, transferred them to sterile 1.5 mL centrifugal tubes, and frozen them at – 80℃. Three samples were placed in parallel.
		We centrifuged BHK-21 suspension cells and CHO-K1 suspension cells at 1000rpm for 1 min respectively, and then removed the supernatant. In the same way, we extracted RNA from the cells, then transfer the liquid to sterile 1.5 mL centrifugal tubes and frozen them at – 80℃. Three samples were placed in parallel.
		RNA-Seq at Oui Biomedical Technology (Shanghai) Co. Ltd. took up about 1 month.</P></DIV></DIV><DIV class="row" id="susweek23"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Suspension Culture</H5><H4>Week 2-3</H4></DIV><P>We spent about 2 weeks to do RNA-Seq data analysis and mathematical modeling.
		(1) Raw data filtering;
		(2) Reference sequence alignment analysis;
		(3) Calculation of gene expression;
		(4) Screening of DEGs;
		(5) SNP analysis;
		(6) GO analysis and KEGG enrichment of DEGs;
		(7) KEGG enrichment analysis of DEGs;
		(8) GeneMANIA analysis of DEGs;</P><DIV class="JFigure"><P><B>Figure 1.</B> Venn diagram showing the number of genes in each analytical cohort.</P></DIV><DIV class="JFigure"><P><B>Figure 2.</B> Network showing interactions among candidate genes responsible for cell suspension feature. The 18 candidate genes were shown in the inner circle and other genes relevant with these candidates were retrieved by GeneMANIA and shown in the outer circle.</P></DIV></DIV></DIV><DIV class="row" id="susweek45"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Suspension Culture</H5><H4>Week 4-5</H4></DIV><P>Day1 qPCR detection 
		We used total RNA extraction kit to extract RNA from BHK-21 cells. After removing the genome, the mRNA of gene PABPC1 was reverse transcribed into cDNA. PABPC1 expression in BHK-21 adherent cells and BHK-21 suspension cells was detected by qPCR.</P><DIV class="JFigure"><P><B>Figure 3.</B> Gene expression of PABPC1 in BHK-21 adherent and suspended cells. PABPC1 level in BHK-21 suspended cells is 0.4 times of that in BHK-21 adherent cells.</P></DIV></DIV></DIV><DIV class="row" id="susweek6"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Suspension Culture</H5><H4>Week 6</H4></DIV><P>We used Lipofectamine 2000 to transfect cells with siRNA but failed as the transfection efficiency is too low. As shown from Figure 4, the relative mRNA expression of PABPC1 in BHK-21 adherent cells were quite different between using Lipofectamine 2000 and Lipofectamine 3000. Thus, we decided to use Lipofectamine 3000 in the following experiments.</P><DIV class="JFigure"><P><B>Figure 4.</B> PABPC1 expression using different transfection reagents. PABPC1 expression was much lower using Lipofectamine 3000, which indicated that Lipofectamine 3000 was better than Lipofectamine 2000 in siRNA transfection.</P></DIV></DIV></DIV><DIV class="row" id="susweek78"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Suspension Culture</H5><H4>Week 7-8</H4></DIV><P>Day1   Cell seeding：We seeded cells when cells’ confluence reached 30%-50% in logarithmic growth stage after cultured for 24 h.
		Day2   Transfection: ‘A’ solution was prepared by mixing 3ul Lipofectamine 3000 with 100ul fresh medium and then adding 0-100 nmol siRNA into 100ul fresh medium to prepare ‘B’ solution. ‘AB’ solution was mixed and incubated in a 6-well plate at room temperature for 1 min, then the supernatant was removed. Incubate cells overnight by adding 900ul complete medium in ‘AB’ solution.
		After transfection for 10-12 hours, the solution was changed to 2ml complete medium (DMEM FULL).
		Day3   After transfection, cells were harvested 48 hours later. Then we extracted RNA and detected the expression of PABPC1.</P><DIV class="JFigure"><P><B>Figure 5.</B> Efficacy of PABPC1 siRNA transfection at different concentrations</P></DIV><P>We used siRNA to inhibit PABPC1 expression in BHK-21 adherent cells and tested the efficacy of siRNAs at different concentrations. As shown from Figure 6, siRNA at the concentration of 50umol/L could effectively reduce PABPC1 to around 0.25 fold, which was used for the following experiments.</P></DIV></DIV><DIV class="row" id="susweek9"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>Suspension Culture</H5><H4>Week 9</H4></DIV><P>Day1   Cell seeding: We seeded cells when cells’ confluence reached 30%-50% in logarithmic growth stage after being cultured for 24 h.
		Day2   Transfection: ‘A’ solution was prepared by mixing 3ul Lipofectamine 3000 with 100ul fresh medium, ‘B’ solution was prepared by adding 50 nmol siRNA into 100ul fresh medium, ‘AB’ solution was mixed and incubated at room temperature for 1 min in a 6-well plate, then the supernatant was removed. Incubate cells overnight by adding 900ul complete medium in ‘AB’ solution.
		After transfection for 10-12 hours, the solution was changed to 2ml complete medium (DMEM FULL).</P><DIV class="JFigure"><P><B>Table 1.</B> The sequence of siRNA used in this study</P></DIV><P>Day3   After transfection, the cells were harvested 48 hours later.
       One part was used to extract RNA and detect the expression of PABPC1.
       The other part was used for suspension culture.
		Day4   Suspension culture cells were counted every day for 5 days, and the growth curve was drawn.</P><DIV class="JFigure"><P><B>Figure 6.</B> Growth curve of BHK-21 cells with reduced PABPC1 expression. BHK-21 adherent and suspended cells were used as the negative and positive control, respectively. After one day's adaptation, suspended and PABPC1 silenced cells entered the logarithmic growth stage. The highest cell density occurs at the 4th day which is the same between adherent and PABPC1 silenced cells and about 4 X 106 cells/ml. </P></DIV><P>Results: The growth curve of adherent cells and suspension cells inhibited by PABPC1 were similar. After one day's adaptation, they all entered the logarithmic growth stage. The highest cell density was reached on the 4th day at about 4106 cells/ml.</P></DIV></DIV><DIV class="row" id="highweek1"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>High Titer</H5><H4>Week 1</H4><P>Week 1 siRNA design</P></DIV><P>The sequences of siRNAs used to instantly inhibit IRF7 expression on MDBK cells are listed in Table 1.</P><DIV class="JFigure"><P><B>Table 1.</B> The siRNA sequences used to instantly inhibit IRF7 expression on MDBK cells.</P></DIV></DIV></DIV><DIV class="row" id="highweek23"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>High Titer</H5><H4>Week 2-3</H4><P>Week 2-3 siRNA efficacy examination</P></DIV><P>Detection of IRF7 expression after transfecting MDBK cells for 48 h</P><DIV class="JFigure"><P>SiIRF7-10nmol：SiRNA final concentration 10nmol/ml
 			SiIRF7-50nmol：SiRNA final concentration 50nmol/ml<B>Figure 1.</B> IRF7 expression after transfecting MDBK cells for 48 h.</P></DIV><P>The results showed that IRF7 expression was effectively suppressed using the designed siRNAs.</P></DIV></DIV><DIV class="row" id="highweek4"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>High Titer</H5><H4>Week 4</H4><P>Week 4 Virus titer test </P></DIV><DIV class="JFigure"><P>C：control    NC：negative control
			SiIRF7-10nmol：SiRNA final concentration 10nmol/ml
			SiIRF7-50nmol：SiRNA final concentration 50nmol/ml<B>Figure 2.</B> Fold changes of IBRV replication after inhibiting IRF7. Virus titer increased over 2 folds when siRNA concentration was 50nmol/ml.</P></DIV><P>folds when siRNA concentration was 50nmol/ml.
		The result showed that IBRV titer is significantly enhanced after suppressing IRF7 expression as detected using qPCR.</P></DIV></DIV><DIV class="row" id="highweek56"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>High Titer</H5><H4>Week 5-6</H4><P>Week 5-6 Fast virus titer examination approach establishment</P></DIV><P>The process of obtaining genomic DNA by dissociating viruses using standard kit（such as GenElute™ Mammalian Genomic DNA Miniprep Kits）s expensive and time-consuming, which is not suitable for high-throughput detection. Thus, we are motivated to develop a fast virus titer examination approach. This approach is established, with details described in our protocol.  
		Tables 3 and 4 show comparisons between the traditional and our fast detection methods.</P><DIV class="JFigure"><P><B>Table 2.</B>  Ct values of virus PCR examined using standard kit. ‘Ct’ refers to Cycle threshold, ‘Ct AVG’ means the average of Cycle threshold, ‘CV’ represents coefficient of variation.</P></DIV><DIV class="JFigure"><P><B>Table 3.</B> Ct values of virus PCR measured by our fast method. ‘Ct’ refers to Cycle threshold, ‘Ct AVG’ means the average of Cycle threshold, CV’ represents coefficient of variation.</P></DIV><P>The CV of our fast examination approach (CV = 0.9 and 0.71 for both samples) has less CV than the traditional approach (CV = 2.2 and 1.42 for both samples), suggesting that they are stable and reproducible. Further, compared with the traditional method, our fast method is less costly, easy to operate and time-saving.
		This approach is now under patent protection in China (201810377315.6) and under patent application in USA.</P></DIV></DIV><DIV class="row" id="highweek7"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>High Titer</H5><H4>Week 7</H4><P>Week 7 Cold atmospheric plasma parameter exploration</P></DIV><P>We firstly tested two treatment durations, i.e., 30s and 40s, with the design shown in Figure 3. 2ul virus was added to each hole and incubated with 1ml PAM for 1h. The PAM was replaced with 1ml complete medium. When cells completely exfoliated, the supernatant was collected and examined.</P><DIV class="JFigure"><P><B>Figure 3.</B> Plate design.</P></DIV><DIV class="JFigure"><P><B>Figure 4.</B> Fold change of virus titer after plasma treatment. Treatment time of 30s and 40s were compared.</P></DIV><P>DNA of IBRV replication increased nearly 2 folds after plasma treatment for 40s. However, it seems that the fold change has not reached the plateau. Thus, we decided to prolong the treatment duration in the following experiments. </P></DIV></DIV><DIV class="row" id="highweek8" style="margin-bottom:0;padding-bottom:2em;"><DIV class="col s7 offset-s2 Jshadow"><DIV class="Jcenter"><H5>High Titer</H5><H4>Week 8</H4><P>Week 8 Cold atmospheric plasma parameter exploration </P></DIV><P>We seeded cells in 12 hole board, with 200,000 / hole, and tested the efficacy of plasma on virus titer at five treatment time points, i.e., 60s, 90s, 120s, 150s and 180s. </P><P>We found that the highest virus titer achieves when the treatment time is 120s.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>