<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Jiangnan_China_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Jiangnan China/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV class="site-header py-2" style="position: fixed;width: 100%;z-index: 999"><DIV class="container d-flex flex-column flex-md-row justify-content-between" style="max-width: 1300px;"><A class="nav-link py-2 d-none d-md-inline-block" href="https://2018.igem.org/Team:Jiangnan_China"> Home</A><DIV class="dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> Team
          </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/TeamMembers">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Sponsors">Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Attributions">Attributions</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><A class="nav-link dropdown-toggle" href="#" id="projectDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> Project
          </A><DIV class="dropdown-menu" aria-labelledby="projectDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Experiments">Experiments</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Protocols">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Demonstrate">Demonstrate</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Model">Model</A></DIV></DIV><DIV class="dropdown"><A class="nav-link dropdown-toggle" href="#" id="projectDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> Safety
          </A><DIV class="dropdown-menu" aria-labelledby="projectDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Safety">Training</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Safety#Protection">Protection</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Safety#Material">Material</A></DIV></DIV><DIV class="dropdown"><A class="nav-link dropdown-toggle" href="#" id="partsDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> Human Pactices
          </A><DIV class="dropdown-menu" aria-labelledby="partsDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Human_Practices">Silver_Human Pactices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Human_Practices#gold">Gold_Human Pactices</A></DIV></DIV><DIV class="dropdown"><A class="nav-link dropdown-toggle" href="#" id="partsDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> Awards
          </A><DIV class="dropdown-menu" aria-labelledby="partsDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Awards">Bronzes</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Awards#Silver">Silver</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Awards#Gold">Gold</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Awards#Prizes">Prizes</A></DIV></DIV><DIV class="dropdown"><A class="nav-link dropdown-toggle" href="#" id="partsDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> Parts
          </A><DIV class="dropdown-menu" aria-labelledby="partsDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Parts">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Jiangnan_China/Parts#Composite">Composite parts</A></DIV></DIV><A class="nav-link py-2 d-none d-md-inline-block" href="https://2018.igem.org/Team:Jiangnan_China/InterLab"> InterLab</A></DIV></NAV><MAIN class="content-wrap"><DIV class="dcpt3" style="font-size:20px;line-height:1.5;font-family: 'spr';margin-top: -1px;"><DIV align="left" style="font-family: 'spr';font-size:40px;border-bottom:2px solid #584b4f;"><STRONG>Part  1. Selection of acid tolerance component ( <SPAN class="font-italic">msmK</SPAN> )</STRONG></DIV>    The key point of this part is selecting the gene related to anti-acid characteristic. Because the mechanism of acid resistance is complicated, and genes are connected to each other, it is hard to  confirm the most significant gene.
     Gnomic mutagenesis and high throughout screening are carried out to obtain anti-acid mutant strains with significant higher survival rate under specific acid environment (pH 4.0 ,5h). Then analyze the diff-genes data between mutant strain and parent strain on pH 4.0 and pH 7.0 respectively. A dimensionality reduction model is established and five possible acid-resistant genes are obtained. Finally, with experiment verification and pathway analysis, <SPAN class="font-italic">msmK</SPAN> gene is shown to be the key anti-acid gene.    <STRONG><FONT size="6">1. Genomic Mutagenesis and Highthroughput Screening</FONT></STRONG><DIV style="text-align:center;"><STRONG>Fig 1</STRONG> Screening procedures.</DIV><DIV style="text-align:center;">We take model strain <SPAN class="font-italic">Lactococcus lactis</SPAN> NZ9000 as parent strain to select</DIV>    <STRONG><FONT size="5">Chemical mutagenesis</FONT></STRONG>    The dose of DES for chemical mutagenesis was 0.5% (v·v-1), the treatment was 30 min, and the mortality rate was 85.3%. Under the condition of pH 5.0, two acid-resistant strains were screened from 20,000 strains, respectively. <SPAN class="font-italic">L. lactis</SPAN> WH101 and <SPAN class="font-italic">L. lactis</SPAN> WH102.    <STRONG><FONT size="5">UV mutagenesis</FONT></STRONG>    The UV mutagenesis conditions were UV lamp power of 15 W, irradiation time of 50 s, irradiation distance of 30 cm, and lethality rate of 92.1%. Under the condition of pH 5.0, the acid-resistant strain <SPAN class="font-italic">L. lactis</SPAN> WH103 was screened from 15000 strains.    <STRONG><FONT size="5">High throughput screening</FONT></STRONG>    With high throughput screening, we got 3 mutants that can survive at pH 5.0 for 4 hours, and we named them as <SPAN class="font-italic">L. lactis</SPAN> WH101、<SPAN class="font-italic">L. lactis</SPAN> WH102、<SPAN class="font-italic">L. lactis</SPAN> WH103.    <STRONG><FONT size="6">2. Compare the growth performance of acid-tolerant strains</FONT></STRONG>    According to the 2% inoculation rate, the original strain <SPAN class="font-italic">L. lactis</SPAN> NZ9000, three mutant strains <SPAN class="font-italic">L. lactis</SPAN> WH101, WH102, WH103 were inoculated into 10 ml of liquid medium under the conditions of pH 4.5, 5.0, and 7.0, respectively. The OD value of the strains were measured every 0.5 hours, and the growth curve of the strains are shown as Fig 2.<DIV style="text-align:center;"><STRONG>Fig 2</STRONG> (A) The growth curve of the four strains at pH 7.0; (B) The growth curve of the four strains at pH 5.0; (C) The growth curve of the four strains at pH 4.5; (D)The survival rate of the four strains at pH 4.0.</DIV>    <STRONG><FONT size="5">Results analysis</FONT></STRONG>
      1)  The growth of acid-resistant strains under normal pH 7.0 conditions was not significantly affected.
      2)  The growth performance of acid-resistant strains at pH 5.0 and pH 4.5 was significantly improved compared to the original strain. 
      3)  At pH 5.0, the biomass of the acid-tolerant strains WH101, WH102, WH103 was 7.0, 4.8 and 5.6 times that of the original strain, respectively. 
      4)  At pH 4.5, they were 5.5, 3.6, and 4.3 times the original strain, respectively.
      5)  Under the condition of pH 4.0, the survival rate of <SPAN class="font-italic">L. lactis</SPAN> WH101, <SPAN class="font-italic">L. lactis</SPAN> WH102 and <SPAN class="font-italic">L. lactis</SPAN> WH103 was significantly higher than that of the original strain. And the survival rate increased gradually with the prolongation of stress time and acid resistance after 5 hours of stress. The strains were 16000, 351.4 and 264.3 times the original strain, respectively.    Finally, through genomic mutation and high-throughput screening, we obtained a strain <SPAN class="font-italic">L. lactis</SPAN> WH101 with good acid resistance from 35,000 strains. Next, we will compare the NZ9000 and WH101 by transcriptomics analysis and mathematical model to obtain key acid resistant components.    <STRONG><FONT size="5">Transcriptomics analysis and modeling</FONT></STRONG><P style="text-align: center;"><A href="https://2018.igem.org/Team:Jiangnan_China/Model" class="btn btn-info">Model </A></P></DIV><DIV id="dcpt4" style="font-size:20px;line-height:1.5;font-family: 'spr';"><DIV align="left" style="font-family: 'spr';font-size:40px;border-bottom:2px solid #584b4f;"><STRONG>Part 2. Add anti-cold gene ( <SPAN class="font-italic">cspD2</SPAN> ) </STRONG></DIV>    According to previous research, we know that <SPAN class="font-italic">cspD2</SPAN> can express cold shock protein in lactic acid bacteria. Therefore, we obtained the <SPAN class="font-italic">cspD2</SPAN> fragment by in vitro synthesis, and constructed the <SPAN class="font-italic">cspD2</SPAN> overexpression strain. Verify the cold resistance performance of the recombined bacteria with nisin as inducer.</DIV><DIV class="dcpt3" style="font-size:20px;line-height:1.5;font-family: 'spr';margin-top:-1px; "><DIV align="left" style="font-family: 'spr';font-size:40px;border-bottom:2px solid #584b4f;"><STRONG>Part 3. Demonstration</STRONG></DIV>    Because we use <SPAN class="font-italic">nisA</SPAN> as promoter to express our parts, 0.5 ng/mL of Nisin is required to induce cell expression in all the following steps.    <STRONG><FONT size="5">Demonstration of <SPAN class="font-italic">msmK</SPAN> gene</FONT></STRONG>    Copy <SPAN class="font-italic">msmK</SPAN> gene from <SPAN class="font-italic">Lactococcus lactis</SPAN> NZ9000 gene group with PCR. Construct <SPAN class="font-italic">msmK</SPAN> overexpression strain <SPAN class="font-italic">L.lactis</SPAN> NZ3900/pNZ8149-<SPAN class="font-italic">msmK</SPAN> with electrotransformation. Take strains <SPAN class="font-italic">L.lactis</SPAN> NZ3900/pNZ8149-<SPAN class="font-italic">msmK</SPAN> and <SPAN class="font-italic">L.lactis</SPAN> NZ3900/pNZ8149 from glycerin tube kept under -80℃, dilute 1/25 in 2 x 10 ml fresh medium (30 °C). Grow until the OD600=0.4. Count their number of colonies per ml at pH 4.0 (More details can be seen in protocol).    We can get the following results:<DIV style="text-align:center;"><STRONG>Fig 3</STRONG> Number of colonies at acid stress (pH 4.0).</DIV><DIV style="text-align:center;"><STRONG>Fig 4</STRONG> Survival rate at acid stress (pH 4.0).</DIV>    The result shows that <SPAN class="font-italic">msmK</SPAN> is an effective anti-acid gene, which can make the recombinant strain has <STRONG>213-fold</STRONG> higher survival rate than parent one.    <STRONG><FONT size="5">Demonstration of composite part <SPAN class="font-italic">msmK-cspD2</SPAN></FONT></STRONG>    It has been reported that <SPAN class="font-italic">cspD2</SPAN> can effectively exert anti-freezing effect in the recipient bacteria. <SPAN class="font-italic">CspD2</SPAN> was ligated to the pNZ8149/<SPAN class="font-italic">msmK</SPAN> plasmid by one-step cloning (seamless ligation), and the recombinant plasmid was introduced into the constructed <SPAN class="font-italic">L. lactis</SPAN> NZ3900/pNZ8149-<SPAN class="font-italic">msmk</SPAN>-cspD2 strain using electroporation.     The <SPAN class="font-italic">gfp</SPAN> gene was inserted as a marker gene, and cell viability was characterized by fluorescence intensity. The strain was tested for acid resistance and freezing resistance using a flow cytometer. The process of acid stress and cold stress is similar with the above demonstration process.    <STRONG><FONT size="5">Acid stress </FONT></STRONG>    Deal with the samples under pH 4.0 with nisin as an inducer.<DIV style="text-align:center;"><STRONG>Fig 5</STRONG> Number of colonies at acid stress (pH 4.0).</DIV><DIV style="text-align:center;"><STRONG>Fig 6</STRONG> Survival rate at acid stress (pH4.0).</DIV>    The result shows that composite part can make the recombinant strain has <STRONG>243-fold</STRONG> higher survival rate than parent one under acid stress.<DIV style="text-align:center;"><STRONG>Fig 7</STRONG> Electron microscopy of <SPAN class="font-italic">L.lactis</SPAN> NZ3900/pNZ8149-<SPAN class="font-italic">msmk-cspD2-gfp</SPAN> and <SPAN class="font-italic">L.lactis</SPAN> NZ3900/pNZ8149 before and after acid stress.</DIV>    Before the acid stress, the cell structure of the control strain and the recombinant strain remained intact. After 3 h of pH 4.0 stress, the cell membrane thickness became thinner and the surface became rough, and the cell membrane of some control strains ruptured. In comparison, the cell structure of the recombinant strain remains more intact, thereby effectively reducing the damage caused by acid stress on the cells.    <STRONG><FONT size="5">Cold stress </FONT></STRONG>    <SPAN class="font-italic">L.lactis</SPAN> NZ3900/pNZ8149-<SPAN class="font-italic">msmk-cspD2-gfp</SPAN> and <SPAN class="font-italic">L.lactis</SPAN> NZ3900/pNZ8149 strains were inoculated with 4% inoculation. When cultured at 30 °C for 2.5 h (OD=0.35), add 0, 0.5 ng/mL of Nisin, and then culture for 8 ∼ 10 h (OD=0.8), centrifuge at 4000 r/min. Resuspend them in the same volume of fresh M17 medium and count the number of colonies. Four freeze-thaw stimulations were performed on all samples by 1 mL of the sample after counting, and placed in a refrigerator at −20 °C to cool rapidly, frozen for 24 h, then slowly frozen at 4 min and 30 °C. Count separately and calculate the survival rate.<DIV style="text-align:center;"><STRONG>Fig 8</STRONG> The Comparison curve of survival rate under cold stress.</DIV>    After 4 consecutive repeated freeze-thaw tests, the recombinant strain was <STRONG>47.5 times</STRONG> more viable than the control strain, indicating the antifreeze survival rate of the strain with increased overexpression of <SPAN class="font-italic">cspD2</SPAN>.<DIV style="text-align:center;"><STRONG>Fig 9</STRONG> Electron microscopy before and after repeated freezing and thawing.</DIV>    Before freezing and thawing, the cell structure of the control strain and the recombinant strain remained intact. After repeated freezing and thawing for 4 times, the cell membrane of the cell became thin and rough, and some intracellular substances overflowed. In comparison, the cell structure of the recombinant strain remains more intact, and the damage of the cell membrane is alleviated, thereby effectively reducing the damage caused by freezing stress on the cells.</DIV></MAIN><FOOTER class="py-4"><DIV class="container" style="text-align: center;"><P>Copyright © jiangnan_China 2018</P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>