"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Eindhoven/Achievements/Submitted Parts/COMPx
From 2014.igem.org
iGEM Team TU Eindhoven 2014
Home
Team
Team Members
Supervisors
Acknowledgements
Sponsors
Project
Background
SPAAC Reaction
Membrane Anchors
FACS
Orthogonal tRNA System
Abstract
Overview Click Coli
Project Description
Design
Membrane Anchor Design
Plasmid Design
Characterization
Click Coli
Cell Viability
Photoreactivity
Fluorescence Microscopy
Rolling Circle Amplification
Introduction
Attaching DNA to DBCO-PEG4
Testing Product
Creating Circular RC Template
RCA on Cells
ZAP
Manual
Achievements
Safety
Our Lab
Biosafety
Kill Switch
Main Results
Submitted Parts
COMPx
COMPy
tRNA Synthetase
Medal Motivation
Future Prospects
Microfluidics
Introduction
Methods
Droplet Device
Cell Encapsulation Device
Results
Results Droplet Device
Results Cell Encapsulation Device
Modeling
Cell Encapsulation Modeling
Bacterial Cell Counter
Society
Synenergene
Introduction Synenergene
Application Scenario
Techno-Moral Vignettes
RIVM Workshop
Outreach
Importance of Outreach
Informal Information Evening
iGEM The Netherlands /li>
Media Attention
Policy & Practices
Children's Book
Workshop Game
Notebook
Timeline
Protocols
iGEM Team TU Eindhoven 2014
COMPx - BBa K1492000
Figure 1. Structure of OmpX alongside a topological depiction of OmpX and CPX.
Usage and Biology
CPX, or Circularly permuted OmpX, was developed as a bacterial display methodology for N- and C- terminal display. It has been shown to enable rapid screening of very large peptide libraries with high precision and efficiency. OmpX possesses four extracellular loops, with loops 2 and 3 forming a semi rigid β-sheets protruding from the cell surface. The native N- and C-termini were fused together by a GGSG linker, and the newly formed N and C termini reside on the cell surface.
CPX was created by Rice et al as a scaffold for peptide libraries. Rice et al added a part at the N terminus that is used for the random mutation needed for Peptide Libraries. Furthermore Rice et al shows that CPX can easily be overexpressed without effecting the cell growth or cell viability. This makes CPX a very useful protein for any type of displaying.
Gene Design
Figure 2. Gene design of COMPx.
To integrate a Non-natural amino acid in the sequence of CPX some modifications were made. Within the part in which originally random mutations were induced a codon was mutated into the amber stop codon TAG. With a specific tRNA it is possible to implement a Non-Natural amino acid at the place of the TAG codon. The TAG codon was introduced into the Peptide Library by Site-Directed Mutagenesis. A HA-tag was also added to the C-terminus of the protein using overhang primers to use in characterization. After the modifications the protein was renamed to Clickable Outer Membrane Protein X (COMPx).
Characterization
Team TU Eindhoven 2014 tried to create a membrane anchor to which molecules could covalently bind. With this protein it is possible to bind anything to the membrane by using a bio-orthogonal “click” reaction. To test the functionality of the protein several assays were done.
For all the assays we used the following vectors pET29a(+) COMPx (Membrane Protein) and pEVOL-PylT-2xTyrRS. (tRNA, tRNA synthetase). Both vectors were transformed into BL21(DE3) strain. Colonies of this transformation were grown on LB. After culturing, glycerol stocks were made. All the assays were done by culturing the bacteria from this glycerol stock. Check our Protocol Page for all the protocols of TU Eindhoven 2014.
Figure 3. COMPx labeled with Anti-HA antibody.
Antibody Confirmation
First assay done was to confirm that the protein was expressed on the membrane. This was done by adding an anti-HA antibody labeled with TAMRA dye to bacteria in solution that had been expressing COMPx. The bacteria were then analyzed with FACS. Results show that COMPx is expressed on the membrane of the bacteria ( Figure 3 ). For the used protocol see Antibody Labelling .
Figure 4. COMPx labeled with DBCO-PEG4-TAMRA.
DBCO-PEG4-TAMRA Confirmation
Now that it is proven that the protein is expressed on the membrane the next step can be taken. This assay is to see whether or not the non-natural amino acid is being incorporated into COMPx. For labeling DBCO-PEG4-TAMRA was used. If the non-natural amino acid is present in the protein then DBCO-PEG4-TAMRA should be measured after washing steps. DBCO-PEG4-TAMRA was added in two different concentrations, in excess, to bacteria in solution that had been expressing COMPx. The bacteria were then analyzed with FACS.  Data clearly shows that DBCO-PEG4-TAMRA is still on the membrane after washing steps. ( Figure 4 ) Therefore it was concluded that the DBCO of the dye is clicked to the Azide, in other words: the dye has covalently bound to the protein. For the used protocol see FACS sorting with DBCO-PEG(10kDa) and FACS sorting with DBCO-TAMRA .
Figure 5. Antibody titration of COMPx
Antibody Titration
In order to verify expression of our designed outer membrane protein COMPx and to obtain further insight the degree of expression, we performed an antibody titration. Out of this titration a Kd can be calculated if the upper and lower plateau is measured. With the Kd it is possible to calculate the number of anchors on the membrane.  This was an important assay for the TU Eindhoven 2014 team to determine how many binding sites there were per bacteria. For this titration several concentration of anti-HA antibody have been used. Bacteria have been analyzed with FACS. For the used protocol see Antibody Titration with FACS .
Unfortunately the upper plateau was not reached. Therefore it is not possible to calculate a Kd from this data. Reasons for this can be that the batch of antibodies has a lower binding affinity than usual. Several other researchers at the TU Eindhoven experienced the same problem with this batch of anti-HA antibodies.
Influence Non-Natural Amino Acid on the Expression of COMPx
To be able to determine the influence of the Non Natural amino acid on COMPx a small modification had to be made to the protein. The TAG codon has been mutated into a TTG codon, which codes for a serine. The mutated protein is called COMPx noTAG. After mutation the new plasmid has been transformed into a BL21 strain and cultured. The original COMPx was also cultured. Protein expression has been induced and the pAzF amino acid has been given to both cultures. After expression both types were labeled with Anti-HA antibody in two concentrations and have been analyzed with the FACS. For the used protocol see Antibody Labelling .
Figure 6. COMPx labeled with Anti-HA antibody.
Figure 7. COMPx noTAG labeled with Anti-HA antibody.
Overall the expression of COMPx is a bit lower than the expression of COMPx noTAG, the peak fluorescence of COMPx is around 1x103 ( Figure 6 ) while the peak of COMPx noTAG is around 2x103 ( Figure 7 ). Hower the frequency of COMPx is a alot higher than the frequency of COMPx noTAG
Cell Viability
An essential part of the characterization of COMPx is to see whether the bacteria live after clicking chemicals to their membrane. This is done to analyze if the bacteria can handle the stress of overexpressing this membrane protein and the stress on the membrane due to the large molecules clicked to it. For this assay two groups of bacteria were grown on agar plates. The first group has been incubated with DBCO PEG 10kDa which reacts with pAzF. The second group has been incubated with PEG 3350 Da which cannot react with pAzF. The bacteria were put on agar plates after incubation in three different concentrations. The bacteria were grown for ~16 hours. This assay has been done twice. For the used protocol see Protocol Cell viability .
Table 1. Results cell viability assay of COMPx.
In this case it is obvious that a part of the cells is still viable in both the reactions. As seen in the results in Table 1 there is a small difference between the viability of the cells clicked with DBCO-PEG compared to those without this molecule. However based on the small amount of data and the method of registering the colonies more research is needed to confirm this difference.  With this assay it is confirmed that the membrane can take the stress caused by the molecules clicked on the membrane and are able to survive. It can be concluded that the cells are viable after protein expression and the click-reaction.
Bibliography
Rice, J. J. "Bacterial display using circularly permuted outer membrane protein OmpX yields high affinity peptide ligands." Protein Science 15.4 (2006): 825-836.
iGEM Team TU Eindhoven 2014
Created by iGEM Team TU Eindhoven 2014
Retrieved from " http://2014.igem.org/Team:TU_Eindhoven/Achievements/Submitted_Parts/COMPx "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
