"
Page
Discussion
View source
History
teams
Log in
Â 
Team:TU Eindhoven/Background/Orthogonal
From 2014.igem.org
iGEM Team TU Eindhoven 2014
Home
Team
Team Members
Supervisors
Acknowledgements
Sponsors
Project
Background
SPAAC Reaction
Membrane Anchors
FACS
Orthogonal tRNA System
Abstract
Overview Click Coli
Project Description
Design
Membrane Anchor Design
Plasmid Design
Characterization
Click Coli
Cell Viability
Photoreactivity
Fluorescence Microscopy
Rolling Circle Amplification
Introduction
Attaching DNA to DBCO-PEG4
Testing Product
Creating Circular RC Template
RCA on Cells
ZAP
Manual
Achievements
Safety
Our Lab
Biosafety
Kill Switch
Main Results
Submitted Parts
COMPx
COMPy
tRNA Synthetase
Medal Motivation
Future Prospects
Microfluidics
Introduction
Methods
Droplet Device
Cell Encapsulation Device
Results
Results Droplet Device
Results Cell Encapsulation Device
Modeling
Cell Encapsulation Modeling
Bacterial Cell Counter
Society
Synenergene
Introduction Synenergene
Application Scenario
Techno-Moral Vignettes
RIVM Workshop
Outreach
Importance of Outreach
Informal Information Evening
iGEM The Netherlands /li>
Media Attention
Policy & Practices
Children's Book
Workshop Game
Notebook
Timeline
Protocols
iGEM Team TU Eindhoven 2014
Orthogonal tRNA System
Normally, all proteins produced by a cell are built with the 20 amino acids naturally present in cells. However, protein engineering has made the incorporation of unnatural amino acids possible. This is done by means of an orthogonal tRNA aminoacyl-tRNA synthetase and tRNA pair. An orthogonal system does not interfere with any other system in the cell. The anticodon on the tRNA corresponds with the TAG codon (amber stop codon). This codon has to be placed in the gene of interest to incorporate the unnatural amino acid.
The aminoacyl-tRNA synthetase is specifically modified to bind the unnatural amino acid to the orthogonal tRNA. The tRNA, in response to an amber stop codon(UAG) on the mRNA, incorporates the unnatural amino acid into the amino acid sequence. (
Figure 1
) [1]
Figure 1. The incorporation of an unnatural amino acid into a protein.
Figure 2. Chemical
structure of pAzF.
Unnatural Amino Acid p-Azido-L-Phenylalanine
The unnatural amino acid used to incorporate an azide in the anchor proteins is p-Azido-L-phenylalanine (pAzF). pAzF is a photocrosslinker which can be incorporated in any protein, irrespective of its size or sequence, by a tRNA synthetase/tRNA pair and the amber codon TAG. The amino acid is incorporated in good yield with high fidelity and can be used to crosslinks interacting proteins.
Both CPX and INPNC meet the requirements for an anchor protein. They are able to display certain unnatural amino acids (in this case pAzF) on the outside of the cell, where the reaction with the DBCO-conjugate takes place.[2]
Bibliography
[1]Davis, Lloyd, and Jason W. Chin.(2012) "Designer proteins: applications of genetic code expansion in cell biology."Nature Reviews Molecular Cell Biology 13 : 168-182.
[2]Chin, Jason W., Stephen W. Santoro, Andrew B. Martin, David S. King, Lei Wang, and Peter G. Schultz. (2002) "Addition of p-Azido-L-phenylalanine to the Genetic Code of Escherichia coli." J. Am. Chem. Soc. 124.31 : 9026-9027
iGEM Team TU Eindhoven 2014
Created by iGEM Team TU Eindhoven 2014
Retrieved from " http://2014.igem.org/Team:TU_Eindhoven/Background/Orthogonal "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
