"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Eindhoven/Achievements/Submitted Parts/COMPy
From 2014.igem.org
iGEM Team TU Eindhoven 2014
Home
Team
Team Members
Supervisors
Acknowledgements
Sponsors
Project
Background
SPAAC Reaction
Membrane Anchors
FACS
Orthogonal tRNA System
Abstract
Overview Click Coli
Project Description
Design
Membrane Anchor Design
Plasmid Design
Characterization
Click Coli
Cell Viability
Photoreactivity
Fluorescence Microscopy
Rolling Circle Amplification
Introduction
Attaching DNA to DBCO-PEG4
Testing Product
Creating Circular RC Template
RCA on Cells
ZAP
Manual
Achievements
Safety
Our Lab
Biosafety
Kill Switch
Main Results
Submitted Parts
COMPx
COMPy
tRNA Synthetase
Medal Motivation
Future Prospects
Microfluidics
Introduction
Methods
Droplet Device
Cell Encapsulation Device
Results
Results Droplet Device
Results Cell Encapsulation Device
Modeling
Cell Encapsulation Modeling
Bacterial Cell Counter
Society
Synenergene
Introduction Synenergene
Application Scenario
Techno-Moral Vignettes
RIVM Workshop
Outreach
Importance of Outreach
Informal Information Evening
iGEM The Netherlands /li>
Media Attention
Policy & Practices
Children's Book
Workshop Game
Notebook
Timeline
Protocols
iGEM Team TU Eindhoven 2014
COMPy - BBa K1492001
Usage and Biology
In 2012, the iGEM team of Penn, University of Pennsylvania, created a generalized BioBrick surface display platform: the Ice Nucleation Protein N-, C-termini only (INPNC), BioBrick part: BBa_K811005 .
INPNC is a truncated Ice Nucleation Protein (INP, a protein which causes ice nucleation and formation but which is also used for its surface display properties). However, INP consists of a C-terminal region that is positioned on the outer membrane, as well as a central 8, 16 or 48 amino acid motif that is responsible for INP’s ice nucleation properties. These central amino acids motifs are not essential for INP’s surface displaying properties. Therefore INP was truncated, retaining only the N- (179 aa) and C-termini (49 aa) to create INPNC. The C-terminal domain is displayed on the cell surface, while the N-terminal domain remains in the outer membrane.
Gene Design
Figure 1. Gene design of COMPy.
The protein has a TAG codon and HA-tag added at the C-terminus of the protein. The TAG codon is for the incorporation of an unnatural amino acid. With the proper tRNA synthetase any Non Natural Amino Acid can be incorporated into the protein and displayed on the membrane. The HA-tag was used for characterization of the protein. Both modifications have been made by using an overhang primer. After the modification the protein has been renamed to Clickable Outer Membrane Protein y (COMPy).
We as team TU Eindhoven 2014 state that we have improved the functionality and “ease of use” of BioBrick part: BBa_K811005 . The functionality has been improved by adding the TAG. This allows the incorporation of any non-natural amino acid. The “ease of use” has been improved by the introduction of a HA-tag to the protein. This tag makes it a lot easier to screen whether or not the protein is expressed using fluorescent anti-HA antibodies.
Characterization
Team TU Eindhoven 2014 tried to create a membrane anchor to which molecules could covalently bind. With this protein it is possible to bind anything to the membrane by using a bio-orthogonal “click” reaction. To test the functionality of the protein several assays were done.
For all the assays we used the following vectors pET29a(+) COMPy (Membrane Protein) and pEVOL-PylT-2xTyrRS. (tRNA, tRNA synthetase). Both vectors were transformed into BL21(DE3) strain. Colonies of this transformation were grown LB. After culturing glycerol stocks were made. All the assays were done by culturing the bacteria from this glycerol stock. Check our Protocol Page for all the protocols of TU Eindhoven 2014.
Figure 2. COMPy labeled with Anti-HA antibody.
Antibody Confirmation
First assay done was to confirm that the protein was expressed on the membrane. This was done by adding an anti-HA antibody labeled with TAMRA dye to bacteria in solution that had been expressing COMPy. The bacteria were then analyzed with FACS. Results show that COMPy is expressed on the membrane of the bacteria ( Figure 2 ). Though it remains a small increase, reasons for this can be that the bacteria are having trouble expression the protein itself or the non-natural amino acid causes problems for expression. For the used protocol see Antibody Labelling .
Figure 3. COMPy labeled with DBCO-PEG4-TAMRA.
DBCO-PEG4-TAMRA Confirmation
Now that it is proven that the protein is expressed on the membrane the next step can be taken. This assay is to see whether or not the non-natural amino acid is being incorporated into COMPy. For labeling DBCO-PEG4-TAMRA was used. If the non-natural amino acid is present in the protein then DBCO-PEG4-TAMRA should be measured after washing steps. DBCO-PEG4-TAMRA was added in two different concentrations, in excess, to bacteria solution after expression of COMPy was induced. The bacteria were then analyzed with FACS.  Data clearly shows that DBCO-PEG4-TAMRA is still on the membrane after washing steps ( Figure 3 ). Therefore the DBCO of the dye is clicked to the azide, in other words: the dye has covalently bound to the protein. The fact that there is also a lot of signal at a lower fluorescence (around 102) goes to show that there is a lot of variance between bacteria expressing COMPy. A peak at high fluorescence is what is expected when the bacteria produce the protein well. For the used protocol see FACS sorting with DBCO-PEG(10kDa) and FACS sorting with DBCO-TAMRA .
Influence Non-Natural Amino Acid on the Expression of COMPy
To be able to determine the influence of the Non Natural amino acid on COMPy a small modification had to be done to the protein. The TAG codon has been mutated into a TTG codon, coding for a serine. The mutated protein is called COMPy noTAG. After mutation the new plasmid has been transformed into a BL21 strain and cultured. The original COMPx was also cultured. Protein expression has been induced and the pAzF amino acid has been given to both cultures. After expression both types were labeled with Anti-HA antibody in two concentrations and have been analyzed with the FACS. For the used protocol see Antibody Labelling .
Figure 4. COMPy labeled with Anti-HA antibody.
Figure 5. COMPy noTAG labeled with Anti-HA antibody.
The expression of COMPy is lower than the expression of COMPy noTAG, in Figure 5 the peaks are at a higher fluorescence than the peaks in Figure 4 . Furthermore this data shows that COMPy and COMPy noTAG do not express well in general. Even for COMPy noTAG, which does not use the non-natural Amino acid, the expression is not that good. When a protein is well expressed a single peak at high fluorescence is expected.
iGEM Team TU Eindhoven 2014
Created by iGEM Team TU Eindhoven 2014
Retrieved from " http://2014.igem.org/Team:TU_Eindhoven/Achievements/Submitted_Parts/COMPy "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
