Team:XJTLU-CHINA/Model
Toggle navigation
Home
Project
Description
Detection
Peptide Production
Cell Lysis
Modeling
Demonstration
Application
Parts
Basic Parts
Composite Parts
Notebook
Lab Book
Protocols
Human Practice
HP_Gold
HP_Silver
Team
Members
Attributions
Collaboration
Safety
Interlab
Modeling
Modeling
Modeling on the sensing device
In the mathematical modeling of quorum sensing, we formulated a system of ordinary equations representing the intracellular and extracellular interactions between the two Agr proteins and AIP (auto-inducing peptide) molecules. Along with numerical simulations, we performed an asymptotic analysis of the time-dependent model in order to characterize whether the AIP molecules produced by Staphylococcus aureus in the intestine would activate our sensing device.
To build the model, we first proposed the following assumptions:
Proteins and mRNA inside the cells are limited by natural degradation.
Housekeeping phosphatases are able to dephosphorylate AgrA at rate αpidi.
Receptor-bound AIP can dissociate spontaneously at rate αunbind.
When an AIP binds to AgrC, we assume that auto-phosphorylation of AgrC happens simultaneously because this process is sufficiently fast. When AgrC transfers its phosphate group to AgrA at rate αpi, it is able to re-auto-phosphorylate.
The resulting equations, together with the definitions of the parameters and variables are shown below.
Table 1 Definitions of the parameters
Parameters
Rate constant for
Value
Units
Note
αpi
Phosphorylation of AgrA
10[1]
μmol-1 ml-1 h-1
αpidi
Dephosphorylation of AgrA
1[1]
h-1
μx
Degradation and dilution
2[1]
h-1
μsfGFP
Degradation of sfGFP
0.378
h-1
Assume the same as GFP
μAm, μCm, μsfGFPm
Degradation of mRNA
17.28[4]
h-1
αcbind
AgrC that anchors to the cell membrane
10
μmol-1 ml-1 h-1
Assume the same as αpi
αbind
Binding of AIP to AgrC
1[1]
μmol-1 ml-1 h-1
αunbind
Separation of AIP from AgrC
0.1[1]
h-1
Parameters
Definitions
Value
Units
Note
X
Nisin
1.42×10-7[2]
μmol ml-1
k2
The Phosphorylated AgrA concentration required for half-maximal transcription rate of P2
1[1]
μmol  ml-1
β1
Maximum transcription rate of pnisA
10
μmol  h-1
Assume the same as β2
β2
Maximum transcription rate of P2
10[1]
μmol h-1
lpinsA
Leakage factor of pinsA
0.02
-
Assume the same as lptet
lP2
Leakage factor of P2
0.02
-
Assume the same as lptet
a
Translation rate
61200[3]
Amino acid residues h-1
SA
Length of AgrA
207
Amino acid residues
SC
Length of AgrC
413
Amino acid residues
SsfGFP
Length of sfGFP
237
Amino acid residues
Table 2 Definitions of the variables
Variables
Concentration of
Units
Am
mRNA of AgrA
μmol  ml-1
A
AgrA
μmol  ml-1
Cm
mRNA of AgrC
μmol  ml-1
C
AgrC
μmol ml-1
Cbind
AgrC that anchors to the cell membrane
μmol ml-1
AIP
Free AIP molecules
μmol ml-1
Cp
AIP-bound AgrC
μmol ml-1
Api
The phosphorylated AgrA
μmol ml-1
sfGFPm
mRNA of sfGFP
μmol ml-1
sfGFP
The product of P2 promoter
μmol ml-1
The three Hill equations represent the rates of transcription of agrA, agrC and sfGFP genes. β1 is the highest efficiency for the promoter pnisA to initiate the transcription of the agrC and agrA genes, and β2 is the highest efficiency for the promoter P2 to initiate the transcription of the sfGFP gene. X is the concentration of nisin which is needed to activate the promoter pnisA, to this extent, k1 equals to the concentration of nisin when the rate of reaction is up to half of Vmax. K2, which is controlled by another regulatory factor, is the concentration of phosphorylated AgrA when the rate of reaction is up to half of Vmax.
By assuming that 0.25 μM of AIP molecules is present in the intestine, we run the MATLAB script to check whether AIP molecules can successfully activate the promoter P2 by binding to AgrC and phosphorylating AgrA. We used a reporter protein sfGFP to show the activation of the sensing device. If there is an apparent increase of sfGFP concentration, the sensing device can be considered as being activated. The results are shown below.
Fig 1.    State values of AgrA, Cbind, AgrC, Cp, Api and sfGFP.
Fig 2.    Individual display of 9 variables
As it is shown in the Figure 1 and 2, the concentration of sfGFP is enough high after several hours. Therefore, we made a conclusion that the amount of AIP molecules can activate the promoter P2 to transcribe the genes downstream.
To verify whether the cross-inhibition （introduced in the description section）can result in a decrease in AIP and             virulence factor in Staphylococcus aureus, we hijacked the modeling of the sensing device into the quorum sensing system. By             assuming the concentration of AIPs decreases to 0.1 μM by the cross-inhibition of another type of AIPs, we run             the MATLAB script again to check whether there is any change in sfGFP production.
Fig 3.    State values of 9 variables.
Fig 4.    Individual display  of 9 variables.
Discussion
As shown in Figures 3 and 4, the concentration of sfGFP is extremely low when compared with the case of 0. 25 μM             AIPs. Therefore, the amount of AIP molecules can dramatically affect the P2 promoter to express the downstream             genes. Based on the result of sfGFP expression, we can also make a conclusion that in the case of the quorum             sensing of Staphylococcus aureus, their signal transduction of AIPs can be cross-inhibited by a different type of AIPs.             As a result, AIPs and virulence factors cannot be further produced.
Modelling on peptide synthesis and cell lysis
Our design uses the tandem repeat strategy to express three copies of each peptide gene, LL-37, GF-17 and Grammistin-Pp1, aiming to produce peptides with a higher rate. To release the peptides to kill Staphylococcus aureus in the intestine, we choose lysis of the cells instead of secretion. A lysis gene is used to open up the cells, then all the peptides will surely be released into the guts. In addition, we plan to use a toggle switch to provide more time for peptide synthesis before lysis. When the cells are lysed, it will result in the release of intracellular proteins and stop all life activities. Therefore, we use modeling to identify:
How much time can the toggle switch provide for the accumulation of AMPs？
Results: Inspired by the team TU-Delft (2013), we simplified the promoters P2 to serve as a binary switch between the active and inactive promoter states instead of continuous activities from fully on to fully off. We used the parameter--s, a binary state descriptor, to refer to the situation when a promoter produces one of the two levels of activity: on or off. In addition, the original amount of TetR  was assumed to be 200 μmol. Due to the unknown concentration of AcmA to lyse the cells, this value was assumed to be 50 μmol.
Table 3 Definitions of parameters
Parameters
Definitions
Value
Units
Note
a
translation rate per amino acid
1020[3]
Amino acids residues min-1
cP2
maximum transcription rate of P2
0.17[1]
μmol  min-1
cptet
maximum transcription rate of ptet
2.79[3]
μmol-1 min-1
cplac
maximum transcription rate of plac
2.79
μmol-1 min-1
Assume the same as cptet
dmRNA
degradation rate of mRNA
0.288[4]
min-1
dLacl
degradation rate of Lacl
0.1386[4]
min-1
dTetR
degradation rate of TetR
0.1386[4]
min-1
dAcmA
degradation rate of AcmA
0.033
min-1
Assume the same as μx
dGFn
degradation rate of GFn
0.011
min-1
Assume the one-third of μx
dGram
degradation rate of Gran
0.011
min-1
Assume the one-third of μx
dLLn
degradation rate of LLn
0.011
min-1
Assume the one-third of μx
lP2
Leakage factor of P2
0.002
-
Assume the same as lptet
lptet
Leakage factor of ptet
0.002[3]
-
lplac
Leakage factor of plac
0.002
-
Assume the same as lptet
SLacl
length of Lacl
371
Amino Acid residues
STetR
length of TetR
226
Amino Acid residues
SAcmA
length of AcmA
438
Amino Acid residues
S
Activation
1[3]
-
kLacI
dissociation constant of LacI
6
μmol
Assume the same as kTetR
kTetR
dissociation constant of TetR
6[3]
μmol
nTetR
Hills coefficient
3[3]
-
nLacl
Hills coefficient
3
-
Assume the same as nTetR
Variables
Concentration of
LacIm
Transcribed LacI
TetRm
Transcribed TetR
AcmAm
Transcribed AcmA
GFnm
Transcribed GF-17 (n=1,2,3)
Granm
Transcribed Grammistin-Pp1 (n=1,2,3)
LLnm
Transcribed LL-37 (n=1,2,3)
LacI
Translated Lacl
TetR
Translated TetR
GFn
Translated GF-17 (n=1,2,3)
Gran
Translated Grammistin-Pp1 (n=1,2,3)
LLn
Translated LL-37 (n=1,2,3)
By running the Matlab script, we obtained the results shown below.
Without the toggle switch:
Fig 5.    State values of LacIm, GFnm, Granm, LLnm, AcmAm, LacI, tetR, AcmA, GFn, Gran, LLn.
With the toggle switch:
Fig 6.    State values of LacIm, GFnm, Granm, LLnm, tetRm, AcmAm, LacI, tetR, AcmA, GFn, Gran, LLn.
Fig 7.    Individual display of transcribed LacIm, GFnm, Granm, LLnm, tetRm and AcmAm
Fig 8.    Individual display of translated LacI, tetR, AcmA, GFn, Gran and LLn.
From these graphs, we can make a general conclusion that the toggle switch provides at least 35 minutes for the peptide accumulation. By the time the promoter P2 initiates the transcription and later efficiently translation of the mRNA of the tandem repeat genes (ll-37, gf-17, and grammistin-Pp1), the antimicrobial peptides are capable of being synthesized at high rates. When the repression of the promoter ptet (tetR) is relieved and the lysis gene acmA (AcmAm) starts to be transcribed, the antimicrobial peptides can be accumulated to high concentrations. Thereafter, enough amounts of antimicrobial peptides will be released to eradicate Staphylococcus aureus through the cell lysis.
References
[1] Z. Cai, et al. “A simulation of Synthetic agr System in E. coli,”in Bioinformatics Research and Applications. Charlotte, NC: Springer, 2013, pp76-86.
[2] NICE Expression System for Lactococcus lactis. MoBITec GmbH, Germany, 2010.
[3] Team: TU-Delft (2013). Timer Plus Sumo [Online]. Available: https://2013.igem.org/Team:TU-Delft/Timer_Plus_Sumo
[4] C. Wu, H. Lee, and B. Chen, "Robust synthetic gene network design via library-based search method," Bioinformatics, vol. 27, pp. 2700-2706, Oct. 2011.
Collaborators and Supporters
Location
Rm 363, Science Building
Xi'an Jiaotong-Liverpool University
111 Ren'ai Road, Suzhou, China
215123
Social
Get in touch
igem@xjtlu.edu.cn
XJTLU-CHINA iGEM 2017
