<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_XJTLU-CHINA_Detection skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:XJTLU-CHINA/Detection</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Detection">Detection</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/PeptideProduction">Peptide Production</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Lysis">Cell Lysis</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Demonstrate">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Applied_Design">Application</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:XJTLU-CHINA/Parts" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:XJTLU-CHINA/Notebook" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Notebook">Lab Book</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practice</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/HP/Gold_Integrated">HP_Gold</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/HP/Silver">HP_Silver</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:XJTLU-CHINA/Team" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Collaborations">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/InterLab">Interlab</A></LI></DIV></DIV><TITLE>Detection</TITLE><DIV class="title text-center"><H1 style="color:white; font-size:80px; padding-top:120px;">Detection</H1></DIV><DIV class="container"><DIV class="para"><H1>Quorum sensing of <I>Staphylococcus aureus</I></H1><P style="font-size:20px;"><I>Staphylococcus aureus</I> is an opportunistic and invasive pathogen that utilizes quorum sensing (QS),
                a cell-to-cell signaling mechanism, to strengthen its ability to cause disease in humans. QS allows <I>S. aureus</I>                to monitor their surroundings and population size, and regulate the production of virulence factors. As shown
                in Fig. 1, QS in <I>S. aureus </I>is regulated by the <I>agr</I> operon which consists of two transcription
                units <I>agr</I>BDCA and RNAIII. The genes <I>agr</I>BDCA are controlled by an inducible promoter termed
                P2. When regulatory proteins bind to P2, these four genes start to be transcribed and then translated to
                give four different proteins which are the transmembrane protein AgrB, the precursor peptide AgrD, the receptor
                protein AgrC and the regulator protein AgrA. The agrD encodes precursor peptides and will be post-transcriptionally
                processed by AgrB to generate a functional QS signal molecule auto-inducing peptides (AIPs) (George and Muir,
                2007). When AIPs are secreted from the cells into the external environment, it can be detected by AgrC present
                on the <I>S. aureus</I> cell surface. The binding of AIP to AgrC phosphorylates AgrA which has a higher affinity
                to interact with P2 than the un-phosphorylated form (Koenig et al., 2004). Upon binding to the P2 promoter,
                AgrA upregulates the transcription of the <I>agr</I> gene and leads to a higher production of AIP.
            </P></DIV><DIV class="fig" align="center"><FIGURE><FIGCAPTION><STRONG>Figure 1 The agr system. The arrows with a filled head show the positive feedback loop.</STRONG></FIGCAPTION></FIGURE></DIV><DIV class="para"><H1>Auto-inducing peptides (AIPs)</H1><P style="font-size:20px;">
                AIPs secreted by <I>Staphylococcus aureus </I>are seven to nine amino acids in length and have the C-terminal
                five residues constrained as a thiolactone ring through a cysteine side chain (Fig. 2). There are four different
                classes in the Agr systems which are referred to as Agr-I, Agr-II, Agr-III, and Agr-IV and each is capable
                of recognizing a unique AIP structure which is respectively AIP-I, AIP-II, AIP-III and AIP-IV. <I>S. aureus</I>                may also be classified into four groups/strains (I to IV) by the class of the AIPs produced. Among the four
                classes of AIPs, the five-residue thiolactone ring structure is always conserved, while the other ring and
                tail residues differ. Similarly, the proteins involved in signal biosynthesis (AgrB and AgrD) and surface
                receptor binding (AgrC) also show variability. Interestingly, different AIP signals cross-inhibit the activity
                of the others (Mayville et al. 1999). For example, group I <I>S. aureus’ </I>s quorum sensing can be activated
                by AIP-I but is inhibited by the AIPs produced by group II or III <I>S. aureus</I> strains. Since AIP-I and
                AIP-IV differ by only one amino acid, they are grouped together and can function interchangeably. Indeed,
                Lyon et al. (2002) demonstrates that, the three AIP groups (group I/IV, group II and group III) can cross-inhibit
                each other with binding constants in the low nanomolar range. Therefore, when conducting experiments, the
                groups of <I>S.aureus</I> strains and the classes of AIPs must match to avoid QS down-regulation. In our
                project, we chose the group I <I>S. aureus</I> strain and AIP-I to verify our parts efficiency.
            </P></DIV><DIV class="fig" align="center"><FIGURE><FIGCAPTION><STRONG>Figure 2 The structure of four AIP signals of S. aureus and their cross-inhibitory interactions.</STRONG></FIGCAPTION></FIGURE></DIV><DIV class="para"><H1>The sensing device</H1><P style="font-size:20px;">
                Our project tried to incorporate a part of the QS system as the sensing device to detect the existence of <I>S. aureus</I>                and upregulate multiple network elements. The detection of AIP-I signal by AgrC results in auto-phosphorylation
                of AgrC followed by transfer of the phosphate group to AgrA. Phosphorylated AgrA has a higher affinity for
                binding to P2 and upregulates the target genes.
            </P></DIV><DIV class="para"><H1>The reporter protein</H1><P style="font-size:20px;">
                In this year’s project, we have checked a range of reporter genes. The luciferase reporter gene and the green fluorescent
                protein (GFP) reporter gene were considered in the initial plan. Thereafter, in the light of long shipment
                time of the luciferase assay kit, we turned to utilize sfGFP as the reporter. Most existing variants of GFP
                often mis-fold when expressed as fusion proteins. Though there exist some better-folded variants of GFPs
                that can be utilized as protein fusion tags, yet the fused proteins can reduce the folding yield and ﬂuorescence
                of these GFPs. Hence, we adopted to employ sfGFP (superfolder GFP) developed by Jean-Denis Pedelacq’s team
                (2006) as our visualizing reporter tool, which is a more robustly folded version of GFP. The merit of sfGFP
                is that the ﬂuorescence from this protein is independent of mis-folding of the fusion partner and is directly
                proportional to total expression level regardless of the solubility of the fusion, thus superfolder GFP makes
                itself a robust reporter of both fusion protein expression and pathway (will show later). Hence, sfGFP was
                used as a reporter in our project this year for its superb folding stability.
            </P></DIV><DIV class="fig" align="center"><FIGURE><FIGCAPTION><STRONG>Figure 3 Stereo structure of sfGFP</STRONG></FIGCAPTION></FIGURE></DIV><DIV class="para"><H1>Nisin production in <I>Lactococcus lactis</I></H1><P style="font-size:20px;">
                We used the promoter PnisA as the promoter of the circuit because it can be induced by nisin. Nisin is a 34 amino acid anti-microbial
                peptide produced by the probiotic bacterium <I>Lactococcus lactis</I>. It can lead to cell
                death by binding to Lipid II, an essential precursor for cell-wall synthesis, and forming small pores in
                the cytoplasmic membrane which can lead to leakage of small molecules such as ATP. It is widely used as food
                preservative due to its broad host spectrum. In the food industry, nisin is not chemically synthesized, but
                it is obtained from the culture of <I>L. lactis</I> in natural substrates, such as milk or dextrose. Biosynthesis
                of nisin is a complex process. It is encoded by a cluster of 11 genes, of which the first gene, nisA, encodes
                the precursor of nisin. The nisB, nisC, nisP, and nisT genes are involved in the modification, translocation
                and processing of nisin. The nisI, nisF, nisE, and nisG genes are for the immunity against nisin. The regulation
                on the expression of the nisin genes are controlled by nisR and nisK, which are, respectively, the response
                regulator and the sensor kinase of the bacterial two-component signal transduction systems (TCS). NisK is
                a histidine–protein kinase that resides in the cytoplasmic membrane and act as a receptor for the mature
                nisin molecule. Upon binding of nisin to NisK, NisK auto-phosphorylates and subsequently transfers the phosphate
                group to NisR. The activated NisR induces transcription from two of the three promoters in the nisin gene
                cluster: PnisA and PnisF.
            </P></DIV><DIV class="para"><H1>The strain <I>Lactococcus lactis</I> NZ9000</H1><P style="font-size:20px;">
                The strain of bacteria we chose, <I>Lactococcus lactis</I> NZ9000, is created by integrating the <I>nisK</I>                and <I>nisR</I> genes for the signal transduction system into the <I>pepN</I> gene (broad range amino peptidase)
                of <I>L. lactis subsp. cremoris</I> MG1363 (nisin-negative). When a gene of interest is subsequently placed
                behind the inducible promoter PnisA in the plasmid pNZ8148, the expression of that gene can be induced by
                adding sub-inhibitory concentrations of nisin (0.1 – 5 ng/ml) to the culture medium.
            </P></DIV><DIV class="para"><H1>The plasmid pNZ8148</H1><P style="font-size:20px;">
                The plasmid we used, pNZ8148, is a broad host range vector. It carries the chloramphenicol resistant gene and has a PnisA
                promoter followed by an NcoI restriction site for translational fusions at the ATG sequence. It came originally
                from the plasmid pSH71 in <I>Lactococcus lactis</I>. The replicon of the vector pNZ8148 contributes
                to the replication of this plasmid in many Gram-positive bacteria, such as <I>Lactobacillus plantarum</I>                and <I>Streptococcus thermophilus</I>. However, its replication in <I>E. coli</I> needs to conduct in a recA+
                strain. A recA- strain, for example, DH5α, cannot be used for these vectors because the nisin promoter PnisA
                is not completely repressed in <I>E. coli</I>, allowing expression of the gene products that can be toxic
                or lethal. Therefore, to this extent, it is preferred to transform a ligation mixture directly to a <I>Lactococcus lactis</I>                strain.
            </P></DIV><DIV class="fig" align="center"><FIGURE><FIGCAPTION><STRONG>Figure 4 The plasmid pNZ8148</STRONG></FIGCAPTION></FIGURE></DIV><DIV class="para"><H1>Testing the pathway</H1><P style="font-size:20px;">
                The biopart BBa_K515005, which contains an RBS and the sfGFP gene, was assembled at the C terminal of our composite part
                K2309003 by the conventional restriction digestion and cloning method. Thereafter, by using Gibson Assembly,
                our composite part was integrated in pNZ8148 at the PstI restriction site. Our detection part was constructed
                as follows: 
            </P></DIV><DIV class="fig" align="center"><FIGURE><FIGCAPTION><STRONG>Figure 5 The Detection Part. </STRONG></FIGCAPTION><FIGCAPTION><STRONG>The yellow figures represent ribosome binding sites and the red figures represent terminators.</STRONG></FIGCAPTION></FIGURE></DIV><H1 class="para">Final Assembly</H1><DIV class="fig" align="center"><FIGURE><FIGCAPTION><STRONG>Figure 6 The Final Assembly. </STRONG></FIGCAPTION></FIGURE></DIV><DIV class="para"><H1>The visualization or detection protocols</H1><P style="font-size:22px;">Nisin induction of gene expression in <I>Lactococcus lactis</I></P><P style="font-size:20px;">1. Grow 5 ml culture in M17 overnight at 30°C.</P><P style="font-size:20px;">2. Dilute 1/25 in 2 x 10 ml fresh medium (30°C).</P><P style="font-size:20px;">3. Grow the culture until OD600 ≈ 0.4.</P><P style="font-size:20px;">4. Induce one 10 ml culture by culturing with 1 ng/ml nisin and keep the other 10 ml culture as negative control.</P><P style="font-size:20px;">5. Incubate 2 - 3 hours, measure OD600 to monitor the growth of the induced and non-induced cultures.</P><P style="font-size:20px;">6. Induce one culture with AIP (at a final concentration of 10 µM) and keep the other induced without inducing.</P><P style="font-size:20px;">7. Measure the fluorescence signals in the culture by a fluorescence plate reader 3 hours later.</P></DIV><DIV class="para" id="r"><H1>References</H1><P style="font-size:20px;">George EA, Muir TW (2007) Molecular mechanisms of agr quorum sensing in virulent Staphylococci. ChemBioChem 8:
                847–855
            </P><P style="font-size:20px;">Koenig RL, Ray JL, Maleki SJ, Smeltzer MS, Hurlburt BK (2004) Staphylococcus aureus AgrA binding to the RNAIII-agr
                regulatory region. J Bacteriol 186: 7549–7555</P><P style="font-size:20px;">Lyon, G. J., J. S. Wright, T. W. Muir, and R. P. Novick. 2002. Key determinants of receptor activation in the
                agr autoinducing peptides of Staphylococcus aureus. Biochemistry 41:10095-10104.</P><P style="font-size:20px;">Mayville, P., G. Ji, R. Beavis, H. Yang, M. Goger, R. P. Novick, and T. W. Muir. 1999. Structure-activity analysis
                of synthetic autoinducing thiolactone peptides from Staphylococcus aureus responsible for virulence. Proc.
                Natl. Acad. Sci. USA 96:1218-1223.</P><P style="font-size:20px;">Pedelacq, J., Cabantous, S., Tran, T. et al. (2015) ‘Engineering and characterization of a superfolder green
                ﬂuorescent protein’, Nature Biotechnology 24, 79 - 88 (2005) [Online]. DOI: 10.1038/nbt1172</P></DIV></DIV><DIV class="text-center" style="background:#cfe5d9;"><H1 style="font-weight:bolder;color:#006934; ">Collaborators and Supporters</H1><FOOTER><DIV class="text-center"><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-4 loaction"><H4>Location</H4><P style="text-align:center;">Rm 363, Science Building
                       Xi'an Jiaotong-Liverpool University
                       111 Ren'ai Road, Suzhou, China
                       215123
                       </P></DIV><DIV class="col-md-4 social"><H4>Social</H4></DIV><DIV class="col-md-4 contact"><H4>Get in touch</H4><P style="text-align:center;">igem@xjtlu.edu.cn</P></DIV></DIV></DIV></DIV><DIV class="text-center" style="background:#003b73;"><P style="text-align:center; color:white;">XJTLU-CHINA iGEM 2017</P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>