Team:XJTLU-CHINA/Detection
Toggle navigation
Home
Project
Description
Detection
Peptide Production
Cell Lysis
Modeling
Demonstration
Application
Parts
Basic Parts
Composite Parts
Notebook
Lab Book
Protocols
Human Practice
HP_Gold
HP_Silver
Team
Members
Attributions
Collaboration
Safety
Interlab
Detection
Detection
Quorum sensing of Staphylococcus aureus
Staphylococcus aureus is an opportunistic and invasive pathogen that utilizes quorum sensing (QS),                 a cell-to-cell signaling mechanism, to strengthen its ability to cause disease in humans. QS allows S. aureus to monitor their surroundings and population size, and regulate the production of virulence factors. As shown                 in Fig. 1, QS in S. aureus is regulated by the agr operon which consists of two transcription                 units agrBDCA and RNAIII. The genes agrBDCA are controlled by an inducible promoter termed                 P2. When regulatory proteins bind to P2, these four genes start to be transcribed and then translated to                 give four different proteins which are the transmembrane protein AgrB, the precursor peptide AgrD, the receptor                 protein AgrC and the regulator protein AgrA. The agrD encodes precursor peptides and will be post-transcriptionally                 processed by AgrB to generate a functional QS signal molecule auto-inducing peptides (AIPs) (George and Muir,                 2007). When AIPs are secreted from the cells into the external environment, it can be detected by AgrC present                 on the S. aureus cell surface. The binding of AIP to AgrC phosphorylates AgrA which has a higher affinity                 to interact with P2 than the un-phosphorylated form (Koenig et al., 2004). Upon binding to the P2 promoter,                 AgrA upregulates the transcription of the agr gene and leads to a higher production of AIP.
Figure 1 The agr system. The arrows with a filled head show the positive feedback loop.
Auto-inducing peptides (AIPs)
AIPs secreted by Staphylococcus aureus are seven to nine amino acids in length and have the C-terminal                 five residues constrained as a thiolactone ring through a cysteine side chain (Fig. 2). There are four different                 classes in the Agr systems which are referred to as Agr-I, Agr-II, Agr-III, and Agr-IV and each is capable                 of recognizing a unique AIP structure which is respectively AIP-I, AIP-II, AIP-III and AIP-IV. S. aureus may also be classified into four groups/strains (I to IV) by the class of the AIPs produced. Among the four                 classes of AIPs, the five-residue thiolactone ring structure is always conserved, while the other ring and                 tail residues differ. Similarly, the proteins involved in signal biosynthesis (AgrB and AgrD) and surface                 receptor binding (AgrC) also show variability. Interestingly, different AIP signals cross-inhibit the activity                 of the others (Mayville et al. 1999). For example, group I S. aureus’ s quorum sensing can be activated                 by AIP-I but is inhibited by the AIPs produced by group II or III S. aureus strains. Since AIP-I and                 AIP-IV differ by only one amino acid, they are grouped together and can function interchangeably. Indeed,                 Lyon et al. (2002) demonstrates that, the three AIP groups (group I/IV, group II and group III) can cross-inhibit                 each other with binding constants in the low nanomolar range. Therefore, when conducting experiments, the                 groups of S.aureus strains and the classes of AIPs must match to avoid QS down-regulation. In our                 project, we chose the group I S. aureus strain and AIP-I to verify our parts efficiency.
Figure 2 The structure of four AIP signals of S. aureus and their cross-inhibitory interactions.
The sensing device
Our project tried to incorporate a part of the QS system as the sensing device to detect the existence of S. aureus and upregulate multiple network elements. The detection of AIP-I signal by AgrC results in auto-phosphorylation                 of AgrC followed by transfer of the phosphate group to AgrA. Phosphorylated AgrA has a higher affinity for                 binding to P2 and upregulates the target genes.
The reporter protein
In this year’s project, we have checked a range of reporter genes. The luciferase reporter gene and the green fluorescent                 protein (GFP) reporter gene were considered in the initial plan. Thereafter, in the light of long shipment                 time of the luciferase assay kit, we turned to utilize sfGFP as the reporter. Most existing variants of GFP                 often mis-fold when expressed as fusion proteins. Though there exist some better-folded variants of GFPs                 that can be utilized as protein fusion tags, yet the fused proteins can reduce the folding yield and ﬂuorescence                 of these GFPs. Hence, we adopted to employ sfGFP (superfolder GFP) developed by Jean-Denis Pedelacq’s team                 (2006) as our visualizing reporter tool, which is a more robustly folded version of GFP. The merit of sfGFP                 is that the ﬂuorescence from this protein is independent of mis-folding of the fusion partner and is directly                 proportional to total expression level regardless of the solubility of the fusion, thus superfolder GFP makes                 itself a robust reporter of both fusion protein expression and pathway (will show later). Hence, sfGFP was                 used as a reporter in our project this year for its superb folding stability.
Figure 3 Stereo structure of sfGFP
Nisin production in Lactococcus lactis
We used the promoter PnisA as the promoter of the circuit because it can be induced by nisin. Nisin is a 34 amino acid anti-microbial                 peptide produced by the probiotic bacterium Lactococcus lactis. It can lead to cell                 death by binding to Lipid II, an essential precursor for cell-wall synthesis, and forming small pores in                 the cytoplasmic membrane which can lead to leakage of small molecules such as ATP. It is widely used as food                 preservative due to its broad host spectrum. In the food industry, nisin is not chemically synthesized, but                 it is obtained from the culture of L. lactis in natural substrates, such as milk or dextrose. Biosynthesis                 of nisin is a complex process. It is encoded by a cluster of 11 genes, of which the first gene, nisA, encodes                 the precursor of nisin. The nisB, nisC, nisP, and nisT genes are involved in the modification, translocation                 and processing of nisin. The nisI, nisF, nisE, and nisG genes are for the immunity against nisin. The regulation                 on the expression of the nisin genes are controlled by nisR and nisK, which are, respectively, the response                 regulator and the sensor kinase of the bacterial two-component signal transduction systems (TCS). NisK is                 a histidine–protein kinase that resides in the cytoplasmic membrane and act as a receptor for the mature                 nisin molecule. Upon binding of nisin to NisK, NisK auto-phosphorylates and subsequently transfers the phosphate                 group to NisR. The activated NisR induces transcription from two of the three promoters in the nisin gene                 cluster: PnisA and PnisF.
The strain Lactococcus lactis NZ9000
The strain of bacteria we chose, Lactococcus lactis NZ9000, is created by integrating the nisK and nisR genes for the signal transduction system into the pepN gene (broad range amino peptidase)                 of L. lactis subsp. cremoris MG1363 (nisin-negative). When a gene of interest is subsequently placed                 behind the inducible promoter PnisA in the plasmid pNZ8148, the expression of that gene can be induced by                 adding sub-inhibitory concentrations of nisin (0.1 – 5 ng/ml) to the culture medium.
The plasmid pNZ8148
The plasmid we used, pNZ8148, is a broad host range vector. It carries the chloramphenicol resistant gene and has a PnisA                 promoter followed by an NcoI restriction site for translational fusions at the ATG sequence. It came originally                 from the plasmid pSH71 in Lactococcus lactis. The replicon of the vector pNZ8148 contributes                 to the replication of this plasmid in many Gram-positive bacteria, such as Lactobacillus plantarum and Streptococcus thermophilus. However, its replication in E. coli needs to conduct in a recA+                 strain. A recA- strain, for example, DH5α, cannot be used for these vectors because the nisin promoter PnisA                 is not completely repressed in E. coli, allowing expression of the gene products that can be toxic                 or lethal. Therefore, to this extent, it is preferred to transform a ligation mixture directly to a Lactococcus lactis strain.
Figure 4 The plasmid pNZ8148
Testing the pathway
The biopart BBa_K515005, which contains an RBS and the sfGFP gene, was assembled at the C terminal of our composite part                 K2309003 by the conventional restriction digestion and cloning method. Thereafter, by using Gibson Assembly,                 our composite part was integrated in pNZ8148 at the PstI restriction site. Our detection part was constructed                 as follows:
Figure 5 The Detection Part.
The yellow figures represent ribosome binding sites and the red figures represent terminators.
Final Assembly
Figure 6 The Final Assembly.
The visualization or detection protocols
Nisin induction of gene expression in Lactococcus lactis
1. Grow 5 ml culture in M17 overnight at 30°C.
2. Dilute 1/25 in 2 x 10 ml fresh medium (30°C).
3. Grow the culture until OD600 ≈ 0.4.
4. Induce one 10 ml culture by culturing with 1 ng/ml nisin and keep the other 10 ml culture as negative control.
5. Incubate 2 - 3 hours, measure OD600 to monitor the growth of the induced and non-induced cultures.
6. Induce one culture with AIP (at a final concentration of 10 µM) and keep the other induced without inducing.
7. Measure the fluorescence signals in the culture by a fluorescence plate reader 3 hours later.
References
George EA, Muir TW (2007) Molecular mechanisms of agr quorum sensing in virulent Staphylococci. ChemBioChem 8:                 847–855
Koenig RL, Ray JL, Maleki SJ, Smeltzer MS, Hurlburt BK (2004) Staphylococcus aureus AgrA binding to the RNAIII-agr                 regulatory region. J Bacteriol 186: 7549–7555
Lyon, G. J., J. S. Wright, T. W. Muir, and R. P. Novick. 2002. Key determinants of receptor activation in the                 agr autoinducing peptides of Staphylococcus aureus. Biochemistry 41:10095-10104.
Mayville, P., G. Ji, R. Beavis, H. Yang, M. Goger, R. P. Novick, and T. W. Muir. 1999. Structure-activity analysis                 of synthetic autoinducing thiolactone peptides from Staphylococcus aureus responsible for virulence. Proc.                 Natl. Acad. Sci. USA 96:1218-1223.
Pedelacq, J., Cabantous, S., Tran, T. et al. (2015) ‘Engineering and characterization of a superfolder green                 ﬂuorescent protein’, Nature Biotechnology 24, 79 - 88 (2005) [Online]. DOI: 10.1038/nbt1172
Collaborators and Supporters
Location
Rm 363, Science Building
Xi'an Jiaotong-Liverpool University
111 Ren'ai Road, Suzhou, China
215123
Social
Get in touch
igem@xjtlu.edu.cn
XJTLU-CHINA iGEM 2017
