<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_XJTLU-CHINA_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:XJTLU-CHINA/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Detection">Detection</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/PeptideProduction">Peptide Production</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Lysis">Cell Lysis</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Demonstrate">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Applied_Design">Application</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:XJTLU-CHINA/Parts" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:XJTLU-CHINA/Notebook" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Notebook">Lab Book</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practice</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/HP/Gold_Integrated">HP_Gold</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/HP/Silver">HP_Silver</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:XJTLU-CHINA/Team" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Collaborations">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:XJTLU-CHINA/InterLab">Interlab</A></LI></DIV></DIV><P><TITLE>Modeling</TITLE></P><DIV class="title text-center"><H1 style="color:white; font-size:80px; padding-top:120px;">Modeling</H1></DIV><DIV class="container"><H1>Modeling on the sensing device</H1><P>In the mathematical modeling of quorum sensing, we formulated a system of ordinary equations representing the intracellular and extracellular interactions between the two Agr proteins and AIP (auto-inducing peptide) molecules. Along with numerical simulations, we performed an asymptotic analysis of the time-dependent model in order to characterize whether the AIP molecules produced by <I>Staphylococcus aureus</I> in the intestine would activate our sensing device.</P><P>To build the model, we first proposed the following assumptions:</P><UL><LI>Proteins and mRNA inside the cells are limited by natural degradation.</LI><LI>Housekeeping phosphatases are able to dephosphorylate AgrA at rate α<SUB>pidi</SUB>.</LI><LI>Receptor-bound AIP can dissociate spontaneously at rate α<SUB>unbind</SUB>.</LI><LI>When an AIP binds to AgrC, we assume that auto-phosphorylation of AgrC happens simultaneously because this process is sufficiently fast. When AgrC transfers its phosphate group to AgrA at rate α<SUB>pi</SUB>, it is able to re-auto-phosphorylate.</LI></UL><P>The resulting equations, together with the definitions of the parameters and variables are shown below. </P><P>Table 1 Definitions of the parameters</P><DIV class="table" align="center"><TABLE border="1" cellpadding="5"><TBODY><TR><TH>Parameters</TH><TH>Rate constant for</TH><TH>Value</TH><TH>Units</TH><TH>Note</TH></TR><TR><TD>α<SUB>pi</SUB></TD><TD>Phosphorylation of AgrA</TD><TD>10<SUP>[1]</SUP></TD><TD>μmol<SUP>-1</SUP> ml<SUP>-1</SUP> h<SUP>-1</SUP></TD></TR><TR><TD>α<SUB>pidi</SUB></TD><TD>Dephosphorylation of AgrA</TD><TD>1<SUP>[1]</SUP></TD><TD>h<SUP>-1</SUP></TD></TR><TR><TD>μ<SUB>x</SUB></TD><TD>Degradation and dilution</TD><TD>2<SUP>[1]</SUP></TD><TD>h<SUP>-1</SUP></TD></TR><TR><TD>μ<SUB>sfGFP</SUB></TD><TD>Degradation of sfGFP</TD><TD>0.378</TD><TD>h<SUP>-1</SUP></TD><TD>Assume the same as GFP</TD></TR><TR><TD>μ<SUB>Am</SUB>, μ<SUB>Cm</SUB>, μ<SUB>sfGFPm</SUB></TD><TD>Degradation of mRNA</TD><TD>17.28<SUP>[4]</SUP></TD><TD>h<SUP>-1</SUP></TD></TR><TR><TD>α<SUB>cbind</SUB></TD><TD>AgrC that anchors to the cell membrane</TD><TD>10</TD><TD>μmol<SUP>-1</SUP> ml<SUP>-1</SUP> h<SUP>-1</SUP></TD><TD>Assume the same as α<SUB>pi</SUB></TD></TR><TR><TD>α<SUB>bind</SUB></TD><TD>Binding of AIP to AgrC</TD><TD>1<SUP>[1]</SUP></TD><TD>μmol<SUP>-1</SUP> ml<SUP>-1</SUP> h<SUP>-1</SUP></TD></TR><TR><TD>α<SUB>unbind</SUB></TD><TD>Separation of AIP from AgrC</TD><TD>0.1<SUP>[1]</SUP></TD><TD>h<SUP>-1</SUP></TD></TR></TBODY></TABLE></DIV><DIV class="table" align="center"><TABLE border="1" cellpadding="5"><TBODY><TR><TH>Parameters</TH><TH>Definitions</TH><TH>Value</TH><TH>Units</TH><TH>Note</TH></TR><TR><TD>X</TD><TD>Nisin</TD><TD>1.42×10<SUP>-7[2]</SUP></TD><TD>μmol ml<SUP>-1</SUP></TD></TR><TR><TD>k<SUB>2</SUB></TD><TD>The Phosphorylated AgrA concentration required for half-maximal transcription rate of P2</TD><TD>1<SUP>[1]</SUP></TD><TD>μmol  ml<SUP>-1</SUP></TD></TR><TR><TD>β<SUB>1</SUB></TD><TD>Maximum transcription rate of pnisA</TD><TD>10</TD><TD>μmol  h<SUP>-1</SUP></TD><TD>Assume the same as β<SUB>2</SUB></TD></TR><TR><TD>β<SUB>2</SUB></TD><TD>Maximum transcription rate of P2</TD><TD>10<SUP>[1]</SUP></TD><TD>μmol h<SUP>-1</SUP></TD></TR><TR><TD>l<SUB>pinsA</SUB></TD><TD>Leakage factor of pinsA</TD><TD>0.02</TD><TD>-</TD><TD>Assume the same as l<SUB>ptet</SUB></TD></TR><TR><TD>l<SUB>P2</SUB></TD><TD>Leakage factor of P2</TD><TD>0.02</TD><TD>-</TD><TD>Assume the same as l<SUB>ptet</SUB></TD></TR><TR><TD>a</TD><TD>Translation rate</TD><TD>61200<SUP>[3]</SUP></TD><TD>Amino acid residues h<SUP>-1</SUP></TD></TR><TR><TD>S<SUB>A</SUB></TD><TD>Length of AgrA</TD><TD>207</TD><TD>Amino acid residues</TD></TR><TR><TD>S<SUB>C</SUB></TD><TD>Length of AgrC</TD><TD>413</TD><TD>Amino acid residues</TD></TR><TR><TD>S<SUB>sfGFP</SUB></TD><TD>Length of sfGFP</TD><TD>237</TD><TD>Amino acid residues</TD></TR></TBODY></TABLE></DIV><P>Table 2 Definitions of the variables</P><DIV class="table" align="center"><TABLE border="1" cellpadding="5"><TBODY><TR><TH>Variables</TH><TH>Concentration of </TH><TH>Units</TH></TR><TR><TD>Am</TD><TD>mRNA of AgrA</TD><TD>μmol  ml<SUP>-1</SUP></TD></TR><TR><TD>A</TD><TD>AgrA</TD><TD>μmol  ml<SUP>-1</SUP></TD></TR><TR><TD>Cm</TD><TD>mRNA of AgrC</TD><TD>μmol  ml<SUP>-1</SUP></TD></TR><TR><TD>C</TD><TD>AgrC</TD><TD>μmol ml<SUP>-1</SUP></TD></TR><TR><TD>C<SUB>bind</SUB></TD><TD>AgrC that anchors to the cell membrane</TD><TD>μmol ml<SUP>-1</SUP></TD></TR><TR><TD>AIP</TD><TD>Free AIP molecules</TD><TD>μmol ml<SUP>-1</SUP></TD></TR><TR><TD>C<SUB>p</SUB></TD><TD>AIP-bound AgrC</TD><TD>μmol ml<SUP>-1</SUP></TD></TR><TR><TD>A<SUB>pi</SUB></TD><TD>The phosphorylated AgrA</TD><TD>μmol ml<SUP>-1</SUP></TD></TR><TR><TD>sfGFPm</TD><TD>mRNA of sfGFP</TD><TD>μmol ml<SUP>-1</SUP></TD></TR><TR><TD>sfGFP</TD><TD>The product of P2 promoter</TD><TD>μmol ml<SUP>-1</SUP></TD></TR></TBODY></TABLE></DIV><P>The three Hill equations represent the rates of transcription of agrA, agrC and sfGFP genes. β<SUB>1</SUB> is the highest efficiency for the promoter pnisA to initiate the transcription of the agrC and agrA genes, and β<SUB>2</SUB> is the highest efficiency for the promoter P2 to initiate the transcription of the sfGFP gene. X is the concentration of nisin which is needed to activate the promoter pnisA, to this extent, k<SUB>1</SUB> equals to the concentration of nisin when the rate of reaction is up to half of V<SUB>max</SUB>. K<SUB>2</SUB>, which is controlled by another regulatory factor, is the concentration of phosphorylated AgrA when the rate of reaction is up to half of V<SUB>max</SUB>.</P><P>By assuming that 0.25 μM of AIP molecules is present in the intestine, we run the MATLAB script to check whether AIP molecules can successfully activate the promoter P2 by binding to AgrC and phosphorylating AgrA. We used a reporter protein sfGFP to show the activation of the sensing device. If there is an apparent increase of sfGFP concentration, the sensing device can be considered as being activated. The results are shown below.</P><P class="form"><B>Fig 1.</B>    State values of AgrA, Cbind, AgrC, Cp, Api and sfGFP.</P><P class="form"><B>Fig 2.</B>    Individual display of 9 variables</P><P>As it is shown in the Figure 1 and 2, the concentration of sfGFP is enough high after several hours. Therefore, we made a conclusion that the amount of AIP molecules can activate the promoter P2 to transcribe the genes downstream.</P><P>To verify whether the cross-inhibition （introduced in the description section）can result in a decrease in AIP and
            virulence factor in
            <I>Staphylococcus aureus</I>, we hijacked the modeling of the sensing device into the quorum sensing system. By
            assuming the concentration of AIPs decreases to 0.1 μM by the cross-inhibition of another type of AIPs, we run
            the MATLAB script again to check whether there is any change in sfGFP production.</P><P class="form"><B>Fig 3.</B>    State values of 9 variables.</P><P class="form"><B>Fig 4.</B>    Individual display  of 9 variables.</P><H2>Discussion</H2><P>As shown in Figures 3 and 4, the concentration of sfGFP is extremely low when compared with the case of 0. 25 μM
            AIPs. Therefore, the amount of AIP molecules can dramatically affect the P2 promoter to express the downstream
            genes. Based on the result of sfGFP expression, we can also make a conclusion that in the case of the quorum
            sensing of
            <I>Staphylococcus aureus</I>, their signal transduction of AIPs can be cross-inhibited by a different type of AIPs.
            As a result, AIPs and virulence factors cannot be further produced.</P><H1>Modelling on peptide synthesis and cell lysis</H1><P>Our design uses the tandem repeat strategy to express three copies of each peptide gene, LL-37, GF-17 and Grammistin-Pp1, aiming to produce peptides with a higher rate. To release the peptides to kill <I>Staphylococcus aureus</I> in the intestine, we choose lysis of the cells instead of secretion. A lysis gene is used to open up the cells, then all the peptides will surely be released into the guts. In addition, we plan to use a toggle switch to provide more time for peptide synthesis before lysis. When the cells are lysed, it will result in the release of intracellular proteins and stop all life activities. Therefore, we use modeling to identify:</P><OL><LI>How much time can the toggle switch provide for the accumulation of AMPs？</LI></OL><P>Results: 
Inspired by the team TU-Delft (2013), we simplified the promoters P2 to serve as a binary switch between the active and inactive promoter states instead of continuous activities from fully on to fully off. We used the parameter--s, a binary state descriptor, to refer to the situation when a promoter produces one of the two levels of activity: on or off. In addition, the original amount of TetR  was assumed to be 200 μmol. Due to the unknown concentration of AcmA to lyse the cells, this value was assumed to be 50 μmol.
</P><P>Table 3 Definitions of parameters</P><DIV class="table" align="center"><TABLE border="1" cellpadding="5"><TBODY><TR><TH>Parameters</TH><TH>Definitions</TH><TH>Value</TH><TH>Units</TH><TH>Note</TH></TR><TR><TD><I>a</I></TD><TD>translation rate per amino acid</TD><TD>1020<SUP>[3]</SUP></TD><TD>Amino acids residues min<SUP>-1</SUP></TD></TR><TR><TD><I>c<SUB>P2</SUB></I></TD><TD>maximum transcription rate of P2</TD><TD>0.17<SUP>[1]</SUP></TD><TD>μmol  min<SUP>-1</SUP></TD></TR><TR><TD><I>c<SUB>ptet</SUB></I></TD><TD>maximum transcription rate of ptet</TD><TD>2.79<SUP>[3]</SUP></TD><TD>μmol<SUP>-1</SUP>  min<SUP>-1</SUP></TD></TR><TR><TD><I>c<SUB>plac</SUB></I></TD><TD>maximum transcription rate of plac</TD><TD>2.79</TD><TD>μmol<SUP>-1</SUP>  min<SUP>-1</SUP></TD><TD>Assume the same as <I>c<SUB>ptet</SUB></I></TD></TR><TR><TD><I>d<SUB>mRNA</SUB></I></TD><TD>degradation rate of mRNA</TD><TD>0.288<SUP>[4]</SUP></TD><TD>min<SUP>-1</SUP></TD></TR><TR><TD><I>d<SUB>Lacl</SUB></I></TD><TD>degradation rate of Lacl</TD><TD>0.1386<SUP>[4]</SUP></TD><TD>min<SUP>-1</SUP></TD></TR><TR><TD><I>d<SUB>TetR</SUB></I></TD><TD>degradation rate of TetR</TD><TD>0.1386<SUP>[4]</SUP></TD><TD>min<SUP>-1</SUP></TD></TR><TR><TD><I>d<SUB>AcmA</SUB></I></TD><TD>degradation rate of AcmA</TD><TD>0.033</TD><TD>min<SUP>-1</SUP></TD><TD>Assume the same as μ<SUB>x</SUB></TD></TR><TR><TD><I>d<SUB>GFn</SUB></I></TD><TD>degradation rate of GFn</TD><TD>0.011</TD><TD>min<SUP>-1</SUP></TD><TD>Assume the one-third of μ<SUB>x</SUB></TD></TR><TR><TD><I>d<SUB>Gram</SUB></I></TD><TD>degradation rate of Gran</TD><TD>0.011</TD><TD>min<SUP>-1</SUP></TD><TD>Assume the one-third of μ<SUB>x</SUB></TD></TR><TR><TD><I>d<SUB>LLn</SUB></I></TD><TD>degradation rate of LLn</TD><TD>0.011</TD><TD>min<SUP>-1</SUP></TD><TD>Assume the one-third of μ<SUB>x</SUB></TD></TR><TR><TD><I>l<SUB>P2</SUB></I></TD><TD>Leakage factor of P2</TD><TD>0.002</TD><TD>-</TD><TD>Assume the same as <I>l<SUB>ptet</SUB></I></TD></TR><TR><TD><I>l<SUB>ptet</SUB></I></TD><TD>Leakage factor of ptet</TD><TD>0.002<SUP>[3]</SUP></TD><TD>-</TD></TR><TR><TD><I>l<SUB>plac</SUB></I></TD><TD>Leakage factor of plac</TD><TD>0.002</TD><TD>-</TD><TD>Assume the same as <I>l<SUB>ptet</SUB></I></TD></TR><TR><TD><I>S<SUB>Lacl</SUB></I></TD><TD>length of Lacl</TD><TD>371</TD><TD>Amino Acid residues</TD></TR><TR><TD><I>S<SUB>TetR</SUB></I></TD><TD>length of TetR</TD><TD>226</TD><TD>Amino Acid residues</TD></TR><TR><TD><I>S<SUB>AcmA</SUB></I></TD><TD>length of AcmA</TD><TD>438</TD><TD>Amino Acid residues</TD></TR><TR><TD><I>S</I></TD><TD>Activation</TD><TD>1<SUP>[3]</SUP></TD><TD>-</TD></TR><TR><TD><I>k<SUB>LacI</SUB></I></TD><TD>dissociation constant of LacI</TD><TD>6</TD><TD>μmol</TD><TD>Assume the same as <I>k<SUB>TetR</SUB></I></TD></TR><TR><TD><I>k<SUB>TetR</SUB></I></TD><TD>dissociation constant of TetR</TD><TD>6<SUP>[3]</SUP></TD><TD>μmol</TD></TR><TR><TD><I>n<SUB>TetR</SUB></I></TD><TD>Hills coefficient</TD><TD>3<SUP>[3]</SUP></TD><TD>-</TD></TR><TR><TD><I>n<SUB>Lacl</SUB></I></TD><TD>Hills coefficient</TD><TD>3</TD><TD>-</TD><TD>Assume the same as <I>n<SUB>TetR</SUB></I></TD></TR></TBODY></TABLE></DIV><DIV class="table" align="center"><TABLE border="1" cellpadding="5"><TBODY><TR><TH>Variables</TH><TH>Concentration of </TH></TR><TR><TD>LacIm</TD><TD>Transcribed LacI</TD></TR><TR><TD>TetRm</TD><TD>Transcribed TetR</TD></TR><TR><TD>AcmAm</TD><TD>Transcribed AcmA</TD></TR><TR><TD>GFnm</TD><TD>Transcribed GF-17 (n=1,2,3)</TD></TR><TR><TD>Granm</TD><TD>Transcribed Grammistin-Pp1 (n=1,2,3)</TD></TR><TR><TD>LLnm</TD><TD>Transcribed LL-37 (n=1,2,3)</TD></TR><TR><TD>LacI</TD><TD>Translated Lacl</TD></TR><TR><TD>TetR</TD><TD>Translated TetR</TD></TR><TR><TD>GFn</TD><TD>Translated GF-17 (n=1,2,3)</TD></TR><TR><TD>Gran</TD><TD>Translated Grammistin-Pp1 (n=1,2,3)</TD></TR><TR><TD>LLn</TD><TD>Translated LL-37 (n=1,2,3)</TD></TR></TBODY></TABLE></DIV><P>By running the Matlab script, we obtained the results shown below.</P><P>Without the toggle switch:</P><P class="form"><B>Fig 5.</B>    State values of LacIm, GFnm, Granm, LLnm, AcmAm, LacI, tetR, AcmA, GFn, Gran, LLn.</P><P>With the toggle switch:</P><P class="form"><B>Fig 6.</B>    State values of LacIm, GFnm, Granm, LLnm, tetRm, AcmAm, LacI, tetR, AcmA, GFn, Gran, LLn.</P><P class="form"><B>Fig 7.</B>    Individual display of transcribed LacIm, GFnm, Granm, LLnm, tetRm and AcmAm</P><P class="form"><B>Fig 8.</B>    Individual display of translated LacI, tetR, AcmA, GFn, Gran and LLn.</P><P>From these graphs, we can make a general conclusion that the toggle switch provides at least 35 minutes for the peptide accumulation. By the time the promoter P2 initiates the transcription and later efficiently translation of the mRNA of the tandem repeat genes (ll-37, gf-17, and grammistin-Pp1), the antimicrobial peptides are capable of being synthesized at high rates. When the repression of the promoter ptet (tetR) is relieved and the lysis gene acmA (AcmAm) starts to be transcribed, the antimicrobial peptides can be accumulated to high concentrations. Thereafter, enough amounts of antimicrobial peptides will be released to eradicate <I>Staphylococcus aureus</I> through the cell lysis.</P><P>References
[1] Z. Cai, et al. “A simulation of Synthetic <I>agr</I> System in <I>E. coli</I>,”<I>in Bioinformatics Research and Applications</I>. Charlotte, NC: Springer, 2013, pp76-86.
[2] NICE Expression System for <I>Lactococcus lactis</I>. MoBITec GmbH, Germany, 2010.
[3] Team: TU-Delft (2013). <I>Timer Plus Sumo</I> [Online]. Available: <A href="https://2013.igem.org/Team:TU-Delft/Timer_Plus_Sumo">https://2013.igem.org/Team:TU-Delft/Timer_Plus_Sumo</A> 
[4] C. Wu, H. Lee, and B. Chen, &quot;Robust synthetic gene network design via library-based search method,&quot; Bioinformatics, vol. 27, pp. 2700-2706, Oct. 2011.</P></DIV><DIV class="text-center" style="background:#cfe5d9;"><H1 style="font-weight:bolder;color:#006934; ">Collaborators and Supporters</H1><FOOTER><DIV class="text-center"><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-4 loaction"><H4>Location</H4><P style="text-align:center;">Rm 363, Science Building
               Xi'an Jiaotong-Liverpool University
               111 Ren'ai Road, Suzhou, China
               215123
               </P></DIV><DIV class="col-md-4 social"><H4>Social</H4></DIV><DIV class="col-md-4 contact"><H4>Get in touch</H4><P style="text-align:center;">igem@xjtlu.edu.cn</P></DIV></DIV></DIV></DIV><DIV class="text-center" style="background:#003b73;"><P style="text-align:center; color:white;">XJTLU-CHINA iGEM 2017</P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>