Team:Duke/Contribution
Toggle navigation
People
Members
Advisors
Attributions
Collaborations
Overview
Gaston Day School
Mini Convention
Reddit Forum
Mosquito Forum
Youtube Channel
Project
Description
Design
Results
Contribution
Improve
Applied Design
Judging Form
Biology
Experiments
Notebook
Safety
Parts
Basic Parts
Part Collection
Modeling
Model Overview
GRFT Targets
Trials
Docking
Expectations
Human Practices
Overview (Sliver)
Integrated (Gold)
Interviews
Social & Political
Economics
Engagement
Engagement Overview
Students in the Lab
Marbles Kids Museum
Youtube Channel
Mosquito Forum
Duke iGEM 2017 Characterization: pSB1C3 Is Not Efficient in Gibson Assembly
At Duke iGEM, we often experiment with using genes in different vectors to achieve optimal metabolic engineering and then we put the genes into pSB1C3 afterwards. In order to do this, we use Gibson assembly and put homology arms on each end of a G-block along with pSB1C3 homology upstream. Last year we designed our G-blocks like this:
Like this, we first put the gene into pSMART via Gibson Assembly because it was better for our experiments and then afterwards we used restriction digest in order to put the gene into pSB1C3.
Our Gibson assemblies had a very low success rate of 6.15% (Figure 1), while the rest of the colonies that grew were empty pSMART vector that had self-ligated. Sequencing the empty vector, we found that there was actually some pSB1C3 between the 5’ and 3’ ends of pSMART.
Figure 1: pSB1C3 has a low Gibson assembly success rate of 6.15%.
The Biobrick prefix and suffix have homology to one another, which greatly reduces the efficacy of Gibson Assembly (n=65). Colony PCR was run on random colonies after transformations of Gibson assemblies to confirm whether the gene insert was present.
Upon closer examination, there are actually 9 base pairs that are homologous between the Biobrick prefix and suffix (Figure 2).  We realized that this micro-homology was cutting out the rest of the G-block during Gibson assembly and creating empty vector like this:
Figure 2: The Biobrick prefix and suffix have 9bp of homology.
Using this construct, the only insert we found was the 9bp of pSB1C3 homology (Figure 3).
Figure 3: pSB1C3 self-homology is in empty vectors
Sequencing the vectors that were empty from colony PCR, we found that unanimously that in place of the gene insert was the 9bp pSB1C3 homology instead.
This suggests that pSB1C3 is not a good vector for Gibson Assembly.
Duke iGEM
People
Members
Advisors
Attributions
Collaborations
Overview
Gaston Day School
Mini Convention
Reddit Forum
Mosquito Forum
Youtube Channel
Project
Description
Design
Results
Contribution
Improve
Applied Design
Judging Form
Biology
Experiments
Notebook
Safety
Parts
Basic Parts
Part Collection
Modeling
Model Overview
GRFT Targets
Trials
Docking
Expectations
Human Practices
Overview (Sliver)
Integrated (Gold)
Interviews
Social & Political
Economics
Engagement
Engagement Overview
Students in the Lab
Marbles Kids Museum
Youtube Channel
Mosquito Forum
