Team:Duke/Improve
Toggle navigation
People
Members
Advisors
Attributions
Collaborations
Overview
Gaston Day School
Mini Convention
Reddit Forum
Mosquito Forum
Youtube Channel
Project
Description
Design
Results
Contribution
Improve
Applied Design
Judging Form
Biology
Experiments
Notebook
Safety
Parts
Basic Parts
Part Collection
Modeling
Model Overview
GRFT Targets
Trials
Docking
Expectations
Human Practices
Overview (Sliver)
Integrated (Gold)
Interviews
Social & Political
Economics
Engagement
Engagement Overview
Students in the Lab
Marbles Kids Museum
Youtube Channel
Mosquito Forum
Duke iGEM 2017 Improvement: Making pSB1C3 Better For Gibson Assembly
In pSB1C3 (and all iGEM vectors) the Biobrick prefix and suffix have homology (Figure 1) to each other and because of this, attempts at Gibson assembly usually end with self-ligation of the vector and rejection of the insert (See Duke iGEM 2017’s Contribution page for more info on this). BBa_K2464000 is pSB1C3 with the typical RFP insert, except we removed “gcggccg” from the suffix, which was homologous to the prefix. The SpeI and PstI cut sites are preserved and this iteration of pSB1C3 that we’ve dubbed pSB1C3_Gibson is highly effective in Gibson assembly (Figure 2). BBa_K2464001 and BBa_K2464002 are the forward and reverse primers, respectively, that we designed to perform Q5 mutagenesis on pSB1C3 to make pSB1C3_Gibson.
Figure 1: The Biobrick prefix and suffix have 9bp of homology.
Figure 2: pSB1C3_Gibson is highly efficient in Gibson Assembly.
Colony PCR was run on random colonies after transformations of Gibson assemblies to confirm whether the gene insert was present (pSB1C3 Homology n = 65, pSB1C3_Gibson Homology n = 30). The Gibson assemblies of pSB1C3_Gibson were done using pSB1C3_Gibson as the homology arms and vector the gene insert was placed in. We have not yet collected data for Gibson assembly into pSMART with upstream homology of pSB1C3_Gibson, which is how the data for pSB1C3 was collected. See Duke iGEM 2017’s Contribution page.
Because iGEM vectors all share common Biobrick prefixes and suffixes, all iGEM vectors would also have this issue of being inefficient in Gibson assembly. However, since our primers bind to the Biobrick and RFP area of the plasmid, they can be used to remove homology in any iGEM vector and make it better for Gibson Assembly.
Duke iGEM
People
Members
Advisors
Attributions
Collaborations
Overview
Gaston Day School
Mini Convention
Reddit Forum
Mosquito Forum
Youtube Channel
Project
Description
Design
Results
Contribution
Improve
Applied Design
Judging Form
Biology
Experiments
Notebook
Safety
Parts
Basic Parts
Part Collection
Modeling
Model Overview
GRFT Targets
Trials
Docking
Expectations
Human Practices
Overview (Sliver)
Integrated (Gold)
Interviews
Social & Political
Economics
Engagement
Engagement Overview
Students in the Lab
Marbles Kids Museum
Youtube Channel
Mosquito Forum
