<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Duke_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Duke/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="example3"><NAV class="navbar navbar-default" style="margin-bottom: 0px"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar3"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar3" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">People </A></LI><LI><A href="https://2017.igem.org/Team:Duke/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Advisors">Advisors</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Collaborations </A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/GastonDaySchool">Gaston Day School</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/MiniConvention">Mini Convention</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/RedditForum">Reddit Forum</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/MosquitoForum">Mosquito Forum</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/YouTubeChannel">Youtube Channel</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project </A></LI><LI><A href="https://2017.igem.org/Team:Duke/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Applied_Design">Applied Design</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Duke">Judging Form</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Biology </A></LI><LI><A href="https://2017.igem.org/Team:Duke/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Part_Collection">Part Collection</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Duke/Model">Model Overview</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Model/Targets">GRFT Targets</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Model/Trials">Trials</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Model/Docking">Docking</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Model/Expectations">Expectations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Duke/HP/Silver">Overview (Sliver)</A></LI><LI><A href="https://2017.igem.org/Team:Duke/HP/Gold_Integrated">Integrated (Gold)</A></LI><LI><A href="https://2017.igem.org/Team:Duke/HP/Silver/Interviews">Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Duke/HP/Silver/Social">Social &amp; Political</A></LI><LI><A href="https://2017.igem.org/Team:Duke/HP/Silver/Economics">Economics</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Engagement </A></LI><LI><A href="https://2017.igem.org/Team:Duke/Engagement">Engagement Overview</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Engagement/EduHighSchoolStudent">Students in the Lab</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Engagement/EngageCommunity">Marbles Kids Museum</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/YouTubeChannel">Youtube Channel</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/MosquitoForum">Mosquito Forum</A></LI></DIV></DIV></NAV></DIV><H1> Experiments </H1><DIV class="column full_size"><H2>Bradford Assay</H2><B>Introduction: </B><P>Method used to measure the protein concentration in solution.</P><B>Materials:</B><UL><LI>Bradford Reagent  </LI><LI> Samples </LI><LI> 96 well plate  </LI><LI> Protein Standards  </LI></UL><B>Procedure:</B><UL><B>Running the Assay</B><OL style="font-size:16px;"><LI>Add 10 ul of each standard (1-5) to five wells.</LI><LI>Add 10 ul of each sample to the other wells.
				<UL>
 					Each sample should be run in triplicate (three different trials of each sample).
				</UL></LI><LI>Add 300 ul of the bradford reagent to each well. 
			</LI><LI>Shake plate for a few minutes or let sit for 15 min then read plate. 
				<UL>
					Read at 595 nm. 
				</UL></LI><LI>Copy into excel sheet and analyze.
	</LI></OL></UL></DIV><DIV class="column full_size"><H2>Colony PCR</H2><B>Introduction:</B><P>The use of PCR to selectively amplify the DNA from E.coli colonies needed to visualize the presence or absence of plasmid constructs. </P><B>Materials:</B><UL><LI>PCR Mastermix Supplies </LI><LI> Thermocycler </LI><LI> PCR Tubes </LI><LI> 1.5mL Eppendorf Tubes  </LI><LI> Pipets </LI><LI>Cell Colonies/ DNA Template  </LI></UL><B>Procedure:</B><UL><OL style="font-size:16px;"><LI>Prepare econotaq master mix with oligos to amplify the region of interest </LI><UL>
			(https://www.lucigen.com/docs/manuals/MA038-EconoTaq-PLUS.pdf)
<UL><UL><UL><TABLE class="table table-bordered table-striped"><THEAD><TR><TH>Reagent</TH><TH>Volumes for 1 Reaction (µL)</TH></TR></THEAD><TBODY><TR><TD>Master Mix</TD><TD>12.5</TD></TR><TR><TD>SL1</TD><TD>0.25</TD></TR><TR><TD>SR2</TD><TD>0.25</TD></TR><TR><TD>H2O</TD><TD>12</TD></TR></TBODY></TABLE></UL></UL></UL><LI>Aliquot out master mix + oligos into a sufficient number of PCR tubes to test colonies
		</LI><LI>If your backbone + insert plates have many more colonies than your backbone only or insert only, you will need to test fewer colonies than if you have lots of colonies on your control plates
		</LI><LI>Aliquot out some LB (+ antibiotic) into another set of tubes,one for each colony you are testing
		</LI><LI>Pick a colony from your transformation plate with a sterile pipette tip
		</LI><LI>Dab the colony into the PCR tube with the econotaq master mix
		</LI><LI>Dab it again in the tube with LB
		</LI><LI>Run the PCR according to manufacturer’s protocol, but with a long (5 min) initial melting step at 95-98
	</LI></UL><B> NOTE: When dealing with DNA samples instead of colony samples. Use 3 uL of 1ng/uL DNA </B><UL>
				If DNA sample is poor, use at most 50 ng/uL concentration
			</UL><B>Thermocycler Protocol</B><OL style="font-size:16px;"><LI>Start: 98°C for 10 min.
		</LI><LI>Cycle (x35)
		</LI><LI>Melt 98°C for 45s
		</LI><LI>Anneal 50°C for 45s
		</LI><LI>Extend 72°C for ___
			<UL>
				Time = (1 min / kbase)*length(longest amplificant)
			</UL></LI><LI>10°C for infinity
	<DIV class="column full_size"><H2>Electrophoresis</H2><B>Introduction:</B><P>Gel electrophoresis is a method to visualize DNA in a cell. </P><B>Materials:</B></DIV></LI><LI>Agarose  </LI><LI> Midori Green Dye</LI><LI> DNA Ladder </LI><LI> Coomassie Blue Stain  </LI><B>Procedure:</B><UL><B>Make Gels</B><OL style="font-size:16px;"><LI>Mix 0.1g Agarose : 10mL 1x TAE
				</LI><LI>Microwave for 45s-1min. Continue adding 15-25s increments until the agarose mixture begins boiling. As soon as it begins boiling, take it out.
				</LI><LI>Mix 1uL DNA Stain : 10mL 1x TAE. Add to still hot agarose mixture
				</LI><LI>Pour into molds
				</LI><LI>Wait ~20 min to solidify
				</LI><LI>When taking out gels, first use razor blade/spatula to disconnect air seal between rubber stoppers and gel to prevent the gel coming apart.
		</LI></OL></UL><UL><B>Loading Gels</B><OL style="font-size:16px;"><LI>Take rubber stoppers off of the gel molds
			</LI><LI>Take the well molds out
			</LI><LI>Place the mold into the electrophoresis machine that is filled with 0.5x TAE (100 mL)
			</LI><LI>Be sure to have the wells on the negative end of the machine (the black end)
			</LI><LI>Draw out a diagram of your well (which samples are going in which lanes - it is helpful to put the ladder, positive, and negative controls directly next to each other and then load samples to the right of them)
			</LI><LI>On a thin strip of parafilm, place 2 uL of loading dye
			</LI><LI>Place 5 uL of DNA from PCR onto each spot of dye, add 5 L of water as well, immediately mix and then place into well
       			 <UL><B>Note: </B> if you are not worried about conserving DNA, can also use 10 uL DNA and no water (just make sure total DNA/Water volume = 10L).</UL></LI><LI>Run electrophoresis machine at:
				<UL>
 					160/180 V are usually run for PCR colony
				<UL>
					 This is a faster run gel 
					More imprecise - when you have a general idea of how the bands are going to turn out</UL></UL><UL>
					If Gel Extraction (120-150 V)
				<UL>
					This is slower so use when bands are expected to be close together
		</UL></UL></LI></OL></UL><DIV class="column full_size"><H2>Freezer Stocks Protocol</H2><B>Introduction:</B><P>Freezer stocks are samples that have been sent for sequencing and should have a corresponding miniprep. </P><B>Materials:</B><UL><LI> Cryotubes  
    		</LI><LI> 20% Glycerol 
    		</LI><LI> Culture Samples
   	  </LI></UL><B>Procedure:</B><UL><OL style="font-size:16px;"><LI>In a cryotube, add 750 mL of culture solution to 750 mL of 20% glycerol</LI><LI>Gently pipette up and down
			</LI><LI>Store in -80 C freezer 
	</LI></OL></UL></DIV><DIV class="column full_size"><H2>Gibson Assembly</H2><B>Introduction:</B><P>Gibson assembly is similar to a restriction digest in goal and product but is a simpler process. </P><B>Materials:</B><UL><LI> Vector 
    		</LI><LI> Insert DNA
    		</LI><LI> Gibson Hi-Fi 2X Mix
    		</LI><LI> Pico Water
   	  </LI></UL><B>Procedure:</B><UL><B>Mix Into Tube</B><OL style="font-size:16px;"><LI>1.5 uL of vector</LI><LI>5 uL of Mix
				</LI><LI>1.5 uL of insert
				</LI><LI>Fill to final volume of 10 uL with pico water
			</LI></OL></UL><UL><B>Heat Block/PCR</B><OL style="font-size:16px;"><LI>Keep at 50˚C for 20-60 minutes</LI><LI>Ice samples for at least 10 minutes before transforming
		</LI></OL></UL></DIV><DIV class="column full_size"><H2>Homemade Bug Buster</H2><B>Introduction:</B><P> Chemical lysis solution needed to break apart bacterial cells. This is probably the most widely used lysis buffer. It relies on the nonionic detergent NP-40 as the major solubilizing agent, which can be replaced by Triton X-100 with similar results. Variations include lowering the detergent concentration or using alternate detergents such as digitonin, saponin, or CHAPS. </P><B>Materials:</B><UL><LI> Used Triton X-100 instead of nonide P-40 </LI><LI>150 mM NaCl</LI><LI>NP-40 lysis</LI><LI>150 mM NaCl</LI><LI>1.0% Nonidet P-40 (NP-40)</LI></UL></DIV><DIV class="column full_size"><H2>LB Agar Plates</H2><B>Introduction:</B><P>This is a recipe for 1 L of LB for plates that are a low salt LB</P><B>Materials:</B><UL><LI> 15 g Agar
    		</LI><LI> 10 g Tryptone
    		</LI><LI> 5 g Yeast Extract
    		</LI><LI> 5 g Sodium Chloride
    			<UL> For a high salt agar, use 10 g of Sodium Chloride </UL></LI><LI> Deionized Water </LI></UL><B>Procedure:</B><UL><B>Media</B><OL style="font-size:16px;"><LI>In a 2000 mL Erlenmeyer Flask, add the agar, tryptone, yeast extract, and sodium chloride.</LI><LI>In a graduated cylinder, measure out 1000 mL of deionized water.
				</LI><LI>Add it to the flask and mix.
				</LI><LI>Autoclave on a liquid cycle for 15 min
					<UL> Make sure to add water or ice to the bin before autoclaving. </UL></LI></OL></UL><UL><B>Recommended Concentrations</B><UL><UL><TABLE class="table table-bordered table-striped"><THEAD><TR><TH>Antibiotic Concentrations</TH></TR></THEAD><TBODY><TR><TD>Commonly Used Antibiotics</TD><TD>Recommended Concentrations (µg/mL) (</TD></TR><TR><TD>Ampicillin</TD><TD>100</TD></TR><TR><TD>Bleocin</TD><TD>5</TD></TR><TR><TD>Carbenicillin</TD><TD>25</TD></TR><TR><TD>Chloramphenicol</TD><TD>25</TD></TR><TR><TD>Coumermycin</TD><TD>25</TD></TR><TR><TD>Gentamycin</TD><TD>10</TD></TR><TR><TD>Kanamycin</TD><TD>50</TD></TR><TR><TD>Spectinomycin</TD><TD>50</TD></TR><TR><TD>Tetracycline</TD><TD>10</TD></TR></TBODY></TABLE></UL></UL><UL><B>KAN (Kanamycin) Plates</B><OL style="font-size:16px;"><LI>Once media is out of the autoclave, allow it to cool.</LI><LI>Add as many uL of KAN 1000X Stock as there is mL of media
					<UL> For a 1 L media batch, add 1 mL of KAN 1000X stock. </UL></LI><LI>Mix the flask well.
				</LI><LI>Pour plates.
					<UL> Make sure the cover the entire bottom of the plate and remove as many bubbles as possible. </UL></LI></OL></UL><UL><B>CM (Chloramphenicol) Plates</B><OL style="font-size:16px;"><LI>Once media is out of the autoclave, allow it to cool.</LI><LI>Add as many uL of CM 1000X Stock as there is mL of media
					<UL> For a 1 L media batch, add 1 mL of CM 1000X stock. </UL></LI><LI>Mix the flask well.
				</LI><LI>Pour plates.
					<UL> Make sure the cover the entire bottom of the plate and remove as many bubbles as possible. </UL></LI></OL></UL><UL><B>AMP (Ampicillin) Plates</B><OL style="font-size:16px;"><LI>Once media is out of the autoclave, allow it to cool.</LI><LI>Add as many uL of AMP 1000X Stock as there is mL of media
					<UL> For a 1 L media batch, add 1 mL of AMP 1000X stock. </UL></LI><LI>Mix the flask well.
				</LI><LI>Pour plates.
					<UL> Make sure the cover the entire bottom of the plate and remove as many bubbles as possible. </UL></LI></OL></UL><UL><DIV class="column full_size"><H2>Lysate Buffer</H2><B>Introduction:</B><P>Recipe to make 50 mL of Lysate Buffer with a final concentration of 2mM Phosphate Buffer </P><B>Materials:</B><UL><LI> Triton X-100 </LI><LI> SDS</LI><LI> Tris HCl</LI><LI> HEPES NaOH</LI></UL><B>Procedure:</B><UL><OL style="font-size:16px;"><LI>Phosphate buffer 100 mM or however high a concentration you have (see math)</LI><LI>EDTA (200ul)
				</LI><LI>Protease inhibiotrs (0.5 pill)
				</LI><LI>water to dilute
				</LI><LI>Made in a 50 ml conical tube and stored in 4 C fridge.
			</LI></OL></UL><UL><B>Math:</B><OL style="font-size:16px;"><LI> 2mM Phosphate Buffer * 50mL = 100mM Phosphate Buffer * x mL</LI><LI>x = 1 mL of 100mM Phosphate Buffer needed
				</LI><LI>Therefore 48.8 mL dH2O
		</LI></OL></UL></DIV><DIV class="column full_size"><H2>Making Competent Cells</H2><B>Introduction:</B><P>Preparing electrocompetent cells for transformation from a cell culture.</P><B>Materials:</B><UL><LI> Cell Stock to grow 
    		</LI><LI> Conical tubes
    		</LI><LI> LB broth
    		</LI><LI> Epi Tubes
    		</LI><LI>10% Glycerol
   	  </LI></UL><B>Procedure:</B><UL><B>Growth Phase</B><OL style="font-size:16px;"><LI>Add cell culture to 50 mL of LB broth</LI><LI>Place on shaker and let grow until there is an optical density of around 2 (this is an absorbance of about OD 0.5-0.6)
			</LI></OL></UL><UL><B>Make Cells Competent</B><OL style="font-size:16px;"><LI>Ice the comp cells and ICE the glycerol</LI><LI>Place broth into several (as many as needed) conical tubes and spin them down in the large centrifuge at 4,000 rpm for 7 minutes 
				</LI><LI>Once you have a pellet decant the supernatant.
				</LI><LI>Resuspend the pellet in a small amount of 10% glycerol (1 mL)
				</LI><LI>Place the resuspension into two epi tubes and chill
				</LI><LI>Centrifuge down for 1 minute at 16,000 rcf and form another pellet
				</LI><LI>Pipet out the supernatant
				</LI><LI>It is ok to remove some of the cells - it is better to remove the cells than leave salt (LB) in the solution
				</LI><LI>Wash with glycerol 3-5 more times
				</LI><LI>On the last wash resuspend only 1 of the tubes in 1 mL of 10% glycerol. Take out everything and add this to the other tube of cells. Resuspend the entire tube of cells.
				</LI><LI>Do a final spin down for 2 minutes at 16,000 rcf, resuspend the tube in the same volume of 10% glycerol as there are cells in the tube. (Tube should have 50% volume be 10% glycerol)
				</LI><LI>The final product should be a clam chowder consistency (creamy but not curd like or cream like)
		</LI></OL></UL><UL><B>Storage</B><OL style="font-size:16px;"><LI>Once you have purified the cells - aliquot them out into 40 uL in 0.6mL or 1.7 mL epi tubes</LI><LI>Store in -80 ˚C freezer
			</LI></OL></UL></DIV><DIV class="column full_size"><H2>Mini Prep Protocol (Zyppy)</H2><B>Introduction:</B><P>Mini prep using the Zyppy Mini Prep Kit. Used to extract DNA from cells. </P><B>Materials:</B><UL><LI> Zyppy Mini Prep Kit  
    		</LI><LI> Cells
   	  </LI></UL><B>Procedure:</B><UL><OL style="font-size:16px;"><LI>Spin cells down at 3500 rpm for 8 minutes.</LI><LI>Add 100 uL of 7X Lysis Buffer (Blue) to 600 UL of E. coli cultrue in a 1.5 ml microcentrifuge tube
				</LI><LI>Mix by inverting the tube 4-6 times and lyse samples for 1-2 min.
				</LI><LI>Add 350 uL of cold Neutralization Buffer (yellow), mix thoroughly.
					<UL> Neutralization is complete when sample becomes yellow and precipitate has formed (~1 min)</UL></LI><LI>Centrifuge at 11,000 - 16,000 g for two minutes.
				</LI><LI>Transfer the supernatant to the spin column
				</LI><LI>Place column into a collection tube and centrifuge at 11,000 g for 15 seconds
				</LI><LI>Discard the flow-through and place the column back into the same collection tube
				</LI><LI>Add 200 uL of Endo-Wash Buffer to the column
				</LI><LI>Centrifuge at 11,000 g for 15 seconds
				</LI><LI>Add 400 uL of zyppy wash buffer to the column.
				</LI><LI>Centrifuge at 11,000 g for 30 seconds
				</LI><LI>Transfer the column into a clean 1.5 ml microcentrifuge tube then add 30 uL of Zyppy Elution Buffer directly to the column matrix and let stand for one minute at room temperature.
				</LI><LI>Centrifuge at or less than 11,000 g for 15 seconds to elute the DNA
		</LI></OL></UL></DIV><DIV class="column full_size"><H2>Minimal Media 2-Stage Evaluations</H2><B>Introduction:</B><P>This standard operating procedure describes the media formulations for preparing media for strain evaluations in various systems as well as the procedures for making concentrated stocks of each media component. </P><B>Materials:</B><UL><LI> Chemical Reagents as Mentioned in Section 1 of this SOP
    		</LI><LI> 250 mL to 1000 mL Bottles
    		</LI><LI> 50 mL to 1000 mL Flasks/Beakers
    		</LI><LI> Ultrapure Water 
    		</LI><LI>0.2 µm Filter Units, Varying Volumes
    		</LI><LI>Autoclave
   	  </LI></UL><B>Procedure:</B><UL><B>Section 1: Preparation of Stock Solutions</B><UL><UL> Prepare the stock solutions to the following concentrations using ultrapure water and ACS grade reagents. </UL></UL><OL style="font-size:16px;"><LI>Prepare 1 liter of a solution of 10X concentrated ammonium-citrate salts, varying components based on the target biomass:
				</LI><LI>Prepare 1 liter of <B>10X Ammonium-Citrate </B> 30 salts by mixing 30 g of (NH4)2SO4 and 1.5 g Citric Acid in water with stirring. Autoclave and store at RT.
				</LI><LI>Prepare 1 liter of <B>10X Ammonium-Citrate </B> 90 salts by mixing 90 g of (NH4)2SO4 and 2.5 g Citric Acid in water with stirring. Autoclave and store at RT.
				 	<UL> Prepare the following buffering reagents: </UL></LI><LI><B>MOPS Buffer </B>: Prepare 1 M Potassium 3-(N-morpholino)propanesulfonic Acid (MOPS) and adjust to pH 7.4 with KOH (~40 mL). Filter sterilize (0.2 um) and store at RT.
				</LI><LI><B>Phosphate</B>: Prepare 0.1 M potassium phosphate buffer, pH 6.8 by mixing 49.7 mL of 1.0 M K2HPO4 and 50.3 mL of 1.0 M KH2PO4 and adjust to a final volume of 1000 mL with ultrapure water. Filter sterilize (0.2 um) and store at RT.
				</LI><LI>Prepare <B>2 M MgSO4 </B> and <B> 10 mM CaSO4 solutions </B>. Filter sterilize (0.2 um) and store at RT.
				</LI><LI><B>500X Trace Metals </B>: Prepare a solution of micronutrients in 1000 mL of water containing 10 mL of concentrated H2SO4 : 0.6 g CoSO4*7H20, 5.0 g CuSO4*5H20, 0.6 g ZnSO4*7H20, 0.2 g Na2MoO4*2H2O, 0.1 g H3BO3, and 0.3 g MnSO4*H2O. Filter sterilize (0.2 um) and store at RT in the dark.
				</LI><LI><B>Iron Sulfate </B>: Prepare a fresh solution of 40 mM ferrous sulfate heptahydrate in water. Filter sterilize (0.2 um) and discard after 1 day.
				</LI><LI><B>Thiamine</B>: Prepare a 50 g/L solution of thiamine-HCl. Filter sterilize (0.2 um) and store at 4°C.
				</LI><LI><B>Glucose</B>: Prepare a 500 g/L solution of glucose by stirring with heat. Cool, filter sterilize (0.2 um), and store at RT.
			</LI></OL></UL><UL><B>Section 2: Final Media Compositions:</B><OL style="font-size:16px;"><LI>Prepare the final working medium by aseptically mixing stock solutions based on the following tables in the order written to minimize precipitation.
				</LI><LI>SM10 ++ Seed Media, pH 6.8:
				<UL><UL><TABLE class="table table-bordered table-striped"><THEAD><TR><TH>Ingredient</TH><TH>Concentration Stock</TH><TH> Volume in 1 L (mL) </TH><TH>Final Concentration </TH></TR></THEAD><TBODY><TR><TH>Ammonium-Citrate 90 Salts, pH 7.5</TH><TH>10 X (90 g Ammonium Sulfate, 2.5 g Citrate)</TH><TH> 100.0 </TH><TH>1 X  (9 g   Ammonium Sulfate, 0.25 g Citrate)</TH></TR><TR><TH>Phosphate Buffer, pH 6.8</TH><TH>500 mM</TH><TH> 10.0 </TH><TH>5.00 mM </TH></TR><TR><TH>Trace Metals</TH><TH>500 X</TH><TH> 4.0 </TH><TH>2 X </TH></TR><TR><TH>Fe (II) Sulfate</TH><TH>40 mM</TH><TH> 4.0 </TH><TH>0.16 mM</TH></TR><TR><TH>MgSO4</TH><TH>2 M</TH><TH> 1.25 </TH><TH>2.50 mM</TH></TR><TR><TH>CaSO4</TH><TH>10 mM</TH><TH> 6.25 </TH><TH>0.06 mM</TH></TR><TR><TH>Glucose</TH><TH>500 g/L</TH><TH> 90.0 </TH><TH>45 .0g/L</TH></TR><TR><TH>MOPS</TH><TH>1 M</TH><TH> 200.0 </TH><TH>200 mM</TH></TR><TR><TH>Thiamine-HCl</TH><TH>50 g/L</TH><TH> 0.2 </TH><TH>0.01 g/L</TH></TR><TR><TH>Yeast Extract</TH><TH>100 g/L</TH><TH> 25.0 </TH><TH>2.5 g/L</TH></TR><TR><TH>Casamino Acids</TH><TH>100 g/L</TH><TH>25.0 </TH><TH>2.5 g/L</TH></TR></TBODY></TABLE></UL></UL></LI></OL></UL></DIV><DIV class="column full_size"><H2>Q5 Mutagenesis</H2><B>Introduction:</B><P>Q5 Mutagenesis allows for alterations to be made to a gblock using primers.  </P><B>Materials:</B><UL><LI> Q5 Mutagenesis Mix 
    		</LI><LI> Forward Primer
    		</LI><LI> Reverse Primer
    		</LI><LI> Template
    		</LI><LI>Pico-water
   	  </LI></UL><B>Procedure:</B><UL><B>Add to a PCR Tube:</B><OL style="font-size:16px;"><LI>25 ul 2X Q5 Mix
				</LI><LI>2.5 uL Forward Primer
				</LI><LI>2.5 uL Reverse Primer
				</LI><LI>1 uL Template
				</LI><LI>19 uL of pico water
			</LI></OL></UL><UL><B>Thermocycler Protocol</B><OL style="font-size:16px;"><LI>95° for 2.5 min
				</LI><LI>35 Cycles: 
					<UL>
						95° for  15 s 
						anneal temp for  for 15s
						72° for 30 s/kb </UL></LI><LI>72° for 2 min
				</LI><LI>4° infinite hold
		</LI></OL></UL></DIV><DIV class="column full_size"><H2>Sequencing</H2><B>Introduction:</B><P>Method used to determine the exact DNA sequence of a plasmid or construct </P><B>Procedure:</B><UL><B>Preparation</B><OL style="font-size:16px;"><LI>Verify concentrations of the plasmid DNA samples
			</LI></OL></UL><UL><B>DNA:</B><OL style="font-size:16px;"><LI>8 uL of 100 ng/uL DNA per reaction 
				</LI><LI>2.2 x 8uL DNA per sample goes into each sequencing tube
					<UL> Reason: we have two primers </UL></LI></OL></UL><UL><B>Primers:</B><OL style="font-size:16px;"><LI>Two tubes, 1 per primer
				</LI><LI>Per reaction: 5 uL of 5 µMol
				</LI><LI>Calculate final volumes and concentrations to reach the above specifications and pipette
				</LI><LI>Send off for sequencing
		</LI></OL></UL></DIV><DIV class="column full_size"><H2>Thermostability Heat Trials</H2><B>Introduction: </B><P>Method used to determine the melting temperature of a protein.</P><B>Materials:</B><UL><LI>Whole Protein Lysate Aliquots </LI><LI> Centrifuge </LI><LI> SDS-PAGE gel  </LI><LI> Protein Ladder  </LI></UL><B>Procedure:</B><UL><B>Running the Heat Trials</B><OL style="font-size:16px;"><LI>The whole lysate aliquots were heated to high temperatures (50,60, 65, 70, 75, 80, 90 degree celcius) for 30 and 60 minutes.</LI><UL>
 					Each sample should be run in triplicate (three different trials of each sample).
				</UL><LI> The samples were centrifuged 15 minutes at 5000 rpm. </LI><LI>The supernatant was run on an SDS-PAGE gel and the gel was analyzed to see if the expected protein band was present.</LI><FOOTER><DIV class="footer" id="footer"><A href="https://2017.igem.org/Team:Duke"><DIV class="container"><CENTER><H2> Duke iGEM </H2></CENTER></DIV></A><DIV class="row"><DIV class="col-sm-4 col-xs-6"><H3> People </H3><UL><LI><A href="https://2017.igem.org/Team:Duke/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Advisors">Advisors</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Attributions">Attributions</A></LI></UL></DIV><DIV class="col-sm-4 col-xs-6"><H3> Collaborations </H3><UL><LI><A href="https://2017.igem.org/Team:Duke/Collaborations">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/GastonDaySchool">Gaston Day School</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/MiniConvention">Mini Convention</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/RedditForum">Reddit Forum</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/MosquitoForum">Mosquito Forum</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/YouTubeChannel">Youtube Channel</A></LI></UL></DIV><DIV class="col-sm-4 col-xs-6"><H3> Project </H3><UL><LI><A href="https://2017.igem.org/Team:Duke/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Applied_Design">Applied Design</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Duke">Judging Form</A></LI></UL></DIV><DIV class="col-sm-4 col-xs-6"><H3> Biology </H3><UL><LI><A href="https://2017.igem.org/Team:Duke/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Part_Collection">Part Collection</A></LI></UL></DIV><DIV class="col-sm-4 col-xs-6"><H3> Modeling </H3><UL><LI><A href="https://2017.igem.org/Team:Duke/Model">Model Overview</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Model/Targets">GRFT Targets</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Model/Trials">Trials</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Model/Docking">Docking</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Model/Expectations">Expectations</A></LI></UL></DIV><DIV class="col-sm-4 col-xs-6"><H3> Human Practices </H3><UL><LI><A href="https://2017.igem.org/Team:Duke/HP/Silver">Overview (Sliver)</A></LI><LI><A href="https://2017.igem.org/Team:Duke/HP/Gold_Integrated">Integrated (Gold)</A></LI><LI><A href="https://2017.igem.org/Team:Duke/HP/Silver/Interviews">Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Duke/HP/Silver/Social">Social &amp; Political</A></LI><LI><A href="https://2017.igem.org/Team:Duke/HP/Silver/Economics">Economics</A></LI></UL></DIV><DIV class="col-sm-4 col-xs-6"><H3> Engagement </H3><UL><LI><A href="https://2017.igem.org/Team:Duke/Engagement">Engagement Overview</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Engagement/EduHighSchoolStudent">Students in the Lab</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Engagement/EngageCommunity">Marbles Kids Museum</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/YouTubeChannel">Youtube Channel</A></LI><LI><A href="https://2017.igem.org/Team:Duke/Collaborations/MosquitoForum">Mosquito Forum</A></LI></UL></DIV></DIV></DIV></FOOTER></OL></UL></DIV></UL></UL></DIV></OL></OL></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>