Team:Duke/Experiments
Toggle navigation
People
Members
Advisors
Attributions
Collaborations
Overview
Gaston Day School
Mini Convention
Reddit Forum
Mosquito Forum
Youtube Channel
Project
Description
Design
Results
Contribution
Improve
Applied Design
Judging Form
Biology
Experiments
Notebook
Safety
Parts
Basic Parts
Part Collection
Modeling
Model Overview
GRFT Targets
Trials
Docking
Expectations
Human Practices
Overview (Sliver)
Integrated (Gold)
Interviews
Social & Political
Economics
Engagement
Engagement Overview
Students in the Lab
Marbles Kids Museum
Youtube Channel
Mosquito Forum
Experiments
Bradford Assay
Introduction:
Method used to measure the protein concentration in solution.
Materials:
Bradford Reagent
Samples
96 well plate
Protein Standards
Procedure:
Running the Assay
Add 10 ul of each standard (1-5) to five wells.
Add 10 ul of each sample to the other wells.
Each sample should be run in triplicate (three different trials of each sample).
Add 300 ul of the bradford reagent to each well.
Shake plate for a few minutes or let sit for 15 min then read plate.
Read at 595 nm.
Copy into excel sheet and analyze.
Colony PCR
Introduction:
The use of PCR to selectively amplify the DNA from E.coli colonies needed to visualize the presence or absence of plasmid constructs.
Materials:
PCR Mastermix Supplies
Thermocycler
PCR Tubes
1.5mL Eppendorf Tubes
Pipets
Cell Colonies/ DNA Template
Procedure:
Prepare econotaq master mix with oligos to amplify the region of interest
(https://www.lucigen.com/docs/manuals/MA038-EconoTaq-PLUS.pdf)
Reagent
Volumes for 1 Reaction (µL)
Master Mix
12.5
SL1
0.25
SR2
0.25
H2O
12
Aliquot out master mix + oligos into a sufficient number of PCR tubes to test colonies
If your backbone + insert plates have many more colonies than your backbone only or insert only, you will need to test fewer colonies than if you have lots of colonies on your control plates
Aliquot out some LB (+ antibiotic) into another set of tubes,one for each colony you are testing
Pick a colony from your transformation plate with a sterile pipette tip
Dab the colony into the PCR tube with the econotaq master mix
Dab it again in the tube with LB
Run the PCR according to manufacturer’s protocol, but with a long (5 min) initial melting step at 95-98
NOTE: When dealing with DNA samples instead of colony samples. Use 3 uL of 1ng/uL DNA
If DNA sample is poor, use at most 50 ng/uL concentration
Thermocycler Protocol
Start: 98°C for 10 min.
Cycle (x35)
Melt 98°C for 45s
Anneal 50°C for 45s
Extend 72°C for ___
Time = (1 min / kbase)*length(longest amplificant)
10°C for infinity
Electrophoresis
Introduction:
Gel electrophoresis is a method to visualize DNA in a cell.
Materials:
Agarose
Midori Green Dye
DNA Ladder
Coomassie Blue Stain
Procedure:
Make Gels
Mix 0.1g Agarose : 10mL 1x TAE
Microwave for 45s-1min. Continue adding 15-25s increments until the agarose mixture begins boiling. As soon as it begins boiling, take it out.
Mix 1uL DNA Stain : 10mL 1x TAE. Add to still hot agarose mixture
Pour into molds
Wait ~20 min to solidify
When taking out gels, first use razor blade/spatula to disconnect air seal between rubber stoppers and gel to prevent the gel coming apart.
Loading Gels
Take rubber stoppers off of the gel molds
Take the well molds out
Place the mold into the electrophoresis machine that is filled with 0.5x TAE (100 mL)
Be sure to have the wells on the negative end of the machine (the black end)
Draw out a diagram of your well (which samples are going in which lanes - it is helpful to put the ladder, positive, and negative controls directly next to each other and then load samples to the right of them)
On a thin strip of parafilm, place 2 uL of loading dye
Place 5 uL of DNA from PCR onto each spot of dye, add 5 L of water as well, immediately mix and then place into well
Note: if you are not worried about conserving DNA, can also use 10 uL DNA and no water (just make sure total DNA/Water volume = 10L).
Run electrophoresis machine at:
160/180 V are usually run for PCR colony
This is a faster run gel
More imprecise - when you have a general idea of how the bands are going to turn out
If Gel Extraction (120-150 V)
This is slower so use when bands are expected to be close together
Freezer Stocks Protocol
Introduction:
Freezer stocks are samples that have been sent for sequencing and should have a corresponding miniprep.
Materials:
Cryotubes
20% Glycerol
Culture Samples
Procedure:
In a cryotube, add 750 mL of culture solution to 750 mL of 20% glycerol
Gently pipette up and down
Store in -80 C freezer
Gibson Assembly
Introduction:
Gibson assembly is similar to a restriction digest in goal and product but is a simpler process.
Materials:
Vector
Insert DNA
Gibson Hi-Fi 2X Mix
Pico Water
Procedure:
Mix Into Tube
1.5 uL of vector
5 uL of Mix
1.5 uL of insert
Fill to final volume of 10 uL with pico water
Heat Block/PCR
Keep at 50˚C for 20-60 minutes
Ice samples for at least 10 minutes before transforming
Homemade Bug Buster
Introduction:
Chemical lysis solution needed to break apart bacterial cells. This is probably the most widely used lysis buffer. It relies on the nonionic detergent NP-40 as the major solubilizing agent, which can be replaced by Triton X-100 with similar results. Variations include lowering the detergent concentration or using alternate detergents such as digitonin, saponin, or CHAPS.
Materials:
Used Triton X-100 instead of nonide P-40
150 mM NaCl
NP-40 lysis
150 mM NaCl
1.0% Nonidet P-40 (NP-40)
LB Agar Plates
Introduction:
This is a recipe for 1 L of LB for plates that are a low salt LB
Materials:
15 g Agar
10 g Tryptone
5 g Yeast Extract
5 g Sodium Chloride
For a high salt agar, use 10 g of Sodium Chloride
Deionized Water
Procedure:
Media
In a 2000 mL Erlenmeyer Flask, add the agar, tryptone, yeast extract, and sodium chloride.
In a graduated cylinder, measure out 1000 mL of deionized water.
Add it to the flask and mix.
Autoclave on a liquid cycle for 15 min
Make sure to add water or ice to the bin before autoclaving.
Recommended Concentrations
Antibiotic Concentrations
Commonly Used Antibiotics
Recommended Concentrations (µg/mL) (
Ampicillin
100
Bleocin
5
Carbenicillin
25
Chloramphenicol
25
Coumermycin
25
Gentamycin
10
Kanamycin
50
Spectinomycin
50
Tetracycline
10
KAN (Kanamycin) Plates
Once media is out of the autoclave, allow it to cool.
Add as many uL of KAN 1000X Stock as there is mL of media
For a 1 L media batch, add 1 mL of KAN 1000X stock.
Mix the flask well.
Pour plates.
Make sure the cover the entire bottom of the plate and remove as many bubbles as possible.
CM (Chloramphenicol) Plates
Once media is out of the autoclave, allow it to cool.
Add as many uL of CM 1000X Stock as there is mL of media
For a 1 L media batch, add 1 mL of CM 1000X stock.
Mix the flask well.
Pour plates.
Make sure the cover the entire bottom of the plate and remove as many bubbles as possible.
AMP (Ampicillin) Plates
Once media is out of the autoclave, allow it to cool.
Add as many uL of AMP 1000X Stock as there is mL of media
For a 1 L media batch, add 1 mL of AMP 1000X stock.
Mix the flask well.
Pour plates.
Make sure the cover the entire bottom of the plate and remove as many bubbles as possible.
Lysate Buffer
Introduction:
Recipe to make 50 mL of Lysate Buffer with a final concentration of 2mM Phosphate Buffer
Materials:
Triton X-100
SDS
Tris HCl
HEPES NaOH
Procedure:
Phosphate buffer 100 mM or however high a concentration you have (see math)
EDTA (200ul)
Protease inhibiotrs (0.5 pill)
water to dilute
Made in a 50 ml conical tube and stored in 4 C fridge.
Math:
2mM Phosphate Buffer * 50mL = 100mM Phosphate Buffer * x mL
x = 1 mL of 100mM Phosphate Buffer needed
Therefore 48.8 mL dH2O
Making Competent Cells
Introduction:
Preparing electrocompetent cells for transformation from a cell culture.
Materials:
Cell Stock to grow
Conical tubes
LB broth
Epi Tubes
10% Glycerol
Procedure:
Growth Phase
Add cell culture to 50 mL of LB broth
Place on shaker and let grow until there is an optical density of around 2 (this is an absorbance of about OD 0.5-0.6)
Make Cells Competent
Ice the comp cells and ICE the glycerol
Place broth into several (as many as needed) conical tubes and spin them down in the large centrifuge at 4,000 rpm for 7 minutes
Once you have a pellet decant the supernatant.
Resuspend the pellet in a small amount of 10% glycerol (1 mL)
Place the resuspension into two epi tubes and chill
Centrifuge down for 1 minute at 16,000 rcf and form another pellet
Pipet out the supernatant
It is ok to remove some of the cells - it is better to remove the cells than leave salt (LB) in the solution
Wash with glycerol 3-5 more times
On the last wash resuspend only 1 of the tubes in 1 mL of 10% glycerol. Take out everything and add this to the other tube of cells. Resuspend the entire tube of cells.
Do a final spin down for 2 minutes at 16,000 rcf, resuspend the tube in the same volume of 10% glycerol as there are cells in the tube. (Tube should have 50% volume be 10% glycerol)
The final product should be a clam chowder consistency (creamy but not curd like or cream like)
Storage
Once you have purified the cells - aliquot them out into 40 uL in 0.6mL or 1.7 mL epi tubes
Store in -80 ˚C freezer
Mini Prep Protocol (Zyppy)
Introduction:
Mini prep using the Zyppy Mini Prep Kit. Used to extract DNA from cells.
Materials:
Zyppy Mini Prep Kit
Cells
Procedure:
Spin cells down at 3500 rpm for 8 minutes.
Add 100 uL of 7X Lysis Buffer (Blue) to 600 UL of E. coli cultrue in a 1.5 ml microcentrifuge tube
Mix by inverting the tube 4-6 times and lyse samples for 1-2 min.
Add 350 uL of cold Neutralization Buffer (yellow), mix thoroughly.
Neutralization is complete when sample becomes yellow and precipitate has formed (~1 min)
Centrifuge at 11,000 - 16,000 g for two minutes.
Transfer the supernatant to the spin column
Place column into a collection tube and centrifuge at 11,000 g for 15 seconds
Discard the flow-through and place the column back into the same collection tube
Add 200 uL of Endo-Wash Buffer to the column
Centrifuge at 11,000 g for 15 seconds
Add 400 uL of zyppy wash buffer to the column.
Centrifuge at 11,000 g for 30 seconds
Transfer the column into a clean 1.5 ml microcentrifuge tube then add 30 uL of Zyppy Elution Buffer directly to the column matrix and let stand for one minute at room temperature.
Centrifuge at or less than 11,000 g for 15 seconds to elute the DNA
Minimal Media 2-Stage Evaluations
Introduction:
This standard operating procedure describes the media formulations for preparing media for strain evaluations in various systems as well as the procedures for making concentrated stocks of each media component.
Materials:
Chemical Reagents as Mentioned in Section 1 of this SOP
250 mL to 1000 mL Bottles
50 mL to 1000 mL Flasks/Beakers
Ultrapure Water
0.2 µm Filter Units, Varying Volumes
Autoclave
Procedure:
Section 1: Preparation of Stock Solutions
Prepare the stock solutions to the following concentrations using ultrapure water and ACS grade reagents.
Prepare 1 liter of a solution of 10X concentrated ammonium-citrate salts, varying components based on the target biomass:
Prepare 1 liter of 10X Ammonium-Citrate 30 salts by mixing 30 g of (NH4)2SO4 and 1.5 g Citric Acid in water with stirring. Autoclave and store at RT.
Prepare 1 liter of 10X Ammonium-Citrate 90 salts by mixing 90 g of (NH4)2SO4 and 2.5 g Citric Acid in water with stirring. Autoclave and store at RT.
Prepare the following buffering reagents:
MOPS Buffer : Prepare 1 M Potassium 3-(N-morpholino)propanesulfonic Acid (MOPS) and adjust to pH 7.4 with KOH (~40 mL). Filter sterilize (0.2 um) and store at RT.
Phosphate: Prepare 0.1 M potassium phosphate buffer, pH 6.8 by mixing 49.7 mL of 1.0 M K2HPO4 and 50.3 mL of 1.0 M KH2PO4 and adjust to a final volume of 1000 mL with ultrapure water. Filter sterilize (0.2 um) and store at RT.
Prepare 2 M MgSO4 and 10 mM CaSO4 solutions . Filter sterilize (0.2 um) and store at RT.
500X Trace Metals : Prepare a solution of micronutrients in 1000 mL of water containing 10 mL of concentrated H2SO4 : 0.6 g CoSO4*7H20, 5.0 g CuSO4*5H20, 0.6 g ZnSO4*7H20, 0.2 g Na2MoO4*2H2O, 0.1 g H3BO3, and 0.3 g MnSO4*H2O. Filter sterilize (0.2 um) and store at RT in the dark.
Iron Sulfate : Prepare a fresh solution of 40 mM ferrous sulfate heptahydrate in water. Filter sterilize (0.2 um) and discard after 1 day.
Thiamine: Prepare a 50 g/L solution of thiamine-HCl. Filter sterilize (0.2 um) and store at 4°C.
Glucose: Prepare a 500 g/L solution of glucose by stirring with heat. Cool, filter sterilize (0.2 um), and store at RT.
Section 2: Final Media Compositions:
Prepare the final working medium by aseptically mixing stock solutions based on the following tables in the order written to minimize precipitation.
SM10 ++ Seed Media, pH 6.8:
Ingredient
Concentration Stock
Volume in 1 L (mL)
Final Concentration
Ammonium-Citrate 90 Salts, pH 7.5
10 X (90 g Ammonium Sulfate, 2.5 g Citrate)
100.0
1 X  (9 g Ammonium Sulfate, 0.25 g Citrate)
Phosphate Buffer, pH 6.8
500 mM
10.0
5.00 mM
Trace Metals
500 X
4.0
2 X
Fe (II) Sulfate
40 mM
4.0
0.16 mM
MgSO4
2 M
1.25
2.50 mM
CaSO4
10 mM
6.25
0.06 mM
Glucose
500 g/L
90.0
45 .0g/L
MOPS
1 M
200.0
200 mM
Thiamine-HCl
50 g/L
0.2
0.01 g/L
Yeast Extract
100 g/L
25.0
2.5 g/L
Casamino Acids
100 g/L
25.0
2.5 g/L
Q5 Mutagenesis
Introduction:
Q5 Mutagenesis allows for alterations to be made to a gblock using primers.
Materials:
Q5 Mutagenesis Mix
Forward Primer
Reverse Primer
Template
Pico-water
Procedure:
Add to a PCR Tube:
25 ul 2X Q5 Mix
2.5 uL Forward Primer
2.5 uL Reverse Primer
1 uL Template
19 uL of pico water
Thermocycler Protocol
95° for 2.5 min
35 Cycles:
95° for  15 s
anneal temp for  for 15s
72° for 30 s/kb
72° for 2 min
4° infinite hold
Sequencing
Introduction:
Method used to determine the exact DNA sequence of a plasmid or construct
Procedure:
Preparation
Verify concentrations of the plasmid DNA samples
DNA:
8 uL of 100 ng/uL DNA per reaction
2.2 x 8uL DNA per sample goes into each sequencing tube
Reason: we have two primers
Primers:
Two tubes, 1 per primer
Per reaction: 5 uL of 5 µMol
Calculate final volumes and concentrations to reach the above specifications and pipette
Send off for sequencing
Thermostability Heat Trials
Introduction:
Method used to determine the melting temperature of a protein.
Materials:
Whole Protein Lysate Aliquots
Centrifuge
SDS-PAGE gel
Protein Ladder
Procedure:
Running the Heat Trials
The whole lysate aliquots were heated to high temperatures (50,60, 65, 70, 75, 80, 90 degree celcius) for 30 and 60 minutes.
Each sample should be run in triplicate (three different trials of each sample).
The samples were centrifuged 15 minutes at 5000 rpm.
The supernatant was run on an SDS-PAGE gel and the gel was analyzed to see if the expected protein band was present.
Duke iGEM
People
Members
Advisors
Attributions
Collaborations
Overview
Gaston Day School
Mini Convention
Reddit Forum
Mosquito Forum
Youtube Channel
Project
Description
Design
Results
Contribution
Improve
Applied Design
Judging Form
Biology
Experiments
Notebook
Safety
Parts
Basic Parts
Part Collection
Modeling
Model Overview
GRFT Targets
Trials
Docking
Expectations
Human Practices
Overview (Sliver)
Integrated (Gold)
Interviews
Social & Political
Economics
Engagement
Engagement Overview
Students in the Lab
Marbles Kids Museum
Youtube Channel
Mosquito Forum
