Team:Edinburgh OG/Notebook
PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens
Toggle navigation
Project
Project Background
Project Design
PHA Synthesis
Downstream Processing
Scaling Up
Model
Glucose Toggle Switch
InterLab Study
Parts List
Improve
Demonstrate
Practices
Integrated Human Practices
Education and Public Engagement
Life Cycle Assessment
Collaborations
Safety
Notebook
Weekly Journal
Protocol
Team
Acknowledgements
Attributions
Medal
 
 
 
 
 
Notebook
Investigation of the best gene order of phaCAB for PHB production(Pepper)
Week 1 - Week 2 Primers design and experimental preparation                                                   
 
Design and order primers
Primers for changing gene order in pha operon are designed in Gibson Assembly website
Preparation of M9 medium
Primers stock solution preparation
Week 3 - Week 5 Building and confirming constructs                                                                     
 
Amplify and extract the fragments of pSB1C3 backbone, pSB1C3-phaB​
Gel electrophoresis and Gel extraction
Amplify and extract fragments of phaA, phaB, phaC with different overlap (failed)
Due to the special primer design from Gibson, except backbone fragment, all of the other fragments were amplified by primers with overhang. In this case, a problem was caused by wrong identity of annealing sequence and overlap sequence. Therefore, it is necessary to produce fragments without overlap firstly, and used them as template, in order to avoid the impact of the overlap and annealing sequence.
Design and order new primers for phaA, phaB and phaC gene template without overhang
Purification of pSB1C3 backbone and pSB1C3-phaB DNA fragments
Amplify phaA, phaB and phaC gene templates by PCR followed by gel electrophoresis and gel extraction
Week 6 - Week 8 Construction establisment                                                                                    
 
Amplify separately phaA, phaB, and phaC genes fragments with different overhangs
Gel electrophoresis of phaA, phaB, and phaC genes fragments
Assemble fragments to build new constructs using Gibson Assembly and transformation ( E. coli DH5α competent cells)
colony PCR and double digestion
Figure 1. Results of colony PCR. The fragments of 4kb size were expected to be the positive results (a)BCA are 12 colonies from plates expressed by phaBAC plasmid. (b)BCA are 12 colonies from phaBCA plasmid. (c) represents 12 colonies from phaABC expression plasmid. 11 colonies in (d) are from phaCBA plasmid. (e) is the result if negative control of this PCR experiment, which represents colonies from expression of empty vector. (f) shows 8 colonies from phaACB plasmid. Finally, (g) is the second negative control of purity water used in this reaction, no band means dH2O used is pure.
Confirmation new plasmids by double digestion. Transfer new plasmids into another strain E. coli BL21(DE3)
Week 9 - Week 10 PHB production and extraction                                                                          
 
Culture the coli strain harboured phaACB operon or phaCBA operon to produce PHB (250 ml flasks)
The optical density was measured every 3 hours for plotting growth curves. Spread bacteria that harboured new constructs phaBAC, phaBCA, phaCBA, phaACB, phaABC or original phaCAB operon on Nile Red plate. Culture E. coli that contained new constructs of phaBAC,phaBCA,phaCBA, phaACB or phaABC for Nile Red plate reader assay.
Optical density measurement and fluorescent intensity measurement
Extract produced PHB from coli that harboured phaACB operon or phaCBA operon after 72h cultivation
Figure 2 Nile red plates images of E. coli that harboured new constructs. On each plate, half plate is spread by negative control- pSB1C3 empty vector, which should not have any fluorescence, another half is plated with new operon plasmids culture.
 
Table 1 OD value detected by Nile Red plate reader
Hours
 
BAC
BCA
CBA
ACB
ABC
24 h
2% Glucose
0.45±0.02
0.48±0.01
0.65±0.01
0.69±0.01
0.42±0.01
 
3% Glucose
0.42±0.01
0.44±0.01
0.64±0.01
0.63±0.01
0.47±0.01 
45 h
2% Glucose
0.66±0.01
0.70±0.01
0.94±0.02
0.99±0.02
0.61±0.01 
 
3% Glucose
0.66±0.01
0.68±0.02
0.94±0.01
0.98±0.03
0.68±0.01 
Table 1 is the OD600 value of Nile Red plate reader, culture fed with 2% glucose and 3% glucose are incubated for 24h and 45h, then stained by Nile Red, and placed into a black 96 well microplate and read by Tecan Infinate M200 (provided by Edinburgh Genome Foundry). Excitation wavelength is 520 nm, and emission wavelength is 590 nm. Data are average number ± SEM (standard error of mean), n=3. P-value<0.05. 
Table 2 Fluorescence intensity measurement by Nile red plate reader
Hours
 
BAC
BCA
CBA
ACB
ABC
24 h
2% Glucose
1,683±132
2,371±90
35,876±850
41,757±586
4,534±311
 
3% Glucose 
1,758±48
2,437±108
40,330±134
31,486±670
4,651±222
45 h
2% Glucose
5,374±97
5,432±236
29,874±685
 42,295±793
5,054±182
 
3% Glucose 
5,415±270
5,164±41
27,890±989
28,311±1988
4,499±95
Table 2 shows the fluorescence value detected by Nile red plate reader, samples stained by Nile Red, and placed into a black 96 well microplate and read by Tecan Infinate M200 (provided by Edinburgh Genome Foundry). Excitation wavelength is 520 nm, and emission wavelength is 590 nm. Data are average number ± SEM (standard error of mean), n=3, P-value<0.05.
The Production of PHB
Figure 3 Image of extracted PHB after cultivation of 45 hours - Figure 3 indicates the PHB products extracted from E. coli BL21 harboring six pha operons and empty vector plasmid, samples shown in figure 3 were incubated for 45h
 
Figure 4 Image of extracted PHB after cultivation of 72 hours - Figure 4 shows the PHB products from E. coli BL21 harboring phaCBA and phaACB, which incubated for 72h
Table 3 Dry weight of produced PHB and melting temperature
Operon construct
Dry weight
Melting temperature start
Finish
                                     Yield of PHB after 45h cultivation
Empty vector
~0
/
/
phaCAB origin order
0.237 g
150
185
phaACB
0.201 g
156
180
phaCBA
0.072 g
160
186
phaBCA
0.021 g
158
184
                                     Yield of PHB after 72h cultivation
phaACB
0.044 g
148
165
phaCBA
0.18 g
176
180
Table 3 indicates the dry weight and melting temperature of all PHB products extracted in this project
 
 
Improving the production of PHBV by introducing bktB  to E. coli  (Ming)
Week 1 - Week 2 Designing the fragments and primers                                                                 
 
A bktB gene from eutropha H16 was codon-optimised for E. coli and synthesised by IDT in two parts (B1 and B2 fragments) with 696 bp overlap.
Order the primers for fragments amplification
M9 medium and 30% glucose stock solution preparation
Week 3 - Week 5 constructs establishment and culture condition optimization                       
 
Establish a new constuct: pSB1C3-phaCAB-bktB (cloning strategy was shown below)
Figure 1. B1 and B2 fragments were codon-optimised for E. coli and synthesised by IDT. The 696 bp overlap was in green and restriction enzyme sites were coloured by red. Two pairs of primers were ordered from IDT to amplify bktB, among which B1 forward primer and B2 reverse primer were used to amplify bktB fragments.
 
Make competent cells ( E. coli BL21 and E. coli DH5)
Determine the tolerance of high concentration of propionic acid.
Three concentrations (8mM, 32mM and 48mM) were tested using E. coli cells that harboured pSB1C3-phaCAB.
Determine the suitable glucose concentration for cultivation. (0.5%, 1%, 2%, 3%, 5%, 10%, 20%)
Establish new control plasmid pSB1C3 by cutting of the RFP gene.
 
Figure 2. Plasmid of control plasmid pSB1C3-RFP. The new control plasmid pSB1C3 was obtained by digesting pSB1C3-RFP with restriction enzyme: NotI followed by self-ligation.
 
Establish another construct pSB1C3-phaCB-bktB
                                             
Figure 3. The bktB fragments and psB1C3-phaCB-bktB fragments were highlighted by red squares.
 
Week 6 - Week 7  Nile Red quantitative measurement                                                                  
 
Transformation of two new constructs.
The transformation was tested by double digestion and colony PCR
Order the Nile red fluorescent dye and prepare the stock solution of 1 mg/ml
Test whether Nile red can be used for PHA production confirmation and establish new Nile red semi-quantitative measurement.
 
Figure 4. Nile red staining plate was exposed to the blue light and UV light respectively. High fluorescent was observed on the right part (cells containing pSB1C3-phaCAB plasmid) due to the produced PHA.
 
Week 8 - Week 9 cell culture and PHA production                                                                           
 
Culture the E. coli strain BL21 harbouring pSB1C3-phaCAB-bktB or pSB1C3-PhaCB-bktB in M9 medium contained different glucose and propionic acid concentration.
Culture the E. coli strain BL21 harbouring pSB1C3-phaCAB that contained different glucose concentration.
Optical density of culture was measured at 16 hours, 24 hours. 32 hours, 48 hours and 56 hours.
Figure 5 Comparison of growth curve of recombinant E. coli harbouring pSB1C3-phaCAB for different concentration of glucose.
Figure 6 Comparison of growth curves with different concentration of glucose and propionic acid. The time of adding propionic acid was pointed out by red arrow.
Figure 7. Time course of cell growth for different construction plasmids. Glucose was added into culture medium as carbon resource with final concentration of 3 %. OD600 was taken after 16 hours, 24 hours, 32 hours, 48 hours and 56 hours. Standard deviation was showed as error bar.
Figure 8. Comparison of cell growth of recombinant E. coli with different concentration of propionic acid concentration. Time of adding propionic acid was pointed out by red arrow. Optical density of cell culture was taken after 16 hours, 24 hours, 32 hours, 48 hours and 56 hours. Error bars represented the standard deviations.
Measure fluorescent intensity of cultures to provide real-time information of PHA production. (semi-quantitative Nile red measurement)
 
Figure 9. Samples were collected at different cultivation times and stained by Nile red fluorescent dye. The fluorescent intensity increased during the cultivation times.
 
Figure 10 The fluorescent intensity of PHA produced with different glucose concentrations. (48 hours cultivation). Error bars represented standard deviations.
Figure 11. Fluorescent intensity of cells harboured pSB1C3 or pSB1C3-phaCAB at different cultivation time.
Week 9 - Week 10 Determination of PHA production                                                                     
 
Nile red plate staining
 
Figure 12. Nile red Plate staining of cells harbouring pSB1C3-phaCAB-bktB or pSB1C3-phaCB-bktB. Compared with control cells harbouring control plasmid, the production of PHA was confirmed.
 
Extraction of produced PHA
Table 1. Yield of PHA of pSB1C3-phaCAB with different glucose concentrations.
 
OD600
Volume (ml)
Total PHA (g)
PHA (mg/ml)
pSB1C3
1.35
100ml
0
0
pSB1C3-phaCAB
1.707
100ml
0.044
0.44
pSB1C3-phaCB-bktB
2.15
100ml
0.023
0.23
pSB1C3-phaCAB-bktB
1.875
100ml
0.021
0.21
Table 2. Yield of PHA with 3 % glucose and 8 mM propionic acid
 
 
Measure the melting point of extracted PHA
 
Tm 1(°C)
Tm 2 (°C)
Tm 3 (°C)
Pure PHB from Sigma
170-179
168-176
168-174
PHBV with 12% 3HV from Sigma
159-161
160-160
161-164
PHA from pSB1C3-PhaCAB
160-168
160-162
161-164
PHA from pSB1C3-PhaCB-bktB
150-155
149-151
149-152
PHA from pSB1C3-PhaCAB-bktB
155-159
156-161
157-159
Table 3. The melting temperature assessment of different extracted product
 
Melting temperature of extracted PHA was measured and compared with PHB and PHBV products which were brought from Sigma. Tm of standard pure PHB product was between 170 °C and 180 °C and Tm of PHBV (12 % 3HV) was between 160 °C and 164 °C.The melting temperature of extracted PHA differed from different constructs. The PHA extracted from cell harbouring pSB1C3-phaCAB plasmids showed higher melting temperature with approximately 165 °C. While the PHA extracted from the cells harbouring pSB1C3-phaCB-bktB or pSB1C3-phaCAB-bktB plasmids presented lower melting temperature at range of 150 °C -155 °C and 158 °C -162 °C respectively.
 
 
Investigation of the effect of phasin autoregulation system on PHB production (Qihui)
Week 1 Experimental preparation                                                                                                        
 
Recover BioBricks from iGEM 2018 distribution kit
Design the construct strategies. Two standard BioBricks (Table 1) were decided to be used for the experiments.
Table 1 Basic information of BioBricks
Biobrick
Well
Plate
Insert Length
Backbone
Antibiotics Resistance
BBa_K390501
19H
6
1136 bp
pSB1C3
chloramphenicol
BBa_K1149051
12M
4
4271 bp
pSB1C3
chloramphenicol
 
Making competent cells ( coli BL21 and DH5α)
Double digestion for confirming the Biobricks
Transformation test
Week 2 Designing the fragments and primers                                                                                  
 
Design and order fragments from IDT
phaR promoter-phaR-phaP promoter (proR-phaR-proP) and primers for amplifying DNA fragment of phaP and HlyA
Confirm the cloning strategy
Figure1. Strategy of R-P-H assembly
Due to the ordered sequence with high content of GC, the first IDT order failed to complete. The sequence of IDT-synthesized DNA was optimized and reordered.
Week 3 - Week 5 Constructs establishment                                                                                      
 
Double digestion of backbone vector with restriction enzymes of SpeI and Pst
Gel electrophoresis and Gel extraction
Amplify phaP and phaP-HlyA followed by gel electrophoresis
   
Figure 2 Images of 1% gel electrophoresis - A.1.0% agarose gel electrophoresis for PCR product of phaP and phaP-HlyA. Lane: 1. 1 kb ladder (molecular weight marker, NEB). 2. 25μL PCR reaction for phaP (569 bp). 3. 25μL PCR reaction for phaP-HlyA (759 bp). B. 1.0% agarose gel electrophoresis for PCR product of proR-phaR-proP. Lane: 1. 100 bp ladder (NEB). 2. 25μL reaction of PCR amplification for proR-phaR-proP (853 bp).
Ligate digested backbone pSB3T5 with phaP and digested backbone with phaP-HlyA, followed by transformation to coli DH5α competent cells.
Order the new primers for sequencing the end of phaP DNA fragment to test whether the stop codon exists.
Send plasmid pSB3T5-phaP-HlyA to Dundee Sequencing Services for Sanger sequencing.
Ligate DNA fragments proR-phaR-proP with pSB3T5-phaP and pSB3T5-phaP-HlyA respectively. Double digest constructs for confirmation.
Figure 3 Image of diagnostic digest of pSB3T5-phaP and pSB3T5-phaP-hlyA with NsiI and PstI. Lanes - 1. 1 kb ladder (molecular weight marker, NEB). 2 and 3. Potential pSB3T5-phaP digested with NsiI and PstI. 4 and 5. Potential pSB3T5-phaP-HlyA digested with NsiI and PstI.
Through sequencing, it was found to contain a stop codon.
Figure 1. The sequence alignment of BBa_K390501, phaR-Phasin-Hly-VR(The sequencing result of our modified part]) and original-phasinHlyA (The sequencing result of BBa_K390501) via Benchling. The selected part of the sequence indicated the stop codon.
Design primers for deleting the stop codon in between the phaP and HlyA.
Week 6 Repeat the experiments and finish constructs establishment                                         
 
Mini-prep for plasmids R-P and R-P-HlyA followed by double digestion. (failed). Results were showed in figure 4a.
Amplify pSB3T5-phaP-HlyA (del) and R-P-HlyA (del) for deleting the stop codon. Running 0.8% agarose gel (figure 4b)
Figure 4 Images of gel electrophoresis. a. Diognostic digest of R-P and R-P-HlyA with EcoRI and PstI. Lanes: 1. 1 kb ladder (molecular weight marker, NEB). 2. Potential R-P digested with EcoRI and PstI. 3. Potential R-P-H digested with EcoRI and PstI. b. 1.0% Agarose gel electrophoresis for PCR product of plasmid pSB3T5-phaP-HlyA (del) and R-P-HlyA (del).Lane: 1. 1kb ladder (NEB). 2. 25μL reaction of PCR amplification pSB3T5-phaP-HlyA. 3. 25μL reaction of PCR amplification R-P-HlyA.
Co-transform the plasmid of phaCAB operon and the construct R-P into coli BL21 competent cells. The recovered cells were spread on the LB agar plates containing 10 μg/ml tetracycline and 25μg/ml chloramphenicol. (failed)
Double digest pSB3T5 backbone to remove the RFP in between the EcoRI and Xba
Transformation:
Repeat the co-transformation of phaCAB operon and R-P with lower concentration of antibiotics (8 μg/ml tetracycline and 20 μg/ml chloramphenicol). 
Transform plasmid R-P into E. coli BL21 competent cells.
Transform the ligation of pSB3T5 without RFP (RFP (-) into E. coli DH5α competent cells.
Colony PCR and double digestion
Figure 5 Image of Agarose gel electrophoresis. a. 1.0% Agarose gel electrophoresis of colony PCR product for screening the positive construct of P.  Lane: 1. 1kb ladder (Promega). 2-14. 25μL reaction of colony PCR amplification for proP-phaP (bp) with different colony extraction.  The positive construct was expected to show a band with size of 1067 bp (proP-phaP + extra sequence in between the VF and VR primers.). b. 1.0% Agarose gel electrophoresis of colony PCR product for screening the positive construct of R.
Mini-prep for plasmid R-P-HlyA (del) and P and transformation.
Week 8 Transformation and colony PCR                                                                                            
 
Measure the optical density of preculture of the recombinant coli BL21 strains that harboured R-P, phaCAB operon+R-P, phaCAB operon+ RFP (-), pSB1C3 (without RFP) +RFP (-) and RFP (-)
Transformation:
Co-transform the phaCAB operon and R-P-HlyA (del) into E. coli BL21 competent cells
Co-transform the phaCAB operon and P into E. coli BL21 competent cells.
Transform the R-P-HlyA (del) into E. coli BL21 competent cells.
Transform the P into E. coli BL21 competent cells.
Colony PCR to fast screen the positive constructs of R.
Figure 6 Image of 1.0% Agarose gel electrophoresis of colony PCR product for screening the positive construct of R - Lane: 1. 1 kb ladder (Promega). 2-14. 25μL reaction of colony PCR amplification for proR-phaR with different colony extraction.
The positive construct was expected to show a band with size of 935 bp (proR-phaR + extra sequence in between the VF and VR primers.
Week 9 - Week 10 Culture bacteria to produce PHB                                                                       
 
Pre-culture all the recombinant coli BL21 strains: phaCABoperon+R-P, phaCAB operon+R-P-HlyA (del), phaCAB operon+p, phaCAB operon+R, phaCAB operon+RFP(-), 1C3-RFP(-)+RFP(-).
The Nile red plate staining
Figure 7 Nile red plates to confirm PHB production (48 hours)
In each picture of a, b, c, d, e, there were 4 Nile red plates. The left plates in the first rows were strains harbouring phaCAB operon and constructs. The right plates in the first row were strains containing the construct only. The left plates in the second row were strains harbouring phaCAB operon only. The right plates in the second row were pSB1C3 and pSB3T5 backbones. phaCAB operon strain showed strong signal of fluorescence because of PHB produced and the backbone strain did not due to no PHB produced.
Inoculate the preculture into 250 ml flasks with 50 ml of M9 medium containing 3.0% glucose, 25 μg/ml chloramphenicol and 10 μg/ml tetracycline.
Measure the optical density to plot the growth curves.
 
Figure 7 The growth curves of E. coli strains that harboured different recombinant constructs
Table 2 PHB production
 
Intracellular PHB
Secreted PHB
Mass of CaCl2
Secreted PHB
phaCAB operon+pSB3T5-R-P
0.071 g
0.057 g
0.05549 g
0.0015 g
phaCAB operon+pSB3T5-R-P-H
0.043 g
0.086 g
0.05549 g
0.0305 g
phaCAB operon+pSB3T5-R
0.065 g
0.058 g
0.05549 g
0.0020 g
phaCAB operon +pSB3T5-P
0.031 g
0.030 g
0.05549 g
0.0000 g
phaCAB operon +pSB3T5
0.010 g
0.038 g
0.05549 g
0.0000 g
pSB1C3+pSB3T5
0.000 g
0.042 g
0.05549 g
0.0000 g
Extract produced PHB (The dry weight of extracted and secreted PHB were listed in the Table 2
Figure 8 Data for dry weight of secreted and intracellular PHB among different stains
Measure the melting point of produced PHB.
Table 3 The melting temperature of produced PHB
Strains
Melting Temperature
phaCAB operon+pSB3T5-R-P
156-164°C
phaCAB operon+pSB3T5-R-P-H
160-168°C
phaCAB operon +pSB3T5-R
156-164°C
phaCAB operon +pSB3T5-P
152-164°C
phaCAB operon+pSB3T5
162-170°C
pSB1C3+pSB3T5
\
 
 
Construction of plasmids harbouring the Sleeping Beauty Mutase operon and methylmalonyl CoA epimerase for producing propionate for PHBV production (Craig)
Week 1 - Week 2 Project Design                                                                                                          
 
Discuss the project aims and decide individual project strategy (shown in Figure 1).
Figure 1 Proposed mechanism for propionate synthesis utilising the Sleeping beauty mutase operon (SBM) and Methylmalonyl-CoA epimerase (MCE) - Succinyl-CoA is converted into Methylmalonyl-CoA-R by the methylmalonyl- CoA mutase ScpA. Methylmalonyl-CoA-R is converted into Methylmalonyl- CoA-S by MCE or an uncharacterised, native pathway. Methylmalonyl-CoA-S is converted into propionyl-CoA by the methylmalonyl-CoA carboxylase ScpB. The CoA from Propionyl-CoA is transferred onto Succinate from the citric acid cycle by the Propionyl-CoA: Succinate CoA transferase ScpC, resulting in the production of propionate and Succinyl-CoA.
Determine whether E. coli  DH5 proposes SBM in genome and if it proposes any change that impact the resulting protein structure and function.
Design and order DNA fragments from IDT to obtain Sleeping Beauty Mutase (SBM) which consists of genes ScpA, ScpB, ScpC, and
Design and order the primers for amplifying SBM operon.
Figure 2 Illustrated diagrams of designed primers - A) The combined primers of Sbm Forward 1 and 2. Sbm Forward 1 annealed to ScpA at its 3’ end, and introduces a Ribosome binding site (RBS; green), AvrII restriction site (black), and half of Promoter BBa_J23110 (orange). Sbm Forward 2 annealed to the amplified sbm operon through containing a complete Promoter BBa_J23110 sequence, and introduced the remaining sequence, alongside XbaI (blue) and EcoRI (red) restriction sites. B) Illustrated diagram of Sbm Reverse Primer. Anneals to the end of ScpC (green), and introduces a SpeI restriction site (orange), and a NsiI restriction site (red).
Week 3 - Week 5 Amplify SBM and construct cloning vectors                                                      
 
Amplify the SBM operon with ordered primers as shown in Figure 1 by tow-step PCR. And analysed by gel electrophoresis.
Figure 3 Gel Electrophoresis of DNA from 2 PCR reactions - PCR-1 is PCR results from the amplification of the SBM operon from E. coli DH5-gDNA, with a band between 6kbp and 5kbp, and a band of below 500bp. PCR-2 used the products from PCR-1 as template DNA and used the Sbm forward 2 and Sbm Reverse primers – a single band of below 500bp can be observed.
TA cloning with SBM to obtain more SBM operon without doing PCR many times.
SBM operon is ‘A-tailed’ prior to ligating into pCRTM2.1-TOPO, and transformed into One Shot® cells and plated. Following incubating overnight, blue colonies and white colonies could be observed. White colonies were used to purify plasmid DNA, and were analysed by digestion with EcoRI.
Figure 4. Analytical digest of TA colonies. Marker ladder used was NEB 1kb ladder, and all samples show a band between 3kbp and 4kbp, and a band below 500bp.
Bands of below 500bp could be observed, suggesting that some of the primer dimers may have been contaminating the extracted SBM operon sample, which were A-tailed, and effectively out- competing the much larger SBM operon fragments to be ligated into pCRTM2.1-TOPO.Again, another cloning method for the utilisation of the SBM operon had to be devised.
Construct pSB3T5: MCE: Promoter- RFP and pSB3T5: Promoter-RFP followed by gel electrophoresis.
Figure 5 Restriction Analysis of various constructs. Marker ladder was Promega 1kb ladder. pSB3T5: MCE: Promoter-RFP was digested with SpeI and/or PstI. Single digests both produced bands of 4757bp, whereas double digest produced 2 bands of 887bp and 3870bp. pSB3T5 was digested with EcoRI and/or PstI. Single digestion results in bands of 4280bp, whereas double digest produced bands of 3211bp and 1069bp. pSB3T5: Promoter-RFP was digested with SpeI and/or PstI. Single digests both produced bands of 4156bp, whereas double digest produced 2 bands of about 3269bp, and 887bp. BBa_J23106 was digested with EcoRI and/or PstI. Single digestion results in bands of 2983bp, whereas double digest produced bands of 2038bp, and 945bp. PSB3T5: MCE was digested with EcoRI and/or PstI. Single digests both produced bands of 3845bp, whereas double digest produced 2 bands of 3211bp and 634bp.
Insert the operon into BioBrick vevtor such as pSB3T5, through digesting with EcoRI and Nsi
Figure 6 a. Diagram illustrating the construction of pSB3T5: MCE. RFP (red) was cleaved from pSB3T5 by digestion with EcoRI and PstI. b. Diagram illustrating the construction of pSB3T5: Promoter-RFP. RFP (red) was excised from pSB3T5 by digesting EcoRI and PstI.
Week 6 - Week 7 Construct pSB3T5: SBM and pSB3T5: MCE: SBM                                            
 
Construct pSB3T5: SBM followed by Colony PCR. The strategy was showed in Figure 7 and gel electrophoresis of colony PCR was showed in Figure 8.
Figure 7 Diagram illustrating the construction of pSB3T5 - SBM. pSB3T5: Promoter- RFP was digested with SpeI and PstI, removing RFP (red). The SBM operon (orange) was digested with AvrII and NsiI. The overhangs generated are illustrated. The SBM operon was ligated into pSB3T5: Promoter downstream of the promoter (purple).
  
Figure 8 a. Image of DNA gel electrophoresis from colony PCR of transformants possessing pSB3T5: SBM using SBM Forward 1 and SBM Reverse Primers. Control lane is the same PCR with no template DNA. Control shows a large empty lane except for a single band below 250bp. All samples show a large smear that is intense from the well down to 5551bp, corresponding to the SBM operon, where it becomes less intense. b. Analytical digest of pSB3T5: SBM using XbaI and SpeI, followed by gel electrophoresis. Uncut sample showed 2 bands, one above 10kbp, and one between 8kbp – 6kbp. XbaI and SpeI single digests show a single band of 8790bp. XbaI and SpeI double digest shows two bands: a pSB3T5 backbone at 3250bp, and the SBM operon at 5540bp.
Two different strategies to construct pSB3T5: MCE: SBM
Figure 9 a. Diagram illustrating the construction of pSB3T5: MCE: SBM. pSB3T5: MCE: Promoter-RFP was digested with SpeI and PstI, removing RFP (red). The SBM operon (orange) was digested with AvrII and NsiI. The overhangs generated are illustrated. The SBM operon was ligated into pSB3T5: Promoter downstream of the promoter (purple). MCE is indicated by black. b. Diagram illustrating the construction of pSB3T5: MCE: SBM. pSB3T5: SBM was digested with EcoRI and XbaI. pSB3T5: MCE was digested with EcoRI and SpeI to excise MCE (brown). The resulting overhangs are displayed. MCE was ligated into the plasmid upstream of the promoter (purple) and the SBM operon (orange), completing the plasmid.
Week 8 Propionate Assay                                                                                                                     
 
Pellets of pSB3T5, pSB3T5: MCE, pSB3T5: SBM, and pSB3T5: MCE: SBM grown in LB broth were resuspended in M9 minimal media with 3% glucose and tetracycline in order to assay for the production of propionate.
Detect the produced propionate by detecting the change in absorbance at 410nm using spectrophotometer.
Fe3+ ions have been shown to react with short chain fatty acids, such as propionate, where the ion complexes with the organic acid and is reduced. This changes the colour of the iron ion, and this change can be detected in a spectrophotometer by a change in absorbance at 410nm. (failed)
Table 1 Absorbance at 410nm of propionate standard solutions
Sample
Absorbance at 410nm (arbitrary units)
0 g/l propionate
0
0.5 g/l propionate
0.017
1.0 g/l propionate
0.003
1.5 g/l propionate
-0.008
2.0 g/l propionate
0.021
2.5 g/l propionate
0.002
3.0 g/l propionate
0.024
3.5 g/l propionate
0.026
4.0 g/l propionate
-0.003
psb3T5
-0.5
psb3T5: MCE
-0.236
psb3T5: SBM
-0.232
psb3T5: MCE: SBM
-0.288
 
 
Investigation of the effects of sucAB and sucCD on the adaption to propionic Acid and PHBV production (Siqi)
Week 1 - Week 2 Amplify essential DNA fragments by ordered primers                                     
 
Design and order primers from IDT
Amplify the sucAB and sucCD from coli DH5genome (Figure 1), sucAB and sucCD are expected to be 4 kb and 2kb respectively.
Figure 1 Gel electrophoresis image of sucAB and sucCD genes from PCR amplification on 1% agarose gel, contrast with 1KB ladder from NEB, generating fragments for the following experiments.
Week 3 - Week 5 Constructs establishment                                                                                      
 
Construct plasmids: pSB3T5-sucAB, pSB3T5-sucCD, plasmid pSB3T5-X
Transform recombinant plasmids to coli DH5
Isolate the plasmids and double digest them with corresponding restriction enzymes for confirmation
Figure 2. A Gel electrophoresis image of pSB3T5-sucAB digested with EcoRI and HindIII, and run on 1% agarose gel, contrast with 1KB ladder from NEB. B.  Gel electrophoresis of pSB3T5-sucCD digested with EcoRI and HindIII, and run on 1% agarose gel, in the third well, there are two bands, the one located above is ringed DNAs are not be digested, sample loaded in this well is not cut completely.
Week 5 - Week 7 Establishment of different E. coli strains                                                            
 
Transform recombinant plasmids to coli forming strains SC1 to SC11. (shown in Table 1)
Table 1 Basic information of engineered E. coli strain
Bacterial strains:
Relevant information
Escherichia coli SC1
Escherichia coli SC2
Escherichia coli SC3
Escherichia coli SC4
Escherichia coli SC5
Escherichia coli SC6
Escherichia coli SC7
Escherichia coli SC8
Escherichia coli SC9
Escherichia coli SC10
Escherichia coli SC11
Escherichia coli BL21 containing pSB3T5-sucAB
Escherichia coli BL21 containing pSB3T5-sucCD
Escherichia coli BL21 containing pSB3T5-X
BL21 containing pSB3T5-sucAB and pSB1C3-PHA
BL21 containing pSB3T5-sucCD and pSB1C3-PHA
BL21 containing pSB3T5-X and pSB1C3-PHA
BL21 containing pSB3T5-AB and empty pSB1C3
BL21 containing pSB3T5-CD empty pSB1C3
BL21 containing pSB3T5-X empty pSB1C3
BL21 containing pSB3T5-sucAB-hypro and pSB1C3-PHA
BL21 containing pSB3T5-sucCD-hypor and pSB1C3-PHA
 
Pre-cultured strain SC1, SC2 and SC3 in LB medium
Inoculate the pre-culture to M9 minimal medium with 1% glucose, 0.01M propionic acid and 10M IPTG.
Optical density at 600nm was measured to plot the growth curve and growth rate was calculated using equation below.
               Table 2 Growth rate of E. coli strains SC1, SC2 and SC3
 
0
2
3
4
5
10
60
72
SC1
0.18
0.503667
0.936667
1.101
1.195333
1.379667
1.3666
1.3666
SC2
0.18
0.489333
1.034
1.238
1.365
1.440667
1.36588
1.36522
SC3
0.28
0.828
1.497333
1.618333
1.755
1.94
1.9
 
Figure 3 Growth curves of strain SC1, strain SC2 and strain SC3 that harboured plasmid pSB3T5-sucAB, plasmid pSB3T5-sucCD and plasmid pSB3T5-X respectively. 1% glucose, 0.01M propionate and 10 μM IPTG were added in the M9 medium.
Week 8 investigate the effect of sucAB and sucCD gene on growth and propionate adaption
 
Pre-culture six different strains to Investigate the effect of sucAB and sucCD gene on growth
Measure the germinate multiple* (GM) of each strain, which represented the proliferation capacity of cells (shown in figure 4)
Figure 4 Images of growth curves and Germinate Multiple - Stacked columns reflect growth rates of each strain under each case. Table at underneath the X-axis diaplays OD600 measurement of six strains under serial propionate concentrations varying from 0 to 0.04 M at the interval of 0.01 M along with time line, time 0: immediately after strains are inoculated into M9 medium, 19: OD600 was measured after cultivation of 19 hours, 24: OD600 was measured after cultivation of 24 hours
Determine the amount of propionic acid in the medium to investigate the effect of sucAB and sucCD gene on propionate uptake. Required standard curve was plotted during the pre-experimental. (shown in Figure 5)
Figure 5 Standard curve and the equation between the absorbance and propionate concentration.
Obtain the amounts of propionic acid that utilized by coli strain SC7, SC8 and SC9.
Figure 6 The bar chart represented the amount of propionate taken by three different strains
*GM defined as final cell concentration / inoculation cell concentration   
Week 9 - Week 10 PHBV production and optimization                                                                    
 
Dry the cells after cultivation of 60 hours.
Nile red plate staining to confirm the production of PHBV
Figure 7 Images of Nile red plate which are exposed to blue light or UV light; all six strains were spread on the plate for overnight culture 
Measure the cell dry weight (CDW).
Figure 8 Absolute cell dry weight of each strain against propionate concentrations, indicating the yield of each strains - It can be seen that when 0.03 M, all three strains have largest absolute dry cell weight, since cultured enough time, the same as PHB production, which corresponding to the largest absorption of propionate in 0.03M
Change the concentration of IPTG. The effect of different IPTG concentration on cell growth and PHBV production were compared
Figure 9 Growth of strain-SC7, SC8 and SC9 under different IPTG concentration, all the cases are cultured in the same condition. The line graph at left is the growth curve of strain-SC7 with IPTG concentration of 0, 0.05M and 0.1 M, the graph in the middle is from strain-SC8 and the line graph at right is strain-SC9.
Measure the fluorescent intensity of three different strains using Plate Reader
Figure 10 Bar chart of Fluorescent intensity - Cells were cultured with different concentrations of IPTG, fluorescent intensity was measured at cultivation of 24 hours and 48 hours
Replace the T7 promoter with hybrid promoter, obtaining plasmids pSB3T5-sucAB-hypro and pSB3T5- sucCD-hypro
 
 
The Phasin and Hemolysin Secretion System (Owen)
Week 1 Design the strategy for plasmid construction                                                                      
 
To investigate and optimized the level of Hemolysin transporter to PHB secretion, PCR strategy and digestion strategy were designed and ;utilised in plasmid construction.
Order the primers
Figure 1 Diagram of new Biobricks development - The development of Lac promoter-Phasin-HlyA without stop codon through PCR strategy. DNA from the Lac promoter-Phasin-HlyA original Biobrick was used as a template to remove the stop codon in the end of Phasin sequence. The PCR product was then digested with DpnI (NEB) to remove the original DNA template then purified with QIAquick PCR Purification Kit (Qiagen), followed by self-ligation.
Figure 2 Diagram of new Biobricks development - The development of pSB3T5-T7-hlyDB-Pro-phaP-hlyA. Several Biobricks were used in this process for assembly, these included T7 promoter, Lac Promoter-PhaP-HlyA, HlyA-tag+Secretion system and pSB3T5-I52001. The purple lines represent the location of enzyme digestion. HlyBD and T7 promoter backbone was first obtained through digestion from their Biobricks then ligated together. The pSB1AK8 backbone of T7 promoter-HlyBD then was replaced by digestion strategy to form T7 promoter-HlyBD/pSB3T5. Lac promoter-Phasin-HlyA without stop codon were used as template to replace its promoter to J23100 promoter through PCT strategy. The parts of J23100 promoter-Phasin-HlyA and T7 promoter-HlyBD/pSB3T5 vector were then ligated together to form T7 promoter-HlyBD-J23100 promoter-Phasin-HlyA/pSB3T5.  
Figure 3 Diagram of new Biobricks development. The development of pSB3T5-T7-hlyDB-phaR-phaP-hlyA 
 
Week 2 - Week 4 Construct establisment                                                                                           
 
Establish a new construct: DNA fragment pSB1AK8-T7-hlyBD (cloning strategy was shown in Figure 2) and transferred in E. coli BL21 (DE3)
Confirmation of successful pSB1AK8-T7-hlyBD plasmid with triple digestion
Establish a new construct: DNA fragment pSB3T5-T7-hlyBD (cloning strategy was shown in Figure 2) and transferred in E. coli BL21 (DE3)
Confirmation of successful pSB3T5-T7-hlyBD plasmid with double digestion
Figure 4 Agarose gel electrophoresis of restriction enzyme-digested Biobricks. A. pSB1AK8-T7-hlyBD plasmid (lane 1: 1 kb DNA ladder; 2: undigested plasmid; 3: EcoR1 digestion; 4: HindIII digestion; 5: PstI digestion; 6: EcoR1, HindIII and PstI triple digestion). B. pSB3T5-T7-hlyBD (lane 0 & 1: 1 kb DNA ladder; 2: undigested plasmid; 3: SpeI digestion; 4: PstI digestion; 5: SpeI and PstI double digestion)
 
Week 4 - Week 6 Construct establishment 2                                                                                     
 
Through sequencing, it was found to contain a stop codon
Figure 1. The sequence alignment of BBa_K390501, LacP-Phasin-Hly-VR-170918-08-27(The sequencing result of BBa K2739007 [our modified part]) and original-phasinHlyA-VR-170918-08-28 (The sequencing result of BBa_K390501) via Benchling. The selected (blue) part of the sequence indicated the stop codon.
Stop codon removal from the original Biobrick (Lac promoter-Phasin-Hemolysin A/ pSB1C3, LPH/pSB1C3) with PCR strategy as shown in Figure 1 and transferred in E. coli BL21(DE3). In the primer design, a HindIII restriction enzyme site, which is not present in the original biobrick, was introduced and allow the confirmation of the success stop codon removal.
Confirmation of with Stop codon removed LPH/pSB1C3 via EcoRI and HindIII double digestion, which the HindIII restriction enzyme was not present in the original Biobrick and introduced through the PCR amplification
Establishment of two new constructs: pSB1C3-T7-hlyBD-Pro-phaP-hlyA and pSB1C3-T7-hlyBD-phaR-phaP-hlyA and transferred in E. coli BL21(DE3)
Confirmation of pSB1C3-T7-hlyBD-JPH and pSB1C3-RPH with EcoRI and PstI double digestion
Week 7 - Week 9 Cell culture for growth study and determination of PHA production              
 
Cell culture for growth study
Culture the E. coli BL21 (DE3), which harbouring the following plasmid(s) in M9 medium that contained 3% glucose with corresponding antibiotic(s) concentration, and assessed their optical density in different time points
pSB1C3, LPH, LPH (without stop codon), pSB1C3 (Red fluorescent protein+), pSB1C3 (RFP-), pSB1C3, PHA operon, PHA operon + pSB1C3 (0, 3, 21, 25, 47, 51 and 71hours);   
T7 promoter-HlyBD - JPH (0, 4, 7.5, 24, 28 and 94 hours); T7 promoter-HlyBD - JPH + PHA operon (0, 3, 20, 24 and 90 hours); in the case of IPTG induction, pSB3T5 (RFP+) and T7 promoter-HlyBD (0, 3.5, 4.5 and 72 hours), T7 promoter-HlyBD - JPH (0, 3.5, 5.5 and 71.5 hours), T7 promoter-HlyBD - JPH + PHA operon (0, 2.5, 19.5, 24.5, 42 and 47 hours), LPH (0, 3.5, 5.5, 71.5, 75.5, 78, 95, 100 and 117 hours), and LPH (without stop codon) (0, 3.5, 5.5, 17, 23 and 71.5 hours)
phaCAB operon + pSB1C3 incubated in 50ml culture (250ml Flask) were measured at 0, 18, 24.5, 39.5, 43.5 and 63.5 hours)
Figure 5 Agarose gel electrophoresis of Phasin-HlyA products. A. PCR product of Lac promoter-Phasin-HlyA with stop codon removal. B. Enzyme digestion of Lac promoter-Phasin-HlyA PCR product for stop codon removal (lane 1: 1 kb DNA ladder; 2: EcoRI and HindIII   double digestion) with the label of HlyA + pSB1C3 Backbone and Phasin. C. PCR product of pSB1C3-Lac promoter-Phasin-HlyA (stop codon -) with J23100 promoter forward and reverse primers 1; D. PCR product of pSB1C3-Lac promoter-Phasin-HlyA (stop codon -) with J23100 promoter forward and reverse primers 2; E. Enzyme digestion of constructed T7 promoter-HlyBD-JPH plasmid andT7 promoter-HlyBD-RPH plasmid (lane 1 & 4: 1 kd DNA ladder; 2-3: EcoRI and PstI double digestion for T7 promoter- HlyBD-JPH plasmid; 5-6 EcoRI and PstI double digestion for T7 promoter-HlyBD-RPH plasmid). 
Determination of PHA production
Nile red plate staining  
Figure 7 The growth study of E. coli Bl21 (DE3) strain with constructed HlyBD-Phasin-HlyA plasmid with and without IPTG (Triplicate). The results are represented as the mean OD600 ± S.E.M.
Figure 8 The growth study of PHA operon - The results are represented as the mean OD600 ± S.E.M. A. The growth study of E. coli BL21 (DE3) strain with pSB1C3, PHA operon, phaCAB operon+ pSB3T5 (RFP-) in 50ml tube (performed in triplicate). B. The growth study of E. coli BL21 (DE3) strain with phaCAB operon+pSB3T5 (RFP-) in 250ml Flask.  
Figure 9 The growth study of E. coli Bl21 (DE3) strain with pSB3T5-T7-hlyDB-phaP-hlyA and phaCAB operon with and without IPTG (Triplicate). The results are represented as the mean OD600 ± S.E.M.    
Week 9 - Week 10 Cell culture for PHA production                                                                          
 
Culture the E. coli BL21 (DE3) to investigate PHA production in different condition under M9 medium that contained 3% glucose with corresponding antibiotic(s) concentration
Measure fluorescent intensity of cultures to provide real-time information of PHA production. (semi-quantitative Nile red measurement) 
 Table 1 PHB production of recombinant E. coli BL21 (DE3) strain with PHA operon from 72 hours bacterial culture in M9 medium with 0.3% glucose (performed in singular). The results of OD600 are represented as the mean OD600 ± S.E.M. 
 
Dry cell mass (g)  
PHB production (g) 
Melting Temperature (°C) 
OD600 
1 litre flask incubation (200ml culture sample) 
0.59 
0.008 
165-170 
2.244 ± 
0.014 
500ml flask incubation (100ml culture sample) 
0.285 
0.001   
168-175 
1.629 ± 
0.035 
250ml flask incubation (50ml culture sample) 
0.17 
Not measurable   
 160-169 
1.913 ± 
0.013 
 
 
Table 2 The PHB production of E. coli BL21 (DE3) strain (PHA operon + pSB3T5) cell culture for 62 hours in M9 medium with 3% glucose (performed in duplicate)  
 
Extracellular fraction 
Intracellular fraction 
PHB Production 
Amount of PHB in 
CaCl2 added 
Corrected amount of PHB 
Amount of PHB 
Melting Temperature 
50ml culture sample X 2 
0.016±0g 
0.110g 
0g 
0.011±0g 
165-170°C 
0.11g/L 
50 ml culture sample 
0.0085±0g 
0.055g 
0g 
0.006±0g 
168-175°C 
0.12g/L 
 
Figure 10 Extracted product via PHB extraction protocol - A. Extracted product from intracellular fragment. B. Extracted product from extracellular fragment.    
 
Figure 11 Fluorescence intensity detection of E. coli BL21(DE3) strain with pSB1C3-T7-hlyBD-JPH and phaCAB operon under Nile red stain (performed in triplicate) - Results are represented as the mean fluorescent strength ± S.E.M. measured at 520 nm excitation and 590 nm emission wavelengths in 24 and 48 hours. Cultured in M9 medium with 3% glucose (performed in duplicate)
The Glucose Toggle Switch, InterLab, and Pot Ale experiments (Jackson)
Week 1 - Week 20                                                                                                                                   
 
Jackson's notebook can be found here
  
 
