<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_OG_PHBV_Production skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh OG/PHBV Production</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV></DIV><NAV class="collapse navbar-collapse navbar-right" role="navigation"><DIV class="main-menu"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A tabindex="-1" href="https://2018.igem.org/Team:Edinburgh_OG/project_background">Project Background</A></LI><LI class="dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown-submenu">Project Design </A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/PHBV_Production">PHA Synthesis</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Downstream_Processing">Downstream Processing</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Scaling_Up">Scaling Up</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Toggle_Switch">Glucose Toggle Switch</A></LI></UL><DIV><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/InterLab">InterLab Study</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Parts">Parts List</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Demonstrate">Demonstrate</A></LI></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Practices </A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Human_Practices">Integrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/life_cycle_assessment">Life Cycle Assessment</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Safety">Safety</A></LI></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Notebook">Weekly Journal</A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Protocol">Protocol</A></LI></NAV></DIV><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Team">Team</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Acknowledgements </A></LI><LI><A href="https://2018.igem.org/Team:Edinburgh_OG/Attributions">Attributions</A></LI></DIV><LI class="medals"><A href="https://2018.igem.org/Team:Edinburgh_OG/Medals">Medal</A></LI></DIV></DIV><DIV><DIV><P> </P></DIV></DIV><DIV><DIV><P> </P></DIV></DIV><DIV><DIV><P> </P></DIV></DIV><DIV><DIV><P> </P></DIV></DIV><DIV><DIV><P> </P></DIV></DIV><H1 style="text-align: center;"><STRONG>Improvement on PHA Production</STRONG></H1><DIV class="column two_third_size"><P style="text-align: justify;">Modern life is reliant on the use of plastics, and since their production began on a large scale over 8.3 million tons have been produced, of which over 6.3 million tons has been thrown away, with the majority accumulating in landfill, or the environment (Geyer et al., 2017). Drawn by increasing demand of plastics and sustainable development request, the general mindset was shifted towards developing completely natural biodegradable plastics. Polyhydroxyalkanoate (PHA) is a huge family of bio-derived and biodegradable polymers belonging to the polyesters class that are also termed “Microbial Plastics” (Bonartsev et al., 2017). The biopolymers are commonly produced among a wide array of gram-positive and gram-negative bacteria but the highest occurrence has been attributed to <EM>Cupriavidus necator</EM> (<EM>C. necator</EM>, previously known as <EM>Rastonia eutropha</EM>), which is the main chassis for current commercial production of PHA-based bioplastics.
</P><P>Because of the characteristics of being water insoluble, nontoxic and degradable, poly(3- hydroxybutyrate) (PHB) and poly(3-hydroxybutyrate-co-3-hydroxyvalerate) (PHBV) became two major members of this family that have attracted our attentions most (Zakaria et al., 2010). PHB is the most prevalent biopolymer of this family and has been investigated fervently as a bio-based alternative to petrochemical-derived plastics. Within specific conditions of nutrients and carbon sources, co-polymers – that is, polyesters comprising of more than one type of monomer – can also be synthesized. PHB is frequently the predominant monomer but various co-polymers exist that are built up from larger species (i.e. polyhydroxyvalerate, polyhydroxyhexanoate, etc.). PHA co-polymers are currently an area of interest due to the diversity of properties (e.g. elasticity, crystallinity, melting point, etc.) that emerge as a result of their relative monomer compositions and their corresponding side chain moieties. It has been shown that a higher molar ratio of the hydroxyvalerate component resulted in increased ductility, strength and a larger thermal processing window. Effectively this means that across a range of monomer compositions, PHBV can be modified to become usable in 3D printers (which operate within specific melting temperatures of the bioplastic “ink), used in consumer packaging or even applied in the biomedical context as components of slow-release drug formulations and orthopedic devices. 
</P><P>
In this project, we aim to construct PHB and PHBV synthesis pathways by introducing <EM>phaCAB</EM>  operon, SBM operon, and other functional genes into <EM>E. coli</EM> to improve the efficiency in production of PHB/PHBV.</P><P style="text-align: center;"><STRONG>Figure 1</STRONG> Comparison between the molecular structure of PHB and PHBV</P><P style="text-align: justify;"> </P><H4 style=" font-size:150% ">Investigation of the optimised culture conditions for  <EM>E.coli</EM>  harbouring <EM>phaCAB</EM>  operon on growth and PHA production</H4><P style="text-align: justify;">When investigating the effect of <EM>phaCAB</EM>  on PHB production, the effect of different culture conditions on recombinant <EM>E. coli</EM>  harbouring the <EM>phaCAB</EM>  operon should be known.  Herein, culture conditions with different glucose concentration were assessed to determine the optimised condition(s) on growth and PHA production. Moreover, due to the importance of propionic acid in producing PHBV, its effect on <EM>E.coli</EM>  growth was also assessed and the result was used as a reference for the Bktb gene research project. Comparison of the different growth curves indicated that cells with 1 %, 2 % and 3 % glucose reached stationary phase with the same level after 50 hours cultivation while culture conditions with glucose concentration higher than 3% or input of propionic acid will compress the cell growth.</P><DIV class="container"><BUTTON class="btn btn-info" type="button" data-toggle="collapse" data-target="#tittle1">Read more</BUTTON><DIV id="tittle1" class="collapse"><H3 style="text-align: justify;"><STRONG>Background</STRONG><STRONG> </STRONG></H3><P style="text-align: justify;"> In order to improve the productivity of PHA, the culture condition optimisation was performed to investigate the ideal glucose concentration and propionic acid concentration.  Currently, PHB and PHBV as biodegradable plastic candidates are being produced in a two-stage glucose/propionate fed batch fermentation progress using <EM>Cupriavidus necator</EM> or recombinant <EM>E. coli</EM> (Masani et. al, 2009). In order to give the real-time data of PHA production during the cultivation, the new real-time productivity measurement was first established and tested with Nile red fluorescent dye through the Image J program, then followed by assessment of the dry PHB weight.</P><H3 style="text-align: justify;"><STRONG>Aims</STRONG> </H3><P style="text-align: justify;">In this sub-project, the main objective is to find the effect of optimised culture conditions for  <EM>E.coli</EM>  harbouring <EM>phaCAB</EM>  operon on growth and PHA production. These results then indicate the best culture condition(s) for the rest of the group.</P><H3 style="text-align: justify;"><STRONG>Materials and Methods</STRONG></H3><P style="text-align: justify;">In the experiment, recombinant <EM>E. coli</EM>  harbouring <EM>phaCAB</EM>  were used to culture with different conditions up to 56 hours.</P><H3 style="text-align: justify;"><STRONG>Results and Discussion</STRONG></H3><P style="text-align: justify;">The iGEM Edinburgh OG 2018 team decided to characterize the ideal glucose concentration for Hybrid promoter with PHA operon. E. coli cells were cultured with different concentrations of glucose at 37 ℃ and optical density was measured at cultivation time of 6 hours, 24 hours, 30 hours 48 hours, 56 hours and 72 hours. In order to assess the Real-time PHA detection by Semi-quantitative analysis (Nile Red Fluorescent intensity-through using Image J program). Cell cultures were harvested at different cultivation hours including 6 hours, 24 hours, 30 hours and 48 hours. The bars in blue and orange represented signal intensity of cells harboured pSB1C3-phaCAB and pSB1C3 respectively. The bars in yellow represented signal intensity resulting from PHA, which were calculated from the difference between intensity of pSB1C3-phaCAB and intensity of pSB1C3. The team then decided to determine the influence of glucose availability on PHA production level, in addition to the growth curves with different glucose concentrations.</P><P style="text-align: center;"><STRONG>Figure 1 </STRONG>Comparison of growth curve of recombinant E. coli harbouring pSB1C3-phaCAB for different concentration of glucose.</P><P> </P><P style="text-align: center;"><STRONG>Figure 2 </STRONG>The fluorescent intensity of PHA produced with different glucose concentrations. (48 hours cultivation). Error bars represented standard deviations.</P><P> </P><P style="text-align:center;"><STRONG>Table 1</STRONG> Yield of PHA of pSB1C3-phaCAB with different glucose concentrations</P><P> </P><P style="text-align: center;"><STRONG>Figure 3. </STRONG>Fluorescent intensity of cells harboured pSB1C3 or pSB1C3-phaCAB at different cultivation time.</P><P> </P><P style="text align:justify;">In order to determine the tolerance of propionic acid, which allow the later investigation with the <EM>Bktb</EM>   gene, E. coli harbouring pSB1C3-phaCAB plasmids were cultured with different concentration of glucose and propionic acid for 56 hours. </P><P style="text-align: center;"><STRONG>Figure 4 </STRONG>Comparison of growth curves with different concentration of glucose and propionic acid. The time of adding propionic acid was pointed out by red arrow. </P><H3 style="text-align: justify;"><STRONG>References</STRONG> </H3><UL><LI style="text-align: justify;">Masani, M.Y.A., Parveez, G.K.A. and Izawati, A.M.D. 2009. Construction of PHB and PHBV multiple-gene vectors driven by an oil palm leaf-specific promoter. <EM>Plasmid</EM>, , 62(3), pp.191-200.</LI></UL></DIV></DIV><DIV><P> </P></DIV><H4 style=" font-size:150% ">Investigation of the best gene order of the <EM>phaCAB</EM>  operon for PHB production</H4><P style="text-align: justify;">When investigating the effect of gene order of <EM>phaCAB</EM>  on PHB production, five new constructs were established by Gibson assembly and succesful constructs were introduced to <EM>E. coli</EM>  BL21 (DE3) strains. Those five recombinant <EM>E. coli</EM>  strains would be cultured and compared with strains that harboured original <EM>phaCAB</EM>  operon. The highest yield of PHB among the new constructs was achieved by <EM>E. coli</EM> that expressed the <EM>phaACB</EM>  plasmid, which had a similar gene order with the original <EM>phaCAB</EM>  operon, while none of the new operons gave rise to higher yields than the original operon.</P><DIV class="container"><BUTTON class="btn btn-info" type="button" data-toggle="collapse" data-target="#tittle2">Read more</BUTTON><DIV id="tittle2" class="collapse"><H3 style="text-align: justify;"><STRONG>Background</STRONG><STRONG> </STRONG></H3><P style="text-align: justify;">Various gram-negative bacteria are known to synthesise PHAs, among which <EM>Ralstonic eutropha</EM>  is regarded as the model bacterium of PHA synthesis because three important genes of <EM>phaA</EM>, <EM>phaB</EM>  and <EM>phaC</EM>  are discovered in its genome (Moorkoth and Nampoothiri, 2016). These three genes which have been organised into the operon <EM>phaCAB</EM>  encode three essential enzymes for production of PHB. Some research showed that genes positioned closer to the promoter could have higher expression than the genes further downtream, and this expression level significant affect the activity of essential enzymes for PHB production, PHB molecular weight and accumulation level (Hiroe et al., 2012).</P><H3 style="text-align: justify;"><STRONG>Aims</STRONG> </H3><P style="text-align: justify;">In this sub-project, the main objective is to find the effect of gene order of <EM>phaCAB</EM>  on PHB production. Moreover, we aim to optimize the order of <EM>phaA</EM> , <EM>phaB</EM>  and <EM>phaC</EM>  genes for obtaining the most balanced PHB production.</P><H3 style="text-align: justify;"><STRONG>Materials and Methods</STRONG></H3><UL><LI><STRONG>Construct establishment</STRONG></LI><P style="text-align: justify;">All the gene fragments with different overhangs were amplified by PCR amplification. Then Gibson Assembly was used to assemble all the fragments and build constructs in new gene orders.</P><P style="text-align: center;"><STRONG>Figure 1 </STRONG>The workflow of Gibson assembly - Fragments are amplified with overlap, followed by incubation at 50˚C for 15-60 min, with Gibson Assembly Master Mix, which includes three essential enzymes (5´ exonuclease, DNA polymerase and DNA ligase). Those essential enzymes enable new construct to be assembled (Chan).</P><P> </P><LI><STRONG>PHB production confirmation and measurement</STRONG></LI><P style="text-align: justify;">Recombinant strains were cultured in M9 medium with 3% glucose and 25g/ml chloramphenicol. By measuring optical density of cells that harboured different constructs including <EM>phaCAB</EM>, <EM>phaCBA</EM>, <EM>phaACB</EM>, <EM>phaACB</EM>, <EM>phaACB </EM>and<EM> phaABC</EM>, the effect of gene order on cell growth could be determined. The production of PHB is confirmed by Nile red plate/culture staining and quantitatively determined by measuring dry weight of extracted PHB.</P></UL><H3 style="text-align: justify;"><STRONG>Results and Discussion</STRONG></H3><P style="text-align: justify;">Four constructs including <EM>phaCAB</EM>, <EM>phaCBA</EM>, <EM>phaACB</EM>  and <EM>phaBCA</EM>  were successfully constructed while the establishment of <EM>phaACB</EM>  and <EM>phaBAC</EM>  failed due the inappropariate primers design. <STRONG> </STRONG>Nile red plate staining confirmed PHB production from cells that harboured four new constructs (<EM>phaCAB</EM>, <EM>phaCBA</EM>, <EM>phaACB</EM>  and <EM>phaBCA</EM>). By measuring the yield of produced PHB, the most balanced PHB production is from <EM>E. coli</EM>  that harboured the original <EM>phaCAB </EM>  and <EM>phaACB</EM>  operon.</P><P> </P><P style="text-align: center;"><STRONG>Table 1 </STRONG>The yield of produced PHB</P><TABLE style="margin-left: auto; margin-right: auto; width: 620.5px;"><TBODY><TR><TD style="width: 113px;"><P style="text-align: center;"><STRONG>Plasmid</STRONG></P></TD><TD style="width: 150px;"><P style="text-align: center;"><STRONG>Culture volume</STRONG></P></TD><TD style="width: 157px;"><P style="text-align: center;"><STRONG>Extracted PHB (g)</STRONG></P></TD><TD style="width: 181.5px;"><P style="text-align: center;"><STRONG>Yield of PHB (mg/ml)</STRONG></P></TD></TR><TR><TD style="width: 113px;"><P><STRONG>pSB1C3</STRONG></P></TD><TD style="text-align: center; width: 150px;"><P style="text-align: center;">150ml</P></TD><TD style="text-align: center; width: 157px;"><P style="text-align: center;">0</P></TD><TD style="text-align: center; width: 181.5px;"><P style="text-align: center;">0</P></TD></TR><TR><TD style="width: 113px;"><P><STRONG><EM>phaCAB</EM></STRONG></P></TD><TD style="text-align: center; width: 150px;"><P style="text-align: center;">150ml</P></TD><TD style="text-align: center; width: 157px;"><P style="text-align: center;">0.237</P></TD><TD style="text-align: center; width: 181.5px;"><P style="text-align: center;">1.58</P></TD></TR><TR><TD style="width: 113px;"><P><STRONG><EM>phaCBA</EM></STRONG></P></TD><TD style="text-align: center; width: 150px;"><P style="text-align: center;">150ml</P></TD><TD style="text-align: center; width: 157px;"><P style="text-align: center;">0.072</P></TD><TD style="text-align: center; width: 181.5px;"><P style="text-align: center;">0.48</P></TD></TR><TR><TD style="width: 113px;"><P><STRONG><EM>phaACB</EM></STRONG></P></TD><TD style="text-align: center; width: 150px;"><P style="text-align: center;">150ml</P></TD><TD style="text-align: center; width: 157px;"><P style="text-align: center;">0.201</P></TD><TD style="text-align: center; width: 181.5px;"><P style="text-align: center;">1.34</P></TD></TR><TR><TD style="width: 113px;"><P><STRONG><EM>phaBCA</EM></STRONG></P></TD><TD style="text-align: center; width: 150px;"><P style="text-align: center;">150ml</P></TD><TD style="text-align: center; width: 157px;"><P style="text-align: center;">0.021</P></TD><TD style="text-align: center; width: 181.5px;"><P style="text-align: center;">0.14</P></TD></TR></TBODY></TABLE><P style="text-align: center;">Note: <EM>E. coli</EM>  that harboured different constructs were cultured in M9 medium with 3% glucose for 48 hours at 37 ℃ shaker.</P><P style="text-align: justify;"><STRONG> </STRONG></P><H3 style="text-align: justify;"><STRONG>Future Work</STRONG></H3><P style="text-align: justify;">Further study about new constructs remains to be performed. For instance, the enzyme activities assays help to figure out how enzyme activities are affected by the different gene order.</P><H3 style="text-align: justify;"><STRONG>References</STRONG> </H3><UL><LI>Geyer, R., Jambeck, J.R. and Law, K.L., 2017. Production, use, and fate of all plastics ever made. <EM>Science advances</EM>, 3(7), p.e1700782.</LI><LI>Bonartsev, A.P., Zharkova, I.I., Yakovlev, S.G., Myshkina, V.L., Mahina, T.K., Voinova, V.V., Zernov, A.L., Zhuikov, V.A., Akoulina, E.A., Ivanova, E.V. and Kuznetsova, E.S., 2017. Biosynthesis of poly (3-hydroxybutyrate) copolymers by Azotobacter chroococcum 7B: A precursor feeding strategy. <EM>Preparative Biochemistry and Biotechnology</EM>, 47(2), pp.173-184.</LI><LI>Zakaria, M.R., Ariffin, H., Johar, N.A.M., Abd-Aziz, S., Nishida, H., Shirai, Y. and Hassan, M.A., 2010. Biosynthesis and characterization of poly (3-hydroxybutyrate-co-3-hydroxyvalerate) copolymer from wild-type Comamonas sp. EB172. <EM>Polymer Degradation and Stability</EM>, 95(8), pp.1382-1386.</LI><LI>Moorkoth, D. and Nampoothiri, K.M., 2016. Production and characterization of poly (3-hydroxy butyrate-co-3 hydroxyvalerate)(PHBV) by a novel halotolerant mangrove isolate. <EM>Bioresource technology</EM>, 201, pp.253-260.</LI><LI>Siu-Hong Chan, Ph.D., New England Biolabs, Inc.: Restriction Endonucleases: Molecular Cloning and Beyond. <A href="https://international.neb.com/tools-and-resources/feature-articles/restriction-endonucleases-molecular-cloning-and-beyond%202018/8/10">https://international.neb.com/tools-and-resources/feature-articles/restriction-endonucleases-molecular-cloning-and-beyond 2018/8/10</A>.</LI><LI>Hiroe, A., Tsuge, K., Nomura, C.T., Itaya, M. and Tsuge, T., 2012. Rearrangement of gene order in the <EM>phaCAB</EM> operon leads to effective production of ultra-high-molecular-weight poly [(R)-3-hydroxybutyrate] in genetically engineered <EM>Escherichia coli</EM>. <EM>Applied and environmental microbiology</EM>, pp.AEM-07715.</LI></UL></DIV></DIV><DIV><P> </P></DIV><H4 style=" font-size:150% ">Improvement of PHBV production by introducing <EM>bktB</EM>  to <EM>E. coli</EM></H4><P style="text-align: justify;">In this study, we constructed a PHBV synthesis pathway (shown in Figure 1) by introducing the <EM>phaA</EM>, <EM>phaB</EM> and <EM>phaC</EM> genes into <EM>E. coli</EM> BL21 (DE3). In order to enhance the 3HV fraction in PHBV, paralog <EM>bktB</EM> was introduced into <EM>E. coli</EM> BL21 (DE3) with co-expression of <EM>phaCAB</EM> operon from <EM>Ralstonia eutropha</EM>. And the effect of <EM>bktB</EM> on PHBV synthesis was further investigated by replacing <EM>phaA </EM>with <EM>bktB. </EM>With the cell culture condition optimisation, 3 % glucose and 8 mM propionic acid was the 
 correct feeding strategy for PHBV production. Cells harbouring pSB1C3-<EM>phaCB</EM>-<EM>bktB</EM> showed great potential to improve production of PHBV with a higher 3HV fraction.</P><DIV class="container"><BUTTON class="btn btn-info" type="button" data-toggle="collapse" data-target="#tittle3">Read more</BUTTON><DIV id="tittle3" class="collapse"><P style="text-align: center;"><STRONG>Figure 1 </STRONG>Schematic illustration of the pathways leading to the PHBV biosynthesis</P><H3><STRONG>Background</STRONG></H3><P style="text-align: justify;">As a typical role in the PHAs family, poly(3-hydroxybutyrate-co-3- hydroxyvalerate) (PHBV) is more likely to be a potential candidate for thermoplastic because of its higher thermal stability and flexibility which could be optimised by adjusting 3-hydroxyvalerate (3HV) fractions (Yu et al., 2005). <EM>Escherichia coli</EM> is one of the best-studied bacteria and is used as an ideal host for PHB and PHBV production because of its well-studied genetics and metabolism. In addition, a high-cell-density cultivation strategy contributed to improve polymer yield and productivity (Shojaosadati et al., 2008). Whole genome analysis of <EM>R. eutropha</EM> H16 identified several genes as paralogous to the <EM>phaA</EM>, and the <EM>bktB</EM> was isolated from <EM>R. eutropha</EM> H16 as the most important paralogous gene for PHBV production since it showed higher substrate specificity to the C5 monomer and used 3-ketovaleryl-CoA more efficiently (Mifune et al., 2010). </P><H3 style="text-align: justify;"><STRONG>Aims</STRONG></H3><P style="text-align: justify;">We aim to enhance the 3HV fraction in PHBV, by co-expressing paralog <EM>bktB</EM> with <EM>phaCAB</EM> operon. In order to improve the productivity of PHBV, the culture condition optimization was performed to investigate the ideal glucose concentration and propionic acid concentration. </P><H3><STRONG>Materials and Methods</STRONG></H3><P style="text-align: justify;">In this study, we constructed a PHBV synthesis pathway by introducing the <EM>phaA</EM>, <EM>phaB</EM> and <EM>phaC</EM> genes into <EM>E. coli</EM> BL21 (DE3). To improve the 3HV fraction in the copolymer, the <EM>phaA</EM> paralog <EM>bktB</EM> from <EM>R. eutropha</EM> H16 was introduced into <EM>E. coli</EM> as co-expression or replacement of <EM>phaA</EM>. To optimise the culture conditions for PHBV production, different concentrations of glucose and propionic acid were applied. The yield of accumulated intercellular PHA was first determined after the extraction and its thermal properties were fist determined by melting temperature measurement. </P><H3><STRONG>Results and Discussion</STRONG></H3><P><EM>E. coli</EM> strain BL21 (DE3) that harboured these two plasmids was spread on the Nile red agar plates with negative control (pSB1C3) respectively, and the two plates were exposed to blue light. Compared with negative control, strong Nile red fluorescence observed from strains that harboured either pSB1C3-<EM>phaCAB</EM>-<EM>bktB</EM> or pSB1C3-<EM>phaCB</EM>-<EM>bktB</EM> indicated that PHA (PHB and PHBV) production was assessed after 24 hours.</P><P style="text-align: center;"><STRONG>Figure 2 </STRONG>Nile red agar plate detection of PHA production - Paralogous gene <EM>bktB</EM> represented similar function with <EM>phaA</EM> gene in the pathway, which showed higher specificity to C5 monomers contributed to the PHBV productivity and 3HV fraction. Although Gas Chromatography remained to be done to analyse PHBV composition, lower melting temperature still gave strong suggestion that replacing <EM>phaA</EM> gene with <EM>bktB</EM> could significantly increase the PHBV content in PHA production and co-expression of two genes would show small increase of PHBV production. Combined with the culture condition optimisation, cells harbouring pSB1C3-<EM>phaCB</EM>-<EM>bktB </EM>showed great potential to improve production of PHBV with higher 3HV fraction.</P><P style="text-align: center;"><STRONG>Table 1  </STRONG>Melting temperatures of PHA from various sources</P><TABLE style="margin-left: auto; margin-right: auto;" width="0"><TBODY><TR><TD style="text-align: center; width: 247px;"><P style="text-align: center;"><STRONG> </STRONG></P></TD><TD style="text-align: center; width: 112px;"><P style="text-align: center;"><STRONG>T<SUB>m</SUB> 1(°C</STRONG><STRONG>)</STRONG></P></TD><TD style="text-align: center; width: 100px;"><P style="text-align: center;"><STRONG>T<SUB>m</SUB> 2 (°C</STRONG><STRONG>)</STRONG></P></TD><TD style="text-align: center; width: 114px;"><P style="text-align: center;"><STRONG>T<SUB>m</SUB> 3 (°C</STRONG><STRONG>)</STRONG></P></TD></TR><TR><TD style="text-align: left; width: 247px;"><P><STRONG>Pure PHB product from Sigma </STRONG></P></TD><TD style="text-align: center; width: 112px;"><P style="text-align: center;">170-179</P></TD><TD style="text-align: center; width: 100px;"><P style="text-align: center;">168-176</P></TD><TD style="text-align: center; width: 114px;"><P style="text-align: center;">168-174</P></TD></TR><TR><TD style="text-align: left; width: 247px;"><P><STRONG>PHBV with 12% 3HV from Sigma</STRONG></P></TD><TD style="text-align: center; width: 112px;"><P style="text-align: center;">159-161</P></TD><TD style="text-align: center; width: 100px;"><P style="text-align: center;">160-160</P></TD><TD style="text-align: center; width: 114px;"><P style="text-align: center;">161-164</P></TD></TR><TR><TD style="text-align: left; width: 247px;"><P><STRONG>PHA from pSB1C3-phaCAB</STRONG></P></TD><TD style="text-align: center; width: 112px;"><P style="text-align: center;">160-168</P></TD><TD style="text-align: center; width: 100px;"><P style="text-align: center;">160-162</P></TD><TD style="text-align: center; width: 114px;"><P style="text-align: center;">161-163</P></TD></TR><TR><TD style="text-align: left; width: 247px;"><P><STRONG>PHA from pSB1C3-phaCB-bktB</STRONG></P></TD><TD style="text-align: center; width: 112px;"><P style="text-align: center;">150-155</P></TD><TD style="text-align: center; width: 100px;"><P style="text-align: center;">149-151</P></TD><TD style="text-align: center; width: 114px;"><P style="text-align: center;">149-152</P></TD></TR><TR><TD style="text-align: left; width: 247px;"><P><STRONG>PHA from pSB1C3-phaCAB-bktB</STRONG></P></TD><TD style="text-align: center; width: 112px;"><P style="text-align: center;">155-159</P></TD><TD style="text-align: center; width: 100px;"><P style="text-align: center;">156-161</P></TD><TD style="text-align: center; width: 114px;"><P style="text-align: center;">157-159</P></TD></TR><TR><TD style="text-align: left; width: 247px;"><P><STRONG>PHB extraction </STRONG></P></TD><TD style="text-align: center; width: 112px;"><P style="text-align: center;">168-180</P></TD><TD style="text-align: center; width: 100px;"><P style="text-align: center;">166-178</P></TD><TD style="text-align: center; width: 114px;"><P style="text-align: center;">169-179</P></TD></TR></TBODY></TABLE><H3><STRONG>Future Works</STRONG></H3><P>Gas chromatography remains to be executed to give more specific information about the composition of extracted PHA products including the percentage of PHBV content and the fraction of 3HV in PHBV, which are essential for confirming the effect of <EM>bktB</EM> on PHBV production.</P><H3><STRONG>References</STRONG></H3><UL><LI>Yu, S.T., Lin, C.C. and Too, J.R., 2005. PHBV production by <EM>Ralstonia eutropha</EM> in a continuous stirred tank reactor. <EM>Process Biochemistry</EM>, 40(8), pp.2729-2734.</LI><LI>Shojaosadati, S.A., Varedi Kolaei, S.M., Babaeipour, V. and Farnoud, A.M., 2008. Recent advances in high cell density cultivation for production of recombinant protein. <EM>Iranian Journal of Biotechnology</EM>, 6(2), pp.63-84.</LI><LI>Mifune, J., Nakamura, S. and Fukui, T., 2010. Engineering of pha operon on <EM>Cupriavidus necator</EM> chromosome for efficient biosynthesis of poly (3-hydroxybutyrate-co-3-hydroxyhexanoate) from vegetable oil. <EM>Polymer Degradation and Stability</EM>, 95(8), pp.1305-1312.</LI></UL></DIV></DIV><DIV><P> </P></DIV><H4 style=" font-size:150% ">Investigation of the effect of phasin autoregulation system on PHB production</H4><P style="text-align: justify;">When investigating the influence of <EM>PhaR</EM> autoregulation system coupled with phasin on PHB production and cell growth, <EM>phaR</EM> and <EM>phaP</EM> which encode the <EM>PhaR</EM> regulation system and phasin respectively were introduced into <EM>E. coli</EM> strain BL21 (DE3) with the <EM>phaCAB</EM> operon, forming new constructs pSB3T5-<EM>ProR</EM>-<EM>phaR</EM>-<EM>ProP</EM>-<EM>phaP</EM>. Furthermore, another construct pSB3T5-<EM>ProR</EM>-<EM>phaR</EM>-<EM>ProP</EM>-<EM>phaP</EM>-<EM>hlyA</EM> was established where the <EM>hlyA</EM> was expressed with the <EM>PhaR</EM> regulation system and phasin to investigate the effect of the <EM>hlyA</EM> depending secretion system coupled with the <EM>PhaR</EM> regulation system and phasin on PHB production.  The highest yield of PHB was achieved by the strain that expressed pSB3T5-<EM>ProR</EM>-<EM>phaR</EM>-<EM>ProP</EM>-<EM>phaP</EM>-<EM>hlyA</EM>. In addition, the presence of the <EM>PhaR</EM> regulator could effectively regulate the expression of <EM>phaP</EM> and moderate the biosynthesis of PHB.</P><DIV class="container"><BUTTON class="btn btn-info" type="button" data-toggle="collapse" data-target="#tittle4">Read more</BUTTON><DIV id="tittle4" class="collapse"><H3><STRONG>Background </STRONG></H3><P style="text-align: justify;">Low molecular weight protein phasin encoded by gene <EM>phaP</EM>, is able to bind to the surface of PHA granules and play essential roles in PHB sysnthesis and granule formation (York et al., 2001). The expression of phasin is regulated by the autoregulated repressor <EM>PhaR</EM> which is able to bind to the region of <EM>phaP</EM> promoter and its own promoter to regulate phasin and itself. In addition, <EM>PhaR</EM> was detected binding on the surface of PHA granules, which potential help cells to save energy by curtailing excessive expression of PHA biosynthesis pathways (Pötter et al., 2002). Protein can be secreted through the type I secretion pathway of <EM>E. coli </EM>by co-expressing <EM>HlyA</EM> signal peptide which is able to interact with <EM>HlyB</EM>/<EM>HlyD</EM> complex (Mergulhao et al., 2005). The sequence amplified from the BioBrick BBA_K390501 was confirmed to have a stop codon between phasin and <EM>hlyA</EM> by sequencing. Due to the existence of stop codon between <EM>phaP</EM> and <EM>hlyA</EM>, the biobrick of HlyA-depending PHB secretion from the previous iGEM team needs to be engineered.</P><H3><STRONG>Aims</STRONG></H3><P style="text-align: justify;">We aim to investigate the effect of phasin and <EM>PhaR</EM> autoregulation system on PHB production and cell growth. In addition, we aim to investigate the influence of co-expressing <EM>phaP</EM>, <EM>phaR</EM> and <EM>hlyA</EM> on the PHB production. Moreover, the stop codon between <EM>phaP</EM> and <EM>hlyA</EM> was removed by PCR amplification.</P><H3 style="text-align: justify;"><STRONG>Materials and Methods</STRONG></H3><P style="text-align: justify;">Two biobricks BBa_K390501 and BBa_K 1149051 were used to establish new constructs including pSB3T5-<EM>ProR</EM>-<EM>phaR</EM>-<EM>ProP</EM>-<EM>phaP</EM>, pSB3T5-<EM>ProR</EM>-<EM>phaR</EM>-<EM>ProP</EM>-<EM>phaP</EM>-<EM>hlyA</EM>, pSB3T5-<EM>ProR</EM>-<EM>phaR</EM> and pSB3T5-<EM>ProP</EM>-<EM>phaP</EM>. New constructs of <EM>phaCAB</EM> operon were introduce into the <EM>E. coli</EM> BL21 (DE3) strain respectively. Optical density was measured to investigate the effect of pSB3T5-<EM>ProR</EM>-<EM>phaR</EM>-<EM>ProP</EM>-<EM>phaP</EM> and pSB3T5-<EM>ProR</EM>-<EM>phaR</EM>-<EM>ProP</EM>-<EM>phaP</EM>-<EM>hlyA</EM> on cell growth. PHB production was confirmed by Nile red staining followed by measurement in a plate reader. The total production of PHB consists of intracellular PHB and secreted PHB, which was measured and compared after the PHB extraction. </P><H3><STRONG>Results and Discussion</STRONG></H3><P style="text-align: justify;">The stop codon between <EM>phaP</EM> and <EM>hlyA</EM> was removed forming a part for new Biobrick that could be submitted to the iGEM registry. The results obtained from cell viability assay demonstrated that the expression of the <EM>phaCAB</EM> operon was likely to pose extra pressure on cells, while the regulation of <EM>PhaR</EM> and phasin can relieve the burden to some extent. The PHB produced by <EM>E. coli</EM> that harboured pSB3T5-<EM>ProR</EM>-<EM>phaR</EM>-<EM>ProP</EM>-<EM>phaP</EM>-<EM>hlyA</EM> is the highest. The comparison of PHB yield listed below, indicating that the presence of the <EM>PhaR</EM> regulator effectively regulate the expression of phasin and moderated the biosynthesis of PHB.</P><P style="text-align: center;"> <STRONG>Table 1 </STRONG>The yield of PHB production</P><TABLE style="margin-left: auto; margin-right: auto; width: 820px;"><TBODY><TR><TD style="text-align: center; width: 156px;"><P style="text-align: center;"><STRONG>Plasmid</STRONG></P></TD><TD style="text-align: center; width: 161px;"><P style="text-align: center;"><STRONG>Culture volume (ml)</STRONG></P></TD><TD style="text-align: center; width: 120px;"><P style="text-align: center;"><STRONG>Intracellular PHB (g)</STRONG></P></TD><TD style="text-align: center; width: 129px;"><P style="text-align: center;"><STRONG>Secreted </STRONG><STRONG>PHB (g)</STRONG></P></TD><TD style="text-align: center; width: 87px;"><P style="text-align: center;"><STRONG>Total </STRONG><STRONG>PHB </STRONG><STRONG>(g)</STRONG></P></TD><TD style="text-align: center; width: 127px;"><P style="text-align: center;"><STRONG>Yield of PHB (mg/ml)</STRONG></P></TD></TR><TR><TD style="text-align: center; width: 156px;"><P><STRONG><EM>phaCAB</EM>+pSB3T5</STRONG></P></TD><TD style="text-align: center; width: 161px;"><P style="text-align: center;">150ml</P></TD><TD style="text-align: center; width: 120px;"><P style="text-align: center;">0.01</P></TD><TD style="text-align: center; width: 129px;"><P style="text-align: center;">0</P></TD><TD style="text-align: center; width: 87px;"><P style="text-align: center;">0.01</P></TD><TD style="text-align: center; width: 127px;"><P style="text-align: center;">0.0667</P></TD></TR><TR><TD style="text-align: center; width: 156px;"><P><STRONG><EM>phaCAB</EM>+pSB3T5-R-P</STRONG></P></TD><TD style="text-align: center; width: 161px;"><P style="text-align: center;">150ml</P></TD><TD style="text-align: center; width: 120px;"><P style="text-align: center;">0.071</P></TD><TD style="text-align: center; width: 129px;"><P style="text-align: center;">0.0015</P></TD><TD style="text-align: center; width: 87px;"><P style="text-align: center;">0.0725</P></TD><TD style="text-align: center; width: 127px;"><P style="text-align: center;">0.4833</P></TD></TR><TR><TD style="text-align: center; width: 156px;"><P><STRONG><EM>phaCAB</EM>+pSB3T5-R-P-<EM>hlyA</EM></STRONG></P></TD><TD style="text-align: center; width: 161px;"><P style="text-align: center;">150ml</P></TD><TD style="text-align: center; width: 120px;"><P style="text-align: center;">0.043</P></TD><TD style="text-align: center; width: 129px;"><P style="text-align: center;">0.0305</P></TD><TD style="text-align: center; width: 87px;"><P style="text-align: center;">0.0735</P></TD><TD style="text-align: center; width: 127px;"><P style="text-align: center;">0.4900</P></TD></TR><TR><TD style="text-align: center; width: 156px;"><P><STRONG><EM>phaCAB</EM>+pSB3T5-R</STRONG></P></TD><TD style="text-align: center; width: 161px;"><P style="text-align: center;">150ml</P></TD><TD style="text-align: center; width: 120px;"><P style="text-align: center;">0.071</P></TD><TD style="text-align: center; width: 129px;"><P style="text-align: center;">0.002</P></TD><TD style="text-align: center; width: 87px;"><P style="text-align: center;">0.073</P></TD><TD style="text-align: center; width: 127px;"><P style="text-align: center;">0.4868</P></TD></TR><TR><TD style="text-align: center; width: 156px;"><P><STRONG><EM>phaCAB</EM>+pSB3T5-P</STRONG></P></TD><TD style="text-align: center; width: 161px;"><P style="text-align: center;">150ml</P></TD><TD style="text-align: center; width: 120px;"><P style="text-align: center;">0.071</P></TD><TD style="text-align: center; width: 129px;"><P style="text-align: center;">0</P></TD><TD style="text-align: center; width: 87px;"><P style="text-align: center;">0.071</P></TD><TD style="text-align: center; width: 127px;"><P style="text-align: center;">0.4733</P></TD></TR></TBODY></TABLE><P>Note: <EM>E. coli</EM> that harboured different constructs were cultured in M9 medium with 3% glucose for 48 hours at 37°C shaker</P><P> </P><H3><STRONG>References</STRONG></H3><UL><LI>York, G.M., Stubbe, J. and Sinskey, A.J., 2001. New insight into the role of the <EM>PhaP</EM> phasin of <EM>Ralstonia eutropha</EM> in promoting synthesis of polyhydroxybutyrate. <EM>Journal of Bacteriology</EM>, 183(7), pp.2394-2397.</LI><LI>Pötter, M., Madkour, M.H., Mayer, F. and Steinbüchel, A., 2002. Regulation of phasin expression and polyhydroxyalkanoate (PHA) granule formation in <EM>Ralstonia eutropha</EM> H16. <EM>Microbiology</EM>, 148(8), pp.2413-2426.</LI><LI>Mergulhao, F.J.M., Summers, D.K. and Monteiro, G.A., 2005. Recombinant protein secretion in <EM>Escherichia coli</EM>. <EM>Biotechnology advances</EM>, 23(3), pp.177- 202.</LI></UL></DIV></DIV><DIV><P> </P></DIV><H4 style=" font-size:150% ">Construction of plasmids harbouring the Sleeping Beauty Mutase operon, and methylmalonyl CoA epimerase for producing propionate for PHBV production</H4><P style="text-align: justify;">Methylmalonyl-CoA Epimerase (MCE) was introduced in the proposed pathway (as shown in Figure 1) resulting into high production of propionate. By expressing the Sleeping Beauty Mutase (SBM) operon and MCE with genes for PHA production in <EM>E. coli</EM> resulted in the production of PHBV with a high ratio of valerate to butyrate. In this study, vectors harbouring the SBM operon with and without MCE was achieved.</P><DIV class="container"><BUTTON class="btn btn-info" type="button" data-toggle="collapse" data-target="#tittle5">Read more</BUTTON><DIV id="tittle5" class="collapse"><H3><STRONG>Background</STRONG></H3><P style="text-align: justify;">There are many bio-based alternatives to the widely used petrochemical-based plastics, which are degradable and therefore are less damaging to the environment, and to health. One such ‘bio-plastic’ is polyhydroxyalkanoates (PHAs), which is a large family with a widerange of properties. However, only one type of microbe produced PHA is economically viable – poly(3- hydroxybutyrate) (PHB) – but too brittle for widespread use. PHA co-polymers, such as poly(3-hydroxybutyrate-co-3-hydroxyvalerate) (PHBV) are more flexible, and could fill niches that PHB does not. However, producing PHBV by microbes typically needs propionate to be supplied exogenously (Babu, et al., 2013). This is unsustainable as the propionate is normally sourced from unnatural sources. <EM>E. coli</EM> encodes many of the genes needed to produce propionate from glucose, in the Sleeping Beauty Mutase (SBM) operon, which encodes an incomplete pathway for propionate production (Kannan, 2008)</P><P style="text-align: center;"><STRONG>Figure 1 </STRONG>Proposed mechanism for propionate synthesis utilising the Sleeping beauty mutase operon (SBM) and Methylmalonyl-CoA epimerase (MCE) - Succinyl-CoA is converted into Methylmalonyl-CoA-R by the methylmalonyl- CoA mutase ScpA. Methylmalonyl-CoA-R is converted into Methylmalonyl-CoA-S by MCE or an uncharacterised, native pathway. Methylmalonyl-CoA-S is converted into propionyl-CoA by the methylmalonyl-CoA carboxylase ScpB. The CoA from Propionyl-CoA is transferred onto Succinate from the citric acid cycle by the Propionyl-CoA: Succinate CoA transferase ScpC, resulting in the production of propionate and Succinyl-CoA.</P><P> </P><H3><STRONG>Aim</STRONG></H3><P style="text-align: justify;">Our aim is to construct plasmids harbouring the SBM operon and MCE. These constructs were to be assayed for propionate production, followed by co-expression with the PHA operon and determination of the composition of the PHA. </P><H3><STRONG>Materials and Methods</STRONG></H3><P style="text-align: justify;">We designed and ordered DNA fragments from Integrated DNA Technologies, Inc. (IDT) to obtain Sleeping Beauty Mutase (SBM) which consist genes <EM>ScpA</EM>, <EM>ScpB</EM>, <EM>ScpC</EM>, and <EM>argK. </EM>Constructs of pSB3T5, pSB3T5:MCE, pSB3T5:SBM, and pSB3T5:MCE:SBM were established and grown in LB medium. Produced propionate was measured by detecting the change in absorbance at 410nm using spectrophotometer.<STRONG> </STRONG></P><H3><STRONG>Results and Discussion</STRONG></H3><P style="text-align: center;"><STRONG>Figure 2 </STRONG>Analytical digest of pSB3T5: SBM using <EM>Xba</EM>I and <EM>Spe</EM>I, followed by gel electrophoresis – Uncut sample showed 2 bands, one above 10kbp, and one between 8kbp – 6kbp. XbaI and SpeI single digests show a single band of 8790bp. XbaI and SpeI double digest shows two bands: a pSB3T5 backbone at 3250bp, and the SBM operon at 5540bp</P><P style="text-align: center;"><STRONG>Figure 3 </STRONG>Colony PCR of 7 colonies of pSB3T5: MCE: SBM – Samples 5, 6, and 7 all have bands between 6kbp – 8kbp, suggesting that they do possess PSB3T5: MCE: Sbm, but would have to be verified by sequencing. </P><H3><STRONG>Future Work</STRONG></H3><P style="text-align: justify;">In this work, constructs containing the SBM operon and MCE were developed. The constructs produced in this work need to be verified by sequencing, in order to determine whether there are any mutations that may impact their function.</P><P><STRONG> </STRONG></P><H3><STRONG>References</STRONG></H3><UL><LI>Babu, R., O'Connor, K. &amp; Seeram, R., 2013. Current progress on bio-based polymers and their future trends. <EM>Progress in Biomaterials,</EM>2(1), p.8.</LI><LI>Kannan, S., 2008. Studies on Methylmalonyl-CoA Mutase from <EM>Escherichia coli</EM>. Doctoral Dissertation. <EM>University of Westminster. </EM></LI></UL></DIV></DIV><DIV><P> </P></DIV><H4 style=" font-size:150% ">Investigation of the effects of <EM>sucAB</EM> and <EM>sucCD</EM> on the adaption to propionic Acid and PHBV production<STRONG> </STRONG></H4><P style="text-align: justify;">In this project, various genes including gene <EM>sucAB</EM> and <EM>sucCD</EM> from <EM>Escherichia coli</EM> were introduced to <EM>E. coli</EM> and co-expressed with <EM>phaCAB</EM> operon from <EM>Rastonis eutropha</EM>. The engineered strains are used to investigate the effects of these genes on PHBV production and strain growth, therein, different effects caused by <EM>sucCD</EM> and <EM>sucAB</EM> respectively are compared, as well as the PHBV yield inside.</P><DIV class="container"><BUTTON class="btn btn-info" type="button" data-toggle="collapse" data-target="#tittle6">Read more</BUTTON><DIV id="tittle6" class="collapse"><H3><STRONG>Background</STRONG></H3><P style="text-align: justify;">As direct precursors, the amount of propionyl-CoA in the <EM>E. coli</EM> determines the accumulation of 3HV branches in the copolymer (Bhatia et al., 2015) and influences the PHBV yield. Thus, enhancement of propionyl-CoA pool is emphasized. The <EM>sucCD</EM> and <EM>sucAB</EM> are two genes participating the tricarboxylic acid (TCA) cycle, the former encodes succinyl-coA synthase catalyzing interconversion between succinate and succinyl-CoA, the latter encodes α-ketoglutarate dehydrogenase responsible for the formation of succinyl-CoA from α-ketoglutarate. According to Yu et al. (2006), either <EM>sucAB</EM> or <EM>sucCD</EM> is viable to produce enough succinyl-CoA, both are essential for cell viability. The interconversions of succinyl-CoA and propionyl-CoA are catalyzed by cluster of enzymes. Therein, playing important role in the conversion from succinyl-CoA and propionyl-CoA, both the proteins from <EM>sucAB</EM> and <EM>sucCD</EM> cooperate with other downstream enzymes, contribute to utilization of propionate, eventually to the PHBV production. </P><H3><STRONG>Aims</STRONG></H3><P style="text-align: justify;">In order to optimize yields and quality of PHBV which will help support the industrialization of PHBV production, we aim to obtain better PHBV production by enhancing the production of precursors via introducing exogenous genes <EM>SucAB</EM> and <EM>sucCD</EM>.</P><H3><STRONG>Materials and Methods</STRONG></H3><P style="text-align: justify;"><EM>sucAB</EM> and <EM>sucCD </EM>were amplified from the genome to build new constructs including pSB3T5-<EM>sucAB</EM>, pSB3T5-<EM>sucCD</EM>, plasmid pSB3T5-X. These new constructs were introduced to <EM>E. coli</EM> BL21 (DE3) and cultured in M9 medium with 1% glucose, 0.01M propionic acid and 10 mM IPTG. Optical density was measured to give information for cell growth. Germinate multiple (GM), or the final cell concentration/inoculation cell concentration, was determined which represented proliferation capacity of cells. The utilization of propionic acid was determined by standard curve and equation between the absorbance and propionic acid concentration.</P><H3><STRONG>Results and Discussion</STRONG></H3><P style="text-align: justify;">The presence of <EM>sucCD</EM> enables strains to adapt to environments with propionate while enhancing the propionate utilization ability, thereby resulting in better PHBV production compared to the existing <EM>sucAB</EM> gene. The amount of propionic acid taken up by thecell reflects the propionate utilization capacity of each strains to some extent, can be said that all three strains present similar property in terms of propionic acid absorption, when the concentration is 0.03 M, every strain reach a stationary phase.</P><P style="text-align: center;"><STRONG>Figure 11 </STRONG>Propionic acid absorbed by three recombinant <EM>E. coli </EM>of pSB3T5-AB+<EM> phaCAB</EM>, pSB3T5-CD+<EM>phaCAB</EM> and pSB3T5+<EM>phaCAB</EM> operon</P><H3><STRONG>Future Works</STRONG></H3><P style="text-align: justify;">This investigation will be meaningful for practical issues like environment protection as well as industrial applications, for example, pot ale, a by-product from whisky distillery is rich in propionate. Currently, selling to local farmers and applying as fertilizer is the main by-products treatment method, PHBV production via recombinant strains fed with pot ale potentially can be a more environmentally-friendly and cost-effective alternative.</P><H3><STRONG>References</STRONG></H3><LI>Bhatia, S.K., Yi, D.H., Kim, H.J., Jeon, J.M., Kim, Y.H., Sathiyanarayanan, G., Seo, H.M., Lee, J.H., Kim, J.H., Park, K. and Brigham, C.J., 2015. Overexpression of succinyl‐CoA synthase for poly (3‐hydroxybutyrate‐co‐3‐hydroxyvalerate) production in engineered <EM>Escherichia coli</EM> BL 21 (DE 3). <EM>Journal of applied microbiology</EM>, 119(3), pp.724-735.</LI><LI>Yu, B.J., Sung, B.H., Lee, J.Y., Son, S.H., Kim, M.S. and Kim, S.C., 2006. <EM>sucAB</EM> and <EM>sucCD</EM> are mutually essential genes in <EM>Escherichia coli</EM>. <EM>FEMS microbiology letters</EM>, 254(2), pp.245-250.</LI></DIV></DIV></DIV></DIV></BODY></HTML>