<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toronto_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toronto/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navigation"><UL><LI><A href="https://2015.igem.org/Team:Toronto"><SPAN>home</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Team"><SPAN>team</SPAN></A></LI><LI class="active"><A href="#"><SPAN>project</SPAN></A></LI><LI class="active"><A href="https://2015.igem.org/Team:Toronto/Description"><SPAN>description</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Experiments"><SPAN>experiments</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Results"><SPAN>results</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Design"><SPAN>design</SPAN></A></LI></UL><LI><A href="https://2015.igem.org/Team:Toronto/Parts"><SPAN>parts</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Notebook"><SPAN>notebook</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Attributions"><SPAN>attributions</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Collaborations"><SPAN>collaborations</SPAN></A></LI><LI><A href="#"><SPAN>practices</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Practices"><SPAN>human practices</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Alberta-Oil-Stories"><SPAN>alberta oil stories</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Community-Outreach"><SPAN>community outreach</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Technology-Applications"><SPAN>technology applications</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/TechnoMoral-Scenarios"><SPAN>technoMoral scenarios</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Safety"><SPAN>safety</SPAN></A></LI><LI><A href="#"><SPAN>software</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Software"><SPAN>software</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Demo"><SPAN>demo</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Generator"><SPAN>wiki generator</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Entrepreneurship"><SPAN>entrepreneurship</SPAN></A></LI></DIV><DIV class="content" id="content-main"><DIV class="row"><DIV class="col col-lg-8 col-md-12"><DIV class="content-main"><H1 id="consortiaflux">ConsortiaFlux</H1><H2 id="introduction">Introduction</H2><P>In nature, it is virtually impossible to find an isolated bacterial species on its own. Across most of the natural world, different bacterial species cluster together in ecologically crucial communities known as microbiomes. In constrast, synthetic biology laboratories often prepare isolated strains for later use in the natural environment. Experimentally, this would require either the creation or simulation of an isolated microbiome.</P><P>Our iGEM team created the web-based tool 'ConsortiaFlux' in order to run microbiome simulations. Our tool allows the user to visualize the effects of the addition of a synthetically engineered bacteria into a predefined microbiome. This visualization is generated through a process called community flux balance analysis (cFBA).</P><P>In concert with our web-based tool, we genetically engineered E. coli bacteria for the purpose of bioremediation. Our engineered E. coli supplements the existing microbiome by efficiently degrading toluene, a highly toxic chemical within Alberta's oil sands tailing ponds.</P><P>All aspects of our project are linked together by analysis of technical, economic, and social applications of our technology within a solid policy framework.</P><H2 id="what-is-fba-and-cfba-">What is FBA and cFBA?</H2><P>FBA (flux balance analysis) is a mathematical tool used to study genome-scale metabolic network reconstructions. By analyzing the flow of metabolites within a given network, FBA can be used to model the expected rates of production for metabolites, metabolic behaviour under optimal growth, and various other aspects of life processes. In a synthetic biology context, FBA is a useful means of assessing the outcomes of in silico transformations.</P><P>cFBA (community based flux-balance analysis) consists of FBA at the community level. Highly interdependent microbial communities share essential metabolites, engage in complex metabolic exchanges, and collectively perform vital ecological functions. Using cFBA, the consequences of genetically modifying a particular species within the microbiome can be predicted.</P><H2 id="math-behind-the-code">Math Behind the Code</H2><P>FBA computes the flow of metabolites at steady state in which mass balance is not changing with time (i.e. (dx/dt)=0). In an FBA with <EM>m</EM> unique compounds and <EM>n</EM> reactions, a stoichiometric matrix(S) with size <EM>m</EM> * <EM>n</EM> represents all the reaction set. Vector fluxes to be computed are represented by a vector v. An objective function (Z) is set to determine which flux to optimize in a given boundary. Hence following is computed linearly,</P><DIV>
     $$ \begin{array}{l l}
        \textrm{Maximize} &amp; c^Tv \\
        \textrm{Subject to} &amp; Sv=0 \\
        \textrm{boundaries} &amp; \textrm{lowerbound} &lt; v &lt; \textrm{upperbound}
     \end{array} $$
</DIV><P class="image-wrapper"><A href="/File:Toronto_2015_FBA_diagram.jpg" title="File:Toronto 2015 FBA diagram.jpg">File:Toronto 2015 FBA diagram.jpg</A></P><P>Taken from:</P><P>Jeffrey D Orth, Ines Thiele, Bernhard Ø Palsson. What is flux balance analysis?
Nature Biotechnology. Nature Publishing Group. Mar 1 2010. Copyright © 2010,
Rights Managed by Nature Publishing Group</P><P>Note: See software section for more</P><H2 id="proof-of-concept-a-genetically-engineered-solution-for-oil-sand-tailings">Proof-of-Concept A Genetically Engineered Solution for Oil Sand Tailings</H2><H3 id="background">Background</H3><P>As conventional oil production declines in the coming decades, unconventional sources of oil such as bitumen will become increasingly important for supplying oil. Currently, Canada possesses the largest of these bitumen reserves in Alberta in the form of oil sands. Alberta’s oil sands have about 168 billion barrels, making it the third largest crude oil reserve in the world. The use and export of this oil significantly contributes to economic growth in Canada, and will continue to do so.</P><H3 id="economic-impact-of-oil-sands-on-canada">Economic impact of oil sands on Canada</H3><UL><LI>Total GDP impacts of all oil sands investment, re-investment and operating revenues is estimated to be $3,865 billion for Canada.</LI><LI>Oil sands related direct employment in Alberta is expected to continue growing from the current level (2014) of 146,000 jobs to a peak of 256,000 jobs in 2024.</LI><LI>For every direct job generated in the Alberta oil sands, 1 additional job is generated by indirect association and 1.5 jobs by induced association in Canada.</LI></UL><P>Despite this huge economic potential, the use of oil sands is raising concerns regarding environmental costs, especially those associated with the creation of oil sands tailings. Since the oil sand tailings are essentially a dumping site, it has a high concentration of toxic organic compounds. The Government of Alberta has a zero discharge policy for open-mined oil operations. This means that all oil sand process-affected water (OSPW) and tailings must be stored on site, which leads to the accumulation of these toxic compounds . According to a report released by WWF, this is a growing problem. From 2006 to 2009, weight percentage of toxic benzene, toluene, ethylbenzene, xylene (BTEX) and Polycyclic Aromatic Hydrocarbons (PAH) present in oil tailings have increased by 29.7% and 15.5% respectively.   Although, the oil sand tailing ponds is supposed to retain the toxic water, a recent Federal study suggests that the waste might be leaching into the ground water and contaminating the Athabasca River. It was estimated that as much as 6.5 litres might be leaching from a single pond per day. Current BTEX concentrations have already exceeded the guidelines provided by Canadian Council for Ministers of Environment (CCME) and thus are toxic to wildlife and humans in the long term.</P><P>BTEX can cause damage to:</P><UL><LI>Liver</LI><LI>Kidneys</LI><LI>Eyes</LI><LI>Central Nervous System</LI></UL><H3 id="current-solutions">Current Solutions</H3><P>The current solutions for BTEX and PAH contamination includes:</P><UL><LI><P><EM>Soil Vapor Extraction (SVE)</EM></P><P>SVE is a physical process that involves sucking up vapors of contaminants. This reduces the vapor pressure of the contaminant hence shifting the equilibrium towards the vapor phase and causing any contaminant in the solid or liquid phase to evaporate. Since the effectiveness of SVE depends strongly on:   </P></LI><LI>The contaminant’s physical and chemical properties</LI><LI>Temperature in the subsurface</LI><LI>Soil properties</LI></UL><P>So SVE is not suitable for all compounds and under all conditions.</P><LI><P><EM>Bioremediation using Microorganisms</EM></P><P>This technique uses microbes in order to break down BTEX compounds and PAHs using enzymes produced by local microbes. This technique is advantageous because it is cheap, occurs on site, and results in the production of non-toxic compounds such as carbon dioxide and acetate. However, the bioremediation process can be hindered as a result of any of the factors listed below:</P></LI><LI>The high toxicity of the pollutant</LI><LI>The high concentration of the pollutant</LI><LI>Low solubility of the pollutant</LI><LI>Low availability of nutrients in the soil</LI><LI>Low concentration of dissolved Oxygen</LI><LI>Fewer electron acceptors.</LI><P>The factors listed essentially limit the bioremediation process.</P><H3 id="our-project">Our Project</H3><P>Our project aims to maximize the efficiency of the bioremediation process by creating a synthetically engineered microorganism that can degrade toxic compounds found in oil sands tailing ponds. Our team focused specifically on toluene due to the prevalence of BTEX compounds (uncluding toluene) in oil sands tailing ponds, as well as fact that toluene is well-studied as a chemical.</P><H3 id="toluene-degradation-in-microbial-communities">Toluene degradation in microbial communities</H3><P>Many microorganisms that survive in Alberta's oil sand tailings use toluene as a nutrient source. As such, these microorganisms have naturally evolved several pathways that can degrade toluene. These degradation pathways can be broadly divided into two categories:</P><UL><LI>Aerobic pathways</LI><LI>Anaerobic pathways</LI></UL><P>We focused specifically on the pathway used by <EM>Pseudomonas putida F1</EM> because it is a well-studied, Biosafety Level 1 organism, and some studies suggest that it is one of the most efficient bacterial species with regards to toluene degradation. <EM>P. putida F1</EM> degrades toluene through an aerobic pathway. In its pathway the reactions involving <EM>todC1C2BA</EM>, <EM>todD</EM>, <EM>todE</EM> and <EM>todF</EM> are specific to <EM>P. putida F1</EM> and are the limiting steps of this pathway. In particular, the accumulation of 3-methylcatechol is known to have severe toxic effects for bacteria. Once 2-Hydroxy-2,4-pentadienoate is formed, it can be degraded by enzymes that can are present in most species, including <EM>E.coli</EM>. A plasmis for the first part of the pathway involving <EM>todC1C2BA</EM> and <EM>todD</EM> (pTDG602) has been created by Zylstra, from whom we obtained it. Finally, we chose to focus on the part of the pathway involving <EM>todE</EM> and <EM>todF</EM>, for which no well-characterized BioBrick exists.</P><H2 id="experimental-design">Experimental Design</H2><H3 id="our-objectives">Our Objectives</H3><UL><LI>To design and construct a plasmid that contains both <EM>todE</EM> and <EM>todF</EM>.</LI><LI>To transform <EM>E.coli</EM> with our created plasmid.</LI><LI>To confirm the synthesis of <EM>todE</EM> and <EM>todF</EM>.</LI><LI>To characterize the activity of <EM>todE</EM> and <EM>todF</EM> (if time permits).</LI><LI>To co-transform <EM>E.coli</EM> with both pTDG602 and our assembled plasmid (if time permits).</LI></UL><H3 id="plasmid-design">Plasmid Design</H3><P>The diagram above shows the plasmid that we designed. We used psB1C3, in green, for the backbone that we obtained from the iGEM registry. The plasmid includes both <EM>todE</EM> and <EM>todF</EM> that have been optimized for <EM>E.coli</EM> K-12 MG1655 strain. Each gene begins with a start codon followed by a His-Tag, and terminates with a stop codon. The purpose of the His-Tag is to confirm the synthesis of our enzymes once we transform <EM>E.coli</EM>. We added a Ribosomal binding site (RBS) before each gene and optimized it using the Salis RBS calculator. UNSs were also added before and after every gene to prevent the formation of a secondary structure.  </P><H3 id="experimental-details">Experimental details</H3><P>To create our plasmid, we used Synbiota’s Rapid Dna Prototyping (RDP) protocol in order to assemble our <EM>todE</EM> and <EM>todF</EM> constructs. We confirmed the identity of our constructs by using gel electrophoresis to compare our constructs against the <EM>todE</EM> and <EM>todF</EM> genes we purified from <EM>P. putida F1</EM>. Once the <EM>todE</EM> and <EM>todF</EM> genes were assembled, they were inserted into the psB1C3 backbone using the Ligase Chain Reaction (LCR) method. Following LCR, gel electrophoresis was once again used to ensure that our plasmid had been assembled correctly. Once our plasmid was assembled, we transformed <EM>E.coli</EM>, which we made chemically competent for transformation. Following the transformation, antibiotic selection was used to identify the transformed colonies. Further, to test for synthesis of the desired enzymes, the transformed colonies were lysed and tested via SDS-PAGE. After confirming enzyme production, the transformed <EM>E.coli</EM> was grown on media rich in 3-methylcatechol, and a colorimetric assay was used to confirm the formation of 2-hydroxy-6-keto-2,4-heptadienoate (substrate of <EM>todE</EM>) and 2-Hydroxy-2,4-pentadienoate (substrate of <EM>todF</EM>). As an aside, gas chromatography and mass spectrometry methods are also applicable to characterizing the activity of these enzymes, where time permits. Finally, we attempted to co-transform our bacteria with our custom-designed plasmid and pTDG602 in order to observe the complete degradation of toluene into carbon dioxide and water.</P></DIV></DIV><DIV id="tableofcontents" class="tableofcontents affix sidebar col-lg-4 hidden-xs hidden-sm hidden-md visible-lg-3"><UL class="nav"><LI><A href="#introduction">Introduction</A></LI><LI><A href="#what-is-fba-and-cfba-">What is FBA and cFBA?</A></LI><LI><A href="#math-behind-the-code">Math Behind the Code</A></LI><LI><A href="#proof-of-concept-a-genetically-engineered-solution-for-oil-sand-tailings">Proof-of-Concept A Genetically Engineered Solution for Oil Sand Tailings</A></LI><LI><A href="#background">Background</A></LI><LI><A href="#economic-impact-of-oil-sands-on-canada">Economic impact of oil sands on Canada</A></LI><LI><A href="#current-solutions">Current Solutions</A></LI><LI><A href="#our-project">Our Project</A></LI><LI><A href="#toluene-degradation-in-microbial-communities">Toluene degradation in microbial communities</A></LI></UL><LI><A href="#experimental-design">Experimental Design</A></LI><LI><A href="#our-objectives">Our Objectives</A></LI><LI><A href="#plasmid-design">Plasmid Design</A></LI><LI><A href="#experimental-details">Experimental details</A></LI></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>