<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toronto_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toronto/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navigation"><UL><LI><A href="https://2015.igem.org/Team:Toronto"><SPAN>home</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Team"><SPAN>team</SPAN></A></LI><LI class="active"><A href="#"><SPAN>project</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Description"><SPAN>description</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Experiments"><SPAN>experiments</SPAN></A></LI><LI class="active"><A href="https://2015.igem.org/Team:Toronto/Results"><SPAN>results</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Design"><SPAN>design</SPAN></A></LI></UL><LI><A href="https://2015.igem.org/Team:Toronto/Parts"><SPAN>parts</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Notebook"><SPAN>notebook</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Attributions"><SPAN>attributions</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Collaborations"><SPAN>collaborations</SPAN></A></LI><LI><A href="#"><SPAN>practices</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Practices"><SPAN>human practices</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Alberta-Oil-Stories"><SPAN>alberta oil stories</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Community-Outreach"><SPAN>community outreach</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Technology-Applications"><SPAN>technology applications</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/TechnoMoral-Scenarios"><SPAN>technoMoral scenarios</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Safety"><SPAN>safety</SPAN></A></LI><LI><A href="#"><SPAN>software</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Software"><SPAN>software</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Demo"><SPAN>demo</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Generator"><SPAN>wiki generator</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Toronto/Entrepreneurship"><SPAN>entrepreneurship</SPAN></A></LI></DIV><DIV class="content" id="content-main"><DIV class="row"><DIV class="col col-lg-8 col-md-12"><DIV class="content-main"><H1 id="project-results">Project Results</H1><H2 id="computational-biology-results">Computational Biology Results</H2><P>Here we present ConsortiaFlux, a framework for investigating and simulating the metabolism of microbial consortiums. ConsortiaFlux was created to explore communities of genome-scale metabolic models. We have created a tool that can provide insights in the metabolism of communities with an intuitive user interface and a novel community level flux balance analysis pipeline.</P><P>ConsortiaFlux automates the rendering of community metabolic networks, created by using HTML5 Canvas, where metabolites, reactions and species are represented as circles and metabolite-reaction relationships are represented as lines. Rendered networks dynamically move to a forced-layout by using the JavaScript library d3.js. The user interface allows the user to intuitively create community metabolic networks from a set of 32 bacterial species and subsequently add/remove metabolites and reactions; the user interface is created with Angular.js. All 32 bacterial metabolic models are genome-scale reconstructions published in literature. Parts from the iGEM registry can be added as reactions to the network. Upon initialization, metabolite networks display species nodes, external metabolites and external reactions where species nodes are connected to reactions that import/export metabolites in/out of the cell. Internal metabolites and reactions can be accessed by clicking the enter button which hovers over species nodes. The metabolism of the community can be simulated by pressing the optimize button.</P><P>Our tool simulates the community’s metabolism by using a community flux balance analysis pipeline we created using Python. We used COnstraint-Based Reconstruction and Analysis (COBRApy) to optimize our models in Python. Our pipeline iteratively optimizes for individual genome-scale metabolic models for each species and computes/applies new constraints that exist in the community model. Once the constraints are calculated through iterations of linear programming and statistical procedures, a community model is built from the collection of species models where species are simply treated as additional compartments. The community model is then optimized with the community objective function. The community objective function is the summation of all the species’ objective functions where the shadow prices of metabolites are multiplied by a biomass coefficient determined from the iterations of linear programming and statistical procedures. Unlike traditional community flux balance analysis, our pipeline uses multiple iterations of linear programming instead of a bi-linear program (Khandelwal et al). The advantage of using our pipeline is that it can accommodate any number of species and that it uses linear systems.</P><P>We used ConsortiaFlux to simulate metabolism of the microbiome of oil sand tailings with the addition of our synthetic Escherichia coli that has the TodABCDEF genes which degrades toluene. With our community flux balance analysis pipeline, we saw an increase in the community biomass objective function compared to a community with E.coli that does not degrade toluene. [<STRONG><EM>Then we just add in reactions that we saw an increase of flux</EM></STRONG>].</P><P>This software can be applied in the future to many aspects of bacterial community studies, including numerous applications in human health and medicine. Our system of community flux balance analysis (cFBA) used the specific example of optimiziation for maximal toluene degradation within the microbial system of Athabasca oil sands tailings pond water, in order to best detoxify this water to be potable to inhabitants living downstream. However, the future uses for this tool could be far-reaching in nature.</P><P>This system has already been used for a wide range of projects including pinpointing pathogenicity islands in strains of bacteria which can be harmful to human populations (Raman and Chandra, 2009), assessing how rice plants can be optimized to best respond to flooding (Lakshmanan et al. 2013), and achieving spatiotemporal amalgamation of overall metabolism in other crop plants (Grafahrend-Belau et al. 2013). Even more significant achievements could prevail with this technology, such as further simple analysis of gut microbiomes in patients with gastrointestinal illness. With the future of cFBA and metabolic mapping, biological and medical research will no doubt leap forward with innovation.</P><H2 id="wetlab-results">Wetlab Results</H2><H3 id="wetlab-goals-">Wetlab Goals:</H3><P>Our main goal in this project was to create standardized and easily customizable synthetic parts coding for the enzymes in the tod pathway, a toluene degrading pathway native to pseudomonas putida F1, in order to improve the efficiency of the pathway. We focused on the tod E (3-methyl catechol 2,3-dioxygenase) and tod F (HODA hydrolase) genes which appear to be important bottlenecks in the toluene degradation pathway. We are using E.Coli MG1655 as a model chassis since it contains the enzymes downstream of TodF in the pathway. As such, plasmid design had to be optimized for E.coli. This would be followed by the transformation of E.coli with the plasmid, the confirmation of protein formation by SDS PAGE and western blot (via His Tags) and the subsequent characterization of enzymatic activity. The characterization of TodE and TodF would be acheived via a spectrophotometric assay that would measure the change in the wavelength of 6-methyl HODA at 367nm. TodE would produce 6-methyl HODA and TodF would show consumption of the metabolite. This would be used to create a Michaeles-Mentens graph by transforming absorbance readings into concentrations by Beer's law. We subsequently also aimed to incorporate the complete toluene degradation pathway in E.coli to observe the complete degradation of toluene to carbon-dioxide and water. This would be achieved by co-transforming our plasmid with TodEF and TodABC1C2D (obtained from Dr. Zylstra).</P><H3 id="achieved-">Achieved:</H3><P>Following are listed the goals we are able to accomplish
<STRONG>Plasmid design</STRONG>:  </P><UL><LI>This was perhaps one of the most critical stages of the project since only a correctly designed plasmid would ultimately allow E.coli to successfully degrade toluene. For this purpose, we sought extensive guidance from our graduate and faculty supervisors who have had extensive experience in these areas. The plasmids were designed to be assembled by Gibson Assembly, Ligase Chain Reaction (LCR) and Synbiota Assembly. (A form of standardized digestion and ligation assembly)</LI></UL><UL><LI><P><STRONG>Plasmid creation</STRONG>:We were able to successfully create our plasmid by PCRing the psb1C3 backbone out of a plasmid containing BBa_J04450. Both TodE and TodF were made compatible for LCR by synthesizing UNS sequences (standard insulating ends) and by PCRing compatible ends for Gibson Assembly. The inserts were checked by using nanodrop readings and gel electrophoresis. At the same time, we needed to verify the correct assembly of our plasmid by digesting with specific enzyme and using gel electrophoresis to confirm the size. This was a bit challenging given that LCR was a new method of assembly and using gibson next to iGEM suffixes (multiple cloning sites) produced low efficiency. Standard protocols used in these experiments were not always completely effective so we had to modify our protocols to achieve the best possible results with respect to plasmid design. This did provide us with major insight regarding how standard protocols need to be modified in novel situations.</P></LI><LI><P><STRONG>E.coli transformation</STRONG>:
We have been able to transform E.coli with our plasmid and we have obtained colonies on antibiotic selection media which indicates that our plasmids have been transformed.</P></LI></UL><H3 id="challenges-faced-">Challenges faced:</H3><UL><LI><P><STRONG>Time Management</STRONG>:This is where we struggled the most. This year we had recruited a fresh team and each member had to be trained in biosafety. At the same time the department had several administrative procedures that took long to clear and hence we were not able to work in lab on time. This has allowed us to formulate a concrete strategy for next year, so we can make best use of our time in the lab.</P></LI><LI><P><STRONG>Division of labour</STRONG>:Due to the relative inexperience of many newer members in the lab, team leads had to take on the role of performing the tasks themselves. Higher efficiency was achieved by making a smaller, more select group to work on protocols throughout the week since large member base and low number of shifts per member led to discontinuity and confusion.</P></LI></UL><H3 id="references-">References:</H3><P>Ramen and Chandra. Flux balance analysis of biological systems: Applications and challenges. <EM>Brief Bioinform</EM>, 10:4, pg.435-449. 2009.Lakshmanan et al. Elucidating rice cell metabolism under flooding and drought stresses using flux-based modelling and analysis. <EM>Plant Physiology</EM>, 162:4, pg.2140-2150. 2013.Grafahrend-Belau et al. Mutliscale metabolic modeling: Dynamic flux balance analysis on a whole-plant scale. <EM>Plant Physiology</EM>, 163:2, pg.637-647. 2013.   </P></DIV></DIV><DIV id="tableofcontents" class="tableofcontents affix sidebar col-lg-4 hidden-xs hidden-sm hidden-md visible-lg-3"><UL class="nav"><LI><A href="#computational-biology-results">Computational Biology Results</A></LI><LI><A href="#wetlab-results">Wetlab Results</A></LI><LI><A href="#wetlab-goals-">Wetlab Goals:</A></LI><LI><A href="#achieved-">Achieved:</A></LI><LI><A href="#challenges-faced-">Challenges faced:</A></LI><LI><A href="#references-">References:</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>