<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Clemson_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Clemson/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Clemson/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Clemson/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Clemson/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Clemson/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Clemson/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>CU iGEM</TITLE></P><DIV class="container_12"><DIV id="sliderWrapper"><DIV id="carouselWrapper" class="grid_12"><UL id="carousel"><LI>Home</LI><LI>Team</LI><LI>Project</LI><LI>Notebook</LI><LI>Attributions</LI><LI>Safety</LI><LI>Ethics</LI><LI>Links</LI></UL></DIV><DIV id="weeklylog"><FONT size="7"><DIV align="center"><U><A name="don">Notebook</A></U></DIV></FONT><FONT size="5"><DIV id="tab" align="center"><TABLE><TBODY><TR><TD><A href="#week1">Primer Design</A><A href="#week2">Competent Cells</A><A href="#week3">DNA Purification</A><A href="#week4">Using Standard Parts</A></TD><TD><A href="#week5">Polymerase Chain Reaction (PCR)</A><A href="#week6">PCR Purification</A><A href="#week7">Joining DNA Fragments</A><A href="#week8">Transformation</A></TD><TD><A href="#week9">Testing Biphenyl Degradation Genes</A><A href="#week10">Testing Rhamnolipid Biosynthesis Genes</A><A href="#week11">BioBricks</A><A href="#week12">References</A></TD></TR></TBODY></TABLE></DIV></FONT></DIV><A name="week1"><U>Primers</U></A>Primers were designed using sequences obtained from the <A href="http://www.ncbi.nlm.nih.gov/"> National Center for Biotechnology Information</A>. To design the primers, we used <A href="http://www.clustal.org/ "> ClustalX </A> to align sequences and observe conserved regions of DNA.  
Organisms:<UL><LI> 	•	Pseudomonas aeruginosa</LI><LI> 	•	Sphingomonas yanoikuyae</LI><LI> 	•	Sphingomonas paucimobilis</LI><LI> 	•	Sphingomonas aromaticivorans</LI><LI> 	•	Novosphingobium pentaromativorans</LI></UL>Operons:<UL><LI> •	Rhamnolipid synthesis genes:</LI><LI>  	o	rmlBDAC</LI><LI>  	o	rhlABG</LI><LI>  	o	MFST-rhlC</LI><LI>  	o	algC</LI><LI> •	Biphenyl degradation:</LI><LI>  	o	bphA1a</LI><LI>  	o	bphA2</LI><LI>  	o	bphA3</LI><LI>   	o	bphA4</LI><LI>   	o	bphB</LI><LI>   	o	bphC</LI><LI>   	o	bphD</LI></UL><A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><A name="week2"><U>Competent Cells</U></A>
	
Electrocompetent cells were created with Escherichia coli JM109 following previously published protocol (Sambrook, 2001). 
	
<A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><A name="week3"><U>DNA Purification</U></A>
	
DNA was extracted and purified from P. aeruginosa, S. yanoikuyae, and S. paucimobilis using Promega’s Wizard Genomic DNA Purification Kit.  

<A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><A name="week4"><U>Using Standard Parts</U></A>

The following standard parts were used:<UL><LI> 	•	BBa_I13521 – RFP</LI><LI> 	•	BBa_I13522 – GFP</LI><LI> 	•	BBa_E0435 – lacZ alpha</LI></UL>
Standard kit instructions were followed to rehydrate the DNA.  The rehydrated plasmid DNA was then transformed into electrocompetent E. coli JM109 and plated onto selective media.  Pure cultures were obtained and stock cultures made.
<A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><A name="week5"><U>Polymerase Chain Reaction (PCR)</U></A>
PCR was done using colony PCR or from template DNA extracted from genomic DNA isolation.  Various PCR conditions were used to optimize the PCR products:
<UL><LI> 	•	Varied annealing temperatures</LI><LI> 	•	Varied magnesium levels to optimize polymerase activity</LI><LI> 	•	Varied buffers</LI></UL>
The following polymerases were used:
<UL><LI> 	•	GoTaq polymerase from Promega</LI><LI> 	•	GoTag Long PCR Master Mix from Promega</LI><LI> 	•	Q5 High Fidelity from New England BioLabs</LI><LI> 	•	OneTaq Master Mix from New England BioLabs</LI></UL>
PCR reactions were checked by DNA gel electrophoresis.  An example of one such gel is below:

     <A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><A name="week6"><U>PCR Purification</U></A>
	PCR products were purified using two different protocols based on the quality of the electrophoresed DNA gels:
	<UL><LI> 	•	QIAGEN QIAquick PCR Purification Kit was used for DNA that gave one clear band on the gel.  </LI><LI> 	•	QIAGEN QIAquick Gel Extract Kit was used to extract DNA that show multiple PCR products.  </LI></UL><A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><A name="week7"><U>Joining DNA Fragments</U></A>

PCR fragments were joined by two methods:<UL><LI> 	•	Chimeric PCR – Fragments of DNA are often joined together by restriction endonuclease digestion followed by ligation, but the fragments of DNA can also be joined together using just PCR. We used the following method to join multiple fragments of double-stranded DNA products together:</LI><LI>   	•	PCR products rhlA and rhlG (for Rhamnolipid biosynthesis) had already been amplified from P. aeruginosa, and we wanted to join them together so they could be part of a single operon on our recombinant plasmid. A new primer was designed that would serve as the forward primer for rhlG such that it contained a 5’ extension with the ending sequence (approximately the last 20 bp) of rhlAB. The original reverse primer for rhlG could be used to amplify, and the resulting PCR product contained a sequence identical to the end of rhlAB.</LI><LI>   	•	Next the two PCR fragments were combined along with only the forward primer for rhlAB and the reverse primer of rhlG. The denaturing step of PCR separates the double-stranded DNA, and during the annealing step, the overlapping sequences of rhlAB and rhlG anneal to each other along with the primers. Then during the elongation step, a single double-stranded DNA product is formed that contains both of the original DNA fragments.
</LI><LI> 	•	Restriction digest and ligation – Restriction enzymes from New England BioLabs and Promega, and T4 ligase from Promega were used.</LI></UL><A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><A name="week8"><U>Transformation</U></A>

DNA fragments were ligated into plasmids pSB1C3 and transformed into E. coli JM109.  Plasmids were purified and sequenced using Sanger’s sequencing to verify successful cloning.

<A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><A name="week9"><U>Testing Biphenyl Degradation Genes</U></A>
To be completed.

<A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><A name="week10"><U>Testing Rhamnolipid Biosynthesis Genes</U></A>
To be completed.

<A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><A name="week11"><U>BioBricks</U></A>
Three BioBricks were designed for rhamnolipid biosynthethis.  Each BioBrick contains a promoter, three to four coding sequences for rhamnolipid biosynthethis enzymes, a reporter gene, ribosome binding sites for all coding sequences, and a double terminator.  



<A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><A name="week12"><U>References</U></A>

Sambrook, J.  2001. Molecular Cloning.  Cold Spring Harbor, Ney York: Cold Spring Harbor Laboratory Press.  
<A href="#don"><P align="center"><FONT size="2">top</FONT></P></A><DIV id="footer"><DIV id="container_121" class="container_121"><TABLE align="left"><TBODY><TR><TD> </TD><TD> </TD></TR><TR><TD> </TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Clemson/Notebook">http://2012.igem.org/Team:Clemson/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Clemson/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Clemson/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Clemson/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Clemson/Notebook&amp;oldid=300605" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>