"
Page
Discussion
View source
History
teams
Log in
 
Team:Clemson/Notebook
From 2012.igem.org
CU iGEM
Home
Team
Project
Notebook
Attributions
Safety
Ethics
Links
Notebook
Primer Design
Competent Cells
DNA Purification
Using Standard Parts
Polymerase Chain Reaction (PCR)
PCR Purification
Joining DNA Fragments
Transformation
Testing Biphenyl Degradation Genes
Testing Rhamnolipid Biosynthesis Genes
BioBricks
References
Primers
Primers were designed using sequences obtained from the National Center for Biotechnology Information . To design the primers, we used ClustalX to align sequences and observe conserved regions of DNA.
Organisms:
  • Pseudomonas aeruginosa
  • Sphingomonas yanoikuyae
  • Sphingomonas paucimobilis
  • Sphingomonas aromaticivorans
  • Novosphingobium pentaromativorans
Operons:
 • Rhamnolipid synthesis genes:
   o rmlBDAC
   o rhlABG
   o MFST-rhlC
   o algC
 • Biphenyl degradation:
   o bphA1a
   o bphA2
   o bphA3
   o bphA4
   o bphB
   o bphC
   o bphD
top
Competent Cells
Electrocompetent cells were created with Escherichia coli JM109 following previously published protocol (Sambrook, 2001).
top
DNA Purification
DNA was extracted and purified from P. aeruginosa, S. yanoikuyae, and S. paucimobilis using Promega’s Wizard Genomic DNA Purification Kit.
top
Using Standard Parts
The following standard parts were used:
  • BBa_I13521 – RFP
  • BBa_I13522 – GFP
  • BBa_E0435 – lacZ alpha
Standard kit instructions were followed to rehydrate the DNA.  The rehydrated plasmid DNA was then transformed into electrocompetent E. coli JM109 and plated onto selective media.  Pure cultures were obtained and stock cultures made.
top
Polymerase Chain Reaction (PCR)
PCR was done using colony PCR or from template DNA extracted from genomic DNA isolation.  Various PCR conditions were used to optimize the PCR products:
  • Varied annealing temperatures
  • Varied magnesium levels to optimize polymerase activity
  • Varied buffers
The following polymerases were used:
  • GoTaq polymerase from Promega
  • GoTag Long PCR Master Mix from Promega
  • Q5 High Fidelity from New England BioLabs
  • OneTaq Master Mix from New England BioLabs
PCR reactions were checked by DNA gel electrophoresis.  An example of one such gel is below:
top
PCR Purification
PCR products were purified using two different protocols based on the quality of the electrophoresed DNA gels:
  • QIAGEN QIAquick PCR Purification Kit was used for DNA that gave one clear band on the gel.
  • QIAGEN QIAquick Gel Extract Kit was used to extract DNA that show multiple PCR products.
top
Joining DNA Fragments
PCR fragments were joined by two methods:
  • Chimeric PCR – Fragments of DNA are often joined together by restriction endonuclease digestion followed by ligation, but the fragments of DNA can also be joined together using just PCR. We used the following method to join multiple fragments of double-stranded DNA products together:
    • PCR products rhlA and rhlG (for Rhamnolipid biosynthesis) had already been amplified from P. aeruginosa, and we wanted to join them together so they could be part of a single operon on our recombinant plasmid. A new primer was designed that would serve as the forward primer for rhlG such that it contained a 5’ extension with the ending sequence (approximately the last 20 bp) of rhlAB. The original reverse primer for rhlG could be used to amplify, and the resulting PCR product contained a sequence identical to the end of rhlAB.
    • Next the two PCR fragments were combined along with only the forward primer for rhlAB and the reverse primer of rhlG. The denaturing step of PCR separates the double-stranded DNA, and during the annealing step, the overlapping sequences of rhlAB and rhlG anneal to each other along with the primers. Then during the elongation step, a single double-stranded DNA product is formed that contains both of the original DNA fragments.
  • Restriction digest and ligation – Restriction enzymes from New England BioLabs and Promega, and T4 ligase from Promega were used.
top
Transformation
DNA fragments were ligated into plasmids pSB1C3 and transformed into E. coli JM109.  Plasmids were purified and sequenced using Sanger’s sequencing to verify successful cloning.
top
Testing Biphenyl Degradation Genes
To be completed.
top
Testing Rhamnolipid Biosynthesis Genes
To be completed.
top
BioBricks
Three BioBricks were designed for rhamnolipid biosynthethis.  Each BioBrick contains a promoter, three to four coding sequences for rhamnolipid biosynthethis enzymes, a reporter gene, ribosome binding sites for all coding sequences, and a double terminator.
top
References
Sambrook, J.  2001. Molecular Cloning.  Cold Spring Harbor, Ney York: Cold Spring Harbor Laboratory Press.
top
 
 
 
 
 
Retrieved from " http://2012.igem.org/Team:Clemson/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
