Team:BGU ISRAEL/Notebook
PlastiCure
Toggle navigation
Home
Project
Description
Design
Experiments
Results
Protocols
Notebook
Achievements
Team
Members
Meet Our University
Meet Our Town
Collaborations
Parts
Overview
Basic Parts
Improved Parts
Human Practices
Integrated Practices
Plastic Art
Ethics & Safety
Attributions
Awards
Entrepreneurship
Measurements
Proof Of Concept
Demonstrate
HP - Silver
HP - Gold
Engagements
Notebook
October
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
August
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
July
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
May
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
April
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
March
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
When - 18/10/16
Who's In the lab today: Eyal and Dor
Colony PCR for the dialysis bag experiment.
We used 2 genes to identify if e.coli and putida can grow together, 
aldA (1440bp) gene for e.coli and pcaD (792bp) gene for putida.
 
aldA gene of E. coli, plates 3,7,10,1,6 were positive
pcaD gene of P. putida, plates 8,2,3,5,1 were positive
When - 17/10/16
Who's In the lab today: Eyal, Dor, Liran and Tomer
Ligation of 7K transporter inserts and pSEVA 434 vector
 
50ng vectur cut with EcorI and PstI
142ng insert cut with EcorI and PstI
2ul T4 DNA ligase Buffer X10
1ul T4 DNA ligase
add UPW for total volum of 20ul
 
16C over night
 
Transformation of 7K trensporter + pSEVA 434 Ligation mix
Tubes of compatent BL21
1ul from each plasmid for each tubes and one tube without plasmid.
Mix with tip.
Incubation in ice for 20 min.
Heat shock 30 sec 42C
Incubation in ice for 3 min
800ul SOC without antibiotic
1h in 37C
-centrifugation 5000 rpm 2min
-take off 500ul from the liquid.
-mix the residue with the rest liquid.
-sowing in LB+antibiotic
Incubation 37C O/N
When - 15/10/16
Who's In the lab today: Eyal and Dor
TPA detection assay
Took 2 sample of 200uL one inside the bag, and one outside the bag into 96 wellplate
 
TPA detection for kinetics measurements:
We transformed WT and variants LC-Cutinase to E. coli BL21 and we raised each one of them in M9+PET medium+CMP+IPTG.
We took samples of 200uL daily from each Erlenmeyer into 96 wellplate anti UV.
We took the plate to a UV table for 1 hour, and insert it into a plate reader for checking the absorption according to the time and date samples, and the change in TPA concentration in time according to our TPA detection protocol. The results show significant raising in the absorption but we didn’t succeed to detect the TPA.
The reasons why we didn’t succeed to detect TPA can be contamination of the medium with another carbon source or maybe because we didn’t filter the samples so there are some materials that can cause quenching and disturb the fluorescence.
 
When - 14/10/16
Who's In the lab today: Eyal and Dor
TPA detection assay
Took 2 sample of 200uL one inside the bag, and one outside the bag into 96 wellplate.
When - 13/10/16
Who's In the lab today: Tomer
Restriction reaction for pSEVA 434
The total reaction mix volum were 50ul.
every reaction contain:
1000ng DNA
1ul of every enzyme.
5ul buffer
 
 
Plasmid
E1
E2
Buffer
1
434
AvrII
-
3.1
2
434
EcorI
-
3.1
3
434
PstI
-
3.1
4
434
EcorI
PstI
3.1
5
434
-
-
3.1
When - 11/10/16
Who's In the lab today: Liran and Tomer
Produced pSEVA plasmids from starters using mini prep kit.
 
 
Concentration ng/ul
224
100
434
118
414
126
 
Restriction reaction for pSEVA 242, pSEVA 414 and pSEVA 434:
The total reaction mix volum were 50ul.
every reaction contain:
1000ng DNA
1ul of every enzyme.
5ul buffer
 
 
Plasmid
E1
E2
Buffer
1
434
AvrII
-
cutsmart
2
434
EcorI
-
cutsmart
3
434
PstI
-
cutsmart
4
434
EcorI
PstI
cutsmart
5
434
SpeI
SacII
cutsmart
6
434
-
-
cutsmart
9
414
SpeI
-
cutsmart
10
414
SacII
-
cutsmart
11
414
SacII
SpeI
cutsmart
12
414
-
-
cutsmart
13
224
AvrII
-
cutsmart
14
224
EcorI
-
cutsmart
15
224
PstI
-
cutsmart
16
224
EcorI
PstI
cutsmart
17
224
-
-
cutsmart
 
1h 37C
20min 65C
 
When - 10/10/16
Who's In the lab today: Liran and Inbar B
Colony-PCR for DH10B competent cells after transformation with ligation mix of 7K trensporter + pSEVA 434
 
 
Templat
pr Rev
pr Fw
taq readymix
UPW
colony
colony
1ul
1ul
10ul
7ul
7K insert posetiv control
5ul 7K insert
1ul
1ul
10ul
3ul
no templat
-
1ul
1ul
10ul
8ul
 
Gibson assembly of EG insert and pSEVA434 vector with Gibson assembly kit
Total volum 5ul
 
EG insert after DpnI
pSEVA434
posetive control mix
Gibson assembly mix
UPW
EG + 434
45ng
27ng
-
2.5ul
-
434 no insert- negativ control
-
27ng
-
2.5ul
1ul
Posetive control from the kit
-
-
2.5ul
2.5ul
-
 
20min 50C
 
Transformation of GA reaction mix:
Tubes of compatent DH10B
1ul from each plasmid for each tubes.
Mix with tip.
Incubation in ice for 20 min.
Heat shock 30 sec 42C
Incubation in ice for 3 min
800ul SOC without antibiotic
1h in 37C
-centrifugation 5000 rpm 2min
-take off 500ul from the liquid.
-mix the residue with the rest liquid.
-sowing in LB+antibiotic
Incubation 37C O/N
 
5 colonys in posetive control, no colonys on the other plates.
When - 09/10/16
Who's In the lab today: Liran, Eyal and Dor
We sent 4 samples contain cutinase WT, R7, F4 and CO for mass spec.
Transformation of 7K trensporter + pSEVA 434 Ligation mix
Tubes of compatent DH10B
1ul from each plasmid for each tubes and one tube without plasmid.
Mix with tip.
Incubation in ice for 20 min.
Heat shock 30 sec 42C
Incubation in ice for 3 min
800ul SOC without antibiotic
1h in 37C
-centrifugation 5000 rpm 2min
-take off 500ul from the liquid.
-mix the residue with the rest liquid.
-sowing in LB+antibiotic
Incubation 37C O/N
 
Produced pSEVA plasmids from starters using mini prep kit.
 
 
Concentration ng/ul
414 stab 2
69
414 col2
72
414 col2
61
414 stab 2
130
434 stab
136
224 gly
195
 
Restriction reaction for pSEVA 242, pSEVA 414 and pSEVA 434:
The total reaction mix volum were 50ul.
every reaction contain:
1000ng DNA
1ul of every enzyme.
5ul buffer
 
 
Plasmid
E1
E2
Buffer
1
434 stab
AvrII
-
cutsmart
2
434 stab
EcorI
-
2.1
3
434 stab
PstI
-
3.1
4
434 stab
EcorI
PstI
2.1
5
434 stab
SpeI
-
cutsmart
6
434 stab
SacII
-
cutsmart
7
434 stab
SacII
SpeI
cutsmart
8
434 stab
-
-
cutsmart
9
414 stab 2
SpeI
-
cutsmart
10
414 stab 2
SacII
-
cutsmart
11
414 stab 2
SacII
SpeI
cutsmart
12
414 stab 2
-
-
cutsmart
13
224 gly
AvrII
-
cutsmart
14
224 gly
EcorI
-
2.1
15
224 gly
PstI
-
3.1
16
224 gly
EcorI
PstI
2.1
17
224 gly
-
-
cutsmart
 
1h 37C
20min 65C
 
When - 08/10/16
Who's In the lab today: Eyal, Dor, Liran  and Efrat
We prepared the dialysis bag experiment one again according to our protocol for kinetics measurements using our TPA detection assay- we will take sample from inside and outside the bag twice a day for 1 week and then we will expose it to UV light according to the TPA detection protocol to measure the kinetics index of cutinase CO on PET.
We used dialysis bag to check if e coli can live with P. putida- e coli inside the dialysis bag and putida outside of the dialysis bag in LB medium.- we will take samples inside the dialysis bag and outside the dialysis bag and sow it on  cam, kan, amp plate to see if they are still alive together (putida has a endogenic resistant to cam and amp, and we insert pACYC to e coli for cam resistant).
Restriction reaction for pSEVA 242, pSEVA 414 and pSEVA 434:
The total reaction mix volum were 50ul.
every reaction contain:
1000ng DNA
1ul of every enzyme.
5ul buffer
 
 
Plasmid
E1
E2
Buffer
1
434 stab 2
AvrII
-
cutsmart
2
434 stab 2
EcorI
-
2.1
3
434 stab 2
PstI
-
3.1
4
434 stab 2
EcorI
PstI
2.1
5
434 stab 2
-
-
cutsmart
6
434 stab 2
SpeI
-
cutsmart
7
434 stab 2
SacII
-
cutsmart
8
434 stab 2
SacII
SpeI
cutsmart
9
414 stab 2
SpeI
-
cutsmart
10
414 stab 2
SacII
-
cutsmart
11
414 stab 2
SacII
SpeI
cutsmart
12
414 stab 2
-
-
cutsmart
13
224 gly1
PstI
-
cutsmart
14
224 gly2
EcorI
-
2.1
 
1h 37C
20min 65C
 
Ligation of 7K transporter inserts and pSEVA 434 vector
 
50ng vectur cut with EcorI and PstI
142ng insert cut with EcorI and PstI
2ul T4 DNA ligase Buffer X10
1ul T4 DNA ligase
add UPW for total volum of 20ul
 
16C over nighet.
 
BioBricks: produced plasmids from starters using mini prep kit.
 
plasmid
concentration(ng/ul)
260\230
260\280
F4 col1
301
2.06
1.85
F4 col3
283
2.04
1.87
F7 col3
327
2.23
1.87
F7 col4
218
1.80
1.87
R4 col2
291
2.10
1.90
R4 col4
314
2.10
1.87
R7 col5
313
2.07
1.86
R7 col6
313
2.07
1.86
PcaC col1
267
1.92
1.88
When - 07/10/16
Who's In the lab today: Tomer, Efrat, Liran and Inbar B
Examine PET degradation bay LCC CO:
3 PET bids were incubate with 0.5L LB contaning cmp and BL21 with LCC CO on pACYC plasmid.
When OD reach 0.5-0.7 add IPTG.
For control, repeat with pACYC plasmid without insert, and without bacteria.
 
C.O
pAYAC
 
Testing BL21 that contains LCC variants groth on PET plates:
PET plates:
For 1L:
200ml M9 salts
2ml 1M mgSO4
20ml glucose 20%
100ul 1M CaCl2
(5ml soft agar for each plate)
The 19 plates:
 
IPTG &PET
IPTG no PET
PET no IPTG
Glucose
F4
V
V
-
-
F7
V
V
-
-
R4
V
V
-
-
R7
V
V
V
-
CO
V
V
V
-
pACYC
V
V
V
-
WT
V
V
V
V
 
Produced pSEVA plasmids from starters using mini prep kit.
 
Concentration ng/ul
 
47
434 stab 1
85
434 stab 2
36
414 stab 1
159
414 stab 2
38
414 col1
121
414 col2
62
414 col3
370
414 col4
32
224 gly1
24
224 gly2
 
transformation of pSEVA 224 from gly1, pSEVA 414 from col2, pSEVA 414 from stab 2 and pSEVA 434 from stab 2:
Tubes of compatent DH10B
1ul from each plasmid for each tubes and one tube without plasmid.
Mix with tip.
Incubation in ice for 20 min.
Heat shock 30 sec 42C
Incubation in ice for 3 min
800ul SOC without antibiotic
1h in 37C
-centrifugation 5000 rpm 2min
-take off 500ul from the liquid.
-mix the residue with the rest liquid.
-sowing in LB+antibiotic
Incubation 37C O/N
 
PCR for amplification EG insert for Gibson with KAPA HIFI Hot Start, after previous reaction ended with Unspecific product
Template –EG 1ul to 49 ul UPW (EG first concentration- 572 ng\ul)
Primers- 10uM
Annealing temperature 70C, 1ul DMSO.
 
 
Upw (ul)
Pr fwd (ul)
Pr rev
template
KAPA Hiti hot start (ul)
EG
20
1.5
1.5
1
25
 
PCR to amplify TphA1, TphA2, TphA3 and TphB for Gibson assembly
Primers- 10uM
Annealing temperature 68C  
 
Upw (ul)
Pr fwd (ul)
Pr rev
template
KAPA Hiti hot start (ul)
TphA1
21
1.5
1.5
1
25
TphA2
21
1.5
1.5
1
25
TphA3
21
1.5
1.5
1
25
TphB
21
1.5
1.5
1
25
 
Using DpnI on EG, TphA1, TphA2, TphA3 and TphB to eliminate closed plasmid from the PCR product. pSB1C3 as control
 
44ul of PCR reaction mix
5ul x10 CutSmart buffer
1ul DpnI
37C for 1h
80C for 20min
 
TphA1- Unspecific product TphA2 -no product
When - 06/10/16
Who's In the lab today: Inbar B and Liran
Testing bacteriom growth on antibiotics stocks:
 
antibiotics
KT2440 from Glycerol stock
KT2440 from petri dish
Cmp
V
V
Cmp+Amp
V
V
Cmp+Strep
-
V
Spect
-
V
Cmp+Strep+Amp
-
-
Km
-
-
Strep X2
-
-
Spect X2
-
V
Control-no antibiotic
V
V
Amp
V
V
 
antibiotics
W.T
Control – no antibiotic
V
Cmp+Amp
V
Amp
V
Strep
-
 
 
antibiotics
E.Coli 224
Control – no antibiotic
V
Km 25 ug\ul
V
Km 50 ug\ul
V
Strep
-
Spect
-
 
Restriction reaction for pAC75 in order to get EG insert for Gibson after PCR product was Unspecific 
 
 
reaction 1
reaction 2
reaction 3
reaction 4
reaction 5
pAC75
3.5ul
3.5ul
3.5ul
3.5ul
5ul
cutsmart x10
10ul
10ul
10ul
10ul
-
UPW
83.5ul
85.5ul
85.5ul
85.5ul
-
KpnI
1ul
1ul
-
-
-
HindIII
1ul
-
1ul
-
-
AatII
1ul
-
-
1ul
-
 
1h 37C
20min 65C
 
When - 05/10/16
Who's In the lab today: Efrat and Liran
Preparing antibiotics stocks:
Spectinomycin 100mg\ml
cmp 34mg\ml
amp 100mg\ml
Kn 25mg\ml
 
Colony-PCR for DH10B competent cells after transformation with ligation mix of pSB1C3 and the inserts F4, F7, R4, R7 and CO:
 
template- toch colony, then reaction
 
Annealing temperature
 (C°)
Upw (ul)
Pr fwd (ul)
Pr rev
KAPA Hiti hot start (ul)
C.O
58.4
22
1.5
1.5
25
F4
58.4
22
1.5
1.5
25
F7
58.4
22
1.5
1.5
25
R4
58.4
22
1.5
1.5
25
R7
58.4
22
1.5
1.5
25
When - 04/10/16
Who's In the lab today: Inbar B, Tomer and Liran
PCR for amplification EG insert for Gibson with KAPA HIFI Hot Start, after previous reaction ended with Unspecific product
Template –EG 1ul to 49 ul UPW (EG first concentration- 572 ng\ul)
Primers- 10uM
Annealing temperature 68C
Using DpnI on EG insert to eliminate closed plasmid from the PCR product
 
44ul of EG PCR reaction mix
5ul x10 CutSmart buffer
1ul DpnI
37C for 1h
80C for 20min
 
Unspecific product
 
Restriction reaction for pSEVA 242, pSEVA 414, pSB1C3 and pUC19 as control:
The total reaction mix volum were 50ul.
every reaction contain:
1000ng DNA
1ul of every enzyme.
5ul buffer
Duplicates for every reaction.
 
 
Plasmid
E1
E2
Buffer
1
224
AvrII
-
cutsmart
2
224
EcorI
-
2.1
3
224
PstI
-
3.1
4
224
EcorI
PstI
2.1
5
224
DpnI
-
cutsmart
C1
224
-
-
cutsmart
6
414
SpeI
-
cutsmart
7
414
SacII
-
cutsmart
8
414
SpeI
SacII
cutsmart
9
414
DpnI
-
cutsmart
C2
414
-
-
cutsmart
10
pSB1C3
PstI
-
3.1
11
pSB1C3
EcorI
-
2.1
12
pSB1C3
EcorI
PstI
2.1
C3
pSB1C3
-
-
cutsmart
13
pUC19
PstI
-
3.1
14
pUC19
DpnI
-
cutsmart
C4
pUC19
-
-
cutsmart
 
1h 37C
20min 65C
 
 
transformation of LCC varients:
Tubes of compatent BL21
1ul from each plasmid for each tubes and one tube without plasmid.
Mix with tip.
Incubation in ice for 20 min.
Heat shock 30 sec 42C
Incubation in ice for 3 min
800ul SOC without antibiotic
1h in 37C
100ul from each tube to 5ml LB+5ul cmp.
Incubation 37C O/N
When - 03/10/16
Who's In the lab today: Efrat
BioBricks after PCR cleanup:
 
Concentration ng/ul
260/230
260/280
F7
313
1.96
1.88
R4
374
1.92
1.88
R7
294
2.00
1.9
F4
232
2.01
1.88
 
Restriction reaction for BioBriks inserts
 
250ng DNA (insert or PSB1C3 vector)
3ul NEB Buffer II
0.5ul PstI
0.5ul EcorI
add UPW for total volum of 20ul
 
1h 37C
20min 80C
 
BioBricks restriction product after PCR cleanup:
 
Concentration ng/ul
260/230
260/280
F7
92
1.49
1.88
R4
70
1.06
1.89
R7
91
2.01
1.88
F4
74
1.53
1.85
 
ligation of BioBriks F7, R4, R7 and F4 inserts and pSB1C3 vector
 
50ng vectur cut with EcorI and PstI
150ng insert cut with EcorI and PstI
0.5ul T4 DNA ligase Buffer
1ul T4 DNA ligase
add UPW for total volum of 50ul
 
16C over nighet.
 
Colony-PCR for DH10B competent cells after transformation with ligation mix of pSB1C3 and the inserts F4, F7, R4, R7 and CO:
 
template- toch colony, then reaction
 
Annealing temperature
 (C°)
Upw (ul)
Pr fwd (ul)
Pr rev
KAPA Hiti hot start (ul)
C.O
58.4
22
1.5
1.5
25
F4
58.4
22
1.5
1.5
25
F7
58.4
22
1.5
1.5
25
R4
58.4
22
1.5
1.5
25
R7
58.4
22
1.5
1.5
25
pACYC as negative control
58.4
22
1.5
1.5
25
 
When - 01/10/16
Who's In the lab today: Inbar B Eyal and Dor
Cutinase kinetics measurement:
We were trying to get the kinetics index of our cutinase- WT. F4, CO using the first velocity way (calculating the shape of the graph in the linear measurement) and exposing the V by that way.
After that, using the linear burk plot to reveal the kinetics index.
We tried to do this tests for more than 10 days, but we didn’t succeed to get a stable indexes. We tried to change the enzyme concertation to see measure the process more accurately, but still, we didn’t succeed to get stable indexes.
Using DpnI on EG insert to eliminate closed plasmid from the PCR product
 
44ul of EG PCR reaction mix
5ul x10 CutSmart buffer
1ul DpnI
37C for 1h
80C for 20min
 
When - 30/09/16
Who's In the lab today: Inbar B Liran and Tomer
PCR for amplification EG insert for doing Gibson reaction with KAPA HIFI Hot Start
Template –EG 1ul to 49 ul UPW (EG first concentration- 572 ng\ul)
Primers- 10uM
 
Restriction for pSEVA 224 and pSEVA414:
 
Plasmids sizes:
pSEVA 224 5250bp
pSEVA 414 3479bp
Transporter 4368bp
pUC19 2568bp
pSB1C3 2070 bp
 
For Gibson reactions:
Enzyme
Plasmid
Concentration
AvrII
pSEVA 224
112ng/ul
SpeI+SacII
pSEVA 414
173ng/ul
 
For restriction and ligation:
Enzyme
Plasmid
Concentration
EcoRI +PstI
pSEVA 224
112ng/ul
 
Enzyme
Buffer
EcoRI
3.1-100%
2.1-100% + star activity
PstI
3.1-100%
2.1-75%
AvrII
Cut smart-100%
1.1-100%
SacII
Cut smart-100%
2.1-100%
SpeI
Cut smart-100%
2.1-100%
 
Reactions:      
DNA
pSEVA224 10ul
pSEVA414 6ul
Buffer 3.1
5ul
Enzyme
1ul
DDW
pSEVA224 34ul
pSEVA414 38ul
Total
50ul
When - 27/09/16
Who's In the lab today: Eyal, Dor and Inbar B
Activity test for cutinase:
We tasted cleaned cutinase F4, CO and WT, and uncleaned cutinse F4, CO and WT in this test.
We used 96DW and a plate reader.
We added 50ul pNPB without cutinase, Uncleaned cutinase WT with pNPB (50ul and 50ul), Uncleaned cutinase R7 with pNPB (50ul and 50ul), Uncleaned cutinase F4 with pNPB (50ul and 50ul), pNPB and cleaned cutinase WT(50ul an 50ul), pNPB and cleaned cutinase
 
Restriction for pSEVA
 
Concentration ug/ul
260/230
260/280
pSEVA 224 EcoRI
50
3.14
1.72
pSEVA 224 EcoRI and PstI
67
0.82
1.70
pSEVA 414 EcoRI and PstI
90
1.99
1.86
PcaG EcoRI and PstI
63
0.97
1.74
pSEVA 514 EcoRI and PstI
19
0.30
1.41
EG
9
0.17
1.62
When - 26/09/16
Who's In the lab today: Liran and Tomer
Preparing Strep stock for P.putida
 
Stock concentration 100mg/ml
 
2g powder
20 ml UPW
 
Preparing TET stock for P.putida:
 
Stock concentration 10mg\ml
0.8g powder
30ml UPW
 
pSEVA plasmids concentration after restriction and cleanup:
 
pSEVA224 8.68 ng\ul
pSEVA224 after mini prep 84.48 ng\ul
pSEVA414 6.31 ng\ul
pSEVA414 after mini prep 3.01 ng\ul
pSEVA514 after mini prep 63.13 ng\ul
When - 25/09/16
Who's In the lab today: Efrat Liran and Tomer
BioBricks after PCR cleanup:
 
Concentration ug/ul
260/230
260/280
C.O
11.2
0.73
1.92
F4
235.05
1
1.98
F7
371.92
1.63
1.93
R4
57.18
0.38
2.18
R7
146.84
0.81
1.93
pCaH
106.59
0.86
1.98
pCaC
61.25
0.42
1.88
pCaD
27.55
0.12
2.32
pCaG
-
-
-
Transporter
254.54
1.55
1.93
Pseva 514
8.98
0.06
8.91
 
Ligation for pcaD\pcaH\pacC
 
PcaC
PcaD
PcaH
Control
Vector
2.8ul
28ul
28ul
28ul
Insert
1.4ul
2ul
3.1ul
-
Buffer
2ul
2ul
2ul
2ul
DDW
12.8ul
12.2ul
11.1ul
14.2ul
Ligase
1ul
1ul
1ul
1ul
 
Overnight 10C
Restriction- Transporter 7k (TPA)
DNA
10ul
Buffer 3.1
10ul
EcoRI
1ul
PstI
1ul
DDW
78ul
Total
100ul
 
Restriction time- 60 min in 37 °C
Inactivation- 20 min in 65°C
When - 24/09/16
Who's In the lab today: Inbar B Liran and Tomer
Produced the pSBIC3 plasmid from the starter and cleaned it using mini prep kit in 6 eppendorf:
 
Concentration ug/ul
260/230
260/280
1
215
2.04
1.85
2
401
2.09
1.88
3
272
1.88
1.84
4
346
2.14
1.87
5
265
2.15
1.89
6
263
2.12
1.87
 
PCR-Transporter for TPA (location 7K from iGEM plates)
 
Transporter
Control
Template
0.5ul
-
P.RW
1.5ul
0.75ul
P.FOW
1.5ul
0.75ul
Hot start
25ul
12.5ul
DDW
21.5ul
11ul
Total
50ul
25ul
 
PCR Program:
 
Temp
Time
Cycle
1.
95°C
3 min
1
2.
98°C
20 sec
25
3.
65°C
15 sec
25
4.
72°C
180 sec
25
5.
72°C
4 min
1
When - 22/09/16
Who's In the lab today: Efrat
Restriction:
Plasmid: pSB1C3
 
Plasmids
Buffer
EcoRI
PstI
SacI
DDW
Total
7.06ul
10ul
1ul
1ul
1ul
79.94ul
100ul
7.06ul
10ul
1ul
1ul
-
80.94ul
100ul
7.06ul
10ul
-
1ul
-
81.94ul
100ul
7.06ul
10ul
1ul
-
-
81.94ul
100ul
7.06ul
10ul
1ul
-
1ul
80.94ul
100ul
Gene
Buffer
EcoRI
PstI
SacI
DDW
Total
10ul
10ul
1ul
1ul
-
78ul
100ul
 
Concentration after clean up:
Gene
Concentration ng/ul
pcaH
5.34
pcaG
8
pcaD
-5.34
pcaC
19.99
C.O
-5.91
F4
9.59
F7
18.55
R4
12.10
R7
44.28
When - 21/09/16
Who's In the lab today: Ben and Inbar B
Runing pSEVA vectors on gel in order to confirm thire exsistens in old sempels
 
PCR clean up
 
gene
concentration(ng/ul)
260\230
pSB1C3 cut EcorI PstI
7
0.02
When - 20/09/16
Who's In the lab today: Inbar B, Ben, Liran Eyal and Dor
Prepering XL1 heat shock competent cells, after the previous XL1 competent cells fail at the test kit.
Repeat PCR to amplify TphA2 for Gibson assembly
template- old PCR product
 
Annealing temperature
(C°)
Upw (ul)
Pr fwd (ul)
Pr rev
template
KAPA Hiti hot start (ul)
TphA2
68
21
1.5
1.5
1
25
TphA2X
68
5.5
0.38
0.38
-
6.25
 
 
Runing pSEVA vectors on gel in order to confirm thire exsistens in old sempels
 
Repeat PCR to amplify LCC variants for BioBriks because efret used all the sempels
Primers 10 uM.
 
 
Annealing temperature
 (C°)
Upw (ul)
Pr fwd (ul)
Pr rev
template
KAPA Hiti hot start (ul)
C.O
58.4
21
1.5
1.5
1
25
F4
58.4
21
1.5
1.5
1
25
F7
58.4
21
1.5
1.5
1
25
R4
58.4
21
1.5
1.5
1
25
R7
58.4
21
1.5
1.5
1
25
 
Activity test for cutinase:
We tasted cleaned cutinase F4, R7 and WT, and uncleaned cutinse F4, R7 and WT in this test.
We used 96DW and a plate reader.
We added 100ul pNPB without cutinase, Uncleaned cutinase WT with pNPB (100ul and 100ul), Uncleaned cutinase R7 with pNPB (100ul and 100ul), Uncleaned cutinase F4 with pNPB (100ul and 100ul), pNPB and cleaned cutinase WT(100ul an 100ul), pNPB and cleaned cutinase R7(1 00ul an 100ul) and pNPB and cleaned cutinase F4(100ul an 100ul). ). [We duplicate every sample]
25/9
We prepared a stock of TPA 50 ml 20mM according to our protocol (TPA detection protocol)/ after that, we diluted the stock to get concentration of 15mM, 10mM, 5mM, 1mM, 0.5mM, 0.1mM, 0.05mM, 0.01mM, 0.005mM, 0.001mM.
We added 200uL from each concentration to a different wall in 96W anti-UV plate.
We exposed the plate to UV according to the protocol and search for fluorescence emission we saw in the literature.
After few days of testing this method we wrote the TPA detecting assay.
When - 19/09/16
Who's In the lab today: Inbar B, Ben and Efret
Produced plasmids from the starters using mini prep kit.
 
plasmid
concentration(ng/ul)
260\230
260\280
pSEVA 514
151
2.00
2.05
pSB1C col1
272
2.15
1.93
pSB1C col2
282
2.19
1.91
 
PCR to amplify TphA1, TphA2, TphA3 and TphB for Gibson assembly
 
iGEM competent cell test kit
XL1
BL21
 
Ligation of BioBriks inserts and pSB1C3 vector
 
50ng vectur cut with EcorI and PstI
150ng insert cut with EcorI and PstI
0.5ul T4 DNA ligase Buffer
1ul T4 DNA ligase
add UPW for total volum of 50ul
 
16C over nighet.
When - 18/09/16
Who's In the lab today: Tomer and Liran
PCR to amplify 7K transporter from plasmid
 
 
Annealing temperature
 (C°)
Upw (ul)
Pr fwd (ul)
Pr rev
template
KAPA Hiti hot start (ul)
7K transporter
68
21
1.5
1.5
1
25
RFP as posetiv control with pSV1C as template
68
21
1.5
1.5
1
25
When - 17/09/16
Who's In the lab today: Eyal and Dor
Dialysis experiment:
We prepared the dialysis experiment according to the “Dialysis bag experiment” using the starters we raised two day ago. We tested OD every day.
 
Date
time
Outside the dialysis bag
Inside the dialysis bag
17/9
13:00
0.09
0.108
18/9
13:30
0.1
0.31
19/9
12:15
0.1
0.4
20/9
14:00
0.107
0.42
21/9
15:00
0.1
0.24( the bag was fallen)
22/9
11:00
0.2
0.4
23/9
14:00
0.2
0.6
24/9
13:30
0.5
0.64
When - 15/09/16
Who's In the lab today: Idit, Eyal and Dor
the dialysis bag content was loded onto a cation seperation colon
 
We prepared starters with e-coli and pACYC.
 After one week we saw that the M9 outside of the dialysis bag was contaminated with e coli.
 
Transformation of pSEVA plasmids to P.putida KT2440 competent cells and groing cells on diffrent antibiotics in order to check the competent cells
Compatent P.putida KT2440 from -80C to ice for 10min.
1 ul from each plasmid to each tube and a few tubes without plasmid(negative control on diffrent antibiotics)
mix with tip
incubate in ice for 20 min
heat shock 42C 3min.
incubate in ice for 3min
SOC 900ul without antibiotic
37C for 2h
centrifugate 500rpm 2min
carrying of 900ul liquid from each tube.
mix the residue with the liquid
sowin on LB plates with relavent antibiotic.
 
number of times that bacterias grows on difrent antibiotics, after trensformesuon of diffrent plasmids.
When - 14/09/16
Who's In the lab today: Efrat
protein seperation
 
the solution was centefuges in 16000g  for an hour
the sediment was re-suspendent in TAE buffer pH 7.
the solution was inserted into a dialysis bag over night.
When - 13/09/16
Who's In the lab today: Efrat
Restriction reaction for BioBriks inserts
 
250ng DNA (insert or PSB1C3 vector)
3ul NEB Buffer II
0.5ul PstI
0.5ul EcorI
add UPW for total volum of 20ul
 
1h 37C
20min 80C
 
protein seperation
 
500 ml of each varient was centrifuged in 8000g for 10 minutes.
amonium solfate was added to the supernatant and incubated over night on a magnetic plate with a stirerer.
When - 12/09/16
Who's In the lab today: Efrat
Repeat PCR to amplify PcaH, PcaG ,PcaD, PcaC from P.putida KT2440
 
PCR clean up
 
gene
concentration(ng/ul)
260\230
PcaH
86
1.03
PcaG
96
0.98
PcaD
25
1.32
PcaC
81
1.39
protein seperation
 
the starters were diluted in 500 ml LB
After achieving 0.9-1 OD values IPTG was added and the incubated overnight in 37 degrees Celsius and agitation of 300 rpm
When - 11/09/16
Who's In the lab today: Inbar B and Efrat
Prepering P.putida KT2440 competent cells:
protein seperation
 
Three variants were chosen CO, F4, R7 for separation.
First a transformation into BL21 competents was performed
1 µl of plasmid was added into 50 µl of competent batch and incubated on ice for 30 min.
Heat shock for 30 sec in 42 degrees Celsius
Incubation for 3 min.
950 µl of SOC medium was add and placed into incubation for an hour in 37 degrees Celsius.
After incubation the bacteria were precipitated and them re-suspended in 200 µl in LB
Starters of 5 ml were made from the transformation to use in the following day
 
PCR clean up
 
gene
concentration(ng/ul)
260\230
PcaH
1
0.01
PcaG
28
0.6
PcaD
33
0.27
PcaC
30
0.2
CO
133
0.99
F4
123
0.93
F7
152
0.63
R4
140
0.38
R7
157
0.47
When - 10/09/16
Who's In the lab today: Eyal and Dor
Activity test for cutinase:
We tasted cleaned cutinase WT and uncleaned cutinase WT in this test.
We used 96DW and a plate reader.
In one wall we added 100ul pNPB without cutinase. In the second wall we added uncleaned cutinase WT with pNPB (100ul and 100ul), and in the third wall we added pNPB and cleaned cutinase WT (100ul an 100ul). [We duplicate every sample]
 
Dialysis experiment:
Transformation was done to insert pACYC with CMP resistant and cutinase WT gene into BL21 e-coli.
We prepared starters with this e-coli.
We prepared M9+PET (we crashed the PET with coffee grinder).
When - 06/09/16
Who's In the lab today: Inbar B and Efrat
PCR to amplify PcaH, PcaG ,PcaD, PcaC from P.putida KT2440 (05.09.16): for BioBriks
Primers 10 uM template- toch colony, then reaction.
KAPA Hiti hot start (ul)
Pr rev
Pr fwd (ul)
Upw (ul)
Annealing temperature
(C°)
 
 
25
1.5 PcaH
1.5 PcaH
22
61.6
PcaH
1
25
1.5
1.5
22
58.4
PcaG
2
25
1.5
1.5
22
62.3
PcaD
3
25
1.5
1.5
22
58.4
PcaC
4
6.25
0.38
0.38
5.5
61.6
HX
5
6.25
0.38
0.38
5.5
58.4
GX
6
6.25
0.38
0.38
5.5
62.3
DX
7
6.25
0.38
0.38
5.5
58.4
CX
8
KAPA Hiti hot start (ul)
template
Pr rev
Pr fwd (ul)
Upw (ul)
Annealing temperature
(C°)
 
25
1
1.5
1.5
21
58.4
C.O
25
1
1.5
1.5
21
58.4
F4
25
1
1.5
1.5
21
58.4
F7
25
1
1.5
1.5
21
58.4
R4
25
1
1.5
1.5
21
58.4
R7
6.25
-
0.38
0.38
5.5
58.4
C.OX
6.25
-
0.38
0.38
5.5
58.4
F4X
6.25
-
0.38
0.38
5.5
58.4
F7X
6.25
-
0.38
0.38
5.5
58.4
R4X
6.25
-
0.38
0.38
5.5
58.4
R7X
When - 04/09/16
Who's In the lab today: Everybody!
Successfully ligated:
 
Sample
Ligated to pSB1C3
pcaC
 
pcaD
Yes (Sample 2)
pcaG
 
pcaH
 
CO
 
R4
 
R7
Yes (Sample 6)
F4
 
F7
 
When - 29/08/16
Who's In the lab today: Everybody!
we made PET plates: For 1L:
200ml M9 salts
2ml 1M mgSO4
20ml glucose 20%
100ul 1M CaCl2
(5ml soft agar for each plate)
the plate that we made:
50
100
250
500
no PET
Glucose
When - 17/08/16
Who's In the lab today: Everybody!
We concentrated the cutinase with centricons.
When - 11/08/16
Who's In the lab today: Everybody!
we made M9 plates without carbon source.
we made a new stock for chloramphenicol.
we sow from the rhodococus experiment on natrient broth plates.
gel weight:
pcaC
pcaD
pcaG
pcaH
CO
R4
R7
F4
F7
0.2816
0.2801
0.1882
0.2811
0.3890
0.2891
0.2882
0.3887
0.2855
Nano drop sup: (calibration by pACYC)
Cutinase variants
Protein concentration
WT
0.83
CO
0.43
R4
-0.21
F4
-0.37
R7
-0.09
F7
-0.95
protein concentration after centrecon:
Cutinase variants
Protein concentration
WT
4.46
R4
2.41
F4
2.23
R7
2.54
F7
2.41
pACYC
3.9
When - 10/08/16
Who's In the lab today: Everybody!
BL21 growth and induction for expression LCC for activity test:
Cutinase variants
14:37
15:15
pACYC
0.215
0.458
WT
0.267
0.497
CO
0.236
0.440
R4
0.221
0.413
F4
0.236
0.446
R7
0.195
0.376
F7
0.208
0.401
15:35-5ul IPTG 1M for each 5ml preparation of IPTG:
0.1gr +420ul DDW
fiber with 0.22um filter.
When - 09/08/16
Who's In the lab today: Everybody
Cutinase experiment:
nano-drop protein measurement (calibration by pACYC)
Cutinase variants
Protein concentration
pACYC
0
WT
0.22
CO
2.16
R4
-1.68
F4
-0.37
R7
0.25
F7
0.4
F4
0.14
BL21
0.76
When - 070816
Who's In the lab today: Everybody!
Competent cells
5ml LB were taken from erlenmeyer 500ml for calibration the spectophotometer before O.D test.
dilution 5ml BL21 culture starters to 500 ml LB.
when the O.D reach to 0.4-0.6 we mix and save in ice for 10min
we transfer the LB to surbal tubes.
centrifuge 10 min, 6000 rpm, 4C
we removed the sup and gentle fluidization in 25ml solution A for 125ml culture.
30min incubation in ice.
centrifuge 5min, 5000, 4C
gentle fluidization all the plet in 10ml solution B.
we divided the solution to ependorf, 50 ul for each ependorf and throw it quickly to liquid nitrogen.
transformation for each LCC variants and control for activity test:
we took 8 tubes of competent BL21.
1ul from each plasmid for each tubes and one tube without plasmid.
mix with tip.
incubation in ice for 20 min.
heat shock 30 sec 42C.
incubation in ice for 3 min.
800ul SOC without antibiotic.
1h in 37C.
100ul from each tube to 5ml LB+5ul cmp. for negative control-plate without antibiotic.
incubation 37C O/N.
When - 01/08/16
Who's In the lab today: Everybody!
the starters diluted to a new starter. O.D tests before the induction:
Cutinase variants
O.D 14:00
pACYC
0.286
WT
0.240
CO
0.239
R4
0.244
F4
0.250
R7
0.252
F7
0.225
F4
0.14
BL21
0.438
when we checked the O.D at 15:30 all the samples O.D where more than 1.0. we assumed that something is wrong with this experiment, maybe the O.D check in 14:00
When - 31/07/16
Who's In the lab today: Ben
we made starters for LCC activity tests. we are doing the same test like we have done before.
When - 27/07/16
Who's In the lab today: Ben
protocol for comasi x10:
30 gr Tris base.
144 gr glycine
10 gr SDS
we dissolve those component in 800ml.
after we dissolove all the component we complete the solution to 1000 ml.
Experiment order:
we need to grow up 5ml varients, W.T, and only plasmid(according to the protocol)
after 1 day we disolve and collect the sup and transfer it to centricon.
we concentrated till 0.5ml(x10)
we swiched the buffer with another Tris
we centrifuge every 5 min for keep the materials in the solution
When - 26/07/16
Who's In the lab today: Tomer
pNPB testing according to the protcol
WT
0.489
CO
0.54
F7
0.5
R7
0.52
O.D testing for LCC varients
WT
0.514
F7
0.525
BL21
0.67
biobrick primers arrived:
for each primer we added DDW as writen in the form to get the concentration of 100mM.
we diluted each primer to a new eppendorf to get a concentration of 10mM by mix 5ul primer+45ul DDW.
When - 18/07/16
Who's In the lab today: Tomer
activity test like we had done before.
When - 17/07/16
Who's In the lab today: Tomer
transformation for all the varients +BL21 without control.
we added 16 starters ( 2 for each plasmid+2 control)- 8 pNPB tubes, 8 PET tubes.
When - 11/07/16
Who's In the lab today: Eyal
Turbidity check and induction:
7 ml LB+ 7ul CP+ 70ul from each starter. 200 rpm, 37C.
dilution of x2→ 1 ml culture+ 1 ml LB
 
11:40
12:40
LCC WT
0.117x2
0.371x2
pACYC
0.116x2
0.360x2
BL21
0.153x2
0.450x2
at 12:40→ we added 1M IPTG for each tube.
Activity test:
centrifuge each culture with 4C 30min 7100g. we transfer thw sup for new tubes.
A. we had made stock of pNPB 100 mM→ 17.5 ul pNPB+ 982.5 ul analytical ethanol
B. we had made Tris pH=8 0.25mM→ dilution Tris 1M- 1ml to 39ml UPW
we added 100 ul of the sup near the plate reader with multichannel
we prepared pNBP:
for concentration A- 50 ul from the stock to 9.95ml Tris
for concentration B-100 ul from the stock to 9.9ml Tris
we added pNPB for each tube. 100 ul for each tube
checking absorption in 405nm per min for 50 min
When - 10/07/16
Who's In the lab today: Eyal
transformation for activity test LCC, plasmid pACYC without insert and pACYC LCC WT C6:
The aim of this test- transformation for compatent bacteria BL21 and activity test will check how much pNPB we need to use in further experiment.
3 tubes, compatent BL21 for -80C to ice for 10min.
1 ul from each plasmid to each tube and 1 tube without plasmid(negative control)
mix with tip
incubate in ice for 20 min
heat shock 42C 30sec
incubate in ice for 3min
SOC 900ul without antibiotic
37C for 1h
100ul from each tubes to LB 5ml+cmp 5 ul
centrifugate 500rpm 2min
carrying of 800ul liquid from each tube
mix the residue with the liquid
sowing LB+cmp
10 min absorption RT
reverse the plates and incubate in 37C
When - 23/06/16
Who's In the lab today: Ben and Dar
glycerol stock protocol:
preparing glycerol:
add 10 ml LB to 10 ml glycerol
preparing the stock:
take 500 ul of starter (with required plasmid transfected)
add 500 ul of the glycerol (from stck)
freeze in -80 C
When - 15/06/16
Who's In the lab today: Ben and Dar
We isolated the plasmids from each cell culture using Geneaid
Again we incubated them with XhoI and XbaI (same protocol as on 14/06)
We ran the samples on a 1% agarose gel with EtBr but using a new loading buffer.
In lanes 6,7,8 we can see good results.
Lane 6 shows the plasmid circular and uncut.
Lane 7 shows the plasmid cut in one location using XbaI.
Lane 8 shows the plasmid circular and uncut (as it does not have a XhoI restriction site).
starters growth for activity testing of LCC:
LB 50 ml+ CP 50 ul.
each tube will get 5 ml.
In addition, LB 5ml without antibiotic for BL21 growth without insertion.
For each tube. we added colony with another varient+2 control tube.
the colony that we are testing:
pACYC LCC WT C6
pACYC LCC C.O C4
pACYC LCC F4 C2
pACYC LCC F7 C3
pACYC LCC R4 C7
pACYC LCC R7 C2
pACYC no insert
BL21 (without antibiotic)
Turbidity check and induction:
For LB 7 ml + CP 7ul we added 70 ul for each starter. ( 37C 200 rpm)
 
PET degradation test:
we cheked if there is any change in PET wight after one day:
 
When - 14/06/16
Who's In the lab today: Ben and Dar
We isolated the plasmids from each cell culture using Geneaid Presto mini plasmid kit.
We Incubated each of the plasmids with XhoI and XbaI restriction enzymes (2 hours incubation in 37 degrees and 20 minutes in 65 degrees).
We ran the results in a 1% agarose gel with EtBr. Both plasmids show on the gel uncut (lanes 2-3). We have no ability to determine their size (as they are circular).
We put new starters for another preparation of plasmid DNA.
When - 13/06/16
Who's In the lab today: Ben and Dar
We incubated starters for the pSEVA224 and pSEVA414 in 30 degrees.
When - 11/06/16
Who's In the lab today: Eyal and inbar b
compentent cells prepretion
 
When - 10/06/16
Who's In the lab today: Eyal
preparation of TRIS-HCL:
for prvpB degredation---> C1V1=C2V2→ 1M*V1= 25mM*9.9ml→ V1=247.5 ul, DDW=9.6525ml
for PET degredation---> C1V1=C2V2→ 1M*V1= 500mM*2ml→ V1=1ml, DDW=1ml
preparation of pNP-B:
pNPB - 35ul -----------> 1 ml of pNPB ---------> 100 mM into
acetonitrite - 965 ul --------> in 200 mM conc. ----->9.9 TRIS 25 nM
we left 8 tubes with LCC variants for incubation at 10:30 AM for the weekend (50 C)
When - 08/06/16
Who's In the lab today: Eyal
 
 
When - 06/06/16
Who's In the lab today: Eyal
we designed an expriment to test differnet carbon surce from our metabolic pathway for putida.
We used 96DW and we orgenized it as seen in the picture:
 
 
this experiment will last for one week we will check the change in the turbidity with plate-reader.
When - 31/05/16
Who's In the lab today: Eyal
we cut De-lorenzo plasmids (414+224), we run thus plasmids on agar gel for analazing if thus plasmids were cut.
When - 28/05/16
Who's In the lab today: Eyal
starters for Putida KT2440.
5 ml LB+5 ul amp, 100ng/ul, 30C
transformetion for pACYC plasmid without insertion(negative control for coplement cells) and with insertion LCC WT C6, LCC CO C4, LCC F4C2, LCCF7C3, LCC R4C7, LCC R7C2 to BL21 bacteria for protein exprection.
8 tubes with compatent BL21 bacteria, 80C,10 min.
1 ul from each plasmid for each tube.
mixed with tip.
keep in ice for 20 min.
keep in ice for 3 min more.
500 ul SOC without antbiotic.
27C 1h
centrifugation 5000 rpm 2min
take off 500ul from the liquid.
mix the residue with the rest liquid.
sowing in LB+CMP
When - 24/05/16
Who's In the lab today: Eyal
seq of LCC variants 3 ul primer in each reaction 300 ng DNA.
(each with Rw\Fw primer).
When - 19/05/16
Who's In the lab today: Roee and Neomi
We made plates with : LB+TET, LB+Strep. we took the starters with De-Lorenzo bacteria to swo on thus plates.
After all the starters and glycerol stocks were found contaminated a new plate of rhodococcus ruber was recieved from alex sivans lab. a new SM medium was made and we have set the tools inorder to grow new starters on sunday may the 22nd.
 
 
When - 13/05/16
Who's In the lab today: Eyal
Constructing a growth-curve for the Putida on Cathecol. The experiment was contaminated. we will have to repete it.
 
 
When - 11/05/16
Who's In the lab today: Eyal
pSEVA starters from 10.5, and sowing on LB+KM 2 ml starters 300 rpm, 2 min, RT, using sup and sowing the output.
When - 03/05/16
Who's In the lab today: Eyal, Roee and Dor
one colony from each plate was sent fot seq(because of the bw concentration, 15 ul from each stock was sent)
2 starters of rhodococcus ruber were made from two different glycerol stocks to check for contamination. each sample was plated on a nutrient agar plate that was divided in to two parts one for each glycerol stock. the starters were put in ramons lab in the incubator shaker 37 degrees and the plate was put in our incubator at 37 degrees as well.
we prepered glycerol stocks of BL21.
When - 30/04/16
Who's In the lab today: Eyal
 
When - 29/04/16
Who's In the lab today: Ben
 
When - 27/04/16
Who's In the lab today: Ben
 
When - 25/04/16
Who's In the lab today: Eyal
 
reaction 1
reaction 2
reaction 3
reaction 4
reaction 5
reaction 6
reaction 7
 
LCC W.T
LCC C.O
LCC F4
LCC R4
LCC F7
LCC R7
control
pACYC
1.8
1.8
1.8
1.8
1.8
1.8
1.8
Insert
0.9
1.3
0.6
0.8
0.9
0.8
-
Buffer
2
2
2
2
2
2
1.4
DDW
14.3
13.9
14.6
14.4
14.3
14.4
15.2
Ligase
1
1
1
1
1
1
1
incubeted at 16C
M9 salts add to 800 ml H2O:
31.64 gr Na2HPO4*2H20
15 gr Kh2PO4
2.5 gr NaCl
5.0 gr NH4Cl
stirr until dissolved
adjust to 100 ml with distilled H2O
make M9:
700 ml DDW
200ml M9 salts
2 ml 1M MgSO4
100ul 1M CaCl2
adjust to 980 ml with DDW
When - 19/04/16
Who's In the lab today: Efrat
Cyclic Voltametry was preformed in oreder to mesure Cathecol’s redox porential conditions:
2% wt Catechol solution in M9. Potential range:
0-0.5 V.E step: 0.005 V
scan rate: 0.01, 0.005, 0.001 V. Room temperature, pH 7
 
0.005 V
0.01 V
0.001 V
E for I max
0.324797
0.334791
0.41574
E for I min
0.119925
0.099937
0.131918
Delta E
0.204872
0.234854
0.283822
E 0
0.222361
0.217364
0.273829
When - 18/04/16
Who's In the lab today: Ben
I loaded the PCR products from yesterday onto an agarose gel (1.5%).
The results:
Lane 1: 1kb plus ladder
Lane 2: pcaG (~660bp)
Lane 3: pcaH (~760bp)
Lane 4: pcaC (~450bp)
Lane 5: pcaD (~840bp)
When - 17/04/16
Who's In the lab today: Ben
I ran a colony-PCR on P. putida with primers for the 4 protocatechuate degredation genes (pcaG/H, pcaC/D)
The reaction mix was:
25μL of ReadyMix Taq PCR reaction mix
22μL of UPW
1.5μL of each primer
A swab from a P. putida colony.
(total reaction volume 50μL)
Cycle parameters were according to Sigma-Aldrich protocol.
ligation of LCC variants into pACYC vector.
Using a ligation calculator in 3:1 ration: we will use about 36 ng of insert for each ligation reaction:
 
reaction 1
reaction 2
reaction 3
reaction 4
reaction 5
reaction 6
 
LCC W,T
LCC C.O
LCC F4
LCC R4
LCC F7
LCC R7
pACYC
1.4
1.4
1.4
1.4
1.4
1.4
Insert
0.9
1.3
0.6
0.8
0.9
0.8
Buffer
2
2
2
2
2
2
DDW
14.7
14.3
15
14.8
14.7
14.8
Ligase
1
1
1
1
1
1
+control: pACYC 1.4, Buffer 2, DDW 1, Ligase 1
-Incubated overnight at 16C
prepered LB agar plates whith CAM resistance.
When - 16/04/16
Who's In the lab today: Eyal
 
When - 13/04/16
Who's In the lab today: Inbar B, Inbar S and Eyal
we took Rodococus rubber from glycerol stock and plated on te LB plate at the biological hood.
When - 12/04/16
Who's In the lab today: Eyal, Inbar B and Inbar S
we have made 2 plates containing only LB.
we have made 8 plates containing LB+CRB.
colonies were present.
two- 5 ml starters from CAM plate (pACYC) for midiprep and miniprep.
starter for midiprep 200 ml
midiprep.
When - 11/04/16
Who's In the lab today: Inbar B, Inbar S and Eyal
we took new pACYC from eyal arbely lab, we transformed E. coli it to mg1655 and seed it on CAM plate.
When - 08/04/16
Who's In the lab today: Tomer
we have made 3 medium with glucose, PET, PolyEthylene and palced them in 37C shaker.
When - 06/04/16
Who's In the lab today: Tomer and Ben
restriction of pACYC and E1B GFP
E1B GFp 1 ul
3.1 buffer 5 ul
XhoI 1ul
DDW 43 ul
pACYC 5 ul
3.1 5 ul
XhaI 1 ul
DDW 39 u
Added 1ul BglII incubeted at 37 C
Glycerol stock made for putida:
500 ul from starters
500 ul glycerol 30 % +LB
making a startersfor Rodococus rubber:
We took 50 ml of SM+Glucose and added to it some of the glycerol stock.
When - 05/04/16
Who's In the lab today: Dar, Tomer and Ben
we prepared new LB medium (1L), gathered and prepared glassware for autoclave.
we set up startes for the Putida - for growth experiments and for preparing new glycerol stocks.
We prepared a 20% (w) Catechol solution using 10gr of Catechol (Pyrocatechol from Sigma-Aldrich) and 50ml of DDW.
We prepared M9 Catechol medium using:
10ml M9 Salts
5μL CaCl2 (1M)
100μL MgSO4
39ml DDW
165μL Catechol 20%
When - 31/03/16
Who's In the lab today: Dar
checking whether the Putida from the experiment last week (Growth curves for Putida and E.Coli on EG) are still alive (and that the OD we’ve measured isn’t the result of dead bacteria).
at both times samples were taken from the liquid solution in the tub and put on plates (LB+amp). at both times growth on the plates was observed - the bacteria were still alive.
When - 28/03/16
Who's In the lab today: Inbar S and Eyal
Try #3 of the pACYC restriction
This time - 1 hour with each enzyme and run on agarose gel 2ul after each cut.
pACYC 2000ng 30ul(66ng/ul)
DDW 58ul
3.1 Buffer 10ul
XhoI 1ul
1 hour 37 deg
20 min 65 deg
1 ul BalII
1hour -> run on agarose gel 2ul.
When - 28/03/16
Who's In the lab today: Inbar S and Eyal
Another restriction reaction for the pACYC plasmid.
We used the pACYC2 product from 25/03(68ng/ul) - Restriction requires 2000ng of plasmid DNA so we used a volume 29.4ul.
Reaction protocol:
DNA 29.4ul
3.1 Buffer (NEB) 10ul
BalII 1ul
XhoI 1ul
DDW 58.8ul
No Bands again.
When - 27/03/16
Who's In the lab today: Dar
Setting up the growth-curve experiment for both the Putida and E.Coli (MG1655) on Ethylene Glycol.
This time the experiment will take a week - 2-3 measurements per day.
When - 25/03/16
Who's In the lab today: Inbar
Mini-prep for pACYC.
 
Concentration (ng/uL)
260/280nm
pACYC1
66
1.81
pACYC2
68
1.78
When - 24/03/16
Who's In the lab today: Tomer and Inbar Bariah
Removing the putida starter from the incubator to the fridge.
DNA extraction from reaction mix- LC-cutinase variants.
total volume 30ul
LC-cutinase variant
concentration(ng/ul)
260\280
lc-cutinase - WT
43
1.63
lc-cutinase - WT CO
28
1.52
FreeRun4
61
1.71
FreeRun7
47
1.66
ResRun4
41
1.69
ResRun7
46
1.64
 
WT, WTCO, F4, F7, R4, R7, 1KbPlus led
 
In addition, starters for pACYC were prepared:
10ml LB+10uL CAM for 2 starters.
When - 23/03/16
Who's In the lab today: Inbar Segal, Dar, Ben, Inbar B, Roee, Dor, Tomer and Eyal
Produced the pACYC plasmid from the starter and cleaned it using mini prep kit.
Concentration: 52ng/ul 260\280: 1.83
another try at constructing a growth-curve for the Putida and E.coli on Ethylene glycol and Glucose.
 
Setting up new starters for the Putida and E.coli (MG1655)
When - 22/03/16
Who's In the lab today: Eyal and Inbar Segal
Cutting LC-cutinase and pACYC vector with restriction enzymes, no bands were visible on electrophoresis gel.
 
When - 18/03/16
PCR extraction of LC-cutinase variants
total volume 30ul
LC-cutinase variant
concentration(ng/ul)
260\280
lc-cutinase - WT CO
134
1.81
FreeRun4
143
1.79
FreeRun7
133
1.80
ResRun4
110
1.78
ResRun7
138
1.81
When - 17/03/16
Who's In the lab today: Inbar Bariah
PCR to amplify LC-cutinase variants: WT CO, FreeRun4, FreeRun7, ResRun4, ResRun7.
WTCO, WTCOX,F4,F4X,F7,F7X,R4,R4X,R7,R7X,1KbPlus led
 
When - 15/03/16
Who's In the lab today: Ben, Dar, Roee and Tomer
We tried constructing a growth curve for E. coli and P. putida on Glucose and Ethylene Glycol.
The Experiment failed (the O.D was declining instead of increasing), probably because the starters we used were kept in the refrigerator for more than 4 days (the bacteria were probably dead).
When - 08/03/16
Who's In the lab today: Eyal and Inbar Segal
we did a transformation to PACYC, using 50microliter competent bacteria.
100 microliters were used for plating.
an activity test was made of DpnI - we did a transformation in order to check that the plasmid indeed dissolved (nothing is supposed to grow).
concentrations of the genes we received from Weizmann institute:
LC-cutinase - WT - 26.55 ng/ul
LC-cutinase - FreeRun4 - 19.15 ng/ul
LC-cutinase - FreeRun7 - 15.575 ng/ul
LC-cutinase - ResRun4 - 20.175 ng/ul
LC-cutinase - ResRun7 - 25.575 ng/ul
When - 07/03/16
Who's In the lab today: Ben, Dar, Eyal and Inbar Segal
Starters did not grow.
12 Petri Dishes of LB+CAM were prepared.
When - 06/03/16
Who's In the lab today: Ben, Tomer and Dar
Preparation of starters for growth curve experiment. (P. putida and E. coli) Glucose and Ethylene Glycol.
When - 25/02/16
Who's In the lab today: Ben
went to the supply room to bring in supplies:
one measuring cup 2L made of glass, and one 1L measuring cup made of plastic.
20ml syringes and 10ml syringes - 10 from each volume. and a few more 2.5 ml because they didn't have 5ml syringes.
around 30 minisart filters.
latex gloves Medium size x5.
A bundle of glass pipettes 10ml (they didn't have 20ml ones) we need to order especially.
big tank for water (needs a wash and then we can use it for ddw or upw).
When - 24/02/16
Who's In the lab today: Neomi, Tomer and Dor
a glycerol stock of rhodoccocus ruber was prepared from the starter made yesterday. I didn't know if the glycerol in our fridge was autoclaved so I borrowed from Ramon's lab, the stock is at (-80) degrees Celsius at Ramon's shelf.
Removed the putida from the incubator after growing on ethylene glycol and glucose and checked the OD (600nm)
Results:
Tube
OD (600nm)
Glucose 20%
0.686
Glucose 15% + Ethylene Glycol 5%
0.420
Glucose 10% + EG 10%
0.859
Glucose 5% + EG 15%
0.112
Ethylene Glycol 20%
0.465
because the measurements were inconsistent I made several measurements. especially tubes 1 and 5:
Tube
OD (600nm)
Glucose 20%
first re-measurement 0.459 second re-measurement 0.611
Glucose 15% + Ethylene Glycol 5%
first re-measurement 0.019 second re-measurement 0.032
When - 23/02/16
Who's In the lab today: Ben and Roee
a starter of Rhodococcus ruber was made. it included 50ml SM(medium) + glucose + microelements. the work was done in a biological hood afterwards it was placed in a shaker at 37 degrees Celsius overnight.
taking the putida out of the shaker after growing it on glucose and ethylene glycol and checking OD (600nm).
Results:
Tube
OD (600nm)
Glucose 20%
0.560
Glucose 15% + Ethylene Glycol 5%
0.671
Glucose 10% + EG 10%
0.465
Glucose 5% + EG 15%
0
Ethylene Glycol 20%
0.09
When - 21/02/16
Who's In the lab today: Roee, Dar, Dor and Neomi
how to make competent bacteria?
all the process need to occur on ice, in a centrifuge set to 4degree Celsius, and the glycerol need to be cold.
first of all, we grow the bacteria over night with antibiotics -> in 50 ml falcon tube.
centrifuge - maximum speed for 15 min and remove the supernatant.
we filled the falcon tube again with glycerol 10%. centrifuge once again for 15 min at max speed and again remove the supernatant.
repeat step 2.
we combine both tubes (if we have more than one, if not we wash it again with glycerol as we did in 2+3) after we fluidize the bacteria in glycerol we complete the volume with 10% glycerol, we centrifuge and remove the supernatant.
fluidize the sediment with 2 ml glycerol and divide into tubes 50 microliter in each one and freeze with liquid nitrogen.
a medium for growing Rhodococcus Ruber was made in the solution are missing microelements that we are supposed to receive from Valentina and there is no carbon source. the solution is in the fridge door with a sticker “rhodococus ruber solution”.
 
there was no CaCl2 2H2O si we used 680microliter of CaCl2 1M from the fridge.
When - 22/02/16
Who's In the lab today: Roee, Dar, Dor, Tomer, Inbar Segal and Eyal
competent bacteria were made from the putida KT2440 strain. the bacteria were divided into 25 Eppendorf's with 50microliter each (the volume needed for each transformation) the Eppendorf's are in the freezer (-80 degrees) of Ramons lab in a red box the with the inscription “Ashalim”.
starting the experiment of growing the Pseudomonas putida on ethylene glycol:
we made the minimal medium where each one has different rates of carbon sources.
we inserted to each medium one colony of Pseudomonas Putida.
we made a blank of 1ml to measure OD.
we put the bacteria in a Shaker set to 37 degrees Celsius in Naama's lab.
We did a PCR reaction for the LC-Cutinase + leader Sequence Gene. in order to take it from the iGEM plasmid and insert it into a PACYC vector. we ran the PCR products on an agarose gel using electrophoresis and took a picture. we got one big band in the size we wanted 1KBP~ and a second band, weaker one in the size of the entire plasmid.
When - 15/02/16
Who's In the lab today: Inbar Bariah
main action - growing two strains of Pseudomonas putida (EM383, KT2440) on SOC :
Inserting a Q-tip that touched the glycerol stock of the relevant culture into 3ml of SOC with ampicillin (near fire) and grow overnight. -results- the bacteria grew and the medium became very murky. the bacteria were afterwards grown on petri dishes with LB + amp from every strain we inserted 20 microliters and we also made plates where we isolated via striking.
colonies from every strain (KT2440, EM383) were moved into a liquid LB medium for incubation overnight in 37 degrees Celsius.
When - 14/02/16
Who's In the lab today: Inbar Bariah, Neomi and Tomer
main action - making M9 medium for growing Pseudomonas Putida
Instead of glucose as a carbon source we planned an experiment where the ratios between glucose and ethylene glycol vary in order to check if the bacteria can use ethylene glycol as a carbon source. if so, we won't need to insert the two plasmids we took from E. coli.
planning the experiment - we divided the medium into 5 different test tubes where the carbon source will be different in each one:
100% glucose.
75% glucose, 25% ethylene glycol.
50% glucose, 50% ethylene glycol.
75% ethylene glycol, 25% glucose.
100% ethylene glycol.
making the medium:
with the assistance of Idit we made MgSo4 and a Salt sterile solutions using autoclave.
we used Sterile CaCl that already gone through autoclave and also DDW that we already had in the lab.
we made 20% glucose and filtered it.
we filtered ethylene glycol and made a 20% ethylene glycol solution.
concentrations:
main action - making ampicillin:
Weighing 1g ampicillin.
Adding 10ml DDW.
Filtering with syringe 0.24 wide.
Dividing it into 20 Eppendorf's (500microliter each).
A protocol for making petri dishes with LB medium:
 
When - 02/02/16
Who's In the lab today: Roee, Inbar Bariah and Tomer
main action - re-sending the two plasmids that got low concentrations to sequencing again.
After checking once again the sequences in blast of the two plasmids that got low percentage and after consulting with Dr.Birnbaum and Naama we understood that the way we looked at the percentage wasn't correct and that the sequencing was actually correct and we had the sequence we wanted on the plasmid. (before we knew about the sequencing being correct we made Chloramphenicol in working dosage in order to grow the bacteria that carried the plasmid on the medium we wanted).
instructions for sending DNA to sequencing:
Plasmid Dosage need to be between 75-100 ng/ul.
volume needed: ten microliters per plasmid.
Primers - 2 microliters per sequenced plasmid (minimum volume 5 ul).
When - 18/01/16
Who's In the lab today: Inbar Segal
Preparation of competent bacteria: E. coli MG1655
Add all of the starter to 500 ml of LB
Use 1 ml of LB as blank, measure OD at time zero at wavelength of 600 nm. (time zero was 0.064 nm)
Measure OD every hour until the bacteria gets to the logarithmic phase (about 3 hours).
The Incubater, Shaker, Spectofotometer were used in Ofer yifrach lab (3 floor).
At 11:20 OD was 0.54 nm, we took it out of the shaker and put it in ice.
We checked if the bacteria can grow on agar with kan and amp - we did not see any colony.
When - 17/01/17
Who's In the lab today: Dor, Inbar Barih and Neomi
How to make a solution for competent bacteria:
 
When - 16/01/16
Who's In the lab today: Dor, Roee and Tomer
Checking the results of the sequencing:
 
 
When - 14/01/16
Who's In the lab today: Dor, Roee, Tomer and Naama
How to check the results of sequencing:
Open the file with the graphs displaying the output.
Go to edit->copy seq->FASTA format.
Go to Blast and paste what we copied in the first window, if we copied first the sequence of the foreword primer after we will do for the reverse primer and vice-versa.
Open the file with the graphs displaying the output, but this time for the reverse primer if we opened its first for the foreword and vice-versa.
Go to edit->copy seq->FASTA format.
Go to Blast press enter, and paste this part in the same window one row below.
Take the reference and put in a different box/one row below.
Press Blast
Pay attention:
In the top of the page in Blast you can check if R or F fits the reference.
It's better to check if R and F match (we expect it to be similar not the same).
Notice the values of query-how similar are the samples, ident-percentage of identical bases.
Look where most of the different bases are located (hopefully in the end or the beginning).
Small letters instead of big ones (atg and not ATG) marks that this sequence repeats itself.
If you to write comments, write according to the locations of the reference.
Planning a Vector:
We looked at the map of the vector that was sent (for the Putida) and tried to see which restriction enzymes are relevant.
we looked at the nebcutter website to check for each of the fragments which restriction enzymes are relevant (each fragment separately, according to the reference).
press custom digest for the list of enzymes, check that this enzyme don't cut inside of the fragments.
choose restriction enzymes.
Plan primers with tails that will match the enzymes.
pcr and ligation.
transformation.
When - 04/01/16
Who's In the lab today: Dor and Roee
We prepared working stock primers to 10μM, and sent 11 plasmids to sequencing:
LLK - lc cutinase + led seq + his tag
LL - lc cutinase + led seq
When - 31/12/15
Who's In the lab today: Liran and Tomer
Measuring DNA concentration of the plasmids (the numbers match the ones from above):
Plasmid
1
2
3
4
5
6
7
8
9
10
11
concentration (ng/ul)
222.96
188.87
159.67
125.30
98.86
293.56
153.54
189.91
155.54
157.45
166.29
When - 30/12/15
Who's In the lab today: Ben, Liran, Dar, Tomer and Dor
PCR for two genes (aldA, flco) of the E.coli strain MG1655:
Prepration of a DNA gel
Well
1
3
4
6
7
Sample
ladder
flco
flco control (no template)
aldA control (no template)
aldA
Extraction of plasmids (using the yellow/orange kit):
Number
Plasmid
1
tph A1
2
tctA A1
3
tph A3
4
LC cutinase
5
LC cut + lead seq + his tag
6
tph A2
Number
Plasmid
7
tphB
8
tctA A2
9
rctB B1
10
tph C
11
LC cut + lead seq
When - 29/12/15
Who's In the lab today: Inbar Barih, Inbar Segal, Eyal, Liran and Ben
Making starters for bacteria after Transformation:
Add 5 ml of LB to a 50ml falcon.
Add 5 ul of the matching antibiotic (the ratio is 1:1000).
With a toothpick stab one colony spread it on another Agar plate (in case the starter wont work we will be able to use that colony again) and then throw it inside of the 50 ml flacon.
Address:
Ben-Gurion University of the Negev
Ben Gurion 1, Beer Sheva 8410501, Israel
Mail: igembgu2016@gmail.com
Connect With Us!
