<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BGU_ISRAEL_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BGU ISRAEL/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>PlastiCure</TITLE></P><NAV class="navbar navbar-inverse navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="mainmenu nav navbar-nav"><LI class="dropdown" id="HomeB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL">Home</A></LI><LI class="dropdown" id="projectB" style="background-color: #CB99C9;"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Achievements">Achievements</A></LI></UL><LI class="dropdown" id="teamB"><A href="https://2016.igem.org/Team:BGU_ISRAEL/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/TeamMembers">Members</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/MeetUniversity">Meet Our University</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/MeetTown">Meet Our Town</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Collaborations">Collaborations</A></LI><LI class="dropdown" id="partsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Parts#Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/ImprovedParts">Improved Parts</A></LI><LI class="dropdown" id="humanB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Integrated_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/PlasticArt">Plastic Art</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/EthicsAndSafety">Ethics &amp; Safety</A></LI><LI class="dropdown" id="attributionsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Attributions">Attributions</A></LI><LI class="dropdown" id="awardsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Awards">Awards</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Measurements">Measurements</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Proof">Proof Of Concept</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/HP/Silver">HP - Silver</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/HP/Gold">HP - Gold</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Engagement">Engagements</A></LI></DIV></DIV></NAV><DIV class="container-fluid"><DIV class="row"><DIV class="projectColor"><P>Description</P></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row whiteback"><DIV class="col-lg-8"><DIV class="descText"><P>
                            Many date the invention of plastic by Alexander Parkes (Parkesine) as far as 1856, back then plastic was made almost entirely of cellulose,
                            a natural substance found in plants. With the advancements made in the field of polymer chemistry during the 20th century, 
                            many new polymers were introduced into industry and are known to us today as Polystyrene, Polyethylene and Polyethylene terephthalate (PET). 
                            Plastic immediately gained popularity, being cheap, durable and easily molded into different solid shapes, and can be even used as an ink in 3D printers. 
                            However, one of its best features was found to be one of its greatest drawbacks, the durability of plastic makes it virtually non-degradable. 
                            An average bottle of mineral water takes roughly half a millennium to decompose, thus, leading to a global accumulation of plastic waste. 
                            Many ideas were considered in dealing with plastic waste such as burning plastic or burying it, but these solutions are considered damaging to the environment due to plastics toxicity. 
                            Since the introduction of plastics, some microbial communities or species have evolved to successfully degrade plastics, however, 
                            from an evolutionary point of view, probably due to the relatively short period of exposure to plastics, 
                            they are yet to be efficient in plastic biodegradation.
                        </P><P>
                            Our goal as the Ben-Gurion University IGEM team of 2016 is to overcome this evolutionary hurdle by devising several approaches using synthetic biology
                            tools for efficient plastic biodegradation. In addition, we plan to utilize the high energy stored in PET molecules, for electricity production.
                            In order to achieve that, the following research scheme has been devised: 
                        </P><DIV class="descTitle"><U>A Protein Engineering Approach</U></DIV><P> We have decided to engineer the LC-Cutinase enzyme. LC-Cutinase is an enzyme discovered from an unknown organism in leaf-branch compost, and has been found to be one of the most efficient enzymes in breaking down PET polymers with relatively high crystallinity into degradable products, the monomers ethylene glycol and terephthalic acid. Based on the LC-Cutinase structure that was solved in 2014 (Sulaiman et al 2014), we have chosen to use a rational mutagenesis approach for its improvement. 
                            Using this approach, we made various mutations using an algorithm developed by Dr. Sarel Fleishman of the Weizmann Institute of Science (Goldenzweig et al. 2016). The algorithm compares the sequence of the original protein with that of other homologous proteins and then chooses a set of mutations. 
                            The algorithm then calculates the differences in free energy (ΔΔG) of each mutated variant compared to the free energy of the W.T. protein and selects for a library of variants that are thermodynamically stable. 
                            Using this algorithm, we received 4 different variants that we further tested for improvements in activity, stability and expression levels. In addition, the pelB leader sequence was fused to the enzyme in its N terminus and was expressed and secreted to the growth media of <I>E. coli</I>.
                        </P><DIV class="descTitle"><U>Genetic Engineering of Metabolic Pathways</U></DIV><P>
                            Next, we wanted to fully degrade the resulting monomers to CO2, this way no toxic molecules will remain as products of the degradation process. We decided to achieve this by genetic engineering of metabolic pathways of the soil bacterium <I>Pseudomonas putida</I> (<I>P. putida</I>). We plan to insert a degradation pathway for terephthalate into <I>P. putida</I> using genetic engineering, while the other monomer of PET, ethylene glycol, is utilized by <I>E. coli</I> that secretes our improved LC-Cutinase protein. The two bacteria will metabolize the two PET degradation products - leading to the conversion of PET to CO<SUB>2</SUB>. The terephthalate degradation pathway, derived from a strain of <I>Commamonas testosteroni</I> , terminates in protocatechuate, a toxic molecule for most bacteria, however, <I>P. putida</I>, is able to utilize it as a carbon source for its growth (Jimenez et al 2002). The ethylene glycol degradation pathway, present in our chosen <I>E. coli</I> strain, BL-21, supplies a carbon source for its growth while degrading PET to its respective monomers with LC-Cutinase. 
                            This way, we hope to achieve a full degradation of the two PET products and with it to drive the PET biocatalysis reaction by LC-Cutinase forward. 
                            We have chosen to work with <I>P. putida</I>, which is a gram-negative bacterium, for its diverse metabolism, including the ability to degrade organic solvents, especially protocatechuate, a toxic substance for most bacteria, its similarity to <I>E. coli</I> in most laboratory protocols and its electrochemical properties which allow it to be used in our fuel cell. 
                            Our goal is to genetically engineer <I>P. putida</I> so it will contain two plasmids which will encode for two essential components: a membrane transporter that will carry the terephthalate molecule into the cell and the necessary genes for its degradation. 
                            In order to achieve symbiosis between our two chosen bacterial species we have separated them using a dialysis membrane so that the <I>E. coli</I>, secreting the LC-Cutinase and utilizing ethylene glycol is enclosed and separated from the <I>P. putida</I> which is utilizing the terephthalic acid that diffuses out of the dialysis bag. Their mutual dependence depends on the fact that terephthalate will be the sole carbon source for <I>P. putida</I>'s growth, and cannot be generated in the absence of <I>E. coli</I> secreting LC-cutinase, while <I>E. coli</I> will not survive in elevated levels of terephthalate, that has to be degraded by <I>P. putida</I>, thus engineering a symbiotic dependence between the two bacterial species. 
                        </P><DIV class="descTitle"><U>Bioelectrochemical PET Degradation System</U></DIV><P>
                            Manufacturing PET from fossil fuels is an energy consuming process. 
                            In addition, current solutions for the disposal of PET, such as recycling and burying are also energy consuming as they require means such as transportation, 
                            sorting and initial processing. In order to offer a better alternative to the existing solutions we decided to explore options which would allow us to maintain a positive energy balance. 
                            Knowing our bacteria requires certain conditions to maintain viability, such as temperature, 
                            we knew we needed to supply them in ways that will maintain a positive energy balance. 
                            Since our bacterium of choice - <I>P. putida</I> is considered an exoelectrogen (i.e. bacteria that is able to respire its excess electrons through electrodes), 
                            and since PET is a polymer that contains a large amount of energy in its carbon-carbon bonds, 
                            excess energy released by our engineered <I>P. putida</I> from terephthalate degradation will be harnessed and utilized in a microbial fuel cell (MFC), 
                            leading to an efficient and energy producing, rather than consuming, degradation of PET. 
                            This energy can be utilized in a future device for the maintenance of growth conditions or the pretreatment of PET to render it easily degraded by the engineered bacteria.
                        </P><OL><LI>
                                Sulaiman, S., You, D. J., Kanaya, E., Koga, Y., &amp; Kanaya, S. (2014). Crystal structure and thermodynamic and kinetic stability of metagenome-derived LC-cutinase. Biochemistry, 53(11), 1858-1869.
                            </LI><LI>
                                Goldenzweig, A., Goldsmith, M., Hill, S. E., Gertman, O., Laurino, P., Ashani, Y., ... &amp; Lieberman, R. L. (2016). Automated Structure-and Sequence-Based Design of Proteins for High Bacterial Expression and Stability.ֲ Molecular Cell,63(2), 337-346.ג€
                            </LI><LI>
                                Jimenez, J. I., Minambres, B., Garcia, J. L., &amp; Diaz, E. (2002). Genomic analysis of the aromatic catabolic pathways from <I>Pseudomonas putida</I> KT2440. Environmental microbiology, 4(12), 824-841.
                            </LI></OL></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row footerColor"><DIV class="col-lg-4 text-center"><H4>Address:</H4><P>
                        Ben-Gurion University of the Negev
                        Ben Gurion 1, Beer Sheva 8410501, Israel
                    </P><H5><B><U>Mail:</U></B> igembgu2016@gmail.com
                    </H5></DIV><DIV class="col-lg-4 text-center"><H4>Connect With Us!</H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>