<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BGU_ISRAEL_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BGU ISRAEL/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>PlastiCure</TITLE></P><NAV class="navbar navbar-inverse navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="mainmenu nav navbar-nav"><LI class="dropdown" id="HomeB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL">Home</A></LI><LI class="dropdown" id="projectB" style="background-color: #CB99C9;"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Achievements">Achievements</A></LI></UL><LI class="dropdown" id="teamB"><A href="https://2016.igem.org/Team:BGU_ISRAEL/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/TeamMembers">Members</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/MeetUniversity">Meet Our University</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/MeetTown">Meet Our Town</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Collaborations">Collaborations</A></LI><LI class="dropdown" id="partsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Parts#Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/ImprovedParts">Improved Parts</A></LI><LI class="dropdown" id="humanB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Integrated_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/PlasticArt">Plastic Art</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/EthicsAndSafety">Ethics &amp; Safety</A></LI><LI class="dropdown" id="attributionsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Attributions">Attributions</A></LI><LI class="dropdown" id="awardsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Awards">Awards</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Measurements">Measurements</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Proof">Proof Of Concept</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/HP/Silver">HP - Silver</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/HP/Gold">HP - Gold</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Engagement">Engagements</A></LI></DIV></DIV></NAV><DIV class="container-fluid"><DIV class="row"><DIV class="projectColor"><P>Experiments</P></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="experimentText"><H3><U><I>Rhodococcus ruber</I> plastic degradation assays:</U><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results#rRuber" class="btn btn-primary infobtnExpToRes" role="button">Results</A></H3><P>
                            Since plastic is a new synthetic polymer introduced to the environment by mankind only recently, 
                            not many organisms have adapted into using plastic as a sole carbon source. Furthermore, 
                            organisms that are able to utilize plastic still consume it in an inefficient manner, thus slowing down their metabolism significantly. Our goal was to use an organism which has adapted into degrading plastic,
                            although slowly, and to examine its mode of action. We choose to work with <I>Rhodococcus ruber</I>
                            C208 which was found to be able to utilize polyethylene (PE) by Prof. Alex Sivan from BGU (Orr et al 2004).
                        </P><P>
                            Because our project focuses on the biodegradation of PET, and not PE, 
                            we first wanted to test Rhodococcus ruber’s ability to utilize PET as a sole carbon source, 
                            an ability that was thus far unreported. We designed an experiment to test the ability of this bacterium to degrade different carbon sources and characterize its growth rate utilizing each carbon source.
                        </P><DIV class="row"><DIV class="col-lg-8"><FIGURE><FIGCAPTION class="expFigCaption">
                                        *No additional carbon source added.
                                    </FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="experimentText"><H3><U>pNP-Butyrate degradation assay:</U><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results#pNPButy" class="btn btn-primary infobtnExpToRes" role="button">Results</A></H3><P>
                            The pNP-B degradation assay is a fast and simple method to estimate LCC’s (LC-Cutinase) variants’ degradation activity. 
                            Under the tested conditions, LCC has a high pNP-B substrate specificity. During the degradation of the pNP-B by LCC, pNP is released, 
                            the quantity of which can be simply and accurately determined via measuring absorbance at 405nm. This method, 
                            allows a quick scan of many variants’ degradation activity under different conditions, and the determination of said variants’ kinetic constants. 
                            The experiments were conducted in room temperature, since the enzyme is designated for use under similar conditions and because the enzyme was reported to function well in room temperature.
                        </P><P>
                            Due to the similarity between the bonds degraded by LCC in PET and pNP-B, LCC’s effectiveness in degrading PET can be assessed by its efficiency using pNP-B degradation.
                            With that in mind, since PET has a more complex chemical structure (it is a polymer),
                            this method alone is not accurate enough to determine the enzyme’s efficiency with PET as a substrate. 
                            For this reason, pNP-B degradation assay was used as a preliminary scan of the different variants and as method to determine their kinetic constants, alongside PET degradation assays.
                            Using various assays and substrates to examine the variants gave a deep comprehensive understanding regarding the variants and their degradation activity.
                        </P></DIV></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="experimentText"><H3><U>PET degradation characterization using electron microscopy:</U><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results#electronM" class="btn btn-primary infobtnExpToRes" role="button">Results</A></H3><P>
                            After our experiments with R. ruber and characterizing how a naturaly evolved plastic degrading bacterium utilizes PET, 
                            we turned to examining our bacteria with the same methods.
                        </P><P>
                            We incubated 3 PET pellets in a 500ml liquid LB medium with codon-optimzed LC-Cutinase expressing bacteria, 
                            to create a large enzyme to substrate ratio.
                        </P><P>
                            To ensure that the results are in fact from the action of the LC-Cutinase enzyme we set two controls:
                        <OL><LI>
                                LB liquid medium with 3 PET pellets incubated at the same conditions, with no bacteria added.
                            </LI><LI>
                                LB liquid medium with 3 PET pellets incubated at the same conditions, with bacteria transformed with the pACYC vector without the Cutinase gene.
                            </LI></OL></P><P>
                            The same test was performed using the W.T. LC-Cutinase and F4 variant with PET and PE but the results were inconclusive. 
                            We planned on performing the experiment again, but were unable to a lack of time and resources. 
                        </P><DIV class="row"><DIV class="col-lg-8"><FIGURE><FIGCAPTION class="expFigCaption">
                                        *No additional carbon source added.
                                    </FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="experimentText"><H3><U><I>P. putida</I> KT2440 antibiotic resistance tests:</U><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results#putiAnti" class="btn btn-primary infobtnExpToRes" role="button">Results</A></H3><P><I>P. putida</I> KT2440 is a versatile soil bacterium with diverse metabolism, 
                            that is capable of degrading many toxic substances. 
                            Additionally, it has natural resistance to the antibiotics Ampicillin and Chloramphenicol. 
                            In a preliminary phase, before transforming it with the pSEVA vectors, we decided to test the bacterium’s response to various antibiotics that would later be used as selection markers.
                        </P><P>
                            We used bacteria acquired from Dr. Halim Jubran from the laboratory of Prof. D. Taufik at the Weizmann Institute. 
                            The growth medium used was a liquid LB medium with Ampicillin in a concentration of 100μg/ml. 
                            After 24 hours of growth at 30<SUP>o</SUP>C, the medium was diluted in a 1:100 ratio and transferred to Snap-Cap tubes with LB liquid medium with various antibiotics at different concentrations.
                        </P><P>
                            The antibiotics that we tested were - Ampicillin, Chloramphenicol, Kanamycin, Streptomycin and Spectinomycin.
                        </P></DIV></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="experimentText"><H3><U><I>P. putida</I> carbon source utilization experiments:</U><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results#putiCarbon" class="btn btn-primary infobtnExpToRes" role="button">Results</A></H3><P><I>P. putida</I> has a crucial role in our project’s designs. 
                            Our symbiotic approach relies on <I>E. coli</I> utilizing one of PET’s monomer, 
                            ethylene glycol, while <I>P. putida</I>, utilizes the other – terephthalate. 
                            For <I>P. putida</I> to utilize TPA (terephthalate) it need to be engineered and inserted with a degradation pathway that terminates in protocatechuate (PCA).
                        </P><P><I>P. putida</I> is documented in previously published articles as able to utilize PCA as a sole carbon source (Jiménez et al 2002), 
                            but we wanted to verify the optimal concentrations.
                        </P><P>
                            Moreover, as <I>P. putida</I> is known to degrade various toxic molecules we decided to test if it can naturally degrade TPA, which possibly eliminates the need for modifying its metabolic pathways by genetic engineering.
                        </P><P>
                            We used liquid M9 minimal media with either TPA or PCA as carbon sources.
                        </P><P>
                            The concentrations of PCA tested were 5 and 10mM and the concentrations of TPA tested were 5, 20 and 30mM.
                        </P><P>
                            for detailed instruction on media preparation see our <A href="https://2016.igem.org/Team:BGU_ISRAEL/Protocols">Protocols</A>)
                        </P><P>References</P><OL class="referenceExpe"><LI>
                                Jiménez, J. I., Miñambres, B., García, J. L., &amp; Díaz, E. (2002). Genomic analysis of the aromatic catabolic pathways from <I>Pseudomonas putida</I> KT2440. Environmental microbiology, 4(12), 824-841.
                            </LI></OL></DIV></DIV></DIV></DIV><DIV class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="experimentText"><H3><U>Ethylene Glycol utilization by <I>E. coli</I>:</U><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results#ecoliEG" class="btn btn-primary infobtnExpToRes" role="button">Results</A></H3><P>
                            As part of our symbiotic approach, <I>E. coli</I> is required to utilize ethylene glycol, a monomer of PET, 
                            as a carbon source. Some strains of <I>E. coli</I> are documented as able to utilize ethylene glycol (Boronat et al 1983) as a carbon source, 
                            but we did not find articles describing the BL21 as one of them. 
                        </P><P>
                            Additionally, we wanted to determine the optimal concentration of EG (ethylene glycol) for growth using the BL21 strain.
                        </P><P>
                            The experiment was performed using a 96DW plate with 200ul of liquid M9 minimal media and EG as a sole carbon source. The concentrations of EG tested were 5,30,40mM and each concentration was tested 4 times. The test was performed for 100 hours.
                            (For detailed instructions for preparation of the media see <A href="’https://2016.igem.org/Team:BGU_ISRAEL/Protocols’">Protocols</A>)
                        </P><P>References</P><OL class="referenceExpe"><LI>
                            Boronat, A., Caballero, E., &amp; Aguilar, J. (1983). Experimental evolution of a metabolic pathway for ethylene glycol utilization by <I>Escherichia coli</I>. Journal of bacteriology, 153(1), 134-139
                            </LI></OL></DIV></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="experimentText"><H3><U>Examining the utilization of PET by <I>E. coli</I> expressing LC-Cutinase:</U><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results#utilPET" class="btn btn-primary infobtnExpToRes" role="button">Results</A></H3><P>
                            After testing the LC-Cutinase (wild type, codon optimized and variant genes) ability to degrade pNP-Butyrate, 
                            we also wanted to examine the possibility that by expressing the enzyme, 
                            our <I>E. coli</I> strain (BL21) has gained the ability to utilize PET, by degrading it to its monomer, ethylene glycol and utilizing it as a sole carbon source.
                        </P><P>
                            During our review of existing literature regarding plastic degradation and the degradation of its monomers, 
                            we learned that most <I>E. coli</I> strains have the ability to consume ethylene glycol, one of the PET degradation products 
                            (Boronat, A. et. al, 1983). We hypothesized that if <I>E. coli</I> can express and secrete LC-Cutinase, 
                            the enzyme will degrade PET to TPA (terephthalic acid) and EG (ethylene glycol), 
                            and the bacteria will be able to consume EG and grow without any other carbon source.
                        </P><P>
                            We needed to develop an assay that will test this hypothesis. Our goal was to build a relatively simple experiment that could confirm the fact that LC-Cutinase is expressed and secreted by the <I>E. coli</I>, 
                            and that it successfully degrades PET to its respective monomers. We have managed to create a new assay using M9 medium with soft agar plates containing shredded PET. 
                        </P><DIV class="row"><DIV class="col-lg-10"><FIGURE><FIGCAPTION>
                                        *No additional carbon source added.
                                    </FIGCAPTION></FIGURE></DIV></DIV><P>
                            Bacteria can only grow inside the soft agar if:
                        </P><OL><LI>
                                LC-Cutinase is expressed 
                            </LI><LI>
                                LC-Cutinase is secreted using the pelB leader sequence
                            </LI><LI>
                                LC-Cutinase can degrade the PET to EG and TPA
                            </LI></OL><P>
                            If one of the conditions in this process is not met, there should not be any growth of <I>E. coli</I> on PET. 
                            This assay gave us a simple way to confirm that the expression, secretion and degradation of LC-Cutinase occurs as hypothesized. 
                        </P><P>References</P><OL class="referenceExpe"><LI>
                            Boronat, A., Caballero, E., &amp; Aguilar, J. (1983). Experimental evolution of a metabolic pathway for ethylene glycol utilization by<I>Escherichia coli</I>. Journal of bacteriology, 153(1), 134-139.‏
                            </LI></OL></DIV></DIV></DIV></DIV><DIV class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="experimentText"><H3><U>Terephthalic acid detection assay:</U></H3><P>
LC-Cutinase degrades PET into its two monomers, ethylene glycol and terephthalic acid (TPA).
We wanted to find a method to detect TPA and to measure the kinetics of the LC-Cutinase variants using different levels of TPA. To do that we conducted an experiment using previously published assay (Barreto et al 1994), that is used to detect the presence of radicals using TPA, we reversed its usage to find the concentration of TPA. We prepared different known concentrations of TPA and irradiated them with UV for 1 hour and by measuring emissions we were able to make a calibration curve for TPA concentrations. We then took samples from a M9 medium that contains shredded PET serving as a sole carbon source and <I>Escherichia coli</I> that secretes LC-Cutinase into a 96 well plate.
</P><P>
We then irradiated the plate with UV for 1 hour, inserted the plate into a plate reader and by using wavelength of 312 nm to excite and wavelentgh of 426 nm to measure the emission we had a result we could 'insert' to our calibration curve and get a concentration.
                        </P><DIV class="row"><DIV class="col-lg-10"><FIGURE><FIGCAPTION class="expFigCaption">                                      *No additional carbon source added.
                                    </FIGCAPTION></FIGURE></DIV></DIV><P>References</P><OL class="referenceExpe"><LI>
                            Barreto, J. C., Smith, G. S., Strobel, N. H., McQuillin, P. A., &amp; Miller, T. A. (1994). Terephthalic acid: a dosimeter for the detection of hydroxyl radicals in vitro. Life sciences, 56(4), PL89-PL96.
                            </LI></OL></DIV></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="experimentText"><H3><U>Fuel cell construction and testing:</U><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results#fuelCell" class="btn btn-primary infobtnExpToRes" role="button">Results</A></H3><P>
                            Designing our fuel cell, we looked for electron transfer mediators that were previously used in MFCs with <I>P. putida</I> KT 2440, 
                            under the assumption that it will improve current generation and power densities (Logan et al 2006). We decided to use Protocatechuic acid (PCA), 
                            an intermediate in the terephthalate metabolic pathway that was planned to be inserted into <I>P. putida</I> KT 2440. Furthermore, 
                            it was shown that <I>P. putida</I> can utilize PCA, ensuring its ability to permeate through the bacterium's outer membrane (Jimenez et al 2002). 
                            We measured its potential via cyclic voltammetry (CV) to assess its compatibility as an electron mediator.
                        </P><P>
                            A M9 medium with 1% Protocatechuic acid (20% w/v) as a carbon source was prepared. 
                            Potential was measured by cyclic voltammetry. The reference electrode used was an Ag/AgCl electrode, 
                            while the working electrode and the counter electrode were 1 mm graphite rods.
                        </P><P>
                            After verifying our assumption regarding the electron mediator, we constructed an anodic half-cell:
                        </P><P>
                            First, we used graphite 2cmx2cm plates as anodes with <I>P. putida</I> KT 2440 that was induced to form a biofilm onto it a week prior to the experiment. 
                            The anode was immersed in 10 mL of LB medium in a 50 mL glass beaker. 
                            The media was inoculated with 50 µL of a day old starter of <I>P. putida</I> KT 2440. The sample was incubated in 30<SUP>o</SUP>C and stired at 300 RPM. 
                            The growth medium had 5 mL of LB replaced with a fresh medium every two days.
                        </P><P>
                            After achieving a substantial biofilm growth, an anodic half-cell was assembled. 
                            The cell was composed of a 100 mL glass beaker, Ag/AgCl reference electrode, a counter electrode, made of a 0.5 mm graphite rod, 
                            a 2cmx2cm graphite plate anode with the biofilm and a 3cmx2.5cm graphite disc cathode. 
                            The electrodes were immersed in 45mL of M9 medium with 0.01% Protocatechuic acid and phosphate buffer at pH 7.2. 
                            The cells were kept at room temperature on a magnetic plate with a stirrer. 
                            The cathode was biased to a potential of 700 mV vs. Ag/AgCl using a potentiostat. 
                        </P><P>
                            Voltage was measured under different resistances ranging from 0-10M Ω using a resistance decade and a voltmeter for a period of three days to follow the changes in power output and current.  
                        </P><P>References</P><OL><LI>
                                Logan, B. E., Hamelers, B., Rozendal, R., Schröder, U., Keller, J., Freguia, S., ... &amp; Rabaey, K. (2006). Microbial fuel cells: methodology and technology.           Environmental science &amp; technology, 40(17), 5181-5192.
                            </LI><LI>
                                Jimenez, J. I., Minambres, B., Garcia, J. L., &amp; Diaz, E. (2002). Genomic analysis of the aromatic catabolic pathways from <I>Pseudomonas putida</I> KT2440. Environmental microbiology, 4(12), 824-841.
                            </LI></OL></DIV></DIV></DIV></DIV><DIV class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="experimentText"><H3><U>Dialysis bag growth experiment:</U><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results#ecoligrowdial" class="btn btn-primary infobtnExpToRes" role="button">Results</A></H3><P>
                            PET degradation by the LC-Cutinase enzyme can only be accomplished if the enzyme is secreted from the bacterial cell.
                        </P><P>
                            Our original designs included transforming the LC-Cutinase gene to our <I>P. putida</I> bacterium for its secretion and utilization of PET. Due to lack of record on the usage of the pelB leader sequence in this bacterium and lack of time for testing it ourselves, we have altered the design.
                        </P><P><I>E. coli</I> strain BL-21 is known as an efficient chassis for protein secretion, and has been previously used with the LC-Cutinase enzyme and the pelB leader sequence. Therefore, we have chosen it for LC-Cutinase expression and secretion. 
                        </P><P>
                            Our final design includes <I>E. coli</I> secreting LC-Cutinase and utilizing ethylene glycol as a carbon source, while <I>P. putida</I> utilizes terephthalate. In order to achieve this symbiosis, we designed a system that uses a dialysis membrane to separate these two bacteria.
                        </P><P>
                            Two experiments were designed for assessing the bacterial growth inside the dialysis bag and the symbiosis:
                        </P><P><I>E. coli</I> BL-21 transformed with the LC-Cutinase gene in a pACYC vector was grown inside a dialysis bag suspended in a M9 liquid medium with PET as a sole carbon source and induced to secret LC-Cutinase using IPTG. (see protocols) O.D measurements were taken regularly. In addition to measurements of bacterial growth, kinetics of the PET degradation was tested by measuring TPA concentrations in the medium. (using the TPA detection protocol we developed)
                            </P><P>References</P><OL><LI>
                                The same <I>E. coli</I> strain was grown without induction inside a dialysis bag suspended in a liquid LB medium with <I>P. putida</I> grown outside the bag. Samples were taken 24 and 72 hours after the start of the experiment to check for viability of the two species.
                            </LI></OL></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row footerColor"><DIV class="col-lg-4 text-center"><H4>Address:</H4><P>
                        Ben-Gurion University of the Negev
                        Ben Gurion 1, Beer Sheva 8410501, Israel
                    </P><H5><B><U>Mail:</U></B> igembgu2016@gmail.com
                    </H5></DIV><DIV class="col-lg-4 text-center"><H4>Connect With Us!</H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>