<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BGU_ISRAEL_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BGU ISRAEL/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>PlastiCure</TITLE></P><NAV class="navbar navbar-inverse navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="mainmenu nav navbar-nav"><LI class="dropdown" id="HomeB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL">Home</A></LI><LI class="dropdown" id="projectB" style="background-color: #CB99C9;"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Achievements">Achievements</A></LI></UL><LI class="dropdown" id="teamB"><A href="https://2016.igem.org/Team:BGU_ISRAEL/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/TeamMembers">Members</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/MeetUniversity">Meet Our University</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/MeetTown">Meet Our Town</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Collaborations">Collaborations</A></LI><LI class="dropdown" id="partsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Parts#Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/ImprovedParts">Improved Parts</A></LI><LI class="dropdown" id="humanB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Integrated_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/PlasticArt">Plastic Art</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/EthicsAndSafety">Ethics &amp; Safety</A></LI><LI class="dropdown" id="attributionsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Attributions">Attributions</A></LI><LI class="dropdown" id="awardsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Awards">Awards</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Measurements">Measurements</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Proof">Proof Of Concept</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/HP/Silver">HP - Silver</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/HP/Gold">HP - Gold</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Engagement">Engagements</A></LI></DIV></DIV></NAV><DIV class="container-fluid"><DIV class="row"><DIV class="projectColor"><P>Notebook</P></DIV></DIV></DIV><DIV class="ProjectCon container-fluid"><DIV class="row"><DIV class="col-lg-8"><DIV class="calendar table-responsive"><TABLE class="calendar_table"><TBODY><TR><TH colspan="7">October</TH></TR><TR><TD class="calendar_day"><A href="#011016">1</A></TD></TR><TR><TD class="calendar_day">2</TD><TD class="calendar_day"><A href="#031016">3</A></TD><TD class="calendar_day"><A href="#041016">4</A></TD><TD class="calendar_day"><A href="#051016">5</A></TD><TD class="calendar_day"><A href="#061016">6</A></TD><TD class="calendar_day"><A href="#071016">7</A></TD><TD class="calendar_day"><A href="#081016">8</A></TD></TR><TR><TD class="calendar_day"><A href="#091016">9</A></TD><TD class="calendar_day"><A href="#101016">10</A></TD><TD class="calendar_day"><A href="#111016">11</A></TD><TD class="calendar_day">12</TD><TD class="calendar_day"><A href="#131016">13</A></TD><TD class="calendar_day"><A href="#141016">14</A></TD><TD class="calendar_day"><A href="#151016">15</A></TD></TR><TR><TD class="calendar_day">16</TD><TD class="calendar_day"><A href="#171016">17</A></TD><TD class="calendar_day"><A href="#181016">18</A></TD><TD class="calendar_day">19</TD><TD class="calendar_day">20</TD><TD class="calendar_day">21</TD><TD class="calendar_day">22</TD></TR><TR><TD class="calendar_day">23</TD><TD class="calendar_day">24</TD><TD class="calendar_day">25</TD><TD class="calendar_day">26</TD><TD class="calendar_day">27</TD><TD class="calendar_day">28</TD><TD class="calendar_day">29</TD></TR><TR><TD class="calendar_day">30</TD><TD class="calendar_day">31</TD></TR></TBODY></TABLE><TABLE class="Month_table"><TBODY><TR><TH colspan="7">September</TH></TR><TR><TD class="calendar_day">1</TD><TD class="calendar_day">2</TD><TD class="calendar_day">3</TD></TR><TR><TD class="calendar_day"><A href="#040916">4</A></TD><TD class="calendar_day">5</TD><TD class="calendar_day"><A href="#060916">6</A></TD><TD class="calendar_day">7</TD><TD class="calendar_day">8</TD><TD class="calendar_day">9</TD><TD class="calendar_day"><A href="#100916">10</A></TD></TR><TR><TD class="calendar_day"><A href="#110916">11</A></TD><TD class="calendar_day"><A href="#120916">12</A></TD><TD class="calendar_day"><A href="#130916">13</A></TD><TD class="calendar_day"><A href="#140916">14</A></TD><TD class="calendar_day"><A href="#150916">15</A></TD><TD class="calendar_day">16</TD><TD class="calendar_day"><A href="#170916">17</A></TD></TR><TR><TD class="calendar_day"><A href="#180916">18</A></TD><TD class="calendar_day"><A href="#190916">19</A></TD><TD class="calendar_day"><A href="#200916">20</A></TD><TD class="calendar_day"><A href="#210916">21</A></TD><TD class="calendar_day"><A href="#220916">22</A></TD><TD class="calendar_day">23</TD><TD class="calendar_day"><A href="#240916">24</A></TD></TR><TR><TD class="calendar_day"><A href="#250916">25</A></TD><TD class="calendar_day"><A href="#260916">26</A></TD><TD class="calendar_day"><A href="#270916">27</A></TD><TD class="calendar_day">28</TD><TD class="calendar_day">29</TD><TD class="calendar_day"><A href="#300916">30</A></TD></TR></TBODY></TABLE><TABLE class="Month_table"><TBODY><TR><TH colspan="7">August</TH></TR><TR><TD class="calendar_day"><A href="#010816">1</A></TD><TD class="calendar_day">2</TD><TD class="calendar_day">3</TD><TD class="calendar_day">4</TD><TD class="calendar_day">5</TD><TD class="calendar_day">6</TD></TR><TR><TD class="calendar_day"><A href="#070816">7</A></TD><TD class="calendar_day">8</TD><TD class="calendar_day">9</TD><TD class="calendar_day"><A href="#100816">10</A></TD><TD class="calendar_day"><A href="#110816">11</A></TD><TD class="calendar_day">12</TD><TD class="calendar_day">13</TD></TR><TR><TD class="calendar_day">14</TD><TD class="calendar_day">15</TD><TD class="calendar_day">16</TD><TD class="calendar_day"><A href="#170816">17</A></TD><TD class="calendar_day">18</TD><TD class="calendar_day">19</TD><TD class="calendar_day">20</TD></TR><TR><TD class="calendar_day">21</TD><TD class="calendar_day">22</TD><TD class="calendar_day">23</TD><TD class="calendar_day">24</TD><TD class="calendar_day">25</TD><TD class="calendar_day">26</TD><TD class="calendar_day">27</TD></TR><TR><TD class="calendar_day">28</TD><TD class="calendar_day"><A href="#290816">29</A></TD><TD class="calendar_day">30</TD><TD class="calendar_day">31</TD></TR></TBODY></TABLE><TABLE class="Month_table"><TBODY><TR><TH colspan="7">July</TH></TR><TR><TD class="calendar_day">1</TD><TD class="calendar_day">2</TD></TR><TR><TD class="calendar_day">3</TD><TD class="calendar_day">4</TD><TD class="calendar_day">5</TD><TD class="calendar_day">6</TD><TD class="calendar_day">7</TD><TD class="calendar_day">8</TD><TD class="calendar_day">9</TD></TR><TR><TD class="calendar_day"><A href="#100716">10</A></TD><TD class="calendar_day"><A href="#110716">11</A></TD><TD class="calendar_day">12</TD><TD class="calendar_day">13</TD><TD class="calendar_day">14</TD><TD class="calendar_day">15</TD><TD class="calendar_day">16</TD></TR><TR><TD class="calendar_day"><A href="#170716">17</A></TD><TD class="calendar_day"><A href="#180716">18</A></TD><TD class="calendar_day">19</TD><TD class="calendar_day">20</TD><TD class="calendar_day">21</TD><TD class="calendar_day">22</TD><TD class="calendar_day">23</TD></TR><TR><TD class="calendar_day">24</TD><TD class="calendar_day">25</TD><TD class="calendar_day"><A href="#260716">26</A></TD><TD class="calendar_day"><A href="#270716">27</A></TD><TD class="calendar_day">28</TD><TD class="calendar_day">29</TD><TD class="calendar_day">30</TD></TR><TR><TD class="calendar_day"><A href="#310716">31</A></TD></TR></TBODY></TABLE><TABLE class="Month_table"><TBODY><TR><TH colspan="7">June</TH></TR><TR><TD class="calendar_day">1</TD><TD class="calendar_day">2</TD><TD class="calendar_day">3</TD><TD class="calendar_day">4</TD></TR><TR><TD class="calendar_day">5</TD><TD class="calendar_day"><A href="#060616">6</A></TD><TD class="calendar_day">7</TD><TD class="calendar_day"><A href="#080616">8</A></TD><TD class="calendar_day">9</TD><TD class="calendar_day"><A href="#100616">10</A></TD><TD class="calendar_day"><A href="#110616">11</A></TD></TR><TR><TD class="calendar_day">12</TD><TD class="calendar_day"><A href="#130616">13</A></TD><TD class="calendar_day"><A href="#140616">14</A></TD><TD class="calendar_day"><A href="#150616">15</A></TD><TD class="calendar_day">16</TD><TD class="calendar_day">17</TD><TD class="calendar_day">18</TD></TR><TR><TD class="calendar_day">19</TD><TD class="calendar_day">20</TD><TD class="calendar_day">21</TD><TD class="calendar_day">22</TD><TD class="calendar_day"><A href="#230616">23</A></TD><TD class="calendar_day">24</TD><TD class="calendar_day">25</TD></TR><TR><TD class="calendar_day">26</TD><TD class="calendar_day">27</TD><TD class="calendar_day">28</TD><TD class="calendar_day">29</TD><TD class="calendar_day">30</TD></TR></TBODY></TABLE><TABLE class="Month_table"><TBODY><TR><TH colspan="7">May</TH></TR><TR><TD class="calendar_day">1</TD><TD class="calendar_day">2</TD><TD class="calendar_day"><A href="#030516">3</A></TD><TD class="calendar_day">4</TD><TD class="calendar_day">5</TD><TD class="calendar_day">6</TD><TD class="calendar_day">7</TD></TR><TR><TD class="calendar_day">8</TD><TD class="calendar_day">9</TD><TD class="calendar_day">10</TD><TD class="calendar_day"><A href="#110516">11</A></TD><TD class="calendar_day">12</TD><TD class="calendar_day"><A href="#130516">13</A></TD><TD class="calendar_day">14</TD></TR><TR><TD class="calendar_day">15</TD><TD class="calendar_day">16</TD><TD class="calendar_day">17</TD><TD class="calendar_day">18</TD><TD class="calendar_day"><A href="#190516">19</A></TD><TD class="calendar_day">20</TD><TD class="calendar_day">21</TD></TR><TR><TD class="calendar_day">22</TD><TD class="calendar_day">23</TD><TD class="calendar_day"><A href="#240516">24</A></TD><TD class="calendar_day">25</TD><TD class="calendar_day">26</TD><TD class="calendar_day">27</TD><TD class="calendar_day"><A href="#280516">28</A></TD></TR><TR><TD class="calendar_day">29</TD><TD class="calendar_day">30</TD><TD class="calendar_day"><A href="#310516">31</A></TD></TR></TBODY></TABLE><TABLE class="Month_table"><TBODY><TR><TH colspan="7">April</TH></TR><TR><TD class="calendar_day">1</TD><TD class="calendar_day">2</TD></TR><TR><TD class="calendar_day"><A href="#030416">3</A></TD><TD class="calendar_day">4</TD><TD class="calendar_day">5</TD><TD class="calendar_day"><A href="#060416">6</A></TD><TD class="calendar_day">7</TD><TD class="calendar_day"><A href="#080416">8</A></TD><TD class="calendar_day">9</TD></TR><TR><TD class="calendar_day">10</TD><TD class="calendar_day"><A href="#110416">11</A></TD><TD class="calendar_day">12</TD><TD class="calendar_day"><A href="#130416">13</A></TD><TD class="calendar_day">14</TD><TD class="calendar_day">15</TD><TD class="calendar_day"><A href="#160416">16</A></TD></TR><TR><TD class="calendar_day"><A href="#170416">17</A></TD><TD class="calendar_day"><A href="#180416">18</A></TD><TD class="calendar_day"><A href="#190416">19</A></TD><TD class="calendar_day">20</TD><TD class="calendar_day">21</TD><TD class="calendar_day">22</TD><TD class="calendar_day">23</TD></TR><TR><TD class="calendar_day">24</TD><TD class="calendar_day"><A href="#250416">25</A></TD><TD class="calendar_day">26</TD><TD class="calendar_day"><A href="#270416">27</A></TD><TD class="calendar_day">28</TD><TD class="calendar_day"><A href="#290416">29</A></TD><TD class="calendar_day"><A href="#300416">30</A></TD></TR></TBODY></TABLE><TABLE class="Month_table"><TBODY><TR><TH colspan="7">March</TH></TR><TR><TD class="calendar_day">1</TD><TD class="calendar_day">2</TD><TD class="calendar_day">3</TD><TD class="calendar_day">4</TD><TD class="calendar_day">5</TD></TR><TR><TD class="calendar_day"><A href="#060316">6</A></TD><TD class="calendar_day"><A href="#070316">7</A></TD><TD class="calendar_day"><A href="#080316">8</A></TD><TD class="calendar_day">9</TD><TD class="calendar_day">10</TD><TD class="calendar_day">11</TD><TD class="calendar_day">12</TD></TR><TR><TD class="calendar_day">13</TD><TD class="calendar_day">14</TD><TD class="calendar_day"><A href="#150316">15</A></TD><TD class="calendar_day">16</TD><TD class="calendar_day"><A href="#170316">17</A></TD><TD class="calendar_day"><A href="#180316">18</A></TD><TD class="calendar_day">19</TD></TR><TR><TD class="calendar_day">20</TD><TD class="calendar_day">21</TD><TD class="calendar_day"><A href="#220316">22</A></TD><TD class="calendar_day"><A href="#230316">23</A></TD><TD class="calendar_day"><A href="#240316">24</A></TD><TD class="calendar_day"><A href="#250316">25</A></TD><TD class="calendar_day">26</TD></TR><TR><TD class="calendar_day"><A href="#270316">27</A></TD><TD class="calendar_day"><A href="#280316">28</A></TD><TD class="calendar_day">29</TD><TD class="calendar_day">30</TD><TD class="calendar_day"><A href="#310316">31</A></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV id="October" class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 18/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal and Dor</STRONG></P><P>Colony PCR for the dialysis bag experiment. We used 2 genes to identify if e.coli and putida can grow together, </P><P>aldA (1440bp) gene for e.coli and pcaD (792bp) gene for putida.</P><P> </P><DIV class="row"><DIV class="col-lg-5"><P>aldA gene of E. coli, plates 3,7,10,1,6 were positive</P></DIV><DIV class="col-lg-5"><P>pcaD gene of P. putida, plates 8,2,3,5,1 were positive</P></DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 17/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal, Dor, Liran and Tomer</STRONG></P><P><SPAN style="text-decoration: underline;">Ligation of 7K transporter inserts and pSEVA 434 vector</SPAN></P><P> </P><P>50ng vectur cut with EcorI and PstI</P><P>142ng insert cut with EcorI and PstI</P><P>2ul T4 DNA ligase Buffer X10</P><P>1ul T4 DNA ligase</P><P>add UPW for total volum of 20ul</P><P> </P><P>16C over night</P><P> </P><P><SPAN style="text-decoration: underline;">Transformation of 7K trensporter + pSEVA 434 Ligation mix</SPAN></P><P>Tubes of compatent BL21</P><P>1ul from each plasmid for each tubes and one tube without plasmid.</P><P>Mix with tip.</P><P>Incubation in ice for 20 min.</P><P>Heat shock 30 sec 42C</P><P>Incubation in ice for 3 min</P><P>800ul SOC without antibiotic</P><P>1h in 37C</P><P>-centrifugation 5000 rpm 2min</P><P>-take off 500ul from the liquid.</P><P>-mix the residue with the rest liquid.</P><P>-sowing in LB+antibiotic</P><P>Incubation 37C O/N</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 15/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal and Dor</STRONG></P><P>TPA detection assay</P><P>Took 2 sample of 200uL one inside the bag, and one outside the bag into 96 wellplate</P><P> </P><P><SPAN style="text-decoration: underline;">TPA detection for kinetics measurements:</SPAN></P><P>We transformed WT and variants LC-Cutinase to E. coli BL21 and we raised each one of them in M9+PET medium+CMP+IPTG.</P><P>We took samples of 200uL daily from each Erlenmeyer into 96 wellplate anti UV.</P><P>We took the plate to a UV table for 1 hour, and insert it into a plate reader for checking the absorption according to the time and date samples, and the change in TPA concentration in time according to our TPA detection protocol. The results show significant raising in the absorption but we didn’t succeed to detect the TPA.</P><P>The reasons why we didn’t succeed to detect TPA can be contamination of the medium with another carbon source or maybe because we didn’t filter the samples so there are some materials that can cause quenching and disturb the fluorescence.</P><P> </P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 14/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal and Dor</STRONG></P><P>TPA detection assay</P><P>Took 2 sample of 200uL one inside the bag, and one outside the bag into 96 wellplate.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 13/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Tomer</STRONG></P><P><SPAN style="text-decoration: underline;">Restriction reaction for pSEVA 434</SPAN></P><P>The total reaction mix volum were 50ul.</P><P>every reaction contain:</P><P>1000ng DNA</P><P>1ul of every enzyme.</P><P>5ul buffer</P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="120"><P><STRONG> </STRONG></P></TD><TD width="120"><P><STRONG>Plasmid</STRONG></P></TD><TD width="120"><P><STRONG>E1</STRONG></P></TD><TD width="120"><P><STRONG>E2</STRONG></P></TD><TD width="120"><P><STRONG>Buffer</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>1</STRONG></P></TD><TD width="120"><P>434</P></TD><TD width="120"><P>AvrII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>3.1</P></TD></TR><TR><TD width="120"><P><STRONG>2</STRONG></P></TD><TD width="120"><P>434</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>3.1</P></TD></TR><TR><TD width="120"><P><STRONG>3</STRONG></P></TD><TD width="120"><P>434</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>3.1</P></TD></TR><TR><TD width="120"><P><STRONG>4</STRONG></P></TD><TD width="120"><P>434</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>3.1</P></TD></TR><TR><TD width="120"><P><STRONG>5</STRONG></P></TD><TD width="120"><P>434</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>3.1</P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 11/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Liran and Tomer</STRONG></P><P><SPAN style="text-decoration: underline;">Produced pSEVA plasmids from starters using mini prep kit.</SPAN></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="143"><P><SPAN style="text-decoration: underline;"> </SPAN></P></TD><TD width="154"><P><STRONG>Concentration ng/ul</STRONG></P></TD></TR><TR><TD width="143"><P>224</P></TD><TD width="154"><P>100</P></TD></TR><TR><TD width="143"><P>434</P></TD><TD width="154"><P>118</P></TD></TR><TR><TD width="143"><P>414</P></TD><TD width="154"><P>126</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">Restriction reaction for pSEVA 242, pSEVA 414 and pSEVA 434:</SPAN></P><P>The total reaction mix volum were 50ul.</P><P>every reaction contain:</P><P>1000ng DNA</P><P>1ul of every enzyme.</P><P>5ul buffer</P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="120"><P><STRONG> </STRONG></P></TD><TD width="120"><P><STRONG>Plasmid</STRONG></P></TD><TD width="120"><P><STRONG>E1</STRONG></P></TD><TD width="120"><P><STRONG>E2</STRONG></P></TD><TD width="120"><P><STRONG>Buffer</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>1</STRONG></P></TD><TD width="120"><P>434</P></TD><TD width="120"><P>AvrII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>2</STRONG></P></TD><TD width="120"><P>434</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>3</STRONG></P></TD><TD width="120"><P>434</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>4</STRONG></P></TD><TD width="120"><P>434</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>5</STRONG></P></TD><TD width="120"><P>434</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>6</STRONG></P></TD><TD width="120"><P>434</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>9</STRONG></P></TD><TD width="120"><P>414</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>10</STRONG></P></TD><TD width="120"><P>414</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>11</STRONG></P></TD><TD width="120"><P>414</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>12</STRONG></P></TD><TD width="120"><P>414</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>13</STRONG></P></TD><TD width="120"><P>224</P></TD><TD width="120"><P>AvrII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>14</STRONG></P></TD><TD width="120"><P>224</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>15</STRONG></P></TD><TD width="120"><P>224</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>16</STRONG></P></TD><TD width="120"><P>224</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>17</STRONG></P></TD><TD width="120"><P>224</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR></TBODY></TABLE><P> </P><P>1h 37C</P><P>20min 65C</P><P> </P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 10/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Liran and Inbar B</STRONG></P><P><SPAN style="text-decoration: underline;">Colony-PCR for DH10B competent cells after transformation with ligation mix of 7K trensporter + pSEVA 434</SPAN></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="100"><P> </P></TD><TD width="100"><P>Templat</P></TD><TD width="100"><P>pr Rev</P></TD><TD width="100"><P>pr Fw</P></TD><TD width="100"><P>taq readymix</P></TD><TD width="100"><P>UPW</P></TD></TR><TR><TD width="100"><P>colony</P></TD><TD width="100"><P>colony</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>10ul</P></TD><TD width="100"><P>7ul</P></TD></TR><TR><TD width="100"><P>7K insert posetiv control</P></TD><TD width="100"><P>5ul 7K insert</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>10ul</P></TD><TD width="100"><P>3ul</P></TD></TR><TR><TD width="100"><P>no templat</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>10ul</P></TD><TD width="100"><P>8ul</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">Gibson assembly of EG insert and pSEVA434 vector with Gibson assembly kit</SPAN></P><P>Total volum 5ul</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="100"><P> </P></TD><TD width="100"><P>EG insert after DpnI</P></TD><TD width="100"><P>pSEVA434</P></TD><TD width="100"><P>posetive control mix</P></TD><TD width="100"><P>Gibson assembly mix</P></TD><TD width="100"><P>UPW</P></TD></TR><TR><TD width="100"><P>EG + 434</P></TD><TD width="100"><P>45ng</P></TD><TD width="100"><P>27ng</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>2.5ul</P></TD><TD width="100"><P>-</P></TD></TR><TR><TD width="100"><P>434 no insert- negativ control</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>27ng</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>2.5ul</P></TD><TD width="100"><P>1ul</P></TD></TR><TR><TD width="100"><P>Posetive control from the kit</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>2.5ul</P></TD><TD width="100"><P>2.5ul</P></TD><TD width="100"><P>-</P></TD></TR></TBODY></TABLE></DIV><P> </P><P>20min 50C</P><P> </P><P><SPAN style="text-decoration: underline;">Transformation of GA reaction mix:</SPAN></P><P>Tubes of compatent DH10B</P><P>1ul from each plasmid for each tubes.</P><P>Mix with tip.</P><P>Incubation in ice for 20 min.</P><P>Heat shock 30 sec 42C</P><P>Incubation in ice for 3 min</P><P>800ul SOC without antibiotic</P><P>1h in 37C</P><P>-centrifugation 5000 rpm 2min</P><P>-take off 500ul from the liquid.</P><P>-mix the residue with the rest liquid.</P><P>-sowing in LB+antibiotic</P><P>Incubation 37C O/N</P><P> </P><P>5 colonys in posetive control, no colonys on the other plates.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 09/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Liran, Eyal and Dor</STRONG></P><P>We sent 4 samples contain cutinase WT, R7, F4 and CO for mass spec.</P><P><SPAN style="text-decoration: underline;">Transformation of 7K trensporter + pSEVA 434 Ligation mix</SPAN></P><P>Tubes of compatent DH10B</P><P>1ul from each plasmid for each tubes and one tube without plasmid.</P><P>Mix with tip.</P><P>Incubation in ice for 20 min.</P><P>Heat shock 30 sec 42C</P><P>Incubation in ice for 3 min</P><P>800ul SOC without antibiotic</P><P>1h in 37C</P><P>-centrifugation 5000 rpm 2min</P><P>-take off 500ul from the liquid.</P><P>-mix the residue with the rest liquid.</P><P>-sowing in LB+antibiotic</P><P>Incubation 37C O/N</P><P> </P><P><SPAN style="text-decoration: underline;">Produced pSEVA plasmids from starters using mini prep kit.</SPAN></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="143"><P><SPAN style="text-decoration: underline;"> </SPAN></P></TD><TD width="154"><P><STRONG>Concentration ng/ul</STRONG></P></TD></TR><TR><TD width="143"><P>414 stab 2</P></TD><TD width="154"><P>69</P></TD></TR><TR><TD width="143"><P>414 col2</P></TD><TD width="154"><P>72</P></TD></TR><TR><TD width="143"><P>414 col2</P></TD><TD width="154"><P>61</P></TD></TR><TR><TD width="143"><P>414 stab 2</P></TD><TD width="154"><P>130</P></TD></TR><TR><TD width="143"><P>434 stab</P></TD><TD width="154"><P>136</P></TD></TR><TR><TD width="143"><P>224 gly</P></TD><TD width="154"><P>195</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">Restriction reaction for pSEVA 242, pSEVA 414 and pSEVA 434:</SPAN></P><P>The total reaction mix volum were 50ul.</P><P>every reaction contain:</P><P>1000ng DNA</P><P>1ul of every enzyme.</P><P>5ul buffer</P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="120"><P><STRONG> </STRONG></P></TD><TD width="120"><P><STRONG>Plasmid</STRONG></P></TD><TD width="120"><P><STRONG>E1</STRONG></P></TD><TD width="120"><P><STRONG>E2</STRONG></P></TD><TD width="120"><P><STRONG>Buffer</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>1</STRONG></P></TD><TD width="120"><P>434 stab</P></TD><TD width="120"><P>AvrII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>2</STRONG></P></TD><TD width="120"><P>434 stab</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>2.1</P></TD></TR><TR><TD width="120"><P><STRONG>3</STRONG></P></TD><TD width="120"><P>434 stab</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>3.1</P></TD></TR><TR><TD width="120"><P><STRONG>4</STRONG></P></TD><TD width="120"><P>434 stab</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>2.1</P></TD></TR><TR><TD width="120"><P><STRONG>5</STRONG></P></TD><TD width="120"><P>434 stab</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>6</STRONG></P></TD><TD width="120"><P>434 stab</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>7</STRONG></P></TD><TD width="120"><P>434 stab</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>8</STRONG></P></TD><TD width="120"><P>434 stab</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>9</STRONG></P></TD><TD width="120"><P>414 stab 2</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>10</STRONG></P></TD><TD width="120"><P>414 stab 2</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>11</STRONG></P></TD><TD width="120"><P>414 stab 2</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>12</STRONG></P></TD><TD width="120"><P>414 stab 2</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>13</STRONG></P></TD><TD width="120"><P>224 gly</P></TD><TD width="120"><P>AvrII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>14</STRONG></P></TD><TD width="120"><P>224 gly</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>2.1</P></TD></TR><TR><TD width="120"><P><STRONG>15</STRONG></P></TD><TD width="120"><P>224 gly</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>3.1</P></TD></TR><TR><TD width="120"><P><STRONG>16</STRONG></P></TD><TD width="120"><P>224 gly</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>2.1</P></TD></TR><TR><TD width="120"><P><STRONG>17</STRONG></P></TD><TD width="120"><P>224 gly</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR></TBODY></TABLE></DIV><P> </P><P>1h 37C</P><P>20min 65C</P><P> </P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 08/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal, Dor, Liran  and Efrat</STRONG></P><P>We prepared the dialysis bag experiment one again according to our protocol for kinetics measurements using our TPA detection assay- we will take sample from inside and outside the bag twice a day for 1 week and then we will expose it to UV light according to the TPA detection protocol to measure the kinetics index of cutinase CO on PET.</P><P>We used dialysis bag to check if <EM>e coli</EM> can live with <EM>P. putida</EM>- e coli inside the dialysis bag and putida outside of the dialysis bag in LB medium.- we will take samples inside the dialysis bag and outside the dialysis bag and sow it on  cam, kan, amp plate to see if they are still alive together (putida has a endogenic resistant to cam and amp, and we insert pACYC to e coli for cam resistant).</P><P><SPAN style="text-decoration: underline;">Restriction reaction for pSEVA 242, pSEVA 414 and pSEVA 434:</SPAN></P><P>The total reaction mix volum were 50ul.</P><P>every reaction contain:</P><P>1000ng DNA</P><P>1ul of every enzyme.</P><P>5ul buffer</P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="120"><P><STRONG> </STRONG></P></TD><TD width="120"><P><STRONG>Plasmid</STRONG></P></TD><TD width="120"><P><STRONG>E1</STRONG></P></TD><TD width="120"><P><STRONG>E2</STRONG></P></TD><TD width="120"><P><STRONG>Buffer</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>1</STRONG></P></TD><TD width="120"><P>434 stab 2</P></TD><TD width="120"><P>AvrII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>2</STRONG></P></TD><TD width="120"><P>434 stab 2</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>2.1</P></TD></TR><TR><TD width="120"><P><STRONG>3</STRONG></P></TD><TD width="120"><P>434 stab 2</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>3.1</P></TD></TR><TR><TD width="120"><P><STRONG>4</STRONG></P></TD><TD width="120"><P>434 stab 2</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>2.1</P></TD></TR><TR><TD width="120"><P><STRONG>5</STRONG></P></TD><TD width="120"><P>434 stab 2</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>6</STRONG></P></TD><TD width="120"><P>434 stab 2</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>7</STRONG></P></TD><TD width="120"><P>434 stab 2</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>8</STRONG></P></TD><TD width="120"><P>434 stab 2</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>9</STRONG></P></TD><TD width="120"><P>414 stab 2</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>10</STRONG></P></TD><TD width="120"><P>414 stab 2</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>11</STRONG></P></TD><TD width="120"><P>414 stab 2</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>12</STRONG></P></TD><TD width="120"><P>414 stab 2</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>13</STRONG></P></TD><TD width="120"><P>224 gly1</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>14</STRONG></P></TD><TD width="120"><P>224 gly2</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>2.1</P></TD></TR></TBODY></TABLE></DIV><P> </P><P>1h 37C</P><P>20min 65C</P><P> </P><P><SPAN style="text-decoration: underline;">Ligation of 7K transporter inserts and pSEVA 434 vector</SPAN></P><P> </P><P>50ng vectur cut with EcorI and PstI</P><P>142ng insert cut with EcorI and PstI</P><P>2ul T4 DNA ligase Buffer X10</P><P>1ul T4 DNA ligase</P><P>add UPW for total volum of 20ul</P><P> </P><P>16C over nighet.</P><P> </P><P><SPAN style="text-decoration: underline;">BioBricks: produced plasmids from starters using mini prep kit.</SPAN></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="150"><P><STRONG>plasmid</STRONG></P></TD><TD width="150"><P><STRONG>concentration(</STRONG><STRONG>ng/ul)</STRONG></P></TD><TD width="150"><P><STRONG>260\230</STRONG></P></TD><TD width="150"><P><STRONG>260\280</STRONG></P></TD></TR><TR><TD width="150"><P>F4 col1</P></TD><TD width="150"><P>301</P></TD><TD width="150"><P>2.06</P></TD><TD width="150"><P>1.85</P></TD></TR><TR><TD width="150"><P>F4 col3</P></TD><TD width="150"><P>283</P></TD><TD width="150"><P>2.04</P></TD><TD width="150"><P>1.87</P></TD></TR><TR><TD width="150"><P>F7 col3</P></TD><TD width="150"><P>327</P></TD><TD width="150"><P>2.23</P></TD><TD width="150"><P>1.87</P></TD></TR><TR><TD width="150"><P>F7 col4</P></TD><TD width="150"><P>218</P></TD><TD width="150"><P>1.80</P></TD><TD width="150"><P>1.87</P></TD></TR><TR><TD width="150"><P>R4 col2</P></TD><TD width="150"><P>291</P></TD><TD width="150"><P>2.10</P></TD><TD width="150"><P>1.90</P></TD></TR><TR><TD width="150"><P>R4 col4</P></TD><TD width="150"><P>314</P></TD><TD width="150"><P>2.10</P></TD><TD width="150"><P>1.87</P></TD></TR><TR><TD width="150"><P>R7 col5</P></TD><TD width="150"><P>313</P></TD><TD width="150"><P>2.07</P></TD><TD width="150"><P>1.86</P></TD></TR><TR><TD width="150"><P>R7 col6</P></TD><TD width="150"><P>313</P></TD><TD width="150"><P>2.07</P></TD><TD width="150"><P>1.86</P></TD></TR><TR><TD width="150"><P>PcaC col1</P></TD><TD width="150"><P>267</P></TD><TD width="150"><P>1.92</P></TD><TD width="150"><P>1.88</P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 07/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Tomer, Efrat, Liran and Inbar B</STRONG></P><P><SPAN style="text-decoration: underline;">Examine PET </SPAN><SPAN style="text-decoration: underline;">degradation bay LCC CO:</SPAN></P><P>3 PET bids were incubate with 0.5L LB contaning cmp and BL21 with LCC CO on pACYC plasmid.</P><P>When OD reach 0.5-0.7 add IPTG.</P><P>For control, repeat with pACYC plasmid without insert, and without bacteria.</P><P> </P><DIV class="row"><DIV class="col-lg-5"><P>C.O</P></DIV><DIV class="col-lg-5"><P>pAYAC</P></DIV></DIV><P> </P><P><SPAN style="text-decoration: underline;">Testing BL21 that contains LCC variants groth on PET plates:</SPAN></P><P>PET plates:</P><P>For 1L:</P><P>200ml M9 salts</P><P>2ml 1M mgSO4</P><P>20ml glucose 20%</P><P>100ul 1M CaCl2</P><P>(5ml soft agar for each plate)</P><P>The 19 plates:</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="120"><P><STRONG> </STRONG></P></TD><TD width="120"><P><STRONG>IPTG &amp;PET</STRONG></P></TD><TD width="120"><P><STRONG>IPTG no PET</STRONG></P></TD><TD width="120"><P><STRONG>PET no IPTG</STRONG></P></TD><TD width="120"><P><STRONG>Glucose</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>F4</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>-</STRONG></P></TD><TD width="120"><P><STRONG>-</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>F7</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>-</STRONG></P></TD><TD width="120"><P><STRONG>-</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>R4</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>-</STRONG></P></TD><TD width="120"><P><STRONG>-</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>R7</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>-</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>CO</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>-</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>pACYC</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>-</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>WT</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD><TD width="120"><P><STRONG>V</STRONG></P></TD></TR></TBODY></TABLE></DIV><P><STRONG> </STRONG></P><P><SPAN style="text-decoration: underline;">Produced pSEVA plasmids from starters using mini prep kit.</SPAN></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="286"><P><STRONG>Concentration ng/ul</STRONG></P></TD><TD width="286"><P><SPAN style="text-decoration: underline;"> </SPAN></P></TD></TR><TR><TD width="286"><P>47</P></TD><TD width="286"><P>434 stab 1</P></TD></TR><TR><TD width="286"><P>85</P></TD><TD width="286"><P>434 stab 2</P></TD></TR><TR><TD width="286"><P>36</P></TD><TD width="286"><P>414 stab 1</P></TD></TR><TR><TD width="286"><P>159</P></TD><TD width="286"><P>414 stab 2</P></TD></TR><TR><TD width="286"><P>38</P></TD><TD width="286"><P>414 col1</P></TD></TR><TR><TD width="286"><P>121</P></TD><TD width="286"><P>414 col2</P></TD></TR><TR><TD width="286"><P>62</P></TD><TD width="286"><P>414 col3</P></TD></TR><TR><TD width="286"><P>370</P></TD><TD width="286"><P>414 col4</P></TD></TR><TR><TD width="286"><P>32</P></TD><TD width="286"><P>224 gly1</P></TD></TR><TR><TD width="286"><P>24</P></TD><TD width="286"><P>224 gly2</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">transformation of pSEVA 224 from gly1, pSEVA 414 from col2, pSEVA 414 from stab 2 and pSEVA 434 from stab 2:</SPAN></P><P>Tubes of compatent DH10B</P><P>1ul from each plasmid for each tubes and one tube without plasmid.</P><P>Mix with tip.</P><P>Incubation in ice for 20 min.</P><P>Heat shock 30 sec 42C</P><P>Incubation in ice for 3 min</P><P>800ul SOC without antibiotic</P><P>1h in 37C</P><P>-centrifugation 5000 rpm 2min</P><P>-take off 500ul from the liquid.</P><P>-mix the residue with the rest liquid.</P><P>-sowing in LB+antibiotic</P><P>Incubation 37C O/N</P><P> </P><P><SPAN style="text-decoration: underline;">PCR for amplification EG insert for Gibson with KAPA HIFI Hot Start, after previous reaction ended with Unspecific product</SPAN></P><P>Template –EG 1ul to 49 ul UPW (EG first concentration- 572 ng\ul)</P><P>Primers- 10uM</P><P>Annealing temperature 70C, 1ul DMSO.</P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="95"><P><STRONG> </STRONG></P></TD><TD width="95"><P><STRONG>Upw (ul)</STRONG></P></TD><TD width="95"><P><STRONG>Pr fwd (ul)</STRONG></P></TD><TD width="95"><P><STRONG>Pr rev</STRONG></P></TD><TD width="95"><P><STRONG>template</STRONG></P></TD><TD width="95"><P><STRONG>KAPA Hiti hot start (ul)</STRONG></P></TD></TR><TR><TD width="95"><P><STRONG>EG</STRONG></P></TD><TD width="95"><P>20</P></TD><TD width="95"><P>1.5</P></TD><TD width="95"><P>1.5</P></TD><TD width="95"><P>1</P></TD><TD width="95"><P>25</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">PCR to amplify TphA1, TphA2, TphA3 and TphB for Gibson assembly</SPAN></P><P>Primers- 10uM</P><P>Annealing temperature 68C<STRONG><SPAN style="text-decoration: underline;">  </SPAN></STRONG></P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="96"><P><STRONG> </STRONG></P></TD><TD width="96"><P><STRONG>Upw (ul)</STRONG></P></TD><TD width="96"><P><STRONG>Pr fwd (ul)</STRONG></P></TD><TD width="96"><P><STRONG>Pr rev</STRONG></P></TD><TD width="96"><P><STRONG>template</STRONG></P></TD><TD width="96"><P><STRONG>KAPA Hiti hot start (ul)</STRONG></P></TD></TR><TR><TD width="96"><P><STRONG>TphA1</STRONG></P></TD><TD width="96"><P>21</P></TD><TD width="96"><P>1.5</P></TD><TD width="96"><P>1.5</P></TD><TD width="96"><P>1</P></TD><TD width="96"><P>25</P></TD></TR><TR><TD width="96"><P><STRONG>TphA2</STRONG></P></TD><TD width="96"><P>21</P></TD><TD width="96"><P>1.5</P></TD><TD width="96"><P>1.5</P></TD><TD width="96"><P>1</P></TD><TD width="96"><P>25</P></TD></TR><TR><TD width="96"><P><STRONG>TphA3</STRONG></P></TD><TD width="96"><P>21</P></TD><TD width="96"><P>1.5</P></TD><TD width="96"><P>1.5</P></TD><TD width="96"><P>1</P></TD><TD width="96"><P>25</P></TD></TR><TR><TD width="96"><P><STRONG>TphB</STRONG></P></TD><TD width="96"><P>21</P></TD><TD width="96"><P>1.5</P></TD><TD width="96"><P>1.5</P></TD><TD width="96"><P>1</P></TD><TD width="96"><P>25</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">Using DpnI on EG, TphA1, TphA2, TphA3 and TphB to eliminate closed plasmid from the PCR product. pSB1C3 as control</SPAN></P><P> </P><P>44ul of PCR reaction mix</P><P>5ul x10 CutSmart buffer</P><P>1ul DpnI</P><P>37C for 1h</P><P>80C for 20min</P><P> </P><DIV class="row"><DIV class="col-lg-5"><P>TphA1- Unspecific product TphA2 -no product</P></DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 06/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar B and Liran</STRONG></P><P><SPAN style="text-decoration: underline;">Testing bacteriom growth on antibiotics stocks:</SPAN></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="151"><P><STRONG>antibiotics</STRONG></P></TD><TD width="151"><P><STRONG>KT2440 from Glycerol stock</STRONG></P></TD><TD width="151"><P><STRONG>KT2440 from petri dish</STRONG></P></TD></TR><TR><TD width="151"><P><STRONG>Cmp</STRONG></P></TD><TD width="151"><P><STRONG>V</STRONG></P></TD><TD width="151"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="151"><P><STRONG>Cmp+Amp</STRONG></P></TD><TD width="151"><P><STRONG>V</STRONG></P></TD><TD width="151"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="151"><P><STRONG>Cmp+Strep</STRONG></P></TD><TD width="151"><P><STRONG>-</STRONG></P></TD><TD width="151"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="151"><P><STRONG>Spect</STRONG></P></TD><TD width="151"><P><STRONG>-</STRONG></P></TD><TD width="151"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="151"><P><STRONG>Cmp+Strep+Amp</STRONG></P></TD><TD width="151"><P><STRONG>-</STRONG></P></TD><TD width="151"><P><STRONG>-</STRONG></P></TD></TR><TR><TD width="151"><P><STRONG>Km</STRONG></P></TD><TD width="151"><P><STRONG>-</STRONG></P></TD><TD width="151"><P><STRONG>-</STRONG></P></TD></TR><TR><TD width="151"><P><STRONG>Strep X2</STRONG></P></TD><TD width="151"><P><STRONG>-</STRONG></P></TD><TD width="151"><P><STRONG>-</STRONG></P></TD></TR><TR><TD width="151"><P><STRONG>Spect X2</STRONG></P></TD><TD width="151"><P><STRONG>-</STRONG></P></TD><TD width="151"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="151"><P><STRONG>Control-no antibiotic</STRONG></P></TD><TD width="151"><P><STRONG>V</STRONG></P></TD><TD width="151"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="151"><P><STRONG>Amp</STRONG></P></TD><TD width="151"><P><STRONG>V</STRONG></P></TD><TD width="151"><P><STRONG>V</STRONG></P></TD></TR></TBODY></TABLE></DIV><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="301"><P><STRONG>antibiotics</STRONG></P></TD><TD width="301"><P><STRONG>W.T</STRONG></P></TD></TR><TR><TD width="301"><P><STRONG>Control – no antibiotic</STRONG></P></TD><TD width="301"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="301"><P><STRONG>Cmp+Amp</STRONG></P></TD><TD width="301"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="301"><P><STRONG>Amp</STRONG></P></TD><TD width="301"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="301"><P><STRONG>Strep</STRONG></P></TD><TD width="301"><P><STRONG>-</STRONG></P></TD></TR></TBODY></TABLE></DIV><P> </P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="301"><P><STRONG>antibiotics</STRONG></P></TD><TD width="301"><P><STRONG>E.Coli 224</STRONG></P></TD></TR><TR><TD width="301"><P><STRONG>Control – no antibiotic</STRONG></P></TD><TD width="301"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="301"><P><STRONG>Km 25 ug\ul</STRONG></P></TD><TD width="301"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="301"><P><STRONG>Km 50 ug\ul</STRONG></P></TD><TD width="301"><P><STRONG>V</STRONG></P></TD></TR><TR><TD width="301"><P><STRONG>Strep</STRONG></P></TD><TD width="301"><P><STRONG>-</STRONG></P></TD></TR><TR><TD width="301"><P><STRONG>Spect</STRONG></P></TD><TD width="301"><P><STRONG>-</STRONG></P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">Restriction reaction for pAC75 in order to get EG insert for Gibson after PCR product was Unspecific </SPAN></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="100"><P> </P></TD><TD width="100"><P><STRONG>reaction 1</STRONG></P></TD><TD width="100"><P><STRONG>reaction 2</STRONG></P></TD><TD width="100"><P><STRONG>reaction 3</STRONG></P></TD><TD width="100"><P><STRONG>reaction 4</STRONG></P></TD><TD width="100"><P><STRONG>reaction 5</STRONG></P></TD></TR><TR><TD width="100"><P>pAC75</P></TD><TD width="100"><P>3.5ul</P></TD><TD width="100"><P>3.5ul</P></TD><TD width="100"><P>3.5ul</P></TD><TD width="100"><P>3.5ul</P></TD><TD width="100"><P>5ul</P></TD></TR><TR><TD width="100"><P>cutsmart x10</P></TD><TD width="100"><P>10ul</P></TD><TD width="100"><P>10ul</P></TD><TD width="100"><P>10ul</P></TD><TD width="100"><P>10ul</P></TD><TD width="100"><P>-</P></TD></TR><TR><TD width="100"><P>UPW</P></TD><TD width="100"><P>83.5ul</P></TD><TD width="100"><P>85.5ul</P></TD><TD width="100"><P>85.5ul</P></TD><TD width="100"><P>85.5ul</P></TD><TD width="100"><P>-</P></TD></TR><TR><TD width="100"><P>KpnI</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>-</P></TD></TR><TR><TD width="100"><P>HindIII</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>-</P></TD></TR><TR><TD width="100"><P>AatII</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>-</P></TD><TD width="100"><P>1ul</P></TD><TD width="100"><P>-</P></TD></TR></TBODY></TABLE></DIV><P> </P><P>1h 37C</P><P>20min 65C</P><P> </P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 05/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Efrat and Liran</STRONG></P><P><SPAN style="text-decoration: underline;">Preparing antibiotics stocks:</SPAN></P><P>Spectinomycin 100mg\ml</P><P>cmp 34mg\ml</P><P>amp 100mg\ml</P><P>Kn 25mg\ml</P><P> </P><P><SPAN style="text-decoration: underline;">Colony-PCR for DH10B competent cells after transformation with ligation mix of pSB1C3 and the inserts F4, F7, R4, R7 and CO:</SPAN></P><P> </P><P>template- toch colony, then reaction</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="99"><P><STRONG> </STRONG></P></TD><TD width="99"><P><STRONG>Annealing temperature</STRONG></P><P><STRONG> (</STRONG><STRONG>C°</STRONG><STRONG>)</STRONG></P></TD><TD width="99"><P><STRONG>Upw (ul)</STRONG></P></TD><TD width="99"><P><STRONG>Pr fwd (ul)</STRONG></P></TD><TD width="99"><P><STRONG>Pr rev</STRONG></P></TD><TD width="99"><P><STRONG>KAPA Hiti hot start (ul)</STRONG></P></TD></TR><TR><TD width="99"><P><STRONG>C.O</STRONG></P></TD><TD width="99"><P>58.4</P></TD><TD width="99"><P>22</P></TD><TD width="99"><P>1.5</P></TD><TD width="99"><P>1.5</P></TD><TD width="99"><P>25</P></TD></TR><TR><TD width="99"><P><STRONG>F4</STRONG></P></TD><TD width="99"><P>58.4</P></TD><TD width="99"><P>22</P></TD><TD width="99"><P>1.5</P></TD><TD width="99"><P>1.5</P></TD><TD width="99"><P>25</P></TD></TR><TR><TD width="99"><P><STRONG>F7</STRONG></P></TD><TD width="99"><P>58.4</P></TD><TD width="99"><P>22</P></TD><TD width="99"><P>1.5</P></TD><TD width="99"><P>1.5</P></TD><TD width="99"><P>25</P></TD></TR><TR><TD width="99"><P><STRONG>R4</STRONG></P></TD><TD width="99"><P>58.4</P></TD><TD width="99"><P>22</P></TD><TD width="99"><P>1.5</P></TD><TD width="99"><P>1.5</P></TD><TD width="99"><P>25</P></TD></TR><TR><TD width="99"><P><STRONG>R7</STRONG></P></TD><TD width="99"><P>58.4</P></TD><TD width="99"><P>22</P></TD><TD width="99"><P>1.5</P></TD><TD width="99"><P>1.5</P></TD><TD width="99"><P>25</P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 04/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar B, Tomer and Liran</STRONG></P><P><SPAN style="text-decoration: underline;">PCR for amplification EG insert for Gibson with KAPA HIFI Hot Start, after previous reaction ended with Unspecific product</SPAN></P><P>Template –EG 1ul to 49 ul UPW (EG first concentration- 572 ng\ul)</P><P>Primers- 10uM</P><P>Annealing temperature 68C</P><P><SPAN style="text-decoration: underline;">Using DpnI on EG insert to eliminate closed plasmid from the PCR product</SPAN></P><P> </P><P>44ul of EG PCR reaction mix</P><P>5ul x10 CutSmart buffer</P><P>1ul DpnI</P><P>37C for 1h</P><P>80C for 20min</P><P> </P><DIV class="row"><DIV class="col-lg-5"><P>Unspecific product</P></DIV></DIV><P> </P><P><SPAN style="text-decoration: underline;">Restriction reaction for pSEVA 242, pSEVA 414, pSB1C3 and pUC19 as control:</SPAN></P><P>The total reaction mix volum were 50ul.</P><P>every reaction contain:</P><P>1000ng DNA</P><P>1ul of every enzyme.</P><P>5ul buffer</P><P>Duplicates for every reaction.</P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="120"><P><STRONG> </STRONG></P></TD><TD width="120"><P><STRONG>Plasmid</STRONG></P></TD><TD width="120"><P><STRONG>E1</STRONG></P></TD><TD width="120"><P><STRONG>E2</STRONG></P></TD><TD width="120"><P><STRONG>Buffer</STRONG></P></TD></TR><TR><TD width="120"><P><STRONG>1</STRONG></P></TD><TD width="120"><P>224</P></TD><TD width="120"><P>AvrII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>2</STRONG></P></TD><TD width="120"><P>224</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>2.1</P></TD></TR><TR><TD width="120"><P><STRONG>3</STRONG></P></TD><TD width="120"><P>224</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>3.1</P></TD></TR><TR><TD width="120"><P><STRONG>4</STRONG></P></TD><TD width="120"><P>224</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>2.1</P></TD></TR><TR><TD width="120"><P><STRONG>5</STRONG></P></TD><TD width="120"><P>224</P></TD><TD width="120"><P>DpnI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>C1</STRONG></P></TD><TD width="120"><P>224</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>6</STRONG></P></TD><TD width="120"><P>414</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>7</STRONG></P></TD><TD width="120"><P>414</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>8</STRONG></P></TD><TD width="120"><P>414</P></TD><TD width="120"><P>SpeI</P></TD><TD width="120"><P>SacII</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>9</STRONG></P></TD><TD width="120"><P>414</P></TD><TD width="120"><P>DpnI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>C2</STRONG></P></TD><TD width="120"><P>414</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>10</STRONG></P></TD><TD width="120"><P>pSB1C3</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>3.1</P></TD></TR><TR><TD width="120"><P><STRONG>11</STRONG></P></TD><TD width="120"><P>pSB1C3</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>2.1</P></TD></TR><TR><TD width="120"><P><STRONG>12</STRONG></P></TD><TD width="120"><P>pSB1C3</P></TD><TD width="120"><P>EcorI</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>2.1</P></TD></TR><TR><TD width="120"><P><STRONG>C3</STRONG></P></TD><TD width="120"><P>pSB1C3</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>13</STRONG></P></TD><TD width="120"><P>pUC19</P></TD><TD width="120"><P>PstI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>3.1</P></TD></TR><TR><TD width="120"><P><STRONG>14</STRONG></P></TD><TD width="120"><P>pUC19</P></TD><TD width="120"><P>DpnI</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR><TR><TD width="120"><P><STRONG>C4</STRONG></P></TD><TD width="120"><P>pUC19</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>-</P></TD><TD width="120"><P>cutsmart</P></TD></TR></TBODY></TABLE></DIV><P> </P><P>1h 37C</P><P>20min 65C</P><P> </P><P> </P><P><SPAN style="text-decoration: underline;">transformation of LCC varients:</SPAN></P><P>Tubes of compatent BL21</P><P>1ul from each plasmid for each tubes and one tube without plasmid.</P><P>Mix with tip.</P><P>Incubation in ice for 20 min.</P><P>Heat shock 30 sec 42C</P><P>Incubation in ice for 3 min</P><P>800ul SOC without antibiotic</P><P>1h in 37C</P><P>100ul from each tube to 5ml LB+5ul cmp.</P><P>Incubation 37C O/N</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 03/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Efrat</STRONG></P><P><SPAN style="text-decoration: underline;">BioBricks after PCR cleanup:</SPAN></P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="143"><P><SPAN style="text-decoration: underline;"> </SPAN></P></TD><TD width="154"><P><STRONG>Concentration ng/ul</STRONG></P></TD><TD width="149"><P><STRONG>260/230</STRONG></P></TD><TD width="149"><P><STRONG>260/280</STRONG></P></TD></TR><TR><TD width="143"><P>F7</P></TD><TD width="154"><P>313</P></TD><TD width="149"><P>1.96</P></TD><TD width="149"><P>1.88</P></TD></TR><TR><TD width="143"><P>R4</P></TD><TD width="154"><P>374</P></TD><TD width="149"><P>1.92</P></TD><TD width="149"><P>1.88</P></TD></TR><TR><TD width="143"><P>R7</P></TD><TD width="154"><P>294</P></TD><TD width="149"><P>2.00</P></TD><TD width="149"><P>1.9</P></TD></TR><TR><TD width="143"><P>F4</P></TD><TD width="154"><P>232</P></TD><TD width="149"><P>2.01</P></TD><TD width="149"><P>1.88</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">Restriction reaction for BioBriks inserts</SPAN></P><P> </P><P>250ng DNA (insert or PSB1C3 vector)</P><P>3ul NEB Buffer II</P><P>0.5ul PstI</P><P>0.5ul EcorI</P><P>add UPW for total volum of 20ul</P><P> </P><P>1h 37C</P><P>20min 80C</P><P> </P><P><SPAN style="text-decoration: underline;">BioBricks restriction product after PCR cleanup:</SPAN></P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="143"><P><SPAN style="text-decoration: underline;"> </SPAN></P></TD><TD width="154"><P><STRONG>Concentration ng/ul</STRONG></P></TD><TD width="149"><P><STRONG>260/230</STRONG></P></TD><TD width="149"><P><STRONG>260/280</STRONG></P></TD></TR><TR><TD width="143"><P>F7</P></TD><TD width="154"><P>92</P></TD><TD width="149"><P>1.49</P></TD><TD width="149"><P>1.88</P></TD></TR><TR><TD width="143"><P>R4</P></TD><TD width="154"><P>70</P></TD><TD width="149"><P>1.06</P></TD><TD width="149"><P>1.89</P></TD></TR><TR><TD width="143"><P>R7</P></TD><TD width="154"><P>91</P></TD><TD width="149"><P>2.01</P></TD><TD width="149"><P>1.88</P></TD></TR><TR><TD width="143"><P>F4</P></TD><TD width="154"><P>74</P></TD><TD width="149"><P>1.53</P></TD><TD width="149"><P>1.85</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">ligation of BioBriks F7, R4, R7 and F4 inserts and pSB1C3 vector</SPAN></P><P> </P><P>50ng vectur cut with EcorI and PstI</P><P>150ng insert cut with EcorI and PstI</P><P>0.5ul T4 DNA ligase Buffer</P><P>1ul T4 DNA ligase</P><P>add UPW for total volum of 50ul</P><P> </P><P>16C over nighet.</P><P> </P><P><SPAN style="text-decoration: underline;">Colony-PCR for DH10B competent cells after transformation with ligation mix of pSB1C3 and the inserts F4, F7, R4, R7 and CO:</SPAN></P><P> </P><P>template- toch colony, then reaction</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="107"><P> </P></TD><TD width="107"><P>Annealing temperature</P><P> (C°)</P></TD><TD width="107"><P>Upw (ul)</P></TD><TD width="87"><P>Pr fwd (ul)</P></TD><TD width="81"><P>Pr rev</P></TD><TD width="114"><P>KAPA Hiti hot start (ul)</P></TD></TR><TR><TD width="107"><P>C.O</P></TD><TD width="107"><P>58.4</P></TD><TD width="107"><P>22</P></TD><TD width="87"><P>1.5</P></TD><TD width="81"><P>1.5</P></TD><TD width="114"><P>25</P></TD></TR><TR><TD width="107"><P>F4</P></TD><TD width="107"><P>58.4</P></TD><TD width="107"><P>22</P></TD><TD width="87"><P>1.5</P></TD><TD width="81"><P>1.5</P></TD><TD width="114"><P>25</P></TD></TR><TR><TD width="107"><P>F7</P></TD><TD width="107"><P>58.4</P></TD><TD width="107"><P>22</P></TD><TD width="87"><P>1.5</P></TD><TD width="81"><P>1.5</P></TD><TD width="114"><P>25</P></TD></TR><TR><TD width="107"><P>R4</P></TD><TD width="107"><P>58.4</P></TD><TD width="107"><P>22</P></TD><TD width="87"><P>1.5</P></TD><TD width="81"><P>1.5</P></TD><TD width="114"><P>25</P></TD></TR><TR><TD width="107"><P>R7</P></TD><TD width="107"><P>58.4</P></TD><TD width="107"><P>22</P></TD><TD width="87"><P>1.5</P></TD><TD width="81"><P>1.5</P></TD><TD width="114"><P>25</P></TD></TR><TR><TD width="107"><P>pACYC as negative control</P></TD><TD width="107"><P>58.4</P></TD><TD width="107"><P>22</P></TD><TD width="87"><P>1.5</P></TD><TD width="81"><P>1.5</P></TD><TD width="114"><P>25</P></TD></TR></TBODY></TABLE></DIV><P> </P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 01/10/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar B Eyal and Dor</STRONG></P><P>Cutinase kinetics measurement:</P><P>We were trying to get the kinetics index of our cutinase- WT. F4, CO using the first velocity way (calculating the shape of the graph in the linear measurement) and exposing the V by that way.</P><P>After that, using the linear burk plot to reveal the kinetics index.</P><P>We tried to do this tests for more than 10 days, but we didn’t succeed to get a stable indexes. We tried to change the enzyme concertation to see measure the process more accurately, but still, we didn’t succeed to get stable indexes.</P><P><SPAN style="text-decoration: underline;">Using DpnI on EG insert to eliminate closed plasmid from the PCR product</SPAN></P><P> </P><P>44ul of EG PCR reaction mix</P><P>5ul x10 CutSmart buffer</P><P>1ul DpnI</P><P>37C for 1h</P><P>80C for 20min</P><P> </P></DIV></DIV></DIV></DIV><DIV id="September" class="container-fluid"><DIV class="row"><DIV class="col-lg-8"><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 30/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar B Liran and Tomer</STRONG></P><P><SPAN style="text-decoration: underline;">PCR for amplification EG insert for doing Gibson reaction with KAPA HIFI Hot Start</SPAN></P><P>Template –EG 1ul to 49 ul UPW (EG first concentration- 572 ng\ul)</P><P>Primers- 10uM</P><P> </P><P><SPAN style="text-decoration: underline;">Restriction for pSEVA 224 and pSEVA414:</SPAN></P><P> </P><P><STRONG>Plasmids sizes:</STRONG></P><P>pSEVA 224 5250bp</P><P>pSEVA 414 3479bp</P><P>Transporter 4368bp</P><P>pUC19 2568bp</P><P>pSB1C3 2070 bp</P><P> </P><P><SPAN style="text-decoration: underline;">For Gibson reactions:</SPAN></P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="198"><P><STRONG>Enzyme</STRONG></P></TD><TD width="197"><P><STRONG>Plasmid</STRONG></P></TD><TD width="200"><P><STRONG>Concentration</STRONG></P></TD></TR><TR><TD width="198"><P>AvrII</P></TD><TD width="197"><P>pSEVA 224</P></TD><TD width="200"><P>112ng/ul</P></TD></TR><TR><TD width="198"><P>SpeI+SacII</P></TD><TD width="197"><P>pSEVA 414</P></TD><TD width="200"><P>173ng/ul</P></TD></TR></TBODY></TABLE><P> </P><P><SPAN style="text-decoration: underline;">For restriction and ligation:</SPAN></P><TABLE width="595"><TBODY><TR><TD width="197"><P><STRONG>Enzyme</STRONG></P></TD><TD width="198"><P><STRONG>Plasmid</STRONG></P></TD><TD width="200"><P><STRONG>Concentration</STRONG></P></TD></TR><TR><TD width="197"><P>EcoRI +PstI</P></TD><TD width="198"><P>pSEVA 224</P></TD><TD width="200"><P>112ng/ul</P></TD></TR></TBODY></TABLE></DIV><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="205"><P><STRONG>Enzyme</STRONG></P></TD><TD width="205"><P><STRONG>Buffer</STRONG></P></TD></TR><TR><TD width="205"><P>EcoRI</P></TD><TD width="205"><P>3.1-100%</P><P>2.1-100% + star activity</P></TD></TR><TR><TD width="205"><P>PstI</P></TD><TD width="205"><P>3.1-100%</P><P>2.1-75%</P></TD></TR><TR><TD width="205"><P>AvrII</P></TD><TD width="205"><P>Cut smart-100%</P><P>1.1-100%</P></TD></TR><TR><TD width="205"><P>SacII</P></TD><TD width="205"><P>Cut smart-100%</P><P>2.1-100%</P></TD></TR><TR><TD width="205"><P>SpeI</P></TD><TD width="205"><P>Cut smart-100%</P><P>2.1-100%</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">Reactions</SPAN>:     </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="297"><P>DNA</P></TD><TD width="298"><P>pSEVA224 10ul</P><P>pSEVA414 6ul</P></TD></TR><TR><TD width="297"><P>Buffer 3.1</P></TD><TD width="298"><P>5ul</P></TD></TR><TR><TD width="297"><P>Enzyme</P></TD><TD width="298"><P>1ul</P></TD></TR><TR><TD width="297"><P>DDW</P></TD><TD width="298"><P>pSEVA224 34ul</P><P>pSEVA414 38ul</P></TD></TR><TR><TD width="297"><P>Total</P></TD><TD width="298"><P>50ul</P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 27/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal, Dor and Inbar B</STRONG></P><P><SPAN style="text-decoration: underline;">Activity test for cutinase:</SPAN></P><P>We tasted cleaned cutinase F4, CO and WT, and uncleaned cutinse F4, CO and WT in this test.</P><P>We used 96DW and a plate reader.</P><P>We added 50ul pNPB without cutinase, Uncleaned cutinase WT with pNPB (50ul and 50ul), Uncleaned cutinase R7 with pNPB (50ul and 50ul), Uncleaned cutinase F4 with pNPB (50ul and 50ul), pNPB and cleaned cutinase WT(50ul an 50ul), pNPB and cleaned cutinase</P><P> </P><P style="text-decoration: underline;">Restriction for pSEVA</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="155"><P> </P></TD><TD width="138"><P><STRONG>Concentration ug/ul</STRONG></P></TD><TD width="81"><P><STRONG>260/230</STRONG></P></TD><TD width="92"><P><STRONG>260/280</STRONG></P></TD></TR><TR><TD width="155"><P>pSEVA 224 EcoRI</P></TD><TD width="138"><P>50</P></TD><TD width="81"><P>3.14</P></TD><TD width="92"><P>1.72</P></TD></TR><TR><TD width="155"><P>pSEVA 224 EcoRI and PstI</P></TD><TD width="138"><P>67</P></TD><TD width="81"><P>0.82</P></TD><TD width="92"><P>1.70</P></TD></TR><TR><TD width="155"><P>pSEVA 414 EcoRI and PstI</P></TD><TD width="138"><P>90</P></TD><TD width="81"><P>1.99</P></TD><TD width="92"><P>1.86</P></TD></TR><TR><TD width="155"><P>PcaG EcoRI and PstI</P></TD><TD width="138"><P>63</P></TD><TD width="81"><P>0.97</P></TD><TD width="92"><P>1.74</P></TD></TR><TR><TD width="155"><P>pSEVA 514 EcoRI and PstI</P></TD><TD width="138"><P>19</P></TD><TD width="81"><P>0.30</P></TD><TD width="92"><P>1.41</P></TD></TR><TR><TD width="155"><P>EG</P></TD><TD width="138"><P>9</P></TD><TD width="81"><P>0.17</P></TD><TD width="92"><P>1.62</P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 26/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Liran and Tomer</STRONG></P><P><SPAN style="text-decoration: underline;">Preparing Strep stock for P.putida</SPAN></P><P> </P><P>Stock concentration 100mg/ml</P><P> </P><P>2g powder</P><P>20 ml UPW</P><P> </P><P><SPAN style="text-decoration: underline;">Preparing TET stock for P.putida:</SPAN></P><P> </P><P>Stock concentration 10mg\ml</P><P>0.8g powder</P><P>30ml UPW</P><P> </P><P><SPAN style="text-decoration: underline;">pSEVA plasmids concentration after restriction and cleanup:</SPAN></P><P> </P><P>pSEVA224 8.68 ng\ul</P><P>pSEVA224 after mini prep 84.48 ng\ul</P><P>pSEVA414 6.31 ng\ul</P><P>pSEVA414 after mini prep 3.01 ng\ul</P><P>pSEVA514 after mini prep 63.13 ng\ul</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 25/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Efrat Liran and Tomer</STRONG></P><P><SPAN style="text-decoration: underline;">BioBricks after PCR cleanup:</SPAN></P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="106"><P> </P></TD><TD width="130"><P><STRONG>Concentration ug/ul</STRONG></P></TD><TD width="86"><P><STRONG>260/230</STRONG></P></TD><TD width="86"><P><STRONG>260/280</STRONG></P></TD></TR><TR><TD width="106"><P>C.O</P></TD><TD width="130"><P>11.2</P></TD><TD width="86"><P>0.73</P></TD><TD width="86"><P>1.92</P></TD></TR><TR><TD width="106"><P>F4</P></TD><TD width="130"><P>235.05</P></TD><TD width="86"><P>1</P></TD><TD width="86"><P>1.98</P></TD></TR><TR><TD width="106"><P>F7</P></TD><TD width="130"><P>371.92</P></TD><TD width="86"><P>1.63</P></TD><TD width="86"><P>1.93</P></TD></TR><TR><TD width="106"><P>R4</P></TD><TD width="130"><P>57.18</P></TD><TD width="86"><P>0.38</P></TD><TD width="86"><P>2.18</P></TD></TR><TR><TD width="106"><P>R7</P></TD><TD width="130"><P>146.84</P></TD><TD width="86"><P>0.81</P></TD><TD width="86"><P>1.93</P></TD></TR><TR><TD width="106"><P>pCaH</P></TD><TD width="130"><P>106.59</P></TD><TD width="86"><P>0.86</P></TD><TD width="86"><P>1.98</P></TD></TR><TR><TD width="106"><P>pCaC</P></TD><TD width="130"><P>61.25</P></TD><TD width="86"><P>0.42</P></TD><TD width="86"><P>1.88</P></TD></TR><TR><TD width="106"><P>pCaD</P></TD><TD width="130"><P>27.55</P></TD><TD width="86"><P>0.12</P></TD><TD width="86"><P>2.32</P></TD></TR><TR><TD width="106"><P>pCaG</P></TD><TD width="130"><P>-</P></TD><TD width="86"><P>-</P></TD><TD width="86"><P>-</P></TD></TR><TR><TD width="106"><P>Transporter</P></TD><TD width="130"><P>254.54</P></TD><TD width="86"><P>1.55</P></TD><TD width="86"><P>1.93</P></TD></TR><TR><TD width="106"><P>Pseva 514</P></TD><TD width="130"><P>8.98</P></TD><TD width="86"><P>0.06</P></TD><TD width="86"><P>8.91</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">Ligation for pcaD\pcaH\pacC</SPAN></P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="122"><P> </P></TD><TD width="120"><P>PcaC</P></TD><TD width="120"><P>PcaD</P></TD><TD width="111"><P>PcaH</P></TD><TD width="122"><P>Control</P></TD></TR><TR><TD width="122"><P>Vector</P></TD><TD width="120"><P>2.8ul</P></TD><TD width="120"><P>28ul</P></TD><TD width="111"><P>28ul</P></TD><TD width="122"><P>28ul</P></TD></TR><TR><TD width="122"><P>Insert</P></TD><TD width="120"><P>1.4ul</P></TD><TD width="120"><P>2ul</P></TD><TD width="111"><P>3.1ul</P></TD><TD width="122"><P>-</P></TD></TR><TR><TD width="122"><P>Buffer</P></TD><TD width="120"><P>2ul</P></TD><TD width="120"><P>2ul</P></TD><TD width="111"><P>2ul</P></TD><TD width="122"><P>2ul</P></TD></TR><TR><TD width="122"><P>DDW</P></TD><TD width="120"><P>12.8ul</P></TD><TD width="120"><P>12.2ul</P></TD><TD width="111"><P>11.1ul</P></TD><TD width="122"><P>14.2ul</P></TD></TR><TR><TD width="122"><P>Ligase</P></TD><TD width="120"><P>1ul</P></TD><TD width="120"><P>1ul</P></TD><TD width="111"><P>1ul</P></TD><TD width="122"><P>1ul</P></TD></TR></TBODY></TABLE></DIV><P> </P><P>Overnight 10C</P><P><STRONG><SPAN style="text-decoration: underline;">Restriction</SPAN></STRONG>- Transporter 7k (TPA)</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="297"><P>DNA</P></TD><TD width="298"><P>10ul</P></TD></TR><TR><TD width="297"><P>Buffer 3.1</P></TD><TD width="298"><P>10ul</P></TD></TR><TR><TD width="297"><P>EcoRI</P></TD><TD width="298"><P>1ul</P></TD></TR><TR><TD width="297"><P>PstI</P></TD><TD width="298"><P>1ul</P></TD></TR><TR><TD width="297"><P>DDW</P></TD><TD width="298"><P>78ul</P></TD></TR><TR><TD width="297"><P>Total</P></TD><TD width="298"><P>100ul</P></TD></TR></TBODY></TABLE></DIV><P> </P><P>Restriction time- 60 min in 37 °C</P><P>Inactivation- 20 min in 65°C</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 24/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar B Liran and Tomer</STRONG></P><P>Produced the pSBIC3 plasmid from the starter and cleaned it using mini prep kit in 6 eppendorf:</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="143"><P><SPAN style="text-decoration: underline;"> </SPAN></P></TD><TD width="154"><P><STRONG>Concentration ug/ul</STRONG></P></TD><TD width="149"><P><STRONG>260/230</STRONG></P></TD><TD width="149"><P><STRONG>260/280</STRONG></P></TD></TR><TR><TD width="143"><P>1</P></TD><TD width="154"><P>215</P></TD><TD width="149"><P>2.04</P></TD><TD width="149"><P>1.85</P></TD></TR><TR><TD width="143"><P>2</P></TD><TD width="154"><P>401</P></TD><TD width="149"><P>2.09</P></TD><TD width="149"><P>1.88</P></TD></TR><TR><TD width="143"><P>3</P></TD><TD width="154"><P>272</P></TD><TD width="149"><P>1.88</P></TD><TD width="149"><P>1.84</P></TD></TR><TR><TD width="143"><P>4</P></TD><TD width="154"><P>346</P></TD><TD width="149"><P>2.14</P></TD><TD width="149"><P>1.87</P></TD></TR><TR><TD width="143"><P>5</P></TD><TD width="154"><P>265</P></TD><TD width="149"><P>2.15</P></TD><TD width="149"><P>1.89</P></TD></TR><TR><TD width="143"><P>6</P></TD><TD width="154"><P>263</P></TD><TD width="149"><P>2.12</P></TD><TD width="149"><P>1.87</P></TD></TR></TBODY></TABLE></DIV><P><SPAN style="text-decoration: underline;"> </SPAN></P><P>PCR-Transporter for TPA (location 7K from iGEM plates)</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="198"><P><STRONG> </STRONG></P></TD><TD width="199"><P><STRONG>Transporter</STRONG></P></TD><TD width="198"><P><STRONG>Control</STRONG></P></TD></TR><TR><TD width="198"><P><STRONG>Template</STRONG></P></TD><TD width="199"><P>0.5ul</P></TD><TD width="198"><P>-</P></TD></TR><TR><TD width="198"><P><STRONG>P.RW</STRONG></P></TD><TD width="199"><P>1.5ul</P></TD><TD width="198"><P>0.75ul</P></TD></TR><TR><TD width="198"><P><STRONG>P.FOW</STRONG></P></TD><TD width="199"><P>1.5ul</P></TD><TD width="198"><P>0.75ul</P></TD></TR><TR><TD width="198"><P><STRONG>Hot start</STRONG></P></TD><TD width="199"><P>25ul</P></TD><TD width="198"><P>12.5ul</P></TD></TR><TR><TD width="198"><P><STRONG>DDW</STRONG></P></TD><TD width="199"><P>21.5ul</P></TD><TD width="198"><P>11ul</P></TD></TR><TR><TD width="198"><P><STRONG>Total</STRONG></P></TD><TD width="199"><P>50ul</P></TD><TD width="198"><P>25ul</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">PCR Program:</SPAN></P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="143"><P> </P></TD><TD width="154"><P><STRONG>Temp</STRONG></P></TD><TD width="153"><P><STRONG>Time</STRONG></P></TD><TD width="144"><P><STRONG>Cycle</STRONG></P></TD></TR><TR><TD width="143"><P>1.</P></TD><TD width="154"><P>95°C</P></TD><TD width="153"><P>3 min</P></TD><TD width="144"><P>1</P></TD></TR><TR><TD width="143"><P>2.</P></TD><TD width="154"><P>98°C</P></TD><TD width="153"><P>20 sec</P></TD><TD width="144"><P>25</P></TD></TR><TR><TD width="143"><P>3.</P></TD><TD width="154"><P>65°C</P></TD><TD width="153"><P>15 sec</P></TD><TD width="144"><P>25</P></TD></TR><TR><TD width="143"><P>4.</P></TD><TD width="154"><P>72°C</P></TD><TD width="153"><P>180 sec</P></TD><TD width="144"><P>25</P></TD></TR><TR><TD width="143"><P>5.</P></TD><TD width="154"><P>72°C</P></TD><TD width="153"><P>4 min</P></TD><TD width="144"><P>1</P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 22/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Efrat </STRONG></P><P><STRONG><SPAN style="text-decoration: underline;">Restriction:</SPAN></STRONG></P><P>Plasmid: pSB1C3</P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="92"><P><STRONG>Plasmids</STRONG></P></TD><TD width="86"><P><STRONG>Buffer</STRONG></P></TD><TD width="85"><P><STRONG>EcoRI</STRONG></P></TD><TD width="80"><P><STRONG>PstI</STRONG></P></TD><TD width="82"><P><STRONG>SacI</STRONG></P></TD><TD width="87"><P><STRONG>DDW</STRONG></P></TD><TD width="83"><P><STRONG>Total</STRONG></P></TD></TR><TR><TD width="92"><P>7.06ul</P></TD><TD width="86"><P>10ul</P></TD><TD width="85"><P>1ul</P></TD><TD width="80"><P>1ul</P></TD><TD width="82"><P>1ul</P></TD><TD width="87"><P>79.94ul</P></TD><TD width="83"><P>100ul</P></TD></TR><TR><TD width="92"><P>7.06ul</P></TD><TD width="86"><P>10ul</P></TD><TD width="85"><P>1ul</P></TD><TD width="80"><P>1ul</P></TD><TD width="82"><P>-</P></TD><TD width="87"><P>80.94ul</P></TD><TD width="83"><P>100ul</P></TD></TR><TR><TD width="92"><P>7.06ul</P></TD><TD width="86"><P>10ul</P></TD><TD width="85"><P>-</P></TD><TD width="80"><P>1ul</P></TD><TD width="82"><P>-</P></TD><TD width="87"><P>81.94ul</P></TD><TD width="83"><P>100ul</P></TD></TR><TR><TD width="92"><P>7.06ul</P></TD><TD width="86"><P>10ul</P></TD><TD width="85"><P>1ul</P></TD><TD width="80"><P>-</P></TD><TD width="82"><P>-</P></TD><TD width="87"><P>81.94ul</P></TD><TD width="83"><P>100ul</P></TD></TR><TR><TD width="92"><P>7.06ul</P></TD><TD width="86"><P>10ul</P></TD><TD width="85"><P>1ul</P></TD><TD width="80"><P>-</P></TD><TD width="82"><P>1ul</P></TD><TD width="87"><P>80.94ul</P></TD><TD width="83"><P>100ul</P></TD></TR><TR><TD width="92"><P><STRONG>Gene</STRONG></P></TD><TD width="86"><P><STRONG>Buffer</STRONG></P></TD><TD width="85"><P><STRONG>EcoRI</STRONG></P></TD><TD width="80"><P><STRONG>PstI</STRONG></P></TD><TD width="82"><P><STRONG>SacI</STRONG></P></TD><TD width="87"><P><STRONG>DDW</STRONG></P></TD><TD width="83"><P><STRONG>Total</STRONG></P></TD></TR><TR><TD width="92"><P>10ul</P></TD><TD width="86"><P>10ul</P></TD><TD width="85"><P>1ul</P></TD><TD width="80"><P>1ul</P></TD><TD width="82"><P>-</P></TD><TD width="87"><P>78ul</P></TD><TD width="83"><P>100ul</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><STRONG><SPAN style="text-decoration: underline;">Concentration after clean up:</SPAN></STRONG></P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="297"><P><STRONG>Gene</STRONG></P></TD><TD width="298"><P><STRONG>Concentration ng/ul</STRONG></P></TD></TR><TR><TD width="297"><P>pcaH</P></TD><TD width="298"><P>5.34</P></TD></TR><TR><TD width="297"><P>pcaG</P></TD><TD width="298"><P>8</P></TD></TR><TR><TD width="297"><P>pcaD</P></TD><TD width="298"><P>-5.34</P></TD></TR><TR><TD width="297"><P>pcaC</P></TD><TD width="298"><P>19.99</P></TD></TR><TR><TD width="297"><P>C.O</P></TD><TD width="298"><P>-5.91</P></TD></TR><TR><TD width="297"><P>F4</P></TD><TD width="298"><P>9.59</P></TD></TR><TR><TD width="297"><P>F7</P></TD><TD width="298"><P>18.55</P></TD></TR><TR><TD width="297"><P>R4</P></TD><TD width="298"><P>12.10</P></TD></TR><TR><TD width="297"><P>R7</P></TD><TD width="298"><P>44.28</P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 21/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben and Inbar B</STRONG></P><P><SPAN style="text-decoration: underline;">Runing pSEVA vectors on gel in order to confirm thire exsistens in old sempels</SPAN></P><P> </P><P><SPAN style="text-decoration: underline;">PCR clean up</SPAN></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="201"><P><STRONG>gene</STRONG></P></TD><TD width="201"><P><STRONG>concentration(</STRONG><STRONG>ng/ul)</STRONG></P></TD><TD width="201"><P><STRONG>260\230</STRONG></P></TD></TR><TR><TD width="201"><P>pSB1C3 cut EcorI PstI</P></TD><TD width="201"><P>7</P></TD><TD width="201"><P>0.02</P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 20/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar B, Ben, Liran Eyal and Dor</STRONG></P><P>Prepering XL1 heat shock competent cells, after the previous XL1 competent cells fail at the test kit.</P><P><SPAN style="text-decoration: underline;">Repeat PCR to amplify TphA2 for Gibson assembly</SPAN></P><P>template- old PCR product</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="91"><P><STRONG> </STRONG></P></TD><TD width="91"><P><STRONG>Annealing temperature</STRONG></P><P><STRONG>(C°)</STRONG></P></TD><TD width="91"><P><STRONG>Upw (ul)</STRONG></P></TD><TD width="91"><P><STRONG>Pr fwd (ul)</STRONG></P></TD><TD width="91"><P><STRONG>Pr rev</STRONG></P></TD><TD width="91"><P><STRONG>template</STRONG></P></TD><TD width="91"><P><STRONG>KAPA Hiti hot start (ul)</STRONG></P></TD></TR><TR><TD width="91"><P><STRONG>TphA2</STRONG></P></TD><TD width="91"><P>68</P></TD><TD width="91"><P>21</P></TD><TD width="91"><P>1.5</P></TD><TD width="91"><P>1.5</P></TD><TD width="91"><P>1</P></TD><TD width="91"><P>25</P></TD></TR><TR><TD width="91"><P><STRONG>TphA2X</STRONG></P></TD><TD width="91"><P>68</P></TD><TD width="91"><P>5.5</P></TD><TD width="91"><P>0.38</P></TD><TD width="91"><P>0.38</P></TD><TD width="91"><P>-</P></TD><TD width="91"><P>6.25</P></TD></TR></TBODY></TABLE></DIV><P> </P><P> </P><P>Runing pSEVA vectors on gel in order to confirm thire exsistens in old sempels</P><P> </P><P><SPAN style="text-decoration: underline;">Repeat PCR to amplify LCC variants for BioBriks because efret used all the sempels</SPAN></P><P>Primers 10 uM.</P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="89"><P><STRONG> </STRONG></P></TD><TD width="89"><P><STRONG>Annealing temperature</STRONG></P><P><STRONG> (</STRONG><STRONG>C°</STRONG><STRONG>)</STRONG></P></TD><TD width="89"><P><STRONG>Upw (ul)</STRONG></P></TD><TD width="89"><P><STRONG>Pr fwd (ul)</STRONG></P></TD><TD width="89"><P><STRONG>Pr rev</STRONG></P></TD><TD width="89"><P><STRONG>template</STRONG></P></TD><TD width="89"><P><STRONG>KAPA Hiti hot start (ul)</STRONG></P></TD></TR><TR><TD width="89"><P><STRONG>C.O</STRONG></P></TD><TD width="89"><P>58.4</P></TD><TD width="89"><P>21</P></TD><TD width="89"><P>1.5</P></TD><TD width="89"><P>1.5</P></TD><TD width="89"><P>1</P></TD><TD width="89"><P>25</P></TD></TR><TR><TD width="89"><P><STRONG>F4</STRONG></P></TD><TD width="89"><P>58.4</P></TD><TD width="89"><P>21</P></TD><TD width="89"><P>1.5</P></TD><TD width="89"><P>1.5</P></TD><TD width="89"><P>1</P></TD><TD width="89"><P>25</P></TD></TR><TR><TD width="89"><P><STRONG>F7</STRONG></P></TD><TD width="89"><P>58.4</P></TD><TD width="89"><P>21</P></TD><TD width="89"><P>1.5</P></TD><TD width="89"><P>1.5</P></TD><TD width="89"><P>1</P></TD><TD width="89"><P>25</P></TD></TR><TR><TD width="89"><P><STRONG>R4</STRONG></P></TD><TD width="89"><P>58.4</P></TD><TD width="89"><P>21</P></TD><TD width="89"><P>1.5</P></TD><TD width="89"><P>1.5</P></TD><TD width="89"><P>1</P></TD><TD width="89"><P>25</P></TD></TR><TR><TD width="89"><P><STRONG>R7</STRONG></P></TD><TD width="89"><P>58.4</P></TD><TD width="89"><P>21</P></TD><TD width="89"><P>1.5</P></TD><TD width="89"><P>1.5</P></TD><TD width="89"><P>1</P></TD><TD width="89"><P>25</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">Activity test for cutinase:</SPAN></P><P>We tasted cleaned cutinase F4, R7 and WT, and uncleaned cutinse F4, R7 and WT in this test.</P><P>We used 96DW and a plate reader.</P><P>We added 100ul pNPB without cutinase, Uncleaned cutinase WT with pNPB (100ul and 100ul), Uncleaned cutinase R7 with pNPB (100ul and 100ul), Uncleaned cutinase F4 with pNPB (100ul and 100ul), pNPB and cleaned cutinase WT(100ul an 100ul), pNPB and cleaned cutinase R7(1 00ul an 100ul) and pNPB and cleaned cutinase F4(100ul an 100ul). ). [We duplicate every sample]</P><P>25/9</P><UL><LI>We prepared a stock of TPA 50 ml 20mM according to our protocol (TPA detection protocol)/ after that, we diluted the stock to get concentration of 15mM, 10mM, 5mM, 1mM, 0.5mM, 0.1mM, 0.05mM, 0.01mM, 0.005mM, 0.001mM.</LI></UL><P>We added 200uL from each concentration to a different wall in 96W anti-UV plate.</P><P>We exposed the plate to UV according to the protocol and search for fluorescence emission we saw in the literature.</P><P>After few days of testing this method we wrote the TPA detecting assay.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 19/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar B, Ben and Efret</STRONG></P><P><SPAN style="text-decoration: underline;">Produced plasmids from the starters using mini prep kit.</SPAN></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="150"><P><STRONG>plasmid</STRONG></P></TD><TD width="150"><P><STRONG>concentration(</STRONG><STRONG>ng/ul)</STRONG></P></TD><TD width="150"><P><STRONG>260\230</STRONG></P></TD><TD width="150"><P><STRONG>260\280</STRONG></P></TD></TR><TR><TD width="150"><P>pSEVA 514</P></TD><TD width="150"><P>151</P></TD><TD width="150"><P>2.00</P></TD><TD width="150"><P>2.05</P></TD></TR><TR><TD width="150"><P>pSB1C col1</P></TD><TD width="150"><P>272</P></TD><TD width="150"><P>2.15</P></TD><TD width="150"><P>1.93</P></TD></TR><TR><TD width="150"><P>pSB1C col2</P></TD><TD width="150"><P>282</P></TD><TD width="150"><P>2.19</P></TD><TD width="150"><P>1.91</P></TD></TR></TBODY></TABLE></DIV><P> </P><P><SPAN style="text-decoration: underline;">PCR to amplify TphA1, TphA2, TphA3 and TphB for Gibson assembly</SPAN></P><P> </P><P><SPAN style="text-decoration: underline;">iGEM competent cell test kit</SPAN></P><P><STRONG>XL1</STRONG></P><P><STRONG>BL21</STRONG></P><P> </P><P><SPAN style="text-decoration: underline;">Ligation of BioBriks inserts and pSB1C3 vector</SPAN></P><P> </P><P>50ng vectur cut with EcorI and PstI</P><P>150ng insert cut with EcorI and PstI</P><P>0.5ul T4 DNA ligase Buffer</P><P>1ul T4 DNA ligase</P><P>add UPW for total volum of 50ul</P><P> </P><P>16C over nighet.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 18/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Tomer and Liran</STRONG></P><P><SPAN style="text-decoration: underline;">PCR to amplify 7K transporter from plasmid</SPAN></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="142"><P><STRONG> </STRONG></P></TD><TD width="95"><P><STRONG>Annealing temperature</STRONG></P><P><STRONG> (C°)</STRONG></P></TD><TD width="95"><P><STRONG>Upw (ul)</STRONG></P></TD><TD width="95"><P><STRONG>Pr fwd (ul)</STRONG></P></TD><TD width="95"><P><STRONG>Pr rev</STRONG></P></TD><TD width="95"><P><STRONG>template</STRONG></P></TD><TD width="95"><P>KAPA Hiti hot start (ul)</P></TD></TR><TR><TD width="142"><P><STRONG>7K transporter</STRONG></P></TD><TD width="95"><P>68</P></TD><TD width="95"><P>21</P></TD><TD width="95"><P>1.5</P></TD><TD width="95"><P>1.5</P></TD><TD width="95"><P>1</P></TD><TD width="95"><P>25</P></TD></TR><TR><TD width="142"><P><STRONG>RFP as posetiv control with pSV1C as template</STRONG></P></TD><TD width="95"><P>68</P></TD><TD width="95"><P>21</P></TD><TD width="95"><P>1.5</P></TD><TD width="95"><P>1.5</P></TD><TD width="95"><P>1</P></TD><TD width="95"><P>25</P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 17/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal and Dor</STRONG></P><P><SPAN style="text-decoration: underline;">Dialysis experiment:</SPAN></P><P>We prepared the dialysis experiment according to the “Dialysis bag experiment” using the starters we raised two day ago. We tested OD every day.</P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="146"><P>Date</P></TD><TD width="147"><P>time</P></TD><TD width="149"><P>Outside the dialysis bag</P></TD><TD width="149"><P>Inside the dialysis bag</P></TD></TR><TR><TD width="146"><P>17/9</P></TD><TD width="147"><P>13:00</P></TD><TD width="149"><P>0.09</P></TD><TD width="149"><P>0.108</P></TD></TR><TR><TD width="146"><P>18/9</P></TD><TD width="147"><P>13:30</P></TD><TD width="149"><P>0.1</P></TD><TD width="149"><P>0.31</P></TD></TR><TR><TD width="146"><P>19/9</P></TD><TD width="147"><P>12:15</P></TD><TD width="149"><P>0.1</P></TD><TD width="149"><P>0.4</P></TD></TR><TR><TD width="146"><P>20/9</P></TD><TD width="147"><P>14:00</P></TD><TD width="149"><P>0.107</P></TD><TD width="149"><P>0.42</P></TD></TR><TR><TD width="146"><P>21/9</P></TD><TD width="147"><P>15:00</P></TD><TD width="149"><P>0.1</P></TD><TD width="149"><P>0.24( the bag was fallen)</P></TD></TR><TR><TD width="146"><P>22/9</P></TD><TD width="147"><P>11:00</P></TD><TD width="149"><P>0.2</P></TD><TD width="149"><P>0.4</P></TD></TR><TR><TD width="146"><P>23/9</P></TD><TD width="147"><P>14:00</P></TD><TD width="149"><P>0.2</P></TD><TD width="149"><P>0.6</P></TD></TR><TR><TD width="146"><P>24/9</P></TD><TD width="147"><P>13:30</P></TD><TD width="149"><P>0.5</P></TD><TD width="149"><P>0.64</P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 15/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Idit, Eyal and Dor</STRONG></P><P>the dialysis bag content was loded onto a cation seperation colon</P><P> </P><P>We prepared starters with e-coli and pACYC.</P><P> After one week we saw that the M9 outside of the dialysis bag was contaminated with e coli.</P><P> </P><P><SPAN style="text-decoration: underline;">Transformation of pSEVA plasmids to P.putida KT2440 competent cells and groing cells on diffrent antibiotics in order to check the competent cells</SPAN></P><UL style="list-style: disc;"><LI>Compatent P.putida KT2440 from -80C to ice for 10min.</LI><LI>1 ul from each plasmid to each tube and a few tubes without plasmid(negative control on diffrent antibiotics)</LI><LI>mix with tip</LI><LI>incubate in ice for 20 min</LI><LI>heat shock 42C 3min.</LI><LI>incubate in ice for 3min</LI><LI>SOC 900ul without antibiotic</LI><LI>37C for 2h</LI><LI>centrifugate 500rpm 2min</LI><LI>carrying of 900ul liquid from each tube.</LI><LI>mix the residue with the liquid</LI><LI>sowin on LB plates with relavent antibiotic.</LI></UL><P> </P><P>number of times that bacterias grows on difrent antibiotics, after trensformesuon of diffrent plasmids.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 14/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Efrat</STRONG></P><P>protein seperation</P><P> </P><P>the solution was centefuges in 16000g  for an hour</P><P>the sediment was re-suspendent in TAE buffer pH 7.</P><P>the solution was inserted into a dialysis bag over night.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 13/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Efrat</STRONG></P><P><SPAN style="text-decoration: underline;">Restriction reaction for BioBriks inserts</SPAN></P><P> </P><P>250ng DNA (insert or PSB1C3 vector)</P><P>3ul NEB Buffer II</P><P>0.5ul PstI</P><P>0.5ul EcorI</P><P>add UPW for total volum of 20ul</P><P> </P><P>1h 37C</P><P>20min 80C</P><P> </P><P>protein seperation</P><P> </P><P>500 ml of each varient was centrifuged in 8000g for 10 minutes.</P><P>amonium solfate was added to the supernatant and incubated over night on a magnetic plate with a stirerer.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 12/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Efrat</STRONG></P><P><SPAN style="text-decoration: underline;">Repeat PCR to amplify PcaH, PcaG ,PcaD, PcaC from P.putida KT2440 </SPAN></P><P> </P><P><SPAN style="text-decoration: underline;">PCR clean up</SPAN></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="201"><P><STRONG>gene</STRONG></P></TD><TD width="201"><P><STRONG>concentration(</STRONG><STRONG>ng/ul)</STRONG></P></TD><TD width="201"><P><STRONG>260\230</STRONG></P></TD></TR><TR><TD width="201"><P>PcaH</P></TD><TD width="201"><P>86</P></TD><TD width="201"><P>1.03</P></TD></TR><TR><TD width="201"><P>PcaG</P></TD><TD width="201"><P>96</P></TD><TD width="201"><P>0.98</P></TD></TR><TR><TD width="201"><P>PcaD</P></TD><TD width="201"><P>25</P></TD><TD width="201"><P>1.32</P></TD></TR><TR><TD width="201"><P>PcaC</P></TD><TD width="201"><P>81</P></TD><TD width="201"><P>1.39</P></TD></TR></TBODY></TABLE></DIV><P>protein seperation</P><P> </P><P>the starters were diluted in 500 ml LB</P><P>After achieving 0.9-1 OD values IPTG was added and the incubated overnight in 37 degrees Celsius and agitation of 300 rpm</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 11/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar B and Efrat</STRONG></P><P><U>Prepering P.putida KT2440 competent cells:</U></P><P>protein seperation</P><P> </P><P>Three variants were chosen CO, F4, R7 for separation.</P><P>First a transformation into BL21 competents was performed</P><P>1 µl of plasmid was added into 50 µl of competent batch and incubated on ice for 30 min.</P><P>Heat shock for 30 sec in 42 degrees Celsius</P><P>Incubation for 3 min.</P><P>950 µl of SOC medium was add and placed into incubation for an hour in 37 degrees Celsius.</P><P>After incubation the bacteria were precipitated and them re-suspended in 200 µl in LB</P><P>Starters of 5 ml were made from the transformation to use in the following day</P><P> </P><P><U>PCR clean up</U></P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="201"><P><STRONG>gene</STRONG></P></TD><TD width="201"><P><STRONG>concentration(</STRONG><STRONG>ng/ul)</STRONG></P></TD><TD width="201"><P><STRONG>260\230</STRONG></P></TD></TR><TR><TD width="201"><P>PcaH</P></TD><TD width="201"><P>1</P></TD><TD width="201"><P>0.01</P></TD></TR><TR><TD width="201"><P>PcaG</P></TD><TD width="201"><P>28</P></TD><TD width="201"><P>0.6</P></TD></TR><TR><TD width="201"><P>PcaD</P></TD><TD width="201"><P>33</P></TD><TD width="201"><P>0.27</P></TD></TR><TR><TD width="201"><P>PcaC</P></TD><TD width="201"><P>30</P></TD><TD width="201"><P>0.2</P></TD></TR><TR><TD width="201"><P>CO</P></TD><TD width="201"><P>133</P></TD><TD width="201"><P>0.99</P></TD></TR><TR><TD width="201"><P>F4</P></TD><TD width="201"><P>123</P></TD><TD width="201"><P>0.93</P></TD></TR><TR><TD width="201"><P>F7</P></TD><TD width="201"><P>152</P></TD><TD width="201"><P>0.63</P></TD></TR><TR><TD width="201"><P>R4</P></TD><TD width="201"><P>140</P></TD><TD width="201"><P>0.38</P></TD></TR><TR><TD width="201"><P>R7</P></TD><TD width="201"><P>157</P></TD><TD width="201"><P>0.47</P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 10/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal and Dor</STRONG></P><P><STRONG>Activity test for cutinase:</STRONG></P><P><SPAN style="font-weight: 400;">We tasted cleaned cutinase WT and uncleaned cutinase WT in this test.</SPAN></P><P><SPAN style="font-weight: 400;">We used 96DW and a plate reader.</SPAN></P><P><SPAN style="font-weight: 400;">In one wall we added 100ul pNPB without cutinase. In the second wall we added uncleaned cutinase WT with pNPB (100ul and 100ul), and in the third wall we added pNPB and cleaned cutinase WT (100ul an 100ul). [We duplicate every sample]</SPAN></P><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Dialysis experiment:</STRONG></P><P><SPAN style="font-weight: 400;">Transformation was done to insert pACYC with CMP resistant and cutinase WT gene into BL21 </SPAN><EM><SPAN style="font-weight: 400;">e-coli</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></P><P><SPAN style="font-weight: 400;">We prepared starters with this e-coli.</SPAN></P><P><STRONG><SPAN style="font-weight: 400;">We prepared M9+PET (we crashed the PET with coffee grinder).</SPAN></STRONG></P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 06/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar B and Efrat</STRONG></P><P><SPAN style="text-decoration: underline;">PCR to amplify PcaH, PcaG ,PcaD, PcaC from P.putida KT2440 (05.09.16): for BioBriks </SPAN></P><P>Primers 10 uM template- toch colony, then reaction.</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD><P><SPAN style="font-weight: 400;">KAPA Hiti hot start (ul)</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">Pr rev</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">Pr fwd (ul)</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">Upw (ul)</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">Annealing temperature</SPAN></P><P><SPAN style="font-weight: 400;"> (</SPAN><SPAN style="font-weight: 400;">C°</SPAN><SPAN style="font-weight: 400;">)</SPAN></P></TD><TD> </TD><TD> </TD></TR><TR><TD><P><SPAN style="font-weight: 400;">25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5 PcaH</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5 PcaH</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">22</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">61.6</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">PcaH</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">22</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">PcaG</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">2</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">22</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">62.3</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">PcaD</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">3</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">22</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">PcaC</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">4</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">6.25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">5.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">61.6</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">HX</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">5</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">6.25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">5.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">GX</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">6</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">6.25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">5.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">62.3</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">DX</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">7</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">6.25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">5.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">CX</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">8</SPAN></P></TD></TR></TBODY></TABLE></DIV><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD><P><SPAN style="font-weight: 400;">KAPA Hiti hot start (ul)</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">template</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">Pr rev</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">Pr fwd (ul)</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">Upw (ul)</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">Annealing temperature</SPAN></P><P><SPAN style="font-weight: 400;"> (</SPAN><SPAN style="font-weight: 400;">C°</SPAN><SPAN style="font-weight: 400;">)</SPAN></P></TD><TD> </TD></TR><TR><TD><P><SPAN style="font-weight: 400;">25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5 </SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5 </SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">21</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">C.O</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">21</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">F4</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">21</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">F7</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">21</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">R4</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">1.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">21</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">R7</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">6.25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">-</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">5.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">C.OX</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">6.25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">-</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">5.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">F4X</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">6.25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">-</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">5.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">F7X</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">6.25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">-</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">5.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">R4X</SPAN></P></TD></TR><TR><TD><P><SPAN style="font-weight: 400;">6.25</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">-</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">0.38</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">5.5</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">58.4</SPAN></P></TD><TD><P><SPAN style="font-weight: 400;">R7X</SPAN></P></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 04/09/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Everybody!</STRONG></P><P>Successfully ligated:</P><P> </P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD width="301"><P><STRONG>Sample</STRONG></P></TD><TD width="301"><P>Ligated to pSB1C3</P></TD></TR><TR><TD width="301"><P><STRONG>pcaC</STRONG></P></TD><TD width="301"><P> </P></TD></TR><TR><TD width="301"><P><STRONG>pcaD</STRONG></P></TD><TD width="301"><P>Yes (Sample 2)</P></TD></TR><TR><TD width="301"><P><STRONG>pcaG</STRONG></P></TD><TD width="301"><P> </P></TD></TR><TR><TD width="301"><P><STRONG>pcaH</STRONG></P></TD><TD width="301"><P> </P></TD></TR><TR><TD width="301"><P><STRONG>CO</STRONG></P></TD><TD width="301"><P> </P></TD></TR><TR><TD width="301"><P><STRONG>R4</STRONG></P></TD><TD width="301"><P> </P></TD></TR><TR><TD width="301"><P><STRONG>R7</STRONG></P></TD><TD width="301"><P>Yes (Sample 6)</P></TD></TR><TR><TD width="301"><P><STRONG>F4</STRONG></P></TD><TD width="301"><P> </P></TD></TR><TR><TD width="301"><P><STRONG>F7</STRONG></P></TD><TD width="301"><P> </P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV id="August" class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 29/08/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Everybody!</STRONG></P><STRONG>we made PET plates:</STRONG> For 1L:
<OL><LI>200ml M9 salts</LI><LI>2ml 1M mgSO4</LI><LI>20ml glucose 20%</LI><LI>100ul 1M CaCl2</LI><LI>(5ml soft agar for each plate)</LI></OL><STRONG>the plate that we made:</STRONG><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD>50</TD><TD>100</TD><TD>250</TD><TD>500</TD><TD>no PET</TD><TD>Glucose</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 17/08/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Everybody!</STRONG></P>
We concentrated the cutinase with centricons.</DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 11/08/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Everybody!</STRONG></P><UL style="list-style: disc;"><LI>we made M9 plates without carbon source.</LI><LI>we made a new stock for chloramphenicol.</LI><LI>we sow from the rhodococus experiment on natrient broth plates.</LI></UL><STRONG>gel weight:</STRONG><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TD>pcaC</TD><TD>pcaD</TD><TD>pcaG</TD><TD>pcaH</TD><TD>CO</TD><TD>R4</TD><TD>R7</TD><TD>F4</TD><TD>F7</TD></TR><TR><TD>0.2816</TD><TD>0.2801</TD><TD>0.1882</TD><TD>0.2811</TD><TD>0.3890</TD><TD>0.2891</TD><TD>0.2882</TD><TD>0.3887</TD><TD>0.2855</TD></TR></TBODY></TABLE></DIV><STRONG>Nano drop sup:</STRONG> (calibration by pACYC)
<DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Cutinase variants</TH><TH>Protein concentration</TH></TR><TR><TD>WT</TD><TD>0.83</TD></TR><TR><TD>CO</TD><TD>0.43</TD></TR><TR><TD>R4</TD><TD>-0.21</TD></TR><TR><TD>F4</TD><TD>-0.37</TD></TR><TR><TD>R7</TD><TD>-0.09</TD></TR><TR><TD>F7</TD><TD>-0.95</TD></TR></TBODY></TABLE></DIV> protein concentration after centrecon:
<DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Cutinase variants</TH><TH>Protein concentration</TH></TR><TR><TD>WT</TD><TD>4.46</TD></TR><TR><TD>R4</TD><TD>2.41</TD></TR><TR><TD>F4</TD><TD>2.23</TD></TR><TR><TD>R7</TD><TD>2.54</TD></TR><TR><TD>F7</TD><TD>2.41</TD></TR><TR><TD>pACYC</TD><TD>3.9</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 10/08/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Everybody!</STRONG></P><STRONG>BL21 growth and induction for expression LCC for activity test:</STRONG><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Cutinase variants</TH><TH>14:37</TH><TH>15:15</TH></TR><TR><TD>pACYC</TD><TD>0.215</TD><TD>0.458</TD></TR><TR><TD>WT</TD><TD>0.267</TD><TD>0.497</TD></TR><TR><TD>CO</TD><TD>0.236</TD><TD>0.440</TD></TR><TR><TD>R4</TD><TD>0.221</TD><TD>0.413</TD></TR><TR><TD>F4</TD><TD>0.236</TD><TD>0.446</TD></TR><TR><TD>R7</TD><TD>0.195</TD><TD>0.376</TD></TR><TR><TD>F7</TD><TD>0.208</TD><TD>0.401</TD></TR></TBODY></TABLE></DIV>
15:35-5ul IPTG 1M for each 5ml <STRONG>preparation of IPTG:</STRONG><UL style="list-style: disc;"><LI>0.1gr +420ul DDW</LI><LI>fiber with 0.22um filter.</LI></UL></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 09/08/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Everybody</STRONG></P><STRONG>Cutinase experiment:</STRONG><STRONG>nano-drop protein measurement (calibration by pACYC)</STRONG><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Cutinase variants</TH><TH>Protein concentration</TH></TR><TR><TD>pACYC</TD><TD>0</TD></TR><TR><TD>WT</TD><TD>0.22</TD></TR><TR><TD>CO</TD><TD>2.16</TD></TR><TR><TD>R4</TD><TD>-1.68</TD></TR><TR><TD>F4</TD><TD>-0.37</TD></TR><TR><TD>R7</TD><TD>0.25</TD></TR><TR><TD>F7</TD><TD>0.4</TD></TR><TR><TD>F4</TD><TD>0.14</TD></TR><TR><TD>BL21</TD><TD>0.76</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 070816</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Everybody!</STRONG></P><STRONG>Competent cells</STRONG><OL><LI>5ml LB were taken from erlenmeyer 500ml for calibration the spectophotometer before O.D test.</LI><LI>dilution 5ml BL21 culture starters to 500 ml LB.</LI><LI>when the O.D reach to 0.4-0.6 we mix and save in ice for 10min</LI><LI>we transfer the LB to surbal tubes.</LI><LI>centrifuge 10 min, 6000 rpm, 4C</LI><LI>we removed the sup and gentle fluidization in 25ml solution A for 125ml culture.</LI><LI>30min incubation in ice.</LI><LI>centrifuge 5min, 5000, 4C</LI><LI>gentle fluidization all the plet in 10ml solution B.</LI><LI>we divided the solution to ependorf, 50 ul for each ependorf and throw it quickly to liquid nitrogen.</LI></OL><STRONG>transformation for each LCC variants and control for activity test:</STRONG><OL><LI>we took 8 tubes of competent BL21.</LI><LI>1ul from each plasmid for each tubes and one tube without plasmid.</LI><LI>mix with tip.</LI><LI>incubation in ice for 20 min.</LI><LI>heat shock 30 sec 42C.</LI><LI>incubation in ice for 3 min.</LI><LI>800ul SOC without antibiotic.</LI><LI>1h in 37C.</LI><LI>100ul from each tube to 5ml LB+5ul cmp. for negative control-plate without antibiotic.</LI><LI>incubation 37C O/N.</LI></OL></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 01/08/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Everybody!</STRONG></P><STRONG>the starters diluted to a new starter. O.D tests before the induction:</STRONG><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Cutinase variants</TH><TH>O.D 14:00</TH></TR><TR><TD>pACYC</TD><TD>0.286</TD></TR><TR><TD>WT</TD><TD>0.240</TD></TR><TR><TD>CO</TD><TD>0.239</TD></TR><TR><TD>R4</TD><TD>0.244</TD></TR><TR><TD>F4</TD><TD>0.250</TD></TR><TR><TD>R7</TD><TD>0.252</TD></TR><TR><TD>F7</TD><TD>0.225</TD></TR><TR><TD>F4</TD><TD>0.14</TD></TR><TR><TD>BL21</TD><TD>0.438</TD></TR></TBODY></TABLE></DIV><P>when we checked the O.D at 15:30 all the samples O.D where more than 1.0. we assumed that something is wrong with this experiment, maybe the O.D check in 14:00</P></DIV></DIV></DIV></DIV><DIV id="July" class="container-fluid"><DIV class="row"><DIV class="col-lg-8"><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 31/07/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben</STRONG></P><P>we made starters for LCC activity tests. we are doing the same test like we have done before.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 27/07/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben</STRONG></P><P>protocol for comasi x10:</P><OL><LI>30 gr Tris base.</LI><LI>144 gr glycine</LI><LI>10 gr SDS</LI></OL><P>we dissolve those component in 800ml. after we dissolove all the component we complete the solution to 1000 ml.</P><P>Experiment order:</P><OL><LI>we need to grow up 5ml varients, W.T, and only plasmid(according to the protocol)</LI><LI>after 1 day we disolve and collect the sup and transfer it to centricon.</LI><LI>we concentrated till 0.5ml(x10)</LI><LI>we swiched the buffer with another Tris</LI><LI>we centrifuge every 5 min for keep the materials in the solution</LI></OL></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 26/07/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Tomer</STRONG></P><P>pNPB testing according to the protcol</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>WT</TH><TD>0.489</TD></TR><TR><TH>CO</TH><TD>0.54</TD></TR><TR><TH>F7</TH><TD>0.5</TD></TR><TR><TH>R7</TH><TD>0.52</TD></TR></TBODY></TABLE></DIV><P>O.D testing for LCC varients</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>WT</TH><TD>0.514</TD></TR><TR><TH>F7</TH><TD>0.525</TD></TR><TR><TH>BL21</TH><TD>0.67</TD></TR></TBODY></TABLE></DIV><P><SPAN style="text-decoration: underline;"> biobrick primers arrived:</SPAN>for each primer we added DDW as writen in the form to get the concentration of 100mM. we diluted each primer to a new eppendorf to get a concentration of 10mM by mix 5ul primer+45ul DDW.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 18/07/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Tomer</STRONG></P><P>activity test like we had done before.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 17/07/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Tomer</STRONG></P><P>transformation for all the varients +BL21 without control. we added 16 starters ( 2 for each plasmid+2 control)- 8 pNPB tubes, 8 PET tubes.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 11/07/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><P><SPAN style="text-decoration: underline;">Turbidity check and induction:</SPAN>7 ml LB+ 7ul CP+ 70ul from each starter. 200 rpm, 37C. dilution of x2→ 1 ml culture+ 1 ml LB</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH> </TH><TH>11:40</TH><TH>12:40</TH></TR><TR><TH>LCC WT</TH><TD>0.117x2</TD><TD>0.371x2</TD></TR><TR><TH>pACYC</TH><TD>0.116x2</TD><TD>0.360x2</TD></TR><TR><TH>BL21</TH><TD>0.153x2</TD><TD>0.450x2</TD></TR></TBODY></TABLE></DIV><P>at 12:40→ we added 1M IPTG for each tube.</P><P><SPAN style="text-decoration: underline;"> Activity test:</SPAN></P><OL><LI>centrifuge each culture with 4C 30min 7100g. we transfer thw sup for new tubes.</LI><LI>A. we had made stock of pNPB 100 mM→ 17.5 ul pNPB+ 982.5 ul analytical ethanol B. we had made Tris pH=8 0.25mM→ dilution Tris 1M- 1ml to 39ml UPW</LI><LI>we added 100 ul of the sup near the plate reader with multichannel</LI><LI>we prepared pNBP: for concentration A- 50 ul from the stock to 9.95ml Tris for concentration B-100 ul from the stock to 9.9ml Tris</LI><LI>we added pNPB for each tube. 100 ul for each tube</LI><LI>checking absorption in 405nm per min for 50 min</LI></OL></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 10/07/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><P>transformation for activity test LCC, plasmid pACYC without insert and pACYC LCC WT C6: The aim of this test- transformation for compatent bacteria BL21 and activity test will check how much pNPB we need to use in further experiment.</P><UL style="list-style: disc;"><LI>3 tubes, compatent BL21 for -80C to ice for 10min.</LI><LI>1 ul from each plasmid to each tube and 1 tube without plasmid(negative control)</LI><LI>mix with tip</LI><LI>incubate in ice for 20 min</LI><LI>heat shock 42C 30sec</LI><LI>incubate in ice for 3min</LI><LI>SOC 900ul without antibiotic</LI><LI>37C for 1h</LI><LI>100ul from each tubes to LB 5ml+cmp 5 ul</LI><LI>centrifugate 500rpm 2min</LI><LI>carrying of 800ul liquid from each tube</LI><LI>mix the residue with the liquid</LI><LI>sowing LB+cmp</LI><LI>10 min absorption RT</LI><LI>reverse the plates and incubate in 37C</LI></UL></DIV></DIV></DIV></DIV><DIV id="June" class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 23/06/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben and Dar</STRONG></P><P>glycerol stock protocol:</P><UL style="list-style: disc;"><LI>preparing glycerol:
</LI><LI>add 10 ml LB to 10 ml glycerol</LI></UL><LI>preparing the stock:
</LI><LI>take 500 ul of starter (with required plasmid transfected)</LI><LI>add 500 ul of the glycerol (from stck)</LI><LI>freeze in -80 C</LI></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 15/06/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben and Dar</STRONG></P><DIV class="row"><DIV class="col-lg-7"><P>We isolated the plasmids from each cell culture using Geneaid Again we incubated them with XhoI and XbaI (same protocol as on 14/06) We ran the samples on a 1% agarose gel with EtBr but using a new loading buffer. In lanes 6,7,8 we can see good results. Lane 6 shows the plasmid circular and uncut. Lane 7 shows the plasmid cut in one location using XbaI. Lane 8 shows the plasmid circular and uncut (as it does not have a XhoI restriction site).</P></DIV></DIV><P>starters growth for activity testing of LCC: LB 50 ml+ CP 50 ul. each tube will get 5 ml. In addition, LB 5ml without antibiotic for BL21 growth without insertion. For each tube. we added colony with another varient+2 control tube. the colony that we are testing: pACYC LCC WT C6 pACYC LCC C.O C4 pACYC LCC F4 C2 pACYC LCC F7 C3 pACYC LCC R4 C7 pACYC LCC R7 C2 pACYC no insert BL21 (without antibiotic) </P><P>Turbidity check and induction: For LB 7 ml + CP 7ul we added 70 ul for each starter. ( 37C 200 rpm)</P><DIV class="row"><DIV class="col-lg-2"> </DIV></DIV><P>PET degradation test: we cheked if there is any change in PET wight after one day:</P><DIV class="row"><DIV class="col-lg-2"> </DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 14/06/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben and Dar</STRONG></P><DIV class="row"><DIV class="col-lg-7"><P>We isolated the plasmids from each cell culture using Geneaid Presto mini plasmid kit. We Incubated each of the plasmids with XhoI and XbaI restriction enzymes (2 hours incubation in 37 degrees and 20 minutes in 65 degrees). We ran the results in a 1% agarose gel with EtBr. Both plasmids show on the gel uncut (lanes 2-3). We have no ability to determine their size (as they are circular). We put new starters for another preparation of plasmid DNA.</P></DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 13/06/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben and Dar</STRONG></P><P>We incubated starters for the pSEVA224 and pSEVA414 in 30 degrees.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 11/06/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal and inbar b</STRONG></P><P>compentent cells prepretion</P><DIV class="row"><DIV class="col-lg-2"> </DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 10/06/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><P>preparation of TRIS-HCL:</P><OL><LI>for prvpB degredation---&gt; C1V1=C2V2→ 1M*V1= 25mM*9.9ml→ V1=247.5 ul, DDW=9.6525ml</LI><LI>for PET degredation---&gt; C1V1=C2V2→ 1M*V1= 500mM*2ml→ V1=1ml, DDW=1ml</LI></OL><P>preparation of pNP-B:</P><UL style="list-style: disc;"><LI>pNPB - 35ul -----------&gt; 1 ml of pNPB ---------&gt; 100 mM into</LI><LI>acetonitrite - 965 ul --------&gt; in 200 mM conc. -----&gt;9.9 TRIS 25 nM</LI></UL><P>we left 8 tubes with LCC variants for incubation at 10:30 AM for the weekend (50 C)</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 08/06/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><DIV class="row"><DIV class="col-lg-1"> </DIV><DIV class="col-lg-1"> </DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 06/06/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><P>we designed an expriment to test differnet carbon surce from our metabolic pathway for putida. We used 96DW and we orgenized it as seen in the picture:</P><DIV class="row"><DIV class="col-lg-1"> </DIV><DIV class="col-lg-1"> </DIV></DIV><P>this experiment will last for one week we will check the change in the turbidity with plate-reader.</P></DIV></DIV></DIV></DIV><DIV id="May" class="container-fluid"><DIV class="row"><DIV class="col-lg-8"><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 31/05/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><P>we cut De-lorenzo plasmids (414+224), we run thus plasmids on agar gel for analazing if thus plasmids were cut.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 28/05/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><P>starters for Putida KT2440. 5 ml LB+5 ul amp, 100ng/ul, 30C</P><P>transformetion for pACYC plasmid without insertion(negative control for coplement cells) and with insertion LCC WT C6, LCC CO C4, LCC F4C2, LCCF7C3, LCC R4C7, LCC R7C2 to BL21 bacteria for protein exprection.</P><UL style="list-style: disc;"><LI>8 tubes with compatent BL21 bacteria, 80C,10 min.</LI><LI>1 ul from each plasmid for each tube.</LI><LI>mixed with tip.</LI><LI>keep in ice for 20 min.</LI><LI>keep in ice for 3 min more.</LI><LI>500 ul SOC without antbiotic.</LI><LI>27C 1h</LI><LI>centrifugation 5000 rpm 2min</LI><LI>take off 500ul from the liquid.</LI><LI>mix the residue with the rest liquid.</LI><LI>sowing in LB+CMP</LI></UL></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 24/05/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><P>seq of LCC variants 3 ul primer in each reaction 300 ng DNA. (each with Rw\Fw primer).</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 19/05/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Roee and Neomi</STRONG></P><P>We made plates with : LB+TET, LB+Strep. we took the starters with De-Lorenzo bacteria to swo on thus plates.</P><P>After all the starters and glycerol stocks were found contaminated a new plate of rhodococcus ruber was recieved from alex sivans lab. a new SM medium was made and we have set the tools inorder to grow new starters on sunday may the 22nd.</P><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV><DIV class="col-lg-1"> </DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 13/05/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><P>Constructing a growth-curve for the Putida on Cathecol. The experiment was contaminated. we will have to repete it.</P><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV><P> </P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 11/05/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><P>pSEVA starters from 10.5, and sowing on LB+KM 2 ml starters 300 rpm, 2 min, RT, using sup and sowing the output.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 03/05/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal, Roee and Dor</STRONG></P><P>one colony from each plate was sent fot seq(because of the bw concentration, 15 ul from each stock was sent)</P><P>2 starters of rhodococcus ruber were made from two different glycerol stocks to check for contamination. each sample was plated on a nutrient agar plate that was divided in to two parts one for each glycerol stock. the starters were put in ramons lab in the incubator shaker 37 degrees and the plate was put in our incubator at 37 degrees as well.</P><P>we prepered glycerol stocks of BL21.</P></DIV></DIV></DIV></DIV><DIV id="April" class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 30/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 29/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben</STRONG></P><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 27/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben</STRONG></P><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 25/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH> </TH><TH>reaction 1</TH><TH>reaction 2</TH><TH>reaction 3</TH><TH>reaction 4</TH><TH>reaction 5</TH><TH>reaction 6</TH><TH>reaction 7</TH></TR><TR><TH> </TH><TH>LCC W.T</TH><TH>LCC C.O</TH><TH>LCC F4</TH><TH>LCC R4</TH><TH>LCC F7</TH><TH>LCC R7</TH><TH>control</TH></TR><TR><TH>pACYC</TH><TD>1.8</TD><TD>1.8</TD><TD>1.8</TD><TD>1.8</TD><TD>1.8</TD><TD>1.8</TD><TD>1.8</TD></TR><TR><TH>Insert</TH><TD>0.9</TD><TD>1.3</TD><TD>0.6</TD><TD>0.8</TD><TD>0.9</TD><TD>0.8</TD><TD>-</TD></TR><TR><TH>Buffer</TH><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>1.4</TD></TR><TR><TH>DDW</TH><TD>14.3</TD><TD>13.9</TD><TD>14.6</TD><TD>14.4</TD><TD>14.3</TD><TD>14.4</TD><TD>15.2</TD></TR><TR><TH>Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR></TBODY></TABLE></DIV><P>incubeted at 16C</P><P>M9 salts add to 800 ml H2O:</P><UL style="list-style: disc;"><LI>31.64 gr Na2HPO4*2H20</LI><LI>15 gr Kh2PO4</LI><LI>2.5 gr NaCl</LI><LI>5.0 gr NH4Cl</LI></UL><P>stirr until dissolved adjust to 100 ml with distilled H2O</P><P>make M9:</P><UL style="list-style: disc;"><LI>700 ml DDW</LI><LI>200ml M9 salts</LI><LI>2 ml 1M MgSO4</LI><LI>100ul 1M CaCl2</LI><LI>adjust to 980 ml with DDW</LI></UL></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 19/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Efrat</STRONG></P><P>Cyclic Voltametry was preformed in oreder to mesure Cathecol’s redox porential conditions:</P><UL style="list-style: disc;"><LI>2% wt Catechol solution in M9. Potential range:</LI><LI>0-0.5 V.E step: 0.005 V</LI><LI>scan rate: 0.01, 0.005, 0.001 V. Room temperature, pH 7</LI></UL><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH> </TH><TH>0.005 V</TH><TH>0.01 V</TH><TH>0.001 V</TH></TR><TR><TH>E for I max</TH><TD>0.324797</TD><TD>0.334791</TD><TD>0.41574</TD></TR><TR><TH>E for I min</TH><TD>0.119925</TD><TD>0.099937</TD><TD>0.131918</TD></TR><TR><TH>Delta E</TH><TD>0.204872</TD><TD>0.234854</TD><TD>0.283822</TD></TR><TR><TH>E 0</TH><TD>0.222361</TD><TD>0.217364</TD><TD>0.273829</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 18/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben</STRONG></P><P>I loaded the PCR products from yesterday onto an agarose gel (1.5%).</P><P>The results:</P><DIV class="row"><DIV class="col-lg-5"><UL style="list-style: disc;"><LI>Lane 1: 1kb plus ladder</LI><LI>Lane 2: pcaG (~660bp)</LI><LI>Lane 3: pcaH (~760bp)</LI><LI>Lane 4: pcaC (~450bp)</LI><LI>Lane 5: pcaD (~840bp)</LI></UL></DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 17/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben</STRONG></P><P>I ran a colony-PCR on P. putida with primers for the 4 protocatechuate degredation genes (pcaG/H, pcaC/D)</P><P>The reaction mix was:</P><UL style="list-style: disc;"><LI>25μL of ReadyMix Taq PCR reaction mix</LI><LI>22μL of UPW</LI><LI>1.5μL of each primer</LI><LI>A swab from a P. putida colony.</LI></UL><P>(total reaction volume 50μL)</P><P>Cycle parameters were according to Sigma-Aldrich protocol.</P><P>ligation of LCC variants into pACYC vector. Using a ligation calculator in 3:1 ration: we will use about 36 ng of insert for each ligation reaction:</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH> </TH><TH>reaction 1</TH><TH>reaction 2</TH><TH>reaction 3</TH><TH>reaction 4</TH><TH>reaction 5</TH><TH>reaction 6</TH></TR><TR><TH> </TH><TH>LCC W,T</TH><TH>LCC C.O</TH><TH>LCC F4</TH><TH>LCC R4</TH><TH>LCC F7</TH><TH>LCC R7</TH></TR><TR><TH>pACYC</TH><TD>1.4</TD><TD>1.4</TD><TD>1.4</TD><TD>1.4</TD><TD>1.4</TD><TD>1.4</TD></TR><TR><TH>Insert</TH><TD>0.9</TD><TD>1.3</TD><TD>0.6</TD><TD>0.8</TD><TD>0.9</TD><TD>0.8</TD></TR><TR><TH>Buffer</TH><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH>DDW</TH><TD>14.7</TD><TD>14.3</TD><TD>15</TD><TD>14.8</TD><TD>14.7</TD><TD>14.8</TD></TR><TR><TH>Ligase</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR></TBODY></TABLE></DIV><P>+control: pACYC 1.4, Buffer 2, DDW 1, Ligase 1 -Incubated overnight at 16C</P><P>prepered LB agar plates whith CAM resistance.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 16/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal</STRONG></P><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 13/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar B, Inbar S and Eyal</STRONG></P><P>we took Rodococus rubber from glycerol stock and plated on te LB plate at the biological hood.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 12/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal, Inbar B and Inbar S</STRONG></P><P>we have made 2 plates containing only LB. we have made 8 plates containing LB+CRB.</P><P>colonies were present. two- 5 ml starters from CAM plate (pACYC) for midiprep and miniprep.</P><P>starter for midiprep 200 ml midiprep.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 11/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar B, Inbar S and Eyal</STRONG></P><P>we took new pACYC from eyal arbely lab, we transformed E. coli it to mg1655 and seed it on CAM plate.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 08/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Tomer</STRONG></P><P>we have made 3 medium with glucose, PET, PolyEthylene and palced them in 37C shaker.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 06/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Tomer and Ben</STRONG></P><P>restriction of pACYC and E1B GFP</P><OL><LI>E1B GFp 1 ul</LI><LI>3.1 buffer 5 ul</LI><LI>XhoI 1ul</LI><LI>DDW 43 ul</LI><UL style="list-style: square;"><LI>pACYC 5 ul</LI><LI>3.1 5 ul</LI><LI>XhaI 1 ul</LI><LI>DDW 39 u</LI></UL><LI>Added 1ul BglII incubeted at 37 C</LI><LI>Glycerol stock made for putida:
</LI><LI>500 ul from starters</LI><LI>500 ul glycerol 30 % +LB</LI></OL><P>making a startersfor Rodococus rubber:  We took 50 ml of SM+Glucose and added to it some of the glycerol stock.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 05/04/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Dar, Tomer and Ben</STRONG></P><P>we prepared new LB medium (1L), gathered and prepared glassware for autoclave.  we set up startes for the Putida - for growth experiments and for preparing new glycerol stocks.</P><P>We prepared a 20% (w) Catechol solution using 10gr of Catechol (Pyrocatechol from Sigma-Aldrich) and 50ml of DDW.</P><P>We prepared M9 Catechol medium using:</P><UL style="list-style: disc;"><LI>10ml M9 Salts</LI><LI>5μL CaCl2 (1M)</LI><LI>100μL MgSO4</LI><LI>39ml DDW</LI><LI>165μL Catechol 20%</LI></UL></DIV></DIV></DIV></DIV><DIV id="March" class="container-fluid"><DIV class="row"><DIV class="col-lg-8"><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 31/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Dar</STRONG></P><P>checking whether the Putida from the experiment last week (Growth curves for Putida and E.Coli on EG) are still alive (and that the OD we’ve measured isn’t the result of dead bacteria).  at both times samples were taken from the liquid solution in the tub and put on plates (LB+amp). at both times growth on the plates was observed - the bacteria were still alive.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 28/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar S and Eyal</STRONG></P><P>Try #3 of the pACYC restriction  This time - 1 hour with each enzyme and run on agarose gel 2ul after each cut.</P><UL style="list-style: disc;"><LI>pACYC 2000ng 30ul(66ng/ul)</LI><LI>DDW 58ul</LI><LI>3.1 Buffer 10ul</LI><LI>XhoI 1ul</LI></UL><OL><LI>1 hour 37 deg</LI><LI>20 min 65 deg</LI><LI>1 ul BalII</LI><LI>1hour -&gt; run on agarose gel 2ul.</LI></OL></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 28/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar S and Eyal</STRONG></P><P>Another restriction reaction for the pACYC plasmid. We used the pACYC2 product from 25/03(68ng/ul) - Restriction requires 2000ng of plasmid DNA so we used a volume 29.4ul.</P><P><STRONG>Reaction protocol:</STRONG></P><UL style="list-style: disc;"><LI>DNA 29.4ul</LI><LI>3.1 Buffer (NEB) 10ul</LI><LI>BalII 1ul</LI><LI>XhoI 1ul</LI><LI>DDW 58.8ul</LI></UL><P>No Bands again.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 27/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Dar </STRONG></P><P>Setting up the growth-curve experiment for both the Putida and E.Coli (MG1655) on Ethylene Glycol. This time the experiment will take a week - 2-3 measurements per day.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 25/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar </STRONG></P><P>Mini-prep for pACYC.</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH> </TH><TH>Concentration (ng/uL)</TH><TH>260/280nm</TH></TR><TR><TH>pACYC1</TH><TD>66</TD><TD>1.81</TD></TR><TR><TH>pACYC2</TH><TD>68</TD><TD>1.78</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 24/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Tomer and Inbar Bariah</STRONG></P><P>Removing the putida starter from the incubator to the fridge.</P><P>DNA extraction from reaction mix- LC-cutinase variants.  total volume 30ul</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>LC-cutinase variant</TH><TH>concentration(ng/ul)</TH><TH>260\280</TH></TR><TR><TD>lc-cutinase - WT</TD><TD>43</TD><TD>1.63</TD></TR><TR><TD>lc-cutinase - WT CO</TD><TD>28</TD><TD>1.52</TD></TR><TR><TD>FreeRun4</TD><TD>61</TD><TD>1.71</TD></TR><TR><TD>FreeRun7</TD><TD>47</TD><TD>1.66</TD></TR><TR><TD>ResRun4</TD><TD>41</TD><TD>1.69</TD></TR><TR><TD>ResRun7</TD><TD>46</TD><TD>1.64</TD></TR></TBODY></TABLE></DIV><DIV class="row"><DIV class="col-lg-2"> </DIV><P><STRONG>WT, WTCO, F4, F7, R4, R7, 1KbPlus led</STRONG></P></DIV><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV><P>In addition, starters for pACYC were prepared: 10ml LB+10uL CAM for 2 starters.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 23/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar Segal, Dar, Ben, Inbar B, Roee, Dor, Tomer and Eyal</STRONG></P><P>Produced the pACYC plasmid from the starter and cleaned it using mini prep kit.  Concentration: 52ng/ul 260\280: 1.83</P><P>another try at constructing a growth-curve for the Putida and E.coli on Ethylene glycol and Glucose.</P><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV><P>Setting up new starters for the Putida and E.coli (MG1655)</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 22/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal and Inbar Segal</STRONG></P><P>Cutting LC-cutinase and pACYC vector with restriction enzymes, no bands were visible on electrophoresis gel.</P><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 18/03/16</SPAN></STRONG></P><P>PCR extraction of LC-cutinase variants  total volume 30ul</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>LC-cutinase variant</TH><TH>concentration(ng/ul)</TH><TH>260\280</TH></TR><TR><TD>lc-cutinase - WT CO</TD><TD>134</TD><TD>1.81</TD></TR><TR><TD>FreeRun4</TD><TD>143</TD><TD>1.79</TD></TR><TR><TD>FreeRun7</TD><TD>133</TD><TD>1.80</TD></TR><TR><TD>ResRun4</TD><TD>110</TD><TD>1.78</TD></TR><TR><TD>ResRun7</TD><TD>138</TD><TD>1.81</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 17/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar Bariah</STRONG></P><P>PCR to amplify LC-cutinase variants: WT CO, FreeRun4, FreeRun7, ResRun4, ResRun7. WTCO, WTCOX,F4,F4X,F7,F7X,R4,R4X,R7,R7X,1KbPlus led</P><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 15/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben, Dar, Roee and Tomer</STRONG></P><P>We tried constructing a growth curve for E. coli and P. putida on Glucose and Ethylene Glycol.</P><P>The Experiment failed (the O.D was declining instead of increasing), probably because the starters we used were kept in the refrigerator for more than 4 days (the bacteria were probably dead).</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 08/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Eyal and Inbar Segal</STRONG></P><P>we did a transformation to PACYC, using 50microliter competent bacteria.</P><P>100 microliters were used for plating.</P><P>an activity test was made of DpnI - we did a transformation in order to check that the plasmid indeed dissolved (nothing is supposed to grow).</P><P><STRONG>concentrations of the genes we received from Weizmann institute:</STRONG></P><UL style="list-style: disc;"><LI>LC-cutinase - WT - 26.55 ng/ul</LI><LI>LC-cutinase - FreeRun4 - 19.15 ng/ul</LI><LI>LC-cutinase - FreeRun7 - 15.575 ng/ul</LI><LI>LC-cutinase - ResRun4 - 20.175 ng/ul</LI><LI>LC-cutinase - ResRun7 - 25.575 ng/ul</LI></UL></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 07/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben, Dar, Eyal and Inbar Segal</STRONG></P><P>Starters did not grow.</P><P>12 Petri Dishes of LB+CAM were prepared.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 06/03/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben, Tomer and Dar</STRONG></P><P>Preparation of starters for growth curve experiment. (P. putida and E. coli) Glucose and Ethylene Glycol.</P></DIV></DIV></DIV></DIV></DIV><DIV id="other" class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV id="Fabruary"><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 25/02/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben</STRONG></P>
went to the supply room to bring in supplies:
<OL><LI>one measuring cup 2L made of glass, and one 1L measuring cup made of plastic.</LI><LI>20ml syringes and 10ml syringes - 10 from each volume. and a few more 2.5 ml because they didn't have 5ml syringes.</LI><LI>around 30 minisart filters.</LI><LI>latex gloves Medium size x5.</LI><LI>A bundle of glass pipettes 10ml (they didn't have 20ml ones) we need to order especially.</LI><LI>big tank for water (needs a wash and then we can use it for ddw or upw).</LI></OL></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 24/02/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Neomi, Tomer and Dor</STRONG></P><P>a glycerol stock of rhodoccocus ruber was prepared from the starter made yesterday. I didn't know if the glycerol in our fridge was autoclaved so I borrowed from Ramon's lab, the stock is at (-80) degrees Celsius at Ramon's shelf.</P><P>Removed the putida from the incubator after growing on ethylene glycol and glucose and checked the OD (600nm)  Results:</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Tube</TH><TH>OD (600nm)</TH></TR><TR><TD>Glucose 20%</TD><TD>0.686</TD></TR><TR><TD>Glucose 15% + Ethylene Glycol 5%</TD><TD>0.420</TD></TR><TR><TD>Glucose 10% + EG 10%</TD><TD>0.859</TD></TR><TR><TD>Glucose 5% + EG 15%</TD><TD>0.112</TD></TR><TR><TD>Ethylene Glycol 20%</TD><TD>0.465</TD></TR></TBODY></TABLE></DIV><P>because the measurements were inconsistent I made several measurements. especially tubes 1 and 5:</P><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Tube</TH><TH>OD (600nm)</TH></TR><TR><TD>Glucose 20%</TD><TD>first re-measurement 0.459 second re-measurement 0.611</TD></TR><TR><TD>Glucose 15% + Ethylene Glycol 5%</TD><TD>first re-measurement 0.019 second re-measurement 0.032</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 23/02/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben and Roee</STRONG></P><UL style="list-style: disc;"><LI>a starter of Rhodococcus ruber was made. it included 50ml SM(medium) + glucose + microelements. the work was done in a biological hood afterwards it was placed in a shaker at 37 degrees Celsius overnight.</LI><LI>taking the putida out of the shaker after growing it on glucose and ethylene glycol and checking OD (600nm).  Results:</LI></UL><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Tube</TH><TH>OD (600nm)</TH></TR><TR><TD>Glucose 20%</TD><TD>0.560</TD></TR><TR><TD>Glucose 15% + Ethylene Glycol 5%</TD><TD>0.671</TD></TR><TR><TD>Glucose 10% + EG 10%</TD><TD>0.465</TD></TR><TR><TD>Glucose 5% + EG 15%</TD><TD>0</TD></TR><TR><TD>Ethylene Glycol 20%</TD><TD>0.09</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 21/02/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Roee, Dar, Dor and Neomi</STRONG></P><STRONG>how to make competent bacteria?</STRONG><UL><LI>all the process need to occur on ice, in a centrifuge set to 4degree Celsius, and the glycerol need to be cold.</LI><LI>first of all, we grow the bacteria over night with antibiotics -&gt; in 50 ml falcon tube.</LI></UL><OL><LI>centrifuge - maximum speed for 15 min and remove the supernatant.</LI><LI>we filled the falcon tube again with glycerol 10%. centrifuge once again for 15 min at max speed and again remove the supernatant.</LI><LI>repeat step 2.</LI><LI>we combine both tubes (if we have more than one, if not we wash it again with glycerol as we did in 2+3) after we fluidize the bacteria in glycerol we complete the volume with 10% glycerol, we centrifuge and remove the supernatant.</LI><LI>fluidize the sediment with 2 ml glycerol and divide into tubes 50 microliter in each one and freeze with liquid nitrogen.</LI></OL><P>a medium for growing Rhodococcus Ruber was made in the solution are missing microelements that we are supposed to receive from Valentina and there is no carbon source. the solution is in the fridge door with a sticker “rhodococus ruber solution”.</P><DIV class="row"><DIV class="col-lg-1"> </DIV><DIV class="col-lg-4">there was no CaCl2 2H2O si we used 680microliter of CaCl2 1M from the fridge.</DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 22/02/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Roee, Dar, Dor, Tomer, Inbar Segal and Eyal</STRONG></P><P>competent bacteria were made from the putida KT2440 strain. the bacteria were divided into 25 Eppendorf's with 50microliter each (the volume needed for each transformation) the Eppendorf's are in the freezer (-80 degrees) of Ramons lab in a red box the with the inscription “Ashalim”.</P><STRONG>starting the experiment of growing the Pseudomonas putida on ethylene glycol:</STRONG><OL><LI>we made the minimal medium where each one has different rates of carbon sources.</LI><LI>we inserted to each medium one colony of Pseudomonas Putida.</LI><LI>we made a blank of 1ml to measure OD.</LI><LI>we put the bacteria in a Shaker set to 37 degrees Celsius in Naama's lab.</LI></OL><P>We did a PCR reaction for the LC-Cutinase + leader Sequence Gene. in order to take it from the iGEM plasmid and insert it into a PACYC vector. we ran the PCR products on an agarose gel using electrophoresis and took a picture. we got one big band in the size we wanted 1KBP~ and a second band, weaker one in the size of the entire plasmid.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 15/02/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar Bariah</STRONG></P><STRONG>main action - growing two strains of Pseudomonas putida (EM383, KT2440) on SOC :</STRONG><P>Inserting a Q-tip that touched the glycerol stock of the relevant culture into 3ml of SOC with ampicillin (near fire) and grow overnight. -results- the bacteria grew and the medium became very murky. the bacteria were afterwards grown on petri dishes with LB + amp from every strain we inserted 20 microliters and we also made plates where we isolated via striking.</P><P>colonies from every strain (KT2440, EM383) were moved into a liquid LB medium for incubation overnight in 37 degrees Celsius.</P></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 14/02/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar Bariah, Neomi and Tomer</STRONG></P><STRONG>main action - making M9 medium for growing Pseudomonas Putida</STRONG><P>Instead of glucose as a carbon source we planned an experiment where the ratios between glucose and ethylene glycol vary in order to check if the bacteria can use ethylene glycol as a carbon source. if so, we won't need to insert the two plasmids we took from E. coli.</P><STRONG>planning the experiment </STRONG> - we divided the medium into 5 different test tubes where the carbon source will be different in each one:
<OL><LI>100% glucose.</LI><LI>75% glucose, 25% ethylene glycol.</LI><LI>50% glucose, 50% ethylene glycol.</LI><LI>75% ethylene glycol, 25% glucose.</LI><LI>100% ethylene glycol.</LI></OL><STRONG>making the medium: </STRONG><OL><LI>with the assistance of Idit we made MgSo4 and a Salt sterile solutions using autoclave.</LI><LI>we used Sterile CaCl that already gone through autoclave and also DDW that we already had in the lab.</LI><LI>we made 20% glucose and filtered it.</LI><LI>we filtered ethylene glycol and made a 20% ethylene glycol solution.</LI></OL><DIV class="row"><DIV class="col-lg-2">concentrations:</DIV></DIV><STRONG>main action - making ampicillin:</STRONG><OL><LI>Weighing 1g ampicillin.</LI><LI>Adding 10ml DDW.</LI><LI>Filtering with syringe 0.24 wide.</LI><LI>Dividing it into 20 Eppendorf's (500microliter each).</LI></OL><STRONG>A protocol for making petri dishes with LB medium:</STRONG><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 02/02/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Roee, Inbar Bariah and Tomer</STRONG></P><STRONG>main action - re-sending the two plasmids that got low concentrations to sequencing again.</STRONG><P>After checking once again the sequences in blast of the two plasmids that got low percentage and after consulting with Dr.Birnbaum and Naama we understood that the way we looked at the percentage wasn't correct and that the sequencing was actually correct and we had the sequence we wanted on the plasmid. (before we knew about the sequencing being correct we made Chloramphenicol in working dosage in order to grow the bacteria that carried the plasmid on the medium we wanted).</P><STRONG>instructions for sending DNA to sequencing:</STRONG><OL><LI>Plasmid Dosage need to be between 75-100 ng/ul.</LI><LI>volume needed: ten microliters per plasmid.</LI><LI>Primers - 2 microliters per sequenced plasmid (minimum volume 5 ul).</LI></OL></DIV></DIV><DIV id="January"><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 18/01/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar Segal</STRONG></P><STRONG>Preparation of competent bacteria: E. coli MG1655</STRONG><OL><LI>Add all of the starter to 500 ml of LB</LI><LI>Use 1 ml of LB as blank, measure OD at time zero at wavelength of 600 nm. (time zero was 0.064 nm)</LI><LI>Measure OD every hour until the bacteria gets to the logarithmic phase (about 3 hours).</LI><LI>The Incubater, Shaker, Spectofotometer were used in Ofer yifrach lab (3 floor).</LI><LI>At 11:20 OD was 0.54 nm, we took it out of the shaker and put it in ice.</LI><LI>We checked if the bacteria can grow on agar with kan and amp - we did not see any colony.</LI></OL></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 17/01/17</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Dor, Inbar Barih and Neomi</STRONG></P><STRONG>How to make a solution for competent bacteria:</STRONG><DIV class="row"><DIV class="col-lg-1"> </DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 16/01/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Dor, Roee and Tomer</STRONG></P><STRONG>Checking the results of the sequencing:</STRONG><DIV class="row"><DIV class="col-lg-1"> </DIV><DIV class="col-lg-1"> </DIV></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 14/01/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Dor, Roee, Tomer and Naama</STRONG></P><STRONG>How to check the results of sequencing:</STRONG><OL><LI>Open the file with the graphs displaying the output.</LI><LI>Go to edit-&gt;copy seq-&gt;FASTA format.</LI><LI>Go to Blast and paste what we copied in the first window, if we copied first the sequence of the foreword primer after we will do for the reverse primer and vice-versa.</LI><LI>Open the file with the graphs displaying the output, but this time for the reverse primer if we opened its first for the foreword and vice-versa.</LI><LI>Go to edit-&gt;copy seq-&gt;FASTA format.</LI><LI>Go to Blast press enter, and paste this part in the same window one row below.</LI><LI>Take the reference and put in a different box/one row below.</LI><LI>Press Blast</LI></OL><STRONG>Pay attention:</STRONG><OL><LI>In the top of the page in Blast you can check if R or F fits the reference.</LI><LI>It's better to check if R and F match (we expect it to be similar not the same).</LI><LI>Notice the values of query-how similar are the samples, ident-percentage of identical bases.</LI><LI>Look where most of the different bases are located (hopefully in the end or the beginning).</LI><LI>Small letters instead of big ones (atg and not ATG) marks that this sequence repeats itself.</LI><LI>If you to write comments, write according to the locations of the reference.</LI></OL><STRONG>Planning a Vector:</STRONG><OL><LI>We looked at the map of the vector that was sent (for the Putida) and tried to see which restriction enzymes are relevant.</LI><LI>we looked at the nebcutter website to check for each of the fragments which restriction enzymes are relevant (each fragment separately, according to the reference).</LI><LI>press custom digest for the list of enzymes, check that this enzyme don't cut inside of the fragments.</LI><LI>choose restriction enzymes.</LI><LI>Plan primers with tails that will match the enzymes.</LI><LI>pcr and ligation.</LI><LI>transformation.</LI></OL></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 04/01/16</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Dor and Roee</STRONG></P><STRONG>We prepared working stock primers to 10μM, and sent 11 plasmids to sequencing:</STRONG><UL style="list-style: disc;"><LI>LLK - lc cutinase + led seq + his tag</LI><LI>LL - lc cutinase + led seq</LI></UL></DIV></DIV><DIV id="December"><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 31/12/15</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Liran and Tomer</STRONG></P><STRONG>Measuring DNA concentration of the plasmids (the numbers match the ones from above):</STRONG><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Plasmid</TH><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD><TD>8</TD><TD>9</TD><TD>10</TD><TD>11</TD></TR><TR><TH>concentration (ng/ul)</TH><TD>222.96</TD><TD>188.87</TD><TD>159.67</TD><TD>125.30</TD><TD>98.86</TD><TD>293.56</TD><TD>153.54</TD><TD>189.91</TD><TD>155.54</TD><TD>157.45</TD><TD>166.29</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dayContainer"><P><STRONG><SPAN style="text-decoration: underline;">When - 30/12/15</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Ben, Liran, Dar, Tomer and Dor</STRONG></P><STRONG>PCR for two genes (aldA, flco) of the E.coli strain MG1655:</STRONG><OL><LI>Prepration of a DNA gel</LI></OL><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Well</TH><TD>1</TD><TD>3</TD><TD>4</TD><TD>6</TD><TD>7</TD></TR><TR><TH>Sample</TH><TD>ladder</TD><TD>flco</TD><TD>flco control (no template)</TD><TD>aldA control (no template)</TD><TD>aldA</TD></TR></TBODY></TABLE></DIV><STRONG>Extraction of plasmids (using the yellow/orange kit):</STRONG><DIV class="row"><DIV class="col-lg-6"><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Number</TH><TH>Plasmid</TH></TR><TR><TD>1</TD><TD>tph A1</TD></TR><TR><TD>2</TD><TD>tctA A1</TD></TR><TR><TD>3</TD><TD>tph A3</TD></TR><TR><TD>4</TD><TD>LC cutinase</TD></TR><TR><TD>5</TD><TD>LC cut + lead seq + his tag</TD></TR><TR><TD>6</TD><TD>tph A2</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-lg-6"><DIV class="table-responsive"><TABLE class="table"><TBODY><TR><TH>Number</TH><TH>Plasmid</TH></TR><TR><TD>7</TD><TD>tphB</TD></TR><TR><TD>8</TD><TD>tctA A2</TD></TR><TR><TD>9</TD><TD>rctB B1</TD></TR><TR><TD>10</TD><TD>tph C</TD></TR><TR><TD>11</TD><TD>LC cut + lead seq</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="dayContainer"><P dir="LTR"><STRONG><SPAN style="text-decoration: underline;">When - 29/12/15</SPAN></STRONG></P><P><STRONG>Who's In the lab today: Inbar Barih, Inbar Segal, Eyal, Liran and Ben</STRONG></P><STRONG>Making starters for bacteria after Transformation:</STRONG><OL><LI>Add 5 ml of LB to a 50ml falcon.</LI><LI>Add 5 ul of the matching antibiotic (the ratio is 1:1000).</LI><LI>With a toothpick stab one colony spread it on another Agar plate (in case the starter wont work we will be able to use that colony again) and then throw it inside of the 50 ml flacon.</LI></OL></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row footerColor"><DIV class="col-lg-4 text-center"><H4>Address:</H4><P>
                        Ben-Gurion University of the Negev
                        Ben Gurion 1, Beer Sheva 8410501, Israel
                    </P><H5><B><U>Mail:</U></B> igembgu2016@gmail.com
                    </H5></DIV><DIV class="col-lg-4 text-center"><H4>Connect With Us!</H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>