<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BGU_ISRAEL_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BGU ISRAEL/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>PlastiCure</TITLE></P><NAV class="navbar navbar-inverse navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="mainmenu nav navbar-nav"><LI class="dropdown" id="HomeB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL">Home</A></LI><LI class="dropdown" id="projectB" style="background-color: #CB99C9;"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Achievements">Achievements</A></LI></UL><LI class="dropdown" id="teamB"><A href="https://2016.igem.org/Team:BGU_ISRAEL/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/TeamMembers">Members</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/MeetUniversity">Meet Our University</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/MeetTown">Meet Our Town</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Collaborations">Collaborations</A></LI><LI class="dropdown" id="partsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Parts#Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/ImprovedParts">Improved Parts</A></LI><LI class="dropdown" id="humanB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Integrated_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/PlasticArt">Plastic Art</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/EthicsAndSafety">Ethics &amp; Safety</A></LI><LI class="dropdown" id="attributionsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Attributions">Attributions</A></LI><LI class="dropdown" id="awardsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Awards">Awards</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Measurements">Measurements</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Proof">Proof Of Concept</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/HP/Silver">HP - Silver</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/HP/Gold">HP - Gold</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Engagement">Engagements</A></LI></DIV></DIV></NAV><DIV class="container-fluid"><DIV class="row"><DIV class="projectColor"><P>Results</P></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="col-lg-8"><DIV class="row protoLists"><DIV class="col-lg-10"><P class="medHeader">
                            Our Results
                        </P><UL><LI><A href="#rRuber"><I>R. ruber</I> Growth experiments - A new discovery!</A></LI><LI><A href="#pNPButy">pNP-butyrate Assay</A></LI><LI><A href="#electronM">Electron microscopy imaging of PET degradation</A></LI><LI><A href="#putiAnti"><I>P. putida</I> antibiotic resistances test results</A></LI><LI><A href="#putiCarbon"><I>P. putida</I> carbon source utilization results</A></LI><LI><A href="#ecoliEG"><I>E. coli</I> ethylene glycol utilization results</A></LI><LI><A href="#utilPET">Examining the utilization of PET by <I>E. coli</I> expressing LC-Cutinase</A></LI><LI><A href="#fuelCell"><I>P. putida</I> Fuel Cell experiments</A></LI><LI><A href="#ecoligrowdial">Characterizing <I>E. coli</I> growth inside a dialysis bag</A></LI><LI><A href="#pAndE"> Characterization of the co-culturing of <I>E. coli</I> and <I>P. putida</I> - a controlled symbiotic approach</A></LI><LI><A href="#TPAin"> Detection of TPA in plates containing PET and E. coli using fluorescence</A></LI></UL></DIV></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="resultsText"><DIV class="resultsHeadLine"><I>R. ruber</I> Growth experiments - a new discovery!
                        </DIV><P>
                            The bacterium <I>Rhodococcus ruber</I><I>R. ruber</I>, known to utilize polyethylene (PE) as a carbon source, was grown in various liquid growth
                            media with and without amino-acids and containing PE or PET pellets for a period of 30 days. 
                        </P><P>
                            For control we used a SM liquid growth medium with glucose, glucose and amino-acids and a SM medium with no carbon
                            source (for detailed instructions on preparation of the media see <A href="https://2016.igem.org/Team:BGU_ISRAEL/Protocols">Protocols</A> ).
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 1:</B><I>R. ruber</I> grown on SM liquid media with various carbon sources for a period of 30 days. Positive control is glucose and negative control is SM medium with no carbon source.
                           </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            As expected the results show that growth on glucose media was substantially faster and yielded high O.D<SUB>600</SUB> measurements throughout the experiment. 
                            We have observed similar results when the bacterium was grown on media containing amino acids (AA), suggesting the bacteria can utilize the AA as a carbon source. 
                            The media containing plastic as the sole carbon source (PET, PE), reached higher O.D<SUB>600</SUB> values from that of a medium with no carbon source (SM medium) but lower O.D<SUB>600</SUB> than on glucose media or plastic enriched with AA media. 
                            All samples have shown a steep rise in the absorbance on the first few hours of the experiment. We can assume that these results are due to the trace of rich medium that was left from the starter that was used prior to the begining of the experiment. 
                            This might also explain the increase in O.D<SUB>600</SUB> values in the SM medium. Lack of values of certain samples is due to contaminations or broken equipment.
                        </P><P>
                            After 30 days we ceased measuring O.D<SUB>600</SUB> values, but continued monitoring our samples for bacterial growth. 
                            Samples were left to incubate unopened for 3 months to test for the formation of biofilms.
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 2:</B><I>R. ruber</I> biofilm formation on a PE pellet.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            As seen in the picture (fig. 2), after 3 months <I>R. ruber</I> developed a biofilm surrounding the pellet (in this case – PE). The orange color is a unique property of <I>R. ruber</I>. 
                            We have witnessed similar results with PET pellets, suggesting <I>R. ruber</I> can also utilize PET as a sole carbon source.
                        </P><P>
                            We took samples from both PET and PE media (with pellets) and examined them with a Phase-contrast microscope and scanning electron microscope (SEM):
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 3:</B> An image taken with a Phase-contrast microscope from SM media with PET after incubation with <I>R. ruber</I>.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            First, we could see that <I>R. ruber</I> was present in its planktonic form in the media.
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 4:</B> An image taken with a Phase-contrast microscope from SM media with PET after incubation with <I>R. ruber</I>. In the center we notice a large vacuole we bacteria inside.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            After further examining the images we noticed the formation of vacuoles close to aggregates of bacteria. 
                            We assumed the vacuoles are made up of Extracellular polymeric substance (EPS) that was secreted by the bacteria into the media. 
                            We speculated that in this case the EPS is used for the absorption of the PET micro particles and allows the accumulation of secreted enzymes. 
                            The enzymes are used to catalyze the initial cleavage of the polymer and allow absorption of nutrients (Laspidou et al 2002). 
                            These images suggest a mechanism to improve degradation by increasing the extracellular degrading enzyme concentrations.
                        </P><P>
                            More samples from the same experiment were analyzed using SEM:
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 5:</B> A scanning electron microscope image of an untreated PE pellet in a X1000 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 6:</B> A scanning electron microscope image of a PE pellet after 30 days of incubation with <I>R. ruber</I> in a X1000 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            As seen from the images above, there is a substantial difference between the surfaces textures. The control's (fig. 5) surface is rough while the treated one is smoother (fig. 6). 
                            It can be explained by the ability of bacteria, i.e. its enzymes, to degrade exposed and less dense polymers with more ease, smoothening the outer layer.
                        </P><P>
                            We performed SEM on the PET samples in order to examine if the same effect on the polymer's surface is observed:
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 7:</B> A scanning electron microscope image of an untreated PET pellet in a X10,000 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 8:</B> A scanning electron microscope image of a PET pellet after 30 days of incubation with <I>R. ruber</I> in a X10,000 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            Contrary to our expectations, the untreated PET surface is smooth while the treated sample’s surface seemed to have gone through degradation (fig. 8).  
                        </P><P>
                            Furthermore, we have come to notice the appearance of crystalline rods near bacteria aggregation sites. 
                            That can be explained by the preferential degradation of the amorphous material exposing the better defined (higher crystallinity) material that goes through a slower degradation (Shah et al 2008).
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 9:</B> A scanning electron microscope image of a PET pellet after 30 days of incubation with <I>R. ruber</I> in a X2,500 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 10:</B> A scanning electron microscope image of a PET pellet after 30 days of incubation with <I>R. ruber</I> in a X2,500 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 11:</B> A scanning electron microscope image of a PET pellet after 30 days of incubation with <I>R. ruber</I> in a X2,500 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            Our results suggest that <I>R. ruber</I> is able to degrade PET, having presented substantial changes before and after incubation with PET. It should be mentioned that it is not yet established that <I>R. ruber</I> is able to utilize PET as a carbon source. 
                            In order to verify this possibility, more tests are needed. 
                            After identifying <I>R. ruber</I>'s mechanisms of plastic degradation such as secretion of polymer degrading enzymes and forming vacuoles from EPS to increase the extracellular enzyme concentration, 
                            we decided to concentrate our efforts on improving a PET degrading enzyme. 
                        </P><P><B>It is important to note, that to the best of our knowledge, this is the first time that PET degradation by <I>R. ruber</I> is reported.</B></P><P>References</P><OL class="referenceExpe"><LI>
                                Laspidou, C. S., &amp; Rittmann, B. E. (2002). A unified theory for extracellular polymeric substances, soluble microbial products, and active and inert biomass. Water Research, 36(11), 2711-2720.
                            </LI><LI>
                                Shah, A. A., Hasan, F., Hameed, A., &amp; Ahmed, S. (2008). Biological degradation of plastics: a comprehensive review. Biotechnology advances, 26(3), 246-265.
                            </LI></OL></DIV></DIV></DIV></DIV><DIV class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="resultsText"><DIV class="resultsHeadLine">
                            pNP-butyrate Assay
                </DIV><P>
                            One method to assess the activity of our designed mutants was using the pNP-butyrate (pNP-B) degradation assay
                            (see <A href="https://2016.igem.org/Team:BGU_ISRAEL/Protocols">Protocols</A>).
                            
                            We ran the tests on all 5 variants of the LC-Cutinase protein with different concentrations of the substrate: 
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 12:</B> pNP-Butyrate degradation activity of all LC-Cutinase variants and W.T. at a substrate concentration of 50μM. 
                                            For control we used E. coli strain BL-21 without any vector (&quot;BL-21&quot;) with pACYC plasmid backbone only (&quot;pACYC&quot;).
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 13:</B> pNP-Butyrate degradation activity of all LC-Cutinase variants and W.T. at a substrate concentration of 125μM. 
                                            Controls are the same as with 50μM concentration.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 14:</B> pNP-Butyrate degradation activity of all LC-Cutinase variants and W.T. at a substrate concentration of 250μM. 
                                            Controls are the same as with 50μM concentration.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            As seen from these results, all of our variants, including the codon-optimized Cutinase, have shown increased pNP-B degradation activity compared to the W.T. LC-Cutinase in all concentrations tested. 
                            We can also see that the highest activity is displayed by the codon-optimized(CO) Cutinase, followed by the F4 mutant.
                        </P><P>
                            These test were preformed by incubating the substrate with a supernatant from LC-Cutinase expressing bacteria and not taking into consideration the different expression levels of each variant. 
                            Even though later tests with <I>R. ruber</I> show that the concentration of enzyme is an important part in efficiency of degradation, 
                            we still decided to run these tests again, with equal enzyme concentrations on our 2 fastest variants - CO and F4:
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 15:</B> pNP-Butyrate degradation activity of 3 LC-Cutinase variants - CO, F4 and the W.T., at a substrate concentration of 125μM. 
                                            Enzyme concentration is 0.11mg/ml.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            Again, we can see that the CO and F4 variants have higher activity in regards to pNP-B degradation than the W.T. Cutinase. The CO Cutinase is once again with the highest activity, suggesting that expression levels are not a major determinant in its pNP-B degradation activity.
                        </P><P>
                            To calculate the amount of improvement in activity of our variants we constructed a Lineweaver-burk plot using the following protocol: 
                            A pNP-Butyrate stock were prepared with 25 mM Tris-HCl (pH 8.0) and pNP-Butyrate in concentrations ranging from 50 μM to 2000 μM.
                            Enzymatic activity was determined at 26.8<SUP>o</SUP>C:
                        </P><DIV class="table-responsive"><TABLE class="table table-bordered"><THEAD><TR><TH>Previously documented W.T.</TH><TH>Our W.T. enzyme</TH><TH>Codon-optimized enzyme</TH><TH>F4 variant</TH></TR></THEAD><TBODY><TR><TH>K<SUB>m</SUB> [mM]</TH><TD>0.21-0.24</TD><TD>0.19</TD><TD>0.98</TD><TD>1.17</TD></TR><TR><TH>K<SUB>cat</SUB> [s<SUP>-1</SUP>]</TH><TD>232±20</TD><TD>48±10</TD><TD>2700±100</TD><TD>2095±100</TD></TR><TR><TH>K<SUB>cat</SUB>/K<SUB>m</SUB>[s<SUP>-1</SUP>*M<SUP>-1</SUP>]</TH><TD>1050</TD><TD>252±20</TD><TD>2755±100</TD><TD>1791±100</TD></TR></TBODY></TABLE></DIV><P>
                            As seen from the results, the K<SUB>m</SUB> values of our variants compared to the W.T. is of the same magnitude, so we conclude that there is no significant change in affinity. 
                            Moreover, when comparing the K<SUB>cat</SUB> values of our variant to the W.T. we see a significant improvement which is consistent with our previous results.
                        </P><P>
                            Finally, we can see that the catalytic efficiency, portrayed by the K<SUB>cat</SUB>/K<SUB>m</SUB> value, has been significantly improved in our variants, compared to the W.T. enzyme, both ours and the previously documented.
                        </P><P>
                            We can conclude from all these results that we managed to dramatically improve LC-Cutinase's activity, with regards to pNP-Butyrate degradation.
                        </P></DIV></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="resultsText"><DIV class="resultsHeadLine">
                            Electron microscopy imaging of PET degradation
                        </DIV><P>
                            To further characterize LC-Cutinase's PET degradation activity we used Scanning-electron microscopy of PET pellets incubated with <I>E. coli</I> expressing LC-Cutinase.
                        </P><P><B>Control:</B></P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 16:</B> A scanning electron microscope (SEM) image of a PET pellet after 2 days of incubation in a liquid LB broth with no bacteria. X1000 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 17:</B> A scanning electron microscope (SEM) image of a PET pellet after 2 days of incubation in a liquid LB broth with no bacteria. X1000 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 18:</B> A scanning electron microscope (SEM) image of a PET pellet after 2 days of incubation in a liquid LB broth with no bacteria. X2500 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 19:</B> A scanning electron microscope (SEM) image of a PET pellet after 2 days of incubation in a liquid LB broth with no bacteria. X5000 magnification.A scanning electron microscope (SEM) image of a PET pellet after 2 days of incubation in a liquid LB broth with no bacteria. X5000 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            One control was PET pellets incubated for 2 days in LB liquid media with no bacteria. 
                            As seen from the images, the surface of the PET pellets is relatively smooth, there are some high crystallinity rod shaped structures but they are fairly covered.
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 20:</B> A scanning electron microscope (SEM) image of a PET pellet after 2 days of incubation in a liquid LB broth with <I>E. coli</I> transformed with the pACYC vector with no insert. X1000 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 21:</B> A scanning electron microscope (SEM) image of a PET pellet after 2 days of incubation in a liquid LB broth with <I>E. coli</I> transformed with the pACYC vector with no insert. X2500 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            As a second control we used <I>E. coli</I> transformed with the pACYC vector with no insert (the backbone alone), to assess the effects of bacteria themselves on the PET.  
                        </P><P><B>PET incubated with LC-Cutinase expressing bacteria:</B></P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 22:</B> A scanning electron microscope (SEM) image of a PET pellet after 2 days of incubation in a liquid LB broth with <I>E. coli</I> expressing the codon-optimized LC-Cutinase protein. X1000 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 23:</B> A scanning electron microscope (SEM) image of a PET pellet after 2 days of incubation in a liquid LB broth with <I>E. coli</I> expressing the codon-optimized LC-Cutinase protein. X1000 magnification
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 24:</B> A scanning electron microscope (SEM) image of a PET pellet after 2 days of incubation in a liquid LB broth with <I>E. coli</I> expressing the codon-optimized LC-Cutinase protein. X5000 magnification.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            As seen above, after incubation with the LC-Cutinase expressing bacteria, the surface of the PET is rougher than the control. We also notice that the surface of the PET looks more porous and the rod shaped, high crystallinity PET, is more exposed than in the control.    
                            Moreover, when comparing the bacteria expressing the LC-Cutinase gene and the bacteria transformed with an empty vector backbone, we see a clear difference in the texture of the PET. We assume that this is the result of the PET degradation activity of the LC-Cutinase enzyme. 
                        </P><P>
                            Please note, that when comparing the appearance of degraded PET between LC-Cutinase degraded PET and <I>R. ruber</I> degraded PET, they looked the                
                            same, however, the difference was that it took a month to <I>R. ruber</I> to degrade PET to the observed picture (Fig. 8-10) while it only took two days to 
                            LC-Cutinase to reach the same. Although, this is not a quantitative measure, it is an indication of the enzyme’s high activity.  
                        </P></DIV></DIV></DIV></DIV><DIV class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="resultsText"><DIV class="resultsHeadLine"><I>P. putida</I> antibiotic resistances test results
                        </DIV><P>
                            As part of our work with the <I>P. putida</I> KT2440 strain we wanted to test its antibiotic resistances, so we can select the appropriate cloning vectors.
                        </P><P>
                            Here are our results:
                        </P><DIV class="table-responsive"><TABLE class="table table-bordered"><THEAD><TR><TH>Antibiotics</TH><TH>Concentration [μg/mL]</TH><TH>Resistant</TH></TR></THEAD><TBODY><TR><TD><B>Ampicillin</B></TD><TD>100</TD><TD><B>Yes</B></TD></TR><TR><TD>Chloramphenicol</TD><TD>34.1</TD><TD><B>Yes</B></TD></TR><TR><TD>Ampicillin + Chloramphenicol</TD><TD>100 + 34.1</TD><TD><B>Yes</B></TD></TR><TR><TD>Spectinomycin</TD><TD>100</TD><TD><B>No</B></TD></TR><TR><TD>Spectinomycin X2</TD><TD>200</TD><TD><B>No</B></TD></TR><TR><TD>Streptomycin</TD><TD>100</TD><TD><B>No</B></TD></TR><TR><TD>Streptomycin X2</TD><TD>200</TD><TD><B>No</B></TD></TR><TR><TD>Kanamycin</TD><TD>25</TD><TD><B>No</B></TD></TR><TR><TD>Kanamycin X2</TD><TD>50</TD><TD><B>No</B></TD></TR><TR><TD>Ampicillin + Chloramphenicol + Streptomycin</TD><TD>100 + 34.1 + 100</TD><TD><B>No</B></TD></TR><TR><TD>Control- No Antibiotics</TD><TD>0</TD><TD><B>Yes</B></TD></TR></TBODY></TABLE></DIV><P>
                            Analysis of these results showed that <I>P. putida</I> KT2440 is sensitive to Kanamycin, Streptomycin and Spectinomycin.
                        </P><P>
                            These results led us to choose the pSEVA224 and pSEVA434 vectors - with Kanamycin and Spectinomycin resistances respectively.</P></DIV></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="resultsText"><DIV class="resultsHeadLine"><I>P. putida</I> carbon source utilization results
                        </DIV><P>
                            In order to test <I>P. putida</I>'s ability to utilize TPA and PCA as a carbon source we measured O.D<SUB>610</SUB> values for a period of ~100 hours with <I>P. putida</I> in liquid M9 media with either TPA or PCA as a carbon source.
                            To prevent contamination, we used Ampicillin and Chloramphenicol (at 100μg/mL and 34.1μg/mL respectively) as selection markers.
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 25:</B><I>P. putida</I> KT2440 grown in liquid M9 media with different concentrations of PCA for a period of 96 hours. 5 and 10mM concentration were tested twice (&quot;a/b&quot;).
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 26:</B><I>P. putida</I> KT2440 grown in liquid M9 media with different concentrations of TCA for a period of 136 hours. 20 and 30mM concentration were tested twice (&quot;a/b&quot;).
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            From these results we conclude that <I>P. putida</I> can utilize PCA as a sole carbon source, as previously reported. In contrast, our results show no growth with TPA, which lead us to the conclusion that <I>P. putida</I> cannot utilize it as a sole carbon source.
                        </P><P>
                            Reviewing these results leads us to the conclusion that the TPA degradation pathway must be cloned into the <I>P. putida</I> bacterium for efficient PET degradation. 
                        </P></DIV></DIV></DIV></DIV><DIV class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="resultsText"><DIV class="resultsHeadLine"><I>E. coli</I> ethylene glycol (EG) utilization results
                        </DIV><P>
                            Here are the results of <I>E. coli</I> strain BL21's growth with liquid M9 media containing ethylene glycol as a sole carbon source:
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 27:</B><I>E. coli</I> strain BL21 grown in liquid M9 media with different concentrations of EG for a period of 96 hours.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            We can conclude from these results that our <I>E. coli</I> strain BL21, like other  <I>E. coli</I> strains, can utilize EG as a sole carbon source, with an optimal concentration of 30mM.
                        </P></DIV></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="resultsText"><DIV class="resultsHeadLine">
                            Examining the utilization of PET by <I>E. coli</I> expressing LC-Cutinase
                        </DIV><P>
                            In order to test the PET degradation ability of the LC-Cutinase enzyme variants, all variants were grown on M9 minimal medium plates with shredded PET pellets as a sole carbon source to test their ability to degrade PET. 
                            (for detailed instructions on the preparation of the media see <A href="https://2016.igem.org/Team:BGU_ISRAEL/Protocols">Protocols</A>).
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 28:</B><I>E. coli</I> expressing the codon-optimized(CO) LC-Cutinase grown on a M9 soft agar plate with shredded PET pellets as a sole carbon source. 
                                            Colonies are marked in red.
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 29:</B><I>E. coli</I> expressing the F4 LC-Cutinase variant grown on a M9 soft agar plate with shredded PET pellets as a sole carbon source. 
                                            Colonies are marked in red.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 30:</B><I>E. coli</I> expressing the F7 LC-Cutinase variant grown on a M9 soft agar plate with shredded PET pellets as a sole carbon source. 
                                            Colonies are marked in red.
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 31:</B><I>E. coli</I> expressing the R4 LC-Cutinase variant grown on a M9 soft agar plate with shredded PET pellets as a sole carbon source. 
                                            Colonies are marked in red.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 32:</B><I>E. coli</I> expressing the R7 LC-Cutinase variant grown on a M9 soft agar plate with shredded PET pellets as a sole carbon source. 
                                            Colonies are marked in red.
                                        </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 33:</B><I>E. coli</I> expressing the W.T LC-Cutinase grown on a M9 soft agar plate with shredded PET pellets as a sole carbon source. 
                                            Colonies are marked in red.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 34:</B><B>Control.</B><I>E. coli</I> expressing the W.T. LC-Cutinase grown on a M9 agar plate no carbon source.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            As seen in the images above, all LC-Cutinase variants were able to grow on the plates containing shredded PET, suggesting that the PET is degraded and <I>E. coli</I> is utilizing at least one of its products (we assume - ethylene glycol). 
                            We cannot conclude which variant degrades the PET with the highest efficiency, as the PET in the plates is not evenly spread and the amount of PET degraded cannot be measured in this assay. </P><P>
                            However, we have clearly shown that our bacteria can utilize PET as a carbon source using the LC-Cutinase protein, which is a proof-of-concept for one part of our project.
                            Looking at the entire design- our biofuel cell will consist of E. coli expressing LC-Cutinase that degrades the PET and consumes ethylene glycol (it includes the previous results in the E. coli ethylene glycol utilization results). We have managed to show that out E. coli will be able to break down PET in conditions similar to those in our designed biofuel cell and to survive without any other carbon source.
                        </P><P>
                            Moreover, our new LC-Cutinase genes consist of a PelB leader sequence that has been codon optimized to maximize expression levels in E. coli. The PelB leader sequence is a previously registered part (<A href="http://partsregistry.org/wiki/index.php/Part:BBa_J32015">Part:BBa_J32015</A>).
                        </P></DIV></DIV></DIV></DIV><DIV class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="resultsText"><DIV class="resultsHeadLine"><I>P. putida</I> Fuel Cell experiments
                        </DIV><P>
                            To verify our hypothesis that PCA (Protocatechuic acid) is a suitable electron transfer mediator for our fuel cells we measured its potential using cyclic voltammetry (CV). 
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 35:</B> Voltagramm of PCA.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            Calculating the values from the CV measurements we found that the substance had a middle point Potential of E<SUP>0'</SUP>= 174 mV vs. an Ag/AgCl reference electrode.  
                            We concluded that the potential was low enough to function as an electron transfer mediator in the system's anode, although not an ideal one. 
                        </P><P>
                            In order to test the ability of <I>P. putida</I> to grow in a biofuel cell anode we had to assess its ability to produce an electroactive biofilm on graphite plates used in a biofuel cell. 
                            We incubated the bacteria in a liquid LB medium and hanged a graphite plate half inside the medium and half outside.
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 36:</B> A <I>P. putida</I> biofilm on a graphite disc.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            As seen from the picture above, our bacterium is able to form a biofilm on the graphite electrode.  
                        </P><P>
                            After testing the compatibility of each component of our fuel cell, we ran several tests to assess its efficiency:
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 37:</B> Power-output as a function of voltage using an Ag-AgCl reference electrode.
                                        </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                            We can see a large increase in power output induced in the sample containing bacteria in the 250-500 mV range, 
                            compared to the control. In the second day of measurements, there was a decline in power output probably due to cell death. 
                            The last measurement is not shown due to contamination in the control cell.  
                        </P><P>
                            The internal resistance was measured in the first day with values of 60 Ω for the cell containing bacteria and 1000 Ω for the control. Which is an indication to the electroactivity of <I>P. putida</I> compared to control, with relatively high internal resistance.                        </P><P>
                            As proof of concept in the biofuel part of our project, Our test results may suggest that there is a potential in constructing a biofuel cell based on the electrochemical activity of <I>P. putida</I> KT 2440 with protocatechuic acid as an electron mediator. 
                            That being said, more experiments are required in order to confirm our hypothesis and determine the proper concentrations of the protocatechuic acid and characterize the mechanisms with which it mediates <I>P. putida</I>'s electron transfer. 
                            It should be mentioned that protocatechuic acid polymerizes in the presence of amine groups, a reaction that could interfere with the electron transfer, hence other more potent mediators will be explored including the operation of the biofuel cell without mediators, since <I>P. putida</I> are reported to produce their endogenous electron transfer mediators.
                        </P><P>
We have demonstrated that creating a biofilm on a graphite anode can be easily done, making this part of our system applicable. Furthermore, we succeeded in generating electricity using a relatively unfavorable condition for the bacterium. Hence showing a potential in in building the final system in which the bacteria will be incubated in similar conditions and under the similar stresses.    
</P></DIV></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="resultsText"><DIV class="resultsHeadLine">
                            Characterizing <I>E. coli</I> growth inside a dialysis bag
                        </DIV><P>
                            Our symbiotic approach relies on mutual growth of <I>E. coli</I> and <I>P. putida</I>. For this approach to succeed <I>E. coli</I> must successfully grow inside a dialysis bag.
                        </P><P>
                            In this experiment we grew <I>E. coli</I> strain BL-21 transformed with the pACYC vector containing the LC-Cutinase W.T. gene and induced with IPTG in a liquid M9 medium with PET.
                        </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><B>Figure 38:</B><I>E. coli</I> strain BL-21 transformed with the the pACYC vector containing the LC-Cutinase W.T. gene grown in liquid M9 media with PET as a sole carbon source. The drop in O.D value at 61 hours was caused by spillage of some of the content from the dialysis bag.
      </FIGCAPTION></FIGURE></DIV></DIV><P>
                            As seen from the results, our <I>E. coli</I> was able to grow with only PET as a sole carbon source, suggesting it utilized PET's degradation product, ethylene glycol, as a carbon source.
                        </P><P>
                            Moreover, we see that <I>E. coli</I> remained viable throughout the experiment, verifying our hypothesis that it can grow inside a dialysis bag, and suggesting that our symbiotic design using a dialysis bag is plausible.
                        </P><P>
We consider that result as an important step and a proof-of-concept of our final goal.
</P></DIV></DIV></DIV></DIV><DIV class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="resultsText"><DIV class="resultsHeadLine">
                        Characterization of the co-culturing of <I>E. coli</I> and <I>P. putida</I> separated by a dialysis bag - a controlled symbiotic approach.
                    </DIV><P>
                        In order to assess the ability of our two chosen bacterial species - <I>E. coli</I> and <I>P. putida</I> to grow in symbiosis, a key element in our project's design, we cultivated both bacteria in a liquid LB medium, separated by a dialysis bag. 
                    </P><P>
                        After 72 hours of growth we transfered samples taken from inside and outside the dialysis bag to petri dishes. The petri dishes were incubated over-night at 37<SUP>o</SUP>C. Samples were taken for analysis using colony-PCR to identify the strains. 
                    </P><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><B>Figure 39:</B> Colony-PCR results used to the detect the presence of the pcaD gene(792bp) from <I>P. putida</I> shown on 1% agarose gel. Samples taken from plates 1,2,3,5 and 8 were positive, suggesting they contained <I>P. putida</I>.
                                </FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><B>Figure 40:</B> Colony-PCR results used to the detect the presence of the aldA gene(1440bp) from <I>E. coli</I> shown on 1% agarose gel. Samples taken from plates 1,3,6,7 and 10 were positive, suggesting they contained <I>E. coli</I>.
                                </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 41:</B> Plate number 1 - samples taken from inside and outside the dialysis bag and grown over-night on a petri dish containing Chloramphenicol.  As seen from the image, both samples show growth, corresponding to <I>P. putida</I>'s endogenic resistance and <I>E. coli</I>'s plasmid acquired resistance to Chloramphenicol.
                                    </P></FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 42:</B> Plate number 2 - samples grown on a petri dish containing Ampicillin. As seen from the image, both samples show growth, suggesting that both samples contain <I>P. putida</I>, which has endogenic resistance to Ampicillin. 
                                    </P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-lg-6"><FIGURE><FIGCAPTION><P><B>Figure 43:</B> Plate number 4 - samples taken from inside and outside the dialysis bag and grown over-night on a petri dish containing Kanamycin. As seen from the image, no growth is visible in either sample. This is expected as neither <I>P. putida</I> nor <I>E. coli</I> is resistant to Kanamycin.
                                    </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                        As seen from the results above, both bacteria maintained viability using our dialysis bag apparatus during the experiment.
                    </P><P><B> Some results show presence of both bacteria inside and outside the bag. Those results are a consequence of a rupture in the dialysis membrane which occurred during the experiment. </B></P><P><B> Nevertheless, our experiment clearly demonstrates the ability of <I>E. coli</I> and <I>P. putida</I> to exist in symbiosis. </B></P><P>
                        Our two dialysis bag experiments clearly demonstrate the plausibility of our designs. We managed to demonstrate PET degradation and utilization of its product, EG, by <I>E. coli</I> surrounded by a dialysis membrane and a consortium consisted of <I>E. coli</I> and <I>P. putida</I> with the same apparatus. Results from both experiments show that our designs work under conditions similar to our final product and proves our project's applicability.
                    </P></DIV></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="resultsText"><DIV class="resultsHeadLine">
                        Detection of TPA in plates containing PET and E. coli using fluorescence
                    </DIV><P>
After seeing bacterial growth on our PET plates, we wanted to combine our protocols of PET utilization and to detect the presence of TPA in our plates. If our speculation was correct, the LC-Cutinase was supposed to be expressed from the <I>E. coli</I>, degrade the PET, and the bacterium could consume ethylene glycol from the degradation products. If that was the case, TPA would be present in the plate. 
</P><DIV class="row"><DIV class="col-lg-10"><FIGURE><FIGCAPTION><P>
                                        Figure 44: PET plates (prepared as previously described in the PET Utilization experiment (<A href=" https://2016.igem.org/Team:BGU_ISRAEL/Experiments">Experiments Page</A>), after colony growth, were exposed to UV light (as described in the Terephthalic acid detection assay (<A href=" https://2016.igem.org/Team:BGU_ISRAEL/Experiments">Experiments Page</A>)), then tested in a fluorescence gel imager to detect TPA presence by excitation in 314nm end detecting emission at 426nm. For control, TPA presence was detected in plates containing <I>E. coli</I> transformed with an empty vector plasmid (the same vector used for expression of LC-Cutinase), and no bacteria at all. 
                                    </P></FIGCAPTION></FIGURE></DIV></DIV><P>
                        As shown, plates containing <I>E. coli</I> expressing LC-Cutinase showed higher emission levels, which indicates of higher presence of TPA in the plates in question. The control plates still have certain background emission, but it can be due to background emission from the polystyrene plates. This experiment, although it cannot quantify the degradation efficiency, is another indication of the existence and activity of the LC-Cutinase enzyme, and the ability of <I>E. coli</I> expressing our LC-Cutinase to utilize its degradation product- ethylene glycole. 
                    </P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row footerColor"><DIV class="col-lg-4 text-center"><H4>Address:</H4><P>
                        Ben-Gurion University of the Negev
                        Ben Gurion 1, Beer Sheva 8410501, Israel
                    </P><H5><B><U>Mail:</U></B> igembgu2016@gmail.com
                    </H5></DIV><DIV class="col-lg-4 text-center"><H4>Connect With Us!</H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>