<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BGU_ISRAEL_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BGU ISRAEL/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>PlastiCure</TITLE></P><NAV class="navbar navbar-inverse navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="mainmenu nav navbar-nav"><LI class="dropdown" id="HomeB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL">Home</A></LI><LI class="dropdown" id="projectB" style="background-color: #CB99C9;"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Achievements">Achievements</A></LI></UL><LI class="dropdown" id="teamB"><A href="https://2016.igem.org/Team:BGU_ISRAEL/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/TeamMembers">Members</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/MeetUniversity">Meet Our University</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/MeetTown">Meet Our Town</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Collaborations">Collaborations</A></LI><LI class="dropdown" id="partsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Parts#Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/ImprovedParts">Improved Parts</A></LI><LI class="dropdown" id="humanB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Integrated_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/PlasticArt">Plastic Art</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/EthicsAndSafety">Ethics &amp; Safety</A></LI><LI class="dropdown" id="attributionsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Attributions">Attributions</A></LI><LI class="dropdown" id="awardsB"><A class="dropdown-toggle" href="https://2016.igem.org/Team:BGU_ISRAEL/Awards">Awards</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Measurements">Measurements</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Proof">Proof Of Concept</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/HP/Silver">HP - Silver</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/HP/Gold">HP - Gold</A></LI><LI><A href="https://2016.igem.org/Team:BGU_ISRAEL/Engagement">Engagements</A></LI></DIV></DIV></NAV><DIV class="container-fluid"><DIV class="row"><DIV class="projectColor"><P>Design</P></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><P class="bigAssHeader">Rational Mutagenesis</P><DIV class="DesignText"><P><B>Protein Mutagenesis</B></P><P>
                            Based on the structure of LC-Cutinase, we decided to use a rational mutagenesis approach, in which we introduce various mutations to the enzyme's sequence. The mutations were designed using the PROSS (Protein Repair One Stop Shop) algorithm, developed by the Fleishman Lab from the Weizmann Institute (Goldenzweig et al 2016).
                          </P><P>
                         The algorithm uses the following steps in selecting the mutations:
                           </P><P>
                            A sequence alignment was generated between the target wild-type sequence and other homologous sequences, from which several mutations were chosen.
                        </P><P>
                            Each mutant’s structure was compared to the wild-type structure, 
                            and the energy difference between them was calculated (ΔΔG<SUB>calc</SUB>).
                        </P><P>
                            By computing the effects of each mutation separately, 
                            rather than in combination with others, design choices which included several ‘Potentially Stabilizing’ mutations (‘Potentially Stabilizing’ are mutations with ΔΔG<SUB>calc</SUB> ≤ -0.45), 
                            were chosen, rather than designs that included several mutations, 
                            computed together and could include one or more non ‘Potentially Stabilizing’ mutations, but with a total ΔΔG<SUB>calc</SUB> ≤ -0.45.
                        </P><P>
                            Here are a few examples of mutations present in our mutated variants:
                        </P><DIV class="row"><DIV class="col-lg-4"><FIGURE><FIGCAPTION>
                                        A249P - Proline (shown in yellow) is added to increase the rigidity of the protein structure.
                                    </FIGCAPTION></FIGURE></DIV><DIV class="col-lg-4"><FIGURE><FIGCAPTION>
                                        M259F - A ‘hole’ inside the protein core could be destabilizing to the protein structure.
                                        Phenylalanine (shown in yellow) with a benzyl side group (hydrophobic) is used to ‘fill’ the hole, thus making the protein more stable.
                                    </FIGCAPTION></FIGURE></DIV><DIV class="col-lg-4"><FIGURE><FIGCAPTION>
                                        S257A - Polar side groups, such as a hydroxyl in Serine, with no hydrogen bonds, are destabilizing as the protein core should
                                        be hydrophobic. A mutation changing Serine to Alanine should increase stability. 
                                    </FIGCAPTION></FIGURE></DIV></DIV><P><B>Cells, Plasmids and Enzymes</B></P><P><I>E. coli</I> strain MG1655 was used for cloning of the LC-Cutinase genes into the pACYC vector backbone. <I>E. coli</I> strain BL21 was used for protein expression. 
                            The wild type LC-Cutinase gene with the pelB leader sequence was obtained from the <A href="https://2012.igem.org/Team:UC_Davis">UC-Davis 2012 iGEM team</A> (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K936013">Part:BBa_K936013</A>) and was amplified using primers containing restriction sites that match the pACYC backbone. 
                            The amplified gene was restricted with XhoI and BglII and ligated into pACYC using the T4 ligase enzyme.
                        </P><P>
                            After validating positive clones, activity tests were performed using a pNP-butyrate assay and PET degradation assays.
                        </P></DIV></DIV></DIV></DIV><DIV class="container-fluid lightgold"><DIV class="row"><DIV class="col-lg-8"><DIV class="DesignText"><P class="bigAssHeader">Metabolic Pathway Engineering</P><P>
                            Two inserts were designed to transform the genes necessary for the terephthalate metabolic pathway to the 
                            <I>Pseudomonas putida</I> bacterium. Two vectors for cloning were chosen and ordered from the Standard European Vector 
                            Architecture’s (SEVA) collection. The 2 vectors chosen are pSEVA224 and pSEVA434. We have chosen the pSEVA plasmids because
                            they have been previously used in the <I>P. putida</I> KT2440 strain by the laboratory of Prof. D. Taufik at the Weizmann Institute.
                        </P><P>
                            Two inserts were designed –
                        </P><OL class="orderedListDesign"><LI>
                                Containing 4 genes from the same strain of <I>C. testosteroni</I> for the degradation of Terephthalic acid to
                                Protocatechuic acid – tphA1, tphA2, tphA3, tphB. 
                                </LI><LI>Containing 3 genes from <I>C. testosteroni</I> strain KF-1 for the transport of Terephthalic acid from the periplasmic 
                                space into the cytoplasm – tphC, tctA, tctB. 
                                </LI></OL><P>
                            The genes involved in the Terephthalate degradation pathway and transporter were acquired from the Darmstadt iGEM team, 2012.
                            (<A href="https://2012.igem.org/Team:TU_Darmstadt">Link</A>). 
                        </P><P>
                            Proximal to each gene, we added the broad-host G10L ribosome binding site, derived from bacteriophage T7 to increase expression. 
                            To provide us with a versatile vector, that will allow excision of a specific sequence from the designed insert if needed, 
                            we added a restriction site between each element (gene or RBS). 
                            The restriction site had to be unique to avoid digestion of the genes or the vector backbone.
                        </P><P>
                            Flanking each insert we designed and added homologous sequences to the desired target vector (224 or 434), for cloning with the NEBuilder® HiFi 
                            DNA Assembly Cloning Kit. Furthermore, we designed an alternative cloning strategy - using restriction and ligation. The flanking sequences also contained  restriction sites for cloning into the multiple cloning site(MCS) of each vector.
                        </P><P>
                            Our design for efficient degradation of PET required the symbiosis our our two chosen bacteria - <I>E. coli</I> and
                            <I>P. putida</I>, so we devised an apparatus to enable their mutual growth without contact. 
                            Our chosen approach utilized a dialysis membrane with a cut-off of 12-14kDa, which is lower than LC-Cutinase's molecular mass,
                            to enable the diffusion of the PET degradation products outside the bag for utilization by <I>P. putida</I>, while enabling
                            a large concentration of LC-Cutinase inside the bag for efficient degradation of PET.
                            Shredded PET was added to the dialysis bag and a magnetic stirrer was used to prevent precipitation.
                        </P></DIV></DIV></DIV></DIV><DIV class="container-fluid whiteback"><DIV class="row"><DIV class="col-lg-8"><DIV class="DesignText"><P class="bigAssHeader">Microbial Fuel Cell</P><P>
                            In order to build a self-sustaining system in which part of the energy from PET could be utilized to make clean energy, we 
                            decided to use microbial fuel cells (MFC). 
                        </P><P>
                            We chose a bacterium that is proven to be electrochemically active (Timur et al 2007). Furthermore, it was essential that
                            the bacterium would be able to form a biofilm over the electrode in attempt to improve the MFCs performance (Nielsen, L. 2012).
                            <I>P. putida</I> KT2440 meets these requirements.
                        </P><P>
                            At first, we examined the possibility of adding an electron carrier to the fuel cell, 
                            as it was shown to be effective in enhancing the MFCs performance and was demonstrated using this species. We decided to use
                            protocatechuic acid as it is part of the terephthalate metabolic pathway, which enables us to reduce the use of external
                            reagents and to create a self-sustaining system. Protocatechuic acid had to meet four criteria: 
                        </P><OL><LI>
                                The mediator redox couple should have a well-defined stoichiometry.
                            </LI><LI>
                                The mediator should be soluble and stable under the conditions present in the fuel cell.
                            </LI><LI>
                                The mediator’s formal potential should be adequate to the enzyme potential.
                            </LI><LI>
                                It should have a fast heterogeneous electrode exchange with the electrode and a fast homogenous electron exchange with the enzyme redox center or enzyme cofactor (Nuñez, M. 2005). 
                            </LI></OL><P>                       To examine some of those criteria regarding protocatechuic acid we plan to use cyclic voltammetry. 
                        </P></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row footerColor"><DIV class="col-lg-4 text-center"><H4>Address:</H4><P>
                        Ben-Gurion University of the Negev
                        Ben Gurion 1, Beer Sheva 8410501, Israel
                    </P><H5><B><U>Mail:</U></B> igembgu2016@gmail.com
                    </H5></DIV><DIV class="col-lg-4 text-center"><H4>Connect With Us!</H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>