<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_WashU_StLouis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:WashU StLouis</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Washington University - Penn State iGEM</TITLE></P><HEADER class="row"><DIV class="container"><DIV class="intro-text"><DIV class="intro-lead-in">Welcome To Our Website!</DIV><DIV class="intro-heading"> WashU and Penn State   iGEM 2015</DIV><A href="https://2015.igem.org/Team:WashU_StLouis/description#description" class="page-scroll btn btn-xl">Project Description</A><A href="https://2015.igem.org/Team:WashU_StLouis#contact" class="page-scroll btn btn-xl">Let's Talk</A><A href="https://static.igem.org/mediawiki/2015/d/d2/2016_WashU_iGEM_Application.pdf" class="btn btn-xl">Apply for the 2016 iGEM Team!</A></DIV></DIV></HEADER><NAV class="navbar navbar-default navbar-fixed-top"><DIV class="container"><DIV class="navbar-header page-scroll"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand page-scroll" href="#page-top">Nitrogen</A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A class="page-scroll" href="#overview">Overview</A></LI><LI><A class="page-scroll" href="#team">Team</A></LI><LI><A class="page-scroll" href="#collaboration">Collaboration</A></LI><LI><A class="page-scroll" href="#sponsors">Sponsors</A></LI><LI><A class="page-scroll" href="#contact">Contact</A></LI><LI><A href="#reportBugsModal" class="overview-link" data-toggle="modal">Report Bugs</A></LI><LI><A class="page-scroll" href="#sitemap">More</A></LI></UL></DIV></DIV></NAV><DIV class="container"><DIV class="col-lg-12 text-center"><H3 class="section-subheading text-muted"><A class="page-scroll" href="https://static.igem.org/mediawiki/2015/d/d2/2016_WashU_iGEM_Application.pdf">Click here to apply for the 2016 iGEM Team!</A></H3></DIV><SECTION id="overview" class="bg-light-gray row sectionNum1"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 text-center"><H2 class="section-heading">Project Overview</H2><H3 class="section-subheading text-muted">Washington University and Penn State iGEM</H3></DIV></DIV><DIV class="row"><DIV class="col-md-4 col-sm-6 overview-item"><DIV class="overview-caption"><H4>Last Year</H4><P class="text-muted">Learn from history</P></DIV></DIV><DIV class="col-md-4 col-sm-6 overview-item"><DIV class="overview-caption"><H4>Mission Statement</H4><P class="text-muted">The goal in a sentence</P></DIV></DIV><DIV class="col-md-4 col-sm-6 overview-item"><DIV class="overview-caption"><H4>Nitrogen Project</H4><P class="text-muted">Ain't broke, but we got to fix it</P></DIV></DIV><DIV class="col-md-4 col-sm-6 overview-item"><DIV class="overview-caption"><H4>The Big Picture</H4><P class="text-muted">Why did we do this?</P></DIV></DIV><DIV class="col-md-4 col-sm-6 overview-item"><DIV class="overview-caption"><H4>Materials and Methods</H4><P class="text-muted">Imitation is the sincerest form of flattery.</P></DIV></DIV><DIV class="col-md-4 col-sm-6 overview-item"><DIV class="overview-caption"><H4>Discussion</H4><P class="text-muted">Explaining the results</P></DIV></DIV></DIV></DIV></SECTION><SECTION id="team" class="bg-white row sectionNum2"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 text-center"><H2 class="section-heading">Team</H2><H3 class="section-subheading text-muted">NitroGeniuses</H3></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><UL class="timeline"><LI class="row"><DIV class="timeline-panel"><DIV class="timeline-heading"><H4>Jessica O'Callaghan</H4><H4 class="subheading">Chemical Engineering</H4></DIV><DIV class="timeline-body"><P class="text-muted">Penn State</P></DIV></DIV></LI><LI class="row timeline-inverted"><DIV class="timeline-panel"><DIV class="timeline-heading"><H4>Laura Beebe</H4><H4 class="subheading">Biochemistry and Molecular Biology</H4></DIV><DIV class="timeline-body"><P class="text-muted">Penn State</P></DIV></DIV></LI><LI class="row "><DIV class="timeline-panel"><DIV class="timeline-heading"><H4>Blake Actkinson</H4><H4 class="subheading">Biomedical Engineering</H4></DIV><DIV class="timeline-body"><P class="text-muted">Washington University in St. Louis</P></DIV></DIV></LI><LI class="row timeline-inverted"><DIV class="timeline-panel"><DIV class="timeline-heading"><H4>Charlotte Bourg</H4><H4 class="subheading">Biochemistry and Spanish</H4></DIV><DIV class="timeline-body"><P class="text-muted">Washington University in St. Louis</P></DIV></DIV></LI><LI class="row "><DIV class="timeline-panel"><DIV class="timeline-heading"><H4>David Ayeke</H4><H4 class="subheading">Computer Engineering</H4></DIV><DIV class="timeline-body"><P class="text-muted">Washington University in St. Louis</P></DIV></DIV></LI><LI class="row  timeline-inverted"><DIV class="timeline-panel"><DIV class="timeline-heading"><H4>Mike Toomey</H4><H4 class="subheading">Biomedical Engineering</H4></DIV><DIV class="timeline-body"><P class="text-muted">Washington University in St. Louis</P></DIV></DIV></LI><LI class="row "><DIV class="timeline-panel"><DIV class="timeline-heading"><H4>Caroline Focht</H4><SPAN class="text-muted">- 2014 iGEM Team Member</SPAN><H4 class="subheading">Chemistry and Biochemistry</H4></DIV><DIV class="timeline-body"><P class="text-muted">Washington University in St. Louis</P></DIV></DIV></LI><LI class="row"><DIV class=" col-lg-12" style="text-align:center;"><H4>Be Part
                    Of Our
                    Story!</H4><A href="#collaboration" class="page-scroll btn btn-sm btn-warning">Collaboration</A></DIV></LI></UL></DIV></DIV></DIV></SECTION><SECTION id="collaboration" class="bg-light-gray row  sectionNum3"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 text-center"><H2 class="section-heading">Collaboration</H2><H3 class="section-subheading text-muted"><A class="page-scroll" href="#contact">Looking for Collaborators?</A></H3></DIV></DIV><DIV class="row text-center"><DIV class="col-lg-12"><H2 class="collaboration-heading">Vanderbilt<P>Our team worked closely with Vanderbilt University’s iGEM team over the summer to help them troubleshoot issues they had been having regarding a promoter and to come up with questions to ask our panel. Read more <A href="https://2015.igem.org/Team:WashU_StLouis/collaborations">here</A></P></H2><H2 class="collaboration-heading">Help Us Out?</H2><P>Washington University in St. Louis genetic engineering panel
                  Hey iGEM teams!
                  Here at Wash U we’re planning a panel on genetic engineering to be held in the fall for the Wash U community. We’re hoping to address some gray areas of synthetic biology and genetic engineering in agriculture, from safety and regulation to ethics and implications. We’re still finalizing our panelists, but hope to have experts from Monsanto and the Danforth Plant Science Center. We’re also in touch with some Wash U professors from the biology, philosophy, and anthropology departments.
                  We plan to start off the discussion with our own questions before opening it up to the audience. We’ve come up with a list of questions, but would love to get the input of some other iGEM teams. If you have questions you think would start interesting discussion at the panel, we’d love to hear them! Email us at <EMAIL>washu.igem@gmail.com</EMAIL>.
                  Best, Wash U iGEM
                </P></DIV></DIV></DIV></SECTION><SECTION id="advisors" class="bg-white row sectionNum4"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 text-center"><H2 class="section-heading">Our Amazing P.I's</H2><H3 class="section-subheading text-muted">The people that made it all possible</H3></DIV></DIV><DIV class="row"><DIV class="col-sm-4"><DIV class="team-member"><H4>Tae Seok Moon</H4><P class="text-muted">Faculty Advisor - Washington University</P></DIV></DIV><DIV class="col-sm-4"><DIV class="team-member"><H4>Fuzhong Zhang</H4><P class="text-muted">Faculty Advisor - Washington University</P></DIV></DIV><DIV class="col-sm-4"><DIV class="team-member"><H4>Costas Maranas</H4><P class="text-muted">Faculty Advisor - Pennsylvania State University</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-4 col-xs-6"><DIV class="team-member"><H4>Cheryl Immethun</H4><P class="text-muted">Washington University iGEM Advisor</P></DIV></DIV><DIV class="col-sm-4 col-xs-6"><DIV class="team-member"><H4>Thomas Mueller</H4><P class="text-muted">Penn State iGEM Advisor</P></DIV></DIV><DIV class="col-sm-4 col-xs-6"><DIV class="team-member"><H4>Andrea Balassy</H4><P class="text-muted">Washington University iGEM Advisor</P></DIV></DIV><DIV class="col-sm-4 col-xs-6"><DIV class="team-member"><H4>Carlos Barba</H4><P class="text-muted">Washington University iGEM Advisor</P></DIV></DIV><DIV class="col-sm-4 col-xs-6"><DIV class="team-member"><H4>Yi Xiao</H4><P class="text-muted">Washington University iGEM Advisor</P></DIV></DIV><DIV class="col-sm-4 col-xs-6"><DIV class="team-member"><H4>Ray Henson</H4><P class="text-muted">Washington University iGEM Advisor</P></DIV></DIV><DIV class="col-sm-4 col-xs-6"><DIV class="team-member"><H4>Young Je Lee</H4><P class="text-muted">Washington University iGEM Advisor</P></DIV></DIV></DIV></DIV></SECTION><SECTION id="contact" class="bg-white row  sectionNum6 sectionNum7"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 text-center"><H2 class="section-heading">Contact Us</H2><H3 class="section-subheading text-muted">We can't do this alone</H3><DIV class="alert alert-danger"><BUTTON type="button" class="close" data-dismiss="alert" aria-hidden="true">×</BUTTON><STRONG>This form will open your default mail client (i.e Outlook). If you do not have one installed, email us at  WashU iGEM washu.igem@gmail.com</STRONG></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><FORM name="sentMessage" id="contactForm" novalidate=""><DIV class="row"><DIV class="col-lg-12 text-center"><BUTTON type="submit" class="btn btn-xl">Send Message</BUTTON></DIV></DIV></FORM></DIV></DIV></DIV></SECTION><DIV class="overview-modal modal fade" id="overviewModal1" tabindex="-1" role="dialog" aria-hidden="true"><DIV class="modal-content"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><DIV class="modal-body"><H2>2014 iGEM Project</H2><P class="item-intro text-muted">Fixing Nitrogen and Sensing Light</P><H4> What was our goal?</H4><P>There were two key components of our project: </P><P>Richard and Caroline were working in the <A href="https://pages.wustl.edu/photo.synth.bio">Pakrasi </A>lab, and had been using genes from cyanobacteria to get nitrogen fixation working in <EM>E. coli</EM>. They were testing nitrogen fixation by running acetylene reduction assays and designing experiments to test the optimal criteria (ie. <EM>E. coli</EM> strains, temperature, nitrogen source) to get maximal results.
                      </P><STRONG> Read more <A href="https://2014.igem.org/Team:WashU_StLouis/Project/nif"> here </A></STRONG><P>Ben and Jeffery worked in the <A href="http://moon.eece.wustl.edu/"> Moon </A> lab, and cloned plasmids to create a system that represses and expresses a fluorescent protein with the presence and absence of light, and were running experiments to test the induction levels compared to various positive and negative controls.</P><STRONG> Read more <A href="https://2014.igem.org/Team:WashU_StLouis/Project/light"> here </A></STRONG><BUTTON type="button" class="btn btn-primary" data-dismiss="modal"> Close Modal</BUTTON></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="overview-modal modal fade" id="overviewModal2" tabindex="-1" role="dialog" aria-hidden="true"><DIV class="modal-content"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><DIV class="modal-body"><H2>Mission Statement</H2><P class="item-intro text-muted">tl;dr Fix Nitrogen</P><SPAN>Caroline Focht</SPAN><P class="text-muted">- 2014 Washington University iGEM member</P><A href="https://www.youtube.com/watch?v=5B73_8vcENI&amp;list=PL6fJjAsbg4Sz1V-Xu0moedDxG9Hbbqx8N">More Videos!!!</A><P> The goal of this year's project is to take the nitrogen fixing <EM>nif</EM> cluster from <EM>Cyanothece</EM>, a cyanobacterium, minify it, and make it work in <EM>E. coli</EM>. </P><UL style="text-align:left;"><LI><A href="#">Designing the plasmid TODO ADD LINkS</A></LI><LI><A href="#">Making it work in E. Coli TODO ADD LINKS</A></LI></UL><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close Modal</BUTTON></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="overview-modal modal fade" id="overviewModal3" tabindex="-1" role="dialog" aria-hidden="true"><DIV class="modal-content"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><DIV class="modal-body"><H2>Summary of the project to date</H2><H3> Phase 1: Electro-Transformation of plasmid pNif51142 into E. Coli strains</H3><EM> - 2014 </EM><P>To successfully transform pNif51142, we used Electro-Transformation, also known as Electroporation.  Electroporation provides a method of transforming E. coli to efficiencies greater than are possible with the best chemical methods. By subjecting mixtures of cells and DNA to exponentially decaying fields of very high initial amplitude, we were able to deliver the plasmid into all of E. coli strains that were tested in the project.</P><STRONG>Results</STRONG><P>According to the gel running and antibiotic testing, bands in the gel and the survival of all strains transformed in antibiotic Kanamycin (there was an cluster of Kanamycin-resistant marker gene in sequence of pNif51142) both prove that the Electro-Transformation was successful.</P><H3> Phase 2: Determine the optimal conditions for cell survival with plasmid pNif51142 in <EM>E. Coli</EM></H3><EM> - 2014</EM><P>Due to the oxidative properties of oxygen, most nitrogenases are irreversibly inhibited by dioxygen, which degradatively oxidizes the Fe-S cofactors. This requires mechanisms for nitrogen fixers to protect nitrogenase from oxygen in vivo. Hence in our experiment, we firstly selected anaerobic condition as part of preparation step for the nitrogenase activity testing</P><UL><LI><STRONG>Testing Media</STRONG>: Minimal M9</LI><LI><STRONG> Carbon Source:</STRONG> Glucose(1mM, 10mM, 100mM)</LI><LI><STRONG> Nitrogen Source:</STRONG> Glutamine(1mM, 10mM, 100mM), Glutamate(1mM, 10mM, 100mM), NH4Cl(1mM, 10mM, 100mM)</LI><LI><STRONG> Temperature:</STRONG> 30°C, 37°C, 40°C</LI><LI><STRONG>pH:</STRONG>6,7,8</LI><LI><STRONG>O<SUB>2</SUB> level:</STRONG>Anaerobic or Aerobic </LI><LI><STRONG>Strains of <EM>e. Coli</EM>:</STRONG> JM109, BL21(DE3), WM1788, Top 10 DH5Α</LI></UL><TABLE class="table table-bordered table-hover table-responsive"><TBODY><TR><TH>Target Strain</TH><TH>JM109</TH><TH>BL21(DE3)</TH><TH>WM1788</TH><TH>Top 10 </TH><TH>DH5Α</TH></TR><TR><TD>Experimental Plates </TD><TD>JM 109 strain w/ plasmid
                                    Antibiotic </TD><TD>BL21(DE3) strain w/ plasmid           Antibiotic
                                  </TD><TD>WM1788 strain w/ plasmid
                                    Antibiotic </TD><TD>Top 10 strain w/ plasmid
                                    Antibiotic</TD><TD>DH5α strain w/ plasmid
                                    Antibiotic
                                  </TD></TR><TR><TD>Positive Control</TD><TD>JM 109 strain w/o plasmid
                                    No antibiotic
                                  </TD><TD>BL21(DE3) strain w/o plasmid
                                    No antibiotic
                                  </TD><TD> WM1788 strain w/o plasmid
                                    No antibiotic
                                  </TD><TD>Top 10 straing w/o plasmid
                                    No Antibiotic</TD><TD>DH5α strain w/o plasmid
                                    No antibiotic</TD></TR><TR><TD>Negative Control </TD><TD> JM 109 strain w/o plasmid
                                      Antibiotic
                                    </TD><TD>BL21(DE3) strain w/o
                                      plasmid 
                                      Antibiotic
                                    </TD><TD>WM1788 strain w/o plasmid
                                      Antibiotic
                                    </TD><TD>Top 10 strain w/o plasmid
                                      Antibiotic</TD><TD>DH5α strain w/out plasmid
                                      Antibiotic
                                    </TD></TR></TBODY></TABLE><STRONG>Results</STRONG><P>In the minimal M9 media, all possible combinations of parameters listed above were tested.
                                None of the concentrations of glucose had any aﬀect on the growth of E. coli. It was expected that as the concentration of glucose increased, the growth of E. coli also increased. However, the variation between the concentrations of glucose may have been too small for a noticeable increase in E. coli growth as the concentration of glucose increased. Also, even the maximum concentration of glucose tested, 100mM, may have been too low to aﬀect the growth of E. coli to an observable extent. Eventually, 10mM was determined to be the optimal glucose concentration for the purpose of least interference possible in solution.
                                NH4Cl as minimal nitrogen source was proven to be not suitable for E. coli growth at any concentration as the OD600 testing results showed that cell density didn’t change throughout the time. It was probably due to the permeability of cell membrane was limited for NH4+ and Cl- ions. Eventually Glutamate at concentration of 10mM supported cell growth the best and thus chosen as part of optimal environment condition.

                                With CASAmino solution’s buffering utility, the pH was controlled little bit below 7 but close to 7.

                                To protect the iron core of the nitrogenase, temperature of 30°C and Anaerobic were both determined not for cell growth but for nitrogenase activity testing, which is the next phase.
                              </P><H3>Phase 3: Measurign nitrogen fixation activity under determined optimal conditions in <EM>E. Coli</EM> Strains. </H3><EM>-2014</EM><P>
                                We used  an Acetylene Reduction Assay to examine the nitrogenase activity for JM109, BL21(DE3), Top10, DH5α at different cell density referred by OD600 values.

                                Acetylene (C2H2) has an triple bond similar to that of atmospheric nitrogen (N2). Because of this structural similarity, the nitrogenase enzyme can cleave the triple bond in acetylene just as it would cleave the triple bond in N2. Ethylene (C2H4) is produced from this enzymatic activity, so a gas chromatograph can be used to detect the presence of ethylene and, consequently, nitrogenase activity.
                              </P><STRONG>Results</STRONG><OL><LI>Of the five E. coli strains tested, JM109 and WM1788 showed strongest nitrogenase activity.</LI><LI>The linear relationship between nitrogen fixation activity and time matches that seen in nature. </LI><LI>Optimal conditions determined:
                                  </LI><LI>Glucose as carbon-source</LI><LI>glutamate as nitrogen source </LI><LI>LB as inoculating media </LI><LI>minimal M9 as testing media for GC assay</LI><LI>anaerobic enviornment</LI><LI> 30°C during the overnight preperation</LI></OL><H3> Phase 4/5 Computationally Modelling an E. Coli system that has Nitrogen Fixation (Penn) and Targeting Nif genes for Overexpression and Silencing (WashU)</H3><EM>-2015</EM><H4 class="text-muted section-heading"><A href="description.html">IGEM 2015 Project Description.</A></H4><P>While last year’s team was able to take the nif cluster from a cyanobacteria and get it to function in E. coli, nitrogenase activity was minimal.  This year’s team is dedicated to increasing the nitrogenase activity of E. coli with the nif cluster from Cyanothece 51142. The team members at Pennsylvania State will use high level computational modeling to optimize metabolic pathways within E. Coli, which will relieve the strain of nitrogen fixation. The team members at Washington University will create a minimal nif cluster of only those genes necessary for nitrogenase production and activation. In addition, they will silence and overexpress several genes in order to maximize nitrogenase activity.</P><P>In doing such, we hope to create a system for nitrogen fixation for transformation into a photosynthetic system. After we come to a greater understanding of how the system works and perfect it, we can move on to working in a more complex organism, such as a cyanobacteria like Synecosystisspp. 6803. </P><P>The end goal is to create plants that can fix their own nitrogen by moving from the cyanobacteria into the chloroplast of the plant. Endosymbiotic theory postulates that cyanobacteria are the ancestors to chloroplasts, so this is the natural progression.     </P><H5>Results</H5><UL><LI>Washington University was able to design and synthesize a new more minimal nif cluster optimize for Nitrogenase activity within E. Coli. Read more on our <A href="https://2015.igem.org/Team:WashU_StLouis/design">Design</A> page.</LI><LI>Penn State was able to compile a list of gene knockouts and metabolic addatives that could promote Nitrogenase activity within E. Coli. Read more on our <A href="https://2015.igem.org/Team:WashU_StLouis/modeling">Modeling</A> page.</LI></UL><BUTTON type="button" class="btn btn-primary" data-dismiss="modal"> Close Project</BUTTON></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="overview-modal modal fade" id="overviewModal4" tabindex="-1" role="dialog" aria-hidden="true"><DIV class="modal-content"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><DIV class="modal-body"><H2>The Big Picture</H2><P class="item-intro text-muted">Haber-Bosch is so 1909</P><H3>Why are we doing this project</H3>
                                Synthetic biology is an exciting area of research that aims to genetically improve organisms to make them more efficient and hopefully more useful to us as well. The human population in 1950 was 2.5 billion, yet it is predicted to surpass 9 billion by 2050 <A href="#" data-container="body" data-toggle="popover" data-placement="top" data-content="Rogers, Oldroyd. Journal of Experimental Botany March 2014 doi:10.1093/jxb/eru098"><SUP>[2]</SUP></A>&quot;. Even with population growth slowing, increasing life spans and standards of living will soon tax our natural resources. One of the most concerning is our food supply. The agriculture industry needs a revolution in order to keep up with our expected growth rates. Currently about 80% of chemically fixated nitrogen is used as agricultural fertilizers, the majority in developed lands<A href="#" data-container="body" data-toggle="popover" data-placement="top" data-content="Rogers, Oldroyd. Journal of Experimental Botany March 2014 doi:10.1093/jxb/eru098"><SUP>[1]</SUP></A>. Intracellular nitrogen fixation in crops could help to sustain the burgeoning world population, especially in areas with less fertile soil without taxing the planet’s waterways. The exponential increase in nitrogen fertilizer has led to more runoff into rivers and oceans. Fertilizers then provide nutrition for algal blooms that result in hypoxia and form oceanic dead zones. These dead zones lead to the death of marine species and have potentially large economic consequences.

                                The ramifications of nitrogen fertilizer runoff can be averted by genetically engineering plant crops to fix their own nitrogen. Some cyanobacteria fix nitrogen for nutritional needs, while most organisms can only acquire it from the food it consumes. Synthetic biology allows us to transfer this ability to fix nitrogen to a heterologous host that has many genetic tools, Escherichia coli, so that we can learn how to give single cell organisms, and eventually chloroplasts the ability to create their own nitrogen fertilizer.

                                Diazotrophic (organisms that fix nitrogen) cyanobacteria such Nostoc Punctiforme or Anabaena use heterocysts (specialized nitrogen fixing cells) to create a mini-anaerobic environment to aid nitrogen fixation. However, Cyanothece 51142, a non-heterocyst, fixes nitrogen in the same cell as photosynthesis by relying on a circadian metabolic process, when there is less oxygen byproduct from photosynthesis. This process is both fascinating and necessary since the key enzyme in nitrogen fixation, nitrogenase, is poisoned by oxygen. Our goal this summer is to engineer the regulation of the proteins necessary for nitrogen fixation so that they are highly repressed when activated by broad spectrum light (such as the sun), and are highly active when there is no light around, mimicking the cycle where photosynthesis occurs during the day and nitrogen fixation occurs at night.

                                <BUTTON type="button" class="btn btn-primary" data-dismiss="modal"> Close Modal</BUTTON></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="overview-modal modal fade" id="overviewModal5" tabindex="-1" role="dialog" aria-hidden="true"><DIV class="modal-content"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><DIV class="modal-body"><H2>Materials and Methods</H2><P class="item-intro text-muted">How we did what we did</P><P dir="ltr">
                                  WashU iGEM Induction Protocol
                                </P><P dir="ltr">
                                  -Pipette 4 mL sterile LB/CM into four 14 mL culture tubes, R0010-RFP (in BL21 and MG1655) and K314100-RFP (in BL21 and MG1655). Also pipette 4 mL sterile
                                  LB to two 14 mL cultures tubes, wild type BL21 and MG1655
                                </P><P dir="ltr">
                                  -Place the cap on the culture tubes to the first position to allow aeration.
                                </P><P dir="ltr">
                                  -Place the tubes into the incubator and allow them to grow overnight (16 hours) at 37 deg C and 250 RPM
                                </P><P dir="ltr">
                                  Next day...
                                </P><P dir="ltr">                         -Make frozen stock for all cultures grown
                                </P><P dir="ltr">
                                  -Measure OD at 600 nm in cuvettes for each culture (OD O/N) (900 uL water and 100 uL culture well mixed)
                                </P><P dir="ltr">
                                  -Dilute cultures to OD of 0.1 in a total of 4 mL fresh LB/antibiotic (just LB for the BL21 and MG1655) in new 14 mL culture tubes
                                </P><P dir="ltr">
                                  -Volume of overnight culture = (0.1*4)/(OD overnight)
                                </P><P dir="ltr">
                                  -Volume of LB/antibiotic (just LB if negative control) = 4mL - V overnight
                                </P><P dir="ltr">
                                  -Incubate 2 hours at 37 deg C and 250 rpm
                                </P><P dir="ltr">
                                  -While waiting for the incubation to complete, label eighteen 15 mL conicals, cm-1 through cm-9 and wt-1 through wt-9 to create 5x dilutions.
                                </P><P dir="ltr">
                                  -Add 10 mL LB and 10 μL cm to conical cm-9.
                                </P><P dir="ltr">
                                  -Add 8 mL LB and 8 μL cm to conicals cm-1 through cm-8.
                                </P><P dir="ltr">
                                  -Add 10 mL LB to conical wt-9.
                                </P><P dir="ltr">
                                  -Add 8 mL LB to conicals wt-1 through wt-8.
                                </P><P dir="ltr">                             -Calculate volume to take from the 0.5 M IPTG stock to achieve the max [IPTG] with the following equation.
                                </P><P dir="ltr">
                                  -Vstock = (10 mL total in tube)*(2.5mM IPTG)*(0.6 mL total in deep well plate)/((500 mM IPTG)*(0.5 mL inducer/LB/ antibiotic in deep well plate)
                                </P><P dir="ltr">
                                  -Add 60 μL of 500 mM IPTG to conicals cm-9 and wt-9.
                                </P><P dir="ltr">
                                  -Vortex both conicals.
                                </P><P dir="ltr">
                                  -Transfer 2 mL of conical cm-9 to conical cm-8, and transfer 2 mL of conical wt-9 to conical wt-8.
                                </P><P dir="ltr">
                                  -Vortex conical cm-8 and wt-8.
                                </P><P dir="ltr">
                                  -Transfer 2 mL of conical cm-8 to conical cm-7, and transfer 2 mL of conical wt-8 to conical wt-7.
                                </P><P dir="ltr">
                                  -Vortex conical cm-7 and wt-7.
                                </P><P dir="ltr">
                                  -Repeat for conical cm-6 through cm-2 and conical wt-6 through wt-2.
                                </P><P dir="ltr">
                                  -Do NOT transfer anything into conical cm-1 or conical wt-1.
                                </P><P dir="ltr">
                                  -Pour each conical (cm-1 through cm-9 and wt-1 through wt-9) into its own trough – one at a time. With the multi-channel pipette, transfer 500 μL per well
                                  into the deep well plates.
                                </P><P dir="ltr">
                                  -After the diluted cultures have grown 2 hours, pour each into its own trough. Using the multi-channel pipette, transfer 100 μL per well into the correct
                                  rows.
                                </P><P dir="ltr">
                                  -Cover the deep well plates with breathable membranes.
                                </P><P dir="ltr">
                                  -Incubate 6-8 hours at 37°C and 250 rpm.
                                </P><P dir="ltr">
                                  To measure...
                                </P><P dir="ltr">
                                  -Centrifuge at 3000 g for 15 minutes. Remember to balance with another plate.
                                </P><P dir="ltr">
                                  -Discard supernatant.
                                </P><P dir="ltr">
                                  -Resuspend all cells in 200 μL 1x PBS.
                                </P><P dir="ltr">
                                  -Transfer 200 μL of wells to a black 96-well plate.
                                </P><P dir="ltr">
                                  -Measure with a Tecan(get proper product name) set to 580 nm excitation and 610 nm emission for mRFP, absorbance at 600 nm and the gain set to 100.
                                </P><P dir="ltr">
                                  -Divide fluorescence by absorbance per well.
                                </P><P dir="ltr">            -Subtract average fluorescence/absorbance of corresponding wild type strain from the fluorescence/absorbance of the strains with the fluorescent protein.
                                </P><P dir="ltr">
                                  -Calculate average fluorescence/absorbance and standard deviation based on the entire population for each strain at each inducer concentration.
                                </P><P dir="ltr">
                                  -Plot average fluorescence/absorbance versus inducer concentration for each strain.
                                </P><P dir="ltr">
                                  Vanderbilt Induction Protocol
                                </P><P dir="ltr">
                                  -Pipette 4 mL sterile LB/CM into four 14 mL culture tubes, R0010-RFP (in BL21 and MG1655) and K314100-RFP (in BL21 and MG1655). Also pipette 4 mL sterile
                                  LB to two 14 mL cultures tubes, wild type BL21 and MG1655
                                </P><P dir="ltr">
                                  -Place the cap on the culture tubes to the first position to allow aeration.
                                </P><P dir="ltr">
                                  -Place the tubes into the incubator and allow them to grow overnight (16 hours) at 37 deg C and 250 RPM
                                </P><P dir="ltr">
                                  Next day...
                                </P><P dir="ltr">
                                  -Make frozen stock for all cultures grown
                                </P><P dir="ltr">
                                  -Measure OD at 600 nm in cuvettes for each culture (OD O/N) (900 uL water and 100 uL culture well mixed)
                                </P><P dir="ltr">
                                  -Label eighteen 15 mL conicals, cm-1 through cm-9 and wt-1 through wt-9 to create 5x dilutions
                                </P><P dir="ltr">
                                  -Add 10 mL LB and 10 uL cm to conical cm-9
                                </P><P dir="ltr">
                                  -Add 8 mL LB and 8 uL cm to conicals cm-1 to cm-8
                                </P><P dir="ltr">
                                  -Add 10 mL LB to conical wt-9.
                                </P><P dir="ltr">
                                  -Add 8 mL LB to conicals wt-1 through wt-8.
                                </P><P dir="ltr">
                                  -Add 50 μL of 500 mM IPTG to conicals cm-9 and wt-9 (conical concentration will be 2.5 mM IPTG)
                                </P><P dir="ltr">
                                  -Vortex both conicals.
                                </P><P dir="ltr">
                                  -Transfer 2 mL of conical cm-9 to conical cm-8, and transfer 2 mL of conical wt-9 to conical wt-8.
                                </P><P dir="ltr">
                                  -Vortex conical cm-8 and wt-8.
                                </P><P dir="ltr">
                                  -Transfer 2 mL of conical cm-8 to conical cm-7, and transfer 2 mL of conical wt-8 to conical wt-7.
                                </P><P dir="ltr">
                                  -Vortex conical cm-7 and wt-7.
                            </P><P dir="ltr">
                                  -Repeat for conical cm-6 through cm-2 and conical wt-6 through wt-2.
                                </P><P dir="ltr">
                                  -Do NOT transfer anything into conical cm-1 or conical wt-1.
                                </P><P dir="ltr">
                                  -Pour each conical (cm-1 through cm-9 and wt-1 through wt-9) into its own trough – one at a time. With a multi-channel pipette, transfer 600 μL per well
                                  into the deep well plates.
                                </P><P dir="ltr">
                                  -Pour each overnight culture into its own trough. Using the low volume multi-channel pipette, transfer 0.6 μL per well into the correct rows.
                                </P><P dir="ltr">
                                  -Cover the deep well plates with breathable membranes.
                                </P><P dir="ltr">
                                  -Incubate 24 hours at 37°C and 250 rpm.
                                </P><P dir="ltr">
                                  To measure...
                                </P><P dir="ltr">
                                  -Centrifuge at 3000 g for 15 minutes. Remember to balance with another plate.
                                </P><P dir="ltr">
                                  -Discard supernatant.
                                </P><P dir="ltr">
                                  -Resuspend all cells in 200 μL 1x PBS.
                                </P><P dir="ltr">
                                  -Transfer 200 μL of wells to a black 96-well plate.
   </P><P dir="ltr">
                                  -Measure with a Tecan(get proper product name) set to 580 nm excitation and 610 nm emission for mRFP, absorbance at 600 nm and the gain set to 100.
                                </P><P dir="ltr">
                                  -Divide fluorescence by absorbance per well.
                                </P><P dir="ltr">
                                  -Subtract average fluorescence/absorbance of corresponding wild type strain from the fluorescence/absorbance of the strains with the fluorescent protein.
                                </P><P dir="ltr">
                                  -Calculate average fluorescence/absorbance and standard deviation based on the entire population for each strain at each inducer concentration.
                                </P><P dir="ltr">
                                  -Plot average fluorescence/absorbance versus inducer concentration for each strain.
                                </P><P dir="ltr">
                                  Making LB Plates
                                </P><P dir="ltr">
                                  -12.5 g Miller LB (25g/Liter of solution)
                                </P><P dir="ltr">
                                  -7.5 g Agar (15g/Liter of solution)
                                </P><P dir="ltr">
                                  -Put magnetic stir bar into bottle
                                </P><P dir="ltr">
                                  -Pour UV deionized water, 500 mL, into bottle (25g/L miller LB, 15 g/L Agar)
                       </P><P dir="ltr">
                                  -Take the bottle to an autoclave
                                </P><P dir="ltr">
                                  -Autoclave bottle on agar cycle, loose cap, for two hours
                                </P><P dir="ltr">
                                  Electrocompetent Cell Prep
                                </P><P dir="ltr">
                                  -2.50 mL of cultures of DH10B were prepared beforehand
                                </P><P dir="ltr">
                                  -0.25 ml of each culture were transferred into 500 mL of LB
                                </P><P dir="ltr">
                                  -These flasks were then grown at 37 deg C, 250 rpm for 1 hour, 30 minutes in incubator
                                </P><P dir="ltr">
                                  i. Start time: 11:05 A.M
                                </P><P dir="ltr">
                                  ii. End time: 12:35 P.M.
                                </P><P dir="ltr">
                                  -Absorbances
                                </P><P dir="ltr">
                                  i. Cuv1 – 0.1055
                                </P><P dir="ltr">
                                  ii. Cuv2 – 0.1084
                                </P><P dir="ltr">
                                  -After absorbances were measured, cultures were added to ice bucket and allowed to cool for 15 minutes.
                                </P><P dir="ltr">
                                  -Centrifuge bottles were centrifuged for 25 min. at 4 deg. C
                                </P><P dir="ltr">
                                  -After centrifuge, bottles were taken to cold room
                                </P><P dir="ltr">
                                  i. Chilled 10% glycerol was added, 55 ml each, into each bottle
                                </P><P dir="ltr">
                                  ii. 70 mL of water were added to each bottle
                                </P><P dir="ltr">
                                  (NOTE: % glycerol in each bottle is roughly 4.4%)
                                </P><P dir="ltr">
                                  -Bottles were placed back into centrifuge for another 25 minutes at 4 deg C
                                </P><P dir="ltr">
                                  -After centrifuge, bottles were taken to cold room once more
                                </P><P dir="ltr">
                                  -Supernatant was drained
                                </P><P dir="ltr">
                                  ii. 10% glycerol was added to 4 tubes
                                </P><P dir="ltr">
                                  iii. Roughly 15 mL were allotted to each tube
                                </P><P dir="ltr">
                                  iv. Tubes were taken and centrifuged
                                </P><P dir="ltr">
                                  -After centrifuge, conical tubes were taken to cold room
                                </P><P dir="ltr">               i. Supernatant was drained
                                </P><P dir="ltr">
                                  ii. 1 mL of 10% glycerol was added to each tube, pellet allowed to resuspend
                                </P><P dir="ltr">
                                  iii. Tubes were taken to centrifuge again, 3000 RPM @ 4deg C
                                </P><P dir="ltr">
                                  Transformation: Electroporation
                                </P><P dir="ltr">
                                  -Obtain plasmid DNA
                                </P><P dir="ltr">
                                  i. Add 2.5 uL of mini-prepped DNA to aliquot containing electrocompetent cells
                                </P><P dir="ltr">
                                  ii. Else, add 1.0 uL if DNA is suspended in buffer solution
                                </P><P dir="ltr">
                                  -Pipet entirety of electrocompetent cell mixture into electro cuvette
                                </P><P dir="ltr">
                                  -Shock cell/DNA mixture using an Eporator
                                </P><P dir="ltr">
                                  -Take out strain and pour into 100 uL of LB to be grown for 1 hour
                                </P><P dir="ltr">
                                  -After 1 hour of growth, plate strain onto LB/Agar/Antibiotic
                                </P><P dir="ltr">
                                  Frozen Stock Preparation
          </P><P dir="ltr">
                                  -Mix 500 uL of cell culture with 500 uL of 30% glycerol
                                </P><P dir="ltr">
                                  -Store in -80 deg C freezer
                                </P><P dir="ltr">
                                  Plasmid Miniprep
                                </P><P dir="ltr">
                                  -Take bacterial culture and centrifuge it
                                </P><P dir="ltr">
                                  -Discard LB growing solution that is supernatant
                                </P><P dir="ltr">
                                  -Resuspend cells in water and add lyze buffer
                                </P><P dir="ltr">
                                  -Immediately after lyze buffer addition, add cold neutralization buffer
                                </P><P dir="ltr">
                                  -Centrifuge tubes at 16000 RPM for 4 minutes
                                </P><P dir="ltr">
                                  -Transfer resulting supernatant to a separation column
                                </P><P dir="ltr">
                                  -Centrifuge columns for 30 seconds at 16000 RPM
                                </P><P dir="ltr">
                                  -Discard flow-through and place spin columns back into collecting tubes
                                </P><P dir="ltr">
                                  -Add 200 uL of Endo-Wash Buffer to each tube
                                </P><P dir="ltr">
                                  -Centrifuge tubes at 16000 RPM for 30 seconds
                                </P><P dir="ltr">
                                  -After centrifugation, add 400 uL of Zyppy’s Wash Buffer to each tube
                                </P><P dir="ltr">
                                  -Centrifuge tubes for 2 minutes at 16000 RPM
                                </P><P dir="ltr">
                                  -After centrifugation is complete, add high pure molecular water to each column and let each column sit for 2 minutes
                                </P><P dir="ltr">
                                  -Centrifuge columns at 16000 RPM for 2 minutes
                                </P><P dir="ltr">
                                  -Take the elution and determine its concentration/purity
                                </P><P dir="ltr">
                                  Polymerase Chain Reaction
                                </P><P dir="ltr">
                                  Components for a 50 uL reaction, per reaction:
                                </P><P dir="ltr">
                                  -4 uL of DMSO
                                </P><P dir="ltr">
                                  -10 uL of buffer
                                </P><P dir="ltr">
                                  -1 uL of dNTPs
                                </P><P dir="ltr">
                                  -2.5 uL of forward primer
     </P><P dir="ltr">
                                  -2.5 uL of reverse primer
                                </P><P dir="ltr">
                                  -1 uL of template DNA
                                </P><P dir="ltr">
                                  -0.5 uL of phusion polymerase
                                </P><P dir="ltr">
                                  -28.5 uL of cloning water
                                </P><P dir="ltr">
                                  **Need time of PCR components
                                </P><P dir="ltr">
                                  Making an Agarose Gel
                                </P><P dir="ltr">
                                  -Measure 1 g of Agarose
                                </P><P dir="ltr">
                                  -Mix agarose with 100 mL of buffer (100 mL of buffer/g of Agarose)
                                </P><P dir="ltr">
                                  -Microwave gel with agarose for 1:30 and then let cool to room temperature
                                </P><P dir="ltr">
                                  -Add 5 uL of cybersafe (5uL/100 mL of buffer) once the mixture cools
                                </P><P dir="ltr">
                                  -Put in combs, dependent on how many lanes you want to construct
                                </P><P dir="ltr">
                                  -Pour agarose gel mixture into tray
                                </P><P dir="ltr">
                                  -Let gel cool for 1 hour
                                </P><P dir="ltr">
                                  -After 1 hour, add 10 uL of 6x loading dye to each sample
                                </P><P dir="ltr">
                                  -Add buffer until the top of the hardened gel plate was covered
                                </P><P dir="ltr">
                                  -Insert ladder
                                </P><P dir="ltr">
                                  i. Make sure you have ladder for each row of running lanes
                                </P><P dir="ltr">
                                  -Then, insert DNA/dye mixture into each well
                                </P><P dir="ltr">
                                  -Let DNA run down the gel at 125V for 30 min
                                </P><P dir="ltr">
                                  i. Make sure there are bubbles forming on the negatively charged end
                                </P><P dir="ltr">
                                  Colony PCR
                                </P><P dir="ltr">
                                  -Obtain necessary amounts of 50 uL of PCR tubes
                                </P><P dir="ltr">
                                  -Add 50 uL of sterile water to each
                                </P><P dir="ltr">
                                  -Prepare a 1 mL/uL LB:AMP mixture with 5 mL of mixture for every reaction
                                </P><P dir="ltr">                 -Pick 3 colonies from plates containing colonies and put 1 into each PCR tube
                                </P><P dir="ltr">
                                  -Then, pipet 5 mL of LB/AMP mixture into corresponding culture tubes and label them with respect to each colony picked
                                </P><P dir="ltr">
                                  -Drop picked colony into its corresponding, labeled culture tube and set aside
                                </P><P dir="ltr">
                                  -Components for a 50 uL reaction, per reaction:
                                </P><P dir="ltr">
                                  i. 22.5 uL of cloning water
                                </P><P dir="ltr">
                                  ii. 4 uL of DMSO
                                </P><P dir="ltr">
                                  iii. 1 uL of dNTPs
                                </P><P dir="ltr">
                                  iv. 0.5 uL of Phusion Polymerase
                                </P><P dir="ltr">
                                  v. 10 uL of buffer
                                </P><P dir="ltr">
                                  vi. 7 uL of template
                                </P><P dir="ltr">
                                  vii. 2.5 uL forward primer
                                </P><P dir="ltr">
                                  viii. 2.5 uL reverse primer
                                </P><P dir="ltr">
                                  -Pipet culture tube contents into corresponding labeled PCR tubes
                                </P><P dir="ltr">
                                  **Need PCR protocols
                                </P><P dir="ltr">
                                  Dpn1 Digest
                                </P><P dir="ltr">
                                  -Obtain Dpn1 enzyme, 10x CutSmart Buffer, and clean DNA
                                </P><P dir="ltr">
                                  Components for a 50 uL reaction, per reaction:
                                </P><P dir="ltr">
                                  i. 40 uL cleaned DNA
                                </P><P dir="ltr">
                                  ii. 5 uL of 10x CutSmart Buffer
                                </P><P dir="ltr">
                                  iii. 5 uL of Dpn1 enzyme
                                </P><P dir="ltr">
                                  -Add buffer before enzyme
                                </P><P dir="ltr">
                                  -Let sit in 37 deg C incubator for 2 hours
                                </P><P dir="ltr">
                                  Ligation Reaction
                                </P><P dir="ltr">
                                  -Obtain T4 Ligase Buffer, DNA to be ligated, P4 Kinase, and T4 Ligase
                                </P><P dir="ltr">
                                  Components for a 50 uL reaction, per reaction:
                                </P><P dir="ltr">
                                  i. 1 uL of T4 Ligase Buffer
                                </P><P dir="ltr">
                                  ii. 8 uL of DNA to be ligated
                                </P><P dir="ltr">
                                  iii. 0.2 uL of P4 Kinase
                                </P><P dir="ltr">
                                  iv. 1 uL of T4 Ligase
                                </P><P dir="ltr">
                                  -Let each reaction sit at room temperature for 1 hour
                                </P><P dir="ltr">
                                  DNA Clean-Up
                                </P><P dir="ltr">
                                  -Load each sample to be cleaned with 250 uL of DNA binding buffer
                                </P><P dir="ltr">
                                  -Vortex
                                </P><P dir="ltr">
                                  -Pour tube contents into spin columns
                                </P><P dir="ltr">
                                  -Centrifuge each tube for 30 seconds at 16000 RPM
                                </P><P dir="ltr">
                                  -Discard flow-through in a clean beaker and set aside
                                </P><P dir="ltr">
                                  -Add 200 uL of DNA Wash Buffer to each spin column
                                </P><P dir="ltr">
                                  -Spin tubes for 30 seconds at 16000 RPM
                                </P><P dir="ltr">
                                  -Add 200 uL more of DNA Wash Buffer
                                </P><P dir="ltr">
                                  -Centrifuge tubes for 2:30 at 16000 RPM
                                </P><P dir="ltr">
                                  -Discard flow-through in beaker
                                </P><P dir="ltr">
                                  -Add 30 uL of high pure molecular water to each tube
                                </P><P dir="ltr">
                                  -Let tubes sit on counter 2 mins.
                                </P><P dir="ltr">
                                  -After waiting period is up, centrifuge tubes for 2 minutes at 16000 RPM
                                </P><P dir="ltr">
                                  -Measure DNA concentrations
                                </P><P dir="ltr">
                                  DNA Gel Extraction
                                </P><P dir="ltr">
                                  -Take each gel, add 600 uL of ADB to each
                                </P><P dir="ltr">
                                  -Put into heat block at 55 deg C and let melt, vortexing every 5 minutes
       </P><P dir="ltr">
                                  -After melting is complete, let each sample sit in heat block for another five minutes
                                </P><P dir="ltr">
                                  -Take each sample out, our contents into a spin column
                                </P><P dir="ltr">
                                  -Spin each down 1:00 min at 16000 rpm
                                </P><P dir="ltr">
                                  -Take each out, discard flow-through in fresh, clean beaker
                                </P><P dir="ltr">
                                  -Add 200 uL of DNA Wash Buffer to each tube
                                </P><P dir="ltr">
                                  -Spin for 30 s at 16000 rpm
                                </P><P dir="ltr">
                                  -Discard flow-through from each collecting tube
                                </P><P dir="ltr">
                                  -Add 200 uL of DNA Wash Buffer to each tube
                                </P><P dir="ltr">
                                  -Spin for 30s at 16000 rpm
                                </P><P dir="ltr">
                                  -Take each out and discard the flow-through
                                </P><P dir="ltr">
                                  -Do a dry run for 2 minutes at 16000 rpm
                                </P><P dir="ltr">
                                  -Take out each sample and discard its flow through
                                </P><P dir="ltr">                                 -Elute each spin column with 40 uL of high pure molecular water
                                </P><P dir="ltr">
                                  -Let each sample sit for 4 minutes
                                </P><P dir="ltr">
                                  -Spin down each for another 4 minutes
                                </P><P dir="ltr">
                                  -Gel recovery is complete
                                </P><BUTTON type="button" class="btn btn-primary" data-dismiss="modal"> Close Modal</BUTTON></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="overview-modal modal fade" id="overviewModal6" tabindex="-1" role="dialog" aria-hidden="true"><DIV class="modal-content"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><DIV class="modal-body"><H2>Discussion</H2><P class="item-intro text-muted">Where do we go from here</P><A href="https://xkcd.com/1501/"><P class="muted">XKCD</P></A><P>The overarching goal behind our project is to introduce nitrogen fixation within plants. Through the introduction of this technology, the need for fertilizers would be greatly diminished, which would provide many positive environmental impacts and cut costs.</P><P>Most of our fertilizer today is produced through the Haber Process. The Haber Process is widely believed to have triggered a population explosion, fueling the growth of human population around the world from 1.6 million to 7 million.<A href="#" data-container="body" data-toggle="popover" data-placement="top" data-content="Smil, Vaclav Detonator of the population explosion"><SUP>[1]</SUP></A> However, there is evidence for detrimental effects surrounding the use of nitrogen compounds. Near areas of heavy industrialization, the development of dead zones have been observed.<A href="#" data-container="body" data-toggle="popover" data-placement="top" data-content="Howarth, Robert Coastal nitrogen pollution: A review of sources and trends globally and regionally"><SUP>[2]</SUP></A> Eutrophication results in the development of harmful algal blooms, triggering hypoxia and hurting the delicate balance of aquatic ecosystems. </P><P>Through our project, we hope to undercut the production of fertilizer and reduce its harmful environmental impact. Through having plants synthesize their own nitrogen compounds we eliminate the need for that artificial nitrogen source. By working in E. coli, it is the team’s hope to develop tools for experimentation with nitrogen fixation in other organisms. E. coli holds the distinct advantage of having rapid growth cycles, enabling quick experimentation with different biological technologies. </P><H4>References</H4><OL><LI><A href="http://www.nature.com/nature/journal/v400/n6743/full/400415a0.html">http://www.nature.com/nature/journal/v400/n6743/full/400415a0.html</A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S1568988308001078">http://www.sciencedirect.com/science/article/pii/S1568988308001078</A></LI></OL><BUTTON type="button" class="btn btn-primary" data-dismiss="modal"> Close Modal</BUTTON></DIV></DIV></DIV></DIV></DIV></DIV><FOOTER class="bg-white row"><DIV class="overview-modal modal fade" id="reportBugsModal" tabindex="-1" role="dialog" aria-hidden="true"><DIV class="modal-content"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><DIV class="modal-body"><SECTION id="report_bugs" class="bg-white row"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 text-center"><H2 class="section-heading">Report Bugs</H2><H3 class="section-subheading text-muted">Thanks for your help with out site!</H3><DIV class="alert alert-danger"><BUTTON type="button" class="close" data-dismiss="alert" aria-hidden="true">×</BUTTON><STRONG>This form will open your default mail client (i.e Outlook). If you do not have one installed, email us at  WashU iGEM washu.igem@gmail.com</STRONG></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><FORM name="sentMessage" id="reportForm" novalidate=""><DIV class="row"><DIV class="col-lg-12 text-center"><BUTTON type="submit" class="btn btn-xl">Send Bug</BUTTON></DIV></DIV></FORM></DIV></DIV></DIV></SECTION><BUTTON type="button" class="btn btn-primary" data-dismiss="modal"> Close Modal</BUTTON></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><SECTION class="row sectionNum5" id="sitemap"><DIV class="col-xs-12"><H2>Discover</H2><TABLE class="table table-responsive"><TBODY><TR><TD>Parts</TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Basic_Part">Basic Parts</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Part_Collection">Parts Collection</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Parts">Parts</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Composite_Part">Composite Part</A></TD></TR><TR><TD>Team</TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Team">Members</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Notebook">Journals</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Attributions">Attributions</A></TD></TR><TR><TD>Project</TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Description">Description</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Design">Design</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Modeling">Modeling</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Measurement">Measurement</A></TD></TR><TR><TD>Policies and Practices</TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Practices">Practices</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Safety">Safety</A></TD></TR><TR><TD>Misc.</TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Software">Software</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Collaborations">Collaborations</A></TD></TR></TBODY></TABLE></DIV></SECTION><SECTION id="sponsors" class="row  sectionNum6"><DIV class="container"><DIV class="row"><H2>Sponsors</H2></DIV><DIV class="row"><P>Special Thanks to ​the Center for Biological Systems Engineering, Department of Chemistry, Focht Family, Heeney/Toomey Family, Bourg Family, NSF-MCB Award #1331194</P></DIV></DIV></SECTION><DIV class="row"><DIV class="col-md-4 col-md-offset-4"><SPAN class="copyright"> Washington University in St. Louis and Pennsylvania State University iGEM 2015</SPAN></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>