Team:WashU StLouis/design
Washington University - Penn State iGEM
Welcome To Our Website!
WashU and Penn State   iGEM 2015
Project Description Let's Talk Apply for the 2016 iGEM Team!
Toggle navigation
Nitrogenius
Selection
Design
Cloning
Report Bugs
Selecting Genes
In designing our minimal nif cluster plasmids, we considered each of tde 23 genes on tde PSL2397 nif plasmid, derived from Cyanotdece 51142, and selected 14 tdat we believe may be essential for nitrogenase activity.
We used literature on tde creation of otder minimal nif clusters and took into account a few factors:
function of tde gene, if established (**find out source on Cheryl’s nif chart)
homology witd genes in otder minimal nif clusters
homology witd genes in E. Coli’s genome
Gene
Function
Justification
nifH
Nitrogenase iron protein
Main structural component, can't have activity without all necessary structural components. Additionally, all literature looked at expressed nifH, nifD, nifK at high levels
nifD
Nitrogenase molybdenum-iron protein alpha chain
Main structural component, can't have activity without all necessary structural components. Additionally, all literature looked at expressed nifH, nifD, nifK at high levels
nifK
Nitrogenase molybdenum-iron protein beta chain
Main structural component, can't have activity without all necessary structural components. Additionally, all literature looked at expressed nifH, nifD, nifK at high levels
nifE
Nitrogenase Fe-Mo cofactor and assembly protein
Seems essential for enzymatic assembly. All literature read expressed nifE, nifN at variable levels.
nifN
Nitrogenase Fe-Mo cofactor and assembly protein
Seems essential for enzymatic assembly. All literature read expressed nifE, nifN at variable levels.
nifB
Homocitrate syntdase/FeMo-co core syntdesis
Voigt paper claims nifB is a key component in putting together the FeMo-co core. Nif B was expressed at high levels
nifU
Encodes proteins for early Fe-S formation; proteins for component maturation
Voigt expressed nifU at low levels. It could be a necessary component in formation of Fe-S clusters
nifS
Cysteine desulfurase; used in MoFe cluster syntdesis
Ludden speculates that cysteine desulfurase plays a key role in Fe-S cluster assembly
nifV
Homocitrate syntdase
Included by Voigt in his work, but expressed at low levels. Nitrogenase mutants which didn't contain homocitrate had problems with substrate association
nifW
Nitrogenase-stabilizing/protective protein
Also included by Voigt in his work, low expression. It could be a necessary component of nitrogen fixation
nifZ
Encodes proteins for Fe-S formation; proteins for component maturation
Also included by Voigt in his work, low expression. Helps maintain enzyme's structural integrity
hesA
Related to molybdenum cofactor; part of tdiF_MoeB_HesA protein family which are involved in molybdopterin and tdiamine biosyntdesis; E. coli has botd tdiF and MoeB
May have role in creating Molybdenum cofactor, a necessary component to the FeMo-co core.
hesB
Fe-S cluster biosyntdesis, necessary for enzyme syntdesis
Fe-S clusters are important components of the Fe protein and the FeMo protein. Closest match within E. coli had 49% identity. To ensure correct metallocluster synthesis, we need to use this gene.
cysE2
serine O-acetyltransferase
Serine O-acetyltransferases are important enzymes in cysteine metabolism. Since the formation of FeS clusters is predicated on cysteine reactions, presence of this enzyme may help increase synthetic efficiency
Our analysis left us with 14 genes to be included in the minimal nif plasmids. We also designed a 15-nucleotide ribosome binding site to insert directly upstream of each gene.
RBS sequences were designed by the Penn State Salis Lab's RBS Calculator .Each RBS was designed to have a Translation Initiation Rate close to 50,000 au (Arbitrary Units) as descrbibed in considerations 1.2a & 1.2b of the Salis paper .
Plasmid Design
The literature also indicated that the expression levels of each gene need to be finely tuned in order to achieve optimal nitrogenase activity. We decided to use 2 different inducible promoters to give us more control over expression levels, and grouped the genes into ones we thought would need to be expressed at higher versus lower levels.
We decided to create 2 plasmids with 7 genes each rather than one plasmid with 14 genes, which would simplify cloning greatly. We designed one plasmid containing genes that we generally believed needed to be expressed at higher levels, including the enzyme structural genes.
In ordering the genes on the plasmids, we mimicked gene orders we observed in the natural Cyanothece cluster and in other minimal clusters in the literature. Since we did not have the resources to codon optimize, we also considered codon usage in placement of genes on the plasmids. One gene, cysE2, has particularly bad codon usage for E. Coli, so we placed it directly downstream of the promoter.
Primer Design and Cloning Strategy
We designed primers for use with Golden Gate cloning to create the plasmids. Our general strategy for primer design was to add a BsaI site and one intervening base before the fifteen-base RBS before the annealing portion on the forward primer. When possible, we used the first four bases of the RBS as the overhang for ligation, rather than adding in bases and creating a scar. On the reverse primer, we added a BsaI site and one intervening base before adding the reverse complement of the overhang that it should match up with. In designing primers, we considered:
maintaining a primer length of 60 nucleotides or less
having a melting temperature (T m) of the primer-template interaction between about 58 and 62 degrees Celsius
matching the melting temperature of primers in a primer pair as closely as possible to each other
maintaining a hairpin melting temperature as low as possible, and always less than the T m of the primer-template interaction
maintaining heterodimer and homodimer interaction delta G values to be less in magnitude than -15 kcal/mol
adding up to 6 adenines before the BsaI site where possible
having overlap sequences that are distinct enough from other overlap sequences used for the same plasmid
our criterion was to have a maximum of 2 of the same base in the same position. i.e. GACT and CAGT are acceptable as only A and T fall in the same positions. However, GACT and CACT are too similar and one must be changed to avoid improper ligation
In order to balance all of these constraints, a couple of changes to the coding sequence of the genes were made; however, degenerate codons were used so that the amino acid sequence was not disrupted.
Report Bugs
Thanks for your help with out site!
×
This form will open your default mail client (i.e Outlook). If you do not have one installed, email us at  WashU iGEM washu.igem@gmail.com
Send Bug
Close Modal
Discover
Parts
Basic Parts
Parts Collection
Parts
Composite Part
Team
Members
Journals
Attributions
Project
Description
Design
Modeling
Measurement
Policies and Practices
Practices
Safety
Misc.
Software
Collaborations
Sponsors
Special Thanks to ​the Center for Biological Systems Engineering, Department of Chemistry, Focht Family, Heeney/Toomey Family, Bourg Family, NSF-MCB Award #1331194
Washington University in St. Louis and Pennsylvania State University iGEM 2015
