Team:WashU StLouis/Notebook
Washington University - Penn State iGEM
Welcome To Our Website!
WashU and Penn State   iGEM 2015
Project Description Let's Talk
charlotte
blake
jessica
mike
david
laura
Charlotte Bourg         Washington University in St. Louis
- Experiments From Jun 02
Make CμLtures
C_exp2
The Moon Lab needed, strains JM109 and MG1655 for their experiments
Make overnight cμLtures from frozen stock to make electrocompetent cells the next day (in Moon lab):
from glycerol stock, set up two 50mL cμLtures in 250-mL flasks
grow overnight
- Experiments From Jun 03
Make Lb
C_exp4
The Moon Lab needed, strains JM109 and MG1655 for their experiments
Transformation (in Moon lab):
- Take MG1655 and WM1788 electrocompetent cells out of -80°C and place on ice
- Take PSL2397 (plasmid) out of -80°C and place on ice
- Add 2 μL of PSL2397 directly into MG1655 tube
- Set pipette over 40 μL and draw up the MG1655 and PSL2397 mixture and place into electroporation cuvette
- Tap cuvette to ensure cells are at bottom of cuvette—shoμLd check to see that there are no gaps—and place into electroporator; turn on
- Immediately add 500 μL of LB into the cuvette then pour into cμLture tube
- Place cμLture tubes of WM1788 and MG1655 into incubator for 1 hour
- Experiments From Jun 04
Make Lb
C_exp6
There are 10 genes in the Nif cluster that have unknown functions. Overexpressing these can shead light on what they do
Use SnapGene to design primers for the 14 sequences to be overexpressed
- Check for EcoRI and XhoI restriction sites within each of the sequences
- Check for directionality on the plasmid: direct or complementary
- If a restriction enzyme does not have sites within the sequence, add a site for that restriction enzyme to the appropriate primer and add 6 adenines beyond the site on the primer so that the restriction enzyme will work properly
- If the restriction enzyme does have a site within the sequence, end the primer at the end of the sequence to be amplified to leave the ends blunt
- If direct, add EcoRI to the 5’ end and XhoI to the 3’ end as appropriate
- If complementary, add XhoI to the 5’ end and EcoRI to the 3’ end as appropriate
- Maintain a Tm for each primer above 60°C and approximately match the Tm of paired primers
- Ensure that there is only 1 binding site on the plasmid for each primer
- Experiments From Jun 09
Run PCR
C_exp8
Run PCR on 3 of the 14 amplicons
- Experiments From Jun 10
Run PCR
C_exp9
Run PCR on remaining amplicons
- Experiments From Jun 11
Overnight cμLtures
C_exp14
Start an overnight cμLture in 5 mL of LB at 37°C for transformation of ligated plasmids
- Experiments From Jun 12
Plate Cells
C_exp18
- Take cells out of 37°C room and spin down all but the control for efficiency of transformation (PE5A) for 4 minutes
- Take 100 μL from PE5A control and add to 900 μL of LB
-Plate 100 μL of that dilution on the amp plate
-Add sterile glass beads and shake laterally to spread around the cμLture
-Dump beads into nonsterile glass beads container
- from centrifuged cμLtures, pipette off the media to the 100 μL mark and resuspend the pellet in that amount of media
-Add fμLl resuspended quantity to the correct labeled plate
-Add beads and shake
- Place all plates in 37°C room
- Experiments From Jun 13
Store plates
C_exp19
Take plates out of 37°C room, check for colonies, and place those that grew into the 4°C room for the remainder of the weekend
- Experiments From Jun 15
Run Gel
C_exp25
- Run digested plasmid through the gel
- Saw no band for digested plasmid and a band in unexpected location for the undigested plasmid need to redo the digestion of the plasmid
- Experiments From Jun 17
Confirm Inserts
C_exp26
Make 20 μL PCR reactions for plasmids purified yesterday (to attempt to confirm correct inserts)
- Experiments From Jun 18
Digest
C_exp33
Digest the blunted cut plasmid with Xho1 in a 25 uL reaction
Add the following to the Klenow reaction mixture (without purifying first)
2 uL digested buffers
2 uL Xho1
1 uL water
Place at 37C for ~3 hours
- Experiments From Jun 19
Check plates
C_exp34
Carlos checked plated cells for growth
1 colony on control; ~30-50 colonies on other plates
Placed plates in 4C room
- Experiments From Jun 20
Pick colonies
C_exp35
Carlos picked 2 colonies per plate and started overnight cultures in LB
- Experiments From Jun 22
Transform
C_exp38
Transform plasmids from overnight
Purify plasmid from overnight cultures of colonies
Quantify cocentrations of plasmids
Set up 20 uL PCR reaction for the purified plasmids to confirm inserts
transform ligated plasmids
Add the ligation reaction products produced to labled vials of competent cells
Incubate cells with ligation products added on ice for 25 minutes
Heat shock at 42C for 1 minue; replace on ice for ~2 minutes
Add 900 uL of LB to each tube (in the hood)
Place on shaker at 37C for 1 hour
Pour plates
Melt LB agar in a secondary container of water in the microwave
Wait for agar to cool before adding 1 uL of chloramphenicol per mL LB agar
Pour plates with ~20mL agar with antibiotic
Allow plates to dry in hood
Plate cells
Centrifuge, pipette off the supernatant to 100 uL mark, then resuspend the pellet in that amount
Add resuspended cells to labeled plates; spread with with sterile glass beads. Place plates in 37C room
Run a gel of PCR products (to confirm inserts in transformed cells)
- Experiments From Jun 23
Check colonies
C_exp39
Check for colonies from plated cells
Control grew colonies, as did the others
Pick a colony with a pipette tip and resuspend colony in the media
Add 50mL of LB to a conical tube; add 50 uL of chloramphenicol to that
Add 5 mL of that mix to each of the 10 labeled sterile culture tubes.
Pick a colony with a pipette tip and then either resuspend the cells in tube or drop the tip into the tube
Plate on shaker in 37C room at 250 rpm
Add 5 mL LB to another steril culture tube. Add 5 uL LB to the tube. Scrape frozen stock of PA2C-TesA with a pipette tip and place the tip in the Lb; place on shaker in 37C room
- Experiments From Jun 24
Miniprep PA2C-TesA
C_exp44
Miniprep PA2C-TesA (using 10X the amount of buffers as suggested by the kit)
Pour the 50 mL culture into a 50 mL conical and centrifuge to a pellet
Pour off the supernatant
Resuspend in 2500 uL of resuspension buffer and vortex or pipette in and out until no clumps remain
Add 2500 uL of lysis buffer and invert 10 times to mix (no vortexing). Incubate on ice for 5 minutes
Centrifuge at 13,000 rpm and 4C for 10 minutes
Pour the supernatant through a syringe with a cotton ball in it to filter
Transfer 800 uL of supernatant to a column in a collection tube and centrifuge at 13,000 rpm for 1 minute.
Discard the filtrete and replace in the same collection tube
Repeat previous steps with additional 800 uL of filtered supernatant until all supernatant has gone trhough column
Add 500 uL of wash buffer A and centrifuge at 13,000 rpm for 1 minute. Discard the flow through andreplace column in the tube
Centrifuge at 13,000 rpm for 1 minute
Place on thermomixer at 65C for 5 minutes
Place column in a new  1.7 mL microcentrifuge tube and add 50 uL of water warmed to 65C; let sit at room temperature for 1 minute
Centrifuge at 13,000 rpm and 1 minute
Quantify concentration on nanodrop
On 2% agarose gel, add: 2.5 uL 1kb pluss ladder to wells on either side. 8uL of each PCR product
- Experiments From Jun 25
Run gel
C_exp44
Run a gel to see if PA2C-TesA purified yesterday is okay
Pour a gel with 10 wells of ~35 mL 1% agarose with 2 uL EtBr
Run 100 ng of plasmid max -> 2.5 uL with .5 uL loading dye
2.5 uL ladder on each side
- Experiments From Jun 26
Repeat transformation
C_exp45
Purify digested PA2C-TesA with DNA clean & concentrator kit
Start a Klenow reaction
Digest the plasmid from the Klenow with Xho1
load the digested plasmid into a gel and gel purify the correctly sized band
Ligate the digested insert with digested plasmid
Transform the ligation products into chemically competant cells
- Experiments From Jun 30
Start overnight of more PA2C-TesA
C_exp46
- Experiments From Jul 01
Attempt to PCR the inserts from the HDK overexpression plasmid using the nifHDK F and R primers
C_exp48
- Experiments From Jul 02
Redo HDK PCR
C_exp50
Redo HDK PCR from nif cluster plasmid to attempt construction of the HDK overexpression plasmid again
- Experiments From Jul 03
Primer design for minimal nif plasmid
C_exp51
- Experiments From Jul 06
Redo PCR of HDK and cce_0550
C_exp53
Cut and purified band for 0550, but HDK band was incorrect
- Experiments From Jul 07
Ligate 0550 and HDK
C_exp54
Ligate genes into plasmid
Repurify 0550 PCR product (due to a bad peak on nanodrop)
Digest repurified 0550 with Xho1
Clean up digest with DNA clean & concentrator kit
Set up ligation for 0550 and HDK with PA2C-TesA
- Experiments From Jul 08
PCR genes for minimal nif plasmid-grouped by size into 3 sets
C_exp55
PCR for small and large size groups
Gradient PCR from 58C to 65C
100 uL rxn split into 4 tubes of 25 mL each, at different temperatures in gradient thermocycler
Run gels of a small amount of each of tube. Saw bands for very few of the expected products
- Experiments From Jul 09
PCr medium size group
C_exp56
- Experiments From Jul 10
PCR for minimal nif plasmids continued
C_exp57
- Experiments From Jul 13
PCR more 0550 and HDK for overexpression plasmids
C_exp58
EcoR1 digest of PA2C-TesA
Run 0550 PCR product in a gel and gel purify it
Digest 0550 with Xho1
Run HDK PCR product in a gel and gel purify it
- Experiments From Jul 14
Prepare 1 plasmid of nif cluster
C_exp59
PCR amplify tetracyclin resistance gene off of one plasmid and pB1a except for the amplicilin resistance gene, using primers designed for Gibson ligation
Blunt the ends of PA2C-TesA cut with EcoR1 using S1 nuclease
Add at least 10 units of S1 nuclease per microgram of plasmid, but use 10 units if you use less than 1 microgram
Use a 10x final concentration of buffer
Add water to a 30 uL final reaction volume
Place at room temperature for 45 minutes, then add 4 uL of .25M EDTA and incubate at 75C for 20 minutes to stop the reaction
After blunting with S1 nuclease, clean up with DNA  clean & concentrator kit, then start an Xho1 digest
Set up Gibson assembly for backbone resistance change for one of the minimal plasmids
Use 7.5 uL of Gibson mix premade by another member of the lab
Dilute to a 10 uL final reaction volume
Place at 50C for 1 hour to ligate
Add 50 ng of each piece to be ligaged
Based on the concentrations of DNA we had, we were unable to add 50ng of each piece, and maximized the amount of DNA added by not diluting with water
transform Gibson assembly products using chemically competent transformation protocol
Cut PA2C-TesA backbone band from gel and gel purify
Make new tetracyclin stocks for Gibson assembly products transformation
Measure 50 mg of tetracycline hydrochloride on weighing paper
Add 70% EtOH to a 10 mL final volume; mix
Add 4 uL per mL to 20 mL Lb (80 uL of tetracycline)
Checked on 7/16/2015, no colonies from Gibson products
- Experiments From Jul 16
Transform ps8k plasmid into DH10B to make more
C_exp61
- Experiments From Jul 17
Golden Gate reactions
C_exp63
Set up a 10 uL Golden Gate reaction of all 8 pieces for the pS8K cysE2-hesB minimal plasmid, using a molar ratio that compares concentration/size(bp) of each piece. Inserts should all be in a 3 to 1 ratio to the backbone
mix reaction mixtures and DNA together
DNA should be 7 uL
Add: 1 uL T4 ligase buffer, .5 uL Bsa1, .5 uL T4 ligase, .3 uL Dpn1, .5 uL ATP at 10 uM, .3 uL BSA
Spin down and put in thermocycler
37C for 3 minutes
16C for 4 minutes
Repeat first 2 steps 50 times
50C for 5 minutes
80C for 5 minutes
Hold at 4C
- Experiments From Jul 20
Pick colonies
C_exp64
Pick 5 colonies from DH10B plate of golden gate reaction mixture transformation. Pick 1 colony of ps8k
- Experiments From Jul 21
Work on HDK & 0550
C_exp65
Digest PA2C-TesA with EcoR1, PCR more 0550 and HDK
- Experiments From Jul 22
Golden gate cysE2-hesB
C_exp67
Run a gel to check for cysE2-hesB PCR product
Run a gel to cut the backbone from the PA2C-TesA Xho1 digest
Digest ps8k minipreps with pst1 and Pme1 to check for proper sizes
Set up a golden gate of the seven inserts only, excluding the backbone from this reaction, using a 1 to 1 molar ratio of all Golden Gate inserts
- Experiments From Jul 23
Repeat Golden Gate
C_exp68
Set up a PCR using the 7 insert Golden Gate reaction as template: make 2 15 microliter reactions, one using 1 microliter of the golden gate as template and the other using 1 microliter of a 10x dilution of the template
Run a gel of a small amount of the ligation mix(template for the PCR)
Start digest of HDK and 0550 PCR products with Xho1
Clean and concentrate the digest
PCR of inserts with 10x dilution of template looks good -- start a 100 uL reaction scaled up
Run a small amount of cleaned up digest products on gel to check for proper bands -- got expected bands
PCR tetracyclin and pB1a for the backbone resistance change with the newly-arrived Golden Gate primers(rather than the Gibson primers)
- Experiments From Jul 24
More Golden Gate
C_exp70
Re PCR the ps8k backbone to ligate cysE2-hesB into
transform golden gate ligation of pB1a and tetracycline into DH10B chemically competent cells
Cut and purify the 7 gene ligation PCR from the gel
Set up a Golden Gate ligation of the 7 insert PCR product and the amplified pS8k backbone in a 10 uL reaction
- Experiments From Jul 25
Transform the Golden Gate product into chemically competent DH10B
C_exp71
- Experiments From Jul 27
Check for inserts
C_exp72
Miniprep the overnight of ps8k, ps8k cysE2-hesB 1 through 5 ( 5 colonies picked from the plate), and pB1a-tet
Start a Golden Gate reaction of the 7 inserts for the second plasmid
Digest pB1t with Spe1 and BamH1 (alone and together) to check for correct insert
Digest pS8k cysE2-hesB with Pme1 and Pst1 (alone and together) to check for insets
- Experiments From Jul 28
Send in sequencing primers
C_exp74
Dilute sequencing primers for ps8k cysE2-hesB
Sequence miniprep #2 of ps8k cysE2-hesB (clearest bands on the gel from digest check)
Sequencing PcR reactions
50 uL reactions using ps8k cysE2-hesB #2 as template
Amplify ~1 kb per primer pair, with each read overlapping by about 100 bp (the way the sequencing primers were designed)
- Experiments From Jul 29
Check Plasmid
C_exp75
Run a gel of a small amount of nifh to nifn golden gate mixture and all of the pB1t PCR
Saw no visible band for the Golden Gate mixture where expected for full ligation
Cut bands of pB1t PCR
Miniprep the overnight clulture of PA2C-TesA
Digest PA2C-TesA with EcoR1
Clean and Concentrate the digest, then start a klenow reaction
Resuspend newly-arrived sequencing primers for the next minimal nif plasmid, using TE buffer
- Experiments From Jul 30
Repeat Golden Gate
C_exp77
Try inserts in groups of 3 and 4 rather than all 7
nifH, nifD, nifK and nifB, hesA, nifE, nifN
- Experiments From Jul 31
Do Goldan Gate 2 at a time
C_exp78
Do Golden gate 2 inserts at a time, in pairs as follows (leaving out nifK from the groupings for now): nifH & nifK, nifB & hesA, nifE & nifN
- Experiments From Aug 01
Run PCRs to confirm
C_exp79
EN did not work. Try PCR with varied MgCl concentration and annealing temperatures.
Cut HD and BA from gel and gel purify
Start a Golden Gate reaction of HD with K
- Experiments From Aug 04
Attempt to digest then ligate separately
C_exp81
Digest
1.5 hour digestion
20 uL reaction
Bsa1, digest buffer, and 1 to 2 uL of insert to be digested
Clean and Concentrate
Ligate cleaned up pieces
T4 ligase
T4 ligase buffer
.5 uL ATP
10 uL reaction
- Experiments From Aug 05
More EN
C_exp83
Run PCRs of nifE and nifN ligation from separate digestion and ligation
Use E F and N R primers
Use 2 additional pairs of primers from the sequencing primers designed for this plasmid to amplify smaller portions of E and N that cross the junction, to check for appropriate ligation
Start overnight ligation of digested E and digested N at 4C
- Experiments From Aug 06
Check on ligation reactions
C_exp84
Ran gel of HDK BA, HDK BA E N, and overnight 4C E N ligations
Start PCRs from each, in addition to the room temperature EN ligation
Both amplifications of EN look good - cut and purify together
Neither HDKBA nor HDKBAEN worked
PCR more HDK from golden gate template
Start digest of BA and EN with Bsa1 for 2 step digestion then ligation
Jun_02
C_exp1
C_exp2
Jun_03
C_exp3
C_exp4
Jun_04
C_exp5
C_exp6
Jun_09
C_exp7
C_exp8
Jun_10
C_exp9
Jun_11
C_exp10
C_exp11
C_exp12
C_exp13
C_exp14
Jun_12
C_exp15
C_exp16
C_exp17
C_exp18
Jun_13
C_exp19
Jun_15
C_exp20
C_exp21
C_exp22
C_exp23
C_exp24
C_exp25
Jun_17
C_exp26
Jun_18
C_exp27
C_exp28
C_exp29
C_exp30
C_exp31
C_exp32
C_exp33
Jun_19
C_exp34
Jun_20
C_exp35
Jun_22
C_exp36
C_exp37
C_exp38
Jun_23
C_exp39
Jun_24
C_exp40
C_exp41
C_exp42
C_exp43
C_exp44
Jun_25
C_exp44
Jun_26
C_exp45
Jun_30
C_exp46
Jul_01
C_exp47
C_exp48
Jul_02
C_exp49
C_exp50
Jul_03
C_exp51
Jul_06
C_exp52
C_exp53
Jul_07
C_exp54
Jul_08
C_exp55
Jul_09
C_exp56
Jul_10
C_exp57
Jul_13
C_exp58
Jul_14
C_exp59
Jul_16
C_exp60
C_exp61
Jul_17
C_exp62
C_exp63
Jul_20
C_exp64
Jul_21
C_exp65
Jul_22
C_exp66
C_exp67
Jul_23
C_exp68
Jul_24
C_exp69
C_exp70
Jul_25
C_exp71
Jul_27
C_exp72
Jul_28
C_exp73
C_exp74
Jul_29
C_exp75
Jul_30
C_exp76
C_exp77
Jul_31
C_exp78
Aug_01
C_exp79
Aug_04
C_exp80
C_exp81
Aug_05
C_exp82
C_exp83
Aug_06
C_exp84
Blake Actkinson         Washington University in St. Louis
- Experiments From Jun 01
Preparation of Electrocompetent Cells
B_exp1
These are the strains we put the nif cluster in this summer.
Prepare cultures of DH10B
25 ml of each culture were transferred into 500 mL of LB
-These flasks were then grown at 37 deg C, 250 rpm for 1 hour, 30 min in incubator
Measure absorbance
Cuv1-.1055, Cuv2-.1084
After absorbances were measured, cultures were added to ice bucket and allowed to cool for 15 minutes.
-Centrifuge bottles were centrifuged for 25 min. at 4 deg.
After centrifuge, bottles were taken to cold room
Chilled 10% glycerol was added, 55 ml each, into each bottle
It’s worth noting that % glycerol in each bottle is roughly 4.4%
70 Ml of water were added to each bottle
-Bottles were placed back into centrifuge for another 256 minutes at 4 deg C-After centrifuge, bottles were taken to cold room once more
Supernatant was drained
10% glycerol was added to 4 tubes
Roughly 15 mL were allotted to each tube
Tubes were taken and centrifuged
After centrifuge, conical tubes were taken to cold room
Supernatant was drained
1 mL of 10% glycerol was added to each tube, pellet allowed to resuspend
Tubes were taken to centrifuge again, 3000 RPM @ 4deg C
- Experiments From Jun 02
Designed program to find sgRNAs in Java,
B_exp2
Need to find a place for dCAS9 to bind
Designed program to find sgRNAs in Java, key feature is finding the PAM sequence
- Experiments From Jun 03
More electrocompetent cell prep
B_exp3
EC cell prep for Strains: MG1655 and JM109
Two culture were incubated overnight
-In the morning, 26 mL of culture were placed into 500 mL of LB
-These flasks were incubated for an hour beginning at 10:34
-Both flasks were taken out around 11:35 P.M.
-Optical measurements were performed on both colonies using a 10x dilution factor
i. The optical density of JM109 was 0.287 ii. The optical density of MG1655 was 0.39
-JM109 went back to be incubated at 11:56 A.M.
-Optical measurement performed on JM109 once again
Optical density was 0.38
-Optical density for both strains was determined to be satisfactory -Thus, each flask was equally partitioned into two centrifuge bottles
-Centrifuge bottles were left to sit for 20 minutes
-After sitting for 30 minutes, bottles were centrifuged at 3000 RPM for 25 minutes
-Upon centrifugation, supernatant was drained, pellet was resuspended in 125 mL of water
-Bottles were centrifuged again for another 25 minutes
-Upon centrifugation, supernatant was drained and 55 mL of 10% glycerol were added to each bottles
-Pellet was resuspended in each bottles
-After resuspension, each bottles was partitioned into 4 15 mL conical tubes
-Conical tubes were centrifuged under same conditions
-From a combined conical tube, 40uL aliquots were created for JM109 and MG1655
-These aliquots were stored in -80 deg C conditions Electroporation
-We transformed MG1655 and WM1788
-To do this, we obtained copies of pSL2397 plasmids
-2.5 uL of this plamsmid DNA was collected and mixed with pipettor
-After this, the plasmid DNA was inserted into a cuvette
-The entirety of the aliquot containing the E.Coli strains was dumped into the cuvette
-The plasmid was shocked into the strain for 5.5s
-Strain was taken out and put into conical tube for an hour
-The same process was repeated with MG1655 strain
- Experiments From Jun 04
Picked Colonies
B_exp5
Picked colonies from yesterday's prep
- Experiments From Jun 05
Preparation of Frozen Stock
B_exp6
Store Cells from yesterday
-Cells were taken from picked colonies on June 4
-We took 500 uL of that culture, added it to 500 uL of 30% glycerol solution, then place din -80k deg C conditions.
-We then prepared 30% glycerol solution for later use
-Next, we prepared the CRIPSR/dCas9 plasmid for PCR
-To do this, we separated the plasmid from its host bacterium
-First, we took the bacterial culture and centrifuged it
-Then, we discarded LB growing solution that was supernatant
-Next, we resuspended bacterial cells in water
-Buffer to lyze the cell was added
-Immediately after, cold neutralization buffer was added to neutralize the process
-Then, tubes were centrifuged at 16000 g for 43 minutes
-Supernatant was transferred to a separation column
-These columns were then centrifuged for 30 seconds
-The flow through was discarded and the colonies were placed back into the collecting tubes
-200 uL of Endo-Wash Buffer were added to each tubes
-Each tubes was centrifuged under same conditions for 30 seconds
-Upon centrifugation, 400 uL of Zappy’s buffer were added to each tube
-Each tube was centrifuged for another 2 minutes
-Upon centrifugation, high pure molecular water was added to each column and each column sat for two minutes
-Each column was centrifuged, same condition, for another 2 minutes
-After this step, dissolved plasmid DNA had collected on bottom of column
-Each column was taken to spectrometer to determine concentration and purity
-Concentrations and purities, in order
Tube
Conc (ng/uL)
Purity
Tube 1
1.87 127.1 Tube 3 1.79 82.7
After that, each DNA sample was placed into freezer to be stored
- Experiments From Jun 08
Prepare Gel
B_exp9
We prepared a gel for DNA purification
-Mix 1g of Agarose with 100 mL of TAE buffer.
Note that you can cool the mixture with cold, running water
-Add 5 uL of cybersafe (5uL/100 mL of buffer) once mixture cools
-Put in combs
-Tape side of tray
-Pour agarose gel in tray
-Let gel cool for 1 hour
Run gels for 1 hour
- Experiments From Jun 08
Ran gels
B_exp13
­6x loading dye, 10 uL, was added to each sample
­Take yellow tape off of sides
­Buffer was added until the top of the hardened gel plate was covered
­Ladder was inserted
i. Make sure you have ladder for each row of running lanes
­Then,inset DNA into each well
Make sure not to creat bubbles. Do not tap plate
­125V, 30 min
i. Make sure there are bubbles forming on the negatively charged end
- Experiments From Jun 16
Another miniprep of plasmid
B_exp14
­Grow stock overnight
­Add 600 uL of culture to 1.5 mL mc tube
­Centrifuge for 30 seconds to pellet cells
Do at max speed
­Discard supernatant
­Add 600 uL of deionized water to cell pellet and resuspend by vortexing the tubes
­Add 100 uL of 7x lysis buffer and mix by inverting 4­6 times proceed to next step within 2 minutes
­Solution should change from opaque to clear blue
­Add 350 uL of cold neutralization buffer and shake hard to mix
­Sample will turn yellow when neutralization is complete and a yellowish precipitate will form
­Invert the sample an additional 2­3 times to ensure complete neutralization
­Centrifuge at 16,000 rpm for 2 min 30 seconds
­Transfer the supernatant into Zymo Spin column (small, without cups) by simply dumping it from microcentrifuge tube into column and avoiding the pellet
­Place into a collection tube and centrifuge for 30 seconds
­Discard the liquid, protect the filter and place the column back into the same collection tube
­Add 20 uL of endo­wash buffer to the column and centrifuge for 30 seconds
­Add 400 uL of Zyppy wash Buffer to the column and centrifuge for 30 seconds
­Discard liquid and centrifuge for 2 minutes
­Transfer column into clean 1.5 mL microcentrifuge tube and add 30 uL of high­pure molecular  water directly to column and let stand at room temp. for 2 min
­Centrifuge for 30 seconds to elute plasmid
­Boot up Nano2000 program and prepare a blank by place 1 uL of high pure molecule water on stand
­Test each sample
- Experiments From Jun 17
Post ligation, transformation
B_exp20
­Need: 500 uL of LB
­NO MORE THAN 2 uL OF LIGATION PRODUCT (had to start over on 4)
­DH10B competent cells, 2 aliquots
­Sanity A flame
Obtain <2 uL of ligation product
Insert ligation product into DH10B cell tube
Put mixture into chilled cuvette
­Ensure mixture @ bottom and no air bubbles
WIPE OFF CUVETTE WITH KIMWIPE
Insert into Eporator set @ 2500
While cuvette is being transformed, hover 400 uL pipette tip over flame then proceed to fill 500
Once transformation is complete immediately insert LB into cuvette
Dump contents of cuvette into falcon tube and let incubate for 1 hour
4 was a failute because too much buffer was present. Salts caused archs
- Experiments From Jun 18
Colony PCR
B_exp22
­First, 12 PCR tubes obtained
­Added 50 uL of HP water to each
­Prepared a 1ml/uL LB:AMP mixture and used 60 mL of mixture total
­Obtained 1F, 2F, 3F, 4F plates
­Picked 3 colonies from each plate and put 1 into each PCR tube
­Then, I pipetted 5 mL of LB/AMP mixture into a culture tube
­After this was complete, dropped pipet tip corresponding to each culture tube and set  them aside
­Next, PCR reaction was prepped
­Reaction was done with 7 uL of template (not entirely DNA)
­292.5 uL of water
­52 uL of DMSO
­130 uL of Buffer
­13 uL of dNTPs
­42.5 uL of forward primer
­42.5 uL of reverse primer
­6.5 uL of Phusion polymerase
­These were components of master mix
­Each added to 7 uL of template DNA strand
­Annealing temperature of PCR:60 degrees
- Experiments From Jun 24
Make Amp/Kan plates
B_exp23
­Made Amp/KAN plates:
i. Protocol on pg. 7
ii. 12.5 g Miller LB
iii. 7.5 g Agar
iv. put magnetic stir bar into bottle
v. Pour UV deionized water, 500 mL, into bottle (25g/L miller LB, 15 g/L Agar)
vi. Took bottle to be autoclaved
vii. Bottle autoclaved on agar cycle, loose cap, for two hours
­Grew culture of sgRNA plasmids for miniprep the following day
- Experiments From Jun 25
­­Went to Pakrasi lab to observe acetylene injection
B_exp25
- Experiments From Jun 26
­Prepped M9 media
B_exp26
- Experiments From Jun 28
­Prepped culture with aTc
B_exp27
- Experiments From Jun 29
­Prepared for assay
B_exp28
­Prepped for assay
­Spun down cells @ 3200g @ 4 deg C for 15 minutes
­Took out culture tubes, discarded supernatant
­Added 4mL of M9 media to each tube and resuspended
­Centrifuged cells 3200g @ 4 deg C for 20 minutes
­Took out culture tubes, discarded supernatant
­Added 4 mL of M9 media to each tube and resuspended
­Centrigued cells 3200 g @ 4 deg C for 25 minutes
­Toook out culture tubes, discarded superantant
­Added 4 mL of M9 media to each tube and resuspend
Centrifuge cells 3200g @ 4 deg C for 30 min.
­Took out culture tubes, discard supernatant
- Experiments From Jul 06
­Prepped cultures of JM109 with pSL2397 for a miniprep
B_exp29
- Experiments From Jul 07
­­Miniprepped pSL2397
B_exp30
- Experiments From Jul 08
iGEM characterization of BBa_K577895
B_exp31
­Transformed colonies of DH10B containing pSB1C3 with 577895, pAH34, and pAH35
­Prepared cultures of these colonies
- Experiments From Jul 09
Started induction experiment
B_exp32
­Started with induction experiment, but halfway through realized I forgot to make frozen stock, so the experiment was nixed and restarted for the following day
- Experiments From Jul 10
Started induction experiment
B_exp33
­ODs of overnight cultures were measured in the TECAN
­More culture tubes were made containing each overnight culture diluted to an OD of 0.2
­Put these new, diluted culture tubes into incubator for two hours
­While this was happening, prepared well plates for induction
­Calculations were made such that I serially diluted aTc stock concentration 5x across each well
­0.1 mL of each cell culture was added to its respective well
­Well plates were taken and incubated for 6 hours
­At the end of the 6 hours, fluorescence was measured using
­Results showed that part expressed RFP constitutively
- Experiments From Jul 12
Another induction
B_ex34
Transformed iGEM part into another strain of DH10B
- Experiments From Jul 13
Prepped sgRNA cultures
B_exp36
­Prepped cultures of sgRNA plasmid (pTet­pCon­sgRNAT1­p15a­dCas9­I4), DH10B, pAH34, pAH35, 2 cultures of pSB1C3 (one was the first transformation, the other the second)
- Experiments From Jul 14
Induction Experiment
B_exp37
July 14 ­Cultures pAH34, pAH35, and pSB1C3 did not grow again last night, ­Ran an induction experiment with just the retransformed DH10B with pSB1C3(refer to pSB1C3 dd)
­Results showed that iGEM part did not express RFP at all across any inducer concentrations
- Experiments From Jul 17
Induction experiment
B_exp38
­Prepared new stocks of pAH34, pAH35, and pSB1C3. ­Did another induction experiment involving two colonies of pSB1C3 dd, pSB1C3, pAH34, pAH35, and DH10B
­Results were consistent with earlier findings; pSB1C3 dd is colorless, pSB1C3 is red
- Experiments From Jul 18
Prepare more sgRNA
B_exp39
- Experiments From Jul 19
Checked samples
B_exp40
Did Dpn1 of 23 samples. Did chemical transformation for 33 samples. Did Eporation of 8 samples
All Failed
- Experiments From Jul 20
Ran PCR
B_exp41
Ran a PCR for 9 samples, Did gel recovery.
- Experiments From Jul 22
Ran PCR
B_exp42
Did DNA cleanup of 9 samples. Loaded 26 samples for colony PCR. Ran a PCR for 9 samples
- Experiments From Jul 23
Prepped gel for samples
B_exp42
Prepped gel for 9 samples. Prepped forzen stock from colony PCR for 17 samples
- Experiments From Jul 27
Prepped frozen stock of 7 samples
B_exp43
Dpn1 digested two samples, Prepped frozen stock of 7 samples
- Experiments From Jul 29
Dpn1/Gel recovered 8 samples
B_exp45
Dpn1/Gel recovered 8 samples. Retransformed 13 other samples
- Experiments From Jul 31
Did another colony PCR
B_exp46
Did another colony PCR for troublesome samples. Did another PCR for 15 other samples; concentrations were too low post clean-up
- Experiments From Aug 04
Vanderbilt Induction Exp
B_exp47
­Want to run an induction experiment with troublesome Vanderbilt promoters transformed into two separate strains:BL21 and MG1655 ­Grew cultures of these two parts transformed into these two strains as well as wild type strains of MG1655 and BL21
­Results obtained from Vanderbilt induction were nebulous at best
- Experiments From Aug 06
Grew more cultures for Vanderbilt induction experiment
B_exp48
Negative control didn't grow; experiment nixed
- Experiments From Aug 20
Grew more cultures for Vanderbilt induction experiment
B_exp49
­Grew cultures for another round of induction experiments on the Vanderbilt part using OUR protocol.
- Experiments From Aug 23
Grew more cultures for Vanderbilt induction experiment
B_exp50
- Experiments From Aug 24
­Induced with IPTG and put each plate into the incubator to grow for 24 hours, roughly
B_exp51
- Experiments From Aug 25
­Prepared media
B_exp52
­Obtain frozen stock
­Grow overnight in LB and selected antibiotic
­Spin down at 3000 RPM for 15 minutes
­Discard supernatant
­Resuspend in M9 media, 4 mL
­Spin down at 3000 RPM for 20 minutes
­Discard supernatant again
­Resuspend in M9 media
­Spin down at 3000g for 25 minutes
­Discard supernatant
­Resuspend in M9
­Spin down at 3000g for 30 minutes
­Discard supernatant
­Take OD
­Calculate how much you’ll need to have 0.1 OD in 5mL
­Prepare all bottles by adding 10 uL of antibiotic
­Place stopper in bottle
­Place in 30 degree shaker
­Grow overnight
­Next morning, take OD again
­Goal is to have it at 0.2
­React away oxygen in anaerobic chamber
­Put them back in 30 degree shaker
­Next day, inject acetylene
­Take zero point
­Put back in shaker
­Take 24 hour point
Jun_01
B_exp1
Jun_02
B_exp2
Jun_03
B_exp3
Jun_04
B_exp4
B_exp5
Jun_05
B_exp6
Jun_08
B_exp7
B_exp8
B_exp9
Jun_08
B_exp10
B_exp11
B_exp12
B_exp13
Jun_16
B_exp14
Jun_17
B_exp15
B_exp16
B_exp17
B_exp18
B_exp19
B_exp20
Jun_18
B_exp21
B_exp22
Jun_24
B_exp23
Jun_25
B_exp24
B_exp25
Jun_26
B_exp26
Jun_28
B_exp27
Jun_29
B_exp28
Jul_06
B_exp29
Jul_07
B_exp30
Jul_08
B_exp31
Jul_09
B_exp32
Jul_10
B_exp33
Jul_12
B_ex34
Jul_13
B_exp35
B_exp36
Jul_14
B_exp37
Jul_17
B_exp38
Jul_18
B_exp39
Jul_19
B_exp40
Jul_20
B_exp41
Jul_22
B_exp42
Jul_23
B_exp42
Jul_27
B_exp43
Jul_29
B_exp45
Jul_31
B_exp46
Aug_04
B_exp47
Aug_06
B_exp48
Aug_20
B_exp49
Aug_23
B_exp50
Aug_24
B_exp51
Aug_25
B_exp52
Jessica O'Callaghan         Pennsylvania State University
- Experiments From Jun 15
Learn The Enviornment
J_exp1
This was my first week with the iGem team, even though I got to work on some segments of the project from home. My first two days consisted of getting acclimated with the computer programming (Komodo and Cyberduck), and specifically running python and gams files from excel data on the E. coli model. I had the opportunity to work with Laura, who was very helpful in explaining the scripts we were running. Tom was also able to send me information on python and unix commands, so navigating through these programs was easier than expected. I’ve worked on and off with MATLAB, but I actually found python and gams to be a lot more user friendly.
- Experiments From Jun 16
Biomass and ATP calculations
J_exp2
Once I generated my input files using a python code provided to me (I only had to personalize my directory information), I was able to run FBA scripts for both the WM1788 wild type and nitrogen fixation models. I was able to determine maximum biomass values for both, as well as maximum ATP values from these biomass outputs. For the WM1788 wild type model, I got a maximum biomass value of 2.97 and a maximum ATP production value of 547.03. For the WM1788 nitrogen fixation model I got a maximum biomass value of 2.52 and a maximum ATP production value of 391.274. These values checked with Laura’s. From there, I ran FVA on both  the wild type and nitrogen fixation models, and generated spreadsheets for flux data on each reaction in the model.
model, I got a maximum biomass value of 2.97 and a maximum ATP production value of 547.03. For the WM1788 nitrogen fixation model I got a maximum biomass value of 2.52 and a maximum ATP production value of 391.274. These values checked with Laura’s.
- Experiments From Jun 17
The Galactose Pathway
J_exp3
On Wednesday, I started to look at the pathway taken by galactose as it enters the cell, thereafter initiating glycolysis. Given our model predicted higher cell growth rates for galactose as a sole energy source than for glucose, which contradicts most available literature, we were trying to determine if there were any reactions that would help explain this phenomena (for example if there is a reaction supplying additional energy in the galactose model). However, even though I was able to identify the series of reactions the take place immediately following the uptake of galactose, which include converting alpha D­galactose to beta D­galactose, and elementary reactions depending on enzymes such as 4­epimerase and galactokinase, a greater amount of energy was required to initiate the conversion of galactose to glucose to G6P than was required to initiate the conversion of just glucose to G6P. We are still in the process of determining where these inconsistencies are coming from by looking into the first set of reactions that takes place when glucose is introduced to E. coli.
I was able to identify the series of reactions the take place immediately following the uptake of galactose, which include converting alpha D­galactose to beta D­galactose, and elementary reactions depending on enzymes such as 4­epimerase and galactokinase, a greater amount of energy was required to initiate the conversion of galactose to glucose to G6P than was required to initiate the conversion of just glucose to G6P. We are still in the process of determining where these inconsistencies
- Experiments From Jun 18
Knocking out PDH and PFL
J_exp4
Lastly, Laura was working on finding an alternate path in the model that produced acetyl­coA. However, the only reactions listed involve the conversion of pyruvate to acetyl­coA. We are interested in determining which reactions can be potentially knocked out of the model to maximum the production of acetyl­coA. Tom wanted me to run gams files that removed the PDH (pyruvate dehydrogenase) and PFL (pyruvate formate lyase) reactions from the model, which are directly involved in the production of acetyl­coA, to determine if whether or not knocking one out will maximum flux through pyruvate synthase (POR5), which is the only other reaction involved in the production of acetyl­coA. However, our findings show that knocking either PDH or PFL from the model has no effect on the flux through this reaction, or on one another. Considering the flux ranges remained the same (0 ­ 10,000) for the PDH and PFL reactions, these reactions can potentially be removed from the model. When both PDH and PFL were both knocked from the model, the flux ranges varied slightly through the pyruvate synthase reaction­ from ­9,948.9478 ­ 444.251305 (for the lower and upper bounds respectively) to ­4068.693 ­ 444.251. Even though the lower bound was less negative, its magnitude was still significantly larger than the upper bound. You can refer to the PDH, PFL, and POR5 reactions below.
Lastly, we identified that the main reaction converting glucose into G6P was glc_DASH_D_c + atp_c → g6p_c + h_c + adp_c (involves hexokinase) and not pep_c + glc_DASH_D_c → pyr_c + g6p_c ( 0 - 7155.54 to 0 - 20 respectfully)
- Experiments From Jun 19
Knocking out PDH and PFL
J_exp4
Lastly, Laura was working on finding an alternate path in the model that produced acetyl­coA. However, the only reactions listed involve the conversion of pyruvate to acetyl­coA. We are interested in determining which reactions can be potentially knocked out of the model to maximum the production of acetyl­coA. Tom wanted me to run gams files that removed the PDH (pyruvate dehydrogenase) and PFL (pyruvate formate lyase) reactions from the model, which are directly involved in the production of acetyl­coA, to determine if whether or not knocking one out will maximum flux through pyruvate synthase (POR5), which is the only other reaction involved in the production of acetyl­coA. However, our findings show that knocking either PDH or PFL from the model has no effect on the flux through this reaction, or on one another. Considering the flux ranges remained the same (0 ­ 10,000) for the PDH and PFL reactions, these reactions can potentially be removed from the model. When both PDH and PFL were both knocked from the model, the flux ranges varied slightly through the pyruvate synthase reaction­ from ­9,948.9478 ­ 444.251305 (for the lower and upper bounds respectively) to ­4068.693 ­ 444.251. Even though the lower bound was less negative, its magnitude was still significantly larger than the upper bound. You can refer to the PDH, PFL, and POR5 reactions below.
Lastly, we identified that the main reaction converting glucose into G6P was glc_DASH_D_c + atp_c → g6p_c + h_c + adp_c (involves hexokinase) and not pep_c + glc_DASH_D_c → pyr_c + g6p_c ( 0 - 7155.54 to 0 - 20 respectfully)
- Experiments From Jun 22
Adding Flavodoxin to the Equation
J_exp6
At the end of last week, Laura added an additional reaction to the N2 WM1788 model that expends flavodoxin:
We found that the flux ranges changed for over 1,300 reactions. And as for the pyruvate synthase reaction, there was hardly any variation in the flux ranges ( ­9948.95 to 443.698 and ­9948.904 to 443.703 for max biomass and max flavodoxin respectively). We started to look at reactions whose flux ranges varied by more than a factor of 1.0 × 10­1.
- Experiments From Jun 23
Knockout PDH and PFL
J_exp7
After the meeting with iGem, I started researching potential growth implications of knocking out the pyruvate dehydrogenase and pyruvate formate lyase reactions from the model. Our data supports that knocking out either of these reactions individually has little to no effect on the flux through the corresponding reaction, or on the flux through the pyruvate synthase reaction. Research shows that pyruvate dehydrogenase deficiency has been associated with a buildup of lactic acid in the cell. The enzyme has also been identified for its essential role in initiating the series of reactions that make up the TCA cycle. According to a study conducted by Anderson ME, Marshall of RT JFood Safety in 1990, increased concentrations of organic acids have been shown to reduce cellular activity in E. coli, leading to higher levels of population reduction. As for the pyruvate formate lyase reaction, one study examined the effects of knocking out it’s associated enzyme on the yield of butanediol in a strain of E. coli known as Klebsiella pneumoniae. Even though the yield was 92.2% of it’ theoretical value, mutant growth was slightly reduced to the parental strain. Similar studies on other mutants of E. coli suggest that this growth deficiency is due to a redox imbalance rather than a reduced level of acetyl­coA. These findings would eliminate the possibility of knocking out either of these reactions from the model.
After removing this reaction, we had to rerun all of our previous codes to update our figures.
- Experiments From Jun 24
Knockout Malate Oxidase
J_exp8
After knocking out malate oxidase, Laura and I reran FBA codes for the WT and N2WM1788 models. Our updated max biomass and ATP values for the wild type model were 0.821233 and 3.155221, respectively. And for the nitrogen fixation model, 0.454531 and 3.155221, respectively. Our updated ATPmreq constraint was then set from 10.0 to 3.15. We also reran the flavodoxin scripts, and found that our max biomass value, which we derived from maximizing flux through the flavodoxin reaction (0.010442), was 0.4545. I continued rerunning the pyruvate codes with the malate oxidase knockout­ however, I was having some difficulty with running the codes to completion­ several errors kept popping up. I later found out that these issues were arising from available storage space in my terminal, which was later resolved after I consulted Tom.
Original values (N2 outputs)
PDH
PFL
PDH and PFL
Max biomass: 0.454531
Flux range for PDH: 0 to 10,000
Flux range for PFL: 0 to 59.205
Flux range for POR5: ­9986.136 to 9.202
Max biomass: 0.454531
Flux range for PDH: 26.0981627712 to 10,000
Flux range for POR5: ­9964.701 to 9.202
Max biomass: 0.310129
Flux range for POR5: 24.575 to 24.588
Even though knocking out the PDH reaction seemed favorable when compared to the existing N2 WM1788 model, literature reveals that the removal of this reaction is associated with diminished cell growth, resulting from a buildup of organic acids. Therefore, we can not perform either of these knockouts.
- Experiments From Jun 26
Checking directionality
J_exp9
I began comparing the FVA outputs for maximizing biomass and maximizing flux through the flavodoxin reaction. I was interested in identifying which reactions changed directionality with the addition of this reaction. Compared to our original FVA outputs (before knocking out malate oxidase), which detailed some reactions with variations of up to 1.0 × 101 in flux ranges, these updated figures only had variations of up to 1.0 × 10­2. So even though almost 1,000 reactions in the model did have different bound values, these differences were incredibly slight. I was also to identify 14 reaction whose directionality changed upon the addition of the flavodoxin reaction­ I included reactions that had an upper/lower bound of zero that then went in the positive/negative direction.
These reactions include:
- Experiments From Jun 29
Maximizing ATP
J_exp11
Laura and I also began running FVA codes at max ATP, and then comparing those outputs to our existing max biomass outputs.
Even though we identified 976 reactions that changed directionality, 220 of those being non­transporter reactions (independent of ATP cell transport), the slight changes in flux ranges led up to the conclusion that the model is already ATP­limited.
- Experiments From Jun 30
Reporting to Igem
J_exp12
Overview­: Laura and I finished up comparing the FVA outputs for the N2 WM1788 model at max biomass vs. max flavodoxin, but we had no conclusive feedback to report to iGem. Following my updates on the pyruvate knockouts, we reported to Cheryl that there is a potential to remove pyruvate dehydrogenase from the model. But we would need to counterattack any additional buildup of pyruvate from this knockout. We surmised that pushing flux through the pyruvate synthase reaction, and thus maximizing the production of flavodoxin, could account for any compromised cell growth.
The WashU branch was in the process of reworking the existing model to account for non recognition of certain promoters by E coli. These changes did not affect the WM1788 model that we are working with, given the nitrogen fixation reactions already account for any discrepancies. Laura and I completed all of our weekly tasks at the time, and were only responsible for summarizing our results and sending them to Cheryl. In addition to my reports, Tom, Laura, and I came to the conclusion that the pyruvate dehydrogenase reaction can potentially be removed from the model if we account for the build­up of pyruvate in the cell as a result. One way to counterattack this build­up of pyruvate, which will be converted into lactic acid and compromise cell growth/energy, is to push flux through the pyruvate synthase reaction and maximize the production of flavodoxin reductase. This will expend the excess pyruvate concentrations.
- Experiments From Jul 02
Try to maximize ATP
J_exp13
7/2­ Laura and I began running FVA codes for maximizing ATP. We compared these outputs to the max biomass outputs, which we used from our previous flavodoxin trials.
We noticed that most of the reactions whose flux ranges changed by more than 0.001 also changed directionality. We also noticed that many reactions at max ATP had lower bounds that were more positive than their upper bounds. We treated these reactions as if their reaction bounds were reversed. For example, instead of reading the reaction bounds at max ATP for XYLI2 as 1.44E­07 to ­5.893E­7, we read it as ­5.893E­7 to 1.44E­07. A majority of reactions that changed directionality were transporter reactions. Because ATP is indirectly involved with transporting proteins and sugars into the cell, it would make sense as to why maximizing its production would potentially alter the directionality of transporter reactions. This is why we only reported non­transporter reactions that changed directionality, which still consisted of roughly 220 reactions. Of the non­transporter reactions we identified, we only took those reactions whose flux ranges varied by more than 0.005 into account. Reactions that varied by less than 0.005 we regarded as essentially having no change in flux. Analyzing a list this complex is incredibly tedious and would not reveal anything about the effects of maximizing ATP on the model. There are too many factors to take into account. However, our greater difference in flux ranges was 0.02, so there was no need to do a thorough analysis of the data comparisons.
- Experiments From Jul 06
Analysis of high delta flux reactions
J_exp14
Laura and I did a quick analysis of the 317 nonoverlapping reactions which change by more than 0.001 between the N2 fixing and WT WM1788 models. We each generated spreadsheets detailing the number of nonoverlapping reactions in each metabolic system/subsystem, what percentage of the total they make up, and what was the greatest change for each system. Overall, amino acid metabolism had the highest number of changing reactions (78), but the greatest change was only 0.637. Membrane transport, exchange, and inorganic ion transport only made up roughly 15% of changing reactions, but had the largest gap in flux ranges (14.6). We also started brainstorming about potential routes to go down, given we were wrapping up our flavodoxin and ATP analyses. Laura proposed looking more closely at dependence of the model on ATP, given flavodoxin, by contrast, is only involved in a handful of reactions. We started compiling a list of the top 10 reactions that produce/consume ATP. I personally was responsible for identifying consummation reactions. Initially I generated an excel file of reactions that only included ATP, and from there I identified which reactions had the most positive flux ranges, given ATP almost always appears as a reactant in the model.
- Experiments From Jul 07
Igem meeting consequences
J_exp15
In our meeting with iGem, Laura and I reported that the model is ATP­limited, a conclusion that we drew from our max biomass vs. max ATP analyses. We also shared some of the main reactions that either produce or consume ATP, and identified that a majority of these reactionsare apart of the carbon metabolism system. After consulting with Tom, and as a step moving forward, we agreed upon knocking out each individual gene from the model to observe its effect on FBA. Him and Maggie also suggested adding additional metabolites in the model, potentially aiding cell growth. Laura and I decided to divvy up these tasks, with her taking the former and me the latter. My experience with python and gams coding is very limited, so I took it upon myself to begin with reviewing the existing templates that Maggie provided at the beginning ofWeek 1. With the help of a python syntax guide, I was able to get a feel for how to generateoutputs files that can then be used to run FBA and FVA.
- Experiments From Jul 08
Figuring out Overexpression
J_exp16
I continued reviewing my existing python and gams to come up with a way to overexpress each of the metabolites in the model. Tom suggested created a “dummy” transporter reaction in my existing python code, which would carry each individual metabolite into the cytoplasm. I decided to take the list of metabolites already defined in the code and generate a new list which only included metabolites with cytoplasm compartments (Tom and I eventually realized that these metabolites are the only ones we are interested in given we are only concerned with “dummy” exchange reactions). With time and effort, I was finally able to generate a new list entitled metname. And from this list, after defining the new metabolite ids as exchange reactions, I was also to make new sij.txt, rxn.txt, and rxntype.txt output files.
- Experiments From Jul 09
Recalculating with new Metabolites
J_exp17
After compiling my new metabolite input files, I started tweaking with the max biomass code that Laura and I have been using to run FBA. Other than including these additional files in the code, I also included a loop statement, which iteratively runs through each metabolite. The data generated from each loop was then placed in an output file, which included max biomass, max ATP, and max flavodoxin.
Initially I set my max biomass constraint to the max biomass determined from running the code. However, setting it to this parameter overconstrained my output file. For example, only 9 of the metabolites had maximum ATP values other than the ATPmreq (3.15), and only 4 of them produced some amount of flavodoxin (all other metabolites produced none). Tom suggested setting the constraint to the original N2 WM1788 max biomass to see if I would get a wider range of figures. Sure enough, there was a much more diverse set of outputs, and almost all of the metabolites produced flavodoxin.
- Experiments From Jul 10
Carbon Constraints
J_exp18
In order to standardize my outputs, and make any viable recommendations to the UWash team for improving the model, Tom recommended that I constrain the amount of substrate in each exchange reaction. And to do so by the number of carbon molecules in each metabolite. We used glucose as a template, and set the supply at 60 (half the amount supplied by the 6 carbon atoms in glucose (20 per molecule)). I also compressed the list of exchange reactions to only include those metabolite who have at least one carbon atom. We planned on dividing the supply by the number of carbon atoms in each metabolite, so dividing by zero carbon atoms would result in computational errors. It took some time reformatting my input files to only include these carbon­containing metabolites, and I encountered several programming issues along the way. For example, I originally only updated the input file that regulates my substrates, and this resulted in flux outputs that were significantly larger than expected. I’m still in the process of readjusting the parameters in the code to account for the addition of these exchange reactions.
- Experiments From Jul 13
Carbon Overview
J_exp19
Last week, I started looking at the effects of adding supplemental metabolites to the WM1788 N2 fixing model. After creating a set of exchange reactions, and regulating the amount of substrate being imported by each reaction, I was able to generate max biomass, max ATP, and max flavodoxin outputs. We normalized our data by the number of carbons in each supplemented metabolite. This allowed the code to constrain our substrate by the number of additional carbon atoms supplied to the model. In my 7/10 journal, I mentioned that we used glucose as a template for regulating our substrate. I failed to mention why we used only 50%. I want to point out that we felt it was a good level to supplement growth. I was having some coding errors last Friday, and I realized that it was a formatting issue in one of my input files. I was still working through some technical issues by the end of the day. I also started working with Laura on our slides for the Monsanto powerpoint. Her and I wrote up a brief overview of FBA and FVA (refer to term sheet if unfamiliar with abbreviations), our general findings thus far, and current projects we are working on.
- Experiments From Jul 14
iGEM meeting and Monsanto Slides
J_exp20
In our meeting with iGem, Laura and I gave a brief summary on each of our projects, specifically what we’re doing and what our objectives are. After the meeting, we updated our Monsanto slides from Tom’s recommendations (mostly visual layout edits). I continued editing my GAMS code from the updates I made in my input files (only include carbon-containing metabolites).
- Experiments From Jul 15
Compile and Run?
J_exp21
After double checking my GAMS parameters, variables, and input files, I was finally able to run my code and generate flux outputs.
While I was reading through them, I realized that some of my carbon-number figures were off. I originally generated a carbon number input file from the iAF model. Specifically, I ran a python code that identifies the number of carbon atoms in each metabolites chemical formula. However, I realized that some of the metabolites were defined differently in the iAF model than in the WM1788 model. I had to manually go through the model and update any figures that were not matching up.
- Experiments From Jul 16
Finding Metabolites
J_exp22
After tediously updating my carbon number input file, I reran my GAMS code and collected my updated outputs in an excel file. Tom recommended I generate a scatter plot of max biomass vs. max ATP to see if I could identify any observable trends. Him and I noticed a portion of the plot that followed a rigid linear trend. Of the metabolites that were along this trend line, one was glucose, our models primary source of energy. Because our objective was to look into supplemental metabolites/energy sources, specifically those with higher flux values than glucose, we were only interested in those metabolite above glucose on the trend line. This consisted of 73 metabolites.
Given several metabolites in the model are either intermediates or impractical to isolate and provide the cell, I started doing a literature check on each one to determine its candidacy. Even though I only ruled out a handful of metabolites that are intermediates, all of the remaining metabolites were either phosphorylated or sugars, specifically glucose derivatives. We were only interested in these six sugars, which include maltotriose, maltotetraoise, maltose, maltopentaose, maltohexaose, and maltoheptaose.
- Experiments From Jul 17
Writing a summary
J_exp23
I completed my summary of metabolite additives for Cheryl, including a project overview, the general procedure I followed, constraints and reasons for such, and findings. We believe that the addition of these sugars, providing additional sources of energy, can promote cell growth and activity. I also added mine and Laura’s slides to the Monsanto powerpoint.
- Experiments From Jul 21
Meeting with iGEM and Succinate miracle?
J_exp24
Laura and I summarized the results of individual trials in our meeting with iGem. After mentioning the 6 sugars I identified from the addition of our metabolite exchange reactions, one member of the WashU team mentioned hearing of an outside branch having success with replacing glucose as a sole energy source with succinate. From this, I started transitioning on to a new topic, specifically other carbon-containing substrates that can replace glucose in our model. Tom also sat down with Dr. Maranas, who would orchestrate funding for our trip to the Jamboree. Even though Dr. Maranas oversees student research in our branch of the department, he does not play a direct role in the work Laura and I do. To verify that her and I are making productive use of our time here, he requested that each of us write a report detailing our project with iGem, along with our findings and future objectives.
- Experiments From Jul 22
Reporting Results
J_exp25
I began writing my report for Dr. Maranas, detailing the motive behind our work here and our ultimate goal of giving plants the ability to produce their own nitrogen. I also mentioned that our branch specifically oversees genome-scale modeling to help guide potential genetic interventions and facilitate nitrogen fixation in E. coli. From our model, we can use FBA to determine the flow of metabolites, or flux, through the network. And from gene knockouts/overexpressions, we can identify available pathways for producing/consuming a metabolite of interest, such as ATP.. As for our results, I focused primarily on how ATP and reduced flavodoxin are the limiting factors in our nitrogenase reactions. Specifically, I mentioned how we attempted to maximize reduced flavodoxin by manipulating pyruvate metabolism. I also focused heavily on my recent work with the metabolite exchange reactions, and how we are looking for ways to help aid cell growth/activity. From this, I have begun looking into other carbon-containing substrates that can replace glucose as the model’s source carbon source.
- Experiments From Jul 23
Checking out Succinate
J_exp27
I started editing my codes to begin running an FBA analysis on replacing glucose with succinate as our model’s sole carbon source. However, after normalizing our substrate to the number of carbon atoms in succinate, 30 mmol/g DW hr succinate vs. 20 mmol/g DW hr glucose, the model proved infeasible. After consulting with Tom, he suggested looking into studies on E. coli growth on succinate to determine if any there are any additional parameters we need to account for, like the presence of another carbon source.
- Experiments From Jul 24
Succinate + Fumarate < Glucose?
J_exp28
I stumbled upon a few online studies that detailed pairing succinate and fumarate as  carbon sources for E. coli. Growth deficiency on succinate in our model, from my interpretation, has to do with the essentiality of two enzymes: succinate dehydrogenase and fumarate reductase. Fumarate is derived from the breakdown of glucose in the glycolysis pathway, so replacing glucose with succinate results in a deficiency of fumarate in the cell. I assumed that these two enzymes, which are responsible for promoting microbial metabolism and play an essential role in the citric acid cycle, can not catalyze cell activity if both succinate and fumarate are not at sufficient levels. When I added fumarate as an additional energy source, the model did experience some level of growth, but the max biomass value was only 0.269648 in comparison to 0.454531 for glucose. Tom recommended that I look into the pathway taken by succinate in E. coli determine if any transport reactions have been overlooked in our model. On a side note, I found one study that redirected the metabolism of glucose to succinate, and detailed increased succinate yields from increasing the expression of phosphoenolpyruvate carboxykinase and inactivating the glucose phosphoenolpyruvate- dependent phosphotransferase system. This resulted in increased ATP production and increased concentrations of phosphoenolpyruvate for carboxylation. Inactivating the pflB gene, which encodes pyruvate formate lyase, was also associated with increased succinate yields. However, the primary goal of this particular study was to increase the production of succinate, so I’m not yet sure if these same steps can be taken to maximize growth in our model.
- Experiments From Jul 27
OptKnock comes knocking
J_exp30
While Laura continued running her double gene knockout code, Tom and Maggie mentioned a code called OptKnock. Optknock suggests a list of potential reaction knockouts to couple growth production with the production of a desired product, in our case max biomass and reduced flavodoxin, respectively. Biomass values will vary depending on the percent biolevel we set in our code (refer to 7/29 for more details). Laura and I spent some time reviewing a chapter on OptKnock to become more familiar with it’s structure and objectives. The goal of running this code is to provide another route for identifying potential knockouts in our model.
- Experiments From Jul 28
Succinate could help Respiration
J_exp31
During our meeting with iGem, Charlotte clarified that using succinate as a carbon source is meant to promote respiration in the cell. I want to note that prior to the meeting, I also ran FBA on our N2 model with both glucose and succinate as carbon sources. I found that my max biomass and max ATP figures were identical to my original nitrogen fixing figures (with glucose as the sole carbon source). However, increased respiration would not appear in FBA, so Maggie suggested running FVA and comparing flux ranges for reactions in the citric acid cycle. The citric acid cycle provides a pool of chemical energy from the oxidation of pyruvate, so changes in flux ranges for reactions associated with this subsystem would imply changes in cellular respiration. After running FVA with the addition of succinate, I found that a majority of reactions in this subsystem had altered flux ranges in comparison to their original flux values. However, these changes were incredibly slight- i.e. less than a factor of 1*10^-2.
- Experiments From Jul 29
Start working with OptKnock
J_exp32
Laura and I began editing a OptKnock template that Maggie provided us. Some of these edits included our reaction input files and our ATP and percent biolevel constraints. We initially encountered some issues with getting our code to run, but we quickly identified them and were able to generate a list of potential double knockouts, which consisted of roughly 400 couples. At first glance, all of the double knockouts had the same max biomass and reduced flavodoxin levels. We set our percent biolevel to .9, so each double knockout had a max biomass value of 0.409 (.9 of 0.4545- i.e. our original max biomass value). We wanted to double check and see what effect these knockouts would have on max biomass and ATP levels in our model by running them through an iteration. Laura wrote a python code that splits a reaction input file into a series of coupled-knockouts, given our previous OptKnock output file only lists the couples without pairing them. She originally had to generate this code to run her gene knockouts, so all we really had to do was substitute in our new input file. While Laura edited her python code, I went ahead and ran OptKnock for triple reaction knockouts.
- Experiments From Jul 30
Knockouts did not have expected Results
J_exp33
After running our python code for both double and triple reaction knockouts, we found that none of the proposed knockouts from our OptKnock code were coupled with reduced flavodoxin. In other words, every knockout had the same max biomass and ATP level as our original nitrogen fixation model- i.e. without any knockouts. The only useful piece of information we drew from our outputs was that several reactions involved in pyruvate metabolism were included in the suggested knockouts. This would confirm some of our previous suggestions on increasing reduced flavodoxin production by manipulating pyruvate metabolism in the cell.
- Experiments From Jul 31
Summary for Website
J_exp35
While Laura resolved these issues in her code, I began our modeling summary for the website. Laura and I felt that an introduction, associated methods, formulation, and results section were appropriate. Because Laura is still working on her python code, I decided to cover the first two sections and my results from the metabolite exchange reactions.
- Experiments From Aug 03
Fixed another Bug
J_exp36
8/3­ Laura regenerated outputs for her double­gene knockout code, but inconsistencies were still popping up in our data when we manually tested knockouts coupled with flavodoxin. We were finding that for non­infeasible knockouts, the maximum biomass, ATP, and reduced flavodoxin values were completely off from those in the iteration outputs. We finally were able to figure out that the code was coupling each gene with itself, resulting in several infeasible outputs. However, instead of skipping over these knockouts, the code outputted the previous set of output values, resulting in mismatched data. We resolved this issue but removing duplicate genes in our code.
- Experiments From Aug 04
Work on Website Summary
J_exp37
Following our meeting with iGem, Laura and I finalized our travels grants and began working on the modeling summary for our wiki page. We wanted to draw attention specifically to what genome­scale modeling is and what we’re hoping to accomplish using methods such as FBA (navigating potential genetic interventions to optimize cofactors of nitrogen­fixing in E. coli). Because most of our results were inconclusive, we wanted to focus on our discoveries with pyruvate metabolism, and how we can promote the production of reduced flavodoxin by redistributing flux through reactions involving pyruvate. We also included brief summaries of our recent work on metabolite exchange reactions and gene/reaction knockouts.
- Experiments From Aug 05
Work on Website summary
J_exp38
Laura and I finalized the first draft of our wiki page, after reworking it to include more visuals and minimal text. Even though computational modeling is extensive, we tried to summarize key points in flow charts and supplement them with brief descriptions. Laura also collected all of the outputs for the double gene knockout code. Even though our outputs were finally starting to match up, we found, unfortunately, that any knockout coupled with flavodoxin outputted an infeasible model.
- Experiments From Aug 06
Review First Draft
J_exp39
Laura and I sat down together and reviewed the completed first draft of our modeling summary. Considering Laura had to leave early, I finalized any remaining edits, such as redesigning our section on genome­scale modeling. We wanted to break up any big chunks of texts. Before Laura left, her and I also wrote up our section on computational modeling for the abstract. Given the length constraint, we focused primarily on what our goal was in using computational modeling to promote nitrogenase activity.
- Experiments From Aug 07
Finished First draft
J_exp40
I reviewed the final first draft of our wiki modeling page, and any edits that the WashU team made to our abstract.
Jun_15
J_exp1
Jun_16
J_exp2
Jun_17
J_exp3
Jun_18
J_exp4
Jun_19
J_exp4
Jun_22
J_exp6
Jun_23
J_exp7
Jun_24
J_exp8
Jun_26
J_exp9
Jun_29
J_exp10
J_exp11
Jun_30
J_exp12
Jul_02
J_exp13
Jul_06
J_exp14
Jul_07
J_exp15
Jul_08
J_exp16
Jul_09
J_exp17
Jul_10
J_exp18
Jul_13
J_exp19
Jul_14
J_exp20
Jul_15
J_exp21
Jul_16
J_exp22
Jul_17
J_exp23
Jul_21
J_exp24
Jul_22
J_exp25
Jul_23
J_exp26
J_exp27
Jul_24
J_exp28
Jul_27
J_exp29
J_exp30
Jul_28
J_exp31
Jul_29
J_exp32
Jul_30
J_exp33
Jul_31
J_exp34
J_exp35
Aug_03
J_exp36
Aug_04
J_exp37
Aug_05
J_exp38
Aug_06
J_exp39
Aug_07
J_exp40
Mike Toomey         Washington University in St. Louis
- Experiments From Jun 01
Preparation of Electrocompetent Cells
B_exp1
These are the strains we put the nif cluster in this summer.
Prepare cultures of DH10B
25 ml of each culture were transferred into 500 mL of LB
-These flasks were then grown at 37 deg C, 250 rpm for 1 hour, 30 min in incubator
Measure absorbance
Cuv1-.1055, Cuv2-.1084
After absorbances were measured, cultures were added to ice bucket and allowed to cool for 15 minutes.
-Centrifuge bottles were centrifuged for 25 min. at 4 deg.
After centrifuge, bottles were taken to cold room
Chilled 10% glycerol was added, 55 ml each, into each bottle
It’s worth noting that % glycerol in each bottle is roughly 4.4%
70 Ml of water were added to each bottle
-Bottles were placed back into centrifuge for another 256 minutes at 4 deg C-After centrifuge, bottles were taken to cold room once more
Supernatant was drained
10% glycerol was added to 4 tubes
Roughly 15 mL were allotted to each tube
Tubes were taken and centrifuged
After centrifuge, conical tubes were taken to cold room
Supernatant was drained
1 mL of 10% glycerol was added to each tube, pellet allowed to resuspend
Tubes were taken to centrifuge again, 3000 RPM @ 4deg C
- Experiments From Jun 02
Designed program to find sgRNAs in Java,
B_exp2
Need to find a place for dCAS9 to bind
Designed program to find sgRNAs in Java, key feature is finding the PAM sequence
- Experiments From Jun 03
More electrocompetent cell prep
B_exp3
EC cell prep for Strains: MG1655 and JM109
Two culture were incubated overnight
-In the morning, 26 mL of culture were placed into 500 mL of LB
-These flasks were incubated for an hour beginning at 10:34
-Both flasks were taken out around 11:35 P.M.
-Optical measurements were performed on both colonies using a 10x dilution factor
i. The optical density of JM109 was 0.287 ii. The optical density of MG1655 was 0.39
-JM109 went back to be incubated at 11:56 A.M.
-Optical measurement performed on JM109 once again
Optical density was 0.38
-Optical density for both strains was determined to be satisfactory -Thus, each flask was equally partitioned into two centrifuge bottles
-Centrifuge bottles were left to sit for 20 minutes
-After sitting for 30 minutes, bottles were centrifuged at 3000 RPM for 25 minutes
-Upon centrifugation, supernatant was drained, pellet was resuspended in 125 mL of water
-Bottles were centrifuged again for another 25 minutes
-Upon centrifugation, supernatant was drained and 55 mL of 10% glycerol were added to each bottles
-Pellet was resuspended in each bottles
-After resuspension, each bottles was partitioned into 4 15 mL conical tubes
-Conical tubes were centrifuged under same conditions
-From a combined conical tube, 40uL aliquots were created for JM109 and MG1655
-These aliquots were stored in -80 deg C conditions Electroporation
-We transformed MG1655 and WM1788
-To do this, we obtained copies of pSL2397 plasmids
-2.5 uL of this plamsmid DNA was collected and mixed with pipettor
-After this, the plasmid DNA was inserted into a cuvette
-The entirety of the aliquot containing the E.Coli strains was dumped into the cuvette
-The plasmid was shocked into the strain for 5.5s
-Strain was taken out and put into conical tube for an hour
-The same process was repeated with MG1655 strain
- Experiments From Jun 04
-We finalized sgRNA sequences
B_exp4
-In picking sgRNA sequences, we determined that sequences close to the beginning of the strand would be the most effective -Using a computer program, we ran gene sequences through and found the most effective sgRNA sequences i. Java program that predicates itself on finding the PAM sequence
- Experiments From Jun 05
Preparation of Frozen Stock
B_exp6
Store Cells from yesterday
-Cells were taken from picked colonies on June 4
-We took 500 uL of that culture, added it to 500 uL of 30% glycerol solution, then place din -80k deg C conditions.
-We then prepared 30% glycerol solution for later use
-Next, we prepared the CRIPSR/dCas9 plasmid for PCR
-To do this, we separated the plasmid from its host bacterium
-First, we took the bacterial culture and centrifuged it
-Then, we discarded LB growing solution that was supernatant
-Next, we resuspended bacterial cells in water
-Buffer to lyze the cell was added
-Immediately after, cold neutralization buffer was added to neutralize the process
-Then, tubes were centrifuged at 16000 g for 43 minutes
-Supernatant was transferred to a separation column
-These columns were then centrifuged for 30 seconds
-The flow through was discarded and the colonies were placed back into the collecting tubes
-200 uL of Endo-Wash Buffer were added to each tubes
-Each tubes was centrifuged under same conditions for 30 seconds
-Upon centrifugation, 400 uL of Zappy’s buffer were added to each tube
-Each tube was centrifuged for another 2 minutes
-Upon centrifugation, high pure molecular water was added to each column and each column sat for two minutes
-Each column was centrifuged, same condition, for another 2 minutes
-After this step, dissolved plasmid DNA had collected on bottom of column
-Each column was taken to spectrometer to determine concentration and purity
-Concentrations and purities, in order
Tube
Conc (ng/uL)
Purity
Tube 1
1.87 127.1 Tube 3 1.79 82.7
After that, each DNA sample was placed into freezer to be stored
- Experiments From Jun 08
Prepare Gel
B_exp9
We prepared a gel for DNA purification
-Mix 1g of Agarose with 100 mL of TAE buffer.
Note that you can cool the mixture with cold, running water
-Add 5 uL of cybersafe (5uL/100 mL of buffer) once mixture cools
-Put in combs
-Tape side of tray
-Pour agarose gel in tray
-Let gel cool for 1 hour
Run gels for 1 hour
- Experiments From Jun 09
Made culture
M_exp10
Made culture for the transformed DH10B so that we could create frozen stock
- Experiments From Jun 10
Helped out in Zhangs lab
M_exp11
Helped Charlotte and Carlos in the Zhang lab as they ran PCR's, gels, DNA purification
- Experiments From Jun 11
Zhang RE Digest (30 microl)
M_exp12
- Experiments From Jun 12
PCR
M_exp13
Preparing samples
Add HP molecular water 10x the amount of DNA in primer tube to get a 100 uM concentration after centrifuging
Vortex each tube
Preparing solutions
Create 10 uM solutions of DNA by transferring primer DNA from 100 uM solution to a microcentrifuge (10 microl) and add 90 microl water
Make a mastermix for the PCR, which should have an amount of liquid n+ 1 times the number of rexn you have
Per rxn
1 microl template
.5 microl phusion polymerase (mastermix)
2.5 microl forward primer
2.5 microl reverse primer
1 microl dNTP's (mastermix)
4 microl DMSO (mastermix)
Fill rest of 50 microl with water (micromix)
Get out 150 ml mcf tube per reaction and add mastermix plus DNA
Thermocycler
Put tubes in top tray and close lid
Tighten the dial on lid so that it's tight, but doesn't break off
Choose the new protocol button
Make sure to change the vol
Make sure to change the polymerization time (72 C) to make sure phusion can extend whole DNA strand
Phusion's rate is 1000 bp/15 sec
Change primer annealing temp. to what you want
Should be 4 degrees C less for DMSO solutions
Making an agarose gel
Measure out 1 g of agarose and add to 100 ml of TAE in a 250 ml flask
2. Put in microwave for 1.5 min.
3. Let cool after microwaving and then add 7 microl Sybersafe
4. Make a gel plate by drying sides off and taping twice
5. Pour gel into plate until it reaches comb inserts level
6. Make sure no air bubbles are in gel by combing the gel with comb
7. Place comb into gel and let sit for 1.5 hrs.
8. Throw out all trash into gel bin
Loading a gel
1. Take comb out perpendicularly and place plate in clean buffer
2. Grab samples from Thermocycler on 50 microl tray
3. Dilute samples with loading dye from fridge to 1x to 6x
a. If PCR products are 50 microl, this means adding 10 microl dye
4. Mix by drawing up ~60 microl from the bottom of tube, not middle
5. Put in well, approaching perpendicularly
a. Do not go in so far as to poke gel!
6. Continuously push the product into the well and pull the pipetter out as
soon as the first stop is hit
7. Load DNA ladder (5 microl) (1 kb)
8. Place black end of electrode onto side with DNA
9. Press mode on machine 3 times to find time, set to 30 min., let sit
PCR
1X
9x
Template
1 microl
-
Pusion
.5 microl
4.5 microl
F Primer
2.5 microl
-
R Primer
2.5 microl
-
dNTP's
1 microl
9 microl
HF Buffer
10 microl
90 microl
DMSO
4 microl
36 microl
Water
28.5 microl
256.5 microl
- Experiments From Jun 13
DNA ligation
M_exp18
i. Want 100 ng backbone (bb) with 5x insert (in)
ii. (5)x(100 ng bb/(6819bp/bb))x((820 bp/in)/(46.7ng/microl)) = 1.28 microl of insert
iii. 100 ng bb/45.1 ng/microl = 2.22 microl bb
iv. Totals
8 microl
1 microl T4 ligase
1 microl buffer
.2 microl PNK
let ligation reaction sit for 1-1.5 hrs.
transform cells with plasmid
Jun_01
B_exp1
Jun_02
B_exp2
Jun_03
B_exp3
Jun_04
B_exp4
Jun_05
B_exp6
Jun_08
M_exp6
M_exp7
M_exp8
B_exp9
Jun_09
M_exp10
Jun_10
M_exp11
Jun_11
M_exp12
Jun_12
M_exp13
Jun_13
M_exp14
M_exp15
M_exp16
M_exp17
M_exp18
David Ayeke         Washington University in St. Louis
- Experiments From Jun 02
Make CμLtures
D_exp2
The Moon Lab needed, strains JM109 and MG1655 for their experiments
Make overnight cμLtures from frozen stock to make electrocompetent cells the next day (in Moon lab):
from glycerol stock, set up two 50mL cμLtures in 250-mL flasks
grow overnight
- Experiments From Jun 03
Make Lb
D_exp4
The Moon Lab needed, strains JM109 and MG1655 for their experiments
Transformation (in Moon lab):
- Take MG1655 and WM1788 electrocompetent cells out of -80°C and place on ice
- Take PSL2397 (plasmid) out of -80°C and place on ice
- Add 2 μL of PSL2397 directly into MG1655 tube
- Set pipette over 40 μL and draw up the MG1655 and PSL2397 mixture and place into electroporation cuvette
- Tap cuvette to ensure cells are at bottom of cuvette—shoμLd check to see that there are no gaps—and place into electroporator; turn on
- Immediately add 500 μL of LB into the cuvette then pour into cμLture tube
- Place cμLture tubes of WM1788 and MG1655 into incubator for 1 hour
- Experiments From Jun 04
Make Lb
D_exp6
There are 10 genes in the Nif cluster that have unknown functions. Overexpressing these can shead light on what they do
Use SnapGene to design primers for the 14 sequences to be overexpressed
- Check for EcoRI and XhoI restriction sites within each of the sequences
- Check for directionality on the plasmid: direct or complementary
- If a restriction enzyme does not have sites within the sequence, add a site for that restriction enzyme to the appropriate primer and add 6 adenines beyond the site on the primer so that the restriction enzyme will work properly
- If the restriction enzyme does have a site within the sequence, end the primer at the end of the sequence to be amplified to leave the ends blunt
- If direct, add EcoRI to the 5’ end and XhoI to the 3’ end as appropriate
- If complementary, add XhoI to the 5’ end and EcoRI to the 3’ end as appropriate
- Maintain a Tm for each primer above 60°C and approximately match the Tm of paired primers
- Ensure that there is only 1 binding site on the plasmid for each primer
- Experiments From Jun 09
Run PCR
D_exp8
Run PCR on 3 of the 14 amplicons
- Experiments From Jun 10
Run PCR
D_exp9
Run PCR on remaining amplicons
- Experiments From Jun 11
Overnight cμLtures
D_exp14
Start an overnight cμLture in 5 mL of LB at 37°C for transformation of ligated plasmids
- Experiments From Jun 12
Plate Cells
D_exp18
- Take cells out of 37°C room and spin down all but the control for efficiency of transformation (PE5A) for 4 minutes
- Take 100 μL from PE5A control and add to 900 μL of LB
-Plate 100 μL of that dilution on the amp plate
-Add sterile glass beads and shake laterally to spread around the cμLture
-Dump beads into nonsterile glass beads container
- from centrifuged cμLtures, pipette off the media to the 100 μL mark and resuspend the pellet in that amount of media
-Add fμLl resuspended quantity to the correct labeled plate
-Add beads and shake
- Place all plates in 37°C room
- Experiments From Jun 13
Store plates
D_exp19
Take plates out of 37°C room, check for colonies, and place those that grew into the 4°C room for the remainder of the weekend
- Experiments From Jun 15
Run Gel
D_exp25
- Run digested plasmid through the gel
- Saw no band for digested plasmid and a band in unexpected location for the undigested plasmid need to redo the digestion of the plasmid
- Experiments From Jun 17
Confirm Inserts
D_exp26
Make 20 μL PCR reactions for plasmids purified yesterday (to attempt to confirm correct inserts)
- Experiments From Jun 18
Digest
D_exp33
Digest the blunted cut plasmid with Xho1 in a 25 uL reaction
Add the following to the Klenow reaction mixture (without purifying first)
2 uL digested buffers
2 uL Xho1
1 uL water
Place at 37C for ~3 hours
- Experiments From Jun 19
Check plates
D_exp34
Carlos checked plated cells for growth
1 colony on control; ~30-50 colonies on other plates
Placed plates in 4C room
- Experiments From Jun 20
Pick colonies
D_exp35
Carlos picked 2 colonies per plate and started overnight cultures in LB
- Experiments From Jun 22
Transform
D_exp38
Transform plasmids from overnight
Purify plasmid from overnight cultures of colonies
Quantify cocentrations of plasmids
Set up 20 uL PCR reaction for the purified plasmids to confirm inserts
transform ligated plasmids
Add the ligation reaction products produced to labled vials of competent cells
Incubate cells with ligation products added on ice for 25 minutes
Heat shock at 42C for 1 minue; replace on ice for ~2 minutes
Add 900 uL of LB to each tube (in the hood)
Place on shaker at 37C for 1 hour
Pour plates
Melt LB agar in a secondary container of water in the microwave
Wait for agar to cool before adding 1 uL of chloramphenicol per mL LB agar
Pour plates with ~20mL agar with antibiotic
Allow plates to dry in hood
Plate cells
Centrifuge, pipette off the supernatant to 100 uL mark, then resuspend the pellet in that amount
Add resuspended cells to labeled plates; spread with with sterile glass beads. Place plates in 37C room
Run a gel of PCR products (to confirm inserts in transformed cells)
- Experiments From Jun 23
Check colonies
D_exp39
Check for colonies from plated cells
Control grew colonies, as did the others
Pick a colony with a pipette tip and resuspend colony in the media
Add 50mL of LB to a conical tube; add 50 uL of chloramphenicol to that
Add 5 mL of that mix to each of the 10 labeled sterile culture tubes.
Pick a colony with a pipette tip and then either resuspend the cells in tube or drop the tip into the tube
Plate on shaker in 37C room at 250 rpm
Add 5 mL LB to another steril culture tube. Add 5 uL LB to the tube. Scrape frozen stock of PA2C-TesA with a pipette tip and place the tip in the Lb; place on shaker in 37C room
- Experiments From Jun 24
Miniprep PA2C-TesA
D_exp44
Miniprep PA2C-TesA (using 10X the amount of buffers as suggested by the kit)
Pour the 50 mL culture into a 50 mL conical and centrifuge to a pellet
Pour off the supernatant
Resuspend in 2500 uL of resuspension buffer and vortex or pipette in and out until no clumps remain
Add 2500 uL of lysis buffer and invert 10 times to mix (no vortexing). Incubate on ice for 5 minutes
Centrifuge at 13,000 rpm and 4C for 10 minutes
Pour the supernatant through a syringe with a cotton ball in it to filter
Transfer 800 uL of supernatant to a column in a collection tube and centrifuge at 13,000 rpm for 1 minute.
Discard the filtrete and replace in the same collection tube
Repeat previous steps with additional 800 uL of filtered supernatant until all supernatant has gone trhough column
Add 500 uL of wash buffer A and centrifuge at 13,000 rpm for 1 minute. Discard the flow through andreplace column in the tube
Centrifuge at 13,000 rpm for 1 minute
Place on thermomixer at 65C for 5 minutes
Place column in a new  1.7 mL microcentrifuge tube and add 50 uL of water warmed to 65C; let sit at room temperature for 1 minute
Centrifuge at 13,000 rpm and 1 minute
Quantify concentration on nanodrop
On 2% agarose gel, add: 2.5 uL 1kb pluss ladder to wells on either side. 8uL of each PCR product
- Experiments From Jun 25
Run gel
D_exp44
Run a gel to see if PA2C-TesA purified yesterday is okay
Pour a gel with 10 wells of ~35 mL 1% agarose with 2 uL EtBr
Run 100 ng of plasmid max -> 2.5 uL with .5 uL loading dye
2.5 uL ladder on each side
- Experiments From Jun 26
Repeat transformation
D_exp45
Purify digested PA2C-TesA with DNA clean & concentrator kit
Start a Klenow reaction
Digest the plasmid from the Klenow with Xho1
load the digested plasmid into a gel and gel purify the correctly sized band
Ligate the digested insert with digested plasmid
Transform the ligation products into chemically competant cells
- Experiments From Jun 30
Start overnight of more PA2C-TesA
D_exp46
- Experiments From Jul 01
Attempt to PCR the inserts from the HDK overexpression plasmid using the nifHDK F and R primers
D_exp48
- Experiments From Jul 02
Redo HDK PCR
D_exp50
Redo HDK PCR from nif cluster plasmid to attempt construction of the HDK overexpression plasmid again
- Experiments From Jul 03
Primer design for minimal nif plasmid
D_exp51
- Experiments From Jul 06
Redo PCR of HDK and cce_0550
D_exp53
Cut and purified band for 0550, but HDK band was incorrect
- Experiments From Jul 07
Ligate 0550 and HDK
D_exp54
Ligate genes into plasmid
Repurify 0550 PCR product (due to a bad peak on nanodrop)
Digest repurified 0550 with Xho1
Clean up digest with DNA clean & concentrator kit
Set up ligation for 0550 and HDK with PA2C-TesA
- Experiments From Jul 08
PCR genes for minimal nif plasmid-grouped by size into 3 sets
D_exp55
PCR for small and large size groups
Gradient PCR from 58C to 65C
100 uL rxn split into 4 tubes of 25 mL each, at different temperatures in gradient thermocycler
Run gels of a small amount of each of tube. Saw bands for very few of the expected products
- Experiments From Jul 09
PCr medium size group
D_exp56
- Experiments From Jul 10
PCR for minimal nif plasmids continued
D_exp57
- Experiments From Jul 13
PCR more 0550 and HDK for overexpression plasmids
D_exp58
EcoR1 digest of PA2C-TesA
Run 0550 PCR product in a gel and gel purify it
Digest 0550 with Xho1
Run HDK PCR product in a gel and gel purify it
- Experiments From Jul 14
Prepare 1 plasmid of nif cluster
D_exp59
PCR amplify tetracyclin resistance gene off of one plasmid and pB1a except for the amplicilin resistance gene, using primers designed for Gibson ligation
Blunt the ends of PA2C-TesA cut with EcoR1 using S1 nuclease
Add at least 10 units of S1 nuclease per microgram of plasmid, but use 10 units if you use less than 1 microgram
Use a 10x final concentration of buffer
Add water to a 30 uL final reaction volume
Place at room temperature for 45 minutes, then add 4 uL of .25M EDTA and incubate at 75C for 20 minutes to stop the reaction
After blunting with S1 nuclease, clean up with DNA  clean & concentrator kit, then start an Xho1 digest
Set up Gibson assembly for backbone resistance change for one of the minimal plasmids
Use 7.5 uL of Gibson mix premade by another member of the lab
Dilute to a 10 uL final reaction volume
Place at 50C for 1 hour to ligate
Add 50 ng of each piece to be ligaged
Based on the concentrations of DNA we had, we were unable to add 50ng of each piece, and maximized the amount of DNA added by not diluting with water
transform Gibson assembly products using chemically competent transformation protocol
Cut PA2C-TesA backbone band from gel and gel purify
Make new tetracyclin stocks for Gibson assembly products transformation
Measure 50 mg of tetracycline hydrochloride on weighing paper
Add 70% EtOH to a 10 mL final volume; mix
Add 4 uL per mL to 20 mL Lb (80 uL of tetracycline)
Checked on 7/16/2015, no colonies from Gibson products
- Experiments From Jul 16
Transform ps8k plasmid into DH10B to make more
D_exp61
- Experiments From Jul 17
Golden Gate reactions
D_exp63
Set up a 10 uL Golden Gate reaction of all 8 pieces for the pS8K cysE2-hesB minimal plasmid, using a molar ratio that compares concentration/size(bp) of each piece. Inserts should all be in a 3 to 1 ratio to the backbone
mix reaction mixtures and DNA together
DNA should be 7 uL
Add: 1 uL T4 ligase buffer, .5 uL Bsa1, .5 uL T4 ligase, .3 uL Dpn1, .5 uL ATP at 10 uM, .3 uL BSA
Spin down and put in thermocycler
37C for 3 minutes
16C for 4 minutes
Repeat first 2 steps 50 times
50C for 5 minutes
80C for 5 minutes
Hold at 4C
- Experiments From Jul 20
Pick colonies
D_exp64
Pick 5 colonies from DH10B plate of golden gate reaction mixture transformation. Pick 1 colony of ps8k
- Experiments From Jul 21
Work on HDK & 0550
D_exp65
Digest PA2C-TesA with EcoR1, PCR more 0550 and HDK
- Experiments From Jul 22
Golden gate cysE2-hesB
D_exp67
Run a gel to check for cysE2-hesB PCR product
Run a gel to cut the backbone from the PA2C-TesA Xho1 digest
Digest ps8k minipreps with pst1 and Pme1 to check for proper sizes
Set up a golden gate of the seven inserts only, excluding the backbone from this reaction, using a 1 to 1 molar ratio of all Golden Gate inserts
- Experiments From Jul 23
Repeat Golden Gate
D_exp68
Set up a PCR using the 7 insert Golden Gate reaction as template: make 2 15 microliter reactions, one using 1 microliter of the golden gate as template and the other using 1 microliter of a 10x dilution of the template
Run a gel of a small amount of the ligation mix(template for the PCR)
Start digest of HDK and 0550 PCR products with Xho1
Clean and concentrate the digest
PCR of inserts with 10x dilution of template looks good -- start a 100 uL reaction scaled up
Run a small amount of cleaned up digest products on gel to check for proper bands -- got expected bands
PCR tetracyclin and pB1a for the backbone resistance change with the newly-arrived Golden Gate primers(rather than the Gibson primers)
- Experiments From Jul 24
More Golden Gate
D_exp70
Re PCR the ps8k backbone to ligate cysE2-hesB into
transform golden gate ligation of pB1a and tetracycline into DH10B chemically competent cells
Cut and purify the 7 gene ligation PCR from the gel
Set up a Golden Gate ligation of the 7 insert PCR product and the amplified pS8k backbone in a 10 uL reaction
- Experiments From Jul 25
Transform the Golden Gate product into chemically competent DH10B
D_exp71
- Experiments From Jul 27
Check for inserts
D_exp72
Miniprep the overnight of ps8k, ps8k cysE2-hesB 1 through 5 ( 5 colonies picked from the plate), and pB1a-tet
Start a Golden Gate reaction of the 7 inserts for the second plasmid
Digest pB1t with Spe1 and BamH1 (alone and together) to check for correct insert
Digest pS8k cysE2-hesB with Pme1 and Pst1 (alone and together) to check for insets
- Experiments From Jul 28
Send in sequencing primers
D_exp74
Dilute sequencing primers for ps8k cysE2-hesB
Sequence miniprep #2 of ps8k cysE2-hesB (clearest bands on the gel from digest check)
Sequencing PcR reactions
50 uL reactions using ps8k cysE2-hesB #2 as template
Amplify ~1 kb per primer pair, with each read overlapping by about 100 bp (the way the sequencing primers were designed)
- Experiments From Jul 29
Check Plasmid
D_exp75
Run a gel of a small amount of nifh to nifn golden gate mixture and all of the pB1t PCR
Saw no visible band for the Golden Gate mixture where expected for full ligation
Cut bands of pB1t PCR
Miniprep the overnight clulture of PA2C-TesA
Digest PA2C-TesA with EcoR1
Clean and Concentrate the digest, then start a klenow reaction
Resuspend newly-arrived sequencing primers for the next minimal nif plasmid, using TE buffer
- Experiments From Jul 30
Repeat Golden Gate
D_exp77
Try inserts in groups of 3 and 4 rather than all 7
nifH, nifD, nifK and nifB, hesA, nifE, nifN
- Experiments From Jul 31
Do Goldan Gate 2 at a time
D_exp78
Do Golden gate 2 inserts at a time, in pairs as follows (leaving out nifK from the groupings for now): nifH & nifK, nifB & hesA, nifE & nifN
- Experiments From Aug 01
Run PCRs to confirm
D_exp79
EN did not work. Try PCR with varied MgCl concentration and annealing temperatures.
Cut HD and BA from gel and gel purify
Start a Golden Gate reaction of HD with K
- Experiments From Aug 04
Attempt to digest then ligate separately
D_exp81
Digest
1.5 hour digestion
20 uL reaction
Bsa1, digest buffer, and 1 to 2 uL of insert to be digested
Clean and Concentrate
Ligate cleaned up pieces
T4 ligase
T4 ligase buffer
.5 uL ATP
10 uL reaction
- Experiments From Aug 05
More EN
D_exp83
Run PCRs of nifE and nifN ligation from separate digestion and ligation
Use E F and N R primers
Use 2 additional pairs of primers from the sequencing primers designed for this plasmid to amplify smaller portions of E and N that cross the junction, to check for appropriate ligation
Start overnight ligation of digested E and digested N at 4C
- Experiments From Aug 06
Check on ligation reactions
D_exp84
Ran gel of HDK BA, HDK BA E N, and overnight 4C E N ligations
Start PCRs from each, in addition to the room temperature EN ligation
Both amplifications of EN look good - cut and purify together
Neither HDKBA nor HDKBAEN worked
PCR more HDK from golden gate template
Start digest of BA and EN with Bsa1 for 2 step digestion then ligation
Jun_02
D_exp1
D_exp2
Jun_03
D_exp3
D_exp4
Jun_04
D_exp5
D_exp6
Jun_09
D_exp7
D_exp8
Jun_10
D_exp9
Jun_11
D_exp10
D_exp11
D_exp12
D_exp13
D_exp14
Jun_12
D_exp15
D_exp16
D_exp17
D_exp18
Jun_13
D_exp19
Jun_15
D_exp20
D_exp21
D_exp22
D_exp23
D_exp24
D_exp25
Jun_17
D_exp26
Jun_18
D_exp27
D_exp28
D_exp29
D_exp30
D_exp31
D_exp32
D_exp33
Jun_19
D_exp34
Jun_20
D_exp35
Jun_22
D_exp36
D_exp37
D_exp38
Jun_23
D_exp39
Jun_24
D_exp40
D_exp41
D_exp42
D_exp43
D_exp44
Jun_25
D_exp44
Jun_26
D_exp45
Jun_30
D_exp46
Jul_01
D_exp47
D_exp48
Jul_02
D_exp49
D_exp50
Jul_03
D_exp51
Jul_06
D_exp52
D_exp53
Jul_07
D_exp54
Jul_08
D_exp55
Jul_09
D_exp56
Jul_10
D_exp57
Jul_13
D_exp58
Jul_14
D_exp59
Jul_16
D_exp60
D_exp61
Jul_17
D_exp62
D_exp63
Jul_20
D_exp64
Jul_21
D_exp65
Jul_22
D_exp66
D_exp67
Jul_23
D_exp68
Jul_24
D_exp69
D_exp70
Jul_25
D_exp71
Jul_27
D_exp72
Jul_28
D_exp73
D_exp74
Jul_29
D_exp75
Jul_30
D_exp76
D_exp77
Jul_31
D_exp78
Aug_01
D_exp79
Aug_04
D_exp80
D_exp81
Aug_05
D_exp82
D_exp83
Aug_06
D_exp84
Laura Beebe         Pennsylvania State University
- Experiments From Jun 01
Find a method of calculating NAD/NADH
L_exp1
Searched literature for a method of calculating NAD/NADH reaction
found incorrect ratio according to literature.  Need new method to calculate this
- Experiments From Jun 02
Find Relatives of JM109 and WM1788
L_exp2
Literature/database search for relatives of JM109 and WM1788 to construct models.
Learned they are K-12 derivatives.
- Experiments From Jun 03
Researched K-12 model
L_exp3
Researched which reactions to remove from K-12 model in order to construct WM1788 and JM109 models using genotypes provided by Wash U
- Experiments From Jun 05
Checked ATP production
L_exp4
Made final WM1788 model, found max biomass and ATP production.  Made K-12 model from Monk paper, found max biomass and ATP production. Made additional corrections to DH10B model, found new max biomass and ATP.   Had lower biomass (growth) than K-12 derivatives.  Continued work on JM109 model with help from Jess.
found new max biomass and ATP.   Had lower biomass (growth) than K-12 derivatives.  Continued work on JM109 model with help from Jess.
- Experiments From Jun 08
Compared Carbon Sources
L_exp5
Compared strain growth and ATP production on various carbon sources.
Noticed high biomass/high ATP when using galactose as a carbon source.  Need a literature search to explore this finding.
- Experiments From Jun 09
Compared Carbon Sources
L_exp6
Continued comparing strains based on nutrient differences.  Started researching differences in E. coli metabolism for glucose vs. galactose.
- Experiments From Jun 10
Added N2 fixation to WM1788
L_exp7
Added N2 fixation reactions to WM1788 model, using flavodoxin as electron donor.  Created a set of nutrients for the N2 fixing strain (contains free nitrogen and lacks fixed nitrogen).  Found max biomass (2.52) and max ATP (391.3) for N2 fixing WM1788.
Normalizing wild type biomass indicates a difference in ATP production of 202 mmol ATP/(g cell dry weight x hr) between WT and N2 fixing cells.  This value makes sense according to nitrogenase flux range.
- Experiments From Jun 11
Ran FVA on WT WM1788
L_exp8
Ran FVA on WT WM1788 and N2 fixing WM1788.  Starting comparing the flux range overlaps between the two outputs using Excel.
- Experiments From Jun 12
Strange result
L_exp9
Observed CO2 and H2O export fixed for N2 fixing WM1788.  Started checking pathways (glycolysis, PPP, TCA cycle, pyruvate metabolism, reactions involving flavodoxin) to find out why.
- Experiments From Jun 15
Compared flux of N2 fixing of WM1788
L_exp10
Compared flux ranges of N2 fixing and wild type WM1788 within main metabolic pathways (glycolysis, PPP, TCA cycle, pyruvate metabolism) as well as within reactions that included oxidized or reduced flavodoxin.
Main result- pyruvate synthase (produces acetyl coA from pyruvate) in the N2 fixing strain has a higher flux than in wild type.  Additionally, many reactions within TCA cycle have a lower flux in the N2 fixing strain than in wild type (this is expected- N2 fixing WM1788 should have a depressed metabolism), except for malate dehydrogenase (produces oxaloacetate from malate).
- Experiments From Jun 16
Continued analysis of WM1788
L_exp11
Continued analysis of differences between wild type and N2 fixing WM1788.
Determined neither strain is using the glyoxylate shunt.  Performed comparisons of all reactions involving malate, oxaloacetate, acetyl coA, and pyruvate.  Malate oxidase (reaction at equilibrium) has a much more negative flux in N2 fixing WM1788 than in wild type.  Seems that oxaloacetate from malate dehydrogenase (higher flux in N2) feeds malate oxidase reaction to produce more malate and oxygen in the N2 fixing strain.
- Experiments From Jun 17
Exploring relationship between pyruvate and acetyl coA in N2 fixing strain versus wild type
L_exp12
Continued analysis of differences between wild type and N2 fixing WM1788.
Identified two other reactions containing both acetyl coA and pyruvate (besides pyruvate synthase), however both have the same flux in both N2 fixing and wild type.  The flux is 0-10,000, meaning they are cycling.  Jess explored deletion of these two reactions to see if removing them would help to push flux through pyruvate synthase and aid nitrogen fixation. Also looked at other ways to produce acetyl coA and other ways to spend pyruvate; however, no result yet for viable knockouts.
- Experiments From Jun 18
Exploring relationship between pyruvate and acetyl coA in N2 fixing strain versus wild type
L_exp13
Continued identifying reactions that involve either the production of acetyl coA or the expenditure of pyruvate.  Looked for other ways to make products in reactions that spend pyruvate, indicating the potential for a knockout.  No conclusive result.
- Experiments From Jun 19
Created a flavodoxin expenditure reaction
L_exp14
Created a flavodoxin expenditure reaction (analogous to ATP maintenance reaction) in order to determine maximum possible flavodoxin production in the cell.  Compared FVA outputs for N2 fixing WM1788 at fixed max biomass producing maximum flavodoxin and regular N2 fixing WM1788 at fixed max biomass.
Created a flavodoxin expenditure reaction (analogous to ATP maintenance reaction) in order to determine maximum possible flavodoxin production in the cell.  Compared FVA outputs for N2 fixing WM1788 at fixed max biomass producing maximum flavodoxin and regular N2 fixing WM1788 at fixed max biomass.
- Experiments From Jun 22
Made comparisons between N2 fixing WM1788 at fixed biomass and N2 fixing WM1788 with maximized flavodoxin at fixed biomass.
L_exp15
Made comparisons between N2 fixing WM1788 at fixed biomass and N2 fixing WM1788 with maximized flavodoxin at fixed biomass. Had to spend some time troubleshooting when the comparison didn’t yield values that we expected.
Did not notice any large differences between values at max flavodoxin and values without max flavodoxin-- ie. can conclude that maximizing flavodoxin as much as possible is already a way for the cell to maximize biomass.
- Experiments From Jun 23
Research malate dehydrogenase and malate oxidase
L_exp16
Based on information that malate dehydrogenase and malate oxidase reactions had a much higher flux in the N2 fixing cells than in wild type, did a database search to find information about ∆G values for these reactions in order to potentially change their directionality in the model.
Malate dehydrogenase directionality could not be changed.  After some searching, found malate oxidase reaction had been deleted from major E. coli databases in the past year.  Determined that the reaction should be removed from the model.
- Experiments From Jun 24
New maximum biomass and maximum ATP
L_exp17
Obtained new maximum biomass and maximum ATP production values for all strains after removing the malate oxidase reaction from the model.
Values were significantly smaller than before, but are more logically consistent with the model.  Strain DH10B now had the highest biomass and ATP production, but only by .0002 mmol/(g cell dry weight X hour).
- Experiments From Jun 25
Ran FVA and began comparing flux ranges
L_exp18
Ran FVA and began comparing flux ranges at maximum biomass for N2 fixing WM1788 and WT WM1788. Looked for differences in pathways between the N2 fixing and wild type strains.  Also ran FVA for N2 fixing WM1788 at fixed biomass with maximum flavodoxin production, and N2 fixing WM1788 at mixed biomass without maximum flavodoxin (did not find many differences in flux ranges, as before).
- Experiments From Jun 26
Compared flux ranges between N2 fixing and WT WM1788 at a fixed max biomass
L_exp19
Compared flux ranges between N2 fixing and WT WM1788 at a fixed max biomass, looking particularly for reactions whose directionality changes between the N2 fixing and wild type strains.  Assembled a list of these reactions, how much they changed by, and the metabolic systems they belong to.  Next week will need to perform the same comparisons with both strains set at maximum biomass, in order to eliminate wide flux variations in the wild type WM1788 for reactions that contribute directly to biomass.
- Experiments From Jun 29
Continuing flux range comparisons
L_exp20
Compared directionality of fluxes for N2 fixing and wild type WM1788 at maximum biomass.  This eliminates potentially large changes in reactions which are not a component of biomass.
Major result
- Experiments From Jun 30
Finish flux range comparisons
L_exp21
Finished up comparisons.  Looked through fluxes of transporter reactions in search of an explanation for ATP synthase directionality change in N2 fixing WM1788 vs. wild type.  Did not find any conclusive result.
- Experiments From Jul 02
Finished flux range comparisons
L_exp22
Did a flux range comparison of N2 fixing WM1788 at maximum ATP production vs normal N2 fixing WM1788.  Many reactions changed their directionalities, but the largest change was .02.  Can determine from this that the N2 fixing strain is already ATP limiting-- ie. there are no major differences when ATP is maximized.
- Experiments From Jul 06
Performed an analysis on all reactions with non-overlapping flux ranges between N2 fixing WM1788 and the wild type
L_exp23
Performed an analysis on all reactions with non-overlapping flux ranges between N2 fixing WM1788 and the wild type.  Determined which metabolic system each reaction was in and what percentage of the non-overlaps each system made up.
Amino acid metabolism was the system with the most non-overlaps, but it had the smallest changes.  Ion transport, exchange, and inner/outer membrane transport made up a little over 15% of the non-overlaps together, but had the highest changes.
- Experiments From Jul 07
Analyzed ATP production/consumption in N2 fixing WM1788.
L_exp24
Analyzed ATP production/consumption in N2 fixing WM1788.  Compiled a ranked list of reactions which produce ATP.  Jess compiled a list of the top 10 reactions in the model which consume ATP.
- Experiments From Jul 08
Started working on the Python script
L_exp25
Started working on the Python script for knocking out all of the genes in the N2 fixing model.  Started by writing a code which takes the GPRs (gene protein relationships) present in the model and creates a list of all the unique genes.
- Experiments From Jul 09
Continued working on the Python script
L_exp26
- Experiments From Jul 10
Continued working on the Python script
L_exp27
- Experiments From Jul 13
Continued working on the Python script.
L_exp28
At this point it can generate a list of reactions that will be deleted for each gene knockout.
- Experiments From Jul 14
Monsanto Slides
L_exp30
Made a rough draft of slides for the team’s presentation to Monsanto next week.
- Experiments From Jul 15
Continued working on Python script
L_exp31
Continued working on the Python script and accompanying GAMS code.  Modified the GAMS code so that it will output the values for biomass, ATP, and flavodoxin in one file, instead of requiring multiple GAMS codes for each value.  Continued working on the slides for Monsanto.
- Experiments From Jul 16
Continued working on the Python script
L_exp32
Fixed various errors in the GAMS code and continued working on the Python script.  Ran the full code for the first time. It gives output in correct formatting, but values are incorrect.  Finished the slides for Monsanto presentation and added notes.
- Experiments From Jul 17
Worked on debugging the final code.
L_exp33
- Experiments From Jul 20
Fixed all remaining errors
L_exp34
Fixed all remaining errors in the single gene knockout code, ran the code, and received the results, which are summarized as follows:
Single Gene Knockout Results:
For 8 genes, max flavodoxin was 2.07
All other genes
- Experiments From Jul 21
Report for Dr. Maranas and KO code
L_exp35
- Experiments From Jul 22
Continued working on the report and code.
L_exp36
- Experiments From Jul 23
Continued working on the report and code.
L_exp37
- Experiments From Jul 24
Finished both the report and code.
L_exp37
- Experiments From Jul 27
Worked on code
L_exp38
Tried running my gene double knockouts code but it timed out.Broke up the code into smaller pieces that will be run simultaneously.  Also started reading about OptKnock.
- Experiments From Jul 28
Worked on code
L_exp39
Started setting up and debugging the OptKnock code with Jess.  We want to try to couple flavodoxin production with biomass.
- Experiments From Jul 29
Worked on code
L_exp40
Ran OptKnock for double reaction knockouts and continued to debug.  Wrote a Python code that will determine if the OptKnock suggested knockouts couple flavodoxin with biomass or not. Jess ran an OptKnock code for triple reaction knockouts.
- Experiments From Jul 30
Worked on code
L_exp41
Wrote a new Python code to determine of the OptKnock suggested triple reaction knockouts couple flavodoxin production to biomass.
- Experiments From Jul 31
First OptKnock results
L_exp42
Analyzed OptKnock results; however, none couple flavodoxin to biomass.  Note that OptKnock suggests reaction, not gene knockouts.
- Experiments From Aug 03
Second OptKnock results
L_exp43
In my final week, I fixed the issues with my double gene knockout codes (so there are no longer discrepancies and the outputs are correct).  Unfortunately, we did not find any double knockouts that coupled flavodoxin production to biomass. Future research could look into why this is the case.  Additionally, Jess and I worked on getting travel grants and funding in order for our trip to the iGEM Giant Jamboree in September. compiled our wiki modeling summary and worked on making edits for it.
Jun_01
L_exp1
Jun_02
L_exp2
Jun_03
L_exp3
Jun_05
L_exp4
Jun_08
L_exp5
Jun_09
L_exp6
Jun_10
L_exp7
Jun_11
L_exp8
Jun_12
L_exp9
Jun_15
L_exp10
Jun_16
L_exp11
Jun_17
L_exp12
Jun_18
L_exp13
Jun_19
L_exp14
Jun_22
L_exp15
Jun_23
L_exp16
Jun_24
L_exp17
Jun_25
L_exp18
Jun_26
L_exp19
Jun_29
L_exp20
Jun_30
L_exp21
Jul_02
L_exp22
Jul_06
L_exp23
Jul_07
L_exp24
Jul_08
L_exp25
Jul_09
L_exp26
Jul_10
L_exp27
Jul_13
L_exp28
Jul_14
L_exp29
L_exp30
Jul_15
L_exp31
Jul_16
L_exp32
Jul_17
L_exp33
Jul_20
L_exp34
Jul_21
L_exp35
Jul_22
L_exp36
Jul_23
L_exp37
Jul_24
L_exp37
Jul_27
L_exp38
Jul_28
L_exp39
Jul_29
L_exp40
Jul_30
L_exp41
Jul_31
L_exp42
Aug_03
L_exp43
Report Bugs
Thanks for your help with out site!
×
This form will open your default mail client (i.e Outlook). If you do not have one installed, email us at  WashU iGEM washu.igem@gmail.com
Send Bug
Close Modal
Discover
Parts
Basic Parts
Parts Collection
Parts
Composite Part
Team
Members
Journals
Attributions
Project
Description
Design
Modeling
Measurment
Project
Policies and Practices
Human Practices
Practices
Safety
Misc.
Entrepreneurship
Software
Collaborations
Sponsors
Special Thanks to the:The Focht Family, The Bourg Family, The Heeney/Toomey Family, and NSF-MCB Award #1331194
Washington University in St. Louis and Pennsylvania State University iGEM 2015
