<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_WashU_StLouis_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:WashU StLouis/Design</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Washington University - Penn State iGEM</TITLE></P><HEADER class="row"><DIV class="container"><DIV class="intro-text"><DIV class="intro-lead-in">Welcome To Our Website!</DIV><DIV class="intro-heading"> WashU and Penn State   iGEM 2015</DIV><A href="https://2015.igem.org/Team:WashU_StLouis/description#description" class="page-scroll btn btn-xl">Project Description</A><A href="https://2015.igem.org/Team:WashU_StLouis#contact" class="page-scroll btn btn-xl">Let's Talk</A><A href="https://static.igem.org/mediawiki/2015/d/d2/2016_WashU_iGEM_Application.pdf" class="btn btn-xl">Apply for the 2016 iGEM Team!</A></DIV></DIV></HEADER><NAV class="navbar navbar-default navbar-fixed-top"><DIV class="container"><DIV class="navbar-header page-scroll"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand page-scroll" href="#page-top">Nitrogenius</A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A class="page-scroll" href="#selection">Selection</A></LI><LI><A class="page-scroll" href="#design">Design</A></LI><LI><A class="page-scroll" href="#cloning">Cloning</A></LI><LI><A href="mailto:ayekedavidr@wustl.edu?subject=Bug">Report Bugs</A></LI></UL></DIV></DIV></NAV><DIV class="container"><SECTION id="selection" class="bg-light-gray row sectionNum1"><H2>Selecting Genes</H2><P>In designing our minimal nif cluster plasmids, we considered each of the 23 genes on the PSL2397 nif plasmid, derived from Cyanotdece 51142, and selected 14 tdat we believe may be essential for nitrogenase activity. </P><P>We used literature on the creation of other minimal nif clusters and took into account a few factors: </P><UL><LI>function of the gene, if established </LI><LI>homology with genes in other minimal nif clusters</LI><LI>homology with genes in E. Coli’s genome</LI></UL><TABLE class="table"><TBODY><TR><TH>Gene</TH><TH>Function</TH><TH>Justification</TH></TR><TR><TD>nifH</TD><TD>Nitrogenase iron protein</TD><TD>Main structural component, can't have activity without all necessary structural components. Additionally, all literature looked at expressed nifH, nifD, nifK at high levels</TD></TR><TR><TD>nifD</TD><TD>Nitrogenase molybdenum-iron protein alpha chain</TD><TD>Main structural component, can't have activity without all necessary structural components. Additionally, all literature looked at expressed nifH, nifD, nifK at high levels</TD></TR><TR><TD>nifK</TD><TD>Nitrogenase molybdenum-iron protein beta chain</TD><TD>Main structural component, can't have activity without all necessary structural components. Additionally, all literature looked at expressed nifH, nifD, nifK at high levels</TD></TR><TR><TD>nifE</TD><TD>Nitrogenase Fe-Mo cofactor and assembly protein</TD><TD>Seems essential for enzymatic assembly. All literature read expressed nifE, nifN at variable levels.</TD></TR><TR><TD>nifN</TD><TD>Nitrogenase Fe-Mo cofactor and assembly protein</TD><TD>Seems essential for enzymatic assembly. All literature read expressed nifE, nifN at variable levels.</TD></TR><TR><TD>nifB</TD><TD>Homocitrate syntdase/FeMo-co core syntdesis</TD><TD>Voigt paper claims nifB is a key component in putting together the FeMo-co core. Nif B was expressed at high levels</TD></TR><TR><TD>nifU</TD><TD>Encodes proteins for early Fe-S formation; proteins for component maturation</TD><TD>Voigt expressed nifU at low levels. It could be a necessary component in formation of Fe-S clusters </TD></TR><TR><TD>nifS</TD><TD>Cysteine desulfurase; used in MoFe cluster syntdesis</TD><TD>Ludden speculates that cysteine desulfurase plays a key role in Fe-S cluster assembly</TD></TR><TR><TD>nifV</TD><TD>Homocitrate syntdase</TD><TD>Included by Voigt in his work, but expressed at low levels. Nitrogenase mutants which didn't contain homocitrate had problems with substrate association</TD></TR><TR><TD>nifW</TD><TD>Nitrogenase-stabilizing/protective protein</TD><TD>Also included by Voigt in his work, low expression. It could be a necessary component of nitrogen fixation</TD></TR><TR><TD>nifZ</TD><TD>Encodes proteins for Fe-S formation; proteins for component maturation</TD><TD>Also included by Voigt in his work, low expression. Helps maintain enzyme's structural integrity</TD></TR><TR><TD>hesA</TD><TD>Related to molybdenum cofactor; part of tdiF_MoeB_HesA protein family which are involved in molybdopterin and tdiamine biosyntdesis; E. coli has botd tdiF and MoeB</TD><TD>May have role in creating Molybdenum cofactor, a necessary component to the FeMo-co core. </TD></TR><TR><TD>hesB</TD><TD>Fe-S cluster biosyntdesis, necessary for enzyme syntdesis</TD><TD>Fe-S clusters are important components of the Fe protein and the FeMo protein. Closest match within E. coli had 49% identity. To ensure correct metallocluster synthesis, we need to use this gene.</TD></TR><TR><TD>cysE2</TD><TD>serine O-acetyltransferase</TD><TD>Serine O-acetyltransferases are important enzymes in cysteine metabolism. Since the formation of FeS clusters is predicated on cysteine reactions, presence of this enzyme may help increase synthetic efficiency</TD></TR></TBODY></TABLE><P>Our analysis left us with 14 genes to be included in the minimal nif plasmids. We also designed a 15-nucleotide ribosome binding site to insert directly upstream of each gene. </P><P>RBS sequences were designed by the Penn State Salis Lab's <A href="https://salislab.net/software/">RBS Calculator</A>.Each RBS was designed to have a Translation Initiation Rate close to 50,000 au (Arbitrary Units) as descrbibed in considerations 1.2a &amp; 1.2b of the Salis <A href="#" data-container="body" data-toggle="popover" data-placement="top" data-content="Howard, Salis. The ribosome binding site calculator.">paper</A>. </P></SECTION><SECTION id="design" class="bg-white row sectionNum2"><H2>Plasmid Design</H2><P>The literature also indicated that the expression levels of each gene need to be finely tuned in order to achieve optimal nitrogenase activity. We decided to use 2 different inducible promoters to give us more control over expression levels, and grouped the genes into ones we thought would need to be expressed at higher versus lower levels. </P><P>We decided to create 2 plasmids with 7 genes each rather than one plasmid with 14 genes, which would simplify cloning greatly. We designed one plasmid containing genes that we generally believed needed to be expressed at higher levels, including the enzyme structural genes. </P><P>In ordering the genes on the plasmids, we mimicked gene orders we observed in the natural Cyanothece cluster and in other minimal clusters in the literature. Since we did not have the resources to codon optimize, we also considered codon usage in placement of genes on the plasmids. One gene, cysE2, has particularly bad codon usage for E. Coli, so we placed it directly downstream of the promoter. </P></SECTION><SECTION id="cloning" class="bg-light-gray row sectionNum3"><H2>Primer Design and Cloning Strategy</H2><P>We designed primers for use with Golden Gate cloning to create the plasmids. Our general strategy for primer design was to add a BsaI site and one intervening base before the fifteen-base RBS before the annealing portion on the forward primer. When possible, we used the first four bases of the RBS as the overhang for ligation, rather than adding in bases and creating a scar. On the reverse primer, we added a BsaI site and one intervening base before adding the reverse complement of the overhang that it should match up with. In designing primers, we considered:</P><UL><LI>maintaining a primer length of 60 nucleotides or less</LI><LI>having a melting temperature (T m) of the primer-template interaction between about 58 and 62 degrees Celsius</LI><LI>matching the melting temperature of primers in a primer pair as closely as possible to each other</LI><LI>maintaining a hairpin melting temperature as low as possible, and always less than the T m of the primer-template interaction</LI><LI>maintaining heterodimer and homodimer interaction delta G values to be less in magnitude than -15 kcal/mol</LI><LI>adding up to 6 adenines before the BsaI site where possible</LI><LI>having overlap sequences that are distinct enough from other overlap sequences used for the same plasmid</LI><UL><LI>our criterion was to have a maximum of 2 of the same base in the same position. i.e. GACT and CAGT are acceptable as only A and T fall in the same positions. However, GACT and CACT are too similar and one must be changed to avoid improper ligation</LI></UL></UL><P>In order to balance all of these constraints, a couple of changes to the coding sequence of the genes were made; however, degenerate codons were used so that the amino acid sequence was not disrupted. </P></SECTION></DIV><FOOTER class="bg-white row"><DIV class="overview-modal modal fade" id="reportBugsModal" tabindex="-1" role="dialog" aria-hidden="true"><DIV class="modal-content"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><DIV class="modal-body"><SECTION id="report_bugs" class="bg-white row"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 text-center"><H2 class="section-heading">Report Bugs</H2><H3 class="section-subheading text-muted">Thanks for your help with out site!</H3><DIV class="alert alert-danger"><BUTTON type="button" class="close" data-dismiss="alert" aria-hidden="true">×</BUTTON><STRONG>This form will open your default mail client (i.e Outlook). If you do not have one installed, email us at  WashU iGEM washu.igem@gmail.com</STRONG></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><FORM name="sentMessage" id="reportForm" novalidate=""><DIV class="row"><DIV class="col-lg-12 text-center"><BUTTON type="submit" class="btn btn-xl">Send Bug</BUTTON></DIV></DIV></FORM></DIV></DIV></DIV></SECTION><BUTTON type="button" class="btn btn-primary" data-dismiss="modal"> Close Modal</BUTTON></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><SECTION class="row sectionNum5" id="sitemap"><DIV class="col-xs-12"><H2>Discover</H2><TABLE class="table table-responsive"><TBODY><TR><TD>Parts</TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Basic_Part">Basic Parts</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Part_Collection">Parts Collection</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Parts">Parts</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Composite_Part">Composite Part</A></TD></TR><TR><TD>Team</TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Team">Members</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Notebook">Journals</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Attributions">Attributions</A></TD></TR><TR><TD>Project</TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Description">Description</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Design">Design</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Modeling">Modeling</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Measurement">Measurement</A></TD></TR><TR><TD>Policies and Practices</TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Practices">Practices</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Safety">Safety</A></TD></TR><TR><TD>Misc.</TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Software">Software</A></TD><TD><A href="https://2015.igem.org/Team:WashU_StLouis/Collaborations">Collaborations</A></TD></TR></TBODY></TABLE></DIV></SECTION><SECTION id="sponsors" class="row  sectionNum6"><DIV class="container"><DIV class="row"><H2>Sponsors</H2></DIV><DIV class="row"><P>Special Thanks to ​the Center for Biological Systems Engineering, Department of Chemistry, Focht Family, Heeney/Toomey Family, Bourg Family, NSF-MCB Award #1331194</P></DIV></DIV></SECTION><DIV class="row"><DIV class="col-md-4 col-md-offset-4"><SPAN class="copyright"> Washington University in St. Louis and Pennsylvania State University iGEM 2015</SPAN></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>