Team:WashU StLouis
Washington University - Penn State iGEM
Welcome To Our Website!
WashU and Penn State   iGEM 2015
Project Description Let's Talk Apply for the 2016 iGEM Team!
Toggle navigation
Nitrogen
Overview
Team
Collaboration
Sponsors
Contact
Report Bugs
More
Click here to apply for the 2016 iGEM Team!
Project Overview
Washington University and Penn State iGEM
Last Year
Learn from history
Mission Statement
The goal in a sentence
Nitrogen Project
Ain't broke, but we got to fix it
The Big Picture
Why did we do this?
Materials and Methods
Imitation is the sincerest form of flattery.
Discussion
Explaining the results
Team
NitroGeniuses
Jessica O'Callaghan
Chemical Engineering
Penn State
Laura Beebe
Biochemistry and Molecular Biology
Penn State
Blake Actkinson
Biomedical Engineering
Washington University in St. Louis
Charlotte Bourg
Biochemistry and Spanish
Washington University in St. Louis
David Ayeke
Computer Engineering
Washington University in St. Louis
Mike Toomey
Biomedical Engineering
Washington University in St. Louis
Caroline Focht
- 2014 iGEM Team Member
Chemistry and Biochemistry
Washington University in St. Louis
Be Part
Of Our
Story!
Collaboration
Collaboration
Looking for Collaborators?
Vanderbilt
Our team worked closely with Vanderbilt University’s iGEM team over the summer to help them troubleshoot issues they had been having regarding a promoter and to come up with questions to ask our panel. Read more here
Help Us Out?
Washington University in St. Louis genetic engineering panel                   Hey iGEM teams!                   Here at Wash U we’re planning a panel on genetic engineering to be held in the fall for the Wash U community. We’re hoping to address some gray areas of synthetic biology and genetic engineering in agriculture, from safety and regulation to ethics and implications. We’re still finalizing our panelists, but hope to have experts from Monsanto and the Danforth Plant Science Center. We’re also in touch with some Wash U professors from the biology, philosophy, and anthropology departments.                   We plan to start off the discussion with our own questions before opening it up to the audience. We’ve come up with a list of questions, but would love to get the input of some other iGEM teams. If you have questions you think would start interesting discussion at the panel, we’d love to hear them! Email us at
washu.igem@gmail.com
.                   Best, Wash U iGEM
Our Amazing P.I's
The people that made it all possible
Tae Seok Moon
Faculty Advisor - Washington University
Fuzhong Zhang
Faculty Advisor - Washington University
Costas Maranas
Faculty Advisor - Pennsylvania State University
Cheryl Immethun
Washington University iGEM Advisor
Thomas Mueller
Penn State iGEM Advisor
Andrea Balassy
Washington University iGEM Advisor
Carlos Barba
Washington University iGEM Advisor
Yi Xiao
Washington University iGEM Advisor
Ray Henson
Washington University iGEM Advisor
Young Je Lee
Washington University iGEM Advisor
Contact Us
We can't do this alone
×
This form will open your default mail client (i.e Outlook). If you do not have one installed, email us at  WashU iGEM washu.igem@gmail.com
Send Message
2014 iGEM Project
Fixing Nitrogen and Sensing Light
What was our goal?
There were two key components of our project:
Richard and Caroline were working in the Pakrasi lab, and had been using genes from cyanobacteria to get nitrogen fixation working in E. coli. They were testing nitrogen fixation by running acetylene reduction assays and designing experiments to test the optimal criteria (ie. E. coli strains, temperature, nitrogen source) to get maximal results.
Read more here
Ben and Jeffery worked in the Moon lab, and cloned plasmids to create a system that represses and expresses a fluorescent protein with the presence and absence of light, and were running experiments to test the induction levels compared to various positive and negative controls.
Read more here
Close Modal
Mission Statement
tl;dr Fix Nitrogen
Caroline Focht
- 2014 Washington University iGEM member
More Videos!!!
The goal of this year's project is to take the nitrogen fixing nif cluster from Cyanothece, a cyanobacterium, minify it, and make it work in E. coli.
Designing the plasmid TODO ADD LINkS
Making it work in E. Coli TODO ADD LINKS
Close Modal
Summary of the project to date
Phase 1: Electro-Transformation of plasmid pNif51142 into E. Coli strains
- 2014
To successfully transform pNif51142, we used Electro-Transformation, also known as Electroporation.  Electroporation provides a method of transforming E. coli to efficiencies greater than are possible with the best chemical methods. By subjecting mixtures of cells and DNA to exponentially decaying fields of very high initial amplitude, we were able to deliver the plasmid into all of E. coli strains that were tested in the project.
Results
According to the gel running and antibiotic testing, bands in the gel and the survival of all strains transformed in antibiotic Kanamycin (there was an cluster of Kanamycin-resistant marker gene in sequence of pNif51142) both prove that the Electro-Transformation was successful.
Phase 2: Determine the optimal conditions for cell survival with plasmid pNif51142 in E. Coli
- 2014
Due to the oxidative properties of oxygen, most nitrogenases are irreversibly inhibited by dioxygen, which degradatively oxidizes the Fe-S cofactors. This requires mechanisms for nitrogen fixers to protect nitrogenase from oxygen in vivo. Hence in our experiment, we firstly selected anaerobic condition as part of preparation step for the nitrogenase activity testing
Testing Media: Minimal M9
Carbon Source: Glucose(1mM, 10mM, 100mM)
Nitrogen Source: Glutamine(1mM, 10mM, 100mM), Glutamate(1mM, 10mM, 100mM), NH4Cl(1mM, 10mM, 100mM)
Temperature: 30°C, 37°C, 40°C
pH:6,7,8
O2 level:Anaerobic or Aerobic
Strains of e. Coli: JM109, BL21(DE3), WM1788, Top 10 DH5Α
Target Strain
JM109
BL21(DE3)
WM1788
Top 10
DH5Α
Experimental Plates
JM 109 strain w/ plasmid
Antibiotic
BL21(DE3) strain w/ plasmid
Antibiotic
WM1788 strain w/ plasmid
Antibiotic
Top 10 strain w/ plasmid
Antibiotic
DH5α strain w/ plasmid
Antibiotic
Positive Control
JM 109 strain w/o plasmid
No antibiotic
BL21(DE3) strain w/o plasmid
No antibiotic
WM1788 strain w/o plasmid
No antibiotic
Top 10 straing w/o plasmid
No Antibiotic
DH5α strain w/o plasmid
No antibiotic
Negative Control
JM 109 strain w/o plasmid
Antibiotic
BL21(DE3) strain w/o                                       plasmid
Antibiotic
WM1788 strain w/o plasmid
Antibiotic
Top 10 strain w/o plasmid
Antibiotic
DH5α strain w/out plasmid
Antibiotic
Results
In the minimal M9 media, all possible combinations of parameters listed above were tested.                                 None of the concentrations of glucose had any aﬀect on the growth of E. coli. It was expected that as the concentration of glucose increased, the growth of E. coli also increased. However, the variation between the concentrations of glucose may have been too small for a noticeable increase in E. coli growth as the concentration of glucose increased. Also, even the maximum concentration of glucose tested, 100mM, may have been too low to aﬀect the growth of E. coli to an observable extent. Eventually, 10mM was determined to be the optimal glucose concentration for the purpose of least interference possible in solution.                                 NH4Cl as minimal nitrogen source was proven to be not suitable for E. coli growth at any concentration as the OD600 testing results showed that cell density didn’t change throughout the time. It was probably due to the permeability of cell membrane was limited for NH4+ and Cl- ions. Eventually Glutamate at concentration of 10mM supported cell growth the best and thus chosen as part of optimal environment condition.                                  With CASAmino solution’s buffering utility, the pH was controlled little bit below 7 but close to 7.                                  To protect the iron core of the nitrogenase, temperature of 30°C and Anaerobic were both determined not for cell growth but for nitrogenase activity testing, which is the next phase.
Phase 3: Measurign nitrogen fixation activity under determined optimal conditions in E. Coli Strains.
-2014
We used  an Acetylene Reduction Assay to examine the nitrogenase activity for JM109, BL21(DE3), Top10, DH5α at different cell density referred by OD600 values.                                  Acetylene (C2H2) has an triple bond similar to that of atmospheric nitrogen (N2). Because of this structural similarity, the nitrogenase enzyme can cleave the triple bond in acetylene just as it would cleave the triple bond in N2. Ethylene (C2H4) is produced from this enzymatic activity, so a gas chromatograph can be used to detect the presence of ethylene and, consequently, nitrogenase activity.
Results
Of the five E. coli strains tested, JM109 and WM1788 showed strongest nitrogenase activity.
The linear relationship between nitrogen fixation activity and time matches that seen in nature.
Optimal conditions determined:
Glucose as carbon-source
glutamate as nitrogen source
LB as inoculating media
minimal M9 as testing media for GC assay
anaerobic enviornment
30°C during the overnight preperation
Phase 4/5 Computationally Modelling an E. Coli system that has Nitrogen Fixation (Penn) and Targeting Nif genes for Overexpression and Silencing (WashU)
-2015
IGEM 2015 Project Description.
While last year’s team was able to take the nif cluster from a cyanobacteria and get it to function in E. coli, nitrogenase activity was minimal.  This year’s team is dedicated to increasing the nitrogenase activity of E. coli with the nif cluster from Cyanothece 51142. The team members at Pennsylvania State will use high level computational modeling to optimize metabolic pathways within E. Coli, which will relieve the strain of nitrogen fixation. The team members at Washington University will create a minimal nif cluster of only those genes necessary for nitrogenase production and activation. In addition, they will silence and overexpress several genes in order to maximize nitrogenase activity.
In doing such, we hope to create a system for nitrogen fixation for transformation into a photosynthetic system. After we come to a greater understanding of how the system works and perfect it, we can move on to working in a more complex organism, such as a cyanobacteria like Synecosystisspp. 6803.
The end goal is to create plants that can fix their own nitrogen by moving from the cyanobacteria into the chloroplast of the plant. Endosymbiotic theory postulates that cyanobacteria are the ancestors to chloroplasts, so this is the natural progression.
Results
Washington University was able to design and synthesize a new more minimal nif cluster optimize for Nitrogenase activity within E. Coli. Read more on our Design page.
Penn State was able to compile a list of gene knockouts and metabolic addatives that could promote Nitrogenase activity within E. Coli. Read more on our Modeling page.
Close Project
The Big Picture
Haber-Bosch is so 1909
Why are we doing this project
Synthetic biology is an exciting area of research that aims to genetically improve organisms to make them more efficient and hopefully more useful to us as well. The human population in 1950 was 2.5 billion, yet it is predicted to surpass 9 billion by 2050 [2] ". Even with population growth slowing, increasing life spans and standards of living will soon tax our natural resources. One of the most concerning is our food supply. The agriculture industry needs a revolution in order to keep up with our expected growth rates. Currently about 80% of chemically fixated nitrogen is used as agricultural fertilizers, the majority in developed lands [1] . Intracellular nitrogen fixation in crops could help to sustain the burgeoning world population, especially in areas with less fertile soil without taxing the planet’s waterways. The exponential increase in nitrogen fertilizer has led to more runoff into rivers and oceans. Fertilizers then provide nutrition for algal blooms that result in hypoxia and form oceanic dead zones. These dead zones lead to the death of marine species and have potentially large economic consequences.                                  The ramifications of nitrogen fertilizer runoff can be averted by genetically engineering plant crops to fix their own nitrogen. Some cyanobacteria fix nitrogen for nutritional needs, while most organisms can only acquire it from the food it consumes. Synthetic biology allows us to transfer this ability to fix nitrogen to a heterologous host that has many genetic tools, Escherichia coli, so that we can learn how to give single cell organisms, and eventually chloroplasts the ability to create their own nitrogen fertilizer.                                  Diazotrophic (organisms that fix nitrogen) cyanobacteria such Nostoc Punctiforme or Anabaena use heterocysts (specialized nitrogen fixing cells) to create a mini-anaerobic environment to aid nitrogen fixation. However, Cyanothece 51142, a non-heterocyst, fixes nitrogen in the same cell as photosynthesis by relying on a circadian metabolic process, when there is less oxygen byproduct from photosynthesis. This process is both fascinating and necessary since the key enzyme in nitrogen fixation, nitrogenase, is poisoned by oxygen. Our goal this summer is to engineer the regulation of the proteins necessary for nitrogen fixation so that they are highly repressed when activated by broad spectrum light (such as the sun), and are highly active when there is no light around, mimicking the cycle where photosynthesis occurs during the day and nitrogen fixation occurs at night.
Close Modal
Materials and Methods
How we did what we did
WashU iGEM Induction Protocol
-Pipette 4 mL sterile LB/CM into four 14 mL culture tubes, R0010-RFP (in BL21 and MG1655) and K314100-RFP (in BL21 and MG1655). Also pipette 4 mL sterile                                   LB to two 14 mL cultures tubes, wild type BL21 and MG1655
-Place the cap on the culture tubes to the first position to allow aeration.
-Place the tubes into the incubator and allow them to grow overnight (16 hours) at 37 deg C and 250 RPM
Next day...
-Make frozen stock for all cultures grown
-Measure OD at 600 nm in cuvettes for each culture (OD O/N) (900 uL water and 100 uL culture well mixed)
-Dilute cultures to OD of 0.1 in a total of 4 mL fresh LB/antibiotic (just LB for the BL21 and MG1655) in new 14 mL culture tubes
-Volume of overnight culture = (0.1*4)/(OD overnight)
-Volume of LB/antibiotic (just LB if negative control) = 4mL - V overnight
-Incubate 2 hours at 37 deg C and 250 rpm
-While waiting for the incubation to complete, label eighteen 15 mL conicals, cm-1 through cm-9 and wt-1 through wt-9 to create 5x dilutions.
-Add 10 mL LB and 10 μL cm to conical cm-9.
-Add 8 mL LB and 8 μL cm to conicals cm-1 through cm-8.
-Add 10 mL LB to conical wt-9.
-Add 8 mL LB to conicals wt-1 through wt-8.
-Calculate volume to take from the 0.5 M IPTG stock to achieve the max [IPTG] with the following equation.
-Vstock = (10 mL total in tube)*(2.5mM IPTG)*(0.6 mL total in deep well plate)/((500 mM IPTG)*(0.5 mL inducer/LB/ antibiotic in deep well plate)
-Add 60 μL of 500 mM IPTG to conicals cm-9 and wt-9.
-Vortex both conicals.
-Transfer 2 mL of conical cm-9 to conical cm-8, and transfer 2 mL of conical wt-9 to conical wt-8.
-Vortex conical cm-8 and wt-8.
-Transfer 2 mL of conical cm-8 to conical cm-7, and transfer 2 mL of conical wt-8 to conical wt-7.
-Vortex conical cm-7 and wt-7.
-Repeat for conical cm-6 through cm-2 and conical wt-6 through wt-2.
-Do NOT transfer anything into conical cm-1 or conical wt-1.
-Pour each conical (cm-1 through cm-9 and wt-1 through wt-9) into its own trough – one at a time. With the multi-channel pipette, transfer 500 μL per well                                   into the deep well plates.
-After the diluted cultures have grown 2 hours, pour each into its own trough. Using the multi-channel pipette, transfer 100 μL per well into the correct                                   rows.
-Cover the deep well plates with breathable membranes.
-Incubate 6-8 hours at 37°C and 250 rpm.
To measure...
-Centrifuge at 3000 g for 15 minutes. Remember to balance with another plate.
-Discard supernatant.
-Resuspend all cells in 200 μL 1x PBS.
-Transfer 200 μL of wells to a black 96-well plate.
-Measure with a Tecan(get proper product name) set to 580 nm excitation and 610 nm emission for mRFP, absorbance at 600 nm and the gain set to 100.
-Divide fluorescence by absorbance per well.
-Subtract average fluorescence/absorbance of corresponding wild type strain from the fluorescence/absorbance of the strains with the fluorescent protein.
-Calculate average fluorescence/absorbance and standard deviation based on the entire population for each strain at each inducer concentration.
-Plot average fluorescence/absorbance versus inducer concentration for each strain.
Vanderbilt Induction Protocol
-Pipette 4 mL sterile LB/CM into four 14 mL culture tubes, R0010-RFP (in BL21 and MG1655) and K314100-RFP (in BL21 and MG1655). Also pipette 4 mL sterile                                   LB to two 14 mL cultures tubes, wild type BL21 and MG1655
-Place the cap on the culture tubes to the first position to allow aeration.
-Place the tubes into the incubator and allow them to grow overnight (16 hours) at 37 deg C and 250 RPM
Next day...
-Make frozen stock for all cultures grown
-Measure OD at 600 nm in cuvettes for each culture (OD O/N) (900 uL water and 100 uL culture well mixed)
-Label eighteen 15 mL conicals, cm-1 through cm-9 and wt-1 through wt-9 to create 5x dilutions
-Add 10 mL LB and 10 uL cm to conical cm-9
-Add 8 mL LB and 8 uL cm to conicals cm-1 to cm-8
-Add 10 mL LB to conical wt-9.
-Add 8 mL LB to conicals wt-1 through wt-8.
-Add 50 μL of 500 mM IPTG to conicals cm-9 and wt-9 (conical concentration will be 2.5 mM IPTG)
-Vortex both conicals.
-Transfer 2 mL of conical cm-9 to conical cm-8, and transfer 2 mL of conical wt-9 to conical wt-8.
-Vortex conical cm-8 and wt-8.
-Transfer 2 mL of conical cm-8 to conical cm-7, and transfer 2 mL of conical wt-8 to conical wt-7.
-Vortex conical cm-7 and wt-7.
-Repeat for conical cm-6 through cm-2 and conical wt-6 through wt-2.
-Do NOT transfer anything into conical cm-1 or conical wt-1.
-Pour each conical (cm-1 through cm-9 and wt-1 through wt-9) into its own trough – one at a time. With a multi-channel pipette, transfer 600 μL per well                                   into the deep well plates.
-Pour each overnight culture into its own trough. Using the low volume multi-channel pipette, transfer 0.6 μL per well into the correct rows.
-Cover the deep well plates with breathable membranes.
-Incubate 24 hours at 37°C and 250 rpm.
To measure...
-Centrifuge at 3000 g for 15 minutes. Remember to balance with another plate.
-Discard supernatant.
-Resuspend all cells in 200 μL 1x PBS.
-Transfer 200 μL of wells to a black 96-well plate.
-Measure with a Tecan(get proper product name) set to 580 nm excitation and 610 nm emission for mRFP, absorbance at 600 nm and the gain set to 100.
-Divide fluorescence by absorbance per well.
-Subtract average fluorescence/absorbance of corresponding wild type strain from the fluorescence/absorbance of the strains with the fluorescent protein.
-Calculate average fluorescence/absorbance and standard deviation based on the entire population for each strain at each inducer concentration.
-Plot average fluorescence/absorbance versus inducer concentration for each strain.
Making LB Plates
-12.5 g Miller LB (25g/Liter of solution)
-7.5 g Agar (15g/Liter of solution)
-Put magnetic stir bar into bottle
-Pour UV deionized water, 500 mL, into bottle (25g/L miller LB, 15 g/L Agar)
-Take the bottle to an autoclave
-Autoclave bottle on agar cycle, loose cap, for two hours
Electrocompetent Cell Prep
-2.50 mL of cultures of DH10B were prepared beforehand
-0.25 ml of each culture were transferred into 500 mL of LB
-These flasks were then grown at 37 deg C, 250 rpm for 1 hour, 30 minutes in incubator
i. Start time: 11:05 A.M
ii. End time: 12:35 P.M.
-Absorbances
i. Cuv1 – 0.1055
ii. Cuv2 – 0.1084
-After absorbances were measured, cultures were added to ice bucket and allowed to cool for 15 minutes.
-Centrifuge bottles were centrifuged for 25 min. at 4 deg. C
-After centrifuge, bottles were taken to cold room
i. Chilled 10% glycerol was added, 55 ml each, into each bottle
ii. 70 mL of water were added to each bottle
(NOTE: % glycerol in each bottle is roughly 4.4%)
-Bottles were placed back into centrifuge for another 25 minutes at 4 deg C
-After centrifuge, bottles were taken to cold room once more
-Supernatant was drained
ii. 10% glycerol was added to 4 tubes
iii. Roughly 15 mL were allotted to each tube
iv. Tubes were taken and centrifuged
-After centrifuge, conical tubes were taken to cold room
i. Supernatant was drained
ii. 1 mL of 10% glycerol was added to each tube, pellet allowed to resuspend
iii. Tubes were taken to centrifuge again, 3000 RPM @ 4deg C
Transformation: Electroporation
-Obtain plasmid DNA
i. Add 2.5 uL of mini-prepped DNA to aliquot containing electrocompetent cells
ii. Else, add 1.0 uL if DNA is suspended in buffer solution
-Pipet entirety of electrocompetent cell mixture into electro cuvette
-Shock cell/DNA mixture using an Eporator
-Take out strain and pour into 100 uL of LB to be grown for 1 hour
-After 1 hour of growth, plate strain onto LB/Agar/Antibiotic
Frozen Stock Preparation
-Mix 500 uL of cell culture with 500 uL of 30% glycerol
-Store in -80 deg C freezer
Plasmid Miniprep
-Take bacterial culture and centrifuge it
-Discard LB growing solution that is supernatant
-Resuspend cells in water and add lyze buffer
-Immediately after lyze buffer addition, add cold neutralization buffer
-Centrifuge tubes at 16000 RPM for 4 minutes
-Transfer resulting supernatant to a separation column
-Centrifuge columns for 30 seconds at 16000 RPM
-Discard flow-through and place spin columns back into collecting tubes
-Add 200 uL of Endo-Wash Buffer to each tube
-Centrifuge tubes at 16000 RPM for 30 seconds
-After centrifugation, add 400 uL of Zyppy’s Wash Buffer to each tube
-Centrifuge tubes for 2 minutes at 16000 RPM
-After centrifugation is complete, add high pure molecular water to each column and let each column sit for 2 minutes
-Centrifuge columns at 16000 RPM for 2 minutes
-Take the elution and determine its concentration/purity
Polymerase Chain Reaction
Components for a 50 uL reaction, per reaction:
-4 uL of DMSO
-10 uL of buffer
-1 uL of dNTPs
-2.5 uL of forward primer
-2.5 uL of reverse primer
-1 uL of template DNA
-0.5 uL of phusion polymerase
-28.5 uL of cloning water
**Need time of PCR components
Making an Agarose Gel
-Measure 1 g of Agarose
-Mix agarose with 100 mL of buffer (100 mL of buffer/g of Agarose)
-Microwave gel with agarose for 1:30 and then let cool to room temperature
-Add 5 uL of cybersafe (5uL/100 mL of buffer) once the mixture cools
-Put in combs, dependent on how many lanes you want to construct
-Pour agarose gel mixture into tray
-Let gel cool for 1 hour
-After 1 hour, add 10 uL of 6x loading dye to each sample
-Add buffer until the top of the hardened gel plate was covered
-Insert ladder
i. Make sure you have ladder for each row of running lanes
-Then, insert DNA/dye mixture into each well
-Let DNA run down the gel at 125V for 30 min
i. Make sure there are bubbles forming on the negatively charged end
Colony PCR
-Obtain necessary amounts of 50 uL of PCR tubes
-Add 50 uL of sterile water to each
-Prepare a 1 mL/uL LB:AMP mixture with 5 mL of mixture for every reaction
-Pick 3 colonies from plates containing colonies and put 1 into each PCR tube
-Then, pipet 5 mL of LB/AMP mixture into corresponding culture tubes and label them with respect to each colony picked
-Drop picked colony into its corresponding, labeled culture tube and set aside
-Components for a 50 uL reaction, per reaction:
i. 22.5 uL of cloning water
ii. 4 uL of DMSO
iii. 1 uL of dNTPs
iv. 0.5 uL of Phusion Polymerase
v. 10 uL of buffer
vi. 7 uL of template
vii. 2.5 uL forward primer
viii. 2.5 uL reverse primer
-Pipet culture tube contents into corresponding labeled PCR tubes
**Need PCR protocols
Dpn1 Digest
-Obtain Dpn1 enzyme, 10x CutSmart Buffer, and clean DNA
Components for a 50 uL reaction, per reaction:
i. 40 uL cleaned DNA
ii. 5 uL of 10x CutSmart Buffer
iii. 5 uL of Dpn1 enzyme
-Add buffer before enzyme
-Let sit in 37 deg C incubator for 2 hours
Ligation Reaction
-Obtain T4 Ligase Buffer, DNA to be ligated, P4 Kinase, and T4 Ligase
Components for a 50 uL reaction, per reaction:
i. 1 uL of T4 Ligase Buffer
ii. 8 uL of DNA to be ligated
iii. 0.2 uL of P4 Kinase
iv. 1 uL of T4 Ligase
-Let each reaction sit at room temperature for 1 hour
DNA Clean-Up
-Load each sample to be cleaned with 250 uL of DNA binding buffer
-Vortex
-Pour tube contents into spin columns
-Centrifuge each tube for 30 seconds at 16000 RPM
-Discard flow-through in a clean beaker and set aside
-Add 200 uL of DNA Wash Buffer to each spin column
-Spin tubes for 30 seconds at 16000 RPM
-Add 200 uL more of DNA Wash Buffer
-Centrifuge tubes for 2:30 at 16000 RPM
-Discard flow-through in beaker
-Add 30 uL of high pure molecular water to each tube
-Let tubes sit on counter 2 mins.
-After waiting period is up, centrifuge tubes for 2 minutes at 16000 RPM
-Measure DNA concentrations
DNA Gel Extraction
-Take each gel, add 600 uL of ADB to each
-Put into heat block at 55 deg C and let melt, vortexing every 5 minutes
-After melting is complete, let each sample sit in heat block for another five minutes
-Take each sample out, our contents into a spin column
-Spin each down 1:00 min at 16000 rpm
-Take each out, discard flow-through in fresh, clean beaker
-Add 200 uL of DNA Wash Buffer to each tube
-Spin for 30 s at 16000 rpm
-Discard flow-through from each collecting tube
-Add 200 uL of DNA Wash Buffer to each tube
-Spin for 30s at 16000 rpm
-Take each out and discard the flow-through
-Do a dry run for 2 minutes at 16000 rpm
-Take out each sample and discard its flow through
-Elute each spin column with 40 uL of high pure molecular water
-Let each sample sit for 4 minutes
-Spin down each for another 4 minutes
-Gel recovery is complete
Close Modal
Discussion
Where do we go from here
XKCD
The overarching goal behind our project is to introduce nitrogen fixation within plants. Through the introduction of this technology, the need for fertilizers would be greatly diminished, which would provide many positive environmental impacts and cut costs.
Most of our fertilizer today is produced through the Haber Process. The Haber Process is widely believed to have triggered a population explosion, fueling the growth of human population around the world from 1.6 million to 7 million. [1] However, there is evidence for detrimental effects surrounding the use of nitrogen compounds. Near areas of heavy industrialization, the development of dead zones have been observed. [2] Eutrophication results in the development of harmful algal blooms, triggering hypoxia and hurting the delicate balance of aquatic ecosystems.
Through our project, we hope to undercut the production of fertilizer and reduce its harmful environmental impact. Through having plants synthesize their own nitrogen compounds we eliminate the need for that artificial nitrogen source. By working in E. coli, it is the team’s hope to develop tools for experimentation with nitrogen fixation in other organisms. E. coli holds the distinct advantage of having rapid growth cycles, enabling quick experimentation with different biological technologies.
References
http://www.nature.com/nature/journal/v400/n6743/full/400415a0.html
http://www.sciencedirect.com/science/article/pii/S1568988308001078
Close Modal
Report Bugs
Thanks for your help with out site!
×
This form will open your default mail client (i.e Outlook). If you do not have one installed, email us at  WashU iGEM washu.igem@gmail.com
Send Bug
Close Modal
Discover
Parts
Basic Parts
Parts Collection
Parts
Composite Part
Team
Members
Journals
Attributions
Project
Description
Design
Modeling
Measurement
Policies and Practices
Practices
Safety
Misc.
Software
Collaborations
Sponsors
Special Thanks to ​the Center for Biological Systems Engineering, Department of Chemistry, Focht Family, Heeney/Toomey Family, Bourg Family, NSF-MCB Award #1331194
Washington University in St. Louis and Pennsylvania State University iGEM 2015
