<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Caltech_Project_Population_Variation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Caltech/Project/Population_Variation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Caltech/Project/Population_Variation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Project/Population_Variation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Project/Population_Variation&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Caltech/Project/Population_Variation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Caltech/Project/Population Variation</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE><TBODY><TR valign="top"><TD><DIV style="position: relative; width: 960px; height: 36px; overflow: hidden;"><DIV style="position: absolute; top: 0; left: 0; font-size: 1000pt; width: 960px; height: 36px; overflow: hidden; line-height: 36px; z-index: 3;"><A href="/Team:Caltech" title="Team:Caltech"> </A></DIV></DIV></TD></TR></TBODY></TABLE><DIV style="font-size:24pt;"><FONT face="verdana" style="color:#BB4400"><CENTER>iGEM 2008
</CENTER></FONT></DIV><TABLE><TBODY><TR valign="top"><TD><DIV style="position: relative; width: 960px; height: 48px; overflow: hidden;"><DIV style="position: absolute; top: 0; left: 0; font-size: 1000pt; width: 960px; height: 48px; overflow: hidden; line-height: 48px; z-index: 3;"><A href="/Team:Caltech" title="Team:Caltech"> </A></DIV></DIV></TD></TR></TBODY></TABLE><DIV><TABLE cellspacing="0"><TBODY><TR><TD style="background-color: #ffffff" width="75px" valign="top"><CENTER><A href="/Team:Caltech" title="Team:Caltech"><FONT face="verdana" style="color:#BB4400"><B>Home</B></FONT></A><P><A href="/Team:Caltech/Members" title="Team:Caltech/Members"><FONT face="verdana" style="color:#BB4400"><B>People</B></FONT></A><A href="/Team:Caltech/Project" title="Team:Caltech/Project"><FONT face="verdana" style="color:#BB4400"><B>Project Details</B></FONT></A><A href="/Team:Caltech/Protocols" title="Team:Caltech/Protocols"><FONT face="verdana" style="color:#BB4400"><B>Protocols</B></FONT></A><A href="/Team:Caltech/Parts" title="Team:Caltech/Parts"><FONT face="verdana" style="color:#BB4400"><B>Completed Systems</B></FONT></A><A href="/Team:Caltech/Biosafety" title="Team:Caltech/Biosafety"><FONT face="verdana" style="color:#BB4400"><B>Biosafety</B></FONT></A><A href="/Team:Caltech/Acknowledgments" title="Team:Caltech/Acknowledgments"><FONT face="verdana" style="color:#BB4400"><B>Support</B></FONT></A></P></CENTER></TD><TD width="880" valign="top" style="padding: 10px; border: 5px solid #FFFFFF; color: #000; background-color: white"><DIV style="font-size:18pt;line-height:1.1"><FONT face="verdana" style="color:#CC3300">Population Variation:</FONT></DIV><DIV><DIV style="font-size:14pt;line-height:1.1"><FONT face="verdana" style="color:#CC3300">Construction of an Asynchronous Random State Generator</FONT></DIV><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>Biological circuits are commonly introduced into cells to produce non-native functions. Such functions include the production of antimalarial drugs in yeast cells<SUP>1</SUP>, the detection of light by bacterial cells<SUP>2</SUP>, and the enzymatic breakdown of bacterial biofilms by bacteriophages<SUP>3</SUP>. However, a problem arises when dealing with functions that inhibit the reproduction of cells. For example, the overproduction of certain compounds may burden cells with such a heavy load that they grow at a slower rate. Such production may involve lysing the cell to release compounds into solution. Once a cell is lysed, it can no longer produce progeny. If all the cells in a colony lyse, then the cell line may eventually die. Our solution is to introduce cellular differentiation. Two states can be created in a cell: one in which the cell simply grows and replicates, and another in which the fatal function is turned on. Cells in the former state can produce progeny, and the progeny that they produce can either be in either of the two states. Cells that differentiate into the latter state can die off without directly affecting the viability of the cell line. Thus, differentiation allows for the incorporation of fatal functions into cells.
</P><H3><SPAN class="mw-headline" id="Random_State"> Random State </SPAN></H3><P>Differentiation can also be used to combine multiple non-compatible functions into a single cell line. In this case, only one function will be turned on in each cell at any time, but the cell population will express the entire set of functions. Two functions may be non-compatible if they compete for the same resources inside a cell or if they signal through the same intracellular components. These two functions could be simply placed into separate cell lines, but if these two cell lines have different growth rates, one cell line would outcompete the other. Differentiation separates different functions into individual cells and allows the entire cell line to express multiple functions. In such a system, cells start off in an undifferentiated state. As undifferentiated cells replicate, their offspring can stay in the current state or switch into one of the mutually-exclusive states. As long as cells divide faster than they differentiate, the reproduction of undifferentiated cells maintains the undifferentiated population, which ensures that the entire cell line is kept alive. Thus, fatal and mutually-exclusive functions can be incorporated into cell lines by utilizing cellular differentiation.
</P><H3><SPAN class="mw-headline" id="Random_Time"> Random Time </SPAN></H3><P>It is also important to have cells differentiate at random times. Having asynchrony ensures that a large portion of the population will not differentiate at once. Doing so might leave too few undifferentiated cells to maintain the population. It is hard to ensure that cells divide faster than they differentiate if there are bursts in time in which many cells differentiate into terminal states. In addition, if the environment the cells are in continually changes, cells may differentiate into a particular state that is incompatible with the current environment. For example, that particular function may be useless in the current environment, or performing that function in such an environment may lead to the cell’s death. As such, spreading out the time during which cells differentiate increases the chances that an individual cell committed to a state finds itself in a hospitable environment. Thus, both random state and random time are needed for differentiation.
</P><H3><SPAN class="mw-headline" id="System_Outline">System Outline</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Figure 1: The probabilistic switch controls the production of a trigger that activates the random state generator.</DIV></DIV></DIV><P>To implement cellular differentiation, we designed two BioBrick<SUP>TM</SUP> devices. The first device is a probabilistic switch that randomly activates the second device. The second device is a random state generator that, when triggered, determines the final state of the bacterial cell line. Combined, the two devices form a system that allows for the integration of multiple <I>de novo</I> functions into bacterial cell lines.
</P><H2><SPAN class="mw-headline" id="Probabilistic_Switch:_Slipped_Stand_Mutations">Probabilistic Switch: Slipped Stand Mutations</SPAN></H2><P>To construct the first device, the probabilistic switch, we exploit the fact that DNA polymerase slips when replicating long stretches of short nucleotide repeats<SUP>4,5</SUP>. This phenomenon is called slipped-strand mispairing (SSM). We have created two families of such switches  based on where the long repeat is integrated. The first family, called coding SSMs, places a repeat in between the start codon of a gene and the remaining coding region. To create an on-to-off switch, the gene is initially placed in-frame with the start codon, and thus the gene is correctly translated and expressed. When DNA polymerase slips and miscopies the stretch of nucleotide repeats, the coding sequence shifts out of frame with respect to the start codon, and the gene is no longer correctly translated<SUP>6,7</SUP>. In an off-to-on switch, the coding sequence is initially out of frame and can either go into the correct frame or stay out of frame when SSM occurs, depending on whether the stretch of repeats becomes a multiple of three. 
</P><P>The second family, called regulatory SSMs, places the repeat between the -10 and -35 elements of a promoter. To create an on-to-off switch, the distance between the -10 and -35 elements is initially optimal for sigma factor binding. The sigma factor helps recruit RNA polymerase to the promoter and initiate transcription. When the polymerase slips and changes the number of nucleotide repeats, the distance between the -10 and -35 elements changes. Suboptimal spacing between promoter elements reduces sigma factor recognition, resulting in less frequent transcription events<SUP>7,8</SUP>. 
</P></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>Figure 2: The two families of SSMs, shown with particular repeats. For the coding SSM (top), the three possible outcomes from DNA replication are shown; the switch is initially off, and has a chance of turning on with each replication. GFP is used to determine the state of the switch.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>Figure 3: Example of an coding SSM transition in which slippage causes the downstream GFP gene to go into frame, turning on the gene.
</P></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Results">Results</SPAN></H3><P>To test the functionality of SSMs as genetic switches, we used the green fluorescent protein (GFP). By measuring the fluorescence of bacterial cells that contain our constructs, we could determine the expression of GFP in the on and off states. We constructed four classes of coding SSMs, and one class of regulatory SSMs. Each class contains one in-frame or ‘on’ variant, and one or more out-of-frame or ‘off’ variants. The fluorescence of these constructs was then measured for the population and at the single-cell level. We found that AGTC coding repeats and TA regulatory repeats display detectable GFP fluorescence in the ‘on’ state and significantly reduced fluorescence in the ‘off’ state. From flow cytometry data, a broad distribution in fluorescence was observed in cells with AGTC constructs covering three orders-of-magnitude. Since the constructs were placed in high copy plasmids, these distributions might reflect hundreds of individual GFP coding sequences shifting in and out of frame within each cell.
</P><TABLE class="gallery" cellspacing="0" cellpadding="0" align="center"><TBODY><TR><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>Table 1: In green are the four classes of coding SSMs, and in red is the class of regulatory SSM created. These repeats were placed into their respective constructs.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>Figure 4: The measured fluorescence of the coding AGTC construct and the regulatory TA construct on a plate reader. These two classes were found to have significant differences between the on and off state. Error bars represent the standard deviation of two samplings from the same culture.
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>Figure 5: The fluorescence spectrum of individual cells containing the AGTC ‘in’ and ‘out’ coding SSM constructs. Spectrum of positive control is in green, negative control is in blue, AGTC ‘in’ construct is in brown, and AGTC ‘out’ is in red. Note that the ‘in’ repeat spectrum has a peak at a higher value than the ‘out’ repeat spectrum. In addition, note the broad distribution of the spectrum. The constructs were placed in a high copy plasmid (&gt;100 copies per cell). Thus, each one of those plasmids has a chance of slipping, and the total number of GFP transcripts from all the plasmids in a cell determines the overall fluorescence of the cell.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>Figure 6: The fluorescence spectrum of individual cells containing the TA ‘on’ and ‘off’ regulatory SSM constructs. Spectrum of negative control is in red, TA ‘on’ is in blue, and TA ‘off’ is in green. The TA ‘on’ distribution lies to the right of the TA ‘off’ distribution, indicating that the TA ‘on’ construct is a better promoter than its counterpart. These distributions were taken at a higher PMT value than the distributions in the left image. The peaks in between 10<SUP>0</SUP> and 10<SUP>1</SUP> fluorescence may be debris the flow cytometer picked up.  Our flow cytometer is commonly used with eukaryotic cells and has not yet been optimized for use with bacterial cells. The cells measured with the flow cytometer in this figure are the same ones measured with the plate reader.
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>
Sequencing results of long repeats suggest the occurrence of slipped-strand mispairing. If a slipped-strand mispairing occurs early on in a cell line’s history, then a large proportion of the population will have shifted to a construct with a differing number of repeats, and the resulting sequencing of the miniprep will show mixed products. If a slipped-strand mutation occurs later on in the colony’s lineage, then a smaller proportion of the population will have shifted, and the resulting sequencing will have clear peaks, indicating single nucleotides at those locations. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 7: Sequencing results suggest slipped-strand mutation. Cells were transformed with just the coding SSM construct (AGTC)<SUB>9</SUB>. However, the sequencing trace for a colony transformed with (AGTC)<SUB>9</SUB> also shows the presence of (AGTC)<SUB>8</SUB>, suggesting that slippage at the AGTC repeat had occurred. The sequence for GFP after the start codon follows the AGTC repeats.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Future_Work">Future Work</SPAN></H3><P>In future work, we would use the SSM as a switch to turn on the gene that codes the trigger for the second device. To implement a coding SSM, we would place repeats after the start codon of the trigger gene. The trigger would initially be off and turned on when the polymerase slips. Or, to implement a regulatory SSM, we would place the trigger gene downstream of a SSM promoter. Then the gene would be transcribed when the promoter slips into the right length to bind to a transcriptional activator.
</P><P>We would also place coding and regulatory SSMs in front of LacZ, and then chromosomally integrating the construct into <I>E. coli</I> so that there is only one copy of the construct in a cell. Our current work utilizes a high copy number plasmid, and it is difficult to resolve individual slipped strand mutations and calculate slippage frequency in such cases. With LacZ, blue-white screening can then be conducted to determine the switching frequency of the SSMs. In addition, we plan to place the SSMs in front of the gene that codes for tetracycline resistance (tetA). We can then select for and against cells with tetracycline resistance. 
</P><H2><SPAN class="mw-headline" id="Random_State_Generator:_FimE_Recombinase">Random State Generator: FimE Recombinase</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Figure 8: FimE flips the region flanked by IRL and IRL. A tetR promoter is placed between the two binding sites facing upstream. When the DNA segment is flipped, the promoter faces downstream, and GFP is produced. Once the in and out components of a binding site do not match, FimE no longer recognizes that site. The length of the recombination region (in red) was varied from 250 bp to 850 bp in increments of 100 or 200 bp.</DIV></DIV></DIV><P>FimE is the trigger that will act on the random state generator. FimE is a recombinase protein that can flip DNA flanked by two binding sites. The random state generator consists of  recombination regions flanked by FimE binding sites. FimE recognizes binding sites called IRR (inverted repeat right) and IRL (inverted repeat left). Each IRL and IRR consists of an inner and outer sequence, and fimE will only recognize a site that has matching inner and outer sequences.  After FimE flips the DNA in between two correctly formed binding sites, the binding sites become distorted, and FimE can no longer act on the region. Taking advantage of this effect, one could place a promoter initially pointing upstream in between the two binding sites, and a gene downstream of the recombination region. Since the promoter is pointing in the opposite direction of the gene, the downstream gene will not be transcribed. However, once FimE flips the flanked recombination region, the promoter then points in the correct direction and can transcribe the downstream gene<SUP>9</SUP>.
</P><H3><SPAN class="mw-headline" id="Results_2">Results</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Figure 9: The fluorescence of cells with constructs in Figure 6, with varying distances between the two FimE binding sites. A peak at around 250 bp is observed. Fluorescence is correlated to the proportion of plasmids that contain flipped promoter. Cells were measured on plate reader, and error bars show the standard deviation of experiments performed in triplicate.</DIV></DIV></DIV><P>Given that we were placing a different DNA segment in between the two binding sites than is found in wild-type genes, we wanted to determine the range of lengths FimE will act on and how the length affects FimE activity. In addition, we want to determine if the ratio of determined states could be skewed by changing the lengths of different recombination sites.  Thus, we tested how the length of the recombination site affects FimE’s ability to detect the two flanking binding sites. The fluorescence of cells with each construct was used as an indicator of FimE activity. We observed a relationship between GFP fluorescence and separation length (Figure 9). Maximum fluorescence was observed to occur at about a separation length of 250 bp. This distance may be a function of entropy and enthalpy. The chances that two DNA segments on the same strand come together statistically decrease as the two segments are separated by intervening sequences. In addition, two sites very close together are unlikely to come into contact with each other, as it is energetically unfavorable to bend DNA to such a great degree.  These two opposing forces might explain the presence of an optimal separation length.
</P><H3><SPAN class="mw-headline" id="Future_Work_2">Future Work</SPAN></H3><P>Utilizing the fact that FimE flips DNA and cannot recognize binding sites that it has distorted, we can design a system that uses two IRR and two IRL sites to produce a total of three possible states, depending on how FimE acts. Once FimE flips any region in this system, it cannot flip the DNA segment again. Hence, the state of the cell is established after FimE acts. This assumes that FimE is unable to flip adjacent IRL and IRR sites, as it is difficult to bend around such a short DNA segment. The system we propose is shown below. By varying the length of the segment containing the promoter and the length of the segment containing the terminator, we may be able to vary the ratio of determined states.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:598px;"><DIV class="thumbcaption">Figure 10: a) The device design for the population variation generator. Gene A and Gene B (in opposite orientation) are initially off. b) Depending which IRR and IRL sites FimE binds to, different DNA segments are flipped. The bracket indicates the region that was flipped in each case. In the first case, gene B is turned on. In the second case, gene A is turned on. In the third case, gene A and gene B remain off and stay off, since FimE no longer has a valid IRR and IRL to chose from (the middle two binding sites are too close for FimE to act upon). Terminators at the ends of gene A and B are not shown.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H2><P>We have evidence that short nucleotides repeats can be placed in the promoter or behind a start codon and function as randomized genetic switches through slipped-strand mutation. We have also shown that FimE can flip a segment of non-native DNA, and have evidence that the probability that FimE flips the segment depends on the length between the two binding sites. In the future, selected SSMs can be integrated into a FimE expression construct; thus, FimE expression can be randomly switched on in cells. FimE can then interact with the random state generator. Linked together, the probabilistic switch and the random state generator allow for the asynchronous differentiation of cells into mutually exclusive and fatal states.
</P><P>Shown below is the final system design, incorporating the other subprojects in our iGEM team. Initially, <I>fimE</I> is out of frame. When (AGTC)<SUB>10</SUB> through SSM becomes (AGTC)<SUB>9</SUB>, <I>fimE</I> falls into frame, and the FimE trigger protein is expressed. It interacts with the population variation generator, either committing the cell into a state of <A href="/Team:Caltech/Project/Oxidative_Burst" title="Team:Caltech/Project/Oxidative Burst"><FONT style="color:#BB4400">ROS production</FONT></A> or <A href="/Team:Caltech/Project/Lactose_intolerance" title="Team:Caltech/Project/Lactose intolerance"><FONT style="color:#BB4400">curing lactose intolerance</FONT></A>. In the default, undifferentied state, cell will produce a basal level of <A href="/Team:Caltech/Project/Vitamins" title="Team:Caltech/Project/Vitamins"><FONT style="color:#BB4400">folate biosynthesis enzymes</FONT></A>. <A href="/Team:Caltech/Project/Phage_Pathogen_Defense" title="Team:Caltech/Project/Phage Pathogen Defense"><FONT style="color:#BB4400">Phage induction</FONT></A> will be controlled by another AGTC coding SSM. Thus, this population variation machinery links the <A href="/Team:Caltech/Project" title="Team:Caltech/Project"><FONT style="color:#BB4400">subprojects</FONT></A> produced by our team into one coherent system.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">The final system design, linking the subprojects in this iGEM together. The folate biosynthesis pathway will be continually expressed in all states, and the phage induction system will be controlled by a seperate coding SSM (not shown).</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>[1] Ro DK, Paradise EM, Ouellet M, Fisher KJ, Newman KL, Ndungu JM, Ho KA, Eachus RA, Ham TS, Kirby J, Chang MC, Withers ST, Shiba Y, Sarpong R, and Keasling JD. <B>Production of the antimalarial drug precursor artemisinic acid in engineered yeast</B>. Nature 2006 Apr 13; 440(7086) 940-3.
</P><P>[2] Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, Davidson EA, Scouras A, Ellington AD, Marcotte EM, and Voigt CA. <B>Synthetic biology: engineering Escherichia coli to see light</B>. Nature 2005 Nov 24; 438(7067) 441-2.
</P><P>[3] Lu TK and Collins JJ. <B>Dispersing biofilms with engineered enzymatic bacteriophage</B>. Proc Natl Acad Sci U S A 2007 Jul 3; 104(27) 11197-202.
</P><P>[4] Henderson IR, Owen P, and Nataro JP. <B>Molecular switches -- the ON and OFF of bacterial phase variation</B>. Mol Microbiol 1999 Sep; 33(5) 919-32.
</P><P>[5] Levinson G and Gutman GA. <B>Slipped-strand mispairing: a major mechanism for DNA sequence evolution</B>. Mol Biol Evol 1987 May; 4(3) 203-21.
</P><P>[6] Levinson G and Gutman GA. <B>High frequencies of short frameshifts in poly-CA/TG tandem repeats borne by bacteriophage M13 in Escherichia coli K-12</B>. Nucleic Acids Res 1987 Jul 10; 15(13) 5323-38.
</P><P>[7] Torres-Cruz J and van der Woude MW. <B>Slipped-strand mispairing can function as a phase variation mechanism in Escherichia coli</B>. J Bacteriol 2003 Dec; 185(23) 6990-4.
</P><P>[8] van Ham SM, van Alphen L, Mooi FR, and van Putten JP. <B>Phase variation of H. influenzae fimbriae: transcriptional control of two divergent genes through a variable combined promoter region</B>. Cell 1993 Jun 18; 73(6) 1187-96. 
</P><P>[9] Ham TS, Lee SK, Keasling JD, and Arkin AP. <B>A tightly regulated inducible expression system utilizing the fim inversion recombination switch</B>. Biotechnol Bioeng 2006 May 5; 94(1) 1-4.
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Caltech/Project/Population_Variation">http://2008.igem.org/Team:Caltech/Project/Population_Variation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Caltech/Project/Population_Variation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Caltech/Project/Population_Variation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Caltech/Project/Population_Variation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Caltech/Project/Population_Variation&amp;oldid=59742" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>