<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Caltech_Project_Vitamins"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Caltech/Project/Vitamins">Page               </A></LI><LI><A href="/Talk:Team:Caltech/Project/Vitamins">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Project/Vitamins&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Project/Vitamins&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Caltech/Project/Vitamins" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Caltech/Project/Vitamins</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE><TBODY><TR valign="top"><TD><DIV style="position: relative; width: 960px; height: 36px; overflow: hidden;"><DIV style="position: absolute; top: 0; left: 0; font-size: 1000pt; width: 960px; height: 36px; overflow: hidden; line-height: 36px; z-index: 3;"><A href="/Team:Caltech" title="Team:Caltech"> </A></DIV></DIV></TD></TR></TBODY></TABLE><DIV style="font-size:24pt;"><FONT face="verdana" style="color:#BB4400"><CENTER>iGEM 2008
</CENTER></FONT></DIV><TABLE><TBODY><TR valign="top"><TD><DIV style="position: relative; width: 960px; height: 48px; overflow: hidden;"><DIV style="position: absolute; top: 0; left: 0; font-size: 1000pt; width: 960px; height: 48px; overflow: hidden; line-height: 48px; z-index: 3;"><A href="/Team:Caltech" title="Team:Caltech"> </A></DIV></DIV></TD></TR></TBODY></TABLE><DIV><TABLE cellspacing="0"><TBODY><TR><TD style="background-color: #ffffff" width="75px" valign="top"><CENTER><A href="/Team:Caltech" title="Team:Caltech"><FONT face="verdana" style="color:#BB4400"><B>Home</B></FONT></A><P><A href="/Team:Caltech/Members" title="Team:Caltech/Members"><FONT face="verdana" style="color:#BB4400"><B>People</B></FONT></A><A href="/Team:Caltech/Project" title="Team:Caltech/Project"><FONT face="verdana" style="color:#BB4400"><B>Project Details</B></FONT></A><A href="/Team:Caltech/Protocols" title="Team:Caltech/Protocols"><FONT face="verdana" style="color:#BB4400"><B>Protocols</B></FONT></A><A href="/Team:Caltech/Parts" title="Team:Caltech/Parts"><FONT face="verdana" style="color:#BB4400"><B>Completed Systems</B></FONT></A><A href="/Team:Caltech/Biosafety" title="Team:Caltech/Biosafety"><FONT face="verdana" style="color:#BB4400"><B>Biosafety</B></FONT></A><A href="/Team:Caltech/Acknowledgments" title="Team:Caltech/Acknowledgments"><FONT face="verdana" style="color:#BB4400"><B>Support</B></FONT></A></P></CENTER></TD><TD width="880" valign="top" style="padding: 10px; border: 5px solid #FFFFFF; color: #000; background-color: white"><DIV style="font-size:18pt;"><FONT face="verdana" style="color:#BB4400"><I>In vivo</I> Folate Production</FONT></DIV></TD></TR><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Background_Information_on_Folate"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Background Information on Folate</SPAN></A></LI><LI class="toclevel-1"><A href="#Folate_Biosynthesis_Pathway"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Folate Biosynthesis Pathway</SPAN></A></LI><LI class="toclevel-1"><A href="#System_Design"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">System Design</SPAN></A></LI><LI class="toclevel-1"><A href="#Folate_Detection_Methods"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Folate Detection Methods</SPAN></A></LI><LI class="toclevel-1"><A href="#pABA_Detection_Methods"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">pABA Detection Methods</SPAN></A></LI><LI class="toclevel-1"><A href="#Results"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2"><A href="#Modifications_to_System_Design"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Modifications to System Design</SPAN></A></LI><LI class="toclevel-2"><A href="#Folate_Assay_Results_for_folB_AND_folKE"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Folate Assay Results for folB AND folKE</SPAN></A></LI><LI class="toclevel-2"><A href="#para-Aminobenzoic_Acid_.28pABA.29_HPLC_Assay_Results_for_pabA"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">para-Aminobenzoic Acid (pABA) HPLC Assay Results for pabA</SPAN></A></LI><LI class="toclevel-2"><A href="#Conclusions_and_Future_Work"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">Conclusions and Future Work</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Relevant_Parts"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Relevant Parts</SPAN></A></LI><LI class="toclevel-1"><A href="#Basic_Parts"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Basic Parts</SPAN></A></LI><LI class="toclevel-1"><A href="#Construction_Intermediates"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Construction Intermediates</SPAN></A></LI><LI class="toclevel-2"><A href="#Composite_Parts"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Composite Parts</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Background_Information_on_Folate">Background Information on Folate</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">The structure of tetrahydrofolate.</DIV></DIV></DIV><P>Folate, the generic term for the various forms of Vitamin B9, is an essential vitamin involved in single-carbon transfer reactions, which are important for many pathways including amino acid synthesis. Folate deficiencies in women can result in birth defects such as neural tube defects and other spinal cord malformations. As important as folate is, humans are unable to produce folate, and so must obtain it from eating foods such as green leafy vegetables or folate-fortified cereals. An engineered strain of bacteria that would constantly release folate into the gut would reduce the need to fortify breads and cereals with folate, as well as reduce folate-related birth defects in regions with little access to folate-containing foods. In addition to all the reasons stated above, folate is an ideal vitamin to be produced in the gut because, unlike many other vitamins, it has been shown to be absorbed in physiologically relevant quantities in the large intestine<SUP>1, 2</SUP>. 
</P><P>Structurally, folate consists of three main parts: pteridine (derived from GTP), p-aminobenzoic acid (PABA, derived from chorismate), and a poly-glutamyl tail (derived from linking glutamate). 
</P><H2><SPAN class="mw-headline" id="Folate_Biosynthesis_Pathway">Folate Biosynthesis Pathway</SPAN></H2><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:797px;"><DIV class="thumbcaption">The folate gene cluster from <I>L.lactis</I>. Black arrows represent genes which have been tested in metabolic engineering studies, shaded arrows represent genes involved in folate biosynthesis, and white arrows represent genes not involved in folate synthesis<SUP>3</SUP>.</DIV></DIV></DIV></DIV><P>Although folate is naturally produced in <I>E. coli</I>, the folate biosynthesis pathway in the bacteria <I>Lactococcus lactis</I> has been more heavily characterized and studied. In previous studies, this folate gene cluster has been successfully transformed into the folate-consuming bacteria <I>L. gasseri</I>, turning the bacteria into folate-producers<SUP>4</SUP>. Using the folate gene cluster from <I>L. lactis</I> also offers the additional benefit of removing the operon from its natural regulatory context. There are six major enzymatic activities involved in folate synthesis, which, in <I>L. lactis</I>, are contained in five genes: <I>folB</I>, <I>folKE</I>, <I>folP</I>, <I>folC</I>, and <I>folA</I><SUP>1</SUP>. The first four, which we have chosen to focus on, are located in a gene cluster approximately 4.4kb long. We have chosen not to test overexpression of <I>folA</I> because it encodes for an enzyme which turns one form of folate (dihydrofolate) into another form of folate (tetrahydrofolate). Since our assay would detect both types of folate as part of the total folate production, <I>folA</I> was not a prime target for overexpression of folate. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">The folate biosynthesis pathway from <I>L.lactis</I><SUP>3</SUP>.</DIV></DIV></DIV><P>Our strategy is to clone the entire folate operon from the <I>L.lactis</I> genome and to transform the entire operon into <I>E.coli</I>. However, because we are unsure of whether or not the ribosomal binding sites (RBS) within the <I>L.lactis</I> operon would work in <I>E.coli</I>, we are also going to unpack the operon by cloning each of the four genes individually, placing them behind <I>E.coli</I> RBSs, and then recombine them into a single empty BioBricks™ plasmid. In addition to the main folate operon, we will also be experimenting with overexpression of the para-aminobenzoic acid (pABA) synthesis pathway from chorismate. Wegkamp <I>et al.</I> have shown that in order to increase overall total levels of folate, both the pABA synthesis genes and certain folate production genes need to be overexpressed<SUP>5</SUP>. The pABA pathway involves three enzymatic activities, <I>pabA</I>, <I>pabB</I>, and <I>pabC</I> – though in <I>L.lactis</I>, <I>pabB</I> is actually a bifunctional enzyme containing the activities for both <I>pabB</I> and <I>pabC</I><SUP>5</SUP>.
</P><H2><SPAN class="mw-headline" id="System_Design">System Design</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Final folate biosynthesis plasmid</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Final PABA biosynthesis plasmid</DIV></DIV></DIV><P>The overall system design for testing folate production in <I>E. coli</I> is to construct several plasmids: one for each individual gene, one for the folate biosynthesis pathway, and one for the PABA synthesis pathway. Each gene, or gene cluster, would be cloned in an inducible-copy plasmid, which would be low copy by default, but can be switched to high copy via the addition of Isopropyl β-D-1-thiogalactopyranoside (IPTG)to the media. This will allow us to test overexpression of each plasmid separately. In addition, each plasmid contains a constitutive promoter, to ensure the constant production of folate. 
</P><H2><SPAN class="mw-headline" id="Folate_Detection_Methods">Folate Detection Methods</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Image of <I>Lactobacillus rhamnosus.</I> Itself a probiotic strain, <I>L. rhamnosus</I> is commonly used in yogurt <A href="http://www.yaklasansaat.com/resimler/2008haberresimleri/haber_subat_resim/subat19_lactobacillus_rhamnosus.jpg" class="external autonumber" rel="nofollow">[1]</A><A href="http://en.wikipedia.org/wiki/Lactobacillus_rhamnosus" class="external autonumber" rel="nofollow">[2]</A>.</DIV></DIV></DIV><P>We measured folate production, and thus the relative success of our engineering efforts, via a microbiological assay involving the folate-dependent strain <I>Lactobacillus rhamnosus</I> (ATCC 7469)<SUP>6</SUP>. The assay uses growth of the folate-dependent strain <I>L. rhamnosus</I> as an indicator of folate concentrations in the sample. The assay media specifically lacks folate, so folate becomes the limiting factor in the growth of <I>L. rhamnosus</I>, with the only possible source being the bacterial lysates of the engineered <I>E. coli</I>. In order to quantify the relative growth of the folate-dependent strain <I>L. rhamnosus</I>, a standard growth curve must first be characterized using known quantities of folic acid in assay media. Once the standard curve has been established, then experimental growth levels, as quantified by spectrophotometry, can be interpolated. Protocols are based off of BD Biosciences Folate Assay Medium datasheet<SUP>7</SUP>. Growth was measured by taking the OD at 546nm.
</P><P>Further details are available on the <A href="/Team:Caltech/Protocols/Folate_assay" title="Team:Caltech/Protocols/Folate assay"><FONT style="color:#BB4400">folate microbiological assay protocol</FONT></A> page.
</P><H2><SPAN class="mw-headline" id="pABA_Detection_Methods">pABA Detection Methods</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">para-Aminobenzoic Acid <A href="http://en.wikipedia.org/wiki/4-Aminobenzoic_acid" class="external autonumber" rel="nofollow">[3]</A></DIV></DIV></DIV><P>para-Aminobenzoic Acid (PABA) can be detected using high performance liquid chromatography (HPLC)<SUP>8</SUP>. Using a 14 minute protocol, we were able to detect PABA peaks coming off a C18 column at 4.9 min. A standard curve was made by running a series of 1:3 PABA dilutions starting at a 10μg/ml concentration. The PABA for the standard curve was spiked into wild type <I>E. coli</I> cell lysate, which by itself did not show any detectable PABA peaks. Samples were run using the same HPLC protocol as the standards, and both lysate and supernatant were tested for PABA peaks. 
</P><P>Again, details on the PABA detection method can be found on the <A href="/Team:Caltech/Protocols/PABA_HPLC_assay" title="Team:Caltech/Protocols/PABA HPLC assay"><FONT style="color:#BB4400">para-aminobenzoic acid (pABA) HPLC protocol</FONT></A> page.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Modifications_to_System_Design">Modifications to System Design</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Target constructs for folate biosynthesis and pABA synthesis gene overexpression. We were able to successfully clone <I>folB</I> in an inducible-copy plasmid and <I>folKE</I> and <I>pabA</I>, in high-copy plasmids. Unfortunately, we were unable to complete the <I>folBKE</I>, <I>pabB</I>, or <I>pabA + pabB</I> constructs.</DIV></DIV></DIV><P>We were able to successfully extract and clone the following three genes: <I>folB</I>, <I>folKE</I>, and <I>pabA</I>. We aimed to create individual constructs of each gene in the IPTG inducible-copy plasmid pSB2K3, as well as constructs with the two folate genes combined and with the two PABA synthesis genes combined. However, we had issues cloning our genes into the pSB2K3 + B0015 terminator construct, and so switched to completing the constructs in the high copy plasmid pSB1AK3 + B0015 terminator.
</P><P>Our final constructs, shown on the right, are <I>folB</I> in pSB2K3, and <I>folKE</I> and <I>pabA</I> in pSB1AK3. We were unable to sequence confirm our <I>folBKE</I> and <I>pabB</I> constructs, and so unfortunately could not include their data. Previous studies on folate overexpression have shown that both folate synthesis and PABA synthesis genes need to be overexpressed simultaneously in order to increase total folate levels<SUP>3</SUP>. Therefore, tests on <I>folB</I> and <I>folKE</I> were done with and without the addition of PABA during the <I>E. coli</I>inoculation; detection of higher folate levels on addition of PABA would indicate PABA as a limiting factor for folate production.
</P><P>We were unable to use PCR to extract several of the target genes: the entire folate gene cluster, <I>folP</I>, and <I>folC</I>. After analyzing the sequencing results of our other folate biosynthesis genes, we realized that the genetic sequences had several point mutations which were not `stop' codons. This indicated that we had a homologous, but not identical, genomic DNA than the one that we has used for PCR primer design. We believe that this homologous sequence could have resulted in too many differences between the primer and the genomic DNA, resulting in little to no binding. This may explain why we were unable to successfully extract <I>folP</I>, <I>folC</I>, and subsequently, the entire folate gene cluster (since <I>folP</I> and <I>folC</I> are the last two genes in the cluster) from the genomic DNA of <I>L. lactis</I>. The revised target constructs are shown . 
</P><H3><SPAN class="mw-headline" id="Folate_Assay_Results_for_folB_AND_folKE">Folate Assay Results for <I>folB</I> AND <I>folKE</I></SPAN></H3><P>The experimental setup included running a standard curve with known amounts of folic acid (0-10ng) simultaneously with the samples, in order to have the same basis for comparison. The hope was that the standard curve growth OD values would correspond with known concentrations, and so sample concentrations could be interpolated based upon the standard. <I>folKE</I>, which was cloned into a constitutive high-copy plasmid, was tested with and without the addition of 500 ng of PABA during inoculation. <I>folB</I>, which is in an inducible copy plasmid, it was tested induced to high-copy as well as uninduced low-copy, with and without 500 ng of PABA during inoculation. We assayed both the supernatant and the cell lysate, though only the supernatant had measurable results. 
</P><P>The growth data for the samples are very encouraging since the relative folate levels match what we would expect. We see that the addition of 500 ng of PABA during inoculation dramatically increases overall folate levels for <I>folKE</I> relative to the samples without PABA. Furthermore, adding PABA to the wild type controls did not affect growth at all, suggesting that the assay bacteria <I>L.rhamnosus</I> was not metabolizing the extra PABA. The extremely high OD values for all the samples were possibly the result of not completely washing out all of the culture media prior to adding the <I>L.rhamnosus</I> to the assay samples. 
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>Test #1: Folate Standard Curve from 0-10ng. This standard curve is pretty much useless.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>Test #1: Relative folate levels for <I>folKE</I> compared to <I>E. coli</I> wild type. Here we see encouraging confirmation of all the expected trends - the addition of PABA during inoculation increases total folate levels but only in the samples with overexpression of our overexpressed folate constructs.
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>When we repeated the assay in duplicate, this time on <I>folB</I> as well, we were able to observe the same trends in growth with and without the addition of PABA. Again, we see that for <I>folKE</I>, the addition of PABA does increase total folate levels, though not as dramatically as before. The rampant growth of the wild type control is disconcerting, but wild type folate production again appears to be unaffected by the addition of PABA. Furthermore, folate levels for <I>folKE</I> are still higher than the control where <I>L.rhamnosus</I> was added to only media without supernatant.
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>Test #2: Folate Standard Assay.A beautiful linear range between .1 and 10 ng can be seen, but overall growth does not correspond to sample growth. Could we be saturating the standard?
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>Test #2: Folate Assay Results with <I>folKE</I> and <I>folB</I>. Although the wild type growth is mystifying, the folate levels for <I>folKE</I> exhibit the same trends as before, where increased folate is observed with the addition of PABA. <I>FolB</I> was tested in low (no IPTG) and high (with IPTG) copy since it is in an inducible-copy plasmid. In the folB data, induced high copy without PABA shows higher growth than low copy without PABA, and the addition of PABA shows the highest growth of all. It is interesting to note that that the two +PABA levels for <I>folB</I> appear to be the same.
</P></DIV></DIV></TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Increasing the flux of both pathways upstream of pABA integration into the pterin (derived from GTP) component of folate may have created a bottleneck. Unfortunately we were unable to test this hypothesis since we could not overexpress <I>folP</I><SUP>3</SUP></DIV></DIV></DIV><P>And what of <I>folB</I>? Recall that <I>folB</I> was the only gene to be successfully cloned into an inducible-copy plasmid, and so it was tested both induced and uninduced. The folate levels in the induced sample (high-copy) are higher than in the uninduced (low-copy) sample, which is consistent with what we would expect. The addition of PABA to both induced and uninduced increases relative levels of folate, which is also consistent. However, it is interesting to note that folate levels for the +PABA samples are the same for induced and uninduced. Of course, given the small sample size, this could just be due to variability, but it could also suggest that folate production reached an upper limit. This explanation seems even more likely if we reconsider the folate biosynthesis pathway, and we see that <I>folB</I> and <I>folKE</I> are both located upstream of actual integration of PABA into the molecule, accomplished by <I>folP</I>. It is very possible that we are increasing the flux of both pathways going into the <I>folP</I> junction, but as we were unable to overexpress <I>folP</I> as well, we may have created a bottleneck.
</P><H3><SPAN class="mw-headline" id="para-Aminobenzoic_Acid_.28pABA.29_HPLC_Assay_Results_for_pabA">para-Aminobenzoic Acid (pABA) HPLC Assay Results for <I>pabA</I></SPAN></H3><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>HPLC peaks for the <I>pabA</I> sample. The blue trace is the control cell supernatant, which has no peak at 4.9 min, the elution time we found for PABA. The area for the <I>pabA</I> peak, shown in green, is 82.55.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 410px;"><DIV class="gallerytext"><P>Standard Curve with linear trendline (y = 117.39x). Using this linear approximation, the concentration of pABA is 703 ng/mL for <I>pabA</I> overexpression.
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>Using high performance liquid chromatography (HPLC) we were able to detect PABA peaks from overexpression of <I>pabA</I> compared to a control sample of wild type supernatant. Compared to the peak areas of the standard curve, we were able to use a linear approximation to determine the concentration of PABA to be 703 ng/mL for <I>pabA</I> overexpression. 
</P><H3><SPAN class="mw-headline" id="Conclusions_and_Future_Work">Conclusions and Future Work</SPAN></H3><P>We have shown, through very preliminary tests, the feasibility of overproduction of folate in <I>E. coli</I> using genes originally derived from <I>L. lactis</I>. Our data have confirmed previous studies in the necessity of overexpressing both folate and PABA synthesis genes, and we have shown that folate appears to be mostly present in the supernatant. Our data, again confirming previous results<SUP>9</SUP>, also suggest that overexpression of <I>folKE</I> is the most effective, definitely more effective than <I>folB</I>.
</P><P>We have also shown that it is possible to overexpress para-aminobenzoic acid production in <I>E. coli</I> and that overexpression of <I>pabA</I> increases total levels of para-aminobenzoic acid (PABA).
</P><P>Further work on this project would include repeating the folate assay to generate more data, perfecting the assay in order to quantify folate levels with the standard curve, making the <I>pabA</I> + <I>pabB</I> and <I>folB</I> + <I>folKE</I> constructs, making a <I>folB</I> + <I>folKE</I> + <I>pabA</I> + <I>pabB</I> construct, and extracting and cloning <I>folP</I> from the <I>L. lactis</I> genome. 
</P><H2><SPAN class="mw-headline" id="Relevant_Parts">Relevant Parts</SPAN></H2><H2><SPAN class="mw-headline" id="Basic_Parts">Basic Parts</SPAN></H2><TABLE border="1"><CAPTION> Basic Parts (Extracted from <I>L.lactis subspecies IL1403</I> genome)
</CAPTION><TBODY><TR><TH> Part Name </TH><TH> Registry # </TH><TH> Description </TH><TH> Cloned? </TH><TH> Sequence confirmed?
</TH></TR><TR><TH> Entire folate synthesis operon
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137002" class="external text" rel="nofollow">BBa_K137002</A></TD><TD> Includes folB+folKE+folP+ylgG+folC </TD><TD> NO </TD><TD> NO
</TD></TR><TR><TH> folB
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137009" class="external text" rel="nofollow">BBa_K137009</A></TD><TD> dihydroneopterin aldolase </TD><TD> YES</TD><TD> YES
</TD></TR><TR><TH> folKE
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137011" class="external text" rel="nofollow">BBa_K137011</A></TD><TD> fusion gene: GTP cyclohydrolase &amp; 2-amino-4-hydroxy-6- hydroxymethyldihydropteridine pyrophosphokinase  </TD><TD> YES</TD><TD> YES
</TD></TR><TR><TH> folP
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137012" class="external text" rel="nofollow">BBa_K137012</A></TD><TD> Dihydropteroate synthase  </TD><TD> NO</TD><TD> NO
</TD></TR><TR><TH> folC
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137013" class="external text" rel="nofollow">BBa_K137013</A></TD><TD> folate synthetase/polyglutamyl folate synthetase  </TD><TD> NO</TD><TD> NO
</TD></TR><TR><TH> pabA
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137005" class="external text" rel="nofollow">BBa_K137005</A></TD><TD> para-aminobenzoate synthetase component II </TD><TD> YES</TD><TD> YES
</TD></TR><TR><TH> pabB
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137006" class="external text" rel="nofollow">BBa_K137006</A></TD><TD> para-aminobenzoate synthetase component I </TD><TD> YES</TD><TD> YES
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Construction_Intermediates">Construction Intermediates</SPAN></H2><TABLE border="1"><CAPTION> Construction Intermediates: Adding B0034 (strong RBS) before each individual gene
</CAPTION><TBODY><TR><TH> Part Name </TH><TH> Registry # </TH><TH> Description </TH><TH> Cloned? </TH><TH> Sequence confirmed?
</TH></TR><TR><TH> folB + B0034
</TH><TD><A href="http://partsregistry.org/Part:BBa_S03957" class="external text" rel="nofollow">BBa_S03957</A></TD><TD> RBS + dihydroneopterin aldolase </TD><TD> YES</TD><TD> YES
</TD></TR><TR><TH> folKE + B0034
</TH><TD><A href="http://partsregistry.org/Part:BBa_S03958" class="external text" rel="nofollow">BBa_S03958</A></TD><TD> RBS + fusion gene: GTP cyclohydrolase &amp; 2-amino-4-hydroxy-6- hydroxymethyldihydropteridine pyrophosphokinase  </TD><TD> YES</TD><TD> YES
</TD></TR><TR><TH> folP + B0034
</TH><TD><A href="http://partsregistry.org/Part:BBa_S03959" class="external text" rel="nofollow">BBa_S03959</A></TD><TD> RBS + Dihydropteroate synthase  </TD><TD> NO</TD><TD> NO
</TD></TR><TR><TH> folC + B0034
</TH><TD><A href="http://partsregistry.org/Part:BBa_S03960" class="external text" rel="nofollow">BBa_S03960</A></TD><TD> RBS + folate synthetase/polyglutamyl folate synthetase  </TD><TD> NO</TD><TD> NO
</TD></TR><TR><TH> pabA + B0034
</TH><TD><A href="http://partsregistry.org/Part:BBa_S03976" class="external text" rel="nofollow">BBa_S03976</A></TD><TD> RBS + para-aminobenzoate synthetase component II </TD><TD> YES</TD><TD> YES
</TD></TR><TR><TH> pabB + B0034
</TH><TD><A href="http://partsregistry.org/Part:BBa_S03977" class="external text" rel="nofollow">BBa_S03977</A></TD><TD> RBS + para-aminobenzoate synthetase component I </TD><TD> YES</TD><TD> YES
</TD></TR><TR><TH> B0034 + folB + B0034 + folKE
</TH><TD><A href="http://partsregistry.org/Part:BBa_S03961" class="external text" rel="nofollow">BBa_S03961</A></TD><TD> Combining folB (with RBS) + folKE (with RBS) </TD><TD> YES</TD><TD> YES
</TD></TR></TBODY></TABLE><TABLE border="1"><CAPTION> Construction Intermediates: Adding J23100 (constitutive promoter) to each gene (with RBS already)
</CAPTION><TBODY><TR><TH> Part Name </TH><TH> Registry # </TH><TH> Description </TH><TH> Cloned? </TH><TH> Sequence confirmed?
</TH></TR><TR><TH> folB + J23100
</TH><TD><A href="http://partsregistry.org/Part:BBa_S04032" class="external text" rel="nofollow">BBa_S04032</A></TD><TD> Promoter(j23100) + RBS(b0034) + dihydroneopterin aldolase </TD><TD> YES</TD><TD> YES
</TD></TR><TR><TH> folKE + J23100
</TH><TD><A href="http://partsregistry.org/Part:BBa_S04033" class="external text" rel="nofollow">BBa_S04033</A></TD><TD> Promoter + RBS + fusion gene: GTP cyclohydrolase &amp; 2-amino-4-hydroxy-6- hydroxymethyldihydropteridine pyrophosphokinase  </TD><TD> YES</TD><TD> YES
</TD></TR><TR><TH> pabA + J23100
</TH><TD><A href="http://partsregistry.org/Part:BBa_S04034" class="external text" rel="nofollow">BBa_S04034</A></TD><TD> Promoter + RBS + para-aminobenzoate synthetase component II </TD><TD> YES</TD><TD> YES
</TD></TR><TR><TH> folBKE + J23100
</TH><TD><A href="http://partsregistry.org/Part:BBa_S04035" class="external text" rel="nofollow">BBa_S04035</A></TD><TD> Promoter + folB (with RBS) + folKE (with RBS) </TD><TD> YES </TD><TD> YES
</TD></TR><TR><TH> pabB + J23100
</TH><TD><A href="http://partsregistry.org/Part:BBa_S04039" class="external text" rel="nofollow">BBa_S04039</A></TD><TD> Promoter + pabB  </TD><TD> YES </TD><TD> YES
</TD></TR><TR><TH> pabB + B0015
</TH><TD><A href="http://partsregistry.org/Part:BBa_S04038" class="external text" rel="nofollow">BBa_S04038</A></TD><TD> pabB + b0015 terminator </TD><TD> MAYBE </TD><TD> NO
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Composite_Parts">Composite Parts</SPAN></H3><TABLE border="1"><CAPTION> Composite Parts: Adding B0015 (double terminator) to complete constructs with promoter + RBS already
</CAPTION><TBODY><TR><TH> Part Name </TH><TH> Registry # </TH><TH> Description </TH><TH> Cloned? </TH><TH> Sequence confirmed?
</TH></TR><TR><TH> folB + B0015
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137053" class="external text" rel="nofollow">BBa_K137053</A></TD><TD> Promoter(j23100) + RBS(b0034) + folB (dihydroneopterin aldolase) + double terminator (b0015) </TD><TD> YES </TD><TD> YES
</TD></TR><TR><TH> folKE + B0015
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137054" class="external text" rel="nofollow">BBa_K137054</A></TD><TD> Promoter + RBS + folKE (fusion gene: GTP cyclohydrolase &amp; 2-amino-4-hydroxy-6- hydroxymethyldihydropteridine pyrophosphokinase) + double terminator (b0015)  </TD><TD> YES </TD><TD> YES
</TD></TR><TR><TH> pabA + B0015
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137055" class="external text" rel="nofollow">BBa_K137055</A></TD><TD> Promoter + RBS + pabA (para-aminobenzoate synthetase component II) </TD><TD> YES  </TD><TD> YES
</TD></TR><TR><TH> pabB + B0015
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137109" class="external text" rel="nofollow">BBa_K137109</A></TD><TD> Promoter + RBS + pabB (para-aminobenzoate synthetase component I) </TD><TD> YES  </TD><TD> NO
</TD></TR><TR><TH> folBKE + B0015
</TH><TD><A href="http://partsregistry.org/Part:BBa_K137056" class="external text" rel="nofollow">BBa_K137056</A></TD><TD> Promoter + RBS + folB + folKE + b0015 </TD><TD> YES </TD><TD> NO
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>[1] Asrar FM and O’Connor DL. <B>Bacterially synthesized folate and supplemental folic acid are absorbed across the large intestine of piglets.</B> J Nutr Biochem 2005 Oct; 16(10) 587-93.
</P><P>[2] Camilo E, Zimmerman J, Mason JB, Golner B, Russell R, Selhub J, and Rosenberg IH. <B>Folate synthesized by bacteria in the human upper small intestine is assimilated by the host.</B> Gastroenterology 1996 Apr; 110(4) 991-8.
</P><P>[3] Sybesma W, Starrenburg M, Kleerebezem M, Mierau I, de Vos WM, and Hugenholtz J. <B>Increased production of folate by metabolic engineering of Lactococcus lactis.</B> Appl Environ Microbiol 2003 Jun; 69(6) 3069-76.
</P><P>[4] Wegkamp A, Starrenburg M, de Vos WM, Hugenholtz J, and Sybesma W. <B>Transformation of folate-consuming Lactobacillus gasseri into a folate producer.</B> Appl Environ Microbiol 2004 May;70(5) 3146-8.
</P><P>[5] Wegkamp A, van Oorschot W, de Vos WM, and Smid EJ. <B>Characterization of the role of para-aminobenzoic acid biosynthesis in folate production by Lactococcus lactis.</B> Appl Environ Microbiol 2007 Apr; 73(8) 2673-81.
</P><P>[6] Horne DW and Patterson D. <B>Lactobacillus casei microbiological assay of folic acid derivatives in 96-well microtiter plates.</B> Clin Chem 1988 Nov; 34(11) 2357-9.
</P><P>[7] BD Biosciences. <B>Folic Acid Assay Medium.</B><A href="http://www.bd.com/ds/technicalCenter/inserts/Folic_Acid_Assay_Medium.pdf" class="external autonumber" rel="nofollow">[4]</A></P><P>[8] Zhang GF, Mortier KA, Storozhenko S, Steene JVD, Straeten DVD, Lambert WE. <B>Free and total para-aminobenzoic acid analysis in plants with high-performance liquid chromatography/tandem mass spectrometry.</B> Rapid Communications in Mass Spectrometry: Volume 19, Issue 8 , Pages 963 - 969, 2005.
</P><P>[9] Sybesma W, Burgess C, Starrenburg M, van Sinderen D, and Hugenholtz J. <B>Multivitamin production in Lactococcus lactis using metabolic engineering.</B> Metab Eng 2004 Apr; 6(2) 109-15.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Caltech/Project/Vitamins">http://2008.igem.org/Team:Caltech/Project/Vitamins</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Caltech/Project/Vitamins" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Caltech/Project/Vitamins" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Caltech/Project/Vitamins&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Caltech/Project/Vitamins&amp;oldid=59051" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>