<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Shiyan_SY_China_Project_html skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Shiyan SY China/Project.html</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>IGEM-PROJECT</TITLE></P><DIV class="top"><DIV class="nav_bar"><UL style="margin:0px auto;" class="nav clearfix"><LI class="m n1"><H3><A target="_blank" style="cursor: pointer;" onclick="location.href ='https://2015.igem.org/Team:Shiyan_SY_China';" title="">HOME</A></H3></LI><LI class="m n2"><H3><A target="_blank" style="cursor: pointer;" onclick="location.href ='https://2015.igem.org/Team:Shiyan_SY_China/Project';" title="">PROJECT</A></H3></LI><LI><A style="background:#27AE60" href="https://2015.igem.org/Team:Shiyan_SY_China/Project#t1">Description</A></LI><LI><A style="background:#27AE60" href="https://2015.igem.org/Team:Shiyan_SY_China/Project#t2">Background</A></LI><LI><A style="background:#27AE60" href="https://2015.igem.org/Team:Shiyan_SY_China/Project#t3">Design</A></LI><LI><A style="background:#27AE60" href="https://2015.igem.org/Team:Shiyan_SY_China/Project#t4">Experiment&amp;Steps</A></LI><LI><A style="background:#27AE60" href="https://2015.igem.org/Team:Shiyan_SY_China/Project#t5">Results&amp;Future</A></LI><LI><A style="background:#27AE60" href="https://2015.igem.org/Team:Shiyan_SY_China/Project#t6">Parts</A></LI></UL><LI class="m n3"><H3 style="line-height: 2;"><A target="_blank" style="cursor: pointer;back" onclick="location.href ='https://2015.igem.org/Team:Shiyan_SY_China/Practices';" title="">HUMAN PRACTICES</A></H3></LI><LI><A style="background:#F39C12" href="https://2015.igem.org/Team:Shiyan_SY_China/Practices#t1">T-shirt</A></LI><LI><A style="background:#F39C12" href="https://2015.igem.org/Team:Shiyan_SY_China/Practices#t2">Biology Speech</A></LI><LI><A style="background:#F39C12" href="https://2015.igem.org/Team:Shiyan_SY_China/Practices#t3">IGEM Magazine</A></LI><LI><A style="background:#F39C12" href="https://2015.igem.org/Team:Shiyan_SY_China/Practices#t4">Questionnaire</A></LI><LI><A style="background:#F39C12" href="https://2015.igem.org/Team:Shiyan_SY_China/Practices#t5">Sina Weibo</A></LI><LI class="m n4"><H3><A target="_blank" style="cursor: pointer;" onclick="location.href ='https://2015.igem.org/Team:Shiyan_SY_China/Safety';" title="">SAFETY</A></H3></LI><LI><A style="background:#BE382A" href="https://2015.igem.org/Team:Shiyan_SY_China/Safety#t1">Safety Design</A></LI><LI><A style="background:#BE382A" href="https://2015.igem.org/Team:Shiyan_SY_China/Safety#t2">Lab security</A></LI><LI><A style="background:#BE382A" href="https://2015.igem.org/Team:Shiyan_SY_China/Safety#t3">Safety Delivery</A></LI><LI class="m n5"><H3><A target="_blank" style="cursor: pointer;" onclick="location.href ='https://2015.igem.org/Team:Shiyan_SY_China/Team';" title="">TEAM</A></H3></LI><LI><A style="background:#8E44AD" href="https://2015.igem.org/Team:Shiyan_SY_China/Team#t1">Members</A></LI><LI><A style="background:#8E44AD" href="https://2015.igem.org/Team:Shiyan_SY_China/Team#t2">Instructors</A></LI><LI><A style="background:#8E44AD" href="https://2015.igem.org/Team:Shiyan_SY_China/Team#t3">Attributions</A></LI><LI><A style="background:#8E44AD" href="https://2015.igem.org/Team:Shiyan_SY_China/Team#t4">Collaborations</A></LI><LI class="m n6"><H3><A target="_blank" style="cursor: pointer;" onclick="location.href ='https://2015.igem.org/Team:Shiyan_SY_China/Notebook';" title="">NOTEBOOK</A></H3></LI><LI><A style="background:#0498F9" href="https://2015.igem.org/Team:Shiyan_SY_China/Notebook#t1">Notebook</A></LI></DIV><DIV><P style="width:800px; height:100px; padding:50px 80px; color:#000; text-align:center; margin:0 auto; font-size:48px; font-family:'Trebuchet MS', Helvetica, sans-serif;">You have to believe in yourself. That’s the secret of success. </P><P style="height:100px; color:#000; width:960px; line-height:100px; margin: 0 auto; text-align:right; font-size:24px; font-family:'Trebuchet MS', Helvetica, sans-serif;">
—— Charles Chaplin</P></DIV></DIV><DIV id="bd"><DIV id="content"><DIV class="wz_pb"><DIV class="mainTitleHeader"><A name="t1"><P style="font-weight: bold;">Description</P></A></DIV><DIV class="mainContainer"><DIV><P>
With increased agriculture activities around the world, it becomes a common practice to use pesticides to manage pest problem. Runoff can carry field pesticide into aquatic environment while wind can carry them to other fields, potentially affecting other species. Over time, repeated application increases pest resistance and facilitate the pest resurgence. Further, especially in China, toxic pesticide residues on green vegetable and fruits become a major public health problem. 
</P></DIV><P>
In order to provide a solution, we design an engineered bacterium, secreting the OpdA enzyme to degrade the common toxic pesticide residues. Its secretion is under the temperature control and can only be activated at specified temperature. To avoid the secondary pollution, a UV-induced suicide gene is inserted into the bacteria: upon exposure to UV or sunshine, the suicide procedure is induced. This purpose of the design is to remove toxic pesticide safely without affecting the environment. 
</P></DIV></DIV><DIV class="wz_pb"><DIV class="mainTitleHeader"><A name="t2"><P style="font-weight: bold;">Background</P></A></DIV><DIV class="mainContainer"><DIV><P>
With the increased environmental pollution and climate change, pest problem becomes a serious issue in agriculture. According to the statistical data from Chinese Academy of Agricultural Sciences, in recent years, over seventy pest species have been recorded on vegetables and fruits. If not controlled, these pests could lead to complete wipeout of various crops, such as vegetables, fruits and grains. Therefore， to deal with the pest issue, the farmers have to constantly spray pesticides, and even use the high-toxic pesticides which are forbidden by China.  As a result, a majority of vegetables and fruits, containing over dosed pesticides which are way above the permitted limits, get onto the dining tables of  thousands of families through channels such as farmer’s markets, supermarkets and roadside stalls.
</P></DIV><DIV><P>
According to the statistics from World Health Organization, there are at least 500,000 pesticide poison accidents per yearwith 115,000 death due to pesticide poisoning all over the world; furtherover 85% of cancer cases and 80 kinds of diseases are suspected to related to over-dosed pesticide usage. In many major cities in China, it is reported that over 47% vegetables and fruits on the market have over-dosed pesticide residues.
</P></DIV><P>
Currently, the mostly used pesticides are phosphorus pesticides and ester pesticides. The phosphorus and ester in these two pesticides can both cause human body damages to . 1. Organophosphorus pesticides, are organic composites to prevent plant diseases and insect pests. These pesticides have multiple varieties, high pesticide effects, wide applications and are easy to be broken down. Additionally, they generally will not accumulate in human bodies and animal bodies, and are one of the most widely used pesticides. The organophosphorus pesticide productscurrently are mostly insecticides. Most commone ones areparathion, demeton, malathion, dimethoate, trichlorphon and dichlorvos. In recent years, organophosphorus pesticides such as bactericide and rodenticide have also been synthesized. Organophosphorus pesticides have many varieties and can be divided into high-toxic, medium-toxic and low-toxic according to their degrees of toxicity. A majority of organophosphorus pesticides belong to high- and medium-toxic varieties while a minority of them are low-toxic. Only a gentle contact of small amount of high-toxic organophosphorus pesticides can cause poisoning, while in the contract, damages can only be caused if a large amount of low-toxic organophosphorus pesticides enter human bodies. The amounts of organophosphorus within human bodies which can cause poisoning or even death, varies upon individual conditions. The poisoning symptoms caused by oral intake of organophosphorus pesticides’ usually are severer than the ones caused by respiratory intake or skin contacts. Further, the onset speed is faster in oral intake comparing to other. However, if organophosphorus pesticides were respiratory intake in large amounts or with high concentration, it can cause death within five minutes. The molecular mechanism behind the toxic effects of organophosphorus pesticides is that  organophosphorus phosphorylates cholinesterase, an enzyme to degrade acetylcholine, ; this phosphorylation suppresses the degradation activity of cholinesterase . . Consequently, accumulated acetylcholine overly stimulates  the cholinergic nerves, which leads to muscarinic, nicotinic and other central nervous system symptoms.  
</P><P>
Many studies indicated that over dosed pesticides could cause many diseases such as various cancers, Children's mental retardation, meningitis Parkinson’s disease, cardiovascualr and cerebrovascular diseases, diabetes and infertility. Acute symptoms associated with over dosed pesticides include headache, dizziness, vomit, stomachache and diarrhea. 
</P></DIV></DIV><DIV class="wz_pb"><DIV class="mainTitleHeader"><A name="t3"><P style="font-weight: bold;">Design</P></A></DIV><DIV class="mainContainer"><DIV><H style="font-size:15px;font-weight: bold;"> Experimental Design: </H></DIV><DIV> </DIV><DIV> </DIV><DIV> </DIV><P>
Note: F1:EcoRI-Constitutive promoter-RNA thermometer-rbs-ompA-Hind III,F2:HindIII-opdA-DNA spacer-Spe, F3:Xbal-RecA(SOS)-rbs-Hind III, F4: Hind III-ccdB-Pstl.</P><DIV> </DIV><H style="font-size:14px;font-weight: bold;">Experimental Procedure: </H><DIV> </DIV><DIV> </DIV><DIV><H style="font-size:14px;font-weight: bold;">
1)Micro-organisms degrades organophosphorus, and organophosphorus-degradation enzyme opdA
</H><P>
Faced with the stress of human pollution such as pesticides, nature itself has evolved many methods to deal with these problems. For example, many natural micro-organisms contain enzymes to degrade organophosphorus pesticides. Currently the micro-organisms which are capable to degrade organophosphorus pesticides include bacteria, fungus, actinomycete and alga. As the research goes further, people find that these degrading effects come from secreting an enzyme, which can hydrolyze phosphoester bonds, organophosphorus degradation enzyme. Because each organophosphorus pesticide has similar structure and protein sequence, one kind of organophosphorus degradation enzyme is capable todegrade multiple kinds of organophosphorus pesticides. Organophosphorus-degradation enzyme has been mostly recognized as the best method to eliminate pesticide residues currently. At present, many enzymes have been identified to be used to degrade organophosphate pesticides. Among these enzymes, the organophosphorus-degradation enzyme (opdA) which comes from Agrobacterium radiobacter P230 has wider targets and higher enzyme-catalyst efficiency. In recent years, the research on the structure and function of organophosphorus-degradation enzyme has gained promising progress,.   Thus, it is possible to improve the properties of organophosphorus-degradation enzyme through genetic engineering and protein engineering method, which meet requirements of different applications.
</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
2)In this project, we will use organophosphorus-degradation enzyme opdA to eliminate residual organophosphorus pesticides on fruits and vegetables.
</H><P>
The organophosphorus-degradation enzyme (opdA) gene opdA （NCBI genbank：Accession: AY043245.2） programmed by Agrobacterium radiobacter contains 1,155 nucleic acids, programming 384 amino acid residues. The N-terminal of protein sequence is the signal peptide while the C-terminal is the degradation-enzyme sequence. The nucleic acid sequence and amino acid sequence are as follows:
</P><H style="font-size:14px;font-weight: bold;">
Nucleotide sequence
</H><P>
at gcaaacgaga agagatgcac ttaagtctgc ggccgcaata actctgctcg gcggcttggc tgggtgtgca agcatggccc gaccaatcgg tacaggcgat ctgattaata ctgttcgcgg ccccattcca gtttcggaag cgggcttcac actgacccat gagcatatct gcggcagttc ggcgggattc ctacgtgcgt ggccggagtt tttcggtagc cgcaaagctc tagcggaaaa ggctgtgaga ggattacgcc atgccagatc ggctggcgtg caaaccatcg tcgatgtgtc gactttcgat atcggtcgtg acgtccgttt attggccgaa gtttcgcggg ccgccgacgt gcatatcgtg gcggcgactg gcttatggtt cgacccgcca ctttcaatgc gaatgcgcag cgtcgaagaa ctgacccagt tcttcctgcg tgaaatccaa catggcatcg aagacaccgg tattagggcg ggcattatca aggtcgcgac cacagggaag gcgaccccct ttcaagagtt ggtgttaaag gcagccgcgc gggccagctt ggccaccggt gttccggtaa ccactcacac gtcagcaagt cagcgcgatg gcgagcagca ggcagccata tttgaatccg aaggtttgag cccctcacgg gtttgtatcg gtcacagcga tgatactgac gatttgagct acctaaccgg cctcgctgcg cgcggatacc tcgtcggttt agatcgcatg ccgtacagtg cgattggtct agaaggcaat gcgagtgcat tagcgctctt tggtactcgg tcgtggcaaa caagggctct cttgatcaag gcgctcatcg accgaggcta caaggatcga atcctcgtct cccatgactg gctgttcggg ttttcgagct atgtcacgaa catcatggac gtaatggatc gcataaaccc agatggaatg gccttcgtcc ctctgagagt gatcccattc ctacgagaga agggcgtccc gccggaaacg ctagcaggcg taaccgtggc caatcccgcg cggttcttgt caccgaccgt gcgggccgtc gtgacacgat ctgaaacttc ccgccctgcc gcgcctattc cccgtcaaga taccgaacga tga
</P><H style="font-size:14px;font-weight: bold;">
Amino acid sequence 
</H><P style="word-break:break-all;">
MQTRRDALKSAAAITLLGGLAGCASMARPIGTGDLINTVRGPIPVSEAGFTLTHEHICGSSAGFLRAWPEFFGSRKALAEKAVRGLRHARSAGVQTIVDVSTFDIGRDVRLLAEVSRAADVHIVAATGLWFDPPLSMRMRSVEELTQFFLREIQHGIEDTGIRAGIIKVATTGKATPFQELVLKAAARASLATGVPVTTHTSASQRDGEQQAAIFESEGLSPSRVCIGHSDDTDDLSYLTGLAARGYLVGLDRMPYSAIGLEGNASALALFGTRSWQTRALLIKALIDRGYKDRILVSHDWLFGFSSYVTNIMDVMDRINPDGMAFVPLRVIPFLREKGVPPETLAGVTVANPARFLSPTVRAVVTRSETSRPAAPIPRQDTER
</P><H style="font-size:14px;font-weight: bold;">
DNA SEQUENCE
</H><P>
at gcaaacgaga agagatgcac ttaagtctgc ggccgcaata actctgctcg gcggcttggc tgggtgtgca agcatggccc gaccaatcgg tacaggcgat ctgattaata ctgttcgcgg ccccattcca gtttcggaag cgggcttcac actgacccat gagcatatct gcggcagttc ggcgggattc ctacgtgcgt ggccggagtt tttcggtagc cgcaaagctc tagcggaaaa ggctgtgaga ggattacgcc atgccagatc ggctggcgtg caaaccatcg tcgatgtgtc gactttcgat atcggtcgtg acgtccgttt attggccgaa gtttcgcggg ccgccgacgt gcatatcgtg gcggcgactg gcttatggtt cgacccgcca ctttcaatgc gaatgcgcag cgtcgaagaa ctgacccagt tcttcctgcg tgaaatccaa catggcatcg aagacaccgg tattagggcg ggcattatca aggtcgcgac cacagggaag gcgaccccct ttcaagagtt ggtgttaaag gcagccgcgc gggccagctt ggccaccggt gttccggtaa ccactcacac gtcagcaagt cagcgcgatg gcgagcagca ggcagccata tttgaatccg aaggtttgag cccctcacgg gtttgtatcg gtcacagcga tgatactgac gatttgagct acctaaccgg cctcgctgcg cgcggatacc tcgtcggttt agatcgcatg ccgtacagtg cgattggtct agaaggcaat gcgagtgcat tagcgctctt tggtactcgg tcgtggcaaa caagggctct cttgatcaag hgcgctcatcg accgaggcta caaggatcga atcctcgtct cccatgactg gctgttcggg ttttcgagct atgtcacgaa catcatggac gtaatggatc gcataaaccc agatggaatg gccttcgtcc ctctgagagt gatcccattc ctacgagaga agggcgtccc gccggaaacg ctagcaggcg taaccgtggc caatcccgcg cggttcttgt caccgaccgt gcgggccgtc gtgacacgat ctgaaacttc ccgccctgcc gcgcctattc cccgtcaaga taccgaacga tga
</P><H style="font-size:14px;font-weight: bold;">
PROTEIN SEQUENCE
</H><P style="word-break:break-all;">
MQTRRDALKSAAAITLLGGLAGCASMARPIGTGDLINTVRGPIPVSEAGFTLTHEHICGSSAGFLRAWPEFFGSRKALAEKAVRGLRHARSAGVQTIVDVSTFDIGRDVRLLAEVSRAADVHIVAATGLWFDPPLSMRMRSVEELTQFFLREIQHGIEDTGIRAGIIKVATTGKATPFQELVLKAAARASLATGVPVTTHTSASQRDGEQQAAIFESEGLSPSRVCIGHSDDTDDLSYLTGLAARGYLVGLDRMPYSAIGLEGNASALALFGTRSWQTRALLIKALIDRGYKDRILVSHDWLFGFSSYVTNIMDVMDRINPDGMAFVPLRVIPFLREKGVPPETLAGVTVANPARFLSPTVRAVVTRSETSRPAAPIPRQDTER
</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
3)Genetically engineered E.Colibacteria
</H><P>
In this project, we will use E. Coli to construct genetically engineered bacteria which can secrete organophosphorus-degradation enzyme opdA protein, to  eliminate the pesticides.
</P><P>
Genetically engineered bacteria are bacteria which can channel target gene into bacteria to express the genes and produce required protein. 
</P><P>
Currently, the mostly used genetically engineered bacteria all over the world are still E. Coli.  E. Coli have explicit genetic background, fast growth rate, limited antibiotics resistance, Thus, E. Coli, are easy to be used in any production magnitude from laboratory to industry production. (For example: scientists introduced human insulin gene into E. Coli genome. E. Coli can express functional human insulin protein, which is used as a medicine for diabetes treatment. Human insulin production from E. coli has been applied to industry and this method is also widely used in biotech industry for other drug purposes In 1981, human insulin gene products were put into market and solved the problem of lack of insulin sources).
</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
4）RNA Thermometer 
</H><P>
An RNA thermometer (or RNA thermosensor) is a temperature-sensitive non-coding RNA molecule which regulates gene expression. RNA thermometers often regulate genes required during either a heat shock or cold shock response. In general, RNA thermometers operate by changing their secondary structure in response to temperature fluctuations. This structural transition can then expose or occlude important regions of RNA such as a ribosome binding site, which then affects the translation rate of a nearby protein-coding gene.
</P><DIV> </DIV><P>
Below is a schematic diagram of Thermometer RNA:
</P><DIV> </DIV><DIV> </DIV><P>
In this project, the Thermometer RNA we use has a sensitive temperature of 32℃, which means in environment above 32C, RNA translation process can be started. Its DNA sequence is as follows:
</P><P style="word-break:break-all;">
Ccgggcgcccttcgggggcccggcggagacgggcgccggaggtgtccgacgcctgctcgtccagtctttgctcagtggaggattactag
</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
5)ompA signal peptide
</H><P>
In this project, to secrete the opdA enzyme from inside the bacteria to outside, an ompA signal peptide is introduced into the construction.  By adding this secreting peptide, OpdA is secreted outside the E. Coli, without its accumulating inside. 
</P><P>
Signal peptide often refers to a N-terminal amino acid sequence which is used to direct the trans-membrane process in newly synthesized proteins. There is a segment of RNA area programming hydrophobic amino acid sequence which is generally behind the start codon. This amino acid sequence is called signal peptide sequence, which introduces  protein into sub-cellular organelles which contain different membrane structures. If the function of this signal peptide is to secrete protein outside the cell walls, this signal peptide is called secretion signal peptide.
</P><P>
As we know, on natural conditions, ompA which exists inside Agrobacterium radiobacter bacteria has its own signal peptide. However, this original signal peptide is only used by Agrobacterium radiobacter and is not necessarily functional inE. Coli. Therefore, we need to select an E. colisignal peptides to achieve our plan.
</P><P>
The ompA signal peptide used in this project is one of the E. coli secreting signal peptides and can lead  secretion of its downstream protein outside of its host E. coli. The DNA sequence of this signal peptide is as follows:
</P><P style="word-break:break-all;">
Atgaaaaaaaccgctatcgcgatcgcagttgcactggctggtttcgctaccgttgcgcaggcc
</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
6)Suicide gene ccdb
</H><P>
In this project, we consider not only how to use genetically engineered bacteria to eliminate organophosphorus, but also after organophosphorus is eliminated, how we should deal with the rest genetically engineered bacteria. Genetically engineered bacteria, after all, come from E. coli. We don’t want them released into environment or remaining on the surface of fruits and vegetables. Thus we must design a method to eliminate them. Consequently, we decide to design a suicide gene into our genetically engineered bacteria, which is inactivated under normal condition and only activated under special environments to eliminate the bacteria and avoid “secondary pollution”.
</P><P>
To achieve the above goal, we select suicide gene ccdb. ccdB is a known toxin system, which exists in pathogenic E. coli F plasmid. The ccdB gene programs a toxin protein CcdB. On conditions where there is a lack of antitoxin, CcdB poisons gyrase inside cells to interfere with the DNAsynthesis and damage host cells. The ccdb sequence we use in this project is as follows:
</P><P style="word-break:break-all;">
Atgcagtttaaggtttacacctataaaagagagagccgttatcgtctgtttgtggatgtacagagtgatattattgacacgcccgggcgacggatggtgatccccctggccagtgcacgtctgctgtcagataaagtctcccgtgaactttacccggtggtgcatatcggggatgaaagctggcgcatgatgaccaccgatatggccagtgtgccggtctccgttatcggggaagaagtggctgatctcagccaccgcgaaaatgacatcaaaaacgccattaacctgatgttctggggaatataa
</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
7)RecA(SOS) promoter
</H><P>
In the project we chose a RecA(SOS) promoter to control to suicide gene ccdB to express. When the bacteria receive ultraviolet lights, DNA injurｙ activate the RecA repairing system. The system can activate RecA(SOS) promoter to drive the downstream ccdB suicide gene’s transcription. In this way, we can control our engineering bacteria by eliminating them with ultraviolet rays to avoid secondary pollution.
</P><P>
The ccdb sequence we use in this project is as follows:
</P><P style="word-break:break-all;">
Aacaatttctacaaaacacttgatactgtatgagcatacagtataattgcttcaacagaacatattgactatccggtattacccggcatgacaggagtaaaaatggctatcgacgaaaacaaacagaaagcgttggcggcagcactgggccagattgagaaacaatttggtaaaggctccatcatgtaataa
</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
8)The design of the biobricks in this project referred to a lot of information from other igem teams in past years and from other scientific projects in the society. The information was coordinated and integrated in this project, and formed the research design and contents of this project.
</H><P>
(1)The strong initiation sequence used in this project referred to BB_J23106 from igem
</P><P>
(2)The RNA thermometer sequence used in this project referred to BB_K115017 from igem
</P><P>
(3)The organophosphorus-degradation enzyme opdA gene sequence used in this project referred to BB_K21509, BB_K215091 and BB_K1010008 from igem
</P><P>
(4)The repair enzyme induced promoter  RecA (SOS) sequence used in this project referred to BB_J22106 from igem
</P><P>
(5)The suicide gene ccdB sequence used in this project referred to BB_K145151 and BB_K1010007 from igem
</P><P>
(6)The ompA signal peptide sequence used in this project referred to sequence with the code AJ617284.1 from GenBank
</P></DIV></DIV></DIV><DIV class="wz_pb"><DIV class="mainTitleHeader"><A name="t4"><P style="font-weight: bold;">Experiment&amp;Steps</P></A></DIV><DIV class="mainContainer"><DIV><H style="font-size:14px;font-weight: bold;">
Protocols on basic experiment steps
</H><DIV> </DIV><H style="font-size:14px;font-weight: bold;">
Plasmid extraction (using AXYgene kit)
</H><P>
1.Collect bacteria from　1-3ml bacteria solution, centrifuge 30” at 10,000rpm 
</P><P>
2.Pour away supernatant and resuspend in 250μl S1 solution
</P><P>
3.Add 250μl S2 solution, mix the solution gently, incubate at room temperature (RT) for no more than 5’
</P><P>
4.Add 350μl S3 solution, mix the solution gently, centrifuge 12’at 12,000rpm
</P><P>
5.Transfer supernatant into nucleic acid reaction column, incubate at RT for 1’ 
</P><P>
6.Centrifuge the column 30” at  12,000rpm, and trash the elute
</P><P>
7.Add 500μl W1 washing buffer into column, and centrifuge 30” at 12,000rpm, trash the elute  
</P><P>
8.Add 750μl W2 washing buffer into column, and centrifuge 30” at 12,000rpm, trash the elute
</P><P>
9.Repeat step 8 once more 
</P><P>
10.Centrifuge 1’ at 12,000rpm, and completely remove residual W2
</P><P>
11.Transfer  column into new 1.5ml centrifuge tube, add 50-100μl water, and incubate at RT for 1’ 
</P><P>
12.Centrifuge 1’ at 12,000rpm, and collect the elute
</P><H style="font-size:14px;font-weight: bold;">
PCR amplification Protocol:
</H><P>
1. PCR reaction system: (25μl)
</P><P>
10 X Taq buffer           2.5μl
</P><P>
 5’ primer        1μl
</P><P>
3’ primer      1μl
</P><P>
dNTP                 2μl
</P><P>
 Template     1μl
</P><P>
Taq            0.2μl
</P><P>
 ddH2O  added up to 25μl
</P><P>
2. PCR reaction conditions:
</P><P>
Denature for 5’ at 95°C;
</P><P>
Denature for30” at 95°C℃;
</P><P>
Anneal for 30” at  55°C℃;
</P><P>
Extension at 72C,℃, extension time is determined by the length of extension sequence;
</P><P>
Return to (2), and cycle 29 times;
</P><P>
Extension for 10’ at 72°C, store at 4°C;
</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
PCR products gel retraction
</H><P>
1.Under UV light, carefully cut agarose block containing target DNA, put it into 1.5ml centrifuge tube;
</P><P>
2.Add two-times volume of  solution A (add 200μl solution A into each 100mg agarose gel), heat it in heat block at 80°C until the gel is completely dissolved, mix evenly and cool to room temperature;
</P><P>
3.Put dissolved gel solution into gel recycle column, and centrifuge 30” at 10,000rpm;
</P><P>
4.Add 450μl washing solution B into column, and centrifuge 1’ at 12,000rpm, trash the elute;
</P><P>
5.Repeat step 4 once more;
</P><P>
6.Transfer column into a clean EP tube, add 30μl elution buffer C into the column;
</P><P>
7.Incubate at 37°C for 5’, centrifuge 2’ at 12,000rpm, and collect the eluted DNA solution;
</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
Double enzymatic digestion of plasmid and PCR product
</H><P>
Double enzymatic digestion on PCR product or plasmid DNA, the digestive products are purified and stored for future use. Enzymatic digestion conditions are as follows:
</P><P>10xH Buffer                  5μl</P><P>Enzyme 1                2μl</P><P>Enzyme 2                2μl</P><P>Template (plasmid or PCR product)                 less than 1μg</P><P>ddH2O         added upto 50μl</P><P>37°C, 3-4 hours.</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
Ligation
</H><P>
Ligation of linear plasmid and inserted fragment double digested with same pair of enzymes, the ligation protocol is as follows:
</P><P>10 x T4 ligase buffer          2μl</P><P>Linear plasmid               3μl</P><P>Inserted fragment        3-10 times Mol of plasmid </P><P>T4 DNA ligase                    1μl</P><P>ddH2OSterile super-pure water            added upto 20μl</P><P>16°C , 4 hours.</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
Protocol to prepare competent cells
</H><P>
(1)Inoculate a single colony of E. coli (or the ratio of E. coli solution and culture medium is 1:1000) in to 1000ml LB culture medium, culture at37°C for 4 hours at 200rpm.
</P><P>(2)Collect E. coli into 50ml centrifuge tube, cool the tube  on ice for 10’, and centrifuge for 10’ at 4,000rpm.
</P><P>(3)Remove the supernatant completely, re-suspend the E. coli cells with pre-cooled 0.1M calcium chloride solution , do not use oscillator, pool resuspend E. coli solution into one centrifuge tube.
</P><P>(4)Centrifuge for 10’ at 4,000rpm.</P><P>(5)Remove the supernatant completely, re-suspend the E. coli cells using 10ml pre-cooled 0.1M calcium chloride solution, add glycerin to a final concentration of 10%, and split them into 100 µl/tube and store them at -80°C.
</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
Transformation of ligation product
</H><P>
(1)Thaw 0.1ml competent cells on ice;
</P><P>(2)Add 10μl ligation product, and incubate on ice for 30’;
</P><P>(3)Heat shock at 42°Cfor 90”, and immediately incubate it back on ice for 2’;
</P><P>(4)Add 0.8ml LB culture, and slowly shake at 37°C for 60’;</P><P>(5)Pipet ~100-200 μl solution onto LB pate coated with certain antibiotic (eg. 45μg/ml chlorampenicol), plate evenly;
</P><P>
(6)Place the plate upside down, 37° Covernig,The next day, select positive single cloning colonies;
</P></DIV></DIV></DIV><DIV class="wz_pb"><DIV class="mainTitleHeader"><A name="t5"><P style="font-weight: bold;">Results&amp;Future</P></A></DIV><DIV class="mainContainer"><DIV><H style="font-size:14px;font-weight: bold;">
Project Achievement
</H><DIV style="height:160px"><P>
The project designed a biology module which integrates multiple bio-function components. This module can make the genetically engineered bacteria induced by temperature to produce organophosphorus-degradation enzyme opdA to eliminate the organophosphorus pesticides pollution in the environment. Later the bacteria is killed by activated suicide gene under UV light, which avoids the secondary pollution posed by bacteria.
</P></DIV><P>
The project using synthetic biology successfully assembled this biology module, and transformed into E. coli, which can proliferate enormously.
</P></DIV><DIV><H style="font-size:14px;font-weight: bold;">
Future Plans for Project
</H><P>
Evaluate the function of the engineered E. coli. Whether the engineered E. coli can eliminate the pesticide? Whether UV or sunlight can induce the self-death of the engineered E. coli? If it works, what the best working condition?
</P><P>
Optimize the growth conditions for E. coli which contain this biology module, and determine the best culture conditions for manufacture scale growth.
</P><P>
On the basis of this biology module, the project further extend its functions. For example, integrate other degradation enzyme systems for other pesticides or other organic pollutants, or combine different enzymes to degrade multiple pollutants.
</P><P>
Failure experience (difficulties we met) and our solutions:
</P><P>
1)Because the linear plasmid provided by IGEM had a low purity, using the plasmid construction method recommended by IGEM protocol caused large quantities of false positives in the process and interfered greatly to select positive clones. Additionally, because the linear plasmid provided by IGEM cannot reproduce, the quantities used in experiment and times of experiment repeatswere severely restricted.
</P><P>
Solution: Counseled with other IGEM, we gave up  the linear plasmid provided by IGEM and turned to J04450 plasmid to carry out plasmid extraction, and recieved large amounts of J04450 . By carrying out EcoRI and PstI double digestion on and cutting off mRFP, and replacing with our biology module, we finally successfully constructed plasmid which contained the biology module we designed.
</P><P>
2)Ligation of multiple segments, our biology module contains four different segments, which are F1, F2, F3 and F4 with a total length of over 1800bp. Given current DNA synthetic technology, it is inefficient and takes a long time to synthesize a length of over 1800bp. Moreover, ligating four segments to plasmid pSB1C3 also requires four repetitive cloning transformation processes, which takes a lot of time and resources.
</P><P>
Solution: We designed methods of separately synthesizing four segments, which greatly increased synthesizing speed and saved time. Then, we utilized the enzyme digestion site pre-designed at each ends of each segment, to achieved the mutual connection among segments without vector. Then we used PCR technology without repeated ligation, transformations, clone selection processes to gain final F1+2+3+4 segments and clone them to pSB1C3 plasmid at a single step.
</P></DIV></DIV></DIV><DIV class="wz_pb"><DIV class="mainTitleHeader"><A name="t6"><P style="font-weight: bold;">Parts</P></A></DIV><DIV class="mainContainer"><H style="font-size:15px;font-weight: bold;"> Our submission parts: </H><DIV> </DIV><H style="font-size:15px;font-weight: bold;"> BBa_K1667005</H><P>
This part includes two individual DNA domains. Constitutive promoter tunes the expression of downstream opdA gene with further help from RNA thermometer. RNA thermometer provides a temperature sensitive post-transcriptional regulation on opdA gene, which initiate the opdA translation around 32°C. OmpA signal peptide guides the secretion of opdA protein to the outside of host strain. Then opdA enzyme specifically degrades organophosphorus pesticide appeared, through hydrolysis. Upon UV light, RecA（SOS）promoter drives the transcription of downstream ccdB suicide gene, whose protein expression interferes DNA sysnthesis and lead to cell dealth. In this way, we can wipe out our genetic engineered bacteria by giving UV lights under manual control to avoid secondary pollution. Without UV light, RecA promoter won’t be activated, so the normal growth and activity of genetic engineering bacteria will be preserved well to release functional opdA protein under temperature control. </P><DIV> </DIV><H style="font-size:15px;"> More information please see our parts form:  </H><DIV> </DIV><TABLE cellspacing="0" cellpadding="0" style="width: 100%;border:2px solid #414141; "><THREAD><TBODY><TR><TH style="width: 10%;border-right:1px solid #414141;"> Name</TH><TH style="width: 20%;border-right:1px solid #414141;"> Type</TH><TH style="width: 40%;border-right:1px solid #414141;"> Description</TH><TH style="width: 10%;"> Length</TH></TR></TBODY></THREAD><TBODY><TR><TD style="text-align:left;border-top:1px solid #414141;;border-right:1px solid #414141;"><A style="color:#002BB8;" href="http://parts.igem.org/Part:BBa_K1667005">BBa_K1667005</A></TD><TD style="text-align:left;border-top:1px solid #414141;border-right:1px solid #414141;"> DNA</TD><TD style="text-align:left;border-top:1px solid #414141;border-right:1px solid #414141;"> OpdA encoding gene with ompA signal peptide</TD><TD style="text-align:left;border-top:1px solid #414141;"> 1776</TD></TR></TBODY></TABLE></DIV></DIV><DIV> </DIV><DIV class="footer"><P>© 2015 Shiyan_SY_China iGEM Team. All rights reserved. 
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>