Team:Shiyan SY China/Notebook
IGEM-SAFETY
HOME
PROJECT
Description
Background
Design
Experiment&Steps
Results&Future
Parts
HUMAN PRACTICES
T-shirt
Biology Speech
IGEM Magazine
Questionnaire
Sina Weibo
Community&Video
SAFETY
Safety Design
Lab security
Safety Delivery
TEAM
Members
Instructors
Attributions
Collaborations
NOTEBOOK
Notebook
You have to believe in yourself. That’s the secret of success.
—— Charles Chaplin
Notebook
November, 2014
It was the first time we heard about iGEM and became interested in it. We wanted to involved into this competition to test our ability and challenge our limits.
December, 2014
We sought advice from International Teenage Competition and Communication Center (ITCCC), where Dr. Mao Yong suggested us that we should establish a team and find a suitable lab or build a high school lab by ourselves.
January, 2015
We discussed with our teacher about building a competition team for iGEM. Although strongly supported by Zhang Shuang, teacher of international department of Liaoning Province Shiyan High School, we can’t build a lab in our high school. Therefore, we need to find a lab outside.
February, 2015
It took so much money to build a lab by ourselves and our school doesn’t have the resource to do so either. So we turned to life science department of Liaoning University for help in February. Professor Wang Qiuyu learned what we wanted, how we loved biology, math and modeling and our wishes to get opportunities to put the theory of synthetic biology into practice. He support our idea and approved that we work together with his team to study synthetic biology.
March, 2015
We used Weibo, QQ and posters to recruit members into our team. Finally we have a team of nine. We divided up work based on individual’s specialty. Some liked biology, some loved math, and also some preferred software. Together, we are IGEM team shiyan_SY_China.
April, 2015
We brainstormed projects in the whole April. We checked past IGEM projectsrace and selected topics which we were truly interested in. Each of us picked 4 or 5 interested topics, which varied from environmental protection, sewage treatment, fat degradation, glowing and power-saving yeast to biodegradable plastics. But teacher Wang warned us that those experiments would be too hard to be finished.
We seized an opportunity to visit a lab in the Chinese Academy of Sciences and discovered a more valuable field for research, food safety! We received explicit support from Dr. Xu Mingkai, from the Chinese Academy of Sciences who finally agreed to be our director . Until now, we got another lab in higher standard with more advanced facilities.
At the end of April, we confirmed our project and divided 9 of us into experimental and software group. Meantime, we began wiki design and scheme. Soon, we decided to work on a special issue about iGEM together with magazine World.
May, 2015
To get safer food, we decided to create an engineered bacteria to degrade farm chemicals, . So when to degrade farm chemicals? What kinds of farm chemicals were sprayed on the vegetable and fruit? Were there any residues? Through investigation on a bigger scale, we found the problem of excessive farm chemicals were so severe that would drop the quality of vegetable and fruit. Nearly 70% of farm chemicals was pesticides. Organophosphorus insecticides accounted for 70%, and 70% of them were supertoxic with high-persistent pesticides.
We were going to edit plasmid in E. colito solve the issue of over dosed pesticide residue.
Would the secondary pollution becaused ? Teacher Xu Mingkai suggested that we could design the segment to make the engineered bacteria suicide. Thus, the secondary pollution wouldn’t show up. We found OPDA enzyme and ccdB in the plasmid repository of iGEM useful to us. On May 11th, we entrusted BGI with total gene sequences synthesis of F1 and F4.
June, 2015
We contacted with director in IDT Beijing office, the official partner of iGEM. We confirmed the sequences of designed, synthesis methods and time, and placed the order to IDT. The ordered synthesized by BGI arrived in Shenyang. We set out to do preliminary work.
The first step we took is to discuss special IGEM issue on World, covering a wide range of matters about ethnic principles, advantages and disadvantages, safety of synthetic biology.
We had revised our project in June.
July, 2015
（2015-07-14）PCR amplification of F1 segment:
Template: segment synthesized by BGI(No. WHC15353 )
Primer: F1F：CCGGAATTCTTTACGGCTA
F1R：CCGAAGCTTGGCCTGCGCAA
Segment length: 201bp
Extension of time: 30s
（2015-07-14）PCR amplification of F2 segment:
Template: segment synthesized by IDT(No.134096554)
Primer: F2F：CCGAAGCTTGGCGATCTGATC
F2R：AAGGACTAGTTCGATTATGCG
Segment length: 1053bp
Extension of time: 1 min 20 s
（2015-07-14）PCR amplification of F3 segment:
Template: segment synthesized by BGI(WHC15421); segment synthesized by IDT(310114)
Primer: F3F：GGGTCTAGAAACAATTTCTAC
F3R：CCCAAGCTTGGTATATCTCC
Segment length: 237bp
Extension of time: 30s
（2015-07-14）PCR amplification of F4 segment:
Template: segment synthesized by BGI(WHC15354)
Primer: F4F：CCCAAGCTTATGCAGTTTAAG
F4R：CAACTGCAGTTATATTCCCCAG
Segment length: 318bp
Extension of time: 30 s
Figure 1. PCR amplification results of F1(track 2,3), F2(track 4,5), F3(track 6,7) and F4(track 8,9) segments. M: DL2000Mm ladder
（2015-07-15）Fragments of F1, F2, F3, F4 recovered from gel extraction:
Figure 2. Gel extraction results of F1(track 1), F2(track 2), F3(track 3) and F4(track 4) segments. M: DL2000Mm ladder
（2015-07-15）Single HindIII enzyme digestion after gel extraction of F1 and F2:
HindIII enzyme Cato No：Takara 1060A
10 X M buffer     5 ul
Hind III          2 ul
F1 orF2 < 1000 ng
dd H2O           added upto 50 ul
Reaction 4 hours at 37°C
（2015-07-16）After digesting by HindIII enzyme, F1 and F2 fragments went through gel extraction process again. Then the two fragments were ligation with T4 ligase:
T4 DNA ligase Cato No：Takara 2011A
10 X T4 buffer          2 ul
T4 DNA ligase          1 ul
F1 + F2 fragments  molecular ratio 1:1
dd H2O         added upto 20 ul
reaction overnight at 16°C
（2015-07-17）PCR amplification of F1+F2 fragments:
Template: ligation product of F1 and F2 segments(diluted 50 times)
Primer Pair：CCGGAATTCTTTACGGCTA
F2R：AAGGACTAGTTCGATTATGCG
Fragment length: about 1260 bp
Extension of time: 1m30s
Figure 3. PCR amplification result of F1+2 segments (track 2,3). M: DL2000 Mm ladde
（2015-07-20）Gel extraction of PCR product from F1+2 segments:
Gel extraction of 1200bp-main band
Figure 4.PCR gel extraction result of F1+2 fragments after PCR gel extraction(track 2,3). M: DL2000Mm ladder
（2015-07-21）After digesting by HindIII enzyme, F3 and F4 fragments went through gel extraction again:
HindIII enzyme Cato No：Takara 1060A
10 X M buffer     5 ul
Hind III          2 ul
F3 or F4 segment      the amount less than 1000 ng
dd H2O               added upto 50 ul
Reaction 4 hours at 37°C
（2015-07-22）After digesting by HindIII enzyme, F3 and F4 fragments went through gel extraction again. Then the F3 and F4 were ligated by T4 ligase:
T4 DNA ligase Cato No：Takara 2011A
10 X T4 buffer          2 ul
T4 DNA ligase          1 ul
F3+ F4 segments  molecular ratio 1:1
dd H2O         added upto 20 ul
reaction overnight at 16°C
（2015-07-23）PCR amplification of F3+F4 segments:
Template: ligation product of F3 and F4 fragments(diluted 50 times)
Primer: F3F：GGGTCTAGAAACAATTTCTAC
F4R：CAACTGCAGTTATATTCCCCAG
Segment length: about 555 bp
Extension of time: 45 s
Figure 5. PCR amplification result of F3+4 segments (track 2). M: DL2000Mm ladder
（2015-07-23）Gel extraction of PCR product from F3+4 segments:
Gel extraction of 555bp main band
Figure 6. PCR gel extraction result of F3+F4 segments (track 2,3). M: DL2000Mm ladder
（2015-07-24）After gel extraction, F1+F1 was single digested  by Spe I enzyme, F3+F4 was single digested by XbaI:
Spe I enzyme Cato No：Takara 1080A
10 X M buffer            5 ul
Spe I                   2 ul
F1+F2 fragments   the amount less than 1000 ng
dd H2O                added upto 50 ul
Reaction 4 hours at 37°C
 
Xba I enzyme Cato No：Takara 1093A
10 X M buffer            5 ul
Xba I                   2 ul
F3+F4 fragments        the amount less than 1000 ng
dd H2O              added upto 50 ul
Reaction 4 hours at 37°C
（2015-07-27）After single digestion by Spe I or Xba I enzyme, F1+2 and F3+4 segments went through gel extraction . Then F1+F2 and F3+F4 were ligated by T4 ligase:
T4 DNA ligase Cato No：Takara 2011A
10 X T4 buffer               2 ul
T4 DNA ligase               1 ul
F1+2 + F3+4 segments  molecular ratio 1:1
dd H2O              added upto 20 ul
reaction overnight at 16°C
（2015-07-28）PCR amplification of F1+2+3+4 segments:
Template: ligation product of F1+2 and F3+4 fragments(diluted 50 times)
Primer: F1F：CCGGAATTCTTTACGGCTA
F4R：CAACTGCAGTTATATTCCCCAG
Extension of time: 1m50s
Figure 7. PCR amplification result of F1+2+3+4 segments (track 2,3). M: DL2000Mm ladder
（2015-07-28）F1+2+3+4 fragment was recovered from gel extraction:
Gel extraction of 1815bp-main band
Figure 8. PCR gel extractionresult of F1+2+3+4 segments (track 2). M: DL2000Mm ladder
（2015-07-29）EcoR I/Pst I double digestion in the segments of F1+2+3+4:
EcoR I enzyme Cato No：1040A
Pst I enzyme Cato No：1073A
10 X H buffer             5 ul
EcoR I                   2 ul
Pst I                     2 ul
F1+2+3+4 segment      the amount less than 1000 ng
dd H2O                added upto 50 ul
Reaction: 4 hours at 37°C
（2015-07-29）Gel extraction after double digestion for the fragments of F1+2+3+4:
Figure 9. Gel extraction results after double digestion in the segments of F1+2+3+4 (track 2). M: DL2000Mm ladder
August , 2015
（2015-08-03）J04450 plasmid transformation:
Transformed J04450 into DH5α competent cells and plate on the LB plate with 45ug/ml chloramphenicol. Cultured them for 24 hours at 37°C and observed the appearance of red bacterial colony.
（2015-08-04）J04450 selection of monoclonal plasmid transformant:
Picked up a red colony (where black circle shown in figure 10 ) and inoculate it into the LB medium with 45ug/ml chloramphenicol. Cultured it for 24 hours at 37°C in shaker.
Figure 10. the photo of J04450 plasmid transformed into DH5α competent cells after 24 hours.
（2015-08-05）J04450 plasmid extraction:
Picked up a red colony (where black circle shown in figure 10 ) and inoculated into the LB medium with 45ug/ml chloramphenicol. Cultured it for 24 hours at 37°C in shaker. Bacterial medium presented a red color (shown in figure 11). Centrifuged bacteria, aspirate the supernatant and collected pellets. Extracted J04450 plasmid DNA using DNA extraction kits (as shown in figure 12).
Figure 11. the bacterial liquid photo of J04450 plasmid cultivated in liquid LB medium after 24 hours.
Figure 12. DNA extraction result of J04450 plasmid (track 1), M：λ-T14Mm ladder; M2：DL2000Mm ladder .
（2015-08-05）EcoR I / Pst I double digestion of J04450 plasmid:
EcoR I enzyme Cato No：1040A
Pst I enzyme Cato No：1073A
Collect the large segment of plasmid in 2200 bp after double digestion, which is the linear pSB1C3 plasmid
10 X H buffer             5 ul
EcoR I                   2 ul
Pst I                     2 ul
J04450 plasmid         the amount less than 1000 ng
dd H2O               added upto 50 ul
Reaction 4 hours at 37°C
Figure 13. EcoR I/Pst I double digestion result (track 2), M：λ-T14 Mm ladder; M2：DL2000 Mm ladder.
（2015-08-06）ligation and transformation of pSB1C3 linear plasmid and F1+2+3+4 fragments:
T4 DNA ligase Cato No：Takara 2011A
10 X T4 buffer               2 ul
T4 DNA ligase               1 ul
F1+2+3+4 fragments              molecular ratio 10
pSB1C3 linear plasmid  molecular ratio 1
dd H2O                        added upto 20 ul
reaction overnight at 16°C
（2015-08-07）ligation product of pSB1C3 linear plasmid and F1+2+3+4 fragments:
ligation product of pSB1C3 linear plasmid and F1+2+3+4 fragments transformed into DH5α competent cells and plated on the LB plate with 45ug/ml chloramphenicol. Cultured them for 24 hours at 37°C and observed the appearance of white bacterial colonies.
（2015-08-08）PCR verification of selected bacterial colonines:
Picked up some white colonies (as shown in the figure 14) and used the bacterial colony as template to do PCR.
Figure 14. the bacterial colony photo of ligation product of pSB1C3 linear plasmid and F1+2+3+4 segments transforming into DH5α competent cells after 24 hours.
bacterial colony PCR operation
Template: a tip of single colony
Primer Pair: F1F：CCGGAATTCTTTACGGCTA
F4R：CAACTGCAGTTATATTCCCCAG
Segment length: about 1815 bp
Extension of time: 1m50s
Figure 15. Selected bacterial colony PCR results. M: DL2000 Mm ladder; track 2-6: 6 various bacterial colony PCR products; track 9: positive control; track 10: negative control.
（2015-08-09）positive transformant in liquid culture:
Picked up three white positive colonies ( as bacterial colony shown in track 2,3,4 of the figure 10 ) and inoculated them v into the LB medium with 45ug/ml chloramphenicol. Cultured them for 24 hours at 37°C on shaker.
（2015-08-10）plasmid extraction of positive colonies:
The bacterial liquids were milk white after cultivated for 24 hours (as shown in figure 16). Centrifuged bacteria, aspirate the supernatant and collected pellets. Extracted plasmid DNA using DNA extraction  kits.
Figure 16. the bacterial liquid photo of three positive colonies cultivated in LB medium after 24 hours.
（2015-08-10）verification of  positive plasmids using double digestion:
EcoR I enzyme Cato No：1040A
Pst I enzyme Cato No：1073A
10 X H buffer             5 ul
EcoR I                   2 ul
Pst I                     2 ul
Positive palsmid transformant  the amount less than 1000 ng
dd H2O                    added upto 50 ul
Reaction 4 hours at 37°C
Figure 17. Verification result of three positive plasmids after double digestion. M: DL2000 molecular mass ladder; track 2: positive plasmid 1 digested by PstI enzyme; track 3: positive plasmid 1 digested by EcoRI + PstI enzymes; track 4: positive plasmid 1 without digestion (negative control); track 5: positive plasmid 2 digested by PstI enzyme; track 6: positiveplasmid 2 digested by EcoRI + PstI enzymes; track 7: positive plasmid 2 without digestion (negative control); track 8 positive plasmid 3 digested by PstI enzyme; track 9: positive plasmid 3 digested by EcoRI + PstI enzymes; track 10: positive plasmid 3 without digestion (negative control).
（2015-08-11）Validation of DNA sequencing of positive plasmids in Sanger way:
Picked up three positive plasmids and delivered them to BGI in Beijing to be sequenced. To ensure the correct sequencing results, way of “bidirectional measure” was applied in the two groups of primers:
F1F：CCGGAATTCTTTACGGCTA
F4R：CAACTGCAGTTATATTCCCCAG
VF2：tgccacctgacgtctaagaa
VR：attaccgcctttgagtgagc
Sequence primers:
Results of sequencing proved clone were correct.
Figure 18. Sequencing results of primer F1F.
Figure 19. Sequencing results of primer F4R.
Figure 20. Sequencing results of primer VF2.
Figure 21. Sequencing results of primer VR.
（2015-08-20）submitted the plasmid:
plasmid was proved by sequencing
pSB1C3-EcoRI-strongpromoter-RNAthermometer-ompA-opdA-RecA(SOS)promoter-ccdB-PstI
Extracted plasmids DNA put into 96 well plate provided by IGEM.
Placed the plate on the clean bench for air drying, then capped it with microplate sealer
Prepared to ship them to IGEM
Figure 22. air-drying of plasmid
After a busy month of August, we had made great progress on the experiment. Eight modules in the website (wiki) were on their way；The first draft of magazine had been set typed. We then proofread the content and layout once more.
We handed in the project overview in August, Final Safety Form  and chose tracks for competition. After reviewing, we re-organized into experimental group, defense and software group, which were in the charge of Liu Yiyang, Zhao Fanglin and Dai Zhehao.
September, 2015
We started to prepare the judge form, PPT and poster. Banner has been submitted. We are going to work on the final checkup of the Wiki.
Work on the final preparation for the competition: make uniform shirts, book airline tickets and hotel; simulate defense.
 
© 2015 Shiyan_SY_China iGEM Team. All rights reserved.
