"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/notebookdata
From 2009.igem.org
<?xml version="1.0" encoding="utf-8"?> <events> <event> <date>29-06-2009</date> <time>9:00 - 17:00 GMT</time> <title>First day in Wet Lab!</title> <image></image> <description><![CDATA[
Today, we grew donor strains of E. coli to commence work on making double mutants. We require a strain of E. coli that is has the transmembrane histidine kinase EnvZ and ribose binding protein functionally deleted. This will be done using P1 lysate. We prepared 6 agar plates with 1.5M LB. Additionally, 0.8% LB agar were prepared and stored at 65C. Work continues tomorrow.
We also made our first batch of competent cells. They are nesting in the -80C freezer. We will need to test them by transforming with EdinBrick 1 (lacZ' in pSB1A2). Fingers crossed, we will have an abundace of blue colonies!
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>30-06-2009</date> <time>9:00 - 17:00 GMT</time> <title>Revival</title> <image></image> <description><![CDATA[ We revived DNA from the registry plates today! DNA for the ompR-controlled promoter (R0082) and EYFP construct (E0430) were rehydrated and trasformed into competent cells previously prepared by C. French. Both BioBrick parts are carried on pSB1A2, hence, cells were left to grow overnight on amp100 agar plates.
Work on the double mutant continues. ]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>01-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Ethics Card Game</title> <image></image> <description><![CDATA[ We did something super fun and meaningful today. We participated in the beta test of a 'card game' that is being jointly developed by Dr Donald Bruce of Edinethics and the Scottish government. The aim of the card game is to stimulate discussion about Synthetic Biology among the general population. There is a possibility that this card game will be introduced in pubs (haha!).  The issues we discussed: - Legal concerns, patents, regulation - Commercialization. Is it acceptable? - Potential dangers (bioweapons, terrorism) - How accessible should the technology be - Applications of the technology in the medical/ environmental field Our P-ompR (R0082) and EYFP (E0430) constructs grew on the ampicillin plates. We picked 3 colonies from each plate and patched cloned them onto a fresh plate (Plate 5).  The clones grew and were transferred into liquid broth in preparation for miniprep. We also transformed our competent cells with EdinBrick1 (Plates, 6, 7 & 8).
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>02-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Our first miniprep!</title> <image></image> <description><![CDATA[ We did miniprep on plasmids revived from registry plates.  After miniprep, samples were analysed by gel electrophoresis. The expected size of PompR (R0082) and the EYFP construct are 108bp and 878bp respectively. Since the gel analysis was unable to conclusively resolve that the PompR is carried by the vector, samples were sent for sequencing. Also, CORRECT SIZE does not reflect CORRECT insert!!!
We also patch cloned LuxAB-transformed colonies from Plates 7 & 8. We added decanal  (aldehyde substrate for luciferase encoded by LuxAB) in the hope of visualizing light. No such luck ): Colonies were cultured overnight in liquid broth in preparation for miniprep tomorrow.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>03-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Our first miniprep!</title> <image></image> <description><![CDATA[ We did miniprep for the LuxAB genes encoding for the Photobacterium phosphoreum luciferase gene.  The expected size of LuxAB insert is 2147bp. Gel analysis revealed no positive result. Once again, decanal was added (to the overnight liquid culture), no light was observed. We have decided to do the transformation again.
Our competent cells prepared on 29-06 turned out to be super competent.
Unfortunately, the double mutant transduction was not successful.
Aims for next week: - TNT/ DNT receptor plasmid received - Nirite-sensitive repressor & promoter in transit - Primers containing standard BioBrick prefix and suffix to be designed - Ligate ompR-controlled promoter and EYFP construct to make composite BioBrick part
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>06-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Work on EYFP</title> <image></image> <description><![CDATA[ The gel analysis of the EYFP construct E0430 (02-07) was not convincing, hence, we decided to digest the miniprep samples (M1004-M1006) with 2 enzymes. We expect that the vector (2079bp) and the insert (878bp) will be separated by gel electrophoresis. Gel analysis revealed no positive results.
PCR was conducted to amplify the EYFP gene.  PCR product detected on gel.
Double-mutant work The transduction of E.coli Bla21 (DE3) with lysates from E.coli JW 3730 (Rbp-) and JW3767 (EnvZ-) gave no KmR colonies (perhaps the phage titre was too low). Decided to introduce the second mutation to E.coli JW 3767. ]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>07-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Our first composite BioBrick part!</title> <image></image> <description><![CDATA[ We sent M1002 (PompR, R0082) and M1005 (EYFP construct, E0430) for sequencing today.
Meanwhile, M1002 was digested with PstI and SpeI and J33202 (carrying lacZ’) was digested with XbaI and PstI. The J33202 BioBrick part was inserted upstream of the PompR. The ligation was labelled L1001. L1001 was incubated overnight at 16C.
Double-mutant work Transformation of E.coli JW3730 with the plasmid pCP20 to remove the Km resistance cassette.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>08-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Engineers got a Cellucidate tutorial</title> <image></image> <description><![CDATA[ We got some help with modelling today. Elaine (Kim’s friend) gave us a tutorial a tutorial on the <A HREF="www.cellucidate.com">Cellucidate software</A>.
Cells were transformed with L1001 and left to incubate on amp100 + b/w overnight. Hopefully, there will be blue colonies when we check back tomorrow.
Double-mutant work Plate transformed colonies on Km (30 ug/l) and Ap plates, 30єC.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>09-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Wish our luck with LuxAB would improve</title> <image></image> <description><![CDATA[ We did more miniprep of P. phosphoreum LuxAB today. Gel analysis: No positive result. This is becoming a little discouraging. We are pressing on…We have been reading up on the luciferase light-emitting system of P. phosphoreum and related bacteria. Vas has suggested cloning the whole operon (LuxCDABFEG). This sounds like a feasible plan.
Perhaps our miniprep technique needs to be improved. We will try out the QIAGEN miniprep kit.
Double-mutant work Only 4 colonies (out of 100) grew on Ap plates but did not grow on Km plates. To remove the plasmid I placed the plates at 37єC (pCP20 has temperature-sensitive replication).
Revive the tubes with E.coli TG1 with a) pNSRR (AmpR); b) pPnir (SmR); c) pNSRR & pPnir (AmpR & SmR).
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>10-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Miniprep of PompR-lacZ' construct</title> <image></image> <description><![CDATA[ We did miniprep of PompR-lacZ’ (L1001) transformed cells using the ‘Spin miniprep kit using a microcentrifuge) by Qiagen.
Gel analysis Expected size of band = 189 + 243 + 2079 = ~2.5kb
1/6 colonies returned positive (M1025). Going to send for sequencing (update: yay, sequencing results returned positive).
Double-mutant work Transduction of E.coli JW3730 with the P1 lysate from E.coli JW3763. Reference: Datsenko, K.A., & Wanner, B. L. (2000). One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. PNAS 97, 6640-6645.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>11-07-2009</date> <time>20:00 till late GMT</time> <title>TEAM DINNER!</title> <image></image> <description><![CDATA[ WE HAD DINNER AT RACHEL'S. YUM YUM. NACHO'S, VEG GRILL AND PAVLOVA. MONOPOLY AFTER (: We need to do this againnnn guys.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>13-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Will we have BioBrick parts by the end of this week???</title> <image></image> <description><![CDATA[ Chris will be away attending a conference this week. We will be left to our own devices, oh no.
Today, we inserted the PRC products of LuxAB (yes more) and Cherry-RFP into pSB1A2 vector (derived from EdinBrick 1). The ligations (L1002, L1003) are incubated overnight at 16C.
Concurrently, we are working to insert the pLac-lacZ’ (from EdinBrick 1) upstream of EYFP (M1005) to create a composite BioBrick part. EdinBrick1 was digested with EcoRI and SpeI and the restriction products were separated by gel electrophoresis. The 650kb band was excised, purified and ligated to the purified restriction products of M1005 + EcoRI + XbaI. Similarly, the ligation was left to incubate at 16C overnight. Update: Transformed cells with ligation, plated on amp100 + b/w agar. No blue colonies observed. Since PompR-lacZ’ transformants produced blue colonies, proving that PompR is a functional promoter, we decided to ligate PompR to the EYFP construct. This saves us ‘one step’ of creating a pLac-EYFP construct.
</font>]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>14-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Slowly but surely</title> <image></image> <description><![CDATA[
Transformed cells with L1002 (LuxAB-Ed1) and L1003 (P650-Ed1) ---> screen for white colonies on amp100+b/w plate.   Transformed cells with pLac-lacZ'-EYFP construct ---> screen for blue colonies on amp100+b/w plate.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>15-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Going back to the start</title> <image></image> <description><![CDATA[ White colonies were detected for the LuxAB-Ed1 and CherryRFP-Ed1 transformants. These were patch cloned and transferred into liquid broth and cultured overnight.
No blue colonies from the pLac-lacZ’-EYFP transformants. In light of this, and that the initial gel analysis of M1005 (EYFP) was dodgy, we decided to patch clone colonies from Plate 3. In the evening, we transferred the cells into liquid broth and cultured them overnight. By doing so, we hope to obtain more E0430 plasmids through miniprep and to know for <b>certain</b> that this is the correct BioBrick part through sequencing.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>16-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Let's get minipreppy!</title> <image></image> <description><![CDATA[ Miniprep of 1) CherryRFP-Ed1 (M1031-M1036), 2) LuxAB-Ed1 (M1037-M1042) and 3) EYFP (M1043-M1048)
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>17-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Sequencing</title> <image></image> <description><![CDATA[ We sent 4 samples for sequencing: - <b>M1035</b> CherryRFP - <b>M1037</b> LuxAB - <b>M1025</b> PompR-lacZ - <b>M1043</b> EYFP construct with RBS and terminators
<b>Update</b>: M1025 and M1043 revealed that plasmids carried the desired inserts whereas M1035 and M1037 returned as vector sequence from EdinBrick1. ]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>20-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Bumper crop of BioBrick parts...hopefully</title> <image></image> <description><![CDATA[ We ran PCR reactions for 7 inserts: <b>P1007</b> TNT.R1 <b>P1008</b> TNT.R3 <b>P1009</b> onr, PTEN reductase <b>P1010</b> Pnir, nitrite-sensitive promoter from N. europa <b>P1011</b> nsrR, repressor of Pnir from N. europa <b>P1012</b> PyeaR, nitrite-sensitive promoter from E. coli
PCR products confirmed by gel analysis for all except <b>P1011</b>. The primers were of incorrect sequence and new primers have been ordered. Confirmed products were purified from solution.
We are also trying to obtain a batch of purifed vector because previous digestion of both vector and insert simultaneously in the same reaction followed by ligation has not proven to be effecient. EdinBrick1 and pSB1A3-RFP were digested with EcoRI and SpeI. The ~2kb fragments were excised and purified. The purified products were analysed by gel electrophoresis. Unfortunately, purification was unsuccessful as no bands could be visualised. Will repeat tomorrow! Oh, a scientist’s life.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>21-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Cloning of not 1, not 2, not 3 but 6 PCR products into BioBrick vector!!! WOW.</title> <image></image> <description><![CDATA[ The PRC products of TNT.R1, TNT.R3, onr and PyeaR were digested with EcoRI and SpeI.
The PCR products of Cherry-RFP and LuxAB were also digested with EcoRI and SpeI so that they can be inserted into a BioBrick compliant vector. This is a repeat of what we have done before as the sequence results for our earlier Cherry-RFP (M1035) and LuxAB (M1037) clones returned with only vector sequence.
Today, we managed to obtain purified Ed-1 vector (see 20-07). Ed-1 was ligated with the restriction products of the aforementioned PCR products.
]]></description> <link><![CDATA[]]></link>
</event> <event>
<date>22-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Of transformations, digestions and ligations...</title> <image></image> <description><![CDATA[ We transformed cells with the ligation products (L1003-L1009) from 21-07 and allowed them to grow on amp100 + b/w plates at 37C overnight. Tomorrow, we will screen for white colonies.
We also commenced work to create the composite BioBrick part PompR-EYFP. PompR (M1002) and EYFP (M1043) were digested to obtain M1002 front vector and M1043 back insert. Unfortunately, after purification, no bands were confirmed by gel electrophoresis. WHY??? Will repeat tomorrow by purifying DNA from gel using glass beads instead of Qiagen kit.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>23-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>We have a batch of pSB1A2 and pSB1A3 vectors!</title> <image></image> <description><![CDATA[
White colonies transformed with pSB1A2 carrying TNT.R1, TNT.R3, onr and PyeaR, LuxAB and Cherry-RFP were patch cloned and transfered into liquid culture.
After three atempts starting 20-07, we finally managed to purify a 7ul batch of pSB1A2 vector from EdinBrick 1 and another batch of pSB1A3 from pSB1A3-RFP.
However, the purification of M1002 vector and M1043 insert using glass beads was unsuccessful. We will explore the option of using fusion PCR.
<b>Fusion PCR</b> We tried cloning BioBrick parts using fusion PCR. The two inserts chosen for our maiden fusion PCR was LuxAB and the PyeaR  nitrite-sensitive promoter. The forward primers of each insert was used together with the reverse primer of pSB1A2 vector in the respective reactions. As templates, the respective ligation reactions were used. The products were confirmed using gel electrophoresis. As expected, PyeaR fusion PCR (P1014) gel analysis revealed a single ~2.1kb band. LuxAB fusion PCR (P1013) gel analysis showed two bands, one of the expected size of ~4kb. As such, P1014 was purified from solution and P1013 will be purified from a gel (stained with SYBR-safe) tomorrow.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>24-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>MmMmMMMMmmmMmore miniprep</title> <image></image> <description><![CDATA[ Miniprep of liquid cultures from yesterday. 5/6 samples look great on the gel anaysis! Again, the LuxAB clone showed an incorrect size. The other 5 samples will be sent for sequencing on Monday.
LuxAB fusion PCR product (P1014) was purified from gel.
PyeaR fusion PCR product (P1013) and P1014 were digested with EcoRI and self-ligated at low concentration, these will be used to transformed cells.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>27-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Busy Bee</title> <image></image> <description><![CDATA[
We performed double digestion on the TNT.R1, TNT.R3, onr, PyeaR and Cherry-RFP clones. The sizes of insert and vector were as expected for each clone. We will proceed to sequence the samples. Check back this Thursday for the results! Fingers crossed.
PCR was performed to screen for double mutants using EnvZ and RbsS primers. One candidate was identified but we will have to order more specific primers to confirm this.
Cells were transformed with ligation L1011 (LuxAB) and L1012 (PyeaR), see 24/07. <b>Note</b> Because the competent cells gave poor results for the last few transformations, the protocol was modified. 150 ul of competent cells were used (instead of 100) and 1350 ul of LB was added in the later step (instead of 900). Total volume 1500 ul. Amount of ligation added to the competent cells: 5 ul. The cells were plated on ampicillin plates. Plating volumes were also altered, 150 ul for the low concentration plate followed by centrifution and removal of 1200ul. The 150ul was resuspended and plates on a high concentration plate.
In other <b>very exciting</b> news, the Trg-EnvZ fusion protein plasmid has arrived in the form of a stab. Courtesy of Dr. Gerald L. Hazelbauer from The Department of Biochemistry, University of Missouri-Columbia. Thanks Lilly Angela for making the arrangements. They were streaked onto amp100 plates in preparation for miniprep. Hopefully this works better that the stab we received from the registry... The sequencing results are questionable and the cells didn’t grow. We’re keen to characterise and improve an existing BioBrick part!
We also contacted the International Campaign to Ban Landmines to obtain information on landmines.
<b>LUNCH AT KB HOUSE</b> The blond girl wasn’t there today. She dishes much bigger portions compared to her college.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>28-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Dear Lux, why won't you work</title> <image></image> <description><![CDATA[
The cells transformed with L1011 (LuxAB fusion) and L1012 (PyeaR fusion) were checked. Strangely, no L1011 transformants grew. We will perform another transformation when we make a new batch of competent cells. 6 colonies of L1012 transformants were patched and cultured in liquid broth overnight in preparation for miniprep.
The plate with cells streak from the stab carrying Trg-EnvZ fusion protein did not grow. Not surprising as we made a <b>boo boo</b> but streaking in on cml100 plates instead of amp100 plates. Will streak more cells!
PCR of the Lux operon (luxCDABFEG did not work, i.e. no bands were detected by gel analysis. We think that this might be due to differences between P. phosphoreum strains. We ordered a new strain and this PCR will be repeated when it arrives.
Primers for Trg-EnvZ fusion protein have been designed.
Cells were transformed with  transformed with the self-ligation of PompR-EYFP fusion PCR products (L1013).
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>29-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Slow day</title> <image></image> <description><![CDATA[
No colonies from L1013 transformants could be found. <b>WE NEED TO MAKE A NEW BATCH OF COMPETENT CELLS…and so we did.</b>
Miniprep performed of the selected colonies (M1055 to M1060). Gel analysis revealed band of expected size. Need to sequence.
Cells from Trg-EnvZ stab growing well. Will patch clone tomorrow.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>30-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>(Small) Success!</title> <image></image> <description><![CDATA[
<b>We have BioBrick Parts! The sequence of TNT.R1, onr and PyeaR have returned positive.</b>
This means that we can proceed to create a composite BioBrick Part of PyeaR (M1052) and J33202. Once this is done, we can characterize the promoter activity of PyeaR. Will start work today!
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>31-07-2009</date> <time>9:00 - 17:00 GMT</time> <title>Transformers</title> <image></image> <description><![CDATA[ We did several transfomations today. <b>L1011</b> LuxAB fusion PCR product <b>L1013</b> PompR-EYFP fusion PCR product <b>L1014</b> PyeaR-lacZ fusion PCR product
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>03-08-2009</date> <time>9:00 - 17:00 GMT</time> <title>Plans for promoter characterization</title> <image></image> <description><![CDATA[ We have decided to characterize the promoters, PompR (R0082) and PyeaR (nitrite-sensitive E. coli promoter), using Jasson Kelly's promoter characterization kit. Tomorrow, the required parts will be revived from the Resgistry plates.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>04-08-2009</date> <time>9:00 - 17:00 GMT</time> <title>Revivvaaal</title> <image></image> <description><![CDATA[ We did miniprep of Trz1 fusion protein carried on its original plasmid and revived  BBa_E0240, pSB3k3+RFP (BBa_J04450)¬¬ and BBa_I20260 from the Registry plates.
The fusion PCR product of LuxAB (P1013) was also digested and self-ligated. As a little experiment (because LuxAB cloning has proved to be much more difficult than expected), ligation reactions with 3 different volumes of DNA were set up: 8ul, 4ul and 2ul.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>05-08-2009</date> <time>9:00 - 17:00 GMT</time> <title>LumP, new BioBrick part</title> <image></image> <description><![CDATA[ The LuxAB ligation (from yesterday) was on a confirmatory gel because the band was weak after the digestion.
Cells transformed with the LuxAB ligations (8ul, 4ul, 2ul) were plated twice (100ul and 900ul). Hopefully, something will grow this time.
LumP (lumazine) PCR product was purified. The purified product was digested, heat treated and ligated to pSB1A3 vector. Again, we tried different amount of insert DNA (2.5ul and 4.0ul) to see which reaction gave more efficient transformation.
]]></description> <link><![CDATA[]]></link>
</event>
<event> <date>06-08-2009</date> <time>9:00 - 17:00 GMT</time> <title>Sequence results</title> <image></image> <description><![CDATA[ Some sequence results have returned. We have successfully made a PompR-EYFP construct and cloned PyeaR promoter by fusion PCR. Unfortunately, we still no luck with LuxAB. Only vector sequence returned. We also have not been able to transform cells with PyeaR-lacZ compostite BioBrick part by conventional ligation. We will do fusion PCR using the ligation as the template at a later date.
]]></description> <link><![CDATA[]]></link>
</event>
</events>
Retrieved from " http://2009.igem.org/Team:Edinburgh/notebookdata "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
