<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_biology_biobricks"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Edinburgh/biology(biobricks)">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/biology(biobricks)&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/biology(biobricks)&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/biology(biobricks)&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/biology(biobricks)" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/biology(biobricks)</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main"><DIV id="leftspace"><DIV id="Upgradebox"><DIV id="Upgradenote" align="left">
                      Our WIKI is best viewed with (click title):
                      * <A target="_blank" style="color:#00008B;text-decoration:underline;" href="http://www.microsoft.com/windows/Internet-explorer/default.aspx">Internet Explorer 8</A>
                      * <A target="_blank" style="color:#00008B;text-decoration:underline;" href="http://www.mozilla.com/en-US/">Firefox 3</A>
                      * <A target="_blank" style="color:#00008B;text-decoration:underline;" href="http://get.adobe.com/flashplayer/">Adobe Flash Player 10</A>
                      If you encounter any visual problems please upgrade your browser to the latest version in order to get the most out of our WIKI. Thank you!
                    </DIV></DIV></DIV><DIV id="mainmenu"><DIV id="menuitem"><A href="https://2009.igem.org/Team:Edinburgh/biology%28overalldescription%29"> Overall Description &amp; Design </A></DIV><DIV id="menuitem"><A href="https://2009.igem.org/Team:Edinburgh/biology%28tntsensing%29"> TNT-Sensing Pathway</A></DIV><DIV id="menuitem"><A href="https://2009.igem.org/Team:Edinburgh/biology%28nitritenitratesensing%29"> Nitrite/Nitrate-Sensing Pathway</A></DIV><DIV id="menuitem"><A href="https://2009.igem.org/Team:Edinburgh/biology%28biobricks%29"> Biobrick Parts </A></DIV><DIV id="menuitem"><A href="https://2009.igem.org/Team:Edinburgh/biology%28results%29"> Results </A></DIV><DIV id="menuitem"><A href="https://2009.igem.org/Team:Edinburgh/biology%28solvedproblems%29"> Problem Solving and Tips </A></DIV><DIV id="menuitem"><A href="https://2009.igem.org/Team:Edinburgh/biology%28materialsandmethods%29"> Materials and Methods </A></DIV><DIV id="menuitem"><A href="https://2009.igem.org/Team:Edinburgh/biology%28references%29"> References </A></DIV></DIV><DIV id="Edinburghcontainer"><DIV id="contentmenu"><UL id="nav" class="dropdown dropdown-horizontal"><LI><A href="https://2009.igem.org/Team:Edinburgh">Home</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/biology%28overalldescription%29" class="dir">Biology</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/biology%28overalldescription%29">Overall Description and Design</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/biology%28tntsensing%29">TNT-Sensing Pathway</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/biology%28nitritenitratesensing%29">Nitrite/Nitrate-Sensing Pathway</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/biology%28biobricks%29">Biobrick Parts</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/biology%28results%29">Results</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/biology%28solvedproblems%29">Problem Solving and Tips</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/biology%28materialsandmethods%29">Materials and Methods</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/biology%28references%29">References</A></LI></UL><LI><A href="https://2009.igem.org/Team:Edinburgh/modelling%28overalldescription%29" class="dir">Modelling</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/modelling%28overalldescription%29">Overall Description</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/modelling%28generegulatorynetwork%29">Gene Regulatory Network</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/modelling%28reallifemodelling%29">Real Life Modelling</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/reallifeapplication%28scaleup%29">Scale Up</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/modelling%28results%29">Results</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/modelling%28references%29">References</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/ethics%28publicperception%29" class="dir">Underlying Philosophy</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/ethics%28publicperception%29">Public Perception</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/ethics%28legislationissues%29">Legislation issues</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/ethics%28biosafety%29">Biosafety</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/ethics%28summary%29">DEMOCS Card Game</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/newinformatics%28introduction%29" class="dir">Informatics</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/newinformatics%28introduction%29">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/newinformatics%28globetutorial%29">Globe Tutorial</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/newinformatics%28igemwikhacks%29">iGEM WIKI Hacks</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/newinformatics%28conclusions%29">Blog Entry</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/team%28teammembers%29" class="dir">Team</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/team%28teammembers%29">Team Members</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/team%28advisors%29">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/team%28supervisors%29">Supervisors</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/team%28gallery%29">Gallery</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/team%28contacts%29">Contacts</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/team%28acknowledgements%29">Acknowledgements</A></LI></DIV><FONT color="#323131" style="font-size:14px;float:left;margin-left:20px;margin-top:20px;"><B>Biology - Biobrick Parts</B></FONT><DIV id="Edinburghcontent"><DIV id="abstarct" style="background-color:#e4f5ca;border:1px solid #595151;width:755px;height:180px;margin-top:10px;margin-left:20px;font-size:11px;text-align:justify;padding-left:5px;padding-right:5px;padding-top:5px;padding-bottom:5px;"><B> Personal note </B>
We worked really hard over this summer, but don't think it was all work and no play. One
of the highlights was making our promotional video clip. If you haven't already seen it
check it out and it will hopefully make your day! For other activities check our Gallery!

<FONT color="green" style="float:right">Vasilis</FONT></DIV><DIV id="text" style="margin-left:20px;margin-top:10px;padding-bottom:15px;"><CENTER><B><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2009&amp;group=Edinburgh">Click here</A> to view our parts on the Registry or click on the links below the images of each part</B></CENTER><DIV id="left" style="width:40%;padding-bottom:45px;height:320px;"><B>Quick links:</B><A href="#tnt">tnt.R1</A> (BBa_K216002)<A href="#tnt">tnt.R3</A> (BBa_K216003)<A href="#fusion">trz fusion protein</A> (BBa_K216004) <A href="#ompc">ompC Promoter</A> (BBa_R0082) <A href="#enhanced">Enhanced Yellow Fluorescent Protein</A> (BBa_E0430)<A href="#onr">onr</A> (BBa_K216006) <A href="#year">yeaR promoter</A> (BBa_K216005) <A href="#luxab">luxAB-gfp fusion protein </A>(BBa_xxxx) <A href="#luxabxeno">luxAB from Xenorhabdus luminescens</A> (BBa_K216008)<A href="#lump">lump</A> (BBa_K216007) <A href="#luxcde">luxCDE from <I>X. luminescence</I></A> (BBa_xxxx) <A href="#nsrr">nsrR from Nitrosomonas Europaea</A> (BBa_xxxx) <A href="#constitutive">Constitutive promoter </A> (BBa_J23105) <A href="#pompc">PompC-eyfp-onr</A> (BBa_K216012) <A href="#pompceyfp">PompC-eyfp</A> (BBa_K216011) <A href="#pompclacz">PompC-lacZ’</A> (BBa_K216010) <A href="#pyearlacz">PyeaR-lacZ’</A> (BBa_K216009) <A href="#pyearrenilla">PyeaR-Renilla luciferase</A> (BBa_K216014) <A href="#pyearfirefly">PyeaR-firefly luciferase</A> (BBa_K216015) <A href="#tntr1trz">tnt.r1-trz</A> (BBa_K216013) </DIV><DIV id="right" style="width:59%;padding-bottom:75px;height:220px;"><B>Key</B> - new BioBrick
    - existing BioBrick - hypothethical BioBrick
    - composite BioBrick - fixed BioBrick - functional and/or characterized BioBrick</DIV></DIV><DIV id="text" style="margin-left:20px;margin-top:10px;padding-bottom:15px;"><TABLE cellspacing="0" cellpadding="0"><TBODY><TR><TH><CENTER>Biobrick Name</CENTER></TH><TH><CENTER>Key</CENTER></TH><TH><CENTER>Description</CENTER></TH></TR><TR><TD><CENTER><B><A name="tnt">tnt.R1 &amp; tnt.R3</A> (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216002">BBa_K216002</A>&amp;<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216003">BBa_K216003</A>)</B></CENTER></TD><TD>TNT.R1 and TNT.R3 are computationally derived ligand receptors specific for TNT. The ribose-binding pocket of ribose-binding protein, a member of the E. coli periplasmic binding protein (PBP) family, was reconfigured so that each receptor recognizes TNT instead of the wild-type ligand.
<FONT style="font-size:10px">Looger, L. L., Dwyer, M. A., Smith J. J., and Hellinga, H. W. Computational design of receptor and sensor proteins with novel functions. Nature 423, 185-190 (2003).</FONT></TD></TR><TR><TD><CENTER><B><A name="fusion">trz fusion protein</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216004">(BBa_K216004)</A></B></CENTER></TD><TD>This hybrid protein contains the periplasmic and transmembrane domains of the chemoreceptor Trg and the cytoplasmic domain of the osmosensor EnvZ. Upon interaction with the TNT-TNT.R1/R3 complex, Trg-EnvZ undergoes a conformational change and autophosphorylates. Subsequently, it phosphorylates the second messenger ompR.

This part was previously submitted into the Registry <A href="http://partsregistry.org/wiki/index.php/Part:BBa_J58104">(BBa_J58104)</A>. However, the sequence information is inconsistent. We requested stabs from the Registry, but failed to obtain the correct construct. Hence, we contacted the following authors and they kindly sent us plasmids carrying the construct.
<FONT style="font-size:10px">Baumgartner, J. W., Kim, C., Brisette, R. E., Inoue, M., Park, C., and Hazelbauer, G. L. Transmembrane signalling by a hybrid protein: communication from the domain of chemoreceptor Trg that recognises sugar-binding proteins to the kinase/phosphatase domain of osmosensor EnvZ. Journal of Bacteriology, 1157-1163 (1994).</FONT></TD></TR><TR><TD><CENTER><B><A name="ompc">ompC promoter</A><A href="http://partsregistry.org/Part:BBa_R0082">(BBa_R0082)</A></B></CENTER></TD><TD>The <I>E. coli</I> EnvZ-OmpR two-component system is a well-characterized signaling pathway. EnvZ, a membrane protein, regulates the levels of phosphorylated OmpR (OmpR-P), which in turn regulates gene transcription. The best-studied genes regulated by this system are <I>ompF</I> and <I>ompC</I>. There are several OmpR binding sites at the <I>ompF</I> and <I>ompC</I> promoters.

In our project, a fusion protein (Trz, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216004">BBa_K216004</A>) carrying the EnvZ histidine kinase activity will phosphorylate ompR in response to TNT binding to a receptor. ompR-P will bind to the upstream binding sequences, thereby controlling genes of interest. The genes that we have chosen to put under the control of the ompR-controlled promoter are those coding for enhanced yellow fluorescent protein and PETN reductase <I>(onr)</I>.

It is believed that the native EnvZ-OmpR senses changes in osmolarity. High osmolarity activates EnvZ, thereby generating more ompR-P that binds to the upstream operator sites of ompC. However, in 2006, Batchelor and Goulian compared the effects of osmolarity and procaine and concluded that procaine activates EnvZ-OmpR signaling whereas osmolarity only has a weak effect on the system.
<A href="#">Click here</A> for the characterization results for this promoter.
<FONT style="font-size:10px">
Batchelor, E., and Goulian, M. Imaging OmpR localization in <I>Escherichia coli</I>. Molecular Microbiology 59(6),1767-78 (2006).

Maeda, S., and Mizuno T. Evidence for multiple Omp-R binding sites in the upstream activation sequence of the ompC promoter in <I>Escherichia coli</I>: a single OmpR-binding site is capable of activating the promoter. J. Bacteriol. 172 (1), 501-503 (1990).
</FONT></TD></TR><TR><TD><CENTER><B><A name="enhanced">enhanced yellow fluorescent protein</A><A href="http://partsregistry.org/Part:BBa_E0430">(BBa_E0430)</A></B></CENTER></TD><TD>In our system, EYFP is expressed in the presence of TNT and under the control of the ompC promoter <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_R0082">(BBa_R0082)</A>. The protein is excited by light emitted by the LuxAB.GFP fusion protein that is produced in the presence of nitrites.
                               </TD></TR><TR><TD><CENTER><B><A name="onr">onr</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216006">(BBa_K216006)</A></B></CENTER></TD><TD><I>onr</I> codes for PETN (pentaerythritol tetranitrate) reductase that degrades components of landmines.
PETN reductase was cloned from Enterobacter cloacae PB2, a strain isolated from explosive-contaminated soil (Binks, 1996). PETN reductase is a small monomeric flavoprotein that reduces a wide variety of electron-deficient multi-nitro compounds including pentaerythritol tetranitrate (PETN), nitroglycerine (glycerol trinitrate, GTN), ethylene glycol dinitrate (EGDN), and trinitrotoluene (TNT). In the case of nitrate esters, the products are nitrite and alcohol; in the case of nitroaromatics such as TNT, the initial product is a hydrode adduct (hydride-Meisenheimer complex), which is further reduced to the dihydride adduct. This then degrades in an unknown way with liberation of nitrite and non-aromatic products. 

<FONT style="font-size:10px">
Binks, P.R., French, C.E., Nicklin, S., &amp; Bruce, N.C. Degradation of pentaerythritol tetranitrate by Enterobacter cloacae PB2. Applied and Environmental Microbiology 62(4), 1214-1219 (1996).</FONT></TD></TR><TR><TD><CENTER><B><A name="year">yeaR promoter</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216005">(BBa_K216005)</A></B></CENTER></TD><TD>Microarray studies have identified the yeaR-yoaG operon among genes whose transcription is induced in response to nitrate, nitrite or nitric oxide. 

Nitrate and nitrite regulate gene expression in anaerobic conditions via the two-component systems NarX-NarL and NarQ-NarP. Typically, Nar-activated genes depend on the oxygen-responsive Fnr transcription activator. However, the transcription from the year-yoaG operon is independent of Fnr activation. This makes the upstream regulatory sequence a good candidate for use as a nitrite-sensor in aerobic conditions. Nitrite is a product of natural TNT degradation. Elevated levels of environmental nitrite will indicate the presence of TNT.

The response to nitrate and nitrite is regulated through the Nar regulatory system while response to nitric oxide acts through the NsrR repressor. The binding sites for phospho-NarL and –NarP activators and that for the NsrR repressor overlap in a region 62 nt upstream of the transcription initiation site. 
<A href="#">Click here</A> for the characterization results for this promoter.
<FONT style="font-size:10px">
Lin, H. Y., Bledsoe, P.J., and Stewart V. Activation of yeaR-yoaG operon transcription by the nitrate-responsive regulator NarL is independent of oxygen- responsive regulator Fnr in <I>Escherichia coli</I> K-12. J Bacteriol. 189(21),7539-48 (2007).
</FONT></TD></TR><TR><TD><CENTER><B><A name="luxab">luxAB-gfp fusion protein</A> (Bba_xxxx)</B></CENTER></TD><TD><I>Photobacterium phosphoreum</I> is a luminescent marine bacterium. It has the ability to produce light via the action of a luciferase enzyme. This enzyme is a heterodimer constituting two homologues, LuxA and LuxB. The reaction catalyzed by luciferase is shown below:
                                     Luciferase
RCHO + FMNH2+ O2 ------------------&gt; RCOOH + FMN + H2O + light

where R is the alkyl residue, FMNH2 is reduced flavin mononucleotide, and FMN is the flavin mononucleotide. 

The aldehyde substrate for the reaction can be produced by enzymes encoded by <I>luxCDE</I> (see below).

The reason why Photobacterium phosphoreum luciferase was chosen over other luciferases is because the light it emits is relatively more intense. The wavelength of light emitted is λ max 478 nm. 

We have decided to make a LuxAB.GFP fusion protein as we hypothesize that this will increase the intensity of light. This hypothesis is founded upon an observation made by Miyawaki. Miyawaki created a fusion protein comprising of coelenterazine luciferase and yellow fluorescent protein. This resulted in a 7-fold increase in luminescence of the construct. We adopted GFP in our system as the activation spectrum closely matches the emission spectrum of <I>Photobacterium phosphoreum</I> luciferase.
<FONT style="font-size:10px">
Mancini, J. A., Boylan, M., Soly, R. R., Graham, A. F., and Meighen, E. A. Cloning and Expression of the <I>Photobacterium phosphoreum</I> Luminescence System Demonstrates a Unique lux Gene Organization. The Journal of Biological Chemistry 263 (28), 14308-14314 (1988).

Daubner, S. C., Astorga, A. M., Leisman, G. B. &amp; Baldwin, T. O. Yellow light emission of vibrio fischeri strain y-1: purification and characterization of the energy-accepting yellow fluorescent protein. Proceedings of the National Academy of Sciences of the United States of America 84, 8912-8916 (1987). 

Miyawaki, A., Bringing bioluminescence into the picture. Nature Methods 4, 616 - 617 (2007).
</FONT></TD></TR><TR><TD><CENTER><B><A name="luxabxeno">luxAB from Xenorhabdus luminescens</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216008">(BBa_K216008)</A></B></CENTER></TD><TD>We worked extremely hard to clone <I>luxAB</I> from Photobacterium phosphoreum, unfortunately, we were unsuccessful. As such, we cloned <I>luxAB</I> from Xenorhabdus luminescens. This was successful- hosts expressing <I>luxAB</I> (from X, luminescens) under the control of the yeaR promoter emitted light in the presence of nitrites and aldehyde substrate (decanal).
</TD></TR><TR><TD><CENTER><B><A name="lump">lump </A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216007">(BBa_K216007)</A></B></CENTER></TD><TD>LumP encodes for lumazine protein. Lumazine interacts with the <I>Photobacterium phosphoreum</I> luciferase and shift the emission peak from 478 nm to 495 nm. This is crucial for our project because the blue light emitted by luciferase is required to activate the Aequorea victoria GFP.
<FONT style="font-size:10px">
Daubner, S. C., Astorga, A. M., Leisman, G. B. &amp; Baldwin, T. O. Yellow light emission of vibrio fischeri strain y-1: purification and characterization of the energy-accepting yellow fluorescent protein. <I>Proceedings of the National Academy of Sciences of the United States of America</I> 84, 8912-8916 (1987).

O'Kane, D. J., Woodward, B., Lee, J., and Prasher, D. C. Borrowed proteins in bacterial bioluminescence. <I>Proc Natl Acad Sci U S A</I>. 88(4), 1100–1104 (1991).
</FONT></TD></TR><TR><TD><CENTER><B><A name="luxcde">luxCDE</A> from X. luminescence (Bba_xxxx)</B></CENTER></TD><TD>The <I>P. phosphoreum</I> genes coding for enzymes that produce aldehydes required for the luciferase reaction are expressed at very low levels in E. coli. As such, we adopted the aldehyde producing system from X. luminescens.

The reasons for this low expression are not clear, but it could be due to the presence of repressors in E. coli or the usage of codons (by P. phosphoreum) where corresponding t-RNas are of very low abundance in E. coli. As both luciferases are able to use decanal (C9H19CHO) as a substrate, we anticipate that luciferase from P. phosphoreum will be able to use the aldehyde produced by X. luminescens’ s enzymes.
<FONT style="font-size:10px">
Colepicolo, P., Cho, K. W., Poinar, G. O., and Hastings, J.W. Growth and luminescence of the bacterium Xenorhabdus luminescens from a human wound. Appl Environ Microbiol. 55(10), 2601–2606 (1989).

Lee, C. Y., and Meighrn, E. A. Expression and DNA Sequence of the Gene Coding for the lux-Specific Fatty Acyl-CoA Reductase from <I>Photobacterium phosphoreum</I>. The Journal of Microbiology. 38(2), 80-87 (2000).

Miyamoto, C., Byers, D., Graham, A. F., and Meighen E. A. Expression of bioluminescence by <I>Escherichia coli</I> Containing Recombinant Vibrio harveyi DNA. Journal of Bacteriology. 169(1), 247-253 (1987).
 
</FONT></TD></TR><TR><TD><CENTER><B><A name="nsrr">nsrR from <I>Nitrosomonas Europaea</I></A>(BBa_xxxx)</B></CENTER></TD><TD>NsrR was first described as a nitrite-responsive regulator of the nirK gene encoding nitrite reductase in <I>Nitrosomonas europaea</I> (Beaumont et al., 2004). NsrR sensitivity to nitrite was increased under acid conditions, so there is the possibility that NsrR is inactivated by the NO formed enzymatically as a by-product of nitrate and nitrite reduction or by disproportionation (Spiro, 2007), but no experimental evidence supporting the latter observation has been received up to now.
br /&gt;
NsrR orthologues belong to the wider Rrf2 family of transcriptional repressors. The best characterized member of the family is the E. coli IscR protein, which contains a <A href="#">2Fe-2S</A> cluster. IscR has only three cysteine residues, which presumably provide three of the ligands to the <A href="#">Fe-S</A> luster. These cysteines are conserved in NsrR (with some variation in spacing) so it has been suggested that NsrR contains an <A href="#">Fe-S</A> cluster (Spiro, 2007).
<FONT style="font-size:10px">
Beaumont, H. J. E., Lens, S. I., Reijnders, W. N. M., Westerhoff, H. V., and Spanning, R. J. M. Expression of nitrite reductase in Nitrosomonas europaea involves NsrR, a novel nitrite-sensitive transcription repressor. Molecular Microbiology 54 (1), 148-158, 2004.

Spiro, S. Regulators of bacterial responses to nitric oxide. FEMS Microbiology Reviews 31, 193-211, (2007).
</FONT></TD></TR><TR><TD><CENTER><B><A name="nirk">nirK promoter </A>(BBa_xxxx) </B></CENTER></TD><TD>The repressor described above (NsrR) binds to this promoter in the absence of nitrites thus preventing transcription of the luciferase gene downstream.  In the presence of nitrites the repressor releases from DNA and allows transcription of the luciferase gene. Thus in nitrite-rich medium the cells will be able to make blue light. 

This promoter is in the intergeneric region between the gene encoding the repressor NsrR (<B>yhdE</B>, NE0928 in the genome sequence, which is divergently transcribed), and the gene <B>'pan'</B> (NE0927) in the genome sequence (Beaumont et al., 2004).
<FONT style="font-size:10px">
Beaumont, H. J. E., Lens, S. I., Reijnders, W. N. M., Westerhoff, H. V., and Spanning, R. J. M. Expression of nitrite reductase in Nitrosomonas europaea involves NsrR, a novel nitrite-sensitive transcription repressor. Molecular Microbiology 54 (1), 148-158, 2004.
</FONT></TD></TR><TR><TD><CENTER><B><A name="constitutive">Constitutive promoter</A><A href="http://partsregistry.org/Part:BBa_J23105">(BBa_J23105)</A></B></CENTER></TD><TD>This will control the expression of <I>tnt.r1</I>, <I>trz</I> and <I>luxCDE</I> genes. As TNT.R1 is a periplasmic protein and Trz is a transmembrane protein, elevated amounts of these protein can potentially disrupt the cell membrane. Hence, we chose a weak constitutive promoter.
                               </TD></TR><TR><TD><CENTER><B><A name="pompc">PompC-eyfp-onr</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216012">(BBa_K216012)</A></B></CENTER></TD><TD>Final construct in system.</TD></TR><TR><TD><CENTER><B><A name="pompceyfp">PompC-eyfp</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216011">(BBa_K216011)</A></B></CENTER></TD></TR><TR><TD><CENTER><B><A name="pompclacz">PompC-lacZ’</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216010">(BBa_K216010)</A></B></CENTER></TD><TD>Construct carrying  PompC <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_R0082">(BBa_R0082)</A> and <I>lacZ'</I><A href="http://partsregistry.org/Part:BBa_J33202">(BBa_J33202)</A>. Used for Miller’s assay.</TD></TR><TR><TD><CENTER><B><A name="pyearlacz">PyeaR-lacZ’</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216009">(BBa_K216009)</A></B></CENTER></TD><TD>Construct carrying  PyeaR <A href="http://partsregistry.org/Part:BBa_K216005">(BBa_K216005)</A> and <I>lacZ'’</I><A href="http://partsregistry.org/Part:BBa_J33202">(Bba_J33202)</A>. Used for Miller’s assay.</TD></TR><TR><TD><CENTER><B><A name="pyearrenilla">PyeaR-<I>Renilla luciferase</I></A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216014">(BBa_K216014)</A></B></CENTER></TD><TD>Construct carrying  PyeaR <A href="http://partsregistry.org/Part:BBa_K216005">(BBa_K216005)</A> and <I>Renilla luciferase</I><A href="http://partsregistry.org/Part:BBa_J52008">(BBa_J52008)</A>. As we were enountered great difficulty in cloing <I>Photobacterium phosphoreum luciferase</I>, we revived <A href="http://partsregistry.org/Part:BBa_J52008">BBa_J52008</A> from the Registry plates as a back-up. </TD></TR><TR><TD><CENTER><B><A name="pyearfirefly">PyeaR-<I>firefly luciferase</I></A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216015">(BBa_K216015)</A></B></CENTER></TD><TD>Construct carrying  PyeaR <A href="http://partsregistry.org/Part:BBa_K216005">(BBa_K216005)</A> and firefly luciferase <A href="http://partsregistry.org/Part:BBa_I712019">(BBa_I712019)</A>. As we were enountered great difficulty in cloing <I>Photobacterium phosphoreum luciferase</I>, we revived <A href="http://partsregistry.org/Part:BBa_I712019">BBa_I712019</A> from the Registry plates as a back-up.</TD></TR><TR><TD><CENTER><B><A name="tntr1trz">tnt.r1-trz</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216013">(BBa_K216015)</A></B></CENTER></TD><TD>Construct carrying <I>tnt.r1</I><A href="http://partsregistry.org/Part:BBa_K216002">(BBa_ K216002)</A> and <I>trz</I><A href="http://partsregistry.org/Part:BBa_K216004">(BBa_ K216004)</A>.</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="bottombox"><DIV id="bottomboxtext" align="center"><FONT size="2">Edinburgh University iGEM Team 2009</FONT></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Edinburgh/biology(biobricks)">http://2009.igem.org/Team:Edinburgh/biology(biobricks)</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/biology(biobricks)" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/biology(biobricks)" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/biology(biobricks)&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/biology(biobricks)&amp;oldid=158743" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>