<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Purdue_The_Solution_Gene_Design"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Purdue/The_Solution/Gene_Design">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Purdue/The_Solution/Gene_Design&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Purdue/The_Solution/Gene_Design&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Purdue/The_Solution/Gene_Design&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Purdue/The_Solution/Gene_Design" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Purdue/The Solution/Gene Design</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wrapper"><NAV class="menu"><UL><LI><A href="javascript:;"><SPAN>Introduction</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Introduction/Project_Overview"><SPAN>Project Overview</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Introduction/Accomplishments"><SPAN>Accomplishments</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Introduction/The_2014_Purdue_Biomakers"><SPAN>The 2014 Purdue Biomakers</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Introduction/Purdue_University_iGEM"><SPAN>Purdue University iGEM</SPAN></A></LI></UL><LI><A href="javascript:;"><SPAN>The Problem</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/The_Problem/Food_Security"><SPAN>Corn and Food Security</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/The_Problem/Current_Agriculture_Practices"><SPAN>Agriculture Practices</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/The_Problem/Plant_Nutrition"><SPAN>Iron Nutrition</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/The_Problem/Iron_Deficiency"><SPAN>Iron Deficiency</SPAN></A></LI><LI><A href="javascript:;"><SPAN>The Solution</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/The_Solution/Our_Microbe"><SPAN>Our Microbe</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/The_Solution/Project_Development"><SPAN>Project Development</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/The_Solution/Gene_Design"><SPAN>Gene Design</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/The_Solution/Parts_Submitted"><SPAN>Parts Submitted</SPAN></A></LI><LI><A href="javascript:;"><SPAN>Results</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Results/Wetlab_Experiments"><SPAN>Wetlab Experiments</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Results/Greenhouse_Experiments"><SPAN>Greenhouse Experiments</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Results/Future_Directions"><SPAN>Future Directions</SPAN></A></LI><LI><A href="javascript:;"><SPAN>Policy and Practices</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Policy_and_Practices/Introduction"><SPAN>Introduction</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Policy_and_Practices/Projects"><SPAN>Projects</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Policy_and_Practices/Scaling_It_Up"><SPAN>Scaling It Up</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Policy_and_Practices/Cost-Benefit_Analysis"><SPAN>Cost-Benefit Analysis</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Policy_and_Practices/Policy"><SPAN>Policy</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Policy_and_Practices/Outreach"><SPAN>Outreach</SPAN></A></LI><LI><A href="javascript:;"><SPAN>Safety</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Safety/Safety"><SPAN>Minecrobe Safety Concerns</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Safety/Safety_Forms_and_Certification"><SPAN>Lab Safety Practices</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Safety/Protocols"><SPAN>Protocols</SPAN></A></LI><LI><A href="javascript:;"><SPAN>Acknowledgments</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Acknowledgements/Academic_Collaborators"><SPAN>Academic Collaborators</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Acknowledgements/Sponsors"><SPAN>Sponsors</SPAN></A></LI><LI><A href="https://2014.igem.org/Team:Purdue/Acknowledgements/Other_Acknowledgements"><SPAN>Other Acknowledgements</SPAN></A></LI></NAV><DIV id="prev_button" class="nav_button"><A href="#" id="prev_link"><SPAN class="arrow">←</SPAN><SPAN class="cap">Previous Page</SPAN></A></DIV><DIV id="next_button" class="nav_button"><A href="#" id="next_link"><SPAN class="arrow">→</SPAN><SPAN class="cap">Next Page</SPAN></A></DIV><DIV class="gutter"><H1><SPAN>Plasmid Design</SPAN></H1><P>Our plasmids encode the enzymes necessary for the biochemical synthesis of phytosiderophores.[i] Two plasmids were used because the number of DNA base pairs for all of the genes (9,535 base pairs) was too high to fit on a single plasmid. The genes were split to make the plasmids as equal in length as possible while maintaining an appropriate configuration. This may lead to different expression of the genes on each respective plasmid, but we chose to take this risk and compensate for it in our characterization. Sequences for our enzymes were obtained from corn and codon optimized for Bacillus subtilis.</P><P>The first plasmid had 4,832 base pairs and included two of the corn genes involved in the biosynthesis of phytosiderophores, the corn gene responsible for transporting phytosiderophores out of the cell, and a reporter. It starts with a constitutive promoter (“Pveg”) which has a medium level of gene expression in Bacillus subtilis[ii]. Following the promoter is an RBS; the RBS’s used in this plasmid and the second were all “SpoVG RBS”, which has been shown to have high levels of translation in Bacillus subtilis[iii]. The corn genes are naat1 (1566 base pairs), which encodes nicotianamine aminotransferase[iv], DMAS (942 bp), which encodes deoxymugineic acid synthase[v], and TOM1 (1431 bp), a transporter of mugineic acid family phytosiderophores[vi]. Following them is a reporter, sfGFP, or superfolder Green Fluorescent Protein, which has been documented as extremely effective in Bacillus subtilis (720 bp)[vii]. Following the sfGFP is a terminator (39 bp).</P><P>The second plasmid had 4,703 base pairs and included the kill switch, two of the corn genes involved in the biosynthesis of phytosiderophores, and a reporter. It starts with a promoter induced by lactose (“PLlac0”, 55 bp). Following the lactose promoter is Hyb5, a holin which causes lysing [cell-death] in Bacillus subtilis (465 bp)[viii].  A terminator concludes the kill switch section of the plasmid. Following the kill switch is the constitutive promoter Pveg, the corn genes SAM (1689 bp), which encodes S-adenosylmethionine synthase,[ix] and nas1 (984 bp), which encodes nicotianamine synthase,[x] a reporter mRFP (mutant red fluorescent protein, 706 bp)[xi] which has been successfully used in Bacillus subtilis before, and a concluding terminator.</P></DIV></DIV><DIV class="gutter"><H1><SPAN>Organization</SPAN></H1><P>The plasmids were designed strategically so that they were roughly equal in number of base pairs, and the orders of specific were considered carefully. Shorter plasmids are more likely to be successful and cheaper, which is why we tried to minimize the length of each plasmid. Terminators are not always effective, so the kill switch is located at the beginning of a plasmid to avoid being accidently translated by an mRNA due to a previous section not being properly terminated. The reporters are placed after the corn genes because gene expression decreases linearly as the distance from the promoter increases, and expressing the corn genes is much more important than expressing the reporters.</P></DIV></DIV><DIV class="gutter"><H1><SPAN>Registry Part Selection</SPAN></H1><P>Using a conditional promoter so that the modified Bacillus subtilis would only produce phytosiderophores in the presence of plants roots was considered, but it was decided that conditional promoters would not be reliable enough. A constitutive promoter of medium strength was chosen for the corn genes because a weak promoter might not have produced enough phytosiderophores to make a difference, and a strong promoter might have overtaxed the cell in the production of phytosiderophores as to be counterproductive.</P><P>A strong RBS was chosen so that when a promoter was induced, the RBS would not be the weak link. This particular RBS was selected as a strong one because it has been shown to have high levels of translation specifically in Bacillus subtilis,[xii] even though many parts are E. coli specific and do not work well, if at all, in Bacillus subtilis.</P><P>sfGFP and mRFP were chosen as the reporters because they both are expressed in Bacillus subtilis (research revealed that RFP is not expressed in Bacillus subtilis and thus should not be used with it),[xiii] visible reporters are easiest to observe, and fluorescent proteins in particular are easiest of the visible reporters, especially since the bacteria would be in a dark soil medium. Two reporters were chosen instead of using the same one over so that it could be observed whether one or both plasmids were being expressed.</P><P>A relatively strong forward terminator with few base pairs was selected for its relative effectiveness, the fact that no reverse termination was required, and the number of base pairs was to be minimized as much as possible without impairing function.</P></DIV></DIV><DIV class="gutter"><H1><SPAN>Phytosiderophore Genes</SPAN></H1><P>Different organisms have slightly different DNA sequences for phytosiderophores production, and corn DNA was selected as our source for two main reasons. Rice experiences less iron deficiency than corn (M. Levy, personal communication, June 30, 2014), and in case the phytosiderophores could only be used by the organism from which the DNA came, we wanted to make it more likely corn would be aided. Another reason was that the corn genes were shorter than their rice counterparts. The corn genes were codon optimized for Bacillus subtilis using information on codon usage.[xiv] Rare codon optimization occurs because different organisms use the various codons which code for the same amino acid at different frequencies; switching out a codon that the organism from which the DNA came from uses often but the new organism uses rarely for a codon that the new organism uses more frequently can increase the rate of gene expression.</P><P>Two of the genes, nas and naat, had separate forms. nas has variants nas1, nas2, nas3, nas4, nas5-1, nas5-2, and nas6, while naat has naat-A, naat-B, and naat1, among others. nas1 was selected because, in a previous experiment, it was able to, alone, produce NAS.[xv] naat1 from corn was selected because a previous experiment revealed that the rice naat1 gene encodes functional NAAT and is expressed in cells of Fe-deficient leaves, strongly suggesting that it is involved in phytosiderophores production and sufficient to produce NAAT on its own, without other naat genes.[xvi]</P></DIV></DIV><DIV class="gutter"><H1><SPAN>Killswitch</SPAN></H1><P>The killswitch was included as a safety precaution so that the modified bacteria could be easily killed. We wanted to include the kill switch because the modified bacteria are intended to be released into fields, not kept in a confined environment like a laboratory. With a kill switch, the bacteria could still be killed out in the field if they were not working as expected or if they spread farther than desired. Finding a promoter for the kill switch was complicated by the fact that most kill switches rely on things such as dramatically altering the pH or exposing the bacteria to red light, but this project required something that could get to the bacteria underground and is economically feasible to spread across an area as big as a field while not harming native plants or the natural soil microbiome. Lactose was chosen because it fulfills all of these requirements and would not work as a food source for the Bacillus subtilis.[xvii] The holin Hyb5 was selected because it is effective on Bacillus subtilis at neutral pH around field temperature and had fewer base pairs than its counterpart, the endolysin Lyb5[xviii].</P></DIV><SECTION id="igem_content" class="content"><DIV class="gutter"><H1><SPAN>Biochemical Pathway</SPAN></H1>Phytosiderophores are not directly produced by cells but instead formed from methionine being chemically altered in a specific order by four different enzymes. L-methionine is altered by S-adenosylmethione synthase (SAM synthase) to have an adenosyl bonded to a sulfer on the L-methionine, producing S-adenosylmethione (SAM). From there, nicotianamine synthase (NAS) catalyzes the trimerization of SAM molecules to form nicotianamine (NA). Nicotianamine aminotransferase ­(NAAT) catalases the conversion of nictotianamine to 3”deamino-3”-oxonicotianamine (a 3”-keto intermediate) and L-glutamate using 2-oxoglutarate. The 3”-carbon of the keto intermediate is reduced by deoxymugineic acid synthase (DMAS) to produce 2’-Deoxymugineic Acid (DMA). </DIV></SECTION><DIV class="gutter"><H1><SPAN>Citations</SPAN></H1><P>[i] Kobayahi, T. &amp; Nishizawa, N.K. (30 January, 2012). Iron uptake, translocation, and regulation in higher plants. Annu. Rev. Plant Biol. 2012. 63:131-52.  DOI: 10.1146/annurev-arplant-042811-105522. <A href="http://www.annualreviews.org/doi/pdf/10.1146/annurev-arplant-042811-105522" class="external free" rel="nofollow">http://www.annualreviews.org/doi/pdf/10.1146/annurev-arplant-042811-105522</A></P><P>[ii] iGEM Registry of Standard Biological Parts. Part: BBa_K143012; Promoter veg: Constitutive Promoter for B. subtilis. Added 11 September, 2008 by Imperial College of London. <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K143012" class="external free" rel="nofollow">http://parts.igem.org/wiki/index.php?title=Part:BBa_K143012</A></P><P>[iii] iGEM Registry of Standard Biological Parts. Part: BBa_K143021; SpoVG Ribosome Binding Site (RBS) for B. subtilis. Added 17 September, 2008 by Imperial College of London. <A href="http://parts.igem.org/Part:BBa_K143021" class="external free" rel="nofollow">http://parts.igem.org/Part:BBa_K143021</A></P><P>[iv] NCBI. (19 October, 2011). Zea mays ZmNAAT1 mRNA for putative nicotianamine aminotransferase, complete cds. <A href="http://www.ncbi.nlm.nih.gov/nuccore/166788521" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/nuccore/166788521</A></P><P>[v] NCBI. (2014). deoxymugineic acid synthase1 [Zea mays]. <A href="http://www.ncbi.nlm.nih.gov/protein/162460852" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/protein/162460852</A></P><P>[vi] NCBI. (10 December, 2008). Zea mays clone 222622 protein transporter mRNA, complete cds. <A href="http://www.ncbi.nlm.nih.gov/nuccore/195621011" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/nuccore/195621011</A></P><P>[vii] iGEM Registry of Standard Biological Parts. Part: BBa_K515005; superfolder GFP (sfGFP). Added 7 September, 2008 by Imperial College of London.
</P><P>[viii] NCBI. (2 December,  2008). Lactobacillus phage YB5 holin gene, complete cds. <A href="http://www.ncbi.nlm.nih.gov/nuccore/EF623894?report=GenBank" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/nuccore/EF623894?report=GenBank</A></P><P>[ix] NCBI. (2014). S-adenosylmethionine synthetase 1 [Zea mays]. <A href="http://www.ncbi.nlm.nih.gov/protein/NP_001148708.1" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/protein/NP_001148708.1</A></P><P>[x] NCBI. (5 February, 2003). Zea mays ZmNAS1 mRNA, complete cds. <A href="http://www.ncbi.nlm.nih.gov/nuccore/19911063" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/nuccore/19911063</A></P><P>[xi] iGEM Registry of Standard Biological Parts. Part: BBa_E1010; **highly** engineered mutant of red fluorescent protein from Discosoma striata (coral). Added 28 July, 2004 by Drew Endy.
</P><P>[xii] METU Turkey 2013 iGEM team. (2013). Bee subtilis. <A href="https://2013.igem.org/Team:METU_Turkey/project.html" class="external free" rel="nofollow">https://2013.igem.org/Team:METU_Turkey/project.html</A></P><P>[xiii] Groningen 2012 iGEM team. (2012). Submitted biobricks. <A href="https://2012.igem.org/Team:Groningen/OurBiobrick" class="external free" rel="nofollow">https://2012.igem.org/Team:Groningen/OurBiobrick</A></P><P>[xiv] Sharp, P.M.; Cowe, E.; Higgins, D.G.; Shields, D.C.; Wolfe, K.H.; &amp; Wright, F. (12 September, 1988). Codon usage patterns in Escherichia coli, Bacillus subtilis, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Drosophila melanogaster and Homo sapiens; a review of the considerable within-species diversity. Nucleic Acids Research 16(17); 8207-8211. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC338553/" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC338553/</A></P><P>[xv] Higuchi, K.; Suzuki, K.; Nakanishi, H.; Yamaguchi, H.; Nishizawa, N.K.; &amp; Mori, S. (February 1999). Cloning of nicotianamine synthase genes, novel genes involved in the biosynthesis of phytosiderophores. Plant Physiology 119(2) 471-480. DOI: ​10.​1104/​pp.​119.​2.​471. <A href="http://www.plantphysiol.org/content/119/2/471/F1.expansion.html" class="external free" rel="nofollow">http://www.plantphysiol.org/content/119/2/471/F1.expansion.html</A></P><P>[xvi] Inoue, H.; Takahashi, M.; Kobayashi, T.; Suzuki, M.; Nakanishi, H.; Mori, S.; &amp; Nishizawa, N.K. (2008). Identification and localization of the rice nicotianamine aminotransferase gene OsNAAT1 expression suggests the site of phytosiderophore synthesis in rice. Plant Mol. Biol. 66(1-2) 193-203. DOI: 10.1007/s11103-007-9262-8.  <A href="http://www.ncbi.nlm.nih.gov/pubmed/18034312/" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/18034312/</A></P><P>[xvii] Krispin, O. &amp; Allmansberger, R. (April, 1998). The Bacillus subtilis gale gene is essential in the presence of glucose and galactose. J Bacteriol 180(8); 2265-2270. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC107161/#B18" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC107161/#B18</A></P><P>[xviii] Wang, S.; Kong, J.; &amp; Zhang, X. (19 November, 2008). Identification and characterization of the two-component cell lysis cassette encoded by temperate bacteriophage ϕPYB5 of Lactobacillus fermentum. Journal of Applied Microbiology, Volume 105, Issue 6, pages 1939-1944. DOI: 10.1111/j.1365-2672.2008.03953.x <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2008.03953.x/full" class="external free" rel="nofollow">http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2008.03953.x/full</A></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Purdue/The_Solution/Gene_Design">http://2014.igem.org/Team:Purdue/The_Solution/Gene_Design</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Purdue/The_Solution/Gene_Design" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Purdue/The_Solution/Gene_Design" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Purdue/The_Solution/Gene_Design&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Purdue/The_Solution/Gene_Design&amp;oldid=349274" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>