<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LMU-Munich_Notebook_Apoptosis"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:LMU-Munich/Notebook/Apoptosis">Page               </A></LI><LI><A href="/Talk:Team:LMU-Munich/Notebook/Apoptosis">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Notebook/Apoptosis&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Notebook/Apoptosis&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LMU-Munich/Notebook/Apoptosis" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LMU-Munich/Notebook/Apoptosis</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="Layer1"><TABLE bgcolor="black" width="979" border="0" cellspacing="0" cellpadding="0" align="center"><TBODY><TR><TD valign="top" bgcolor="black"><DIV class="lmunavi"><UL><LI><B><A href="/Team:LMU-Munich/Cut%27N%27survive" title="Team:LMU-Munich/Cut'N'survive">Cut'N'Survive System</A></B></LI><LI><A href="/Team:LMU-Munich/Cut%27N%27survive/Functional_Principle" title="Team:LMU-Munich/Cut'N'survive/Functional Principle">Functional Principle</A></LI><LI><A href="/Team:LMU-Munich/Cut%27N%27survive/Sequences" title="Team:LMU-Munich/Cut'N'survive/Sequences">Sequences</A></LI><LI><A href="/Team:LMU-Munich/Cut%27N%27survive/Schedule" title="Team:LMU-Munich/Cut'N'survive/Schedule">Schedule</A></LI></UL><LI><B><A href="/Team:LMU-Munich/Jump-or-die" title="Team:LMU-Munich/Jump-or-die">Jump-or-Die System</A></B></LI><LI><A href="/Team:LMU-Munich/Jump-or-die/Functional_Principle" title="Team:LMU-Munich/Jump-or-die/Functional Principle">Functional Principle</A></LI><LI><A href="/Team:LMU-Munich/Jump-or-die/Seqences" title="Team:LMU-Munich/Jump-or-die/Seqences">Sequences</A></LI><LI><A href="/Team:LMU-Munich/Jump-or-die/Schedule" title="Team:LMU-Munich/Jump-or-die/Schedule">Schedule</A></LI><LI><B><A href="/Team:LMU-Munich/ProSearch" title="Team:LMU-Munich/ProSearch">ProSearch System</A></B></LI><LI><A href="/Team:LMU-Munich/ProSearch/Schedule" title="Team:LMU-Munich/ProSearch/Schedule">Schedule</A></LI></DIV></TD></TR><TR><TD><DIV class="lmunavi"><UL><LI><A href="/Team:LMU-Munich/Functional_Principle" title="Team:LMU-Munich/Functional Principle">Functional Principle</A></LI><LI><A href="/Team:LMU-Munich/Sequences" title="Team:LMU-Munich/Sequences">Sequences</A></LI><LI><A href="/Team:LMU-Munich/Schedule" title="Team:LMU-Munich/Schedule">Schedule</A></LI></UL><UL><LI><B><A href="/Team:LMU-Munich/Azo_Switch/crosslinks/Picture_gallery" title="Team:LMU-Munich/Azo Switch/crosslinks/Picture gallery">picture gallery</A></B></LI></UL></DIV></TD></TR></TBODY></TABLE><TABLE width="807" height="1210" border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD colspan="2" align="left" valign="top" bgcolor="#FFFFFF" style="padding:1em;"><H2><SPAN class="mw-headline" id="ApoControl_Notebook"><FONT color="#9933CC"><B>ApoControl Notebook</B></FONT></SPAN></H2><H2><SPAN class="mw-headline" id="What_we_did"><B>What we did</B></SPAN></H2><P><B>Short description of our work, our results and our supporters</B></P><P>
The creation of certain constructs was necessary for our two systems for cell selection by means of apoptosis: “Cut’N’Survive” and “Jump-Or-Die”. We searched for sources of the DNA sequences we needed and found several supporters which are listed below.
</P><P>Most genes and promoters were amplificated via PCR with overhang-primers with the BioBrick prefix or suffix. If the sequence contained a EcoR1-, Pst1-, Xba1-, Spe1- or Not1- restriction site, we used mutagenesis primers and fusioned both DNA parts by fusion PCR. All PCRs worked out, even the fusion PCRs.
</P><P>The length of the PCR products were tested by agarose gel electrophoresis. We tried to sequence our PCR products, but obtained poor results and resorted to sequencing the plasmids.
</P><P>In parallel, we made competent cells and multiplied ccdB (death gene)-vectors with different antibiotic resistances. All components were digested with the appropriate restriction enzymes. The samples were cleaned with a PCR clean up kit or dephosphorylated to reduce false ligations.
</P><P>We ligated our constructs and several interim stages with the 3A-assembly according to our schedule. The ligations were transformed to E.coli DH5α strains and selected by antibiotics. Afterwards, some colonies were picked and we tested the insertion of the construct by colony PCR. 
</P><P>If the colony PCR resulted in bands of the right size, we extracted the plasmids from overnight cultures and sequenced the samples with forward and reverse BioBrick primers.
</P><P>Unfortunately, not all BioBricks were cloned succesfully. However, we were able to produce 4 BioBricks, one of which represents a full construct while the other three are intermediates. The system wasn't completed on time, so we weren´t able to test them in eukarytic cell lines. 
</P><P><B>The protocols we used are listed here: </B><A href="/Team:LMU-Munich/Notebook#Protocols" title="Team:LMU-Munich/Notebook">Protocols</A></P><P><B>These Biobricks we submitted: </B></P><UL><LI>BBa_K368004: attP+eGFP+SV40PA
</LI><LI>BBa_K368011: eGFP+SV40PA
</LI><LI>BBa_K368016: TEVrecognition site+N-degron+SF3b155
</LI><LI>BBa_K368019: TEV-Protease+p14*+TEVrecognition site
</LI></UL><P><B>Sources, helpers and supporters:</B></P><UL><LI>	Prof. Dr. Angelika Böttger : 
</LI><LI>	prevTRE (tet-on CMV promoter; inducible by doxycycline in special cell lines)
</LI><LI>	supported the construction ideas and would have given us the cells and mediums we would have needed
</LI><LI>	SV40PA (Polyadenylation site): gave us a vector containing it
</LI><LI>	Human Bak:  her assistant Erika Clement gave us appropriate cDNA
</LI></UL><UL><LI>	Dr. Arnim Weber: submitted us a vector with human Bak
</LI><LI>	Dr. Philipe Soriano: 
<A href="http://www.ncbi.nlm.nih.gov/pubmed/17225864?dopt=Abstract"> (Raymond CS et al: High-Efficiency FLP and PhiC31 Site-Specific Recombination in Mammalian Cells (2007))</A></LI><LI>	Sequences of attB and attP site
</LI><LI>	PhiC31o was bought via addgene
</LI></UL><LI>	Knop, M (Heidelberg): 
<A href="http://www.ncbi.nlm.nih.gov/pubmed?term=Efficient%20protein%20depletion%20by%20genetically%20controlled%20deprotection%20of%20a%20dormant%20N-degron">(Knop et al.: Efficient protein depletion by genetically controlled deprotection of a dormant N-degron (2009))</A></LI><LI>	TEVrecognition site+N-degron+SF3b155
</LI><LI>	TEV-Protease+p14*+TEVrecognition site
</LI><LI>	Prof. Dr. Thorsten Mascher:
</LI><LI>	Helped with primer design, agarose gel electrophoresis apparatuses and trouble shooting
</LI><LI>	Prof. Dr. Kirsten Jung:
</LI><LI>	Helped with ideas and fundraising
</LI><LI>	Dr. Susanne Gebhard:
</LI><LI>	Helped with trouble-shooting and materials
</LI><LI>	Prof. Dr. Andreas Brachmann: 
</LI><LI>	Sequenced our samples
</LI><LI>	Partsregistry:
</LI><LI>	eGFP (BBa_I714891)
</LI><LI>	CMV-Promoter (BBa_J52034: this part was wrong: its lacI !!!)
</LI><LI>	ccdB amp, cam, tet, kan in E.coli DH3
</LI><H2><SPAN class="mw-headline" id="Contents.3E"><B>Contents</B>&gt;</SPAN></H2></TD></TR><TR><TH> Week                       </TH><TH colspan="7">Days
</TH></TR><TR><TD style="text-align:center">Monday
</TD><TD style="text-align:center">Tuesday
</TD><TD style="text-align:center">Wednesday
</TD><TD style="text-align:center">Thursday
</TD><TD style="text-align:center">Friday
</TD><TD style="text-align:center">Saturday
</TD><TD style="text-align:center">Sunday
</TD></TR><TR><TD style="text-align:center"> 31
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-02-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-02-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-03-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-03-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-04-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-04-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-05-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-05-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-06-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-06-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-07-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-07-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-08-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-08-2010</A></TD></TR><TR><TD style="text-align:center"> 32
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-09-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-09-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-10-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-10-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-11-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-11-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-12-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-12-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-13-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-13-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-14-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-14-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-15-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-15-2010</A></TD></TR><TR><TD style="text-align:center"> 33
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-16-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-16-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-17-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-17-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-18-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-18-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-19-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-19-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-20-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-20-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-21-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-21-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-22-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-22-2010</A></TD></TR><TR><TD style="text-align:center"> 34
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-23-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-23-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-24-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-24-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-25-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-25-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-26-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-26-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-27-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-27-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-28-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-28-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-29-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-29-2010</A></TD></TR><TR><TD style="text-align:center"> 35
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-30-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-30-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#8-31-2010" title="Team:LMU-Munich/Notebook/Apoptosis">8-31-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-01-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-01-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-02-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-02-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-03-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-03-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-04-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-04-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-05-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-05-2010</A></TD></TR><TR><TD style="text-align:center"> 36
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-06-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-06-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-07-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-07-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-08-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-08-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-09-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-09-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-10-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-10-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-11-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-11-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-12-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-12-2010</A></TD></TR><TR><TD style="text-align:center"> 37
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-13-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-13-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-14-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-14-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-15-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-15-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-16-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-16-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-17-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-17-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-18-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-18-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-19-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-19-2010</A></TD></TR><TR><TD style="text-align:center"> 38
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-20-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-20-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-21-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-21-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-22-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-22-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-23-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-23-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-24-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-24-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-25-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-25-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-26-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-26-2010</A></TD></TR><TR><TD style="text-align:center"> 39
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-27-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-27-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-28-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-28-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-29-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-29-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#9-30-2010" title="Team:LMU-Munich/Notebook/Apoptosis">9-30-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-01-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-01-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-02-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-02-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-03-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-03-2010</A></TD></TR><TR><TD style="text-align:center"> 40
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-04-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-04-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-05-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-05-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-05-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-06-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-07-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-07-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-08-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-08-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-09-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-09-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-10-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-10-2010</A></TD></TR><TR><TD style="text-align:center"> 41
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-11-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-11-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-12-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-12-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-13-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-13-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-14-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-14-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-15-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-15-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-16-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-16-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-17-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-17-2010</A></TD></TR><TR><TD style="text-align:center"> 42
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-18-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-18-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-19-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-19-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-20-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-20-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-21-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-21-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-22-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-22-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-23-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-23-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-24-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-24-2010</A></TD></TR><TR><TD style="text-align:center"> 43
</TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-25-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-25-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-26-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-26-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-27-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-27-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-28-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-28-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-29-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-29-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-30-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-30-2010</A></TD><TD style="text-align:center"><A href="/Team:LMU-Munich/Notebook/Apoptosis#10-31-2010" title="Team:LMU-Munich/Notebook/Apoptosis">10-31-2010</A></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="8-10-2010"> 8-10-2010 </SPAN></H2><P><FONT color="#009933">Transforming competent cells</FONT></P><P>-	eGFP Biobrick: BBa_I714891 SDY_eGFP (Kanamycin)
</P><P>-	TEV recogn N Degron SF3 = pDS7 (Ampicillin)
</P><P>-	TEV p14 recogn = 190-6 (Ampicillin)
</P><P>-&gt;	Protocol: (<A href="/Team:LMU-Munich/Notebook/Protocols/3_Transformation" title="Team:LMU-Munich/Notebook/Protocols/3 Transformation">3 Transformation</A>)
</P><P>-	We added 2 µl DNA
</P><P>-	We plated out 200 µl
</P><P><FONT color="#009933">Plasmid Isolation</FONT></P><P>-	CMV-Promoter Biobrick: BBa_J52034
</P><P>-&gt;	Protocol:(<A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A>)
</P><P>-	Prepared overnight culture, measured concentration of DNA
</P><P>-&gt;	Poor results -&gt; thrown away
</P><H2><SPAN class="mw-headline" id="8-11-2010"> 8-11-2010 </SPAN></H2><P><FONT color="#009933">New Plasmid Extraction</FONT></P><P>-	CMV-Promoter Biobrick: BBa_J52034
</P><P>-&gt;	Protocol: (<A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A>)
</P><P>-	Plasmid concentration: 143ng/µl
</P><P><FONT color="#009933">Prepared overnight culture of eGFP BBa_I714891</FONT></P><P>-	3 ml LB-Media + 4 µl Kanamycin
</P><P>-	Inoculated with 1 colony of BBa_I714891 -&gt; 37°C
</P><P><FONT color="#009933">Prepared overnight culture of 190-6 and pDS7 and eGFP (BBa_I714891) in falcons</FONT></P><P>-	for 190-6 and pDS7: 10µl Ampicillin + 10 ml LB-Media + colony of plate
</P><P>-	for eGFP: 13,3 µl Kanamycin + 10 ml LB-Media + 1 colony of plate
</P><P><FONT color="#009933">Restriction digestion of CMV-Promoter BBa_J52034 with EcoRI and PstI</FONT></P><TABLE><TBODY><TR><TD>H<SUB>2</SUB>O<SUB>ddest, sterile</SUB></TD><TD>10,3 µl
</TD></TR><TR><TD>RE10 + Buffer H
</TD><TD>2,0 µl
</TD></TR><TR><TD>acetylated BSA (18ng/µl)
</TD><TD>0,2 µl
</TD></TR><TR><TD>DNA (0,143µg/µl)
</TD><TD>6,0 µl
</TD></TR></TBODY></TABLE><P>-&gt;	mixed
</P><P>-	plus: EcoRI (10µg/µl): 0,5 µl resp. PstI (10µg/µl): 0,5 µl
</P><P>-	incubated at room temperature from 12:10 to 15:00, 1 hour at 37°C, 2 hours at 60°C
</P><P>-	frozen at -20°C
</P><P><FONT color="#009933">Prepare new/fresh overnight culture of CMV-Promoter Biobrick: BBa_J52034</FONT></P><P>-	1 ml of &quot;old&quot; culture + 3 ml LB-Media + 4 µl Kanamycin -&gt; 37°C
</P><H2><SPAN class="mw-headline" id="8-12-2010"> 8-12-2010 </SPAN></H2><P><FONT color="#009933">Plasmid Extraction of pDS7, eGFP, 190-6</FONT></P><P>-&gt;	Protocol: (<A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraktion from cells</A>)
</P><P>-       pDS7 (458ng/µl), eGFP (55ng/µl), 190-6 (193ng/µl)
</P><P><FONT color="#009933">Restriction digest of pDS7, eGFP, 190-6</FONT></P><P>-	with EcoRI and PstI in buffer H (for testing DNA is correct)
</P><P>-&gt;      Protocol: (<A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digest</A>)
</P><P>-       10µg DNA: pDS7 (2µl), eGFP (15µl), 190-6 (10µl)
</P><P><FONT color="#009933">Plate colonies for plasmid extraction</FONT></P><P>-	CMV (Kanamycin), eGFP (Kanamycin), pDS7 (Ampicillin), 190-6 (Ampicillin)) 
</P><P>-	PhiC31o plated on Ampicillin-Agar, stored at 37°C
</P><P><FONT color="#009933">50% Glycerol made</FONT></P><P>-       for PhiC31o glycerol stock (produced later)
</P><H2><SPAN class="mw-headline" id="8-13-2010"> 8-13-2010 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gelfoto from the EcoR1 and Pst1 Restrictiondigest of 190-6, eGFP, pDS7 and CMV</DIV></DIV></DIV><P><FONT color="#009933">Inoculate CMV into LB medium with ampicillin</FONT></P><P>- CMV (BBa_J52034) from 10.8.2010 inoculated into LB medium with ampicillin, as falsly inoculated in Kanamycin
</P><P><FONT color="#009933">Agarosegelelectrophoresis with digestions</FONT></P><P>-&gt;Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis">11 Agarose gel electrophoresis</A>)
</P><P>- Agarosegelelectrophoresis with the digestions (CMV, eGFP, pDS7, 190-6), 125V for 30 minutes and then for 20 minutes;
</P><P>- expected DNA bands: 190-6 (4840bp, 1903bp), pDS7 (8027bp, 6bp), CMV (654 bp (Insert), 2079bp (Plasmid)), eGFP (720bp (Insert), 2750bp (Plasmid))
</P><P>- <FONT color="#CC33CC"><B>Correct DNA bands for 190-6 (~4800bp, ~1900bp, ~6700bp (undigested plasmid)) and eGFP (~2000bp (Plasmid), ~750 bp (Insert))</B></FONT>; CMV probably not digested (two bands; one probably normal, one supercoiled) and pDS7 not clear
</P><P><FONT color="#009933">Restriction digest from CMV and pDS7</FONT></P><P>-&gt; Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digest</A>)
</P><P>- Restriction digest from CMV (EcoR1, Pst1; 6µl DNA, buffer H) and pDS7 (EcoR1, Spe1; 2µl DNA, buffer B)
</P><P><FONT color="#009933">Agarosegelectrophoresis with digestions</FONT></P><P>-&gt;Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis">11 Agarose gel electrophoresis</A>)
</P><P>- Agarosegelelectorphoresis for 30 minutes, 150V
</P><P>- Expected DNA bands: CMV see above, pDS7 (3647bp, 3369bp, 1011bp, 6bp)
</P><P>- false DNA bands CMV (~1200 bp, ~2000 bp) and pDS7 (~8000bp two bands, ~1100 bp); required to isolate a new colony for these two Plasmidextractions
</P><P><FONT color="#009933">Plated CMV on Ampicllin-Agar</FONT></P><P>- Plated the colony from CMV (BBa_J52034) for Plasmidextraction (Ampicillin), as falsly plated on Kanamycin
</P><H2><SPAN class="mw-headline" id="8-14-2010"> 8-14-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="8-15-2010"> 8-15-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="8-16-2010"> 8-16-2010 </SPAN></H2><P><FONT color="#009933">Planting colonies</FONT></P><P>- transfer 1 ml PhiC31o culture to new LB medium + Amp, 37°C
</P><P>- pick up CMV and pDS7 colonies from plates and transfer to LB medium+Amp, 37°C
</P><P><FONT color="#009933">Plasmid Extraction of PhiC31o</FONT></P><P>-&gt;Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraktion from cells</A>)
</P><P>- plasmid extraction of PhiC310 
</P><P>-&gt;27,5ng/µl DNA and second plasmid extraction of PhiC310 (i. o. to get more DNA); first eluation-step with first eluation-extraction 
</P><P>-&gt; 60ng/µl DNA 
</P><P><FONT color="#009933">Restriction digest</FONT></P><P>-&gt;Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digest</A>) 
</P><P>- restriction digest of PhiC310 with EcoR1 and Spe1
</P><TABLE><TBODY><TR><TD>H<SUB>2</SUB>O<SUB>ddest, sterile</SUB></TD><TD>0 µl
</TD></TR><TR><TD>Buffer B
</TD><TD>2,0 µl
</TD></TR><TR><TD>BSA (1:10)
</TD><TD>2 µl
</TD></TR><TR><TD>DNA (0,06µg/µl)
</TD><TD>15,0 µl
</TD></TR><TR><TD>EcoR1
</TD><TD>0,5 µl
</TD></TR><TR><TD>Spe1
</TD><TD>0,5µl
</TD></TR></TBODY></TABLE><P>restriction digest in the thermo cycler (program &quot;Verdau&quot;, see protocol)
</P><P><FONT color="#009933">Handling primers after arrival (1,2,3,4,5,6,11,12)</FONT></P><P>-&gt;Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/9_Handling_primers" title="Team:LMU-Munich/Notebook/Protocols/9 Handling primers">9 Handling primers</A>)
</P><P><FONT color="#009933">PCR preparations</FONT></P><P>- 10mM dNTP mix made from 100 mM dATP, dGTP, dCTP, dTTP by taking 100µl of each and adding 600µl H <SUB>2</SUB> O
</P><P><FONT color="#009933">PCR 1 and 6</FONT></P><P>- PCR of the tet inducible CMV minimal promotor out of prevTRE (=PCR 1 with Primer 1 and 2) and SV40PA out of pcDNA3 (=PCR 6 with Primer 11 and 12)
</P><P>-&gt;Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu">10 PCR with Pfu</A>)
</P><P>Mixture:
</P><TABLE><TBODY><TR><TD>pTRERev (0,15µg/µl)
</TD><TD>pcDNA3 (0,6 µg/µl)
</TD></TR><TR><TD>Primer
</TD><TD>2*2,5µl (P1+P2)
</TD><TD>&quot; (P11+P12)
</TD></TR><TR><TD>300ng template
</TD><TD> 0,5µl
</TD><TD> 2µl
</TD></TR><TR><TD>10x Buffer Pfu
</TD><TD>5µl
</TD><TD>&quot;
</TD></TR><TR><TD>dNTP Mix
</TD><TD>1µl
</TD><TD>&quot;
</TD></TR><TR><TD>Pfu Polymerase (3u/µl)
</TD><TD>0,5µl
</TD><TD>&quot;
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>40,5µl
</TD><TD>39µl
</TD></TR><TR><TD>summ
</TD><TD>52,5µl
</TD><TD>52,5µl
</TD></TR></TBODY></TABLE><P>
Programme:
</P><TABLE><TBODY><TR><TD>Denaturation
</TD><TD>95°C
</TD><TD>2min
</TD></TR><TR><TD>30 times:
</TD><TD>Denaturation
</TD><TD>95°C
</TD><TD>1min
</TD></TR><TR><TD>Annealing
</TD><TD>45°C
</TD><TD>30sec
</TD></TR><TR><TD>Extension
</TD><TD>73°C
</TD><TD>2min
</TD></TR><TR><TD>Final Extension
</TD><TD>73°C
</TD><TD>5min
</TD></TR><TR><TD>Soak (end)
</TD><TD>12°C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P><FONT color="#009933">Glycerolstock of PhiC31o</FONT></P><P>- Glycerolstock of the colony of PhiC31o for the plasmidextraction
</P><TABLE><TBODY><TR><TD>bacterial culture
</TD><TD>800µl
</TD></TR><TR><TD>Glycerol (50%)
</TD><TD>500µl
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="8-17-2010"> 8-17-2010 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:115px;"><DIV class="thumbcaption">Agarose gel electrophoresis of PCR6 which shows that PCR6 is about 200bp</DIV></DIV></DIV><P><FONT color="#009933">Plate CMV and pDS7 colonies on Ampicillin-Agar</FONT></P><P>- colonies for plasmidextraction of CMV and pDS7 plated on Ampicillinplates
</P><P><FONT color="#009933">Plasmid Extraction of CMV and pDS7</FONT></P><P>- plasmidextraction of CMV (2,5ng/µl) and pDS7 (10ng/µl) the A260/A280 value was 1.333, which means that it was 90% Protein and only 10% DNA (should be 1,8); new plasmidextraction needed
</P><P><FONT color="#009933">new overnight cultures of CMV and pDS7 for a new plasmidextraction made</FONT></P><P><FONT color="#009933">Agarose gel electrophoresis</FONT></P><P>-&gt; Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis">11 Agarose gel electrophoresis</A>)
</P><P>- Agarose gel electrophoresis of the restriction digest of PhiC31o and PCR 1 and 6
</P><P>- <FONT color="#CC33CC"><B>the right bands found for PhiC31o (~2900,~2400,~250)</B></FONT></P><P>- <FONT color="#CC33CC"><B>the right band found for PCR1 (~450)</B></FONT></P><P>- no band found for PCR6; new electrophoresis needed with more DNA loaded
</P><P><TABLE><TBODY><TR><TD>
Agarose gel electrophoresis of (from left to right) PhiC31o, PCR1 and PCR6
</TD><TD>
Agarose gel electrophoresis of (from left to right) PhiC31o, PCR1 and PCR6 which shows that PCR1 is between 250 and 500 bp
</TD></TR></TBODY></TABLE></P><P>- new agarose gel electrophoresis from PCR6 with 5µl DNA instead of 3µl (image not yet shown)
</P><P>-<FONT color="#CC33CC"><B>the right band found for PCR6 (~200)</B></FONT></P><P><FONT color="#009933">New overnight cultures of CMV and pDS7</FONT></P><P>- the overnight colonies didn't grow; new colonies (CMV and pDS7) picked from plate and inoculated in LB Ampicillin
</P><P><FONT color="#009933">PCR purification of PCR 1 and 6</FONT></P><P>-&gt; Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/12_Gel_extraction_or_PCR_Clean_up" title="Team:LMU-Munich/Notebook/Protocols/12 Gel extraction or PCR Clean up">12 Gel extraction or PCR Clean up</A>)
</P><P>- DNA concentration of the PCR 1 and 6 products measured: PCR1: 410ng/µl (A260/A280=1.253) PCR6: 568ng/µl (A260/A280=1.275)
</P><P>- PCR Purification with Promega Kit
</P><P>-&gt; PCR1: 230ng/µl (A260/A280=1.769)
</P><P>-&gt; PCR6: 37.5ng/µl (A260/A280=1.667)
</P><H2><SPAN class="mw-headline" id="8-18-2010"> 8-18-2010 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Agarose gel electrophoresis of (from left to right) CMV and pDS7 showing the right bands for pDS7</DIV></DIV></DIV><P><FONT color="#009933">Plasmid Extraction of CMV and pDS7</FONT></P><P>-&gt; Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A>)
</P><P>- Plasmid extraction of CMV (97.5ng/µl; A260/A280=1.857) and pDS7 (212ng/µl; A260/A280=1.848)
</P><P><FONT color="#009933">Restriction digestion</FONT></P><P>-&gt; Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digestion</A>)
</P><P>- Restriction digestion of CMV (EcoR1 + Pst1; 10µl DNA, buffer H) and pDS7 (EcoR1 + Spe1; 5µl DNA, buffer B)
</P><P>-&gt; expected DNA bands: CMV: 2079bp (plasmid) + 654bp (Insert); pDS7: 7022bp + 1011bp 
</P><P><FONT color="#009933">Agarose Gel electrophoresis of digested CMV and pDS7</FONT></P><P>-&gt; Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis">11 Agarorse gel electrophoresis</A>)
</P><P>-&gt;<FONT color="#CC33CC"><B>right DNA bands for pDS7 (~7000bp, ~1000bp)</B></FONT></P><P>-&gt; false DNA bands for CMV
</P><P>- Starting PCR 2a and 2b (replication and mutagenesis of pDS7): 3 µl DNA and 50°C Annealing Temperatur (other same as 8-16-2010)
</P><H2><SPAN class="mw-headline" id="8-19-2010"> 8-19-2010 </SPAN></H2><P><FONT color="#009933">Agarose gel electrophoresis of PCR 2a and 2b</FONT></P><P>-&gt; Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis">11 Agarose gel electrophoresis</A>)
(150V, 30min)
</P><TABLE><TBODY><TR><TD>Agarose gel electrophoresis of (from left to right) PCR2a and PCR2b
</TD></TR></TBODY></TABLE><P>-&gt;<FONT color="#CC33CC"><B>the right bands for PCR2a (~300bp) and PCR2b (~700bp)</B></FONT></P><P>- New agarose gel electrophoresis with all of the PCR product for gel extraction (150V, 30min)
</P><P><FONT color="#009933">Gel extraction of the DNA from PCR2a and PCR2b</FONT></P><P>-&gt; Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/12_Gel_extraction_or_PCR_Clean_up" title="Team:LMU-Munich/Notebook/Protocols/12 Gel extraction or PCR Clean up">12 Gel extraction or PCR Clean up</A>)
</P><P>- DNA concentration measured; problem with nanodrop as too low concentration; lyophille used to reduce volume
</P><P>- DNA concentration measured again: PCR2a: 70ng/µl A260/A280=1.647;  PCR2b: 45ng/µl A260/A280=1.5  
</P><P><FONT color="#009933">PCR 3 (joining PCR of 2a and 2b)</FONT></P><P>- PCR3 (the joining PCR of PCR2a and 2b; Joining of the TEVrecogn-N-Degron-SF3 part) done: 1.3 µl of PCR2a and 4.7 µl of PCR2b makes 300ng of a 1:1 solution of both to be joined DNA parts. Annealing temperature: 50°C
</P><P>-&gt; Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu">10 PCR with Pfu</A>)
</P><H2><SPAN class="mw-headline" id="8-20-2010"> 8-20-2010 </SPAN></H2><P><FONT color="#009933">Agarose gel electrophoresis of PCR3</FONT></P><P>left column: marker; rightmost column: PCR3
</P><P>-&gt; Protocol:   <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A> (150V, 30min)
</P><P>-&gt; expected band: ~1000bp
</P><P>-&gt; false band: ~500bp
</P><P>- probable reason: mini photometre was influenced by gel extraction chemicals, therefore it measured false DNA concentrations and false template masses were calculated
</P><P>-&gt; New 2a and 2b PCR
</P><P><FONT color="#009933">New PCR (2a and 2b)</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P>(see 8-18-2010, but 35,5µl water)
</P><H2><SPAN class="mw-headline" id="8-21-2010"> 8-21-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="8-22-2010"> 8-22-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="8-23-2010"> 8-23-2010 </SPAN></H2><P><FONT color="#009933">Agarose gel electrophoresis of PCR 2b</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>- expected band: 700bp
</P><P>-&gt; no band shown on gel -&gt; new PCR 2b
</P><P><FONT color="#009933">PCR 2b</FONT></P><P>- start PCR 2b with PCR 2b from 8-13-10 as template ( 1:20 and 1:100 diluted; 1µl)
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P>- annealing temperature: 50°C; amount of water: 37,5µl
</P><P><FONT color="#009933">Agarose gel electrophoresis of PCR 2b 1:20 and 1:100</FONT></P><P>-&gt; Protocol:  <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>- expected bands: each ~ 700bp
</P><P>- false bands: ~ 200bp
</P><P>-&gt; new PCR with 2ng, 5ng, 10ng template pDS7
</P><P><FONT color="#009933">PCR 2b  with 2ng, 5ng, 10ng template pDS7</FONT></P><P>- pDS7 1:100 diluted(-&gt; 2,1 ng/µl)
</P><P>Mixture:
</P><TABLE><TBODY><TR><TD>2ng
</TD><TD>5ng
</TD><TD>10ng
</TD></TR><TR><TD>Primer
</TD><TD>2*2,5µl (P5+P6)
</TD><TD>2*2,5µl (P5+P6)
</TD><TD>2*2,5µl (P5+P6)
</TD></TR><TR><TD>10x Buffer Pfu
</TD><TD>5µl
</TD><TD>5µl
</TD><TD>5µl
</TD></TR><TR><TD>dNTP Mix
</TD><TD>1µl
</TD><TD>1µl
</TD><TD>1µl
</TD></TR><TR><TD>template
</TD><TD>pDS7 (dil.)
</TD><TD>1µl
</TD><TD>2,5µl
</TD><TD>5µl
</TD></TR><TR><TD>Pfu Polymerase (3u/µl)
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD><TD>1,25µl
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>33,25µl
</TD><TD>30,25µl
</TD><TD>25,25µl
</TD></TR><TR><TD>sum
</TD></TR></TBODY></TABLE><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P><FONT color="#009933">PCR 2a gel extraction</FONT></P><P>- Quiagen kit (QuiaexII)
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/14_QIAEX_II_gel_extraction" title="Team:LMU-Munich/Notebook/Protocols/14 QIAEX II gel extraction"> 14 QIAEX II gel extraction</A></P><P><FONT color="#009933">Start 3 CMV overnight cultures</FONT></P><H2><SPAN class="mw-headline" id="8-24-2010"> 8-24-2010 </SPAN></H2><P><FONT color="#009933">agarose gel electrophoresis of PCR 2b</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Agarose gel electrophoresis of (from left to right) PCR2b (2ng (cut out), 10ng, 5ng template) showing the right bands for 2ng, 5ng template</DIV></DIV></DIV><P>- expected bands:<FONT color="#CC33CC"><B>right bands with 2ng and 5ng template (~700bp)</B></FONT>, no band with 10ng template
</P><P><FONT color="#009933">CMV plasmid extraction</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><P>Plasmid extractionof 3 different overnight cultures.
</P><P>- results: 
</P><UL><LI> 52,5 ng/µl A260/A280= 1.312
</LI><LI> 133 ng/µl A260/A280= 1.710
</LI><LI> 80 ng/µl A260/A280= 1.600
</LI></UL><P><FONT color="#009933">CMV restriction digestion</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digestion</A></P><P>- CMV restriction digest: EcoRI, PstI, buffer H
</P><UL><LI> 19µl, H<SUB>2</SUB>O : 0µl
</LI><LI> 6µl, H<SUB>2</SUB>O : 9.5µl
</LI><LI> 12.5µl, H<SUB>2</SUB>O : 3µl
</LI></UL><P><FONT color="#009933">PCR 2b gel extraction</FONT></P><P>- PCR2b was gel extracted (with Qiagen gel extraction kit), 17.5 ng/µl a260/A280= 1.750
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/14_QIAEX_II_gel_extraction" title="Team:LMU-Munich/Notebook/Protocols/14 QIAEX II gel extraction"> 14 QIAEX II gel extraction</A></P><P><FONT color="#009933">PCR 3 (fusion of 2a and 2b)</FONT></P><P>- PCR3: conducted again at 52°C annealing temperature
</P><UL><LI> 10.5 ng (from PCR2b) 0.6µl
</LI></UL><UL><LI> 4.5 ng  (from PCR2a) 0.9µl (1:10 diluted)
</LI></UL><DL><DD><TABLE><TBODY><TR><TD>PCR2a
</TD><TD>0.9 µl
</TD></TR><TR><TD>PCR2b
</TD><TD>0.6 µl
</TD></TR><TR><TD>primer3
</TD><TD>2.5 µl
</TD></TR><TR><TD>primer6
</TD><TD>2.5 µl
</TD></TR><TR><TD>dNTPs
</TD><TD>1   µl
</TD></TR><TR><TD>Pfu
</TD><TD>0.5 µl
</TD></TR><TR><TD>10xbuffer
</TD><TD>5 µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>37 µl
</TD></TR></TBODY></TABLE></DD></DL><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P><FONT color="#009933">agarose gel electrophoresis  of CMV digestion</FONT></P><P>- agarose gel electrophoresis (150V, 25 min) of the CMV digestion
</P><P>-&gt; bands are wrong again ( ~ 1200bp, 2000bp)
</P><H2><SPAN class="mw-headline" id="8-25-2010"> 8-25-2010 </SPAN></H2><P><FONT color="#009933">Agarose gel electrophorese of PCR 3</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>- expected band: ~1000bp
</P><P>- false band: ~400bp
</P><P><FONT color="#009933">Plasmid extraction of ccdB tet and ccdB strep</FONT></P><P>Plasmid extraction of pSB1C3 with BBa_P1010 
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><P>- results:
</P><TABLE><TBODY><TR><TD>ccdB tet:
</TD><TD>50ng/µl;
</TD><TD>A260/A280= 1,818
</TD></TR></TBODY></TABLE><P><FONT color="#009933">Plate ccdB amp, cam, tet</FONT></P><P>- plate ccdB with ampicilline, chloramphenicol, tetracycline resistance on LB agar with appropiate antibiotic.
</P><P><FONT color="#009933">Overnight culture of ccdB kan</FONT></P><P>- Overnight culture of ccdB with kanamycine resistance in LB medium with kanamycine
</P><P><FONT color="#009933">PCR 7a, 7b, 9, 10</FONT></P><P>-&gt;Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><TABLE><TBODY><TR><TD>PCR nr.
</TD><TD>template
</TD><TD>concentration
</TD><TD>dilution
</TD><TD>primer
</TD></TR><TR><TD>7a
</TD><TD>190-6
</TD><TD>~200ng/µl
</TD><TD>1:100
</TD><TD>13,14
</TD></TR><TR><TD>7b
</TD><TD>190-6
</TD><TD>~200ng/µl
</TD><TD>1:100
</TD><TD>15,16
</TD></TR><TR><TD>9
</TD><TD>eGFP
</TD><TD>55ng/µl
</TD><TD>1:25
</TD><TD>20,21
</TD></TR><TR><TD>10
</TD><TD>PhiC31o
</TD><TD>20ng/µl
</TD><TD>1:10
</TD><TD>22,23
</TD></TR></TBODY></TABLE><P>
Mixture
</P><TABLE><TBODY><TR><TD>template (~2ng)
</TD><TD>1µl
</TD></TR><TR><TD>Pfu
</TD><TD>0,5µl
</TD></TR><TR><TD>Primer *2
</TD><TD>2,5µl *2
</TD></TR><TR><TD>10x buffer
</TD><TD>5µl
</TD></TR><TR><TD>dNTP Mix
</TD><TD>1µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>37,5µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE><P>Standard PCR; annealing temperature: 60°C
</P><H2><SPAN class="mw-headline" id="8-26-2010"> 8-26-2010 </SPAN></H2><P><FONT color="#009933">Agarose gelelectrophoresis of PCR 7a, 7b, 9, 10</FONT></P><P>-&gt;Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>- 150V, 25min
</P><TABLE><TBODY><TR><TD>PCR nr.
</TD><TD>expected bands
</TD><TD>result
</TD></TR><TR><TD>7a
</TD><TD>850bp
</TD><TD>no band
</TD></TR><TR><TD>7b
</TD><TD>402bp
</TD><TD>false band (200bp)
</TD></TR><TR><TD>9
</TD><TD>808bp
</TD><TD>no band
</TD></TR><TR><TD>10
</TD><TD>1888bp
</TD><TD>no band
</TD></TR></TBODY></TABLE><P><FONT color="#009933">Plasmid extraction of ccdB kan</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><P>-result: concentration: 25ng/µl; A260/A280= 2,0
</P><P><FONT color="#009933">New PCR 7a, 7b, 9, 10</FONT></P><P>Mixture
</P><TABLE><TBODY><TR><TD>template (~2ng)
</TD><TD>1µl
</TD></TR><TR><TD>Pfu
</TD><TD>0,5µl
</TD></TR><TR><TD>Primer *2
</TD><TD>2,5µl *2
</TD></TR><TR><TD>10x buffer
</TD><TD>5µl
</TD></TR><TR><TD>dNTP Mix
</TD><TD>1µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>36,25µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE><P>Program:
</P><P>gradient PCR, 42-69°C annealing temp.
</P><P>-&gt;Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P><FONT color="#009933">Overnight culture of ccdB amp, tet, cam</FONT></P><P>Inoculate one colony each in 5ml medium with approptraite antibiotic.
</P><H2><SPAN class="mw-headline" id="8-27-2010"> 8-27-2010 </SPAN></H2><P><FONT color="#009933">Agarose gel electrophoresis of PCR 7a, 7b, 9, 10</FONT></P><P>-&gt;Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>150V, 25min, 75mA
</P><P>from left to right: 7a, 7b, 9, 10, Marker
</P><TABLE><TBODY><TR><TD>PCR nr.
</TD><TD>expected bands
</TD><TD> result
</TD></TR><TR><TD>7a
</TD><TD>850bp
</TD><TD>no band
</TD></TR><TR><TD><FONT color="#CC33CC"><B>7b</B></FONT></TD><TD><FONT color="#CC33CC"><B>402bp</B></FONT></TD><TD><FONT color="#CC33CC"><B>right band (~400bp)</B></FONT>+ false band (~150bp)
</TD></TR><TR><TD>9
</TD><TD>808bp
</TD><TD>false band (~200bp)
</TD></TR><TR><TD><FONT color="#CC33CC"><B>10</B></FONT></TD><TD><FONT color="#CC33CC"><B>1888bp</B></FONT></TD><TD><FONT color="#CC33CC"><B>right band (~1900bp)</B></FONT>+false band (~500bp)
</TD></TR></TBODY></TABLE><P><FONT color="#009933">Plasmid extraktion of ccdB amp, tet, cam</FONT></P><P>-&gt;Protocol:  <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><P>results:
</P><TABLE><TBODY><TR><TD>Plasmid
</TD><TD>concentration
</TD><TD>A260/A280
</TD></TR><TR><TD>ccdB amp
</TD><TD>57,5 ng/µl
</TD><TD>1,917
</TD></TR><TR><TD>ccdB cam
</TD><TD>70,0 ng/µl
</TD><TD>1,867
</TD></TR><TR><TD>ccdB tet
</TD><TD>50,0 ng/µl
</TD><TD>1,818
</TD></TR></TBODY></TABLE><P><FONT color="#009933">New PCR 7a, 9</FONT></P><P>Mixture:
</P><P>- 2ng template: see 26-8-10
</P><P>- 4ng template: see 26-8-10, but 2µl template and 35,25µl water
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P><FONT color="#009933">Restriction digestion of ccdB amp, kan, cam, tet</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digestion</A></P><P>- only 90min 37°C incubation
</P><P>- EcoRI, PstI, Buffer H
</P><TABLE><TBODY><TR><TD>template
</TD><TD>volume
</TD><TD>mass
</TD></TR><TR><TD>ccdB amp
</TD><TD>16µl
</TD><TD>930ng
</TD></TR><TR><TD>ccdB cam
</TD><TD>14,3µl
</TD><TD>1µg
</TD></TR><TR><TD>ccdB tet
</TD><TD>16µl
</TD><TD>800ng
</TD></TR><TR><TD>ccdB kan
</TD><TD>16µl
</TD><TD>400ng
</TD></TR></TBODY></TABLE><P><FONT color="#009933">Agarose gelelectrophoresis of PCR 7a, 9, ccdB restriction digestion</FONT></P><P>150v, 25min, 75mA
</P><P>-&gt; Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>results:
</P><P>- PCR7a, 9: false band at 200bp
</P><P>- <FONT color="#CC33CC"><B>ccdB: each digestion leads to a right band with ~ 650bp</B></FONT></P><H2><SPAN class="mw-headline" id="8-28-2010"> 8-28-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="8-29-2010"> 8-29-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="8-30-2010"> 8-30-2010 </SPAN></H2><P><FONT color="#009933">New PCR 7a and 9</FONT></P><P>
Mixture
</P><TABLE><TBODY><TR><TD>template (~4ng)
</TD><TD>2µl
</TD></TR><TR><TD>Pfu
</TD><TD>0,5µl
</TD></TR><TR><TD>Primer *2
</TD><TD>2,5µl *2
</TD></TR><TR><TD>10x buffer
</TD><TD>5µl
</TD></TR><TR><TD>dNTP Mix
</TD><TD>1µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>35,25µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE><P>
-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P>
PCR program
</P><P>PCR 7: Annealing Temperature 60°C - 25 x 1 min Annealing time and 5x 1,30 min Annealing time
</P><P>PCR 9:  Annealing Temperature 55°C - 25 x 1 min Annealing time and 5x 1,30 min Annealing time
</P><P><FONT color="#009933">Gel extraction of PCR 7b, 10</FONT></P><P>-&gt;Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/14_QIAEX_II_gel_extraction" title="Team:LMU-Munich/Notebook/Protocols/14 QIAEX II gel extraction"> 14 QIAEX II gel extraction</A></P><P>results:
</P><TABLE><TBODY><TR><TD>PCR nr.
</TD><TD>concentration
</TD><TD>A260/A280
</TD></TR><TR><TD>7b
</TD><TD>10 ng/µl
</TD><TD>2,0
</TD></TR><TR><TD>10
</TD><TD>17,5 ng/µl
</TD><TD>1,4
</TD></TR></TBODY></TABLE><P><FONT color="#009933">Agarose gel electrophoresis of new PCR 7a, 9</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>150V, 25min
</P><P>- results:
</P><P>- 7a: no band shown
</P><P>- 9: false band (~200bp)
</P><P><FONT color="#009933">New PCR 3</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><DL><DD><TABLE><TBODY><TR><TD>PCR2a
</TD><TD>0.9 µl
</TD></TR><TR><TD>PCR2b
</TD><TD>0.6 µl
</TD></TR><TR><TD>dNTPs
</TD><TD>1   µl
</TD></TR><TR><TD>Pfu
</TD><TD>0.5 µl
</TD></TR><TR><TD>10xbuffer
</TD><TD>5 µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>36,75 µl
</TD></TR><TR><TD>sum
</TD><TD>45µl
</TD></TR></TBODY></TABLE></DD></DL><P>- new method: standard PCR without primers (10 cycles, 56°C annealing temp.)
</P><P>- then add 2,5µl of primer 3 and 6
</P><P>- 30 cycles standard PCR (54°C annealing temp.)
</P><H2><SPAN class="mw-headline" id="8-31-2010"> 8-31-2010 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel photo of (left to right) PCR4a(2.5ng template), PCR4a(5ng template),PCR4b(2.5ng template), PCR4b(5ng template), PCR3(Pfu), PCR3(Phusion)</DIV></DIV></DIV><P><FONT color="#009933">Agarose gel electrophoresis of PCR 4a, PCR4b, PCR3(Pfu), PCR3(Phusion)</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>150V, 25min
</P><P>- results:
</P><P>- PCR4a(2.5ng template), PCR4a(5ng template),PCR4b(2.5ng template), PCR4b(5ng template), PCR3(Pfu): no band shown
</P><P>- <FONT color="#CC33CC"><B>PCR3 (Phusion): right band (~1000bp)</B></FONT></P><P><FONT color="#009933">New PCR  PCR4a, PCR4b, PCR7a, PCR9</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P>PCR mixture for PCR4a, PCR4b
</P><DL><DD><TABLE><TBODY><TR><TD>template
</TD><TD>37.25 µl (200ng)
</TD></TR><TR><TD>dNTPs
</TD><TD>1   µl
</TD></TR><TR><TD>Pfu
</TD><TD>0.5 µl
</TD></TR><TR><TD>10xbuffer
</TD><TD>5 µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE></DD></DL><P>Standard PCR program with annealing temperature PCR4a: 51.1°C, PCR4b: 48.5°C. 
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/15_PCR_with_Phusion" title="Team:LMU-Munich/Notebook/Protocols/15 PCR with Phusion"> 15 PCR with Phusion</A></P><P>PCR mixture for PCR7a, PCR9
</P><DL><DD><TABLE><TBODY><TR><TD>template
</TD><TD>4 µl (8ng)
</TD></TR><TR><TD>dNTPs
</TD><TD>1   µl
</TD></TR><TR><TD>Phusion
</TD><TD>0.5 µl
</TD></TR><TR><TD>5xbuffer
</TD><TD>10 µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>28.25 µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE></DD></DL><P>PCR program: Phu62
</P><DL><DD><TABLE><TBODY><TR><TD>98°C
</TD><TD>1 min
</TD></TR><TR><TD>98°C
</TD><TD>10 sec
</TD></TR><TR><TD>62°C
</TD><TD>20 sec
</TD></TR><TR><TD>73°C
</TD><TD>30 sec
</TD></TR><TR><TD>return to step 2 for 29 cycles
</TD></TR><TR><TD>73°C
</TD><TD>10 min
</TD></TR><TR><TD>12°C
</TD><TD>forever
</TD></TR></TBODY></TABLE></DD></DL><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel photo of (left to right) PCR3, ladder, PCR7a, empty, PCR9</DIV></DIV></DIV><P><FONT color="#009933">Agarose gel electrophoresis of PCR 3, PCR7a, PCR9 for gel extraction</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>120V, 30min
</P><P>- results:
</P><P>-<FONT color="#CC33CC"><B>PCR3; right band (~1000bp)</B></FONT> and side-product
</P><P>- PCR7a: no band
</P><P>- PCR9: right band (~800bp) and side-product
</P><P><FONT color="#009933">Gel extraction of the DNA from PCR3 and PCR9</FONT></P><P>-&gt; Protocol (<A href="/Team:LMU-Munich/Notebook/Protocols/12_Gel_extraction_or_PCR_Clean_up" title="Team:LMU-Munich/Notebook/Protocols/12 Gel extraction or PCR Clean up">12 Gel extraction or PCR Clean up</A>)
</P><P>results:
</P><P>PCR 9: 22,5ng/µl; A260/A280=1,8
</P><P>PCR 3: 22,5ng/µl; A260/A280=2,25
</P><P><FONT color="#009933">New PCR  4a, 4b, 7a with DreamTaq</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/16_PCR_with_DreamTaq" title="Team:LMU-Munich/Notebook/Protocols/16 PCR with DreamTaq"> 16 PCR with DreamTaq</A></P><P>PCR mixture for PCR7a 
</P><DL><DD><TABLE><TBODY><TR><TD>template
</TD><TD>5 µl (10ng)
</TD></TR><TR><TD>dNTPs
</TD><TD>5   µl
</TD></TR><TR><TD>DreamTaq
</TD><TD>0.33 µl
</TD></TR><TR><TD>10xbuffer
</TD><TD>5 µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>28.5 µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE></DD></DL><P>Primers for PCR 7a: 13,14
</P><P>Annealing temp: 60°C
</P><P>PCR mixture for PCR4a,4b
</P><DL><DD><TABLE><TBODY><TR><TD>template
</TD><TD>36,5 µl (180ng HeLa cDNA)
</TD></TR><TR><TD>dNTPs
</TD><TD>5   µl
</TD></TR><TR><TD>DreamTaq
</TD><TD>0.33 µl
</TD></TR><TR><TD>10xbuffer
</TD><TD>5 µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE></DD></DL><P>PCR 4a
</P><P>Primers for PCR 4a: 7,8
</P><P>Primers for PCR 4b: 9,10
</P><P>Annealing temp: 50°C
</P><P>PCR program: 
</P><DL><DD><TABLE><TBODY><TR><TD>95°C
</TD><TD>1 min
</TD></TR><TR><TD>95°C
</TD><TD>30 sec
</TD></TR><TR><TD>50/60°C
</TD><TD>30 sec
</TD></TR><TR><TD>72°C
</TD><TD>1 min (1kb/min)
</TD></TR><TR><TD>return to step 2 for 29 cycles
</TD></TR><TR><TD>72°C
</TD><TD>10 min
</TD></TR><TR><TD>12°C
</TD><TD>forever
</TD></TR></TBODY></TABLE></DD></DL><H2><SPAN class="mw-headline" id="9-01-2010"> 9-01-2010 </SPAN></H2><P><FONT color="#009933">Agarose gel electrophoresis of PCR4a, PCR4b, PCR7 (DreamTaq)</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>150V, 25min
</P><P>results: no product
</P><P><FONT color="#009933">New PCR  4a, 4b with DreamTaq, Pfu, with concentration gradient and touch-down PCR </FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/16_PCR_with_DreamTaq" title="Team:LMU-Munich/Notebook/Protocols/16 PCR with DreamTaq"> 16 PCR with DreamTaq</A>; <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P>PCR mixture for DreamTaq 
</P><DL><DD><TABLE><TBODY><TR><TD>Concentration
</TD><TD>Low
</TD><TD>Middle
</TD><TD>High
</TD></TR><TR><TD>template
</TD><TD>5 µl (1:1000)
</TD><TD>31.75µl (1:1000)
</TD><TD>2µl (1:10)
</TD></TR><TR><TD>dNTPs
</TD><TD>5 µl
</TD></TR><TR><TD>DreamTaq
</TD><TD>0.33 µl
</TD></TR><TR><TD>10xbuffer
</TD><TD>5 µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>26.75 µl
</TD><TD>0
</TD><TD>29.75
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE></DD></DL><P>PCR mixture for Pfu
</P><DL><DD><TABLE><TBODY><TR><TD>Concentration
</TD><TD>Low
</TD><TD>Middle
</TD><TD>High
</TD></TR><TR><TD>template
</TD><TD>5 µl (1:1000)
</TD><TD>37.25µl (1:1000)
</TD><TD>2µl (1:10)
</TD></TR><TR><TD>dNTPs
</TD><TD>1 µl
</TD></TR><TR><TD>Pfu
</TD><TD>0.5 µl
</TD></TR><TR><TD>10xbuffer
</TD><TD>5 µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>32.25 µl
</TD><TD>0
</TD><TD>35.25 µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE></DD></DL><P>Primers for PCR 4a: 7,8; PCR 4b: 9,10
</P><P>-&gt; Protocol: Thermal cycler program: <A href="/Team:LMU-Munich/Notebook/Protocols/17_Special_thermal_cycler_programs/Touch_down" title="Team:LMU-Munich/Notebook/Protocols/17 Special thermal cycler programs/Touch down">Touch down</A></P><P><FONT color="#009933">Agarose gel electrophoresis of PCR4a, PCR4b</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>150V, 25min
</P><P>from left to right: 4a: P1, P2, P3, D1, D2, D3; 4b: P1, P2, P3, D1, D2, D3
</P><P>key:
</P><P>&quot;P&quot;= PCR with Pfu
</P><P>&quot;D&quot;= PCR with DreamTaq
</P><P>&quot;1&quot;= low template concentration
</P><P>&quot;2&quot;= middle template concentration
</P><P>&quot;3&quot;= high template concentration
</P><P>expected bands: 
</P><UL><LI><FONT color="#CC33CC"><B>4a: 330bp -&gt; P2 and P3 show right bands </B></FONT>and &quot;primer clouds&quot;(?)
</LI></UL><UL><LI><FONT color="#CC33CC"><B>4b: 376bp -&gt; P1, P2, P3 show right bands</B></FONT> and &quot;primer clouds&quot; (?)
</LI></UL><H2><SPAN class="mw-headline" id="9-02-2010"> 9-02-2010 </SPAN></H2><P><FONT color="#009933">Agarose gel electrophorese of PCR 4a P2, 4b P2</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>- 120V, 45min, 1,5% Agarose gel
</P><P>Agarose gel electrophoresis of (from left to right) PCR4aP2, Marker and PCR4bP2
</P><P>
- Cut out bands at ~~ 350bp and extract
</P><P><FONT color="#009933">PCR Agarose gel extraction</FONT></P><P>-&gt; Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/14_QIAEX_II_gel_extraction" title="Team:LMU-Munich/Notebook/Protocols/14 QIAEX II gel extraction"> 14 QIAEX II gel extraction</A></P><P>results:
</P><TABLE><TBODY><TR><TD>PCR nr.
</TD><TD>concentration
</TD><TD>A260/A280
</TD></TR><TR><TD>4aP2
</TD><TD>12,5 ng/µl
</TD><TD>1,67
</TD></TR><TR><TD>4bP2
</TD><TD>72,5 ng/µl
</TD><TD>1,53
</TD></TR></TBODY></TABLE><P><FONT color="#009933">New PCR 7a with Pfu and Phusion</FONT></P><P>template: 190-6, Primer 13,14
</P><P>Mixture with Pfu
</P><TABLE><TBODY><TR><TD>template (~2ng)
</TD><TD>1µl
</TD></TR><TR><TD>Pfu
</TD><TD>0,5µl
</TD></TR><TR><TD>Primer *2
</TD><TD>2,5µl *2
</TD></TR><TR><TD>10x buffer
</TD><TD>5µl
</TD></TR><TR><TD>dNTP Mix
</TD><TD>1µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>36,25µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE><P>
-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P>PCR mixture with Phusion
</P><DL><DD><TABLE><TBODY><TR><TD>template (~2ng)
</TD><TD>1,0 µl
</TD></TR><TR><TD>dNTPs
</TD><TD>1   µl
</TD></TR><TR><TD>Phusion
</TD><TD>0.5 µl
</TD></TR><TR><TD>5xbuffer
</TD><TD>10 µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>31,25µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE></DD></DL><P>-&gt; Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/15_PCR_with_Phusion" title="Team:LMU-Munich/Notebook/Protocols/15 PCR with Phusion"> 15 PCR with Phusion</A></P><P><FONT color="#009933">New PCR 4a, 4b with Pfu</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P>-&gt; Protocol: Thermal cycler program: <A href="/Team:LMU-Munich/Notebook/Protocols/17_Special_thermal_cycler_programs/Touch_down" title="Team:LMU-Munich/Notebook/Protocols/17 Special thermal cycler programs/Touch down">Touch down</A></P><P>Mixture see 9-1-10, twice 4a and 4b
</P><P><FONT color="#009933">Agarose gel electrophorese of PCR 4a, 4b, 7a, gel extracted 4a and 4b</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>- 25min, 150V
</P><P>from left to right: 4a*, 4a, 4b*, 4b, 7a Phusion, 7a Pfu, Ladder
</P><P>-&gt;<FONT color="#CC33CC"><B>result: 4b, 4b*: right bands (~330bp)</B></FONT></P><P>-remain: false bands/no band
</P><P>from left to right: ladder, 4 columns pathway, 4a gelextr., 4b gelextr.
</P><P>-&gt;<FONT color="#CC33CC"><B>results: slight right bands for 4a and 4b</B></FONT>, no &quot;primer clouds&quot; anymore.
</P><H2><SPAN class="mw-headline" id="9-03-2010"> 9-03-2010 </SPAN></H2><P><FONT color="#009933">Overlapping PCR 5 with Pfu and Phusion</FONT></P><P>template: 4a, 4b Primer 7,10
</P><P>Mixture with Pfu
</P><TABLE><TBODY><TR><TD>PCR4a (~15ng)
</TD><TD>1.2µl
</TD></TR><TR><TD>PCR4b (~15ng)
</TD><TD>2µl (1:10)
</TD></TR><TR><TD>Pfu
</TD><TD>0.5µl
</TD></TR><TR><TD>Primer *2
</TD><TD>2.5µl *2
</TD></TR><TR><TD>10x buffer
</TD><TD>5µl
</TD></TR><TR><TD>dNTP Mix
</TD><TD>1µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>34.05µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P>PCR program: standard PCR program for Pfu, Annealing temperature: 54°C
</P><P>PCR mixture with Phusion
</P><DL><DD><TABLE><TBODY><TR><TD>PCR4a (~15ng)
</TD><TD>1.2µl
</TD></TR><TR><TD>PCR4b (~15ng)
</TD><TD>2µl (1:10)
</TD></TR><TR><TD>dNTPs
</TD><TD>1   µl
</TD></TR><TR><TD>Phusion
</TD><TD>0.5 µl
</TD></TR><TR><TD>5xbuffer
</TD><TD>10 µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>29.05µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE></DD></DL><P>-&gt; Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/15_PCR_with_Phusion" title="Team:LMU-Munich/Notebook/Protocols/15 PCR with Phusion"> 15 PCR with Phusion</A></P><P>PCR program: standard PCR program for Phusion, Annealing temperature: 58°C
</P><P><FONT color="#009933">New PCR7a with Pfu</FONT></P><P>template: 190-6; Primer: 13,14
</P><P>Mixture with Pfu
</P><TABLE><TBODY><TR><TD>template (~2ng)
</TD><TD>1µl
</TD></TR><TR><TD>Pfu
</TD><TD>0.5µl
</TD></TR><TR><TD>Primer *2
</TD><TD>2.5µl *2
</TD></TR><TR><TD>10x buffer
</TD><TD>5µl
</TD></TR><TR><TD>dNTP Mix
</TD><TD>1µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>36.25µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P>PCR program: standard PCR program for Pfu, Gradient: 54.4°C, 57.8°C, 61.4°C, 65.0°C
</P><P>-&gt; results:
</P><P>-7a: only &quot;primer cluods&quot;
</P><P>-5 Pfu: no defined product (slurred?)
</P><P>-5 Phusion: &quot;primer clouds&quot;
</P><H2><SPAN class="mw-headline" id="9-04-2010"> 9-04-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="9-05-2010"> 9-05-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="9-06-2010"> 9-06-2010 </SPAN></H2><P>charges for sequencing
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> name
</TH><TH> 4a-7
</TH><TH> 4a-8
</TH><TH> 4b-9
</TH><TH> 4b-10
</TH><TH> 3-3
</TH><TH> 3-6
</TH><TH> 6-11
</TH><TH> 6-12
</TH></TR><TR><TD> DNA
</TD><TD> 4a; 2.4µl
</TD><TD> 4a; 2.4µl
</TD><TD> 4b; 0.5µl
</TD><TD> 4b; 0.5µl
</TD><TD> 3; 2µl
</TD><TD> 3; 2µl
</TD><TD> 6; 0.5µl
</TD><TD> 6; 0.5µl
</TD></TR><TR><TD> primer [1pmol/µl]
</TD><TD> 7; 3.2µl
</TD><TD> 8; 3.2µl
</TD><TD> 9; 3.2µl
</TD><TD> 10; 3.2µl
</TD><TD> 3; 3.2µl
</TD><TD> 6; 3.2µl
</TD><TD> 11; 3.2µl
</TD><TD> 12; 3.2µl
</TD></TR><TR><TD> Tris (10mM); pH 8.2
</TD><TD> 1.4µl
</TD><TD> 1.4µl
</TD><TD> 3.3µl
</TD><TD> 3.3µl
</TD><TD> 1.8µl
</TD><TD> 1.8µl
</TD><TD> 3.3µl
</TD><TD> 3.3µl
</TD></TR></TBODY></TABLE><P>every charge 7µl
</P><P><FONT color="#009933">New overlapping PCR 5 with Pfu</FONT></P><P>template: 4a, 4b; 15ng
</P><P>Primer 7,10
</P><P>Mixture
</P><TABLE><TBODY><TR><TD>PCR4a (~7ng)
</TD><TD>0.56µl
</TD></TR><TR><TD>PCR4b (~8ng)
</TD><TD>1.1µl (1:10)
</TD></TR><TR><TD>Pfu
</TD><TD>0.5µl
</TD></TR><TR><TD>Primer *2
</TD><TD>2.5µl *2
</TD></TR><TR><TD>10x buffer
</TD><TD>5µl
</TD></TR><TR><TD>dNTP Mix
</TD><TD>1µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>36.6µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P>PCR program: standard PCR program for Pfu, Annealing temperature: 46.1°C,48.5°C, 51.1°C
</P><P>-&gt; results: &quot;primer clouds&quot;
</P><P><FONT color="#009933">New PCR 7a with Pfu</FONT>
new dilution of 190-6; 1:100
</P><TABLE><TBODY><TR><TD>template[2ng/µl]
</TD><TD>1µl
</TD><TD>2µl
</TD></TR><TR><TD>pfu polymerase
</TD><TD>0.5µl
</TD><TD>0.5µl
</TD></TR><TR><TD>primer13
</TD><TD>2.5µl
</TD><TD>2.5µl
</TD></TR><TR><TD>primer14
</TD><TD>2.5µl
</TD><TD>2.5µl
</TD></TR><TR><TD>buffer
</TD><TD>5µl
</TD><TD>5µl
</TD></TR><TR><TD>dNTP Mix
</TD><TD>1µl
</TD><TD>1µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD><TD>1.25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>36.25µl
</TD><TD>35.25
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD><TD>50µl
</TD></TR></TBODY></TABLE><P><FONT color="#009933">Agarose gel electrophoresis of 7a</FONT></P><P>-&gt; results: &quot;primer clouds&quot;
</P><P><FONT color="#009933">charges for sequencing</FONT> for PCR 1,7b and 9
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> name
</TH><TH> 1-1
</TH><TH> 1-2
</TH><TH> 7b-15
</TH><TH> 7b-16
</TH><TH> 9-20
</TH><TH> 9-21
</TH><TH> 10-22
</TH><TH> 10-23
</TH></TR><TR><TD> DNA
</TD><TD> 1[1:10]; 1.74µl
</TD><TD> 1[1:10]; 1.74µl
</TD><TD> 7b; 3µl
</TD><TD> 7b; 3µl
</TD><TD> 9; 1.78µl
</TD><TD> 9; 1.78µl
</TD><TD> 10; 3.8µl
</TD><TD> 10; 3.8µl
</TD></TR><TR><TD> primer [1pmol/µl]
</TD><TD> 1; 3.2µl
</TD><TD> 2; 3.2µl
</TD><TD> 15; 3.2µl
</TD><TD> 16; 3.2µl
</TD><TD> 20; 3.2µl
</TD><TD> 21; 3.2µl
</TD><TD> 22; 3.2µl
</TD><TD> 23; 3.2µl
</TD></TR><TR><TD> Tris (10mM); pH 8.2
</TD><TD> 2.06µl
</TD><TD> 2.06µl
</TD><TD> 0.8µl
</TD><TD> 0.8µl
</TD><TD> 2.02µl
</TD><TD> 2.02µl
</TD><TD> 0µl
</TD><TD> 0µl
</TD></TR></TBODY></TABLE><P>every charge 7µl
</P><H2><SPAN class="mw-headline" id="9-07-2010"> 9-07-2010 </SPAN></H2><P><FONT color="#009933">Agarose gel electrophorese of PCR gel extractions</FONT></P><P>150V, 25min
</P><P>-&gt; Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>-result:
</P><P>from left to right: PCR 1,2a,2b,3,4a,/,7b,9,10,/,ladder,pathway
</P><TABLE><TBODY><TR><TH>PCR nr.
</TH><TH>1
</TH><TH><FONT color="#CC33CC">2a</FONT></TH><TH><FONT color="#CC33CC">2b</FONT></TH><TH><FONT color="#CC33CC">3</FONT></TH><TH><FONT color="#CC33CC">4a</FONT></TH><TH><FONT color="#CC33CC">7b</FONT></TH><TH>9
</TH><TH><FONT color="#CC33CC">10</FONT></TH></TR><TR><TD>expected band (bp)
</TD><TD>492
</TD><TD>332
</TD><TD>772
</TD><TD>1087
</TD><TD>330
</TD><TD>402
</TD><TD>808
</TD><TD>1888
</TD></TR><TR><TD>shown band(s)
</TD><TD>550,200
</TD><TD><FONT color="#CC33CC"><B>300</B></FONT></TD><TD><FONT color="#CC33CC"><B>750</B></FONT></TD><TD><FONT color="#CC33CC"><B>1100</B></FONT></TD><TD><FONT color="#CC33CC"><B>300</B></FONT></TD><TD><FONT color="#CC33CC"><B>450</B></FONT></TD><TD>900,1500
</TD><TD><FONT color="#CC33CC"><B>1900</B></FONT></TD></TR><TR><TD>clean charge
</TD><TD><FONT color="#CC33CC"><B>x</B></FONT></TD><TD><FONT color="#CC33CC"><B>x</B></FONT></TD><TD><FONT color="#CC33CC"><B>~x</B></FONT></TD><TD><FONT color="#CC33CC"><B>x</B></FONT></TD><TD><FONT color="#CC33CC"><B>x</B></FONT></TD><TD><FONT color="#CC33CC"><B>x</B></FONT></TD></TR></TBODY></TABLE><P><FONT color="#009933">new PCR for PCR6</FONT></P><P>PCR mixture for PCR6
</P><DL><DD><TABLE><TBODY><TR><TD>Concentration
</TD><TD>Low
</TD><TD>High
</TD></TR><TR><TD>template
</TD><TD>1 µl (1:100)
</TD><TD>15µl (1:100)
</TD></TR><TR><TD>primer (11,12)
</TD><TD>2.5 µl*2
</TD></TR><TR><TD>dNTPs
</TD><TD>1 µl
</TD></TR><TR><TD>Pfu
</TD><TD>0.5 µl
</TD></TR><TR><TD>10xbuffer
</TD><TD>5 µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>36.25 µl
</TD><TD>22.25 µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE></DD></DL><P>-&gt; Protocol: Thermal cycler program: <A href="/Team:LMU-Munich/Notebook/Protocols/17_Special_thermal_cycler_programs/Touch_down" title="Team:LMU-Munich/Notebook/Protocols/17 Special thermal cycler programs/Touch down">Touch down</A>, 62°C-52°C, 30 cycles by 55°C
</P><H2><SPAN class="mw-headline" id="9-08-2010"> 9-08-2010 </SPAN></H2><P><FONT color="#009933">Restriction digestion of eGFP, PCR6, ccdBamp</FONT></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel photo of (left to right) PCR6(5ng/100ng) template</DIV></DIV></DIV><P>as prepatation for ligation
</P><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digestion</A></P><TABLE><TBODY><TR><TH>eGFP
</TH><TH>SV40PA=PCR6
</TH><TH>ccdBamp
</TH></TR><TR><TD>DNA
</TD><TD>5µl=300ng
</TD><TD>DNA
</TD><TD>1µl=37,5ng
</TD><TD>DNA
</TD><TD>2µl=115ng
</TD></TR><TR><TD>Buffer MC
</TD><TD>2µl
</TD><TD>Buffer D
</TD><TD>2µl
</TD><TD>Buffer H
</TD><TD>2µl
</TD></TR><TR><TD>BSA 1:10
</TD><TD>2µl
</TD><TD>BSA 1:10
</TD><TD>2µl
</TD><TD>BSA 1:10
</TD><TD>2µl
</TD></TR><TR><TD>EcoRI
</TD><TD>0,5µl
</TD><TD>XbaI
</TD><TD>0,5µl
</TD><TD>EcoRI
</TD><TD>0,5µl
</TD></TR><TR><TD>SpeI
</TD><TD>0,5µl
</TD><TD>PstI
</TD><TD>0,5µl
</TD><TD>PstI
</TD><TD>0,5µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>13µl
</TD><TD>H<SUB>2</SUB>O
</TD><TD>14µl
</TD><TD>H<SUB>2</SUB>O
</TD><TD>13µl
</TD></TR><TR><TD>sum
</TD><TD>23µl
</TD><TD>sum
</TD><TD>20µl
</TD><TD>sum
</TD><TD>20µl
</TD></TR></TBODY></TABLE><P>1:30h 37°C, 20min 80°C
</P><P><FONT color="#009933">Agarose gel electrophoresis of PCR6</FONT></P><P>-&gt; Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>25min, 150V
</P><P><FONT color="#009933">Ligation 1</FONT></P><TABLE><TBODY><TR><TD>eGFP
</TD><TD>14,4µL
</TD><TD>(144ng)
</TD></TR><TR><TD>SV40PA
</TD><TD>4,8µL
</TD><TD>(48ng)
</TD></TR><TR><TD>ccdBamp
</TD><TD>2,9µL
</TD><TD>(100ng)
</TD></TR><TR><TD>T4 Buffer 10x
</TD><TD>3µL
</TD></TR><TR><TD>T4 Ligase
</TD><TD>0,5µL
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>4,4µL
</TD></TR><TR><TD>sum
</TD><TD>30µL
</TD></TR></TBODY></TABLE><P>
Ligation at 22,5°C for 30 min, denaturation at 65°C for 10 min.
</P><P>Concentration of DNA in Ligation 1:
</P><TABLE><TBODY><TR><TD>540ng
</TD><TD>A260/A280=2,4
</TD></TR></TBODY></TABLE><P><FONT color="#009933">Transformation</FONT></P><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/18_Competent_cells2" title="Team:LMU-Munich/Notebook/Protocols/18 Competent cells2">18 competent cells2</A></P><P>The incubation time for the cells is here 1 hour.
</P><H2><SPAN class="mw-headline" id="9-09-2010"> 9-09-2010 </SPAN></H2><P><FONT color="#009933">Agarose gel electrophoresis of PCR 1, 3, 4a, 5, 7a, 9</FONT>
Agarose gel electrophoresis of PCR 1, 3, 4a, 5, 7a, 9
</P><P>-&gt; protocol <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>results: &quot;primer-clouds&quot;, PCR 9: no band
</P><P><FONT color="#009933">Plasmid extraction of ccdBcam, ccdBamp, Bak, CMV, PhiC31o</FONT></P><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><TABLE><TBODY><TR><TD>Plasmid
</TD><TD>concentration [ng/µL]
</TD><TD>A260/A280
</TD></TR><TR><TD>ccdBcam
</TD><TD>22.5
</TD><TD>1.0
</TD></TR><TR><TD>ccdBamp
</TD><TD>42.5
</TD><TD>1.2
</TD></TR><TR><TD>Bak
</TD><TD>17.5
</TD><TD>0.8
</TD></TR><TR><TD>CMV
</TD><TD>10.0
</TD><TD>0.7
</TD></TR><TR><TD>PhiC310
</TD><TD>60
</TD><TD>1.4
</TD></TR></TBODY></TABLE><P>Eluated with H<SUB>2</SUB>O instead of the Eluation Buffer.
</P><P><FONT color="#009933">New PCR 1, 3, 4a, 5, 7a, 9, 10 with phusion</FONT></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> PCRnr.
</TH><TH> 1
</TH><TH> 3
</TH><TH> 4a
</TH><TH> 4b
</TH><TH> 5
</TH><TH> 7a
</TH><TH> 9
</TH><TH> 10
</TH></TR><TR><TD> template
</TD><TD> pDS7 (1:100); 4µl
</TD><TD> 2a; 0.9µl+2b; 0.6µl
</TD><TD> Bak; 0.5µl
</TD><TD> Bak; 0.5µl
</TD><TD> 4a; 0.8µl+4b; 1µl
</TD><TD> 190-6 (1:100); 1µl
</TD><TD> eGFP (1:25); 2µl
</TD><TD> PhiC31o (1:10); 1µl
</TD></TR><TR><TD> primer [10pmol/µl]
</TD><TD> 1,2; 2.5µl
</TD><TD> 3,6; 2.5µl
</TD><TD> 7,8; 2.5µl
</TD><TD> 9,10; 2.5µl
</TD><TD> 7,10; 2.5µl
</TD><TD> 13,14; 2.5µl
</TD><TD> 20,21; 2.5µl
</TD><TD> 22,23; 2.5µl
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> 28µl
</TD><TD> 30.5µl
</TD><TD> 31.5µl
</TD><TD> 31.5µl
</TD><TD> 30.2µl
</TD><TD> 31µl
</TD><TD> 30µl
</TD><TD> 31µl
</TD></TR><TR><TD>Annealing temperature
</TD><TD>51.5°C
</TD><TD>53.1°C
</TD><TD>53.1°C
</TD><TD>51.5°C
</TD><TD>53.1°C
</TD><TD>57.5°C
</TD><TD>61°C
</TD><TD>57.5°C
</TD></TR></TBODY></TABLE><P>+ in each assay:
</P><P>dNTP mix: 1µl, 5x Phusion buffer: 10µl, DMSO:1.5µl, Phusion:0.5µl
</P><P>sum: 50µl
</P><P>Program: standard Phusion PCR, 29 cycles; annealing temperatures: see above
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/15_PCR_with_Phusion" title="Team:LMU-Munich/Notebook/Protocols/15 PCR with Phusion"> 15 PCR with Phusion</A></P><P>[from left to right: PCR 1, 3, 4a, 5, 7a, 9, 10, Ladder]
</P><P><FONT color="#CC33CC"><B>PCR3, 9, 10 with right bands.</B></FONT></P><P><FONT color="#009933">New PCR 1, 4a, 4b, 5, 7a with phusion</FONT></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> PCRnr.
</TH><TH> 1
</TH><TH> 4a
</TH><TH> 4b
</TH><TH> 5
</TH><TH> 7a
</TH></TR><TR><TD> template
</TD><TD> pDS7 (1:10); 1µl
</TD><TD> Bak 1 ; 1µl
</TD><TD> Bak 1; 1µl
</TD><TD> 4a, 4b; 2 x 1.5µl
</TD><TD> pCT 190-6 (1:40); 1µl
</TD></TR><TR><TD> primer
</TD><TD> 1, 2; 2 x 2.5µl
</TD><TD> 7, 8; 2 x 2.5µl
</TD><TD> 9, 10; 2 x 2.5µl
</TD><TD> 7, 10; 2 x 2.5µl
</TD><TD> 13, 14; 2x 2.5µl
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> 31µl
</TD><TD> 31µl
</TD><TD> 31µl
</TD><TD> 30µl
</TD><TD> 30µl
</TD></TR><TR><TD> Annealing temperature
</TD><TD> 52°C
</TD><TD> 52°C
</TD><TD> 48°C
</TD><TD> 50°C
</TD><TD> 55°C
</TD></TR></TBODY></TABLE><P>+ each assay with dNTP-mix (1µl), 5xBuffer (10µl, DMSO (1.5µl), Phusion (0.5µl)
-&gt; sum: 50µl
</P><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/17_Special_thermal_cycler_programs/Touch_down_phusion" title="Team:LMU-Munich/Notebook/Protocols/17 Special thermal cycler programs/Touch down phusion">Touch down 59 with phusion</A>, 30 cycles with gradient appropriate for the annealing temperatures above.
</P><H2><SPAN class="mw-headline" id="9-10-2010"> 9-10-2010 </SPAN></H2><P><FONT color="#009933">Agarose gel electrophoresis of PCR 1, 4a, 4b, 5, 7a</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>[From left to right: Ladder, 1, 4a, 4b, 5, 7a]
</P><P>Only<FONT color="#CC33CC"><B>4a has been amplified successfully.</B></FONT></P><P><FONT color="#009933">Agarose gel extraction of PCR 3, 4a, 5, 9, 10</FONT></P><P>from left to right:<FONT color="#CC33CC"><B>PCR 3, band ~700bp, PCR 4a, band ~300bp, PCR 5, bands ~550bp (5*), ~650bp (5), PCR 9, band ~800bp, PCR 10, band ~1900bp</B></FONT></P><P>results:
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> PCR 3
</TH><TH> PCR 4a
</TH><TH> PCR 5
</TH><TH> PCR 5*
</TH><TH> PCR 9
</TH><TH> PCR 10
</TH></TR><TR><TD> concentration [ng/µl]
</TD><TD> 20
</TD><TD> 10
</TD><TD> 5
</TD><TD> 30
</TD><TD> 35
</TD><TD> 25
</TD></TR><TR><TD> A260/A280
</TD><TD> 1.6
</TD><TD> 1.333
</TD><TD> 2.0
</TD><TD> 2.0
</TD><TD> 1.750
</TD><TD> 2.0
</TD></TR></TBODY></TABLE><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/12_Gel_extraction_or_PCR_Clean_up" title="Team:LMU-Munich/Notebook/Protocols/12 Gel extraction or PCR Clean up"> 12 Gel extraction or PCR Clean up (Promega kit)</A></P><H2><SPAN class="mw-headline" id="9-11-2010"> 9-11-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="9-12-2010"> 9-12-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="9-13-2010"> 9-13-2010 </SPAN></H2><P>charges for sequencing (retry of 6.9.)
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> name
</TH><TH> 4a-7
</TH><TH> 4a-8
</TH><TH> 4b-9
</TH><TH> 4b-10
</TH><TH> 3-3
</TH><TH> 3-6
</TH><TH> 6-11
</TH><TH> 6-12
</TH></TR><TR><TD> DNA
</TD><TD> 4a; 2.4µl
</TD><TD> 4a; 2.4µl
</TD><TD> 4b; 0.5µl
</TD><TD> 4b; 0.5µl
</TD><TD> 3; 2µl
</TD><TD> 3; 2µl
</TD><TD> 6; 0.5µl
</TD><TD> 6; 0.5µl
</TD></TR><TR><TD> primer [1pmol/µl]
</TD><TD> 7; 3.2µl
</TD><TD> 8; 3.2µl
</TD><TD> 9; 3.2µl
</TD><TD> 10; 3.2µl
</TD><TD> 3; 3.2µl
</TD><TD> 6; 3.2µl
</TD><TD> 11; 3.2µl
</TD><TD> 12; 3.2µl
</TD></TR><TR><TD> Tris (10mM); pH 8.2
</TD><TD> 1.4µl
</TD><TD> 1.4µl
</TD><TD> 3.3µl
</TD><TD> 3.3µl
</TD><TD> 1.8µl
</TD><TD> 1.8µl
</TD><TD> 3.3µl
</TD><TD> 3.3µl
</TD></TR></TBODY></TABLE><P>every charge 7µl
</P><P><FONT color="#009933">New PCR7a with Taq</FONT></P><P>template: p190-6
</P><P>Primer 13,14
</P><P>Mixture
</P><TABLE><TBODY><TR><TD>template (~4ng)
</TD><TD>2µl (1:100)
</TD></TR><TR><TD>MasterMix for Taq
</TD><TD>10µl
</TD></TR><TR><TD>Primer *2
</TD><TD>1.5µl *2
</TD></TR><TR><TD>DMSO
</TD><TD>0,5µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>4.5µl
</TD></TR><TR><TD>sum
</TD><TD>20µl
</TD></TR></TBODY></TABLE><P>PCR program: touchdown PCR with Taq
</P><DL><DD><TABLE><TBODY><TR><TD>1: 94°C 2'
</TD></TR><TR><TD>2: 94°C 30&quot;
</TD></TR><TR><TD>3: 64°C/62°C/60°C/58°C/56°C/54° 30&quot;
</TD></TR><TR><TD>4: 72°C 2'
</TD></TR><TR><TD>5: (for each temperature)repeat 2-4 2x
</TD></TR><TR><TD>6: 94°C 30&quot;
</TD></TR><TR><TD>7: 58°C 30&quot;
</TD></TR><TR><TD>8: 72°C 2'
</TD></TR><TR><TD>9: repeat 6-8 29x
</TD></TR><TR><TD>10: 72°C 10'
</TD></TR><TR><TD>11: 15°C break
</TD></TR></TBODY></TABLE></DD></DL><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/21_PCR_with_Taq_Mastermix" title="Team:LMU-Munich/Notebook/Protocols/21 PCR with Taq Mastermix">21 PCR with Taq Mastermix</A></P><P><FONT color="#009933">overnight culture inoculated of</FONT></P><DL><DD>CMV (amp)
</DD><DD>ccdB (amp)
</DD><DD>ccdB (cam)
</DD><DD>pC31o (amp)
</DD><DD>Bak (amp)
</DD></DL><P><FONT color="#009933">New PCR7a with Phusion Hot Start</FONT></P><P>with 2ng and 4 ng template
</P><DL><DD><TABLE><TBODY><TR><TD>template
</TD><TD>2ng
</TD><TD>4ng
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>54µl
</TD><TD>53µl
</TD></TR><TR><TD>Buffer 5x
</TD><TD>20µl
</TD><TD>20µl
</TD></TR><TR><TD>Primer (13,14)
</TD><TD>2*10µl
</TD><TD>2*10µl
</TD></TR><TR><TD>DMSO
</TD><TD>3µl
</TD><TD>3µl
</TD></TR><TR><TD>Hot Start
</TD><TD>1µl
</TD><TD>1µl
</TD></TR></TBODY></TABLE></DD></DL><P>total: 100µl each, divided into 5 charges
</P><P>program: 
</P><DL><DD>98°C 30sec
</DD><DD>----
</DD><DD>98°C 10sec
</DD><DD>gradient: 50°C, 53°C, 56,6°C, 60,2°C, 64,5°C  30sec   30 cycles</DD><DD>72 15sec
</DD><DD>----
</DD><DD>72°C 5min
</DD><DD>12°C forever
</DD></DL><P>-&gt; Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/20_PCR_with_Phusion_Hot_Start" title="Team:LMU-Munich/Notebook/Protocols/20 PCR with Phusion Hot Start">20 PCR with Phusion Hot Start</A></P><H2><SPAN class="mw-headline" id="9-14-2010"> 9-14-2010 </SPAN></H2><P><FONT color="#009933">Agarose gelelectrophoresis of PCR 7a</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>bands 1 to 5: 2ng of template DNA
</P><P>bands 6 to 10: 4ng of template DNA
</P><P>-&gt; no bands
</P><P><FONT color="#009933">new PCR 7a and PCR 8 (without mutation)</FONT>
with mastermix, without DMSO
</P><TABLE><TBODY><TR><TD>number:
</TD><TD>7a diluted
</TD><TD>7a undiluted
</TD><TD>8
</TD></TR><TR><TD>mastermix
</TD><TD>50µl
</TD><TD>50µl
</TD><TD>10µl
</TD></TR><TR><TD>primer
</TD><TD>2*10µl
</TD><TD>2*10µl
</TD><TD>2*2µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>29µl
</TD><TD>29µl
</TD><TD>5µl
</TD></TR><TR><TD>template
</TD><TD>1µl 190-6 (1:100)
</TD><TD>1µl 190-6
</TD><TD>1µl 190-6 (1:100)
</TD></TR></TBODY></TABLE><P>7a diluated and undiluated: divided into 3 charges
</P><P>7a diluated: 1,2,3; 7a undiluated: 4,5,6
</P><TABLE><TBODY><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>8
</TD></TR><TR><TD>48°C
</TD><TD>52°C
</TD><TD>56,1°C
</TD><TD>48°C
</TD><TD>52°C
</TD><TD>56,1°C
</TD><TD>52°C
</TD></TR></TBODY></TABLE><P>program:
</P><DL><DD><TABLE><TBODY><TR><TD>94°C 2'
</TD></TR><TR><TD>94°C 30&quot;
</TD></TR><TR><TD>48°C/52°C/56,1°C 30&quot;
</TD></TR><TR><TD>72°C 1,5'
</TD></TR><TR><TD>(for each temperature)repeat 30x
</TD></TR><TR><TD>72°C 5'
</TD></TR></TBODY></TABLE></DD></DL><P><FONT color="#009933">Plasmid extraction of ccdB (amp) and ccdB (cam)</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><P>results:
</P><DL><DD>ccdB(amp): 105ng/µl A260/280: 2,00
</DD><DD>cddB(cam): 102ng/µl A260/280: 1,952
</DD></DL><H2><SPAN class="mw-headline" id="9-15-2010"> 9-15-2010 </SPAN></H2><P><FONT color="#009933">Agarose gelelectrophoresis of PCR 7a diluted &amp;7a undiluted &amp; &quot;8&quot;(without mutation)</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>from left to right: 7a: 1:100 diluted: 48°C, 52°C, 56.1°C, undiluted: 48°C, 52°C, 56.1°C, 8 without Mutation
</P><P><FONT color="#CC33CC"><B>weak right band (with fuzz)for 7a </B></FONT>undiluated with best result for 56°C
</P><P>-&gt;<FONT color="#009933">new PCR 7a and &quot;8&quot;</FONT></P><P>with mastermix Taq
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/21_PCR_with_Taq_Mastermix" title="Team:LMU-Munich/Notebook/Protocols/21 PCR with Taq Mastermix">21 PCR with Taq Mastermix</A></P><P>template: 190-6, 1:10 and 1:5
</P><P>temp: 56°C and 58°C
</P><TABLE><TBODY><TR><TD>number
</TD><TD>7a
</TD><TD>7a
</TD><TD>&quot;8&quot;
</TD><TD>&quot;8&quot;
</TD></TR><TR><TD>Mastermix
</TD><TD>20µl
</TD><TD>20µl
</TD><TD>20µl
</TD><TD>20µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>15µl
</TD><TD>15µl
</TD><TD>15µl
</TD><TD>15µl
</TD></TR><TR><TD>Primer
</TD><TD>(13,14) 2*2µl
</TD><TD>(13,14) 2*2µl
</TD><TD>(13,16) 2*2µl
</TD><TD>(13,16) 2*2µl
</TD></TR><TR><TD>Template
</TD><TD>190-6 (1:5) 1µl
</TD><TD>190-6 (1:10) 1µl
</TD><TD>190-6 (1:5) 1µl
</TD><TD>190-6 (1:10) 1µl
</TD></TR></TBODY></TABLE><P>2 charges each: one for 56°C and one for 58°C annealing temperature
</P><P>
program:
</P><DL><DD><TABLE><TBODY><TR><TD>94°C 2'
</TD></TR><TR><TD>94°C 30&quot;
</TD></TR><TR><TD>56°C/58°C 30&quot;
</TD></TR><TR><TD>72°C 1,5'
</TD></TR><TR><TD>(for each temperature)repeat 30x
</TD></TR><TR><TD>72°C 5'
</TD></TR></TBODY></TABLE></DD></DL><P><FONT color="#009933">Restriction Digestion of PCR 1 (17.8.), PCR 3 (10.9.), PCR 5<SUB>1</SUB> (10.9.), PCR 5<SUB>2</SUB> (10.9.), PCR 6 (17.8.), PCR 9 (10.9.), PCR 10 (30.8.) for ligation with vector</FONT></P><P>EcoR1 + Pst1 with Buffer H; 50ng DNA
</P><TABLE><TBODY><TR><TD>1
</TD><TD>3
</TD><TD>5<SUB>1</SUB></TD><TD>5<SUB>2</SUB></TD><TD>6
</TD><TD>9
</TD><TD>10
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>13µl
</TD><TD>12,5µl
</TD><TD>5µl
</TD><TD>13µl
</TD><TD>13,5µl
</TD><TD>13,5µl
</TD><TD>12,5µl
</TD></TR><TR><TD>Buffer H
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD></TR><TR><TD>BSA (1:10)
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD></TR><TR><TD>DNA
</TD><TD>2µl
</TD><TD>2,5µl
</TD><TD>10µl
</TD><TD>2µl
</TD><TD>1,5µl
</TD><TD>1,5µl
</TD><TD>2,5µl
</TD></TR><TR><TD>EcoR1
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD></TR><TR><TD>Pst1
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD></TR></TBODY></TABLE><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digestion</A></P><P><FONT color="#009933">Agarose gelelectrophoresis of PCR 7a 1-4 &amp; &quot;8&quot; 1-4(without mutation)</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>-&gt; bad results
</P><P><FONT color="#009933">purification of restriction digestion</FONT></P><TABLE><TBODY><TR><TD>concentration (ng/µl)
</TD><TD>A<SUB>260</SUB>/A<SUB>280</SUB></TD><TD>(supposed) lenght
</TD></TR><TR><TD>1
</TD><TD>20
</TD><TD>1,000
</TD><TD>492
</TD></TR><TR><TD>3
</TD><TD>5
</TD><TD>1,000
</TD><TD>1087
</TD></TR><TR><TD>5<SUB>1</SUB></TD><TD>22,5
</TD><TD>1,8
</TD><TD>688
</TD></TR><TR><TD>5<SUB>2</SUB></TD><TD>7,5
</TD><TD>1,5
</TD><TD>688
</TD></TR><TR><TD>6
</TD><TD>10
</TD><TD>1,333
</TD><TD>237
</TD></TR><TR><TD>9
</TD><TD>20
</TD><TD>1,6
</TD><TD>808
</TD></TR><TR><TD>10
</TD><TD>17,5
</TD><TD>1,75
</TD><TD>1888
</TD></TR></TBODY></TABLE><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/12_Gel_extraction_or_PCR_Clean_up" title="Team:LMU-Munich/Notebook/Protocols/12 Gel extraction or PCR Clean up"> 12 Gel extraction or PCR Clean up (Promega kit)</A></P><P><FONT color="#009933">Ligation with pSB1C3 (2072bp)</FONT></P><TABLE><TBODY><TR><TD>for
</TD><TD>#
</TD><TD>backbone
</TD><TD>insert (µl)
</TD><TD>charge (µl)
</TD><TD>Buffer 10x (µl)
</TD><TD>H<SUB>2</SUB>O (µl)
</TD></TR><TR><TD>100ng
</TD><TD>1
</TD><TD>4
</TD><TD>7,12
</TD><TD>20
</TD><TD>2
</TD><TD>6,38
</TD></TR><TR><TD>50ng
</TD><TD>3
</TD><TD>2
</TD><TD>31,48
</TD><TD>40
</TD><TD>4
</TD><TD>2,02
</TD></TR><TR><TD>100ng
</TD><TD>5<SUB>1</SUB></TD><TD>4
</TD><TD>8,85
</TD><TD>20
</TD><TD>2
</TD><TD>4,65
</TD></TR><TR><TD>100ng
</TD><TD>5<SUB>2</SUB></TD><TD>4
</TD><TD>26,56
</TD><TD>40
</TD><TD>4
</TD><TD>4,94
</TD></TR><TR><TD>100ng
</TD><TD>6
</TD><TD>4
</TD><TD>6,86
</TD><TD>20
</TD><TD>2
</TD><TD>13,36
</TD></TR><TR><TD>100ng
</TD><TD>9
</TD><TD>4
</TD><TD>11,7
</TD><TD>20
</TD><TD>2
</TD><TD>1,8
</TD></TR><TR><TD>100ng
</TD><TD>10
</TD><TD>4
</TD><TD>31,24
</TD><TD>40
</TD><TD>4
</TD><TD>4,26
</TD></TR></TBODY></TABLE><P>plus 1 µl T4 ligase in each charge
</P><P>-&gt; Protocol:  <A href="/Team:LMU-Munich/Notebook/Protocols/22_Ligation" title="Team:LMU-Munich/Notebook/Protocols/22 Ligation">22 Ligation</A></P><P><FONT color="#009933">new PCR 7a &amp; &quot;8&quot;</FONT></P><P>with mastermix Taq 56°C
</P><TABLE><TBODY><TR><TD>7a-1
</TD><TD>7a-2
</TD><TD>&quot;8&quot;-1
</TD><TD>&quot;8&quot;-2
</TD></TR><TR><TD>mastermix (µl)
</TD><TD>10
</TD><TD>10
</TD><TD>10
</TD><TD>10
</TD></TR><TR><TD>template
</TD><TD>190-6 1µl
</TD><TD>190-6 2µl
</TD><TD>190-6(1:10) 1µl
</TD><TD>190-6(1:10) 2µl
</TD></TR><TR><TD>primer
</TD><TD>13&amp;14 2*1µl
</TD><TD>13&amp;14 2*1µl
</TD><TD>13&amp;16 2*1µl
</TD><TD>13&amp;16 2*1µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>7µl
</TD><TD>6µl
</TD><TD>7µl
</TD><TD>6µl
</TD></TR></TBODY></TABLE><P>sum: 20µl each
</P><P>pcr-program:
</P><DL><DD><TABLE><TBODY><TR><TD>94°C 2'
</TD></TR><TR><TD>94°C 30&quot;
</TD></TR><TR><TD>56°C 30&quot;
</TD></TR><TR><TD>72°C 1,5'
</TD></TR><TR><TD>(for each temperature)repeat 30x
</TD></TR><TR><TD>72°C 5'
</TD></TR><TR><TD>12°C forever
</TD></TR></TBODY></TABLE></DD></DL><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/21_PCR_with_Taq_Mastermix" title="Team:LMU-Munich/Notebook/Protocols/21 PCR with Taq Mastermix">21 PCR with Taq Mastermix</A></P><H2><SPAN class="mw-headline" id="9-16-2010"> 9-16-2010 </SPAN></H2><P><FONT color="#009933">Agarose gelelectrophoresis of PCR 7a-1, PCR 7a-2, PCR 8-1, 8-2 and of all three PCR 6 we ever made (in order to control whether we have extracted the right fragment (237) or just the primerdimers)</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>From left to right: 6<SUB>2</SUB>, 6<SUB>1</SUB>, 6, Ladder, 7a-1, 7a-2, 8-2, 8-1
</P><DL><DD>results:
</DD></DL><DL><DD>6-1 and 6-2: ?
</DD></DL><DL><DD><FONT color="#CC33CC"><B>6: ok</B></FONT></DD><DD>7a-1,7a-2,&quot;8&quot;-1,&quot;8&quot;-2: primerdimer-problem -&gt; we ordered new primers!
</DD></DL><P><FONT color="#009933">transformation of the ligations 1,3,5<SUB>1</SUB>, 5<SUB>2</SUB>, 6, 9, 10</FONT></P><P>-&gt; Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/3_Transformation" title="Team:LMU-Munich/Notebook/Protocols/3 Transformation">3 Transformation</A></P><P>-&gt;plated on plates with chloramphenicol
</P><P><FONT color="#009933">again PCR 1,3,5,6,9,10 in order to increase the profit (Ausbeute) for ligation</FONT></P><TABLE><TBODY><TR><TD>1
</TD><TD>3
</TD><TD>5
</TD><TD>6
</TD><TD>9
</TD><TD>10
</TD></TR><TR><TD>template
</TD><TD>prev TRE(1:30) 1µl
</TD><TD>2a(1:5) 1µl + 2b 0,5µl
</TD><TD>4a 0,5µl + 4b 0,5µl
</TD><TD>SV40PA (pcDNA3)(1:50) 1µl
</TD><TD>eGFP(1:5) 1µl
</TD><TD>PhiC310(1:6) 1µl
</TD></TR><TR><TD>primer
</TD><TD>1&amp;2 2*2,5µl
</TD><TD>3&amp;6 2*2,5µl
</TD><TD>7&amp;10 2*2,5µl
</TD><TD>11&amp;12 2*2,5µl
</TD><TD>20&amp;21 2*2,5µl
</TD><TD>22&amp;23 2*2,5µl
</TD></TR><TR><TD>T<SUB>m</SUB></TD><TD>54°C
</TD><TD>50°C
</TD><TD>49°C
</TD><TD>56°C
</TD><TD>58°C
</TD><TD>58°C
</TD></TR><TR><TD>mastermix (µl)
</TD><TD>25
</TD><TD>25
</TD><TD>25
</TD><TD>25
</TD><TD>25
</TD><TD>25
</TD></TR><TR><TD>DMSO
</TD><TD>1,25µl
</TD><TD>1,25µl
</TD><TD>1,25µl
</TD><TD>1,25µl
</TD><TD>1,25µl
</TD><TD>1,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O (µl)
</TD><TD>17,75
</TD><TD>17,25
</TD><TD>17,75
</TD><TD>17,75
</TD><TD>17,75
</TD><TD>17,75
</TD></TR></TBODY></TABLE><P>pcr-program:
</P><DL><DD><TABLE><TBODY><TR><TD>94°C 2'
</TD></TR><TR><TD>94°C 30&quot;
</TD></TR><TR><TD>T<SUB>m</SUB> (see above) 30&quot;
</TD></TR><TR><TD>72°C 1,5'
</TD></TR><TR><TD>(for each temperature)repeat 30x
</TD></TR><TR><TD>72°C 5'
</TD></TR></TBODY></TABLE></DD></DL><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/21_PCR_with_Taq_Mastermix" title="Team:LMU-Munich/Notebook/Protocols/21 PCR with Taq Mastermix">21 PCR with Taq Mastermix</A></P><H2><SPAN class="mw-headline" id="9-17-2010"> 9-17-2010 </SPAN></H2><P><FONT color="#009933">Analysis of the transformation from yesterday</FONT></P><P>Colonies on plates when 100µl or pellet plated:
</P><TABLE><TBODY><TR><TD>1
</TD><TD>3
</TD><TD>5
</TD><TD>6
</TD><TD>9
</TD><TD>10
</TD></TR><TR><TD>pellet
</TD><TD>no
</TD><TD>yes
</TD><TD>yes
</TD><TD>yes
</TD><TD>yes
</TD><TD>yes
</TD><TD>yes
</TD></TR><TR><TD>100µl
</TD><TD>no
</TD><TD>yes
</TD><TD>yes
</TD><TD>yes
</TD><TD>yes
</TD><TD>yes
</TD><TD>yes
</TD></TR></TBODY></TABLE><P><FONT color="#009933">again: transformation of ligation 1</FONT></P><P>-&gt;Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/3_Transformation" title="Team:LMU-Munich/Notebook/Protocols/3 Transformation">3 Transformation</A></P><P><FONT color="#009933">Agarose gelelectrophoresis of yesterday's PCR 1,3,5,6,9,10(without mutation)</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>From left to right: PCR1, PCR3, PCR5, Ladder, PCR6, PCR9, PCR10
</P><P>-&gt;<FONT color="#CC33CC"><B>right band for PCR 3,5,6,9,10</B></FONT>; no band for PCR 1
</P><P>From left to right: Ladder, PCR3, PCR5, PCR9, PCR10
</P><P>Bands took: 3<SUB>2</SUB>: Band short over 1000bp, 3<SUB>1</SUB>: Band short under 1000bp (probably 3<SUB>2</SUB> right), 5 the highest band, 9 upper band, 10 upper band
</P><P><FONT color="#009933">new PCR 1,5,6</FONT></P><TABLE><TBODY><TR><TD>1
</TD><TD>5
</TD><TD>6
</TD></TR><TR><TD>template
</TD><TD>pTRE Rev 0,5µl
</TD><TD>4a 1,5µl + 4b 1,5µl
</TD><TD>pcDNA3 0,5µl
</TD></TR><TR><TD>primer
</TD><TD>1&amp;2 2*2,5µl
</TD><TD>7&amp;10 2*2,5µl
</TD><TD>11&amp;12 2*2,5µl
</TD></TR><TR><TD>buffer 10x
</TD><TD>5µl
</TD><TD>5µl
</TD><TD>5µl
</TD></TR><TR><TD>dNTPs
</TD><TD>1µl
</TD><TD>1µl
</TD><TD>1µl
</TD></TR><TR><TD>Pfu
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD><TD>0,5µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>26,75 µl
</TD><TD>24,25 µl
</TD><TD>26,75 µl
</TD></TR><TR><TD>T&lt;sub&lt;m&lt;/sub&gt;</TD><TD>51°C
</TD><TD>50°C
</TD><TD>55°C
</TD></TR></TBODY></TABLE><P>-&gt; Protocol:
pcr-program: standard Pfu <A href="/Team:LMU-Munich/Notebook/Protocols/10_PCR_with_Pfu" title="Team:LMU-Munich/Notebook/Protocols/10 PCR with Pfu"> 10 PCR with Pfu</A></P><P><FONT color="#009933">Restriction digestion of PCR products, ccdB Plasmids and Biobricks for the 3A Method</FONT></P><TABLE><TBODY><TR><TD>Name
</TD><TD>H<SUB>2</SUB>O
</TD><TD>Buffer (each 2µl)
</TD><TD>BSA (1:10)
</TD><TD>DNA Volume
</TD><TD>DNA Mass
</TD><TD>Enzyms (2*0.5µl)
</TD></TR><TR><TD>ccdBa
</TD><TD>25 µl
</TD><TD>H
</TD><TD>2µl
</TD><TD>20µl
</TD><TD>~600ng
</TD><TD>E+P
</TD></TR><TR><TD>ccdBa
</TD><TD>25µl
</TD><TD>H
</TD><TD>2µl
</TD><TD>20µl
</TD><TD>~700ng
</TD><TD>E+P
</TD></TR><TR><TD>1
</TD><TD>43µl
</TD><TD>MC
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>~500ng
</TD><TD>E+S
</TD></TR><TR><TD>3
</TD><TD>25µl
</TD><TD>B
</TD><TD>2µl
</TD><TD>20µl
</TD><TD>~200ng
</TD><TD>X+P
</TD></TR><TR><TD>Primer 18+19
</TD><TD>25µl
</TD><TD>B
</TD><TD>2µl
</TD><TD>10µl+10µl
</TD><TD>?
</TD><TD>X+P
</TD></TR><TR><TD>5<SUB>1</SUB></TD><TD>25µl
</TD><TD>MC
</TD><TD>2µl
</TD><TD>20µl
</TD><TD>~100ng
</TD><TD>E+S
</TD></TR><TR><TD>5<SUB>2</SUB></TD><TD>25µl
</TD><TD>MC
</TD><TD>2µl
</TD><TD>20µl
</TD><TD>~600ng
</TD><TD>E+S
</TD></TR><TR><TD>6
</TD><TD>35µl
</TD><TD>B
</TD><TD>2µl
</TD><TD>10µl
</TD><TD>~400ng
</TD><TD>X+P
</TD></TR><TR><TD>9
</TD><TD>25µl
</TD><TD>MC
</TD><TD>2µl
</TD><TD>20µl
</TD><TD>~700ng
</TD><TD>E+S
</TD></TR><TR><TD>10
</TD><TD>25µl
</TD><TD>MC
</TD><TD>2µl
</TD><TD>20µl
</TD><TD>~400ng
</TD><TD>E+S
</TD></TR><TR><TD>eGFP
</TD><TD>35µl
</TD><TD>MC
</TD><TD>2µl
</TD><TD>10µl
</TD><TD>~550ng
</TD><TD>E+S
</TD></TR><TR><TD>CMV1
</TD><TD>35µl
</TD><TD>H
</TD><TD>2µl
</TD><TD>10µl
</TD><TD>~530ng
</TD><TD>E+P
</TD></TR></TBODY></TABLE><P>-&gt; Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digestion</A></P><P><FONT color="#009933">Colony PCR of Ligations 3, 5<SUB>1</SUB>, 5<SUB>2</SUB>, 6, 9, 10 and PCR of CMV1 (to test if right)</FONT></P><P>10 1-4, 9 1-4, 6 1-4, 5<SUB>1</SUB> 1-4, 5<SUB>2</SUB> 1-4, 3 1-4 Colonies picked and put in following Mix:
</P><TABLE><TBODY><TR><TD>PCR Mastermix
</TD><TD>10µl
</TD></TR><TR><TD>Biobrick Primer F
</TD><TD>1.5µl
</TD></TR><TR><TD>Biobrick Primer R
</TD><TD>1.5µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>7µl
</TD></TR></TBODY></TABLE><P>CMV1:
</P><TABLE><TBODY><TR><TD>PCR Mastermix
</TD><TD>10µl
</TD></TR><TR><TD>Biobrick Primer F
</TD><TD>1.5µl
</TD></TR><TR><TD>Biobrick Primer R
</TD><TD>1.5µl
</TD></TR><TR><TD>Template
</TD><TD>2µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>5µl
</TD></TR></TBODY></TABLE><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/21_PCR_with_Taq_Mastermix" title="Team:LMU-Munich/Notebook/Protocols/21 PCR with Taq Mastermix">21 PCR with Taq Mastermix</A></P><P><FONT color="#009933">Agarose Gel electrophoresis of 3, 5<SUB>1</SUB>, 5<SUB>2</SUB>, 6, 9, 10 and CMV1 (to test if right)</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>Above: From left to right: 3.1, 3.2, 3.3, 3.4, 5<SUB>1</SUB>.1, 5<SUB>1</SUB>.2, 5<SUB>1</SUB>.3, 5<SUB>1</SUB>.4, Ladder, 5<SUB>2</SUB>.1, 5<SUB>2</SUB>.2, 5<SUB>2</SUB>.3, 5<SUB>2</SUB>.4
</P><P>Below: From left to right: 6.1, 6.2, 6.3, 6.4, 9.1, 9.2, 9.3, 9.4, Ladder, 10.1, 10.2, 10.3, 10.4, CMV1
</P><P>-&gt;<FONT color="#CC33CC"><B>right bands for 6.3 and 6.4, CMV1 ~1200bp (we think that this is right, as Biobrick sequenzing information indicates that it isn't 654bp but about 1200bp)</B></FONT></P><P><FONT color="#009933">PCR clean up of PCR Gel extraction 3<SUB>1</SUB>, 3<SUB>2</SUB>, 5, 9, 10, PCR Product 6 and digestion PCR1, Primer 18+19, R5<SUB>1</SUB>, R5<SUB>2</SUB>, PCR6, PCR9, PCR10, PCR3</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/12_Gel_extraction_or_PCR_Clean_up" title="Team:LMU-Munich/Notebook/Protocols/12 Gel extraction or PCR Clean up"> 12 Gel extraction or PCR Clean up (Promega kit)</A></P><H2><SPAN class="mw-headline" id="9-18-2010"> 9-18-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="9-19-2010"> 9-19-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="9-20-2010"> 9-20-2010 </SPAN></H2><P><FONT color="#009933">Dephosphorylation of linearized ccdB amp and ccdB cam</FONT></P><P>Add 1µl TSAP to digested vectors.
</P><P>Incutation: 37°C 15min; Inhibition: 74°C 15min
</P><P><FONT color="#009933">Ligation and 3A-Assemblies</FONT></P><P>Jump-or-Die Ligations: 1a, 1b (5<SUB>1</SUB>),1b  (5<SUB>2</SUB>), 2, 3
</P><P>Cut'N'Survive Ligations: 1a, 2b, 
</P><P>Biobricks for both Systems: CMV, PCR1 (tet-on-promoter)
</P><P>Mixtures:
</P><P>- 3A Assemblies (everything except CMV, PCR1):
</P><P>Inserts: each 8µl; ccdB vector: 1µl; T4 Ligase: 0.5µl; T4 Ligase Buffer: 2µl; H<SUB>2</SUB>O: 1µl 
</P><P>- Biobricks:
</P><P>Insert: 10µl; ccdB vector: 1µl; T4 Ligase: 0.5µl; T4 Ligase Buffer: 2µl; H<SUB>2</SUB>O: 7µl 
</P><P>Incubation: 2:30h 16°C; Inhibition: 10 min 65°C
</P><P>-&gt;Protocols: <A href="/Team:LMU-Munich/Notebook/Protocols/22_Ligation" title="Team:LMU-Munich/Notebook/Protocols/22 Ligation">22 Ligation</A>, <A href="/Team:LMU-Munich/Notebook/Protocols/13_3A_Method_for_Biobrick_assembly" title="Team:LMU-Munich/Notebook/Protocols/13 3A Method for Biobrick assembly"> 13 3A Method for Biobrick assembly</A></P><P><FONT color="#009933">Colony PCR of PCR product ligations with pSB1C3</FONT></P><P>Mixture:
</P><P>PCR Mastermix:130µl ; PrimerF:19,5µl ; PrimerR:19,5µl ; H<SUB>2</SUB>O:91µl
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/21_PCR_with_Taq_Mastermix" title="Team:LMU-Munich/Notebook/Protocols/21 PCR with Taq Mastermix">21 PCR with Taq Mastermix</A></P><P><FONT color="#009933">Agarose Gel Electrophoresis of Colony PCRs</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>above from left to right: pathway/ladder/3.5/3.6/3.7/3.8/5<SUB>1</SUB>.5/5<SUB>1</SUB>.6/5<SUB>1</SUB>.7/5<SUB>1</SUB>.8/5<SUB>2</SUB>.5/5<SUB>2</SUB>.6/5<SUB>2</SUB>.7/5<SUB>2</SUB>.8
</P><P>below from left to right: ladder/6.5/6.6/6.7/6.8/9.5/9.6/9.7/9.8/10.5/10.6/10.7/10.8
</P><H2><SPAN class="mw-headline" id="9-21-2010"> 9-21-2010 </SPAN></H2><P><FONT color="#009933">Agarose Gel Electrophoresis of PCRs 1, 5, 6, and Colony PCR 3.8, 5<SUB>1</SUB>.7,  5<SUB>2</SUB>.6, 9.7, 10.5, 10.6</FONT></P><P>-&gt; Bad results, probably too much cells in PCR mix
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P><FONT color="#009933">Colony PCR of 3.8, 5<SUB>1</SUB>.7,  5<SUB>2</SUB>.6, 9.7, 10.5, 10.6 again and of the 3A Ligations of yesterday as well as the Ligation of PCR1 and CMV with pSB1C3</FONT></P><P>PCR Master Mix: 60µl
</P><P>Primer F: 9µl
</P><P>Primer R: 9µl
</P><P>H<SUB>2</SUB>O: 42µl
</P><P>-&gt;22charges à 5µl
</P><P>-&gt; Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/21_PCR_with_Taq_Mastermix" title="Team:LMU-Munich/Notebook/Protocols/21 PCR with Taq Mastermix">21 PCR with Taq Mastermix</A></P><P><FONT color="#009933">PCR 7a with new Primers</FONT></P><TABLE><TBODY><TR><TD>template 190-6 (1:10)
</TD><TD>1µl
</TD></TR><TR><TD>Master Mix
</TD><TD>25µl
</TD></TR><TR><TD>Primer 13k (short)
</TD><TD>3,75µl
</TD></TR><TR><TD>Primer 14
</TD><TD>3,75µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>16,5µl
</TD></TR></TBODY></TABLE><P>sum: 50µl
</P><P>same with Primer 13l (long)
</P><P>-&gt;Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/21_PCR_with_Taq_Mastermix" title="Team:LMU-Munich/Notebook/Protocols/21 PCR with Taq Mastermix">21 PCR with Taq Mastermix</A></P><P><FONT color="#009933">Agarose Gel Electrophoresis of Colony PCRs and PCR 7a</FONT></P>
different ladder: <DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">fermentas dna ladder mix (source:<A href="http://www.fermentas.com/en/products/all/dna-electrophoresis/generuler-dna-ladders/sm0333-generuler-mix" class="external free" rel="nofollow">http://www.fermentas.com/en/products/all/dna-electrophoresis/generuler-dna-ladders/sm0333-generuler-mix</A></DIV></DIV></DIV><P>
key: JD=Jump-or-Die; CS=Cut'N'Survive; first number=Ligation number (see schedule); second number=colonie number (marked on the plate)
</P><P>from left to right:7ak(short primer), 7al(long primer), JD.1a.1, JD.1a.2,JD.1b5<SUB>1</SUB>.1, JD.1b5<SUB>2</SUB>.2, JD.1b5<SUB>2</SUB>.1, JD.1b5<SUB>2</SUB>.2, JD.2.1, JD.2.2, JD.3<SUB>1</SUB>.1, JD.3<SUB>1</SUB>.2, CS.1a.1, CS.1a.2
</P><P><FONT color="#CC33CC"><B>verified products:7a (~800bp); JD.2.1; JD.2.2(each ~1000bp);JD.3<SUB>1</SUB>.1(~2000bp); CS.1a.2(~1500bp)</B></FONT></P><P>no bands shown: CS.2b.1, CS.2b.2, PCR1.1, PCR1.CMV,3.8, 5<SUB>1</SUB>.7, 5<SUB>2</SUB>.6, 9.7, 10.5, 10.6
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><H2><SPAN class="mw-headline" id="9-22-2010"> 9-22-2010 </SPAN></H2><P><FONT color="#009933">Inoculate Colonies</FONT></P><P>inoculated in 4ml LB-medium with appropriate antibiotic
</P><P><FONT color="#009933">Plated residual transformated E. colis (9-20-10) (where we had few colonies on plates) </FONT></P><P><FONT color="#009933">Colony PCRs</FONT></P><P>for 22*5µl charges:
</P><TABLE><TBODY><TR><TD>PCR Mastermix
</TD><TD>55µl
</TD></TR><TR><TD>Primer F
</TD><TD>8,25µl
</TD></TR><TR><TD>Primer R
</TD><TD>8,25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>38,5µl
</TD></TR><TR><TD>sum
</TD><TD>110µl
</TD></TR></TBODY></TABLE><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/21_PCR_with_Taq_Mastermix" title="Team:LMU-Munich/Notebook/Protocols/21 PCR with Taq Mastermix">21 PCR with Taq Mastermix</A></P><P>Annealing temperature: 53°C
</P><P><FONT color="#009933">PCR Purification of 7a</FONT></P><P>charges 7ak and 7al had not been labeled, so they were renamed.
</P><P>results:
</P><P>7a<SUB>1</SUB>: 135ng/µl; A260/A280=1.7
</P><P>7a<SUB>2</SUB>: 133ng/µl; A260/A280=1.7
</P><P><FONT color="#009933">PCR8 with Phusion Hot Start II</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/20_PCR_with_Phusion_Hot_Start" title="Team:LMU-Munich/Notebook/Protocols/20 PCR with Phusion Hot Start">20 PCR with Phusion Hot Start</A></P><TABLE><TBODY><TR><TD>template 7a 1:100
</TD><TD>3µl ~4ng (850bp)
</TD></TR><TR><TD>template 7b 1:10
</TD><TD>2µl ~2ng (402bp)
</TD></TR><TR><TD>Buffer 5X
</TD><TD>10µl
</TD></TR><TR><TD>Phusion
</TD><TD>0.5µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>27µl
</TD></TR><TR><TD>Primer 13
</TD><TD>2.5µl
</TD></TR><TR><TD>Primer 16
</TD><TD>2.55µl
</TD></TR><TR><TD>DMSO)
</TD><TD>1.5µl
</TD></TR><TR><TD>dNTP mix
</TD><TD>1µl
</TD></TR></TBODY></TABLE><P>Program: annealing temperature: 63°C, elongation time: 40sec
</P><P><FONT color="#009933">Agarose gel electrophoresis of Colony PCRs and PCR8</FONT></P><P>new ladder: fermentas FastRuler (TM) Middle Range DNA Ladder <A href="http://www.fermentas.com/en/products/all/dna-electrophoresis/fastruler-dna-ladders/sm111-fastruler-middle" class="external free" rel="nofollow">http://www.fermentas.com/en/products/all/dna-electrophoresis/fastruler-dna-ladders/sm111-fastruler-middle</A></P><P>some results:
</P><P>PCR 8 (lanes 23, 24): no band
</P><P>Colony PCRs CS.2b.10, PCR1cam.3, PCR1cam.4, CMVcam.4, CMVcam.4, JD.1b5<SUB>1</SUB>.10   (lanes 7,8,9,10,11,14): show right bands
</P><P><FONT color="#009933">Plasmid extraction of &quot;over-day&quot; cultures ligation-7(=BB4),-8(=BB4),-9(=BB5),-12(=BB9) and pSB1C3</FONT></P><P>-&gt;Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><P>concentrations (ng/µl - A<SUB>260/280</SUB>): 
</P><DL><DD>BB4(7):57,5 - 1,643
</DD><DD>BB4(8):15 - 1,5
</DD><DD>BB5(9):173 - 1,816
</DD><DD>BB7(12):55,0 - 1,571
</DD><DD>pSB1C3: 12,5 - 1,250
</DD></DL><P><FONT color="#009933">new PCR 7b in order to get better results</FONT></P><TABLE><TBODY><TR><TD>Master Mix
</TD><TD>25µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>16,5µl
</TD></TR><TR><TD>Primer 15
</TD><TD>3,75µl
</TD></TR><TR><TD>Primer 16
</TD><TD>3,75µl
</TD></TR><TR><TD>template 190-6 (1:10)
</TD><TD>1µl
</TD></TR></TBODY></TABLE><P><FONT color="#009933">overnight-cultures</FONT></P><P>inoculated of: CS2bK10=BB11; PCP1K3+4=BB1; CMV-C-K3+4; JD1b5<SUB>1</SUB>K10=BB2; pSB1C3
</P><H2><SPAN class="mw-headline" id="9-23-2010"> 9-23-2010 </SPAN></H2><P><FONT color="#009933">Restriction digestion of BB5 for the 3A Method</FONT></P><TABLE><TBODY><TR><TD>Name
</TD><TD>H<SUB>2</SUB>O
</TD><TD>Buffer D
</TD><TD>BSA (1:10)
</TD><TD>DNA Volume
</TD><TD>DNA Mass
</TD><TD>Enzyms (2*0.5µl)
</TD></TR><TR><TD>BB5
</TD><TD>11 µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>4µl
</TD><TD>~500ng
</TD><TD>X+P
</TD></TR></TBODY></TABLE><P>sum: 20µl
</P><P>-&gt; Protocol:<A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digestion</A></P><P><FONT color="#009933">Plasmid extraction of overnight cultures BB2, BB11, CMV in cam and pSB1C3</FONT></P><P>-&gt;Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><P>concentrations (ng/µl - A<SUB>260/280</SUB>): 
</P><DL><DD>BB2:232 - 1,603
</DD><DD>BB11:170 - 1,889
</DD><DD>CMV in cam:272 - 1,703
</DD><DD>pSB1C3: 290 - 1,758
</DD></DL><P>The pSB1C3 colony was pink (produced a red pigment or something like that, so we make a new transformation in order to extract only pSB1C3 and not some other vectors.
</P><P><FONT color="#009933">Agarose gelelectrophoresis of PCR 7b and pathway restriction digestions</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gelphoto</DIV></DIV></DIV><P>From left to right: PCR7b, pathway (3x),ladder (5000,2000,850,400,100bp)
</P><P>result: <FONT color="#CC33CC"><B>PCR 7b shows right band with ~400bp </B></FONT></P><P><FONT color="#009933">PCR clean up of PCR 7b</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/12_Gel_extraction_or_PCR_Clean_up" title="Team:LMU-Munich/Notebook/Protocols/12 Gel extraction or PCR Clean up"> 12 Gel extraction or PCR Clean up (Promega kit)</A></P><P>result: 90ng/µl; A260/A280= 1,895
</P><P><FONT color="#009933">Restriction digestion of BB2 and BB11</FONT></P><TABLE><TBODY><TR><TD>Name
</TD><TD>H<SUB>2</SUB>O
</TD><TD>Buffer D
</TD><TD>BSA (1:10)
</TD><TD>DNA Volume
</TD><TD>DNA Mass
</TD><TD>Enzyms (2*0.5µl)
</TD></TR><TR><TD>BB2
</TD><TD>10 µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>5µl
</TD><TD>~1.2mg
</TD><TD>X+P
</TD></TR></TBODY></TABLE><P>sum: 20µl
</P><TABLE><TBODY><TR><TD>Name
</TD><TD>H<SUB>2</SUB>O
</TD><TD>Buffer D
</TD><TD>BSA (1:10)
</TD><TD>DNA Volume
</TD><TD>DNA Mass
</TD><TD>Enzyms (2*0.5µl)
</TD></TR><TR><TD>BB11
</TD><TD>7 µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>8µl
</TD><TD>~1.3mg
</TD><TD>X+P
</TD></TR></TBODY></TABLE><P>sum: 20µl
</P><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digestion</A></P><P><FONT color="#009933">Ligation of BB1 and BB6</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/22_Ligation" title="Team:LMU-Munich/Notebook/Protocols/22 Ligation">22 Ligation</A></P><TABLE><TBODY><TR><TD>Name
</TD><TD>H<SUB>2</SUB>O
</TD><TD>T4 Ligase-Buffer
</TD><TD>PCR1
</TD><TD>Primer 18+19
</TD><TD>ccdBAmp
</TD><TD>T4 Ligase
</TD></TR><TR><TD>BB1
</TD><TD>9.5 µl
</TD><TD>2µl
</TD><TD>5µl
</TD><TD>2µl
</TD><TD>1µl
</TD><TD>0.5µl
</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Name
</TD><TD>H<SUB>2</SUB>O
</TD><TD>T4 Ligase-Buffer
</TD><TD>CMV E+S
</TD><TD>BB5
</TD><TD>ccdBCam
</TD><TD>T4 Ligase
</TD></TR><TR><TD>BB6
</TD><TD>6.5 µl
</TD><TD>2µl
</TD><TD>8µl
</TD><TD>2µl
</TD><TD>1µl
</TD><TD>0.5µl
</TD></TR></TBODY></TABLE><P><FONT color="#009933">PCR 8 with 7b(new) plus 7a-2 with Phusion Hot Start II; Primer 16 plus 13k and 13l</FONT></P><TABLE><TBODY><TR><TD>Buffer 5x
</TD><TD>10µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>27µl
</TD></TR><TR><TD>dNTPs
</TD><TD>1µl
</TD></TR><TR><TD>Primer 16
</TD><TD>2,5µl
</TD></TR><TR><TD>Primer 13k short
</TD><TD>2,5µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,5µl
</TD></TR><TR><TD>template 7a(1:100)(850bp)
</TD><TD>3µl -&gt; 4ng
</TD></TR><TR><TD>template 7b(new)(1:100)(402bp)
</TD><TD>2µl -&gt; 2ng
</TD></TR><TR><TD>Phusion Hot Start II
</TD><TD>0,5µl
</TD></TR></TBODY></TABLE><DL><DD>-&gt;same with Primer 13l long&lt;-
</DD></DL><P><FONT color="#009933">Overnight culture of PCR1 in cam- Vector</FONT></P><P>new inoculation of PCR1 in cam, Colony 3 in 5ml LB+cam
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">From left to right: PCR8k,PCR8l,ladder (5000,2000,850,400,100bp)</DIV></DIV></DIV><P><FONT color="#009933">Transformation of pSB1C3, BB1, BB6</FONT></P><P>New transformation of pSB1C3 from Spring distribution because the old colony was pink.
</P><P>Transformation of BB1 and BB6.
</P><P>
BB1 plated on amp-Agar, BB6 and pSB1C3 plated on cam-Agar
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/19_Transformation_SSS" title="Team:LMU-Munich/Notebook/Protocols/19 Transformation SSS">19 Transformation (Sina Science Services)</A></P><P><FONT color="#009933">Agarose gelelectrophoresis of PCR8 (long and short primer 13)</FONT></P><P>PCR8k=short primer 13, PCR8l=long primer 13
</P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/11_Agarose_gel_electrophoresis" title="Team:LMU-Munich/Notebook/Protocols/11 Agarose gel electrophoresis"> 11 Agarose gel electrophoresis</A></P><P>result:<FONT color="#CC33CC"><B>PCR 8k and 8l shows right band with ~1200bp</B></FONT></P><P><FONT color="#009933">PCR purification of PCR 8</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/12_Gel_extraction_or_PCR_Clean_up" title="Team:LMU-Munich/Notebook/Protocols/12 Gel extraction or PCR Clean up"> 12 Gel extraction or PCR Clean up (Promega kit)</A></P><P>result: ~70ng/µl; A260/A280=1.75
</P><H2><SPAN class="mw-headline" id="9-24-2010"> 9-24-2010 </SPAN></H2><P><FONT color="#009933">New &quot;over-day&quot; culture of pSB1C3</FONT></P><P>Inoculate one culture from the ligation yesterday in 4ml LB+cam
</P><P><FONT color="#009933">Ligation of BB7<SUB>2</SUB></FONT></P><P>Mixture:
</P><P>Inserts: each 5µl; ccdB vector: 1µl; T4 Ligase: 0.5µl; T4 Ligase Buffer: 2µl; H<SUB>2</SUB>O: 7µl 
</P><P>Incubation: 5h 18°C; Inhibition: 20 min 65°C
</P><P>-&gt;Protocols: <A href="/Team:LMU-Munich/Notebook/Protocols/22_Ligation" title="Team:LMU-Munich/Notebook/Protocols/22 Ligation">22 Ligation</A>, <A href="/Team:LMU-Munich/Notebook/Protocols/13_3A_Method_for_Biobrick_assembly" title="Team:LMU-Munich/Notebook/Protocols/13 3A Method for Biobrick assembly"> 13 3A Method for Biobrick assembly</A></P><P><FONT color="#009933">Colony PCRs of BB1 and BB6</FONT></P><P>2 Colonys were picked from each plate -&gt; 4 charges
</P><P>Mixture for 4 charges with 5µl:
</P><TABLE><TBODY><TR><TD>PCR Mastermix 2x
</TD><TD>10µl</TD></TR><TR><TD>Primer F
</TD><TD>1.5µl
</TD></TR><TR><TD>Primer R
</TD><TD>1.5µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>7µl
</TD></TR><TR><TD>sum
</TD><TD>20µl
</TD></TR></TBODY></TABLE><P>+ Colony
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">from left to right: 1<SUB>1</SUB>, 1<SUB>2</SUB>, 6<SUB>1</SUB> and 6<SUB>2</SUB></DIV></DIV></DIV><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/21_PCR_with_Taq_Mastermix" title="Team:LMU-Munich/Notebook/Protocols/21 PCR with Taq Mastermix">21 PCR with Taq Mastermix</A></P><P>Annealing temperature: 53°C, standard protocol
</P><P><FONT color="#009933">Plasmid extraction of new PCR1 in cam vector</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><P>results:
</P><TABLE><TBODY><TR><TD> concentration
</TD><TD> 82.5ng/µl
</TD></TR><TR><TD> A260/A280
</TD><TD> 1.737
</TD></TR></TBODY></TABLE><P><FONT color="#009933">Agarose Gelelektrophoresis of 1<SUB>1</SUB>, 1<SUB>2</SUB>, 6<SUB>1</SUB> and 6<SUB>2</SUB></FONT></P><P><FONT color="#009933">Plasmid extraction of pSB1C3 in cam vector</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><P>results:
</P><TABLE><TBODY><TR><TD> concentration
</TD><TD> 22.5ng/µl
</TD></TR><TR><TD> A260/A280
</TD><TD> 1.5
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="9-25-2010"> 9-25-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="9-26-2010"> 9-26-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="9-27-2010"> 9-27-2010 </SPAN></H2><P><FONT color="#009933">Colony PCR of BB7<SUB>2</SUB>, BB6 and BB1 plus a negativ charge as reference</FONT></P><TABLE><TBODY><TR><TD> PCR Mastermix
</TD><TD> 17.5µl
</TD></TR><TR><TD> Primer forward
</TD><TD> 2.625µl
</TD></TR><TR><TD> Primer revers
</TD><TD> 2.7µl
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> 12.1µl
</TD></TR><TR><TD> sum
</TD><TD> 35µl = 5µl per charge
</TD></TR></TBODY></TABLE><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/24_Colony_PCR" title="Team:LMU-Munich/Notebook/Protocols/24 Colony PCR">24 Colony PCR</A></P><P><FONT color="#009933">Ligation of BB10</FONT></P><TABLE><TBODY><TR><TD> CMV© E+S
</TD><TD> 5µl
</TD></TR><TR><TD> PCR 8l X+P
</TD><TD> 5µl
</TD></TR><TR><TD> ccdBamp
</TD><TD> 1µl
</TD></TR><TR><TD> T4 Ligase-Buffer
</TD><TD> 2µl
</TD></TR><TR><TD> T4 Ligase
</TD><TD> 0.5µl
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> 6.5µl
</TD></TR><TR><TD> sum
</TD><TD> 20µl
</TD></TR></TBODY></TABLE><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/22_Ligation" title="Team:LMU-Munich/Notebook/Protocols/22 Ligation">22 Ligation</A></P><P><FONT color="#009933">Ligation of BB001(PCR1 in ©), BB017(PCR5 in ©), BB018(PCR6 in ©) and BB021(PCR9 in ©)</FONT></P><TABLE><TBODY><TR><TD> ccdB
</TD><TD> 1µl
</TD></TR><TR><TD> BB001/017/018/021
</TD><TD> 5µl
</TD></TR><TR><TD> T4 Ligase-Buffer
</TD><TD> 2µl
</TD></TR><TR><TD> T4 Ligase
</TD><TD> 0.5µl
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> 11.5µl
</TD></TR><TR><TD> sum
</TD><TD> 20µl
</TD></TR></TBODY></TABLE><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/22_Ligation" title="Team:LMU-Munich/Notebook/Protocols/22 Ligation">22 Ligation</A></P><P><FONT color="#009933">Restriction digestion of PCR3<SUB>2</SUB>, 8l, 10 and Primer 18+19 with E+P</FONT></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> template
</TH><TH> H<SUB>2</SUB>O
</TH><TH> BSA [1:10]
</TH><TH> Buffer H
</TH><TH> EcoRI
</TH><TH> PstI
</TH></TR><TR><TD> PCR3<SUB>2</SUB></TD><TD> 10µl
</TD><TD> 5µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 0.5µl
</TD><TD> 0.5µl
</TD></TR><TR><TD> PCR8l
</TD><TD> 10µl
</TD><TD> 5µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 0.5µl
</TD><TD> 0.5µl
</TD></TR><TR><TD> Primer 18+19
</TD><TD> 2x7.5µl
</TD><TD> --
</TD><TD>2µl
</TD><TD> 2µl
</TD><TD> 0.5µl
</TD><TD> 0.5µl
</TD></TR><TR><TD> PCR10
</TD><TD> 10µl
</TD><TD> 5µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 0.5µl
</TD><TD> 0.5µl
</TD></TR></TBODY></TABLE><P>-&gt; Protocol  <A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digestion</A></P><H2><SPAN class="mw-headline" id="9-28-2010"> 9-28-2010 </SPAN></H2><P><FONT color="#009933">Plating BB017, BB021, BB018, BB001, BB7<SUB>2</SUB>, BB10</FONT></P><TABLE class="wikitable" border="1"><TBODY><TR><TD> Amp
</TD><TD> BB7<SUB>2</SUB></TD><TD> BB10
</TD></TR><TR><TD> Cam
</TD><TD> BB001
</TD><TD> BB017
</TD><TD> BB018
</TD><TD> BB021
</TD></TR></TBODY></TABLE><P><FONT color="#009933">Colony PCR  BB1, BB6, BB7<SUB>2</SUB> with DreamTaq (each 2 colonies)</FONT></P><P>-&gt; Protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/16_PCR_with_DreamTaq" title="Team:LMU-Munich/Notebook/Protocols/16 PCR with DreamTaq"> 16 PCR with DreamTaq</A></P><P>PCR mixture for PCR7a 
</P><DL><DD><TABLE><TBODY><TR><TD>template
</TD><TD>pick up colony from plate
</TD></TR><TR><TD>primers*2
</TD><TD>0.25µl*2
</TD></TR><TR><TD>dNTPs
</TD><TD>0.5µl
</TD></TR><TR><TD>DreamTaq
</TD><TD>0.05µl
</TD></TR><TR><TD>10xbuffer
</TD><TD>0.5µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>3.45µl
</TD></TR><TR><TD>sum
</TD><TD>5µl
</TD></TR></TBODY></TABLE></DD></DL><P>Primers for colony PCR: BBseqF, BBseqR
</P><P>Annealing temperature: 53°C
</P><P><FONT color="#009933">Ligation of BB2, BB019, BB020, BB016, BB022</FONT></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> BB2
</TH><TH> BB019
</TH><TH> BB020
</TH><TH> BB016
</TH><TH> BB022
</TH><TH> Volume
</TH></TR><TR><TD> PCR
</TD><TD> 5<SUB>2</SUB></TD><TD> 8
</TD><TD> 3<SUB>2</SUB></TD><TD> 10
</TD><TD> each 5µl
</TD></TR><TR><TD> PCR
</TD><TD> 6
</TD><TD> each 5µl
</TD></TR><TR><TD> Primer
</TD><TD> 18+19
</TD><TD> each 5µl
</TD></TR><TR><TD> ccdBamp
</TD><TD> 1µl
</TD></TR><TR><TD> ccdBcam
</TD><TD> 1µl
</TD><TD> 1µl
</TD><TD> 1µl
</TD><TD> 1µl
</TD></TR><TR><TD> T4 Ligase-Buffer
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD></TR><TR><TD> T4 Ligase
</TD><TD> 0.5µl
</TD><TD> 0.5µl
</TD><TD> 0.5µl
</TD><TD> 0.5µl
</TD><TD> 0.5µl
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> 6.5µl
</TD><TD> 11.4µl
</TD><TD> 11.4µl
</TD><TD> 11.4µl
</TD><TD> 11.4µl
</TD></TR><TR><TD> sum
</TD><TD> 20µl
</TD><TD> 20µl
</TD><TD> 20µl
</TD><TD> 20µl
</TD><TD> 20µl
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="9-29-2010"> 9-29-2010 </SPAN></H2><P>plates BB7<SUB>2</SUB>, BB017, BB10, BB021, BB001, BB018: nothing grown -&gt; thrown into trash
</P><P><FONT color="#009933">colony-PCR of BB2/8, BB019, BB016, BB020, BB022 with Taq</FONT></P><P>-&gt;protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/25_PCR_with_Taq" title="Team:LMU-Munich/Notebook/Protocols/25 PCR with Taq">PCR with Taq</A></P><TABLE><TBODY><TR><TD>H<SUB>2</SUB>O
</TD><TD>6,7µl
</TD></TR><TR><TD>Buffer 10x
</TD><TD>1µl
</TD></TR><TR><TD>MgCl<SUB>2</SUB></TD><TD>1µl
</TD></TR><TR><TD>dNTPs
</TD><TD>0,2µl
</TD></TR><TR><TD>Primer forw. seq f
</TD><TD>0,5µl
</TD></TR><TR><TD>Primer rev. seq r
</TD><TD>0,5µl
</TD></TR><TR><TD>Taq
</TD><TD>0,1µl
</TD></TR></TBODY></TABLE><P>template: pick colony
</P><P>sum: 10µl; we made 100µl -&gt;10 charges (2 colonies of each plate)
</P><P>tube numbers: 
</P><TABLE class="wikitable" border="1"><TBODY><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD></TR><TR><TD>BB2/BB8 - 1
</TD><TD>BB2/BB8 - 2
</TD><TD>BB016 - 1
</TD><TD>BB016 - 2
</TD><TD>BB019 - 1
</TD><TD>BB019 - 2
</TD><TD>BB020 - 1
</TD><TD>BB020 - 2
</TD><TD>BB022 - 1
</TD><TD>BB022 - 2
</TD></TR></TBODY></TABLE><P>PCR-program: taq53
</P><P>agarose-gelelektrophoresis -&gt; no bands visible
</P><P><FONT color="#009933">new plates made of BB1, BB4, BB6, BB7</FONT></P><P><FONT color="#009933">Ligations BB7<SUB>2</SUB>, BB017 (mit PCR5<SUB>1</SUB>), BB10, BB021, BB001, BB018</FONT></P><TABLE class="wikitable" border="1"><TBODY><TR><TD>PCR (5 µl each)
</TD><TD>PCR (5µl each)
</TD><TD>ccdB(amp)
</TD><TD>ccdB©
</TD><TD>Buffer
</TD><TD>T4 Ligase
</TD><TD>H<SUB>2</SUB>O
</TD></TR><TR><TD>BB7<SUB>2</SUB>O
</TD><TD>1
</TD><TD>3<SUB>2</SUB></TD><TD>1µl
</TD><TD>2µl
</TD><TD>0,5µl
</TD><TD>6,5µl
</TD></TR><TR><TD>BB017
</TD><TD>5<SUB>1</SUB></TD><TD>1µl
</TD><TD>2µl
</TD><TD>0,5µl
</TD><TD>11,5µl
</TD></TR><TR><TD>BB10
</TD><TD>CMV©
</TD><TD>PCR8
</TD><TD>1µl
</TD><TD>2µl
</TD><TD>0,5µl
</TD><TD>6,5µl
</TD></TR><TR><TD>BB021
</TD><TD>9
</TD><TD>1µl
</TD><TD>2µl
</TD><TD>0,5µl
</TD><TD>11,5µl
</TD></TR><TR><TD>BB001
</TD><TD>1
</TD><TD>1µl
</TD><TD>2µl
</TD><TD>0,5µl
</TD><TD>11,5µl
</TD></TR><TR><TD>BB018
</TD><TD>6
</TD><TD>1µl
</TD><TD>2µl
</TD><TD>0,5µl
</TD><TD>11,5µl
</TD></TR></TBODY></TABLE><P><FONT color="#009933">restriction degestions</FONT></P><P>of PCR6, PCR5<SUB>2</SUB>, PCR3<SUB>2</SUB>, PCR1, ccdB(amp), ccdB(cam)
</P><TABLE class="wikitable" border="1"><TBODY><TR><TD>template
</TD><TD>digestion with
</TD><TD>volume
</TD></TR><TR><TD>PCR 6
</TD><TD>Xba1 &amp; Pst1
</TD><TD>I. 10µl   II. 3µl
</TD></TR><TR><TD>PCR5<SUB>2</SUB></TD><TD>EcoR1 &amp; Pst1
</TD><TD>20µl
</TD></TR><TR><TD>PCR3<SUB>2</SUB></TD><TD>Xba1 &amp; Pst1
</TD><TD>10µl
</TD></TR><TR><TD>PCR1
</TD><TD>EcoR1 &amp; Spe1
</TD><TD>2µl
</TD></TR><TR><TD>ccdB(amp)
</TD><TD>EcoR1 &amp; Pst1
</TD><TD>20µl
</TD></TR><TR><TD>cddB(cam)
</TD><TD>EcoR1 &amp; Pst1
</TD><TD>20µl
</TD></TR></TBODY></TABLE><P>-&gt;protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/5_Restriction_digest" title="Team:LMU-Munich/Notebook/Protocols/5 Restriction digest">5 Restriction digestion</A></P><P><FONT color="#009933">colony-PCR of BB4/c3, BB6/c7, BB7<SUB>2</SUB>/c5, BB1/c5, (BB11 as control) with Taq</FONT></P><P>-&gt;protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/25_PCR_with_Taq" title="Team:LMU-Munich/Notebook/Protocols/25 PCR with Taq">PCR with Taq</A></P><P>Mastermix:
</P><TABLE><TBODY><TR><TD>H<SUB>2</SUB>O
</TD><TD>32µl
</TD></TR><TR><TD>Buffer 10x
</TD><TD>5µl
</TD></TR><TR><TD>MgCl<SUB>2</SUB></TD><TD>5µl
</TD></TR><TR><TD>dNTPs
</TD><TD>1µl
</TD></TR><TR><TD>Primer forw. seq f
</TD><TD>2,5µl
</TD></TR><TR><TD>Primer rev. seq r
</TD><TD>2,5µl
</TD></TR><TR><TD>Taq
</TD><TD>0,5µl
</TD></TR><TR><TD>DMSO
</TD><TD>1,5µl
</TD></TR><TR><TD>sum
</TD><TD>50µl
</TD></TR></TBODY></TABLE><P>-&gt;10µl for each charge
</P><P>template: pick colony; BB11: 2ng
</P><P>tube numbers: 
</P><TABLE class="wikitable" border="1"><TBODY><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD></TR><TR><TD>BB4-c3
</TD><TD>BB6-c7
</TD><TD>BB7<SUB>2</SUB>-c5
</TD><TD>BB1-c5
</TD><TD>BB11
</TD></TR></TBODY></TABLE><P>PCR program:T<SUB>A</SUB> = 53°C; elongation time=1min
</P><H2><SPAN class="mw-headline" id="9-30-2010"> 9-30-2010 </SPAN></H2><P><FONT color="#009933">Colony PCR of BB2/BB8, BB6 K7&amp;K8 and test PCR of BB11 with new dilutions and primers</FONT></P><TABLE><TBODY><TR><TD> Colony/BB11
</TD><TD> 2.5µl
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> 32µl
</TD></TR><TR><TD> dNTP
</TD><TD> 1µl
</TD></TR><TR><TD> Primer F
</TD><TD> 2.5µl
</TD></TR><TR><TD> Primer R
</TD><TD> 2.5µl
</TD></TR><TR><TD> MgCl<SUB>2</SUB></TD><TD> 5µl
</TD></TR><TR><TD> DMSO
</TD><TD> 1.5µl
</TD></TR><TR><TD> Buffer 10x (with KCl)
</TD><TD> 5µl
</TD></TR><TR><TD> Taq Polymerase
</TD><TD> 0.5µl
</TD></TR><TR><TD> sum
</TD><TD> 56µl
</TD></TR></TBODY></TABLE><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/24_Colony_PCR" title="Team:LMU-Munich/Notebook/Protocols/24 Colony PCR">24 Colony PCR</A></P><P><FONT color="#009933">PCR purification of the digestions of 6I, 6II, 5<SUB>2</SUB>, 3<SUB>2</SUB>, 1</FONT></P><TABLE><TBODY><TR><TD> concentration [ng/µl]
</TD><TD> A260/A280
</TD></TR><TR><TD> 6II
</TD><TD> 32.5
</TD><TD> 1.444
</TD></TR><TR><TD> ccdBamp
</TD><TD> 0.0
</TD><TD> ---
</TD></TR><TR><TD> ccdBcam
</TD><TD> 5.0
</TD><TD> 1.000
</TD></TR><TR><TD> 6I
</TD><TD> 0.0
</TD><TD> ---
</TD></TR><TR><TD> 5<SUB>2</SUB></TD><TD> 20.0
</TD><TD> 1.143
</TD></TR><TR><TD> 3<SUB>2</SUB></TD><TD> 27.5
</TD><TD> 1.57
</TD></TR><TR><TD> 1
</TD><TD> 25
</TD><TD> 1.667
</TD></TR></TBODY></TABLE><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/12_Gel_extraction_or_PCR_Clean_up" title="Team:LMU-Munich/Notebook/Protocols/12 Gel extraction or PCR Clean up"> 12 Gel extraction or PCR Clean up (Promega kit)</A></P><P><FONT color="#009933">Transformation of BB018, BB7<SUB>2</SUB>, BB001, BB017, BB10 and BB021</FONT></P><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/19_Transformation_SSS" title="Team:LMU-Munich/Notebook/Protocols/19 Transformation SSS">19 Transformation SSS</A></P><P><FONT color="#009933">Plasmidextraction of the following overnight cultures: BB020(K1), BB018(K5), BB6(K5), BB022(K1), BB016(K1), BB019(K1), BB2(K1) and BB018(K3)</FONT></P><TABLE><TBODY><TR><TD> concentration [ng/µl]
</TD><TD> A260/A280
</TD></TR><TR><TD> BB020(K1)
</TD><TD> 163
</TD><TD> 1.711
</TD></TR><TR><TD> BB018(K5)
</TD><TD> 92.5
</TD><TD> 1.85
</TD></TR><TR><TD> BB6(K5)
</TD><TD> 120
</TD><TD> 1.778
</TD></TR><TR><TD> BB022(K1)
</TD><TD> 105
</TD><TD> 1.75
</TD></TR><TR><TD> BB016(K1)
</TD><TD> 85
</TD><TD> 1.889
</TD></TR><TR><TD> BB019(K1)
</TD><TD> 105
</TD><TD> 1.909
</TD></TR><TR><TD> BB2(K1)
</TD><TD> 190
</TD><TD> 1.81
</TD></TR><TR><TD> BB018
</TD><TD> 110
</TD><TD> 1.833
</TD></TR></TBODY></TABLE><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><H2><SPAN class="mw-headline" id="10-01-2010"> 10-01-2010 </SPAN></H2><P><FONT color="#009933">new test PCR of BB5 and BB11 with 20µl and 50µl volume</FONT></P><P>Master Mix
</P><TABLE><TBODY><TR><TD> H<SUB>2</SUB>O
</TD><TD> 67µl
</TD></TR><TR><TD> dNTP
</TD><TD> 2µl
</TD></TR><TR><TD> Primer F
</TD><TD> 5µl
</TD></TR><TR><TD> Primer R
</TD><TD> 5µl
</TD></TR><TR><TD> MgCl<SUB>2</SUB></TD><TD> 10µl
</TD></TR><TR><TD> Buffer 10x (with KCl)
</TD><TD> 10µl
</TD></TR><TR><TD> Taq Polymerase
</TD><TD> 1µl
</TD></TR><TR><TD> sum
</TD><TD> 100µl
</TD></TR></TBODY></TABLE><P>template: BB5/50µl: 5ng, 20µl: 1.5ng; BB11/20µl: 3ng
</P><P>-&gt;protocol: <A href="/Team:LMU-Munich/Notebook/Protocols/25_PCR_with_Taq" title="Team:LMU-Munich/Notebook/Protocols/25 PCR with Taq">PCR with Taq</A></P><P>from left to right: ladder, BB5/50µl, BB5/20µl, BB11/20µl. BB5/50µl and BB11/20µl show the right band.
</P><H2><SPAN class="mw-headline" id="10-02-2010"> 10-02-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="10-03-2010"> 10-03-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="10-04-2010"> 10-04-2010 </SPAN></H2><P>sequencing
</P><P>cycle, clean and run
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> CMV (c)
</TH><TH> PCR 1 (c)
</TH><TH> BB4 <SUB>7</SUB></TH><TH> BB4 <SUB>8</SUB></TH><TH> BB018K5
</TH><TH> BB018K3
</TH><TH> BB6K5
</TH><TH> BB2K1
</TH></TR><TR><TD> template
</TD><TD> 1µl
</TD><TD> 2µl
</TD><TD> 3µl
</TD><TD> 3,8µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 1µl
</TD></TR><TR><TD> primer (1:100)
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD></TR><TR><TD> 10mM TrispH 8,5
</TD><TD> 2,8 µl
</TD><TD> 1,8 µl
</TD><TD> 0,8 µl
</TD><TD> 0 µl
</TD><TD> 1,8 µl
</TD><TD> 1,8 µl
</TD><TD> 1,8 µl
</TD><TD> 1,8 µl
</TD></TR><TR><TD>name of tube
</TD><TD> 028 FIR
</TD><TD> 001 F/R
</TD><TD> 4-7 F/R
</TD><TD> 4-8 F/R
</TD><TD> 018-5 F/R
</TD><TD> 018-3 F/R
</TD><TD> 6-5 F/R
</TD><TD> 2-1 F/R
</TD></TR></TBODY></TABLE><P>
PCR
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> 019
</TH><TH> 022
</TH><TH> 016
</TH><TH> 020
</TH></TR><TR><TD> template (1:100 = 2ng)
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 2,5µl
</TD><TD> 1,5µl
</TD></TR><TR><TD> Primer F
</TD><TD> 1µl
</TD><TD> 1µl
</TD><TD> 1µl
</TD><TD> 1µl
</TD></TR><TR><TD> Primer R
</TD><TD> 1µl
</TD><TD> 1µl
</TD><TD> 1µl
</TD><TD> 1µl
</TD></TR><TR><TD> dNTPs
</TD><TD> 0,4µl
</TD><TD> 0,4µl
</TD><TD> 0,4µl
</TD><TD> 0,4µl
</TD></TR><TR><TD> Taq
</TD><TD> 0,2µl
</TD><TD> 0,2µl
</TD><TD> 0,2µl
</TD><TD> 0,2µl
</TD></TR><TR><TD> Buffer
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD></TR><TR><TD>MgCl<SUB>2</SUB></TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD> 11,4µl
</TD><TD> 11,4µl
</TD><TD> 10,9µl
</TD><TD> 11,
</TD></TR><TR><TD>colour of tube
</TD><TD> orange
</TD><TD> rose
</TD><TD> green
</TD><TD> blue
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="10-05-2010"> 10-05-2010 </SPAN></H2><P><FONT color="#009933">Colony PCRs of BB6, BB4, BB021, BB022, CMV©, BB001, BB016, BB1, BB2<SUB>5<SUB>2</SUB></SUB>, BB7<SUB>2</SUB>, BB017, BB018, BB019, BB020</FONT></P><P><FONT color="#009933">agarose gel electrophoresis</FONT></P><P>schedule of the agarose gel electrophoresis:
</P><TABLE class="wikitable" border="1"><TBODY><TR><TD> gel 1
</TD><TD> BB6K9
</TD><TD> BB6K10
</TD><TD> BB6K11
</TD><TD> BBK12
</TD><TD> BB6K13
</TD><TD> BB4K5
</TD><TD> ladder
</TD><TD> BB4K6
</TD><TD> BB4K7
</TD></TR><TR><TD> BB021K14
</TD><TD> BB021K13
</TD><TD> BB021K12
</TD><TD> BB021K11
</TD><TD> BB022K18
</TD><TD> BB022K19
</TD><TD> ladder
</TD><TD> BB022K17
</TD><TD> BB022K16
</TD></TR><TR><TD> gel 2
</TD><TD> BB022K12
</TD><TD> BB022K11
</TD><TD> CMV©K4
</TD><TD> CMV©K5
</TD><TD> BB001K5
</TD><TD> BB001K6
</TD><TD> BB001K7
</TD><TD> BB001K8
</TD><TD> ladder
</TD></TR><TR><TD> BB016K17
</TD><TD> BB1K16
</TD><TD> BB1K15
</TD><TD> BB1K8
</TD><TD> BB1K7
</TD><TD> BB2<SUB>5<SUB>2</SUB></SUB>K1
</TD><TD> BB2<SUB>5<SUB>2</SUB></SUB>K2
</TD><TD> BB2<SUB>5<SUB>2</SUB></SUB>K3
</TD><TD> ladder
</TD></TR><TR><TD> gel 3
</TD><TD> BB017K14
</TD><TD> BB017K13
</TD><TD> BB017K12
</TD><TD> BB017K11
</TD><TD> BB018K17
</TD><TD> BB018K16
</TD><TD> BB018K15
</TD><TD> BB018K14
</TD><TD> ladder
</TD></TR><TR><TD> BB019K6
</TD><TD> BB019K5
</TD><TD> BB019K4
</TD><TD> ladder
</TD><TD> BB019K3
</TD><TD> BB020K7
</TD><TD> BB020K8
</TD><TD> BB020K6
</TD><TD> BB020K5
</TD></TR></TBODY></TABLE><TABLE class="wikitable" border="1"><TBODY><TR><TD> gel 1
</TD><TD> BB4K8
</TD><TD> BB4K9
</TD><TD> BB4K10
</TD><TD> BB021K10
</TD><TD> BB021K15
</TD></TR><TR><TD> BB022K15
</TD><TD> BB022K14
</TD><TD> BB022K13
</TD><TD> ---
</TD><TD> ---
</TD></TR><TR><TD> gel 2
</TD><TD> BB001K9
</TD><TD> BB001K10
</TD><TD> BB016K11
</TD><TD> BB016K12
</TD><TD> BB016K13
</TD><TD> BB016K14
</TD><TD> BB016K15
</TD><TD> BB016K16
</TD><TD> ---
</TD></TR><TR><TD> ---
</TD><TD> BB2<SUB>5<SUB>2</SUB></SUB>K4
</TD><TD> BB2<SUB>5<SUB>2</SUB></SUB>K5
</TD><TD> BB7<SUB>2</SUB>K12
</TD><TD> BB7<SUB>2</SUB>K11
</TD><TD> BB7<SUB>2</SUB>K10
</TD><TD> BB7<SUB>2</SUB>K9
</TD><TD> BB017K16
</TD><TD> BB017K15
</TD></TR><TR><TD>gel 3
</TD><TD> BB018K13
</TD><TD> BB018K12
</TD><TD> BB018K11
</TD><TD> BB018K10
</TD><TD> BB018K9
</TD></TR><TR><TD> BB020K4
</TD><TD> BB020K3
</TD><TD> ---
</TD><TD> ---
</TD></TR></TBODY></TABLE><P>Gelbilder und Auswertung einfügen
</P><P><FONT color="#009933">overnight cultures of BB6K3, BB4K10, CMV©K4, BB001K5, BB001K9, BB016K11, BB1K8, BB2<SUB>5<SUB>2</SUB></SUB>K3, BB7<SUB>2</SUB>, BB017K14, BB018K10, BB019K6</FONT></P><H2><SPAN class="mw-headline" id="10-06-2010"> 10-06-2010 </SPAN></H2><P><FONT color="#009933">Plasmidextraction of the following overnight cultures: BB6(K13), BB4(K13), CMVc (K4), BB001(K5), BB001(K9), BB016(K11), BB1(K8), BB2(K3), BB7(K10), BB017(K14), BB018(K10) and BB019(K6) </FONT></P><TABLE><TBODY><TR><TD> concentration [ng/µl]
</TD><TD> A260/A280
</TD></TR><TR><TD> BB6(K13)
</TD><TD> 265
</TD><TD> 1.8
</TD></TR><TR><TD> BB4(K10)
</TD><TD> 40
</TD><TD> 2.0
</TD></TR><TR><TD> CMVc(K4)
</TD><TD> 97.5
</TD><TD> 1.8
</TD></TR><TR><TD> BB001(K5)
</TD><TD> 30
</TD><TD> 1.7
</TD></TR><TR><TD> BB001(K9)
</TD><TD> 17.5
</TD><TD> 1.7
</TD></TR><TR><TD> BB016(K11)
</TD><TD> 45
</TD><TD> 1.7
</TD></TR><TR><TD> BB1(K8)
</TD><TD> 57.5
</TD><TD> 1.9
</TD></TR><TR><TD> BB2(K3)
</TD><TD> 60
</TD><TD> 1.8
</TD></TR><TR><TD> BB7(K10)
</TD><TD> 30
</TD><TD> 1.7
</TD></TR><TR><TD> BB017(K14)
</TD><TD> 12.5
</TD><TD> 1.5
</TD></TR><TR><TD> BB018(K10)
</TD><TD> 35
</TD><TD> 1.8
</TD></TR><TR><TD>BB019(K6)
</TD><TD> 37.5
</TD><TD> 1.5
</TD></TR></TBODY></TABLE><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><H2><SPAN class="mw-headline" id="10-07-2010"> 10-07-2010 </SPAN></H2><P><FONT color="#009933"> colony PCR of BB021, BB022 with Taq (53°C) </FONT></P><TABLE><TBODY><TR><TD> Primer fw
</TD><TD> 0.5 µl
</TD></TR><TR><TD> Primer rev
</TD><TD> 0.5 µl
</TD></TR><TR><TD> dNTPs
</TD><TD> 0.2 µl
</TD></TR><TR><TD> Taq
</TD><TD> 0.1 µl
</TD></TR><TR><TD> 10xbuffer
</TD><TD> 1 µl
</TD></TR><TR><TD> MgCl<SUB>2</SUB></TD><TD> 1 µl
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> 6.7 µl
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="10-08-2010"> 10-08-2010 </SPAN></H2><P><FONT color="#009933"> 
plasmide extractions from 10-06-2010 were put in a lyophilizer and redissolved in less water (10-20 µl)
</FONT></P><TABLE><TBODY><TR><TD> concentration [ng/µl]
</TD><TD> A260/A280
</TD></TR><TR><TD> BB4(K10)
</TD><TD>     258
</TD><TD>     1,981
</TD></TR><TR><TD> BB001(K5)
</TD><TD>     325
</TD><TD>     1.94
</TD></TR><TR><TD> BB001(K9)
</TD><TD>     205
</TD><TD>     1.907
</TD></TR><TR><TD> BB7(K10)
</TD><TD>     170
</TD><TD>     1.744
</TD></TR><TR><TD> BB017(K14)
</TD><TD>     215
</TD><TD>     1.955
</TD></TR><TR><TD> BB018(K10)
</TD><TD>     500
</TD><TD>     1.88
</TD></TR><TR><TD>BB019(K6)
</TD><TD>     293
</TD><TD>     1.95
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="10-09-2010"> 10-09-2010 </SPAN></H2><P><FONT color="#009933">Plasmid extraction BB018K5, BB018K6, BB021K17 and BB021K24</FONT></P><P>concentration of these samples to be measured on Monday
</P><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/4_Plasmid_extraction_from_cells" title="Team:LMU-Munich/Notebook/Protocols/4 Plasmid extraction from cells">4 Plasmid extraction from cells</A></P><H2><SPAN class="mw-headline" id="10-10-2010"> 10-10-2010 </SPAN></H2><P>weekend
</P><H2><SPAN class="mw-headline" id="10-11-2010"> 10-11-2010 </SPAN></H2><P><FONT color="#009933">Plasmid extraction BB018K5, BB018K6, BB021K24 and BB021K17 from 10-09-2010
</FONT></P><TABLE><TBODY><TR><TD> concentration [ng/µl]
</TD><TD> A260/A280
</TD></TR><TR><TD> BB018K5
</TD><TD>     55
</TD><TD>     1.571
</TD></TR><TR><TD> BB018K6
</TD><TD>    92,5
</TD><TD>     1.762
</TD></TR><TR><TD> BB021K24
</TD><TD>     27.5
</TD><TD>     1.833
</TD></TR><TR><TD> BB021K17
</TD><TD>     37.5
</TD><TD>     1.508
</TD></TR></TBODY></TABLE><P><FONT color="#009933">Colony PCR BB01, BB018, BB6, BB252</FONT></P><TABLE><TBODY><TR><TD> stripe 1
</TD><TD> BB001K5
</TD><TD> K6
</TD><TD> K7
</TD><TD> K8
</TD><TD> K9
</TD><TD> K9
</TD><TD> K10
</TD><TD> BB252 K2
</TD></TR><TR><TD> stripe 2
</TD><TD> BB252 K3
</TD><TD> K4
</TD><TD> K5
</TD><TD> BB6 K9
</TD><TD> K10
</TD><TD> K11
</TD><TD> K12
</TD><TD> K13
</TD></TR><TR><TD> stripe 3
</TD><TD> BB018 K9
</TD><TD> K10
</TD><TD> K11
</TD><TD> K12
</TD><TD> K13
</TD><TD> K14
</TD><TD> K15
</TD><TD> K16
</TD></TR><TR><TD>single sample
</TD><TD>BB018 K17
</TD></TR></TBODY></TABLE><P>
gel picture colony PCR
</P><P>foto einfügen!
</P><P><FONT color="#009933">Sequencing 8,9 and 10 (cycle, clean and run)</FONT></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> BB001K9 for
</TH><TH> BB001K9 rev
</TH><TH> BB4K16 for
</TH><TH> BB4K16 rev
</TH><TH> BB001K5 for
</TH><TH> BB001K5 rev
</TH><TH> BB2<SUB>5<SUB>2</SUB></SUB>K3
</TH><TH> BB2<SUB>5<SUB>2</SUB></SUB>K3
</TH></TR><TR><TD> template
</TD><TD> 1µl
</TD><TD> 1µl
</TD><TD> 1µl
</TD><TD> 1µl
</TD><TD> 0,5µl
</TD><TD> 0,5µl
</TD><TD> 3,8µl
</TD><TD> 3,8µl
</TD></TR><TR><TD> primer (1:100)
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD></TR><TR><TD> 10mM TrispH 8,5
</TD><TD> 2,8 µl
</TD><TD> 2,8 µl
</TD><TD> 2,8 µl
</TD><TD> 2,8 µl
</TD><TD> 3,3 µl
</TD><TD> 3,3 µl
</TD><TD> 0 µl
</TD><TD> 0 µl
</TD></TR><TR><TD>name of tube
</TD><TD> 31-F
</TD><TD> 31-R
</TD><TD> 44-F
</TD><TD> 44-R
</TD><TD> 24-F
</TD><TD> 24-R
</TD><TD> 32-F
</TD><TD> 32-R
</TD></TR></TBODY></TABLE><TABLE class="wikitable" border="1"><TBODY><TR><TH> BB7<SUB>2</SUB>K10 for
</TH><TH> BB7<SUB>2</SUB>K10 rev
</TH><TH> BB017K14 for
</TH><TH> BB017K14 rev
</TH><TH> BB018K10 for
</TH><TH> BB018K10 rev
</TH><TH> BB019K6 for
</TH><TH> BB019K6 rev
</TH></TR><TR><TD> template
</TD><TD> 1,5µl
</TD><TD> 1,5µl
</TD><TD> 1µl
</TD><TD> 1µl
</TD><TD> 0,5µl
</TD><TD> 0,5µl
</TD><TD> 0,8µl
</TD><TD> 0,8µl
</TD></TR><TR><TD> primer (1:100)
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD></TR><TR><TD> 10mM TrispH 8,5
</TD><TD> 2,3 µl
</TD><TD> 2,3 µl
</TD><TD> 2,8 µl
</TD><TD> 2,8 µl
</TD><TD> 3,3 µl
</TD><TD> 3,3 µl
</TD><TD> 3 µl
</TD><TD> 3 µl
</TD></TR><TR><TD>name of tube
</TD><TD> 14-F
</TD><TD> 14-R
</TD><TD> 21-F
</TD><TD> 21-R
</TD><TD> 13-F
</TD><TD> 13-R
</TD><TD> 34-F
</TD><TD> 34-R
</TD></TR></TBODY></TABLE><TABLE class="wikitable" border="1"><TBODY><TR><TH> CMV for
</TH><TH> CMV rev
</TH><TH> BB018K5 for
</TH><TH> BB018K5 rev
</TH><TH> BB018K6 for
</TH><TH> BB018K6 rev
</TH><TH> BB021K17
</TH><TH> BB021K17
</TH></TR><TR><TD> template
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 3,8µl
</TD><TD> 3,8µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 3,8µl
</TD><TD> 3,8µl
</TD></TR><TR><TD> primer (1:100)
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD><TD> 3,2 µl
</TD></TR><TR><TD> 10mM TrispH 8,5
</TD><TD> 1,8 µl
</TD><TD> 1,8 µl
</TD><TD> 0 µl
</TD><TD> 0 µl
</TD><TD> 1,8 µl
</TD><TD> 1,8 µl
</TD><TD> 0 µl
</TD><TD> 0 µl
</TD></TR><TR><TD>name of tube
</TD><TD> CMV-F
</TD><TD> CMV-R
</TD><TD> 185-F
</TD><TD> 185-R
</TD><TD> 186-F
</TD><TD> 186-R
</TD><TD> 2117-F
</TD><TD> 2117-R
</TD></TR></TBODY></TABLE><P><FONT color="#009933">ligation BB020 BB022, Bb6 and BB5<SUB>2</SUB></FONT></P><P>-&gt; Protocol <A href="/Team:LMU-Munich/Notebook/Protocols/19_Transformation_SSS" title="Team:LMU-Munich/Notebook/Protocols/19 Transformation SSS">19 Transformation SSS</A></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> BB020
</TH><TH> BB022
</TH><TH> BB6
</TH><TH> BB5<SUB>2</SUB></TH></TR><TR><TD> insert (5µl)
</TD><TD> 18+19
</TD><TD> PCR 10
</TD><TD> CMV
</TD><TD> PCR5&lt;sub&lt;2&lt;/sub&gt;</TD></TR><TR><TD> insert (5µl)
</TD><TD> -
</TD><TD> -
</TD><TD> BB5
</TD><TD> PCR6
</TD></TR><TR><TD> ccdB (1µl)
</TD><TD> c
</TD><TD> c
</TD><TD> c
</TD><TD> a
</TD></TR><TR><TD> buffer
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD><TD> 2µl
</TD></TR><TR><TD> T4-ligase
</TD><TD> 0,5µl
</TD><TD> 0,5µl
</TD><TD> 0,5µl
</TD><TD> 0,5µl
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> 11,5µl
</TD><TD> 11,5µl
</TD><TD> 6,5µl
</TD><TD> 6,5µl
</TD></TR></TBODY></TABLE><P><FONT color="#009933">PCR to test self-made taq</FONT></P><TABLE><TBODY><TR><TD> template eGFP
</TD><TD> 1µl
</TD></TR><TR><TD> Primer 20,21
</TD><TD> each 1µl
</TD></TR><TR><TD> buffer (10x)
</TD><TD> 2µl
</TD></TR><TR><TD> MgCl
</TD><TD> 2µl
</TD></TR><TR><TD> dNTPs
</TD><TD> 0,2µl
</TD></TR><TR><TD> taq
</TD><TD> 0,3µl to 0,7µl (0,1µl steps)
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> fill to 20 µl
</TD></TR></TBODY></TABLE><P>anealing temperature 58°C, elongation time 30
</P><P> </P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:LMU-Munich/Notebook/Apoptosis">http://2010.igem.org/Team:LMU-Munich/Notebook/Apoptosis</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LMU-Munich/Notebook/Apoptosis" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LMU-Munich/Notebook/Apoptosis" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LMU-Munich/Notebook/Apoptosis&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LMU-Munich/Notebook/Apoptosis&amp;oldid=190797" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>