<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_London_BioHackspace_Project_Background"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:London_BioHackspace/Project_Background">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:London_BioHackspace/Project_Background&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:London_BioHackspace/Project_Background&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:London_BioHackspace/Project_Background&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:London_BioHackspace/Project_Background" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:London BioHackspace/Project Background</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TABLE width="100%"><TBODY><TR heigth="75px"><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:London_BioHackspace" style="color:#000000">Home </A></TD><TD style="border:1px solid black" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:London_BioHackspace/Project" style="color:#000000"> Project</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:London_BioHackspace/Team" style="color:#000000"> Team </A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:London_BioHackspace/Policy_Practice" style="color:#000000">Policy &amp; Practice</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:London_BioHackspace/Protocols" style="color:#000000">Protocols</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:London_BioHackspace/Safety" style=" color:#000000"> Safety </A></TD></TR></TBODY></TABLE></P><H1>Background</H1><H2>Bacterial Cellulose</H2><P>Bacterial cellulose (BC) is an extracellular product of a number number of species of bacteria, however the most common methods of BC production rely on a select few species of the <EM>Acetobacteraceae</EM> family (Legge, 1990). Unlike the cellulose found in plant tissue such as wood or cotton, BC has a very high crystalinity due to the lack of impurities such as the lignin and hemicellulose produced in plant tissues (Valla et al., 2009).</P><P>Bacterial cellulose is formed of extruded fibrils of cellulose which are produced in a trail-like fashion by the bacteria (see Figure 1)(Chawla and Bajaj, 2009). These fibrils are interwoven and organised randomly, producing macro-scale sheets without a directional &quot;grain&quot;. As a result, sheets of bacterial cellulose are mechanically strong allowing. Additionally, BC is biocompatible and has been shown to promote mammalian cell proliferation effectively without inducing significant pro-inflammatory cytokine production. This has lead to interest in using BC as a scaffold for grafting, in vitro 3D cell culture, or mammalian tissue growth (Keshk, 2014). </P><P>[Figure1 - chawla<EM>bajaj</EM>figure]</P><H2>G. hansenii</H2><P><A href="http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&amp;id=714995&amp;lvl=3&amp;lin=f&amp;keep=1&amp;srchmode=1&amp;unlock"><EM>Gluconacetoabacter hansenii</EM> (LMG1524, ATCC 23769)</A>, previously classified as <EM>Acetobacter xylinum</EM> is an alphaproteobacter originally isolated from vinegar which is well studied for its production of bacterial cellulose. <EM>G. hansenii</EM> is a model organism for the study of bacterial cellulose production and like other BC producing bacteria, <EM>G. hansenii</EM>, manufactures extracellular cellulose in a thin layer at the air-medium interface of a liquid medium.</P><P><EM>G. hansenii</EM> has been used in numerous studies relating to bacterial cellulose production and has an fully sequenced genome (Iyer et al., 2010). We are using <EM>G. hansenii</EM> over the related <EM>G. xylinus</EM>, a more prodigious producer of BC, due to the more complete genomic and metabolic information available. <EM>G. hansenii</EM> will therefore provide our starting point for the production of a light sensitive extracellular cellulose producing organism for use in our light based 3D cellulose printer: JuicyPrint.</P><H2>3D Printing</H2><P>3D printing is a technology which has started to become a common place tool for a large range of engineering or fabrication projects. This is largely due to the open source movement producing software and hardware that is easy to use without the need for extensive training. Current 3D printers allow the rapid design and production of high quality 3-dimensional objects made of a broad range of materials.</P><P>JuicyPrint hopes to follow in the model of successful opensource 3D printer projects such as <A href="http://reprap.org">RepRap</A> which are designed to be constructed from easily available generic parts and, where that is not possible, from parts that can be produced from a pre-existing unit.</P><P>Our project is inspired by photosensitive resin 3D printers which utilise a liquid resin that can be cured with exposure to intense light. In these printers, the object to be printed is built layer by layer through the projection of a pattern of light onto the photosensitive resin's surface. This cures a thin layer of resin in a corresponding pattern to that of the projected light which adheres to the previous layer. By altering the pattern of projected light for each layer, intricate 3-dimensional objects can therefore be built out of resin.</P><P>In our printer 3D printer JuicyPrint, light sensitive <EM>G. hansenii</EM> produces a macro-scale layer of extracellular cellulose at the surface of the liquid medium. As with the photosensitive resin printer, we control the pattern of the forming layer of solid material using a projected pattern of light. The major differences between JuicyPrint and photosensitive resin printers are:
<UL><LI>3D structures created are formed of gelatinous bacterial cellulose rather than rigid resin</LI><LI>the process of converting the liquid feedstock into structural material relies on biological signal transduction and metabolism rather than a rapid chemical change</LI></UL></P><H2>Genetics of Engineered Light Sensitivity</H2><P>Our project relies on previous contributions to the iGEM Registry of Standard Biological parts in order to engineer a light sensitive transcription pathway into bacteria. This pathway, developed by Levskaya et al. (2005), is based on the well characterised EnvZ-OmpR two-component signal transduction system found in <EM>E. coli</EM>. The modified pathway replaces the naturally present EnvZ osmolarity sensor with an engineered protein Cph8, a fusion of the light responsive domain of Cph1 (a cyanobacterial phytochrome ) and the histidine kinase domain of EnvZ. This allows light mediated transcription of genes promoted by the ompC promoter (ompCp). </P><P>The Cph8 signal transduction pathway is a simple pathway which has been used in numerous iGEM projects to introduce light sensitivity to bacteria (e.g. <A href="https://2013.igem.org/Team:NCTU_Formosa">E.colightuner</A>, <A href="https://2010.igem.org/Team:Edinburgh">Communicating Through Bridges</A>). This pathway forms the basis of introducing light mediated extracellular cellulose production into <EM>G. hansenii</EM> in our project. </P><P>Table 1 gives an overview of the genes involved in the light sensitive signal transduction system.</P><TABLE class="wikitable" width="100%"><TBODY><TR><TH> Gene </TH><TH> Function </TH><TH width="60%"> Description </TH><TH> Origin </TH></TR><TR><TD><I>envZ</I></TD><TD> Encodes <B>EnvZ</B></TD><TD> EnvZ is a histidine kinase/phosphatase found in <I>E. coli</I>. EnvZ responds to the osmolarity changes in the medium through variations in membrane surface tension triggering conformational changes and phosphorylates/dephosphorylates OmpR. Low osmolarity -&gt; dephosphorylation of OmpR. High osmolarity -&gt; phosphorylation of OmpR.
</TD><TD><I>E. coli</I></TD></TR><TR><TD><I>cph1</I></TD><TD> Encodes <B>Cph1</B></TD><TD> Cph1 is a cyanobacterial phytochrome (protein in which exposure to light induces conformational changes).
</TD><TD><I>Synechocystis</I></TD></TR><TR><TD><I>cph8</I></TD><TD> Encodes <B>Cph8</B></TD><TD> Cph8 is a fusion of the light responsive domain of Cph1 and the histidine kinase domain of EnvZ. The light responsive domain (Cph1) has maximal response to light near 660nm. Exposure to red light inhibits the activity of the EnvZ histidine kinase domain. When active (in the dark) its kinase domain phosphorylates endogenous OmpR. Light sensitivity is only functional in the presence of <B>Phycocyanobilin (PCB)</B>.
</TD><TD> Engineered
</TD></TR><TR><TD><I>ho1</I></TD><TD> Encodes <B>Ho1</B></TD><TD> Heme oxygenase (Ho1) is a product of ho1. One of two proteins required for the biosynthesis of PCB from heme, thus is required to produce functional Cph1 or Cph8.
</TD><TD><I>Synechocystis</I></TD></TR><TR><TD><I>pcyA</I></TD><TD> Encodes <B>PcyA</B></TD><TD> Phycocyanobilin:ferredoxin oxidoreductase (PcyA) is the second of two proteins required for the biosynthesis of PCB from heme. Thus it is required to produce functional Cph1 or Cph8.
</TD><TD><I>Synechocystis</I></TD></TR><TR><TD><I>ompR</I></TD><TD> Encodes <B>OmpR</B></TD><TD> OmpR is a DNA-binding protein (transcription factor) that is phosphorylated or dephosphorylated by EnvZ depending the osmolarity of the medium. Phosphorylated OmpR (OmpR-P) binds to sites within <I>ompCp</I> and <I>ompFp</I>. Low levels of OmpR-P upregulate ompF transcription. High levels of OmpR-P upregulate ompC transcription and repress ompF transcription (see Egger et al., 1997).<U>BEWARE: The iGEM repository uses ompR and OmpR where they mean to say ompCp/OmpR-binding site.</U></TD><TD><I>E. coli</I></TD></TR><TR><TD><I>ompC</I></TD><TD> Encodes <B>OmpC</B></TD><TD> OmpC is an outer membrane porin found in E. coli.
</TD><TD><I>E. coli</I></TD></TR><TR><TD><I>ompCp</I></TD><TD> Regulates <I>ompC</I> transcription
</TD><TD><I>ompCp</I> is the upstream operator-promoter region of <I>ompC</I>. <I>ompCp</I> contains three OmpR-P binding sites: the C1, C2, and C3 sites located in the the -100 to -38 region of <I>ompC</I>. Higher levels of OmpR-P lead to increased rate of binding and lead to upregulation of ompC transcription.
</TD><TD><I>E. coli</I></TD></TR><TR><TD><I>ompF</I></TD><TD><I>Encodes <B>OmpF</B></I></TD><TD> OmpF is an outer membrane porin found in E. coli.
</TD><TD><I>E. coli</I></TD></TR><TR><TD><I>ompFp</I></TD><TD> Regulates <I>ompF</I> transcription
</TD><TD><I>ompFp</I> is the upstream operator-promoter region of <I>ompF</I>. <I>ompFp</I> contains four OmpR-P binding sites: the -380 to the -361 region (F4 site), and the -100 to -39 region (F1, F2, and F3 sites) of ompF. With low levels of OmpR-P only the F1, F2, and F3 sites are bound leading to upregulated transcription of <I>ompF</I>. With high levels of OmpR-P the F4 site is also bound and leads to repression of <I>ompF</I> transcription.
</TD><TD><I>E. coli</I></TD></TR></TBODY></TABLE><H2>Genetics of Cellulose Production in <EM>G. hansenii</EM></H2><P>A number of the core genes involved in extracellular cellulose production in <EM>G. hansenii</EM> have recently been characterised using transposon mutagenesis (Deng et al., 2013). Of these genes, <EM>dgc1</EM> is a primary candidate for use as a genetic switch for controlling the production of extracellular cellulose. <EM>dgc1</EM> encodes diguanylate cyclase (Dgc) which is <STRONG>necessary</STRONG> for the production of extracellular cellulose in <EM>G. hansenii</EM> - <EM>dgc1</EM> knockout mutants of <EM>G. hansenii</EM> do not produce extracellular cellulose. Dgc catalyses the formation of c-di-GMP which is known to regulate biofilm formation, motility, and the production of extracellular polysaccharides (RÃ¶mling and Amikam, 2006).</P><H2>References</H2><P><STRONG>Chawla, P.</STRONG>, Bajaj, I., 2009. Microbial cellulose: Fermentative production and applications. Food Technology and â¦ 47, 107â124.</P><P><STRONG>Deng, Y.</STRONG>, Nagachar, N., Xiao, C., Tien, M., Kao, T.-H., 2013. Identification and Characterization of Non-Cellulose-Producing Mutants of Gluconacetobacter hansenii Generated by Tn5 Transposon Mutagenesis. Journal of bacteriology. doi:10.1128/JB.00767-13</P><P><STRONG>Egger, L.A.</STRONG>, Park, H., Inouye, M., 1997. Signal transduction via the histidyl-aspartyl phosphorelay. Genes Cells 2, 167â184.</P><P><STRONG>Iyer, P.R.</STRONG>, Geib, S.M., Catchmark, J., Kao, T., Tien, M., 2010. Genome sequence of a cellulose-producing bacterium, Gluconacetobacter hansenii ATCC 23769. Journal of bacteriology 192, 4256â7. doi:10.1128/JB.00588-10</P><P><STRONG>Keshk, S.M.</STRONG>, 2014. Bacterial Cellulose Production and its Industrial Applications. Journal of Bioprocessing &amp; Biotechniques 04. doi:10.4172/2155-9821.1000150</P><P><STRONG>Legge, R.L.</STRONG>, 1990. Microbial cellulose as a speciality chemical. Biotechnology Advances 8, 303â319. doi:10.1016/0734-9750(90)91067-Q</P><P><STRONG>Levskaya, A.</STRONG>, Chevalier, A.A., Tabor, J.J., Simpson, Z.B., Lavery, L.A., Levy, M., Davidson, E.A., Scouras, A., Ellington, A.D., Marcotte, E.M., Voigt, C.A., 2005. Synthetic biology: engineering Escherichia coli to see light. Nature 438, 441â442. doi:10.1038/nature04405</P><P><STRONG>RÃ¶mling, U.</STRONG>, Amikam, D., 2006. Cyclic di-GMP as a second messenger. Curr. Opin. Microbiol. 9, 218â228. doi:10.1016/j.mib.2006.02.010</P><P><STRONG>Valla, S.</STRONG>, ErtesvÃ¥g, H., Tonouchi, N., FjÃ¦rvik, E., 2009. Bacterial cellulose production: biosynthesis and applications, in: Rehm, B.H.A. (Ed.), . Caister Academic Press, Norfolk, UK, pp. 43â77.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:London_BioHackspace/Project_Background">http://2014.igem.org/Team:London_BioHackspace/Project_Background</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:London_BioHackspace/Project_Background" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:London_BioHackspace/Project_Background" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:London_BioHackspace/Project_Background&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:London_BioHackspace/Project_Background&amp;oldid=392773" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>