Team:Nanjing NFLS/Notebook
Team:Nanjing_NFLS 2017-igem.org
HOME
PROJECT
OVERVIEW
DESIGN
RESULT
VERIFICATION
SAFETY
TEAMS
MEMBERS
INSTRUCTORS
ATTRIBUTIONS
COLLABORATIONS
HUMAN PRACTICE
NOTEBOOK
PROTOCOL
LAB PICTURES
PARTS
Team:Nanjing_NFLS 2017-igem.org/Notebook
HOME
PROJECT
OVERVIEW
DESIGN
RESULT
VERIFICATION
SAFETY
TEAMS
MEMBERS
INSTRUCTORS
ATTRIBUTIONS
COLLABORATIONS
HUMAN PRACTICE
NOTEBOOK
PROTOCOL
LAB PICTURES
PARTS
Notebook
Protocol
1) DNA extraction of contaminated soil
Use PowerSoil DNA Isolation Kit（#12888-50，MOBIO）
2) Random Mutation PCR（Domain mutagenesis kit）
Thaw Taq, dNTP, primers, template DNA on ice.
To a new PCR tube, add:
Mix solution well.
Place tube in PCR thermocycler. Set thermocycler program:
Inititial denaturation: 5 min at 95℃;
Loop (35 cycles), Denaturation: 30s at 95℃,Annealing: 30s at 60℃,Elongation: 1min at 72℃;
Final elongation: 10min at 72℃;
Store: 10℃.
We use 5μL of the PCR product for electrophoresis and 45μL for purification.
3) Agarose Gel Electrophoresis
Weigh agarose powder and TBE buffer according to a proper portion, and add them to a 100mL conical flask (we usually make 1.5% Agarose Gel).
Melt the mixture in a microwave until the solution becomes clear (don’t leave the microwave).
Let the solution cool down to about 40-50℃ and add DNA gel stain (usually we use EB), pour the solution into the gel casting tray with appropriate comb.
Let the gel cool until it becomes solid.
PμLl out the comb carefμLly.
Place the gel in the electrophoresis chamber.
Add enough TAE Buffer so that there is about 2-3mm of buffer over the gel.
Pipette DNA samples mixed with appropriate amount of DNA loading buffer (the dye/GeneFinder is in the loading buffer) into wells on the gel.
Run the gel at 135V for about twenty minutes.
4) PCR products cleanup using AMPure XP beads
Warm the Ampure beads to room temperature and mix thoroughly before use.
Prepare 3.6 mls of 70% ethanol for each sample. The 70% ethanol solution should be prepared fresh.
Add 70 μl （1.8×） of Ampure beads to each sample, mix thoroughly and rotate for 10 minutes at room temperature.
Place samples on a magnetic separator. When the beads have collected to the wall of the tube and the solution is clear, transfer the liquid to a new tube. Be careful not to disturb the beads. The liquid contains the end repaired DNA.
Add 110 μl of Ampure beads to each sample, mix and rotate for 10 minutes at room temperature.
Place the samples on a magnetic separator, when the beads have collected to the wall of the tube and the solution is clear, remove and discard the liquid. The end repaired DNA is now bound to the beads.
Add 300 μl of 70% ethanol. Wash the beads by turning the tube 180° and allowing the beads to re-collect on the side of the tube. Turn the tube 6 times.
Remove and discard the ethanol.
Remove and discard the ethanol.
Repeat steps 7 and 8 two more times.
Remove the tubes from the magnetic separator, quick spin the beads, place back on the magnet and remove any remaining liquid. The quick spin will aid in drying the beads.
Keeping the tubes on the magnet and the caps open, dry the beads at room temperature for 20-30 minutes. Cracks will be observed in the bead pellet when drying is complete.
Add 30 μl of water to the dried beads and vortex to mix thoroughly.
Place the samples on a magnetic separator, when the beads have collected to the wall of the tube and the solution is clear, transfer the liquid to a fresh tube. The liquid contains your purified library.
5）QIAGEN MinElute® Gel Extraction Kit
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg or approximately 100 μl). For example, add 300 μl of Buffer QG to each 100 mg of gel. For >2% agarose gels, add 6 volumes of Buffer QG. The maximum amount of gel slice per spin column is 400 mg; for gel slices >400 mg use more than one MinElute column.
Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation. IMPORTANT: Solubilize agarose completely. For >2% gels, increase incubation time.
After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). Note: If the color of the mixture is orange or violet, add 10 μl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn to yellow. The adsorption of DNA to the membrane is efficient only at pH ≤7.5. Buffer QG contains a pH indicator which is yellow at pH ≤7.5 and orange or violet at higher pH, allowing easy determination of the optimal pH for DNA binding.
Add 1 gel volume of isopropanol to the sample and mix by inverting the tube several times. For example, if the agarose gel slice is 100 mg, add 100 μl isopropanol. Do not centrifuge the sample at this stage.
Place a MinElute column in a provided 2 ml collection tube in a suitable rack.
To bind DNA, apply the sample to the MinElute column, and centrifuge for 1 min. For maximum recovery, transfer all traces of sample to the column. The maximum volume of the column reservoir is 800 μl. For sample volumes of more than 800 μl, simply load and spin again.
Discard the flow-through and place the MinElute column back in the same collection tube.
Add 500 μl of Buffer QG to the spin column and centrifuge for 1 min.
Discard the flow-through and place the MinElute column back in the same collection tube.
To wash, add 750 μl of Buffer PE to the MinElute column and centrifuge for 1 min.
Discard the flow-through and centrifuge the MinElute column for an additional 1 min at ≥10,000 x g.
Place the MinElute column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 10 μl of Buffer EB (10 mM Tris·Cl, pH 8.5) or water to the center of the membrane, let the column stand for 1 min, and then centrifuge for 1 min.
If the purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel.
6) Enzyme digestion reaction and reaction of ligation between target gene and plasmid
1. Enzyme digestion
Incubate the tube at 37℃ for about 2hours, then cleanup according to the previous approaches.
2. Ligation
Mixed thoroughly according to above table, and incubate the tube at 16℃ over night. The ligation products were also cleaned according to the previous approaches.
7) Transformation of E.coli cells
1. Preparation of chemically competent E.coli cells
InocμLate 2ml LB broth with an aliquot (about 50μL)of the desired E.coli from the -80℃ freezer stock of cells.
Incubate for 2h at 37℃.
Add the 2ml seed cμLture to 250ml LB broth and grow at 37℃, shaking (about 200rpm) until OD600 of 0.3-0.4 (about 5 hours).
Pre-cool the 50ml polypropylene tube, 80 EP tubes, CaCl2-glycerine (0.1mol/L CaCl2) and CaCl2- MgCl2 (80mmol/L MgCl2, 20mmol/L CaCl2). Set the centrifuge and prepare the ice tray.
Transfer the bacteria into the 50ml polypropylene tube. Place it on ice for 10 minutes.
Centrifuge at 4℃, 4100rpm for 10 minutes.
Discard supernatant, then place the tube upside down to make sure trace liquid medium runs out.
Add 30ml of pre-cooled CaCl2- MgCl2 per 50ml of initial liquid medium to resuspend bacteria cell pellet.
Centrifuge at 4℃, 4100rpm for 10 minutes.
Discard supernatant then place the tube upside down to make sure trace liquid medium runs out.
Add 2ml of pre-cooled CaCl2 per 50ml of initial liquid medium to resuspend bacteria cell pellet.
Transfer to EP tubes (50μL every tube) and store at -80℃.
2. Transformation of E.coli DH-5α
Thaw competent cells rapidly by immersing frozen tubes in a 37℃ water bath after remove from 70℃ refrigerator. Draw about 50μL of the competent cells in a clean tube and add 5μL recombined plasmid, throw on ice for about 30 min. then put them in 42℃ for 90 second, immediately turn on ice for 2min. add 900μL LB medium and incubate in a roller drum at 37℃ fro 1hours. Took 100μL LB medium contain ampicillin, upside down when dried, put in 37℃ incubator over night. Validation by agarose electrophoresis after the plasmid DNA was extracted from transforming Escherichia coli.
8) Determination of the basic resistance of kanamycin
Kanamycin was prepared in water at 0µg/mL、5µg/mL、10µg/mL、15µg/mL、20µg/mL and stored at -20℃ in 10-µL aliquots. An aliquot was thawed as needed and used once without refreezing.
Equivalently draw the cyanobacterium at logarithmic growth stage and inoculate in BG-11 liquid medium containing kanamycin with all above concentration respectively, the cyanobacterium solution was cultured under the optimum conditions for one week and observation of growth status was carried out.
It is determined that the cell of cyanobacterium cannot resist certain concentration of kanamycin if no algae community were found after one week of culture. Basic kanamycin resistance test of Microcystis aeruginosa shows that the algae cannot resist concentration of 5μg/mL and above on BG11. Therefore, we choose 10-15μg/mL kanamycin for screening after shuttle plasmid pPKE2 containing modified GvpA1 gene is transformed into microcysitis aeruginosa.
9) Triparental matting of P.putida KT2440
Inoculated with pET-30a, PRK2013, and the donor LB contained 10mL (Amp, Km, Gm) in liquid culture 200 rpm overnight culture was maintained at 37℃in the nutrient medium.
Taken 300µL donor bacteria and 600µL cells, 600µL receptor bacteria liquid and bacterial liquid in 1.5mL centrifuge tube, centrifugalization at 7000 rpm for 10min to collect bacteria.
The supernatant was washed out, and the cells were cleaned with sterile water. The cells were collected by centrifugation at the same condition as 10min.
Repeat step 3 once.
with 500µL sterile water suspension cell, draw 100µL bacterial suspension in 10mL liquid medium without antibiotics LB. 37℃, cultured for 24-36 hours under 200 rpm.
Prepare double resistance plate, dissolve the solid medium prepared in microwave oven, cool to the non hot temperature, to 100mL. The resistance of receptor medium added solid culture (Amp) antibiotics and antibiotics resistance of donor strain (Gm), shake well, poured into the Petri dish out good bacteria, (about 5 per 100mL culture dish).
From steps 5 in bacteria from 100µL bacteria, in the middle of double plate coated with anti, burned after cooling rods. Uniform coating. Placed around 10min, inverted in 37℃ incubator cultured overnight.
The blue bacteria on the plate were lagged behind, and single colonies were picked out by inoculating rings to another plate containing double resistance Observe its stability effect.
Note: each tablet is marked with antibiotics, the time of plate treatment, and the number of recipient bacteria. During the specified period of time, the culture medium was dried and the bacteria died.
10) Sample preparation for SDS-PAGE
5mL of induced products was centrifuged with 10000rpm for 10 min under 4℃, collecting the cells.
Resuspend the precipitate with 2mL 0.5M Tris-HCl (pH 6.8), then centrifuged with 10000rpm for 5 min, repeat this step twice.
Resuspend the precipitate with 5mL buffer for μLtrasonication under 200W, 1second, with 2 second interval, 30 cycles, total times with 6 min. then centrifuged with 12000rpm for 25 min, collected the supernatant and precipitation respectively.
Resuspend the precipitate with 80µL ddw, add 20µL 5×loading buffer and 5µL DTT, mixed thoroughly. Prepare another tube with the supernatant with the same procedure. Then boiling in 100℃ for 6min for SDS-PAGE detection.
Procedure of PAGE
Preparation of the separation and concentration gels
AP and TEMED shoμLd be the last to added. When it was added, mixed thoroughly immediately, and after the gel polymerization, pμLl out the comb and wash the sample hole with double distilled water.
Load 5μL samples, Start electrophoresis under 80V until the bromophenol blue moving into separation gel, then increase the voltage to 100V.
After electrophoresis, strip the gel and immerge in the fixed solution until the bromophenol blue change to faint yellow; replace the fixed solution with staining solution, then dyeing and decolorizing for 4 hours.
Removing the staining solution, following another decolonization step, it coμLd be watch and took photo after the band of protein was complete clear. The gel coμLd be stored in distilled water.
11) Western Blotting
Apparatus: Apparatus of SDS-PAGE, Electroblotting Apparatus, Power supply, PVDF membrane（Millipore Immobion-P #IPVH 000 10）, Whatman 3MM paper, Additional Tools: Forceps, sponge pad, scissor, gloves, small plastic or glass container, Shallow tray.
Cell extracts were prepared by homogenizing cells or tissues in the lysis buffer (50 mM Tris–HCl, pH 7.5, 5 mM EDTA, 150 mM NaCl, 0.5% NP-40) for 45 min. The soluble protein concentration was measured using Bio-Rad Protein Assay (Bio-Rad). The lysates (50 lg) were separated by 12% sodium dodecyl sμLfate (SDS)-polyacrylamide gel (SDS–PAGE) and transferred to PVDF membranes for immunoblotting assay. The membrane was blocked in 5% fat-free dry milk, and probed with antibodies against the interest proteins. The blots were visualized using the enhanced ECL reagent. The levels of protein expression were semi-quantified by optical densitometry using Image J software.
12）RNA Extraction
Use Trizol reagent Kit according to user manual. then treated with DNase, quality control by electrophoresis.
13) Reverse transcription
Firstly, reverse transcription with 6mer random primer and poly T primer according to the following table:
Thermal cycle parameter as follows: 65℃ for 5min, put on ice immediately; then add 5×buffer，RNase Inhibitor and RTase, finally add ddw to 20uL, 30℃ for 10min, following 42℃for 1hours, 70℃ for 10min. the products stored at -20℃ or using for qPCR immediately.
14）Fluorogenic quantitative PCR
qPCR was performed according to the user manual, 16s rRNA was used as internal control. Preparation of the reaction system according to the following table:
Place in ABI 7500 and run the following cycles: 95℃for 30s, 60℃ for 34s, 40cycles, 72℃ 8min.
Lib Pictures
Home
Project
Overview
Design
Result
Verification
Safety
Teams
Members
Instructors
Attributions
Collaborations
Human Practice
Notebook
Protocol
Lab pictures
Parts
Contact us:
Email:  NFLS_iGEM@126.com
Mobile: +86 13913850670
Address: 30 East Beijing Road, Nanjing,
Jiangsu, China
