<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Design_and_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Design and Background</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Research </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Project_Description">Project description</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Design_and_Background">Design and background</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Lab_summary">Lab summary</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/labbook">Lab book</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Demonstrate">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Outlook">Outlook</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Safety">Lab Safety</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/InterLab">Interlab study</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Measurement">Measurement</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/HP/Silver">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Handbook">Handbook</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Policy">Policy</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Entrepreneurship </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Entrepreneurship">Product Development</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Applied_Design">Applied Design</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Achievements">Achievements</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/About_Us">About Us</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Supervisors_and_Advisors">Supervisors and Advisors</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Collaborations">iGEM Team Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Other_Collaborations">Other Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/SGEM">SGEM</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Future">Future</A></LI></DIV></DIV><P><SECTION class="igem-prolung-promo"><VIDEO id="igem-prolung-video" poster="img/head.jpg" playsinline="" autoplay="" muted="" loop=""><SOURCE src="https://static.igem.org/mediawiki/2017/d/dc/Stockholm_mucus.mp4" type="video/mp4">
              Your browser does not support video.
            </SOURCE></VIDEO></SECTION></P><DIV class="container igem-prolung-logo"><DIV class="logo-slogan"><DIV id="slogan">A mucus degrading lung probiotic</DIV></DIV></DIV><DIV id="wrapper"><DIV class="igem-content"><H1>Design and Background</H1></DIV><SECTION class="container igem-content-section"><DIV class="igem-content"><H2>Biocontainment</H2><H3>Why do we need a biocontainment system?</H3><P class="igem-text">
One of the most important issues of creating genetically modified organisms is their behaviour outside of the controlled space. Unintended cellular proliferation and dissemination of transgenic material are long-standing concerns for scientists and society. In many cases, it is hard or even impossible to predict how a synthetic organism will respond to stimuli from the environment. Proper risk assessments of the GMO should always precede its release into the market to evaluate its impact and the worst-case scenarios. Nowadays, several efforts have been made to reduce those risks, yet they are far from being perfect at foreseeing every possible event (Mandell et al., 2015).
    </P><P class="igem-text">
PROlung holds a lot of promise for a great number of people affected by several respiratory diseases. However, considering that it is a GMO intended to be introduced into the human body, it poses a risk to an individual, as well as an environmental level. Even though the general concept of probiotics and their beneficial effects has been around for decades, complications have still been linked to the use of bacteria (Sanders et al., 2010). Subsequently, one has to consider the inherent risks of the transgenic material of a GMO, as well as its pathogenicity.
    </P><P class="igem-text">
Our team has received inputs from our advisors regarding the technicalities of such a system, as well as the ethical aspects of releasing genetic material outside of the lab. Therefore, we aim to develop a multilayered strategy for a well-regulated biocontainment system, attempting to minimize the negative impact on the environment and the human health.
    </P></DIV></SECTION><SECTION class="container igem-content-section"><DIV class="igem-content"><H3>How will our biocontainment system work?</H3><P class="igem-text">
There is a broad range of biocontainment systems designed with the sole purpose of ensuring that the spreading of GMOs is as limited as possible (Mandell et al., 2015). The use of a toxin-antitoxin regulation mechanism is a prime example of such a system (Li and Wu, 2009), with several iGEM projects already having attempted to use and characterize these systems, for instance <A target="_blank" href="https://2013.igem.org/Team:Uppsala/LactonutritiousWorld">Lactonutritious</A> and <A target="_blank" href="https://2015.igem.org/Team:BABS_UNSW_Australia/endosymbionts">Endosymbiont</A>. Toxin expression or action is hindered by its cognate antitoxin, resulting in a controlled system which could be used to eliminate the bacteria when it is not needed, hence the name kill-switch.
    </P><P class="igem-text">
Our biocontainment system is based on two principles: genome integration and toxin-antitoxin gene switches. It is a two-component feedback-regulated kill-switch mechanism, with toxin expression induced by the addition of two chemical substances - cumate and tryptophan. Additionally, to minimize spreading of transgenic material, we added the option of integrating the system into a specific site in the genome.
    </P><H4>What is genome integration and why is it needed?</H4><P class="igem-text">
The first issue we addressed was horizontal gene transfer (HGT), which is a mechanism used by bacteria for an exchange of genetic material (Keese, 2008). To reduce spreading of the transgenes encoding our system to native lung bacteria, integration into the bacterial chromosome was employed as a first-in-line strategy against HGT, specifically the mechanism of bacterial conjugation. 
    </P><P class="igem-text">
Looking into the available methods for genome integration described in the literature and in the iGEM community, we took advantage of the characterized <A target="_blank" href="https://2011.igem.org/Team:UPO-Sevilla/Foundational_Advances/MiniTn7/Overview">mini-Tn7 BioBrick toolkit</A> developed by the <A target="_blank" href="https://2011.igem.org/Team:UPO-Sevilla">iGEM team UPO-Sevilla 2011</A> and previously described by Choi et al., 2005. In brief, the toolkit contains two suicide delivery vectors (medium and high copy number plasmids) in which one could insert any BioBrick between two distinct sequences (Tn7L and Tn7R) and integrate it into the genome at a designated site, attTn7. The attTn7 site is highly conserved in nearly all bacterial genomes (Milewski, 2002). The integration is executed by a transposase complex expressed in another ‘’helper’’ suicide plasmid (<A target="_blank" href="https://www.addgene.org/64968/">pTNS2</A>), containing a set of enzymes carrying out the transposition. The transposase complex recognizes the Tn7L and Tn7R sites, flanking the sequence of interest and finally inserts it into the unique conserved site in the genome (attTn7).
</P><DIV class="igem-legend">Figure 1. Delivery plasmid map and integration into the conserved attTn7 site using a transposase complex.</DIV><H3>How will our kill-switch work?</H3><P class="igem-text">
At the same time, we came to realize that any selected system would have intrinsic disadvantages which may compromise our kill-switch. The downside of introducing toxins into bacteria is the selective pressure to improve fitness through deactivation of the toxic gene, as well as leaky expression of the toxin, which could hinder the effectiveness of the biocontainment system (Pasotti et al, 2011).
    </P><P class="igem-text">
Therefore, we searched for potential well-described gene switches where we could employ a toxin-antitoxin system. We decided to use two switches instead of one to ensure improved regulation and advantageous GMO containment. 
    </P><P class="igem-text">
More specifically, we built upon the cumate gene switch of the <A target="_blank" href="https://2012.igem.org/Team:Trieste">iGEM Trieste 2012</A> team, previously developed by Mullick et al. 2006. It is a tightly controlled inducible system where we aim to insert a toxin, Colicin E2, previously used in many iGEM projects, such as <A target="_blank" href="https://2016.igem.org/Team:TU_Darmstadt">iGEM TU Darmstadt 2016</A> and <A target="_blank" href="https://2012.igem.org/Team:Paris_Bettencourt">iGEM Paris Bettencourt 2012</A>. Although leaky expression is minimized by the tight regulation, we believe that expression of a DNA endonuclease might still pose a threat. 
    </P><P class="igem-text">
Thereby, we aim to add another layer of protection by controlling the expression of its cognate Immunity protein 2 (Im2) by a Trp operon and its negative repressible feedback mechanism. The tryptophan operon is a well-known gene switch regulated by the levels of tryptophan and a tryptophan repressor (Rose et al, 1973). 
</P><P class="igem-text">
Overall, our goal is to prove that the inherent weaknesses of the cumate switch can be compensated by the tryptophan switch and vice versa.
    </P><H3>How is our kill-switch regulated?</H3><P class="igem-text">
Our biocontainment kill-switch is divided into two parts: the cumate regulatory system and the tryptophan regulatory system. Their names refer to the compound which is used to induce toxin expression by deactivating the safety switches.
    </P><H4>Cumate system</H4><P class="igem-text">
The repressor of the system (<A target="_blank" href="http://parts.igem.org/Part:BBa_K415202">CymR</A>) is constitutively expressed and is bound to the <A target="_blank" href="http://partsregistry.org/Part:BBa_K875001">T5 cumate operon</A>, repressing the expression of the downstream gene. Once cumate is added it binds to CymR, while the latter dissociates from the T5 cumate operator, enablng the expression of the downstream gene. It should be noted that we have added two reporter proteins, <A target="_blank" href="http://parts.igem.org/Part:BBa_K1368011">RNA spinach aptamer</A> and blue fluorescent protein (<A target="_blank" href="http://parts.igem.org/Part:BBa_K592100">BFP</A>). Moreover, we have introduced two compatible multiple cloning sites (MCS), surrounding BFP, facilitating the subcloning of a gene of interest in its place. Our aim is to characterize a fully functional composite BioBrick part which can be used by any iGEM team by simply introducing and having upon chemical regulation their gene of interest.  
    </P><DIV class="igem-legend">Figure 2. Cumate system with BFP before and after induction with cumate.</DIV><P class="igem-text">
In our project, we are planning to remove BFP and introduce the toxin <A target="_blank" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1976048">Colicin E2</A> once the system is adequately characterized. At last, we have added an additional MCS between the two gene regulation units in order to enable further subcloning. 
    </P><H4>Tryptophan system</H4><P class="igem-text"><A target="_blank" href="http://parts.igem.org/Part:BBa_K588001">Tryptophan operon</A>, comprised of tryptophan promoter and operator, is regulated by a constitutively expressed tryptophan repressor (<A target="_blank" href="http://parts.igem.org/Part:BBa_K588000">TrpR</A>). However, the TrpR alone is unable to bind to the operator allowing expression of the downstream gene. Only upon addition of tryptophan does the TrpR associate with it and bind to the operator, disabling further gene expression. We placed a red fluorescent protein (RFP) downstream of the tryptophan operon as a reporter gene for characterization of the system. As in the cumate system, we have introduced MCS before and after the RFP sequence, facilitating further subcloning steps. Again, this composite BioBrick could potentially incorporate any gene of interest for future iGEM teams who  are interested in a chemical gene switch.
    </P><DIV class="igem-legend">Figure 3. Tryptophan system with RFP before and after induction with tryptophan.</DIV><P class="igem-text">
Our gene of interest is Im2, which would potentially be introduced in the place of RFP. Therefore, our goal is to develop a system in which the antitoxin will be expressed in the absence of tryptophan.
    </P><H4>What is our ultimate goal?</H4><P class="igem-text">
The main objective is to fuse the two composites into one large BioBrick part and clone them into our workhorse (TOP10, BL21). Furthermore, we intend to integrate the whole system into the bacterial genome via the mini-Tn7 toolkit. It is imperative to determine whether the expression of the toxin is well-regulated in the absence of tryptophan and cumate, whilst the presence of the two chemicals induces cell death. 
    </P><H4>What are our limitations?</H4><P class="igem-text">
We should mention that our design is not perfect and there are intrinsic disadvantages which we cannot avoid and may compromise our system such as:
    </P><UL class="igem-general-list"><LI>Selective pressure to improve fitness</LI><LI>Leaky expression of toxin</LI><LI>Other mechanisms of HGT: transduction, gene transferring agents</LI><LI>Mutagenic drift</LI></UL></DIV></SECTION><SECTION class="container igem-content-section"><DIV class="igem-content"><H3>What additional steps did we perform?</H3><P class="igem-text">
We have also considered the possibility of our lung probiotic being exhaled or ingested. To avoid any incident of spreading, the toxin can be inserted downstream of different promoters that are activated by changes in physiological conditions, such as temperature sensitive and pH-responsive promoters. Subsequently, the expression of toxin would then be induced by the acidic conditions of the stomach if ingested, or by room temperature if exhaled, effectively eliminating the released bacteria. 
    </P><H4>pH-sensitive promoter</H4><P class="igem-text">
We aim to employ the AsR promoter (<A target="_blank" href="http://parts.igem.org/Part:BBa_K1231000">BBa_K1231000</A>), which activates transcription in acidic conditions (~pH 5.5), making it suitable for the kill-switch (Sužiedėlienė et al, 1999). By introducing the colicin E2 downstream of this BioBrick, we can determine whether acidic pH (~ 5.5) induces the expression of the toxin and subsequently cell death.
    </P><DIV class="igem-legend">Figure 4. Activity of AsR promoter and expression of colicin in pH 5.5.</DIV></DIV></SECTION><SECTION class="container igem-content-section"><DIV class="igem-content"><H3>Collaboration - modelling</H3><P class="igem-text">
We came in contact with <A href="https://2017.igem.org/Team:NUS_Singapore">iGEM NUS 2017</A> to collaborate and employ their characterized temperature sensitive promoter (Piraner et al, 2016). 
    </P><P class="igem-text">
Lastly, NUS 2017 also offered their modelling expertise as part of our collaboration and they have modelled how the bacteria would react in defined environments. Specifically, they  modelled our two gene switches (cumate and tryptophan) expressing the toxin and antitoxin, respectively. The main objective is to determine whether the two systems would work as expected upon chemical addition.
    </P></DIV></SECTION><SECTION class="container igem-content-section"><DIV class="igem-content"><H2>References</H2><UL class="igem-references"><LI>Choi, K., Gaynor, J., White, K., Lopez, C., Bosio, C., Karkhoff-Schweizer, R. and Schweizer, H. (2005). A Tn7-based broad-range bacterial cloning and expression system. Nature Methods, 2(6), pp.443-448.</LI><LI>Keese, P. (2008). Risks from GMOs due to Horizontal Gene Transfer. Environmental Biosafety Research, 7(3), pp.151-151.</LI><LI>Li, Q. and Wu, Y. (2009). A fluorescent, genetically engineered microorganism that degrades organophosphates and commits suicide when required. Applied Microbiology and Biotechnology, 82(4), pp.749-756.</LI><LI>Mandell, D., Lajoie, M., Mee, M., Takeuchi, R., Kuznetsov, G., Norville, J., Gregg, C., Stoddard, B. and Church, G. (2015). Biocontainment of genetically modified organisms by synthetic protein design. Nature, 518(7537), pp.55-60.</LI><LI>Milewski, S. (2002). Glucosamine-6-phosphate synthase—the multi-facets enzyme. Biochimica et Biophysica Acta (BBA) - Protein Structure and Molecular Enzymology, 1597(2), pp.173-192.</LI><LI>Mullick, A., Xu, Y., Warren, R., Koutroumanis, M., Guilbault, C., Broussau, S., Malenfant, F., Bourget, L., Lamoureux, L., Lo, R., Caron, A., Pilotte, A. and Massie, B. (2006). The cumate gene-switch: a system for regulated expression in mammalian cells. BMC Biotechnology, 6(43).</LI><LI>Pasotti, L., Zucca, S., Lupotto, M., Cusella De Angelis, M. and Magni, P. (2011). Characterization of a synthetic bacterial self-destruction device for programmed cell death and for recombinant proteins release. Journal of Biological Engineering, 5(1), p.8.
</LI><LI>Piraner, D., Abedi, M., Moser, B., Lee-Gosselin, A. and Shapiro, M. (2016). Tunable thermal bioswitches for in vivo control of microbial therapeutics. Nature Chemical Biology, 13(1), pp.75-80.</LI><LI>Rose, J., Squires, C., Yanofsky, C., Yang, H. and Zubay, G. (1973). Regulation of in vitro Transcription of the Tryptophan Operon by Purified RNA Polymerase in the Presence of Partially Purified Repressor and Tryptophan. Nature New Biology, 245(144), pp.133-137.</LI><LI>Sanders, M., Akkermans, L., Haller, D., Hammerman, C., Heimbach, J., Hörmannsperger, G. and Huys, G. (2010). Safety assessment of probiotics for human use. Gut Microbes, 1(3), pp.164-185.</LI><LI>Sužiedėlienė, E., Sužiedėlis, K., Garbenčiūtė, V., &amp; Normark, S. (1999). The Acid-Inducible asr Gene in Escherichia coli: Transcriptional Control by the phoBR Operon. Journal of Bacteriology, 181(7), 2084–2093.</LI></UL></DIV></SECTION><SECTION class="container igem-content-section"><DIV class="igem-content"><H2 id="sensing">Osmotic pressure sensing</H2><H3>Why do we want to sense changes in osmotic pressure?</H3><P class="igem-text">
Patients suffering from respiratory diseases such as CF, asthma and COPD have one thing in common; accumulation of excessively thick mucus. When the mucus is thickened, the cilia covering the lung surface have difficulties removing dirt and pathogens from the airways (Hendersen et al, 2014). This results in more frequent infections and causes breathing difficulties. The thick mucus has a higher mucin content, ie. more mucin molecules, than that of mucus in healthy individuals. More mucin molecules mean that there is a higher total number of solutes in the mucus, resulting in increased osmotic pressure in the mucus.
    </P><P class="igem-text">
Therefore, our goal is to trigger the expression of our mucus-degrading enzymes after sensing this increase in osmotic pressure, helping patients in clearing the airways.
    </P><H3>How is this possible?</H3><P class="igem-text">
We will take advantage of the already well characterized OmpR/EnvZ two-component system used by many bacteria, including <I>E. coli</I> (figure 1) (Pratt et al. 1996). The system’s function is to keep the osmotic pressure stable inside of the bacteria by responding to changes in osmotic pressure and regulating the in and outflow of molecules with the help of two membrane porins, OmpC and OmpF. The expression of these membrane porins is regulated by the OmpR/EnvZ system. EnvZ is a histidine kinase, able to sense the external osmotic pressure and communicate this information to the OmpR protein, by phosphorylation or dephosphorylation. When the osmotic pressure is high, the phosphorylated OmpR-P will inhibit the expression of OmpF and simultaneously bind to the promoter upstream of OmpC, increasing the expression of this membrane porin and hence stabilizing the osmotic pressure due to increased inflow of solutes.
    </P><DIV class="igem-legend"><STRONG>Figure 1.</STRONG> Mechanism of promoter activation upon increased osmotic pressure.
</DIV><P class="igem-text">
By replacing the OmpC gene with the gene for our mucus-degrading enzyme, we take advantage of this OmpR responsive promoter and introduce a system capable of self-regulating its enzyme expression. Consequently, the higher the osmotic pressure (the thicker the mucus), the more mucus-degrading enzymes will be expressed.
    </P><H3>How do we know that the activity of the OmpR promoter is truly dependent on changes in osmotic pressure?</H3><P class="igem-text">
iGEM Stockholm 2015 characterized the OmpR responsive promoter with RFP (BioBrick BBa_M30011). By cultivating the transformed bacteria in a sucrose gradient, ranging from 5% to 15%, and thereafter measuring the fluorescence of the RFP, they observed increasing fluorescence with increasing sucrose concentration. To prove that the increased activity of the OmpR promoter was truly dependent on the OmpR/EnvZ system, they performed the same experiment in an EnvZ deficient strain. They found a significant difference in fluorescence and could, therefore, conclude that the increased fluorescence was due to increased osmotic pressure and not due to more available nutrients.
    </P><P class="igem-text">
The promoter was also characterized by iGEM UCL 2015. However, instead of testing it in a sucrose gradient, they performed their activity test in a salt gradient ranging from 0.05% to 0.4% NaCl. They reached the conclusion that the NaCl concentrations were too high for the bacteria to survive as they did not retrieve any conclusive results regarding the activity of the promoter. 
    </P><H3>How will we test our system?</H3><P class="igem-text">
To demonstrate that the activity of the promoter is increased when the osmotic pressure rises, we will use the same BioBrick, OmpR responsive promoter with downstream RFP, and protocol as iGEM Stockholm 2015. We will also test the system in a salt gradient, ranging from 0.00625% to 0.1% NaCl to elaborate on the work done by iGEM UCL 2015. By using a sucrose and salt gradient to represent the increase in osmotic pressure, we will try to mimic the environment in a lung with an accumulation of excessively thick mucus.
    </P><H3>What will we contribute with?</H3><P class="igem-text">
Apart from confirming the work done by iGEM Stockholm 2015, we will also elaborate on the work by testing the activity at more bacterial concentrations. While 2015 proved that the activity of the promoter is completely EnvZ dependent and that expression of the downstream gene is increased at rising osmotic pressure, they only tested this at one OD measurement (OD 0.3). We will further characterize the BioBrick by investigating the activity throughout the growth curve. In addition to using the same sucrose gradient as iGEM Stockholm 2015, we will also use a salt gradient with lower concentrations of NaCl, investigating if this will improve the results of iGEM UCL 2015. 
    </P></DIV></SECTION><SECTION class="container igem-content-section"><DIV class="igem-content"><H2>References</H2><UL class="igem-references"><LI>Henderson, A. G., Ehre, C., Button, B., Abdullah, L. H., Cai, L.-H., Leigh, M. W.,  DeMaria, G., Matsui, H., Donaldson, S., Davis, W., Sheehan, J., Boucher, R., Kesimer, M. (2014). Cystic fibrosis airway secretions exhibit mucin hyper-concentration and increased osmotic pressure. The Journal of Clinical Investigation, 124(7), p. 3047–3060. </LI><LI>Pratt, L., Hsing, W., Gibson, K., Silhavy, T., (1996), From acids to osmZ multiple factors influence synthesis of the OmpF and OmpC porins in Escherichia, Molecular Biology 20(5), p. 911-917</LI></UL></DIV></SECTION><SECTION class="container igem-content-section"><DIV class="igem-content"><H2 id="degradation">Mucus degrading enzymes</H2><H3>What is mucus and mucin?</H3><P class="igem-text">
Mucus is a mixture consisting of 90-95% water, 5-10% molecules called mucins, and a remaining 1% of lipids, proteins and salts. Among the constituents, mucins belong to a class of large, heterogeneous, and heavily glycosylated proteins (figure 1). The protein backbone of a mucin is covered with chains of O-linked glycans, which are polysaccharides bound by an oxygen to the protein. The mucins’ glycans prevents protein-protein interaction, forcing the backbone to extend itself. The extended conformation of the mucin enables a larger contact area for glycans to bind water, enabling the mucus to form a gel with both viscous and elastic properties. Removing glycans or part of the glycan could induce protein aggregation of the mucin backbone and thus loss of gelatinous attributes.
    </P><P class="igem-text">
The sugar groups of the mucins often differ, making them heterogeneous constructs. The majority of the mucins found in the respiratory tract contain a large amount of sialic acid, a negatively charged monosaccharide. Sialic acids are located at the end of the mucins’ saccharide chains (figure 1) and give the mucin an overall negative surface charge, which increases the stability of the mucin through charge repulsion (Zanin, 2016). 
    </P><DIV class="igem-legend"><STRONG>Figure 1.</STRONG> Mucin schematic representation.
</DIV><H3>How will we degrade the mucus?</H3><P class="igem-text">
With the ultimate goal of degrading the mucins using a genetically engineered bacteria, a literature search was conducted, aiming to identify mucin degrading enzymes successfully expressed in <I>E. coli</I> by other researchers. We mainly researched enzymes that degrade the saccharides rather than the protein backbone. Considering our project aims to expose these enzymes to the human body, we decided to avoid proteases. Our objective was a controlled bacterial system that will only degrade the mucins and not harm the lungs. Several relevant enzymes were found in literature, but after the first trial experiments it was decided to further research two of them.
    </P><P class="igem-text">
Firstly, the enzyme sialidase (EC 3.2.1.18) from <I>Arthrobacter ureafaciens</I> (Egebjerg, 2005). This enzyme hydrolyses glycosidic linkages of terminal sialic acid residues in glycoproteins (figure 2). In other words, it removes the sialic acids from the ends of the mucin constructs. Removal of sialic acids results in an exposure of other sugar groups on the mucins to other enzymes. Thus, the removal of sialic acids may result in even further mucus degradation by bacteria using different enzymes (Derrien, 2010).
    </P><DIV class="igem-legend"><STRONG>Figure 2.</STRONG> Sialidase reaction mechanism.

    <P class="igem-text">
Secondly, we studied a specific Endo-β-galactosidase (EBG) from <I>Clostridium perfringens</I>. This enzyme has been shown, in literature, to release saccharide chains from glycans expressed in gastric mucus (Ashida, 2002). Therefore, we assumed it would behave similarly with the lungs’ mucus.
    </P><P class="igem-text">
EBG releases the saccharides by hydrolysing the bonds next to galactose in the polysaccharide chains of mucins (figure 3). This enzyme interacts only with saccharides that do not contain sialic acids. Mucins contain such glycans, but a combination of both enzymes is believed to result in mucin degradation.
    </P><DIV class="igem-legend"><STRONG>Figure 2.</STRONG> Endo-β-galactosidase reaction mechanism.
</DIV><P class="igem-text">
To summarize, applying one of the enzymes or a combination of both, is believed to degrade different sugar groups on the mucins, consequently decreasing the mucins’ ability to bind water, and thus making the mucus less viscous.
    </P><H3>How will we measure the degradation?</H3><P class="igem-text">
Firstly, to measure the rate of degradation, a method called high-performance anion-exchange chromatography was used. Using this method, we were able to quantify the amount of saccharides degraded from mucin samples. 
    </P><P class="igem-text">
Secondly, to confirm that the degradation of sugar groups really adjusts the visco-elastic properties of the mucus, we used the method rheology: the study of the flow of matter. By applying sialidase or EBG, or both, onto a mucus sample, we hope to prove that our enzymes are capable of making the mucus less viscous. Several studies have shown that small differences in the mucin concentration can affect the mucus viscoelasticity tremendously (e.g. 2-4 fold increase in mucin concentration can result in 100-fold higher viscosity) (Lai et al., 2009). In cystic fibrosis for instance, a disease with an abnormally high mucin concentration, the viscosity at low shear rates (close to standstill) can be compared to that of rubber  (Lai et al., 2009) (Mestecky et al., 2015).  
    </P><H3>How will we secrete our enzymes?</H3><P class="igem-text">
To achieve the goal of degrading mucus in the lung, the engineered bacteria, in this case <I>E. coli</I>, should be able to secrete the enzymes out of the cell. <I>E. coli</I> is not natively equipped with the proper machinery to secrete large enzymes. Therefore, the possibility of making <I>E. 0coli</I> secrete the desired enzymes was studied. 
    </P><P class="igem-text">
We chose to work with a type I secretion system, which is also known as the ABC transporter (figure 4). The system works in a continuous secretion process across both the inner and the outer membrane of Gram-negative bacteria. The proteins involved in type I secretion system form a channel that exports proteins from the cytoplasm to the extracellular environment (Green, 2016). Among all of the type I secretion systems, the α-hemolysin(HlyA) secretion system is the best characterized, studied and most used. Also, the type I secretion system is able to transport various molecules, from ions, drugs, to proteins of various sizes (20-900 kDa). Because of these two reasons, we chose to use the α-hemolysin(HlyA) secretion system for the secretion of our enzymes (≈50 kDa).
    </P><DIV class="igem-legend"><STRONG>Figure 4.</STRONG> ABC transporter, secretion system schematic representation.

    <P class="igem-text">
The system contains 4 parts: the HlyA, HlyB, HlyD and TolC respectively. HlyA is a α-hemolysin, which contains a C-terminal signal sequence and is attached to the desired enzyme. The attachment of HlyA contributes to the recognition by HlyB, an ATP-binding cassette. HlyD is a membrane fusion protein linking between the outer and the inner membrane. TolC is a specific outer membrane protein, which forms a long channel throughout the outer membrane and the periplasm, largely open towards the extracellular medium.
    </P></DIV></DIV></DIV></SECTION><SECTION class="container igem-content-section"><DIV class="igem-content"><H2>References</H2><UL class="igem-references"><LI>Ashida, H., Maskos, K., Li, S.-C., &amp; Li, Y.-T. (2002). Characterization of a Novel Endo-β-galactosidase Specific for Releasing the Disaccharide GlcNAcα1→4Gal from Glycoconjugates†,‡. Biochemistry, 41(7), 2388–2395. http://doi.org/10.1021/bi011940e</LI><LI>Derrien, M., Passel, M. W. V., Bovenkamp, J. H. V. D., Schipper, R., Vos, W. D., &amp; Dekker, J. (2010). Mucin-bacterial interactions in the human oral cavity and digestive tract. Gut Microbes, 1(4), 254–268. http://doi.org/10.4161/gmic.1.4.12778</LI><LI>Egebjerg, J., &amp; Christensen, S. (2005). Cloning, expression and characterization of a sialidase gene from Arthrobacter ureafaciens. Biotechnology and Applied Biochemistry, 41(3), 225. http://doi.org/10.1042/ba20040144</LI><LI>Green, E. R., &amp; Mecsas, J. Bacterial Secretion Systems: An Overview. Virulence Mechanisms of Bacterial Pathogens, Fifth Edition, 215–239. http://doi.org/10.1128/microbiolspec.vmbf-0012-2015</LI><LI>Lai, S., Wang, Y., Wirtz, D. and Hanes, J. (2009). Micro- and macrorheology of mucus. Advanced Drug Delivery Reviews, 61(2), pp.86-100.</LI><LI>Mestecky, J., Strober, W., Russell, M., Cheroutre, H., Lambrecht, B. and Kelsall, B. (2015). Mucosal Immunology. 4th ed. Elsevier.</LI><LI>Zanin, M., Baviskar, P., Webster, R., &amp; Webby, R. (2016, October). The interaction between respiratory pathogens and mucus. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4752725/#R77</LI></UL></DIV></SECTION></DIV><DIV class="container"><DIV class="igem-sponsors igem-sponsors-first"><H1> Sponsors </H1></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>