<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Research </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Project_Description">Project description</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Design_and_Background">Design and background</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Lab_summary">Lab summary</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/labbook">Lab book</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Demonstrate">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Outlook">Outlook</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Safety">Lab Safety</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/InterLab">Interlab study</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Measurement">Measurement</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/HP/Silver">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Handbook">Handbook</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Policy">Policy</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Entrepreneurship </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Entrepreneurship">Product Development</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Applied_Design">Applied Design</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Achievements">Achievements</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/About_Us">About Us</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Supervisors_and_Advisors">Supervisors and Advisors</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Collaborations">iGEM Team Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Other_Collaborations">Other Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/SGEM">SGEM</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Future">Future</A></LI></DIV></DIV><P><SECTION class="igem-prolung-promo"><VIDEO id="igem-prolung-video" poster="img/head.jpg" playsinline="" autoplay="" muted="" loop=""><SOURCE src="https://static.igem.org/mediawiki/2017/d/dc/Stockholm_mucus.mp4" type="video/mp4">
              Your browser does not support video.
            </SOURCE></VIDEO></SECTION></P><DIV class="container igem-prolung-logo"><DIV class="logo-slogan"><DIV id="slogan">A mucus degrading lung probiotic</DIV></DIV></DIV><DIV id="wrapper"><DIV class="igem-content"><H1>Results</H1><H2>Biocontainment</H2><H3>Cumate regulatory system</H3><TABLE style="width:60%"><TBODY><TR><TH>No</TH><TH>Hypothesis</TH><TH>Status</TH></TR><TR><TH>1</TH><TH>Cloning of cumate regulatory system gBlock into pSB1C3</TH><TH>Positive</TH></TR><TR><TH>2</TH><TH>Bacterial proliferation of cumate regulatory system with BFP in medium containing cumate in a concentration gradient</TH><TH>Positive</TH></TR><TR><TH>3</TH><TH>Measuring BFP fluorescence of cumate regulatory system in medium containing cumate in a concentration gradient</TH><TH>Inconclusive</TH></TR><TR><TH>4</TH><TH>Cloning of colicin into pSB4A5</TH><TH>Negative</TH></TR><TR><TH>5</TH><TH>Subcloning of colicin into cumate regulatory system</TH><TH>Positive</TH></TR><TR><TH>6</TH><TH>Bacterial proliferation of cumate regulatory system with colicin in medium containing cumate in a concentration gradient</TH><TH>Not started</TH></TR></TBODY></TABLE><H4>Hypothesis 1: Cloning of cumate regulatory system gBlock into pSB1C3</H4><P class="igem-text">
The cumate regulatory system gBlock was amplified via PCR, digested with EcoRI and PstI, and ligated into the pSB1C3 iGEM backbone. The colony PCR performed on the transformed bacteria shows a band matching the size of the fragment (1909 bp). The purified plasmid was sent for sequencing. The results showed a mutation from a hydrophilic amino acid codon (R for arginine) to a hydrophobic one (L for leucine) in the sequence of the CymR regulatory protein; at position 498 of the sequence the nucleotide has mutated from G to T. The mutation appears not to affect the DNA binding site of the repressor according to protein databases. However, no information regarding the binding site of p-cumate was found.
</P><P class="igem-text">
The successful cloning was confirmed with colony PCR using prefix/suffix primers showing the amplification of an insert of the appropriate size, sequencing of the isolated plasmids from said culture (sequence contains a mutation from R to L, which does not affect the DNA-binding site, but it may affect the binding of p-cumate), and a comparison to a positive control (PCR of cumate).
</P><DIV class="igem-legend"><STRONG>Figure 1.</STRONG> a. Cumate system PCR: 1. <SPAN class="underline">M</SPAN> ladder, 2.-3. Cumate PCR product; b. Cumate system colony PCR: 1. M ladder, 2.-5. Cumate system colony PCR; 5. Cumate system at ~1900 bp; c. Ladder <SPAN class="underline">M</SPAN></DIV><H4>Hypothesis 2: Bacterial proliferation of cumate regulatory system with BFP in medium containing cumate in a concentration gradient</H4><H4>Hypothesis 3: Measuring BFP fluorescence of cumate regulatory system in medium containing cumate in a concentration gradient
</H4><P class="igem-text">
The main objective was to determine whether the gene expression of BFP was dose-dependent over time during the exponential phase and whether the presence of cumate in the bacterial medium affected growth. The cells were treated with different concentrations of p-cumate and the fluorescence was measured every 30 minutes after the initial incubation time for a total of 3 hours.
</P><P class="igem-text">
Bacterial growth was not affected by the cumate in the liquid medium. The fluorescence intensity follows the same trend in all samples, therefore the cumate-induced expression of BFP is not dose-dependent. Normally, we would expect to observe fluorescence only in the treated samples because with cumate present, the repressor is dissociated from the operon, enabling the transcription of the downstream gene. However, we assume that the CymR is mutated at a specific site based on the sequencing results; thus, the expressed protein is unable to bind to cumate in the first place, the T5 operon is not inhibited, and BFP is not expressed.
</P><DIV class="igem-legend"><STRONG>Figure 2.</STRONG>Growth curve of <I>E.coli</I> TOP10 over time in different cumate concentrations. There is no effect of cumate on the proliferation of the bacteria over time.
</DIV><DIV class="igem-legend"><STRONG>Figure 3.</STRONG>Expression of BFP in <I>E.coli</I> TOP10 in different cumate concentrations over time. The fluorescence intensity is higher in all samples compared to the negative control. Therefore, the expression of BFP is not dose-dependent and fluorescence can be detected even in the untreated sample.
</DIV><P class="igem-text">
Subsequent experiments should follow with a positive control (bacteria with tagBFP) to confirm our assumption that there is BFP expression in our system. Furthermore, another experimental setup could be employed for the same endpoint, such as longer incubation time with cumate.
</P><H4>Hypothesis 4: Cloning of colicin into pSB4A5</H4><P class="igem-text">
The colicin gBlock was amplified via PCR, digested with EcoRI and PstI, and ligated into the linearized pSB4A5 low-copy-number plasmid backbone.
</P><DIV class="igem-legend"><STRONG>Figure 4.</STRONG>Transformation of colicin+pSB4A5 low-copy-number plasmid into <I>E. coli</I> BL21 cells.
</DIV><P class="igem-text">
A purified plasmid sample was sent for sequencing. However, the results showed numerous mutations.
</P><H4>Hypothesis 5: Subcloning of colicin into cumate regulatory system</H4><P class="igem-text">
Plasmids from the transformed bacterial colonies containing the cumate system were digested with XmaI and NdeI in order to remove the BFP and insert colicin in its place. After the digestion a gel extraction was performed, followed by ligation with the colicin DNA and a subsequent transformation.
</P><P class="igem-text">
The resulting colonies contained an insert of the appropriate size (~1600 bp), which was the same size as the positive control (the ligated product - cumate and colicin).
The results from the sequencing contained multiple mutations at variable sites along the sequence.
</P><DIV class="igem-legend"><STRONG>Figure 5.</STRONG>Plate with overnight transformation of ligated cumate+colicin+pSB1C3 showing one colony. 
</DIV><P class="igem-text">
Successful cloning was demonstrated with colony PCR with positive control as a reference (ligated product cumate and colicin) and sequencing results, which, however, showed multiple mutations due to continuous exposure to UV and repetitive PCRs.
</P><DIV class="igem-legend"><STRONG>Figure 6.</STRONG>Cumate+colicin colony PCR: 1. Ladder <SPAN class="underline">M</SPAN>, 2. colony PCR cumate+colicin, 3. ligated product cumate+colicin, 4. Ladder <SPAN class="underline">M</SPAN></DIV><H4>Hypothesis 6: Bacterial proliferation of cumate regulatory system with colicin in medium containing cumate in a concentration gradient</H4><P class="igem-text">
The next step would be to measure the toxicity of colicin by growing the bacteria in cumate-containing medium. Unfortunately, due to time constraints we were unable to test out our hypothesis.
</P><H3>Tryptophan regulatory system</H3><TABLE style="width:60%"><TBODY><TR><TH>No</TH><TH>Hypothesis</TH><TH>Status</TH></TR><TR><TH>1</TH><TH>Cloning of tryptophan regulatory system gBlock into pSB4A5</TH><TH>Positive</TH></TR><TR><TH>2</TH><TH>Bacterial proliferation of tryptophan regulatory system with RFP in medium containing tryptophan in a concentration gradient</TH><TH>Not started</TH></TR><TR><TH>3</TH><TH>Measuring RFP fluorescence of tryptophan regulatory system in medium containing tryptophan in a concentration gradient</TH><TH>Not started</TH></TR><TR><TH>4</TH><TH>Cloning of Im2 into pSB1C3</TH><TH>Positive</TH></TR><TR><TH>5</TH><TH>Subcloning of Im2 into tryptophan regulatory system</TH><TH>Not started</TH></TR><TR><TH>6</TH><TH>Bacterial proliferation of tryptophan regulatory system with Im2 in medium containing tryptophan in a concentration gradient</TH><TH>Not started</TH></TR></TBODY></TABLE><H4>Hypothesis 1: Cloning of tryptophan regulatory system gBlock into pSB4A5</H4><P class="igem-text">
After several unsuccessful transformations, we attempted to clone the tryptophan system into a low-copy-number plasmid provided by iGEM HQ (pSB4A5). PCR-amplified product was digested and ligated with the backbone, followed by an overnight transformation. A self-ligation was used as a negative control. 
</P><DIV class="igem-legend"><STRONG>Figure 7.</STRONG> Plate with transformation of Trp and pSB4A5 plasmid backbone. 
</DIV><P class="igem-text">
A colony PCR was performed, both on the negative control plate and on the desired product. The results showed the presence of the fragment of interest, along with other unknown products. 
</P><DIV class="igem-legend"><STRONG>Figure 8.</STRONG> Colony PCR tryptophan system. 1. Ladder M, 2-5. Negative control (self-ligation), 6-13. Colonies from Trp plate; 14. Ladder <SPAN class="underline">M</SPAN></DIV><P class="igem-text">
Plasmids containing the tryptophan system were sent for sequencing. However, the results exhibited many mutations in various genes.
</P><H4>Hypothesis 2: Bacterial proliferation of tryptophan regulatory system with RFP in medium containing tryptophan in a concentration gradient</H4><H4>Hypothesis 3: Measuring RFP fluorescence of tryptophan regulatory system in medium containing tryptophan in a concentration gradient</H4><P class="igem-text">
The follow-up steps after cloning of the tryptophan regulatory system would be to measure the bacterial proliferation and the expression of RFP in cumate-containing medium. Unfortunately, the sequencing results showed multiple mutations and due to time constraints we were unable to optimize the system and perform the tests.
</P><H4>Hypothesis 4: Cloning of Im2 into pSB1C3</H4><P class="igem-text">
The immunity protein 2 gBlock was PCR amplified, digested with EcoRI and PstI, and ligated with the pSB1C3 plasmid backbone from iGEM HQ. The ligated product was transformed and confirmed with colony PCR and sequencing. No mutations were detected in the sequencing results.
</P><DIV class="igem-legend"><STRONG>Figure 9.</STRONG> Im2 PCR: 1-2. Ladder <SPAN class="underline">M</SPAN>, 3-4. Im2 PCR product; Im2: 304 bp.
</DIV><DIV class="igem-legend"><STRONG>Figure 10.</STRONG> Transformation of Im2+pSB1C3.
</DIV><H4>Hypothesis 5: Subcloning of Im2 into tryptophan regulatory system</H4><H4>Hypothesis 6: Bacterial proliferation of tryptophan regulatory system with Im2 in medium containing tryptophan in a concentration gradient</H4><P class="igem-text">
The next step would be to subclone the immunity protein 2 into the tryptophan regulatory system construct and perform proliferation tests. Unfortunately, due to time constraints, we were unable to complete these experiments.
</P><H3>Genome integration</H3><TABLE style="width:60%"><TBODY><TR><TH>No</TH><TH>Hypothesis</TH><TH>Status</TH></TR><TR><TH>1</TH><TH>Genome integration of RFP as a pilot study</TH><TH>Positive</TH></TR><TR><TH>2</TH><TH>Genome integration of cumate regulatory system</TH><TH>Not started</TH></TR><TR><TH>3</TH><TH>Genome integration of tryptophan regulatory system</TH><TH>Not started</TH></TR><TR><TH>4</TH><TH>Removal of Gm antibiotic resistance from genome integration cassette in delivery plasmids from UPO-Sevilla 2011</TH><TH>Positive</TH></TR></TBODY></TABLE><H4>Hypothesis 1: Genome integration of RFP as a pilot study</H4><P class="igem-text">
A genome integration study was performed with the plasmids from iGEM HQ and the transposase plasmid (<A target="_blank" href="https://www.addgene.org/64968/">pTNS2</A>) provided by Addgene. The purpose of this experimental part was to determine the viability of both plasmids when implemented into the same bacteria and the success rate of a study utilizing machinery for genome integration. The RFP construct was cloned into the UPO-Sevilla 2011 plasmids (<A target="_blank" href="http://parts.igem.org/Genome_Integration">C18Sfi and C18R6</A>) which were subsequently transformed with the transposase-containing suicide plasmid.
</P><DIV class="igem-legend"><STRONG>Figure 11.</STRONG> Plate showing transformation of R6 plasmid from iGEM HQ with RFP construct. Colonies circled in green: white, self-ligation. Colonies circled in red: red, containing RFP. All red colonies contain RFP, but only some of them contain both RFP and pTNS2; the white colonies are empty carrier plasmids.
</DIV><P class="igem-text">
The integration was confirmed by colony PCR using primers that anneal to the genome of the bacteria. The forward primer anneals to the gene following the attTn7 integration site (glmS), while the reverse primer binds to the transposition cassette. A formed product means that the two sites are present in the same template. Given that one of the primers anneals only to the genome, this must mean that the product from the colony PCR reaction comes from the chromosome, not a plasmid.
</P><P class="igem-text">
Successful integration was proven with transformed bacteria exhibiting red color, where RFP is found in genome integration plasmids only, because they have ampicillin resistance, while the pSB1C3 backbone is contains chloramphenicol resistance. The integration into the specific attTn7 site was also confirmed with colony PCR using primers covering the downstream glmS gene (genome site) and part of the Tn7R site (plasmid site).
</P><DIV class="igem-legend"><STRONG>Figure 12.</STRONG> Colony PCR genome integration. PCR performed with primers annealing to the genome (forward primer glmS gene) and to the Tn7R sequence in the transposition cassette (reverse primer Tn7 sequence). A formed product means that the insert has been successfully integrated into the genome. 1. Ladder <SPAN class="underline">M</SPAN>, 2-11. Colony PCR red colonies, ladder <SPAN class="underline">M</SPAN>.
</DIV><P class="igem-text">
Annotated region shows bands at approx. 200 bp, which represents amplification of a product where both primers anneal, indicating genome integration. Wells 4-9. contain DNA from colonies with successful integration.
</P><H4>Hypothesis 2: Genome integration of cumate regulatory system</H4><H4>Hypothesis 3: Genome integration of tryptophan regulatory system</H4><P class="igem-text">
After successful cloning of both systems and follow-up experiments, such as proliferation and dose-dependent protein expression tests, the next stage would be to integrate them into the bacterial chromosome. Due to time constraints this step was not performed.
</P><H4>Hypothesis 4: Removal of Gm antibiotic resistance from genome integration cassette in delivery plasmids from UPO-Sevilla 2011</H4><P class="igem-text">
The gentamicin cassette was removed from the iGEM HQ plasmids with the goal to limit the spreading of antibiotic resistance to the genome. The genes coding for the resistance are located inside the transposable cassette of the plasmids, meaning they get transferred along with the fragment of interest.
</P><P class="igem-text">
We have removed the 800 bp insert by digesting the plasmids with SphI, whose restriction sites flank both sides of the gentamicin cassette. The digestion was followed by religation and transformation. An informative digestion was performed, where a presence of two bands, one of which at ~800 bp would indicate an unsuccessful reaction. The gel shows only a single band in each well, signifying a removed resistance gene.
</P><P class="igem-text">
Gentamicin resistance removal was confirmed with a gel showing the results from a SphI restriction digestion of C18 and R6 plasmids containing cumate.
</P><DIV class="igem-legend"><STRONG>Figure 13.</STRONG> Informative digestion of C18 and R6 with Gm gene (C18/R6 Gm+) and C18 and R6 without Gm gene (C18/R6 Gm-) using SphI. SphI restriction site flanks the Gm gene.  Therefore, unsuccessful digestion would yield a product around 800 bp. 1. Ladder <SPAN class="underline">M</SPAN>, 2. C18 Gm+, 3. C18 Gm-, 6. R6 Gm+, 7. R6 Gm-, 8. Ladder M. Red: C18 and R6 backbones. Green: Gm cassette (approx. 800 bp). Wells 3. and 7. show no band at 800 bp, therefore the removal of the Gm cassette has been successful.
</DIV><H3>Expression of colicin in a pH-controlled system</H3><TABLE style="width:60%"><TBODY><TR><TH>No</TH><TH>Hypothesis</TH><TH>Status</TH></TR><TR><TH>1</TH><TH>Cloning of AsR pH-sensitive promoter with colicin and double terminator into pSB1C3</TH><TH>Negative</TH></TR></TBODY></TABLE><H4>Hypothesis 1: Cloning of AsR pH-sensitive promoter with colicin and double terminator into pSB1C3</H4><P class="igem-text">
We attempted to clone the AsR promoter with its native RBS <A target="_blank" href="http://parts.igem.org/Part:BBa_K1231000">BBa_K1231000</A> obtained from iGEM HQ with colicin E2 and <A target="_blank" href="http://parts.igem.org/Part:BBa_B0015">double terminator</A> into a pSB1C3 backbone. The idea is to create a complete system that would be controlled by a pH gradient, potentially applicable in an acidic environment (e.g. the stomach).
</P><P class="igem-text">
Digestion was performed to remove the backbone and isolate the fragment of interest. The AsR promoter has a length of 140 bp. However, the gel shows the AsR promoter with a length of 1300 bp.
</P><DIV class="igem-legend"><STRONG>Figure 14.</STRONG> Digestion of AsR (2.) and digested backbone of pSB1C3 (6.). In green: pSB1C3 backbone. In red: suspected AsR insert at 1300 bp, instead of 140 bp. Ladder for reference: Ladder <SPAN class="underline">M</SPAN> (7.)
</DIV><P class="igem-text">
As the length of the digestion product did not match the expected one, we performed a colony PCR and confirmed a fragment with a length of 1300 bp, not 140 bp.
</P><DIV class="igem-legend"><STRONG>Figure 15.</STRONG> Colony PCR AsR. Ladder for reference: <SPAN class="underline">M</SPAN>. Well 5-18. AsR colony PCR. Annotated band at 1300 bp. 
</DIV><P class="igem-text">
A purified plasmid sample was sent for sequencing and after an informative digestion of the BioBrick part along with our plasmids showed identical results, iGEM HQ was contacted about the issue. It would appear that the BioBrick was submitted to HQ with modified prefix and suffix sequences and the iGEM prefix and suffix could not be found. Because of these issues, we discontinued our work with the abovementioned BioBrick.
</P></DIV><SECTION class="container igem-content-section"><DIV class="igem-content"><H2>Sensing</H2><TABLE style="width:60%"><TBODY><TR><TH>No</TH><TH>Hypothesis</TH><TH>Status</TH></TR><TR><TH>1</TH><TH>Cloning osmosensitive promoter + YFP (BBa_I6211) into pSB4A5</TH><TH>Positive</TH></TR><TR><TH>2</TH><TH>Increased expression of YFP due to increased osmotic pressure</TH><TH>Inconclusive</TH></TR><TR><TH>3</TH><TH>Cloning osmosensitive promoter + RFP (BBa_M30011) into pSB4A5</TH><TH>Positive</TH></TR><TR><TH>4</TH><TH>Increased expression of RFP due to increased osmotic pressure</TH><TH>Positive</TH></TR><TR><TH>5</TH><TH>Cloning sialidase downstream of osmosensitive promoter</TH><TH>Positive</TH></TR><TR><TH>6</TH><TH>Increased expression of sialidase due to increased osmotic pressure, using a sucrose gradient</TH><TH>Inconclusive</TH></TR><TR><TH>7</TH><TH>Increased expression of sialidase due to increased osmotic pressure, using a mucin concentration gradient</TH><TH>Not started</TH></TR></TBODY></TABLE><H3>YFP</H3><H4>Hypothesis 1: Successful cloning osmosensitive promoter + YFP (BBa_I6211) into low copy number plasmid (pSB4A5)</H4><P class="igem-text">
We cloned the construct OmpR responsive promoter-YFP into a low copy number plasmid (pSB4A5) to yield higher levels of fluorescence. pSB4A5 originally contained a RFP insert, making it easy to screen for successful ligation after transformation. The white/yellow colonies contain the right insert, red colonies are re-ligated plasmids (figure 1).
</P><DIV class="igem-legend"><STRONG>Figure 1.</STRONG>TOP10 cells transformed with the construct OmpR-YFP in pSB4A5. White/yellow colonies are successful ligations and red colonies are re-ligations of the original insert. 
</DIV><H4>Hypothesis 2: Increased expression of YFP due to increased osmotic pressure</H4><P class="igem-text">
We cultivated TOP10 cells, transformed with the successful clonings, in sucrose and salt gradients and thereafter measured fluorescence (excitation wavelength 485-12 nm ± 10, emission wavelength 520 nm ± 30) of YFP at each OD600 from 0.1 to 0.6, expecting to see higher levels of fluorescence in higher sucrose and salt concentrations. We used both negative and positive controls in the highest concentrations of sucrose and NaCl and in a 0% sample. Our negative control was untransformed TOP10 cells and positive control TOP10 cells transformed with constitutively expressed YFP (BBa_K592101).
</P><DIV class="igem-legend"><STRONG>Figure 2.</STRONG> Average fluorescence of YFP at OD 0.3, only sucrose and controls.
</DIV><DIV class="igem-legend"><STRONG>Figure 3.</STRONG> Average fluorescence of YFP at OD 0.3, only NaCl and controls.
</DIV><P class="igem-text">
However, the obtained results from the fluorescence measurements were not conclusive. First of all, the negative controls showed higher fluorescence than some of the samples. This is further demonstrated in figure 2 and 3. As the negative control contained no YFP, we concluded that the bacteria itself have a lot of background fluorescence. Second, we observed no trend regarding increasing fluorescence with rising osmotic pressure, neither in sucrose nor in the salt gradient. We could therefore not draw any conclusions regarding the activity of the OmpR promoter when using YFP as reporter and decided to replace the reporter protein to RFP, the same as iGEM Stockholm 2015 had used.
</P><H3>RFP</H3><H4>Hypothesis 3: Successful cloning OmpR+RFP (BBa_M30011) into low copy number plasmid (pSB4A5)</H4><P class="igem-text">
After deciding to replace the YFP reporter with RFP reporter, we began working with the already existing biobrick BBa_M30011 which contains the construct OmpR-RFP. We performed the same cloning procedure as we designed for the earlier construct, and successfully retrieved OmpR-RFP in the low copy number plasmid pSB4A5 (figure 4). The red colonies contain the correct ligation product, and the white/yellow colonies have re-ligated.
</P><DIV class="igem-legend"><STRONG>Figure 4.</STRONG> TOP10 cells transformed with the construct OmpR-RFP in pSB4A5. Red colonies are successful ligations and white/yellow colonies are re-ligations of the previous insert.
</DIV><H4>Hypothesis 4: Increased expression of RFP due to increased osmotic pressure</H4><P class="igem-text">
After successfully cloning the RFP+OmpR construct into the low copy number plasmid pSB4A5, we tested the activity of the promoter by using the same protocol as for the YFP testing.
</P><P class="igem-text">
When using RFP instead of YFP, we obtained reliable and conclusive results (figure 5). Now we could observe a significant increase in fluorescence of the samples, in comparison to the negative control, as shown in figure 7. It was noted that, the fluorescence intensity (at wavelength 580 nm ± 10 and emission length 627 nm ± 30) of the salt gradient cultures was very low in comparison to those in the sucrose gradient.
</P><P class="igem-text">
Throughout the growth curve, the fluorescence also increased with the increasing sucrose concentration (figure 6). However, the bacteria grown in 15% sucrose did not grow past OD 0.3 and therefore no results were obtained regarding the highest sucrose concentration for higher OD values. It is still to be noted that there is a significant increase in fluorescence when increasing sucrose concentration.
</P><DIV class="igem-legend"><STRONG>Figure 5.</STRONG> Average fluorescence of RFP at OD 0.3, only sucrose and controls.
</DIV><DIV class="igem-legend"><STRONG>Figure 6.</STRONG> Fluorescence of RFP over OD values, samples cultivated in sucrose gradient.
</DIV><P class="igem-text">
We could unfortunately not draw any conclusions from the salt gradient. As previously mentioned, the fluorescence intensity was very low in comparison to the sucrose gradient and at the same time no trend could be observed (figure 8). As demonstrated by figure 9, there was almost no difference in fluorescence between the samples, at any point in the growth curve.
</P><P class="igem-text">
We had designed the NaCl experiment on the results obtained by iGEM UCL 2015 where the team had used NaCl concentrations between 0.05% and 0.1% but had concluded that these were too high for the OmpR promoter. We therefore designed our experiment with lower concentrations, between 0.00625% and 0.1%, but still we did not see any conclusive results.
</P><DIV class="igem-legend"><STRONG>Figure 7.</STRONG> Average fluorescence of RFP at OD 0.3, only NaCl and controls.
</DIV><DIV class="igem-legend"><STRONG>Figure 8.</STRONG> Fluorescence of RFP over OD values, samples cultivated in salt gradient.
</DIV><H4>Growth curve</H4><P class="igem-text">
When analysing the growth curves of the sucrose and salt gradient cultures, we can observe a much greater difference between the samples in the sucrose gradient than those in the salt gradient. As the sucrose concentration increases, the growth is inhibited (figure 10). This trend cannot be observed in the salt gradient (figure 11). However, the salt concentrations were very low in comparison to the sucrose concentrations, which could explain the small difference.
</P><DIV class="igem-legend"><STRONG>Figure 9.</STRONG> Growth curve of samples cultivated in sucrose gradient.
</DIV><DIV class="igem-legend"><STRONG>Figure 10.</STRONG> Growth curve of samples cultivated in salt gradient.
</DIV><P class="igem-text">
In conclusion, we have proved that the activity of the OmpR promoter is increased in higher osmotic pressure with the results from the sucrose gradient, meaning that our bacteria would be able to sense the changes in osmotic pressure in the mucus. However the RFP was just used as a reporter gene to test the activity of the promoter. The next step was to replace RFP with our mucus-degrading enzyme sialidase.
</P><H3>Expressing sialidase with OmpR promoter</H3><H4>Hypothesis 5: Successful cloning sialidase downstream of osmosensitive promoter</H4>
We digested a PCR product with RBS-sialidase and ligated it downstream of the OmpR responsive promoter. After screening using colony PCR, we confirmed successful cloning as we saw bands at around 1700 bp (figure 12). The control is slightly smaller than the insert, which is because it does not contain the OmpR responsive promoter.
<DIV class="igem-legend"><STRONG>Figure 11.</STRONG> Gel of colony PCR after cloning. From left to right: DNA ladder, well 2-6: successful clonings of RBS-sialidase downstream of OmpR, well 8: re-ligation of OmpR to the backbone, well 9, 10: successful clonings of RBS-sialidase downstream of OmpR, well 11,12: successful cloning of RBS-sialidase in pSB1C3 backbone, control.
</DIV><H4>Hypothesis 6: Increased expression of sialidase due to increased osmotic pressure, using a sucrose gradient</H4><P class="igem-text">
After cloning sialidase downstream of the OmpR responsive promoter, we executed the same osmolarity test by procedure, but this time using only a sucrose gradient as this yielded conclusive results before. However, when analyzing the SDS-PAGE gels of our expressed protein, we found only a very faint band at the expected size, in 10% sucrose (figure 12).
</P><DIV class="igem-legend"><STRONG>Figure 12.</STRONG> SDS-PAGE gel of sialidase expression. From left to right: ladder, well 2-6: expression in 10% sucrose, well 7-11: expression in 15% sucrose, well 12: positive control.
</DIV><P class="igem-text">
There could be several explanations for the failure of this experiment. Firstly, it was the first try, meaning that, due to lack of experience, many errors could have been made during the experiment and the procedure was not optimized. Second, there are many bands on the gel, meaning that the sample was impure and hence the purification was improper. Due to time constraints, we could unfortunately not repeat the experiment to avoid the mistakes made during the first try.
</P><H4>Hypothesis 7: Increased expression of sialidase due to increased osmotic pressure, using a mucin concentration gradient</H4><P class="igem-text">
To further connect the parts of our project, we aimed to express sialidase with our OmpR responsive promoter in a mucin concentration gradient. However, due to time constraints we were unable to perform this experiment.
</P></DIV></SECTION><SECTION class="container igem-content-section"><DIV class="igem-content"><H2>Degradation</H2><TABLE style="width:60%"><TBODY><TR><TH>No</TH><TH>Hypothesis</TH><TH>Status</TH></TR><TR><TH>1</TH><TH>Sialidase can be successfully expressed in <I>E. coli</I></TH><TH>Positive</TH></TR><TR><TH>2</TH><TH>Endo-β-galactosidase can be successfully expressed in <I>E. coli</I></TH><TH>Positive</TH></TR><TR><TH>3</TH><TH>Sialidase can be secreted using HylA secretion system in <I>E. coli</I></TH><TH>Inconclusive</TH></TR><TR><TH>4</TH><TH>Endo-β-galactosidase can be secreted using HylA secretion system in <I>E. coli</I></TH><TH>Not started</TH></TR><TR><TH>5</TH><TH>Successfully expressed sialidase shows enzymatic activity on mucin</TH><TH>Positive</TH></TR><TR><TH>6</TH><TH>Successfully expressed endo-β-galactosidase shows enzymatic activity on mucin</TH><TH>Inconclusive</TH></TR><TR><TH>7</TH><TH>Glycan removal decreases mucus viscosity.</TH><TH>Positive</TH></TR><TR><TH>8</TH><TH>Mucin samples enzymatically treated with endo-β-galactosidase and/or sialidase show a decrease in viscosity.</TH><TH>Not started</TH></TR><TR><TH>9</TH><TH><I>E. coli</I> can grow and express enzymes in mucin samples</TH><TH>Not started</TH></TR></TBODY></TABLE><H3>Hypothesis 1: Sialidase can be successfully expressed in <I>E. coli</I></H3><P class="igem-text">
Our first step was to express sialidase in <I>E.coli</I> BL21(DE3) from a plasmid (au54) that we received from Dr. Christensen (Egebjerg, 2005). This expression was induced at OD600 of 0.4 and an IPTG concentration of 0.5 mM. Samples from fractions collected after IMAC purification were analysed by SDS-PAGE (figure 1). The band at 60 kDa is slightly larger than the expected size of sialidase in literature (54 kD.
</P><DIV class="igem-legend"><STRONG>Figure 1.</STRONG> SDS-PAGE gel and a protein ladder. From left to right: protein ladder, the remaining lanes contain samples from consecutive fractions after IMAC purification.
</DIV><P class="igem-text">
Next, we cloned our gBlock containing T7 promoter-RBS-sialidase into the iGEM compatible backbone (pSB1C3). To confirm successful cloning, we performed double digestion on the plasmid (figure 2) and could see one band at ~1600 bp and one at ~2000 bp. The band at 1600 bp corresponds to the size of sialidase and the band at 2000 bp to the linearized plasmid backbone.
</P><DIV class="igem-legend"><STRONG>Figure 2.</STRONG> From left to right: DNA ladder, the next two are digested BBa_K2235009 plasmids.
</DIV><P class="igem-text">
We thereafter induced expression of our sialidase (BBa_K2235009), using the same method as when expressing the au54 plasmid, and could observe a band at 60 kDa (figure 3). While this also is slightly larger than the designed sialidase (55 kDa) sequence, it has a similar size as the sialidase we expressed from the literature plasmid (au54).
</P><DIV class="igem-legend"><STRONG>Figure 3.</STRONG> SDS-PAGE gel and a protein ladder. From left to right: protein ladder, the remaining three are IMAC purification fractions of sialidase.
</DIV><P class="igem-text">
To demonstrate that the sialidase we expressed from our designed biobrick (BBa_K2235009) is similar to the size of the au54 sialidase, we analysed both samples using SDS-PAGE. The au54 sialidase and our designed biobrick sialidase, with molecular sizes of 54 and 55 kDa, respectively. SDS-PAGE results show no observable difference in size between the two proteins (figure 4). Sialidase is a protein with a high content of basic amino acids, therefore it was hypothesized that this might affect migration through the gel.
</P><DIV class="igem-legend"><STRONG>Figure 4.</STRONG> SDS-PAGE gel and a protein ladder.From left to right: first well is a protein ladder, second well is protein expressed from the au54 plasmid, third well contains sialidase expressed from the biobrick.
</DIV><H3>Hypothesis 2: Endo-β-galactosidase can be successfully expressed in <I>E. coli</I></H3><P class="igem-text">
We designed our endo-β-galactosidase (EBG) biobrick (BBa_K2235010) by modifying the sequence of professor Li (Ashida, 2002). The sequence received did not contain a His6-tag, which was added to the sequence for later IMAC purification steps. The biobrick containing T7 promoter-RBS-EBG was cloned into an iGEM compatible plasmid backbone (pSB1C3). To confirm successful cloning, we double digested plasmids from five different colonies (figure 5). For each colony, two bands could be observed. One at ~1400 bp, corresponding to the size of EBG, and one at ~2000 bp, corresponding to the size of the plasmid backbone.
</P><DIV class="igem-legend"><STRONG>Figure 5.</STRONG> Agarose gel. From left to right: DNA ladder, empty well, the next five are digested BBa_K2235010 plasmids.
</DIV><P class="igem-text">
After confirming that the cloning worked, the biobrick plasmid (BBa_K2235010) was transformed into <I>E. coli</I> BL21(DE3) and expression was induced at multiple combinations of OD600 and IPTG concentrations. SDS-PAGE results from one of the successful expressions (at OD600 of 0.4 and an IPTG concentration of 0.5 mM) show the expression of EBG, a 47 kDa protein (figure 6). The band on lane three is believed to be EBG.
</P><DIV class="igem-legend"><STRONG>Figure 6.</STRONG> SDS-PAGE gel and a protein ladder. From left to right: protein ladder, the remaining five are IMAC purification fractions of EBG.
</DIV><H3>Hypothesis 3: Sialidase can be secreted using HylA secretion system in <I>E. coli</I></H3><P class="igem-text">
By carrying out the experiments above we demonstrated successful cloning and expression of both of our mucus degrading enzymes: sialidase and EBG. The next step was to secrete the enzymes, using an already existing biobrick for HylA <I>E.coli</I> secretion system (BBa_K1166002) from the iGEM 2017 distribution kit. Firstly, we removed the stop codon at the end of the sialidase gblock sequence using PCR and thereafter cloned sialidase without the stop codon upstream of the secretion system (BBa_K2235011). To confirm successful cloning, we double digested the plasmid (figure 7). Two bands were observed, one at ~7000 bp, corresponding to the size of T7 promoter-RBS-Sialidase-HylA <I>E.coli</I> secretion system, and one at ~2000 bp, corresponding to the size of the plasmid backbone.
</P><DIV class="igem-legend"><STRONG>Figure 7.</STRONG> Agarose gel. From left to right: DNA ladder, the next five are digested BBa_K2235011 plasmids.
</DIV><P class="igem-text">
The newly cloned plasmid was transformed into <I>E.coli</I> and expression in flask was induced with 0.5 mM IPTG. The enzyme was extracted from the medium using IMAC purification. SDS-PAGE results (figure 8) shows no secretion of a protein resembling the correct size (≈ 55 kDa).
</P><DIV class="igem-legend"><STRONG>Figure 8.</STRONG> SDS-PAGE gel and a protein ladder. From left to right: protein ladder, following four are IMAC purification fractions of a control that wasn’t induced with IPTG, the last four show the IPTG induced sample fractions.
</DIV><H3>Hypothesis 4: Endo-β-galactosidase can be secreted using HylA secretion system in <I>E. coli</I></H3><P class="igem-text">
Due to time restraints this part of the project was never reached and focus was instead put on the parts of the project which were showing more promising results.
</P><H3>Hypothesis 5: Successfully expressed sialidase shows enzymatic activity on mucin</H3><P class="igem-text">
With the goal of testing the enzymatic activity of sialidase on mucin, an assay was developed and optimized. The objective was to measure the concentration of sialic acid released after digestion, which was quantified using high performance anion exchange chromatography (HPAEC).
</P><P class="igem-text">
Using industrially purchased sialidase to treat bovine submaxillary mucin (BSM) gave a positive result. Sialic acid was proved to be digested from the mucin. Therefore, the next step was to repeat the experiment with sialidase that we expressed in <I>E. coli</I>. A range of different sialidase concentrations were used and their respective sialic acid digestion quantified (figure 9). We were expecting a linear increase of substrate degraded with increase of enzyme. It is believed that excessive enzyme was used in the experiment. The sialic acid concentration released even at the lowest enzyme concentration is large when compared to the positive control (deglycosylation using sulfuric acid).
</P><DIV class="igem-legend"><STRONG>Figure 9.</STRONG> Concentrations of sialic acid digested from BSM in comparison to the concentration of sialidase used.
</DIV><H3>Hypothesis 6: Successfully expressed endo-β-galactosidase shows enzymatic activity on mucin</H3><P class="igem-text">
To determine the concentration of sugars after the EBG digestion of PGM a colorimetric assay was performed. Industrially available EBG was used firstly to test the assay. The results from the assay were inconclusive as a result of a strong response from the negative control.
</P><P class="igem-text">
Due to time restraints this part of the project was never completed. The next step after fixing the assay would have been to test the expressed EBG from <I>E. coli</I>.
</P><H3>Hypothesis 7: Glycan removal decreases mucus viscosity</H3><P class="igem-text">
Mucins are densely covered with glycans (polysaccharides) which constitute up to 80% of dry mucus weight. Therefore, we aimed to reveal information on how the glycosylation state of mucins affects mucus viscosity (Lamblin et al., 1991). For this, we degraded pig gastric mucins (PGM) with a deglycosylating reagent, as done by mucus degrading enzymes, and measured the viscoelastic properties by performing rheology measurements.
</P><P class="igem-text">
Figure 10 displays how the viscosity of native PGM is slightly higher at low shear rate (close to standstill), compared to deglycosylated PGM. However, with increasing shear rate, which approximates physiological condition, the reduction in viscosity is faster in deglycosylated (slope: -0.181) compared to native PGM (slope: -0.371). Furthermore, visual fluidity comparison of the tested samples (figure 11) strengthened the findings of the rheology testing.  
</P><P class="igem-text">
Since this effect was even detectable at shear rates well below physiological conditions, we hypothesize that the viscosity of deglycosylated PGM will continue to drop faster than native PGM with increasing shear rates. If that is the case, then high shear rate events such as coughing, which causes a shear rate of 103−104 s−1 (this is 1000-10’000 fold higher than what we measured), would cause deglycosylated mucins to form much less viscous mucus than native mucins (Lai et al. 2009).
</P><DIV class="igem-legend"><STRONG>Figure 10.</STRONG> Deglycosylated PGM shows decreased viscosity with increasing shear rate.
</DIV><DIV class="igem-legend"><STRONG>Figure 11.</STRONG> Visual representation of the decrease in viscosity of PGM after chemical treatment. Left tube is deglycosylated PGM and right tube is untreated PGM.
</DIV><H3>Hypothesis 8: Mucin samples enzymatically treated with endo-β-galactosidase and/or sialidase show a decrease in viscosity.</H3><P class="igem-text">
Our results support the hypothesis of lowering mucus viscosity when mucin-associated glycans are removed. Due to time restraints we were not able to test our expressed enzymes.Thus, we would like to further explore the beneficial impact of enzymatic mucus degradation by glycosidases. In particular, the stepwise treatment of mucins with sialidase and endo-beta-galactosidase.
</P><H3>Hypothesis 9: <I>E.coli</I> can grow and express enzymes in mucin samples</H3><P class="igem-text">
Due to time restraints this part of the project was never reached.
</P><H1>References</H1><UL class="igem-references"><LI>Ashida, H., Maskos, K., Li, S.-C., &amp; Li, Y.-T. (2002). Characterization of a Novel Endo-β-galactosidase Specific for Releasing the Disaccharide GlcNAcα1→4Gal from Glycoconjugates†,‡. Biochemistry, 41(7), 2388–2395. http://doi.org/10.1021/bi011940e</LI><LI>Egebjerg, J., &amp; Christensen, S. (2005). Cloning, expression and characterization of a sialidase gene from Arthrobacter ureafaciens. Biotechnology and Applied Biochemistry, 41(3), 225. http://doi.org/10.1042/ba20040144</LI><LI>Cone R. Mucus. In: Michael WS, Lamm E, McGhee Jerry R, Mayer Lloyd, Mestecky Jiri, Bienenstock John, editors. Mucosal Immunology. San Diego: Academic Press; 1999. pp. 43–64.</LI><LI>Lamblin G, Lhermitte M, Klein A, Houdret N, Scharfman A, Ramphal R, Roussel P. The carbohydrate diversity of human respiratory mucins: a protection of the underlying mucosa? The American Review of Respiratory Disease. 1991;144:S19–S24.</LI></UL></DIV></SECTION></DIV><DIV class="container"><DIV class="igem-sponsors igem-sponsors-first"><H1> Sponsors </H1></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>