<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Research </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Project_Description">Project description</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Design_and_Background">Design and background</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Lab_summary">Lab summary</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/labbook">Lab book</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Demonstrate">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Outlook">Outlook</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Safety">Lab Safety</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/InterLab">Interlab study</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Measurement">Measurement</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/HP/Silver">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Handbook">Handbook</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Policy">Policy</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Entrepreneurship </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Entrepreneurship">Product Development</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Applied_Design">Applied Design</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Achievements">Achievements</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/About_Us">About Us</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Supervisors_and_Advisors">Supervisors and Advisors</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Collaborations">iGEM Team Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Other_Collaborations">Other Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/SGEM">SGEM</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Future">Future</A></LI></DIV></DIV><P><SECTION class="igem-prolung-promo"><VIDEO id="igem-prolung-video" poster="img/head.jpg" playsinline="" autoplay="" muted="" loop=""><SOURCE src="https://static.igem.org/mediawiki/2017/d/dc/Stockholm_mucus.mp4" type="video/mp4">
              Your browser does not support video.
            </SOURCE></VIDEO></SECTION></P><DIV class="container igem-prolung-logo"><DIV class="logo-slogan"><DIV id="slogan">A mucus degrading lung probiotic</DIV></DIV></DIV><DIV id="wrapper"><DIV class="igem-content"><H1>Demonstration</H1><P class="igem-text">
Many respiratory diseases such as asthma, chronic obstructive pulmonary disease (COPD) and cystic fibrosis are characterized by an excessive accumulation of thick mucus. Afflicted people suffer repeated lung infections and breathing difficulties caused by clogged airways, resulting in severely reduced quality of life. We address this issue by developing an unprecedented probiotic approach that controls the mucus thickness ina self-regulating manner to protect and promote respiratory health. We envision a system capable of clearing the airways and consequently removing entrapped pathogens and harmful particles.
</P><P class="igem-text">
Our aim is to demonstrate that our genetically engineered bacteria can sense the pathologically altered osmotic pressure caused by thickened mucus, which will trigger the expression of mucus-degrading enzymes. The objective of the mucus degradation is to reduce the viscosity of mucus by removing glycans, aiming to restore the mucociliary clearance. 
</P><P class="igem-text">
While the holistic demonstration of the project yet remains, we have demonstrated that key parts of our system are working as expected, as seen in the table below:
</P><TABLE style="width:60%">

  <TBODY><TR><TH>Key parts</TH><TH>Status</TH>
  </TR><TR><TH>1</TH><TH>Our engineered bacteria can sense changes in osmotic pressure </TH>
    <TH>Demonstrated</TH>
  </TR>

  <TR><TH>2</TH><TH>Our engineered bacteria can express mucus degrading enzymes</TH><TH>Demonstrated</TH>
  </TR><TR><TH>3</TH><TH>Our expressed enzymes can degrade mucus</TH><TH>Demonstrated</TH>
  </TR>

  <TR><TH>4</TH><TH>The viscosity of mucus is reduced after removal of glycans</TH><TH>Demonstrated</TH>  
  </TR><TR><TH>5</TH><TH>BioBricks can be integrated into the genome at a specific site</TH>
    <TH>Demonstrated</TH>
  </TR></TBODY></TABLE><H2>1: Our engineered bacteria can sense changes in osmotic pressure</H2><P class="igem-text">
To sense excessively thick mucus, we employed a promoter (BBa_M30011) sensitive to changes in osmotic pressure. Characterization of this promoter was carried out by increasing osmotic pressure. Using RFP as a reporter, we confirmed increasing activity of the promoter along a sucrose gradient in a dose-dependent manner, demonstrating that our system is able to sense changes in osmotic pressure. Additionally, we showed that this is valid throughout the growth curve (figure 1). 
</P><DIV class="igem-legend"><STRONG>Figure 1.</STRONG> Fluorescence of RFP over OD. Increased fluorescence intensity is observed when osmotic pressure (as demonstrated by increasing sucrose concentration) is increased. The trend is verified throughout the growth curve. 
</DIV><H2>2: Our engineered bacteria can express mucus degrading enzymes</H2><P class="igem-text">
In order to successfully degrade thick mucus, it is essential for our engineered bacteria to express the enzymes, sialidase and endo-β-galactosidase (EBG). After induction and protein extraction from our test chassis <I>E. coli</I>, we could confirm expression of both mucus-degrading enzymes with SDS-PAGE (Figure 2 and 3).
</P><DIV class="igem-legend"><STRONG>Figure 2.</STRONG> SDS-PAGE gel verifying expression of sialidase
</DIV><DIV class="igem-legend"><STRONG>Figure 3.</STRONG> SDS-PAGE gel verifying expression of Endo-β-galactosidase
</DIV><P class="igem-text">
Towards the end of our project, we also managed to express our mucus-degrading enzyme sialidase, under regulation of the OmpR responsive promoter (figure 4; well 5 and 6).
</P><DIV class="igem-legend"><STRONG>Figure 4.</STRONG> SDS-PAGE gel of sialidase expression. From left to right: ladder, well 2-3: expression in 5% sucrose, well 4-8: expression in 10% sucrose, well 9-13: expression in 15% sucrose well 14: positive control
</DIV><H2>3: Our expressed enzymes can degrade mucus</H2><P class="igem-text">After successfully expressing sialidase in our E.coli chassis, we investigated its potential to degrade mucus. In order to achieve this goal, we used high performance anion exchange chromatography (HPAEC) to quantify the sialic acid released after treating bovine submaxillary mucin (BSM) with our expressed sialidase. Although the quantification of the sialidase activity returned inconclusive results, we were able to consistently demonstrate its ability to digest terminal sialic bonds from mucin (figure 5), and, as a consequence, its potential to alter mucin composition and properties.</P><DIV class="igem-legend"><STRONG>Figure 5.</STRONG>Concentration of sialic acid of BSM after digestion with sialidase
</DIV><H2>4: Removal of glycans reduces the mucus viscosity</H2><P class="igem-text">
Since HPAEC analysis confirmed that our expressed sialidase enzymatically degrades mucus, we further investigated the impact of deglycosylation on the mucus viscoelastic properties by rheology testing. To demonstrate that removal (deglycosylation) of mucin associated glycans, as done by the mucus degrading enzymes, leads to reduced viscosity, we treated pig gastric mucins (PGM) with a deglycosylating agent. As shown by figure 6, the viscosity of treated PGM was decreased at shear rates well below physiological conditions. This finding not only supports the hypothesis that removal of mucin-associated glycans lowers the  mucus viscosity, but also suggests improvement of mucociliary clearance. Thus, targeting glycan removal could not only pose a potential alternative to currently available mucolytics, but is also a promising approach to remove mucus from patients suffering from excessive accumulation of thick mucus. Having this novel strategy in mind, we would like to further explore the beneficial impact of enzymatic mucus degradation by glycosidases. In particular, we would like to investigate the stepwise treatment of mucins with sialidase and endo-beta-galactosidase.
</P><DIV class="igem-legend"><STRONG>Figure 6.</STRONG> Rheology measurement displaying viscosity and stress of untreated mucus (native PGM) versus treated mucus (deglycosylated PGM)
</DIV><DIV class="igem-legend"><STRONG>Figure 7.</STRONG> Visual representation of the decrease in viscosity of PGM after chemical treatment. Left tube is deglycosylated PGM and right tube is untreated PGM.
</DIV><H2>5: BioBricks can be integrated into the genome at a specific site</H2><DIV class="igem-text">
Our objective was to demonstrate that genome integration of any BioBrick part is successful by using the sophisticated mini-Tn7 system in combination with the pTNS2 transposase plasmid. The results confirm successful integration of the reporter gene construct <A href="http://parts.igem.org/Part:BBa_J04450">BBa_J04450</A> into the bacterial chromosome (figure 8).
</DIV><DIV class="igem-legend"><STRONG>Figure 8.</STRONG> Colony PCR of genome integration. PCR performed with primers annealing to the genome (forward primer glmS gene) and to the Tn7R sequence in the transposition cassette (reverse primer Tn7 sequence). A formed product means that the insert has been successfully integrated into the genome. (1. Ladder <SPAN class="underline">M</SPAN>, 2-11. Colony PCR red colonies, 12. Ladder <SPAN class="underline">M</SPAN>). Annotated region shows bands at approx. 200 bp, which represents amplification of a product where both primers anneal, indicating genome integration. Wells 4-9. contain DNA from colonies with successful integration.
</DIV><P class="igem-text">
To further emphasize the concept of biocontainment and safe GMOs, we have removed the gentamicin cassette using a restriction enzyme specific for the sequence (figure 9). This action eliminates the risk of spreading of antibiotic resistance to other bacteria via horizontal gene transfer (HGT).
</P><DIV class="igem-legend"><STRONG>Figure 9.</STRONG> Informative digestion of C18 and R6 with Gm gene cassette (C18/R6 Gm+) and C18 and R6 without Gm gene cassette (C18/R6 Gm-) using SphI. SphI restriction site flanks the Gm gene. Therefore, unsuccessful digestion would yield a product of around 800 bp. (1. Ladder M, 2. C18 Gm+, 3. C18 Gm-, 6. R6 Gm+, 7. R6 Gm-, 8. Ladder <SPAN class="underline">M</SPAN>.) Red: C18 and R6 backbones. Green: Gm cassette (approx. 800 bp). Wells 3. and 7. show no band at 800 bp, therefore the removal of the Gm cassette has been successful. 
</DIV></DIV><DIV class="container"><DIV class="igem-sponsors igem-sponsors-first"><H1> Sponsors </H1></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>