<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Lab_summary skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Lab summary</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Research </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Project_Description">Project description</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Design_and_Background">Design and background</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Lab_summary">Lab summary</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/labbook">Lab book</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Demonstrate">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Outlook">Outlook</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Safety">Lab Safety</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/InterLab">Interlab study</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Measurement">Measurement</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/HP/Silver">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Handbook">Handbook</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Policy">Policy</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Entrepreneurship </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Entrepreneurship">Product Development</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Applied_Design">Applied Design</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Achievements">Achievements</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/About_Us">About Us</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Supervisors_and_Advisors">Supervisors and Advisors</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Collaborations">iGEM Team Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Other_Collaborations">Other Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/SGEM">SGEM</A></LI><LI><A href="https://2017.igem.org/Team:Stockholm/Future">Future</A></LI></DIV></DIV><P><SECTION class="igem-prolung-promo"><VIDEO id="igem-prolung-video" poster="img/head.jpg" playsinline="" autoplay="" muted="" loop=""><SOURCE src="https://static.igem.org/mediawiki/2017/d/dc/Stockholm_mucus.mp4" type="video/mp4">
              Your browser does not support video.
            </SOURCE></VIDEO></SECTION></P><DIV class="container igem-prolung-logo"><DIV class="logo-slogan"><DIV id="slogan">A mucus degrading lung probiotic</DIV></DIV></DIV><DIV id="wrapper"><DIV class="igem-content"><H1>Biocontainment</H1><H3>Testing gBlocks</H3><P class="igem-text">Initially, we had two gBlocks synthesized by our sponsor, IDT, which contained the whole biocontainment system, along with the Tn7 cassette used for genome integration. During the first weeks of working with them, we did not obtain any good results. The PCR was not optimized, resulting in many byproducts, unspecific binding of the primers along the sequence, smears on the agarose gels and low concentrations. Transformations always yielded few to no colonies.</P><P class="igem-text">After going back to the initial gBlock sequences, we realized we had used the same promoter and terminator for many genes, which meant that the DNA strands kept reannealing at the incorrect sites, resulting in random products. It was particularly difficult to clone the two gBlocks together, and whenever we succeeded, the yield was too low. We also found a few other mistakes in the design of the sequences as well, like repeating some restriction sites or not adding the correct overlapping regions. Working with these fragments required a lot of consecutive PCRs, purifications, and extraction steps, leading to loss of product and many mutations.</P><P class="igem-text">As the first constructs proved flawed, we decided to design and order a new set of gBlocks with the biocontainment system in two smaller subsystems: the cumate regulatory system and the tryptophan regulatory system. The Tn7 transposition cassette was not part of the gBlocks this time. Instead, we decided to use the genome integration plasmids from the <A target="_blank" href="https://2011.igem.org/Team:UPO-Sevilla">2011 iGEM UPO-Sevilla</A> team.</P><H4>Cloning</H4><P class="igem-text">We began by trying to clone each of the systems separately into the pSB1C3 backbone. Afterwards, we attempted to replace the reporter genes in the cumate and tryptophan systems with colicin and immunity protein 2, respectively.</P><H4>Characterizing the cumate system</H4><P class="igem-text">For a few weeks, we optimized our procedures and protocols before successfully cloning the cumate system and after, the cumate system where BFP was replaced with colicin. Successful cloning was confirmed by gel electrophoresis, and the DNA construct was sent for sequencing. Whilst waiting for sequencing results, we began characterizing the function of the systems. We grew <I>E. coli</I> transformed with the cumate system on plates containing a concentration gradient of p-cumate. However, the results were very ambiguous - the plates with the higher concentration of cumate, which should have no colonies due to toxin expression, showed the highest yield. After receiving the sequencing results, we found mutations in some nucleotides and hypothesized that these could have caused the questionable results. We determined that the mutations were due to long exposure times under UV light when doing gel extraction, or too many PCRs on the same product. Therefore, we returned to the first stock of cloned cumate. However, the sequencing came back with a key mutation - one that results in the change of a hydrophilic amino acid to a hydrophobic one in the sequence of the CymR regulatory protein. The mutation was not in the DNA binding site, but it may have been in the site that binds p-cumate, which would explain why many colonies could still be seen in higher p-cumate concentration. We also performed BFP fluorescence measurements to characterize the cumate system but were unable to draw any conclusions from the results, which could mean that the mutation indeed affected the binding of p-cumate.</P><H4>Characterizing the tryptophan system</H4><P class="igem-text">The tryptophan system proved a lot more difficult to work on. Cloning was complicated due to errors in sequence design, where some restriction sites were present more than once. This led to multiple digestions and gel extractions, use of alternative restriction sites, and series of consecutive transformations. The results from the sequenced fragments had numerous mutations and very low quality overall. There was not enough time to finish both of the biocontainment subsystems, therefore we decided to focus on the cumate system, where we had made more progress. </P><H4>pH-sensitive promoter</H4><P class="igem-text">Furthermore, we attempted to clone the pH-responsive promoter AsR with RBS, and then clone the part into the pSB1C3 plasmid containing colicin and a double terminator. The promoter has a total length of 140 bp, but the fragment on the gel continuously appeared at 1500 bp. After contacting iGEM HQ, we discovered the part was faulty. Therefore, we decided not to continue working with it.</P><H4>Genome integration</H4><P class="igem-text">We simultaneously started working with the genome integration plasmids we requested from iGEM HQ by performing a pilot study and successfully integrating RFP into the genome of <I>E. coli</I>. To further emphasize the importance of biocontainment, we removed the gentamicin resistance genes from the genome integration plasmids to limit the spreading of antibiotic resistance to the bacterial chromosome.</P></DIV><SECTION class="container"><DIV class="igem-content"><H1>Sensing</H1><H3>Characterizing promoter with YFP</H3><P class="igem-text">Our initial plan was to test the activity of the OmpR responsive promoter, BBa_R0082, using a YFP reporter. To do this, we used the already existing BioBrick BBa_I6211, containing the construct osmosensitive promoter-RBS-YFP-double terminator in the pSB1C3 backbone. As opposed to other constructs of reporter molecule downstream of the OmpR responsive promoter, YFP had no previous characterization. Therefore, we decided to work with the YFP reporter, to contribute with more information to the iGEM community regarding this BioBrick.</P><H4>Cloning into low copy number plasmid</H4><P class="igem-text">After consulting with the <A target="_blank" alt="iGEM Stockholm 2015" href="https://2015.igem.org/Team:Stockholm">iGEM Stockholm 2015</A> team, who also worked with the OmpR responsive promoter, we decided to clone the construct from the high copy number plasmid pSB1C3 into the low copy number plasmid pSB4A5. According to their work, the fluorescence would yield much higher levels when expressing the fluorescent protein in a low copy number plasmid.</P><P class="igem-text">We tested the OmpR responsive promoter by cultivating TOP10 cells in both a sucrose and a salt gradient, ranging from 5% to 15% sucrose and 0.00525% to 0.1% NaCl. After cultivating our transformed bacteria cells in the respective osmolarity media, we expected to see more fluorescence from the YFP when the solute concentration was increased. However, our results were very inconsistent and we could see no correlation between the activity of the promoter and increasing osmotic pressure. We also observed higher fluorescence in our negative control than in some of the samples, which could indicate high background fluorescence, making it difficult to distinguish between background and fluorescence of YFP.</P><P class="igem-text">After multiple unsuccessful tries of obtaining any conclusive results regarding the activity of the promoter when using YFP as a reporter, we decided to change the reporter to RFP instead. Luckily, there was an already existing and well characterized BioBrick with the construct osmosensitive promoter-RBS-RFP-double terminator (BBa_M30011). We successfully performed the same cloning procedure to get our construct in the low copy number plasmid pSB4A5.</P><H4>Testing promoter with RFP</H4><P class="igem-text">To test the RFP construct, we performed the same osmotic pressure test as when the promoter was connected to YFP. This time our test was much more successful. We measured the fluorescence of every sucrose and salt concentration at OD 0.1-0.6, with an interval of 0.1, and saw that the fluorescence gradually increased with the increasing sucrose concentration at every OD value. However, we could not draw any conclusions from the salt gradient.</P><H4>Combining sensing of osmotic pressure with expression of mucus degrading enzymes</H4><P class="igem-text">At this point, we had characterized an increased activity of the OmpR responsive promoter when the osmotic pressure was increased. Our next step was to clone one of our mucus degrading enzymes, sialidase, downstream of the OmpR responsive promoter. We thereafter continued to perform the same osmolarity test as with the reporter genes, to verify that increased osmotic pressure results in an increased expression of sialidase.</P><P class="igem-text">As we only had sialidase in the construct T7 promoter-RBS-sialidase, we had to remove the T7 promoter prior to cloning. We did this with PCR using a forward primer, annealing to the RBS site with XbaI restriction site overhang, and a reverse primer, annealing to the end of sialidase with suffix overhang for cloning downstream of the OmpR responsive promoter. However, the cloning procedure turned out to be very difficult and for several weeks we did not succeed at cloning them together. After multiple troubleshooting sessions, we started from the beginning by isolating RBS-sialidase using PCR. However, this time we used a different reverse primer, which annealed directly to the suffix and contained a scar overhang. Later, when we tried cloning the new PCR product, we confirmed the successful cloning of the OmpR responsive promoter with sialidase with colony PCR.</P><P class="igem-text">The final step in the sensing part of the project was to express sialidase with our osmosensitive promoter. We performed an osmolarity test similar to that of the RFP expression. However, this time we only used a sucrose gradient and additionally used a mutant strain, ΔEnvZ, which lacks the EnvZ/OmpR two-component system, to further demonstrate that increased expression is purely dependent on osmotic pressure changes. After the expression attempt, almost no sialidase was expressed by TOP10.</P></DIV></SECTION><SECTION class="container"><DIV class="igem-content"><H1>Degradation</H1><H4>Designing gBlocks</H4><P class="igem-text">When designing the BioBricks for the sialidase and endo-β-galactosidase (EBG) enzymes, we ordered gBlocks containing the desired enzyme sequence, along with a T7 promoter and RBS, in order to save time and avoid repetitive cloning. The T7 promoter is used for protein production and is activated by addition of IPTG. Additionally, we added a His-tag on the opposite side of our enzymes’ catalytic domains to enable protein purification with IMAC before validation of enzymatic activity.</P><H4>Expressing plasmids from other researchers</H4><P class="igem-text">Vectors containing the two enzymes were donated to us by the researchers we based our gBlock design on (Egebjerg, 2005) (Ashida, 2002). We transformed these vectors into <I>E. coli</I> and and grew them in flask cultures induced with IPTG. The produced proteins were extracted from the cells by sonication and purified using IMAC before confirming expression with SDS-PAGE.</P><P class="igem-text">Seeing no expression, we tried multiple combinations of OD600 and IPTG concentration for the sialidase vector. One combination (OD600 = 0.4 and c(IPTG) = 0.5 mM) showed a high expression and was used for all future experiments. Since the donated EBG vector design did not contain a His-tag, no further expression and purification was tried out. Simultaneously, we discovered that a substance called DTT used in the sonication lysis buffer, destroys the matrix resin of the IMAC purification. Therefore, DTT was replaced with β-mercaptoethanol, which does not harm the matrix.</P><H4>Expressing enzyme from our gBlocks</H4><P class="igem-text">The gBlocks were cloned into the pSB1C3 plasmid backbones. Successful clonings were transformed into <I>E. coli</I>. Extraction, purification and confirmation of the enzyme expression were performed using SDS-PAGE, as mentioned above. For some reason, the size of the protein appeared larger than it should be on the gel. Therefore, to show that the correct protein was expressed, we compared the researchers’ (Egebjerg, 2005) plasmid sialidase to our own sialidase BioBrick, and both were observed at the same size.</P><P class="igem-text">EBG BioBrick cultures were grown at different OD600 and IPTG concentration combinations. Extraction, purification and confirmation of enzyme expression was performed the same way as mentioned earlier by using SDS-PAGE. Some of the cultures were shown to express EBG.</P><H4>Testing our secretion system</H4><P class="igem-text">We looked into the possibility of making <I>E.coli</I> secrete our desired protein instead of extracting the proteins using sonication. We used the secretion system BioBrick (BBa_K1166002) from the iGEM 2017 distribution kit and used 3A assembly to clone the sialidase upstream of the secretion system device. After the ligation proved successful, the new plasmid was transformed. A flask culture containing the new plasmid was grown and the medium was purified, whilst the cells were lysed and the purified intracellular content was used as a reference. A SDS-PAGE was performed to prove expression and secretion, but no protein resembling the correct size was observed. </P><H4>Activity testing of sialidase and EBG</H4><P class="igem-text">We developed two assays to test the activity of sialidase and EBG, based on the activity of industrially produced enzymes. Sialidase activity was measured using high-performance anion-exchange chromatography (HPAEC) on pig gastric mucin (PGM). However, the assay showed very low signals. Hypothesizing that the PGM’s sialic acid content was too low to give useful data, we decided to replace it with bovine submaxillary mucin (BSM). Assay tests performed with BSM as a substrate instead of PGM yielded better results. Apart from testing the industrial enzymes, we also tested our own enzyme, produced by our sialidase BioBrick. The results confirmed an activity for our enzyme.</P><P class="igem-text">To test the activity of the EBG, we tried using a colorimetric assay. When performing the assay, we observed a unusually strong signal in our negative control. This was explained by the fact that the filter used to extract the sugar residues was covered in glycerine which in itself gave a signal, even when no degradation had been accomplished. 
</P><H4>Rheology</H4><P class="igem-text">To test the properties of PGM and their change as a result of enzyme treatment, rheology was employed. With rheology, we can measure the viscoelastic properties of a given structure. Samples treated with and without mucus degrading enzymes were to be prepared.</P><P class="igem-text">For time reasons, we were not able to test the effect of our self-produced sialidase and endo-β-galactosidase on PGM. Instead, to partly prove our hypothesis of the decrease in viscoelasticity, we degraded the whole glycans from PGM samples. This was done using a β-Elimination kit, which is a non-reducing chemical reagent mixture that efficiently cleaves the O-linked glycans. Rheology measurement was performed on cleaved PGM samples and untreated PGM samples. The results were clear in this case and a decrease in glycosylation was proved to decrease the viscosity.</P><H2>References</H2><P class="igem-text">Ashida, H., Maskos, K., Li, S.-C., &amp; Li, Y.-T. (2002). Characterization of a Novel Endo-β-galactosidase Specific for Releasing the Disaccharide GlcNAcα1→4Gal from Glycoconjugates†,‡. Biochemistry, 41(7), 2388–2395. http://doi.org/10.1021/bi011940e</P><P class="igem-text">Egebjerg, J., &amp; Christensen, S. (2005). Cloning, expression and characterization of a sialidase gene from Arthrobacter ureafaciens. Biotechnology and Applied Biochemistry, 41(3), 225. http://doi.org/10.1042/ba20040144</P></DIV></SECTION></DIV><DIV class="container"><DIV class="igem-sponsors igem-sponsors-first"><H1> Sponsors </H1></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>