Team:Vilnius-Lithuania/Description
project
parts
notebook
human practices
team
Description
Describe the Impossible
Cell-free systems are becoming an increasingly popular in vitro tool to study biological processes as it is accompanied by less intrinsic and extrinsic noise. Relying on fundamental concepts of synthetic biology, we apply a bottom-up forward engineering approach to create a novel cell-free system for unorthodox protein-evolution. The core of this system is cell-sized liposomes that serve as excellent artificial membrane models. By encapsulating genetic material and full in vitro protein transcription and translation systems within the liposomes, we create reliable and incredibly efficient nanofactories for the production of target proteins. Even though there are many alternative proteins that can be synthesized, our main focus is directed towards membrane proteins, which occupy approximately one third of living-cells’ genomes. Considering their significance, membrane proteins are spectacularly understudied since synthesis and thus characterization of them remain prevailing obstacles to this day. We aim to utilize liposomes as nanofactories for directed evolution of membrane proteins. Furthermore, by means of directed membrane protein-evolution, a universal exposition system will be designed in order to display any protein of interest on the surface of the liposome. This way, a system is built where a phenotype of a particular protein is expressed on the outside while containing its genotype within the liposome. To prove the concept, small antibody fragments will be displayed to create a single-chain variable fragment (scFv) library for rapid screening of any designated target.
Read More
Description
Fig. 1 Fig. 1. Schematic overview of the SynDrop. Using microfluidic technology, we synthesize liposomes at a high throughput within which we encapsulate an in vitro transcription/translation system with DNA encoding a target membrane protein. In addition we encapsulate purified cellular membrane protein machinery and chaperones - they facilitate the insertion of synthesized target membrane proteins into the membrane. The system is also capable of displaying small molecules on the surface of the liposome, which makes SynDrop applicable for a novel liposome surface-display method.
“What I cannot create, I do not understand”
R. Feynmann, February 1988
Brief overview of the SynDrop - Synthetic Droplets for Membrane Protein Research.
SynDrop started from the idea of working towards developing a minimal synthetic cell. However it was soon realized that synthetic life is not something that will be made in one go - it will be the culmination of all the small, separate systems that will come together and work in unison. As such, it was understood that these systems need to be independently functional and well described first, before more complex systems are built on top of them. One of the most fundamental differences between life and synthetic systems is the responsivity and communication with the surrounding environment. This function in living cells is mostly performed by membrane proteins. We quickly realized that in order to make a significant impact on synthetic life development, membrane proteins are that understudied field that holds great potential for future applications in synthetic biology. Fig. 1 beautifully summarizes the workflow, complexity and at the same time minimalism of SynDrop. We utilized the emerging technology of microfluidics to synthesize cell-sized liposomes and provide them with the minimal set of all the necessary tools and machineries for the successful synthesis of membrane proteins. These fully equipped liposomes form the core of the SynDrop. Within them are encapsulated purified BAM complex proteins and the chaperone SurA which facilitate beta-barrel bearing protein assembly. Liposomes also contain genetically engineered membrane-associating ribosomes which increase the yields of target protein expression. SynDrop liposomes contain an in vitro transcription-translation system and custom DNA. Their inner aqueous environment is suitable for molecular reactions to occur. Finally, SynDrop provides a novel platform for protein display, whether they were antibodies, single chain fragments, globular proteins, or peptides. It is a huge step forward in membrane protein research and perhaps another resolved puzzle towards the creation of synthetic cell.
Background
Synthetic biologists have come a long way since 1912 Stéphane Leduc’s La Biologie Synthétique. Throughout the course of synthetic biology, there were many stepping stones that lead to greater things, such as the discovery of restriction enzymes which lead to the simplified construction of recombinant DNA molecules and arrangements of new genes, or the creation of synthetic biological circuit devices by combining different genes within E. coli. Through these canonical inventions we acquired not only better tools, but also greater ambitions. From understanding how genes work and that they can be modified or replaced, to programming cellular behavior with external impulses, we have reached a point, where no longer the singular modal elements like switches, cascades, pulse generators or oscillators matter. We have reached a point where scientists are ready to face the biggest challenge of synthetic biology - creating an artificial cell.
However designing complex, several layered circuitries resembling the behavior of a natural cell is still an overwhelming challenge due to many limitations like crosstalk, mutations, ambiguous intracellular and extracellular conditions, and biological noise. Therefore we propose to start from something simpler and more minimal.. Although the journey of creating a synthetic minimal cell has already begun, we hoped to contribute to this ultimate goal as well by investing our time and effort. This year we are engineering liposomes, lipid-coated vesicles, that are perfect models to study the initial steps for creating synthetic cells. Liposomes can offer a system with fully controllable experimental parameters and only the exact elements for our custom circuit design without the need to ever worry about the crosstalk and noise. We believe that most of the future synthetic biology a pplications will rely on bottom-up engineering solutions. Having mastered some hard-core bottom-up liposome engineering, we won’t take long to create the first synthetic cell.
Keeping that in mind we raised a question - what is the trivial difference between completely artificial systems like liposomes and living cells? The answer has pushed us to develop the SynDrop system the way it is being presented today. That major difference between synthetic and natural systems is the capacity to develop an active interface between outside and inside, which is vital for communication, transport, signaling, growth, and proliferation. Most of this communication is mediated by integral membrane proteins. Surprisingly, though membrane proteins are key players in many cellular functions and even human health and disease, they are greatly understudied when compared to e.g. globular proteins. Membrane protein integration into the membranes in vitro is still a particularly delicate issue that halts more rapid scientific development in this field. Having realized this, we knew that our goal this year was going to be the creation of an in vitro membrane protein synthesis system in liposomes in order to not only broaden the research possibilities of integral proteins but also to move one step further in building a minimal cell, which with what SynDrop offers - now will be capable of executing at least minimal .communication between its inside (genotype) and surrounding environment (display phenotype).
SynDrop - Synthetic Droplets for Membrane Protein Research
Current methods for studying MPs are mostly based on cellular systems, which are really not the best environment to study membrane proteins. Large intrinsic and extrinsic cellular noise makes it awfully difficult to characterize discrete MPs as well as other limitations like toxicity and limited yields. Only a few self-integrating membrane proteins have been utilized for research as the majority of them aggregate or are toxic to the host. The capacity to vary parameters is too restrictive for an in-depth investigation of a complete spatiotemporal arrangement of a particular protein or mechanism.
As mentioned above, to untangle these problems we decided not to try to decipher and modulate natural cells. We chose something much more simple - a cell-free synthetic system with well defined, easily controlled parameters, that could be engineered from scratch. We called it SynDrop - Synthetic Droplets for Membrane Protein Research. SynDrop edges not only cellular systems, but also other current cell-free systems in terms of membrane protein research and studying their behavior, including folding and membrane insertion.
Thus SynDrop is a novel bottom-up liposome-based platform created to empower the manipulation of membrane proteins. It is carefully though and bares the exact minimum components to successfully perform its function and not become too unpredictable or metabolically unstable. Transcriptional and translational machineries together with the plasmid DNA, chaperones, energy regeneration systems, and cellular MP insertion facilitators are encapsulated within the liposomes, enabling synthesis, integration, and display of target membrane proteins. It is also equipped with synthetic tools that regulate, attenuate and modulate the whole system.
Applications
As our project focuses on a novel platform for membrane protein research it offers various future applications.
1. Membrane protein characterization
Since liposome systems contain no unknown variables, SynDrop could be used to characterize membrane proteins and their biogenesis. This could possibly unlock new and innovative breakthrough in proteomics.
2. Membrane protein-lipid interactions
Microfluidics allow to synthetize liposomes with modifying their lipid composition rather simply. By using SynDrop we can quickly get huge quantities of liposomes with various lipid compositions. Further, membrane proteins translated in these liposomes will interact accordingly with these lipids. This characterization of membrane protein interactions with custom lipids can be modelled and then implemented in in vivo systems.
3. Membrane protein-membrane protein interactions
Expanding even further of the membrane protein interactions. The amount of genetic material that gets encapsulated in the liposome can be regulated. Therefore multiple genes coding for different membrane proteins can be translated inside liposomes. It yields multiple membrane protein-protein interactions that can be measured and modelled accordingly.
4. Molecular evolution of the exposed particle
Liposomes could be effectively used in molecular evolution. We constructed a few membrane proteins that proved to display a protein particle on the surface of bacteria. By incorporating a few membrane protein integration helpers, we believe that liposome exposition could serve as a better alternative to current molecular evolution methods.
5. Molecular evolution of the membrane protein
Molecular evolution could be also applied not only for the exposed protein particle, but also for the membrane protein itself. Since liposomes have no additional noise and contains a changeable lipid bilayer, it could be one of the most feasible methods of membrane protein evolution.
6. Membrane protein biogenesis regulation
DNA or RNA that gets encapsulated inside liposomes can be altered. Therefore we can insert modifications that change the rate and volume of membrane protein biogenesis. For example, modifications might include thermoswitches that we used in our project to regulate the rate of translation inside the liposomes, so that one membrane protein has the priority to get translated over the other. These or similar modifications might lead to a fully functional one or more genetic circuit incorporation into liposomes that regulate the biogenesis of membrane proteins.
7. Liposome communication
Microfluidic field enables the production of similar or different composition liposomes. Theoretically, these liposomes could cross-react with each other or even share signals. This could create an autonomous artificial ecosystem and could lead towards the creation of artificial life.
Background of other display systems
An extensive literature search was performed to analyze every display system type that currently exists in order to be sure about the necessity and applicability of our project.
Phage display
Library size
107
Transformation required
Yes
Mechanism
Phage display systems can be grouped into two classes: true phage vectors and phagemid vectors. In both cases, the protein to be displayed is fused to the capsid protein.
Evolution
Typically the phage library screening entails several (usually three) consecutive rounds of panning and phage amplification before the selected phage and the polypeptide that they present are individually analyzed.
Protein displayed
Fv, scFv or Fab fragments
Proteins to be displayed
Soluble, non-toxic, compatible with crossing membranes
Surface anchorage
Capsid proteins
Properties of protein enhanced
Affinity, enzymatic activity, stability, folding, selection from cDNA libraries
Stability
Stable
Applications
To enhance catalytic activity, proteolytic stability, or attach conjugation site and photoaffinity labels
Affinity selection of protein fragments expressed from cDNA fragments
Immunotherapy
Developing diagnostic or therapeutic reagents in medicine
Developing nanomaterials in material science
Identifying receptor agonists or antagonists in cell biology
Advantages
Biopanning of phage libraries on whole cells
Wide range of pH could be used for selection
Disadvantages
The low display rate can be a drawback. In the first round of panning where very few binders are to be enriched from a huge excess of unwanted phages, phages that are mostly “bald” can reduce the accessible molecular diversity of the library and the efficiency of the system
If selection is done on cells, non specific phage binding is observed, due to phage protein interaction with cell membrane proteins
In phage display, libraries must be transformed into bacteria, limiting the number of possible independent sequences to 109–1010
Membrane protein research
Compatible
Nogo-66 - monotopic membrane protein (7,5 kDa)
ShuA - beta barrel forming protein, which pass through membrane 22 times (72.5 kDa)
MOMP - beta barrel forming protein, which pass through membrane 16 times (40 kDa)
Nef - peripheral protein (23 kDa)
Neuromodulin - peripheral protein (25 kDa)
Caveolin-1 - monotopic membrane protein (22 kDa)
References
1. Imai, S. et al. Development of an antibody proteomics system using a phage antibody library for efficient screening of biomarker proteins. Biomaterials 32, 162-169 (2011). 
2. Petrenko, V. Evolution of phage display: from bioactive peptides to bioselective nanomaterials. Expert Opinion on Drug Delivery 5, 825-836 (2008).
3. Paschke, M. Phage display systems and their applications. Applied Microbiology and Biotechnology 70, 2-11 (2005).
4. Reiersen, H. Covalent antibody display--an in vitro antibody-DNA library selection system. Nucleic Acids Research 33, 10 (2005).
5. Vithayathil, R., Hooy, R., Cocco, M. & Weiss, G. The Scope of Phage Display for Membrane Proteins. Journal of Molecular Biology 414, 499-510 (2011).
6. Morrison, K. & Weiss, G. Combinatorial alanine-scanning. Current Opinion in Chemical Biology 5, 302-307 (2001).
7. Clackson, T. & Wells, J. In vitro selection from protein and peptide libraries. Trends in Biotechnology 12, 173-184 (1994).
8. Galán, A. et al. Library-based display technologies: where do we stand?. Molecular BioSystems 12, 2342-2358 (2016).
Ribosome Display
Library size
1013-14
Transformation required
No
Mechanism
DNA sequence of protein does not have a STOP codon and features a linker sequence at the C-terminus. Instead of detaching, the transcribed RNA and translated protein remain connected to the ribosome
Evolution
High affinity binders can be generated after 1st round of selection, but most of the time, multiple rounds are done
Protein displayed
Antibody scFv fragments, "Designed Ankyrin Repeat Proteins"- DARPins, camelid nanobodies, DNA-binding proteins, receptors, membrane proteins
Proteins to be displayed
Most proteins including cytotoxic, chemically modified and membrane proteins
Surface anchorage
Ribosome
Properties of protein enhanced
Affinity, enzymatic activity (used rarely; water-in-oil emulsions have more advantages in this field), stability
Stability
Stable in Mg2+
Applications
Cancer treatment, antibody engineering, proteomics, diagnostics and therapeutics
Advantages
Fast generation of big library as the complex is stabilized by Mg2+ and can be readily dissociated by the addition of EDTA
A signiﬁcant advantage of in vitro translation methods is the ability to modify the genetic code to allow the incorporation of non-canonical, unnatural amino acids, to give molecules with novel properties, such as cyclic peptides with increased serum, stability
Rapid isolation and direct evolution of high-affinity functional proteins, particularly antibodies
Disadvantages
By introducing mutations during PCR, a stop codon might appear in the middle of the sequence - the protein is unable to fold properly and mutant is lost. Moreover, RNA is less stable than DNA and requires an RNAse free environment, otherwise the majority of RNA will be degraded before reverse transcribed to DNA
 In order to incorporate non canonical amino acids, DNA sequence has to be modified by introducing amber codons and specifically modified ribosomes have to be used
Stem loops have to be added to the 5' and 3' ends, to stabilize RNA and a linker at the 3' end before the loops
Polysomes can reduce the library size, because the protein is not formed and aggregates.
Membrane protein research
Incompatible. Membrane proteins are too hydrophobic and will not function if displayed. No pore forming activity or transfer activity would be evaluated
References
1. Lipovsek, D. & Plückthun, A. In-vitro protein evolution by ribosome display and mRNA display. Journal of Immunological Methods 290, 51-67 (2004).
2. He, M. & Khan, F. Ribosome display: next-generation display technologies for production of antibodies in vitro. Expert Review of Proteomics 2, 421-430 (2005).
3. Zahnd, C., Amstutz, P. & Plückthun, A. Ribosome display: selecting and evolving proteins in vitro that specifically bind to a target. Nature Methods 4, 269-279 (2007).
Cis-Activity (Cis) Display
Library size
1010
Transformation required
No
Mechanism
Replication initiator protein (RepA) binds to the DNA template from which it has been expressed, a property called cis-activity. The protein of interest is binded to RepA protein C-terminus.
Protein displayed
Antibody scFv fragments
Proteins to be displayed
Soluble, including cytotoxic, chemically modified
Surface anchorage
In vitro
Properties of protein enhanced
Affinity, stability, resistance to degradation, longer half-life
Stability
Stable (>2 days)
Applications
For the selection of high affinity peptides and folded protein domains, including antibody fragments
Advantages
Does not require any compartmentalization of the library-encoding nucleic acid
The use of DNA to encode the displayed peptides provides advantages over RNA-based in vitro selection methods
Very large libraries can be rapidly constructed and screened without separating transcription and translation steps and without purification of the protein–DNA complexes before selection
Complexes do not require incubation under sterile or ribonuclease-free conditions
Optimal ligands after 3-5 successive rounds
References
Odegrip, R. et al. CIS display: In vitro selection of peptides from libraries of protein-DNA complexes. Proceedings of the National Academy of Sciences 101, 2806-2810 (2004).
mRNA Display
Library size
1014
Transformation required
No
Mechanism
Puromycin with a DNA linker is ligated to RNA after mRNA library generation. Translation is carried out and the puromycin molecule enters the P-site of ribosome. It takes the role of tRNA, which leads to the growing peptide chain being covalently connected to the puromycin molecule.
Evolution
To produce mRNA-displayed proteins requires ~3 days, to subject them to selection and evolution of enzymes for bond-forming reactions requires ~4-10 weeks
Protein displayed
Antibody scFv, RasIn1 and RasIn2 proteins
Proteins to be displayed
Soluble, including cytotoxic, chemically modified
Surface anchorage
In vitro
Properties of protein enhanced
Affinity, stability
Stability
Covalent
Applications
Investigation of protein-protein interactions and the development and selection of peptides, enzymes, scFvs, and novel binders based on alternative scaffolds
Protein evolution
Advantages
Useful for performing in vitro selections in harsh environments that are not compatible with ribosome display. If the RNA stability is of concern in certain environments, the RNA can be replaced by its cDNA, as described in a variation of the mRNA display
The mRNA display system allows libraries with sequence complexity approximately 10,000-fold that of phage display, 106-fold over yeast display or yeast two- and three- hybrid systems, and approximately 109-fold over colony screening approaches
Disadvantages
Time consuming, technically demanding
mRNA display has only been used for bond-forming enzymes (not to select enzymes that catalyze bond-breaking reactions or other covalent modification reactions)
Membrane protein research
Incompatible. Due to the limited expression of membrane-bound proteins by in vitro translation systems, mRNA-display cannot be utilized to address membrane protein related questions
References
1. Takahashi, T., Austin, R. & Roberts, R. mRNA display: ligand discovery, interaction analysis and beyond. Trends in Biochemical Sciences 28, 159-165 (2003). 
2. Seelig, B. mRNA display for the selection and evolution of enzymes from in vitro-translated protein libraries. Nature Protocols 6, 540-552 (2011). 
3. Cetin, M. et al. RasIns: Genetically Encoded Intrabodies of Activated Ras Proteins. Journal of Molecular Biology 429, 562-573 (2017).
4. Lin, H. & Cornish, V. Screening and Selection Methods for Large-Scale Analysis of Protein Function. ChemInform 34, (2003).
Covalent Antibody Display
Library size
107
Transformation required
No
Mechanism
A fusion protein - P2A and an scFv antibody bind to the same molecule of DNA from which it has been expressed. Following in vitro coupled transcription and translation, the P2A protein makes a covalent link between scFv genotype and scFv phenotype, by producing a stable protein–DNA complex
Evolution
Fast cycle time: in a few hours, unique scFvs can be enriched, isolated and directly amplified for the next rounds of selection
Protein displayed
Antibody scFv fragments
Proteins to be displayed
Developed for antibody display
Surface anchorage
In vitro
Properties of protein enhanced
Affinity selections
Stability
Covalent
Applications
Advantages
scFv can be genetically fused to the P2A protein creating the smallest imaginable antibody selection particle: a protein and its gene
CAD is the only antibody selection method providing a direct covalent link between an scFv gene and its protein, increasing the chemical stability of the panning complex
Disadvantages
Product inhibition of uncomplexed fusion proteins without DNA
Membrane protein research
Incompatible - only used for antibody display and no membrane proteins can be evolved with this system
References
Reiersen, H. Covalent antibody display--an in vitro antibody-DNA library selection system. Nucleic Acids Research 33, 10 (2005).
Yeast Explay
Library size
Up to 1014
Transformation required
Yes
Mechanism
The most common yeast display system employs fusion of the protein of interest to the C-terminus of the Aga2p subunit. Induction of protein expression results in surface display of the fusion protein through disulfide bond formation of Aga2p to the β1,6-glucan-anchored Aga1p domain of agglutinin. The epitope tags allow quantification of fusion protein expression, and thus normalization of protein function to expression level by flow cytometry using fluorescently labeled antibodies
Protein displayed
Antibody scFv fragments, Fab, single-chain T cell receptors, major histocompatibility complex (MHC)
Proteins to be displayed
Soluble and membrane, nontoxic, compatible with crossing membranes
Surface anchorage
Agglutination proteins, flocculation proteins
Properties of protein enhanced
Affinity, specificity
Applications
Indispensable and efficient way for affinity maturation in antibody engineering
A method to immobilize enzymes and pathogen-derived proteins for vaccine development  
Advantages
Yeast as eukaryotic microorganism owns post-translational modifications as well as processing machinery and conditions homologous to the mammals
Possesses eukaryotic machinery to express, assemble and export onto the surface both monomeric and oligomeric proteins in a fully native-like state
In a direct comparison with phage display using the same antibody library and target, the yeast system sampled the library repertoire more comprehensively
Compatibility with flow cytometric analysis, which allows quantitative measurements of equilibrium binding constants, dissociation kinetics, stability, and specificity of the displayed proteins without the laborious requirements of soluble protein expression and purification
Disadvantages
The potential drawbacks are similar to phage display in that the expression of extracellular proteins is favoured and affinity maturation is complicated by avidity effects due to the multiplicity of the displayed peptides or proteins on the cell surface
Membrane protein research
Compatible
References
1. van Rosmalen, M. et al. Affinity Maturation of a Cyclic Peptide Handle for Therapeutic Antibodies Using Deep Mutational Scanning. Journal of Biological Chemistry 292, 1477-1489 (2016). 
2. Gai, S. & Wittrup, K. Yeast surface display for protein engineering and characterization. Current Opinion in Structural Biology 17, 467-473 (2007). 
3. Bowley, D., Labrijn, A., Zwick, M. & Burton, D. Antigen selection from an HIV-1 immune antibody library displayed on yeast yields many novel antibodies compared to selection from the same library displayed on phage. Protein Engineering, Design and Selection 20, 81-90 (2007).     
4. Shusta, E., Pepper, L., Cho, Y. & Boder, E. A Decade of Yeast Surface Display Technology: Where Are We Now?. Combinatorial Chemistry & High Throughput Screening 11, 127-134 (2008).
Eukaryotic Display
Library size
107
Transformation required
Yes
Mechanism
For the purpose of displaying foreign proteins on the surface of baculovirus particles as well as on infected insect cells, gp64 serve as a fusion partner that together with a chosen target protein gets incorporated into the cell membrane
Proteins to be displayed
Soluble and membrane, nontoxic, compatible with crossing membranes
Surface anchorage
Agglutination proteins, flocculation proteins
Properties of protein enhanced
Increased binding affinity and improved catalytic properties
Applications
Therapeutics
Eukaryotic surface display libraries based on the baculovirus system allow selecting for specific binding proteins while providing post translational modifications
Baculoviruses are non-pathogenic to mammals and thus, are an ideal carrier for antigenic epitopes or proteins intended to induce neutralizing immune response when administered as vaccines
Eukaryotic expression libraries are a powerful tool for a variety of applications such as finding new ligands, identification of cellular interaction partners and affinity maturation of antibody and antibody fragments
Advantages
The mammalian expression system is inherently competent for processing and secreting eukaryotic proteins. Consequently, mammalian cell display offers vast potential to display full-length human antibodies;
Insect cells might be more feasible to express complex protein structures than e.g. yeast;
As compared to mammalian cell lines, insect cells such as Sf9 and T. ni cells may be considered as a more robust and efficient display scaffold, since handling is less laborious, less costly and target proteins are incorporated more efficiently and homogeneously into the cellular membrane;
Selection of specific clones by FACS is a fast and attractive method and does not require the cells to stay intact, but instead baculoviruses can be recovered by filtration and be used for re-infection
Disadvantages
An obvious challenge remains to be solved that relatively low transformation efficiency of mammalian cells diminishes the actual repertoire size in contrast with phage display, leading to unlikely straightforward isolation of  antibodies with remarkable affinity. Moreover, the mammalian cell proliferation rate is slower, and such cells require more specific culture conditions in vitro than microbial cells. These drawbacks necessitate great efforts to improve the mammalian cell display platform.
Membrane protein research
Compatible
References
Oker-Blom, C. Baculovirus display strategies: Emerging tools for eukaryotic libraries and gene delivery. Briefings in Functional Genomics and Proteomics 2, 244-253 (2003).
Water-In-Oil Emulsions
Library size
1010
Transformation required
No
Mechanism
Experimental conditions are adjusted so that, in most cases, one compartment contains one DNA molecule. The DNA fragments encode fusion proteins containing a DNA-methyltransferase, which can form a covalent bond with a 5-fluoro deoxycytidine base at the extremity of the DNA fragment. The resulting library of DNA–protein fusions is extracted from the emulsion and DNA molecules displaying a protein with desired binding properties are selected from the pool of DNA–protein fusions by affinity panning on target antigens.
Typical enrichment factor per round
>1000
Protein displayed
In vitro compartmentalization  using water-in-oil emulsions has been used for the selection of peptide ligands and for the directed evolution of DNA methyltransferases, bacterial phosphotriesterase, Taq polymerase, luciferase and human telomerase.
Proteins to be displayed
-
Surface anchorage
-
Properties of protein enhanced
Enzymatic activity, thermal stability
Stability
Stable
Applications
Protein evolution, to enhance their enzymatic activity, and thermal stability.
Advantages
Since DNA is added at limiting dilution, each aqueous droplet contains a single gene, and acts as a unique, independent reaction vessel
Disadvantages
Many proteins of interest, in particular more complex, multidomain proteins are more efficiently expressed in eukaryotic expression systems
More than one DNA molecule might be present in one water compartment of the water-in-oil emulsion, which would lead to the incorrect assignment of genotype and phenotype and, therefore, to the selection of DNA molecules coding for non-binding proteins
Membrane protein research
Incompatible. Membrane proteins cannot be reconstituted into surfactant layer of w/o emulsion
References
Ghadessy, F. & Holliger, P. A novel emulsion mixture for in vitro compartmentalization of transcription and translation in the rabbit reticulocyte system. Protein Engineering, Design and Selection 17, 201-204 (2004).
Bertschinger, J. & Neri, D. Covalent DNA display as a novel tool for directed evolution of proteins in vitro. Protein Engineering Design and Selection 17, 699-707 (2004).
Liposome Display
Library size
107
Transformation required
No
Mechanism
Membrane protein is inserted to lipid bilayer. Protein can insert by himself or with the help of BAM complex and additional helper proteins
Evolution
Single round of selection can be achieved within 1 day
Protein displayed
Alpha-hemolysin pore forming protein, G protein-coupled receptor, caveolin
Proteins to be displayed
Soluble and membrane, non-toxic, toxic, compatible with crossing membranes
Surface anchorage
Membrane proteins
Properties of protein enhanced
Pore-forming activity, enzymatic activity, ...
Stability
Stable
Applications
Membrane protein evolution, therapeutics
Advantages
In vitro translation system produces proteins without using living cells. One DNA sequence can be trapped in one liposome and toxic, bacterial growth inhibiting protein can be translated
Rapid and efficient mutant selection
Proteins that affect cell growth can also be engineered because the protein is synthesized in vitro
Conditions appropriate for the target membrane protein can be adopted because the lipid composition of the liposomes is adjustable and internal solution can be adjusted as needed
No membrane proteins other than the target one are present
Disadvantages
Size variety of liposomes reduce enrichment factor if multiple genes are trapped inside one big lisopome
Membrane protein research
Alpha-hemolysin was translated inside liposome, self-inserted to membrane and formed pore. By doing evolution of this protein, mutants showing better pore forming and self-inserting properties were selected by FACS
References
1.Fujii, S., Matsuura, T., Sunami, T., Kazuta, Y. & Yomo, T. In vitro evolution of  alpha-hemolysin using a liposome display. Proceedings of the National Academy of Sciences 110, 16796-16801 (2013).
2.Goodsell, D. & Olson, A. Structural Symmetry and Protein Function. Annual Review of Biophysics and Biomolecular Structure 29, 105-153 (2000).
invert
