<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania-OG_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania-OG/Design</SPAN></H1></DIV></DIV></DIV><TITLE>Collaborations</TITLE><DIV id="" class="clearfix"><HEADER id="header" class="page-section full-header"><DIV id="header-wrap" style="padding-left:1%"><DIV class="clearfix"><NAV id="primary-menu"><UL><LI id="homeheader"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG">Home</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Project</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design_Regulatory">Design for Regulatory Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Model">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Demonstrate">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Nucleotide_Design">Substrate Design Prospects</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Entrepreneurship">Entrepreneurship</A></LI></UL><LI id="softwareh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Gan_Introduction">Generative Adversarial Network – GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ProteinGAN">Protein GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ReactionGAN">Reaction GAN</A></LI><LI id="partsh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-basic">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-composite">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-collection">Collection</A></LI><LI id="notebookh"><A href="/pages/notebook/index.html">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/catseqguide">Using the CAT-Seq</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/labnotebook">Lab Journal</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Safety">Safety</A></LI><LI id="humanh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Collaborations">Collaborations</A></LI><LI id="peopleh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">People</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">Team</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Sponsors">Sponsors</A></LI></NAV></DIV></DIV></HEADER><SECTION id="slider" class="slider-element slider-parallax full-screen force-full-screen with-header swiper_wrapper page-section clearfix"><DIV class="slider-parallax-inner"><DIV class="swiper-container swiper-parent"><DIV class="swiper-wrapper"><DIV class="swiper-slide" style="background-image: url('https://static.igem.org/mediawiki/2018/3/3f/T--Vilnius-Lithuania-OG--design.jpg');"><DIV class="container clearfix"><DIV class="slider-caption slider-caption-center"><H2 data-caption-animate="fadeInUp">Design</H2></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><DIV id="page-menu"><DIV id="page-menu-wrap"><DIV class="container clearfix"><DIV class="menu-title">Explore <SPAN>DESIGN</SPAN></DIV><NAV class="one-page-menu"><UL><LI><A href="#" style="font-size: 90%" data-href="#section-introduction">Introduction</A></LI><LI><A href="#" style="font-size: 90%" data-href="#section-library">Library</A></LI><LI><A href="#" style="font-size: 90%" data-href="#section-encapsulation">Encapsulation</A></LI><LI><A href="#" style="font-size: 90%" data-href="#section-production">Biomolecule Production</A></LI><LI><A href="#" style="font-size: 90%" data-href="#section-conversion">Substrate Conversion</A></LI><LI><A href="#" style="font-size: 90%" data-href="#section-recording">Recording</A></LI><LI><A href="#" style="font-size: 90%" data-href="#section-reading">Reading</A></LI></UL></NAV></DIV></DIV></DIV><SECTION style="overflow:hidden; position:relative; background-color: #FFF;"><DIV class="content-wrap"><DIV class="single-post nobottommargin"><DIV class="container clearfix"><SECTION id="section-introduction" class="page-section"><DIV class="tekstodydis"><DIV class="fancy-title title-border-color"><H2>Introduction</H2></DIV><P>In this section you will find out what is <STRONG>Catalytic Activity Sequencing</STRONG> (CAT-Seq). Firstly, the CAT-Seq is introduced in brief to give a quick overview of how the system works and what are the different parts it consists of. Next, every step of the system is described and illustrated in detail. Finally, we show how we have applied this design in order to build the proof of concept of CAT-Seq.</P><P>It should be noted that only the final form of the system design is described here. Yet, the design of a new method is a highly <STRONG>dynamic </STRONG>process which requires an immense amount of theoretical and practical iterations that closely follow the main principles of engineering. Therefore, in order to see the <STRONG>CAT-Seq system engineering story</STRONG> - how it began, how early designs have looked, how and why did the system design change during the whole project year and what influenced those changes - please head <STRONG>to the Human Practices section</STRONG>!</P><P>CAT-Seq allows to <STRONG>rapidly </STRONG>record the information about billions catalytic biomolecule activities<STRONG> in parallel</STRONG>. Each measured activity is then <STRONG>simultaneously </STRONG>written into DNA sequence encoding the particular biomolecule. After the recording, both the biomolecule sequence and it’s activity can be read using <STRONG>nanopore sequencing</STRONG>.</P><P>Currently, droplet microfluidics offer an efficient and high-throughput way to parallelise high number of reactions. As each molecule is encapsulated in a unique water droplet, various physically separated reactions can be carried out at the same time. Yet, what is <STRONG>missing</STRONG>, is a way to record the activity information in each droplet, so that after the droplets are collected together and broken, the sequence and function relationship is not lost. In other words, the main goal of CAT-Seq is to <STRONG>connect catalytic biomolecule’s phenotype to genotype in ultra-high throughput manner</STRONG>.</P><P>During the theoretical design phase it became apparent that recording such information in mutant’s own DNA sequence from which it was synthesized would be the smartest choice. That is because <STRONG>DNA already contains half of the information</STRONG> we want to save - the sequence that codes the biomolecule. As we want to associate the sequence information with the information of catalytic activity, there did not seem to be a better object than the <STRONG>DNA</STRONG>.</P><H4 class="excepth4">But how can we write such information into the DNA? Substrate Nucleotides!</H4><P>While navigating the vast sea of scientific literature we have stumbled upon a fact that some polymerases do not incorporate nucleotides into DNA sequence <STRONG>if they are modified</STRONG>. We have then come up with an idea to <STRONG>attach the enzyme substrate to the nucleotides </STRONG>and term this compound<STRONG> “Substrate nucleotide”.</STRONG></P><P>Such modified nucleotides then act as <STRONG>a target for various catalytic biomolecules</STRONG> - they would first need to recognise it, bind to it and then chemically convert it, resulting in the removal of the substrate. We call this chemically converted nucleotide a “<STRONG>Product Nucleotide</STRONG>”.</P><P>The removal of the different substrates can be achieved by different types of catalytic biomolecules that catalyse a vast range of different chemical reactions. For example, all of the six major classes of enzymes can be used. To explore the prospects of recording the activities of different types of catalytic biomolecules, please head to the <A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Nucleotide_Design"><STRONG>Substrate Design Prospects</STRONG></A> page!</P><P>The catalytic molecule acting on the<STRONG> substrate nucleotide</STRONG> is the main principle behind CAT-Seq. A substrate nucleotide <STRONG>cannot be incorporated</STRONG> by DNA polymerase. In contrast - <STRONG>Product Nucleotide can </STRONG>be incorporated.</P><P>Yet, such concept alone is not enough to record catalytic activities of the biomolecules. What else is needed? Let’s dive right in.</P><DIV class="fancy-title title-border-color"><H2>CAT-Seq design</H2></DIV><P>Every part of the CAT-Seq design page is divided in two main sections: </P><OL class="sarasas"><LI class="listukas"><STRONG>Design description</STRONG> - an in depth explanation of how a particular part of our system works, how it should be used in real-life situation and why such design choices were adopted. </LI><LI class="listukas"><STRONG>Proof of Concept</STRONG> -  a story about how we applied the design using specific components and parts to build CAT-Seq and demonstrate that it works as intended.</LI></OL><H3>Design Overview</H3><H3>Design description</H3><SECTION id="section-library" class="page-section"><DIV class="fancy-title title-border-color"><H4 class="didesnish4">1. Library preparation</H4></DIV><P class="didesnist"><STRONG>General description</STRONG></P><P>A library of is prepared <STRONG>(1)</STRONG> for a specific or group of catalytic biomolecules - <STRONG>enzymes</STRONG>, <STRONG>catalytic RNAs</STRONG> or <STRONG>catalytic DNAs</STRONG>. For example, such library may consist of billions of randomly mutated enzyme sequences, a few computationally derived enzymes or a <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="Metagenomics is the study of genetic material recovered directly from environmental samples">metagenomic</A> enzyme library.</P><P>The next step, after building the library, is adding the essential genetic <STRONG>regulatory parts</STRONG> - <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="a promoter is a region of DNA that initiates transcription of a particular gene">RNA polymerase promoter</A> and a <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="A ribosome binding site is a sequence of nucleotides upstream of the start codon of an mRNA transcript that is responsible for the recruitment of a ribosome during the initiation of protein translation">ribosome binding site</A><STRONG>(2)</STRONG>. These parts are necessary and sufficient to synthesize the biomolecules in both cells and in-vitro. For example, this can be done by ligating the regulatory parts onto DNA fragments or by using overhang PCR, which adds additional sequences to the end of your amplified product.</P><P>After the regulatory parts are added the fragments must then be <STRONG>circularized (3)</STRONG>. DNA fragments are circularized for two main reasons:</P><P class="listin">1.	First of all, in order to amplify the DNA inside microfluidic droplets an <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="Isothermal amplification reactions covers multiple methods which utilize the amplification reaction at a single-temperature, usually are extremely fast and do not require thermocyclers">isothermal amplification reaction</A> must be used. That is because droplets become highly unstable and can break easily when the temperature is quickly altered - just like during the popular polymerase chain reaction (PCR). If droplets would break during the initial stages of amplification, all of the information from different droplets would get mixed in the tube and the knowledge about which activity belongs to which sequence would be lost, as it was not yet recorded by the means of amplification (before the amplification, the activity measurement information is in form of modified substrate nucleotides).        That is why CAT-Seq uses a multiple displacement amplification (MDA), which is a non-thermocycling (isothermal) based DNA amplification technique. MDA was chosen because of its ability to rapidly amplify single templates of DNA into long double-stranded molecules which contain multiple repeats of the starting template. That said, one of the core requirements of MDA reaction is that starting DNA template <STRONG>must be circular</STRONG>.</P><P class="listin">2.	Secondly, as seen in the later steps of CAT-Seq, in order to efficiently express a catalytic biomolecule in droplets, <STRONG>Rolling Cycle Transcription</STRONG> (described in catalytic biomolecule production section) is used, which immensely increases mRNA synthesis yield from a single DNA template. Such technique also <STRONG>requires the DNA to be circular</STRONG>, as an RNA polymerase travels around it in circles producing mRNA.</P><P class="didesnist"><STRONG>Proof of concept</STRONG></P><P>Choosing the first catalytic biomolecule for creating and optimizing CAT-Seq system was a classical <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="Which came first: the chicken or the egg? The dilemma stems from the observation that all chickens hatch from eggs and all chicken eggs are laid by chickens">“chicken or the egg”</A> paradox. At the very beginning, when system design was purely theoretical and we have been preparing to build it, the main question we had was - what is the combination of substrate nucleotide and catalytic biomolecules we should use, in order to confidently know if our system or its parts are working as intended?</P><P>For example, if we would have tried to build CAT-Seq using a random library of different catalytic biomolecules and at the same time tested the system performance, we would have no idea whether if it’s the catalytic biomolecule and substrate nucleotide combination that is influencing our positive or negative results, or is it the CAT-Seq method itself. In other words, we had two groups of variables. For that reason, we wanted to reduce the variables to one group - system performance variables.</P><P>In order to achieve that, we have decided to use standard, already well established low-throughput methods to find a particular catalytic biomolecule and a substrate nucleotide pair. After that, we can then use that well-characterized catalytic biomolecule and substrate pair to build and optimize CAT-Seq.</P><P><STRONG>Obtaining the first catalytic biomolecule</STRONG></P><P>In order to find a correct combination of Substrate Nucleotide and the Catalytic Biomolecule that can catalyze the removal of the substrate, we have decided to look for a hydrolase. The is because a direct removal of the substrate from nucleotide seemed like the most timewise efficient choice.</P><P>    First, we synthesized a cytidine which had a benzoyl substrate attached to it: <STRONG>N4-benzoyl-2'-deoxycytidine triphosphate</STRONG>. Instructions on how such substrate nucleotide can be synthesized can be found here.</P><P>Then, an <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="An esterase is a hydrolase enzyme that splits esters into an acid and an alcohol in a chemical reaction with water called hydrolysis">esterase</A> library was screened manually using microwell plates in order to find the hydrolase enzyme which can catalyse the removal of the substrate. Once such enzyme was found and its sequence was determined. We have then used this pair of <STRONG>esterase enzyme</STRONG> and <STRONG>substrate cytidine</STRONG> to build the first version of CAT-Seq.</P><P>Next, we have performed an in-silico modelling of our Esterase enzyme.  The goal of the in-silico analysis was to create the Esterase mutants which would have different catalytic activities. Once such mutants were constructed, their relative catalytic activities were measured again using the same standard methods we have used to find the initial esterase enzyme. This was done in order to construct a small Esterase mutant library which would have precisely-defined catalytic activities by already established low-throughput measuring methods. Then, with CAT-Seq, we can also measure the activities of those mutants in high-throughput, which can then be <STRONG>compared to our previous measurements</STRONG> to confirm that CAT-Seq system is working as intended and can precisely discriminate different mutant activities.</P><P>    After acquiring the esterase enzyme and its mutants, <STRONG>regulatory sequences </STRONG>- RNA polymerase promoter and ribosome binding site -  were added to each of the library members. Then, the fragments were <STRONG>circularized </STRONG>for the MDA and RCT reactions.</P><P><STRONG>After acquiring the initial esterase for building CAT-Seq, why were the mutations added manually and not randomly?</STRONG></P><P>The goal of the in-silico modelling of the esterase was to determine mutants that would have different catalytic activities. If we had a randomized library of esterase mutants, it would have been difficult to precisely tune CAT-Seq parts as we would not be sure what output of the system we should expect. In contrast, having mutants with precisely measured catalytic activities beforehand using well-established and precise low-throughput methods enables us to recognise how well our system is performing. </P><P>    For example, if <STRONG>mutant A</STRONG> was determined to be a little bit <STRONG>more active then mutant B</STRONG>, CAT-Seq would have to be <STRONG>sensitive enough</STRONG> to discriminate the difference and also <STRONG>precise enough</STRONG> to determine that mutant A is indeed more active then mutant B.</P><SECTION id="section-encapsulation" class="page-section"><DIV class="fancy-title title-border-color"><H4 class="didesnish4">2. Library encapsulation into droplets</H4></DIV><P class="didesnist"><STRONG>General description</STRONG></P><P>For the precise measurement of multiple biomolecule’s catalytic activities they need to be physically separated. For example, if we would want to measure the activities of two mutants, we probably would not mix them together - we would measure them in separate tubes. The same logic applies with bigger libraries. Droplet microfluidics offers a way to quickly parallelize billions of reactions by physically separating reactants in pico- to nano-liter water droplets.</P><P>Using a microfluidic device chip whole DNA library can be encapsulated into the droplets, together with specific reaction reagents. Water phase flows through the microfluidic device channels with the help of pressure created by special pumps.</P><P>As water passes through oil flow, it breaks into defined size droplets. Their size depends on dimensions of the device, water and oil flow rates. On average,<STRONG> 300 000</STRONG> of droplets can be generated in <STRONG>under 10 minutes</STRONG>. Generated droplets are collected into a simple 1.5mL tube and can be incubated at a required temperature or stored in a fridge.</P><P>        In CAT-Seq, multiple of things must be encapsulated into the droplets:</P><P class="listin"><STRONG>1. Catalytic Biomolecule Library</STRONG> - in the form of nucleic acids or cells that encode a particular catalytic biomolecule library.</P><P class="listin"><STRONG>2. Reagents for biomolecule production - </STRONG> described with more details in later sections.</P><P class="listin"><STRONG>3. Substrate Nucleotides</STRONG> - the target for catalytic biomolecule. It can be attached to any of the four <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="dCTP, dGTP, dATP, dTTP">dNTPs</A>. All four different nucleotides can be used in order to record the activity information using 4 different substrates.The goal of the catalytic biomolecule is to recognise the substrate and catalyze a particular chemical conversion which depends on the substrate and biomolecules type. In many cases such conversion helps the substrate to detach from the nucleotide. Such detachment can be achieved by the first catalyzed reaction or subsequent spontaneous reactions that follow the initial one.</P><P>Please see the Methods and Protocols section for a more detailed description of microfluidics techniques.</P><P class="didesnist"><STRONG>Proof of concept</STRONG></P><P>As we have already chosen a combination of esterase enzyme and a substrate nucleotides, our initial encapsulation consisted of following:</P><P class="listin"><STRONG>1. Esterase mutant library </STRONG>created with the help of in-silico modelling.</P><P class="listin"><STRONG>2. In-vitro transcription and translation (IVTT) mix</STRONG> for catalytic biomolecule production.</P><P class="listin"><STRONG>3. Substrate Nucleotides</STRONG> (N4-benzoyl-2'-deoxycytidine triphosphates) as the targets for the esterase enzyme. </P><P>How such encapsulation looks can be seen in the video below (Video slowed down 2000x):</P><P style="margin-top: 5%;">After the encapsulation each droplet on average contains single copy of a mutant DNA sequence from the Esterase library. Also every droplet contains the same amount of IVTT reagents and Substrate Nucleotides. We call these droplets the <STRONG>Substrate Droplets</STRONG>.</P><DIV class="toggle toggle-bg"><DIV class="togglet"><STRONG>After encapsulation, does every droplet contain only a single DNA fragment from the library?</STRONG></DIV><DIV class="togglec"><P>In order to ensure the fidelity of the data collected using CAT-Seq only every 10th droplet houses a DNA template after droplet generation. If the concentration of DNA molecules in the starting encapsulation mix is too high, the likelihood of encapsulating multiple mutant templates per droplet increases drastically. Such events disrupt the data acquired by CAT-Seq or any droplet microfluidics technique. Encapsulation events can be precisely described by Poisson distribution:</P><P>P(X = k) is the probability for the single droplet to house k DNA molecules. λ shows the average number of DNA molecules per droplet.  For example, if you want to have a single template per droplets, you might choose λ to be 1. The Poisson distribution predicts, that 36.7% of droplets will have one molecule. However, it also predicts that 18.4% of droplets will house 2 molecules and 6.1% - 3 molecules. Following this notation, In order to avoid artificial data, caused by multiple DNA templates in one droplet, the concentration of the DNA template was kept at 1 per 10 droplets (λ = 0.1).  As seen from the graph, 90.4% of droplets will be empty and 9% will have 1 template. On the other hand, 0.4%. of droplets will house 2 DNA molecules, meaning that droplets with 2 DNA templates are generated rarely</P></DIV></DIV><SECTION id="section-production" class="page-section"><DIV class="fancy-title title-border-color"><H4 class="didesnish4">3. Catalytic biomolecule production</H4></DIV><P class="didesnist"><STRONG>General description</STRONG></P><P> Once the encapsulation is finished, biomolecules are then produced in the droplets. In principle, there are <STRONG>two major approaches of producing</STRONG> catalytic biomolecule in the CAT-Seq system:</P><P class="listin">1. <STRONG> Nucleic Acids</STRONG>: encapsulating DNA or RNA molecules encoding the catalytic biomolecule. After the encapsulation the droplets can be incubated in order to produce the biomolecule. If the catalytic biomolecule is an enzyme, <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="the production of protein using biological machinery in a cell-free system, that is, without the use of living cells">in-vitro protein synthesis</A> (<B>A</B>) kit can be used in order to produce it. Otherwise, if it is a <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="RNA molecule capable of catalyzing a chemical reaction">Ribozyme</A> (<B>B</B>), it can be produced using <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="the production of RNA using biological machinery in a cell-free system, that is, without the use of living cells">in-vitro transcription</A> reagents. Also, there are cases where the catalytic biomolecule is a <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="DNA molecule capable of catalyzing a chemical reaction">Deoxyribozyme</A> (<B>C</B>). Then, all that is required is to encapsulate the DNA library!</P><P>    During one of the collaborative meetings with experts from microfluidic technology field in Life Sciences Center located in Vilnius, we have received information that synthesizing biomolecules from a single DNA template is an <STRONG>extremely inefficient process</STRONG>. That is because when there is only a single  template of DNA, the mRNA synthesis yield is extremely low and there is <STRONG>not enough mRNA </STRONG>to make an enzyme.</P><P>After doing in-depth literature research to try to solve this problem, we have found out that around 1990s people immensely increased short microRNA synthesis yields by circularizing DNA templates. As RNA polymerase goes around the DNA in circles it produces long strands of RNA molecules. Within those long molecules were the repeating small units of microRNA. They coined this method <STRONG>Rolling Circle Transcription</STRONG> (RCT). </P><P>Following this discovery, we have figured out we can apply <STRONG>RCT </STRONG>in order to fix the current fundamental issues with biomolecule expression from single DNA templates in the field of droplet microtechnologies. As we are already circularizing DNA molecules in order to perform the MDA reaction in later steps, all that is left to do is to <STRONG>remove the transcription terminator</STRONG> from the catalytic biomolecule library members. Such removal of the terminator highly increases the mRNA concentration from a single template. Therefore, enzymes can then be <STRONG>efficiently synthesized</STRONG> using a <STRONG>single template of DNA</STRONG>.</P><P class="listin">2. <STRONG> Cells</STRONG>: catalytic biomolecules can also be provided into droplets in the form of cells, which have the DNA library <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="transformation is the genetic alteration of a cell resulting from the direct uptake and incorporation of genetic material from its surroundings through the cell membrane"><STRONG>transformed</STRONG></A> into them (one library variant per cell on average). In turn, the cells may produce the protein of interest. After cells are encapsulated into droplets, cell-specific reagents can lyse the cells. After the cell lysis, an enzyme of interest is released into the droplet together with DNA molecules that encoded it.</P><P class="didesnist"><STRONG>Proof of concept</STRONG></P><P>After the esterase <STRONG>mutant DNA library</STRONG> and <STRONG>Substrate Nucleotides </STRONG>are encapsulated together with protein <STRONG>expression reagents</STRONG> the droplets are incubated in order for protein expression to occur.</P><SECTION id="section-conversion" class="page-section"><DIV class="fancy-title title-border-color"><H4 class="didesnish4">4. Catalysis of Substrate Conversion</H4></DIV><P class="didesnist"><STRONG>General description</STRONG></P><P>At this stage, each droplet contains produced catalytic biomolecules. These catalytic biomolecules can now act on Substrate Nucleotides. After the chain of catalyzed chemical reactions is over, some of the nucleotides have their substrates removed - they are now <STRONG>Product Nucleotides</STRONG>. Depending on the catalytic biomolecule activity in each droplet, there can be a different amount of product nucleotides depending on the activity of the biomolecule variant.</P><P class="didesnist"><STRONG>Proof of concept</STRONG></P><P>In each droplet esterase mutants are synthesized. Depending on their activities, they will remove different amount of substrates from the Substrate Cytidines in different droplets.</P><DIV class="toggle toggle-bg"><DIV class="togglet"><DIV style="; font-size: 90%; margin-right: 5%; ">Does the design choice of removing substrate  limit the different class of biomolecules we can analyse?</DIV></DIV><DIV class="togglec"><P>Not at all! CAT-Seq allows to screen biomolecules catalyzing numerous of distinct reactions. Please head to the  <A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Nucleotide_Design">Substrate Nucleotide design page</A>  for more information and examples!</P></DIV></DIV><SECTION id="section-recording" class="page-section"><DIV class="fancy-title title-border-color"><H4 class="didesnish4">5.  Information recording</H4></DIV><P class="didesnist"><STRONG>General description</STRONG></P><P>At this point, each droplet contains a <STRONG>defined amount of nucleotides</STRONG> with their substrate removed, which is <STRONG>proportional to how strong the catalytic activity</STRONG> was in the particular droplet, of that particular biomolecule.</P><P>This information must then be stored into the biomolecule coding DNA sequence in the process of DNA amplification.</P><P>There are two main things that need to be considered in order to precisely record the activity information: <STRONG>effective DNA amplificatio</STRONG>n and the <STRONG>addition of reference information</STRONG> which would serve as an activity baseline or a point of reference.</P><P>First of all, the efficiency of amplification reaction. After an in-depth discussion with microfluidics expert Karolis Leonavičius we have decided that DNA amplification reagents must be added after the initial droplet generation and protein production. That is because protein synthesis reagents usually do not work well with various amplification mixes.</P><P><STRONG>Droplet merging</STRONG> can solve this problem. Main principle of droplet merging techniques, is that by using a specially designed microfluidics device, droplets can be merged with the help of electrical field. In CAT-Seq, the initial population of <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="the initially formed droplets with expression reagents and substrate nucleotides"><STRONG>substrate droplets</STRONG></A> must be merged with freshly generated <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="droplets containing DNA amplification reagents and Reference Nucleotides"><STRONG>amplification droplets</STRONG></A>. All the the amplification droplets contain the same amount of reagents that are required to amplify DNA.</P><P>Another important thing to consider is creating a reference point which would help to precisely determine the activities of different catalytic biomolecules. For CAT-Seq, such reference point is nucleotides with reference modification. This modification can be any small modification which still allows a DNA polymerase to incorporate it during the amplification. We call them the <STRONG>Reference Nucleotides. </STRONG></P><P>For example, If we would try to amplify DNA template in each droplet that contains different amounts of nucleotides that can be incorporated, we would get a sequencing <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="Bias is disproportionate weight in favour of or against one thing compared with another">bias</A> that would be more favourable for biomolecules that are highly active.</P><P>That is because biomolecules with higher activity would produce more nucleotides for polymerase and the amplified DNA amount would be much bigger than compared to lower activity mutants which would produce less. Reference nucleotides can easily remove this bias - after merging the Substrate Droplet with Amplification droplets that contain Reference Nucleotides, they allow the synthesis of a similar amount of amplified DNA in each droplet.</P><P>Thus, two important parts of CAT-Seq has now been established - <STRONG>droplet merging</STRONG> and <STRONG>Reference Nucleotides</STRONG>.</P><P style="margin-bottom: 0%;">Going back, we now have a population of <STRONG>substrate droplets</STRONG> that have a defined amount <STRONG>Product Nucleotides</STRONG> (the chemically converted substrate nucleotides). Those must then be merged with <STRONG>Amplification Droplets</STRONG>, wherein every droplet contains the same amount of amplification reagents and <STRONG>Reference Nucleotides</STRONG>. After the droplet merging, each droplet now contains a <STRONG>ratio of Product nucleotides to Reference Nucleotides</STRONG>. If the catalytic biomolecule was highly active, the ratio of Product Nucleotides to Reference Nucleotides will be high. In contrast, if the biomolecule activity was low, the ratio will also be low.</P><P style="margin-bottom">These specific nucleotide ratios in droplets can then be <STRONG>recorded into DNA, by the means of DNA amplification</STRONG>. As DNA is amplified in each droplet, specific ratio of nucleotides is incorporated into the copy of initial template. Such amplification results in molecules that both encode the <STRONG>sequence of a specific biomolecule </STRONG>and information about that particular biomolecules’ <STRONG>catalytic activity</STRONG>.</P><P class="didesnist"><STRONG>Proof of concept</STRONG></P><P>We now have a population of droplets, wherein each droplet contains a <STRONG>specific amount of Product Nucleotides</STRONG> or 2'-deoxycytidine triphosphates.</P><P>As already mentioned earlier, we have chosen to use MDA reaction in order to amplify the DNA isothermally. Isothermal amplification allows us to amplify the DNA without disrupting the stability of the droplets. Also, for the Reference Nucleotides, we have decided to use Methylated Cytidines (2'-deoxy-5-methylcytidine 5'-triphosphates). These Reference Nucleotides can be readily incorporated by multiple polymerases, including Phi29 which we use for our MDA reaction.</P><P>Next, we need to merge our Substrate Droplets with the Amplification Droplets containing the MDA reaction mix. Droplet merging is not an easy task to achieve, as efficient merging requires precise tuning of multiple parameters, such as 4 different droplet and oil flow rates and the strength of the electrical field that helps the droplets to merge. Searching for these parameters experimentally can often be like searching for a needle in a haystack. For this reason, before starting to do droplet merging experiments in the laboratory, we have written a thorough mathematical model which describes the droplet merging in detail and has the ability to predict the correct experimental parameters. We have successfully determined those <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="Water and oil phase flow rates, different droplet volume ratios, the strength of the electrical field and other">parameters</A> and showed experimentally, that they were indeed well-performing conditions for the droplet merging.</P><P>Substrate Droplets could then be merged with Amplification Droplets containing MDA mix and Reference Nucleotides.</P><P>Each merged droplet now contains a specific ratio of Product Cytidines to Methylated Cytidines. Such information can then be recorded by initiating MDA reaction. After the reaction, the amplified DNA molecules now have the specific ratio of nucleotides, wherein the ratio of those nucleotides depend on the specific esterase mutant activity.</P><SECTION id="section-reading" class="page-section"><DIV class="fancy-title title-border-color"><H4 class="didesnish4">6.  Reading the activity information - sequencing and data analysis</H4></DIV><P class="didesnist"><STRONG>General description</STRONG></P><P>The last step of the CAT-Seq is the retrieval of recorded activity information from the amplified DNA. In theory, this may be achieved by any sequencing method that can incorporate a DNA modification recognition step. Yet, CAT-Seq uses <STRONG>Nanopore Sequencing</STRONG> for several reasons:</P><P class="listin">1. Nanopore sequencing allows to sequence<STRONG> long molecules</STRONG>, which eliminates the need of biomolecule fragmentation. This is an extremely important characteristic to us because when working with large libraries of mutants in single droplets, fragmenting amplified DNA in those droplets means that all of those fragments in droplets would need to be barcoded. While in theory, this is a possible thing to do, it would require an extra merging and amplification step which would immensely decrease the accuracy of the activity determination. That is because in general, every step of the biotechnological method never is one hundred percent efficient. In turn, the lower amount of steps your system needs to take in order to produce results - the more accurate it will be. That is why reducing the number of steps in CAT-Seq by <STRONG>omitting fragmentation and barcoding</STRONG> was extremely important for us.</P><P class="listin">2. Secondly, nanopore sequencing can detect and discriminate different modified nucleotides with ease, because nanopore can detect slight changes in nucleotide structure and assign a different value to that signal.</P><P>Going back, the droplets containing the amplified DNA molecules can then be <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="The procedure of re-separating the oil and water phases">broken</A> and the amplified DNA is purified. The purified DNA from water phase can be directly sequenced using a nanopore sequencer. As the library passes through the nanopore, it retrieves the <STRONG>biomolecule’s sequence</STRONG> and the<STRONG> information about its activity</STRONG> that is encoded in the form of Product to Reference Nucleotide ratio.</P><P style="margin-bottom: 2%">Using the same principle, billions of different catalytic biomolecule sequences and their corresponding activities can be recorded.</P><P class="didesnist"><STRONG>Proof of concept</STRONG></P><P style="margin-bottom: 0%">The last step of the proof of concept story is the <STRONG>sequencing of amplified DNA</STRONG> using the nanopore. This is an important step, because it can provide the information about the CAT-Seq system precision. In this case, nanopore must <STRONG>accurately distinguish mutant sequences </STRONG>that contain different point mutations and <STRONG>accurately detect Product </STRONG>and <STRONG>Reference Cytidines</STRONG>.</P><P style="margin-bottom: 0%">After the sequencing we have successfully retrieved information about both the different mutant sequences and also their activities. After that, it was extremely important to compare the sequences and activities retrieved by CAT-Seq and the precise, low-throughput esterase mutant measurements we have performed earlier using standard methods. The proof of concept was a huge success, as <STRONG>activities determined by CAT-Seq closely followed the beforehand low-throughput measurements by standard methods.</STRONG></P><P>To read about more about proof of concept results, please <A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Demonstrate">click here</A>.</P></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></DIV></SECTION></DIV></DIV></DIV></SECTION></DIV></BODY></HTML>