<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="project-wrapper"><HEADER class="header-area"><NAV class="navigation"><UL class="navigation-list"><LI id="project-list-item" class="navigation-list-item"><A class="navigation-list-item-anchor" href="https://2018.igem.org/Team:Vilnius-Lithuania/Description"><SPAN class="image-logo-text">project</SPAN></A></LI><LI id="parts-list-item" class="navigation-list-item"><A class="navigation-list-item-anchor" href="https://2018.igem.org/Team:Vilnius-Lithuania/Overview"><SPAN class="image-logo-text">parts</SPAN></A></LI><LI id="notebook-list-item" class="navigation-list-item"><A class="navigation-list-item-anchor" href="https://2018.igem.org/Team:Vilnius-Lithuania/LabBook"><SPAN class="image-logo-text">notebook</SPAN></A></LI><LI id="hp-list-item" class="navigation-list-item"><A class="navigation-list-item-anchor" href="https://2018.igem.org/Team:Vilnius-Lithuania/Human_Practices"><SPAN class="image-logo-text">human practices</SPAN></A></LI><LI id="team-list-item" class="navigation-list-item"><A class="navigation-list-item-anchor" href="https://2018.igem.org/Team:Vilnius-Lithuania/Members"><SPAN class="image-logo-text">team</SPAN></A></LI></UL></NAV></HEADER><DIV class="content-area"><DIV class="text-wall-carrot-container"><DIV class="text-wall-area"><H1 class="text-wall-heading">Design and Results</H1><DIV class="text-wall-area-box"><H2 class="text-wall-area-box-heading">Results</H2><DIV class="scroll-area"><P class="text-content">Cell-free, synthetic biology systems open new horizons in engineering biomolecular systems which feature complex, cell-like behaviors in the absence of living entities. Having no superior genetic control, user-controllable mechanisms to regulate gene expression are necessary to successfully operate these systems. We have created a small collection of synthetic RNA thermometers that enable temperature-dependent translation of membrane proteins, work well in cells and display great potential to be transferred to any in vitro protein synthesis system.
         </P><BUTTON class="read-more-button">Read More</BUTTON></DIV></DIV><DIV class="pagination"><DIV class="pagination-item-wrapper"><A class="pagination-anchor"><SPAN class="pagination-text">Description</SPAN></A></DIV></DIV><DIV class="modal"><DIV class="modal-content"><SECTION class="design_subsections"><H1 id="Liposomes">Liposomes</H1><DIV class="third_level_links"><A href="#Liposomes">Liposomes</A><A href="#Ribosome_modifications">Ribosome modifications</A><A href="#BAM complex">BAM complex</A><A href="#RNA_Thermoswitches">RNA Thermoswitches</A><A href="#Mistic_fusion_protein">Mistic fusion protein</A><A href="#Surface_display_system">Surface display system</A></DIV><DIV><H2>Liposomes as closed containers for bottom up research</H2><P>
                    At the core of SynDrop lays a liposome. Liposomes are essentially synthetic vesicles, artificially synthesized droplets of liquid, separated from the environment by a lipid bilayer (Fig. 1). They act as containers that encapsulate purified transcriptional and translational machinery and other vital elements that enable complex circuitry design. They have become increasingly popular due to various applications such as being carriers for medicinal drugs<SUP>1</SUP>, closed environments for protein engineering<SUP>2</SUP> and characterization of RNAs<SUP>3</SUP>, as biosensors<SUP>4</SUP> and molecular diagnostic tools<SUP>5</SUP>. The growing perspectives of liposomes as scaffolds for synthetic circuitry and membrane protein research are compelling as they have a multitude of different parameters that can be controlled. These include size, composition of a lipid membrane and interior composition. 
            </P><DIV class="image-container"><P><STRONG>Fig. 1</STRONG> The composition of a liposome with encapsulated machinery for membrane protein integration. Size, membrane composition and interior composition can be easily varied.</P></DIV><H2>Requirements for liposomes</H2><P>
                    There are many different ways to synthesize liposomes, resulting in a vast range of diverse outputs. For our project, we set particular requirements to suit our scientific design. Ideally, our liposomes must be stable, cell-sized, monodisperse, have a single bilayered membrane and demonstrate excellent encapsulation efficiency. The only approach to optimally combine these attributes proved to be microfluidics. In recent years, droplet-based microfluidic techniques have proved to achieve a robust control over physical parameters of the final product, therefore showing superiority compared to the macroscale bulk methods<SUP>6</SUP>. We successfully customized and integrated a novel octanol-assisted liposome assembly (OLA)<SUP>7</SUP> method for high-throughput production of our liposomes. 
            </P><H2>
                    AutoCAD design for creating photomasks
            </H2><P>
                    The first step we had to take towards the production of liposomes was to design, fabricate and prepare a unique microfluidic device. A microfluidic channel design was created with the AutoCAD platform. The prototype acts as a photomask during the photolithography to create a master for the fabrication of microfluidic chips.  Our design consists of an array of 16 separate microfluidic channel devices distributed parallelly in groups of four on a single chip (Fig. 2). The dimensions of the microchannels limit the size range of the synthesized liposomes. To dig deeper into the details of how the dimensions of the microfluidic channels influence liposome size we created a phase-field based <STRONG><VAR><A href="https://2018.igem.org/Team:Vilnius-Lithuania/Model#COMSOL_model">SynFlow</A></VAR></STRONG> model with COMSOL Multiphysics. Auxiliary parametric sweeps were performed that defined the dimensions needed to attain cell – sized liposomes, a range from 5 µm to 30 µm. The final design can be downloaded here and used by anyone interested in the synthesis of liposomes.
            </P><DIV class="image-container"><P><STRONG>Fig. 2 a </STRONG>AutoCAD design for the photomask. There are 16 individual microchannel devices on a
                                   
   single chip. <STRONG>b</STRONG> One device consists of three inlets, an outlet and a star-shaped junction.</P></DIV><H2>Photolithography as a tool for microfluidic chip fabrication</H2><P>
                            After calculating the exact parameters for microfluidic channels and receiving a printed photomask, photolithography is performed to create a master for microfluidic chip preparation. After completing this step, PDMS (<VAR>polydimethylsiloxane</VAR>) is poured on to the master left in a thermostat overnight. Inlets and outlets are punched with a biopsy puncher, and the PDMS is cleaned and plasma treated before attaching it to the PDMS coated microscope glass slides. Fig. 3 presents a simplified scheme demonstrating photolithography and other +6steps towards creating a microfluidic chip. To learn more details about the fabrication process, refer to <A href="https://2018.igem.org/Team:Vilnius-Lithuania/Protocols">our Protocols</A>. We called our chip LipoDrop.  The final form of LipoDrop is shown in Fig. 4.  
                    </P><DIV class="image-container"><P><STRONG>Fig. 3 </STRONG> Simplified scheme for microfluidic device preparation. <STRONG>a-b</STRONG> the silicon wafer is cleaned and spin-coated with photoresist; <STRONG>c</STRONG> the photomask is aligned on the sample and exposed to UV light. <STRONG>d</STRONG> sample is submerged to a developer – only the sections that were exposed to the UV light remain intact  on the wafer; <STRONG>e</STRONG> PDMS is poured onto the master to create a PDMS mold and left for a bake in the oven; <STRONG>f</STRONG> the mold is then separated and prepared further by cleaning and punching inlets and outlets; <STRONG>e-f</STRONG> a microscopic slide is prepared by applying a thin layer of PDMS on top; <STRONG>i</STRONG> PDMS mold and PDMS covered microscopic slide are plasma treated and connected to each other to produce a final microfluidic chip.</P></DIV><DIV class="image-container"><P><STRONG>Fig. 4 </STRONG> Final form of Lipodrop.</P></DIV><H2>Coating LipoDrop with PVA</H2><P>
                            Final and critical step in preparing the chip is the selective coating of post-junction channels with PVA (<VAR>Polyvinyl alcohol</VAR>) to render them hydrophilic. It is required to prevent the lipid/octanol solution from wetting the inherently hydrophobic inner channel surface of the device. Without this additional coating, liposomes are unable to form. To coat only a part of microfluidic channels is a challenge: PVA must only remain in post-junction channels without any leakage to the other side. To counter the spreading of PVA to pre-junction channels, air is introduced from separate inlets. A correct interphase between air and PVA must form and stay stable for at least several minutes to let the PVA molecules adhere to the surface (Fig. 5). Any PVA contamination to the opposite side makes the whole device unusable. Not only is this process time consuming (at least 15-20 minutes for each microfluidic device) and requires constant supervision, the procedure quite often fails due to human error while controlling the infusion rates of air and PVA.
                    </P><DIV class="image-container"><P><STRONG>Fig. 5 </STRONG> A schematic representation of the interphase of air and PVA at the star shaped junction of LipoDrop.</P></DIV><H2>Lipovision software for fully automated microfluidic experiments</H2><P>
                            In pursuit to substantially reduce the manual effort in performing coating procedure, we have designed a software called LipoVision. LipoVision software reduces human labor down to the bare minimum and optimizes the coating procedure entirely. The LipoVision software is based on the open standard computer library OpenCV and is written in Go. It is available on all operating systems and made to be accessible to any custom microfluidic experiment. Eventually, we are planning to apply this software for the fully automated synthesis of the liposomes. To learn more about LipoVision, refer to <A href="https://2018.igem.org/Team:Vilnius-Lithuania/Software">Software</A>.
                    </P><H2>The synthesis of liposomes</H2><P>
                            After successfully coating the chips, the devices are finally ready for liposome synthesis. An experiment is conducted utilizing an octanol-assisted liposome assembly (OLA) method. Liposomes are formed when three unique phases (solutions - OA phase, LO phase and IA phase) form a correct interphase at the junction (Fig. 6). The <STRONG>IA</STRONG> phase (inner aqueous) occupies the inner part of the liposome and contains an IVTT transcription/translation system, DNA, membrane protein integration machinery, chaperones and salts needed for protein synthesis and integration into the membrane of the liposome. The <STRONG>LO</STRONG> phase contains lipids (i.e. DOPC, Cholesterol) and organic solvent (i.e. 2-octanol); it can also contain fluorescent lipids, such as Rh PE, for imaging. This phase introduces all of the components that will form the liposomes’ membrane. Octanol acts as an organic solvent for phospholipids. In the OLA method, initially double emulsions are formed; the excess octanol and lipids dewet and separate from the droplet leaving double-layered liposomes.  Octanol removal from liposomes is crucial as the correct bilayer cannot form in excess organic solvent.  Lastly, the <STRONG>OA</STRONG> phase (outer aqueous) contains surfactants that help stabilize the droplets at the initial formation and propagation through the microfluidic channels. 
                    </P><DIV class="image-container"><P><STRONG>Fig. 6</STRONG> A  close-up of the phase interface during liposome synthesis; <STRONG>IA</STRONG> phase contains elements required for the synthesis 
                            and integration of membrane proteins; <STRONG>LO</STRONG> phase consists of octanol and lipids that form a lipid bilayer; OA solution 
                            carries surfactants that stabilize the initial formation and propagation of the droplets along the microfluidic device.</P></DIV><H2>Optimized flow rates for high throughput synthesis</H2><P>
                            With this method, we effectively produce monodisperse bilayered liposomes with high throughput. It is important to note that together with the microchannel dimensions, the infusion rates of the different phases also have an impact on the size of the droplets. Additionally, the infusion rates regulate the throughput of liposome production. The already mentioned <STRONG><VAR>SynFlow</VAR></STRONG> model determines the optimized flow rates for the most stable and highest frequency synthesis. With the right dimensions and flow rates we reach the production rate that is up to 2000 Hz. The results suggest that this method could be successfully adapted for mass production of liposomes. Fig. 7 reveals the slowed down process of droplet formation.
                    </P><DIV class="image-container"><VIDEO width="50%" height="240" controls=""><P><STRONG>Fig. 7 </STRONG> High throughput formation of cell-sized liposomes. The video is 60x slowed down </P></VIDEO></DIV>

                            A simple liposome size frequency distribution was determined with an image analysis software ImageJ. A plugin SpotCaliper was utilized to identify circular objects and measure their diameters (Fig. 8a). Gaussian distribution was fitted to the frequency histogram. Results verify that the size of the liposomes follows the Gaussian distribution (Fig. 8b). It proves that the droplets are highly homogeneous. Average diameter of a liposome (results from a single batch experiment) is around 12 µm, with standard deviation of 0.4 µm which fits our requirements very well. 
                    <DIV class="image-container"><P><STRONG>Fig. 8 </STRONG> An automatic detection of droplets with SpotCaliper: the droplets are marked with teal colored circles and the 
                            diameter of each is measured; <STRONG>b</STRONG> size frequency distribution histogram fitted to Gaussian distribution (teal fit) proves 
                            the homogeneity of the liposomes; μ=11.853 &gt;µm±0.017 &gt;µm ; SD=0.442 µm ±0.017 µm. </P></DIV><H2>Characterization: encapsulation efficiency and internal synthesis</H2><P>
                            To prove the encapsulation efficiency and test whether protein synthesis and folding are feasible within the liposomes we employed green fluorescent protein (GFP). First, plasmid DNA coding GFP (3.0 ng/µL) was encapsulated together with PURE<VAR>frex</VAR> 2.0 transcription translation system. The total reaction mixture was 40 µL. The microtubes were kept on ice during the production and collection of the liposomes. After all the reaction mixture was used up, the collected suspension was incubated in 37℃ for six hours. During the incubation time, negative and positive controls were prepared. The negative control contained the same reaction mixture as previously, excluding the DNA. Positive control, on the other hand, contained purified GFP proteins and no included DNA. The outer solution of all samples contained RNAse to ensure that no synthesis occurred outside the droplets (in case some liposomes erupted and released DNA to the outside). After the incubation period, fluorescent microscopy was utilized to reveal the results. Excitation and emission wavelengths of GFP are 488 nm and 510 nm, respectfully, therefore a suitable FITC filter was used. 
                    </P><P>
                            Brightfield images of the synthesized liposomes are presented in Fig. 9a. The same area was then imaged under the FITC filter Fig. 9b, showing prominent fluorescence. It confirms that the liposomes are biocompatible, and synthesis occurs within them successfully . These results, together with positive control (Fig. 9c) validated that the encapsulation efficiency is immensely effective, as all the liposomes exhibit fluorescence signal. Negative control exhibited no measurable fluorescence with FITC filter, as expected (data not shown), confirming that no contamination was present to distort the results. 
                    </P><DIV class="image-container"><P><STRONG>Fig. 9 </STRONG> brightfield image of the liposomes that contain IVTT system and plasmid GFP DNA (after incubation);
                            scale bar is 10 µm; <STRONG>b</STRONG> liposomes imaged with FITC: fluorescence confirms that transcription and translation
                            reactions occur inside them; scale bar is 10 µm; <STRONG>c</STRONG> liposomes containing purified GFP protein: all the
                            liposomes exhibit fluorescence validating excellent encapsulation efficiency; scale bar is 20 µm.</P></DIV><H2>Characterization: unilamellarity validation using α-hemolysin protein pores</H2><P>
                            Most membrane proteins insert, fold and function properly only in unilamellar membranes, therefore unilamellarity is an absolutely critical parameter for studying MPs. To validate that our synthesized liposomes are truly unilamellar, we recruited the pore forming membrane protein α-hemolysin (2 mM) from Staphylococcus aureus. Monomers of α-hemolysin self-assemble to form mushroom-shaped heptameric 1.5 nm wide pores in the phospholipid bilayer. The integration of these proteins make the bilayer selectively permeable to small molecules sized 2 kDa or less<SUP>8</SUP>. This protein pore was chosen as it can perforate into only one bilayer, therefore a multilamellar membrane would not allow the internal content of the liposome to escape<SUP>9</SUP>.  
                    </P><P>
                            To test the incorporation of α-hemolysin into the liposome membrane we used a membrane-impermeable fluorescent probe calcein. Calcein is a self quenching fluorophore - a unique molecule that exhibits fluorescence only after it is diluted. That means that if we encapsulate concentrated calcein solution inside of our liposomes, we should observe that while looking through a fluorescence filter they appear darker than the outer solution (Fig. 10a). That happens because a small fraction of liposomes inevitably falls apart, releasing calcein into the outside, thus diluting it and therefore we measure the fluorescence in the outer solution. Hypothetically, inserting α-hemolysin into the liposome membrane will increase the fluorescence of the outer solution to a much greater degree, as the calcein molecules can now leave the liposomes via the α-hemolysin pore.
                    </P><P>
                            To test this hypothesis, we prepared two sets of liposomes. Lipid composition of liposomes was composed of DOPC and Cholesterol (cholesterol is necessary for the integration of α-hemolysin). Both sets of liposomes were prepared from the same batch microfluidic experiment. The same volume of liposomes was transferred into a solution with α-hemolysin and to an identical solution without the protein. Plate-reader measurements of fluorescence were then recorded, and results analyzed. As expected, in the absence of α-hemolysin (control), some background fluorescence was observed. However, in the solution where α-hemolysin was included, the measured fluorescence was significantly stronger (p &lt; 0.0001 Fig. 10b) compared to the control group. That confirms the successful integration of α-hemolysin pore into the liposome membrane, explaining the increased fluorescence of the measured outer solution. 
                   
                        </P><DIV class="image-container"><P><STRONG>Fig. 10 a</STRONG> concentrated calcein encapsulated within liposomes: the outer solution fluoresces as some of the liposomes
                                inevitably burst releasing calcein into the outside; <STRONG>b</STRONG> box plot comparison of the control (without α-hemolysin) and
                                a group with inserted α-hemolysin; nonparametrical Mann-Whitney U test was used for the statistical evaluation:
                                the group with α-hemolysin shows statistically significant (p &lt; 0.0001) increase in fluorescence</P></DIV><H3>References</H3><P><OL><LI>
                                               Torchilin, V. P. Recent advances with liposomes as pharmaceutical carriers. Nat. Rev. Drug Discov. 4, 145–160 (2005).
                            </LI><LI>
                                               Shin, J. &amp; Noireaux, V. An E. coli cell-free expression toolbox: Application to synthetic gene circuits and artificial cells. ACS Synth. Biol. 1, 29–41 (2012).
                            </LI><LI>
                                               Kapoor, M., Burgess, D. J. &amp; Patil, S. D. Physicochemical characterization techniques for lipid based delivery systems for siRNA. Int. J. Pharm. 427, 35–57 (2012).
                            </LI><LI>
                                               Michener, J. K., Thodey, K., Liang, J. C. &amp; Smolke, C. D. Applications of genetically-encoded biosensors for the construction and control of biosynthetic pathways. Metab. Eng. 14, 212–222 (2012).
                            </LI><LI>
                                               Pardee, K. et al. Paper-based synthetic gene networks. Cell 159, 940–954 (2014).
                            </LI><LI>
                                               Carugo, D., Bottaro, E., Owen, J., Stride, E. &amp; Nastruzzi, C. Liposome production by microfluidics: Potential and limiting factors. Sci. Rep. 6, (2016).
                            </LI><LI>
                                               Deshpande, S., Caspi, Y., Meijering, A. E. C. &amp; Dekker, C. Octanol-assisted liposome assembly on chip. Nat. Commun. 7, 1–9 (2016).
                            </LI><LI>
                                               Noireaux, V. &amp; Libchaber, A. A vesicle bioreactor as a step toward an artificial cell assembly. Proc. Natl. Acad. Sci. 101, 17669–17674 (2004).
                            </LI><LI>  Lu, L., Schertzer, J. W. &amp; Chiarot, P. R. Continuous microfluidic fabrication of synthetic asymmetric vesicles. Lab Chip 15, 3591–3599 (2015).
                            </LI></OL></P></DIV></SECTION><SECTION class="design_subsections"><H1 id="Ribosome_modifications">Ribosome modifications</H1><DIV class="third_level_links"><A href="#Liposomes">Liposomes</A><A href="#Ribosome_modifications">Ribosome modifications</A><A href="#BAM complex">BAM complex</A><A href="#RNA_Thermoswitches">RNA Thermoswitches</A><A href="#Mistic_fusion_protein">Mistic fusion protein</A><A href="#Surface_display_system">Surface display system</A></DIV><DIV><H1>
                Background
            </H1><P>
                    Insertion of many membrane proteins in prokaryotes as well as the endoplasmic reticulum of eukaryotes is facilitated by various translocons. These complexes interact with the ribosome during protein synthesis and direct the newly forming peptide into the translocon pore, or directly into the membrane. However, for the translocon to function properly, it requires many auxiliary components (signalling sequence, chaperones, insertion mechanism), which would be far too many for our system to remain stable. Since we could not simply transfer the entire translocation system into SynDrop, we hypothesized that some of its functionality could be emulated in different ways.
            </P><H2>
                        Nickel-chelating lipids and polyhistidine tags as potential solution
                </H2><P>
                    Specifically to localize the ribosomes near the membrane and reduce the exposure of MP transmembrane domains to the aqueous environment, we eventually chose a method that has already been successfully used to attach other proteins to membranes - via a his-tag to nickel ions, chelated by specific lipids composing membrane<SUP>1,2</SUP>. Our reasoning for this was threefold. Firstly, this would have allowed ribosomes to be correctly positioned. If the ribosomes were localized near the membrane, but the peptide exit tunnel hadn’t been consistently pointed towards the membrane, the entire benefit of reducing MP aggregation would be lost. By being able to create his-tagged fusion proteins specifically of the ribosomal subunits localized near the exit tunnel, we guaranteed that the exit tunnel could never point away from the membrane. Secondly, general protein synthesis should be the least impeded, compared to alternatives, such as introducing a lipid anchor via an enzyme, or creating a fusion protein with a self inserting MP. These latter methods would completely immobilize the ribosome onto the membrane, removing degrees of freedom. With the polyhistidine-nickel interaction the ribosome would simply exhibit a much greater affinity towards the membrane, while still being able to detach and reattach as needed. Lastly, the simplicity for this method seemed to be the greatest, as only modified ribosomes as well as the incorporation of nickel-chelating lipids during liposome synthesis was needed - this was far more important than it can appear, because it was vital for our overall system to be as robust as possible.
            </P><DIV class="image-container"><P><STRONG>Fig. 1 </STRONG> Principle of ribosome attachment to the liposome membrane. The ribosome exit tunnel is localized near the membrane, resulting in transmembrane domains of newly synthesized peptides interacting with the membrane, reducing aggregation</P></DIV><H1>Results</H1><H2>Designing the modifications</H2><P>
                    The basic procedure we designed to obtain our modified ribosomes is as follows: first, using CRISPR-Cas9, we modify the genome of E. coli (native strain MG1655) to incorporate his-tags onto the ribosomal subunits, as well as a Strep-tag onto the L12 subunit for ease of purification<SUP>3</SUP>. In literature a his-tag was described to aid purification when fused to subunit L12, however we exchanged it for a Strep tag, as it is also short in sequence and while being biologically does not interfere with the functionality of other his-tags. The subunits localized near the exit tunnel we chose as potential targets, in order of priority, were L24, L23, and L29. These subunits were chosen as they had free, unfolded C ends exposed to the surface at nearest proximity to the exit tunnel. These subunits are also not integral to the overall function of the ribosome, meaning that our modifications would have little to no impact on peptide synthesis<SUP>4,5,6,7</SUP>. After the E. coli genome is successfully modified, the ribosomes are purified with the help of the previously incorporated Strep tag, and the final product is used for IVTT reactions.
            </P><H2>
                    Modification procedure
                </H2><P>
                    CRISPR-Cas9 was utilized via pCas9 and pTargetF plasmids <SUP>8</SUP>. pCas9 constitutively expresses Cas9, which induces a double-strand DNA break at a specific target sequence, that is complementary to the guide RNA. The guide RNA sequences for our targets were introduced via reverse PCR into the pTargetF plasmid series, which then supplies it to Cas9. After the double stranded break occurs, the HDR (homology directed repair) mechanism is activated, which repairs the genome according to a supplied donor sequence. This process is highly efficient with the assistance λ-red proteins, expressed from pCas9. The donor sequence has ~300 bp length homology arms and the insertion sequence that can include either the His or Strep tag to be fused with the chosen ribosomal subunit C’-end, as well as an in frame selection marker (select antibiotic resistance genes). pCas9 expresses a gRNA targeting the ori of pTargetF, therefore the cells are automatically cured from pTargetF after each modification. Additionally, pCas9 has a temperature-sensitive ori, it is cured by growing the cells at 37 <SUP>o</SUP>C.
            </P><DIV class="image-container"><P><STRONG>Fig. 2 </STRONG> Scheme of the genome modification process:
                   <OL><LI>
                           1.  pCas9 is introduced into the target cells
                       </LI><LI>
                           2. pTargetF and the according donor DNA are electroporated into the cells
                       </LI><LI>
                           3. Cas9 and our custom gRNA form a complex that cuts the genome DNA at the target. The genome is repaired via HDR according to our donor DNA sequence - X1 (homology arm of target ribosomal subunit) with fused tag, X3 (antibiotic resistance gene) and X2 (second homology arm).
                       </LI><LI>
                           4. pTargetF is cured and the process can be repeated with a new target.
                       </LI></OL></P></DIV><P>For multi-gene editing, we chose to supply the donor sequence as a linear DNA strand (PCR product). Due to financial reasons, to construct the donor DNA sequence we performed separate PCRs of the homology arms (from the E. coli genome), selection marker (antibiotic resistance genes from available plasmids) (Fig. 4). The oligomers had the his and strep tag sequences incorporated into them alongside 2 different restriction sites. In case the distance between the ribosomes and the membrane wall was too small for our system to be efficient, we also designed alternative variants the would feature the his-tags connected via a highly flexible two-glycine-four-serine linker (GGSSSS), which is a highly popular linker for artificial fusion proteins.
            </P><P>
                    When the homology arm and marker gene pcr products are restricted and ligated, the end result is a single sequence with an antibiotic resistance gene flanked on either side by ~300 bp homology arms, in addition to the according tag fused to the selected subunit’s gene. The entire linear DNA product was amplified via PCR using primers that annealed to the ends of the strand. Large amounts of nonspecific product was present even after performing PCR of the whole ligated sequence, therefore gel purification was necessary after each amplification (Fig. 5).
            </P><DIV class="image-container"><P><STRONG> Fig.3 </STRONG> Example of a constructed donor sequence. The sequence of the selected tag is present in primer used for the PCR of the homology arm that encompasses the target subunit. As a result, the tag sequence is fused to the ribosomal subunit gene.</P></DIV><DIV class="image-container"><P><STRONG>Fig. 4 </STRONG> PCR of homology arms, and antibiotic resistance genes</P></DIV><DIV class="image-container"><P><STRONG>Fig. 5 </STRONG> Constructed donor DNA sequences. The L29 donor DNA was not further revisited due to time constraints</P></DIV><P>
                    The genome modifications were then carried according to <A href="https://2018.igem.org/Team:Vilnius-Lithuania/Protocols">our protocol</A>. Although cPCR gave us mixed results, we could not verify any colonies that afterwards grew on our selected marker antibiotics, and thus could not continue our experiments with them. It appears most likely that the genome modifications were not entirely successful, due to the somewhat unstable nature of the ligated linear DNA used for the donor sequence.
            </P><H1>Conclusion and Discussion</H1><P>
                    Although well described and planned in theory, our ribosome attachment system did not yield desired results. We hypothesize, that the underlying cause may include a flawed implementation of the CRISPR-Cas9 system: in our case we were forced to incorporate selection markers due to the functionality of the specific expression plasmids we used, which lead to the construction of unstable donor sequences. While this is unfortunate, we are still confident that our idea is worth further investigation: as we could not reproduce genome modifications on the subunit that has already been reportedly modified in a similar manner, it does not suggest that our primary concept is unworkable. Moving forward, we will most likely move to an alternative CRISPR-Cas9 expression system that would allow for optimized modifications, such as removing the requirement for a selection marker, which in turn would allow us to construct more robust donor sequences. Nonetheless, herein we provide a concept and initial tools for ribosome engineering. This demonstrates great potential in synthetic biology, especially in cell-free biomolecular systems like SynDrop.
            </P><H2>References</H2><P><OL><LI>1.Chikh, G., Li, W., Schutze-Redelmeier, M., Meunier, J. &amp; Bally, M. Attaching histidine-tagged peptides and proteins to lipid-based carriers through use of metal-ion-chelating lipids. Biochimica et Biophysica Acta (BBA) - Biomembranes 1567, 204-212 (2002).
                    </LI><LI>2.Blanchette, C., Fischer, N., Corzett, M., Bench, G. &amp; Hoeprich, P. Kinetic Analysis of His-Tagged Protein Binding to Nickel-Chelating Nanolipoprotein Particles. Bioconjugate Chemistry 21, 1321-1330 (2010).
                    </LI><LI>3.Ederth, J., Mandava, C., Dasgupta, S. &amp; Sanyal, S. A single-step method for purification of active His-tagged ribosomes from a genetically engineered Escherichia coli. Nucleic Acids Research 37, e15-e15 (2008).
                    </LI><LI>4.Spillmann, S. &amp; Nierhaus, K. The ribosomal protein L24 of Escherichia coli is an assembly protein. Journal of Biological Chemistry (1978). at http://www.jbc.org/content/253/19/7047.long    
                    </LI><LI>5.GU, S. The signal recognition particle binds to protein L23 at the peptide exit of the Escherichia coli ribosome. RNA 9, 566-573 (2003).
                    </LI><LI>6.STOFFLER-MEILICKE, M., DABBS, E., ALBRECHT-EHRLICH, R. &amp; STOFFLER, G. A mutant from Escherichia coli which lacks ribosomal proteins S17 and L29 used to localize these two proteins on the ribosomal surface. European Journal of Biochemistry 150, 485-490 (1985).
                    </LI><LI>7.Noeske, J. et al. Synergy of Streptogramin Antibiotics Occurs Independently of Their Effects on Translation. Antimicrobial Agents and Chemotherapy 58, 5269-5279 (2014).
                    </LI><LI>8.Jiang, Y. et al. Multigene Editing in the Escherichia coli Genome via the CRISPR-Cas9 System. Applied and Environmental Microbiology 81, 2506-2514 (2015)
                    </LI></OL></P></DIV></SECTION><SECTION class="design_subsections"><H1 id="BAM complex">BAM complex</H1><DIV class="third_level_links"><A href="#Liposomes">Liposomes</A><A href="#Ribosome_modifications">Ribosome modifications</A><A href="#BAM complex">BAM complex</A><A href="#RNA_Thermoswitches">RNA Thermoswitches</A><A href="#Mistic_fusion_protein">Mistic fusion protein</A><A href="#Surface_display_system">Surface display system</A></DIV><DIV><H1>Background</H1><P>Proteins that belong to a small group referred to as nonconstitutive membrane proteins can independently integrate into the membrane from the aqueous phase without the help of other proteins. This is because they possess a stable soluble form, that can bind to membranes and then insert and refold into another stable form.[1] However, most of the membrane proteins do not have a stable soluble form and rapidly aggregate when synthesized in the cytoplasm. That is why living organisms require additional machinery which facilitates MP insertion into membranes and catalyzes their folding.</P><H2>Structure of BAM complex
</H2><P>Assembly of the β-barrel bearing integral membrane proteins (MPs) into the target membrane is catalyzed by the β-barrel assembly machinery (BAM) complex. It contains five subunits, BamA–E.[2]  BamA is composed of a 16-strand β-barrel integral membrane part and a periplasmic domain, which consists of five globular subdomains called POTRA motifs that are essential for complex formation and interaction with a substrate β-barrel proteins.[3] BamB-E are lipoproteins, each attaching to the inner leaflet of the OM via an N-terminal lipid moiety and playing an important role in promoting the folding of OMP.[4] (Fig.1)
</P><DIV class="image-container"><P><STRONG>Fig. 1</STRONG> Structure of BAM complex</P></DIV><H2>Outer membrane protein (OMP) insertion into membrane in bacteria cells
</H2><DIV class="image-container"><P><STRONG>Fig. 2</STRONG> 5 steps scheme of OMP insertion into OM.</P></DIV><P>Generally MPs are integrated during translation with the assistance of the Sec Translocon. However, as there is no protein translation within the periplasm, OMPs require an alternative integration mechanism. 
</P><P>In the beginning, OMPs are synthesized in the cytoplasm with N-terminal signal sequence that directs them to the Sec translocon, which transfers OMPs through the inner membrane into periplasm.[5] (Fig.2, step 1)
</P><P>Integral membrane proteins forming β-barrel structures are prone to aggregate in aqueous environments. Therefore, after they transit a Sec channel, chaperones are required to bind OMPs to transfer them through the periplasmic compartment, while keeping them in an unfolded state to prevent aggregation.(Fig.2, step 2) The periplasmic chaperone, SurA has been shown to transfer most of the OMPs to the OM.[6] It has been shown that SurA directly participates in Bam-mediated OMP assembly by associating with BamA POTRA domain.[7] (Fig.2, step 3)</P><P>Final folding takes place in BAM complex. However, the mechanism how BAM complex catalyzes the insertion of β-barrel proteins into the OM still remains not fully understood. Structure analysis implies that the cavity seems to be too small to accomodate a fully folded OMP substrate, even though it is large enough to house couple of substrate β-hairpins.[2] (Fig.2, step 4)Based on the structure, it has been suggested that the rotation of the ring-like structure by POTRA domains and lipoproteins leads to the opening of a junction between the first and last β-strands of the BamA β-barrel (lateral gate) which promotes the insertion of OMPs into the lipid bilayer.[8] (Fig. 2, step 5)
</P><H2>Our approach</H2><P>In order to reconstitute the fully working Bam complex we relied on the mechanisms elucidated before. SurA is a periplasmic chaperone which binds to unfolded β-barrel proteins and retains them in unfolded state, thus preventing aggregation. BamB or BamD lipoproteins can bind BamA-SurA, direct the complex to the membrane, and catalyze BamA insertion as well as correct folding in the membrane. [9] (Fig. 3, step 1) Knowing that in bacteria most of the Bam lipoproteins are found in BAM complex and full five proteins complex can be purified with one tag without any cross-links, we expected high complex association constants among the components (Fig.3, step 2) and hypothesized that BAM complex could be assembled in vitro simply by encapsulating BamA-SurA associatives and Bam lipoproteins.</P><DIV class="image-container"><P><STRONG>Fig. 3</STRONG> Bam lipoproteins assemble BamA in vitro</P></DIV><H1>Results</H1><H2>Plasmid construction</H2><P>To purify the components of BAM complex we have constructed 6 plasmids:</P><P><OL><LI>pET28b-BamA</LI><LI>pET22b-BamA</LI><LI>pET22b-BamB</LI><LI>pCDFDuet-BamC-BamD</LI><LI>pET22b-BamE</LI><LI>pET28b-SurA(more details in protocols)</LI></OL>
    which encode products of BamA-HisN6, BamA, BamB-HisC6, BamC, BamD, BamE-HisC8, SurA-HisN6. (Fig. 4 and Fig. 5) 
</P><DIV class="image-container"><P><STRONG>Fig. 4</STRONG> Maps of constructed plasmids</P><P><STRONG>Fig. 5</STRONG> PCR products of genes of BAM complex 1,18 – GeneRuler 1 kb DNA ladder;  4,5 – SurA (1250bp); 6,7 -  BamE (390bp); 8,9 – BamB (1201bp); 10, 11 – BamC (1056bp); 12, 13 – BamD(756bp); 14,16 – BamA (2455bp)</P></DIV><H2>
    Protein purification
</H2><P>We relied on three different strategies to purify the separate BAM complex components: </P><UL><LI>For our experiments we needed an unfolded BamA. Therefore, we overexpressed BamA , which we isolated in the form of inclusion bodies and then dissolved in 8M urea without any further purification steps.</LI></UL><DIV class="image-container"><P><STRONG>Fig. 6</STRONG> BamA after purification
    1 – BamA, 2 – BamA-HisN6, L – PageRuler Unstained Broad Range Protein Ladder</P></DIV><UL><LI>Bam B-D lipoproteins were expressed with the pelB signal sequence, leading them to be exported to the periplasm where lipidation takes place. We then isolated the proteins from the membrane fraction, which we solubilised with specific detergents before purification using Ni-NTA (Fig.7 and Fig.9) and gelfiltration (size-exclusion) (Fig. 8 and Fig. 9) columns. BamCDE were purified as a single subcomplex via one octahistidine tag on BamE.</LI></UL><H3>After purification with Ni-NTA column:</H3><DIV class="image-container"><P><STRONG>Fig. 7</STRONG> BamB after Ni-NTA column purification Lane L – PageRuler Unstained Protein Ladder, Lane 1 – Sample  loaded on Ni-NTA Column,
    Lanes 2-3 – Flow through fractions, Lanes 4-5 – washing fractions, Lanes 6-9 – Elution fractions
    </P></DIV><H3> After gelfiltration:</H3><DIV class="image-container"><P><STRONG>Fig. 8</STRONG> BamB fractions after gelfiltration
Lane L – PageRuler Unstained Protein Ladder, Lanes 1-14 Elution fractions
</P></DIV><H3> After gelfiltration:</H3><DIV class="image-container"><P><STRONG>Fig. 9</STRONG> BamCDE after Ni-NTA column purification Lane L – PageRuler Unstained Protein Ladder, Lane 1,2 - Flow through fractions, Lanes 3-4 – washing fractions, Lanes 5-7 – Elution fractions
</P><DIV><H3>After gelfiltration:</H3><DIV class="image-container"><P><STRONG>Fig. 10</STRONG> BamCDE fractions after gelfiltration
    Lane 1 – PageRuler Unstained Protein Ladder,  2-15 elution fractions</P></DIV><UL><LI>
        While SurA is a periplasmic protein, we had no issues overexpressing it in the cytoplasm for increased yield. We used a hexahistidine tag and purified using Ni-NTA (Fig.11) and gelfiltration (Fig.12) columns.
    </LI></UL><H3>After Ni-NTA column:</H3><DIV class="image-container"><P><STRONG>Fig. 11 </STRONG>SurA after purification with Ni-NTA column
    L - PageRuler Unstained Protein Ladder, 1 - Protein loaded on Ni-NTA Column, Lane 2 – Flow through fraction, 3 - washing fraction, 4-9 elution fractions
    </P></DIV><H3>After gelfiltration</H3><DIV class="image-container"><P><STRONG>Fig. 12 </STRONG>SurA fractions after gelfiltration
    L - PageRuler Unstained Protein Ladder, 1-9 elution fractions 
    </P></DIV><H2>Folding assay</H2><P>To determine whether the purified proteins act as expected we conducted a specific a folding assay. Proteins possessing β-barrel structures exhibit a unique characteristic - when mixed with SDS (for SDS-PAGE) but unboiled, the β-barrel structure remains intact, which causes the protein to move differently in the SDS-PAGE gel in comparison to the same protein lacking these structures - which occurs when it is denatured or did not originally fold. Exploiting this characteristic makes it possible to observe and quantify protein folding levels.</P><P>For the first experiment we observed if BamB and BamCDE can incorporate the unfolded BamA protein into the membrane and reconstitute the complete BAM complex. This was accomplished by incubating SurA with BamA denatured in urea, then transferring it into a solution featuring liposomes, BamB and BamCDE,  then further incubating for 2 hours. As BamA was expressed with a his-tag, we performed a blot to determine the level of protein folding (Fig. 13). </P><DIV class="image-container"><P><STRONG>Fig. 13</STRONG> Western blot of BamA folding. 1U - sample 1 unboiled, 1B - sample 1 boiled, 2U - sample 2 unboiled, 2B - sample 2 boiled, L - ladder
</P></DIV><P>As we can see from the results, after only 2 hours of incubation over 50% of BamA was processed and correctly folded, which is an indicator of proper functionality. At higher concentrations or within more enclosed environments, such as encapsulated within the liposome, efficiency is bound to increase.</P><H1>Conclusions</H1><P>We managed to successfully isolate the BAM protein complex at a high purity with relatively few steps. The BAM complex not only shows activity in vitro, it shows efficient activity within the presence of liposomes, which shows that it is functional and suitable for our system.
</P><H1>Discussion</H1><P>The study of MPs is and continues to be a difficult area of study, due to the sheer difficulty in handling them. As we have emphasized before, the integration of MP’s is a particularly pronounced issue, being borderline impossible for some cases in vitro. However, we have managed to demonstrate, that with our newly designed approach utilizing the BAM complex, the SynDrop system can allow for much more efficient insertion of MP’s as well as greatly expanding the total amount of viable MP’s for high throughput in vitro studies.</P><H2>References</H2><P><OL><LI>White, S. &amp; Wimley, W. MEMBRANE PROTEIN FOLDING AND STABILITY: Physical Principles. Annual Review of Biophysics and Biomolecular Structure 28, 319-365 (1999).</LI><LI>Noinaj, N., Rollauer, S. &amp; Buchanan, S. The β-barrel membrane protein insertase machinery from Gram-negative bacteria. Current Opinion in Structural Biology 31, 35-42 (2015).
    </LI><LI>Fleming, P. et al. BamA POTRA Domain Interacts with a Native Lipid Membrane Surface. Biophysical Journal 110, 2698-2709 (2016).
    </LI><LI>Hussain, S. &amp; Bernstein, H. The Bam complex catalyzes efficient insertion of bacterial outer membrane proteins into membrane vesicles of variable lipid composition. Journal of Biological Chemistry 293, 2959-2973 (2018).
    </LI><LI>Driessen, A. &amp; Nouwen, N. Protein Translocation Across the Bacterial Cytoplasmic Membrane. Annual Review of Biochemistry 77, 643-667 (2008).
    </LI><LI>Sklar, J., Wu, T., Kahne, D. &amp; Silhavy, T. Defining the roles of the periplasmic chaperones SurA, Skp, and DegP in Escherichia coli. Genes &amp; Development 21, 2473-2484 (2007).
    </LI><LI> Bennion, D., Charlson, E., Coon, E. &amp; Misra, R. Dissection of β-barrel outer membrane protein assembly pathways through characterizing BamA POTRA 1 mutants of Escherichia coli. Molecular Microbiology 77, 1153-1171 (2010).
    </LI><LI>Gu, Y. et al. Structural basis of outer membrane protein insertion by the BAM complex. Nature 531, 64-69 (2016).
    </LI><LI>Hagan, C., Westwood, D. &amp; Kahne, D. Bam Lipoproteins Assemble BamA in Vitro. Biochemistry 52, 6108-6113 (2013).
    </LI></OL></P>-
            </DIV></DIV></DIV></SECTION><SECTION class="design_subsections"><H1 id="RNA_Thermoswitches">RNA Thermoswitches</H1><DIV class="third_level_links"><A href="#Liposomes">Liposomes</A><A href="#Ribosome_modifications">Ribosome modifications</A><A href="#BAM complex">BAM complex</A><A href="#RNA_Thermoswitches">RNA Thermoswitches</A><A href="#Mistic_fusion_protein">Mistic fusion protein</A><A href="#Surface_display_system">Surface display system</A></DIV><DIV><DIV class="image-container"><P><STRONG>Fig. 1</STRONG> A simplified mechanism of action of RNA thermometers. At lower temperatures the secondary messenger RNA (mRNA) stem-loop masks the ribosome binding site (RBS). Higher temperature induces melting of the hairpin which reveals the RBS to allow ribosome binding and initiation of translation.</P></DIV><H1>Background</H1><P>
                  RNA thermometers are RNA-based genetic control tools that react to temperature changes<SUP>1</SUP>. Low temperatures keep the mRNA at a conformation that masks the ribosome binding site within the 5’ end untranslated region (UTR). Masking of the Shine-Dalgarno (SD) sequence restricts ribosome binding and subsequent protein-translation. Higher temperatures melt the hairpins of RNA secondary structure allowing the ribosomes to access SD sequence to initiate translation <SUP>1</SUP>. In terms of applicability of RNA thermometers in <VAR>in vitro</VAR> systems, they display certain advantages over ribo- or toehold switches: they do not require binding of a ligand, metabolite or trigger RNA to induce the conformational change<SUP>2,3</SUP>, therefore are especially compatible with our liposome IVTT system. Keeping that in mind we have explored literature <SUP>1,4</SUP> and found five different RNA thermoswitches that we decided to test and build into our system in order to delay the translation of fusion construct bearing beta-barrel membrane protein. Furthermore, understanding the importance of expanding the library of well characterized and widely-applicable biobricks, we have <VAR>de novo</VAR> designed (<A href="https://2018.igem.org/Team:Vilnius-Lithuania/Model&lt;/a">) six completely unique heat-inducible RNA thermometers.
               </A></P><H1>Results</H1><P>
                  With custom IDT primers with overhangs bearing thermoswitch sequences we performed a PCR  from a GFP gene containing plasmid and inserted RNA thermometers GJ<SUB>x</SUB> (link: <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622010">BBa_K2622010</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622011">BBa_K2622011</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622012">BBa_K2622012</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622013">BBa_K2622013</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622014">BBa_K2622014</A>) upstream the GFP gene. Another set of primers was used to produce RNA thermometers Sw<SUB>x</SUB> (link: <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622016">BBa_K2622016</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622017">BBa_K2622017</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622018">BBa_K2622018</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622019">BBa_K2622019</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622020">BBa_K2622020</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622021">BBa_K2622021</A>). PCR was successful and all products were the same size as expected for Sw<SUB>x</SUB> constructs ~76 bp (Fig. 2). DNA gel electrophoresis was not performed for GJx constructs, because whole plasmid was multiplied and only 40-60 bp were inserted. 
               </P><P><STRONG>Fig. 2 </STRONG> Electrophoresis gel of PCR products: 6 - Sw2, 7 - Sw3, 8 - Sw6, 9 - Sw7, 10 - Sw9, 11 - Sw11.</P><P>
                  pRSET plasmid and Sw<SUB>x</SUB> PCR products were digested with restriction enzymes and ligated, while GJ<SUB>x</SUB> PCR products were phosphorylated and ligated to produce plasmids from linear products. DH5α competent cells were transformed and plated on lysogeny broth (LB) media with ampicillin (Amp) and grown for 16 hours. Positive colonies were selected by colony PCR or restriction analysis (Fig. 3 and Fig. 4) and grown in 5 mL LB media. Plasmids were purified and BL21 competent cells were transformed. Three tubes of every construct plus plasmid with GFP without RNA thermometer were grown till OD<SUB>600</SUB> reached 0.4. Control samples were taken and protein expression was induced with Isopropyl β-D-1-thiogalactopyranoside (IPTG). One tube of every construct was grown in 24 ˚C, 30 ˚C, and 37 ˚C. Samples were taken after 1 and 2 hours. SDS-PAGE was run (for elaborate protocol see <A href="https://2018.igem.org/Team:Vilnius-Lithuania/LabBook">Notebook</A>/<A href="https://2018.igem.org/Team:Vilnius-Lithuania/Protocols">Protocols</A>). Fig. 5, Fig. 6 and Fig. 7 depicts GFP expression at different temperatures. Although our RNA thermometers were designed to melt at 37 ˚C, some displayed leakiness to different extent. GJ3 (link:<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622011">BBa_K2622011</A>) RNA thermometer was the leakeast and allowed for GFP translation at lower temperatures. On the other hand, when grown at 37 ˚C, it unlocked the translation of GFP to highest yields. GJ2 (link:<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622010">BBa_K2622010</A>) was less leaky, but inhibited protein translation more strictly when grown at 37 ˚C. GJ6 (link: <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622012">BBa_K2622012</A>), GJ9 (link:<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622013">BBa_K2622013</A>), and GJ10 (link: <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2622014">BBa_K2622014</A>) suppressed GFP production at 24 ˚C and 30 ˚C at similar level. They also inhibited translation to some extent at higher temperatures, meaning their melting temperature was not reached. Altogether these results prove, that our synthetic thermoswitches are temperature-responsive and act in physiological temperature range needed for IVTT reaction and also for BamA folding and membrane insertion.
               </P><P><STRONG>Fig. 3 </STRONG> Restriction analysis of GJ<SUB>x</SUB> constructs</P><P><STRONG>Fig. 4 </STRONG> Colony PCR of RNA thermometers in pSB1C3 plasmid.</P><P><STRONG>Fig. 5 </STRONG> expression at 24 ˚C. On the right you can see GFP expression without RNA thermometer.</P><DIV class="image-container"><P><STRONG>Fig. 6 </STRONG> GFP expression at 30 ˚C. On the right you can see GFP expression without RNA thermometer.</P></DIV><DIV class="image-container"><P><STRONG>Fig. 7 </STRONG> GFP expression in 37 ˚C. On the right you can see GFP expression without RNA thermometer.</P></DIV><H1>Discussion</H1><P>
                  As described in other sections of the Design and results page , beta-barrel bearing proteins are assembled into the membrane by the BAM protein complex machinery. The key protein BamA is itself a membrane protein, whose folding and insertion into membrane where it helps assemble target proteins, last up to two hours. In order to prevent the aggregation of our fusion proteins after encapsulating their gene-bearing plasmids and purified BamA mRNA into liposomes, we needed to develop a modulatory regulatory tool to lock the translation of our membrane proteins to allow enough time for the encapsulated BamA to fold and insert into the liposome membrane. 
               </P><DIV class="image-container"><P><STRONG>Fig. 8</STRONG> Associational scheme of thermoswitches’ action in the SynDrop system. Not locking the concomitant translation of our target protein and BamA results in target protein aggregation due to insufficient membrane insertion and  assembling potential of BamA.</P></DIV><DIV class="image-container"><P><STRONG>Fig. 9</STRONG> Associational scheme of thermoswitches’ action in the SynDrop system. Locking up translation gives time for proper folding and insertion of BamA and prevents undesirable aggregation of target membrane proteins.</P></DIV><P>
                  Additionally, while creating SynDrop, we have considered various options on how to make our complex cell-free system more user-controllable and predictable. Cell-free systems are becoming an attractive platform for <VAR>in vitro</VAR> compartmentalization and protein research, and although usually compositionally sensitive, they also offer a platform for building synthetic genetic regulatory tools or logic gates. Both the need to control the translation time of target genes and desire to provide more modularity for our synthetic system, led us to exploring RNA thermometers as a viable option to perform these tasks. They have minimal molecular burden and are easy to modulate. These properties encouraged us to developed a library of synthetic RNA thermometers suitable to translationally regulate the expression of our fusion constructs in bacteria with a further possibility to transfer them to IVTT systems. All of the RNA thermometers including those we found in literature and our <VAR>de novo</VAR> modelled ones were optimized for best performance at 37<SUP>o</SUP>C, bearing in mind their future transition to IVTT system, whose optimum performance temperature is also 37<SUP>o</SUP>C. Consequently, our experiments showed that our synthetic RNA thermometers, despite their simplistic structures compared to naturally occurring ones, efficiently triggered the expression of target constructs at 37<SUP>o</SUP>C, and successfully locked it at lower temperatures having made them an ideal complement to our liposome IVTT system. All of our thermoswitches unlocked the expression to similarly high levels at 37<SUP>o</SUP>C, but differed in terms of leakiness and success at inhibiting translation at lower temperatures.
               </P><H1>Conclusions</H1><P>
                  We proved that herein described synthetic RNA thermometers enable high-yield expression of our constructs in an inducible temperature range. What is more important, this spectrum of temperature is compatible with currently used in vitro transcription and translation systems. Synthetic Thermoswitches allow the user-controllable and responsive protein translation for custom experiments. Finally, we introduced six de novo designed RNA thermoswitches which will have been used by future iGEM teams having to work both with cell-free and in vivo synthetic biology systems.
               </P><H2>References</H2><P><OL><LI> 1. Neupert J, Karcher D, Bock R. Design of simple synthetic RNA thermometers for temperature-controlled gene expression in Escherichia coli. Nucleic Acids Res. Oxford University Press; (2008); 36:e124–e124.</LI><LI>  2. Narberhaus F, Waldminghaus T, Chowdhury S. RNA thermometers. FEMS Microbiol. Rev. Wiley/Blackwell (10.1111); (2006); 30:3–16.</LI><LI> 3. Storz G. An RNA thermometer. Genes Dev. Cold Spring Harbor Laboratory Press; (1999); 13:633–6.</LI><LI>    4. Sen S, Apurva D, Satija R, Siegal D, Murray RM. Design of a Toolbox of RNA Thermometers. ACS Synth. Biol. (2017); 6:1461–70.</LI></OL></P></DIV></SECTION><SECTION class="design_subsections"><H1 id="Mistic_fusion_protein">Mistic fusion protein</H1><DIV class="third_level_links"><A href="#Liposomes">Liposomes</A><A href="#Ribosome_modifications">Ribosome modifications</A><A href="#BAM complex">BAM complex</A><A href="#RNA_Thermoswitches">RNA Thermoswitches</A><A href="#Mistic_fusion_protein">Mistic fusion protein</A><A href="#Surface_display_system">Surface display system</A></DIV><DIV><H1>Mistic Fusion Protein</H1><P>
                  Self-inserting <VAR>Bacillus subtilis</VAR> protein called Mistic (MstX) is originally involved in biofilm formation<SUP>1</SUP>. It is thought that MstX might directly chaperon the membrane insertion of potassium ion channel YugO. Together these proteins create a positive autoregulatory feedback loop that assists biofilm assembly in a population of cells and is mediated by a pathway involving potassium ion efflux<SUP>2</SUP>.
               </P><P>
                  MstX comprises 110 residues that are arranged into a four-helix bundle exposing numerous polar and charged amino acids. This α-helical protein is characterized by an uncommonly hydrophilic surface. Until this day there is a great debate on how MstX is able to autonomously associate with a lipid bilayer despite its hydrophilic surface <SUP>2</SUP>. It is known that three of the four MstX helices are much shorter than transmembrane helices of canonical integral MPs. In general, the four helices of this protein show no apparent differences in hydrophobicity or charge distribution among each other. 
               </P><P>
                  MstX was identified back in 2005 by Rooslid and colleagues. Interestingly, until this day little is known about how MstX promotes integral protein targeting to the membrane<SUP>3</SUP>. Recently it has found a novel application as a fusion tag supporting the recombinant production and bilayer insertion of other membrane proteins (MPs)<SUP>1</SUP>. MstX, when fused to the N-terminus of integral MPs, enables the cargo proteins to fold into their native conformations in the membrane, thus yielding high-level expression. It is known that MstX autonomously targets proteins to the membrane bypassing the canonical secretory apparatus, like Sec translocon. In addition to this, it was indirectly presumed that MstX lacks any recognizable signal sequence <SUP>2</SUP>. 
               </P><P>
                  According to this, we have decided to implement the advantages of MstX into our project. In order to boost protein expression yield MstX was fused with target integral membrane protein Outer membrane protein A (OmpA). Prior using these recombinant proteins in cell-free system, it has to be ensured that proteins are expressed in bacteria. To do so, <VAR>E. coli</VAR> BL21 stain cells were transformed and induced for 2 and 4 hours with isopropyl β-D-1-thiogalactopyranoside (IPTG). Results were analyzed by sodium dodecyl sulphate-polyacrylamide gel electrophoresis (SDS-PAGE). This experiment was conducted in order to detect if there are any differences in protein production yield comparing native and MstX-fused proteins. 
               </P><P>
                  First of all, it was concluded that MstX-OmpA-X and OmpA-X (X - additional part of the construct) are expressed in cells (Fig. 1). 
               </P><DIV class="image-container"><P><STRONG>Fig. 1 </STRONG> OmpA-X and MstX-OmpA-X (X - additional part of the construct; shown with arrows) expression in <VAR>E. coli</VAR>; SDS-PAGE after induction with IPTG for 2 hours and 4 hours; K - control, M - protein ladder </P></DIV><P>Also, we checked if we could fuse MstX with other integral membrane proteins. In this case, IgA protein (Fig. 3). 
               </P><DIV class="image-container"><P><STRONG>Fig. 2 </STRONG> IgA, IgA-MstX, and X-IgA-MstX (X - additional part of the construct; shown with arrows) expression in <VAR>E. coli</VAR>; SDS-PAGE; U4-2 - samples affected with 8M urea, S2-4 - samples affected with protein denaturation dye, K - control, M - protein ladder </P></DIV><P>
                  Analyzing electrophoresis gel, differences of native IgA protein and  recombinant IgA fused with MstX can be observed. It is extremely important that MstX in this case ensures higher-level yield of integral membrane proteins in <VAR>E. coli</VAR> as well.
               </P><P>
                  According to these results, we validated that chosen integral membrane proteins are expressed in cells. In addition to this, their expression in cell-free system could be also expected. Moreover, MstX increases these MPs yield by functioning as a chaperone and enabling our target proteins to fold into their native conformations in the membrane even more efficiently. 
               </P><P>
                  Using MstX in cell-free systems is extremely advantageous as it allows effortless protein research without any need for additional protein purification step. Also, this fusion tag is universal as MstX it is compatible with different kinds of membrane proteins. In addition to this, the final task was to refine our cell-free system with MstX in order to make system more efficient and even more suitable for protein research.
               </P><P>
                  To demonstrate that MstX is beneficial to cell-free system not only for membrane protein expression, it was fused with single-chain variable fragment (scFv; Fig. 2). It was decided to do so as scFvs are usually prone to form aggregates and lose their function <VAR>in vitro</VAR>. We thought that MstX could stabilize scFvs and prevent aggregation. As a result, we observed that MstX fusion could enhance scFv solubility allowing it to use in cell-free systems. 
               </P><DIV class="image-container"><P><STRONG>Fig. 3 </STRONG> Single-chain variable fragment (scFv) expression in IVTT system; SDS-PAGE. M - protein ladder, + - positive control DHFR, 1 - scFv, 2 - MstX-scFv, - negative control (without template DNA)</P></DIV><P>
                  Analyzing the reaction samples, sediments in scFv were observed, which meant that scFv aggregated. However, in MstX-scFv sample there were no sediments. By analysing electrophoresis results (Fig. 3) it can be seen that MstX prevented formation of the aggregates which resulted in higher scFv expression yield.
               </P><H2>References</H2><P><OL><LI>1. Broecker, J., Fiedler, S., Gimpl, K. &amp; Keller, S. Polar Interactions Trump Hydrophobicity in Stabilizing the Self-Inserting Membrane Protein Mistic. <VAR>Journal of the American Chemical Society</VAR> 136, 13761-13768 (2014).  </LI><LI>2. Textor., M. Reconstitution and Membrane Topology  of Mistic from <VAR>Bacillus subtilis</VAR> (Doctoral dissertation). <VAR>University of Kaiserslautern</VAR>. Retrieved from https://kluedo.ub.uni-kl.de </LI><LI>3. Lundberg, M. E. Biochemical and functional characterization of MISTIC. (Doctoral dissertation). <VAR>UC San Diego</VAR>. Retrieved from https://escholarship.org/uc/item/5b594287 (2013).                        </LI></OL></P></DIV></SECTION><SECTION class="design_subsections"><H1 id="Surface_display_system">Surface display system</H1><DIV class="third_level_links"><A href="#Liposomes">Liposomes</A><A href="#Ribosome_modifications">Ribosome modifications</A><A href="#BAM complex">BAM complex</A><A href="#RNA_Thermoswitches">RNA Thermoswitches</A><A href="#Mistic_fusion_protein">Mistic fusion protein</A><A href="#Surface_display_system">Surface display system</A></DIV><DIV><H1>ScFv Antibody</H1><H1>Background</H1><P>scFv consists of a minimal functional antigen-binding domain of an antibody (~30 kDa) (Fig. 1) , in which the heavy variable chain (VH) and light variable chain (VL) are connected by Ser and Gly rich flexible linker. [1] In most cases scFv is expressed in bacteria, where it is produced in cytoplasm, a reducing environment, in which disulfide bonds are not able to form and protein is quickly degraded or aggregated. Although poor solubility and affinity limit scFvs’ applications, their stability can be improved by merging with other proteins. [2] When expressed in cell free system, scFv should form disulfide bonds with the help of additional molecules. Merging to a membrane protein would provide additional stability and would display scFv on liposome membrane, where its activity could be detected. These improved qualities make ScFv recombinant proteins a perfect tool to evaluate, if SynDrop system acts in an anticipated manner. Of all possible scFvs we decided to use scFv-anti vaginolysin, which binds and neutralizes toxin vaginolysin (VLY). Its main advantage is rapid (&lt; 1 h) and cheap detection of activity by inhibition of erythrocyte lysis (Fig. 2). Looking into future applications, scFvs are also attractive targets of molecular evolution, because one round of  evolution would last less than one day thus generating a and wide range of different scFv mutants. Those displaying the highest affinity for antigens could be selected and used as drugs or drug carriers. </P><DIV class="image-container"><P><STRONG>Fig. 1 </STRONG>Simplified structure of scFv Antibody</P></DIV><DIV class="image-container"><P><STRONG>Fig. 2 </STRONG>Scheme of scFv_antiVLY and VLY interaction. Left- scFv_antiVLY binds to VLY, erythrocytes stay intact, Right- scFv_antiVLY does not bind and VLY lyse erythrocytes.</P></DIV><H1>Results</H1><P>scFv constructs were created <A href="http://parts.igem.org/Part:BBa_K2622004"> BBa_K2622004</A>. and checked by <A href="https://2018.igem.org/Team:Vilnius-Lithuania/Protocols"> colony PCR and DNA sequencing</A>. scFv synthesis was performed in a cell free system. Validation of protein expression was done by running a sodium dodecyl sulfate–polyacrylamide gel electrophoresis (SDS-PAGE), see (Fig. 3)</P><P><STRONG>Fig. 3 </STRONG> SDS-PAGE of scFv. GFP is used as positive control, C- chaperone DnaK.</P></DIV><P>Red arrows in the photo indicate scFv anti-vaginolysin (~27 kDa). As successful synthesis was confirmed, the next step was to check if protein folded correctly and was able to bind its antigen -  vaginolysin. We examined this by erythrocyte-lysis test, which was performed by comparing erythrocytes incubated with VLY (erythrocytes burst open) and erythrocytes incubated with VLY that was previously incubated with scFv anti-vaginolysin (less or no erythrocyte lysis). Results revealed that scFv binded to vaginolysin and inhibited cell lysis. Graph in (Fig. 4) demonstrates that scFv indeed attenuated the lysis of erythrocytes. These result prove scFv activity in IVTT system.</P><DIV class="image-container"><P><STRONG>Fig. 4 </STRONG> Percentage of erythrocyte lysis at different +/-scFv dilutions.</P></DIV><P>We then went one step further and constructed MstX-scFv_antiVLY <A href="http://parts.igem.org/Part:BBa_K2622038"> BBa_2622038</A>, fusion protein, aiming to increase the stability of scFv having in mind future applications and experiments of exposing it on liposome surface. Fusion protein was expressed in E.coli cells; yellow to red arrows in (Fig. 5A) indicate MstX-scFv expression after induction with IPTG.</P><P>Finally, we expressed the protein in a cell free system (Fig. 5B) along with scFv in order to compare how well scFv accomplishes its function alone or binded to other protein. In this case MstX-scFv_antiVLY fusion did not show superior activity than scFv_antiVLY alone (Fig. 6). These results also reveal that scFv_antiVLY is very sensitive and loses its activity with time. Ist and IInd attempts were separated by 1-2 hours. This amount of time is enough to measure decreasing activity. This must be taken into account when performing future experiments.</P><P><STRONG>Fig. 5 </STRONG>A- MstX-scFv_antiVLY expression in Escherichia coli. B- scFv_antiVLY and MstX-scFv_antiVLY expression in cell-free system.</P><P><STRONG>Fig. 6 </STRONG> Percentage of erythrocyte lysis at different scFv/MstX-scFv dilutions.</P><H1>Conclusions</H1><P>We successfully expressed scFv_antiVLY antibody and MstX-scFv_antiVLY construct in E.coli cells as well as in a cell free system. We demonstrated that our scFv anti-vaginolysin can bind to its target antigen vaginolysin and inhibit erythrocyte-lysis reaction. Based in scFv general properties in IVTT activity, we conclude that scFvs are an elegant addition to SynDrop liposome display system.</P><H1>Discussion</H1><P>Small size of scFv makes it a widely researched antibody. It’s ability to penetrate deeply into tissues and trait to elicit low to none organism’s immune response, makes scFv the one of the best candidates for medical, diagnostic, and research applications [3]. Efficient and fast method for scFv generation is in demand. SynDrop liposome display system offers an ability to produce scFv in IVTT system and display them on membranes to facilitate rapid antigen binding. scFv on the other hand can also help us prove that our system work either cost efficiently or with extreme precision. scFv surface display is compatible with using fluorescence assisted cell sorting (FACS) to detect well functioning liposomes. [4] This would reduce amount of time needed for mutant sorting compared with enzyme-linked immunosorbent assay. Second, scFv display is compatible with the experiments described in this section. We have performed erythrocyte-lysis tests to prove the functional activity of scFv anti-vaginolysin that was synthesized in the IVTT system. Not all experiments with VLY indicated positive antibody activity. We hypothesized that proteins could have aggregated very quickly after IVTT expression and the amount of active antibody left in the solution was not enough to inhibit VLY in quantities, what would have been detectable. This hypothesis was further supported by several experiments, which revealed decreasing scFv antibody’s functional activity with time. Moreover, not every experiment was done just after IVTT reaction completed and spend few hours in +4 ˚C. Another option to test scFv, single or displayed on liposomes to gain most reliable results, is an ELISA test. It requires specific antibodies and tags (His-6x or Strep-tag) on scFv or MstX-scFv. </P><H2>References</H2><P><OL><LI>Monnier, P., Vigouroux, R. &amp; Tassew, N. In Vivo Applications of Single Chain Fv (Variable Domain) (scFv) Fragments. Antibodies 2, 193-208 (2013).</LI><LI>Wang, R. et al. Engineering production of functional scFv antibody in E. coli by co-expressing the molecule chaperone Skp. Frontiers in Cellular and Infection Microbiology 3, (2013).</LI><LI>Ahmad, Z. et al. scFv Antibody: Principles and Clinical Application. Clinical and Developmental Immunology 2012, 1-15 (2012).</LI><LI>Vorauer-Uhl, K., Wagner, A., Borth, N. &amp; Katinger, H. Determination of liposome size distribution by flow cytometry. Cytometry 39, 166 (2000).</LI></OL></P></SECTION></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="invert-box"><SPAN class="invert-text">invert</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>