Team:Vilnius-Lithuania-OG/Demonstrate
Collaborations
Home
Project
Overview
Design
Design for Regulatory Parts
Modelling
Results
Proof of Concept
Measurement
Improve
Substrate Design Prospects
Entrepreneurship
Software
Motivation
Generative Adversarial Network – GAN
Protein GAN
Reaction GAN
Parts
Overview
Basic Parts
Composite Parts
Collection
Notebook
Using the CAT-Seq
Lab Journal
Interlab
Safety
Human practices
Human Practices
Education and Public Engagement
Collaborations
People
Team
Attributions
Sponsors
Proof of concept
Explore PROOF OF CONCEPT
Introduction
Mutant library activity
Regulatory part activity
Conclusions
Introducton
To demonstrate the CAT-Seq is working as intended, we chose an Esterase enzyme (CAT-Seq Esterase) capable of hydrolyzing N4-benzoyl-2'-deoxycytidine triphosphate to 2'-deoxycytidine triphosphate. We have then designed a library of the esterase mutants with the help of in-silico modelling and also acquired libraries of different ribosome binding sites and toehold switches. We have successfully shown that we can assess the derived esterase mutant and regulatory sequence activities and sequences by recording those activities into their DNA sequences in a high-throughput manner using N4-benzoyl-2'-deoxycytidine triphosphate as a Substrate Nucleotide and 5-methyl-2’-deoxycytidines as reference nucleotides. The activity recording is based on the ratio of catalytically converted and reference nucleotide incorporated into the  DNA during the amplification reaction and can be extracted during the analysis of Nanopore sequencing data.
Mutant library activity and sequence assessment
First of all, the constructed in silico designed mutant library was subjected to catalytic activity sequencing. By applying the data preparation and analysis pipeline, the mean methylation scores arising from different ratios of catalytically converted and reference nucleotides for each barcoded mutant DNA template were filtered and extracted from the DNA embedded with catalytic activity information (in a form of incorporated reference to catalytically converted nucleotide ratio). The collected data was normalized over Wild Type Esterase and  K227R mutant (lowest activity).
Figure 1. Comparison of In bulk and CAT-Seq measured esterase mutant relative activity. In silico generated Esterase mutant library was subjected to catalytic activity sequencing. The mean methylation scores for each barcoded mutant DNA template were filtered and extracted. The collected data was normalized over Wild Type CAT seq Esterase and K227R mutant (lowest activity). The relative activity, extracted from the mean methylation score of each mutant read is compared to measurement data gathered in standard sized reactions (in bulk).
The relative methylation score (reference nucleotide count) of each mutant read corresponds to the activity of the enzyme it encodes. The higher the activity of the expressed enzyme, the lower methylation score is assigned, due to the catalytic conversion of substrate nucleotides. The comparison of the results, gathered with CAT-Seq catalytic activity sequencing method and in standard sized reactions (in bulk) spectrophotometric data (Fig 1.) conclude the viability of CAT-Seq approach. The activity reading, extracted from the DNA sequence correlates with the in kinetic measurement data perfectly. The activity of each Esterase mutant is measured accurately and is assigned to the corresponding DNA sequence. These results prove and conclude, the validity of CAT-Seq as a method for screening the activity of millions of enzyme sequences and assigning the phenotype of each variant to the genotype it arises from.
Regulatory part activity and cross-interaction assessment
We have then adjusted the CAT-Seq for regulatory part characterization and showed that we can successfully record characteristics of different regulatory parts. The constructed Ribosome Binding site library ( BBa_B0030 , BBa_B0032 , BBa_B0034 , BBa_K2621038 with a downstream CAT-Seq esterase gene BBa_K2621000 ) was subjected to catalytic activity sequencing method. The mean methylation scores (reference nucleotide count) for each barcoded DNA template, housing different RBS were filtered and extracted from the DNA embedded with catalytic activity information (in a form of incorporated reference to catalytically converted nucleotide ratio). The collected activity data was normalized to BBa_B0034 data and is shown in Fig. 2.
Figure 2. Comparison of in bulk and CAT-Seq measured ribosome binding site relative strength.The catalytic activity of the esterase gene, regulated by a library of ribosome binding sites was measured using cell-free expression system in standard sized reactions (in bulk) or CAT-Seq approach and compared side by side. The mean methylation scores for each barcoded mutant DNA template were filtered and extracted. The collected data was normalized BBa_B0034 corresponding to mean strength of 1.
Stronger ribosome binding sites increase the yield of translated proteins and in turn increase the number of catalytically converted substrate nucleotides. This increase is inversely proportional to the assigned mean methylation score. Based on this fact, the activity results can be extracted from mean methylation scores (reference nucleotide count) and correspond to ribosome binding site strength. The catalytic activity sequencing results were compared to earlier measured in bulk RBS strength results. The comparison once again concludes the viability of CAT-Seq approach. The ribosome binding site strength, extracted from the DNA sequence reference nucleotide count correlates with the in bulk measurement data. These results display the validity of CAT-Seq as a method for screening the strength of regulatory sequences and its ability to assign accurate phenotype to genotype linkage.
In addition to ribosome binding sites, we have constructed Toehold regulatory sequence library constituted of a different toehold and triggers pairs was constructed subjected to catalytic activity sequencing method. The mean methylation scores (reference nucleotide count) for each barcoded DNA template, housing different regulatory sequence were filtered and extracted from the DNA embedded with catalytic activity information (in a form of incorporated reference to catalytically converted nucleotide ratio).
Figure 3. The evaluation of Toehold-Trigger riboregulatory sequence orthogonality using CAT-Seq. The catalytic activity of esterase genes, regulated by different Toehold switches were measured using CAT-Seq. The mean methylation scores for each barcoded regulatory construct DNA template was filtered and assigned. Low methylation scores correspond to actively expressed protein and are only assigned when both Toehold and trigger sequences from the same group are present verifying the already measured orthogonality of regulatory parts.
The graph displays the mean methylation (reference nucleotide) scores assigned to each barcoded toehold-trigger construct read. Based on the results, low methylation score is only assigned when both Toehold and trigger sequences from the same group are present, due tue esterase being expressed. These results correlate perfectly to the standard (not in droplet) measurement results, carried out earlier. Based on this fact it can be concluded that CAT-Seq activity sequencing method can be utilized as a precise and accurate way to screen and assign the activity and orthogonality of regulatory sequences.
Conclusion
The three proof of concept experiments presented above - mutant library activity and sequencing recording, ribosome binding site strength determination and toehold regulatory sequence cross-interaction evaluation - conclude that CAT-Seq catalytic activity sequencing workflow is a precise and high throughput method to screen millions of catalytic or regulatory molecules that allows to precisely link the measured absolute phenotypic data to the sequenced genotype.
