<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="project-wrapper"><HEADER class="header-area"><NAV class="navigation"><UL class="navigation-list"><LI id="project-list-item" class="navigation-list-item"><A class="navigation-list-item-anchor" href="https://2018.igem.org/Team:Vilnius-Lithuania/Description"><SPAN class="image-logo-text">project</SPAN></A></LI><LI id="parts-list-item" class="navigation-list-item"><A class="navigation-list-item-anchor" href="https://2018.igem.org/Team:Vilnius-Lithuania/Overview"><SPAN class="image-logo-text">parts</SPAN></A></LI><LI id="notebook-list-item" class="navigation-list-item"><A class="navigation-list-item-anchor" href="https://2018.igem.org/Team:Vilnius-Lithuania/LabBook"><SPAN class="image-logo-text">notebook</SPAN></A></LI><LI id="hp-list-item" class="navigation-list-item"><A class="navigation-list-item-anchor" href="https://2018.igem.org/Team:Vilnius-Lithuania/Human_Practices"><SPAN class="image-logo-text">human practices</SPAN></A></LI><LI id="team-list-item" class="navigation-list-item"><A class="navigation-list-item-anchor" href="https://2018.igem.org/Team:Vilnius-Lithuania/Members"><SPAN class="image-logo-text">team</SPAN></A></LI></UL></NAV></HEADER><DIV class="content-area"><DIV class="text-wall-carrot-container"><DIV class="text-wall-area"><H1 class="text-wall-heading">Description</H1><DIV class="text-wall-area-box"><H2 class="text-wall-area-box-heading">Describe the Impossible</H2><DIV class="scroll-area"><P class="text-content">Cell-free systems are becoming an increasingly popular in vitro tool to study biological processes as it is accompanied by less intrinsic and extrinsic noise. Relying on fundamental concepts of synthetic biology, we apply a bottom-up forward engineering approach to create a novel cell-free system for unorthodox protein-evolution. The core of this system is cell-sized liposomes that serve as excellent artificial membrane models. By encapsulating genetic material and full in vitro protein transcription and translation systems within the liposomes, we create reliable and incredibly efficient nanofactories for the production of target proteins. Even though there are many alternative proteins that can be synthesized, our main focus is directed towards membrane proteins, which occupy approximately one third of living-cells’ genomes. Considering their significance, membrane proteins are spectacularly understudied since synthesis and thus characterization of them remain prevailing obstacles to this day. We aim to utilize liposomes as nanofactories for directed evolution of membrane proteins. Furthermore, by means of directed membrane protein-evolution, a universal exposition system will be designed in order to display any protein of interest on the surface of the liposome. This way, a system is built where a phenotype of a particular protein is expressed on the outside while containing its genotype within the liposome. To prove the concept, small antibody fragments will be displayed to create a single-chain variable fragment (scFv) library for rapid screening of any designated target.</P><BUTTON class="read-more-button">Read More</BUTTON></DIV></DIV><DIV class="pagination"><DIV class="pagination-item-wrapper"><A class="pagination-anchor"><SPAN class="pagination-text">Description</SPAN></A></DIV></DIV><DIV class="modal"><DIV class="modal-content"><P><STRONG>Fig. 1</STRONG> Fig. 1. Schematic overview of the SynDrop. Using microfluidic technology, we synthesize liposomes at a high throughput within which we encapsulate an in vitro transcription/translation system with DNA encoding a target membrane protein. In addition we encapsulate purified cellular membrane protein machinery and chaperones - they facilitate the insertion of synthesized target membrane proteins into the membrane. The system is also capable of displaying small molecules on the surface of the liposome, which makes SynDrop applicable for a novel liposome surface-display method.
</P><H2><VAR>“What I cannot create, I do not understand”</VAR></H2><H6>R. Feynmann, February 1988</H6><H2 style="color: #61afaa; font-size:1.8em">Brief overview of the SynDrop - Synthetic Droplets for Membrane Protein Research.</H2><P>
            SynDrop started from the idea of working towards developing a minimal synthetic cell. However it was soon realized that synthetic life is not something that will be made in one go - it will be the culmination of all the small, separate systems that will come together and work in unison. As such, it was understood that these systems need to be independently functional and well described first, before more complex systems are built on top of them. One of the most fundamental differences between life and synthetic systems is the responsivity and communication with the surrounding environment. This function in living cells is mostly performed by membrane proteins. We quickly realized that in order to make a significant impact on synthetic life development, membrane proteins are that understudied field that holds great potential for future applications in synthetic biology. Fig. 1 beautifully summarizes the workflow, complexity and at the same time minimalism of SynDrop. We utilized the emerging technology of microfluidics to synthesize cell-sized liposomes and provide them with the minimal set of all the necessary tools and machineries for the successful synthesis of membrane proteins. These fully equipped liposomes form the core of the SynDrop. Within them are encapsulated purified BAM complex proteins and the chaperone SurA which facilitate beta-barrel bearing protein assembly. Liposomes also contain genetically engineered membrane-associating ribosomes which increase the yields of target protein expression. SynDrop liposomes contain an <VAR>in vitro</VAR> transcription-translation system and custom DNA. Their inner aqueous environment is suitable for molecular reactions to occur. Finally, SynDrop provides a novel platform for protein display, whether they were antibodies, single chain fragments, globular proteins, or peptides. It is a huge step forward in membrane protein research and perhaps another resolved puzzle towards the creation of synthetic cell.

    </P><H1>Background</H1><P>
            Synthetic biologists have come a long way since 1912 Stéphane Leduc’s <VAR>La Biologie Synthétique</VAR>. Throughout the course of synthetic biology, there were many stepping stones that lead to greater things, such as the discovery of restriction enzymes which lead to the simplified construction of recombinant DNA molecules and arrangements of new genes, or the creation of synthetic biological circuit devices by combining different genes within E. coli. Through these canonical inventions we acquired not only better tools, but also greater ambitions. From understanding how genes work and that they can be modified or replaced, to programming cellular behavior with external impulses, we have reached a point, where no longer the singular modal elements like switches, cascades, pulse generators or oscillators matter. We have reached a point where scientists are ready to face the biggest challenge of synthetic biology - creating an artificial cell.
    </P><P>
            However designing complex, several layered circuitries resembling the behavior of a natural cell is still an overwhelming challenge due to many limitations like crosstalk, mutations, ambiguous intracellular and extracellular conditions, and biological noise. Therefore we propose to start from something simpler and more minimal.. Although the journey of creating a synthetic minimal cell has already begun, we hoped to contribute to this ultimate goal as well by investing our time and effort. This year we are engineering liposomes, lipid-coated vesicles, that are perfect models to study the initial steps for creating synthetic cells. Liposomes can offer a system with fully controllable experimental parameters and only the exact elements for our custom circuit design without the need to ever worry about the crosstalk and noise. We believe that most of the future synthetic biology a
pplications will rely on bottom-up engineering solutions. Having mastered some hard-core bottom-up liposome engineering, we won’t take long to create the first synthetic cell.
    </P><P>
            Keeping that in mind we raised a question - what is the trivial difference between completely artificial systems like liposomes and living cells? The answer has pushed us to develop the SynDrop system the way it is being presented today. That major difference between synthetic and natural systems is the capacity to develop an active interface between outside and inside, which is vital for communication, transport, signaling, growth, and proliferation. Most of this communication is mediated by integral membrane proteins. Surprisingly, though membrane proteins are key players in many cellular functions and even human health and disease, they are greatly understudied when compared to e.g. globular proteins. Membrane protein integration into the membranes <VAR>in vitro</VAR> is still a particularly delicate issue that halts more rapid scientific development in this field. Having realized this, we knew that our goal this year was going to be the creation of an <VAR>in vitro</VAR> membrane protein synthesis system in liposomes in order to not only broaden the research possibilities of integral proteins but also to move one step further in building a minimal cell, which with what SynDrop offers - now will be capable of executing at least minimal .communication between its inside (genotype) and surrounding environment (display phenotype).
    </P><H1><VAR>SynDrop - Synthetic Droplets for Membrane Protein Research</VAR></H1><P>
            Current methods for studying MPs are mostly based on cellular systems, which are really not the best environment to study membrane proteins. Large intrinsic and extrinsic cellular noise makes it awfully difficult to characterize discrete MPs as well as other limitations like toxicity and limited yields. Only a few self-integrating membrane proteins have been utilized for research as the majority of them aggregate or are toxic to the host. The capacity to vary parameters is too restrictive for an in-depth investigation of a complete spatiotemporal arrangement of a particular protein or mechanism.
    </P><P>
            As mentioned above, to untangle these problems we decided not to try to decipher and modulate natural cells. We chose something much more simple - a <VAR>cell-free</VAR> synthetic system with well defined, easily controlled parameters, that could be engineered from scratch. We called it SynDrop - Synthetic Droplets for Membrane Protein Research. SynDrop edges not only cellular systems, but also other current cell-free systems in terms of membrane protein research and studying their behavior, including folding and membrane insertion.
    </P><P>
            Thus SynDrop is a novel bottom-up liposome-based platform created to empower the manipulation of membrane proteins. It is carefully though and bares the exact minimum components to successfully perform its function and not become too unpredictable or metabolically unstable. Transcriptional and translational machineries together with the plasmid DNA, chaperones, energy regeneration systems, and cellular MP insertion facilitators are encapsulated within the liposomes, enabling synthesis, integration, and display of target membrane proteins. It is also equipped with synthetic tools that regulate, attenuate and modulate the whole system.
    </P><H1>Applications</H1><P>As our project focuses on a novel platform for membrane protein research it offers various future applications.
    </P><H2>1. Membrane protein characterization</H2><P>Since liposome systems contain no unknown variables, SynDrop could be used to characterize membrane proteins and their biogenesis. This could possibly unlock new and innovative breakthrough in proteomics.</P><H2>2. Membrane protein-lipid interactions</H2><P>Microfluidics allow to synthetize liposomes with modifying their lipid composition rather simply. By using SynDrop we can quickly get huge quantities of liposomes with various lipid compositions. Further, membrane proteins translated in these liposomes will interact accordingly with these lipids. This characterization of membrane protein interactions with custom lipids can be modelled and then implemented in <VAR>in vivo</VAR> systems.</P><H2>3. Membrane protein-membrane protein interactions</H2><P> Expanding even further of the membrane protein interactions. The amount of genetic material that gets encapsulated in the liposome can be regulated. Therefore multiple genes coding for different membrane proteins can be translated inside liposomes. It yields multiple membrane protein-protein interactions that can be measured and modelled accordingly.</P><H2>4. Molecular evolution of the exposed particle</H2><P> Liposomes could be effectively used in molecular evolution. We constructed a few membrane proteins that proved to display a protein particle on the surface of bacteria. By incorporating a few membrane protein integration helpers, we believe that liposome exposition could serve as a better alternative to current molecular evolution methods.</P><H2>5. Molecular evolution of the membrane protein</H2><P>Molecular evolution could be also applied not only for the exposed protein particle, but also for the membrane protein itself. Since liposomes have no additional noise and contains a changeable lipid bilayer, it could be one of the most feasible methods of membrane protein evolution.</P><H2>6. Membrane protein biogenesis regulation</H2><P>DNA or RNA that gets encapsulated inside liposomes can be altered. Therefore we can insert modifications that change the rate and volume of membrane protein biogenesis. For example, modifications might include thermoswitches that we used in our project to regulate the rate of translation inside the liposomes, so that one membrane protein has the priority to get translated over the other. These or similar modifications might lead to a fully functional one or more genetic circuit incorporation into liposomes that regulate the biogenesis of membrane proteins.</P><H2>7. Liposome communication</H2><P>Microfluidic field enables the production of similar or different composition liposomes. Theoretically, these liposomes could cross-react with each other or even share signals. This could create an autonomous artificial ecosystem and could lead towards the creation of artificial life.</P><H1>Background of other display systems</H1><P>
            An extensive literature search was performed to analyze every display system type that currently exists in order to be sure about the necessity and applicability of our project.
    </P><H2>Phage display</H2><TABLE class="c65"><TBODY><TR class="c25"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Library size </SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c7"><SPAN class="c17">10</SPAN><SPAN class="c17 c31"><SUP>7</SUP></SPAN></P></TD></TR><TR class="c1"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Transformation required</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Yes</SPAN></P></TD></TR><TR class="c55"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Mechanism </SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Phage display systems can be grouped into two classes: true phage vectors and phagemid vectors. In both cases, the protein to be displayed is fused to the capsid protein. </SPAN></P></TD></TR><TR class="c25"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Evolution</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Typically the phage library screening entails several (usually three) consecutive rounds of panning and phage amplification before the selected phage and the polypeptide that they present are individually analyzed.</SPAN></P></TD></TR><TR class="c54"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Protein displayed </SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Fv, scFv or Fab fragments</SPAN></P></TD></TR><TR class="c25"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Proteins to be displayed </SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Soluble, non-toxic, compatible with crossing membranes</SPAN></P></TD></TR><TR class="c25"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Surface anchorage </SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Capsid proteins</SPAN></P></TD></TR><TR class="c25"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Properties of protein enhanced</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Affinity, enzymatic activity, stability, folding, selection from cDNA libraries</SPAN></P></TD></TR><TR class="c92"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Stability</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Stable</SPAN></P></TD></TR><TR class="c80"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Applications</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><UL class="c20 lst-kix_8x5880tr1n6b-0 start"><LI class="c11"><SPAN class="c2">To enhance catalytic activity, proteolytic stability, or attach conjugation site and photoaffinity labels </SPAN></LI><LI class="c11"><SPAN class="c2">Affinity selection of protein fragments expressed from cDNA fragments</SPAN></LI><LI class="c11"><SPAN class="c2">Immunotherapy</SPAN></LI><LI class="c11"><SPAN class="c2">Developing diagnostic or therapeutic reagents in medicine</SPAN></LI><LI class="c11"><SPAN class="c2">Developing nanomaterials in material science</SPAN></LI><LI class="c45 c67"><SPAN class="c2">Identifying receptor agonists or antagonists in cell biology</SPAN></LI></UL></TD></TR><TR class="c19"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Advantages</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><UL class="c20 lst-kix_r1elq1tnko7i-0 start"><LI class="c7 c67"><SPAN class="c2">Biopanning of phage libraries on whole cells</SPAN></LI><LI class="c7 c67"><SPAN class="c2">Wide range of pH could be used for selection</SPAN></LI></UL></TD></TR><TR class="c25"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Disadvantages</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><UL class="c20 lst-kix_vaxyq0tqgdpb-0 start"><LI class="c34"><SPAN class="c2">The low display rate can be a drawback. In the first round of panning where very few binders are to be enriched from a huge excess of unwanted phages, phages that are mostly “bald” can reduce the accessible molecular diversity of the library and the efficiency of the system</SPAN></LI><LI class="c34"><SPAN class="c2">If selection is done on cells, non specific phage binding is observed, due to phage protein interaction with cell membrane proteins </SPAN></LI><LI class="c62"><SPAN class="c17">In phage display, libraries must be transformed into bacteria, limiting the number of possible independent sequences to 10</SPAN><SPAN class="c17 c31"><SUP>9</SUP></SPAN><SPAN class="c17">–10</SPAN><SPAN class="c17 c31"><SUP>10</SUP></SPAN></LI></UL></TD></TR><TR class="c105"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Membrane protein research</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Compatible</SPAN></P><P class="c12"><SPAN class="c2">Nogo-66 - monotopic membrane protein (7,5 kDa)ShuA - beta barrel forming protein, which pass through membrane 22 times (72.5 kDa)MOMP - beta barrel forming protein, which pass through membrane 16 times (40 kDa)Nef - peripheral protein (23 kDa)Neuromodulin - peripheral protein (25 kDa)Caveolin-1 - monotopic membrane protein (22 kDa)</SPAN></P></TD></TR><TR class="c25"><TD class="c18" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">References</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c12"><SPAN class="c17">1. </SPAN><SPAN class="c17 c29">Imai, S. et al. Development of an antibody proteomics system using a phage antibody library for efficient screening of biomarker proteins. </SPAN><SPAN class="c5 c29">Biomaterials</SPAN><SPAN class="c17 c29"> 32, 162-169 (2011).</SPAN><SPAN class="c17"> 2. </SPAN><SPAN class="c17 c29">Petrenko, V. Evolution of phage display: from bioactive peptides to bioselective nanomaterials. </SPAN><SPAN class="c5 c29">Expert Opinion on Drug Delivery</SPAN><SPAN class="c17 c29"> 5, 825-836 (2008).</SPAN></P><P class="c12"><SPAN class="c17 c29">3. Paschke, M. Phage display systems and their applications. </SPAN><SPAN class="c5 c29">Applied Microbiology and Biotechnology</SPAN><SPAN class="c17 c29"> 70, 2-11 (2005).</SPAN><SPAN class="c17 c29">4. Reiersen, H. Covalent antibody display--an in vitro antibody-DNA library selection system. </SPAN><SPAN class="c5 c29">Nucleic Acids Research</SPAN><SPAN class="c17 c29"> 33, 10 (2005).</SPAN><SPAN class="c17 c29">5. Vithayathil, R., Hooy, R., Cocco, M. &amp; Weiss, G. The Scope of Phage Display for Membrane Proteins. </SPAN><SPAN class="c5 c29">Journal of Molecular Biology</SPAN><SPAN class="c17 c29"> 414, 499-510 (2011).</SPAN><SPAN class="c17 c29">6. Morrison, K. &amp; Weiss, G. Combinatorial alanine-scanning. </SPAN><SPAN class="c5 c29">Current Opinion in Chemical Biology</SPAN><SPAN class="c17 c29"> 5, 302-307 (2001).</SPAN><SPAN class="c17 c29">7. Clackson, T. &amp; Wells, J. In vitro selection from protein and peptide libraries. </SPAN><SPAN class="c5 c29">Trends in Biotechnology</SPAN><SPAN class="c17 c29"> 12, 173-184 (1994).</SPAN><SPAN class="c17 c29">8. Galán, A. et al. Library-based display technologies: where do we stand?. </SPAN><SPAN class="c5 c29">Molecular BioSystems</SPAN><SPAN class="c17 c29"> 12, 2342-2358 (2016).</SPAN></P></TD></TR></TBODY></TABLE><H2>Ribosome Display</H2><TABLE class="c65"><TBODY><TR class="c44"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Library size </SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c7"><SPAN class="c17">10</SPAN><SPAN class="c17 c31"><SUP>13-14</SUP></SPAN></P></TD></TR><TR class="c44"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Transformation required</SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">No</SPAN></P></TD></TR><TR class="c44"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Mechanism </SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">DNA sequence of protein does not have a STOP codon and features a linker sequence at the C-terminus. Instead of detaching, the transcribed RNA and translated protein remain connected to the ribosome</SPAN></P></TD></TR><TR class="c44"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Evolution</SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c12"><SPAN class="c17">High affinity binders can be generated after 1</SPAN><SPAN class="c17 c31"><SUP>st</SUP></SPAN><SPAN class="c2">round of selection, but most of the time, multiple rounds are done</SPAN></P></TD></TR><TR class="c44"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Protein displayed </SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Antibody scFv fragments, &quot;Designed Ankyrin Repeat Proteins&quot;- DARPins, camelid nanobodies, DNA-binding proteins, receptors, membrane proteins</SPAN></P></TD></TR><TR class="c44"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Proteins to be displayed </SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Most proteins including cytotoxic, chemically modified and membrane proteins</SPAN></P></TD></TR><TR class="c44"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Surface anchorage </SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Ribosome</SPAN></P></TD></TR><TR class="c44"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Properties of protein enhanced</SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Affinity, enzymatic activity (used rarely; water-in-oil emulsions have more advantages in this field), stability</SPAN></P></TD></TR><TR class="c44"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Stability </SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c7"><SPAN class="c17">Stable in Mg</SPAN><SPAN class="c17 c31"><SUP>2+</SUP></SPAN></P></TD></TR><TR class="c44"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Applications</SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Cancer treatment, antibody engineering, proteomics, diagnostics and therapeutics</SPAN></P></TD></TR><TR class="c82"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Advantages</SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><UL class="c20 lst-kix_8k5bwrjmi1qe-0 start"><LI class="c34"><SPAN class="c17">Fast generation of big library as the complex is stabilized by Mg</SPAN><SPAN class="c17 c31"><SUP>2+</SUP></SPAN><SPAN class="c2">and can be readily dissociated by the addition of EDTA</SPAN></LI><LI class="c34"><SPAN class="c17">A signiﬁcant advantage of </SPAN><SPAN class="c5"><VAR>in vitro</VAR></SPAN><SPAN class="c2"> translation methods is the ability to modify the genetic code to allow the incorporation of non-canonical, unnatural amino acids, to give molecules with novel properties, such as cyclic peptides with increased serum, stability </SPAN></LI><LI class="c62"><SPAN class="c2">Rapid isolation and direct evolution of high-affinity functional proteins, particularly antibodies</SPAN></LI></UL></TD></TR><TR class="c112"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Disadvantages</SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><UL class="c20 lst-kix_qsrhfxweq4fz-0 start"><LI class="c11"><SPAN class="c2">By introducing mutations during PCR, a stop codon might appear in the middle of the sequence - the protein is unable to fold properly and mutant is lost. Moreover, RNA is less stable than DNA and requires an RNAse free environment, otherwise the majority of RNA will be degraded before reverse transcribed to DNA</SPAN></LI><LI class="c11"><SPAN class="c2"> In order to incorporate non canonical amino acids, DNA sequence has to be modified by introducing amber codons and specifically modified ribosomes have to be used</SPAN></LI><LI class="c11"><SPAN class="c2">Stem loops have to be added to the 5' and 3' ends, to stabilize RNA and a linker at the 3' end before the loops</SPAN></LI><LI class="c45 c67"><SPAN class="c2">Polysomes can reduce the library size, because the protein is not formed and aggregates.</SPAN></LI></UL></TD></TR><TR class="c95"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Membrane protein research </SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Incompatible. Membrane proteins are too hydrophobic and will not function if displayed. No pore forming activity or transfer activity would be evaluated</SPAN></P></TD></TR><TR class="c44"><TD class="c24" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">References</SPAN></P></TD><TD class="c0" colspan="1" rowspan="1"><P class="c12"><SPAN class="c17 c29">1. Lipovsek, D. &amp; Plückthun, A. In-vitro protein evolution by ribosome display and mRNA display. Journal of Immunological Methods 290, 51-67 (2004).</SPAN><SPAN class="c17 c29">2. He, M. &amp; Khan, F. Ribosome display: next-generation display technologies for production of antibodies in vitro. Expert Review of Proteomics 2, 421-430 (2005).</SPAN><SPAN class="c17 c29">3. Zahnd, C., Amstutz, P. &amp; Plückthun, A. Ribosome display: selecting and evolving proteins in vitro that specifically bind to a target. Nature Methods 4, 269-279 (2007).</SPAN></P></TD></TR></TBODY></TABLE><H2>Cis-Activity (Cis) Display</H2><TABLE class="c65"><TBODY><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Library size </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c17">10</SPAN><SPAN class="c17 c31"><SUP>10</SUP></SPAN></P></TD></TR><TR class="c54"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Transformation required</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">No</SPAN></P></TD></TR><TR class="c60"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Mechanism </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Replication initiator protein (RepA) binds to the DNA template from which it has been expressed, a property called cis-activity. The protein of interest is binded to RepA protein C-terminus. </SPAN></P></TD></TR><TR class="c72"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Protein displayed </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Antibody scFv fragments</SPAN></P></TD></TR><TR class="c54"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Proteins to be displayed </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Soluble, including cytotoxic, chemically modified</SPAN></P></TD></TR><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Surface anchorage </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c5 c23">In vitro</SPAN></P></TD></TR><TR class="c69"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Properties of protein enhanced</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Affinity, stability, resistance to degradation, longer half-life</SPAN></P></TD></TR><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Stability </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Stable (&gt;2 days)</SPAN></P></TD></TR><TR class="c37"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Applications</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">For the selection of high affinity peptides and folded protein domains, including antibody fragments</SPAN></P></TD></TR><TR class="c86"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Advantages</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><UL class="c20 lst-kix_dpgpa2ohx90u-0 start"><LI class="c34"><SPAN class="c2">Does not require any compartmentalization of the library-encoding nucleic acid</SPAN></LI><LI class="c34"><SPAN class="c2">The use of DNA to encode the displayed peptides provides advantages over RNA-based in vitro selection methods </SPAN></LI><LI class="c34"><SPAN class="c2">Very large libraries can be rapidly constructed and screened without separating transcription and translation steps and without purification of the protein–DNA complexes before selection </SPAN></LI><LI class="c34"><SPAN class="c2">Complexes do not require incubation under sterile or ribonuclease-free conditions</SPAN></LI><LI class="c62"><SPAN class="c2">Optimal ligands after 3-5 successive rounds </SPAN></LI></UL></TD></TR><TR class="c100"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">References</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c12"><SPAN class="c17 c29">Odegrip, R. et al. CIS display: In vitro selection of peptides from libraries of protein-DNA complexes. </SPAN><SPAN class="c5 c29">Proceedings of the National Academy of Sciences</SPAN><SPAN class="c17 c29"> 101, 2806-2810 (2004).</SPAN></P></TD></TR></TBODY></TABLE><H2>mRNA Display</H2><TABLE class="c65"><TBODY><TR class="c1"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Library size </SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c7"><SPAN class="c17">10</SPAN><SPAN class="c17 c31"><SUP>14</SUP></SPAN></P></TD></TR><TR class="c54"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Transformation required</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">No</SPAN></P></TD></TR><TR class="c32"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Mechanism </SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Puromycin with a DNA linker is ligated to RNA after mRNA library generation. Translation is carried out and the puromycin molecule enters the P-site of ribosome. It takes the role of tRNA, which leads to the growing peptide chain being covalently connected to the puromycin molecule. </SPAN></P></TD></TR><TR class="c99"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Evolution</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">To produce mRNA-displayed proteins requires ~3 days, to subject them to selection and evolution of enzymes for bond-forming reactions requires ~4-10 weeks</SPAN></P></TD></TR><TR class="c1"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Protein displayed </SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Antibody scFv, RasIn1 and RasIn2 proteins</SPAN></P></TD></TR><TR class="c70"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Proteins to be displayed</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Soluble, including cytotoxic, chemically modified</SPAN></P></TD></TR><TR class="c1"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Surface anchorage</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c7"><SPAN class="c23 c5">In vitro</SPAN></P></TD></TR><TR class="c19"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Properties of protein enhanced</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Affinity, stability</SPAN></P></TD></TR><TR class="c1"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Stability</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Covalent</SPAN></P></TD></TR><TR class="c77"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Applications</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><UL class="c20 lst-kix_w6shjt5topg6-0 start"><LI class="c7 c67"><SPAN class="c2">Investigation of protein-protein interactions and the development and selection of peptides, enzymes, scFvs, and novel binders based on alternative scaffolds</SPAN></LI><LI class="c7 c67"><SPAN class="c2">Protein evolution </SPAN></LI></UL></TD></TR><TR class="c82"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Advantages</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><UL class="c20 lst-kix_dpgpa2ohx90u-0"><LI class="c34"><SPAN class="c17">Useful for performing </SPAN><SPAN class="c5">in vitro</SPAN><SPAN class="c2"> selections in harsh environments that are not compatible with ribosome display. If the RNA stability is of concern in certain environments, the RNA can be replaced by its cDNA, as described in a variation of the mRNA display</SPAN></LI><LI class="c62"><SPAN class="c2">The mRNA display system allows libraries with sequence complexity approximately 10,000-fold that of phage display, 106-fold over yeast display or yeast two- and three- hybrid systems, and approximately 109-fold over colony screening approaches</SPAN></LI></UL></TD></TR><TR class="c101"><TD class="c38" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Disadvantages</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><UL class="c20 lst-kix_dpgpa2ohx90u-0"><LI class="c11"><SPAN class="c2">Time consuming, technically demanding </SPAN></LI><LI class="c45 c67"><SPAN class="c2">mRNA display has only been used for bond-forming enzymes (not to select enzymes that catalyze bond-breaking reactions or other covalent modification reactions)</SPAN></LI></UL></TD></TR><TR class="c110"><TD class="c38" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Membrane protein research</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Incompatible. Due to the limited expression of membrane-bound proteins by in vitro translation systems, mRNA-display cannot be utilized to address membrane protein related questions</SPAN></P></TD></TR><TR class="c97"><TD class="c38" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">References</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c12"><SPAN class="c17">1. </SPAN><SPAN class="c17 c29">Takahashi, T., Austin, R. &amp; Roberts, R. mRNA display: ligand discovery, interaction analysis and beyond. </SPAN><SPAN class="c5 c29">Trends in Biochemical Sciences</SPAN><SPAN class="c17 c29"> 28, 159-165 (2003).</SPAN><SPAN class="c17"> 2. </SPAN><SPAN class="c17 c29">Seelig, B. mRNA display for the selection and evolution of enzymes from in vitro-translated protein libraries. </SPAN><SPAN class="c5 c29">Nature Protocols</SPAN><SPAN class="c17 c29"> 6, 540-552 (2011).</SPAN><SPAN class="c2"> </SPAN></P><P class="c12"><SPAN class="c17">3. </SPAN><SPAN class="c17 c29">Cetin, M. et al. RasIns: Genetically Encoded Intrabodies of Activated Ras Proteins. </SPAN><SPAN class="c5 c29">Journal of Molecular Biology</SPAN><SPAN class="c17 c29"> 429, 562-573 (2017).</SPAN><SPAN class="c17">4. </SPAN><SPAN class="c17 c29">Lin, H. &amp; Cornish, V. Screening and Selection Methods for Large-Scale Analysis of Protein Function. </SPAN><SPAN class="c5 c29">ChemInform</SPAN><SPAN class="c17 c29"> 34, (2003).</SPAN></P></TD></TR></TBODY></TABLE><H2>Covalent Antibody Display</H2><TABLE class="c65"><TBODY><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Library size </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c17">10</SPAN><SPAN class="c17 c31"><SUP>7</SUP></SPAN></P></TD></TR><TR class="c28"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Transformation required</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">No</SPAN></P></TD></TR><TR class="c81"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Mechanism </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">A fusion protein - P2A and an scFv antibody bind to the same molecule of DNA from which it has been expressed. Following in vitro coupled transcription and translation, the P2A protein makes a covalent link between scFv genotype and scFv phenotype, by producing a stable protein–DNA complex</SPAN></P></TD></TR><TR class="c73"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Evolution</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Fast cycle time: in a few hours, unique scFvs can be enriched, isolated and directly amplified for the next rounds of selection</SPAN></P></TD></TR><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Protein displayed </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Antibody scFv fragments</SPAN></P></TD></TR><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Proteins to be displayed</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Developed for antibody display</SPAN></P></TD></TR><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Surface anchorage</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c23 c5">In vitro</SPAN></P></TD></TR><TR class="c19"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Properties of protein enhanced</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Affinity selections</SPAN></P></TD></TR><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Stability</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Covalent</SPAN></P></TD></TR><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Applications</SPAN></P></TD></TR><TR class="c61"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Advantages</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><UL class="c20 lst-kix_itgz30egtqyy-0 start"><LI class="c34"><SPAN class="c2">scFv can be genetically fused to the P2A protein creating the smallest imaginable antibody selection particle: a protein and its gene</SPAN></LI><LI class="c34"><SPAN class="c2">CAD is the only antibody selection method providing a direct covalent link between an scFv gene and its protein, increasing the chemical stability of the panning complex</SPAN></LI></UL></TD></TR><TR class="c95"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Disadvantages</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Product inhibition of uncomplexed fusion proteins without DNA</SPAN></P></TD></TR><TR class="c19"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Membrane protein research </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Incompatible - only used for antibody display and no membrane proteins can be evolved with this system</SPAN></P></TD></TR><TR class="c110"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">References</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c12"><SPAN class="c17 c29">Reiersen, H. Covalent antibody display--an in vitro antibody-DNA library selection system. </SPAN><SPAN class="c5 c29">Nucleic Acids Research</SPAN><SPAN class="c17 c29"> 33, 10 (2005).</SPAN></P></TD></TR></TBODY></TABLE><H2>Yeast Explay</H2><TABLE class="c65"><TBODY><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Library size </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c17">Up to 10</SPAN><SPAN class="c17 c31"><SUP>14</SUP></SPAN></P></TD></TR><TR class="c54"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Transformation required</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Yes</SPAN></P></TD></TR><TR class="c107"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Mechanism </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">The most common yeast display system employs fusion of the protein of interest to the C-terminus of the Aga2p subunit. Induction of protein expression results in surface display of the fusion protein through disulfide bond formation of Aga2p to the β1,6-glucan-anchored Aga1p domain of agglutinin. The epitope tags allow quantification of fusion protein expression, and thus normalization of protein function to expression level by flow cytometry using fluorescently labeled antibodies</SPAN></P></TD></TR><TR class="c19"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Protein displayed </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Antibody scFv fragments, Fab, single-chain T cell receptors, major histocompatibility complex (MHC)</SPAN></P></TD></TR><TR class="c25"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Proteins to be displayed</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Soluble and membrane, nontoxic, compatible with crossing membranes</SPAN></P></TD></TR><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Surface anchorage </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Agglutination proteins, flocculation proteins</SPAN></P></TD></TR><TR class="c19"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Properties of protein enhanced</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Affinity, specificity</SPAN></P></TD></TR><TR class="c49"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Applications</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><UL class="c20 lst-kix_or03z4mjd684-0 start"><LI class="c34"><SPAN class="c2">Indispensable and efficient way for affinity maturation in antibody engineering</SPAN></LI><LI class="c62"><SPAN class="c2">A method to immobilize enzymes and pathogen-derived proteins for vaccine development  </SPAN></LI></UL></TD></TR><TR class="c102"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Advantages</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><UL class="c20 lst-kix_rxypapex9qc0-0 start"><LI class="c34"><SPAN class="c2">Yeast as eukaryotic microorganism owns post-translational modifications as well as processing machinery and conditions homologous to the mammals</SPAN></LI><LI class="c11"><SPAN class="c2">Possesses eukaryotic machinery to express, assemble and export onto the surface both monomeric and oligomeric proteins in a fully native-like state</SPAN></LI><LI class="c11"><SPAN class="c2">In a direct comparison with phage display using the same antibody library and target, the yeast system sampled the library repertoire more comprehensively</SPAN></LI><LI class="c34"><SPAN class="c2">Compatibility with flow cytometric analysis, which allows quantitative measurements of equilibrium binding constants, dissociation kinetics, stability, and specificity of the displayed proteins without the laborious requirements of soluble protein expression and purification</SPAN></LI></UL></TD></TR><TR class="c32"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Disadvantages</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">The potential drawbacks are similar to phage display in that the expression of extracellular proteins is favoured and affinity maturation is complicated by avidity effects due to the multiplicity of the displayed peptides or proteins on the cell surface </SPAN></P></TD></TR><TR class="c1"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Membrane protein research </SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Compatible</SPAN></P></TD></TR><TR class="c26"><TD class="c22" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">References</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c12"><SPAN class="c17 c29">1. van Rosmalen, M. et al. Affinity Maturation of a Cyclic Peptide Handle for Therapeutic Antibodies Using Deep Mutational Scanning. </SPAN><SPAN class="c5 c29">Journal of Biological Chemistry</SPAN><SPAN class="c17 c29"> 292, 1477-1489 (2016).</SPAN><SPAN class="c17"> </SPAN><SPAN class="c17 c29">2. Gai, S. &amp; Wittrup, K. Yeast surface display for protein engineering and characterization. </SPAN><SPAN class="c5 c29">Current Opinion in Structural Biology</SPAN><SPAN class="c17 c29"> 17, 467-473 (2007).</SPAN><SPAN class="c17"> 3. </SPAN><SPAN class="c17 c29">Bowley, D., Labrijn, A., Zwick, M. &amp; Burton, D. Antigen selection from an HIV-1 immune antibody library displayed on yeast yields many novel antibodies compared to selection from the same library displayed on phage. </SPAN><SPAN class="c5 c29">Protein Engineering, Design and Selection</SPAN><SPAN class="c17 c29"> 20, 81-90 (2007).</SPAN><SPAN class="c17">   </SPAN><SPAN class="c17 c29">4. Shusta, E., Pepper, L., Cho, Y. &amp; Boder, E. A Decade of Yeast Surface Display Technology: Where Are We Now?. </SPAN><SPAN class="c5 c29">Combinatorial Chemistry &amp; High Throughput Screening</SPAN><SPAN class="c17 c29"> 11, 127-134 (2008).</SPAN></P></TD></TR></TBODY></TABLE><H2>Eukaryotic Display</H2><TABLE class="c65"><TBODY><TR class="c1"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Library size </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c17">10</SPAN><SPAN class="c17 c31"><SUP>7</SUP></SPAN></P></TD></TR><TR class="c19"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Transformation required</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Yes</SPAN></P></TD></TR><TR class="c84"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Mechanism </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">For the purpose of displaying foreign proteins on the surface of baculovirus particles as well as on infected insect cells, gp64 serve as a fusion partner that together with a chosen target protein gets incorporated into the cell membrane</SPAN></P></TD></TR><TR class="c54"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Proteins to be displayed </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Soluble and membrane, nontoxic, compatible with crossing membranes</SPAN></P></TD></TR><TR class="c1"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Surface anchorage </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Agglutination proteins, flocculation proteins</SPAN></P></TD></TR><TR class="c73"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Properties of protein enhanced</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Increased binding affinity and improved catalytic properties </SPAN></P></TD></TR><TR class="c74"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Applications</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><UL class="c20 lst-kix_grfzg7jsfuxi-0 start"><LI class="c11"><SPAN class="c2">Therapeutics</SPAN></LI><LI class="c34"><SPAN class="c2">Eukaryotic surface display libraries based on the baculovirus system allow selecting for specific binding proteins while providing post translational modifications</SPAN></LI><LI class="c34"><SPAN class="c17">Baculoviruses are non-pathogenic to mammals and thus, are an ideal carrier for antigenic epitopes or proteins intended to induce neutralizing immune response when administered as </SPAN><SPAN class="c88 c63">vaccines</SPAN></LI><LI class="c34"><SPAN class="c2">Eukaryotic expression libraries are a powerful tool for a variety of applications such as finding new ligands, identification of cellular interaction partners and affinity maturation of antibody and antibody fragments</SPAN></LI></UL></TD></TR><TR class="c66"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Advantages</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><UL class="c20 lst-kix_4is1obpzgfj4-0 start"><LI class="c34"><SPAN class="c2">The mammalian expression system is inherently competent for processing and secreting eukaryotic proteins. Consequently, mammalian cell display offers vast potential to display full-length human antibodies;</SPAN></LI><LI class="c34"><SPAN class="c2">Insect cells might be more feasible to express complex protein structures than e.g. yeast;</SPAN></LI><LI class="c34"><SPAN class="c2">As compared to mammalian cell lines, insect cells such as Sf9 and T. ni cells may be considered as a more robust and efficient display scaffold, since handling is less laborious, less costly and target proteins are incorporated more efficiently and homogeneously into the cellular membrane;</SPAN></LI><LI class="c62"><SPAN class="c17">Selection of specific clones by FACS is a fast and attractive method and does not require the cells to stay intact, but instead baculoviruses can be recovered by filtration and be used for </SPAN><SPAN class="c63 c88">re-infection </SPAN></LI></UL></TD></TR><TR class="c98"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Disadvantages</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">An obvious challenge remains to be solved that relatively low transformation efficiency of mammalian cells diminishes the actual repertoire size in contrast with phage display, leading to unlikely straightforward isolation of  antibodies with remarkable affinity. Moreover, the mammalian cell proliferation rate is slower, and such cells require more specific culture conditions in vitro than microbial cells. These drawbacks necessitate great efforts to improve the mammalian cell display platform.</SPAN></P></TD></TR><TR class="c1"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Membrane protein research </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Compatible</SPAN></P></TD></TR><TR class="c108"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">References</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c12"><SPAN class="c17 c29">Oker-Blom, C. Baculovirus display strategies: Emerging tools for eukaryotic libraries and gene delivery. </SPAN><SPAN class="c5 c29">Briefings in Functional Genomics and Proteomics</SPAN><SPAN class="c17 c29"> 2, 244-253 (2003).</SPAN></P></TD></TR></TBODY></TABLE><H2>Water-In-Oil Emulsions</H2><TABLE class="c65"><TBODY><TR class="c1"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Library size </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c17 c51">10</SPAN><SPAN class="c17 c51 c31"><SUP>10</SUP></SPAN></P></TD></TR><TR class="c19"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Transformation required</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">No</SPAN></P></TD></TR><TR class="c113"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Mechanism </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Experimental conditions are adjusted so that, in most cases, one compartment contains one DNA molecule. The DNA fragments encode fusion proteins containing a DNA-methyltransferase, which can form a covalent bond with a 5-fluoro deoxycytidine base at the extremity of the DNA fragment. The resulting library of DNA–protein fusions is extracted from the emulsion and DNA molecules displaying a protein with desired binding properties are selected from the pool of DNA–protein fusions by affinity panning on target antigens.</SPAN></P></TD></TR><TR class="c19"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Typical enrichment factor per round</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">&gt;1000</SPAN></P></TD></TR><TR class="c79"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Protein displayed </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">In vitro compartmentalization  using water-in-oil emulsions has been used for the selection of peptide ligands and for the directed evolution of DNA methyltransferases, bacterial phosphotriesterase, Taq polymerase, luciferase and human telomerase. </SPAN></P></TD></TR><TR class="c106"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Proteins to be displayed </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">-</SPAN></P></TD></TR><TR class="c1"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Surface anchorage </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">-</SPAN></P></TD></TR><TR class="c19"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Properties of protein enhanced</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Enzymatic activity, thermal stability</SPAN></P></TD></TR><TR class="c1"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Stability </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Stable </SPAN></P></TD></TR><TR class="c92"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Applications</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Protein evolution, to enhance their enzymatic activity, and thermal stability. </SPAN></P></TD></TR><TR class="c1"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Advantages</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Since DNA is added at limiting dilution, each aqueous droplet contains a single gene, and acts as a unique, independent reaction vessel</SPAN></P></TD></TR><TR class="c96"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Disadvantages</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><UL class="c20 lst-kix_m43ls9r672lj-0 start"><LI class="c87"><SPAN class="c2">Many proteins of interest, in particular more complex, multidomain proteins are more efficiently expressed in eukaryotic expression systems</SPAN></LI><LI class="c87"><SPAN class="c2">More than one DNA molecule might be present in one water compartment of the water-in-oil emulsion, which would lead to the incorrect assignment of genotype and phenotype and, therefore, to the selection of DNA molecules coding for non-binding proteins </SPAN></LI></UL></TD></TR><TR class="c59"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Membrane protein research </SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Incompatible. Membrane proteins cannot be reconstituted into surfactant layer of w/o emulsion</SPAN></P></TD></TR><TR class="c103"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">References</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><OL class="c20 lst-kix_mfoakyfn81yj-0 start" start="1"><LI class="c12 c67" id="h.gjdgxs"><SPAN class="c17 c29">Ghadessy, F. &amp; Holliger, P. A novel emulsion mixture for in vitro compartmentalization of transcription and translation in the rabbit reticulocyte system. </SPAN><SPAN class="c5 c29">Protein Engineering, Design and Selection</SPAN><SPAN class="c17 c29"> 17, 201-204 (2004).</SPAN></LI><LI class="c12 c67" id="h.9jtl5i5liufz"><SPAN class="c17 c29">Bertschinger, J. &amp; Neri, D. Covalent DNA display as a novel tool for directed evolution of proteins in vitro. </SPAN><SPAN class="c5 c29">Protein Engineering Design and Selection</SPAN><SPAN class="c17 c29"> 17, 699-707 (2004).</SPAN></LI></OL></TD></TR></TBODY></TABLE><H2>Liposome Display</H2><TABLE class="c65"><TBODY><TR class="c1"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Library size </SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c7"><SPAN class="c17">10</SPAN><SPAN class="c17 c31"><SUP>7</SUP></SPAN></P></TD></TR><TR class="c28"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Transformation required</SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">No</SPAN></P></TD></TR><TR class="c104"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Mechanism </SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Membrane protein is inserted to lipid bilayer. Protein can insert by himself or with the help of BAM complex and additional helper proteins</SPAN></P></TD></TR><TR class="c50"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Evolution</SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Single round of selection can be achieved within 1 day</SPAN></P></TD></TR><TR class="c104"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Protein displayed </SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Alpha-hemolysin pore forming protein, G protein-coupled receptor, caveolin</SPAN></P></TD></TR><TR class="c92"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Proteins to be displayed </SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Soluble and membrane, non-toxic, toxic, compatible with crossing membranes</SPAN></P></TD></TR><TR class="c1"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Surface anchorage </SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Membrane proteins</SPAN></P></TD></TR><TR class="c19"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Properties of protein enhanced</SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Pore-forming activity, enzymatic activity, ...</SPAN></P></TD></TR><TR class="c1"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Stability</SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Stable</SPAN></P></TD></TR><TR class="c1"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Applications</SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Membrane protein evolution, therapeutics</SPAN></P></TD></TR><TR class="c47"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Advantages</SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><UL class="c20 lst-kix_l6pi2eue8il6-0 start"><LI class="c34"><SPAN class="c5">In vitro</SPAN><SPAN class="c2"> translation system produces proteins without using living cells. One DNA sequence can be trapped in one liposome and toxic, bacterial growth inhibiting protein can be translated</SPAN></LI><LI class="c34"><SPAN class="c2">Rapid and efficient mutant selection</SPAN></LI><LI class="c34"><SPAN class="c17">Proteins that affect cell growth can also be engineered because the protein is synthesized </SPAN><SPAN class="c5">in vitro</SPAN></LI><LI class="c34"><SPAN class="c2">Conditions appropriate for the target membrane protein can be adopted because the lipid composition of the liposomes is adjustable and internal solution can be adjusted as needed</SPAN></LI><LI class="c62"><SPAN class="c2">No membrane proteins other than the target one are present</SPAN></LI></UL></TD></TR><TR class="c19"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Disadvantages</SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Size variety of liposomes reduce enrichment factor if multiple genes are trapped inside one big lisopome</SPAN></P></TD></TR><TR class="c90"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">Membrane protein research </SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c12"><SPAN class="c2">Alpha-hemolysin was translated inside liposome, self-inserted to membrane and formed pore. By doing evolution of this protein, mutants showing better pore forming and self-inserting properties were selected by FACS</SPAN></P></TD></TR><TR class="c91"><TD class="c3" colspan="1" rowspan="1"><P class="c7"><SPAN class="c2">References</SPAN></P></TD><TD class="c13" colspan="1" rowspan="1"><P class="c12"><SPAN class="c17 c29">1.Fujii, S., Matsuura, T., Sunami, T., Kazuta, Y. &amp; Yomo, T. In vitro evolution of  alpha-hemolysin using a liposome display. </SPAN><SPAN class="c5 c29">Proceedings of the National Academy of Sciences</SPAN><SPAN class="c17 c29"> 110, 16796-16801 (2013).</SPAN><SPAN class="c17 c29">2.Goodsell, D. &amp; Olson, A. Structural Symmetry and Protein Function. </SPAN><SPAN class="c5 c29">Annual Review of Biophysics and Biomolecular Structure</SPAN><SPAN class="c17 c29"> 29, 105-153 (2000).</SPAN></P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="invert-box"><SPAN class="invert-text">invert</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>