<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania-OG_Inhibition_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania-OG/Inhibition Model</SPAN></H1></DIV></DIV></DIV><TITLE>Collaborations</TITLE><DIV id="" class="clearfix"><HEADER id="header" class="page-section full-header"><DIV id="header-wrap" style="padding-left:1%"><DIV class="clearfix"><NAV id="primary-menu"><UL><LI id="homeheader"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG">Home</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Project</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design_Regulatory">Design for Regulatory Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Model">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Demonstrate">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Nucleotide_Design">Substrate Design Prospects</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Entrepreneurship">Entrepreneurship</A></LI></UL><LI id="softwareh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Gan_Introduction">Generative Adversarial Network – GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ProteinGAN">Protein GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ReactionGAN">Reaction GAN</A></LI><LI id="partsh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-basic">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-composite">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-collection">Collection</A></LI><LI id="notebookh"><A href="/pages/notebook/index.html">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/catseqguide">Using the CAT-Seq</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/labnotebook">Lab Journal</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Safety">Safety</A></LI><LI id="humanh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Collaborations">Collaborations</A></LI><LI id="peopleh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">People</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">Team</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Sponsors">Sponsors</A></LI></NAV></DIV></DIV></HEADER><SECTION id="slider" class="slider-element slider-parallax full-screen force-full-screen with-header swiper_wrapper page-section clearfix"><DIV class="slider-parallax-inner"><DIV class="swiper-container swiper-parent"><DIV class="swiper-wrapper"><DIV class="swiper-slide" style="background-image: url('https://static.igem.org/mediawiki/2018/4/4b/T--Vilnius-Lithuania-OG--model.jpg');"><DIV class="container clearfix"><DIV class="slider-caption slider-caption-center"><H2 data-caption-animate="fadeInUp" style="color:aliceblue">Modified Nucleotide inhibitory effect model</H2></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><DIV id="page-menu"><DIV id="page-menu-wrap"><DIV class="container clearfix"><DIV class="menu-title">Explore <SPAN>MODEL</SPAN></DIV><NAV class="one-page-menu"><UL><LI><A href="#" data-href="#section-introduction">Introduction</A></LI><LI><A href="#" data-href="#section-model">Modelling </A></LI><LI><A href="#" data-href="#section-exp">Experminental validation</A></LI><LI><A href="#" data-href="#section-concl">Conclusions</A></LI></UL></NAV></DIV></DIV></DIV><SECTION style="overflow:hidden; position:relative; background-color: #FFF;"><DIV class="content-wrap"><DIV class="single-post nobottommargin"><DIV class="container clearfix"><DIV class="tekstodydis"><DIV class="fancy-title title-border-color"><H2>Introduction</H2></DIV><P>During the development of CAT-Seq, we have met various biotechnology field experts with the hope of finding design flaws and improvements to our system. One of these experts was Audrius Laurinėnas. At that time we had a big problem of not having enough amplified DNA in each droplet for robust sequencing. Audrius has suggested the MDA reaction. Yet, he also knew that using modified nucleotides might inhibit the MDA reaction, as phi29 DNA polymerase does not incorporate many modified nucleotides. It is a logical assumption to think that some of the <STRONG>Substrate Nucleotides</STRONG> might somehow inhibit the DNA replication reaction. We have hypothesised that the inhibitory effect of modified nucleotides might skew the amplified <STRONG>DNA fragment size</STRONG> and <STRONG>product yield </STRONG>distribution. This an important aspect to consider, as long fragments are crucial for high quality when Nanopore Sequencing is employed. Yet, if we don’t use a high enough concentration of Substrate Nucleotides, the dynamic range of our measurement might decrease dramatically.</P><DIV class="fancy-title title-border-color"><H2>Modelling of inhibitory nucleotide incorporation</H2></DIV><P>Therefore, we have written a <STRONG>mathematical description of Phi29 DNA amplification</STRONG> which takes into account the possibility of the Phi29 inhibition by <STRONG>Substrate Nucleotide</STRONG>, in order to investigate the maximum amount of substrate nucleotides we can use without worrying about short sequencing reads.</P><P>The mathematical Phi29 model imitates the amplification of the CAT-Seq Esterase DNA template with random hexamers. First of all, random hexamer primer sequences are generated and aligned to the DNA template. This maps the beginning of the DNA amplification. After that, newly synthesized DNA templates can be immediately primed again with random hexamers. The re-priming of DNA template happens every few iterations of synthesis. Each site housing a Guanine nucleotide acts as a potential spot for modified nucleotide incorporation.</P><P>The incorporation of nucleotides is based on the probabilities set by initial ratios of the substrate and reference nucleotides. The model assumes that incorporation of the modified nucleotide into a DNA sequence during replication initiates the termination and dissociation of the phi29 polymerase from the DNA template. Every incorporation of Substrate Nucleotides is followed by a probability recalculation.</P><P>Using different ratios of the substrate and reference nucleotides as an input, the distribution of DNA fragments was generated. Because the kinetics parameters and features of phi29 polymerase and multiple displacement amplification reaction are complex and not quite understood, the generated data was normalized to the output distribution of DNA fragment size when zero substrate of nucleotides.</P><DIV style="margin-top: 3%; text-align: justify; font-size: 80%; margin-bottom: 5%; margin-left:5%;margin-right:5%;"><P><STRONG>Figure 1.</STRONG><STRONG> Normalized mathematical model output DNA fragment distribution. </STRONG>The distribution of DNA fragment size for different initial substrate and reference nucleotide ratios was generated using the mathematical phi29 model. The data is normalized to the distribution output generated with zero substrate nucleotides.</P></DIV><P>As seen from the distribution violin plots shown in figure 1, the increase of potentially inhibiting substrate nucleotide concentration (its ratio to reference nucleotides) decreases the fragment size distribution as expected. Such amplification inhibition and reduced DNA fragment size distribution could pose a difficult task during the Nanopore sequencing of amplified DNA product and short sequencing read mapping and analysis.</P><DIV class="fancy-title title-border-color"><H2>Experimental validation</H2></DIV><P>Now that we are aware of such possible <STRONG>degree Substrate Nucleotide inhibition</STRONG>, we set out to test and titrate the possible nucleotide concentration and ratios used in CAT-Seq workflow for best possible DNA fragment distribution results.</P><DIV style="margin-top: 3%; text-align: justify; font-size: 80%; margin-bottom: 5%; margin-left:5%;margin-right:5%;"><P><STRONG>Figure 2. Amplified DNA correlation to substrate nucleotide concentration in the reaction.</STRONG>DNA, amplified using multiple displacement amplification was synthesised using 750µM dNTP concentration with varied substrate nucleotide concentration. Decreasing the concentration of substrate nucleotides the amount of DNA amplified increases as seen by the increased intensity of the DNA band and smear.</P></DIV><P>First of all, the lower limit of substrate nucleotides (Sub-dCTP) was determined. MDA reactions were performed with different dNTP and substrate-dCTP concentrations and the amplified DNA was analyzed on agarose gel. Results shown on Fig. 2 hint that large concentrations of substrate-bound nucleotides might inhibit the reaction, as the larger amplified DNA product bands fade as more substrate nucleotides are added, <STRONG>just as the model predicted</STRONG>. The DNA smear seen in the gel is a characteristic part of the rolling circle amplification. The final concentration of substrate-bound dCTP molecules was chosen to be 25 µM in the amplification reaction.</P><DIV style="margin-top: 3%; text-align: justify; font-size: 80%; margin-bottom: 5%; margin-left:5%;margin-right:5%;"><P><STRONG>Figure 3. Phi29 amplified DNA dependence on the concentration of nucleotide concentrations.</STRONG> DNA, amplified using multiple displacement amplification was synthesised using 750 - 100µM dNTP concentration with or without 25µM substrate nucleotide concentration. Decreasing the concentration of dNTP’s decreases the amount of DNA amplified only when concentrations lower than 200 µM are used. Red dashed line highlight the chosen condition for MDA reaction.</P></DIV><P>Next, we seeked to determine the lowest reference nucleotide concentration for the MDA reaction usable with the corresponding 25 µM concentration of substrate nucleotides. While substrate dCTP concentration was constant, the concentration of dNTPs was gradually lowered. In addition to this, reaction without modified nucleotides was carried too, to determine whether the nucleotide concentration or ratio to substrate-dCTP is the product limiting factor. The results indicate that <STRONG>200 µM</STRONG> and <STRONG>100 µM</STRONG> final concentrations are too low to be used with <STRONG>25 µM</STRONG> sub-dCTP (ratios 1:8 and 1:4), because the upper DNA product fades considerably. Taking this into consideration, <STRONG>300 µM</STRONG> was chosen to be the final working dNTP concentration.</P><DIV class="fancy-title title-border-color"><H2>Conclusions</H2></DIV><P>To conclude, the phi29 mathematical description solutions allowed us to acknowledge the potential DNA fragment size shift as a result of inhibitory effects of substrate nucleotides. Based on the output of the model we set out to investigate the inhibitory effects caused by substrate nucleotides in the laboratory. The results generated by the model fit with the experimental data well - higher substrate nucleotide concentrations indeed inhibit the production of long DNA molecules and lead to the synthesis of short ones. This model has immensely <STRONG>helped us to improve our CAT-Seq prototype </STRONG>by showing the possible <STRONG>crucial </STRONG>negative effects of substrate nucleotides. This has <STRONG>helped us to successfully optimize</STRONG> the substrate nucleotide concentration in laboratory in order to <STRONG>receive best possible and sequencing quality and the dynamic range</STRONG> for activity measurements.</P><P>The mathematical model for <B>Modified Nucleotide Inhibitory Effect</B> can be found <A href="https://github.com/donatasrep/igem/tree/master/scripts/Modified%20nucleotide%20inhibitory%20effect%20model">here</A></P></DIV></DIV></DIV></DIV></SECTION></DIV></BODY></HTML>