<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania-OG_Design_Regulatory skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania-OG/Design Regulatory</SPAN></H1></DIV></DIV></DIV><TITLE>Collaborations</TITLE><DIV id="" class="clearfix"><HEADER id="header" class="page-section full-header"><DIV id="header-wrap" style="padding-left:1%"><DIV class="clearfix"><NAV id="primary-menu"><UL><LI id="homeheader"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG">Home</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Project</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design_Regulatory">Design for Regulatory Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Model">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Demonstrate">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Nucleotide_Design">Substrate Design Prospects</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Entrepreneurship">Entrepreneurship</A></LI></UL><LI id="softwareh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Gan_Introduction">Generative Adversarial Network – GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ProteinGAN">Protein GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ReactionGAN">Reaction GAN</A></LI><LI id="partsh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-basic">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-composite">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-collection">Collection</A></LI><LI id="notebookh"><A href="/pages/notebook/index.html">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/catseqguide">Using the CAT-Seq</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/labnotebook">Lab Journal</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Safety">Safety</A></LI><LI id="humanh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Collaborations">Collaborations</A></LI><LI id="peopleh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">People</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">Team</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Sponsors">Sponsors</A></LI></NAV></DIV></DIV></HEADER><SECTION id="slider" class="slider-element slider-parallax full-screen force-full-screen with-header swiper_wrapper page-section clearfix"><DIV class="slider-parallax-inner"><DIV class="swiper-container swiper-parent"><DIV class="swiper-wrapper"><DIV class="swiper-slide" style="background-image: url('https://static.igem.org/mediawiki/2018/5/54/T--Vilnius-Lithuania-OG--Design.jpeg');"><DIV class="container clearfix"><DIV class="slider-caption slider-caption-center"><H2 data-caption-animate="fadeInUp" style="color: white">Regulatory CAT-Seq Design</H2></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><DIV id="page-menu"><DIV id="page-menu-wrap"><DIV class="container clearfix"><DIV class="menu-title">Explore <SPAN>DESIGN</SPAN></DIV><NAV class="one-page-menu"><UL><LI><A href="#" data-href="#section-introduction">Introduction</A></LI><LI><A href="#" data-href="#section-general">General description</A></LI><LI><A href="#" data-href="#section-proof">Proof of Concept</A></LI></UL></NAV></DIV></DIV></DIV><SECTION style="overflow:hidden; position:relative; background-color: #FFF;"><DIV class="content-wrap"><DIV class="single-post nobottommargin"><DIV class="container clearfix"><SECTION id="section-introduction" class="page-section"><DIV class="tekstodydis"><DIV class="fancy-title title-border-color"><H3>Introduction</H3></DIV><P>While Catalytic Activity Sequencing began as a method for catalytic biomolecule activity recording, we have also created a way to adjust CAT-Seq to record activities of <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="genetic elements that influence the transcription or translation of the gene of interest. For example, such elements could be promoter, ribosome binding site, Riboswitches and other">regulatory part</A> libraries. In addition to the activities, cross-interactions of different regulatory parts can also be measured.</P><P>We highly suggest to read the main design section of CAT-Seq first, as we will not go over the main design scheme here and will only build upon it further.</P><DIV class="fancy-title title-border-color"><H3>Adjusting CAT-Seq system for Regulatory Sequences</H3></DIV><P>Just like in the main CAT-Seq design page, we again divide the Regulatory CAT-Seq design page into two parts:</P><P class="listin">1.	<STRONG>Design description</STRONG> - an in depth explanation of how a particular part of our system works, how it should be used in real-life situation and why such design choices were adopted.</P><P class="listin">2.	<STRONG>Proof of Concept</STRONG> - a story about how we applied the design using specific components and parts to build Regulatory CAT-Seq and demonstrate that it works as intended.</P><SECTION id="section-general" class="page-section"><P class="didesnist"><STRONG>General description</STRONG></P><P>As already established, the Catalytic Activity Sequencing records the activity of different catalytic biomolecules in separate water droplets simultaneously. That is achieved with the help of <STRONG>Substrate Nucleotides</STRONG>. In turn, unique catalytic biomolecules produce different amounts of <STRONG>Product Nucleotides</STRONG>, which can then be quantified by adding them to their own amplified template DNA and later sequencing those molecules.</P><P>Yet, the activity of the catalytic biomolecule is not the only aspect that can influence the amount of Product Nucleotides that are produced. If all of the droplets would contain <STRONG>the same catalytic biomolecule</STRONG>, but each droplet would have a <STRONG>different concentration of that biomolecule</STRONG>, we would get <STRONG>different amounts</STRONG> of Product Nucleotides. For example, droplets with large amount of biomolecules may produce a large number of Product Nucleotides and <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="Vice versa is a Latin phrase that means “the other way around.”">vice-versa</A>.</P><P>Using such logic, we can hypothesise that any <STRONG>sequence element that can adjust the expression</STRONG> of the catalytic biomolecule <STRONG>can be measured </STRONG>and its influence on the Product Nucleotide - recorded. In other words, this influence is the <STRONG>activity</STRONG>, or <STRONG>strength </STRONG>of <STRONG>regulatory element</STRONG>.</P><SECTION id="section-proof" class="page-section"><P class="didesnist"><STRONG>Proof of concept</STRONG></P><P>To demonstrate that CAT-Seq can be applied to record the activity and cross interactions of regulatory sequences we have chosen two main objects: <STRONG>Ribosome Binding Sites </STRONG>and <STRONG>synthetic Toehold Switches</STRONG>.</P><P><STRONG>1.	Regulatory part activity recording</STRONG></P><P><STRONG>Ribosome binding sites (RBS) </STRONG>are sequences that are responsible for recruiting the ribosomes during the initiation of protein translation. RBS can have different sequences, and depending on those sequences, the frequency of ribosome binding rate to the RBS will be different. In this way, gene of interest translation rate can be controlled in a constitutive manner.</P><P>Having a large RBS library with known activities would allow to precisely pick an RBS sequence with a specific activity, while engineering sensitive and complex genetic circuits. Yet, it is not easy to screen large libraries of RBS, as each individual <STRONG>known </STRONG>sequence must be selectively placed in a physically separated compartment and then measured.</P><P>With CAT-Seq, such activity recording becomes much easier. While keeping all the workflow from the main Catalytic Activity Sequencing design, all we need to do is to change the way we prepare the library. Firstly, instead of using a library of catalytic biomolecules, we can use a single esterase enzyme. Then, specific or random RBS sequence library can be prepared and attached to the esterase enzyme sequence. After that, every step of CAT-Seq method is the same.</P><P style="clear:both">After the library is encapsulated into droplets, depending on the <STRONG>RBS sequence strength </STRONG>some droplets may have<STRONG> different amounts </STRONG>of the esterase. For example, the droplets with <STRONG>more esterase</STRONG> will produce <STRONG>more Product Nucleotides </STRONG>and vice-versa. After the amplification and sequencing steps we can learn two main things about the measured library - the <STRONG>RBS sequence</STRONG> and its relative <STRONG>RBS strength</STRONG> in the form of Product Nucleotide concentration for every library variant screened.</P><P>For our proof of concept experiments, we have chosen to measure multiple already established RBS sequences from the <STRONG>Registry of Standard Biological Parts</STRONG>. First, we measured the activities of RBS using standard methods as a control. Next, we have measured the same constructs using the CAT-Seq system.</P><P><STRONG>2.	Regulatory part interaction recording</STRONG></P><P>Next, we want to give an example on how to record regulatory part <STRONG>interactions </STRONG>using CAT-Seq. In this case, we will be using <STRONG>Toehold Switches</STRONG>.</P><P>The Toehold Switch systems are composed of two RNA strands referred to as the Switch and Trigger. The Switch RNA contains the coding sequence of the gene being regulated. The Switch RNA forms a hairpin structure that includes the RBS site. While the hairpin structure is formed, the translation is inhibited. The Trigger RNA is a molecule that can selectively bind to the Switch RNA region and expose the gene RBS site for ribosomes. Once that happens, the protein translation can be initiated.</P><P>While these riboregulators are extremely useful in engineering synthetic biology circuits, there is still a huge lack of <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="Orthogonality means that two pairs of Toehold Switches should not interact. For example - Trigger RNA A should only be able to activity Toehold Switch A, and not Toehold Switch B">orthogonal</A> Toehold Switch and Trigger pairs. The orthogonal pairs are needed in order to create complex circuits, which require a large amount of logic functions. Yet, finding properly working pairs of Toehold Switches is still difficult, as in-silico methods still require more experimental data to generate well-working Toehold Switch pairs from scratch. Moreover, creating a Toehold Switch pair that is not only working by itself, but also does not cross interact with other pairs is an even more daunting task.</P><P>To give an example, if we had <STRONG>50 working Toeholds</STRONG> and wanted to see how they would interact with each other using currently available methods, we would have to do 2500 separate measurements. For that reason, we want to demonstrate that CAT-Seq allows to measure regulatory part cross-interactions simultaneously and with ease.</P><P>Once again, only the first part of general CAT-Seq design needs to be changed - the <STRONG>library preparation</STRONG>. Instead of using a catalytic biomolecule library, a single enzyme is used. Then, <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="random or specific">libraries</A> need to be prepared - one for Toehold Switches and another for Trigger RNA. The Trigger RNA libraries also require a separate T7 promoter for RNA expression. Then, both of those libraries must be ligated to the enzyme DNA fragment. Resulting library contains fragments which have the <STRONG>same catalytic biomolecule</STRONG>, yet each of them have a <STRONG>random combination </STRONG>of a specific <STRONG>Toehold Switch </STRONG>and <STRONG>RNA Trigger</STRONG>.</P><P>After the encapsulation, the droplets which have the correct Trigger RNA and Toehold Switch combination (in which Trigger RNA binds to that specific Toehold Switch), can produce catalytic biomolecules. In turn, those catalytic biomolecules can produce the<STRONG> Product Nucleotides</STRONG>.</P><P>In other droplets, where the Trigger RNA binds to the Toehold Switch with weaker affinity, there are also less catalytic biomolecules and Product Nucleotides produced. Finally, the droplets in which the Trigger RNA does not bind to the Toehold Switch, there are no biomolecules or product nucleotides produced.</P><P>After the Amplification and Sequencing steps, from each unique sequenced single molecule we can receive the information about the <STRONG>Toehold Switch and Trigger RNA sequence pairs</STRONG>, and also the information about how that regulatory pair interacted. In this case the interaction stands for how strongly the specific Trigger RNA bound to the Toehold Switch.</P><P>For our proof of concept, we have chosen multiple Toehold Switches which were already described in literature. We have first prepared a small library of different combinations of Toehold Switches and Trigger RNAs. Then, we measured those interactions using standard, low-throughput methods. Lastly, we have used CAT-Seq to determine those cross-interactions in high-throughput.</P><P>In this way, CAT-Seq can be applied to conveniently <STRONG>record the activities of large number of regulatory parts</STRONG> and at the same time <STRONG>assess their cross-interactions</STRONG>.</P></SECTION></SECTION></DIV></SECTION></DIV></DIV></DIV></SECTION></DIV></BODY></HTML>