<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Rice_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Rice/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><NAV><UL class="nav-list"><LI><A href="https://2018.igem.org/Team:Rice" style="text-decoration:none">Home</A></LI><LI><A href="#!" style="text-decoration:none">Project</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Description" style="text-decoration:none">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Design" style="text-decoration:none">Design</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Background" style="text-decoration:none">Background</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Model" style="text-decoration:none">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Software" style="text-decoration:none">Software</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Notebook" style="text-decoration:none">Notebook</A></LI><LI><A href="https://static.igem.org/mediawiki/2018/0/05/T--Rice--Protocols.pdf" style="text-decoration:none">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Results" style="text-decoration:none">Results</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Demonstrate" style="text-decoration:none">Demonstration</A></LI><LI><A href="https://2018.igem.org/Team:Rice/FutureDirections#!" style="text-decoration:none">Future Directions</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Safety" style="text-decoration:none">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Rice/InterLab" style="text-decoration:none">InterLab</A></LI></UL><LI><A href="#!" style="text-decoration:none">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Human_Practices" style="text-decoration:none">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Public_Engagement" style="text-decoration:none">Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Collaborations" style="text-decoration:none">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Integrated_HP" style="text-decoration:none">Integrated Human Practices</A></LI><LI><A href="#!" style="text-decoration:none">Achievements</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Parts" style="text-decoration:none">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Medal_Requirements" style="text-decoration:none">Medal Requirements</A></LI><LI><A href="#!" style="text-decoration:none">Team</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Members" style="text-decoration:none">Members</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Mentors" style="text-decoration:none">Advisors</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Instructors" style="text-decoration:none">Instructors</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Attributions" style="text-decoration:none">Attributions</A></LI></NAV></DIV><DIV class="igem_2017_content_wrapper"><DIV id="topbox"><H1>Project Description
            </H1></DIV><DIV class="container" align="center"><DIV class="center caption" width:="" 80=""><B class="caption">Overview of PORTAL.</B> The vision of PORTAL is to create a virtual machine which functions inside the cell independently of the host operating system. Based on the advice from experts, we selected strains that are safe and relevant for industry and science. Strain genomes are used as inputs to our software which generates the design for orthogonal translation. Mathematical modelling allowed us to determine appropriate parameters for orthogonal transcription and translation systems. 
</DIV><DIV class="center"><H2>Introduction</H2><P align="left"> Engineering genetic circuits that function consistently across strains and are reliable in different environmental conditions is an important goal of synthetic biology. However, the majority of biological parts have only been characterized in specific model organisms such as <I>Escherichia coli</I> or <I>Saccharomyces cerevisiae</I>. These parts often fail or have inconsistent behavior in other species. The goal of our project is to expand the range of synthetic biology applications by developing tools which would facilitate the expression of genetic constructs in a wider variety of strains. We have chosen to test our designs in a number of organisms relevant for applications in areas such as agriculture, therapeutics, bioremediation, and bioelectronics.</P><DIV class="center caption" width:="" 80=""><B class="caption">Figure 1:</B><B class="caption">Strains used in the project. </B>From left to right: <I>Escherichia coli</I>, <I> Bacillus subtilis </I>, <I>Lactococcus lactis</I>, <I>Pseudomonas putida</I>, <I>Shewanella oneidensis</I>, <I>Sinorhizobium meliloti</I>, <I>Corynebacterium glutamicum</I></DIV><P align="left"> A central obstacle to the development of universal cross-species parts and devices is the dependence of genetic circuit expression on host resources and gene expression machinery, which vary from strain to strain. Introducing synthetic genetic constructs into a cell may result in unpredictable outcomes such as cell growth defects, gene coupling, and alterations in host metabolic processes<SUP>1</SUP> (Figure 2). In order to address the problem of undesirable circuit-host interactions, our projects aims to create tools which would allow for a partial insulation of genetic circuit function from the nuances of host processes.</P><DIV class="center caption"><B class="caption">Figure 2: Orthogonal expression resources help reduce undesirable circuit-host interactions. </B> Introducing synthetic genetic constructs may result in unpredictable behavior due to circuit interactions with host machinery. Addition of circuit-specific expression resources can help partially insulate the circuit from the nuances of host processes. Figure modified from Brynildsen <I> et al. </I><SUP> 1 </SUP><P>
When we introduce genetic circuit into a cell, cellular resources such as ribosomes and polymerases are shared between circuit and host genes, creating a competition for resources which may lead to undesired effects mentioned above (Figure 3, left). To develop a solution to this problem, we have designed a portable transcription-translation resource (PORTAL), which uses orthogonal transcription and translation to create a separate pool of resources in order to minimize crosstalk between circuit and host components (Figure 3, right). PORTAL function is analogous to a virtual machine that can works independently of host operating system. 

</P><DIV class="center caption"><B class="caption">Figure 3: circuit expression with host vs. orthogonal components.</B> When circuit machinery is used for circuit expression, undesired effects may arise due to competition for resources between host and circuit genes (left). An alternative way is to introduce circuit-specific resources to minimize cross-talk between host and circuit processes (right).  
</DIV><DIV class="center"><H2>Orthogonal Transcription</H2></DIV><P align="left">
To implement orthogonal transcription, we decided to use T7 expression system. Circuits involving phage T7 RNA polymerase (T7 RNAP)  have been commonly used for expression in non-model organisms. T7 expression systems are  convenient for cross-species use since transcription from T7 promoters is independent of host RNA polymerase. We optimize the design for a universal bacterial expression resource (UBER) described in Kushwaha &amp; Salis <SUP> 2 </SUP>, which utilizes T7 RNAP to drive the expression of a reporter. Circuit architecture is shown in Figure 3. A negative feedback loop reduces the toxicity associated with overexpression of the polymerase and enhances the consistency of cross-species circuit performance by modulating the expression levels. 
</P><P align="left">
Our design aims to enhance the universality of UBER by making the following alterations: 
1) Replacing the existing origin with a broad host origin 
2) Using an antibiotic resistance cassette that has broader functionality across strains 
3) Removing components associated with eukaryotic gene expression for optimal performance in bacteria 
</P><P><A href="https://2018.igem.org/Team:Rice/Design">Click here</A> to learn more about modifications introduced in UBER deign. </P><DIV class="center caption"><B class="caption">Figure 3: UBER (Universal Bacterial Expression Resource). </B>Figure modified from Kushwaha &amp; Salis.<SUP>2</SUP></DIV><DIV class="center"><H2>Orthogonal Translation</H2></DIV><P align="left">
In addition to a cross-species transcription resource, we will incorporate an orthogonal translation system which utilizes 16S rRNA with altered Anti-Shine-Dalgarno (ASD) sequence and a corresponding Shine-Dalgarno sequence (SD) of the ribosome binding site (RBS). This will advance our understanding of the benefits of orthogonal translation in non-model organisms since expression of orthogonal ribosomes in strains other than <I>E. coli</I> has not been previously reported in the literature. <A href="https://2018.igem.org/Team:Rice/Background"> Click here </A> to learn more about orthogonal gene expression in bacteria. 
</P><P align="left">
We developed a software tool based on the algorithm described in Chubiz &amp; Rao <SUP>3</SUP>, which can be used to design pairs of SD-ASD sequences to generate orthogonal ribosomes for any strain of bacteria. The algorithm takes into account the binding energies of SD-ASD pairs and selects only those candidates that are expected to have a sufficiently strong SD-ASD interaction while avoiding cross-reactivity with wild-type sequences (Figure 4).

</P><DIV class="center caption"><B class="caption">Figure 4: Algorithm for designing orthogonal ribosomes. </B>(a) The algorithm creates a library of all possible SD-ASD pairs. (b) Candidates with binding energies similar to the wild-type SD-ASD pair are selected; candidates which favor interaction with wild-type SD-ASD sequences are eliminated. (d) The remaining SD-ASD pairs are ranked based on the strength of interaction with host sequences. Figure modified from Chubiz &amp; Rao <SUP> 4 </SUP>.</DIV><P align="left">

We have developed a graphical user interface to make the use of algorithm convenient for other researchers. The inputs required include genome and CDS files for a given organism, which can be easily obtained from NCBI. The algorithm will output the top 10 ASD candidates for a given strain. 

</P><P align="left">
Our project will ultimately consist of a combined orthogonal transcription-translation system (Figure 5). Orthogonal transcription system, involving modified UBER is used to transcribe the genes under T7 promoter, including T7 RNAP, mKate2 reporter, TetR repressor, and the orthogonal 16S rRNA. Orthogonal ribosomes created through association of the altered 16S rRNA with large ribosomal subunit selectively translate the reporter mRNA containing the orthogonal ribosome binding site. This design ensures that native RNA polymerases and ribosomes minimally interact with our system, reducing the unwanted interactions between cricut expression and host processes. As shown in the models from Darlington <I>et al.</I><SUP>4</SUP>, the minimization of crosstalk between host and synthetic processes is desirable as it decreases gene coupling, decreases mutation rate, and decreases sensitivity to changes in nutrient availability in <I>E. coli</I> without adversely affecting cell growth.
</P><DIV class="center caption"><B class="caption">Figure 5: Orthogonal transcription-translation system.</B></DIV><DIV class="center"><DIV class="center caption"><H2>Modeling</H2></DIV><P align="left"> The design for our model requires three primary features: orthogonal expression, physiological output, and adjustable parameters to change host species. We base our model on that used in Darlington <I>  et al. </I><SUP>4</SUP>. This model incorporates a system of ordinary differential equations that describe simplified energy production, ribosome synthesis, translation of host and circuit genes, and cell growth. In addition, the model includes orthogonal translation by allowing orthogonal 16S rRNAs to compete with host rRNAs in ribosome synthesis. 
 
</P><P align="left">
We extend the model to incorporate RNA transcription in order to more naturally model orthogonal transcription with T7 RNA polymerases. Transcription follows Michaelis-Menten kinetics based on the concentration of polymerase molecules in the cell, and we choose constants to reproduce accurate steady-state behavior. Since translation is the most energy intensive process in <I> E. coli </I><SUP>5</SUP>, we assume in our model that transcription does not use cellular energy. We also include transcription and translation of RNA polymerase genes to produce feedback behavior. Our model predicted that low levels of orthogonal 16S rRNA transcription will results in reduced PORTAL sensitivity to parameter changes, which informed our choice to put 16S rRNA under IPTG-inducible promoter in order to allow the control over the levels of 16S rRNA produced.  <A href="https://2018.igem.org/Team:Rice/Model"> Click here </A> to learn more about our model. 
</P><DIV class="center caption"><B class="caption">Figure 6: Main components of the model.</B>  The model includes metabolism, host expression, circuit expression, and orthogonal expression.  </DIV></DIV><DIV class="center caption"><H2>References</H2></DIV><P>
[1] Aedo, S. J., Gelderman, G., &amp; Brynildsen, M. P. (2017). Tackling host–circuit give and take.<I> Nature Microbiology </I>, 2(12), 1584–1585.
[2] Kushwaha, M., &amp; Salis, H. M. (2015). A portable expression resource for engineering cross-species genetic circuits and pathways.<I> Nature Communications</I>, 6(1), 7832. 
[3] Chubiz, L. M., &amp; Rao, C. V. (2008). Computational design of orthogonal ribosomes.<I>Nucleic Acids Research</I>, 36(12). 
[4] Darlington, A.P.S., Kim, J., Jiménez, J.I., &amp; Bates, D.G. (2018). Dynamic allocation of orthogonal ribosomes facilitates uncoupling of co-expressed genes. Nature Communications, 9, 695.  
[5] Weiße, A.Y., Oyarzún, D.A., Danos, V., Swain, P.S. (2015). Mechanistic links between cellular trade-offs, gene expression, and growth.<I> Proc. Natl. Acad. Sci. USA</I>, 112(9), 1038-47.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>