<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Rice_Software_Step1 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Rice/Software/Step1</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><NAV><UL class="nav-list"><LI><A href="https://2018.igem.org/Team:Rice" style="text-decoration:none">Home</A></LI><LI><A href="#!" style="text-decoration:none">Project</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Description" style="text-decoration:none">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Design" style="text-decoration:none">Design</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Background" style="text-decoration:none">Background</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Model" style="text-decoration:none">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Software" style="text-decoration:none">Software</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Notebook" style="text-decoration:none">Notebook</A></LI><LI><A href="https://static.igem.org/mediawiki/2018/0/05/T--Rice--Protocols.pdf" style="text-decoration:none">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Results" style="text-decoration:none">Results</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Demonstrate" style="text-decoration:none">Demonstration</A></LI><LI><A href="https://2018.igem.org/Team:Rice/FutureDirections#!" style="text-decoration:none">Future Directions</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Safety" style="text-decoration:none">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Rice/InterLab" style="text-decoration:none">InterLab</A></LI></UL><LI><A href="#!" style="text-decoration:none">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Human_Practices" style="text-decoration:none">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Public_Engagement" style="text-decoration:none">Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Collaborations" style="text-decoration:none">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Integrated_HP" style="text-decoration:none">Integrated Human Practices</A></LI><LI><A href="#!" style="text-decoration:none">Achievements</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Parts" style="text-decoration:none">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Medal_Requirements" style="text-decoration:none">Medal Requirements</A></LI><LI><A href="#!" style="text-decoration:none">Team</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Members" style="text-decoration:none">Members</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Mentors" style="text-decoration:none">Advisors</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Instructors" style="text-decoration:none">Instructors</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Attributions" style="text-decoration:none">Attributions</A></LI></NAV></DIV><DIV class="igem_2017_content_wrapper"><DIV id="topbox"><H1>Library Initiation Details
            </H1></DIV><DIV class="container" align="center"><DIV class="center"><P align="left">
	To determine the ideal Shine-Dalgarno (SD) / Anti-Shine-Dalgarno (ASD) sequence for use as a truly orthogonal ribosome in a particular organism, we first start by generating a dictionary of all possible SD sequences. Since the SD sequence is 6 base-pairs and each position has 4 possible options (A, U, C, G), we generate a total of 4096 different possible sequences. This starting dictionary will be narrowed down in future steps to ultimately determine which complementary ASD candidates are the best for use in an orthogonal ribosome. </P><DIV class="center"><H2>Function: libbuild</H2></DIV><P align="left">Step 1 was done by using six nested for loops, each of which choose a number between 1 and 4. A 6-digit number was generated from this process, each number of which was then translated into a specific base pair (1 = A, 2 = U, 3 = C, 4 = G). Repeating this loop for all possible 6-digit combinations of numbers between 1 and 4 gave us all 4096 possible pre-ASD sequences. The ASD is generated by wrapping each pre-ASD sequence in a 4 (prefix) and 2 bp (suffix) sequence, respectively. For example, the ASD for E. coli was generated by wrapping each 6 bp SD sequence in AUCA (prefix) and UA (suffix), to ultimately generate the ASD in the form “AUCA------UA”, where the dashes represent the 6 randomized bases (see description of getASD function to see how we found the correct prefix and suffix). We then take the reverse complement of those 6-base sequences to get the corresponding SD sequences. We then format the entire dictionary so that the first SD sequence can be called by the key “SD1”, and the first ASD sequence by “ASD1”, and so on. 
</P><DIV class="center caption"><B class="caption">Figure S1. Randomization of six base in the anti Shine-Dalgarno sequence. </B></DIV><DIV class="center"><H2>Function: getASD</H2></DIV><P align="left">It has been known that “ACCUCC” is a highly prevalent sequence found on rRNA that is complementary to the SD sequence found on the mRNA. To determine the prefix and suffix of the ASD of a particular organism, we took the highly conserved 16s rRNA sequence and found the “ACCUCC” motif within it (using the finditer method, which locates patterns within a particular string). We then looked 3 bps upstream of this motif to find the prefix and 3 bps downstream of the motif to find the suffix of the ASD. The function also returns the index of the starting base of the ASD sequence in the chromosome.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>