<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Rice_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Rice/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><NAV><UL class="nav-list"><LI><A href="https://2018.igem.org/Team:Rice" style="text-decoration:none">Home</A></LI><LI><A href="#!" style="text-decoration:none">Project</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Description" style="text-decoration:none">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Design" style="text-decoration:none">Design</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Background" style="text-decoration:none">Background</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Model" style="text-decoration:none">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Software" style="text-decoration:none">Software</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Notebook" style="text-decoration:none">Notebook</A></LI><LI><A href="https://static.igem.org/mediawiki/2018/0/05/T--Rice--Protocols.pdf" style="text-decoration:none">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Results" style="text-decoration:none">Results</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Demonstrate" style="text-decoration:none">Demonstration</A></LI><LI><A href="https://2018.igem.org/Team:Rice/FutureDirections#!" style="text-decoration:none">Future Directions</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Safety" style="text-decoration:none">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Rice/InterLab" style="text-decoration:none">InterLab</A></LI></UL><LI><A href="#!" style="text-decoration:none">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Human_Practices" style="text-decoration:none">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Public_Engagement" style="text-decoration:none">Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Collaborations" style="text-decoration:none">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Integrated_HP" style="text-decoration:none">Integrated Human Practices</A></LI><LI><A href="#!" style="text-decoration:none">Achievements</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Parts" style="text-decoration:none">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Medal_Requirements" style="text-decoration:none">Medal Requirements</A></LI><LI><A href="#!" style="text-decoration:none">Team</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Members" style="text-decoration:none">Members</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Mentors" style="text-decoration:none">Advisors</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Instructors" style="text-decoration:none">Instructors</A></LI><LI><A href="https://2018.igem.org/Team:Rice/Attributions" style="text-decoration:none">Attributions</A></LI></NAV></DIV><DIV class="igem_2017_content_wrapper"><DIV id="topbox"><H1>Design
            </H1></DIV><DIV class="container" align="center"><H2>Introduction</H2><P> The goal of our project is to create a portable transcription-translation resource (PORTAL) which will insulate the expression of genetic circuit from host machinery and reduce the competition for resources which occurs between circuit and host genes (Figure 1). To create portal, we design a genetic circuit that incorporates orthogonal transcription and translation modules that together constitute an expression resource to drive the production of a fluorescent protein. Fluorescence measurements will be performed across strains to determine whether PORTAL improves gene expression consistency in non-model bacteria. We also decided to create intermediate genetic constructs to allow for independent characterization of orthogonal transcription and translation systems. </P><DIV class="center caption"><B class="caption">Figure 3: circuit expression with host vs. orthogonal components.</B> When circuit machinery is used for circuit expression, undesired effects may arise due to competition for resources between host and circuit genes (left). An alternative way is to introduce circuit-specific resources to minimize cross-talk between host and circuit processes (right).  
<H2>Improving Orthogonal Transcription System for Cross-Species Expression </H2><P>Orthogonal transcription has previously been implemented by a number of studies. For example, Kushwaha &amp; Salis developed a universal bacterial expression resource (UBER) and demonstrated its function in <I>E. coli</I>, <I>P. putida</I>, and <I>B. subtilis</I><SUP> 1</SUP> (Figure 1).The first part of our project consists of improving this system to make it more suitable for cross-species expression. </P><DIV class="center caption"><B class="caption">
Figure 1: Universal bacterial expression resource (UBER).
</B> Constitutive priming promoter (green) is used to provide original levels of T7 RNAP.  T7 RNAP then transcribes genes under T7 promoter, including itself, GFP reporter, and TetR repressor. Repression by TetR is used to provide negative feedback on T7 RNAP levels to help reduce toxicity and improve consistency of expression across strains. 
</DIV><P> 
We address the following flaws of UBER: (1) Presence of eukaryotic sequences such as nuclear localization signals (NLSs)  which may interfere with cross-species expression; (2) The use of eukaryotic priming promoter with unknown host range; (3) requirement of two origins of replication to work in a given strain (it may be problematic to find two compatible origin of replication for some undomesticated bacteria); (4) replacement of origin depending on host destination (different origins of replication was used for <I>P. putida</I> and <I>S. oneidendis</I> and <I>B. subtilis</I> required chromosomal integration).(Figure 3).  
</P><P> To address those problems, we have introduced the following modifications: (1) Removed nuclear localization signals; (2) Used one of the broad host range regulatory elements characterized by Johns <I> et al. </I><SUP> 2 </SUP> as a priming promoter; (3) assembled the system on a single plasmid; (4) used a well-characterized broad host origin of replication RSF1010 which works across a wide variety of bacteria.</P><DIV class="center caption"><B class="caption">
Figure 2: modifications introduced to address the problems of UBER design.
</B> Two-plasmid system on the left represents the original UBER from Kushwaha &amp; Salis, which includes T7 RNAP and TetR expression cassettes (blue box) on the first plasmid and a reporter expression cassette on the second plasmid (purple box). Circuit on the right shows the modified orthogonal transcription system which is optimized for cross-species expression. 
</DIV><DIV class="center"><H2>Orthogonal Translation</H2></DIV><P align="left"> Orthogonal translation has previously been tested in <I>E. coli</I>: as demonstrated by Darlington <I>et al.</I>, the advantages of orthogonal translation driving circuit expression include a possibility to increase circuit protein production with lower metabolic burden on the host cell, decreased gene coupling through reduced competition between expression of host and circuit genes, and creation of dynamic allocation of resources to modulate the rates of circuit proteins production<SUP>3</SUP>. To date, orthogonal ribosomes have only been used in <I>E. coli</I> and we hypothesize that the advantages of orthogonal translation can be extended to other bacterial strains. Our final goal is to use orthogonal translation to construct a combined orthogonal transcription-translation system.</P><P> Before assembling the full orthogonal transcription-translation system, we have decided to test the orthogonal ASD (anti Shine-Dalgarno) and SD (Shine-Dalgarno) sequences predicted by the <A href="https://2018.igem.org/Team:Rice/Software"> algorithm </A> written by our team in <I> E. coli </I>. For this purpose, we designed two expression cassettes (Figure 3). A plasmid containing mKate2 and orthogonal RBS can be co-transformed with a plasmid containing orthogonal 16S rRNA under the control of inducible Plac promoter to assess translation level from the orthogonal ribosome. Background expression by host ribosomes can be measured by transforming cells only with plasmid containing mKate2 and measuring fluorescence. <A href=" https://2018.igem.org/Team:Rice/Background"> Click here to learn more about design considerations related to orthogonal translation.</A></P><DIV class="center caption"><DIV class="center caption"><B class="caption">
Figure 1: Constructs used to test orthogonal translation in <I> E. coli. </I></B> Host ribosomes are unable to bind the orthogonal RBS of mKate2 reporter and no fluorescence is observed unless a second plasmid containing a cognate orthogonal 16S rRNA is introduced. The expression of the orthogonal 16S rRNA results in creation of orthogonal ribosomes which can translate mKate2 mRNA, resulting in detectable fluorescence.
</DIV><DIV class="center"><H2>Circuit Design for Combined Orthogonal Transcription-Translation System</H2></DIV><P> 
The final design of our system involves a combined orthogonal transcription-translation system (Figure 4). To further improve the consistency of genetic circuit behavior, we have added a number of insulators. SccJ ribozyme is placed upstream of the RBS to remove the context of 5’ UTR which may affect the translation differently in different strains. We have also placed bidirectional terminators on either side of each expression cassette to prevent undesired effects of possible transcriptional read-through from one cassette to the next. This feature is important in our design since we are using T7 promoters, which result in high transcription rates. 
</P><DIV class="center caption"><B class="caption">Figure 5: Circuit design for combined orthogonal transcription-translation system.</B> 
Constitutive priming promoter (orange) provides original levels of T7 RNAP. T7 RNAP then transcribes genes under T7 promoter, including itself, mKate2 reporter, and TetR repressor. mKate2 translation is initiated when IPTG is added to the system to induce the production of orthogonal 16S rRNA. 16S rRNA associates with large ribosomal subunit to create orthogonal ribosomes, which translate mKate2 mRNA with orthogonal RBS, resulting in detectable fluorescence.  
</DIV><DIV class="center caption"><H2>References</H2></DIV><P>
[1] Kushwaha, M., &amp; Salis, H. M. (2015). A portable expression resource for engineering cross-species genetic circuits and pathways. Nature Communications, 6(1), 7832. 
</P><P>
[2] Johns, N. I., Gomes, A. L. C., Yim, S. S., Yang, A., Blazejewski, T., Smillie, C. S., … Wang, H. H. (2018). Metagenomic mining of regulatory elements enables programmable species-selective gene expression. Nature Methods, 15(5), 323–329
</P><P>
[3] Aedo, S. J., Gelderman, G., &amp; Brynildsen, M. P. (2017). Tackling host–circuit give and take. Nature Microbiology , 2(12), 1584–1585.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>