Team:Rice/Demonstrate
Home
Project
Overview
Design
Background
Modeling
Software
Notebook
Protocols
Results
Demonstration
Future Directions
Safety
InterLab
Human Practices
Overview
Education & Engagement
Collaborations
Integrated Human Practices
Achievements
Parts
Medal Requirements
Team
Members
Advisors
Instructors
Attributions
Demonstration
Broad host range regulatory elements
During our project, we have successfully assembled constructs for and characterized five broad host range regulatory elements from Johns et al1(Figure 1). The results demonstrate that most regulatory elements exhibit variability across species, with the exception of the Strength 8 construct which resulted in the most consistent fluorescence levels across strains. Our data contributes to the characterization of broad host range parts and can aid other researchers in the design of cross-species expression constructs.
Figure 1: broad host range plasmid for mKate2 expression with HW elements. A construct for mKate2 expression with HW elements was assembled on a broad host range plasmid with RSF1010 origin to allow for characterization in a range of bacteria.
Orthogonal Transcription
We have also successfully introduced modifications to the universal bacterial expression resource (UBER) from Kushwaha & Salis. The two-plasmid system we made performed well in E. coli DH10B (Figure 2). In addition, we have demonstrated that the final construct, in which orthogonal transcription and reporter cassettes are assembled on a single plasmid, functions in P. putida. Visit our results page to learn more about the performance of the orthogonal transcription system we built.
Figure 2: Performance of original vs. modified UBER in E. coli mKate expression driven by either original or modified UBER was measured over 15 hours. Modifications introduced do not interfere with UBER function and result in higher fluorescence/OD compared to the original system.
Orthogonal Translation
Our team created software and graphical user interface which allows a user to easily generate anti-Shine-Dalgarno sequence for orthogonal ribosome for any bacteria whose genome is available at NCBI (Figure 4).
Figure 4: User Interface for Portal software.
As demonstrated by our data, the anti-Shine-Dalgarno sequence predicted by our software allowed us to create a functional orthogonal ribosome which selectively initiated translation at the oRBS of mKate2. As can be concluded from the lack of fluorescence when no orthogonal 16S rRNA is expressed, host ribosome is unable to translate the mRNA containing the orthogonal RBS.
Figure 10: constructs used to test orthogonal translation in E. coli. Shine-Dalgarno sequence of the oRBS is unable to interact with host ribosome and no fluorescence is observed when o16S rRNA is not present (negative control). When plasmid containing o16S rRNA is introduced, orthogonal ribosomes are created which are able to translate mRNA containing oRBS, resulting in reporter expression.
Figure 11: characterization of orthogonal translation constructs in E. coli . IPTG-dependent mKate2 fluorescence is observed when plamids containing o16S rRNA controlled by Plac promoter and oRBS-mKate2 are co-transformed into E. coli (left). At 0.1 mM IPTG, ~100 fold fluorescence over negative control is observed, demonstrating that translation is orthogonal (right).
BioBricks
We have built BioBricks for broad host range regulatory elements, modified UBER, and orthogonal ribosome binding site. Visit our parts page to learn more about the parts we submitted.
References
[1] Johns, N. I., Gomes, A. L. C., Yim, S. S., Yang, A., Blazejewski, T., Smillie, C. S., Wang, H. H. (2018). Metagenomic mining of regulatory elements enables programmable species-selective gene expression. Nature Methods, 15(5), 323â€“329.
