Team:Rice/Software
Home
Project
Overview
Design
Background
Modeling
Software
Notebook
Protocols
Results
Demonstration
Future Directions
Safety
InterLab
Human Practices
Overview
Education & Engagement
Collaborations
Integrated Human Practices
Achievements
Parts
Medal Requirements
Team
Members
Advisors
Instructors
Attributions
Software
Software Overview
This algorithm is extended from an algorithm used to design orthogonal ribosomes for Escherichia coli[1]. The software component of our project aims to extend this algorithm to be applicable to a variety of bacterial strains, and to make the software easier to access and use for fellow synthetic biologists. Our released software can be downloaded from the IGEM Judging Release (Click here for more details.) and the associated GitHub repositories.
Library Initialization
In order to choose specific sequences that would work best in the Anti Shine-Dalgarno (ASD) region of orthogonal ribosomes, we first consider all 4096 possibilities of the 6 base-pair Shine-Dalgarno (SD) sequence in our initial library. All the SD sequences are wrapped with a 4 base-pair “prefix” and 2 base-pair “suffix” which, together, creates the 12 base pair ASD sequence. The prefix and suffix are determined by analyzing the genome for SD motifs, and determining the wild-type prefix and suffix for that particular organism. Click here for more details.
Figure 1: Creating a library of 4096 candidate pairs.
Candidate Elimination Based on Energy Constraints
The library is first narrowed down based on two energy constraints.
The first energy constraint is to ensure that the candidate pairs' binding energy is close enough to the wild type binding energy. This is to ensure translation initiation. We use RNAduplex from the ViennaRNA package to calculate binding energies for all 4096 pairs of SD/ ASD sequences, and eliminate pairs that have binding energies deviating too far from the wild type's value. Click here for more details.
Figure 2a: Energy constraint: Ensuring similary of candidates' and wild type's binding energy.
The second energy constraint is to ensure orthogonality by imposing a binding threshold between the wild type SD with candidate ASD. Again, we use RNAduplex to calculate binding energies, and eliminate candidates which have low, favourable wild type SD-candidate ASD binding energies. Click here for more details.
Figure 2b: Energy constraint: Ensuring weak interaction between candidate ASD and host SD.
Candidate Elimination Based on Structural Constraints
We use RNAfold from the ViennaRNA package to calculate the secondary structure for the full 16s rRNA.  Candidates with secondary structure in the ASD regions are discarded, as this would impair their ability to carry out translation. Click here for more details.
Figure 3: Structural constraint: Eliminating candidates with secondary structures forming in ASD region.
Candidate Ranking Based on Host Interactions
Figure 4: Ranking candidates based on number of strong host interactions.
In order to rank the remaining candidates, we first obtain all the translation initiation regions (TIRs) from the genome.
We use these TIRs in conjunction with RNAduplex once again to calculate the binding energies of the remaining candidate ASDs with those TIRs.
Candidates are then ranked based on their binding energies with the host TIRs. Candidates with higher binding energies (which are less likely to bind with host TIRs) are given preference, since they are more likely to remain orthogonal to the host processes. Click here for more details.
Graphical User Interface
In order to make the algorithm more accessible and easier to use, we have developed a graphical user interface (GUI) for our software. The detailed instructions on how to use the software can be found here. The GUI is developed on a Windows 10 system.
Figure 5: User Interface for Portal software.
Software Results
We have run our software for 7 different bacterial strains: Escherichia Coli K-12 MG1655, Pseudomonas putida F1, Shewanella oneidensis MR-1, Sinorhizobium meliloti 1021, Vibrio natriegens 14048, Corynebacterium glutamicum ATCC 13032 and Bacillus subtilis 6051-HGW. The top candidate for each strain is reflected below in figure 6.
Figure 6: Top candidates for each strand.
We also carried out some preliminary analysis with unranked libraries of those 7 strains. Surprisingly, we found a common candidate for 6 of those 7 strains (except the Bacillus Subtilis strain). The candidate has the complementary "UGCUGC" Shine-Dalgarno sequence. We strongly believe this candidate can be our first step towards a "universal" 16s ribosomal candidate. Our team succeeded to verify that this candidate when used to design an orthogonal ribosome, results in desired expression of the orthogonal circuit. (Click here to be directed to the project results page for more information.)
Fitting PORTAL Software into Orthogonal Transcriptional-Translational Circuit Work Flow
The PORTAL software is created to replace the traditional method of using random mutagenesis-based approaches to identify the correct orthogonal mutants of 16s rRNA.
In the workflow, the first step is choosing a bacterial strain most suitable for the application (Figure 7a). Then, the PORTAL software will computationally identify the mutated sequence for an orthogonal 16s rRNA. This mutant is cloned through PCR (Figure 7b), and then is combined with an oRBS to create the orthogonal translational portion. This is then combined with the orthogonal transcriptional section to form a single plasmid, which when transformed will lead to the expression of the desired reporter/protein.
Figure 7: Workflow for creating an Orthogonal Transcriptional-Translational Circuit.
References
[1] Darlington, A.P.S., Kim, J., Jiménez, J.I., & Bates, D.G. (2018). Dynamic allocation of orthogonal ribosomes facilitates uncoupling of co-expressed genes. Nature Communications, 9, 695.
[2] Ding, Y., Chan, C. Y., & Lawrence, C. E. (2005). RNA secondary structure prediction by centroids in a Boltzmann weighted ensemble. RNA, 11(8), 1157–1166. http://doi.org/10.1261/rna.2500605
[3] “TBI - RNAduplex - Manpage.” Accessed September 12, 2018. https://www.tbi.univie.ac.at/RNA/RNAduplex.1.html.
<
