Team:Rice/Background
Home
Project
Overview
Design
Background
Modeling
Software
Notebook
Protocols
Results
Demonstration
Future Directions
Safety
InterLab
Human Practices
Overview
Education & Engagement
Collaborations
Integrated Human Practices
Achievements
Parts
Medal Requirements
Team
Members
Advisors
Instructors
Attributions
Background
Orthogonal Gene Expression in Bacteria
Overview
The process of engineering living organisms is often complicated by the fact that biological components can participate in multiple reactions that are hard to control, creating unwanted crosstalk between different parts of the system. Therefore, orthogonality (an idea of creating synthetic pathways whose components have minimized number of undesirable interactions among each other or with the host) is an important concept in synthetic biology.  Our goal is to achieve orthogonality of genetic circuit expression on the levels of transcription and translation in order to minimize crosstalk between circuit and host components and thus reduce host-dependent variability of circuit performance. The orthogonal transcription-translation system we use is based on T7 RNA polymerase and orthogonal ribosomes (Figure 1).
Figure 1: Overview of orthogonal transcription-translation. The expression of circuit genes controlled by orthogonal regulatory elements only occurs in presence of T7 RNA polymerase and orthogonal ribosomes due to the lack of interaction between circuit promoters and ribosome binding sites with host machinery.
Expressing RNA polymerase from T7 phage allows to insulate transcription from the host machinery. This polymerase is specific to T7 promoters and the promoters are fully inactive in the absence of the T7 polymerase. T7 promoters can function in a variety of strains and therefore are suitable for creating cross-species genetic circuits1.
Prokaryotic ribosome consists of two subunits: small 30S and large 50S. The large ribosomal subunit contains 5S and 23S rRNA and participates in peptide bond synthesis while the small ribosomal subunit is involved in mRNA recognition through 16S rRNA. Specifically, the anti-Shine Dalgarno region (a short sequence found near the end of 16S rRNA) recognizes and binds to the Shine-Dalgarno region on the mRNA ribosome binding site, resulting in translation initiation2. The free energy of binding between SD and ASD sequences determine the translation initiation rate and mutation in the SD sequence which results in sufficiently high ASD-SD binding energy will lead to the lack of mRNA translation. This principle can be used to create orthogonal ribosomes3. A mutation in SD region of mRNA RBS is introduced to prevent the binding of wild-type ribosome. 16S rRNA with a corresponding ASD mutation can then be expressed to compete with the wild-type 16S rRNA for the large ribosomal subunits, resulting in a pool of orthogonal ribosomes targeted specifically to the mRNAs containing SD mutation (Figure 2).
Figure 2: Mutations in SD and ASD sequences result in creation of orthogonal ribosomes. When SD sequence of RBS is altered to have weak interaction with host ribosome, corresponding mutation can be introduced into the 16S rRNA ASD to create a pool of orthogonal ribosomes specifically targeted to the translation of mRNA containing the SD mutation.
Orthogonal Ribosomes for Cross-Species Expression
Our project aims to use orthogonal translation for cross-species cricut expression. Considering the nuances of the interaction between orthogonal 16S rRNA expressed from a circuit and large ribosomal subunit, an optimal design might be to express 16S rRNA from each organism to facilitate its association with the large ribosomal subunit. However, evidence exists which suggests that 16S rRNA can be transferred across organism without the loss of function4. According to Kitahara et al. , 16S rRNA from a variety of organisms was successfully expressed in E. coli even if the similarity between the 16S sequences was as low as 80%. After comparing 16S rRNA sequences from the bacteria used in our project, we hypothesized that E. coli 16S rRNA can be expressed in other gram-negative bacteria we have selected due to high sequence similarity (Figure 3). If E. coli 16S rRNA fails in gram-positive bacteria, future work would involve expressing native 16S rRNA for each species.
Figure 3: Pairwise percent identities of 16S rRNA sequences from organisms used in the project. The comparison of pairwise percent identities demonstrates that 16S rRNA sequences among gram-negative bacteria have > 80% similarity.
Truncation point of the ribosomal operon to express 16S rRNA separately from other ribosomal genes was identified based on previous work by An & Chin 5 . They have demonstrated that 16S rRNA can be expressed without loss of function if the ribosomal operon is truncated to include a region downstream of 16S rRNA containing RNAse III sites where the sequence is naturally cleaved.
References
[1] Kushwaha, M., & Salis, H. M. (2015). A portable expression resource for engineering cross-species genetic circuits and pathways. Nature Communications, 6(1), 7832.
[2] Fox, G. E. (2010). Origin and evolution of the ribosome. Cold Spring Harbor Perspectives in Biology, 2(9).
[3] Rackham, O., & Chin, J. W. (2005). A network of orthogonal ribosome·mRNA pairs. Nature Chemical Biology, 1(3), 159–166.
[4] Kitahara, Kei, Yoshiaki Yasutake, and Kentaro Miyazaki (2012). Mutational Robustness of 16S Ribosomal RNA, Shown by Experimental Horizontal Gene Transfer in Escherichia Coli.” Proceedings of the National Academy of Sciences of the United States of America 109 (47), 19220–25.
[5] An, W., & Chin, J. W. (2009). Synthesis of orthogonal transcription-translation networks. Proceedings of the National Academy of Sciences of the United States of America, 106(21), 8477–8482.
