<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Davis_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UC_Davis/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Davis/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Davis/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Davis/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Davis/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Davis/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><UL class="nav"><LI class="dropdown leftbookend"><A href="/Team:UC_Davis" style="font-size:13px;">Home</A></LI><LI class="dropdown leftbookend rightbookend"><A href="https://2013.igem.org/Team:UC_Davis/Team_Overview" style="font-size:13px;">Team</A></LI><LI><A href="https://2013.igem.org/Team:UC_Davis/Team_Overview">Overview</A></LI><LI><A href="https://2013.igem.org/Team:UC_Davis/Bios">Bios</A></LI><LI><A href="/Team:UC_Davis/Contact">Contact Us</A></LI><LI><A href="https://igem.org/Team.cgi?id=1212">Official Team Profile</A></LI></UL><LI class="dropdown leftbookend rightbookend"><A href="https://2013.igem.org/Team:UC_Davis/Project_Overview" style="font-size:13px;">Project</A></LI><LI><A href="https://2013.igem.org/Team:UC_Davis/Project_Overview">Overview</A></LI><LI><A href="/Team:UC_Davis/Modeling">Modeling</A></LI><LI><A href="/Team:UC_Davis/Assembly">Assembly</A></LI><LI><A href="/Team:UC_Davis/Parts">Parts</A></LI><LI><A href="/Team:UC_Davis/Safety">Safety</A></LI><LI class="dropdown leftbookend rightbookend"><A href="/Team:UC_Davis/Data" style="font-size:13px;">Data</A></LI><LI><A href="/Team:UC_Davis/Data">Testing Constructs</A></LI><LI><A href="/Team:UC_Davis/AndersonPromoters">Anderson Promoters</A></LI><LI><A href="/Team:UC_Davis/AndersonPromoters2">Anderson II</A></LI><LI class="dropdown leftbookend rightbookend"><A href="/Team:UC_Davis/Notebook_Overview" style="font-size:13px;">Notebook</A></LI><LI><A href="/Team:UC_Davis/Notebook">Notebook</A></LI><LI><A href="/Team:UC_Davis/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:UC_Davis/Gallery">Gallery</A></LI><LI><A href="https://2013.igem.org/Team:UC_Davis/Attributions">Attributions</A></LI><LI class="dropdown rightbookend"><A href="/Team:UC_Davis/HumanPracticesOverview" style="font-size:13px;">Human Practices</A></LI><LI><A href="/Team:UC_Davis/HumanPracticesOverview">Overview</A></LI><LI><A href="/Team:UC_Davis/Outreach">Outreach</A></LI><LI><A href="/Team:UC_Davis/Database">The Depot</A></LI></P><DIV class="floatboxwide"><H1>Protocols</H1><DIV id="firstpane" class="menu_list"><P class="menu_head"> LB Media </P><DIV class="menu_body"><TABLE class="gray"><TBODY><TR><TD><LI>950 mL  dH<SUB>2</SUB>0</LI><LI>10 g       Tryptone</LI><LI>10 g       NaCl</LI><LI>5 g         Yeast Extract</LI>
1 L  Total (add dH<SUB>2</SUB>0 to reach total volume)
<LI>Add 15 g Agar, if being poured into plates.</LI><LI>Autoclave, when cool add antibiotics if desired.</LI></TD></TR></TBODY></TABLE></DIV><P class="menu_head"> Antibiotic Stock Solutions</P><DIV class="menu_body"><P>Materials</P>
Chloramphenicol
<LI>Working Concentration 12.5 μg/ml</LI><LI>Stock solutions can be made at 35 mg/ml in ethanol and kept at -20º C</LI>

Kanamycin
<LI>Filter sterilize for kanamycin</LI>
Working Concentration:
<LI>25 μg/mL for low-copy plasmids</LI><LI>35 µg/mL for high-copy plasmids</LI><LI>Stock solution is 35 mg/ml in water and kept at -20º C</LI>

Spectinomycin 
<LI>Filter sterilize</LI><LI>Working Concentration 50 μg/mL</LI><LI>Stock solution is 100 mg/mL in water and kept at -20º C</LI></DIV><P class="menu_head"> Heat Shock Transformation</P><DIV class="menu_body"><P>Procedure</P><OL><LI>Preheat water bath to 42º C.</LI><LI>Thaw competent cells on ice for 10 minutes.</LI><LI>Use 50 µL chemically competent cells, add transforming DNA [up to 25 ng per 50 µL of cells, volume not exceeding 2.5 µL (5%)]. For control add 1 µL of control DNA (e.g. PUC19). Swirl gently to mix, store on ice 5 minutes. </LI><LI>Heat shock in 42º C water bath for 45 seconds.</LI><LI>Cool cells in ice bath for 1-2 minutes.</LI><LI>Add 800 µL LB to each tube, set in shaker at 37º C for 45 minutes. </LI><LI>Transfer 200 µL of culture to a plate (containing the appropriate antibiotic).</LI><LI>Spread using glass beads.</LI><LI>Invert plates and incubate overnight (12-16 hrs) at 37º C.</LI></OL></DIV><P class="menu_head"> Making Chemically Competent Cells</P><DIV class="menu_body"><P>Procedure</P><OL><LI>Prepare 0.5 M PIPES (pH 6.7) (piperazine-1,2-bis[2-ethanesulfonic acid]) by dissolving 15.1 g of PIPES in 80 ml of pure H2O (Milli-Q, or equivalent). Adjust the pH of the solution to 6.7 with 5 M KOH, and then add pure H<SUB>2</SUB>O to bring the final volume to 100 ml. Sterilize the solution by filtration through a disposable pre-rinsed Nalgene filter (0.45-µm pore size). Divide into aliquots and store frozen at -20°C.
        Prepare Inoue transformation buffer by dissolving all of the solutes listed below in 800 mL of pure H2O and then add 20 ml of 0.5 M PIPES (pH 6.7). Adjust the volume of the Inoue transformation buffer to 1 liter with pure H<SUB>2</SUB>O.</LI><TABLE><TBODY><TR><TH>Reagent</TH><TH>Amount per liter</TH><TH>Final Concentration</TH></TR><TR><TD>MnCl<SUB>2</SUB>•4H<SUB>2</SUB>O</TD><TD>10.88 g</TD><TD>55 mM</TD></TR><TR><TD>CaCl<SUB>2</SUB>•2H<SUB>2</SUB>O</TD><TD>2.20 g</TD><TD>15 mM</TD></TR><TR><TD>KCl</TD><TD>18.65 g</TD><TD>250 mM</TD></TR><TR><TD>PIPES (0.5 M, pH 6.7)</TD><TD>20 ml</TD><TD>10 mM</TD></TR></TBODY></TABLE>
Add H<SUB>2</SUB>O to fill solution to 1 liter. Sterilize Inoue transformation buffer by filtration through a prerinsed 0.45-µm Nalgene filter. Divide into aliquots and store at -20°C.

<LI>Pick a single bacterial colony (2-3 mm in diameter) from a plate that has been incubated for 16-20 hours at 37°C. Transfer the colony into 25 ml of SOB medium (LB may be used instead) in a 250-ml flask. Incubate the culture for 6-8 hours at 37°C with vigorous shaking (250-300 rpm).</LI><LI>At about 6 o'clock in the evening, use this starter culture to inoculate three 1-liter flasks, each containing 250 ml of SOB. The first flask receives 10 ml of starter culture, the second receives 4 ml, and the third receives 2 ml. Incubate all three flasks overnight at 18-22°C with moderate shaking.</LI><LI> The following morning, read the OD<SUB>600</SUB> of all three cultures. Continue to monitor the OD every 45 minutes.</LI><LI>When the OD<SUB>600</SUB> of one of the cultures reaches 0.55, transfer the culture vessel to an ice-water bath for 10 minutes. Discard the two other cultures.
</LI><LI>Harvest the cells by centrifugation at 2500g (3900 rpm in a Sorvall GSA rotor) for 10 minutes at 4°C.</LI><LI>Pour off the medium and store the open centrifuge bottle on a stack of paper towels for 2 minutes. Use a vacuum aspirator to remove any drops of remaining medium adhering to walls of the centrifuge bottle or trapped in its neck.</LI><LI>Resuspend the cells gently in 80 ml of ice-cold Inoue transformation buffer.</LI><LI>Harvest the cells by centrifugation at 2500g (3900 rpm in a Sorvall GSA rotor) for 10 minutes at 4°C.</LI><LI>Pour off the medium and store the open centrifuge tube on a stack of paper towels for 2 minutes. Use a vacuum aspirator to remove any drops of remaining medium  adhering to the walls of the centrifuge tube or trapped in its neck.</LI><LI>Resuspend the cells gently in 20 ml of ice-cold Inoue transformation buffer.</LI><LI>Add 1.5 ml of DMSO. Mix the bacterial suspension by swirling and then store it in ice for 10 minutes.</LI><LI>Working quickly, dispense aliquots of the suspensions into chilled, sterile microcentrifuge tubes. Immediately snap-freeze the competent cells by immersing the tightly closed tubes in a bath of liquid nitrogen. Store the tubes at -70°C until needed.
</LI></OL></DIV><P class="menu_head">Double Restriction (Fast) Digest</P><DIV class="menu_body"><P>Materials</P><LI>~20 µL DNA (as much as possible)</LI><LI>1 µL Restriction Enzyme 1</LI><LI>1 µL Restriction Enzyme 2</LI><LI>5 µL 10X Universal Buffer (with loading dye)</LI><LI>Add appropriate amount of dH<SUB>2</SUB>0 (µL)</LI>

50 µL  Total                       → incubate at 37º C for 3 hrs.

<P>Procedure</P><LI>Treat insert with XbaI and PstI</LI><LI>Treat vector with SpeI and PstI</LI><LI>After incubating at 37º C for 3 hrs, either heat inactivate the enzymes or proceed directly to gel extraction and purification.</LI></DIV><P class="menu_head"> Ligation</P><DIV class="menu_body"><P>Materials</P>
Ligation reaction:
<LI>__ µL vector DNA</LI><LI>__ µL insert DNA</LI><LI>2 µL T4 10x Buffer</LI><LI>1 µL DNA ligase</LI><LI>Add appropriate amount of dH<SUB>2</SUB>0 (µL) to reach total volume</LI>
20 µL Total             →Leave at room temperature for 20 minutes

Vector control:
<LI>__ µL vector DNA (same volume as ligation reaction)</LI><LI>2 µL Buffer</LI><LI>1 µL ligase</LI><LI>Add appropriate amount of dH<SUB>2</SUB>0 (µL) to reach total volume</LI>
20 µL total 

Insert control:
<LI>__ µL insert DNA (same volume as ligation reaction)</LI><LI>2 µL Buffer</LI><LI>1 µL ligase</LI><LI>Add appropriate amount of dH<SUB>2</SUB>0 (µL) to reach total volume</LI>
20 uL Total
<P>Procedure</P><LI>Mix these materials in the amounts determined by the reaction volume calculator for the vector and insert DNA <A href="https://static.igem.org/mediawiki/2011/7/73/UC_Davis_Reaction_Volume_Calculator.xls">here</A>.
    </LI></DIV><P class="menu_head">Gel Electrophoresis</P><DIV class="menu_body"><TABLE class="gray"><TBODY><TR><TD><P>Procedure</P><OL><LI>Add 0.5 grams of agarose to 50 mL of 1X TAE buffer. (1% agarose gel)</LI><LI>Microwave agarose/TAE until agarose is completely dissolved. (Caution: Will be hot)</LI><LI>Cool under water or let sit until cool.</LI><LI>Add SYBR safe dye (2.5-3µL).</LI><LI>Pour into mold with appropriate comb.</LI><LI>Wait 15 minutes for gel to solidify.</LI><LI>Transfer gel to a gel electrophoresis box.</LI><LI>Load DNA and appropriate DNA standard or ladder with dye into wells while submerged in 1X TAE buffer.</LI><LI>Run gel at a constant 120V.</LI><LI>Check gel periodically until the dye has run the desired length of the gel.</LI><LI>View gel under short wavelength or UV light, with proper protection, to check for bands.</LI></OL></TD></TR></TBODY></TABLE></DIV><P class="menu_head">Gel Extraction and Purification</P><DIV class="menu_body"><P>Procedure</P><OL><LI>Prepare agarose gel (see gel electrophoresis) and use additional or larger combs to make larger wells.</LI><LI>Once the gel has run, use preferably a blue light lamp, or UV for a very short time (it will degrade your DNA), (in the dark, wearing goggles and skin protection) to identify bands. </LI><LI>Cut out the desired bands with a stamp pipette tip or clean razor and transfer to a clean tube.  The stamp pipette tip can be left in the tube to be cleaned out with a smaller pipette tip. </LI><TABLE class="gray"><TBODY><TR><TD><LI>Weigh gel fragments and add 200 µL Buffer NTI for every 100 mg agarose gel.</LI>
 Incubate sample for 5-10 min at 50º C, vortexing every 2-3 min until the gel is completely dissolved.<LI>Load sample onto column over collection tube.  Centrifuge for 30 sec at 11,000 x g.</LI><LI>Discard flow-through and replace column on tube.</LI><LI>Add 700 µL Buffer NT3 and centrifuge for 30 sec. at 11,000 x g. Discard flow-through.</LI><LI>Centrifuge for 2 min at 11,000 x g to completely remove buffer.</LI><LI>Transfer column to a 1.5 mL tube and add 20 µL ddH<SUB>2</SUB>O.</LI><LI>Incubate at room temperature for 10 min.</LI><LI>Centrifuge for 1 min at 11,000 x g. </LI></TD></TR></TBODY></TABLE><P class="menu_head">PCR Amplification</P><DIV class="menu_body"><P>Materials</P><LI>10 µL 5x HF Buffer</LI><LI>1   uL dNTPs</LI><LI>2.5 µL Forward Primer</LI><LI>2.5 µL Reverse Primer</LI><LI>100 ng Template DNA (2 ng/µL)</LI><LI>0.5 µL DNA Phusion Polymerase</LI><LI>Add appropriate amount of ddH<SUB>2</SUB>O to reach total volume</LI>
50 µL Total 
 
<P>
or for DNA with high G/C content we found a combination of Taq and Pfu gave us better results.
</P><LI>5 µL 10x Buffer</LI><LI>10 uL Q solution</LI><LI>2.5 µL Forward Primer</LI><LI>2.5 µL Reverse Primer</LI><LI>1.25 µL DNTPs</LI><LI>100 ng Template DNA (2 ng/µL)</LI><LI>0.3 µL Taq DNA Polymerase</LI><LI>0.1 µL Cloned Pfu DNA Polymerase</LI><LI>Add appropriate amount of ddH<SUB>2</SUB>O to reach total volume</LI>
50 µL Total 
<P>
PCR program
<OL><LI>98º C 30 sec</LI><LI>98º C 10 sec</LI><LI>___º C 30 sec  (Temperature depends on Tm of your primers)</LI><LI>72º C 1 min / kb Repeat Steps 2-4 29x (30x total)</LI><LI>72º C 5 min</LI><LI>4º C Hold</LI></OL></P><LI>Run PCR products on a 1% agarose gel for verification. If the gel is good, perform PCR clean up with your kit of choice.</LI></DIV><P class="menu_head">Golden Gate Assembly</P><DIV class="menu_body"><P>Materials</P><LI>40 fmol of DNA for each part (or 100 ng if your parts are all roughly the same size)</LI><LI>.75 µL BsaI</LI><LI>.2 µL 100x BSA</LI><LI>1 µL T4 DNA ligase</LI><LI>2 µL 10X T4 ligase buffer</LI><LI>Add appropriate amount of ddH<SUB>2</SUB>0 to reach total volume.</LI>
20 µL Total
<P>
To convert your DNA concentration to fmol/µL, use the equation 1µg of 1kb DNA = 1.52 pmol.
</P><P>PCR Program</P><OL><LI>37º C 2 min</LI><LI>16º C 3 min Repeat Steps 1-2 49x (50x total)</LI><LI>50º C 5 min</LI><LI>80º C 5 min</LI><LI>4º C Hold</LI></OL><LI>Following PCR, directly transform 5-10 µL of your product into competent cells.</LI></DIV><P class="menu_head">DNA Extraction (Minipreps)</P><DIV class="menu_body"><P>Procedure (Invitrogen Quick Plasmid Miniprep Kit)</P><OL><LI>Sediment the cells by centrifuging 1-5 mL of overnight LB-culture. Remove all medium.</LI><LI> Add 250 µL Resuspension Buffer (R3) with RNase A to the cell pellet and resuspend the pellet until it is homogeneous.</LI><LI>Add 250 µL Lysis Buffer (L7). Mix gently by inverting the capped tube five time. Do not vortex. Incubate the tube at room temperature for 5 minutes.</LI><LI>Add 350 µL Precipitation Buffer (N4). Mix immediately, or for large pellets, vigorously shaking the tube until the mixture is homogeneous. Do not vortex. Centrifuge the lysate at &gt;12,000 x g for 10 minutes.</LI><LI>Load the supernatant from the prior step on a spin column in a 2-mL wash tube. Centrifuge at the column for 12,000 x g for 1 minute. Discard the flow-through and place the column back into the wash tube.</LI><LI>Add 700 µL Wash Buffer (W9) with ethanol to the column. Centrifuge the column at 12,000 x g for 1 minute. Discard the flow-through and place the column into the wash tube. Centrifuge the column at 12,000 x g for 1 minute. Discard the wash tube with the flow-through.</LI><LI>Place the spin column in a clean 1.5 mL recovery tube. Add 30 µL of ddH2O to the center of the column. Incubate the column for 10 minutes at room temperature.</LI><LI>Centrifuge the column at 12,000 x g for 2 minutes. Discard the column. Store plasmid DNA at 4º C (short-term) or store the DNA in aliquots at -20º C (long-term).</LI></OL></DIV><P class="menu_head">Making and Reviving Glycerol Stocks</P><DIV class="menu_body"><P>Procedure</P><LI>Add equal volumes (500-700 µL) of overnight cell culture and 80% glycerol into a cryotube, keep sterile with a flame.</LI><LI>Store at -80º C.</LI><LI>When reviving a glycerol stock, keep the glycerol stock on dry ice.  
</LI><LI>Use a pipette tip to poke and/or slightly swirl glycerol stock and drop tip into 5 mL LB culture with appropriate antibiotic and shake overnight.</LI></DIV><P class="menu_head">Sequencing Preparation</P><DIV class="menu_body"><P>Procedure  (Quintara Biosciences)</P><LI>Primer will have [ng] content printed on label: add 10x H<SUB>2</SUB>0 for DNA at 100 µM.</LI><LI>Need 10 µM solution for sequencing, so dilute a portion of the hydrated primer solution 10x (e.g. 1 µL primer + 9 µL H<SUB>2</SUB>0).</LI><LI>Determine DNA concentration of template DNA.</LI><LI>(Premixed) in 0.5 µL tube</LI><LI>.6 µg    DNA (final concentration: 50 ng/µL)</LI><LI>8 pmol primer (10 µM = .8 µL)</LI><LI>Add appropriate amount of H<SUB>2</SUB>O for total volume.</LI>
12 µL total
    </DIV><P class="menu_head">Measuring DNA Concentration</P><DIV class="menu_body"><TABLE class="gray"><TBODY><TR><TD><P>Procedure</P><OL><LI>Log in to nanodrop program and select Nucleic Acids/DNA.</LI><LI>Moisten a Kimwipe and clean the pedestal.</LI><LI>Apply 2 µL H<SUB>2</SUB>O to pedestal and click 'OK'.</LI><LI>Press 'Blank' button.</LI><LI>Gently wipe blank from pedestal using a Kimwipe.</LI><LI>Apply 2 µL of desired sample to pedestal.</LI><LI>Click 'Measure'.</LI><LI>Gently wipe pedestal using a Kimwipe and continue measuring remaining samples.</LI><LI>Record/print results. </LI></OL></TD></TR></TBODY></TABLE></DIV><P class="menu_head">Tecan Fluorescence Testing</P><DIV class="menu_body"><P>Procedure</P><OL><LI>Grow cultures overnight, or until OD<SUB>600</SUB>&gt;1, in LB at 37 C, 150 RPM. </LI><LI>Measure OD<SUB>600</SUB> and dilute to get &lt;0.01 OD<SUB>600</SUB>.</LI><LI>Grow until the OD<SUB>600</SUB> approaches 0.5.</LI><LI>Load 96 well plate with LB, or other media of choice, depending on the experiment, as well as the appropriate antibiotic, inducer stock solutions, and the appropriate volume of culture so as to reach an OD<SUB>600</SUB> of 0.1 in 200 µL. If changing media, first spin down the culture to pellet, decant and aspirate any remaining media, and resuspend in new media to reach the desired OD<SUB>600</SUB> before loading in plate.</LI><LI>Be sure to include the appropriate positive and negative controls (e.g. blank media, strain control, etc.).</LI></OL><TABLE class="gray"><TBODY><TR><TD><P>Tecan Program Parameters</P><UL><LI>Temperature: 37 C</LI><LI>Orbital Shake Frequency: 244.5 rpm (Amplitude of 2.5 mm)</LI><LI>Orbital Shake Duration: 600 sec</LI><LI>Excitation wavelength: 485 nm</LI><LI>Emission wavelength: 535 nm</LI><LI>Absorbance wavelength (for OD readings): 595 nm</LI><LI>Gain: 35</LI></UL></TD></TR></TBODY></TABLE></DIV><P class="menu_head">Primer Design for Site-Directed Mutagenesis PCR</P><DIV class="menu_body"><P>Procedure</P><OL><LI> Identify site that needs to be mutated.</LI><LI>Check the amino acid sequence to create a silent mutation, generally the last base in a codon.</LI><LI>Check a codon usage table for your chassis to help choose how the codon should be changed, try to pick a frequently used codon. </LI><LI>Take about 20 base pairs upstream and 20 base pairs downstream of your desired mutation site to create your primer. The sequence should be identical to the template except for the one changed base you are trying to mutate at the center.  </LI><LI>The reverse primer will be the reverse complement of this sequence.</LI></OL></DIV><P class="menu_head">Site-Directed Mutagenesis PCR</P><DIV class="menu_body"><P>Materials</P><LI>Primer will have [µg] content printed on label: add dH<SUB>2</SUB>0 1:1 for DNA at 1 µg/µL.</LI><LI>Need 0.1 µg/µL for PCR reaction, so dilute a portion of the hydrated primer solution 10x (e.g. 1 µL primer + 9 µL dH<SUB>2</SUB>0).</LI><LI>Determine DNA concentration of template DNA.</LI><LI>50 ng     Template DNA</LI><LI>5 µL       10x Turbo Buffer</LI><LI>1 µL       Forward Primer (0.1 ug/uL)</LI><LI>1 µL       Reverse Primer (0.1 ug/uL)</LI><LI>1 µL       dNTPs (10 mM)</LI><LI>1 µL       Pfu Turbo (enzyme)</LI><LI>Add appropriate amount of dH<SUB>2</SUB>O to reach total volume.</LI>
50 µL Total
<P>PCR program</P><OL><LI>95º C     1 min</LI><LI>95º C    50 sec</LI><LI>60º C    50 sec   Repeat Steps 2-4 17x (18x total) </LI><LI>68º C    1 min / kb of plasmid    </LI><LI>68º C    7 min</LI><LI>4º C      Hold </LI></OL></DIV><P class="menu_head">Electroporation Transformation</P><DIV class="menu_body"><P>Procedure</P><OL><LI>Thaw electrocompetent cells on ice, keep on ice when thawed.</LI><LI>Aliquot 20-30µL of competent cells into separate 1.5mL microcentrifuge tubes on ice.</LI><LI>Pipette appropriate amount of DNA (1 µL) into tube on ice.</LI><LI>Transfer cell/DNA into prechilled electroporation cuvette, keep on ice for 1 minute.</LI><LI>Make sure the electroporator is set to:
    <UL>Time constant = 4.5-5.0 ms</UL><UL>Resistance = 200 W</UL><UL>Capacitance = 25 mFD</UL><UL>Volts = 1.7 kV (for 1mm gap cuvettes)</UL></LI><LI>Electrocute cells once.</LI><LI>Add 1mL of LB to cuvette, pipette up and down to mix, and transfer mixture to 14mL Falcon culture tube.</LI><LI>Incubate for 1hr at 37C shaking at 150rpm.</LI><LI>Plate cells on appropriate antibiotic.</LI></OL></DIV><P class="menu_head">SOE PCR</P><DIV class="menu_body"><P>Materials</P><LI>Primer will have [µg] content printed on label: add H<SUB>2</SUB>0 1:1 for DNA at 1 µg/µL.</LI><LI>Need 0.1 µg/µL for PCR reaction, so dilute a portion of the hydrated primer solution 10x.</LI><LI>Determine DNA concentration of template DNA.</LI><P>Amplification</P><LI>100 ng     Template DNA</LI><LI>10 µL      HF 5x Buffer</LI><LI>2.5 µL     Forward Primer (0.1 µg/µL)</LI><LI>2.5 µL     Reverse Primer (0.1 µg/µL)</LI><LI>1 µL       dNTPs (10 mM)</LI><LI>0.5 µL     Phusion Polymerase (enzyme)</LI><LI>Add appropriate amount of dH<SUB>2</SUB>O.</LI>
50 µL Total
<P>PCR program</P><OL><LI>98º C      30 sec</LI><LI>98º C      10 sec</LI><LI>55º C      30 sec    </LI><LI>72º C      1 min / kb    Repeat Steps 2-4 29x (30x total)   </LI><LI>72º C      5 min</LI><LI>4º C       Hold </LI></OL>Purify the PCR product and determine the resultant concentration<P>SOE PCR</P><LI>300 ng     Template DNA (larger part)</LI><LI>xx  ng     Template DNA (smaller part) keeping a 1:1 molar ratio </LI><LI>10 µL      HF 5X Buffer</LI><LI>2.5 µL     Forward Primer (0.1 µg/µL)</LI><LI>2.5 µL     Reverse Primer (0.1 µg/µL)</LI><LI>1 µL       dNTPs (10 mM) </LI><LI>0.5 µL     Phusion Polymerase (enzyme)</LI><LI>Add appropriate amount of dH<SUB>2</SUB>O.</LI>
50 µL Total


<P>PCR program</P><LI>Same as above.</LI></DIV><P class="menu_head">Colony PCR</P><DIV class="menu_body"><P>When to Use</P>
Transformation of competent cells may yield false positives. Through colony PCR, one may screen for the presence of the assembled sequence by amplifying with the universal forward and reverse primers and checking the length of the fragments through gel electrophoresis.

<P>Materials</P><LI>2 µL     10x Buffer</LI><LI>4 µL     Q solution</LI><LI>1 µL     Universal Forward Primer (0.1 µg/µL)</LI><LI>1 µL     Universal Reverse Primer (0.1 µg/µL)</LI><LI>0.5 µL   dNTPs (10 mM)</LI><LI>1.375 µL     dH<SUB>2</SUB>O</LI>Pick a colony with a sterile tip and set the tip in 10 µL dH<SUB>2</SUB>O in a PCR tube for 10 minutes. Pipet up and down to resuspend the cell material and then add the reagent, DNA, and buffer mixture. While picking the colony, some cells should also be transferred to a new plate with the same antibiotic so that if the screen results are positive, the colony may be identified and grown up.
20 µL Total<P>PCR program</P><OL><LI>98º C      30 sec</LI><LI>98º C      10 sec</LI><LI>55º C      30 sec    </LI><LI>72º C      1 min / kb    Repeat Steps 2-4 29x (30x total)   </LI><LI>72º C      5 min</LI><LI>4º C       Hold </LI></OL></DIV><P class="menu_head">M9 Minimal Media</P><DIV class="menu_body"><P>Materials</P><UL><LI>1x <A href="http://openwetware.org/wiki/M9_salts">M9 salts</A></LI><LI>2 mM MgSO<SUB>4</SUB></LI><LI>0.1 mM CaCl<SUB>2</SUB></LI><LI>0.4% carbon source (eg. glucose)</LI><LI>Dissolve in sterile dH<SUB>2</SUB>0</LI><DIV id="sitemapbox" class="floatboxwide"><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UC_Davis/Protocols">http://2013.igem.org/Team:UC_Davis/Protocols</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Davis/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Davis/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Davis/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Davis/Protocols&amp;oldid=322230" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>