"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Davis/Assembly
From 2013.igem.org
Home
Team
Overview
Bios
Contact Us
Official Team Profile
Project
Overview
Modeling
Assembly
Parts
Safety
Data
Testing Constructs
Anderson Promoters
Anderson II
Notebook
Notebook
Protocols
Gallery
Attributions
Human Practices
Overview
Outreach
The Depot
Assembly
We used three different methods of assembly over the course of this project, the detailed methods and materials of which may be found on the Protocols page. Below is an overview of the different assembly methods used.
Golden Gate Assembly
Golden Gate Assembly involves amplifying out the DNA sequence of interest with forward and reverse primers that add to both ends a BsaI restriction site and a 4 bp overhang. The reverse 4 bp overhang is designed to be complimentary to the forward 4 bp overhang of the intended downstream sequence. Likewise, the forward 4 bp overhang is designed to be complimentary to the reverse 4 bp overhang of the intended upstream sequence. After Golden Gate PCR amplification with the appropriate primers, the parts can be assembled in a one-pot reaction involving the BsaI restriction enzyme, T4 DNA ligase, T4 DNA ligase buffer, and BSA.
The image included here illustrates the Golden Gate Assembly approach, where the 'nnnn' sequences indicate the 3' end of the upstream part and the 5' beginning of the downstream part, and the '1234' sequences indicate the arbitrary 4 bp overhangs that will be added through Golden Gate amplification.
The paper Generation of Families of Construct Variants Using Golden Gate Shuffling (Engler C., Marillonnet S., Methods in Molecular Biology Volume 729, 2011, pp 167-181) provides a detailed description of the Golden Gate assembly approach and its applications.
Splice Overhang Extension (SOE)
The SOE approach to assembly involves PCR amplification of the sequences of interest using primers that will add to both ends sequences of about 20 bp in length that are complimentary to the intended upstream and downstream sequences. After SOE amplification, a second round of PCR amplification is carried out in which the amplified sequences act as their own primers at the assembly points. The flanking primers for the assembled sequence may be used to increase the yield of PCR product. No restriction enzymes are required. This process may be repeated until all the parts in the full construct are assembled. If desired, the flanking primers at the beginning and terminus of the full construct may be designed so as to include restriction sites. After the final round of SOE assembly by PCR, the full construct can be digested and inserted into the BioBrick plasmid of choice. It is, however, also possible to assemble through SOE PCR an insert and a vector.
The paper Gene Splicing by Overlap Extension: Tailor-Made Genes Using the Polymerase Chain Reaction (Horton M. et al, BioTechniques, Vol. 54, No. 3, March 2013, pp. 129–133) provides a thorough explanation of the SOE assembly approach and its advantages.
Standard Assembly
Standard Assembly is a widely used and well-documented method of assembly involving the use of four restriction sites corresponding to the restriction enzymes EcoRI, XhoI, SpeI, and PstI. One key point is that after digestion, the XhoI and SpeI 'sticky ends' are capable of annealing to each other, forming a scar. A part may thus be cut out of its vector through digestion with XhoI and PstI, for example, isolated via gel extraction and purification, and ligated to a second plasmid that has been previously digested with SpeI and PstI. The SpeI and XhoI 'sticky ends' will anneal, as well as the 'sticky ends' resulting from digestion with PstI, effectively placing the part downstream of whatever part may have been present in the second plasmid. After ligation, the product can then be transformed into competent cells and selected for with the antibiotic corresponding to the resistance provided by the second plasmid.
Alternately, the insert may be digested with EcoRI and SpeI, while the vector is digested with EcoRI and XhoI, so that through ligation the insert is placed upstream of whatever part may have been present in the vector. The image below illustrates this method.
The Registry of Standard Biological Parts provides many resources pertaining to the use of Standard Assembly [RFC 10] , as well as other assembly techniques.
Retrieved from " http://2013.igem.org/Team:UC_Davis/Assembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
