<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Davis_Data"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UC_Davis/Data">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Davis/Data&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Davis/Data&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Davis/Data&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Davis/Data" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Davis/Data</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><UL class="nav"><LI class="dropdown leftbookend"><A href="/Team:UC_Davis" style="font-size:13px;">Home</A></LI><LI class="dropdown leftbookend rightbookend"><A href="https://2013.igem.org/Team:UC_Davis/Team_Overview" style="font-size:13px;">Team</A></LI><LI><A href="https://2013.igem.org/Team:UC_Davis/Team_Overview">Overview</A></LI><LI><A href="https://2013.igem.org/Team:UC_Davis/Bios">Bios</A></LI><LI><A href="/Team:UC_Davis/Contact">Contact Us</A></LI><LI><A href="https://igem.org/Team.cgi?id=1212">Official Team Profile</A></LI></UL><LI class="dropdown leftbookend rightbookend"><A href="https://2013.igem.org/Team:UC_Davis/Project_Overview" style="font-size:13px;">Project</A></LI><LI><A href="https://2013.igem.org/Team:UC_Davis/Project_Overview">Overview</A></LI><LI><A href="/Team:UC_Davis/Modeling">Modeling</A></LI><LI><A href="/Team:UC_Davis/Assembly">Assembly</A></LI><LI><A href="/Team:UC_Davis/Parts">Parts</A></LI><LI><A href="/Team:UC_Davis/Safety">Safety</A></LI><LI class="dropdown leftbookend rightbookend"><A href="/Team:UC_Davis/Data" style="font-size:13px;">Data</A></LI><LI><A href="/Team:UC_Davis/Data">Testing Constructs</A></LI><LI><A href="/Team:UC_Davis/AndersonPromoters">Anderson Promoters</A></LI><LI><A href="/Team:UC_Davis/AndersonPromoters2">Anderson II</A></LI><LI class="dropdown leftbookend rightbookend"><A href="/Team:UC_Davis/Notebook_Overview" style="font-size:13px;">Notebook</A></LI><LI><A href="/Team:UC_Davis/Notebook">Notebook</A></LI><LI><A href="/Team:UC_Davis/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:UC_Davis/Gallery">Gallery</A></LI><LI><A href="https://2013.igem.org/Team:UC_Davis/Attributions">Attributions</A></LI><LI class="dropdown rightbookend"><A href="/Team:UC_Davis/HumanPracticesOverview" style="font-size:13px;">Human Practices</A></LI><LI><A href="/Team:UC_Davis/HumanPracticesOverview">Overview</A></LI><LI><A href="/Team:UC_Davis/Outreach">Outreach</A></LI><LI><A href="/Team:UC_Davis/Database">The Depot</A></LI></P><DIV class="floatbox"><TABLE class="showbox"><TBODY><TR><TD><DIV><A href="https://2013.igem.org/Team:UC_Davis/Data"><H3>Testing Constructs</H3></A><P>Check out our initial experiments with our testing constructs that served as a proof of concept for RiboTAL function.
                     </P></DIV></TD><TD><A href="https://2013.igem.org/Team:UC_Davis/AndersonPromoters"><H3>Anderson Promoters</H3></A><P>Find out how we controlled the Anderson family of promoters through induction.
                        Also, see the secondary data page, <A href="https://2013.igem.org/Team:UC_Davis/AndersonPromoters2">here</A>. 
                     </P></TD></TR></TBODY></TABLE></DIV><DIV class="floatbox2"><H2>Quick Links</H2><H2><UL><LI><A href="#graph1">RiboTALe Activity</A></LI><LI><A href="#graph2">TALe Tunability</A></LI><LI><A href="#graph3">Riboswitch Activation</A></LI><LI><A href="#widget">RiboTALe KO3D</A></LI></UL></H2><H1 id="studies">Proof of Concept: Our Testing Construct</H1>To characterize the behavior of a RiboTALe device, we acquired cells containing the sequences for TAL repressors from the Segal Lab and Tagkopoulos Lab at UC Davis, with which we have worked closely. We placed the TAL repressors downstream of theophylline-responsive riboswitches, the sequences of which were taken from the studies <A href="http://www.ncbi.nlm.nih.gov/pubmed/19033367">A flow cytometry-based screen for synthetic riboswitches</A><A href="#ref"> [1]</A>, and <A href="http://aem.asm.org/content/76/23/7881.abstract">Synthetic Riboswitches That Induce Gene Expression in Diverse Bacterial Species</A><A href="#ref"> [2]</A>. The riboswitch-TALe sequences were placed under the regulation of a pBAD promoter.We inserted previously engineered TALe binding sites corresponding to the TAL repressors used in our characterization experiments upstream of a reporter, GFP. This target sequence was placed under the regulation of a pTET promoter.We tested our construct by subjecting the pBAD promoter, the theophylline riboswitch, and the pTET promoter to a range of induction levels with arabinose, theophylline, and aTc, respectively. It was expected that at low levels of arabinose and theophylline, but at high levels of aTc, GFP expression would be maximal due to the very low production of TAL repressor protein. On the other hand, at high levels of arabinose and theophylline it was expected that fluorescence levels would be greatly reduced due the higher rate of TAL repressor production. We also expected to see many instances of neither total GFP expression or total GFP repression, depending on the relative states of induction of the pBAD promoter, the theophylline riboswitch, and the pTET promoter.</DIV><DIV class="floatboxwide"><H1 id="graph1">Translation is modulated by theophylline concentrations <A href="#top" class="to_top">^back to top</A></H1>We subjected <A href="http://parts.igem.org/Part:BBa_K1212015">our testing construct</A> to the induction condition of 100 ng/mL aTc, which would result in constitutive and maximal expression of GFP given no repression. We also subjected the construct to the induction condition of 0.1% arabinose, which would produce a nominal level of RiboTALe transcript. We varied only the concentration of theophylline, over a range of 0 mM to 10 mM. Thus, difference in fluorescence between induction conditions would be due only to the RiboTALe repression activity.  We measured the fluorescence of our construct in E. Coli strain MG1655Z1 over a course of 9-10 hours using the Tecan Infinite 200Pro microplate reader. Please refer to the <A href="https://2013.igem.org/Team:UC_Davis/Protocols">Protocols</A> page for details on our culture preparation and Tecan testing parameters. <P id="itworks">The image above illustrates that GFP fluorescence is inversely related to theophylline concentrations, indicating that the <A href="http://parts.igem.org/Part:BBa_K1212007">TAL repressor</A> is in fact being translated at rates corresponding to the theophylline induction levels, and effectively binding to its target site. At maximal theophylline concentrations, the expression of GFP is reduced 2.6 fold.
</P><P>Next, we investigated what difference in system response we could achieve by altering the binding affinity of the TAL repressor protein.
</P></DIV><DIV class="floatboxwide"><H1 id="graph2">Binding affinities of the TAL repressors provide tunability<A href="#top" class="to_top">^back to top</A></H1>The <A href="http://parts.igem.org/Part:BBa_K1212007">TAL repressor protein</A> expressed by our <A href="http://parts.igem.org/Part:BBa_K1212015">RiboTALe device</A> in our initial experiment has a dissociation constant K<SUB>D</SUB> 1.3 ± .03 nM. We compared the activity of this RiboTALe to one under the control of the same <A href="http://parts.igem.org/Part:BBa_K1212001">theophylline riboswitch</A>, but that expressed a <A href="http://parts.igem.org/Part:BBa_K1212004">TAL repressor</A> with K<SUB>D</SUB> 240 ± 40 nM. We subjected both RiboTALes to the induction condition of 100 ng/mL aTc, which would result in constitutive and maximal expression of GFP given no repression. At the same time, we subjected both RiboTALes to the following conditions:<TABLE class="black"><TBODY><TR><TH>Arabinose Concentration (%)</TH><TH>Theophylline Concentration (mM)</TH><TH>Expected Result</TH></TR><TR><TD>0.0</TD><TD>0.0</TD><TD>Full GFP expression due to a lack of RiboTALe transcript and thus TAL repressor</TD></TR><TR><TD>1.0</TD><TD>0.0</TD><TD>Full GFP expression due to a lack of riboswitch inducer, and thus translation of the TAL protein. Decreased GFP expression could be attributed to riboswitch leakiness</TD></TR><TR><TD>1.0</TD><TD>10.0</TD><TD>Full GFP repression, due the nominal expression of the RiboTALe transcript and the TAL repressor</TD></TR></TBODY></TABLE>The image above displays the peak fluorescence of two RiboTALe constructs, <A href="http://parts.igem.org/Part:BBa_K1212014">one</A> expressing <A href="http://parts.igem.org/Part:BBa_K1212004">TALe 1</A> and the <A href="http://parts.igem.org/Part:BBa_K1212015">other</A> expressing <A href="http://parts.igem.org/Part:BBa_K1212007">TALe 8</A>, under different induction conditions for arabinose and theophylline. Both RiboTALes exhibit the expected behavior pattern given the induction conditions, but at consistently different levels of fluorescence. We have attributed this to the difference in binding affinities of the two TAL repressors to their respective binding sites.This variable, if well characterized for different TAL repressors, will provide a powerful means to control the tunability of these devices.It is similarly interesting to note that under conditions of 1% arabinose, but no theophylline, there was clearly some reduction in fluorescence. We concluded that the <A href="http://parts.igem.org/Part:BBa_K1212001">riboswitch</A> we used in this experiment had some degree of leakiness. We next investigated the possibility of altering riboswitch leakiness as another means to increase the tunability of our RiboTALe devices.</DIV><DIV class="floatboxwide"><H1 id="graph3">Riboswitch leakiness modulates RiboTALe activity<A href="#top" class="to_top">^back to top</A></H1>
We proceeded to investigate the differences achievable in RiboTALe system response by varying the riboswitch controlling the translation of the TAL repressor. To this end we tested two RiboTALe devices, both of which expressed <A href="http://parts.igem.org/Part:BBa_K1212007">TALe 8</A>. <A href="http://parts.igem.org/Part:BBa_K1212015">One</A> of these RiboTALes was under the control of theophylline riboswitch <A href="http://parts.igem.org/Part:BBa_K1212001">Clone E</A> and the <A href="http://parts.igem.org/Part:BBa_K1212012">other</A> was under the control of theophylline riboswitch <A href="http://parts.igem.org/Part:BBa_K1212000">Clone 8.1*</A>.  We subjected both RiboTALes to the induction condition of 100 ng/uL aTC, which would result in constitutive and maximal expression of GFP given no repression. At the same time, we subjected both RiboTALes to the following conditions:
<TABLE class="black"><TBODY><TR><TH>Arabinose Concentration (%)</TH><TH>Theophylline Concentration (mM)</TH><TH>Expected Result</TH></TR><TR><TD>0.0</TD><TD>0.0</TD><TD>Full GFP expression due to a lack of RiboTALe transcript and thus TAL repressor</TD></TR><TR><TD>1.0</TD><TD>0.0</TD><TD>Full GFP expression due to a lack of riboswitch inducer, and thus translation of the TAL protein. Decreased GFP expression could be attributed to riboswitch leakiness</TD></TR><TR><TD>1.0</TD><TD>10.0</TD><TD>Full GFP repression, due the nominal expression of the RiboTALe transcript and the TAL repressor</TD></TR></TBODY></TABLE>
The image above displays the fluorescence results for the two RiboTALe devices tested in this experiment. According to the literature both riboswitches have similar reported fold activation ratios<A href="#ref">[1,2]</A>. But it is clear that the two RiboTALe devices, differing only in the riboswitch controlling the translation of the TAL repressor, exhibit consistently different behavior. The data show that at 1% arabinose (the inducer for the RiboTALe transcript), but in the absence of theophylline, the RiboTALe under control of riboswitch <A href="http://parts.igem.org/Part:BBa_K1212001">Clone E</A> is active. Under identical induction conditions, the RiboTALe under riboswitch <A href="http://parts.igem.org/Part:BBa_K1212000">Clone 8.1*</A> exhibits no repression activity. The fluorescence measured was in fact <I>higher</I> than the baseline for reasons not understood. From the data we conclude that <A href="http://parts.igem.org/Part:BBa_K1212001">Clone E</A> is leakier and yet stronger than <A href="http://parts.igem.org/Part:BBa_K1212000">Clone 8.1*</A>, generating a 3.68 fold reduction in fluorescence as opposed to a 2.42 fold reduction. These data indicate that differences riboswitch leakiness and strength do impact RiboTALe system behavior, and can be engineered into RiboTALe designs as sources of tunability. 

</DIV><DIV class="floatboxwide"><H1 id="widget">3D RiboTALe Data Plot<A href="#top" class="to_top">^back to top</A></H1><P>Here is a graphical representation of some of our RiboTALe characterization data. The graph can be toggled between 2D and 3D plot modes. The data sets plotted can also be turned on or off through the use of the corresponding buttons in the upper right of the graph. Feel free to click the navigation buttons or drag the 3D graph in order to get a better view.
</P><DIV id="mutantwidget" class="floatbox"><SPAN class="dataMax">90000</SPAN><H3>100 ng/mL aTc</H3><SPAN class="xdata">0, 1, 5, 10</SPAN><SPAN class="ydata">81896.13, 78666.88, 74182.52, 31385.93</SPAN><SPAN class="xdata_3d">0, .01, .1, .25, .5, 1</SPAN><SPAN class="ydata_3d">0, 1, 2, 5, 10</SPAN><SPAN class="zdata_3d">
81399.99914, 82312.01989, 81896.13437, 70965.09262, 71753.41851, 52082.65688,
82598.77679, 90191.2318, 78666.87521, 59658.53566, 53189.6567, 43608.73416,
89341.08341, 88566.62349, 89727.21455, 58754.65109, 55356.67872, 59500.96347,
71372.62047, 72710.06759, 74182.52136, 39969.45281, 41701.09012, 58907.74009,
22868.50882, 26697.24765, 31385.92672, 12146.13773, 13292.34931, 29857.02244

</SPAN><H3>0 ng/mL aTc</H3><SPAN class="xdata">0, 1, 5, 10</SPAN><SPAN class="ydata">1629.559472, 1700.829702, 1701.217487, 1116.65006</SPAN><SPAN class="xdata_3d">0, .01, .1, .25, .5, 1</SPAN><SPAN class="ydata_3d">0, 1, 2, 5, 10</SPAN><SPAN class="zdata_3d">
1629.559472,1634.441888,1565.606412,1568.030923,1549.936772,1550.608345,
1700.829702,1756.172098,1716.475113,1714.999974,1687.625775,1673.997409,
1792.94241,1759.305223,1757.164411,1746.514631,1719.24679,1750.824618,
1701.217487,1990.586906,2022.536824,2028.438816,2074.652724,2075.872599,
1116.65006,1816.239221,1806.522633,1788.908717,1827.918679,1805.555476
</SPAN></DIV><DIV class="playme"><P id="play">Play With Me</P></DIV></DIV><DIV class="floatbox"><TABLE class="showbox"><TBODY><TR><TD><A href="https://2013.igem.org/Team:UC_Davis/Project_Overview"><H3>Project Overview</H3></A><P>Learn about how we combine riboswitches and TALs into robust orthogonal mechanisms for     inducible repression. 
                     </P></TD><TD><A href="https://2013.igem.org/Team:UC_Davis/Data"><H3>Results</H3></A><P>Check out the cool results of our experiments with RiboTALs. 
                     </P></TD></TR><TR><TD><A href="https://2013.igem.org/Team:UC_Davis/HumanPracticesOverview"><H3>Human Practices</H3></A><P>Take a look at how we promote sharing in iGEM through The Depot, an open BioBrick characterization database.<A href="http://dilbert.cs.ucdavis.edu/Depot" class="bold">Visit the Depot!</A></P></TD><TD><A href="https://2013.igem.org/Team:UC_Davis/Criteria"><H3>Judging Criteria</H3></A><P>Here's the criteria that we met for this year's team. 
                     </P></TD></TR></TBODY></TABLE></DIV><DIV class="floatboxwide"><H3>References</H3><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/?term=A+flow+cytometry-based+screen+for+synthetic+riboswitches">[1]	S. A. Lynch and J. P. Gallivan, &quot;A flow cytometry-based screen for synthetic riboswitches,&quot; Nucleic Acids Research, vol. 37, pp. 184-192, Jan 2009.</A><A href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Synthetic+Riboswitches+That+Induce+Gene+Expression+in+Diverse+Bacterial+Species">[2]	S. Topp, C. M. K. Reynoso, J. C. Seeliger, I. S. Goldlust, S. K. Desai, D. Murat, et al., &quot;Synthetic Riboswitches That Induce Gene Expression in Diverse Bacterial Species (vol 76, pg 7881, 2010),&quot; Applied and Environmental Microbiology, vol. 77, pp. 2199-2199, Mar 2011.</A></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UC_Davis/Data">http://2013.igem.org/Team:UC_Davis/Data</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Davis/Data" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Davis/Data" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Davis/Data&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Davis/Data&amp;oldid=357504" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>