<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TokyoTech_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TokyoTech/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Coli Sapiens</TITLE></P><NAV class="w3-sidebar w3-red w3-collapse w3-top w3-large w3-padding" style="z-index:3;width:300px;font-weight:bold;" id="mySidebar"><A href="javascript:void(0)" onclick="w3_close()" class="w3-button w3-hide-large w3-display-topleft" style="width:100%;font-size:20px">Close Menu</A><DIV style="padding-top: 15px; padding-left: 25px; id=" wrap=""><DIV id="contents" style="z-index: 10"><UL class="click_menu_checkbox vertm_down"><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px;"><LABEL for="vmcb-a"><A href="https://2017.igem.org/Team:TokyoTech">Home</A></LABEL></LI><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-b"><A href="https://2017.igem.org/Team:TokyoTech/Medal">Achievements</A></LABEL></LI><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-c"><A>Project</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Description" class="w3-bar-item w3-button w3-hover-white">Description</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Project/Basic_Parts" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Basic Parts</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Project/Composite_Parts" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Composite Parts</A></LI></UL><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-d"><A>Experiment</A></LABEL></LI><LI><LABEL for="vmcb-d1"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Overview" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white" style="text-align: center;">Overview</A></LABEL></LI><LI style="padding-bottom: 10px; padding-top: 5px"><LABEL for="vmcb-d2"><A style="text-align: center;">Bacteria to Human Cells ▼</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraI Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Improvement" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraI Improvement Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraR_Reporter_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraR Reporter Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Transcriptome_Analysis" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Transcriptome Analysis</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Chimeric_Transcription_Factor" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Chimeric  Transcription  Factor Assay</A></LI><LI style="padding-bottom: 3px"><LABEL for="vmcb-d3"><A style="text-align: center;">Human Cells to Bacteria ▼</A></LABEL></LI><LI><A href="https://2017.igem.org/Team:TokyoTech/Experiment/AHK4_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">AHK4 Assay</A></LI><LI><LABEL for="vmcb-d4"><A href="https://2017.igem.org/Team:TokyoTech/InterLab" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white" style="text-align: center;">InterLab</A></LABEL></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-e"><A href="https://2017.igem.org/Team:TokyoTech/Model">Modelling</A></LABEL></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-f"><A>Human Practices</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Overview</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP/Silver" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Silver</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP/Gold_Integrated" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Gold (Integrated)</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Demonstrate" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Demonstrate</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Collaborations" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Collaborations</A></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-g"><A>About us</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Team" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Team</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Attributions" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Attributions</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Sponsors" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Sponsors</A></LI></DIV></DIV></NAV><HEADER class="w3-container w3-top w3-hide-large w3-red w3-xlarge w3-padding"><SPAN style="padding-top: 5px">iGEM Tokyo Tech</SPAN></HEADER><DIV class="w3-main" style="margin-left:340px;margin-right:40px"><DIV class="w3-container" id="contact" style="margin-top:30px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px" align="center">Project Description</H1></DIV><DIV class="w3-container" id="contact" style="margin-top:30px"><H2 class="w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><SPAN>Contents</SPAN></H2><H3 style="text-indent: 1em;font-size: 16px"><A href="#intro">i. Introduction</A></H3><H3 style="text-indent: 1em;font-size: 16px"><A href="#goal">ii. Goal and Approach</A></H3><H3 style="text-indent: 1em;font-size: 16px"><A href="#mecha">iii. Mechanism</A></H3><H3 style="text-indent: 1em;font-size: 16px"><A href="#result">iv. Results</A></H3><H3 style="text-indent: 1em;font-size: 16px"><A href="#hp">v. Human Practices</A></H3></DIV><DIV class="w3-container" id="intro" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Introduction</B></H1><P style="font-size:16px;font-size: 16px; text-indent:1em">
    How can we define a human organism? Is it simply a group of human cells? It's said that in our body, there exist not only 3.0*10<SUP>13</SUP> human cells but also 3.8*10<SUP>13</SUP> bacteria. That means the mass of bacteria reaches 0.2 kg. In other words, humans are not solely composed of human cells. However, in iGEM community, it's been a standard to use single organism in project and it's not an overstatement that most teams don't take it into account that in a real world, multiple kinds of organisms co-exist and the ecosystem is sustained by their mutual dependence. Therefore, to target &quot;true human organism&quot;, it's necessary to establish the system that human cells and bacteria co-exist under in vitro conditions. Therefore, we decided to establish co-culture system between human cells and bacteria.</P><P style="font-size:16px;font-size: 16px; text-indent:1em">
    If we can establish a co-culture system, we can find a way to achieve population balance to sustain the co-existence and apply for a medical field like a cancer treatment. If you can co-exist with photosynthetic bacteria or nitrogen fixing bacteria, you can photosynthesize or produce protein from air. If you could co-exist with bacteria, you could be a super human. We named this new type of human 'Coli Sapiens.' 
    </P></DIV><DIV class="w3-container" id="goal" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Goal and Approach</B></H1><P style="font-size:16px;text-indent: 1em; padding-bottom: 15px">Our original goals are as follows:</P><CENTER><DIV style="padding: 10px; margin-bottom: 10px; border: 1px dotted #333333;width: 90%; border-radius: 10px;"><H3 style="text-align: center; padding-bottom: 15px">
Establishing an artificial cross-kingdom communication system between human cells and bacteria.</H3><P style="font-size:16px;font-size: 16px; text-indent:1em;text-align: center;">
    To achieve the first goal, we needed a new cell-to-cell communication system because native and direct communication systems between human cells and bacteria were little known. Thus, we decided to integrate signal transduction system among three kingdoms. 
</P></DIV><DIV style="padding: 10px; margin-bottom: 10px; border: 1px dotted #333333;width: 90%; border-radius: 10px"><H3 style="text-align: center; padding-bottom: 15px">
Creating a co-culture model using the cross-kingdom communication and designing ‘Coli Sapiens,’ a new type of human strengthened by bacteria</H3><P style="font-size:16px;font-size: 16px; text-indent:1em;text-align: center;">
    To achieve the second goal, we chose the essential parts in a complex co-culture system between bacteria and human cells. The reason why co-existence between them has not been developed under <SPAN style="font-style: italic;">in vitro</SPAN> conditions is that a growth rate of bacteria surpasses that of human cells. Thus, when we designed the mathematical model, we emphasized a population of bacteria as one of the biggest factors to establish a co-culture system.
</P></DIV></CENTER></DIV><DIV class="w3-container" id="mecha" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Mechanism</B></H1><P style="font-size:16px;font-size: 16px; text-indent:1em;padding-bottom: 15px">
      We established the following two systems.
      </P><CENTER><DIV style="padding: 10px; margin-bottom: 3px; border: 1px dotted #333333;width: 90%; border-radius: 10px"><H4 style="text-align: center">Signal transduction system from bacteria to humans</H4><H5 style="text-align: center">~ Integration of systems derived from bacteria and humans ~</H5><P style="font-size:16px;font-size: 16px; text-indent:1em;padding-top: 15px">
      In this signal transduction system, the transcription level is controlled by integrating quorum sensing (bacterial cell-to-cell communication) and NF-kB, transcription factor in mammalian cell. We used this system the signal transduction from bacteria to human cells.
      </P><DIV class="w3-xxxlarge" style="padding-bottom: 3px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 1 Mechanism of signal transduction system from bacteria to human</FIGCAPTION></FIGURE></DIV><DIV onclick="obj=document.getElementById('menu1').style; obj.display=(obj.display=='none')?'block':'none';" style="text-align: center;" style="margin-bottom: 30px"><BUTTON class="w3-button w3-red w3-padding-large w3-hover-black" style="font-size: 20px; margin-bottom: 30px"><A style="cursor:pointer;">▼ Click here for more detail</A></BUTTON></DIV><DIV id="menu1" style="display:none;clear:both;padding-top: 20px;text-align: center;"><P style="font-size: 16px; "><U>Step1</U>: We transduced <SPAN style="font-style: italic">traI</SPAN> which codes C8 synthetase and as <SPAN style="font-style: italic">E. coli</SPAN> grow, C8 is synthesized and secreted. In human cells, the following two genes are transduced. In the fusional protein, TraR works as a DNA-binding domain and a part of RelA works as a chimeric transcription factor.
    </P><P style="font-size: 16px; "><U>Step2</U>: After the complex receives C8, the chimeric transcription factor structurally changes and the active dimer is formed.
    </P><P style="font-size: 16px; "><U>Step3</U>: TraR in the active factor binds tra box located upstream of the target gene.
    </P><P style="font-size: 16px; "><U>Step4</U>: RelA Trans Activation Domain (TAD) activates CMV minimal promoter and the expression of the downstream gene is induced. As a result, iP is produced and diffuses out of the cell.
    </P></DIV></DIV><DIV style="padding: 10px; margin-bottom: 10px; border: 1px dotted #333333;width: 90%;border-radius: 10px"><H4 style="text-align: center">Signal transduction system from human cells to bacteria</H4><H5>~ Integration of systems derived from bacteria and plants ~</H5><P style="font-size:16px;font-size: 16px; text-indent:1em;padding-top: 15px">
      In this signal transduction system, the transcription level is controlled by integrating signal transduction systems derived from bacteria and plants. We used this system the signal transduction from human cells to bacteria.
      </P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 2 Mechanism of signal transduction system from human cells to bacteria</FIGCAPTION></FIGURE></DIV><DIV onclick="obj=document.getElementById('menu2').style; obj.display=(obj.display=='none')?'block':'none';" style="text-align: center;" style="margin-bottom: 30px"><BUTTON class="w3-button w3-red w3-padding-large w3-hover-black" style="font-size: 20px; margin-bottom: 30px"><A style="cursor:pointer;">▼ Click here for more detail</A></BUTTON></DIV><DIV id="menu2" style="display:none;clear:both;padding-top: 20px;text-align: center;"><P style="font-size: 16px; "><U>Step1</U>: AHK4 binding to iP performs an autophosphorylation reaction, transferring a phosphoryl group from ATP to a histidine residue of Histidine kinase (HK) domain. AHK4 transfers the phosphoryl group to its own internal receiver domain.
        </P><P style="font-size: 16px; "><U>Step2</U>: The phosphoryl group of AHK4 is transferred the histidine-containing phosphotransmitter, RcsD.
        </P><P style="font-size: 16px; "><U>Step3</U>: The phosphoryl group of RcsD is transferred the RcsB and RcsB is activated. Activated RcsB and another RR, RcsA form a hetero dimer and bind to cps operon promoter (which controls the production of polysaccharides). This series of phosphoryl group transmission is called the His-to-Asp phosphorelay (Read <A href="https://2017.igem.org/Team:TokyoTech/Experiment/AHK4_Assay">AHK4 Assay</A> page).
        </P><P style="font-size: 16px; "><U>Step4</U>: The transcription of the gene downstream cps operon promoter is activated. In our genetic circuits, the gene downstream cps operon promoter is <SPAN style="font-style: italic;">mazF</SPAN>. <SPAN style="font-style: italic;">mazF</SPAN> is the gene of  a toxin-antitoxin system. Please read <A href="https://2017.igem.org/Team:TokyoTech/Model">Modelling</A> page about the details of toxin-antitoxin system and <SPAN style="font-style: italic;">mazF</SPAN>.
        </P></DIV></DIV><DIV style="padding: 10px; margin-bottom: 10px; border: 1px dotted #333333;width: 90%;border-radius: 10px"><H4 style="text-align: center">Co-culture system</H4><P style="font-size:16px;font-size: 16px; text-indent:1em;padding-top: 15px">
      We conducted experiments to validate the two systems above. Based on the results, we virtually integrated the two systems and conducted population change simulations.
      </P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 3 Mechanism of co-culture system</FIGCAPTION></FIGURE></DIV><DIV style="text-align: center;" style="margin-bottom: 30px"><BUTTON class="w3-button w3-red w3-padding-large w3-hover-black" style="font-size: 20px; margin-bottom: 30px"><A href="https://2017.igem.org/Team:TokyoTech/Model" target="_blank">Go to Modelling Page</A></BUTTON></DIV></DIV></CENTER></DIV><DIV class="w3-container" id="result" style="margin-top:20px"><H2 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Results</B></H2><H3 style="text-align: center; margin-top:40px;margin-bottom:20px"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Improvement">TraI Improvement</A></H3><P style="font-size:16px;font-size: 16px; text-indent:1em">At an early stage of our project, we simulated the whole co-culture system using parameters from the C8 production rate of <SPAN style="font-style: italic">E. coli</SPAN>, the iP production rate of human cells and growth inhibition rate of <SPAN style="font-style: italic">mazF</SPAN>. The simulation showed that the C8 production rate is not enough to induce the iP production and as a result, <SPAN style="font-style: italic">E. coli</SPAN> overgrow.
    </P><P style="font-size:16px;font-size: 16px; text-indent:1em">To increase the C8 production rate, we improved the previous genetic circuits in two ways.
    </P><UL style="padding-left: 2em"><LI style="font-size: 16px">- 
        Introducing various point mutations into CDS of the <SPAN style="font-style: italic">traI</SPAN> gene and finding a strain whose C8 production rate increases
      </LI><LI style="font-size: 16px">- 
        Adding SAM (one of the C8 materials) to culture medium and promoting the C8 production
      </LI></UL><P style="font-size:16px;font-size: 16px; text-indent:1em">As a result of the improvement, the concentration of C8 which <SPAN style="font-style: italic">E. coli</SPAN> produce increased by about 3-fold and it has been possible to induce iP synthesis in human cells from an early stage of <SPAN style="font-style: italic">E. coli</SPAN>'s growth.
    </P><CENTER><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 4 Improvement of C8 production by the K34G mutant (37℃ culture)</FIGCAPTION></FIGURE></CENTER><H3 style="text-align: center; margin-top:40px;margin-bottom:20px"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Chimeric_Transcription_Factor">Chimeric Transcription Factor</A></H3><P style="font-size:16px;font-size: 16px; text-indent:1em"> As for human cells' constructs, we synthesized chimeric transcription factor and iP synthetase genes. In the assay, first, we transduced the constructs. Then, we cultured the cells in which the constructs are successfully transduced and added C8 from <SPAN style="font-style: italic">E. coli</SPAN>. After the addition, we checked the transcription of <SPAN style="font-style: italic">atIPT4</SPAN> and <SPAN style="font-style: italic">log1</SPAN> (part of iP synthetase genes) using transcriptome analysis. From this result, we concluded that human cells received C8 from bacteria and successfully produced iP.
    </P><CENTER><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 5 Result of the qualitative experiment</FIGCAPTION></FIGURE></DIV><P style="font-size: 16px">
  The term “Cont” means the control cells that are not electroporated, while “EP” means the electroporated cells. The concentrations of added C8 are indicated below the bars. 
 </P></CENTER><H3 style="text-align: center; margin-top:40px;margin-bottom:20px"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/AHK4_Assay">AHK4 Assay</A></H3><P style="font-size:16px;font-size: 16px; text-indent:1em">We transduced <SPAN style="font-style: italic">ahk4</SPAN> into <SPAN style="font-style: italic">E. coli</SPAN> (KMI002 strain) and cultured them. Then, we added iP and after AHK4 received iP, cps promoter was activated and downstream <SPAN style="font-style: italic">lacZ</SPAN> is expressed. (<SPAN style="font-style: italic">lacZ</SPAN> expression was confirmed by blue-white screening.) In conclusion, it turned out that AHK4 can receive iP and induce the gene expression of the downstream genes, which means in a larger scale, <SPAN style="font-style: italic">E. coli</SPAN> can receive growth inhibition factors from human cells and inhibit the own growth. 
    </P><CENTER><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 6  Result of the qualitative experiment</FIGCAPTION></FIGURE><P style="font-size:16px;font-size: 16px; text-indent:1em">Cells were grown at room temperature on LB agar plates with and without iP. β-galactosidase activity was monitored by X-gal. Photographs were taken after 25h incubation.</P></DIV></CENTER><H4 style="text-align: center;margin-top:40px"><A href="https://2017.igem.org/Team:TokyoTech/Model">Simulation</A></H4><P style="font-size:16px;font-size: 16px; text-indent:1em">We again simulated the whole co-culture system using the parameter from assay data. The simulation showed that human cells have potential to control the population of <SPAN style="font-style: italic">E. coli</SPAN>, and both population settle to an appropriate ratio. In Fig. 7, u means the number of human cells and f means the flow rate.

    </P><CENTER><DIV class="w3-xxxlarge" style="padding-bottom: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 7 Condition of co-existence </FIGCAPTION></FIGURE></DIV></CENTER></DIV><DIV class="w3-container" id="hp" style="margin-top:20px"><H2 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Human Practices</B></H2><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 8 Roadmap: How we integrated Human Practices and our experiment</FIGCAPTION></FIGURE></DIV><P style="font-size: 16px; text-indent: 1em;padding-bottom:20px">
        From our full year experience in iGEM, we realized the necessity of verifying from a different point of view. In other words, we realized that we researchers ourselves must also continuously reflect on the risks and costs &amp; benefits of the science we discover. In the workshop that we attended as our initial activity in iGEM, we learned from social scientists, the danger of grounding on the deficit model, which fixes on the idea that the general public is ignorant, and the importance of the two-way dialogue between society and researchers.
      </P><DIV style="text-align: center;" style="margin-bottom: 30px;margin-top:20px"><BUTTON class="w3-button w3-red w3-padding-large w3-hover-black" style="font-size: 20px; margin-bottom: 30px;margin-top:20px"><A href="https://2017.igem.org/Team:TokyoTech/HP" target="_blank">Go to Human Practices Page</A></BUTTON></DIV></DIV></DIV></DIV><DIV class="w3-light-grey w3-container w3-padding-32" style="margin-top:75px;padding-right:58px"><P class="w3-right"><A href="http://96haji.me/" title="W3.CSS" target="_blank" class="w3-hover-opacity">Hajime Fujita with W3.CSS: All Rights Reserved</A></P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>