<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TokyoTech_Experiment_TraI_Improvement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TokyoTech/Experiment/TraI Improvement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Coli Sapiens</TITLE></P><NAV class="w3-sidebar w3-red w3-collapse w3-top w3-large w3-padding" style="z-index:3;width:300px;font-weight:bold;" id="mySidebar"><A href="javascript:void(0)" onclick="w3_close()" class="w3-button w3-hide-large w3-display-topleft" style="width:100%;font-size:20px">Close Menu</A><DIV style="padding-top: 15px; padding-left: 25px; id=" wrap=""><DIV id="contents" style="z-index: 10"><UL class="click_menu_checkbox vertm_down"><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px;"><LABEL for="vmcb-a"><A href="https://2017.igem.org/Team:TokyoTech">Home</A></LABEL></LI><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-b"><A href="https://2017.igem.org/Team:TokyoTech/Medal">Achievements</A></LABEL></LI><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-c"><A>Project</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Description" class="w3-bar-item w3-button w3-hover-white">Description</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Project/Basic_Parts" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Basic Parts</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Project/Composite_Parts" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Composite Parts</A></LI></UL><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-d"><A>Experiment</A></LABEL></LI><LI><LABEL for="vmcb-d1"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Overview" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white" style="text-align: center;">Overview</A></LABEL></LI><LI style="padding-bottom: 10px; padding-top: 5px"><LABEL for="vmcb-d2"><A style="text-align: center;">Bacteria to Human Cells ▼</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraI Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Improvement" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraI Improvement Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraR_Reporter_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraR Reporter Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Transcriptome_Analysis" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Transcriptome Analysis</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Chimeric_Transcription_Factor" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Chimeric  Transcription  Factor Assay</A></LI><LI style="padding-bottom: 3px"><LABEL for="vmcb-d3"><A style="text-align: center;">Human Cells to Bacteria ▼</A></LABEL></LI><LI><A href="https://2017.igem.org/Team:TokyoTech/Experiment/AHK4_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">AHK4 Assay</A></LI><LI><LABEL for="vmcb-d4"><A href="https://2017.igem.org/Team:TokyoTech/InterLab" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white" style="text-align: center;">InterLab</A></LABEL></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-e"><A href="https://2017.igem.org/Team:TokyoTech/Model">Modelling</A></LABEL></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-f"><A>Human Practices</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Overview</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP/Silver" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Silver</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP/Gold_Integrated" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Gold (Integrated)</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Demonstrate" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Demonstrate</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Collaborations" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Collaborations</A></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-g"><A>About us</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Team" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Team</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Attributions" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Attributions</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Sponsors" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Sponsors</A></LI></DIV></DIV></NAV><HEADER class="w3-container w3-top w3-hide-large w3-red w3-xlarge w3-padding"><SPAN style="padding-top: 5px">iGEM Tokyo Tech</SPAN></HEADER><DIV class="w3-main" style="margin-left:340px;margin-right:40px"><DIV class="w3-container" id="contact" style="margin-top:30px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px" align="center">
TraI Improvement Assay</H1></DIV><DIV class="w3-container" id="overview" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Introduction</B></H1><P style="font-size: 16px; text-indent:1em">
In the previous wiki page (Read <A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Assay">TraI Assay</A> page), we describe that the productivity of C8 in <SPAN style="font-style: italic">E. coli</SPAN> depends on the culture temperatures. However, to complete our co-culture system, the current 3OC8HSL (hereafter C8) productivity at 37℃ was not enough to transmit the AHL signal to human cells, because human cells are usually grown at 37℃. Therefore, we tried to mutate the <SPAN style="font-style: italic">traI</SPAN> gene and increase the productivity of C8 at 37℃.</P></DIV><DIV class="w3-container" id="overview" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Summary of experiment</B></H1><P style="font-size: 16px; text-indent:1em">
TraI has not been characterized extensively, and thus, it is unclear what kind of mutations is appropriate for the above purpose. A preceding study describes that, in the case of LuxI, the amino acid substitution at the 34th and 63rd positions (both are substitutions from glutamate to glycine; E34G and E63G) increase the productivity of C6 (2). Since TraI has homology to LuxI over the entire amino acid sequences, we speculate that the same amino acid substitutions in TraI can increase the productivity of C8. Also, we here describe the modification of the culture conditions and the host strain choice for increased C8 production. </P></DIV><DIV class="w3-container" id="results" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Results</B></H1><P style="font-size: 16px; text-indent:1em">
When amino acid sequences of TraI and LuxI were aligned using the clustal W program (1), the E34 and E63 residues of LuxI were found to correspond to K34 and Q63 residues of TraR. According to this information, oligonucleotide primers to create TraI-K34G, TraI-Q63G, and TraI-K34G,Q63G mutants were designed. The primer sequences are shown in Fig. 1. The mutations were introduced to the pSB1C3-based traI plasmid using the inverse-PCR method, and successful introduction of the mutations were confirmed with Sanger sequencing.</P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 1 Sequences of the primers. Note that each primer set is divergent for inverse-PCR</FIGCAPTION></FIGURE><P style="font-size: 16px; text-indent:1em">
The sequences of <SPAN style="font-style: italic">traI</SPAN> mutants and wild-type are shown in Fig. 2. The plasmids we used are shown in Fig. 3 and 4. The Sender and the Reporter strains were prepared in the same way as described in the previous wiki page (Read<A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Assay">TraI Assay</A> page).<FIGURE><FIGCAPTION style="font-size: 16px">Fig. 2 Sequence of <SPAN style="font-style: italic">traI</SPAN> wild type gene and mutant</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 3 Structure of the plasmids used for creating the “Reporter” strain. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 4 Structure of the plasmids used for creating the “Sender” and “Mutated Sender” strains. One of these plasmids was used for preparing each transfromant</FIGCAPTION></FIGURE></P></DIV><P style="font-size: 16px; text-indent:1em">
In the experiments shown in this page, one additional modification was made in experimental conditions; 1 microM of SAM (S-adenosylmethionine; structure is shown in Fig. 5) was added to the culture of the Sender. Since C8 is synthesized from SAM and ACP (acyl carrier protein) through the action of TraI in bacterial cells (3), we expected that the addition of SAM may increase the productivity.	</P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 5 Chemical structure of SAM (S‐adenosylmethionine) </FIGCAPTION></FIGURE><P style="font-size: 16px; text-indent:1em">
The result of C8 production using the TraI wild-type and the mutants is shown in Fig. 6. &quot;W.T.&quot; means native <SPAN style="font-style: italic">traI</SPAN>.
The RFU value of the TraI(K34G)-expressing cells was about 3-fold higher than that of the TraI-expressing cells. <SPAN style="font-style: italic">E. coli</SPAN> introduced empty vector was used as Negative Control.
Other mutant did not show improvement of C8 production (data was not shown).
When these RFU values were converted to C8 concentrations using the calibration curve obtained in the reagent assay (Read <A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Assay">TraI Assay</A> page), they were calculated as 28 nM and 42 nM, respectively.<FIGURE><FIGCAPTION style="font-size: 16px">Fig. 6 Improvement of C8 production by the K34G mutant (37℃ culture)</FIGCAPTION></FIGURE></P><P style="font-size: 16px; text-indent:1em">
Also, the other modification was made concerning the host strain of the Sender. The preceding iGEM study has shown that the amount of AHL produced by the <SPAN style="font-style: italic">luxI</SPAN> gene highly depends on the host strain; depending on the used strains as the Sender, there was approximatly 100-fold difference in cell number for obtaining the same activation level of the lux promoter (4). Therefore, we here used the other strain, MG1655hapB, as the Sender.
Note that, in the experiments of the previous wiki page (Read <A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Assay">TraI Assay</A> page
), only the DH5α strain was used as a host. The MG1655hapB strain is a mutant of MG1655 (the representative wild-type K12 strain) and has higher membrane permeability for hydrophobic compounds compared to the parent.
As a result (Fig. 7), we found that amount of C8 produced is dependent on strains. The RFU value in the MG1655hapB strain was about 3-fold higher than that in the DH5α strain. <SPAN style="font-style: italic">E. coli</SPAN> introduced empty vector was used as Negative Control.
When these RFU values were converted to C8 concentrations using the calibration curve obtained in the reagent assay (Read <A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Assay">TraI Assay</A> page), they were calculated as 28 and 37 nM, respectively.
	</P><P style="font-size: 16px; text-indent:1em">
Strain dependence of AHL production
We found that amount of C8 production is depend on <SPAN style="font-style: italic">E. coli</SPAN>’s strain. RFU is 2-fold higher than DH5α. 
Calculated from the graph obtained in the reagent assay, C8 concentration of DH5α culture was 28 nM and MG1655hapB culture was 36 nM.
    	<FIGURE><FIGCAPTION style="font-size: 16px">Fig. 7 Strain dependence of C8 production</FIGCAPTION></FIGURE></P></DIV><DIV class="w3-container" id="results" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Discussion</B></H1><P style="font-size: 16px; text-indent:1em">
In the previous study, it was considered that the E34G mutation of LuxI most likely enhances the interactions between the enzyme and the ACP substrate. Therefore, we thought that K34G mutation of TraI also has the same effect.  
It was also showed that the MG1655hapB strain produced more C8 than the DH5α strain. We speculate that the difference in permeability of hydrophobic compounds through the cell membrane is the main reason for this result.
Taken together, we conclude that increasing the productivity of C8 at 37℃ was successful. Notably, generation and functional identification the mutant traI (TraI(K34G)) meet the medal criteria of ”parts improvement”, because the wild-type <SPAN style="font-style: italic">traI</SPAN> parts was registered in iGEM parts collection earlier (Read <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K553001">BBa_K553001</A> page). However, further improvement of C8 production is necessary to transmit the signal from bacteria to human cells. Such improvement is possible through tuning the experimental conditions further.</P></DIV><DIV id="mtt" class="w3-container" id="overview" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B> Appendix: Material and Method</B></H1><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Reference</B></H1><P style="font-size: 16px; text-indent:1em">&quot;&gt;
(1). http://www.genome.jp/tools-bin/clustalw
(2). Pavan Kumar Reddy Kambam, Daniel J. Sayut, Yan Niu, Dawn T. Eriksen, Lianhong Sun (2008) Directed evolution of LuxI for enhanced OHHL production. Biotechnology and Bioengineering Volume 101, Issue 2 1 October 2008 Pages 263-272
(3). MATTHEW R. PARSEK, DALE L. VAL, BRIAN L. HANZELKA, JOHN E. CRONAN, E. P. GREENBERG (1999) Acyl homoserine-lactone quorum-sensing signal generation. Proc. Natl. Acad. Sci. USA Vol. 96, pp. 4360-4365, April 1999 Biochemistry
(4). https://2007.igem.org/wiki/index.php/Chiba/Quorum_Sensing</P></DIV></DIV><DIV class="w3-light-grey w3-container w3-padding-32" style="margin-top:75px;padding-right:58px"><P class="w3-right"><A href="http://96haji.me/" title="W3.CSS" target="_blank" class="w3-hover-opacity">Hajime Fujita with W3.CSS: All Rights Reserved</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>