<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TokyoTech_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TokyoTech/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Coli Sapiens</TITLE></P><NAV class="w3-sidebar w3-red w3-collapse w3-top w3-large w3-padding" style="z-index:3;width:300px;font-weight:bold;" id="mySidebar"><A href="javascript:void(0)" onclick="w3_close()" class="w3-button w3-hide-large w3-display-topleft" style="width:100%;font-size:20px">Close Menu</A><DIV style="padding-top: 15px; padding-left: 25px; id=" wrap=""><DIV id="contents" style="z-index: 10"><UL class="click_menu_checkbox vertm_down"><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px;"><LABEL for="vmcb-a"><A href="https://2017.igem.org/Team:TokyoTech">Home</A></LABEL></LI><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-b"><A href="https://2017.igem.org/Team:TokyoTech/Medal">Achievements</A></LABEL></LI><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-c"><A>Project</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Description" class="w3-bar-item w3-button w3-hover-white">Description</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Project/Basic_Parts" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Basic Parts</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Project/Composite_Parts" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Composite Parts</A></LI></UL><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-d"><A>Experiment</A></LABEL></LI><LI><LABEL for="vmcb-d1"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Overview" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white" style="text-align: center;">Overview</A></LABEL></LI><LI style="padding-bottom: 10px; padding-top: 5px"><LABEL for="vmcb-d2"><A style="text-align: center;">Bacteria to Human Cells ▼</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraI Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Improvement" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraI Improvement Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraR_Reporter_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraR Reporter Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Transcriptome_Analysis" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Transcriptome Analysis</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Chimeric_Transcription_Factor" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Chimeric  Transcription  Factor Assay</A></LI><LI style="padding-bottom: 3px"><LABEL for="vmcb-d3"><A style="text-align: center;">Human Cells to Bacteria ▼</A></LABEL></LI><LI><A href="https://2017.igem.org/Team:TokyoTech/Experiment/AHK4_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">AHK4 Assay</A></LI><LI><LABEL for="vmcb-d4"><A href="https://2017.igem.org/Team:TokyoTech/InterLab" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white" style="text-align: center;">InterLab</A></LABEL></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-e"><A href="https://2017.igem.org/Team:TokyoTech/Model">Modelling</A></LABEL></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-f"><A>Human Practices</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Overview</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP/Silver" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Silver</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP/Gold_Integrated" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Gold (Integrated)</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Demonstrate" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Demonstrate</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Collaborations" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Collaborations</A></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-g"><A>About us</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Team" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Team</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Attributions" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Attributions</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Sponsors" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Sponsors</A></LI></DIV></DIV></NAV><HEADER class="w3-container w3-top w3-hide-large w3-red w3-xlarge w3-padding"><SPAN style="padding-top: 5px">iGEM Tokyo Tech</SPAN></HEADER><DIV class="w3-main" style="margin-left:340px;margin-right:40px"><DIV class="w3-container" id="contact" style="margin-top:30px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px" align="center">Modelling</H1></DIV><DIV class="w3-container" id="overview" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Overview</B></H1><P style="font-size: 16px; text-indent:1em">In iGEM history, many teams including Tokyo_Tech tried information processing between bacteria. This year we tried to establish an artificial cross-kingdom communication between human cell and bacteria then enable their co-culture. We call this new artificial human-bacteria co-culture living system “Coli Sapiens.” Many factors like growth rate of human cells and bacterial cells are quite different and it is difficult to consider the all parameters to the model. In complex systems, only essential parameters were selected and an abstract model was designed. To evaluate this model, drylab comprehensively simulated the property of the system using data from the experiments. As a result, our simulation contributed to the suggestion of part improvement to wetlab. This part improvement increased the feasibility of the model and it indicated that concentration of <SPAN style="font-style: italic">E. coli</SPAN> could be controlled by human cells and the condition for co-culture. Thus, we succeeded in engineering a new living system of co-existence between human cells and <SPAN style="font-style: italic">E. coli</SPAN>. This could be a progress for iGEM. </P><CENTER><FIGURE><FIGCAPTION style="font-size: 20px">Fig. 1 Coli Sapiens System</FIGCAPTION></FIGURE></CENTER></DIV><DIV class="w3-container" id="results" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Simulation</B></H1><P style="font-size: 16px; text-indent:1em">We developed our models with two main goals.
1. In wetlab, a lot of noise will affect to the results because co-culture between human cells and <SPAN style="font-style: italic">E. coli</SPAN> is very complicated. Constructing the model containing only essential mechanism and integrating the data from wetlab, we improve the genetic circuits and engineer the model which enables co-culture. 
2. Calculate the condition of co-existence between the human cells and <SPAN style="font-style: italic">E. coli</SPAN> changing the value of variable parameters.
    </P><P style="font-size: 20px">2-1 Introduction</P><P style="font-size: 16px; text-indent:1em">In our project, we use two signaling molecules, 3OC8HSL (hereafter C8) and isopentenyl adenine, to establish an artificial cross-kingdom communication between human cells and bacteria and MazF to control cell growth of <SPAN style="font-style: italic">E. coli</SPAN>. The details are described below.</P><P style="font-size: 18px">Signaling Molecules</P><P style="font-size: 16px"> -  C8</P><P style="font-size: 16px; text-indent:1em">C8 is one of AHLs, signaling molecules of quorum sensing. Quorum sensing is the cell-to-cell communication system used by a variety of bacteria to detect the population of cells around them. The system consists of three procedure: production of signal molecules, sensing the molecules, and responding to the signals. 3OC6HSL (C6), derived from <SPAN style="font-style: italic">Vibrio fischeri</SPAN>, and C8, derived from <SPAN style="font-style: italic">Agrobacterium fumigatus</SPAN>, are the most used signal molecules in the system and produced by LuxI and TraI proteins, respectively. C6 and C8 are the compounds called acyl-homoserine lactone (AHL) and chemical structures of these molecules are shown in Fig. 1 (Read <A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Assay">TraI Assay</A> page). 
	</P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 2 Chemical structures of AHL-type signal molecules </FIGCAPTION></FIGURE></DIV><P style="font-size: 16px"> -  Isopentenyl adenine (iP)</P><P style="font-size: 16px; text-indent:1em"> Isopentenyl adenine (iP) is a kind of cytokinin and we use it as a signal molecule from human cells to <SPAN style="font-style: italic">E. coli</SPAN> in the cross-kingdom communication. Cytokinins are the signaling molecules (or Phytohormones) that plants produce and play important roles in cell growth and differentiation. In the case of <SPAN style="font-style: italic">Arabidopsis thaliana</SPAN>, extracellular iP is received by a transmembrane receptor, AHK4 (Read  <A href="https://2017.igem.org/Team:TokyoTech/Experiment/AHK4_Assay">AHK4 Assay</A> page).</P><P style="font-size: 18px">Growth Inhibition Molecule</P><P style="font-size: 16px"> -  Toxin-antitoxin system</P><P style="font-size: 16px; text-indent:1em"> A toxin-antitoxin system is composed of two or more cognate genes that encode toxins and antitoxins. Toxins are proteins, whereas antitoxins are either proteins or non-coding RNAs. Many prokaryotes harbor toxin-antitoxin systems on the genomes, typically in multiple copies. Changes in the physiological conditions, such as stress conditions or viral infection trigger antitoxin degradation by cytosolic proteases. Unleashed toxin proteins impede or alter cellular processes including translation, cell division, DNA replication, ATP synthesis, mRNA stability, or cell wall synthesis and lead to dormancy. This dormant state probably enables bacteria to survive in unfavorable conditions. In general, toxin proteins are more stable than antitoxin proteins, but antitoxins are expressed at a higher level in cells. </P><P style="font-size: 16px"> -  MazF</P><P style="font-size: 16px; text-indent:1em">MazF is a toxin protein. MazF is a ribosome-independent endoribonuclease whose activity leads to bacterial growth arrest. MazF dimer cleaves mRNAs at ACA sequences.</P><P style="font-size: 20px">2-2 Mathematical model</P><P style="font-size: 16px; text-indent:1em">In order to simulate our gene circuits, we developed an ordinary differential equation model. The equations and parameters to simulate our genetic circuits are shown below.</P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGCAPTION style="font-size: 20px">Table. 1 Parameters </FIGCAPTION><FIGURE><FIGCAPTION style="font-size: 20px">Fig. 3 <SPAN style="font-style: italic">E. coli</SPAN></FIGCAPTION><FIGCAPTION style="font-size: 20px">Fig. 4 3OC8HSL (C8) </FIGCAPTION><FIGCAPTION style="font-size: 20px">Fig. 5 Isopentenyl adenine (iP)  </FIGCAPTION><FIGCAPTION style="font-size: 20px">Fig. 6 MazF</FIGCAPTION></FIGURE></DIV><P style="font-size: 20px">2-3 Analysis</P><P style="font-size: 16px; text-indent:1em">We first obtained result that E. coli cells temporally multiple excessively, which means E. coli cells are no longer controllable (Fig. 7). According to the detailed analysis, this E. coli explosion was due to the low concentration of C8 and insufficiency of MazF expression. Therefore, we proposed a wet experimental improvement of traI coding C8 synthetic rate.</P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 7 Result of modelling (Before improvement of TraI)</FIGCAPTION></FIGURE></DIV><P style="font-size: 16px; text-indent: 1em">This result indicated that C8 synthetic quantity was needed to increase. Thus, we proposed improvement of <SPAN style="font-style: italic">traI</SPAN> coding C8 synthetic enzyme to wetlab.

 In wetlab, the C8 synthetic quantity was improved by introducing a single point mutation to <SPAN style="font-style: italic">traI</SPAN> (Read <A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Improvement">TraI Improvement</A> page). Using this experiment data, the excessive <SPAN style="font-style: italic">E. coli</SPAN> growth was suppressed. We confirmed the desirable behavior of the whole system by modifying and improving a part. 

    </P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 8 Result of modelling (After improvement of TraI)</FIGCAPTION></FIGURE></DIV><P style="font-size: 20px">2-4 Explore the condition of co-existence</P><P style="font-size: 16px; text-indent:1em">We confirm that the human cell and the <SPAN style="font-style: italic">E. coli</SPAN> can co-exist in our model, but the condition to become co-existence is supposed to be severe. And that to clarify the condition will make a substantial contribution to application of our co-culture system. Therefore, we investigate the values of variable parameters in our model, flow(x axis, f in Fig. 9) and the concentration of human cells(y axis, u in Fig. 9), to co-exist. The result graph was shown in the below, Fig. 9. The yellow area means failure of co-existence due to excessive growth of <SPAN style="font-style: italic">E. coli</SPAN>. The purple are means that failure of co-existence due to <SPAN style="font-style: italic">E. coli</SPAN> extermination. The green area means that success of co-existence. When we apply the co-culture system (Read <A href="https://2017.igem.org/Team:TokyoTech/Description">Project page</A>), we can choose the best condition to satisfy co-existence for the application.</P></DIV><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 9 Condition of Co-existence </FIGCAPTION></FIGURE></DIV><DIV class="w3-container" id="results" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Reference</B></H1><P style="font-size: 16px">[1]  <A href="https://2016.igem.org/Team:Tokyo_Tech/Model">iGEM 2016 Tokyo_Tech</A>, accessed October 17, 2017</P><P style="font-size: 16px">[2]  <A href="https://roche-biochem.jp/catalog/category_33061/product_3.5.3.18.6.1">The protocol of β-Gal ELISA kit</A>, accessed October 17, 2017</P><P style="font-size: 16px">[3] Optimal tuning of bacterial sensing potential, Molecular Systems Biology (2009) 5, 286 Anand Pai, Lingchong You</P><P style="font-size: 16px">[4] Cytokinin Oxidase and the Regulation of Cytokinin Degradation (1994) in Cytokinins Chemistry, Activity, and Function  David W. S. Mok、Machteld C. Mok (eds) CRCPress
    </P></DIV></DIV></DIV><DIV class="w3-light-grey w3-container w3-padding-32" style="margin-top:75px;padding-right:58px"><P class="w3-right"><A href="http://96haji.me/" title="W3.CSS" target="_blank" class="w3-hover-opacity">Hajime Fujita with W3.CSS: All Rights Reserved</A></P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>