<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TokyoTech_Experiment_TraI_Assay skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TokyoTech/Experiment/TraI Assay</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Coli Sapiens</TITLE></P><NAV class="w3-sidebar w3-red w3-collapse w3-top w3-large w3-padding" style="z-index:3;width:300px;font-weight:bold;" id="mySidebar"><A href="javascript:void(0)" onclick="w3_close()" class="w3-button w3-hide-large w3-display-topleft" style="width:100%;font-size:20px">Close Menu</A><DIV style="padding-top: 15px; padding-left: 25px; id=" wrap=""><DIV id="contents" style="z-index: 10"><UL class="click_menu_checkbox vertm_down"><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px;"><LABEL for="vmcb-a"><A href="https://2017.igem.org/Team:TokyoTech">Home</A></LABEL></LI><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-b"><A href="https://2017.igem.org/Team:TokyoTech/Medal">Achievements</A></LABEL></LI><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-c"><A>Project</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Description" class="w3-bar-item w3-button w3-hover-white">Description</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Project/Basic_Parts" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Basic Parts</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Project/Composite_Parts" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Composite Parts</A></LI></UL><LI style="margin-bottom: 10px; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-d"><A>Experiment</A></LABEL></LI><LI><LABEL for="vmcb-d1"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Overview" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white" style="text-align: center;">Overview</A></LABEL></LI><LI style="padding-bottom: 10px; padding-top: 5px"><LABEL for="vmcb-d2"><A style="text-align: center;">Bacteria to Human Cells ▼</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraI Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraI_Improvement" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraI Improvement Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/TraR_Reporter_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">TraR Reporter Assay</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Transcriptome_Analysis" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Transcriptome Analysis</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Experiment/Chimeric_Transcription_Factor" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Chimeric  Transcription  Factor Assay</A></LI><LI style="padding-bottom: 3px"><LABEL for="vmcb-d3"><A style="text-align: center;">Human Cells to Bacteria ▼</A></LABEL></LI><LI><A href="https://2017.igem.org/Team:TokyoTech/Experiment/AHK4_Assay" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">AHK4 Assay</A></LI><LI><LABEL for="vmcb-d4"><A href="https://2017.igem.org/Team:TokyoTech/InterLab" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white" style="text-align: center;">InterLab</A></LABEL></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-e"><A href="https://2017.igem.org/Team:TokyoTech/Model">Modelling</A></LABEL></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-f"><A>Human Practices</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Overview</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP/Silver" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Silver</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/HP/Gold_Integrated" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Gold (Integrated)</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Demonstrate" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Demonstrate</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Collaborations" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Collaborations</A></LI><LI style="margin-bottom: 10px; text-align: center; border: 5px double #fff; border-radius: 10px"><LABEL for="vmcb-g"><A>About us</A></LABEL></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Team" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Team</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Attributions" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Attributions</A></LI><LI style="text-align: center;"><A href="https://2017.igem.org/Team:TokyoTech/Sponsors" onclick="w3_close()" class="w3-bar-item w3-button w3-hover-white">Sponsors</A></LI></DIV></DIV></NAV><HEADER class="w3-container w3-top w3-hide-large w3-red w3-xlarge w3-padding"><SPAN style="padding-top: 5px">iGEM Tokyo Tech</SPAN></HEADER><DIV class="w3-main" style="margin-left:340px;margin-right:40px"><DIV class="w3-container" id="contact" style="margin-top:30px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px" align="center">TraI Assay</H1></DIV><DIV class="w3-container" id="overview" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Introduction</B></H1><P style="font-size: 16px; text-indent:1em">Quorum sensing is the cell-to-cell communication system used by a variety of bacteria to detect the population of cells around them. The system consists of three procedure: production of signal molecules, sensing the molecules, and responding to the signals. 3OC6HSL (C6), derived from <SPAN style="font-style: italic">Vibrio fischeri</SPAN>, and 3OC8HSL (C8), derived from <SPAN style="font-style: italic">Agrobacterium fumigatus</SPAN>, are the most used signal molecules in the system and produced by LuxI and TraI proteins, respectively. C6 and C8 are the compounds called acyl-homoserine lactone (AHL) and chemical structures of these molecules are shown in Fig. 1. 
	</P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 1 Chemical structures of AHL-type signal molecules </FIGCAPTION></FIGURE></DIV><P style="font-size: 16px; text-indent:1em">The production of C6 by LuxI is constitutive and C6 can be sensed by an intracellular receptor, LuxR, in other cells. The complex of LuxR and C6 forms the homodimer and promotes the activation of corresponding promoter, <SPAN style="font-style: italic">lux</SPAN> promoter, and the transcription of downstream genes. It should be noted that LuxI is encoded by one of those genes, meaning that the production of C6 molecules is promoted by receiving C6 molecules. Therefore, as the population of bacteria grows, the production of C6 is amplified and the expression of the target genes will increase exponentially. As a consequence, bacterial cells can sense their population density and carry out cell-density specific behaviors such as luminescence emission and pathogenicity exerting.</P><P style="font-size: 16px; text-indent:1em">A similar mechanism presents for C8 molecule, and in this case, the TraR protein is the receptor of the molecule.</P><DIV style="margin-top:16px"><P style="font-size: 16px; text-indent:1em">In a previous study, an AHL-inducible eukaryotic gene expression system was developed based on TraR (1). In this system, expression from the eukaryotic promoter (CMV minimal promoter) is induced only in the presence of C8. Therefore, we here chose C8 as a signal molecule and tried to engineer <SPAN style="font-style: italic">E. coli</SPAN> cells to produce C8. 
 </P></DIV><DIV class="w3-container" id="overview" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Summary of experiment</B></H1><P style="font-size: 16px; text-indent:1em">
In this section, we investigated whether <SPAN style="font-style: italic">E. coli</SPAN> cells expressing TraI protein produce a practical amount of C8. 
To this end, two <SPAN style="font-style: italic">E. coli</SPAN> strains were constructed; one is the “Sender” strain which produces C8 and the other is the “Reporter” strain which expresses GFP in the presence of C8.
To begin with, it was investigated whether the “Reporter” cells expressed GFP when cultured in the medium containing various concentrations of C8 (0.1 nM -1000 nM).
In the previous similar experiment, the intensities of GFP fluorescence (Relative Fluorescence Units; RFU) have shown to follow Hill's equation (2). Therefore, in this study, the parameters of Hill's equation were obtained from the data and the concentrations of AHL were calculated from the values of RFU. 
Then, the ability of the Sender to produce AHL was analyzed. The supernatant of the Sender was mixed with the actively growing culture of the Reporter, and the production of AHL was evaluated by observing the expression of GFP.
The following plasmids were introduced into <SPAN style="font-style: italic">E. coli</SPAN>. 
Reporter <SPAN style="font-style: italic">E. coli</SPAN>
By introducing the plasmids shown in Fig. 2, <SPAN style="font-style: italic">E. coli</SPAN> cells are expected to produce GFP in response to C8 and C6. Note that Ptet is the constitutive promoter. Also, note that LuxR can accept C8 as well as the natural ligand, C6 (3); we here employed LuxR, but not TraR, because LuxR had been characterized far better than TraR in the preceding iGEM projects.
	</P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 2 Structure of the plasmids used for creating the “Reporter”</FIGCAPTION></FIGURE></DIV><P style="font-size: 16px; text-indent:1em">
Sender <SPAN style="font-style: italic">E.coli</SPAN>
We created the Sender by introducing the plasmid shown in Fig. 3.
The Sender is expected to produce C8 constantly, because the <SPAN style="font-style: italic">traI</SPAN> gene is placed at downstream of the constitutive promoter, Ptet.
	</P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 3 Construction of TraI gene</FIGCAPTION></FIGURE></DIV></DIV><DIV class="w3-container" id="results" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Results</B></H1><P style="font-size: 16px; text-indent:1em">
Assay using reagent AHLs
In order to analyze the ability of the Reporter to receive AHLs and to express GFP depending on AHL, defined concentrations of reagent AHLs were added to growing culture of the Reporter. It was confirmed that LuxR responded to C8 in a similar level to C6. RFU of the Reporter at various AHL concentrations (0.1 nM - 1000 nM) is shown in Fig. 4. Detection limit was over 10 nM for both cases. The C8 concentration 0 means adding DMSO (solvent for AHLs is DMSO).
	</P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 4 Concentration dependance of Reletive Fluoroscent Units </FIGCAPTION></FIGURE></DIV><P style="font-size: 16px; text-indent:1em">
The data are presented as mean ± SD from triplicate experiments.
Based on the data which is shown in Fig. 4, parameter was obtained to fit Hill’s equation.
Hill’s equation is shown in Eq. 1
	</P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Eq.1 Hill's equation  </FIGCAPTION></FIGURE></DIV><P style="font-size: 16px; text-indent:1em">
The values of parameters are shown in Table. 1
The parameter “a” represents leakiness of the GFP expression in the Reporter. Even in the absence of AHL, it is known that downstream genes below Plux are transcribed slightly. The parameter “b” is the value of RFU when AHL binds to all receptors and is completely induced. The parameter “n“ is the Hill coefficient, and when this value is 1 or more, it is said that there are multiple binding sites. “Km” is the AHL concentration where half of the receptor molecules are bound to the AHL molecules, and this value represent the detection sensitivity of the Reporter. It was found that both AHLs can be detected with a sensitivity of order 10 nM. </P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Table. 1 Parameters of Hill’s equation </FIGCAPTION></FIGURE></DIV><P style="font-size: 16px; text-indent:1em">
The values of parameters are shown in Table. 1
The parameter “a” represents leakiness of the GFP expression in the Reporter. Even in the absence of AHL, it is known that downstream genes of Plux are transcribed slightly. The parameter “b” is the value of RFU when AHL binds to all receptors and is completely induced. The parameter “n“ is the Hill coefficient, and when this value is 1 or more, it is said that there are multiple binding sites. “Km” is the AHL concentration where half of the receptor molecules is bound to the AHL molecules, and this value represent the detection sensitivity of the Reporter. It was found that both AHLs can be detected with a sensitivity of order 10 nM. </P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 5 Actual measurement value and Theoretical formula </FIGCAPTION></FIGURE></DIV><P style="font-size: 16px; text-indent:1em">
Supernatant Assay 
Temperature dependence of AHL production. 
During the trial-and-error process for increasing the productivity of AHL in the Sender, we found that the amount of C8 was higher when the Sender was grown at a lower temperature; the RFU value was 14-fold higher. <SPAN style="font-style: italic">E. coli</SPAN> introduced an empty vector was used as Negative Control.
C8 concentrations were calculated from the RFU values using calibration curve, 34 nM of C8 was produced in the 37℃ culture, whereas C8 concentration exceeded the detection limit in the 25℃ culture.
	</P><DIV class="w3-xxxlarge" style="padding-bottom: 10px;padding-top: 10px;text-align: center"><FIGURE><FIGCAPTION style="font-size: 16px">Fig. 6 Temperature dependency of C8 production</FIGCAPTION></FIGURE></DIV></DIV><DIV class="w3-container" id="results" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Discussion</B></H1><P style="font-size: 16px; text-indent:1em">
We confirmed that <SPAN style="font-style: italic">E. coli</SPAN> cells produces over 200 nM of C8. However, as shown in other wiki pages (Read <A href="https://2017.igem.org/Team:TokyoTech/Description">Project Description</A> page and Read <A href="https://2017.igem.org/Team:TokyoTech/Experiment/Chimeric_Transcription_Factor">Chimeric Transcription Factor Assay</A> page), the final objective of our project is inducing gene expression with C8 in human cells. From the <A href="https://2017.igem.org/Team:TokyoTech/Model">Simulation</A>, more C8 was required to do so. Therefore, we need to improve the production amount of C8 further. 
The result in Fig. 5 shows that temperature dependency of C8 production. This result may reflect that the <SPAN style="font-style: italic">traI</SPAN> gene is derived from a soil bacterium <SPAN style="font-style: italic">A. tumefaciens</SPAN>; in nature, the temperature of soil hardly reaches  37 ℃, and the TraI protein may be unstable at 37℃. Indeed, growth of <SPAN style="font-style: italic">A. tumefaciens</SPAN> occurs optimally at 28°C, and at above 30°C, <SPAN style="font-style: italic">A. tumefaciens</SPAN> becomes heat-shock state (5).</P></DIV><DIV id="mtt" class="w3-container" id="overview" style="margin-top:20px"><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B> Appendix: Material and Method</B></H1><H1 class="w3-xxxlarge w3-text-red" style="padding-bottom: 10px;padding-top: 10px"><B>Reference</B></H1><P style="font-size: 16px">
(1). Neddermann P1, Gargioli C, Muraglia E, Sambucini S, Bonelli F, De Francesco R, Cortese R (2003) A novel, inducible, eukaryotic gene expression system based on the quorum-sensing transcription factor TraR. EMBO Rep. 2003 Feb;4(2):159-65.
(2). https://2014.igem.org/Team:ETH_Zurich/modeling/qs
(3). https://2016.igem.org/Team:Tokyo_Tech/AHL_Assay/AHL_Reporter_Assay 
(4). Elise R. Morton and Clay Fuqua (2012) UNIT 3D.1 Laboratory Maintenance of Agrobacterium. Curr Protoc Microbiol. 2012 Feb; CHAPTER: Unit3D.1.</P></DIV></DIV><DIV class="w3-light-grey w3-container w3-padding-32" style="margin-top:75px;padding-right:58px"><P class="w3-right"><A href="http://96haji.me/" title="W3.CSS" target="_blank" class="w3-hover-opacity">Hajime Fujita with W3.CSS: All Rights Reserved</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>