"
Page
Discussion
View source
History
teams
Log in
 
Team:Penn State/MSC Design
From 2012.igem.org
Multiple Start Codons Overview
Home | Team | Projects | Parts | Main Results | Notebook | Other
Multiple Start Codons
A frustrating yet commonly observed phenomenon in the lab is the production of unexpected proteins. These occurrences may be explainable by multiple start codons in the mRNA strand. Codon slippage is a theory practically untouched by research, and this project aspires to shed some light on the issue.
Multiple Start Codons
Overview | Design | Results
Circuit Model
Circuit Design
To begin construction of this plasmid, the DNA sequence was generated and edited in ApE, a DNA editing software available through the University of Utah, from several existing plasmid sequences available through other labs on the Penn State campus. The construct was rationally designed to include two start codons within 7 nucleotides of each other, and thus in different open reading frames.
Each codon, labeled RFP-coding ATG and sfGFP-coding ATG for simplicity, codes for a different reporter in the respective reading frame of each start codon. The first start codon, RFP-coding ATG, if recognized by the ribosome, will translate the reporter RFP. The second start codon, sfGFP-codon ATG, will translate the reporter sfGFP if recognized by the ribosome. Super-folder GFP (sfGFP) is used to allow for the nonsense codons of the out-of-frame RFP sequence preceding the sfGFP reporter which would inevitably be translated prior to the green reporter sequence should the ribosome recognize the sfGFP-coding ATG. The super-folder protein will fold regardless of this nonsense sequence, facilitating the measurement of codon slippage through the measurement of fluorescence
To prevent premature termination of the translation of either reporter, all stop codons in the respective reading frames of each codon were eliminated through amino acid sequence substitution.
Circuit Construction
Initially, 4 gBlocks, available through IDT as oligos with overlapping 40bp sequences complimentary to the adjacent oligo, were used to perform a Gibson Assembly reaction with a digested dRBS1 vector. After several failed attempts, a second approach was implemented. Oligos for the missing circuit sequence were designed and annealed, according to their complimentary 20bp overlaps, and inserted into the nearly-complete vector using Gibson Assembly. This attempt was successful, and the initial construct was verified with sequencing.
Check these Out!
Judging Criteria
Parts
Human Practices
Projects
Multiple Start Codons
Bidirectional Promoters
Codon Optimization
© Hannah Jepsen-Burger | Follow us on Twitter
Retrieved from " http://2012.igem.org/Team:Penn_State/MSC_Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
