"
Page
Discussion
View source
History
teams
Log in
 
Team:Penn State/Projects
From 2012.igem.org
Penn State iGEM 2012
Navigation
Home
Team
Official Team Profile
Projects
Parts Submitted to the Registry
Modeling
Notebook
Safety
Attributions
Contents
1 Objective
2 Project 1: Codon Optimization
2.1 Project Overview
2.2 Experiments
2.3 Results
3 Project 2: Multidirectional Promoters
3.1 Project Overview
3.2 Experiments
3.3 Results
4 Project 3: Multiple Start Codons
4.1 Project Overview
4.2 Experiments
4.3 Results
Objective
The focus of this years' projects is to question the central dogma of biology. This is the commonly held belief that information flows from the ordering of the bases within a cell's DNA to the ordering of the bases within RNA. This information can then be translated to the order of Amino Acids that make up proteins. We are currently working on projects that question aspects of this dogma.
Project 1: Codon Optimization
Project Overview
All of the proteins around us, with few exceptions, are made up of 20 fundamental building blocks - Amino Acids. Different arrangements and combinations of these basic building blocks gives us the diversity of proteins that we see, but to make these we need information. We need to know the order of the bases. This information is held within the order of a messenger RiboNucleic Acid's (mRNA) bases; Adenine, Guanine, Thymine, and Uracil. (mRNA is in essence a "photocopy" of DNA that codes for a gene. <p/> <p> Since there are only 4 bases in RNA and there are 20 Amino Acids, there are not enough bases to directly code for all of the Amino Acids. Nature has a solution for this - codons. Codons are groups of three bases that sode for a single amino acid. There are 64 possible combinations of codons (4 x 4 x 4 = 64). These combinations are degenerate, meaning that you can have more than one codon code for the same Amino Acid.
Proteins are assembled by ribosomes, which read the mRNA and catalyze the binding between Amino Acids. However, they cannot read the codon code themselves, and require the help of tRNA (transfer RNA). Each tRNA codes for a single codon, or a set of similar codons. Each type of tRNA can only carry a single amino acid at a time, and can only ever carry that given Amino Acid. Each tRNA has bases that are a complement to the codon that they code for. The tRNA molecules carry Amino Acids to the ribosome where the Amino Acids are bound together to form a protein.
We have mentioned how there can be a number of codons that can code for the same Amino Acid, and subsequently, a number of tRNA molecules that can carry a given Amino Acid. In nature, and in many organism, only a select few of these tRNA and codon combinations are used instead of all possibilities for the same amino acid. This is called codon bias.
Our goal for this project is to see which codons are preferred, or biased. We also want to investigate if this bias can change over time or in different circumstances or stresses. For this we will be using the standard lab stain of E. coli, DH10B.
Experiments
Coming soon…
Results
Coming soon…
Project 2: Multidirectional Promoters
Project Overview
Before we can make a protein, we need an mRNA to carry the information about the order of the amino acids. But before we can make mRNA we need to know where the genes are in the DNA of a cell. Before RNA polymerase can make a copy it needs to bind to the DNA. This is assisted by a variety of factors, other proteins, that look for a specific sequence in the DNA. This sequence is called a promoter because it promotes the transcription of the DNA into RNA by the RNA polymerase. These sequences are generally upstream, or ahead, of a gene's coding sequence.
However, not all promoters cause RNA polymerase to transcribe downstream in the expected "forward direction". Some promoters can cause RNA polymerase to go in the opposite direction from what is expected, or go in both directions. This is what we are trying to find out; do different promoters go in different directions, and what is the directional preference of different promoters.
Experiments
Coming soon…
Results
Coming soon…
Project 3: Multiple Start Codons
Project Overview
mRNA is the molecule that carries information about the sequence of amino acids in a protein. However, much like the lines on a sheet of paper, the protein codeine sequence of an mRNA molecule does not start right at the beginning, or top of the page. Instead, once the mRNA is bound by a ribosome, a start codon must first be read before the protein can be translated. This start codon is generally AUG, or Methionine.
Once this start codon in read the ribosome will continue reading and building the polypeptide (protein) until a stop codon is reached. But what happens if you have two AUG codons close together? That is the question we are attempting to answer.
We are trying to understand what happens when there are two start codons very close together, but out of frame. Out of frame refers to how the ribosome reads the mRNA. Remember those codons and how they are groups of three bases on the mRNA? The reading frame refers to which group of three. If you start at one base and read the bases in groups of three from that point on, that is one frame of reference. If you then move your start point ahead one base, then you are reading in a new reading frame. If you advance you starting point one more base, that is the third reading frame. If you advance it again you are now back in your first reading frame, but you have skipped the first codon. We are looking into what happens when you have multiple start codons close together, but in different reading frames. Which frame will be preferred?
Experiments
Coming soon…
Results
Coming soon…
Retrieved from " http://2012.igem.org/Team:Penn_State/Projects "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
