<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Eindhoven_LEC_Lab"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Eindhoven/LEC/Lab">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Eindhoven/LEC/Lab&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Eindhoven/LEC/Lab&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Eindhoven/LEC/Lab&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Eindhoven/LEC/Lab" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Eindhoven/LEC/Lab</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="wrap"><DIV id="header" class="clickableDiv"><A href="https://2012.igem.org/Team:TU-Eindhoven"> </A></DIV><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#"> Introduction</A></LI><LI><A href="/Team:TU-Eindhoven">Home page</A></LI><LI><A href="/Team:TU-Eindhoven/Background">Backgroundinformation</A></LI><LI><A href="/Team:TU-Eindhoven/Team">About us</A></LI><LI><A href="/Team:TU-Eindhoven/Contact">Contact us</A></LI><LI><A href="/Team:TU-Eindhoven/Photos">Our photos</A></LI></UL><LI><A class="head" href="#">Project</A></LI><LI><A href="/Team:TU-Eindhoven/Project">Project description</A></LI><LI><A href="/Team:TU-Eindhoven/LEC/LabTheory">Lab approach</A></LI><LI><A href="/Team:TU-Eindhoven/LEC/Device">Device information</A></LI><LI><A href="/Team:TU-Eindhoven/LEC/ModellingFirst">Initial model</A></LI><LI><A href="/Team:TU-Eindhoven/Modeling">Yeast model</A></LI><LI><A href="/Team:TU-Eindhoven/LEC/Lab">Lab results</A></LI><LI><A href="/Team:TU-Eindhoven/Parts">Our BioBricks<SUP>TM</SUP></A></LI><LI><A href="/Team:TU-Eindhoven/LEC/Accomplishments">Ouraccomplishments</A></LI><LI><A class="head" href="#">Considerations</A></LI><LI><A href="/Team:TU-Eindhoven/Human_Practices">Public outreach</A></LI><LI><A href="/Team:TU-Eindhoven/Safety">Safety questions</A></LI><LI><A href="/Team:TU-Eindhoven/Thoughts">Other thoughts</A></LI><LI><A class="head" href="#">Future applications</A></LI><LI><A href="/Team:TU-Eindhoven/Future_applications/Human_interest">Human interest</A></LI><LI><A href="/Team:TU-Eindhoven/Future_applications/Developments">Furtherdevelopments</A></LI><LI><A class="head" href="/Team:TU-Eindhoven/Notebook/Week1">Notebook</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Before">Before week 1</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week1">Week 1</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week2">Week 2</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week3">Week 3</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week4">Week 4</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week5">Week 5</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week6">Week 6</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week7">Week 7</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week8">Week 8</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week9">Week 9</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week10">Week 10</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week11">Week 11</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week12">Week 12</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/FurtherWork">After week 12</A></LI><LI><A href="/Team:TU-Eindhoven/Protocols">Lab protocols</A></LI><LI><A href="/Team:TU-Eindhoven/Partners" class="head">Partners</A></LI><LI><A href="/Team:TU-Eindhoven/ICMS">ICMS TU/e</A></LI><LI><A href="/Team:TU-Eindhoven/Collaborations">Team collaborations</A></LI><LI><A href="/Team:TU-Eindhoven/Attributions">Attributions and Contributions</A></LI><LI><A href="/Team:TU-Eindhoven/Partners">Our sponsors</A></LI></DIV><DIV id="right"><H3><SPAN class="mw-headline" id="GECO_protein_expression_and_isolation">GECO protein expression and isolation</SPAN></H3><P>The 3 GECOs have been expressed in E. coli BL21 to yield a large amount of the proteins for characterization. Cells were <SPAN class="lightblue">cultured</SPAN> in large Erlenmeyer flasks. The <SPAN class="lightblue">results were visually impressive</SPAN>: After lysis the flasks turned vibrantly red and green respectively. The blue GECO however looked much the same as the green GECO.</P><P>More information about the GECOs can be found at <A href="/Team:TU-Eindhoven/Parts" title="Team:TU-Eindhoven/Parts">our BioBricks<SUP>TM</SUP></A>.</P><H3><SPAN class="mw-headline" id="Yeast_transformation">Yeast transformation</SPAN></H3><P>A weekly item on our schedule was <SPAN class="lightblue">yeast transformation</SPAN>. It takes about a week to complete a transformation, that is, to add one plasmid to an existing strain variant. Because success is not guaranteed we choose to introduce plasmids in various orders in parallel. This resulted in <SPAN class="lightblue">many variants</SPAN> that we assigned a unique number for convenience. The same number was used on plates, cultures and cryostocks. In total <SPAN class="lightblue">49 variants</SPAN> were made.</P><P>To reach the aim of this project and prepare a multi-colored screen, three kind of cells, each with a different color, are needed. Therefore, one single plasmid strain would have to consist of <SPAN class="lightblue">both calcium channel parts MID1 and CCH1 sequences and one of the fluorescent GECO protein sequence</SPAN>. Each of these DNA sequences, coding for the three different parts, is transformed into a yeast plasmid (InvSC1). Moreover, each of these sequences contain a small sequence coding for an amino acid, causing the yeast strain to be able to synthesize this specific amino acid. The synthesis of this specific amino acid will allow strain selection by retaining this amino acid from the culture media. Cells containing the  coding CCH1, MID1 and GECO sequence are able to <SPAN class="lightblue">synthesize leucine, uracil and tryptophan</SPAN> respectively. </P><P>Looking at Scheme 1, which represents the <SPAN class="lightblue">transformations</SPAN> needed to come to the desired yeast strain, it seems and is logical to start with the MID1 or CCH1 vectors. By doing so, less transformations can be performed due to the transformation intensive introduction of the three different GECO proteins at last. To make sure that the transformations would lead to the desired strain, it was decided to start the first step with transforming <SPAN class="lightblue">all possibilities</SPAN>. When step 1 was transformed successfully, visualized by the arrows in Scheme 1, the transformation of step 2 was performed and so on.</P><P><I>Scheme1: Transformation Scheme</I></P><P>The several transformation steps are clarified in Scheme 1. The arrows represent a <SPAN class="lightblue">successful transformation</SPAN> as at the same time the lack of an arrow represents a <SPAN class="lightblue">transformation failure</SPAN>. Of course, not every transformation was needed to obtain the desired strain. E.g. in step 1 the transformation of the R-GECO failed, however, the strain containing the R-GECO was obtained in step 2 due to other successful transformations. 
As can be seen, in step 2 the first co-transformation are performed. This brought some complications because the regular transformation protocol did not work for these co-transformations. For the co- and co-co-transformations (performed in step 3) we used <SPAN class="lightblue">another high-efficiency</SPAN> transformation <A href="/Team:TU-Eindhoven/Protocols" title="Team:TU-Eindhoven/Protocols">protocol</A> which worked out successfully. After 3 steps of transformations, a couple of different transformation paths reached the 3 desired strains a complete calcium channel and 1 of the 3 GECO proteins.</P><P>Initially, yeast transformations failed or yielded only several colonies, a lot less than expected. A review of transformation protocols showed that the <SPAN class="lightblue">shock protocol</SPAN> we used should be more efficient than electroporation or other common methods. We tried to transform another yeast strain with our plasmids which yielded the expected high transformation efficiency. Unfortunately that strain was not compatible with the auxotrophic markers on the plasmids we wanted to introduce. Our problem was remedied by using <SPAN class="lightblue">more plasmid DNA</SPAN> for the transformation, as described in our <A href="/Team:TU-Eindhoven/Protocols" title="Team:TU-Eindhoven/Protocols">modified yeast transformation protocol</A>.</P><H3><SPAN class="mw-headline" id="Device_tests">Device tests</SPAN></H3><P>The device was tested with yeast containing <SPAN class="lightblue">all three plasmids</SPAN>, thus over expressing channels and expressing GECOs. A quick inspection with the naked eye did not show any sign of visual light coming off the device, not even in a dark room. We think that the yeast was <SPAN class="lightblue">not concentrated enough</SPAN> to show a clear response.</P><H3><SPAN class="mw-headline" id="Detection_of_GECO_protein_in_yeast">Detection of GECO protein in yeast</SPAN></H3><P>Fluorescent proteins like our GECOs can be detected by <SPAN class="lightblue">spectrophotometry</SPAN> if the intensity is insufficient to be observed by the naked eye. If many samples have to be measured these can be put onto a 96-wells plate and measured all at once in a special type of spectrophotometer called a <SPAN class="lightblue">plate reader</SPAN>. A plate reader is also convenient for use with small sample volumes, which was beneficial in our case, since it allowed us to concentrate the yeast from a large culture volume into a small volume through centrifugation and obtain a better measurement.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Samples of yeast on a 96-wells plate.</DIV></DIV></DIV><P>The expression of GECOs in yeast did <SPAN class="lightblue">not yield obvious fluorescence</SPAN> when <A href="/Team:TU-Eindhoven/LEC/Device" title="Team:TU-Eindhoven/LEC/Device">stimulated in the device</A>. To check if the protein was actually present in our cells, we made spectrograms of cells concentrated to 1.25 x 10<SUP>8</SUP> cells/ml, which amounts to an OD<SUB>600</SUB> of approximately 125.</P><P>We wanted to demonstrate the presence of GECOs in yeast cells carrying GECO, CCH1 and MID1 plasmids, cultured on induction medium. For this we used a plate reader to detect the possible fluorescence of any GECO protein present within the cells. To <SPAN class="lightblue">exclude</SPAN> other sources of fluorescence we also measured yeast cells without either GECOs, without over expressed CCH1-MID1 or without any plasmids. Furthermore we compared cells carrying GECO- and CCH1-MID1 plasmids cultured in 2% galactose induction medium with cells cultured in 2% glucose growth medium, which represses induction.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Emission spectra of INVSc1 yeast cells carrying one GECO plasmid and CCH1 and MID1 plasmids cultured on induction medium (solid lines) or culture medium which represses induction (dashed lines). Excitation peak was centered around 495 nm with a bandwidth of 5 nm.</DIV></DIV></DIV></DIV><P>When excited at 495 nm, around the <SPAN class="lightblue">peak in the excitation spectrum of G-GECO</SPAN>, both cells carrying G-GECO and cells carrying B-GECO plasmids cultured in induction medium fluoresced, whereas cells carrying R-GECO cultured in induction medium did not fluoresce. Cells cultured in medium that represses the expression of GECO protein did not fluoresce. Furthermore the spectra of G-GECO and B-GECO seem to overlap, which confirms our earlier observation that spectra of G-GECO and B-GECO protein produced in E. coli overlapped.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Emission spectra of INVSc1 yeast cells carrying one GECO plasmid and CCH1 and MID1 plasmids cultured on induction medium (solid lines) or culture medium which represses induction (dashed lines). Excitation peak was centered around 565 nm with a bandwidth of 5 nm.</DIV></DIV></DIV></DIV><P>When excited at 565 nm, around the <SPAN class="lightblue">peak in the excitation spectrum of R-GECO</SPAN>, only cells carrying R-GECO plasmids cultured in induction medium fluoresced, whereas cells carrying G-GECO or B-GECO plasmids cultured in induction medium did not fluoresce. Cells cultured in medium that represses the expression of GECO protein did not fluoresce.</P><P>There is a <SPAN class="lightblue">clear difference</SPAN> in spectra between R-GECO and G-GECO, which means they can be <SPAN class="lightblue">used together</SPAN> in a multi-colored screen without disturbing each others emission. The difference between G-GECO and B-GECO is very small, perhaps another excitation wavelength could better separate the emission spectra and make them better suited for simultaneous use in the same screen.</P><P>The following non-BioBrick plasmids have been constructed:</P><DIV><TABLE class="pgrouptable tablesorter" style="width: 100%;" cellpadding="0" cellspacing="0"><THEAD><TR><TH style="width:100px;">name</TH><TH style="width:80px;">vector</TH><TH style="width:200px;">inserts</TH><TH style="width:suto;">description</TH></TR></THEAD><TBODY><TR><TD>pYES2/CT-X-GECO</TD><TD>pYES2/CT</TD><TD>R-GECO1, G-GECO1.1, B-GECO1</TD><TD>High copy-number shuttle vector </TD></TR><TR><TD>pYES3/CT-X-GECO</TD><TD>pYES3/CT</TD><TD>R-GECO1, G-GECO1.1, B-GECO1</TD><TD>High copy-number shuttle vector </TD></TR><TR><TD>pET<SUB>28a</SUB></TD><TD>X-GECO/pET<SUB>28a</SUB></TD><TD>R-GECO1, G-GECO1.1, B-GECO1</TD><TD>Expression vector for E. coli </TD></TR></TBODY></TABLE><P>The following plasmids were received from Addgene.org:</P><P><TABLE class="pgrouptable tablesorter" style="width: 100%;" cellpadding="0" cellspacing="0"><THEAD><TR><TH style="width:70px;">name</TH><TH style="width:40px;">vector</TH><TH style="width:170px;">inserts</TH><TH style="width:auto;">description</TH></TR></THEAD><TBODY><TR><TD>CMV-X-GECO</TD><TD>CMV </TD><TD>R-GECO1, G-GECO1.1, B-GECO1</TD><TD>Shipping vector use by addgene.org, CMV vector was not used in this project</TD></TR></TBODY></TABLE></P><P>These plasmids were kindly donated by H. Iida &amp; K. Iida:</P><P><TABLE class="pgrouptable tablesorter" style="width: 100%;" cellpadding="0" cellspacing="0"><THEAD><TR><TH style="width:110px;">name</TH><TH>vector</TH><TH>inserts</TH><TH>description</TH></TR></THEAD><TBODY><TR><TD>pBCT -CCH1H</TD><TD>pBCT </TD><TD>CCH1H</TD><TD>Low copy-number shuttle vector, encodes the CCH1 protein</TD></TR><TR><TD>pBCT-CCH1H-HA4</TD><TD>pBCT</TD><TD>CCH1H-HA4</TD><TD>Low copy-number shuttle vector, encodes HA4-tagged CCH1</TD></TR><TR><TD>pBCT-CCH1H-EGFP</TD><TD>pBCT</TD><TD>CCH1H-EGFP</TD><TD>Low copy-number shuttle vector, encodes EGFP-tagged CCH1</TD></TR><TR><TD>YCpT-MID1</TD><TD>YCpT</TD><TD>MID1</TD><TD>Low copy-number shuttle vector, encodes the MID1 protein</TD></TR><TR><TD>YCpT-MID1-EGFP</TD><TD>YCpT</TD><TD>MID1-EGFP</TD><TD>Low copy-number shuttle vector, encodes EGFP-tagged MID1</TD></TR></TBODY></TABLE></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:TU-Eindhoven/LEC/Lab">http://2012.igem.org/Team:TU-Eindhoven/LEC/Lab</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Eindhoven/LEC/Lab" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Eindhoven/LEC/Lab" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Eindhoven/LEC/Lab&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Eindhoven/LEC/Lab&amp;oldid=232645" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>