<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Eindhoven_LEC_LabTheory"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Eindhoven/LEC/LabTheory">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Eindhoven/LEC/LabTheory&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Eindhoven/LEC/LabTheory&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Eindhoven/LEC/LabTheory&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Eindhoven/LEC/LabTheory" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Eindhoven/LEC/LabTheory</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="wrap"><DIV id="header" class="clickableDiv"><A href="https://2012.igem.org/Team:TU-Eindhoven"> </A></DIV><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#"> Introduction</A></LI><LI><A href="/Team:TU-Eindhoven">Home page</A></LI><LI><A href="/Team:TU-Eindhoven/Background">Backgroundinformation</A></LI><LI><A href="/Team:TU-Eindhoven/Team">About us</A></LI><LI><A href="/Team:TU-Eindhoven/Contact">Contact us</A></LI><LI><A href="/Team:TU-Eindhoven/Photos">Our photos</A></LI></UL><LI><A class="head" href="#">Project</A></LI><LI><A href="/Team:TU-Eindhoven/Project">Project description</A></LI><LI><A href="/Team:TU-Eindhoven/LEC/LabTheory">Lab approach</A></LI><LI><A href="/Team:TU-Eindhoven/LEC/Device">Device information</A></LI><LI><A href="/Team:TU-Eindhoven/LEC/ModellingFirst">Initial model</A></LI><LI><A href="/Team:TU-Eindhoven/Modeling">Yeast model</A></LI><LI><A href="/Team:TU-Eindhoven/LEC/Lab">Lab results</A></LI><LI><A href="/Team:TU-Eindhoven/Parts">Our BioBricks<SUP>TM</SUP></A></LI><LI><A href="/Team:TU-Eindhoven/LEC/Accomplishments">Ouraccomplishments</A></LI><LI><A class="head" href="#">Considerations</A></LI><LI><A href="/Team:TU-Eindhoven/Human_Practices">Public outreach</A></LI><LI><A href="/Team:TU-Eindhoven/Safety">Safety questions</A></LI><LI><A href="/Team:TU-Eindhoven/Thoughts">Other thoughts</A></LI><LI><A class="head" href="#">Future applications</A></LI><LI><A href="/Team:TU-Eindhoven/Future_applications/Human_interest">Human interest</A></LI><LI><A href="/Team:TU-Eindhoven/Future_applications/Developments">Furtherdevelopments</A></LI><LI><A class="head" href="/Team:TU-Eindhoven/Notebook/Week1">Notebook</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Before">Before week 1</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week1">Week 1</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week2">Week 2</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week3">Week 3</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week4">Week 4</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week5">Week 5</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week6">Week 6</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week7">Week 7</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week8">Week 8</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week9">Week 9</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week10">Week 10</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week11">Week 11</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/Week12">Week 12</A></LI><LI><A href="/Team:TU-Eindhoven/Notebook/FurtherWork">After week 12</A></LI><LI><A href="/Team:TU-Eindhoven/Protocols">Lab protocols</A></LI><LI><A href="/Team:TU-Eindhoven/Partners" class="head">Partners</A></LI><LI><A href="/Team:TU-Eindhoven/ICMS">ICMS TU/e</A></LI><LI><A href="/Team:TU-Eindhoven/Collaborations">Team collaborations</A></LI><LI><A href="/Team:TU-Eindhoven/Attributions">Attributions and Contributions</A></LI><LI><A href="/Team:TU-Eindhoven/Partners">Our sponsors</A></LI></DIV><DIV id="right"><H3><SPAN class="mw-headline" id="Design_challenge">Design challenge</SPAN></H3><P>The aim of this project is to design and produce a new multi-colored display in which genetically engineered cells function as pixels analogous to a conventional display. In the lab we will create cells that emit light in response to an electric stimulus. This can be achieved by <SPAN class="lightblue">genetic modification of yeast cells</SPAN>, through the introduction of fluorescent calcium sensors and utilizing the already present calcium channels. Already anticipating on the probably slow response time of our 'pixel' we also intend to over express the calcium channels together with the expression of the GECO proteins as a means to enhance the calcium flux. On top of our great endeavor an other innovative aspect is the fact that our system responds to <SPAN class="lightblue">exogeneous stimuli</SPAN> instead of the more traditional systems which respond to intracellular processes.</P><P>The plasma membrane of the brewer's yeast <I>Saccharomyces cerevisiae</I> contains the <SPAN class="lightblue">CCH1-MID1 channel</SPAN> that is homologous to mammalian voltage-gated calcium channels (VGCCs). It is hypothesized that upon depolarization of the plasma membrane <SPAN class="lightblue">calcium ions selectively enter the cytoplasm</SPAN> through these channels<A href="#ref_Iida" name="text_Iida"><SUP>[1]</SUP></A>. Light will be emitted by fluorescence of the <SPAN class="lightblue">GECO protein</SPAN><A href="#ref_zhao" name="text_zhao"><SUP>[2]</SUP></A>, a calcium dependent fluorescent protein that is expressed from a genetically engineered plasmid. When this calcium sensor is exposed to an elevated intracellular calcium concentration its fluorescence will increase significantly; consequently when the calcium concentration drops its fluorescence will diminish. The calcium can enter the cell's cytoplasm upon electrical stimulation of the calcium channel, after which the GECO protein will start to fluoresce. Finally the excess of calcium will be removed by active transport within the cell to restore it's <SPAN class="lightblue">homeostatic level</SPAN>, and the amount of fluorescence will decrease.</P><P>Challenges in the laboratory can be found in creating yeast cells with the GECO protein as well as a sufficient amount of calcium channels which consist of two separate proteins. The important design choices regarding the biological work are further motivated in the text below.</P><P>Fig. 1 Yeast cell with the needed plasmids</P><H3><SPAN class="mw-headline" id="Chassis">Chassis</SPAN></H3><P>Before this light emitting cell display project could start, it was necessary to decide on a suitable chassis.  Candidates were E. coli and S. cerevisiae. Both are common model organisms that can be used for protein expression and are cheap to culture. To reach the goal of this project expression of <SPAN class="lightblue">voltage-gated calcium channel</SPAN> is needed, which luckily is already the case for S. cerevisiae. Therefore, it was decided to use S. cerevisiae as our chassis throughout the project. More specifically, we used the <SPAN class="lightblue">INVSc1 yeast strain</SPAN> which is compatible with the choice of vectors. Additional benefits when using yeast is that there are still only a <SPAN class="lightblue">few BioBricks<SUP>TM</SUP></SPAN> available, which creates a greater opportunity to contribute to the Registry, but also the drawback of having less BioBricks<SUP>TM</SUP> available to ourselves. Another benefit is that, at the moment, the manipulation of yeast is <SPAN class="lightblue">less standardized</SPAN> than it is for E. coli, which ultimately means that our research is <SPAN class="lightblue">more innovative</SPAN>.</P><H3><SPAN class="mw-headline" id="Plasmids_and_transformations">Plasmids and transformations</SPAN></H3><P>Since we choose yeast as the chassis we preferred to clone all required genes into <SPAN class="lightblue">shuttle vectors</SPAN>. A shuttle vector is a special type of vector that can be propagated both in yeast and in bacteria. For our purposes we choose to use <SPAN class="lightblue">high copy-number vector</SPAN> in order to achieve high concentration of GECO protein.</P><P>The proteins CCH1 and MID1 together constitute a high-affinity Ca<SUP>2+</SUP>-channel. Both proteins are brought to expression on a <SPAN class="lightblue">low copy-number shuttle vector</SPAN>. We are thankful to <SPAN class="lightblue">Hidetoshi Iida and Kazuko Iida</SPAN> for their donation of the shuttle vectors pBCT-CCH1H and YCpT-MID1 which they constructed in earlier research.</P><P>The GECO proteins that were engineered in the group of <SPAN class="lightblue">Robert E. Campbell</SPAN> (Zhao et al., 2011<SUP>[2]</SUP>) are available from Addgene.org, a non-profit plasmid sharing service. Via PCR we cloned the GECOs from the shipping plasmid into a pYES3 shuttle vector acquired from Invitrogen (Fig. 1).</P><P>Transformation of S. cerevisiae cells can be done with a heat shock. To obtain a cell containing all the 3 different plasmids, <SPAN class="lightblue">3 consecutive transformations</SPAN> are needed. Since we have 3 different GECOs, in the end we will have <SPAN class="lightblue">3 different variants</SPAN>, all containing the CCH1-MID1 calcium channel but each with a different color of GECO.</P><P>The order in which plasmids are introduced can be varied. The route requiring the least transformations introduces first the CCH1 and MID1 proteins and finally the 3 different GECOs. Doing so, one can obtain all 3 variants by using 5 transformations, if the GECOs are first introduced the CCH1 and MID1 proteins, they need to be introduced separate to every GECO strain; this results in <SPAN class="lightblue">9 transformations in total</SPAN>. Even though the second approach is more cumbersome, trying other routes in parallel provides a backup in case certain transformations fail.</P><P>Selection of successful transformants is done by making use of <SPAN class="lightblue">auxotrophic markers</SPAN>, a different one for each vector. The INVSc1 strain we use for transformations is auxotrophic in <SPAN class="lightblue">histidine, leucine, tryptophan and uracil</SPAN>. These usually have to be added to the medium for the yeast to survive. The uptake of a vector however, will restore the ability of the cell to synthesize the missing amino acid or nucleic acid, and therefore it can survive in a selective medium.</P><H3><SPAN class="mw-headline" id="Compatibility_of_yeast_and_device">Compatibility of yeast and device</SPAN></H3><P>The genetically modified S. cerevisiae cells will be put to the test in our <SPAN class="lightblue">home-made</SPAN><A href="/Team:TU-Eindhoven/LEC/Device" title="Team:TU-Eindhoven/LEC/Device">device</A>, designed for providing electrical stimuli to the yeast cells. These tests are done in a single bath filled with SC medium containing nutrients for our genetically modified yeast and additional free Ca<SUP>2+</SUP> ions. After <SPAN class="lightblue">electrical stimulation</SPAN> of yeast at different positions in the device, optical signals will be expected to be visible to the naked eye. In a more <SPAN class="lightblue">sensitive analysis</SPAN>, isolated protein will be characterized on a <SPAN class="lightblue">spectrophotometer</SPAN>.</P><H3><SPAN class="mw-headline" id="GECOs">GECOs</SPAN></H3><P>Real-time imaging of biochemical events inside living cells is important for understanding the molecular basis of physiological processes and diseases<A href="#ref_merkx" name="text_merkx"><SUP>[3]</SUP></A>. Genetically encoded sensors based on fluorescent proteins (FPs) are frequently used for <SPAN class="lightblue">molecular recognition</SPAN>. In this iGEM project we use fluorescent proteins to provide the light in our display. More specifically we make use the so called GECOs.</P><P>Fig. 2 Cartoon representation of the calcium dependence</P><P>A GECO is a protein which is fluorescent in the presence of Ca<SUP>2+</SUP><A href="#ref_zhao" name="text_zhao"><SUP>[2]</SUP></A> (Fig. 2). There are two important classes of <SPAN class="lightblue">genetically encoded Ca<SUP>2+</SUP> indicators</SPAN>. One is called the Forster Resonance Energy Transfer (FRET)-based cameleon type<A href="#ref_miyawaki" name="text_miyawaki"><SUP>[4]</SUP></A> and the other one is called the single Green Fluorescent Protein (GFP) type<A href="#ref_nakai" name="text_nakai"><SUP>[5]</SUP></A>. The GECO protein belongs to the single GFP type. Research has shown that Ca<SUP>2+</SUP> indicators targeted to the E. coli periplasm can be shifted toward the Ca<SUP>2+</SUP>-free or Ca<SUP>2+</SUP>-bound states by <SPAN class="lightblue">manipulation of the intracellular Ca<SUP>2+</SUP> concentration</SPAN><A href="#ref_zhao" name="text_zhao"><SUP>[2]</SUP></A>. Robert E. Campbell et al. named these Ca<SUP>2+</SUP> calcium sensors GECOs. <SPAN class="lightblue">R-GECO, G-GECO and B-GECO</SPAN> emit red, green and blue light respectively, each with their own distinct excitation and emission spectrum (Fig. 3).</P><P>Fig. 3 Excitation and emission spectra of the GECO's</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P><UL><LI><A href="#text_Iida" name="ref_Iida">[1]</A>  K. Iida, et al., Essential, completely conserved glycine residue in the domain III S2S3 linker of voltage-gated calcium channel α1 subunits in yeast and mammals, Journal of Biological Chemistry 282: 25659-25667, (2007)</LI><LI><A href="#text_zhao" name="ref_zhao">[2]</A> Y. Zhao, et al., An expanded palette of genetically encoded Ca2+ indicators, Science 333: 1888-1891, (2011)</LI><LI><A href="#text_merkx" name="ref_merkx">[3]</A> L. Lindenburg and M. Merkx, Colorful calcium sensors, (2012)</LI><LI><A href="#text_miyawaki" name="ref_miyawaki">[4]</A> A. Miyawaki, et al., Nature 338, (1997)</LI><LI><A href="#text_nakai" name="ref_nakai">[5]</A> J. Nakai, M. Ohkura, K. Imoto, Nat. Biotechnol., (2001)</LI></UL></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:TU-Eindhoven/LEC/LabTheory">http://2012.igem.org/Team:TU-Eindhoven/LEC/LabTheory</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Eindhoven/LEC/LabTheory" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Eindhoven/LEC/LabTheory" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Eindhoven/LEC/LabTheory&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Eindhoven/LEC/LabTheory&amp;oldid=229893" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>