"
Page
Discussion
View source
History
teams
Log in
 
Team:TU-Eindhoven/Lab test
From 2012.igem.org
 
Introduction
Home page
Background
information
About us
Contact us
Our photos
Project
Project description
Lab approach
Device information
Initial model
Yeast model
Lab results
Our BioBricksTM
Our
accomplishments
Considerations
Public outreach
Safety questions
Other thoughts
Future applications
Human interest
Further
developments
Notebook
Before week 1
Week 1
Week 2
Week 3
Week 4
Week 5
Week 6
Week 7
Week 8
Week 9
Week 10
Week 11
Week 12
After week 12
Lab protocols
Partners
ICMS TU/e
Team collaborations
Attributions and Contributions
Our sponsors
GECO protein expression and isolation
The three GECOs have been expressed in E. coli BL21 to yield a large amount of the proteins for characterization. Cells were cultured in large Erlenmeyer flasks. The results were visually impressive: After lysis the flasks turned vibrantly red and green respectively. The blue GECO however looked much the same as the green GECO.
More information about the GECOs can be found at Our BioBricks .
Yeast transformation
A weekly item on our schedule was yeast transformation. It takes about a week to complete a transformation, that is, to add one plasmid to an existing strain variant. Because success is not guaranteed we choose to introduce plasmids in various orders in parallel. This resulted in many variants that we assigned a unique number for convenience. The same number was used on plates, cultures and cryostocks. In total 49 variants were made.
To reach the aim of this project and prepare a multi-colored screen, three kind of cells, each with a different color, are needed. Therefore, one single plasmid strain would have to consist of both calcium channel parts MID1 and CCH1 sequences and one of the fluorescent GECO protein sequence. Each of these DNA sequences, coding for the three different parts, is transformed into a yeast plasmid (InvSC1). Moreover, each of these sequences contain a small sequence coding for an amino acid, causing the yeast strain to be able to synthesize this specific amino acid. The synthesis of this specific amino acid will allow strain selection by retaining this amino acid from the culture media. Cells containing the  coding CCH1, MID1 and GECO sequence are able to synthesize leucine, uracil and tryptophan respectively.
Looking at Scheme 1, which represents the transformations needed to come to the desired yeast strain, it seems and is logical to start with the MID1 or CCH1 vectors. By doing so, less transformations can be performed due to the transformation intensive introduction of the three different GECO proteins at last. To make sure that the transformations would lead to the desired strain, it was decided to start the first step with transforming all possibilities. When step one was transformed successfully, visualized by the arrows in Scheme 1, the transformation of step two was performed and so on.
Scheme1: Transformation Scheme
The several transformation steps are clarified in Scheme 1. The arrows represent a successful transformation as at the same time the lack of an arrow represents a transformation failure. Of course, not every transformation was needed to obtain the desired strain. E.g. in step one the transformation of the R-GECO failed, however, the strain containing the R-GECO was obtained in step two due to other successful transformations.  As can be seen, in step 2 the first co-transformation are performed. This brought some complications because the regular transformation protocol did not work for these co-transformations. For the co- and co-co-transformations (performed in step 3) we used another high-efficiency transformation protocol which worked out successfully. After 3 steps of transformations, a couple of different transformation paths reached the three desired strains a complete calcium channel and one of the three GECO proteins.
Initially, yeast transformations failed or yielded only several colonies, a lot less than expected. A review of transformation protocols showed that the shock protocol we used should be more efficient than electroporation or other common methods. We tried to transform another yeast strain with our plasmids which yielded the expected high transformation efficiency. Unfortunately that strain was not compatible with the auxotrophic markers on the plasmids we wanted to introduce. Our problem was remedied by using more plasmid DNA for the transformation, as described in our modified yeast transformation protocol .
Device tests
The device was tested with yeast containing all three plasmids, thus over expressing channels and expressing GECOs. A quick inspection with the naked eye did not show any sign of visual light coming off the device, not even in a dark room. We think that the yeast was not concentrated enough to show a clear response.
Detection of GECO protein in yeast
Fluorescent proteins like our GECOs can be detected by spectrophotometry if the intensity is insufficient to be observed by the naked eye. If many samples have to be measured these can be put onto a 96-wells plate and measured all at once in a special type of spectrophotometer called a plate reader. A plate reader is also convenient for use with small sample volumes, which was beneficial in our case, since it allowed us to concentrate the yeast from a large culture volume into a small volume through centrifugation and obtain a better measurement.
The expression of GECOs in yeast did not yield obvious fluorescence when stimulated in the device . To check if the protein was actually present in our cells, we made spectrograms of cells concentrated to 1.25 x 108 cells/ml, which amounts to an OD600 of approximately 125.
Samples of yeast on a 96-wells plate.
We wanted to demonstrate the presence of GECOs in yeast cells carrying GECO, CCH1, and MID1 plasmids, cultured on induction medium. For this we used a plate reader to detect the possible fluorescence of any GECO protein present within the cells. To exclude other sources of fluorescence we also measured yeast cells without either GECOs, without over expressed CCH1-MID1 or without any plasmids. Furthermore we compared cells carrying GECO- and CCH1-MID1 plasmids cultured in 2% galactose induction medium with cells cultured in 2% glucose growth medium, which represses induction.
Emission spectra of INVSc1 yeast cells carrying one GECO plasmid and CCH1 and MID1 plasmids cultured on induction medium (solid lines) or culture medium which represses induction (dashed lines). Excitation peak was centered around 495 nm with a bandwidth of 5 nm.
When excited at 495 nm, around the peak in the excitation spectrum of G-GECO, both cells carrying G-GECO and cells carrying B-GECO plasmids cultured in induction medium fluoresced, whereas cells carrying R-GECO cultured in induction medium did not fluoresce. Cells cultured in medium that represses the expression of GECO protein did not fluoresce. Furthermore the spectra of G-GECO and B-GECO seem to overlap, which confirms our earlier observation that spectra of G-GECO and B-GECO protein produced in E. coli overlapped.
Emission spectra of INVSc1 yeast cells carrying one GECO plasmid and CCH1 and MID1 plasmids cultured on induction medium (solid lines) or culture medium which represses induction (dashed lines). Excitation peak was centered around 565 nm with a bandwidth of 5 nm.
When excited at 565 nm, around the peak in the excitation spectrum of R-GECO, only cells carrying R-GECO plasmids cultured in induction medium fluoresced, whereas cells carrying G-GECO or B-GECO plasmids cultured in induction medium did not fluoresce. Cells cultured in medium that represses the expression of GECO protein did not fluoresce.
There is a clear difference in spectra between R-GECO and G-GECO, which means they can be used together in a multi-color screen without disturbing each other’s emission. The difference between G-GECO and B-GECO is very small, perhaps another excitation wavelength could better separate the emission spectra and make them better suited for simultaneous use in the same screen.
References
Retrieved from " http://2012.igem.org/Team:TU-Eindhoven/Lab_test "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
