"
Page
Discussion
View source
History
teams
Log in
 
Team:UANL Mty-Mexico/Safety/genetic modifications
From 2013.igem.org
Carousel Template for Bootstrap
Home
Judging
Team
Members
Profile
Attributions
Project
Abstract
Applications
In our Community
Full Description
Modeling
Results
At a glance
Parts
Wetlab
Notebook
Protocols
Human Practices
Overview
Activities
Exhibitions
Workshops
Genobiotec13
Media
Collaboration
Safety
Overview
Organism Description
Genetic Modifications
Potential Risks
Laboratory
Stability Test
ID Tags
Safety
Genetic Modifications
The project “Thermocoli” consists in a circuit of transcription factors and reporter genes, some of which are under the post-transcriptional regulation of thermoregulable RNA elements, also known as RNA thermometers. These genes are arrange in a circuit constructed in such a way that three distinguishable states should emerge, characterized by the expression and repression of two different reporter fluorescent proteins.
In this work, we intend to regulate the expression not only reporter proteins, but also the expression of at least one transcription factor using RNA thermometers. If proved possible, the RNA-thermoregulation of transcription factors will widen the spectrum of genetic circuit topologies that can be used for a number of purposes, most remarkably, the research of basic cellular processes and the replacement of chemical inducers for industrial-scale processes. Our system can be subdivided in five different modules that can be characterized separately. Figure 1 shows those submodules labeled from A to E. Here we describe each one of them:
Figure 1. The submodules of the iGEM UANL 2013 project. Description in text.
This synthesized construction has a gene that codes for the green fluorescent protein (GFP) variant that has an LVA degradation tag; the gene is under the regulation of a pLac promoter and a thermolabile ribosome binding site (RBS) that should allow for translation only at temperatures above 32°C. At the 3’ end of the gene dubbed “GFP-LVA” are two consecutive transcription termination regions.
The GFP sequence is similar to the one found in part BBa_K145015 and is a mutant derived from the green fluorescent protein found in jellyfish (Andersen et al., 1998) . The LVA tag that is in frame with the coding sequence of the TetR gene, is actually a derivation of the C-terminal AANDENYALAA tagging sequence, which makes proteins susceptible to degradation by the ClpX and ClpA proteases (McGiness, Baker and Sauer, 2006) .
After testing different variants for the last three amino acid residues in the C-terminal part of the peptide, it was found that AANDENYALAA variants show different degradation rates. We chose the LVA because is the most commonly employed in iGEM projects and we attached it to the coding sequence of GFP using a spacer (or scar) similar to the one present in part BBa_J04450. The transcription termination sequences employed are two: part BBa_B0010 (derived from T1 of E. coli rrnB) and part BBa_B0012 (derived from TE of bacteriophage T7). Both sequences are stem loops that hinder the processivity of E. coli RNA polymerase.
A construction that has a gene coding for the LacI transcription factor, which also has a LVA degradation tag. The LacI gene is transcribed through a constitutive promoter and a thermolabile RBS with an optimal translation temperature of 37°C. Two termination sites were added at the 3’ end of the gene.
The constitutive promoter corresponds to the sequence of part BBa_J23119, the most potent member of a family of promoter variants obtained from a combinatorial library. Transcription factor LacI is the same one found in E. coli lac operon and the sequence employed is the same as the one from part BBa_C0012, which is the same as the one employed in the Elowitz and Leibler 2000 . study as stated in the “design” section of the registry entry for part BBa_C0012, this LacI variant differs from the wild-type in that the GTG start present in nature was changed to an ATG start.
This is a synthesized construction that also codes for an LVA-tagged transcription factor, which in this case is protein TetR. The gene is under the regulation of a pcI promoter, which is in turn regulated by transcription factor cI and a generic (non-thermolabile) RBS.
The pcI promoter is derived from the pR promoter of bacteriophage lambda and is registered as BBa_R0051 in the biological parts registry.
The RBS sequence is derived from part BBa_B0034, which features in the 2010 study by Elowitz and Leibler 2000 . This RBS is defined as the standard for RBS activity and is assigned an efficiency of 1.0.
The coding sequence of TetR was derived from part BBa_C0040. TetR is a member of a family of transcriptional repressors present in gram-positive, alpha-,beta-, and gamma-proteobacteria, cyanobacteria and archea. The function of a TetR family member can be quite complex. However, the TetR sequence we are using should have a straightforward inhibiting activity upon the pTet promoter region, which is alleviated by the binding of tetracycline, or its analogue, aTc, to TetR. (Ramos, et al., 2005) . The TetR sequence we are using is similar (as per a BLAST search) to other plasmids employed in Synthetic Biology projects.
A construction similar to A., but that has a red fluorescent protein-coding gene mCherry instead of GFP. This mCherry gene also has an in-frame degradation tag. However, the mCherry is under the regulation of a pTet promoter and a thermolabile ribosome binding site with an optimal translation temperature of 37 °C. Two transcription termination sites were also added at the 3’ end of the gene.
The pTet promoter is the binding site for TetR, the transcription repressor, and it’s identical to the sequence found in part BBa_R0040. Promoter pTet shows constitutive transcriptional activity until TetR binds to it. The sequences for TetR and the pTet promoter are similar to the ones present in E. coli Tn10 (tet) operon (Lutz and Bujard, 1997).
This construction is actually part BBa_K098995 and was the only construction which we didn’t synthesize. It codes for a variant of E. coli transcription factor cI which is thermolabile. In principle, at 42°C this thermolabile cI should denature and stop its inhibiting action upon the pcI promoter. The cI gene is transcribed through a constitutive promoter and a generic (non-thermolabile) RBS. Two transcription termination sites were added at the 3’ end of the gene.
References
Ramos, JL, et al., (2005), The TetR family of transcriptional repressors, Microbiol Mol Biol Rev, 69(2):326-56.
McGiness, KE; Baker, TA and Sauer, RT, (2007), Engineering Controllable Protein Degradation, Mol. Cell., 22, 701-707.
Elowitz, MB and Leibler, S, (2000), A synthetic oscillatory network of transcriptional regulators, Nature, 20;403(6767):335-8.
Lutz, R, and Bujard, H, (1997), Independent and Tight Regulation of Transcriptional Units in Escherichia Coli Via the LacR/O, the TetR/O and AraC/I1-I2 Regulatory Elements, Nucleic Acids Research, 25 (6): 1203-1210.
Andersen, JB, et al., (1998), New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria, Applied and Environmental Microbiology, vol. 64 no. 6 2240-2246
Retrieved from " http://2013.igem.org/Team:UANL_Mty-Mexico/Safety/genetic_modifications "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
