"
Page
Discussion
View source
History
teams
Log in
 
Team:UANL Mty-Mexico/Project/Overview
From 2013.igem.org
Carousel Template for Bootstrap
Home
Judging
Team
Members
Profile
Attributions
Project
Abstract
Overview
Applications
Full Description
Safety
Abstract
Overview
Applications
Full Description
Math
Introduction
Equations
Fitting
Proposal
Conclusions
Results
Parts
Wetlab
Notebook
Protocols
Outreach
Overview
Genobiotec13
SynBio Tunnel
Talks
Surveys
RNATs
Synthetic RNATs
Project circuit
Types of natural RNATs
References
RNA thermometers
RNA thermometers (RNATs) are RNA sequences that range from 40 to more than a 100 nucleotides commonly found in the 5' untranslated region of some genes and that regulate in cis their translation without the need of other factors [ Kortmann and Narberhaus, (2012) ; Narberhaus, (2009) ]. These RNAT sequences show certain three dimensional structures, some of which interact with the ribosome binding site (RBS) of their regulated genes and hinders the proccessivity of the ribosome complex at certain temperatures. The dynamics of the formation of these structures is temperature dependent and is the basis of the regulation of the translation rate of a given transcript [ Chowdhury, S., et al.,(2006) ; Narberhaus, F., et al.,(2006) ].
Functional RNAT have been found in different organisms, mainly pathogenic bacteria, and many others have been predicted in almost everyfrom a number of bioinformatic studies. They have been found to regulate the expression of virulence factors, heat and cold shock response factors and even proteins involved the development of some bacteriophages.
Their apparent widespread presence in living organisms has made RNATs attractive for some applications, specially the ones related to the replacement of chemical inducers and for the development of new drugs.
However, from the experience of those who have been working extensively with RNAT in the later years, the accurate bioinformatic prediction of functional RNAT has proven to be an exceptionally difficult task; the reasons for this are pointed to be the poor sequence conservation observed among RNATs and the gaps in our current understanding of the RNAT function, their structural diversity and the effect of other regulatory sequences far from the RBS region [ Kortmann and Narberhaus, (2012) ; Waldminghaus, et al., (2007) ].
Types of natural RNA thermometers   Back to top
Although RNATs show almost no sequence similarity among them, a number of structural features can be used to classify them. Here we enlist the most described RNATs structural groups described to date [ Kortmann and Narberhaus, (2012) ]:
ROSE.- ROSE stands for "Regulation Of heat Shock Expression". ROSE elements are 60 to >100 nucleotide sequences found upstream of heat shock proteins. They have been found to be conserved in alpha and gamma-proteobacteria. Among the structural features of the ROSE element family are: a) their folding in 2 to 4 stemloop structures; b) a short conserved sequence (UU/CGCU) near the Shine-Dalgarno sequence; and c) the presence of a number of non-cannonical base interactions (the G83-G94 pair; a triple bair among U96-C80-C81; the U79-U97; and the interaction of the AUG codon and C71, G72 and U73. Functional ROSE elements have been found in E. coli (rpoH and ibpA) and B. japonicum (hspA).
FourU elements.- these elements are characterized by a short motif composed of four uridines that pair with the Shine-Dalgarno region and is embedded in a hairpin that shows temperature-induced conformational changes. FourU elements have only one A-G non-cannonical base interaction. Among the structural features that characterize FourU elements are a) the A-G pair and b) the G34-C46 pair that regulates melting. Functional FourU elements have been described in Salmonella (agsA) and Yersina pseudotuberculosis (lcrF).
Synechocystis hsp17 element.- with a length of 46 nucleotides, this is the shortest RNAT described so far. The distinctive structural features essential for the function of this element are a) the pairing of a UCCU sequence with the AGGA in the Shine-Dalgarno sequence and b) the presence of two loops in its stems.
Coding region spanning RNATs.- RNATs are not exclusively found in the 5'UTR of genes; they can also span into the coding region and even be intergenic. Functional coding region spanning RNATs have been found in E. coli (rpoH), phage lambda (cIII) and Lysteria monocytogenes (prfA).
Cold shock RNATs.- cold shock RNATs also depend on the dynamics of the folding of different loops, but in contrast to heat shock RNATs, the conformation that prevents the binding of the ribosome is found at high temperatures, while at low temperatures, the RNAT folds into a conformation that allows for the ribosome to proceed. An example of a cold shock RNAT is the element found upstream and inside the coding region of E. coli gene cspA.
Synthetic RNATs
The discovery of new RNATs has relied on a mixed approach that involves bioinformatics and experimental validation, as well as approaches that involve mutational libraries, synthetic constructions and directed evolution. Likewise, de novo designed synthetic RNATs have been proved to regulate genetic expression at different temperature ranges. Successful approaches are based on structural design of sequence loops that mask the Shine-Dalgarno (SD) sequence inside the RBS.  Specifically, Neupert et al. (2008) achieved to build simple RNATs that consist of a single small stem-loop structures containing the RBS and blocking the SD sequence. Additionally, 2011 iGEM team BYU_Provo also worked with the design of synthetic RNATs, managing to select sequences sensitive to narrow temperature ranges.
Our project builds upon these two works to prove that RNATs can also be employed to effectively regulate the expression of transcription factors in synthetic circuits. Even when the naturally found RNATs usually regulate the expression of transcription factors, the synthetic constructions made so far have focused mainly on the characterization of the effect of a given RNAT by placing a reporter protein (LacZ or a fluorescent protein) directly downstream. On the other hand, our proposal points at possible applications for the circuit topologies that would be made feasible through the combination of transcription factors and temperature sensitive translational regulation.
The ThermoColi project  
Back to top
Our genetic circuit combines two synthetic RNATs and three transcription factors, through which we intend to gain temperature-mediated control over the transition of three discrete states, signaled by the expression of distinct fluorescent proteins. The figure below shows the topology of the proposed circuit. 
At temperatures below 32ºC, called state 0, no reporter protein is produced. Above 32ºC, the first RNAT is melted thus allowing the translation of GFP transcripts (figure 1a) and the system enters state 1. Note that, despite the fact that GFP transcription is under the regulation of a Lac promoter, LacI is not being produced so far as its translation is prevented by a different RNAT. 
When the system reaches 37ºC, mCherry’s RNAT melts allowing it to be translated (figure 1b) and the system enters state 2. At this point, LacI repressor -whose translation is regulated by the same RNAT- is produced, therefore repressing GFP. Again, note that even if mCherry is transcribed from a Tet promoter, Tet repressor’s transcription is blocked by cI represor. 
Finally, if the system reaches a temperature above 42ºC, the system is shut down (figure 1c). This is achieved because the cI repressor herein used is thermolabile, and above 42ºC it is no longer able to bind its operator and repress transcription. Hence, both LacI and Tet repressors are again produced blocking the transcription of GFP and mCherry, respectively.  
References  
Back to top
Kortmann J, Narberhaus F (2012) Bacterial RNA thermometers: molecular zippers and switches. Nat Rev Micro, 10:255-265
Saheli Chowdhury, Christophe M, Allain F, Franz Narberhaus F (2006) Molecular basis for temperature sensing by an RNA thermometer. The EMBO Journal, 25:2487-2497
Neupert J, Karcher D, Bock R (2008) Design of simple synthetic RNA thermometers for temperature- controlled gene expression in Escherichia coli. Nucleic Acids Res, 36:e124.
BYU Provo iGEM 2008 https://2011.igem.org/Team:BYU_Provo
Retrieved from " http://2013.igem.org/Team:UANL_Mty-Mexico/Project/Overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
