<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Technion-Israel_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Technion-Israel/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Technion-Israel/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Technion-Israel/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Technion-Israel/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Technion-Israel/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Technion-Israel/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Safie by Technion-Israel</TITLE></P><DIV id="header-wrapperP" class="wrapperP"><DIV id="header"><DIV id="logo"><H1 style="color: #ebebeb">The Project</H1><H2 style="color: #ebebeb">&quot;Success consists of going from failure to failure without loss of enthusiasm.&quot;--Winston Churchill</H2></DIV><NAV id="nav"><UL><LI id="parent"><A href="https://2014.igem.org/Team:Technion-Israel">Home</A></LI><LI id="parent"><A href="https://2014.igem.org/Team:Technion-Israel/Project">Project</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Project#idea">The idea</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Project#howitworks">How it works</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Project#alpha">Alpha System</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Project#beta">Beta System</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Project#rna">RNA splint</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Project#azo">Azobenzene</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Project#hk">Histidine Kinase</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Project#newmethod">New Standard</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Project#protocol">Protocols</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Project#notebook">Lab Notebook</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Project#safety">Safety</A></LI></UL><LI id="parent"><A href="https://2014.igem.org/Team:Technion-Israel/Modeling">Modeling</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Modeling#whyworks">Why it should work</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Modeling#whyfail">Why it should fail</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Modeling#splint">RNA Splint</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Modeling#biofilm">Synthetic BiofilmFormation</A></LI><LI id="parent"><A href="https://2014.igem.org/Team:Technion-Israel/Experiments">Experiments</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Experiments#gate1">Gate 1</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Experiments#gate2">Gate 2</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Experiments#rna">RNA Splint</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Experiments#pompc">Pompc-RFP</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Experiments#taz">TaZ</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Experiments#mcherry">mCherry</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Experiments#amilcp">amilCP</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Experiments#azo">Azobenzene</A></LI><LI id="parent"><A href="https://2014.igem.org/Team:Technion-Israel/HP">Policy &amp; Practices</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/HP#3lang">SynBio in 3 Languages</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/HP#highschool">iGEM High Schools</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/HP#lectures">SynBio Lectures</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/HP#collab">Collaborations</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/HP#art">Art</A></LI><LI id="parent"><A href="https://2014.igem.org/Team:Technion-Israel/Team">The Team</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Team#members">Members</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Team#mentors">Mentors</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Team#gallery">Gallery</A></LI><LI id="child1"><A href="https://igem.org/Team.cgi?id=1343" target="_blank">Official TeamProfile</A></LI><LI id="child1"><A href="https://www.facebook.com/iGEM2014.Technion" target="_blank">Official FacebookPage</A></LI><LI id="parent"><A href="https://2014.igem.org/Team:Technion-Israel/Support">Special Thanks</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Support#headstart">Headstart Supporters</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Support#attributions">Attributions</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Support#sponsors">Sponsors</A></LI><LI id="parent"><A href="https://2014.igem.org/Team:Technion-Israel/Judging">Judging</A></LI><LI id="child1"><A href="https://igem.org/2014_Judging_Form?id=1343" target="_blank">Judging Form</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Judging#biobrick">BioBricks</A></LI><LI id="child1"><A href="https://2014.igem.org/Team:Technion-Israel/Judging#results">Results</A></LI></NAV></DIV></DIV><DIV class="wrapper style2"><DIV class="title" id="idea">The Idea</DIV><DIV id="main" class="container"><CENTER><H2 style="font-size:2em; line-height:1.25em">Using a network of E. coli to form a smart materialfor low concentration detection!</H2><DIV style="font-size:1.2em; line-height:1.75em;"><P>Bio-detectors have been a big part of the iGEM projects ever since the competition first started, it's easy to see why: One of the simplest systems to build using our current tools for synthetic biology is a simple Input→Output &quot;linear&quot; (Promoter→Signaling Gene) bio-detector but this method has a major flaw:
In order to get a detection signal that's visible to the naked eye, we must have a LOT of bacteria change color (or any other signal). With the linear approach we find ourselves <B>needing high concentration of the detected material</B> for our system to be effective!
Now, while this issue is far from new and various teams have already tried to tackle this exact problem before, our team worked for a year on a new approach utilizing things like quorum sensing for inter-bacteria communication and signal amplification which is possible thanks to our creation of a synthetic bio-film using a revolutionary organic molecule called Azobenzene, resulting in what we refer to as a</P></DIV><H2 style="box-shadow:0px 0px 1px 1px rgba(0,0,0,0.25); border-radius:0.35em; font-weight:700; padding:1em; font-size:2em;">'smart, self assembling material'</H2></CENTER></DIV></DIV><DIV class="wrapper style3"><DIV class="title" id="howitworks">How It works</DIV></DIV><DIV id="footer-wrapper" class="wrapper"><DIV class="title" id="alpha">Alpha System</DIV><DIV id="footer" class="container"><HEADER class="style1info"><CENTER><H2 style="box-shadow:0px 0px 1px 1px rgba(255,255,255,0.25); border-radius:0.35em; font-weight:700; padding:1em;">Alpha System</H2><P style="font-size: 1.2em;">The Alpha System is the preliminary design we planned for a biosensor that can detect various materials at low concentrations.
The main goal of our design is to make the bio-sensor capable of detecting harmful substances at far lower concentrations than existing bio-sensors. To do this, we designed a system which produces a chain reaction resulting from the input of a single molecule. This system utilizes a Cell-to-cell communication channel called quorum sensing, to make it possible for a cell which sensed a single molecule of the material detected to “inform” the surrounding cells of the existence of this toxin, causing them to produce a signaling molecule, and then inform their neighbors and so on, creating a chain reaction.
Unlike other biosensors, our system is meant to be as versatile as possible- with a minor change at the genetic level it can detect molecule A instead of molecule B. That saves us the need to construct an entirely different detector for every dangerous material out there.
The alpha system consists of three gates.
Each gate plays an important role in the detection process inside the cell, but the system as a whole is also based on communication between bacteria via quorum sensing.</P><H2><U>Gate 1</U></H2><H1>Built and Biobricked <A href="http://parts.igem.org/Part:BBa_K1343000" target="_blank"><U>(BBa_K1343000)</U></A></H1><H2>The goal of the first gate is to produce the quorum sensing molecule, AHL, when a material of interest is present in the bacterias' environment. As a result, this part can be replaced by any bio-sensor which uses AHL production as its output.</H2><P style="font-size: 1.2em;">The construct includes a changeable promoter for the detection of the material of interest. In the example above the promoter is Ptet which is repressed by tetR; when tetracycline or its analog aTc is present, tetR is removed
When the material is present it will get into the cell, attach to the promoter and the transcription of the LuxI gene will begin. LuxI gene encodes for an AHL synthase. AHL is a signal molecule that can diffuse freely into the cell and out.
The NheI restriction site was added to the construct so that the promoter activating the gate can be easily changed. EcoRI restriction site is found at the beginning of the prefix, and by a simple restriction reaction with the two enzymes we could take the promoter out and replace it with a different one.
                        </P><P><B>But why do we want our bacteria to produce AHL molecules? How does it help us to see if a toxin or an allergen is present in our sample?</B>
Continue reading about gate 2 and 3 to find out</P></CENTER><CENTER><H2><U>Gate 2</U></H2><H1>Built and Biobricked <A href="http://parts.igem.org/Part:BBa_K1343003" target="_blank"><U>(BBa_K1343003)</U></A></H1><H2>The gate serves as a sensitivity tuner, allowing us to control on the timing of the measurement. When we want to start our measurement we just need to add IPTG to the bacteria solution.</H2><P style="font-size: 1.2em;">The gate consists of the Plac promoter and the LuxR gene. When IPTG is added to a sample containing our bacteria, LuxR gene is transcribed. LuxR is a transcriptional activator that binds directly to the AHL molecules, so that a LuxR-AHL complex is formed.
                        </P><P><B>And what happens in our system when the complex LuxR-AHL is formed?</B>
Read the next sections about gates 3 to find out</P></CENTER><CENTER><H2><U>Gate 3</U></H2><H1>Built alternative construct</H1><H2>Gate 3 enables us to see if our material of interest is present in a random sample by producing GFP and amplifying the signal.</H2><P style="font-size: 1.2em;">This construct is the final part of our alpha system. 
The promoter of the gate is the Plux promoter which undergoes activation when the complex LuxR-AHL binds to it. If the promoter is activated we get the production of a reporter gene (GFP) and the LuxI gene. 
As we mentioned in the section on gate 1, LuxI transcription results in AHL molecules.</P><P style="font-size: 1.2em;">We experienced many difficulties in building this gate.  Therefore we built an alternative Gate 2-Gate 3 construct: Pcat_luxR_Plux_mCherry_luxI</P><H3 style="font-size: 1.5em; line-height:1.75em">In conclusion, the Alpha System is a 3 gate construct that produces GFP when a substance is detected and signals the other cells around it to produce GFP as well</H3></CENTER></HEADER></DIV></DIV><DIV class="wrapper style2"><DIV class="title" id="beta">Beta System</DIV><DIV id="main" class="container"><CENTER><H2 style="box-shadow:0px 0px 1px 1px rgba(0,0,0,0.25); border-radius:0.35em; font-weight:700; padding:1em; font-size:2em;">Beta System</H2><P style="line-height:1.75em;">
According to the model <A href="https://2014.igem.org/Team:Technion-Israel/Modeling#whyfail">&quot;Why it Should Fail&quot;</A> of the Alpha System, we can see that it has some problems. We decided to test new methods to reduce the noise in our system. One idea was a new design – the Beta System, inspired by the noise reduction mechanism described by Goni-Moreno and Amos. (Goni-Moreno &amp; Amos, 2012). 
We used a double repression Toggle Switch similar to that described by Gardner et al. (Gardner, Cantor, &amp; Collins, 2000)), to filter the inputs of our system. This makes the cell-to-cell communication more accurate, while affording them an internal memory capacity.
This system consists of three main circuits:<B>(1)</B> The Computation Circuit (which acts as the CPU and determines whether or not to activate the Toggle Switch).<B>(2)</B> A Toggle Switch (which acts as the internal memory bank, by keeping the system active or inactive over long periods of time).<B>(3)</B> A Signal Circuit (which acts as the &quot;antenna&quot; used for broadcasting and receiving cell to cell signals).</P><H1 style="font-size: 1.2em;">Computation Circuit</H1><P style="line-height:1.75em;">
The Computation Circuit has been programmed to function as an OR gate:
Its first part consists of the gene LacI, which activates the Toggle Switch, and whose synthesis is regulated by the changeable promoter Pchangeable. This is the promoter activated by the substance we want to detect.
Its second part consists of the gene LacI, whose synthesis is regulated by the promoter Plux, which is activated by LuxR-AHL dimers, which result from the cell-to-cell communication.
This circuit determines whether or not to activate the toggle switch, according to the external signal (the substance being detected by the Pchangeable promoter), and the signals coming from the surrounding cell population (which activate the Plux promoter).</P><H1 style="font-size: 1.2em;">Toggle Switch</H1><P style="line-height:1.75em;">
The Toggle Switch is how the cells &quot;remember&quot; data. As a default, the switch stays &quot;off&quot;, until it receives a signal (in the form of LacI) from the Computation Circuit telling it to switch “on”. Once the Toggle Switch is &quot;on&quot; it remains activated indefinitely, unless it receives an external signal (IPTG) to switch “off”.
We based our system on the Toggle Switch from Gardner et. al. (Gardner, Cantor, &amp; Collins, 2000):
&quot;The toggle switch is composed of two repressors and two constitutive promoters [LacI and CI]. Each promoter is inhibited by the repressor that is transcribed by the opposing promoter. We selected this design for the toggle switch because it requires the fewest genes and cis-regulatory elements to achieve robust bistable behavior&quot;.
The system is switched &quot;on&quot; when the computation circuit produces the repressor LacI, which binds to Plac, thus inhibiting the expression of CI.
Without the presence of CI, the promoter PCI is uninhibited, and LacI and T7 RNA polymerase are synthesized. LacI inhibits the expression of CI, keeping our system in an &quot;on&quot; state, while the T7 RNA polymerase activates the Signal Circuit.
The system can be switched “off” by using IPTG, which induces Plac, causing CI expression and therefore repression of PCI, LacI and T7 RNA polymerase.</P><H1 style="font-size: 1.2em;">Signal circuit</H1><P style="line-height:1.75em;">
The Signal Circuit is designed to facilitate cell-to-cell communication, which is based on the diffusion of a small quorum sensing molecule called AHL (N-Acyl Homoserine Lactose). AHL can diffuse through cell walls. For effective cell-to-cell communication, the cells must have two things:<B>(1)</B>	LuxR, a protein which binds to AHL (a &quot;receiver&quot;).<B>(2)</B>	A genetic gate which can produce large amounts of AHL (an &quot;antenna&quot;).<B>This circuit contains two parts:</B>
The first part consists of the promoter Pcat, a constitutive promoter, which regulates LuxR expression in excess at all times. The LuxR protein can bind to AHL produced by neighboring cells, activating the Computational Circuit.
The second part consists of the promoter, PT7 RNA polymerase, which is controlled by the T7 polymerase synthesized by the Toggle Switch, and regulates the expression of LuxI – an enzyme that produces AHL. When the PT7 promoter is activated, it produces large amounts of AHL. This amplifies the signal produced by the toggle switch, before it is diffuses out through the lossy channel.</P><P style="line-height:1.75em;">
We believe that using the toggle switch is an improvement compared to the Alpha System, because Goni-Moreno and Amos’s analysis of the use of toggle switches for noise reduction showed positive results (Goni-Moreno &amp; Amos, 2012).
 We believe that amplified AHL production (using the T7 polymerase) would reduce the noise in our system because the error term in the Fokker-Planck equation (the equation commonly used for modeling noisy system) for this system, is reduced by a factor inversely proportional to this amplification (link to Ittai’s reference [13]).
</P></CENTER></DIV></DIV><DIV id="footer-wrapper" class="wrapper"><DIV class="title" id="rna">RNA Splint</DIV><DIV id="footer" class="container"><HEADER class="style1info"><CENTER><H2>RNA Splint for protein split and assembly - luxI as inspiration and CMR for proof of concept</H2><P>
Our system is made for the detection of substances in very low concentration. It is based on a signal transduction and amplification between the individual bacterium. The major risk in systems like this one is that a false positive result will activate the whole system, rapidly. In order to solve this problem, we came up with few methods. One of those methods is the split of a key component protein, which activates the system: LuxI. It allows the communication of an activation signal to between all of the cells in the sample. We thought it would be feasible to solve it by ensuring the production of LuxI only when a toxin is in the sample. To avoid the leakiness of the promoters we have decided to split the LuxI gene so each part will be regulated under a different promoter (of the same type). This way the probability for leakiness of the two promoters in the same cell and assembly of the two parts of the protein is very low. That way LuxI will be produced only when the cell detects a toxin.

To screen for the positive colony with the positive RNA splint and the functionality of LuxI, the detector strain assay will be done (the same screening method from gate 1)</P><H1>LuxI-Inspiration</H1><P><B>RNA splint:</B> bridges between A protein RNA and B protein RNA. The splint enables the ligation of two RNA molecules. In order for the ligation of the 2 RNA molecules to occur a T4 RNA ligase is needed. T4 RNA Ligase is the enzyme from the phage that makes this ligation between two RNA molecules, it recognizes a specific site (this will be addressed later on). This is the preferable method.<B>The design using luxI:</B></P><H1>CM Resistance</H1><P>RNA splint is an in-vitro method described for the ligation of 2 RNA molecules. (M.R. Stark, J.A. Pleiss, (2006)
For a simple and robust screen CM split is preferred over the split of luxI</P><H1>Some background about the components needed for the described system</H1><P style="text-align:left;">• Role of T4 Ligase in nature: to repair tRNA damage during the invasion of the bacteriophage (maybe cause of different anti codon usage in the phage) (“Thus, reprocessing could be yet another T4 device to adapt the translation apparatus to post-infection codon usage”) (Amitsur et al., 1987)(C. Kiong Ho, Li Kai Wang 2004)• Problem: Ribosome will get stuck because of the double strand RNA
Solution: RNA Helicase-exist in E-coli K-12• The 5’ of the mRNA is phosphorylated - it is important for the ligation• The terminator should be removed from the first part of the CM resistance, or else it will be translated in the middle of the gene-will be done by Hammerhead  Ribozyme • Chloramphenicol acetyltransferase is the protein for the resistance - There is a His residue in the C-terminus that is important for the mechanism of the enzyme so it is possible to knock the activity out by splitting the sequence• We used Rnl1 for this ligation (ligates tRNA)(T4 nucleic acid ligases: Bullard &amp; Bowater 2006)• It's enough to use 7 bases for each side (14 total) we did 20bp totally (E. Paredes et al. / Methods 54 (2011))• GTN for the Ribozyme cut, N needs to be purine (but could be any base except G)• TTC is the loop of tRNA-Lys (UUC) - Needs TTCN for the T4 Ligase• The first part ends with TTCGTC• In luxI there is TTCTC so it is supposed to be suitable for the approach• This is Type 3 Ribozyme.</P><P>For extended information follow the <A href="https://static.igem.org/mediawiki/2014/c/c5/Technion-Israel-RNA_Splint2.pdf" target="_blank">link</A></P></CENTER><P style="text-align:left; font-size:1.0em;">
1. M.R. Stark, J.A. Pleiss, (2006), RNA
2. Amitsur et al., 1987
3. C. Kiong Ho, Li Kai Wang 2004
4. NCBI, Gene ID: 948290, &lt; http://www.ncbi.nlm.nih.gov/gene/948290&gt;, updated on 12-Oct-2014
5. NCBI, GenBank: AF158101.6, <HTTP:>17.10.14
6. E. Paredes et al. / Methods 54 (2011)
7. Christian Hammann, Marcos De La Pena, (2012), The ubiquitous hammerhead ribozyme. RNA
8. William G. Scott (RNA Technologies 2010)
9. Jim Nolan, &lt; http://www.tulane.edu/~biochem/nolan/lectures/rna/ham.htm&gt; 17.10.14</HTTP:></P></HEADER></DIV></DIV><DIV class="wrapper style2"><DIV class="title" id="azo">Azobenzene</DIV><DIV id="main" class="container"><DIV id="content"><ARTICLE class="box post"><HEADER class="style1"><CENTER><H2>Azobenzene Photo-induced Magical Molecule</H2><H1>From NPs Aggregation to Synthetic Biofilm</H1><DIV id="azoimage"><P>
(1) Our bacterium has two main features - one is a sensor and the second is azobenzene attached to the LPS. When the bacterium detects a substance it changes color by producing green luciferase
(2) The light emitted from the bacterium causes the azobenzene molecules to change conformation to a &quot;sticky&quot; form
(3) The azobenzene molecules cause the bacteria to aggregate by forming bonds through azobenzene, allowing fast diffusion of quorum sensing molecules and the rest of the bacteria turn green as well
</P></DIV></CENTER><CENTER><P style="font-size:1.1em;">Azobenzene is an organic molecule which responds to light and selectively photoswitches from an extended Trans conformation to a more compact Cis conformation.</P><H1>Background</H1><P style="font-size:1.1em;">In the dark, at equilibrium, azobenzene is &gt;99% trans. Irradiation at specific short wavelength causes ~90% to switch to the cis isomer. Irradiation at longer wavelengths and/or thermal relaxation causes the molecule to return to the trans isomer.</P><P style="font-size:1.0em;">Figure1: Photochromism of azobenzene derivatives and energetic profile for the switching process.</P><H1>iGEM new azobenzene synthesis breakthrough</H1><P style="font-size:1.1em;">Most azobenzene-based photoswitches use UV light for photoisomerization. This limits their application in biological systems, where UV light be harmful to the cell. Substitution of all four ortho positions with methoxy groups in an amidoazobenzene derivative leads to a substantial (~35 nm) red shift. This red shift makes trans-to-cis photoswitching possible using green light (530-560 nm). The cis state is thermally stable with a half-life of ~2.4 days in the dark in aqueous solution. Reverse (cis-to-trans) photoswitching can be accomplished with blue light (460 nm), so bidirectional photoswitching between thermally stable isomers is possible without using UV light at all. <B>Our team synthesized this azobenzene molecule <U>by ourselves</U>.</B></P><H1>Bacteria-azobenzene connection</H1><P style="font-size:1.1em;">We attached the azobenzene molecules to the bacteria's modified lipopolysaccharide (LPS) on the outer-membrane. Azobenzene reacts with the Kdo sugar of the LPS giving stable amide bonds of azo-bacteria connections. Amide bonds cannot form at room temperature. To overcome this we used an EDC catalyst. EDC is used to form amide bonds between peptides. We could not find any information about how much EDC molecules are safe for E.coli so we performed a series of experiments to check the concentration range of EDC that is safe for a bacterial culture and used it to attach the azobenzene to the E.coli's LPS.Diagram A: azobenzene attaches to KDO sugers</P><P style="font-size:1.1em;">We run a concentration gradient of EDC along with Top10 bacteria, 1 is the highest concentration of EDC, 2 is the lowest concentration of EDC. As you can see in the tubes 1 and 2 we did not get a bacterial growth, but in 3-10 tubes we did. Therefore, we chose to work with the concentration of tube 4 for the rest of the experiment where we use EDC with azobenzene and bacteria.</P><H1>Synthetic biofilm formation</H1><P style="font-size:1.1em;">After mixing azobenzene with bacteria, we exposed them to 530-560 nm wavelength and we saw the “sticky” bacteria aggregate to form a biofilm.  The quorum sensing molecules, AHL can diffuse faster between them leading to a faster information exchange.
left: E.coli sample with azobenzene, right: E.coli sample with-out azobenzene
The dipole force between azobenzene rings are more powerful than the natural negative repulsion of bacteria.
We are the first people ever to demonstrate this technique. Our vision can help many researchers study the kinetics of biofilm formation in real time.</P><H1>Signal focusing by azobenzene</H1><P style="font-size:1.1em;">Quorum sensing bacteria release chemical signal molecules called autoinducers (AHL molecules) that increase in concentration as a function of cell density.
By making the bacteria aggregate, they exchange AHL molecules faster, which speeds up the detection process.
We engineered E. coli bacteria to emit light at a wavelength range of 530-560nm (green) when they detect the desired substance. The green light is absorbed by azobenzene, photo-switches to its “sticky” form causing the bacteria clump together forming a reversible synthetic biofilm.  The AHL molecules diffuse quickly to all the other bacteria and they all produce green light, which can be seen by the naked eye.</P><H1>Azobenzene aggregate Nano-Particles (NPs)</H1><P style="font-size:1.1em;">We established our iGEM azobenzene biological conceptions based on the Nano-word. We collaborated with Weizmann institute to test azobenzene molecules. We have seen that azobenzene can aggregate various NPs like iron oxide, gold and big particles like silica (see reference and TEM figures), and based on this behaviors we established our vision to use azobenzene as a photo-induced molecule to aggregate bacteria forming a synthetic biofilm.</P><P>For full azobenzene protocol follow the <A href="https://static.igem.org/mediawiki/2014/0/0e/Technion-Israel-Azobenzene.pdf" target="_blank">link</A></P><P>For extended information follow the <A href="https://static.igem.org/mediawiki/2014/f/fa/Technion-Israel-Simple_azo.pdf" target="_blank">link</A></P></CENTER><P style="font-size:1.0em; text-align:left;">
1. Azobenzene Photoswitching without Ultraviolet Light,Andrew A. Beharry , Oleg Sadovski , and G. Andrew Woolley
2. Hammet LP (1937) The effect of sturcure upon the reactions of organic compounds. Benzene derivatives. J Am Chem Soc 59: 96-103. http://dx.doi.org/10.1021/ja01280a022
3. Sykes P (1986) A guidebook to mechanism in organic chemistry, 6th Ed. Harlow, England:   Pearson Education Limited. 416 p.
4. Salonen LM, Ellermann M, Diederich F (2011) Aromatic rings in chemical and biological recognition: energetics and structures. Angew Chem Int Ed 50: 4808-4842. http://dx.doi.org/10.1002/anie.201007560
5. Hunter C, Sanders J (1990) The nature of π-π interactions. J Am Chem Soc 112: 5525-5534. http://dx.doi.org/10.1021/ja00170a016
6. Nicolas L, Beugelmans-Verrier M, Guilhem J (1981) Interactions entre groupes aromatique et polaire voisins – III. Tetrahedron 37: 3847-3860. http://dx.doi.org/10.1016/S0040-4020(01)98883-0
7. A simple route to fluids with photo-switchable viscosities based on a reversible transition betweenvesicles and wormlike micelles†Hyuntaek Oh,‡a Aimee M. Ketner,‡a Romina Heymann,b Ellina Kesselman,c Dganit Danino,c Daniel E. Falveyb and Srinivasa R. Raghavan
8. http://www.piercenet.com/product/edc (EDC Reagent)</P></HEADER></ARTICLE></DIV></DIV></DIV><DIV class="wrapper style3"><DIV class="title" id="hk">Histidine Kinase</DIV><DIV id="highlights" class="container"><CENTER><H1 style="font-size:1.5em;">Introduction</H1><P style="font-size:1.1em; line-height:1.75em;">
Some substances that we want to detect cannot diffuse into the cell or they do not activate promoters.  To test for these substances we want utilize the modularity of E.coli’s EnvZ/ompR two-component signaling system by creating chimera proteins that detect the desired substance.<B>Figure 1: How a chimaera protein would use the EnvZ/ompR two-component signalling system to trigger our system</B>
Taz is a chimaera protein of the cytoplasmic domain of EnvZ fused with the sensory domain of the transmemebrane aspartate receptor (TAR) (Tabor, Groban, &amp; Voigt, 2009)</P><H1 style="font-size:1.5em;">TaZ Construct</H1><P style="font-size:1.1em; line-height:1.75em;"><B>Completed and Biobricked</B></P><P style="font-size:1.1em; line-height:1.75em;">We found the receptor, tar-envZ biobrick (Bba_C0082) which contains the coding sequence for Taz. In order to use the Taz we added the promoter Pcat (Bba_I14033), an RBS (Bba_B0034) and double terminator (Bba_B0015).  Thus we created the Taz construct biobrick <A href="http://parts.igem.org/Part:BBa_K1343003" target="_blank">BBa_K1343016</A>. Click on the link to continue reading about our <A href="https://2014.igem.org/Team:Technion-Israel/Experiments#taz">TaZ experimentation</A>.</P><P>Two different E. coli strains were tested:
(1)BW25113 - parent strain for the Keio collection
(2)JW3367-3 - Keio collection mutant with EnvZ deletion
(These strains were given to us by Lior Zelcbuch, Elad Hertz from Ron Milo’s lab at the Weizmann Institute of Science)
The goal was to compare the expression in the wild type and in the ΔEnvZ mutant.  We expected that in the wild type the expression will be greater than in the mutant since the natural EnvZ/ompR system will cause expression of the RFP.</P><P>For extended information follow the <A href="https://static.igem.org/mediawiki/2014/7/7d/Technion-Israel-Histidine_Kinase_-_long_for_wiki.pdf" target="_blank">link</A></P></CENTER><P style="font-size:1.0em;">
1. Baba, T., Ara, T., Hasegawa, M., Takai, Y., &amp; Okumura, Y. (2006). Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection. Molecular Systems Biology, 1-11.
2. Forst, S. A., &amp; Roberts, D. L. (1994). Signal transduction by the EnvZ-OmpR phosphotransfer system in bacteria. Research in Microbiology, 145, 363-373.
3. Harrison, K. (2013). Reporter ompC-GFP. Retrieved from Toximop: https://2013.igem.org/Team:Dundee/Project/ReporterOmpC
4. Heyde, M., Laloi, P., &amp; Portalier, R. (2000). Involvement of Carbon Source and Acetyl Phosphate in the External-pH-Dependent Expression of Porin Genes in Escherichia coli. Journal of Bacteriology, 182(1), 198-202.
5. Levskaya, A., Chevalier, A. A., &amp; Tabor, J. J. (2005). Engineering Escherichia Coli to see light. Nature, 438(24), 441-442.
6. Michalodimitrakis, K. M., Sourjik, V., &amp; Serrano, L. (2005). Plasticity in amino acid sensing of the chimeric receptor Taz. Molecular Microbiology, 58(1), 257–266.
7. Tabor, J. J., Groban, E. S., &amp; Voigt, C. A. (2009). Performance Characteristics for Sensors and Circuits Used to Program E. coli. In S. Y. Lee (Ed.), Systems Biology and Biotechnology of Escherichia coli (pp. 401-439). Springer Science+Business Media B.V.
8.Yoshida, T., Phadtare, S., &amp; Inouye, M. (2007). The Design and Development of Tar-EnvZ Chimeric Receptors. Methods in Enzymology, 423, 166-183.</P><CENTER><H1 style="font-size:1.5em;">Gene Deletion</H1></CENTER><CENTER><P><B>Failed to delete ackA-pta genes</B></P><CENTER><CENTER><P>
To be able to utilize the EnvZ/OmpR two-component-signaling   system for our project, we need to ensure that the natural EnvZ/OmpR system does not interfere, introducing noise to the system, giving a false signal.<B>How did we change make sure the natural EnvZ/OmpR system doesn’t disrupt our system?</B>
We needed to use a strain of E. coli that has an EnvZ deletion (ΔEnvZ).  The Keio Collection  (Baba, Ara, Hasegawa, Takai, &amp; Okumura, 2006) contains a strain of E. coli with exactly this deletion (strain JW3367-3).Great! So we are all set right?
Wrong.
OmpR can be phosphorylated not only by the histidine kinase EnvZ but also by an acetyl phosphate dependent mechanism. (Heyde, Laloi, &amp; Portalier, 2000) This would introduce a low level of noise into the system.  Since our detector needs to be precise to be able to detect low concentrations, even a low level of noise would be problematic.<B>What did we do about this?</B>
We needed a bacteria which not only had the EnvZ knockout, but also had the genes for the Phosphate acetyl transferase (pta) and Acetate kinase (ackA) enzymes which are involved in the acetyl phosphate pathway (Heyde, Laloi, &amp; Portalier, 2000)
Unfortunately we had some trouble finding a strain with the deletions we needed so we decided to make one ourselves.<B>How did we do this?</B>
Lior Zelcbuch and Elad Hertz from Ron Milo’s lab at the Weizmann Institute of Science suggested we take the E. coli strain JW3367-3 (ΔEnvZ) from the Keio Collection and use the Lamda Red technique to delete the genes for ackA and pta.
Since the genes for ackA and pta are adjacent to each other on the E. coli chromosome, we decided to delete them in one go.
With Lior Z. and Elad’s guidance and help from Edna Kler from the Technion, we attempted to delete the genes.
We tried several times, once we even went all the way to the Weizmann Institute in Rehovot where Lior Z., Elad and Sagit Yahav helped us.  But to no avail!  We just couldn’t manage to knock out the genes! </P><P>For extended information follow the <A href="https://static.igem.org/mediawiki/2014/1/1e/Gene_Deletion_-_wiki.pdf" target="_blank">link</A></P></CENTER><P style="text-align:left;">
1. Baba, T., Ara, T., Hasegawa, M., Takai, Y., &amp; Okumura, Y. (2006). Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection. Molecular Systems Biology, 1-11.
2. Datsenko, K. A., &amp; Wanner, B. L. (2000). One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proceedings of the National Academy of Sciences of the United States of America, 97(12), 6640-6645.
3. Forst, S. A., &amp; Roberts, D. L. (1994). Signal transduction by the EnvZ-OmpR phosphotransfer system in bacteria. Research in Microbiology, 145, 363-373.
4. Heyde, M., Laloi, P., &amp; Portalier, R. (2000). Involvement of Carbon Source and Acetyl Phosphate in the External-pH-Dependent Expression of Porin Genes in Escherichia coli. Journal of Bacteriology, 182(1), 198-202.
5. Kenney, L. (n.d.). Welcome to the Kenney Lab. Retrieved from University of Illinois at Chicago: http://www.uic.edu/labs/kenneyl/)
</P></CENTER></CENTER></DIV></DIV><DIV id="footer-wrapper" class="wrapper"><DIV class="title" id="newmethod">New Standard</DIV><DIV id="footer" class="container"><HEADER class="style1info"><CENTER><H2>Bio-Builder: Can we build it? Yes We Can!</H2><P>Throughout this project, we had many genetic constructs to build, including: the alpha system, the beta system, the RNA splint, and testing Toggle Switches in order to optimize the beta system. As you may know, building these circuits is not easy: it requires a lot of lab-work, and has a high rate of failure, in addition to the fact that each reaction requires a different method, and individual planning.
To solve these problems, we designed a new standard for the iGEM community, which will allow teams to construct many different parts, using only a small set of almost identical Gibson reactions which we have shown to have a <STRONG>94% success rate</STRONG> over the construction of 21 gates. In addition, because these reactions have been standardized, they are compatible with high-throughput methodologies. Finally, we implemented a simple method for making existing parts fit our standard.</P><H1>The Standard</H1><P>Our standard is based upon a set of <B>Gibson Addresses</B> – <B>&quot;barcodes&quot;</B> which we have strategically placed along the genetic sequence, between the different promoters and genes. It is possible to use these barcodes (which have been optimized for PCR and Gibson reactions), to amplify pieces of DNA by PCR, and combine them using Gibson reactions. There are two types of parts in our standard:</P><H1>How to build a circuit</H1><P>This allows us to construct any one of the following constructs:                                Where P is any promoter in our system, G1 is any gene of class 1, and G2 is any gene of class 2.
To see how we build any one of the constructs click on its image. *Figure*

In addition, it is quite simple to convert genes into our standard:
Just PCR the desired gene with barcoded primers and then Gibson with any other construct

In addition to designing a standardized methodology, we also optimized the sequences of our barcodes for PCRs and for Gibson Reactions. This is evident from the success rate.
Overall, using our new method, we have successfully built 21 genetic gates (of which 12 were bio-bricked), and only 1 reaction failed along the way, giving us a success rate of 94.2%. As a control we did 14 Gibson reactions using the iGEM prefix and suffix, of which 6 were successful (and bio-bricked), giving it a success rate of 42% (meaning that a Gibson reaction with the iGEM prefix and suffix is 10 times more likely to fail than a Gibson reaction using our optimized overlaps).
See the full list of all the constructs built using our methodology, which addresses were used, and which templates were used:</P><TABLE style="background:none; text-align:center; color:#919499; line-height:2.5em;"><TBODY><TR style="border: 1px solid #919499;"><TD>Status/BioBrick#</TD><TD>Part name</TD><TD>First comp</TD><TD>Second comp</TD><TD>Barcodes</TD><TD>Status/BioBrick#</TD><TD>Part name</TD><TD>First comp</TD><TD>Second comp</TD><TD>Barcodes</TD></TR><TR><TD>BBa_K134004</TD><TD>Plac-CI-T7</TD><TD>1C3:Plac-CI</TD><TD>T7 RNA poly</TD><TD>3,4</TD><TD>BBa_K134021</TD><TD>Plux-amilCP-CI</TD><TD>1C3:Plux-amilCP</TD><TD>CI</TD><TD>2,4</TD></TR><TR><TD>BBa_K134005</TD><TD>PCI-tetR</TD><TD>1C3:PCI</TD><TD> tetR</TD><TD>5,6</TD><TD>BBa_K134022</TD><TD>Plux-amilCP-tetR</TD><TD>1C3:Plux- amilCP</TD><TD>TetR</TD><TD>2,4</TD></TR><TR><TD>BBa_K134006</TD><TD>Plac-tetR</TD><TD>1C3:Plac</TD><TD> tetR</TD><TD>5,6</TD><TD>ReactionSuccessful*</TD><TD>Plux-mCherry</TD><TD>----</TD><TD>-------</TD><TD>-------</TD></TR><TR><TD>BBa_K134007</TD><TD>Ptet-CI</TD><TD>1C3:Tet</TD><TD> CI</TD><TD>5,6</TD><TD>Reaction Successful</TD><TD>Plux-g.luc-CI</TD><TD>1C3:Plux-g.luc</TD><TD>CI</TD><TD>2,4</TD></TR><TR><TD>BBa_K134008</TD><TD>Ptet-lacI</TD><TD>1C3:Tet</TD><TD>lacI</TD><TD>5,6</TD><TD>Reaction Successful</TD><TD>Plux-amilCP-lacI</TD><TD>1C3:Plux- amilCP</TD><TD>lacI</TD><TD>2,4</TD></TR><TR><TD>BBa_K134012</TD><TD>Plux-amilCP</TD><TD>1C3:plux</TD><TD>amilCP</TD><TD>5,6</TD><TD>Reaction Successful</TD><TD>Plux-mcherry-tetR</TD><TD>1C3:Plux-mCherry</TD><TD>tetR</TD><TD>2,4</TD></TR><TR><TD>BBa_K134013</TD><TD>PT7-amilCP</TD><TD>1C3:pT7</TD><TD>amilCP</TD><TD>5,6</TD><TD>Reaction Successful</TD><TD>Plux-mcherry-CI</TD><TD>1C3:Plux-mCherry</TD><TD>CI</TD><TD>2,4</TD></TR><TR><TD>BBa_K134018</TD><TD>Ptet-tetR-T7</TD><TD>1C3:Ptet-tetR</TD><TD>T7 RNA poly</TD><TD>3,4</TD><TD>Reaction Successful</TD><TD>Plux-mcherry-LacI</TD><TD>1C3:Plux-mCherry</TD><TD>lacI</TD><TD>2,4</TD></TR><TR><TD>BBa_K134019</TD><TD>Plux-G.luc-TetR</TD><TD>1C3:Plux-g.luc</TD><TD>TetR</TD><TD>2,4</TD><TD>Reaction Successful</TD><TD>Plux-mcherry-luxI</TD><TD>1C3:Plux-mCherry</TD><TD>luxI</TD><TD>2,4</TD></TR><TR><TD>BBa_K134020</TD><TD>Plux-G.luc-lacI</TD><TD>1C3:Plux-g.luc</TD><TD>lacI</TD><TD>2,4</TD><TD>Reaction Successful</TD><TD>Plac-CI-mcherry</TD><TD>1C3:Plac-CI</TD><TD>mCherry</TD><TD>2,3</TD></TR></TBODY></TABLE><P>*mCherry contains restriction sites which makes it unbio-brickable</P><P>For the sequences of the barcodes, and their implementation in our project, see our <A href="https://2014.igem.org/Team:Technion-Israel/Judging#biobrick">bio-bricks page</A></P></CENTER></HEADER></DIV></DIV><DIV class="wrapper style2"><DIV class="title" id="protocol">Protocols</DIV><DIV id="main" class="container"><DIV id="content"><ARTICLE class="box post"><HEADER class="style1"><H2>Here are all the protocols we used in the project</H2><P>to view the files you will need <A href="http://get.adobe.com/reader/">Adobe Acrobat Reader</A> or similar</P></HEADER><CENTER><DIV id="proto"><A href="https://static.igem.org/mediawiki/2014/8/8f/Technion-Israel-Antibiotics.pdf" target="_blank">Antibiotics Preparation</A><A href="https://static.igem.org/mediawiki/2014/5/57/Technion-Israel-Chemical_transformation.pdf" target="_blank">Chemical Trasformation</A><A href="https://static.igem.org/mediawiki/2014/0/03/Technion-Israel-DNA_Kit_Plate.pdf" target="_blank">DNA Kit Plate Instructions</A><A href="https://static.igem.org/mediawiki/2014/4/49/Technion-Israel-MixandGo.pdf" target="_blank">Z-Competent™ cells and Mix&amp;Go</A><A href="https://static.igem.org/mediawiki/2014/2/27/Technion-Israel-Gel_and_PCR_Extraction.pdf" target="_blank">Gel/PCR Extraction</A><A href="https://static.igem.org/mediawiki/2014/b/b3/Technion-Israel-Gel_Prep.pdf" target="_blank">Gel Preparation</A><A href="https://static.igem.org/mediawiki/2014/d/dc/Technion-Israel-manual_NEB_%28gibson_assembly%29.pdf" target="_blank">Gibson Assembly</A><A href="https://static.igem.org/mediawiki/2014/d/dd/Technion-Israel-Glycerol_stock.pdf" target="_blank">Glycerol stock</A><A href="https://static.igem.org/mediawiki/2014/a/a6/Technion-Israel-LB_SOB_SOC_BA_protocol.pdf" target="_blank">LB, BA, SOB, SOC</A><A href="https://static.igem.org/mediawiki/2014/c/c1/Technion-Israel-Ligation.pdf" target="_blank">Ligation</A><A href="https://static.igem.org/mediawiki/2014/b/b7/Technion-Israel-Presto_Mini_Plasmid_Kit_Protocol.pdf" target="_blank">Presto Mini-prep</A><A href="https://static.igem.org/mediawiki/2014/1/1b/Technion-Israel-PCR.pdf" target="_blank">PCR</A><A href="https://static.igem.org/mediawiki/2014/d/dd/Technion-Israel-Phosphorylation_%28PNK%29_and_blunt_ligation.pdf" target="_blank">Phosphorylation and blunt ligation</A><A href="https://static.igem.org/mediawiki/2014/7/78/Technion-Israel-Restriction_enzyme.pdf" target="_blank">Restriction enzyme</A><A href="https://static.igem.org/mediawiki/2014/a/aa/Technion-Israel-Transformation_to_3300_LG_using_electroporation.pdf" target="_blank">Electroporation Trasformation</A><A href="http://openwetware.org/wiki/NanoBio:_Protocol_for_gene_knockout" target="_blank">Gene deletion using λ red</A><A href="http://www.geneaid.com/sites/default/files/DF12_0.pdf" target="_blank">Genome extaraction</A></DIV></CENTER></ARTICLE></DIV></DIV></DIV><DIV class="wrapper style3"><DIV class="title" id="notebook">Lab Notebook</DIV><DIV id="highlights" class="container"><CENTER><P>to view the files you will need <A href="http://get.adobe.com/reader/">Adobe Acrobat Reader</A> or similar</P></CENTER><DIV class="row oneandhalf"><DIV class="4u"><SECTION class="highlight"><H3><A>Gene Deletion &amp; Histidine Kinase</A></H3><P>This is Rebecca's and Karen's lab notebook for gene deletion attempts and TaZ biobrick building.</P><UL class="actions"><LI><A href="https://static.igem.org/mediawiki/2014/8/88/Technion-Israel-HK.pdf" target="_blank" class="button style1">Learn More</A></LI></UL></SECTION></DIV><DIV class="4u"><SECTION class="highlight"><H3><A>Alpha System</A></H3><P>These are a few notebooks arranged together of all Alpha System gate constructs. Lab work done by Tal, Rica, Ronen, Shira, Noa and Alex</P><UL class="actions"><LI><A href="https://static.igem.org/mediawiki/2014/a/a4/Technion-Israel-Gates.pdf" target="_blank" class="button style1">Learn More</A></LI></UL></SECTION></DIV><DIV class="4u"><SECTION class="highlight"><H3><A>Azobenzene-Robot</A></H3><P>This is the lab notebook of all the experiments done with azobenzene, by Ittai, Yair, Faris and the Robot.</P><UL class="actions"><LI><A href="https://static.igem.org/mediawiki/2014/1/1c/Technion-Israel-robot.pdf" class="button style1">Learn More</A></LI></UL></SECTION></DIV></DIV><DIV class="row oneandhalf"><DIV class="4u"><SECTION class="highlight"><CENTER><H3><A>RNA Splint</A></H3><P>This is Ronen's lab notebook for the RNA splint</P><UL class="actions"><LI><A href="https://static.igem.org/mediawiki/2014/3/39/Technion-Israel-RNA.pdf" target="_blank" class="button style1">Learn More</A></LI></UL></CENTER></SECTION></DIV><DIV class="4u"><SECTION class="highlight"><CENTER><H3><A>Beta System</A></H3><P>This the Beta System lab notebook of Rica, Tal and Ittai</P><UL class="actions"><LI><A href="https://static.igem.org/mediawiki/2014/b/b8/Technion-Israel-batasys.pdf" target="_blank" class="button style1">Learn More</A></LI></UL></CENTER></SECTION></DIV></DIV></DIV></DIV><DIV class="wrapper style2"><DIV class="title" id="safety">Safety</DIV><DIV id="main" class="container"><CENTER><DIV style="line-height:1.75em;"><DIV style="font-size:1.1em; line-height:1.75em;"><P>							
One of the biggest concerns regarding synthetic biology in the general public is &quot;Will the genetically modified organism be safe for me? What happens when you release the organism you designed into the environment? What if you create something you cannot control?&quot;
These are valid questions that need to be answered when creating genetically modified bacteria. 
We tackled the important safety aspect in the project in three different ways:<B>(1)</B> Teaching and trying to understand synthetic biology.<B>(2)</B> Safety in the lab.<B>(3)</B> Our system's safety.</P></DIV><H1 style="font-size:1.2em;">Synthetic Biology Education</H1><P>We understand the fear and concern of the public about GMOs. Therefore we wanted to expose the public to synthetic biology. 
We gave lectures to teens and adults from different backgrounds about synthetic biology, its great potential and safety concerns.
We also emphasized the importance of safe lab work as part of our lab activity for children.</P><H1 style="font-size:1.2em;">Safety in the Lab</H1><P>We took all the necessary precautions such as lab coats, safety goggles when using liquid Nitrogen and always wore gloves.
No food was allowed in the lab and there was a separate area for computer work.
The dress code was also strict- when working in the lab we wore closed shoes and long pants/skirt.
While working in the lab we used <A href="http://www.sciencelab.com/msds.php?msdsId=9927667" target="_blank">Ethidium Bromide (EB)</A> for using gel electrophoresis and analysis. This substance is a potent mutagen that is used as a nucleic acid stain.
Therefore, we took special precautions such as working with EB only in the chemical hood, and having separate disposal for EB. We also had have a separate area on the bench where we ran the gels. This area has its own equipment such as tips, pipettors and gloves.
When performing gel extraction, we were exposed to UV light for short periods of time. To minimize the exposure we used protection equipment such as face protection shields and full body lab coats.<B>Non-biological Safety</B>Another safety aspect of our project is the chemical one.
The Azobenzene production was done using a few chemicals that needed special caution such as 70% Nitric Acid. This substance is hazardous when it comes in contact with skin. Therefore, a face shield, full suit and all the appropriate protection was worn.
Another substance was chloroform which is carcinogenic. All the safety measures was taken, including personal protection and exhaust ventilation in the chemical hood. We also used AgO, THF and Zinc dust. AgO is irritating to the eyes and respiratory system, THF is hazardous when it comes in contact with skin and has carcinogenic effects, Zinc dust is an irritant when it comes in contact with skin. Therefore, the use of all these substances was under the guidance of our mentors and every step was evaluated by experienced chemists (We consulted Ruth Goldschmidt from Professor Livney's lab and also Emma Gerts, who is in charge of organic chemistry labs in the Technion) who advised us on all the necessary safety measures needed to be taken. The whole process was always done according to all safety precautions, in a chemical hood and the disposal was according to MSDS of the reagents.
Other chemical materials we used in the production of Azobenzene were not hazardous.
The Azobenzene as a product is not hazardous and is biologically safe. The product is not volatile and is not hazardous when it comes in contact with skin (according to Woolley Group, Department of Chemistry in the University of Toronto, Canada).
Our project combines both synthetic biology and chemistry. We think it's important to have a safety program in the iGEM competition for chemistry, not only for biology, which will allow the iGEM HQ to supervise chemical lab work as well.</P><H1 style="font-size:1.2em;">Our System's Safety</H1><P>Now that we discussed the general safety, the safety of our project needs to be assessed.
In our project we used E. coli strains as model organisms for our systems. 
We chose this bacteria since it is common in laboratory use and the strains we used (BL21, Top 10-DH10β, DH5αz1, JW3367-3, BW25113) are non-pathogenic and safe to work with.<B>In the future</B>, when we finish testing the whole system, adding a kill switch into the system  would be a MUST to ensure a safe use of the bacteria as a detector.</P></DIV></CENTER></DIV></DIV><DIV id="copyright"><UL><LI><A href="https://www.facebook.com/iGEM2014.Technion" target="_blank">Visit our Facebook page</A></LI><LI>© All rights reserved</LI><LI>Design: <A href="http://html5up.net" target="_blank">HTML5 UP</A>&amp; Technion_Israel Team</LI><LI>igem14il@gmail.com</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Technion-Israel/Project">http://2014.igem.org/Team:Technion-Israel/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Technion-Israel/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Technion-Israel/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Technion-Israel/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Technion-Israel/Project&amp;oldid=400178" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>