Team:Shenzhen SFLS/PROJECT
Project
Motivation
Hepatitis C is a liver infection caused by the Hepatitis C virus (HCV). For some people, hepatitis C is a short-term illness but for 70%–85% of people who become infected with Hepatitis C, it becomes a long-term, chronic infection. Chronic Hepatitis C is a serious disease that can result in long-term health problems, even death. The majority of infected persons might not be aware of their infection because they are not clinically ill.Anestimated 130~150 million people worldwide have chronic hepatitis C infection (WHO,2015). The number of hepatitis C virus infection-related deaths reached an all-time high of 19,659 in 2014, making hepatitis C the number one infectious disease that kills people. There is no vaccine for Hepatitis C and therefore the detection of Hepatitis C is vital to reduce the harm of the silent killer.  However, there are several defects of current detection methods: 1.Due to the relatively high developing and manufacturing costs and specific equipment required, the promotion of Hepatitis detection is hindered, especially in low-resource environments. 2.Though anti-HCV screening tests are convenient and quick, due to the relatively low specificity and persons with early HCV infection might not yet have developed antibody levels high enough that the test can measure, false-positive and false-negative results appear often. 3.Antibody-based diagnostics don'toffer a sequence-based method of detection, which meansresearch and clinical tools can't be designed rationally,and therefore lead to higher development costs and longer design-to-production cycles(JB, 2016).  Witnessing such a situation, we seek to develop a new type of HCV detection method that is highly specific, cheap and efficient.
Design
We decide to develop a new detection method based on cell-free systems(Keith Pardee, 2015) , which are placed on paper with the fundamental transcription and translation properties of a cell but that are sterile and abiotic. Our goal is to realize sequence-specific detection of HCV by linking isothermal RNA amplification to toehold switch RNA sensors, which regulate measurable colorimetric outputs for semi-quantitative and qualitative detection. A toehold switch is a transducer RNA that enables translation of the downstream protein only upon binding of a cognate trigger RNA(Alexander A. Green，2014). By programming the toehold switches to pair with specific short sequences of HCV, we ensure that our device will be exclusively activated by HCV, and moreover, distinguish different genotypes of HCV. Using LacZ as reporter gene, we are able to detect outputs by eyes, and we also develop a mathematic model to further analyze them by electronic readers.
Input
HCV is a small, enveloped, positive-sense single-stranded RNA virus. Each particle consists of a core of genetic material (RNA), surrounded by a protective shell of protein, and further encased in a lipid envelope. HCV is a blood-borne virus which presents in blood and other body fluids. We chose serum, which carries a high concentration of HCV RNA, as our detection substrate. As for the extraction method, boiling our sample for a short period(1~2 minutes) should be enough to reach the standard of an amplification reaction, which requires cracking the capsids of the HCV viruses which qualifies the exposure of HCV RNA without damaging it. We also added RNase Inhibitor to reaction in order to prevent possible degeneration of RNA samples.
In order to offer a sequence-specific detection, we focused on the region of HCV RNA which encodes the virus' nucleocapsid due to its high specificity and low mutation rate. Considering not all regions of HCV RNA were suitable because of the complex structure, we used a software to predict its secondary structure [Figure A]  and picked four short sequences that were likely to be ideal for detection of HCV 1b and HCV 2a respectively. We then performed a BLAST search of each sequence, and the results showed that they were highly specific to each genotype of HCV. Thus, we based the detection directly on short sequences of HCV RNA.  To get the short sequences from HCV RNA and amplify them to increase the sensitivity of the detection, we employed an isothermal amplification method designed to detect RNA targets—nucleic acid sequence-based amplification (NASBA). NASBA is known to be extremely sensitive and has a proven track record in field-based diagnostic applications (Cordray and Richards-Kortum, 2012). The amplification process begins with reverse transcription of a target RNA that is mediated by a sequence-specific reverse primer to create an RNA/DNA duplex. RNase H then degrades the RNA template, allowing a forward primer containing the T7 promoter to bind and initiate elongation of the complementary strand, generating a double-stranded DNA product. T7- mediated transcription of the DNA template then creates copies of the target RNA sequence. Importantly, each new target RNA can be detected by the toehold switch sensors and also serve as starting material for further amplification cycles. The whole reaction takes approximately 2 hours. NASBA requires an initial heating step (65 C), followed by isothermal amplification at 41 C(Guatelli et al., 1990). Compared with other amplification methods, NASBA is more applicable, especially in low-resource environments(Keith Pardee, 2016).
Figure A (the blue parts were the chosen short sequences from HCV 2a RNA)
Although the NASBA is the first-choice method for amplification in field tests, we haven’t got the chance to test it in our lab due to the limited resources we own. Instead, we applied in vitro transcription kit(MEGAscript® T7 Transcription Kit, Invitrogen™) to our project as an alternative choice. For the safety of lab, we weren’t able to experiment on HCV. Therefore, we designed short DNA sequences as substitutes, which correspond to our sensors and were synthesized as primers. Then we assembled them through oligo annealing ligation and transcribed them into short RNA sequences as substitutes to activate our sensors. The final concentration of our trigger RNA after purification is verified to be applicable in our testing experiments. After that, we diluted the purified RNA trigger to different detection levels, such as 10^1M, 10^2M, 10^3M etc., and applied them to the paper in order to test the impact of trigger concentration on the outputs.
Sensor
To target on the short sequences of HCV RNA, we used toehold switches as our sensor. Toehold switches are RNA-based triggers that can be rationally programmed to recognize sequence-specific RNA and integrated into the genome to regulate endogenous genes. Folding onto itself, the toehold switch represses translation by sequestering the ribosomal binding site (RBS) of the transducer RNA within a loop region of a hairpin. This hairpin is unwound upon binding of a cognate trigger RNA, exposing the RBS and enabling translation of the downstream protein. By designing the interaction domain (30 nt) of the toehold switch to be complementary to the target RNA, we were able to detect the presence of the target RNA through the expression of downstream protein.
The initial domain of the switch is known as a toehold at its 5’ end. This toehold domain provides the initial reaction site for binding between the trigger and switch RNAs and greatly improves the ON/OFF ratio of the switches (Cordray, 2012). According to the short sequences of HCV RNA, we synthesized DNAs with specific sequence that could be translated into functional toehold switches. Due to the great programmability of the toehold, it can be designed to respond to cognate RNAs with arbitrary sequences. Thus, toehold switches have great advantage over other sensors because of their short development cycles and their relatively low development costs, especially for viruses with high mutation rate. Besides diagnostics, toehold switches also offer diverse applications in other fields of synthetic biology.
In order to test the orthogonality of the toehold switches, we arrayed each of the 8 toehold switches (along with negative control, plain water) with their corresponding triggers (along with negative control, plain water) into a 9*9 square. If the orthogonality is specific, activation of the toehold switches will be rigorously restricted to the diagonal.  With excellent orthogonality, our devices are then able to rule out cross-activation or possible false positive results.
Output
To achieve rapidity and convenience, we put our circuits on an in vitro paper-based platform. The method is achieved by freeze drying cell-free systems (including enzymes, ribosomes,dNTPs, tRNAs, amino acids, buffers, and synthetic gene network etc.) into paper and other porous substrates to create materials with the fundamental transcription and translation properties of a cell but that are sterile and abiotic. Stable at room temperature, these embedded materials are readily stored, distributed, and can be activated by simply adding water (Pardee, 2014). This platform affords many benefits for our project. Without interference caused by cell division, the detection will be more stable and precise. Moreover, paper-based synthetic gene networks are also potentially less expensive to manufacture and operate than most of the options currently available.
To begin, we tested whether the cell-free system we intended to use, E. coli S30 Extract System for Circular DNA (Promega), worked well. We used the standard plasmid, pBEST, supplied by the system, with firefly luciferase, which catalyzes light-emitting reactions as reporter gene. Light is produced by converting the chemical energy of luciferin oxidation through an electron transition, forming the product molecule oxyluciferin. The constitutive expression of firefly luciferase is supported in solution-phase reaction and the light generated can be identified by naked eyes. Since it serves only as a qualitative test, we haven’t developed more precise assay method for luciferase.
In order to turn our system into a widely applied device with the function of both qualitative and quantitative detection, we used enzyme b-galactosidase (LacZ) to create a synthetic gene network that generates a dramatic enzyme-mediated color change in response to conditional inputs. LacZ cleaves the yellow substrate, chlorophenol red-b-D-galactopyranoside, embedded into the freeze-dried paper discs, to produce a purple chlorophenol red product that is visible to the naked eye and can be measured on standard plate readers by tracking the generation of signal in the blue channel, relative to yellow signal (red + green channels) for quantitative detection (Pardee, 2014).
Next, we applied all the necessary components for in vitro transcription and translation onto filter paper, along with plasmids of different switches and RNase Inhibitor. Then we treated the paper with  corresponding triggers to test orthogonality. After kicking out those switches with low activation rates, we replaced luciferase with enzyme b-galactosidase (LacZ) to generate colorimetric output. Since triggers in higher concentration activate toehold switches more efficiently, the rate of color change of the reaction also varies with respect to the same variable. To precisely determine how they vary, we “switched on” the reaction with triggers of different conc. , 10^1M, 10^2M,10^3M, etc. Then we put the reaction into a standard plate reader with absorbance monitored at 570nm. We plotted graphs from the data table and determine the detection rate from the slope. With these data and graphs, we could find out time points significant to field detection: the time point where the colorimetric output starts to be recognizable for naked eyes, which will differ due to different reaction rates; the time point at which the color change activated by triggers of different conc. all become recognizable but with different color depth, when quantitative detections are carried out; and the time points when outputs are maximized.
Application
Prototype
We came up with the idea of turning our detection method into a consumer product that can be conveniently and efficiently used. Our detection paper is named “HCV hunter”. By printing hydrophobic barriers on paper (Slomovic, 2015), we created hydrophilic channels, where the cell-free system and plasmids were placed. The paper is divided into three areas—two detection areas for HCV 1b and HCV 2a respectively, and a quality control area. Plasmids with toehold switches corresponding to short RNA sequences from HCV 1b and HCV 2a are separated so that the paper can distinguish between the two major genotypes of HCV prevalent in Asia, providing accurate information. Plasmids in the quality control area are positive controls, in which toehold switches aren’t inserted. When adding water to the paper, the quality control area should turn purple due to the expression of b-galactosidase under normal circumstances. Through observing whether the quality area turns purple, users can determine if the paper is damaged and whether the results are credible. The operation of the test paper is simple and concise. A specific amount of serum is used as detection substrate. Going through the process of dilution, boiling, and NASBA, it is ready to be added onto the paper, where the colorimetric reaction takes place. The resulting color in test areas, which can be distinguished by naked eye, qualitatively reveals whether the user carries HCV and discerns the genotype of HCV [Figure B]. For further analysis, devices such as electronic readers, or even cellphones with cameras, can provide a robust and quantitative measurements of sensor outputs at a relative low cost. On the basis of experimental data (Pardee, 2014), we built a mathematic model to correlate the color intensity with the concentration of HCV RNA in the samples, so as to provide semi-quantitative detection. We set intervals based on the concentration gradient. The highly approximate HCV RNA concentration calculated from color intensity falls into one of these intervals. The magnitude of the intervals is represented by different numbers of “+”s in the final result displayed, with more “+”s indicating higher concentration [Figure C].
Figure B
The result I. indicates that the patient carries HCV 2a;                  II. indicates that the patient carries HCV 1b;                   III. indicates that the test paper is damaged;                   IV. indicates multiple infections with HCV 1b and HCV 2a
Figure C
Application
Paper-based HCV detection has a wide range of application. In places where strict detection of blood borne viruses are required like drug rehab centers and blood centers, or places with high prevalence of HCV but low resources like some underdeveloped areas, our method offers not only a low-cost and efficient detection, for it isn’t as demanding in aspect of equipments and operating staff as other methods such as RT-PCR, but also guarantees the sensitivity, reducing the false-positive and false-negative rate greatly compared with other qualitative test paper. Since the diagnosis of HCV requires multiple parameters (virus load, liver function, etc.), our semi-quantitative detection results provides significant information for future treatment of HCV.
Future Work
Though at present serum is the most common detection substrate, we’ve seen potential in other body fluids such as saliva and gingival crevicular fluid (GCF). Besides carrying a considerable concentration of HCV RNA (Suzuki, 2005), saliva and GCF are safer to collect compared with serum, reducing the risk of health care workers getting infected after needlesticks involving HCV-positive blood. We expect advances in sample collection techniques to be combined with isothermal nucleic acid sequence- based amplification to solve existing challenges of maintaining sample stability and raising sample concentration to achieve sensitivity.
Future work should also include optimization of reporter gene. We selected LacZ as reporter gene, creating circuits with colorimetric outputs for rapid and low-cost detection. However, defects in sensitivity limited the function to semi-quantitative detection. One of the possible improvements is to use dual-luciferase reporter system (Promega) to normalize the activity of the experimental reporter to the activity of the internal control to minimize experimental variability caused by differences in cell viability or transfection efficiency.
Rather than competing with technically demanding and expensive lab-based techniques like ELISA, PCR, and mass spectrometry, with DNA technologies becoming cheaper and more advanced, we envision paper-based systems creating a new, low-cost generation of sensors that could be embedded ubiquitously into daily life.
Proof of Concept
The proof of concept was performed using trigger RNA to activate our device. [Figure D]The glow indicates the expression of firefly luciferase, and thus validates that the hairpin of the toehold switch is unwound upon binding of a trigger RNA. This demonstrates that our device is valid.
Figure D
Reference
Pardee, K., Green, A.A., Ferrante, T., Cameron, D.E., DaleyKeyser, A., Yin, P., and Collins, J.J. (2014). Paper-based synthetic gene networks. Cell 159, 940–954.
Green, A.A., Silver, P.A., Collins, J.J., and Yin, P. (2014). Toehold switches: de- novo-designed regulators of gene expression. Cell 159, 925–939.
Pardee, K., Green, A.A., Takahashi, M.K.,O’Connor, D.O., Gehrke,L., Collins,J.J.(2016).Rapid, Low-Cost Detection of Zika Virus Using Programmable Biomolecular Components. Cell 165, 1–12
WHO (2016). WHO to fact sheets about Hepatitis Chttp://www.who.int/mediacentre/factsheets/fs164/en/
Joy JB, et al. Lancet Infect Dis. (2016);doi:10.1016/S1473­3099(16)00124­9.
Cordray, M.S., and Richards-Kortum, R.R. (2012). Emerging nucleic acid- based tests for point-of-care detection of malaria. Am. J. Trop. Med. Hyg. 87, 223–230.
Guatelli, J.C., Whitfield, K.M., Kwoh, D.Y., Barringer, K.J., Richman, D.D., and Gingeras, T.R. (1990). Isothermal, in vitro amplification of nucleic acids by a multienzyme reaction modeled after retroviral replication. Proc. Natl. Acad. Sci. USA 87, 1874–1878.
Slomovic, S., Pardee, K., and Collins, J.J. (2015). Synthetic biology devices for in vitro and in vivo diagnostics. Proc. Natl. Acad. Sci. USA 112, 14429–14435.
Suzuki, T., Omata, K., Satoh, T.,Miyasaka, T.,Arai, C.,Maeda, M., Matsuno, T.,Miyamura, T. (2005).Quantitative Detection of Hepatitis C Virus (HCV) RNA in Saliva and Gingival Crevicular Fluid of HCV-Infected Patients. JOURNAL OF CLINICAL MICROBIOLOGY, Sept. 2005, p. 4413–4417
