Team:Waterloo/Design
Toggle navigation
Project
Description
Results
Design
Requirements
Lab & Design
Overview
sgRNA Modification
dCas9 Modification
Plant Defense
Parts
Experiments & Protocols
Notebook
Modeling
Overview
CaMV Biology
PAM Flexibility
Cas9 Dynamics
CaMV Replication
Viral Spread
Software
Practices
Overview
Commercialization
Human Practices
Lab Safety
Opinions
Survey
Outreach
Team
Our Team
Attributions
Sponsors
Collaborations
Example Page
Design
Simple sgRNA Exchange
The ability re-purpose sgRNAs to guide Cas9 to new target sites currently requires re-cloning the entire 500 base pair sgRNA sequence. The turnaround time needed to test different sgRNA sequences is a problem because different sgRNA are known to have very different biochemical activity
, in some studies ranging from 9%-60% cleavage efficiency, and these efficiencies cannot always be predicted computationally
.
Biochemical determinants in sgRNA activity: certain nucleotides tend to occupy specific positions in highly-active sgRNA. This shows "p-values of observing the conditional probability of a guide with a percent-rank activity of >0.8 under the null distribution"
.
i
Briner et al., 2014
Adapted from Figure 3
Link to Original Photo
By inserting restriction enzyme sites flanking the 20 base pair guide sequence of sgRNA, researchers will have the ability to more easily swap in new targets for Cas9. By taking advantage of the existence  of a single "G" nucleotide at the bottom of the first stem-loop that doesn't follow complementarity with the opposing nucleotide in the sgRNA scaffold, we were able to insert a restriction enzyme site without also introducing that site downstream in the original DNA sequence. We then placed another restriction site after the U6 promoter but before the gRNA to complete the swap. Our experiments showed that this modified version of the sgRNA was still capable of working to silence gene expression with a dCas9 in similar level as the non-modified. You can read about the design in detail on the Simple sgRNA Exchange Page .
sgRNA scaffold structure, different modules identified by Briner et al.
.
Modified sgRNA structure, including restriction site in the lower stem and before the 20nt gRNA sequence.
Cas9 PAM Flexibility
Being able to target DNA at different protospacer adjacent motif (PAM) site will allow scientists to target a larger portion of the genome. "Dead" Cas9 (dCas9) allows for easier analysis of products because the DNA still remains intact. Kleinstiver et al proposed an EQR variant of Cas9 with three amino acid substitutions. This new proposed version of Cas9 targeted an NGAG PAM sequence. Our design was to make these three amino acid substitutions in dCas9 and try to target an NGAG PAM site in GFP.
Kleinstiver et al. recently demonstrated modified spCas9 with altered PAM specificity
. Their results motivated us to explore computational methods of assessing spCas9 mutants for altered PAM specificity profiles. We created an analysis pipeline using python that makes use of PyRosetta, a well known molecular dynamics toolkit. Our suite of scripts is described in our Modeling and Software pages.
A PyMOL generated image of the wild type residues near the PAM binding site of spCas9.
A PyMOL generated image of the three mutations found in the EQR spCas9 variant
A PyMOL generated image of the three mutations found in the VQR spCas9 variant
CRISPR Plant Defense
By integrating an expression cassette that included Cas9 and three sgRNAs that target the CaMV genome in an important coding sequence into the shuttle vector pCAMBIA, we'll be able to integrate that cassette into the Arabidopsis genome using nature's genetic engineer, Agrobacterium.
To model our antiviral application, we looked at the antiviral effects of CRISPR/Cas9 targeting on three scales: CaMV genomes, plant cells and plant leaves. Details of our modeling approach, and how it influenced our design, are provided in our Modeling pages.
CaMV Genomes
Plant Cells
Plant Leaves
References
Top
Sitemap External Site Â© 2015 Waterloo iGEM
