Team:Waterloo/Lab/dCas9
Toggle navigation
Project
Description
Results
Design
Requirements
Lab & Design
Overview
sgRNA Modification
dCas9 Modification
Plant Defense
Parts
Experiments & Protocols
Notebook
Modeling
Overview
CaMV Biology
PAM Flexibility
Cas9 Dynamics
CaMV Replication
Viral Spread
Software
Practices
Overview
Commercialization
Human Practices
Lab Safety
Opinions
Survey
Outreach
Team
Our Team
Attributions
Sponsors
Collaborations
Example Page
dCas9 Modification
Introduction
One of our goals this year was to engineer a version of the Cas9 protein that could recognize different protospacer adjacent motifs (PAMs) other than the regular NGG. Kleinstiver et al.
recently published data on Cas9 mutants that effectively cut unique PAM sites. A specific EQR variant in the publication recognized a NGAG PAM site after three different amino acids in the original Cas9 were changed. The EQR variant includes the changes D1135E, R1335Q, and T1337R. Our goal was to try and replicate these mutations but instead with dCas9, which is commonly used since it represses a gene rather than cutting one. Target sites were designed in promoters of green fluorescent protein (GFP) with both an NGG PAM sequence and an NGAG PAM sequence.
Experimental Design
All of our experiments to study GFP expression with and without addition of dCas9 were performed on the Amnis Image Stream machine (Amnis 2015). The Image Stream is a fluorescence microscope combined with a flow cytometer, letting one to perform a quantitative method of fluorescence detection while visualizing the actual cells. By measuring an average fluorescence intensity of cells, we can determine the average intensity of a cell population.
Our first step was to create the EQR variant in dCas9 by using site directed mutagenesis . This was done on a dCas9 plasmid ordered from AddGene. Two other plasmids were constructed with both a gene producing GFP and an sgRNA target recognizing either an NGG or an NGAG PAM site on the GFP promoter J23101.
The first experiment that was performed was to co-transform a plasmid containing both GFP and the sgRNA targeting the GFP promoter into the same cell as dCas9. This was a control experiment that will explain that the mutated EQR dCas9 performs worse than                 the original dCas9 when targeting an NGG PAM site.
The next experiment involved transforming the plasmid with GFP and an sgRNA targeting a region in the GFP promoter with an NGAG PAM site into E. coli. This was also co-transformed with both the orignial and the mutated versions of dCas9. This time, we expect to see the mutated EQR dCas9 perform better than the original in terms of repression of GFP.
Constructs
Visualization of site-directed mutagenesis protocol, showing modification to dCas9 coding sequence.
GFP with NGAG sgRNA target
Target plasmid with GFP sgRNA
Mutated dCas9
dCas9 from AddGene
Results
The EQR muatated dCas9 was successfully constructed using the QuickChange protocol twice and sending for sequencing. Co-transformations were also accomplished and cells were analyzed using Flow cytometry.
NGG PAM site
NGAG PAM site
These graphs show the results of the mean of the GFP intensity when the co-transformed cells were run on flow cytometry. A lower mean points to the result that the dCas9 is more effective at repressing expression of GFP. These results are inconclusive and we need to repeat the experiment multiple times to understand the effect of the EQR mutations on dCas9.
The results are rather inconclusive, as can be seen even more clearly from the bar graph of the data, shown below. It was expected that the original dCas9 would target NGG but not NGAG, while the modified dCas9 would target NGAG but not NGG. Instead roughly the same signal was measured for both dCas9 variants at both target sites.
Mean RFP Intensity after targeting an NGAG and NGG PAM site (using different complementary sgRNA sequences) with both dCas9 and the modified dCas9 variant.
References
Top
Sitemap External Site Â© 2015 Waterloo iGEM
