<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Waterloo_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Waterloo/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project</A></LI><LI><A href="/Team:Waterloo/Description">Description</A></LI><LI><A href="/Team:Waterloo/Results">Results</A></LI><LI><A href="/Team:Waterloo/Design">Design</A></LI><LI><A href="/Team:Waterloo/Requirements">Requirements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Lab &amp; Design</A></LI><LI><A href="/Team:Waterloo/Lab_Overview">Overview</A></LI><LI><A href="/Team:Waterloo/Lab/sgRNA">sgRNA Modification</A></LI><LI><A href="/Team:Waterloo/Lab/dCas9">dCas9 Modification</A></LI><LI><A href="/Team:Waterloo/Lab/Plants">Plant Defense</A></LI><LI><A href="/Team:Waterloo/Parts">Parts</A></LI><LI><A href="/Team:Waterloo/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="/Team:Waterloo/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Modeling</A></LI><LI><A href="/Team:Waterloo/Modeling">Overview</A></LI><LI><A href="/Team:Waterloo/Modeling/CaMV_Biology">CaMV Biology</A></LI><LI><A href="/Team:Waterloo/Modeling/PAM_Flexibility">PAM Flexibility</A></LI><LI><A href="/Team:Waterloo/Modeling/Cas9_Dynamics">Cas9 Dynamics</A></LI><LI><A href="/Team:Waterloo/Modeling/CaMV_Replication">CaMV Replication</A></LI><LI><A href="/Team:Waterloo/Modeling/Intercellular_Spread">Viral Spread</A></LI><LI><A href="/Team:Waterloo/Software">Software</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Practices</A></LI><LI><A href="/Team:Waterloo/Practices">Overview</A></LI><LI><A href="/Team:Waterloo/Practices/Commercialization">Commercialization</A></LI><LI><A href="/Team:Waterloo/Practices/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Waterloo/Safety">Lab Safety</A></LI><LI><A href="/Team:Waterloo/Practices/Ethics">Opinions</A></LI><LI><A href="/Team:Waterloo/Practices/Survey">Survey</A></LI><LI><A href="/Team:Waterloo/Outreach">Outreach</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team</A></LI><LI><A href="/Team:Waterloo/Team">Our Team</A></LI><LI><A href="/Team:Waterloo/Attributions">Attributions</A></LI><LI><A href="/Team:Waterloo/Sponsors">Sponsors</A></LI><LI><A href="/Team:Waterloo/Collaborations">Collaborations</A></LI><LI><A href="/Team:Waterloo/Example">Example Page</A></LI></DIV></DIV><DIV class="container main-container"><H1>Experiments &amp; Protocols</H1><SECTION id="agroinfiltration" title="Agroinfiltration"><H2>Agroinfiltration</H2><OL><LI>
                Seeds of <I>Arabidopsis thaliana</I></LI><LI><I>Agrobacterium tumefaciens</I> strains such as C58C1
            </LI><LI>
                Luria-Bertani (LB) media (broth and agar plate)
            </LI><LI>
                Induction medium: 10.5 g/L K2HPO4, 1 g/L (NH4)2SO4, 0.5 g/L NaCitrate, 1 g/L glucose, 1 g/L fructose, 4 g/L glycerol, 1 mM MgSO4, 10 mM 2-(N-morpholino) methanesulfonic acid (MES); adjust pH to 5.6, autoclave before use
            </LI><LI>
                Infiltration medium: 10 mM MgSO4, 10 mM MES; adjust pH to 5.6, autoclave before use
            </LI><LI>
                Rifampicin: dissolve in methanol to make 25 mg/mL stock solution and store in -20 C freezer
            </LI><LI>
                Tetracycline: dissolve in 70% ethanol to make 12.5 mg/mL stock solution and store in -20 C freezer
            </LI><LI>
                Kanamycin: dissolve in water to make 50 mg/mL stock solution and store in -20 C freezer
            </LI><LI>
                Acetosyringone (3,5-dimethoxy-4-hydroxyacetophenone, Aldrich): dissolve in 196.2 mg acetosyringone in 1 mL dimethylsulfoxide to make 1 M stock solution and store in -20 C freezer
            </LI><LI>
                Dexamethasone (DEX, USB, Cleveland, OH): dissolve 78.4 mg DEX in 1 mL ethanol to make 200 mM stock solution
            </LI><LI>
                5-bromo-4-chloro-3-indolyl B-D-glucuronide (X-Glu, Sigma, St. Louis, MO): dissolve X-Glu in dimethylformamide to make 200 mM stock solution and store in -20 C freezer
            </LI><LI>
                GUS staining solution: 50 mM sodium phosphate (pH 7.0), 1 mM EDTA, 2 mM X-Glu, 0.05% SDS, 0.1% sodium N-lauryl-sarcosine, 0.1% Triton-X-100
            </LI><LI>Vacuum pump</LI></OL></SECTION><SECTION id="floral-dip" title="Floral Dip"><H2><EM>Arabidopsis</EM> Transformation (Floral Dip)</H2><H3>Day 0</H3><OL><LI>Make Rif/Gm/Km plates</LI></OL><H3>Day 1</H3><OL><LI>
                Electroporate your constructs into <I>Agrobacterium</I>. Make sure you use appropriate antibiotic plates. For our <I>Agrobacterium</I> it needs Rif and Gm and as long as we have pCAMBIA in it we need to add Km. Refer to the lab book with sticky notes.
            </LI></OL><H3>Day 2</H3><OL><LI>Wait</LI></OL><H3>Day 3</H3><OL><LI>
                Streak your plate if you dip on the 8th day. If you are in a rush just inoculate a single colony into 5mL of YEP media using appropriate antibiotic. You can dip in day 6 if you inoculate directly from the transformation plate.
            </LI><LI>Incubate the tubes at 28°C</LI></OL><H3>Day 4</H3><OL><LI>
                Miniprep 1-2mL of your inoculated culture and run a pcr check+diagnostic gel. when minipreping <I>Agrobacterium</I> make sure to wash 3-4 times.
            </LI><LI>Inoculate about 20µl from your last night culture into new 5mL of YEP</LI><LI>
                Make 1-2ml of YEP into 500ml baffled flask (depending on how many constructs you have. for example. One flask for pCAMBIA in <I>Agrobacterium</I>, one for pCAMBIA+sgRNA, one for pCAMBIA+Cas9, one for pCAMBIA+Cas9+sgRNA-A2, one for <I>Agrobacterium</I> alone). Fill up the flasks with 250mL of media and cover it with cap or sponge and foil. Put the media in your flasks and autoclave them together.
            </LI><LI>Inoculate <I>Agrobacterium</I> into 5mL YEP using only Rif and GM</LI></OL><H3>Day 5</H3><OL><LI>Take all your 5mL tubes that you inoculated overnight and all your ready flasks.</LI><LI>Add your antibiotics to the flasks. When you have your constructs add all three Km/Gm/Rif, for your <I>Agrobacterium</I> use only Rif/Gm.</LI><LI>
                From your 5mL tube add 2mL of culture to the 250mL dilution. Do this at 4pm so you can start the dip at noon on the next day.
            </LI><LI>
                Incubate the flasks at 28°C with shaker on. Refer to the lab book for antibiotic concentrations.
            </LI><LI>
                Get all your solutions ready. Do not autoclave your sucrose, it can be made on the day of the dip. Make sure that all other solutions are made. Don't forget the other solutions, but this one needs to be made and autoclaved on this day.
            </LI></OL><H3>Day 6</H3><OL><LI>Get to the lab 1 hour before the dip</LI><LI>Turn on the big centrifuge make sure it is at 4°C.Then take your cultures and spin down for 15min at 4 degrees.</LI><LI>Make sucrose.</LI><LI>Take cultures and read the OD</LI><LI>
                Calculate dilutions and amount of Silwet that needs to be added. For resuspending, add 50ml of solution, and pipette up and down to break the pellet.
            </LI><LI>Have fun dipping! <STRONG>Make sure you autoclave the trays first</STRONG></LI></OL></SECTION><SECTION id="colonypcr" title="Colony PCR"><H2>Colony PCR (cPCR)</H2><H3>Day 1</H3><OL><LI>Set-up an aseptic work area using either a bunsen burner or flow-hood. Use 70% ethanol to clean the bench top.</LI><LI>Use a marker to circle all the colonies you want to screen on the transformation plate(s), and label each one with a number. </LI><LI>Streak purify each of these colonies on LB agar plates with the corresponding antibiotic. Number these plates according to the colonies they came from.</LI><LI>Seal the plate’s edge with parafilm and incubate 37C overnight (at least 12hrs)</LI></OL><H3>Day 2</H3><OL><LI>
                Set-up an aseptic work area using either a bunsen burner or flow-hood. Use 70% ethanol to clean the bench top. Remove plates from the incubator.
            </LI><LI>
                Based on the number of colonies you want to screen, use a permanent marker to draw multiple ~1 cm diameter circles on the LB agar plate. This is the &quot;patch plate.&quot; If you are screening too many colonies to fit on one patch plate, make a second patch plate. Label the circles on the patch plates with numbers that will correspond to the colony they came from.
            </LI><LI>
                Add 20µL of the Cell Lysis Solution to a 1.5mL microfuge tube. Repeat this step for however many colonies you want to screen.
            </LI><LI>
                Scoop up one selected colony from each of the streak purification plates using an inoculation stick.
            </LI><LI>
                Mini-streak the colony into a patch plate circle with the corresponding number.
            </LI><LI>
                Dip the same stick into the Cell Lysis Solution and swirl stick to get all bacteria off the stick. Discard stick into waste jar for autoclaving.
            </LI><LI>Repeat steps 7-10 for all colonies to be screened.</LI><LI>Ensure all the caps of the tubes are closed and vortex quickly to resuspend cells.</LI><LI>
                Place tubes into an 80°C water bath (heat block containing water, or water bath) and let them sit for at least 15 minutes.
            </LI><LI>
                You may also choose to microwave your tubes, caps open, for 30s to guarantee all cells have lysed.
            </LI><LI>
                Vortex tubes again to resuspend cells, and centrifuge at 13,300RPM for 4-5 mins to pellet the cellular debris. Remove from centrifuge when complete.
            </LI><LI>
                Use the following recipe to prepare the colonies' DNA for cPCR:
                </LI><LI>Cell Template 2µL</LI><LI>Forward Primer 1.25µL</LI><LI>Reverse Primer 1.25µL</LI><LI>Q5 HF Master Mix 10µL</LI><LI>PCR MilliQ Water 5.5µL</LI><LI>TOTAL VOLUME 20µL</LI>
                Repeat this for all colonies to be screened. <STRONG>The primers should be designed to amplify your insert. Remember to do a positive AND negative control</STRONG><LI>
                Use the following PCR program to amplify the insert.
                </LI><LI>First Cycle: 98°C for 30s</LI><LI>Second Cycle: 98°C for 10-15s, 68°C for 25-35s, 72°C for [30s/kb]</LI><LI>Third Cycle: 72°C for 3-5m, 4°C Forever and ever.</LI><LI>
                Upon completion of cPCR, run the samples in a diagnostic 1.0% agarose gel. Load 1µL of cPCR product, 4µL of de-ionized water, and 1µL of 6X Loading Dye.
            </LI><LI>Image gel and cry for happiness or sadness.</LI></OL></SECTION><SECTION id="electroporation" title="Electroporation"><H2>Electroporation of <EM>Agrobacterium</EM></H2><OL><LI>Pipette 100-600 ng plasmid into new tube and resuspend in 5µL dH2O, leave on ice</LI><LI>
                Take <I>Agrobacterium</I> electro competent cells and thaw 40µL (1 aliquot per transformation) on ice
            </LI><LI>Put cuvettes on ice before use with electroporator </LI><LI>Turn on electroporator, and set voltage at 1.25V, then press the CONST button </LI><LI>
                Pipette the 5µL plasmid mixture into the 40µL of competent cells. Mix by pipetting up and down a couple times, then transfer to cuvette
            </LI><LI>
                Wipe cuvette with a kimwipe, then transfer to electroporation unit. Press both buttons on the electroporator simultaneously and hold them down until you hear a beep and see a constant time on the screen. Record this value when finished (should be 3.7-4 ideally). <STRONG>Note:</STRONG> Lower values than 3.7 mean the cells were not washed with water sufficiently. If you see a spark or arc it means you have salt in your sample and need to precipitate and rinse it well with 70% ethanol.
            </LI><LI>
                As quickly as possible, at 1mL of YEP media to the cuvette and gently aspirate up and down to mix gently. Transfer to labelled growth tube.
            </LI><LI>Incubate at 27°C for 2 hours</LI><LI>
                Gently mix the culture, pipette 20µL of culture and spreak on YEP agar with appropriate antibiotics and incubate at 28°C (or room temp) for 3 days for colony to grow <STRONG>Note: These should also be in the dark.</STRONG></LI></OL></SECTION><SECTION id="protoplasts" title="Protoplast Transfection"><H2>PEG-Mediated Transfection of <EM>Arabidopsis</EM> Protoplasts</H2><OL><LI>Harvest 20 well-expanded leaves from 3- to 4-week-old plants before flowering</LI><LI>Cut 0.5 to 1 mm leaf strips from the middle part of a leaf on a clean white paper using a fresh, sharp razor blade without tissue crushing at the cutting site. Change the blade after cutting 4-5 leaves.</LI><LI>Transfer the leaf strips to 10 mL of enzyme buffer in a Petri plate by dipping both sides of the strips using a pair of flat-tip foreceps</LI><LI>Vacuum infiltrate the leaf strips for 10-15 minutes in the dark using a desiccator</LI><LI>Continue digestion at room temperature in the dark for &gt;3 hours without shaking. The enzyme solution should turn green after a gentle swirling motion.</LI><LI>Check the release of protoplasts in the solution under a microscope (size = 30-50 um). Check the healthiness of protoplasts (unhealthy protoplasts show chloroplast clumping, irregularity in shape, and sometimes plasmolysis).</LI><LI>Filter the enzyme/protoplast solution through a piece of 75-um nylon mesh into a 50-mL tube. </LI><LI>Divide the protoplasts into two 15 mL tubes and pellet the protoplasts by centrifugation at 100 g (i.e. ~700 rpm) for 2 minutes using the swinging-bucket centrifuge H-20</LI><LI>(Optional) To remove unhealthy protoplasts and cell debris, resuspend the protoplast pellets in 10 mL of CS-Sucrose buffer and float the healthy protoplasts by centrifugation at 100 g for 2 minutes using the swinging-bucket centrifuge H20. Remove the &quot;internatant&quot; and pellet as much as possible without disturbing the floating layer of healthy protoplasts.</LI><LI>Resuspend healthy protoplasts in 1 mL WI buffer and count cells using haemocytometer.</LI><LI>Resuspend the pellet (or floating layer of healthy protoplasts in step 10) in 1 mL of W5 solution by gentle swirling. Incubate on ice for 30 minutes. In the meantime, check the cell number using a haemocytometer. </LI><LI>The protoplasts should have settled to the bottom. Remove the supernatant and resuspend the pellet in Mg-Man buffer to get a concentration of 200 000 protoplasts per mL. Keep at room temperature and check the cell number again.</LI><LI>Add 5-10 µg of plasmid DNA in 10 µL to a 2 mL tube.</LI></OL></SECTION><SECTION id="electrocomp" title="Electrocompetent Cells"><H2>Preparation of Electrocompetent <EM>Agrobacterium</EM></H2><OL><LI> Grow seed culture to saturation (36-48 hours), need approximately 5mL of this </LI><LI> Inoculate 500mL of YEP media at a dilution of 1:100 <I>Agrobacterium</I>, and grow culture to an OD600 of 1.5 (takes approximately 24 hours)</LI><STRONG>Note: The rest of the protocol should be completed on ice</STRONG><LI> Spin cells at 5000 rpm for 15 minutes, resuspend in 500 mL cold, sterile, H2O </LI><LI> Spin cells at 5000 rpm for 15 minutes, resuspend in 250 mL cold, sterile, H2O</LI><LI> Spin cells at 5000 rpm for 15 minutes, resuspend in 10 mL sterile 10% (v/v) glycerol</LI><LI> Spin cells at 6000 rpm for 15 minutes, resuspend (wash) in 0.5 mL cold, sterile 10% (v/v) glycerol. Pipette the glycerol onto the cells and gently stir until well mixed. Cells will be viscous, and the final volume will be 0.003X original culture volume. </LI><LI> Dispense cells in 40µL aliquots into screw top tubes - sample is viscous so try to use a P1000 and it will make pipetting quicker and less difficult </LI><LI> Store at -80C and thaw on ice before use </LI></OL></SECTION><SECTION id="quick" title="Quick Change"><H2>Quick Change</H2><OL><LI>Synthesize two complementary oligonucleotides containing desired mutation, flanked by unmodified nucleotide sequence.</LI><LI>Prepare control reaction as follows:
                </LI><LI>5 µL of 10x reaction buffer</LI><LI>2 µL (10 ng) of pWhitescript 4.5kb control plasmid (5 ng/µL)</LI><LI>1.25 (125 ng) of oligonucleotide control primer #1 (34-mer, 100 ng/µL)</LI><LI>1.25 µL (125 ng) of oligonucleotide control primer #2 (34-mer, 100 ng/µL)</LI><LI>1 µL of dNTP mix</LI><LI>38.5 µL ddH20 </LI><LI>Total reaction volume should be 50 µL</LI><LI>Then add 1 µL <I>PfuUltra</I> HF DNA Polymerase (2.5 U/µL)</LI><LI>Prepare the sample reactions as follows:
                </LI><LI>5 µL of 10x reaction buffer</LI><LI><I>X</I>µL (5-50ng) of dsDNA template</LI><LI><I>X</I>µL (125 ng) of oligonucleotide primer #1</LI><LI><I>X</I>µL (125 ng) of oligonucleotide primer #2</LI><LI>1 µL of dNTP mix</LI><LI>ddH20 to final volume of 50 µL</LI><LI>Then add 1 µL <I>PfuUltra</I> HF DNA Polymerase (2.5 U/µL)</LI><LI>Cycle each reaction using the parameters below:
                </LI><LI>Segment 1: 1 cycle at 95 C for 30s</LI><LI>Segment 2: 12 cycles at 95 C for 30s, then 1 minute at 55 C, then 1 minute/kb of plasmid length at 68 C</LI><LI>Place the reaction on ice for 2 minutes to cool the reaction</LI><LI>Add 1 µL of DPNI to each reaction tube, and place in 37 C incubator for one hour</LI><LI>Transform this reaction mixture, and send for sequencing following colony growth</LI></OL></SECTION><SECTION id="western" title="Western Blot"><H2>Western Blot</H2><H3> Gel Preparation: </H3><OL><LI>Clean and scrub glass plates with ethanol and Kimwipes.</LI><LI>Parafilm bottom of glass cassettes and test leakage with deionized water.</LI><LI>Cast resolving gel, overlay with ~300 µL, and let stand for &gt;= 15 minutes. Do not move.</LI><LI>Rinse with deionized water and cast stacking gel, topping up if need be. Water with low concentrations of acrylamide will evaporate noticeably during solidification.</LI><LI>Wrap in wet paper towel and saran wrap, and store at 4°C for up to one week.</LI></OL><H3>Gel Electrophoresis: </H3><OL><LI>
                Test cassettes for leakage by filling up with 1X Running Buffer or DI water. Piece together the apparatus you are working with, and fill whole rig with 1X Running Buffer. Ensure that the gel cassette remains topped up throughout electrophoresis.
            </LI><LI>
                Assuming samples have been quantitated and denatured, add amounts to the wells of gel. Keep samples on ice and vortex immediately before addition. Add slowly and do not pipet any air bubbles into buffer/well. Work quickly to limit diffusion of samples through stacking gel. Do not forget a ladder, used unstained if proceeding to Coomassie.
                </LI><LI>
                        If doing sensitive densitometry, leave a lane between your ladder and first lane. Some chemiluminescent reactions such as HRP-luminol will cause the ladder to fluoresce and interfere with background signal.
                    </LI><LI>Connect to power supply, and choose your voltage. Some tips for choosing the right voltage:
                </LI><LI>
                        The higher the voltage, the faster the run and poorer the resolution. You will get separation, but heat can cause &quot;smiling&quot; of bands, smearing due to melting, or even cracks in glass plates. 160V is considered high but doable. <STRONG>A 12% gel will complete within 75 minutes</STRONG></LI><LI>
                        The lower the voltage, the slower the run,but resolution is markably better. Low voltage is always a safe bet, and can allow one to even see phosphorylation states of proteins. 50-80V is considered low. <STRONG> A 12% gel will complete with 2.5-3 hours. </STRONG></LI><LI>
                        A good median to use is 100-120V. Phosphorylation states may not be seen, but resolution will be acceptable. <STRONG> A 12% gel will complete within 1.5-2 hours </STRONG></LI><LI>
                During electrophoresis, prepare Whatmann paper, nitrocellulose, transfer cassettes, and <STRONG> 1X Transfer Buffer </STRONG> for Western Blot. Refer ahead to &quot;Wet protein transfer.&quot;
            </LI><LI>
                Gently separate glass plates by using a razor on the corners and gently prying open. If at any point force is needed, stop – the plates may break. Cut off any unused lanes with a razor by pushing down in straight lines. Never drag a razor, as the gel is very delicate and will rip.
            </LI><LI>If proceeding to Coomassie staining, submerse the gel in Coomassie and follow company’s protocol.</LI><LI>Buffer can be sealed, stored in the fridge, and reused up to 4 times.</LI></OL><H3>Wet Protein Transfer:</H3><P>It is essential that from this point, both gel and membrane are kept wet <STRONG> until they are discarded </STRONG></P><OL><LI>
                In a tray or Pyrex dish, lay down a transfer cassette <STRONG>with the black side underneath</STRONG>. Put down a sponge and cut a piece of Whatmann paper, and cover with <STRONG>1X Transfer Buffer</STRONG>. The Whattman paper should be rectangular and larger than the gel, and the nitrocellulose should be large enough to completely cover the gel.
            </LI><LI>
                At the same time, place nitrocellulose in a tray of <STRONG>1X Transfer Buffer</STRONG> until ready to use.
            </LI><LI>             The gel has likely stuck to one glass plate during separation. Apply the gel to the Whatmann paper by pressing it while the paper is on the sponge and wet with <STRONG>1X Transfer Buffer.</STRONG> One can easily peel the gel off the glass and form a smooth adhesion between the paper and gel. Take care as to not rip the gel – sometimes it may stick to impurities on the glass and rip.
            </LI><LI>
                Using tweezers, place nitrocellulose on the gel and roll flat to remove any air bubbles. A 15 mL tube or Pasteur pipette barrel work great for this. <U>Air bubbles are the bane of your existence!</U>Two passes each way will suffice.
            </LI><LI>Place Whatmann paper on top of nitrocellulose and roll flat again.</LI><LI>Place sponge on top and close cassette. Ensure the whole cassette is wet.</LI><LI>
                Place in cassette in proper orientation with electrode (black to black) and fill the remaining <STRONG> 1X Transfer Buffer</STRONG>. Add stir bar and run at 50V. It is essential that this process be kept cool by both addition (and replenishment) of ice packs or by running in a 4°C environment. Complete transfer takes no more than 3 hours in these parameters, or can be done overnight in a cold room.
            </LI><LI>
                One can check the status of the transfer by pausing, and carefully separating the ladder-side corner of the membrane from the gel to see how much has transferred. High molecular weight protein takes longer than low molecular weight protein, and can easily be seen by the ladder.
            </LI><LI>
                When complete, discard Whatmann paper, gel and buffer and place transferred membrane in a Western blotting tray. Trim any excess membrane that wasn’t in contact with the gel by razor or scissors.
            </LI><LI>
                Some laboratories will discard the buffer through biohazardous waste since it contains methanol, but most will flush down the drain. Consult your technician.
            </LI></OL><H3>Western Blotting Protocol</H3><OL><LI>Quickly stain blot in an excess of Ponceau S. Leave stationary for 5-10 minutes.</LI><LI>Drain Ponceau S back into stock, and rinse in DI water. This can be done by gently flooding the membrane and allowing it to sit for 5 minutes. The excess stain will lift off, while protein will be stained a bright pink.</LI><LI>Discard and flood again 2 more times for 10 seconds to remove excess stain.</LI><LI>Submerge in a final flood and image on a scanner. This is only to keep the blot wet.</LI><LI>Destain with shaking in enough <STRONG>1X TBS-T</STRONG> to cover the surface of the blot. This is generally 10 mL in a 10x10 cm tray.</LI><LI>When the stain disappeared, apply <STRONG>Blocking Buffer</STRONG> to blot and incubate for 1 hour (room temperature with gentle shaking) or overnight (4°C, static).</LI><LI>Discard <STRONG>Blocking Buffer</STRONG> and apply <STRONG>Primary Antibody</STRONG>. Incubate with gentle shaking at RT for 1-2 hours.
                </LI><LI>
                        Some antibodies are very strong and only require an hour or less. Some are very weak and require overnight incubations. Troubleshoot for the optimal incubation time with your antibody and the dilution used.</LI><LI>Remove <STRONG>Primary Antibody</STRONG> and wash with <STRONG>1X TBS-T </STRONG> 3 times for 5 minutes each.</LI><LI>Apply <STRONG>Secondary Antibody</STRONG> and incubate with gentle shaking at RT for 1-2 hours. </LI><LI>Remove <STRONG>Secondary Antibody</STRONG> and wash with <STRONG>1X TBS-T</STRONG> 3 times for 5 minutes each. </LI><LI>Develop or image your blot
                </LI><LI><EM>Chemiluminescence – HTP</EM>: Air-dry the blot by grabbing by the ladder end and gently flapping the blot back and forth until complete dry. Apply ECL, wait 5 minutes in the dark, and image.
                    </LI><LI><EM>Chemiluminescence – AP</EM>:Incubate blot in AP buffer for 5 minutes, drain, then add <STRONG> AP Working Solution</STRONG> in the dark and wait until purple bands appear. Flood with DI water for ~10 minutes to stop reaction.
                    </LI><LI><EM>Fluorophore – Alexa</EM>: Incubate <STRONG>Secondary Antibody</STRONG> containing steps in the dark. Image with BioRad Typhoon
                    </LI><LI>Blots can be stored <STRONG> Blocking Buffer </STRONG> in the fridge.</LI></OL><H3>Reagent Recipes</H3><H4>Lysis (RIPA) buffer:</H4><UL><LI>50 mM Tris-HCl, pH 8</LI><LI>150 mM NaClDEvelop </LI><LI>1% NP-40</LI><LI>0.5% Na-Deoxycholate </LI><LI>0.1% SDS </LI><LI> 2 µL Protease Inhibitor Cocktail per mL of buffer, added at time of lysis </LI></UL><H4>Running Buffer:</H4><UL><LI>72 g Glycine</LI><LI>15 g Tris Base</LI><LI>5 g SDS</LI></UL><OL><LI>Dissolve in 500 mL DI water and pH to 8-8.3</LI><LI>Dilute 1:10 in DI water for 1X Running Buffer </LI></OL><H4>10X Transfer Buffer:</H4><UL><LI>72 g Glycine</LI><LI>5 g Tris Base </LI></UL><OL><LI>Dissolve in 500 mL DI water. </LI></OL><H4>1X Transfer Buffer:</H4><UL><LI>100 mL 10X Transfer Buffer</LI><LI>200 mL Methanol </LI><LI>700 mL DI water </LI></UL><H4>10X TBS:</H4><UL><LI>40 g NaCl </LI><LI>1 g KCl </LI><LI> 15 g Tris Base </LI></UL><OL><LI>Dissolve in 500 mL DI water </LI><LI>For 1X TBS-T: Dilute 1:10 in DI water, and add 1 mL of Tween-20 per 1L</LI></OL><H4>Blocking Buffer and Antibody Solutions:</H4><UL><LI>Blocking Buffer: 0.5 g skim milk in 10 mL 1X TBS-T</LI><LI>
                Antibody Solutions: Specified dilution in 10 mL Blocking Buffer – 2 mg of Sodium Azide can be added to <STRONG>in primary antibody solution</STRONG> to limit bacterial growth.
            </LI></UL><H4>AP Buffer:</H4><UL><LI>6.06 Tris Base </LI><LI>2.9 NaCl</LI><LI>0.51 g magnesium chloride</LI></UL><OL><LI>Dissolve in 500 mL DI water and pH to 9.5</LI></OL><H4>Working Solution:</H4><UL><LI>66 µL NBT </LI><LI>33 µL BCIP </LI><LI>10 mL AP Buffer </LI></UL></SECTION></DIV><A href="#0" class="cd-top">Top</A><FOOTER><P class="footer-links" style="font-size:0.8em;"><A href="https://2015.igem.org/Team:Waterloo/Sitemap" style="font-size:1em;">Sitemap</A><A style="padding-right:0;font-size:1em;" href="http://igem.uwaterloo.ca">External Site</A>© 2015 Waterloo iGEM
    </P></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>