<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Waterloo_Modeling_CaMV_Replication skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Waterloo/Modeling/CaMV Replication</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project</A></LI><LI><A href="/Team:Waterloo/Description">Description</A></LI><LI><A href="/Team:Waterloo/Results">Results</A></LI><LI><A href="/Team:Waterloo/Design">Design</A></LI><LI><A href="/Team:Waterloo/Requirements">Requirements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Lab &amp; Design</A></LI><LI><A href="/Team:Waterloo/Lab_Overview">Overview</A></LI><LI><A href="/Team:Waterloo/Lab/sgRNA">sgRNA Modification</A></LI><LI><A href="/Team:Waterloo/Lab/dCas9">dCas9 Modification</A></LI><LI><A href="/Team:Waterloo/Lab/Plants">Plant Defense</A></LI><LI><A href="/Team:Waterloo/Parts">Parts</A></LI><LI><A href="/Team:Waterloo/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="/Team:Waterloo/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Modeling</A></LI><LI><A href="/Team:Waterloo/Modeling">Overview</A></LI><LI><A href="/Team:Waterloo/Modeling/CaMV_Biology">CaMV Biology</A></LI><LI><A href="/Team:Waterloo/Modeling/PAM_Flexibility">PAM Flexibility</A></LI><LI><A href="/Team:Waterloo/Modeling/Cas9_Dynamics">Cas9 Dynamics</A></LI><LI><A href="/Team:Waterloo/Modeling/CaMV_Replication">CaMV Replication</A></LI><LI><A href="/Team:Waterloo/Modeling/Intercellular_Spread">Viral Spread</A></LI><LI><A href="/Team:Waterloo/Software">Software</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Practices</A></LI><LI><A href="/Team:Waterloo/Practices">Overview</A></LI><LI><A href="/Team:Waterloo/Practices/Commercialization">Commercialization</A></LI><LI><A href="/Team:Waterloo/Practices/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Waterloo/Safety">Lab Safety</A></LI><LI><A href="/Team:Waterloo/Practices/Ethics">Opinions</A></LI><LI><A href="/Team:Waterloo/Practices/Survey">Survey</A></LI><LI><A href="/Team:Waterloo/Outreach">Outreach</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team</A></LI><LI><A href="/Team:Waterloo/Team">Our Team</A></LI><LI><A href="/Team:Waterloo/Attributions">Attributions</A></LI><LI><A href="/Team:Waterloo/Sponsors">Sponsors</A></LI><LI><A href="/Team:Waterloo/Collaborations">Collaborations</A></LI><LI><A href="/Team:Waterloo/Example">Example Page</A></LI></DIV></DIV><DIV class="container main-container"><H1>CaMV Replication</H1><SECTION id="motivation" title="Motivation"><DIV class="row"><DIV class="col-sm-8">
        Intracellular replication of CaMV may be strongly affected by the addition of CRISPR/Cas9 and in turn may influence viral spread. Therefore, it's crucial for us to accurately model the replication process to predict viral production over time. In this way, our model of replication will connect our work on <A href="https://2015.igem.org/Team:Waterloo/Modeling/Cas9_Dyanmics">CRISPR/Cas9 Dynamics</A> with our agent-based modelling of <A href="https://2015.igem.org/Team:Waterloo/Modeling/Intercellular_Spread">Intercellular Viral Spread</A>. The first goal of the model is to demonstrate the reduction of viral load over time due to CRISPR/Cas9 modification of ORFVI. This reduction in viral load will hopefully allow for plant defenses to effectively contain the virus before the onset of systemic infection.</DIV><DIV class="col-sm-4"><FIGURE><FIGCAPTION class="model-caption">Plant Cells</FIGCAPTION></FIGURE></DIV></DIV><P>This leads to our second goal which is to integrate the intracellular replication and intercellular spread models to fully understand impact of CRISPR/Cas9. Altogether, we are attempting to demonstrate the feasibility of our anti-viral system and use our findings to direct the project design. </P></SECTION><SECTION id="formation" title="Model Formation"><H2>Model Formation</H2><DIV class="row"><DIV class="col-sm-8"><P>There are several core processes involved in CaMV replication within an infected host cell. The details of CaMV replication are discussed on the <A href="https://2015.igem.org/Team:Waterloo/Modeling/CaMV_Biology">CaMV Biology</A> page and shown in the figure. In brief, however, we determined that in addition to typical processes such as transcription, constitutive translation, and degradation we must also account for the following processes in our model:</P><UL><LI>Virions reinfecting the nucleus</LI><LI>Repair of pdsDNA</LI><LI>Splicing of 35S RNA</LI><LI>RNA interference</LI><LI>Packaging of 35S RNA</LI><LI>P6 transactivating production of P1-P5</LI><LI>Anchoring of P3 to the viral capsid</LI><LI>Virions exiting the cell</LI></UL><P>Additionally, since our project uses CRISPR to produce a targeted mutation in the P6 gene, we track both wild-type and mutated versions of genomes, pregenomes, and virions.</P></DIV><DIV class="col-sm-4"><FIGURE><DIV class="img-att">
                i
                <UL class="img-att-bubble"><LI>Photo ©<A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1364-3703.2002.00136.x/full">Haas et al. (2005)</A></LI><LI>Figure 3 of <EM>Cauliflower mosaic virus</EM>: still in the news </LI></UL></DIV></FIGURE></DIV></DIV><H3>Network and Equations</H3><P>The replication pathway of CaMV, simplified to contain only the processes of interest, is shown in the figure below. Gapped CaMV genomes ($g_{gap}$) are closed ($g_{cc}$) and modified by CRISPR/Cas9 ($g_{m.gap}$, $g_{m.cc}$). The modified genomes produce 35S transcripts $35S_m$, but do not produce P6, which is needed to activate translation of the $35S$ transcript into $P3$, $P4$ and $P5$. Proteins $P1$ and $P2$ are disregarded because they mostly play a role in aphid transmission, which we did not model.</P><FIGURE><FIGCAPTION>CaMV replication pathway, showing main steps of CRISPR/Cas9 modification, transcription, translation and viral packaging.</FIGCAPTION></FIGURE><P>The proteins produced from either 35S transcript ($35S$ or $35S_m$) may be used to package virions, though virions produced from $35S_m$, while they are packaged and transported to other cells, will not be able to initiate infection due to their deactivated P6 gene. It should be noted that transport in and out of the nucleus is not modelled explicitly, though certain species only reside in the nucleus and others only in the cytoplasm.</P><P>The principles of mass-action kinetics, reviewed by Ingalls , were used to extract a series of ordinary differential equations from this network. The construction of the network and extraction of equations required a number of assumptions, which are detailed below.</P><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="systemODEs"><H3 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#bio background" href="#collapseODEs" aria-expanded="false" aria-controls="collapseODEs">
                        List of Ordinary Differential Equations
                        <SPAN class="hidden-xs toggle-arrow pull-right">Panel Toggle</SPAN></A></H3></DIV><DIV id="collapseODEs" class="panel-collapse collapse" role="tabpanel" aria-labelledby="systemODEs"><DIV class="panel-body"><H4>State Variables</H4><UL><LI>$d_g$: Gapped DNA in nucleus</LI><LI>$d_{cc}$: cccDNA in nucleus</LI><LI>$d_{m.g}$: Modified gapped DNA in nucleus</LI><LI>$d_{m.cc}$: Modified cccDNA in nucleus</LI><LI>$r_{19S}$: 19S RNA</LI><LI>$r_{35S}$: 35S RNA</LI><LI>$r_{35Sm}$: Modified 35S RNA</LI><LI>$P_3$: Protein 3</LI><LI>$P_4$: Protein 4</LI><LI>$P_5$: Protein 5</LI><LI>$P_6$: Protein 6</LI><LI>$V_i$: Intermediate virions</LI><LI>$V_{im}$: Intermediate virions with modified genome</LI><LI>$V_m$: Modified virions</LI><LI>$V$: Virions</LI></UL><H4>CaMV DNA</H4><P>CaMV DNA is present in both gapped and ungapped forms in the nucleus and may be converted to &quot;modified&quot; form by CRISPR/Cas9, which removes its P6 activity. There is assumed to be a carrying capacity for viruses within the nucleus because of limited cellular resources and the nucleus may be reinfected by virions packaged in the cytoplasm.</P><P>
                    $$\frac{d d_g}{dt} = k_v V (d_{max} - d_{total}) - \alpha_c d_g - k_g d_g - \gamma_d d_g$$
                    $$\frac{d d_{cc}}{dt} = \alpha_c d_g - k_c d_{cc} - \gamma_d d_{cc}$$
                    $$\frac{d d_{m.g}}{dt} = k_v V_m (d_{max} - d_{total}) - \alpha_c d_{m.g} + k_g d_g - \gamma_d d_{m.g}$$
                    $$\frac{d d_{m.cc}}{dt} = \alpha_c d_{m.g} + k_c d_{cc} - \gamma_d d_{m.cc}$$
                    </P><H4>RNA</H4><P>We consider three transcripts, all assumed to be of importance only in the cytoplasm. The 19S mRNA transcript, the 35S transcript and the modified 35S transcript. These transcripts are degraded both passively, according to the $\gamma_{19S}$ and $\gamma_{35S}$ parameters, and actively by the plant RNAi defenses, captured in the $\gamma_{r}$ term. The 35S transcripts are also packaged into virions by a P4/P5 complex present on the inclusion bodies.</P><P>
                    $$\frac{d r_{19S}}{dt} = \alpha_{19S} d_{cc} - (\gamma_{19S}+\gamma_{r}) r_{19S}$$
                    $$\frac{d r_{35S}}{dt} = \alpha_{35S} d_{cc} - k_p p_4 p_5 f_u r_{35S} - (\gamma_{35S}+\gamma_{r}) r_{35S}$$
                    $$\frac{d r_{35Sm}}{dt} = \alpha_{35S} d_{m.cc} - k_p p_4 p_5 f_u r_{35Sm} - (\gamma_{35S}+\gamma_{r}) r_{35Sm}$$
                    </P><H4>Protein</H4><P>The concentrations of four proteins are modelled. Translation of P3, P4 and P5 from the either of the 35S transcripts occurs because of activation by P6, while P6 is translated from the 19S transcript. All transcripts proteins are degraded and P3, P4 and P5 participate in viral packaging.</P><P>
                    $$\frac{d p_3}{dt} = \beta_3 \left( \frac{p_6}{p_6+K_6} \right) (r_{35S} + r_{35Sm}) - k_a p_3 (V_i+V_{im}) - \delta_3 p_3$$
                    $$\frac{d p_4}{dt} = \beta_4 \left( \frac{p_6}{p_6+K_6} \right) (r_{35S} + r_{35Sm}) - k_p p_4 p_5 f_u (r_{35S} + r_{35Sm}) - \delta_4 p_4$$
                    $$\frac{d p_5}{dt} = \beta_5 \left( \frac{p_6}{p_6+K_6} \right) (r_{35S} + r_{35Sm}) - k_p p_4 p_5 f_u (r_{35S} + r_{35Sm}) - \delta_5 p_5$$
                    $$\frac{d p_6}{dt} = \beta_6 r_{19S} - \delta_6 p_6$$
                    </P><H4>Virions</H4><P>The concentrations of four proteins are modelled. Translation of P3, P4 and P5 from the either of the 35S transcripts occurs because of activation by P6, while P6 is translated from the 19S transcript. All transcripts proteins are degraded and P3, P4 and P5 participate in viral packaging. Due to the complicated nature and relative lack of characterization of viral packaging, we decided in our model to combine the packaging reaction into one term, dependent on P4, P5, and unspliced 35S RNA concentration.</P><P>
                    $$\frac{d V_i}{dt} = k_p p_4 p_5 f_u r_{35S} - k_a p_3 V_i$$
                    $$\frac{d V}{dt} = k_a p_3 V_i - k_v V (d_{max} - d_{total}) - v_e V - \delta_v V$$
                    $$\frac{d V_{im}}{dt} = k_p p_4 p_5 f_u r_{35Sm} - k_a p_3 V_{im}$$
                    $$\frac{d V_m}{dt} = k_a p_3 V_{im} - k_v V_m (d_{max} - d_{total}) - v_e V_m - \delta_v V_m$$
                    </P><H4>Algebraic Equations</H4><P>Not all equations pertinent to the dynamics are differential: in particular, the carrying capacity of genomes in the nucleus and effect of RNAi are modelled using algebraic equations.</P><P>
                    $$d_{total} = d_g + d_{cc} + d_{m.g} + d_{m.cc}$$
                    $$\gamma_r = \frac{L}{1+e^{k (p_6-x_0)}}$$
                    $$x_0 = \frac{1}{2} p_6 ^{ss}$$
                    $$p_6 ^{ss} = \frac{\beta_6}{\delta_6} \frac{\alpha_{19}}{\gamma_{19}} d_{max}$$
                    </P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="modelAssumptions"><H3 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#bio background" href="#collapseAssumptions" aria-expanded="false" aria-controls="collapseAssumptions">
                         Assumptions Contained in ODE Equations
                        <SPAN class="hidden-xs toggle-arrow pull-right">Panel Toggle</SPAN></A></H3></DIV><DIV id="collapseAssumptions" class="panel-collapse collapse" role="tabpanel" aria-labelledby="modelAssumptions"><DIV class="panel-body"><UL><LI>Cell concentrations are continuous</LI><LI>Molecules in the cytosplasm are well-mixed</LI><LI>No outside infection</LI><UL><LI>This ODE model only tracks replication within one cell, it cannot track multiple cells. This is handled by the viral spread model instead</LI></UL><LI>Limited number of genomes in nucleus</LI><UL><LI>&quot;Another pool of viral genomes, in the order of 10-100 copies of minichromosomes comprising supercoiled circular viral DNA and host histones, accumulates in the nucleus.&quot; </LI></UL><LI>Rate of repair of gapped DNA follows law of mass-action</LI><LI>Rate of P6 gene mutation is proportional to number of wild-type genomes</LI><LI>DNA, RNA, proteins, and complete virions degrade, incomplete virions do not</LI><UL><LI>We assume P3 binds to virions rapidly enough for incomplete virion degradation to be negligible</LI></UL><LI>We are only targeting 19S RNA</LI><LI>RNA production follows mass-action</LI><LI>Concentration of spliced/unspliced RNA is at rapid equilibrium</LI><LI>Spliced and unspliced 35S RNA degrade at the same rate</LI><LI>Encapsidation and reverse transcription occur simultaneously</LI><UL><LI>Information for CaMV is lacking, but for HBV (another pararetrovirus) reverse transcription is initiated during encapsidation and encapsidation is initiated by RT . The complexity of this packaging process is not fully captured in this model</LI></UL><LI>Only unspliced RNA is packaged</LI><UL><LI>Spliced RNA lacks P1 and P2 and so, although able to replicate within the cell, virions which package spliced RNA will be unable to propagate</LI></UL><LI>P1 and P2 do not affect replication dynamics</LI><UL><LI>P1 is primarily involved in cell-to-cell movement and P2 with host-to-host movement. Neither have a significant impact on the replication process</LI></UL><LI>P6 is only translated from 19S RNA, translation rate is proportional to cccDNA copy number</LI><LI>P3, P4, and P5 are translated from 35S RNA, translation is activated by P6</LI><UL><LI>This is very well established </LI></UL><LI>All P6 is incorporated in inclusion bodies</LI><LI>Mass-action anchoring of P3 to virions</LI><LI>All P4 is instantaneously spliced</LI><UL><LI>As discussed above the subtleties of encapsidation are not captured by this model</LI></UL><LI>Virions may reinfect nucleus</LI><UL><LI>This is very well established </LI></UL><LI>Virions leave the cell at a constant rate</LI><H3>Model Parameters</H3><P>
        Finding and implementing accurate parameter values is crucial when trying to draw conclusions from a model. Our model contains nearly thirty parameters, each of which needed to be found to exactly reproduce the viral replication process <EM>in silico</EM>. Model dynamics may be drastically affected by our choice in parameters and this must be accounted for in our analysis. However, as with many models in systems biology our network suffers from the &quot;parameter problem&quot; described in chapter 1 of the book by Gunawardena  which arises from the complexity of biological processes and the difficulty in measuring parameter values.</P><P>A number of parameter values are the same as those used in an HBV model developed by Nakabayashi &amp; Sasaki . Parameters that have a reference in the &quot;Source&quot; column are ones we were able to find or derive accurate values for from the literature. Unfortunately, there are a few parameter values that could not be found (e.g. the half-saturation constant of P6 activation) and so we explicitly mention the absence of data in the table. In order to compensate for the lack of accurate parameters, we analyze our model over a wide range of parameters to determine how behaviour will be affected.</P><P> Lastly, for simplicity we normalized each concentration by the volume of the cell (volume=1.414x10$^{-13}$m$^3$).</P><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="paramTable"><H3 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#biobackground" href="#collapseTable" aria-expanded="false" aria-controls="collapseTable">
                         Full Table of Parameters
                        <SPAN class="hidden-xs toggle-arrow pull-right">Panel Toggle</SPAN></A></H3></DIV><DIV id="collapseTable" class="panel-collapse collapse" role="tabpanel" aria-labelledby="paramTable"><DIV class="panel-body"><DIV class="table-responsive"><TABLE class="table table-condensed"><THEAD><TR><TH>Symbol</TH><TH>Value</TH><TH>Units</TH><TH>Description</TH><TH>Source</TH></TR></THEAD><TBODY><TR><TD>$k_v$</TD><TD>0.1</TD><TD>min$^{-1}$</TD><TD>Rate at which virions produced by the cell reinfect the nucleus.</TD><TD>No source found, parameter is further analyzed in the results section</TD></TR><TR><TD>$d_{max}$</TD><TD>100</TD><TD>molecules/volume</TD><TD>Maximum concentration of viral genomes in the nucleus.</TD><TD>&quot;Another pool of viral genomes, in the order of 10-100 copies of minichromosomes comprising supercoiled circular viral DNA and host histones, accumulates in the nucleus&quot; </TD></TR><TR><TD>$\alpha_c$</TD><TD>0.1</TD><TD>min$^{-1}$</TD><TD>Rate at which gaps are repaired in gapped DNA to form cccDNA.</TD><TD>HBV model by Nakabayashi &amp; Sasaki </TD></TR><TR><TD>$k_g$</TD><TD>0.01</TD><TD>min$^{-1}$</TD><TD>Rate at which the P6 gene on the gapped DNA is modified.</TD><TD>No source found, parameter is further analyzed in the results section</TD></TR><TR><TD>$k_c$</TD><TD>0.01</TD><TD>min$^{-1}$</TD><TD>Rate at which the P6 gene on cccDNA is modified.</TD><TD>No source found, parameter is further analyzed in the results section</TD></TR><TR><TD>$\gamma_d$</TD><TD>0.001</TD><TD>min$^{-1}$</TD><TD>DNA Degradation rate.</TD><TD>HBV model by Nakabayashi &amp; Sasaki </TD></TR><TR><TD>$\alpha_{19S}$</TD><TD>0.01</TD><TD>min$^{-1}$</TD><TD>Transcription rate of 19S RNA.</TD><TD>HBV model by Nakabayashi &amp; Sasaki </TD></TR><TR><TD>$\alpha_{35S}$</TD><TD>0.09</TD><TD>min$^{-1}$</TD><TD>Transcription rate of 35S RNA.</TD><TD>HBV model by Nakabayashi &amp; Sasaki </TD></TR><TR><TD>$\gamma_{19S}$, $\gamma_{35S}$</TD><TD>0.001</TD><TD>min$^{-1}$</TD><TD>Degradation rate of 19S RNA, 35S RNA.</TD><TD>HBV model by Nakabayashi &amp; Sasaki </TD></TR><TR><TD>$f_u$</TD><TD>0.3</TD><TD>unitless</TD><TD>Fraction of unspliced 35S RNA in the cell (assumed at equilibrium).</TD><TD>Froissart et al. 2004 <CITE ref="Froissart2004">&lt;\cite&gt; and Bouton et al. 2015 <CITE ref="Bouton2015">&lt;\cite&gt;</CITE></CITE></TD></TR><TR><TD>$\beta_3$, $\beta_4$, $\beta_5$, $\beta_6$</TD><TD>0.1</TD><TD>min$^{-1}$</TD><TD>Translation rate of P3, P4, P5, P6.</TD><TD>HBV model by Nakabayashi &amp; Sasaki </TD></TR><TR><TD>$K_6$</TD><TD>5000</TD><TD>molecules/volume</TD><TD>Half-saturation constant for transactivation of P1-P5 production.</TD><TD>No source found, parameter is further analyzed in the results section</TD></TR><TR><TD>$\delta_3$, $\delta_4$, $\delta_5$, $\delta_6$</TD><TD>0.001</TD><TD>min$^{-1}$</TD><TD>Degradation rate of P3, P4, P5, P6.</TD><TD>HBV model by Nakabayashi &amp; Sasaki </TD></TR><TR><TD>$k_p$</TD><TD>0.1</TD><TD>molecules$^{-2}$ min$^{-1}$</TD><TD>Packaging rate.</TD><TD>No source found, parameter is further analyzed in the results section</TD></TR><TR><TD>$k_a$</TD><TD>1</TD><TD>molecules$^{-1}$min$^{-1}$ (converted)</TD><TD>Rate of P3 anchoring to virions.</TD><TD>No source found, parameter is further analyzed in the results section</TD></TR><TR><TD>$v_e$</TD><TD>0.1</TD><TD>min$^{-1}$</TD><TD>Rate at which virions exit the cell.</TD><TD>No source found, parameter is further analyzed in the results section</TD></TR><TR><TD>$\delta_v$</TD><TD>0.001</TD><TD>min$^{-1}$</TD><TD>Rate of virion degradation.</TD><TD>HBV model by Nakabayashi &amp; Sasaki </TD></TR><TR><TD>$L$</TD><TD>0.005</TD><TD>min$^{-1}$</TD><TD>Maximum RNAi effectiveness.</TD><TD>Estimated to be on the order of $\gamma_{35S}$, but slightly larger since RNAi is more effective than degradation</TD></TR><TR><TD>$k$</TD><TD>10$^{-4}$</TD><TD>molecules$^{-1}$</TD><TD>Slope of logistic function describing RNAi effectiveness</TD><TD>No source found, parameter is further analyzed in the results section.</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></SECTION><SECTION id="results" title="Results"><H2>Results</H2><H3>Time Course of CaMV Infection in a Single Cell</H3><P>The effects of RNAi and CRISPR/Cas9 the time course of an infection by CaMV in a single cell were explored.</P><P>Time Series</P><FIGURE><FIGCAPTION>Figure 1 - Time series data of k = $10^{-5}$ and varied genome modification rates </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 2 - Time series data of k = $10^{-4}$ and varied genome modification rates </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 3 - Time series data of k = $10^{-3}$ and varied genome modification rates </FIGCAPTION></FIGURE><P>In descending graphs, RNAi effectiveness is increased by a factor of 10.   Likewise, shifting to the right increases the CRISPR genome modification rate by a factor of 10.  Three observations can be made:</P><OL><LI>When the genome modification rate is 0 $min^{-1}$ (Figure 1A, Figure 2A &amp; Figure 3A), the total virions reach a constant value of ~80-85.  Increasing the RNAi effectiveness from $10^{-5}$ to $10^{-4}$ decreases the virion capacity by ~5 virions. </LI><LI>When the genome modification rate is 0.001 $min^{-1}$ (Figure 1B, Figure 2B &amp; Figure 3B), the total virions reach the same constant value of ~80-85.  However, the total virions encompass both pure virions and impure virions.  The pure virions will tend to 0, and the impure virions will reach the ~80-85 constant and then tend to 0. </LI><LI>When the genome modification rate is 0.01 $min^{-1}$ (Figure 1C, Figure 2C &amp; Figure 3C), the total virions reach a peak of 20-45 then tend to 0.  This is because all genomes produced are converted to impure genomes. Increasing the RNAi effectiveness from $10^{-5}$ to $10^{-4}$ decreases the virion peak by ~20 virions.</LI></OL><P>In conclusion, an RNA silencing constant of $10^{-5}$ to $10^{-3}$ $molecules^{-1}$ does not have a meaningful impact on virion concentration.  In contrast, increasing the CRISPR modification rate to 0.01 $min^{-1}$ causes the total virion amount to tend to 0.</P><H3>Investigating Final Virion Concentration</H3><P>Two inferences can be made from the virion concentration heat maps.  The X-axis (RNAi) represents the RNAi effectiveness and the Y-axis (pure2mod) represents the P6 targeted genome modification rate.  Furthermore, the virion concentrations were collected over a period of 16 hours. </P><FIGURE><FIGCAPTION>Heat map of genome modification and RNAi effectiveness</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Heat map of genome modification and maximum RNAi effectiveness</FIGCAPTION></FIGURE><P>The CRISPR genome modification rate has a greater effect on the final virion concentration than the RNA silencing.  This is because the virion concentration gradient changes along genome modification rate axis, not the RNAi axis. </P><UL><LI>For all k values, low virion numbers (0-10) occur when the genome modification rate is high (~0.07 $min^{-1}$).  High virion numbers (60-70) occur when the genome modification rate is low (0.04 $min^{-1}$).</LI></UL><P>The CRISPR genome modification rate has a greater effect on the final virion concentration than the maximum RNAi silencing.  Again, this is because the virion concentration gradient changes along genome modification rate axis, not the RNA silencing axis.</P><UL><LI>When the maximum RNAi effectiveness is low (&lt; 6 x $10^{-4}$ $min^{-1}$), low virion numbers (0-10) occur when the genome modification rate is high (0.095 $min^{-1}$).  High virion numbers (60-70) occur when the genome modification rate is low (0.063 $min^{-1}$).  </LI><LI>When the maximum RNAi effectiveness is high (&gt; 6 x $10^{-4}$ $min^{-1}$), both the RNAi and the genome modification affect virion concentration.  For example, at a maximum RNAi effectiveness of 10 x $10^{-4}$ $min^{-1}$, a modification rate of only 0.05 $min^{-1}$ is necessary to reduce virion numbers to 0.</LI></UL><P>This serves as another confirmation that using a CRISPR system to reduce viral efficacy is a worthwhile endeavor, as CRISPR genome modification has a more significant impact on virion levels than the primary plant defense, gene silencing.  An ideal genome modification rate is ~0.095 $min^{-1}$, as this value is the threshold genome modification rate that decreases the virion numbers to 0 on both heat maps.</P><H3>Sensitivity Analysis: Effect of parameter values on steady-state</H3><P>Exploration of parameter values is crucial to determining how robust our system is, especially for parameters which we were unable to find values for. Below is a table showing the relative sensitivity of our system to each parameter at the value listed in the parameter table above. Note we omitted $f_u$, $k_p$, $k_a$, $\delta_3$, $\delta_4$, and $\delta_5$ since for each of those parameters the relative sensitivity was less than 0.01. $k_p$ and $k_a$ are two parameters from our parameter table that we were unable to find values for. The fact that the system is insensitive to changes in their values is good, it indicates our system is robust around these parameters. Biologically this is expected since $k_p$ and $k_a$ are mostly limited by the availability of substrates available for packaging and anchoring respectively.</P><P>We note that virion concentration at 16 hours is increased the most by increasing the transcription rate of 35S RNA. This is intuitive to us since this leads to more 35S RNA, which means more pregenomes to be packaged and more P3, P4, and P5 available to produce virions. We also see a significant increase from increasing the maximum number of genomes in the nucleus, transcription rates for P3, P4, P5, and P6, and the transcription rate of 19S RNA.</P><P>Virion concentration was also very sensitive to $k_g$ and $k_c$, our rates of P6 deactivation by CRISPR/Cas9. An increase in $k_g$ and $k_c$ effectiveness leads to a relatively larger decrease in virion concentrations. This is exciting since our application is focused around the implementation of the CRISPR/Cas9 system. Other parameters that will cause a large relative decrease in virion concentration are the maximum RNAi effectiveness ($L$) and the half-saturation constant for P6 activation.</P><H4>Relative Sensitivity at 16 hours</H4><TABLE class="table table-condensed"><THEAD><TR><TH>Parameter</TH><TH>Relative Sensitivity</TH></TR></THEAD><TBODY><TR><TD>$\alpha_{35}$</TD><TD>$1.48$</TD></TR><TR><TD>$d_{max}$</TD><TD>$1.26$</TD></TR><TR><TD>$\beta_3, \beta_4, \beta_5, \beta_6$</TD><TD>$1.06$</TD></TR><TR><TD>$\beta_6$</TD><TD>$1.03$</TD></TR><TR><TD>$\alpha_{19}$</TD><TD>$1.03$</TD></TR><TR><TD>$\alpha_c$</TD><TD>$0.20$</TD></TR><TR><TD>$k_v$</TD><TD>$0.01$</TD></TR><TR><TD>$...$</TD><TD>$...$</TD></TR><TR><TD>$\delta_v$</TD><TD>$-0.01$</TD></TR><TR><TD>$k$</TD><TD>$-0.05$</TD></TR><TR><TD>$\gamma_d$</TD><TD>$-0.09$</TD></TR><TR><TD>$\gamma_{19}$</TD><TD>$-0.14$</TD></TR><TR><TD>$\gamma_{35}$</TD><TD>$-0.18$</TD></TR><TR><TD>$\delta_6$</TD><TD>$-0.34$</TD></TR><TR><TD>$v_e$</TD><TD>$-0.99$</TD></TR><TR><TD>$K_6$</TD><TD>$-1.03$</TD></TR><TR><TD>$L$</TD><TD>$-1.55$</TD></TR><TR><TD>$k_g, k_c$</TD><TD>$-1.74$</TD></TR></TBODY></TABLE><H3>Phase Plane Analysis: Effect of initial conditions on steady state</H3><P>We want to investigate how the total and relative concentrations of wild-type or $\Delta$-P6 virions will vary over time using different founder population compositions and varying the P6-deactivation rate. To investigate this we produced phase plane diagrams comparing the amount of wild-type virions in the cell against the amount of modified ones.</P><P>Below are three sections, each with a different $k_g$ and $k_c$, which show how the composition of virions in the cell will change over time with or without our CRISPR/Cas9 system. In each case, the lines are colour-coded by the initial concentration of virions. If the founder population is mostly wild-type the curve is dark blue, if founders are mostly mutants the curve will be green, and mixed populations will have a colour in between. Founder populations are varied from 0 to 10 genomes for each of the trajectories.</P><P>This analysis fits nicely with our viral spread model, as it will tell us how initial composition of founders will affect viral load. It also fits nicely with the CRISPR/Cas9 analysis as it shows how behaviour changes given different values for $k_g$ or $k_c$.</P><P><B>Note:</B> In each plot below “pure virions” refers to the number of virions in the cell which contain the wild-type genome while “impure virions” refers to the number of virions containing the genome without functional P6.</P><H4>Phase plane with no P6 deactivation $k_g = k_c = 0$</H4><P>As shown in the four graphs below, phase planes with no P6-deactivation are not particularly interesting. Since there's no conversion from wild-type to modified genomes the lines are perfectly straight. Their slope is determined by the initial composition of wild-type and modified genomes while the final position after a given time are determined by the initial wild-type genomes (since only wild-type genomes produce P6).</P><P>After 4 hours there are no virions present in the cell which corresponds to the lag between initial infection and virion production. After 6 hours the cell begins to produce virions at fairly low concentrations. At 8 hours the cell produces slightly more. Finally, by 16 hours the number of virions has reached steady state, which is indicated by the regular line length. The lines do decrease in length as the slope increases since lines with a larger slope have more modified genomes which leads to less P6 and thus fewer total virions at steady state.</P><FIGURE><FIGCAPTION>Phase diagram with $k_g$ = 0</FIGCAPTION></FIGURE><H4>Phase plane with no P6 deactivation $k_g = k_c = 0.001$</H4><P>Much more interesting is when P6-deactivation is non-zero. At 6 hours there is a low amount of virions produced in the cell. At 16 hours we see most virion trajectories have reached steady state (seen in the graph by the top right limit), but these curves have another interesting property. Due to the conversion of wild-type virions the lines are curving upwards as there are more mutated genomes to produce them.</P><P>However, over time (seen clearly at about 48 hours) the total amount of virions curves back on itself. This is due to having all genomes having P6 deactivated and thus no transactivation from P6 to produce P3, P4, and P5. Once P6 has started degrading the concentrations of the other proteins decrease and so the total viral load lowers. After 80 hours we see that the total amount of virions will drop down to zero.</P><FIGURE><FIGCAPTION>Phase diagram with $k_g$ = 0.001</FIGCAPTION></FIGURE><H4>Phase plane with high P6 deactivation $k_g = k_c = 0.01$</H4><P>Here we see what happens when the P6-deactivation rate is high. Again, at 4 hours there are no virions present. At 8 hours there are virions present and again all the lines curve upwards due to the conversion of wild-type genomes to modified versions. Note also the “feathery” pattens in the trajectories at 8 hours which are caused by the fact that we varied both wild-type and modified genomes from 0 to 10.</P><P>At 16 hours we see that with this high P6 deactivation rate the virions start to curve inwards much earlier than they did in the previous section. Finally, at 24 hours the viral load is in sharp decline due to the total lack of P6 in the cell.</P><FIGURE><FIGCAPTION>Phase diagram with $k_g$ = 0.01</FIGCAPTION></FIGURE></SECTION><SECTION id="references" title="References"><H2>References</H2></SECTION></DIV><A href="#0" class="cd-top">Top</A><FOOTER><P class="footer-links" style="font-size:0.8em;"><A href="https://2015.igem.org/Team:Waterloo/Sitemap" style="font-size:1em;">Sitemap</A><A style="padding-right:0;font-size:1em;" href="http://igem.uwaterloo.ca">External Site</A>© 2015 Waterloo iGEM
    </P></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>