Team:Waterloo/Results
Toggle navigation
Project
Description
Results
Design
Requirements
Lab & Design
Overview
sgRNA Modification
dCas9 Modification
Plant Defense
Parts
Experiments & Protocols
Notebook
Modeling
Overview
CaMV Biology
PAM Flexibility
Cas9 Dynamics
CaMV Replication
Viral Spread
Software
Practices
Overview
Commercialization
Human Practices
Lab Safety
Opinions
Survey
Outreach
Team
Our Team
Attributions
Sponsors
Collaborations
Example Page
Results
The Waterloo iGEM team was able to achieve results for all three project focus areas. This page offers a general summary, while more details can be found on our wet lab overview , mathematical modelling and policy & human practices overview pages.
For the Simple sgRNA Exchange, we were able to show that adding restriction sites and a single mutation in the scaffold region of the sgRNA did not reduce targeting of the sgRNA in CRISPR dCas9 system. The graphs of flow cytometry data can be seen below and more details on the results are available on the sgRNA Exchange page .
RFP Intensity after targeting with dCas9 + standard sgRNA and dCas9 + modified sgRNA in blue. Non-targeted RFP control measurements are shown in grey. Replicate measurements taken on three separate days are overlayed on the same axes.
A software pipeline was developed to determine the PAM-binding selectivity of variant Cas9 proteins. The PyRosetta molecular dynamics software suite was used to perform an energy minimization at the interface of the protein structure and the DNA PAM site structure. The energy minimization provided a score that was then compared to the results of empirical characterization of mutant PAM selectivity
. The pipeline is described in more detail on the Pam Flexibility page and is visualized below.
PAM Flexibility software pipeline: the 3D Cas9 structure is mutated, then 256 variants with different PAM DNA structures are created, then PyRosetta is used to score PAM affinities.
In addition, we mutated dCas9 at three sites 1135, 1335, and 1337 to alter its PAM site from native NGG to NGAG using the Quick Change protocol. The mutations were confirmed using sequencing. Preliminary experiments were done using sgRNA targeting the J23101 promoter upstream of the GFP using NGAG PAM site, however the results, graphed below, were inconclusive. It was expected that the original dCas9 would target NGG but not NGAG, while the modified dCas9 would target NGAG but not NGG. Instead roughly the same signal was measured for both dCas9 variants at both target sites.
Mean RFP Intensity after targeting an NGAG and NGG PAM site (using different complementary sgRNA sequences) with both dCas9 and the modified dCas9 variant.
The wet lab was able to agroinfiltrate a pCAMBIA + sgRNA vector into Arabidopsis, but the generation time of the plant prevented proper testing and verification of the results. However, a protoplast model was used that demonstrated successful expression of Cas9 in plant cells, as shown on the dot blot below.
Two dot blots using anti-Cas9 antibody demonstrate protoplast expression. Top row is chemiluminescence, bottom row is visible light.
Mathematical models are not limited by the same time constraints as growing plants, so they were used to validate the CRISPR Plant Defense aspect of our project. The team created a multi-scale model that showed the effect of CRISPR/Cas9 on viral genomes, viral replication and viral spread across multiple plant cells.
A stochastic model of Cas9 cleavage was able to estimate the effects of CRISPR-Cas9 on the viral genome. Averaging over many simulations, it was apparent that the loss-of-function in viral genomes could be expected to follow an exponential trend. The viral genomes were then simulated with an exponential decay from functional to non-functional forms.
CRISPR effect parameter (t1/2) is derived from an exponential fit to the fraction of CaMV genomes active at each timestep across 1000 stochastic genome simulations.
The CRISPR effect parameter (t1/2) is passed on to a model of the intracellular viral replication cycle. This model determines the time-series and steady state virion concentration. This virion concentration was impacted by plant defenses such as RNA silencing and our CRISPR system. The number of virions present in one cell over time, with a changing CRISPR parameter from the biological parameter to one that might be achieved with an optimized CRISPieR system, is illustrated in the figures below.
Replication model shows total CaMV virions within a cell over time with and without a CRISPR/CRISPieR defense system.  Non-functional virion production is lowered because fewer functional proteins that protect against RNAi are produced.
Both the time series data and heat maps highlight how the CRISPR genome modification has a greater effect on the virion concentration than the plant defenses alone. These results were then carried forward into an agent-based model on the level of plant leaves, discussed in detail on the Intracellular Spread page.
Top
Sitemap External Site Â© 2015 Waterloo iGEM
