<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Waterloo_Modeling_CaMV_Biology skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Waterloo/Modeling/CaMV Biology</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project</A></LI><LI><A href="/Team:Waterloo/Description">Description</A></LI><LI><A href="/Team:Waterloo/Results">Results</A></LI><LI><A href="/Team:Waterloo/Design">Design</A></LI><LI><A href="/Team:Waterloo/Requirements">Requirements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Lab &amp; Design</A></LI><LI><A href="/Team:Waterloo/Lab_Overview">Overview</A></LI><LI><A href="/Team:Waterloo/Lab/sgRNA">sgRNA Modification</A></LI><LI><A href="/Team:Waterloo/Lab/dCas9">dCas9 Modification</A></LI><LI><A href="/Team:Waterloo/Lab/Plants">Plant Defense</A></LI><LI><A href="/Team:Waterloo/Parts">Parts</A></LI><LI><A href="/Team:Waterloo/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="/Team:Waterloo/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Modeling</A></LI><LI><A href="/Team:Waterloo/Modeling">Overview</A></LI><LI><A href="/Team:Waterloo/Modeling/CaMV_Biology">CaMV Biology</A></LI><LI><A href="/Team:Waterloo/Modeling/PAM_Flexibility">PAM Flexibility</A></LI><LI><A href="/Team:Waterloo/Modeling/Cas9_Dynamics">Cas9 Dynamics</A></LI><LI><A href="/Team:Waterloo/Modeling/CaMV_Replication">CaMV Replication</A></LI><LI><A href="/Team:Waterloo/Modeling/Intercellular_Spread">Viral Spread</A></LI><LI><A href="/Team:Waterloo/Software">Software</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Practices</A></LI><LI><A href="/Team:Waterloo/Practices">Overview</A></LI><LI><A href="/Team:Waterloo/Practices/Commercialization">Commercialization</A></LI><LI><A href="/Team:Waterloo/Practices/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Waterloo/Safety">Lab Safety</A></LI><LI><A href="/Team:Waterloo/Practices/Ethics">Opinions</A></LI><LI><A href="/Team:Waterloo/Practices/Survey">Survey</A></LI><LI><A href="/Team:Waterloo/Outreach">Outreach</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team</A></LI><LI><A href="/Team:Waterloo/Team">Our Team</A></LI><LI><A href="/Team:Waterloo/Attributions">Attributions</A></LI><LI><A href="/Team:Waterloo/Sponsors">Sponsors</A></LI><LI><A href="/Team:Waterloo/Collaborations">Collaborations</A></LI><LI><A href="/Team:Waterloo/Example">Example Page</A></LI></DIV></DIV><DIV class="container main-container"><H1>Cauliflower Mosaic Virus (CaMV) Biology</H1><P>Cauliflower Mosaic Virus (CaMV) is a member of the caulimovirus genus of pararetroviruses. CaMV is an ideal virus for the study of CRISPR plant defense for several reasons. <OL><LI>Most importantly, CaMV only infects plants, making it significantly safer to work with.</LI><LI>Second, it can infect <EM>Arabidopsis thaliana</EM>, the plant model organism for genetics, meaning many of the tools for genetic engineering already work.</LI><LI>Finally, the CaMV genome is double stranded for much of the virus lifecycle, which means it can be easily targeted by CRISPR-Cas9 for cutting.</LI></OL></P><SECTION id="replication" title="Replication"><H2>CaMV Replication</H2><P>The lifecycle of a pararetrovirus such as CaMV is rather similar to that of a retrovirus such as HIV. Both types of viruses use a reverse transcriptase to turn viral mRNA into DNA. However, unlike retroviruses, pararetroviruses do not integrate into the host genome. Instead, thousands of minichromosomes of viral genome are imported into the nucleus, where they are transcribed by host machinery.</P><P>Here we present a brief overview of the CaMV replication cycle, compiled from Khelifa et al. 2010 , Yasaka et al. 2014 , Hohn, Fütterera &amp; Hull (1997) , and Citovsky, Knorr, &amp; Zambryski (1991) :</P><DIV class="row"><DIV class="col-sm-8"><OL><LI>The virion enters the host cell, moves to the nuclear pore, and the viral genome is imported in the nucleus</LI><LI>Discontinuities in the genomes are repaired to form minichromosomes</LI><LI>Minichromosomes are used as templates to produce 35S RNA, 19S RNA, and 8S RNA</LI><LI>P6 is translated from 19S RNA and accumulates in electron-dense inclusion bodies (edIBs)</LI><LI>P6 transactivates production of P1-P5 from 35S RNA</LI><LI>Packaging occurs. The exact process is complicated, but either :</LI><UL><LI>35S RNA is packaged into virions along with P5 then is reverse transcribed within the capsid, or</LI><LI>35S RNA is reverse transcribed in the edIBs before packaging</LI></UL><LI>The virion may either reinfect the nucleus to increase the copy number of the minichromosome (early in the infection) or it may leave the cell.</LI></OL><P>The team created an ordinary differential replication model of CaMV replication which is described on <A href="https://2015.igem.org/Team:Waterloo/Modeling/CaMV_Replication">its own page</A>.</P></DIV><DIV class="col-sm-4"><FIGURE><DIV class="img-att">
                i
                <UL class="img-att-bubble"><LI>Photo ©<A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1364-3703.2002.00136.x/full">Haas et al. (2005)</A></LI><LI>Figure 3 of <EM>Cauliflower mosaic virus</EM>: still in the news </LI></UL></DIV></FIGURE></DIV></DIV></SECTION><SECTION id="genome" title="Genome and Proteins"><H2>CaMV Genome and Proteins</H2><P>
    The CaMV genome consists of circular, double-stranded DNA approximately 8000 bp in length . The genome contains three discontinuities typical of pararetroviruses . Discontinuities are repaired within the nucleus to form covalently closed circular DNA (cccDNA) . There are seven open reading frames (ORFs) which code for six proteins of known function; P1-P6 (labeled gp2-gp7 in the diagram below) . Two main RNAs are produced from cccDNA: pregenomic 35S RNA and subgenomic 19S RNA, as well as the small 8S RNA .
    </P><FIGURE><FIGCAPTION>A Visualization of the CaMV Genome produced by importing the NCBI record into Benchling</FIGCAPTION><DIV class="img-att">i
            <UL class="img-att-bubble"><LI><A href="#">Link to original Photo</A></LI></UL></DIV></FIGURE><P>Details on the RNA transcripts and proteins produced by CaMV are given in the sections below.</P><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="rna"><H3 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#bio background" href="#collapseRNA" aria-expanded="false" aria-controls="collapseRNA">
        RNA Transcripts
        <SPAN class="hidden-xs toggle-arrow pull-right">Panel Toggle</SPAN></A></H3></DIV><DIV id="collapseRNA" class="panel-collapse collapse" role="tabpanel" aria-labelledby="rna"><DIV class="panel-body"><H4>Pregenomic 35S RNA</H4><P>
            35S RNA is a complete transcript of the CaMV genome with additional 180 nt terminal repeats . It has two primary functions. The first is to act as an intermediate in genome replication. That is, it's reverse transcribed either in a viral factory or within the capsid to recover gapped dsDNA . It's second function is as polycistronic mRNA coding for P1-P5. Translation of P1-P5 from the 35S promoter must be transactivated by P6 . 35S RNA also undergoes alternative splicing to regulate P1 and P2, which are potentially toxic in high enough concentrations .
            </P><H4>Subgenomic 19S RNA</H4><P>
            The 19S RNA is a 2.3 kb  transcript which contains the ORF for P6 . P6 is constitutively produced from 19S RNA and it's many functions are described in it's own section.
            </P><H4>8S RNA</H4><P>
            8S RNA is 610-680 nt long  and is transcribed from the leader sequence of 35S RNA. It's believed that it serves as a decoy for RNA interference, which normally targets the 35S leader .
            </P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="pro"><H3 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#bio background" href="#collapseProtein" aria-expanded="false" aria-controls="collapseProtein">
        CaMV Proteins
        <SPAN class="hidden-xs toggle-arrow pull-right">Panel Toggle</SPAN></A></H3></DIV><DIV id="collapseProtein" class="panel-collapse collapse" role="tabpanel" aria-labelledby="rna"><DIV class="panel-body"><P>CaMV produces six proteins, five of which (P1-P5) are produced from the 35S RNA. The sixth, P6, is produced from the 19S RNA.</P><H4>Movement Protein (P1)</H4><P>
            The movement protein (MP) of CaMV facilitates intercellular spread of virions . P1 binds to pectin methyltranferase, a cell wall associated protein , and ultimately forms intercellular tubules though which virions are transported to neighbouring cells . This protein may also bind cooperatively to RNA to direct 35S and/or 19S RNA to neighbouring cells , which suggests the protein-RNA complex may also move between cells without a capsid.
            </P><H4>Aphid Transmission Factor (P2)</H4><P>
            The aphid transmission factor (ATF) is a protein which allows for host-to-host transmission of the virus using aphids as vectors. P2 binds to the aphid stylets and P3, which allows virions to associate with the stylet upon feeding . Its primary function is host-to-host transmission, but it does produce one electron-lucent inclusion body (elIB) which has minor effects on the system. To form the elIB, P2 associates with microtubules and are transported along the cytoskeleton to a central location. In this process some P3 associates with the P2 and is transported to the elIB . Finally, a small percent (~5%) of virions find their way to the elIB .
            </P><H4>Virion-Associated Protein (P3)</H4><P>
            The virion associated protein (VAP) is a 15 kDa protein is weakly associated with the viral capsid . It's implicated in both host-to-host  and cell-to-cell  transmission.
            </P><H4>Capsid Protein (P4)</H4><P>
            The capsid protein (CP) is a precursor to numerous other polypeptides which we refer to as P4 subunits. The P4 subunits include P37, P44, P40, P80, P90, and P96, with P80 and P96 being homodimers composed of P37 and P44 respectively . A viral capsid is formed from 420 subunits composed into 3 layers.
            </P><H4>Reverse Transcriptase (P5)</H4><P>
            The P6 protein reverse transcribes 35S RNA to form partially double stranded DNA (pdsDNA) . The pdsDNA is then encapsidated by P4 subunits to produce virions. It's unknown if reverse transcription occurs within the capsid or in the viroplasm . There is evidence that in Hepatitis B (a similar pararetrovirus) that RT first binds to the pregenome, then the RT-RNA complex is recognized by CP to form a capsid, after which reverse transcription finally occurs .
            </P><H4>Transactivator/Viroplasmin Protein (P6)</H4><P>
            The transactivator/viroplasmin protein (TAV), as it's name suggests, is a protein primarily involved in transactivating the five other proteins  and condenses into viroplasms . The viroplasms, aka viral factories or electron-dense inclusion bodies (edIBs), sequester 35S RNA transcripts and produces the other proteins required for viral spread and assembly . The purpose is two-fold: concentrating the nucleic acid and protein in a smaller volume to increase reaction rates, and protecting the replication machinery from cell defenses .
            </P></DIV></DIV></DIV><SECTION id="spread" title="Transport and Spread"><H2>Viral Transport and Spread</H2><P>CaMV exploits host transport mechanisms in order to spread. Neighbouring plant cells share cytoplasmic connections through connections called plasmodesmata. These symplastic connections are exploited by CaMV to spread to neighbouring cells, but it is a fairly slow process. The virions must travel between the plasma membrane and the desmotubule (cytoplasmic channel) of the plasmodesmata, a very narrow passage. The size of molecule able to diffuse through the plasmodesmata is determined by the size exclusion limit (SEL), which is affected by the gating properties of the plasmodesmata. In another slow cell-to-cell process, the virus can move through tubules formed through the cell wall. However, CaMV is able to spread to different regions of the plant quite quickly using the plant’s phloem (sugar-transport network). </P><P>In addition to exploiting the host transport machanisms, CaMV can induce the formation of microtubules in order to increase its rate of spread . Microtubules are a key component of intercellular transmission as well as intracellular motion. They form viral inclusions that encourage uptake by aphids (the major plant-to-plant transport agent) and of viral factories .</P><P>Modelling the spread of infection through a series of plant cells was the focus of the<A href="https://2015.igem.org/Team:Waterloo/Modeling/Intercellular_Spread"> Intercellular Viral Spread Model</A>. </P></SECTION><SECTION id="defense" title="Plant Defenses"><H2>Natural Plant Defenses</H2><P>The CRISPR Plant Defense application aims to introduce a CRISPR/Cas9 as an antiviral defense in plants. Plants do, of course, have their own defenses against viral infection and we wanted to ensure that CRISPR/Cas9 provided <EM>additional</EM> defenses for the plants. To that end, two forms of natural plant defense were modelled: intercellular signalling and RNA interference.

 </P><H3>Intercellular Defense Signalling</H3><P>Chemical signaling with chemicals such as salicylic acid (SA), NO and  H<SUB>2</SUB>O<SUB>2</SUB> within plants is extremely important due to the huge role it plays in triggering and maintaining defense mechanisms to allow the plant cells to fight off pathogens.  </P><P>The main plant defenses triggered by signalling are systemic acquired resistance (SAR) and the hypersensitive response (HR). SAR is a broad, long-term increased resistant to future infections, similar to the increased resistance against diseases in mammals post-infection.  This mechanism is only activated after a pathogen is detected within the plant, triggering defenses through signals like SA and NO.  During the initial wave of defense after pathogen detection, there is an “oxidative burst” wherein levels of oxidative species suddenly increase, accompanied by cell wall protein linkage, the activation of kinase and increased defensive gene expression. </P><P>NO and SA play an interconnected role with each other, where reactive oxygen species (ROS) such as NO have been observed accumulating SA, and in turn, SA triggers ROS production (including NO and H<SUB>2</SUB>O<SUB>2</SUB>, the main signal for HR). </P><P> In the hypersensitive response (HR), cells infected by a pathogen undergo apoptosis, forming lesions on the plant. H<SUB>2</SUB>O<SUB>2</SUB> allows for the localization of the HR-induced apoptosis by triggering the chemical processes required for cell degradation in addition to increasing the expression of defense genes, which it modulates during defense response . </P><P>
HR and SAR are modeled extensively in the <A href="https://2015.igem.org/Team:Waterloo/Modeling/Intercellular_Spread">viral spread model</A> in order to produce a reasonable simulation of the infection of Arabidopsis Thaliana by CaMV. The different signalling chemicals have been simplified to one process each for HR and SAR, instead of having multiple reactions and interactions between chemicals, and are used to visualize the spread of the defenses throughout the plant.
</P><H3>RNA Interference</H3><P>RNA interference is a method of gene silencing utilized in gene regulation and viral defense.  Cytoplasmic double-stranded RNA (dsRNA) is cleaved by a Dicer-like enzyme (DCL) into short lengths, facilitated by a dsRNA-binding protein (DRB) . These short segments are called small interfering RNA (siRNA) or micro RNA (miRNA) and are approximately 20-25 base pairs with a 2-nucleotide overhang at the 3' end  . Specifically in <EM>Arabidopsis thaliana</EM>, DLC3 and DLC4 cleave CaMV virus-derived short interfering RNA (viRNA) into 24-nt and 21-nt long lengths respectively .  The viRNA are split into single strands and integrated into a RNA-induced silencing complex, which targets and cleaves messenger RNA (mRNA). This prevents the use of mRNA in translation templates, effectively inhibiting gene expression .  To avoid this plant defense, P6 located in the nucleus interacts with DBR4 to eliminate Dicer activity . </P><P>RNA interference is incorporated into the <A href="https://2015.igem.org/Team:Waterloo/Modeling/CaMV_Replication">CaMV Replication</A> and <A href="https://2015.igem.org/Team:Waterloo/Modeling/Intercellular_Spread"> Intercellular Viral Spread Model</A>.</P></SECTION><H2>References</H2></SECTION></DIV></DIV><A href="#0" class="cd-top">Top</A><FOOTER><P class="footer-links" style="font-size:0.8em;"><A href="https://2015.igem.org/Team:Waterloo/Sitemap" style="font-size:1em;">Sitemap</A><A style="padding-right:0;font-size:1em;" href="http://igem.uwaterloo.ca">External Site</A>© 2015 Waterloo iGEM
    </P></FOOTER></DIV></DIV></DIV></BODY></HTML>