<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Waterloo_Modeling_Cas9_Dynamics skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Waterloo/Modeling/Cas9 Dynamics</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project</A></LI><LI><A href="/Team:Waterloo/Description">Description</A></LI><LI><A href="/Team:Waterloo/Results">Results</A></LI><LI><A href="/Team:Waterloo/Design">Design</A></LI><LI><A href="/Team:Waterloo/Requirements">Requirements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Lab &amp; Design</A></LI><LI><A href="/Team:Waterloo/Lab_Overview">Overview</A></LI><LI><A href="/Team:Waterloo/Lab/sgRNA">sgRNA Modification</A></LI><LI><A href="/Team:Waterloo/Lab/dCas9">dCas9 Modification</A></LI><LI><A href="/Team:Waterloo/Lab/Plants">Plant Defense</A></LI><LI><A href="/Team:Waterloo/Parts">Parts</A></LI><LI><A href="/Team:Waterloo/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="/Team:Waterloo/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Modeling</A></LI><LI><A href="/Team:Waterloo/Modeling">Overview</A></LI><LI><A href="/Team:Waterloo/Modeling/CaMV_Biology">CaMV Biology</A></LI><LI><A href="/Team:Waterloo/Modeling/PAM_Flexibility">PAM Flexibility</A></LI><LI><A href="/Team:Waterloo/Modeling/Cas9_Dynamics">Cas9 Dynamics</A></LI><LI><A href="/Team:Waterloo/Modeling/CaMV_Replication">CaMV Replication</A></LI><LI><A href="/Team:Waterloo/Modeling/Intercellular_Spread">Viral Spread</A></LI><LI><A href="/Team:Waterloo/Software">Software</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Practices</A></LI><LI><A href="/Team:Waterloo/Practices">Overview</A></LI><LI><A href="/Team:Waterloo/Practices/Commercialization">Commercialization</A></LI><LI><A href="/Team:Waterloo/Practices/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Waterloo/Safety">Lab Safety</A></LI><LI><A href="/Team:Waterloo/Practices/Ethics">Opinions</A></LI><LI><A href="/Team:Waterloo/Practices/Survey">Survey</A></LI><LI><A href="/Team:Waterloo/Outreach">Outreach</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team</A></LI><LI><A href="/Team:Waterloo/Team">Our Team</A></LI><LI><A href="/Team:Waterloo/Attributions">Attributions</A></LI><LI><A href="/Team:Waterloo/Sponsors">Sponsors</A></LI><LI><A href="/Team:Waterloo/Collaborations">Collaborations</A></LI><LI><A href="/Team:Waterloo/Example">Example Page</A></LI></DIV></DIV><DIV class="container main-container"><H1>Modeling Genomic Effects of CRISPR/Cas9</H1><SECTION id="motivation" title="Motivation"><DIV class="row"><DIV class="col-sm-8"><P>CRISPR/Cas9 has been extensively studied for its applications in eukaryotic genome editing and gene expression control. Last year, the <A href="https://2014.igem.org/Team:Waterloo/Math_Book/CRISPRi">Waterloo iGEM team</A> created an ODE model of dCas9 binding and control of gene expression. This year, however, the modelling team chose to investigate the effects of CRISPR/Cas9 on an genomic rather than molecular level. Specifically, we wanted to model the <STRONG>accumulation of mutations</STRONG> in a target genome and <STRONG>eventual deactivation</STRONG> of target genes after cutting by CRISPR/Cas9 and repair by Non-Homologous End Joining (NHEJ).</P></DIV><DIV class="col-sm-4"><FIGURE><FIGCAPTION class="model-caption">Genomes</FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION id="formation" title="Model Formation"><H2>Model Formation</H2><P>When bound to a single guide RNA (sgRNA), the <EM>S. pyogenes</EM> Cas9 nuclease diffuses through the cell in three dimensions, searching for the sequence 'NGG' in the target genome . When it finds an 'NGG', known as a <EM>PAM site</EM>, Cas9 binds and undergoes a conformational change that allows it to unwind the DNA helix and compare the sequence of its sgRNA with the DNA. If the sgRNA matches well, Cas9 cleaves the DNA, producing a double-stranded break (DSB) 3-4 bp upstream of the PAM site .</P><P>In the absence of a template, DSBs are repaired by Non-Homologous End Joining (NHEJ), which is an error-prone process that sometimes creates indels at the site of repair . This effect has recently been exploited to target double-stranded viruses such as HBV . Though there have been extensive efforts to characterize the factors that contribute to effective targeting and deactivation by CRISPR/Cas9 and NHEJ, they have not, to the best of our knowledge, been synthesized into a single model.</P><P>The aim of the model is thus to capture the cutting events initiated by Cas9 and predict the outcomes of these events. We model each genome as containing multiple <EM>domains</EM> of interest, such as promoters or ORFs, and track whether these domains have been deactivated by CRISPR/Cas9 activity. There may be more than one sgRNA <EM>target</EM> in each domain and many domains can be targeted at once.</P><FIGURE><FIGCAPTION>Genomes contain domains, such as promoters or ORFs, which we endeavour to deactivate by directing Cas9 to a target or targets within them.</FIGCAPTION></FIGURE><P>If Cas9 successfully cuts at a target site, the double-stranded break may be resolved in three ways. The most common resolution is for NHEJ to successfully repair the DSB without creating any indels </P>However, NHEJ repair is error-prone and will often induce indels at the target site. Finally, since multiple sgRNA targets are considered, it is possible that large deletions will occur between two targets that are simultaneously cut.

        <FIGURE><FIGCAPTION>Possible events after double-stranded breaks caused by CRISPR-Cas9: repair, indels or a large deletion between sites.</FIGCAPTION></FIGURE><P>At each timestep, the model considered the state (cut or uncut) and sequence of all targets and computes the probability of the following events at each target: CRISPR/Cas9 cutting, NHEJ repair or large deletion. The remainder of the model formation section discusses how we determined the probability of each event.</P><H3>Probability of Double-Stranded Cuts made by CRISPR/Cas9</H3><P>The probability of a target being cut in a given time step was modelled as dependent on Cas9 concentration and sgRNA mismatches. The average time for a cut was also considered using the reported from 2-minute half-life Hemphill et al. .</P><P>Since Cas9 binds to PAM sites according to an approximately first order  three dimensional diffusion  we expect increasing concentration of Cas9 protein to lead to a higher probability of cutting. Concentration was incorporated into our model using a multiple regression on data from Kusku et al.(2014) , which related the proportion bound as a function of concentration and number of mismatches between the target and the sgRNA.</P><P>The effect of mismatches in the sgRNA (important for the residual targeting after indels have been introduced by NHEJ) was further considered using the relationship found by Hsu et al. . These effects were assumed to be independent, which likely overestimates mismatch effects and underestimated CRISPR/Cas9 efficacy.</P><H3>Error-Prone Repair by Non-Homologous End Joining</H3><P>Open breaks in the DNA were repaired according to an exponential decay, following the model of Reynolds et al. , which found that DSBs remained with a half-life of 8 minutes. Insertion and deletion sizes were chosen based on a distributions of indels observed by deep-sequencing of repaired targets (see data in ). When there is a net insertion, we selected from a uniform distribution of [ACGT] to add new nucleotides.</P><H3>Large Deletions</H3><P>The probability of large deletions was estimated using a study on large deletions, which measured the percentage of large deletions observed at 3 days and 10 days at several targets . We chose not to account for the effect of the distance between targets for large deletions, though it could be incorporated in future studies .</P><H3>No Interaction Between Genomes</H3><P>We expect there to be multiple viral genomes in our plant defense example and it is possible that simultaneous cuts on different genomes could result two genomes being joined together. However, we chose to disregard this possibility and averaged results from multiple stochastic simulations could be averaged to get an overall picture of gene deactivation by Cas9.</P></SECTION><SECTION id="model" title="Model Implementation"><H2>Software Implementation</H2><H3>Genome Classes</H3><P>The code uses three classes to model the genome. Genomes have domains which have targets. Targets handle probabilities, domains track functionality and genome modifies everything.</P><PRE>class Target():
    is associated with a domain

class Domain():
    has targets
    is associated with a genome

class Genome():
    has domains</PRE><H3>Genome Simulation</H3><P>The simulation calls these classes to check if events have occurred and the details of each event. At the end it compiles the data logs into CSVs, plots and visualizations.</P><PRE>for dt in time_steps:
    call genome_classes to check if there was a cut, repair  or large deletion
    if event:
        add to log
generate CSVs, plots and visualizations</PRE><P>To see all the code for the simulation, check out our <A href="https://github.com/igem-waterloo/uwaterloo-igem-2015/tree/master/models/targeting">GitHub Page</A></P></SECTION><SECTION id="results" title="Results"><H2>Results</H2><H3>Model Validation</H3><P>The model matched experimental results well and helped to shed light on the details of indels due to NHEJ. The clips bellow show three different examples of gene deactivation ranging from few cuts, many deactivations to high cuts, few deactivations.</P><DIV class="jumbotron hp-landing"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><VIDEO style="width:100%" controls=""><SOURCE src="/wiki/images/b/b4/Waterloo_genome_360min_15fps_stoch1.mp4" type="video/mp4">
                            Your browser does not support the video tag.
                        </SOURCE></VIDEO></DIV></DIV><DIV class="row"><DIV class="col-sm-6" style="text-align:center"><FIGURE><FIGCAPTION>Gene States</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6" style="text-align:center"><FIGURE><FIGCAPTION>Target States</FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><VIDEO style="width:100%" controls=""><SOURCE src="https://static.igem.org/mediawiki/2015/f/f0/Waterloo_genome_360min_30fps_stoch2.mp4" type="video/mp4">
                            Your browser does not support the video tag.
                        </SOURCE></VIDEO></DIV></DIV><DIV class="row"><DIV class="col-sm-6" style="text-align:center"><FIGURE><FIGCAPTION>Gene States</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6" style="text-align:center"><FIGURE><FIGCAPTION>Target States</FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><VIDEO style="width:100%" controls=""><SOURCE src="https://static.igem.org/mediawiki/2015/9/92/Waterloo_genome_360min_15fps_stoch3.mp4" type="video/mp4">
                            Your browser does not support the video tag.
                        </SOURCE></VIDEO></DIV></DIV><DIV class="row"><DIV class="col-sm-6" style="text-align:center"><FIGURE><FIGCAPTION>Gene States</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6" style="text-align:center"><FIGURE><FIGCAPTION>Target States</FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV><H3>Importance of Large Deletions</H3><P>Our interest in modeling large deletions was to investigate whether they would be a viable method for deactivating viral targets. The infrequency of large deletions according to experimental results and inputed in our model showed that they are not a reliable method of deactivating genes.</P><H3>Effect of Cas9 Concentration</H3><P>Through experimentation with the model we found that varying the Cas9 concentration had a surprisingly small efect on the strength of the system. This is likely because the standard concentration is already nearly saturated and increasing this value does not significantly increase binding.</P><H3>Predicting CRISPR Plant Defense</H3><P>This model was applied to the CRISPR Plant Defense aspect of our project, investigating whether the P6 protein of Cauliflower Mosaic Virus (CaMV) could be deactivated by frameshift mutations. The P6 protein was chosen as a focus of the investigation because it suppresses natural plant RNAi defenses  and trans-activates translation of other CaMV proteins . Details on P6 and the CaMV genome can be found on <A href="https://2015.igem.org/Team:Waterloo/Modeling/CaMV_Biology">CaMV Biology page</A>.</P><P>The model was run with three targets in the P6 gene of the simulated CaMV genome described. The particular sgRNA target locations in P6 are those described as <STRONG>Design II</STRONG> on the related <A href="https://2015.igem.org/Team:Waterloo/Lab/Plants#target-sites">wet lab page</A>. We tracked the percent of simulated genomes with functional P6 across 1000 runs fo the model, giving a general prediction of how long it will take before the P6 of a particular CaMV genome is rendered non-functional by our Plant Defense system.</P><FIGURE><FIGCAPTION>Percent of functional P6 genomes observed over 1000 simulations with three targets are shown in black, while an exponential decay fit done with the R nls package is shown in green.</FIGCAPTION></FIGURE><P>Based on the fit to our 1000-simulation average, we considered our CRISPR Plant Defense system to render the P6 gene of CaMV non-functional according to an exponential decay with a decay constant of 6.36x10<SUPER>3</SUPER> min<SUPER>-1</SUPER>. </P><H3>Future Research</H3><P>An interesting pattern we noticed when determining how to measure gene activation was that recording the sum of the target shifts as opposed to the individual shifts lead to reactivation after a few hours. This is likely due to some active targets being re-shifted and actually fixing the gene. The first figure shows gene activation measured by the sum of target shifts and the second figure shows gene activation measured by individual shifts (where any non multiple 3 shift causes deactivation.)</P><CENTER><P>1000 runs of target sum deactivation</P><P>1000 runs of single target deactivation</P></CENTER><P>Further developments we would like to pursue are adding proximity of targets to their large deletion probabilities and making the model simpler to apply to non-viral genomes.</P></SECTION><SECTION id="references" title="References"><H2>References</H2></SECTION></DIV><A href="#0" class="cd-top">Top</A><FOOTER><P class="footer-links" style="font-size:0.8em;"><A href="https://2015.igem.org/Team:Waterloo/Sitemap" style="font-size:1em;">Sitemap</A><A style="padding-right:0;font-size:1em;" href="http://igem.uwaterloo.ca">External Site</A>© 2015 Waterloo iGEM
    </P></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>