Team:Freiburg/Project/Cellfree Expression
""
Home
Project
Team
Results
Human Practice
Notebook
Introduction
Overview
Diagnostics today
The DiaCHIP
DNA Engineering
Cell-Free Expression
Protein Purification
Surface Chemistry
Optical Detection
Outlook
Members
Acknowledgements
Partners
The DiaCHIP
Main Results
Diagnostics
Our Device
DNA Array
Cell-Free Expression
Binding on Surface
Modeling
BioBricks
Collaborations
Interlab Study
Achievements
Science Fair
Theater Project
Public Opinion
College Talk
Ethics
Labjournals
Techniques
Gibson
E.coli Strains
Protocols
Materials
Safety
Wiki-Tricks
Glossary
Home
Project
Results
Team
Practice
Notebook
Menu
text
next
Cell-Free Expression
Why Do We Use Cell-Free Expression?
Cell-free expression offers a possibility to overcome several challenges of conventional protein expression and has many advantages, in particular for our project. 1)
Cell-based protein expression is a well-established method to obtain large amounts of a target protein. It enables high yield expression and purification of the protein in quantities sufficient for various in vitro applications 2) . Nevertheless, it is a tedious task to generate all the genetically modified organisms if many different proteins need to be expressed. Additionally, purification cannot be performed with a generalized protocol, and usually requires separate optimization for each protein 1) .
In contrast, cell-free expression saves a lot of time and money. It avoids the generation of genetically modified organisms for every protein. At the purification step, cell-free expression avoids the need for cell lysis and therefore circumvents this harsh procedure, thus preserving the integrity of the protein 3) .  In cell-based expression, too strong induction often results in aggregation of protein, rendering it non-functional. This risk is minimized by using cell-free expression, since the expressed protein is diluted in a far larger volume than the intracellular space and inclusion bodies do not form 4) .
For translating DNA templates into protein microarrays in a microfluidic setup, cell-free expression is the method of choice. This system is capable of expressing many different sequences at once. Additionally, the microfluidic setup provides the opportunity to constantly supplement the expression. Replacing depleted components, like dNTPs, amino acids or energy sources like creatine phosphate, allows higher protein yields 5) .
Basics of Cell-Free Expression
The Components
Figure 1: Basic Components of a Cell-Free Expression Mix. A standard cell-free expression systems consists of a DNA template, the cell extract and additives containing all the components needed for energy regeneration, buffering, and generation of the protein.
Two basic components are needed to conduct in vitro protein expression as seen in figure 1:
the genetic template (mRNA or DNA) encoding the target protein
a reaction solution containing the transcriptional and translational molecular machinery
Cell extracts supply the reaction with most of the beforehand mentioned molecules, including:
RNA polymerases for mRNA transcription
ribosomes for polypeptide translation
tRNA and amino acids
enzymatic cofactors and an energy source
cellular components essential for proper protein folding
Regular cell lysate already contains most of the components needed for cell-free expression. The machineries that usually conduct the translation and transcription of various proteins in the organism now only produce the protein of choice. The basic components, the amino acids and dNTPs, are already present but are also supplemented to increase the efficiency. As the normal energy regeneration system is missing in a cell lysate, an artificial one is added; for this we use creatine phosphokinase.  Other components present in a cell-free expression mix buffer the sensitive system and imitate the cytosolic environment.
Figure 2: Prokaryotic vs eukaryotic cell-free expression. While prokaryotic expression enables higher yields and is more cost efficient, eukaryotic expression offers more advanced features.
Comparing the currently available cell-free expression systems reveals advantages and disadvantages depending on the field of application. An E. coli based system convinces with simplicity, cost-efficiency and its high yield production. However, for the production of complex eukaryotic proteins, it might not be the best option as it does not provide post-translational modifications such as core glycosylation or phosphorylation, and natural membrane components (e.g. microsomes) for membrane protein insertion. Eukaryotic cell-free systems based on wheat germ embryos, rabbit reticulocytes, insect cells or other animal and human cell lines would be the better choice for expression of eukaryotic proteins. 4)
The Reaction
Figure 3: Batch vs Dialysis mode. In batch mode, the cell-free mix is assembled in one complete batch, whereas a membrane in dialysis mode allows feeding of the reaction with additional components over time, thus enabling higher expression.
There are several ways of performing a cell-free reaction. In batch reactions, transcription and translation are carried out in one reaction vessel containing all the necessary components. Due to factors like fast depletion of energy supply, degradation of components (e.g. nucleotides), and decreasing concentrations of free Mg2+-ions, the reaction in the batch system usually reaches a plateau after about 1-2 hours 1) .
In dialysis mode, the cell-free transcription and translation reaction is carried out in a small reaction chamber that is separated by a dialysis membrane from a 10-20x larger reservoir containing low molecular weight reagents. The reservoir supplies the reaction chamber with ions, energy substrates, nucleotides and amino acids. In turn, the low molecular weight by-products are efficiently diluted via the membrane from the reaction chamber into the reservoir. This setup will drive the reaction longer (usually 20-24 hrs) and results in higher levels of produced protein 1) .
Design of the DNA Template
In general, it is possible to use RNA, plasmids, or linear templates (e.g. PCR products). In all cases, the structural design of the DNA or RNA is of high importance for the yield and quality of the protein produced. In bacterial 5'-UTRs optimization can be achieved by placing a Shine-Dalgarno (SD) sequence at an optimal distance, typically eight nucleotides, from the AUG start codon. The SD sequence interacts with the 3'-end of 16S ribosomal RNA (rRNA) that is part of the small ribosomal (30S) subunit, to facilitate initiation. Furthermore, the integration of an A/U-rich enhancer sequence further upstream allows the SD sequence to interact more effectively with the rRNA.  Another attempt in this context is the use of different tags to enhance expression, as reported by Haberstock et al (2012). 6) This design was used for our DNA templates for cell-free expression of antigens.
In eukaryotes, the Internal Ribosome Entry Site (IRES) is a highly structured element found within viral mRNA that is able to induce eukaryotic initiation. This solves the problem of ineffective capping, which presents a major restriction of high production. For example, the IRES of the Cricket Paralysis Virus (CrPV) showed the ability to increase expression substantially across several species 7) .
Our DiaMIX
We developed a low-budget protocol for iGEM teams to express proteins in a prokaryotic cell-free expression system. Due to the prokaryotic origin of most of the proteins expressed during our project, we decided to base our cell-free expression system, the DiaMIX, on an E. coli culture.
Figure 4: E. coli lysate production. The strain BL21 was transformed with the plasmid pRARE2 , induced with IPTG to express T7 polymerase, lysed, centrifuged and the supernatant was flash frozen in liquid N2.
For producing the DiaMIX lysate, we used the E. coli strain BL21 , as it is carrying a gene coding for the T7 RNA polymerase, which improves protein expression. The cells were induced with IPTG prior to further treatment to initialize T7 polymerase expression, therefore allowing a high rate of protein synthesis. 6)
Since codon usage is a limiting factor, especially when expressing proteins of eukaryotic origin, the E. coli BL21 cells were additionally transformed with the plasmid pRARE2 containing the coding sequence of tRNAs for the translation of rare codons (AGA, AGG, AUA, CUA, GGA, CCC and CGG)under control of their native promoters. We prepared the lysate according to an optimized protocol adapted from the European Molecular Biology Lab (EMBL) Heidelberg (figure 4)  This lysate contains the whole transcription and translation apparatus of the cell, allowing successful expression of proteins from DNA templates.
Verification of Protein Expression
To validate whether the expression in the DiaMIX yields correctly folded proteins, we used various methods like fluorescence measurements , luciferase assays and Western Blots .
Our first approach was the expression of GFP , which can be detected via its fluorescence. We used a plasmid template that was kindly provided by AG Roth. For the results of this experiment, have a look at our Results Page . As a second test we performed a luciferase assay. In contrast to the expression of GFP which can be followed by a gradual rise of fluorescence over time, the luciferase assay is based on the emission of light when the substrate luciferin is converted into oxyluciferin (figure 5). This reaction is catalyzed by the reporter luciferase. Since it depends on inducing the reaction with luciferin, it does not allow a tracking of expression over time. Check out our results for more information about the outcome. This analysis helped us to evaluate the concentration of expressed protein in the cell-free expression system.
Figure 5: Luciferase reaction. The luciferase catalyzes the reaction of D-luciferin to D-luciferyl using ATP as an energy source. D-luciferyl is instantly converted into oxyluciferin, whereby energy is emitted as light. The intensity of bioluminescence is proportional to the amount of expressed luciferase.
Cell-free Expression of Antigens
An important goal was to express antigens with the required tags in a concentration and folding state that suffices for an iRIf measurement. Here, we focused on a C. tetani epitope flagged by a double His and a Spy-tag. For cloning of this construct we used a newly designed cloning site and a vector that we had optimized for cell-free expression.
Testing Different Conditions
To identify the optimal DNA concentration that yields in a maximum protein expression, we tested different conditions. For this experiment, we used DNA coding for the enzyme luciferase.
Another component that is crucial for optimal cell-free expression is magnesium. To evaluate the effects of different magnesium concentrations, we started multiple cell-free expression mixes using different magnesium starting concentrations. For results of our optimization, go to our Results Page.
Cell-Free Results
Step by Step Validation
Figure 6: Immobilization of DNA. PCR product coding for GFP with a Cy3 and an amino label was spotted onto the activated PDMS slide. Binding of the amino group to the specific surface allows for direct DNA immobilization.
Since we aimed at using cell-free expression in combination with the DiaCHIP, we decided to validate the process in individual steps.
First, the DNA templates had to be immobilized on the PDMS slide (figure 6). In another step we expressed GFP cell-free in a tube and spotted the produced protein on a iRIf slide to examine whether the interaction with the respective antibody can be detected (figure 7). Furthermore, to evaluate the interference of the specific surface with our expression system, we spotted our expression system onto the slide without starting the reaction beforehand (figure 8). Lastly, we performed a cell-free expression from immobilized DNA in the finished setup of glass and PDMS slide (figure 9).
Figure 7: Spotting of expressed GFP. Cell-free expressed GFP lysate was spotted onto the activated glass slide after expression was performed.
Figure 8: On-slide expression of GFP. Cell-free expression mix was spotted on the activated glass slide and expression was performed.
Figure 9: In-chamber expression of GFP. Cell-free expression mix was pipetted into the flow chamber with immobilized DNA on the silicone slide and an activated glass slide. Then, the expression was performed.
References
1) Bernhard, F., & Tozawa, Y., 2013. Cell-free expression-making a mark. Current Opinion in Structural Biology, 23(3), 374–380.
2) Carlson et al., 2011. Cell-free protein synthesis: applications come of age. Biotechnology advances.
3) Kubick., et al., 2015. Cell-Free Synthesis Meets Antibody Production: A Review. Antibodies.
4) Rosenblum et al., 2015. Engine out of the Chassis: Cell-Free Protein Synthesis and its Uses.
5) Stech et al., 2014.A Continuous-Exchange Cell-Free Protein Synthesis System Based on Extracts from Cultured Insect Cells. PLOS.
6) Haberstock, S. et al., 2012. A systematic approach to increase the efficiency of membrane protein production in cell-free expression systems. Protein Expression and Purification.
7) Brödel, A. K., et al., 2013. IRES-mediated translation of membrane proteins and glycoproteins in eukaryotic cell-free systems.
