<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Project_DNA_Engineering skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Project/DNA Engineering</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>&quot;&quot;</TITLE></P><DIV id="header"><DIV id="naviconwrapper"><DIV id="naviconbar"><DIV id="navicons"><UL><LI id="home"><A href="https://2015.igem.org/Team:Freiburg/Home">Home</A></LI><LI id="project"><A href="https://2015.igem.org/Team:Freiburg/Project/Overview">Project</A></LI><LI id="team"><A href="https://2015.igem.org/Team:Freiburg/Team">Team</A></LI><LI id="results"><A href="https://2015.igem.org/Team:Freiburg/Results">Results</A></LI><LI id="policy" style="font-size:11px"><A href="https://2015.igem.org/Team:Freiburg/Practices">Human Practice</A></LI><LI id="notebook"><A href="https://2015.igem.org/Team:Freiburg/Notebook">Notebook</A></LI></UL></DIV></DIV></DIV><DIV id="navtextwrapper"><DIV id="navtext"><UL id="project-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/Overview">Introduction</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/System">Overview</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Design">Diagnostics today</A></LI><LI><A class="navsub1 subsubmenu" href="#">The DiaCHIP</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/DNA_Engineering">DNA Engineering</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Cellfree_Expression">Cell-Free Expression</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Protein_Purification">Protein Purification</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Surface_Chemistry">Surface Chemistry</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/iRIf">Optical Detection</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/Future_Directions">Outlook</A></LI><UL id="team-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Team">Members</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Attributions">Acknowledgements</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Partners">Partners</A></LI></UL><UL id="results-sub"><LI><A class="navsub1 subsubmenu" href="#">The DiaCHIP</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results">Main Results</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Diagnostics">Diagnostics</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Own_Device">Our Device</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Immobilization">DNA Array</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Cellfree">Cell-Free Expression</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Surface">Binding on Surface</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Modeling">Modeling</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Parts">BioBricks</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Collaborations">Collaborations</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/InterLab_Study">Interlab Study</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Achievements">Achievements</A></LI><UL id="policy-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Science_Fair">Science Fair</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Theater_Project">Theater Project</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Public_Opinion">Public Opinion</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/College">College Talk</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Ethics">Ethics</A></LI></UL><UL id="notebook-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Labjournals">Labjournals</A></LI><LI><A class="navsub1 subsubmenu" href="#">Techniques</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Classic_vs_Gibson">Gibson</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Coli_Strains">E.coli Strains</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Protocols">Protocols</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Materials">Materials</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Wikimigration">Wiki-Tricks</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Glossary">Glossary</A></LI></DIV></DIV></DIV><DIV class="clearfix" id="smallnav"><UL class="clearfix"><LI id="home_small"><A href="https://2015.igem.org/Team:Freiburg/Home">Home</A></LI><LI id="project_small"><A href="https://2015.igem.org/Team:Freiburg/Project/Overview">Project</A></LI><LI id="results_small"><A href="https://2015.igem.org/Team:Freiburg/Results">Results</A></LI><LI id="team_small"><A href="https://2015.igem.org/Team:Freiburg/Team">Team</A></LI><LI id="policy_small"><A href="https://2015.igem.org/Team:Freiburg/Practices">Practice</A></LI><LI id="notebook_small"><A href="https://2015.igem.org/Team:Freiburg/Notebook">Notebook</A></LI></UL><A href="#" id="pull">Menu</A></DIV><DIV class="content_background_wrapper"><DIV class="central_position_wrap"><DIV id="bubble_wrapper"><DIV id="bubble_sidebar" class="bubble2"><P><A id="bubble2" href="#firstHeading">text</A></P></DIV><DIV id="bubble_sidebar" class="bubble3"><P><A id="bubble3" href="">next</A></P></DIV></DIV><DIV class="page_content"><DIV class="content_box" style="margin-top:0"><H1 class="sectionedit1">DNA Engineering</H1><P>
A protein array containing different antigens specific for distinct diseases is one of the main parts of the DiaCHIP. 
To obtain all the DNA constructs requires a lot of cloning, especially because <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Project/Protein_Purification" title="ProtPur">conventional protein expression</A> and expression using a <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Project/Cellfree_Expression" title="cell_free">cell-free system</A> are based on different plasmid backbones. 

To reduce this ambitious task to a minimum of effort, we elaborated a well-structured <A class="wikilink1" href="#detailed_cloning_anchor" title="Detailed_cloning">cloning strategy</A> including a self-designed <A class="wikilink1" href="#cloning_site_anchor" title="Cloning site">multiple cloning site</A>. This site was incorporated into the commercial expression vector pET22b+ resulting in <A class="wikilink1" href="#vector_design_anchor" title="vector_design">pET_iGEM</A>. 

We soon realized that  efficient protein expression is a problem many iGEM Teams around the world may be facing during their projects. Therefore, we provide the Registry with <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Description" title="„pOP“">pOP</A>, an expression vector suitable for iGEM standard cloning procedures.	

To help future iGEM Teams with their decision, which cloning method to use, we compared and contrasted classical cloning and Gibson Assembly in a <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Project/Classic_vs_Gibson" title="Review">short review</A>. 
</P><DIV class="flexbox" align="center" width="600px"><DIV class="horizontal_menu"><UL><LI><A href="#cloning_site_anchor" title="Cloning site">Our Cloning Site</A></LI><LI><A href="#vector_design_anchor" title="vector_design">Design of pET_iGEM</A></LI><LI><A href="#detailed_cloning_anchor" title="Detailed_cloning">Detailed Cloning Strategy</A></LI><LI><A href="https://2015.igem.org/Team:Freiburg/Description" title="„pOP“">   pOP - Protein Expression Meets iGEM Standards   </A></LI><LI><A href="https://2015.igem.org/Team:Freiburg/Project/Classic_vs_Gibson" title="Review">Short Review on Cloning Methods</A></LI></UL></DIV></DIV></DIV><DIV id="cloning_site" class="content_box"><H1>Our Cloning Site</H1><DIV class="floatbox left"><P>
The cloning site we developed enables the assembly of different antigens with a number of tags in a standardized manner making it simple for many people to work with. A first calculation based on the assumption of using about 15 antigens and four different tag systems resulted in a total of 120 constructs to be cloned. This underlines the need for a simplified cloning procedure. 
        </P><P>
We did not use one of the available iGEM standards as a new cloning site for several reasons.
        </P></DIV><DIV class="floatbox right"><P>
The RFCs are designed to enable serial assembly of various sequences, but the exchange of single parts at a later time point is not possible<SUP><A class="fn_top" href="#fn__1" id="fnt__1" name="fnt__1">1)</A></SUP>. In order to be able to exchange tags at the N-terminus as well as at the C-terminus, distinct restriction sites need to persist between the parts. Moreover, we first wanted to establish an expression cassette inserted between BioBrick pre- and suffix of the submission backbone <A class="urlextern" href="http://parts.igem.org/Part:pSB1C3" target="_blank" title="pSB1C3">pSB1C3</A>. Therefore, making use of an additional RFC was not possible.
	</P></DIV><DIV class="flexbox"><DIV class="thumb2 trien" style="width:680px"><DIV class="thumbinner"><P><STRONG> Figure 1: Self-designed cloning site.</STRONG> Our cloning site containing the restriction sites <I>Acc</I>65I, <I>Bam</I>HI, <I>Hind</I>III and <I>Afl</I>II, that are assembled in a way that facilitates exchange of tags without getting a frame-shift.</P></DIV></DIV></DIV></DIV><DIV class="floatbox left"><P>
The basic idea was to have distinct restriction sites which can be used for the insertion of antigens, while others can be used for the exchange of tags. Finally, this resulted in a cloning site containing <I>Acc</I>65I, <I>Bam</I>HI, <I>Hind</I>III and <I>Afl</I>II restriction sites (figure 1). Using those sites for the assembly of different antigen-tag combinations will never result in a frameshift, if the coding sequence is in frame with the restriction sites flanking it. 

The restriction sites have been chosen according to the amino acids that result from the translation of their sequence. <I>Acc</I>65I (G/GTACC) and <I>Bam</I>HI (G/GATCC) are translated into glycine and either serine or threonine. 
        </P></DIV><DIV class="floatbox right"><P>
Those are hydrophilic, soluble amino acids, which are commonly used in flexible linkers. The translation of <I>Hind</I>III (A/AGCTT) results in lysine and leucin. Especially lysine is highly hydrophobic, which is why some soluble amino acids should be used as a linker in front of a C-terminal tag. The last restriction site, <I>Afl</I>II (C/TTAAG), is also translated to leucin and lysine. This will not highly affect protein function or folding since these are the last translated codons. Anyways, in order to prevent rapid protein degradation according to the N-end-rule in bacteria<SUP><A class="fn_top" href="#fn__2" id="fnt__2" name="fnt__2">2)</A></SUP>, a few additional amino acids have been inserted between the <I>Afl</I>II site and the stop codon (TGA).
	</P></DIV><DIV class="footnotes"><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__1" id="fn__1" name="fn__1">1)</A></SUP><A class="urlextern" href="http://dspace.mit.edu/handle/1721.1/21168" rel="nofollow" target="_Blank" title="http://dspace.mit.edu/handle/1721.1/21168">Knight T 2003. Idempotent Vector Design for Standard Assembly of BioBricks.</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__2" id="fn__2" name="fn__2">2)</A></SUP><A class="urlextern" href="http://www.sciencemag.org/content/254/5036/1374.long" rel="nofollow" target="_Blank" title="http://www.sciencemag.org/content/254/5036/1374.long">Tobias JW 1991. The N-end rule in bacteria. Science.</A></DIV></DIV></DIV><DIV id="vector_design" class="content_box"><H1>Design of pET_iGEM</H1><DIV class="image_box right"><DIV class="thumb2 trien" style="width:260px"><DIV class="thumbinner"><P><STRONG>Figure 2: pIG15_001.</STRONG><A class="urlextern" href="http://parts.igem.org/Part:pSB1C3" target="_blank" title="pSB1C3">pSB1C3</A> backbone containing an altered cloning site in the middle of an expression cassette for improved assembly of protein coding sequences with different tags (N- or C-terminal), which avoids the formation of frameshifts. Important parts for overexpression of proteins in <I>E. coli</I> were added inside the RFC[10].</P></DIV></DIV></DIV></DIV><P>
		For protein expression, we designed two vectors based on the submission backbone <A class="urlextern" href="http://parts.igem.org/Part:pSB1C3" target="_blank" title="pSB1C3">pSB1C3</A>. pIG15_001 (figure 2) was supposed to be adapted for inducible protein overexpression in <I>E. coli</I>. Therefore, we inserted main features derived from the commercial expression vector pET22b+ between the BioBrick pre- and suffix of pSB1C3. These include a T7 promoter and terminator, a ribosomal binding site, the self-designed cloning site and a lacI expression cassette. Additionally, a pelB signal sequence for periplasmic translocation was added 5’ to the cloning site. 
</P><DIV class="image_box left"><DIV class="thumb2 trien" style="width:260px"><DIV class="thumbinner"><P><STRONG>Figure 3: pIG15_104.</STRONG> For cell-free expression the <A class="urlextern" href="http://parts.igem.org/Part:pSB1C3" target="_blank" title="pSB1C3">pSB1C3</A> backbone was altered similarly, but without a pelB secretion signal or lacI coding sequence and with a CMV promoter instead of a T7 promoter.In this case, the coding sequence for TurboYFP is inserted.</P></DIV></DIV></DIV></DIV><P>
The vector for cell-free expression (pIG15_104) was designed analogously, but lacking the lacI expression cassette and the signal sequence. The T7 promoter and terminator were exchanged by a CMV promoter and a WPRE terminator region for secretion into the medium. In figure 3, the vector is shown with an inserted coding sequence for TurboYFP.
</P><P>
Unfortunately, after the first expression experiment in <I>E. coli</I>, we realized that the yields of protein expression are not sufficient for our purposes. Instead of wasting time trying to optimize our own expression vector, we went on using the original vector and modified the cloning site to fit our requirements. The original vector pET22b+ and the modified version pET_iGEM are compared in figure 4.
</P><P>
Using an adapted commercial expression vector finally resulted in sufficiently high protein yields. The advantages of expression vectors like pET22b+ in contrast to cloning vectors like pSB1C3 are discussed in detail on the introduction page for <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Description" title="„pOP“">pOP</A>. This is a plasmid backbone we submitted to the iGEM Registry that is optimized for protein overexpression purposes and fully compatible with iGEM idempotent cloning standards of RFC[25].

The backbone for cell-free expression was not changed because a PCR product of the expression cassette serves as a template for the actual expression purpose making the vector it is assembled on irrelevant. Using a high-copy plasmid such as pSB1C3 rather simplifies cloning efforts. 
</P><DIV class="flexbox"><DIV class="thumb2 trien" style="width:1000px"><DIV class="thumbinner"><P><STRONG>Figure 4: Comparison of pET_iGEM and pET22b+.</STRONG> The two vectors differ in the cloning site that contains only four restriction sites in pET_iGEM (<I>Acc</I>65I, <I>Bam</I>HI, <I>Hind</I>III, <I>Afl</I>II) compared to the more complex multiple cloning site of the original pET22b+ (<I>BamHI</I>, <I>Eco</I>RI, <I>Sac</I>I, <I>Sal</I>I, <I>Hind</I>III, <I>Not</I>I, <I>Ava</I>I, <I>Xho</I>I). The 6xHis-Tag from pET22b+ was also removed to enable easy tag-exchanges with our system.</P></DIV></DIV></DIV></DIV></DIV><DIV id="detailed_cloning" class="content_box"><H1>Detailed Cloning Strategy</H1><P>
As mentioned before, we established a cloning strategy that is easy to handle for a team of researchers working together on the same project. It facilitates easy exchange of tags for custom purposes. All our cloning efforts began with either pET_iGEM (the modified version of pET22b+) for <I>E. coli</I>-based protein expression or pIG15_104, where the expression site derived from pET22b+ was inserted into pSB1C3.

        </P><H2>The Basic Constructs (Protein Purification Using <I>E. coli</I>)</H2><H3>pET22b+ with our cloning site (pET_iGEM)</H3><P>
To modify the multiple cloning site of pET22b+ for our purposes, we performed a Gibson Assembly with only one fragment. Therefore, we designed primers for the amplification of the whole plasmid, except the MCS. The cloning site we designed ourselves was part of the primer extension.
        </P><H3>pET_iGEM with His-tagged Herpes simplex antigen (pET_803)</H3><P>
To insert the very small but commonly used His-tag, we again used Gibson Assembly because classical cloning of such small parts can be a challenging task. The respective primers were designed for amplification of <A class="media" href="http://parts.igem.org/Part:BBa_K1621002" target="_blank" title="BBa_1621002">antigen 8 (Herpes Simplex Virus Type 1, glycoprotein G1)</A> to combine the insertion of the tag and an antigen. While the forward primer was a regular Gibson primer, the reverse primer additionally contained the sequence of a 10xHis-tag. After amplification, the part was inserted into the <I>Bam</I>HI and <I>Afl</I>II digested backbone pET_iGEM resulting in the first antigen with N-terminal His-tag.
        </P><H3>pET_iGEM with Spy-tagged Herpes simplex antigen (pET_804)</H3><P>
Another tag we wanted to use for immobilization of the antigens on the surface is the <A class="media" href="http://parts.igem.org/Part:BBa_K1159201" target="_blank" title="BBa_K1158201">SpyTag</A>. This part of 39 bp length was inserted into pET_iGEM analogous to the 10xHis-tag. The only difference was that the forward primer for amplification of antigen 8 included a standard 6xHis-tag, which enables the purification of the protein. Consistent with our cloning strategy, the amplified part was inserted between <I>Acc</I>65I and <I>Afl</I>II as it contains an N-terminal as well as a C-terminal tag. <I>Bam</I>HI and <I>Hind</I>III restriction sites were retained flanking the antigen, making it freely exchangeable.
        </P><H3>pET_iGEM with Herpes simplex antigen with HaloTag (pET_805)</H3><P>
The only tag we used that was not inserted via Gibson Assembly was the HaloTag. With a length of about 900 bp, classical cloning of this part was no trouble. The <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Project/Surface_Chemistry#halo_surface_anchor" titlre="halo">HaloTag</A> exists in two variants, one being optimized for C-terminal tagging of the protein, the other for N-terminal tagging. As the tag we had access to was the C-terminal one, the SpyTag in pET_804 was exchanged using the restriction sites <I>Hind</I>III and <I>Afl</I>II resulting in pET_805.
        </P>

A list of all our plasmids can be found <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Labjournals/Plasmids">here.</A><DIV class="flexbox"><DIV class="thumb2 trien" style="width:680px"><DIV class="thumbinner"><P><STRONG> Figure 1: Detailed view of our self-designed cloning site.</STRONG> Our cloning site containing the restriction sites <I>Acc</I>65I, <I>Bam</I>HI, <I>Hind</I>III and <I>Afl</I>II, with initiation site and termination site. Different antigen coding sequences or tags can be easily exchanged. We used this system in our constructs for protein expression in <I>E. coli</I> as well as in constructs we used for cell-free expression.</P></DIV></DIV></DIV></DIV><H2 class="sectionedit2">Tagged Constructs (Protein Purification Using <I>E. coli</I>)</H2><P>
A tag-system minimizes unspecific binding of proteins to the chip surface, therefore we tested many different <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Project/Surface_Chemistry" title="tag_systems">tag systems</A>.
</P><H3>His-tag</H3><P>
Every antigen was genetically fused to a C-terminal 10xHis-tag, which could be used for its immobilization on the surface as well as for its purification. The basic construct for this purpose was pET_803. The Herpes Simplex Virus derived antigen, which was the very first we finished cloning, had to be excised by restriction with <I>Bam</I>HI and <I>Hind</I>III. Thus, all the other antigens digested in the same way could be ligated into this backbone.
</P><H3>SpyTag</H3><P>
Another tag for surface immobilization of the antigen is the SpyTag. To enable protein purification via Ni-NTA columns, proteins fused to the SpyTag were additionally fused to a C-terminal 6xHis-tag. The basic vector for cloning these constructs was pET_804. Excision of the Herpes Simplex Virus derived antigen and insertion of other antigens was performed analogous to the antigen exchange in His-tag constructs.
</P><H3>HaloTag</H3><P>
As a second covalent tag-system for immobilization of the antigens on the surface we used the HaloTag, which specifically binds to a chloroalkane surface. Using our own cloning strategy, the HaloTag could easily be inserted between <I>Hind</I>III and <I>Afl</I>II, using pET_803 as basis. Cloning of the HaloTag into pET_803 leads to a new vector, pET_805. This vector is then used to substitute the antigens. As in the SpyTag construct, this new vector contains a C-terminal 6xHis-tag for purification of the antigens.
</P><H2 class="sectionedit2">Cell-Free Backbone</H2><P>
As described above we decided to use the <A class="media" href="http://parts.igem.org/Part:pSB1C3" target="_blank" title="pSB1C3">pSB1C3</A> standard vector as backbone for our <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Project/Cellfree_Expression" title="cellfree">cell-free expression</A> constructs. Inserted between the <A class="wikilink1" href="http://parts.igem.org/Help:Standards/Assembly/RFC10" target="_blank" title="RFC10">RFC 10</A> prefix and suffix is our own cloning site, containing the following restriction sites: <I>Acc</I>65I, <I>Bam</I>HI, <I>Hind</I>III and <I>Afl</I>II. We included a T7 promoter and a ribosomal binding site between the RFC 10 prefix and the <I>Acc</I>65I restriction site. A stop codon cassette and a T7 terminator were inserted between the <I>Afl</I>II restriction site and the RFC 10 suffix. These similarities to the expression vector facilitate the easy exchange of fragments between the standard expression vector and the cell-free expression vector. Using our cloning site it is rather simple to exchange different tag-combinations. The advantage of the pSB1C3 backbone is the high copy origin, allowing a rapid multiplication and easy cloning. For the cell-free expression itself the vector backbone is dispensable as only the part amplified via PCR between the prefix and the suffix is mandatory for cell-free expression.
</P><H2 class="sectionedit2">The Basic Constructs (Cell-Free Expression)</H2><H3>Vector for cell-free expression with tYFP with SpyTag (pIG15_104)</H3><P>
For the first basic construct we ordered a gBlock from <A class="urlextern" href="https://eu.idtdna.com/site" target="_blank" title="IDT">IDT</A> containing the T7 promoter, ribosomal binding site, our cloning site and the T7 terminator with adjoining stop codon cassette. Between <I>Bam</I>HI and <I>Hind</I>III the coding sequence for tYFP and between <I>Hind</I>III and <I>Afl</I>II the SpyTag sequence was inserted. This gBlock was then inserted into the pJET1.2 vector. Afterwards, it was digested out of pJET1.2 and classically cloned into the pSB1C3 backbone resulting in the first basic construct for cell-free expression. We inserted tYFP as default fragment into the cell-free expression backbone because we already had some anti-tYFP antibody available. Hence, first measurements for analyzing the binding of anti-tYFP to tYFP could be performed as soon as the basic construct was finished. Besides, tYFP could be used as reporter. Using <I>Bam</I>HI and <I>Hind</I>III restriction sites tYFP could be excised and replaced by antigen sequences analogous to the cloning system used for the expression vector.
</P><H3>Vector for cell-free expression with tYFP with HaloTag (pIG15_105)</H3><P>
Using the <I>Hind</I>III and <I>Afl</I>II restriction sites, the SpyTag sequence was excised from pIG15_104. For our second covalent tag system we inserted the sequence for the HaloTag via classical cloning and thereby producing our next basic construct for cell-free expression, pIG15_105. This backbone again contains tYFP as default insert, this time with a C-terminal HaloTag.
Using identical cloning strategies as with the first basic construct (pIG15_104) all antigen sequences could be inserted into this vector using <I>Bam</I>HI and <I>Hind</I>III.
</P></DIV></DIV></DIV></BODY></HTML>