Team:Freiburg/Project/Coli Strains
""
Home
Project
Team
Results
Human Practice
Notebook
Introduction
Overview
Diagnostics today
The DiaCHIP
DNA Engineering
Cell-Free Expression
Protein Purification
Surface Chemistry
Optical Detection
Outlook
Members
Acknowledgements
Partners
The DiaCHIP
Main Results
Diagnostics
Our Device
DNA Array
Cell-Free Expression
Binding on Surface
Modeling
BioBricks
Collaborations
Interlab Study
Achievements
Science Fair
Theater Project
Public Opinion
College Talk
Ethics
Labjournals
Techniques
Gibson
E.coli Strains
Protocols
Materials
Safety
Wiki-Tricks
Glossary
Home
Project
Results
Team
Practice
Notebook
Menu
text
next
E.coli genotypes
We are using E.coli strains for overexpression of our target proteins. E. coli is suitable for expression of a wide range of heterologous proteins. Our aim was the enrichment of active, soluble recombinant protein. Therefore, we used different E.coli strains and adapted different expression temperatures, durations and IPTG concentrations to get as much soluble target protein as possible.  Of course, many challenges can arise during expression of recombinant proteins. That is why most expression strains have a few mutations in common to obtain a high yield of expressed proteins.  Expression strains often harbor mutations that make them deficient in outer membrane protease VII and delete the Ion protease. Both of this reduces proteolysis of the expressed recombinant proteins. Additionally the native restriction/methylation system is often inactivated 1) 2) .   Below we introduce the bacterial strains we used for protein overexpression:
Bl21 (DE3) pLysS
BL21 (DE3) is the basic IPTG-inducible strain containing T7 RNA-polymerase (DE3) for general protein expression 3) . The E.coli expression strain BL21 (DE3) pLysS is useful for the expression of toxic proteins.  The pLysS plasmid contains a chloramphenicol resistance. The pLysS plasmid also encodes T7 phage lysozyme, an inhibitor for T7 polymerase. In the absence of IPTG induction this inhibitor reduces and almost eliminates expression from a transformed T7 promoter containing plasmids . This inducable expression is suitable to stabilize recombinants encoding particularly toxic proteins 4) 5) .
Rosetta(DE3)pLysS
Rosetta™ host strains are BL21 derivatives designed to enhance the expression of eukaryotic proteins that contain codons rarely used in E. coli. These strains supply tRNAs for AGG, AGA, AUA, CUA, CCC, GGA codons on a compatible chloramphenicol-resistant plasmid. The Rosetta strains provide a better translation for eukaryotic proteins that have not been codon optimized. In Rosetta(DE3)pLysS, the rare tRNA genes are present on the same plasmid that carries the T7 lysozyme gene for inducable expression 6) .
C43 (DE3)
This overexpression strain derives from the C41 (DE3) E.coli strain.  These strains contain an uncharacterized genetic mutation that allows a higher tolerance to toxic proteins. The mutation prevents E.coli cell death often associated with expression of many recombinant toxic proteins.
The advantage of this strain to BL21 (DE3) is that even membrane proteins, often particularly problematic for protein expression, can be overexpressed 7) .
Received from the Toolbox - BIOSS (Centre for Biological Signalling Studies).
ArcticExpress (DE3)RP
ArcticExpress Competent Cells are engineered to address the common bacterial gene expression hurdle of protein insolubility. These cells are derived from the high-performance Stratagene BL21-Gold competent cells, enabling efficient high-level expression of heterologous proteins in E. coli. With these cells we were able to express our target proteins o/n at 10°C. Low-temperature cultivation represents one strategy for increasing the recovery of soluble protein. This is due to the fact that E. coli chaperonins, which facilitate proper protein folding by binding and stabilizing unfolded or partially folded proteins, lose activity at reduced temperatures 8) 9) 10) .
Received from AG Weber - BIOSS (Centre of Biological Signalling Studies).
Top10
The Top10 cloning strain harbors some mutations especially useful for high efficiency cloning purposes and for propagation of plasmids. In this strain, the gene for the type I endonuclease complex KI that recognizes unmethylated DNA is not functional 11) . So genes from PCR amplification are not degraded by this defense-system and can be efficiently transformed into the bacteria.
Additionally, a mutation in a nonspecific endonuclease (endA) yields cleaner preparations due to less unspecifically cut double-strand DNA 12) .
Finally the rate of homologous recombination that can hinder efficient cloning is reduced by a mutation in the DNA strand exchange and recombination protein recA 13) .
info
1) Demerec M. et al. (1966). A proposal for a uniform nomenclature in bacterial genetics. Genetics. 54(1):61-76
2) Grodberg, J. and Dunn, J. J. (1988) J Bacteriol 170(3):1245-53
3) Moffatt BA, Studier FW (1987). T7 lysozyme inhibits transcription by T7 RNA-polymerase. Cell. 49(2):221-7
4) Jerpseth, M., Jerpseth, B., Briester, L. and Greener, A. (1998) Strategies 11(1):3-4
5) Jerpseth, B., Callahan, M. and Greener, A. (1997) Strategies10(2):37–38
6) Looman et al. (1987) EMBO J. 6, 2489-2492
7) B Miroux and J E Walker, ‘Over-Production of Proteins in Escherichia Coli: Mutant Hosts That Allow Synthesis of Some Membrane Proteins and Globular Proteins at High Levels.’, Journal of molecular biology, 260 (1996), 289–98
8) Carstens, C.-P., Bonnardel, J., Allen, R. and Waesche, A. (2001) Strategies 14:50
9) Schein, C. H. (1989) Biotechnology7:1141-8
10) Ferrer, M., Chernikova, T. N., Yakimov, M.M., Golyshin, P. N.and Timmis, K. N. (2003) Nat Biotechnol.21(11):1266-7
11) http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&object=EG10459
12) http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&object=EG11336
13) http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&object=EG10823
