Team:Freiburg/Labjournals/Cellfree/June
""
Home
Project
Team
Results
Human Practice
Notebook
Introduction
Overview
Diagnostics today
The DiaCHIP
DNA Engineering
Cell-Free Expression
Protein Purification
Surface Chemistry
Optical Detection
Outlook
Members
Acknowledgements
Partners
The DiaCHIP
Main Results
Diagnostics
Our Device
DNA Array
Cell-Free Expression
Binding on Surface
Modeling
BioBricks
Collaborations
Interlab Study
Achievements
Science Fair
Theater Project
Public Opinion
College Talk
Ethics
Labjournals
Techniques
Gibson
E.coli Strains
Protocols
Materials
Safety
Wiki-Tricks
Glossary
Home
Project
Results
Team
Practice
Notebook
Menu
<< Back
Labjournal Cellfree June
Tue, 30.06.2015
Mini prep of pJet with Halo PCR-Product was repeated
buffer two of biozym miniprep kit ran out and buffer two of roche kit was substituted
miniprep did not work, product not pure, poor 260/280 values and no DNA-resembling curve on NanoDrop
Mon, 29.06.2015
Mini Pep of pJet with Halo PCR-Product
4 Different Clones were prepped
unfortunately ethnol-containing buffer made inscriptions invisible –> numbers don't match plates any more
each four clones were picked from the classical cloning and cloning plate. For further identification numbers were added.
Sun, 28.06.2015
Cloning of pIG15_105 approaches:
pJET with Halo-Tag: Picking of Clones from Trafo
Fusion PCR approach –> no correct bands after test digest –> failed
Gibson: small colonies –> transfer tomorrow
classical cloning: only 30 minute ligation at night, so one part was transformed, while the other part proceeded with ligation @4°C
Sat, 27.06.2015
Continuation of Western blot from 26.06.2015
1st antibody incubation. 1h @ RT  (AB-dilution 1:1 ,000 in blocking buffer with 1% milk powder)
only Anti-Gfp (monoclonal) was used ( AB received from Phillip Schwenk)
Washing membrane 3x 15 min in blotting buffer
2nd antibody incubation. 1 h @ RT (AB-dilution 1: 10,000 in Blotting buffer anti-mouse, 1:20,000for ECL)
Washing membrane 3x 10min in blotting buffer
mesuring in ECL reader for 7,5 min (every 30 sec)in serial mode –> only noise on picture
measuring for 3 min with 8×8 binning –> empty blot
new approaches for pIG15_104:
blunt end ligation of halo and pJET for better chances in restriction digestion → repetition of halo pcr
fusion pcr of pIG15_104 and halo from original plasmid
Gibson of pIG15_104 and halo pcr product
classical: digestion of halo pcr product and pIG15_104
30 minute ligation,transformation _> rest left in fridge over night at 4°C for second transformation
ligation of empty vector as control
Fri, 26.06.2015
Westernblot of first and second cellfree Expression
20 µl per sample (16µl probe and 4µl Sds-Puffer) besides neg control of first expression there were only 10µl left
pipetting sheme
YFP
GFP
negativ
marker
YFP
GFP
negativ
1.EX
1.Ex
1.Ex
-
2.Ex
2.Ex
2.Ex
Cutting of PVDF membrane and 4 whatman-papers in gel size
PVDF membranes were  activated in 100% MeOH before use
Semidry blotting
Stack: 2x Whatmann, Filter, Gel, 2x Whatman
everything was soaked in blotting buffer prior to use
(no bubbles after adding each layer)
reduction of buffer necessary due to electrical short
Blot for at 0,2 mA <note tip>tip</note>h  (0.8mA/cm2 )
Blot was dismanteled
whatmann paper were washed and drieed for reuse
Gel was stained with Comassie and dried in Ethanol solution for storage
Block membrane ca 30 min in blocking buffer (Blottin Buffer with 5% Milk pulver)
Washing with Blotting Buffer 3 x 15 min
last wash overnight @ 4°C
Thu, 25.06.2015
Minipreps of the 8 colonies
Control digest of possible pIG15_105 and gel with SpeI-HF and HindIII-HF (expected band size:1077 bp 2935 bp)
2nd cellfree expression
100µl per reaction, 3´.5 h duration @37°C in the thermomixer with 300rpm
as DNA templates:
pQE-60 with GFP-His from the toolbox as outside standard (164ng/µl)
pIG15_104 with HIS-tYfp-Spy (134ng/µl)
and water as negative control
everything was pipetted on ice reaction was stared by addition of DNA and immiditae placement in the thermomixer
DNA
1x [µl]
3x [µl]
Premix
10
30
Amino Acids
10
30
PEP
4,2
12,6
CoA
1,212
3,636
tRNA
17,5
52,5
ATP
2,5
7,5
CTP
1,7
5,1
GTP
1,7
5,1
UTP
1,7
5,1
Lysate
24
72
DNA
DNA [µl]
Water [µl]
Mastermix
pIG15_104
11,194
14,294
74,512
pQE-60
9,146
16,342
74,512
-Con
0
25,488
74,512
Afterwards 0,5 µl were pipetted on a objectslide and analyzed under the fluoresence microscope (Gfp filter)
only GFP and negative control were observed
clear difference to negative control
pictures of GFP –> saved as nef format (in future Tiff)
Problems with to ast bleaching for taking a photo
variation of shutter speed etc.
Rest of the samples was frozen at -20°C
Wed, 24.06.2015
1st cellfree expression
25 µl per reaction, 2 h duration @37°C in the thermomixer with 300rpm
As DNA templates:
pQE-60 with GFP-His from the toolbox as outside standard (164ng/µl)
pIG15_104 with HIS-tYfp-Spy (134ng/µl)
and water as negative control
everything was pipetted on ice reaction was stared by addition of DNA and immiditate placement in the thermomixer
DNA
1x [µl]
3x [µl]
Premix
2,5
7,5
Amino Acids
2,5
7,5
PEP
1,05
3,15
CoA
0,303
0,909
tRNA
4,375
13,125
ATP
0,625
1,875
CTP
0,425
1,27
GTP
0,425
1,275
UTP
0,425
1,275
Lysate
6
18
DNA
DNA [µl]
Water [µl]
Mastermix
pIG15_104
3,760
3,760
18,628
pQE-60
2,134
4,238
18,628
neg control
0
6,372
18,628
Afterwards 0,5 µl were pipetted on coverslip and analyzed under the fluoresence microscope (Gfp filter)
only GFP and negative control were observed
both shine green without differnence
Rest of the samples was frozen at -80°C
Tue, 23.06.2015
Picking of 8 colonied from all plates with the different cloning strategies
Mon, 22.06.2015
Trafo of overnight ligation for pIG15_105
Digest of pJET with Halo-Tag insert
two step process with first one hour with AflII and than one hour HindIII
more DNA used for sigest (1µg Halo and 0,5µg Backbone)
digest of pIG15_104 in two steps
Ligation of new digest
Trafo of new digest
Sun, 21.06.2015
over day digest of Halo in pJET1.2 blunt vector for pIG15_105
Fri, 19.06.2015
sending clones out for sequencing
preparing extract buffers with DTT and ßME
ligation of HaloTag into pJET1.2 blunt vector from yesterday  * failed due to wrong buffer or no cells in transformation mix
ligation of HaloTag into pJET1.2 blunt vector
Thu, 18.06.2015
sending clones out for sequencing:
preparing extract buffers with DTT and ßME
ligation of HaloTag into pJET1.2 blunt vector
Wed, 17.06.2015
lab today slightly less occupied by strange forces from other worlds.
miniprep of  all 4 A2 + halo (pIG15_105)</todo>
comment:
1: 67,2 mg/ml
2: 78,8 mg/ml
3: 66,1 mg/ml
4: 60,1 mg/ml
control digest of pIG15_105 with HindIII and Afl II
control with gel:
at all digests: visible band at 3kb, which would be the size of the digested backbone, but no insert (a.k.a. HaloTag) at expected 880bp visible on the gel.
today the final chemicals arrived: folinic acid, cAMP, K-glutamate and the 20-L-Amino-Acid kit from Sigma were given to us from thr AG Roth.
the aminos were then dissolved to 80 mM, which is 10x the concentration of the stock of the „Bernhard-protocol“ and stored at 4°C in a 15ml falcon.
1l of the culture needed for the cellfree extract-preparation was prepared following the EMBL protocol.
Tue, 16.06.2015
our lab was still occupied by strange forces from other worlds.
replating of four colonies on plate 1 of transformation (A2 + halo)
miniprep of A2 liquid colonies
Mon, 15.06.2015
today our lab was occupied by strange forces from other worlds.
both HaloTag PCR-clean-up and the original A2 plasmid backbone were digested
HaloTag insert
component
amount [µl]
plasmid (concentration=47,8ng/µl)
2
CutSmart buffer
2
Hind III HF
1
Afl II
1
ddH2O ad 10µl
4
A2 backbone
component
amount [µl]
plasmid (concentration=45ng/µl)
2
CutSmart buffer
2
Hind III HF
1
Afl II
1
Phosphatase Buffer
2
Antarctic Phosphatase
1
ddH2O to 20µl
1
The digested parts were then ligated:
component
amount [µl]
plasmid backbone
2
plasmid insert
16
T4 DNA Ligase buffer
4 (because old)
T4 DNA Ligase
1
Ligation performed at RT for 1h.
The A2 backbone alone was religated as negative control.
component
amount [µl]
plasmid backbone
1
T4 DNA Ligase buffer
2
T4 DNA Ligase
1
H2O ad 20
17
A transformation was performed using 25µl TOP10 competent cells with 10µl of plasmid-ligation product following our standard protocol. 2 plates were streaked and are grown o/n.
To produce more of the plasmid backbone, a liquid culture of the A2 clone was inoculated.
Sun, 14.06.2015
gelelectrophoresis of pcr product showed no/ wrong bands.
lane1: empty - lane2: 2log dna ladder - lane3+4: sample+ midoridirect - lane5: ladder  - lane6:empty
repeating PCR, using same amounts and program as yesterday Sat, 13.06.
repeating gelelectrophoresis (1% agarose), this time using in-gel staining via 4µl MidoriGreen Adavnce instead of sample-loaded staining.
second PCR and gelectrophoresis showed the right band at about 880 bp:
band was cut out and purified using qiaquick gelextraction kit. (30µl water instead of elution buffer)
only 5µl of the PCR product was loaded onto the gel, the other 15µl were directly purified using the Zymo DNA-Clean&Concentration Kit.
Changes: Instead of 6µl elution buffer, 8µl of warm water was used. After that, the column was eluted for a second time with the same amount of water. The second elution still  contained 10ng/µl of DNA.
Sat, 13.06.2015
Due to lack of growth on plates, restart of procedure from PCR of HaloTag onwards.
Template dilution 1:20 → 1,4 ng/µl
* PCR of Halo tag
Temperature °C
duration
98
5 min
98
10 sec
63.5
20 sec
72
22 sec
72
5
4
infinite
buffer - Phusion HF
5
dNTPs
0,5
Primer forward
1,25
Primer reverse
1,25
Phusion Polymerase
0,25
Template <250 ng
1
H2O to 25µl
10,75
Primers used: 105f + 106r
Product placed in cooler at 4°C.
Thu, 11.06.2015
no growth on plates. Transformation repeated with 25µl aliquot of TOP10 cells.
Wed, 10.06.2015
sequencing results for sample two of both plates were positive
both clones 2 (plate A and B) were replated onto new plates
Test Digestion of Halo PCR cleanup product and „A2“ clone plasmid
with Afl I + Hind III, following standard amounts
Separation using 1% Agarose Gel, 120V, 40min
Imaging showed two clear bands, each one at the right size: the halo product around 800bp, the linearized backbone of the A2 clone backbone at 3000bp:
Gelextraction using Qiagen Qiaquick Kit, elution with 30µl ddH20 at 30°C
concentration: 6,6ng/µl for Halo digest, 45ng/µl for A2 digest
Ligation: 30µl of Halo plasmid, 1µl of A2 plasmid, 3h at RT
Transformation into 100µl of TOP10 competent cells
Tue, 02.06.2015
pIG_003 sequencing didn't work bc primers with gibson overhang were mistakingly chosen
pIG_003 sequencing is repeated with cloning group primers oIG_s001 ans oIG_s003
Tue, 02.06.2015
Halo tag pcr didn't work → wrong bands on gel
Template dilution 1:20 → 1,4 ng/µl
* PCR of Halo tag
Temperature °C
duration
98
5 min
98
10 sec
63.5
20 sec
72
22 sec
72
5
4
infinite
buffer - Phusion HF
5
dNTPs
0,5
Primer forward
1,25
Primer reverse
1,25
Phusion Polymerase
0,25
Template <250 ng
1
H2O to 25µl
10,75
PCR product was loaded on 1% agarose gel and let run for 35min at 120V
though no ladder was visible, the only visible band was cut out from the gel and extracted using the QiaQuick Gel-Extraction Kit.
The elution should now contain the Halo tag
pIG_003 was sent out for sequencing
Mon, 01.06.2015:
miniprep of colonies from g-block + pSB1C3 ligation
control digest, gel
prepped plasmids from the colonies were prepared for sequencing
PCR of Halo tag
Temperature °C
duration
98
5 min
98
10 sec
63
20 sec
72
40 sec
72
5
4
infinite
Sun, 31.05.2015:
* Re-ligation of pIG15_001 backbone with gBlock inser  * Transformation of overnight-ligation from Fri, 29.05 in TOP10 cells
checking for colonies:  both plates grew about 20 - 50 colonies. 4 colonies were picked from each plate and re-plated.
