Team:Freiburg/Description
""
Home
Project
Team
Results
Human Practice
Notebook
Introduction
Overview
Diagnostics today
The DiaCHIP
DNA Engineering
Cell-Free Expression
Protein Purification
Surface Chemistry
Optical Detection
Outlook
Members
Acknowledgements
Partners
The DiaCHIP
Main Results
Diagnostics
Our Device
DNA Array
Cell-Free Expression
Binding on Surface
Modeling
BioBricks
Collaborations
Interlab Study
Achievements
Science Fair
Theater Project
Public Opinion
College Talk
Ethics
Labjournals
Techniques
Gibson
E.coli Strains
Protocols
Materials
Safety
Wiki-Tricks
Glossary
Home
Project
Results
Team
Practice
Notebook
Menu
text
next
pOP – Protein Expression Meets iGEM Standards
Kann manda am Anfang irgendein Bild reinmachen? Weil sonst wird man dirket mal von der wall.of-text erschlagen. Hba auhc noch ein paar typos rausgeworfen.. (ps1709) Und in der Registry fehlen bisher die Bilder...
All the constructs we cloned for this year’s iGEM Competition were meant to be used for overexpression of the respective proteins, either in a cell-free expression system or in an E. coli expression strain. Usually, vectors that are optimized for protein overexpression are low- to medium-copy plasmids. In contrast to cloning purposes, this is sufficient for protein expression. When a certain copy number is reached, the expression machinery of the cell is saturated and a further increase will not result in higher expression yields. High-copy plasmids result in lower expression rates as more metabolic capacity is used for replication.
Many E. coli strains optimized for overexpression carry a chloramphenicol resistance as a selective marker. Plasmids carrying the same resistance are not suitable for protein expression for two reasons. First, double selection for successfully transformed colonies on a medium with two antibiotics is not possible. Second, there is no selective pressure that stops the cells from releasing the plasmid.
The iGEM standard vector pSB1C3 was designed for cloning purposes and thus feature an origin of replication yielding high plasmid copy numbers. Moreover, it carries a chloramphenicol resistance gene as selection marker what makes it a tough task to use this vector for successful expression purposes.
Using a modified version of pSB1C3 (pIG15_803) for overexpression resulted in an unsatisfying protein amount . Therefore, we decided to use a common expression vector for further experiments.
Obviously, it is a lot of work to clone every part we want to send to the Registry into an expression vector and additionally into pSB1C3 , which exhibits a completely different cloning site. Due to those reasons we aimed to add a vector backbone to the iGEM Registry which is suitable for efficient protein overexpression and compatible for cloning parts derived from the Registry in a standardized procedure.
Figure 1: pOP (plasmid for Overexpression of Proteins). Expression vector based on pET22b+ 1) that is fitted to iGEM Standards by incorporation of the Freiburg cloning standard (RFC[25]) and removal of forbidden restriction sites.
The improved vector, called pOP – plasmid for Overexpression of Proteins (figure 1) combines features needed for efficient overexpression of proteins with standardized elements derived from pSB1C3. Based on the commonly used expression vector pET22b+ 1) and the BioBrick pSB6A1 , some changes had to be applied to fit all the iGEM Standards.
First, one of the standard cloning sites had to be inserted instead of the original multiple cloning site of pET22b+. RFC[25] seemed to be the most suitable one for protein expression purposes. It allows the assembly of several coding sequences as the scar left between the sequences does not result in a frameshift or a stop codon. This is an advantage for expression of fusion proteins on the one hand and enables to genetically fuse the protein of interest to a specific tag for affinity purification or similar applications on the other hand. Additionally, signal sequences can be added for example to cause periplasmic secretion of the protein, which is a tool commonly used for simplification of protein purification .
Next, recognition sites for restriction enzymes that are used for the insertion of coding sequences had to be eliminated in the backbone, in order for those enzymes (namely AgeI, EcoRI, NgoMIV, NotI, SpeI, PstI and XbaI) to remain single cutters.
To allow sequence analysis in a standardized way, the binding sites of the primers VF2 and VR , as they are used by the iGEM Registry, have been inserted in an appropriate distance to the insertion site.
The final vector was designed theoretically based on pET22b+ and pSB6A1 . As the sites where adaptations had to be applied were distributed all over the vector, a multi-step mutagenesis approach would have been hard to realize. Instead, the vector was divided into five fragments that were pieced together in two steps by fusion PCR and Gibson Assembly. The resulting plasmid is shown in figure 1.
Taken together, we designed an improved plasmid for the overexpression of proteins that is compatible with iGEM cloning standards. We are happy to provide this new backbone to the iGEM Registry and help future iGEM Teams to reach high protein expression yields in an easy way.
Link to GenBank file: BBa_K1621009.gb .
Link to Registry: BBa_K1621009
pOP – Successful Induction of Protein Expression in E. coli BL21
We inserted the coding sequence for GFP ( BBa_I13504 ) into the improved expression vector pOP using EcoRI and PstI for restriciton. After ligation of the digested pOP plasmid with GFP we transformed E. coli BL21 cells with the ligation mix.
Correct clones were selected for further analysis. To show that the new improved backbone is capable of protein overexpression upon induction with IPTG, two cultures of 5 ml each were inoculated from one colony. Both cultures were incubated at 37°C until they reached an OD600 of 0.5. At this time point one culture was induced with IPTG in a final concentration of 1 mM, whereas in the other culture protein expression was not induced. Both cultures were grown for several hours after induction on 37°C.
Successful expression of GFP was observed by eye (figure 2) and analyzed using SDS-PAGE (figure 3).
Figure 3: SDS-PAGE of pOP containing GFP coding sequence. GFP exhibits a molecular weight of 27 kDa, which is indicated by the arrow. SDS-PAGE was performed using 1 ml of cell culture. The enrichment of the target protein is clearly visible for the induced culture.
We could show that the improved backbone pOP can be used to efficiently overexpress proteins in E. coli. Therefore, future iGEM Teams can now access a standardized backbone to express large amounts of protein in a simple way.
Figure 2: Pelleted liquid culture pOP containing GFP coding sequence. The difference in the amount of expressed GFP with and without GFP in the pOP vector can easily be observed by the yellowish color of the pellet of the induced culture.
References
1) pET system. Novagen.
info
