<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Project_Methods_Cloning skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Project/Methods/Cloning</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>&quot;&quot;</TITLE></P><DIV id="header"><DIV id="naviconwrapper"><DIV id="naviconbar"><DIV id="navicons"><UL><LI id="home"><A href="https://2015.igem.org/Team:Freiburg/Home">Home</A></LI><LI id="project"><A href="https://2015.igem.org/Team:Freiburg/Project/Overview">Project</A></LI><LI id="team"><A href="https://2015.igem.org/Team:Freiburg/Team">Team</A></LI><LI id="results"><A href="https://2015.igem.org/Team:Freiburg/Results">Results</A></LI><LI id="policy" style="font-size:11px"><A href="https://2015.igem.org/Team:Freiburg/Practices">Human Practice</A></LI><LI id="notebook"><A href="https://2015.igem.org/Team:Freiburg/Notebook">Notebook</A></LI></UL></DIV></DIV></DIV><DIV id="navtextwrapper"><DIV id="navtext"><UL id="project-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/Overview">Introduction</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/System">Overview</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Design">Diagnostics today</A></LI><LI><A class="navsub1 subsubmenu" href="#">The DiaCHIP</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/DNA_Engineering">DNA Engineering</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Cellfree_Expression">Cell-Free Expression</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Protein_Purification">Protein Purification</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Surface_Chemistry">Surface Chemistry</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/iRIf">Optical Detection</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/Future_Directions">Outlook</A></LI><UL id="team-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Team">Members</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Attributions">Acknowledgements</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Partners">Partners</A></LI></UL><UL id="results-sub"><LI><A class="navsub1 subsubmenu" href="#">The DiaCHIP</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results">Main Results</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Diagnostics">Diagnostics</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Own_Device">Our Device</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Immobilization">DNA Array</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Cellfree">Cell-Free Expression</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Surface">Binding on Surface</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Modeling">Modeling</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Parts">BioBricks</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Collaborations">Collaborations</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/InterLab_Study">Interlab Study</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Achievements">Achievements</A></LI><UL id="policy-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Science_Fair">Science Fair</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Theater_Project">Theater Project</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Public_Opinion">Public Opinion</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/College">College Talk</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Ethics">Ethics</A></LI></UL><UL id="notebook-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Labjournals">Labjournals</A></LI><LI><A class="navsub1 subsubmenu" href="#">Techniques</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Classic_vs_Gibson">Gibson</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Coli_Strains">E.coli Strains</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Protocols">Protocols</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Materials">Materials</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Wikimigration">Wiki-Tricks</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Glossary">Glossary</A></LI></DIV></DIV></DIV><DIV class="clearfix" id="smallnav"><UL class="clearfix"><LI id="home_small"><A href="https://2015.igem.org/Team:Freiburg/Home">Home</A></LI><LI id="project_small"><A href="https://2015.igem.org/Team:Freiburg/Project/Overview">Project</A></LI><LI id="results_small"><A href="https://2015.igem.org/Team:Freiburg/Results">Results</A></LI><LI id="team_small"><A href="https://2015.igem.org/Team:Freiburg/Team">Team</A></LI><LI id="policy_small"><A href="https://2015.igem.org/Team:Freiburg/Practices">Practice</A></LI><LI id="notebook_small"><A href="https://2015.igem.org/Team:Freiburg/Notebook">Notebook</A></LI></UL><A href="#" id="pull">Menu</A></DIV><DIV class="content_background_wrapper"><DIV class="central_position_wrap"><DIV class="page_content"><DIV class="content_box"><H1 class="sectionedit1">Cloning methods</H1><H3 class="sectionedit2">Theoretical cloning</H3><DIV class="level3"><P>
All our constructs were theoretically cloned before we ordered primers and started with the wet-lab work. For this we used Geneious (Firma???). Detailed cloning strategies can be found next to the respective plasmid maps (?).
</P><P>
For classical cloning approaches primers were designed with an annealing temperature of approximately 60°C to facilitate polymerase chain reactions. Important restriction sites or linker regions were included via primer overhangs. Using the tools provided by Geneious the cloning strategy was tested extensively to discover potential mistakes or other occurring problems.
</P><P>
For Gibson cloning we decided to stay consistent with our classical cloning strategy and designed our primers with an annealing temperature of approximately 60°C as well. Attached to the binding region a 32 basepair overlap was added. Potential linkers or additional restriction sites were introduced between the binding region and the overlap. 
All Gibson constructs were theoretically cloned with Geneious to avoid mistakes and improve the wet-lab work.
</P></DIV></DIV><DIV class="content_box"><H3 class="sectionedit3">Primer Design</H3><DIV class="level3"><UL><LI class="level1"><DIV class="li"><EM class="u">DNA amplification:</EM> All primers used for amplification of DNA from a template were designed with Geneious software. The binding site of each primer is indicated to have a melting temperature of about 60°C. Possible extentions have not been included for melting temperature calculations. When an extention was needed to produce further recognition sites for restriction enzymes or to add overlaps compatible for Gibson Assembly, they were allways fused to the 5' end of the respective primer.</DIV></LI><LI class="level1"><DIV class="li"><EM class="u">Mutagenesis:</EM> Primers that were created for site-directed mutagenesis were also designed with Geneious software. Such primers were supposed to have length of about 50 bp and the site which has to be mutated should be located in the middle of the sequence to make sure the primer binds to the template over its full length.</DIV></LI><LI class="level1"><DIV class="li"><EM class="u">Sequencing:</EM> Sequences for primers designed for sequencing purposes only were obtained by the GATC tool for custom primers. </DIV></LI></UL><P>
All primers were synthesized by Sigma Aldrich.
</P></DIV></DIV><DIV class="content_box"><H3 class="sectionedit4">PCR</H3><DIV class="level3"><P>
For amplification of different DNA fragments from plasmid templates, various PCR approaches were used. In general, PCRs were performed with Phusion High Fidelity DNA Polymerase (NEB). The components of the reaction mixture were used in defined amounts/concentrations recommended by the manufacturer:
</P><DIV class="table sectionedit5"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0"> Component </TH><TH class="col1"> Concentration/amount </TH></TR><TR class="row1"><TD class="col0"> Phusion polymerase </TD><TD class="col1"> 1 U </TD></TR><TR class="row2"><TD class="col0"> Phusion polymerase buffer </TD><TD class="col1"> 1X </TD></TR><TR class="row3"><TD class="col0"> Forward primer </TD><TD class="col1"> 0.5 µM </TD></TR><TR class="row4"><TD class="col0"> Reverse primer </TD><TD class="col1"> 0.5 µM </TD></TR><TR class="row5"><TD class="col0"> dNTPs </TD><TD class="col1"> 200 µM </TD></TR><TR class="row6"><TD class="col0"> DMSO </TD><TD class="col1"> 3% </TD></TR><TR class="row7"><TD class="col0"> Template DNA </TD><TD class="col1"> 1 ng </TD></TR><TR class="row8"><TD class="col0"> ddH2O </TD><TD class="col1"> up to 50 µl </TD></TR></TBODY></TABLE></DIV><P>
Primers for DNA amplification were designed to have a melting temperature of about 60°C (detailed information can be found in the <EM>Oligo list</EM>). The following cycling conditions have been used for standard PCRs:
</P><DIV class="table sectionedit6"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0"> Step </TH><TH class="col1"> Temperature [°C] </TH><TH class="col2"> Duration [s] </TH></TR><TR class="row1"><TD class="col0"> Initial denaturation </TD><TD class="col1"> 98 </TD><TD class="col2"> 30 </TD></TR><TR class="row2"><TD class="col0"> Denaturation </TD><TD class="col1"> 98 </TD><TD class="col2"> 20 </TD></TR><TR class="row3"><TD class="col0"> Annealing </TD><TD class="col1"> 60 </TD><TD class="col2"> 20 </TD></TR><TR class="row4"><TD class="col0"> Extention </TD><TD class="col1"> 72 </TD><TD class="col2"> 30 s/kb* </TD></TR><TR class="row5"><TD class="col0"> Final extention </TD><TD class="col1"> 72 </TD><TD class="col2"> 300 </TD></TR><TR class="row6"><TD class="col0"> Cooling </TD><TD class="col1"> 4 </TD><TD class="col2"> infinite </TD></TR></TBODY></TABLE></DIV><P>
* Extention times were calculated  according to NEB's recommendations for Phusion High-Fidelity Polymerase. </P><P>
Upon unsuccessful amplification, variations of the protocol have been performed. Those inculde a temperature gradient from 55°C to 65°C in the annealing step and the use of Q5 Polymerase (NEB).
</P></DIV></DIV><DIV class="content_box"><H3 class="sectionedit7">Agarose Gel Electrophoresis</H3><DIV class="level3"><P>
The analysis of PCR products as well as enzymatic DNA digests was routinely performed via gel electrophoresis using 1 % agarose (w/v) in 1x TAE buffer. Before polymerization of the gel 4-8 µl MidoriGreen Advance (Biozym) were added. Commonly, a voltage between 80 and 120 V was applied to the gel for 20 to 45 minutes before analysis on a UV scanner.
</P></DIV><H3 class="sectionedit8">Gel Extraction</H3><DIV class="level3"><P>
To extract DNA fragments obtained by PCR or enzymatic digest from agarose gels, the QIAquick Gel extraction Kit from Qiagen was used according to the manufacturers instructions. Before elution of the DNA from the column an additional incubation step of 10 min (shaking with 300 rpm at 40°C) was added. DNA concentrations were measured by photospectrometry (NanoDrop). 
</P></DIV><H3 class="sectionedit9">Restriction digest</H3><DIV class="level3"><P>
Enzymatic digests of DNA were performed for two purposes. The first is to verfiy the successful ligation of DNA fragments into a certain backbone. The second is to obtain those fragments by excision from another vector. To be able to ligate fragments together, they need to be digested with enzymes producing the same compatible ends. In general, restriction digests were performed with ~ 1 µg of DNA in a total reaction volume of 50 µl. Therefore, 1 µl of each enzyme were added as well as 5 µl of the the corresponding 10 X buffer and the appropriate amount of dH2O to reach a final volume of 50 µl.
The utilized enzymes were provided by NEB and Fermentas. Reactions with NEB enzymes were incubated at 37°C for 1 h, while reactions with Fermentas FastDigest enzymes were performed for 30 min at 37°C. Afterwards, the samples were analyzed by agarose gel electrophoresis.
</P></DIV><H3 class="sectionedit10">Ligation</H3><DIV class="level3"><P>
To ligate a certain DNA fragment into a vector which had been opened by enzymatic digest with the same restriction enzymes, 50 ng of the backbone DNA was used. An 8-fold molar amount of the insert was added as well as 1 µl T4 DNA ligase and 2 µl T4 DNA ligase buffer (NEB). dH2O was added up to a final volume of 20 µl. The reaction was incubated either at RT for 1 h or at 16°C overnight. Subsequently, 5 µl of the ligation product were transformed into chemically competent <EM>E. coli</EM> TOP10. 
</P></DIV></DIV><DIV class="content_box"><H3 class="sectionedit11">Gibson Assembly</H3><DIV class="level3"><P>
Besides classical cloning methods, we used Gibson Assembly to obtain desired plasmids. The operating mode of this method is devided into three major parts:
</P><OL><LI class="level1"><DIV class="li"> An exonuclease removes bases from the 5' end of each DNA strand.</DIV></LI><LI class="level1"><DIV class="li"> Complementary regions of different DNA strands can anneal and a polymerase fills up the gaps.</DIV></LI><LI class="level1"><DIV class="li"> The fragments a ligated together.</DIV></LI></OL><P>
To enable this these three steps, DNA strands with compatible ends of about 32 bp are needed. Those can either be incorporated by primer overhangs or by gene synthesis. 

A 5 µl mix of the DNA parts that are supposed to be assembled is prepared. Its composition is calculated based on the length and concentration of every single fragment. The insert(s) should at least be contained in a 4 - 8 fold molar amount of the antibiotic resistance containing backbone. 

This DNA mix is then added to a 15 µl aliquot of the Assmbly Master Mix. 10 µl of this mix are then used to transform 25 µl of chemically competent <EM>E. coli</EM> TOP10 cells according to the following protocol.
</P><P><STRONG>Preparation of the Assembly Master Mix:</STRONG></P><DIV class="table sectionedit12"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0"> Ingredient </TH><TH class="col1"> Concentration of the stock solution </TH><TH class="col2"> Volume for 80 aliquots [µl] </TH><TH class="col3"> Volume for 25 aliquots [µl] </TH></TR><TR class="row1"><TD class="col0"><ACRONYM title="International Organization for Standardization">ISO</ACRONYM> buffer </TD><TD class="col1"> 5X </TD><TD class="col2"> 320 </TD><TD class="col3"> 100 </TD></TR><TR class="row2"><TD class="col0"> T5 exonuclease </TD><TD class="col1"> 10 U/µl </TD><TD class="col2"> 0.64 </TD><TD class="col3"> 0.2 </TD></TR><TR class="row3"><TD class="col0"> Phusion polymerase </TD><TD class="col1"> 2 U/µl </TD><TD class="col2"> 20 </TD><TD class="col3"> 6.25 </TD></TR><TR class="row4"><TD class="col0"> Taq DNA Ligase </TD><TD class="col1"> 40 U/µl </TD><TD class="col2"> 160 </TD><TD class="col3"> 50 </TD></TR><TR class="row5"><TD class="col0"> dH2O </TD><TD class="col1"> - </TD><TD class="col2"> ad 1.2 ml </TD><TD class="col3"> ad 375 µl </TD></TR></TBODY></TABLE></DIV><P>
Aliquots of 15 µl are stored at -20°C.
</P><P><STRONG>Preparation of 5x ISOthermal buffer:</STRONG></P><DIV class="table sectionedit13"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0"> Ingredient </TH><TH class="col1"> Concentration of the stock solution </TH><TH class="col2"> Amount </TH></TR><TR class="row1"><TD class="col0"> Tris-HCL </TD><TD class="col1"> 3 M </TD><TD class="col2"> 3 ml </TD></TR><TR class="row2"><TD class="col0"> MgCl2 </TD><TD class="col1"> 2 M </TD><TD class="col2"> 150 µl </TD></TR><TR class="row3"><TD class="col0"> dNTP </TD><TD class="col1"> 100 mM of each </TD><TD class="col2"> 60 µl </TD></TR><TR class="row4"><TD class="col0"> DTT </TD><TD class="col1"> 1 M </TD><TD class="col2"> 300 µl </TD></TR><TR class="row5"><TD class="col0"> PEG-8000 </TD><TD class="col1"> - </TD><TD class="col2"> 1.5 g </TD></TR><TR class="row6"><TD class="col0"> NAD+ </TD><TD class="col1"> 100 mM </TD><TD class="col2"> 300 µl </TD></TR></TBODY></TABLE></DIV><P>
Aliquots of 320 µl are stored at -20°C.
</P></DIV></DIV><DIV class="content_box"><H3 class="sectionedit14">Transformation</H3><DIV class="level3"><P>
Chemically competent <EM>E. coli</EM> cells were transformed with a certain vector by heat shock. The cells were thawed on ice and the appropriate amount of DNA was added. After 10 min of incubation on ice, the cells were heat shocked for 45 s at 42°C. They were chilled on ice for 2 min, before 500 µl LB medium without antibiotics were added. Following 1 h incubation at 37°C (500 rpm), the cells were plated on LB medium containing the appropriate anibiotic(s) and grown at 37°C over night. 
</P></DIV><H3 class="sectionedit15">Growing conditions</H3><DIV class="level3"><P><EM>E.coli</EM> cells were grown in LB medium (10 g/l NaCl, 10 g/l Trypton, 5 g/l yeast extract) at 37°C, shaking with 200 rpm. For inoculation of liquid cultures, a single colony was taken with a sterile pipette tip which was then <STRONG>pitched/thrown/put</STRONG> into a test tube with 5 ml liquid LB. Appropriate antibiotics were added (see table below).
For long-term storage 1 ml of an <EM>E.coli</EM> o/n culture was mixed with 300 µl  50% glycerol, immediately frozen in liquid nitrogen and stored at -80°C.
</P><DIV class="table sectionedit16"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0 leftalign"> Additive  </TH><TH class="col1 centeralign">  Concentration [µg/ml]  </TH><TH class="col2 centeralign">  Storage  </TH></TR><TR class="row1"><TD class="col0">kanamycin</TD><TD class="col1">50</TD><TD class="col2">-20°C (H2O)</TD></TR><TR class="row2"><TD class="col0">ampicillin</TD><TD class="col1">100</TD><TD class="col2">-20°C (H2O)</TD></TR><TR class="row3"><TD class="col0">chloramphenicol</TD><TD class="col1">30</TD><TD class="col2">-20°C (Ethanol)</TD></TR><TR class="row4"><TD class="col0">gentamycin</TD><TD class="col1">25</TD><TD class="col2">-20°C (H2O)</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="content_box"><H3 class="sectionedit17">Competent cells</H3><DIV class="level3"><P>
To prepare competent <EM>E.coli</EM> cells, <EM>E.coli</EM> cells were streaked out on LB plates containing appropriate antibiotics. The plates were incubated at 37°C over night and then 5 ml LB medium with antibiotic was inoculated with a single colony. After o/n incubation at 37°, 100 ml LB medium without antibiotic was inoculated with 1 ml of the o/n culture. Cells were grown for 2-3 hours until they reached an OD(600) of 0.4 to 0.6.
The <EM>E.coli</EM> cells were then transferred into a pre-cooled, sterile centrifuge tube (250 ml) and pelleted for 5 minutes (5000 rpm, 4°C). After discarding the supernatant the cells were resuspended in 20 ml of ice cold TFB1 with cut tips, followed by 5 minutes of incubation on ice. Then, the resuspended cells were again centrifuged (5 min, 5000 rpm, 4°C) and afterwards resuspended in 4 ml ice-cold TFB2 with cut tips. 
The resuspension of <EM>E.coli</EM> cells was divided in 25 µl or 50 µl aliquots that were immediately frozen in liquid nitrogen and stored at -80°C.
</P><P>
Preparation of buffers:<STRONG>TBF1</STRONG>:
</P><DIV class="table sectionedit18"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0"> Chemical </TH><TH class="col1"> Final concentration </TH><TH class="col2"> Amount needed for 200 ml </TH></TR><TR class="row1"><TD class="col0"> KAc </TD><TD class="col1"> 30 mM </TD><TD class="col2"> 0.589 g </TD></TR><TR class="row2"><TD class="col0"> MnCl2 </TD><TD class="col1"> 50 mM </TD><TD class="col2"> 1.6919 g </TD></TR><TR class="row3"><TD class="col0"> KCl </TD><TD class="col1"> 100 mM </TD><TD class="col2"> 1.482 g </TD></TR><TR class="row4"><TD class="col0"> CaCl2 </TD><TD class="col1"> 10 mM </TD><TD class="col2"> 0.294 g </TD></TR><TR class="row5"><TD class="col0"> Glycerol </TD><TD class="col1"> 15 % </TD><TD class="col2"> 30 g </TD></TR></TBODY></TABLE></DIV><UL><LI class="level1"><DIV class="li"> sterile filtration and storage at 4°C</DIV></LI></UL><P><STRONG>TBF2</STRONG>:
</P><DIV class="table sectionedit19"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0"> Chemical </TH><TH class="col1"> Final concentration </TH><TH class="col2"> Amount needed for 200 ml</TH></TR><TR class="row1"><TD class="col0"> MOPS </TD><TD class="col1"> 10 mM </TD><TD class="col2"> 0.419 g </TD></TR><TR class="row2"><TD class="col0"> KCl </TD><TD class="col1"> 10 mM </TD><TD class="col2"> 0.149 g </TD></TR><TR class="row3"><TD class="col0"> CaCl2 </TD><TD class="col1"> 75 mM </TD><TD class="col2"> 2.205 g </TD></TR><TR class="row4"><TD class="col0">Glycerol </TD><TD class="col1"> 15 % </TD><TD class="col2"> 30 g </TD></TR></TBODY></TABLE></DIV><UL><LI class="level1"><DIV class="li"> autoclaving and storage at 4°C</DIV></LI></UL></DIV></DIV><DIV class="content_box"><H3 class="sectionedit20">Plasmid isolation</H3><DIV class="level3"><P>
For isolation of plasmid DNA from <EM>E.coli</EM> cells plasmid isolation kits from Peqlab(???), Biozym (???), Qiagen (???) and Zymo Research (???) were used. Plasmid islolation was done according to manufacturers information.
</P></DIV><H3 class="sectionedit21">Oligo-annealing</H3><DIV class="level3"><P>
Primers for oligo annealing were designed using Geneious. Primers were diluted according to information from Sigma-Aldrich. From this dilution a 5 mM and a 2.5 mM primer solution was prepared. Primer dilutions were incubated at 95°C for 5 minutes, followed by a slow cool-down phase for approximately one hour.
</P></DIV><H3 class="sectionedit22">Blunt-end ligation</H3><DIV class="level3"><P>
For blunt-end ligation of PCR fragments into pJet the following components were added to a reaction mix:
</P><DIV class="table sectionedit23"><TABLE class="inline"><TBODY><TR class="row0"><TD class="col0"> Volume [µl] </TD><TD class="col1"> Ingredient </TD></TR><TR class="row1"><TD class="col0"> 1 </TD><TD class="col1"> ligation fragment (10 ng/µl)</TD></TR><TR class="row2"><TD class="col0"> 0.5 </TD><TD class="col1"> pJet </TD></TR><TR class="row3"><TD class="col0"> 1 </TD><TD class="col1"> T4 ligase</TD></TR><TR class="row4"><TD class="col0"> 5 </TD><TD class="col1"> 2x reaction buffer</TD></TR><TR class="row5"><TD class="col0"> 2.5 </TD><TD class="col1"> H2O </TD></TR></TBODY></TABLE></DIV><P>
The ligation mix was incubated at room temperature for 5 minutes. Approximately 5 µl of the ligation was used to transform 25 µl of chemically competent <EM>E.coli</EM> T10. Transformation was done according to the protocol delivered with the respective kit. The remaining ligation mix was stored at -20°C as backup.
</P></DIV><DIV class="tags"><SPAN><A class="wikilink1" href="/igem2015/doku.php?id=tag:info&amp;do=showtag&amp;tag=info" rel="tag" title="tag:info">info</A></SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>