<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Results_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Results/Modeling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>&quot;&quot;</TITLE></P><DIV id="header"><DIV id="naviconwrapper"><DIV id="naviconbar"><DIV id="navicons"><UL><LI id="home"><A href="https://2015.igem.org/Team:Freiburg/Home">Home</A></LI><LI id="project"><A href="https://2015.igem.org/Team:Freiburg/Project/Overview">Project</A></LI><LI id="team"><A href="https://2015.igem.org/Team:Freiburg/Team">Team</A></LI><LI id="results"><A href="https://2015.igem.org/Team:Freiburg/Results">Results</A></LI><LI id="policy" style="font-size:11px"><A href="https://2015.igem.org/Team:Freiburg/Practices">Human Practice</A></LI><LI id="notebook"><A href="https://2015.igem.org/Team:Freiburg/Notebook">Notebook</A></LI></UL></DIV></DIV></DIV><DIV id="navtextwrapper"><DIV id="navtext"><UL id="project-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/Overview">Introduction</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/System">Overview</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Design">Diagnostics today</A></LI><LI><A class="navsub1 subsubmenu" href="#">The DiaCHIP</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/DNA_Engineering">DNA Engineering</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Cellfree_Expression">Cell-Free Expression</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Protein_Purification">Protein Purification</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Surface_Chemistry">Surface Chemistry</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/iRIf">Optical Detection</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/Future_Directions">Outlook</A></LI><UL id="team-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Team">Members</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Attributions">Acknowledgements</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Partners">Partners</A></LI></UL><UL id="results-sub"><LI><A class="navsub1 subsubmenu" href="#">The DiaCHIP</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results">Main Results</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Diagnostics">Diagnostics</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Own_Device">Our Device</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Immobilization">DNA Array</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Cellfree">Cell-Free Expression</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Surface">Binding on Surface</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Modeling">Modeling</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Parts">BioBricks</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Collaborations">Collaborations</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/InterLab_Study">Interlab Study</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Achievements">Achievements</A></LI><UL id="policy-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Science_Fair">Science Fair</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Theater_Project">Theater Project</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Public_Opinion">Public Opinion</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/College">College Talk</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Ethics">Ethics</A></LI></UL><UL id="notebook-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Labjournals">Labjournals</A></LI><LI><A class="navsub1 subsubmenu" href="#">Techniques</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Classic_vs_Gibson">Gibson</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Coli_Strains">E.coli Strains</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Protocols">Protocols</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Materials">Materials</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Wikimigration">Wiki-Tricks</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Glossary">Glossary</A></LI></DIV></DIV></DIV><DIV class="clearfix" id="smallnav"><UL class="clearfix"><LI id="home_small"><A href="https://2015.igem.org/Team:Freiburg/Home">Home</A></LI><LI id="project_small"><A href="https://2015.igem.org/Team:Freiburg/Project/Overview">Project</A></LI><LI id="results_small"><A href="https://2015.igem.org/Team:Freiburg/Results">Results</A></LI><LI id="team_small"><A href="https://2015.igem.org/Team:Freiburg/Team">Team</A></LI><LI id="policy_small"><A href="https://2015.igem.org/Team:Freiburg/Practices">Practice</A></LI><LI id="notebook_small"><A href="https://2015.igem.org/Team:Freiburg/Notebook">Notebook</A></LI></UL><A href="#" id="pull">Menu</A></DIV><DIV class="content_background_wrapper"><DIV class="central_position_wrap"><DIV id="bubble_wrapper"><DIV id="bubble_sidebar" class="bubble2"><P><A id="bubble2" href="#firstHeading">text</A></P></DIV><DIV id="bubble_sidebar" class="bubble3"><P><A id="bubble3" href="">next</A></P></DIV></DIV><DIV class="page_content"><DIV class="content_box"><H1> Modeling</H1><H2> Life as Complex Biochemical Network </H2><P>
Nowadays, more and more information on biochemical networks is collected, and a lot is already known. Understanding biochemical networks is in general important for understanding signaling pathways within organisms, especially regarding medical health issues. Furthermore, it is needed for building new networks from scratch using synthetic biology.

However, the more components a network involves, the harder it gets to estimate how a complex network reacts to changes from both the inside and the outside. This is especially a problem, if entities change in both time and space. Experimentally studying all different components of a network for different possible conditions often is not only time-consumptive but also very expensive and prove often to be impossible at the desired level. 
Mathematical modeling of biological networks therefore is a powerful tool to support experimental research, predicting the behavior of networks concerning both time and space. These predictions can then be validated experimentally.
</P><P>
Two crucial biological processes within the DiaCHIP are cell-free expression and the diffusion of proteins to the specific chemical surface. Cell-free expression limits the DiaCHIP concerning amount and production rate. Protein diffusion limits the system regarding space thereby determining the maximum protein spot density one can still resolve. Therefore, we decided to model both cell-free expression and protein diffusion to predict the behavior of protein concentration in time and space. 
</P><P>
With a model like this, it is possible to find the minimum time needed to get a sufficiently high concentration of protein for antibody detection. This reduces the time needed for diagnosis by the DiaCHIP. Furthermore, not only protein, but also other molecules' concentrations can be simulated. Therefore, bottlenecks within the biochemical system as well as limiting substrates can be predicted. Specifically increasing the concentration of such limiting substrates as well as substances involved in limiting processes may not only increase the overall protein yield but also speeds up synthesis.
Moreover, a cell-free expression model has huge potential for application on all systems dealing with protein expression in general. Although designed for cell-free systems, it represents the central dogma of biology and therefore mathematically describes a lot of processes involved in many biochemical networks.
</P><H2>Overview</H2><P>
Our system consists of the three biochemical and biophysical processes cell-free transcription, cell-free translation and diffusion of the protein to the opposite surface, as shown in figure 1. With our model we aim to simulate cell-free expression to predict the amount of protein synthesized in the course of time. We set up a system of ordinary differential equations (ODE System) based on the law of mass action, describing the kinetics of the single processes occurring during transcription and translation. Starting from a detailed description including <STRONG>95</STRONG> different entities and 100 different parameters, we defined additional assumptions to simplify our system to a total of 65 entities and 44 parameters. Kinetic constants and all other parameter values needed were researched in literature. If no values were found, appropriate values were suggested. As the final model can not be solved analytically, it was solved numerically using Mathematica.
</P><DIV class="thumb2 trien" style="width:900px"><DIV class="thumbinner"><DIV class="thumbcaption">
                       		Figure 1: Overview of the DiaCHIP as biochemical system. 
                   		 </DIV></DIV></DIV><H1 style="text-align:left"> Model Description </H1><H3> 1. Transcription</H3><P>
Transcription describes the processes of mRNA synthesis from a DNA template, the first part of the central dogma of biology, as shown in figure 2. Unspecific binding is also displayed here. The detailed system inherits <STRONG>33</STRONG> different entities described by their corresponding ODEs and <STRONG>31</STRONG> parameters. Using a set of assumptions, the system was simplified to <STRONG>14</STRONG> entities and  <STRONG>10</STRONG> parameters.
</P><H4> Detailed System </H4><DIV class="image_box center"><DIV class="thumb2 trien" style="width:900px"><DIV class="thumbinner"><DIV class="thumbcaption">
                       		Figure 2: Reactions and processes inherited in the biochemical network of transcription. Unspecific binding is displayed. The nomenclature of the single entities' and parameters' abbreviations shown is given in the section 'nomenclature'.
                   		 </DIV></DIV></DIV></DIV><DIV class="accordion" style="margin-top:50px"><DIV class="accordion-section"><A class="accordion-section-title" href="#accordion-nomenclature_transc">Nomenclature (Transcription)</A></DIV></DIV><DIV class="accordion" style="margin-top:100px"><DIV class="accordion-section"><A class="accordion-section-title" href="#accordion-1">ODE System Transcription (Detailed)</A></DIV></DIV><H4> Simplified System </H4><DIV class="accordion"><DIV class="accordion-section"><A class="accordion-section-title" href="#accordion-2">ODE System Transcription (Simplified)</A></DIV></DIV><H3>2. Translation</H3><P>
Translation describes the processes of protein synthesis from an mRNA template, the second part of the central dogma of biology, as shown in figure 3. Unspecific binding is neither displayed in the figure nor included into the mathematical description due to the sheer amount of entities and parameters inherited. The detailed system consists of <STRONG>62</STRONG> different entities described by their corresponding ODEs and <STRONG>69</STRONG> parameters. Using a set of assumptions, the system was simplified to <STRONG>51</STRONG> entities and  <STRONG>34</STRONG> parameters. 
</P><H4> Detailed System </H4><DIV class="image_box center"><DIV class="thumb2 trien" style="width:900px"><DIV class="thumbinner"><DIV class="thumbcaption">
               		Figure 3: Reactions and processes inherited in the biochemical network of translation. Unspecific binding is not displayed. The nomenclature of the single entities' and parameters' abbreviations shown is given in the section 'nomenclature'.
           		 </DIV></DIV></DIV></DIV><DIV class="accordion" style="margin-top:50px"><DIV class="accordion-section"><A class="accordion-section-title" href="#accordion-nomenclature_translat">Nomenclature (Translation)</A></DIV></DIV><DIV class="accordion" style="margin-top:100px"><DIV class="accordion-section"><A class="accordion-section-title" href="#accordion-3">ODE System Translation (Detailed)</A></DIV></DIV><H4> Simplified System </H4><DIV class="accordion"><DIV class="accordion-section"><A class="accordion-section-title" href="#accordion-4">ODE System Translation (Simplified)</A></DIV></DIV><H2>3. Diffusion</H2><H3>3.1 Introduction and Motivation</H3><P>
In the final step of cellfree expression proteins being produced are diffusing inside the microfluidic chamber. We modeled an ideal case to provide a tool:

On the PDMS slide spots of immobilized DNA produces proteins with a steadily decreasing production
rate. The product is distributed homogeneously on the spot and starts diffusing freely in the cell-free
mix. Furthermore besides convection through gravitation any interaction is assumed to be negligibly small. The coated iRIf glass is expected to be an ideal sink. Any proteins reaching the slide are bound and therefore do not contribute to diffusion anymore.
</P><P>
	What knowledge did we want to gain by the diffusion?
</P><UL><LI>Time optimization: When is the most efficient time to stop the expression?</LI><LI>Product optimization: How much of the totally produced proteins does bind to the surface?</LI><LI>Spot distance optimization: How is the bound protein distributed on the glass slide?</LI></UL>

In order to achieve this we constructed the following system.
<H3>3.2 Model System</H3><P>
Effective modeling depends on wisely chosen assumptions and a minimally complex system. The main assumptions we considered can be seen in chapter 3.3 and will be referenced by A1,A2, ...

Assume the following setup: A microfluidic system is given consisting of a spotted PDMS slide (multiple spots) and the chemically treated iRIf slide. Yet instead of observing this system of multiple spots we reduce our problem to a cylindrical area around one spot. This area has a radius of L and still the same width W as the whole system. Ideally the spot is a perfect circle which allows us to reduce the problem to a twodimensional one: No matter from which angle the cylinder is observed the identical behaviour is seen. Therefore a crosssection through it provides enough information.
</P><DIV class="image_box right"><DIV class="thumb2 trien" style="width:376px"><DIV class="thumbinner"><DIV class="thumbcaption"><P><STRONG>Figure: Illustrated above is a crosssection of the system around one spot (A1). Due to assumption A2 the number of geometrical degrees of freedom can be reduced to 2. </STRONG></P></DIV></DIV></DIV></DIV><P>
While the expression is done the system lies as in the picture shown above, parallel to the ground. Therefore we included in our model system possible movements due to gravitation. The analytical equation describing this process is the diffusion convection equation:
</P><P>
It is an inhomogeneous parabolic partial differential equation (PDE). The diffusion constant D depends on the media and materials involved in the system, the inhomogenity describes sources of the system. The velocity vector v is assumed to be a possible sedimentation velocity (A4) and therefore a constant. The initial and boundary conditions are the following: The PDMS slide as well as the iRIf slide are closed surfaces while the left and right edges of the system are not. On the PDMS slide proteins are produced by a logistic function (A5). The solution to this system is only numerically solvable. Therefore it has to be prepared mathematically by discretizing the whole setup. Finally we used the Crank-Nicholson approach on a finite-differences-method to solve this system.
</P><H3>3.3 Numerical approach</H3><P>
Let us start by describing the crosssection in terms of mathematics: It is a subset M of the set of twodimensional real vectors. Of this subset we define two more subsets S and B describing the DNA spot and the glass slide respectively.
</P><P>
Two time δ is the spot width, its thickness is ε. The thickness of the binding area is ξ yet we choose them to be the same (A6). Next we have to simplify our PDE: It consists of three terms on the right side which can be (in our case) shortened to the following components:
</P><P>
The next step is the change from a continuous to a discrete system: First we describe our concentration function as one depending on three integers (places x and z, time t):
</P><P>
Also, instead of handling every component individually we define a concentration vector containing all concentration values of M at a static time step:
</P><P>
We do the same for our source by defining a production vector at the time t:
</P><P>
Why is this done? Because we can split our grid into a connected line of rows. The first row of the grid then would be given by the first components of the vector, followed by the second row and so on.
</P><P>
Next we discretize the derivations of the PDE by exchanging the differentials by finite differences. In this case differentials can be seen as the rate of change in one (x-,z- or) t-step (secants so to say). For a first and second derivation this would be written as:
</P><P>
Using the integer multiplicated step values for our PDE components this results in:
</P><P>
Logically this method is called &quot;finite differences method&quot;. For conveniences sake two parameters λ and μ are defined and used in the upcoming simplifications:
</P><P>
There are different approaches with the finite differences method to approximately solve the PDE. We decided to use an ansatz by Crank and Nicholson which has the following form: A functions change in one time step is determined by the average of momentary and following time step of the right hand side of the PDE. Or, in mathematical terms:
</P><P>
Combining the PDE parts and splitting the concentration values into following and momentary time steps the problem can be solved by solving a linear system of coupled equations in place components of the concentration:
</P><P>
This is an iterative method: The values of the upcoming time step are calculated by the ones from the former time step. The equations cannot be solved independently from one another. In a grid of NW rows and NL columns this a system of NW times NL equations that have to be solved every step.
</P><P>

What can be seen in the scheme above is the underlying geometrical approach: Diffusion and convection are done only with direct grid neighbours (two vertical and two horizontal ones). Logically this holds true only for non-border grid points. For the systems borders our boundary conditions have to be considered which manifest in the following conditions:
</P><P>
Inserting those in the main Crank-Nicholson-scheme leads to the border descriptions:
</P><P>
Combining two border conditions leads to corner conditions which have to be considered too. They follow the same calculation yielding:
</P><P>
Being a linear system of equations this problem can be replaced by a short form using matrix-vector multiplication (hence the definition of the vectors from before):
</P><P>
Where the two matrices A and B consist of the factors accompanying the concentration values. They are rather large and are sketched below:
</P><P>
This concludes the theoretical part of the diffusion. The next step was to implement this system using python 3.4.2 including numpy and matplotlib. Source codes can be obtained here:
</P><A href="https://static.igem.org/mediawiki/2015/9/94/Freiburg_2D_Diffusion_Convection.zip"> Python source code</A><P>
The first file produces two output files. One of them contains the protein distribution on the glass slide at the finishing time. The other one consists of the total bound protein at the surface of the slide. The second file returns pictures of the ongoing diffusion giving an insight of the diffusion process in the crosssection system.
</P><H3>3.4 Assumptions</H3><P>
A1: The spots are equally distributed and distance to the borders of the microfluidic chamber is at least one
spot-to-spot distance.
</P><P>
A2: The spots are perfect circles and the binding site at the iRIf glass is homogeneous -&gt; Around one spot
cylindrical symmetry is given.
</P><P>
A3: The area of produced proteins is as thick as the one of bound ones -&gt; χ = ε</P><P>
A4: Movement by virtue of gravitation is considered to already be in a stationary status (constant movement).
</P><P>
A5: Protein production is described by a logistic function with a minimal value close to zero and maximum value of 100 (percent).
</P><P>
A6: ε = ξ = Δ x = Δ z
</P><H1 style="text-align:left"> Results </H1><H2> 1. Transcription and Translation </H2><P>
We wanted to test our model's ability to predict the concentration of the cell-free expression biochemical network entities and to adapt it to different conditions. Given these properties, our model has the potential to reveal limiting resources and bottleneck reactions and to test optimized parameter settings for the resulting effects on the biochemical network behavior.
</P><P>
We run the model using default starting concentrations and parameter values as given in the following, tracking the concentration time course of free RNA polymerase, free 30s and 50s ribosomal subunits, free 70s ribosome, free amino acids, free tRNA, free initiation factors, free elongation factors, free nucleotides, free mRNA and free protein, to examine the network's predicted behavior. Afterwards, we tracked the protein and mRNA concentrations in three more simulations, while changing one starting concentration each, to examine the network's predicted reaction and compare it to the reaction expected.
</P><DIV class="accordion" style="margin-top:50px"><DIV class="accordion-section"><A class="accordion-section-title" href="#accordion-values">Default Parameter Values</A></DIV></DIV><H4 style="margin-top:50px"> Network Behaviour Under Default Parameter Values </H4><P>
Protein expression occurs between 220 s and 230 s and reaches a maximal protein concentration of about 2.8 µM. Interestingly, free mRNA concentration remains close to zero but reaches a short peak between 230 s and 240 s. The peak correlates with the saturation of protein concentration, denoting the end of protein expression. This can be explained by an increased release of mRNA molecules from translation complexes at the end of expression, which are still bound by evolving initiation complexes that can not be further processed and therefore decrease strongly again.
</P><DIV class="flexbox"><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: Timecourse of Protein Expression </STRONG> ...

			</DIV></DIV></DIV><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: Timecourse of Free RNAP Concentration </STRONG> ...

			</DIV></DIV></DIV><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: Timecourse of Free mRNA concentration </STRONG> ...

			</DIV></DIV></DIV></DIV><P>
The concentration of free RNA polymerase drops down close to zero almost immediately, making it a possible limiting entity and therefore a candidate for optimization.
</P><P>
The time course of free 30s and 70s ribosomal species behaves similar to the RAN polymerase. Already at the beginning droppping down to zero, whereas the 50s ribosomal subunit concentration remains quite high. This points to a faster kinetic of the beginning of translation initiation compared to the end of translation initiation, binding a lot of 30s subunits but leaving the 50s subunits unbound. Again, a peak can be observed for all three ribosomal species at the end of protein expression, caused by an increased dissociation of translation complexes.
</P><P>
The concentration of initiation factors 1 and 3 already drops in the beginning of the reaction, whereas initiation factor 2 concentration drops only during protein expression. This can be explained by the limited mRNA concentration in the beginning of the reaction, which has to bind to the evolving initiation complex before IF-2-GTP can be bound. As IF-2-GTP concentration is not restored at the end of protein expression, this hints to GTP limiting the overall reaction rather than amino acids.
</P><DIV class="flexbox"><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: Timecourse of Free Ribosomal species concentration </STRONG> ...

			</DIV></DIV></DIV><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: Timecourse of Free Initiation Factors Concentration </STRONG> ...

			</DIV></DIV></DIV><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: Timecourse of Free Elongation Factors Concentration </STRONG> ...

			</DIV></DIV></DIV></DIV><P>
As for initiation factor 2, concentrations of all elongation factors remain constant until the beginning of protein expression, due to the limited amount of mRNA. Interestingly, only EF-G-GTP experiences a severe drop during protein expression. This could be due to EF-Tu-GTP regeneration kinetics being faster than EF-G-GTP regeneration. As EF-G-GTP is not restored as it is the case for IF-2-GTP, this is another hint on GTP being a limiting entity.
</P><P>
The time courses of amino acid and tRNA concentration show a slightly decreasing trend, caused by processing of amino acids and tRNAs to aminoacyl-tRNAs. In contrast to the tRNA concentration, the amino acid concentration shows a drop in the very beginning probably due to the binding of amino acids by the aminoacyl-tRNA synthetases. In contrast to the amino acid concentration, tRNA concentration experiences an increase during protein expression between 220 s and 230 s, that can be explained by the release of empty tRNAs during translation.
</P><P>
Looking at the time courses of nucleotide concentrations, it can be observed that the ATP concentration remains almost constant, whereas the GTP concentration drops extremely during protein expression. This drop seems to be responsible for the drop in total nucleotide concentration. Therefore, GTP is a possible limiting entity. 
</P><DIV class="flexbox"><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: Timecourse of Free tRNA and Amino Acid Concentration </STRONG> ...

			</DIV></DIV></DIV><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: Timecourse of Nucleotide Species Concentration </STRONG> ...

			</DIV></DIV></DIV></DIV><H4> Network Reaction To Changed Parameter Values </H4><P>
As observed for time courses under default parameter values, ribosomal species and GTP may be limiting the cell-free expression process. Therefore, mRNA and protein expression under both lower and higher concentrations of those entities was simulated. 
</P><P>
Protein expression is expected to be faster with higher concentration of ribosomes. This is also predicted by the model. Also, mRNA concentration is lower for higher ribosomal species concentration, confirming a faster process. In the course of protein concentrations, the differences are rather small.
</P><P>
Protein expression is expected to gain a higher yield using higher GTP concentrations. This behaviour is clearly predicted also by the model, confirming GTP to be a limiting entity for the cell-free expression described by the model. Both mRNA and protein concentrations are much higher compared to lower GTP concentrations.
</P><DIV class="flexbox"><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: Protein Concentration Timecourse Depending on 70s Concentration </STRONG> ...

			</DIV></DIV></DIV><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: mRNA Concentration Timecourse Depending on 70s Concentration </STRONG> ...

			</DIV></DIV></DIV><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: Protein Concentration Timecourse Depending on GTP Concentration </STRONG> ...

			</DIV></DIV></DIV><DIV class="thumb2 trien" style="width:250px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure: mRNA Concentration Timecourse Depending on GTP Concentration </STRONG> ...

			</DIV></DIV></DIV></DIV><H4> Summary and Model Optimizations </H4><P>
Our model describing cell-free expression, including 65 entites and 44 parameters, showed the ability of both predicting network behavior and network reaction to changed conditions. Although the model's predicted output does not fit to real cell-free expression yet, there is a huge capability of making the model more specific. Using the set of assumptions defined, the model can be extended step by step, bringing it closer to the cell-free expression biochemical network.
</P><H2>2. Diffusion results</H2><P>
First, we made an animation of the diffusion using exemplary values. You can see a video of the diffusion process down below:
</P><DIV class="flexbox"><DIV class="thumbinner" style="width:700px;"><VIDEO poster="https://static.igem.org/mediawiki/2015/c/c4/Freiburg_Diffusion_1_preview.jpg" width="700" height="350" controls=""><SOURCE src="https://static.igem.org/mediawiki/2015/5/5a/Freiburg_Diffusion_1_720p.mp4" type="video/mp4">
          Your browser does not support the HTML5 video tag.
        </SOURCE><DIV class="thumbcaption">Video 1: Demonstration of the diffusion model.   
        </DIV></VIDEO></DIV></DIV><DIV class="image_box right"><DIV class="thumb2 trien" style="width:500px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure:</STRONG> Total bound protein on the crosssection area of the glass slide depending on the time. Plotted for four different spot diameters yet the same produced amount of proteins.
			</DIV></DIV></DIV></DIV><DIV class="image_box right"><DIV class="thumb2 trien" style="width:500px"><DIV class="thumbinner"><DIV class="thumbcaption"><STRONG>Figure:</STRONG> Distribution of bound proteins on the crosssection area of the glass slide. The y axis describes the total bound concentration of proteins at the specific spot on the slide.
			</DIV></DIV></DIV></DIV><P>
We also checked the protein binding for different spot widths: 20%, 40%, 60% and 80% of the total crosssection width. While the binding distribution returned gaussian profiles the total binding - like the production - followed a logistic function. 

The total binding concentration was lower the broader the spot was. This can be explained by diffusion to the left and right borders of the system. In other words the broader the spots (while holding constant distances between the spots) the more &quot;contamination&quot; we have to expect at neighboring spots. Therefore, theory supports the expectation.  The binding distribution showed a similar result: The broader the DNA spot is distributed the broader a gaussian profile of the bound protein concentration (lower μ and higher σ). Here we neglected the fact that there are no infinite binding positions. So at a specific point a plateau would limit the distribution. Also, a broader profile is the result of the higher width. On one hand it is good to have a big enough detection spot. On the other hand being too broad could lead to overlaps with neighboring spots. Due to a lack of time gaining further insight into how much protein concentration is needed was not possible. 
</P></DIV><DIV class="content_box"><DIV class="footnotes"><H3> References </H3><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__1" id="fn__1" name="fn__1">1)</A></SUP>
Lehninger et al., 2005. Principles of Biochemistry, fourth edition
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__2" id="fn__2" name="fn__2">2)</A></SUP>
Metzler, 2003. Biochemistry, Volume 1 and 2, second edition
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__3" id="fn__3" name="fn__3">3)</A></SUP>
Pieter L. deHaseth et al., 1998. J. Bacteriol. RNA Polymerase-Promoter Interactions: the Comings and Goings of RNA Polymerase
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__4" id="fn__4" name="fn__4">4)</A></SUP>
Malcolm Buckle et al., 1999. J. Mol. Biol. The Kinetics of Sigma Subunit Directed Promoter Recognition by E. coli RNA Polymerase
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__5" id="fn__5" name="fn__5">5)</A></SUP>
Steen Pedersen et al., 1978. Cell. Patterns of protein synthesis in E. coli: a catalog of the amount of 140 individual proteins at different growth rates
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__6" id="fn__6" name="fn__6">6)</A></SUP>
J G Howe et al., 1981. The Journal of Biological Chemistry. A sensitive immunoblotting method for measuring protein synthesis initiation factor levels in lysates of Escherichia coli
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__7" id="fn__7" name="fn__7">7)</A></SUP>
Frances M. Adamski et al., 1994. Journal of Molecular Biology. The Concentration of Polypeptide Chain Release Factors 1 and 2 at Different Growth Rates of Escherichia coli
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__8" id="fn__8" name="fn__8">8)</A></SUP>
Irina L. Grigorova et al., 2006. PNAS. Insights into transcriptional regulation and sigma competition from an equilibrium
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__9" id="fn__9" name="fn__9">9)</A></SUP>
Mantu Santra et al., 2012. J. Phys. Chem. Catalysis of tRNA Aminoacylation: Single Turnover to Steady-State Kinetics of tRNA Synthetases
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__10" id="fn__10" name="fn__10">10)</A></SUP>
Pohl Milon et al., 2015. Methods in Enzymology. Transient Kinetics, Fluorescence, and FRET in Studies of Initiation of Translation in Bacteria.
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__11" id="fn__11" name="fn__11">11)</A></SUP>
Pohl Milon et al., 2010. EMBO reports. The ribosome-bound initiation factor 2 recruits initiator tRNA to the 30S initiation complex
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__12" id="fn__12" name="fn__12">12)</A></SUP>
Chandra Sekhar Mandava et al., 2012. Nucleic Acids Research. Bacterial ribosome requires multiple L12 dimers for efficient initiation and elongation of protein synthesis involving IF2 and EF-G
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__13" id="fn__13" name="fn__13">13)</A></SUP>
Tomsic J et al., 2000. The EMBO Journal. Late events of translation initiation in bacteria: a kinetic analysis
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__14" id="fn__14" name="fn__14">14)</A></SUP>
Guillermo Romero et al., 1985. The Journal of Biological Chemistry. Kinetics and Thermodynamics of the Interaction of Elongation Factor Tu with Elongation Factor Ts, Guanine Nucleotides, and AminoacyltRNA
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__15" id="fn__15" name="fn__15">15)</A></SUP>
Kirill B. Gromadski et al., 2002. Biochemistry. Kinetic Mechanism of Elongation Factor Ts-Catalyzed Nucleotide Exchange in Elongation Factor Tu
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__16" id="fn__16" name="fn__16">16)</A></SUP>
Andreas Savelsbergh et al., 2003. Molecular Cell. An Elongation Factor G-Induced Ribosome Rearrangement Precedes tRNA-mRNA Translocation
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__17" id="fn__17" name="fn__17">17)</A></SUP>
Andrei V Zavialov et al., 2002. Molecular Cell. Release of Peptide Promoted by the GGQ Motif of Class 1 Release Factors Regulates the GTPase Activity of RF3
</DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__18" id="fn__18" name="fn__18">18)</A></SUP>
Felicia Y. H. Wu et al., 1976. Biochemistry. Conformational transition of Escherichia coli RNA polymerase induced by the interaction of σ subunit with core enzyme
</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>