<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Project_Coli_Strains skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Project/Coli Strains</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>&quot;&quot;</TITLE></P><DIV id="header"><DIV id="naviconwrapper"><DIV id="naviconbar"><DIV id="navicons"><UL><LI id="home"><A href="https://2015.igem.org/Team:Freiburg/Home">Home</A></LI><LI id="project"><A href="https://2015.igem.org/Team:Freiburg/Project/Overview">Project</A></LI><LI id="team"><A href="https://2015.igem.org/Team:Freiburg/Team">Team</A></LI><LI id="results"><A href="https://2015.igem.org/Team:Freiburg/Results">Results</A></LI><LI id="policy" style="font-size:11px"><A href="https://2015.igem.org/Team:Freiburg/Practices">Human Practice</A></LI><LI id="notebook"><A href="https://2015.igem.org/Team:Freiburg/Notebook">Notebook</A></LI></UL></DIV></DIV></DIV><DIV id="navtextwrapper"><DIV id="navtext"><UL id="project-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/Overview">Introduction</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/System">Overview</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Design">Diagnostics today</A></LI><LI><A class="navsub1 subsubmenu" href="#">The DiaCHIP</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/DNA_Engineering">DNA Engineering</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Cellfree_Expression">Cell-Free Expression</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Protein_Purification">Protein Purification</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Surface_Chemistry">Surface Chemistry</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/iRIf">Optical Detection</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/Future_Directions">Outlook</A></LI><UL id="team-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Team">Members</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Attributions">Acknowledgements</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Partners">Partners</A></LI></UL><UL id="results-sub"><LI><A class="navsub1 subsubmenu" href="#">The DiaCHIP</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results">Main Results</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Diagnostics">Diagnostics</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Own_Device">Our Device</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Immobilization">DNA Array</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Cellfree">Cell-Free Expression</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Surface">Binding on Surface</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Modeling">Modeling</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Parts">BioBricks</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Collaborations">Collaborations</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/InterLab_Study">Interlab Study</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Achievements">Achievements</A></LI><UL id="policy-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Science_Fair">Science Fair</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Theater_Project">Theater Project</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Public_Opinion">Public Opinion</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/College">College Talk</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Ethics">Ethics</A></LI></UL><UL id="notebook-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Labjournals">Labjournals</A></LI><LI><A class="navsub1 subsubmenu" href="#">Techniques</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Classic_vs_Gibson">Gibson</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Coli_Strains">E.coli Strains</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Protocols">Protocols</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Materials">Materials</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Wikimigration">Wiki-Tricks</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Glossary">Glossary</A></LI></DIV></DIV></DIV><DIV class="clearfix" id="smallnav"><UL class="clearfix"><LI id="home_small"><A href="https://2015.igem.org/Team:Freiburg/Home">Home</A></LI><LI id="project_small"><A href="https://2015.igem.org/Team:Freiburg/Project/Overview">Project</A></LI><LI id="results_small"><A href="https://2015.igem.org/Team:Freiburg/Results">Results</A></LI><LI id="team_small"><A href="https://2015.igem.org/Team:Freiburg/Team">Team</A></LI><LI id="policy_small"><A href="https://2015.igem.org/Team:Freiburg/Practices">Practice</A></LI><LI id="notebook_small"><A href="https://2015.igem.org/Team:Freiburg/Notebook">Notebook</A></LI></UL><A href="#" id="pull">Menu</A></DIV><DIV class="content_background_wrapper"><DIV class="central_position_wrap"><DIV id="bubble_wrapper"><DIV id="bubble_sidebar" class="bubble2"><P><A id="bubble2" href="#firstHeading">text</A></P></DIV><DIV id="bubble_sidebar" class="bubble3"><P><A id="bubble3" href="">next</A></P></DIV></DIV><DIV class="page_content"><DIV class="content_box"><H1 class="sectionedit1">E.coli genotypes</H1><DIV class="level1"><P>
We are using <I>E.coli</I> strains for overexpression of our target proteins. <I>E. coli</I> is suitable for expression of a wide range of heterologous proteins. Our aim was the enrichment of active, soluble recombinant protein. Therefore, we used different <I>E.coli</I> strains and adapted different expression temperatures, durations and IPTG concentrations to get as much soluble target protein as possible.

Of course, many challenges can arise during expression of recombinant proteins. That is why most expression strains have a few mutations in common to obtain a high yield of expressed proteins.

Expression strains often harbor mutations that make them deficient in outer membrane protease VII and delete the Ion protease. Both of this reduces proteolysis of the expressed recombinant proteins. Additionally the native restriction/methylation system is often inactivated <SUP><A class="fn_top" href="#fn__1" id="fnt__1" name="fnt__1">1)</A></SUP><SUP><A class="fn_top" href="#fn__2" id="fnt__2" name="fnt__2">2)</A></SUP>. 

Below we introduce the bacterial strains we used for protein overexpression:
</P></DIV><H2 class="sectionedit2">Bl21 (DE3) pLysS</H2><DIV class="level2"><P>
BL21 (DE3) is the basic IPTG-inducible strain containing T7 RNA-polymerase (DE3) for general protein expression <SUP><A class="fn_top" href="#fn__3" id="fnt__3" name="fnt__3">3)</A></SUP>. The <EM>E.coli</EM> expression strain BL21 (DE3) pLysS is useful for the expression of toxic proteins.

The pLysS plasmid contains a chloramphenicol resistance. The pLysS plasmid also encodes T7 phage lysozyme, an inhibitor for T7 polymerase. In the absence of IPTG induction this inhibitor reduces and almost eliminates expression from a transformed T7 promoter containing plasmids .
This inducable expression is suitable to stabilize recombinants encoding particularly toxic proteins <SUP><A class="fn_top" href="#fn__4" id="fnt__4" name="fnt__4">4)</A></SUP><SUP><A class="fn_top" href="#fn__5" id="fnt__5" name="fnt__5">5)</A></SUP>. 
</P></DIV><H2 class="sectionedit3">Rosetta(DE3)pLysS</H2><DIV class="level2"><P>
Rosetta™ host strains are BL21 derivatives designed to enhance the expression of eukaryotic proteins that contain codons rarely used in <I>E. coli</I>. These strains supply tRNAs for AGG, AGA, AUA, CUA, CCC, GGA codons on a compatible chloramphenicol-resistant plasmid. The Rosetta strains provide a better translation for eukaryotic proteins that have not been codon optimized. In Rosetta(DE3)pLysS, the rare tRNA genes are present on the same plasmid that carries the T7 lysozyme gene for inducable expression <SUP><A class="fn_top" href="#fn__6" id="fnt__6" name="fnt__6">6)</A></SUP>.
</P></DIV><H2 class="sectionedit4">C43 (DE3)</H2><DIV class="level2"><P>
This overexpression strain derives from the C41 (DE3) <I>E.coli</I> strain. 
These strains contain an uncharacterized genetic mutation that allows a higher tolerance to toxic proteins. The mutation prevents <I>E.coli</I> cell death often associated with expression of many recombinant toxic proteins.
</P><P>
The advantage of this strain to BL21 (DE3) is that even membrane proteins, often particularly problematic for protein expression, can be overexpressed <SUP><A class="fn_top" href="#fn__7" id="fnt__7" name="fnt__7">7)</A></SUP>. 
</P><P>
Received from the Toolbox - BIOSS (Centre for Biological Signalling Studies).
</P></DIV><H2 class="sectionedit5">ArcticExpress (DE3)RP</H2><DIV class="level2"><P>
ArcticExpress Competent Cells are engineered to address the common bacterial gene expression hurdle of protein insolubility. These cells are derived from the high-performance Stratagene BL21-Gold competent cells, enabling efficient high-level expression of heterologous proteins in <I>E. coli</I>. With these cells we were able to express our target proteins o/n at 10°C. Low-temperature cultivation represents one strategy for increasing the recovery of soluble protein. This is due to the fact that <I>E. coli</I> chaperonins, which facilitate proper protein folding by binding and stabilizing unfolded or partially folded proteins, lose activity at reduced temperatures <SUP><A class="fn_top" href="#fn__8" id="fnt__8" name="fnt__8">8)</A></SUP><SUP><A class="fn_top" href="#fn__9" id="fnt__9" name="fnt__9">9)</A></SUP><SUP><A class="fn_top" href="#fn__10" id="fnt__10" name="fnt__10">10)</A></SUP>.
</P><P>
Received from AG Weber - BIOSS (Centre of Biological Signalling Studies).
</P></DIV><H2 class="sectionedit6">Top10</H2><DIV class="level2"><P>
The Top10 cloning strain harbors some mutations especially useful for high efficiency cloning purposes and for propagation of plasmids.
In this strain, the gene for the type I endonuclease complex KI that recognizes unmethylated DNA is not functional <SUP><A class="fn_top" href="#fn__11" id="fnt__11" name="fnt__11">11)</A></SUP>. So genes from PCR amplification are not degraded by this defense-system and can be efficiently transformed into the bacteria.

Additionally, a mutation in a nonspecific endonuclease (endA) yields cleaner preparations due to less unspecifically cut double-strand DNA <SUP><A class="fn_top" href="#fn__12" id="fnt__12" name="fnt__12">12)</A></SUP>.

Finally the rate of homologous recombination that can hinder efficient cloning is reduced by a mutation in the DNA strand exchange and recombination protein recA<SUP><A class="fn_top" href="#fn__13" id="fnt__13" name="fnt__13">13)</A></SUP>.
</P><DIV class="tags"><SPAN><A class="wikilink1" href="/igem2015/doku.php?id=tag:info&amp;do=showtag&amp;tag=info" rel="tag" title="tag:info">info</A></SPAN></DIV></DIV><DIV class="footnotes"><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__1" id="fn__1" name="fn__1">1)</A></SUP><A href="http://www.genetics.org/content/54/1/61.full.pdf+html;" target="_blank">Demerec M. et al. (1966). A proposal for a uniform nomenclature in bacterial genetics. Genetics. 54(1):61-76</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__2" id="fn__2" name="fn__2">2)</A></SUP><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC210899/pdf/jbacter00181-0229.pdf;" target="_blank">Grodberg, J. and Dunn, J. J. (1988) J Bacteriol 170(3):1245-53</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__3" id="fn__3" name="fn__3">3)</A></SUP><A href="http://www.sciencedirect.com/science/article/pii/0092867487905630" target="_blank">Moffatt BA, Studier FW (1987). T7 lysozyme inhibits transcription by T7 RNA-polymerase. Cell. 49(2):221-7</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__4" id="fn__4" name="fn__4">4)</A></SUP><A href="https://www.agilent.com/cs/library/usermanuals/Public/230191.pdf" target="_blank">Jerpseth, M., Jerpseth, B., Briester, L. and Greener, A. (1998) Strategies 11(1):3-4</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__5" id="fn__5" name="fn__5">5)</A></SUP><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2731896/pdf/gkp498.pdf" target="_blank">Jerpseth, B., Callahan, M. and Greener, A. (1997) Strategies10(2):37–38</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__6" id="fn__6" name="fn__6">6)</A></SUP><A href="http://nar.oxfordjournals.org/content/20/17/4668.full.pdf" target="_blank">Looman et al. (1987) EMBO J. 6, 2489-2492</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__7" id="fn__7" name="fn__7">7)</A></SUP><A href="http://www.sciencedirect.com/science/article/pii/S002228369690399X" target="_blank">B Miroux and J E Walker, ‘Over-Production of Proteins in Escherichia Coli: Mutant Hosts That Allow Synthesis of Some Membrane Proteins and Globular Proteins at High Levels.’, Journal of molecular biology, 260 (1996), 289–98</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__8" id="fn__8" name="fn__8">8)</A></SUP><A href="http://download.springer.com/static/pdf/195/art%253A10.1007%252Fs00253-015-6744-5.pdf?originUrl=http%3A%2F%2Flink.springer.com%2Farticle%2F10.1007%2Fs00253-015-6744-5&amp;token2=exp=1442320220~acl=%2Fstatic%2Fpdf%2F195%2Fart%25253A10.1007%25252Fs00253-015-6744-5.pdf%3ForiginUrl%3Dhttp%253A%252F%252Flink.springer.com%252Farticle%252F10.1007%252Fs00253-015-6744-5*~hmac=662f291fc8a55dcdbd8d5cea91926d4aa442bf111b9b8be2c8f048c364fe8b9d" target="_blank">Carstens, C.-P., Bonnardel, J., Allen, R. and Waesche, A. (2001) Strategies 14:50</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__9" id="fn__9" name="fn__9">9)</A></SUP><A href="http://www.chem-agilent.com/pdf/strata/230193.pdf" target="_blank">Schein, C. H. (1989) Biotechnology7:1141-8</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__10" id="fn__10" name="fn__10">10)</A></SUP><A href="http://www.nature.com/nbt/journal/v21/n11/full/nbt1103-1266.html" target="_blank">Ferrer, M., Chernikova, T. N., Yakimov, M.M., Golyshin, P. N.and Timmis, K. N. (2003) Nat Biotechnol.21(11):1266-7</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__11" id="fn__11" name="fn__11">11)</A></SUP><A class="urlextern" href="http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10459" rel="nofollow" target="_Blank" title="http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10459">http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10459</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__12" id="fn__12" name="fn__12">12)</A></SUP><A class="urlextern" href="http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG11336" rel="nofollow" target="_Blank" title="http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG11336">http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG11336</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__13" id="fn__13" name="fn__13">13)</A></SUP><A class="urlextern" href="http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10823" rel="nofollow" target="_Blank" title="http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10823">http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10823</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>