<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Project_Cellfree_Expression skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Project/Cellfree Expression</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>&quot;&quot;</TITLE></P><DIV id="header"><DIV id="naviconwrapper"><DIV id="naviconbar"><DIV id="navicons"><UL><LI id="home"><A href="https://2015.igem.org/Team:Freiburg/Home">Home</A></LI><LI id="project"><A href="https://2015.igem.org/Team:Freiburg/Project/Overview">Project</A></LI><LI id="team"><A href="https://2015.igem.org/Team:Freiburg/Team">Team</A></LI><LI id="results"><A href="https://2015.igem.org/Team:Freiburg/Results">Results</A></LI><LI id="policy" style="font-size:11px"><A href="https://2015.igem.org/Team:Freiburg/Practices">Human Practice</A></LI><LI id="notebook"><A href="https://2015.igem.org/Team:Freiburg/Notebook">Notebook</A></LI></UL></DIV></DIV></DIV><DIV id="navtextwrapper"><DIV id="navtext"><UL id="project-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/Overview">Introduction</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/System">Overview</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Design">Diagnostics today</A></LI><LI><A class="navsub1 subsubmenu" href="#">The DiaCHIP</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/DNA_Engineering">DNA Engineering</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Cellfree_Expression">Cell-Free Expression</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Protein_Purification">Protein Purification</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Surface_Chemistry">Surface Chemistry</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/iRIf">Optical Detection</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Project/Future_Directions">Outlook</A></LI><UL id="team-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Team">Members</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Attributions">Acknowledgements</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Partners">Partners</A></LI></UL><UL id="results-sub"><LI><A class="navsub1 subsubmenu" href="#">The DiaCHIP</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results">Main Results</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Diagnostics">Diagnostics</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Own_Device">Our Device</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Immobilization">DNA Array</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Cellfree">Cell-Free Expression</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Surface">Binding on Surface</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Results/Modeling">Modeling</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Parts">BioBricks</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Collaborations">Collaborations</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/InterLab_Study">Interlab Study</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Achievements">Achievements</A></LI><UL id="policy-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Science_Fair">Science Fair</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Theater_Project">Theater Project</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Public_Opinion">Public Opinion</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/College">College Talk</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Human_Practice/Ethics">Ethics</A></LI></UL><UL id="notebook-sub"><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Labjournals">Labjournals</A></LI><LI><A class="navsub1 subsubmenu" href="#">Techniques</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Classic_vs_Gibson">Gibson</A></LI><LI><A class="navsub2" href="https://2015.igem.org/Team:Freiburg/Project/Coli_Strains">E.coli Strains</A></LI></UL><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Protocols">Protocols</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Materials">Materials</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Wikimigration">Wiki-Tricks</A></LI><LI><A class="navsub1" href="https://2015.igem.org/Team:Freiburg/Glossary">Glossary</A></LI></DIV></DIV></DIV><DIV class="clearfix" id="smallnav"><UL class="clearfix"><LI id="home_small"><A href="https://2015.igem.org/Team:Freiburg/Home">Home</A></LI><LI id="project_small"><A href="https://2015.igem.org/Team:Freiburg/Project/Overview">Project</A></LI><LI id="results_small"><A href="https://2015.igem.org/Team:Freiburg/Results">Results</A></LI><LI id="team_small"><A href="https://2015.igem.org/Team:Freiburg/Team">Team</A></LI><LI id="policy_small"><A href="https://2015.igem.org/Team:Freiburg/Practices">Practice</A></LI><LI id="notebook_small"><A href="https://2015.igem.org/Team:Freiburg/Notebook">Notebook</A></LI></UL><A href="#" id="pull">Menu</A></DIV><DIV class="content_background_wrapper"><DIV class="central_position_wrap"><DIV id="bubble_wrapper"><DIV id="bubble_sidebar" class="bubble2"><P><A id="bubble2" href="#firstHeading">text</A></P></DIV><DIV id="bubble_sidebar" class="bubble3"><P><A id="bubble3" href="">next</A></P></DIV></DIV><DIV class="page_content"><DIV class="content_box"><H1>Cell-Free Expression</H1><H2>Why Do We Use Cell-Free Expression?</H2><P><STRONG> Cell-free</STRONG> expression offers a possibility to overcome several challenges of conventional protein expression and has many advantages, in particular for our project.<SUP><A class="fn_top" href="#fn__1" id="fnt__1" name="fnt__1">1)</A></SUP><STRONG> Cell-based</STRONG> protein expression is a well-established method to obtain large amounts of a target protein. It enables high yield expression and purification of the protein in quantities sufficient for various <I>in vitro</I> applications<SUP><A class="fn_top" href="#fn__2" id="fnt__2" name="fnt__2">2)</A></SUP>.
Nevertheless, it is a tedious task to generate all the genetically modified organisms if many different proteins need to be expressed. Additionally, purification cannot be performed with a generalized protocol, and usually requires separate optimization for each protein
<SUP><A class="fn_top" href="#fn__1" id="fnt__1" name="fnt__1">1)</A></SUP>. 
In contrast, cell-free expression saves a lot of time and money. It avoids the generation of genetically modified organisms for every protein. At the purification step, cell-free expression avoids the need for cell lysis and therefore circumvents this harsh procedure, thus preserving the integrity of the protein<SUP><A class="fn_top" href="#fn__3" id="fnt__3" name="fnt__3">3)</A></SUP>.  In cell-based expression, too strong induction often results in aggregation of protein, rendering it non-functional. This risk is minimized by using cell-free expression, since the expressed protein is diluted in a far larger volume than the intracellular space and inclusion bodies do not form<SUP><A class="fn_top" href="#fn__4" id="fnt__4" name="fnt__4">4)</A></SUP>. 
For translating DNA templates into protein microarrays in a microfluidic setup, cell-free expression is the method of choice. This system is capable of expressing many different sequences at once. Additionally, the microfluidic setup provides the opportunity to constantly supplement the expression. Replacing depleted components, like dNTPs, amino acids or energy sources like creatine phosphate, allows higher protein yields<SUP><A class="fn_top" href="#fn__5" id="fnt__5" name="fnt__5">5)</A></SUP>. 
</P><H2>Basics of Cell-Free Expression</H2><H3>The Components</H3><DIV class="image_box right"><DIV class="thumb2 trien" style="width:390px"><DIV class="thumbinner"><P><STRONG>Figure 1: Basic Components of a Cell-Free Expression Mix.</STRONG> A standard cell-free expression systems consists of a DNA template, the cell extract and additives containing all the components needed for energy regeneration, buffering, and generation of the protein. </P></DIV></DIV></DIV></DIV><P style="margin-bottom: 0">
Two basic components are needed to conduct <I>in vitro</I> protein expression as seen in figure 1: 
<UL><LI>the genetic template (mRNA or DNA) encoding the target protein
</LI><LI>a reaction solution containing the transcriptional and translational molecular machinery </LI></UL></P><P style="margin-bottom: 0">
Cell extracts supply the reaction with most of the beforehand mentioned molecules, including:
<UL><LI>RNA polymerases for mRNA transcription
</LI><LI>ribosomes for polypeptide translation
</LI><LI>tRNA and amino acids 
</LI><LI> enzymatic cofactors and an energy source
</LI><LI>cellular components essential for proper protein folding 
</LI></UL></P><P>
Regular cell lysate already contains most of the components needed for cell-free expression. The machineries that usually conduct the translation and transcription of various proteins in the organism now only produce the protein of choice. The basic components, the amino acids and dNTPs, are already present but are also supplemented to increase the efficiency. As the normal energy regeneration system is missing in a cell lysate, an artificial one is added; for this we use creatine phosphokinase. 
Other components present in a cell-free expression mix buffer the sensitive system and imitate the cytosolic environment.  
</P><DIV class="image_box left"><DIV class="thumb2 trien" style="width:280px"><DIV class="thumbinner"><P><STRONG>Figure 2: Prokaryotic vs eukaryotic cell-free expression.</STRONG> While prokaryotic expression enables higher yields and is more cost efficient, eukaryotic expression offers more advanced features. </P></DIV></DIV></DIV></DIV><P> 
Comparing the currently available cell-free expression systems reveals advantages and disadvantages depending on the field of application.
An <I>E. coli</I> based system convinces with simplicity, cost-efficiency and its high yield production. However, for the production of complex eukaryotic proteins, it might not be the best option as it does not provide post-translational modifications such as core glycosylation or phosphorylation, and natural membrane components (e.g. microsomes) for membrane protein insertion.
Eukaryotic cell-free systems based on wheat germ embryos, rabbit reticulocytes, insect cells or other animal and human cell lines would be the better choice for expression of eukaryotic proteins.<SUP><A class="fn_top" href="#fn__4" id="fnt__4" name="fnt__4">4)</A></SUP></P><H3>The Reaction</H3><DIV class="image_box right"><DIV class="thumb2 trien" style="width:300px"><DIV class="thumbinner"><P><STRONG>Figure 3: Batch vs Dialysis mode.</STRONG> In batch mode, the cell-free mix is assembled in one complete batch, whereas a membrane in dialysis mode allows feeding of the reaction with additional components over time, thus enabling higher expression.</P></DIV></DIV></DIV></DIV><P>
There are several ways of performing a cell-free reaction. In <B>batch reactions</B>, transcription and translation are carried out in one reaction vessel containing all the necessary components. Due to factors like fast depletion of energy supply, degradation of components (e.g. nucleotides), and decreasing concentrations of free Mg<SUP>2+</SUP>-ions, the reaction in the batch system usually reaches a plateau after about 1-2 hours<SUP><A class="fn_top" href="#fn__1" id="fnt__1" name="fnt__1">1)</A></SUP>. 
</P><P>
In <B>dialysis mode</B>, the cell-free transcription and translation reaction is carried out in a small reaction chamber that is separated by a dialysis membrane from a 10-20x larger reservoir containing low molecular weight reagents. The reservoir supplies the reaction chamber with ions, energy substrates, nucleotides and amino acids. In turn, the low molecular weight by-products are efficiently diluted via the membrane from the reaction chamber into the reservoir. This setup will drive the reaction longer (usually 20-24 hrs) and results in higher levels of produced protein<SUP><A class="fn_top" href="#fn__1" id="fnt__1" name="fnt__1">1)</A></SUP>.
</P><H3>Design of the DNA Template</H3><P>
In general, it is possible to use RNA, plasmids, or linear templates (e.g. PCR products). In all cases, the structural design of the DNA or RNA is of high importance for the yield and quality of the protein produced.
In bacterial 5'-UTRs optimization can be achieved by placing a Shine-Dalgarno (SD) sequence at an optimal distance, typically eight nucleotides, from the AUG start codon. The SD sequence interacts with the 3'-end of 16S ribosomal RNA (rRNA) that is part of the small ribosomal (30S) subunit, to facilitate initiation. Furthermore, the integration of an A/U-rich enhancer sequence further upstream allows the SD sequence to interact more effectively with the rRNA. 
Another attempt in this context is the use of different tags to enhance expression, as reported by Haberstock <I>et al</I> (2012).<SUP><A class="fn_top" href="#fn__6" id="fnt__6" name="fnt__6">6)</A></SUP>
This design was used for our <A href="https://2015.igem.org/Team:Freiburg/Labjournals/Plasmids" target="blank">DNA</A> templates for cell-free expression of antigens.
</P><P>
In eukaryotes, the Internal Ribosome Entry Site (IRES) is a highly structured element found within viral mRNA that is able to induce eukaryotic initiation. This solves the problem of ineffective capping, which presents a major restriction of high production. For example, the IRES of the Cricket Paralysis Virus (CrPV) showed the ability to increase expression substantially across several species<SUP><A class="fn_top" href="#fn__7" id="fnt__7" name="fnt__7">7)</A></SUP>.
</P><H2 id="diamix" class="sectionedit3">Our DiaMIX</H2><P>
We developed a low-budget <A href="https://2015.igem.org/Team:Freiburg/Protocols/Cellfex" target="blank">protocol</A> for iGEM teams to express proteins in a prokaryotic cell-free expression system.
Due to the prokaryotic origin of most of the proteins expressed during our project, we decided to base our cell-free expression system, the DiaMIX, on an <I>E. coli</I> culture.</P><DIV class="image_box right"><DIV class="thumb2 trien" style="width:330px"><DIV class="thumbinner"><P><STRONG>Figure 4: <I>E. coli </I> lysate production. </STRONG> The strain <A href="https://2015.igem.org/Team:Freiburg/Project/Coli_Strains" target="blank">BL21</A> was transformed with the plasmid <A href="https://2015.igem.org/Team:Freiburg/Labjournals/Plasmids" target="blank">pRARE2</A>, induced with IPTG to express T7 polymerase, lysed, centrifuged and the supernatant was flash frozen in liquid N<SUB>2</SUB>.</P></DIV></DIV></DIV></DIV><P> 
For producing the DiaMIX lysate, we used the <I>E. coli</I> strain <A href="https://2015.igem.org/Team:Freiburg/Project/Coli_Strains" target="blank">BL21</A>, as it is carrying a gene coding for the T7 RNA polymerase, which improves protein expression. The cells were induced with IPTG prior to further treatment to initialize T7 polymerase expression, therefore allowing a high rate of protein synthesis. <SUP><A class="fn_top" href="#fn__6" id="fnt__6" name="fnt__6">6)</A></SUP>
Since codon usage is a limiting factor, especially when expressing proteins of eukaryotic origin, the <I>E. coli</I> BL21 cells were additionally transformed with the plasmid <A href="https://2015.igem.org/Team:Freiburg/Labjournals/Plasmids" target="blank">pRARE2</A>  containing the coding sequence of tRNAs for the translation of rare codons (AGA, AGG, AUA, CUA, GGA, CCC and CGG)under control of their native promoters. We prepared the lysate according to an <A class="Wikilink1" href="https://2015.igem.org/Team:Freiburg/Protocols/Cellfex" title="cellfree_protocol" target="blank">optimized protocol</A> adapted from the <A class="http://www.embl.de/pepcore/pepcore_services/protein_expression/ecoli/lysate/" target="_blank" title="EMBL" target="blank">European Molecular Biology Lab (EMBL) Heidelberg</A>(figure 4) 
This lysate contains the whole transcription and translation apparatus of the cell, allowing successful expression of proteins from DNA templates.
</P><H3>Verification of Protein Expression</H3><P>
To validate whether the expression in the DiaMIX yields correctly folded proteins, we used various methods like <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Protocols/MesurementGFP" target="blank">fluorescence measurements</A>, <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Protocols/LUC" target="blank">luciferase assays</A> and <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Protocols/Western_Blot" target="blank">Western Blots</A>. 
 Our first approach was the expression of <A href="https://2015.igem.org/Team:Freiburg/Results/Cellfree" target="blank">GFP</A>, which can be detected via its fluorescence. We used a plasmid template that was kindly provided by <A class="urlextern" href="https://www.zbsa.uni-freiburg.de/projects/ag-roth" target="blank">AG Roth.</A>
For the results of this experiment, have a look at <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Results/Cellfree" title="cellfree_results">our Results Page</A>.

As a second test we performed a luciferase assay. In contrast to the expression of GFP which can be followed by a gradual rise of fluorescence over time, the luciferase assay is based on the emission of light when the substrate luciferin is converted into oxyluciferin (figure 5). This reaction is catalyzed by the reporter luciferase. Since it depends on inducing the reaction with luciferin, it does not allow a tracking of expression over time. 

Check out our <A class="wikilink1" href="https://2015.igem.org/Team:Freiburg/Results/Cellfree" title="cellfree_results" target="blank">results</A> for more information about the outcome. This analysis helped us to evaluate the concentration of expressed protein in the cell-free expression system. 

</P><DIV class="flexbox"><DIV class="thumb2 trien"><DIV class="thumbinner"><P><STRONG>Figure 5: Luciferase reaction.</STRONG> The luciferase catalyzes the reaction of D-luciferin to D-luciferyl using ATP as an energy source. D-luciferyl is instantly converted into oxyluciferin, whereby energy is emitted as light. The intensity of bioluminescence is proportional to the amount of expressed luciferase. </P></DIV></DIV></DIV></DIV><H3>Cell-free Expression of Antigens</H3><P>
An important goal was to express antigens with the required tags in a concentration and folding state that suffices for an iRIf measurement. Here, we focused on a <A href="https://2015.igem.org/Team:Freiburg/Project/Diseases#Tetanus_anchor" target="blank"><I>C. tetani</I></A> epitope flagged by a double His and a Spy-tag. For cloning of this construct we used a newly designed <A href="https://2015.igem.org/Team:Freiburg/Methods/Cloning" target="blank">cloning site</A> and a vector that we had optimized for cell-free expression.</P><H3>Testing Different Conditions</H3><P>
To identify the optimal DNA concentration that yields in a maximum protein expression, we tested different conditions. For this experiment, we used DNA coding for the enzyme luciferase.
Another component that is crucial for optimal cell-free expression is magnesium. To evaluate the effects of different magnesium concentrations, we started multiple cell-free expression mixes using different magnesium starting concentrations.
For results of our optimization, go to our Results Page.

              </P><DIV class="link_button link_button_arrow"><P><A href="https://2015.igem.org/Team:Freiburg/Results/Cellfree" title="cell-free results">Cell-Free Results</A></P></DIV><H2>Step by Step Validation</H2><DIV class="image_box right"><DIV class="thumb2 trien" style="width:330px"><DIV class="thumbinner"><P><STRONG>Figure 6: Immobilization of DNA. </STRONG> PCR product coding for GFP with a <A href="https://2015.igem.org/Team:Freiburg/Glossary" target="blank">Cy3</A> and an amino label was spotted onto the activated <A href="https://2015.igem.org/Team:Freiburg/Glossary" target="blank">PDMS</A> slide. Binding of the amino group to the specific surface allows for direct DNA immobilization.</P></DIV></DIV></DIV></DIV><P>
Since we aimed at using cell-free expression in combination with the DiaCHIP, we decided to validate the <A href="https://2015.igem.org/Team:Freiburg/Project/System" target="blank">process</A> in individual steps. 
First, the DNA templates had to be immobilized on the PDMS slide (figure 6).
In another step we expressed GFP cell-free in a tube and spotted the produced protein on a iRIf slide to examine whether the interaction with the respective antibody can be detected (figure 7). Furthermore, to evaluate the interference of the specific surface with our expression system, we spotted our expression system onto the slide without starting the reaction beforehand (figure 8). Lastly, we performed a cell-free expression from immobilized DNA in the finished setup of glass and PDMS slide (figure 9).
</P><DIV class="image_box left"><DIV class="thumb2 trien" style="width:270px"><DIV class="thumbinner"><P><STRONG>Figure 7: Spotting of expressed GFP.</STRONG> Cell-free expressed GFP lysate was spotted onto the activated glass slide after expression was performed.</P></DIV></DIV></DIV></DIV><DIV class="image_box left"><DIV class="thumb2 trien" style="width:270px"><DIV class="thumbinner"><P><STRONG>Figure 8: On-slide expression of GFP.</STRONG> Cell-free expression mix was spotted on the activated glass slide and expression was performed.</P></DIV></DIV></DIV></DIV><DIV class="image_box left"><DIV class="thumb2 trien" style="width:270px"><DIV class="thumbinner"><P><STRONG>Figure 9: In-chamber expression of GFP.</STRONG> Cell-free expression mix was pipetted into the flow chamber with immobilized DNA on the silicone slide and an activated glass slide. Then, the expression was performed.</P></DIV></DIV></DIV><DIV class="footnotes"><H3>References</H3><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__1" id="fn__1" name="fn__1">1)</A></SUP><A href=" http://doi.org/10.1016/j.sbi.2013.03.012" target="_blank"> Bernhard, F., &amp; Tozawa, Y., 2013. Cell-free expression-making a mark. Current Opinion in Structural Biology, 23(3), 374–380.</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__2" id="fn__2" name="fn__2">2)</A></SUP><A href="http://www.ncbi.nlm.nih.gov/pubmed/22008973" target="_blank"> Carlson et al., 2011. Cell-free protein synthesis: applications come of age. Biotechnology advances.</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__3" id="fn__3" name="fn__3">3)</A></SUP><A href="http://www.mdpi.com/2073-4468/4/1/12">Kubick., et al., 2015. Cell-Free Synthesis Meets Antibody Production: A Review. Antibodies.</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__4" id="fn__4" name="fn__4">4)</A></SUP><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4133780/" target="_blank">Rosenblum et al., 2015. Engine out of the Chassis: Cell-Free Protein Synthesis and its Uses. </A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__5" id="fn__5" name="fn__5">5)</A></SUP><A href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0096635" target="_blank">Stech et al., 2014.A Continuous-Exchange Cell-Free Protein Synthesis System Based on Extracts from Cultured Insect Cells. PLOS.</A></DIV><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__6" id="fn__6" name="fn__6">6)</A></SUP><A href="http://www.ncbi.nlm.nih.gov/pubmed/22342679" target="_blank"> Haberstock, S. et al., 2012. A systematic approach to increase the efficiency of membrane protein production in cell-free expression systems. Protein Expression and Purification.</A><DIV class="fn"><SUP><A class="fn_bot" href="#fnt__7" id="fn__7" name="fn__7">7)</A></SUP><A href=" http://doi.org/10.1371/journal.pone.0082234" _blank="">Brödel, A. K., et al., 2013. IRES-mediated translation of membrane proteins and glycoproteins in eukaryotic cell-free systems.</A></DIV></DIV></DIV></BODY></HTML>