Team:Freiburg/Project/Methods/Cloning
""
Home
Project
Team
Results
Human Practice
Notebook
Introduction
Overview
Diagnostics today
The DiaCHIP
DNA Engineering
Cell-Free Expression
Protein Purification
Surface Chemistry
Optical Detection
Outlook
Members
Acknowledgements
Partners
The DiaCHIP
Main Results
Diagnostics
Our Device
DNA Array
Cell-Free Expression
Binding on Surface
Modeling
BioBricks
Collaborations
Interlab Study
Achievements
Science Fair
Theater Project
Public Opinion
College Talk
Ethics
Labjournals
Techniques
Gibson
E.coli Strains
Protocols
Materials
Safety
Wiki-Tricks
Glossary
Home
Project
Results
Team
Practice
Notebook
Menu
Cloning methods
Theoretical cloning
All our constructs were theoretically cloned before we ordered primers and started with the wet-lab work. For this we used Geneious (Firma???). Detailed cloning strategies can be found next to the respective plasmid maps (?).
For classical cloning approaches primers were designed with an annealing temperature of approximately 60°C to facilitate polymerase chain reactions. Important restriction sites or linker regions were included via primer overhangs. Using the tools provided by Geneious the cloning strategy was tested extensively to discover potential mistakes or other occurring problems.
For Gibson cloning we decided to stay consistent with our classical cloning strategy and designed our primers with an annealing temperature of approximately 60°C as well. Attached to the binding region a 32 basepair overlap was added. Potential linkers or additional restriction sites were introduced between the binding region and the overlap.  All Gibson constructs were theoretically cloned with Geneious to avoid mistakes and improve the wet-lab work.
Primer Design
DNA amplification: All primers used for amplification of DNA from a template were designed with Geneious software. The binding site of each primer is indicated to have a melting temperature of about 60°C. Possible extentions have not been included for melting temperature calculations. When an extention was needed to produce further recognition sites for restriction enzymes or to add overlaps compatible for Gibson Assembly, they were allways fused to the 5' end of the respective primer.
Mutagenesis: Primers that were created for site-directed mutagenesis were also designed with Geneious software. Such primers were supposed to have length of about 50 bp and the site which has to be mutated should be located in the middle of the sequence to make sure the primer binds to the template over its full length.
Sequencing: Sequences for primers designed for sequencing purposes only were obtained by the GATC tool for custom primers.
All primers were synthesized by Sigma Aldrich.
PCR
For amplification of different DNA fragments from plasmid templates, various PCR approaches were used. In general, PCRs were performed with Phusion High Fidelity DNA Polymerase (NEB). The components of the reaction mixture were used in defined amounts/concentrations recommended by the manufacturer:
Component
Concentration/amount
Phusion polymerase
1 U
Phusion polymerase buffer
1X
Forward primer
0.5 µM
Reverse primer
0.5 µM
dNTPs
200 µM
DMSO
3%
Template DNA
1 ng
ddH2O
up to 50 µl
Primers for DNA amplification were designed to have a melting temperature of about 60°C (detailed information can be found in the Oligo list). The following cycling conditions have been used for standard PCRs:
Step
Temperature [°C]
Duration [s]
Initial denaturation
98
30
Denaturation
98
20
Annealing
60
20
Extention
72
30 s/kb*
Final extention
72
300
Cooling
4
infinite
* Extention times were calculated  according to NEB's recommendations for Phusion High-Fidelity Polymerase.
Upon unsuccessful amplification, variations of the protocol have been performed. Those inculde a temperature gradient from 55°C to 65°C in the annealing step and the use of Q5 Polymerase (NEB).
Agarose Gel Electrophoresis
The analysis of PCR products as well as enzymatic DNA digests was routinely performed via gel electrophoresis using 1 % agarose (w/v) in 1x TAE buffer. Before polymerization of the gel 4-8 µl MidoriGreen Advance (Biozym) were added. Commonly, a voltage between 80 and 120 V was applied to the gel for 20 to 45 minutes before analysis on a UV scanner.
Gel Extraction
To extract DNA fragments obtained by PCR or enzymatic digest from agarose gels, the QIAquick Gel extraction Kit from Qiagen was used according to the manufacturers instructions. Before elution of the DNA from the column an additional incubation step of 10 min (shaking with 300 rpm at 40°C) was added. DNA concentrations were measured by photospectrometry (NanoDrop).
Restriction digest
Enzymatic digests of DNA were performed for two purposes. The first is to verfiy the successful ligation of DNA fragments into a certain backbone. The second is to obtain those fragments by excision from another vector. To be able to ligate fragments together, they need to be digested with enzymes producing the same compatible ends. In general, restriction digests were performed with ~ 1 µg of DNA in a total reaction volume of 50 µl. Therefore, 1 µl of each enzyme were added as well as 5 µl of the the corresponding 10 X buffer and the appropriate amount of dH2O to reach a final volume of 50 µl. The utilized enzymes were provided by NEB and Fermentas. Reactions with NEB enzymes were incubated at 37°C for 1 h, while reactions with Fermentas FastDigest enzymes were performed for 30 min at 37°C. Afterwards, the samples were analyzed by agarose gel electrophoresis.
Ligation
To ligate a certain DNA fragment into a vector which had been opened by enzymatic digest with the same restriction enzymes, 50 ng of the backbone DNA was used. An 8-fold molar amount of the insert was added as well as 1 µl T4 DNA ligase and 2 µl T4 DNA ligase buffer (NEB). dH2O was added up to a final volume of 20 µl. The reaction was incubated either at RT for 1 h or at 16°C overnight. Subsequently, 5 µl of the ligation product were transformed into chemically competent E. coli TOP10.
Gibson Assembly
Besides classical cloning methods, we used Gibson Assembly to obtain desired plasmids. The operating mode of this method is devided into three major parts:
An exonuclease removes bases from the 5' end of each DNA strand.
Complementary regions of different DNA strands can anneal and a polymerase fills up the gaps.
The fragments a ligated together.
To enable this these three steps, DNA strands with compatible ends of about 32 bp are needed. Those can either be incorporated by primer overhangs or by gene synthesis.
A 5 µl mix of the DNA parts that are supposed to be assembled is prepared. Its composition is calculated based on the length and concentration of every single fragment. The insert(s) should at least be contained in a 4 - 8 fold molar amount of the antibiotic resistance containing backbone.
This DNA mix is then added to a 15 µl aliquot of the Assmbly Master Mix. 10 µl of this mix are then used to transform 25 µl of chemically competent E. coli TOP10 cells according to the following protocol.
Preparation of the Assembly Master Mix:
Ingredient
Concentration of the stock solution
Volume for 80 aliquots [µl]
Volume for 25 aliquots [µl]
ISO buffer
5X
320
100
T5 exonuclease
10 U/µl
0.64
0.2
Phusion polymerase
2 U/µl
20
6.25
Taq DNA Ligase
40 U/µl
160
50
dH2O
-
ad 1.2 ml
ad 375 µl
Aliquots of 15 µl are stored at -20°C.
Preparation of 5x ISOthermal buffer:
Ingredient
Concentration of the stock solution
Amount
Tris-HCL
3 M
3 ml
MgCl2
2 M
150 µl
dNTP
100 mM of each
60 µl
DTT
1 M
300 µl
PEG-8000
-
1.5 g
NAD+
100 mM
300 µl
Aliquots of 320 µl are stored at -20°C.
Transformation
Chemically competent E. coli cells were transformed with a certain vector by heat shock. The cells were thawed on ice and the appropriate amount of DNA was added. After 10 min of incubation on ice, the cells were heat shocked for 45 s at 42°C. They were chilled on ice for 2 min, before 500 µl LB medium without antibiotics were added. Following 1 h incubation at 37°C (500 rpm), the cells were plated on LB medium containing the appropriate anibiotic(s) and grown at 37°C over night.
Growing conditions
E.coli cells were grown in LB medium (10 g/l NaCl, 10 g/l Trypton, 5 g/l yeast extract) at 37°C, shaking with 200 rpm. For inoculation of liquid cultures, a single colony was taken with a sterile pipette tip which was then pitched/thrown/put into a test tube with 5 ml liquid LB. Appropriate antibiotics were added (see table below). For long-term storage 1 ml of an E.coli o/n culture was mixed with 300 µl  50% glycerol, immediately frozen in liquid nitrogen and stored at -80°C.
Additive
Concentration [µg/ml]
Storage
kanamycin
50
-20°C (H2O)
ampicillin
100
-20°C (H2O)
chloramphenicol
30
-20°C (Ethanol)
gentamycin
25
-20°C (H2O)
Competent cells
To prepare competent E.coli cells, E.coli cells were streaked out on LB plates containing appropriate antibiotics. The plates were incubated at 37°C over night and then 5 ml LB medium with antibiotic was inoculated with a single colony. After o/n incubation at 37°, 100 ml LB medium without antibiotic was inoculated with 1 ml of the o/n culture. Cells were grown for 2-3 hours until they reached an OD(600) of 0.4 to 0.6. The E.coli cells were then transferred into a pre-cooled, sterile centrifuge tube (250 ml) and pelleted for 5 minutes (5000 rpm, 4°C). After discarding the supernatant the cells were resuspended in 20 ml of ice cold TFB1 with cut tips, followed by 5 minutes of incubation on ice. Then, the resuspended cells were again centrifuged (5 min, 5000 rpm, 4°C) and afterwards resuspended in 4 ml ice-cold TFB2 with cut tips.  The resuspension of E.coli cells was divided in 25 µl or 50 µl aliquots that were immediately frozen in liquid nitrogen and stored at -80°C.
Preparation of buffers:
TBF1:
Chemical
Final concentration
Amount needed for 200 ml
KAc
30 mM
0.589 g
MnCl2
50 mM
1.6919 g
KCl
100 mM
1.482 g
CaCl2
10 mM
0.294 g
Glycerol
15 %
30 g
sterile filtration and storage at 4°C
TBF2:
Chemical
Final concentration
Amount needed for 200 ml
MOPS
10 mM
0.419 g
KCl
10 mM
0.149 g
CaCl2
75 mM
2.205 g
Glycerol
15 %
30 g
autoclaving and storage at 4°C
Plasmid isolation
For isolation of plasmid DNA from E.coli cells plasmid isolation kits from Peqlab(???), Biozym (???), Qiagen (???) and Zymo Research (???) were used. Plasmid islolation was done according to manufacturers information.
Oligo-annealing
Primers for oligo annealing were designed using Geneious. Primers were diluted according to information from Sigma-Aldrich. From this dilution a 5 mM and a 2.5 mM primer solution was prepared. Primer dilutions were incubated at 95°C for 5 minutes, followed by a slow cool-down phase for approximately one hour.
Blunt-end ligation
For blunt-end ligation of PCR fragments into pJet the following components were added to a reaction mix:
Volume [µl]
Ingredient
1
ligation fragment (10 ng/µl)
0.5
pJet
1
T4 ligase
5
2x reaction buffer
2.5
H2O
The ligation mix was incubated at room temperature for 5 minutes. Approximately 5 µl of the ligation was used to transform 25 µl of chemically competent E.coli T10. Transformation was done according to the protocol delivered with the respective kit. The remaining ligation mix was stored at -20°C as backup.
info
