Team:Freiburg/Project/Classic vs Gibson
""
Home
Project
Team
Results
Human Practice
Notebook
Introduction
Overview
Diagnostics today
The DiaCHIP
DNA Engineering
Cell-Free Expression
Protein Purification
Surface Chemistry
Optical Detection
Outlook
Members
Acknowledgements
Partners
The DiaCHIP
Main Results
Diagnostics
Our Device
DNA Array
Cell-Free Expression
Binding on Surface
Modeling
BioBricks
Collaborations
Interlab Study
Achievements
Science Fair
Theater Project
Public Opinion
College Talk
Ethics
Labjournals
Techniques
Gibson
E.coli Strains
Protocols
Materials
Safety
Wiki-Tricks
Glossary
Home
Project
Results
Team
Practice
Notebook
Menu
text
next
Short Review: Classical Cloning vs. Gibson Assembly
Synthetic biology is based on the combination of various DNA sequences to obtain complex genetic networks. The single parts may originate from different organisms to broaden the toolbox for the improvement of such networks on several levels. Of course, these combinations cannot be found in nature and therefore have to be assembled in vitro. Over time, many different methods have been established to simplify the production of synthetic gene networks. Among others, scientists have to decide between the classical cloning method based on the activity of restriction enzymes and the more recently developed method making use of an exounuclease, called Gibson Assembly. This short review compares and contrasts the two methods to simplify this decision for future iGEM Teams.
The basic idea behind classical cloning is to generate DNA fragments supposed to be assembled that are carrying short compatible single stranded overhangs that allow their annealing. After specific annealing of the fragments, they can be covalently bound to each other.
This strategy was invented and established shortly after the discovery of DNA ligases and restriction enzymes in 1967 1) 2) .  The original function of DNA ligases is to form a covalent bond between viral and bacterial DNA allowing the virus to insert its DNA into the host’s genome for efficient replication.
Restriction enzymes are characterized by their ability to recognize short specific DNA sequences and cut the DNA double strand. Thereby, they produce short DNA overhangs, so called sticky ends, either on the 5’ or the 3’ end of a DNA strand.
Synthetic biology makes use of the properties of restriction enzymes in two ways. On the one hand, DNA is cut at desired positions to produce defined fragments. On the other hand, different DNA fragments with compatible sticky ends specifically anneal with each other and thus can be pieced together by a DNA ligase.
Figure 1: Schematical overview of a classical cloning approach. Two different restriction enzymes (red and green) are used to cut open a plasmid and to digest the desired insert. Due to the compatible overhangs originating from the restriction the insert can be ligated (blue) with the plasmid, forming a new plasmid containing an additional part.
Classical cloning was invented a long time before Gibson cloning and countless restriction enzymes with individual specificities have been discovered and established for laboratory use. Additionally, nearly all vectors for recombinant DNA assembly contain a multiple cloning site that allows choosing between several restriction enzymes, dependent on the insert that is supposed to be incorporated.
Nonetheless, classical cloning also bears some disadvantages. First, the definition of a backbone is needed. Each DNA fragment that is supposed to be assembled has to be inserted into this backbone one after the other. The assembly of fragments of about the same size as well as the assembly of more than two fragments in one step is not possible.
Next, classical cloning is dependent on recognition sites for restriction enzymes that are positioned exactly in the appropriate way. Often, there are either no suitable restriction sites available at all, or using the sites will result in frameshift mutations when assembling several coding sequences. In these cases, suitable restriction sites have to be attached for example to the insert by amplification of the fragment using primers with specific overhangs.
A rather new method for the assembly of recombinant DNA is called Gibson Assembly and was invented in 2009 by Gibson et al 3) . This system is based on the activity of three enzymes, exonucleases, polymerases and ligases, acting together in an isothermal reaction. The DNA fragments that are supposed to be assembled have to be overlapping. Accordingly, the removal of bases from the 5’ end by exonuclease activity results in compatible single stranded DNA overhangs. These complementary regions should anneal at about 50°C. If the exonuclease recessed the DNA further than until the complementary sequence's end, the polymerase fills up the gaps. The DNA ligase is used to form a covalent bond between the DNA fragments afterwards.
The main advantage of Gibson Assembly over classical cloning is the ability to assemble more than two fragments in one step. Therefore, the only requirement is to append suitable overlaps to the DNA fragments what can be obtained by PCR amplification using primers with specific overhangs. This saves a lot of work and time when trying to assemble many different DNA fragments as it is common in synthetic biology where complex gene networks are often combined in a single plasmid.
Figure 2: Schematical overview of Gibson cloning. Sequences derived from three different plasmids are assembled in a single step after PCR        amplification.
Additionally, Gibson Assembly allows generating recombinant DNA up to more than 100 kb, which is not possible by classical cloning methods. It is also less problematic to insert very small DNA fragments into larger backbones.
On the other side, the overlapping regions have to be attached to every DNA fragment individually. This can be achieved by PCR using primers with specific overhangs, which have to be designed individually for every cloning experiment.
Regarding the sequence of the overlapping regions, it can cause problems if they contain repetitive elements as this promotes the chance of incorrect annealing. In the worst case, this results in a frameshift mutation leading to a wrong amino acid sequence in the encoded protein. Sometimes this problem can be solved by changing the overlapping region to a part of the sequence without repetitive elements.
All in all, the cloning method of choice has to be selected individually dependent on the kind of experiment that is planned. Gibson Assembly is advantageous if there are many or very long DNA fragments that have to be joined. Additionally, it bears the possibility to insert very small fragments, like protein tags for example, into the primer overhang and amplify it by PCR before cloning the whole fragment into the backbone. Last but not least, it is more flexible than classical cloning because it does not depend on specific restriction sites.
Classical cloning could be preferred if only one DNA fragment is supposed to be inserted into a backbone and the required restriction sites are available. But it can also be worthwhile to obtain these sites via PCR if DNA fragments are supposed to be exchanged in a standardized manner, for example.
info
1) Fareed et al., 1967. Enzymatic breakage and joining of deoxyribonucleic acid. II. The structural gene for polynucleotide ligase in bacteriophage T4.Proc Natl Acad Sci U S A
2) Smith et al., 1970. A restriction enzyme from Hemophilus influenzae. I. Purification and general properties. J Mol Biol
3) Gibson et al., 2009. Enzymatic assembly of DNA molecules up to several hundred kilobases.Nat Methods
