Team:USP-Brazil/Results/Detection
HOME
TEAM
Team Collaborations Attributions Achievements
CONCEPTUAL
FRAMEWORK
Overview Modeling Chassis BioContainment Detection Elimination
RESULTS
Overview Modeling Chassis BioContainment Detection Elimination Proof of Concept
HUMAN
PRACTICES
Overview Integrated Practices Public Engagement Brazilian Meetup
INTERLAB
Overview Plate Reader Flow Citometry Microscopy DIY Photo Measurement
RESOURCES
Parts Collection of Parts Notebook Safety
SPONSORS
HOME
TEAM
Team
Collaborations
Attributions
Achievements
CONCEPTUAL FRAMEWORK
Overview
Modeling
Chassis
BioContainment
Detection
Elimination
RESULTS
Overview
Modeling
Chassis
BioContainment
Detection
Elimination
Demonstrate
HUMAN PRACTICES
Overview
Integrated Practices
Public Engagement
Brazilian Meetup
INTERLAB
Overview
Plate Reader
Flow Citometry
Microscopy
DIY Photo Measurement
RESOURCES
Parts
Collection of Parts
Notebook
Safety
Toehold Switch
Description
We decided to use a toehold switch system as a method to simulate RNA-dependent response in our cells. The image below briefly describes how this system works:
Figure 1 - A toehold-switch system scheme. The system is comprised of two RNA molecules, named Switch and Trigger RNA. The Switch RNA contains a hairpin structure that hides a ribosome binding site and its activated when the so called trigger RNA is present, allowing translation of a protein of interest.
In our system, the Trigger RNA represents an overexpressed messenger RNA which could be, for instance, a bacterial transport system related transcript, only expressed when the cell is under certain stress conditions and aimed to increase the bacterial load.  On the other hand, the Switch RNA represents the part capable of producing the desired effector, which would then be able to eliminate the related pathogen.
Firstly, we’ve carried out some experiments in order to evaluate the possibility to use this system in our work.  For this purpose, two plasmids containing compatible origins of replications and adequate copy number were designed. pETDuet and pRSFDuet plasmids were chosen for the Switch Part:BBa_K1639000 and the Trigger RNA Part:BBa_K1639001 , respectively.
Figure 2 - Two plasmids were chosen for this system. pETDuet was the plasmid containing the switch RNA while pRSFDuet contains the trigger RNA. Since both have compatible origins of replication, they could be cotransformed in our chassis.
Results
Our team has successfully demonstrated the RNA-dependent response of our cells using a toehold switch circuit.
Figure 3 - By looking at the plate using blue light, it was possible to visualize the fluorescence difference between different samples (1-6) and their respective controls containing an empty plasmid in place of the trigger RNA.
We then decided to assess if this response is IPTG-dependent, in an attempt to evaluate how much RNA is needed to activate the system:
Figure 4 - Expression profile from our constructions with different IPTG concentration.
As shown by the graph above, IPTG concentrations as low as 0,01mM could activate the system! To assess if the observed expression was mostly related to the interaction between the trigger and the switch RNAs (and not due to some leakage of the Switch RNA), we compared the fluorescence profiles of a strain transformed with the two plasmids and another transformed with both a plasmid containing the Switch RNA and an empty plasmid.
Figure 5 - Expression profile comparison between strains with and without the trigger RNA.
Our last question was: would this system be sensitive enough to work under real conditions? In other words, mRNA expression variations observed in bacteria are really able to promote the activation of the Switch RNA? For this purpose, we decided to do some modeling, which you can check it out here!
Results in our Chassis
Lastly but no least, we decided to design a new toehold switch for expression in our chassis, P. agglomerans! For that purpose, we needed to change the promoter and cotransform our bacteria with these new plasmids.
We then replaced the T7 promoter by a lac promoter from both the switch and trigger RNA, resulting in two new parts: BBa_K2486177 and BBa_K2486179 To evaluate if these new system could actually work in P. agglomerans, we decided to do some microscopy!
For our surprise, the new chassis was not only able to be cotransformed with the new plasmids but also to exhibit a toehold-switch like behavior!
Figure 6 - Microscopy study of our P. agglomerans strain . Analysis was performed by a Leica DMI8 epi-fluorescent microscope using a 60x high NA objective, 500ms at 30% intensity exposure time. Images were analysed by "MicrobeJ" plugin in ImageJ. Cell contours were detected and mean GFP intensity was calculated and background corrected. Black line indicates mean GFP fluorescence of the population
As expected, E. coli BL21 also exhibited a similar behavior using these new parts.
Figure 6 - Microscopy study of our E.coli strain . Analysis was performed by a Leica DMI8 epi-fluorescent microscope using a 60x high NA objective, 500ms at 30% intensity exposure time. Images were analysed by "MicrobeJ" plugin in ImageJ. Cell contours were detected and mean GFP intensity was calculated and background corrected. Black line indicates mean GFP fluorescence of the population
Even though the expression profile in P. agglomerans was not as high as in E.coli BL21, it is still possible to conclude that our chassis can recognize our two plasmids and exhibit an IPTG dependent toehold-switch behavior!
References
Green, A. A., Silver, P. A., Collins, J. J. & Yin, P. Toehold Switches: De-Novo-Designed Regulators of Gene Expression. Cell 159, 925–939 (2014).
Iron and Lactate Biosensors
Another way to sense diseases is through the detection of metabolites that are qualitatively or quantitatively altered in the specific condition. As a way of detecting  Plasmodium-contaminated blood in the mosquito gut, we designed the concept of a Fe2+ and Lactate presence AND gate.
Our approach was inspired by two previous iGEM projects:
Iron Coli from 2013 Evry Team
MicroBeacon from 2015 ETH_Zurich Team
The team from Evry developed an interesting iron biosensor based on E. coli fur regulator, a protein that represses its cognate operator in the presence of iron. Our chassis (P. agglomernas presents motifs for recognition of fur protein, which can be explained by it's genetic the similarity to E. coli. In order to avoid cross-talk between endogenous gene regulation and the signal from our biosensor, we developed a new system for Fe2+ detection based on dtxR regulator from Corynebacterium diphtheriae.
ETH_Zurich's 2015 team developed a lactate biosensor based on the LldR protein, wich we applied to our project.
Figure 1 - Rationale behind our AND gate.
Figure 2 - We designed a lactate and iron (Fe2+) AND gate as a way of detecting the contaminated blood. (A) Our biosensor device composed of generators of DtxR (BBa_K2486004) and LldR (BBa_K2486007), a PdtxR inverter based on tetR () and a reporter module with a hybrid promoter regulated by TetR and LldR ().
Once we were able to sucessfuly transform P.  agglomerans with pSB1C3 vector we designed our  detection module inside pSB1C3 plasmid. Our designed plasmid, the pTROJAN, is composed by 4 modules: (1)a DtxR  generator, (2) a LldR generator, (3)a TetR-based DtxR inverter and (4) a  reporter module that processes lactate and Fe2+ presence through our  designed promoter and it's cis-elements.
Figure 3 - The pTROJAN vector, a Fe2+ and Lactate and gate circuit.
We decided to firstly characterize our new promoters and parts before appling our time and resources in the construction of pTROJAN. So we designed 3 devices that would be used to validate our new parts (figure 4).
Figure 4 - Our 3 validation circuits for (1)DtxR system wich works as a Fe2+ biosensor circuit . (2)LldR system wich works as a bionsensor circuit for lactate. (3)TetR system wich works asa biosensor circuit for tetracycline
Figure 5 - BBa_K2486019: The presence of LlldR and lactate affects the circuit as expected, repressing it in the absence of lactate.
Figure 6 - BBa_K2486020: The presence of TetR and tetracycline worked as expected too, sugesting that the part could work with both regulators.
Figure 7 - BBa_K2486021: Promoter activity (GFP/OD600) under different concentrations of Iron Chloride (Fe2+) shows a decrease in promoter activity when (Fe2+) is added to the system, wich is expected by the funciotn mechanism of the regulator. The binding of the DtxR protein to iron enhances it's affinity to the DNA repressing the GFP expression.
All of our biosensors worked as expected, they are a great aditions to the registry once they create new pissibilities of combining sensorsa to paratransgenesis applications.
References
DtxR
White, A., Ding, X., Murphy, J. R., & Ringe, D. (1998). Structure of the metal-ion-activated diphtheria toxin repressor/tox operator complex. Nature, 394(6692), 502-506.
Love, J. F., Marin, V., Guerrero, L., Logan, T. M., & Murphy, J. R. (2004). Genetic and biophysical studies of diphtheria toxin repressor (DtxR) and the hyperactive mutant DtxR (E175K) support a multistep model of activation. Proceedings of the National Academy of Sciences of the United States of America, 101(8), 2506-2511.
Brune, I., Werner, H., Hüser, A. T., Kalinowski, J., Pühler, A., & Tauch, A. (2006). The DtxR protein acting as dual transcriptional regulator directs a global regulatory network involved in iron metabolism of Corynebacterium glutamicum. BMC genomics, 7(1), 21.
lldR
https://2015.igem.org/Team:ETH_Zurich/Results
Cournac, A., & Plumbridge, J. (2013). DNA looping in prokaryotes: experimental and theoretical approaches. Journal of bacteriology, 195(6), 1109-1119.
Aguilera, L., Campos, E., Giménez, R., Badía, J., Aguilar, J., & Baldoma, L. (2008). Dual role of LldR in regulation of the lldPRD operon, involved in L-lactate metabolism in Escherichia coli. Journal of bacteriology, 190(8), 2997-3005.
Xin, B., Wu, G., Zhang, K., He, Y., Tang, H., Gao, C., ... & Ma, C. (2016). Sequence similarity network analysis, crystallization, and X-ray crystallographic analysis of the lactate metabolism regulator LldR from Pseudomonas aeruginosa. Bioresources and Bioprocessing, 3(1), 1-9.
TetR
Evans, J. C., & Mizrahi, V. (2015). The application of tetracyclineregulated gene expression systems in the validation of novel drug targets in Mycobacterium tuberculosis. Frontiers in microbiology, 6.
Orth, P., Schnappinger, D., Hillen, W., Saenger, W., & Hinrichs, W. (2000). Structural basis of gene regulation by the tetracycline inducible Tet repressor–operator system. Nature Structural & Molecular Biology, 7(3), 215-219.
Bertram, R., & Hillen, W. (2008). The application of Tet repressor in prokaryotic gene regulation and expression. Microbial biotechnology, 1(1), 2-16.
