<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Saclay_Modeling_FNR"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Saclay/Modeling/FNR">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Saclay/Modeling/FNR&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Saclay/Modeling/FNR&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Saclay/Modeling/FNR&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Saclay/Modeling/FNR" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Saclay/Modeling/FNR</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><NAV class="psmenu"><UL class="psmenu_mainul"><LI><A href="https://2013.igem.org/Team:Paris_Saclay">Home</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Team">Presentation</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Paris_Saclay">Official profile</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Extras">Extras</A></LI></UL><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/PCBs">What are PCBs ?</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Project">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Modeling">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Modeling/Simulator">Simulator</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Modeling/FNR">Oxygen sensor</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/PS-PCR">PS-PCR</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/PS-PCR">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/PS-PCR/detailed_description">Detailed description</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Labwork">Labwork</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Labwork">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Parts">Biobricks</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Primers">Primers</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Human_Practices">Human practices</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Human_Practices">Reflection on the project</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Human_Practices">Reflection</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Poll">Poll</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Sewers">Sewers of Paris</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Open_source">Reflection on open source</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Open_source">Intro</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/opensourcereflexion">Reflection</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/ellen_interview">Interview</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/communication">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Saclay/Team_Collaboration">Team collaboration</A></LI></NAV></P><DIV class="menug"><DIV class="navig">


Project
<A href="/Team:Paris_Saclay/PCBs" title="Team:Paris Saclay/PCBs">What are PCBs ?</A><A href="/Team:Paris_Saclay/Project" title="Team:Paris Saclay/Project">Overview</A><A class="selflink">Modeling</A><A href="/Team:Paris_Saclay/PS-PCR" title="Team:Paris Saclay/PS-PCR">PS-PCR</A><A href="/Team:Paris_Saclay/Achievements" title="Team:Paris Saclay/Achievements">Achievements</A></DIV><DIV class="navig">


Modeling
<A href="/Team:Paris_Saclay/Modeling" title="Team:Paris Saclay/Modeling">Overview</A><A href="/Team:Paris_Saclay/Modeling/Simulator" title="Team:Paris Saclay/Modeling/Simulator">Simulator</A><STRONG class="selflink">Oxygen sensor</STRONG></DIV></DIV><DIV class="contenu"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#FNR_oxygen_sensor_system"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">FNR oxygen sensor system</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Translate_biological_knowledges_into_functional_scheme"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Translate biological knowledges into functional scheme</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Transcribe_functional_description_into_equation_system"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Transcribe functional description into equation system</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Summary_for_equations:"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Summary for equations:</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Details"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Details</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Transcribe_equations_to_Hsim_code"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Transcribe equations to Hsim code</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Set_of_parameters"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Set of parameters</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Scenarios_and_results"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Scenarios and results</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Online_simulation"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Online simulation</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Simple_simulation"><SPAN class="tocnumber">1.8.1</SPAN><SPAN class="toctext">Simple simulation</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Complex_simulation"><SPAN class="tocnumber">1.8.2</SPAN><SPAN class="toctext">Complex simulation</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-12"><A href="#Conclusion"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Reference"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="FNR_oxygen_sensor_system"><B>FNR oxygen sensor system</B></SPAN></H1><P>As we use Hsim for the modeling, a multi-agents programme developped at our university in order to follow the assembly, movements and dissociation of a large number of molecules in a virtual cell. It replaces the Ordinary Differential Equations and mimics chemical reactions of the real system by rewriting specials rules. If you want to dig in our model we suggest you to read <A href="https://www.lri.fr/~pa/Hsim/" class="external text" rel="nofollow">this page</A>, written by Mr. Patrick Amar who is the developer of Hsim. .</P><P>Roughly speaking, in our model we define a sampling volume, size in nanometer and we declare some molecules with their sizes, their moving speed and their number. Once the simulation starts, those molecules move randomly in the space and trigger some possible reaction between them with some probabilities. There is always somehow deviation from the reality, but we are all excited about this particular simulating modeling.</P><P>We construct our FNR regulator model by 3 steps.At first, We extract the biological information, draw some schemes for helping us to understand the biological. Then we translate those biological concept into specials equations or pseudo code of Hsim. Finally, we write the Hsim code and run it in different scenarios for the test. We also have developed a web similator which allows user change and test our model with their own parameter.</P><H2><SPAN class="mw-headline" id="Translate_biological_knowledges_into_functional_scheme"><B>Translate biological knowledges into functional scheme</B></SPAN></H2><DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER> Tolla and Savageau Regulation of Aerobic to Anaerobic Transitions by the FNR cycle in Escherichia coli</CENTER></DIV></DIV></DIV></DIV><P>This scheme from the Dean A.Tolla and Michael A.Savageauthe's paper shows clearly the FNR transformation cycle. After the fnr translation, apoFNR(inactive) is produced. And the dimerization of 2 apoFNR which id catalyzed by ClpXP create the active 4Fe-FNR. The 4Fe-FNR can be oxidized by oxygen to 2Fe-FNR then reduce to apoFNR with another oxygen. 2Fe-FNR plus ClpXP can also generate 4Fe-FNR. So there is a cycle for this 3 FNR derivatives. All proteins sustain a degradation effect. Of cause, if there are too mang active FNR (4Fe-FNR), those active FNR will suppress the translating of fnr.</P><P>Now we combine the FNR cycle and the color reporter system together:</P><P>The FNR behave differently in aerobic and anaerobic condition. In the case that oxygen is absent, a pair of 4Fe-FNR fix on the green reporter DNA sequence and the green reporter gene is expressed, so we will observe this color for sample colonies. However a group of 4 4Fe-FNR comes to fix at red reporter gene and prevents the expression of red reporter protein. If oxygen is sufficient, so in aerobic condition, FNR derivatives keep in inactive form, there is no fixing of FNR on mRNA. The red reporter protein always produces the red protein but production of green reporter protein ceases.</P><H2><SPAN class="mw-headline" id="Transcribe_functional_description_into_equation_system"><B>Transcribe functional description into equation system</B></SPAN></H2><P>We transcribe biological informations into chemical reaction by simpliying the system which means that we chose to ignore some molecules(especially the enzymes) and focus on the IN and OUT of the system.</P><H2><SPAN class="mw-headline" id="Summary_for_equations:">Summary for equations:</SPAN></H2><P><UL><LI>Translation of fnr gene into FNR protein (FNR without Fe)
</LI><LI>Dimerisation of apoFNR (action of Ise protein)
</LI><LI>Oxydation: in aerobic conditions, oxygen inactivates FNR but cell continues to reactivate it (action of Ise protein)
</LI><LI>Degradation of each kind of FNR (FNR without Fe, FNR activated, and FRN inactivated), green and red proteins (action of ClpXP
</LI></UL>
protein)
<UL><LI>Activation/inactivation of a gene under a repressor promotor (green) and production or not of the cognate protein
</LI><LI>Activation/inactivation of a gene under an activator promotor (red) and production or not of the cognate protein
</LI></UL></P><H2><SPAN class="mw-headline" id="Details">Details</SPAN></H2><BLOCKQUOTE><UL><LI>Translating fnr to FNR 
</LI></UL><DL><DD> fnr mRNA generates steadily apoFNR(inactive)
</DD><DD><I>fnr -&gt; fnr + apoFNR </I></DD></DL><UL><LI>Dimerization of apoFNR
</LI></UL><DL><DD>2 apoFNRs form active 4Fe-FNR(QFeFNRa) with catalyzer Ise.
</DD><DD><I>apoFNR + apoFNR + Ise -&gt; QFeFNRa + Ise</I></DD></DL><UL><LI>Oxidation, from 4Fe-FNR to apoFNR(inactive)
</LI></UL><DL><DD> In aerobical condition, oxygen inactivates FNR, 4Fe-FNR generates to 2 inactive Fe-FNR(DFeFNRi)
</DD><DD><I>QFeFNRa + o2 -&gt; DFeFNRi +o2 </I></DD></DL><DL><DD> Then oxidize FNR from 2Fe-FNR to apoFNR
</DD><DD><I>DFeFNRi + o2 -&gt; apoFNR +o2 </I></DD></DL><DL><DD> 2Fe-FNR can be reactivate by Ise 
</DD><DD><I>DFeFNRi + DFeFNRi + Ise -&gt; QFeFNRa + Ise</I></DD></DL><UL><LI>Degradation FNR
</LI></UL><DL><DD>Protein FNR and its derivation submit a steady degrading rate
</DD><DD><I>apoFNR + ClpXP -&gt; ClpXP</I></DD><DD><I>DFeFNRi + ClpXP -&gt; ClpXP</I></DD><DD><I>QFeFNRa + ClpXP -&gt;ClpXP</I></DD></DL><UL><LI>Activation/inactivation of green reporter gene
</LI></UL><DL><DD> Association between 2 4Fe-FNR(active) and green reporter protein gene
</DD><DD><I>QFeFNRa + QFeFNRa + green_reporter_protein_gene -&gt; DQFeFNRa_green_binded</I></DD></DL><DL><DD> Dissociation between 4Fe-FNR(active) and green reporter protein gene
</DD><DD><I>DQFeFNRa_green_binded -&gt; QFeFNRa + QFeFNRa + green_reporter_protein_gene </I></DD></DL><DL><DD> Production of green reporter protein
</DD><DD><I>DQFeFNRa_green_binded -&gt; green_reporter_protein + DQFeFNRa_green_binded</I></DD></DL><UL><LI>Green protein degradation 
</LI></UL><DL><DD><I>green_reporter_protein + proteinase1 -&gt; proteinase1 </I></DD></DL><UL><LI>Repression of red protein geneÂ : Association between 4Fe-FNR(active) and red reporter protein gene
</LI></UL><DL><DD><I>QFeFNRa + QFeFNRa + red_reporter_pretein_gene -&gt; DQFeFNRa_red_binded</I></DD></DL><DL><DD> Dissociation between 4Fe-FNR(active) and red protein reporter gene
</DD><DD><I>DQFeFNRa_red_binded -&gt; QFeFNRa + QFeFNRa + red_reporter_pretein_gene </I></DD></DL><DL><DD>Association of 4 4Fe-FNR(active) binded, prevent the production of red reporter protein
</DD><DD><I>DQFeFNRa_red_binded + DQFeFNRa_red_binded -&gt; repression_red </I></DD></DL><DL><DD>Dissociation of 4 4Fe-FNR(active)
</DD><DD><I>repression_red -&gt; DQFeFNRa_red_binded + DQFeFNRa_red_binded </I></DD></DL><UL><LI>Expression the red gene in aerobic condition
</LI></UL><DL><DD> Production of red protein without repression
</DD><DD><I>DQFeFNRa_red_binded -&gt; DQFeFNRa_red_binded + red_reporter_protein </I></DD></DL><UL><LI>Degradation of red protein
</LI></UL><DL><DD><I>red_reporter_protein + proteinase2 -&gt; proteinase2 </I></DD></DL></BLOCKQUOTE><H2><SPAN class="mw-headline" id="Transcribe_equations_to_Hsim_code"><B>Transcribe equations to Hsim code</B></SPAN></H2><P>Thanks to the equations that we wrote, we have all the informations needed for Hsim. In the Hsim code, we simplify some equations by deleting some objets, Ise for example. Because Hsim has some limit for maximum element number in one rule, we have to declare some new objets such like FNRa_binded,FNRa_binded_green_protein. In the case that we want change speed of degradation, we give 3 different names for different proteinase.</P><P>You can check our source code <A href="https://2013.igem.org/Team:Paris_Saclay/Modeling/FNR/hsimcode" class="external text" rel="nofollow">here</A></P><H2><SPAN class="mw-headline" id="Set_of_parameters"><I>Set of parameters</I></SPAN></H2><P>In our model Hsim, Kinect of reaction is represented by probability of rules (probability that the reaction takes place). Those probability is the result of a ratio between the Km of 2 side of chemical equilibrium equation, however in our model there are lots of no reversal reaction, we have some trouble to deal with them, for e.g. the dimensions of some units. But somehow we can peep some reality, the article of Dean A.Tolla and Michael A.Savageau show us some relation among their system. For example, we can estimate which reaction is faster than others: the Kinect of oxidation for active FNR is much quick than dimerization of 2 inactive FNR. So what we did for those probability is a large amount of test.</P><P>We had some difficults in the beginning, the model did not accord with our estimation. However, we figured out at last some reliable combination of paremeters. And we have chosen one for our testing scenarios.</P><P>You can check our paremeters setting <A href="https://2013.igem.org/Team:Paris_Saclay/simulation#simulation.hsim" class="external text" rel="nofollow">here</A> in source code of second scenario.</P><H2><SPAN class="mw-headline" id="Scenarios_and_results"><B>Scenarios and results</B></SPAN></H2><P>Here we prepared 3 scenarios in order to testing our model. Each of them represent one specific condition which could happen in our experiments. The first scenario is about the degradation of red reporter protein in aerobic condition. In the second scenario, the model submit to aerobic then anaerobic condition.  The last one is a rare situation, we want to see how our model acts.</P><P>In the beginning of each scenario, we let our system run in the distribution mode for few generation, the molecules will spread homogeneously in the virtual bacterium.</P><DL><DD><UL><LI> Degradation of red reporter protein
</LI></UL></DD></DL><P>In this first scenario, we have a situation that the system has been in a anaerobic condition for a while, there is already some of red protein molecules. What we wish to see is a decline of red protein from initial value to 0, and it is what we observed in our model. There is also a latency due to the time needed for the formation of binding FNR molecules. The synthesis of green protein begins before oxygen running out.</P><DIV class="center"><DIV class="floatnone"><A href="/File:PSscenario1.gif" title="File:PSscenario1.gif">File:PSscenario1.gif</A></DIV></DIV><P>The red and green sphere represent respectively the red reporter protein and green reporter protein. 4 FNRa combine with 2 red reporter protein gene and form those large pink spheres. In our configuration, those combination is stable, once created, it is relatively hard to dissimilate them. This characteristic help the system represse the production of red reporter protein in anaerobic condition. </P><DL><DD><UL><LI> Alternation of reporter protein
</LI></UL></DD></DL><P>In this scenario, we changed the nature of oxygen from âmetaboliteâ to âmoleculeâ which means the quantity of oxygen in our system is no more constant, and distribution of oxygen become heterogeneous. We begin with 5000 molecules of oxygen, and 0 reporter protein. The red reporter protein is synthesized immediately. Oxygen is used to oxidant the FNR active to inactive state, however, when all oxygen is consumed, FNR n-mer begin to synthesize green protein and prevent red one. So we observe a change of dominance in the scheme. This scenario reveals a part of result in our experiments.</P><P><A href="/File:PSscenario2.gif" title="File:PSscenario2.gif">File:PSscenario2.gif</A></P><P>Here you can find our configuration files for HSIM: <A href="https://2013.igem.org/Team:Paris_Saclay/simulation#simulation.hsim" class="external text" rel="nofollow">link</A>.</P><DL><DD><UL><LI> Green reporter protein in aerobic condition
</LI></UL></DD></DL><P>This scenario is a little bizarre, we set at the presence of oxygen, there is still a lot of green reporter in the system. What will happen? It is not quite explicit. So how about a simulation, we amused ourtheves by testing some weird values. We try to find its limit.</P><P><A href="/File:PSscenario3.gif" title="File:PSscenario3.gif">File:PSscenario3.gif</A></P><P>And as we can see, after a period of latency, the system has at first systhesized red reporter protein, the number of green reporter protein goes down. But after the system consume all its oxygen, the red reporter protein begin to re dominate the cell.</P><H2><SPAN class="mw-headline" id="Online_simulation"><I>Online simulation</I></SPAN></H2><P>We used our homegrown wiki-integrated <A href="https://2013.igem.org/Team:Paris_Saclay/Modeling/Simulator#Wiki-integrated_simulator" class="external text" rel="nofollow">simulator</A> to provide an interactive way of seeing our model in action.
</P><H3><SPAN class="mw-headline" id="Simple_simulation"> Simple simulation </SPAN></H3><P>The first simulation presented here uses the following simplified modelÂ :
</P><BLOCKQUOTE><UL><LI>Effective inactivation of FNR by oxydation
</LI></UL><DL><DD><I>FNR + O2 -&gt; FNR_ox </I></DD></DL><UL><LI>Effective reactivation of FNR by reduction
</LI></UL><DL><DD><I>FNR_ox -&gt; FNR + O2 </I></DD></DL><UL><LI>Active FNR binds to the Pfnr activator
</LI></UL><DL><DD><I>FNR + Pfnr_activator -&gt; Pfnr_activator_bond</I></DD></DL><UL><LI>FNR detaches from the Pfnr activator
</LI></UL><DL><DD><I>Pfnr_activator_bond -&gt; FNR + Pfnr_activator</I></DD></DL><UL><LI>A green reporter protein (PV) is expressed when active FNR is bound to Pfnr_activator, so when the oxygen level is low.
</LI></UL><DL><DD><I>Pfnr_activator_bond -&gt; Pfnr_activator_bond + PV</I></DD></DL><UL><LI>Active FNR binds to the Pfnr repressor
</LI></UL><DL><DD><I>FNR + Pfnr_repressor -&gt; Pfnr_repressor_bond</I></DD></DL><UL><LI>FNR detaches from the Pfnr repressor
</LI></UL><DL><DD><I>Pfnr_repressor_bond -&gt; FNR + Pfnr_repressor</I></DD></DL><UL><LI>A red reporter protein (PV) is expressed when <B>no</B> active FNR is bound to Pfnr_repressor, so when the oxygen level is high.
</LI></UL><DL><DD><I>Pfnr_repressor -&gt; Pfnr_repressor + PR</I></DD></DL></BLOCKQUOTE><P>Now you just have to choose an oxygen level and push START in the simulator below.
</P><DIV class="simbox" data-width="785" data-height="300" data-load="https://2013.igem.org/Team:Paris_Saclay/modeling/fnr0.xml">Loading simulation...</DIV>
Get the <A href="https://2013.igem.org/Team:Paris_Saclay/modeling/fnr0.xml?action=raw&amp;ctype=text/css" class="external text" rel="nofollow">configuration file</A><H3><SPAN class="mw-headline" id="Complex_simulation"> Complex simulation </SPAN></H3><P>A more complete simulation is presented here, which takes into account the detail of the FNR oxydation and dimerization processes.
</P><P>FNR is the quantity of FNR monomers.
DFNRa represents the quantity of non-oxydated FNR dimers.
</P><DIV class="simbox" data-width="785" data-height="300" data-load="https://2013.igem.org/Team:Paris_Saclay/modeling/fnr1.xml">Loading simulation...</DIV><P>Get the <A href="https://2013.igem.org/Team:Paris_Saclay/modeling/fnr1.xml?action=raw&amp;ctype=text/css" class="external text" rel="nofollow">configuration file</A></P><H2><SPAN class="mw-headline" id="Conclusion"><B>Conclusion</B></SPAN></H2><P>Our model successfully revealed the different states of FNR regulation. As you can see, in a anaerobic condition, the number of FNR active is significantly raised, so do the number of green reporter protein. In aerobic condition, because of the presence of oxygen, FNR inactive dominates in the model, the production of green protein ceases so the number of green protein drops, at the same time the repression for the red reporter disappear leading to the restart of the red protein production.</P><P>Back to our wet experiments, we used LacZ (blue color BBa_K11550007) and AmilCP (pink color BBa_K11550003) as 2 independent expression reporters which correspond to green reporter protein in our model. The results of modeling for green reporter protein conform to our experiments (PS: green reporter protein in our model correspond to LacZ and AmilCP, there is a deviation). And we had not perform the red reporters protein in the wet experiments.</P><H2><SPAN class="mw-headline" id="Reference"><I>Reference</I></SPAN></H2><UL><LI>Wenmao Meng, Jeffrey Green and John R.Guest FNR-dependent repression of ndh gene expression requires two upstream FNR-binding sites Microbiology (1997), 143, 1521-1532
</LI><LI>Parick Amar, Gilles Bernot, Victor Norris HSIM: a simulation programme to study large assemblies of proteins
</LI></UL><P>Article written by Zhou and Damir

</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Paris_Saclay/Modeling/FNR">http://2013.igem.org/Team:Paris_Saclay/Modeling/FNR</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Saclay/Modeling/FNR" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Saclay/Modeling/FNR" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Saclay/Modeling/FNR&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Saclay/Modeling/FNR&amp;oldid=314491" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>