"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Saclay/Modeling/FNR
From 2013.igem.org
Home
Team
Presentation
Official profile
Attributions
Sponsors
Extras
Project
What are PCBs ?
Overview
Modeling
Overview
Simulator
Oxygen sensor
PS-PCR
Overview
Detailed description
Achievements
Labwork
Overview
Biobricks
Protocols
Primers
Safety
Notebook
Human practices
Reflection on the project
Reflection
Poll
Sewers of Paris
Reflection on open source
Intro
Reflection
Interview
Outreach
Team collaboration
Project What are PCBs ? Overview Modeling PS-PCR Achievements
Modeling Overview Simulator Oxygen sensor
Contents
1 FNR oxygen sensor system
1.1 Translate biological knowledges into functional scheme
1.2 Transcribe functional description into equation system
1.3 Summary for equations:
1.4 Details
1.5 Transcribe equations to Hsim code
1.6 Set of parameters
1.7 Scenarios and results
1.8 Online simulation
1.8.1 Simple simulation
1.8.2 Complex simulation
1.9 Conclusion
1.10 Reference
FNR oxygen sensor system
As we use Hsim for the modeling, a multi-agents programme developped at our university in order to follow the assembly, movements and dissociation of a large number of molecules in a virtual cell. It replaces the Ordinary Differential Equations and mimics chemical reactions of the real system by rewriting specials rules. If you want to dig in our model we suggest you to read this page , written by Mr. Patrick Amar who is the developer of Hsim. .
Roughly speaking, in our model we define a sampling volume, size in nanometer and we declare some molecules with their sizes, their moving speed and their number. Once the simulation starts, those molecules move randomly in the space and trigger some possible reaction between them with some probabilities. There is always somehow deviation from the reality, but we are all excited about this particular simulating modeling.
We construct our FNR regulator model by 3 steps.At first, We extract the biological information, draw some schemes for helping us to understand the biological. Then we translate those biological concept into specials equations or pseudo code of Hsim. Finally, we write the Hsim code and run it in different scenarios for the test. We also have developed a web similator which allows user change and test our model with their own parameter.
Translate biological knowledges into functional scheme
Tolla and Savageau Regulation of Aerobic to Anaerobic Transitions by the FNR cycle in Escherichia coli
This scheme from the Dean A.Tolla and Michael A.Savageauthe's paper shows clearly the FNR transformation cycle. After the fnr translation, apoFNR(inactive) is produced. And the dimerization of 2 apoFNR which id catalyzed by ClpXP create the active 4Fe-FNR. The 4Fe-FNR can be oxidized by oxygen to 2Fe-FNR then reduce to apoFNR with another oxygen. 2Fe-FNR plus ClpXP can also generate 4Fe-FNR. So there is a cycle for this 3 FNR derivatives. All proteins sustain a degradation effect. Of cause, if there are too mang active FNR (4Fe-FNR), those active FNR will suppress the translating of fnr.
Now we combine the FNR cycle and the color reporter system together:
The FNR behave differently in aerobic and anaerobic condition. In the case that oxygen is absent, a pair of 4Fe-FNR fix on the green reporter DNA sequence and the green reporter gene is expressed, so we will observe this color for sample colonies. However a group of 4 4Fe-FNR comes to fix at red reporter gene and prevents the expression of red reporter protein. If oxygen is sufficient, so in aerobic condition, FNR derivatives keep in inactive form, there is no fixing of FNR on mRNA. The red reporter protein always produces the red protein but production of green reporter protein ceases.
Transcribe functional description into equation system
We transcribe biological informations into chemical reaction by simpliying the system which means that we chose to ignore some molecules(especially the enzymes) and focus on the IN and OUT of the system.
Summary for equations:
Translation of fnr gene into FNR protein (FNR without Fe)
Dimerisation of apoFNR (action of Ise protein)
Oxydation: in aerobic conditions, oxygen inactivates FNR but cell continues to reactivate it (action of Ise protein)
Degradation of each kind of FNR (FNR without Fe, FNR activated, and FRN inactivated), green and red proteins (action of ClpXP
protein)
Activation/inactivation of a gene under a repressor promotor (green) and production or not of the cognate protein
Activation/inactivation of a gene under an activator promotor (red) and production or not of the cognate protein
Details
Translating fnr to FNR
fnr mRNA generates steadily apoFNR(inactive)
fnr -> fnr + apoFNR
Dimerization of apoFNR
2 apoFNRs form active 4Fe-FNR(QFeFNRa) with catalyzer Ise.
apoFNR + apoFNR + Ise -> QFeFNRa + Ise
Oxidation, from 4Fe-FNR to apoFNR(inactive)
In aerobical condition, oxygen inactivates FNR, 4Fe-FNR generates to 2 inactive Fe-FNR(DFeFNRi)
QFeFNRa + o2 -> DFeFNRi +o2
Then oxidize FNR from 2Fe-FNR to apoFNR
DFeFNRi + o2 -> apoFNR +o2
2Fe-FNR can be reactivate by Ise
DFeFNRi + DFeFNRi + Ise -> QFeFNRa + Ise
Degradation FNR
Protein FNR and its derivation submit a steady degrading rate
apoFNR + ClpXP -> ClpXP
DFeFNRi + ClpXP -> ClpXP
QFeFNRa + ClpXP -> ClpXP
Activation/inactivation of green reporter gene
Association between 2 4Fe-FNR(active) and green reporter protein gene
QFeFNRa + QFeFNRa + green_reporter_protein_gene -> DQFeFNRa_green_binded
Dissociation between 4Fe-FNR(active) and green reporter protein gene
DQFeFNRa_green_binded -> QFeFNRa + QFeFNRa + green_reporter_protein_gene
Production of green reporter protein
DQFeFNRa_green_binded -> green_reporter_protein + DQFeFNRa_green_binded
Green protein degradation
green_reporter_protein + proteinase1 -> proteinase1
Repression of red protein gene : Association between 4Fe-FNR(active) and red reporter protein gene
QFeFNRa + QFeFNRa + red_reporter_pretein_gene -> DQFeFNRa_red_binded
Dissociation between 4Fe-FNR(active) and red protein reporter gene
DQFeFNRa_red_binded -> QFeFNRa + QFeFNRa + red_reporter_pretein_gene
Association of 4 4Fe-FNR(active) binded, prevent the production of red reporter protein
DQFeFNRa_red_binded + DQFeFNRa_red_binded -> repression_red
Dissociation of 4 4Fe-FNR(active)
repression_red -> DQFeFNRa_red_binded + DQFeFNRa_red_binded
Expression the red gene in aerobic condition
Production of red protein without repression
DQFeFNRa_red_binded -> DQFeFNRa_red_binded + red_reporter_protein
Degradation of red protein
red_reporter_protein + proteinase2 -> proteinase2
Transcribe equations to Hsim code
Thanks to the equations that we wrote, we have all the informations needed for Hsim. In the Hsim code, we simplify some equations by deleting some objets, Ise for example. Because Hsim has some limit for maximum element number in one rule, we have to declare some new objets such like FNRa_binded,FNRa_binded_green_protein. In the case that we want change speed of degradation, we give 3 different names for different proteinase.
You can check our source code here
Set of parameters
In our model Hsim, Kinect of reaction is represented by probability of rules (probability that the reaction takes place). Those probability is the result of a ratio between the Km of 2 side of chemical equilibrium equation, however in our model there are lots of no reversal reaction, we have some trouble to deal with them, for e.g. the dimensions of some units. But somehow we can peep some reality, the article of Dean A.Tolla and Michael A.Savageau show us some relation among their system. For example, we can estimate which reaction is faster than others: the Kinect of oxidation for active FNR is much quick than dimerization of 2 inactive FNR. So what we did for those probability is a large amount of test.
We had some difficults in the beginning, the model did not accord with our estimation. However, we figured out at last some reliable combination of paremeters. And we have chosen one for our testing scenarios.
You can check our paremeters setting here in source code of second scenario.
Scenarios and results
Here we prepared 3 scenarios in order to testing our model. Each of them represent one specific condition which could happen in our experiments. The first scenario is about the degradation of red reporter protein in aerobic condition. In the second scenario, the model submit to aerobic then anaerobic condition.  The last one is a rare situation, we want to see how our model acts.
In the beginning of each scenario, we let our system run in the distribution mode for few generation, the molecules will spread homogeneously in the virtual bacterium.
Degradation of red reporter protein
In this first scenario, we have a situation that the system has been in a anaerobic condition for a while, there is already some of red protein molecules. What we wish to see is a decline of red protein from initial value to 0, and it is what we observed in our model. There is also a latency due to the time needed for the formation of binding FNR molecules. The synthesis of green protein begins before oxygen running out.
File:PSscenario1.gif
The red and green sphere represent respectively the red reporter protein and green reporter protein. 4 FNRa combine with 2 red reporter protein gene and form those large pink spheres. In our configuration, those combination is stable, once created, it is relatively hard to dissimilate them. This characteristic help the system represse the production of red reporter protein in anaerobic condition.
Alternation of reporter protein
In this scenario, we changed the nature of oxygen from “metabolite” to “molecule” which means the quantity of oxygen in our system is no more constant, and distribution of oxygen become heterogeneous. We begin with 5000 molecules of oxygen, and 0 reporter protein. The red reporter protein is synthesized immediately. Oxygen is used to oxidant the FNR active to inactive state, however, when all oxygen is consumed, FNR n-mer begin to synthesize green protein and prevent red one. So we observe a change of dominance in the scheme. This scenario reveals a part of result in our experiments.
File:PSscenario2.gif
Here you can find our configuration files for HSIM: link .
Green reporter protein in aerobic condition
This scenario is a little bizarre, we set at the presence of oxygen, there is still a lot of green reporter in the system. What will happen? It is not quite explicit. So how about a simulation, we amused ourtheves by testing some weird values. We try to find its limit.
File:PSscenario3.gif
And as we can see, after a period of latency, the system has at first systhesized red reporter protein, the number of green reporter protein goes down. But after the system consume all its oxygen, the red reporter protein begin to re dominate the cell.
Online simulation
We used our homegrown wiki-integrated simulator to provide an interactive way of seeing our model in action.
Simple simulation
The first simulation presented here uses the following simplified model :
Effective inactivation of FNR by oxydation
FNR + O2 -> FNR_ox
Effective reactivation of FNR by reduction
FNR_ox -> FNR + O2
Active FNR binds to the Pfnr activator
FNR + Pfnr_activator -> Pfnr_activator_bond
FNR detaches from the Pfnr activator
Pfnr_activator_bond -> FNR + Pfnr_activator
A green reporter protein (PV) is expressed when active FNR is bound to Pfnr_activator, so when the oxygen level is low.
Pfnr_activator_bond -> Pfnr_activator_bond + PV
Active FNR binds to the Pfnr repressor
FNR + Pfnr_repressor -> Pfnr_repressor_bond
FNR detaches from the Pfnr repressor
Pfnr_repressor_bond -> FNR + Pfnr_repressor
A red reporter protein (PV) is expressed when no active FNR is bound to Pfnr_repressor, so when the oxygen level is high.
Pfnr_repressor -> Pfnr_repressor + PR
Now you just have to choose an oxygen level and push START in the simulator below.
Loading simulation...
Get the configuration file
Complex simulation
A more complete simulation is presented here, which takes into account the detail of the FNR oxydation and dimerization processes.
FNR is the quantity of FNR monomers. DFNRa represents the quantity of non-oxydated FNR dimers.
Loading simulation...
Get the configuration file
Conclusion
Our model successfully revealed the different states of FNR regulation. As you can see, in a anaerobic condition, the number of FNR active is significantly raised, so do the number of green reporter protein. In aerobic condition, because of the presence of oxygen, FNR inactive dominates in the model, the production of green protein ceases so the number of green protein drops, at the same time the repression for the red reporter disappear leading to the restart of the red protein production.
Back to our wet experiments, we used LacZ (blue color BBa_K11550007) and AmilCP (pink color BBa_K11550003) as 2 independent expression reporters which correspond to green reporter protein in our model. The results of modeling for green reporter protein conform to our experiments (PS: green reporter protein in our model correspond to LacZ and AmilCP, there is a deviation). And we had not perform the red reporters protein in the wet experiments.
Reference
Wenmao Meng, Jeffrey Green and John R.Guest FNR-dependent repression of ndh gene expression requires two upstream FNR-binding sites Microbiology (1997), 143, 1521-1532
Parick Amar, Gilles Bernot, Victor Norris HSIM: a simulation programme to study large assemblies of proteins
Article written by Zhou and Damir
Retrieved from " http://2013.igem.org/Team:Paris_Saclay/Modeling/FNR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
