<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Part_Collection skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Part Collection</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><NAV><UL class="nav-list"><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec">Home</A></LI><LI><A href="#!">Project</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Motivation">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Description">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Theory">Toxicity</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation">Accumulation</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA">Silencing</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Ferritin">Nanoparticles</A></LI></UL><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="#!">Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Results">Results Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Results">Toxicity Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation_Results">Accumulation Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA_Results">Silencing Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Ferritin_Results">Nanoparticles Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate</A></LI><LI><A href="#!">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Parts Collection</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Improve">Parts Improvement</A></LI><LI><A href="#!">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Public_Engagement">Outreach</A></LI><LI><A href="#!">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Design">Product Design</A></LI><LI><A href="#!">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Experiments">Experiments</A></LI><LI><A href="#!">Team</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Bielefeld">Bielefeld</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/CeBiTec">CeBiTec</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Bielefeld-University">Bielefeld University</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=Bielefeld-CeBiTec">Judging Form ⇗</A></LI></NAV></DIV><DIV class="container"><DIV class="main_content"><DIV class="title">Part Collection</DIV><H2>Short Summary</H2><ARTICLE>

In synthetic biology the control of transcription and translation is of enormous importance. Therefore, promoters and ribosome binding sites (RBS) play a central role in each iGEM project. Choosing the optimal promoter and RBS combination for a gene of interest can be crucial, since small changes in the protein expression level can lead to large changes in the resulting effect inside synthetic gene circuits. To address the challenge of choosing the right promoter, we designed a promoter-RBS library as this year’s parts collection as well as a suitable measurement system to analyze the expression strength of the chosen promoter-RBS combination. With our measurement vector the library could be easily expanded by future iGEM teams and the results are comparable due to normalization of the measured signal to a second reporter protein. We submitted our designed vector (<A href="http://parts.igem.org/Part:BBa_K2638560">BBa_K2638560</A>) to assess the promoter-RBS combination expression strength accurately, based on two reporter genes. 
Our collection contains a variety of iGEM standard promoters like the Anderson promoter library, as well as inducible promoters. This collection is integrated in our whole project. We tested all of our promoter-RBS combinations which are important for different parts of our project. By combining different RBS and promoters, the individual strength of the RBS and promoter parts can be checked, too. 
With our part collection we improved our <A href="http://parts.igem.org/Promoters/Catalog/Anderson">Anderson promoter library</A>, which offers the probability to choose the strength of a knock-down using a specific promoter. Furthermore, we used the promoter-RBS combination measurement to determine the optimal expression level of our <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation">membrane proteins</A> and our <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Theory">anti-toxicity</A> project.. To sum up, we analyzed 26 promoter-RBS combinations, modeled 37 more and therefore provided the iGEM community with detailed information regarding their future projects. In addition, we designed a database that allows us to easily find a promoter or promoter-RBS combination. If you want to express a slightly toxic protein, for example, you can find a weak combination. If you are looking for a suitable expression system for your reporter gene, you can choose the optimal strength with the help of our data.





</ARTICLE><H2>Design</H2><ARTICLE>
Analyzing the expression strength of individual promoter-RBS combinations is quite challenging. The main reasons hindering accurate promoter-RBS characterization, are fluctuating copy-number changes of the expression plasmid (Jahn, M. et al,2016) or growth phase specific expression changes due to effects of sigma factors (Bervoets, I. <I>et al.</I>, 2018). To avoid these errors, we designed a measurement vector carrying two reporter genes, which enables us to normalize the expression strength of the measured promoter-RBS combination to a constant expression level of the reference reporter gene. In this way, the effect of a varying plasmid copy number inside the cells can be taken into account.. Our measurement vector is based on the expression strength of the different promoter-RBS combinations from our library cloned upstream of <I>mRFP</I> and a double terminator (BBa_K2638426) into the pSB1C3 BioBrick site. Furthermore, our measurement vector carries a eCFP (<A href="http://parts.igem.org/Part:BBa_E0022">BBa_E0022</A>) under control of a strong/weak Anderson promoter (<A href="http://parts.igem.org/Part:BBa_J23100">BBa_J23100</A>) and the RBS <A href="http://parts.igem.org/Part:BBa_J61100">BBa_J61100</A> followed by a double terminator (Fig. 1). Thus, the constitutive eCFP expression is proportional to the plasmid’s copy-number. This enables a normalization of the <I>mRFP</I> expression to the plasmid’s copy-number and direct assessment of our library’s promoter-RBS combinations expression strength. As this measurement is independent of plasmid effects it enables comparison with our modeling as well as with other expression constructs.

</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 1:</B> Map of Biobrick <A href="http://parts.igem.org/Part:BBa_K2638560">BBa_K2638560</A>. The vektor of our measurement system with changeable promoterand RBS between prefix and suffix.
                      </FIGCAPTION></FIGURE><ARTICLE>
Due to the DNA submission requirements, we choose the reporter gene <I>mRFP</I> as insert, to enable successful submission of our designed and constructed plasmid backbone. It also enables a quick and easy control if the cloning was successful. In regard to the use of a second fluorophore, the emission and absorption spectra should not interact with each other. Therefore, we did not use GFP, one of the most used reporters in iGEM. The use of GFP and mRFP would enable a FRET (Foerster Resonance Transfer)(Bajar, B. T. <I>et al</I>, 2016), and thus an interference regarding the detection signal. Since the difference between the absorption spectrum of mRFP and the emission of the eCFP is larger than the distance between the emission of GFP and the absorption of mRFP, we chose eCFP and mRFP as a good combination for our purposes Thus, eCFP and mRFP fluorescence should be detectable at the same time without any interferences.
</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 3:</B> Emission and absorption spectrum of GFP, CFP and RFP. The dashed line shows the emission and the solid line shows the absortion. Picture from <A href="https://www.thermofisher.com/de/de/home/life-science/cell-analysis/labeling-chemistry/fluorescence-spectraviewer.html">Thermo Fisher fluorescence spectraviewer</A></FIGCAPTION></FIGURE><H2>Modeling</H2><ARTICLE>
For the modeling of our promoter-RBS combinations we used the given strengths of the Anderson promoters BBa_J23119, BBa_J23100 to BBa_J23110) and the strengths of different RBS (BBa_J61100, BBa_B0030, BBa_B0031) to determine an estimate for their absolute strength.
Prior to the experimental validation, we modeled the expression strength of different promoter and RBS combinations to create a database for our further experiments. Therefore, we used the given strength of the Anderson promoters and the strength of the different known RBS to determine and visualize their absolute strength shown in Figure 2. Especially for our siRNA system, it was interesting to see the differences between inducible and constitutive promoters.


</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 2:</B> Modeling of the expression strength different the Anderson promoters (BBa_J23119,BBa_J23100 to BBa_J23110) in combination with three different RBS (BBa_J61100, BBa_B0030, BBa_B0031) respectively.
                      </FIGCAPTION></FIGURE><ARTICLE>
The visualization of the modeled expression strength is shown in Figure 2. The expression is influenced by the different used RBS, which are indicated with different colors. The modeling shows a significant influence of the different RBS on the expression strength, independent from the use of different promoters.  In theory, the RBS has a stronger influence of the expression strength, than the promoter, which only influences the transcription, while the RBS influences the translation. When the J61100 RBS is used, the expression strength of the construct is statistically larger (approximately eight times higher) than in the other modeled RBS. The modeling shows a relatively small influence on the expression strength whether the RBS B0030 or B0031 is used.
</ARTICLE><H2>Results</H2><ARTICLE>
In addition to the modeled expression strengths we also tested the influence of gene expression on the bacterial growth by the measurement of the OD<SUB>600</SUB>. After cloning, all parts were checked by Sanger sequencing. The correct plasmids containing the different promoter-RBS combinations were transformed into <I>E. coli</I> DH5α and grown in LB media. 5mL cultures were inoculated at OD<SUB>600</SUB> of 0.1 and incubated at 37°C and 300 rpm. After one hour, the fluorescence signals of mRFP and eCFP were measured using the Tecan Reader with excitation wavelengths of 558 nm and 435 nm and emission detection at 608 nm and 485 nm respectively. In a second experiment the samples were cultivated under the same conditions over 14 hours, showing similar results. The fluorescence signals of the cells with the plasmid containing the Anderson promoter BBa_J23119 encoding the <I>E. coli</I> consensus promoter was set as reference for the analysis of the expression strength of all other RBS promoter combinations.

</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 3:</B> Results of the expression strength analysis of different Anderson promoters (BBa_J23119, and BBa_J23100 to BBa_J23110) in combination with different RBS (BBa_J61100, BBa_B0030, BBa_B0031). The plasmid BBa_K2638560 was used for the expression of the fluorescence protein mRFP under the control of the mentioned promoter-RBS combinations. The fluorescence signals were normalized to the fluorescence of eCFP of the corresponding cells. The measured values were also normalized to the measured OD<SUB>600</SUB>. The relative strength regarding the the <I>E. coli</I>consensus promoter BBa_J23119 is shown. The known promoter strength of the Anderson collection is listed in brackets. 
                      </FIGCAPTION></FIGURE><ARTICLE>
The analyzed expression strengths of mRFP under control of the different promoters-RBS combinations are shown in Figure 3. The relative expression level is plotted regarding the <I>E. coli</I> consensus promoter BBa_J23119 The different RBS are shown in different colors. The Anderson promoter BBa_J23119 with different RBS was set as reference, and is therefore shown with a relative expression level of 1.0. All other combinations are compared and referenced to the consensus promoter construct with the corresponding RBS. The strongest difference between the expression level determined by Anderson et al. and our analyzed expression level could be observed for the BBa_J23100 promoter. Its expression level was expected to be as strong as the BBa_J23119 promoter, used as basis. But the measured relative expression level showed a significant lower expression of 0.47. This variation of the stated and analyzed relative expression level shows the importance of a validation of different promoter strengths and their combinations with different RBS. Using the BBa_B0030, described as a strong RBS, the relative expression level of the fluorophore of most of the analyzed constructs is higher than when combined to the RBS J23119 or B0031, except for promoters with a relatively low expression level. Our experiments show, that the relative expression level of the constructs is mainly influenced by the choice of RBS. That is why our measurement system is of such importance for the analysis of promoters and RBS.
</ARTICLE><TABLE id="t01" class="centern"><ARTICLE><CAPTION><B>Table 1: </B>Results of the Anderson promoter (BBa_J23119,BBa_J23100 to BBa_J23110)in combination with the RBS (BBa_J61100, BBa_B0030, BBa_B0031). </CAPTION></ARTICLE><TBODY><TR><TH>Name</TH><TH>Sequence</TH><TH>Measured Strength RBS J61100</TH><TH>Measured Strength RBS B0030</TH><TH>Measured Strength RBS B0031</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23119">BBa_J23119</A></TD><TD>ttgacagctagctcagtcctaggtataatgctagc</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23100">BBa_J23100</A></TD><TD>ttgacggctagctcagtcctaggtacagtgctagc</TD><TD>0.41631</TD><TD>0.47372</TD><TD>0.40999</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A></TD><TD>tttacagctagctcagtcctaggtattatgctagc</TD><TD>0.41057</TD><TD>0.38392</TD><TD>0.35113</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23102">BBa_J23102</A></TD><TD>ttgacagctagctcagtcctaggtactgtgctagc</TD><TD>0.32899</TD><TD>0.55225</TD><TD>0.49386</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23103">BBa_J23103</A></TD><TD>ctgatagctagctcagtcctagggattatgctagc</TD><TD>0.05543</TD><TD>0.07914</TD><TD>nd</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23104">BBa_J23104</A></TD><TD>ttgacagctagctcagtcctaggtattgtgctagc</TD><TD>0.66601</TD><TD>0.84445</TD><TD>0.66691</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23105">BBa_J23105</A></TD><TD>tttacggctagctcagtcctaggtactatgctagc</TD><TD>0.11405</TD><TD>0.00532</TD><TD>0.07979</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23106">BBa_J23106</A></TD><TD>tttacggctagctcagtcctaggtatagtgctagc</TD><TD>0.18257</TD><TD>0.2062</TD><TD>0.15317</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23107">BBa_J23107</A></TD><TD>tttacggctagctcagccctaggtattatgctagc</TD><TD>0.05682</TD><TD>0.01321</TD><TD>0.02459</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23108">BBa_J23108</A></TD><TD>ctgacagctagctcagtcctaggtataatgctagc</TD><TD>0.16366</TD><TD>0.25364</TD><TD>0.17165</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23109">BBa_J23109</A></TD><TD>tttacagctagctcagtcctagggactgtgctagc</TD><TD>0.00928</TD><TD>0.01173</TD><TD>nd</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23110">BBa_J23110</A></TD><TD>tttacggctagctcagtcctaggtacaatgctagc</TD><TD>0.29643</TD><TD>0.29876</TD><TD>0.29423</TD></TR></TBODY></TABLE><H2>Outlook</H2><ARTICLE>
With this database we can predict the right Promoter and RBS for every system. We have weak combinations for proteins which are, for example, slightly toxic for the cells or cause stress responses. On the other hand, we have strong combinations for the expression of anti-toxicity or membrane proteins. During our experiments we realized, that protein expression is influenced by several aspects. Besides plasmid copy-number variations and influences regarding growth, also mRNA and protein stability should be considered. We have found, that especially protein half-life of the fluorescence proteins is a problem during expression strength analyses. Especially for weak promoters and RBS accumulation of the protein to be measured has a disturbing effect on the expression analysis. Therefore, we thought about alternative methods to measure gene expression. In this context, we thought that the quantification of transcripts would be the best method to measure real transcription rates. This approach is independent of protein accumulating effects. Only mRNA degradation has to be considered. Fast sampling would be needed to analyze the real time status of the cells transcriptional landscape. Hence, we decided to perform quantitative real-time PCRs (qRT-PCR) to measure the transcript amount and thus the real promoter strength. Of course, it is no longer possible to take into account the strength of the RBS in this way. Unfortunately, due to time limitations we were not able to carry out the real-time PCR experiments for all of our constructs. But it would be a suitable method for future iGEM teams to analyze their promoters using our designed vector system with qRT-PCR. For this, both the transcript amount of mRFP and of eCFP as a reference should be determined. Using eCFP as a reference again would enable the measurement of a promoter strength independent of the plasmid copy number and the growth phase. 

</ARTICLE><BUTTON onclick="myFunction()" class="refbtn"> References ▾</BUTTON><DIV id="myDIV" class="reftext"><B>De Mey, M., Maertens, J., Lequeux, G. J., Soetaert, W. K., &amp; Vandamme, E. J. (2007). </B>Construction and model-based analysis of a promoter library for E. coli: an indispensable tool for metabolic engineering. BMC biotechnology, 7(1), 34.
<B>Ipsaro, J. J., &amp; Joshua-Tor, L. (2015).</B> From guide to target: molecular insights into eukaryotic RNA-interference machinery. Nature structural &amp; molecular biology, 22(1), 20.
<B>Jahn, M., Vorpahl, C., Hübschmann, T., Harms, H., &amp; Müller, S. (2016). </B>  Copy number variability of expression plasmids determined by cell sorting and Droplet Digital PCR. Microbial cell factories, 15(1), 211.
<B>Kannan, S., Sams, T., Maury, J., &amp; Workman, C. T. (2018). </B>  Reconstructing dynamic promoter activity profiles from reporter gene data. ACS synthetic biology, 7(3), 832-841.
<B>Köker, T., Fernandez, A., &amp; Pinaud, F. (2018). </B>  Characterization of Split Fluorescent Protein Variants and Quantitative Analyses of Their Self-Assembly Process. Scientific reports, 8(1), 5344.
<B>Rizzo, M. A., Springer, G. H., Granada, B., &amp; Piston, D. W. (2004). </B>  An improved cyan fluorescent protein variant useful for FRET. Nature biotechnology, 22(4), 445.
<B>Rudge, T. J., Brown, J. R., Federici, F., Dalchau, N., Phillips, A., Ajioka, J. W., &amp; Haseloff, J. (2016). </B>  Characterization of intrinsic properties of promoters. ACS synthetic biology, 5(1), 89-98.
<B>Bajar, B. T., Wang, E. S., Zhang, S., Lin, M. Z., &amp; Chu, J. (2016).</B>  A guide to fluorescent protein FRET pairs. Sensors, 16(9), 1488.
              
                   




              
                   </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>