Team:Bielefeld-CeBiTec/Design
Home
Project
Motivation
Overview
Toxicity
Accumulation
Silencing
Nanoparticles
Modeling
Results
Results Overview
Toxicity Results
Accumulation Results
Silencing Results
Nanoparticles Results
InterLab
Demonstrate
Parts
Parts Overview
Basic Parts
Composite Parts
Parts Collection
Parts Improvement
Human Practices
Human Practices
Outreach
Awards
Hardware
Software
Measurement
Product Design
Notebook
Notebook
Experiments
Team
Members
Bielefeld
CeBiTec
Bielefeld University
Collaborations
Attributions
Partners
Judging Form ⇗
Introduction
Testing the system
Usage protocol
Product Design - The TACE System as open source alternative to CRISPR/Cas
Introduction
During our projekt we designed out TACE system. A new open source and easy to use Tool for all iGEM Teams  that wish to perform genome editing , that does not rely on CRISPR/Cas and has the potential to rise as a new standard among iGEM techniques.
CRISPR/Cas has become a standard in Genome editing techniques. But there are several Problems with the system that are currently being worked on, like off-target events that can lead to mutations and rearrangements of the genome. (Fu et al.,2013) While knockouts are invaluable for genome editing, they can also lead to Problems when important Household Genes are concerned as knockouts can greatly diminish growth rates. There is a range of knockouts that can not be performed as they would be lethal to the cell. Because CRISPR/Cas is a commercial product it can create legal difficulties to share Parts or CRISPR systems, which contradicts the open source mindset of the iGEM contest. Additionally, some iGEM Laboratories with insufficient funds might not have access to CRISPR.
To open up new possibilities and design a system that can be spread through the iGEM network we want to introduce an alternative approach to genome editing. Instead of knocking out genes at a DNA level, we want to silence genes by knocking down the transcribed mRNA introducing a whole new set of possibilities. Not only can we inducibly knock down genes this way, multiple siRNAS can be used to shift an organisms metabolism to comply with personal needs. As an example, a knockout required for the fabrication of a biotechnological product needs a knockout to be able to form that product, but this knockout extremely slows the growth of the organism. Using our System the cells can grow to a desired optical density or cell number. When a certain condition is met, the knockdown can be induced to start the improved production.
This is why we developed our TACE system . For further informations visit our Tace project page .
We designed this system to test as many siRNAs as possible with a high sample throughput. To achieve this, we constructed expression vectors which allow comparable expressions of different siRNAs, as well as a target vector which grants easy measurement of the silencing effectivity of a given siRNA.   It is important to make all expressed siRNAs comparable to each other. Therefore we designed four Biobricks featuring the same promoter to establish the same expression rate for all siRNAs. As a first Generation, we designed the four Biobricks BBa_K2638701 , BBa_K2638702 , BBa_K2638758 and BBa_K2638759 (Figure 1-4) for our expression vectors. They contain a Golden Gate Assembly (GGA) cassette which can be cut out using the restriction enzyme BbsI. So it is possible to replace the whole GGA cassette with a specific siRNA with the usage of our GGA protocol . The transcription of the siRNA is terminated by the strong Terminator BBa_B0015 to make sure that all expression processes are completely terminated. While the Biobrick BBa_K2638701 (Figure 1) is supposed to transcribe an siRNA without further modifications, the Biobrick BBa_K2638702 (Figure 2) also includes the Hfq binding sequence originating of the MicC-siRNA (Chen et al., 2004).  The Biobrick BBa_K2638758 (Figure 3) contains the 5’-UTR from ompA as a protective sequence upstream of the siRNA insertion site and the Biobrick BBa_K2638759 (Figure 4) with both features surrounding the insertion site as well as sequences combined to add further functions to the siRNA.
Figure 1: Illustration of Biobrick BBa_K2638701 , an expression Vector for siRNAs. The Biobrick includes a Golden Gate cassette which can be cut out using the BbsI restriction enzyme to seamlessly fuse an siRNA into the Vector.
Figure 2: Illustration of Biobrick BBa_K2638702 , an expression Vector for siRNAs. The Biobrick includes a Golden Gate cassette which can be cut out using the BbsI restriction enzyme to seamlessly fuse an siRNA to the Hfq binding Sequence inside the expression Vector.
Figure 3: Illustration of Biobrick BBa_K2638758 , an expression Vector for siRNAs. The Biobrick includes a golden Gate Cassette which can be cut out using the BbsI restriction enzyme. This seamlessly fuse an siRNA into the expression Vector which contains the ompA 5’- untranslated region (5’ UTR) upstream of the site of insertion. The ompA 5’ UTR acts as a protective sequence to protect an siRNA from 5’-dependend degradation.
Figure 4: Illustration of Biobrick BBa_K2638759 , an expression Vector for siRNAs. The Biobrick includes a Golden Gate cassette which can be cut out using the BbsI restriction enzyme. This seamlessly fuse an siRNA into the expression Vector which contains the ompA 5’- untranslated region (UTR)upstream, and the Hfq binding Sequence downstream of the site of siRNA insertion. The ompA 5’ UTR acts as a protective sequence to protect an siRNA from 5’-dependend degradation.
Therefore, we can choose one of these explained expression vectors to be the first part of the complete TACE-system. The second part of our TACE  system is a target vector, pTale, which transcribes one specific mRNA. The chosen mRNA should be silenced by the constructed siRNAs using one of the described Biobricks above. To get the optimal conditions for measuring the silencing effect of our siRNA we scheduled and constructed two Generations of target vectors. For the first generation of the target vectors we used two different Reporter Proteins: the chromoprotein AmilCp (BBa_K592009) and the blue fluorescent protein BFP (BBa_K592100). Additionaly feature the target Biobrick a linker between the GGA cassette and the reporter protein. This way, the inserted target mRNA forms a CDS fusion with the reporter protein without losing any function. If the mRNA is destroyed, no reporter protein is formed. This results in no measurement of the BFP fluorescents which is proportional to the silencing strength of the siRNA. The Target Vectors were cloned with 4 different linkers ( GGGGS, BBa_K2638721 ; EAAAK, BBa_K2638722<(a>; XP, BBa_K2638723 ; cMyc, BBa_K2638724 ), so users of this System can choose the perfect linker for their own system. The structure of the first generation of pTale is shown in Figure 5.
Figure 5: Structure of the first generationof the target vector pTale. Four different Linkers were cloned. BFP and AmilCP were tested as Reporter Proteins. The Golden Gate Cassette, the Linker and the reporter protein form a fused unit and do not contain Biobrick scars.
While the measurable Fluorescence is diminished with a higher silencing effectivity when using a first generation pTale, the second generation of the target vector does not feature a direct fusion of the target gene to the reporter gene. The difference to the first generation is that the reporter is cloned behind a pLac promotor while the target sequence is fused to the CDS of the lacI inhibitor. This construct works as an inverter to increase the measurable fluorescence with higher silencing effectivity. Additionally, the plac promotor can be induced and repressed, introducing another layer of control into the system. The vector was constructed with BFP as a reporter protein. The structure of the second generation of pTale can be found in Figure 6. This construct prevents false positive results generated by non-functional reporter proteins, as the fluorescence increases with higher silencing activity instead of decreasing with higher silencing effectivity.
Figure 6: Structure of the second generation of the target vector pTale. The Golden Gate Cassette, the Linker and the LacI inhibitor are fused and do not feature scars between parts.
All Vectors with corresponding linkers and reporter proteins can be found in Table 1:
Table 1: List of Expression vectors (pGuide) and target vectors (pTale) that were constructed in this project and their corresponding linkers, reporter proteins and other features.
Biobrick number
Tag
Linker
Reporter Protein
Other features
BBa_K2638711
pTale_A_GS
GGGGS
AmilCP
-
BBa_K2638712
pTale_A_EA
EAAAK
AmilCP
-
BBa_K2638713
pTale_A_CM
cMyc
AmilCP
-
BBa_K2638714
pTale_A_XP
XP
AmilCP
-
BBa_K2638707
pTale_B_GS
GGGGS
BFP
-
BBa_K2638708
pTale_B_EA
EAAAK
BFP
-
BBa_K2638709
pTale_B_CM
cMyc
BFP
-
BBa_K2638710
pTale_B_XP
XP
BFP
-
BBa_K2638707
pTale2_B_GS
GGGGS
BFP
2. Generation
BBa_K2638708
pTale2_B_EA
EAAAK
BFP
2. Generation
BBa_K2638709
pTale2_B_CM
cMyc
BFP
2. Generation
BBa_K2638710
pTale2_B_XP
XP
BFP
2. Generation
BBa_K2638701
pGuide
-
-
-
BBa_K2638702
pGuide_Hfq
-
-
Hfq scaffold
BBa_K2638758
pGuide_Omp
-
-
OmpA 5’-UTR
BBa_K2638759
pGuide_Hfq_Omp
-
-
Hfq Scaffold, Hfq Scaffold
Testing the System
As we had great difficulties to assemble the vectors, we had very little time left to test the system. Amongst other small hinderances, major faults in IDT gene syntheses were the cause of great time losses. In the end we were able to assemble and transform all vectors belonging to the system. We were able to test the Target vectors cloning GFP as a target sequence into all target vectors. The OD600 and the fluorescence of BFG and GFP were measured in 96 well plates as described in the usage Protocol below using a Teacon reader. 3 replicates were measured for each sample. The results of the measurement of pTale1_B_CM are in figures 7 and 8 shown below. Two clones were tested, one of them showing expression of BFP, and one showing expression of GFP. No clones expressing both proteins were detected
Figure 7: Meassurement of the pTale_B_CM vector. To demonstrate its function, GFP was cloned into the vector as a target sequence. Three replicates ware measured for each sample. The measured clone was sequenced, showing that only 50 BP of GFB were inserted as target sequence.
Figure 8: Measurement of the pTale_B_CM vector. To demonstrate its function, GFP was cloned into the vector as a target sequence. Three replicates ware measured for each sample. The measured clone was sequenced, showing that a complete GFB was inserted as target sequence.
The results presented in figures 7 and 8 show that the cells were growing very slowly in the 12 h of measurement time owing to the oxygen limited culture conditions in the 96 well plate used. In Figure 7 a significant increase in the Fluorescence of BFP was visible, but no noteworthy increase in the Fluorescence signal of GFP was detectable. Sequencing showed that only 50 BP of the GFP were inserted into the Target vector, explaining the missing fluorescence signal. Though it could not be shown that our system works with larger protein fusions, we were able to show that our vector works as expected with short target sequences. Vice versa, in Figure 8 a strong GFP signal was detectable, while no BFP was produced. Possibly the larger GFP protein keeps the BFP from folding correctly or inhibits the fluorophore formation in another way. Both figures show a strong increase in Fluorescence upon induction with ahTc. This confirms that the strain is producing the TetR repressor, but not on a level that is high enough to repress the promotor. Possibly the copy number of pSb1C3 proved to be too high, so that not enough TetR was abundant.
Next, we tried to clone the Guide vectors into competent cells containing the target vectors shown above. As the cells were constantly expressing GFP, we were not able to perform the blue white screening following the Golden Gate Assembly, and therefore were not able to test the pTale vectors in combination with the pGuide vectors. We were however able to clone the pGuide Biobricks into a pSB1K3 plasmid which had its native origin of replication (ori) replaced with a synthetic one ( BBa_K2638751 )  made from parts of Team Vilnius 2017. We were able to grow cells containing that vector and sequence the ori.
As we ran out of time, we were not able to perform further tests on these vectors.
Usage protocol
Figure 9: Schematic overview that shows how to use our siRNA Testing System.
All experiments should be conducted in an E. coli strain producing the TetR repressor. Alternatively, a construct producing TetR can be cloned into the pTale vectors.
Prepare Biobricks
To use this system it is important that a stable integration of two different vectors in the E. coli cell is possible. Therefore,  the usage of two selection markers like antibiotic resistances, and optimally the usage of different Origins of Replications with about equal strength is necessary. The parts of iGEM Vilnius 2017 are extremely useful for this purpose.
Preparation of the target sequences
Prior to the experiments, the target sequences and siRNAs need to be designed. First decide for a target vector to use. It might be useful to try cloning the target sequence into vectors with different linkers and test which works best for a given target sequence.
There are two ways to clone a target sequence into a target vector:
By Gibson Assembly: Linearize the vector using BbsI or amplification with PCR using the primers in Table 2 corresponding to the chosen vector.
By Golden Gate assembly: Specially suitable for short target sequences which can be ordered as oligonucleotides, as overlaps of 4 nucleotides are needed to assemble the vector. Dimerize the oligos by Oligo Annealing and perform the Golden Gate assembly as described in our protocols. This might be useful when only a fragment of a coding sequence is used as a target sequence.
To screen several vectors for the best compartibility with a target sequence transform all vectors with the Target gene inserted.
For pTale vectors: induce with anhydrotetracycline (ahTc) and compare the levels of formed protein markers.
For pTale2: Induce with IPTG. Let the cells grow until the BFP can be detected. Induce with ahTc and measure and compare the decline of the BFPs fluorescence to find out if a correctly folded LacI fusion is expressed.
Once a target vector is chosen, prepare competent cells harboring this vector.
Design siRNAs
Prior to the siRNA design, a mechanism for the silencing should be chosen. Our vector system can be used with any siRNA that has overlaps to the pGuide expression vector. At present, our system also provides three vectors with already integrated sequences to give an siRNA further features. If an siRNA shall be used to prevent the translation of the mRNA, it is advantageous to use the pGuide_Omp vector ( BBa_K2638758 ), as the OmpA 5-UTR significantly increases the half-life of the siRNA. The Hfq adapting scaffold present in the Expression vectors pGuide_Hfq and pGuide_OmpA_Hfq is recommendable in most cases, as it protects the siRNA and strengthens the bond to the target sequence. The vector used determines which overlaps to the vector need to be included into the siRNAs. The overlaps for the different vectors can be seen in Table 3.
Design the siRNAs. Either by using our siRCon tool to predict siRNAs, or by adapting externally designed siRNAs to our system.
Order siRNAs as Oligonucleotides
Transformation and measurement
Perform an oligoannealing and a Golden Gate assembly as described in our protocols to insert the siRNAs into the expression vector.
Transform the Golden Gate Assembly into the competent cells containing the target vector.
Measure the reporter protein and the OD600. BFP has an excitation peak at 399 nm and an emission peak at 456 nm. AmilCP has a maximum absorption at 588 nm.
Table 2: Primers for sequencing, colony PCR and amplification of the Tace vectors.
Vektor
Linker
Forward Primer
Reverse Primer
Tm
pGuide
-
gattatttgcacggcgtcac
gaggaagcctgcataacgc
57°C
pTale1_BFP
all
gtgatagagattgacatccctatcagtg
ccctgagtatggttaatgaacgttttg
57°C
pTale1_AmilCP
all
gtgatagagattgacatccctatcagtg
cagtgagctttaccgtctgc
57°C
pTale2
all
gtgatagagattgacatccctatcagtg
gtggcaacgccaatcagc
57°C
pTale2_amplification
GGGGS
gggggtggaggttcgg
catctttcctgtgtgagtgctcag
57°C
pTale2_amplification
EAAAK
gaggcggctgcaaaagag
catctttcctgtgtgagtgctcag
57°C
pTale2_amplification
cMyc
gaacagaagctgattagcgaagaag
catctttcctgtgtgagtgctcag
57°C
pTale2_amplification
XP
gctcccgctccgaagc
catctttcctgtgtgagtgctcag
57°C
pTale1_ amplification
GGGGS
gggggtggaggttcgg
catctttcctgtgtgagtgctcag
57°C
pTale1_ amplification
EAAAK
gaggcggctgcaaaagag
catctttcctgtgtgagtgctcag
57°C
pTale1_ amplification
xMyc
gaacagaagctgattagcgaagaag
catctttcctgtgtgagtgctcag
57°C
pTale1_ amplification
XP
gctcccgctccgaagc
catctttcctgtgtgagtgctcag
57°C
Table 3: Table 3: The overlaps needed to insert DNA into several vectors.
Vektor
Linker
Forward overlap
Reverse overlap
pTale1, pTale2
GGGGS
GATG
CCCC
pTale1, pTale2
EAAAK
GATG
CCTC
pTale1, pTale2
cMyc
GATG
GTTC
pTale1, pTale2
XP
GATG
GAGC
