<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><NAV><UL class="nav-list"><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec">Home</A></LI><LI><A href="#!">Project</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Motivation">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Description">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Theory">Toxicity</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation">Accumulation</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA">Silencing</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Ferritin">Nanoparticles</A></LI></UL><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="#!">Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Results">Results Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Results">Toxicity Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation_Results">Accumulation Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA_Results">Silencing Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Ferritin_Results">Nanoparticles Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate</A></LI><LI><A href="#!">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Parts Collection</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Improve">Parts Improvement</A></LI><LI><A href="#!">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Public_Engagement">Outreach</A></LI><LI><A href="#!">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Design">Product Design</A></LI><LI><A href="#!">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Experiments">Experiments</A></LI><LI><A href="#!">Team</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Bielefeld">Bielefeld</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/CeBiTec">CeBiTec</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Bielefeld-University">Bielefeld University</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=Bielefeld-CeBiTec">Judging Form ⇗</A></LI></NAV></DIV><DIV class="sidenavi" id="side_bar"><LI class="side_list"><A href="#tox">Toxicity modeling</A></LI><LI class="side_list"><A href="#fdes">Design of Ferritin</A></LI><LI class="side_list"><A href="#sir">siRNA promoter model</A></LI><LI class="side_list"><A href="#dis">Discussion</A></LI></DIV><DIV class="container"><DIV class="main_content"><DIV class="title">Modeling</DIV><ARTICLE> „<I>Any modeling project should be tempered by the morality of laziness.</I>“ (Barnes <I>et al.</I>, 2010)</ARTICLE><H2>Short Summary</H2><ARTICLE> 
This phrase illustrates that lab results can be predicted before actually working in the lab, saving precious time.
</ARTICLE><ARTICLE>
Our project revolves around new discovered or theoretical biological systems like for example RNAi in prokaryotes or newly discovered copper transporters.
Our project required four different modeling approaches:
</ARTICLE><UL style="margin-left:20%; font-size:18px; line-height:2em;"><LI> toxicity modeling</LI><LI> siRNA modeling</LI><LI> ferritin structure modeling</LI></UL><H2>Toxicity modeling</H2><DIV class="article">
The toxicity of copper ions on the cell is well characterized (Ning <I>et al.</I>, 2015) but for important parts of this project like the <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Hardware">crossflow reactor</A> we needed to know the exact point of time when our cells will die to achieve highest possible yields of copper. 
</DIV><DIV class="article">
The residence time should not exceed the rate of dying and cell lysis in <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Hardware">the system</A>. If cell lysis kicks in copper gets released back again into the substrate media and the yield minimizes. The modeling started with the copper uptake in our cells containing the BioBrick BBa_K2638204, which expresses <I>oprC</I> under pAra​BAD (<A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Hardware">BBa_I0500)</A>) control and induction at 1.0 % arabinose in. The toxicity is calculated for a single cell.</DIV><DIV class="article">
The first step was to calculate the rate of expression of <I>oprC</I>. Therefore the characterization of  <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Hardware">BBa_I0500)</A> of <A href="https://2011.igem.org/Team:Groningen/project_characterisation_promoters_pbad">Groningen</A> was used to calculate the expression speed. At 1.0 % arabinose induction a raise of fluorescence of approx. <I>ΔF</I> = 82,000 within of t = 36,000 s was measurable. 
The conversion from fluorescence units to concentration in mol/L was calculated as <I>k</I> = 2.5 * 10<SUP>-6</SUP> mol*L<SUP>-1</SUP> (Furtado and Henry, 2002) and the volume of the used capillaries was <I>V</I> = 3.14 * 10<SUP>-9</SUP> L The rate of protein expression with 1.0 % arabinose is:
</DIV>
$$\frac{3.14 * 10^{-9} L * 2.5 * 10^{-6} mol L^{-1} * 87,500}{36,000 s} = 7.85 * 10^{-14} mol*s^{-1} (1)$$

<ARTICLE>According to (1) the increase of OprC with t<SUB>0</SUB> = 0 can be described with (2) and (3):</ARTICLE>

$$\frac{dOprC}{dt} = 7.85 * 10-14  mol*s^{-1} (2)$$  
 
$$OprC_t = OprC_0 + t * 7.85 * 10^{-14}   mol*s^{-1} (3)$$  

<ARTICLE>
The next step is to determine the increase of the copper concentration inside the cell. OprC transports copper ions with a rate of v<SUB>max</SUB> = 0.144 mol/s into the cell and has a substrate affinity of K<SUB>M­</SUB> = 1.66 mM (Parmar et al., 2018). The volume per cell in this model contains <I>V<SUB>C</SUB></I> = 1*10<SUP>-15</SUP> L<SUP>-1</SUP>. The starting concentration was taken from the first <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation_Results">accumulation</A> test with [Cu<SUP>2+</SUP><SUB>0</SUB>] = 0.003 M. Due to the missing catalytic interchange number <I>k<SUB>cat</SUB></I> the other definition of the Michaelis-Menten equation had to be used:
</ARTICLE>

$$\frac{d(Cu^{2+}_{inside})}{dt} = \frac{-d(Cu^{2+}_{outside})}{dt} =  v_{max} * \frac{Cu^{2+}_{outside}}{Cu^{2+}_{outside} + K_M}	(4)$$

<ARTICLE>
After solving the differential equation (4) with the conditions t<SUB>0</SUB> = 0:</ARTICLE>

$$[C_t]- K_M * ln{[C_t]} = v_{max} * t + [C_0] - K_M * ln {[C_0]}			(5)$$

<ARTICLE> 
Due to the fact that this only describes the kinetic of a single molecule of OprC the (3) and the Avogadro number <I>N<SUB>A</SUB></I> = 6.022 * 10<SUP>23</SUP> mol<SUP>-1</SUP> . That way the addition was possible of the both terms on the right side of (5) and the end equation results in:</ARTICLE>


$$[C_t]- K_M * ln{[C_t]} = v_{max}*t^2*N_A*7.85*10^{-14} mol*s^{-1} + [C_0]-K_M*ln{[C_0]} (6)$$
<ARTICLE>
At this point the equation is not further to simplify. Because of this a short python script trying out multiple different number constellations had to be tried out.
</ARTICLE><ARTICLE>kurve = []
for each in listCt[:end+1]:
	for t in ts:
		nOPRC = t*5*(10**(-29))
		
		#print each
		#print t
		gleichung1 = vmax*t*nOPRC*6.022*(10**(23)) - C0 - math.log(C0)*KM 
		gleichung2 = -each - KM * math.log(each)
		gleichung2small = gleichung2 - gleichung2*0.00001
		gleichung2big = gleichung2 + gleichung2*0.00001
		if gleichung2small &lt;= gleichung1 and gleichung2big &gt;= gleichung1:
			kurve.append((each,t))
			print each
			print t
print kurve
with open (&quot;Tupel_fuer_Kurve3.txt&quot;, &quot;w&quot;) as out:
</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 1:</B> The results of the copper uptake modeling. On the y axis the concentration in mol*L<SUP>-1</SUP> is depicted, on the x axis the time <I>t</I> in s. The concentration of 1 * 10<SUP>-5</SUP> mol/L is reached at 800 s.  
                      </FIGCAPTION></FIGURE><ARTICLE>This way the toxic copper concentration <I>c<SUB>tox</SUB></I> = 1 * 10<SUP>-5</SUP> mol/L (Ning, 2015) inside the cell is approx. reached 800 s (Figure 1) after the induction with 1.0 % arabinose.</ARTICLE><H2>Design decisions for an improved disassembly and reassembly of ferritin</H2><H2>Disassembly:</H2><ARTICLE>
Since pH induced ferritin disassembly using a regular ferritin requires strong acids a modified version was developed by H. Chen et. al. To allow for the disassembly of ferritin at a pH closer to neutral compared to the wild type the E-helix and DE turn (Fig. 2 marked in green) of the ferritin subunit where removed.
</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 2:</B> Parts marked in green where removed using primers binding on a ferritin sequence ordered via IDT synthesis and the short ferritin was aquired by PCR.  
                      </FIGCAPTION></FIGURE><ARTICLE>
By removing those structural elements the overall stability of ferritin is reduced and a disassembly at higher pH of around 4 instead of 2 should be the result. 
The removal of the E-helix and DE turn in all the 24 subunits should lower the melting temperature off the assembled ferritin from 80 °C to 72,7 °C 
</ARTICLE><H2>Assembly:</H2><ARTICLE>
Enhanced stability of the binding between the ferritin subunits is another field of research which sets out to enhance the binding strength between the 24 subunits to one another. The method developed by Wang et. al introduces 4 amino acid changes to the binding region between two adjacent subunits. The 4 histidine amino acids that are introduced (Fig. 3 histidine colored in pink) allow for the incorporation of Cu(II) ions into the binding region without severely changing the alignment of the subunits to each other.
</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 3:</B> 4 amino acid in the binding region between the ferritin subunits where changed to histidines (pink). The histidines can bind copper ions which in  turn enhances binding strength.  
                      </FIGCAPTION></FIGURE><H2>Combining both approaches</H2><ARTICLE>
From combining both approaches a mutant ferritin variant which disassembles at pH values closer to neutral and assembles with high strength when copper ions are added could be developed opening up new areas where the encapsulation of pH-sensitive proteins or molecules is required.
</ARTICLE><H2>siRNA promoter model</H2><ARTICLE>
		   First the most fitting siRNAs for our project has to be calculated. A program will be written which returns a selection of compatible siRNAs for a gene you want to silence. Possible secondary structures should also be considered (Mathews, 1999) because they could prevent the silencing mechanism of siRNAs.</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 4:</B> The siRNA with the compatible secondary structure binds to the mRNA and enables its digestion and blocks the translation of the sequence (A). The wrong secondary structure can’t bind and that way the mRNA can be translated and won’t be digested (B).
                      </FIGCAPTION></FIGURE><DIV class="article">We assume that the rate of silencing by siRNAs depends in the first place on the promoter region in front of the RNAi gene. By gathering expression rates of the most popular promoters in the iGEM parts registry valuable information for modelling will be acquired. Furthermore additional  factors like secondary structures, diffusion rates and degradation rates of RNA fragments. With that a model of the actual expression rate of proteins can be modeled with a given sequence that shall be silenced.
This modeling approach turned due to the high potential into our <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Software">software project</A>. The experimental part was shown on the <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA_Results">silencing results site</A>.</DIV><H2>Discussion</H2><ARTICLE>
The modeling for the point of time where the toxicity kicks in showed, that due to the troublesome way to find the right promoter strength a universal promoter library would be an important addition to the synthetic biology community. That way the promoter library in the <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">parts collection</A> represents an universal standard for the iGEM community to characterize their promoters.</ARTICLE><DIV class="article">
The <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Hardware">crossflow reactor</A> design was strongly influenced by the toxicity modeling. Another way to approach the high copper uptake were our <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Theory">anti toxicity measures </A> the BioBricks <A href="http://parts.igem.org/Part:BBa_K2638101">BBa_K2638101</A>, <A href="http://parts.igem.org/Part:BBa_K2638100">BBa_K2638100</A>, <A href="http://parts.igem.org/Part:BBa_K2638103">BBa_K2638103</A>, <A href="http://parts.igem.org/Part:BBa_K2638105">BBa_K2638105</A>, <A href="http://parts.igem.org/Part:BBa_K2638106">BBa_K2638106</A>, <A href="http://parts.igem.org/Part:BBa_K2638120">BBa_K2638120</A> and <A href="http://parts.igem.org/Part:BBa_K2638121">BBa_K2638121</A> enrich now the parts registry.</DIV><ARTICLE>The <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Software">siRNA/RNAi</A> modeling went over into our software project because it showed way more potential like this.</ARTICLE><ARTICLE>
The ferritin modeling was a important measure to make sure the characterization of the ferritin molecules didn’t encountered any issues.</ARTICLE><BUTTON onclick="myFunction()" class="refbtn"> References ▾</BUTTON><DIV id="myDIV" class="reftext"><B><A href="https://books.google.de/books?hl=de&amp;lr=&amp;id=sLflFWCkA1kC&amp;oi=fnd&amp;pg=PR8&amp;dq=Barnes,+D.+J.,+%26+Chu,+D.+(2010)&amp;ots=lfRIrKPpFN&amp;sig=s4_mjSadfCgpPlF3OMF71heZ1xQ#v=onepage&amp;q=Barnes%2C%20D.%20J.%2C%20%26%20Chu%2C%20D.%20(2010)&amp;f=false">Barnes, D. J., &amp; Chu, D. (2010)</A></B>. Introduction to modeling for biosciences. Springer Science &amp; Business Media.<B><A href="https://www.sciencedirect.com/science/article/pii/S0022283601950296">Delihas, N., &amp; Forst, S. (2001)</A></B>. MicF: an antisense RNA gene involved in response of Escherichia coli to global stress factors. Journal of molecular biology, 313(1), 1-12.<B><A href="https://www.sciencedirect.com/science/article/pii/S0003269702002816">Furtado, A., &amp; Henry, R. (2002)</A></B>. Measurement of green fluorescent protein concentration in single cells by image analysis. Analytical biochemistry, 310(1), 84-92.<B><A href="https://www.ncbi.nlm.nih.gov/pubmed/26258952">Ning, C., Wang, X., Li, L., Zhu, Y., Li, M., Yu, P., ... &amp; Zhang, Y. (2015)</A></B>. Concentration ranges of antibacterial cations for showing the highest antibacterial efficacy but the least cytotoxicity against mammalian cells: implications for a new antibacterial mechanism. Chemical research in toxicology, 28(9), 1815-1822.<B><A href="https://www.ncbi.nlm.nih.gov/pubmed/10329189">Mathews, D. H., Sabina, J., Zuker, M., &amp; Turner, D. H. (1999)</A></B>. Expanded sequence dependence of thermodynamic parameters improves prediction of RNA secondary structure. Journal of molecular biology, 288(5), 911-940.<B><A href="">Parmar, J. H., Quintana, J., Ramírez, D., Laubenbacher, R., Argüello, J. M., &amp; Mendes, P. (2018)</A></B>. An important role for periplasmic storage in Pseudomonas aeruginosa copper homeostasis revealed by a combined experimental and computational modeling study. Molecular microbiology.<B><A href="https://pubs.acs.org/doi/abs/10.1021/acs.jpca.7b02186">Phan, C. M., &amp; Nguyen, H. M. (2017)</A></B>. Role of capping agent in wet synthesis of nanoparticles. The Journal of Physical Chemistry A, 121(17), 3213-3219.<B><A href="">Stach, J. E., &amp; Good, L. (2011)</A></B>. Synthetic RNA silencing in bacteria–antimicrobial discovery and resistance breaking. Frontiers in microbiology, 2, 185.
                  <B><A href="https://www.ncbi.nlm.nih.gov/pubmed/29271453">Wang, Z., Dai, Y., Wang, Z., Jacobson, O., Zhang, F., Yung, B. C., ... &amp; Chen, X. (2018).</A></B>. Metal ion assisted interface re-engineering of a ferritin nanocage for enhanced biofunctions and cancer therapy. Nanoscale, 10(3), 1135-1144.<B><A https:="">Chen, H., S. Zhang, C. Xu, and G. Zhao.(2016)</A></B>. Engineering protein interfaces yields ferritin disassembly and reassembly under benign experimental conditions. Chemical Communications 52, no. 46 (2016): 7402-7405.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>