Team:Bielefeld-CeBiTec/Measurement
Home
Project
Motivation
Overview
Toxicity
Accumulation
Silencing
Nanoparticles
Modeling
Results
Results Overview
Toxicity Results
Accumulation Results
Silencing Results
Nanoparticles Results
InterLab
Demonstrate
Parts
Parts Overview
Basic Parts
Composite Parts
Parts Collection
Parts Improvement
Human Practices
Human Practices
Outreach
Awards
Hardware
Software
Measurement
Product Design
Notebook
Notebook
Experiments
Team
Members
Bielefeld
CeBiTec
Bielefeld University
Collaborations
Attributions
Partners
Judging Form ⇗
Test your own Promoter RBS
Design
Results
Outlook
Measurement
Short Summary
In order to enable precise quantification of promoter strengths, we designed a sophisticated measurement system (pCeBiTec18) comprising two independent reporters for accurate normalization. Fluorescence measurements are a common way to determine promoter strengths but are influenced by plasmid copy number and other factors. Having a reference reporter in the plasmid backbone allows correction of many of these effects. This system was subjected to absolute expression quantification in multiple replicates. Thus, our system equips every iGEM team with a general and easily applicable tool to analyze the expression strength of any promoter-RBS combination. pCeBiTec18 comes along with a detailed protocol for fast and efficient measurement. This system provides an urgently needed universal standard to compare promoter strengths.
Figure 1: Map of Biobrick BBa_K2638560 . The vektor of our measurement system with changeable promoterand RBS between prefix and suffix.
Test your own Promoter RBS
A strong promoter leads to an increased transcription level and therefore often to high level of gene expression, but a strong expression is not always the best choice for your construct. If you cannot decide, whether your choice of promoter and RBS offers the optimal level of gene expression, you can use our new designed vector to test your own promoter-RBS construct.  The only thing you have to do is to exchange the promoter-RBS combination of the vectors insert with your sequence. This can easily be done by restriction digest with XbaI and EcoRI to remove our insert. To insert the construct which should be tested oligonucleotides containing the construct are dimerized. These oligonucleotides contain an overhang to the backbone. Thus, the oligonucleotide dimer could be cloned in the backbone by Gibson assembly.  With the cultures containing this measurement plasmid, you can easily analyze the strength of expression of your promoter-RBS constructs measuring the fluorescence of the mRFP. You can also compare different promoter-RBS combinations by scaling the different mRFP signals through the detected eCFP signal. This even allows you to compare measurements, performed under different cultivation conditions.
Figure 2: Construction of the Promoter-RBS combination with the overhang to the Plasmid pCeBiTec18 ( BBa_K2638560 )
Design
Analyzing the expression strength of individual promoter-RBS combinations is quite challenging. The main reasons hindering accurate promoter-RBS characterization, are fluctuating copy-number changes of the expression plasmid (Jahn, M. et al,2016) or growth phase specific expression changes due to effects of sigma factors (Bervoets, I. et al., 2018). To avoid these errors, we designed a measurement vector carrying two reporter genes, which enables us to normalize the expression strength of the measured promoter-RBS combination to a constant expression level of the reference reporter gene. In this way, the effect of a varying plasmid copy number inside the cells can be taken into account.. Our measurement vector is based on the expression strength of the different promoter-RBS combinations from our library cloned upstream of mRFP and a double terminator (BBa_K2638426) into the pSB1C3 BioBrick site. Furthermore, our measurement vector carries a eCFP ( BBa_E0022 ) under control of a strong/weak Anderson promoter ( BBa_J23100 ) and the RBS BBa_J61100 followed by a double terminator (Fig. 1). Thus, the constitutive eCFP expression is proportional to the plasmid’s copy-number. This enables a normalization of the mRFP expression to the plasmid’s copy-number and direct assessment of our library’s promoter-RBS combinations expression strength. As this measurement is independent of plasmid effects it enables comparison with our modeling as well as with other expression constructs.
Figure 3: Map of Biobrick BBa_K2638560 . The vektor of our measurement system with changeable promoterand RBS between prefix and suffix.
Due to the DNA submission requirements, we choose the reporter gene mRFP as insert, to enable successful submission of our designed and constructed plasmid backbone. It also enables a quick and easy control if the cloning was successful. In regard to the use of a second fluorophore, the emission and absorption spectra should not interact with each other. Therefore, we did not use GFP, one of the most used reporters in iGEM. The use of GFP and mRFP would enable a FRET (Foerster Resonance Transfer)(Bajar, B. T. et al, 2016), and thus an interference regarding the detection signal. Since the difference between the absorption spectrum of mRFP and the emission of the eCFP is larger than the distance between the emission of GFP and the absorption of mRFP, we chose eCFP and mRFP as a good combination for our purposes Thus, eCFP and mRFP fluorescence should be detectable at the same time without any interferences.
Figure 4: Emission and absorption spectrum of GFP, CFP and RFP. The dashed line shows the emission and the solid line shows the absortion. Picture from Thermo Fisher fluorescence spectraviewer
Results
In addition to the modeled expression strengths we also tested the influence of gene expression on the bacterial growth by the measurement of the OD600. After cloning, all parts were checked by Sanger sequencing. The correct plasmids containing the different promoter-RBS combinations were transformed into E. coli DH5α and grown in LB media. 5mL cultures were inoculated at OD600 of 0.1 and incubated at 37°C and 300 rpm. After one hour, the fluorescence signals of mRFP and eCFP were measured using the Tecan Reader with excitation wavelengths of 558 nm and 435 nm and emission detection at 608 nm and 485 nm respectively. In a second experiment the samples were cultivated under the same conditions over 14 hours, showing similar results. The fluorescence signals of the cells with the plasmid containing the Anderson promoter BBa_J23119 encoding the E. coli consensus promoter was set as reference for the analysis of the expression strength of all other RBS promoter combinations.
Figure 5: Results of the expression strength analysis of different Anderson promoters (BBa_J23119, and BBa_J23100 to BBa_J23110) in combination with different RBS (BBa_J61100, BBa_B0030, BBa_B0031). The plasmid BBa_K2638560 was used for the expression of the fluorescence protein mRFP under the control of the mentioned promoter-RBS combinations. The fluorescence signals were normalized to the fluorescence of eCFP of the corresponding cells. The measured values were also normalized to the measured OD600. The relative strength regarding the the E. coliconsensus promoter BBa_J23119 is shown. The known promoter strength of the Anderson collection is listed in brackets.
The analyzed expression strengths of mRFP under control of the different promoters-RBS combinations are shown in Figure 3. The relative expression level is plotted regarding the E. coli consensus promoter BBa_J23119 The different RBS are shown in different colors. The Anderson promoter BBa_J23119 with different RBS was set as reference, and is therefore shown with a relative expression level of 1.0. All other combinations are compared and referenced to the consensus promoter construct with the corresponding RBS. The strongest difference between the expression level determined by Anderson et al. and our analyzed expression level could be observed for the BBa_J23100 promoter. Its expression level was expected to be as strong as the BBa_J23119 promoter, used as basis. But the measured relative expression level showed a significant lower expression of 0.47. This variation of the stated and analyzed relative expression level shows the importance of a validation of different promoter strengths and their combinations with different RBS. Using the BBa_B0030, described as a strong RBS, the relative expression level of the fluorophore of most of the analyzed constructs is higher than when combined to the RBS J23119 or B0031, except for promoters with a relatively low expression level. Our experiments show, that the relative expression level of the constructs is mainly influenced by the choice of RBS. That is why our measurement system is of such importance for the analysis of promoters and RBS.
Table 1: Results of the Anderson promoter (BBa_J23119,BBa_J23100 to BBa_J23110)in combination with the RBS (BBa_J61100, BBa_B0030, BBa_B0031).
Name
Sequence
Measured Strength RBS J61100
Measured Strength RBS B0030
Measured Strength RBS B0031
BBa_J23119
ttgacagctagctcagtcctaggtataatgctagc
1
1
1
BBa_J23100
ttgacggctagctcagtcctaggtacagtgctagc
0.41631
0.47372
0.40999
BBa_J23101
tttacagctagctcagtcctaggtattatgctagc
0.41057
0.38392
0.35113
BBa_J23102
ttgacagctagctcagtcctaggtactgtgctagc
0.32899
0.55225
0.49386
BBa_J23103
ctgatagctagctcagtcctagggattatgctagc
0.05543
0.07914
nd
BBa_J23104
ttgacagctagctcagtcctaggtattgtgctagc
0.66601
0.84445
0.66691
BBa_J23105
tttacggctagctcagtcctaggtactatgctagc
0.11405
0.00532
0.07979
BBa_J23106
tttacggctagctcagtcctaggtatagtgctagc
0.18257
0.2062
0.15317
BBa_J23107
tttacggctagctcagccctaggtattatgctagc
0.05682
0.01321
0.02459
BBa_J23108
ctgacagctagctcagtcctaggtataatgctagc
0.16366
0.25364
0.17165
BBa_J23109
tttacagctagctcagtcctagggactgtgctagc
0.00928
0.01173
nd
BBa_J23110
tttacggctagctcagtcctaggtacaatgctagc
0.29643
0.29876
0.29423
Outlook
With this database we can predict the right Promoter and RBS for every system. We have weak combinations for proteins which are, for example, slightly toxic for the cells or cause stress responses. On the other hand, we have strong combinations for the expression of anti-toxicity or membrane proteins. During our experiments we realized, that protein expression is influenced by several aspects. Besides plasmid copy-number variations and influences regarding growth, also mRNA and protein stability should be considered. We have found, that especially protein half-life of the fluorescence proteins is a problem during expression strength analyses. Especially for weak promoters and RBS accumulation of the protein to be measured has a disturbing effect on the expression analysis. Therefore, we thought about alternative methods to measure gene expression. In this context, we thought that the quantification of transcripts would be the best method to measure real transcription rates. This approach is independent of protein accumulating effects. Only mRNA degradation has to be considered. Fast sampling would be needed to analyze the real time status of the cells transcriptional landscape. Hence, we decided to perform quantitative real-time PCRs (qRT-PCR) to measure the transcript amount and thus the real promoter strength. Of course, it is no longer possible to take into account the strength of the RBS in this way. Unfortunately, due to time limitations we were not able to carry out the real-time PCR experiments for all of our constructs. But it would be a suitable method for future iGEM teams to analyze their promoters using our designed vector system with qRT-PCR. For this, both the transcript amount of mRFP and of eCFP as a reference should be determined. Using eCFP as a reference again would enable the measurement of a promoter strength independent of the plasmid copy number and the growth phase.
References ▾
De Mey, M., Maertens, J., Lequeux, G. J., Soetaert, W. K., & Vandamme, E. J. (2007). Construction and model-based analysis of a promoter library for E. coli: an indispensable tool for metabolic engineering. BMC biotechnology, 7(1), 34. Ipsaro, J. J., & Joshua-Tor, L. (2015). From guide to target: molecular insights into eukaryotic RNA-interference machinery. Nature structural & molecular biology, 22(1), 20. Jahn, M., Vorpahl, C., Hübschmann, T., Harms, H., & Müller, S. (2016). Copy number variability of expression plasmids determined by cell sorting and Droplet Digital PCR. Microbial cell factories, 15(1), 211. Kannan, S., Sams, T., Maury, J., & Workman, C. T. (2018). Reconstructing dynamic promoter activity profiles from reporter gene data. ACS synthetic biology, 7(3), 832-841. Köker, T., Fernandez, A., & Pinaud, F. (2018). Characterization of Split Fluorescent Protein Variants and Quantitative Analyses of Their Self-Assembly Process. Scientific reports, 8(1), 5344. Rizzo, M. A., Springer, G. H., Granada, B., & Piston, D. W. (2004). An improved cyan fluorescent protein variant useful for FRET. Nature biotechnology, 22(4), 445. Rudge, T. J., Brown, J. R., Federici, F., Dalchau, N., Phillips, A., Ajioka, J. W., & Haseloff, J. (2016). Characterization of intrinsic properties of promoters. ACS synthetic biology, 5(1), 89-98. Bajar, B. T., Wang, E. S., Zhang, S., Lin, M. Z., & Chu, J. (2016). A guide to fluorescent protein FRET pairs. Sensors, 16(9), 1488.
