<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_siRNA skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/siRNA</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><NAV><UL class="nav-list"><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec">Home</A></LI><LI><A href="#!">Project</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Motivation">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Description">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Theory">Toxicity</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation">Accumulation</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA">Silencing</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Ferritin">Nanoparticles</A></LI></UL><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="#!">Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Results">Results Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Results">Toxicity Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation_Results">Accumulation Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA_Results">Silencing Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Ferritin_Results">Nanoparticles Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate</A></LI><LI><A href="#!">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Parts Collection</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Improve">Parts Improvement</A></LI><LI><A href="#!">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Public_Engagement">Outreach</A></LI><LI><A href="#!">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Design">Product Design</A></LI><LI><A href="#!">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Experiments">Experiments</A></LI><LI><A href="#!">Team</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Bielefeld">Bielefeld</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/CeBiTec">CeBiTec</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Bielefeld-University">Bielefeld University</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=Bielefeld-CeBiTec">Judging Form ⇗</A></LI></NAV></DIV><DIV class="sidenavi" id="side_bar"><LI class="side_list"><A href="#abstract">Abstract</A></LI><LI class="side_list"><A href="#crispr">CRISPR/Cas9</A></LI><LI class="side_list"><A href="#sirna">siRNAs in <I>E. coli </I></A></LI><LI class="side_list"><A href="#rnai">Mimicking RNAi in <I>E. coli </I></A></LI></DIV><DIV class="container"><DIV class="main_content"><DIV class="title">Silencing</DIV><H2>Short Summary</H2><ARTICLE>
CRISPR/Cas is a widely used tool in genome engineering and strain development. Among other tools it can be used for knock-outs and insertion of new genes into the genome. But CRISPR/Cas has faced growing criticism in the past, partly due to the large off-target activity. The CRISPR/Cas system is a licensed product, which makes sharing related parts difficult and contradicts the iGEM spirit.
Here we want to propose RNA interference (RNAi) as an open source and free to use alternative to CRISPR/Cas for the iGEM contest. We connected several known processes taking place in <I>Escherichia coli</I>  to mimic the eukaryotic RNAi mechanisms. As we used the homologous proteins Hfq, RNase E and RppH, we did not need to express any new proteins in <I>Escherichia coli</I> to keep the impact on the cell’s metabolism as low as possible. Additionally, we designed a two vector system to express and test siRNAs in <I>E. coli</I> and developed a tool to predict and rate possible siRNAs and their potential to silence a given target sequence. 

                  </ARTICLE><H2> CRISPR/Cas9</H2><ARTICLE>
When Emmanuelle Charpentier and Jennifer Doudna introduced the CRISPR/Cas9 technique in 2012 (Hille &amp; Charpentier, 2016), it offered a seemingly infinite number of possibilities for genetic engineering. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeat) systems are part of the immune system of around 40 % of all bacteria and archaea (Zetsche <I>et al.</I>, 2015). The mechanism is based on the single endonuclease protein Cas9 (CRISPR associated protein) recognizes specific sites in DNA due to a complementary guide RNA (crRNA). The Cas9 protein introduces a double strand break at the desired site (Cong <I>et al.</I>, 2013). As a prerequisite the Cas9 protein needs the crRNA, tracrRNA and a PAM sequence. The PAM sequence consists of the nucleotides (nts) NGG and is found next to the protospacer DNA which gets processed to become the crRNA. Since a double strand break is lethal, a repair pathway needs to be triggered: Non-homologous end joining (NHEJ), microhomology-mediated end joining (MMEJ), homology-directed repair (HDR), or a combination of these pathways. The DNA fragment which serves as a template for the recombination mechanism needs to be homologous to the site where the double strand break was introduced. However, since some aberrations are tolerated it is possible to introduce insertions and/or deletions into the sequence. Therefore, the CRISPR/Cas9 technology is a powerful tool for genetic engineering while being cheap and easy to perform (Cong <I>et al.</I>, 2013; Hille &amp; Charpentier, 2016). 
However, unwanted off-target events can occur for sequences on the genome if there are sequences who share homology with the guide RNA. These events can lead to several mutations revealing the main problem of the CRISPR technology (Fu <I>et al.</I>, 2013; Hsu <I>et al.</I>, 2013; Pattanayak <I>et al.</I>, 2013; Tsai <I>et al.</I>, 2015). Whereas efforts have been made to reduce the off-target activity of the Cas9 protein (Kleinstiver <I>et al.</I>, 2016), researchers still were able to detect deletions and large-scale rearrangements on the genome after the repair of a CRISPR/Cas9-induced double strand break which could ultimately lead to cancer (Kosicki <I>et al.</I>, 2018).
By introducing point mutations into two sites of the Cas9 protein researchers have been able to alter the proteins function: A mutation in the HNH domain that cleaves the target DNA strand and in the RuvC domain that cleaves the non-target DNA strand lead to the dead Cas9 protein dCas9. The dCas9 protein lost its ability to cleave DNA but is still able to bind DNA (Barrangou <I>et al.</I>, 2014). If the guide RNA is designed to bind in a coding DNA sequence or in the upstream region, transcription initiation or elongation is going to be inhibited, leading to the repression of the gene of interest by sterically hindering. This process is called CRISPR interference based on the similar concept of RNA interference. Further applications of dCas9 include the fusion to a transcriptional activator in order to stimulate the transcription of an adjacent gene by recruiting the RNA polymerase (Hille <I>et al.</I>, 2016; Doudna &amp; Charpentier, 2014).  Since most of prokaryotic organisms do not have access to the RNA interference mechanism like eukaryotic organisms an effective silencing mechanism for prokaryotic organisms would be desired.

In synthetic biology it is often required to perform knock-outs in the context of metabolic engineering. Currently, knock-outs are usually carried out by using a CRISPR/Cas9 system. While it is often required and wanted to generate permanent knock-outs, those can generate problems like inhibited growth when genes that are important for the metabolism of the cell are knocked out. In extreme cases, a knock-out can even lead to the death of the cell, making it impossible to knock this gene out for metabolic engineering. We want to propose an alternative: siRNAs and RNAi. The RNA interference system (RNAi) can be used to knock down genes by degradation of the corresponding mRNA, rendering the knockdown inducible by using an inducible promoter. This enables us to let production cells grow until a desired cell density or production phase and induce the gene knockdown when needed. This method can be used to induce changes in the cell metabolism and therefore, adapt the cell the to individual needs of a production process.

</ARTICLE><H2>siRNAs in<I> Escherichia coli</I></H2><ARTICLE>
Small interfering RNAs (siRNAs) are very common in all types of organism and a part of the regulatory networks of a broad range of organism. Also known as small RNAs or small regulatory RNAs (sRNAs), these are short, about 50 to 200 nts long RNAs, which are not part of protein coding sequences but have several biological functions. The term “noncoding RNAs” which is commonly used in eukaryotes is not used as often, as some of the siRNAs also encode Proteins (Waters &amp; Storz, 2009). Many siRNAs work as posttranslational regulators of gene expression, either by binding to proteins or by acting as an antisense RNA and binding to complementary mRNAs. siRNAs are amongst other things involved in plasmid replication, response to environmental changes or metabolic changes. (Raghavan et al., 2014; Argaman et al., 2001)
siRNAs can influence transcription, translation, mRNA stability and more through a range of mechanisms, like changes in RNA conformation, Protein Binding, base pairing with other RNAs and interactions with DNA. (Waters &amp; Storz, 2009) 
We focused on Base Pairing siRNA to introduce new regulatory mechanisms to the iGEM community. There are two types of base pairing siRNAs: the cis or trans encoded siRNAs. Cis siRNAs, which are encoded on the reverse strand of the target gene often share broad complementarity with the target gene and can be 75 in length or even longer. Trans encoded base pairing siRNAs usually have limited complementarity to the target mRNAs and mainly negatively regulate protein levels by translational inhibition or mRNA degradation. Most characterized siRNAs bind to the 5’- Untranslated region (5’-UTR) and block the ribosome-binding site (RBS).  (Waters &amp; Storz, 2009) 
While blocking the translation using siRNAs is a good way to control Protein expression, it has certain drawbacks. The siRNA and the mRNA can seperated again, and the still intact mRNA can continue to be translated. Having the siRNAs flow around in the cell also means that the transition from a repressed translation to an unrestricted expression takes some time, as all the siRNAs can block mRNAs and are only degraded through standard degradation systems. 
Therefore we decided to mimic RNAi in E. coli to introduce a feasible non-permanent control mechanism for gene expression to the iGEM community

</ARTICLE><H2>Mimicking Eukaryotic RNAi in <I>Escherichia Coli </I></H2><ARTICLE>
There are 3 known RNAi pathways: MicroRNA (miRNA), small interfering RNA (siRNA), and Piwi-interacting (piRNA)  pathways. As an example, the miRNA RNAi pathway in eukaryotic cells consists of several steps. First a miRNA is produced and modified by several processing steps. A long primary miRNA (pri-miRNA) containing hairpins is cropped to a precursor miRNA  (pre-miRNA) with a length of 70 nt by a heterodimeric microprocessor complex. The complex comprised the proteins Drosha and DGCR8. The pre-miRNA is then exported from the nucleus and processed by a Dicer family enzyme to produce a mature double-stranded RNA of about 21 to 25 nt which is prepared for RISC loading. The RNA-induced silencing complex (RISC) is a ribonucleoprotein complex minimally composed of a small single-stranded RNA of 20 to 31 nt in length and an Argonaute (Ago) protein. The RNA in this complex acts as a ‘guide’ to determine the target mRNA. While there are several RNAi pathways differing in several points as the source of the guide RNA, this Ago-complex is the base of every known example. While they differ in several aspects it is always possible to differentiate three main stages: The production of a pre-guide RNA, the ripening to a functional guide RNA, and the silencing of the target mRNA. (Ipsaro &amp; Joshua-Tor, 2015)
We searched for a way to establish a mechanism similar to RNAi in <I>E. coli</I> while using enzymes which are already abundant to minimize the negative impact on the metabolism of the cell. In the system we envisioned a siRNA is transcribed into a functional form without need for further processing. This siRNA is activated by the pyrophosphohydrolase RppH and bound by the RNA chaperone Hfq. When the siRNA binds to a target mRNA RNase E can use this siRNA as a primer to break down the mRNA.
The crucial step in this mechanism is the adaption of RNase E to cut the mRNA. It has been shown that an siRNA can lead RNase E to a specific target and allosterically activates it through a 5’-monophosphate group (Callaghan <I>et al.</I>, 2005).
To add the RNase E recruiting and activating functionality to our siRNA we added a 4 nt non complementary  5’ overlap to the siRNA, as this is thought to enhance mRNA cleavage. The catalytic activity of RNAse E is dependend on the group found at the 5’ terminus of the siRNA. While a triphosphate and an OH-group protect the siRNA, a monophosphate increases the catalytic activity. Therefore, it is essential to have a 5’ monophosphate on our siRNA to increase the effectiveness of this mechanism (Bandyra <I>et al.</I>, 2012). This can be accomplished in several ways and is directly related to the way the siRNA is transcribed. A pre-siRNa can be cut at a target sequence by RNase E without full degradation of the siRNA leaving behind a 5’ monophosphate at the truncated siRNA (Chao <I>et al.</I>, 2017).
Nevertheless, we decided to transcribe a siRNA which needs no further truncation. Therefore, the maturation process is limited to the cleavage of the 5’ pyrophosphate to optain a 5’ monophosphate. This is accomplished by the <I>E. coli</I> pyrophosphohydrolase RppH. This is also the first step in RNA degradation but does not seem to be the rate determining step of RNA degradation as the half-life of the RNA is not changed by increased abundance of monophosphorylated transcripts (Foley <I>et al.</I>, 2015). 
The <I>E. coli</I> RppH has a broad substrate range, but it was shown by  Foley <I>et al.</I> (2015) that changes in the last 5’-terminal nt can change the rate at which the transcript is processed.  Foley <I>et al.</I> determined the ratio of monophosphorylated to triphosphorylated yeiP RNA to be 0.78 +- 0.02. Changing the U at the second position of yeiP RNa to a G increased this ratio threefold to 2.4 +- 0.2 in accordance with their expectations and as RppH shows a higher affinity for this terminal sequence. In their experiments substitutions of the nt at position 2 had the most significant impact on  pyrophosphate removal, while RppH has a moddest preference for an A at position 1 and substitutions of the nt at positions 3 and 4 had little effect. Additionaly, RppH needs at least 2 unpaired nt at the 5' end, but prefers three or more nt (Fu <I>et al.</I>, 2013).
In accordance with these findings siRNAs for our silencing system should have a 4 nt noncomplementary 5’ overlap, with the first 5’ nt being AG. This design parameter was included in our  <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Software">software tool</A> which predicts siRNAs through rational design and the Ui-Tei rules and rates the silencing probability of possible siRNAs to achieve a maximum silencing efficiency.
RNAs can also be designed to have longer lifetimes through protective loops at the 5’ terminus and intact 5’-triphosphates. While siRNAs are destroyed together with the mRNA during RNAi it is also possible to silence genes by blocking transcription or ribosome binding. In this case the siRNA should be as stable as possible and should be protected against decomposition. The 5’ untranslated region ( 5’ UTR) was found to stabilize mRNAs <I>in vivo</I>. While the <I>ompA</I> mRNA has a half-life of about 17 minutes, which is significantly longer than the average of 2-4 minutes in growing <I>E. coli</I> cells, deletion of the first 115 nt reduces the half life of the <I>ompA</I> mRNA to 3.6 minutes. When transferred in front of the β-lactamase (<I>bla</I>) mRNA, the half-life of this mRNA was raised from 3.7 minutes to 17 minutes. (Hansen <I>et al.</I>,1994) 
A great diversity of siRNAs uses the RNA chaperone Hfq to increase their effectiveness or even require it to function as intended. Hfq facilitates RNA hybridization of the siRNA and the mRNA and increases the silencing effectivity by increasing the local concentration of the siRNA. Aiba (2007) found that Hfq also speeds up the dublex formation between siRNA and mRNA. It is thought that Hfq might keep an siRNA and an mRNA in close proximity and give them the possibility to cycle through several possible base pairings to find an optimum conformation (Aiba, 2007).
While there are several siRNAs in <I>E. coli</I> that can bind to Hfq, their scaffold sequences for the adaption of Hfq differ greatly in sequence and length and can be positioned upstream or downstream of the silencing part of the siRNA. These scaffolds are mostly AU rich regions, which form secondary structure loops that can bind Hfq. Na <I>et al.</I> tested several different scaffolds and  discovered that the <I>micC</I> siRNA with a downstream Hfq scaffold had a superior repression capability compared to 100 known siRNAs (Na <I>et al.</I>, 2013). We included the MicC Hfq-scaffold sequence in our system to maximize the silencing efficiency. As an additional advantage, Hfq protects siRNAs from degradation (Carpousis, 2007). </ARTICLE><H2>The results of the silencing experiments can be found <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA_Results">here</A></H2><BUTTON onclick="myFunction()" class="refbtn"> References ▾</BUTTON><DIV id="myDIV" class="reftext"><B>  Aiba, H. (2007).</B> Mechanism of RNA silencing by Hfq-binding small RNAs. <I>Current opinion in microbiology, 10</I>(2), 134-139. <B>  Argaman, L., Hershberg, R., Vogel, J., Bejerano, G., Wagner, E. G. H., Margalit, H., &amp; Altuvia, S. (2001).</B> Novel small RNA-encoding genes in the intergenic regions of Escherichia coli. <I>Current Biology, 11</I>(12), 941-950. <B>  Bandyra, K. J., Said, N., Pfeiffer, V., Górna, M. W., Vogel, J., &amp; Luisi, B. F. (2012).</B> The seed region of a small RNA drives the controlled destruction of the target mRNA by the endoribonuclease RNase E. <I>Molecular cell, 47</I>(6), 943-953. <B>  Barrangou, R., &amp; Marraffini, L. A. (2014). </B>CRISPR-Cas systems: prokaryotes upgrade to adaptive immunity. <I>Molecular cell, 54</I>(2), 234-244 <B>  B. Zetsche, J.S. Gootenberg, O.O. Abudayyeh, I.M. Slaymaker, K.S. Makarova, P. Essletzbichler, et al.</B> Cpf1 is a single RNA-guided endonuclease of a class 2 CRISPR–Cas system.<I> Cell, 163</I> (2015), pp. 759-771. <B>  Callaghan, A. J., Marcaida, M. J., Stead, J. A., McDowall, K. J., Scott, W. G., &amp; Luisi, B. F. (2005).</B> Structure of Escherichia coli RNase E catalytic domain and implications for RNA turnover. <I>Nature, 437</I>(7062), 1187. <B>  Carpousis, A. J. (2007).</B> The RNA degradosome of Escherichia coli: an mRNA-degrading machine assembled on RNase E. <I>Annu. Rev. Microbiol., 61</I>, 71-87 <B>  Chao, Y., Li, L., Girodat, D., Förstner, K. U., Said, N., Corcoran, C., ... &amp; Luisi, B. F. (2017).</B> In vivo cleavage map illuminates the central role of RNase E in coding and non-coding RNA pathways. <I>Molecular cell, 65</I>(1), 39-51.  <B>  Deana, A., Celesnik, H., &amp; Belasco, J. G. (2008).</B> The bacterial enzyme RppH triggers messenger RNA degradation by 5′ pyrophosphate removal. <I>Nature, 451</I>(7176), 355. <B>  Doudna, J. A., &amp; Charpentier, E. (2014).</B> The new frontier of genome engineering with CRISPR-Cas9. <I>Science, 346</I>(6213), 1258096. <B>  Foley, P. L., Hsieh, P. K., Luciano, D. J., &amp; Belasco, J. G. (2015).</B> Specificity and evolutionary conservation of the Escherichia coli RNA pyrophosphohydrolase RppH. <I>Journal of Biological Chemistry, jbc-M114</I>.  <B>  Fu, Y., Foden, J. A., Khayter, C., Maeder, M. L., Reyon, D., Joung, J. K., &amp; Sander, J. D. (2013).</B> High-frequency off-target mutagenesis induced by CRISPR-Cas nucleases in human cells.<I> Nature biotechnology, 31</I>(9), 822. <B>Hansen, M. J., Chen, L. H., Fejzo, M. L., &amp; Beiasco, J. G. (1994).</B> The ompA 5′ untranslated region impedes a major pathway for mRNA degradation in Escherichia coli. <I>Molecular microbiology, 12</I>(5), 707-716.

             <B>  Hille, F., &amp; Charpentier, E. (2016).</B> CRISPR-Cas: biology, mechanisms and relevance. <I>Phil. Trans. R. Soc. B, 371</I>(1707), 20150496. <B>  Hsu, P. D., Scott, D. A., Weinstein, J. A., Ran, F. A., Konermann, S., Agarwala, V., ... &amp; Cradick, T. J. (2013).</B> DNA targeting specificity of RNA-guided Cas9 nucleases. <I>Nature biotechnology, 31</I>(9), 827 <B>  Ipsaro, J. J., &amp; Joshua-Tor, L. (2015).</B> From guide to target: molecular insights into eukaryotic RNA-interference machinery. <I>Nature structural &amp; molecular biology, 22</I>(1), 20. <B>  Jinek, M., Chylinski, K., Fonfara, I., Hauer, M., Doudna, J. A., &amp; Charpentier, E. (2012).</B> A programmable dual-RNA–guided DNA endonuclease in adaptive bacterial immunity. <I>Science, 1225829.</I><B>  Kleinstiver, B. P., Pattanayak, V., Prew, M. S., Tsai, S. Q., Nguyen, N. T., Zheng, Z., &amp; Joung, J. K. (2016).</B> High-fidelity CRISPR–Cas9 nucleases with no detectable genome-wide off-target effects. <I>Nature, 529</I>(7587), 490. <B>  Kosicki, M., Tomberg, K., &amp; Bradley, A. (2018).</B> Repair of double-strand breaks induced by CRISPR–Cas9 leads to large deletions and complex rearrangements. <I>Nature biotechnology, 36</I>(8), 765. <B>  L. Cong, F.A. Ran, D. Cox, S.L. Lin, R. Barretto, N. Habib, et al. </B> Multiplex genome engineering using CRISPR/Cas systems.<I> Science, 339</I><B>  Na, D., Yoo, S. M., Chung, H., Park, H., Park, J. H., &amp; Lee, S. Y. (2013).</B> Metabolic engineering of Escherichia coli using synthetic small regulatory RNAs. <I>Nature biotechnology, 31</I>(2), 170.  <B>  Pattanayak, V., Lin, S., Guilinger, J. P., Ma, E., Doudna, J. A., &amp; Liu, D. R. (2013).</B> High-throughput profiling of off-target DNA cleavage reveals RNA-programmed Cas9 nuclease specificity. <I>Nature biotechnology, 31</I>(9), 839. <B>  Raghavan, R., Groisman, E. A., &amp; Ochman, H. (2011).</B> Genome-wide detection of novel regulatory RNAs in E. coli. <I>Genome research, gr-119370.</I><B>  Tsai, S. Q., Zheng, Z., Nguyen, N. T., Liebers, M., Topkar, V. V., Thapar, V., ... &amp; Aryee, M. J. (2015).</B> GUIDE-seq enables genome-wide profiling of off-target cleavage by CRISPR-Cas nucleases. <I>Nature biotechnology, 33</I>(2), 187. <B>  Waters, L. S., &amp; Storz, G. (2009).</B> Regulatory RNAs in bacteria. <I>Cell, 136</I>(4), 615-628.4 https://www.nature.com/articles/nsmb.2931  </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>