<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Accumulation_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Accumulation Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><NAV><UL class="nav-list"><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec">Home</A></LI><LI><A href="#!">Project</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Motivation">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Description">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Theory">Toxicity</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation">Accumulation</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA">Silencing</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Ferritin">Nanoparticles</A></LI></UL><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="#!">Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Results">Results Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Results">Toxicity Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation_Results">Accumulation Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA_Results">Silencing Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Ferritin_Results">Nanoparticles Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate</A></LI><LI><A href="#!">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Parts Collection</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Improve">Parts Improvement</A></LI><LI><A href="#!">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Public_Engagement">Outreach</A></LI><LI><A href="#!">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Design">Product Design</A></LI><LI><A href="#!">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Experiments">Experiments</A></LI><LI><A href="#!">Team</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Bielefeld">Bielefeld</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/CeBiTec">CeBiTec</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Bielefeld-University">Bielefeld University</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=Bielefeld-CeBiTec">Judging Form ⇗</A></LI></NAV></DIV><DIV class="sidenavi" id="side_bar"><LI class="side_list"><A href="#toxa">Toxicity Assays</A></LI><LI class="side_list"><A href="#memb">Membrane Permeability Assays</A></LI><LI class="side_list"><A href="#spec">Specific Uptake Assay</A></LI><LI class="side_list"><A href="#disc">Discussion</A></LI></DIV><DIV class="container"><DIV class="main_content"><DIV class="title">Accumulation Results</DIV><H2>Short Summary</H2><ARTICLE>To test whether our copper accumulation system with the importers <I>oprC</I>, <I>hmtA</I>, <I>copC</I> and <I>copD</I> works as intended, we conducted several experiments analyzing Cu(II) ion uptake. We conducted growth experiments and observed hindered cell growth when expressing our transporter proteins, which indicates import of toxic copper into the cell. Furthermore, we measured membrane permeability thereby, showing the ion channel nature of the transporter proteins in the outer membrane. Finally we showed the proteins specifity for taking up Cu(II).</ARTICLE><H2>Toxicity Assays</H2><DIV class="article">
As intracellular copper leads to toxic effects in the cell (read more on our <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Theory" target="_blank">toxicity</A>) page, an increased uptake of Cu(II) ions should slow down and exacerbate cell growth. Therefore, we examined the growth of <I>E. coli</I> KRX expressing <I>copC</I>, <I>copD</I>, <I>oprC</I> and <I>hmtA</I> in lysogeny broth (LB) media with 30 ng/µL chloramphenicol supplemented with different concentrations of CuSO<SUB>4</SUB> (0 mM, 1 mM, 2 mM, 3 mM, 4 mM, 8 mM) by measuring the optical density at a wavelength of 600 nm (OD600). As a control we decided to use the pSB1C3 null vector under same conditions as well. The measurement was performed with the <A href="https://lifesciences.tecan.com/plate_readers/infinite_200_pro" target="_blank"> Infinite® 200 PRO</A> in a 24 wellplate with flat bottom (Greiner®). For expression of the BioBricks either a T7 promoter and a ribosome binding site (RBS) (BBa_K525998) or a combination of the <I>pBAD/araC</I> promoter (BBa_I0500) and a RBS (BBa_B0030) (RBS) were cloned upstream of <I>copC</I> (BBa_K2638001), <I>copD</I> (BBa_K2638002), (<I>oprC</I> (BBa_K2638200) and <I>hmtA</I> (BBa_K2638000). The resulting parts are shown in table 1: 
</DIV><TABLE id="t01" class="centern" style="margin-top:30px; margin-bottom:30px;"><CAPTION style="line-height:1.5; text.align:left;"><B>Table 1: </B>Parts used in toxicity assay (growth curves)</CAPTION><TBODY><TR><TH>Biobrick number</TH><TH>Components</TH><TH>Content</TH></TR><TR><TH>BBa_K2638003</TH><TD>BBa_K525998, BBa_K2638001</TD><TD>T7 promoter, RBS, <I>copC</I></TD></TR><TR><TH>BBa_K2638004</TH><TD>BBa_K525998, BBa_K2638002</TD><TD>T7 promoter, RBS, <I>copD</I></TD></TR><TR><TH>BBa_K2638016</TH><TD>BBa_K525998, BBa_K2638000</TD><TD>T7 promoter, RBS, <I>hmtA</I></TD></TR><TR><TH>BBa_K2638201</TH><TD>BBa_K525998, BBa_K2638200</TD><TD>T7 promoter, RBS, <I>oprC</I></TD></TR><TR><TH>BBa_K2638005</TH><TD>BBa_I0500, BBa_B0030, BBa_K2638001</TD><TD>pBAD/araC, RBS, <I>copC</I></TD></TR><TR><TH>BBa_K2638006</TH><TD>BBa_I0500, BBa_B0030, BBa_K2638002</TD><TD>pBAD/araC, RBS, <I>copD</I></TD></TR><TR><TH>BBa_K2638204</TH><TD>BBa_I0500, BBa_B0030, BBa_K2638200</TD><TD>pBAD/araC, RBS, <I>oprC</I></TD></TR></TBODY></TABLE><ARTICLE>All toxicity measurements were performed by inoculating <I>E. coli</I> KRX cultures harbouring the plasmid of interest with a starting OD600 of 0.01 from an overnight culture. Afterwards cells were grown (37 °C, 350 rpm) for 3 h and were induced either with 0.1 % rhamnose and 0.1 mM IPTG for the T7 constructs or 1 % arabinose for the arabinose constructs. The OD600 of the cultures were measured every hour for up to 12 hours.
 
Figure 1 shows the growth of <I>E. coli</I> KRX expressing <I>oprC</I> (BBa_K2638201). The right graph shows the growth after induction in comparison to the left graph without induction. All strains with induced expression showed lower maximum cell densities, even in the culture without added Cu(II). This effect is a consequence of the high metabolic burden bearing on the cells due to strong expression of the T7 promoter controlled genes. When growing in copper-containing media the cells expressing <I>oprC</I> show a stronger growth inhibition. This effect gets stronger with increasing Cu(II) concentrations. Especially interesting is the growth curve progression at a concentration of 2 mM Cu(II) (figure 1). The optical density does not only increase at a reduced rate, it even decreases after approximately 220 minutes, which indicates cell death. Both growth inhibitions cannot be observed with <I>E. coli</I> carrying pSB1C3.</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 1:</B> Growth curves of <I>E. coli</I> KRX carrying BBa_K2638201, the <I>oprC</I> expression device, at different CuSO<SUB>4</SUB> concentrations. Left: No induction. Right: Induction started simultaneously at t = 0 min with 0.1 % rhamnose and 0.1 mM IPTG.
                      </FIGCAPTION></FIGURE><ARTICLE>The observed effect could be reproduced with an induction 30 or 60 minutes before inoculation (Figure 2). A decrease of optical density can be detected in either measurement series, but the exact moment of decreasing cell density differs between experiments, thus making the determination of exact mean values impossible.</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 2:</B> Growth curves of <I>E. coli</I> KRX BBa_K2638201 at different CuSO<SUB>4</SUB> concentrations and different induction timepoints. Left: Induction with 0.1 % rhamnose and 0.1 mM IPTG 30 minutes before inoculation. Right: Induction with 0.1 % rhamnose and 0.1 mM IPTG 60 minutes before inoculation.
                      </FIGCAPTION></FIGURE><ARTICLE>The growth curves for <I>E. coli</I> KRX expressing <I>copC</I> from BBa_K2638003 show slightly decreased growth at a Cu(II) ion concentration of 2 mM at around 300 minutes. As the noninduced culture does not show this decrease this result indicates expression of <I>copC</I> and increased copper uptake (Figure 3).</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 3:</B> Growth curves of <I>E. coli</I> KRX BBa_K2638003, the <I>copC</I> expression device, at different CuSO<SUB>4</SUB> concentrations. Left: No induction. Right: Induction started simultaneously with inoculation with 0.1 % rhamnose and 0.1 mM IPTG.
                      </FIGCAPTION></FIGURE><ARTICLE>Growth curves of <I>hmtA</I> (BBa_K2638016) only show a very slight effect difference upon expression. But interestingly the expression itself does not effect cell growth at all (see figure 4). This experiment should be repeated, to see if the absent effect could be reproduced. </ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 4:</B> Growth curves of <I>E. coli</I> KRX BBa_K26380016, the <I>hmtA</I> expression device, at different CuSO<SUB>4</SUB> concentrations. Left: No induction. Right: Induction started simultaneously with inoculation with 0.1 % rhamnose and 0.1 mM IPTG.
                      </FIGCAPTION></FIGURE><ARTICLE>With expression of <I>copD</I> (BBa_K2638004) decreased growth upon expression but also slightly higher sensitivity towards copper could be observed (Figure 5). Furthermore, the effect of cell death was detected again, although the reaction is hardly measurable with a fast recovery of the cells after about 100 minutes. With <I>copD</I> in combination with pBAD/araC/RBS (BBa_K2638006) this can be reproduced and the effect there is more obvious (see figure 6).</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 5: </B>Growth curves of <I>E. coli</I> KRX BBa_K2638004 at different CuSO<SUB>4</SUB> concentrations. Left: No induction. Right: Induction started simultaneously with inoculation with 0.1 % rhamnose and 0.1 mM IPTG.
                      </FIGCAPTION></FIGURE><FIGURE role="group"><FIGCAPTION><B>Figure 6: </B>Growth curves of <I>E. coli</I> DH5α BBa_K2638006 at different CuSO<SUB>4</SUB> concentrations. Left: No induction. Right: Induction started simultaneously with inoculation with 1 % arabinose.
                      </FIGCAPTION></FIGURE><ARTICLE>Using the growth curve data from <I>E. coli</I>DH5α carrying BBa_K2638006 we also compared relative growth of a strain at 2 mM or 3 mM Cu(II) against the growth at 0 mM Cu(II) for not induced and induced cultures (figure 7). This type of plot shows the negative effect of Cu(II) ions on the growth. The cells grow only to a fraction of 0.785 ± 0.033 at 2 mM Cu(II) and 0.580 ± 0.054 with 3 mM Cu(II) of the culture in media without Cu(II). This shows the titratable effect of copper resulting in decreased growth in correspondence to the amount in copper inside the media.
</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 7: </B>Relative OD<SUB>600</SUB> of <I>E. coli</I> KRX strains carrying the <I>copD</I> expression device (BBa_K2638006). Measured OD<SUB>600</SUB> of the strain under varying copper concentrations and induction conditions were normalized to OD<SUB>600</SUB> of the strain growing in media without Cu(II) and without induction of gene expression to the corresponding time point. This included cultures from the strain in 2 mM or 3 mM Cu(II) media either not induced (n.i. 2 mM, n.i. 3 mM) or induced with 1 % arabinose (i. 2 mM, i. 3 mM).
                      </FIGCAPTION></FIGURE><H2>Membrane Permeability Assays</H2><ARTICLE>
1-N-phenylnaphthylamine membrane-permeabilization (NPN) assays are a fast method to measure the permeability of outer cell membranes. 
The NPN assays were all performed under the same conditions. The gene expression in the cells was either induced with  0.1 % rhamnose and 0.1 mM IPTG or with 1 % arabinose. Fluorescence was measured after excitation at 355 nm in a range from 380 – 550 nm. The fluorescence values for every measurement were normalized to the fluorescence value at 382 nm. The fluorescence data reached at 382 nm a minimum which indicates the starting point of the measurement. The equation (1) was furthermore used to calculate the percental increase of the fluorescence making it easier to determine fluorescence changes and compare between samples.</ARTICLE>


$$[\frac{F_x}{F_{382}}-1]*100 = F_{+\%}    (1)$$


<ARTICLE>Equation (1) was used to calculate the percental increase of the fluorescence. 
The NPN assays showed a higher percental fluorescence increase for all outer membrane transport systems compared to the strain with the empty vector (pSB1C3) as control.</ARTICLE><TABLE id="t01" class="centern" style="margin-top:30px; margin-bottom:30px;"><CAPTION style="line-height:1.5; text.align:left;"><B>Table 2: </B>NPN assay was carried out for different expression systems for OprC, CopC, CopD and HmtA and fluorescence was detected at 408 nm at exitation at 355 nm.</CAPTION><TBODY><TR><TH>Biobrick number</TH><TD>contains</TD><TD>Fluorescence at 408 nm</TD><TD>ΔFluorescence to pSB1C3</TD></TR><TR><TH>--</TH><TD>pSB1C3</TD><TD>35.12 ± 7.78</TD><TD>--</TD></TR><TR><TH>BBa_K2638201</TH><TD>T7 promoter<I>oprC</I></TD><TD>71.24 ± 9.52</TD><TD>36.11</TD></TR><TR><TH>BBa_K2638204</TH><TD><I>pBAD/araC</I> RBS <I>oprC</I></TD><TD>57.41 ± 17.55</TD><TD>22.29</TD></TR><TR><TH>BBa_K2638003</TH><TD>T7 promoter<I>copC</I></TD><TD>75.32 ± 10.59</TD><TD>40.20</TD></TR><TR><TH>BBa_K2638005</TH><TD><I>pBAD/araC</I> RBS <I>copC</I></TD><TD>51.42 ± 2.85</TD><TD>16.30</TD></TR><TR><TH>BBa_K2638004</TH><TD>T7 promoter<I>copD</I></TD><TD>68.11 ± 10.89</TD><TD>32.98</TD></TR><TR><TH>BBa_K2638006</TH><TD><I>pBAD/araC</I> RBS <I>copD</I></TD><TD>94.16 ± 4.47</TD><TD>59.04</TD></TR><TR><TH>BBa_K2638016</TH><TD>T7 <I>hmtA</I></TD><TD>62.35 ± 7.13</TD><TD>27.23</TD></TR></TBODY></TABLE><ARTICLE><B><A href="http://parts.igem.org/Part:BBa_K2638002" target="_blank">CopD (BBa_K2638002, BBa_K2638004 and BBa_K2638006)</A></B></ARTICLE><DIV class="article">
CopD is an active copper transport protein in the inner cell membrane. 
Both strains expressing <I>copD</I> (BBa_2638004 (T7 promoter + <I>copD</I>) and BBa_K2638006 (pBAD/araC promoter + <I>copD</I>)) show a higher increase in the fluorescence than the pSB1C3 control strain (yellow curve figure 8).
The strain expressing <I>copD</I> under control of the <I>pBAD</I>/<I>araC</I> promoter with RBS (<A href="http://parts.igem.org/Part:BBa_K2638006" target="_blank">BBa_K2638006</A>, blue curve in figure 8) showed a maximum of 94.16 ± 10.89 % in the fluorescence emission at 408 nm. This is an increase of 59.03 % to the empty control vector pSB1C3.
The <I>copD</I> strain with the T7 promoter (<A href="http://parts.igem.org/Part:BBa_K2638004" target="_blank">BBa_2638004</A>, red curve in figure 8) showed a maximum regarding the fluorescence of 68.10 ± 2.84 % at 408 nm. This is an increase of 32.99 % in comparison to the  empty vector control.
The difference increased fluorescence at 408 nm of both <I>copD</I> expressing strains compared to the empty vector control show a substantial increased of the membrane permeability of <I>E. coli</I>.
</DIV><FIGURE role="group"><FIGCAPTION><B>Figure 8: </B>Relative fluorescence increase of <I>E. coli</I> KRX strains with BBa_K2638004 (<I>copD</I> containing plasmid with T7 promoter control, red curve), BBa_K2638006 (<I>copD</I> containing plasmid with <I>pBAD</I>/<I>araC</I> promoter with RBS control, blue curve) and as an empty vector control an empty pSB1C3 in <I>E. coli</I> KRX (yellow curve). The excitation wavelength was 355 nm. The fluorescence was measured from 382 nm to 454 nm and the relative fluorescence was calculated with equation 1. The OD<SUB>600</SUB> of cells was adjusted with the NPN stock solution. The measurement was performed with the <A href="https://lifesciences.tecan.com/plate_readers/infinite_200_pro" target="_blank">Infinite® 200 PRO</A> in a 24 wellplate with flat bottom (Greiner®).
                      </FIGCAPTION></FIGURE><ARTICLE><B><A href="http://parts.igem.org/Part:BBa_K2638200" target="_blank">OprC (BBa_K2638200, BBa_K2638201 and BBa_K2638204)</A></B></ARTICLE><DIV class="article">OprC is a copper transport protein in the outer membrane.
In both strains which expressed the composite parts <A href="http://parts.igem.org/Part:BBa_K2638201" target="_blank">BBa_2638201</A> and <A href="http://parts.igem.org/Part:BBa_K2638004" target="_blank">BBa_K2638204</A> a stronger fluorescence than in the pSB1C3 control strain (yellow curve figure 9) was measurable.
The strain expressing <I>oprC</I> under control of the <I>pBAD</I>/<I>araC</I> promoter with RBS (BBa_K2638201, blue curve in figure 9) showed a maximum of 71.23 ± 7.78 % in the fluorescence emission at 408 nm. This is an increase of 36.12 % in comparison to the empty vector control.
The <I>oprC</I> strain with the T7 promoter (BBa_2638004, red curve in figure 9) showed a maximum fluorescence of 57.41 % ± 9.52 % at 408 nm. This is an increase of 22.29 % in comparison to the empty vector control.
Due to the stronger fluorescence at 408 nm of both <I>oprC</I> expressing strains compared to the empty vector control the NPN assay indicates a substantial increase of the membrane permeability of <I>E. coli</I> after expression of the transporter protein gene.</DIV><FIGURE role="group"><FIGCAPTION><B>Figure 9: </B> Relative fluorescence increase of <I>E. coli</I> KRX strains with BBa_K2638201 (<I>oprC</I> containing plasmid with T7 promoter control, red curve), BBa_K2638204 (<I>oprC</I> containing plasmid with <I>pBAD</I>/<I>araC</I> promoter and RBS control, blue curve) and as an empty vector control an empty pSB1C3 (yellow curve). The excitation wavelength was 355 nm. The fluorescence was measured from 382 nm to 454 nm and the relative fluorescence was calculated with equation (1). The OD<SUB>600</SUB> of cells was adjusted with the NPN stock solution. The measurement was performed with the <A href="https://lifesciences.tecan.com/plate_readers/infinite_200_pro" target="_blank">Infinite® 200 PRO</A> in a 24 wellplate with flat bottom (Greiner®).
                      </FIGCAPTION></FIGURE><ARTICLE><B><A href="http://parts.igem.org/Part:BBa_K2638001" target="_blank">CopC (BBa_K2638001, BBa_K2638003 and BBa_K2638005)</A></B></ARTICLE><DIV class="article">CopC is a copper mediator protein which is localized in the periplasm.
In both strains which expressed the composite parts <A href="http://parts.igem.org/Part:BBa_K2638003" target="_blank&quot;">BBa_2638003</A> and <A href="http://parts.igem.org/Part:BBa_K2638005" target="_blank">BBa_K2638005</A> a stronger fluorescence than in the pSB1C3 control strain (yellow curve, figure 10) was measured.
The <I>copC</I> strain with the T7 promoter (BBa_2638003, red curve in figure 10) showed a maximal fluorescence of 75.23 % ± 17.55 % at 408 nm. This is an increase of 40.20 % to the  empty control vector.
The strain expressing <I>copC</I> under control of the <I>pBAD</I>/<I>araC</I> promoter with RBS (BBa_K2638003, blue curve in figure 10) showed a maximum of 51.42 ± 10.59 % of fluorescence emission at 408 nm. This is an increase of 16.30 % in comparison to the empty control vector. In comparison between both <I>copC</I> expressing strains the device with the T7 promoter showed slightly higher fluorescence than the <I>pBAD</I>/<I>araC</I> device indicating a potential connection between the stronger T7 promoter and a higher membrane permeability.
</DIV><FIGURE role="group"><FIGCAPTION><B>Figure 10: </B>Relative fluorescence increase of <I>E. coli</I> KRX strains with BBa_K2638003 (<I>copC</I> containing plasmid with T7 promoter control, red curve), BBa_K2638005 (<I>copC</I> containing plasmid with <I>pBAD</I>/<I>araC</I> promoter with RBS control, blue curve) and as an empty vector control an empty pSB1C3 (yellow curve). The excitation wavelength was 355 nm. The fluorescence was measured from 382 nm to 454 nm and the relative fluorescence was calculated with equation 1. The OD<SUB>600</SUB> of the cells was adjusted with the NPN stock solution. The measurement was performed with the <A href="https://lifesciences.tecan.com/plate_readers/infinite_200_pro" target="_blank">Infinite® 200 PRO</A> in a 24 wellplate with flat bottom (Greiner®).
                      </FIGCAPTION></FIGURE><ARTICLE><B><A href="http://parts.igem.org/Part:BBa_K2638000" target="_blank">HmtA (BBa_K2638000 and BBa_K2638016)</A></B></ARTICLE><DIV class="article">HmtA is a copper and zinc specific transporter in the inner cell membrane. The strain which expressed the composite part <A href="http://parts.igem.org/Part:BBa_K2638016" target="_blank&quot;">BBa_2638016</A> showed a stronger fluorescence than the strain with the empty pSB1C3 (blue curve, figure 11).
The <I>hmtA</I> strain with the T7 promoter (BBa_2638016, red curve in figure 11) showed a maximal fluorescence of 62.35 ± 4.47 % at 408 nm. This is an increase of 27.23 % to the  empty control vector.
</DIV><FIGURE role="group"><FIGCAPTION><B>Figure 11: </B>Relative fluorescence increase of <I>E. coli</I> KRX strains with BBa_K2638016 (<I>hmtA</I> containing plasmid with T7 promoter control, red curve) and as an empty vector control an empty pSB1C3 (yellow curve). The excitation wavelength was 355 nm. The fluorescence was measured from 382 nm to 454 nm and the relative fluorescence was calculated with equation 1. The OD<SUB>600</SUB> of cells was adjusted with the NPN stock solution. The measurement was performed with the <A href="https://lifesciences.tecan.com/plate_readers/infinite_200_pro" target="_blank">Infinite® 200 PRO</A> in a 24 wellplate with flat bottom (Greiner®).
                      </FIGCAPTION></FIGURE><H2>Specific Uptake Assay</H2><ARTICLE>An <I>ortho</I>-Nitrophenyl-β-galactoside (ONPG) assay was performed with eight different pSB1C3 constructs to show that our copper transport BioBricks do not unspecifically take up other substrates.

The ONPG assay shows no unspecific uptake for all eight analyzed cultures (figure 12). As a reference an <I>E. coli</I> KRX with the empty vector pSB1C3 was used. Over a period of <I>t</I> = 3000 s no increase of absorption (λ = 400 nm) could was detected, thus the ONPG assay shows no unspecific uptake for all eight analyzed cultures (figure 12).



<FIGURE role="group"><FIGCAPTION><B>Figure 12: </B>Results of the ONPG assay. The absorption spectra are measured at λ = 400 nm over <I>t</I> = 3000 s. The measurement was performed with the <A href="https://lifesciences.tecan.com/plate_readers/infinite_200_pro" target="_blank">Infinite® 200 PRO</A> in a 24 wellplate with flat bottom (Greiner®).
                      </FIGCAPTION></FIGURE><H2>Discussion</H2><ARTICLE>In summary our data suggests that our copper uptake BioBricks work as expected. The import of copper can be shown by increasing toxic effects on cells, as the growth curves indicate an obvious growth inhibition for OprC and CopD, which begins at lower Cu(II) concentrations when expressing the copper transporter genes. The CopD encoding Biobricks BBa_K2638002, BBa_K2638004 and BBa_K2638006 show the most impressive results both in the toxicity test as well as in the NPN assay. This result was expected, as CopD imports Cu(II) actively across the inner cell membrane into the cytoplasm (Lawton <I>et al.</I>, 2016), where toxic effects have the strongest impact. The OprC encoding Biobricks BBa_K263200, BBa_K2638201 and BBa_K2638204 also show weaker, but also clear results, as it is a passive importer into the periplasm (Yoneyama &amp; Nakae, 2001). 

An interesting observation is, that the composite BioBrick of <I>copD</I> with <I>pBAD</I>/<I>araC</I> promoter and RBS BBa_K2638006) shows a higher impact from copper ions during the toxicity tests and a higher fluorescence emission in the NPN assay than the corresponding T7 promoter construct (BBa_K2638004) and all other copper transport constructs. In this collection this behavior is out of norm and different results were assumed because of the higher promoter strength of the T7 promoter (Balzer <I>et al.</I>, 2013). A possible reason for that could be that the chaperone concentration and the concentration of proteins for the buildup of membrane proteins in <I>E. coli</I> KRX just cannot keep up with the expression level of the T7 promoter and the rather short <I>copD</I> (Bukau <I>et al.</I>, 1990, Gräslund <I>et al.</I>, 2008). The other membrane proteins which we equipped with T7 promoters, <I>oprC</I> and <I>hmtA</I>, have an about 1000 bp longer coding sequence. Therefore, gene expression takes longer for those two constructs, giving the cell more time to cope with each newly expressed protein. The missing increase of absorption during the ONPG assay reinforces the assumption that CopD is located in the inner membrane and is specific to metal ions.

HmtA (BBa_K2638000), which fulfills in particular the same function as CopD, shows in the growth curve (Figure 7) as composite Part BBa_K2638016 an influence on the toxicity, but because of long lasting cloning complications we had no time to repeat that experiment. The ONPG assay shows that HmtA is probably located in the inner membrane and is specific to metal ions as mentioned in the literature (Lewinson <I>et al.</I>, 2009).

The NPN assays of CopD and HmtA show positive signals which does not accord to the literature. The NPN assay is only affected by outer membrane proteins (Helander, 2000). A further experiment is needed by analyzing CopD and HmtA with GFP marker and repeating the NPN assays. If the results are reproducible and the BioBricks are correctly localized in the inner cell membrane two reasons for the observed behavior are possible: 
On the one hand the NPN assay would also work for inner membrane proteins or on the other hand both proteins have regulatory functions which results in the expression of further outer membrane transporter.
<I>OprC</I> (BBa_K2638200) expression shows lower effects on the dying behavior under increased copper concentrations than <I>copD</I> (BBa_K2638002) and <I>hmtA</I> (BBa_K2638000). This could be since OprC is an outer membrane transporter (Yoneyama &amp; Nakae, 1996) and the copper is only transported into the periplasm, where it has lower toxic effects on the cell. The expression of BBa_K2638201 shows a bigger toxicity effect and a higher fluorescence emission  than BBa_K2638204 as expected for a T7 promoter. The NPN assay shows a higher membrane permeability that is not higher than the signal that CopD-cultures are emitting which is rather concerning in face of the fact that OprC is an outer membrane protein. The NPN assay should show a higher permeability in this case. The protein probably was not expressed in the same amount or to the same degree of functionality as CopD. The ONPG assay showed that OprC is not just an unspecific, unselective porin but a specific transporter for certain metals through the outer cell membrane.

CopC shows debatable results. Copper toxicity is nearly not measurable and even if it is just a mediator of copper in the periplasm and not a membrane protein it creates a higher permeability as measured in the NPN assay. This leaves room for interpretation that CopC either has a regulatory function for expression of membrane proteins or it changes the condition of the membrane to a higher permeability. The ONPG assay shows no signal so it seems to be a periplasmatic mediator protein. One probable function of the periplasmic soluble protein is to deliver Cu(II) to CopD and hence increase its effectivity in copper uptake (Lawton <I>et al.</I>, 2016). As CopC and CopD are found ubiquitous in a huge amount of unrelated species (Lawton <I>et al.</I>, 2016), the presumption is close that <I>E. coli</I> also expresses a <I>copD</I>-like gene. The associated protein may interact with our introduced CopC and could support its activity. To verify this thesis bioinformatic approaches could help identify such a gene in <I>E. coli</I> and repeat the experiments in knock-out mutants of those genes. Another possible explanation may be a regulatory virtue of CopC.

Further interesting experiments would be the combination of OprC (BBa_K2638200), CopC (BBa_K2638001) with CopD (BBa_2638002) in one BioBrick. The combination of an importer into the periplasm and into the cytoplasm could improve the effectivity of uptake dramatically. To further support CopD, CopC could deliver Cu(II) to the transporter through the periplasm. The already performed growth experiments could be performed again with such a BioBrick and measurements of intracellular copper may be conducted.
</ARTICLE><BUTTON onclick="myFunction()" class="refbtn"> References ▾</BUTTON><DIV id="myDIV" class="reftext"><B> Balzer, S., Kucharova, V., Megerle, J., Lale, R., Brautaset, T., &amp; Valla, S. (2013).</B>. A comparative analysis of the properties of regulated promoter systems commonly used for recombinant gene expression in Escherichia coli. <I>Microbial cell factories</I>, <I>12</I>(1), 26.<B> Bukau, B., &amp; Walker, G. C. (1990).</B>. Mutations altering heat shock specific subunit of RNA polymerase suppress major cellular defects of E. coli mutants lacking the DnaK chaperone.  <I>The EMBO journal</I>, <I>9</I>(12), 4027-4036.<B> Gräslund, S., Nordlund, P., Weigelt, J., Hallberg, B. M., Bray, J., Gileadi, O., ... &amp; Ming, J. (2008).</B>. Protein production and purification. <I>Nature methods</I>, <I>5</I>(2), 135.<B> Helander, I. M., &amp; Mattila-Sandholm, T. (2000).</B> Fluorometric assessment of Gram-negative bacterial permeabilization. <I>Journal of applied microbiology</I>, <I>88</I>(2), 213-219.<B> Lawton, T. J., Kenney, G. E., Hurley, J. D., &amp; Rosenzweig, A. C. (2016).</B> The CopC family: structural and bioinformatic insights into a diverse group of periplasmic copper binding proteins.<I> Biochemistry</I>, <I>55</I>(15).<B> Lewinson, O., Lee, A. T., &amp; Rees, D. C. (2009).</B> A P-type ATPase importer that discriminates between essential and toxic transition metals.<I> Proceedings of the National Academy of Sciences</I>, <I>106</I>(12).<B> Yoneyama, H., &amp; Nakae, T. (1996).</B> Protein C (OprC) of the outer membrane of Pseudomonas aeruginosa is a copper-regulated channel protein.<I> Microbiology</I>, <I>142</I>(8), 2137-2144.</DIV></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>