<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Accumulation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Accumulation</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><NAV><UL class="nav-list"><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec">Home</A></LI><LI><A href="#!">Project</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Motivation">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Description">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Theory">Toxicity</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation">Accumulation</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA">Silencing</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Ferritin">Nanoparticles</A></LI></UL><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI><A href="#!">Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Results">Results Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Toxicity_Results">Toxicity Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation_Results">Accumulation Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/siRNA_Results">Silencing Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Ferritin_Results">Nanoparticles Results</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Demonstrate</A></LI><LI><A href="#!">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Parts Collection</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Improve">Parts Improvement</A></LI><LI><A href="#!">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Public_Engagement">Outreach</A></LI><LI><A href="#!">Awards</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Design">Product Design</A></LI><LI><A href="#!">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Experiments">Experiments</A></LI><LI><A href="#!">Team</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Bielefeld">Bielefeld</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/CeBiTec">CeBiTec</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Bielefeld-University">Bielefeld University</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=Bielefeld-CeBiTec">Judging Form ⇗</A></LI></NAV></DIV><DIV class="sidenavi" id="side_bar"><LI class="side_list"><A href="#intro">Introduction</A></LI><LI class="side_list"><A href="#prev">Preventing Copper Export</A></LI><LI class="side_list"><A href="#copim">Copper Import</A></LI><LI class="side_list"><A href="#con">Conclusion</A></LI></DIV><DIV class="container"><DIV class="main_content"><DIV class="title">Accumulation</DIV><H2>Short Summary</H2><ARTICLE>Increasing the accumulation of copper ions in living cells requires enhanced copper uptake as well as inhibition of copper export systems. We knocked out <I>cusCFBA</I>, the dominant copper export system from <I>Escherichia coli</I> which facilitates Cu(I) export from the cytoplasm out of the cell. In addition, we supplemented the native copper accumulation system of <I>E. coli</I> by expression of the outer membrane importers <I>copC</I> and <I>oprC</I> as well as the inner membrane transporters <I>copD</I> and <I>hmtA</I>. 
</ARTICLE><H2>Introduction</H2><ARTICLE>
One important objective of nanoFactory is to construct an efficient accumulation system for heavy metal ions in order to subsequently form nanoparticles in <I>E. coli</I>. As a proof of principle we concentrated on the accumulation of Cu(II) ions. Besides the knockout of copper exporters, the main component for such a system is a highly efficient uptake system.

Since copper is toxic to all types of cells (Ladomersky &amp; Petris, 2015), the existence of transporters for the active and selective uptake of copper ions may initially come as a surprise, but copper also carries out important functions in many cells. As a cofactor for many different enzymes such as superoxide dismutase it is primarily involved in electron transfer, dioxygen transport and activation (Solomon <I>et al.</I>, 2014). Accordingly, there must be natural absorption mechanisms for the trace element both into the periplasm and the cytoplasm. Nevertheless, as non-bonded, dissolved copper in particular is very toxic, there are efficient export systems in <I>E. coli</I> to keep a tightly controlled copper homeostasis and remove excess copper from the cell (Rensing &amp; Grass, 2003). In order to create an efficient accumulation system for copper ions, highly selective uptake systems must be expressed on the one hand and existing export systems must be suppressed on the other.</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 1:</B> Overview on the different copper accumulation components we worked with. The active copper export system cusCBA is knocked out and prevents Cu(I) loss. Uptake is promoted from the outer membrane importer OprC supported by the inner membrane importers CopD and HmtA. CopC is a metallochaperone delivering Cu(II) to CopD. Toxic Cu(I) ions in the periplasm are oxidated to Cu(II) in the periplasm by the native copper oxidase Cue
                      </FIGCAPTION></FIGURE><H2>Preventing Copper Export</H2><ARTICLE>In order to prevent the loss of the effortfully imported Cu(II) ions by transport or diffusion, the <I>cus</I> operon of <I>E. coli</I> was knocked out using CRISPR/Cas9. It  consists of <I>cusCFBA</I> which belongs to the resistance-nodulation-cell division superfamily (Tseng <I>et al.</I>, 1999) and the two-component regulatory system <I>cusRS</I>. The response regulator <I>cusR</I> and its associated membrane-bound kinase <I>cusS</I> regulate the expression of the opposed directed <I>cusCFBA</I> genes directly upstream of <I>cusRS</I> (Munson <I>et al.</I>, 2000, Xiao <I>et al.</I>, 2017). The export of Cu(I) and also Ag(I) is carried out by the tripartite protein complex CusCBA (Gudipaty <I>et al.</I>, 2012) which is spanning through both the inner and outer membrane of the cell (Delmar, Su &amp; Yu, 2013). CusA is a pump for Cu(I)//Ag(I) ions located in the inner cell membrane. It pumps Cu(I)/Ag(I) ions, driven by proton-motive force, through the adapter-like CusB and CusC which span the outer membrane (Franke <I>et al.</I>, 2003, Delmar, Su &amp; Yu, 2013). Cu(I)/Ag(I) ions from the periplasmic space are delivered to CusB by CusF metallochaperones (Bagai <I>et al.</I>, 2008). 

We constructed a knock out mutant of <I>E. coli</I> DH5α using CRISPR/Cas9 to remove the whole <I>cus</I> operon ultimately prohibiting Cu(I) export. Consequently Cu(I) ions can not leave the periplasmic space. Subsequently the oxidization of Cu(I) to Cu(II) occurs spontaneously in the oxidative environment of aerobically grown cultures and is catalyzed by multicopper oxidase CueO which is native to <I>E. coli</I> (Outten <I>et al.</I>, 2001).</ARTICLE><FIGURE role="group"><FIGCAPTION><B>Figure 2:</B> Structure of the <I>E. coli cus</I> operon. The two-component regulatory system <I>cusRS</I> is located directly downstream of the <I>cusCFBA</I> coding sequences.
                      </FIGCAPTION></FIGURE><H2>Copper Import</H2><ARTICLE>The absorption of copper takes place in two steps: In the gram-negative chassis organism <I>E. coli</I>, Cu(II) ions are first transported across the outer cell membrane into the periplasm and subsequently into the cytoplasm over the inner cell membrane if necessary. Due to its toxicity an accumulation of copper ions in the periplasm has many advantages over accumulating it in the cytoplasm, where damage to the DNA and several proteins might occur (Ladomersky &amp; Petris, 2015). 

As comparable uptake system are not known in <I>E. coli</I> (Rensing &amp; Grass, 2003) we identified the outer membrane porin OprC from <I>Pseudomonas aeruginosa</I> as a good candidate for this purpose (Yoneyama &amp; Nakae, 2001). OprC belongs to the superfamily of TonB-dependent receptors to which TonB binds as a signal protein (Postle &amp; Good, 1983). It was shown that OprC allows the selective uptake of Cu(I/II) into the periplasmic space whilst other toxic ions like Ag(I) and Hg(II) are not imported (Yoneyama &amp; Nakae, 1996, Quintana, Novoa-Aponte &amp; Argüello, 2017). For our project we used a OprC homolog of <I>P. brassicacearum</I> 3Re2-7 (strain not pulished yet, for more information please contact Prof. Dr. Alfred Pühler) with 71% protein identity to avoid working with an organism from risk group two. 

In order to further increase the intracellular concentration of copper ions, the P-type ATPase HmtA from <I>P. aeruginosa</I> can be used. It actively transports copper ions across the cytoplasmic membrane into the cytoplasm (Lewinsohn, Lee &amp; Rees, 2009). As with <I>oprC</I> the homolog from the non-pathogenic <I>P. brassicacearum</I> 3Re2-7 with a protein identity of 80% was used. <I>HmtA</I> expression was found to lead to acute hypersensitivity for Cu(II) and Zn(II), which is a result of Cu(II)/Zn(II) uptake consequently leading to increased intracellular concentrations of these metal ions (Lewinsohn, Lee &amp; Rees, 2009). Uptake of other cations into the cytoplasm like toxic Ag(I) and Cd(II) ions is not facilitated by <I>hmtA</I> expression (Lewinson, Lee &amp; Rees, 2009). As <I>hmtA</I> expression depends on the extracellular Zn(II) concentration and Zn(II) is transported as well as Cu(II) by HmtA (Pederick <I>et al.</I>, 2015), it can be assumed that HmtA is a zinc importer which also transports copper ions due to the similarity in size and charge of Zn(II) and Cu(II) ions. 

Another transportation system is <I>copCD</I> from the <I>copABCD</I> operon for copper homeostasis in <I>Pseudomonas syringae</I> pv. <I>tomato</I>. This operon encodes for a precise regulation system of intracellular Cu(II) concentration (Lawton <I>et al.</I>, 2016). The two genes <I>copC</I> and <I>copD</I> exist in various forms like in a <I>copABCD</I> operon, standing alone as a <I>copCD</I> operon or even build up CopCD fusion proteins (Arnesano <I>et al.</I>, 2002, Lawton <I> et al.</I>, 2016) and their function is not fully understood yet. One proposed function is the uptake of Cu(II) ions into the cytoplasm (Cha &amp; Cooksey, 1993). CopC is a putative metallochaperone and exists in different forms with one or two copper binding sites (Lawton <I>et al.</I>, 2016) and CopD is a inner membrane protein that is presumably able to pump Cu(II) into the cell. We identified both genes in the genome of <I>P. brassicacearum</I> 3Re2-7 and transferred them into <I>E. coli</I> to increase the uptake of Cu(II) and accumulate intracellular copper.
</ARTICLE><H2>Conclusion</H2><ARTICLE>Our copper accumulation system works in two ways. We prevent the export of Cu(I) by knocking out the native <I>E. coli</I> copper export <I>cus</I> operon. In addition we identified genes involved in copper uptake, <I>hmtA</I>, <I>oprC</I>, <I>copC</I> and <I>copD</I>, and cloned them into <I> E. coli</I>. This combined system provides and represents an highly efficient and specific accumulation system for copper ions.
</ARTICLE><H2>The results of the accumulation experiments can be found <A href="https://2018.igem.org/Team:Bielefeld-CeBiTec/Accumulation_Results">here</A>.</H2><BUTTON onclick="myFunction()" class="refbtn"> References ▾</BUTTON><DIV id="myDIV" class="reftext"><B>Arnesano, F., Banci, L., Bertini, I., &amp; Thompsett, A. R. (2002).</B> Solution structure of CopC: a cupredoxin-like protein involved in copper homeostasis. <I>Structure</I>, <I>10</I>(10), 1337-1347.<B>Bagai, I., Rensing, C., Blackburn, N. J., &amp; McEvoy, M. M. (2008).</B> Direct metal transfer between periplasmic proteins identifies a bacterial copper chaperone. <I>Biochemistry</I>, <I>47</I>(44), 11408-11414.<B>Cha, J. S., &amp; Cooksey, D. A. (1993).</B> Copper hypersensitivity and uptake in Pseudomonas syringae containing cloned components of the copper resistance operon. <I>Applied and environmental microbiology</I>, <I>59</I>(5), 1671-1674.<B>Delmar, J. A., Su, C. C., &amp; Edward, W. Y. (2013).</B> Structural mechanisms of heavy-metal extrusion by the Cus efflux system. <I>Biometals</I>, <I>26</I>(4), 593-607.<B>Franke, S., Grass, G., Rensing, C., &amp; Nies, D. H. (2003).</B> Molecular analysis of the copper-transporting efflux system CusCFBA of Escherichia coli. <I>Journal of bacteriology</I>, <I>185</I>(13), 3804-3812.<B>Gudipaty, S. A., Larsen, A. S., Rensing, C., &amp; McEvoy, M. M. (2012).</B> Regulation of Cu (I)/Ag (I) efflux genes in Escherichia coli by the sensor kinase CusS. <I>FEMS microbiology letters</I>, <I>330</I>(1), 30-37.<B>Ladomersky, E., &amp; Petris, M. J. (2015).</B> Copper tolerance and virulence in bacteria. <I>Metallomics</I>, <I>7</I>(6), 957-964.<B>Lawton, T. J., Kenney, G. E., Hurley, J. D., &amp; Rosenzweig, A. C. (2016).</B> The CopC family: structural and bioinformatic insights into a diverse group of periplasmic copper binding proteins. <I>Biochemistry</I>, <I>55</I>(15), 2278-2290.<B>Lewinson, O., Lee, A. T., &amp; Rees, D. C. (2009).</B> A P-type ATPase importer that discriminates between essential and toxic transition metals. <I>Proceedings of the National Academy of Sciences</I>, <I>106</I>(12), 4677-4682.<B>McDevitt, C. A. (2015).</B> ZnuA and zinc homeostasis in Pseudomonas aeruginosa. <I>Scientific reports</I>, <I>5</I>, 13139.<B>Munson, G. P., Lam, D. L., Outten, F. W., &amp; O'Halloran, T. V. (2000).</B> Identification of a copper-responsive two-component system on the chromosome of Escherichia coli K-12. <I>Journal of Bacteriology</I>, <I>182</I>(20), 5864-5871.<B>Outten, F. W., Huffman, D. L., Hale, J. A., &amp; O'Halloran, T. V. (2001).</B> The independent cue and cusSystems confer copper tolerance during aerobic and anaerobic growth inEscherichia coli. <I>Journal of Biological Chemistry</I>, <I>276</I>(33), 30670-30677.<B>Pederick, V. G., Eijkelkamp, B. A., Begg, S. L., Ween, M. P., McAllister, L. J., Paton, J. C., &amp; McDevitt, C. A. (2015).</B> ZnuA and zinc homeostasis in Pseudomonas aeruginosa. <I>Scientific reports</I>, <I>5</I>, 13139.<B>Postle, K., &amp; Good, R. F. (1983).</B> DNA sequence of the Escherichia coli tonB gene. <I>Proceedings of the National Academy of Sciences</I>, <I>80</I>(17), 5235-5239.<B>Quintana, J., Novoa-Aponte, L., &amp; Argüello, J. M. (2017).</B> Copper homeostasis networks in the bacterium Pseudomonas aeruginosa. <I>Journal of Biological Chemistry</I>, jbc-M117.<B>Rensing, C., &amp; Grass, G. (2003).</B> Escherichia coli mechanisms of copper homeostasis in a changing environment. <I>FEMS microbiology reviews</I>, <I>27</I>(2-3), 197-213.<B>Solomon, E. I., Heppner, D. E., Johnston, E. M., Ginsbach, J. W., Cirera, J., Qayyum, M., ... &amp; Tian, L. (2014).</B> Copper active sites in biology. <I>Chemical reviews</I>, <I>114</I>(7), 3659-3853.<B>Tseng, T. T., Gratwick, K. S., Kollman, J., Park, D., Nies, D. H., Goffeau, A., &amp; Saier Jr, M. H. (1999).</B> The RND permease superfamily: an ancient, ubiquitous and diverse family that includes human disease and development proteins. <I>Journal of molecular microbiology and biotechnology</I>, <I>1</I>(1), 107-125.<B>Xiao, M., Zhu, X., Fan, F., Xu, H., Tang, J., Qin, Y., ... &amp; Zhang, X. (2017).</B> Osmotolerance in Escherichia coli is improved by activation of copper efflux genes or supplementation with sulfur containing amino acids. <I>Applied and environmental microbiology</I>, AEM-03050.<B>Yoneyama, H., &amp; Nakae, T. (1996).</B> Protein C (OprC) of the outer membrane of Pseudomonas aeruginosa is a copper-regulated channel protein. <I>Microbiology</I>, <I>142</I>(8), 2137-2144.
                   </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>