<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCUT_CadmiumAbsorption skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCUT/CadmiumAbsorption</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:SCUT</TITLE></P><DIV id="wrapper"><HEADER id="header"><DIV class="centered-wrapper"><DIV class="two-third column-last"><NAV id="navigation"><UL id="mainnav"><LI><A href="https://2015.igem.org/Team:SCUT"><SPAN>HOME</SPAN></A></LI><LI><A href="#"><SPAN>TEAM</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/OurUniversity">Our University</A></LI></UL><LI><A href="#"><SPAN>PROJECT</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SCUT/BackgroundOverview">Background &amp; Overview</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/CadmiumAdsorption">Cadmium Adsorption</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/SensorSystem">Sensor System</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Protocol">Protocol</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Achievement">ACHIEVEMENT</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Modeling"><SPAN>MODELING</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Biosensor">Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Bioeffector">Bioeffector</A></LI><LI><A href="#"><SPAN>RESULT</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Basic_Part">Basic Part</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Inthelab">In the lab</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Measurement">Measurement</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Practices"><SPAN>PRACTICES</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SCUT/EconomicsandEthics">Economics and ethics</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Outdoors">OUTDOORS</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/SynBiologyStudy">SYN-BIOLOGY STUDY</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Visits">VISITS</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Questionnaire">Questionnaire</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Safety"><SPAN>SAFETY</SPAN></A></LI></NAV></DIV></DIV></HEADER></DIV><DIV id="wrapper"><SECTION class="page-title"><DIV class="title-wrapper"><DIV class="title-bg"><DIV class="title-content"><DIV class="three-fourth"><H2>Cadmium Adsorption</H2></DIV></DIV></DIV></DIV></SECTION><DIV class="centered-wrapper"><DIV class="portfolio-content"><DIV id="majorcontent"><H3>Overview</H3><P>
Wild type Escherichia coli can assemble extracellular amyloid fibers on their cell surface. Here we performed the molecular programming of the bacterial extracellular matrix material by genetically appending peptide domains to the amyloid protein CsgA, the major curli subunit in E.coli biofilms. Attaching metal-binding peptide to CsgA can allow E.coli to capture specific heavy metal in the water. By creating fusion protein of CsgA with synthetic phytochelatins (ECs), we could characterize the ability of heavy metal cadmium adsorption.</P><H3>csgA</H3><P>
Curli, the first identified functional amyloid fibres, are extracellular protein fibers produced by many enteric bacteria including Escherichia coli and Salmonella species [1]. The curli system exhibits numerous features that make it an ideal platform for the type of materials engineering by way of synthetic biology that we envision. First, as the curli nanofibre is composed primarily from the self-assembly of one small protein, it presents a tractable entry point towards creating a large diversity of biofilm extracellular matrices with conventional genetic engineering methods. Second, the functional amyloid fibres formed by CsgA are extremely robust, being able to withstand boiling in detergents and extended incubation in solvents, increasing their potential utility in harsh environments. Finally, recent findings have shown that the curli system can be used to efficiently export natively unfolded polypeptides and can be used in a broad and modular way for the display of various functional peptides throughout the E. coli biofilm [2]. CsgA, the dominant proteinaceous component in E. coli biofilms, is capable of self-polymerizing in vitro into β-sheet-rich amyloid fibers that bind to the amyloid specific dye Congo red (CR), resulting in a red shift and green birefringence under polarized light, and thioflavin T (ThT), leading to increased fluorescence at certain wavelengths [3].</P><P>Fig. 1. Curli production contributes to E. coli biofilms. A. E. coli k-12 strain BW25113 grown on a low salt agar plate at 26 °C produce cell surface associated curli fibers that are visible by transmission electron microscopy( Margery L. Evans. 2013).</P><P>Fig. 2. Model of curli biogenesis. Excluding CsgD, the master curli regulator, all Csg proteins have Sec-dependent signal sequences allowing their secretion into the periplasm. The lipoprotein CsgG forms a pore-like structure in the outer membrane. The major subunit protein CsgA and the nucleator CsgB are secreted to the cell surface in a CsgG- and CsgE-dependent manner. CsgF associates with the outer membrane and is required for cell association of the minor curli fiber subunit CsgB. Situated at the cell surface, CsgB nucleates soluble, unstructured CsgA into a highly ordered amyloid fiber. Curli production can be visualized by CR binding, which is absent in a csgA mutant, and by transmission electron microscopy (left inserts). Also shown are two CsgA subunits interacting in a cross-β conformation, with the R1–R5 interaction depicted (right inset) (Luz P. Blanco. 2011).</P><H3>Synthetic phytochelatins (ECs)</H3><P>Phytochelatins (PCs) are naturally occurring thiol-rich peptides containing gamma (γ) peptide bonds(γGlu-Cys)n Gly (n=2-11) and are well known for their metal-binding and detoxification capabilities. These peptides contain a γ- bond between the glutamic acid and cysteine and are synthesized enzymatically by the phytochelatin synthase enzyme using reduced glutathione (GSH) or related thiols [4]. Whether synthetic phytochelatins (ECs), the analogs of phytochelatin (Glu-Cys)n Gly, can be used as an alternative approach for enhancing the metal-binding capacity of bacteria has been investigated in some study. Previously, the difference between α- and γ- bond was thought to influence the metal-binding affinity between ECs and PCs. However, detailed experiments demonstrated that these peptides bind various metals in a similar manner as PCs [4]. A novel strategy using synthetic phytochelatins is described for the purpose of developing microbial agents for enhanced bioaccumulation of toxic metals.The binding of metals to ECs weakens in the order of Cd(II) &gt; Zn(II) &gt; Ni(II) &gt; Co(II) [5]. Thus, we utilized ECs which synthesis easily for microbes to adsorb Cd2+ in the water efficiently.</P><P>Fig. 3. (A) The general structure of PCs with a γ peptide bond between the glutamic acid and cysteine. (B) The chemical structure of ECs with an α peptide bond between the glutamic acid and cysteine (Devesh Shukla. 2013).</P><H3>Adsorption system</H3><P>
We constructed CsgA-ECs adsorption system to capture Cd2+ in the water. First, we knockout CsgA gene on the genome of E.coli k-12 strain W3110. Then we expressed CsgA-ECs fusion in the E.coli. The linker region was placed at C-terminus of the CsgA and the amino acid sequence of the linker was “GSGGSG” . </P><P>
ECs of different chain lengths can be produced to provide peptides with different metal binding capacities. So, using synthetic genes we have recently synthesized ECs of different chain lengths for removal of Cd2+. We expressed different lengths of EC genes encoding EC1, EC2, EC3, EC5, and EC8 in ΔCsgA cells. In this way, we will be able to study the binding capabilities of ECs  of different chain lengths. We used thioflavin T (ThT) binding assay to analyze the expression of CsgA.</P><P>Fig 4. Genetic programming and modularity of the CsgA-ECs system.</P><H3>Reference</H3><P>
[1] Evans, M.L., and Chapman, M.R. (2014). Curli biogenesis: order out of disorder. Biochimica et biophysica acta 1843, 1551-1558.
[2] Nguyen, P.Q., Botyanszki, Z., Tay, P.K., and Joshi, N.S. (2014). Programmable biofilm-based materials from engineered curli nanofibres. Nature communications 5, 4945.
[3] Blanco, L.P., Evans, M.L., Smith, D.R., Badtke, M.P., and Chapman, M.R. (2012). Diversity, biogenesis and function of microbial amyloids. Trends in microbiology 20, 66-73.
[4] Shukla, D., Tiwari, M., Tripathi, R.D., Nath, P., and Trivedi, P.K. (2013). Synthetic phytochelatins complement a phytochelatin-deficient Arabidopsis mutant and enhance the accumulation of heavy metal(loid)s. Biochemical and biophysical research communications 434, 664-669.
[5] Viswanathan, K., Schofield, M.H., Teraoka, I., and Gross, R.A. (2012). Surprising metal binding properties of phytochelatin-like peptides prepared by protease-catalysis. Green Chemistry 14, 1020.</P></DIV></DIV></DIV><FOOTER id="footer1" style="border-top:5px solid #e1472f;"><DIV class="centered-wrapper"><DIV id="topfooter"><DIV class="one-half"><H3>About Us</H3><P class="percent-two-third">
In 2015, we SCUT teams won top ten innovative and entrepreneurial team set up by SCUT.Because of the strong support of the college, our team is being on the right track, and increasing understanding of the subject and experience. 
</P></DIV><DIV class="one-half column-last"><DIV class="percent-one-half column-last"><H3 style="margin-left:20px;">Thanks</H3><UL><LI>Zhang Zhenwu,Prof. Guo Shouqian,Dr. Li, Dr. Li Cheng,Dr. Wang Meng,Chen Kejie</LI><LI>Guangzhou Municipal Environmental Protection Bureau</LI></UL></DIV></DIV></DIV></DIV><DIV id="bottomfooter"><DIV class="centered-wrapper"><DIV class="one-half"><P>COPYRIGHT ©2015-SCUT</P></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>