<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCUT_Protocol skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCUT/Protocol</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:SCUT</TITLE></P><DIV id="wrapper"><HEADER id="header"><DIV class="centered-wrapper"><DIV class="two-third column-last"><NAV id="navigation"><UL id="mainnav"><LI><A href="https://2015.igem.org/Team:SCUT"><SPAN>HOME</SPAN></A></LI><LI><A href="#"><SPAN>TEAM</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/OurUniversity">Our University</A></LI></UL><LI><A href="#"><SPAN>PROJECT</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SCUT/BackgroundOverview">Background &amp; Overview</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/CadmiumAdsorption">Cadmium Adsorption</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/SensorSystem">Sensor System</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Protocol">Protocol</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Achievement">ACHIEVEMENT</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Modeling"><SPAN>MODELING</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Biosensor">Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Bioeffector">Bioeffector</A></LI><LI><A href="#"><SPAN>RESULT</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Basic_Part">Basic Part</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Inthelab">In the lab</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Measurement">Measurement</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Practices"><SPAN>PRACTICES</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SCUT/EconomicsandEthics">Economics and ethics</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Outdoors">OUTDOORS</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/SynBiologyStudy">SYN-BIOLOGY STUDY</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Visits">VISITS</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Questionnaire">Questionnaire</A></LI><LI><A href="https://2015.igem.org/Team:SCUT/Safety"><SPAN>SAFETY</SPAN></A></LI></NAV></DIV></DIV></HEADER></DIV><DIV id="wrapper"><SECTION class="page-title"><DIV class="title-wrapper"><DIV class="title-bg"><DIV class="title-content"><DIV class="three-fourth"><H2>Protocol</H2></DIV></DIV></DIV></DIV></SECTION><DIV class="centered-wrapper"><DIV class="portfolio-content"><DIV id="majorcontent"><H3>Related experimental method of molecular construction</H3><H3>1. Acquisition of the purpose gene</H3><H3>1.1 Objective gene extraction and PCR amplification</H3><P>Use the DNA Mini Kit to extract the whole genome of the purpose strain. Please refer to the instruction of the DNA Mini Kit for the details of the procedure.
Precede PCR using the whole genome as the DNA template. The reaction system is as follows: </P><P>Table 1</P><P>The parameters of reaction condition:</P><P>Table 2</P><P>Use agarose gel electrophoresis to separate and analyze the PCR product. And then, use DNA Gel Extraction Kit to extract the purpose DNA fragment.</P><H3>1.2 DNA agarose gel electrophoresis:</H3><H4>1.2.1 Casting a gel</H4><P>
1)Weigh 0.25g agarose in a conical flask. Add 1×TAE 25mL, cover a aluminized paper and dissolve the agarose in a microwave oven for 1~2min taking care not to boil the solution out of the flask.
2)Swirl to mix the gel.
3)Install the gel tray and the comb exactly.
4)Let the agarose solution cool down. Once the solution is touchable, add the DNA stain. Check the stock concentration and add the appropriate amount to give the desired final concentration.
5)Pour the gel solution into the gel tray. Remove any air bubbles with a pipette tip. Put in comb.
6)The gel will solidify while cooling down to room temperature. Remove the comb carefully and place the gel tray into the buffer chamber. Add 1x TBE buffer until the gel is completely covered.</P><H4>1.2.2 Running the gel:</H4><P>
1)Take part of your DNA samples (~0.2 µg) and mix with loading dye. This can be done either in 1.5 mL tubes or, if the volumes are very small, on a piece of parafilm.
2)Load the size marker mixed in 1x loading dye (~6 µL final volume) into a well.
3)Load your samples into the other wells while writing down which lanes have which samples.
4)Put the lid onto the buffer chamber and connect it to the power supply. Make sure to put it in the right direction so that your DNA runs towards the positive (red) electrode.
5)Run the gel at 120 V for 30–60 min. Neither of the dyes should be run off the gel. If the electrophoresis runs correctly you will notice air bubbles coming from the negative (black) electrode.
6)Stop the run and bring your gel to a UV table to visualize your gel bands and take a picture of your gel. And then cut and extract the gel which contains the purpose DNA fragment. Use protective glasses. If sufficient separation was not achieved, put the gel back into the buffer chamber and run it for longer.</P><H4>1.2.3 Extract the purpose DNA fragment:</H4><P>
Please refer to the instruction of the DNA Gel Extraction Kit for the details of the procedure, but taking care to cut the gel which just contains all purpose bands as small as possible.</P><H3>2. Construct the gene circuits of expression system</H3><P>
Get the standard biological parts of promoter, RBS, and terminator. Assemble the plasmid from each standard biological parts and purpose gene by genetic engineering.</P><P>Digestion system:</P><P>Table 3</P><P>Attention:</P><P>
1.Make sure of the volume of the each component before you add it into a tube and add the endonucleases last.
2.Mix the components in the tubes and centrifuge for a few seconds to spin down the liquid.
3.The quantity what is said above should be adjusted according to the actual situation.
4.React 2 hours in 37 °C.</P><P>Ligation system:</P><P>Table 4</P><P>Attention:</P><P>1.Add the T4 DNA ligase last.
2.Mix reaction components and overnight react in 16°C.
3. The expression, screening and identification of the recombinant plasmid:</P><H3>3.1 The transformation and screening of the plasmid:</H3><P>
Turn the recombinant plasmid into the E.coli Top Ten competent cell by CaCl2, in culture medium containing 10 mg/L kanamycin on a preliminary screening, get the E.coli Top Ten with a pET - 30 a recombinant plasmid. Pick monoclonal recombinant E. coli to 2 ml LB liquid medium, after 37 ℃ under the cultivate after 6 h, extraction with alkali splitting method recombinant plasmid.</P><H4>3.1.1 The preparation of competent E.coli cells by CaCl2:</H4><P>
1)Get the E.coli Top Ten from the cryopreserved tube, streak LB plate, 37 ℃ for the night cultivate.
2)Pick those form is full single colony, inoculation 3 ml LB liquid medium, 250 RPM, 37 ℃ for the night cultivate.
3)1:100 transfer to 100 ml LB liquid medium, 37 ℃, 250 RPM cultivate 2 ~ 3 h, measuring OD600 = 0.35 ~ 0.4 (culture began to take liquid measuring OD after 2.5 hours, every 10 min to measure time, until the OD = 0.4, immediately stop training).
4)Ice precooling one 2 ml EP pipe, take the fungus to EP tubes, each tube take 1.5 ml bacteria liquid, placed on the ice for 5 min.
5)4000 RPM, 4 ℃, centrifugal 5 min.
6)Carefully pour out on the clear, the tube inversion 1 min flow residual liquid on paper towels.
7)Add 500μL precooling CaCl2 0.1 M - l MgCl2 resuspended precipitation to each tube to suspension cells transferred to an ice precooling 2 ml centrifuge tube, the ice bath for 10 min.
8)Centrifugal, 4 ℃ 4000 RPM 5 min, carefully pour out on the qing dynasty, the tube inversion 1 min flow residual liquid on paper towels.
9)Add 100 ice precooling 0.1 M CaCL2 to each tube to suspension bacteria.</P><H4>3.1.2 The transformation of the recombinant DNA:</H4><P>
1)Add 10ligation product to the E.coli Top Ten competent cell, gently blowing with mixed content, placed on the ice for 30 minutes.
2)Tube to be included in the 42 ℃ water bath, just the 90 s. (accurate timing)
3)Fast move the tube to the ice bath, cooling 2 min.
4)Add 400μL LB culture medium to each tube, (heating medium to 37 ℃ in advance), and then transfer the tube to 100 RPM gently table concentrator, 45 min.
5)Use the pipettor to gently blowing even the liquid. The appropriate volume (150μL a tablet, another tablet 250μL) has transformed cells transferred to LB culture medium containing kanamycin on the tablet.
6)Put tablet at room temperature until the liquid is absorbed.
7)Inversion tablet, 37 ℃ cultivating for the night, observe colony.</P><H3>4. SDS alkaline lysis method to prepare small amount of plasmid:</H3><P>
1)Remove the conical flask containing the colonies.
2)Add about 1.5mL bacterium liquid to a 2mL centrifuge tube. Don't put the pollution caused by bacteria liquid spilling out.
3)10000 RPM centrifugal for 2 minutes put on the supernatant liquid in the bottle.
4)Centrifugal again, 10000 RPM for 1 minute, with liquid move on gettering clear as far as possible.
5)Add 100μL cold solution I, then violent oscillation blender, put on ice.
6)Add 200μL solution II, soft reverse centrifuge tube 5 times (do not use the vortex generator), put the ice 4 minutes.
7)Add 150μLcold solution III, soft reverse centrifuge tube 5 times (do not use the vortex generator), put on ice for 5 minutes.
8)12000 RPM centrifuge for 5 minutes. The supernatant fluid is transferred to another 2 ml (about 450 ul) in the centrifuge tube.
9)In the fume hood, add 200μLphenol and 200μL chloroform, isoamyl alcohol (24:1) mixture, violent oscillation, 12000 RPM centrifuge for 5 minutes, be careful for the upper solution (about 450 ul) in another 1.5 ml centrifuge tube.
10)Add 2 times the volume (900 ul) ice precooling of anhydrous ethanol, put on ice for 10 minutes, the precipitation of plasmid DNA.
11)4 ℃ 12000 RPM centrifuge for 5 minutes, abandon the supernatant.
12)Use 1mL ice precooling of 70% ethanol to sediment, gently inverted centrifuge tube washing precipitation. 4 ℃ 12000 RPM centrifugal 1 minute, abandon the supernatant.
13)Room temperature dry precipitation to see no fluid in the tube wall. Add 20μL TE dissolve plasmid, - 20 ℃.</P><H3>5. Identification of plasmid</H3><P>
Recombinant plasmid was identified and sequenced by the method of PCR and double enzyme digestion. Carry on Bi-directional sequencing to the recombinant plasmid which was correct by identification. We contrast the amino acid sequence translated by the DNA sequence with the amino acid sequence of reported Lactococcus lactis alpha acetolactate synthase in the process, to check if mutation is induct to the bacteria before PCR.</P><P>
PCR identification method:</P><P>Table 5</P><P>Attention:</P><P>
1. The enzyme is added at the end, and then put into the PCR for amplification.
2. Electrophoretic analysis of PCR product:
Specific operation is introduced in 3.2: agarose gel electrophoresis, without the need for plastic recycling.
Double enzyme digestion identification method:</P><P>Table 6</P><P>Attention:</P><P>
a)Centrifuge to the bottom of the tube after mixing the components in the tube.
b)Put in 37℃ about 2 hours.
Gel electrophoresis analysis of DNA of plasmid:
Specific operation is introduced in 1: agarose gel electrophoresis, without the need for plastic recycling.</P><H3>6. SDS-PAGE Gel electrophoresis</H3><P>
1.Assemble the gel-irrigating divice according to the instructions and mark the position under the comb 0.7cm.
2.Refer to the tab1, mix the ingredients successively to make the gel, and add the TEMED to the mixture before irrigating the gel.</P><P>Table 7</P><P>
3.Once the TEMED is added, the gel begins to polymerize. It's appropriate to turn the mixture around and continue the next operation.
4.Irrigate solution to two glass plate gap with 1000ul pipette, to the surface to comb 0.7cm, covered with a layer of water in the sol is a small tip carefully.
5.After the polymerization (about 30min) completed, pour the cover layer, use filter paper to absorb the residual moisture content.
6.According to the following formula and order,mix various ingredients for preparing concentrated glue in turn, glue before adding TEMED.</P><P>Table 8</P><P>
7.Irrigate the stacking gel solution directly to the separation gel polymerized directly, stick into a clean comb therein immediately, be careful not to mix the bubbles, and then add some concentrated gum solution completely fills the gap between the comb.
8.After the stacking gel polymerized (about 30min) completely, pull off the comb carefully, wash loading slot with deionized water immediately to remove unpolymerized acrylamide.
9.The glass plate is fixed on the gel electrophoresis apparatus, adding Tris- glycine buffer in the upper and lower slot.
10.Add sample according to the order, each sample with 10ul.Wash pipette tips in the next slot buffer after adding sample every time. In the hole without the sample add an equal volume of 1 x SDS gel sample buffer.
11.The electrophoresis device is connected with the power supply (positive electrode is connected with lower slot), plus 100V voltage on the gel. Until the front of the dye into the separating gel, boost the voltage to 150V, until the bromphenol blue reached the bottom of the separation gel, and then turn off the power.
12.Remove the glass from the electrophoresis apparatus, pry the glass plate carefully, cut off a corner of gel, and remember the cutting corner.
13.The gel is in at least 5 times volume of dye decolorization table, placed on decoloring shaker for at least 30min.
14.Remove the stain, the gel was soaked in the liquid, gently shake the 1-2 hours, replace the destainer 3-4 times.
15.After bleaching, the gel is stored in the water, sealed in plastic bags (adding 20% glycerol preservation).</P><H3>7. Transformation and Selection</H3><P>
Take 200ul competent cell suspension from the -80℃ fridge, unfreeze it in room temperature, and put in on ice at once after unfreezing.
Add pZ plasmid DNA solution (content no more than 50ng, volume no more than 10ul), shake it slightly, and put it on ice for 30min.
After that, heat it in electric-heated thermostatic water bath in 42℃, 90s or 37℃, 5min, then put it on the ice to cool down for 3-5min.
Add 1ml LB liquid medium (without antibiotic), mix it and cultivate it in shaking environment, 37℃ for 1h, which can make the bacteria grow normal again and express the antibiotics resistance gene coded by the plasmids.
Take the mentioned above bacteria solution 100ul after being shaken up and spread it on a plate which includes 3 antibiotics mixed culture, then put it upward for 30min. After the bacteria is absorbed by the medium absolutely, put the petri dish upside down, and cultivate it in 37℃, 16-24h.</P><P>Set the control groups:</P><P>Group 1:</P><P>Use double distilled water to replace the DNA solution, maintain the same operation mentioned above. Under normal circumstances, it's suggested that no colony will appear on the LB plate containing antibiotic.</P><P>Group2:</P><P>
Use double distilled water to replace the DNA solution, and spread 5ul bacteria solution on the LB plate without antibiotic. Under normal circumstances, it's suggested that many colonies will grow on the LB plate.</P><P>Test of results:</P><P>After the whole night cultivation, some colonies may appear on both of the experimental group and the control group. It's a good omen if the number of the experimental group's colonies is more than the control group's. But whether the colonies are wanted depends on the authenticating of DNA recons.</P><H3>8. DNA authenticating</H3><P>Pick up the single colony:
Pick up the single colony from the plate, and inoculate it into 5ml liquid medium containing antibiotic. Then cultivate it for a whole night, 37℃. After that, begin the basic operations to gain the plasmids. Last, use agarose gel electrophoresis (containing 1% agarose gel) to check the results.</P><H3>9.Agarose gel electrophoresis:</H3><P>
The preparation of 1% agarose gel:
(1)Weigh 0.7g (or 0.5g) agarose and put it into an erlenmeyer, add 70ml (or 50ml) 1×TAE, cover the bottleneck with a small beaker. Use a microware oven to heat and boil it three times until the agarose melts absolutely, shake it equably. The solution we get is the 1% agarose gel solution. (Normally, the volume of big gel is 70ml, while the small is 50ml)
(2)The preparation of gel plate:
Take the perspex inside groove (the groove which is used to make gel), wash and dry it, then put it on the glass plate used to making gel. Attach the plate to the bottom of the inside groove with tape, which come into a model.
Put the inside groove on horizontal plane, and put a comb in settled position. Cool down the agarose gel to 65℃ and mix it, then pour it into the inside groove, making the gel solution expand slowly, until the homogeneous gel level appear on the whole plate. Keep the gel quiescent until it becomes solid entirely in room temperature, then pull out the comb vertically and take off the tape. Last, put the gel and inside groove together into the electrophoresis tank. 
Add 1×TAE electrophoresis buffer until it covers the gel plate.
(3)Adding the samples to the groove:
Mix the DNA samples and buffer solution on a sample plate or parafilm. The dilution multiple of the buffer solution should not be less than 1X. Use 10ul micropipet to add the samples to the holes of the gel plate respectively. When we finish the addition of every sample, we should change the tip to avoid pollution. In addition, we should not destroy the gel level around the holds when adding the samples. 
(Notice: Record the sequence of the samples before adding)
(4) Electrophoresis:
After adding the samples, we should electrophorese the gel plate at once. The voltage is 60-100V.
And the samples will move from cathode (Black) to anode (Red). When the voltage rises, the effective segregation of agarose gel will decrease. When bromophenol blue stripes reach the last 1cm of the opposite side of the gel, we stop electrophoresing. 
(5)Detection:
After the electrophoresis, take out the gel, and dye it with 1×TAE solution containing 0.5ug/ml ethidium bromide for 20 min, then rinse it with water for 10 min.
Observation and photography: Observe the gel under ultraviolet lamp, there will be red fluorescent strips if DNA existed, then use gene genius bioimaging system to take photos and save.</P><H3>10.Protocol of EC orthogonal experiment Parameter Measurement And Data Processing</H3><P>
A.Parameter Measurement
1.Choose 4 kinds of Escherichia coli K12 W3110:
（1）PJ23100 - merR - ter &amp; PcadA - CsgA-EC0 - terR- RFP - ter &amp;Ptet-cjblue-ter
（2）PJ23100 - merR - ter &amp; PcadA - CsgA-EC1 - terR- RFP - ter &amp;Ptet-cjblue-ter
（3）PJ23100 - merR - ter &amp; PcadA - CsgA-EC2 - terR- RFP - ter &amp;Ptet-cjblue-ter
（4）PJ23100 - merR - ter &amp; PcadA - CsgA-EC3 - terR- RFP - ter &amp;Ptet-cjblue-ter
（5）PJ23100 - merR - ter &amp; PcadA - CsgA-EC5 - terR- RFP - ter &amp;Ptet-cjblue-ter
（6）Wild tpye E.coli K12
 Put these 4 kinds of Escherichia coli into 4 bottles respectively,37 ℃ shaking bed for the night train, then inoculate them which can express into LB medium, and incubate at 37°C with shaking until the OD600=0.1~0.2.Use the colorimetric utensil to measure O.D., results were compared with LB.

2.Use the LB and CdCl2 to constant volume the concentration of Cd+ into 0、10-8、10-7、10-6、10-5、10-4mol/L,50mL.Parallel to the two groups, a total of 12 bottles.One inoculate the strain whose EC=0,other inoculate EC=3.

3.Cultivate EC = 0,1,2,3,5 and wild type of E. Coli with LB, constant volume the concentration of Cd+ into 0 and 10-6mol/L,50mL,each 5, a total of 10 tube.

4.During the incubation, sample the mixture after 4h、5h、6h、7h、8h、9h、10h、11h、12h、13h、14h、15h、16h、17h、18h、19h、20h、21h、22h、23h、24h.Each sample was 200μL, a total of three parallel samples.

5.Put the sample into a 96 hole coated wells, use the ELIASA to measure O.D., compared with HMM medium. Use the ELIASA to measure the total fluorescence of RFP.

6.Set another sample 2mL, centrifuge at 7500 rpm for 2 min, collect the  supernatant and filtrate.

7. Use the atomic spectrophotometer to measure the concentration of the Cd+ (after filtrate), then deal with the data.</P><H3>11.Measuring and Data processing Scheme of Promoter cadA Parameters</H3><P>
1. Select two kinds of W3110 Escherichia coli.
   （1）J23100 - merR - ter
   （2）J23100 - merR - ter  &amp;  PcadA - RFP - ter
2.Put these four kinds of Escherichia coli respectively in four bottles of LB (200 ml),cultivating in 37 ℃ shaking bed to plateau (OD600 = 1 ~ 1.2).Using the  colorimetric utensil to measure O.D., results were compared with LB.
3. Preparation of Heavy Metal MOPS medium (HMM medium)
   （1）40μM MOPS；
   （2）50 mM KCl；
   （3）10 mM NH4Cl；
   （4）0.5 mM MgSO4；
   （5）0.4% glucose；
   （6）1 mM β-glycerophosphate disodium hydrate；
   （7）1μM FeCl3；
   （8）0.2μg·mL-1 vitaminB1；
   （9）0.25μg·mL-1 carbamide.
4.Using the HMM medium and CdCl2 to constant volume to the concentration of Cd+ into 0, 10-8 , 10-7, 10-6,10-5, 10-4mol·L-1 ,30 ml.
5.Split charging each bottle of LB into 6 tube,30mL each tube, wash for three times with the HMM medium respectively,5000rpm centrifuge for 8 minutes, abandon the supernatant, heavy suspension with the above solution respectively, and eventually made into 30 ml cell suspension. Put the suspension at 100 ml conical flask, 250 rpm, cultivating in 37 ℃ shaking bed and hatch for 24 hours.
6.After starting to hatch, set sample at 4h、5h、6h、7h、8h、9h、10h、11h、12h、13h、14h、15h、16h、17h、18h、19h、20h、21h、22h、23h、24h respectively.
7.Set 200μL sample each time, three parallel samples in total. Then put the sample into a 96 hole coated wells, use the ELIASA to measure O.D., compared with HMM medium. Use the ELIASA to measure the total fluorescence of RFP.</P></DIV></DIV></DIV><FOOTER id="footer1" style="border-top:5px solid #e1472f;"><DIV class="centered-wrapper"><DIV id="topfooter"><DIV class="one-half"><H3>About Us</H3><P class="percent-two-third">
In 2015, we SCUT teams won top ten innovative and entrepreneurial team set up by SCUT.Because of the strong support of the college, our team is being on the right track, and increasing understanding of the subject and experience. 
</P></DIV><DIV class="one-half column-last"><DIV class="percent-one-half column-last"><H3 style="margin-left:20px;">Thanks</H3><UL><LI>Zhang Zhenwu,Prof. Guo Shouqian,Dr. Li, Dr. Li Cheng,Dr. Wang Meng,Chen Kejie</LI><LI>Guangzhou Municipal Environmental Protection Bureau</LI></UL></DIV></DIV></DIV></DIV><DIV id="bottomfooter"><DIV class="centered-wrapper"><DIV class="one-half"><P>COPYRIGHT ©2015-SCUT</P></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>