<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HokkaidoU_Japan_Project_Aggregation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HokkaidoU_Japan/Project/Aggregation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HokkaidoU_Japan/Project/Aggregation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HokkaidoU_Japan/Project/Aggregation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HokkaidoU_Japan/Project/Aggregation&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HokkaidoU_Japan/Project/Aggregation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HokkaidoU Japan/Project/Aggregation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="hokkaidou-header-wrapper"><DIV id="hokkaidou-header"><DIV id="hokkaidou-header-nav"><UL><LI class="hokkaidou-header-nav-alone"><A href="/Team:HokkaidoU_Japan">Home</A></LI><LI class="hokkaidou-header-nav-parent"><A href="/Team:HokkaidoU_Japan/Project">Project</A></LI><LI><A href="/Team:HokkaidoU_Japan/Project/Aggregation">Aggregation module</A></LI><LI><A href="/Team:HokkaidoU_Japan/Project/PHB_Synthesis">PHB synthesis</A></LI><LI><A href="/Team:HokkaidoU_Japan/Project/Biocapsule">Biocapsule</A></LI></UL></DIV><LI class="hokkaidou-header-nav-alone"><A href="/Team:HokkaidoU_Japan/Parts">Parts</A></LI><LI class="hokkaidou-header-nav-alone"><A href="/Team:HokkaidoU_Japan/Human_Practice">Human Practice</A></LI><LI class="hokkaidou-header-nav-parent"><A href="/Team:HokkaidoU_Japan/Notebook">Notebook</A></LI><LI><A href="/Team:HokkaidoU_Japan/Notebook/Spring_Boot_Camp">Boot Camp</A></LI><LI><A href="/Team:HokkaidoU_Japan/Notebook/Lab_Diary">Lab Diary</A></LI><LI><A href="/Team:HokkaidoU_Japan/Notebook/overall_protocols">Protocols</A></LI></DIV><LI class="hokkaidou-header-nav-alone"><A href="/Team:HokkaidoU_Japan/Achievements">Achievements</A></LI><LI class="hokkaidou-header-nav-alone"><A href="/Team:HokkaidoU_Japan/Safety">Safety</A></LI><LI class="hokkaidou-header-nav-alone"><A href="/Team:HokkaidoU_Japan/Team">Team</A></LI></DIV></DIV></DIV><DIV id="hokkaidou-wrapper-bg"><DIV id="hokkaidou-wrapper"><DIV id="hokkaidou-contents"><DIV id="hokkaidou-column-nav"><UL id="hokkaidou-column-nav-accordion"><LI><A href="/Team:HokkaidoU_Japan/Project" title="Team:HokkaidoU Japan/Project">Project Top</A></LI><LI><STRONG class="selflink">Aggregation</STRONG></LI><LI><A href="/Team:HokkaidoU_Japan/Project/PHB_Synthesis" title="Team:HokkaidoU Japan/Project/PHB Synthesis">PHB Synthesis</A></LI><LI><A href="/Team:HokkaidoU_Japan/Project/Biocapsule" title="Team:HokkaidoU Japan/Project/Biocapsule">Biocapsule</A></LI></UL></DIV><DIV id="hokkaidou-column-main"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Planning_of_.22Aggregation_Module.22"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Planning of &quot;Aggregation Module&quot;</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Background_of_.22Aggregation_Module.22"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background of &quot;Aggregation Module&quot;</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Experiments"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Experiments</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Planning_of_.22Aggregation_Module.22">Planning of &quot;Aggregation Module&quot;</SPAN></H2><DIV class="hokkaidou-section"><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig. 1  Large amount of cultivated cells is time-consuming to harvest by centrifugation</DIV></DIV></DIV><P>
In modern industry, two major methods to harvest cultured cells that produce high-value added macromolecules are centrifugation and filtration. Since sizes of Escherichia coli cells are very small, harvesting them by filtration is difficult. So the most popular method for harvesting E. coli is centrifugation. However centrifugation is time-consuming process particularly when harvest them from large volume of culture medium. (Fig. 1)
</P><P>
The purpose of this project is to make more efficient method for macromolecule production in bacterial cells by eliminating time-consuming centrifugation step, through enlarging sizes of E. coli cell clusters (large enough to be captured), named “Bio-capsule”, for harvesting them by filtration on nylon mesh.
</P><P>
For this purpose, we made E. coli expressing adhesion molecule, Antigen 43 (Ag43), on the surface of outer membrane of it for enhancing cell-cell interaction. We have succeeded in harvesting large amount of E. coli cells as filter cake on nylon mesh after continuous rapid strokes of injector (Fig. 2). The “Bio-capsule” resulted from function of the “Aggregation Module” will contribute to the industrial production in near future.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig. 2  Our capturing tools and harvesting method</DIV></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Background_of_.22Aggregation_Module.22">Background of &quot;Aggregation Module&quot;</SPAN></H2><DIV class="hokkaidou-section"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Fig. 3  Cell-to-cell aggregation by Ag43 complex. α43 domain exists at 10 nm above on the E. coli cell surface by non-covelent bond with β43 domain (right). Left picture is actual cluster observed with optical microscopy.</DIV></DIV></DIV></DIV><P>
Ag43 is one of a kind of outer membrane proteins and belongs to auto-transporter family. This protein was derived from E. coli locus: flu, mapping on the E. coli K-12 chromosome. Other E. coli strains also have same gene, but this K-12 strain’s Ag43 cause the most rapid aggregation compared with other strain’s Ag43 [2]. Typical auto-transporter domains: signal peptides, N-proximal passenger domain (α43 : processing sites), autochaperon domain and C-terminal β-barrel domain (β43 : translocation unit) are found in the protein. These domains promote cell-to-cell auto-aggregation through the intermediate function.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig. 4  Structure of Ag43 domain. This image is referenced from [1]. 5 motifs: RGD (orange), aspartyl protease (green), leucine zipper (pink), P-loop (blue) and another RGD (purple) exist in Ag43 complex.</DIV></DIV></DIV><P>
Signal peptide is present at the N-terminus of this protein and translated with α43 and β43. In the presence of this signal, polypeptides consisted of other domains can cross from inner membrane to outer membrane. β43 is mainly composed of β-barrel structure. Some of membrane protein, particularly auto-transporter family has this domain which inserts into the outer membrane to work as anchor for passenger domain, and as pathway that allows passenger domain to cross from periplasm to outer membrane. Another domain which exists in the β43, the auto-chaperon domain stabilizes in near the outer pore of  β-barrel then the domain associates folding of secreted passenger domain. α43 has β-helical structure and some identified motifs, e.g., aspartyl protease site, a leucine zipper motif and RGD motif. But none of the motifs are conserved in all agn43 alleles.(Fig. 4) That means the motifs does not play a functional role but plays a role in the structural integrity [2]. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig. 5  Image of secretion of Ag43. IM is Innner Membrane, OM is Outer Membrane, SP is Signal Peptides, α is α43 domain and β is β43 domain. α43 forms β-helix structure and β43 forms β-barrel structure.</DIV></DIV></DIV><P>
The actual secretional function of Ag43 is not still known. But about AIDA-I, One of the auto-transporter protein belonging to same group as Ag43, it is said that the AIDA-I passenger domain is auto-catalytically cleaved from complex with β-barrel and autochaperon after secreted, then remain on the outside of cell membrane with non-covalently bound to translocator β-barrel structure. This process would correspond to Ag43 because Ag43 is belonging to same family and has sequence similarity with AIDA-I [3]. It is generally confirmed that the structural and sequential similarity is correlated with functional similarity in proteomics. And the fact that the passenger domain is cleaved in the 60C will be additional evidence of non-covalent binding between each cell’s Ag43 [4]. The presence of transported Ag43 α domain is predicted in about 10nm of cell surface, so the cells that will aggregate with each other needs to be very close. If there were some obstacles near cells, they cannot aggregate. In other words, if there were rich other large proteins or cell surface structure, e.g., type 1 fimbriae and flagella on the cell membrane, Ag43 would not promote aggregation (Fig. 5) [5].
</P></DIV><H2><SPAN class="mw-headline" id="Experiments">Experiments</SPAN></H2><DIV class="hokkaidou-section"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:382px;"><DIV class="thumbcaption">Fig. 6  Schematics of pBAD-RBS-Ag43-dT and pBAD-RBS-eCFP-RBS-Ag43-dT construct</DIV></DIV></DIV><P>
We appreciate to previous iGEM teams, especially iGEM 2010 Peking University team. We could easily get characterized Ag43 gene derived from K-12 strain flu gene. Compared with aggregation module of PKU iGEM 2010, our aggregation module is induced directly and executes aggregation more rapidly. We assembled a construct which is composed of pBAD+AraC (BBa_I0050) as promoter, ribosome binding site: RBS (BBa_B0034), Ag43 as protein coding region (BBa_K346007) and double terminator: dT (BBa_B0015) on a low copy plasmid pSTV28. (Fig. 6)
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig. 7  An image of Ag43 expression induced by L-arabinose. If you added L-arabinose to cell culture, AraC binds with it and reveal from pBAD region and the construct is started to transcription.</DIV></DIV></DIV><P>
The induction of pBAD promoter is negatively controlled by AraC repressor. In the absence of L-arabinose, an AraC protein remains in pBAD promoter region and prevents transcription. In the presence of L-arabinose, AraC is released from pBAD by conformational change, and then RNA polymerase is allowed to start transcription of downstream sequence. Consequently, in this devise, we can induce Ag43 protein by addition of L-arabinose. And we inserted additional gene, eCFP: Cyan Fluorescent protein (BBa_E0020), upstream of Ag43 gene as indicator of expression of “Aggregation Module” (Fig. 7).
</P><P>
Furthermore, we executed codon optimization of Ag43 by JCat [7] and replaced 6 recognition sites of PstI which originally exists in wild-type Ag43 sequence with sequences suitable for BioBrick assembly. By this optimization, we would be able to improve the translational frequency of Ag43 and accelerate the time required for aggregation.
</P><P>
To aggregate cells each other, L-arabinose was added into LB liquid medium with appropriate antibiotics for induction of the function of “Aggregation Module”, and cultured in glass test tubes. The plastic tube was not suitable for producing large cluster of E. coli cells, since a strong affinity of E. coli expressing Ag 43 was observed onto the surface of inside of plastic test tube as shown in image 8. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Image 8  Picture of cultivation in plastic tubes for 12 hrs and 16 hrs. Plasmid structure is pBAD-RBS-Ag43-dT-pSB1C3</DIV></DIV></DIV></DIV><P>
Meanwhile, we tested the same observation using 300 ml Erlenmeyer flask containing 50 ml LB liquid medium and cultivated at 34C (See <A href="https://2012.igem.org/Team:HokkaidoU_Japan/Notebook/aggregation_protocols" class="external text" rel="nofollow">protocol page</A> for experimental procedure).
</P><P>
We found that optimum temperature required for efficient aggregation by “Aggregation Module” is 34~37C. We didn’t observe aggregation at 30C even after 24 hr-culture. 
Reciprocal shaking of culture flask seemed to be better than rotation.
</P><P>
After culturing the aggregated E. coli, the clusters were harvested on nylon mesh by continuous strokes of injector (Fig. 9)(Image10). 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:332px;"><DIV class="thumbcaption">Fig. 9  Image of harvesting method using nylon mesh by continuous strokes of injector. Only large clusters are harvested on nylon mesh</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:292px;"><DIV class="thumbcaption">Image10  Picture of our actual harvesting tools</DIV></DIV></DIV><P>
We succeeded in filtering them by nylon mesh which has 250 um pore size then we have gained E. coli cake (Image11). And as our experimental fact, we filtered them by 200 um pore size mesh too. This result shows that we would be able to gain target product by this devise if we cultivated E. coli which has ability to produce target product.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Image11  Cell cake on a nylon mesh (pore size: 250 um)</DIV></DIV></DIV></DIV><P>
Induction of function of “Aggregation Module” was tested by observing appearance of aggregated E. coli cluster in the presence (right) or absence (left) of L-arabinose in culture medium (See Image12).  Aggregated E. coli clusters were observed only in the culture medium containing L-arabinose. The results suggest that the auto-aggregation of E. coli is mediated by overexpressed Ag43.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Image12  Induction of function of “Aggregation Module”. Appearance of aggregated E. coli in the presence (right) or absence (left) of L-arabinose in culture medium. Plasmid structure is pBAD-RBS-eCFP-RBS-Ag43-dT-pSTV28 (low copy plasmid), and culture condition is at 37C for 24 hrs at 130 rpm.</DIV></DIV></DIV></DIV><P>
Besides above experiments, we tried to confirm a relationship between Ag43 expression and aggregation by monitoring signal associated with eCFP (indicator module) fused to “Aggregation Module”. During culture of the cells, we monitored change of OD600 as indicator of cell growth, and measured emission associated with eCFP as indicator of expression of “Aggregation Module” (See <A href="https://2012.igem.org/Team:HokkaidoU_Japan/Notebook/aggregation_protocols" class="external text" rel="nofollow">protocol page</A> for experimental procedure). Plasmid structure is pBAD-RBS-eCFP-RBS-Ag43-dT-pSB1C3 and we cultivated  at 37C for 12 hrs at 180 rpm.
</P><P>
The result showed there is relation between optical density and aggregation. At the same time cultivating cells, OD600 of each cell culture's suspension are increasing, but increasing ratio of arabinose added culture (Arabinose+) became lower than negative control (Arabinose-) (Image13). On the other hand, we couldn't observe the expression of eCFP.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Image13  Graph of OD600 of suspension and culture time (hours).</DIV></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><DIV class="hokkaidou-section"><P>[1]  van der Woude, Marjan W Henderson, Ian R. Regulation and function of Ag43 (flu). Annu. Rev. Microbiol. 2008. 62:153–69 <A href="http://www.ncbi.nlm.nih.gov/pubmed/18785838" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/18785838</A></P><P>
[2] P Caffrey and P Owen. Purification and N-terminal sequence of the alpha subunit of antigen 43, a unique protein complex associated with the outer membrane of Escherichia coli. J. Bacteriol. 1989, 171(7):3634. <A href="http://www.ncbi.nlm.nih.gov/pubmed/2661530" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/2661530</A></P><P>
[3] Inga Benz, M. Alexander Schmidt. Structures and functions of autotransporter proteins in microbial pathogens. International Journal of Medical Microbiology 301 (2011) 461– 468. <A href="http://www.ncbi.nlm.nih.gov/pubmed/21616712" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/21616712</A></P><P>
[4]Glen C. Ulett, Jaione Valle, Christophe Beloin, Orla Sherlock, Jean-Marc Ghigo and Mark A. Schembri. Functional Analysis of Antigen 43 in Uropathogenic Escherichia coli Reveals a Role in Long-Term Persistence in the Urinary Tract Infect. Immun. 2007, 75(7):3233. DOI: 10.1128/IAI.01952-06. <A href="http://iai.asm.org/content/75/7/3233.abstract" class="external free" rel="nofollow">http://iai.asm.org/content/75/7/3233.abstract</A></P><P>
[5] <A href="https://2010.igem.org/Team:Peking/Project/Bioabsorbent/InductiveAggregation" class="external free" rel="nofollow">https://2010.igem.org/Team:Peking/Project/Bioabsorbent/InductiveAggregation</A></P><P>
[6] <A href="http://openwetware.org/wiki/Arking:JCAOligoTutorial8" class="external free" rel="nofollow">http://openwetware.org/wiki/Arking:JCAOligoTutorial8</A></P><P>
[7] <A href="http://www.jcat.de/" class="external free" rel="nofollow">http://www.jcat.de/</A></P></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:HokkaidoU_Japan/Project/Aggregation">http://2012.igem.org/Team:HokkaidoU_Japan/Project/Aggregation</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HokkaidoU_Japan/Project/Aggregation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HokkaidoU_Japan/Project/Aggregation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HokkaidoU_Japan/Project/Aggregation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HokkaidoU_Japan/Project/Aggregation&amp;oldid=236843" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>