<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Michigan_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Michigan/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV class="navbar navbar-default"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:Michigan">Home</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Michigan/Project" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Results">Results &amp; Discussion</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Parts">Parts</A></LI></UL><LI class="dropdown"><A href="https://2015.igem.org/Team:Michigan/Notebook" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Notebook/June">June</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Notebook/July">July</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Notebook/August">August</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Notebook/September">September</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Modeling">Modeling</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Michigan/Practices" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Diseases">Diseases and Applications</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Current">Current Detection Methods</A></LI><LI class="dropdown dropdown-submenu"><A href="https://2015.igem.org/Team:Michigan/Informed" class="dropdown-toggle" data-toggle="dropdown">Informed Design </A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Survey">Survey</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Biochemical">Biochemical Industry</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Food">Food Science Industry</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Safety">Safety</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Michigan/Team" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Meet">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Advisors">Advisors</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Michigan/Outreach">Outreach</A></LI></DIV></NAV><H2 style="text-align: center">Experiments and Protocols</H2><H3>Ligation Protocol with T4 DNA Ligase</H3>
1.	Set up the following reaction in a microcentrifuge tube on ice.
2.	(T4 DNA Ligase should be added last. Note that the table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.) 
* The T4 DNA Ligase Buffer should be thawed and resuspended at room temperature.
3.	Gently mix the reaction by pipetting up and down and microfuge briefly.
4.	For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
5.	For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours
6.	Heat inactivate at 65°C for 10 minutes.
7.	Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells.<H3>High Efficiency Transformation Protocol</H3>
1.	Thaw a tube of NEB 5-alpha Competent E. coli cells on ice for 10 minutes. 
2.	Thaw a tube of NEB 5-alpha Competent E. coli cells on ice until the last ice crystals disappear. Mix gently and carefully pipette 50 µl of cells into a transformation tube on ice.
3.	Add 1-5 µl containing 100 ng of plasmid DNA to the cell mixture. Carefully flick the tube 4-5 times to mix cells and DNA. Do not vortex.
4.	Place the mixture on ice for 30 minutes. Do not mix.
5.	Heat shock at exactly 42°C for exactly 30 seconds. Do not mix.
6.	Place on ice for 5 minutes. Do not mix.
7.	Pipette 950 µl of room temperature SOC into the mixture.
8.	Place at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate.
9.	Warm selection plates to 37°C.
10.	Mix the cells thoroughly by flicking the tube and inverting, then perform several 10-fold serial dilutions in SOC.
11.	Spread 50-100 µl of each dilution onto a selection plate and incubate overnight at 37°C. Alternatively, incubate at 30°C for 24-36 hours or 25°C for 48 hours.<H3>QIAquick PCR Purification Kit Protocol using a microcentrifuge</H3>
This protocol is designed to purify single- or double-stranded DNA fragments from PCR and other enzymatic reactions. Fragments ranging from 100 bp to 10 kb are purified from primers, nucleotides, polymerases, and salts using QIAquick spin columns in a microcentrifuge.<B>Important points before starting:</B>
■ Add ethanol (96–100%) to Buffer PE before use (see bottle label for volume). 
■ All centrifugation steps are carried out at 17,900 x g (13,000 rpm) in a conventional tabletop microcentrifuge at room temperature. 
■ Add 1:250 volume pH indicator I to Buffer PB (i.e., add 120 µl pH indicator I to 30 ml Buffer PB or add 600 µl pH indicator I to 150 ml Buffer PB). The yellow color of Buffer PB with pH indicator I indicates a pH of #7.5. 
■ Add pH indicator I to entire buffer contents. Do not add pH indicator I to buffer aliquots. 
■ If the purified PCR product is to be used in sensitive microarray applications, it may be beneficial to use Buffer PB without the addition of pH indicator I.<B>Procedure</B>
1.	 Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix. It is not necessary to remove mineral oil or kerosene. For example, add 500 µl of Buffer PB to 100 µl PCR sample (not including oil). 
2.	If pH indicator I has beein added to Buffer PB, check that the color of the mixture is yellow. If the color of the mixture is orange or violet, add 10 µl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn to yellow. 
3.	Place a QIAquick spin column in a provided 2 ml collection tube. 
4.	To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s. 
5.	Discard flow-through. Place the QIAquick column back into the same tube. Collection tubes are re-used to reduce plastic waste. 
6.	To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s. 
7.	Discard flow-through and place the QIAquick column back in the same tube. Centrifuge the column for an additional 1 min. IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation. PCR Purification Spin Protocol 
8.	Place QIAquick column in a clean 1.5 ml microcentrifuge tube. 
9.	To elute DNA, add 50 µl Buffer EB (10 mM Tris·Cl, pH 8.5) or water (pH 7.0–8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min. Alternatively, for increased DNA concentration, add 30 µl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge. IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volume is 48 µl from 50 µl elution buffer volume, and 28 µl from 30 µl elution buffer. Elution efficiency is dependent on pH. The maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C as DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE buffer (10 mM Tris·Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions. 
10.	If the purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel. Loading dye contains 3 marker dyes (bromophenol blue, xylene cyanol, and orange G) that facilitate estimation of DNA migration distance and optimization of agarose gel run time. Refer to Table 2 (page 15) to identify the dyes according to migration distance and agarose gel percentage and type.<H3>QIAquick Gel Extraction Kit Protocol using a microcentrifuge</H3>
This protocol is designed to extract and purify DNA of 70 bp to 10 kb from standard or low-melt agarose gels in TAE or TBE buffer. Up to 400 mg agarose can be processed per spin column. This kit can also be used for DNA cleanup from enzymatic reactions. For DNA cleanup from enzymatic reactions using this protocol, add 3 volumes of Buffer QG and 1 volume of isopropanol to the reaction, mix, and proceed with step 6 of the protocol. Alternatively, use the new MinElute Reaction Cleanup Kit.<B>Notes:</B>
• The yellow color of Buffer QG indicates a pH ≤7.5. 
• Add ethanol (96–100%) to Buffer PE before use (see bottle label for volume). 
• Isopropanol (100%) and a heating block or water bath at 50°C are required. 
• All centrifugation steps are carried out at ≥10,000 x g (~13,000 rpm) in a conventional table-top microcentrifuge. 
• 3 M sodium acetate, pH 5.0, may be necessary.<B>Protocol</B>
1.	Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose. 
2.	Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 µl). For example, add 300 µl of Buffer QG to each 100 mg of gel. For &gt;2% agarose gels, add 6 volumes of Buffer QG. The maximum amount of gel slice per QIAquick column is 400 mg; for gel slices &gt;400 mg use more than one QIAquick column. 
3.	Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation. IMPORTANT: Solubilize agarose completely. For &gt;2% gels, increase incubation time. 
4.	After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 µl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn to yellow. The adsorption of DNA to the QIAquick membrane is efficient only at pH ≤7.5. Buffer QG contains a pH indicator which is yellow at pH ≤7.5 and orange or violet at higher pH, allowing easy determination of the optimal pH for DNA binding. 
5.	Add 1 gel volume of isopropanol to the sample and mix. For example, if the agarose gel slice is 100 mg, add 100 µl isopropanol. This step increases the yield of DNA fragments &lt;500 bp and &gt;4 kb. For DNA fragments between 500 bp and 4 kb, addition of isopropanol has no effect on yield. Do not centrifuge the sample at this stage. Protocol QIAquick Spin Handbook 03/2001 24 
6.	Place a QIAquick spin column in a provided 2 ml collection tube. 
7.	To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min. The maximum volume of the column reservoir is 800 µl. For sample volumes of more than 800 µl, simply load and spin again. 
8.	Discard flow-through and place QIAquick column back in the same collection tube. Collection tubes are re-used to reduce plastic waste. 
9.	(Optional): Add 0.5 ml of Buffer QG to QIAquick column and centrifuge for 1 min. This step will remove all traces of agarose. It is only required when the DNA will subsequently be used for direct sequencing, in vitro transcription or microinjection. 
10.	To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min. Note: If the DNA will be used for salt sensitive applications, such as blunt-end ligation and direct sequencing, let the column stand 2–5 min after addition of Buffer PE, before centrifuging. 
11.	Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at ≥10,000 x g (~13,000 rpm). IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation. 
12.	Place QIAquick column into a clean 1.5 ml microcentrifuge tube. 
13.	To elute DNA, add 50 µl of Buffer EB (10 mM Tris·Cl, pH 8.5) or H2O to the center of the QIAquick membrane and centrifuge the column for 1 min at maximum speed. Alternatively, for increased DNA concentration, add 30 µl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min. IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volume is 48 µl from 50 µl elution buffer volume, and 28 µl from 30 µl. Elution efficiency is dependent on pH. The maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C as DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE (10 mM Tris·Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions.<H3>Phenol | Chloroform Extraction</H3>
1.	Add one volume of phenol:chloroform:isoamyl alcohol (25:24:1) to your sample, and vortex or shake by hand thoroughly for approximately 20 seconds.
2.	Centrifuge at room temperature for 5 minutes at 16,000 × g. Carefully remove the upper aqueous phase, and transfer the layer to a fresh tube. Be sure not to carry over any phenol during pipetting.
3.	Proceed to Ethanol Precipitation, below.<H3>Ethanol Precipitation</H3>
1.	Add the following reagents to the aqueous phase, in the listed order in above table
2.	Place the tube at –20°C overnight to precipitate the DNA from the sample.  Note: If you wish to continue with the protocol, place the tube in dry ice or at –80°C for at least 1 hour.
3.	Centrifuge the sample at 4°C for 30 minutes at 16,000 × g to pellet the cDNA.
4.	Carefully remove the supernatant without disturbing the cDNA pellet.
5.	Add 150 μL of 70% ethanol. Centrifuge the sample at 4°C for 2 minutes at 16,000 × g. Carefully remove the supernatant.
6.	Repeat Step 3 once. Remove as much of the remaining ethanol as possible.
7.	Dry the cDNA pellet in a SpeedVac® for 2 minutes or at room temperature for 5–10 minutes.
8.	Resuspend the cDNA pellet in 300 μL of TEN buffer by pipetting up and down 30–40 times.
9.	Centrifuge briefly to collect the sample, and place the tube on ice.<H3>In Vitro Cell Free Expression Reactions Protocol</H3>
These formulations allow an increase in the &quot;user added&quot; volume (for template, supplements, etc.); tolerating up to 20% over volume (30 μl reaction total) without an appreciable drop in productivity. 
					
Template DNA, particularly plasmid DNA prepared by mini-prep (e.g. Qiagen) is often the major source of RNase contamination. We strongly recommend adding 20 units RNase Inhibitor, Murine (NEB #M0314) in each reaction. 
									
Add Solution B to Solution A, do not dilute Solution B unbuffered. We recommend a starting concentration of 250 ng template DNA per 25 μl reaction. The optimal amount of input DNA can be determined by setting up multiple reactions and titrating the amount of template DNA added to the reaction. Typically, the optimal amount will fall in a range of 25–1000 ng template. 
	
Incubate at 37°C for 2 hours, stop reactions by placing solutions in ice						
We recommend using an incubator rather than a water bath, to prevent evapora- tion. Some reactions can benefit from an additional 2 hours (4 hours total) of incubation to achieve maximum yield. Some proteins are also more soluble at reduced temperatures; however, incubating reactions below 37°C will likely reduce yield.<H3>PURExpress® In Vitro Protein Synthesis</H3>
Complete manual can be found here: https://www.neb.com/~/media/Catalog/All-Products/0D1F4E4BB3F14EFC9DF22C6463654CE4/Datacards%20or%20Manuals/manualE6800.pdf

<H3>Plasmid DNA Purification using the QIAprep Spin Miniprep Kit</H3>
1.	Resuspend pelleted bacterial cells in 250 μl Buffer P1 and transfer to a micro- centrifuge tube						
2.	Add 250 μl Buffer P2 and mix thoroughly by inverting the tube 4–6 times.
							
3.	Mix gently by inverting the tube. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear. Do not allow the lysis reaction to proceed for more than 5 min.						
4.	Add 350 μl Buffer N3 and mix immediately and thoroughly by inverting the tube 4–6 times.							
5.	To avoid localized precipitation, mix the solution thoroughly, immediately after addition of Buffer N3. Large culture volumes (e.g. ≥5 ml) may require inverting up to 10 times. The solution should become cloudy.
							
6.	Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge. A compact white pellet will form.
		
7.	Apply 800 μl of the supernatant from step 4 to the QIAprep 2.0 spin column by pipetting.			
8.	Centrifuge for 30–60 s. Discard the flow-through.
			
9.	Recommended: Wash the QIAprep 2.0 spin column by adding 0.5 ml Buffer PB and centrifuging for 30–60 s. Discard the flow-through.
										
10.	Wash QIAprep 2.0 spin column by adding 0.75 ml Buffer PE and centrifuging for 30–60 s.
			
11.	Discard the flow-through, and centrifuge at full speed for an additional 1 min to remove residual wash buffer.
							
12.	Important: Residual wash buffer will not be completely removed unless the flow-through is discarded before this additional centrifugation. Residual ethanol from Buffer PE may inhibit subsequent enzymatic reactions.
		
Place the QIAprep 2.0 column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 25ul of Ultra Pure water, let it sit at room temperature for one minute and centrifuge for 1 minute. Repeat.<H3>LB medium (1L liquid)</H3>
■	10 g tryptone
■	10 g NaCl
■	5 g yeast extract
■	Water<H3>Setting up a Double Digestion </H3>
Double digests with NEB's restriction enzymes can be set up in CutSmart Buffer. Otherwise, choose an NEBuffer that results in the most activity for both enzymes.

Set up reaction according to recommended protocol. The final concentration of glycerol in any reaction should be less than 5% to minimize the possibility of star activity. For example, in a 50 µl reaction, the total amount of enzyme added should not exceed 5 µl.

If two different incubation temperatures are necessary, choose the optimal reaction buffer and set up reaction accordingly. Add the first enzyme and incubate at the desired temperature. If the enzyme is heat inactivatable, a heat inactivation step is recommended. Add the second enzyme and incubate at the recommended temperature. 



*All protocols in this section were obtained online at qiagen.com, thermofisher.com,  and neb.com for their respective products.</DIV></DIV></DIV></DIV></BODY></HTML>