Team:Michigan/Design
Home
Project
Design
Experiments & Protocols
Results & Discussion
Collaborations
Parts
Notebook
June
July
August
September
Modeling
Human Practices
Background
Diseases and Applications
Current Detection Methods
Informed Design
Survey
Biochemical Industry
Food Science Industry
Safety
Team
Meet the Team
Advisors
Attributions
Sponsors
Achievements
Outreach
Design
While there are many genetic switches for detecting proteins, to our knowledge, every genetic switch is unique to the protein it detects. Aptapaper seeks to create a switch that could have the same logic adapted to any protein, similar to how toehold switches can be adapted and respond to any sequence of trigger RNA. Toehold switches refer to an RNA switch that works by sequestering the ribosomal binding site and start codon in a hairpin, while leaving a portion of what the RNA trigger binds to exposed in a “toehold” region.  This toehold region dramatically changes the kinetics of the system, making it much easier for the trigger to bind.  After the initial binding, the hairpin is “unzipped” by the trigger1. Exact design specs of the previously optimized second generation toehold design are detailed in Figure 1.  Because toehold switches can be adapted to respond to any RNA trigger, Aptapaper attempted to use aptamers in conjunction with toehold switches in the Switch 1.0 design. While Switch 1.0 was unsuccessful, it inspired our Switch 2.0 design that responded to the lowest protein concentrations tested. See different design approaches below.
Figure 1: Forward Engineered Switches Inspiring Aptapaper Switch Design A.) RNA switch without trigger forms a stable hairpin structure, featuring a 15nt toehold region at 5’ end, 18nt total hairpin stem, 3nt AUG bubble (yellow), 15nt bubble sequestering 8nt ribosome binding site (green).  Bubbles for AUG and ribosomal site allow switch to be activated by RNA triggers with any sequence.   B.) RNA trigger (red) easily binds exposed toehold region.  C.) RNA trigger binds to hairpin region of switch, forcing hairpin to unbind.  D.) RNA trigger has fully bound switch, exposing the ribosome binding site so translation can be initiated1.
Switch 1.0 - Trigger-Aptamer Release for Toehold
Figure 2: Switch 1.0 Design A.) Protein of specificity is not present, so aptamer (blue) binds trigger (red).  Toehold switch in hairpin conformation sequesters ribosome binding site.  B.) Protein of specificity (gray) binds aptamer (blue), forcing trigger (red) to be released.  Trigger can then easily bind exposed toehold region of switch.    C.) Trigger binds to hairpin region of switch, forcing hairpin to unbind.  D.) Trigger has fully bound switch, exposing the ribosome binding site so translation can be initiated.
In this design, a DNA trigger is bound by a DNA aptamer and “junk” DNA.  The aptamer binds the protein of specificity when it is present, displacing the trigger, which can then activate the toehold switches previously discussed.  While the system was based on the same DNA release sequence as validated for protein detection via fluorophore unquenching (FRET)2, the 5’ 3’ orientation is reversed so that no part of toehold region is unbound when hybridized to the trigger.  The toehold region of the switch used in our system is also 12nt instead of the 15nt seen in forward engineered switches for the same reason. Additionally, this design used a DNA trigger to activate the RNA switch, a previously untested configuration.
Because the toehold binding region of the trigger is complementary to the aptamer, this design necessitates that an RNA sequence identical to a portion of the DNA aptamer is exposed in the toehold region of the switch.  It was unclear if the RNA version of the DNA aptamer would maintain some affinity for thrombin.  The results from this design suggest that the toehold region may in fact be functioning as an aptamer, which immediately suggests the two designs seen below.
Switch 2.0 - Toehold with Protein “Trigger”
Figure 3: Switch 2.0 Design A.) When no protein of specificity (gray) is bound, switch is hairpin conformation, sequestering the ribosome binding site (green).  B.) Protein binds aptamer (blue), which is exposed in a “toehold” region to making binding initiation easier.  Aptamer comprises 5’ side of hairpin, so hairpin is forced to unbind when aptamer conforms to bind protein.
This design takes advantage of the aptamer  in the toehold and hairpin regions of the switch by having the protein intentionally bind to this region, acting like a trigger to unfold the hairpin. In Switch 2.0, the 3’ end of the aptamer ends at the start of the ribosome binding site bubble, extends into the 15nt hairpin, and then the remainder of the aptamer forms the toehold region.  The hairpin was destabilized from the forward engineered design by shortening the hairpin from 18nt to 15nt and adding 3nt to the ribosome binding site loop.  One important parameter which must be optimized in this design is the number of bases in the aptamer which are exposed in the toehold region. If too much aptamer is exposed the protein could bind without unfolding the hairpin; too little, however, and the protein may not bind at all. Additionally, the hairpin structure could be too weak to stay bound under the low salt, high temperature conditions suited for in vitro translation. Additionally, depending on the way the aptamer recognizes the protein, a 3D structure critical for protein detection could be sequestered in the hairpin.  Despite these concerns, the switch worked well for one protein thus far (see Results and Discussion section), with another currently being tested.
Reaction Conditions
Temperatures ideal for hybridization of short DNA oligos are lower and salt concentrations are higher than that recommended for in vitro translation.  In the Switch 1.0 design, it is likely that the trigger and aptamer did not bind, allowing uninhibited switch induction in the negative controls.  In order to find conditions that are acceptable for in vitro translation that also allow hybridization of short DNA oligos, higher salt concentrations are currently being tested with a control switch.  Lower temperatures could also allow hybridization to occur.
References
Green A, Silver P, Collins J, Yin P. Toehold Switches: De-Novo-Designed Regulators of Gene Expression. Cell. 2014;159(4):925-939. doi:10.1016/j.cell.2014.10.002.
Nutiu R, Li Y. Structure-Switching Signaling Aptamers: Transducing Molecular Recognition into Fluorescence Signaling. J Am Chem Soc. 2003;125:4771-4778. doi:10.1021/ja028962.
