"
Page
Discussion
View source
History
teams
Log in
 
Team:Colombia/Notebook/Journal
From 2012.igem.org
Home
The Team
The Team
Undergrads
Grad Students
Instructors
Pest Busters
The Problem
The Basics
Our Design
Experiments and Results: Toxin Antitoxin module
Experiments and Results: Ralstonia
Experiments and Results: Aliivibrio and Streptomyces
Parts
Modeling
Diferential Equations
Parameters
Stochastic Model
Scripting
Results
Implementation Model
Human Practices
Social: Schools and Coffee Growers
Research in Colombia
SynthEthics: A Phylosophical Debate
Essay: The Impact of Genetic Technologies on Society
Notebook
Journal
Meetings
Classes
Protocols
How To: Parts
Human Practices
Safety
Biosafety Provisions
Hazard
Biological Parts and Devices
Attributions
Gallery
University & Jamboree
UniAndes
Latin American Jamboree 2012
Our Sponsors
iGEM
Contents
1The Journal
1.1Ralstonia solanacearum
1.1.1June 8
1.1.2June 12
1.1.3June 13
1.1.4June 15
1.1.5June 16
1.1.6June 18
1.1.7June 19
1.1.8June 20
1.1.9June 21
1.1.10June 22
1.1.11June 24
1.1.12June 26
1.1.13June 27
1.1.14June 28
1.1.15June 29
1.1.16July 3
1.1.17July 5
1.1.18July 6
1.1.19July 7
1.1.20July 10
1.1.21July 12
1.1.22July 13
1.1.23July 16
1.1.24July 17
1.1.25July 18
1.1.26July 23
1.1.27July 27
1.1.28July 28
1.1.29July 30
1.1.30August 1
1.1.31August 2
1.1.32August 3
1.1.33August 4
1.1.34August 5
1.1.35August 6
1.1.36August 7
1.1.37August 10
1.1.38August 13=
1.1.39August 15
1.1.40August 17
1.1.41August 18
1.1.42August 19
1.1.43August 20
1.1.44August 21
1.1.45August 23
1.1.46September 3=
1.1.47September 4
1.1.48September 7
1.1.49September 8
1.1.50September 10
1.1.51September 12
1.1.52September 13
1.1.53September 17
1.1.54September 18
1.1.55September 19
1.1.56September 20
1.1.57September 21
1.1.58September 22
1.1.59September 23
1.1.60September 24
1.2Mathematical Models
1.2.1Up to June 6
1.2.2July 3
1.2.3July 10
1.2.4July 12
1.2.5July 16
1.2.6July 19
1.2.7August 8
1.2.8August 15
1.2.9August 26
1.3Cut and Paste
1.4Aliivibrio/Streptomyces
1.5Chitinase
The Journal
Ralstonia solanacearum
June 8
Today we had our first meeting! We introduced ourselves and basically talked about Ralstonia solanacearum detection system and made a little research to find a proper culture medium in order to grow our bacteria. Here is a little sketch we made.
June 12
We decided to prepare Casamino acid-Peptone-Glucose (CPG) media, which is a rich medium that provides everything that Ralstonia solanacearum needs to grow and we have all the components in the lab.  This is the recipe for 1L of CPG:
Reactives
Amount (g)
Casamino acid
1
Peptone
10
Glucose
5
Agar
15
June 13
We grew a Ralstonia solanacearum strain in solid medium from the REVCO, it belongs to the phytopathogen bacteria strain store at the LAMFU, it will fully grow in 2 days, so we have to wait until then…everything is pretty easy so far.
June 15
We were going to extract DNA from Ralstonia tomorrow, so today we spent a lot of time preparing all the solutions needed. Ralstonia strain didn’t grow, we are kind of upset. We also designed the primers to amplify the promoter of xpsR(PxpsR) We used the sequence of Ralstonia solanacearum AW (gi:3132834)as this promoter sequence has been previously characterized by Huanget al. in 1998.
June 16
Today is a new day, we decided to grow a different accession of Ralstonia solanacearum and see what happens. Hopes up!
June 18
Our bacterium is alive! it doesnt matter that today’s hollyday anymore and we are at lab….We are finally extracting DNA tomorrow, so we inoculated 5 ml of liquid CPG with a few colonies from the solid culture and left it growing ON.
June 19
Today was DNA extraction day! We used Xam’s DNA extraction protocol(see protocols) and here it is, our Ralstonia’s DNA, we think is a little bit degraded but still amplifiable…
June 20
Today we were supposed to amplify all the genes for the detection system but PxpsR (those primers haven’t  arrive yet). We used Fermentas Pfu (see protocols) and the Ta for each gene was calculated taking the lower primer Tm of each couple and subtracting it 2 degrees (phcS=61°C, phcR=57°C, phcA=54°C). Nonetheless, none amplified. We are no longer sure if the primers anneal in all Ralstonia strains, we haven’t consider that before.
June 21
We performed a primer search in NCBI and confirmed our suspects … there are some base changes at the 3’ of our primers in the sequence of different strains of R. solanacearum…that will explain why it didn’t amplified a thing yesterday. The most conserved sequence appear to be PxpsR and then phcA, so, while PxpsR primers arrive, we will perform a screening of differtent Ralstonia solanacearum strains using phcA, then we will try to amplify the other genes from positive strains. Here is our decision tree.
June 22
Today we prepared the selective media for the screening; we had to make substantial variations due to a lack of compounds in our lab. This was the final composition:
Reactives
Amount
Mannitol
1g
Na2HPO4
3g
KH2PO4
3g
NH4Cl
1g
MgSO4
0.25g
FeSO4
5mg
Crystal Violet
3mg
Cycloheximide
5mg
Chloramphenicol
1mg
Bacitracine
0.25mg
Agar
15g
Distilled water
1L
June 24
We decided to make a pilot experiment in order to standardize  the conditions of phcA amplification before starting with the massive screening . We choose #37 strain randomly for the standardization.
June 26
In order to standardize  the conditions of phcA amplification before starting with the massive screening , we determine the annealing temperature (71°C) by performing an in silico PCR (FAST PCR) and used that temperature to calculate a  temperature gradient (form 64 to 71), reactions were carried out with and without DMSO. The results weren’t expected, none of the temperatures amplified.
June 27
We perfomed phcA PCRs using boiling from the solid culture as a source of DNA. The results remained the same.  Due to results we decided to change of strains and we grew on solid 3 new accessions.
June 28
The primers for the promoter of xpsR finally arrived! We intented to amplify this promoter due to the conservation of the secuence using the DNA previously extracted and using a Ta of 46°C. We expected a band of 389pb but we couldn’t even see primer dimmers.
June 29
We are finally in the right direction! Today we massively performed PCR of the 3 new strains to amplify PxpsR using a temperature gradient (form 45 to 69). All but one seemed to amplify in almost all the temperatures! Thats the one!!
July 3
Now that we know that PxpsR amplifies, we continue following our decision tree and try with phcA, we only used the strains where PxpsR amplified. We used the temperature gradient previously named for phcA. Fortunately for us all the strains were positive.
July 5
Today we tried to amplify both PxpsR and phcA with Pfu in order to clone in the backbone (pBS1C3), as both genes had amplified previously at 64°C we choose that temperature as the Ta, allthough it didn’t work for PxpsR.
We also tried to amplify phcR and phcS using a gradient temperature from 57 to 67 with the strain #75 that always amplified better than the others. Surprisingly both genes amplified just fine at all temperatures, but there was double band, even in the highest temperatures, so we will have to cut band after amplifying with Pfu.
Finally we digested (see protocols) phcA with EcoRI and SpeI, and the backbone with EcoRI, SpeI and DpnI, and we also ligated (see protocols) with T4 ligase through the night.
July 6
Today we amplified PxpsR, phcR and phcS with Pfu, we used the lowest  temperature for PxpsR (45°C), but once again PxpsR didn’t amplify as we expected (more than 1 band).
We also transformed by electroporation (see protocols) the ligation of phcA into the backbone, we hope to see colonies in the morning.
July 7
Nothing grew into the plate! We are not sure if the restriction enzymes aren’t working well or the T4 ligase is failing… We tested the restriction enzymes using as target the salicylic acid biobrick, we performed different combinations of enzymes (PstI-EcoRI, EcoRI-SpeI, XbaI-PstI, SpeI-XbaI) that excises a fragment of 1500bp. Conclusion: Everything  seems to be fine with the enzymes, it must be the T4 ligase.
We also repeated the PCR for PxpsR this time using as Ta 50°C, it amplified!
July 10
We are back in the game, we have a different ligase and we are making new electrocompetent cells!
July 12
Today we are going to transform phcA+BB, phcR+BB, phcR+BB and PxpsR+BB in our just prepared competent cells. Tomorrow we will make passes of the colonies.
July 13
There are not colonies in the plates…maybe the cells aren’t that competent, we will try again with other cells…
July 16
We triple digested the BB sng XbaI, SpeI and DpnI and double digested phcA with XbaI and SpeI.... We ligated overnight!!... We started phosphatating the BB using alkaline phospatase to avoid the religation of the BB...Fingers crossed!
July 17
Today we transformed using new competent cells. Hopefully the new cells and the phospatase will help!
Just in case we repeated the PCR for phcA and PxpsR using Pfu.
July 18
We have colonies!!! Now we  need to make passes and confirm!
July 23
Today we run a pcr confirmation for phcA and xpsR... Out of 30 reactions only four bands!
We think maybe DpnI is cutting the BB instead of just mCherry. We digested the BB using only XbaI and SpeI, and we will purify the band. We ligated again using the purified BB band and phcA.
July 27
We are running low on pcR products, so we reamplified all our genes using PFU today. Also just on colony out of our  latest attempt with the double digested BB and phcA... We just need to get one right, right?
Lated that same day... The confirmation pcr worked! Now we have to confirm using restictive enzymes, to verify directionality. All genes amplified! ...except for xpsr!
July 28
Pcr to amplify pcr once again. All these digestions took almost the BB miniprep we had, and we are running low on supplies, so today we did miniprep of the BB again to stock up. We also did miniprep of our phcA+BB colony to confirm.
First day of class is just around the corner, so today we did as much as we could. We digested the backbone, phcA+BB with PstI and XbaI-SpeI, and our pcr products!
Confirmation digestion for phcA didn' work! At least we have all new BB miniprep!
Guess phcA went the other way and ligated backwards!
We have xpsR colonies!!
July 30
Back to school and still no results!
Almost of our latest ligations worked! See those two bands next to the DNA Ladder, that´s supposed to be just one! We confirmted our suspition, DpnI cuts the BB in multiple fragemts. Guess we'll have to purify band every time we run out of BB!
We transformed new cells with phcA+BB
August 1
We haven't forgotten about phcR and phcS. Today we made passes of phcR colonies to confirm later with pcr. We digested and ligated xpsR once again.
August 2
We transformed xpsR once again and made passes of phcA colonies.  Now that we are back to school our rhythm has really slowed down.
August 3
Confirmation pcr of phcR colonies.
Finally, there aren't many but they are definitely there.
August 4
More confirmation Pcrs of phcA, phcS and xpsR! Today we resuspended more primers, all these Pcrs took all we had!
August 5
We run a gel with yesterdays pcr.... Nothing! Except for on little band next to the DNA Ladder!
We made passes of phcA+BB!
August 6
Miniprep of the little colony that amplified in the last Pcr. We digested it to confirm.  Once again we are digesting phcS and phcR and ligating overnight.
Pcr to confirm phcA passes!
Later that same day.... Nothing on the confirmation digestion.
August 7
We diced to try confirming more xpsR colonies.
Looks like we have 4 bands!! Fingers crossed!
August 10
Today Laura woke up really early and started working at 5:00 am. You know what they say, the early bird catches the worm! She digested phcR, phcS using XbaI and SpeI. We ligated that same day and tranformed in the night.
We confirmed a colony of xpsR both by Pcr and digestion!!! We have our first gene of the Ralstonia System!!!!!!!
August 13=
Once again we did Pcrs of phcS and phcR. No bands on the gel!
August 15
Today we are screening colonies of phcR and phcS once again.
Primer Dimers..... Again!
To confirm the basal actvity of xpsR promoter we are trying to ligate a fluorescence gene (GYFP) upstream of the promoter. Today we made our first attemp.
August 17
We run out of Pcr products, again, Today we are amplifying phcA, phcR and phcS with PFU.
We also run a gel to confirm phcS colonies.
It seems we have to clones, although there are some bands with a lower molecular weight than we expected.
We transformed competent cells with the xpsR+eYFP ligation.
August 18
Today we did a PCR to the xpsR+eYFP transformation.
Today we strated using a new PCR kit, which is so much easier!
August 19
We ran a gel for the PCRs of phcA, phcR and phcS. The new kit is great! So intese bands on the gel.
August 20
Today we ligated phcA, phcR and phcS using NEB T4 Ligase. We transformed and hoped for the best.
August 21
We a lot of colonies from our last tranformation, so we made a lot of PCRs to confirm. Unfortunetly after 94 reactions, we found nothing.
August 23
Since nothing amplified we started using new primers, maybe that's the problem.
We digested the miniprep of xpsR+GYFP using EcoRI and PstI and we were able to confirm that xpsR ligated correctly! Now we just need to see our bacteria under a fluorescence microscope and hope they shine!
September 3=
We have had a lot problems to use the fluorescence miccroscope, guess we'll have to wait a little more! For now we are preserving our bacteria with the xpsR and xpsR+GYFP vectors.
We are re-amplifying old PCR products using new primers!
September 4
Our last PCRs worked, now we are digesting them using EcoRI and SpeI. We decided to try this new set of enzymes to avoid the vectors not ligating directioanlly. We ligated overnight and tranformed competent cells.
September 7
PCR Marathon!!! We have a lot of colonies from our last ligations, so we are screening as much as we can hoping to find a winner!
Almost nothing! Loos like a couple of colonies did amplify.
September 8
Today we could finally use the fluorescence microscope.... It worked, our colonies were fluorescing and just some of them which tells us the promoter is inducible! If we just had phcA to test the response to an enhancer!
September 10
We need to transfer xpsR+GYFP to a chloramphenicol backbone since it was transformed in an ampicillin backbone. Today we began by digesting the miniprep.
Simultaneously we are doing new ligation for phcA, phcR and phcS.
September 12
We digested the backbone were we are going to put xpsr+eYFP. We are still using EcoRI and SpeI to digest.
Wow! The backbone digestion was very effective!
September 13
We continued to do massive PCR confirmation out of all the colonies we got form out last transformations.
September 17
We are still trying to confirm phcR and phcA from the colonies that grew during our last transformation.  But still have nothing!
September 18
Our last gel amplified a fragment of 900 bp! So we did a little experiment to test the purity of the primers...And they were contaminated! We'll have to  start using specific primers for each gene to confirm and repeat all the PCRs using these primers.
September 19
We repeated all confirmation PCRs using specific primers, digested the backbone to ligate xpsR+eYFP. Ligations of phcR, phcS, phcA and PxpsR+eYFP were left overnight.  We evaluate different broad host range plasmids including pHM1, pDD62, pBAV178 and pML123 wich could replicate in Pseudomonas/Ralstonia, finding information and sequences about plasmids it is so difficult, so we decided to use pML123, the best documented plasmid we had.
September 20
We obtained colonies of PxpsR+GYFP!!! Colonies 2, 7 and 10 were confirmed by PCR... now we have to continue screening for phcA, phcR and phcS. We start working into the in vivo PxpsR promoter (Experiment). We growth the mute Ralstonia solanascearum and make miniprep of the plasmid ML123.
September 21
Today we cut pML123 with XbaI and ligated with the xpsR+eYPF cut with XbaI and SpeI, we did it this way hopping some inserts ligate in the opposite direction to the strong promoter npt. Still not good news about the other genes...
September 22
We decide to use an alternative strategy to confirm phcA, phcR and phcS clones... we digested severely minipreps looking for excise the insert of the right size... no chance with the digestions today.
September 23
We are running off time and options, big massive PCR today, we screen nearly to 100 colonies per gene, tomorrow we'll ckeck if our chance is changing.
September 24
First good news!!!! and only 3 days left to the deadline!!! we have bands!!!
Mathematical Models
Up to June 6
Up to June 6 we organized the variables and parameters of our design, formulated the differential equations for each of the molecules of the model and set out to look for parameter values in literature. This link is the depurated version summary of the model that should be included in the Modeling section of the wiki.
July 3
Today’s subject was the parameters to be used at differential equations that had been previously developed. In order to do this, a set of parameters, to be searched from different sources, had been assigned to each member of the team. According to the results obtained it was established to categorize each group in a range as it is shown below:
α group: Since no value was found it would be supposed at first that all values are the same.
γ group: For this parameters, Gabriel proposed to bear in mind the lifetime of a E. coli. As a backup, rates of normal destruction in proteins (in general) and salicylic acid could be looked for.
Kinetic constants of reaction: We will focus on ChiA and look for any number of cellular kinetic trying to find a maximum.
βx group: Search concentrations of the most produced protein in E. coli, RubisCo and the least produced to establish a range.
Parameters of export and import: To reference export and import rates of any protein and then try to establish a range.
July 10
The following document has all the information we could find about parameter approximate values. It is in spanish, but it should be translated and included in the Modeling section of the wiki. File .
Our first matlab code was written today too. It is a two file code:
Code for ecuaDif.m
%Codigo que contiene todas las ecuaciones diferenciales de los procesos %dados en los 3 plasmidos function y=ecuaDif(t,v) % Funcion que devuelve un vector y con todas las ecuaciones, tiene como  parametro un vector con los parametros y un vector v  %las variables.                        QQ=10;  %------ Variables%------                         Ao=v(1); %Cocentration of chitinase outside the cell Ai=v(2); %Concentration of chitinase inside the cell P=v(3);  %Concentration of chitiporin C=v(4);  %Concentratio of chitin binding protein (CBP) CS=v(5); %Concentration the complex CBP-s R=v(6);  %Concentration of LuxR Ii=v(7); %Concentration of LuxI inside the cell Io=v(8); %Concentration of LuI outsied the cell IR=v(9); %Concentration of the complex LuxI-LuxR  CI=v(10);%Concentration of the protein CI  HA=v(11);%Concentration of HipA7 HB=v(12);%Concnetratio of HipB AS=v(13);%Concentration of salicylic acid Q=v(14); %Concentratio of quitin monomers  %----Parameters----%                          %All the parameter came in a vector given by other function.    alfA=0.4;  %Basal concentration of Chitinase inside the cell (micromolar) alfP=0.4;  %Basal concnetration of chitoporin  alfC=0.4 ;  %Basal concentration of the CBP alfR=0.4;  %Basal concentration of LuxR alfI=0.4;  %Basal concentration of LuxI  alfCI=0.4;  %Basal concentration of CI  alfHA=0.4;  %Basal concnetration of HipA7 alfHB=0.4;  %Basal concnetration of HipB  alfAS=0.4;  %Basal concnetration of Salycilic acid     gammaA=0.01;  %Degradation of Chitinase inside the cell gammaP=0.01;  %Degradation of chitoporin  gammaC=0.01;  %Degradation concentration of the CBP gammaR=0.01;  %Degradation of LuxR gammaI=0.01;  %Degradation of LuxI  gammaCI=0.01;  %Degradation of CI  gammaHA=0.01;  %Degradation of HipA7 gammaHB=0.01;  %Degradation of HipB  gammaAS=0.01;  %Dergradation of Salycilic acid gammaCS=0.01;  %Degradation of the complex CS     mCS=5;  %Kinetic constant for the formation of the complex CS  mCSQ=5;  %Kinetic constant of the reaction of the complex CS with the chitin  mAQQ=5;  %Kinetic constant for the reaction of the chitinase and th chitin mIR=5;  %Kinetic constant for the formation of the complex LuxILuxR mI=5;  %Constant that represent the union of the complex LuxILuxR with the promoter mHAHB=5;  %Kinetic constant for the inhibition of HipA7  betaP=10;  %Max production of the chitoporin  betaA=10;  %Max production of chitinase betaI=10;  %Max production of LuxI betaCI=10;  %Max production of CI  betaHB=10;  %Max peoduction of HipB betaHA=10;  %Max production of HipA7  betaAS=10;  %Max production of Salicylic acid   kS=1;  %Constant k of the hill ecuation for the promoter promoted by S kIR=0.5;  %Constant k of the hill equiation for the promorer prmoted by the complex luxIluxR kCI=0.1;  %Cosntant k of the hill equation for the promoter promoted by CI    hS=1;  %Hill constant for the promoters promoted by S  hIR=3;  %Hill constant for the promoter promoted by the complex IR hCI=2.3;  %Hill constant fot the promoter CI    eA=0.5;  %Exportation factor of the chitinase  jQ=0.8;  %Importation factor of the chitin monomers  deltaA=0.2; %Difusion factor of the chinitanse outside the cell  eI=0.5;  %Exportation factor of LuxI jI=0.8;  %Importation factor of LuxI deltaI=0.2;%Difusion of LuxI outside the cell  Stotal= 1;  %Total concentration of the sensor in the cell   numcel=100;  %number of cells  %---Equations---%                                  S=Stotal-CS;                                                                     dC=alfC- gammaC*C - mCS*C*S; %Change of CBP   dCS=mCS*C*S- mCSQ*CS*Q-gammaCS*CS;  %Change of the complex CS   dP=alfP - gammaP*P + (betaP*(S^hS))/(kS+(S^hS));%Change of chitoporin  dAi=alfA- gammaA*Ai+ (betaA*(S^hS))/(kS+(S^hS))- eA*Ai; %Change of chitinase inside the cell   dAo= eA*Ai-deltaA*Ao- mAQQ*Ao*QQ; %Change of chitinase outside the cell  dQ= jQ*P*(mAQQ*QQ*Ao)-mCSQ*CS*Q; %Change of chitin monomer inside the cell  dIi= alfI+ (betaI*(S^hS))/(kS+(S^hS)) -gammaI*Ii +jI*Io- eI*Ii- mIR*Ii*R; %Change of LuxI inside the cell  dIo= numcel*(eI*Ii-jI*Io)-deltaI*Io; %Change of LuxI outside the cell  dIR= mIR*Ii*R - mI*IR;  %Change of the complex LuxI luxR  dR= alfR-gammaR*R -mIR*Ii*R; %Change of LuxR  dCI= alfCI -gammaCI*CI+ (betaCI*(CI^hCI))/(kCI+(CI^hCI))+(betaCI*(IR^hIR))/(kIR+(IR^hIR));%Change of CI  dHB=alfHB-gammaHB*HB+(betaHB*(CI^hCI))/(kCI+(CI^hCI))+(betaHB*(IR^hIR))/(kIR+(IR^hIR))-mHAHB*HA^2*HB^2; %Chanche of HipB   dHA=alfHA-gammaHA*HA+ (betaHA*(CI^hCI))/(kCI+(CI^hCI))-mHAHB*HA^2*HB^2; %Change of HipA7  dAS=alfAS-gammaAS*AS +(betaAS*(CI^hCI))/(kCI+(CI^hCI));  %Change of Salicylic acid    y1(1)=dC; y1(2)=dCS; y1(3)=dP; y1(4)=dAi; y1(5)=dAo; y1(6)=dQ; y1(7)=dIi; y1(8)=dIo; y1(9)=dIR; y1(10)=dR; y1(11)=dCI; y1(12)=dHB; y1(13)=dHA; y1(14)=dAS; y=y1';   end
MATLAB script for solvEqua.m
%File that solves the differential equations and graphs them  h=1000; %Tiempo maximo  Chi=zeros(1,h); %Vector que almacena la concentracion de quitina  t=0:0.1:h; %Vector tiempo  conInd=[0.4,0.4 0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4];   for j=1:h-1   if j<900       Chi(1,j)=10;   else       Chi(1,j)=0;   end end [T Y] =ode45(@ecuaDif,[0 h],conInd);  C=Y(:,1);  for i = 1:length(C)    if C(i) < 0        C(i) = 0;    end end CS=Y(:,2); P=Y(:,3); Ai=Y(:,4); Ao=Y(:,5); Q=Y(:,6); Ii=Y(:,7); Io=Y(:,8); IR=Y(:,9); R=Y(:,10); CI=Y(:,11); HB=Y(:,12); HA=Y(:,13); AS=Y(:,14);  subplot(14,1,1); plot (T,C,T,CS,T,P,T,Ai,T,Ao,T,Q) subplot(4,1,2); plot (T,R,T,Ii,T,Io,T,IR) subplot(4,1,3); plot (T,CI,T,HA,T,HB) subplot(4,1,4); plot (T,AS)
July 12
We had little luck graphing the code we wrote before. It seems we need to many specifications and restrictions to just use the ode function so it seemed we had to write the differential equation solver method ourselves.
We modified the ecuaDif.m file so that it may receive a chitin pulse:
if (t<(0.1) || t>0.3)         QQ=0; else     QQ=1; end
We also came up with a 4th order Runge-Kutta method for the solution of differential equations:
Code for solvEquaRK.m
%File that solves the differential equations and graphs them   h=0.44; %Tiempo maximo   t=0:0.1:h; %Vector tiempo   conInd=[0.02,0.001,0.002,0.002,0.02,0.002,0.001,0,0.001,0.04,0.003,0.04,0.04,0.02];     m=h/100; %Longitud de paso [s] l=(0:m:h)'; %Vector de longitudes x=zeros(length(l),length(conInd)); %Matriz de variables, en las columnas varia  %la variable y en las filas varia la longitud QQ=zeros(1,length(l));    x(1,:)=conInd;                                                for k=1:length(l)-1    xk=x(k,:); %Captura de la ultima posicion de la matirz, es decir, los  %valores actuales de las variables        k1=ecuaDif(l(k),xk); %Primera pendiente del metodo de RK4    k2=ecuaDif(l(k)+m/2,xk+(m/2*k1)'); %Segunda pendiente del metodo de RK4    k3=ecuaDif(l(k)+m/2,xk+(m/2*k2)'); %Tercera pendiente del metodo de RK4    k4=ecuaDif(l(k)+m,xk+(m*k3)'); %Cuarta pendiente del metodo de RK4        xk1=xk+m/6*(k1+2*k2+2*k3+k4)'; %Calculo de nuevos valores para las  %variables                      x(k+1,:)=xk1; %Actualizacion del nuevo vector de variables en la matriz                     end  for j=1:length(l)     if (l(j)<(0.1) || l(j)>(0.3))             QQ(j)=0;         else             QQ(j)=1;              end   end  C=x(:,1); CS=x(:,2); P=x(:,3); Ai=x(:,4); Ao=x(:,5); Q=x(:,6); Ii=x(:,7); Io=x(:,8); IR=x(:,9); R=x(:,10); CI=x(:,11); HB=x(:,12); HA=x(:,13); AS=x(:,14);     figure(1)  plot (l,QQ,l,C,l,CS,l,P,l,Ai,l,Ao,l,Q)  legend('QQ','C','CS','P','Ai','Ao','Q')   figure(2)   plot (l,QQ,l,R,l,Ii,l,Io,l,IR)  legend('QQ','R','Ii','Io','IR')    figure(3)  plot (l,QQ,l,CI,l,HA,l,HB)  legend('QQ','CI','HA','HB')   figure(4)  plot (l,QQ,l,AS)  legend('QQ','Acido Salicilico')
We found three more problems with this code:
As with the ode function, graph solutions work until 0.44 time units.
Some graphs show negative numbers, which should not be possible. It seems, however, to be solved using appropriate parameters.
First results show that CI and salicylic acid concentration do not depend heavily in chitin concentration. We have to check it there is a mistake with our differential equations or whether we should change aspects of the design.
July 16
The following is the July 10 entry all translated and ready. Again, it should be posted in the modelling section of the site.
During this meeting there were determined the ranges in which each group of parameters can be found. For this, there were used different sources, obtaining the results shown below: iGEM parameters determination
1. Basal levels of the protein: For this value, it will be assumed the same for all proteins. Range: 0-0.8µM Value taken from: http://bionumbers.hms.harvard.edu/bionumber.aspx?&id=104520&ver=10&trm=protein Where there is an average level, the lowest levels on that range were taken, since the concentration is basal. 2. Protein degradation: For all the proteins is assumed in the same way.
Range: 0-0.02sec-1
Degradation constant for RubisCo, maybe is needed to include in the constant the cell division value.
Taken from: http://www.tesisenred.net/bitstream/handle/10803/9518/marin.pdf?sequence=1
3. Reaction constant (association): Each of the kinetic constants are independent from each other.
Range: 0.1-10µM-1sec-1 Range was taken from: http://www.biokin.com/dynafit/scripting/html/node23.html#SECTION00431000000000000000
4. Maximum cell concentration (β Hill): Having in mind that RubisCo concentration is 20 times more than a normal protein, and the average range of protein concentration in the cell, we have: Range: 1.4-20µM  Taken from: http://bionumbers.hms.harvard.edu/bionumber.aspx?&id=107431&ver=1&trm=rubisco http://bionumbers.hms.harvard.edu/bionumber.aspx?&id=104520&ver=10&trm=protein
5. Hill coefficient k: As it is the concentration to the maximum expression, the minimum value was taken from a paper where there had been calculated the parameters for CI and the maximum value was half of the maximum concentration calculated previously.
Range: 0.05-10µM  Taken from:  Gene Regulation at the Single-Cell Level. Nitzan Rosenfeld,Jonathan W. Young, Uri Alon, Peter S. Swain, Michael B. Elowitz. Science 307, 1962 (2005);
6. Hill coefficient (n): Having in mind the CI paper and approximately the amount of molecules involves in the gen activation, it was established that:
Range of genes activated by S: 0-2 Range of genes activated by LuxI-LuxR: 0-4 Range of genes activated by CI: 1-3
Finally, the last modifications to the program code were made so it could be possible to graph, in order to continue with a detailed scan of the parameters. The next meeting will take place on Thursday, July 12th at 10:00 am. For this meeting, tasks are to have already introduced the chitin pulse to the program’s code and to have a matrix with all the possible combination of parameters.
July 19
Today’s meeting was focused on solving the problem of the code on Matlab. Previously, the functions were not compared in the expected way so the following steps were taken:
• A separated new file was created, saving the initial conditions of the system, so it would take into account that the values must start from a stable state.
• It was verified that the order in which the variables were introduced on all the documents was correct.
• We continued verifying the values of the parameters.
Finally, several advances were achieves:
• The program gets done graphs with large times.
• Negative values are not obtained anymore.
• The response of most proteins is the desired.
At the moment, two errors are still being detected, which are necessary to verify:
• Salicylic acid does not depend on the impulse of the chitin.
• Certain values on the parameters generate responses with imaginary values.
It was left as homework to each member of the group to find ways to solve the current problems and then make a parameters scan.
August 8
Even if our genetic circuit model works, we decided we should expand our simulations to a population scale. For this, and with the help of Juan Cordovez, biomedical engineering professor, we came up with the idea of generating an ecological computational model of the interactions between plant-pathogen and our engineered bacteri.
General Objective: to generate a computational model that simulate the most important ecological relationships between the engineered bacteria and the pathogenic microorganisms in the corresponsing habitat.
Specific Objectives:
1. To limit the ecological multifactorial problem in such a way that a simple mathematical model, capable of answering relevant questions may be built.
2. To find the populational relationships between our engineered microorganisms, pathogens, and the plant host that optimize our biological control. The corresponding parameters will have to be taken from literature or determined experimentally.
3. Generate interesting hypotheses for latter experimental confirmation.
August 15
Today we had  a meeting with all the group of iGEM Colombia. There each group presented their work and how it is going to end before de deadline.  There are new things to consider in the mathematical models due to some little changes in the order of the genes and some aspects that were not taken into account.
1. There is a change in the detection model of chitin, now is going to have two copies of chitinase and chitoporine, and will be regulated by 2 different promoters. First they are going to be downstream the genes that code for the sensor and de CBP (constitutive promoters) and to create a positive feedback when the chitin is present they are going to have an inducible promoter by chitin.
This modification makes us change the parameters of the basal production of the proteins involved but the equations remain the same.
2. We haven’t considered the change in metabolism of the bacteria when the toxin is more concentrated than the antitoxin. This event makes the cell have a lower metabolism and all the parameters change. We need to read the literature to find the levels at which the metabolism slows down and how much it does it.
3. The promoter for LuxR was not established; it depended of the modeling results. Even though we don’t have a final result we have seen that the system behaves better when LuxR is inducible and not when is downstream de constitutive promoter. The cut and paste group is taking this into account.
4. Before screening the parameters we need to know the concentration at which the detection system of vibrio is turned on and the concentration of chitin in a leaf of coffee. The group one is going to do this experiment and after that the screening will be made. Meanwhile this Saturday (08-18-2012)  we will start with the stochastic model.
August 26
We had a first meeting with Juan Cordovez about the ecological model. We explained to him the project thoroughly. We are to begin looking for relevant modelling parameters until a next meeting.
Cut and Paste
Until June 20th we did biobricks transformations of parts from the registry for confirmation.
This procedement was repeated a lot of times without success because we weren't using a proper kit for DNA purification from agarose geles. It started to being succesful after using the Wizard® Genomic DNA Purification Kit.
2.We obtained colonies, but we realized they were false positives, so we designed primers and tried cloning parts from PCR.
3.PCR Standardization. We did digestions with enzymes of the PCR products since September 5th getting good results.
4.The results with this protocol were satisfactory, so we was ligating the biobricks by couples and repeting until get the desired construction.
Aliivibrio/Streptomyces
The first issue faced by the team was to clarify which genes were already cloned in the previous year. As the parts were missing or incongruent, we decided to start from the beginning.
Due to many complications for getting the strain, we decided to postpone the cloning of the genes from S. coelicolor and work towards the characterization of the genes in A. fischeri.
ChiP and pChitin
Chitoporin gene, chiP, was the first amplified gene once the ES114 strain arrived. The pChitin gene was amplified along with chiP using a primer designed for Colombia Team 2011 and the reverse from chiP. They both were properly purified and digested. The transformation resulted with several candidate colonies but we currently don't have postitive confirmation.
Gradient amplification of pChitin (Lanes 2 to 7)
chiA and CBP:
We achieved amplification of chiA and CBP with success in a broad range of temperatures with the OneTaq 2X MasterMix (New England Biolabs)PCR kit.
Amplification of CBP (Lanes 2 to 10) and chiA (Lanes 11 to 19
Due to a mistake in the sequence of the part (there are two chitin binding proteins, one in each chromosome of the bacterium), the digestions of CBP were giving "wrong" band sizes.
Restrictions of chiA, CBP, chiS, backbone and chiP and pChitin
Chitinase
To determine which chitinase is going to be used, we screened chitinase from 3 different species since the chitinase of different Vibrio spp. was not suitable or the strain was found (Alivibrio fischeri ES114, Alivibrio fischeri M11):
- Arabidopsis thaliana
- Colletotrichum spp.
- Trichoderma sbpp.
For each one we obtained accesion numbers:
- Arabidopsis thaliana: AY099810.2
- Colletotrichum spp. : GW342409.1
- Trichoderma spp. : BM077089.2
Each one was tested using codon usage for bacteria (Translation table 11) to determine if the chitinase could be used. All of the sequences were suitable for use.
We selected Colletotrichum spp. and Trichoderma spp. sequences to design primers.
Primers
- Colletotrichum spp. :
- Trichoderma spp. :
Retrieved from " http://2012.igem.org/Team:Colombia/Notebook/Journal "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
