Team:Pittsburgh/Protocols
Home
People
Team Collaborations InterLab Attributions
Medals
Project
Overview Experiments Results Proof of Concept Notebook Protocols
Parts
Parts Basic Parts Description
Safety
Human Practices
Human Practices Silver Gold Engagement Model
Home
People
Team
Collaborations
InterLab
Attributions
Medals
Project
Overview
Experiments
Results
Proof of Concept
Notebook
Protocols
Parts
Parts
Basic Part
Description
Safety
Human Practices
Human Practices
Silver
Gold
Engagement
Model
Contact Us
pitt.igem.2016@gmail.com
Thank you to our sponsors!
The protocols we reference in our Notebook
Contents
Reporter
Pouring Plates
Competent Cells
Transformations
Liquid Cultures
Glycerol Stocks
Minipreps
Sequencing
Restriction Digests
Agarose Gel Electrophoresis
Gel Extraction and DNA Cleanup
Dephosphorylation
Ligation
Polymerase Chain Reaction
Colony PCR
Site-Directed Mutagenesis
Cell-Free Expression
Annealing
Phosphorylation
10X Annealing Buffer
Annealing in Buffer
Annealing in Water
DNAzyme
10X TBE Buffer
Denaturing Polyacrylamide Gel Electrophoresis
Native Polyacrylamide Gel Electrophoresis
Buffer B
Cleavage Reaction
Collaborations
InterLab Study
1X PBS
UGA Archaeal InterLab
W&M Genetic Circuits Toolbox
50X TAE Buffer
Reporter
Pouring Plates
In 1L of DI water, add 25 grams of LB broth and 15 grams of agar. Adjust accordingly if making a different volume
Swirl gently to dissolve some of the LB broth
Autoclave for 20 minutes
Remove from the autoclave and let cool
When cool, add antibiotic to working concentration
Ampicillin - 100 µg/mL
Chloramphenicol -  25 µg/mL
Kanamycin - 50 µg/mL
Pour plates
Let them sit for ~30 min to solidify
Store in cold room in original plate sleeve. Make sure agar side is up
Back to Top
Competent Cells
We follow the protocol for the preparation of competent E. coli cells using calcium chloride from Ivaan.com . We use the Competent Cell Test Kit from iGEM to determine our cells' competency.
Back to Top
Transformations
We follow the transformation protocol provided by iGEM. Any deviations from this protocol are noted in the Lab Notebook.
Back to Top
Liquid Cultures
In a 15 mL centrifuge tube, add 5 mL of LB Broth
Add necessary antibiotic to working concentration
Using a pipette tip, lift colony of interest
Place tip in centrifuge tube
Tape on the lid. Be sure not to twist tightly
Incubate at 37°C in the shaker overnight
Back to Top
Glycerol Stocks
Combine 1 mL of 40% glycerol with 1 mL fresh bacterial culture in a cryogenic vial
Gently vortex or pipette to mix
Store culture in -80°C
Back to Top
Minipreps
We follow the miniprep protocol provided by Thermo Scientific for their GeneJET Plasmid Miniprep Kit. We elute the DNA in 25 µL of Elution Buffer, not 50 µL as stated in the protocol, to achieve a higher concentration.
Back to Top
Sequencing
We use Genewiz for sequencing and follow their sample submission guidelines .
Back to Top
Restriction Digests
We use the protocol provided by Thermo Scientific for the fast digestion of DNA as a guide.
For a 20 µL reaction, add:
2 µL enzyme (1 µL each if doing a double digest)
2 µL 10x buffer
1 µg plasmid
Nuclease-free water to volume
Incubate at 37°C for 30 min
Incubate at 65°C for 20 min to deactivate enzymes
Back to Top
Agarose Gel Electrophoresis
All gels are 1% agarose unless otherwise stated. We use the 1 kb DNA ladder from NEB.
Dissolve 0.5 g agarose in 50 mL 1X TAE buffer, using the microwave to heat
When cool to the touch, add 5 µL ethidium bromide and pour gel
Run in 1X TAE buffer for 45 min
Back to Top
Gel Extraction and DNA Cleanup
We follow the protocols provided by Thermo Scientific for their GeneJET  Gel Extraction and DNA Cleanup Micro Kit.
Back to Top
Dephosphorylation
Combine
10 µL DNA
1 µL buffer
0.5 µL rSAP (recombinant shrimp alkaline phosphatase)
Incubate at 37°C for 30 min
Incubate at 65°C for 5 min to deactivate enzymes
Transform 5 uL of each reaction
Back to Top
Ligation
Combine
1 µL plasmid
amount of insert for desired ratio
2 µL buffer
Nuclease-free water to 20 µL
1 µL T4 DNA ligase
Incubate at room temperature for 10 min
Incubate at 65°C for 10 min to deactivate enzymes
Transform 5 µL of each reaction
Back to Top
Polymerase Chain Reaction
We follow the PCR protocol provided by NEB for Phusion High-Fidelity DNA Polymerase.
Back to Top
Colony PCR
We use the Colony PCR Protocol for plated colonies from the 2013 iGEM page.
Back to Top
Site-Directed Mutagenesis
Phosphorylate Oligos
T4 PNK 1 µl (10 units)
10X T4 Ligase buffer 5 µl
DNA (20 mer) 1-2 µg
Nuclease-free water to 50 µl
Incubation 37°C for 30 minutes
Mix in a PCR tube
0.5 µl forward primer, 2.5 pmoles/µl
0.5 µl reverse primer, 2.5 pmoles/µl
0.25 µl 40 mM dNTP mix, (10 mM each)
1.25 µl Phusion Buffer
1 µl Template DNA, 2 ng/µl
0.25 ul Phusion
8.75 µl sterile H2O
12.5 µl total
Incubate according to the following PCR Program:
5 min at 95°C
Repeat 18 times:
50 s at 95°C
50 s at 60°C
1 min + 1 min/1 kb template at 68 °C
7 min at 68 °C
Gel Check: Run 2.5 µl of the reaction on a gel. There should be a band corresponding to your product. Even if you don’t see a reaction product, you can still try the rest of the protocol, but you may not get any colonies.
DpnI Digest: Add 0.25 µl of DpnI (20 U/µl, New England Biolabs) to the reaction. Incubate at 37◦C for 1 hr.
Transform
Back to Top
Annealing
Phosphorylation
Resuspend oligos to a stock concentration of 100µM with nuclease-free water.
To a PCR tube, add
2 µl of the proper Top or Bottom strand oligo
2 µl of 10X T4 DNA Ligase Buffer
1 µl of T4 Polynucleotide Kinase
15 µl of water
Mix well and spin down. Oligo final concentration is 10 uM.
Incubate the PCR tube in the thermocycler with the program at 37°C for 60 minutes, then at 65°C for 20 minutes then end.
Back to Top
10X Annealing Buffer
For 4 mL, add
400 µL 1M Tris pH 8
80 µL 0.5M EDTA pH 8
800 µL 2.5M NaCl
2720 µL water
Back to Top
Annealing in Buffer
Combine
5 µL forward oligo
5 µL reverse oligo
5 µL 10X annealing buffer
35 µL water
Mix well and spin down. The final oligo concentration is 1µM.
Back to Top
Annealing in Water
Combine
1 µg oligos
5 µL 10X T4 DNA Ligase Buffer
Nuclease-free water to 50 µL
Incubate at 85°C for 10 min
Cool to 20°C over 30 min
Store annealed oligos at -20°C
Any deviations from this protocol are noted in the Lab Notebook.
Back to Top
Cell-Free Expression
We follow the protocol provided by NEB for the protein synthesis reaction using PURExpress (E6800) .
Back to Top
DNAzyme
10X TBE Buffer
For 1 liter, combine
108 g Tris base
55 g boric acid
40 mL 0.5 M EDTA (pH = 8.0)
water to 1 liter
Back to Top
Denaturing Polyacrylamide Gel Electrophoresis
We use Novex precast gels in TBE buffer for our dPAGE assays. We follow the guidelines provided with the gel loading buffer we use from ThermoFisher. We use SYBR Gold from ThermoFisher to stain.
Back to Top
Native Polyacrylamide Gel Electrophoresis
For a 15% gel, combine
4 mL 40% acrylamide
6 mL 1X TBE buffer
100 µL 10% APS
15 uL TEMED
in a 15-mL Falcon tube
Invert tube to mix
Pour between plates and insert comb
Gel solidifies in about 20 minutes
Load gel and run at 100V for one hour
Stain for 20 minutes with SYBR Gold from ThermoFisher
Thank you to Nick from the Deiters lab for his help!
Back to Top
Buffer B
Combine
25 mM NaCl
50 mM MOPS
NaOH to adjust pH to 7.5
water to volume
Back to Top
Cleavage Reaction
Combine
0.7 μM annealed duplex
10 μM ErCl3 dissolved in water
Buffer B to volume
Incubate at room temperature for 20 minutes
Quench with equal volume of gel loading buffer II from ThermoFisher
Back to Top
Collaborations
InterLab Study
We followed the updated Plate Reader Protocol from iGEM.
Back to Top
1X PBS
For 1 liter, dissolve the following in 800 mL of water
8 g NaCl
0.2 g KCl
1.44 g Na2HPO4
0.24 g KH2PO4
Adjust pH to 7.4 using NaOH
Add water to volume
Back to Top
University of Georgia: Archaeal InterLab
We were given five samples from UGA. 50 μL of sample was added to a 96-well plate for each triplicate of each sample. Fluorescence was read at 590 nm excitation, 645 nm emission, and 630 nm cutoff.
Back to Top
William & Mary: Genetic Circuits Toolbox
Back to Top
50X TAE Buffer
Materials
242 grams Tris free base
18.61 grams Disodium EDTA
57.1 mL glacial acetic acid
DI water
Procedure
Add Tris free base and EDTA to ~700 mL of DI water
Stir until dissolved
Autoclave for 20 minutes
Add acetic acid
Adjust the volume with DI water until the volume is 1 L
Back to Top
