"
Page
Discussion
View source
History
teams
Log in
 
Team:CU-Boulder/Project/Kit/RestrictionEnzymes
From 2013.igem.org
Home
Team
Team
Members
Collaboration
Official Profile
The University of Colorado - Boulder
Project
Project Description
Restriction Enzymes
Protein Purification
DNA Purification
Safety
Happy 4th!
Attribution
Parts
Notebook
Protocols
Experiments
Outreach
Sponsors
iGEM
\
Abstract
The main focus of our project was to create the constructs and purification methods necessary to produce and isolate the restriction enzyme EcoRI. EcoRI is commonly used for research and is one of the primary restriction endonucleases used in the Biobrick standard. Synthesizing a plasmid containing the gene for EcoRI and producing a simple purification method would provide a cheaper option for obtaining the enzyme.
Background Information
Bacteria lack an immune system so rely on restriction-modification (R-M) systems for defense against foreign viruses. The first enzyme involved in the system is the restriction endonuclease (REase). It is specific to a single DNA sequence that is typically four to eight nucleotides in length and is usually palindromic. When the REase recognizes this sequence, it will cleave both strands of the DNA at the backbone, resulting in “blunt” or “sticky” ends, depending on the exact function of the specific enzyme. When a bacterial cell is infected by a virus, the REase will bind to and cut its recognition sequence if present in the virus DNA; thereby, hindering or stopping the effects of the virus on the host. But this process is not specific to host or virus DNA.
To protect their own genetic information, bacterial cells also produce a methyltransferase (MTase). This enzyme is specific to the same site as its corresponding REase, but instead of cutting the DNA at this site, it tags the DNA with a methyl group, which sterically inhibits the binding of the REase. In type I R-M systems, the REase and MTase are apart of a single enzyme whereas in the simpler, type II system, they are found as two individual enzymes. When isolated, the REase from a type II system can be used independently of the methylase to digest DNA for analysis and synthesis making these enzymes a valuable tool in synthetic biology.
General Considerations
In order to produce restriction enzymes in vivo, it is necessary to protect the host DNA from auto-restriction and cell death. This is possible by expressing an excess of the MTase in relation to the REase that is being produced within the cell. Since REases form dimers (whereas MTases are monomeric) in their active form there is a natural lag phase before enzymatic activity is observed. This also implies a 2:1 excess of active MTase within the cell if both enzymes are expressed at the same rate. We intend to create a construct that expresses both enzymes on the same promoter in order to test if MTase can effectively protect the cell's DNA through this mechanism alone. It is possible that this will not provide the MTase with the advantage needed to sufficiently protect the host DNA from auto-restriction. In this is the case, it may be possible for the MTase to out compete the REase by simply expressing the MTase on a strong constitutive promoter and the REase on a weak constitutive promoter. Alternatively, it should be possible to express the MTase on a constitutive promoter and use an inducible promoter system to delay the expression of the REase until the host genome is protected.
Methods of Production
We ordered 500bp segments of DNA that, when connected through a Gibson Assembly, contained the full sequence of our genes of interest (EcoRI and EcoRI methylase). After Gibson Assembly into plasmids failed to produce any transformants, we attempted to PCR amplify our constructs from linear intermediates in the Gibson Reaction.  These PCR reactions resulted in smearing and unexpected bands (below left).  Finally, we used a PCR purification kit to clean up the Gibson Reactions before PCR amplification. Smearing disappeared and our expected bands became more prominent (below right) although multiple unexpected bands remained, most likely due to the DNA blocks in the Gibson Assembly annealing incorrectly, especially in the EcoRI Methylase where three 500bp DNA blocks were required.
To remedy this problem, we gel extracted the band of the correct size to isolate it from the other products. The resulting DNA was at such a low concentration that a second round of PCR was required and a subsequent purification.
In order to synthesize a plasmid containing the desired genes, we digested EcoRI, its methylase, and a previously made Freiburge pSB1C3 backbone with XbaI and AgeI. All three DNA sequences were extracted and purified with an extraction kit.
We then ligated EcoRI into the pSB1C3 backbone and EcoRI methylase into the pSB1C3 backbone at 16C for 12 hours. The ligation products were then transformed into DH10B cells and selected for on Chloramphenicol plates. We received over 200 colonies for EcoRI and about 150 colonies for EcoRI methylase.
The samples with the correct band size on the gel were sequenced. Both EcoRI and EcoRI methylase were sequence confirmed; however, we found a nonsense mutation in the methylase, which rendered it non-functional. At this point, we successfully produced and submitted a Freiburg 1C3 backbone with an EcoRI coding gene.
In order to fix the deletion in our methylase gene, we performed Site Directed Mutagenesis (SDM) using primers that flanked the deletion. One of these primers had an extra guanine on its tail in order to add back the missing nucleotide. This product was then transformed into chemically competent cells and selected for on Ampicillin. To test our new methylase in vivo, we grew two overnights: one of the non-functional methylase cells and one of the new methylase cells. The DNA was isolated through mini-prep. The plasmids were then digested with PstI and XbaI as a control to determine the expected bands when the plasmid was cut twice. To test the methylase’s ability to methylate the EcoRI site we also digested with PstI and EcoRI. The results can be seen below.
As can be seen by the two bands in lane 2, the non-functional methylase does not protect the DNA from EcoRI. However, in lane 4, there is a single band of about 4000bp indicating that the DNA was cut only once. The now functional methylase protected the EcoRI cut site showing that it is functional in vivo.
Next we tested our methylase for activity in vitro. We grew a fresh 250ml over night culture then lysed the cells using a French press. 25ul of the lysate were loaded into a 1%, no EtBr gel and ran for 5 minutes at 120V. 25ul were then removed from the gel.  We hypothesized that because enzymes moved slower through agarose gels than DNA, much of the DNA would enter the gel during this short time, leaving most of our methylase. This is referred to as our gel purified lysate.
We then set up four samples. In the first we added plasmid DNA (amilCP on a vector) to our lysate. In the second, we added only lysate so we could see what background showed up. The third contained only plasmid that would be unprotected from EcoRI so we could compare our results to uncut plasmid. Lastly, we added plasmid DNA to our gel purified lysate. These samples were incubated at 37C for 1 hour to allow the methylase time to methylate the DNA.
Each of these samples were then digest with PstI and EcoRI. The results can be seen below.
As can be seen in lane 2, much genomic DNA and the plasmid DNA from our methylase is present in the lysate; however, in lane 4, most of this DNA is removed by our purification method. We do loose some functionality of the methylse possibly by loosing it in the gel or decrease in concentration but it mostly works
Since the start of summer we have successfully isolated the EcoRI restriction enzyme gene. We have also isolated the EcoRI methylase gene and confirmed that it functions in vivo and in vitro.
The Next Step
Continuing with our project, we would like to obtain a functional EcoRI methylase to then express with a promoter RBS. We then would transform plasmids with the expressed EcoRI and EcoRI methylase into bacterial cells and observe any changes in growth rates and to purify out the produced EcoRI protein.
Other restriction endonucleases we would like to express through a plasmid are: ApoI, PstI, SpeI, and XbaI.
ApoI for Malaria Testing
Another application of our project is for testing for antimicrobial resistant malaria strains. Malaria is an infection of Plasmodium falciparum and caused approximately 660,000 deaths in 2010 [1]. Chloroquine, an antimicrobial drug, is widely used to treat malaria; however, an increasing prevalence of chloroquine resistance has complicated malaria treatment. This increased resistance is linked to a mutation in one of the membrane transporters of Plasmodium falciparum (pfmdr-1) where a tyrosine residue is replaced by an arginine residue and a mutation of the transporter pfcrt where lysine is replaced by threonine [2]. The same mutation in pfmdr-1 may also be involved in lumefantrine resistance and serves as a marker for mefloquine vulnerability [3].
In order to treat malaria, the infecting bacteria must be tested for the presence of these mutations. One possible method of testing for and monitoring the spread of malaria strains is by digestion with the enzyme ApoI [4]. In wild-type strains, digestion with ApoI results in two bands during gel electrophoresis; however, chloroquine resistant strains have a mutation that destroys this cut site so only one band is produced [5].
If we applied the same procedure that we did for EcoRI we could produce a plasmid containing the ApoI gene. We could then express it by growing cultures and later harvest the enzyme for uses in restriction digests. If this restriction enzyme could be readily available, in cheap and resource limited conditions, it would make restriction enzyme assay viable for testing for antimicrobial resistance in malaria.
1. Organization WH: World Malaria Report 2012. 2012.
2. H.H. Abruquah FYB, S.C.K. Tay, and B.W.L. Lawson: Resistance-Mediating Polymorphisms of Plasmodium Falciparum Among Isolates from Children with Severe Malaria in Kumasi, Ghana. Ghana Medical Journal 2010, 44:52-58.
3. Laufer MTaM: Resistance to Antimalarial Drugs: Molecular, Pharmacologic, and Clinical Considerations. Pediatric Research 2009, 65:64-70.
4. Durand R, Huart V, Jafari S, Le Bras J: Rapid Detection of a Molecular Marker for ChloroquineResistant Falciparum Malaria. Antimicrobial Agents and Chemotherapy 2002, 46:2684-2686.
5. Abruquah, HH: Resistance-Mediatign Polymorphisms of Plasmodium Falciparum Among Isolates fraom Children with Severe Malaria in Kumasi, Ghana. Ghana Medical Journal 2010.
Retrieved from " http://2013.igem.org/Team:CU-Boulder/Project/Kit/RestrictionEnzymes "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
