<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNSW_Australia_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNSW Australia/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-main-collapse">
                    Menu </BUTTON></DIV><DIV class="collapse navbar-collapse navbar-main-collapse"><UL class="nav navbar-nav"><LI><A class="page-scroll" href="https://2016.igem.org/Team:UNSW_Australia">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Description">★ Description</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Demonstrate">★ Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Notebook">Notebook</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Team">Our Team</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Collaborations">★ Collaboration</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Basic_Part">★ Basic Parts</A></LI><LI><A class="page-scroll" href="https://2016.igem.org/Team:UNSW_Australia/Safety">Safety</A></LI><LI><A class="page-scroll" href="https://2016.igem.org/Team:UNSW_Australia/Attributions">★ Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/HP/Silver">★ Silver</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/HP/Gold">★ Gold</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Integrated_Practices">★ Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Engagement">★ Engagement</A></LI></DIV></DIV><NAV class="navbar col-sm-3 nav-wrapper"><UL class="sidebar-nav nav" data-spy="affix" data-offset-top="700"><LI><A class="page-scroll" href="#week1">Week 1</A></LI><LI><A class="page-scroll" href="#week2">Week 2</A></LI><LI><A class="page-scroll" href="#week3">Week 3</A></LI><LI><A class="page-scroll" href="#week4">Week 4</A></LI><LI><A class="page-scroll" href="#week5">Week 5</A></LI><LI><A class="page-scroll" href="#week6">Week 6</A></LI><LI><A class="page-scroll" href="#week7">Week 7</A></LI><LI><A class="page-scroll" href="#week8">Week 8</A></LI><LI><A class="page-scroll" href="#week9">Week 9</A></LI><LI><A class="page-scroll" href="#week10">Week 10</A></LI><LI><A class="page-scroll" href="#week11">Week 11</A></LI><LI><A class="page-scroll" href="#week12">Week 12</A></LI><LI><A class="page-scroll" href="#week13">Week 13</A></LI><LI><A class="page-scroll" href="#week14">Week 14</A></LI><LI><A class="page-scroll" href="#week15">Week 15</A></LI><LI><A class="page-scroll" href="#week16">Week 16</A></LI><LI><A class="page-scroll" href="#week17">Week 17</A></LI><LI><A class="page-scroll" href="#week18">Week 18</A></LI><LI><A class="page-scroll" href="#week19">Week 19</A></LI><LI><A class="page-scroll" href="#week20">Week 20</A></LI><LI><A class="page-scroll" href="#week21">Week 21</A></LI></UL></NAV><DIV class="row"><SECTION id="week1" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2 col-lg-offset-2"><H3 id="week1">Week 1: May 30 - June 5</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>Labwork begins in earnest! We received three plasmids from AddGene: pKDsgRNA-p15, pKDsgRNA-ack, and pCas9-CR4. These were <B>transformed</B> into our cloning strain, streaked out for single colonies, and <B>miniprepped</B> to build up stocks.</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week2" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3>Week 2: June 6 - June 12</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>We took last week's minipreps and ran them on a gel. It took three attempts, but we got there!</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week3" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3>Week 3: June 13 - June 19</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>Labwork took a backseat as the six of us began our exam period. We still managed to <B>transform</B> the pCas9 into the two protein expression strains: NEB T7-Express, and Thermofisher BL21-Star</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week4" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week4">Week 4: June 20 - June 26</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>All we managed to fit in was <B>making glycerol stocks</B> of all the plasmid-bearing cells we've prepared thus far</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week5" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week5">Week 5: June 27 - July 2</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>With exams finally ending, we sat down to plan where we would be going over the next few weeks</P></DIV></LI><LI class="media"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>We (finally!) received all our ordered parts from IDT. However, we didn't use them in any labwork, merely planning out the best approach, hoping to start fresh next week</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Human Practices</H4><P>Two big events this week! On Monday, Gordon, Jess, Maddie, and Shivani presented at the <B>B.Inspiring STEM</B> conference, while on Tuesday Gavin and Charlotte followed up with an interactive workshop with the <B>Aspire Program</B>. See more at our Public Engagement page!</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week6" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week6">Week 6: July 4 - July 10</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>This week, we made plasmids expressing sgRNAs that target TolA and DegS in the E. coli genome. To do so, we performed a <B>SLiM PCR</B> on the pKD-sgRNA-ack plasmid. We <B>digested with DpnI</B> the products of these PCRs to remove reaction template, and ran a 3-hour long PCR reaction that <B>hybridised</B> the reactants together.</P><P>These were then <B>transformed</B> into the cloning and two expression strains. Unfortunately, the TolA product failed to work, so <B>we retried the process</B>; a gel then revealed that no DNA was present, so we ordered new primers for the SLiM reaction. For DegS, we <B>miniprepped</B> and submitted the product for sequencing at the Ramaciotti Centre </P></DIV></LI><LI class="media"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>We used <B>PCR</B> to amplify the parts received from IDT: expression constructs for g3p, tolR, and alMGS. For the latter, we tried several <B>overlapping PCRs</B> to fuse the parts together, but gels revealed bands running at incorrect sizes. The other two were successfully <B>ligated</B> into pSB1C3 (both), pSB1T3 (tolR only), and pSB1K3 (g3p only)</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week7" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week7">Week 7: July 11 - July 17</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>We <B>retried the degS-ack miniprep</B>, and got a yield good enough to resubmit for sequencing. We also <B>rehybridised the SLiM PCR products </B>for tolA and <B>retransformed </B>them into our cloning strain, but no colonies grew; we believed that there was something wrong with the primers, so we designed new ones and put them on order. At the same time, we ordered primers for making a pKD-sgRNA-degP plasmid, to knockout a related protein to degS called degP.</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>For alMGS, we reattempted fusion of its two fragments via <B>overlapping PCR</B>. However, running the product on a gel revealed that the reaction didn't complete and still contained more reactants than products. We <B>retried this procedure twice</B>, tweaking parameters such as molar ratios and the number of cycles, but neither helped. We then tried <B>a PCR on the overlapping PCR's product</B>, hoping to amplify the full product specifically from the mix; this only marginally improved the outcome, so <B>we did another PCR using that as template</B>, only to again see little improvement; so we shelved work on alMGS for now.</P><P> We did, however, have more luck with g3p and tolR, as we <B>digested </B> their PCR products with EcoRI and PstI, <B>ligated</B> them into the linearised plasmid backbones from the DNA distribution kit, and <B>transformed</B> them into our cloning strain. We put g3p into the kanamycin backbone, and tolR into the tetracycline backbone, such that down the line we could pop both plasmids into a single cell and see how the two proteins interact</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week8" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week8">Week 8: July 18 - July 24</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>Primers arrived on Thursday, and so we <B>repeated the process of converting pKD-sgRNA-ack </B>, this time to -degP and to -tolA, which was quite arduous (see Week 6). On a nicer note, sequencing results revealed that -degS plasmid had been taken up by transformants 24, 25, and 26, and so we could continue working with these. </P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>We began the process of <B>screening our transformants</B> for g3p- and tolR-bearing plasmids. This involved performing <B>colony PCRs</B> to amplify the insert, running the products on a gel, and observing for the correct size. We ran over 50 reactions; nothing seemed to be running consistently, making it seem like there were no positive hits, but we then <B>ran a gel in which ladders were placed in every lane</B>, revealing that there was an inconsistent, random effect of lane number on migration distance. So <B>we got a new gel tank, reran the colony PCRs</B>, and finally identified a putative positive hits for two of our g3p constructs and one of our tolR constructs; we then <B>miniprepped</B> the plasmids from these. By Friday we were exhausted of repeating the same procedure over and over again, but we managed to <B>PCR</B> amplify the insert (again) from these minipreps, and submit them for sequencing - always good to get more confirmation than just size!</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week9" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week9">Week 9: July 25 - July 31</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>We ran a gel of what we thought should be new pKD-sgRNA-tolA and -degP plasmids, but no band appeared. Hence, we <B>retried entire conversion process (see Week 6)</B>, got a positive result, and then DpnI digested it. As for pKD-sgRNA-tolA, we wanted to put them into the expression strains that already had the pCas9 plasmid, but these stubbornly grew really slowly in and were unable to reach the correct OD to make them competent.</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P><B>More colony PCRs this week!</B> We continued with the screening process from last week, and eventually found the final putative positive hits that had the correct size insert. For these, again, we <B>miniprepped</B> the plasmids, <B>PCR amplified</B> the insert again, and submitted them for sequencing. We also <B>transformed</B> the cloning strain with an RFP-expression plasmid, hoping to use that for microscopy later.</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week10" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week10">Week 10: August 1 - August 7</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>We managed to <B>make our pCAS9-bearing cells competent</B> using calcium chloride, and <B>transform</B> them with pKD-sgRNA-degS, too. Once we managed to grow them up sufficiently, we performed a <B>colony PCR</B> to sequence their plasmids and double check that everything was perfect</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>We took a new approach to fusing the two alMGS fragments into the full gene: <B>amplification using 5'-phosphoprimers</B>, such that the two products could be ligated together. Running the results on this procedure on a <B>gel</B> revealed the same result as last time - the reactants were still present at a much higher abundance than the product we wanted (i.e. the full gene). We tried a <B>gel extraction</B> of the correct-sized band, but the gel itself had an unexpected error and we lost all our product...</P><P> In what was good news, though, we managed to order the gene in from fragment from IDT, negating the need to fuse it together, hence we decided to wait until that arrived.</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Characterisation Team</H4><P>We <B>made competent the cloning strain</B> bearing either g3p or tolR expression constructs, and <B>transformed</B> these with RFP-expression plasmids. We took these double transformants, as well as the respective single transformants, to the <B>Zeiss LSM780 confocal microscope</B> at UNSW's Lowy Centre, taking a variety of cool snaps of fluorescent, hypervesiculating E. coli! However, we realised once we got there that we forgot to bring our negative control - a strain with just RFP - and so the strength of our data was weakened.</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week11" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week11">Week 11: August 8 - August 14</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>We first <B>sequenced</B> our expression strains to check for the genes degS, degP, and tolA. Then, we used arabinose and aTc to <B>activate CRISPR-Cas9</B> with the sgRNA for degS! We also managed to <B>miniprep</B> the degP- and tolA-sgRNA plasmids, but we saw nothing when we ran them on a gel; we then <B>PCR amplified</B> the sgRNA region of this plasmid, ran that on a gel and saw that it was present, then <B>sent it off for sequencing</B></P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>Despite the orders we made last week, we still <B>retried the phospho-primer PCR</B> of the two alMGS fragments, <B>ligated</B> them, and ran the product on a gel. However, the gel turned out as a smear, meaning no clear band could be excised for purification.</P><P> We also <B>transformed</B> our tolR and g3p expression products into the expression strains, ready for further analysis next week!</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week12" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week12">Week 12: August 15 - August 21</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>We ran into a roadblock where our double transformants (with both Cas9 and any of the sgRNA plasmids) began to grow really slowly, taking days to appear on the relevant double-antibiotic plates. This week was spent <B>trying out different growth conditions, media, antibiotic concentrations</B>, to try to help our cells grow. </P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>Lots of work this week! First, we <B>cured both of our expression strains</B> of a thermolabile plasmid - this gave us a plasmid-free version (tested for its antibiotic sensitivity) of the expression strain, to use as a negative control. We then ran our first <B>SDS-PAGE</B> using total <B>protein samples extracted</B> from empty, tolR-expression, and g3p-expressing versions of both expression strains. We stained with <B>GelCode Blue protein stain</B>, looking for extra bands at 9kDA for each of tolR and g3p relative to the negative control. With the help of our advisor we sent in one of these bands to the <B>Mass Spectrometer</B> (which later came back negative).</P><P>The following day, we <B>reran the protein samples on the SDS-PAGE</B>, and this time performed a <B>Western Blot</B>, with the primary antibody targeting g3p's and tolR's His-tag. Unfortunately, while the positive control sample worked, we couldn't detect either g3p or tolR, implying that they weren't being expressed.</P><P>And, to cap off a busy week, we <B>transformed</B> J04450 (RFP), J23119 (a promoter), B0034 (RBS), and B0017 (double terminator) into our cloning strain, for later BioBrick assembly.</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week13" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week13">Week 13: August 22 - August 28</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>While still figuring out how to accelerate our cells' growth (or, at least, return it to a normal rate for E. coli), we <B>remade the sgRNA-tolA plasmid</B> (see Week 6), and <B>transformed</B> it into our cloning strain. We <B>miniprepped</B> a few of the transformants, and verified them by running on a gel and submitting for sequencing. By Friday, though, our cells started growing at a normal rate, the result of much retransforming, restreaking, and reculturing, and so we <B>activated the Cas9 plasmid</B> to knockout DegS and DegP.</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>From the overnight cultures of last week, we made <B>minipreps</B>. Each of the J04450s were <B>digested with EcoRI and PstI</B>, to give us an empty backbone into which we could pop our inserts, as if it were to self-ligate the transformants would grow as red.</P><P>Then, on Wednesday, taking our 3 recently-synthesised GBLOCKs (for alMGS and g3p) from IDT, we <B>PCR amplified, digested, ligated </B>into a the J04450-derived backbone, and <B>transformed</B> into the cloning strain.</P><P>On Thursday we then <B>re-extracted separate total and soluble protein fractions</B> from our cloning and expressions strains purportedly expressing g3p and tolR. Coming in on Sunday, these were run on an <B>SDS-PAGE</B>, but the negative control lane we ran wound up as a smear, making it tough to see if we had an extra band present. We also ran a separate <B>SDS-PAGE</B> to transfer to a <B>Western Blot</B>, but again we were unable to detect our proteins.</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week14" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week14">Week 14: August 29 - September 4</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRIPSR Team</H4><P>On Tuesday we <B>redid our activation of Cas9</B> in all three cell lines, for knocking out degS or degP (i.e., six knockouts in total). Unfortunately, when we plated these out, no serial dilutions were made, giving us lawns of bacteria. We thus had to <B>isolate single colonies by restreaking</B>, but also began to <B>screen for successes using colony PCR</B> to check the size of the genes (knockouts being smaller). We also ran more <B>pkD-sgRNA-TolA minipreps on gels</B> and submitted them for sequencing at the Ramaciotti Centre </P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>Beginning Monday, we started <B>screening last week's transformants</B> from  for the presence of the alMGS or g3p inserts we wanted; this involved <B>colony PCRs</B> and submitting for sequencing. On Tuesday, we received 12 more GBLOCKs from IDT (for inducible or membrane-localised parts), which we <B>PCR amplified, digested, and ligated</B> into the BioBrick plasmids. We then <B>transformed</B> these, and <B>pick'n'patched</B> for single colonies which we could begin to screen.</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Characterisation Team</H4><P>On Friday, we tested out our <B>FN-464</B> membrane dye under the confocol microscope. The images turned out beautifully! Unfortunately, we only imaged strains lacking any hypervesiculation modifications, as we only had 30 minutes of microscope time and merely wanted to check if the dye worked</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week15" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week15">Week 15: September 5 - September 11</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>This week was devoted to sorting out whether our Cas9 system actually was operation, and to do so we devised a series of tests. We <B>induced the system and looked on an SDS-PAGE</B> for extra bands; we <B>compared the growth rate of the cells with or without added aTC</B> (which would reactivate the system and cut the genome); the rate of growth would be proportional to the percentage of cells that had already been cut, as they would be immune to recutting). Results were inconclusive...</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>To start the week, we received sequencing results telling us that our alMGS insertions were, well, junk, as they came back with a sequence matching to nothing. We began to <B>screen more and more colonies</B> for their sizes using colony PCR and gels, but the bands came back in a range on inconsistent sizes - despite testing white colonies that should show no self-ligation. We concluded that the new backbone preparation we used was not working well, and, given how little time remained, we decided to step back to transforming using the iGEM-supplied linearised backbone. We then <B>retransformed</B> the most critical parts (inducible tolR, and periplasm-localising GFP) into the cloning strain on Sunday. Sunday also saw us <B>screen some of the old colonies</B> from several weeks back for their parts, to try and start putting together our BioBricks for submission</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Characterisation Team</H4><P>On Thursday and Friday, <B>Michael Carnell</B> trained Charlotte and Gavin in how to use the Zeiss LSM 780, a confocal microscope. Each session was around three hours in length, covering the theoretical underpinnings, advantages, required sample treatment, parameters to consider, and use of the machine. </P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week16" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week16">Week 16: September 12 - September 18</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>We used <B>colony PCR</B> to screen 11 colonies that were potential degP knockouts, but none showed the band size for which we were hoping. </P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>The glut of <B>colony screening</B> for this week began on Monday. As more sequencing results came back as junk, we began <B>rescreening more tolR and g3p transformants</B>. The new transformants of the critical inducible and periplasm-localising parts were also screened. Then, on Thursday, we received two new plasmids: pET-Duet, and pRSF-Duet. We <B>transformed</B> these into our cloning strain, to start building up a stock with which we can start working. Much of our time was then devoted to <B>designing primers</B> to make our existing parts compatible with these Duet plasmids, as our intention next week is to start cloning our genes into this, and hopefully successfully express that way</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week17" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week17">Week 17: September 19 - September 25</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>We set aside Tuesday for another growth rate analysis of our Cas9-edited cells, as per Week 15. Again, the data were inconclusive, leaving us with no direction for this part of the project</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>Starting Monday, we began <B>rescreening colonies</B> to find those bearing our T7-controlled construct. We hit various snags, as colony PCRs using VF2 and VR revealed multiple bands of the same size across all 7 colonies screened. We then <B>miniprepped</B> all these colonies, and performed the same PCR on the pure plasmid template, but saw the same pattern of bands; the consistency of this perhaps hinted at contamination of the primers, but more than likely it was due to some quirk of the sequence. We found out the VR binds two locations (once at its correct site and once in the terminator), but even that accounted for only two of the many bands. As such, we set this aside for now...</P><P>But we had other colonies to <B>screen</B>, too. We tested 11 colonies from each of tolR and g3p in pSB1C3, to hopefully submit to the registry. For each we identify one with the correct sized insert, and so we <B>miniprepped</B>, <B>PCR amplified the insert</B>, and submitted for sequencing</P>At the same time we also began working on isolating our alMGS-bearing cells. For this we took two parallel approaches: using (yet more) <B>colony PCRs</B> to screen transformants, but also <B>redigesting alMGS</B> (i.e. restarting the 3A assembly) as we realised that alMGS has an EcoRI site in its middle and hence was cut into multiple fragments during EcoRI digestion, thus vastly reducing the likelihood of a transformant taking up the full alMGS product. To solve this, we <B>digested weakly with EcoRI</B>, and <B>gel extracted</B> the band that corresponded to a single rather than a double cut. Late in this process, though, we also realised that having an EcoRI site excludes it from being submitted to the registry, and so stopped work on it...<P>Also, regarding our new Duet Expression Plasmids, we <B>miniprepped</B> colonies, ran these on a <B>gel</B> to confirm their sizes, <B>digested</B> to open them up to our insert, and finally <B>phosphatase</B> treated.</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week18" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week18">Week 18: September 26 - October 2</H3><UL class="media-list"><LI class="media"><DIV class="media-body"><H4 class="media heading">CRISPR Team</H4><P>We finally decided to pull the plug on the CRISPR side of our project; it'd simply proven too fiddly and tough to optimise. Instead, we <B>ordered KOs of degP and tolA</B> from Yale's E.coli Genetic Stock Centre. As mentioned elsewhere, we acknowledge that the genetic background of these strains will be different to those in which we will test our other parts, which will present somewhat of a confound, but we believe that collecting the data but making that caveat clear is still important</P></DIV></LI><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P> Charlotte managed to take <B>images of the DegP KO</B> that recently arrived. It was her first time flying solo with the AiryScan confocal microscope, and her machine time was limited on the day, meaning that the images turned out too blurry to be conclusive and strong enough to submit - while the E.coli were imaged well, we just couldn't get enough clarity to see smaller OMVs (if they're actually there...). We also began to <B>harvest OMVs</B> using our centrifugation protocol</P><P> And, due to the folding of the CRISPR team, all hands were devoted to cloning and expressing our parts. Three of us <B>tried to clone our constructs into pSB1C3</B> (to submit to the registry), and the other three <B>put it into our Duet plasmids</B> (to express).</P><P> To put our six parts into pSB1C3, we <B>started from scratch</B>: 1) PCR amplify six of our synthesised parts, 2) PCR amplify the backbone, 3) digest both with EcoRI and PstI, 4) phosphatase-treat the backbone, 5) cleanup both, 6) ligate, and 7) transform. This failed the first time, so we <B>tried the transformation again</B>, this time with new SOC media, as we used the wrong type of glucose in it. It failed, so we <B>retried the ligations</B> with <B>QuickLigase, T4 Ligase, and a Gibson</B> - none of those transformations worked! It was an enormous effort, though, so despite the setbacks we were all proud</P><P> To put our six parts into the expression vector, we did <B>exactly the same process as above</B>, only using different primers for the initial PCR amplification (to add on specific restriction enzyme sites), and the above step (2) was not needed. About twenty transformants were made, but <B>colony PCR</B> revealed they had not taken up the insert.</P><P> Further, <B>we began work on a part we received from the registry</B>: INPNC. We 1) transformed, 2) miniprepped, 3) digested with BamHI and PstI, 4) ligated in GFP and RFP that had been prepared to add in a BamHI-overhang, and 5) transformed. As above, this didn't work either</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week19" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week19">Week 19: October 3 - October 9</H3><UL class="media-list"><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>We first <B>retried reinserting our parts into the Duet expression vectors</B>, but this time <B>we ligated using a Gibson</B>. Hundreds of transformants were made, and we stayed in late <B>screening as many as we could using colony PCRs</B></P><P>And we got several hits! Our next task was to <B>test for expression</B> from our colony PCR hits. To do so, we first grew up our cells with TolA inserted into the Duet vector, and <B>induced at OD 0.4 with 0.5mM IPTG</B>. After two hours, we <B>extracted both soluble and membrane protein </B>fractions. The next day we ran <B>two SDS-PAGEs</B> with these samples, plus some samples received on USYD as part of our collaboration. One SDS-PAGE was <B>stained with Coomasie</B>, while the other was run through a <B>Western Blot </B>assaying for the His-tag. Both of USYD's proteins were found to be expressing, while none of ours were. We then realised that we used the cloning strain for our expression test, which lacks the ability to be induced by IPTG...</P><P>We thus <B>miniprepped</B> the plasmids out of these, and <B>transformed</B> them into the expression strain the next day. Rather than test for expression via SDS-PAGEs, we began to look for GFP expression under UV lamps, but couldn't see anything</P><P>We also began the arduous task of <B>harvesting OMVs from our KO strains</B>. To do so, we set up 500mL cultures of delta-tolA, delta-degP, and a negative control. After a night of growth, we spun down briefly to pellet the cells, and ran the supernatant through a 0.22 micron filter. This clarified supernatant was then spun through a 3kDA filter, at 3400g for 20 minutes, <I> 20mL at a time</I>. As you can imagine, this took an age, and was spaced over several days. </P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week20" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week20">Week 20: October 10 - October 16</H3><UL class="media-list"><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>As this week rolled in, we finished <B>colony PCRs</B> for all our remaining transformants with Gibson plasmids (and there were well over 100). Disappointingly, we received no hits for anything cloned into Duet's second multiple cloning site. As such, we <B>redid the Gibson ligation</B> reaction for these plus (g3p, and periplasm-localised RFP), this time doing an <B>overnight DpnI digest</B> to lower the background, and balancing the stoichiometries as best we could. We then <B>transformed</B> our expression strain with these.</P><P>The following day we began screening these via <B>colony PCR</B>. In what might be the first stroke of luck, our first gel screening 10 g3p-in-Duet colonies got 10 hits! We <B>miniprepped</B> two of these, performed a <B>PCR</B> to sequence the insert, and at the end of week received sequencing data confirming that these were both successfully inserted.</P><P>During the week we also tried <B>a simple assay for quantifying OMVs</B>. Taking our solutions of harvested OMVs, we <B>loaded them into a 96-well plate </B>and added a dye that fluoresces strongly when embedded into lipid membranes. This was a simple first pass at comparing OMV production between delta-tolA, delta-degP, and a negative control, but the data pointed to a need for further testing when we get a different set of filters.</P><P>And on Wednesday we went <B>back to the Airyscan microscope</B>, this time to image our harvested OMVs at high resolution. Unfortunately, we saw nothing, and upon further digging realised that the microscope could never hit a good enough resolution to clearly resolve OMVs, which typically average around 15nm in diameter. It's tough to get this heartbreak so late in the project, but we had to look for alternatives and located a device called the NanoSight, which we will try next week.</P></DIV></LI></UL></DIV></DIV></SECTION><SECTION id="week21" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2"><H3 id="week21">Week 21: October 17 - October 23</H3><UL class="media-list"><LI class="media dry"><DIV class="media-body"><H4 class="media heading">Overexpression Team</H4><P>The final week of iGEM saw us in a mad rush to finish parts. Here, we reattempted <B>Gibson ligations</B> and <B>transformations</B> to put missing parts into pSB1C3, and began the screening process by <B>colony PCRs</B>.</P><P>We also completed <B>two runs of harvesting OMVs</B>, using our latest protocol with a 100kDa filter. The first of these was <B>given to the team at EnGeneIc</B> to image using their <B>Nanosight</B>, while the rest were used for nanodropping, to test whether this could be adapted to measure OMV concentration</P><P>And in the few hours before Wiki freeze, we will be going to the <B>Airyscan microsope</B> for one last run. Let's hope it's good!</P></DIV></LI></UL></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>