Team:UNSW Australia/Experiments
Menu
Home
Project
★ Description
Experiments
★ Demonstrate
Results
Notebook
Team
Our Team
★ Collaboration
Parts
Parts
★ Basic Parts
Safety
★ Attributions
Human Practices
Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
Protocols
CRISPR Protocols
Site-directed, Ligase-Independent Mutagenesis (SLIM) Hybridisation
Materials
SLIM primers, including two gene-specific primers (FS and RS) and two tailed primers (FT and RT). The tailed primers have identical gene-specific sequences found in FS and Rs, but also carry an adaptor sequence 5’ to the sequence-specific nucleotides which is complementary between FT and RT
Template plasmid
DpnI
H-buffer (300mM NaCl, 20mM Tris, pH 9.0, 20mM EDTA, pH 8.0)
Method
Two PCR amplification reactions were performed, using:
FS & RT primers
FT & RS primers
The PCR products were DpnI digested for 60mins at 37 degrees
SLIM hybridation was set up with 10μL of each DpnI-treated PCR reaction product, 10μL of H-buffer, and sterile water to a final volume of 50μL
Hybridisation was performed using two cycles of 65 degrees for 5 mins and 30 degrees for 15 mins
An aliquot of 20μL from the reaction was used to transform competent E. coli
Scarless Cas9 Assisted Recombineering (No-SCAR) Genome Editing in E. coli
Materials
pCas9-CR4
pKDsg-DegP or pKDsgTolA
NEB T7 Express Competent E. coli
Oligonucleotides for homologous recombination with the flanking regions of genomic DegP or TolA devoid of the gene
Method
E. coli was transformed with pCas9cr4 plasmid, plated on LB + CM and incubated overnight at 37 degrees
Transformants were made chemically competent and transformed with either the pKDsg-DegP or pKDsg-TolA plasmid prepared earlier. These were plated on LB + CM + SM and incubated overnight at 30 degrees
Double transformants were then grown in SOB at 30 degrees until OD600 of approximately 0.5
50mM L-arabinose was added to the to induce the lambda-red system
After 20 mins of incubation with L-arabinose, cells were then made chemically competent
Oligonucleotides were added to 50μL of the experimental cells at a final concentration of 10μL. Negative control cells received no oligonucleotides.
Cells were recovered in SOC for 2 hours, then plated on LB + CM + SM + aTc and incubated overnight. The addition of aTc was for induction of Cas9.
The following day, significantly less growth should be observed on the negative control plate. Colonies on the experimental plate were screened for genomic DegP or TolA knockout by colony PCR.
G-Block Cloning Protocols
Restriction Enzyme Digestion
Materials
NEB RE Buffer (2.1, 3.1 or cutsmart)
Restriction Enzymes e.g. EcoR1, Pst1
MiliQ Water
DNA Template
Method
Add into each tube: 250ng of DNA, 2.5μl Buffer, 0.5μl of each RE
Make up the total volume to 20μl with MiliQ Water.
Incubate at 37 degrees for 30 minutes.
Heat inactivate at 80 degrees for 20 minutes.
Perform Clean-up protocol. Then store in freezer.
E.Coli Transformation
Materials
Competent E.coli cells
Purified plasmid or ligation reaction
SOC Medium
LB Agar plates (with appropriate antibiotics)
Method
Thaw competent cells on ice.
Add 30-50μl of competent cell to a chilled 1.5ml tube containing 1μl of purified plasmid (2μl if a ligation plasmid)
Leave on ice for 30 minutes.
Heat shock at 42 degrees for 90 seconds.
Leave on ice for 2 minutes.
Add 400μl of SOC
Shake at 37 degrees for 1 hour.
In biosaftey cabinet, plate the full volume onto an LB agar plate and spread
Ligation
Materials
Digested plasmid backbone
Digested insert fragment
T4 DNA ligase buffer
T4 DNA Ligase
MiliQ Water
Method
Add the following into a 1.5ml tube: ~25ng of digested plasmid, equimolar equivalent digested fragment (<3ul), 1ul 10x T4 ligase Buffer, 0.5μl T4 Ligase.
Make up the volume to 10μl with MiliQ Water.
Leave at room temperature for 30-60 minutes.
Heat inactivate at 80 degrees for 20 minutes.
Perform Clean-up protocol. Then store in freezer.
Colony PCR
Materials
Lysis Buffer (TE + 0.1% Triton-X)
Bacterial Single Colonies
Method
Prepare 1.5ml tubes with 15μl of lysis buffer for each colony.
In biosaftey cabinet, gently touch a colony with a sterile tip and place the tip the the tube.
Leave for 2 minutes.
Remove tips, and boil cells at 90-100 degrees for 15 minutes.
Spin down the cell debris for 10 minutes at 14,000 g.
Use 1μl of the supernantant for the template of a 10μl PCR reaction.
Miniprep
Materials
15ml Falcon Tube
Qiagen QIAprep Spin Miniprep Kit
1.5ml tubes
Method
Follow the QIAGEN QIAprep Spin Miniprep Kit Instructions
Gibson Assembly
Materials
Microcentrifuge tubes
Gibson reaction mix
Parts for assembly
Method
Add 2.5μl Gibson reaction mix to a microcentrifuge tube
Then add parts for assembly in a 3:1 ratio of insert:vector
Incubate at 50 degrees celcius for 15 mins
Proceed to transformation
Protein Assay Protocols
Total Protein Extraction
Materials
BugBuster
2-mercaptoethanol
LDS loading buffer
Heating block
Tabletop centrifuge (15000g)
Method
Grow cells to an OD of 0.8-1 (induced or non-induced)
Pellet 1ml of cells by centrifuging for 5 minutes
Remove supernatant
Add 100μL bugbuster, 40μL 4x LDS, 10μL 2-mercaptoethanol
Vortex thoroughly
Boil at 95C for 5 minutes
Centrifuge 15 minutes, 15000 rpm
Store at 4C until use
Western Blot
Materials
PBS-T
Skim milk powder
SDS-PAGE Gel
Blot transfer device
Primary antibody with conjugated horseradish peroxidase
3, 3’, 5, 5’ tetramethylbenzidine (TMB)
Method
Separate protein samples on SDS-PAGE
Soak gel in Towbin buffer for 10 minutes
Place gel in transfer stack, and blot it to a membrane
Wash membrane in PBS-T
Soak membrane in 10% w/v skim milk in PBS-T, shaking, for one hour
Wash membrane in PBS-T, 20 minutes, shaking. Repeat once more.
Soak membrane in 10% skim milk in PBS-T, with added primary antibody (Sigma a7058)
Wash membrane in PBS-T, 20 minutes, shaking. Repeat once more.
Pour TMB onto the membrane.
After bands develop, photograph in a geldoc
Running and protein staining of SDS-Page
Materials
SDS Buffer
Protein Sample
SDS-PAGE Gel
MES Buffer
Isopropanol (50% methanol, 10% acetic acid) fixing solution
MiliQ Water
Coomassie Blue
Method
Create a solution of 5:1 protein sample to SDS buffer ratio
Boil samples for 5 minutes
Centrifuge at maximum speed for 1 minute
Assemble an SDS-Page tank with a 4-12% gel and fill with MES buffer, remove the comb
Carefully deposit sample and ladder in the wells
Connect the tank to the power supply
Run gel at 100V for 60 minutes
Dissaemble the tank and break the gel case
Incubate the gel in isopropanol fixing solution for 10 minutes
Wash in MiliQ water for 10 minutes, then discard water and repeat 2 times
Stain gel with coomassie blue for 1 hour
Wash gel with MiliQ water until excess coomassie is removed
Gel is ready to be imaged or to be analysed with mass spectrometry
Sample Preparation for Mass Spectrometry
Materials
0.1 M NH4CO3
0.1 M NH4CO3/50% acetonitrile
100% acetonitrile
SDS page gel that needs to be analyzed
10 mM DTT
55 mM iodoacetamide
MiliQ Water
Proteases and their corresponding buffers
0.1% trifluoroacetic acid (TFA)
0.1% TFA/60% acetonitrile
0.1% formic acid
Method
Cut bands out of the SDS page gel and slice them into 3 or 4 smaller pieces to increase the surface exposed to the reagents
Equilibrate in 0.1 M NH4CO3 for 20 mins at room temperature. Discard liquid afterwards.
Equilibrate in 0.1 M NH4CO3/50% acetonitrile at room temperature until the Coomassie or silver staining comes off. Discard liquid afterwards.
Dehydrate in 100% acetonitrile at room temperature until gel pieces are completely dry
Reduce with 10 mM DTT for 30 mins at 55 degrees Celsius. Discard liquid
Alkylate with 55 mM iodoacetamide for 45 minutes at room temperature, in the dark. Discard liquid
Rinse with MiliQ water
Wash with 0.1 M NH4CO3 for 20 mins at room temperature. Discard liquid
Dehydrate in 100% acetonitrile at room temperature until gel pieces are completely dry
Add sufficient protease in the appropriate buffer in a volume sufficient to cover the gel pieces.
Digest overnight at 37 degrees Celsius (trypsin) or 30 degrees Celsius (chymotrypsin). Avoid overdigesting depending on the protease.
Remove digest solution into a new silanised tube
Add 50μl 0.1% trifluoroacetic acid (TFA) for 30 mins at room temp. Remove liquid.
Add 50μl 0.1% TFA/60% acetonitrile for 30 mins at room temp. Discard liquid
Add 50μl 100% acetonitrile for 10 mins at room temp, removing liquid afterwards.
Vacuum dry sample and resuspend in 10-20μl of 0.1 formic acid
Sample is now ready to be analyzed by mass spectrometry
OMV Purification
OMV Harvesting
Materials
Shaking Incubator
Beckman Avanti Swinging bucket centrifuge
100kDa centrifugal filter units (Millipore)
0.2μm  syringe filters (Millipore)
Sterile 20 ml syringes
PBS
Method
100mL of LB with appropriate antibiotics (if required) was inoculated with each strain for analysis, and placed on a shaking incubator at 37 degrees, shaken at 200 rpm
After 3 hours of incubation, any necessary inducing agents (IPTG at 0.5 mM) were added to the cultures, which were then left on the shaking incubator at 37 degrees overnight
The next day, the OD600 of each culture was measured before OMV harvest
Each culture was then centrifuged for 20 mins at 6000g in a fixed angle rotor to pellet the biomass
The supernatant was recovered and then passed through a 0.2μm syringe-driven filter
40mL of the syringe-filtered supernatant was then loaded into a 100kDa centrifugal filter unit
The filter unit was then centrifuged at 3500g in a swinging bucket rotor for 15 minutes
The volume of the filtrate was recorded and then discarded
PBS of equal volume to the discarded filtrate was loaded into the filter unit on the retentate-side
The filter unit was then centrifuged at 3500g for 10 minutes in the same swinging bucket rotor
The filtrate was again discarded. The retentate was recovered by fitting a retentate collection cap to the filter unit and centrifuging with the filter in the opposite to normal orientation for 5 minutes at 1000g. This was recovered and the volume recorded.
400μL of PBS was loaded directly onto each of the 2 filter units of the filter unit
The sample above was combined with the retentate collected in step 11
OMVs were stored at 4 degrees until analysis
General Protocols
Making E. coli Chemically Competent
Materials
E. coli strain to make competent
LB media
100mM CaCl2 solution
SOC media
Agar plates with LB + appropriate antibiotics
Method
Grow the relevant strain of E. coli in LB overnight (+ antibiotics, if necessary)
The following day, inoculate 5mL of LB + appropriate antibiotics with the overnight culture (1/100 dilution)
After approximately 3 hours, pellet the cells and discard the supernatant
Resuspend the pellet in 20μL of sterile water + 100μL 100mM CaCl2 solution
Leave on ice for 60 minutes
For each 40μL of competent cells, add 1μL of plasmid (per plasmid being transformed)
Leave on ice for 30 minutes
Heat shock at 42 degrees for 90 seconds
Leave on ice for 2 minutes
Add 600μL of SOC media
Recover for one hour at 37 degrees
Pellet the cells, and remove most of the supernatant (leave 50-100L)
Resuspend the pellet in the remaining media, and plate on LB + appropriate antibiotics
Preparation of Lysogeny broth (LB)
Materials
5 grams Yeast Extract
10 grams peptone
10 grams NaCl (anhydrous)
1 liters of RO Water
Method
Weigh out components and add to a 1 liter schott bottle
Make up to 1 liter with RO Water
Mix with a magnetic mixer until fully dissolved
Seal with autoclave tape and autoclave LB for use
Preparation of Agar Plates
Materials
LB agar for E.Coli
Stock concentration of relavant antibiotic
Method
Melt solid agar in microwave and allow to cool.
In the biosaftey cabinet add 1μl of stock concentration antibiotic for every 1ml of agar.
Pour agar into sterile petri dishes until 1/2 full. For a standard 10 cm dish, this will be approximately 20ml.
Cover plates and allow to set at room temperature.
Seal with parafilm, and place in a 4 degrees fridge. An extra covering of foil helps prolong self life.
SOC Medium Preparation
Materials
Anhydrous MgCl2
Glucose Powder
Tryptone
Yeast Extract
NaCl
KCl
RO Water
Method
Measure 0.476 grams of anhydrous MgCl2 and add RO water to a final volume of 10ml.
Filter Sterilize.
Measure 1.8 grams of glucose and add RO water to a final volume of 10ml.
Filter Sterilize.
Add into 96ml of RO water: 2 grams tryptone, 0.5 grams Yeast extract, 58.5 mg NaCl, 18.6mg KCl.
Mic til dissolved with magnetic stirrer.
Autoclave
Aseptically add 2ml of 0.5M Magnesium Chloride and 2ml of 1M Glucose solution to the 96ml of autoclaved solution.
Project Development
As our iGEM project developed, so too did the methods we used. Our project began with a journey through the literature on OMVs, to see what had already been done, and to try to spot the gaps which our project could plug. From this, we identified a range of proteins that could either be overexpressed or knocked-out to induce hypervesiculation. Our aim was to compare the size, number, and stability of OMVs induced by these mutations.
Proteins that could be overexpressed we synthesised from IDT, and cloned into pSB1C3 using 3A assemblly, as for such small parts we felt this was the fastest approach (and free!); proteins that could be knocked-out we designed guide RNAs that fit could be cloned into the noSCAR CRISPR Kit from AddGene, with support from a recent Nature Methods paper [1]. Each of these approaches, however, proved troublesome.
Regarding overexpression parts, a Western Blot revealed that they weren’t expressing from pSB1C3, and so had to clone into the Duet Expression Vectors using Gibson assembly before reliable expression was achieved. The NoSCAR CRISPR system, likewise, seemed to be going well, but we wound up having to pull the plug as deadlines fast approached, where we needed strains to characterise; instead, we ordered knockouts from the Yale Coli Genetic Stock Centre, recommended to us by Dr Jai Tree, a lecturer at UNSW.
Now that we had strains, it was time to begin characterisation. Our first avenue was to try the Zeiss Airyscan LSM 880 microscope; it was only late in the project that we realised that this didn’t have the resolution to see nanoscale-OMVs. This microscope still had use in seeing our localising-fluorescence parts, though. Other papers that characterised OMVs seemed to use Electron Microscopes [2], but we didn’t have access to these. Instead, we contacted biotech companies and found out about the NanoSight, which we wound up successfully using in the closing days of the project.
In sum, like all iGEM teams, we hit numerous roadblocks; it was through contacts and literature searches that we overcame these, to give us the data we have today!
References
Reisch, C.R. and Prather, K.L., 2015. The no-SCAR (Scarless Cas9 Assisted Recombineering) system for genome editing in Escherichia coli. Scientific reports, 5.
Henry, T., Pommier, S., Journet, L., Bernadac, A., Gorvel, J.P. and Lloubès, R., 2004. Improved methods for producing outer membrane vesicles in Gram-negative bacteria. Research in microbiology, 155(6), pp.437-446.
