<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNSW_Australia_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNSW Australia/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-main-collapse">
                    Menu </BUTTON></DIV><DIV class="collapse navbar-collapse navbar-main-collapse"><UL class="nav navbar-nav"><LI><A class="page-scroll" href="https://2016.igem.org/Team:UNSW_Australia">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Description">★ Description</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Demonstrate">★ Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Notebook">Notebook</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Team">Our Team</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Collaborations">★ Collaboration</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Basic_Part">★ Basic Parts</A></LI><LI><A class="page-scroll" href="https://2016.igem.org/Team:UNSW_Australia/Safety">Safety</A></LI><LI><A class="page-scroll" href="https://2016.igem.org/Team:UNSW_Australia/Attributions">★ Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/HP/Silver">★ Silver</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/HP/Gold">★ Gold</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Integrated_Practices">★ Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:UNSW_Australia/Engagement">★ Engagement</A></LI></DIV></DIV><SECTION id="about" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-10 col-lg-offset-1"><H2 style="font-family: Slim-Joe;">Protocols</H2></DIV></DIV></SECTION><SECTION><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading panel-main"><H4 class="panel-title">
						CRISPR Protocols
				</H4></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse1">
						Site-directed, Ligase-Independent Mutagenesis (SLIM) Hybridisation
					</A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>SLIM primers, including two gene-specific primers (FS and RS) and two tailed primers (FT and RT). The tailed primers have identical gene-specific sequences found in FS and Rs, but also carry an adaptor sequence 5’ to the sequence-specific nucleotides which is complementary between FT and RT</LI><LI>Template plasmid</LI><LI>DpnI</LI><LI>H-buffer (300mM NaCl, 20mM Tris, pH 9.0, 20mM EDTA, pH 8.0)</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>
								Two PCR amplification reactions were performed, using: 
								</LI><LI>FS &amp; RT primers</LI><LI>FT &amp; RS primers</LI><LI>The PCR products were DpnI digested for 60mins at 37 degrees</LI><LI>SLIM hybridation was set up with 10μL of each DpnI-treated PCR reaction product, 10μL of H-buffer, and sterile water to a final volume of 50μL</LI><LI>Hybridisation was performed using two cycles of 65 degrees for 5 mins and 30 degrees for 15 mins</LI><LI>An aliquot of 20μL from the reaction was used to transform competent E. coli</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse2">
						Scarless Cas9 Assisted Recombineering (No-SCAR) Genome Editing in E. coli
					</A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>pCas9-CR4</LI><LI>pKDsg-DegP or pKDsgTolA</LI><LI>NEB T7 Express Competent E. coli</LI><LI>Oligonucleotides for homologous recombination with the flanking regions of genomic DegP or TolA devoid of the gene</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>E. coli was transformed with pCas9cr4 plasmid, plated on LB + CM and incubated overnight at 37 degrees</LI><LI>Transformants were made chemically competent and transformed with either the pKDsg-DegP or pKDsg-TolA plasmid prepared earlier. These were plated on LB + CM + SM and incubated overnight at 30 degrees</LI><LI>Double transformants were then grown in SOB at 30 degrees until OD600 of approximately 0.5</LI><LI>50mM L-arabinose was added to the to induce the lambda-red system</LI><LI>After 20 mins of incubation with L-arabinose, cells were then made chemically competent</LI><LI>Oligonucleotides were added to 50μL of the experimental cells at a final concentration of 10μL. Negative control cells received no oligonucleotides.</LI><LI>Cells were recovered in SOC for 2 hours, then plated on LB + CM + SM + aTc and incubated overnight. The addition of aTc was for induction of Cas9.</LI><LI>The following day, significantly less growth should be observed on the negative control plate. Colonies on the experimental plate were screened for genomic DegP or TolA knockout by colony PCR.</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-main"><H4 class="panel-title">
						G-Block Cloning Protocols
				</H4></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse3">
						Restriction Enzyme Digestion
					</A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>NEB RE Buffer (2.1, 3.1 or cutsmart)</LI><LI>Restriction Enzymes e.g. EcoR1, Pst1</LI><LI>MiliQ Water</LI><LI>DNA Template</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Add into each tube: 250ng of DNA, 2.5μl Buffer, 0.5μl of each RE</LI><LI>Make up the total volume to 20μl with MiliQ Water.</LI><LI>Incubate at 37 degrees for 30 minutes.</LI><LI>Heat inactivate at 80 degrees for 20 minutes.</LI><LI>Perform Clean-up protocol. Then store in freezer.</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse4">
						E.Coli Transformation
					</A></H4></DIV><DIV id="collapse4" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>Competent E.coli cells</LI><LI>Purified plasmid or ligation reaction</LI><LI>SOC Medium</LI><LI>LB Agar plates (with appropriate antibiotics)</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Thaw competent cells on ice.</LI><LI>Add 30-50μl of competent cell to a chilled 1.5ml tube containing 1μl of purified plasmid (2μl if a ligation plasmid)</LI><LI>Leave on ice for 30 minutes.</LI><LI>Heat shock at 42 degrees for 90 seconds.</LI><LI>Leave on ice for 2 minutes.</LI><LI>Add 400μl of SOC</LI><LI>Shake at 37 degrees for 1 hour.</LI><LI>In biosaftey cabinet, plate the full volume onto an LB agar plate and spread</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse5">
						Ligation
					</A></H4></DIV><DIV id="collapse5" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>Digested plasmid backbone</LI><LI>Digested insert fragment</LI><LI>T4 DNA ligase buffer</LI><LI>T4 DNA Ligase</LI><LI>MiliQ Water</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Add the following into a 1.5ml tube: ~25ng of digested plasmid, equimolar equivalent digested fragment (&lt;3ul), 1ul 10x T4 ligase Buffer, 0.5μl T4 Ligase.</LI><LI>Make up the volume to 10μl with MiliQ Water.</LI><LI>Leave at room temperature for 30-60 minutes.</LI><LI>Heat inactivate at 80 degrees for 20 minutes.</LI><LI>Perform Clean-up protocol. Then store in freezer.</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse6">
						Colony PCR
					</A></H4></DIV><DIV id="collapse6" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>Lysis Buffer (TE + 0.1% Triton-X)</LI><LI>Bacterial Single Colonies</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Prepare 1.5ml tubes with 15μl of lysis buffer for each colony.</LI><LI>In biosaftey cabinet, gently touch a colony with a sterile tip and place the tip the the tube.</LI><LI>Leave for 2 minutes.</LI><LI>Remove tips, and boil cells at 90-100 degrees for 15 minutes.</LI><LI>Spin down the cell debris for 10 minutes at 14,000 g.</LI><LI>Use 1μl of the supernantant for the template of a 10μl PCR reaction.</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse7">
						Miniprep
					</A></H4></DIV><DIV id="collapse7" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>15ml Falcon Tube</LI><LI>Qiagen QIAprep Spin Miniprep Kit</LI><LI>1.5ml tubes</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Follow the QIAGEN QIAprep Spin Miniprep Kit Instructions</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse8">
						Gibson Assembly
					</A></H4></DIV><DIV id="collapse8" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>Microcentrifuge tubes</LI><LI>Gibson reaction mix</LI><LI>Parts for assembly</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Add 2.5μl Gibson reaction mix to a microcentrifuge tube</LI><LI>Then add parts for assembly in a 3:1 ratio of insert:vector</LI><LI>Incubate at 50 degrees celcius for 15 mins</LI><LI>Proceed to transformation</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-main"><H4 class="panel-title">
						Protein Assay Protocols
				</H4></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse9">
						Total Protein Extraction
					</A></H4></DIV><DIV id="collapse9" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>BugBuster</LI><LI>2-mercaptoethanol</LI><LI>LDS loading buffer</LI><LI>Heating block</LI><LI>Tabletop centrifuge (15000g)</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Grow cells to an OD of 0.8-1 (induced or non-induced)</LI><LI>Pellet 1ml of cells by centrifuging for 5 minutes</LI><LI>Remove supernatant</LI><LI>Add 100μL bugbuster, 40μL 4x LDS, 10μL 2-mercaptoethanol</LI><LI>Vortex thoroughly</LI><LI>Boil at 95C for 5 minutes</LI><LI>Centrifuge 15 minutes, 15000 rpm</LI><LI>Store at 4C until use</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse10">
						Western Blot
					</A></H4></DIV><DIV id="collapse10" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>PBS-T</LI><LI>Skim milk powder</LI><LI>SDS-PAGE Gel</LI><LI>Blot transfer device</LI><LI>Primary antibody with conjugated horseradish peroxidase</LI><LI>3, 3’, 5, 5’ tetramethylbenzidine (TMB)</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Separate protein samples on SDS-PAGE</LI><LI>Soak gel in Towbin buffer for 10 minutes</LI><LI>Place gel in transfer stack, and blot it to a membrane</LI><LI>Wash membrane in PBS-T</LI><LI>Soak membrane in 10% w/v skim milk in PBS-T, shaking, for one hour</LI><LI>Wash membrane in PBS-T, 20 minutes, shaking. Repeat once more.</LI><LI>Soak membrane in 10% skim milk in PBS-T, with added primary antibody (Sigma a7058)</LI><LI>Wash membrane in PBS-T, 20 minutes, shaking. Repeat once more.</LI><LI>Pour TMB onto the membrane. </LI><LI>After bands develop, photograph in a geldoc</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse11">
						Running and protein staining of SDS-Page
					</A></H4></DIV><DIV id="collapse11" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>SDS Buffer</LI><LI>Protein Sample</LI><LI>SDS-PAGE Gel</LI><LI>MES Buffer</LI><LI>Isopropanol (50% methanol, 10% acetic acid) fixing solution</LI><LI>MiliQ Water</LI><LI>Coomassie Blue</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Create a solution of 5:1 protein sample to SDS buffer ratio</LI><LI>Boil samples for 5 minutes</LI><LI>Centrifuge at maximum speed for 1 minute</LI><LI>Assemble an SDS-Page tank with a 4-12% gel and fill with MES buffer, remove the comb</LI><LI>Carefully deposit sample and ladder in the wells</LI><LI>Connect the tank to the power supply</LI><LI>Run gel at 100V for 60 minutes</LI><LI>Dissaemble the tank and break the gel case</LI><LI>Incubate the gel in isopropanol fixing solution for 10 minutes</LI><LI>Wash in MiliQ water for 10 minutes, then discard water and repeat 2 times</LI><LI>Stain gel with coomassie blue for 1 hour</LI><LI>Wash gel with MiliQ water until excess coomassie is removed</LI><LI>Gel is ready to be imaged or to be analysed with mass spectrometry</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse17">
						Sample Preparation for Mass Spectrometry
					</A></H4></DIV><DIV id="collapse17" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>0.1 M NH4CO3</LI><LI>0.1 M NH4CO3/50% acetonitrile</LI><LI>100% acetonitrile</LI><LI>SDS page gel that needs to be analyzed</LI><LI>10 mM DTT</LI><LI>55 mM iodoacetamide</LI><LI>MiliQ Water</LI><LI>Proteases and their corresponding buffers</LI><LI>0.1% trifluoroacetic acid (TFA)</LI><LI>0.1% TFA/60% acetonitrile</LI><LI>0.1% formic acid</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Cut bands out of the SDS page gel and slice them into 3 or 4 smaller pieces to increase the surface exposed to the reagents</LI><LI>Equilibrate in 0.1 M NH4CO3 for 20 mins at room temperature. Discard liquid afterwards.</LI><LI>Equilibrate in 0.1 M NH4CO3/50% acetonitrile at room temperature until the Coomassie or silver staining comes off. Discard liquid afterwards.</LI><LI>Dehydrate in 100% acetonitrile at room temperature until gel pieces are completely dry</LI><LI>Reduce with 10 mM DTT for 30 mins at 55 degrees Celsius. Discard liquid</LI><LI>Alkylate with 55 mM iodoacetamide for 45 minutes at room temperature, in the dark. Discard liquid</LI><LI>Rinse with MiliQ water</LI><LI>Wash with 0.1 M NH4CO3 for 20 mins at room temperature. Discard liquid</LI><LI>Dehydrate in 100% acetonitrile at room temperature until gel pieces are completely dry</LI><LI>Add sufficient protease in the appropriate buffer in a volume sufficient to cover the gel pieces. </LI><LI>Digest overnight at 37 degrees Celsius (trypsin) or 30 degrees Celsius (chymotrypsin). Avoid overdigesting depending on the protease. </LI><LI>Remove digest solution into a new silanised tube</LI><LI>Add 50μl 0.1% trifluoroacetic acid (TFA) for 30 mins at room temp. Remove liquid. </LI><LI>Add 50μl 0.1% TFA/60% acetonitrile for 30 mins at room temp. Discard liquid</LI><LI>Add 50μl 100% acetonitrile for 10 mins at room temp, removing liquid afterwards.</LI><LI>Vacuum dry sample and resuspend in 10-20μl of 0.1 formic acid</LI><LI>Sample is now ready to be analyzed by mass spectrometry</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-main"><H4 class="panel-title">
						OMV Purification
				</H4></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse12">
						OMV Harvesting
					</A></H4></DIV><DIV id="collapse12" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>Shaking Incubator</LI><LI>Beckman Avanti Swinging bucket centrifuge</LI><LI>100kDa centrifugal filter units (Millipore)</LI><LI>0.2μm  syringe filters (Millipore)</LI><LI>Sterile 20 ml syringes</LI><LI>PBS</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>100mL of LB with appropriate antibiotics (if required) was inoculated with each strain for analysis, and placed on a shaking incubator at 37 degrees, shaken at 200 rpm</LI><LI>After 3 hours of incubation, any necessary inducing agents (IPTG at 0.5 mM) were added to the cultures, which were then left on the shaking incubator at 37 degrees overnight</LI><LI>The next day, the OD600 of each culture was measured before OMV harvest</LI><LI>Each culture was then centrifuged for 20 mins at 6000g in a fixed angle rotor to pellet the biomass</LI><LI>The supernatant was recovered and then passed through a 0.2μm syringe-driven filter</LI><LI>40mL of the syringe-filtered supernatant was then loaded into a 100kDa centrifugal filter unit</LI><LI>The filter unit was then centrifuged at 3500g in a swinging bucket rotor for 15 minutes</LI><LI>The volume of the filtrate was recorded and then discarded</LI><LI>PBS of equal volume to the discarded filtrate was loaded into the filter unit on the retentate-side</LI><LI>The filter unit was then centrifuged at 3500g for 10 minutes in the same swinging bucket rotor</LI><LI>The filtrate was again discarded. The retentate was recovered by fitting a retentate collection cap to the filter unit and centrifuging with the filter in the opposite to normal orientation for 5 minutes at 1000g. This was recovered and the volume recorded.</LI><LI>400μL of PBS was loaded directly onto each of the 2 filter units of the filter unit</LI><LI>The sample above was combined with the retentate collected in step 11</LI><LI>OMVs were stored at 4 degrees until analysis</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-main"><H4 class="panel-title">
						General Protocols
				</H4></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse13">
						Making E. coli Chemically Competent
					</A></H4></DIV><DIV id="collapse13" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>E. coli strain to make competent</LI><LI>LB media</LI><LI>100mM CaCl2 solution</LI><LI>SOC media</LI><LI>Agar plates with LB + appropriate antibiotics</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Grow the relevant strain of E. coli in LB overnight (+ antibiotics, if necessary)</LI><LI>The following day, inoculate 5mL of LB + appropriate antibiotics with the overnight culture (1/100 dilution)</LI><LI>After approximately 3 hours, pellet the cells and discard the supernatant</LI><LI>Resuspend the pellet in 20μL of sterile water + 100μL 100mM CaCl2 solution</LI><LI>Leave on ice for 60 minutes</LI><LI>For each 40μL of competent cells, add 1μL of plasmid (per plasmid being transformed)</LI><LI>Leave on ice for 30 minutes</LI><LI>Heat shock at 42 degrees for 90 seconds</LI><LI>Leave on ice for 2 minutes</LI><LI>Add 600μL of SOC media</LI><LI>Recover for one hour at 37 degrees</LI><LI>Pellet the cells, and remove most of the supernatant (leave 50-100L)</LI><LI>Resuspend the pellet in the remaining media, and plate on LB + appropriate antibiotics</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse14">
						Preparation of Lysogeny broth (LB)
					</A></H4></DIV><DIV id="collapse14" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>5 grams Yeast Extract</LI><LI>10 grams peptone</LI><LI>10 grams NaCl (anhydrous)</LI><LI>1 liters of RO Water</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Weigh out components and add to a 1 liter schott bottle</LI><LI>Make up to 1 liter with RO Water</LI><LI>Mix with a magnetic mixer until fully dissolved</LI><LI>Seal with autoclave tape and autoclave LB for use</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse15">
						Preparation of Agar Plates
					</A></H4></DIV><DIV id="collapse15" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>LB agar for E.Coli</LI><LI>Stock concentration of relavant antibiotic</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Melt solid agar in microwave and allow to cool.</LI><LI>In the biosaftey cabinet add 1μl of stock concentration antibiotic for every 1ml of agar.</LI><LI>Pour agar into sterile petri dishes until 1/2 full. For a standard 10 cm dish, this will be approximately 20ml.</LI><LI>Cover plates and allow to set at room temperature.</LI><LI>Seal with parafilm, and place in a 4 degrees fridge. An extra covering of foil helps prolong self life.</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse16">
						SOC Medium Preparation
					</A></H4></DIV><DIV id="collapse16" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="col-sm-6"><H3>Materials</H3><UL><LI>Anhydrous MgCl2</LI><LI>Glucose Powder</LI><LI>Tryptone</LI><LI>Yeast Extract</LI><LI>NaCl</LI><LI>KCl</LI><LI>RO Water</LI></UL></DIV><DIV class="col-sm-6"><H3>Method</H3><OL><LI>Measure 0.476 grams of anhydrous MgCl2 and add RO water to a final volume of 10ml.</LI><LI>Filter Sterilize.</LI><LI>Measure 1.8 grams of glucose and add RO water to a final volume of 10ml.</LI><LI>Filter Sterilize.</LI><LI>Add into 96ml of RO water: 2 grams tryptone, 0.5 grams Yeast extract, 58.5 mg NaCl, 18.6mg KCl.</LI><LI>Mic til dissolved with magnetic stirrer.</LI><LI>Autoclave</LI><LI>Aseptically add 2ml of 0.5M Magnesium Chloride and 2ml of 1M Glucose solution to the 96ml of autoclaved solution.</LI></OL></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="about" class="container content-section text-center"><DIV class="row"><DIV class="col-lg-10 col-lg-offset-1"><H2 style="font-family: Slim-Joe;">Project Development</H2><P> As our iGEM project developed, so too did the methods we used. Our project began with a journey through the literature on OMVs, to see what had already been done, and to try to spot the gaps which our project could plug. From this, we identified a range of proteins that could either be overexpressed or knocked-out to induce hypervesiculation. Our aim was to compare the size, number, and stability of OMVs induced by these mutations.</P><P> Proteins that could be overexpressed we synthesised from IDT, and cloned into pSB1C3 using 3A assemblly, as for such small parts we felt this was the fastest approach (and free!); proteins that could be knocked-out we designed guide RNAs that fit could be cloned into the noSCAR CRISPR Kit from AddGene, with support from a recent Nature Methods paper [1]. Each of these approaches, however, proved troublesome. </P><P> Regarding overexpression parts, a Western Blot revealed that they weren’t expressing from pSB1C3, and so had to clone into the Duet Expression Vectors using Gibson assembly before reliable expression was achieved. The NoSCAR CRISPR system, likewise, seemed to be going well, but we wound up having to pull the plug as deadlines fast approached, where we needed strains to characterise; instead, we ordered knockouts from the Yale Coli Genetic Stock Centre, recommended to us by Dr Jai Tree, a lecturer at UNSW.</P><P> Now that we had strains, it was time to begin characterisation. Our first avenue was to try the Zeiss Airyscan LSM 880 microscope; it was only late in the project that we realised that this didn’t have the resolution to see nanoscale-OMVs. This microscope still had use in seeing our localising-fluorescence parts, though. Other papers that characterised OMVs seemed to use Electron Microscopes [2], but we didn’t have access to these. Instead, we contacted biotech companies and found out about the NanoSight, which we wound up successfully using in the closing days of the project. </P><P> In sum, like all iGEM teams, we hit numerous roadblocks; it was through contacts and literature searches that we overcame these, to give us the data we have today!</P></DIV></DIV></SECTION><SECTION id="project" class="container intro-section text-center"><DIV class="row"><DIV class="col-lg-10 col-lg-offset-1"><H2 style="font-family: Slim-Joe;">References</H2><OL><LI>Reisch, C.R. and Prather, K.L., 2015. The no-SCAR (Scarless Cas9 Assisted Recombineering) system for genome editing in Escherichia coli. Scientific reports, 5.</LI><LI>Henry, T., Pommier, S., Journet, L., Bernadac, A., Gorvel, J.P. and Lloubès, R., 2004. Improved methods for producing outer membrane vesicles in Gram-negative bacteria. Research in microbiology, 155(6), pp.437-446.</LI></OL></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>