<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Todai-Tokyo_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Todai-Tokyo/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Todai-Tokyo/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Todai-Tokyo/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Todai-Tokyo/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Todai-Tokyo/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Todai-Tokyo/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><TABLE style="color:#008fff;background-color:#ffd200;" cellpadding="3" cellspacing="2" width="69%" align="left"><TBODY><TR><TH align="center"><A href="/Team:Todai-Tokyo" title="Team:Todai-Tokyo">Home</A></TH><TH align="center"><A href="/Team:Todai-Tokyo/Team" title="Team:Todai-Tokyo/Team">The Team</A></TH><TH align="center"><STRONG class="selflink">The Project</STRONG></TH><TH align="center"><A href="/Team:Todai-Tokyo/Parts" title="Team:Todai-Tokyo/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><A href="/Team:Todai-Tokyo/Modeling" title="Team:Todai-Tokyo/Modeling">Modeling</A></TH><TH align="center"><A href="/Team:Todai-Tokyo/Notebook" title="Team:Todai-Tokyo/Notebook">Notebook</A></TH><TH align="center"><A href="/Team:Todai-Tokyo/Protocols" title="Team:Todai-Tokyo/Protocols">Protocols</A></TH><TH align="center"><A href="/Team:Todai-Tokyo/Ethics" title="Team:Todai-Tokyo/Ethics">Ethics</A></TH></TR></TBODY></TABLE><P><TITLE>the notebook</TITLE></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Aim"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Aim</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Overall_projects"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Overall projects</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Details_of_Each_Projects"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Details of Each Projects</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Bioclock"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Bioclock</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Tobacco"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Tobacco</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Bread"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Bread</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Arterisclerosis"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Arterisclerosis</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Aim"><B>Aim</B></SPAN></H2><P><B>To Prevent lifestyle diseases using synthetic living systems</B></P><P>Lifestyle diseases, diseases caused by unhealthy living habits, comprise one of the major problems in modern society, especially as they may lead to fatal heart problems or even cancer. However, preventing or curing these diseases is presently of extreme difficulty.
</P><P>
Our team, Todai-Tokyo, has been tackling treatment of lifestyle diseases such as hypercholesterolemia, diabetes, circadian rhythm dysfunction, and bad smoking habits by using synthetic living systems, utilizing their ability to incorporate complex logic functions and dispensability of external control once in operation. To do this, we aim to create the following: cells that ingest cholesterol to decrease blood cholesterol levels, healthy low calorie breads, a system in which periodic gene expression is controlled, and bacteria that encourage smokers to quit smoking, respectively.
</P><P>
By applying similar synthetic biology methodologies to these, prevention of numerous lifestyle-related diseases may become reality, serving as a first step towards their eradication.
</P><H2><SPAN class="mw-headline" id="Overall_projects"><B>Overall projects</B></SPAN></H2><P>We are planing 4 projects.
1.bioclock
2.tobacco
3.Bread
4.Arterisclerosis
1.We will produce E.coli that can let me know what time it is now by expressing Fluorescent Protein.
2.If you broken your promise and smoked, your mouth would begin to smell.We are growing E&gt;coli or bacterias which express the enzyme that reacts Nicotine and catalyzes a smelling substract.
3.We are making food which has additional nutritions.Without adding sugar,Bread becomes sweet by enzymes of bread yeast. Food has more vitamins. etc.
4.Arteriosclerosis is caused by LDL cholesterols. So, we are growing S.cerevisiae which express LDL cholesterol receptor and inhibits the absorption of LDLcholesterols in the blood vassel.</P><H2><SPAN class="mw-headline" id="Details_of_Each_Projects"><B>Details of Each Projects</B></SPAN></H2><H3><SPAN class="mw-headline" id="Bioclock"> Bioclock </SPAN></H3><P><B>Synthetic Oscillatory Gene Circuit in E.coli Cells</B></P><P>We are working on an oscillatory gene circuit in each individual E.coli cell according to the circuit design which was reported by J. Sticker and his colleagues in 2008. Also, we are making the switch to turn on and off the oscillator. The oscillator sould be turned on after UV exposure, and turned off automatically after a few hours.
</P><P>1.	Activating the Oscillator by UV Radiation
</P><P>The switch consists of two genes derived from bacteriophage lambda, cI and cro. The expression of two genes is mutually exclusive. The initial state is the cI expression, which means the oscillator is turned off. When UV light is exposed, cI is cleaved by RecA, one of the SOS genes of E.coli, and cro starts its own expression. As a result, the oscillator is turned on. The reason why these two gene can control the oscillation is that cI enhances lacI which is a component of the oscillator while cro doesn’t, as explained just now.
</P><P>2.	Oscillatory Expression. (J. Sticker et al, Nature 2008).
</P><P>The oscillator consists of one negative feedback roop by lacI gene and one positive feedback roop by araC gene. In this circuit, araC, lacI, and GFP are under control of the same promoter (denoted as P_araC/lacI) which is suppressed by lacI and enhanced by araC. The oscillatory period is from 10 min to 40 min, and dependent on the concentration of IPTG, arabinose, and temperature.
</P><P>3.	Silencing the Oscillator
</P><P>A gene called cIII enhances cI and turns off the oscillator. CIII is also controlled by P_araC/lacI.
Here is the mechanism: When cro is expressed (which means oscillator is on), still other gene cII is expressed together. cII is capable of enhancing cI expression, but is unstable under normal condition, because it is attacked by Hfl protease of E.coli. cIII prevent cII degradation by competitive inhibition of Hfl. Consequently, cII can induce cI expression only when cIII exists in the concentration high enough to inhibit Hfl activity. So, when the cIII concentration hits the critical amount, the oscillator is turned off.
</P><P><B>Experimental procedure</B></P><P>Oscillator (Sticker,J et al. <I>Nature</I>, 2009 <A href="http://www.nature.com/nature/journal/v456/n7221/abs/nature07389.html" class="external autonumber" rel="nofollow">[1]</A>)
</P><P>The ParaC/lacI promoter is enhanced by araC and repressed by lacI. (Lutz,R et al. <I>Nucleic Acid Research</I>, 1997 <A href="http://nar.oxfordjournals.org/cgi/content/abstract/25/6/1203" class="external autonumber" rel="nofollow">[2]</A>)
</P><P>
The promoter, RBS, NcoI and XhoI restriction site, and double terminator was first synthesized.
</P><P>araC, lacI and GFP was then each amplified with degradation tag. Using homologous recombination, these three genes were inserted into the synthesized DNA, between the NcoI and XhoI site.
</P><P>araC and GFP were ligated in tandem, inserted into a plasmid vector which has ColE1 origin.　
lacI were inserted into a plasmid vector with p15A origin.
</P><P>(* In order to make the genetic oscillatory circuit, strict control of the total number of the mRNA molecules transcribed is required.  The replication origin ColE1 has medium copy number, while p15A has low. We used these origins following the instruction of the paper of J.Sticker.)
</P><P>UV induced Switch
</P><P>All the genes described above, except RecA (which is endogenous bacterial gene) was cloned from lambda phage genome. they are all sub-cloned into plasmid of the p15A origin. 
</P><P>araC+lacI double knockout bacteria construction
</P><P>As mentioned above, We made bacterial strain that has no functional araC nor lacI proteins, for the strict control of protein expression is crutial in this gene circuit.
</P><H3><SPAN class="mw-headline" id="Tobacco"> Tobacco </SPAN></H3><P><B>Effective Way to Quit Smoking Using Smelling Yeast Cells</B></P><P><B>ABSTRACT</B>
We will make bacteria to help those who want to quit smoking to attain their goal. 
In order to achieve this, we are constructing the bacteria that begin to smell when exposed to smoke. 
The way to use these bacteria is very simple. 
1. At first, we get those who have to break their habit of smoking to take these bacteria. 
2. And then, if they break the ban and smoke, they will be worried by strong smell like a cheese or dirty socks. 
3. Thereafter they will never smoke anymore.</P><P><B>CONSTRUCT</B></P><P><B>AhR,   ARNT coding</B>
AhR (The Aryl hydrocarbon receptor) is a member of the family of basic-helix-loop-helix transcription factors. 
AhR is a cytosolic transcription factor that is normally inactive, bound to several co-chaperones. 
Upon ligand binding to chemicals such as 2,3,7,8-tetrachlorodibenzo-p-dioxin (TCDD), the chaperones dissociate resulting in AhR translocating into the nucleus and dimerizing with ARNT (AhR nuclear translocator), leading to changes in gene transcription.<B>XRE (xenobiotic response element)</B>
The classical recognition motif of the AhR/ARNT complex, referred to XRE, contains the core sequence 5'-GCGTG-3' within the consensus sequence 5'-T/GNGCGTGA/CG/CA-3' in the promoter region of AhR responsive genes.
The AhR/ARNT entity directly binds the XRE core sequence in a manner such that ARNT binds to 5'-GTG-3' and AhR binding 5'-TC/TGC-3'. <B>yqiT</B>
YqiT codes Leucine dehydrogenase descended from Bacillus subtilis. 
The Leucine dehydrogenase makes isovaleric acid from Leucine. <B># Isovaleric acid</B>
Isovaleric acid is a natural fatty acid found in a wide variety of plants and essential oils. 
Isovaleric acid has a strong pungent cheesy or sweaty smell. It is a major component of the cause of unpleasant foot odor.
</P><P><B>MECHANISM</B>1. First, dioxin in smoke reacts the activator protein Ahr. 
2. Then, Ahr and protein ARNT binds the enhancer XRE, and yqiT is expressed. </P><UL><LI>YqiT codes Leucine dehydrogenase descended from Bacillus subtilis. </LI><LI>The Leucine dehydrogenase makes isovaleric acid from Leucine. </LI><LI>Isovaleric acid is a major component of the cause of unpleasant foot smell.</LI></UL><H3><SPAN class="mw-headline" id="Bread"> Bread </SPAN></H3><P><B>We make yeast that can make bread without sugar.</B>
Sugar is essential for bread making but may cause diabetes; therefore bread made from our yeast is healthier.
</P><P>In diabetes, the body either doesn't respond properly to its own insulin, doesn't make enough insulin, or both. 
This causes glucose to accumulate in the blood, often leading to various complications. Too much intake of sugar is one of the major causes of this disease. We focused on lowering sugar ingestion and planned to make sweet bread without adding any sugar. Because bread contains sugar and we take it almost everyday. To make sweet bread, we usually have to add sugar. It is necessary to make bread sweet but also as a source of the fermentation process. In detail yeast digests sugar into glucose and uses it in the fermentation. Without sugar, bread can’t be sweet and in the first place dough can’t expand.
</P><P>We think of substituting other things for sugar in both processes. First we make baker’s yeast that uses starch instead of sugar for the fermentation. It is realized by transforming GLU1, a gene of glucoamylase, into yeast. This enzyme resolves starch into glucose. Starch is contained in flour and it’s a main ingredient of bread, so this yeast gets able to perform fermentation without sugar.
</P><P>Then as an origin of sweetness we direct our attention to two sugar, mannitol and psicose.
</P><UL><LI>mannitol
</LI></UL><P>First plan is changing glucose to mannitol.It is an alcohol sugar that is as sweet as glucose, lower in energy and slowly absorbed from the intestinal tract. So products sweetened with mannitol in place of sugar may be useful in providing a wider variety of reduced calorie and sugar-free choices to people with diabetes.  We’ll introduce a gene of an enzyme that converts glucose into mannitol to the yeasts. Using the rest of glucose after the fermentation, they produce mannitol.
</P><UL><LI>psicose
</LI></UL><P>The other plan we constructed is converting glucose to psicose. It's an ultralow-energy monosaccharide sugar which yields only 0.3% the metabolic energy of the equivalent amount of sucrose. As in mannitol plan, we make yeast produce psicose from glucose using two enzymes.
</P><P>
By using these yeasts, we can offer you the way to lead healthier life with sweet, tasty bread.
</P><P><B>Experimental Procedure</B></P><UL><LI>mannitol
</LI></UL><P>
Clone mtlD (mannitol synthase) from E.coli and insert it in the yeast chromosome by homologous recombination.
Replace gpd1 gene by mtlD and gpd2 gene by glu1.
Yeast responds to osmotic stress by increasing the synthesis and accumulation of glycerol. Gpd encodes Glycerol-3Phosphate Dehydrogenase, which is essential for forming glycerol. 
Mannitol fuctions as a osmolytes instead of glycerol. 
Kan4 and His3 markers are used to confirm recombination.</P><UL><LI>psicose
</LI></UL><P>
Clone xylose isomerase gene and D-tagatose 3-epimerase gene from Mesorhizobium loti.
Replace gpd1 gene by D-tagatose 3-epimerase gene and gpd2 gene by glu1. Transform a plasmid coding xylose isomerase gene.
Psicose protects yeast against osmotic stress as mannitol.</P><H3><SPAN class="mw-headline" id="Arterisclerosis"> Arterisclerosis </SPAN></H3><P><B>~Reducing the Risk of Arteriosclerosis with &quot;Cholesterol-sucking Yeast&quot;~</B></P><P><B>ABSTRACT</B>
Arteriosclerosis is caused by LDL; therefore we decided to reduce LDL concentration in a blood stream.
We try to reduce LDL by introducing LDL receptor to yeast. LDL receptor catches LDL therefore reduces LDL concentration.</P><P><B>CONSTRUCT</B></P><P><B>Gal1</B>
Gal1 is a promotor that is triggered by galactose; works well under galactose-rich environment.</P><P><B>LDLR (Low Density Lipoprotein Receptor)</B>
LDLR is a receptor for LDL descended from human beings. 
LDLR bonded with LDL causes endocytosis.</P><P><B># LDL(Low Density Lipoprotein)</B>
LDL carries cholesterol from the liver to other tissues.
LDL contains a lot of cholesterol molecules therefore sometimes called as &quot;bad cholesterol&quot;.</P><P><B>MECHANISM</B>
1. Yeast with LDLR catches LDL in a blood stream 
2. Leads to endocytosis, taking in LDL with LDLR
3. LDL concentration in the blood stream decreases
4. Prevents arteriosclerosis</P><H2><SPAN class="mw-headline" id="Results"><B>Results</B></SPAN></H2><P><B>Tobacco</B>
We have made E.coli which has &quot;Constitutive promoter (strong) - RBS - yqiT - double terminator&quot; in its plasmid.
and
Associate Prof. Kawanishi offered us the yeast, whose chromosome contains AhR and ARNT.
Next, We will have made yeast which has Ahr and ARNT in its chromosome, and &quot;XRE - Gal1p - yqiT&quot; in its plasmid by 21 October.<B>We had a questionnaire that asked the participants what they thought of our &quot;TOBACCO project&quot;.</B> The participants are the students and graduates of the University of Tokyo&quot;.
Questions
1. Do you smoke more than once a week?
2. Have you ever thought of quitting smoking? (only for smokers)
3. Is there any smoker, whom you want to quit smoking, around you?
4. We are trying to make yeast that emits such a strong smell when you smoke for people who cannot quit smoking. The smell will traumatize you to smoke, and therefore you should be able to quit smoking!
If such yeast are created, do you think it will help smokers to quit smoking?</P><P>Here are the answers;</P><P>According to the data above, 
more than 70% of smokers (13 out of 18) wants to quit smoking, and 
approximately 60% (99 out of 169) people are troubled by smokers... 
Yes, there are many people who wants to quit smoking!! </P><P>However, the rate of success to quit smoking is relatively low... doctors say that only 20~30% people are able to quit smoking. 
On the other hand, 60% people believe our project would successfully help smoking cessation! <B>We measured the intensity of smell of cleansed air, air, air in the test tube in which we were shaking culture of E.coli “negative control”, and air in the test tube in which we were shaking culture of E.coli which has plasmid containing “promoter + RBS + yqiT + double terminator”</B>
Data
Time for shake culture (each of sample) 15 hours
Negative control has plasmid containing “yqiT + double terminator” (Not including promoter)
Cleansed air is cleansed with active carbon filter.
Equipment</P><UL><LI>Trade name: ODOR CONCENTRATION METER</LI><LI>Model: XP-329</LI><LI>Serial No. 311059</LI><LI>Manuf. NEW COSMOS ELECTRIC CO,LTD.</LI><LI>Setting: Peak hold</LI></UL><P><B>Bread</B></P><UL><LI>mtlD gene from E.coli made a debut as iGEM part!</LI><LI>We made a gene cassette containing Kan4 marker and mtlD for recombination with gpd1 gene of S.cerevisiae.</LI><LI>We're making a gene cassette of His3 marker and Glu1 and have finished PCR of marker plasmid for amplifying of  GFPKan4MX6 gene.</LI></UL><P><B>LDLR</B></P><UL><LI>We have made Gal1 promoter +LDLR ,using iGEM part, plate1-7D.</LI><LI>Now we are replasing  Gal1 promoter +LDLR in YCplacIII.</LI><LI>We will check whether it works in S.cerevisiae.</LI></UL><P><B>Bioclock</B></P><UL><LI>We made these constructs.</LI></UL><P>-(pUC57)-placI/araC-araC+ssra tag-double terminator-
-(pUC57)-placI/araC-GFP+ssra tag-double terminator-
- (pUC57)-placI/araC-lacI+ssra tag-double terminator- 
-(p15A ori)-cI~cII-
-(pUC57)-placI/araC-cIII-double terminator-
We are ligating the upper ones in order to make what we wrote on notebook page<A href="https://2009.igem.org/Team:Todai-Tokyo/Notebook/bioclock" class="external autonumber" rel="nofollow">[3]</A>.</P><UL><LI>We analyzed whether this genetic circuit really work or not using mathematical model(see <A href="/Team:Todai-Tokyo/Modeling" title="Team:Todai-Tokyo/Modeling">Modeling</A>).
</LI></UL><P>These are  data we've got so far... BUT! WE STILL ARE COLLECTING DATA!!
Please expect our results at Jamboree.</P><TABLE style="color:#008fff;background-color:#ffd200;" cellpadding="3" cellspacing="2" width="69%" align="left"><TBODY><TR><TH align="center"><A href="/Team:Todai-Tokyo" title="Team:Todai-Tokyo">Home</A></TH><TH align="center"><A href="/Team:Todai-Tokyo/Team" title="Team:Todai-Tokyo/Team">The Team</A></TH><TH align="center"><STRONG class="selflink">The Project</STRONG></TH><TH align="center"><A href="/Team:Todai-Tokyo/Parts" title="Team:Todai-Tokyo/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><A href="/Team:Todai-Tokyo/Modeling" title="Team:Todai-Tokyo/Modeling">Modeling</A></TH><TH align="center"><A href="/Team:Todai-Tokyo/Notebook" title="Team:Todai-Tokyo/Notebook">Notebook</A></TH><TH align="center"><A href="/Team:Todai-Tokyo/Protocols" title="Team:Todai-Tokyo/Protocols">Protocols</A></TH><TH align="center"><A href="/Team:Todai-Tokyo/Ethics" title="Team:Todai-Tokyo/Ethics">Ethics</A></TH></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Todai-Tokyo/Project">http://2009.igem.org/Team:Todai-Tokyo/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Todai-Tokyo/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Todai-Tokyo/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Todai-Tokyo/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Todai-Tokyo/Project&amp;oldid=168539" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>