"
Page
Discussion
View source
History
teams
Log in
 
Team:Todai-Tokyo/Notebook/isoleucine
From 2009.igem.org
Home
The Team
The Project
Parts Submitted to the Registry
Modeling
Notebook
Protocols
Ethics
the notebook
Contents
1Plan
2June
2.16/5
2.26/6
2.36/7
2.46/14
2.56/21
3July
3.17/5
3.27/19
3.37/20
3.47/27
3.57/30~8/2
4August
4.18/3
4.28/7~9
4.38/12~14
4.48/15,16
4.58/19
5September
5.19/15
5.29/16
5.39/17
5.49/19
5.59/20
5.69/21
5.79/22
5.89/23
5.99/24
5.109/25
5.119/26
5.129/27
5.139/29
5.149/30
6October
6.110/2
6.210/7
6.310/9
6.410/10
6.510/11
6.610/12
6.710/13
6.810/14
6.910/15
6.1010/16
6.1110/18
6.1210/19
6.1310/20~
6.14Result
Plan
Aim: Create bacteria that produce a noxious odour when induced to do so
Methods:
Clone the yqiT gene from Bacillus subtilis into a biobrick vector.
Make constructs that express this gene constitutively and under the control of an inducible promoter.
June
First, we haven't decided what promoter to use.
While looking for a good promoter, we were cloning YqiT and useful parts.
6/5
Constructs to be created:
yqiT
ptetR-RBS-yqiT-dterm
pAraC-RBS-yqiT-dterm
pLacI-RBS-yqiT-dterm
Obtaining DNA:
Resuspended DNA in the following wells with 10ul water:
Plate 1 1D
AraC regulated promoter Plate 1 12E
LacI regulated promoter
Plate 1 1H
Strong RBS
Plate 1 13B
TetR repressed POPS generator
Plate 2 24C
Double terminator
Transformed 1ul of each of the above into DH5a competent cells:
Transformation
Mix 1ul of DNA with 100ul of competent cells on ice.
Leave on ice for 30 minutes.
Heat shock at 42°Cfor 45 seconds.
Leave on ice for 2 minutes.
Add 500ul of LB and incubate at 37°C for 1 hour.
Plate on LB-ampicillin plates.
6/6
No colonies grew from the 5 transformations of 6/5.
6/7
Creating a biobrick part out of yqiT
Strategy: PCR out the yqiT gene from the Bacillus subtilis genome using primers to attach the biobrick preffix/suffix and clone this into a biobrick vector. The vector utilized is the GFP generator which houses a sufficient-sized insert:
Plate 1 16E (from 2007 iGEM distribution provided by Chiba University)
GFP generator
Overview:
PCR yqiT gene from genome
purify DNA from the PCR product
digest GFP generator and purified PCR product by XbaI/PstI
gel-purify the digested vector (GFP generator without the insert i.e. empty) and the yqiT gene
ligate the above 2 fragments of DNA together
transform into E. coli and select for ampicillin resistance
check for transformation success using colony PCR by yqiT primers
PCR of yqiT gene
The Bacillus subtilis genome was provided by Bacillus subtilis 168
The following primers were used to amplify an approx. 1500bp fragment from the Bacillus subtilis genome.
yqiT EX: ccggaattctctagaatggaactttttaaatatatggaacgatacg(bold text is a sequence binding yqiT gene)
yqiT SP: ctgcagcggccgctactagtattagcgacgacttaaaatatgttgg(bold text is a sequence binding yqiT gene)
PCR protocol
The following was mixed in a PCR tube:
1ul 20uM yqiT EX primer
1ul 20uM yqiT SP primer
2ul 10X Pfu Ultra buffer
1.6ul dNTP mix
0.15ul Bacillus subtilis genome
0.5ul Pfu Ultra enzyme
13.85ul MilliQ
Performed PCR using the following program:
1. 95°C 2 minutes
2. 95°C 30 seconds
3. 50°C 30 seconds
4. 72°C 40 seconds
5. Repeat 2-4 23 times
6. 25°C forever
Purified PCR product using the Promega PCR purification kit.
Ran on gel to visualize bands:
PCR unsuccessful: will digest the GFP generator but cannot perform the ligation without the PCR product.
Digestion
Digested purified PCR product and the GFP generator both with XbaI/PstI:
yqiT
3ul PCR product
1ul High buffer
0.5ul XbaI
0.5ul PstI
5ul MilliQ
GFP generator
6ul DNA
2ul High buffer
1ul XbaI
1ul PstI
10ul MilliQ
Incubated mixtures at 37°C for 1 hour.
Ran on gel to gel purify:
Lanes 1, 12:marker
Lanes 2, 5, 9:GFP
Lanes 3, 6, 10:yqiT
Cut out the second band from the bottom and dissolved the gel using the Promega gel purification kit to extract DNA from.
This DNA is stored for later usage as the GFP generator X/P vector fragment.
6/14
Creating a biobrick part out of yqiT
Strategy: PCR out the yqiT gene from the Bacillus subtilis genome using primers to attach the biobrick preffix/suffix and clone this into a biobrick vector. The vector utilized is the GFP generator which houses a sufficient-sized insert:
Plate 1 1D (from 2009 iGEM distribution provided by Tokyo University) promoter (lacI regulated)
Overview:
PCR yqiT gene from genome
gel-purify DNA from the PCR product
digest lacI regulated promoter and purified PCR product by XbaI/PstI
gel-purify the digested vector (lacI regulated promoter without the insert i.e. empty) and the yqiT gene
ligate the above 2 fragments of DNA together
transform into E. coli and select for ampicillin resistance
check for transformation success using colony PCR by yqiT primers
PCR of yqiT gene The Bacillus subtilis genome was provided by Bacillus subtilis 168
The following primers were used to amplify an approx. 1095bp fragment from the Bacillus subtilis genome.
yqiT EX: ccggaattctctagaatggaactttttaaatatatggaacgatacg(bold text is a sequence binding yqiT gene)
yqiT SP: ctgcagcggccgctactagtattagcgacgacttaaaatatgttgg(bold text is a sequence binding yqiT gene)
PCR protocol
The following was mixed in a PCR tube:
1ul 20uM yqiT EX primer
1ul 20uM yqiT SP primer
2ul 10X Pfu Ultra buffer
1.6ul dNTP mix
0.15ul Bacillus subtilis genome
0.5ul Pfu Ultra enzyme
13.75ul MilliQ
Performed PCR using the following program:
1. 95°C 2 minutes
2. 95°C 30 seconds
3. 50°C 30 seconds
4. 72°C 40 seconds
5. Repeat 2-4 29 times
6. 25°C forever
Purified PCR product using the Promega PCR purification kit.
Ran on gel to gel purify:
Lanes 1:marker 6ul
Lanes 2:yqiT PCR product 10ul + 10xLoading Dye 2ul
PCR successful
Ligation
Ligated gel purified yqiT and vector with TaKaRa Solutions 1:
lacI regulated promoter (vector) 2ul
yqiT 8ul
TaKaRa Solutions 1 6ul
Incubated mixtures at 37°C for 10 minutes.
Transformation
6/21
Creating a biobrick part out of yqiT
Strategy: PCR out the yqiT gene from the Bacillus subtilis genome using primers to attach the biobrick preffix/suffix and clone this into a biobrick vector. The vector utilized is the GFP generator which houses a sufficient-sized insert:
Plate 1 1D (from 2009 iGEM distribution provided by Tokyo University)
promoter (lacI regulated)
Overview:
PCR yqiT gene from genome
gel-purify DNA from the PCR product
digest lacI regulated promoter and purified PCR product by XbaI/PstI
gel-purify the digested vector (lacI regulated promoter without the insert i.e. empty) and the yqiT gene
ligate the above 2 fragments of DNA together
transform into E. coli and select for ampicillin resistance
check for transformation success using colony PCR by yqiT primers
PCR of yqiT gene
The Bacillus subtilis genome was provided by Bacillus subtilis 168
The following primers were used to amplify an approx. 1095bp fragment from the Bacillus subtilis genome.
yqiT EX: ccggaattctctagaatggaactttttaaatatatggaacgatacg(bold text is a sequence binding yqiT gene)
yqiT SP: ctgcagcggccgctactagtattagcgacgacttaaaatatgttgg(bold text is a sequence binding yqiT gene)
PCR protocol
The following was mixed in a PCR tube:
1ul 20uM yqiT EX primer
1ul 20uM yqiT SP primer
2ul 10X Pfu Ultra buffer
1.6ul dNTP mix
0.15ul Bacillus subtilis genome
0.5ul Pfu Ultra enzyme
13.75ul MilliQ
Performed PCR using the following program:
1. 95°C 2 minutes
2. 95°C 30 seconds
3. 50°C 30 seconds
4. 72°C 40 seconds
5. Repeat 2-4 29 times
6. 25°C forever
Purified PCR product using the Promega PCR purification kit.
Ran on gel to gel purify:
Lanes 6:marker 6ul
Lanes 2, 4:yqiT PCR product 10ul + 10xLoading Dye 2ul
PCR successful
Digestion
Digested purified PCR product and the lacI regulated promoter both with XbaI/PstI:
yqiT
3ul PCR product
1ul High buffer
0.5ul XbaI
0.5ul PstI
5ul MilliQ
lacI regulated promoter
4ul DNA
4ul High buffer
2ul XbaI
2ul PstI
28ul MilliQ
Incubated mixtures at 37°C for 1 hour.
Ran on gel to gel purify:
Lanes 3:marker 6ul
Lanes 2, 5:lacI regulated promoter 40ul + 10xLoading Dye 8ul
Lanes 1, 4:yqiT 10ul + 10xLoading Dye 2ul
Cut out the band and dissolved the gel using the Promega gel purification kit to extract DNA from.
Ligation
Ligated gel purified yqiT and vector with TaKaRa Solutions 1:
lacI regulated promoter (vector) 0.5ul
yqiT 5.5ul
TaKaRa Solutions 1 4ul
Incubated mixtures at 37°C for 10 minutes.
Transformation
July
We tried to read the sequence of YqiT for the first time.
But the trial ended to failure....
7/5
TA cloning of YqiT
PCR of YqiT
0.25ul Takara ExTaq
5ul 10xExTaq Buffer
4ul dNTPmix
0.5ul Bacillus subtilis genome
0.5ul 5'primer(It is the same sequence as that on 6/21)
0.5ul 3'primer(It is the same sequence as that on 6/21)
up to 50ul MilliQ
Performed PCR using the following program: 1. 94°C 2 minutes
2. 94°C 30 seconds
3. 52°C 80 seconds
4. 72°C 60 seconds
5. Repeat 2-4 29 times
6. 25°C forever
lane1:marker
lane2~5:YqiT
PCR successful!
But we couldn't do ligation because TA cloning kit was not left.
We purified PCR products and put them in -20°C.
7/19
YqiT colony PCR
put a small amount of single colony into each tube with 5ul MilliQ water
|
v
95°C 5min
|
v
PCR reaction
1ul 10×buffer
0.8ul 2.5mMdNTP
0.08ul Ex-Taq
0.1ul 5’-primer
0.1ul 3’-primer
2.92ul MilliQ water
|
v
added the PCR reaction to each tube.
|
v
Performed PCR using the following program:
1. 95°C 2min
2. 95°C 30sec
3. 52°C 30sec
4. 72.5°C 80sec
5. repeat 2-4 29times
6. 25°C forever
PCR successful!
added a small amount of colony number 2 and 3 to each test tube with 4ml LB broth.
cultured them over night.
7/20
Miniprep of E.coli cells containing YqiT gene with Promega, Wizard Plus SV Miniprep DNA Purification System
7/27
Sequencing YqiT by BIG DYE
1.8ul 5xB.D.3.1.buffer
0.4ul B.D.3.1.
6.3ul MilliQ water
1ul plasmid(0.15ug/ul)
0.5ul 5'or3'primer(3.2pmol/ul)
|
v
PCR Program
1.96°C 2min
2.96°C 10sec
3.55°C 5sec
4.60°C 3min
5.go to 2.29times
6.25°C forever
|
v
add 0.5ul PHOSPHATASE ALKALINE shrimp
|
v
37°C 1hr incubate
|
v
add 1ul 3MNaOAc
|
v
add 25ul EtOH
|
v
20000xg 4°C 10min centrifugation
|
v
put off supernatant
|
v
dry tubes
|
v
add 15ul HiDi
|
v
put them in the sequence machine
7/30~8/2
Sequencing YqiT by BigDye
unsuccessful
August
Sequence reading succeed!
We made YqiT + double terminator.
8/3
sequence YqiT by BigDye
successful
8/7~9
Digestion
insert YqiT gene into a terminator of iGEM parts
terminator
double terminator Miniprep ., using Promega kit
plate1 23L
8/12~14
PCR of YqiT for restriction enzyme fragmentation and infusion
PCR protocol
The following was mixed in a PCR tube:
0.2ul 100uM yqiT EX primer/yqiT infusion 5'primer
0.2ul 100uM yqiT SP primer/yqiT infusion 3'primer
2ul 10X Pfu Ultra buffer
1.6ul dNTP mix
0.2ul plasmid containing YqiT gene(that sequence matches YqiT gene of Batillus genome)
0.5ul Pfu Ultra enzyme
15.3ul MilliQ
Performed PCR using the following program:
1. 95°C 2 minutes
2. 95°C 30 seconds
3. 55°C 30 seconds
4. 72.5°C 30 seconds
5. Repeat 2-4 29 times
6. 25°C forever
restriction enzyme fragmentation of double terminator by XbaI
1ug plasmid
10ul BSA
10ul 10xM buffer
2.5ul XbaI
up to 100ul MilliQ water
37°C over night
restriction enzyme fragmentation of double terminator by EcoRI
1ug plasmid
10ul 10xH buffer
2.5ul EcoRI
up to 100ul MilliQ water
37°C over night
restriction enzyme fragmentation of YqiT gene by SpeI
1ug DNA
10ul 10xH buffer
2.5ul SpeI
up to 100ul MilliQ water
37°C over night
restriction enzyme fragmentation of YqiT gene by EcoRI
1ug DNA
10ul 10xH buffer
2.5ul EcoRI
up to 100ul MilliQ water
37°C over night
8/15,16
ligate YqiT and double terminator
8/19
1 colony PCR of YqiT
Result: successful
September
Aim
We decided to use YqiT as a reporter of dioxin!
So, we began to create this construct
1.-XRE-Gal1 promoter-yqiT-
2.-Ahr-←Gal1promoter/Gal10 promoter→-Arnt-
PCR of yqiT and insert it in iGEM parts(plate 1-7D:Gal1 promoter)
Performed PCR using the following program:
1. 95°C 2 minutes
2. 95°C 30 seconds
3. 55°C 30 seconds
4. 72.5°C 30 seconds
5. Repeat 2-4 29 times
6. 25°C forever
cut YqiT off with EcoRI and SpeI
cut P1-7D off with EcoRI and XbaI
ligate the upper two DNA
read the Sequencing of Gal1 promoter+yqiT
->the result conforms with NCBI datebase
9/15
1. Restriction enzyme digestion
P1 7L with E
2. ligation + transformation
P1 7L + yqit
3. restriction enzyme digestion
P1 7L with S
4. restriction enzyme digestion
yqit with X and P
P1 7D with S and P
9/16
1. Colony PCR
P1 7D+yqit
2. gel extraction (product of restriction enzyme digestion)
P1 7L (E,X)
P1 7D (S,P)
yqit (X,P)
3. ligation and transformation
yqit + P1 7D
9/17
1. colony PCR
P1 7D + yqit
2. restriction enzyme digestion
yqit with X and P
P1 7D with S and P
P1 7L with E and S
9/19
1. Ligation->transformation
P1 7D+yqit
9/20
1. Sequencing
yqit->failure
2. colony PCR
P1 7D+yqit->success
9/21
1. Miniprep
P1 7D+yqit
9/22
1. Sequencing
P1 7D+yqit->failure
9/23
1. restriction enzyme digestion
P1 7D      with E and X
P1 7L+yqit with E and X
2. Ligation->transformation
P1 7D+XRE
P1 7D+yqit+XRE
9/24
1. Miniprep
P2 10F
2. restriction enzyme digestion
P1 7D      with E and X
P1 7L+yqit with E and X
9/25
insert XRE in P1-7D and P1-7D including yqiT using Infusion kit(clonthech)
gel purification of P1-7D and P1-7D including yqiT
9/26
preculture of E.coli cells containing Ahr and Arnt gene for Miniprep
9/27
Miniprep of E.coli cells containing Ahr and Arnt gene with Promega, Wizard Plus SV Miniprep DNA Purification System
Aim
TA cloning of Ahr, Arnt and gal1/10 promoter
PCR of Ahr and Arnt for TA cloning(Ex-taq , Funakoshi)
template:human Ahr cDNA and human Arnt cDNA
1,95°C 2min
2,95°C 30sec
3,55°C 30sec
4,72.5°C 1.5min
5,go°C to 2, 29times
6,25°C forever
9/29
ligation of P17D+YqiT and XRE
ligation of P17D and XRE
restriction enzyme digest
P17D with S and P
sequencing of XRE+P17D
9/30
miniprep of ARNT and AMR
colony PCR of P17D+XRE
primer:
EX_XRE3_5_5'
J630101P7Dseq3'
restriction enzyme digest
YqiT+D.T. with Pst1 and Xba1
again, colony PCR of P17D+XRE
primer:
EX_XRE3_5_5'
J630101P7Dseq3'
October
We made the plasmid including XRE+Gal1p+YqiT.
Cloning of Ahr, Arnt and gal1/10p were difficult...
10/2
colony PCR of YqiT
10/7
PCR of gal1/10 promoter (Ex-taq , Funakoshi)
template:S.serevisiae genome DNA
1,95°C 2min
2,95°C 30sec
3,55°C 30sec
4,72.5°C1min
5,go°C to 2, 29times
6,25°C forever
10/9
PCR of gal1/10 promoter (Ex-taq , Funakoshi)
template:S.serevisiae genome DNA
1,95°C 2min
2,95°C 30sec
3,55°C 30sec
4,72.5°C1min
5,go°C to 2, 29times
6,25°C forever
10/10
TA cloning of Ahr, Arnt and gal1/10 promoter
ligate PCR product and T bector
->
put each plasmid into different E.coli
->
Miniprepof them by using Promega kit
->
read each sequence in order to check whether there is plasmids including Ahr, Arnt or gal1/10 promoter
10/11
check YqiT expression
Using iGEM parts, create the construct that produces YqiT steadily or reguratory
P2-10F-YqiT-double terminator
P2-16E-YqiT-double terminator
P1-22A-YqiT-double terminator
P1-21B-YqiT-double terminator
cut YqiT-double terminator off with XbaI and PstI
cut P2-10F, P2-16E, P1-22A and P1-21B off with SpeI and PstI
|
v
ligate YqiT-double terminator and iGEM part
|
v
put each plasmid into different E.coli
|
v
Miniprep of them by using Promega kit
|
v
read each sequence in order to check whether there is plasmids including the construct
10/12
Miniprep of P1,10F with Promega Kit
10/13
Sequencing of Arnt, Ahr
10/14
Colony PCR of Arnt, Aht, and pGal1/10
Sequencing Ahr, Arnt, and pGal1
preculture of Arnt and Ahr
10/15
Sequencing Arnt, Ahr and pGal1/10
10/16
Miniprep of Arnt,Ahr and pGal1/10 with Promega Kit
10/18
1. Sequencing
read the following sequences;
P1-23A+yqit
P1-21B+yqit
p1-16E+yqit
XRE+Gal1+yqit
P2 10F
2. transformation
P2 10F+yqit+double terminator
3. Miniprep
P2 10F
4. Restriction enzyme digestion
yqit with X and P
XRE+P1 7D with S and P
5. Gel extraction(after gel electrophoresis)
yqit (processed with restriction enzymes)
P1 7D processed with restriction enzymes)
6. Restriction enzyme digestion
yqit with P
7. Restriction enzyme digestion
P1 7D with S and P
P1 22A with E and P
8. Sequencing
P1 14L
P1 16E
P1 21B
P1 22A
P2 5F
P2 9H
P2 10F
P2 16E
XRE-Gal1-yqit
9. Restriction enzyme digestion
P1 7D with S and P
10. Ligation
XRE and yqit
11. Transformation
XRE + yqit
12. restriction enzyme digest
yqit with X and P
P2 10F with S and P
10/19
1. colony PCR
P1 7D+XRE+yqit+double terminator
2. restriction enzyme digest
P2 10F with S and P
yqit with X and P
3. sequencing
P2 10F
4. Ligation
P2 10F + yqit
5. transformation
P2 10F + yqit
yqit
6. restriction enzyme digest
XRE+P1 7D+yqit with E and P
7. infusion
yqit + PET
8. colony PCR
P1 7D+XRE+yqit
9. colony PCR
P2 10F+yqit
10/20~
1. colony PCR
P2 10F+yqit
2. Smell Test 2. Smell Test
Time for shake culture (each of sample) 15 hours
Negative control has plasmid containing “yqiT + double terminator” (Not including promoter)
Cleansed air is cleansed with active carbon filter.
Equipment
Trade name: ODOR CONCENTRATION METER
Model: XP-329
Serial No. 311059
Manuf. NEW COSMOS ELECTRIC CO,LTD.
Setting: Peak hold
the following values indicate intensity of the smell
cleansed air:200
air:515
normal E.coli: 582
E.coli with constitutive promotor+RBS+yqit+double terminator: 686
yqit successfully worked!!
Associate Prof. Kawanishi offered us the yeast, whose chromosome contains AhR and ARNT.
Result
We have made E.coli which has "Constitutive promoter (strong) - RBS - yqiT - double terminator" in its plasmid.
and
We will have made yeast which has Ahr and ARNT in its chromosome, and "XRE - Gal1p - yqiT" in its plasmid by 21 October.
Home
The Team
The Project
Parts Submitted to the Registry
Modeling
Notebook
Protocols
Ethics
Retrieved from " http://2009.igem.org/Team:Todai-Tokyo/Notebook/isoleucine "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
