<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Vanderbilt_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Vanderbilt/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Vanderbilt/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Vanderbilt/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Vanderbilt/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Vanderbilt/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Vanderbilt/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TABLE width="70%" align="center" margin-bottom:=""><TBODY><TR heigth="75px"><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#000000'" bgcolor="#000000"><A href="https://2014.igem.org/Team:Vanderbilt" style="color:#CC9900">Home</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#000000'" bgcolor="#000000"><A href="https://2014.igem.org/Team:Vanderbilt/Team" style="color:#CC9900">Team</A></TD><TD style="border:1px solid black" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#000000'" bgcolor="#000000"><A href="https://2014.igem.org/Team:Vanderbilt/Project" style="color:#CC9900">Project</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#000000'" bgcolor="#000000"><A href="https://2014.igem.org/Team:Vanderbilt/Parts" style="color:#CC9900">Parts</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#000000'" bgcolor="#000000"><A href="https://2014.igem.org/Team:Vanderbilt/Notebook" style="color:#CC9900">Notebook</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#000000'" bgcolor="#000000"><A href="https://2014.igem.org/Team:Vanderbilt/Safety" style=" color:#CC9900">Safety</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#000000'" bgcolor="#000000"><A href="https://2014.igem.org/Team:Vanderbilt/Attributions" style="color:#CC9900">Attributions</A></TD></TR></TBODY></TABLE></P><H3> Introduction </H3><P>
The production of plant essential oils and their derivatives represents an over 9 billion dollar industry when considering their applications in the food and fragrance industries alone <SUP>1</SUP>. Up until only a couple decades ago, the production of these essential oils was done almost exclusively by chemical extraction from plant material. However, the sudden emergence of synthetic biology a versatile and efficient tool has the potential to transform this immense industry,whose products are an unavoidable staple of everyday life. 
</P><P>
By harnessing existing biosynthetic pathways and introducing enzymes taken from plants into malleable model systems, it is possible to significantly improve on current methods for synthesizing the active components of essential oils, most notably the terpenoids. While most plants express terpenes in the range of parts per million and thus require very large scale operations to be commercially viable, early forays into the biological production of terpenes have proven that it is possible to improve yields 100-fold <SUP>2</SUP>. We selected a total of nine different terpenes to produce (see table) to prove the feasibility of our distinct approach. To demonstrate the strengths of our approach, we purposefully chose what amounted to, relative to most iGEM projects, a large number of pathways to be working on simultaneously. The list includes both well-characterized terpenes and more exotic varieties, all of which share direct practical applications that make them prime candidates for alternative modes of manufacture. The first aspect of our project involves using the great potential of synthetic biology to design a commercially-viable strategy for the production of numerous terpene varieties.
</P><P>
Just as important to the economic benefit of this approach is its environment benefit. With chemical terpene extraction being such a relatively inefficient process, it is necessary to process large amounts of plant material to get a substantive yield. This may not pose as significant of an issue for citrus growers, but many of the most prized compounds are taken from the rarest species of plant. The continuation of the status quo in terms of terpene extraction is not an environmentally sustainable solution. In our selection of terpenes, were conscious of the value of our approach from the perspective of sustainability. From a small amount of plant tissue as a starting sample, we could create theoretically limitless amount of terpene in our microbes, and by starting with gene extractions from plants, there is left open the possibility to use techniques like degenerate primers or sequences conserved across terpene classes to identify terpene genes from rare or endangered species that may not have genome sequences readily available. 
</P><P>
The best example of this idea behind our project can be seen in the gene for santalene synthase. The only species know to have genes to produce this terpene are found in remote regions of India and Australia, and one of them is listed as a vulnerable species by the IUCN. The trees can live for hundreds of years, but are the target of widespread over-exploitation, to the point that, for example, the Indian government has banned the export of sandalwood. Synthetic biology can produce the exact same active ingredients of sandalwood oil in a way that is both more economically and environmentally sound. 
</P><P>
In order for our idea to truly be applicable to exotic and endangered species of plant, we had to take an approach that was quite different from the one most iGEM teams have historically taken. What we were looking for was a quick, inexpensive method of cloning genes that was also compatible with species that have not had their entire genomes sequenced. Often, iGEM teams resort to synthesizing their genes through third parties. However, this can be fairly costly especially given the moderately large size of many of these synthase genes, may take several weeks or even a month to finish synthesizing, and cannot be done unless the gene's sequence is known in its entirety.  By going back to basics and taking raw plants as our source material, we were able to avoid these issues and demonstrated how our approach was a practically viable one. 
</P><TABLE width="500" border="1" cellpadding="5"><TBODY><TR><TD align="center" valign="center"><B>Cadenine</B>
Main essential oil: Cade oil
Applications: Antifungal, bactericidal, and antioxidant
Plant Species: <I>Gossypium hirsutum</I> (cotton)
Synthase Gene: δ-cadenine synthase (E.C. 4.2.3.13)
</TD><TD align="center" valign="center"><B>Carene</B>
Main essential oils: Rosemary and Cedar oil
Applications: Insecticide, anti-inflammatory, and central nervous system depressant
Plant Species: <I>Picea abies</I> (norway spruce)
Synthase Gene: carene synthase (E.C. 4.2.3.107)
</TD><TD align="center" valign="center"><B>Humelene</B>
Main essential oils: Hops oil
Applications: Culinary spice, and anti-inflammatory
Plant Species: <I>Zingiber zerumbet</I> (shampoo ginger)
Synthase Gene: α-humulene synthase (E.C. 4.2.3.104)
</TD></TR><TR><TD align="center" valign="center"><B>Myrcene</B>
Main essential oils: Thyme and Hops oil 
Applications: Fragrance, analgesic, and anti-inflammatory
Plant Species: <I>Perilla frutescens</I> (Green Shiso)
Synthase Gene: myrcene synthase (E.C. 4.2.3.15)
</TD><TD align="center" valign="center"><B>(R)-Linalool</B>
Main essential oils: Lavender oil
Applications: Fragrance, and insecticide
Plant Species: <I>Mentha citrata</I> (lemon mint)
Synthase Gene: (R)-linalool synthase (E.C. 4.2.3.26)
</TD><TD align="center" valign="center"><B>(S)-Linalool</B>
Main essential oils: Citrus and Coriander oil
Applications: Fragrance, and insecticide
Plant Species Chosen: <I>Arabidopsis thaliana</I> (thale cress)
Synthase Gene: (S)-linalool synthase (E.C. 4.2.3.25)
</TD></TR><TR><TD align="center" valign="center"><B>Sabinene</B>
Main essential oils: Junier Coriander oil
Applications: Spice, and antimicrobial
Plant Species Chosen: <I> Salvia officinalis</I> (sage)
Synthase Gene: (+)-sabinene synthase (E.C. 4.2.3.110) 
</TD><TD align="center" valign="center"><B>Santalene</B>
Main essential oils: Sandalwood oil
Applications: Fragrance, antiviral, and tumor-suppressant
Plant Species Chosen: <I> Santalum album</I> (sandalwood tree)
Synthase Gene: α-santalene synthase (E.C. 4.2.3.82)
</TD><TD align="center" valign="center"><B>Zingiberene</B>
Main essential oils: Ginger oil
Applications: Flavoring and pesticide
Plant Species Chosen: <I> Ocimum basilicum</I> (basil)
Synthase Gene: α-zingiberene synthase (E.C. 4.2.3.65)
</TD></TR></TBODY></TABLE><H3>Design</H3><P>
Terpene biosynthesis in plants is part of larger pathways that metabolize isoprenoid intermediates. Genes encoding for enzymes known as synthases catalyze the terminal step in these pathways, from a precursor (commonly farnesyl pyrophosphate (FPP) or garnyl pyrophosphate (GPP)) to the final terpene product. As it happens, two well established and genetically manipulable model organisms- the bacterium <I> Escherichia coli </I> and brewer's yeast <I> Saccharomyces cerevisiae</I>- produce moderate amounts of GPP and FPP as part of their endogenous non-mevalonate pathway (MEP) and mevalonate pathway (MEV) respectively<SUP>3</SUP>. All that is required for either of these organisms to begin producing terepenes is to introduce that single synthase gene.
</P><P>
Yeast initially was our main target for a few advantages it appeared to have as a production platform. First, the MEV pathway is found in all eukaryotes including plants and fungi, so better yield were expected by choosing it over the MEP pathway in prokaryotes<SUP>4</SUP>. Second, it would be possible to physically integrate our gene into one of the yeast's chromosomes by using homologous recombination. An inducible promoter could be included to further increase production. Third, as a diploid the yeast could be made homozygous for the terpene gene. We soon found out that no exiting vector had all of the features we would want. Therefore, we designed our own new plasmid vector, pVU140006, that contained a number of important features and advances over previous plasmids for this purpose. See our parts page for more information on what special features we included and their relevance to the project
</P><H3>Methods</H3><P>
Our project had several co-dependent sub-projects that, thanks to our unusual design, could all be worked on in parallel. These can roughly be divided into two categories: the first involving work on our synthase genes and the second involving the construction of a new, specially designed plasmid vector. We tried two different team structures over the year to see which would give the best results. For the Spring, we had the original idea of dividing members into independent groups, each working on a specific terpene. Each group was headed by a single group manager who would teach 4-5 new members the protocol that was to be preformed and then supervise that the work was carried out correctly. On occasion either the group managers or the organization president or wetware director would also given lessons to teach members about the techniques and theory involved at each step. All group managers were in turn trained by either the president or wetware director, whom had come up with the experiment, acquired all the necessary primers and reagents, wrote up the protocol, and had preformed it prior to any group-phase work for the purposes of troubleshooting and predicting where issues may come up. The president or wetware director also assisted the group manager by being present during all experiments for answering questions, preparing materials, and troubleshooting. 
</P><P>
Each group first planted seeds under the appropriate soil, humidity, and temperature conditions at the Vanderbilt Greenhouse. Once the majority of these grew into saplings with green leaves, samples were flash frozen in liquid nitrogen in preparation for a genomic DNA extraction. After the extractions, nanodrop concentration readings and agarose gels confirmed the presence of high molecular weight genomic DNA. Groups then ran a PCR on their genomic DNA with primers specific to their synthase gene. More gels were run to check for the presence of PCR product. At this point, the semester was coming to an end, so groups were disbanded before most managed to isolate their synthase gene. Over the summer, the president and wetware director continued troubleshooting those genes which were not amplifying and eventually got each to the point where consistent PCR product bands were produced. </P><P>
Once we had clear banding patterns, it became clear that the number of introns in each of our genes (a variable which was unknown since most of the plants we worked with have not had their genomes sequenced) was too great for cloning and expression to be practical. Therefore, as soon as the fall semester began, we shifted strategy to isolating RNA from our plants. This RNA could be converted to cDNA by reverse transcription, which would eliminate the issue with introns we were having. Several of our greenhouse plants were no longer available, so we reduced the number of target terpenes we were focusing on. After extracting RNA and running an RT-PCR, several samples produced bands that corresponded roughly to where the synthase gene should be. These were gel extracted. Because almost every synthase gene had restriction sites in them that prevented them from being RFC10 compatible, we ligated the genes in pUC19 for site directed mutagenesis. After that, a second processing step would have been necessary to add the correct restriction sites to each gene to allow them to be integrated into pSB1C3 as a biobrick. To ensure the part submission deadline would be met, we synthesized a codon-optimized santalene synthase gene in order to skip these RFC10 processing steps, even though we had already successfully reverse transcribed cDNA of the santalene synthase gene.
</P><P>
In the spring concurrent to work by the terpene groups, work began on plasmid construction. This was preformed by the president, wetware director, and a handful of others rather than in group format. Each gene cassette for our final plasmid was first identified in an existing, readily available plasmid. All of these cassettes were extracted by PCR using those plasmids as templates. Overlap extension PCR was then done on the gel-purified product to add restriction sites and homology regions for the purposes of eventually combining all of the cassettes together into a single plasmid. By the end of the summer, only one final fragment remained to be inserted to complete the intermediate plasmid pVU14004. Upon the successful creation of pVU14004, several restriction enzyme sites had to be removed by site directed mutagensis in order to make the plasmid RFC10 compatible. 
</P><H3> Results and Directions </H3><P>
Several factors contributed to the difficulty we experienced during the final phase of the project. First, member engagement suffered a significant decline between the spring and fall semesters, to the point where only a small handful of people were left to preform all experiments. Second, the late realization that we had to change our cloning strategy to modified cDNA inserts effectively meant we had to start anew in late August despite having what was a good head start when we began in early March. Third, the RFC10 requirements added a substantial dimension of difficulty to the project since all of our starting material (both the extracted gene cassettes for plasmid construction and the synthase genes) contained multiple sites that made them incompatible with the biobrick standard. Nevertheless, our team accomplished an enormous amount during our first year in competition. </P><TABLE width="90%"><TBODY><TR><TD><I>Terpene </I></TD><TD><I>Plant Genomic DNA Successfully Extracted</I></TD><TD><I>Synthase Gene Successfully PCR Isolated </I></TD><TD><I>Plant RNA Successfully Extracted</I></TD><TD><I>Synthase gene Successfully Reverse Transcribed</I></TD><TD><I>Terpene Successfully Produced in E. coli or yeast </I></TD></TR><TR><TD><B>Cadinene</B></TD></TR><TR><TD><B>Carene</B></TD></TR><TR><TD><B>Humelene</B></TD></TR><TR><TD><B>Myrcene</B></TD></TR><TR><TD><B>(R)-Linalool</B></TD></TR><TR><TD><B>(S)-Linalool</B></TD></TR><TR><TD><B>Sabinene</B></TD></TR><TR><TD><B>Santalene</B></TD></TR><TR><TD><B>Zingiberene</B></TD></TR></TBODY></TABLE><P>
We successfully constructed pVU14004, which contains all of the cassettes needed to have all of functionality we had intended. However, this form of the plasmid has XbaI and EcoRI sites that make it incompatible with RFC10. These sites were successfully changed to missense mutations by site directed mutagenesis. The final step required to make pVU14006 is changing the multiple cloning site (MCS) to have the biobrick prefix and suffix. In its current form, the plasmid has a functional MCS but it does not yet have the specific order of EcoRI, XbaI, SpeI, and PstI sites required by RFC10. 
</P><P>
For a listing of all the medal requirements we successfully fulfilled over the course of our project, please visit <A href="https://2014.igem.org/Team:Vanderbilt/Project/Requirements" style="color:#000000"><U> this page </U></A>.
</P><H3> Collaborations </H3><P>
In addition to our own wetware project, our team led fruitful collaborations with a total of three other iGEM teams. First, we played a major role in assisting Vanderbilt's microfluidics division with the genetics aspect of their project. We prepared the biobrick parts they tested in their microfluidic device, including transforming the E. coli they used to study their quorum-sensing fluorescent oscillator circuit.  Second, we provided feedback to Vanderbilt's software division about their own project involving a program to aid in the manipulation of genetic sequences. We used the program as if it were for a real project and gave them suggestions on how to make their program easier to use and more useful to biologists.
</P><P> 
Last but not least, we provided significant assistance and guidance to the Ravenwood Raptors high school iGEM team. In a series of conversations with Dr. Amanda Benson, we planned a smaller scale version of our own project that involved only a single terpene synthase gene. After visiting the high school and presenting our idea, the students voted to choose it for their project. We also provided the primers and sage genomic DNA template they used in their experiments. 
</P><B> References: </B><I>1. USDA Industrial Uses Reports. Essential Oils Widely Used in Flavors and Fragrances. September 1995. 
2. Ajikumar PK, Tyo K, Carlsen S, Mucha O, Phon TH, Stephanopoulos G. Terpenoids: opportunities for biosynthesis of natural product drugs using engineered microorganisms. Mol Pharm. 2008;5(2):167-90. 
3. Dudareva N, Klempien A, Muhlemann JK, Kaplan I. Biosynthesis, function and metabolic engineering of plant volatile organic compounds. New Phytol. 2013;198(1):16-32. 
4.Martin VJ, Pitera DJ, Withers ST, Newman JD, Keasling JD. Engineering a mevalonate pathway in Escherichia coli for production of terpenoids. Nat Biotechnol. 2003;21(7):796-802. </I><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Vanderbilt/Project">http://2014.igem.org/Team:Vanderbilt/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Vanderbilt/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Vanderbilt/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Vanderbilt/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Vanderbilt/Project&amp;oldid=406073" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>