"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Vanderbilt/Notebook
From 2014.igem.org
Home
Team
Project
Parts
Notebook
Safety
Attributions
Lab Notebook
Spring 2014
March 27th
Extracted genomic DNA from 100 mg of Arabidopsis thaliana leaves using a Viogene DNA extraction kit. Samples were labeled ZIN for zingiberene.
Two samples were prepared and nanodropped. The concentration of the first was 2.6 ng/ul, and the second was 3.5 ng/ul of DNA, indicating a minimal yield of DNA.
March 30th
Extracted genomic DNA from 100 mg of Picea abies needles using the same extraction kit and protocol. Samples were labeled CAR for carene.
Two samples were prepared and nanodropped. The concentration of the first was 0.9 ng/ul, and the second was 3.5 ng/ul of DNA, indicating a minimal yield of DNA.
March 31st
Ran a 0.6 % argarose gel on the DNA extracted from ZIN and CAR, as well as the column flow-through from the kit.
ZIN1, ZIN2, CAR1, and CAR2 all show a faint but distinct DNA band above the highest rung on the DNA ladder (>10kb), showing the presence of DNA. No bands were seen on the kit flow-through. Indicates successful genomic extraction.
Preformed a second genomic extraction on Picea abies to improve yield. Nanodrop shows CAR3 to be at 6.2 ng/ul and CAR4 to be 11.5 ng/ul.
Extracted genomic DNA from Gossypium hirsutum . Samples nanodroppped: CAD1 1.8 ng/ul and CAD2 7.8 ng/ul.
April 1st
Ran a gel on CAR3, CAR4, CAD1, and CAD2. Brighter genomic DNA bands were seen on the cadinene camples than before, but cadinene samples showed significant smearing near the top of the gel.
April 2nd
Extracted genomic DNA from Salvia officinalis. Samples nanodropped: SAB1 7.5 ng/ul, and SAB 11.4 ng/ul.
Extracted genomic DNA from Mentha citrata. Samples nanodropped: LNR1 3.4 ng/ul, LNR2 6.2 ng/ul, LNR3 7.3 ng/ul
E. coli containing p404GALS and pDZ207 from Addgene were grown on LB plates with ampicilin. These were miniprepped using a Viogen kit. Samples nanodropped: p404GALS (A) 130.2 ng/ul, p404GALS (B) 112.3 ng/ul, pDZ207 (A) 145.3 ng/ul, pDZ207 (B) 84.1 ng/ul.
April 3rd
Extracted genomic DNA again from Arabidopsis thaliana . Sample nanodropped: LNS1 53.4 ng/ul, LNS2 585.2 ng/ul
Ran gel on LNR1, LNR2, LNR3, LNS1, LNS2, SAB1, and SAB2. All show high weight DNA bands, with Linalool (S) samples having the brightest.
April 4th
Extracted genomic DNA from Zingiber zermbet . Sample nanodropped: HUM1 31.9 ng/ul and HUM2 18.6 ng/ul
Extracted genomic DNA from Ocimum basilicum . Sample nanodropped: GER1 2.3 ng/ul
April 5th
Extracted genomic DNA from Santalum album seeds since the sapling was still not fully grown. Sample nanodropped: SAN1 53.8 ng/ul and SAN2 28.4 ng/ul
April 7th
Extracted genomic DNA from Perilla frutescens . Sample nanodropped: MYR1 632.8 ng/ul and MYR2 958.6 ng/ul
Ran gel on MYR1, MYR2, SAN1, and SAN2. Myrcene samples show bright band at high weight as well as smearing toward bottom. Santelene samples have visible but much fainter genomic DNA band.
April 24th
Preformed PCR using zingiberene synthase primers on LNS2 genomic DNA and using linalool (S) synthase primers on the sample template.
Ran gel on PCR product. Resulted in no visible bands formed.
April 25th
Took 1.8 ul of HUM1 genomic DNA and preformed a PCR with humelene synthase primers.
Ran gel on PCR product. Resulted in three total bands, one faint around 1.2 kb, and two bands very close in size just under 3.0 kb.
Both ~3 kb bands were gel extracted, combining  across all four lanes. Samples nanodropped: HUM-top 8.5 ng/ul, HUM-bottom 10.6 ng/ul.
April 27th
Preformed PCR using linalool (R) synthase primers on LNR3 genomic DNA.
Ran gel on PCR product. Resulted in no visible bands formed.
April 29th
Preformed PCR using myrcene synthase primers on MYR2 genomic DNA.
Ran gel on PCR product. Smeared bands on gel but no distinct bands.
Preformed overlap-extension PCR on the gel extracted humelene samples (both HUM-top and HUM-bottom) to add epitope tag sequence
Ran gel on OE-PCR product. Only extremely faint bands were visible.
Summer 2014
May
Continued troubleshooting PCR reaction conditions for all of the terpenes that failed to amplify. Tried adjusting template concentration, adding DMSO, changing thermocycler program, hot start PCR, new polymerase and dNTPs
Re-did genomic extractions for those that produced less than 50 ng/ul of genomic DNA. Used this new template in further PCRs
Extracted the Gal10 gene from our template plasmid and a kanomycin resistance gene bordered by LoxP sites.
June
All of the gene cassettes for plasmid construction that were successfully extracted so far were ligated together and inserted into the MCS of pUC19. This formed our first intermediate plasmid.
July
Finally reached the point that all terpene genes were consistently amplifying with the synthase gene primers. The genomic DNA sample for sabinene was completely used before this point, although all other of the 8 terpenes showed clear bands.
The results of the genomic DNA PCR indicated each gene had a large fraction of introns. None of the genes had a distinct band at exactly the right weight corresponding with what the intron-less cDNA size would be.
Fall 2014
August
Moved the lab into its new space before the start of the semester
Created the plasmid intermediate pVU1400A, which is missing only a single insert to become pVU14004.
September
Ran RNA extraction on all the plants that were still available. This excluded Myrcene and Linalool (R) since both Perilla frutescens and Mentha aquatica had withered over the summered.
Repeated RNA extractions on those which showed appreciable concentration on the nanodrop. Eventually all 7 remanining terpenes had plant RNA in appreciable quantities (most between 20-50 ng/ul, with a few less than 10 and a few more than 100 ng/ul).
September 17th
Digested plasmid intermediate grown in demethylated bacteria with ClaI. Ligated final insert into vector. No transfomants grow after 24 hours.
Diagnostic digest shows the ClaI enzyme is cutting properly. pUC19 positive control for transformations show that the competent cells are working.
September 18th
Ran reverse transcription PCR on extracted RNA to isolate synthase cDNA. Humelene and sabinene show clear positive results, santalene shows amplification at smaller region, and cadinene shows no cDNA bands.
September 19th
Made liquid cultures of K546546 in preparation for mutagenesis. Also made glycerol stock to store at -80.
Diagnostic digest of ligation of pVU1400A intermediate and the final insert needed to make finished plasmid. Gel clearly shows bands in exactly the correct positions for each of three comparison conditions, proving that the creation of pVU14004 was finally successful.
September 20th
Miniprep of K546546 liquid cultures (1 ml ). First culture tube concentration of 85.7 ng/ul DNA, second 105.1 ng/ul
RNA extracted arabadopsis and Picea abies to improve yield and quality. Carene still failed to get an RNA concentration greater than 10 ng/ul, while Arabidopsis produced 107 ng/ul with a good A260/A280 ratio.
September 20th
Miniprep of K546546 liquid cultures (1 ml). First culture tube concentration of 85.7 ng/ul DNA, second 105.1 ng/ul
September 26th
RT-PCR done on humelene, linalool (S), sabinene, and zingiberene. Sabinene produces clear bands, while zingiberene shows one faint band at roughly the correct size. Positive controls are also run to confirm that the reverse transcription step is not the cause of any failures to amplify.
October 5th
RT-PCR done on humelene,  sabinene, and santalene. Sabinene again produces clear bands,and santalene does as well although much  fainter. Both bands were gel extracted to yield a small (<10 ng/ul) amount of DNA.
Extracted DNA was ligated into pUC19 and transformed into E. coli.
October 7th
Site direction mutagenesis kit and specially designed primers were used to mutagenize K546546 at its BglI site, sabinene cDNA at  its XbaI and EcoRI sites, and pVU14004 at its EcoRI and XbaI sites.
Mutagenesis product was transformed into E. coli.
October 9th
Minipreps were done on 5 ml liquid cultures of mutagenized pVU14004, sabinene, and K546546. 4 liquid cultures were made for each, and both sabinene and pVU14004 were done in duplicate.
Diagnostic digests were done on miniprepped plasmid to check if the restriction sites were mutagenized. pVU14004 appeared to have lost its XbaI site but not its EcoRI site, sabinene shows a size that suggests it failed to ligate as an insert into pUC19, and K546 shows only a single band at around its starting weight.
October 10th
All 8 minipreps of sabinene ligated into pUC19 were digested with SpeI and ApaI to check for the synthase insert. Only one, Sab B2, shows a second band at the right size.
October 11th
Santalene synthase was ligated into pVU14004 and pSB1C3. E. coli was transformed and incubated.
The sites that failed to show mutagenesis were mutagenized again using and transformed into E. coli. K546546 had its AgeI site mutagenized.
October 12th
All liquid cultures were miniprepped, producing 8 samples of pVU14004 with confirmed XbaI mutagenesis, 4 pVU14004 with no sites confirmed, 4 sabinene, and 6 samples produced from santalene in pVU14004. None of the plates with santalene in pSB1C3 produced colonies.
Diagnostic digests were run on all miniprepped plasmid (K546- AgeI, BglI, SphI.  Sabinene- EcoRI, BamH1. Santalene in pVU- ApaI, XbaI. pVU- EcoRI, XbaI, BamHI, KpnI). Santalene appeared not to have ligated into pVU14004. Sabinene did not have its EcoRI site removed by mutagenesis. K546546 had at least one cut, but the second sample may have had one site mutagenized.
Santalene was re-ligated into pSB1C3 and transformed.
October 13th
Santalene in pSB1C3 was miniprepped to good yield. Each of 4 replicates was digested with SpeI and ApaI to test for ligation. The ApaI enzyme appears not have cut, but the fourth sample showed an uncut plasmid size which corresponded to that of pSB1C with santalene successfully inserted.
October 14th
Ligated santalene synthase again into pVU14004 and transformed into E. coli
October 15th
One colony grew and was put in liquid culture.
Culture miniprepped and digested. Again no gene insertion was detectable.
October 16th
Transformed pVU14004 into a dam- strain of E. coli to address the methylation sensitivity of ApaI.
Finished planning and acquiring materials for GC-MS confirmation of terpene presence.
Retrieved from " http://2014.igem.org/Team:Vanderbilt/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
