"
Page
Discussion
View source
History
teams
Log in
 
Team:Waterloo/Silence
From 2014.igem.org
Project ▾
Overview
Deliver
Silence
Translate
Safety
Notebook ▾
Lab Book
Protocols
Math Book
Outreach
Accomplishments ▾
Biobricks
Collaborations
Checklist
Acknowledgements
Team ▾
Our Team
Sponsors
Gallery
Silence
OVERVIEW
CRISPRi
sRNA
DESIGN
RESULTS
Overview: Silencing Antibiotic Resistance
By silencing the expression of genes responsible for antibiotic resistance, antibiotic-resistant bacteria can be converted to antibiotic-sensitive bacteria. In the case of MRSA, silencing the expression of mecA and its regulatory elements create a population of antibiotic-sensitive MRSA (Meng et al., 2006; Hou et al., 2007; Sakoulas et al., 2001). To model silencing in MRSA, our team aimed to silence YFP expression in S. epidermidis using CRISPRi and RNAi.
CRISPRi: Silencing Transcription
CRISPR systems (clustered regularly interspaced short palindromic sequences) are involved in the adaptive immune systems of bacteria and archaea. Mature CRISPR RNA (crRNA) binds to trans-activating crRNA (tracrRNA) to form a complex that is recognized by the CRISPR associated protein (Cas9). The Cas9 protein is directed to a target DNA site where it performs a double stranded break (Karginov and Hannon, 2010). An improvement to the original CRISPR system is the use of sgRNA which is a chimera of the crRNA and tracrRNA complex and this simplified version has been shown to be effective more (Jinek et al., 2012).
The CRISPR system can be modified further to regulate gene expression. The CRISPRi (CRISPR interference) system involves a catalytically dead Cas9 protein (dCas9) paired with a single guide RNA.  Together, they are able to interfere with transcription and halt the expression of the target gene (Qi et al., 2013).  This process is reversible and nonfatal; since dCas9 is catalytically dead, it does not cleave targeted DNA in the way an endonuclease normally would (Qi et al., 2013).
CRISPRi: dCas9-sgRNA complex binds to DNA and interferes with transcription.
Allowing the MRSA cells to survive is ironically in our favor; we want our silencing plasmid to propagate throughout the entire infected population exponentially and S. aureus cells that have been conjugated-into are also capable of conjugating the construct into others. The desired result is therefore a population in which most cells have been infected with the conjugative silencing plasmid, actively suppressing their chromosomal antibiotic resistance. Application of β-lactam antibiotics would then be effective against a previously resistant population.
We created a mathematical model of the CRISPRi system based on the network below, in which the bound complex has a repressive effect on YFP production. The purpose of the model was to examine the system for possible engineered improvements and to identify time-series repression data.
CRISPR Interference Network used to create Mathematical Model
The time series dynamics of the model showed that repression of YFP was acheivable, but incomplete. To improve our repression levels, we conducted a local and global sensitivity analysis of the model parameters. The sensitivity analysis informed us that changes to the mRNA degradation rate would have the greatest effect on overall repression efficiency.
The results of the sensitivity analysis inspired an investigation into silencing of the mRNA itself, i.e. RNAi.
sRNA: Silencing Translation
Gene regulation at the post-transcriptional level can be accomplished through RNA interference (RNAi) mechanisms (Aiba, 2007). RNAi occurs when bits of non-coding small interfering RNA (siRNA) - also known as microRNA (miRNA) - produced by the cell complements with targeted mRNA, regulating its translation (Shan, 2010). In bacteria, sRNA or small regulatory RNAs are members of a wide and divergent class of RNAi processes. Translation repression and/or target mRNA degradation occurs when sRNA binds in close proximity to the ribosomal binding site of its target mRNA (Yoo et al., 2013). This inhibits the initiation of translation by outcompeting the ribosome and subsequently leads to the destabilization of the target mRNA. Furthermore, in order for the sRNA to efficiently anneal its target mRNA in bacterial cells, a RNA chaperone protein, hfq, is required (Aiba, 2007). The translation repressing mechanism of hfq-dependent sRNA is shown in figure below.
We modelled the RNAi system to determine the amount of time it would take to repress YFP. The figure below is a time series graph of the concentration of YFP over time with our RNAi system.
The response of YFP concentration when sRNA is activated at time 0. After approximately 18hours, there is a 99.8% silencing of protein. Additionally, we did a sensitivity analysis of the network and determined that the system was relatively most sensitive to the YFP degradation rate and the YFP mRNA transcription rate. To read more on our analysis please see the sRNA Math Book
Design
Strain Considerations
Working with Staphylococcus aureus may produce more direct results for our project. However, S. aureus is a level 2 pathogen, and is thus a safety concern for a team of undergraduate students. To avoid this issue, we chose to conduct our research using Staphylococcus epidermidis, a level 1 organism. S. epidermidis is a close relative of S. aureus, and it can conjugate with populations of S. aureus (Forbes and Schaberg, 1983). Marraffini and Sontheimer (2009) used S. epidermidis strain ATCC 12228 in an experiment that studied the indigenous CRISPR system that exists in S. epidermidis. This particular strain has the ability to accept foreign DNA more readily than other S. epidermidis strains due to its lack of the CRISPR system (Marraffini and Sontheimer, 2009). The modified CRISPRi system therefore has a promising chance of success in ATCC 12228.
Target
In the case of MRSA, silencing the expression of mecA and its regulatory elements would create a population of antibiotic-sensitive MRSA (Meng et al., 2006; Hou et al., 2007; Sakoulas et al., 2001). To model silencing in MRSA, our team aimed to silence expression of a reporter gene, YFP, in S. epidermidis using CRISPRi and RNAi. YFP (yellow fluorescent protein) has been shown to be a reliable reporter in S. aureus (Malone et al., 2009). The coding sequence for YFP (BBa_K132310) was codon optimized for Staphylococcus aureus by Bose et al. (2013) and we had it sequenced by BioBasic to be controlled by the strong constitutive sarA P1 promoter (BBa_K1323021) and TIR RBS (BBa_K1323016). To model silencing of a chromosomal gene such as mecA as closely as possible, it is necessary to express YFP in single copy, ideally integrated into the genome of S. epidermidis. However, preliminary experiments were conducted using a low copy plasmid containing the YFP expression cassette.
CRISPRi
In order to test the CRISPRi system, we required a dCas9 protein and sgRNA molecules that were functional in S. epidermidis.
The dCas9 protein we used is from S. pyrogens and we codon optimized it for S. epidermidis. It was synthesized under a xylose inducible promoter (BBa_K1323014) and sodA RBS (BBa_K1323022) to create an expression cassette that was sent as a biobrick to the registry: BBa_K1323002.
We designed two sgRNA constructs that targeted 20 bp of the YFP gene on the non-template strand at positions 43-62 and 166-185 (BBa_K1323000 and BBa_K1323001 respectively). It has been shown that targets closer to the transcription start site are more effective at silencing a gene (Qi et al., 2013) and these were the first two sites proceeding a PAM site, a site necessary for dCas9 to bind. Both sgRNAs were put under the sarA P1 promoter (BBa_K1323021) and were designed according to the suggestions outlined in the 2013 paper by Larson et al. Besides the YFP target sequnce, they contain the following parts: a dCas9 handle hairpin which is necessary for dCas9 binding and an S. pyrogens terminator which mimics the natural RNA complex when crRNA and tracrRNA come together.
sRNA Interference
In order to test the RNAi system, we required an Hfq protein and sRNA molecules that were functional in S. epidermidis.
Synthetic sRNA contains two functional components: the target-binding sequence that is complementary to the target mRNA and the scaffold structure responsible for recruiting the Hfq protein (Yoo et al., 2013).
We have designed three sRNA constructs that will bind to different target sites at the positions shown in the figure below based on the works of Yoo et al. (2013). Our constructs were placed under the constitutive sarA promoter, similar to our other designs. All three target sites will be tested and the construct that results in the greatest decrease in fluorescence will have demonstrated the greatest silencing effect and will be used in our final design. The testing of the three sites will allow us to identify the optimal binding region on a mRNA which can be used to design the optimal sRNA to target genes associated with antibiotic resistance in MRSA.
Location of the target sites of three sRNA targets. On the mRNA, the orange region indicates the 3’ untranslated region which includes the RBS (underlined). The green region is the biobrick scar region and the yellow region is the YFP coding sequence, where the bolded bases indicate the start codon. The blue, pink, and purple sequences indicate the target-binding regions coded into sRNA construct 1 (BBa_K1323005), 2 (BBa_K1323006) and 3 (BBa_K1323007) respectively.
The scaffold used in our sRNA constructs is the scaffold of the MicC sRNA found in E.coli. As shown by Yoo et al.(2013), incorporation of the MicC scaffold in synthetic sRNA results in much higher repression compared to other scaffold regions (Yoo et al., 2013). The scaffold region used in our sRNA constructs is specific to the E coli Hfq proteins and will not bind the Hfq proteins found in Staphylococcus aureus and Staphylococcus epidermidis . Furthermore, S.aureus Hfq is proposed to be functionally inactive (Horstmann et al., 2012). Therefore, the Hfq protein we used is from E coli and we codon optimized it for S. epidermidis. It was synthesized under a xylose inducible promoter (BBa_K1323014) and sodA RBS (BBa_K1323022) to create an expression cassette that was sent as a biobrick to the registry: BBa_K1323004.
Assaying for YFP Repression
To test the efficiency of our CRISPRi and RNAi silencing systems, the RNA target binding molecule must be combined with their associated protein. Each CRISPRi sgRNA construct should be cloned with dCas9, and each RNAi sRNA construct should be cloned with Hfq. We also proposed constructs that contains both sgRNAs with dCas9 because it has been shown that having multiple sgRNAs that do not overlap greatly increases the efficiency of silencing (Larson et al., 2013).
An E coli and S. epidermidis shuttle vector is necessary for this assay so that parts can be constructed in E coli then tested in S. epidermidis. The part BBa_K1323017 (see figure below) was developed as an improvement of the pSB1C3 backbone by turning it into a shuttle vector. An erythromycin resistance gene for Staphylococcal organisms and a Staphylococcal origin of replication was cloned in between the pMB1 replication origin and VR regions. The RFP cassette (BBa_J04450) is present between the prefix and suffix of this backbone as a place holder.
A schematic of the E coli - Staphylococcus shuttle vector derived from pSB1C3 (blue parts)  (BBa_K1323017). The green parts indicate the Staphylococcal parts needed for selection and replication.
The CRISPRi and RNAi silencing constructs should be cloned into this shuttle vector and electroporated into S. epidermidis containing the YFP expression cassette. The repression of YFP can be observed by measuring and comparing the fluorescence of strains containing the silencing mechanisms to strains that do not.
Results and Future Work
Given that the proof of concept for the two silencing mechanisms in our project are designed to silence the YFP coding sequence, attaining a standard for fluorescence over time will give us a base from which to compare future silencing assays. It is assumed that the level of YFP fluorescence detected is directly proportional to the levels of YFP in the cell. Therefore, our YFP cassette was successful at producing YFP fluorescence.
A schematic of the E coli - Staphylococcus shuttle vector derived from pSB1C3 (blue parts)  (BBa_K1323017). The green parts indicate the Staphylococcal parts needed for selection and replication.
A graph of YFP fluorescence compared to E coli fluorescence - expected not to change over time - measured in one hour intervals.
A graph comparing the expression of sarA P1 promoter against J23101, the suggested reference promoter for promoter characterization.
sarA P1 Characterization
Both promoters were placed in front of a GFP generator (BBa_E0240) and the resulting amount of expression after 3 hours was compared. J23101 promoter expression was standardized to 1.00 and sarA P1 promoter expression had a value of 2.43 RFU suggesting that the sarA P1 promoter is 2.43 times as strong as the J23101.
E coli - Staphylococcus Shuttle Vector
The shuttle vector was constructed and confirmed through a restriction digest (see image below) as well as through sequencing.
A schematic of the E coli - Staphylococcus shuttle vector derived from pSB1C3 (blue parts)  (BBa_K1323017). The green parts indicate the Staphylococcal parts needed for selection and replication.
The most apparent result here is the resistance to erythromycin that the shuttle vector confers - transformed S. epidermidis is resistant to concentrations up to and including 100 ug/mL. Also, the chloramphenicol within the shuttle vector is in fact intended for use in E coli, and it is known that S. epidermidis contains an endogenous chloramphenicol resistance gene. It is therefore interesting to note that resistance to chloramphenicol was diminished when the E coli version was included. A further experiment could be devised to control for the effect of E coli chloramphenicol resistance on the endogenous version.
Preliminary RNAi silencing in E coli
Restriction enzyme digest with EcoRI and PstI.
This diagnostic gel shows that both the YFP cassette and the sRNA constructs were successfully co-transformed into DH5α. Expected sizes: 2.1 kb for the SB1A3 backbone, 900 bp for the YFP, and 230 bp (on average) for the varying sRNA constructs. The control sRNA and YFP cassette in pUC57 was not used for the RNAi assay*. The first colonies from each group of three was chosen for the actual silencing assay.
Testing the sRNAi system by measuring YFP fluorescence.
This graph shows YFP expression (normalized by optical density) for the samples tested. YFP expression was significantly lower in sRNA 2 and 3. Although the control sRNA targeted an intergenic non-coding region of the YFP, it seems it interfered with YFP expression levels. Therefore, these results suggest that the sRNA cassettes were successful at targeting and eliminating YFP mRNA.
Future Work
To further our understanding of the regulatory networks of the mecA cassette, we plan to modify the Type 4 mecA cassette by replacing the mecA gene with our YFP gene. This enables us to quantify the interaction of a mecA cassette and our silencing systems.
Project
Overview
Deliver
Silence
Translate
Safety
Outreach
Accomplishments
Biobricks
Collaborations
Checklist
Acknowledgements
Team
Our Team
Sponsors
Gallery
Notebook
Lab Book
Protocols
Math Book
Retrieved from " http://2014.igem.org/Team:Waterloo/Silence "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
