"
Page
Discussion
View source
History
teams
Log in
 
Team:Waterloo/Math Book/sRNA
From 2014.igem.org
Project ▾
Overview
Deliver
Silence
Translate
Safety
Notebook ▾
Lab Book
Protocols
Math Book
Outreach
Accomplishments ▾
Biobricks
Collaborations
Checklist
Acknowledgements
Team ▾
Our Team
Sponsors
Gallery
Math Book : Silencing RNA (sRNA)
Formation
Reduction
Parameters
Results
Conclusion
The ordinary differential equation model for ribonucleic acid interference (RNAi) gene silencing was formulated for the purpose of:
Model Formation
Model Reductions and Steady State Behaviour
Parameters
Results
Conclusion
Model Formation
Inspiration for the model came from the metabolic pathway reported in the literature by Abia in 2007 [17]. In the network, small RNA (sRNA) binds to Hfq, a chaperone protein which increases the binding rate between sRNA and its target mRNA substantially. Once bound, the Hfq-sRNA-mRNA complex is broken down by a degradosome, a specialized quaternary structure in sRNA-regulated gene expression.
At least, this is how the pathway works in E. coli. A major difficulty is that Hfq in S. aureus does not seem to play any major physiological role [18]. To make matters more difficult, the existence of a chaperone protein for sRNA in S. aureus has yet to be discovered [19]. Additionally, the proteins that make up the degradosome in E. coli are not present in S. aureus.
Our solution to these problems was to simple provide S. aureus the Hfq present in E. coli. In this way, a model of sRNA gene-regulation could be implemented to aid with laboratory design, and respond to the purposes of the model. Since Hfq would need to be expressed in the target cell, the reaction network took the form shown below.
Applying the usual mass action to the reaction network in the figure below, we arrive at the model equations:
Model Reduction and Steady State Behaviour
In our model equations presented previously, if we define the total amount of Hfq present in the cell as HT = H + Hs + Hms, we find:
In this way, the steady state concentration of Hfq is then:
Applying a quasi-steady state approximation on the last three equations in the model yields a system of linear equations:
Or, equivalently:
This system has least-squares solution:
We can then substitute these expressions into the first two equations of the model to (ultimately) arrive at a reduced model. After simplification:
Where, Vm = k3, K1 = k3/k2 and Km = (k-1k3) / (k1k2). We could use this simplified model to explore a phase space, however, it is much more valuable to explore the steady state behaviour of the model.
Inspired by [20], where the authors examined the steady state concentration of target mRNA exposed to sRNA regulation as a function of sRNA transcription, we also seek the steady state concentration of mRNA. The major difference is that the steady state expression of mRNA in this case will be controlled by two expressions, those of Hfq as well as sRNA, as opposed to simply sRNA. In our simplified model, it can be shown that this steady state concentration  of mRNA obeys the cubic equation:
Where:
The solutions of this equation describe how the expression rates of Hfq and sRNA control the steady state concentration of target mRNA.
Parameters
Our parameters, and their citations, are tabulated in the table below.
Parameter
Value
Description
Reference
αh
1/600 nM/s
transcription mRNA
Fender, A et al.
βh
2.31*10-3 s-1
degradation mRNA
Hussein, R. & Lim, H.N.
αH
0.9 s-1
degradation rates of Hfq mRNA, sRNA, mRNA
Hussein, P.S. et al.
αs
1/600 nM/s
transcription sRNA
Fender, A et al.
αm
1/600 nM/s
transcription Hfq mRNA
Fender, A et al.
αM
1/600 s-1
translation of the mRNA
Fender, A et al.
βHm, βs, βm
2.31*10-3 s-1
degradation rates of Hfq mRNA, sRNA, mRNA
Swain, P.S. et al.
βH, βM, βHs, βHms
6.42*10-5 s-1
Degradation rate of Hfq and Target Protein (YFP)
<
Swain, P.S. et al.
βms
2.31*10-2 s-1
transcription mRNA
?
k1
10-3 (nM*s)-1
association constant of Hfq and sRNA
Fender, A et al.
k-1
0.7*10-4 s-1
dissociation constant of Hs to Hfq and sRNA
Fender, A et al.
k2
3.5*106 (nM*s)-1
association constant of Hs and mRNA
Fender, A et al.
k3
0.7*10-4 s-1
dissociation constant of Hms, assuming that the Hfq-sRNA binding to the MicC region is independent of the sRNA, mRNA binding. Therefore, k-1=k3
Fender, A et al.
k4
? (nM*s)-1
production rate of Target mRNA-sRNA complex
???
Results
The resulting time-history of the concentration of Yellow Fluorescent Protein (YFP), the target protein in this case, when sRNA gene-silencing is introduced is displayed in the figure below. As can be seen from the graph, the target protein exhibits an exponential decay until it reaches an almost negligible steady state. The approximate time it takes to do this is on the order of 18 hours, which is an artefact of the half life of the protein. The small RNA and Hfq in the cell effectively destroy YFP’s mRNA, turning off expression, forcing the protein concentration dynamics to be mostly governed by the decay.
The response of YFP concentration when sRNA is activated at time 0. After approximately 18hours, there is a 99.8% silencing of protein.
In using Equation 6 to construct a surface relating sRNA and Hfq transcription to the steady state concentration of target mRNA, we generate the surface pictured in the figure below.
The figure above seems to indicate, qualitatively, that the steady state concentration of target mRNA is much more sensitive to changes in Hfq expression than sRNA expression. To explore this relationship, we performed a sensitivity analysis on the model to each of the parameters.
Metabolic Control Analysis
Since a major purpose of this model was to elucidate potential avenues for silencing a target protein, our Metabolic Control Analysis focused primarily on how the parameters of the model and affect the steady state concentration of the target protein (in this case YFP). Listed to the right are the Concentration Control Coefficients for the steady state concentration of YFP for each parameter in the model:
The most notable of these are the concentration control coefficients for the following parameters: βM the degradation rate of YFP), αm (the transcription rate of the YFP mRNA transcript). These relatively large flux control coefficient values demonstrate the system is highly sensitive to changes in these variables. Therefore, these would make particularly good choices for synthetic intervention, as they would induce the largest changes on the steady state concentration of YFP. In order for sRNA gene suppression to be most efficient, a minimal amount of YFP-flux through the system is desirable. However, since we want to use this sRNA system to control a protein whose degradation rate and transcription rate that we could not physically change, we need to turn our attention to some other potential confounding factors.
Some other notable concentration control coefficients are αs, αM, K-1. These constants are respectively the rate of transcription of sRNA, the rate constant for translation of YFP, and the rate constant of dissociation of Hfq-sRNa to Hfq and sRNA. These rates have the least impact on the system and are not good targets for optimization of sRNA.
Conclusion
Using parameters from the literature we were able to construct a model of sRNA gene repression in Staphylococcus aureus, using Hfq from E. coli. In the subsequent analysis of the model, the relationships between the expression rate of sRNA, Hfq, and the subsequent steady state concentration of a target protein, YFP, was elucidated.
The model was successful in providing an estimate of the amount of suppression as well as the approximate amount of time until maximum suppression was obtained. Additionally, in the Metabolic Control Analysis, we were able to deduce the best regions of the metabolic pathway to target in order to reduce the steady state concentration of YFP; unfortunately, however, these ended up being parameters outside of our control.
In the end, the model of sRNA repression was able to inform the conjugation model of the magnitude and temporal characteristics of sRNA gene regulation.
References
[1]D. Bikard et al. “Programmable repression and activation of bacterial gene expression using an engineered CRISPR-Cas system”. In: Nucleic Acids Res. 41.15 (Aug. 2013), pp. 7429–7437.
[2]Florian Brandt et al. “The Native 3D Organization of Bacterial Polysomes”. In: Cell 136.2 (2009), pp. 261 –271. issn: 0092-8674. doi: 10.1016/j.cell.2008.11.016 .
[3]A. G. Cheng, D. Missiakas, and O. Schneewind. “The giant protein Ebh is a determinant of Staphylococcus aureus cell size and complement resistance”. In: J. Bacteriol. 196.5 (2014), pp. 971–981.
[4]A. L. Cheung, K. Nishina, and A. C. Manna. “SarA of Staphylococcus aureus binds to the sarA promoter to regulate gene expression”. In: J. Bacteriol. 190.6 (Mar. 2008), pp. 2239–2243.
[5]G. Domingue, J. W. Costerton, and M. R. Brown. “Bacterial doubling time modulates the effects of opsonisation and available iron upon interactions between Staphylococcus aureus and human neutrophils”. In: FEMS Immunol. Med. Microbiol. 16.3-4 (Dec. 1996), pp. 223–228.
[6]S. Michalik et al. “Life and death of proteins: a case study of glucose-starved Staphylococcus aureus”. In: Mol. Cell Proteomics 11.9 (Sept. 2012), pp. 558–570.
[7]R. Milo et al. “BioNumbers-the database of key numbers in molecular and cell biology”. In: Nucleic Acids Res. 30 (Jan. 2010), pp. D750–D753. url: http://bionumbers.hms.harvard.edu/bionumber.aspx?id=107869 }.
[8]L. S. Qi et al. “Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression”. In: Cell 152.5 (Feb. 2013), pp. 1173–1183.
[9]C. Roberts et al. “Characterizing the effect of the Staphylococcus aureus virulence factor regulator, SarA, on log-phase mRNA half-lives”. In: J. Bacteriol. 188.7 (Apr. 2006), pp. 2593–2603. doi: 10.1128/JB.188.7.2593-2603.2006
[10]Marlena Siwiak and Piotr Zielenkiewicz. “Transimulation - Protein Biosynthesis Web Service”. In: PLoS ONE 8.9 (Sept. 2013), e73943. doi: 10.1371/journal.pone.0073943.
[11]S.H. Sternberg et al. “DNA interrogation by the CRISPR RNA-guided endonuclease Cas9”. In: Nature 7490 (2014), 6267. doi: 10.1038/nature13011 . url: http://www.nature.com/nature/journal/v507/n7490/full/nature13011.html .
[12]Freiburg iGEM Team. dCas9. BBa K1150000 Standard Biological Part. 2013. url: http://parts.igem.org/Part:BBa_K1150000 .
[13]UCSF iGEM Team. Operation CRISPR: Decision Making Circuit Model. 2013. url: https://2013.igem.org/Team:UCSF/Modeling .
[14]Jian-Qiu Wu and Thomas D. Pollard. “Counting Cytokinesis Proteins Globally and Locally in Fission Yeast”. In: Science 310.5746 (2005), pp. 310–314. doi: 10.1126/science.1113230 .
[15]Jianfang Jia and Hong Yue. “Sensitivity Analysis and Parameter Estimation of Signal Transduction Pathways Model”. In: Proceedings of the 7th Asian Control Conference (Aug. 2009), pp. 1357–1362.
[16]Fi-John Chang and J. W. Delleur. “Systematic Parameter Estimation Of Watershed Acidification Model”. In: Hydrological Processes 6. (1992), pp. 29–44. doi: 10.1002/hyp.3360060104 .
[17]Aiba, H. (2007). Mechanism of RNA silencing by Hfq-binding small RNAs. Current opinion in microbiology, 10 (2), 134-139.
[18]Horstmann, N., Orans, J., Valentin-Hansen, P., Shelburne, S. A., & Brennan, R. G. (2012). Structural mechanism of Staphylococcus aureus Hfq binding to an RNA A-tract. Nucleic acids research, gks809.
[19]Eyraud, A., Tattevin, P., Chabelskaya, S., & Felden, B. (2014). A small RNA controls a protein regulator involved in antibiotic resistance in Staphylococcus aureus. Nucleic acids research, gku149.
[20]Shimoni, Y., Friedlander, G., Hetzroni, G., Niv, G., Altuvia, S., Biham, O., & Margalit, H. (2007). Regulation of gene expression by small non‐coding RNAs: a quantitative view. Molecular Systems Biology, 3 (1)
[21]Fender, A., Elf, J., Hampel, K., Zimmermann, B., & Wagner, E. G. H. (2010). RNAs actively cycle on the Sm-like protein Hfq. Genes & Development, 24 (23),2621-2626.
[22] Swain, P. S. (2004). Efficient attenuation of stochasticity in gene expression through post-transcriptional control. Journal of molecular biology, 344 (4),965-976.
[23] Hussein, R., & Lim, H. N. (2012). Direct comparison of small RNA and transcription factor signaling. Nucleic acids research, 40 (15), 7269-7279.
[24] Levin, B.R., Stewart, F.M. and Rice, V.A. 1979. “The Kinetics of Conjugative Plasmid Transmission: Fit of a Simple Mass Action Model.” In: Plasmid. 2. pp. 247-260.
[25]Projan, S.J. and Archer, G.L. 1989. “Mobilization of the Relaxable Staphylococcus aureus Plasmid pC221 by the Conjugative Plasmid pGO1 Involves Three pC221 Loci.” In: Journal of Bacteriology. pp. 1841-1845.
[26]Phornphisutthimas, S., Thamchaipenet, A., and Panijpan, B. 2007. “Conjugation in Escherichia coli.” In: The International Union of Biochemistry and Molecular Biology. 35. 6. pp. 440-445.
[27]Phornphisutthimas, S., Thamchaipenet, A., and Panijpan, B. 2007. “Conjugation in Escherichia coli.” In: The International Union of Biochemistry and Molecular Biology. 35. 6. pp. 440-445.
[28]P Chung P., McNamara P.J., Campion J.J., Evans M.E. 2006. “Mechanism-based pharmacodynamic models of fluoroquinolone resistance in Staphylococcus aureus.” In: In:  Antimicrobial Agents Chemotherapy. 50. pp. 2957-2965.
[29] Chang H., Wang L. “A Simple Proof of Thue's Theorem on Circle Packing” In: arXiv:1009.4322v1.
Project
Overview
Deliver
Silence
Translate
Safety
Outreach
Accomplishments
Biobricks
Collaborations
Checklist
Acknowledgements
Team
Our Team
Sponsors
Gallery
Notebook
Lab Book
Protocols
Math Book
Retrieved from " http://2014.igem.org/Team:Waterloo/Math_Book/sRNA "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
