"
Page
Discussion
View source
History
teams
Log in
 
Team:Waterloo/Modeling/Silencing
From 2014.igem.org
(Redirected from Team:Waterloo/Modeling/CRISPRi )
CRISPR Interference Model
Motivation
We decided to create a model of the CRISPR system for two main reasons:
Identifying the parts of the network that could be targeted by our lab team to improve repression efficiency
To approximate time-series mecA repression data for use in modelling the overall vulnerability of a S. aureus population
Model Construction
After a literature review we were able to construct the CRISPR interference system network. The targeted single guide RNA (sgRNA) associates with nuclease-deficient Cas9 protein (dCas9) to form a complex that binds with the DNA complementary to the sgRNA target
. The bound complex prevents transcription elongation by RNA polymerase, repressing YFP mRNA expression
. The chemical network is shown below:
IMG: reaction netowrk
Using standard mass-action kinetics, the network simplifies into the following set of differential equations:
Differential System Here
The choice of a largely first-order model is supported by findings of several recent studies
. To simplify the model further, we made a quasi-steady-state assumption (QSSA) about the formation of  of the dCas9-sgRNA complex b. That is, we assume that dCas9 and sgRNA associate on a faster timescale than the other reactions (i.e. transcription, translation and the binding of the complex to the DNA). Therefore, we disregard the kinetics of the complex formation reaction and assume that it is always at steady-state relative to the other time-dependent species concentrations.  Under the QSSA, the concentration of the complex is given by:  see latex 4 equation  This is the same assumption made by previous teams
. Our model then simplifies to:  see latex 4 equation
Modeling Incomplete Repression
A recent study by Bikard et al
. found that maximal repression (on the order of 100 fold) was achieved when the promoter was targeted. However, targeting the promoter is not viable in this project since an essential promoter from elsewhere in the genome has been harnessed to produce the fluorescent promoter. Instead, we model the incomplete repression (ranging from 6-fold to 35-fold) observed when the off-promoter regions, specifically on the non-coding strand, are targeted.
There are two possible approaches for modelling the incomplete repression, each reflecting a different physical mechanism that allows leaky YFP expression. In the first mechanism, RNA polymerase is sometimes able to cleave the bound dCas9-sgRNA complex from the DNA. In the second mechanism, the complex binds inefficiently and is sometimes separated from the DNA, permitting transcription to continue.
We assumed that the incomplete repression is accounted for by the first mechanism. This assumption was based on several studies
showing radically different repression rates if the complex targets the promoter, preventing transcription initiation, rather than targeting the DNA further downstream and impeding transcription elongation. The differences in the system behaviour depending on whether or not RNA polymerase has the opportunity to bind suggest that the "cleavage" mechanism may more closely resemble the chemical reality.
Consequently, we modelled incomplete repression using a leaky expression term proportional to the expected YFP expression when the complex is saturated. The differential equation model was updated with the following term: When the concentration of the complex is zero, YFP mRNA is produced at the rate expected from the unrepressed sarA promoter. At a large concentration of the complex, the YFP mRNA is produced at only 60% of the possible rate from sarA.
Parameter Search and Fitting
We turned to the literature to find parameters for our model. We first looked for exact parameter values in S. aureus. If these could not be found, we next looked for ways to to estimate the parameters using other available data for S. aureus and finally searched for the parameters in other gram-positive bacteria. Aggregating parameters from many experiments across the literature is by nature a somewhat uncertain endeavour, but those parameters about which we are very uncertain are marked with asterisks. A general rationale is given for each parameter, but details on the more circuitously estimated parameters are given after the table.
Parameter
Value
Description
Source/Rationale
0.0011 nM min-1
mRNA production from sarA P1 Promoter
Determined based on linear fitting to the time-series fluorescence measurements from YFP/P2-P3-P1 fusion, as reported in
and fluorescence per molecule from
The only model parameters without some basis in the literature are the association rates for dCas9 and sgRNA. However, since the model is based on the QSSA that those dynamics are much faster than the others in the model, we were able to define a range for those parameters based on the other.  Details on the more roundabout estimations are given below:
dCas9 Production Rates from mRNA
We were unable to find a peptide chain elongation rate for S. aureus, so instead we used the values reported in BioNumber 107869
which gives a range of 0.59-3.17 amino acids per second per ribosome in Streptomyces coelicolor, another gram-positive bacteria. Freiburg's dCas9 part from last year
is composed of 1372 amino acids. This translates to a range of 0.0258 to 0.1386 dCas9 molecules per minute per ribosome. We were unable to find ribosome densities in S. aureus, but found two different estimates for ribsosome density in Bionumbers: 0.22 ribosomes per 100 codons (i.e. per 3 nt coding sequence)
and 3.46 ribosomes per 100 codons
. Using our assumption of 3 nt:1 amino acid, we then multiply to get the 0.0057-0.4797 range of dCas9 molecules per minute.
Degradation Rates of dCas9
We were unable to find any specific data on dCas9 degradation, so instead we used a protein half-life of sarA measured in S. Aureus by Michelik et al.
. We chose sarA rather than a protein more chemically similar to dCas9 because data on sarA was readily available and because dCas9 is transcribed using the sarA promoter, which allows us to at least capture sensitivity of the degradation rate to production.
Production Rates from the sarA Promoter
We used the time-series data given by Cheung et al.
to estimate the rate of production from the sarA P2-P3-P1 promoter in S. aureus. The figure from their paper is reproduced below. After diluting 1:100, the S. aureus strains were serially monitored for OD_650. We used data from the sarA+ strain, as that's more like a wild-type S. aureus strain.  Using the laboratory-conditions doubling time of 24 minutes given in given in
, we found that the bacteria would re-enter stationary phase after 2.5 hours; for time-points after 3 hours, the number of number of sarA genes producing fluorescence could be assumed as constant. For this reason, we excluded time-points prior to 3 hours. We then converted from fluorescence units to number of fluorescent molecules using the quantization measurements provided by Wu & Pollard
and, using our assumption of a fixed number of active sarA genes, considered the relative change in number of molecules to be representative of the per-promoter rate.  We were interested, however, in the changes of concentration rather than the changes in the raw number of molecules. As the name suggests, Staphylococcus aureus are spherical in shape. Assuming that all S. aureus are spheres, the volume of the cell can be determined. The diameter of a USA300 S. aureus cell was previously measured as 1.1 μ•m
resulting in the overall cell volume to be calculated as 5.575•10-15 L. The number of molecules were thus converted to units of molar concentration in the cell, specifically nanomoles per litre (nM). The exponential fit used to find the rate constant is shown below:  This resulted in a exponential model a•ebt with a b rate constant of 0.0011 nM/min.
References
[1]D. Bikard et al. “Programmable repression and activation of bacterial gene expression using an engineered CRISPR-Cas system”. In: Nucleic Acids Res. 41.15 (Aug. 2013), pp. 7429–7437.
[2]Florian Brandt et al. “The Native 3D Organization of Bacterial Polysomes”. In: Cell 136.2 (2009), pp. 261 –271. issn: 0092-8674. doi: 10.1016/j.cell.2008.11.016 .
[3]A. G. Cheng, D. Missiakas, and O. Schneewind. “The giant protein Ebh is a determinant of Staphylococcus aureus cell size and complement resistance”. In: J. Bacteriol. 196.5 (2014), pp. 971–981.
[4]A. L. Cheung, K. Nishina, and A. C. Manna. “SarA of Staphylococcus aureus binds to the sarA promoter to regulate gene expression”. In: J. Bacteriol. 190.6 (Mar. 2008), pp. 2239–2243.
[5]G. Domingue, J. W. Costerton, and M. R. Brown. “Bacterial doubling time modulates the effects of opsonisation and available iron upon interactions between Staphylococcus aureus and human neutrophils”. In: FEMS Immunol. Med. Microbiol. 16.3-4 (Dec. 1996), pp. 223–228.
[6]S. Michalik et al. “Life and death of proteins: a case study of glucose-starved Staphylococcus aureus”. In: Mol. Cell Proteomics 11.9 (Sept. 2012), pp. 558–570.
[7]R. Milo et al. “BioNumbers-the database of key numbers in molecular and cell biology”. In: Nucleic Acids Res. 30 (Jan. 2010), pp. D750–D753. url: http://bionumbers.hms.harvard.edu/bionumber.aspx?id=107869 }.
[8]L. S. Qi et al. “Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression”. In: Cell 152.5 (Feb. 2013), pp. 1173–1183.
[9]C. Roberts et al. “Characterizing the effect of the Staphylococcus aureus virulence factor regulator, SarA, on log-phase mRNA half-lives”. In: J. Bacteriol. 188.7 (Apr. 2006), pp. 2593–2603. doi: 10.1128/JB.188.7.2593-2603.2006
[10]Marlena Siwiak and Piotr Zielenkiewicz. “Transimulation - Protein Biosynthesis Web Service”. In: PLoS ONE 8.9 (Sept. 2013), e73943. doi: 10.1371/journal.pone.0073943.
[11]S.H. Sternberg et al. “DNA interrogation by the CRISPR RNA-guided endonuclease Cas9”. In: Nature 7490 (2014), 6267. doi: 10.1038/nature13011 . url: http://www.nature.com/nature/journal/v507/n7490/full/nature13011.html .
[12]Freiburg iGEM Team. dCas9. BBa K1150000 Standard Biological Part. 2013. url: http://parts.igem.org/Part:BBa_K1150000 .
[13]UCSF iGEM Team. Operation CRISPR: Decision Making Circuit Model. 2013. url: https://2013.igem.org/Team:UCSF/Modeling .
[14]Jian-Qiu Wu and Thomas D. Pollard. “Counting Cytokinesis Proteins Globally and Locally in Fission Yeast”. In: Science 310.5746 (2005), pp. 310–314. doi: 10.1126/science.1113230 .
[15]Jianfang Jia and Hong Yue. “Sensitivity Analysis and Parameter Estimation of Signal Transduction Pathways Model”. In: Proceedings of the 7th Asian Control Conference (Aug. 2009), pp. 1357–1362.
[16]Fi-John Chang and J. W. Delleur. “Systematic Parameter Estimation Of Watershed Acidification Model”. In: Hydrological Processes 6. (1992), pp. 29–44. doi: 10.1002/hyp.3360060104 .
[17]Aiba, H. (2007). Mechanism of RNA silencing by Hfq-binding small RNAs. Current opinion in microbiology, 10 (2), 134-139.
[18]Horstmann, N., Orans, J., Valentin-Hansen, P., Shelburne, S. A., & Brennan, R. G. (2012). Structural mechanism of Staphylococcus aureus Hfq binding to an RNA A-tract. Nucleic acids research, gks809.
[19]Eyraud, A., Tattevin, P., Chabelskaya, S., & Felden, B. (2014). A small RNA controls a protein regulator involved in antibiotic resistance in Staphylococcus aureus. Nucleic acids research, gku149.
[20]Shimoni, Y., Friedlander, G., Hetzroni, G., Niv, G., Altuvia, S., Biham, O., & Margalit, H. (2007). Regulation of gene expression by small non‐coding RNAs: a quantitative view. Molecular Systems Biology, 3 (1)
[21]Fender, A., Elf, J., Hampel, K., Zimmermann, B., & Wagner, E. G. H. (2010). RNAs actively cycle on the Sm-like protein Hfq. Genes & Development, 24 (23),2621-2626.
[22] Swain, P. S. (2004). Efficient attenuation of stochasticity in gene expression through post-transcriptional control. Journal of molecular biology, 344 (4),965-976.
[23] Hussein, R., & Lim, H. N. (2012). Direct comparison of small RNA and transcription factor signaling. Nucleic acids research, 40 (15), 7269-7279.
[24] Levin, B.R., Stewart, F.M. and Rice, V.A. 1979. “The Kinetics of Conjugative Plasmid Transmission: Fit of a Simple Mass Action Model.” In: Plasmid. 2. pp. 247-260.
[25]Projan, S.J. and Archer, G.L. 1989. “Mobilization of the Relaxable Staphylococcus aureus Plasmid pC221 by the Conjugative Plasmid pGO1 Involves Three pC221 Loci.” In: Journal of Bacteriology. pp. 1841-1845.
[26]Phornphisutthimas, S., Thamchaipenet, A., and Panijpan, B. 2007. “Conjugation in Escherichia coli.” In: The International Union of Biochemistry and Molecular Biology. 35. 6. pp. 440-445.
[27]Phornphisutthimas, S., Thamchaipenet, A., and Panijpan, B. 2007. “Conjugation in Escherichia coli.” In: The International Union of Biochemistry and Molecular Biology. 35. 6. pp. 440-445.
[28]P Chung P., McNamara P.J., Campion J.J., Evans M.E. 2006. “Mechanism-based pharmacodynamic models of fluoroquinolone resistance in Staphylococcus aureus.” In: In:  Antimicrobial Agents Chemotherapy. 50. pp. 2957-2965.
[29] Chang H., Wang L. “A Simple Proof of Thue's Theorem on Circle Packing” In: arXiv:1009.4322v1.
Retrieved from " http://2014.igem.org/Team:Waterloo/Modeling/Silencing "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
