"
Page
Discussion
View source
History
teams
Log in
 
Team:Newcastle/Modeling/Population
From 2009.igem.org
(Redirected from Team:Newcastle/Modelling/Population )
Home
Team
Project
Parts
Modelling
Sponsors
Lab Book
Project
Overview
Choices Rationale
Characterisation
Safety
Judging Comments
Sub-projects
Metal Intake/Efflux
Metal Sensing
Sporulation Tuning
Population Dynamics
Stochastic Switch
Metal Sequester
Chassis
Promoter Library
Wet Lab
Lab Book
Protocols
Meetings
Meetings Calendar
Planning
Timeline
Ideas
Links
Helping other teams
Ethics
T-Shirts
iGEM Meetup
Our City
Fun and Games
Sponsors
Useful Links
Contact Us
Press Coverage
Contents
1 Population modelling
1.1 Distributed Computing
1.2 Code
1.2.1 Pseudo code
1.3 Results
1.3.1 Visualisation
1.3.2 Further work
1.4 References
1.5 External links
Population modelling
A simple view of our bacteria's possible states. Note how when the sequestering spore state is reached, the cell cannot germinate back to being a vegetative cell.
One of the models which we have produced is one concerning the population numbers of our bacteria. It looks at how our additions to the DNA may affect the growth of the bacterial population.
This model has been implemented in the Java programming language, and connects to other models which we have written in CellML, using the JSim application. Due to the design of the program, the team has enlisted the help of some very powerful computers, and final iteration of the program has been to use distributed computing and the "Amazon Cloud" to extend the size of the bacterial population.
Distributed Computing
Distributed Computing (also known as Cloud or Grid computing) is the use of multiple computing processors, which can be distributed around the world, to accomplish a single given task. The processes on the machines may also communicate with each other, instead of simply acting independently.
Due to early versions of our bacterial population simulation program using a lot of CPU time and RAM, we attempted to think of ways to be able to speed up the processing times and expand the simulation. Although these early simulations were fairly small scale, each bacterial cell runs as an independent thread, so every system that we ran it on began to slow down dramatically.
Newcastle's School of Computing Science Integrative Bioinformatics Research Group has developed a software system for grid computing applications in the scientific field, called Microbase .  The Microbase project is a grid-based system that uses web-service-based technologies to integrate distributed components and ensure that they interoperate.[3] Funded by the BBSRC and the DTI, Microbase has been used in Microbial Genome Comparison and Analysis.[1] Microbase works by splitting a task into a number of 'Jobs' and managing how these Jobs are run on grid-based systems.[2]
Keith Flanagan, a Researcher at Newcastle University and member of the Microbase Project, suggested that Microbase could aid us in the development of a distributed version of our bacterial population simulation. Using Microbase meant that almost all of the programming concerned with the smooth running of our modelling was simple to carry out, and we were easily able to convert our single machine program to a distributed one.
Using Microbase we are running our simulation on the Amazon Elastic Compute Cloud, a service provided by the e-commerce company Amazon.com, which gives users access to a large number of powerful machines. The development of Microbase for use on Amazon's web services has been a part of the Microbase project to provide further processing capacity.[4] Prior to running our simulations using Amazon's computers, we tested our program using some of the machines in some of the University's computer clusters.
Code
Advice about the code for our population simulation model is available on request. Please the model requires specific libraries, database setup and Microbase setup, etc to build and run correctly. Please email Morgan Taschuk , one of our advisors, and visit the Microbase homepage for more information.
The Java source code for our model is available here:
Image:Newcastle Distributed Microbase Population src.zip
Pseudo code
Please click here to see an overview of our population simulation.
Results
This is an example graph created using some of the output from a small run of the population model, with a maximum of 27 cells of bacteria.
The results from various runs of our population simulation have been fed back into the development of other parts of our project, namely tuning sporulation. We have used the outputs to more closely determine how much of the population we can allow to become non-germinating metal sequestering spores.
Visualisation
Alongside the actual simulation we are developing an application to visualise the output of our simulation in a three dimensional environment.
Below is a basic visualisation of one of the early versions of our simulation, where the different states of the bacterial cells are represented by differing colours. This early graphic was produced by feeding the output of our population simulations into another program which we have developed in Java, using the Java3D technology.
In this animation, a population of cells is seen to proliferate in three dimensions from a single cell. Vegetative cells are represented by white spheres. There is cadmium in the environment (not shown), and also sufficient food to allow proliferation. Cells can make choices either to grow and proliferate, or to sporulate due to the cadmium. If cells choose to sporulate, they make a further choice whether or not to sequester cadmium. Since there is sufficient food in the environment, ordinary spores (represented in black) germinate immediately after sporulation. Spores that cannot germinate because they have sequestered cadmium are shown as red spheres. Germinating spores (that have previously been non-sequestering spores) are shown in green.
Also below is an image of the more sophisticated graphical output which we have been producing. The Graphics package Autodesk 3ds Max has been used in the production of these graphics.
Further work
The current version of the model provides the basis of a bacterial population simulator, which can be improved and adapted for use by other people and projects in the future. Currently parts of the model are highly focused on our project, but it can easily be generalised. We have developed a framework for integrating agent based and biochemical models on a distributed grid system. The way in which the bacterial threads run in the program makes it very easy to customise and improve the behaviour of the bacteria throughout their simulated life cycles.
References
Microbase in the Wild , Microbase. Newcastle University. Accessed on 21 October 2009.
A Grid-based System for Microbial Genome Comparison and Analysis, the 5th IEEE International Symposium on Cluster Computing and the Grid , (CCGrid 2005), Cardiff, UK, May 9-12 2005. Yudong Sun, Anil Wipat, Matthew Pocock, Pete A. Lee, Paul Watson, Keith Flanagan and James T. Worthington. Accessed on 21 October 2009.
Exploring Microbial Genome Sequences to Identify Protein Families on the Grid , IEEE Transactions on Information Technology in Biomedicine, vol. 11 (4), July 2007. Yudong Sun, Anil Wipat, Matthew Pocock, Peter A. Lee, Keith Flanagan, and James T. Worthington. Accessed on 21 October 2009.
Developing Grid-based Systems for Microbial Genome Comparisons: The Microbase Project , UK e-Science All Hands Meeting 2004. Nottingham, August 31 - September 3, 2004. Anil Wipat, Yudong Sun, Matthew Pocock, Pete Lee, Paul Watson and Keith Flanagan. Accessed on 21 October 2009.
External links
Amazon Elastic Compute Cloud
Microbase
News
Events
20 – 21 June 2009 - Europe workshop (London)
23 – 24 June 2009 - UK iGEM meetup (Edinburgh)
23 October Practice Presentation (Newcastle)
23 October T-shirts are ready
27 October Practice Presentation (Sunderland)
27 October Poster is ready
30 October – 2 November 2009 - Jamboree (Boston)
Social Net
Newcastle iGEM Twitter
Newcastle on Facebook
Newcastle Youtube Channel
Retrieved from " http://2009.igem.org/Team:Newcastle/Modeling/Population "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
