"
Page
Discussion
View source
History
teams
Log in
 
Team:Newcastle/SporulationTuning
From 2009.igem.org
Home
Team
Project
Parts
Modelling
Sponsors
Lab Book
Project
Overview
Choices Rationale
Characterisation
Safety
Judging Comments
Sub-projects
Metal Intake/Efflux
Metal Sensing
Sporulation Tuning
Population Dynamics
Stochastic Switch
Metal Sequester
Chassis
Promoter Library
Wet Lab
Lab Book
Protocols
Meetings
Meetings Calendar
Planning
Timeline
Ideas
Links
Helping other teams
Ethics
T-Shirts
iGEM Meetup
Our City
Fun and Games
Sponsors
Useful Links
Contact Us
Press Coverage
Sporulation Tuning
Introduction
The bacterium Bacillus subtilis used in our project is a gram-positive soil bacterium which, under certain conditions, commits itself to a developmental pathway leading to the production of spores.[1] In this part of our project, we aim to control sporulation in our bacterial population, so that we can decide how much of the population becomes spores, and how much continue as vegetative cells. Should the cell sporulate, it becomes a ‘metal container’ , trapping the sequestered cadmium in its spore.
After the cell sequesters cadmium into its spore, it should not germinate, or the sequestered cadmium will be released back into the environment. Therefore, the chassis comes into play, where the sleB and cwlJ germination-defective mutants are put into use.
In order to control sporulation, our team proposed the idea of inducing the synthesis of kinA, with IPTG as a sporulation initiation signal.
KinA is a major kinase which provides phosphate input to a phosphorelay, which in turn, activates the sporulation pathway upon starvation via the phosphorylated Spo0A transcription factor,[2] which governs entry into the sporulation pathways of the bacterium Bacillus subtilis.[3]
... Click to read more ...
Novelty in this sub-project
In this sub-project, instead of allowing the cell to decide whether or not to sporulate, we influence its decision. In order to execute our plan, we used the concentration of kinA, induced by IPTG, to control sporulation.
This sub-project consists of two main models: the Sporulation Tuning and the Sin Operon models. These two models are meant to work together, as mentioned below in the modelling section , with the Sporulation Tuning model controlling sporulation, and the Sin Operon model repressing sporulation, creating a more realistic model.
Modelling
The Sin (sporulation inhibition) Operon Model was one of the earlier models built. As its name suggests, it models the repression of sporulation. The Sin Operon Model was built in CellML.
The second model built was the simple model of KinA expression. After satisfactory results were obtained, the sporulation phosphorelay was modelled into the KinA Expression Model, and is known as the Sporulation Tuning Model. Both the KinA Expression and Sporulation Tuning models were built using COPASI.
The Sin Operon and Sporulation Tuning model work hand in hand as components of the Population Dynamics model .
KinA Expression Model
Under normal conditions, LacI represses kinA.
However, in the presence of IPTG, KinA can be expressed, as IPTG binds to lacI, deactivating it. Equations (a) and (b) describes how IPTG binds to LacI, forming LacI*, which is the deactivated form of LacI
Click to view more of the KinA Expression Model equations
Results
Click to view the KinA Expression results
Sporulation Tuning Model
The expression of KinA has been modelled as seen above, therefore we can now proceed further into the Sporulation Tuning Model, which is built from the KinA Expression Model using COPASI.
To proceed with the modelling of our Sporulation Tuning Model, we have decided that in response to an unidentified stimulus, where KinA  autophosphorylates and then donates its phosphate groups to the response regulator Spo0F, the unidentified stimuli will be termed as 'sporulation signal'.[5]
The following equations describe the model:
Equation 1
Equation 2
Equation 3
Click to view more of the Sporulation Tuning Model equations
Results
Click to view the Sporulation Tuning results
Sin (sporulation inhibition) Operon Model
In order to create a more realistic model of our sporulation system the team has decided to include the Sin (sporulation inhibition) Operon Model, which the team designed in CellML.
The sin operon controls the production and activity of the repressor SinR, which in its active tetrameric form, inhibits sporulation by repressing stage II and spo0A promoters. On the other hand, the accumulation of Spo0A~P induces the expression of SinI, which binds to and inactivates SinR.
Diagram 1: Simplifed Schematic of the sin Operon[9]
Click to view more of the Sin Operon Model equations
Results
Click to view the Sin Operon Model results
BioBrick constructs
The BioBrick we have designed is to contain an IPTG inducable kinA gene, using pSpac, allowing us to test the theory about KinA in the lab.
BBa_K174010
KinA
Length: 1818bp
Click here for more information on this part.
BBa_K174011
IPTG inducable KinA sporulation trigger
Length: 1953bp
Click here for more information on this part.
Lab Work Strategies
The lab work executed was be to induce sporulation in the presence of IPTG, and this involved microscopy and testing cultures for sporulation (e.g. by their heat resistance).
To find out more about our lab work strategies,
see Lab Work Strategies
References
[1] Predich, M., Nair, G., Smith, I. (1992) Bacillus subtilis Early Sporulation Genes kinA, spo0F, and spo0A Are Transcribed by the RNA Polymerase Containing σH. Journal of Bacteriology. Pp 2771-2778
[2] Veening, J-W., Smits, W. K., Kuipers, O. P. (2008) Bistability, Epigenetics, and Bet-Hedging in Bacteria. Annu. Rev. Microbiol. 62: 193-210
[3] Fujita, M., Losick, R. (2005) Evidence that Entry into Sporulation in Bacillus subtilis is Governed by a Gradual Increase in the Level and Activity of the Master Regulator Spo0A. 19: 2236–2244
[4] Sonenshein, A.L., Hoch, J.A., Losick, R., (2002) Bacillus subtilis and Its Closest Relatives From Genes to Cells. ASM Press, United States of America. Pp 476–477
[5] Hilbert, D.W., Piggot, P.J., (June 2004) Compartmentalization of Gene Expression during Bacillus subtilis Spore Formation. Microbiology and Molecular Biology Reviews. Vol. 68, No. 2. Pp 234-262
[6] Eswaramoorthy, P., Guo, T., Fujita, M. (2009) In Vivo Domain-Based Functional Analysis of the Major Sporulation Sensor Kinase, KinA, in Bacillus subtilis. Journal of Bacteriology. Pp 5358-5368
[7] Stephenson, K., Hoch, J. A. (2001) PAS-A domain of phosphorelay sensor kinase A: A catalytic ATP-binding domain involved in the initiation of development in Bacillus subtilis. PNAS. Vol. 98, no. 26: 15251-15256
[8] Veening, J-W., Hamoen, L. W., Kuipers, O. P. (2005) Phosphatases modulate the bistable sporulation gene expression pattern in Bacillus subtilis. Molecular Microbiology 56(6), 1481-1494
[9] Voigt, C. A., Wolf, D. M., Arkin, A. P. (2004) The Bacillus subtilis sin Operon: An Evolvable Network Motif. Genetics 169: 1187-1202
Retrieved from " http://2009.igem.org/Team:Newcastle/SporulationTuning "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
