"
Page
Discussion
View source
History
teams
Log in
 
Team:Newcastle/ Stochastic Switch cloning strategy
From 2009.igem.org
Home
Team
Project
Parts
Modelling
Sponsors
Lab Book
Project
Overview
Choices Rationale
Characterisation
Safety
Judging Comments
Sub-projects
Metal Intake/Efflux
Metal Sensing
Sporulation Tuning
Population Dynamics
Stochastic Switch
Metal Sequester
Chassis
Promoter Library
Wet Lab
Lab Book
Protocols
Meetings
Meetings Calendar
Planning
Timeline
Ideas
Links
Helping other teams
Ethics
T-Shirts
iGEM Meetup
Our City
Fun and Games
Sponsors
Useful Links
Contact Us
Press Coverage
Contents
1 Stochastic Switch Cloning Strategy
1.1 Cloning strategies: Stochastic switch
1.2 Cloning strategies: Degradation controller
1.3 SspB
1.3.1 Primers
1.4 Ara Promoter/Operator
1.4.1 Primers
1.5 SacA integration region
1.5.1 Primers
1.6 Degradation controller device
1.6.1 Primers
Stochastic Switch Cloning Strategy
We have the following cloning strategies for testing our construct.
Cloning strategies: Stochastic switch
The following is the clone manager file for our synthesised brick; using clone manager it is easy to simulate digests and ligation reactions, in order to make sure the enzymes you use in the will give the result you expect.
The stochastic brick fragment is to be cloned into the integration vector pGFP-rrnB. This is because plasmid DNA is very unstable in bacillus. From pGFP-rrnb it will integrate into the 168 chromosome at amyE. We can clone the fragment by cutting pGFP-rrnB with EcoRI and NheI, cutting the fragment with EcoRI and NheI, and ligating.
pGFP-rrnb
We also need to clone the brick into a BioBrick compatible vector for submission to MIT. Digesting the fragment with EcoR1 and Pst1 and ligation into a EcoR1 and Pst1 cut biobrick vector will do this.
Cloning strategies: Degradation controller
SspB
SspB proteintargets proteins tagged with modified degradation tag, ssrA, and deliver them to ClpXP for proteolysis.  Modified ssrA tags are fused onto the 3’ end of a gene. In our project Hin protein is tagged with the modified ssrA.  Hence we will be able to control the degradation of Hin proteins by controlling the expression of SspB.The sequence is taken from E. coli strain MG1655. Genbank accession number is NC_000913.2.
The 498 bp long sspB CDS is amplified by PCR with dangling end primers with EcoRI-NotI-XbaI restriction sites at the 5’ and SpeI at the 3’. The resulting PCR product is cut with EcoRI and SpeI and inserted into a Biobrick compatible vector cut with the same enzymes.
Primers
We used the following primers to amplify SspB:
Forward primer: GATCTGGAATTCGCGGCCGCTTCTAGATGGATTTGTCACAGCTAAC
Reverse Primer: TGTGACACTAGTATTACTTCACAACGCGTAATGC
Ara Promoter/Operator
The purpose of this system is to enable us to control the expression of sspB, the degradation controller. In this way we have a third level of control over hin expression levels within the cell. The system is inducible by arabinose, therefore we can alter the arabinose levels to regulate hin degradation.
The 156bp long bindingsite_promoter_bindingsite_RBS  is amplified by PCR with dangling end primers with EcoRI-NotI-XbaI restriction site at the 5’ and SpeI at the 3’ and inserted into a Biobrick compatible vector. The sequence is taken from wildtype Bacillus subtilis.
Primers
We used the following primers to amplify the ara promoter/operator:
Forward:GATCTGGAATTCGCGGCCGCTTCTAGAGCACAGCGTTCTTTGTAAG
Reverse:TGTGACACTAGTAGCCCTCCCGAATGTTGAG
SacA integration region
The SacA CDS codes for a dispensable protein, similar to AmyE, and therefore can be disrupted and used as an integration site without harming cell metabolism. When inserted into Bacillus subtilis integration vector, the sac region will provide a single crossover site copying the whole plasmid into the bacterial chromosome. The following sequencher diagram shows our plans for PCR.
The 366 bp long sacA sequence is selected from 1440 bp long sacA CDS. This sacA sequence is amplified by PCR with dangling end primers with EcoRI-NotI-XbaI restriction site at the 5’ and the SpeI at 3’ and inserted into a Biobrick compatible vector. The sequence is taken from B. subtilis strain 168 and the Genbank accession number is AL009126.
Primers
We used the following primers to amplify the sacA region:
Forward: GATCTGGAATTCGCGGCCGCTTCTAGAGCACAGCGTTCTTTGTAAG
Reverse: TGTGACACTAGTAAGGATCGGATGTGCTGATTG
Degradation controller device
Finally we need to put all of our manual bricks into one plasmid to create our degradation controller device. When inserted into Bacillus subtilis integration vector, it will provide a single crossover site with arabinose controlled sspB expression.
Sac, araR promoter+RBS, and sspB biobricks are combined as a device into the Biobrick compatible vector pSB1AT3. The sequence of the device (without the biobrick prefix and suffix) is then amplified by PCR with dangling end primers with PacI at the 5’ and SacII at the 3’.  Pmutin4 is cut with PacI and SacII and the backbone of pmutin4 without terminator_0 and pspac is ligated with the PCR product.
Firstly digest plasmid part araR (pSB1AT3) with SpeI and PstI and digest plasmid part sspB (pSB1AT3) with XbaI and PstI. These fragments can then be ligated (Ligation will be tested by gel electrophoresis).
Now plasmid part sacA (pSB1AT3) can be cut with SpeI and PstI and plasmid part araR promoter+rbs+sspB (pSB1AT3)is cut with XbaI and PstI.
Primers
Forward primer: GGTGCTGTTAATTAATTCTCATTTCTGCGCATGGC Reverse Primer: TGTGATCCGCGGTTACTTCACAACGCGTAATGCC
News
Events
20 – 21 June 2009 - Europe workshop (London)
23 – 24 June 2009 - UK iGEM meetup (Edinburgh)
23 October Practice Presentation (Newcastle)
23 October T-shirts are ready
27 October Practice Presentation (Sunderland)
27 October Poster is ready
30 October – 2 November 2009 - Jamboree (Boston)
Social Net
Newcastle iGEM Twitter
Newcastle on Facebook
Newcastle Youtube Channel
Retrieved from " http://2009.igem.org/Team:Newcastle/_Stochastic_Switch_cloning_strategy "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
