"
Page
Discussion
View source
History
teams
Log in
 
Team:Newcastle/Labwork/3 September 2009
From 2009.igem.org
Home
Team
Project
Parts
Modelling
Sponsors
Lab Book
Project
Overview
Choices Rationale
Characterisation
Safety
Judging Comments
Sub-projects
Metal Intake/Efflux
Metal Sensing
Sporulation Tuning
Population Dynamics
Stochastic Switch
Metal Sequester
Chassis
Promoter Library
Wet Lab
Lab Book
Protocols
Meetings
Meetings Calendar
Planning
Timeline
Ideas
Links
Helping other teams
Ethics
T-Shirts
iGEM Meetup
Our City
Fun and Games
Sponsors
Useful Links
Contact Us
Press Coverage
Formal Lab Session - 3rd September 2009
Overview
Sporulation Tuning/Chassis Team -
Stochastic Switch Team -
Promoter Library Sub-Project -
Metal Sensor Team -
Sporulation Tuning/Chassis Team
Today we continued the lab work on our PCR primers. We again attempted to use two pairs of our primers; the ones to amplify the sleB and cwlJ genes from the B. subtilis genome. We prepared the solutions needed, and followed the PCR protocol. We rehydrated our primers, and diluted them accordingly. The TMs were lowered, following yesterday's failed attempt to 50 and 45 degrees. We prepared four lots of solutions, two with genomic DNA and two without (our controls), containing 2 and 3ul of primer. They were loaded into the PCR machine, which ran for around 2 hours 23 minutes.
When we ran the products on a gel.
Test concentration of PCR primers
Using NanoDrop spectrophotometer
sleBForward: 202.8ng/ul  19pmol/ul
sleBReverse: 272.3ng/ul  30pmol/ul
cwlJForward: 318.8ng/ul  28pmol/ul
cwlJReverse: 248.9ng/ul  27pmol/ul
Redo PCR
Exp. PCR experiment for gene cwlJ and sleB
No.
Water (ul)
PCR Mix (ul)
Forward Primer (ul)
Reverse Primer (ul)
Template (ul)
Polymerase used (ul)
Total Volume (ul)
1
24
15
sleBForward 3ul
sleBReverse 2ul
168 DNA 5ul
MolTaq (1u/ul) 1ul
50
2
29
15
sleBForward 3ul
sleBReverse 2ul
0
MolTaq (1u/ul) 1ul
50
3
24
15
sleBForward 3ul
sleBReverse 2ul
168 DNA 5ul
MolTaq (1ul)
50
4
29
15
sleBForward 3ul
sleBReverse 2ul
0
MolTaq (1u/ul) 1ul
50
5
23
15
cwlJForward 3ul
cwlJReverse 3ul
168 DNA 5ul
MolTaq (1u/ul) 1ul
50
6
28
15
cwlJForward 3ul
cwlJReverse 3ul
0
MolTaq (1u/ul) 1ul
50
7
23
15
cwlJForward 3ul
cwlJReverse 3ul
168 DNA 5ul
MolTaq ((1u/ul) 1ul
50
8
28
15
cwlJForward 3ul
cwlJReverse 3ul
0
MolTaq (1u/ul) 1ul
50
PCR procedure
No.1,2,5,6:
95C 2min -->[95C 30S --> 50C 30S --> 75C 1.5min] 30 runs --> 75C 5min --> store at 4C
No.3,4,7,8:
95C 2min -->[95C 30S --> 45C 30S --> 75C 1.5min] 30 runs --> 75C 5min --> store at 4C
PCR result
Still no bands when we run the PCR result on 0.8% agarose gel.
conclusion
Try to change the Tag enzyme instead of MotTag
Try to redesign Primers
Stochastic Switch Team
Jess attaching electrodes to the gel electrophoresis apparatus
Summary
Today we ran the PCR products for sac, ara and sspb on a 0.8% gel with wide wells, which we then cut out using the transilluminator.  Goksel also did a ligation of the sspb brick that was purified from the gel using the sigma gel extraction protocol. We ligated the product with the pSB1AT3 plasmid backbone prepared last week. Both the sspb insert and the backbone were cut with EcoRI and SpeI.  Today Arun also ran another PCR including our ara region, it worked well so we believe we can use the resulting PCR product for a digest and a ligation.
sac, araR and sspB PCR products were run on the gel. The order of the weels is: ladder, sac, araR, sspB, ladder.
Ligation
Goksel preparing ligations
Add water, 14ul
Add ligation buffer, 3ul
Mix well
Add the sspB insert, 10ul
Add the psB1AT3 plasmid backbone, 2ul
Add the ligase, 1ul
Vortex, pippet up&down
Store at the fridge overnight suspending in water
We also did a control without the insert to measure the background level of religation of the vector. It should be zero.
Promoter Library Sub-Project
Introduction
From the previous lab session the products of the PCR reactions attempting to amplify the sigA promoter library (synthesized sequences of the sigA library containing degeneracies) were run through 1.5% agarose gel in electrophoresis. Analysis of the gel showed that although there were hazy bands forming, these may have not been the result of the PCR products and may have been caused by the primer-dimer complexes formed. The promoter fragments are roughly 100bp however, meaning that there may be little distinction between the promoters and the primer-dimers in 1.5% agarose gel! In light of this experiment, today's task will involve cleaning up the PCR products for another run of DNA gel electrophoresis tomorrow.
PCR Product Cleanup
Mathew preparing the binding columns used for PCR clean-ups
The PCR Product Kit used to purify the PCR results was the GenElute™ PCR Clean-Up Kit, purchased from Sigma-Aldrich. The protocol used in this experiment is the one given with the kit. There were no changes to the protocol and the steps taken to perform the purification can be seen here:
Took four Mini-prep Binding Columns and placed them into collection tubes; to each 0.5ml of Column Preparation Solution was added. The tubes were then centrifuged for 1 minute under 12,000g.
Once the eluate in all of the collection tubes was removed, 200ul of Binding Solution was added to the four PCR Eppendorf tubes containing the PCR DNA (the four tubes containing 40ul of C0, V1, V2 and V3 PCR DNA). These four PCR DNA mixtures were then transferred to the four prepared MiniPrep Binding Columns and centrifuged for 1 minute under 13,000rpm. The eluate was discarded once again with the binding columns returned to their respective collection tubes.
0.5ml of Wash Solution was then added to the four binding columns. These binding columns (within the collection tubes) were then centrifuged for 1 minute under 13,000rpm with the resulting eluate discarded. The binding columns were then placed in their original collection tubes and centrifuged again at 13,000rpm; this time with no solutions added.
The binding columns were then transferred to four new collection tubes (appropriately labelled) whilst the original collection tubes (plus eluate from the 2 minute centrifugation) were discarded. 50ul of Elution Solution was then added to the binding columns and allowed to sit within the tubes on the bench for 1 minute.
The columns (within their new collection tubes) were then centrifuged for 1 minute under 13,000 rpm but this time it was the eluates which were kept and the binding columns discarded. The four collection tubes containing the eluated PCR products of C0, V1, V2 and V3 PCR reactions were then stored in the -20°C.
Conclusion
With the PCR products purified, another attempt at DNA gel electrophoresis under 1.5% agarose gel should confirm whether the PCR reactions to amplify our synthesized sigA promoter library has worked.
Metal Sensor Team
Introduction
Today we shall again run the products from yesterdays PCR reaction on agarose gel (through DNA gel electrophoresis) to determine whether we have been successful this time. If we are successful the team will purify the PCR product; if not the team will reattempt a PCR reaction.
Practical Outline
By the end of the day the Metal Sensing team hope to accomplish these objectives:
Run fragments from PCR reaction (carried out yesterday) on gel through electrophoresis.
If PCR reaction successful clean up products; if unsuccessful reattempt PCR reaction
Observations
We ran the products from yesterday's PCR reaction on gel and after about 30-40 minutes of DNA gel electrophoresis, the team analysed the gel using GelDoc:
The lanes of the gel were as follows:
Lane 1 = blank
Lane 2 = HindIII DNA ladder
Lane 3 = czrA PCR product
Lane 4 = czrA PCR control
Lane 5 = ara PCR product
Lane 6 = ara PCR control
Lane 7 = Bacillus subtilis genomic DNA
Lane 8 = Bacillus subtilis genomic DNA
Lane 9 = Bacillus subtilis genomic DNA
Lane 10 = HindIII DNA ladder
As far as the Metal Sensing team are concerned the PCR reaction involving the gene czrA did not work at all - this means we need to reattempt the PCR reaction. Meanwhile the Stochastic Switch team's ara coding sequence PCR did work.
Procedure
PCR
Today, after having run the gel for yesterdays PCR products, we are carrying out another PCR to amplify czrA gene which didn't work yet at different annealing temperatures: 40C 45C and 50C.
Exp.No.5 PCR experiment plan
No.
Water (ul)
PCR Mix (ul)
Forward Primer (ul)
Reverse Primer (ul)
Template (ul)
Polymerase used (ul)
Total Volume (ul)
1
24
15
pMA1 2.5ul
pMA2 2.5ul
168 DNA 5ul
MolTaq (1ul)
50
2
29
15
pMA1 2.5ul
pMA2 2.5ul
0
MolTaq (1ul)
50
3
24
15
pMA1 2.5ul
pMA2 2.5ul
168 DNA 5ul
MolTaq (1ul)
50
4
29
15
pMA1 2.5ul
pMA2 2.5ul
0
MolTaq (1ul)
50
5
24
15
pMA1 2.5ul
pMA2 2.5ul
168 DNA 5ul
MolTaq (1ul)
50
6
29
15
pMA1 2.5ul
pMA2 2.5ul
0
MolTaq (1ul)
50
PCR conditions:
For sample 1 and 2
1- 95C for 2min 2- 95C for 30sec 3- 40C for 1min30sec  4- 75C for 2min     Step 2-3-4 x 30 5- 75C for 5min 6- 4C Pause
For sample 3 and 4
1- 95C for 2min 2- 95C for 30sec 3- 45C for 1min30sec  4- 75C for 2min     Step 2-3-4 x 30 5- 75C for 5min 6- 4C Pause
For sample 5 and 6
1- 95C for 2min 2- 95C for 30sec 3- 50C for 1min30sec   4- 75C for 2min     Step 2-3-4 x 30 5- 75C for 5min 6- 4C Pause
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
News
Events
20 – 21 June 2009 - Europe workshop (London)
23 – 24 June 2009 - UK iGEM meetup (Edinburgh)
23 October Practice Presentation (Newcastle)
23 October T-shirts are ready
27 October Practice Presentation (Sunderland)
27 October Poster is ready
30 October – 2 November 2009 - Jamboree (Boston)
Social Net
Newcastle iGEM Twitter
Newcastle on Facebook
Newcastle Youtube Channel
Retrieved from " http://2009.igem.org/Team:Newcastle/Labwork/3_September_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
