"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Newcastle/Labwork/21 August 2009
From 2009.igem.org
Home
Team
Project
Parts
Modelling
Sponsors
Lab Book
Project
Overview
Choices Rationale
Characterisation
Safety
Judging Comments
Sub-projects
Metal Intake/Efflux
Metal Sensing
Sporulation Tuning
Population Dynamics
Stochastic Switch
Metal Sequester
Chassis
Promoter Library
Wet Lab
Lab Book
Protocols
Meetings
Meetings Calendar
Planning
Timeline
Ideas
Links
Helping other teams
Ethics
T-Shirts
iGEM Meetup
Our City
Fun and Games
Sponsors
Useful Links
Contact Us
Press Coverage
Formal Lab Session - 21st August 2009
Overview
Stochastic Switch Team - mini-preps of the BBa_C0178 BioBrick were carried out as well as restriction enzyme digests on BBa_R0062, BBa_R0079, BBa_C0161, BBa_C0179, and BBa_J44000
Metal Sensor Team - conducted DNA gel electrophoresis on yesterday's mini-prep digests, conducted midi-preps based on analysis of digests and also carried out test PCR on pGFP-rrnB transformant B. subtilis genome
Stochastic Switch Team
Summary
Today the stochastic team split into two, to fully utilise the time in the lab. Jess carried out minipreps for the BBa_C0178 in E.coli DH5alpha cultures that were set up yesterday. Goksel carried oud restriction digests of the BioBricks R0062; R0079; C0161; C0179; J44000 which were miniprepped yesterday.
Miniprep of C0178 cultures
Goksel has certainly picked up his lab skills very quickly, making a very neat job of filling the gel wells
As it happens only 2 of the 3 cultures grew from yesterdays overnights so these were spun down (1ml of each). It was decided that these cultures would be treated separately until after being run on a gel so their identities could be confirmed.
After spinning the cultures down after the addition of solution III, it was observed that there was not much of a protein pellet to be seen which should precipitate after solution III is added. The tubes were shaken well and put in to centrifuge for a further 10 miutes and a larger pellet was seen.
N.B: During our minipreps we have always kept solution III on the bench, however Chris seeing this advised us to always keep solution III in the fridge for the best results. As well as this after the addition of solution III the tubes must be shaken well in order for the proteins to precipitate properly. Remember that Phil's protocols may not always be explicit and although care should be taken in following them, there may be important points missed out that can effect results.
The supernatant was taken and Isopropanol added, the solution was spun, ethanol added, aspirated and speed vac'd according to the protocol. The DNA pellet was resuspended in 50ul water, labelled, and frozen in the -20 freezer.
Metal Sensor Team
Introduction
Mathew resuspending the pellet with Resuspension + RNAse Solution
In yesterday's lab session the Metal Sensing team carried out a miniprep of the plasmid pSB1A3, which contained the BioBrick BBa_J33206 (taken from the Parts Registry). This BioBrick contains the arsR binding site as well as the arsR gene - both of which are needed in our system. Once the minipreps were conducted, 10 microlitres of plasmid DNA were taken from three plasmid (pSB1A3) samples (the three samples originating from the three cultures of E.coli containing BBa_J33206 extracted from the plate)and placed into one set of 3 Eppendorf tubes; and then another 10 microlitres were taken from the three samples and put into another set of 3 Eppendorf tubes.
To the first set of 3 Eppendorf tubes, no enzyme was added and to the second tube, a combination of EcoRI and PstI was added. Once the restriction enzyme digests had taken place they were placed in the -20C freezer for use today.
Another thing which was carried out yesterday was preparation of agarose gel; the agarose gel was made up in the usual way, poured in the usual way and when it had set was stored in the fridge (this was done by sprinkling 1 x TAE buffer on the gel surface and then wrapping it in cling film). This gel will be used to analyse the restriction enzyme digests of the pSB1A3 plasmid.
There are still two tasks which need to be resolved:
Run both the undigested and digested plasmids on agarose gel in DNA gel electrophoresis.
Conduct midi-preps of the plasmids if gel results are positive.
In addition to this, the Metal Sensing team are hoping to carry out a test PCR reaction on the genome of Bacillus subtilis with the pGFP-rrnb plasmid integrated within it. The Metal Sensing team have designed primers which are embedded within the amyE region of the Bacillus subtilis chromosome so any integration at that site (the pGFP-rrnb plasmid integrates in the amyE gene in B. subtilis) will be detected.
Practical Outline
Mathew loading the pSB1A2 + BBa_J33206 DNA into the wells of the gel
The tasks needed to be completed by the Metal Sensing team by the end of the day are:
Run both the undigested and digested plasmids on agarose gel in DNA gel electrophoresis.
Conduct midi-preps of the plasmids if gel results are positive.
Run a test PCR reaction on a crude extraction of the genome of the Bacillus subtilis which had successfully been transformed by the plasmid pGFP-rrnb
Procedure
DNA gel electrophoresis
The gel loaded with DNA samples; the top row of wells belong to the Metal Sensor team whereas the bottom set of wells belong to the Stochastic Switch team
DNA gel electrophoresis was used to analyse the restriction enzyme digests for the three samples of the plasmid pSB1A3 containing BioBrick BBa_J33206. Three samples set aside for electrophoresis contained no restriction enzymes where as three samples of the BioBrick were treated with both EcoRI and PstI. The protocol used to prepare the DNA and to load the samples into the gel can be found at these links.
The way in which the samples were set up in terms of DNA gel wells were as follows:
Lane 1 = blank
Lane 2 = DNA ladder (HindIII ladder)
Lane 3 = Sample 1 (BBa_J33206) + no enzymes
Lane 4 = Sample 2 (BBa_J33206) + no enzymes
Lane 5 = Sample 3 (BBa_J33206) + no enzymes
Lane 6 = blank
Lane 7 = Sample 1 (BBa_J33206) + EcoRI + PstI
Lane 8 = Sample 2 (BBa_J33206) + EcoRI + PstI
Lane 9 = Sample 3 (BBa_J33206) + EcoRI + PstI
Lane 10 = DNA ladder (HindIII ladder)
The only change to the protocol is that 8 microlitres of ach of the prepared samples were placed into the wells. This is because the pSB1A3 plasmid is a high copy plasmid so the concentration of DNA will be high. Apart from that, the protocol was adhered to.
The results from the gel can be seen in the 'Results' section below.
Midi Preps
Mathew adding other Midi-Prep solutions to the transformed E.coli cells
The cell lysate floats at the top of the syringe barrel - eventually all of the clear solution below it will be transferred into the binding column
When looking at the mini-prep restriction enzyme digests on the gel, it was noted that although the bandwidths seen weren't what was expected the bands that did form were consistent for all three cultures. This could be down to a number of factors - the most likely factor, in these circumstances, are the restriction enzymes (see below in 'Results' section). Because of the consistency of the samples in the gel, both digested and undigested, it was decided that the Metal Sensor team prepare midi-preps of the pSB1A3 plasmid.
The protocol used was SigmaAldrich's MidiPrep protocol . The Spin Method (i.e. using a centrifuge to move solutions through the column) was the preferred route in this protocol and all the steps involved in this procedure were carried out, up until step 11b. The DNA was then stored in the freezer for use tomorrow.
While the Metal Sensor team had prepared three overnight cultures to be used in the midiprep today (the three cultures derived from three colonies of E. coli all containing pSB1A3 and BBa_J33206) only one of these colonies was taken forward for midipreps. It was decided that the overnight culture grown from colony 1 would be used for midipreps - this was because of the clarity of the bands produced by colony 1 in both the undigested and digested samples.
A summary of the steps were as follows:
50ml of culture 1 was spun down for 10 minutes at 5,000g. The supernatant was aspirated off and the pellet was resuspended in 4ml Resuspension/RNAse solution.
4ml of lysis solution was immediately applied and after inverting the falcon tube 6 times, the solution was allowed to sit on the bench for 4 minutes. 4ml neutralisation solution was added once the time had expired and the tube inverted 6 times again.
3ml of binding solution was added and after 2 inversions the solution was placed into the filter syringe. With the plunger removed from the syringe, the lysate remained in the syringe for 5 minutes. During this time, 4ml of Column Preparation Solution was allowed to be run through the midiprep binding column (by centrifugation).
The plunger was then applied to the syringe and half of the lysate (which had been sitting in the syringe barrel for 5 minutes) was injected into the midiprep binding column. The binding column was placed in the centrifuge at 3,000g for 2 minutes and once the eluate had filtered through the remaining half of the lysate was added and treated in the same way.
4ml of Wash Solution 1 was then added to the binding column and centrifuged for 2 minutes for 2 minutes before 4ml of Wash Solution 2 was added and then placed in the centrifuge for 2 minutes again.
1 ml of Elution solution was then added to the midiprep binding column before the column was spun for 5 minutes at 3,000g. The eluate was then harvested at the bottom of the binding column and transferred to the freezer for storage.
PCR Reactions
Primers are sent by the oligo company in dehydrated form, and a sheet is attached with each primer's properties.
Once we receive the primers from the oligo company, we need to enter it into PA' primer database. Only Chris have access to the database, so we need to kindly ask him to open the database for us.
Data needed to be entered into the database: primer name, sequence, primer length and oligo company's name.
The database will give a primer number which we need to write on the tube's lid and on the primer's properties sheet. In today's case, we got:
pGFPconf1Â : primer 501 pGFPconf2Â : primer 502 pGFPconf3Â : primer 503 pGFPconf4Â : primer 504
Then, we need to rehydrate the primers. In order to do that, we took the primer properties sheet and found the primer concentration in nMoles. Multiply that number by 5 and you get the amount of water to be added to the primer in order to make it into the standard PA lab's primer concentration of 200ng/ul.
For todays's case:
pGFPconf1Â : 20.2nmoles > added 20.2 x 5 = 101 ul of PCR water pGFPconf2Â : 30.3nmoles > added 30.3 x 5 = 151.5 ul of PCR water pGFPconf3Â : 28.5nmoles > added 28.5 x 5 = 142.5 ul of PCR water pGFPconf4Â : 21.7nmoles > added 21.7 x 5 = 108.5 ul of PCR water
The rehydrated primer stock then needs to be diluted 1:10 for working concentration; therefore, we got 50ul from the primer stock and added 450ul of PCR water.
Once rehydrated, the primers stock can be stored in the -20C freezer in the primer box, the 1:10 primer dilution can be stored the -20C freezer in the diluted primer box.
We made PCR water: got 50ml of UV-sterilised water from the stock and filter sterilised it into a 50ml Falcon tube.
Results
The first and last wells in this photograph which are yielding bands are the HindIII ladder DNA, used for comparisons. The first three lanes next to the starting HindIII lane are the BBa_J33206 BioBrick in pSB1A2 plasmid with no enzymes added. The next three lanes (missing out the blank lane used as a spacer) are the BBa_J33206 BioBrick in pSB1A2 plasmid with restriction enzymes EcoRI + PstI added.
The samples of BBa_J33206 BioBrick in pSB1A2 plasmid with no enzymes added show consistency as do the samples of BBa_J33206 BioBrick in pSB1A2 plasmid with restriction enzymes EcoRI + PstI added. However the band sizes for the digested BioBrick samples don't appear to match the expected bands; the BioBrick insert, according to the parts registry entry , should be 956 bp and exist as one band whereas the gel suggests it could be between 125bp and 2027bp and could be any of the two bands present. No matter which band you look at, the insert on the gel doesn't match the expected BioBrick size.
Nevertheless, due to the consistency of the bands it is likely that there is no contamination in the samples. These samples can therefore be used for midi-preps and then analysed using better restriction enzymes.
The probable reason for the bands looking incorrect is the restriction enzymes used. In a previous experiment we had aliquoted some EcoRI and PstI enzymes into separate Eppendorf tubes; for this experiment we used these aliquoted enzymes rather than enzymes taken from the original container. The aliquoting and subsequent exposure to room temperature may have lead to the degradation of enzyme efficiency.
July
M
T
W
T
F
S
S
Â 
Â 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
Â 
Â 
Â 
Â 
Â 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
Â 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
Â 
Â 
Â 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
News
Events
20 â 21 June 2009 - Europe workshop (London)
23 â 24 June 2009 - UK iGEM meetup (Edinburgh)
23 October Practice Presentation (Newcastle)
23 October T-shirts are ready
27 October Practice Presentation (Sunderland)
27 October Poster is ready
30 October â 2 November 2009 - Jamboree (Boston)
Social Net
Newcastle iGEM Twitter
Newcastle on Facebook
Newcastle Youtube Channel
Retrieved from " http://2009.igem.org/Team:Newcastle/Labwork/21_August_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
