"
Page
Discussion
View source
History
teams
Log in
 
Team:Newcastle/Characterisation
From 2009.igem.org
Home
Team
Project
Parts
Modelling
Sponsors
Lab Book
Project
Overview
Choices Rationale
Characterisation
Safety
Judging Comments
Sub-projects
Metal Intake/Efflux
Metal Sensing
Sporulation Tuning
Population Dynamics
Stochastic Switch
Metal Sequester
Chassis
Promoter Library
Wet Lab
Lab Book
Protocols
Meetings
Meetings Calendar
Planning
Timeline
Ideas
Links
Helping other teams
Ethics
T-Shirts
iGEM Meetup
Our City
Fun and Games
Sponsors
Useful Links
Contact Us
Press Coverage
Characterization of IPTG inducable KinA sporulation trigger device - part BBa_K174011
We successfully characterized our IPTG inducible kinA sporulation tuning device BBa_K174011 . Although Spo0A is the master regulator of sporulation in B. subtilis, only an increase in phosphorylated Spo0A can trigger the sporulation(1). kinA is a kinase that will phosphorylate Spo0A. Thus, we achieved the increase in sporulation rate by artificially inducing the gene, kinA, with the pSpac promoter controlled by IPTG. We used growth curves and fluorescence microscopy to verify our results by assaying the expression of the gfp gene that is transcriptionally fused to kinA and by counting the spores visible by brightfield microscopy.
For more information about the design of this device go to our sporulation tuning page.
Our synthesised device was cloned into the integration vector pGFP-rrnb constructed by Dr. JW Veening. By placing our device just before the gfp CDS in the integration vector, we hoped to see GFP when we induce the device with IPTG. However for IPTG to work, we needed LacI expressed in the cells.
When we first did our transformation for B. subtilis we forgot this link and we got similar results with cells induced with IPTG and not induced with IPTG. Obviously LacI was not present in the cells and so adding IPTG did not make any difference.
We then used a Bacillus subtilis mutant, BFS687, which has pMutin4 integrated into the chromosomal DNA (2). We selected this mutant especially since it does not have any phenotype under a wide range of conditions as described in the Micado database. Since pMutin4 has the lacI gene, we were able to get LacI expressed in the cells. We successfully transformed the mutant with pGFP-rrnb carrying out kinA based sporulation rate tuning brick . amyE homologous regions in the integration vector mediated a double crossover into amyE in the chromosome. As a result of this crossover, cells lose the amyE gene and cannot break down starch. When transformed colonies are plated into starch plates, the transformed colonies that have successfully inetgrated the plasmid can be seen without any halos around them when we expose the plates to iodine.
We then selected two transformed colonies from the starch plate and used them to induce sporulation by adding IPTG.
To select the transformed colonies we prepared our plates with LB + Erythromycin + Chloramphenicol. Chloramphenicol was used to select pGFP-rrnb based transformations and Erythromycin  was used to select the colonies with pMutin4 intgrated. Hence, by adding these two antibiotics we made sure that we were using BFS867 mutants transformed with pgfp-rrnb integration vector containing our biobrick as an insert.
Concentrations used to characterize our device:
IPTG : 1mM Erythromycin :0.3ug/ml Chloramphenicol: 5ug/ml
Experiment Overview
Each 10ml of overnight cultures was split into two flasks with 60ml of LB+Em+CHL. Every half an hour we measured the optical density of the cells to plot the growth curve of the cells. We also stored samples on ice at each time point. We added 1mM IPTG after one hour to one set of the cultures but not the other. We also used wild type B. subtilis and wild type cells transformed with pGFP-rrnb (vector only) as the negative and positive controls respectively.
After time point 5, before the cells reached stationary phase, we used the samples from time point 1 and time point 5 and prepared them for microscopy by resuspending the cells in SMM medium and absorbing them on to agarose slabs on microscope slides.
Growth Curves
The growth curves below show that growth of the strain with BBa_K174011 integrated grows more slowly that the wild type, presumably because of increased sporulation rates. The addition of IPTG slows the growth rate still further.
OD's above 1.0 need to be discounted since the spectrophotometer we used is not accurate above this OD.
Shows Wild type B. subtilis, WT. B. Subtilis transformed with pGFP-rrnb vector only, and BFS867 mutant transformed with pGFP-rrnb + BBa_K174011. (+ sign denotes IPTG added)
Growth curve of the BBa_K174011 mutant with and without IPTG
Microscopy results
Without IPTG
At time point 1, the cells did not express gfp and there was little evidence of sporulation (see figure below). Examination of the sample taken at time point 5 (see figure below) revealed the presence of a few spores inside the mother cells and only minimal gfp expression. Some spores would be expected since natural sporulation was not disabled and the cultures were approaching stationary phase.
With 1mM IPTG
Again, at time point 1, the cells did not express gfp and there was little evidence of sporulation (see figure below).
As can be seen, at time point 5 (shown below) with the addition of 1mM IPTG there are many spores due to the increased expression levels of KinA. In addition, many of the cells are showing high levels of gfp expression and many of the gfp expressing cells can be seen to contain spores.
Hence we can conclude that the sporulation was mainly triggered by our device.
We therefore believe that the sporulation rate tuning device works as expected.
If we had time we would like to characterise this brick even further by carrying our sporulation assays (using the heat resistance of spores as a selective measure) and by flow cytometry to accurately measure the sporulation rate.
Bright Field image of WT B.subtilis at time point 1, No IPTG
Gfp expression shown for WT B.subtilis at time point 1, No IPTG
Bright Field image of BFS 867 + BBa_K174011 at time point 5, No IPTG
Gfp expression shown for BFS 867 + BBa_K174011 at time point 5, No IPTG
Bright Field image of BFS 867 + BBa_K174011 at time point 5, WithIPTG
Gfp expression shown for BFS 867 + BBa_K174011 at time point 5, With IPTG
Zoom into Bright Field image of BFS 867 + BBa_K174011 at time point 5, WithIPTG. The red arrows show spores
Fujita, M. and R. Losick (2005). "Evidence that entry into sporulation in Bacillus subtilis is governed by a gradual increase in the level and activity of the master regulator Spo0A." Genes & Development 19(18): 2236-2244.
Retrieved 20/10/2009, from http://bacillus.genome.jp/bsorf_mutant_list/Page12.htm
News
Events
20 – 21 June 2009 - Europe workshop (London)
23 – 24 June 2009 - UK iGEM meetup (Edinburgh)
23 October Practice Presentation (Newcastle)
23 October T-shirts are ready
27 October Practice Presentation (Sunderland)
27 October Poster is ready
30 October – 2 November 2009 - Jamboree (Boston)
Social Net
Newcastle iGEM Twitter
Newcastle on Facebook
Newcastle Youtube Channel
Retrieved from " http://2009.igem.org/Team:Newcastle/Characterisation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
