"
Page
Discussion
View source
History
teams
Log in
 
Team:Newcastle/Labwork/7 September 2009
From 2009.igem.org
Home
Team
Project
Parts
Modelling
Sponsors
Lab Book
Project
Overview
Choices Rationale
Characterisation
Safety
Judging Comments
Sub-projects
Metal Intake/Efflux
Metal Sensing
Sporulation Tuning
Population Dynamics
Stochastic Switch
Metal Sequester
Chassis
Promoter Library
Wet Lab
Lab Book
Protocols
Meetings
Meetings Calendar
Planning
Timeline
Ideas
Links
Helping other teams
Ethics
T-Shirts
iGEM Meetup
Our City
Fun and Games
Sponsors
Useful Links
Contact Us
Press Coverage
Formal Lab Session - 7th September 2009
Overview
Metal Sensor Team -
Stochastic Switch Team -
Sporulation Tuning/Chassis Team -
Metal Sensor Team
Introduction
In last week's lab session, we had received some BBa_J33206 + pSB1A2 DNA in it's original form by Chris French (the person who submitted the BioBrick to the Parts Registry) as well as both PCR and Sequencing primers. Whilst we used the sequencing primers to determine the sequence of the midi-prepped BBa_J33206 DNA derived from the Parts Registry, we also attempted to transform some DH5-alpha E. coli bacteria with the original BBa_J33206 BioBrick sent to us.
In today's lab session, the plates on which the transformed cells were placed will be examined for colonies; if colonies should be present then they will be used to inoculate LB media in preparation for mini-preps.
Observations
Both of the LB+amp plates (one with 200ul of transformants and one with 500ul of transformants - left photograph and right photograph respectively) used to grow the transformant DH5-alpha E. coli cells were covered in hundreds of colonies - almost like a lawn of bacteria could be seen on the plates. This suggests that the transformation rate was very high; this is expected considering we were sent pure DNA. The only thing to note was that the 500ul plate seemed to contain dense, large round colonies in addition to the E. coli cells; this may be contamination.
Procedure
In light of these observations, a total of 6 tubes of 5ml LB solution were innoculated with the transformed E. coli cells. The first three tubes were inoculated by 3 cultures found on the 200ul LB+agar plate and the next three tubes were inoculated with 3 cultures found on the 500ul LB+agar plate. This was all done under aspetic conditions.
These tubes were then placed in the orbital shaking incubator at 37C in the morning of inoculations. The tubes were as follows:
Tube 1 - culture 1 from 200ul transformants on LB+amp plate
Tube 2 - culture 2 from 200ul transformants on LB+amp plate
Tube 3 - culture 3 from 200ul transformants on LB+amp plate
Tube 4 - culture 1 from 500ul transformants on LB+amp plate
Tube 5 - culture 2 from 500ul transformants on LB+amp plate
Tube 6 - culture 3 from 500ul transformants on LB+amp plate
Further Observations
When the tubes in the orbital incubator were observed in the afternoon, it was found that all of them had gone cloudy. This opacity meant that the E. coli cells had grown sufficiently within the space of a day.
Further Procedure
Inoculating 50ml LB flasks
Because it is likely that the Metal Sensing Team will be using BBa_J33206 in plasmid pSB1A2 DNA in further project work it is a good idea that midi-preps of the DNA are made. 3 flasks, each containing 50ml of LB solution (+ ampicillin), were innoculated each with 50ul from a chosen 5ml LB-grown E.coli culture.
The three 5ml cultures chosen to inoculate the three 50ml LB+amp flasks were taken from Tube 1 (culture 1 from the 200ul plate), Tube 2 (culture 2 from the 200ul plate) and Tube 5 (culture 2 from the 500ul plate). This was all done under aseptic conditions. The resulting inoculated flasks were placed in the orbital shaking incubator at 37C for overnight growth.
Attempting mini-prep
Seen as the six tubes containing LB+amp appeared to all contain sufficient amounts of transformed DH5-alpha E. coli cells, mini-preps of the samples were attempted. This was done using Phil Aldridge's mini-prep protocol and there were no changes to the protocol. However the procedure had to be called off at step 6 due to a problem with the samples.
The problem was the online wiki protocol was correct except for one thing; it missed out the 'shake rigourously' step which comes immediately after the addition of isopropanol. This meant that the proteins didn't properly aggregate and so the centrifugation step didn't properly pellet all of the unwanted substances. Because the mini-preps were conducted in the afternoon, time soon ran out and rather than trying to put the samples right the procedure was abandoned.
Conclusions
Whilst most of the 5ml culture samples were consumed by the mini-prep attempt there was still the 3 flasks of 50ml LB+amp solution growing in the orbital incubator. Therefore tomorrow the Metal Sensing Team will focus on conducting midi-preps without doing mini-preps and will take a small sample of the resulting DNA for analysis.
The online wiki protocol has since been changed to ensure mistakes like these don't happen again.
Stochastic Switch Team
Transformations from last week worked. We had transformed colonies from plates with Tube 1 solution which had the backbone + insert ligaton product. (4 colonies from the plate with 400ul of ligation product and 1 from the plate with 20ul of ligation product)
We also did not have any transformation from the plate(LB + Amp) with cut backbone (pSB1AT3 cut with EcoRI and SpeI). So there was no uncut pSB1AT3.
Today we will plate out transformed colonies and leave them in LB + Amp overnight for tomorrow's minipreps. (Remember to dry the plates having the colonies before using them). We mixed the colonies in 3ml LB + Amp solution and left them overnight in shaking incubator for minipreps.
We prepared 5 fresh plates from 5 transformed colonies and 10 LB + Amp tubes, 2 for each transformed colony. The plates were left in 37C incubator.
For LB+Amp solution we used 5ml of LB + 250ul Amp.
Today we also did fast ligations of the cut sac and ara fragments:
For the ara ligation:
5ul cut pSB1AT3 (12ng/ul)
35ul cut ara (5.5ng/ul)
7ul PCR water
4ul 5X buffer
19ul PCRwater
1ul T4 ligase
For the sac ligation:
5ul cut pSB1AT3 (12ng/ul)
5ul cut sac (31ng/ul)
7ul PCR water
4ul 5X buffer
19ul PCRwater
1ul T4 ligase
Vortexed pulsed these and left for 1 hour at 22C.  2-5 ul of this solution can be used for transformations tomorrow.
Sporulation Tuning/Chassis Team
Introduction
Run last Friday's PCR result
Purify sleB gene using PCR Clean-up Kit(Sigma)
PCR cwlJ gene with new parameter
Experiment prcedure
Run PCR product on 0.8% agarose gel:
lane8: Lamda DNA ladder lane9: sleB PCR product  lane10: sleB PCR control lane11: cwlJ PCR product lane12: cwlJ PCR control
Clean up sleB gene PCR product
Follow the PCR Clean up Kit standard procedure.
NanoDrop sleB gene concentration
sleB PCR product using primers sleBForward and sleBReverse : 39.3ng/ul -20C storage
Redo PCR for gene cwlJ
Using 55C for elongation with GoTag.
Conclusion
The reason for no PCR result in last week is MotTag's problem. When we changed the polymerase, we got the PCR result for sleB gene.
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
News
Events
20 – 21 June 2009 - Europe workshop (London)
23 – 24 June 2009 - UK iGEM meetup (Edinburgh)
23 October Practice Presentation (Newcastle)
23 October T-shirts are ready
27 October Practice Presentation (Sunderland)
27 October Poster is ready
30 October – 2 November 2009 - Jamboree (Boston)
Social Net
Newcastle iGEM Twitter
Newcastle on Facebook
Newcastle Youtube Channel
Retrieved from " http://2009.igem.org/Team:Newcastle/Labwork/7_September_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
