"
Page
Discussion
View source
History
teams
Log in
 
Team:Newcastle/SporulationTuning/Introduction
From 2009.igem.org
Home
Team
Project
Parts
Modelling
Sponsors
Lab Book
Project
Overview
Choices Rationale
Characterisation
Safety
Judging Comments
Sub-projects
Metal Intake/Efflux
Metal Sensing
Sporulation Tuning
Population Dynamics
Stochastic Switch
Metal Sequester
Chassis
Promoter Library
Wet Lab
Lab Book
Protocols
Meetings
Meetings Calendar
Planning
Timeline
Ideas
Links
Helping other teams
Ethics
T-Shirts
iGEM Meetup
Our City
Fun and Games
Sponsors
Useful Links
Contact Us
Press Coverage
Contents
1 Introduction
1.1 Spo0A
1.2 Sporulation
1.3 KinA
2 References
Introduction
The bacteria, Bacillus subtilis, used in our project is a gram-positive soil bacterium that, under certain conditions, would commit itself to a developmental pathway leading to the production of spores.[1] Therefore, in this section of our project, we hope to control sporulation in our bacterial population, such that we can decide how much of the population becomes spores, and how much continue as vegetative cells. Should the cell sporulate, it would become a ‘metal container’ , trapping the sequestered cadmium in its spore.
After the cell sequesters cadmium into its spore, it should not germinate or the sequestered cadmium will be released back into the environment as a result. Therefore, the role of chassis comes into play, where the sleB and cwlJ germination-defective mutants are put into use.
In order to control sporulation, our team is proposing the idea of inducing the synthesis of kinA, with IPTG as a sporulation initiation signal.
KinA is a major kinase which provides phosphate input to the phosphorelay, which in turn, activates the sporulation pathway upon starvation via the phosphorylated Spo0A transcription factor,[2] which governs entry into the sporulation pathways of the bacterium Bacillus subtilis.[3]
Spo0A
From our research, we found that entry into the sporulation pathway is governed by a member of the response regulator family of transcription facts, known as Spo0A.[3] Spo0A is activated by phosphorylation on an aspartyl residue located in the N-terminal portion of the protein.[3]
Unlike other response regulators, Spo0A is indirectly phosphorylated by a multicomponent phosphorelay involving at least three kinases called kinA, kinB, and kinC, which phosphorylate Spo0F, and the resulting Spo0F~P, in turn, transfers the phosphoryl group to Spo0B. Finally, Spo0B~P transfers the phosphoryl group to, and thereby activates, Spo0A.[3] As such, the phophoryl groups are drained from the relay by the action of dedicated phosphotases that dephosphorylate Spo0F~P and Spo0A~P.[1][3]
Spo0A is also subjected to control at the levels of its synthesis and activity by a positive feedback loop in which the response regulator stimulates the synthesis of the RNA polymerase σ factor σH, which, in turn stimulates transcription of the gene for Spo0A, as well as the genes for the phosphorelay components KinA and Spo0F.[2]
It is important to note that activating Spo0A via the phosphorelay is essential as it is responsible for allowing Spo0A to accumulate in a gradual manner, and this slow accumulation plays a critical role in the ability of the regulatory protein to trigger sporulation.[3] Experiments have also been carried out and results have shown that the activated form of Spo0A failed to trigger sporulation during growth as it had bypassed phosphorylation, and only a small proportion of cells progressed to the early stage of sporulation.[3]
Sporulation
From our research, we found that phosphatases may be viewed as negative regulators that provide access for negative signals to influence the cell’s decision whether to sporulate or to continue vegetative growth.[4] The phosphotases that dephosphorylates Spo0A, preventing its activation are Spo0E[1], YisI, and YnzD.[5] Therefore, in order to ensure that sporulation occurs under the appropriate conditions, the phosphorelay must integrate the competition between signal input provided by the kinases and signal cancellation carried out by the phosphatises, which determines the decision to sporulate or not,[4][5] by governing flux through the relay and hence the level of Spo0A~P, which must reach a threshold concentration to trigger sporulation.[1][3]
Also, the Spo0F lacks an output domain and is incapable of activating transcription, therefore it serves only as an intermediary in the phosphorelay.[5]
Other than the external and internal signals that are integrated into the phosphorelay, it is important to note that sporulation is also initiated by nutrient starvation , cell density , and cell cycle progression .[5]
KinA
Sporulation can be triggered with high efficiency in cells in the exponential phase of growth in rich medium by artificial induction of the synthesis of any one of three histidine kinases tha feed phosphoryl groups into the relay.[3] For our project, we are using kinA, a major histidine kinase responsible for activating the sporulation pathway in Bacillus subtilis.[6]
The sensor kinase, kinA was chosen to induce sporulation as it is responsible for the majority of phosphate input into the phosphorelay, in response to as yet unknown signal ligands.[7] It has also been demonstrated that the fraction of cells that initiate sporulation is decreased in a kinA mutant background.[8]
KinA is a soluble cytoplasmic protein that appears to be active as a dimer and is composed of an amino-terminal sensor domain and a carboxy-terminal autokinase domain.[6][7]
References
[1] Predich, M., Nair, G., Smith, I. (1992) Bacillus subtilis Early Sporulation Genes kinA, spo0F, and spo0A Are Transcribed by the RNA Polymerase Containing σH. Journal of Bacteriology. Pp 2771-2778
[2] Veening, J-W., Smits, W. K., Kuipers, O. P. (2008) Bistability, Epigenetics, and Bet-Hedging in Bacteria. Annu. Rev. Microbiol. 62: 193-210
[3] Fujita, M., Losick, R. (2005) Evidence that Entry into Sporulation in Bacillus subtilis is Governed by a Gradual Increase in the Level and Activity of the Master Regulator Spo0A. 19: 2236–2244
[4] Sonenshein, A.L., Hoch, J.A., Losick, R., (2002) Bacillus subtilis and Its Closest Relatives From Genes to Cells. ASM Press, United States of America. Pp 476–477
[5] Hilbert, D.W., Piggot, P.J., (June 2004) Compartmentalization of Gene Expression during Bacillus subtilis Spore Formation. Microbiology and Molecular Biology Reviews. Vol. 68, No. 2. Pp 234-262
[6] Eswaramoorthy, P., Guo, T., Fujita, M. (2009) In Vivo Domain-Based Functional Analysis of the Major Sporulation Sensor Kinase, KinA, in Bacillus subtilis. Journal of Bacteriology. Pp 5358-5368
[7] Stephenson, K., Hoch, J. A. (2001) PAS-A domain of phosphorelay sensor kinase A: A catalytic ATP-binding domain involved in the initiation of development in Bacillus subtilis. PNAS. Vol. 98, no. 26: 15251-15256
[8] Veening, J-W., Hamoen, L. W., Kuipers, O. P. (2005) Phosphatases modulate the bistable sporulation gene expression pattern in Bacillus subtilis. Molecular Microbiology 56(6), 1481-1494
[9] Voigt, C. A., Wolf, D. M., Arkin, A. P. (2004) The Bacillus subtilis sin Operon: An Evolvable Network Motif. Genetics 169: 1187-1202
News
Events
20 – 21 June 2009 - Europe workshop (London)
23 – 24 June 2009 - UK iGEM meetup (Edinburgh)
23 October Practice Presentation (Newcastle)
23 October T-shirts are ready
27 October Practice Presentation (Sunderland)
27 October Poster is ready
30 October – 2 November 2009 - Jamboree (Boston)
Social Net
Newcastle iGEM Twitter
Newcastle on Facebook
Newcastle Youtube Channel
Retrieved from " http://2009.igem.org/Team:Newcastle/SporulationTuning/Introduction "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
