"
Page
Discussion
View source
History
teams
Log in
 
Team:Newcastle/Labwork/1 September 2009
From 2009.igem.org
Home
Team
Project
Parts
Modelling
Sponsors
Lab Book
Project
Overview
Choices Rationale
Characterisation
Safety
Judging Comments
Sub-projects
Metal Intake/Efflux
Metal Sensing
Sporulation Tuning
Population Dynamics
Stochastic Switch
Metal Sequester
Chassis
Promoter Library
Wet Lab
Lab Book
Protocols
Meetings
Meetings Calendar
Planning
Timeline
Ideas
Links
Helping other teams
Ethics
T-Shirts
iGEM Meetup
Our City
Fun and Games
Sponsors
Useful Links
Contact Us
Press Coverage
Formal Lab Session - 1st September 2009
Overview
Stochastic Switch Team - analysed cut pSB1AT3 plasmid (successful) and also cleaned up ssbp PCR product. Also froze down strains of transformant E. coli containing BioBricks
Metal Sensor Team - prepared the newly arrive pMA1 and pMA2 primers and then used these primers to amplify czrA from B. subtilis genome (PCR). Also used PCR to amplify sac, ara and sspb for Stochastic Switch team.
Promoter Library Sub-Project - attempted to amplify the three sets of sigA promoter using PCR
Stochastic Switch Team
Goksel cleaning up products from a successful PCR reaction involving sspB
Today we run the gel for pSB1AT3 plasmid backbone purified last week from the gel. It was cut with EcoRI and SpeI sites. The length of the band confirmed that we have the right product.
We also used PCR clean-up kit to extract sspB from last week's PCR. We used sigma-d's PCR clean-up protocol. The final volume was measured as 48ul. We measured the concentration and run it on the gel using 5ul of sspb + 4ul of water + 1ul of the loading buffer. The band confirmed that we had the right product after the clean-up process.
The concentration of sspB was 37.5ng/ul
Jess freezing down E.coli cells containing 6 BioBricks
Today we also made ON cultures of the following strains in order to freeze them down into the TPA collection. First the TPA book had to be filled out to record the strains:
2489- BBa_CO178
2490- BBa_ROO62
2491- BBa_CO161
2492- BBa_CO179
2493- BBa_ROO79
2494- BBa_J44000
These will be frozen down in DMSO tomorrow.
Metal Sensor Team
Introduction
So far, the Metal Sensing Team have successfully digested the pSB1A2 plasmid (which contains the BBa_J33206 BioBrick - the arsR gene and binding site) so that the plasmid has had it's promoter removed. With this linearised fragment we hope to insert the czrA gene along with the cadA promoter; these genes will be extracted from the genome of Bacillus subtilis via PCR reactions. The primers capable of this are pMA1 and pMA2 (designed by Mathew and Arun).
Today we will carry out PCR in order to amplify czrA gene for our team and sac, ara and sspb genes for the stochastic switch team. However we must first of all hydrate and prepare the primers which have just arrived (i.e. pMA1 and pMA2). We will also need to prepare for the possibility of the PCR reactions not working - overnight cultures of B. subtilis in LB medium will be prepared for a possible PCR tomorrow.
Practical Outline
These are the three things which the Metal Sensing Team need to have accomplished by today:
Hydrate the newly-arrived primers pMA1 and pMA2, adding water according to their concentrations (in nMoles); then dilute them to 10x dilution.
Using the 10x hydrated primers, carry out a czrA amplification PCR reaction on cells taken from a plate containing wild type Bacillus subtilis
Inoculate LB cultures with B. subtilis cells and grow overnight - this is for if the PCR reactions today don't work
Procedure
1) Preparing primers pMA1 and pMA2
pMA1 and pMA2 need to be rehydrated by adding sterile distilled water. The volume of water to be added is simply (the concentration (in nMoles) of the primers multiplied by 5) ul. The concentrations of the primers are as follows
pMA1 = 30.6 nMoles
pMA2 = 31.3 nMoles
Therefore the amount of water to be added to each of the primers is:
pMA1 = (30.6 x 5) = 153ul
pMA2 = (31.3 x 5) = 156.5ul
The next thing to do is make 10x dilutions by extracting 50ul of each of these hydrated primers and adding them to 450ul of sterile distilled water.
2) PCR
We carried out PCR reactions in the same way as described on 21/08/2009 along with the stochastic switch team. It must be noted that the annealing temperatures for pMA1 and pMA2 are 63.1°C and 62.9°C respectively.
Exp.No.3 PCR experiment plan
No.
Water (ul)
PCR Mix (ul)
Forward Primer (ul)
Reverse Primer (ul)
Template (ul)
Polymerase used (ul)
Total Volume (ul)
1
24
15
pMA1 2.5ul
pMA2 2.5ul
168 DNA 5ul
MolTaq (1ul)
50
2
29
15
pMA1 2.5ul
pMA2 2.5ul
0
MolTaq (1ul)
50
3
24
15
sac_forward 2.5ul
sac_reverse 2.5ul
168 DNA 5ul
MolTaq (1ul)
50
4
29
15
sac_forward 2.5ul
sac_reverse 2.5ul
0
MolTaq (1ul)
50
5
24
15
araR_forward 2.5ul
araR_reverse 2.5ul
168 DNA 5ul
MolTaq (1ul)
50
6
29
15
araR_forward 2.5ul
araR_reverse 2.5ul
0
MolTaq (1ul)
50
7
24
15
sspb_forward 2.5ul
sspb_reverse 2.5ul
168 DNA 5ul
MolTaq (1ul)
50
8
29
15
sspb_forward2.5
sspb_reverse 2.5
0
MolTaq (1ul)
50
Setting up the PCR machine
Mathew pouring two gels at the same time!
PCR conditions:
For sample 1 and 2:
1- 95C for 2min 2- 95C for 30sec 3- 63C for 1min  4- 75C for 2min     Step 2-3-4 x 30 5- 75C for 5min 6- 4C Pause
For sample 3, 4, 7 and 8:
1- 95C for 2min 2- 95C for 30sec 3- 57C for 1min  4- 75C for 2min     Step 2-3-4 x 30 5- 75C for 5min 6- 4C Pause
For sample 5 and 6:
1- 95C for 2min 2- 95C for 30sec 3- 59C for 1min  4- 75C for 2min     Step 2-3-4 x 30 5- 75C for 5min 6- 4C Pause
Tomorrow the team will run the products on the gel and analyse them.
3) Inoculating LB with wt B. subtilis
We added 3ml of LB to a test-tube and, under aseptic conditions, inoculated it with a colony taken from a plate containing wild type Bacillus subtilis. The tube was then transferred to the orbital incubator (37°C) and allowed to grow overnight.
Promoter Library Sub-Project
Introduction
Up until now, this sub-project has seen the pGFP-rrnB plasmid backbone digested and linearised with restriction enzymes EcoRI and NheI. The resulting 8kb fragment (which is the plasmid backbone) has been successfully excised from agarose gel and purified; it is into this plasmid backbone in which we will place our sigA promoters.
Today's work involves the amplification of the promoter library (a collection of sigA promoters specified by degenerate sequences) using PCR.
Preparing the Primers
The first task carried out was the rehydration of the primers involved with this sub-project. A final concentration of 200uM is the desired value and to get this for each primer, the following calculation was observed:
Amount of water to be added (ul) = amount of primer (nMoles) x 5
The table below shows the oligos involved in this practical along with the primer amounts (in nMoles) and the amount of PCR water needed (in ul) to make the primers at a concentration of 200uM
A table showing the oligos with the amount of water needed
Oligo
Purpose
Amount (nMoles)
PCR Water added (ul)
IDT 60396420
Forward Primer
19.6
98.0
IDT 60396419
Reverse Primer
21.9
109.5
alta D68423AW
Control (C0)
0.06
300
alta D68425AW
Variant 2 (V2)
0.06
300
alta D68424AW
Variant 3 (V3)
0.06
300
alta D68426AW
Variant 1 (V1)
0.06
300
alta D68427AW
19mer Primer
0.08
400
All of these oligos were then aliquoted and further diluted 1:100 with PCR water and stored in the -20C freezer. The primer stocks are kept in Neil's yellow box whereas the dilutions in enzymes are kept in the a separate box.
Preparing the PCR
The table below shows the amounts of substances added to each PCR reaction:
PCR Reaction Table
Sample
No.
Water (ul)
PCR Mix (ul)
Forward Primer (ul)
Reverse Primer (ul)
Template (ul)
Polymerase used (ul)
Total Volume (ul)
C0
1
31
15
1
1
200uM (1ul)
Taq (1ul)
50
V1
2
31
15
1
1
200uM (1ul)
Taq (1ul)
50
V2
3
31
15
1
1
200uM (1ul)
Taq (1ul)
50
V3
4
31
15
1
1
200uM (1ul)
Taq (1ul)
50
The PCR Mix was composed of the following substances:
30ul dNTP's (6 x 5ul)
30ul Buffer (6 x 5ul)
30ul DMSO (6 x 5ul)
The Taq polymerase used needed to be prepared too. 6ul of pure Taq enzyme was taken and 1ul was diluted in 5ul of PCR water. 1ul of this diluted Taq was used per tube.
Carrying out the PCR
The cycle for the PCR was as follows:
One cycle involving the reverse primer pGFGrrnBrev only with the large degenerate oligo as a template - this is to convert the single-stranded oligo into double stranded DNA. To denature the oligo 90C was needed, to anneal the oligo 60C was needed and to extend the oligo 72C was needed.
The forward primer should then be added and the mixture given 30 cycles (each cycle lasting 1 minute) of 90C denaturing, 54C annealing and 72C extending.
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
News
Events
20 – 21 June 2009 - Europe workshop (London)
23 – 24 June 2009 - UK iGEM meetup (Edinburgh)
23 October Practice Presentation (Newcastle)
23 October T-shirts are ready
27 October Practice Presentation (Sunderland)
27 October Poster is ready
30 October – 2 November 2009 - Jamboree (Boston)
Social Net
Newcastle iGEM Twitter
Newcastle on Facebook
Newcastle Youtube Channel
Retrieved from " http://2009.igem.org/Team:Newcastle/Labwork/1_September_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
