"
Page
Discussion
View source
History
teams
Log in
 
Team:Hendrix Arkansas/Project
From 2014.igem.org
2014
Home
Project
Project Page
Parts
Notebook
Safety
Team
Team
Official Team Profile
Attributions
Using synthetic biology to engineer a visible alkane-detecting strain of Yarrowia  lipolytica yeast
Abstract
This summer, Hendrix College entered the first Arkansas team to compete in the International  Genetically Engineered Machine competition (iGEM). The overall goal of our  project is to engineer a biological machine that can detect cancer. Skin cancers, such as  melanoma, are known to give off volatile compounds that can be detected by trained dogs. The  volatile compounds given off by cancer cells include several alkanes. The purpose of our  experiment is to engineer a strain of Yarrowia lipolytica that is capable of detecting and growing  on alkanes to turn color in the presence of these volatile compounds. It is our hope that we can  use this yeast to cheaply and non-invasively (perhaps in a cancer detecting band-aid) to detect  melanoma. In order to achieve our goal, we will build a reporter construct that uses an alkane  sensitive promoter to drive the expression of a blue chromoprotein from the coral Acropora  millepora. This construct should cause the yeast to turn blue in the presence of alkanes. We first  want to test the ability of the blue chromoprotein to be expressed in yeast cells, so we have  generated an expression vector to express the gene in Saccharomyces cerevisiae. We have also  obtained a sample of Yarrowia lipolytica and are in the process of generating the parts required  to build the reporter construct. These parts include an alkane response element (3xARE1), a leu  minimal promoter, our reporter gene (blue chromoprotein), and a terminator (XPR2). Once the  reporter construct is constructed, we will clone it into a Yarrowia lipolytica expression vector  and integrate it into the yeast. If our experiment is successful, our cancer detecting band-aid  could allow for easier and earlier detection of melanoma.
Smelling Cancer
For the iGEM competition, the Hendrix team has decided to engineer an alkane responsive  strain of yeast, Yarrowia lipolytica, to activate the expression of a visible reporter gene in the  presence of volatile alkanes. We hope that this biological machine would be useful for the  detection of skin cancer in a non-invasive approach. This approach was motivated by stories of  dogs that could successfully identify cancer in humans. According to McCulloch et al. (2006),  the first documented instance of dog-patient interactions leading to a diagnosis of cancer  occurred in 1989. Canines are able to detect the presence of volatile compounds in  concentrations as low as parts per trillion and have been trained to detect lung, breast, and  ovarian cancer in exhaled breath samples (McCulloch et al., 2006). To determine which volatile  compounds are given off by cancer cells, Abaffy et al. (2011) used gas chromatography/mass  spectroscopy (GC/MS) on the volatile compounds given off by melanoma samples as  compared to non-cancerous skin. They identified 23 volatile compounds that were detected  only in melanoma samples and these compounds included the alkanes decane and undecane  (Abaffy et al., 2011). Based on this research, we decided to try to develop a biological machine  to detect the presence of alkanes in the hope that it will be able to specifically detect  melanoma.
Alkane Detecting Yeast
In order to detect the volatile alkanes given off by melanoma cells, we took advantage of a strain of yeast that can grow using alkanes as a sole carbon source. Yarrowia lipolytica is a non-conventional, non-pathogenic yeast that can utilize n-alkanes as a carbon source (Barth and Gaillardin, 1997). Once inside the cell, the n-alkanes induce the expression of cytochrome P-450 which hydroxylates the n-alkanes as the initial step in their utilization (Barth and Gaillardin, 1997). By comparing the promoter sequences of the cytochrome P-450 and other alkane inducible genes, Yamagami et al. (2004) identified a cis-acting alkane responsive element (ARE) and showed that it could drive the expression of a lacZ reporter gene in the presence of alkanes. We would like to take advantage of this finding and engineer a strain of Yarrowia lipolytica to express a visible reporter gene in the presence of alkanes. To accomplish this, we plan to generate a reporter construct that is similar to the one used by Yamagami et al. (2004)(Figure 2). Our construct will contain three copies of the alkane responsive element upstream of a minimal promoter sequence from the leu gene. We will use this promoter to drive the expression of a blue chromoprotein from the coral Acropora millepora (Alieva et al., 2008). After the coding region of the chromoprotein, we will include a terminator sequence from the alkaline extracellular protease gene (XPR2). This construct is similar to the one that was shown to work by Yamagami et al. (2004) except for the use of a different reporter gene (blue chromoprotein vs. lacZ). We will integrate the final construct into Yarrowia lipolytica and test for the ability of volatile alkanes to induce the expression of the reporter gene.
Figure 2: The design for our final device with its parts in order, which include: three copies of the alkane responsive element (3xARE1), the Leu minimal promoter (mLeu), the blue chromoprotein reporter gene (Chromoprotein), and the terminator sequence from the XPR2 gene (XPR2t).
Building a Yeast Blue Chromoprotein Expression Vector
To test the ability of yeast cells to express the Acropora millepora blue chromoprotein, we first constructed an expression vector that would express the reporter gene in the conventional yeast Saccharomyces cerevisiae. We obtained a plasmid vector, pSB1C3, containing the blue chromoprotein (part K592009, Figure 3) from the iGEM parts repository. We transformed the plasmid into bacteria, grew up overnight cultures, and performed a miniprep to purify the plasmid DNA. To isolate the coding sequence of the blue chromoprotein, we digested the construct with XbaI and SpeI. We also obtained an S. cerevisiae expression vector (p416Gal1, Figure 4) that uses the Gal1 inducible promoter to express heterologous genes (generous gift from Dr. Andrea Duina). We grew up an overnight culture and performed a miniprep to purify the plasmid DNA. To prepare the vector for subcloning, we digested the vector with XbaI and dephosphorylated the ends to prevent recircularization. We ligated the XbaI – SpeI fragment of the blue chromoprotein into the linearized p416Gal1 vector (SpeI and XbaI have compatible overhangs allowing hybridization to occur but both restriction enzyme recognition sites are lost after ligation) and transformed the construct into bacteria. Since the ligation was not directional, we screened the resulting colonies for the direction of the insert by digesting the miniprep DNA with XbaI and SpeI. Since the hybrid XbaI/SpeI site cannot be digested by either enzyme, we would expect to get an ~700bp fragment from an XbaI – SpeI digest only if the K592009 is in the correct orientation. Figure 6 shows that we obtained one clone with the insert in the correct orientation for expression.
Figure 3: The pSB1C3 plasmid vector containing the blue chromoprotein reporter gene (K592009), the pMB1 origin of replication, chloramphenicol resistance gene, and the EcoRI, XbaI, SpeI, and PstI restriction sites.
Figure 4: The backbone of the p416Gal1 yeast expression vector with a Gal1 promoter, XbaI and SpeI restriction sites, cyc1 terminator, URA+ selectable marker, ampicillin resistance gene, the CEN6/ARSH4 centromere for replication in yeast cells and the pMB1 origin for replication in bacteria.
Figure 5: The backbone of the blue chromoprotein expression vector containing the blue chromoprotein reporter gene (K592009) flanked by Xba1 and Spe1 restriction sites, and the hybrid XbaI/SpeI restriction site that cannot be digested.
Figure 6: Gel electrophoresis of several clones of the blue chromoprotein expression vector. DNA from each clone was digested with XbaI and SpeI and separated on the gel. Clone A (lane 1) contained the insert in the correct orientation as evidenced by the presence of an ~700 bp band on the gel.
Can Yeast Express a Blue Chromoprotein From Coral?
To test whether the blue chromoprotein from Acropora millepora would be expressed in yeast cells, we transformed the blue  chromoprotein expression construct and the p416Gal1 vector alone into a ura-minus strain of S. cerevisiae and selected for growth on SC -Ura media. We selected 3 colonies from each transformation and streaked them onto fresh SC -Ura plates to grow. We then made patches of each clone on an SC -Ura plate and after they had grown up, we replica plated the patches onto SC -Ura with glucose and SC -Ura with galactose and rafinose as a carbon source. In the presence of galactose, and the absence of glucose, the Gal1 promoter is activated and leads to the expression of the blue chromoprotein (Figure 7). While S. cerevisiae does express the blue chromoprotein, it took 72 hours for the color to be expressed. In the interest of finding a better reporter gene we are currently testing a GFP, an RFP, and a blue/green chromoprotein to see if they result in a more obvious color change more quickly.
Figure 7: Two SC –URA plates with glucose (A) and galactose/rafinose (B) as a carbon source. Three clones of S. cerevisiae transformed with the blue chromoprotein expression vector (1) or with the p416Gal1 vector alone (2).
Building Our Device
To build our Yarrowia lipolytica expression construct, we have to generate several parts. These include an alkane response element (3xARE1), a leu minimal promoter, our reporter gene (blue chromoprotein), and a terminator (XPR2t) that we can assemble in the correct order. The 3xARE1 part is 157 bp in length so we will synthesize this part by synthesizing four oligonucleotides of 73+ base pairs each, and then annealing them in pairs to form two double stranded molecules approximately 80 base pairs long with compatible overhangs. We will then ligate these two fragments into the pSB1C3 vector to create the part. To generate the Leu minimal promoter and the XPR2t terminator, we have ordered oligonucleotides to PCR amplify the sequences from Y. lipolytica genomic DNA. We designed restriction sites into the primers so both parts can be cloned independently into pSB1C3. The blue chromoprotein was previously prepared by an iGEM team and is available from their parts registry. We will use either the blue chromoprotein, GFP, RFP, or the blue/green chromoprotein in our construct depending on which one shows the highest level and fastest expression in yeast cells. Once we have created each of the parts, we will assemble them using standard biobrick cloning techniques to generate a Y. lipolytica expression vector (Figure 2).
Conclusion and Future Directions
We successfully engineered a strain of S. cerevisiae that was able to express the blue chromoprotein (K592009) reporter gene. Although the yeast was able to express the blue chromoprotein, the blue color was not apparent until 72+ hours of incubation. Due to the extended incubation period necessary for expression of the blue chromoprotein, we have decided to explore other reporter genes such as the red fluorescent protein (RFP), green fluorescent protein (GFP), and a blue/green chromoprotein.
References
Abaffy, T., Möller, M., Riemer, D. D.,  Milikowski, C., and DeFazio, R. A. A case  report - Volatile metabolomic signature of  malignant melanoma using matching skin as  a control. J Cancer Sci Ther. 2011; 3:140- 144.
Alieva, N. O., K.A. Konzen, S.F. Field, E. A. Meleshkevitch, M. E. Hunt, V. Beltran-Ramiraz, D. J.  Miller, J. Wiedermann, A. Salid, M. V. Matz (2008) Diversity and Evolution of Coral Flourescent  Proteins. PLoS vol 3, issue 7.
Barth, G., C. Gaillardin (1997) Physiology and Genetics of the Dimorphic Fungus Yarrowia  lipolytica. FEMS Microbiology Reviews vol. 19, pp. 219-237.
Blanchin-Roland, S., R. R. Cordero Otero, C. Gaillardin (1994) Two Upstream Activation  Sequences Control the Expression of the XPR2 Gene in the Yeast Yarrowia lipolytica. Molecular  and Cellular Biology, vol. 14, NO. 1, pp. 327-338.
Blazeck, J., L. Liu, H. Redden, H. Alper (2011) Tuning Gene Expression in Yarrowia lipolytica by a  Hybrid Promoter Approach. Applied and Environmental Microbiology, vol. 77, NO. 22, pp. 7905- 7914.
Davidow, L. S., M. M. O’Donnell, F. S. Kaczmarek, D. A. Pereira, J. R. DeZoeuw, A. E. Franke  (1987) Cloning and Sequencing of the Alkaline Extracellular Protease Gene of Yarrowia lipolytica.  Journal of Bacteriology, vol. 169, NO. 1, pp. 4621-4629.
Hirakawa, K., S. Kobayashi, T. Inoue, S. Endoh-Yamagami, R. Fukuda, A. Ohta (2009) Yas3p, an OPi1 Family Transcription Factor, Regulates Cytochrome P450 Expression in Response to n-Alkanes in Yarrowia lipolytica. The Journal of Biological Chemistry, vol. 284, NO. 11, pp. 7126-7137.
Iida, T., A. Ohta, M. Takagi (1998) Cloning and Characterization of an n-Alkane-Inducible  Cytochrome P450 Gene Essential for n-Decane Assimilation by Yarrowia lipolytica. Yeast, vol. 14,  pp. 1387- 1397.
Juretzek, T., H. Wang, J. Nicaud, S. Mauersberger, G. Barth, (2000) Comparison of Promoters  Suitable for Regulated Overexpression of Beta-Galactose in the Alkane-Utilizing Yeast Yarrowia  lipolytica. Biotechnol. Bioprocessing Eng. vol.5, pp. 320-326.
Juretzek, T., M. Le Dall, S. Mauersberger, C. Gaillardin, G. Barth, J. Nicaud (2001) Vectors for  Gene Expression and Amplification in the Yeast Yarrowia lipolytica. Yeast, vol. 18, pp. 97-113.
Madzak, C., S. Blanchin-Roland, R. R. Cordero Otero, C. Gaillardin (1999) Functional Analysis of  Upstream Regulating Regions from the Yarrowia lipolytica XPR2 Promoter. Microbiology, vol.  145, pp. 75-87.
Madzak, C., C. Gaillardin, J. M. Beckerich (2004) Heterologous Protein Expression and Secretion  in the Non-Conventional Yeast Yarrowia lipolytica: a Review. Journal of Biotechnology vol. 109,  pp. 63-81.
Madzak, C., B. Treton, S. Blanchin-Rolland (2000) Strong Hybrid Promoters and Integrative  Expression/Secretion for Quasi-Constitutive Expression of the Heterogeneous Proteins in the  Yeast Yarrowia lipolytica. Journal of Molecular Microbiology Biotechnology, vol. 2, pp. 207-216.
Mumberg, D., R. Muller, M. Funk (1994) Regulatable Promoters of Saccharomyces cerevisiae:  Comparison of Transcriptional Activity and their Use for Heterologous Expression. Nucleic Acids  Research, vol. 22, NO. 25, 5767-5768.
Sumita, T., T. Iida, S. Yamagami, H. Horiuchi , M. Takagi, A. Ohta (2002) YIALK1 Encoding the  Cytochrome P450 ALK1 in Yarrowia lipolytica is Transcriptionally Induced by the n-Alkane  through Two Distinct cis-Elements on its Promoter. Biochemical and Biophysical Research  Communications, vol. 294, pp. 1071-1078.
Yamagami, S., T. Iida, Y. Nagata, A. Ohta, M. Takagi (2001) Isolation and Characterization of  Acetoacetyl-CoA Thiolase Gene Essential for n-Decane Assimilation in Yeast Yarrowia lipolytica.  Biochemical and Biophysical Research Communications vol. 282, pp. 832-838.
Yamagami, S., D. Morioka, R. Fukuda, A. Ohta (2004) A Basic Helix-Loop-Helix Transcription  Factor Essential for Cytochrome P450 Induction in Response to Alkanes in Yeast Yarrowia  lipolytica. Journal of Biol. Chem. vol. 279, pp. 22183-22189.
Retrieved from " http://2014.igem.org/Team:Hendrix_Arkansas/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
