Team:Dalhousie Halifax NS/Experiments
Toggle navigation
About Us
Our Team
The Blog(current)
The Project
Description
Experiments
Proof of Concept
Results
Notebook
Outreach
Human Practices
Engagement
Shubenacadie Wildlife Park
Microbiome Map
Safety
Attributions
Parts
Papers and Presentations
Awards
Dalhousie iGEM 2016
Experiments
High-Throughput Sequencing
Fecal Sample DNA Extraction
To extract environmental DNA from fecal samples. This extraction will purify genomic DNA from prokaryotic and eukaryotic cells found in the fecal sample.
DNA Concentration
Concentration protocol to concentrate DNA isolating using the PowerFecal Extraction kit.
16S Illumina Miseq Sequencing
Illumina MiSeq Sequencing was used to identify the sequences of the 16S rRNA genes in the fecal samples. This step was undertaking by the Integrated Microbiome Resource at Dal. There protocol was provided to us.
Metagenomic Shotgun Illumina Miseq Sequencing
This protocol was used by the Integrated Microbiome Resource to sequence the metagenomic DNA in our fecal samples from the porcupine, beaver, arctic wolf and coyote.
16S Sequencing Analysis
This protocol was used for analyzing the 16S Illumina Sequencing data from the illumina sequencer. Provided by the Integrated Microbiome Resource at Dalhousie University.
Metagenomic Sequencing Analysis
This protocol was used for analyzing the Metagenomic Illumina Sequencing data from the illumina sequencer. Provided by the Integrated Microbiome Resource at Dalhousie University.
Isolation of Bacteria
Preparation of Cellulose Media
Congo Red Cellulose media can be used to isolate bacteria that degrade cellulose. It does this because the only carbon source in the media is cellulose. It uses Whatman Paper and gelatin.
16S Colony PCR
Colony PCR has many applications including; verifying that an insert has been successfully incorporated in plasmid constructs, insert orientation, and species determination of unknown samples. The later was the reason for our use of Colony PCR. After streaking fecal sample onto cellulose plates, we picked colonies of different morphologies to find out which species or genus they were. We did this by using primers that targeted conserved sequence found in bacteria so we could amplify the DNA of almost all bacteria.
ExoSapIt and DNA Cleanup
Instead of PCR clean up, we used ExoSap It. ExoSap It allows for an enzymatic clean-up of left over dNTPs and single stranded DNA. It is a very quick procedure as the only activation ExoSap It needs is a 15 minute 37°C incubation and a 15 minute 80 °C inactivation.
Metagenomic Library and Transformation
Metagenomic Library Construction
Protocol that takes extracted fecal DNA, prepares it for ligation and clones it into a cosmid. This cosmid can be packaged into phage particles and those can be used to place the cosmid into E. coli cells which will express environmental DNA that had been placed into the cosmid.
Heat-Shock Transformation
Protocol for the heat-shock transformation of plasmids into chemically competent E. coli cells.
Chemical Analysis
Steam Distillation
Steam Distillation was used to separate the terpenes from pine tree oleoresin without degrading them. Through GC/MS analysis we confirmed that this procedure worked.
Terpene Inhibition Experiments
These experiments were used to test E. coli and S. cerevisiae tolerance to terpenes, both in pure form and the terpenes isolated from oleoresin.
