<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dalhousie_Halifax_NS_Proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dalhousie Halifax NS/Proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1" style="font-size: 12pt;"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">About Us</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Team">Our Team</A></LI><LI><A href="http://www.daligem2016.wordpress.com/">The Blog<SPAN class="sr-only">(current)</SPAN></A></LI></UL><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">The Project</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Notebook">Notebook</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Shubenacadie">Shubenacadie Wildlife Park</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Integrated_Practices">Microbiome Map</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/PapersandPresentations">Papers and Presentations</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Awards">Awards</A></LI></DIV></DIV><DIV id="topper" class="jumbotron"><H1 id="title">Metagenomic Library Proof Of Concept</H1><P style="font-size:20pt; text-align: center; color: white; font-family:lato-black;"><A id="nav" href="#initial">Initial 16S rRNA Sequencing</A> | <A id="nav" href="#PICRUSt">PICRUSt</A> | <A id="nav" href="#Conclusions">Conclusions</A></P></DIV><DIV class="container"><H3 style="font-family:lato-black;color:#E810A0;"> Would it work? </H3><P id="description">Using gut microbes as a &quot;mine&quot; for enzymes using a metagenomic library approach is a fairly novel idea. This concept does raise some questions.</P><P id="description" style="font-family:lato-heavy"> Is it even possible?</P><P id="description" style="font-family:lato-heavy"> How different are microbiomes to begin with, will different animals have different microbiomes? </P><P id="description" style="font-family:lato-heavy"> What enzymes are we likely to find in what animals? </P><P id="description" style="font-family:lato-heavy">Are the enzymes found in each animal predictable?</P><P id="description">These questions guided our proof of concept experiments and allowed us to gather some evidence for our approach. This page will answer the questions raised above and will act to provide some of evidence to the feasibility of our approach.</P></DIV><DIV class="container"><H3 style="font-family:lato-black;color:#C410E8;"> How did we find evidence? </H3><P>The Integrated Microbiome Resource at Dalhousie does 16S and metagenomic DNA sequencing and is a bioinformatic hub at the University. With their support we sequenced the 16S rRNA genes found in the environmental DNA extracted from feces of 21 mammals at the Shubenacadie Wildlife Park. With this information we were able to address a few of the questions that were mentioned above. We then chose the porcupine, beaver, arctic wolf and coyote samples to sequence in replicate, and then applied a bioinformatic tool called PICRUSt (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) to obtain information of microbial gene content found in these fecal samples. These animals were chosen because the beaver microbiome is similar to the porcupine microbiome according to our 16S data, and the arctic wolf and coyote microbiomes are similar to each other but different from the beaver and porcupine microbiomes.  Our goal is to use microbial diversity and gene content in fecal samples as an approximation of the microbiomes of the mammals at the Wildlife Park. With this approximation we can determine what enzymes we are likely to find in the microbiomes of particular animals.</P><H3 style="font-family:lato-black;color:#10E8C4;">Results</H3><P>These are the results of our sequencing and bioinformatic analysis separated by section and tool:</P><H3 style="text-align:left;font-family:lato-black;color:#10A0E8;" id="initial">Initial 16S Sequencing<P>The initial 16S sequencing provided us with an overall picture of species diversity and similarity, and allowed us to create beta-diversity plots that compare the similarities of the microbiomes that were sequenced. A in figure 1 shows a bar graph that was constructed from the 16S data: </P><P>This bar graph shows us two things. The first is that the microbiomes of these different animals are very diverse. There are significant differences between each sample and there is great microbial diversity. This finding is important because we would want a diverse representation of genetic content to start with when building a metagenomic library. The second is that you can notice similarities between the samples. The porcupine and beaver samples share a microbe that is of the family S24-7 and the order Bacteroidales, which is denoted by a light blue colour. You can also notice that the red deer, moose and elk share a large brown bar that matches up to the order Bacteriodales. This is by far an exhaustive list of similarites, but these two function to illustrate these similarities. To further show similarities, we built a beta-diversity plot (Figure 1. B) that allows us to see how similar or how different each sample is. This is particularly interesting because the microbiomes tend to group by family of the animal they came from. So if indeed their microbiomes contain a useful and diverse group of enzymes, then we could choose multiple animals from a similar family to build a metagenomic library.</P><DIV class="well well-lg" style="background-color:#A9D091;"><P>Figure 1.1. Bar Graph and Beta-Diversity Plot for Microbiome Survey Data of 21 Mammals from the Shubenacadie Wildlife Park. <A href="https://static.igem.org/mediawiki/2016/f/fc/T--Dalhousie_Halifax_NS--BarGraphLegend.pdf" style="padding-top:20px;">Bar Graph Legend here</A></P><P>Figure 1.2. Beta-Diversity Plot of Animal Microbiomes shows relationship between microbiomes.</P></DIV><P>From this initial 16S sequencing, it was noticed that the microbiomes of the Arctic Wolf and Coyote were similar, while those of the beaver and porcupine were similar. Microbiomes of the Canidae and the Rodentia, however, were different from one another. We then decided to re-seqeuence new samples from the coyote, arctic wolf, beaver and porcupine in order to apply these to PICRUSt and see what gene content these samples have. Results of the analysis are found below.</P></H3><H3 style="text-align:left;font-family:lato-black;color:#1034E8;" id="PICRUSt">PICRUSt</H3><P>Using PICRUSt allows us to predict the gene content found in 16S samples. It does this by matching 16S sequences to a metagenomic library of known sequenced organisms or a metagenome whose gene content has been inferred based on the ancestors of that metagenome's gene content. What we found is that, in many cases, the gene content of the microbiomes of the beaver and porcupine was similar, and it was similar in enzymes in which we were particularly interested. These results can be seen in the bar graphs for the cellulobiose phosphorylase enzyme and the endoglucanase enzyme. Cellulobiose phosphorylase is an important enzyme for the degradation of cellulobiose and its maintenance inside the cell. Endoglucanase breaks down the cellulose crystal into cellulobiose, by cleavage of internal bonds.. Expression of these two enzymes in an <EM>E. coli</EM> cell could enable that host cell to degrade cellulose.</P><P style="padding-top:25px; padding-bottom:25px;">This same PICRUSt analysis was done on enzymes that we might expect to find in the Arctic Wolf and Coyote microbiomes more often than in the microbiomes of the beaver and porcupine. These results are found in figure 3. The arctic wolf and the coyote microbiomes would see starch more often than would those of the porcupines and the beavers. For this reason we would expect to see higher levels of the enzyme starch phosphorylase in the arctic wolf and coyote microbiomes. That is, indeed, what we see. The second enzyme that we found at a higher frequency in the arctic wolf and coyote microbiomes was the transketolase enzyme. We do not have a good explanation as to why we see this, but it is interesting, nonetheless, that we see it at a higher frequency in the arctic wolf and coyote microbiomes. This shows that enzyme gene content does vary between microbiomes, and it actually varies similarly between different animals. For example, we see higher gene content for cellulose degrading enzymes in the beaver and porcupine, but we see lower content for starch degrading enzymes. The opposite trend is seen in the arctic wolf and coyote microbiomes.</P><H3 style="font-family:lato-black;color:#5810E8;" id="Conclusions">Conclusions</H3><P>All of the above information leads us to believe that our metagenomic library idea would provide a good approach to cloning desired genes into a host organism. Not only did the gene content in the porcupine microbiome match our predictions, we also found that we could use other animals that are similar to the porcupine to isolate bacteria whose genomes encode cellulose degrading enzymes. We also found that other animals might host genes for useful enzymes in their microbiomes, and those are also predictable. We are confident that a metagenomic library built from fecal sample DNA of animals whose microbiomes we expect to perform a particular function would, indeed, lead us to useful enzymes.
</P></DIV><FOOTER class="footer"><DIV class="container"><P class="text-muted">Dalhousie iGEM 2016</P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>