<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dalhousie_Halifax_NS_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dalhousie Halifax NS/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1" style="font-size: 12pt;"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">About Us</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Team">Our Team</A></LI><LI><A href="http://www.daligem2016.wordpress.com/">The Blog<SPAN class="sr-only">(current)</SPAN></A></LI></UL><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">The Project</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Notebook">Notebook</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Shubenacadie">Shubenacadie Wildlife Park</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Integrated_Practices">Microbiome Map</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/PapersandPresentations">Papers and Presentations</A></LI><LI><A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Awards">Awards</A></LI></DIV></DIV><DIV class="jumbotron" id="header"><H1 id="title" style="font-family:lato-heavy;color:white;opacity:0.85;text-align:center;"> Results </H1><P style="font-size:20pt; text-align: center; color: white; font-family:lato-black;"><A id="nav" href="#resultspart">Our Results</A> | <A id="nav" href="#future">For The Future</A></P></DIV><DIV class="container" id="summary" style="width:80%;"><H3 style="font-family:lato-heavy;text-align:center;">This year we managed to isolate a few cellulose degrading bacteria and identify them. We also did an extensive proof of concept for our plan to use microbiomes as a &quot;mine&quot; for enzymes and gathered evidence that it would, likely, enable us to identify genes that encode cellulose degrading enzymes. We showed that growth of an E. coli strain used to degrade soft-wood lumber waste would not be inhibited by the anti-microbial properties of terpenes found in trees milled for softwood lumber. We also created a couple of new biobricks containing enzymes that we identified bioinformatically from <EM style="font-family:lato-black-italic;">Ruminiclostridium thermocellum</EM>, a cellulose degrader identified in the porcupine microbiome. Next year we intend to move forward with the metagenomic library and isolate more enzymes that would be useful for converting soft-wood lumber waste to a source of efficient fuel (ethanol) or be useful for bioremediation.</H3></DIV><DIV class="jumbotron text-center" id="resultspart"><H1 style="font-family:lato-black; opacity:0.85;"> Our Results </H1></DIV><DIV class="container"><H2 style="font-family:lato-black;text-align:center;color:#1710E8;">DNA Sequencing and Bacterial Isolation</H2><H3 style="color:#E81D10;">Bacterial Isolation:</H3><P>Using the cellulose media, we plated porcupine fecal samples to inspect which species could be cultivated and would degrade cellulose. We did this both aerobically and anaerobically. Pictures of these plates can be found below. Some of these colonies were subjected to colony PCR for identification and those results can be found below. We were able to grow many colonies on the cellulose media plates, showing us that there are many cellulose bacteria in the porcupine gut.</P></DIV><DIV class="container"><H3 style="color:#E88910;">Colony PCR:</H3><P>The colony PCR that was done with the cellulose-degrading colonies can be seen on the gel to the left. A 1kb ladder can also be seen on this image. Most of the PCR reactions were successful here, so the DNA was quantified used the geldoc and was purified with ExoSapIt before being sent to GeneWiz for sequencing.</P><DIV class="col-md-8"><P> All of these samples (1A, 1B, 2A and 2B) were sequenced and identified using NCBI's BLAST. They all came back as the genus <EM>Proteus</EM>. They came from the porcupine fecal sample, on two separate cellulose plates grown aerobically at 37 degrees celcius.</P></DIV><DIV class="col-md-12"><P>Other colony PCR experiments were done on three different colony morphologies found on the porcupine fecal sample cellulose plate: white colonies, yellow colonies and red bulls-eye colonies. After 16S sequencing the yellow colony was identified as <EM>Escherichia coli</EM> and the red bulls-eye colony was identified as <EM>Stenotrophomonas rhizophila</EM>. The sequencing of the white colony morphology proved challenging. Therefore we streaked this bacteria out on LB and McConkey agar in an effort to identify it. The LB plate can be seen below:</P><DIV class="col-md-8"><P>The McConkey streak came back as positive lactose fermenting. With the brown streak pattern and the lactose fermenting we labelled this bacterium as likely a <EM>Sphingomonas</EM> sp.</P></DIV></DIV></DIV><DIV class="container"><H3 style="color:#E8AA10;">High-Throughput Sequencing:</H3><P>The high-throughput sequencing part of our project was done mostly to provide evidence for the feasibility of creating a metagenomic library from microbiome DNA. The results are documented in detail on our <A href="https://2016.igem.org/Team:Dalhousie_Halifax_NS/Proof">Proof of Concept</A> page.</P><P>We were able to sequence the microbiomes of 21 mammals from the Shubenacadie Wildlife Park with the help of the Integrated Microbiome Resource at Dalhousie University. We discovered the diversity of bacteria living in animal microbiomes is very large! This can be seen in A of figure 1 below. We found that the microbiomes of animals that are of similar families, or are closely related evolutionarily, correlate with the similarity of their microbiomes. This can be seen in B of figure 1 below.</P><DIV class="col-md-12 centered" style="border-radius:25px; background-color:#A9D091;margin-bottom:25px;"><H4 style="font-family:lato-black">Figure 1. 16S Bar Graph and Beta-Diversity Plot</H4></DIV><P>We did a PICRUSt analysis on this 16S data from 4 animals in the park whose microbiomes we re-sequenced. These resequenced samples included the beaver, the porcupine, the arctic wolf and the coyote. The PICRUSt analysis specifically looked at the differences in the gene content of endo-glucanase and cellubiose phosphorylase in the re-seqeunced samples. The bar chart produced can be seen below in figure 2. It can be noticed that the beaver and porcupine microbiomes have significantly higher proportions of these cellulose degrading enzymes than the arctic wolf or coyote microbiomes.</P><P>We also did a second analysis looking at enzymes found at a higher proportion in the arctic wolf and coyote. It can be seen in figure 3 that the transketolase enzyme and the starch phosphorylase enzyme are found with higher proportion in the arctic wolf and coyote samples than in the beaver and porcupine samples.</P><P>This information provides some evidence that our future metagenomic library will be a fruitful endeavour. Making a beaver microbiome metagenomic library should yield plenty of useful cellulose-degrading enzymes.</P></DIV></DIV><DIV class="container"><H2 style="color:#10E889;">Chemical Analysis</H2><H3 style="color:#10E81D;">Gas Chromatography/Mass Spectrometry:</H3><P>We analyzed two samples using GC/MS; commercial (store-bought) turpentine, and our home-made (steam-distilled) terpenic extract. In gas chromatography, analytes are first seperated by boiling point - they are then seperated based on their affinity to the column. The analytes are then identified using mass spectrometry. </P><P>We found that the commercial stuff contained mostly D-Limonene, as well as a variety of other terpenes (with alpha-pinene being present in very trace amounts). In comparison, the terpenic extract contained mostly alpha-pinene and beta-pinene in a 3:1 ratio.</P><H3 style="color:#6FE810;">Bacterial Inhibition with Terpenes:</H3><P>We tested for bacterial inhibition first by simply culturing <EM>E. coli</EM> on solid media with a dab of pine tree resin in the center. We expected we would find a zone of inhibition, but did not observe such a thing.</P><P>After isolating the terpenic extract from the resin, we repeated similar experiments by soaking into Whatman filter paper and placing it in the culturing plate. No zone of inhibition was observed.</P><P>We moved on to liquid media, and solubilized the water-insoluble terpenic extract using DMF (N,N-dimethylformamide). A series of culturing tubes were innoculated with <EM>E. coli</EM> and varying amounts of the terpenic extract were added; we started at a concentration of 4.3 mg/mL and worked our way down, and then measured turbidity -- and found no inhibition. This procedure was also carried out using pure alpha-pinene (and we still found no inhibition).</P><P>Additionally, we tested the residue which remained after steam-distillation, and still found no inhibition.</P><P>The afforementioned experiments (in liquid culture using terpenic extracts and pure alpha-pinene) were also performed on <EM>S. cerevisiae</EM>, and clear, distinct inhibition was observed. We therefore conclude that <EM>E. coli</EM> is very resistant to the anti-microbial properties of pine tree resin, and that the terpenes would not interfere with our goals for cellulose degradation. For bioremediation, however, a different species of bacteria should be used to screen the metagenomic library (because <EM>E. coli</EM> can survive terpenes).</P><P>All of the above results, for both <EM>E. coli</EM> and <EM>S. cerevisiae</EM> are summarized in Figure 4 below.</P></DIV><DIV class="jumbotron text-center" id="future"><H1 style="font-family:lato-black; opacity:0.85; color:white;"> For the Future </H1></DIV><DIV class="container"><H2 style="color:#10BAE8;">Metagenomic Library</H2><H3 style="color:#E81075;">How?</H3><P>This year we had a bit of time to perfect the metagenomic library protocol. With this new protocol we believe that we will be able to build the metagenomic library using DNA Extracted from porcupine fecal sample and transforming that library into E. coli. We should be able to screen this library for cellulose degrading enzymes as well as terpene degrading enzymes that could prove useful in our search for new biofuel feedstocks and production solutions</P><H3 style="color:#8310E8;">Why?</H3><P>We have been able to provide evidence that a metagenomic library from DNA extracted from porcupine fecal samples would provide us with a new source of cellulose degrading enzymes. The next step would be to actually attempt the metagenomic library and search for the enzymes we have been able to identify bioinformatically.</P></DIV><FOOTER class="footer"><DIV class="container"><P class="text-muted">Dalhousie iGEM 2016</P></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>