<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IIT-Madras_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IIT-Madras/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><UL><LI><A href="https://2016.igem.org/Team:IIT-Madras"> Home</A></LI><LI><A href="https://2016.igem.org/Team:IIT-Madras/People"> People</A></LI><LI><A href="https://2016.igem.org/Team:IIT-Madras/Description"> Project</A></LI><LI><A href="https://2016.igem.org/Team:IIT-Madras/Parts"> Parts</A></LI><LI><A href="https://2016.igem.org/Team:IIT-Madras/Model"> Modelling</A></LI><LI><A href="https://2016.igem.org/Team:IIT-Madras/Human_Practices"> Human Practises</A></LI><LI><A href="https://2016.igem.org/Team:IIT-Madras/Interlab_study"> Interlab</A></LI><LI><A href="https://2016.igem.org/Team:IIT-Madras/Achievements"> Achievements</A></LI></UL><DIV class="content-wrapper"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Modularity_of_RBS_parts"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Modularity of RBS parts</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Methodology"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Methodology</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Hypothesis_and_Algorithm"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Hypothesis and Algorithm</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Quantification_of_Non-modularity"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Quantification of Non-modularity</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Optimization"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Optimization</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Results"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Conclusion"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#Noise_in_Devices"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Noise in Devices</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Introduction_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2"><A href="#Solution"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Solution</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#References"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Modularity_of_RBS_parts">Modularity of RBS parts</SPAN></H1><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>The <B>non-modular nature</B> of Ribosomal Binding Sites in bacteria is well known to the synthetic biology community. Most of the biological parts have been assigned a strength for their functionality. For example, Promoters have transcriptional score (number of RNA molecules per DNA molecule), RBSs have translational score (number of protein molecules per RNA molecule). Ideally, if we were to use a promoter and a RBS to produce a protein, we should get (<B>transcriptional score times translational score</B>) number of protein molecules. In bacterial cells, transcription and translation processes are coupled and can occur simultaneously. This is why, these process are not independent and, hence, not modular. Apart from this, RNA molecules consisting of RBS and first few codons may form secondary structures, which in turn reduce the translation efficiency. Also, rare codons have also been shown to influence translation efficiency.
</P><P>Secondary structure's and codons' influence on translational score of RBS are overlapping as they are a result of <B>A, U, G, C</B> combinations. Therefore, it is important to decouple both effects to unravel underlying patterns.
</P><P><B>We have successfully validated an empirical model to predict variations in protein expression levels, which could help future iGEM teams</B>. Here, we represent a thorough description of our work.
</P><H2><SPAN class="mw-headline" id="Methodology">Methodology</SPAN></H2><P>The dataset from &quot;Causes and effects of N-terminal codon bias in bacterial genes&quot; paper was taken. Protein expression Data was available for following constructs : 
2 promoters x 3 RBSs x 1781 (137x13) sfGFP variants in first 11 codons at N-terminal (3 RBS parts were B0034, B0032, B0030) and 2 promoter x 137 natural RBSs x 13 sfGFP variants in first 11 codons at N-terminal (2 Promoters were J23100 &amp; J23108)
</P><H2><SPAN class="mw-headline" id="Hypothesis_and_Algorithm">Hypothesis and Algorithm</SPAN></H2><P>At the beginning, we hypothesized following things based on the information available in literature:
1. <B>Expression is inversely proportional to the stability of secondary structure of mRNA near RBS part</B>.
2. <B>Rare codons present in first 11 codons of proteins have the ability to increase or decrease the translational score of RBS parts</B>.
3. <B>Each RBS part has a native strength irrespective of the promoter and protein coding part it can be used with</B>.
</P><P>We designed an algorithm to compute the translational score of a given construct(which expresses a protein) in the following way:
\begin{equation*}
TS = \dfrac{NS*C_{pref}}{1+dG} + \alpha
\end{equation*}
</P><P>\begin{equation*}
C_{pref}= C_{1}*C_{2}*C_{3}*...*C_{11}*C_{sfGFP}
\end{equation*}
</P><P><B>Objective Function</B>: minimize \(\sum \mid TS_{model}-TS_{experiment} \mid\) 
</P><P><B>Outlier Removal</B>: top scores in \(\mid TS_{model}-TS_{experiment} \mid\)
</P><P>where \(C_{i}\) is the codon preference of codon present at \(i^{th}\) position, \(C_{sfGFP}\) a constant for sfGFP protein codons, <B>TS</B> : Translational Score of RBS part, <B>NS</B> : Native Strength of RBS part,  which is equal to TS at \(\delta\)G=0, \(C_{pref}\)=1; <B>\(\delta\)G</B> : Stability of RNA strand consisting of RBS and first 11 codon of protein, 
\(\alpha\) is a constant.
</P><H2><SPAN class="mw-headline" id="Quantification_of_Non-modularity">Quantification of Non-modularity</SPAN></H2><P>As previously mentioned, RBS's have been found to be non-modular with respect to promoter and protein coding parts. A quantification of modularity would enable us to screen better RBS parts to make high order complex genetic circuits in high-throughput manner.
</P><P>\begin{equation*}
NM_p = \frac{\sigma_{NS}}{\langle {NS} \rangle} \\
NM_c = \frac{\sigma_{TS}}{\langle {TS} \rangle}
\end{equation*}
where, <B>NM<SUB>p</SUB></B> is non-modularity of RBS w.r.t. promoters; <B>NM<SUB>c</SUB></B> is non-modularity of RBS w.r.t. protein coding part; <B>NS</B> is the array of native strength of RBS part with given promoters, <B>TS</B> is the array of translational score of RBS part with given protein coding parts.
</P><H2><SPAN class="mw-headline" id="Optimization">Optimization</SPAN></H2><P>Above model was optimized to compute the unknown variables, <B>Native Strength of RBSs</B> and <B>codon preference matrix</B>, using the data from above mentioned paper. In MATLAB, <I>fmincon</I> function was used to minimize the sum of (\({TS_{model}-TS_{experiment})}^2\) for all 14137 constructs. After several iterations of optimization, we achieved following results. Optimization was done in MATLAB on a supercomputer facility at IIT Madras.
Further, the system was optimized by removing 5%, 10% outliers, which were computed as the top scores in (\({TS_{model}-TS_{experiment})}^2\).
</P><UL class="gallery mw-gallery-nolines center"><LI class="gallerycaption">Validation of Model</LI><LI class="gallerybox" style="width: 405px"><DIV style="width: 405px"><DIV class="gallerytext"><P>Experiment model, dark green 90% data, dark green+light dark green 95% data.
</P></DIV></DIV></LI><LI class="gallerybox" style="width: 405px"><DIV style="width: 405px"><DIV class="gallerytext"><P>Null model, dark green 90% data, dark green+light dark green 95% data.
</P></DIV></DIV></LI></UL><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><UL class="gallery mw-gallery-traditional center"><LI class="gallerybox" style="width: 435px"><DIV style="width: 435px"><DIV class="gallerytext"><P>Non-modularity of RBSs with Promoters
</P></DIV></DIV></LI><LI class="gallerybox" style="width: 435px"><DIV style="width: 435px"><DIV class="gallerytext"><P>Popular BioBrick RBS parts
</P></DIV></DIV></LI></UL><P>We found that some codons favor the translation process, while some of this do not. Following is a list of codons, which favor the translation process: 
</P><TABLE class="wikitable" style="font-size:0.8em;@media screen and (max-width: 1000px){font-size:0.6em;}"><TBODY><TR><TH scope="row"> Codon
</TH><TD>AAA</TD><TD>AAT</TD><TD>AGA</TD><TD>AGC</TD><TD>AGT</TD><TD>ATA</TD><TD>GAT</TD><TD>GGC</TD><TD>GGG</TD><TD>GGT</TD><TD>GTA</TD><TD>TCA</TD><TD>TCC</TD><TD>TCT</TD><TD>TGC
</TD></TR><TR><TH scope="row"> Amino Acid
</TH><TD>K</TD><TD>N</TD><TD>R</TD><TD>S</TD><TD>S</TD><TD>I</TD><TD>D</TD><TD>G</TD><TD>G</TD><TD>G</TD><TD>V</TD><TD>S</TD><TD>S</TD><TD>S</TD><TD>C
</TD></TR><TR><TH scope="row"> Preference Value
</TH><TD>1.17</TD><TD>1.23</TD><TD>1.15</TD><TD>1.13</TD><TD>1.21</TD><TD>1.14</TD><TD>1.13</TD><TD>1.19</TD><TD>1.19</TD><TD>1.21</TD><TD>1.14</TD><TD>1.14</TD><TD>1.13</TD><TD>1.19</TD><TD>1.19
</TD></TR></TBODY></TABLE><P>Following is a list of codons, which reduces the translation efficiency:
</P><TABLE class="wikitable" style="font-size:0.8em;@media screen and (max-width: 1000px){font-size:0.6em;}"><TBODY><TR><TH scope="row"> Codon
</TH><TD>CAC</TD><TD>CGC</TD><TD>CTC</TD><TD>GTC</TD><TD>TTC
</TD></TR><TR><TH scope="row"> Amino Acid
</TH><TD>H</TD><TD>R</TD><TD>L</TD><TD>V</TD><TD>F
</TD></TR><TR><TH scope="row"> Preference Value
</TH><TD>0.9768</TD><TD>0.9862</TD><TD>0.979</TD><TD>0.997</TD><TD>0.993
</TD></TR></TBODY></TABLE><P>A complete list of codons and their preference values are <A rel="nofollow" class="external text" href="https://static.igem.org/mediawiki/2016/7/71/Igemiitm16_Codonpref.xlsx">here</A></P><H2><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H2><P>We could achieve a heuristic solution with a <B>correlation of 0.87 with 90% of the data points</B> for our experimental model, while null model gives us a correlation of 0.83 with 90% data points. Null model doesn't incorporate the feature of codon preference (C<SUB>pref</SUB>). As we see that codon preference feature improves our output, hence, it justifies the fact that codons can also vary translational score of RBS parts. Experimental model gives us the strength of promoter-RBS combined strength for 280 (2 promoters x 140 RBSs) combinations. This model provides a way to predict the variations in translational score of RBS parts unlike the existing tools. Using this model, users can get translational score for their expression systems.
</P><H1><SPAN class="mw-headline" id="Noise_in_Devices">Noise in Devices</SPAN></H1><H2><SPAN class="mw-headline" id="Introduction_2">Introduction</SPAN></H2><P>In order to understand the behavior of device components, we need to have at least two signals coming from the device to understand the variations from <B>intrinsic</B> and <B>extrinsic</B> sources. Similarly, we can have two or more protein producing parts in complex biological devices, using which, we can understand the behavior of biological devices. <A rel="nofollow" class="external text" href="http://www.pnas.org/content/99/20/12795.short">Elowtiz et al</A> has done significant work in understanding noise in biological devices. We have observed that if the mean value of one signal is bigger than the other signal, Elowitz formula doesn't give us accurate noise due to under-representation of variations present in low protein expressing device. 
</P><P>\begin{equation*}
Noise_{int}=\frac{\langle {rfp-gfp} \rangle^2}{2*\langle gfp \rangle* \langle rfp \rangle};\\  
if \langle rfp \rangle &gt; \langle gfp \rangle; Noise_{int} = \frac{{\langle rfp \rangle}^2}{2*\langle gfp \rangle*\langle rfp \rangle}; \\ 
Noise_{int} = \frac{\langle {rfp} \rangle}{2*\langle gfp \rangle};
\end{equation*}
</P><UL class="gallery mw-gallery-traditional center"><LI class="gallerybox" style="width: 435px"><DIV style="width: 435px"><DIV class="gallerytext"><P>Signals from biological device
</P></DIV></DIV></LI></UL><H2><SPAN class="mw-headline" id="Solution">Solution</SPAN></H2><P>To solve this problem, we have modified the current formula as follows:
\begin{equation*}
gr_{fold}=\dfrac{\langle rfp \rangle}{\langle gfp \rangle} \\
Noise_{int}=\frac{\langle{rfp-gfp*gr_{fold}}\rangle^2}{2*\langle gfp \rangle*\langle rfp \rangle*gr_{fold}};\\  
\end{equation*}
</P><UL class="gallery mw-gallery-traditional center"><LI class="gallerybox" style="width: 435px"><DIV style="width: 435px"><DIV class="gallerytext"><P>Corrected Signals from biological device
</P></DIV></DIV></LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P><A rel="nofollow" class="external text" href="http://science.sciencemag.org/content/342/6157/475">1.</A> Goodman, Daniel B., George M. Church, and Sriram Kosuri. &quot;Causes and effects of N-terminal codon bias in bacterial genes.&quot; Science 342.6157 (2013): 475-479.

</P></DIV><DIV id="footer"><H1 style="font-family:bold; font-size:1.5em;color:white;">Contact Us</H1><P style="text-align:center;font-family:bold;color:white;font-size:1.1em;"><A target="_blank" style="font-family:bold; font-size:1em;color:white;" href="https://biotech.iitm.ac.in/welcome">Department of Biotechnology, IIT Madras</A></P><P style="text-align:center;font-family:bold;color:white;font-size:1.1em;">email: <A style="font-family:bold; font-size:1em;color:white;" target="_blank" href="mailto:igemiitm16@gmail.com">igemiitm16@gmail.com</A></P><P style="text-align:center;font-family:bold;color:white;font-size:1.1em">twitter: <A style="font-family:bold; font-size:1em;color:white;" target="_blank" href="https://twitter.com/iGEMIITMadras">iGEMIITMadras</A></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>