<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Washington_Tools_Used_Next-Gen_Cloning"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Washington/Tools_Used/Next-Gen_Cloning">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Washington/Tools_Used/Next-Gen_Cloning&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Tools_Used/Next-Gen_Cloning&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Tools_Used/Next-Gen_Cloning&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Washington/Tools_Used/Next-Gen_Cloning" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Washington/Tools Used/Next-Gen Cloning</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Washington/Project/Tools/Klavins&amp;redirect=no" title="Team:Washington/Project/Tools/Klavins">Team:Washington/Project/Tools/Klavins</A>)</DIV><DIV id="uw_leftBar"><DIV id="uw_ss"><UL id="uw_navBar"><LI><A href="/Team:Washington" title="Team:Washington"><SPAN style="color:#F0F7C1">Home</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Negative" title="Team:Washington/Gram Negative"><SPAN style="color:#F0F7C1">Gram(-) Therapeutic</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Negative/Design" title="Team:Washington/Gram Negative/Design"><SPAN style="color:#F0F7C1">Design</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Negative/Build" title="Team:Washington/Gram Negative/Build"><SPAN style="color:#F0F7C1">Build</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Negative/Test" title="Team:Washington/Gram Negative/Test"><SPAN style="color:#F0F7C1">Test Results</SPAN></A></LI></UL><LI><A href="/Team:Washington/Gram_Positive" title="Team:Washington/Gram Positive"><SPAN style="color:#F0F7C1">Gram(+) Therapeutic</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Positive/Design" title="Team:Washington/Gram Positive/Design"><SPAN style="color:#F0F7C1">Design</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Positive/Build" title="Team:Washington/Gram Positive/Build"><SPAN style="color:#F0F7C1">Build</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Positive/Test" title="Team:Washington/Gram Positive/Test"><SPAN style="color:#F0F7C1">Test Results</SPAN></A></LI><LI><A href="/Team:Washington/Tools_Created" title="Team:Washington/Tools Created"><SPAN style="color:#F0F7C1">Tools Created</SPAN></A></LI><LI><A href="/Team:Washington/Tools_Created/New_Vectors" title="Team:Washington/Tools Created/New Vectors"><SPAN style="color:#F0F7C1">New Vectors</SPAN></A></LI><LI><A href="/Team:Washington/Tools_Created/New_Software" title="Team:Washington/Tools Created/New Software"><SPAN style="color:#F0F7C1">New Software</SPAN></A></LI><LI><A href="/Team:Washington/Tools_Used" title="Team:Washington/Tools Used"><SPAN style="color:#F0F7C1">Tools Used</SPAN></A></LI><LI><A href="/Team:Washington/Tools_Used/Software" title="Team:Washington/Tools Used/Software"><SPAN style="color:#F0F7C1">Software</SPAN></A></LI><LI><STRONG class="selflink"><SPAN style="color:#F0F7C1">Next-Gen Cloning</SPAN></STRONG></LI><LI><A href="/Team:Washington/Protocols" title="Team:Washington/Protocols"><SPAN style="color:#F0F7C1">Protocols</SPAN></A></LI><LI><A href="/Team:Washington/Safety" title="Team:Washington/Safety"><SPAN style="color:#F0F7C1">Safety</SPAN></A></LI><LI><A href="/Team:Washington/Team" title="Team:Washington/Team"><SPAN style="color:#F0F7C1">Meet the Team</SPAN></A></LI><LI><A href="/Team:Washington/Accomplishments" title="Team:Washington/Accomplishments"><SPAN style="color:#F0F7C1">Accomplishments</SPAN></A></LI><LI><A href="/Team:Washington/Parts" title="Team:Washington/Parts"><SPAN style="color:#F0F7C1">Parts Submitted</SPAN></A></LI><LI><A href="/Team:Washington/Notebook" title="Team:Washington/Notebook"><SPAN style="color:#F0F7C1">Notebook</SPAN></A></LI><LI><A href="/Team:Washington/Bibliography" title="Team:Washington/Bibliography"><SPAN style="color:#F0F7C1">References</SPAN></A></LI></DIV></DIV><DIV id="uw_content"><H1><SPAN class="mw-headline" id="Gibson_Assembly">Gibson Assembly</SPAN></H1><P>The cloning process is often the most time-intensive task for iGEM teams. Methods to streamline the assembly and tuning of gene circuits can increase productivity and lead to more successful iGEM projects. Traditionally, restriction enzymes and their DNA recognition sequences are used to assemble promoters, ribosome binding sites, and gene coding sequences into gene networks.
</P><P>To create plasmids with more freeform inserts than possible with traditional BioBrick restriction/ligation cloning, we used a method described in 
<A href="http://www.natureprotocols.com/2009/04/16/onestep_enzymatic_assembly_of.php" class="external text" rel="nofollow">Nature Protocols 2009</A> in which parts are extracted from standard biobricks with primers that have homologies introduced on their 5' ends that overlap the parts that they will be next to. Small parts less than 250 bp are then stitched together using &quot;overlap extension&quot; PCR to create large enough fragments (roughly 500bp) that they can be assembled in a one step reaction as described by Gibson.  This procedure allows the construction of plasmids that have no seams between parts and allows for arbitrary numbers of parts to be combined quickly and efficiently.
</P><H2><SPAN class="mw-headline" id="Utility">Utility</SPAN></H2><P>Gibson assembly is particularly useful for the following tasks
</P><UL><LI> Extracting Biobrick parts from multiple source plasmids and assembling them arbitrarily into one plasmid using one cloning step
</LI><LI> Modifying short sequences of promoters, ribosome binding sites, and gene coding sequences  - from point mutations to inserting or replacing moderately large tracts (20-60 bp) of DNA
</LI><LI> Adding ssrA <A href="http://partsregistry.org/Protein_domains/Degradation" class="external text" rel="nofollow">degradation tags</A> to genes
</LI><LI> Creating fusion proteins and operons
</LI><LI> Flipping transcription direction of operons
</LI></UL><H2><SPAN class="mw-headline" id="Part_Extraction">Part Extraction</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:288px;"><DIV class="thumbcaption">Extraction Protocol</DIV></DIV></DIV><P>The first step is to design primers to extract the parts out of standard biobricks. The primers should be designed as usual (i.e. design for whatever Tm is desired over the ends of the part to be extracted. We aimed for around 60C). Then an extra sequence is added to the 5' end of the primers that is homologous to whatever part will be next to it in the final construct. These &quot;overlaps&quot; must have a Tm above 50C for efficient plasmid assembly. In some cases, new parts (i.e. promoters, ribosomal binding sites, ssrA degradation tags, and point mutations) can be introduced in these overlaps, provided they are short enough.
</P><H2><SPAN class="mw-headline" id="Overlap_Extension_for_Short_Amplicons">Overlap Extension for Short Amplicons</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:280px;"><DIV class="thumbcaption">Overlap Extension PCR</DIV></DIV></DIV><P>After two adjacent parts have been extracted as above, if one of them is short (&lt;250 bp) they are put into a standard PCR reaction with two primers that match the added homology on the outside of the desired construct. This results in linear DNA that is the result of the two parts stuck end-to-end with no seam and homologies to parts that will be adjacent in the final plasmid. This process can repeated to build DNA fragments roughly 500bp in length.
</P><H2><SPAN class="mw-headline" id="Gibson_Assembly_Reaction">Gibson Assembly Reaction</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:353px;"><DIV class="thumbcaption">Gibson Reaction</DIV></DIV></DIV><P>The ~500bp pieces of the final plasmid are then all fused together in one reaction with T5 exonuclease, heat-stable <I>Taq</I> ligase, Phusion polymerase, and free nucleotides with buffer. The exonuclease chews back the 5' ends of the strands, leaving &quot;sticky ends.&quot; Due to the introduced homologies, the complementary single-stranded DNA anneal to each other in the desired order. The polymerase repairs any extra nucleotides chewed back by the exonuclease, and - finally - the ligase repairs nicks in the DNA. This process results in a circular plasmid that can be transformed efficiently.
</P><H2><SPAN class="mw-headline" id="Issues_and_Solutions">Issues and Solutions</SPAN></H2><P>This type of procedure relies heavily on introduced homologies on the ends of all the parts involved. Initially we used the standard bioBrick prefix and suffix as the homologous region with which to insert our construct into standard backbones, but this proved problematic because of the NotI site which lies between the E/X and S/P restriction sites in bioBrick backbones, as it is long, consists mostly of Gs and Cs, and is palindromic. This leads to possible mispriming, and an ambiguity in the final configuration of our plasmid (the insert could end up forward, backward, or circular. The backbone could also recircularize without taking up the insert at all) and in turn, a very low yield of our desired construct. 
</P><P>To remedy this, we designed a new prefix and suffix, based on the <A href="http://dspace.mit.edu/handle/1721.1/46747" class="external text" rel="nofollow">BglBrick standard</A> which allows for the elimination of the NotI sites. We developed the prefix gaattcctgctgcggagatct and the suffix ggatccaacagggttctcgag by aiming for roughly 50% GC content and a Tm around 68 as calculated by <A href="http://www.finnzymes.com/tm_determination.html" class="external text" rel="nofollow">Finnzymes Tm calc</A>. We then modified Psb1A3 and Psb3K3 with these prefixes and suffixes, and had much greater success with our cloning.
</P><P>We have found that with inserts of less than 500 bp, one often gets many colonies containing insertless plasmid that has been circularized by DNA ligase. In order to determine if this is an issue in a given gibson reaction, for shorter inserts, we often conducted two reactions at once, one including the vector and the insert, and one containing only the vector, with the difference in volume made up with water. The reaction mixes were transformed into E. coli, and each  tranformant mixture was plated on LB agarose plates with selective media. The number of colonies on the control plate (  the plate with transformants of the reaction without any insert) corresponds roughly to the number of colonies on the the cloning plate ( the plate  with transformants of the vector/insert reaction). If the ratio of colonies on the cloning plate   to colonies on the control is 1:1, it would mean that the vast majority of the colonies on the cloning plate are due to insertless, circularized vector. We were able to obtain correct plasmids when the cloning colony:control colony ratio was as low as 2:1, but this required double restriction digest screening of a large number of colonies ( around 24) in order to obtain vector with insert.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>Daniel Gibson, One-step enzymatic assembly of DNA molecules up to several hundred kilobases in size
</P><P><A href="http://www.natureprotocols.com/2009/04/16/onestep_enzymatic_assembly_of.php" class="external text" rel="nofollow">Nature Protocols 2009</A></P><P>Gibson, D.G., <I>et al.</I> Enzymatic assembly of DNA molecules up to several hundred kilobases
</P><P><A href="http://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html" class="external text" rel="nofollow">Nature Methods 2009</A></P><DIV style="text-align:center"><P><B>←<A href="/Team:Washington/Tools_Used/Software" title="Team:Washington/Tools Used/Software">Software We Used</A></B>   <B><A href="/Team:Washington/Protocols" title="Team:Washington/Protocols">Protocols</A>→</B></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Washington/Tools_Used/Next-Gen_Cloning">http://2010.igem.org/Team:Washington/Tools_Used/Next-Gen_Cloning</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Washington/Tools_Used/Next-Gen_Cloning" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Washington/Tools_Used/Next-Gen_Cloning" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Washington/Tools_Used/Next-Gen_Cloning&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Washington/Tools_Used/Next-Gen_Cloning&amp;oldid=208557" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>