<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Washington_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Washington/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Washington/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Washington/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Washington/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="uw_leftBar"><DIV id="uw_ss"><UL id="uw_navBar"><LI><A href="/Team:Washington" title="Team:Washington"><SPAN style="color:#F0F7C1">Home</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Negative" title="Team:Washington/Gram Negative"><SPAN style="color:#F0F7C1">Gram(-) Therapeutic</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Negative/Design" title="Team:Washington/Gram Negative/Design"><SPAN style="color:#F0F7C1">Design</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Negative/Build" title="Team:Washington/Gram Negative/Build"><SPAN style="color:#F0F7C1">Build</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Negative/Test" title="Team:Washington/Gram Negative/Test"><SPAN style="color:#F0F7C1">Test Results</SPAN></A></LI></UL><LI><A href="/Team:Washington/Gram_Positive" title="Team:Washington/Gram Positive"><SPAN style="color:#F0F7C1">Gram(+) Therapeutic</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Positive/Design" title="Team:Washington/Gram Positive/Design"><SPAN style="color:#F0F7C1">Design</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Positive/Build" title="Team:Washington/Gram Positive/Build"><SPAN style="color:#F0F7C1">Build</SPAN></A></LI><LI><A href="/Team:Washington/Gram_Positive/Test" title="Team:Washington/Gram Positive/Test"><SPAN style="color:#F0F7C1">Test Results</SPAN></A></LI><LI><A href="/Team:Washington/Tools_Created" title="Team:Washington/Tools Created"><SPAN style="color:#F0F7C1">Tools Created</SPAN></A></LI><LI><A href="/Team:Washington/Tools_Created/New_Vectors" title="Team:Washington/Tools Created/New Vectors"><SPAN style="color:#F0F7C1">New Vectors</SPAN></A></LI><LI><A href="/Team:Washington/Tools_Created/New_Software" title="Team:Washington/Tools Created/New Software"><SPAN style="color:#F0F7C1">New Software</SPAN></A></LI><LI><A href="/Team:Washington/Tools_Used" title="Team:Washington/Tools Used"><SPAN style="color:#F0F7C1">Tools Used</SPAN></A></LI><LI><A href="/Team:Washington/Tools_Used/Software" title="Team:Washington/Tools Used/Software"><SPAN style="color:#F0F7C1">Software</SPAN></A></LI><LI><A href="/Team:Washington/Tools_Used/Next-Gen_Cloning" title="Team:Washington/Tools Used/Next-Gen Cloning"><SPAN style="color:#F0F7C1">Next-Gen Cloning</SPAN></A></LI><LI><A href="/Team:Washington/Protocols" title="Team:Washington/Protocols"><SPAN style="color:#F0F7C1">Protocols</SPAN></A></LI><LI><A href="/Team:Washington/Safety" title="Team:Washington/Safety"><SPAN style="color:#F0F7C1">Safety</SPAN></A></LI><LI><A href="/Team:Washington/Team" title="Team:Washington/Team"><SPAN style="color:#F0F7C1">Meet the Team</SPAN></A></LI><LI><A href="/Team:Washington/Accomplishments" title="Team:Washington/Accomplishments"><SPAN style="color:#F0F7C1">Accomplishments</SPAN></A></LI><LI><A href="/Team:Washington/Parts" title="Team:Washington/Parts"><SPAN style="color:#F0F7C1">Parts Submitted</SPAN></A></LI><LI><STRONG class="selflink"><SPAN style="color:#F0F7C1">Notebook</SPAN></STRONG></LI><LI><A href="/Team:Washington/Bibliography" title="Team:Washington/Bibliography"><SPAN style="color:#F0F7C1">References</SPAN></A></LI></DIV></DIV><DIV id="uw_content"><H1><SPAN class="mw-headline" id="Gram_Positive"><B>Gram Positive</B></SPAN></H1><H2><SPAN class="mw-headline" id="07.2F06.2F10"><B>07/06/10</B></SPAN></H2><UL><LI>Kunkel Mutagenesis
</LI></UL><H2><SPAN class="mw-headline" id="07.2F07.2F10"><B>07/07/10</B></SPAN></H2><UL><LI>Stained and scanned protein gel.
</LI></UL><P><B>BioRad Micro Column Protein Prep: Expression</B></P><UL><LI>Arctic Express 13C 24H no anti, Arctic Express 13C 24H kan, DE3* 22C 24H kan, Arctic Express 37C 24H Kan. (Both CapD and CapDCP) 
</LI><LI>CapD 22C kan and 37C kan grew much slower than the rest. (Takes about 3 - 4hrs vs around 1.5-2 hrs for the rest.)
</LI><LI>One of the two actually decreased OD after 30 minutes. 
</LI></UL><LI>Induced with IPTG after reached OD600 between .7-1.0 (CapD 22C kan and 37C reached about 1.3) and incubated at given temperatures.
</LI><H2><SPAN class="mw-headline" id="07.2F08.2F10"><B>07/08/10</B></SPAN></H2><UL><LI>Made glycerol stocks for Kunkel's. Put in -80C Freezer. To be shipped for sequencing.
</LI><LI>Spin down remaining proteins(CapD Artic no Anti, CapD CP Arctic no Anti, CapD* Kan 22, CapD CP* Kan 22, CapD Kan Artic, CapD CP Kan Arctic) and place in -20C, awaiting purification.
</LI></UL><H2><SPAN class="mw-headline" id="07.2F09.2F10"><B>07/09/10</B></SPAN></H2><P><B>Protein Purification</B></P><UL><LI>Lysis
</LI><LI>Bind Protein
</LI><LI>Numbers 6 and 7, CapD CP Anti and CapD CP 22, respectively, had lysate form in the supernatant that had been filtered through the TALON beads. 
</LI><LI>Sometimes centrifuged twice after pouring out supernatant because a significant amount had not filtered through TALON beads
</LI></UL><LI>Wash Protein
</LI><LI>Sometimes centrifuged twice after pouring out supernatant because a significant amount had not filtered through TALON beads
</LI><LI>Elution
</LI><P><I>(Note: Desired proteins have excess histidine chain that binds to TALON beads. Imidazole has higher affinity for TALON beads, so higher imidazole concentration pushes desired protein off, allowing it to fall through filter. This is why higher and higher concentrations of imidazole are used. Cleans off undesired proteins first all the way up to, hopefully, only the desired protein.)</I></P><P><B>Electrophoresis</B></P><UL><LI>Ran two gels. #1 has ladder in well 1. #2 has ladder in well 2. 3 wells per condition being Lysate, Supernatant, Purified, in that order. SDS 6uL+6uL of substance for mix. 10uL per well.
</LI></UL><P>Gel #1
</P><UL><LI>CapD Arctic No Anti, CapD Arctic Kan, CapD* Kan 22, CapD CP* Kan 37
</LI></UL><P>Gel #2
</P><UL><LI>CapD CP Arctic No Anti, CapD CP Arctic Kan, CapD CP* Kan 22, CapD* Kan 37
</LI></UL><H2><SPAN class="mw-headline" id="07.2F12.2F10"><B>07/12/10</B></SPAN></H2><P><B>Nano Drop (Bradford Protein Assay)</B></P><UL><LI>Create normal line with known concentrations of protein (9uL Coomassie dye, 1uL protein solution) (We used .25mM, .5mM, 1mM, 2mM)
</LI><LI>Use 1uL of each protein sample with 9uL Coomassie dye and measure with program.
</LI></UL><P><B>Overnight Glycerol Stocks</B></P><UL><LI>CapD/CapD CP TB (LB?) Kan solutions with a dab of glycerol from -80C (just scrape glycerol with pipette tip and drop into TB Kan solution).
</LI><LI>Incubate in 37C shaker overnight.
</LI></UL><P><B>Dialysis</B></P><UL><LI>Put proteins into dialysis tubes. Left in cold room overnight. (Used to reduce amount of imidazole in solution.)
</LI><LI>1. CapD Arctic no Anti
</LI><LI>2. CapD Arctic Kan
</LI><LI>3. CapD* Kan 22
</LI><LI>4. CapD* Kan 37
</LI><LI>5. CapD CP Arctic no Anti
</LI><LI>6. CapD CP Arctic Kan
</LI><LI>7. CapD CP* Kan 22
</LI><LI>8. CapD CP* Kan 37
</LI></UL><H2><SPAN class="mw-headline" id="07.2F13.2F10"><B>07/13/10</B></SPAN></H2><P><B>Nano Drop (Bradford Protein Assay)</B></P><UL><LI>Run Nano drop to find mg/mL post-dialysis
</LI></UL><P><B>Enzyme Assay</B></P><UL><LI>Diluted small amount of each dialyzed enzyme for Assay reaction components
</LI><LI>Ran fluorescence test with plate reader
</LI><LI>Got weird results (low, delayed, or no activity)
</LI></UL><P><B>Post Assay Electrophoresis</B></P><UL><LI>Ran gel to determine protein concentrations using 1mg/mL, .5mg/mL, .25mg/mL, .125mg/mL, .0625mg/mL, and .03125mg/mL BSA as standard.
</LI><LI>Will enter order of wells tomorrow once paperwork is on hand
</LI></UL><P>Post-assay Gel order ?
</P><UL><LI>1) Ladder
</LI><LI>2) 1mg/mL BSA
</LI><LI>3) 0.5mg/mL BSA
</LI><LI>4) 0.125mg/mL BSA
</LI><LI>5) 0.25mg/mL BSA
</LI><LI>6) 0.06mg/mL BSA 
</LI><LI>7) 0.3mg/mL BSA
</LI><LI>8) CapD Arctic no anti
</LI><LI>9) CapD Arctic Kan 13C
</LI><LI>10) CapD* Kan 22C
</LI><LI>11) CapD* Kan 37C
</LI><LI>12) CapD CP Arctic no anti
</LI><LI>13) CapD CP Arctic Kan 13C
</LI><LI>14) CapD CP* Kan 22C
</LI><LI>15) CapD CP* Kan 37C
</LI></UL><P><B>MiniPrep</B></P><UL><LI>Done by David and Chris to obtain plasmids for PCR
</LI></UL><H2><SPAN class="mw-headline" id="7.2F14.2F10"><B>7/14/10</B></SPAN></H2><P><B>PCR (Polymerase chain reaction)</B></P><UL><LI>Diluted Primers to 10uM (10uL of 100uM primer with 90uL of diH2O)
</LI><LI>Created mixes using the recipe from Table 1a at <A href="http://www.neb.com/nebecomm/products_intl/protocol210.asp" class="external autonumber" rel="nofollow">[1]</A></LI><LI>Primers were CapDOS, CapNSS
</LI><LI>Template DNA was from MiniPrep (Plasmids)
</LI><LI>Added T7 reverse primer to both
</LI><LI>Added Phusion DNA Polymerase last to delay reaction
</LI></UL><P><B>Kunkel Mutagenesis Sequence</B></P><UL><LI>Checked sequences that came back for desired mutations using CLC Sequence viewer <A href="http://www.clcbio.com/index.php?id=479" class="external autonumber" rel="nofollow">[2]</A>. See Google Docs form for mutation information.<A href="http://spreadsheets.google.com/ccc?key=0AvA_ILRbNLPfdEdDUk90ZFBYYjdpU1lCTHNmRl9kMFE&amp;hl=en#gid=0" class="external autonumber" rel="nofollow">[3]</A></LI></UL><P><B>Overnights</B></P><UL><LI>Made 6 overnights from glycerol stocks in -80C with TB Kan CapD CP*, one control, two mutant catalytic knock outs (T2A and T2V), and three mutants (F24Y, L40R, S143R)
</LI><LI>Incubate at 37C in iGEM lab
</LI></UL><P><B>Gel</B></P><UL><LI>Dialyzed proteins seemed to be very low in concentration. (BSA standard might be twice the labeled concentration on gel)
</LI></UL><H2><SPAN class="mw-headline" id="7.2F15.2F10"><B>7/15/10</B></SPAN></H2><P><B>IPTG Induction</B></P><UL><LI>Pulled overnight glycerol stocks to be put into TB Kan solution
</LI><LI>Added 1mL of culture per 50mL flask of TB Kan, 10mL of culture per 500mL flask of TB Kan
</LI><LI>Incubated at 37C until OD600 was 0.6-0.8. 
</LI><LI>Flasks were all incubated at 37C after induction, except the control, for about 1 hour before being moved to 22C.
</LI></UL><H2><SPAN class="mw-headline" id="7.2F16.2F10"><B>7/16/10</B></SPAN></H2><P><B>Scan Gel</B></P><UL><LI>Post Assay Gel scanned
</LI></UL><P><B>Colony PCR</B></P><UL><LI>Boil cells with PCR with &quot;green mix&quot; and +T7/-T7 primers. 
</LI><LI>Run on DNA gel to see if mutation occurred (determine by band size).
</LI></UL><P><B>Spin down cells</B></P><UL><LI>4000rpm for 20 minutes
</LI><LI>Store at -20C until purification
</LI></UL><H2><SPAN class="mw-headline" id="7.2F17.2F10-7.2F25.2F10"><B>7/17/10-7/25/10</B></SPAN></H2><UL><LI>See others for data?
</LI></UL><H2><SPAN class="mw-headline" id="7.2F26.2F10"><B>7/26/10</B></SPAN></H2><P><B>Induction</B></P><UL><LI>Grew up T20S, T2V, T59N, T59Q, T59H_M61A, T59H_M61T at 37C to OD0.5-0.8
</LI><LI>Induced with 500uL IPTG
</LI><LI>Continue incubation for 24hrs at 22C
</LI></UL><H2><SPAN class="mw-headline" id="7.2F27.2F10"><B>7/27/10</B></SPAN></H2><UL><LI>Concentrated purified proteins
</LI><LI>Ran gel
</LI><LI>Ran Bradford and Enzyme Assays
</LI><LI>Spin down T20S, T2V, T59N, T59Q, T59H_M61A, T59H_M61T
</LI></UL><H2><SPAN class="mw-headline" id="7.2F28.2F10"><B>7/28/10</B></SPAN></H2><P><B>Protein Purification</B></P><UL><LI>T59Q sat in centrifuge overnight at 4C, spun down in morning again.
</LI></UL><H2><SPAN class="mw-headline" id="7.2F29.2F10"><B>7/29/10</B></SPAN></H2><P><B>Electrophoresis</B></P><P>Gel #1
</P><UL><LI>Ladder
</LI><LI>A CapDCP
</LI><LI>B T2V
</LI><LI>C T20S
</LI><LI>D T59H_M61A
</LI><LI>E T59H_M61T
</LI><LI>F T59N
</LI><LI>G T59Q
</LI><LI>BSA 2mg/mL
</LI><LI>BSA 1mg/mL
</LI><LI>BSA 0.5mg/mL
</LI><LI>BSA .25mg/mL
</LI><LI>BSA .125mg/mL
</LI><LI>BSA .0625mg/mL
</LI></UL><P>Gel #2
</P><UL><LI>Ladder
</LI><LI>BSA 2mg/mL
</LI><LI>BSA 1mg/mL
</LI><LI>BSA 0.5mg/mL
</LI><LI>1 CapD
</LI><LI>2 CapDCP
</LI><LI>3 CapDNSS
</LI><LI>4 CapDOS
</LI><LI>5 F24Y
</LI><LI>6 L40R
</LI><LI>7 S143R
</LI><LI>8 S22I,T59Q
</LI><LI>9 S57H,M61H
</LI><LI>10 T2A
</LI><LI>11 T2V
</LI></UL><P><I>Notes</I></P><UL><LI>De-staining started at 4 pm
</LI><LI>Used up remaining possibly &quot;bad&quot; gels. Gels have large blue blotch at the bottom.
</LI></UL><P><B>Enzyme Assay</B></P><UL><LI>Row A=H2O
</LI><LI>Row B=L-Glutamate
</LI><LI>1 CapDCP
</LI><LI>2 T2V
</LI><LI>3 T20S
</LI><LI>4 T59H_M61A
</LI><LI>5 T59H_M61T
</LI><LI>6 T59N
</LI><LI>7 T59Q
</LI><LI>8 diH2O
</LI></UL><H2><SPAN class="mw-headline" id="7.2F30.2F10"><B>7/30/10</B></SPAN></H2><P><B>Kunkel Mutagenesis</B></P><UL><LI>1. CapDCP NSS
</LI><LI>2. CapDCP NSS Foldit
</LI><LI>3. CapDCP w/ Thrombin
</LI><LI>4. F24H, R356K
</LI><LI>5. G79A
</LI><LI>6. M61H
</LI><LI>7. M61N
</LI><LI>8. S22Q, T59Q
</LI><LI>9. T18L, T59Q_M61N, F452W
</LI><LI>10. T20S, T59Q_M61T
</LI><LI>11. T59Q_M61Q, F452W
</LI><LI>12. T59S_M61H
</LI><LI>13. H2O
</LI></UL><H2><SPAN class="mw-headline" id="8.2F2.2F10"><B>8/2/10</B></SPAN></H2><UL><LI>First Kunkel's failed for unknown reasons
</LI></UL><P><B>Kunkel Mutagenesis Attempt #2</B></P><UL><LI>1. CapDCP NSS
</LI><LI>2. CapDCP NSS Foldit
</LI><LI>3. CapDCP w/ Thrombin
</LI><LI>4. F24H, R356K
</LI><LI>5. G79A
</LI><LI>6. M61H
</LI><LI>7. M61N
</LI><LI>8. S22Q, T59Q
</LI><LI>9. T18L, T59Q_M61N, F452W
</LI><LI>10. T20S, T59Q_M61T
</LI><LI>11. T59Q_M61Q, F452W
</LI><LI>12. T59S_M61H
</LI><LI>13. H2O
</LI></UL><H2><SPAN class="mw-headline" id="8.2F3.2F10"><B>8/3/10</B></SPAN></H2><UL><LI>Picked Kunkel's colonies and placed into plate with LB+Kan culture.
</LI><LI>Shaking in 37C room
</LI><LI>Scan Gels from 7/29/10
</LI></UL><P><B>Well Order (Four per mutation)</B></P><UL><LI>CapDCP NSS- A1-D1
</LI><LI>CapDCP NSS Foldit- E1-H1
</LI><LI>CapDCP w/ Thrombin- A2-D2
</LI><LI>F24H, R356K- E2-H2
</LI><LI>G79A- A3-D3
</LI><LI>M61H- E3-H3
</LI><LI>M61N- A4-D4
</LI><LI>S22Q, T59Q- E4-H4
</LI><LI>T18L, T59Q_M61N, F452W- A5-D5
</LI><LI>T20S, T59Q_M61T- E5-H5
</LI><LI>T59Q_M61Q, F452W- A6-D6
</LI><LI>T59S_M61H- E6-H6
</LI></UL><H2><SPAN class="mw-headline" id="8.2F4.2F10"><B>8/4/10</B></SPAN></H2><UL><LI>Took Kunkel's overnights out of 37C room
</LI><LI>Sent Glycerol Stocks for Sequencing (100uL 20% Glycerol, 100uL cells)
</LI></UL><H2><SPAN class="mw-headline" id="8.2F5.2F10"><B>8/5/10</B></SPAN></H2><UL><LI>Computational Work from Home
</LI></UL><H2><SPAN class="mw-headline" id="8.2F6.2F10"><B>8/6/10</B></SPAN></H2><UL><LI>Checked all Round 2 sequences. See Google Doc for mutant information.
</LI><LI>Corrected Data from Assay based on Gel #2 protein concentrations.
</LI></UL><H2><SPAN class="mw-headline" id="8.2F9.2F10"><B>8/9/10</B></SPAN></H2><UL><LI>Autoclave 15 flasks of TB (500mL)
</LI><LI>Glycerol Stock Overnights (5mL TB+Kan)
</LI><LI>1. CapD CP
</LI><LI>2. CapD CP w/ Thrombin
</LI><LI>3. CapD NSS
</LI><LI>4. CapD CP NSS
</LI><LI>5. CapD CP NSS Foldit
</LI><LI>6. T2V
</LI><LI>7. F24H, R356K
</LI><LI>8. G79A
</LI><LI>9. M61H
</LI><LI>10. M61N
</LI><LI>11. S22Q, T59Q
</LI><LI>12. T18L, T59Q, M61N, F452W
</LI><LI>13. T20S, T59Q, M61T
</LI><LI>14. T59Q, M61Q, F452W
</LI><LI>15. T59S, M61H
</LI></UL><P><I>Remember to make glycerol stock for T2A and T2V and store in -80C</I></P><H2><SPAN class="mw-headline" id="8.2F10.2F10"><B>8/10/10</B></SPAN></H2><UL><LI>Grow up cells in 500mL TB+Kan
</LI><LI>Induce with 500uL 1M IPTG once at OD 0.5-0.8
</LI><LI>Incubate at 18C for 24 hours
</LI><LI>1. CapD CP
</LI><LI>2. CapD CP w/ Thrombin
</LI><LI>3. CapD NSS
</LI><LI>4. CapD CP NSS
</LI><LI>5. CapD CP NSS Foldit
</LI><LI>6. T2V
</LI><LI>7. F24H, R356K
</LI><LI>8. G79A
</LI><LI>9. M61H
</LI><LI>10. M61N
</LI><LI>11. S22Q, T59Q
</LI><LI>12. T18L, T59Q, M61N, F452W
</LI><LI>13. T20S, T59Q, M61T
</LI><LI>14. T59Q, M61Q, F452W
</LI><LI>15. T59S, M61H
</LI></UL><P><I>G79A had super slow growth rate, about OD 0.1 after 4 hours. T18L, T59Q, M61N, F452W and T59Q, M61Q, F452W about OD 0.3 after 4 hours.</I><I>G79A completed at 7:00pm.</I></P><H2><SPAN class="mw-headline" id="8.2F11.2F10"><B>8/11/10</B></SPAN></H2><UL><LI>Spin down cells 4000rpm for 20 minutes. Store in -20C.
</LI></UL><H2><SPAN class="mw-headline" id="8.2F12.2F10"><B>8/12/10</B></SPAN></H2><UL><LI>Full Purification Process
</LI><LI>Concentrate Proteins
</LI><LI>Run gel of purified/concentrated proteins
</LI></UL><H2><SPAN class="mw-headline" id="8.2F13.2F10"><B>8/13/10</B></SPAN></H2><UL><LI>Interpret protein concentration from gels (scan with Baker Lab scanner)
</LI><LI>Run enzyme/H2O fluorescence assay 
</LI><LI>CapD NSS ran for 2 possible protein concentrations. One for the bottom two bands, one for all three. Bottom two=3BC, All three=3ABC.
</LI></UL><P>L-Glutamate
</P><UL><LI>A1.1 (CapD CP)
</LI><LI>A2.2 (CapD CP w/ Thrombin)
</LI><LI>A3.3BC (CapD NSS: lower two bands)
</LI><LI>A4.3ABC (CapD NSS: All bands)
</LI><LI>A5.4 (CapD CP NSS)
</LI><LI>A6.5 (CapD CP NSS Foldit)
</LI><LI>A7.6 (T2V)
</LI><LI>A8.7 (F24H,R356K)
</LI><LI>A9.8 (G79A)
</LI><LI>A10.9 (M61H)
</LI><LI>A11.10 (M61N)
</LI><LI>A12.11 (S22Q,T59Q) 
</LI><LI>C1.12 (T18L,T59Q,M61N,F452W)
</LI><LI>C2.13 (T20S,T59Q,M61T)
</LI><LI>C3.14 (T59Q,M61Q,F452W)
</LI><LI>C4.15 (T59S,M61H)
</LI><LI>C5.Blank
</LI></UL><P>H2O
</P><UL><LI>B1.1 (CapD CP)
</LI><LI>B2.2 (CapD CP w/ Thrombin)
</LI><LI>B3.3BC (CapD NSS: lower two bands)
</LI><LI>B4.3ABC (CapD NSS: All bands)
</LI><LI>B5.4 (CapD CP NSS)
</LI><LI>B6.5 (CapD CP NSS Foldit)
</LI><LI>B7.6 (T2V)
</LI><LI>B8.7 (F24H,R356K)
</LI><LI>B9.8 (G79A)
</LI><LI>B10.9 (M61H)
</LI><LI>B11.10 (M61N)
</LI><LI>B12.11 (S22Q,T59Q)
</LI><LI>D1.12 (T18L,T59Q,M61N,F452W)
</LI><LI>D2.13 (T20S,T59Q,M61T)
</LI><LI>D3.14 (T59Q,M61Q,F452W)
</LI><LI>D4.15 (T59S,M61H)
</LI><LI>D5.Blank
</LI></UL><LI>Ran Mass spec to determine the characteristics of second band on gel
</LI><H2><SPAN class="mw-headline" id="8.2F16.2F10"><B>8/16/10</B></SPAN></H2><UL><LI>Left proteins out over weekend in ice bucket.
</LI><LI>CapD NSS and CapD CP NSS Foldit both crashed out.
</LI></UL><LI>Regrow CapD NSS and CapD CP NSS Foldit
</LI><LI>Gravity columns left out over weekend.
</LI><LI>Recharged, but ask Justin if should use again.
</LI><LI>Autoclaved 2x 500mL TB in 2L flasks
</LI><H2><SPAN class="mw-headline" id="8.2F17.2F10"><B>8/17/10</B></SPAN></H2><UL><LI>Plate Chlor+Kan for incubating transformed cells
</LI><LI>Transform CJ236 cells for creating ssDNA of CapD NSS and CapD CP NSS Foldit tomorrow
</LI><LI>Inoculate and grow up 500mL TB+Kan cultures of CapD NSS and CapD CP NSS Foldit. Induce with 500uL IPTG at OD 0.5-0.8
</LI><LI>Induced at 2pm. Incubating at 18C in shakers at Schief lab.
</LI></UL><H2><SPAN class="mw-headline" id="8.2F18.2F10"><B>8/18/10</B></SPAN></H2><UL><LI>Spin down cells
</LI><LI>Add phage to ssDNA prep and expand culture
</LI></UL><H2><SPAN class="mw-headline" id="8.2F19.2F10"><B>8/19/10</B></SPAN></H2><UL><LI>Purified and concentrated protein (One concentrated down to 400uL, other to 800uL, instruction mismatch with protocol)[Correct: 200uL]
</LI><LI>New columns used. Covered with parafilm since can't find caps.
</LI></UL><LI>Chris harvested ssDNA for CapD NSS and CapD CP NSS Foldit
</LI><H2><SPAN class="mw-headline" id="8.2F20.2F10"><B>8/20/10</B></SPAN></H2><UL><LI>A280 came up with CapD - 6.1mg/mL and CapDCP - 3.1mg/mL
</LI><LI>Running gel for protein concentrations
</LI><LI>Started staining 2:12 pm
</LI></UL><LI>Flash freeze CapD and CapDCP proteins in 50uL aliquots
</LI><H2><SPAN class="mw-headline" id="8.2F23.2F10"><B>8/23/10</B></SPAN></H2><UL><LI>Ran fluorescence readings to determine correction coefficients for enzyme assay due to product/substrate quenching
</LI></UL><P>Rows
</P><UL><LI>1. Product concentration at 10uM, Substrate decreasing 100uM to 0uM (Concentration cut in half each time, 12th well is 0uM)
</LI><LI>2. 5uM Product, Substrate decreasing 100uM to 0uM (Concentration cut in half each time, 12th well is 0uM) 
</LI><LI>3. Substrate set at 100uM, Product decreasing 100uM to 0uM (Concentration cut in half each time, 12th well is 0uM)
</LI></UL><H2><SPAN class="mw-headline" id="8.2F24.2F10"><B>8/24/10</B></SPAN></H2><UL><LI>Still searching for proper Michaelis-Menten profile. (Abandoned 8/23/10 attempt)
</LI><LI>Original Excitation/Emission standard looks fine (490/520nm). 
</LI><LI>Tested 0.01 and 0.00001 mg/mL protein. First too high, second too low. 
</LI><LI>Tested serial dilution (half) enzyme from 0.01mg/mL decreasing across 12 wells. Substrate hold at 1uM.
</LI><LI>Tested serial dilution (half) substrate 100uM decreasing across 12 wells (no blank). Hold enzyme at 0.001mg/mL.
</LI><LI>Testing serial dilution (half) substrate 1uM decreasing across 12 wells (12 is blank). Hold enzyme at 0.0001mg/mL.
</LI></UL><LI>Kunkel's Mutagenesis
</LI><H2><SPAN class="mw-headline" id="8.2F25.2F10"><B>8/25/10</B></SPAN></H2><P>Reads
</P><UL><LI>Calibration Curve (New Substrate)
</LI><LI>Substrate start 1uM serial dilution (halves) across. Product start 0.1uM serial dilution (thirds) down. PMT High Sensitivity
</LI></UL><LI>Enzyme Assay (New Substrate)
</LI><LI>6 Rows. Two CapDCP, two CapD, two no enzyme. (0.0001mg/mL, 2.3nM)
</LI><LI>First row of each L-Glu. Second row of each H2O.
</LI><LI>Substrate start 1uM serial dilution (halves) across. PMT High Sensitivity. 
</LI><LI>Calibration Curve (Old Substrate)
</LI><LI>Substrate start 100uM serial dilution (halves) across. Product start 1uM serial dilution (thirds) down. PMT High Sensitivity
</LI><LI>Enzyme Assay (Old Substrate)
</LI><LI>3 rows. L-Glutamate in all rows. CapD 0.01mg/mL (200nM), CapD 0.0001mg/mL (2nM), and no enzyme (0nM). PMT High Sensitivity.
</LI><P><I>All Complete. Looks like good results across the board. Begin data analysis tomorrow.</I></P><H2><SPAN class="mw-headline" id="8.2F26.2F10"><B>8/26/10</B></SPAN></H2><UL><LI>Normalized Michaelis-Menten Data which better fit Substrate Inhibition Curves.
</LI><LI>Gave Dr. Baker an update on results from the past month.
</LI></UL><H2><SPAN class="mw-headline" id="8.2F27.2F10"><B>8/27/10</B></SPAN></H2><UL><LI>Presentation Slides
</LI></UL><H2><SPAN class="mw-headline" id="8.2F30.2F10"><B>8/30/10</B></SPAN></H2><UL><LI>Dilute and Kinase Oligos
</LI><LI>A1. CapDCPNF I83T 
</LI><LI>A2. CapDCPNF I83V
</LI><LI>A3. CapDCPNF T2C
</LI><LI>A4. CapDCPNF T2S
</LI><LI>A5. CapDCPNF T18S_T20S
</LI><LI>A6. L40A
</LI><LI>A7. M61A
</LI><LI>A8. M61C
</LI><LI>A9. M61D
</LI><LI>A10. M61G
</LI><LI>A11. M61I
</LI><LI>A12. M61L
</LI><LI>B1. M61N
</LI><LI>B2. M61S
</LI><LI>B3. M61T
</LI><LI>B4. M61V
</LI><LI>B5. T20S_F24H
</LI><LI>B6. T20S_F24Y
</LI><LI>B7. Control
</LI><LI>B8. Tossed
</LI><LI>B9. T20S
</LI><LI>B10. T59N
</LI><LI>B11. T59S_M61S 
</LI></UL><UL><LI><I>Pipetted 3uL of 8-17 kinased oligos into eppendorf tube marked M61X</I></LI></UL><P>Overnights Growing at 37C
</P><UL><LI>1.CapDCPNFDelta
</LI><LI>2.F24A
</LI><LI>3.F24H
</LI><LI>4.F24H,T59N
</LI><LI>5.F24Y,R305K
</LI><LI>6.F24Y,T59N
</LI><LI>7.F24Y,T59N,R305K
</LI><LI>8.L40R
</LI><LI>9.L40S
</LI><LI>10.L40W
</LI><LI>11.M61S
</LI><LI>12.N23K
</LI><LI>13.N23Q
</LI><LI>14.T20A
</LI><LI>15.T20C
</LI><LI>16.T20S,T59N
</LI><LI>17.T59M
</LI><LI>18.T59N
</LI><LI>19.T59S
</LI><LI>20.CapD CP NSS Foldit (Control)
</LI></UL><H2><SPAN class="mw-headline" id="8.2F31.2F10"><B>8/31/10</B></SPAN></H2><UL><LI>Kunkel's up to plate and incubate cells
</LI><LI>1.I83T
</LI><LI>2.I83V
</LI><LI>3.L40A
</LI><LI>4.T2C
</LI><LI>5.T2S
</LI><LI>6.T2S,T18S_T20S
</LI><LI>7.T2S,T18S_T20S,T59S_M61S
</LI><LI>8.T2S,T20S
</LI><LI>9.T18S_T20S
</LI><LI>10.T20S_F24H
</LI><LI>11.T20S_F24Y
</LI><LI>12.T20S_F24Y,T59N
</LI><LI>13.T20S,M61A
</LI><LI>14.T20S,M61C
</LI><LI>15.T20S,M61D
</LI><LI>16.T20S,M61G
</LI><LI>17.T20S,M61I
</LI><LI>18.T20S,M61L
</LI><LI>19.T20S,M61N
</LI><LI>20.T20S,M61S
</LI><LI>21.T20S,M61T
</LI><LI>22.T20S,M61V
</LI><LI>23.T59S_M61S
</LI><LI>24.Control (Blank)
</LI><LI>25.Control (Only Cells)
</LI></UL><P>Grow up proteins(small scale) to OD600 ~0.4 and induce with 25uL IPTG. 
</P><UL><LI>1.CapDCPNFDelta
</LI><LI>2.F24A
</LI><LI>3.F24H
</LI><LI>4.F24H,T59N
</LI><LI>5.F24Y,R305K
</LI><LI>6.F24Y,T59N
</LI><LI>7.F24Y,T59N,R305K
</LI><LI>8.L40R
</LI><LI>9.L40S
</LI><LI>10.L40W
</LI><LI>11.M61S
</LI><LI>12.N23K
</LI><LI>13.N23Q
</LI><LI>14.T20A
</LI><LI>15.T20C
</LI><LI>16.T20S,T59N
</LI><LI>17.T59M
</LI><LI>18.T59N
</LI><LI>19.T59S
</LI><LI>20.CapD CP NSS Foldit (Control)
</LI></UL><H2><SPAN class="mw-headline" id="9.2F1.2F10"><B>9/1/10</B></SPAN></H2><P><I>Have not tested F24W, R305A</I></P><P>Overnight: Kunkel's Plate shake in warm room
</P><UL><LI>1.I83T: A1-D1
</LI><LI>2.I83V: E1-H1
</LI><LI>3.L40A: A2-D2
</LI><LI>4.T2C: E2-H2
</LI><LI>5.T2S: A3-D3
</LI><LI>6.T2S,T18S_T20S: E3-H3
</LI><LI>7.T2S,T18S_T20S,T59S_M61S: A4-D4, E12-F12
</LI><LI>8.T2S,T20S: E4-H4
</LI><LI>9.T18S_T20S: A5-D5
</LI><LI>10.T20S_F24H: E5-H5
</LI><LI>11.T20S_F24Y: A6-D6
</LI><LI>12.T20S_F24Y,T59N: E6-H6
</LI><LI>13.T20S,M61A: A7-D7
</LI><LI>14.T20S,M61C: E7-H7
</LI><LI>15.T20S,M61D: A8-D8
</LI><LI>16.T20S,M61G: E8-H8
</LI><LI>17.T20S,M61I: A9-D9
</LI><LI>18.T20S,M61L: E9-H9
</LI><LI>19.T20S,M61N: A10-D10
</LI><LI>20.T20S,M61S: E10-H10
</LI><LI>21.T20S,M61T: A11-D11
</LI><LI>22.T20S,M61V: E11-H11
</LI><LI>23.T59S_M61S: A12-D12
</LI><LI>24.Blank: E12-H12
</LI></UL><P>Spin Down Cells at 4:00pm. Place in -20C
</P><UL><LI>1.CapDCPNFDelta
</LI><LI>2.F24A
</LI><LI>3.F24H
</LI><LI>4.F24H,T59N
</LI><LI>5.F24Y,R305K
</LI><LI>6.F24Y,T59N
</LI><LI>7.F24Y,T59N,R305K
</LI><LI>8.L40R
</LI><LI>9.L40S
</LI><LI>10.L40W
</LI><LI>11.M61S
</LI><LI>12.N23K
</LI><LI>13.N23Q
</LI><LI>14.T20A
</LI><LI>15.T20C
</LI><LI>16.T20S,T59N
</LI><LI>17.T59M
</LI><LI>18.T59N
</LI><LI>19.T59S
</LI><LI>20.CapD CP NSS Foldit (Control)
</LI></UL><H2><SPAN class="mw-headline" id="9.2F2.2F10"><B>9/2/10</B></SPAN></H2><UL><LI>Made Glycerol Stocks for Kunkel  Set 2 Round 3 and sent in for sequencing
</LI><LI>Small Scale Purification of Kunkel Set 2 Round 2
</LI><LI>Growing overnight of CapD CP NSS Foldit for creating glycerol stock and miniprepping DNA for USAMRIID
</LI></UL><H2><SPAN class="mw-headline" id="9.2F3.2F10"><B>9/3/10</B></SPAN></H2><P><B>Enzyme Assay</B></P><UL><LI>Row A and B=Transpeptidation
</LI><LI>Row C and D=Hydrolysis
</LI><LI>1.CapDCPNFDelta-A1,C1
</LI><LI>2.F24A-A2,C2
</LI><LI>3.F24H-A3,C3
</LI><LI>4.F24H,T59N-A4,C4
</LI><LI>5.F24Y,R305K-A5,C5
</LI><LI>6.F24Y,T59N-A6,C6
</LI><LI>7.F24Y,T59N,R305K-A7,C7
</LI><LI>8.L40R-A8,C8
</LI><LI>9.L40S-A9,C9
</LI><LI>10.L40W-A10,C10
</LI><LI>11.M61S-A11,C11
</LI><LI>12.N23K-A12,C12
</LI><LI>13.N23Q-B1,D1
</LI><LI>14.T20A-B2,D2
</LI><LI>15.T20C-B3,D3
</LI><LI>16.T20S,T59N-B4,D4
</LI><LI>17.T59M-B5,D5
</LI><LI>18.T59N-B6,D6
</LI><LI>19.T59S-B7,D7
</LI><LI>20.CapD CP NSS Foldit (Control)-B8,D8
</LI><LI>21.Blank-B9,D9
</LI></UL><LI>Make Glycerol Stock and MiniPrep DNA of CapD CP for USAMRIID
</LI><LI>Run protein gel to confirm concentrations from Nanodrop
</LI><H2><SPAN class="mw-headline" id="9.2F4.2F10"><B>9/4/10</B></SPAN></H2><UL><LI>Mass Spec confirms that CapDCPNSSFoldit really is awesome.
</LI></UL><H2><SPAN class="mw-headline" id="9.2F7.2F10"><B>9/7/10</B></SPAN></H2><P>Overnights in TB+Kan at 37C in back of small shaker in front of storage room.
</P><UL><LI>Control.CapDCPNSSFoldit
</LI><LI>1.I83T
</LI><LI>2.I83V
</LI><LI>3.L40A
</LI><LI>4.T2C
</LI><LI>5.T2S
</LI><LI>6.T2S,T18S_T20S
</LI><LI>7.T2S,T18S_T20S,T59S_M61S
</LI><LI>8.T2S,T20S
</LI><LI>9.T18S_T20S
</LI><LI>10.T20S_F24H
</LI><LI>11.T20S_F24Y
</LI><LI>12.T20S_F24Y,T59N
</LI><LI>13.T20S,M61A
</LI><LI>14.T20S,M61C
</LI><LI>15.T20S,M61D
</LI><LI>16.T20S,M61G
</LI><LI>17.T20S,M61L
</LI><LI>18.T20S,M61N
</LI><LI>19.T20S,M61S
</LI><LI>20.T20S,M61T
</LI><LI>21.T20S,M61V
</LI><LI>22.T20S,T59S_M61S
</LI></UL><H2><SPAN class="mw-headline" id="9.2F8.2F10"><B>9/8/10</B></SPAN></H2><P>Chris to inoculate ~9am. Induce at OD600 0.4-0.6 w/ 25uL IPTG. Induced at 12:20pm.
</P><UL><LI>Control.CapDCPNSSFoldit
</LI><LI>1.I83T
</LI><LI>2.I83V
</LI><LI>3.L40A
</LI><LI>4.T2C
</LI><LI>5.T2S
</LI><LI>6.T2S,T18S_T20S
</LI><LI>7.T2S,T18S_T20S,T59S_M61S
</LI><LI>8.T2S,T20S
</LI><LI>9.T18S_T20S
</LI><LI>10.T20S_F24H
</LI><LI>11.T20S_F24Y
</LI><LI>12.T20S_F24Y,T59N
</LI><LI>13.T20S,M61A
</LI><LI>14.T20S,M61C
</LI><LI>15.T20S,M61D
</LI><LI>16.T20S,M61G
</LI><LI>17.T20S,M61L
</LI><LI>18.T20S,M61N
</LI><LI>19.T20S,M61S
</LI><LI>20.T20S,M61T
</LI><LI>21.T20S,M61V
</LI><LI>22.T20S,T59S_M61S
</LI></UL><H2><SPAN class="mw-headline" id="9.2F9.2F10"><B>9/9/10</B></SPAN></H2><UL><LI>Spin Down Cells at 12:30pm
</LI><LI>Purification completed ~6:45pm
</LI><LI>1.I83T
</LI><LI>2.I83V
</LI><LI>3.L40A
</LI><LI>4.T2C
</LI><LI>5.T2S
</LI><LI>6.T2S,T18S_T20S
</LI><LI>7.T2S,T18S_T20S,T59S_M61S
</LI><LI>8.T2S,T20S
</LI><LI>9.T18S_T20S
</LI><LI>10.T20S_F24H
</LI><LI>11.T20S_F24Y
</LI><LI>12.T20S_F24Y,T59N
</LI><LI>13.T20S,M61A
</LI><LI>14.T20S,M61C
</LI><LI>15.T20S,M61D
</LI><LI>16.T20S,M61G
</LI><LI>17.T20S,M61L
</LI><LI>18.T20S,M61N
</LI><LI>19.T20S,M61S
</LI><LI>20.T20S,M61T
</LI><LI>21.T20S,M61V
</LI><LI>22.T20S,T59S_M61S
</LI><LI>23.CapDCPNSSFoldit
</LI></UL><H2><SPAN class="mw-headline" id="9.2F10.2F10"><B>9/10/10</B></SPAN></H2><P><B>Enzyme Assay</B></P><UL><LI>Set 2 Round 3
</LI><LI>Row A &amp; B = L-Glu
</LI><LI>Row C &amp; D = H2O
</LI><LI>Nanodrop for protein concentration
</LI><LI>Proteins Diluted with Elution Buffer
</LI><LI><B>L-Glutamate</B></LI><LI>A1.I83T
</LI><LI>A2.I83V
</LI><LI>A3.L40A
</LI><LI>A4.T2C
</LI><LI>A5.T2S
</LI><LI>A6.T2S,T18S_T20S
</LI><LI>A7.T2S,T18S_T20S,T59S_M61S
</LI><LI>A8.T2S,T20S
</LI><LI>A9.T18S_T20S
</LI><LI>A10.T20S_F24H
</LI><LI>A11.T20S_F24Y
</LI><LI>A12.T20S_F24Y,T59N
</LI><LI>B1.T20S,M61A
</LI><LI>B2.T20S,M61C
</LI><LI>B3.T20S,M61D
</LI><LI>B4.T20S,M61G
</LI><LI>B5.T20S,M61L
</LI><LI>B6.T20S,M61N
</LI><LI>B7.T20S,M61S
</LI><LI>B8.T20S,M61T
</LI><LI>B9.T20S,M61V
</LI><LI>B10.T20S,T59S_M61S
</LI><LI>B11.CapDCPNSSFoldit
</LI><LI>B12.Blank
</LI></UL><LI><B>H2O</B></LI><LI>C1.I83T
</LI><LI>C2.I83V
</LI><LI>C3.L40A
</LI><LI>C4.T2C
</LI><LI>C5.T2S
</LI><LI>C6.T2S,T18S_T20S
</LI><LI>C7.T2S,T18S_T20S,T59S_M61S
</LI><LI>C8.T2S,T20S
</LI><LI>C9.T18S_T20S
</LI><LI>C10.T20S_F24H
</LI><LI>C11.T20S_F24Y
</LI><LI>C12.T20S_F24Y,T59N
</LI><LI>D1.T20S,M61A
</LI><LI>D2.T20S,M61C
</LI><LI>D3.T20S,M61D
</LI><LI>D4.T20S,M61G
</LI><LI>D5.T20S,M61L
</LI><LI>D6.T20S,M61N
</LI><LI>D7.T20S,M61S
</LI><LI>D8.T20S,M61T
</LI><LI>D9.T20S,M61V
</LI><LI>D10.T20S,T59S_M61S
</LI><LI>D11.CapDCPNSSFoldit
</LI><LI>D12.Blank
</LI><H2><SPAN class="mw-headline" id="9.2F13.2F10"><B>9/13/10</B></SPAN></H2><P>Designed new set and ordered Oligos
</P><UL><LI>1.F24W
</LI><LI>2.F60E
</LI><LI>3.F60Q
</LI><LI>4.F60W
</LI><LI>5.L40D
</LI><LI>6.L40E
</LI><LI>7.L40F
</LI><LI>8.L40H
</LI><LI>9.L40K
</LI><LI>10.L40Q
</LI><LI>11.L40Y
</LI><LI>12.N81E
</LI><LI>13.N81Q
</LI><LI>14.T59D_M61S
</LI><LI>15.T59K
</LI><LI>16.T59K_M61S
</LI><LI>17.T59N_M61S
</LI><LI>18.T59R
</LI><LI>19.T59I
</LI></UL><H2><SPAN class="mw-headline" id="9.2F14.2F10"><B>9/14/10</B></SPAN></H2><UL><LI>Wiki/Slides
</LI></UL><H2><SPAN class="mw-headline" id="9.2F15.2F10"><B>9/15/10</B></SPAN></H2><P>Kunkel's Mutagenesis
</P><OL><LI>F24H, L40R, T59D_M61S
</LI><LI>F24H, L40R, T59N_M61S
</LI><LI>F24H, L40R, T59S_M61S
</LI><LI>F24H, T59R
</LI><LI>F24W
</LI><LI>F24Y, L40R
</LI><LI>F24Y, L40R, T59N
</LI><LI>F60W
</LI><LI>T20S, F60W
</LI><LI>T20S, L40D
</LI><LI>T20S, L40E
</LI><LI>T20S, L40F
</LI><LI>T20S, L40H
</LI><LI>T20S, L40K
</LI><LI>T20S, L40Q
</LI><LI>T20S, L40R
</LI><LI>T20S, L40R, F60E
</LI><LI>T20S, L40R, F60Q
</LI><LI>T20S, L40R, N81E
</LI><LI>T20S, L40R, N81Q
</LI><LI>T20S, L40R, T59K
</LI><LI>T20S, L40R, T59K_M61S
</LI><LI>T20S, L40R, T59N
</LI><LI>T20S, L40W
</LI><LI>T20S, L40Y
</LI><LI>T20S, T59I
</LI><LI>T20S, T59K
</LI><LI>T20S, T59K_M61S
</LI><LI>T20S, T59R
</LI><LI>Control
</LI></OL><UL><LI><I>11, 12, 28 failed. High ion concentration did not allow for electroporation. Dialysis ligation fail? Continuing to plate and grow, but possibly 1/10 success rate.</I></LI><LI><I>Did not recover after electroporation. Plated immediately.</I></LI></UL><H2><SPAN class="mw-headline" id="9.2F16.2F10"><B>9/16/10</B></SPAN></H2><P>Pick Colonies and grow on plate shaker in 37C <I>No colonies on 28</I></P><P>Plate A
</P><P>A1-D1. F24H, L40R, T59D_M61S
</P><P>E1-H1. F24H, L40R, T59N_M61S
</P><P>A2-D2. F24H, L40R, T59S_M61S
</P><P>E2-H2. F24H, T59R
</P><P>A3-D3. F24W
</P><P>E3-H3. F24Y, L40R
</P><P>A4-D4. F24Y, L40R, T59N
</P><P>E4-H4. F60W
</P><P>A5-D5. T20S, F60W
</P><P>E5-H5. T20S, L40D
</P><P>A6-D6. T20S, L40F
</P><P>E6-H6. T20S, L40H
</P><P>A7-D7. T20S, L40K
</P><P>E7-H7. T20S, L40Q
</P><P>A8-D8. T20S, L40R
</P><P>E8-H8. T20S, L40R, F60E
</P><P>A9-D9. T20S, L40R, F60Q
</P><P>E9-H9. T20S, L40R, N81E
</P><P>A10-D10. T20S, L40R, N81Q
</P><P>E10-H10. T20S, L40R, T59K
</P><P>A11-D11. T20S, L40R, T59K_M61S
</P><P>E11-H11. T20S, L40R, T59N
</P><P>A12-D12. T20S, L40W
</P><P>E12,F12. T20S, L40E
</P><P>
Plate B
</P><P>A1-D1. T20S, L40Y
</P><P>E1-H1. T20S, T59I
</P><P>A2-D2. T20S, T59K
</P><P>E2-H2. T20S, T59R
</P><H2><SPAN class="mw-headline" id="9.2F17.2F10"><B>9/17/10</B></SPAN></H2><UL><LI>Store in -80C. Send for Sequencing Monday.
</LI><LI>100uL 20% Glycerol+100uL Culture
</LI></UL><LI>Plate A &amp; Plate B
</LI><H2><SPAN class="mw-headline" id="9.2F21.2F10"><B>9/21/10</B></SPAN></H2><UL><LI>Autoclaved 30 250mL flasks and 2L TB (No Kan).
</LI></UL><H2><SPAN class="mw-headline" id="9.2F22.2F10"><B>9/22/10</B></SPAN></H2><P>Grrr...angry at FedEx for being slow.
</P><H2><SPAN class="mw-headline" id="9.2F23.2F10"><B>9/23/10</B></SPAN></H2><UL><LI>Genewiz determined glycerol stocks cross contaminated due to sitting in Memphis, Tennessee for two days...OR possibly bad seal.
</LI><LI>Resending glycerol stocks (20uL per stock) for sequencing. Hope for sequencing results by Friday.
</LI><LI>Looks like Plate B is fine, did not send again. Will see sequencing tomorrow.
</LI></UL><H2><SPAN class="mw-headline" id="9.2F24.2F10"><B>9/24/10</B></SPAN></H2><UL><LI>2L flask of TB is tempting me to do wet lab work...
</LI></UL><H2><SPAN class="mw-headline" id="9.2F27.2F10"><B>9/27/10</B></SPAN></H2><P>Overnights started 3pm
</P><UL><LI>DNSS and DCPNSSFoldit have 2mL in order to take 1mL for miniprep and PCR
</LI></UL><OL><LI>F24H
</LI><LI>F24H, L40R, T59D_M61S
</LI><LI>F24H, L40R, T59N_M61S
</LI><LI>F24H, L40R, T59S_M61S
</LI><LI>F24H, T59R
</LI><LI>F24W
</LI><LI>F24Y, L40R
</LI><LI>F24Y, L40R, T59N
</LI><LI>F60W
</LI><LI>L40R
</LI><LI>T20S, L40D
</LI><LI>T20S, L40E
</LI><LI>T20S, L40F
</LI><LI>T20S, L40H
</LI><LI>T20S, L40K
</LI><LI>T20S, L40Q
</LI><LI>T20S, L40R
</LI><LI>T20S, L40R, F60E
</LI><LI>T20S, L40R, F60Q
</LI><LI>T20S, L40R, N81E
</LI><LI>T20S, L40R, N81Q
</LI><LI>T20S, L40R, T59K
</LI><LI>T20S, L40R, T59K_M61S
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><H2><SPAN class="mw-headline" id="9.2F28.2F10"><B>9/28/10</B></SPAN></H2><P>Plasmid for pSB1C3, CapD, and DCP obtained
</P><P>PCRed CapD and DCP
</P><P>PCR Purification for CapD and DCP purified linear double-stranded DNA.
</P><P>All stored in -20C
</P><P>Inoculate and induce. (OD600=0.4-0.6) Start growing at 18C 5:00pm.
</P><OL><LI>F24H
</LI><LI>F24H, L40R, T59D_M61S
</LI><LI>F24H, L40R, T59N_M61S
</LI><LI>F24H, L40R, T59S_M61S
</LI><LI>F24H, T59R
</LI><LI>F24W
</LI><LI>F24Y, L40R
</LI><LI>F24Y, L40R, T59N
</LI><LI>F60W
</LI><LI>L40R
</LI><LI>T20S, L40D
</LI><LI>T20S, L40E
</LI><LI>T20S, L40F
</LI><LI>T20S, L40H
</LI><LI>T20S, L40K
</LI><LI>T20S, L40Q
</LI><LI>T20S, L40R
</LI><LI>T20S, L40R, F60E
</LI><LI>T20S, L40R, F60Q
</LI><LI>T20S, L40R, N81E
</LI><LI>T20S, L40R, N81Q
</LI><LI>T20S, L40R, T59K
</LI><LI>T20S, L40R, T59K_M61S
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><H2><SPAN class="mw-headline" id="9.2F29.2F10"><B>9/29/10</B></SPAN></H2><P>Spin down and Store at -20C. 21 lost lid (possible contamination)
</P><OL><LI>F24H
</LI><LI>F24H, L40R, T59D_M61S
</LI><LI>F24H, L40R, T59N_M61S
</LI><LI>F24H, L40R, T59S_M61S
</LI><LI>F24H, T59R
</LI><LI>F24W
</LI><LI>F24Y, L40R
</LI><LI>F24Y, L40R, T59N
</LI><LI>F60W
</LI><LI>L40R
</LI><LI>T20S, L40D
</LI><LI>T20S, L40E
</LI><LI>T20S, L40F
</LI><LI>T20S, L40H
</LI><LI>T20S, L40K
</LI><LI>T20S, L40Q
</LI><LI>T20S, L40R
</LI><LI>T20S, L40R, F60E
</LI><LI>T20S, L40R, F60Q
</LI><LI>T20S, L40R, N81E
</LI><LI>T20S, L40R, N81Q
</LI><LI>T20S, L40R, T59K
</LI><LI>T20S, L40R, T59K_M61S
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><H2><SPAN class="mw-headline" id="9.2F30.2F10"><B>9/30/10</B></SPAN></H2><UL><LI>Possibly lots of proteins have crashed out...I guess they were much less stable this time around for some reason?
</LI></UL><UL><LI>Continue with only pipetting out remaining supernatent. Left possibly crashed out protein...
</LI><LI>Digested CapD and CapDCP
</LI><LI>Attempted ligation, but did not use digested pSB1C3 vector, so no possibility significant yield.
</LI></UL><H2><SPAN class="mw-headline" id="10.2F1.2F10"><B>10/1/10</B></SPAN></H2><UL><LI>4,5,6,7,10,11,12,14,15,17,20 have weird curves on nanodrop
</LI><LI>Run enzyme assay
</LI><LI>Plated CapD and CapDCP transformed into pSB1C3 w/ RFP
</LI></UL><P>Protein Concentrations(mg/mL) <I>Diluting them all is ridiculous...</I></P><OL><LI>F24H - 1.785
</LI><LI>F24H, L40R, T59D_M61S - 2.064
</LI><LI>F24H, L40R, T59N_M61S - 2.636
</LI><LI>F24H, L40R, T59S_M61S - 0.2679
</LI><LI>F24H, T59R - 1.43
</LI><LI>F24W - 0.03752
</LI><LI>F24Y, L40R - 0.1868
</LI><LI>F24Y, L40R, T59N - 2.029
</LI><LI>F60W - 4.399
</LI><LI>L40R - 0.2323
</LI><LI>T20S, L40D - 0.1174
</LI><LI>T20S, L40E - -0.08503
</LI><LI>T20S, L40F - 1.801
</LI><LI>T20S, L40H - 0.4182
</LI><LI>T20S, L40K - 0.6324
</LI><LI>T20S, L40Q - 1.513
</LI><LI>T20S, L40R - 0.3727
</LI><LI>T20S, L40R, F60E - 1.762
</LI><LI>T20S, L40R, F60Q - 2.578
</LI><LI>T20S, L40R, N81E - 0.4964
</LI><LI>T20S, L40R, N81Q - 1.767
</LI><LI>T20S, L40R, T59K - 1.960
</LI><LI>T20S, L40R, T59K_M61S - 1.818
</LI><LI>CapDNSS - 2.901
</LI><LI>CapDCPNSSFoldit - 2.380
</LI></OL><P>Flash froze all proteins. A blue alpha (lower case greek) denotes them.
</P><H2><SPAN class="mw-headline" id="10.2F4.2F10"><B>10/4/10</B></SPAN></H2><P>Colony PCR on CapD, DCP, and control plate
</P><UL><LI><I>Looks like plates possibly mislabeled? Something messed up along the way since CapD plate has very few colonies, Control has as many as DCP.</I></LI></UL><P>No bands turned up on gel. Redoing Digestion, Ligation, PCR process.
</P><P>Overnights
</P><OL><LI>F24H
</LI><LI>F24H, L40R, T59N_M61S
</LI><LI>F24Y, L40R, T59N
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><H2><SPAN class="mw-headline" id="10.2F5.2F10"><B>10/5/10</B></SPAN></H2><P>Michaelis Menten Curves for 
</P><OL><LI>F24H
</LI><LI>F24H, L40R, T59N_M61S
</LI><LI>F24Y, L40R, T59N
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><UL><LI>Transpeptidation first plate
</LI><LI>Hydrolysis second plate
</LI><LI>0.0001mg/mL of enzyme
</LI><LI>Substrate start 1uM serial dilution (halves) across. PMT High Sensitivity.
</LI><LI><I>Weird curve for transpeptidation</I></LI><LI>PCR amplify insert annealing at 55C, 60C, and 65C
</LI></UL><H2><SPAN class="mw-headline" id="10.2F6.2F10"><B>10/6/10</B></SPAN></H2><UL><LI>Run gel of insert to see which annealing temperature is optimal
</LI><LI>Failed THRICE...hopefully I get it right this time
</LI></UL><LI>DNA gel showed no bands for any of the samples. No inserts PCRed out?
</LI><P>DNA gel=
</P><UL><LI>50mL TAE
</LI><LI>0.5g Agarose
</LI><LI>Microwave max power for 1 minute
</LI><LI>10uL EtBr
</LI><LI>Pour into cartridge
</LI><LI>Place combs in
</LI><LI>Let sit for 20 minutes
</LI><LI>Load samples
</LI><LI>Run for 15-30 minutes
</LI><LI>Look at under UV
</LI></UL><H2><SPAN class="mw-headline" id="10.2F7.2F10"><B>10/7/10</B></SPAN></H2><UL><LI>Michaelis-Menten Curves for
</LI></UL><OL><LI>F24H
</LI><LI>F24H, L40R, T59N_M61S
</LI><LI>F24Y, L40R, T59N
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><UL><LI>Transpeptidation first plate
</LI><LI>Hydrolysis second plate
</LI><LI>0.0001mg/mL of enzyme
</LI><LI>Substrate start 1uM serial dilution (halves) across. PMT High Sensitivity.
</LI></UL><P>PCR again with anneal at 55, 60, and 65. <I>Forgot dNTPs first time</I></P><OL><LI>F24H
</LI><LI>F24H, L40R, T59N_M61S
</LI><LI>F24Y, L40R, T59N
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI><LI>CapDCPNSSFoldit (T7 Forward and Reverse Primers)
</LI></OL><UL><LI>60C anneal seems to be best. 
</LI><LI>CapDNSS failed. Assuming forgot forward primer. Redo at 60C.
</LI><LI>Assay test of BSA dilution showed ~1000RFU better than water dilution
</LI><LI>Run PBS dilution vs H2O dilution vs BSA dilution vs BSA control tomorrow
</LI></UL><P>Run on gel to see what worked. <I>Loaded 6uL of samples. Messed up ladder, so added 10uL ladder and 6uL loading dye.</I></P><P>Think about abandoning F24Y, L40R, T59N due to instability/inactivity.
</P><P>T7 Forward/Reverse DCP tossed since Justin found better cloning idea using RFP
</P><H2><SPAN class="mw-headline" id="10.2F8.2F10"><B>10/8/10</B></SPAN></H2><UL><LI>Got clean curves, but lower activity. Regrow proteins and try again. Can use for iGEM, but better curves are better.
</LI></UL><H2><SPAN class="mw-headline" id="10.2F11.2F10"><B>10/11/10</B></SPAN></H2><UL><LI>Colony PCR. We'll see how it goes...
</LI><LI>Growing up new cells overnight 2mL TB+Kan
</LI></UL><OL><LI>F24H
</LI><LI>L40R
</LI><LI>F24H, L40R, T59N_M61S
</LI><LI>T2V
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><H2><SPAN class="mw-headline" id="10.2F12.2F10"><B>10/12/10</B></SPAN></H2><P>Inoculate at 9am. Induce with 25uL IPTG OD600 at 0.3-0.6. Grow at 18C for 24 hours. Can Spin down after 11am.
</P><OL><LI>F24H
</LI><LI>L40R
</LI><LI>F24H, L40R, T59N_M61S
</LI><LI>T2V
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><P>Miniprep plasmids. (Eluted into wrong tubes once. Luckily grew 2mL...Second attempt running)
</P><OL><LI>F24H
</LI><LI>F24H, L40R, T59N_M61S
</LI><LI>F24Y, L40R, T59N
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><H2><SPAN class="mw-headline" id="10.2F13.2F10"><B>10/13/10</B></SPAN></H2><UL><LI>Spun down cells
</LI><LI>Cells not suspended very well.
</LI></UL><P>Purified Protein
</P><OL><LI>F24H - 2.554
</LI><LI>F24H,L40R,T59N_M61S - 2.591
</LI><LI>L40R - 2.771
</LI><LI>T2V - 0.3800
</LI><LI>CapDNSS - 2.509
</LI><LI>CapDCPNSSFoldit - 3.554
</LI></OL><H2><SPAN class="mw-headline" id="10.2F14.2F10"><B>10/14/10</B></SPAN></H2><P>MM Assay Data (Transpeptidation, then hydrolysis)
</P><OL><LI>F24H
</LI><LI>F24H,L40R,T59N_M61S
</LI><LI>L40R
</LI><LI>T2V
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI><LI>Blank
</LI></OL><P>HEPES not pHed. Must pH to ~8 before use.
</P><H2><SPAN class="mw-headline" id="10.2F15.2F10"><B>10/15/10</B></SPAN></H2><P>MM Assay failed again
</P><H2><SPAN class="mw-headline" id="10.2F18.2F10"><B>10/18/10</B></SPAN></H2><UL><LI>Soluble protein not dead
</LI><LI>Recalibrate protein concentration for PMT=Medium. (Check Tomorrow since fire alarms are annoying...)
</LI><LI>Start Substrate at 1uM. (10x is 10uM)
</LI><LI>Determine if Master Mixes or Individual Wells is better...Master Mixes gets clean, consistent data, Individual Wells is easier...
</LI></UL><H2><SPAN class="mw-headline" id="10.2F19.2F10"><B>10/19/10</B></SPAN></H2><P>Complete Substrate/Product Curve. 
</P><UL><LI>Substrate 10uM (Serial dilution in halves across)
</LI><LI>Product 1uM (Serial dilution thirds down)
</LI></UL><P>Data Works
</P><P>Overnights to Re-Express
</P><OL><LI>F24H
</LI><LI>F24H,L40R,T59N_M61S
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><H2><SPAN class="mw-headline" id="10.2F20.2F10"><B>10/20/10</B></SPAN></H2><P>Overnights (F24H did not grow so re-express)
</P><OL><LI>F24H
</LI><LI>F24H,L40R,T59N_M61S
</LI><LI>L40R
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><H2><SPAN class="mw-headline" id="10.2F21.2F10"><B>10/21/10</B></SPAN></H2><P>Induced with 25uL IPTG at 3pm (OD600 of 0.3)
</P><OL><LI>F24H
</LI><LI>F24H,L40R,T59N_M61S
</LI><LI>L40R
</LI><LI>CapDNSS
</LI><LI>CapDCPNSSFoldit
</LI></OL><H2><SPAN class="mw-headline" id="10.2F22.2F10"><B>10/22/10</B></SPAN></H2><P>Spin down and start purification by 5?
</P><H1><SPAN class="mw-headline" id="Vectors"><B>Vectors</B></SPAN></H1><H2><SPAN class="mw-headline" id="6.2F15.2F10"><B>6/15/10</B></SPAN></H2><P>PCR out Lac I and F1 origin from pet29b vector
</P><P>run following PCR reaction 
0.5ul forward primer
0.5ul reverse primer
0.5ul 25mM dNTP
0.5ul pet29 plasmid 
0.5ul fusion DNA polymerase
10ul 5x polymerase buffer
37.5ul dH20
</P><P>PCR program: 
·	Start: 98 °C for 2 min. (melt) 
·	Cycle 98 °C for 0.5 min (melt) 
·	60 °C for 0.5 min. (anneal) 
·	72 °C for 0.5 min. 
·	No. of Cycles: 29
·	End: keep at 4 °C forever 
</P><P>Check for PCR product 
run product with 100bp ladder on 1.5% agarose gel
load 5ul sample (or ladder) and 1ul 6x loading dye into gel
</P><P>Miniprep psb1A3 plasmids for day 2,
take six 2ml LB broth overnight growths of E. Coli with psb1A3 
pellet cells in centrifuge, pour of supernatant fluid. 
Use instructions from miniprep kit 
store purified psb1A3 plasmids in IGEM vector box for day 2 use
</P><H2><SPAN class="mw-headline" id="6.2F16.2F10"><B>6/16/10</B></SPAN></H2><P>PCR gradients for F1 and LacI
10 ul reactions 
variables: temp (58, 65, 72) for extension and DMSO (0%, 5%, and 10%)
run on 1.5% arganose gel with 1kb ladder and reactions from day 1
</P><P>results of agarose gel
F1 bands on all 72C and 65C temps, 5%DMSO at 58C, Day 1 PCR
LacI all 5% DMSO, 10% DMSO 58C and 65C, Day 1
PCR purified DNA placed in IGEM vector box -20C
</P><H2><SPAN class="mw-headline" id="6.2F17.2F10"><B>6/17/10</B></SPAN></H2><P>digest PCR products to make sticky ends
double digestion with EcoRI and PstI
double digestion of PSB1A3 w/ EcoRI and PstI in NEB buffer 3 with BSA at 37C for 1 hours. PCR purify to remove enzymes
</P><P>digestions
	5ul NEB buffer 3
	2ul each enzyme (EcoRI and PstI)
	15ul PCR purified DNA (day 1 and 1.5) or plasmid DNA
	.5ul BSA
	25.5ul dH20
	50ul total volume
</P><P>ligate PCR products into plasmids (PSB1A3)
	10ul T4 buffer
	30ul PCR purified F1 digest or LacI digest 
	15ul PCR purified psb1A3 digest 
	43.5ul dH20
	1.5ul T4
</P><P>	held at rm temp. 1hour for ligation
</P><P>place ligated plasmids into E. Coli for expression 
	transformed into E. Coli x-10 gold series competent cells
	2 plates Carbacillin held at 37C
</P><P>plate E. Coli on Ampicillin to as a selection (note death gene will ensure non-ligated products don't contaminate sample)
</P><H2><SPAN class="mw-headline" id="6.2F22.2F10"><B>6/22/10</B></SPAN></H2><P>Isolate colonies growing on plates, place part in storage.
PCR test for inserts (scrap 8 colonies from each plate)
suspend colonies in 10ul dH20, 
run PCR with 1ul VF, 1ul VR (biobrick vectors), 5ul premixed 2x Master mix (taq polymerase, taq buffer, dNTP), and 3ul colonies suspension.
</P><P>Run PCR on 1% arganose gel
</P><P>Make suspension of vector expressing E. coli, use for further experiments.
</P><P>2 colonies of F1 and LacI (each) that showed recomination in gel placed in 2ml TB with Carbicillian @37C shaker 
</P><H2><SPAN class="mw-headline" id="6.2F23.2F10"><B>6/23/10</B></SPAN></H2><P>minipreped F1 and LacI broths for sequencing, sent into sequencing at genewiz
</P><H2><SPAN class="mw-headline" id="6.2F24.2F10"><B>6/24/10</B></SPAN></H2><P>PCR out constitutive promoters (Bba_J23100, J23114, J23113) and Lac regulated promoter (R0011) use ordered primer and VF2 primer. 
</P><P>Mixed
0.5ul specific primer
0.5ul VF2 primer
0.5ul 25mM dNTP
0.5ul DNA from distribution plate 
0.5ul fusion DNA polymerase
10ul 5x polymerase buffer
37.5ul dH20
</P><P>(note master mix of H20, buffer and dNTP was used)
</P><P>PCR was ran as follows
</P><P>PCR program: 
·	Start: 98 °C for 2 min. (melt) 
·	Cycle 98 °C for 0.5 min (melt) 
·	58 °C for 0.2 min. (anneal) 
·	72 °C for 0.5 min. 
·	No. of Cycles: 29
·	End: keep at 4 °C forever 
</P><P>
run PCR results on 2% agarose gel to check for products. 
</P><P>Products seen on lac I regualer (R0011) and J23113, no product observed on J23100 and J23114, will wait for results of transformation to decide if rePCRing or requesting new DNA from IGEM
</P><P>Transformed DNA from plate into x10 gold series E. Coli (15ul cells to 1ul DNA)
grown in TB for 1 hour on shaker @ 37C
plated onto antibiotic plates (5 amp/ 1 Kan)
</P><P>Sequencing results of Day 3.5, both colonies show 100% homology with expected DNA from ape sequence 
</P><H2><SPAN class="mw-headline" id="6.2F25.2F10"><B>6/25/10</B></SPAN></H2><P>result of plates, growth on all constitutive promoters and on psb4A5, no growth on psb1k3 and R0011 (lac regulated promoter).  
</P><P>Growth plates were scraped and inoculated into 2.5ml TB for overnight. These broths will be used for a glycerol stock and miniprep.
</P><P>Plates with no growth, the psb1k3 and R0011 were re-transformed using electrophoresis and plated onto the appropriate antibiotic plate (amp and Kan) 
</P><P>Also from the growth plates suspension were made of J23100 and J23114 which were PCRed (with 5% DMSO, all other same as day 5 protocol) and ran on a 2% agarose gel.
Strong band from the J23100, weak band from the J23114.  
</P><P>-over weekened broths were spun down into pellets and glycerol stocks of J23100, J23113, J23114, psb4A5 was made.
</P><H2><SPAN class="mw-headline" id="6.2F28.2F10"><B>6/28/10</B></SPAN></H2><P>PCR purify PCR results and check DNA concentration using Nanodropper, low concentrations on PCR products, will proceed and adjust concentration so there is a 10 to 1 insert to vector concentration for ligations.  
</P><P>Isolate plasmid with F1 origin and Lac I from Day 4 (minipreps, done prior for F1 and LacI, for constitutive promoters done today)
Digest plasmid with SpeI and PstI (F1 with Buffer 2 and BSA)
Lac I and constitutive promoters XbalI and PstI. (Buffer 2 with BSA)
digestions analyzed on nanodropper, 
</P><P>Ligate the digestions. Overnight ligation at 16C
10ul ligation using 
2ul vector
6ul insert (or dH20 for control)
1ul buffer
1ul T4 ligase
</P><H2><SPAN class="mw-headline" id="6.2F29.2F10"><B>6/29/10</B></SPAN></H2><P>finished ligation 
Chemically transformed into gold series x10 cells
Express with 30ul on Ampicillin, grown at 37c overnight
</P><H2><SPAN class="mw-headline" id="6.2F30.2F10"><B>6/30/10</B></SPAN></H2><P>growth plates, no growth on control or J23100 plates, replated with 200ul on amp/ overnight @37C
</P><P>Check for recombinants by PCRing using VF2 and VR in Taq
</P><P>run on 1% agarose gel
</P><P>make a broth(s) of Day 6 E. coli colonies that show signs of recombination for miniprep tommorrow, all but 1 lacI +F1 showed recombination, made broths with carb, set @37C for overnights.
</P><H2><SPAN class="mw-headline" id="7.2F1.2F10"><B>7/1/10</B></SPAN></H2><P>growth plates, 1 colony on control, colonies on J23100, PCRing with biobrick primers (VF2 and VR) to run on 1% agarose gel and make broths of colonies suspensions.
</P><P>Minipreped  broths from J23113, J23114, and LacI constructs with F1 
reinvigorated broth with 1ml TB and stored @4C for latter glycerol stocks
</P><P>Kineased T7 promoter using 
·	i. 9uL Kinase Buffer
·	ii. 3uL of 10mM ATP
·	iii. 3uL T4 Polynucleotide Kinase
·	iv. 52uL ddiH2O
·	v. 21uL Olgio
incubated @37C for 1 hour then set at 95 with a slow cooling to room temp over 1 hour (PCR protocol, 95 then -.5 per 30 seconds)
Ran results on 1% agarose gel. Results showed distinct band around 100bp with a smear below
</P><H2><SPAN class="mw-headline" id="7.2F2.2F10"><B>7/2/10</B></SPAN></H2><P>Minipreped broth of J23100
</P><H2><SPAN class="mw-headline" id="7.2F6.2F10"><B>7/6/10</B></SPAN></H2><P>digest, ligate and inserted into X10 gold cells T7
digest: digest psb1A3 with E and S (T7 ready to ligate from PCR) ?
ligate: 
plate onto Carbicillin plate
</P><H2><SPAN class="mw-headline" id="7.2F7.2F10"><B>7/7/10</B></SPAN></H2><P>digest psb1A3 with F1+lacI genes, at S/P (50ul reaction)
digest R0011 with X/P (50ul reaction)
</P><P>pcr out the T7 from the X10 gold cells grown up on day 9
pcr out GFP (E0040) from Igem 2009 stock
digest Pcr T7 with X/P
</P><UL><LI>purified GFP/ T7 contaminated during purification* 
</LI></UL><P>ran on 1% agarose gel, banding around 700 bp for GFP, and banding around 100bp for colony 1, no dna in colony 2 or 3
</P><P>
overnight ligate F1+lacI with R0011/T7 hold at 16C
</P><P>between digest and ligations made more Carbicillin plates
</P><P>examine sequencing results of J23100/114/113.  113 good stored in Glycerol stock, re-transformed J23100/114 from earlier ligation held at -20C, plated on Carbicillin for overnight growth
</P><H2><SPAN class="mw-headline" id="7.2F8.2F10"><B>7/8/10</B></SPAN></H2><P>plate results F1+ j23100/j23114: no colonies on J23114 redo digestion/ligation
colonies on J23100, Pcr out (on 9 july, stored at 4C)
re-plated j23114 using remaing broth from recovery media
transform overnights into X10 gold series and grow on Carbicillin plates
miniprep T7 in psb1A3, retain some broth for glycerol stock(?)
</P><H2><SPAN class="mw-headline" id="7.2F9.2F10"><B>7/9/10</B></SPAN></H2><P>growth plate results: R0011 w/ F1&amp;LacI shows colonies, 1 colony on J23114, no colonies on T7 recombination.  Plates retained at 4C
</P><P>PCR T7 and GFP, purify after (using VF2 and VR, phusion) (purfied and nanodroped afterwords, looks good)
pcr chosen colonies to check for recombination (use VF2 and specific promoter, green Taq)
</P><P>run on gel (gel results, J23100 1,2,4, J23114, R0011 1,3,4) Broths made of J23100 1,2, J23114, R0011 1,4. Placed at 4C send for sequencing?
</P><H2><SPAN class="mw-headline" id="7.2F26.2F10_2"><B>7/26/10</B></SPAN></H2><P>T7 digestion, ligation, insertion using T7, LacI+F1
using T7 pcr purified VR/VF digest XP , LacI+F1 digest SP
plated on Carbicillin plates at 37C
</P><P>broths from J23100, J23114, R0011 reinvigorated and placed in 37C water bath for growth
</P><H2><SPAN class="mw-headline" id="7.2F27.2F10_2"><B>7/27/10</B></SPAN></H2><P>Miniprep broths from j23100, j23114, R0011
check T7 plate for growth, select 4 colonies to check for inserts via PCR with Taq mix and gel run. Gel ran with Lac I +F1, pictures did not turn out well (need better camera/ backlight) no dicernable difference between control (F1+ lacI) and T7, all 4 showed strong bands.  Colonies 1 and 2 used for making broths held overnight at 37C in shaker waterbath.
</P><H2><SPAN class="mw-headline" id="7.2F28.2F10_2"><B>7/28/10</B></SPAN></H2><P>Nano drop Minipreps from 27 july
Miniprep 2 T7 broths.
Send j23100, j23114, R0011 and T7 in for sequencing.
</P><H2><SPAN class="mw-headline" id="7.2F29.2F10_2"><B>7/29/10</B></SPAN></H2><P>sequencing results, J23100 good, R0011 good, T7 questionable (send in other samples) j23114 bad, redo
</P><H2><SPAN class="mw-headline" id="7.2F30.2F10_2"><B>7/30/10</B></SPAN></H2><P>ligate F1 origin in psb1A3 with j23114 using earlier digestions remains plated out portion of earlier j23114 broth 
</P><H2><SPAN class="mw-headline" id="8.2F2.2F10_2"><B>8/2/10</B></SPAN></H2><P>PCRed cells ran on 1% agarose gel at 155V for 1 hour.  No good T7 (used both ends of PCR primers, causing them to bind to each other and get no results) J23114 looks good on 7 of 8 colonies ran. 2 pick (1 from old 1 ligation, 1 from new ligation) for overnight growth
</P><H2><SPAN class="mw-headline" id="8.2F3.2F10_2"><B>8/3/10</B></SPAN></H2><P>minipreped 6 colonies for sequencing, 4 T7 and 2 j23114 colonies
sequencing results show j23114 cross contaminated with j23113, earlier results confirm this.  T7 checking sequence incorrect, T7 results from earlier good and used for further vector construction.  
</P><H2><SPAN class="mw-headline" id="8.2F9.2F10_2"><B>8/9/10</B></SPAN></H2><P>digest, ligate, transformation, and plate GFP into vectors
transform stock vectors and plate
check on status of other plasmid, other plasmids in stock
</P><H2><SPAN class="mw-headline" id="8.2F10.2F10_2"><B>8/10/10</B></SPAN></H2><P>PCR GFP colonies to check for transformation (VF/VR)
overnights of good colonies, (2 sets, TB and LB) (+ and - GFP)
</P><H2><SPAN class="mw-headline" id="8.2F11.2F10_2"><B>8/11/10</B></SPAN></H2><P>make glycerol stocks of + - minus (TB)
expression psb1A3 (from LB) (see note from Justin), expression stopped due to no GFP signs of Constitutive promoter (j23100) during Miniprep.  
Minipreped +/ - GFP psb1A3 for latter
</P><H2><SPAN class="mw-headline" id="8.2F12.2F10_2"><B>8/12/10</B></SPAN></H2><P>send in J23114 for sequencing.
PCR GFP
</P><H2><SPAN class="mw-headline" id="8.2F13.2F10_2"><B>8/13/10</B></SPAN></H2><P>digest, ligate and plate (grow at room temp over weekend) GFP vectors in psb1A3
sequencing result for J23114, good... found earlier flip in J23114 and J23113 (first sequencing)... which means I tossed out the vector I needed :(
</P><H2><SPAN class="mw-headline" id="8.2F16.2F10_2"><B>8/16/10</B></SPAN></H2><P>ligate, digest, transformation J23114, plated after 45 minutes so could get to meeting in time, remainder of broth left in shaker overnight. 
pick colonies for PCR verification with GFP, J23100 no good pcr results (solid bands, but too low to have GFP included) included anyhow to see if any Green would show up.  Other PCR's had expected results (bands below F1 + lacI for J23113, above for T7 and R0011) ladder shows up week even with doubling the amount normally included...
overnights GFP (TB and LB) overnight colonies from plate with no GFP (LB)
</P><H2><SPAN class="mw-headline" id="8.2F17.2F10_2"><B>8/17/10</B></SPAN></H2><P>pick colony from J23114 and PCR verify/ start overnights
Glycerol stock GFP transformed vectors, J23100 not stocked due to no green pigment shown when spun down, will re-ligate on the 18th (tomorrow) 
</P><H2><SPAN class="mw-headline" id="8.2F18.2F10_2"><B>8/18/10</B></SPAN></H2><P>Miniprep overnights and send to sequencing (J23114)
digestions, ligate, transform J23100 and J23114 with GFP (multiple colonies so you have best for sequencing results the next day)
</P><H2><SPAN class="mw-headline" id="8.2F19.2F10_2"><B>8/19/10</B></SPAN></H2><P>PCR verification of colonies, start overnights (TB for stocks)
sequencing results: j23114 good (2 and 3)
</P><H2><SPAN class="mw-headline" id="8.2F20.2F10_2"><B>8/20/10</B></SPAN></H2><P>make glycerol stocks of J23114
</P><H2><SPAN class="mw-headline" id="8.2F31.2F10_2"><B>8/31/10</B></SPAN></H2><P>grow broths from glycerol stock (in LB)
meet with Justin to discuss plans
</P><H2><SPAN class="mw-headline" id="9.2F1.2F10_2"><B>9/1/10</B></SPAN></H2><P>expresssion assay
notes on assay: T7 used glycerol stocks 10ul into 1ml TB all others 100ul LB overnight into 1ml TB, no stock of j23114 so no j24113 overnight or non GFP for data points. Overnights grown for 24h. +/- 10 minute, grown for 4h at 37C shaker
IPTG added at 2.5h to T7 and R0011, low growth of broths, re-run assay tomorrow.  Check for .1 colony density (OD 600, clear plate, 100ul broth) before induction. 
PCR out vectors 
</P><H2><SPAN class="mw-headline" id="9.2F2.2F2010"><B>9/2/2010</B></SPAN></H2><P>assays psb1a3
overnights grown in 1ml LB w/Carbicillin, grown for approx. 16h., 50ul transferred to 2ml TB w/Carb.  checked for colony density in broth by reading OD 600 at 3h.  all constitive above .1 OD, R0011 GFP low (.068) and GFP control low.  will induce with IPTG in 30min. and wait 4 hours for final read.  
verify pcr: all but j23100 with GFP showed bands at appropriate positions. nano drop of j23100 with GFP is at 16ng/ul   
re PCRing J23100 + GFP, nanodrop had around 34ng/ul
made plates (carb/chlor)
</P><H2><SPAN class="mw-headline" id="9.2F7.2F2010"><B>9/7/2010</B></SPAN></H2><P>digest, ligate, transform vectors (3k3, 4a5), plates of transformation on appropriate media (Kan/ Carb) made
note: possible contamination of T7 +GFP in 4A5 and T7 in 3K3 during purification process
remainer of electro competent cells placed in 4 shelf up, 1 drawer, 2 row upper  box
grow k3k overnight for stock
</P><H2><SPAN class="mw-headline" id="9.2F8.2F2010"><B>9/8/2010</B></SPAN></H2><P>plasmid purfication of 3k3 overnight 
start overnights of 1C3
verify colonies by PCR (done at 1600), 2 gels due to amount of DNA ran (1 hour at 100V 155amps)
note on 4A5 j23114 and j23113 reversed for 4A5 colonies/pcr (labled as 3c and 2c respectivley)
MM 2 used, 1 VF + GFP reverse (for all Gfps) other VF 
replated from broths plates that did not grow, 
start overnights growths of good colonies for glycerol/ plasmid stocks
note: wrong t7 used and gel ran to long, broths with ? area unclear via PCR results, re-run from purified plasmids tomorrow 
</P><H2><SPAN class="mw-headline" id="9.2F9.2F2010"><B>9/9/2010</B></SPAN></H2><P>glycerol stocks of overnights
minipreps of overnights
PCR overnights in question (T7 in 3k3 and 4a5 and j23100 + Gfp in 4a5)
run 1% agarose gel with 1kb ladder to verify vectors (t7 3k3 good, t74a5 and j23100+gfp in 4a5 plates put back in 37c overnight for new colonies selecting)
make plates of Kan
digest 1c3
ligate 1c3 to all, former non good with old digestion. chemical(j23100+gfp 3k3, j23113 +gfp 3k3, r0011 3k3, r0011 +gfp 3k3,  j23114 4a5) electric (t7 +gfp 3k3, t7 for 1c3 and 1a3)
transform T7 1a3, all 1c3, old ligations of non turn outs, 
plate on appropriate antibiotics
</P><H2><SPAN class="mw-headline" id="9.2F10.2F2010"><B>9/10/2010</B></SPAN></H2><P>plates results: no colonies; t7 psb1c3, t7+gfp psb3k3, t7 psb1a3, j23100+gfp psb3k3, R0011 psb3k3. 
	replated no colonies from broth
	lawn of colonies (replate from broth at 100 dillution; R0011+gfp psb1c3, R0011 psb1c3, j23113+gfp psb1c3, j23100 psb1c3, 	j23113 psb1c3, j23114 pab1c3, j23100+gfp psb1c3, j23114+gfp psb1c3, t7+gfp psb1a3)
	good colonies(pcr); R0011+gfp psb3k3, j23113+gfp psb3k3, t7+gfp psb1c3, j23114 psb4a5
	 earlier plates new pcr; t7 psb4a5, j23100+gfp psb4a5 
PCR with taq colonies to verify inserts (2 colonies from each of 6 plates, 11 total)
pcr error, pcr overhead not heating caused no distingishable dna to be visible on gel. no colonies held
make more glycerol stocks (autoclave)
all plates placed in 4c walk in, remove for growth when you return to the lab
</P><H2><SPAN class="mw-headline" id="9.2F20.2F2010"><B>9/20/2010</B></SPAN></H2><P>Clean- up borrowed lab space
PCR colonies from held plates (R0011+gfp psb3k3, j23113+gfp psb3k3, t7+gfp psb1c3, j23114 psb4a5, t7+gfp psb3k3
	 earlier plates new pcr; t7 psb4a5, j23100+gfp psb4a5 )
ran on 1% agarose gel, no psb4a5 results
</P><H2><SPAN class="mw-headline" id="9.2F21.2F2010"><B>9/21/2010</B></SPAN></H2><P>re-do t7 psb4a5, j23114 psb4a5, j23100+gfp psb4a5, j23100 psb3k3, r0011 psb3k3, all 1c3 into dh5a cell line
pick 2 colonies from each good plate and pcr with taq for gel run to verify insert, no pcr results (loss of some product from machine heating problems) for t7 psb1c3 or t7+gfp psb1a3... re do tomorrow in middle area...
made kan, carb, and chorl plates
glycerol stocked t7+gfp psb3k3, t7+gfp psb1c3, R0011+gfp psb3k3
ligate and transformed t7 psb4a5, j23114 psb4a5, j23100+gfp psb4a5, j23100 psb3k3, r0011 psb3k3, all 1c3 into dh5a 
</P><H2><SPAN class="mw-headline" id="9.2F22.2F2010"><B>9/22/2010</B></SPAN></H2><P>check for growth on plates
colony PCR 2 colonies from each good plate and held t7 psb1c3 and t7+gfp psb1a3, colonies on j23114+gfp, t7 psb1a3, j23100 psb1c3, and j23100+ gfp psb1c3
replate from broth all non-growth plates
make more Chlor. and Carb. plates
run PCR on gel, no gel results, not even a ladder... eth bromide going bad?
start overnights from glycerol stocks of all non psb1a3 X10 gold stocks to transform into DH5a
transform t7 psb1c3 from DH5a to BL21 cell line for expression assay(2nd plate set)
transform psb1a3 stocks into DH5a cell line
start overnights for digestion tomorrow of all cell pcr colonies (11 total)
</P><H2><SPAN class="mw-headline" id="9.2F23.2F2010"><B>9/23/2010</B></SPAN></H2><P>check plates for growth, no growth, j23100 psb3k3, r0011 psb3k3, (redo digestion), t7 psb4a5 in BL21 (replated)
pcr (taq) to check for insert
minipreped all X10 gold cells for transforming into DH5a
run gel to verify inserts, results: no colonies j23100 + gfp psb4a5, j23113+gfp psb1c3, t7 psb4a5 (band too low?), r0011+gfp psb1c3, replate for religate those
start overnights for glycerol stocks
miniprep overnights of no pcr result colonies, re ligate/transform j23100 psb1c3
run pcr of last nights no results colonies that broth looks good
</P><H2><SPAN class="mw-headline" id="9.2F24.2F2010"><B>9/24/2010</B></SPAN></H2><P>run pcr results on gel, results: no band j23114+gfp psb1c3 (not even primers?)
made glycerol stocks of t7 psb1a3, t7+gfp psb1a3, and t7 psb1c3
digest (j23100+gfp no results post digestion clean up)/ligate/transform j23100 psb1c3, j23100 psb3k3, r0011 psb3k3, t7 psb4a5 (BL21), r0011+gfp psb1c3, j23100+gfp psb4a5. 20ul digestions (2ul buffer (10x), .5 pstI, .5 specific biobrick enzyme, 4ul DNA in EB, 13ul dh20)
ligate, 10ul 7.5 insert to vector goal
pcr j23113+gfp psb1c3, gel results, no bands
</P><H2><SPAN class="mw-headline" id="9.2F26.2F2010"><B>9/26/2010</B></SPAN></H2><P>transform stocks into DH5a
plate growth t7 psb4a5 only
</P><H2><SPAN class="mw-headline" id="9.2F27.2F2010"><B>9/27/2010</B></SPAN></H2><P>check plates for growth, all but R0011+gfp (psb3k3 and psb4a5) had growth
replated and religated R0011+gfp (psb3k3 and psb4a5)
made Chlor. plates
digestion/ligation/transformation j23100 (psb1c3, psb3k3), j23100+gfp (very low DNA on nanodrop) psb4a5, j23114 psb1c3, j23113+gfp (psb1c3, psb4a5), R0011 psb3k3, R0011+gfp psb1c3
pcr verify colonies all but t7 psb4a5 good
</P><H2><SPAN class="mw-headline" id="9.2F28.2F2010"><B>9/28/2010</B></SPAN></H2><P>digest 50ul, 20ul DNA, extra long (4 hours), clean up all psb's and redigest with xbal I
miniprep 1c3
ligate j23100 (psb1c3, psb3k3), j23100+gfp (very low DNA on nanodrop) psb4a5, j23114 psb1c3, j23113+gfp (psb1c3, psb4a5), R0011 psb3k3, R0011+gfp psb1c3
</P><H2><SPAN class="mw-headline" id="9.2F29.2F2010"><B>9/29/2010</B></SPAN></H2><P>digest f1, lac I and T7 promoter XP
prepare more chem comp cells
ligate T7 psb4a5 transform into bl21
transform all ligated vectors
pcr purify overnight ligation
verify DH5a stocks, see notes, need j23114+gfp psb1c3 and T7+gfp 3k3
check plates (R0011+ gfp), growth R0011+gfp psb3k3, verify tomorrow
digest and ligate all none made 
</P><H2><SPAN class="mw-headline" id="9.2F30.2F2010"><B>9/30/2010</B></SPAN></H2><P>check plates for growth
pcr verify colonies, gel results: good J23113 +gfp psb1c3 &amp; psb4a5, j23100+gfp psb4a5... all other not good, plates retained for tomorrow to pick new colonies for verification
made overnights of good gel results
clean up overnight digestion of f1, lacI and t7
transform all none working vectors into BL21 for assay
</P><H2><SPAN class="mw-headline" id="10.2F1.2F2010"><B>10/1/2010</B></SPAN></H2><P>ligate f1, lacI, t7 into psb1c3
transform f1, lacI, t7 into DH5a
plated f1, lacI, and t7 onto chorl plates
glycerol stocks of all pcr verified overnights
miniprep remainder of broth from pcr verified overnights
examine plate, some colonies on all types, 
pcr verify all colonies, gel results: 4 colonies verified, selected more colonies for overnight pcr
overnight ligation of j23114+gfp psb1c3, j23100+gfp psb1c3, R0011+gfp psb3k3
</P><H2><SPAN class="mw-headline" id="10.2F2.2F2010"><B>10/2/2010</B></SPAN></H2><P>run gel of overnight pcr, no good results
transform overnight ligation and held ligations
make glycerol stocks of overnight broths
pcr verify f1, LacI, and T7 promoter, f1 and LacI verified, t7 ran off gel
</P><H2><SPAN class="mw-headline" id="10.2F3.2F2010"><B>10/3/2010</B></SPAN></H2><P>run gel with t7 psb4a5 and t7+gfp psb3k3 pcr to check for recombants, gel results: no band at right height, all bands below 500kb (primer dimers?)
check plates for growth, all plates but 47 psb4a5 had colonies
run pcr of 4 good colonies from each plate, include t7 biobrick
run gel to verify pcr, gel results: psb1c3, both good, no good psb3k3
check quality of samples in shipping with nano dropper
glycercol stock f1 and LacI biobricks
make overnights of all good pcr/gel results
</P><H2><SPAN class="mw-headline" id="10.2F4.2F2010"><B>10/4/2010</B></SPAN></H2><P>prepare sequecne for psb1c3 parts
send of for sequencing all psb1c3 without gfp
run gel of overnight pcrs (t7's), results: no good bands 
check t7 psb4a5 plate, all colonies red, showing religation of psb4a5 without insert
start overnights for assay
</P><H2><SPAN class="mw-headline" id="10.2F5.2F2010"><B>10/5/2010</B></SPAN></H2><P>Assay (note R0011+gfp in X10 gold cell line), data on assay computer
miniprep psb1c3 and psb3k3
glycerol stock and miniprep t7 
plan f1 testing
sequence results j23113 not present as a promoter, j23114 only... which mean I swapped j23113 and j23114 and only caught j23114 causing a problem instead of fixing it :(
make kan/chlor and amp/chlor plates for f1 testing
</P><H2><SPAN class="mw-headline" id="10.2F6.2F2010"><B>10/6/2010</B></SPAN></H2><P>send for sequencing f1, t7, lacI
run mini gel with digestions of inserts (t7+gfp, R0011+gfp, J23100+gfp) results: gel t7+gfp had a band all other to low concetration
digest psb1c3 f1, j23113, and gfp 
re ligate/ transform all non-working colonies (psb1c3 priority j23100) (psb3k3, r0011+gfp, t7+gfp, j23113 psb1c3)
fill out submission spreadsheet, started...
day 1 f1 test
4 plates, 3k3 w/wo f1, 1a3 w/wo f1 into cj236 cell line
start overnights of j23100+gfp
</P><H2><SPAN class="mw-headline" id="10.2F7.2F2010"><B>10/7/2010</B></SPAN></H2><P>check plates for growth (psb3k3 R0011/T7 +gfp, psb1c3 f1+j23113, psb1c3 j23100+gfp, f1 plates), growth on all plates except control plates
pcr verify colonies, j23113 +f1 good started overnights... all other overnight PCR with more colonies
miniprep j23100+gfp (4 overnights), good nanodropper concentraion (~500)
overnight PCR J23100+gfp psb1c3, R0011+gfp psb3k3, T7+gfp psb3k3
day 2 f1 test
autoclave 12 250ml beakers(?)
3ml broth cultures with anti (kan/ amp)
add phage at approx 6 hour mark (M13K07 helper phage) (approx 6pm pst) added at 7pm due to low culture density in some samples
let grow for 1 hr to allow for phage infection
add 1ml of broth to 40ml kan or kan+amp LB in 250ml flask for overnight growth place in 37C room on shaker for overnight growth
</P><P>sequence results: f1 and lacI did not pass inspection, T7 good, note all sequences should t7, resending f1 and lacI after digest and gel to verify bands 
</P><H2><SPAN class="mw-headline" id="10.2F8.2F2010"><B>10/8/2010</B></SPAN></H2><P>fill out submission spreadsheet
miniprep f1, LacI, j23113+gfp 
send in f1, j23113+gfp and LacI for sequencing
run gel on overnight PCR's: j23100+gfp psb1c3 good, grow 3 overnights 
day 3 f1 test
ODs: psb1a3 .059, .088, .590
	f1 1a3 .115, .574, .357
	psb3k3 .956, 1.021, .927
	f1 3k3 1.453, 1.344, 1.515
submission party, bring munchies
</P><H2><SPAN class="mw-headline" id="10.2F10.2F2010"><B>10/10/2010</B></SPAN></H2><P>run gel of ss dna as a test, band result,#1 1a3, fa, fk looks empty (drop) #2 looks best in all types,  1a3 #3  shows some double banding, run 12 lane mini gel with #2 from each and ladder, run at 90V for 45 min for pic tomorrow
make overnights of psb3k3 (5 for high concentration) 
miniprep j23100+gfp psb1c3
</P><H2><SPAN class="mw-headline" id="10.2F11.2F2010"><B>10/11/2010</B></SPAN></H2><P>run ss dna gel, take pic if good (pic on other lab computer)
VF/VR pcr psb1c3 vectors for 4a5 transformation
digest VF/VR psb1c3 constructs
miniprep 3k3, high concentraion
digest 3k3, xp 
overnight digest of all inserts from 1c3
sequencing results, no good sequences 
</P><H2><SPAN class="mw-headline" id="10.2F12.2F2010"><B>10/12/2010</B></SPAN></H2><P>run colonies from lacI and f1 plates through colony pcr to verify inserts
start overnight of good colonies
clean up overnight digest, digestion crashed out (star activity?)
</P><H2><SPAN class="mw-headline" id="10.2F13.2F2010"><B>10/13/2010</B></SPAN></H2><P>morning (7am) fusion pcr vf/vr all 1c3
minipreped j23100+gfp psb1c3/psb3k3, biobrick f1/lac I
afternoon digest pcr's
clean up digestion
ligate all 4a5 to all inserts, psb3k3 induced with gfp
make more carb and chlor plates
send of for sequencing, f1 lac I
plate transformations (Justin)
plate gfp vectors to verify noise levels
</P><H2><SPAN class="mw-headline" id="10.2F14.2F2010"><B>10/14/2010</B></SPAN></H2><P>run digested inserts on gel to verify inserts presence, good bands with some background on all but j23100 and j23114+gfp
pcr verify inserts on non-gfp colonies, good bands on 4 and R
use illuminator to select gfp colonies
check background noise on 1c3 gfp high background noise. no gfp in 4g
read sequence results, no good results, re digesting original PCR of f1 and LacI to insert into fresh digestion of 1c3
plated all other gfp to check background count
made more carb plates
replated/ retransformed all BL21 ligations and j23100
streaked for isolation t7 gfp, r0011 gfp
</P><H2><SPAN class="mw-headline" id="10.2F15.2F2010"><B>10/15/2010</B></SPAN></H2><P>transform f1/lac I, plated
pcr verify j23100, run on gel, 4 good colonies, 1 broth made
check background level in GS plates, good most, 3k3 questionable (either small or none, no distinct)
turn overnights into gylcerol stocks/ minipreps
plates, no colonies t7 psb4a5
</P><H2><SPAN class="mw-headline" id="10.2F18.2F2010"><B>10/18/2010</B></SPAN></H2><P>pcr verify f1/ Lac I, start overnights of good colonies
transform t7 (note which electro comp. BL21 color you use: RED),  j23114+gfp psb3k3
glycerol stock overnights
send out biobrick parts (use fedEx package on desk)
</P><H2><SPAN class="mw-headline" id="10.2F19.2F2010"><B>10/19/2010</B></SPAN></H2><P>send out sequence parts, f1 &amp; lacI
replate from ligate psb4a5 j23114+gfp, psb4a5 t7+gfp, psb3k3 t7+gfp, psb3k3 j23114+gfp
pcr verify t7 psb4a5, use floresence for gfp constructs, no good colonies rg3k3 and tg 4a5, one questionable colony for tg3k3 set for overnight.  Broths replated. no T7 psb4a5 verified, run more colonies of plate tomorrow give longer extension times (2:30)
</P><H2><SPAN class="mw-headline" id="10.2F20.2F2010"><B>10/20/2010</B></SPAN></H2><P>sequence verify biobricks f1 &amp; Lac I, no verification. rePCRing from psb1a3 stock
start overnights for assay
pcr verify t7 psb4a5 from plate in 4c room
pcr verify gfp in psb4a5
miniprep psb3k3 t7+gfp
</P><H2><SPAN class="mw-headline" id="10.2F21.2F2010"><B>10/21/2010</B></SPAN></H2><P>digest, ligate, transform psb1c3/ f1 and lacI
assay induction: induced at 1530(10 minutes to fully induce all wells)
pcr verify psb4a5 j23114+gfp, one good colony set for overnight broth
</P><H2><SPAN class="mw-headline" id="10.2F22.2F2010"><B>10/22/2010</B></SPAN></H2><P>fill out fosmid registry page
get assay data point
create data tables
fill out registry
</P><H2><SPAN class="mw-headline" id="10.2F23.2F2010"><B>10/23/2010</B></SPAN></H2><P>miniprep f1/ Lac I overnights
set up shipments for registry and sequencing
</P><H2><SPAN class="mw-headline" id="10.2F25.2F2010"><B>10/25/2010</B></SPAN></H2><P>send of second shipment to registry
go to meeting
</P><DIV style="text-align:center"><P><B>←<A href="/Team:Washington/Parts" title="Team:Washington/Parts">Parts We Submitted</A></B>   <B><A href="/Team:Washington/Bibliography" title="Team:Washington/Bibliography">References</A>→</B></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Washington/Notebook">http://2010.igem.org/Team:Washington/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Washington/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Washington/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Washington/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Washington/Notebook&amp;oldid=203505" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>