"
Page
Discussion
View source
History
teams
Log in
 
Team:Washington/Notebook
From 2010.igem.org
Home
Gram(-) Therapeutic
Design
Build
Test Results
Gram(+) Therapeutic
Design
Build
Test Results
Tools Created
New Vectors
New Software
Tools Used
Software
Next-Gen Cloning
Protocols
Safety
Meet the Team
Accomplishments
Parts Submitted
Notebook
References
Gram Positive
07/06/10
Kunkel Mutagenesis
07/07/10
Stained and scanned protein gel.
BioRad Micro Column Protein Prep: Expression
Arctic Express 13C 24H no anti, Arctic Express 13C 24H kan, DE3* 22C 24H kan, Arctic Express 37C 24H Kan. (Both CapD and CapDCP)
CapD 22C kan and 37C kan grew much slower than the rest. (Takes about 3 - 4hrs vs around 1.5-2 hrs for the rest.)
One of the two actually decreased OD after 30 minutes.
Induced with IPTG after reached OD600 between .7-1.0 (CapD 22C kan and 37C reached about 1.3) and incubated at given temperatures.
07/08/10
Made glycerol stocks for Kunkel's. Put in -80C Freezer. To be shipped for sequencing.
Spin down remaining proteins(CapD Artic no Anti, CapD CP Arctic no Anti, CapD* Kan 22, CapD CP* Kan 22, CapD Kan Artic, CapD CP Kan Arctic) and place in -20C, awaiting purification.
07/09/10
Protein Purification
Lysis
Bind Protein
Numbers 6 and 7, CapD CP Anti and CapD CP 22, respectively, had lysate form in the supernatant that had been filtered through the TALON beads.
Sometimes centrifuged twice after pouring out supernatant because a significant amount had not filtered through TALON beads
Wash Protein
Sometimes centrifuged twice after pouring out supernatant because a significant amount had not filtered through TALON beads
Elution
(Note: Desired proteins have excess histidine chain that binds to TALON beads. Imidazole has higher affinity for TALON beads, so higher imidazole concentration pushes desired protein off, allowing it to fall through filter. This is why higher and higher concentrations of imidazole are used. Cleans off undesired proteins first all the way up to, hopefully, only the desired protein.)
Electrophoresis
Ran two gels. #1 has ladder in well 1. #2 has ladder in well 2. 3 wells per condition being Lysate, Supernatant, Purified, in that order. SDS 6uL+6uL of substance for mix. 10uL per well.
Gel #1
CapD Arctic No Anti, CapD Arctic Kan, CapD* Kan 22, CapD CP* Kan 37
Gel #2
CapD CP Arctic No Anti, CapD CP Arctic Kan, CapD CP* Kan 22, CapD* Kan 37
07/12/10
Nano Drop (Bradford Protein Assay)
Create normal line with known concentrations of protein (9uL Coomassie dye, 1uL protein solution) (We used .25mM, .5mM, 1mM, 2mM)
Use 1uL of each protein sample with 9uL Coomassie dye and measure with program.
Overnight Glycerol Stocks
CapD/CapD CP TB (LB?) Kan solutions with a dab of glycerol from -80C (just scrape glycerol with pipette tip and drop into TB Kan solution).
Incubate in 37C shaker overnight.
Dialysis
Put proteins into dialysis tubes. Left in cold room overnight. (Used to reduce amount of imidazole in solution.)
1. CapD Arctic no Anti
2. CapD Arctic Kan
3. CapD* Kan 22
4. CapD* Kan 37
5. CapD CP Arctic no Anti
6. CapD CP Arctic Kan
7. CapD CP* Kan 22
8. CapD CP* Kan 37
07/13/10
Nano Drop (Bradford Protein Assay)
Run Nano drop to find mg/mL post-dialysis
Enzyme Assay
Diluted small amount of each dialyzed enzyme for Assay reaction components
Ran fluorescence test with plate reader
Got weird results (low, delayed, or no activity)
Post Assay Electrophoresis
Ran gel to determine protein concentrations using 1mg/mL, .5mg/mL, .25mg/mL, .125mg/mL, .0625mg/mL, and .03125mg/mL BSA as standard.
Will enter order of wells tomorrow once paperwork is on hand
Post-assay Gel order ?
1) Ladder
2) 1mg/mL BSA
3) 0.5mg/mL BSA
4) 0.125mg/mL BSA
5) 0.25mg/mL BSA
6) 0.06mg/mL BSA
7) 0.3mg/mL BSA
8) CapD Arctic no anti
9) CapD Arctic Kan 13C
10) CapD* Kan 22C
11) CapD* Kan 37C
12) CapD CP Arctic no anti
13) CapD CP Arctic Kan 13C
14) CapD CP* Kan 22C
15) CapD CP* Kan 37C
MiniPrep
Done by David and Chris to obtain plasmids for PCR
7/14/10
PCR (Polymerase chain reaction)
Diluted Primers to 10uM (10uL of 100uM primer with 90uL of diH2O)
Created mixes using the recipe from Table 1a at [1]
Primers were CapDOS, CapNSS
Template DNA was from MiniPrep (Plasmids)
Added T7 reverse primer to both
Added Phusion DNA Polymerase last to delay reaction
Kunkel Mutagenesis Sequence
Checked sequences that came back for desired mutations using CLC Sequence viewer [2] . See Google Docs form for mutation information. [3]
Overnights
Made 6 overnights from glycerol stocks in -80C with TB Kan CapD CP*, one control, two mutant catalytic knock outs (T2A and T2V), and three mutants (F24Y, L40R, S143R)
Incubate at 37C in iGEM lab
Gel
Dialyzed proteins seemed to be very low in concentration. (BSA standard might be twice the labeled concentration on gel)
7/15/10
IPTG Induction
Pulled overnight glycerol stocks to be put into TB Kan solution
Added 1mL of culture per 50mL flask of TB Kan, 10mL of culture per 500mL flask of TB Kan
Incubated at 37C until OD600 was 0.6-0.8.
Flasks were all incubated at 37C after induction, except the control, for about 1 hour before being moved to 22C.
7/16/10
Scan Gel
Post Assay Gel scanned
Colony PCR
Boil cells with PCR with "green mix" and +T7/-T7 primers.
Run on DNA gel to see if mutation occurred (determine by band size).
Spin down cells
4000rpm for 20 minutes
Store at -20C until purification
7/17/10-7/25/10
See others for data?
7/26/10
Induction
Grew up T20S, T2V, T59N, T59Q, T59H_M61A, T59H_M61T at 37C to OD0.5-0.8
Induced with 500uL IPTG
Continue incubation for 24hrs at 22C
7/27/10
Concentrated purified proteins
Ran gel
Ran Bradford and Enzyme Assays
Spin down T20S, T2V, T59N, T59Q, T59H_M61A, T59H_M61T
7/28/10
Protein Purification
T59Q sat in centrifuge overnight at 4C, spun down in morning again.
7/29/10
Electrophoresis
Gel #1
Ladder
A CapDCP
B T2V
C T20S
D T59H_M61A
E T59H_M61T
F T59N
G T59Q
BSA 2mg/mL
BSA 1mg/mL
BSA 0.5mg/mL
BSA .25mg/mL
BSA .125mg/mL
BSA .0625mg/mL
Gel #2
Ladder
BSA 2mg/mL
BSA 1mg/mL
BSA 0.5mg/mL
1 CapD
2 CapDCP
3 CapDNSS
4 CapDOS
5 F24Y
6 L40R
7 S143R
8 S22I,T59Q
9 S57H,M61H
10 T2A
11 T2V
Notes
De-staining started at 4 pm
Used up remaining possibly "bad" gels. Gels have large blue blotch at the bottom.
Enzyme Assay
Row A=H2O
Row B=L-Glutamate
1 CapDCP
2 T2V
3 T20S
4 T59H_M61A
5 T59H_M61T
6 T59N
7 T59Q
8 diH2O
7/30/10
Kunkel Mutagenesis
1. CapDCP NSS
2. CapDCP NSS Foldit
3. CapDCP w/ Thrombin
4. F24H, R356K
5. G79A
6. M61H
7. M61N
8. S22Q, T59Q
9. T18L, T59Q_M61N, F452W
10. T20S, T59Q_M61T
11. T59Q_M61Q, F452W
12. T59S_M61H
13. H2O
8/2/10
First Kunkel's failed for unknown reasons
Kunkel Mutagenesis Attempt #2
1. CapDCP NSS
2. CapDCP NSS Foldit
3. CapDCP w/ Thrombin
4. F24H, R356K
5. G79A
6. M61H
7. M61N
8. S22Q, T59Q
9. T18L, T59Q_M61N, F452W
10. T20S, T59Q_M61T
11. T59Q_M61Q, F452W
12. T59S_M61H
13. H2O
8/3/10
Picked Kunkel's colonies and placed into plate with LB+Kan culture.
Shaking in 37C room
Scan Gels from 7/29/10
Well Order (Four per mutation)
CapDCP NSS- A1-D1
CapDCP NSS Foldit- E1-H1
CapDCP w/ Thrombin- A2-D2
F24H, R356K- E2-H2
G79A- A3-D3
M61H- E3-H3
M61N- A4-D4
S22Q, T59Q- E4-H4
T18L, T59Q_M61N, F452W- A5-D5
T20S, T59Q_M61T- E5-H5
T59Q_M61Q, F452W- A6-D6
T59S_M61H- E6-H6
8/4/10
Took Kunkel's overnights out of 37C room
Sent Glycerol Stocks for Sequencing (100uL 20% Glycerol, 100uL cells)
8/5/10
Computational Work from Home
8/6/10
Checked all Round 2 sequences. See Google Doc for mutant information.
Corrected Data from Assay based on Gel #2 protein concentrations.
8/9/10
Autoclave 15 flasks of TB (500mL)
Glycerol Stock Overnights (5mL TB+Kan)
1. CapD CP
2. CapD CP w/ Thrombin
3. CapD NSS
4. CapD CP NSS
5. CapD CP NSS Foldit
6. T2V
7. F24H, R356K
8. G79A
9. M61H
10. M61N
11. S22Q, T59Q
12. T18L, T59Q, M61N, F452W
13. T20S, T59Q, M61T
14. T59Q, M61Q, F452W
15. T59S, M61H
Remember to make glycerol stock for T2A and T2V and store in -80C
8/10/10
Grow up cells in 500mL TB+Kan
Induce with 500uL 1M IPTG once at OD 0.5-0.8
Incubate at 18C for 24 hours
1. CapD CP
2. CapD CP w/ Thrombin
3. CapD NSS
4. CapD CP NSS
5. CapD CP NSS Foldit
6. T2V
7. F24H, R356K
8. G79A
9. M61H
10. M61N
11. S22Q, T59Q
12. T18L, T59Q, M61N, F452W
13. T20S, T59Q, M61T
14. T59Q, M61Q, F452W
15. T59S, M61H
G79A had super slow growth rate, about OD 0.1 after 4 hours. T18L, T59Q, M61N, F452W and T59Q, M61Q, F452W about OD 0.3 after 4 hours. G79A completed at 7:00pm.
8/11/10
Spin down cells 4000rpm for 20 minutes. Store in -20C.
8/12/10
Full Purification Process
Concentrate Proteins
Run gel of purified/concentrated proteins
8/13/10
Interpret protein concentration from gels (scan with Baker Lab scanner)
Run enzyme/H2O fluorescence assay
CapD NSS ran for 2 possible protein concentrations. One for the bottom two bands, one for all three. Bottom two=3BC, All three=3ABC.
L-Glutamate
A1.1 (CapD CP)
A2.2 (CapD CP w/ Thrombin)
A3.3BC (CapD NSS: lower two bands)
A4.3ABC (CapD NSS: All bands)
A5.4 (CapD CP NSS)
A6.5 (CapD CP NSS Foldit)
A7.6 (T2V)
A8.7 (F24H,R356K)
A9.8 (G79A)
A10.9 (M61H)
A11.10 (M61N)
A12.11 (S22Q,T59Q)
C1.12 (T18L,T59Q,M61N,F452W)
C2.13 (T20S,T59Q,M61T)
C3.14 (T59Q,M61Q,F452W)
C4.15 (T59S,M61H)
C5.Blank
H2O
B1.1 (CapD CP)
B2.2 (CapD CP w/ Thrombin)
B3.3BC (CapD NSS: lower two bands)
B4.3ABC (CapD NSS: All bands)
B5.4 (CapD CP NSS)
B6.5 (CapD CP NSS Foldit)
B7.6 (T2V)
B8.7 (F24H,R356K)
B9.8 (G79A)
B10.9 (M61H)
B11.10 (M61N)
B12.11 (S22Q,T59Q)
D1.12 (T18L,T59Q,M61N,F452W)
D2.13 (T20S,T59Q,M61T)
D3.14 (T59Q,M61Q,F452W)
D4.15 (T59S,M61H)
D5.Blank
Ran Mass spec to determine the characteristics of second band on gel
8/16/10
Left proteins out over weekend in ice bucket.
CapD NSS and CapD CP NSS Foldit both crashed out.
Regrow CapD NSS and CapD CP NSS Foldit
Gravity columns left out over weekend.
Recharged, but ask Justin if should use again.
Autoclaved 2x 500mL TB in 2L flasks
8/17/10
Plate Chlor+Kan for incubating transformed cells
Transform CJ236 cells for creating ssDNA of CapD NSS and CapD CP NSS Foldit tomorrow
Inoculate and grow up 500mL TB+Kan cultures of CapD NSS and CapD CP NSS Foldit. Induce with 500uL IPTG at OD 0.5-0.8
Induced at 2pm. Incubating at 18C in shakers at Schief lab.
8/18/10
Spin down cells
Add phage to ssDNA prep and expand culture
8/19/10
Purified and concentrated protein (One concentrated down to 400uL, other to 800uL, instruction mismatch with protocol)[Correct: 200uL]
New columns used. Covered with parafilm since can't find caps.
Chris harvested ssDNA for CapD NSS and CapD CP NSS Foldit
8/20/10
A280 came up with CapD - 6.1mg/mL and CapDCP - 3.1mg/mL
Running gel for protein concentrations
Started staining 2:12 pm
Flash freeze CapD and CapDCP proteins in 50uL aliquots
8/23/10
Ran fluorescence readings to determine correction coefficients for enzyme assay due to product/substrate quenching
Rows
1. Product concentration at 10uM, Substrate decreasing 100uM to 0uM (Concentration cut in half each time, 12th well is 0uM)
2. 5uM Product, Substrate decreasing 100uM to 0uM (Concentration cut in half each time, 12th well is 0uM)
3. Substrate set at 100uM, Product decreasing 100uM to 0uM (Concentration cut in half each time, 12th well is 0uM)
8/24/10
Still searching for proper Michaelis-Menten profile. (Abandoned 8/23/10 attempt)
Original Excitation/Emission standard looks fine (490/520nm).
Tested 0.01 and 0.00001 mg/mL protein. First too high, second too low.
Tested serial dilution (half) enzyme from 0.01mg/mL decreasing across 12 wells. Substrate hold at 1uM.
Tested serial dilution (half) substrate 100uM decreasing across 12 wells (no blank). Hold enzyme at 0.001mg/mL.
Testing serial dilution (half) substrate 1uM decreasing across 12 wells (12 is blank). Hold enzyme at 0.0001mg/mL.
Kunkel's Mutagenesis
8/25/10
Reads
Calibration Curve (New Substrate)
Substrate start 1uM serial dilution (halves) across. Product start 0.1uM serial dilution (thirds) down. PMT High Sensitivity
Enzyme Assay (New Substrate)
6 Rows. Two CapDCP, two CapD, two no enzyme. (0.0001mg/mL, 2.3nM)
First row of each L-Glu. Second row of each H2O.
Substrate start 1uM serial dilution (halves) across. PMT High Sensitivity.
Calibration Curve (Old Substrate)
Substrate start 100uM serial dilution (halves) across. Product start 1uM serial dilution (thirds) down. PMT High Sensitivity
Enzyme Assay (Old Substrate)
3 rows. L-Glutamate in all rows. CapD 0.01mg/mL (200nM), CapD 0.0001mg/mL (2nM), and no enzyme (0nM). PMT High Sensitivity.
All Complete. Looks like good results across the board. Begin data analysis tomorrow.
8/26/10
Normalized Michaelis-Menten Data which better fit Substrate Inhibition Curves.
Gave Dr. Baker an update on results from the past month.
8/27/10
Presentation Slides
8/30/10
Dilute and Kinase Oligos
A1. CapDCPNF I83T
A2. CapDCPNF I83V
A3. CapDCPNF T2C
A4. CapDCPNF T2S
A5. CapDCPNF T18S_T20S
A6. L40A
A7. M61A
A8. M61C
A9. M61D
A10. M61G
A11. M61I
A12. M61L
B1. M61N
B2. M61S
B3. M61T
B4. M61V
B5. T20S_F24H
B6. T20S_F24Y
B7. Control
B8. Tossed
B9. T20S
B10. T59N
B11. T59S_M61S
Pipetted 3uL of 8-17 kinased oligos into eppendorf tube marked M61X
Overnights Growing at 37C
1.CapDCPNFDelta
2.F24A
3.F24H
4.F24H,T59N
5.F24Y,R305K
6.F24Y,T59N
7.F24Y,T59N,R305K
8.L40R
9.L40S
10.L40W
11.M61S
12.N23K
13.N23Q
14.T20A
15.T20C
16.T20S,T59N
17.T59M
18.T59N
19.T59S
20.CapD CP NSS Foldit (Control)
8/31/10
Kunkel's up to plate and incubate cells
1.I83T
2.I83V
3.L40A
4.T2C
5.T2S
6.T2S,T18S_T20S
7.T2S,T18S_T20S,T59S_M61S
8.T2S,T20S
9.T18S_T20S
10.T20S_F24H
11.T20S_F24Y
12.T20S_F24Y,T59N
13.T20S,M61A
14.T20S,M61C
15.T20S,M61D
16.T20S,M61G
17.T20S,M61I
18.T20S,M61L
19.T20S,M61N
20.T20S,M61S
21.T20S,M61T
22.T20S,M61V
23.T59S_M61S
24.Control (Blank)
25.Control (Only Cells)
Grow up proteins(small scale) to OD600 ~0.4 and induce with 25uL IPTG.
1.CapDCPNFDelta
2.F24A
3.F24H
4.F24H,T59N
5.F24Y,R305K
6.F24Y,T59N
7.F24Y,T59N,R305K
8.L40R
9.L40S
10.L40W
11.M61S
12.N23K
13.N23Q
14.T20A
15.T20C
16.T20S,T59N
17.T59M
18.T59N
19.T59S
20.CapD CP NSS Foldit (Control)
9/1/10
Have not tested F24W, R305A
Overnight: Kunkel's Plate shake in warm room
1.I83T: A1-D1
2.I83V: E1-H1
3.L40A: A2-D2
4.T2C: E2-H2
5.T2S: A3-D3
6.T2S,T18S_T20S: E3-H3
7.T2S,T18S_T20S,T59S_M61S: A4-D4, E12-F12
8.T2S,T20S: E4-H4
9.T18S_T20S: A5-D5
10.T20S_F24H: E5-H5
11.T20S_F24Y: A6-D6
12.T20S_F24Y,T59N: E6-H6
13.T20S,M61A: A7-D7
14.T20S,M61C: E7-H7
15.T20S,M61D: A8-D8
16.T20S,M61G: E8-H8
17.T20S,M61I: A9-D9
18.T20S,M61L: E9-H9
19.T20S,M61N: A10-D10
20.T20S,M61S: E10-H10
21.T20S,M61T: A11-D11
22.T20S,M61V: E11-H11
23.T59S_M61S: A12-D12
24.Blank: E12-H12
Spin Down Cells at 4:00pm. Place in -20C
1.CapDCPNFDelta
2.F24A
3.F24H
4.F24H,T59N
5.F24Y,R305K
6.F24Y,T59N
7.F24Y,T59N,R305K
8.L40R
9.L40S
10.L40W
11.M61S
12.N23K
13.N23Q
14.T20A
15.T20C
16.T20S,T59N
17.T59M
18.T59N
19.T59S
20.CapD CP NSS Foldit (Control)
9/2/10
Made Glycerol Stocks for Kunkel  Set 2 Round 3 and sent in for sequencing
Small Scale Purification of Kunkel Set 2 Round 2
Growing overnight of CapD CP NSS Foldit for creating glycerol stock and miniprepping DNA for USAMRIID
9/3/10
Enzyme Assay
Row A and B=Transpeptidation
Row C and D=Hydrolysis
1.CapDCPNFDelta-A1,C1
2.F24A-A2,C2
3.F24H-A3,C3
4.F24H,T59N-A4,C4
5.F24Y,R305K-A5,C5
6.F24Y,T59N-A6,C6
7.F24Y,T59N,R305K-A7,C7
8.L40R-A8,C8
9.L40S-A9,C9
10.L40W-A10,C10
11.M61S-A11,C11
12.N23K-A12,C12
13.N23Q-B1,D1
14.T20A-B2,D2
15.T20C-B3,D3
16.T20S,T59N-B4,D4
17.T59M-B5,D5
18.T59N-B6,D6
19.T59S-B7,D7
20.CapD CP NSS Foldit (Control)-B8,D8
21.Blank-B9,D9
Make Glycerol Stock and MiniPrep DNA of CapD CP for USAMRIID
Run protein gel to confirm concentrations from Nanodrop
9/4/10
Mass Spec confirms that CapDCPNSSFoldit really is awesome.
9/7/10
Overnights in TB+Kan at 37C in back of small shaker in front of storage room.
Control.CapDCPNSSFoldit
1.I83T
2.I83V
3.L40A
4.T2C
5.T2S
6.T2S,T18S_T20S
7.T2S,T18S_T20S,T59S_M61S
8.T2S,T20S
9.T18S_T20S
10.T20S_F24H
11.T20S_F24Y
12.T20S_F24Y,T59N
13.T20S,M61A
14.T20S,M61C
15.T20S,M61D
16.T20S,M61G
17.T20S,M61L
18.T20S,M61N
19.T20S,M61S
20.T20S,M61T
21.T20S,M61V
22.T20S,T59S_M61S
9/8/10
Chris to inoculate ~9am. Induce at OD600 0.4-0.6 w/ 25uL IPTG. Induced at 12:20pm.
Control.CapDCPNSSFoldit
1.I83T
2.I83V
3.L40A
4.T2C
5.T2S
6.T2S,T18S_T20S
7.T2S,T18S_T20S,T59S_M61S
8.T2S,T20S
9.T18S_T20S
10.T20S_F24H
11.T20S_F24Y
12.T20S_F24Y,T59N
13.T20S,M61A
14.T20S,M61C
15.T20S,M61D
16.T20S,M61G
17.T20S,M61L
18.T20S,M61N
19.T20S,M61S
20.T20S,M61T
21.T20S,M61V
22.T20S,T59S_M61S
9/9/10
Spin Down Cells at 12:30pm
Purification completed ~6:45pm
1.I83T
2.I83V
3.L40A
4.T2C
5.T2S
6.T2S,T18S_T20S
7.T2S,T18S_T20S,T59S_M61S
8.T2S,T20S
9.T18S_T20S
10.T20S_F24H
11.T20S_F24Y
12.T20S_F24Y,T59N
13.T20S,M61A
14.T20S,M61C
15.T20S,M61D
16.T20S,M61G
17.T20S,M61L
18.T20S,M61N
19.T20S,M61S
20.T20S,M61T
21.T20S,M61V
22.T20S,T59S_M61S
23.CapDCPNSSFoldit
9/10/10
Enzyme Assay
Set 2 Round 3
Row A & B = L-Glu
Row C & D = H2O
Nanodrop for protein concentration
Proteins Diluted with Elution Buffer
L-Glutamate
A1.I83T
A2.I83V
A3.L40A
A4.T2C
A5.T2S
A6.T2S,T18S_T20S
A7.T2S,T18S_T20S,T59S_M61S
A8.T2S,T20S
A9.T18S_T20S
A10.T20S_F24H
A11.T20S_F24Y
A12.T20S_F24Y,T59N
B1.T20S,M61A
B2.T20S,M61C
B3.T20S,M61D
B4.T20S,M61G
B5.T20S,M61L
B6.T20S,M61N
B7.T20S,M61S
B8.T20S,M61T
B9.T20S,M61V
B10.T20S,T59S_M61S
B11.CapDCPNSSFoldit
B12.Blank
H2O
C1.I83T
C2.I83V
C3.L40A
C4.T2C
C5.T2S
C6.T2S,T18S_T20S
C7.T2S,T18S_T20S,T59S_M61S
C8.T2S,T20S
C9.T18S_T20S
C10.T20S_F24H
C11.T20S_F24Y
C12.T20S_F24Y,T59N
D1.T20S,M61A
D2.T20S,M61C
D3.T20S,M61D
D4.T20S,M61G
D5.T20S,M61L
D6.T20S,M61N
D7.T20S,M61S
D8.T20S,M61T
D9.T20S,M61V
D10.T20S,T59S_M61S
D11.CapDCPNSSFoldit
D12.Blank
9/13/10
Designed new set and ordered Oligos
1.F24W
2.F60E
3.F60Q
4.F60W
5.L40D
6.L40E
7.L40F
8.L40H
9.L40K
10.L40Q
11.L40Y
12.N81E
13.N81Q
14.T59D_M61S
15.T59K
16.T59K_M61S
17.T59N_M61S
18.T59R
19.T59I
9/14/10
Wiki/Slides
9/15/10
Kunkel's Mutagenesis
F24H, L40R, T59D_M61S
F24H, L40R, T59N_M61S
F24H, L40R, T59S_M61S
F24H, T59R
F24W
F24Y, L40R
F24Y, L40R, T59N
F60W
T20S, F60W
T20S, L40D
T20S, L40E
T20S, L40F
T20S, L40H
T20S, L40K
T20S, L40Q
T20S, L40R
T20S, L40R, F60E
T20S, L40R, F60Q
T20S, L40R, N81E
T20S, L40R, N81Q
T20S, L40R, T59K
T20S, L40R, T59K_M61S
T20S, L40R, T59N
T20S, L40W
T20S, L40Y
T20S, T59I
T20S, T59K
T20S, T59K_M61S
T20S, T59R
Control
11, 12, 28 failed. High ion concentration did not allow for electroporation. Dialysis ligation fail? Continuing to plate and grow, but possibly 1/10 success rate.
Did not recover after electroporation. Plated immediately.
9/16/10
Pick Colonies and grow on plate shaker in 37C No colonies on 28
Plate A
A1-D1. F24H, L40R, T59D_M61S
E1-H1. F24H, L40R, T59N_M61S
A2-D2. F24H, L40R, T59S_M61S
E2-H2. F24H, T59R
A3-D3. F24W
E3-H3. F24Y, L40R
A4-D4. F24Y, L40R, T59N
E4-H4. F60W
A5-D5. T20S, F60W
E5-H5. T20S, L40D
A6-D6. T20S, L40F
E6-H6. T20S, L40H
A7-D7. T20S, L40K
E7-H7. T20S, L40Q
A8-D8. T20S, L40R
E8-H8. T20S, L40R, F60E
A9-D9. T20S, L40R, F60Q
E9-H9. T20S, L40R, N81E
A10-D10. T20S, L40R, N81Q
E10-H10. T20S, L40R, T59K
A11-D11. T20S, L40R, T59K_M61S
E11-H11. T20S, L40R, T59N
A12-D12. T20S, L40W
E12,F12. T20S, L40E
Plate B
A1-D1. T20S, L40Y
E1-H1. T20S, T59I
A2-D2. T20S, T59K
E2-H2. T20S, T59R
9/17/10
Store in -80C. Send for Sequencing Monday.
100uL 20% Glycerol+100uL Culture
Plate A & Plate B
9/21/10
Autoclaved 30 250mL flasks and 2L TB (No Kan).
9/22/10
Grrr...angry at FedEx for being slow.
9/23/10
Genewiz determined glycerol stocks cross contaminated due to sitting in Memphis, Tennessee for two days...OR possibly bad seal.
Resending glycerol stocks (20uL per stock) for sequencing. Hope for sequencing results by Friday.
Looks like Plate B is fine, did not send again. Will see sequencing tomorrow.
9/24/10
2L flask of TB is tempting me to do wet lab work...
9/27/10
Overnights started 3pm
DNSS and DCPNSSFoldit have 2mL in order to take 1mL for miniprep and PCR
F24H
F24H, L40R, T59D_M61S
F24H, L40R, T59N_M61S
F24H, L40R, T59S_M61S
F24H, T59R
F24W
F24Y, L40R
F24Y, L40R, T59N
F60W
L40R
T20S, L40D
T20S, L40E
T20S, L40F
T20S, L40H
T20S, L40K
T20S, L40Q
T20S, L40R
T20S, L40R, F60E
T20S, L40R, F60Q
T20S, L40R, N81E
T20S, L40R, N81Q
T20S, L40R, T59K
T20S, L40R, T59K_M61S
CapDNSS
CapDCPNSSFoldit
9/28/10
Plasmid for pSB1C3, CapD, and DCP obtained
PCRed CapD and DCP
PCR Purification for CapD and DCP purified linear double-stranded DNA.
All stored in -20C
Inoculate and induce. (OD600=0.4-0.6) Start growing at 18C 5:00pm.
F24H
F24H, L40R, T59D_M61S
F24H, L40R, T59N_M61S
F24H, L40R, T59S_M61S
F24H, T59R
F24W
F24Y, L40R
F24Y, L40R, T59N
F60W
L40R
T20S, L40D
T20S, L40E
T20S, L40F
T20S, L40H
T20S, L40K
T20S, L40Q
T20S, L40R
T20S, L40R, F60E
T20S, L40R, F60Q
T20S, L40R, N81E
T20S, L40R, N81Q
T20S, L40R, T59K
T20S, L40R, T59K_M61S
CapDNSS
CapDCPNSSFoldit
9/29/10
Spin down and Store at -20C. 21 lost lid (possible contamination)
F24H
F24H, L40R, T59D_M61S
F24H, L40R, T59N_M61S
F24H, L40R, T59S_M61S
F24H, T59R
F24W
F24Y, L40R
F24Y, L40R, T59N
F60W
L40R
T20S, L40D
T20S, L40E
T20S, L40F
T20S, L40H
T20S, L40K
T20S, L40Q
T20S, L40R
T20S, L40R, F60E
T20S, L40R, F60Q
T20S, L40R, N81E
T20S, L40R, N81Q
T20S, L40R, T59K
T20S, L40R, T59K_M61S
CapDNSS
CapDCPNSSFoldit
9/30/10
Possibly lots of proteins have crashed out...I guess they were much less stable this time around for some reason?
Continue with only pipetting out remaining supernatent. Left possibly crashed out protein...
Digested CapD and CapDCP
Attempted ligation, but did not use digested pSB1C3 vector, so no possibility significant yield.
10/1/10
4,5,6,7,10,11,12,14,15,17,20 have weird curves on nanodrop
Run enzyme assay
Plated CapD and CapDCP transformed into pSB1C3 w/ RFP
Protein Concentrations(mg/mL) Diluting them all is ridiculous...
F24H - 1.785
F24H, L40R, T59D_M61S - 2.064
F24H, L40R, T59N_M61S - 2.636
F24H, L40R, T59S_M61S - 0.2679
F24H, T59R - 1.43
F24W - 0.03752
F24Y, L40R - 0.1868
F24Y, L40R, T59N - 2.029
F60W - 4.399
L40R - 0.2323
T20S, L40D - 0.1174
T20S, L40E - -0.08503
T20S, L40F - 1.801
T20S, L40H - 0.4182
T20S, L40K - 0.6324
T20S, L40Q - 1.513
T20S, L40R - 0.3727
T20S, L40R, F60E - 1.762
T20S, L40R, F60Q - 2.578
T20S, L40R, N81E - 0.4964
T20S, L40R, N81Q - 1.767
T20S, L40R, T59K - 1.960
T20S, L40R, T59K_M61S - 1.818
CapDNSS - 2.901
CapDCPNSSFoldit - 2.380
Flash froze all proteins. A blue alpha (lower case greek) denotes them.
10/4/10
Colony PCR on CapD, DCP, and control plate
Looks like plates possibly mislabeled? Something messed up along the way since CapD plate has very few colonies, Control has as many as DCP.
No bands turned up on gel. Redoing Digestion, Ligation, PCR process.
Overnights
F24H
F24H, L40R, T59N_M61S
F24Y, L40R, T59N
CapDNSS
CapDCPNSSFoldit
10/5/10
Michaelis Menten Curves for
F24H
F24H, L40R, T59N_M61S
F24Y, L40R, T59N
CapDNSS
CapDCPNSSFoldit
Transpeptidation first plate
Hydrolysis second plate
0.0001mg/mL of enzyme
Substrate start 1uM serial dilution (halves) across. PMT High Sensitivity.
Weird curve for transpeptidation
PCR amplify insert annealing at 55C, 60C, and 65C
10/6/10
Run gel of insert to see which annealing temperature is optimal
Failed THRICE...hopefully I get it right this time
DNA gel showed no bands for any of the samples. No inserts PCRed out?
DNA gel=
50mL TAE
0.5g Agarose
Microwave max power for 1 minute
10uL EtBr
Pour into cartridge
Place combs in
Let sit for 20 minutes
Load samples
Run for 15-30 minutes
Look at under UV
10/7/10
Michaelis-Menten Curves for
F24H
F24H, L40R, T59N_M61S
F24Y, L40R, T59N
CapDNSS
CapDCPNSSFoldit
Transpeptidation first plate
Hydrolysis second plate
0.0001mg/mL of enzyme
Substrate start 1uM serial dilution (halves) across. PMT High Sensitivity.
PCR again with anneal at 55, 60, and 65. Forgot dNTPs first time
F24H
F24H, L40R, T59N_M61S
F24Y, L40R, T59N
CapDNSS
CapDCPNSSFoldit
CapDCPNSSFoldit (T7 Forward and Reverse Primers)
60C anneal seems to be best.
CapDNSS failed. Assuming forgot forward primer. Redo at 60C.
Assay test of BSA dilution showed ~1000RFU better than water dilution
Run PBS dilution vs H2O dilution vs BSA dilution vs BSA control tomorrow
Run on gel to see what worked. Loaded 6uL of samples. Messed up ladder, so added 10uL ladder and 6uL loading dye.
Think about abandoning F24Y, L40R, T59N due to instability/inactivity.
T7 Forward/Reverse DCP tossed since Justin found better cloning idea using RFP
10/8/10
Got clean curves, but lower activity. Regrow proteins and try again. Can use for iGEM, but better curves are better.
10/11/10
Colony PCR. We'll see how it goes...
Growing up new cells overnight 2mL TB+Kan
F24H
L40R
F24H, L40R, T59N_M61S
T2V
CapDNSS
CapDCPNSSFoldit
10/12/10
Inoculate at 9am. Induce with 25uL IPTG OD600 at 0.3-0.6. Grow at 18C for 24 hours. Can Spin down after 11am.
F24H
L40R
F24H, L40R, T59N_M61S
T2V
CapDNSS
CapDCPNSSFoldit
Miniprep plasmids. (Eluted into wrong tubes once. Luckily grew 2mL...Second attempt running)
F24H
F24H, L40R, T59N_M61S
F24Y, L40R, T59N
CapDNSS
CapDCPNSSFoldit
10/13/10
Spun down cells
Cells not suspended very well.
Purified Protein
F24H - 2.554
F24H,L40R,T59N_M61S - 2.591
L40R - 2.771
T2V - 0.3800
CapDNSS - 2.509
CapDCPNSSFoldit - 3.554
10/14/10
MM Assay Data (Transpeptidation, then hydrolysis)
F24H
F24H,L40R,T59N_M61S
L40R
T2V
CapDNSS
CapDCPNSSFoldit
Blank
HEPES not pHed. Must pH to ~8 before use.
10/15/10
MM Assay failed again
10/18/10
Soluble protein not dead
Recalibrate protein concentration for PMT=Medium. (Check Tomorrow since fire alarms are annoying...)
Start Substrate at 1uM. (10x is 10uM)
Determine if Master Mixes or Individual Wells is better...Master Mixes gets clean, consistent data, Individual Wells is easier...
10/19/10
Complete Substrate/Product Curve.
Substrate 10uM (Serial dilution in halves across)
Product 1uM (Serial dilution thirds down)
Data Works
Overnights to Re-Express
F24H
F24H,L40R,T59N_M61S
CapDNSS
CapDCPNSSFoldit
10/20/10
Overnights (F24H did not grow so re-express)
F24H
F24H,L40R,T59N_M61S
L40R
CapDNSS
CapDCPNSSFoldit
10/21/10
Induced with 25uL IPTG at 3pm (OD600 of 0.3)
F24H
F24H,L40R,T59N_M61S
L40R
CapDNSS
CapDCPNSSFoldit
10/22/10
Spin down and start purification by 5?
Vectors
6/15/10
PCR out Lac I and F1 origin from pet29b vector
run following PCR reaction  0.5ul forward primer 0.5ul reverse primer 0.5ul 25mM dNTP 0.5ul pet29 plasmid  0.5ul fusion DNA polymerase 10ul 5x polymerase buffer 37.5ul dH20
PCR program:  · Start: 98 °C for 2 min. (melt)  · Cycle 98 °C for 0.5 min (melt)  · 60 °C for 0.5 min. (anneal)  · 72 °C for 0.5 min.  · No. of Cycles: 29 · End: keep at 4 °C forever
Check for PCR product  run product with 100bp ladder on 1.5% agarose gel load 5ul sample (or ladder) and 1ul 6x loading dye into gel
Miniprep psb1A3 plasmids for day 2, take six 2ml LB broth overnight growths of E. Coli with psb1A3  pellet cells in centrifuge, pour of supernatant fluid.  Use instructions from miniprep kit  store purified psb1A3 plasmids in IGEM vector box for day 2 use
6/16/10
PCR gradients for F1 and LacI 10 ul reactions  variables: temp (58, 65, 72) for extension and DMSO (0%, 5%, and 10%) run on 1.5% arganose gel with 1kb ladder and reactions from day 1
results of agarose gel F1 bands on all 72C and 65C temps, 5%DMSO at 58C, Day 1 PCR LacI all 5% DMSO, 10% DMSO 58C and 65C, Day 1 PCR purified DNA placed in IGEM vector box -20C
6/17/10
digest PCR products to make sticky ends double digestion with EcoRI and PstI double digestion of PSB1A3 w/ EcoRI and PstI in NEB buffer 3 with BSA at 37C for 1 hours. PCR purify to remove enzymes
digestions  5ul NEB buffer 3  2ul each enzyme (EcoRI and PstI)  15ul PCR purified DNA (day 1 and 1.5) or plasmid DNA  .5ul BSA  25.5ul dH20  50ul total volume
ligate PCR products into plasmids (PSB1A3)  10ul T4 buffer  30ul PCR purified F1 digest or LacI digest   15ul PCR purified psb1A3 digest   43.5ul dH20  1.5ul T4
held at rm temp. 1hour for ligation
place ligated plasmids into E. Coli for expression   transformed into E. Coli x-10 gold series competent cells  2 plates Carbacillin held at 37C
plate E. Coli on Ampicillin to as a selection (note death gene will ensure non-ligated products don't contaminate sample)
6/22/10
Isolate colonies growing on plates, place part in storage. PCR test for inserts (scrap 8 colonies from each plate) suspend colonies in 10ul dH20,  run PCR with 1ul VF, 1ul VR (biobrick vectors), 5ul premixed 2x Master mix (taq polymerase, taq buffer, dNTP), and 3ul colonies suspension.
Run PCR on 1% arganose gel
Make suspension of vector expressing E. coli, use for further experiments.
2 colonies of F1 and LacI (each) that showed recomination in gel placed in 2ml TB with Carbicillian @37C shaker
6/23/10
minipreped F1 and LacI broths for sequencing, sent into sequencing at genewiz
6/24/10
PCR out constitutive promoters (Bba_J23100, J23114, J23113) and Lac regulated promoter (R0011) use ordered primer and VF2 primer.
Mixed 0.5ul specific primer 0.5ul VF2 primer 0.5ul 25mM dNTP 0.5ul DNA from distribution plate  0.5ul fusion DNA polymerase 10ul 5x polymerase buffer 37.5ul dH20
(note master mix of H20, buffer and dNTP was used)
PCR was ran as follows
PCR program:  · Start: 98 °C for 2 min. (melt)  · Cycle 98 °C for 0.5 min (melt)  · 58 °C for 0.2 min. (anneal)  · 72 °C for 0.5 min.  · No. of Cycles: 29 · End: keep at 4 °C forever
run PCR results on 2% agarose gel to check for products.
Products seen on lac I regualer (R0011) and J23113, no product observed on J23100 and J23114, will wait for results of transformation to decide if rePCRing or requesting new DNA from IGEM
Transformed DNA from plate into x10 gold series E. Coli (15ul cells to 1ul DNA) grown in TB for 1 hour on shaker @ 37C plated onto antibiotic plates (5 amp/ 1 Kan)
Sequencing results of Day 3.5, both colonies show 100% homology with expected DNA from ape sequence
6/25/10
result of plates, growth on all constitutive promoters and on psb4A5, no growth on psb1k3 and R0011 (lac regulated promoter).
Growth plates were scraped and inoculated into 2.5ml TB for overnight. These broths will be used for a glycerol stock and miniprep.
Plates with no growth, the psb1k3 and R0011 were re-transformed using electrophoresis and plated onto the appropriate antibiotic plate (amp and Kan)
Also from the growth plates suspension were made of J23100 and J23114 which were PCRed (with 5% DMSO, all other same as day 5 protocol) and ran on a 2% agarose gel. Strong band from the J23100, weak band from the J23114.
-over weekened broths were spun down into pellets and glycerol stocks of J23100, J23113, J23114, psb4A5 was made.
6/28/10
PCR purify PCR results and check DNA concentration using Nanodropper, low concentrations on PCR products, will proceed and adjust concentration so there is a 10 to 1 insert to vector concentration for ligations.
Isolate plasmid with F1 origin and Lac I from Day 4 (minipreps, done prior for F1 and LacI, for constitutive promoters done today) Digest plasmid with SpeI and PstI (F1 with Buffer 2 and BSA) Lac I and constitutive promoters XbalI and PstI. (Buffer 2 with BSA) digestions analyzed on nanodropper,
Ligate the digestions. Overnight ligation at 16C 10ul ligation using  2ul vector 6ul insert (or dH20 for control) 1ul buffer 1ul T4 ligase
6/29/10
finished ligation  Chemically transformed into gold series x10 cells Express with 30ul on Ampicillin, grown at 37c overnight
6/30/10
growth plates, no growth on control or J23100 plates, replated with 200ul on amp/ overnight @37C
Check for recombinants by PCRing using VF2 and VR in Taq
run on 1% agarose gel
make a broth(s) of Day 6 E. coli colonies that show signs of recombination for miniprep tommorrow, all but 1 lacI +F1 showed recombination, made broths with carb, set @37C for overnights.
7/1/10
growth plates, 1 colony on control, colonies on J23100, PCRing with biobrick primers (VF2 and VR) to run on 1% agarose gel and make broths of colonies suspensions.
Minipreped  broths from J23113, J23114, and LacI constructs with F1  reinvigorated broth with 1ml TB and stored @4C for latter glycerol stocks
Kineased T7 promoter using  · i. 9uL Kinase Buffer · ii. 3uL of 10mM ATP · iii. 3uL T4 Polynucleotide Kinase · iv. 52uL ddiH2O · v. 21uL Olgio incubated @37C for 1 hour then set at 95 with a slow cooling to room temp over 1 hour (PCR protocol, 95 then -.5 per 30 seconds) Ran results on 1% agarose gel. Results showed distinct band around 100bp with a smear below
7/2/10
Minipreped broth of J23100
7/6/10
digest, ligate and inserted into X10 gold cells T7 digest: digest psb1A3 with E and S (T7 ready to ligate from PCR) ? ligate:  plate onto Carbicillin plate
7/7/10
digest psb1A3 with F1+lacI genes, at S/P (50ul reaction) digest R0011 with X/P (50ul reaction)
pcr out the T7 from the X10 gold cells grown up on day 9 pcr out GFP (E0040) from Igem 2009 stock digest Pcr T7 with X/P
purified GFP/ T7 contaminated during purification*
ran on 1% agarose gel, banding around 700 bp for GFP, and banding around 100bp for colony 1, no dna in colony 2 or 3
overnight ligate F1+lacI with R0011/T7 hold at 16C
between digest and ligations made more Carbicillin plates
examine sequencing results of J23100/114/113.  113 good stored in Glycerol stock, re-transformed J23100/114 from earlier ligation held at -20C, plated on Carbicillin for overnight growth
7/8/10
plate results F1+ j23100/j23114: no colonies on J23114 redo digestion/ligation colonies on J23100, Pcr out (on 9 july, stored at 4C) re-plated j23114 using remaing broth from recovery media transform overnights into X10 gold series and grow on Carbicillin plates miniprep T7 in psb1A3, retain some broth for glycerol stock(?)
7/9/10
growth plate results: R0011 w/ F1&LacI shows colonies, 1 colony on J23114, no colonies on T7 recombination.  Plates retained at 4C
PCR T7 and GFP, purify after (using VF2 and VR, phusion) (purfied and nanodroped afterwords, looks good) pcr chosen colonies to check for recombination (use VF2 and specific promoter, green Taq)
run on gel (gel results, J23100 1,2,4, J23114, R0011 1,3,4) Broths made of J23100 1,2, J23114, R0011 1,4. Placed at 4C send for sequencing?
7/26/10
T7 digestion, ligation, insertion using T7, LacI+F1 using T7 pcr purified VR/VF digest XP , LacI+F1 digest SP plated on Carbicillin plates at 37C
broths from J23100, J23114, R0011 reinvigorated and placed in 37C water bath for growth
7/27/10
Miniprep broths from j23100, j23114, R0011 check T7 plate for growth, select 4 colonies to check for inserts via PCR with Taq mix and gel run. Gel ran with Lac I +F1, pictures did not turn out well (need better camera/ backlight) no dicernable difference between control (F1+ lacI) and T7, all 4 showed strong bands.  Colonies 1 and 2 used for making broths held overnight at 37C in shaker waterbath.
7/28/10
Nano drop Minipreps from 27 july Miniprep 2 T7 broths. Send j23100, j23114, R0011 and T7 in for sequencing.
7/29/10
sequencing results, J23100 good, R0011 good, T7 questionable (send in other samples) j23114 bad, redo
7/30/10
ligate F1 origin in psb1A3 with j23114 using earlier digestions remains plated out portion of earlier j23114 broth
8/2/10
PCRed cells ran on 1% agarose gel at 155V for 1 hour.  No good T7 (used both ends of PCR primers, causing them to bind to each other and get no results) J23114 looks good on 7 of 8 colonies ran. 2 pick (1 from old 1 ligation, 1 from new ligation) for overnight growth
8/3/10
minipreped 6 colonies for sequencing, 4 T7 and 2 j23114 colonies sequencing results show j23114 cross contaminated with j23113, earlier results confirm this.  T7 checking sequence incorrect, T7 results from earlier good and used for further vector construction.
8/9/10
digest, ligate, transformation, and plate GFP into vectors transform stock vectors and plate check on status of other plasmid, other plasmids in stock
8/10/10
PCR GFP colonies to check for transformation (VF/VR) overnights of good colonies, (2 sets, TB and LB) (+ and - GFP)
8/11/10
make glycerol stocks of + - minus (TB) expression psb1A3 (from LB) (see note from Justin), expression stopped due to no GFP signs of Constitutive promoter (j23100) during Miniprep.   Minipreped +/ - GFP psb1A3 for latter
8/12/10
send in J23114 for sequencing. PCR GFP
8/13/10
digest, ligate and plate (grow at room temp over weekend) GFP vectors in psb1A3 sequencing result for J23114, good... found earlier flip in J23114 and J23113 (first sequencing)... which means I tossed out the vector I needed :(
8/16/10
ligate, digest, transformation J23114, plated after 45 minutes so could get to meeting in time, remainder of broth left in shaker overnight.  pick colonies for PCR verification with GFP, J23100 no good pcr results (solid bands, but too low to have GFP included) included anyhow to see if any Green would show up.  Other PCR's had expected results (bands below F1 + lacI for J23113, above for T7 and R0011) ladder shows up week even with doubling the amount normally included... overnights GFP (TB and LB) overnight colonies from plate with no GFP (LB)
8/17/10
pick colony from J23114 and PCR verify/ start overnights Glycerol stock GFP transformed vectors, J23100 not stocked due to no green pigment shown when spun down, will re-ligate on the 18th (tomorrow)
8/18/10
Miniprep overnights and send to sequencing (J23114) digestions, ligate, transform J23100 and J23114 with GFP (multiple colonies so you have best for sequencing results the next day)
8/19/10
PCR verification of colonies, start overnights (TB for stocks) sequencing results: j23114 good (2 and 3)
8/20/10
make glycerol stocks of J23114
8/31/10
grow broths from glycerol stock (in LB) meet with Justin to discuss plans
9/1/10
expresssion assay notes on assay: T7 used glycerol stocks 10ul into 1ml TB all others 100ul LB overnight into 1ml TB, no stock of j23114 so no j24113 overnight or non GFP for data points. Overnights grown for 24h. +/- 10 minute, grown for 4h at 37C shaker IPTG added at 2.5h to T7 and R0011, low growth of broths, re-run assay tomorrow.  Check for .1 colony density (OD 600, clear plate, 100ul broth) before induction.  PCR out vectors
9/2/2010
assays psb1a3 overnights grown in 1ml LB w/Carbicillin, grown for approx. 16h., 50ul transferred to 2ml TB w/Carb.  checked for colony density in broth by reading OD 600 at 3h.  all constitive above .1 OD, R0011 GFP low (.068) and GFP control low.  will induce with IPTG in 30min. and wait 4 hours for final read.   verify pcr: all but j23100 with GFP showed bands at appropriate positions. nano drop of j23100 with GFP is at 16ng/ul    re PCRing J23100 + GFP, nanodrop had around 34ng/ul made plates (carb/chlor)
9/7/2010
digest, ligate, transform vectors (3k3, 4a5), plates of transformation on appropriate media (Kan/ Carb) made note: possible contamination of T7 +GFP in 4A5 and T7 in 3K3 during purification process remainer of electro competent cells placed in 4 shelf up, 1 drawer, 2 row upper  box grow k3k overnight for stock
9/8/2010
plasmid purfication of 3k3 overnight  start overnights of 1C3 verify colonies by PCR (done at 1600), 2 gels due to amount of DNA ran (1 hour at 100V 155amps) note on 4A5 j23114 and j23113 reversed for 4A5 colonies/pcr (labled as 3c and 2c respectivley) MM 2 used, 1 VF + GFP reverse (for all Gfps) other VF  replated from broths plates that did not grow,  start overnights growths of good colonies for glycerol/ plasmid stocks note: wrong t7 used and gel ran to long, broths with ? area unclear via PCR results, re-run from purified plasmids tomorrow
9/9/2010
glycerol stocks of overnights minipreps of overnights PCR overnights in question (T7 in 3k3 and 4a5 and j23100 + Gfp in 4a5) run 1% agarose gel with 1kb ladder to verify vectors (t7 3k3 good, t74a5 and j23100+gfp in 4a5 plates put back in 37c overnight for new colonies selecting) make plates of Kan digest 1c3 ligate 1c3 to all, former non good with old digestion. chemical(j23100+gfp 3k3, j23113 +gfp 3k3, r0011 3k3, r0011 +gfp 3k3,  j23114 4a5) electric (t7 +gfp 3k3, t7 for 1c3 and 1a3) transform T7 1a3, all 1c3, old ligations of non turn outs,  plate on appropriate antibiotics
9/10/2010
plates results: no colonies; t7 psb1c3, t7+gfp psb3k3, t7 psb1a3, j23100+gfp psb3k3, R0011 psb3k3.   replated no colonies from broth  lawn of colonies (replate from broth at 100 dillution; R0011+gfp psb1c3, R0011 psb1c3, j23113+gfp psb1c3, j23100 psb1c3,  j23113 psb1c3, j23114 pab1c3, j23100+gfp psb1c3, j23114+gfp psb1c3, t7+gfp psb1a3)  good colonies(pcr); R0011+gfp psb3k3, j23113+gfp psb3k3, t7+gfp psb1c3, j23114 psb4a5   earlier plates new pcr; t7 psb4a5, j23100+gfp psb4a5  PCR with taq colonies to verify inserts (2 colonies from each of 6 plates, 11 total) pcr error, pcr overhead not heating caused no distingishable dna to be visible on gel. no colonies held make more glycerol stocks (autoclave) all plates placed in 4c walk in, remove for growth when you return to the lab
9/20/2010
Clean- up borrowed lab space PCR colonies from held plates (R0011+gfp psb3k3, j23113+gfp psb3k3, t7+gfp psb1c3, j23114 psb4a5, t7+gfp psb3k3   earlier plates new pcr; t7 psb4a5, j23100+gfp psb4a5 ) ran on 1% agarose gel, no psb4a5 results
9/21/2010
re-do t7 psb4a5, j23114 psb4a5, j23100+gfp psb4a5, j23100 psb3k3, r0011 psb3k3, all 1c3 into dh5a cell line pick 2 colonies from each good plate and pcr with taq for gel run to verify insert, no pcr results (loss of some product from machine heating problems) for t7 psb1c3 or t7+gfp psb1a3... re do tomorrow in middle area... made kan, carb, and chorl plates glycerol stocked t7+gfp psb3k3, t7+gfp psb1c3, R0011+gfp psb3k3 ligate and transformed t7 psb4a5, j23114 psb4a5, j23100+gfp psb4a5, j23100 psb3k3, r0011 psb3k3, all 1c3 into dh5a
9/22/2010
check for growth on plates colony PCR 2 colonies from each good plate and held t7 psb1c3 and t7+gfp psb1a3, colonies on j23114+gfp, t7 psb1a3, j23100 psb1c3, and j23100+ gfp psb1c3 replate from broth all non-growth plates make more Chlor. and Carb. plates run PCR on gel, no gel results, not even a ladder... eth bromide going bad? start overnights from glycerol stocks of all non psb1a3 X10 gold stocks to transform into DH5a transform t7 psb1c3 from DH5a to BL21 cell line for expression assay(2nd plate set) transform psb1a3 stocks into DH5a cell line start overnights for digestion tomorrow of all cell pcr colonies (11 total)
9/23/2010
check plates for growth, no growth, j23100 psb3k3, r0011 psb3k3, (redo digestion), t7 psb4a5 in BL21 (replated) pcr (taq) to check for insert minipreped all X10 gold cells for transforming into DH5a run gel to verify inserts, results: no colonies j23100 + gfp psb4a5, j23113+gfp psb1c3, t7 psb4a5 (band too low?), r0011+gfp psb1c3, replate for religate those start overnights for glycerol stocks miniprep overnights of no pcr result colonies, re ligate/transform j23100 psb1c3 run pcr of last nights no results colonies that broth looks good
9/24/2010
run pcr results on gel, results: no band j23114+gfp psb1c3 (not even primers?) made glycerol stocks of t7 psb1a3, t7+gfp psb1a3, and t7 psb1c3 digest (j23100+gfp no results post digestion clean up)/ligate/transform j23100 psb1c3, j23100 psb3k3, r0011 psb3k3, t7 psb4a5 (BL21), r0011+gfp psb1c3, j23100+gfp psb4a5. 20ul digestions (2ul buffer (10x), .5 pstI, .5 specific biobrick enzyme, 4ul DNA in EB, 13ul dh20) ligate, 10ul 7.5 insert to vector goal pcr j23113+gfp psb1c3, gel results, no bands
9/26/2010
transform stocks into DH5a plate growth t7 psb4a5 only
9/27/2010
check plates for growth, all but R0011+gfp (psb3k3 and psb4a5) had growth replated and religated R0011+gfp (psb3k3 and psb4a5) made Chlor. plates digestion/ligation/transformation j23100 (psb1c3, psb3k3), j23100+gfp (very low DNA on nanodrop) psb4a5, j23114 psb1c3, j23113+gfp (psb1c3, psb4a5), R0011 psb3k3, R0011+gfp psb1c3 pcr verify colonies all but t7 psb4a5 good
9/28/2010
digest 50ul, 20ul DNA, extra long (4 hours), clean up all psb's and redigest with xbal I miniprep 1c3 ligate j23100 (psb1c3, psb3k3), j23100+gfp (very low DNA on nanodrop) psb4a5, j23114 psb1c3, j23113+gfp (psb1c3, psb4a5), R0011 psb3k3, R0011+gfp psb1c3
9/29/2010
digest f1, lac I and T7 promoter XP prepare more chem comp cells ligate T7 psb4a5 transform into bl21 transform all ligated vectors pcr purify overnight ligation verify DH5a stocks, see notes, need j23114+gfp psb1c3 and T7+gfp 3k3 check plates (R0011+ gfp), growth R0011+gfp psb3k3, verify tomorrow digest and ligate all none made
9/30/2010
check plates for growth pcr verify colonies, gel results: good J23113 +gfp psb1c3 & psb4a5, j23100+gfp psb4a5... all other not good, plates retained for tomorrow to pick new colonies for verification made overnights of good gel results clean up overnight digestion of f1, lacI and t7 transform all none working vectors into BL21 for assay
10/1/2010
ligate f1, lacI, t7 into psb1c3 transform f1, lacI, t7 into DH5a plated f1, lacI, and t7 onto chorl plates glycerol stocks of all pcr verified overnights miniprep remainder of broth from pcr verified overnights examine plate, some colonies on all types,  pcr verify all colonies, gel results: 4 colonies verified, selected more colonies for overnight pcr overnight ligation of j23114+gfp psb1c3, j23100+gfp psb1c3, R0011+gfp psb3k3
10/2/2010
run gel of overnight pcr, no good results transform overnight ligation and held ligations make glycerol stocks of overnight broths pcr verify f1, LacI, and T7 promoter, f1 and LacI verified, t7 ran off gel
10/3/2010
run gel with t7 psb4a5 and t7+gfp psb3k3 pcr to check for recombants, gel results: no band at right height, all bands below 500kb (primer dimers?) check plates for growth, all plates but 47 psb4a5 had colonies run pcr of 4 good colonies from each plate, include t7 biobrick run gel to verify pcr, gel results: psb1c3, both good, no good psb3k3 check quality of samples in shipping with nano dropper glycercol stock f1 and LacI biobricks make overnights of all good pcr/gel results
10/4/2010
prepare sequecne for psb1c3 parts send of for sequencing all psb1c3 without gfp run gel of overnight pcrs (t7's), results: no good bands  check t7 psb4a5 plate, all colonies red, showing religation of psb4a5 without insert start overnights for assay
10/5/2010
Assay (note R0011+gfp in X10 gold cell line), data on assay computer miniprep psb1c3 and psb3k3 glycerol stock and miniprep t7  plan f1 testing sequence results j23113 not present as a promoter, j23114 only... which mean I swapped j23113 and j23114 and only caught j23114 causing a problem instead of fixing it :( make kan/chlor and amp/chlor plates for f1 testing
10/6/2010
send for sequencing f1, t7, lacI run mini gel with digestions of inserts (t7+gfp, R0011+gfp, J23100+gfp) results: gel t7+gfp had a band all other to low concetration digest psb1c3 f1, j23113, and gfp  re ligate/ transform all non-working colonies (psb1c3 priority j23100) (psb3k3, r0011+gfp, t7+gfp, j23113 psb1c3) fill out submission spreadsheet, started... day 1 f1 test 4 plates, 3k3 w/wo f1, 1a3 w/wo f1 into cj236 cell line start overnights of j23100+gfp
10/7/2010
check plates for growth (psb3k3 R0011/T7 +gfp, psb1c3 f1+j23113, psb1c3 j23100+gfp, f1 plates), growth on all plates except control plates pcr verify colonies, j23113 +f1 good started overnights... all other overnight PCR with more colonies miniprep j23100+gfp (4 overnights), good nanodropper concentraion (~500) overnight PCR J23100+gfp psb1c3, R0011+gfp psb3k3, T7+gfp psb3k3 day 2 f1 test autoclave 12 250ml beakers(?) 3ml broth cultures with anti (kan/ amp) add phage at approx 6 hour mark (M13K07 helper phage) (approx 6pm pst) added at 7pm due to low culture density in some samples let grow for 1 hr to allow for phage infection add 1ml of broth to 40ml kan or kan+amp LB in 250ml flask for overnight growth place in 37C room on shaker for overnight growth
sequence results: f1 and lacI did not pass inspection, T7 good, note all sequences should t7, resending f1 and lacI after digest and gel to verify bands
10/8/2010
fill out submission spreadsheet miniprep f1, LacI, j23113+gfp  send in f1, j23113+gfp and LacI for sequencing run gel on overnight PCR's: j23100+gfp psb1c3 good, grow 3 overnights  day 3 f1 test ODs: psb1a3 .059, .088, .590  f1 1a3 .115, .574, .357  psb3k3 .956, 1.021, .927  f1 3k3 1.453, 1.344, 1.515 submission party, bring munchies
10/10/2010
run gel of ss dna as a test, band result,#1 1a3, fa, fk looks empty (drop) #2 looks best in all types,  1a3 #3  shows some double banding, run 12 lane mini gel with #2 from each and ladder, run at 90V for 45 min for pic tomorrow make overnights of psb3k3 (5 for high concentration)  miniprep j23100+gfp psb1c3
10/11/2010
run ss dna gel, take pic if good (pic on other lab computer) VF/VR pcr psb1c3 vectors for 4a5 transformation digest VF/VR psb1c3 constructs miniprep 3k3, high concentraion digest 3k3, xp  overnight digest of all inserts from 1c3 sequencing results, no good sequences
10/12/2010
run colonies from lacI and f1 plates through colony pcr to verify inserts start overnight of good colonies clean up overnight digest, digestion crashed out (star activity?)
10/13/2010
morning (7am) fusion pcr vf/vr all 1c3 minipreped j23100+gfp psb1c3/psb3k3, biobrick f1/lac I afternoon digest pcr's clean up digestion ligate all 4a5 to all inserts, psb3k3 induced with gfp make more carb and chlor plates send of for sequencing, f1 lac I plate transformations (Justin) plate gfp vectors to verify noise levels
10/14/2010
run digested inserts on gel to verify inserts presence, good bands with some background on all but j23100 and j23114+gfp pcr verify inserts on non-gfp colonies, good bands on 4 and R use illuminator to select gfp colonies check background noise on 1c3 gfp high background noise. no gfp in 4g read sequence results, no good results, re digesting original PCR of f1 and LacI to insert into fresh digestion of 1c3 plated all other gfp to check background count made more carb plates replated/ retransformed all BL21 ligations and j23100 streaked for isolation t7 gfp, r0011 gfp
10/15/2010
transform f1/lac I, plated pcr verify j23100, run on gel, 4 good colonies, 1 broth made check background level in GS plates, good most, 3k3 questionable (either small or none, no distinct) turn overnights into gylcerol stocks/ minipreps plates, no colonies t7 psb4a5
10/18/2010
pcr verify f1/ Lac I, start overnights of good colonies transform t7 (note which electro comp. BL21 color you use: RED),  j23114+gfp psb3k3 glycerol stock overnights send out biobrick parts (use fedEx package on desk)
10/19/2010
send out sequence parts, f1 & lacI replate from ligate psb4a5 j23114+gfp, psb4a5 t7+gfp, psb3k3 t7+gfp, psb3k3 j23114+gfp pcr verify t7 psb4a5, use floresence for gfp constructs, no good colonies rg3k3 and tg 4a5, one questionable colony for tg3k3 set for overnight.  Broths replated. no T7 psb4a5 verified, run more colonies of plate tomorrow give longer extension times (2:30)
10/20/2010
sequence verify biobricks f1 & Lac I, no verification. rePCRing from psb1a3 stock start overnights for assay pcr verify t7 psb4a5 from plate in 4c room pcr verify gfp in psb4a5 miniprep psb3k3 t7+gfp
10/21/2010
digest, ligate, transform psb1c3/ f1 and lacI assay induction: induced at 1530(10 minutes to fully induce all wells) pcr verify psb4a5 j23114+gfp, one good colony set for overnight broth
10/22/2010
fill out fosmid registry page get assay data point create data tables fill out registry
10/23/2010
miniprep f1/ Lac I overnights set up shipments for registry and sequencing
10/25/2010
send of second shipment to registry go to meeting
← Parts We Submitted       References →
Retrieved from " http://2010.igem.org/Team:Washington/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
