"
Page
Discussion
View source
History
teams
Log in
 
Team:Washington/Sandbox
From 2010.igem.org
Home
Gram(-) Therapeutic
Design
Build
Test Results
Gram(+) Therapeutic
Design
Build
Test Results
Tools Created
New Vectors
New Software
Tools Used
Software
Next-Gen Cloning
Protocols
Safety
Meet the Team
Accomplishments
Parts Submitted
Notebook
References
Works Cited
Angelo Scorpio, Donald J Chabot, William A Day, Timothy A Hoover, and Arthur M Friedlander  Capsule depolymerase overexpression reduces Bacillus anthracis virulence  Microbiology 2010 : mic.0.035857-0v1-mic.0.035857-0.
Include references
Play Puzzle Bobble ! Play Pac Man !
will include intro image, idea is currently being fleshed out from sketch into image
21st century antibiotics GENERAL STORYLINE FLOW
The bullet points are the key images that need to be generated.  They should be very general and easy to understand!  The text will basically be explaning the pictures and telling the story.... think of this as a comic book!
gram(+)
summary
why anthrax is bad
proteins fighting anthrax
design
circular permutation
computational design
build
How we made mutants
How we made protiein
Test
gel/mass spec cp vs D
activity cp vs D
catalytic KO of CP vs CP
CP mutations
Specificity switch
Future Directions
Just some bullet points, no figures needed
gram(-)
puzzle bobble with evil bacteria as bubbles?
summary (bugs vs bugs)
type 6 secretion (spear) done-Matthew Harger
Toxin/Antitoxin produced in prescence of XXX (poison) done
design
operon in Pseudomonas —> operon in e coli ( fosmid image)
what parts of this operon are needed? justin has this
sensor —produces—> toxin/antitoxin (classic tinkercell picture) done
build
exporter —> recombineering to promoter  image already made
sensor/toxin: —> PCR —> HSL biobrick vector made by Matthew Harger
Test
Exporter:  Still under construction
Sensor/Toxin
Toxin is toxic in ecoli: (Rachael hood,2010) paper;  Sensor works and produces tse2 as expected ( include image of western blot with annotations)
Antidote still works in e.coli indirect evidence, state lack of mutations, may not need image.;
Future Directions
Just some quick Bullet Points
T7 recombinant cassette insertion
PCR testing for insertion
We want to test for gene insertions and recombinations at several steps of our project.  PCR is the easiest and fastest way to determine the success of Recombineering.
Using a set of primers flanking the intragenic promoter region, we tested the insertion on our recombinants. galK is around 1200 base pairs in size. The T7 double promoter cassette not only deletes galK but around a hundred base pairs of the original sequence.  Therefore, we would expect the galK recombinant PCR to be about 1200 bp larger than the non-recombinant, and the T7 recombinant to be about 100 bp smaller than the non-recombinant. Indeed, this is exactly what we saw.
Analyzing CapD_CP
Figure 4 shows the kinetic constants of CapD_CP's transpeptidation and hydrolysis capability
The two abilities of CapD_CP are transpeptidation and hydrolysis. Based on the Kcat and Km (see figure 4) values of the two, we conclude that CapD_CP is a weak binder and efficient catalyst for the transpeptidation reaction. In terms of  hydrolysis, it shows strong binding but slow catalysis, thus our mutant designs focus on increasing the catalytic efficiency.
Test: Enzyme Assay
The general scheme of our fluorescence-based enzyme assay
After we have the CapD_CP mutants, we tested our mutants for their catalytic activity using our fluorescence-based enzyme assay scheme. Fluorescence-based enzyme assay measures the rate at which fluorescence in the testing media is released and the amount fluorescence depends on the rate at which fluorophore-quencher linkage is disrupted. Our substrate PDGA contains a linked fluorophore-quencher component. The faster fluorophore-quencher component is cleaved, the higher the amount of fluorescence is released and thus the greater the enzymatic activity observed.
Expected weight of CapD_CP without Methionine=55285Da, with Methionine=55417Da. Our mass spec detected a peak at 55274.8Da (no Methionine) well within the 0.02% error limit for our mass spec
Using Recombineering to create a constitutive T6SS
Insertion of T7 promoters
The process for using homologous recombination to replace the T6SS native promoters is simple.  We begin with a previously characterised fosmid (a large, single-copy plasmid based on the F' plasmid) which contains all the genes in our T6SS.  Of particular interest is the fha1-tssA1 intergenic region containing the promoters.  We transform this fosmid into recombineering strain E. coli SW102, which contains heat-inducible lambda phage recombinases and lacks the galactose metabolism gene galK.
Using PCR, we create a cassette containing the galK gene flanked by 50 bp of homology to the intergenic region.  The lambda recombinases integrate the cassette into the fosmid.  The transformants are plated on minimal galactose media to select for the recombinants.
A second cassette is created via PCR, this time containing our new promoters.  In this case, we chose to use T7 phage promoters due to their constitutive expression and small size, making them easier to synthesize.  The promoters are again flanked with homologous regions, and integrated into the fosmid.  This cassette displaces the galK cassette.  The transformants are this time plated on 2-deoxy-galactose (DOG), which is a toxic homolog to galactose.  Cells which can metabolize galactose (that is, those with a functioning galK gene) are killed, while those without the gene are unharmed.
The final step is to transform the recombinant plasmid into a T7 polymerase expression strain, to allow T6SS constitutive expression.
Overall Objective:Using Type VI Secretion as an Antibacterial Agent
Our overall goal is to clone the Type 6 secretion system and the Tse2/Tsi2 toxin/antitoxin system from Pseudomonas aeruginosa into E. coli to make a strain of E. coli that could be used to kill off gram negative pathogens present in the human gut. Ideally, this system would be regulated in such a way that the strain of E. coli would only be able to kill gram negative bacteria when a gram negative pathogen is present. This strain could (ideally) be introduced into the gut either as a preventive measure or as a treatment after a known infection. Ideally, the probiotic would only kill of gram negative bacteria in the area of infection.
Cool Picture goes here
Designing the T6SS for a E. coli probiotic
Justin and or Laura, please write this section short intro, basic plan: -molecular schematic of T6SS (Jesa send to Matthew) -include map of, describe T6SS  putative operon ( there is a map color coded to diagram
-mention that it is in pao1 and state objective overall:  transfer into ecoli so that it functions constitutively ,  what genes
===Objectives:
1. figure out which genes are necessary
Mention not all genes in operon neccessarily needed for T6SS, preferable to be smaller
2. physically transfer genes into E. coli
mention large size makes it difficult, mention/explain fosmid, include fosmid map ( Laura should have This)
3. optimize regulation
mention that fosmid contains natural promoter, may not be trascribed from at all, or at right amount for a plasmid ( more than 1 copy of plasmid)
In order to create a probiotic application for this system, we first attempt to express it heterologously in non-pathogenic E. Coli.  Starting from a Fosmid containing our T6SS, we are using Recombineering to replace the strict native regulation with robust T7 promoters to create strong expression of the T6SS.
All the essential genes for our T6SS are contained within two putative operons, encoded in opposite directions.  The native promoters for both operons are found in the same intergenic region, between fha1 and tssA1.  Therefore, we can easily replace the promoter regions for both operons in one step.
← Accomplishments and Submitted Parts →
Retrieved from " http://2010.igem.org/Team:Washington/Sandbox "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
