"
Page
Discussion
View source
History
teams
Log in
 
Team:Washington/Tools Used/Next-Gen Cloning
From 2010.igem.org
Home
Gram(-) Therapeutic
Design
Build
Test Results
Gram(+) Therapeutic
Design
Build
Test Results
Tools Created
New Vectors
New Software
Tools Used
Software
Next-Gen Cloning
Protocols
Safety
Meet the Team
Accomplishments
Parts Submitted
Notebook
References
Gibson Assembly
The cloning process is often the most time-intensive task for iGEM teams. Methods to streamline the assembly and tuning of gene circuits can increase productivity and lead to more successful iGEM projects. Traditionally, restriction enzymes and their DNA recognition sequences are used to assemble promoters, ribosome binding sites, and gene coding sequences into gene networks.
To create plasmids with more freeform inserts than possible with traditional BioBrick restriction/ligation cloning, we used a method described in Nature Protocols 2009 in which parts are extracted from standard biobricks with primers that have homologies introduced on their 5' ends that overlap the parts that they will be next to. Small parts less than 250 bp are then stitched together using "overlap extension" PCR to create large enough fragments (roughly 500bp) that they can be assembled in a one step reaction as described by Gibson.  This procedure allows the construction of plasmids that have no seams between parts and allows for arbitrary numbers of parts to be combined quickly and efficiently.
Utility
Gibson assembly is particularly useful for the following tasks
Extracting Biobrick parts from multiple source plasmids and assembling them arbitrarily into one plasmid using one cloning step
Modifying short sequences of promoters, ribosome binding sites, and gene coding sequences  - from point mutations to inserting or replacing moderately large tracts (20-60 bp) of DNA
Adding ssrA degradation tags to genes
Creating fusion proteins and operons
Flipping transcription direction of operons
Part Extraction
Extraction Protocol
The first step is to design primers to extract the parts out of standard biobricks. The primers should be designed as usual (i.e. design for whatever Tm is desired over the ends of the part to be extracted. We aimed for around 60C). Then an extra sequence is added to the 5' end of the primers that is homologous to whatever part will be next to it in the final construct. These "overlaps" must have a Tm above 50C for efficient plasmid assembly. In some cases, new parts (i.e. promoters, ribosomal binding sites, ssrA degradation tags, and point mutations) can be introduced in these overlaps, provided they are short enough.
Overlap Extension for Short Amplicons
Overlap Extension PCR
After two adjacent parts have been extracted as above, if one of them is short (<250 bp) they are put into a standard PCR reaction with two primers that match the added homology on the outside of the desired construct. This results in linear DNA that is the result of the two parts stuck end-to-end with no seam and homologies to parts that will be adjacent in the final plasmid. This process can repeated to build DNA fragments roughly 500bp in length.
Gibson Assembly Reaction
Gibson Reaction
The ~500bp pieces of the final plasmid are then all fused together in one reaction with T5 exonuclease, heat-stable Taq ligase, Phusion polymerase, and free nucleotides with buffer. The exonuclease chews back the 5' ends of the strands, leaving "sticky ends." Due to the introduced homologies, the complementary single-stranded DNA anneal to each other in the desired order. The polymerase repairs any extra nucleotides chewed back by the exonuclease, and - finally - the ligase repairs nicks in the DNA. This process results in a circular plasmid that can be transformed efficiently.
Issues and Solutions
This type of procedure relies heavily on introduced homologies on the ends of all the parts involved. Initially we used the standard bioBrick prefix and suffix as the homologous region with which to insert our construct into standard backbones, but this proved problematic because of the NotI site which lies between the E/X and S/P restriction sites in bioBrick backbones, as it is long, consists mostly of Gs and Cs, and is palindromic. This leads to possible mispriming, and an ambiguity in the final configuration of our plasmid (the insert could end up forward, backward, or circular. The backbone could also recircularize without taking up the insert at all) and in turn, a very low yield of our desired construct.
To remedy this, we designed a new prefix and suffix, based on the BglBrick standard which allows for the elimination of the NotI sites. We developed the prefix gaattcctgctgcggagatct and the suffix ggatccaacagggttctcgag by aiming for roughly 50% GC content and a Tm around 68 as calculated by Finnzymes Tm calc . We then modified Psb1A3 and Psb3K3 with these prefixes and suffixes, and had much greater success with our cloning.
We have found that with inserts of less than 500 bp, one often gets many colonies containing insertless plasmid that has been circularized by DNA ligase. In order to determine if this is an issue in a given gibson reaction, for shorter inserts, we often conducted two reactions at once, one including the vector and the insert, and one containing only the vector, with the difference in volume made up with water. The reaction mixes were transformed into E. coli, and each  tranformant mixture was plated on LB agarose plates with selective media. The number of colonies on the control plate (  the plate with transformants of the reaction without any insert) corresponds roughly to the number of colonies on the the cloning plate ( the plate  with transformants of the vector/insert reaction). If the ratio of colonies on the cloning plate   to colonies on the control is 1:1, it would mean that the vast majority of the colonies on the cloning plate are due to insertless, circularized vector. We were able to obtain correct plasmids when the cloning colony:control colony ratio was as low as 2:1, but this required double restriction digest screening of a large number of colonies ( around 24) in order to obtain vector with insert.
References
Daniel Gibson, One-step enzymatic assembly of DNA molecules up to several hundred kilobases in size
Nature Protocols 2009
Gibson, D.G., et al. Enzymatic assembly of DNA molecules up to several hundred kilobases
Nature Methods 2009
← Software We Used       Protocols →
Retrieved from " http://2010.igem.org/Team:Washington/Tools_Used/Next-Gen_Cloning "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
