<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_H14Z1_Hangzhou_Protocol skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:H14Z1 Hangzhou/Protocol</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;</P><DIV id="page_content_container"><DIV class="content"><DIV class="content_box"><H1 class="content_title">Protocol</H1><DIV class="content_conts"><H3 class="content_subtitle">1. Media</H3><P class="content_context" style="text-indent:2em; text-align:justify">
                        Luria-Bertan (LB) medium: 10 g/L tryptone, 10 g/L NaCl, 5 g/L yeast extract;
                        GM17 medium: M17 Broth with 0.2M KH2PO4, adjust pH to 6.8 with 2M NaOH, 10 g/L glucose;
                        G-SGM17 medium: M17-Broth with 0.5 M sucrose, 2.5% glycine, 0.5% glucose.
                    </P><H3 class="content_subtitle">2. PCR</H3><P class="content_context" style="text-indent:2em; text-align:justify">
                        DNA segments were amplified with PrimeSTAR Max DNA Polymerase (Takara). Reaction mixture was
                        prepared as follows:
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Reaction conditions were set as follows:
                    </P><H3 class="content_subtitle">3. SDS-PAGE</H3><P class="content_context" style="text-align:justify">
                        1) Set the casting frames (clamp two glass plates in the casting frames) on the casting stands.
                    </P><P class="content_context" style="text-align:justify">
                        2) Prepare the 12% separating gel as follows:
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Swirl the solution gently but thoroughly. Pipet appropriate amount of separating gel solution
                        into the gap between the glass plates. To make the top of the separating gel be horizontal,
                        fill in isopropanol into the gap until an overflow. Wait for 20-30min to let it gelate.
                    </P><P class="content_context" style="text-align:justify">
                        3) Prepare stacking gel as follows:
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Discard the isopropanol and pipet in stacking gel until an overflow. Insert the well-forming
                        comb without trapping air under the teeth. Wait for 20-30min to let it gelate.
                    </P><P class="content_context" style="text-align:justify">
                        4) Take out the comb after a complete gelation of the stacking gel. Take the glass plates out
                        of the casting frame and set them in the cell buffer dam. Pour the electrophoresis buffer into
                        the inner chamber and keep pouring after overflow untill the buffer surface reaches the
                        required level in the outer chamber.
                    </P><P class="content_context" style="text-align:justify">         5) Prepare the samples:
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Mix the samples with loading buffer. Heat them in boiling water for 5-10 min. The loading
                        buffer contains:
                    </P><P class="content_context" style="text-align:justify">
                        6) Load prepared samples into wells and make sure not to overflow. Load protein marker into the
                        first lane. Then cover the top and connect the anodes.
                    </P><P class="content_context" style="text-align:justify">
                        7) Set the volt as 100 V and run the electrophoresis for 30 min. Then, switch to 180 V and keep
                        running till the blue band reaches the bottom line of the separating gel.
                    </P><P class="content_context" style="text-align:justify">
                        8) Take out the SDS-PAGE gel and put it in staining solution. Stain for 30 min at 100rpm.
                    </P><P class="content_context" style="text-align:justify">
                        9) Pour out the staining solution, wash the gel once with water and immerse it in destaining
                        solution for 30 min at 100 rpm.
                    </P><P class="content_context" style="text-align:justify">
                        Change the destaining solution for several times. When the protein bands become clear, observe
                        it in lights and take pictures.
                    </P><H3 class="content_subtitle">4. Gibson Assembly[1]</H3><P class="content_context" style="text-align:justify">    1) Design the plasmid map by snapgene
                    </P><P class="content_context" style="text-align:justify">
                        2) Design the primers with 20-40bp flank
                    </P><P class="content_context" style="text-align:justify">
                        3) PCR fragments and purified
                    </P><P class="content_context" style="text-align:justify">
                        4) Add the fragmens into 15ul Gibson mix buffter. The final concentration of the fragments are
                        0.03pmol/L each. The total volume is 20 ul.
                    </P><P class="content_context" style="text-align:justify">
                        5) Lignation an hour in 50℃
                    </P><P class="content_context" style="text-align:justify">
                        6) Heat-shock transform into E.coli
                    </P><P class="content_context" style="text-align:justify">
                        7) Screen colony using PCR
                    </P><H3 class="content_subtitle">5. Plasmid DNA isolation from Lactococcus lactis[2]</H3><P class="content_context" style="text-align:justify">
                        1) Use 5 ml full-grown culture with M17 broth containing 10 μg/ml chloramphenicol or
                        erythromycin.
                    </P><P class="content_context" style="text-align:justify">
                        2) Spin down 10 min at 3000-6000 x g.
                    </P><P class="content_context" style="text-align:justify">
                        3) Resuspend pellet in 250 μl THMS buffer + 2 mg/ml lysozyme in reaction tube.
                    </P><P class="content_context" style="text-align:justify">
                        4) Incubate 10 min at 37oC
                    </P><P class="content_context" style="text-align:justify">
                        5) Add 500 μl 0.2 N NaOH + 1% SDS, shake carefully. Incubate 5 min on ice
                    </P><P class="content_context" style="text-align:justify">
                        6) Spin 5 min in benchtop centrifuge
                    </P><P class="content_context" style="text-align:justify">
                        7) Take out supernatant and add to new reaction tube
                    </P><P class="content_context" style="text-align:justify">
                        8) Fill the cup with 2-propanol
                    </P><P class="content_context" style="text-align:justify">
                        9) Incubate 5-10 min at room temperature
                    </P><P class="content_context" style="text-align:justify">
                        10) Spin 10 min in benchtop centrifuge
                    </P><P class="content_context" style="text-align:justify">
                        11) Wash pellet carefully with 70% ethanol and dry the pellet
                    </P><P class="content_context" style="text-align:justify">
                        12) Dissolve the pellet in 50 μl TE or sterile water
                    </P><H3 class="content_subtitle">6. Transformation of Lactococcus lactis with electroporation[2]</H3><P class="content_context" style="font-size: 18px; font-weight: bold">
                        6.1 Preparation of the cells:
                    </P><P class="content_context" style="text-align:justify">
                        Day 1: Inoculate 5 ml of G/L-SGM17B medium with L. lactis glycerol stock from -80 ℃
                        and grow at 30 ℃, without aeration, overnight </P><P class="content_context" style="text-align:justify">
                        Day 2: Inoculate 50 ml of G/L-SGM17B with pre-culture in a dilution of 1:100 and grow at 30 ℃,
                        without aeration, overnight </P><P class="content_context" style="text-align:justify">
                        Day 3: Inoculate 5 ml of G/L-SGM17B medium with L. lactis glycerol stock from -80 ℃
                        and grow at 30 ℃, without aeration, overnight
                    </P><P class="content_context" style="text-align:justify">
                        1) Add 50 ml full-grown culture to 400 ml of G/L-SGM17B medium.
                    </P><P class="content_context" style="text-align:justify">
                        2) Grow the culture until OD600 is 0.2-0.3 (ca. 3 h).
                    </P><P class="content_context" style="text-align:justify">
                        3) Spin down cells for 20 min at 6000 × g, 4 ℃
                    </P><P class="content_context" style="text-align:justify">
                        4) Wash cells with 400 ml of 0.5 M sucrose, 10% glycerol (4 ℃) and spin down at 6000 x g
                        (centrifugation speed may need to be increased during successive washing steps).
                    </P><P class="content_context" style="text-align:justify">
                        5) Resuspend the cells in 200 ml of 0.5 M sucrose, 10% glycerol, 50 mM EDTA (4 ℃), keep the
                        suspension on ice for 15 min and spin down.
                    </P><P class="content_context" style="text-align:justify">
                        6) Wash cells with 100 ml of 0.5 M sucrose, 10% glycerol (4 ℃) and spin down (6000 x g).
                    </P><P class="content_context" style="text-align:justify">
                        7) Resuspend the cells in 4 ml of 0.5 M sucrose, 10% glycerol (4 ℃):
                    </P><P class="content_context" style="text-align:justify">
                        8) Use 40 μl per electroporation (keep on ice) or store the cells in small portions at -80 ℃,
                        let them defreeze on ice before use
                    </P><P class="content_context" style="font-size: 18px; font-weight: bold">
                        6.2 Electroporation:
                    </P><P class="content_context" style="text-align:justify">
                        1) Place 40 μl cells in a pre-chilled electroporation cuvette with 1 μl DNA (100-500 ng vector
                        DNA reconstituted in TE-, Tris-buffer, or distilled water; for transforming cells with ligation
                        product use 500-1000 ng DNA) and keep the cuvette on ice
                    </P><P class="content_context" style="text-align:justify">
                        2) Use Biorad Genepulser with following adjustments: 2000 V 25 μF 200 Ω
                    </P><P class="content_context" style="text-align:justify">
                        3) Pulse (normal reading is 4.5-5 msec)
                    </P><P class="content_context" style="text-align:justify">
                        4) Add 1 ml of G/L-M17B + 20 mM MgCl2 + 2 mM CaCl2
                    </P><P class="content_context" style="text-align:justify">
                        5) Keep the cuvette for 5 min on ice and incubate 1-1.5 h at 30 °C
                    </P><P class="content_context" style="text-align:justify">
                        6) Plate 10 μl, 100 μl, 900 μl on M17agar with glucose or lactose and antibiotics
                    </P><P class="content_context" style="text-align:justify">
                        7) Incubate 1-2 days at 30 °C
                    </P><H3 class="content_subtitle">7. Analysis of process and product</H3><P class="content_context" style="font-size: 18px; font-weight: bold">
                        7.1 Cell growth
            </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Cell growth was monitored by measuring the OD600 of the culture and counting the Cell number
                        using hemocytometer.
                    </P><P class="content_context" style="font-size: 18px; font-weight: bold">
                        7.2 GSH
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        CIntracellular GSH was extracted by resuspending cells in 1.2M perchloric acid and cultivating
                        for 4 hours at 4℃. Cell resuspension was reverse blended every 30 min to prevent cell
                        aggregation.
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Cell extracts were centrifuged at 12,000g for 3 min, the supernatant was taken for HPLC
                        analysis. Agilent 1100 series with a UV absorption detector and a reverse phase Agilent TC-C18
                        (2) column was used for HPLC. The mobile phase was 50 mM potassium phosphate buffer, pH 3.0,
                        with 10 mM sodium heptanesulfonate and 5% methanol. The column temperature was set at 30 ℃ and
                        flow rate at 1.0 ml/min. Quantification of GSH was based on its absorbance at 210 nm
                    </P><P class="content_context" style="font-size: 18px; font-weight: bold">
                        7.3 SAM
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Intracellular SAM was extracted by resuspending cells in 400ul water and ethyl acetate (1:1)
                        mixture and cultivating for 0.5 hour at 30℃, 200rpm. Then, 400ul 0.35M H2SO4 solution was added
                        and cultivated for another 1.5 hours.
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Cell extracts were centrifuged at 12,000g for 3 min, the supernatant was taken for HPLC
                        analysis. Agilent 1100 series with a UV absorption detector and a reverse phase Agilent TC-C18
                        (2) column was used for HPLC. The mobile phase was 40 mM ammonium dihydrogen phosphate, with 10
                        mM sodium heptanesulfonate and 18% methanol. The flow rate was set at 1.0 ml/min and column
                        temperature not controlled. Quantification of SAM was based on its absorbance at 254 nm[5].
                    </P><P class="content_context" style="font-size: 18px; font-weight: bold">
                        7.4 Self-aggregation value[3]
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        To evaluate auto-aggregation rate, cells were collected and washed twice with o.1M PBS buffer
                        (pH=7.0) and resuspended in 4ml PBS buffer (pH=7.0) with an initial OD600=0.5 (A0). The final
                        OD600 (At) of the uppermost layer was measured after 4 hours’ cultivation in 30℃ without
                        shaking. The auto-aggregation value was calculated using the following equation:

                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Autoaggregation (%)=100-(At/A0)×100
                    </P></DIV><H1 class="content_title">References</H1><DIV class="content_conts"><P class="content_context" style="text-indent:2em; text-align:justify">
                        [1] Gibson, D.G., Young, L., Chuang, RY., Venter, J.C., Hutchison, C.A., Smith, H.O., 2009,
                        Enzymatic assembly of DNA molecules up to several hundred kilobases, Nat. Methods 6, 343-345.
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        [2] MoBiTec GmbH, 2015, Expression System for Lactococcus lactis, The effective &amp;
                        easy-to-operate NIsin Controlled gene Expression system.
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        [3] Zhang B., Zuo F., Yu R., Zeng Z., Ma H., Chen S., 2015, Comparative genome-based
                        identification of a cell wall-anchored protein from Lactobacillus plantarum increases adhesion
                        of Lactococcus lactis to human epithelial cells, Sci. Rep-UK 5: 14109.
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        [4] Rui-Yan, F. U., Chen, J., &amp; Yin, L. I. 2004. Production of glutathione by recombinant
                        lactococcus lactis. Chinese J. of Bioprocess Engineering.
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        [5] Han, G., Hu, X., &amp; Wang, X. 2016. Overexpression of methionine adenosyltransferase in
                        corynebacterium glutamicum for production of s‐adenosyl‐l‐methionine. Biotech. App.
                        Biochem. 63(5), 679-689.
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>