Team:H14Z1 Hangzhou/Experiments
<!DOCTYPE html>
Experiments
Overview:
To get our designed “smart yogurt”, plasmid pNZ-GMcA, which can realize enhancement of GSH and                         SAM production and adhesion ability, is going to be constructed finally. In order to examine                         each module (GSH module, SAM module and Adhesion factor module), three gene fragments (gshF,                         metK and cwaA) were cloned to plasmid pNZ8148, separately. Then, the combinations of the three                         modules were carried out to construct plasmid pNZ-GM (containing GSH and SAM module) and                         pNZ-GMcA. Finally, the engineered strains were used for fermentation to get “smart yogurt”.
Details:
1. Origin of the genes in the three modules
Codon optimized gene gshF from Streptococcus thermophilus and cwaA from Lactobacillus, a cell                         wall anchoring protein, were obtained by chemical synthesis; gene metK was amplified from the                         genome of Lactococcus lactis NZ9000.
2. Construction of three modules (gshF, metK and cwaA)
Plasmid pNZ-gshF containing gshF gene was constructed as GSH module. Plasmid pNZ-metK                         containing metK gene was constructed as SAM module. Plasmid pNZ-cwaA containing cwaA gene was                         constructed as adhesion factor module.
Figure1. The schematic diagram for construction of the three modules.
Gene gshF, metK and cwaA were amplified from chemical synthesized gene fragment and genomic DNA                         of Lactococcus lactis NZ9000, separately, using the following primers. The fragments obtained                         from PCR and plasmid pNZ8148 were cut with same restriction enzymes (gshF using HindIII and                         NcoI, metK using KpnI and Hind III, cwaA using Hind III and NcoI). Then the cut fragments and                         plasmid pN8148 were linked with T4 DNA ligase separately and transferred to E.coli DH5α.
gshF-F-Hind III5’-CCCAAGCTTGGATCCTCTAGAGTCGACCT -3’
gshF-R-Nco I5’-CATGCCATGGTTATGATTATCGATCGACTGTT -3’
metK-F-Kpn I5’- CGGGGTACCATATGTCAGAAAAACATTTATT - 3’
metK-R-Hind III5’- CCCAAGCTTTTATTTACCTAAAACAGCTT - 3’
cwaA-F-Nco I5’-CCCAAGCTTGGATCCTCTAGAGTCGACCT -3’
cwaA-R-Hind III5’-CCCAAGCTTTTATCTAGATGCCT -3’
3. Validation of the three plasmids
Colonies on the plates were randomly picked and inoculated in 1ml LB medium for 3 hours at 37℃,                         200 rpm. 1 μl culture were added to the PCR system as template. The length of gene gshF, metK                         and cwaA were 2298 bp, 1200 bp and 2800 bp, respectively. As shown in Figure 2, all the picked                         colonies had gene gshF, metK or cwaA, illustrating that the plasmid pNZ-gshF, pNZ-metK and                         pNZ-cwaA was successfully constructed. Primers used for validation were the same as using for                         amplification, as shown above.
Figure 2. Validation of constructed plasmids. A. pNZ-gshF, B. pNZ-metK, C. pNZ-cwaA. M                         represents marker. 1~5 represent randomly picked colonies.
4. Combinations of the three modules
After validation of the three modules, we further combined two and three modules into one                         plasmid to construct plasmid pNZ-GM and pNZ-GMcA.
Figure 3. The schematic diagram for construction of plasmid pNZ-GM and pNZ-GMcA.
4.1 Construction of plasmid pNZ-GM
Gene gshF and metK were expressed in tandem and controlled by the same inducible promoter                         PnisA. Gibson assembly[1] was used to join gshF and metK to form pNZ-GM which was then                         transformed into E.coli DH5α. RBS site was added in front of gene metK. Primers used were as                         following:
GSH-CZ-F5‘-AGGTAAATAATGTTGCTTTGATTGATAGCC-3’
GSH-CZ-R5‘-CTACCTCCTAAAGCAGTTATAATTCTGGGAAC-3’
cwaA-F-Nco I5‘-TATAACTGCTTTAGGAGGTAGTCCAAATGTCAGAAAAACATTTATT-3’
cwaA-R-Hind III5‘-ATCAAAGCAACATTATTTACCTAAAACAGCTT-3’
RBS sequence is shown in blue.
4.2 Validation of plasmid pNZ-GM
Colonies on the plates were randomly picked and inoculated in 1ml LB medium for 3 hours at 37℃,                         200 rpm. 1 μl culture were added to the PCR system as template. The length of the fragment                         obtained by PCR using the primers for verification was 3494 bp theoretically if plasmid pNZ-GM                         was constructed successfully. As shown in Figure 4, all the picked colonies had gene gshF and                         metK, illustrating that the plasmid pNZ-GM was successfully constructed.
Primers:
METK-R5‘-TTATTTACCTAAAACAGCTT-3’
GSH-F5‘-TTATGATTATCGATCGACTGTT-3’
Figure 4. Validation of plasmid pNZ-GM. M represents marker. 1~6 represent randomly picked                         colonies.
5. Construction of plasmid pNZ-GMcA
In order to improve the colonization ability of Lactococcus lactis while producing GSH and SAM,                         we added adhesion factor module to plasmid pNZ-GM. Plasmid pNZ-GM and adhesion module amplified                         from plasmid pNZ-cwaA using primers below were cut by Pst I and Sal I, then linked with T4 DNA                         ligase and transformed into E.coli DH5α.
Primers:
pNZ-ORF-F-SalI5‘-ACGCGTCGACCAGTCGGTTTTCTAATGTCAC -3’
pNZ-ORF-R-PstI5‘-AAAACTGCAGCGAAAGCGAAATCAAACGAA-3’
6. Verification of plasmid pNZ-GMcA
Colonies on the plates were randomly picked and inoculated in 1ml LB medium for 3 hours at 37℃,                         200 rpm. 1 μl cultures were added to the PCR system as template. The length of the fragment 1                         obtained by PCR using primer pair cwaA-R-HindIII and pNZ-ORF-F-SalI for verification was 3760                         bp theoretically and fragment 2 using primer pair metK-CZ-R-N and pNZ-ORF-F-SalI was 3079 bp if                         plasmid pNZ-GM was constructed successfully. As shown in Figure 5, all the picked colonies                         contains gene gshF, metK and cwaA, illustrating that the plasmid pNZ-GMcA was successfully                         constructed.
Primers:
cwaA-R-HindIII5‘-AAGCTTTTATCTAGATGCCT-3’
pNZ-ORF-F-SalI5‘-GTCGACCAGTCGGTTTTCTAATGTCAC-3
metK-CZ-R-N5‘-ATCAAAGCAACATTATTTACCTAAAACAGCTT-3’
pNZ-ORF-F-SalI5‘-GTCGACCAGTCGGTTTTCTAATGTCAC-3’
Figure 5. Validation of plasmid pNZ-GMcA. M represented marker. gm-1 to gm-5 and cA-1 to cA-5                         all represented the same five randomly picked colonies. The fragments of gm-1 to gm-5 were                         obtained using primer pair cwaA-R-HindIII and pNZ-ORF-F-SalI. The fragments of cA-1 to cA-5                         were obtained using primer pair metK-CZ-R-N and pNZ-ORF-F-SalI.
7. Functional characterization of three modules in Lactococcus                             lactis
All the constructed plasmids were transformed into Lactococcus lactis NZ 9000 respectively with                         standard electroporation method. The recombinant cells were inoculated to evaluate the                         potentials of the targeted genes during the cultivation process using the standard medium which                         was used in L. lactis. SDS-PAGE was employed for the possible expression of our targeted genes. L. lactis was inoculated into flasks containg M17 broth with 0.5% sugar at 30OC. With induction                         of nisin at the initial medium, the expression of three different modules was detected by HPLC                         analyses of GSH and SAM, and the expression of cwaA was characterized by measuring                         self-aggregation value.
8. Preparation of smart yogurt
In our experiments, the same formula (93%raw milk , 7% white granulated sugar) was prepared to                         produce yogurt with different strains. At present three strain modes (Lactococcus lactis NZ                         9000, Lactococcus lactis NZ 9000/PNZ-GMcA, Lactococcus lactis NZ 9000/PNZ-GMcA plus                         Lactobacillus bulgaricus (Dennis L340)) was inoculated into milk medium respectively.
The general process is described in the following:
1) Heat raw milk to 55-60 oC, add a quantity of sugar, then stir then for 10 minutes at 3000                         rpm, finally heat up to 65 oC.
2) Keep the culture medium at 65 oC for two-stage homogenization (5/20MPa), then do the                         sterilization at 90 oC for 5 minutes. Then cool down to 30 oC for the next inoculation.
3) Inoculate the milk-containing medium with suitable amount of seed, the fermentor was                         maintained for 16 hour at 30 oC. For both fermentors, both of the end pH was 4.6.
4) Pack the yoghurt samples into sterile milk cartoon in aseptic manipulation room and put them                         in 4 oC refrigeration for further analysis.
Note: the special amino acid mixture can be added with the seed according to different                         experiment purposes.
