<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_H14Z1_Hangzhou_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:H14Z1 Hangzhou/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;</P><DIV id="page_content_container"><DIV class="content"><DIV class="content_box"><H1 class="content_title">Experiments</H1><DIV class="content_conts"><H3 class="content_subtitle">Overview:</H3><P class="content_context" style="text-indent:2em; text-align:justify">
                        To get our designed “smart yogurt”, plasmid pNZ-GMcA, which can realize enhancement of GSH and
                        SAM production and adhesion ability, is going to be constructed finally. In order to examine
                        each module (GSH module, SAM module and Adhesion factor module), three gene fragments (gshF,
                        metK and cwaA) were cloned to plasmid pNZ8148, separately. Then, the combinations of the three
                        modules were carried out to construct plasmid pNZ-GM (containing GSH and SAM module) and
                        pNZ-GMcA. Finally, the engineered strains were used for fermentation to get “smart yogurt”.
                    </P><H3 class="content_subtitle">Details:</H3><H6 class="content_sub_subtitle">1. Origin of the genes in the three modules</H6><P class="content_context" style="text-indent:2em; text-align:justify">
                        Codon optimized gene gshF from Streptococcus thermophilus and cwaA from Lactobacillus, a cell
                        wall anchoring protein, were obtained by chemical synthesis; gene metK was amplified from the
                        genome of <I>Lactococcus lactis</I> NZ9000.
                    </P><H6 class="content_sub_subtitle">2. Construction of three modules (gshF, metK and cwaA)</H6><P class="content_context" style="text-indent:2em; text-align:justify">
                        Plasmid pNZ-gshF containing gshF gene was constructed as GSH module. Plasmid pNZ-metK
                        containing metK gene was constructed as SAM module. Plasmid pNZ-cwaA containing cwaA gene was
                        constructed as adhesion factor module.
                    </P><P class="content_context" style="text-align:center; font-size:18px">
                        Figure1. The schematic diagram for construction of the three modules.
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Gene gshF, metK and cwaA were amplified from chemical synthesized gene fragment and genomic DNA
                        of <I>Lactococcus lactis</I> NZ9000, separately, using the following primers. The fragments obtained
                        from PCR and plasmid pNZ8148 were cut with same restriction enzymes (gshF using HindIII and
                        NcoI, metK using KpnI and Hind III, cwaA using Hind III and NcoI). Then the cut fragments and
                        plasmid pN8148 were linked with T4 DNA ligase separately and transferred to E.coli DH5α.
                    </P><P class="content_context"><SPAN style="width:150px">gshF-F-Hind III</SPAN><SPAN>5’-CCCAAGCTTGGATCCTCTAGAGTCGACCT -3’</SPAN></P><P class="content_context"><SPAN style="width:150px">gshF-R-Nco I</SPAN><SPAN>5’-CATGCCATGGTTATGATTATCGATCGACTGTT -3’</SPAN></P><P class="content_context"><SPAN style="width:150px">metK-F-Kpn I</SPAN><SPAN>5’- CGGGGTACCATATGTCAGAAAAACATTTATT - 3’</SPAN></P><P class="content_context"><SPAN style="width:150px">metK-R-Hind III</SPAN><SPAN>5’- CCCAAGCTTTTATTTACCTAAAACAGCTT - 3’</SPAN></P><P class="content_context"><SPAN style="width:150px">cwaA-F-Nco I</SPAN><SPAN>5’-CCCAAGCTTGGATCCTCTAGAGTCGACCT -3’</SPAN></P><P class="content_context"><SPAN style="width:150px">cwaA-R-Hind III</SPAN><SPAN>5’-CCCAAGCTTTTATCTAGATGCCT -3’</SPAN></P><H6 class="content_sub_subtitle">3. Validation of the three plasmids</H6><P class="content_context" style="text-indent:2em; text-align:justify">
                        Colonies on the plates were randomly picked and inoculated in 1ml LB medium for 3 hours at 37℃,
                        200 rpm. 1 μl culture were added to the PCR system as template. The length of gene gshF, metK
                        and cwaA were 2298 bp, 1200 bp and 2800 bp, respectively. As shown in Figure 2, all the picked
                        colonies had gene gshF, metK or cwaA, illustrating that the plasmid pNZ-gshF, pNZ-metK and
                        pNZ-cwaA was successfully constructed. Primers used for validation were the same as using for
                        amplification, as shown above.
                    </P><P class="content_context" style="text-align:center; font-size:18px">
                        Figure 2. Validation of constructed plasmids. A. pNZ-gshF, B. pNZ-metK, C. pNZ-cwaA. M
                        represents marker. 1~5 represent randomly picked colonies.
                    </P><H6 class="content_sub_subtitle">4. Combinations of the three modules</H6><P class="content_context" style="text-indent:2em; text-align:justify">
                        After validation of the three modules, we further combined two and three modules into one
                        plasmid to construct plasmid pNZ-GM and pNZ-GMcA.
                    </P><P class="content_context" style="text-align:center; font-size:18px">
                        Figure 3. The schematic diagram for construction of plasmid pNZ-GM and pNZ-GMcA.
                    </P><P class="content_context" style="font-size: 18px; font-weight: bold">
                        4.1 Construction of plasmid pNZ-GM
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Gene gshF and metK were expressed in tandem and controlled by the same inducible promoter
                        PnisA. Gibson assembly[1] was used to join gshF and metK to form pNZ-GM which was then
                        transformed into E.coli DH5α. RBS site was added in front of gene metK. Primers used were as
                        following:
                    </P><P class="content_context"><SPAN style="width:150px">GSH-CZ-F</SPAN><SPAN>5‘-AGGTAAATAATGTTGCTTTGATTGATAGCC-3’</SPAN></P><P class="content_context"><SPAN style="width:150px">GSH-CZ-R</SPAN><SPAN>5‘-CTACCTCCTAAAGCAGTTATAATTCTGGGAAC-3’</SPAN></P><P class="content_context"><SPAN style="width:150px">cwaA-F-Nco I</SPAN><SPAN>5‘-TATAACTGCTTTAGGAGGTAGTCCAAATGTCAGAAAAACATTTATT-3’</SPAN></P><P class="content_context"><SPAN style="width:150px">cwaA-R-Hind III</SPAN><SPAN>5‘-ATCAAAGCAACATTATTTACCTAAAACAGCTT-3’</SPAN></P><P class="content_context">
                        RBS sequence is shown in blue.
                    </P><P class="content_context" style="font-size: 18px; font-weight: bold">
                        4.2 Validation of plasmid pNZ-GM
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        Colonies on the plates were randomly picked and inoculated in 1ml LB medium for 3 hours at 37℃,
                        200 rpm. 1 μl culture were added to the PCR system as template. The length of the fragment
                        obtained by PCR using the primers for verification was 3494 bp theoretically if plasmid pNZ-GM
                        was constructed successfully. As shown in Figure 4, all the picked colonies had gene gshF and
                        metK, illustrating that the plasmid pNZ-GM was successfully constructed.
                    </P><P class="content_context">
                        Primers:
                    </P><P class="content_context"><SPAN style="width:150px">METK-R</SPAN><SPAN>5‘-TTATTTACCTAAAACAGCTT-3’</SPAN></P><P class="content_context"><SPAN style="width:150px">GSH-F</SPAN><SPAN>5‘-TTATGATTATCGATCGACTGTT-3’</SPAN></P><P class="content_context" style="text-align:center; font-size:18px">
                        Figure 4. Validation of plasmid pNZ-GM. M represents marker. 1~6 represent randomly picked
                        colonies.
                    </P><H6 class="content_sub_subtitle">5. Construction of plasmid pNZ-GMcA</H6><P class="content_context" style="text-indent:2em; text-align:justify">
                        In order to improve the colonization ability of <I>Lactococcus lactis</I> while producing GSH and SAM,
                        we added adhesion factor module to plasmid pNZ-GM. Plasmid pNZ-GM and adhesion module amplified
                        from plasmid pNZ-cwaA using primers below were cut by Pst I and Sal I, then linked with T4 DNA
                        ligase and transformed into E.coli DH5α.
                    </P><P class="content_context">
                        Primers:
                    </P><P class="content_context"><SPAN style="width:150px">pNZ-ORF-F-SalI</SPAN><SPAN>5‘-ACGCGTCGACCAGTCGGTTTTCTAATGTCAC -3’</SPAN></P><P class="content_context"><SPAN style="width:150px">pNZ-ORF-R-PstI</SPAN><SPAN>5‘-AAAACTGCAGCGAAAGCGAAATCAAACGAA-3’</SPAN></P><H6 class="content_sub_subtitle">6. Verification of plasmid pNZ-GMcA</H6><P class="content_context" style="text-indent:2em; text-align:justify">
                        Colonies on the plates were randomly picked and inoculated in 1ml LB medium for 3 hours at 37℃,
                        200 rpm. 1 μl cultures were added to the PCR system as template. The length of the fragment 1
                        obtained by PCR using primer pair cwaA-R-HindIII and pNZ-ORF-F-SalI for verification was 3760
                        bp theoretically and fragment 2 using primer pair metK-CZ-R-N and pNZ-ORF-F-SalI was 3079 bp if
                        plasmid pNZ-GM was constructed successfully. As shown in Figure 5, all the picked colonies
                        contains gene gshF, metK and cwaA, illustrating that the plasmid pNZ-GMcA was successfully
                        constructed.
                    </P><P class="content_context">
                        Primers:
                    </P><P class="content_context"><SPAN style="width:150px">cwaA-R-HindIII</SPAN><SPAN>5‘-AAGCTTTTATCTAGATGCCT-3’</SPAN></P><P class="content_context"><SPAN style="width:150px">pNZ-ORF-F-SalI</SPAN><SPAN>5‘-GTCGACCAGTCGGTTTTCTAATGTCAC-3</SPAN></P><P class="content_context"><SPAN style="width:150px">metK-CZ-R-N</SPAN><SPAN>5‘-ATCAAAGCAACATTATTTACCTAAAACAGCTT-3’</SPAN></P><P class="content_context"><SPAN style="width:150px">pNZ-ORF-F-SalI</SPAN><SPAN>5‘-GTCGACCAGTCGGTTTTCTAATGTCAC-3’</SPAN></P><P class="content_context" style="text-align:center; font-size:18px">
                        Figure 5. Validation of plasmid pNZ-GMcA. M represented marker. gm-1 to gm-5 and cA-1 to cA-5
                        all represented the same five randomly picked colonies. The fragments of gm-1 to gm-5 were
                        obtained using primer pair cwaA-R-HindIII and pNZ-ORF-F-SalI. The fragments of cA-1 to cA-5
                        were obtained using primer pair metK-CZ-R-N and pNZ-ORF-F-SalI.
                    </P><H6 class="content_sub_subtitle">7. Functional characterization of three modules in <I>Lactococcus
                            lactis</I></H6><P class="content_context" style="text-indent:2em; text-align:justify">
                        All the constructed plasmids were transformed into <I>Lactococcus lactis</I> NZ 9000 respectively with
                        standard electroporation method. The recombinant cells were inoculated to evaluate the
                        potentials of the targeted genes during the cultivation process using the standard medium which
                        was used in <I>L. lactis</I>. SDS-PAGE was employed for the possible expression of our targeted genes.
                        <I>L. lactis</I> was inoculated into flasks containg M17 broth with 0.5% sugar at 30OC. With induction
                        of nisin at the initial medium, the expression of three different modules was detected by HPLC
                        analyses of GSH and SAM, and the expression of cwaA was characterized by measuring
                        self-aggregation value.
                    </P><H6 class="content_sub_subtitle">8. Preparation of smart yogurt</H6><P class="content_context" style="text-indent:2em; text-align:justify">
                        In our experiments, the same formula (93%raw milk , 7% white granulated sugar) was prepared to
                        produce yogurt with different strains. At present three strain modes (<I>Lactococcus lactis</I> NZ
                        9000, <I>Lactococcus lactis</I> NZ 9000/PNZ-GMcA, <I>Lactococcus lactis</I> NZ 9000/PNZ-GMcA plus
                        Lactobacillus bulgaricus (Dennis L340)) was inoculated into milk medium respectively.
                    </P><P class="content_context" style="text-indent:2em; text-align:justify">
                        The general process is described in the following:
                    </P><P class="content_context">
                        1) Heat raw milk to 55-60 oC, add a quantity of sugar, then stir then for 10 minutes at 3000
                        rpm, finally heat up to 65 oC.
                    </P><P class="content_context">                       2) Keep the culture medium at 65 oC for two-stage homogenization (5/20MPa), then do the
                        sterilization at 90 oC for 5 minutes. Then cool down to 30 oC for the next inoculation. </P><P class="content_context">
                        3) Inoculate the milk-containing medium with suitable amount of seed, the fermentor was
                        maintained for 16 hour at 30 oC. For both fermentors, both of the end pH was 4.6.
                    </P><P class="content_context">
                        4) Pack the yoghurt samples into sterile milk cartoon in aseptic manipulation room and put them
                        in 4 oC refrigeration for further analysis.
                    </P><P class="content_context">
                        Note: the special amino acid mixture can be added with the seed according to different
                        experiment purposes.
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>