Team:H14Z1 Hangzhou/Parts
<!DOCTYPE html>
Parts
Basic parts
-- Part:BBa_K2856001
Modification of bi-functional glutamate--cysteine ligase/glutathione synthase (gshF)
The BBa_K2856001 harbors a coding sequence of bi-functional glutamate--cysteine                         ligase/glutathione synthase (gshF) derived from S. agalactiae. Codon-optimization has been made                         for Lactococcus lactis. gshFp catalyzes the conversion of Cys, Glu and Gly to GSH.
Figure. 1 Schematic diagram of GSH module.
Usage and Biology
Bifunctional glutamate--cysteine ligase/glutathione synthase (gshF) is an enzyme involved and                         responded to synthetic reaction of GSH. In this reaction, one Cysteine and one Glutamate are                         converted to one γ-GC, then one γ-GC and one Glycine are converted to one GSH (Figure 2). The Lactococcus lactis NZ9000 has inability to synthesis GSH. In our project, we construct a                         plasmid harboring gshF in order to produce GSH in Lactococcus lactis NZ9000.
Figure. 2 Enzymatic reaction catalyzed by gshF.
Construction and validation of plasmid pNZ-gshF
Gene gshF was amplified from genomic DNA of S. agalactiae and cut with restriction enzyme Hind                         III and NcoI, and ligased with plasmid pNZ8148 cut with the same enzyme. Then the ligation                         product was transferred to E.coli and spread on plates containing 10 mg/L chloramphenicol.
Colonies on the plates were randomly picked and inoculated in 1ml LB medium for 3 hours at 37℃,                         200 rpm. 1 μl culture were added to the PCR system as template. As shown in Figure. 3, all the                         picked colonies had gene gshF, illustrating that the plasmid pNZ-gshF was successfully                         constructed.
Figure. 3 Validation of plasmid pNZ-gshF. M represented marker. 1, 2 and 3 represented three                         randomly                         picked colonies.
Protein Analysis
After transferring the plasmid pNZ-gshF to L. lactis NZ9000, SDS-PAGE was performed to detect                         the protein expression level of gshF gene. The cells were washed twice with 0.1 M PBS after                         centrifugation. Crude protein was extracted through cell breaking using ultrasonication and                         centrifugation. Then the supernatant of the samples were used to analysis the protein                         expression. As shown in Figure. 4, expected bands of the GshF protein were observed on the gel                         in the lane of recombinant L. lactis containing pNZ-gshF induced with different nisin                         concentration while no GshF protein existed in L. lactis NZ9000.
Figure. 4 SDS PAGE validation of gene gshF expression in L. lactis. M represented marker. WT                         represented L. lactis NZ9000. 1-3 represented L. lactis/pNZ-gshF induced with 100, 50 and 20                         ng/ml nisin.
Validation of glutathione (GSH) by HPLC analysis
To confirm the synthetic glutathione in L. lactis/pNZ-gshF, HPLC was performed to analyze the                         extracts from the strain. Glutathione was identified on the basis of retention times related to                         standard sample. According to the retention time of standard glutathione sample, it can be                         confirmed that glutathione was synthesized in L. lactis/pNZ-gshF.
Figure. 5 Validation of glutathione (GSH) by HPLC.
-- Part:BBa_K2856002
Modification of S-adenosylmethionine synthetase (MetK)
The BBa_K2856001 harbors a coding sequence of S-adenosyl-methionine synthetase (MetK) derived                         from Lactococcus lactis NZ9000 genome. The MetK protein catalyzes methionine and ATP to form                         S-adenosyl-methionine.
Figure. 1 Schematic diagram of SAM module.
Usage and Biology
S-adenosyl-methionine synthetase encoded by gene metK is an enzyme involved and responded to                         the synthetic reaction of S-adenosyl-methionine (SAM). In this reaction, one molecule                         methionine and one molecule ATP are converted to one molecule SAM (Figure 2). The L. lactis                         NZ9000 has the ability to synthesize SAM, but just enough for itself. In our project, we                         constructed a plasmid harboring metK in order to produce more SAM in L. lactis.
Figure. 2 Enzymatic reaction catalyzed by metK.
Construction and validation of plasmid pNZ-metK
Gene metK was amplified from genomic DNA of Lactococcus lactis NZ9000 and cut with restriction                         enzyme Hind III and KpnI, and ligased with plasmid pNZ8148 cut with the same enzyme. Then the                         ligation product was transferred to E.coli and spread on plates containing 10 mg/L                         chloramphenicol.
Colonies on the plates were randomly picked and inoculated in 1ml LB medium for 3 hours at 37℃,                         200 rpm. 1 μl culture were added to the PCR system as template. As shown in Figure. 3, all the                         picked colonies had gene metK, illustrating that the plasmid pNZ-metK was successfully                         constructed.
Figure. 3 Validation of plasmid pNZ-metK. M represented marker. 1-5 represented three randomly                         picked colonies.
Protein Analysis
After transferring the plasmid pNZ-metK to L. lactis NZ9000, SDS-PAGE was performed to detect                         the protein expression level of metK gene. The cells were washed twice with 0.1 M PBS after                         centrifugation. Crude protein was extracted through cell breaking using ultrasonication and                         centrifugation. Then the supernatant of the samples were used to analysis the protein                         expression. As shown in Figure. 4, expected bands of the MetK protein were observed on the gel.                         Recombinant L. lactis containing pNZ-metK induced with different nisin concentration showed                         higher expression of MetK protein.
Figure. 4 SDS PAGE validation of gene metK expression in L. lactis. M represented marker. WT                         represented L. lactis NZ9000. 1-3 represented L. lactis/pNZ-metK induced with 100 50 and 20                         ng/ml nisin.
Validation of S-adenosyl-methionine (SAM) by HPLC analysis
To confirm the synthetic S-adenosyl-methionine in L. lactis/pNZ-metK, HPLC was performed to                         analyze the extracts from the strain. S-adenosyl-methionine was identified on the basis of                         retention times related to standard sample. According to the retention time of standard                         S-adenosyl-methionine sample, it can be confirmed that S-adenosyl-methionine was synthesized                         more in L. lactis/pNZ-metK than original L. lactis NZ9000.
Figure. 5 Validation of S-adenosyl-methionine (SAM) by HPLC.
-- Part:BBa_K2856003
Modification of cell wall anchor domain-containing protein (CwaA)
BBa_K2856003 harbors an adhesion-associated protein called cwaA. The cwaA encodes a protein                         containing multiple domains, including five cell wall surface anchor repeat domains and an                         LPxTG-like cell wall anchor motif.
Figure. 1 Schematic diagram of adhesion factor module.
Usage and Biology
The cwaA gene encodes 923 amino acids with a predicted molecular weight of 93.7 kD. The C                         terminus of CwaA contains an LPQTDE (LPxTG-like cell wall anchoring) motif belonging to the                         gram-positive LPxTG anchor superfamily. CwaA possesses five cell wall surface anchor repeat                         domains. The LPxTG-like motif and three of the five cell wall surface anchor repeat domains.                         Therefore, CwaA is a cell wall-anchored protein. The specific hit domains of CwaA also included                         epiglycanin (tandem-repeating region of mucin, pfam05647), OmpC (outer membrane protein,                         COG3203), PT (the tetrapeptide XPTX repeat, pfam04886) and BF2867_like_N (N-terminal domain                         found in Bacteroides fragilis Nctc 9343 BF2867 and related proteins, cd13120), probably with a                         role in cell adhesion
Figure. 2 Enzymatic reaction catalyzed by metK.
Construction and validation of plasmid pNZ-cwaA
Gene cwaA is from genomic DNA of Lactobacillus plantarum NL42 .In order to remove some illegal                         restriction enzyme sites and add a HIS tag, the gene cwaA was synthesized by Shanghai Generay                         Biotech (Shanghai, China). The gene cut with restriction enzyme Hind III and NcoI was ligased                         with plasmid pNZ8148 cut with the same enzyme. Then the ligation product was transferred to                         E.coli and spread on plates containing 10 mg/L chloramphenicol. Colonies on the plates were                         randomly picked and inoculated in 1ml LB medium for 3 hours at 37℃, 200 rpm. 1 μl culture were                         added to the PCR system as template. As shown in Figure. 3, all the picked colonies had the                         brand of cwaA, illustrating that the plasmid pNZ-cwaA was successfully constructed.
Figure. 3 Validation of plasmid pNZ-cwaA. M represented marker. 1-4 represented three randomly                         picked colonies.
Validation of self-aggregation of the recombinant L.                         lactis/pNZ-cwaA
To evaluate whether CwaA was effective for adhesion, we performed self-aggregation assays. The                         cells were collected and washed twice with o.1M PBS (pH=7.0).Then the cells were resuspended in                         4ml PBS buffer with an initial OD600=0.5(A0) and the final OD600(At) of the upper most layer                         was measured after 4 hour cultivation in 30℃ without shaking.
Self-aggregation (%)=100-(At/A0)X100
Figure. 4 Self-aggregation value of L. lactis NZ9000 and L. lactis/pNZ-cwaA.
Composite parts
Plasmid pNZ-GM
Plasmid pNZ-GM harbors gene gshF and gene metK. Combing the modules in one plasmid to validate                         the function of simultaneously expression of protein GshF and MetK.
Construction of plasmid pNZ-GM
Gene gshF and metK were expressed in tandem and controlled by the same inducible promoter                         PnisA. Gibson assembly method was used to join gshF and metK to form pNZ-GM and then                         transferred to E.coli DH5α. RBS site was added in front of gene metK.
Validation of plasmid pNZ-cwaA
Colonies on the plates were randomly picked and inoculated in 1ml LB medium for 3 hours at 37℃,                         200 rpm. 1 μl culture were added to the PCR system as template. The length of the fragment                         obtained by PCR using the primers for verification was 3494 bp theoretically if plasmid pNZ-GM                         was constructed successfully. As shown in Figure. 2, all the picked colonies had gene gshF and                         metK, illustrating that the plasmid pNZ-GM was successfully constructed.
Figure. 1 Validation of plasmid pNZ-GM. M represented marker. 1 to 6 represented randomly                         picked colonies.
Functional characterization of plasmid pNZ-GM
After we introduced the plasmid pNZ-GM to L. lactis NZ9000, we tested the GSH and SAM                         production. Compared with wild type L. lactis NZ9000, the engineered strain L.lactis/pNZ-GM                         produced more GSH and SAM.
Figure. 2 GSH and SAM production of strain L. lactis NZ9000 and L. lactis/pNZ-GM. Asterisk                         represented none production.
Plasmid pNZ-GMcA
Plasmid pNZ-GMcA harbors gene gshF, gene metK and cwaA. Combing three modules in one plasmid in                         order to improve the colonization ability of Lactococcus lactis while producing GSH and SAM.
Construction of plasmid pNZ-GMcA
Plasmid pNZ-GM and adhesion module amplified from plasmid pNZ-cwaA were cut by Pst I and Sal I,                         then were linked with T4 DNA ligase and transferred to E.coli DH5α.
Validation of plasmid pNZ-GMcA
Colonies on the plates were randomly picked and inoculated in 1ml LB medium for 3 hours at 37℃,                         200 rpm. 1 μl cultures were added to the PCR system as template. The length of the fragment 1                         obtained by PCR for verification was 3760 bp theoretically and fragment 2 was 3079 bp if                         plasmid pNZ-GM was constructed successfully. As shown in Figure. 1, all the picked colonies                         contains gene gshF, metK and cwaA, illustrating that the plasmid pNZ-GMcA was successfully                         constructed.
Figure. 1 Validation of plasmid pNZ-GMcA. M represented marker. gm-1 to gm-5 and cA-1 to cA-5                         all represented the same five randomly picked colonies.
Functional characterization of plasmid pNZ-GMcA
After we introduced the plasmid pNZ-GMcA to L. lactis NZ9000, we tested the GSH and SAM                         production and self-aggregation value to evaluate adhesivity of the strain. As depicted in                         Figure. 2, compared with wild type L. lactis NZ9000, the engineered strain L.lactis/pNZ-GMcA                         produced more GSH and SAM and showed better adhesivity.
Figure. 2 GSH and SAM production and self-aggregation value of strain L. lactis NZ9000 and L.                         lactis/pNZ-GMcA. Asterisk represented none production.
