"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/24 June 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
JUNE 24, 2009
CAROL
Sequencing
Sent both mutated and non-mutated luxCDABE for sequencing using Sp6 and T7 primers. The mutation site was sequenced using a primer that was designed to detect the mutated site.
CHINMOYEE
More Research into Stochasticity
Bridging the Gap between Stochastic and Deterministic Regimes in the Kinetic Simulations of the Biochemical Reaction Networks
Efficient Exact Stochastic Simulation of Chemical Systems with Many Species and Many
Channels
Modelling and Simulation of IntraCellular
Dynamics: Choosing an Appropriate Framework
EMILY
Going Back to Biobricking LuxOD47E
Restriction digest products were mixed with 10x Orange Dye and visualized on a 1% agarose gel with Generuler 1.0 KB+ DNA Ladder and uncut plasmid of the same colonies as a negative conttrol.  Gel photo is below.
Lanes 1,3,5 and 7 are the restriction digest products for colonies 1-4 and lanes 2,4,6 and 8 are uncut plasmid for colonies 1-4.
Analysis: There seems to have been no cutting with NotI as the band sin the digested lanes are the same as the bands in the uncut lanes.  There are also some strange bands at about 12 KB which are much too big to be LuxOD47E.  It looks like my gene may be lacking the NotI restriction sites, which indicates that the BBK Ampliification Gradient PCR to Biobrick my gene if interest may not have been successful.  We will proceed by returning to this PCR and repeating it.
Repeated PCR, see protocol on June 6th 2009.
PCR Products were visualized on a 1% agarose gel run at 120 V with 1.0 KB GeneRuler DNA Ladder.  See gel photo below.
Lanes 1-12 contain LuxOD47E in pCR2.1-TOPO vector.  Lane 13 is a negative control with only Mastermix + ddH2O.
From this gel, it looks like our gene if interest, LuxOD47E is there as we see bads around 1.4 KB, as expected (gene is 1362 base pairs).  We will proceed by isoltaing plasmid from the PCR product and performing a NotI Verification digest.
So I pooled the DNA from the BBK amplification PCR.  Lanes: 2,3,9,11 and 12 went into tube 1 and lanes 4,5,6,7 and 8 went into tube 2.  I did a PCR product isolation (Sigma) according to the manufacturer's instructions and took the concentration using the Nanodrop Spectrophotometer.
JAMIE
Construction of B0015-R0040-luxOU-B0015
Final construction was carried out using BBa_B0015-BBa_R0040 C1 in psBAK3 and luxOU(XbaI)-BBa_B0015 in psB1AC3 C4. B0015-R0040 was digested using EcoRI with SpeI and luxOU(XbaI)-B0015 was digested using EcoRI with XbaI (Invitrogen, CA). Standard phosphatase treatment, ligation and transformation.
JEREMY
Sending LuxPQ in psB1AK3 for DNA Sequencing
Purpose: To verify presence of LuxPQ in psB1AK3. LuxPQ-C4-2 was sent to University of Calgary DNA Sequencing with the BBK-CP-F/R primers.
KEVIN
Transformation
In order to verify the competency of KT1144 cells, I have transformed pBluescript into them. Competent KT1144 cells are needed to test the mutant circuits.
PRIMA
Bacterial Transformation & Response circuit
I shadowed Jeremy today. He was transforming the BBK ligated vector to competent cells (Top 10). I'm not sure exactly what he was transforming but I just hung out to learn the procedures and purposes of transformation.
Jamie and I were working on possible applications of the response circuit. We proposed that we should have bacteria produce wintergreen smell in response to stinky odor. The Wintergreen gene already exists in the registry so we could use a sulfur sensitive promoter in bacteria which would bind to sigma 54 and together bind to Pqrr4 promoter. Since Pqrr4 initiates the response circuit, further down the line, we could add the Winter green gene which would then be activated as sigma 54 latches on the Pqrr4.
I followed up with a few companies. BioAlberta has generously donated $1000.00 to the team. I wrote a Thank-You letter on behalf of the team and sent it to the President of BioAlberta.
VICKI
A sexy circuit discussion
Fahd and I worked together to propose a possible response circuit. We investigated the possiblity of using parts from the registry that can form a bioplastic, which could be useful for encapsulating cells that need to be inserted into the human body to - say - degrade a biofilm, but need to resist an immune and inflammatory response. Some considerations for our next discussion on this front will be whether or not this part works, and if so, what form the bioplastic comes in.
Retrieved from " http://2009.igem.org/Team:Calgary/24_June_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
