"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/5 August 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
AUGUST 5, 2009
CAROL
Verification of the pCS26 vector and Bake Sale Help
helped Jamie out with verifying the surette (pCS26) vector. Unfortunately, the gel only shows results for the uncut plasmid and nothing for the plasmid cut with XhoI and BamHI. Jamie went ahead and cut the vector with NotI. We will knoe the results tomorrow morning
helped the team out with the Bake Sale
CHINMOYEE
Bake Sale day
This was my first bake sale.I made 3 cakes for sale: Marble  and two choclate . The sale was a sucess  . We raised more money than the last sale.
On the Modelling side :  We had a modelling meeting in the morning . Here we made a clear plan on what needs to be done . A tentative timeline was made . Work was designated .  More planning was done for the presentation on Friday.
EMILY
Bake Sale Fun
Today I spent some time updating the Sponsors page on the Wiki, adding Nexen's logo and description to our sponsor of the month section.  I also added Sigma Aldrich's logo and a link to their website to the home page of our Wiki.
The majority of today was spent preparing for and running the bake sale where we made record profits!  Some of us also met with a graduate student from another lab to inquire about the isolation of AI-2 from Salmonella which will be required for testing in a couple of weeks.
FAHD
Marketing for August 5th (2009-08-05)
Today, I concentrated my energies on fundraising for our research project and finding potential media partners.
For the fundraising part, we conducted a bake sale from which, we raised $494.18 CDN Dollars. Every team member had contributed to the endeavour. I also contacted some airline agencies to get a quote for our trip to MIT, Boston.
I also contacted our University of Calgary Community Radio Station called CJSW 90.9 for doing a media coverage for our research project this year. I will do a follow-up later this week.
JAMIE
Verification of Surette vector
Ran digested Surette vector and undigested product on 0.7% agarose gel. No bands in digested product?
Set up overnight NotI digest of pCS26.
Spoke with Margot from Surette lab about AI-2 bioassay.
JEREMY
Verification of cl lambda in psB2K3 and PQ-B-R-OU-B in psB1AC3
Overnight cultures were prepared of cl lambda and sequenced PQ-B-R-OU-B. Different concentrations of IPTG (0.5mM, 1mM, 5mM) were added to the cultures of cl lambda at the mid-exponential phase of growth. Plasmid was isolated using the QIAprep Spin MiniPrep Kit, and the concentration and purity of plasmid was measured using the NanoDrop Spectrophotometer. Restriction digest was then performed cutting with XbaI/PstI and EcoRI. The restriction digest was also set up for an isolated plasmid of cl lambda from July 17, 2009. The figure shows the results:
Noteworthy sizes (all are verified by the gel above):
-cl lambda gene: 987 bp
-psB2K3 for cl lambda: 4.4kb
-PQ-B-R-OU-B: 6.1kb
-psB1AC3: 3.1kb
KATIE
Cleaning Up the Virtual Lab
Most of the work done in the virtual lab today was tidying up some scripts and making minor adjustments and improvements. I was able to complete the third construction zone and now I have returned to my DNA replication animation and I am now working on:
Determining the positions the separate places the pieces have to be situated as well as making sure each piece knows where the others are even if the location of the pieces, especially the DNA strands are moved to a different area.
Setting the positions of gyrase, helicase and primase to their respective places. Presently, the primase can detect a piece of the DNA strand and set its position to it and then rezzes an RNA primer and then obtains its next position.
I took some snapshots around the lab today for the presentation on Friday and I made a rough outline for the PowerPoint slides I will be working on tonight and sometime tomorrow for the presentation. I have decided to make some short video clips as well in order to better demonstrate what can be done in the lab as it is explained on Friday.
The incubator has been added to the movement of the insert tube during restriction digest and the phosphatase treatment station no longer moves on its own. It acts like restriction digest so avatars must chose what the tube needs to be sent to within the lab. I also changed sequencing script and made a list of the note cards that still need to be added to the station’s inventory in order for all items that can be dropped into the sequencing station have a product that may be given in return.
KEVIN
1. Plasmid Isolation of Pqrr4+B0034+K082003
Pqrr4+B0034+K082003 was isolated in order to verify its presence using Restriction digest and Sequencing later on. The product's purity and concentration were measured using Nanodrop utility and spectrophotometer.
Plasmid
260/280
260/230
Concentration [ng/μL]
Pqrr4+B0034+K082003 C9 #1
2.02
2.72
158.0
Pqrr4+B0034+K082003 C9 #2
2.06
3.61
160.4
2. Restriction digest of Pqrr4+B0034+K082003
Restriction digest on Pqrr4+B0034+K082003 to verify its presence. This is our reporter circuit. The product was ran on 1.5% gel at 90V.
The bands in Pqrr4+B0034+K082003 lanes seems to match the size I expected, which is around 1043bp; thus I can move forward with sequencing tomorrow.
3. Plasmid Switch of Pqrr4+I13500
Yesterday, I have done an overnight digestion of Pqrr4+I13500 and Q04510 (inverter) part in pSB2K3 vector. Today, with this product, I have performed phophatase treatment on Q04510 and ligated them together. This product was then transformed into TOP10 cells, and is now growing in the incubator at 37˚C.
PATRICK
Worked on Biobricker, Wrote a blog post about it.
The blog post at Synthetic Blogology covers what I've done today fairly well, http://igemcalgary.blogspot.com/2009/08/second-life-belated-update.html
In short: I completed the second part of the Biobricker I talked about last Friday, the portion of the code that actually assembles Biobricks together into devices. Remaining to do on the Biobricker is in world testing, and to tie up a few loose ends in the code.
I keep a little to do list of elements I would like to have done for the end of this iGEM season, specifically with the biobrick simulator:
level_select
home
parts
interaction_check
biobricker
cofactors for dna binding proteins
The level select will be one of the last to be finished, since choosing a level and using the parts provided by it will require most of the other features to be finished. The same goes with the Home screen, which will display basic information about the current level and serve as a launching point to the other tools available in the simulator. The Parts screen will let you create objects from an inventory of prebuilt devices, plus regulatory proteins and cofactors; I've already implemented an inventory like system in the Biobricker though, so this will be a copy and paste job.
The Biobricker and Interaction Checker are easily the most complicated remaining parts, so I'm working on them first! The Biobricker, as explained, permits assembly of custom devices in world. The Interaction Checker will scan your nearby DNA, proteins, and cofactors, looking for possible binding interactions between them. Basically, if it could happen, it should. This is a tool to prevent users from 'forgetting' the way a system actually would work in the current state, and may be used to determine whether you've understood the system enough to progress to the next level! The checker will be intelligent enough to detect whether a gene is on or off, and hopefully work intelligently with multiple copies of a single coding sequence in different activation states. Of course, the Interaction Checker won't be able to help as much with systems that have no steady state such as the Repressilator... there's always another valid interaction.
Last of all, I want to implement DNA binding proteins that can accept multiple cofactors, mostly so that I can have LacI behave approximately like the real thing with regards to Glucose, Lactose, and IPTG (but also TetR with Tetracycline, AtC). The implementation I have in mind is reprogrammable though, and has no upper bound on the number of cofactors per protein. So if you want to invent a monster with three allosteric inhibitors and five inducers, you'll be able to do it!
PRIMA
Bake Sale Trinity
I emailed a few companies to discuss meeting times. Then i helped to organize the bake sale and sold home baked treats until 2:30pm. We also went out to different parts of the building to sell the left over goods. The team raised a net of $494.18 profit!!!  Next, I worked on the powerpoint presentation slides for marketing and discussed lab with my peers and supervisors. I am doing a construction of aiiA tp the response circuit.  Tomorrow, i'll follow up with some of the older companies regarding sponsorship.
STEFAN
Building the squid
Nothing too exciting today, just constructed my proposed squid at each station, giving them names, choosing textures and animating them. Nothing too technical but animation is quite time consuming, especially when things go wrong and I have to restart.
I also helped out a little bit at the bake sale. Mostly in setting up.
VICKI
A miscellany of engineering and modelling
Editing of APEGGA article
I spent a few hours re-working the promotion article that we will be submitting to APEGGA for publication in September. My focus was primarily on improving the flow of the article and bringing in specific examples that are tangible to people with little to no background in the field.
Another modelling meeting
Carol, Chinee and I came up with a list of what we will need from our lab/modelling teams to collect data for our characterisation/simulation efforts; when we need this done; and who will be responsible for each part of the project. Carol has the hand-written version and will be typing it up and circulating it shortly.
Finding specific examples of where characterisation would be useful
This part was difficult, and after talking to Sonja, we decided a simple explanation of how we need to know how our circuit behaves in the lab so that we can better predict its behaviour out of the lab will suffice for presentation purposes. From the transfer function/static performance, we get a sense of the equilibrium behaviour at different input AI-2 concentrations. From the dynamic performance, we can figure out the response time and compare it to that of an entire system where our part in question is being used as a component.
We also talked about how to account for the effects of cell division, which could confound our plate readings. First, we decided it was best to characterise in the mid-exponential phase (as opposed to plateau), as dead cells would cause unknown effects that could be very difficult to explain. To account for cell division, we plan to take baseline measurements of the mutant + reporter circuits, as well as a baseline of signalling + reporter, over a long period of time. This will give us a sense of how important cell division is in the interpretation of our results.
Retrieved from " http://2009.igem.org/Team:Calgary/5_August_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
