"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/27 May 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
MAY 27, 2009
CAROL
Modeling Readings
Focused on how part BBa_F2620 was characterized and wrote up simple protocols to test our circuit when the circuit is completed.
Some of the characteristics include: gene stability, response time, dynamic response and specificity.
Started reading up on Simbiology and what kind of functionality the platform provides.
No experiments performed.
CHINMOYEE
WHMIS Training
Attended WHMIS training today.
EMILY
Restriction Digest and PCR with gene-specific primers of LuxOD47E in the psB1AC3 vector
Objective: to verify that the gene of interest LuxOD47E has been biobricked (contains the biobrick restriction sites).
Restriction Digest
Protocol: Digest 10 μL LuxOD47E BBK (C1- [60.3 ng/μL] and C2- [92.9 ng/μL]) with 1μL Not1 restriction enzyme (Invitrogen, CA), 2 μL REact 3 Buffer and 7 μL ddH20.  Left for two hours in 37ºC water bath.  Then heat activated in a 65ºC water bath for 10 minutes.
PCR of LuxOD47E in psB1AC3 with gene-specific primers
Objective: To verify the presence of LuxOD47E in the psB1AC3 vector.  If our gene of interest is biobricked and in the psB1AC3 vector, than we can proceed with construction.
Protocol: Made 9x mastermix with 45 μL 10x PCR Buffer, minus Mg2+, 9 μL 10 mM dNTPs, 13.5  μL 50 mM MgCl2, 1.8 μL p Taq DNA Polymerase, 9  μL LuxO-R primer, 9  μL LuxO-F primer and 344.7  μL ddH2O.  Split Mastermix into the  tubes, 49  μL in each.  Put 1  μL BBK LuxOD47E C1 DNA template into 2 tubes, 1 μL BBK LuxOD47E C2 DNA template into two tubes, 1 μL LuxOD47E TOPO 2 tubes (positive contorol) and 1  μL ddH2O into two tubes for a negative control.
Ran PCR with the folowing cycling conditions: 94°C for 3 minutes, 36x (94°C for 30 s, 53°C for 45 s and 72°C for 90 s), 72°C for 10 minutes.  Held at 4°C.
Results: Ran both the restriction digest and the PCR product on a 1% agarose gel at 120 V with 2 μL Orange 10X dye.  Ran the restriction digest with uncut plasmid as  positive control.
Lane 1- BBK LuxOD47E  C1 digested Lane 2- C1 uncut Lane 3- BBK LuxOD47E C2 digested Lane 4- C2 uncut Lane 5- LuxOD47E BBK- C1 Lane 6- LuxOD47E BBK- C2 Lane 7- LuxOD47E TOPO (Positive Control) Lane 8- Negative Control
Analysis:  Looking at the PCR Products, it looks extremely likely that the gene of interest, LuxOD47E, is in the psB1ACM vector, as the bands in the two BBK lanes match up with the band in the TOPO positive control lane.  We will proceed to DNA sequencing for a fnal verification.  For the restriction digest, it does not look like the DNA was cut as the bands in the cut lanes are identical to the bands in the uncut lanes for all colonies.  This may have been because the reaction was not run long enough, there was a problem with the enzymes or there is a problem with the restriction sites themselves.
FAHD
Marketing and Campus Fair Meeting, May 27th 2009
Today I contacted a couple of Oil and Gas Companies and Research and Development companies. I also attended the Campus Fair Meeting that day. The Campus Fair will be held on Saturday June 5th.
JAMIE
Restriction Digest and PCR of biobricked luxOU
Purpose: Verification of last year's parts submitted to the registry.
Digested approximately 1500ng of DNA with NotI for 2 hours and ran the product on a 1.0% agarose gel. pTAq (Invitrogen, #436557) was used in the PCR with gene specific primers.  Expected band size for the PCR was 2.0kb.
JEREMY
Verifying the presence of luxO D47A in the psB1AC3 vector
The purpose of today’s experiments is to verify the presence of luxO D47A in the psB1AC3 vector. A restriction digest (RD) with NotI was set up for 2 hours at 37ºC, with two separate colonies of isolated luxOD47A in psB1AC3 (from May 26). A PCR using LuxO F/R primers (gene specific) was set up using platinum Taq Polymerase (Invitrogen, CA). according to the specifications of the manufacturer. Template from colonies 1 and 2 of luxOD47A in psB1AC3 were used. Conditions were as follows: 94ºC for 3 minutes; 36 X (94ºC for 30 seconds; 53ºC for 30 seconds; 72ºC for 2 minutes); 72ºC for 10 minutes; held at 4ºC. The products from the PCR and the RD were then run on a 1% agarose gel (see below).
For the RD, we should see two bands for the “cut”: a band at ~3kb for the psB1AC3 vector, and a band at ~1.4kb for luxO D47A. However, the “cut” lanes show that NotI was unable to cut out any insert, suggesting the absence of luxO D47A. However, if we look to the PCR for colony 1 and 2, we do see the expected band size of ~1.4kb for the luxO D47A gene. The band size lines up nicely with the positive control, Lux O D47A in TOPO T/A. However, the negative control lane shows contamination.
KATIE
Invisibility
One set back to this method is that any object or texture that the original avatar is wearing that does not have the alpha value set to the float value 1.0 will not be concealed by the invisible primitive. It may be easier to place the original avatar underground (might just be a fluke, more experimentation would be required so it might be returned to if we are not impressed with the use of the sculpted primitives alone for transforming avatars).
I will be gone for four days, but I will send the few notecards I have started to make to the rest of the group so they can be critiqued/criticized and improved further early next week and continue to add to the weekly progress reports to keep track of group activities and conversations.
KEVIN
Verification of Pqrr4 for Reporter circuit
Restriction Digest To verify the isolated Pqrr4, restriction digest was done with NotI (invitrogen, Lot 2803027) enzymes, React Buffer 3, and GeneRuler SM1333 ladder.
Polymerase Chain Reaction (PCR) The Pqrr4 was then amplified using   Polymerase Chain Reaction (PCR) from biobrick AC (pSB1AC3) using primers Pqrr4 F/R and platinum Thermus aquaticus (pTaq; invitrogen) according to the specifications of the manufacturer. The following cycling conditions were used: 94°C for 3 min; 36x (94°C for 30s; 53°C for 45s; 72°C for 1 min;) 72°C for 10 min; held at 4°C.
Figure 1. The Restriction digest and PCR of Pqrr4
Sequencing Lastly, the Colony 1, being Pqrr4 BBK Green, was sent for sequencing with Pqrr4 F/R primers to the University of Calgary DNA Sequencing Facility (University Core DNA Services, Calgary, Alberta, Canada).
Legend:
Vector contamination
Biobrck restriction sites
Part of interest, in this case, Pqrr4
result
ATAGGCGTATCACGAGGCAGAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTG
GAATTCGCGGCCGCTTCTAGA GTATCAGCAAAAACACTACGGTGGATAATCAGTAAAACCATGAAACTAGAGCCCCGCACACTTGCGGGGCTTTTTAATTTTGAAT TTCTTTCTTATTAAAACGCCATTTTTCTGATAAATGTATTAGTAGCAATGCGCATGGTGGCATATTTGCATCATTTTGCATTTTG CAAATGCGATTTGCAAAATGCGTGCTCAATAAAGCACCAATATGCATCAGGATCGAAGAAAAAAGGCGTTTTTAAAAGTTGGCAC GCATCGTGCTTTATACAGATACTAGTAGCGGCCGCT GCAG
TCCGGCAAAAAAGGGCAAGGTGTCACCACCCTGCCCTTTTTCTTTAAAACCGAAAAGATTACTTCGCGT TATGCAGGCTTCCTCGCTCACTGACTCGCTGCGCTCGGTCGTTCGGCTGCGGCGAGCGGTATCAGCTCA CTCAAAGGCGGTAATACGGTTATCCACAGAATCAGGGGATAACGCAGGAAAGAACATGTGAGCAAAAGG CCAGCAAAAGGCCAGGAACCGTAAAAAGGCCGCGTTGCTGGCGTTTTTCCACAGGCTCCGCCCCCCTGA CGAGCATCACAAAAATCGACGCTCAAGTCAGAGGTGGCGAAACCCGACAGGACTATAAAGATACCAGGC GTTTCCCCCTGGAAGCTCCCTCGTGCGCTCTCCTGTTCCGACCCTGCCGCTTACCGGATACCTGTCCGC CTTTCTCCCTTCGGGAAGCGTGGCGCTTTCTCATAGCTCACGCTGTAGGTATCTCAGTTCGGTGTAGGT CGTTCGCTCCAAGCTGGGCTGTGTGCACGAACCCCCCGTTCAGCCCGACCGCTGCGCCTTATCCGGTAA CTATCGTCTTGAGTCCAACCCGGTAAGACACGACTTATCGCCACTGGCAGCAGCCACTGGTAACAGGATTAGCAGA
The above Pqrr4 sequence matches 100% with the expected Pqrr4 sequence; thus I can continue.
MANDY
Second Life Island!
While exploring more of Second Life and the scripting areas (my learning of scripting is so slow) Stefan and I checked out the map and were able to find... OUR ISLAND. Since Dr. Jacob is away, we cannot gain access to be able to build on it, but we were able to 'set foot' on it for the first time, so to speak...
PRIMA
Dragon's Den presentation to Class
Vicky, Jeremy and I told the team about our presentation at the Dragon's Den, what we learned, what we need to improve on and how we should market our project. Dragon's Den was a great experience for us to polish our presentation skills and brainstorm ideas on how to improve marketing strategies.
The good news is that the all 5 dragons thought that our Business plan was the best out of all 3 Alberta Universities. Good job team!
VICKI
NotI restriction digest of LuxPQ BBk on psB1AC3
Purpose:
To verify that a gene that is the same size as LuxPQ can be excised from the plasmids isolated in the miniprep on May 26 by using NotI restriction enzymes
Materials and methods:
Enough volume for around 1ug of DNA was taken from the samples measured in the previous step (4uL from each sample). 2 uL of REact buffer 3 (Invitrogen) and 1 uL of NotI enzyme (Invitrogen) were added. 13 uL of ddHOH was added to bring the total tube volume up to 20 uL. Tubes were incubated in the water bath for 2.5 hours and then left at room temperature for a few more hours before being placed in the freezer.
Results:
These are displayed and discussed on the gel that was run on May 28.
PCR amplification of LuxPQ BBk
Purpose:
To verify that the LuxPQ BBk samples actually consist of LuxPQ BBk
Materials and methods:
The procedure is similar to that from May 20. Platinum Taq polymerase was used instead of Taq polymerase, as a higher degree of fidelity was desired in light of poor results last week (for other people). Platinum Taq has an antibody that prevents the polymerase from acting until the initiation temperature reaches 94 degrees C. This makes the system more stringent and prevents polymerase activity at lower temperatures, such as the 56 degree C environment that is critical for annealing.
TOPO clones of LuxPQ are used as a positive control. Since the gene of interest (LuxPQ) is purportedly present in these (according to sequencing results from last year), you would expect to see a band at 3855 bp. If the BBk bands are also this size, it encourages our presumption that they are indeed LuxPQ BBk and not some other gene that can’t be amplified by primers specific to LuxPQ. ddHOH is our negative control.
Note: the BBk and TOPO LuxPQ are very concentrated. More dilute samples of each were made (100 ng/uL) so that an appropriate volume with lower error could be pitetted into the PCR tubes.
49 uL of master mix was pipetted into each tube, followed by the BBk samples, the TOPO samples and the negative control, according to the following key:
Tubes 1 and 2: LuxPQ c1 BBk
Tubes 3 and 4: LuxPQ c2 BBk
Tubes 5 and 6: Negative control (ddHOH)
Tubes 7 and 8: Positive control (TOPO plasmid)
Template: LuxPQ BBk
Forward primer: LuxPQ BBk forward primer (Tm = 60 degrees C)
Reverse primer: LuxPQ BBk reverse primer (Tm = 60 degrees C)
Master Mix contents (for 9 tubes)
10X PCR Buffer minus Mg 2+ (45 uL)
10mM dNTPs mixture (9 uL)
50mM MgCl2 (13.5 uL)
Forward primer [10 uM] (9 uL)
Reverse primer [10 uM] (9 uL)
pTaq DNA polymerase [5 units/uL] (1.8 uL)
Autoclaved distilled HOH (353.7 uL)
TOTAL: 441 uL of Master Mix, of which 49 uL were spread across 8 tubes
PCR steps:
Denaturation: 94 degrees C, 3 minutes
Amplification: 36 cycles of:
Denaturation (94 degrees C, 30 seconds);
Annealing (53 degrees C, 45 seconds);
Extension (72 degrees C, 4 minutes)}
Final extension: 72 degrees C, 10 minutes
Hold temperature: 4 degrees C
Results:
Like the restriction digest, these will be visualised on a gel tomorrow.
Retrieved from " http://2009.igem.org/Team:Calgary/27_May_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
