<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Lab_Protocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Calgary/Lab/Protocol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Lab/Protocol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Lab/Protocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Lab/Protocol&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Lab/Protocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Lab/Protocol</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>University of Calgary</TITLE></P><DIV class="header"><DIV class="title">
UNIVERSITY OF CALGARY
</DIV></DIV><DIV class="menu"><DIV class="outer"><UL id="nav"><LI><P><A href="https://2009.igem.org/Team:Calgary">  HOME</A></P></LI><LI><P><A href="https://2009.igem.org/Team:Calgary/Team">  TEAM</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team#under">Undergraduates</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team#facil">Facilitators</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team#uni">Our University</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Acknowledgments">Acknowledgments</A></LI></UL><LI><P><A href="#">  PROJECT</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/Project">Overall Project</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Parts_Notebook">Notebook &amp; Parts</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab">Lab</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Modelling">Modelling</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Second_Life">Second Life</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Human_Practices">Human Practices</A></LI><LI><P><A href="#">  EXTRA</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/News">News</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Blog">Synthetic BLOGology</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Gallery">Gallery</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Collaborations">Collaborations</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Mascot">Mr. Super D-Lux</A></LI><LI class="last"><P><A href="https://2009.igem.org/Team:Calgary/Sponsors">  SPONSORS</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#ps">Platinum Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#gs">Gold Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#ss">Silver Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#bs">Bronze Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#cp">Community Partners</A></LI></DIV></DIV><P><TABLE cellspacing="0"><TBODY><TR><DIV id="leftmenu"><TD width="210" rowspan="34" bgcolor="#414141" valign="top"><DIV class="heading">LAB INDEX</DIV><DIV class="desc"><A href="https://2009.igem.org/Team:Calgary/Lab"><B>Overview</B></A><A href="https://2009.igem.org/Team:Calgary/Lab/Blog"><B>Updates</B></A>
Circuits
<UL><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Signalling">
Signalling Circuit</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Reporter">
Reporter Circuit</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Mutant">
Mutant Circuits</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Response">
Response Circuit</A></LI></UL><A href="https://2009.igem.org/Team:Calgary/Lab/Quorum_Sensing"><B>Quorum Sensing</B></A><A href="https://2009.igem.org/Team:Calgary/Lab/Chassis_Selection"><B>Chassis Selection</B></A><A href="https://2009.igem.org/Team:Calgary/Lab/Protocol"><B>Protocol &amp; Safety</B></A><B><A href="https://2009.igem.org/Team:Calgary/Parts_Notebook">Notebook &amp; Parts</A></B></DIV></TD></DIV></TR><TR><TD><DIV class="heading">
Lab Protocols
</DIV><DIV class="desc" width="750">

The following are the protocols that are used in the lab this year. Some modifications may have occurred during this summer. If there are any changes, those changes can be found in the notebook. 

All team members of this year's iGEM team have successfully completed WHMIS and Biosafety Training. 

</DIV></TD></TR><TR><TD valign="top"><UL id="sub"><LI><A href="#rehydration" title="Protocol for rehydrating plasmid from Registry"> Rehydration </A></LI><LI><A href="#pcr" title="Colony Polymerase Chain Reaction (PCR)"> Colony PCR </A></LI><LI><A href="#pcr" title="Gradient PCR"> Gradient PCR </A></LI><LI><A href="#transformation" title="Bacteria Transformation"> Bacterial Transformation </A></LI></UL></TD><TD valign="top"><UL id="sub"><LI><A href="#ct" title="Construction Technique"> Construction Technique</A></LI><LI><A href="#pp" title="Plasmid Isolation"> Plasmid Prep </A></LI><LI><A href="#pp2" title="Plasmid Isolation 2"> Plasmid Prep 2 </A></LI><LI><A href="#ap" title="Antarctic Phosphatase"> Vector Dephosphorylation </A></LI></UL></TD><TD valign="top"><UL id="sub"><LI><A href="#platePrep" title="Antarctic Phosphatase"> LB - Agar Plate </A></LI><LI><A href="#overnight" title="overnight"> Over Night Growth </A></LI><LI><A href="#gsp" title="Glycerol Stock"> Glycerol Stock Preparation </A></LI><LI><A href="#pcrp" title="PCR Purification"> PCR Purification </A></LI></UL></TD><TD valign="top"><UL id="sub"><LI><A href="#age" title="Agarose Gel Electrophoresis"> Agarose Gel Electrophoresis </A></LI><LI><A href="#rd" title="Restriction Digest"> Restriction Digest </A></LI><LI><A href="#lp" title="Ligation"> Ligation </A></LI><LI><A href="#mut" title="Quikchange Site Directed Mutagenesis">Site Directed Mutagenesis</A></LI></UL></TD></TR></TBODY></TABLE></P></DIV><TABLE width="700"><TBODY><TR><TD style="width:85%;"><DIV class="heading">Bacterial Transformation</DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><OL><LI> Thaw 100 μL of competent cells (per transformation) on ice just before they are needed</LI><LI> Add DNA (max 20ul) thawed cells and mix by flicking the side of the tube. Leave on ice for 30 minutes</LI><LI> Heat shock 5 minutes at 37 degrees Celsius</LI><LI> Place on ice for 5 minutes </LI><LI> Add 250ul SOC medium to each tube</LI><LI> Incubate for 30 to 60 minutes with shaking at 37 degrees Celsius. (Note that for Kanamycin containing plasmids always use one hour)</LI><LI> Spin down to remove all supernatant except approximately 100 μL</LI><LI> Plate approximately 30 μL on each of two antibiotic plates </LI><LI> Grow overnight at 37 degrees Celsius </LI></OL><P> For this protocol we used a couple of controls
  <UL><LI><B> Positive Control </B> - pBluescript in TOP10 cells on ampicillin plates </LI><LI><B> Negative Control </B> - TOP10 cells grown on ampcillin plates </LI></UL></P></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD width="85%"><DIV class="heading"> Rehydration</DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><P> Biobrick parts are shipped from the registry in a dehydrated from. As such they must be rehydrated before they can be used. </P><OL><LI> Puncture a hole through the foil with a pipette tip into the well that corresponds to the Biobrick - standard part that you want</LI><LI> Add 15 μL of diH20 (deionized water)</LI><LI> Let the water sit for 5 minutes</LI><LI>Take 2 μL DNA and transform into your desired competent cells, plate out onto a plate with the correct antibiotic and grow overnight. Your goal here is to obtain single colonies</LI></OL></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD width="85%"><DIV class="heading"><P style="font-size:14px; font-weight:bold">Taq PCR Protocol </P></DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><TABLE width="630" border="1px" style="margin-bottom:15px;"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume ( 1x )</B></TD><TD><B>Volume ( 3x )</B></TD><TD><B>Volume ( 5x )</B></TD><TD><B>Volume ( 15x )</B></TD></TR><TR><TD>Sterile H2O</TD><TD>36 μL</TD><TD>108 μL</TD><TD>180 μL</TD><TD>540 μL</TD></TR><TR><TD>10X Taq Buffer</TD><TD>5 μL</TD><TD>15 μL</TD><TD>25 μL</TD><TD>75 μL</TD></TR><TR><TD>2mM dNTPs</TD><TD>5 μL</TD><TD>15 μL</TD><TD>25 μL</TD><TD>75 μL</TD></TR><TR><TD>Forward Primer (100 ug/ul) </TD><TD>1 μL</TD><TD>3 μL</TD><TD>5 μL</TD><TD>15 μL</TD></TR><TR><TD>Reverse Primer (100 ug/ul) </TD><TD>1 μL</TD><TD>3 μL</TD><TD>5 μL</TD><TD>15 μL</TD></TR><TR><TD>50mM MgCl2</TD><TD>1.5 μL</TD><TD>4.5 μL</TD><TD>7.5 μL</TD><TD>22.5 μL</TD></TR><TR><TD>Taq Polymerase (50 ug/ul)</TD><TD>0.5 μL</TD><TD>1.5 μL</TD><TD>2.5 μL</TD><TD>7.5 μL</TD></TR></TBODY></TABLE><P><B> Thermocycler Conditions </B></P><OL><LI> 1 Cycle - 6 minutes at 95 degrees Celsius </LI><LI> 36 cycles of:
      </LI><LI> 1 minute at 95 degrees Celsius </LI><LI> 1 minute at 58 degrees Celsius ( this step done at 65 degrees Celsius for higher GC content ) </LI><LI> 1 minute at 72 degrees Celsius </LI></OL><LI> 1 Cycle - 10 minutes at 72 degrees Celsius then HOLD at 4 degrees Celsius </LI><P> Conditions were varied as needed. For example in cases of longer products all 1 minute times were increased to 1.5 or even 3 minutes</P></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD style="width:85%;"><DIV class="heading"> Plasmid Preparation Protocol </DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE>
The volumes of different solutions may vary depending on the type of kit we used. The main kits that we used this summer was ordered from Sigma and Qiagen
  <OL><LI><EM>Harvest Cells</EM></LI><LI>Pellet 5 mL of an overnight culture.</LI><LI><EM> Resuspend Cells </EM></LI><LI>Completely resuspend the bacterial pellet with 200 uL of resuspension solution. Pipette up and down to throroughly resuspend cells until homogenous. Incomplete suspensions will result in poor recovery. </LI><LI> Another rapid way to resuspend the cell pellet is to scrape teh bottoms of the microcentrifuge tubes back and forth five times across the surface of a polpropylene microcentrifuge tube storage rack with 5 X 16 holes .</LI><LI><EM>Lyse Cells</EM></LI><LI> Lyse resuspended cells by adding 200 μL of the lysis solution. Immediately mix the contents by gentle inversion (6-8 times) until the mixture becomes clear and viscous. <B> Do Not Vortex </B>. The lysis was allowed to proceed for 5 minutes before neutralization.</LI><LI><EM>Neutralize</EM></LI><LI> Precipitate the cell debris by adding 350 μL of the Neutralization/Binding solution. Gently invert the tube 4-6 times. Pellet the cell debris by centrifuging at maximum speed for 10 minutes.</LI><LI><EM> Prepare Column </EM></LI><LI> Insert a GenElute Miniprep Binding Column into a provided microcentrifuge tube. Add 500 μL of the Column Preparation Solution to each miniprep column and centrifuge at max speed for 60 seconds. Discard the flow through liquid.</LI><LI><EM>Load Cleared Lysate</EM></LI><LI> Transfer 600 μL of the cleared lysate from step 4 to the column prepared in step 5 and centrifuge at max speed for 60 seconds. Discard the flow through.</LI><LI><EM>Wash Column</EM></LI><LI>Add 750 μL of the diluted Wash Solution to the column. Centrifuge at max speed for 60 seconds. The column wash step removes residual salt and other contaminants introduced during the column load. Discard the flow through liquid and centrifuge again at maximum speed 2 minutes without any additional wash solution to remove excess alcohol.</LI><LI><EM>Elute DNA </EM></LI><LI>Transfer the column to a fresh collection tube. Add 100 μL of molecular biology reagent water to the column. Centrifuge at Max speed for 2 minutes. DNA is now present in the eluate and is ready for immediate use or storage at -20 degrees Celsius. </LI></OL></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD style="width:85%;"><DIV class="heading"> Construction Technique </DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><P> Determine the order of the two parts you will be putting together; the one in front will be referred to as the insert, while the one behind will be referred to as the vector.  Both the vector and the insert need to have their own separate tube, at least in the beginning. </P><P><EM> Restriction Digest Protocol</EM></P><P> In the Insert Tube...
  <UL><LI>600 ng of DNA (To figure out the volume, the calculation is 600 / concentration of plasmid.  This gives you volume in μL).</LI><LI>Water, so that the volume of both DNA and water in the tube is 35 μL total</LI><LI>4 μL of React 1 Buffer </LI><LI>0.5 μL of EcoR1 </LI><LI>0.5 μL of Spe1</LI></UL></P><P> In the vector Tube...
  <UL><LI>250ng of DNA (To figure out the volume, the calculation is 250 / concentration of plasmid.  This gives you volume in μL).</LI><LI>Water, so that the volume of both DNA and water in the tube is 35 μL total</LI><LI>4 μL of React 2 Buffer</LI><LI>0.5 μL of EcoR1</LI><LI>0.5 μL of Xba1</LI></UL></P><P> Put both tubes into the 37°C water bath for one hour.  After, place them into the 65°C heating block for 10 minutes.  This deactivates any enzymes in the tube (which is ok, because by now they’ve done all they need to).
    Take the insert out, and put it in a -20°C freezer. </P><P><EM> Antarctic Phosphatase Protocol</EM></P><P> To the vector tube, add 5 μL of 10x Antarctic Phosphatase Buffer, 4 μL of water, and 1 μL of Antarctic Phosphatase.  We do this to prevent the vector from closing up again without any insert.
    Put the tube into the 37°C water bath for 30 mins.  After, place it in the 65°C heating block for 10 minutes. </P><P><EM> Ligation Protocol </EM></P><P>Take the insert out of the freezer, and add 5 μL of insert and 5 μL of vector to a new tube.  Label the rest of each tube as Unligated, put the date on the tube, and stick it in the -20°C freezer incase your ligation/transformation doesn’t work.  To the single tube of 10 μL mix, add 10 μL of 2x Quick Ligase Buffer, and 1 μL of Quick Ligase.  Let this sit at room temperature for 5 minutes. </P><P> You are now done.  If you are going to transform this construction product, add all 21μL to a tube of whichever competent bacteria you're using. </P></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD style="width:85%;"><DIV class="heading"><P style="font-size:14px; font-weight:bold"> LB - Agar Plate Preparation Protocol</P></DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><OL><LI>Weigh 35g of LB-Agar powder mix per litre of media desired. One litre makes 40-50 plates</LI><LI>Select an appropriate flask; the lab autoclave will cause flasks half full and above to boil over! Use a 2L flasks for up to .5 L of media, a 4 litre flask for up to 1.5L, etc</LI><LI>Disolve LB-Agar, using water from one of the wall mounted nanopure filters. Add a stir bar and use a magnetic stirrer to speed things up </LI><LI>Cover the flask with aluminum foil, and secure the foil with autoclave tape. The foil should be somewhat loose (to avoid building pressure in the flask while sterilizing and blowing the foil off), but not so loose that lots of liquid can escape </LI><LI>Put the flask in a plastic autoclave tray, load into the autoclave, and sterilize using the 20 minute liquid program</LI><LI>Once the autoclave finishes venting (which can take twice as long as the sterilization proper), check that the foil covering is still in place. If it is not, the media is contaminated! Unload using the insulated oven gloves</LI><LI>Allow the media to cool until it can be handled without the oven mits. The cold room can be used to speed this up. Alternatively, if a large batch of media is prepared flasks may be kept hot in the prep lab water bath, to avoid all of them cooling at once. Agar polymerization cannot be reversed once it starts (and if it begins to set in the flask you're in trouble!), but media can be kept from setting further by keeping it hot.</LI><LI>Once media is cool, add other desired ingredients. Use the magnetic stirrer to mix, but do NOT add a stir bar now, or the media will be contaminated. (If one wasn't added before, you must do without.) Common additions include:
      </LI><LI>ampicillin (stock 100mg/ml, final 100ug/ml)</LI><LI>kanamycin (stock 50mg/ml, final 50ug/ml)</LI><LI>chloramphenicol (stock 50mg/ml, final 10ug/ml)</LI><LI>To achieve final concentrations, add 1mL of stock per 1L of media, except for chloramphenicol, where 0.6mL per 1L of media is added instead</LI><LI>Pour directly from the flask into sterile petri plates. Use a quick pass with a bunsen burner flame to snuff out bubbles that form during pouring. Do not subject the plate to continuous heat or the plate will melt, and the heat sensitive ingredients added in the previous step will be destroyed. Bubbles can allow cells to access nutrients without being exposed to the plate's antibiotic, and should be blown out immediately before the gel can set. It's a good idea for one person to pour while another flames bubbles. </LI><LI>Allow the plates to stand right side up overnight, or until the gel sets if they are needed sooner. Plates should be stored upside down to keep condensation from falling on the media. Store petri plates in the plastic bags they ship in, in the 4 degree cold room. </LI></OL></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD style="width:85%;"><DIV class="heading"><P style="font-size:14px; font-weight:bold"> Over Night Growth </P></DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><P><EM> Adapted from Butanerds Protocols from the University of Alberta iGEM Protocols pdf</EM></P><P> What you will need
  <UL><LI> 10mL culture tube. Use 16mm x 160mm or 16mm x 125mm</LI><LI> 5 mL LB</LI><LI> 5 uL 1000X antibiotics</LI><LI> Single colonies on a plate (best not to start an over night from a glycerol stock)</LI></UL></P><P> Protocol </P><OL><LI> Pipet 5uL 1000X antibiotic into culture tube </LI><LI> Add 5mL non-contaminated LB. Do this first. Then add antibiotic</LI><LI> Select a single colony using a sterile toothpick or flamed loop that has been cooled </LI><LI> Place toothpick or loop in culture tube and stir </LI><LI> Remove toothpick or loop and place culture tube in incubator at 37 C overnight shaking vigorously (250 RPM)</LI></OL></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD style="width:85%;"><DIV class="heading"><P style="font-size:14px; font-weight:bold"> Glycerol Stock Preparation </P></DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><P><EM> Adapted from Butanerds Protocols from the University of Alberta iGEM Protocols pdf</EM></P><P> What you will need </P><UL><LI> Overnight bacterial growth </LI><LI> screw captubes </LI><LI> glycerol </LI></UL><P> Protocol </P><OL><LI> Pipet 0.5mL of 50% glycerol into 3 1.5 screw cap tubes</LI><LI> Add 0.5mL of overnight culture to each tube </LI><LI> Pipet up and down to gently mix</LI><LI> Flash freeze  in liquid N2 or dry ice/ethanol bath </LI><LI> Place in -80 C freezer when frozen </LI></OL></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD style="width:85%;"><DIV class="heading"><P style="font-size:14px; font-weight:bold"> Agarose Gel Electrophoresis Protocol </P></DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><P><EM> Adapted from Butanerds Protocols from the University of Alberta iGEM Protocols pdf</EM></P><P> What you will need </P><UL><LI> 1X TAE </LI><LI> Graduated Cylinder</LI><LI> 125 mL flask </LI><LI> Agarose </LI><LI> Gel Pouring Tray </LI><LI> Tape </LI><LI> Gel rig </LI><LI> Ethidium Bromide </LI></UL><P> Protocol </P><OL><LI> Measure out 120mL of buffer </LI><LI> Transfer buffer to 125 mL flask </LI><LI> Weigh out enough agarose to make a 1% gel (in our case 1.2 g of agarose was the right amount)</LI><LI> Transfer agarose to 125mL flask</LI><LI> Melt agarose in microwave until solution is almose boiling, stirring every 15-20 seconds (should be around 2 minutes)</LI><LI> Allow agarose to cool (do not let it cool to the point where it is hard)</LI><LI> Add 3 uL of Ethidium Bromide to the cooling agarose</LI><LI> Assemble the gel pouring apparatus by inserting gate into slots. Use a pastuer pipet to run a bead of molton agarose along the edges of the gates to seal the box and prevent leaks</LI><LI> Allow gel to cool until flask can be handled comfortabley</LI><LI> Place comb in the gel rig</LI><LI> Pour agarose into gel tray</LI><LI> Allow to solidify. While the gel is solidifying prepare the samples. Add your sample and 1 uL 10x Loading Dye, 4 uL of DNA and 5 uL of water</LI><LI> Pour 1X TAE over gel so that gel is covered by a 3-5mm buffer</LI><LI> Load samples into lane (Don't forget to load a 1kb+ ladder into one of the lanes)</LI><LI> Hook electrodes to gel apparatus</LI><LI> Run the apparatus at 100V for 30 - 45 minutes (make sure to watch that the dye does not run off the gel)</LI><LI> Visualize the gel and record the results</LI></OL></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD style="width:85%;"><DIV class="heading"><P style="font-size:14px; font-weight:bold"> Restriction Digest</P></DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><P> This protocol is also described as a part of our <A href="#ct" title="construction technique"> Construction Technique</A>. Start by selecting the order of the two parts you will be putting together; the one in front will be referred to as the insert, while the one behind will be referred to as the vector.  Both the vector and the insert need to have their own separate tube, at least in the beginning. This is important because it allows for clean addition new parts to a the circuit </P><P> In the Insert Tube...
  <UL><LI>600ng of DNA (To figure out the volume, the calculation is 600 / concentration of plasmid.  This gives you volume in μL).</LI><LI>Water, so that the volume of DNA and water in the tube is 35 μL</LI><LI>4 μL of React 1 Buffer </LI><LI>0.5 μL of EcoR1 </LI><LI>0.5 μL of Spe1</LI></UL></P><P> In the vector Tube...
  <UL><LI>250ng of DNA (To figure out the volume, the calculation is 250 / concentration of plasmid.  This gives you volume in μL).</LI><LI>Water, so that the volume of DNA and water in the tube is 35 μL </LI><LI>4 μL of React 2 Buffer</LI><LI>0.5 μL of EcoR1</LI><LI>0.5 μL of Xba1</LI></UL></P><P> Put both tubes into the 37°C water bath for one hour.  After, place them into the 65°C heating block for 10 minutes.  This destroys any enzymes in the tube (which is ok, because by now they’ve done all they need to).
    Take the insert out, and put it in a -20°C freezer. </P></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD style="width:85%;"><DIV class="heading"><P style="font-size:14px; font-weight:bold"> Ligation Protocol </P></DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><P> This protocol is also described as a part of our <A href="#ct" title="construction technique"> Construction Technique</A>. Start by selecting the order of the two parts you will be putting together; the one in front will be referred to as the insert, while the one behind will be referred to as the vector.  Both the vector and the insert need to have their own separate tube, at least in the beginning. This is important because it allows for clean addition new parts to a the circut </P><OL><LI>Take insert out of the freezer and ad 5 uL of insert and 5 uL f vector to a new tube </LI><LI>Clearly label the remaining tubes of each (insert and vector) as Unligated, put the date on the tube and place in -20 C freezer in case the transformation does not work</LI><LI>To the single tube containing both insert and vector add 10 uL of 2x Quick Ligase Buffer and 1 uL of Quick Ligase.</LI><LI>Let this sit at room temperature for 5 minutes</LI></OL><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD width="85%"><DIV class="heading"> Plasmid Preparation/Isolation (For Higher Concentration of Plasmids)</DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><P>This protocol is used to obtain higher concentrations of plasmids.</P><OL><LI>Pick a single colony and inoculate a starter culture of 2-5 mL LB broth
+ antibiotic and shake for 8 hours at 37 degrees Celsius</LI><LI>Dilute the starter culture by 1/500 to 1/1000</LI><LI>Isolate the cells by spinning at 3200 rpm for 25 mins holding the
temperature at 4 degrees Celcius.</LI><LI>Resuspend the bacterial pellet with 10mL of buffer P1</LI><LI>Add 10 mL of Buffer P2 and invert the tube several times and incubate
at room temperature for no longer than 5 mins. This step is for lysis of
cells.</LI><LI>Add 10 mL of cold Buffer P3 to the lysate and invert several times.</LI>
This step is used to neutralize the lysis reaction.<LI>Pour the neutralized lysate through the barrel of the QIAfilter</LI>
Cartridge (included in kit) for 10 mins. Let this sit so that the later of
proteins, genomic DNA, and detergent will float on top of the solution<LI>Remove the cap from the cartridge outlet nozzle. Gently insert the
plunger into the cartridge and filter the lysate into a 50 mL centrifuge
tube.</LI><LI>Add 2.5 Ml of Buffer ER to the filter lysate, mix by inverting and then
incubate for 30 mins in ice</LI><LI> Equilibrate a Qiagen-tip 500 by adding 10 ml of Buffer QBT and allow
gravity to empty the solution.</LI><LI> Pour the lysate through the filter and allow the solution to flow
through via gravity flow.</LI><LI> Wash the tip with Buffer QC (30 mL) twice.</LI><LI> Elute the DNA with 15mL of Buffer QN</LI><LI> Precipitate the DNA by adding 0.7 volumes room temperature isopropanol
to the eluted DNA. Centrifuge for 90 mins at 3000 rpm.</LI><LI> Empty the supernatant slowly and carefully, the pellet is hard to see.</LI><LI>Wash the pellet with 5mL ethanol (96-100%) and centrifuge for 90 mins
at 3000 rpm. Carefully decant the supernatant without disturbing the
pellet.</LI><LI> Air dry the pellet for 10 mins and then redissolve the plasmid in
buffer TE with a suitable volume.</LI></OL></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD width="85%"><DIV class="heading"> PCR Purification</DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><P>The kit we used over the summer was purchased from Qiagen. This kit is
used to purify the PCR product.
 </P><OL><LI>Add 5 volumes of Buffer PB to 1 volume of PCR product and then mix.</LI><LI> Place a spin column in a 2 mL collection tube.</LI><LI> Apply the mixture of PB buffer + PCR product through the QIA column and
centrifuge for 1 minute.</LI><LI> Discard the flow-through and place the column back in the tube.</LI><LI> Wash the product using 0.75 mL buffer PE and then centrifuge for 1 min.</LI><LI> Discard the flow-through and place the column back into the tube.</LI>
Centrifuge the column inside the tube to discard the additional fluid in
the column.<LI> Elute the DNA into a clean microcentrifuge tube with buffer EB.
</LI></OL></DIV><DIV style="margin-bottom:60px"><TABLE width="700"><TBODY><TR><TD width="85%"><DIV class="heading"> Quikchange Site Directed Mutagenesis</DIV></TD><TD width="15%"><A style="float:right;" href="#top" title="return to top">return to top</A></TD></TR></TBODY></TABLE><P> The kit that we used was purchased from Stratagene.</P><OL><LI>Design and synthesis two complimentary oligonucleotides containing the mutation.</LI><LI>Prepare the control + sample reaction
</LI>
Control:
<UL><LI>5 µL of 10x reaction buffer</LI><LI>2 µl (10ng) of pWhitescript 4.5kb control plasmid</LI><LI>1.25 µL (125 ng) of control primer 1</LI><LI>1.25 µL (125 ng) of control primer 2</LI><LI>1 µL of dNTP mix</LI><LI>3 µL of QuikSolution</LI><LI>36.5 µL double distilled water to a final volume of 50 µL</LI><LI>+ 1 µL of PfuTurbo DNA polymerase</LI></UL>
Sample:
<UL><LI>5 µL of 10X reaction buffer</LI><LI>X µL (10ng) of dsDNA template</LI><LI>X µL (125ng) forward primer</LI><LI>X µL (125ng) reverse primer</LI><LI>1 µL of dNTP mix</LI><LI>3 µL of QuikSolution</LI><LI>ddH20 to a final solution of 50 µL</LI><LI>+ 1 µL of Pfu Turbo DNA polymerase</LI></UL><LI>
Follow the cycling parameters:
</LI><LI>Segment 1: 1 cycle (95 degrees Celsius, 1 minute)</LI><LI>Segment 2: 18 cycles (95 degrees Celsius, 50 secs; 60 degrees Celsius, 50
secs; 68 degrees Celsius, 1 min/kb of plasmid length)</LI><LI>Segment 3: 1 cycle (68 degrees Celsius, 7 minutes)</LI><LI>Add 1 µL of DpnI restriction enzyme to the amplified reaction. Mix
gently by pipeting the solution up and down and then incubate at 37
degrees Celsius for 1 hour.</LI><LI>Thaw the XL-10 Gold ultracompetent cells on ice. Add 45 µL to a
prechilled 14 mL BD Falcon polypropylene round-bottom tube.</LI><LI>Add 2 µL of the â=-ME mix to the 45 µL of cells</LI><LI>Swirl content and incubate cells on ice for 10 mins and gently swirling
every 2 mins.</LI><LI>Transfer 2 µL of the DpnI treated DNA to the cells</LI><LI>Preheat NZY+ broth in a 42 degrees Celsius water bath.</LI><LI> Heat shock cells in a 42 degrees Celsius water bath for 30 seconds</LI><LI>Incubate the tubes on ice for 2 minutes</LI><LI> Add 500 µL of preheated NZY+ broth to each tube then incubate the tube
at 37 degrees Celsius for 1 hour shaking at 225-250 rpm.</LI><LI>For the sample, plate 250 µL of cells on appropriate plate. For the
control, plate 250 µL of cells on the plate. Before plating, X-gal and
IPTG is required for the controls.</LI><LI>Incubate for 37 degrees Celsius for +16 hours.</LI></OL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Calgary/Lab/Protocol">http://2009.igem.org/Team:Calgary/Lab/Protocol</A>&quot;</DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Lab/Protocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Lab/Protocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Lab/Protocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Lab/Protocol&amp;oldid=82148" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>