"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/12 August 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
AUGUST 12, 2009
CAROL
Modelling Meeting
Today, we went over to ICT to discuss about how the two types of modelling can be put together as a whole. We discussed what both projects are capable of doing and the weaknesses in both models. The main task that matlab based modelling can help us achieve is to understand the signalling pathway in more depth. By characterizing the system via testing different parts of the circuit, we can understand more about the AI-2 system. If our results from the lab do not match the model, then either we are missing something in our model or there might be a step that we did not consider in the biological model. However, membrane computing gives us a better picture of the overall picture. It has the power to look at populations of cells as well as focus on one cell. I think the appropriate way to sell both modelling systems is to sell MC as a simulation tool and using the matlab based model, we can understand the signalling cascade within the cell more. There are a list of things we need to research about in the next few days for friday. We have another meeting on Friday for more discussion.
CHINMOYEE
The Recovery Day
We had a meeting with the membrane computing people in the afternoon . It was an interesting discussion that we had. We saw Iman's visulizations. We discussed the differences between Matlab and Mathamatica . We also looked into parameter sensitivity and parameter optimzation. Afshin suggested looking into evolutionary techniques for parameter optimzation . He offered to help with the creation of this technique in Matlab which is why we wanted a copy of it. He also wanted to better know the exact algorithm that Matlab uses to produce stochastic simulations.
I wrote a log about all the details that I remembered from my trip to Fort Mcmurray.
EMILY
Wiki and Overnight Cultures
Today I talked with Mandy about some things that are still missing from the Wiki.  I looked at some other wiki's to get some ideas for the bottom section of ours that has a gap that needs filling.  I also worked on the NEWS section of our Wiki, starting to add stuff about the Fort Mac trip.  I also made overnight cultures of my J13002-LuxOD47E-B0015 T2-C3 construct so that I can make glycerol stocks of this tomorrow.  Fahd and I also met briefly about Ethics to outline what needs to get accomplished over the next couple of days.
FAHD
Media Coverage and Marketing
Today I started my day by contacting and researching on the following companies:
1)Albian Sands Inc.
2)Life Technologies
3)ALPAC
4)Critical Outcome Technologies I also joined the Energy Futures Network, which is an online forum for discussing future innovations in the Oil & Gas industry and debating current issues relating to the energy sector.
For media, I setup a radio interview with our community radio network called CJSW 90.9. I also emailed our University of Calgary iGEM July newsletter  to CTV News and NUTV.
JAMIE
Getting to know the basics of AI-2 isolation
I read a Surette and Bassler paper on isolating "quorum sensing inducing molecules" from the supernatant of Salmonella typhimurium.  A large thanks to Margot from the Surette lab for kindly providing the various reporter strains and Salmonella strains needed for the experiment!
JEREMY
Isolating plasmid of PQ-OU in pCS26 and start of Response Circuit
Plasmid was isolated from eight overnight cultures of PQ-B-R-OU-B in pCS26 using the QIAprep Spin MiniPrep Kit (QIAGEN).  Plasmid purity and concentration were measured using the NanoDrop Spectrophotometer.
The response circuit began today with the start of construction of aiiA with a terminator and eventually ribosome binding site. This construct will eventually be placed behind the Pqrr4 promoter and c1 lambda inverter.
KATIE
Determining the Proper Orientation of Objects
For the majority of today, I spent time:
Working on getting the nucleotides for the DNA replication animation to line up correctly without using a function to set position using the current location of corresponding nucleotides since this will change when the display is moved to the bottom of the spiral for levels. This involved dividing each nucleotide by the same amount of space apart on each strand of DNA. However, the orientation of the nucleotides become off the further the DNA polymerase moves down the leading strand so I may have to have the polymerase listen fo the position of each corresponding nucleotide so it can get the position right.
Primase now rezzes the primers required for the lagging strand in replication and I will make it add a single primer to the beginning of the leading strand tomorrow
Changing positions of the equipment certain things were sent to in the second virtual lab
KEVIN
Attempted at making Pqrr4+I13500 competent
An attempt was made to make Pqrr4+I13500 cells competent again in order to transform in the mutant circuits; however, due to my short term memory, I had put them in an LB media with chloramphenicol, which the cells don't have resistance to. Failure, Failure, and another failure.
Overnight culture of Pqrr4+I13500 C4, C5 cells
Because of today's failure at making those cells competent, I have to grow another overnight culture and give it another try from scratch.
MANDY
Usability of the Wiki
Did some modifications to the wiki, continuing from Monday:
The site map has a complete guide to the navigational tools which are buttons that outline contributions by each sub-team.
A comment widget has been added to the front page to limit the stretch on the other boxes (by our growing list of sponsors) and to allow some communication with our visitors.
Biobricker User Interface Design
I began designing the buttons and indicators that will appear on the biobricker userface for the second component of the island. So far, I have:
promoters
terminators
coding sequences for various proteins
proteins
co-factors
PRIMA
The ultimate lab test & marketing continuation
Today, I spent some time going over my plans for construction of the aiiA part of the response circuit with my Supervisors. This was just to get a better understanding of the lab work, what i'm doing, why i'm doing this, etc. It helped me a lot and made me more confident about myself. We collectively decided that we need to synthesize aiiA specific primers and RBS primers. Jeremy and I will focus on the aiiA primers while Thane does the RBS.  Next, I spent sometime figuring out the sequence for the aiiA forward and reverse primers and calculating their Tm.
With respect to marketing, I called a few companies I was in contact with last week. Unfortuantely, I couldn't get a hold of anyone so i left messages and I will follow up tomorrow. I also sat with the T-shirt design group to decide on the colors, design and layout of the shirt after we found out some information about the cost of the shirts.
Finally, I began the restriciton digest. I'm moving aiiA (C0160 which was originally in psB1A2) in front of B0015, which exists in the psB1AK3 vector. The reason  I moved the larger piece into the smaller is because I need to grow the bacteria on Kan plates since they both have Amp resistance.\
I prepared the insert and vector and left it in the 37 degreees water bath overnight.I cut the insert with EcoR I and Spe I using REact I buffer. I cut the vector (containing B0015) with EcoR I and Xba I using REact 2 buffer.  I'll phosphatase tomorrow.
STEFAN
New station!
So after deciding that the smelly bacteria station was not satisfactory I made something else. You encounter a sick squid that needs some vitamins. Luckily, I have created a bacteria a while ago which can do just that! The person will then create some vitamins by clicking on it and drag them to the squid. This is based on the same principle as the smelly stuff because it's still a gene being introduced that is not native to the bacteria, just has a different function. The notecard was written as well.
VICKI
Meeting with the modelling team
The modelling crew met today to discuss how we could integrate our work into a cohesive section in a presentation. Our main focal points were (a) why model; (b) what questions can be answered by each modelling approach; (c) what does each modelling approach NOT answer; and (d) can we resolve the gaps in one modelling approach by bringing in the other? This will help frame our descriptions of our approaches and help us convince others that what we're doing is worth it.
On the MatLab side, we have found that SimBiology does not provide the simulation power that can be achieved by membrane computing. For instance, the platform does not offer a convenient way of modelling more than one cell, making it very difficult to introduce a space/location component into our model. We are having trouble resolving what questions can be anwered with the SimBiology simulation model that cannot be answered by using membrane computing. Accordingly, we are focussing more heavily on characterisation. As we will be biobricking ~5 different promoter-LuxPQ components, each one of those circuits can be contributed to the registry in characterised form, so that future users will be able to use our characterisation data to compare parts. It is sensible to approach this with MatLab because the membrane computing model cannot answer many of these aspects - for example, what is the best temperature to culture and maintain our cells for optimal performance? Or how does the broth in which our cells are cultured affect the dynamic performance? By collecting that data and fitting it to equations, future users of our registry contributions will have a quantitative way to compare parts and select the best one for their purposes.
Our next meeting will be at noon on Friday. By then, this is what I hope to achieve:
play with SimBiology at the GUI and command line levels to see if we can build in a multicellular approach
check out the optimisation toolbox and see if there would be a nice way to implement some form of evolutionary optimisation in MatLab. This will be useful in helping us optimise reaction constants for the simulation model. Work with Afshin on this - if it can't be done on Matlab, we might be able to implement something in mathematica using the same data.
work with the Fort Mac crew to prepare a cohesive, relevant presentation on our experience on the oilsands tour and how synthetic biology can tie into that. Note that iGEM also focusses on ethics and social responsibility, which will be a critical consideration in forging any future alliances, regardless of the industry with which our potential partners are affiliated.
Retrieved from " http://2009.igem.org/Team:Calgary/12_August_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
