"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/16 July 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
JULY 16, 2009
CAROL
Diluting Primers and Primer Extension Polymerase Chain Reaction (PCR)
To create the synthetic sigma 70 promoter library, we have synthesized 32 different primers (16 different forward primers and 16 different reverse primers) to hopefully form 256 different promoters.
Need to dilute primers to 80μM. The following is the concentration of the original primers that were synthesized once water was added to the primers.
Primer
260/280
260/230
Concentration [ng/μL]
Reading/2.5 [ng/μL]
Stock Concentration [μM]
Volume stock into working [μL]
Volume water into working [μL]
sig70-F1
1.98
1.73
20816
832.64
60.07
6.7
43.3
sig70-F2
1.55
1.80
44284
17713.6
1277.91
3.1
46.9
sig70-F3
1.62
1.86
41323
16529.2
1192.46
3.4
46.6
sig70-F4
1.70
1.99
32365
12946
933.96
4.3
45.7
sig70-F5
1.72
2.07
33688
13475.2
972.14
4.1
45.9
sig70-F6
2.07
1.95
45412
18164.8
1310.46
3.1
46.9
sig70-F7
1.64
1.95
39046
15618.4
1126.76
3.6
46.4
sig70-F8
1.43
1.63
45745
18298
1320.07
3.0
47.0
sig70-F9
1.75
2.16
23628
9451.2
681.84
5.9
44.1
sig70-F10
1.72
2.01
32682
13072.8
943.11
4.2
45.8
sig70-F11
1.71
2.07
31494
12597.6
908.83
4.4
45.6
sig70-F12
1.59
1.82
41202
16480.8
1188.97
3.4
46.6
sig70-F13
1.73
2.03
29224
11689.6
843.3
4.7
45.3
sig70-F14
1.41
1.58
45645
18258
1317.2
3.0
47.0
sig70-F15
1.59
1.85
40681
16272.4
1173.9
3.4
46.6
sig70-F16
1.57
1.82
42241
16896.4
1219.0
3.3
46.7
sig70-R1
2.06
2.17
7516
3006.4
216.9
18.4
31.6
sig70-R2
2.04
2.25
15255
6102.0
440.2
9.1
40.9
sig70-R3
1.76
1.85
42503
17001.2
1226.5
3.3
46.7
sig70-R4
2.01
2.21
24012
9604.8
692.9
5.8
44.2
sig70-R5
2.10
2.27
18530
7412
534.7
7.5
42.5
sig70-R6
1.98
2.19
28452
11380.8
821.0
4.9
45.1
sig70-R7
1.94
2.17
32976
13190.4
951.6
4.2
45.8
sig70-R8
2.00
2.20
25379
10151.6
732.4
5.5
44.5
sig70-R9
2.00
2.24
24141
9656.4
696.6
5.9
44.1
sig70-R10
1.94
2.17
30237
12094.8
872.6
4.6
45.4
sig70-R11
1.92
2.16
31497
12598.8
908.9
4.4
45.6
sig70-R12
1.92
2.17
30045
12018
867.0
4.6
45.4
sig70-R13
2.01
2.18
18726
7490.4
540.4
7.4
42.6
sig70-R14
1.88
2.03
36111
14444.4
1042.1
3.8
46.2
sig70-R15
1.98
2.17
10679
4271.6
308.2
13
37
sig70-R16
1.86
2.01
35969
14387.6
1038
3.9
46.1
The sequences for each primer is listed in the following table:
Primer
Sequence
sig70-F1
GCGCGCTCGAGAATAATTCTTAAAATTTATGCTTCCGGCTCG
sig70-F2
GCGCGCTCGAGAATAATTCTTAATATTTATGCTTCCGGCTCG
sig70-F3
GCGCGCTCGAGAATAATTCTTAAGATTTATGCTTCCGGCTCG
sig70-F4
GCGCGCTCGAGAATAATTCTTAACATTTATGCTTCCGGCTCG
sig70-F5
GCGCGCTCGAGAATAATTCTTTAAATTTATGCTTCCGGCTCG
sig70-F6
GCGCGCTCGAGAATAATTCTTTATATTTATGCTTCCGGCTCG
sig70-F7
GCGCGCTCGAGAATAATTCTTTAGATTTATGCTTCCGGCTCG
sig70-F8
GCGCGCTCGAGAATAATTCTTTACATTTATGCTTCCGGCTCG
sig70-F9
GCGCGCTCGAGAATAATTCTTGAAATTTATGCTTCCGGCTCG
sig70-F10
GCGCGCTCGAGAATAATTCTTGATATTTATGCTTCCGGCTCG
sig70-F11
GCGCGCTCGAGAATAATTCTTGAGATTTATGCTTCCGGCTCG
sig70-F12
GCGCGCTCGAGAATAATTCTTGACATTTATGCTTCCGGCTCG
sig70-F13
GCGCGCTCGAGAATAATTCTTCAAATTTATGCTTCCGGCTCG
sig70-F14
GCGCGCTCGAGAATAATTCTTCATATTTATGCTTCCGGCTCG
sig70-F15
GCGCGCTCGAGAATAATTCTTCAGATTTATGCTTCCGGCTCG
sig70-F16
GCGCGCTCGAGAATAATTCTTCACATTTATGCTTCCGGCTCG
sig70-R1
GCGGGATCCAATTGCACGTAAAATACGAGCCGGAAGCATAAA
sig70-R2
GCGGGATCCAATTGCACGTAATATACGAGCCGGAAGCATAAA
sig70-R3
GCGGGATCCAATTGCACGTAACATACGAGCCGGAAGCATAAA
sig70-R4
GCGGGATCCAATTGCACGTAAGATACGAGCCGGAAGCATAAA
sig70-R5
GCGGGATCCAATTGCACGTATAATACGAGCCGGAAGCATAAA
sig70-R6
GCGGGATCCAATTGCACGTATTATACGAGCCGGAAGCATAAA
sig70-R7
GCGGGATCCAATTGCACGTATGATACGAGCCGGAAGCATAAA
sig70-R8
GCGGGATCCAATTGCACGTATCATACGAGCCGGAAGCATAAA
sig70-R9
GCGGGATCCAATTGCACGTAGAATACGAGCCGGAAGCATAAA
sig70-R10
GCGGGATCCAATTGCACGTAGTATACGAGCCGGAAGCATAAA
sig70-R11
GCGGGATCCAATTGCACGTAGGATACGAGCCGGAAGCATAAA
sig70-R12
GCGGGATCCAATTGCACGTAGCATACGAGCCGGAAGCATAAA
sig70-R13
GCGGGATCCAATTGCACGTACAATACGAGCCGGAAGCATAAA
sig70-R14
GCGGGATCCAATTGCACGTACTATACGAGCCGGAAGCATAAA
sig70-R15
GCGGGATCCAATTGCACGTACGATACGAGCCGGAAGCATAAA
sig70-R16
GCGGGATCCAATTGCACGTACCATACGAGCCGGAAGCATAAA
Stock dATP, dGTP, dCTP, dTTP from Invitrogen was diluted from 100mM to 0.5mM.
See how the primer extension PCR is done in the protocol page.
Both the varying promoters and the vector, pCS26, was digested using enzymes, XhoI and BamHI overnight. The vector was incubated at 37oC and the promoter was digested for 2 hours at 16oC and then overnight at room temperature (~21oC).
CHINMOYEE
Content / Ideas on Modelling
Looked into papers to find Rate Constants . Unfortunately this endeavor was unsuccessful .  After the meeting with Thane, me and Vicki planned out the distribution of gene characteristics . From the meeting with Vicky we both decided to think about some of the content / ideas we discussed for modelling.  I am going to be looking at curve fitting with the hill function and Static performance of the system .
EMILY
Verification NotI Digest
Today I started with a NotI digest to verify the presence of the B0015 terminator in the J13002-LuxOD47E construct.  I ran this on a gel and got good results.  See gel photo below.
Analysis: Lane 1 is J13002-LuxOD47E-B0015 trial 1, colony1, digested with Not1
Lane 2 is J13002-LuxOD47E-B0015 trial 1, colony 1, uncut
Lane 3 is colony 2 digested with NotI
Lane 4 is Colony 2, uncut
Lane 5 is colony 3 digested
Lane 6 is colony 3, uncut
Lane 7 is colony 4, digested
Lane 8 is colony 4, uncut
Lane 9 is J13002-LuxOD47E
Lane 10 is BBK LuxOD47E
The digest looks like it worked as we see bands at around 3000 bp and 1500 bp in the digested lanes, representing the vector and the J13002-LuxOD47E-B0015 construct respectively.  We are going to send colony 3 for sequencing as it has the least amount of random bands in it.
Today I also helped Mandy with some formatting of the Wiki.  We added a template for our header and added it to the other pages.  We also edited the team members pictures and added some text descriptions to the Home page and Team pages.  We also looked into using icons on the top of each page.
Tomorrow I am going to send colony 3 down for sequencing in the morning.
FAHD
Marketing for July 16th 2009
Today, I concentrated my energies at marketing our iGEM project. Here is what I did today:
1) Call VWR, he offered to gives us free lab equipment and discounts on expensive lab equipment. He told me to e-mail him a list which thane said that he will prepare tomorrow.
2) Called Charles Rivers Lab and e-maile him a proposal and will follow up next week
3) Called Boehringer Ingelheim Canada and left her a voicemail
4) Called COTI and left him a voicemail
5) Called Bracco/ E-Z-EM canada and left him a voicemail
6) Called Life-Teac/Applied Biosystems and left her a voicemail
7) Called EMD Chemicals and e-mailed hera sponsorship package. Will folllow-up next week
8) Called Genome Canada and left him a voicemail
9) Called Nexen Inc. and left her a voicemail
10) E-mailed proposals Albian Sands Energy and Bietz Resources Ltd.
11) E-mailed proposals to Lonza Canada Inc., Quidel Corporation and Electron Microscopy Services. Since I did not have specific contacts, i e-mailed the proposals to their generic e-mail addresses and will follow up next week
12) Did my blog of the week for the marketing team
13) Went through the Micheal Smith awards application(NSERC) and will tell everybody abt it tomorrow in class
14) E-mailed a Thank You Card to Mr and Mrs Kubik
JAMIE
July Newsletter and c1lambda verification
Outline for June Newsletter was updated for the July newsletter.
Restriction digest verification of c1lambda with XbaI and PstI indicated an insert in the plasmid which appeared too large (band size of >1kb compared to an expected 987bp).
JEREMY
Colony PCR to verify PQ-B-R-OU-B (July 9) construct in AC plasmid
Purpose: To perform a colony PCR to verify the presence of the PQ-B-R-OU-B. The pTaq colony PCR reaction used two sets of primers (LuxOU F/R and LuxPQ F/R) and the following conditions were used: 94ºC for 6 minutes; 36X (94ºC for 30 seconds; 55ºC for 45 seconds; 72ºC for 4 minutes for LuxPQ F/R (or 2 minutes for LuxOU F/R); 72ºC for 10 minutes; held at 4ºC. The products were then run on a 0.8% agarose gel (see below). For LuxOU F/R primers, we can see contamination in the negative control lane, and even though we see the appropriate PCR product size for all colonies (~2kb), we should not proceed with these results. However, we can see that when using the LuxPQ F/R primers, we have a clean negative control lane, and all colonies except for colony 4 show the expected size (3.8kb).
KATIE
Movement and Modification
The gel electrophoresis script has been modified and now it should be simple for Mandy to add any conditionals she would like to suit the lab activities. Also, the tube for bacteria transformation now has another tube beside it and every time a question for the quiz is answered, the tube will fill up a little more and once it reaches the top it will reset for another person to take the quiz. I would like to change the colours each time a question is answered depending on whether it was answered correctly or not, but for now it fills up using a green colour.
The water bath script is now being modified and I am inserting movement scripts using llSetPos() function into the donor and recipient tubes for restriction digest (due to my horrible attentiveness to saving, I lost this once today when second life froze my laptop – luckily the script is not too long). It is all under comments for now in the recipient tube and the donor tube is undergoing testing with the current script. The water bath listen script has been set up and the top will open using the touch_start event at the moment, which I will transfer to listen so that when the test tube begins to move over to it the top will open.
Once again the restriction digest tubes cannot be used the way they were intended while I add the movement code to the existing script so this activity will probably be under construction until early next week. I will still need to determine the proper placement for the tubes as they move so they do not go through the walls of the water bath, actually end up in the water and do not end up in the exact same place to appear as one tube.
Tomorrow I would like to:
Finish up the listen/response scripts that I started for the heating block and phosphatase treatment that were adapted from the water bath.
Get the movement of the tubes for restriction digest moving smoothly and to the proper positions within the lab. If I am able to complete this I believe I will clean up a couple of my scripts so that they look presentable for the video I have to do for the blog next Tuesday.
If I am able to get restriction digest functioning again, I would like to return to working on the sequencer, which I have neglected this week. I will need to test the note card reading script to make sure it is functioning the way I would like it to and then I will add it to the sequencer.
KEVIN
1. Plasmid Isolation
Pqrr4+I13500 plasmids were isolated from yesterday's overnight culture by using Sigma's GenElute miniprep kit. (sigma) Then the product's purity and concentration was measured with nanodrop utility. The isolated plasmid was pure enough to move on.
2. Restriction Digest
To verify the presence of Pqrr4+I13500, I performed restriction digest on the above plasmid using XbaI and PstI enzymes. The following is a picture of the gel:
Every lane worked except 9. It seems like non specific cutting happend in lane of colony 9. It can be sequenced now to verify for sure that it is in there.
3. Overnight culture of B0034
Overnights of B0034 were grown for tomorrow's Plasmid Isolation and Construction.
MANDY
Wiki Design
Today, we implemented a lot of the style suggestions we received during the wiki workshop including shrinking the header, using top and left navigation, and using templates to layout code neater/easily. The header is now smaller and in a simple to use template so it can be pasted on all pages easily. As well if it gets changed it will be changed on all pages at the same time which is convenient.
I messed around trying to remember how to code tables properly (with cells that span columns and rows). My css spacing wasn’t rendered very well in wiki so I used blank table cells as pretend spaces between sections instead.
We decided we need at least a brief description of iGEM and a follow up page for people such as sponsors, who aren’t part of the competition. We will also be using coloured icons to remind people of sections in the wiki (lab has a green icon, etc. these haven’t been made yet). We will also include a site map.
We also vetoed the multiple languages idea because it’s pointless if you’re just parsing your text through Google Translate (not being sure of if it’s correct or not), and it’s superficial to make a wiki appear to be coded in 10 different languages when only the front page has 10 of the same paragraph in different languages and the rest is in English.
That’s it for wiki for now. It doesn’t look like much, because a lot of today was switching between messing with html and css code to figure out what the wiki site would allow me to do and what I couldn’t do (which is annoying).
PRIMA
Marketing and Media Continued
Researched and Emailed the sponsorship pkg  to the following companies with a brief one-two sentence about how iGEM Calgary’s project can be beneficial to them or their service.
Company 3 – emailed sponsorship pkg –Exec Director and Comm and Research Specialists- follow up july 12
Company 1 – mailed sponsorship and filled out online app – follow up july 20
Innovia biosciences – international company – mailed anyways probably not going to help us. – located in UK
Company  2 – only found phone numbers – so call tomorrow to get a hold of someone – VP of Canadian Forest Lands
I found out that Jay Ingram (co-host of Daily Planet on Discovery Channel) is an U of C alumni and I found his Calgary number. I can try contacting him to see if he can help with sponsorship or media. If you guys can offer suggestions on what I should ask him to help us with, that would be great. I’ll talk to the rest of the team about it tomorrow.
Mandy and I looked up T-shirt companies who could give us quotes on bulk orders. We came up with Wordans.com.  Tomorrow’s plan is to follow up with GE Health (whose CEO I still can’t get a hold of), company 2 – to see if they considered sponsoring us. Also, tomorrow, I’ll talk to the rest of the marketing team to see if they were able to get a hold of anyone. If not, I’ll suggest calling those companies to follow up.
Also Jamie and I will be calling and arranging one-to-one meetings with some Calgarian companies and finish up the July newsletter.
STEFAN
More ethics
So ethics meeting will be coming up shortly so I'm doing a lot of reading. Second Life will pick up again this weekend.
Today I read:
Henkel, J. and Maurer, SM., 2007, The Economics of Synthetic Biology, Molecular Systems Biology 3 (117):1-4.
This paper deals a lot with the economical repercussions of having standard biological parts for synthetic biology and raises several important issues. Since standardized components are a fundamental part of iGEM, this paper will help us look deeper into the pros and cons of an open source system. Detailed notes were taken to later implemented into the paper.
VICKI
Other approaches: neural networks
I haven’t stopped thinking about this kind of representation and way of establishing a model since you brought up systems biology. I accordingly looked into it and checked to see if there is a way that we could implement it into our modelling efforts. From a MatLab standpoint, it’s feasible with the equipment on Main Campus, as MatLab has a neural network toolbox that isn’t part of our iGEM packages but is on the engineering department computers. I was interested mostly because it would be another way to distinguish our efforts from all of the other teams who don’t take their modelling efforts beyond what SimBiology offers (which isn’t all that much). However, after discussing the idea with Thane, we decided it wouldn’t be all that appropriate as it wouldn’t solve our problem of not knowing the rate constants of all of the little reactions that are taking place.
Modelling meeting
We sat down with Thane to determine how to best coordinate our efforts and time, as well as to discuss some of the difficulties that we’ve been running into in finding rate constants and adopting a thorough approach to modelling individual reactions. We also chatted about circuit characterisation measures (and specifically how it probably won’t be realistic to measure genetic stability in time for the competition and in good financial health, unless we can think of a very creative, fast and cost-effective alternative). We divided tasks to check into regarding possible procedures that we could adopt, along with an assessment of the feasibility of attaining that measure in time for the iGEM. As mentioned in my previous email, I am looking into the dynamic response and the response time. ased on the procedures adopted last year, it will be realistic if  we can get our hands on a supply of AI-2 molecules or a very close mimic. The dynamic response and response time are really only effectively measured if we can perturb the system to a noticeable extent (ie, big change from the previous state), which will require a very large change in AI-2 input from the normal basal levels. If we can have that molecule at our disposal, the procedure actually doesn’t look that hard. We will still need to get a sense of the delay (time between successive measurements of a time-dependent system) on the measuring device as that will greatly affect the feasibility of modelling the dynamic response, but we should be able to figure something out.
Retrieved from " http://2009.igem.org/Team:Calgary/16_July_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
