"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/14 July 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
JULY 14, 2009
CAROL
Last Attempt of Transformation
Attempted to transformed plasmid (Quick Ligated product (luxCDABE + pSB1AK3/pSB1AC3)) ibto Top10 cells and then plated on AK/AC plates overnight.
CHINMOYEE
E-mailed Gauntlet about article
I looked at papers  Potential :    Kinetic Analysis of Multi site phosphorylation using Analytic Solutions to Michaelis-Menten Equations Ligand induced Asymmetry in Histidine sensor kinase complex regulates quorum sensing
Dead end : Purification and characterization of a luxO promoter....
Non Paper related stuff :  found a Systems biology toolbox for Matlab ( free)  E-mailed gauntlet about writing a Frosh supplement article
EMILY
Construction of J13002-LuxOD47E-B0015, Glycerol Stocks and Transformation of J13002-LuxOD47E into KT1144 Cells
Today I did another construction digest with J13002-LuxOD7E and B0015.  I then transformed into TOP10 cells and plated on AC plates.  I also made glycerol stocks of LuxOD47E BBK and of J13002-LuxOD47E.
The retreaks of the J13002-LuxOD47E constructs in KT1144 cells worked, however when tested under UV light, they still did not glow.  I transformed a different colony of J13002-LuxOD47E into KT1144 cells again and we'll see if it gives any better results.
Today I also helped Mandy out with some stuff for the Wiki.  I labelled, tagged and resized some photos that we're putting in our gallery and I wrote up a section for our Overall Project page that explains what the iGEM competition is and how our team is divided.
FAHD
Marketing for July 14th 2009
Today, I concentrated my energies at marketing our iGEM project. Here is what I did today:
1) Attended the meeting in the morning
2) Contacted VWR. Will do a follow up call soon.
3) Called EMD Chemicals/Biosciences (Brands include Novagen, Novabiochem and Calbiochem), left a voicemail and i will get back to them tomorrow.
4) Called Critical Outcome Technologies, nobody was available so i left them a voicemail.
5) Called Life Technologies. Left them a voicemail.
6) I called the owner of Neuroimage Inc., I have telephone meeting with him on Friday 6 pm(ET).
7) I e-mailed Jen a thank you letter after consulting my beautiful marketing team.
8) I replied to Mrs.Kubik e-mail about the update on the e-mails that we had sent. Conoco Phillips and Statoil said no i will send them a 'rejection thank you' e-mail.
9) I discussed the template and information for the July's newsletter.
10)I worked on the Air Canada Community investment application. They offer free airline tickets in their community investment programme.
11) I will go to Walmart in the morning(8.30) for my visa photos cuz they messed up the photos last time. Then i will go to Scotia Bank to deposit my visa money hopefully by by 9.30/10.00.
12) I also called Genome Canada, I talked to the President's assistant: she told me to me to e-mail her the proposal so she will forward the e-mail to her boss.
13) I called Astellas Pharma however she is out of her office till next week.
JEREMY
Sequencing of PQ-B-R-OU-B
Colony 12 of the PQ-B-R-OU-B construct in AK was sent to the University of Calgary DNA Sequencing Facility with BBK CP F/R and R0040-R primers.
KATIE
Complete Version of Activities
This morning, I completed the restriction digest activity in its general form. Now, I have to rename materials so they are no longer general and add the notecards to be given to avatars to each object in the activity and set it so that you cannot read what is within the notecard, which will be compared to a master notecard within the sequencer at the end of the activity. This prevents anyone from just renaming objects in their inventory (“cheating”). I am in the process of modifying the notecard reader script for use within the sequencer and depending on how complicated it ends up being I may incorporate it into the rest of the restriction digest activity and check all notecards for every step if possible. My restriction digest notecard has also been completed to the best of my ability, which will be sent to Mandy this week.
I am now working on bacterial transformation, which I would like to be slightly different from PCR/Gel Electrophoresis and Restriction digest, which have functions different from one another as well (for example, order is important for one). So, I believe this activity will ask the user questions to assure they have some idea of what they are doing, which once again also serves the secondary purpose of preventing people from just renaming objects and being able to complete the activity.
I have created an object that has the ability to create the bacteria avatars by touching it.  Problem: Mandy pointed out that the attach avatar permission only worked for me, which I determined was because I am the owner of the object when it is rezzed. So instead of rezzing the object I decided to give the object to people so that they will become the owner. This means they will now have to take it out of their inventory by attaching it to themselves from their inventory. So it is working, just not the way I would prefer it. There is also varying levels of brightness for the bacteria, which I am unsure of whether to keep or just have it glow when there is enough bacteria.
Tomorrow I will modify the PCR script so that the items for each activity are more specific to that single activity as well as scientifically accurate, including the names for each specific PCR. I will be completing bacterial transformation after I brainstorm questions.
I also have to go back to the water bath script and incorporate it into the activities as well because as of now it is just a stand-alone activity. All I will have to do is change the names of what is added and given to and from the water bath respectively and this also means modifying phosphatase treatment. I am also beginning to make more visual instructions for restriction digest. So far sections that are explained will flash as instructions for that specific section are given. I would like to expand on this later this week.
KEVIN
Colony PCR (cPCR) of Pqrr4+I13500
Unlike the R0040 promoter driving the expression of I13500, Pqrr4 promoter is not constitutive, so it cannot be tested visually. Since I do not have any overnight cultures to perform Restriction Digest as of now, the only method I can use today is cPCR. 3 colonies were chosen, and 2 different sets of primers were used: BBK CP F/R primers and Pqrr4 F and BBK CP R. The following is a picture of the gel:
Only colony 3 seemeed to have worked. Pqrr4+I13500 is about 1013bp, and because biobrick primers anneal outside of the circuit (adding about 250 more base pairs), it seems to match the band in the lane of colony 3. Since I only tried 3 colonies, tomorrow, I will pick more and perform another cPCR.
MANDY
Second Life Lab & Interview
Continued scripting DNA extraction. I learned how to rotate objects in scripts, which requires conversion of a quaternion scale to vectors (which I still don’t really get). Anyways, it works, mostly, up to the point where different buffers are eluted through the cartridge to get rid of debris not taken care of by RNAse resuspension fluid or proteinase K. I am still figuring out a way to ensure that it only works if the buffers are added in the correct order. Also, I am considering eliminating the centrifugation step between each elution, because that makes it a very complicated activity.
Stefan and I found some information about Pathfinder Linden, the guy responsible for promotion and helping educational areas in Second Life. We will hopefully be going to his office hour (which starts at 8:30 am) next week to talk to him. Eventually, we hope to prepare a interview with him discussing the future of education in second life, why it is a useful and unique educational platform, and maybe discuss how science education is developing. Tips on how to maximize number of visitors using our area could also be useful. We could videotape this interview (esp if he has a microphone set up) and post it on the wiki (like Heidelberg and the Novel Prize Winner- less exciting… but still an interview). So today, Stefan and I left a message in his ‘Second Life Voice Mail’, and hopefully he will get back to us soon.
As a side note, a couple stylistic changes have been made to the wiki to keep up with what we learned at the conference. Unfortunately, the coding for the page has to be completely redone in HTML tables instead of CSS because absolute positioning does not work.
PRIMA
Cold Calls & Transformation of Cl lambda into xl gold
For marketing:  The marketing team sat down and decided that we should have our third bake on Monday, July 27th and the fourth one on Monday, August 16th. These dates are tentative. Fahd and I have worked on the July newsletter template: Cedarlane is our sponsor of the month. Our goal is to get it done by Friday so we can start sending it out next week.  For companies:  Company 1 – got an email back saying they’re considering our sponsorship packgage  follow up with them on Monday, July 20th  Company 2– couldn’t get a hold of them  call back tomorrow Company 3 – considering our package, needs some time  follow up on Monday July 20th Company 4 – sent package yesterday, will follow up tomorrow to see if they got my email Company 5 – (this was one of the older companies who said no…but I got a hold of someone new) –sent sponsorship package yesterday – will follow up tomorrow to see if they received it Company 6 –package sent to the executive committee, now considering  follow up July 20th Company 7 – sent package  follow up tomorrow to see if they received it Company 8- got an email today …refused to help us. I Sent them a thank you letter anyways and asked for contacts Company 9  - sent pkg, have CEO’s number, called before couldn’t get a hold of him, will call again tomorrow – I wanna set up a meeting with him cuz I think I know where their main office is located.  Tomorrow – call the 4 companies  from Mrs. Kubik’s company contacts. They were Suncor, Syncrude and 2 more whose names I don’t remember and Fahd took the list home. I sent them our package 2 weeks ago, tomorrow I’ll follow up to see if they had considered sponsoring us.
Lab:  I helped Jeremy set up sequencing for LuxPQ-B-R-LuxOU-B in AK for colony 12. We used colony 12 because the restriction digest from yesterday worked perfectly for that colony (correct size and no random fragments) (please refer to attachment). Since last time we only got one colony from transforming Cl lambda couple days ago and the restriction digest we ran yesterday showed multiple bands, we couldn’t judge the size of Cl lambda properly. Therefore, today we decided to transform Cl lambda again into competent cells. Since the inverter plasmid is resistant to Kan, we made more Kan agar plates and then transformed the Cl lambda and let them grow over night.  We also made overnight cultures of the Cl lambda. We’ll isolate the plasmids tomorrow. Then in the afternoon, if we see single colonies on our overnight plates, we ‘ll do restriction digest again to confirm that the Cl lambda is actually there in our plasmids.
STEFAN
Adding life to the Synthetic Kingdom
Today was spent on working on the synthetic kingdom. Difficulty was encountered because moving the giant eukaryotic cell was harder than it should've been. The organelles kept wanting to move to their original places because of the scripts involved so I had to stop all of them and delete them. I will reintroduce them once I have found a suitable place for the cell. We had 3 new "births" today, which increased the population of the bacteria! Squid were added as well as jellyfish and coral. I feel that this is important because of the relation of each to quorum sensing, GFP and RFP respectively so we can show that some genes from animals can be added to E.coli (pretend that is italicized). Also it looks cool. I am attempting to also pretty up the place so it will be more vibrant and full of life. This is going to be carried out by designing underwater plants and adding rocks as well for a more natural look. This will be carried out tomorrow.
Upon my domain reaching satisfactory levels, I did my blog video in one AMAZING take. Unfortunately it didn't save properly so now I have to do it at home tonight.
VICKI
Paper reading
I indulged in learning about the “Interplay of two quorum sensing regulation systems of Vibrio fischeri” in a 2007 paper. It is a long paper for which most of the space is occupied by a derivation as they are looking at modelling the lux and ain pathways with an ODE system, where they consider stationary states, dynamical behaviour and a possible biological meaning. I think that I’m supposed to be looking up phosphorylation and other rate constants, but I’m realising (conveniently after Carol has left for work) that we haven’t really sorted out what we need to be finding out and compiled it into a convenient table for filling in. I’ll make sure that gets done tomorrow. I also went through “A stochastic model of Escherichia coli AI-2 quorum signal circuit reveals alternative synthesis pathways”, which focussed on how cell physiology can be linked with systems-based stochastic models that can be assembled from scratch with some (not complete) knowledge of biochemical pathways. Sadly, LuxS is the only protein in that series that comes up in their differential modelling approach. However, they do describe how they used the MatLab optimisation toolbox to optimise their ODE equation parameters, so I made a note of that.
Circuit characterisation
I took another look at the registry, where they describe how they characterised the AHL signalling pathway. I’m currently looking for the experimental procedures that were adopted in quantifying these values and will keep the team posted on my success on that front. Some of them are going to be challenging – response time, for example, is based on a sudden impulse of the signal in question being introduced to the system. We’ll discuss this soon, but I think there would be great great great great great … great value in having a good supply of AI-2 isolated so that we can try to resolve a step (constant infusion of AI-2 at a certain level) or impulse (sudden, very high infusion of AI-2  liken to dumping a huge amount of AI-2 into a tube with bacteria containing the AI-2 signalling pathway) response. We’d need lots of output data at multiple points of time so the collection might be a pain, but it would be very useful for further practical applications of this signalling pathway.
Plate reader
Thane showed us where this is in the lab and I leafed through the booklet for a little while. If you’re cool with it, I’d like to spend some time playing with the software and figuring out what that machine can do. So much of mathematical modelling depends on how frequently and effectively we can make measurements, and the experimental/data collection approach that we take will ride heavily on the instrumentation that we are working with, especially if we’re dealing with proteins that are degrading and processes that are happening quickly (are they? I still need to see if this is the case). The machine looks like it’s more sophisticated and advanced than the plate reader that I used in Toronto, which took forever to analyse a 96-well plate – which is bad news if we’re dealing with time-dependent data (and which is why I’d be reluctant to fill a 96-well plate completely unless the time between reading well 1 and well 96 is much shorter than the response time of the circuit and the half-life of the protein). But if it can make these measurements faster, then we’ll have much more versatility in what we can test, with (hopefully) more reliable models. I’ll bring this up tomorrow as well.
Retrieved from " http://2009.igem.org/Team:Calgary/14_July_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
