"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Lab/Protocol
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
LAB INDEX
Overview
Updates
Circuits
Signalling Circuit
Reporter Circuit
Mutant Circuits
Response Circuit
Quorum Sensing
Chassis Selection
Protocol & Safety
Notebook & Parts
Lab Protocols
The following are the protocols that are used in the lab this year. Some modifications may have occurred during this summer. If there are any changes, those changes can be found in the notebook.
All team members of this year's iGEM team have successfully completed WHMIS and Biosafety Training.
Rehydration
Colony PCR
Gradient PCR
Bacterial Transformation
Construction Technique
Plasmid Prep
Plasmid Prep 2
Vector Dephosphorylation
LB - Agar Plate
Over Night Growth
Glycerol Stock Preparation
PCR Purification
Agarose Gel Electrophoresis
Restriction Digest
Ligation
Site Directed Mutagenesis
Bacterial Transformation
return to top
Thaw 100 μL of competent cells (per transformation) on ice just before they are needed
Add DNA (max 20ul) thawed cells and mix by flicking the side of the tube. Leave on ice for 30 minutes
Heat shock 5 minutes at 37 degrees Celsius
Place on ice for 5 minutes
Add 250ul SOC medium to each tube
Incubate for 30 to 60 minutes with shaking at 37 degrees Celsius. (Note that for Kanamycin containing plasmids always use one hour)
Spin down to remove all supernatant except approximately 100 μL
Plate approximately 30 μL on each of two antibiotic plates
Grow overnight at 37 degrees Celsius
For this protocol we used a couple of controls
Positive Control - pBluescript in TOP10 cells on ampicillin plates
Negative Control - TOP10 cells grown on ampcillin plates
Rehydration
return to top
Biobrick parts are shipped from the registry in a dehydrated from. As such they must be rehydrated before they can be used.
Puncture a hole through the foil with a pipette tip into the well that corresponds to the Biobrick - standard part that you want
Add 15 μL of diH20 (deionized water)
Let the water sit for 5 minutes
Take 2 μL DNA and transform into your desired competent cells, plate out onto a plate with the correct antibiotic and grow overnight. Your goal here is to obtain single colonies
Taq PCR Protocol
return to top
Reagent
Volume ( 1x )
Volume ( 3x )
Volume ( 5x )
Volume ( 15x )
Sterile H2O
36 μL
108 μL
180 μL
540 μL
10X Taq Buffer
5 μL
15 μL
25 μL
75 μL
2mM dNTPs
5 μL
15 μL
25 μL
75 μL
Forward Primer (100 ug/ul)
1 μL
3 μL
5 μL
15 μL
Reverse Primer (100 ug/ul)
1 μL
3 μL
5 μL
15 μL
50mM MgCl2
1.5 μL
4.5 μL
7.5 μL
22.5 μL
Taq Polymerase (50 ug/ul)
0.5 μL
1.5 μL
2.5 μL
7.5 μL
Thermocycler Conditions
1 Cycle - 6 minutes at 95 degrees Celsius
36 cycles of:
1 minute at 95 degrees Celsius
1 minute at 58 degrees Celsius ( this step done at 65 degrees Celsius for higher GC content )
1 minute at 72 degrees Celsius
1 Cycle - 10 minutes at 72 degrees Celsius then HOLD at 4 degrees Celsius
Conditions were varied as needed. For example in cases of longer products all 1 minute times were increased to 1.5 or even 3 minutes
Plasmid Preparation Protocol
return to top
The volumes of different solutions may vary depending on the type of kit we used. The main kits that we used this summer was ordered from Sigma and Qiagen
Harvest Cells
Pellet 5 mL of an overnight culture.
Resuspend Cells
Completely resuspend the bacterial pellet with 200 uL of resuspension solution. Pipette up and down to throroughly resuspend cells until homogenous. Incomplete suspensions will result in poor recovery.
Another rapid way to resuspend the cell pellet is to scrape teh bottoms of the microcentrifuge tubes back and forth five times across the surface of a polpropylene microcentrifuge tube storage rack with 5 X 16 holes .
Lyse Cells
Lyse resuspended cells by adding 200 μL of the lysis solution. Immediately mix the contents by gentle inversion (6-8 times) until the mixture becomes clear and viscous. Do Not Vortex . The lysis was allowed to proceed for 5 minutes before neutralization.
Neutralize
Precipitate the cell debris by adding 350 μL of the Neutralization/Binding solution. Gently invert the tube 4-6 times. Pellet the cell debris by centrifuging at maximum speed for 10 minutes.
Prepare Column
Insert a GenElute Miniprep Binding Column into a provided microcentrifuge tube. Add 500 μL of the Column Preparation Solution to each miniprep column and centrifuge at max speed for 60 seconds. Discard the flow through liquid.
Load Cleared Lysate
Transfer 600 μL of the cleared lysate from step 4 to the column prepared in step 5 and centrifuge at max speed for 60 seconds. Discard the flow through.
Wash Column
Add 750 μL of the diluted Wash Solution to the column. Centrifuge at max speed for 60 seconds. The column wash step removes residual salt and other contaminants introduced during the column load. Discard the flow through liquid and centrifuge again at maximum speed 2 minutes without any additional wash solution to remove excess alcohol.
Elute DNA
Transfer the column to a fresh collection tube. Add 100 μL of molecular biology reagent water to the column. Centrifuge at Max speed for 2 minutes. DNA is now present in the eluate and is ready for immediate use or storage at -20 degrees Celsius.
Construction Technique
return to top
Determine the order of the two parts you will be putting together; the one in front will be referred to as the insert, while the one behind will be referred to as the vector.  Both the vector and the insert need to have their own separate tube, at least in the beginning.
Restriction Digest Protocol
In the Insert Tube...
600 ng of DNA (To figure out the volume, the calculation is 600 / concentration of plasmid.  This gives you volume in μL).
Water, so that the volume of both DNA and water in the tube is 35 μL total
4 μL of React 1 Buffer
0.5 μL of EcoR1
0.5 μL of Spe1
In the vector Tube...
250ng of DNA (To figure out the volume, the calculation is 250 / concentration of plasmid.  This gives you volume in μL).
Water, so that the volume of both DNA and water in the tube is 35 μL total
4 μL of React 2 Buffer
0.5 μL of EcoR1
0.5 μL of Xba1
Put both tubes into the 37°C water bath for one hour.  After, place them into the 65°C heating block for 10 minutes.  This deactivates any enzymes in the tube (which is ok, because by now they’ve done all they need to).     Take the insert out, and put it in a -20°C freezer.
Antarctic Phosphatase Protocol
To the vector tube, add 5 μL of 10x Antarctic Phosphatase Buffer, 4 μL of water, and 1 μL of Antarctic Phosphatase.  We do this to prevent the vector from closing up again without any insert.     Put the tube into the 37°C water bath for 30 mins.  After, place it in the 65°C heating block for 10 minutes.
Ligation Protocol
Take the insert out of the freezer, and add 5 μL of insert and 5 μL of vector to a new tube.  Label the rest of each tube as Unligated, put the date on the tube, and stick it in the -20°C freezer incase your ligation/transformation doesn’t work.  To the single tube of 10 μL mix, add 10 μL of 2x Quick Ligase Buffer, and 1 μL of Quick Ligase.  Let this sit at room temperature for 5 minutes.
You are now done.  If you are going to transform this construction product, add all 21μL to a tube of whichever competent bacteria you're using.
LB - Agar Plate Preparation Protocol
return to top
Weigh 35g of LB-Agar powder mix per litre of media desired. One litre makes 40-50 plates
Select an appropriate flask; the lab autoclave will cause flasks half full and above to boil over! Use a 2L flasks for up to .5 L of media, a 4 litre flask for up to 1.5L, etc
Disolve LB-Agar, using water from one of the wall mounted nanopure filters. Add a stir bar and use a magnetic stirrer to speed things up
Cover the flask with aluminum foil, and secure the foil with autoclave tape. The foil should be somewhat loose (to avoid building pressure in the flask while sterilizing and blowing the foil off), but not so loose that lots of liquid can escape
Put the flask in a plastic autoclave tray, load into the autoclave, and sterilize using the 20 minute liquid program
Once the autoclave finishes venting (which can take twice as long as the sterilization proper), check that the foil covering is still in place. If it is not, the media is contaminated! Unload using the insulated oven gloves
Allow the media to cool until it can be handled without the oven mits. The cold room can be used to speed this up. Alternatively, if a large batch of media is prepared flasks may be kept hot in the prep lab water bath, to avoid all of them cooling at once. Agar polymerization cannot be reversed once it starts (and if it begins to set in the flask you're in trouble!), but media can be kept from setting further by keeping it hot.
Once media is cool, add other desired ingredients. Use the magnetic stirrer to mix, but do NOT add a stir bar now, or the media will be contaminated. (If one wasn't added before, you must do without.) Common additions include:
ampicillin (stock 100mg/ml, final 100ug/ml)
kanamycin (stock 50mg/ml, final 50ug/ml)
chloramphenicol (stock 50mg/ml, final 10ug/ml)
To achieve final concentrations, add 1mL of stock per 1L of media, except for chloramphenicol, where 0.6mL per 1L of media is added instead
Pour directly from the flask into sterile petri plates. Use a quick pass with a bunsen burner flame to snuff out bubbles that form during pouring. Do not subject the plate to continuous heat or the plate will melt, and the heat sensitive ingredients added in the previous step will be destroyed. Bubbles can allow cells to access nutrients without being exposed to the plate's antibiotic, and should be blown out immediately before the gel can set. It's a good idea for one person to pour while another flames bubbles.
Allow the plates to stand right side up overnight, or until the gel sets if they are needed sooner. Plates should be stored upside down to keep condensation from falling on the media. Store petri plates in the plastic bags they ship in, in the 4 degree cold room.
Over Night Growth
return to top
Adapted from Butanerds Protocols from the University of Alberta iGEM Protocols pdf
What you will need
10mL culture tube. Use 16mm x 160mm or 16mm x 125mm
5 mL LB
5 uL 1000X antibiotics
Single colonies on a plate (best not to start an over night from a glycerol stock)
Protocol
Pipet 5uL 1000X antibiotic into culture tube
Add 5mL non-contaminated LB. Do this first. Then add antibiotic
Select a single colony using a sterile toothpick or flamed loop that has been cooled
Place toothpick or loop in culture tube and stir
Remove toothpick or loop and place culture tube in incubator at 37 C overnight shaking vigorously (250 RPM)
Glycerol Stock Preparation
return to top
Adapted from Butanerds Protocols from the University of Alberta iGEM Protocols pdf
What you will need
Overnight bacterial growth
screw captubes
glycerol
Protocol
Pipet 0.5mL of 50% glycerol into 3 1.5 screw cap tubes
Add 0.5mL of overnight culture to each tube
Pipet up and down to gently mix
Flash freeze  in liquid N2 or dry ice/ethanol bath
Place in -80 C freezer when frozen
Agarose Gel Electrophoresis Protocol
return to top
Adapted from Butanerds Protocols from the University of Alberta iGEM Protocols pdf
What you will need
1X TAE
Graduated Cylinder
125 mL flask
Agarose
Gel Pouring Tray
Tape
Gel rig
Ethidium Bromide
Protocol
Measure out 120mL of buffer
Transfer buffer to 125 mL flask
Weigh out enough agarose to make a 1% gel (in our case 1.2 g of agarose was the right amount)
Transfer agarose to 125mL flask
Melt agarose in microwave until solution is almose boiling, stirring every 15-20 seconds (should be around 2 minutes)
Allow agarose to cool (do not let it cool to the point where it is hard)
Add 3 uL of Ethidium Bromide to the cooling agarose
Assemble the gel pouring apparatus by inserting gate into slots. Use a pastuer pipet to run a bead of molton agarose along the edges of the gates to seal the box and prevent leaks
Allow gel to cool until flask can be handled comfortabley
Place comb in the gel rig
Pour agarose into gel tray
Allow to solidify. While the gel is solidifying prepare the samples. Add your sample and 1 uL 10x Loading Dye, 4 uL of DNA and 5 uL of water
Pour 1X TAE over gel so that gel is covered by a 3-5mm buffer
Load samples into lane (Don't forget to load a 1kb+ ladder into one of the lanes)
Hook electrodes to gel apparatus
Run the apparatus at 100V for 30 - 45 minutes (make sure to watch that the dye does not run off the gel)
Visualize the gel and record the results
Restriction Digest
return to top
This protocol is also described as a part of our Construction Technique . Start by selecting the order of the two parts you will be putting together; the one in front will be referred to as the insert, while the one behind will be referred to as the vector.  Both the vector and the insert need to have their own separate tube, at least in the beginning. This is important because it allows for clean addition new parts to a the circuit
In the Insert Tube...
600ng of DNA (To figure out the volume, the calculation is 600 / concentration of plasmid.  This gives you volume in μL).
Water, so that the volume of DNA and water in the tube is 35 μL
4 μL of React 1 Buffer
0.5 μL of EcoR1
0.5 μL of Spe1
In the vector Tube...
250ng of DNA (To figure out the volume, the calculation is 250 / concentration of plasmid.  This gives you volume in μL).
Water, so that the volume of DNA and water in the tube is 35 μL
4 μL of React 2 Buffer
0.5 μL of EcoR1
0.5 μL of Xba1
Put both tubes into the 37°C water bath for one hour.  After, place them into the 65°C heating block for 10 minutes.  This destroys any enzymes in the tube (which is ok, because by now they’ve done all they need to).     Take the insert out, and put it in a -20°C freezer.
Ligation Protocol
return to top
This protocol is also described as a part of our Construction Technique . Start by selecting the order of the two parts you will be putting together; the one in front will be referred to as the insert, while the one behind will be referred to as the vector.  Both the vector and the insert need to have their own separate tube, at least in the beginning. This is important because it allows for clean addition new parts to a the circut
Take insert out of the freezer and ad 5 uL of insert and 5 uL f vector to a new tube
Clearly label the remaining tubes of each (insert and vector) as Unligated, put the date on the tube and place in -20 C freezer in case the transformation does not work
To the single tube containing both insert and vector add 10 uL of 2x Quick Ligase Buffer and 1 uL of Quick Ligase.
Let this sit at room temperature for 5 minutes
Plasmid Preparation/Isolation (For Higher Concentration of Plasmids)
return to top
This protocol is used to obtain higher concentrations of plasmids.
Pick a single colony and inoculate a starter culture of 2-5 mL LB broth + antibiotic and shake for 8 hours at 37 degrees Celsius
Dilute the starter culture by 1/500 to 1/1000
Isolate the cells by spinning at 3200 rpm for 25 mins holding the temperature at 4 degrees Celcius.
Resuspend the bacterial pellet with 10mL of buffer P1
Add 10 mL of Buffer P2 and invert the tube several times and incubate at room temperature for no longer than 5 mins. This step is for lysis of cells.
Add 10 mL of cold Buffer P3 to the lysate and invert several times.
This step is used to neutralize the lysis reaction.
Pour the neutralized lysate through the barrel of the QIAfilter
Cartridge (included in kit) for 10 mins. Let this sit so that the later of proteins, genomic DNA, and detergent will float on top of the solution
Remove the cap from the cartridge outlet nozzle. Gently insert the plunger into the cartridge and filter the lysate into a 50 mL centrifuge tube.
Add 2.5 Ml of Buffer ER to the filter lysate, mix by inverting and then incubate for 30 mins in ice
Equilibrate a Qiagen-tip 500 by adding 10 ml of Buffer QBT and allow gravity to empty the solution.
Pour the lysate through the filter and allow the solution to flow through via gravity flow.
Wash the tip with Buffer QC (30 mL) twice.
Elute the DNA with 15mL of Buffer QN
Precipitate the DNA by adding 0.7 volumes room temperature isopropanol to the eluted DNA. Centrifuge for 90 mins at 3000 rpm.
Empty the supernatant slowly and carefully, the pellet is hard to see.
Wash the pellet with 5mL ethanol (96-100%) and centrifuge for 90 mins at 3000 rpm. Carefully decant the supernatant without disturbing the pellet.
Air dry the pellet for 10 mins and then redissolve the plasmid in buffer TE with a suitable volume.
PCR Purification
return to top
The kit we used over the summer was purchased from Qiagen. This kit is used to purify the PCR product.
Add 5 volumes of Buffer PB to 1 volume of PCR product and then mix.
Place a spin column in a 2 mL collection tube.
Apply the mixture of PB buffer + PCR product through the QIA column and centrifuge for 1 minute.
Discard the flow-through and place the column back in the tube.
Wash the product using 0.75 mL buffer PE and then centrifuge for 1 min.
Discard the flow-through and place the column back into the tube.
Centrifuge the column inside the tube to discard the additional fluid in the column.
Elute the DNA into a clean microcentrifuge tube with buffer EB.
Quikchange Site Directed Mutagenesis
return to top
The kit that we used was purchased from Stratagene.
Design and synthesis two complimentary oligonucleotides containing the mutation.
Prepare the control + sample reaction
Control:
5 µL of 10x reaction buffer
2 µl (10ng) of pWhitescript 4.5kb control plasmid
1.25 µL (125 ng) of control primer 1
1.25 µL (125 ng) of control primer 2
1 µL of dNTP mix
3 µL of QuikSolution
36.5 µL double distilled water to a final volume of 50 µL
+ 1 µL of PfuTurbo DNA polymerase
Sample:
5 µL of 10X reaction buffer
X µL (10ng) of dsDNA template
X µL (125ng) forward primer
X µL (125ng) reverse primer
1 µL of dNTP mix
3 µL of QuikSolution
ddH20 to a final solution of 50 µL
+ 1 µL of Pfu Turbo DNA polymerase
Follow the cycling parameters:
Segment 1: 1 cycle (95 degrees Celsius, 1 minute)
Segment 2: 18 cycles (95 degrees Celsius, 50 secs; 60 degrees Celsius, 50 secs; 68 degrees Celsius, 1 min/kb of plasmid length)
Segment 3: 1 cycle (68 degrees Celsius, 7 minutes)
Add 1 µL of DpnI restriction enzyme to the amplified reaction. Mix gently by pipeting the solution up and down and then incubate at 37 degrees Celsius for 1 hour.
Thaw the XL-10 Gold ultracompetent cells on ice. Add 45 µL to a prechilled 14 mL BD Falcon polypropylene round-bottom tube.
Add 2 µL of the â=-ME mix to the 45 µL of cells
Swirl content and incubate cells on ice for 10 mins and gently swirling every 2 mins.
Transfer 2 µL of the DpnI treated DNA to the cells
Preheat NZY+ broth in a 42 degrees Celsius water bath.
Heat shock cells in a 42 degrees Celsius water bath for 30 seconds
Incubate the tubes on ice for 2 minutes
Add 500 µL of preheated NZY+ broth to each tube then incubate the tube at 37 degrees Celsius for 1 hour shaking at 225-250 rpm.
For the sample, plate 250 µL of cells on appropriate plate. For the control, plate 250 µL of cells on the plate. Before plating, X-gal and IPTG is required for the controls.
Incubate for 37 degrees Celsius for +16 hours.
Retrieved from " http://2009.igem.org/Team:Calgary/Lab/Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
