"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/13 July 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
JULY 13, 2009
CAROL
Gel Extraction of luxCDABE
The main purpose of gel extraction is to ensure that only luxCDABE is present for cloning. The problem with cloning luxCDABE is that we are cloning in smaller pieces of DNA, rather than the gene of interest.
Prepared 0.7% gel and ran the gel for 90 minutes at 90V.
Followed QIAquick Gel Extraction Kit Protocol (please see protocol page for details)
The concentrations that we ended up getting from the gel extraction is low. We re-ran the 0.7% gel with a larger amount of product, however, the kit is only optimal for 3-4 kbs.
Gradient PCR of luxCDABE with the biobrick sites at the end. (We ran out of PCR products to work with)
EMILY
Biosafety 1 Course, Visualization of Colony PCR and Retreaks
Today we had Biosafety 1 in the morning.  After lunch, I started with re-running my gel from Friday from the PCR products in the freezer.  See gel photo below.
Analysis: Lanes 1-4 are J13002-LuxOD47E-B0015 trial 1 (with the J13002-LuxOD47E construct as the insert) and Lanes 5-8 are J13002-LuxOD47E-B0015 trial 2 (with B0015 as the insert).  Lane 9 is a size control with J13002-LuxOD47E and Lane 10 is a size control with only J13002.  If our transformation was successful, we expect to see lanes 1-8 slightly higher then lane 9, which should be slightly higher than lane 10.
Looking at this gel, we do not see a size difference at all between lanes 1-8 and lane 9.  This indicates that either our transformation was not successful or our initial construction digest was not successful.  Because we have all ready gone back to the unligated product once with no results, we will try this again going back to construction.
Today I also made restreaks of the J13002-Lux0D47E constructs on KT1144 plates (we still have no glow) as well as overnight cultures of J13002-LuxOD47E and LuxOD47E witht he purpose of making glycerol stocks tomorrow.
JAMIE
pCS26 plasmid isolation
pCS26 was isolated using QIAquick miniprep spin kit, eluting in 50uL ddH2O.  The Surette vector (pCS26) will be used to clone the signalling circuit and the promoter library.
Concentrations were approximately 50ng/uL.  Two pools of 10 minipreps were vacufuged to final concentrations of approximately 100ng/uL.
JEREMY
Isolating plasmid from LuxPQ in AK and PQ-B-R-OU-B (see July 10), as well as c1λ and aiiA and performing restriction digest on these parts
Purpose: to perform plasmid isolation in order to proceed with sequencing/restriction digest for sequence verification. The plasmid isolation was done using the SIGMA’s Genelute plasmid Miniprep Kit.  Plasmid purity and concentration were measured using the NanoDrop Spectrophotometer. After isolation, all the parts were digested with XbaI and PstI (REact 2 buffer) and run on a 0.7% gel (see below). Although this gel was clearly part-broken, it reveals and/or confirms important information: (1) that the LuxPQ switch into AK plasmid was successful, as was (2) the construction of PQ-B-R-OU-B and that (3) aiiA is present in what the Registry sent us. The c1λ part; however, we will need to do further verification.
KATIE
Complete Version of Activities
This morning, I completed the restriction digest activity in its general form. Now, I have to rename materials so they are no longer general and add the notecards to be given to avatars to each object in the activity and set it so that you cannot read what is within the notecard, which will be compared to a master notecard within the sequencer at the end of the activity. This prevents anyone from just renaming objects in their inventory (“cheating”). I am in the process of modifying the notecard reader script for use within the sequencer and depending on how complicated it ends up being I may incorporate it into the rest of the restriction digest activity and check all notecards for every step if possible. My restriction digest notecard has also been completed to the best of my ability, which will be sent to Mandy this week.
I am now working on bacterial transformation, which I would like to be slightly different from PCR/Gel Electrophoresis and Restriction digest, which have functions different from one another as well (for example, order is important for one). So, I believe this activity will ask the user questions to assure they have some idea of what they are doing, which once again also serves the secondary purpose of preventing people from just renaming objects and being able to complete the activity.
I have created an object that has the ability to create the bacteria avatars by touching it.  Problem: Mandy pointed out that the attach avatar permission only worked for me, which I determined was because I am the owner of the object when it is rezzed. So instead of rezzing the object I decided to give the object to people so that they will become the owner. This means they will now have to take it out of their inventory by attaching it to themselves from their inventory. So it is working, just not the way I would prefer it. There is also varying levels of brightness for the bacteria, which I am unsure of whether to keep or just have it glow when there is enough bacteria.
Tomorrow I will modify the PCR script so that the items for each activity are more specific to that single activity as well as scientifically accurate, including the names for each specific PCR. I will be completing bacterial transformation after I brainstorm questions.
I also have to go back to the water bath script and incorporate it into the activities as well because as of now it is just a stand-alone activity. All I will have to do is change the names of what is added and given to and from the water bath respectively and this also means modifying phosphatase treatment. I am also beginning to make more visual instructions for restriction digest. So far sections that are explained will flash as instructions for that specific section are given. I would like to expand on this later this week.
KEVIN
Construction of Pqrr4+I13500 (RBS and GFP)
Now that I13500 part has been confirmed working by putting R0040 in front, we can move on to the construction of our reporter circuit, which has Pqrr4 promoter instead of R0040. Because the Pqrr4 promoter is in AC plasmid and I13500 is in A, Pqrr4 was cut with SpeI+PstI, and I13500 was cut with XbaI+PstI. The vector, Pqrr4, was phosphatase treated, and then the two parts were ligated together. Next, the product was transformed into TOP10 cells and were plated on AC plates. Overnight growth is needed.
MANDY
Second Life Lab Challenges/ Missions
Finished most of the second lab challenge, which involves AFP (I love AFP), so looked up exact gene length, etc. Also learnt about extraction of genomic DNA from Thane.  Finished the notecard for competent cell preparation, built a freezer that gives competent cells, and the incubator now gives plates with cell cultures.  Started writing up the procedure for DNA extraction. Also began scripting the DNA extraction activity in the lab. So far, only up to lysozyme and proteinase addition (unfortunately), and still have to figure out how to incorporate centrifugation. Scripted the activity so it can only be done when the avatar is holding a pipette.
PATRICK
Working Away
Retrospective Notebook: This entry was not written on this day, but derived later from working notes I made that day.
Continued work on the revised communication system, and the unified DNA script. Finished adding the 'owner id' check to all of the messages in the system. Finished reorganizing all of the UI screens, and activating most of them (though they are mostly empty and nonfunctional besides navigation buttons).
First realized that there is no way to write notecards with scripts... which meant more changes to the DNA script.
Began to question my sanity for setting out to replace old code with new code that does the same thing but better. "It works already, more or less, doesn't it?" (I'm ultimately much happier with the project for having done these rewrites.)
PRIMA
Mini-prep -Lux PQ in Ak
I shadowed Jeremy today. Purpose: Isolate plasmid from Lux PQ in Ak. PQ-B-R-OU-B, aiiA, C1 lambda Procedure: used Qiagen mini-prep directions. The following table shows the labeling scheme.
Plasmid
260/280
Tube numbers
Contents
1
Lux PQ in AK colony 2
2
Lux PQ in AK colony 5
3
Lux PQ in AK colony 6
4
Lux PQ in AK colony 8
5
PQ-B-R-OU-B Colony 11
6
PQ-B-R-OU-B Colony 12
7
PQ-B-R-OU-B Colony 13
8
C1 lambda Colony 1
9
aiiA colony 1
10
aiiA colony 2
11
aiiA colony 3
12
aiiA colony 4
We used buffer EB (heated at 70 degrees) because it's easy to dilute plasmid for larger pieces. Note: when nano-dropping, Blank with EB b/c i eluted with EB buffer.
Today, I also learned how to look up which enzymes to use in restriction digest using the buffer-enzyme sheet.
Following table shows nano dropping results:
Results: Used Nanodrop 1000 Spectrophotometer read at 260 wavelength to determine DNA concentrations
Plasmid
260/280
260/230
Concentration [ng/μL]
Lux PQ psB1AK3 C2
1.83
2.20
102.7
C5
1.82
2.33
92.7
C6
1.84
2.07
101.7
C8
1.85
2.15
113.8
PQ-B-R-OU-B C11
1.83
2.09
145.2
C12
1.85
2.15
125.7
C13
1.87
2.01
157.4
C1 lambda psBAK3 C1
1.75
1.54
63.3
aiiA psB1A2 C1
1.68
0.77
35.9
aiiA C2
1.71
1.73
30.7
C3
1.70
2.32
31.6
C4
1.84
2.39
28.6
STEFAN
Reading ethics papers
Today I read 2 ethics papers:
Erik Parens, Josephine Johnston, and Jacob Moses, “Ethical Issues in Synthetic Biology: An Overview of the Debates,” Woodrow Wilson International Center for Scholars, June 24, 2009.
Selgelid M., 2007, The tale of two studies: Ethics, Bioterrorism, and the Censorship of Science, Hastings Center Report 37, no. 3:35-43.
and took some notes!
VICKI
Systems biology textbook
This is a slight diversion from the other work, but it does provide an intersting spin on what we're doing. This is really interesting as I’ve seen most of what Alon discusses but always in different contexts. I was distracted for a while on feedforward control (which is fascinating in its applications, by the way). I’m working through it right now.
Retrieved from " http://2009.igem.org/Team:Calgary/13_July_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
