<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Lab_Mutant"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Calgary/Lab/Mutant">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Lab/Mutant&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Lab/Mutant&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Lab/Mutant&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Lab/Mutant" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Lab/Mutant</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>University of Calgary</TITLE></P><DIV class="header"><DIV class="title">
UNIVERSITY OF CALGARY
</DIV></DIV><DIV class="menu"><DIV class="outer"><UL id="nav"><LI><P><A href="https://2009.igem.org/Team:Calgary">  HOME</A></P></LI><LI><P><A href="https://2009.igem.org/Team:Calgary/Team">  TEAM</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team#under">Undergraduates</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team#facil">Facilitators</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team#uni">Our University</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Acknowledgments">Acknowledgments</A></LI></UL><LI><P><A href="#">  PROJECT</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/Project">Overall Project</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Parts_Notebook">Notebook &amp; Parts</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab">Lab</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Modelling">Modelling</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Second_Life">Second Life</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Human_Practices">Human Practices</A></LI><LI><P><A href="#">  EXTRA</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/News">News</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Blog">Synthetic BLOGology</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Gallery">Gallery</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Collaborations">Collaborations</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Mascot">Mr. Super D-Lux</A></LI><LI class="last"><P><A href="https://2009.igem.org/Team:Calgary/Sponsors">  SPONSORS</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#ps">Platinum Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#gs">Gold Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#ss">Silver Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#bs">Bronze Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#cp">Community Partners</A></LI></DIV></DIV><P><TABLE cellspacing="0"><TBODY><TR><DIV id="leftmenu"><TD width="210" rowspan="3" bgcolor="#414141" valign="top"><DIV class="heading">LAB INDEX</DIV><DIV class="desc"><A href="https://2009.igem.org/Team:Calgary/Lab"><B>Overview</B></A><A href="https://2009.igem.org/Team:Calgary/Lab/Blog"><B>Updates</B></A>
Circuits
<UL><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Signalling">
Signalling Circuit</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Reporter">
Reporter Circuit</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Mutant">
Mutant Circuits</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Response">
Response Circuit</A></LI></UL><A href="https://2009.igem.org/Team:Calgary/Lab/Quorum_Sensing"><B>Quorum Sensing</B></A><A href="https://2009.igem.org/Team:Calgary/Lab/Chassis_Selection"><B>Chassis Selection</B></A><A href="https://2009.igem.org/Team:Calgary/Lab/Protocol"><B>Protocol &amp; Safety</B></A><B><A href="https://2009.igem.org/Team:Calgary/Parts_Notebook">Notebook &amp; Parts</A></B></DIV></TD></DIV><TD width="750" bgcolor="#414141"><DIV class="desc"><DIV class="heading">MUTANT CIRCUITS</DIV><DIV class="desc">The purpose of the mutant circuits is to test that the reporter circuit is functional so it can be used to test the functionality of the signalling circuit.
</DIV><CENTER><B>Figure 1. Schematic diagram of LuxO D47A (left) and LuxO D47E (right) mutant circuits.</B></CENTER>
 The circuits are almost identical in that they have the same promoter (BBa_R0040), RBS (BBa_B034) and terminator (BBa_B0015). This design will allow for constitutive expression of these proteins. The LuxO D47A mutant mimics the unphosphorylated and thus inactive form of LuxO, meaning that it will not bind to the qrr4 promoter. The LuxO D47E mutant, however, mimics the phosphorylated and thus active form of LuxO, and will thus bind to the qrr4 promoter and induce expression of downstream genes. Once these circuits are tested with a non-Biobrick reporter (in the KT1144 cells) they are used to test whether the reporter circuit is functional. If in the presence of the LuxO D47A mutant the reporter cells do not glow and if in the presence of the LuxO D47E mutant the reporter cells do glow, we know the reporter circuit is functional.
<B>CHARACTERIZATION</B>
In order to use the mutant circuits to test whether the reporter circuit is functional, we must first verify that the mutant circuits themselves are functional. This was done by transforming each mutant circuit separately into KT1144 cells that have the qrr4 promoter followed by GFP on a cosmid (this was provided by Bonnie Bassler - see acknowledgments). The expectation was such that without LuxO D47E, the KT1144 cells would have basal levels of fluorescence, but upon the addition of the mutant, fluorescence would increase. This is because LuxO D47E mimics the phosphorylated and thus active form of LuxO and should bind to the qrr4 promoter and induce expression of GFP. Below depicts the results of the fluorescent readings and the protocol can be found under the figure. 
<B>Figure 2. Fluorescent readings when testing LuxO D47E mutants in KT1144 cells and testing the reporter circuit with functional LuxO D47E mutants.</B> This graph is divided into two lines of cells and a positive control. The left hand bars depict the KT1144 cells with and without LuxO D47E. This graph shows that without the LuxO D47E mutant in the KT1144 cells, fluorescence reads at 267, whereas with the mutant, fluorescence reads at 4219. This increase in fluorescence shows that the LuxO D47E mutant is functional. These mutants can now be used to test the reporter circuit, which is shown in the next line of cells that shows reporter circuit with and without LuxO D47E in order to determine whether the reporter circuit is functional. Fluorescences levels increase upon the addition of the LuxO D47E mutant and therefore the reporter circuit is functional. See 'reporter circuit' on the side bar for more information on testing the reporter. The positive control is the TetR promoter followd by an RBS and GFP. TOP10 cells with pBluescript were used as a negative control and to blank the plate reader
<B>Protocol for fluorescent readings</B>1. Grow overnight cultures of each sample
2. Power on the Bio-tec Synergy HT plate reader, or another plate reader, and KC4 application.
3. On a black 96 well plate, aliquot samples in required wells.
4. Go to wizard, and change the reading parameters to the following settings:
Reader: absorbance
Reading type: Endpoint
Wavelength: 570nm (it is as close as it gets to OD600)
5. Click ok.
6. Again, go to wizard, then in layout, mark the wells that contain samples and blank. Click ok.
7. Press the read button
8. Match the OD600 levels by diluting with corresponding Luria-Bertani (LB) broth.
9. Measure OD600 again.
10. Once OD600 are matching for all samples, serial dilute them (1 in 10, 1 in 100). To serial dilute, aliquot 100uL of original culture into a new tube containing 900uL of corresponding LB broth (1 in 10). To make 1 in 100, aliquot 100uL of 1 in 10 dilution into a new tube containing 900uL of corresponding LB broth (1 in 100).
11. Go back to wizard, change the reading parameters to the following settings*:
Reader: Fluorescence
Reading type: Endpoint
Excitation: 485/20
Emission: 528/20
Optics position: Top
Sensitivity: automatic adjustment, scale to high or low well. 
Top probe vertical offset: 3mm
12. Click ok.
13. Again, go to wizard, change the layout of the cells.
14. Read.
*GFP reading protocol was obtained from <A href="http://www.mnstate.edu/provost/GFPPlateReaderAssayProtocol.pdf">Minnesota State University</A>
A paper regarding the construction and testing of the mutant circuits can be read below or <A href="https://static.igem.org/mediawiki/2009/3/34/Calgary_LuxO_mutants.pdf">HERE</A>.
<CENTER><A title="View LuxO Mutants on Scribd" href="http://www.scribd.com/doc/21377150/LuxO-Mutants" style="margin: 12px auto 6px auto; font-family: Helvetica,Arial,Sans-serif; font-style: normal; font-variant: normal; font-weight: normal; font-size: 14px; line-height: normal; font-size-adjust: none; font-stretch: normal; -x-system-font: none; display: block; text-decoration: underline;">LuxO Mutants</A></CENTER></DIV></TD></TR></TBODY></TABLE></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Calgary/Lab/Mutant">http://2009.igem.org/Team:Calgary/Lab/Mutant</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Lab/Mutant" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Lab/Mutant" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Lab/Mutant&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Lab/Mutant&amp;oldid=162981" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>