"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/3 July 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
JULY 3, 2009
CAROL
Colony PCR
plates resulted in two colonies and a colony PCR was initiated. After the PCR was completed, we ran the product on a 0.6% agarose gel but no bands appeared suggesting that colonies did not uptake plasmid that had the gene of interest ligated to it.
FAHD
Team Presentations
Today we had our team presentations. I presented on behalf of the Human Practices aspect of our project.
IMAN
DIVISION
As it was mentioned, division function gets an environment as its input. Then it looks at the elements existing in the cells. Those elements that should be doubled before the divisions (like Qrr4 promoter, RNA polymerase, LuxPQ and etc) are doubled and those elements that should be equally divided between new cells (like AI2, LuxU.p, LuxO.p, etc) are divided between cells. For those elements that are divided between new cells, it is not 100% that each new cell gets an equal amount of each one. There is a random generator function which produces a number between 1.90 and 2.10, and this number is used to divide the elements between the cells. Each rule has a label that determines this rule belongs to which cell. When division occurs, rules of the new cells should be labelled. Another challenge is to label all the rules of all the new existing cells. Moreover, we need to determine a time period as life cycle for each generation of the cells. We used the time that Gillespie’s algorithm assigns to the system for this purpose. At this stage, we have added an important aspect of “emergent properties” to our bacteria population by adding division to it.
Once again, two meetings this week. One on Thursday with Lindsay project, and the other one on Friday with iGEM.
JEREMY
Verification of LuxPQ into psB1AC3 construction
Purpose: to verify the presence of LuxPQ into psB1AC3 by colony PCR; to isolate plasmid; to send isolated plasmid for sequencing. A pTaq colony PCR was set up using the following sets of forward and reverse primers: LuxPQ and BBK CP. The following conditions were used for both sets of primers: 94ºC for 6 minutes; 36X (94ºC for 30 seconds; 55ºC for 45 seconds; 72ºC for 4 minutes); 72ºC for 10 minutes; held at 4ºC. The products were then run on a 0.8% agarose gel (see figures below). We can see that colonies 1, 6 and 7 all have bands of the expected size (~3.9kb) for both primers, indicating a successful transformation of LuxPQ into psB1AC3. Plasmid was isolated from these colonies using the GenElute Plasma MinPrep Kit. Colonies 6 and 7 were then sent to the University of Calgary’s DNA sequencing laboratory with BBK-CP-F/R primers.
KEVIN
Transformation of fluorescent proteins
I transformed E0040 (GFP), E0032 (YFP), E0022 (CFP), I13500 (GFP+RBS), and I13502 (RFP+RBS) into TOP10 cells for testing of R0040 and... for drawing bacterial pictures! Orange FP and Blue FP were not available for distribution, and purple FP was not added to the iGEM registry.
PATRICK
iGEM Island Terrain
Retrospective Notebook: This entry was not written on this day, but derived later from working notes I made that day.
We finally gained access to our new home LINDSAY Island not long ago, but did not have any permissions to create objects there, or edit the terrain. That all changed today, after meeting with out supervisor (and keeper of admin passwords) Dr. Christian Jacob. It was a sight to behold, the *instant* the terraforming permission was switched on, the whole team began creating mountains and gouging our canyons everywhere, right from beneath their own feet.
After a while, we got more organized about the island's terrain, and found a program called Backhoe for designing SL terrain externally. Today saw an upgrade to the island's terrain, expanding the underwater zone assigned to the Synthetic Kingdom (which would grow ever larger over the summer, slowly submerging more and more land).
The rest of the day was spent on more HUD development, determining which navigation buttons will belong on which screens, and setting up a script to 'listen' for binding events (such as TetR subunits binding each other, or binding DNA after dimerizing). This unbind list could then be called on to send messages dismissing the binding, giving the user back their separate parts and control over their biobricks.
First occurred to me that, if I meant for multiple copies of this system to be in use by multiple people in the same area, I might have to insulate one person's biobricks from the the others! Began a campaign to upgrade all of the publicly visible messages in the simulator's components to include their owner's ID, and to check for that ID on all incoming messages. Now your friend's HUD won't have the power to unbind your complexes, for example.
PRIMA
Isolate Plasmids
-Shadowed Jeremy today. Purpose: isolate plasmids from overnight cultures containing Top 10 competent cells with Lux PQ in psB1AC3
Protocol: Geneflute Plasmid mini prep kit
After mini-prep we nano-dropped to determine the plasmid concentration at 260 wavelength
Plasmid
260/280
260/230
Concentration [ng/μL]
LuxPQ Top 10 C24C11
1.82
2.23
160.2
LuxPQ Top 10 C24C12
1.89
2.34
112.8
LuxPQ psB1AC3 C11
1.89
2.17
89.4
LuxPQ psB1AC3 C6
1.88
2.11
82.5
LuxPQ psB1AC3 C7
1.88
2.09
79.0
For sequencing: C7 = send 8.34 microL Sequencing Table
Reaction #
Name'
size
Primer
Tm
1
LuxPQ-BBK-C6-F
6680
BBK-CP-F
55
2
LuxPQ-BBK-C6-R
6680
BBK-CP-R
55
3
LuxPQ-BBK-C7-F
6680
BBK-CP-F
55
4
LuxPQ-BBK-C7-R
6680
BBK-CP-R
55
Retrieved from " http://2009.igem.org/Team:Calgary/3_July_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
