<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Lab_Blog"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Calgary/Lab/Blog">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Lab/Blog&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Lab/Blog&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Lab/Blog&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Lab/Blog" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Lab/Blog</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>University of Calgary</TITLE></P><DIV class="header"><DIV class="title">
UNIVERSITY OF CALGARY
</DIV></DIV><DIV class="menu"><DIV class="outer"><UL id="nav"><LI><P><A href="https://2009.igem.org/Team:Calgary">  HOME</A></P></LI><LI><P><A href="https://2009.igem.org/Team:Calgary/Team">  TEAM</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team#under">Undergraduates</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team#facil">Facilitators</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Team#uni">Our University</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Acknowledgments">Acknowledgments</A></LI></UL><LI><P><A href="#">  PROJECT</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/Project">Overall Project</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Parts_Notebook">Notebook &amp; Parts</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab">Lab</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Modelling">Modelling</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Second_Life">Second Life</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Human_Practices">Human Practices</A></LI><LI><P><A href="#">  EXTRA</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/News">News</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Blog">Synthetic BLOGology</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Gallery">Gallery</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Collaborations">Collaborations</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Mascot">Mr. Super D-Lux</A></LI><LI class="last"><P><A href="https://2009.igem.org/Team:Calgary/Sponsors">  SPONSORS</A></P></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#ps">Platinum Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#gs">Gold Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#ss">Silver Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#bs">Bronze Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Sponsors#cp">Community Partners</A></LI></DIV></DIV><P><TABLE cellspacing="0"><TBODY><TR><DIV id="leftmenu"><TD width="210" rowspan="25" bgcolor="#414141" valign="top"><DIV class="heading">LAB UPDATES</DIV><DIV class="desc">
Every Wednesday (started at the end of June), our lab team uses our <A href="http://igemcalgary.blogspot.com"> Synthetic BLOGology</A> blog to update both the public and the rest of the team of their progress in the lab project. For more details regarding Synthetic BLOGology, please click <A href="https://2009.igem.org/Team:Calgary/Blog">HERE</A>.
</DIV><DIV class="heading">INDEX OF UPDATES</DIV><DIV class="desc">
-<A href="https://2009.igem.org/Team:Calgary/Lab/Blog#jul">July</A>
-<A href="https://2009.igem.org/Team:Calgary/Lab/Blog#aug">August</A>
-<A href="https://2009.igem.org/Team:Calgary/Lab/Blog#sep">September</A></DIV><DIV class="heading">LAB INDEX</DIV><DIV class="desc"><A href="https://2009.igem.org/Team:Calgary/Lab"><B>Overview</B></A><A href="https://2009.igem.org/Team:Calgary/Lab/Blog"><B>Updates</B></A>
Circuits
<UL><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Signalling">
Signalling Circuit</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Reporter">
Reporter Circuit</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Mutant">
Mutant Circuits</A></LI><LI><A href="https://2009.igem.org/Team:Calgary/Lab/Response">
Response Circuit</A></LI></UL><A href="https://2009.igem.org/Team:Calgary/Lab/Quorum_Sensing"><B>Quorum Sensing</B></A><A href="https://2009.igem.org/Team:Calgary/Lab/Chassis_Selection"><B>Chassis Selection</B></A><A href="https://2009.igem.org/Team:Calgary/Lab/Protocol"><B>Protocol &amp; Safety</B></A><B><A href="https://2009.igem.org/Team:Calgary/Parts_Notebook">Notebook &amp; Parts</A></B></DIV></TD></DIV><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">
WHAT WE'VE BEEN UP TO IN LAB:
</DIV><DIV class="desc">
These updates have been posted with newest ones at the top.
</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">August 20, 2009 - Jamie</DIV><DIV class="heading">Labbing it up!</DIV><DIV class="desc">
Hello virtual world of synthetic biologists, engineers and the rest of the cool kids floating around in cyber space!

Jamie is back with a short and sweet update from the lab side of things.

This past week Emily (with a little Vicki-probably about 0.1 of her to be exact) alongside with Kevin proceeded to test the reporter circuit with the mutant circuits.  This involved a quick plasmid switch of the mutant Vicki, I mean OD47A into psB1AC3 simply due to antibiotic resistance and selection pressure.  Overnights were grown and tested in the plate reader Synergy HT.

Jeremy and myself (Jeremy did most if not all of the pipetting; I might have plated one or two transformations here and there) just finished up the signalling pathway in pCS26.  Sequencing results were analyzed with tools available over at ncbi.nlm.nih.gov (i.e. BLAST) and the signalling cascade is now officially completed with glycerol stocks, plasmid DNA and restreaks ready to go!

I (sort of) lied when I said the luxPQ-luxOU construct was done.  Just a tad.  Carol's project involving the synthetic sigma70 promoter is currently on hold as not only is Carol off in Vancouver on a <SPAN style="font-weight:bold;">well deserved</SPAN> holiday but new primers need to be made with the degenerate bases.  We hope to resume progress on this after next week. 

Meanwhile, I myself am fiddling around with mineral oil, plate reader, 96 well microplates,<SPAN style="font-style:italic;">Vibrio harveyi</SPAN><SPAN style="font-style:italic;">Salmonella typhimurium</SPAN>, <SPAN style="font-style:italic;">Escherichia coli</SPAN>, centrifuges in hopes of isolating AI-2.  Please note this is all done without the handy dandiness of multi-channel pipettes or access to a VICTOR; nevertheless learning to pipette effectively and how to calibrate a different model of plate is useful and exciting!

Without further ado, on behalf of the lab team I will like to sign off (not permanently though ;) - can't get rid of us THAT easily) for the summer.  Thank you everyone for following us throughout the sunny (and rainy) summer days and I look forward to blogging again once the school year rolls around.  So until next next week a.k.a. September, see you later :D!
</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">August 13, 2009 - Jeremy</DIV><DIV class="heading">CLOSE UP of iGEM Calgary's LAB</DIV><DIV class="desc">
Hey guys! Summer is coming to a close pretty quick here, and so is iGEM! We are working tremendously hard to try and bring our project to completion! Here’s where we stand as of today (August 12, 2009):
<B>Signaling Circuit</B>
LuxPQ-B0015-R0040-LuxOU-B0015 has been successfully verified in psB1AC3. But that’s old news. We have recently cloned this construct into the pCS26 vector, whose cloning site originally had LuxCDABE flanked by NotI sites. We cut our signaling construct with NotI enzyme, but this does not guarantee that our construct was cloned in the appropriate direction. We transformed the product into TOP10 and XL Gold competent cells, and grew them on Kanamycin plates (pCS26 has resistance for this antibiotic). We have isolated plasmid from eight colonies, and we will verify whether the direction of our construct is correct with a primer that anneals just outside the cloning site on the supposed LuxPQ side. This primer will be paired with a LuxPQ Reverse primer. If we see a PCR product of just over 4kb, we have cloned the construct in the right direction, if we get smearing, however, we have not cloned the product in the right direction.

Moving forward, we require the sigma 70 promoter library that will control the expression levels of LuxPQ in our system. We have had trouble cloning this into the pCS26 vector, and will be trying this again from the start.
<B>Reporter Circuit and Mutants</B>
Pqrr4-B0034-GFP has been successfully verified in psB2K3. Both mutants have been verified in the following constructs in psB1AC3: R0040-B0034-LuxO D47A-B0015 and R0040-B0034-LuxO D47E-B0015. The Pqrr4-B0034-GFP is currently in TOP10 cells, and being made competent once again in order to allow the subsequent transformation of the mutants into the cells. The purpose here is to test the mutant circuits to see if they are working.
<B>Acquiring AI-2</B>
The purpose of constructing all these circuits is to make an AI-2 signaling system. So what else do we need? AI-2! We are currently isolating AI-2 from a species of Salmonella by centrifuging and taking the supernatant that should contain the AI-2. Many controls will be used in order to test whether AI-2 is actually present in the supernatant, namely using Vibrio Harveyi as a response which should glow in the presence of AI-2 because it naturally has the AI-2 signaling system. Once verified that we have AI-2, it can then be used to see if our system responds to AI-2.
</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">August 5, 2009 - Carol</DIV><DIV class="heading">Some Progress Finally!!</DIV><DIV class="desc">
Hi Everyone! Thanks for reading our lab blog once again! Just a quick lab update for all you readers this week. Emily has finally completed her mutant circuit for
luxOD47E. Congrats to her! Jeremy and Jamie are currently working on several things right now. First, they finally performed a plasmid switch from pSB1AK3 to pSB1AC3 for luxPQOU. Now that they have successfully switched plasmids, they will be constructing luxPQOU into pCS26 (surette vector) by cutting with NotI enzyme. Finally, they are working on verifying cllamda. Unfortunately, after many enzyme digestions, they are unable to verify that the sequence is in the vector. They are currently trying again and hopefully will get results later this week. Kevin is working on the reporter circuit and he is verifying that circuit today with restriction enzymes. He is going to look into how to test the mutant circuits this week as well. Finally I am still stuck with the sigma 70 promoter. Unfortunately, I am still unable to get any colonies. We are currently looking at other ways to optimize the results. Hopefully, I'll have better results to report next week! Thanks for reading! 
</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">July 22, 2009 - Emily</DIV><DIV class="heading">Update from Circuit World</DIV><DIV class="desc">
Hi, this is Emily again! I'm part of the wetlab portion of our team and today I'm going to be updating you on the progress we've made over the past week or so. I'll start with the signalling circuit where Jeremy, Jamie and Carol have been concentrating their efforts.

This week, Carol has been working on our promoter library which Jeremy touched on last week. Essentially Carol is trying to find a promoter that provides the best expression of Luciferase so that we can find the optimal expression of LuxPQ. We don't know the optimal expression of PQ however as more epression of LuxPQ does not necessarily mean more expression of Lucierase. This is therfore the purpose of the Promoter Loibrary, to find the optimal expression of LuxPQ. To do this, Carol has set up PCR reactions with 16 forward primers and 16 reverse primers. She then digested the PCR products and the Surette vector with ZhoI and BamHI enzymes and is currently in the process of ligating them in order to trasnform them into competant cells. She has tried three different types of Ligation, QuikLigse, T4 NEB ligase and a week long ligase. The first two did not produce cells that glow (that express Luciferase) and there were not many cells that grew. The week long ligation will be done tomorrow and we are hoping that the results will be promising!

Last week Jamie and Jeremy finished the signalling circuit with LuxOU and LuxPQ. This week they have been working on a plasmid switch. Thw signalling circuit was contructed in an AK vector, however we need to now move it into a vector with K resitance (the surrette vector). Because moving something from AK to K does not offer any antibiotic selection pressure, we need to first do a plasmid switch to get it into an AC vector, and then we can move it into the Surrette vector.

Next we have the reporter circuit which Kevin has been working on. This week Kevin has been making a constrcut with pqrr4, RBS and GFP in order to test the functioning of the Signalling circuit. Kevin has been working on this construction this week and is waiting for

Finally we have our two Mutant circuits: LuxOD47E and LuxOD47A, which are being worked on by Emily and Vicki respectively. As of last week, Vicki had seccessfully completed and sequnced her circuit. This week she has been working on her pro paper that, with some additons and changes is well on its way to being done. Unfortunately, LuxOD47E and I are not having so much fun and I am pretty much at the same place that I was last week, trying to get the B0015 terminator in. This week's attempt seems much more promising as yesterday I did a verificatioon digest with goiod results and have since sent off a colony for sequncing. I have my fingers crossed that B0015 is in there and my circuit will finally be done.

Well that's about all for this week. We still have a fair ways to go in the lab, so hopefully we have more successes to report next week!
</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">July 15, 2009 - Jeremy &amp; Jamie</DIV><DIV class="heading">Why Annotations are Useful</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">July 8, 2009 - Carol</DIV><DIV class="heading">Carol Chan Battles Lux CDABE</DIV><DIV class="desc">
Hi everyone, it’s Carol again! I won’t re-introduce myself again since I wrote the modelling blog a few days ago. I don’t have much to report since I’ve been having bad luck in the lab lately, actually from the start! I am working with Kevin (the nice individual) to construct the reporter circuit for the project; however, due to my lack of lab skills, I’m delaying the whole project and leaving Kevin with nothing to do! I’m just kidding. I spent a few weeks trying to concentrate DNA plasmids for LuxCDABE sequence in topo vector. After many failed maxi-prep and many mini-preps (with the help of my favourite lab equipment, the vacifuge), I was able to concentrate my sample. As well, before Biobrick construction, I was left with a difficult task of single site mutagenesis. For some reason (with my luck) after one trial I was able to mutate a specific site within the LuxCDABE gene. This past week, I was unable to successfully clone and transform the LuxCDABE into the Biobrick vector. The lab team are trying to think of other ‘innovative’ ways to get this large gene cloned into Biobrick vector. Hopefully, next week, I’ll be able to show some better results, as of now, only time will tell! Talk to you guys on Monday!
</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading"> July 8, 2009 - Kevin</DIV><DIV class="heading">Kevin Loves Rainbows</DIV><DIV class="desc">
I, Kevin Shin, being a nice individual, am also involved in wet lab part of our team. Carol and I are responsible for our Reporter circuit, which involves Qrr4 promoter and LuxCDABE gene. As of now, I have completed verifying and glycerol stocking last year’s Pqrr4 part, and am waiting for LuxCDABE part (which is much longer, meaning harder to work with) to be done.

While waiting, I am conducting an exciting side project involving a variety of FLUORESCENT proteins!!(and dry ice...) Woot! Although I am really disappointed at not being able to gain access to purple, orange, and blue fluorescent proteins, we have secured a supply of red, green, yellow, and cyan for me to have fun with. Today, I will be transforming plasmids with these genes in them and by tomorrow, I should be able to draw some bacterial pictures! We will keep you posted on those glowing masterpieces.
</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">July 8, 2009 - Emily</DIV><DIV class="heading">How Negative Controls Became Positive</DIV><DIV class="desc">
Hi, I'm Emily and I’m going into second year Biomedical Sciences. When I’m not fighting with my LuxOD47E gene, I’m usually highland dancing or playing the Oboe. In the lab, I’m in charge of the mutant LuxOD47E circuit This week I am excited to say that sequencing has confirmed that I've finally biobricked my gene of interest, LuxOD47E. Yay! This has taken a lot longer than anticipated due to several negative control contamination issues (negative controls are not my friends) and a battle with strange reappearing bands in restriction digest and PCR products. Nevertheless, this gene is now biobricked and it is on to the construction of my circuit! This week I will be trying to get the J13002 promoter in front of LuxOD47E as well as the BOO15 terminator behind. I’ll be doing this through restriction digest with EcoRI, XbaI and SpeI followed by an awful lot of verification. Maybe, just maybe I’ll see some clean negative controls! If this is successful, my circuit should be completed and sequenced by Monday!</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">July 8, 2009 - Jeremy</DIV><DIV class="heading">Jeremy Kubik Wrestles with LuxPQ LIVE!</DIV><DIV class="desc">
My name is Jeremy Kubik and I am part of iGEM Calgary’s wet lab and marketing team. This is my first time participating in iGEM and I am having a great time working with the other students on this awesome team. Outside of the lab, I enjoy competitive sports, all the way from swimming with the Varsity team at the U of C to ripping it up on the streets of Calgary for a little hockey action. I also enjoy long walks on the beach, sunrises and fine wines.

Within the lab, I am taking care of part of the signaling circuit. Our overall project is to create a Quorum Sensing (QS) system with the signaling molecule autoinducer-II (AI-2) as the input to the system. My collaboration with Jamie will eventually lead to the construction of a circuit with the genes LuxPQ and LuxOU, all of which code for important proteins with respect to the transmitting the AI-2 to induce a specified response (which was recently decided to be a protein output that degrades biofilms!). So where am I as of today? Stuck. Well, not really stuck, but I have had some difficulties with LuxPQ. The part is verified to be in the TOPO vector and to have a length of 3.8kb, and I have had quite some trouble simply cloning LuxPQ into a BioBrick Vector (psB1AC3). I performed a construction last week and finally got some colonies, of which I ran a colony PCR, isolated plasmid, and have sent plasmids of two colonies down to sequencing. I will find out in a few hours whether LuxPQ has been successfully cloned into the BioBrick vector. If it is, I can begin construction with Jamie’s circuit (LuxOU with promoter and terminators). If it has not been successfully cloned, it is back to the drawing board: trying different conditions to transfer LuxPQ from TOPO vector into a BioBrick vector.</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">July 8, 2009 - Jamie</DIV><DIV class="heading">Jamie + Synthetic Biology + Blogging = Jamie on Youtube</DIV><DIV class="desc">
So I decided to be lazy and outdo everyone on my team by doing some snazzy Web 2.0 related thing: VIDEO BLOGGING!!!!! Amazing stuff.

One downfall of video blogging is it is HARD to edit stuff out. :( And after reading everyone else's blog post and realizing I forgot to introduce myself OUTSIDE of iGEM (do I even exist???) and being to lazy to re-record anything I will include the following disclosure:

Jamie enjoys doing iGEM from 9-5 Monday to Fridays. Sometime 10-6, dependent on how late I sleep in :D I will include my extracurriculars next week (I will have had enough time by then to think of at least two extracurriculars so I do not look like a complete loser). Anyways, enjoy the video.
</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">July 8, 2009 - Vicki</DIV><DIV class="heading">Building Circuits: A Few Successful Moves to the End</DIV><DIV class="desc">
Hi! You heard from me last week in my third-person commentary and now I’m back in the first-person flesh. My name is Vicki and I recently graduated from the Engineering Science program at the University of Toronto, with a major in biomedical engineering. I grew up in Calgary and am very excited to help my hometown kick some synthetic biology butt. When I am not indulging in the satisfaction that only a properly-sequenced synthetic BioBricked circuit can provide, I am usually either swimming, skating, biking, learning German or a combination of any of the above (ever try riding a bicycle whilst on skates? It’s as perilous as it sounds!).

Over the last few weeks, I have been deeply entrenched in converting a mutant protein sequence in a TOPO plasmid into a fully functional and biobricked sequence in a pSB1AK3 plasmid, complete with the appropriate and properly-integrated promoter, RBS and terminator sequences. Indeed, it has been a most gruelling month of gradient PCRs, colony PCRs, plasmid isolations, restriction digests, ligations and – craziest of all – understanding what I’m doing and explaining it to the lab group in coherent sentences! Because even though the principles of synthetic biology and biobricking are supposed to make genetic engineering so easy that even an engineer like me can work with it, it’s really quite challenging when you cannot see what is really happening in terms of molecular interactions on the nano-scale and smaller. Although many parts of the project have flummoxed me, I am developing a better appreciation of what I am doing as I gain more experience in the lab.

So, here is my work to date.
<B>Battle 1</B>: Vicki vs gradient PCR of LuxOD47A in TOPO. The purpose of this step was to use biobrick gene-specific primers to make copies of a biobricked version of LuxOD47A.

Winner: Vicki
<B>Battle 2</B>: Vicki vs ligation of LuxOD47A into the pSB1AC3 plasmid. This is so that the gene sequence can be integrated into competent TOP 10 bacteria. Prefix-end cuts were made with EcoRI and XbaI (in separate samples).

Winner (VK vs EcoRI cuts): EcoRI cuts L. The restriction digests of these look like someone painted white-out in the 12 kb range just to spite me.

Winner (VK vs XbaI cuts): Vicki. The EcoRI enzyme sample that wasn’t up to par is spending its days in solitary confinement at room temperature. Indeed, things are so much cleaner when enzymes decide to cooperate.
<B>Battle 3</B>: Vicki vs the sequencing machine, part I. After a successful restriction digest of the XbaI-cut samples, I sent them down for confirmation that my PCR and restriction digest results weren’t just lying to me.

Winner: Vicki
<B>Battle 4</B>: Vicki vs integration of promoter and terminator sequences, attempt I. I tried both to see which would work best.

Winner (VK vs J13002 promoter): J13002 promoter L. That one didn’t integrate in any of the colonies that I PCR’d and RD’d

Winner (VK vs B0015 terminator): Vicki. We’ll move forward with a newly-made LuxOD47A-B0015 construct, while the attempted J13002-LuxOD47A samples can go roast in the autoclave.
<B>Battle 5</B>: Vicki vs the sequencing machine, part II. Yea, I know I skipped a lot of steps here. It wouldn’t tell you anything interesting beyond what you’ve already read. After good restriction digests and colony PCR results that would have been fine if not for a superfluous negative control band that didn’t show up anywhere else, we sequenced anyway. We had a scare when a simple [Crtl+F _ (B0015 sequence)] didn’t yield any results on the text file, but were reassured when we blasted the sequences against each other as that algorithm accounts for the reverse sequence given by the reverse primers that I used.

Winner: Vicki, with credit to BLAST
<B>Battle 6</B>: Vicki vs the integration of the nefarious promoter sequence. Lest the J13002 samples go the same way as our uncooperative EcoRI enzyme, we decided to attempt this integration thing one last time. Two colonies behaved appropriately, and the J-part lived to see another day.

Winner: Vicki
<B>Battle 7</B>: Vicki vs the sequencing machine, part III. Although I had to wait four days for my results, they were what we expected.

Winner: Vicki

This takes us to now. I need to make more of the biobricked sequence in question by letting it grow in bacteria, which I’ll do later today so that it can grow overnight. Results to come!
</DIV></TD></TR><TR><TD width="750" bgcolor="#414141" valign="top"><DIV class="heading">July 2, 2009 - Vicki</DIV><DIV class="heading">Some Musings from Mutant-Circuit World</DIV><DIV class="desc">
The last few days have been flavoured by a plethora of successes, not-quite-successes, and other incidences of interest and amusement.
<B>THE GOOD</B>: Jamie has usurped the throne of AI-2 Circuitland. With a complete and properly-sequenced LuxOU component of the circuit, complete with the necessary promoter and terminator sequences, Jamie gets to sit back, chillax and admire his work as he helps the rest of the lab group complete their work. Congrats, Jamie, and B0015-R0040-LuxOU-B0015 FTW!
<B>THE BAD</B>: Negative controls FTL! Emily and Vicki have both been plagued by unsightly bands in the negative control lane, despite using new and purportedly uncontaminated equipment in their PCRs and restriction digests every time. Of particular bamboozlement, Vicki’s negative control in her latest colony PCR had a band at 1kb that did not appear anywhere else in the gel. Nevertheless, the two will continue to move forward with their experiments. The sequencing results of Vicki’s LuxOD47A BBk circuit showed that she does indeed have the proper product present. And Emily is set to conquer the colony PCR, so hopefully we’ll see the awesome results of that at the end of the day.
<B>THE UGLY</B>: Vicki spat into a K-laced plate and placed it in the incubator to see what would happen. She was most disappointed to see that the plate was cleaner when she returned than when she first left it there. She’ll try again with a plate free of antibiotics next week. Results to come!
</DIV></TD></TR></TBODY></TABLE></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Calgary/Lab/Blog">http://2009.igem.org/Team:Calgary/Lab/Blog</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Lab/Blog" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Lab/Blog" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Lab/Blog&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Lab/Blog&amp;oldid=70689" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>