"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/28 July 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
JULY 28, 2009
CAROL
Analyzing ''E.coli'' Colonies
Minimal growth of E.Coli on LB plates with Kan resistance. There were only 5 colonies found. None of the colonies were glowing. We are not sure what 'glowing' would look like.
Ordered primers that would allow us to clone part BBa_B0040 in front of the luciferase operon. The following is the primers that were ordered.
XhoI-R40-F: GCGCTCGAGTCCCTATCAGTGATAGAG BamHI-R40-R: GCGGGATCCGTGCTCAGTATCTCTATC
These primers would allow us to attach XhoI and BamHI sites onto part BBa_B0040 by PCR and then we are able to clone this part into pCS26 vector with luciferase as the reporter. This should allow us to see the glowing from the luciferase operon in the plasmid.
CHINMOYEE
Descriptive Title of What You're Doing
WIKI CODING HERE
EMILY
Colony PCR Verification of J13002-LuxOD47E-B0015
Today I did a Colony PCR in order to verify the presence of the B0015 terminator in our contruct.  I followed the protocol and the cycling conditions from July 7th 2009.  PCR prodcuts were visualized on a 1% agarose gel run with 1.0kb+ GeneRuler DNA Ladder.  See gel photo below.
Analysis: Lane 1 is a size control of BBK LuxOD47E, lane 2 is another size control of J13002-LuxOD47E, Lanes 3-6 are J13002-LuxOD47E-B0015 trial 2, Lanes 7-10 are J13002-LuxOD47E-B0015 trial 3, lane 11 is left blank and lane 12 is a negative control.
From this gel, it looks like the B0015 terminator may have been cloned into colonies 1,2 & 3 of trial 2, as the bands in these lanes appear slightly higher than the band in lane 2 which is J13002-LuxOD47E.  I will proceed by preparing overnight cultures of trial 2, colonies 1-3 in order to isolate plasmid tomorrow and do a verification digest, before hopefully choosing a colony to send down for sequecning.
FAHD
Calling Oil & Gas and Research & Development Pharmaceutical Companies
Following is a list of Research & Development Pharmaceutical Companies I contacted and researched on today:
1) Life Technologies Inc.
2) Charles River Labs
3) Boheringer Ingelheim Canada Ltd.
4) EMD Chemicals
5) Paladin Labs
6) VWR Canada
Following is a list of Oil & Gas Companies I contacted and researched on today:
1) Teck Coal Ltd.
2) Focus Corporation
3) Imperial oil resources
4) Enbridge Pipelines
5) Bietz Resources
6) Bow Valley Energy Ltd.
7) Ithaca Energy Inc.
For today, Boheringer Ingelhiem Canada was the only Research & Development Pharmaceutical Company that said that it might be able to help us. I will do a follow-up on our 2009 iGEM sponsorship proposal next week.
IMAN
Descriptive Title of What You're Doing
WIKI CODING HERE
JAMIE
Competent Cell Testing with Serial Dilutions of pBluescript
Serial dilutions of pBluescript were used to test transformation efficiency of competent cells made yesterday.  Amounts of 1000ng, 100ng and 10ng of DNA were transformed into TOP10 and XL Gold cells which were plated on Amp100 plates.  Standard transformation protocols can be found in Sambrook et. al, 1994.
JEREMY
Transforming PQ-B-R-OU-B construct into TOP10 cells
After the overnight restriction digest set up on July 27, phosphotase and ligation treatment was performed and the plasmid was subsequently transformed into TOP10 competent cells and plated overnight on LB+Chlor.
KATIE
Script Revision
The PCR machine modification took a lot longer than initially anticipated since the machine was having difficulties successfully locating all the materials it required. To fix this, I rewrote the script using a different method to check for inventory inside the object and I now check for the existence of the items that are needed within individual lists at three points rather that two to prevent any mistakes that were being made previously. One positive outcome of taking the time to make these changes is that now a list may contain things that it does not necessarily require to complete the activity meaning more options may be added when they are required. It also makes the script less bulky so I believe that when I have the time I will change the whole script to run just like the PCR activity with a general DNA template.
I am now tempted to go back to restriction digest and change the script, since my initial method appears to only work well within a less complicated script, so I believe I will return to that at a later date as well. I am selecting the substrings I will need from a single note card for comparing the parts that are going to be ligated together. So far I have determined the substrings of sequences for all promoters, rbs and terminators cut to be inserted behind or in front of another part and now have moved on to the gene that will be used for the lab missions.
I was able to get my DNA polymerase to successfully detect an RNA primer and rez nucleotides after the collision for a little while. I have now made a primase to rez the RNA primers and now I have to determine where on the strand they should be rezzed along the lagging strand. I am also considering animating the open strands of DNA so that initially, the single strands are together and then come apart, which will send a message to gyrase and helicase to initiate their own activities, which I believe will just involve them moving to their respective places as their individual functions are explained.
Tomorrow I plan to continue with the replication animation as well as the entire molecular cloning mission. I believe I will try to get the timing organized for the different parts of the replication display so that everything will happen in the correct order and I will finish of obtaining the substrings I need from note cards for construction.
KEVIN
1. Overnight cultures
Purpose: To do a miniprep on cells containing these plasmids, we have to grow an o/n culture. K082003 is being isolated to complete the reporter circuit of Pqrr4+RBS+GFP+LVA. So far, Pqrr4 + RBS have been constructed, and now I have to miniprep GFP+LVA, and construct the final reporter circuit.  Q04510 is being isolated so that Jeremy and Jamie can construct our final response circuit.
2. Restreak of K082003 (GFP + LVA tag) and Q04510 (Inverter)
Purpose: Because some of yesterday’s restreaks did not contain single colonies, I restreaked them again so that I can get single colonies to work with if the single colonies in 1 do not contain the plasmid of interest.
MANDY
Wrapping Up the Second Life Lab
I got together with Katie and we listed out all the things I needed to make available as components for Transformation and Restriction digest. Those have been completed. Unfortunately, the ‘fridges’ in the lab have become increasingly crowded as a result. To make it easier to see, I will be changing the hover text color, and am also considering making a mini ‘map’ on a texture that will show people exactly where things are kept. (EDIT: now that the font has been changed and the boxes more organized, a map is no longer necessary)
Katie also incorporated the necessary components for the lab missions in the PCR, restriction digest, and transformation activities. I’ve completed the un-edited notecards for DNA extraction and all 3 lab missions, so as a result, we are done the notecards necessary for lab and will be sending the rest of the set to you tomorrow (hopefully).  All the ‘lab mission packages’ have been assembled with instructions, necessary starting parts, and accessories (wearable pipettes  :D)
Tomorrow, Katie and I will be testing out all of our lab equipment and making necessary corrections/streamlining activities. Once this is completed, everything can be replicated to the green lab and the lab component will be basically complete. (Katie rocks)
We are also toying with the idea of using Katie’s item script to drop completed lab mission products in the robots, and allowing users to get ‘prizes’ as a result. However, we currently are lacking prizes to be given so that must be brainstormed. Ideally, we want it to have something to do with synthetic bio, our island, and even the lab missions themselves (which are largely elaborate made up applications). These must still be completed.
Wiki Ease of Navigation
Did the daily update for both sponsors and blog/update pages. Editted Fahd’s article about NUTV, and posted that along with the video of our news coverage.  Filling out of notebook began today. I still need to write the description of what the notebook is (I think emphasizing that we want to document our activities OUTSIDE the lab as well in our notebook might be a good idea, as I don’t think it’s been done before).
Kevin noted that it was difficult to navigate between notebook pages, but from the looks of the Team: Warsaw page, the coding is extremely complicated (I might ask Patrick about it tomorrow if he has time). Instead, we have included links to days through the month’s calendar in the sidebar, allowing easy access between days. With the fixed template, I sent it to Jamie, Carol, Emily, and Kevin, and they have helpfully put up the new template on all the days (including weekends) from MAY until AUGUST, making it easier for the people who post afterwards. I am trying to find an easier way to post/code tables, esp. for Jeremy because his tables are GINORMOUS, and we need an easy way to do that. :P
Also prepped the Acknowledgements page for Prima to fill in and the Protocols page for Carol.
Looking at Heidelberg, their ‘guide’ to the Wiki really only takes you through 6 critical areas of their wiki and sometimes he is hard to find, I am thinking of implementing a navigational guide (NOT in mascot form) that opens in a new window when people click that they need help. Still playing with these ideas.
PATRICK
Descriptive Title of What You're Doing
WIKI CODING HERE
PRIMA
Marketing Cold calls, Media & wiki acknowledgements
I updated the gmail contact list. Sent a Thank you letter to Corning Life Sciences.
Called Worley Parsons – the contact they gave me was apparently the wrong person. It was the HR person and she said she’ll find out who deals with sponsorship and get back to me. But I might just inquire and find out myself just in.  Company 2 (Jennifer Alexander)- called today but couldn’t get a hold of her so I left a msg  follow up tomorrow
Cayman Chemicals ..called…needs a final confirmation from their President follow up July 31
Sigma Aldrich I sent her and email and left her a voicemail  follow up tomorrow or day after
COLT Engineering – sent sponsorship pkg and I’ll call them on Thursday
Origene – I called today but they can’t sponsor any projects this year due to economic downfall
Finally sent Ms. Moser an email and the letter to be forwarded to Mr Jay Ingram.
I also wrote up the acknowledgement section of the wiki and sent it to Mandy. Tomorrow I’ll acknowledge each individual mentor/donator person.
I also called Wordans T-shirt company and told them about our project and what we want and asked for quotes and discounts. I got the email address of the manager and sent him and email about what we want on the shirt and if we ordered 30-40 shirts, how much would each cost??  He’ll get back to me soon and I can let you guys know.  I research some companies that I’ll be contacting within a few days and personalized the email to emphasize how iGEM's goal concides with the company's goal/project. Tomorrow I’ll update my notebook on the wiki.
STEFAN
Building Engineered Cells in the Synthetic Kingdom
Construction of a GFP activity has been completed in Second Life. Clicking a jellyfish will dispense GFP, which the person can then put in their inventory. Then they can open the inventory of a bacteria in the area and place the GFP inside. As a result, the bacteria will glow green for a period of time after which the script will reset until the next person places the protein inside. This activity will also serve as a tutorial for inventory management for people new to Second Life. A note card will be added later on explaining the protein so others will understand how it can be expressed in the bacteria even though it has been extracted from a jellyfish.
VICKI
A search for credibility
There were 4 important areas toward which I geared my literature search today:
1. Other applications of LuxO D47A
This is mostly for the discussion: if we’re contributing a mutant to the Registry, it would be nice if we could find a use for it in other areas. After all, there has to be a greater sexiness factor to this lowly mimic of a dephosphorylated protein.
I went on to Google Scholar and clicked the list of papers that cited Freeman and Bassler 1999 (A genetic analysis of LuxO…), and looked at where it has been used in other studies. D47A was a little harder to find, but lots of people use D47E, so I think that it’s reasonable to propose a synthetic biology project where the two are used in conjunction in different contexts
A study where LuxO D47A is used to eliminate density-dependent Lux expression in Vibrio cholerae (“Parallel quorum sensing systems converge to regulate virulence in Vibrio cholerae”), as well as the contents described in “Quorum sensing controls biofilm formation” have potential. I have saved both of the papers. With the former, we have developed biobricks for E. coli, but we can always change our chassis and biobrick a similar system in V. cholerae. Unlike the system in nature, we can tweak different modules and genes as we see fit, using the mutants to help validate our testing process. The latter only brings up D47E to mimic the high cell-density state, so it will be harder to tie that in. I’m not sure if it’ll be too far-fetched to suggest that you still need D47A to validate a reporter circuit, if only to establish a negative control.
2. Versatility of AI-2 signalling
So I was hasty last week and didn’t bother to really read the paper that I thought had an estimate of how prevalent AI-2 signalling is among bacteria (50% of bacterial species, according to the business plan). Imagine my surprise when I didn’t find that statistic when I verified my citation today. Accordingly, I sought to find a way to justify why we’re even bothering to implement AI-2 in the first place. This is mostly for the introduction
First, I found a paper that discusses how AHL is limited to gram-negative bacteria (“Interspecies communication in bacteria”). Next, “LuxS quorum sensing: more than just a numbers game” discusses how AI-2 is in both gram-positive and gram-negative bacteria. It also includes a nice list of bacteria that have functions regulated by LuxS (complete with references), so I’m going to choose 2 of them and use them as examples to support the versatility of this system and draw potential applications
Whereas AI-I production and detection is unique to V. harveyi and V. parahaemolyticus (according to “Cross-species induction of luminescence in the quorum sensing bacteria Vibrio harveyi”) (note that this is an old paper and new discoveries might have been made since then), AI-II is much more wide-spread and can be used in interspecies communication (from “Multiple signalling systems controlling expression of luminescence in Vibrio harveyi: sequence and function of genes encoding a second sensory pathway”).
Someone did this in 2003, but we could do it now…..run a BLAST search of LuxS in bacterial species whose genomes have been sequence. I’m sure that we’ve discovered more bacteria that use AI-II in nature, and could make a pretty convincing impact when we compare the length of the updated list to that of the AHL list.
MAJOR APPLICATION (why we care about versatility of the system): many biofilms embody numerous species of bacteria. If we’re using quorum sensing to destroy the biofilm, we can’t just appeal to the small gram-negative selection that uses AHL -> we need to broaden our horizons and account for the plethora of other species that may be present. AI-II seems to be the best way to do this, given the (very long and BLASTed) list of bacteria in which that signalling pathway is naturally found.
3. Engineering applications of quorum sensing
This is mostly for the discussion and the sexiness factor of our circuit on the whole. In essence, you can use your mutants to test reporter (and in turn, signalling) circuits that are useful in cool applications.
I tried to find some hard numbers on the cost of biofilm and the impact that an anti-biofilm device could have. Finding a credible source for this is a problem! I tracked down a paper that estimates that 65% of human bacterial infections involve biofilm, and another paper that brings that number up to 80%. The first paper is very old and the second one doesn’t have a formal source attached to the number (they attribute it to the NIH, but the only place on NIH where I could find it was on a request for proposals released in 2003, where no study or history was included. The only thing is that if I am focussing on biofilm killers for sex appeal, I need to bring in some credibility as to why we care. We all know that biofilms are a problem, but people outside the field might not and we need to convince them while respecting their intelligence.
I also looked at pattern formation, in a study that made pretty designs with the idea that such pattern formation could eventually be adapted to serve in 3-D tissue engineering, biosensing and biomaterial formation  all of which rely on highly-ordered structural design and fabrication. The paper is “A synthetic multicellular system for programmed pattern formation”.
My third cool example lies in the value in engineering multi-cellular systems, with emphasis on the quorum-based repressilator. When quorum sensing is coupled with the repressilator, system noisiness is reduced. Again, the mutant circuits would be effective in validating (or not) the reporters used in this system. The paper is “Modelling a synthetic multi-cellular clock: Repressilators coupled by quorum sensing”
4. Understanding of what synthetic biology is
This is partly to help give me some focus for the first paragraph of the introduction (do I want to focus on standardising biological parts, circumventing the tedium associated with assembling biological systems from raw materials, allowing for a modular approach in system design, assembly and refinement – or all of the above?). I also used it to help me write my part for our APEGGA article. I went to syntheticbiology.org and saw what was there.
Procedural modifications
Nothing fancy here – I just included details on the restriction enzymes that I used in the construction protocols. I’ll throw in a picture to make this easier for a reader to visualise. The specific enzymes are important so that anyone attempting to replicate the experiment is able to insert the appropriate scar between adjacent parts, and doesn’t mix up the order of the inserts.
Discussion writing
I didn’t make it as far as I would have liked. I have a paragraph that discusses what we accomplished, but when I attempted to write something on what the results tell us, I got stuck. I have included what I wrote at the end of the update.
APEGGA article
I wrote up my section and sent it to Carol and Chinee. We’ll put everything together tomorrow.
Main points in my discussion section
Original purpose of project: develop a BBk form of a gene that codes for a mutant protein that can be used to test our BioBricked version of the AI-2 system (originally from V. harveyi and soon to be in E. coli).
We have developed 3 biobricked versions of that gene, which can be used in conjunction with LuxOD47E for testing
It is useful to have the 3 versions in case somebody else wants to use the functional unit as a whole, or alternatively choose their own promoter/terminator sequences
What's next: precisely where this biobricked gene will be useful
More on what's next: characterisation and modelling of circuit behaviour!
Other potential applications  is there another use for a protein with this shape? Can we tie in the alanine mutation that makes most proteins defective, especially when it replaces an amino acid with a large and/or charged functional group?
Evaluation of challenges in the experiment: not every cloning attempt was successful and we had problems with negative control contamination. Is there a credible way to explain why?
Be sure to mention what this will contribute to the scientific community (this is probably tied into the reason why we want 3 biobricked versions
etc...
Retrieved from " http://2009.igem.org/Team:Calgary/28_July_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
