"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/17 July 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
JULY 17, 2009
CAROL
Ligation of promoter library and pCS26 and Transformation of plasmid in E. coli
Following the overnight enzyme digestion (BamHI and XhoI) of the vector (pCS26) and promoter (created by the 32 different primers), the vector (pCS26) was treated with anarctic phosphatase for 30 minutes (see proper protocol in procedure page) and the products was ligated with 3 different methods:
1. NEB Quick Ligase for 5 minutes 2. Invitrogen T4 Ligase at 16oC overnight 3. Invitrogen T4 Ligase at room temperature (~21oC) for one week
The Quick Ligated product was transformed into Top 10 cells and was grown on LB plates with Kan resistance for 16 hours.
CHINMOYEE
Programing of Hill curve fitting Complete
Watched the Awesome Molecular biology video. Learned quite a bit about cells in our body . Set up Matlab so that when data is present the Hill Equation will automatically fit the data to a curve .
At the end of the Day we had a fire alarm.
EMILY
DNA Sequencing & Wiki Updating
Today I sent one of the colonies from my restriction digest yesterday down for sequencing.  When these results come in, we will be able to see if the B0015 terminator has been successfully cloned into the J13002-LuxOD47E construct.  If this is successful, my circuit will be done!  From here, I can use the parts that Kevin has been constructing with GFP to put my circuit in and test the functioning of my circuit.
This afternoon I helped Mandy with editing more photos for the Wiki.  I also looked about CedarLane Laboratories, our sponsor of the month, and wrote a short paragraph about them to include on our Wiki.  I also sent an Ethics paper that Fahd sent me.
FAHD
Marketing for July 17th 2009
Today, I concentrated my energies at marketing our iGEM project and looking at the ethical aspect of our project. Here is what I did today:
1) Attended Meeting
2) Did my ethics blog
3) Did not call and e-mail any companies because it was too late in the day
4) Talked to Nexen and told her that when she meets with her manager on monday, please tell him that any kind of donation would help because we are developing educational tools and our project has potential applications such as bioremediation.
5) Read Ethics papers
IMAN
Results:
Our Framework is able to produce more results than concentration graphs. Concentration graphs are good to observe the general trends in the system and to gain some numerical data, however there are better ways to present the patterns and trends in our model. For example, we could use matrixes and arrays to show the change of concentration of given agents over the simulation period using colors.
An interesting example would be: AI-2 Binding to the LuxP-LuxQ Protein Complex.
Each column represents one of twenty E.coli bacteria. The state of the modelled bacteria over a period of 50 simulated time steps is depicted along the vertical axis. The color of each cell indicates the binding degree between AI-2 and the LuxP-LuxQ complex. The color spectrum spans from red (no binding) over white to blue (complete binding). As time progresses an increasing amount of AI-2 is produced by LuxS and gets bound to LuxP-LuxQ.
That is what I have been working on this week. As usual I had 2 meetings to go as well.
JEREMY
Plasmid Isolation from colonies of PQ-B-R-OU-B construct (see July 16) and sequencing
Purpose: to perform plasmid isolation in order to proceed with sequencing/restriction digest for sequence verification. The plasmid isolation was done using the QIAprep Spin MiniPrep Kit (QIAGEN).  Plasmid purity and concentration were measured using the NanoDrop Spectrophotometer. After isolation, colony 3 was sent to the University of Calgary DNA Sequencing Facility with BBK CP F/R and R0040-Reverse primers.
KATIE
A Little Bit of Everything
The phosphatase treatment station is now set up to listen for the queue that all ingredients have been added and then moves the phosphatase tube to the water bath, which is also set to listen for the tube as it travels to the bath. Once just outside the water bath will lift the lid and the tube will go inside. At the moment, after this has been completed it just appears at its original position on the a lab bench, but I think I will make it travel back more slowly as phosphatase treatment is explained once I have time to write up some basic instructions.
The listen scripts are complete for restriction digest and the heating block and now all I need to do is incorporate them into the existing scripts within the objects. The test tubes will then have to be moved to the water bath, heating block and freezer (for insert), which should not be a problem since the phosphatase tube is already set up so the script will not be much different.
Basic instructions for construction and bacterial transformation have been completed, which I will still have to add to the touch_start event or have them beside the activity. The bacterial transformation quiz also had to be able to be reset at any time so each question for the quiz now has a reset button for a choice and when touched, it will not only reset any variables for the quiz, but reset the progress test tube to the original white/transparent colour.
I was able to modify the notecard reader script so that it will check that the contents within two notecards are the same, which I think I will use in the sequencer as well as the construction tubes. The notecards can contain a series of bases that will determine if it is a promoter, rbs etc. And how it has been cut so I will not just be relying on checking for the proper name of the notecard, but what it contains. You can also add the information from two notecards together, but this will only be useful if I can determine a way to store the information in a notecard and give it to the individual. I do not know if it is possible to make a new notecard with a script or if there is only one empty notecard, if you put the information into it and give it to a user I do not know if the notecard in the objects inventory would remain empty, which is what I would want. I will be looking into this next week.
I also plan to prepare something for the blog video I have to do by Tuesday where I will display the progress that has been made in the lab as well as some of the scripts that are behind the functionality of the objects.
KEVIN
1. Plasmid Isolation of B0034
I have isolated the plasmids from these overnight cultures, and the purity and concentration of them were measured using nanodrop. They were pure enough to move on with.
2. Construction of Pqrr4+B0034
Since our Luciferase did not turn out great, we turned our attention to GFP:LVA. But because the part with GFP:LVA does not contain RBS, I need to construct Pqrr4+B0034 first. Pqrr4 was cut with SpeI+PstI and B0034 was cut with XbaI+PstI. Pqrr4 was then phosphatase treated, and the two were ligated together. The ligated product was finally transformed into TOP10 cells. Carol promised to take the plates out tomorrow.
MANDY
DNA Extraction in Second Life
In the activity now, users can take cells, suspend them and then add the appropriate enzymes to lyse cells and degrade proteins … Then the solution is poured into a cartridge and eluted…etc. For this activity, a couple of the test tubes used have Katie’s inventory script, which means you have to drop things in e.g. the suspended cells, the proteinase, etc. Dialogue menus (like clicking buttons to choose) are used to identify what cells are being used (haven’t fully implemented this yet), what step is being centrifuged, etc. The entire system works around the centrifuge. I still need to make sure the instructions given at every step are clear, and need to finish the notecard that should give a thorough explanation of extraction. :D
Next week, I will also coordinate with Katie so that we figure out what the combinations of lab activities can be done, and how many ‘types’ of the same activity can be done (eg. PCR of different parts, etc.) depending on the design of the remaining 2 lab missions. (I got half way through the design of the hardest one before I got sidetracked by realizing that we should have DNA extraction as an activity).
Visual Guides for the Wiki
For wiki, I made a couple icons, that will serve as a visual ‘reminder’ of what section you’re in the wiki for (icon for lab work, icon for modelling, icon for ethics, etc.) Because although the site design is cohesive (e.g. practically looks the same on every page), other than the header, there should be other things that serve as a reminder for where you are, especially if the page scrolls vertically a lot.
PATRICK
Repressors and RNA Polymerases
Retrospective Notebook: This entry was not written on this day, but derived later from working notes I made that day.
Spent all of today working out the details of the permission system for binding DNA. Proteins aren't allowed on until the DNA gives permission!
PRIMA
Media Inquiries
After the marketing meeting, I contact SemBioSys, called up the Investor Relations Inquires lead, Abby Garfunkel and told her about our project. She asked me to mail her the sponsorship pkg so she could forward it to Dr. Moloney.  I wrote an email to Dr. Moloney restating our project goal, meeting and that I spoke to Abby.  The rest of the marketing team checked over the email before I sent it out. Hopefully, he gets back to me soon or I’ll call Ms. Abby again on Tuesday.
I found a number for Jay Ingram but I couldn’t find any emails anywhere on the websites. However, I spoke to Dr. Jacob today and he said he knew Mr. Ingram and his wife and he’ll send me her email address and hopefully I can get a hold of him through her. I started writing the email that I’m going to send to Jay Ingram. Dr. Jacob wanted to know how we’re doing in marketing so I told him everything and about our marketing meeting this morning.   Oh and I sent Mandel Scientific an email asking for a pipette set and pipette tips. So next week, I have to follow-up with a WHOLE BUNCH OF companies (about 10 –practically all the ones I’ve mentioned over the past 2 days). I also updated the iGEM marketing company list on gmail spreadsheet.  Finally, I just helped Jamie with plating after he transformed Cl lambda earlier today.
VICKI
Modelling meeting
The purpose of this meeting was to flesh out a procedure for attacking characterization. We decided that the six areas we looked at yesterday and earlier today were all feasible to test, and it looks like most of the evaluation can be done with SimBiology and the optimisation toolboxes, both of which we have on our own computers. We also explored other areas that might be interesting, such as the circuit behaviour in different conditions (ie, vary the temperature or the medium composition and see what happens). It’ll definitely be a lot of work to test all of this, but if we plan our procedures well, we should be able to see results.
Facilitating the modelling methods: finding a good AI-2 supplier
I searched online for an AI-2 supplier, as well as molecules that might mimic its behaviour as well as any corporation that might supply those. I found a series of mimics (mostly variants of AI-2), each of which were patented by the Bassler lab. At that point, I decided that it would be best for us to just e-mail the Bassler lab and find out where they get their AI-2 molecules.
Retrieved from " http://2009.igem.org/Team:Calgary/17_July_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
