"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/19 August 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
AUGUST 19, 2009
CHINMOYEE
Look up the magnitudes of certain kinetic constants
Since it is very hard to find relavent kinetic constants ( these constants change with different conditions)we decided to find an order of magnitude for our kinetic constants in certain situations and the rest of the precision we hope to optimize at a certain point in the future.
We looked through many papers and fortunately we managed to find some magnitudes for the dephosphorylation  and phosphorylation rate. The smaller the kinetic rates the longer the reaction time.
We also looked into finding the precision and limitations of the fluoresecence reading instrument. The protocol was discussed and preparations were made so that the instrument could be tested tomorrow.
Issues of disscussion : Are all reactions really elementary ? How accurate is it to assume that the order of magnitude of the kinetic equations is determined by the number of species present in the reactants ?
After some research into paper : One other math modelling paper looked into modelling the interactions of insulin . The autophosphorylation of (X...), receptor dephosphorylation and degradation of insulin was simply assumed to be first order kinetics.
I did some research into the research of GFP production and found some interesting things: GFP is a very stable protein. It only denatures after 90 degress celcius and with pH of either below 4 or above 12 . Partial to near renaturation occurs with in minutes following a reversal of denaturing conditions by dialysis or neutralization.
The lower excitation peak of GFP changes with a change in conditions of the environment.
Folding rate of GFP depends upon the concentration of GFP present . The more GFP present the slower the maturation rate. Wild Type GFP maturations takes about 2 hours while GFP mutants maturation time can vary to the degree of 5 mins.
Revision of the Math Model Summary done . To be presented for tomorrows modelling meeting .
EMILY
Verification of Plasmid Switch and Overnight Cultures
Today I ran a colony PCR of J13002-LuxOD47E-B0015 in psB1AC3 with BBK- CP forward and reverse primers and p Taq.  This was to verify that the plasmid switch was successful and the construct is now in the psB1AC3 vector.  The PCR product was visualized on a 1% agarose gel run ay 120 V with the construct in the psB1AK3 vector as a size control.  See gel photo below.  Lanes 1 and 2 are the construct in psB1AC3, lane 3 is the construct in psB1AK3 and lane 4 is a negative control.  From this gel, it looks like the plasmid switch was indeed successful as the bands in the first two lanes match up with the band in the control lane and these bands are at ~ 1.8 KB, which is the expected size of the construct using BBK-CP primers. From here I made an overnight culture and restreak of the colony used in the PCR.  Tomorrow I will isolate plasmid and transform into the KTT1144 cells as well as into Kevin's reporter circuit (Pqrr4 + GFP).
Today Kevin and I also prepared various overnight cultures for testing with the plate reader tomorrow for Modeling purposes.  Cultures were: TOP10 with Pbluescript, Top10 with Kevin's reporter alone, Top10 with reporter and LuxOD47E, Top10 with R0040 + GFP (constitutive promter with lots of glow), KT1144 cells (with GFP cosmid), KT1144 cells with GFP cosmid and LuxOD47E mutant circuit.
FAHD
Ethics and Outreach for August 19th 2009
Today, I concentrated my energies on elaborating on the ethics essay further. For example, I read more literature on Self Regulation vs Government regulation and added the extra information to the blog. I will continue to work on the ethics paper over the next few days.
I also worked on improving the program plan for the ethics conference including the type of questions we will be discussing at the conference.
I also looked up a couple of awards that we can apply for as a team which I will be discussing with my team-mates on our Friday’s meeting.
JAMIE
Preparing for V. harveyi AI-2 activity assay #2
I finished up the list of bacterial strains used in the experiment (complete with references!) so that it can be used as a tool for analyzing the results from the assay.  Additionally, more AB media was made alongside the proper overnights needed (i.e. reporter strains, negative controls, AI-2 producers).
KATIE
Continuation of Transcription and Wiki Updates
The transcription activity is almost finished accept if I would like to have an avatar pick their own nucleotides. In order to do this I have made four nucleotides to choose from and when one is touched, it send a message to the RNA polymerase and it will determine if its decision matches the user’s. I made it so that an avatar can choose any nucleotides they would like along the template strand by touching them to change the type and the polymerase is able to detect this change and automatically knows the complementary nucleotide. Initially, I was having difficulties preventing the creation of more than one nucleotide in the same place, but that was solved by changing the location of the polymerase so that it did not continuously detect the same object. Now that this is working I now have to add the rest of the framework to the activity including the two different types of termination that may be involved, which I plan to start tomorrow. If I am able to complete this tomorrow, I will begin on a display for translation to be continued on next week.
Stephan found a much more graceful way of adding something to inventory, which is the way I would prefer people added inventory in areas like the virtual lab since the open window take up so much space on the screen and I am still unsure if people can delete what is in my objects if I give them enough permissions to open them. It appears that all you have to do is control click and drag an item over the object you wish to drop something into, which will be highlighted red. I am very tempted to change the instruction notecards in the lab to instruct people to do this, but I will see what everyone else would prefer first.
I also spent time doing wiki updates for the beginning of June.
KEVIN
1. Plasmid Isolation
Pqrr4+B0034+K082003 (C1, old), (C3, new), (C4, new), (C7, new) was isolated today using Qiagen's miniprep kit (qiagen) according to the manifacture's specifications. Then the purity and the concentration of the product was measured using the nanodrop utility. The result was good enough for me to go on.
2. Restriction digest
The above products of Pqrr4+B0034+K082003 (C1, old), (C3, new), and (C7, new) were then restriction digested using XbaI and PstI. The following is a picture of the gel:
All three bands look approximately the same, so the purist, being (C1,old), was picked to be sent for sequencing. Results will be in by tomorrow.
3. Overnight cultures
The overnight cultures of TOP10 cells with pBluescript, Reporter by itself, Reporter and LuxOD47E, R0040 (constitutive promoter) and GFP, KT1144 cells with GFP cosmid, and KT1144 cells with GFP cosmid and LuxOD47E were grown in order for tomorrow's test with Plate reader.
PATRICK
Getting Ready for a Test Release!
Today saw a great number of tiny usability changes and minor bugfixes, made in preparation for releasing a test version of the Biobricker UI.
Did some thorough testing of the biobricker, rezzed half a dozen different dna molecules, tested out all of the different types.
Tested transcription/translation, and dna binding proteins on promoters and operator sites.
Modified the behaviour of objects that are moved around in world. Before, floating objects (protein or DNA) could accidentally be pushed away from the user, to drift away indefinitely (hey, get back here TetR!). This was patched so that they will stop moving if they travel too far from where they began, but that interfered with the user moving objects further than that cutoff distance. Now objects can be dragged any distance, but the stopping distance is still in effect for drifting items.
Eliminated some annoying behaviour of DNA under construction, where pieces would appear at the wrong location. Now parts appear exactly where they're supposed to be in the device.
Tighened up how the RNAP controls whether it is physical and draggable by the user, or whether it is not draggable (ie, now it can no longer be dragged away from DNA that it is busy transcribing).
Figured out how to get the user interface / heads up display to always appear in the right place on the user's screen, regardless of screensize, computer platform, etc.
Unfortunately, a number of these changes affect scripts in all of the DNA and protein objects in the biobricker so far, and there's no nice way to change the contents of all those nested objects. So I'm stuck with rezzing them all in world again, manually deleting and replacing all the old scripts with the updated ones, and then returning those items to inventory, nesting them within each other all the while until the whole system is ready for deployment again.
So, I'm trying to bundle as many bugfixes and little enhancements together as I can!
PRIMA
Mini-prep, gene-specific primer PCR and RD
First, I made 1% agarose gel and ran last night's gene-specific PCR. It seemed like the negative control was contaminated with the aiiA. However, the expected size (775bp) of the bands were clear. Thus, aiiA exists!.
I also restreaked the 5 colonies which I'll mini prep today. I left them in the incubator at 37 degrees to grow overnight. Next, I minipreped the aiiA-B0015 from my overnights
Purpose: Isolate plasmids from overnights of Colonies 1-5.
I followed the correct procedures and isolated the plasmids. Then I nanodropped. The results were the following:
Results: Used Nanodrop 1000 Spectrophotometer read at 260 wavelength to determine DNA concentrations
Plasmid
260/280
260/230
Concentration [ng/μL]
C1-aiiA-B0015
2.03
3.91
57.3
C2-aiiA-B0015
2.02
3.04
67.8
C3 -aiiA-B0015
1.91
3.86
55.5
C4 -aiiA-B0015
1.99
3.65
59.0
C5 -aiiA-B0015
1.94
2.43
75.3
Next, I did a Restriction Digest Purpose: To verify size of my construct after I mini-prepped.
I digested with REact 3 buffer and Not I because we were out of Pst. I ran both uncut plasmid and the cut construct on 1% agarose gel and ran it at 100V. The following is a picture of the gel:
STEFAN
Fixin' things
After having the team on our island I realized that there's some things that I need to change permissions on. The vitamins are unmovable by anyone except by me. I changed permissions on the vitamins so now they are ok. The GFP inventory drop has also been altered but it seems that there is a strange bug where it won't let people open the inventory of the bacteria but it will let them click and drag things into it. Luckily that is good enough if I give people instruction on how to do it.
I spent most of today working on updating the wiki and this will be carried out for most of the day tomorrow as well.
VICKI
More work on the simulation front
Chinee and I sat down and attempted to incorporate a "sanity check" to our model - in essence, ensure that the time scale is reasonable and that the numbers that we predict are in line with what could be possible in a real system. We also looked at our simulation to try to predict where potential problems might occur in the signalling pathway and reporter system.
I looked up half-lives for GFP:LVA. I mixed it up with luciferase yesterday: the GFP:LVA half-life is not the 20 seconds that I had originally thought, but closer to 40 minutes. This could be a problem, depending on the saturation points of our instrumentation. I brought this up with the lab team and they will have data collected on the saturation limits of the plate reader by the end of the week.
Chinee and I also looked up rate constants for non-specific phosphatases in E. coli. This was difficult to find, especially since we had trouble relating Michaelis constants (in molar units) to reaction constants (in seconds^-1). We were able to approximate a few numbers, at least to an order of magnitude. This includes non-specific phosphatases( ~10^-3 s^-1) and transcription/translation of GFP:LVA. One area where we might run into trouble is the GFP protein folding time, which will take much longer than the transcription/translation steps. This could be another significant limitation to using GFP as a reporter circuit, at least for modelling purposes.
Chinee and I then updated our reaction sheet to better match what really happens, and incorporated that into the SimBiology model.
Tomorrow, we will meet with Afshin and Iman to ensure that our models complement each other and the included reactions are the same.
Retrieved from " http://2009.igem.org/Team:Calgary/19_August_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
