"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/28 May 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
MAY 28, 2009
CAROL
Modeling Readings Continued
Continued with reading up on things to characterize for the AI-2 system
Started familiarization with Single Site-Directed Mutagenesis protocol
Primers for mutagenesis is ordered. The following are the sequences for the mutagenesis:
LuxCDABE-M-F:CCATTAATGAATTGCCGGATAATCTGGATTTTGAAGGCC LuxCDABE-M-R:GGCCTTCAAAATCCAGATTATCCGGCAATTCATTAATGG
Both primers are PAGE purified.
Received licences for Matlab and Simbiology. Downloaded program and was able to practice through examples found in the Matlab package. Focused on G protein cascase model (example in Matlab) in Yeast and prepared a presentation to show other iGEM members how Simbiology works.
EMILY
Sequencing of LuxOD47E in psB1AC3
Objective: To veriffy the presence of the LuxOD47E gene in the psB1AC3 BBK vector
Size of plasmid Backbone:  3055 bp
Size of the gene of interest: 1362 bp
Total size:                            4417 bp
Sequencing needs 100ng of DNA/ 1 kb
4.4kb x 100ng/kb = 440ng
Concentration of C2= 92.9ng/μL
440ng/ 92.9 ng/uL = 4.8 μL DNA
Diluted 4.8 μL DNA (LuxOD47E BBK C2) with 5.2 μL ddH20 and sent  down to Univeristy of Calgary Sequencing Labs with VF2 and VF1 primers.
Results: Sequencing came back and results indicated that the gene if interest, LuxOD47E was not in the psB1AC3 vector.  We will have to go back to the gene in the pCR2.1-TOPO vector and try to get it into the psB1AC3 vector.
File:2009.05.28.D47E BBkVer RD+PCR.tif
FAHD
Ethics paper
Today I helped made our ethics paper handout that we will present the ethics workshop tomorrow. I also made follow up calls to a couple of R&D and O&G development companies.
JAMIE
Sequencing luxOU with VF1 and VF2.
luxOU plasmid DNA was sent to the University of Calgary DNA Sequencing Facility for sequencing.
JEREMY
Send LuxO D47A in psB1AC3 Colony 2 to get sequenced
Although RD and PCR are effective means of verifying the presence of a sequence of DNA, the gold standard for verification is DNA sequencing. LuxO D47A in psB1AC3 Colony 2 was sent to the University of Calgary DNA Sequencing Facility (University Core DNA Services, Calgary, Alberta, Canada). Primers used were VF2 (anneals near the BBK prefix on the BBK vector - used as a forward primer) and VF1 (anneals near the BBK suffix on the BBK vector - used as a reverse primer). The template was prepared such that there was 100ng/1kb of DNA (gene of interest + vector).
After analyzing the sequencing results, it was evident that luxO D47A was absent from the psB1AC3 vector.
KATIE
Continuing to Discover
This week, the island has finished processing, but we do not have permission to build yet so our second life team continues to attempt to obtain a competent understanding of manipulating objects within the various public sandboxes, mainly from the Knights Bishop public group sandbox.
I also experimented with the manipulation of avatars:
Skins, which are skins worn over the avatars shaped in world by the sliders, are made in any raster based image editor capable of producing .tga files, Photoshop, Paintshop Pro, GIMP being the big ones. The skins are then uploaded and worn. Using sculpted prims (sculpties) seems to be the most promising and flexible method to manipulate an avatar’s look. I also had to do a survey to obtain $150 LD.
KEVIN
Matlab and symbiology
Familiarizing with the matlab program provided by MathWorks, and its biological tool called Symbiology.
No experiments were done on this day.
MANDY
More building in Second Life
While waiting for us to get the ability to script/build on our island, we continued in Kings Bishop. We played with a 'door' script in order to make fridges, waterbaths, and other equipment with the ability to be opened, making them more realistic. Also, with our idea to make floating spheres that people could do the biobrick simulator in, I started learning the scripts for teleportation in order for people to get in to the spheres. Also built these spheres.
PRIMA
Marketing
I spoke to the team about logos. We discussed logo ideas and designs. Also, I contacted the Gauntlet (U of C campus newsletter) about publishing a story on iGEM. I spoke to one of the journalists and scheduled a meeting to interview some of my teammates.
VICKI
Gel electrophoresis of PCR and restriction digest products from yesterday
Purpose:
To analyse the PCR and restriction digest products from yesterday by running them on the gel. This will show:
Whether or not the products are free of DNA contamination
Whether or not they actually contain LuxPQ BBk (or LuxPQ TOPO) – note that sequencing is the only way to confirm this, but this will help us eliminate products that clearly do not contain the gene of interest
Materials and methods:
Preparation.
Salvaging the restriction digests, which were hastily left overnight at room temperature and with no orange dye to stop the reaction. This was addressed today, when 4uL of orange dye were introduced to the 20 uL of restriction digest tube contents to deactivate the NotI restriction enzymes. 16 uL of ddHOH were added to bring the total tube volume up to 40 uL
Introducing a negative control. Stock LuxPQ BBk (of the uncut variety) will produce a distinct band on the gel around 7500 bp (accounting for the psB1AC3 plasmid and the LuxPQ gene), with fluctuations in levels based on the degree of coiling. Accordingly, if the NotI restriction digests did not actually digest, the digest lanes will resemble this control. This tube included 2uL of DNA, 1 uL of dye and 7 uL ddHOH
PCR products to evaluate specificity. If there are significant genetic mutations in the BBk sample, the primers will either not anneal at all (resulting in no PCR product) or anneal but give a product that is of a different size (if the mutations are beyond the primer domain).
Gel set-up
A 0.7% agarose gel was made. 10 uL of the aforementioned cocktails were loaded into each well, with the well key included in the results picture. Voltage was set at 90V and left to run for 50 minutes. A GenRuler 1 kb Plus DNA Ladder was run in parallel to provide size calibrations for our results.
Results:
Retrieved from " http://2009.igem.org/Team:Calgary/28_May_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
