"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/18 August 2009
From 2009.igem.org
University of Calgary
UNIVERSITY OF CALGARY
  HOME
  TEAM
Team
Undergraduates
Facilitators
Our University
Acknowledgments
  PROJECT
Overall Project
Notebook & Parts
Lab
Modelling
Second Life
Human Practices
  EXTRA
News
Synthetic BLOGology
Gallery
Collaborations
Mr. Super D-Lux
  SPONSORS
Platinum Sponsors
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Community Partners
THIS MONTH
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
NOTEBOOK PAGE INDEX
Carol
Chinmoyee
Emily
Fahd
Iman
Jamie
Jeremy
Katie
Kevin
Mandy
Patrick
Prima
Stefan
Vicki
CALENDAR
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
AUGUST 18, 2009
CHINMOYEE
Editing the Wiki Notebook
EMILY
Restriction Digest and Q & A Session
Unfortunately my plates from yesterday of Vicki's construct in TOP10 (J13002-LuxOD47A-B0015) did not have any colonies.  So I restrated the plasmid switch from the restriction digest, digesting both the construct in psB1AK3 and the psB1AC3 vector with XbaI and PstI restriction enzymes.  I transformed the ligated product into TOP10 cells and plated on C plates.  Hopefully these plates will have colonies tomorrow and I can proceed with a Colony PCR to verify that the construct is in the vector.
Today we also had a team meeting where we had another Q & A session in preparation for aGEM and iGEM.  I answered some general questions as well as some human practices questions.  This was once again extremely helpful in identifying areas that I don't entirely understand, especially within some of the various sub-projects.
FAHD
Ethics & Outreach and Q & A
Today I continued to work on the ethics paper that I started yesterday. I also added extra information on the precautionary and the pro-actionary framework. I also made a program plan for the ethics conference that we will hold in Second Life ™. The plan includes information such as how we intend to carry out the conference, who are we going to invite, what type of question we will be asking our guest speakers, etc. .
We also had a Q & A with our team today. The questions were about our iGEM in general, synthetic biology, our project this year and specific questions concerning specific aspects of our project. The Q & A was an excellent exercise to prepare our team for the type of question that we will be asked during the aGEM and iGEM competition.
JAMIE
V. harveyi reporter strains for AI-2 Activity in Salmonella typhimurium supernatant
Further analysis of the data provided confusing and contradictory results.  A conscious decision was made to repeat the experiment using a different plate reader model (VICTOR) to collect both cell density and luminosity readings in hopes of figuring out what was really going on with the V. harveyi reporters.
A table was started which listed bacterial strains and the important phenotypes of each (Sensor 1, Sensor 2, AHL producing, AI-2 producing).  I also read up on the V. harveyi BB170 AI-2 assay and the potential limitations and shortfalls.
I also made a glycerol stock of the luxPQ-luxOU construct in pCS26 vector transformed into XL Gold cells.
KATIE
Continuation of August 17th
I was able to move the DNA replication to the base of the spiral, but it seems I have lost permission on gyrase and cannot even delete the script inside so any changes will occur at a later date. I started the basic frame consisting of an RNA polymerase to make a complementary RNA strand and a single strand of DNA this polymerase can “read”. The difference in this animation is that specific bases must be used instead of generic nucleotide in order to make sense so the polymerase must know what the complementary nucleotide is.
I really want users to be able to pick what nucleotide is to be added to the mRNA strand so I believe I will work on a way to do this. For transcription and am in the process of making the polymerase move down the strand, but I may change how it runs at the moment.
I edited the second life blog update and sent it to receive feedback as well as a creative title before posting it and the pictures for it in the evening. I also continued with wiki updates for July and June, but now I believe I will have to make Thursday the day I finish the June updates while May has been completed to the best of my ability disregarding the week spent learning some molecular biology and lab techniques.
After the majority of team members went onto second life today, I was told that the prizes are given regardless of completing a lab mission. I do not have permission to view this script at this time, but I will request it. Also, the lab depends on notecards, which it seems we will need to properly test anything since the robots require giving a folder of objects including instructions that have yet to be added.
KEVIN
cPCR of Pqrr4+B0034+K082003
Yesterday, our reporter circuit with GFP:LVA (Pqrr4+B0034+K082003) was constructed and transformed using colony 1 of Pqrr4+B0034 and already cut and freshly cut K082003. The K082003 part was already cut last time I attempted to construct this circuit, and this was used as well as a freshly cut K082003 to see if there are any differences between the two. On the plate of Pqrr4+B0034+K082003(old), there were noticeably less colonies than on the plate of Pqrr4+B0034+K082003(new). Perhaps most of the old K082003 linear DNA has degraded and was not cloned into Pqrr4+B0034, and most of the phosphotase treated vector remained as a linear piece of DNA, meaning the cells could not take up the plasmid that contains the antibiotic of interest. The following is the picture of the gel:
The band in (C1, old), (C3, new), (C4, new), and (C7, new) lanes seem to be at the expected size, at around 1170bp, including the additional bps that are due to the annealing of the Biobrick CP reverse primer. I will procede with these colonies tomorrow with Restriction digest to further verify the presence.
Measuring the fluorescence of the cells with the reporter and mutant circuits
Today, I played around with the plate reader. Blank LB broth with Kanamycin and Chloramphenicol antibiotics media was used to blank the machine. The following was the result:
A1~A6 consists of the results from TOP10 cells with the reporter and mutant circuits. A7~A8 constists of the results from KT1144 cells with mutant circuits in them. According to http://www.mnstate.edu/provost/GFPPlateReaderAssayProtocol.pdf , the settings for the plate reader should be:
Excitation: 485/20 nm
Emission: 528/20
Just by looking at the results, it seems that the cells indeed do produce GFP. However, since I am yet to know what values I should be expecting to conclude that they are producing GFP, I cannot be sure; thus I will explore some papers in order to find some data to compare with.
PATRICK
The Putting the Biobricker Together
Today was spent on integration of all the parts of the biobricker. It's like putting together a Russian Babushka doll... protein objects are stored inside DNA part objects, which go inside the user interface object, which also has several dozen icons and buttons associated with it. Except for the exceptions... some of the buttons are associated with a protein instead.
Finished loading all of Mandy's buttons into the biobricker. No more letters for buttons!
Went from the simpler test objects used to start with the biobricker to a larger set of Biobrick parts.
Built all of the initial assortment of Biobrick parts: activatable and repressable promoters, some operator sites, coding sequences for fluorescent proteins plus repressor and activator proteins, and finally a terminator. Building each part meant copying template part, encoding the right information in the name, setting its colour, and loading it with up to date scripts and sub parts.
Corrected a few bugs related to using the new parts. Some of the regular parts had longer names than the test part, which weren't correctly handled by the Biobricker to start.
Tracked down a number of typos (when a repressable promoter checks the name of a colliding protein to see if its the right one, or a coding sequence looks for the protein it's supposed to produce in its inventory, the names must match exactly). Swapped some incorrect texture IDs out for the right ones.
PRIMA
Mini-prep of B0015-aiiA construct
Purpose: Isolate plasmid from overnight cultures of each colony in LB medium
I followed the prodedures however, i assumed that the PB buffer in the kit had ethanol in it but it didn't. I also missed a few steps because i was using the mini-prep manifold procedures. All in all, when nano-dropping, my concentrations came out really low.
Results: Used Nanodrop 1000 Spectrophotometer read at 260 wavelength to determine DNA concentrations
Plasmid
260/280
260/230
Concentration [ng/μL]
C1-aiiA-B0015
-13.95
4.14
6.9
C2-aiiA-B0015
10.65
0.81
13.6
C3 -aiiA-B0015
-27.26
3.86
6.7
C4 -aiiA-B0015
2.56
0.69
28.1
C5 -aiiA-B0015
11.0
11.07
9.5
I realized my mistakes after and set up new overnight cultures from the same colonies.
In the meantime, I picked up my aiiA-specific primers and began primer dilution.
Purpose: Dilute primers obtained from the synthesis of aiiA-F and aiiA-R because the primers are in 'powder' form.
I successfully diluted both forward and reverse primers. The stock concentration for the forward primer was 72.07 pmol/microL. The stock concentration for the reverse prime was 78.91 pmol/microL. Then I ran my gene-specific PCR.
Purpose: To verify if aiiA actually exists in the plasmid by running it with aiiA-F and aiiA-R primers which anneal to the ends sequences of aiiA.
I inserted 3 microL of DNA (aiiA) in each tube and kept a negative control. I ran the PCR and held it at 4 degrees overnight. Tomorrow, I'll run it on a gel.
PCR Conditions:
1x - 94 degrees - 6mins
36x -94 degrees - 30 sec
36x -55 degrees- 45 sec
36x -72 degrees -50 sec
1x -72 degrees - 10 mins
4 degrees - hold temp.
STEFAN
Introducing the Team to Second Life
I finished up the Disease Hunting station in the Synthetic Kingdom. This was done by setting up 2 boxes which will rez the Bad Guy and Champion bacteria. This had to be set up in a way which will be controlled. This means putting the script to sleep for 3 seconds so people don't click on eat repeatedly and overload the sim with too many sensors (this may cause lag). I have also used a random rezzer so the interactions between the bacteria and also the cell will be more varied.
The whole team entered Second Life today which was very helpful. Having other people enter the Synthetic Kingdom made me realize any mistakes I may have made. For example, Jamie was confused about which bacteria to click. I then changed it to make it more clear that the bacteria all around are able to be clicked. In addition, some permissions weren't set and people couldn't move the vitamins and that was fixed. The GFP station still needs to be fixed but setting permissions on that is a little bit trickier.
VICKI
Some next steps on the modelling front
Today was dominated by meetings: first, Iman and I met with Anders to discuss where we're at on the project and what to do next. We ran our simulation and noticed that it shows that our GFP levels rise to infinity. This is because we haven't updated our model in a while and hadn't accounted for GFP degradation, but it also raises some interesting questions on the utility of our reporter circuit. If GFP:LVA doesn't degrade or takes a very long time to degrade, it may be very difficult to see any sort of response when we add AI-2 to the system. We also went over the sensitivity analysis and noticed that 2 of the parameters didn't really seem to be important in the overall rates of the system. More specifically, our system displayed a low degree of sensitivity to these parameters. What this means is that if we change the species involved in those reactions to something cheaper, or more readily available, or perform other tweaking, it shouldn't impede the overall performance of the system to any appreciable extent. The modelling team will meet again on Thursday morning to align our reactions and make sure that we really are modelling the same system.
After the modelling meeting, we had a team Q and A session. This was very useful in (a) revealing what I don't know; and (b) clearing up these uncertainties. It also serves as excellent preparation for when we really have to perform on the Q and A front outside of our own team.
I then worked more on our model, first by adding an element to account for GFP degradation and then approximating a degradation constant. For some reason, I had heard something to the effect of 20 seconds for a half-life with an LVA tag, and with that I was able to approximate a degradation constant. After all that, it occured to me to verify the half-life, and it turns out that I mixed it up with luciferase. GFP is much slower - around 40 minutes - which will be a problem if it is produced much more quickly than it degrades and if we surpass the saturation limits on our instrumentation. Tomorrow, I will approximate a degradation constant using the 40 minute value and work with the other members on the lab team to experimentally resolve the saturation limits on our machine.
Retrieved from " http://2009.igem.org/Team:Calgary/18_August_2009 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
