<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Results_Lysates skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Results/Lysates</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="apta-menu-bar"><P style="float:right; color:white; padding-right:60px;padding-top:10px;" class="apta-menu-icon"><B>MENU</B></P></DIV><DIV class="apta-wrapper"><DIV class="apta-menu-item"><UL><LI class="main-submenu-item"><A href="https://2017.igem.org/Team:EPFL">Home</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Project</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Summary">Results overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Outlook">Outlook</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Team</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Team">Our team</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Human practices</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit">Educational CF Kit- Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit/Storyline">Educational CF Kit - The Project</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lysates</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Lysates">Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates/Catalog">Catalog</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Aptamers</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts/Description">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Aptamers">Microfluidics</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers">Microfluidics - Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Beads">Beads</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers2">Beads - Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Toehold</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Toehold">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold">Results- Trigger Modularity</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold/Signal_Generation">Results - Signal Generation</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Software</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software/Demo">Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lab</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Safety">Safety</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Protocols">Protocols</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Notebook">Notebook</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts">Parts</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Medal requirements</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Contribution">Contribution</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Silver">Silver</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Gold Integrated</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Demonstrate">Proof of Concept</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Improve">Improvement</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Prizes</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Engagement">Education &amp; Public Engagement</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Software</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Applied_Design">Applied Design</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Integrated Human Practices</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Contact</LI><LI class="sec-submenu-item">epfl.igem2017@gmail.com</LI><LI class="sec-submenu-item"><I><A href="https://www.facebook.com/igemepfl/?fref=ts" class="fa fa-facebook fa-2x">facebook</A></I></LI><LI class="sec-submenu-item"><I><A href="https://twitter.com/epfl_igem?lang=en" class="fa fa-twitter fa-2x">twitter</A></I></LI></UL></DIV></DIV><DIV class="text-center top-space center-block" style="width:60%"><H1 class="top-space bottom-space"> Results </H1><P>As the project was based on working with cell-free systems, the first 
		objective was to develop an optimized, cheap, easy-to-use and secure protein 
    expression system. The solution to this was to develop functional home-made 
		lysate and to increase the efficiency of protein production in order to achieve 
		an optimal signal generation system.</P><H2 class="top-space bottom-space">What was accomplished ?</H2><P>In order to chronologically summarize our best accomplishments or discoveries 
		during the elaboration of our project before presenting our results, here is a list:</P><UL style="list-style:disc"><LI>Successful production of functional lysates from different cell strains.</LI><LI>Determination of which lysate is the best for protein production.</LI><LI>Characterization of GFP-mut3b and EGFP in cell free expression system.</LI><LI>Successfully shown the positive effect for protein expression upon adding gamS.</LI><LI>Successfully exhibited the two ways of adding gamS to the cell-free reaction: by adding purified gamS or by mixing the lysate with Top10-gamS.</LI><LI>Determination of the best quantity of Top10-gamS to add to the lysate reaction.</LI><LI>Discovery of the importance of T7 RNA polymerase and thus the discovery of a limiting factor for protein expression.</LI><LI>Successfully tested lysates with lacZ reporter and determined which one was the best.</LI><LI>Successful Î±-complementation.</LI><LI>Successful lyophilized cell free reactions.</LI></UL><H2 class="top-space bottom-space"> Lysates preparation and characterisation </H2><P>Preparing functional lysate was the first goal we accomplished. In order to 
		determine which bacterial strains yielded a lysate with the highest protein 
		production efficiency, we made 6 different lysates derived from the following 
		strains: BL21(DE3), M15, M15-T7, Dh5, Top10 and Top10-gamS. For every lysate, 
		we performed a <A href="https://2017.igem.org/Team:EPFL/Protocols#lysates" target="blank">
		Bradford assay</A> in order to determine the 
		concentration of protein present in the lysate. This gives an indication about 
		the efficiency of the lysates, however we also had to test our lysate in an 
		in-vitro transcription-translation reaction to determine if the protein present 
		was still viable. The Bradford assay results for BL21(DE3), M15 and M15-T7 
		always showed a protein concentration between 50 and 80 mg/ml. For Top10 and 
		Top10-gamS, the results were between 30 and 50 mg/ml. The lower protein 
		concentrations in the Top10 cells could be attributed to the fact that our 
		original protocol for lysate production was optimised for BL21 cell strains.
		Although these parameters seemed to work well for both BL21 and M15 cell 
		strains, it is possible that further optimisation could improve the protein 
		yield for Top10 cells. </P><DIV class="row"><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 1a</B>: Before and after sonication.</FIGCAPTION></FIGURE><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 1b</B>: After sonication and centrifugation, 
						lysate is the supernatant, the pellet contains the debris (cell wall, cells and aggregated proteins).</FIGCAPTION></FIGURE></DIV><P>Now that the protein concentrations in lysates are known, the production of
		 protein by the different lysates must be tested. GFP-muts3b (a green fluorescent 
		 protein variant) was chosen as a protein to express for lysate characterization 
		 for two reasons: first, it is a protein that can be measured quantitatively as 
		 the fluorescence measured is proportional to the quantity of GFP expressed and 
		 secondly, it is a bioprick part (<A href="http://parts.igem.org/Part:BBa_E0040"><FONT color="blue">Part:BBa_E0040</FONT></A>) that could be characterized in 
		 cell free for the first time. </P><P>GFP-mut3b was resuspended from the iGEM plate and amplified by PCR with primers 
		that add the T7 promoter, which is a highly efficient promoter, especially in 
		cell-free. However, in order to express DNA with a T7-promoter, only lysates 
		made from cells that produce T7 RNA polymerase (here BL21(DE3) and M15-T7) can 
		be used.</P><P>In this experiment, cell-free GFP-mut3b synthesis was analysed using a 
		home-made E. Coli lysate from strain BL21(DE3). Fluorescence was measured at 
		37Â°C for five hours on a platereader. Here (figures 2a &amp; 2b), we have shown 
		three repeats with a negative control, as well as a shaded error graph with 
		the negative control subtracted, summarizing this result. GFP-mut3b expression 
		yields high fluorescent signal in lysate, enabling it to serve as a reporter 
		while working on a cell-free chassis. Saturation of protein expression occurs 
		after about five hours.</P><DIV class="row align-bottom"><FIGURE class="col-md-6 align-bottom"><FIGCAPTION><B>Figure 2a</B>: GFP-mut3b expression in BL21(DE3) lysate.</FIGCAPTION></FIGURE><FIGURE class="col-md-6 align-bottom"><FIGCAPTION><B>Figure 2b</B>: GFP-mut3b expression in BL21(DE3) lysate with standard deviation and blank subtracted.</FIGCAPTION></FIGURE></DIV><P>GFP-mut3b expression was tested in M15-T7 lysate in order to determine if 
		there was a difference of expression between the BL21(DE3) and M15-T7 lysates 
		(figure 3). The lysate reaction contains M15-T7 lysate, energy solution, buffer A, 
		nuclease free water, DNA coding for GFP-mut3b and purified gamS or Top10-gamS lysate. 
		GamS is a protein that protects the DNA from exonuclease present in the lysates. 
		It can be added in two manners: either as purified gamS protein or as a mixture 
		of top10-gamS lysate with M15-T7 lysate (see next chapter). The control is without 
		coding DNA. The fluorescence was measured for 6 hours at 37Â°C. </P><FIGURE><FIGCAPTION><B>Figure 3</B>: Comparison of EGFP expression in M15-T7 lysate 
				depending on the addition of gamS.</FIGCAPTION></FIGURE><P>Since the units of the two last graphs are different (they were measured on 
		two different platereaders), we cannot make a quantitative comparison. However, 
		a qualitative comparison shows that the expression of GFP-mut3b in M15-T7 reaches 
		its maximum after only 1 hour of incubation while the maximum in BL21(DE3) is 
		reached after 5 hours. Furthermore, the fluorescence of the +DNA measurements 
		is approximatively 5X the âDNA measurements for the M15-T7 lysate and respectively 
		3X for BL21(DE3). Hence, with these two observations, we drew the conclusion that 
		M15-T7 lysate is to be preferred for GFP expression. This was good news for 
		our project as we planned to incorporate the expression of Î²-galactosidase (LacZ), 
		an enzyme which is not present in M15-T7 cell strains, in our diagnostic tool. 
		A second positive result is that we were able to achieve similar expression 
		levels by adding purified gamS or Top10-gamS lysate, meaning that we could bypass 
		the usage of purified gamS proteins, which is a more timely and costly process.</P><P>EGFP (enhanced green fluorescent protein) was chosen as an alternative protein 
		to be expressed for lysate characterization because the progression over time 
		of protein expression is observable, it is a bioprick (<A href="http://parts.igem.org/Part:BBa_K1123017"><FONT color="blue">BBa_K1123017</FONT></A>) part and 
		it gives a higher fluorescent signal than normal GFP. The lysate reaction contains 
		BL21(DE3) lysate, energy solution, buffer A, nuclease free water, coding DNA for 
		EGFP and gamS or Top10-gamS as described in the previous experiment. The control 
		is without coding DNA. The fluorescence was measured for approximatively 5 hours 
		at 37Â°C. </P><FIGURE><FIGCAPTION><B>Figure 4</B>: Comparison of EGFP expression depending on the addition of 
					Top10-gamS lysate. </FIGCAPTION></FIGURE><P>In this experiment (figure 4), three things are important to notice. First,
		 as was seen in the previous experiment with GFP-mut3b, the protein expression 
		 in BL21 (DE3) is slower than in M15-T7 lysate, which confirms the previous 
		 conclusion. Secondly the difference between the +DNA (pink) and âDNA (royal blue) 
		 (when gamS is added) is clearly bigger than the two preceding experiments 
		 (approximatively 3X), probably due to higher fluorescence of EGFP. Finally,
		  we see a very interesting effect due to the presence of gamS: 10x higher 
			expression of protein expression can be observed between the +gamS (pink) 
			and the âgamS (light blue) signal. This experiment confirms the positive 
			effect of gamS that will be further shown in future experiments when using 
			linear template DNA.</P><H2 class="top-space bottom-space"> Optimisation of lysates: GamS</H2><P>The gamS protein from phage lambda increases protein expression when the DNA 
	template is added in linear form because it protects linear DNA from degradation 
	by exonucleases. As gamS protein could have a great potential for lysate optimisation, 
	we decided to characterise this part. Most importantly we needed to prove the importance 
	of adding gamS to our cell-free reactions and define the best quantity of gamS to add.</P><P>In proving that the addition of gamS helps the reaction to take place, we tested 
	two possible scenarios: adding either purified gamS or Top10-gamS lysate. Top10-gamS 
	lysate was derived from the bacterial strain Top10 which we had transformed with a
  plasmid containing the gene encoding the gamS protein, enabling the expression of 
	gamS in these cells. In fact, two of our published biobricks are the gamS gene (<A href="http://parts.igem.org/Part:BBa_K2203001"><FONT color="blue">BBa_K2203001</FONT></A> and <A href="http://parts.igem.org/Part:BBa_K2203000"><FONT color="blue">BBa_K2203000</FONT></A>), therefore any iGEM team can add this to their lysate 
	cell strains for improved expression when using linear DNA templates.</P><P>In order to determine the improvement of protein expression by adding gamS 
		and whether mixing two lysates would also improve the reaction, three separate 
		reactions were tested (figure 5). Each of the three solutions contains M15-T7 
		lysate, energy solution, buffer A, nuclease free water and DNA coding for T7-EGFP. 
		In addition to the aforementioned components, the first reaction additionally 
		contains gamS purified protein (pink bar), the second reaction Top10-gamS mixed 
		with M15-T7 (1.25Â Âµl of each lysates) (purple bar) and the last reaction does not 
		contain gamS in any form (blue bar). The respective controls were subtracted. 
		The measures were taken by platereader measuring fluorescence at 37Â°C for 
		approximatively 5 hours.</P><FIGURE><FIGCAPTION><B>Figure 5</B>: Protein expression in M15-T7 lysate.</FIGCAPTION></FIGURE><P>This experiment (figure 5, and the last two experiments figures 4 &amp; 3) clearly 
		showed that adding gamS in any form to the lysate reaction is incontestably 
		increasing the protein production. The best solution is to add purified gamS 
		but adding the gamS by mixing Top10-gamS with M15-T7 lysate is still improving 
		the reaction. This last solution is cheaper and easier as the protein does not 
		need to be purified. Thus, we are totally independent as we know how to produce 
		Top10-gamS lysate. Hence, the majority of the following experiments were performed 
		by adding Top10-gamS.</P><P>Now that we have shown that the addition of gamS clearly increases protein 
		expression compared to lysate reactions without gamS, we wanted to see if this 
		advantage was consistent in other types of lysates and which ratios of added 
		Top10-gamS were ideal. Here, three lysate reactions were performed (figures 6a &amp; 6b): 
		the first containing BL21(DE3) lysate and purified gamS (pink), 
		the second mixing BL21(DE3) and Top10-gamS lysates (dark blue) 
		and the last mixing BL21(DE3) and Top10 lysates as well as purified 
		gamS (light blue) in order to determine the proportion of protein 
		expression done by Top10 lysate (as Top10-gamS is a home-made recombinant 
		bacteria made from the Top10 strain). The controls were made by doing the 
		same lysate reaction without adding DNA for T7-EGFP. </P><DIV class="row align-bottom"><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 6a</B>: EGFP expression in BL21 lysate upon addition of gamS purified or in Top10-gamS.</FIGCAPTION></FIGURE><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 6b</B>: EGFP expression in BL21 lysate upon addition of gamS purified or in Top10-gamS with standard deviation.</FIGCAPTION></FIGURE></DIV><P>The first observation (figure 6a &amp; 6b) that can be made is that adding either 
		purified gamS or Top10-gamS gives a similar result (respectively pink &amp; dark blue). 
		The light blue measures indicate that Top10 lysate does not significantly help 
		the protein expression but gamS does when adding Top10-gamS lysate. This is probably 
		because that Top10 cells do not produce T7 RNA polymerase and thus cannot participate 
		as much as BL21 in protein expression, which leads us to think that the polymerase and 
		thus the mRNA translated is a limiting factor for the cell-free reaction. Hence, one 
		solution to explore in order to further optimise the lysate reaction is to transform 
		M15-T7 or BL21(DE3) with a plasmid coding for gamS protein in front of an E. Coli 
		promoter in order to have both T7 RNA polymerase and gamS produced simultaneously.</P><P>As we decided to mostly add gamS by mixing lysates with Top10-gamS, the next 
		step for gamS characterisation was to determine which quantity of Top10-gamS 
    is ideal. Hence, EGFP expression was measured according to different ratios of 
		M15-T7 and Top10-gamS lysates with all reactions containing the same total amount 
		of lysate (figure 7). The ratios tested ranged from 1:1 to 1:4. The controls are 
		the lysate reaction without adding DNA. The average of three repeats for all four 
		ratios is shown here (errors bars are not shown for readability).</P><FIGURE><FIGCAPTION><B>Figure 7</B>: EGFP expression upon different ratios of M15-T7 and Top10-gamS lysate.</FIGCAPTION></FIGURE><P>What can be concluded here (figure 7) is that the two best ratios are 1:1 and 1:2. 
		The two other ratios give a lower protein expression, maybe because of the lack of 
		T7 RNA polymerase.</P><H2 class="top-space bottom-space"> LacZ expression </H2><P>As the best functional lysate was determined, we next wanted to choose an ideal 
		reporter for our system. The goal of aptasense was to be able to detect both
		 DNA/RNA and protein in a simple manner. Hence lacZ was chosen as reporter 
		 for our diagnostic tool because it is qualitative and the signal is visible 
		 by eye because when Î²-galactosidase is expressed, it catalyzes the reaction 
		 of a substrate molecule, causing the colour of the lysate reaction to turn 
		 purple. Before choosing this reporter for our diagnostic tool, we needed to 
		 determine with which lysates this reporter could be used, as bacteria usually 
		 also produce Î²-galactosidase. Thus, we must use a lysate that does not already 
		 contain this enzyme. For this, bacteria must have a mutation in the region coding 
		 for the enzyme in order to avoid the production of Î²-galactosidase. 
		 If Î²-galactosidase is not present in the solution, it will be possible to see 
		 the progression of signal with time by measuring the absorbance at 595 nm on a 
		 platereader.</P><P>To determine if Î²-galactosidase is present in a given lysate and to check 
		whether the lysate is contaminated with the enzyme, an experiment was performed 
		where substrate (chlorophenol red) alone or with magnesium chloride was added to 
		the different lysates. We did this for BL21(DE3), M15 and M15-T7 lysates (figure 8a) 
		and saw that the reaction turns purple, Î²-galactosidase was inherently present. </P><DIV class="row align-bottom"><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 8a</B>: Plan of experiment.</FIGCAPTION></FIGURE><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 8b</B>: Test the presence of lacZ or Î²-galactosidase in M15-T7 lysate.</FIGCAPTION></FIGURE></DIV><P>As expected, we discovered that Î²-galactosidase is present only in the first 
	tube containing BL21(DE3) lysate. Otherwise, others lysates (shown here: M15 &amp; M15-T7,
	otherwise: Dh5, Top10 &amp; Top10-gamS) do not contain the enzyme and, thus, can be used 
	for Î²-galactosidase expression. </P><P>Having chosen Ã-galactosidase as the main reporter protein, the choice for 
		the lysate that we will use was made based on the best expression of Î²-galactosidase.
	  Hence, lacZ DNA with an E. Coli promoter was added to the different lysates 
		with energy solution, buffer A, chlorophenol red and nuclease free water 
		(figures 9a &amp; 9b).</P><DIV class="row align-bottom"><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 9a</B>: LacZ expression in BL21(DE3), BL21(DE3) YTPG and Top10-gamS lysates. </FIGCAPTION></FIGURE><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 9b</B>: LacZ expression in M15, M15-T7, M15-T7 YTPG, Dh5Î± and Top10 lysates. </FIGCAPTION></FIGURE></DIV><P>The measurement showed that BL21(DE3) already contains Î²-galactosidase, which 
		was expected (figure 9a). However, the BL21(DE3) lysate derived from cells grown 
		on YTPG medium started with a lower expression level than the normal BL21(DE3) 
		lysate. This is most probably due to the different pathways of metabolism and 
		the inhibition of Ã-galactosidase production in YTPG growth medium. In YTPG, 
		the enzyme Ã-galactosidase is not used for the general metabolism as the cellsâ 
		growth in YTPG is based on glucose.</P><P>The expression levels of Top10-GamS lysate are very unexpected (figure 9a), 
		the expression should be smaller. It is most likely due to overspill or pipetting 
		errors during the preparation of the experiment. Both M15-T7 lysates (pink and blue) 
		reach the best level of Ã-galactosidase expression that produces a colour change 
		visible by naked eye (figure 9b). Thus, the M15-T7 lysate will be our lysate of 
		choice since the expression of Ã-galactosidase is ideal, and the lysate does not 
		contain Ã-galactosidase, but does contain T7 RNA polymerase, enabling expression 
		under a T7 promoter in future experiments.</P><P>Further experiments were done in order to be sure that lacZ can be used as 
		reporter for our toehold (figures 10). Three reactions were made: T7-lacZ DNA 
		is added to BL21(DE3) lysate (figure 10a), to BL21(DE3) YTPG (figure 10b) and 
		to M15-T7 (figure 10c). </P><DIV class="row align-bottom"><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 10a</B>: T7-LacZ expression in BL21(DE3) lysates. </FIGCAPTION></FIGURE><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 10b</B>: T7-LacZ expression in BL21(DE3) YTPG lysates. </FIGCAPTION></FIGURE></DIV><FIGURE><FIGCAPTION><B>Figure 10c</B>: T7-LacZ expression in M15-T7 lysates. </FIGCAPTION></FIGURE><P>The measured absorbance does not change over time in BL21(DE3) lysate and 
		no difference in observed in control. The results from the last experiment 
		in BL21(DE3) YTPG is confirmed here: the absorbance is growing over time and 
		there is a difference between the + and â DNA control. For the experiment in 
		M15-T7, a clear difference of absorbance between the + and â DNA can be seen 
		and this reporter can be used for further experiments.</P><H2 class="top-space bottom-space"> Î±-Complementation </H2><P>In order to optimize our reactions, we thought to characterize the expression 
		of lacZ in cell free systems, something that has not been shown previously. 
		We wanted to know if the expression of only lacZÎ±, instead of lacZ, would 
		lead to the formation of a functional Ã-galactosidase since the entire enzyme 
		is quite long to produce while the Î± part is significantly smaller. 
		B-galactosidase is a tetramer of four subunits (two Î± and two Î© parts) and 
		lysates normally used for blue-white screening do not lack the entire 
		Ã-galactosidase but rather produce the Î© subunit and are mutated only for 
		the lacZÎ± part. </P><P>Hence, in order to see if the Î± complementation (assembly of Î± and Î© parts 
		in order to produce a functional Ã-galactosidase) is possible. We made an 
		experiment that consisted of adding lacZÎ± DNA in front of a T7 promoter in a 
		lysate reaction to see if the solution turned purple. In this experiment 
		(figures 11), we used T7-LacZ as a positive control and toehold-lacZ as the 
		negative control. </P><DIV class="row align-bottom"><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 11a</B>: Colorimetry assay after 1h incubation. </FIGCAPTION></FIGURE><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 11b</B>: Colorimetry assay after 2h incubation. </FIGCAPTION></FIGURE></DIV><FIGURE class="bottom-space"><FIGCAPTION><B>Figure 11c</B>: Colorimetry assay after overnight incubation. </FIGCAPTION></FIGURE><P>B-galactosidase complementation was successful: the colour change was witnessed 
		after two hours of incubation at 37Â°C. Here (figures 11), the T7-lacZÎ± is less 
		efficient than the entire T7-lacZ but later, some additional experiments were done,
		 in fact the colour change can be observed more rapidly, especially when using the
		  toehold (figure 12).</P><FIGURE><FIGCAPTION><B>Figure 12</B>: Comparison of time needed for colour change between lacZ or lacZÎ± expression. </FIGCAPTION></FIGURE><P>In order to improve the characterization of the lacZÎ± fragment in cell-free 
		reactions and make further experiments in order to be sure that the Î± complementation 
		works, lacZÎ± is added in T7-M15 cell lysate. The figure below shows the expression 
		of a functional Ã-galactosidase in T7-M15 cells upon the assembly of the different 
		Î± and Î© parts and confirms the possibility to further use the T7-lacZÎ± for the next 
		experiments.</P><FIGURE><FIGCAPTION><B>Figure 13</B>: T7-lacZÎ± expression in M15-T7 lysate. </FIGCAPTION></FIGURE><H2 class="top-space bottom-space">Lyophilisation</H2><P>Lyophilisation of cell lysate permits long and easy storage although it can
		 reduce the efficiency of protein production. Hence, the goal of the first 
		 experiment was to determine the ultimate loss of protein synthesis due to 
		 lyophilisation (figures 13).</P><DIV class="row align-bottom"><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 13a</B>: EGFP expression in M15-T7 lyophilised lysate.</FIGCAPTION></FIGURE><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 13b</B>: Comparaison between EGFP expression in M15-T7 lyophilised or fresh lysates.</FIGCAPTION></FIGURE></DIV><P>The loss was determined measuring the levels of fluorescence of EGFP protein 
		(enhanced green fluorescent protein) on platereader. The loss of EGFP protein 
		expression by adding T7-EGFP DNA in rehydrated freeze-dried reactions is 60% 
		compared to the non freeze-dried reactions (figure 13b), however the expression 
		is still high enough for detection (figure 13a).</P><P>The next step was to test if the production of Î²-galactosidase in a lyophilised 
		reaction is detectable since this enzyme was the main reporter used. E. Coli lacZ 
		DNA producing Î²-galactosidase was added to M15-T7 lyophilised lysate (figures 14).</P><DIV class="row align-bottom"><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 14a</B>: Measurements on plate reader of Î²-galactosidase expression in M15-T7 lyophilised lysate.</FIGCAPTION></FIGURE><FIGURE class="col-md-6"><FIGCAPTION><B>Figure 14b</B>: Colorimetric assay of Î²-galactosidase expression in M15-T7 lyophilised lysate.</FIGCAPTION></FIGURE></DIV><P>The production of protein was detectable since the substrate for Î²-galactosidase 
		changed colour both by eye and showed an absorbance of 0.5 at OD 595 nm after 5 hours. 
		This result is enough for detection, even if the lost of protein expression in 
		rehydrated freeze-dried reactions is about 50% compared to the fresh lysate 
		reactions (OD about 1).</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>