Team:EPFL/Description/Toehold
MENU
Home
Project
Overview
Results overview
Outlook
Team
Our team
Attributions
Collaborations
Human practices
Overview
Educational CF Kit- Description
Educational CF Kit - The Project
Lysates
Description
Results
Catalog
Aptamers
Description
Microfluidics
Microfluidics - Results
Beads
Beads - Results
Toehold
Description
Results- Trigger Modularity
Results - Signal Generation
Software
Description
Results
Lab
Safety
Protocols
Notebook
Parts
Medal requirements
Attributions
Collaborations
Contribution
Silver
Gold Integrated
Proof of Concept
Improvement
Prizes
Education & Public Engagement
Software
Applied Design
Integrated Human Practices
Contact
epfl.igem2017@gmail.com
facebook
twitter
Toehold
After improving the yield of our cell free protein synthesis system, we   started working on our detection device. We chose to couple toehold switches   with aptamers because toehold switches can relay the signal while aptamers   bind specifically to the target protein. In the following we relay the results   obtained with the toehold switches.
What is a toehold switch and why is it    useful in our detection scheme?
A toehold switch is an RNA fragment with a secondary hairpin structure    that is used to regulate translation. The toehold sequence contains a strong    ribosome binding site (RBS) and a start codon that is followed by the coding    sequence of a reporter gene. In the absence of a complementary sequence,    the toehold is in its native hairpin conformation acting as a translational    repressor and preventing ribosomes from binding to the RBS and thus from    translating the reporter gene downstream.    In the presence of a single stranded trigger complementary to the stem of the    hairpin, the switch unfolds and exposes the ribosome binding site and the start    codon and translation can be initiated. We worked with a Zika virus toehold switch that detects the presence of a specific     Zika sequence within a sample by translating the downstream gene LacZ. This allows a rapid colorimetric output to signal the detection of the target virus. This work is based on a paper written by Keith Pardee et al 1 , where he uses toehold switches to detect Zika viral RNA circulating in infected individuals. In order for our concept to work, we had to show that the two branches of detection work separately (eg. the triggering of toehold switches as well as the detection of the target protein by aptamers). For the beta-galactosidase    assay, we use chlorophenol red-β-d-galactopyranoside, a yellow substrate    that turns purple in the presence of the beta-galactosidase enzyme.
Figure 1: Schematic overview of the triggering of a toehold switch
Improving the signal of our detection device
The toehold we use to initiate translation upon annealing of a trigger is    a large DNA sequence: about 3000 base-pair long, as it is paired to the LacZ gene.    We thought that this may be an obstacle to a fast detection device, as the time it    takes to transcribe a gene is proportional to its length.
Alpha Complementation
To resolve this problem, we opted for the beta-galactosidase alpha    complementation. This complementation reaction makes use of a mutation    introduced in certain E.Coli cell strains. The lacZ delta M15 mutation    encodes a form of beta-galactosidase lacking residues 11-41.   Beta-galactosidase produced without those residues is missing a small part   and is thus not functional. But if this mutated form of beta-galactosidase   is brought together with the missing lacZ alpha part, the two will connect    and form a functional beta-galactosidase part. 2
In fact, beta-galactosidase is a tetramer, which means it is made up of    four identical single units. Each unit needs one LacZalpha part to work.
Figure 2: 3D structure of beta-galactosidase
In our lysate reaction, the toehold lacZ alpha can be triggered by a    complementary nucleic acid sequence, thus the lacZ alpha gene downstream    is translated. As the rest of the protein (the mutated form) is already    present in the T7 M15 cell lysate, the two will form a functional beta-galactosidase    whose activity can be detected by the addition of the substrate chlorophenol    red-β-d-galactopyranoside.
To get started, we first tried to prove that we can obtain a functional    beta-galactosidase starting from a T7 lacZ alpha DNA template added to the    T7 M15 lysate reaction.
After proving that alpha complementation worked in our cell-free platform,    we went on to test this concept for lacZ alpha in a toehold as well.    The complementation worked fine also with the toehold.
Testing different triggers
A toehold switch will stay in its hairpin conformation until a trigger    binds to it. This trigger needs to have a sequence that is complementary    to the toehold switch. A functional trigger needs to contain at least a    36 base pairs long sequence complementary to the toehold switch. It can    however also be longer. For example, the Zika virus spreads by letting    its whole genome circulate in the blood. Somewhere inside that genome is    located the trigger sequence to which the toehold switch will respond.    The trigger (in this case the viral genome) is thus a lot longer than    just the sequence that will bind to the toehold switch.
Throughout we tested several different trigger 'forms'. We started with    reproducing the results from the Pardee paper 1 ,    which meant making RNA trigger. The RNA trigger is obtained by performing an in vitro transcription reaction with a double-stranded trigger template    (for a detailed protocol, click here ).    First we tested the toehold/trigger reaction in a commercial in vitro protein    synthesis kit ( PURExpress )
Figure 3: Trigger sequence for Zika 27B toehold, 391 base pairs
Blue: The 36bp sequence complementary to the toehold switch, Turquoise: A longer stretch of the Zika genome containing the 36bp trigger sequence
The RNA trigger has the length of the green sequence in Figure 3 (as it    is transcribed only between T7 promoter and terminator) and worked well in our setup.     However, since our end goal was to couple the toehold switches to the aptamers     by letting one of the aptamers trigger the switch, and aptamers are single-stranded     DNA (ssDNA) molecules, we had to test if a ssDNA trigger would also work with the     toehold switch.
After ordering ssDNA trigger with a size of 36 base pairs (thus only including    the sequence complementary to the toehold switch and no additional bases,    Blue part on Figure A), we tested it in a toehold reaction in lysate    and were pleased to see that it worked well. This construct we called the    'ssDNA trigger short'.
Last we wanted to see if the toehold still    gets triggered even if the trigger sequence has additional base pairs at    the ends of the 36bp core sequence. We ordered the whole sequence from    Figure A as a ssDNA molecule and tested it in our lysates. This construct    was called 'ssDNA trigger long'.
Aptamer trigger
The most important trigger for us to test was of course the aptamer trigger.    The aptamer trigger is a ssDNA molecule that contains the aptamer sequence at    one end, followed by the trigger sequence for our toehold. Another way to put it :    It's the aptamer with our trigger sequence as an 'extension'. The aptamer trigger    looks like this :
Figure 4: Aptamer trigger
Dark green : toehold short trigger sequence, Light green : aptamer sequence
Signal Amplification
Another endeavour of ours was to amplify the signal : An amplification    step would allow us lower the limit of detection and thus have signal at    lower concentrations. This would render our detection tool more sensitive    and represent the real concentrations inside an infected individual better.    To this end we created the following construct :
Figure 5: T7 aptamer trigger construct
Green: the aptamer sequence, Blue: The trigger sequence, White:    T7 promoter
The goal is to let the construct be amplified by an in vitro transcription    step before adding it to a lysate reaction and thus augmenting the effective    concentration of aptamer trigger in the solution. The aptamer sequence is encoded    on the bottom strand, so the bottom strand would serve as an aptamer to    bind the target protein. Before adding the aptamer trigger to lysate reaction,    it will undergo a linear amplifcation step. This is possible because the sequence    contains a T7 promoter sequence. In presence of a T7 RNA polymerase, transcription    will proceed starting at the T7 promoter. The polymerase however needs a double-stranded    promoter region to bind to the strand. For this reason we add a primer annealing to    just the promoter region (called 'forwardb' in the figure). Once the primer anneals    to the promoter region, making it double-stranded, the T7 RNA polymerase can begin    transcription. After incubating the aptamer trigger with primer and polymerase,    the RNA that was produced can be added to a lysate reaction.
At the same time, the aptamer trigger can be amplified by performing a     PCR where we add the two primers 'forwardb' and 'T7-aptamer-trigger reverse'.     This step will lead to an exponential increase in aptamer trigger concentration.
Changing Buffer Conditions
Continuously looking to improve our output, we thought changing the buffer    conditions in our cell lysate reactions may be one way to do so.
When searching for enzymatic assays conditions in the scientific    literature 3 , we saw that a certain buffer,    Buffer Z (link to protocol) is commonly used for beta-galactosidase assays. Buffer Z    contains 2-Mercaptoethanol, a strong reducing agent that inhibits the oxidation    of free sulfhydryl residues and thus maintains beta-galactosidase’s activity.
FDG : A fluorescent substrate for beta-galactosidase
We were interested to see if our reporter gene could also turn over other substrates.    After researching the topic, we decided to give FDG (Fluorescein Di-β-D-Galactopyranoside)    a shot. FDG is a substrate that will emit fluorescence once it has been    cleaved by beta-galactosidase. By measuring fluorescence, we should be able    to see whether beta-galactosidase is produced or not.
FDG was an     interesting subject to us as it would allow detection with a sensitive     microscope which could pick up weaker signals than the usual platereader.     Thus FDG could be used for the experiments on chip and there picked up by a     fluorescent microscope.
References
1. Pardee, Keith, et al. "Rapid, low-cost detection of Zika virus using programmable biomolecular components." Cell 165.5 (2016): 1255-1266.
2. Broome, Ann-Marie, et al. "Expanding the utility of β-galactosidase complementation: piece by piece." Molecular pharmaceutics 7.1 (2010): 60
3. Stephenson, Frank H. Calculations for molecular biology and biotechnology. Academic press, 2016: Chapter 11
