<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_HP skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/HP</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="apta-menu-bar"><P style="float:right; color:white; padding-right:60px;padding-top:10px;" class="apta-menu-icon"><B>MENU</B></P></DIV><DIV class="apta-wrapper"><DIV class="apta-menu-item"><UL><LI class="main-submenu-item"><A href="https://2017.igem.org/Team:EPFL">Home</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Project</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Summary">Results overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Outlook">Outlook</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Team</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Team">Our team</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Human practices</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit">Educational CF Kit- Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit/Storyline">Educational CF Kit - The Project</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lysates</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Lysates">Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates/Catalog">Catalog</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Aptamers</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts/Description">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Aptamers">Microfluidics</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers">Microfluidics - Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Beads">Beads</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers2">Beads - Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Toehold</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Toehold">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold">Results- Trigger Modularity</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold/Signal_Generation">Results - Signal Generation</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Software</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software/Demo">Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lab</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Safety">Safety</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Protocols">Protocols</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Notebook">Notebook</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts">Parts</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Medal requirements</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Contribution">Contribution</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Silver">Silver</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Gold Integrated</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Demonstrate">Proof of Concept</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Improve">Improvement</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Prizes</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Engagement">Education &amp; Public Engagement</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Software</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Applied_Design">Applied Design</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Integrated Human Practices</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Contact</LI><LI class="sec-submenu-item">epfl.igem2017@gmail.com</LI><LI class="sec-submenu-item"><I><A href="https://www.facebook.com/igemepfl/?fref=ts" class="fa fa-facebook fa-2x">facebook</A></I></LI><LI class="sec-submenu-item"><I><A href="https://twitter.com/epfl_igem?lang=en" class="fa fa-twitter fa-2x">twitter</A></I></LI></UL></DIV></DIV><DIV class="text-center top-space center-block" style="width:60%"><H1 class="top-space bottom-space">Human Practices</H1><P> To ensure that our project would stay on track to tackle real-life problems, 
		we continuously sought input from people working ‘in the field’. This included 
		professionals with backgrounds such as synthetic biology, computer sciences, 
		education, academia and industry. By presenting our project at several events 
		and always engaging in discussions, we received a lot of opinions and feedback 
		and worked to evolve our project according to this interdisciplinary spirit.</P><H2 class="top-space bottom-space">Rapid Prototyping Workshop, EPFL</H2><P> A first step in this direction was taken when one team member went to the 
		Rapid Prototyping Workshop : “Teaching how to develop creative and practical 
		solutions to real-life, complex, scientific and engineering problems” on June 
		8th 2017 to represent EPFL iGEM and gain valuable insights on problem solving, 
		how to approach problems in general and as well as feedback on organizing the 
		Wiki space and our web page. The workshop consisted of small groups having to 
		solve individual problems by making use of a given ‘toolbox’. The problems 
		included making prototypes for a project webpage, improvising an informative 
		video describing a project, organizing the lab/office space optimally, how to 
		improve the team performance and identifying the composition of an ideal curriculum 
		for future (bio-)engineers. We later on implemented a lot of the feedback 
		(for instance in teamwork) and believe it helped us for getting more efficient 
		at organizing ourselves and meeting deadlines.</P><H2 class="top-space bottom-space">Hosting a High School Class in the Lab</H2><DIV class="row bottom-space"><FIGURE class="col-md-5"><FIGCAPTION>The high school students in our lab</FIGCAPTION></FIGURE><P class="col-md-7"> During our summer exam period we were approached by EPFL’s study promotion 
			responsible Maya Frühauf, asking us if we could set up a wet-lab workshop 
			for and supervise a high school class visiting EPFL in the beginning of July. 
			Of course we were more than happy to introduce those high school students to 
			the concepts of synthetic and cell-free biology, and hopefully pass on our 
			motivation for the exciting field of bioengineering. The more than 25 high 
			school students, in 9th grade and from the gymnasium <A href="https://www.lerbermatt.ch/home/" target="_blank">Lerbermatt</A>, 
			Bern, would be spending half a day with us, which gave us plenty of time to 
			let them do fun experiments with our guidance. After discussing the matter 
			within the team, we made a list of several experiments and prepared everything 
			for their arrival; we planned on explaining to them the working of chromoproteins 
			and letting them perform a lysate reaction in tubes. We started with a presentation 
			on what the iGEM competition is, placing it within its context of synthetic biology 
			and explaining how this field is transforming the world we live in.</P></DIV><DIV class="row"><P class="col-md-7"> We then explained the workshop we had set up for them. Experiments included showing 
			the enzyme activity of beta-galactosidase, which is closely related to our project 
			as it is the reporter protein in our detection scheme. Another experiment was 
			performing an E.Coli lysate reaction, where we explained how we make the lysate 
			in the lab and what the advantages of a cell-free environment are. They would also 
			prepare a PCR reaction, after we explained to them how the PCR machine works and 
			that it's something we use every day in the lab, as well as prepare an agarose gel 
			and run the electrophoresis and check the gel under the UV illuminator at the end. 
			The last experiment would serve to explain how recombinant proteins, transformation 
			and cloning all are connected as the students would plate E.Coli cells transformed 
			with chromoproteins on agar plates. Beforehand a safety coordinator at EPFL had 
			approved this workshop, and we worked with P1 equipment and components only. In 
			the lab, we equipped every student with a lab coat, a pair of gloves and glasses 
			to ensure safety and  and divided them into groups of two. In those groups they 
			performed the experiments, with us explaining some select biological principles 
			at work (eg. protein expression, cell-free biology, transformation, cloning) and
		 a paper manual we wrote to resume the course of experiments. At the end of the 
		 day and after a successfull workshop, they gave us feedback on our performance 
		 as well as the experiments we had chosen for them. We took many of their comments 
		 into account for developing the Mini-Kit, which is described in detail below.</P><FIGURE class="col-md-5"><FIGCAPTION> Drawing with Chromoproteins, a high schooler's interpretation</FIGCAPTION></FIGURE></DIV><H2 class="top-space bottom-space">Open Plant Forum and Workshop, Cambridge</H2><P> At a later occasion, three team members went on a trip to Cambridge, 
	UK where we followed the Open Plant Forum in July and more specifically 
	the part of it that treated cell-free synthetic biology. We were able to
	get in touch with Keith Pardee, on whose paper<A href="#1"><SUP>1</SUP></A> 
	we based the toehold part of our project (a toehold switch gets triggered by 
	a complementary trigger sequence, unfolds and let's the downstream reporter 
	gene be transcribed). Together we talked about our experimental setup, because 
	at the time we were quite desperate as we hadn’t managed to get the toehold 
	switch to unfold yet, so it was a relief to hear we were on a good track. 
	We were working on a software to streamline the process of finding toehold/trigger 
	pairs from a given input sequence, which would be the first computer program of its 
	kind and a great tool for anyone working with toehold switches, so we also discussed 
	our approach to the code with Pardee, namely how much weight certain parameters in 
	our scoring function could be attributed.</P><P> By chance we met the Oxford iGEM team at that very same conference, and it 
		was the beginning of a fruitful collaboration as they also work on a cell-free 
		biosensor and we were able to exchange protocols and ideas on improving the 
		E.Coli lysate reaction later on. However the highlight of this trip was definitely 
		the one-day workshop on the integration of cell-free biology in the educational 
		system of the UK. Named “Open Plant curriculum development working group”, this 
		workshop was an interactive opportunity to discuss the advantages of including 
		cell-free biology in the UK’s high school curricula and exchange ideas on the 
		implementation thereof. Our first-hand knowledge of manipulating and making E.Coli 
		lysate was a good starting point for many conversations, as we were closest to the 
		target age group and were able to contribute unique points of view.</P><P>At the end of the day, we were convinced that cell-free biology will provide 
		high schools with an amazing tool to explain certain principles of biology 
		(eg. gene transcription, translation and regulation, transformation, cloning, 
		energy of a system) all in a hands-on way, while at the same time developing 
		an intuition for biology that students cannot get from books. The idea of our 
		Mini-Kit was born: A small toolbox that would enable teachers to give a live 
		demonstration of various biomolecular processes. An easy to handle, high-impact 
		collection of experiments that poses no biohazard as it doesn’t contain living 
		cells, perfect for explaining the concepts listed before to high school kids. 
		A creative and novel way to stimulate each student’s learning process. The workshop 
		was ideal to give us a roadmap, as a lot of technical details were discussed, 
		and it inspired us to take this message home and implement it in our local context.</P><FIGURE><FIGCAPTION>iGEM EPFL at the Curriculum Development Workshop</FIGCAPTION></FIGURE><H2 class="top-space bottom-space">The Educational Cell Free Mini Kit project</H2><P>Ever since the workshop in Cambridge, iGEM EPFL 2017 aimed to difuse cell-free biology as an accessible and 
        exciting scientific field for the younger. Thus arose the idea of the Educational Cell Free Mini Kit 
        <A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit"><FONT color="blue">(ECFK)</FONT></A> that consists in a set of five experiments intended for demonstrations at 
        local high schools. Leaving a footprint after the end of the competition was also something dear to the team, who 
        found solace in the distribution of the minikit to several high schools in the Canton of Geneva, set in the 
        francophone part of Switzerland. For this purpose, we reached out to several high schools 
	located in regions nearby EPFL to ensure they had interests in our kit and that they would 
	add it to their classes. A few days later we received many encouraging responses from high school and university 
        professors in Geneva: François Lombard, Alexandra Suter de Iaco, Bertrand Emery and Jean-Luc Zanasco showed great 
        interest in the implementation of such a kit in their theoretical and practical lectures and provided us with precise 
        counselling on how to improve it in order to make it as accurate and suitable for high school classes as possible. 
        Professor at the University of Geneva and manager of the plateform
	<A href="https://www.bioutils.ch"><FONT color="blue">BiOutils'</FONT></A> Karl Perron also shared his 
        enthusiastic thoughts about the project and reviewed our protocols for the minikit. As BiOutils is a platform which 
        takes care of the production and distribution of experiments for high schools for the whole Canton of Geneva, we 
        transmitted him our will to bring cell-free biology into the classes and he agreed to have BiOutils ship our ECFK 
        project ! </P><P>The ECFK consists of several tubes containing different lysate reaction components, 
	ideal to show a range of biomolecular processes. There are two kits: one of them is freeze dried, 
	which enables simpler manipulations and storage for high schools with low 
	budget; the other one isn't freeze dried, which enables students to learn how to pipette correctly
	and perform more elaborate experiments. The mini kit also contains a detailed description of the experiments that can 
        be performed with it as well as suggestions of topics (such as cell-free biology) that the teacher could find 
        interesting to include in his biology lectures. In fact, the ECFK is a diverse toolbox suitable for the demonstration 
        of biological principles  to groups with basic knowledge in biology, as the teacher is free to provide the 
        students with an adequate background for their grade level. </P><P>We also reached out to our university’s biosafety department to make sure we 
	were complying with national safety requirements. </P><P><A href="https://static.igem.org/mediawiki/2017/d/d8/T--EPFL--page_biosafety_evaluation.pdf"><FONT color="blue">Biosafety evaluation ECFK</FONT></A></P><H2 class="top-space bottom-space">Professional Input on Viral Contenders</H2><P> When the time came to test our software in previously unchartered territory 
		(eg. with new viruses), our first concern was finding a suitable ‘viral contender’. 
		It had to be present in a bodily fluid, if possible with high amounts of viral RNA 
		or DNA circulating. In order to make the best choice and detect a virus having a 
		high impact on today’s health landscape, we contacted several professionals form 
		different organizations and labs for their input. This included the following 
		scientists : Jacques Fellay whose lab is working on Human Genomics of infectious 
		diseases, virus-host interactions and translational genomics at EPFL; Samuel Cordey 
		from the Virology lab at HUG in Geneva; Bruno Correia heading the laboratory of 
		Protein Design and Immunoengineering at EPFL; and Didier Trono from the laboratory 
		of Virology and Genetics at EPFL.</P><P> Their suggestions for a relevant virus that needed quick detection as well 
		as our team brainstorming led us to single out the Hepatitis virus as a main 
		contender, of which we then chose genomic sequence unique to Hepatitis C, 
		genotype 1, as it is the most common of all Hepatitis C (HCV) genotypes. 
		Today, Hepatitis C is diagnosed by taking blood and liver samples that need 
		to be analyzed by a hospital laboratory. Our testing would dramatically reduce 
		the amount of time needed to detect the virus and would also be more reliable, 
		as we are directly targeting the viral genome instead of HCV antibodies secreted 
		by the human immune system (which only get produced after several months of exposure 
		to the antigen).</P><H2 class="top-space bottom-space">Summer School : Shaping the Future of Bioengineering,
	 Davos</H2><FIGURE class="bottom-space"><FIGCAPTION>The summer school's program</FIGCAPTION></FIGURE><DIV class="row bottom-space"><FIGURE class="col-md-4"><FIGCAPTION>Poster Session</FIGCAPTION></FIGURE><DIV class="col-md-8"><P> In September, two of our team members went to Davos, Switzerland, to attend 
		the Summer School on Shaping the Future of Bioengineering. There we presented 
		our project as well as a first poster at a poster session and got feedback from 
		professionals in academia and industry on key experiments and how to present 
		our data in a more scientific manner. </P><P>At the same time we learned about and discussed related applications and expansions 
	to our project (like what a 3D dignostics kit could look like, how to test for several 
	diseases at the same time, who would be interested in this kits, and much more), 
	which decidedly had a big impact on our ideas and is further explained in the 
	Outlook section. </P></DIV></DIV><FIGURE><FIGCAPTION>Group photo of participants, speakers and organisators</FIGCAPTION></FIGURE><H2 class="top-space bottom-space">Computational Biology Symposium, EPFL</H2><P> We also took part in the Computational Biology Symposium 2017 hosted at EPFL 
		in October to present the software part of our project during a poster session. 
		Three team members were present to explain what our iGEM project was about and 
		exchange ideas on how to make our code public in the best way. At the end of the 
		day we were happy to have found the most suitable solution to this problem and 
		having gained absolutely inspiring insights into the world of computational 
		biology.</P><H2 lass="top-space bottom-space">Reaching out to industry</H2><P>Our project was confined in the EPFL and we wanted to reach more people outside 
		the academic world. To figure out how we could improve our project further and 
		whether our concept would be interesting to labs from the industry, we contacted 
		Medisupport, a laboratory group that performs a broad range of medical analyses 
		for hospitals and doctor’s offices, servicing whole Switzerland. As they are 
		in touch with the industry, they have a quite new opinion and they would know 
		exactly the implementability of aptasense and what where the advantages or
		weaknesses. They turned out to be quite interested in our project and our 
		detection scheme of proteins, so that in October one of their representatives 
		came to EPFL to meet with one of our team members and discuss our diagnostics 
		tool in more detail. After two hours of discussion, Medisupport told us they 
		would want to put us in contact with one of their geneticists. Hence, the 
		subsequent week, we had a second meeting and they gave us more details about 
		the outlook’s project and how to improve our project.</P><FIGURE><FIGCAPTION>Meeting with a scientist from Medisupport</FIGCAPTION></FIGURE><P>The found the general idea quite ambitious and new, which was one of our goal. 
		However, as they perfectly know the diagnostic world, they told us that it is 
		really difficult to implement a new tool because of the trust that we must have 
		in the results given by the diagnostic tool. Hence, it requires a huge series of 
		tests that we couldn’t achieve considering the time and ressources that we had. 
		Thus, one anther field to reach was the food industry as the tests are less advanced. 
		Concerning the diagnostic field, Aptasense could have a future in liquid biospy for 
		detecting tumoral DNA in blood or detecting bacterial markers/activity as the methods 
		used nowadays are quite expensive. To conclude, it was a great opportunity to meet 
		them and to have the possibility to implement their advices.</P><P id="1" class="cit">1. Pardee, Keith, et al. &quot;Rapid, low-cost detection of Zika virus using programmable biomolecular components.&quot; Cell 165.5 (2016): 1255-1266. </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>