<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Description_Lysates skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Description/Lysates</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="apta-menu-bar"><P style="float:right; color:white; padding-right:60px;padding-top:10px;" class="apta-menu-icon"><B>MENU</B></P></DIV><DIV class="apta-wrapper"><DIV class="apta-menu-item"><UL><LI class="main-submenu-item"><A href="https://2017.igem.org/Team:EPFL">Home</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Project</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Summary">Results overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Outlook">Outlook</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Team</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Team">Our team</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Human practices</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit">Educational CF Kit- Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit/Storyline">Educational CF Kit - The Project</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lysates</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Lysates">Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates/Catalog">Catalog</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Aptamers</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts/Description">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Aptamers">Microfluidics</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers">Microfluidics - Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Beads">Beads</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers2">Beads - Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Toehold</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Toehold">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold">Results- Trigger Modularity</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold/Signal_Generation">Results - Signal Generation</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Software</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software/Demo">Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lab</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Safety">Safety</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Protocols">Protocols</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Notebook">Notebook</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts">Parts</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Medal requirements</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Contribution">Contribution</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Silver">Silver</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Gold Integrated</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Demonstrate">Proof of Concept</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Improve">Improvement</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Prizes</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Engagement">Education &amp; Public Engagement</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Software</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Applied_Design">Applied Design</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Integrated Human Practices</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Contact</LI><LI class="sec-submenu-item">epfl.igem2017@gmail.com</LI><LI class="sec-submenu-item"><I><A href="https://www.facebook.com/igemepfl/?fref=ts" class="fa fa-facebook fa-2x">facebook</A></I></LI><LI class="sec-submenu-item"><I><A href="https://twitter.com/epfl_igem?lang=en" class="fa fa-twitter fa-2x">twitter</A></I></LI></UL></DIV></DIV><DIV class="text-center top-space center-block" style="width:60%"><H1 class="top-space">Lysates </H1><H2 class="top-space bottom-space"> Cell-free systems</H2><P> Synthetic biology has the main goal of creating artificial biological systems 
 that might eventually be used for application in various domains such as medicine 
 or research. Modifying cells to develop new systems is very labour, time intensive
 and difficult. Synthetic biology approaches problems engineering-driven with design-test 
 cycles, which can still take up considerable amounts of time. Furthermore, the exact 
 behaviour of cells and the influence of interactions between newly engineered and 
 native parts cannot be predicted.</P><P> Cell-free systems use all of the inner workings of a cell without having the 
	constricting boundary of the cell wall and thus the precondition of keeping cells 
	alive. This allows speeding the design-test cycles up. When preparing the cell-free 
	systems, all genomic DNA and membranes are eliminated, resulting in a solution 
	containing all of the cells proteins without the limiting factors of a living cell. 
	This system can be used to express proteins for example. Furthermore, having eliminated 
	the membrane of cells also allows bacteria to detect and sense proteins, RNAs and DNAs 
	that would normally not be found in the cytoplasm.</P><P>One need of the project was to produce the enzyme β-galactosidase as signaling 
	molecule. Working in lysates was one way to do it. Another possibility was to work 
	in <A href="https://www.neb.com/products/e6800-purexpress-invitro-protein-synthesis-kit#Product%20Information"><FONT color="blue">PURExpress</FONT></A>, but it is substantially more expensive, 
	thus it was used only with 'delicate' experiments.</P><P>According to a paper from Sun et al<A href="#1"><SUP>1</SUP></A>, 
	home-made lysate costs approximatively only 0.03$ per μL (depending on 
	the method of cell lysis) instead of 0.79$ for PURExpress. 
	Moreover, they estimated that the production of protein in PURExpress lies at 0.2 mg/ml 
	versu s0.75 mg/ml for home-made lysate, which clearly favours lysate. 
	However, the protein production efficiency can vary between lysate batches, bacterial 
	strains or which protein is produced. PURExpress does have this considerable advantage: 
	The whole environment is a lot more controlled and thus there is less variability with results.</P><FIGURE class="bottom-space top-space"><FIGCAPTION><B>Figure 1: </B>Comparison of prices and protein production for different expression medium (from Sun et al<A href="#1"><SUP>1</SUP></A>).</FIGCAPTION></FIGURE><P>Hence, the first step of the project was to produce functional lysates from 
	different strains of bacteria.</P><H2 class="top-space bottom-space"> Lysate preparation</H2><P>Lysates preparation is quite long but the big advantage is that it gives a 
	protein expression system that is cheap and can be lyophilized for better storage. 
	The first step of production is to make a preculture of cells in growth medium 
	(LB or YPTG) incubating overnight. Then, a sample of the preculture must be added 
	again to growth medium in order to have enough bacteria and must be incubated for 
	4 hours. The next day, the cells are pelleted by centrifugation, washed and 
	sonicated (method based on ultrason in order to break the membrane). Afterwards 
	the solution are again pelleted in order to separate the supernatant - the lysate 
	- from debris. Finally, lysate is extracted from the debris, aliquoted and stored 
	at -80°C. For one good bacterial culture of 200ml, there is usually between 0.5 
	and 0.7 ml of lysates obtained (gives between 200 and 280 reactions). </P><FIGURE class="bottom-space top-space"><FIGCAPTION><B>Figure 2: </B>Workflow of lysate preparation</FIGCAPTION></FIGURE><H2 class="top-space bottom-space"> Lysate reaction</H2><P>The next step after lysate production was to make an functional energy solution 
	in order to be able to test the lysates. As lysates is based on using all the 
	working material of cells without keeping them alive, some components that are 
	consumable need to be added. Hence, an energy solution <A href="#2"><SUP>2</SUP></A> 
	containing amino acids, the four NTPs, tRNA, CoA, NAD, cAMP, salts 
	(magnesium for example) and other things necessary to the cell’s machinery to 
	work is added to the lysate reaction. For an optimized lysate reaction, some 
	buffer is added too.</P><P>Finally, DNA, coding for a protein, can be added to the lysate reaction 
		(lysate, energy solution and buffer A) in order to produce protein.</P><P>In the project, there is two main expressed proteins: enhanced green fluorescent 
		protein (EGFP) and β-galactosidase. For EGFP, the lysate reaction contains DNA 
		coding for EGFP, lysate, energy solution, buffer A and water for correct final 
		reaction volume. EGFP was used for quantitive reactions in order to determine 
		from which bacteria strains the lysates were the most efficient. For β-galactosidase, 
		the reaction is the same except that the DNA is lacZ (codes for β-galactosidase) and 
		that 5-Bromo-4-Chloro-3-Indolyl β-D-Galactopyranoside (chlorophenol red) must be added. 
		Chlorophenol red is a polysaccharide, which gives a yellow color to the substrate and 
		change to purple when cutted by β-galactosidase. The production of this enzyme was 
		mainly used for qualitative experiment because only few enzymes must be produced 
		in order to have a signal. </P><P> Another way to have more sensitive signal was studied: we showed the α-complementation. 
	B-galactosidase is a tetramer of two different subunits: α and Ω parts respectively coded
  by lacZα and lacZΩ genes. These subunits are non-functional alone, but can be assembled 
	in order to produce a functional β-galactosidase.<A href="#3"><SUP>3</SUP></A> Hence, we decided to produce lysates 
	from bacterial strains that can’t produce the α subunit because of a mutation in lacZα 
	gene and to produce this part afterwards in a lysate reaction. We discovered that these 
	two α and Ω parts can be assembled afterwards in order to produce a functional enzyme. 
	As, the entire β-galactosidase, is a quite long protein to express, the alternative 
	of producing only the α subunit gave us a quicker and more sensitive signal.</P><P> We had some problems with the efficiency of protein production. Hence, as lysate 
	contains exonuclease that degrades DNA, we must find out a solution in order to avoid 
	this DNA degradation.</P><P> Thus, the next step of the project were to find a solution for this in order 
	to produce more proteins and thus have more sensibility of our diagnostic tool: 
	it was done by adding GamS in the reaction or by using plasmid DNA that cannot
  be degraded by exonuclease.</P><H2 class="top-space bottom-space"> GamS</H2><P> Linear DNA template that is added to a cell-free expression system is at 
		high risk of being cut by an exonuclease still present in the lysate <A href="#4"><SUP>4</SUP></A>. This 
		has as a result that less protein is being produced. When adding the DNA in 
		plasmid form this issue can be prevented. </P><P> The GamS protein of phage lambda is an inhibitor of the exonuclease. When 
		added purified to a cell-free system containing linear DNA template, protein 
		expression increases substantially. As purified GamS is quite expensive, 
		another way was explored : this effect could also be achieved when the production 
		of the GamS protein was induced in cells before the subsequent lysis of the cell 
		culture. We showed this by mixing two lysates: M15-T7, which is optimised for 
		protein production in cell-free, and Top10-GamS, which contains GamS protein. 
		Bacterial cells used for Top10-GamS lysate production was done by transformation 
		of Top10 cells with a plasmid coding for the gene GamS.</P><H2 class="top-space bottom-space">Lyophilisation</H2><P> Once cells are lysed and their inner workings extracted, storing the cell-free 
		extract and the energy solution becomes a delicate affair as it has to be stored 
		at -80°C. The necessary infrastructures for this specific storage are easily 
		accessible for scientific laboratories and big enterprises, but can quickly 
		become a problem for countries in development. Cell-free expression systems 
		are a valuable diagnostic tool for viral infections and many more pathogens, 
		so bringing this technique to low-resource regions can be essential. 
		Lyophilisation of the cell-free extracts allows storage at room temperature 
		and only requires a source of nuclease-free water for rehydration.</P><P>  We have freeze dried our lysates in different combinations and tested their 
		efficiencies upon rehydration.</P><P> We tried to optimise the lyophilisation by growing the cells used for 
		lysate production in YTPG medium instead of LB. This YTPG medium permits to 
		make the cells grow using glucose for source of energy and, thus, to use a 
		way simpler and cheaper energy solution. Furthermore, it was demonstrated 
		that YTPG medium permits to have an increased protein production after rehydration 
		of lyophilised lysate reaction.</P><H2 class="top-space bottom-space"> Lysates</H2><TABLE class="table table-striped bottom-space"><THEAD><TR><TH>Lysate</TH><TH>Organism</TH><TH>Strain</TH><TH>RNA polymerase</TH><TH>Antibiotic resistance</TH><TH>Induction</TH><TH> Alpha complementation</TH></TR></THEAD><TBODY><TR><TD>BL21(DE3)</TD><TD>E.coli</TD><TD>BL21(DE3)</TD><TD><UL><LI>E.coli</LI><LI>T7</LI></UL></TD><TD>-</TD><TD>IPTG</TD><TD>No</TD></TR><TR><TD>M15</TD><TD>E.coli</TD><TD>E.coli M15 DZ291</TD><TD><UL><LI>E.coli</LI></UL></TD><TD><UL><LI>Kanamycin</LI></UL></TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>M15-T7</TD><TD>E.coli</TD><TD>E.coli M15 DZ291</TD><TD><UL><LI>E.coli</LI><LI>T7</LI></UL></TD><TD><UL><LI>Kanamycin</LI><LI>Ampicillin</LI></UL></TD><TD>IPTG</TD><TD>Yes</TD></TR><TR><TD>DH5α</TD><TD>E.coli</TD><TD>DH5α™ T1R </TD><TD><UL><LI>E.coli</LI></UL></TD><TD>-</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>Top10</TD><TD>E.coli</TD><TD> Top10 </TD><TD><UL><LI>E.coli</LI></UL></TD><TD>-</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>Top10-GamS</TD><TD>E.coli</TD><TD>Top10 </TD><TD><UL><LI>E.coli</LI></UL></TD><TD><UL><LI> Ampicillin</LI></UL></TD><TD><UL><LI>IPTG</LI><LI>Arabinose</LI></UL></TD><TD>Yes</TD></TR></TBODY></TABLE><FIGCAPTION><B>Table 1:</B> A summary of all lysates produced.
	</FIGCAPTION><P class="top-space"> The α complementation, as explained before, is the assembly of the two subunits 
	of the β-galactosidase to produce a functional enzyme. Thus, when a bacteria strain 
	is mutated in the gene that code for the α part, this bacteria will not produce the 
	functional β-galactosidase. Hence, this lysate's strain can be used for colorimetric 
	detection as the chlorophenol red is progressively cut by β-galactosidase produced 
	from the DNA added to the lysate. Bacterial strains that contain not the mutation 
	in α part can not be used to study the expression of β-galactosidase by colorimetry 
	as the enzyme is already present in the lysate and will cut immediately all the 
  cholophenol red. </P><P>The IPTG induction is for T7 RNA polymerase production during the growth of cells 
	used for lysate production. T7 RNA polymerase is a polymerase that is very specific 
	to his promoter, make less mistakes when synthesis of RNA and produce more RNA. 
	These three advantages motived us to use the T7 promoter for optimised protein 
	expression in lysates.</P><P>Hence, we made M15-T7 lysate by transforming M15 cells with plasmid containing 
	the gene for T7 expression for three reasons: first, M15 cells are optimised for 
	protein production in cell free, these cells are mutated in the lacZα part so 
	colorimetric assays were possible and finally the T7 promoter, as explained before 
	is better than the natural present polymerase from e.coli cells. </P><P> In the following chapters, we will sometimes use the abbreviation BL21 instead of 
	BL21(DE3) for simplicity. </P><H2 class="top-space bottom-space"><SMALL> References</SMALL></H2><P id="1" class="cit">1. Sun, Zachary Z., et al. &quot;Protocols for implementing an Escherichia coli based TX-TL cell-free expression system for synthetic biology.&quot; Journal of visualized experiments: JoVE 79 (2013).</P><P id="2" class="cit">2. Kwon, Yong-Chan, and Michael C. Jewett. &quot;High-throughput preparation methods of crude extract for robust cell-free protein synthesis.&quot; Scientific reports 5 (2015).</P><P id="3" class="cit">3. Broome, Ann-Marie, et al. &quot;Expanding the utility of β-galactosidase complementation: piece by piece.&quot; Molecular pharmaceutics 7.1 (2010): 60</P><P id="4" class="cit">4. Murphy, Kenan C. &quot;The λ Gam protein inhibits RecBCD binding to dsDNA ends.&quot; Journal of molecular biology 371.1 (2007): 19-24. </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>