<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Outlook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Outlook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="apta-menu-bar"><P style="float:right; color:white; padding-right:60px;padding-top:10px;" class="apta-menu-icon"><B>MENU</B></P></DIV><DIV class="apta-wrapper"><DIV class="apta-menu-item"><UL><LI class="main-submenu-item"><A href="https://2017.igem.org/Team:EPFL">Home</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Project</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Summary">Results overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Outlook">Outlook</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Team</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Team">Our team</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Human practices</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit">Educational CF Kit- Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit/Storyline">Educational CF Kit - The Project</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lysates</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Lysates">Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates/Catalog">Catalog</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Aptamers</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts/Description">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Aptamers">Microfluidics</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers">Microfluidics - Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Beads">Beads</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers2">Beads - Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Toehold</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Toehold">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold">Results- Trigger Modularity</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold/Signal_Generation">Results - Signal Generation</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Software</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software/Demo">Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lab</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Safety">Safety</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Protocols">Protocols</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Notebook">Notebook</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts">Parts</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Medal requirements</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Contribution">Contribution</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Silver">Silver</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Gold Integrated</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Demonstrate">Proof of Concept</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Improve">Improvement</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Prizes</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Engagement">Education &amp; Public Engagement</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Software</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Applied_Design">Applied Design</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Integrated Human Practices</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Contact</LI><LI class="sec-submenu-item">epfl.igem2017@gmail.com</LI><LI class="sec-submenu-item"><I><A href="https://www.facebook.com/igemepfl/?fref=ts" class="fa fa-facebook fa-2x">facebook</A></I></LI><LI class="sec-submenu-item"><I><A href="https://twitter.com/epfl_igem?lang=en" class="fa fa-twitter fa-2x">twitter</A></I></LI></UL></DIV></DIV><DIV class="text-center top-space center-block" style="width:60%"><H1 class="top-space bottom-space"> Outlook</H1><P>Our project represents a new approach to sensing biomarkers. We propose a toolbox where components can easily be modified to recognize virtually any protein or nucleic acid alike. In the same way this concept allows for developments that make complex diagnostics possible and can be extended to a wide range of uses. Here we have written down our most interesting ideas regarding the potential future applications of our project.
</P><H2 class="top-space bottom-space"> Paper-based diagnosis</H2><P>One such idea was the use of paper-based microfluidics to make our biosensor fully self-contained. Using the principle of a lateral flow assay we could have eliminated the need for any washing steps. This was already demonstrated by Xu et al<A href="#1"><SUP>1</SUP></A>. Building off their result we would have tacked on our signal amplification scheme with the toehold switches and cell-free expression system. The resulting sensor would have been as easy to use and distribute as a pregnancy test.
</P><H2 class="top-space bottom-space"> The Dipstick</H2><P>A slightly more crazy idea of ours included the use of a 'dipstick'. The surface chemistry of the microfluidic chips we used, would performed on the surface of this stick. That way, aptamer 1 would be immobilized on the stick. We could then proceed to dip the stick into tubes containing the different components of our assay: A tube with the blood sample (and thus the target protein), a tube containing aptamer 2 (extended with a trigger sequence, see Fig. 1). The last step would be incubating the dipstick in a tube containing the toehold switch and components of a lysate reaction. If the target protein is present, one would observe a color change from yellow to purple in this tube.</P><FIGURE><FIGCAPTION><B>Figure 1: </B>Aptamer 1 is bound the dipstick surface, while aptamer 2 with the trigger extension will be in solution</FIGCAPTION></FIGURE><P>Had we had a bit more time, we would have definitely explored this idea further. It could also be imagined as similar to a beads experiment where the beads are equivalent to the surface of the dipstick.
</P><H2 class="top-space bottom-space">Double Testing</H2><P>Since we propose detection of proteins and nucleic acids, the two methods could also be used to complement each other when testing for a particular disease. The aptamers could recognize protein biomarkers while the toehold switches would be complementary to the viral genome. This approach takes two independent ways of detecting viral presence. Cross-linked detection would make detection of such systems more reliable with reduced risks of false positives.
</P><H2 class="top-space bottom-space">One assay - Multiple diagnoses</H2><P>For now we propose detection devices for the Hepatitis C and Zika viruses. One can easily imagine how we could put these tests together to have an all-in-one testing device which screens for several diseases at the same time. With our software <A href="https://2017.igem.org/Team:EPFL/Software" target="_blank"><FONT color="pink">Toehold Designer</FONT></A>, finding new toehold switches for any disease is done in a matter of minutes. The multi-test would be useful in case of ambiguous symptoms in a region with prevalence of several dangerous pathogens.</P><H2 class="top-space bottom-space">Differentiating between strains of the same virus</H2><P>We have generated the software <A href="https://2017.igem.org/Team:EPFL/Software" target="_blank"><FONT color="pink">Toehold Designer</FONT></A> that outputs the best toeholds for a specific unique viral sequence within minutes.

We could potentially further develop this tool to discriminate between different strains of a virus or different sub-genotypes. 
As gene variants may cause different symptoms and thus require different treatments, this process could be a way for patients to be treated in a more targeted way, improving the likelihood of successful treatment.
</P><H2 class="top-space bottom-space">Implementing Logic</H2><P>Our project is based on LacZ-α as a reporter gene and the process of α-complementation. This works on the principle of splitting the LacZ gene in two parts and spontaneous rejoining of the protein pieces in solution. So far, we have only tried the alpha variant of splitting it (with alpha being a short sequence at the beginning of the gene). However other variants are possible<A href="#2"><SUP>2</SUP></A> and could be tested with this reporter system. To implement logic circuits in our system, we could express the subparts of LacZ within different toehold switches triggered by aptamers recognizing different biomarkers. Only if all subunits of LacZ are expressed simultaneously will a color change be induced in the sample. This is an example of a simple AND gate. Many other combinations are possible and would add degrees of complexity to our cell-free reporter system.
</P><H2 class="top-space bottom-space"> Protein-regulated gene networks</H2><P>Additionally, while our project focused on making a protein biosensor, it also fits into a broader goal of creating a standard biochemical part that is able to control the transcription of a gene based on the concentration of a target protein, similarly to how a toehold switch can control transcription based on the presence of a trigger sequence<A href="#3"><SUP>3</SUP></A>. This would constitute a sort of protein-signal to genetic-signal transducer. Such a part would be useful in engineering bacteria to react to various proteins in their extracellular as well as intracellular environment, for example. 
</P><H2 class="top-space bottom-space">Applications in agriculture</H2><P>Crop diseases are a serious issue around the globe, and farmers with less means might not be able to detect these diseases in time without access to a lab. An easily distributed biosensor could allow these farmers to quickly identify whether their crops or harvests are affected and react accordingly.<A href="#4"><SUP>4</SUP></A></P><P>
Having more farmers able to report on the health of their crops would also help monitor the spread of crop diseases within a whole region. This could help avoid serious food shortages.
</P><H2 class="top-space bottom-space">Applications in the food industry</H2><P>There are several possibilities in the food industry. For instance our detection scheme could be useful to investigate the authenticity of wines. Wines from a certain region will contain traces of local bacteria strains. By using our toehold switches, we could easily determine if a wine actually originates in a particular  region or whether it is a case of wine fraud. In Germany alone, more than 20% of 6443 German wines tested in the scope of a study<A href="#5"><SUP>5</SUP></A> turned out to be fraudulent.  Our easily distributable detection system could be of great value to those businesses that have neither the means nor the interest to have a lab for those tests.
</P><H2 class="top-space bottom-space"><SMALL>References</SMALL></H2><P id="1" class="cit">1. Xu, Hui, et al. &quot;Aptamer-functionalized gold nanoparticles as probes in a dry-reagent strip biosensor for protein analysis.&quot; Analytical Chemistry 81.2 (2008): 669-675.</P><P id="2" class="cit">2. Broome, Ann-Marie, et al. &quot;Expanding the utility of β-galactosidase complementation: piece by piece.&quot; Molecular pharmaceutics 7.1 (2010): 60</P><P id="3" class="cit">3. Green, Alexander A., et al. &quot;Toehold switches: de-novo-designed regulators of gene expression.&quot; Cell 159.4 (2014): 925-939.</P><P id="4" class="cit">4. Fang, Yi, and Ramaraja P. Ramasamy. &quot;Current and prospective methods for plant disease detection.&quot; Biosensors 5.3 (2015): 537-561.</P><P id="5" class="cit">5. Holmberg, Lars. &quot;Wine fraud.&quot; International Journal of Wine Research 2010.2 (2010): 105-13.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>