<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Software</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="apta-menu-bar"><P style="float:right; color:white; padding-right:60px;padding-top:10px;" class="apta-menu-icon"><B>MENU</B></P></DIV><DIV class="apta-wrapper"><DIV class="apta-menu-item"><UL><LI class="main-submenu-item"><A href="https://2017.igem.org/Team:EPFL">Home</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Project</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Summary">Results overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Outlook">Outlook</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Team</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Team">Our team</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Human practices</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit">Educational CF Kit- Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit/Storyline">Educational CF Kit - The Project</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lysates</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Lysates">Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates/Catalog">Catalog</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Aptamers</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts/Description">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Aptamers">Microfluidics</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers">Microfluidics - Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Beads">Beads</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers2">Beads - Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Toehold</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Toehold">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold">Results- Trigger Modularity</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold/Signal_Generation">Results - Signal Generation</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Software</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software/Demo">Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lab</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Safety">Safety</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Protocols">Protocols</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Notebook">Notebook</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts">Parts</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Medal requirements</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Contribution">Contribution</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Silver">Silver</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Gold Integrated</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Demonstrate">Proof of Concept</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Improve">Improvement</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Prizes</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Engagement">Education &amp; Public Engagement</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Software</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Applied_Design">Applied Design</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Integrated Human Practices</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Contact</LI><LI class="sec-submenu-item">epfl.igem2017@gmail.com</LI><LI class="sec-submenu-item"><I><A href="https://www.facebook.com/igemepfl/?fref=ts" class="fa fa-facebook fa-2x">facebook</A></I></LI><LI class="sec-submenu-item"><I><A href="https://twitter.com/epfl_igem?lang=en" class="fa fa-twitter fa-2x">twitter</A></I></LI></UL></DIV></DIV><DIV class="text-center top-space center-block" style="width:60%"><H2 class="bottom-space top-space">What is <B> Toehold Designer</B>? </H2><P>Generating new <A href="https://2017.igem.org/Team:EPFL/Description/Toehold"><FONT color="pink"><B> toehold</B></FONT></A>
		 switches requires <B>in-silico</B> processing. It is a quick step (~5 min) 
		 if a tool that pipelines the required processes is available. In the scope of our 
		 project we worked with existing toehold switches and also generated our own switches 
		 targeting <B>Hepatitis C viral RNA</B> (see <B><A href="https://2017.igem.org/Team:EPFL/Results/Toehold"><FONT color="pink">results</FONT></A></B>).</P><P>Toehold Designer strictly follows the Series A and B designs of the Zika 
			virus<SUP><A href="#2">2</A></SUP>, generalized for any unique virus sequence.</P><P>Thanks to the NUPACK Design <SUP><A href="#4">4</A></SUP> and 
			Analysis<SUP><A href="#5">5</A></SUP> algorithms this tool analyses 
			the optimality of the structure of the generated sequences
			by evaluating the target's availability for binding with the toehold 
			<B>(1)</B>, the toehold's availiability for binding with the target 
			<B>(2)</B> and the sensor's complex defect <B>(3)</B>. </P><P>By summing up <B>(1), (2)</B> and <B>(3)</B>, and assigning to each of them 
			a different weight <SUP><A href="#1">1</A></SUP> we can order the generated 
			sequences by score from lowest up to the highest score where the sequences 
			with lowest score have the most optimal structures.</P><P>It also makes use of the NUPACK Web application<SUP><A href="#3">3</A></SUP> 
			in order to generate images of the optimal secondary structure of the generated 
			sequence. </P><P>You can then choose the first few Series A or B toeholds and test them in 
			the lab.</P><H2 class="top-space bottom-space">The processing in more details</H2><P> Given a unique sequence for a virus, the required steps to generate a new toehold switch (from the A or B series) are: <UL><LI> Split the sequence to smaller target sequences with a sliding window of length 36<SUP><A href="#2">2</A></SUP></LI><LI> Evaluate these smaller target sequences for their single-strandedness (i.e. their availability for binding with the toehold switch). This step is done by using the <B>pair</B> functionality of NUPACK</LI><LI> For each target sequence build the Series A and B toehold switches<SUP><A href="#2">2</A></SUP></LI><LI> Evaluate the first 30 nucleotides of the generated Series A toehold switches for their single-strandedness (i.e. their availability for binding with the toehold switch).  This step is also done by using the <B>pair</B> functionality of NUPACK </LI><LI> Calculate the complex defect for every Series A toehold. This step is done by using the <B>complex defect</B> functionality of NUPACK</LI><LI> Combine these coefficients in a single score for every toehold by giving them different importance coefficients<SUP><A href="#1">1</A></SUP></LI><LI> Sort the toeholds by score in ascending order and chose the first toehold switches as the best ones</LI><LI>Display the results in the browser </LI><LI>For each Series A toehold give the possibility to see the different score components, 
														the Series B sequence generated for the same target sequence and an image of 
														the secondary structure generated by the NUPACK Web application<A href="#3">3</A></LI></UL></P><P> Watch the video tutorial below to learn more!</P><H2 class="top-space bottom-space">How does it work?</H2><H2 class="top-space bottom-space">Installation in 5 steps!</H2><UL><LI>Install NUPACK 3.0 from <A href="http://nupack.org/downloads"><FONT color="pink">here</FONT></A></LI><LI>Install Python from <A href="https://www.python.org/downloads/release/python-360/"><FONT color="pink">here</FONT></A></LI><LI>Open the command line and run the following commands: 
		                  	python get-pip.py
		                  	pip install beautifulsoup4 Flask</LI><LI>Download and unzip the project <A href="https://static.igem.org/mediawiki/2017/b/bc/T--EPFL--toehold-designer.zip"><FONT color="pink">here</FONT></A></LI><LI>Go to root of this repository and write in the command line:
		                  python toehold_designer.py</LI></UL><P>That's it. You have the toehold designer running in your browser now! </P><P> The Toehold designer is also available on our <A href="https://github.com/EPFLliGem/Toehold-Designer">github page</A>. Any future contributions are welcome!

		</P><H2 class="top-space bottom-space"> References </H2><P><A name="1">1. Green, Alexander A., et al. &quot;Toehold switches: de-novo-designed regulators of gene expression.&quot; Cell 159.4 (2014): 925-939.</A></P><P><A name="2">2. Pardee, Keith, et al. &quot;Rapid, low-cost detection of Zika virus using programmable biomolecular components.&quot; Cell 165.5 (2016): 1255-1266.</A></P><P><A name="3">3. NUPACK Web Application </A>
		 J. N. Zadeh, C. D. Steenberg, J. S. Bois, B. R. Wolfe, M. B. Pierce, A. R. Khan, R. M. Dirks, N. A. Pierce. NUPACK: analysis and design of nucleic acid systems. J Comput Chem, 32:170–173, 2011.</P><P><A name="4">4. NUPACK Analysis Algorithms </A></P><UL><LI> R. M. Dirks, J. S. Bois, J. M. Schaeffer, E. Winfree, and N. A. Pierce. Thermodynamic analysis of interacting nucleic acid strands. SIAM Rev, 49:65-88, 2007. </LI><LI> R. M. Dirks and N. A. Pierce. An algorithm for computing nucleic acid base-pairing probabilities including pseudoknots. J Comput Chem, 25:1295-1304, 2004. </LI><LI> R. M. Dirks and N. A. Pierce. A partition function algorithm for nucleic acid secondary structure including pseudoknots. J Comput Chem, 24:1664-1677, 2003.</LI></UL><P><A name="5">5. NUPACK Design Algorithms</A><UL><LI> B. R. Wolfe, N. J. Porubsky, J. N. Zadeh, R. M. Dirks, and N. A. Pierce. Constrained multistate sequence design for nucleic acid reaction pathway engineering. J Am Chem Soc, 139:3134-3144, 2017. </LI><LI> B. R. Wolfe and N. A. Pierce. Sequence design for a test tube of interacting nucleic acid strands. ACS Synth Biol, 4:1086-1100, 2015. </LI><LI>J. N. Zadeh, B. R. Wolfe, and N. A. Pierce. Nucleic acid sequence design via efficient ensemble defect optimization. J Comput Chem, 32:439–452, 2011. </LI><LI> R. M. Dirks, M. Lin, E. Winfree, and N. A. Pierce. Paradigms for computational nucleic acid design. Nucl Acids Res, 32:1392-1403, 2004. (pdf, supp info, supp seqs)</LI></UL></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>