<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_HP_Mini-Kit_Storyline skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/HP/Mini-Kit/Storyline</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="apta-menu-bar"><P style="float:right; color:white; padding-right:60px;padding-top:10px;" class="apta-menu-icon"><B>MENU</B></P></DIV><DIV class="apta-wrapper"><DIV class="apta-menu-item"><UL><LI class="main-submenu-item"><A href="https://2017.igem.org/Team:EPFL">Home</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Project</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Summary">Results overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Outlook">Outlook</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Team</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Team">Our team</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Human practices</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit">Educational CF Kit- Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit/Storyline">Educational CF Kit - The Project</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lysates</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Lysates">Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates/Catalog">Catalog</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Aptamers</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts/Description">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Aptamers">Microfluidics</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers">Microfluidics - Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Beads">Beads</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers2">Beads - Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Toehold</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Toehold">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold">Results- Trigger Modularity</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold/Signal_Generation">Results - Signal Generation</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Software</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software/Demo">Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lab</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Safety">Safety</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Protocols">Protocols</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Notebook">Notebook</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts">Parts</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Medal requirements</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Contribution">Contribution</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Silver">Silver</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Gold Integrated</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Demonstrate">Proof of Concept</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Improve">Improvement</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Prizes</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Engagement">Education &amp; Public Engagement</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Software</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Applied_Design">Applied Design</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Integrated Human Practices</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Contact</LI><LI class="sec-submenu-item">epfl.igem2017@gmail.com</LI><LI class="sec-submenu-item"><I><A href="https://www.facebook.com/igemepfl/?fref=ts" class="fa fa-facebook fa-2x">facebook</A></I></LI><LI class="sec-submenu-item"><I><A href="https://twitter.com/epfl_igem?lang=en" class="fa fa-twitter fa-2x">twitter</A></I></LI></UL></DIV></DIV><DIV class="text-center top-space center-block" style="width:60%"><H1 class="top-space bottom-space">Educational Cell Free Mini Kit (ECFK)</H1><H2 class="top-space bottom-space">1. Introduction </H2><P>This kit is made to give students the opportunity to perform experiments to
	illustrate what can happen in a cell in a simple and easily accessible way.
	Hence the kit is based on the « cell-free » technique.</P><P>Several experiments are prepared to show four elements:
	<UL style="list-style:disc"><LI>The presence, the functioning and the capacity of enzymes to produce a
		reaction in cells.</LI><LI>How are proteins synthetized in cells?</LI><LI>What do cells really need to synthetize these proteins?</LI><LI>How is the production of proteins regulated?</LI></UL></P><H2 class="bottom-space">2. Theoretical context</H2><P>The cell is a structural and functional unit made of several compartments and
 is confined by the plasma membrane, which separates the interior to the outside
 environment. “Cell-free” is a working method that consists of working without the
 plasma membrane and thus without a living cell. This technique makes transport and
 storage of reactants simpler, improves biosafety and reduces cost of experiments.</P><P>DNA (deoxyribonucleic acid) is a molecule present in every cell. It contains
	all genetic information of an organism, like a book telling a story. It consists
	of a double strand that is composed of a string of nucleotides (A, T, G and C)
	making up a genetic sequence. This genetic sequence is organized into genes
	allowing, when expressed, the expression synthesis of proteins.</P><DIV class="row"><DIV class="col-md-7"><P>A gene is a small portion of DNA that can be transcribed into RNA in order to
	be further translated into a protein. Such a gene can be expressed or regulated,
	depending on the needs of the cell. If the protein gets expressedWhen expressed,
	the gene will first be translated to RNA by a polymerase in order to be translated
	to produce a proteine.</P><P>RNA (ribonucleic acid) is a single-stranded molecule that is composed of the
	nucleotides A, U, G and C and is a translation of DNA.</P><P>The polymerase is a cellular machinery that allows transcription of
	DNA to RNA which then allows production of a protein. In order for the
	for the polymerase to work, it needs a particular sequence on the DNA
	called promoter. The promoter is a binding site that allows the polymerase
	to bind to the DNA and hence produce RNA. Each type of polymerase needs its
	specific promoter to work optimally (experiment 4).</P></DIV></DIV><P>Proteins are macromolecules made up of a sequence of amino acids. Each amino
	acid is encoded by 3 consecutive nucleotides present in a RNA. Proteins are
	necessary for the proper functioning of all biological systems. They are among
	other things responsible for the structure of muscles, the colour of your hair
	or eyes as well as the ability for a chemical reaction to happen – these particular
	proteins are called enzymes.</P><P>In order for protein production to happen (experiment 2), you need certain conditions:
	<UL style="list-style:disc"><LI>The presence of RNA – it’s the code for the protein.</LI><LI>The presence of ribosomes – a little machine that allows translation of the 3 consecutive bases in the RNA into amino acids.</LI><LI>The ribosome binding site so that the ribosome can bind to the RNA, scan it and produce the amino acid sequence (experiment 5).</LI><LI>Amino acids – they are usually present in the cell.</LI><LI>Energy in the form of ATP that the ribosome can use to advance on the RNA and to link the different amino acids that will make up the protein.</LI><LI>A stop codon present in the RNA that will signal the ribosome that all necessary amino acids have been linked and that translation has come to an end.</LI></UL></P><P>Enzymes are proteins that help to speed up a chemical reaction by providing
	the reactants with an environment that is favorable for the reaction. Without
	enzymes, a big number of reactions would never happen (experiment 2).</P><P>B-galactosidase, for example, is an enzyme that can cut sugars such as lactose
	but also a special sugar, chlorophenol red, that changes its colour from yellow
	to purple when cut. The enzyme ß-galactosidase (also called ß-gal) is thus often
	used in laboratories as a reporter meaning that the colour changes it produces
	indicates a reaction taking place. B-galactosidase is made out of structural
	subunits α and Ω. If one is absent, the enzyme is not working. This mechanism
	of complementation is also very popular in laboratories to check for the presence
	of two different components. B-galactosidase is encoded by a gene called lacZ,
	lacZα and lacZΩ respectively for the subunits.</P><P>The principal component of the “cell-free” technique is the lysate, a reactant
	that allows for protein production for example. Lysate is produced out of a
	bacterial cell culture. The cells in this culture are first collected at the
	bottom of a tube, were rid of any contaminants and debris and then suspended.
	In a final step of sonication, the plasma membranes are disrupted by ultrasound.
	As a result, the lysate contains all the inner workings of a cell except the membrane
	and the genome. </P><P>This increases biosafety by a lot because no living entities can be
	found in the solution and all DNA initially contained in the cell has been destroyed.
  The disadvantage of working without living cells is that the energy generating
	metabolism can no longer take place. This means that you need to add energy
	sources to the lysate in order to provide the protein expression machinery
	with sufficient fuel. Additionally, the storage of lysate is a delicate history,
	they need to be kept at -80°C. This problem can be avoided by lyophilizing of the
	lysate, a process that removes all water.</P><H2 class="top-space bottom-space">3. Materials</H2><H6 class="bottom-space"><B><I> Lyophilised kit</I></B></H6><H6 class="bottom-space"><B><I>Non lyophilised kit</I></B></H6><H2 class="top-space bottom-space">4. Experiences</H2><P>To prevent contamination of the experiments and to protect yourself form the
substances you are working with, wearing a lab coat and gloves is highly recommended.
While you are doing experiments, you should avoid touching your face and doorknobs
with your gloves. Furthermore, you should never leave the laboratory with your lab
coat and gloves on. Once done with all the experiments, take the gloves off by turning
them inside out but try not to touch the parts that came into contact with parts of the
experiment with your bare hands. </P><P>Normally, tubes containing reactants are labelled according to their content.
Here, they are labelled with a letter to simplify the experiments and to avoid
situations of confusion about which tube to use.</P><H3 class="top-space bottom-space">4.1 Presence of the enzyme ß-galactosidase in a cell</H3><H4 class="top">4.1.1. Goal</H4><P>To test for the presence of the enzyme ß-galactosidase in a lysate made of a
	cell culture.</P><H4 class="top">4.1.2. Description of the experiment</H4><P>There are 2 tubes containing lysate, energy solution, a buffer (to provide the
	appropriate reaction environment) and the substrate of ß-galactosidase
	(chlorophenol red) that when cuted on will produce a colour change.</P><P>Two lysates made from two different cell lines are provided: one of the cultures
	 contains a full version of ß-galactosidase. The other has a non-functional version
	 of the enzyme as the cells cannot produce one of its subunits as they contain a
	 mutation in the gene and are thus missing the information to code the full enzyme.</P><H4 class="top">4.1.3. Protocol</H4><H6><B><I>Lyophilised kit</I></B><UL><LI>Rehydrate the lyophilised solutions of BL21 and Dh5α by adding 35 µl of
	nuclease-free water.</LI></UL></H6><H6><B><I>Non lyophilised kit</I></B><UL><LI>Make a mastermix containing:
	</LI><LI>10 µl of energy solution (label ES)</LI><LI>10 µl of buffer A (label buffer)</LI><LI>4 µl of nuclease free water (label NFW)</LI><LI>4 µl of substrate (label sub)</LI></UL><LI>Equally separate the mastermix in two tubes and add in one tube 5 µl of
		solution 1.1 and in the other tube 5 µl of solution 1.2, don’t forget to
		label the tubes !</LI></H6><H6><B><I>For both kits</I></B><UL><LI>Let incubate until the colour change becomes obvious (immediately).</LI><LI>Which lysate changed colour ?</LI></UL></H6><H4 class="top">4.1.4. Expected Results<P>In one of the two tubes should be able to observe a change in colour. The change
	from yellow to purple should be produced in the tube containing the lysate produced
	from the cells that have an intact copy of ß-galactosidase that cuts the substrate
	chlorophenol red. The other tube stays yellow because it contains an incomplete
	version of the enzyme and can thus not act on the substrate.</P><UL style="list-style:disc"><LI>Can you guess which culturesolution 1.1 or 1.2 contains the full version of the ß-galactosidase ?</LI><LI>Knowing that the lysate Dh5 has a mutation for the lacZ, guess which tubes 1.1 or 1.2 contain the lysate Dh5  ? </LI><LI>Does BL21 contain functional ß-galactosidase ? Why ?</LI></UL></H4><H3 class="top-space bottom-space">4.2. Expression of β-galactosidase</H3><H4 class="top">4.2.1. Goal</H4><P>Cell-free in vitro expression of ß-galactosidase.</P><H4 class="top">4.2.2. Description of the experiment</H4><P>There are two tubes: one containing a lysate without ß-galactosidase, an energy
	solution, a buffer, the substrate chlorophenol red and DNA LacZ that codes for
	 ß-gal, the other one everything listed before except the DNA. If the DNA is
	 present in a tube, the full version of ß-gal will be produced when the system
	 is rehydrated and thus functioning again. It is up to you to determine which
	 tube contains the DNA coding for ß-galactosidase.</P><H4 class="top">4.2.3. Protocol</H4><H6><B><I>Lyophilised kit</I></B><UL><LI>Rehydrate both tubes 2.1 and 2.2 by adding 35 µl of nuclease free water.</LI></UL></H6><H6><B><I>Non lyophilised kit</I></B><UL><LI>Make a mastermix containing:
	</LI><LI>10 µl of energy solution (label ES)</LI><LI>10 µl of buffer A (label buffer)</LI><LI>10 µl of M15-T7 lysate (label 2.1)</LI><LI>4 µl of substrate (label sub)</LI></UL><LI>Equally separate the mastermix in two tubes and add in one tube the 4 µl of
	solution 2.2 and in the other one, add 4 µl of the solution 2.3, don’t forget
	to label the tubes !</LI></H6><H6><B><I>For both kits</I></B><UL><LI>Let the tubes incubate until you see a change of colour (approx. 30 minutes). </LI><LI>Which tube contains the coding sequence for ß-gal ?</LI></UL></H6><H4 class="top">4.2.4. Expected Results</H4><P>One of the tubes will produce ß-galactosidase due to the presence of LacZ DNA.
	This shows in a colour change. The other tube is a negative control without DNA
	that allows the conclusion that the colour change really is due to the presence of DNA.</P><UL style="list-style:disc"><LI>What would happen if we add BL21 lysate in a protein expression system
			containing LacZ DNA ?</LI><LI>Would it be possible to produce other ß-galactosidase ?</LI><LI>What would happened if we forgot the energy solution ? Response in the next experiment !</LI></UL><H3 class="top-space bottom-space">4.3. The necessity of the energy solution in the lysate</H3><H4 class="top">4.3.1. Goal</H4><P>Definition of the importance of energy solution to produce proteins.</P><H4 class="top">4.3.2. Description of the experiment</H4><P>Energy solution is essential for the production of proteins as will be demonstrated
	in this experiment. The cells in the lysate are “dead” and can no longer produce all
	the required material for protein production by themselves. We thus have to provide
	the lysate will all the essential components for protein synthesis. The energy solution
	contains a mix of the 20 essential amino acids that make up proteins, ATP (a cell’s form
	of energy), different salts and some DNAs and RNAs that are needed and have been degraded
  in the process of making the lysate.</P><H4 class="top">4.3.3. Protocol</H4><H6><B><I>Lyophilised kit</I></B><UL><LI>Rehydrate the tube 3.1 by adding 35 µl of nuclease free water.</LI></UL></H6><H6><B><I>Non lyophilised kit</I></B><UL><LI>Make solution containing:
	</LI><LI>5 µl of buffer A (label buffer)</LI><LI>5 µl of M15-T7 lysate (label 3.1)</LI><LI>2 µl of substrate (label sub)</LI><LI>2 µl of DNA coding for ß-galactosidase (label 3.2) </LI></UL></H6><H6><B><I>For both kits</I></B><UL><LI>Let it incubate (approx. 5 minutes).</LI><LI>Witness the constant yellow colour.</LI></UL></H6><H4 class="top">4.3.4. Expected Results</H4><P>The colour of the solution in the tube should stay yellow. Because the energy
	solution has not been added to the solution, the machinery is not able to produce
	the enzyme ß-gal and as a matter of fact, no other protein either. The energy
	solution is an important aspect disadvantage in the field of “cell-free” as it
	is long and difficult to produce and can only sustain a lysate for so long.</P><UL style="list-style:disc"><LI>Why the solution stayed yellow ?</LI><LI>Try to add BL21 lysat, what happen ? Why ?</LI></UL><H3 class="top-space bottom-space">4.4. The Polymerase</H3><P>The polymerase is the cellular machinery that allows transcription of DNA to
	RNA, which can subsequently be translated to proteins. In order for the polymerase
	to be able to work properly, it needs a binding site, the promoter. Each different
	polymerase has its own specific promoter. If your lysate contains a certain type of
	DNA containing a promoter for another polymerase, almost nothing will be expressed.</P><P> In this subsequent experiment, two polymerases are studied: the E. Coli RNA
	polymerase who need the E. Coli promoter and the T7 RNA polymerase who need T7
	promoter for transcription of DNA to RNA.</P><H4 class="top">4.4.1. Goal</H4><P>Test the principle of the polymerase and its corresponding promoter.</P><H4 class="top">4.4.2. Description of the experiment</H4><P>The two tubes contain a different lysate each, energy solution, substrate and
	LacZ DNA with a promoter called T7. To have protein synthesis you thus need T7
	RNA polymerase in your lysate. The two lysates used in this experiment are M15-T7
	and DH5α. Can you guess which lysate contains the appropriate polymerase?</P><H4 class="top">4.4.3. Protocol</H4><H6><B><I>Lyophilised kit</I></B><UL><LI>Rehydrate the two tubes 4.1 and 4.2 by adding 35 µl of nuclease free water.</LI></UL></H6><H6><B><I>Non lyophilised kit</I></B><UL><LI>Make a mastermix containing:
		</LI><LI>10 µl of energy solution (label ES)</LI><LI>10 µl of buffer A (label buffer)</LI><LI>4 µl of substrate (label sub)</LI><LI>4 µl of lacZ DNA (label 4.1)  </LI><LI>2 µl of nuclease free water (label NFW)</LI></UL><LI>Equally separate the mastermix in two tubes and add in one tube the 5 µl of
		solution 4.2 and in the other one, add 5 µl of the solution 4.3, don’t forget
		to label the tubes !</LI></H6><H6><B><I>For both kits</I></B><UL><LI>Incubate until you see a colour change (approx. 30 minutes).</LI><LI>Observe the colour change.</LI></UL></H6><H4 class="top">4.4.4. Expected Results</H4><P>One of the tubes should change colour due to the presence of T7 polymerase and
	the resulting expression of ß-galactosidase based on the added LacZ DNA. The
	other tube is not able to produce the enzyme due to the missing polymerase.
	What is the interest of using ß-galactosidase to identify the transcription
	and subsequent translation of a certain DNA? When you add the coding sequence
	for ß-gal after your gene of interest you can verify the transcription of the
	gene of interest. Whenever this process happens, the gene for ß-gal is also
	transcribed and a colour change is thus produced, indicating that our protein
	of interest is produced.</P><UL style="list-style:disc"><LI>By observing the change of color, which one of the two tubes 4.2 or 4.3 contain lysate from cells that express T7 RNA polymerase ?
		</LI><LI>Why one solution stayed yellow ? Why the other one turned purple ?
	</LI></UL><H3 class="top-space bottom-space">4.5 The Ribosome Binding Site (RBS) -
	a laboratory technique allowing the regulation of protein expression</H3><H4 class="top">4.5.1. Goal<P>Identification of the presence of a particular trigger RNA – here a small part of the Zika virus RNA (We do not have the complete RNA of the virus present so no potential danger present!).</P></H4><H4 class="top">4.5.2. Description of the experiment</H4><P>There are many techniques to regulate protein expression. The one presented here
	has been specifically developed to be used in laboratories. A toehold switch is
	a piece of RNA in the form of a hairpin. This hairpin has as a result that both
  the ribosome binding site and the start codon are hidden for the ribosome and
	thus the encoded protein isn’t expressed. When a certain trigger RNA is present
	in the solution of the toehold switch, the latter will unfold and release the
	RBS and the start codon. The trigger RNA has a sequence that is complementary
	to the stem of the hairpin and when it binds, it opens up the structure. A
	ribosome can then bind to the RNA and produce proteins. This system is used
	to report the presence of a particular DNA, viral for example, that will trigger
	the toehold and thus protein expression.</P><P>The experiment will be done with a toehold that is followed by the LacZα gene,
		the coding sequence of the α subunit of ß-gal. When the trigger sequence is present
		in the solution, the toehold will unfold and as a result the  subunit is produced
		 and then assembled with the  subunit to produce a functional version of
		 ß-galactosidase. This will induce a colour change. The lysate contains an
		 incomplete version of ß-galactosidase, the Ω subunit. When the  subunit
		 assembles with the  subunit they create a fully working version of the enzyme.
		 This is a very useful mechanism for studying transcription and translation:
		 you add the coding sequence of LacZα after your gene of interest. When your
		 gene of interest is being transcribed and translated, LacZα is so too. If
		 you add this construct into a lysate containing only the  subunit, the two
		 can assemble. </P><H4 class="top">4.5.3. Protocol</H4><H6><B><I>Lyophilised kit</I></B><UL><LI>Rehydrate the two tubes 5.1 and 5.2 by adding 35 µl of nuclease free water.</LI></UL></H6><H6><B><I>Non lyophilised kit</I></B><UL><LI>Make a mastermix containing:
		</LI><LI>10 µl of energy solution (label ES)</LI><LI>10 µl of buffer A (label buffer)</LI><LI>10 µl of M15-T7 lysate (label 5.1)</LI><LI>4 µl of substrate (label sub)</LI><LI>2 µl of toehold (label 5.2)</LI></UL><LI>Equally separate the mastermix in two tubes and add in one tube the 2 µl of
		solution 5.3 and in the other one, add 2 µl of the solution 5.4, don’t forget
		to label the tubes !</LI></H6><H6><B><I>For both kits</I></B><UL style="list-style:disc"><LI>Incubate until you see a change of colour (approx. 30 minutes).</LI><LI>Which tube contains the trigger RNA of the Zika virus ?</LI></UL></H6><H4 class="top">4.5.4. Expected results<P>One of the tubes should change colour as the trigger RNA is present in the
	solution and thus unfolds the toehold switch. This is inducing the production
	of the  subunit of ß-galactosidase and subsequent assembly of a functioning enzyme.
	The other tube does not contain this trigger and thus also no functioning ß-gal. The
	first tube would correspond to an infected individual if the sample would originate
	from blood, while the second one corresponds to a healthy person.</P><UL style="list-style:disc"><LI>Which tube contain the trigger RNA ? Why ?</LI><LI>Does this diagnostic system would be possible if using BL21 lysate for
		ß-galactosidase expression ?</LI><LI>If we added a part of a RNA from the influenza, what would happen ?
		What is the interest ?</LI></UL></H4><H2 class="top-space bottom-space">5. References</H2><P><A href="https://www.ncbi.nlm.nih.gov/pubmed/27160350">Pardee, K., Green, A. A., Takahashi, M. K., Braff, D., Lambert, G., Lee, J. W., ... &amp; Daringer, N. M. (2016). Rapid, low-cost detection of Zika virus using programmable biomolecular components. Cell, 165(5), 1255-1266.</A></P><P>Reece, J. B., Urry, L. A., Cain, M. L., Wasserman, S. A., Minorsky, P. V., &amp; Jackson, R. (2014). Campbell biology (p. 135). Boston: Pearson.</P><H2 class="top-space bottom-space">6. PDF versions</H2><P><A href="https://static.igem.org/mediawiki/2017/f/f3/T--EPFL--page_ECFK_angl.pdf">ECFK PDF english version</A></P><P><A href="https://static.igem.org/mediawiki/2017/b/bd/T--EPFL--page_ECFK_french.pdf">ECFK PDF french version</A></P><P><A href="https://static.igem.org/mediawiki/2017/d/d8/T--EPFL--page_biosafety_evaluation.pdf">Biosafety evaluation ECFK</A></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>