<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="apta-menu-bar"><P style="float:right; color:white; padding-right:60px;padding-top:10px;" class="apta-menu-icon"><B>MENU</B></P></DIV><DIV class="apta-wrapper"><DIV class="apta-menu-item"><UL><LI class="main-submenu-item"><A href="https://2017.igem.org/Team:EPFL">Home</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Project</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Summary">Results overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Outlook">Outlook</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Team</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Team">Our team</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Human practices</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP">Overview</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit">Educational CF Kit- Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Mini-Kit/Storyline">Educational CF Kit - The Project</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lysates</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Lysates">Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Lysates/Catalog">Catalog</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Aptamers</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts/Description">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Aptamers">Microfluidics</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers">Microfluidics - Results</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Beads">Beads</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Aptamers2">Beads - Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Toehold</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Description/Toehold">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold">Results- Trigger Modularity</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Results/Toehold/Signal_Generation">Results - Signal Generation</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Software</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Description</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software/Demo">Results</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Lab</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Safety">Safety</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Protocols">Protocols</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Notebook">Notebook</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Parts">Parts</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Medal requirements</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Contribution">Contribution</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Silver">Silver</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Gold Integrated</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Demonstrate">Proof of Concept</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Improve">Improvement</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Prizes</LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Engagement">Education &amp; Public Engagement</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Software">Software</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/Applied_Design">Applied Design</A></LI><LI class="sec-submenu-item"><A href="https://2017.igem.org/Team:EPFL/HP/Gold_Integrated">Integrated Human Practices</A></LI></UL></DIV><DIV class="apta-menu-item"><UL><LI class="main-submenu-item">Contact</LI><LI class="sec-submenu-item">epfl.igem2017@gmail.com</LI><LI class="sec-submenu-item"><I><A href="https://www.facebook.com/igemepfl/?fref=ts" class="fa fa-facebook fa-2x">facebook</A></I></LI><LI class="sec-submenu-item"><I><A href="https://twitter.com/epfl_igem?lang=en" class="fa fa-twitter fa-2x">twitter</A></I></LI></UL></DIV></DIV><DIV id="title"> Protocols </DIV><DIV class="protocols-menu"><DIV><A href="#agrose-gel">Agarose gel electrophoresis</A></DIV><DIV><A href="#aptamer">Aptamer binding test</A></DIV><DIV><A href="#buffer-a">Buffer A</A></DIV><DIV><A href="#buffer-z">Buffer Z</A></DIV><DIV><A href="#colony-pcr">Colony PCR</A></DIV><DIV><A href="#color">Colorimetric pull-down aptamer triggering toehold</A></DIV><DIV><A href="#ES">Energy solution protocol</A></DIV><DIV><A href="#freeze-dry">Freeze dry</A></DIV><DIV><A href="#gibson-assambly"> Gibson assembly</A></DIV><DIV><A href="#glass-slides"> Glass slides preparation</A></DIV><DIV><A href="#glycerol-stock">Glycerol stock</A></DIV><DIV><A href="#in-vitro-rna-synth">In vitro RNA synthesis and purification </A></DIV><DIV><A href="#LB">LB (Luria-Bertani) medium</A></DIV><DIV><A href="#lysates">Lysate preparation</A></DIV><DIV><A href="#Mitomi">MITOMI chips fabrication</A></DIV><DIV><A href="#PAGE">PAGE Urea gel</A></DIV><DIV><A href="#PCR">PCR</A></DIV><DIV><A href="#plasmid-prep">Plasmid extraction protocol (Miniprep)</A></DIV><DIV><A href="#platereader">Platereader lysate reaction</A></DIV><DIV><A href="#purification">Purification of DNA</A></DIV><DIV><A href="#amplification">Signal Amplification</A></DIV><DIV><A href="#pull-down">Streptavidin bead aptamer pull-down assay</A></DIV><DIV><A href="#transformation">Transformation into DH5α/M15 cells</A></DIV><DIV><A href="#YTPG">YTPG medium protocol</A></DIV></DIV><DIV class="protocol"><P class="protocol-title">Agarose gel electrophoresis</P><P class="protocol-section">Materials </P><UL class="mat"><LI> DNA</LI><LI>RNAse free water</LI><LI>Agarose</LI><LI>1X Tris-acetate-EDTA (TAE) buffer</LI><LI>GelRed</LI><LI>1 kb DNA ladder (ladder must be adapted to DNA analysed)</LI><LI>6X loading buffer</LI><LI>Agarose gel trays</LI><LI>UV transilluminator</LI></UL><P class="protocol-section">Procedure</P><P class="protocol-subsection">Preparation of 1% agarose gel</P><UL class="step"><LI> Melt 600 mg of agarose in 60 ml of 1X TAE buffer in microwave oven.</LI><LI> Add 6 μl GelRed in the melted agarose.</LI><LI> Pour the melted agarose in the gel cast with the two combs set.</LI><LI>  Wait 30 minutes until the gel solidifies (faster in the 4°C fridge).</LI><LI> Cover the gel with 1X TAE buffer and remove the combs carefully.</LI><LI> Load the samples:
			</LI><LI>2 μl of 1 kb DNA ladder</LI><LI>Mix 1 μl of DNA with 1 μl of 2X times loading buffer</LI></UL><LI> Run the gel at 100 volts for 30 minutes.</LI><LI> Remove the gel from the chamber.</LI><LI> Visualize the DNA fragments using a UV transilluminator.</LI></DIV><DIV class="protocol"><P class="protocol-title"> Aptamer binding test </P><P class="protocol-section"> Materials</P><UL class="mat"><LI> Native human a-thrombin, (storage buffer: 50% water and 50% glycerol)</LI><LI>Cy5-labeled aptamer</LI><LI>Cy3-labeled aptamer</LI><LI>Biotinylated aptamer</LI><LI>PBS (Phosphate buffered saline) </LI><LI>Neutravidin</LI><LI>Biotin-BSA</LI><LI>Tris-HCl</LI><LI>NaCl</LI><LI>MgCl2</LI><LI>MITOMI 768 chip</LI></UL><P class="protocol-subsection">References to other protocols</P><UL class="mat"><LI><A href="#Mitomi">MITOMI chips fabrication protocol</A></LI><LI><A href="#glass-slides">Glass slide preparation protocol</A></LI></UL><P class="protocol-section">Procedure</P><P class="protocol-subsection">Aptamer Binding Buffer preparation</P><UL class="mat">
		Aptamer Binding Buffer consists of:
		<LI>50 mM Tris-HCl (pH 7.5)</LI><LI>100 mM NaCl</LI><LI>1 mM MgCl2</LI></UL><P class="protocol-subsection">Neutravidin Surface Chemistry</P><UL class="step"><LI>Set pressure for control layer at 15 Psi, flow layer at 3 Psi.</LI><LI>Fill in the control lines with dH2O one at a time, check that all lines are working. Close chamber valves (neck valves).</LI><LI>Flow 2 mg/ml biotin-BSA with the general outlet open until BSA approximately reaches the outlet. Close the outlet and allow any air inside the chip to escape. Then flow the BSA for 20 minutes in order to passivate the glass surface.</LI><LI>After this point, each time a new reagent is to be flowed through the chip, make sure no air enters the chip. Close the general inlet and open the waste and reagent lines. Allow the reagent to flow through the waste line for 30 seconds to let air escape. Then close the waste line and open the general inlet and allow the reagent to flow.</LI><LI>Wash chip with PBS for 5 minutes.</LI><LI>Flow 500 µg/ml Neutravidin for 20 minutes in order to bind to biotin, buttons up.</LI><LI>Wash chip with PBS for 5 minutes, buttons up.</LI><LI>Flow 2 mg/ml biotin-BSA for 20 minutes in order to block Neutravidin sites outside button area, buttons down.</LI><LI>Wash chip with PBS for 5 minutes, buttons down.</LI><LI>Chip now has biotin-binding neutravidin sites blocked everywhere by biotin-BSA but in the button area.</LI></UL><P class="protocol-subsection">Sandwich test</P><UL class="step"><LI>Multiplex the chip in 4.</LI><LI>Flow the biotinylated aptamer (2 μM) into the chip for 2 minutes buttons down, then 15 minutes buttons up.</LI><LI>Close the button valves.</LI><LI>Wash with aptamer binding buffer for 3 minutes with buttons closed, then 5 minutes with buttons open.</LI><LI>In the first quarter and third quarter of the chip, flow thrombin 2 μM for 2 minutes buttons down, then 15 minutes buttons up.</LI><LI>Flow the Cy5 labeled aptamer into the upper half chip for 5 minutes buttons down.</LI><LI>Remove the flow line from the flow manifold (but not from the chip).</LI><LI>Wash with aptamer binding buffer for 5 minutes, buttons down.</LI><LI>Flow the Cy3 labeled aptamer into the lower half chip for 5 minutes buttons down.</LI><LI>Remove the flow line from the flow manifold (but not from the chip).</LI><LI>Image the chip buttons up.</LI><LI>Wash with aptamer binding buffer for 5 minutes, buttons down.</LI><LI>Image the chip buttons down.</LI></UL><P class="protocol-subsection">Imaging on fluorescent microscope</P><UL class="step"><LI>Use the NIS program.</LI><LI>Align the chip on the microscope with the grid in the software. Make sure that every chip cell is in the field of view when selected from the grid in the software. Set the upper left chip cell as the reference point.</LI><LI>Choose the appropriate laser/channel for imaging.</LI><LI>Choose the exposure time in order to maximize the signal:noise ratio, not to saturate the camera, and not to photobleach the fluorophores.</LI><LI>Choose a filename.</LI><LI>Press “run now&quot;. The program should take an image of each cell in the chip.</LI><LI>When the image is created, select it and export it to a new folder using the “export ND document&quot; option, under “export&quot; in the “file » menu.</LI><LI>Select “mono image&quot; and the name, then click “export ».</LI><LI>Open ImageJ. Go to “plugins&quot;, “stitching&quot;, “grid&quot;. Choose the grid size, no overlap, make sure “display fusion&quot; is selected, and select the folder where you saved the image from the microscope. Choose a file name, but it must end in “xy{iii} ».</LI><LI>Once the stitched image is generated, scale it down by pressing ctrl-E.</LI></UL><P class="protocol-citation"><A href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200261.full.pdf">MITOMI : A Microfluidic Platform for In Vitro Characterization of Transcription Factor-DNA Interaction, Rockel Sylve, Geertz Marcel, Maerkl Sebastian J., 2009, Springer</A></P><P class="protocol-citation"><A href="https://www.ncbi.nlm.nih.gov/pubmed/9368651">Oligonucleotide Inhibitors of Human Thrombin that Bind Distinct Epitopes, Tasset Diane M., Kubik Mark F., Steiner Walter, Journal of Molecular Biology 272, 1997, p.688-698</A></P><P class="protocol-citation"><A href="https://www.ncbi.nlm.nih.gov/pubmed/17218526">Maerkl, S. J. &amp; Quake, S. R. A Systems Approach to Measuring the Binding Energy Landscapes of Transcription Factors. Science (80-315), 233–237 (2007).</A></P></DIV><DIV class="protocol"><P class="protocol-title">Buffer A</P><P class="protocol-section">Materials</P><UL class="mat"><LI>Deionized water</LI><LI>Tris base</LI><LI>Magnesium glutamate  (L-Glutamic acid hemimagnesium salt tetrahydrate)</LI><LI>Potassium glutamate (L-Glutamic acid potassium salt monohydrate)</LI><LI>DTT (1,4-Dithiothreitol)</LI></UL><P class="protocol-section">Procedure </P><P class="protocol-subsection"> Preparation of 1M stock solutions </P><UL class="step"><LI>Weight 12.11 g of tris buffer (121.14 g/mol), add it to 100 ml of deionized water and adjust to pH 8.2 with acetic acid.</LI><LI>Weight 38.86 g of magnesium glutamate (388.61 g/mol), add it to 100 ml of deionized water.</LI><LI>Weight 20.32 g of potassium glutamate (203.23 g/mol), add it to 100 ml of deionized water.</LI></UL><P class="protocol-subsection"> Preparation of 1L of buffer A </P><UL class="step"><LI>Add 10 ml of 1 M tris acetate (final concentration is 10 mM).</LI><LI>Add 14 ml of 1 M magnesium glutamate (final concentration is 14 mM).</LI><LI>Add 60 ml of 1 M potassium glutamate (final concentration is 60 mM).</LI><LI>Store at room temperature.</LI><LI>If necessary, add 2 ml of 1 M DTT (final concentration is 2 mM) in order to have buffer A + DTT and store at -20°C.</LI></UL></DIV><DIV class="protocol"><P class="protocol-title"> Buffer Z</P><P class="protocol-section">Materials</P><UL class="mat"><LI>Phosphate buffered saline (PBS buffer), stored at 4°C</LI><LI>$MgCl_2$ (stock solution at 50 mM), stored at -20°C</LI><LI>Beta-mercaptoethanol (BME, stock solution : 14.3 M), stored at 4°C</LI></UL><P class="protocol-section">Procedure</P><UL class="step">
		Buffer Z should be prepared in small quantities just before use, as the BME is not stable in this buffer. BME should be handled with care as it is an extremely sticky and stinky chemical (keep inside several bags), work under the hood to avoid toxic fumes.
		<LI> To make 1 mL of Buffer Z, add to a 1.5 Eppendorf tube:
			</LI><LI>976.5 μl PBS</LI><LI>20 μl MgCl2</LI><LI>3.5 μl BME</LI></UL><LI>Keep the buffer on ice until use. </LI></DIV><DIV class="protocol"><P class="protocol-title"> Colony PCR</P><P class="protocol-section"> Materials </P><UL class="mat"><LI> Plate containing bacterial colonies </LI><LI>Tips</LI><LI>LB medium</LI><LI>0.5 ml microcentrifuge tube</LI><LI>Thermocycler</LI><LI>Buffer HF</LI><LI>dNTP</LI><LI>DMSO</LI><LI>Forward primer</LI><LI>Backward primer</LI><LI>Phusion polymerase</LI></UL><P class="protocol-subsection"> References to other protocols</P><UL class="mat"><LI><A href="#agrose-gel">Agarose gel electrophoresis protocol</A></LI></UL><P class="protocol-section"> Procedure</P><UL class="step"><LI>Work in sterile environment</LI><LI>Label all colonies to test on the plate</LI><LI>Set up the program on the thermocycler:
			</LI><LI>98°C for 2 minutes for initial denaturation</LI><LI>30 cycles:
					</LI><LI>98°C for 30 seconds for denaturation</LI><LI>30 seconds for primers annealing at temperature depending on primers and polymerase</LI><LI>72°C for 30 seconds per kb of DNA to amplify for elongation</LI></UL><LI>72°C for 7 minutes for final extension</LI><LI>4°C on hold</LI><LI>Make a mastermix with quantity according to number of colony to be tested. For two reactions of 50 μl, mix
			</LI><LI>to 100 μl nuclease free water</LI><LI>20 μl of 5X HF Buffer</LI><LI>2 μl of 10mM dNTPs</LI><LI>5 μl of 100% DMSO</LI><LI>0.5 μl of 100 μM forward primer</LI><LI>0.5 μl of 100 μM backward primer</LI><LI>Touch a bacterial colony with a pipette tip, touch the bottom of a 0.5 ml centrifuge tube, throw the tip in 5 ml of LB medium for overnight bacterial culture and label the tubes.</LI><LI>Aliquot mastermix in each tube that were touched by a pipette tip containing colony and label them.</LI><LI>Vortex and centrifuge.</LI><LI>Add 0.375 μl of 2000 U/ml phusion polymerase to each tube.</LI><LI>Load the tubes in the thermocycler and start the program.</LI><LI>Control the colony PCR by doing an agarose gel electrophoresis (follow <A href="#agrose-gel">Agarose gel electrophoresis protocol</A>).</LI></DIV><DIV class="protocol"><P class="protocol-title"> Colorimetric pull-down aptamer triggering toehold protocol </P><P class="protocol-section">Materials</P><UL class="mat"><LI>Dynabeads™ MyOne™ Streptavidin T1</LI><LI>Biotinylated aptamer 1</LI><LI>Aptamer extended with trigger sequence for toehold (aptamer 2 extended with trigger)</LI><LI>Target protein</LI><LI>DynaMag™ Magnet</LI><LI>Sample mixer allowing tilting and rotation of tubes</LI><LI>Binding and Washing (B&amp;W) Buffer 2X: 
	        </LI><LI>10 mM Tris-HCl (pH 7.5)</LI><LI>1 mM EDTA</LI><LI>2 M NaCl</LI><LI>PBS/0.01%BSA/0.01%Tween20</LI></UL><LI>Aptamer Binding Buffer</LI><LI>Toehold reaction:
	        </LI><LI>Energy solution</LI><LI>Buffer A</LI><LI>Top10-gamS lysate</LI><LI>M15-T7 lysate</LI><LI> Nuclease free water</LI><LI>Toehold-lacZ at 420 ng/μl</LI><LI>X-gal at 15 mg/ml</LI><P class="protocol-section">References to other protocols</P><UL class="mat"><LI><A href="#ES">Energy solution protocol</A></LI><LI><A href="#lysates">Lysate protocol</A></LI><LI><A href="#buffer-a">Buffer A protocol</A></LI><LI><A href="#platereader">Plate-reader protocol</A></LI></UL><P class="protocol-subsection">Aptamer Binding Buffer preparation</P><UL class="mat">
		Aptamer Binding Buffer consists of:
		<LI>50 mM Tris-HCl (pH 7.5)</LI><LI>100 mM NaCl</LI><LI>1 mM MgCl2</LI></UL><P class="protocol-section">Procedure</P><UL><LI>Each time a washing buffer (B&amp;W or PBS) is added, let sit for 2-3 min before discarding supernatant.</LI></UL><P class="protocol-subsection"> Wash Dynabeads™ MyOne™ magnetic beads</P><UL class="step"><LI>Resuspend Dynabeads in vial and vortex for 30 seconds minimum.</LI><LI>Transfer 5 μl of Dynabeads to a 200 μl microcentrifuge tube.</LI><LI>Add 5 μl of 2X Washing Buffer and resuspend.</LI><LI>Place the tube on a magnet for 1 min and discard the supernatant.</LI><LI>Remove tube from the magnet and resuspend the washed magnetic beads in 10 μl of 2X Washing Buffer. </LI><LI>Repeat steps 4-5 once, for a total of 2 washes.</LI><B>IMPORTANT:</B> The two next big steps: « Immobilize biotinylated aptamer » and « Thrombin-aptamer binding » must be performed in parallel.
	</UL><P class="protocol-subsection"> Immobilize biotinylated aptamer</P><UL class="step"><LI>Resuspend washed Dynabeads™ magnetic beads in 10 μl of 2X B&amp;W Buffer.</LI><LI>Discard supernatant.</LI><LI>Add 10 μl of 2 μM biotinylated aptamer 1.</LI><LI>Incubate for 20 min at room temperature using gentle rotation.</LI><LI>Wash once with 10 μl of PBS.</LI><LI>Wash once with 10 μl B&amp;W buffer.</LI></UL><P class="protocol-subsection"> Thrombin-aptamer binding</P><UL class="step"><LI>In a separate tube, prepare a 10 μl solution containing 1 μM target protein and 1 μM of aptamer 2 in Aptamer Binding Buffer.</LI><LI>Incubate the tube for 20 min at room temperature using gentle rotation.</LI></UL><P class="protocol-subsection"> Flow Thrombin and Thrombin-aptamer dimer</P><UL class="step"><LI>Add 10 μl of Aptamer Binding Buffer containing 1 μM of target protein-aptamer 2 complex.</LI><LI>Incubate for 20 min at room temperature using gentle rotation.</LI><LI>Separate beads with a magnet and discard the supernatant.</LI><LI>Wash once with 10 μl of PBS.</LI><LI>Wash the coated beads once with 10 μl of 2X B&amp;W Buffer.</LI></UL><P class="protocol-subsection"> Adding trigger solution to toehold reaction</P><UL class="step"><LI>From the tube with the in vitro transcribed trigger, add 8.1 μl of toehold reaction in tubes containing beads. Toehold reaction should be prepared as follows:
			</LI><LI>2.5 μl of energy solution</LI><LI>2.5 μl of buffer A</LI><LI>1.1 μl of nuclease free water</LI><LI>1.25 μl of Top10-gamS lysate</LI><LI>1.25 μl of M15-T7 lysate</LI><LI>0.6 μl toehold-lacZ</LI><LI>0.2 μl of X-gal</LI></UL><LI>Every 20-30 min, pull beads aside using the magnet and check for colorimetric evolution. Once measurements are taken, mix the solution and the beads and repeat for 4-5 hours.</LI><P class="protocol-citation"><A href="https://www.ncbi.nlm.nih.gov/pubmed/17297945">Centi, S., Tombelli, S., Minunni, M., &amp; Mascini, M. (2007). Aptamer-based detection of plasma proteins by an electrochemical assay coupled to magnetic beads. Analytical chemistry, 79(4), 1466-1473.</A></P></DIV><DIV class="protocol"><P class="protocol-title"> Energy solution protocol</P><P class="protocol-section">Materials </P><P class="protocol-section">Procedure</P><P class="protocol-subsection">Make amino acids stock solution</P><UL class="step"><LI>In one tube weigh each amino acid one by one using a small parafilm paper. </LI><LI>Add all the amino acids into the same tube apart from the tyrosine that should be added in a separate tube by itself. </LI><LI>To the first tube (amino acid – tyrosine ) add 500 ul of water in total: small volumes of water should be added periodically before adding the 500 ul total. After each addition check for the pH, it should be around 5-6. We can also add small volumes of KOH 1% (100 ul max) to dissolve the amino acids in the solution.</LI><LI>To the tyrosine only tube add 400 ul of water and 10 ul of KOH. Add those volume periodically while vortexing to help the amino acid dissolve in solution. The pH should be around 9.</LI><LI>Do not mix the two tubes together at this stage of the preparation to prevent precipitation.</LI></UL><P class="protocol-subsection">Making energy solution</P><UL class="step"><LI>After preparing the amino acids, prepare the rest of the components by weighing each time the component in a tube and adding the amount of water listed in the table below. Vortex well so the component dissolves. </LI><LI>After weighing all the components, mix the amino acids and the rest of the solutions together, according to the table.</LI><LI>Aliquot into small 25ul tube and flash freeze in liquid nitrogen. </LI><LI>Store in a -80°C.</LI></UL></DIV><DIV class="protocol"><P class="protocol-title"> Freeze dry (lyophilisation) </P><P class="protocol-section">Materials</P><UL class="mat"><LI>Freeze dry machine</LI><LI>Microfuge tubes</LI><LI>Drilled caps</LI><LI>Liquid nitrogen</LI><LI>Parafilm</LI><LI>Vacuum bell</LI><LI>Kimwipes</LI><LI>Nuclease free water</LI><LI>Incubator</LI><LI>Any other components you want to freeze dry</LI></UL><P class="protocol-section">Procedure</P><P class="protocol-subsection">Freeze dry</P><UL class="step"><LI>Label tubes according to what will be freeze dried and weigh them.</LI><LI>Mix all reactants that will be freeze dried in the tube and directly flash-freeze them in liquid nitrogen (wear glasses) in order to stop the possible reactions.</LI><LI>Take some caps of microfuge tubes and drill some holes.</LI><LI>Prepare the freeze drying machine (take 20 minutes).</LI><LI>Take out the microfuge tubes from the liquid nitrogen with tweezers (wear glasses) and put the drilled caps on (as fast as possible).</LI><LI>Put the tubes in the machine and run the program overnight.</LI></UL><P class="protocol-subsection">Storage</P><UL class="step"><LI>Take the tubes out ofthe machine when the program is finished and remove the drilled caps.</LI><LI>Weigh the tubes again and calculate how much water is needed for rehydration (difference between the two weights).</LI><LI>Put parafilm on the caps of the tube to seal them off.</LI><LI>Put the tubes under a vacuum bell.</LI></UL><P class="protocol-subsection">Rehydration</P><UL class="step"><LI>Put some kim wipes in a clean box and humidify them with nuclease free water.</LI><LI>Put the tubes open in the box and incubate them for 2 hours at 37°C.</LI><LI>Add the quantity of water evaporated according to calculations at step 2.2 in each tube.</LI></UL></DIV><DIV class="protocol"><P class="protocol-title"> Gibson assembly</P><P class="protocol-section">Materials</P><UL class="mat"><LI>Primers</LI><LI>DNA fragments</LI><LI>DPN1</LI><LI>Linearized plasmid DNA</LI><LI>Gibson assembly master mix</LI><LI>Deionized water</LI><LI>Incubator</LI><LI>Purification of DNA materials</LI><LI>Transformation materials</LI><LI>Colony PCR materials</LI><LI>Extraction of plasmid DNA materials</LI><LI>Agarose gel electrophoresis materials</LI><LI>Glycerol stock materials</LI></UL><P class="protocol-subsection">References to other protocols</P><UL class="mat"><LI><A href="#purification">Purification of DNA protocol</A></LI><LI><A href="#transformation">Transformation protocol</A></LI><LI><A href="#colony-pcr">Colony PCR protocol</A></LI><LI><A href="#plasmid-prep">Extraction of plasmid DNA protocol (Miniprep)</A></LI><LI><A href="#agrose-gel">Agarose gel electrophoresis protocol</A></LI><LI><A href="#glycerol-stock">Glycerol stock protocol</A></LI></UL><P class="protocol-section">Procedure</P><P class="protocol-subsection">Gibson assembly</P><UL class="step"><LI>Do PCRs of DNA fragments to insert with right primers.</LI><LI>Add 1 μl of DPN1 to 50μl of PCR products.</LI><LI>Incubate 1 hour at 30°C.</LI><LI>Do a purification of PCR products using Qiagen purification kit <A href="#purification">Purification of DNA protocol</A>.</LI><LI>Do the calculations in order to have between 50-100 ng of plasmid DNA and two or three-fold of insert in pmols. So, the calculations are:
			</LI><LI>$pmoles_{plasmid} =  \frac{m_{plasmid(ng)} \cdot 1000}{bp_{plasmid} \cdot 650}$</LI><LI>$pmoles_{insert} = pmoles_{plasmid} \cdot fold$</LI><LI>$m_{insert(ng)} = \frac{pmoles_{insert} \cdot 650 \cdot bp_{insert}}{1000}$</LI></UL><LI>Mix 10 μl of Gibson assembly master mix 2X, plasmid DNA and insert and complete reaction with deionized water until 20 μl.</LI><LI>Incubate sample 15 minutes at 50°C.</LI><LI>Store sample at -20°C before doing the transformation.</LI><P class="protocol-subsection">Subsequent steps in order to have plasmid DNA</P><UL class="step"><LI>Transform competent cells with 2 μl of the assembly reaction and follow <A href="#transformation">Transformation protocol</A>.</LI><LI>Select colonies: touch a colony with the tip, touch the bottom of a 1.5 ml microfuge tube for colony PCR and, for bacterial culture, discard the tip in a culturing tube containing 5 ml of LB medium and antibiotics corresponding to the resistance of the cells. Follow protocol: <A href="#colony-pcr">Colony PCR protocol</A>.</LI><LI>Choose colony to do the miniprep according to the right band after loading the colony PCRs in an agarose gel. Follow protocol: <A href="#agrose-gel"> Agarose gel electrophoresis protocol</A>.</LI><LI>Do miniprep in order to have plasmid DNA following <A href="#plasmid-prep">Plasmid extraction protocol (Miniprep)</A>.</LI><LI>Make a glycerol stock of the chosen colony according to <A href="#glycerol-stock">Glycerol stock protocol</A>.</LI></UL><P class="protocol-citation"><A href="https://www.neb.com/protocols/2012/12/11/gibson-assembly-protocol-e5510">Gibson Assembly® Protocol (E5510), New England Biolabs, 2017.</A></P></DIV><DIV class="protocol"><P class="protocol-title">Glass slides preparation </P><P class="protocol-section">Materials</P><UL class="mat"><LI>Standard (76x26x1 mm) microscope glass slides (ECN 631-1550)</LI><LI>MQ water</LI><LI>Ammonium hydroxide ($NH_4OH$; 25%)</LI><LI>Hydrogen peroxide ($H_2O_2$; 30%)</LI><LI>Solvents:
			</LI><LI>Acetone</LI><LI>Toluene</LI><LI>IPA (isopropyl alcoholl)</LI></UL><LI>3-glycidoxypropyl-trimethoxymethylsilane (3-GPS; 97%)</LI><LI>Nitrogen gas supply</LI><P class="protocol-section">Procedure</P><P class="protocol-subsection">Cleaning</P><UL class="step"><LI>Add 600 ml of MQ water + 120 ml of NH4OH (5:1 ratio) in a glass container A (staining bath). Put the container on a hot plate and wait until it reaches 80°C (Set it to 500 initially in the Arec TC until it reaches 80-90°C and then dial down the knob to 350°C; it usually takes about 20 minutes to reach 80°C). Monitor the temperature of the solution using the thermometer from the incubator chamber in the PDMS room.</LI><LI>Add 150 ml of $H_2O_2$ to glass container A.</LI><LI>Add the glass slides into the metallic holder.</LI><LI>Add the holder with the slides into the glass container A. Microscope glass slides must be from VWR with cut edges (ECN 631-1550, 50/pk).</LI><LI>Incubate for 30 minutes.</LI><LI>Take out glass slides and let it cool for 5 minutes.</LI><LI>Let the glass container A cool down also for 5-10 minutes. Dispose the solution into the base aqueous water when the solution has cooled down to room temperature. Rinse it with MQ water and dry it as it will be used in the next step to store the dried glass slides.</LI><LI>Fill the other glass container B with MQ water and load the glass slides into it.</LI><LI>Dry each glass slides with nitrogen and put them into the other glass container A.</LI></UL><P class="protocol-subsection">Epoxysilane Deposition</P><UL class="step"><LI>Turn on oven at 120°C</LI><LI>Use two nitrile gloves in each hand - toluene can penetrate through a single layer of nitrile.</LI><LI>Dump the water from the glass container B. Keep the metallic holder inside. Glass container A will be used to keep the glass slides so keep it always closed.</LI><LI>Rinse glass container B with acetone as well as the glass cylinders (500 ml and 10 ml volumes). No need to rinse with water.</LI><LI>Dump the acetone into the organic (solvent) waste.</LI><LI>Dry glassware at 80°C for a few minutes. Then let it cool down to room temperature. Avoid as much as possible any dust particles inside the containers so always keep them closed.</LI><LI>Prepare solution of 9 ml 3-Glycidoxypropyl-trimethoxymethylsilane (GPS, 97%) in 891 ml toluene in the glass container B.</LI><LI>Incubate glass slides for 20 minutes inside the solution.</LI><LI>Dump toluene-GPS into organic (solvent) waste.</LI><LI>Rinse each glass slides with fresh toluene to remove unbound 3-GPS.</LI><LI>Dry each glass slides with $N_2$.</LI><LI>Bake glass slides at 120°C for 30 minutes.</LI></UL><P class="protocol-citation"><A href="https://www.ncbi.nlm.nih.gov/pubmed/21938622">MITOMI : A Microfluidic Platform for In Vitro Characterization of Transcription Factor-DNA Interaction, Rockel Sylve, Geertz Marcel, Maerkl Sebastian J., 2009, Springer.</A></P><P class="protocol-citation"><A href="https://www.ncbi.nlm.nih.gov/pubmed/17218526">Maerkl, S. J. &amp; Quake, S. R. A Systems Approach to Measuring the Binding Energy Landscapes of Transcription Factors. Science (80-. ). 315, 233–237 (2007).</A></P></DIV><DIV class="protocol"><P class="protocol-title">Glycerol stock</P><P class="protocol-section">Materials</P><UL class="mat"><LI>Bacterial overnight culture</LI><LI>Cryogenic vials</LI><LI>40% glycerol in $H_2O$</LI><LI>Liquid nitrogen</LI><LI>-80°C fridge</LI></UL><P class="protocol-section">Procedure</P><UL class="step"><LI>Work in sterile environment.</LI><LI>Add 0.5 ml of 40% glycerol to a cryogenic vial.</LI><LI>Add 0.5 ml sample from the bacterial culture to the cryogenic vial and pipette up and down in order to mix.</LI><LI>Flash free in liquid nitrogen – wear glasses.</LI><LI>Store at -80°C fridge.</LI></UL></DIV><DIV class="protocol"><P class="protocol-title">In vitro RNA synthesis and purification</P><P class="protocol-section"> Materials </P><UL class="mat"><LI> NTP Buffer Mix</LI><LI> T7 RNA Polymerase</LI><LI> Nuclease Free Water</LI><LI> Template dsDNA</LI><LI> Incubator </LI><LI> DNAse </LI><LI> $LiCl_2$ (2,5 M) </LI><LI> 70% ethanol </LI><LI> MQ water</LI></UL><P class="protocol-subsection"> References to other protocols</P><UL class="mat"><LI><A href="#PAGE">PAGE gel protocol</A></LI></UL><P class="protocol-section"> Procedure</P><UL class="step"><LI> First, wipe everything down with EtOH, bench, tube racks, pipettes and use RNase free tips. </LI><LI> Make 20 μl reaction (first add water and nTPs, then buffer and T7 enzyme mix last):
			</LI><LI> x μl RNase free water (to 10 μl) </LI><LI> 10 μl NTPs Buffer mix</LI><LI> 1 μg template dsDNA</LI><LI> 2 μl RNA polymerase mix</LI><LI> Incubate at 37°C overnight</LI></UL><P class="protocol-subsection">DNase digest run</P><UL class="step"><LI> Add 0.5-1 μl DNAse to in vitro to reaction tube</LI><LI> Incubate at 37°C for 15 minutes</LI></UL><P class="protocol-subsection"> LiCl<SUB>2</SUB> precipitation</P><UL class="step"><LI> Exchange rotter and start cooling the centrifuge to 4°C.</LI><LI> Add 30 μl LiCl<SUB>2</SUB> to RNA product and mix thoroughly. </LI><LI> Incubate at -20°C for 30 minutes. </LI><LI> Centrifuge at 4°C for 15 minutes at max speed to pellet the RNA. </LI><LI> Remove the supernatant. </LI><LI> Wash with 70% EtOH. </LI><LI> Centrifuge at 4°C for 15 minutes. </LI><LI> Remove supernatant and resuspend in MilliQ water. </LI><LI> Aliquot in 5 μl tubes to preserve RNA from degradation from the freezing and unfreezing. </LI><LI> Run an PAGE Urea gel (follow <A href="#PAGE">PAGE gel protocol</A>) with one aliquot of the RNA purification product in order to check the RNA synthesis. </LI><LI> Store at -20°C. </LI></UL></DIV><DIV class="protocol"><P class="protocol-title">LB (Luria-Bertani) medium</P><P class="protocol-section"> Materials </P><UL class="mat"><LI> Autoclave</LI><LI> Yeast extract</LI><LI> Bacto-trypton</LI><LI> NaCl</LI><LI> mQ water</LI></UL><P class="protocol-section"> Procedure</P><UL class="step">
		For 1L of LB medium
		<LI>Mix :
			</LI><LI> 5 g yeast extract</LI><LI> 10 g bacto-tryptone</LI><LI> 10 mg NaCl</LI></UL><LI> Add mQ water until 1 L is reached. </LI><LI> Autoclave the medium for 15 minutes at 121°C. </LI><LI> When working with LB medium, always work next to the flame and sterilise bottle and cap under the flame in order to avoid contaminations. </LI></DIV><DIV class="protocol"><P class="protocol-title"> Lysate preparation </P><P class="protocol-section"> Materials </P><UL class="mat"><LI> Autoclave</LI><LI> LB medium autoclaved</LI><LI> Autoclaved tips and dishes</LI><LI> Flame</LI><LI> Buffer A</LI><LI> Buffer A + DTT</LI><LI> Tube for bacterial culture</LI><LI> Spectrophotometer</LI><LI> Plastic cuvettes</LI><LI> 50 ml Falcon tubes</LI><LI> Liquid nitrogen</LI><LI> Big and small centrifuge</LI><LI> Sonicator</LI></UL><P class="protocol-section"> Procedure </P><P class="protocol-subsection"> Preparation of Materials </P><UL class="step">
	For 2*200 ml of bacterial culture:
		<LI> Prepare minimum 420 ml of LB medium and autoclave for 20 min at 121ºC </LI><LI> Autoclave tips and dishes, which will be used for bacteria growth</LI><LI> Prepare minimum 120 ml buffer A (- DTT) </LI><LI> Prepare minimum 3 ml buffer A + DTT </LI></UL><P class="protocol-subsection">Bacteria growth</P><UL class="step"><LI> When working with bacteria, always work next to the flame to avoid contaminations and wear glasses when working with liquid nitrogen. </LI><LI> At 17h approximatively, add 5 ml of LB medium and antibiotics according to resistance’s cells to a tube and inoculate with a small amount of desired bacteria from the glycerol stock (pick by tip or inoculating loop). Put the top of the bottle of the LB and the cap under the flame to avoid contamination of LB medium (contaminate very easily).</LI><LI> Grow the culture overnight at 37°C, 200rpm, pay attention to keep the lid loose to allow for air exchange.</LI><LI> At 9h approximately, measure absorbance at 600 nm with 10X dilution of the culture (900 μl of LB + 100 μl of bacteria culture) in a plastic cuvette, OD600 should be around 4.</LI><LI> Add 1 ml overnight culture to 200 ml of LB medium and correct antibiotics in a 500 ml Erlenmeyer flask. </LI><LI> Incubate the culture for 2 hours at 37°C, 200 rpm (pay attention to the rpm setting – this speed is optimal for the growth). </LI><LI> Induce if necessary: 400 μl of 100 mM IPTG for T7 polymerase inducing; 5 ml of 10% arabinose for gamS inducing (under pBAD promoter). </LI><LI> Incubate again the culture for 2 hours at 37°C, 20 0rpm and measure absorbance, OD600 should be around 1.5 – 2. </LI></UL><P class="protocol-subsection"> Centrifugation and cleaning </P><UL class="step"><LI> Let the big centrifuge cool down to 4ºC (takes around 10 min). </LI><LI> Weigh one 50 ml falcon tube for one culture (clearly label the tube as well the cap). </LI><LI> Separate the grown culture to four 50 ml falcon tubes and spin at 4000 rpm at 4°C for 20 min.</LI><LI> Keep the bacteria on ice as much as possible.</LI><LI> Discard the supernatant and add 10 ml of buffer A to the bacteria pellet, re-suspend the bacteria by pipetting or by vortexing and transfer all four parts to one tube, spin at 4000 rpm at 4°C for 10 min.</LI><LI> Discard the supernatant and re-suspend the pellet in 10 ml of buffer A, spin at 4000 rpm at 4°C for 10 min.</LI><LI> Re-do the washing step 6 one more times.</LI><LI> Discard the supernatant (remove as much as possible).</LI><LI> Weigh the pellet (wet mass). </LI><LI> Wear protective glasses (danger of tube explosion) and flash freeze with liquid nitrogen, store at –80°C. </LI></UL><P class="protocol-subsection">  Sonication </P><UL class="step"><LI> Re-suspended the thawed bacteria based on their mass in buffer A by vortexing (1ml of buffer A + DTT per grams of thawed bacteria). </LI><LI> Keep the bacteria or lysate on ice as much as possible.</LI><LI> Aliquot 1 ml of re-suspended bacteria to a new 2 ml Eppendorf tube.</LI><LI> Place the tube in an ice bath (ice mixed with water) and place the sonicated tip inside the tube so it is immersed as much as possible in the liquid without touching the surface of the tube.</LI><LI> Sonicate with 50% amplitude and pulsing 10 s:10 s (energy : pause) until you reach 400 J (around 1 min and 24 s).</LI></UL><P class="protocol-subsection"> Separation</P><UL class="step"><LI> Let the small centrifuge cool down to 4ºC (takes around 20 min). </LI><LI> Centrifuge the lysed bacteria at 12000 rpm and 4°C for 10 min. </LI><LI> Transfer the supernatant (top layer) to a new tube. To prevent any transfer of bacteria debris, do not take all the lysate.</LI><LI> Place the lysate to an incubator at 37 °C and 200 rpm for 1h30 min, this run off reaction will lead to degradation of the rest of the DNA.</LI><LI> Centrifuge the run off reaction at 12000 rpm at 4°C for 10 min.</LI><LI> Transfer the supernatant to a new tube. As before, to prevent any transfer of bacteria debris, do not take all the lysate.</LI><LI> Aliquot the lysate to 0.5 ml microfuge tubes according to your needs (around 25 μl is recommended) and keep 1 μl of the lysate for the Bradford assay.</LI><LI> Flash freeze the aliquots in liquid nitrogen and store at -80°C (wear glasses). </LI></UL><P class="protocol-subsection"> Bradford assay </P><UL class="step"><LI> Perform the Bradford assay to determine the protein concentration: dilute 1 μl of the lysate in 99 μl of buffer A </LI><LI> Mix 5 μl of the diluted lysate with 250 μl of the Bradford reagent. </LI><LI> Let the reaction incubate for 5 min </LI><LI> Measure the absorbance using the Nanodrop device, protein concentration should be between 400 and 800 μg/ml </LI></UL><P class="protocol-citation"><A href="https://www.nature.com/articles/srep08663"> Kwon, Y. C., &amp; Jewett, M. C. (2015). High-throughput preparation methods of crude extract for robust cell-free protein synthesis. Scientific reports, 5.</A></P></DIV><DIV class="protocol"><P class="protocol-title"> MITOMI chips fabrication protocol</P><P class="protocol-section">Materials </P><UL class="mat"><LI>Centrifuge</LI><LI>Oven</LI><LI>Scotch Magic Tape</LI><LI>PDMS resin</LI><LI>Trimethylchlorosilane</LI><LI>Thinky Mixer ARE-250 w/ adaptor for 100 ml disposable PP beakers</LI><LI>Programmable spin coater G3P-8</LI><LI>Vacuum desiccator </LI><LI>Stereomicroscope, SMZ1500 / SZX2-TBO</LI><LI>Manual hole punching machine and pin vises, 21 gauge (0.04’’ OD)</LI></UL><P class="protocol-section">Procedure</P><UL class="step"><LI>The control layer mold is placed in a glass Petri dish lined with aluminum foil to facilitate easy removal. Care must be taken that the aluminum foil lining does not contain any holes.</LI><LI>Make sure to press mold down into foil.</LI><LI>To generate a hydrophobic surface, both flow and control mold are exposed to vapor deposits of TMCS for 30 minutes by placing them into a sealable plastic container with 1 ml TMCS filled into a plastic cap. TMCS treatment is repeated for 10 minutes each time prior to PDMS chip fabrication. </LI><LI>For the control layer, 60 g of a 5:1 Sylgard mixture (50 g Part A:10 g Part B) is prepared, mixed for 1 minute at 2,000 rpm (~400 × g) and degassed for 2 minutes at 2,200 rpm (~440 × g) in a centrifugal mixer.</LI><LI>Prepare in a plastic cup, cover in parafilm, and launch centrifugal mixer (program is preset).</LI><LI>The mixture is poured onto the control layer mold and degassed in a vacuum desiccator for 10 min.</LI><LI>Pour as close to the mold as possible to avoid air bubbles.</LI><LI>For the flow layer, 21 g of a 20:1 Sylgard mixture (20 g Part A: 1 g Part B) is prepared, mixed for 1 minute at 2,000 rpm (~400 × g) and degassed for 2 minutes at 2,200 rpm (~440 × g) in a centrifugal mixer. </LI><LI>The mixture is spin coated onto the flow layer with:
			</LI><LI>15 seconds ramp, 2800 RPM, 30 seconds dwell</LI><LI>20 seconds ramp, 100 rpm, 1 second dwell</LI><LI>0.1 second ramp, 100 rpm, 0 dwell</LI></UL><LI>After removing the control layer mold from the vacuum chamber any residual surface bubbles are destroyed by blowing on top of the PDMS layer. Any visible particles on top of the control channel grid are carefully removed by blowing on them with a pasteur pipette. </LI><LI>Both layers are cured in an oven for 20-30 minutes at 80°C. </LI><LI>Following polymerization, both molds are taken from the oven and allowed to cool for 5 minutes. </LI><LI>The control layer is then diced with a scalpel and holes (1–8 and B, S, C, O) are punched at the control input side using a hole puncher or a 21 gauge lure stub. </LI><LI>The channel side of the control layer is thoroughly cleaned with Scotch Magic Tape. </LI><LI>The cleaned control layer is then aligned to the flow layer on the stereomicroscope. </LI><LI>The device is bonded for 90 minutes at 80°C in an oven. </LI><LI>Bonded devices are removed from the oven and allowed to cool for 5 minutes. </LI><LI>Following the outline of the control layer each individual device is cut with a scalpel and peeled off the flow layer. </LI><LI>Holes are punched for the sample inlet and outlet (S1–S7 and O) using a hole puncher, and the chip is cut along the alignment crosses.</LI><LI>The flow channel side is cleaned thoroughly with tape before aligning the device to a spotted glass slide.</LI><LI>Flow mold is cleaned of any residual polymerized PDMS either by peeling off the thin layer of PDMS using a pair of tweezers or by an additional PDMS layer. For the latter, 11 g of a 10:1 Sylgard mixture (10 g Part A:1 g Part B) is mixed for 1 minute at 2,000 rpm (~400 × g), degassed for 2 minute at 2,200 rpm (~440 × g), poured on the flow mold cured in the oven for 30 minute at 80°C, and peeled off after cooling down to room temperature. The control mold is cleaned with a nitrogen air gun of any PDMS debris.</LI><P class="protocol-citation"><A href="https://www.ncbi.nlm.nih.gov/pubmed/21938622">MITOMI : A Microfluidic Platform for In Vitro Characterization of Transcription Factor-DNA Interaction, Rockel Sylve, Geertz Marcel, Maerkl Sebastian J., 2009, Springer.</A></P></DIV><DIV class="protocol"><P class="protocol-title">PAGE Urea gel</P><P class="protocol-section">Materials </P><UL class="mat"><LI>Urea (7M)</LI><LI>40% Acryl/Bisacryl</LI><LI>10x TBE (Tris/Borate/EDTA) buffer</LI><LI>Water (tap)</LI><LI>TEMED (Tetramethylethylenediamine)</LI><LI>30%(w/v) APS (Ammonium Persulfate)</LI><LI>Gel Cast</LI><LI>Incubator</LI><LI>SYBER_SAFE box</LI><LI>Gel red DNA stain (10’000x)</LI><LI>UV transilluminator</LI><LI>Mini-PROTEAN System</LI></UL><P class="protocol-section">Procedure</P><P class="protocol-subsection"> Set up </P><UL class="step"><LI>Generally, we prepare more than one gel in case one leaks.</LI><LI>Make sure every component is washed and cleaned properly.</LI><LI>Beforehand, align the small glass and the bigger glass (the little one should face towards you) in the green clasps on an even surface.</LI><LI>Close the clasps to fixate the glasses and snap it into the big plastic stand (fits two gel next to each other).</LI><LI>Put the gel cast under the laminar hood.</LI></UL><P class="protocol-subsection"> Gel preparation </P><UL class="step"><LI>To prepare the gel add the following to a 50ml falcon tube (calculations for 2 gels):
			</LI><LI>6.72 g of 7M urea</LI><LI>5.3 ml of 40% Acryl/Bisacryl</LI><LI>1.4 ml of 10X TBE</LI><LI>1.4ml of water</LI><LI>5.6 μl of TEMED (will be added after microwave)</LI><LI>46.2 μl of 30%(w/v) APS</LI></UL><LI>For a total volume of 14 ml.</LI><LI>Add everything expect the TEMED and the 30% APS and heat it up in the microwave for about 10 seconds (the tube should be clear).</LI><B>TIP:</B> let the tube in the microwave for 3-4 secondes, take it out and shake it again for 3-4 secondes.
	<P class="protocol-subsection"> Under the laminar hood </P><UL class="step"><LI>Add the TEMED second to last and the 30% APS last as the reaction starts quickly after you add those components.</LI><LI>Gently let the gel mixture flow between the two thin glasses. Add the gel comb. Nothing should spill down.</LI><LI>Let it solidify for about 5 minutes.</LI><LI>Unclip the gels from the clasps and gently.</LI><LI>Wash the glasses with water to remove the urea (make sure not to move the glasses, make sure that no water enter).</LI><LI>Put the glasses into the cassette (movable green sides to fixate the glasses securely).</LI><B>N.B:</B> If you are only running one gel, you should add a little plastic space holder so the space inside the cassette is safely closed.
		<LI>Fill the inside of the cassette with 1X TBE Buffer. Nothing should leak.</LI><LI>Put the cassette into the big plastic container used to run the gel and fill this with 1X TBE buffer until the 2 gels mark.</LI><LI>Wash the wells with the 1xTBE buffer already in the gel cassette.</LI><LI>Let the gel pre run in the incubator for 1 to 2 hours at 300V. (Temperature of the gel should be between 40 and 60°C).</LI></UL><P class="protocol-subsection"> Sample preparation </P><UL class="step"><LI>In a 200 μl PCR tube, put 2 μl of the 2X RNA loading dye and 2 μl of the desired sample or ladder.</LI><LI>It helps to put some dye on the side of the gel (first and last well) to have the samples in between the dye as it protects the RNA.</LI><LI>Let the reaction run for 28 minutes at 300V.</LI><LI>Put 100 ml of 1x TBE buffer to the SYBER_SAFE box and add 10 μl of the gel red stain.</LI><LI>Wash the gel. Slowly take the glasses from the sides (be careful, it can break easily), remove the gel and place it in a plastic container. </LI><LI>Add 1x TBE buffer to wash it (shake for 2 minutes).</LI><LI>Rewash and reshake for another two minutes.</LI><LI>Place the gel inside the SYBER_SAFE box and let it shake for 30 minutes.</LI></UL><P class="protocol-citation"><A href="https://www.neb.com/products/e2040-hiscribe-t7-high-yield-rna-synthesis-kit#Product%20Information">HiScribe™ T7 High Yield RNA Synthesis Kit (2017), New England Biolabs.</A></P></DIV><DIV class="protocol"><P class="protocol-title"> PCR</P><P class="protocol-section">Materials </P><UL class="mat"><LI>0.5ml microcentrifuge tubes</LI><LI>Thermocycler</LI><LI>DNA</LI><LI>Buffer HF</LI><LI>dNTP</LI><LI>DMSO</LI><LI>Forward primer</LI><LI>Backward primer</LI><LI>Phusion polymerase</LI></UL><P class="protocol-section">References to other protocols</P><UL class="mat"><LI><A href="#agrose-gel">Agarose gel electrophoresis protocol</A></LI></UL><P class="protocol-section">Procedure</P><UL class="step"><LI>Set up the program on the thermocycler:
			</LI><LI>98°C for 2 minutes for initial denaturation</LI><LI>30 cycles:
					</LI><LI>98°C for 30 seconds for denaturation</LI><LI>30 seconds for primers annealing at temperature depending on primers and polymerase</LI><LI>72°C for 30 seconds per kb of DNA to amplify for elongation</LI></UL><LI>72°C for 7 minutes for final extension</LI><LI>4°C on hold</LI><LI>Prepare a mastermix. For two reactions of 50μl, mix:
			</LI><LI>20 μl of 5X HF Buffer</LI><LI>2 μl of 10 mM dNTPs</LI><LI>5 μl of 100% DMSO</LI><LI>0.5 μl of 100 μM forward primer</LI><LI>0.5  μl of 100 μM backward primer</LI><LI>1 ng of DNA</LI>
		And complete this reaction with RNase free water until 100 μl.
	  <LI>Aliquote mastermix in the two tubes and add 0.375 μl of 2000 U/ml phusion polymerase.</LI><LI>Load the tubes in the thermocycler and start the program.</LI><LI>Control the PCR by doing an agarose gel electrophoresis (follow <A href="#agrose-gel">Agarose gel electrophoresis protocol</A>).</LI></DIV><DIV class="protocol"><P class="protocol-title">Plasmid extraction (Miniprep)</P><P class="protocol-section">Materials</P><UL class="mat"><LI>Bacterial culture transformed with plasmid to extract</LI><LI>Centrifuge</LI><LI>1.5 ml microcentrifuge tube</LI><LI>QIAprep Spin Miniprep Kit</LI><LI>Buffer P1</LI><LI>Buffer P2</LI><LI>Buffer N3</LI><LI>Buffer PE</LI><LI>Buffer EB</LI><LI>QIAprep spin column</LI><LI>Collection tube</LI></UL><P class="protocol-section">References to other protocols</P><UL class="mat"><LI><A href="#agrose-gel">Agarose gel electrophoresis protocol</A></LI></UL><P class="protocol-section">Procedure</P><UL class="step"><LI>Work sterile until bacteria are not lysed.</LI><LI>Pellet 1-5 ml of the bacterial culture by centrifugation at 8000 rpm for 3 minutes at room temperature.</LI><LI>Resuspend bacteria by adding 250 μl of Buffer P1 and transfer it in a new microcentrifuge tube.</LI><LI>Add 250μl of Buffer P2 and mix by inverting the tube 4-6 times and let incubate for max 5 minutes to lyse bacterial cells.</LI><LI>Add 350 μl of Buffer N3 and mix by inverting the tube 4-6 times.</LI><LI>Centrifuge for 10 minutes at 13’000 rpm.</LI><LI>Transfer 800 μl of supernatant from previous step to the QIAprep 2.0 spin column</LI><LI>Centrifuge the tube for 60 seconds and discard the flow-through.</LI><LI>Add 0.75 ml of Buffer PE to the column for washing.</LI><LI>Centrifuge for 60 seconds and discard the flow-through.</LI><LI>Centrifuge 60 seconds again to remove residual buffer.</LI><LI>Place the column to a clean 1.5 ml microcentrifuge tube.</LI><LI>Elute plasmid DNA by adding 50 μl of Buffer EB to the center of the column and let incubate for 1 minute.</LI><LI>Centrifuge the column for 1 minute.</LI><LI>Control plasmid extraction with agarose gel electrophoresis (follow <A href="#agrose-gel">Agarose gel electrophoresis protocol</A>.)</LI><LI>Measure plasmid DNA concentration using a Nanodrop spectrophotometer.</LI></UL><P class="protocol-citation"><A href="https://www.qiagen.com/us/resources/resourcedetail?id=331740ca-077f-4ddd-9e5a-2083f98eebd5&amp;lang=en">Quick-Start Protocol, QIAprep® Spin Miniprep Kit, cat. No. 27104 and 27106, Qiagen 2015.</A></P></DIV><DIV class="protocol"><P class="protocol-title">Platereader lysate reaction</P><P class="protocol-section">Materials</P><UL class="mat"><LI>Platereader</LI><LI>Lysate</LI><LI>Energy solution</LI><LI>Buffer A</LI><LI>gamS protein</LI><LI>DNA</LI><LI>Wax</LI><LI>384 Nunc plate with film</LI><LI>Centrifuge</LI></UL><P class="protocol-section">References to other protocols</P><UL class="mat"><LI><A href="#lysates">Lysate protocol</A></LI><LI><A href="#ES">Energy solution protocol</A></LI><LI><A href="#buffer-a">Buffer A protocol</A></LI></UL><P class="protocol-section">Procedure</P><UL class="step">
			For 10 μl lysate reaction with linear DNA templates:
		<LI>Configure the platereader program and start preheating the machine to 29°C. Usually, the platereader measure all the wells for 6 hours. </LI><LI>Prepare mastermix. For one reaction:
			</LI><LI>2.5 μl energy solution</LI><LI>2.5 μl lysate</LI><LI>2.5 μl buffer A</LI><LI>0.3 μl gamS</LI><LI>1.7 μl $H_2O$</LI></UL><LI>Prepare a mixture of everything minus the DNA — if 10 reactions will be running, prepare a mixture that would give enough for 10.5 reactions to account for some pipetting errors (if different volumes/concentrations of DNA are added to each reaction then prepare a mixture of everything minus the DNA and the water).</LI><LI>Notes:
			</LI><LI>For different final concentrations of DNA, calculate new volume to be added and compensate by adding more or less $H_2O$ to make the final reaction volume equal to 10 μl</LI><LI>For reactions with plasmid DNA rather than linear templates, exclude the gamS and compensate with $H_2O$</LI><LI>In order to have cheaper experiment, instead of using gamS protein, use Top10-gamS cells lysate (Top10 cells transformed with gamS gene)</LI><LI>Aliquot 9.5 μl of mastermix in wells of the plate (Nunc 384) in one corner of the wells.</LI><LI>Add 0.5 μl of 100 nM DNA template (final concentration of DNA = 5 nM) to the other corner of the well.</LI><LI>Centrifuge the plate for a short time (30-60 seconds) to remove any bubbles and collect the solutions at the bottom of the wells.</LI><LI>Add 35 μl of wax to each well.</LI><LI>Centrifuge again.</LI><LI>Load the plate and start the platereader program.</LI></DIV><DIV class="protocol"><P class="protocol-title">Purification of DNA</P><P class="protocol-section">Materials </P><UL class="mat"><LI> DNA fragments (usually 50-100 μl of PCR reaction)</LI><LI> Centrifuge</LI><LI> Microcentrifuge tube</LI><LI> QIAquick® PCR purification kit (Qiagen) </LI><LI> QIAquick spin column</LI><LI> Collection tube</LI><LI> Buffer PB</LI><LI> Buffer PE with ethanol (96-100%)</LI><LI> Buffer EB</LI><LI> Nanodrop spectrophotometer</LI></UL><P class="protocol-subsection"> References to other protocols</P><UL class="mat"><LI><A href="#agrose-gel">Agarose gel electrophoresis protocol</A></LI></UL><P class="protocol-section">Procedure</P><UL class="step"><LI> Add 5 volumes of Buffer PB to 1 volume of the PCR reaction and mix. If pH indicator is added to the Buffer PB, the solution should be yellow. If the solution is orange or violet, add 10 μl 3 M sodium acetate pH 5 and mix. The colour will turn yellow.</LI><LI> Place QIAquick spin column in a 2 ml collection tube.</LI><LI> Apply the sample to the QIAquick spin column (DNA will bind to the column) and centrifuge for 60 seconds at 13’000 rpm.</LI><LI> Discard the flow-through and put the column back in the collection tube. </LI><LI> Wash the column by adding 750 μl Buffer EP and centrifuge for 60 sec. </LI><LI> Discard the flow through and put the column back in the collection tube. </LI><LI> Centrifuge the column for 1 more minute to remove all wash Buffer. </LI><LI> Place the column in a clean 1.5 ml centrifuge tube. </LI><LI> Add 50μl Buffer EB to the centre of the membrane of the column and centrifuge for 1 minute in order to elute DNA (for increased concentration, add 30 μl and let incubate for 1 minute before centrifugation). </LI><LI> Run a gel to check the purification (follow <A href="#agrose-gel">Agarose gel electrophoresis protocol).</A></LI><LI> Measure the concentration of DNA in the sample using a Nanodrop spectrophotometer.</LI></UL><P class="protocol-citation"><A href="https://www.qiagen.com/us/resources/resourcedetail?id=390a728a-e6fc-43f7-bf59-b12091cc4380&amp;lang=en">Quick-start protocol, QIAquick® PCR purification kit, Qiagen, cat. No 28104 and 28106, 2011.</A></P></DIV><DIV class="protocol"><P class="protocol-title">Signal Amplifcation</P><P class="protocol-section">Materials</P><UL class="mat"><LI>Nuclease-free water</LI><LI>LacZalpha 27B toehold</LI><LI>T7 aptamer trigger 27B</LI><LI>Aptamer trigger 27B (no T7)</LI><LI>M15 T7 lysate</LI><LI>Top10-GamS lysate</LI><LI>Buffer A</LI><LI>Energy solution</LI><LI>Substrate (15 mg/mL)</LI><LI>Plate reader</LI><LI>Phusion polymerase</LI><LI>NTP Buffer Mix</LI><LI>T7 RNA Polymerase Mix</LI><LI>Primer forwardb</LI></UL><P class="protocol-subsection">Procedure</P><UL class="step"><LI>Incubate for 10 minutes at the annealing temperature of 70°C</LI><UL><LI>0.7 uL of ss T7 aptamer trigger at 100uM</LI><LI>10 uL of primer forwardb at 100uM</LI><LI>9.3 uL of nuclease-free water</LI><LI>Control Tube : Add no primer, instead water</LI></UL><LI>Put directly on ice for 10 minutes</LI><LI>Add T7 RNA polymerase and NTP Buffer Mix according to this list :</LI><UL><LI>10 uL ds T7 aptamer mix</LI><LI>10 uL NTP Buffer mix</LI><LI>2 uL T7 RNA polymerase mix</LI></UL><LI>Incubate for at least two hours at 37°C</LI><LI>After incubation : Measure RNA concentration for future reference</LI><LI>Run the amplification product on a 2% agarose gel together with controls for qualitative measure of the amplification</LI><LI>Compare efficiency of amplified trigger to non-amplified trigger quantitatively on a plate reader</LI></UL></DIV><DIV class="protocol"><P class="protocol-title">Streptavidin bead aptamer pull-down assay</P><P class="protocol-section">Materials</P><UL class="mat"><LI>Dynabeads™ MyOne™ Streptavidin T1</LI><LI>Biotinylated target protein</LI><LI>Cy5 labeled aptamer 1 </LI><LI>Cy3 labeled aptamer 2</LI><LI>DynaMag™ Magnet</LI><LI>Sample mixer allowing tilting and rotation of tubes</LI><LI>Binding and Washing (B&amp;W) Buffer 2X:
                </LI><LI>10 mM Tris-HCl (pH 7.5)</LI><LI>1 mM EDTA</LI><LI>2 M NaCl</LI><LI>PBS</LI><LI>PBS/0.01%BSA</LI><LI>PBS/0.01%Tween20</LI></UL><LI>Aptamer Binding Buffer</LI><LI>D-Biotin</LI><LI>Platereader</LI><P class="protocol-subsection">Aptamer Binding Buffer preparation</P><UL class="mat">
		Aptamer Binding Buffer consists of:
		<LI>50 mM Tris-HCl (pH 7.5)</LI><LI>100 mM NaCl</LI><LI>1 mM MgCl2</LI></UL><P class="protocol-section">Procedure</P><UL> Each time a washing buffer (B&amp;W or PBS) is added, let sit for 2-3 min before discarding supernatant. </UL><P class="protocol-subsection"> Wash Dynabeads™ MyOne™ magnetic beads</P><UL class="step"><LI>Resuspend Dynabeads in vial and vortex for 30 seconds minimum.</LI><LI>Transfer 50 μl of Dynabeads to a 1.5 ml microcentrifuge tube.</LI><LI>Add 50 μl of Washing Buffer and resuspend.</LI><LI>Place the tube on a magnet for 1 min and discard the supernatant.</LI><LI>Remove tube from the magnet and resuspend the washed magnetic beads in 100 μl of Washing Buffer. </LI><LI>Repeat steps 4-5 twice, for a total of 3 washes.</LI></UL><P class="protocol-subsection"> Immobilize thrombin </P><UL class="step"><LI>Resuspend washed Dynabeads™ magnetic beads in 100 μl of 2X B&amp;W Buffer.</LI><LI>Discard supernatant.</LI><LI>Add 100 μl of Aptamer Binding Buffer containing 100 nM of biotinylated target protein. Optimal binding occurs when NaCl concentration is reduced from 2 M to 1 M.</LI><LI>Incubated for 20 min (or 15 min for DNA &gt; 30 bp) at room temperature using gentle rotation.</LI><LI>Separate biotinylated protein coated beads with a magnet and discard the supernatant.</LI><LI>Wash the coated beads once with 100 μl of 2X B&amp;W Buffer.</LI></UL><P class="protocol-subsection"> Flow aptamers </P><UL class="step"><LI> Discard supernatant. </LI><LI>Add 100 μL of 2 μM Cy5 labeled aptamer 1.</LI><LI>Incubate for 20 min at room temperature using gentle rotation.</LI><LI>Wash once with 100 μl of PBS.</LI><LI>Wash once with 100 μl B&amp;W buffer.</LI><LI>Add 100 μl of 2 μM Cy3 labeled aptamer</LI><LI>Incubate for 20 min at room temperature using gentle rotation.</LI><LI>Wash once with 100 μl of PBS.</LI><LI>Wash once with 100 μl B&amp;W buffer.</LI><LI>Resuspend beads.</LI><LI>Analyze fluorescence using plate-reader.</LI></UL><P class="protocol-citation"><A href="https://www.ncbi.nlm.nih.gov/pubmed/17297945">Centi, S., Tombelli, S., Minunni, M., &amp; Mascini, M. (2007). Aptamer-based detection of plasma proteins by an electrochemical assay coupled to magnetic beads. Analytical chemistry, 79(4), 1466-1473.</A></P></DIV><DIV class="protocol"><P class="protocol-title"> Transformation into DH5α/M15 cells</P><P class="protocol-section">Materials </P><UL class="mat"><LI> DNA plasmid</LI><LI> Incubator</LI><LI> LB agar selective plates</LI><LI> DH5α competent cells</LI></UL><P class="protocol-subsection">Notes</P><UL class="mat"><LI> Do not mix by pipetting up and down or by vortex, cells are sensitive.</LI><LI> After taking out cells from -80°C fridge, thaw the cells and immediately do the transformation.</LI></UL><P class="protocol-section">Procedure</P><UL class="step"><LI> Thaw the tube of competent cells and 1.5ml microcentrifuge tubes on ice (takes around 2-5 minutes).</LI><LI> Gently mix the competent cells.</LI><LI> Take 50 μl of cells for each transformation into a 1.5 ml tube (put back unused cells to -80°C fridge).</LI><LI> Add 5 μl of DNA to the tubes and mix gently.</LI><LI> Pre-warm incubator at 42°C. </LI><LI> Incubate tubes on ice for 30 minutes. </LI><LI> Heat shock cells for 20 seconds at 42°C. </LI><LI> Place tubes back on ice for 2 minutes. </LI><LI> Add 950μl of pre-warmed LB or SOC medium to each tube. </LI><LI> Incubate tubes at 37°C for 1 hour at 225 rpm. </LI><LI> Spread 20 to 200 μl from each transformation on pre-warmed LB agar selective plates (antibiotic resistance from plasmid). </LI><LI> Store the remaining transformation reaction in +4°C fridge for another plating if necessary (not longer than 24 hours). </LI><LI> Incubate plates overnight at 37°C. </LI></UL><P class="protocol-citation"><A href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/subcloningefficiencydh5alpha_man.pdf"> Subcloning EfficiencyTM DH5αTM Competent Cells, Invitrogen, Cat. No. 18265-017, 2006 .</A></P></DIV><DIV class="protocol" id="YTPG"><P class="protocol-title"> YTPG medium protocol</P><P class="protocol-section"> Materials </P><UL class="mat"><LI> Autoclaved flasks </LI><LI>Tryptone</LI><LI>Yeast extract</LI><LI>Sodium chloride</LI><LI>Potassium phosphate dibasic</LI><LI>Potassium phosphate monobasic</LI><LI>Deionized water</LI><LI>Glucose </LI></UL><P class="protocol-section"> Procedure</P><UL class="step">
      For 1 L of LB medium
    <LI> Prepare in autoclaved flasks. </LI><LI> For 200 ml of 2X YTP medium, mix:
      </LI><LI>3.2 g of tryptone</LI><LI>2.0 g of yeast extract</LI><LI>1 g of sodium chloride</LI><LI>1.4 g of potassium phosphate dibasic</LI><LI>0.6 g of potassium phosphate monobasic</LI></UL><LI> Adjust volume to 150 ml by adding deionized water. </LI><LI> For glucose solution, add 3.6 g of glucose and adjust volume to 50 ml by adding deionized water. </LI><LI> Autoclave the two solutions separately.</LI><LI> Sterilely add the glucose solution to 2X YTP solution after autoclaving.</LI></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>