Team:SDU-CHINA/Notebook
PROJECT ▾
Description
Design
Results
Demonstrate
WETLAB ▾
Protocols
Notebook
Safety
DRYLAB ▾
Model
Hardware
TEAM ▾
Members
Attributoins
Collaborations
INTERLAB
PARTS ▾
Parts
Improve
HUMAN PRACTICES ▾
Silver
Gold & Integrated
Engagement
Notebook
week1
week2
week3
week4
week5
week6
week7
week8
week9
week10
week11
week12
week13
week14
week15
week16
week17
week18
week19
week20
week21
week22
week23
Week 1 (April 25th to April 29th) ▲ back
Plasmid backbones pSR43.6 and pSR58.6 were assembled by PCR and Gibson Assembly and the DNA fragments cloned from original plasmid pJ101700 and paracr15A was examined by electrophoresis. To test whether the backbones were assembled correctly, colony PCR and electrophoresis were required and then the backbones were send to sequencing.
During electrophoresis of the PCR product, we discovered that the maker was not seen clearly.
We made Gibson Assemble: Protocol 3
Assembly master mixture
1.25 ml
5×isothermal reaction buffer
500μl
T5 exonuclease (10U/μl)
1μl
DNA polymerase (2U/μl)
31.25μl
Taq DNA ligase (40U/μl)
ddH2O
DNA fragment
Equal volume
All plasmid backbones of pSR58.6 showed in correct length except pSR58.6 NO.10.
Week 2 (May 2nd to May 8th) ▲ back
We derived our plasmid backbones pSR43.6 and pSR58.6 and then sent them for synthesising the final plasmid. To test the influence of blocking gltA to cell growth, we decided to construct a new plasmid p58.6gRNA containing Para and gRNA for type I-e CRISPRi and gltA.
DNA fragments were copied from pSR43.6 and pcrRNA-bssl-7(provided by our lab), the PCR products are shown below.
New plasmid was assembled by Gibson Assembly, and transformed into E. coli DH5α, then colony PCR and electrophoresis were applied and we sent the prodcuts to sequencing.
The result of electrophoresis showed only pSR58.6gRNA-8 was correct.
To assemble pcrRNA.bbsI-7-phac, we cloned DNA fragments from pcrRNA.bbsI-7 and pBHR68 by PCR, and the two fragments were assembled by Gibson assembly. But we failed in PCR, and it took us some time to repeat PCR.
Considering the failures in assembling pPT7-sfGFP, we cloned four DNA fragments (sfGFP、CmR、Ori、araR) by PCR and tried to assemble them by Gibson assembly at a time. Then this new plasmid was transformed into E. coli DH5α.
Week 3 (May 9th to May 15th) ▲ back
pSR58.6gRNA was transformed into E. coli EE-E15, in which Cas3 has been knocked out, provided by our lab.
(EEgRNA-5 was not correct)
After testing the transformation, growth curve was made to figure out the impact of crRNA along with CRISPRi.
(BC: blank control; c: arabinose without pSR58.6gRNA; a: arabinose and pSR58.6gRNA; concentration of arabinose: 0.2g/L; 0.4g/L; 0.6g/L)
(BC: blank control; c: arabinose without pSR58.6gRNA; a: arabinose and pSR58.6gRNA; concentration of arabinose: 0.02g/L; 0.2g/L; 2g/L)
As for pcrRNA.bbsI-7-phac, the PCR products were still incorrect tested by electrophoresis. So we tried several different PCR systems and redesigned primers.
pPT7-sfGFP was not correct by Gibson, we tried restriction sites and restriction enzymes instead, the new plasmid was transformed into E. coli DH5α and tested by colony PCR and sequencing.
Week 4 (May 16th to May 23rd) ▲ back
We cultured bacteria transformed with pSR58.6gRNA in M9 (commonly used for industrial fermentation), and repeated measuring growth curve following the same protocol as week 3.
We discovered the impact of type I-e CRISPRi to bacteria growth is relatively weak.
pcrRNA.bbsI-7-phac was assembled by both Gibson assembly and restriction enzymes, and it was successfully assembled by Gibson assembly according to sequencing results.
Week 5 (May 24th to May 31st) ▲ back
PT7 was inserted into pcrRNA.bbsI-7-phac through PCR and Gibson assembly, but it was failed. So we tried to assemble the plasmid by restriction enzymes.
Week 6 (June 4th to June 10th) ▲ back
We were appreciated to receive pSR43.6r and pHZ from HZAU-China2017 and they were transformed into E. coli DH5α. pSR43.6r was derived and the ccaS component was improved by deleting about eight hundred base pairs according to paper [1]. But we were stopped by PCR.
pHZphbc, which contains phbCAB gene cluster, was assembled by PCR and Gibson assembly.
phbCAB gene cluster was inserted into pcrRNA.bbsI-7 by both restriction enzymes and Gibson assembly. The sequencing result showed that this new plasmid was assembled successfully by Gibson.
[1]. Nakajima, M., et al., Construction of a Miniaturized Chromatic Acclimation Sensor from Cyanobacteria with Reversed Response to a Light Signal. Scientific Reports, 2016. 6(1).
Week 7 (June 11th to June 17th) ▲ back
pHZphbc was failed in assebling, so we repeated cloning and redesigned the plasmid at the same time. Tested by colony PCR and sequencing, pSR43.6-ccaS#10 was successfully assembled. To insert crRNA into pSR43.6r, we needed to construct an intermediate plasmid called pHZZJ.
pPT7-crRNA was assembled successfully by restriction enzymes after editing PT7 for a long time.
Week 8 (June 18th to June 24th) ▲ back
pHZphbc and pHZZJ were assembled successfully. To construct pSR43.6-ccaS#crRNA, DNA fragments were copied from pSR43.6-ccaS#10 and pHZZJ and then they were assembled through Gibson assembly.
pcrRNA.bbsI-7-opto-phbc was transformed into E. coli DE3, we wanted to test the production of PHB by Nile Red. We did this experiment applying Protocol 15 . Here is the result.
pPT7-sfGFP was assembled successfully by restriction enzymes.
Week 9-11 (June 25th to July 14th) ▲ back
It was time for final examination!
Week 12 (July 15th to July 22th) ▲ back
We were blocked by assembling pSR43.6-ccaS#10-crRNA, we tried both Gibson assembly and restriction enzyme, and we redesigned the plasmid and primers for many times. We had to figure out what was going on.
At the same time, pHZccdA which contains antitoxic protein ccdA but without promoter was assembled by PCR and Gibson assembly.
Week 13 (July 23rd to July 30th) ▲ back
pSR43.6-ccaS#10-crRNA was still under assembling, we suspected that the leakage of crRNA might prevent bacteria from growing, so the Gibson product was transformed into EE-E15, in which Cas3 protein has been knocked out. EE-E15 grew at last but the result of colony PCR was incorrect.
Since the result of Nile Red wasn’t desirable, we tried to change the terminator of phbCAB gene cluster through restriction enzymes, but we failed.
Week 14 (July 31st to August 5th) ▲ back
Promoters with different strength (J23101, J23105, J23108, J23113 and J23114) were inserted into pHZccdA through Gibson assembly. But only J23108 and J23113 were inserted successfully.
We tested the production of PHB with Nile Red again, according to Protocol 15
Result:
Week 15 (August 6th to August 12th) ▲ back
While assembling pHZJ23ccdA series, we characterized the improved ccaS-ccaR two-component system by measuring GFP fluorescence according to Protocol 11
Result:
ccdB and degradation tag was inserted into pcrRNA.bbsI-7 after promoter T7 by Gibson assembly and restriction enzymes.
Week 16 (August 13th to August 20th) ▲ back
We found that there were some problems in pSR43.6-ccaS#10 by characterizing pSR43,6r and pHZ, so we redesigned this plasmid as well as other pSR43.6-ccaS# series (#3,#4), we assembled pSR43.6-ccaS# series by PCR and Gibson assembly. pSR43.6-ccaS# series were assembled successfully.
Result for characterization:
The new plasmid pcrRNA.bbsI-7-ccdb was assembled and tested by colony PCR and sequencing.
The split T7 RNA polymerase was characterized under blue light according to Protocol 13
The result showed that the split T7 RNA polymerase did not work, we held a troubleshooting for that with our advisors and instructors, wanting to figure out the problems and solutions.
Week 17 (August 21st to August 28th) ▲ back
After finishing assembling pSR43.6-ccaS# series, they were transformed into E. coli TOP10 along with pHZ, and the ccaS-ccaR two-component system was characterized again according to Protocol 11 .
Finally pSR43.6-ccaS#-crRNA series were born through Gibson assembly.
we wondered if there was something wrong with PT7 and split RNA polymerase, so pET28a(DE3) was applied to test whether pT7 could work properly, and we did SDS-PAGE to detect the expression of proteins. But pT7 worked properly.
Week 18 (August 29th to September 7th) ▲ back
pHZJ23ccdAB series were assembled by PCR and Gibson assembly. pSR43.6-ccaS#-crRNA and pHZphbc were characterized by measuring the production of PHB according to Protocol 14 .
Blue light components were inserted into a high-copy plasmid backbone with high GFP expression, and this system was tested again under blue light according to Protocol 13 .
We couldn’t figure out the problem of this system and we decided to stop experiment on blue light component due to the limitation of time.
Week 19 (September 10th to September 16th) ▲ back
To make phbCAB gene cluster express constitutively, phbCAB gene cluster was inserted into pHZ3.1 through Gibson Assembly.
Ccas-ccaR system was characterized by measuring fluorescence, and production of PHB was measured according to Protocol 13 .
Inhibition of bacteria growth by type I-e CRISPRi was tested by measuring cell density, but the result was not good enough.
Week 20 (September 17th to September 23rd) ▲ back
We found two mutations in phbCAB gene cluster by sequencing, and we wondered it might be the cause of the low production of PHB. At the same time, plasmids that would be used as control were under assembling.
Week 21 (September 24th to October 1st) ▲ back
We were busy with standardization and part registry, at the same time, bacteria strain responsible for production of PHB induced by green light was under constructing. We were still struggling testing the effect caused by crRNA to bacteria growth. Characterization of Ccas-CcaR system was done in 24-well plate. At the end of this week, part BBa_K2627000 is successfully assembled and finished sequencing.
Week 22 (October 2nd to October 7th) ▲ back
We were happy with the National Day Holiday, but we still need to continue our experiment or we might leave the project unfinished. Bacteria strains which can synthesize PHB induced by light were selected through Nile Red according to https://static.igem.org/mediawiki/2018/d/d3/T--SDU-China--Nile_Red.pdf. Characterization of Ccas-CcaR system was done in 24-well plate, and characterization of crRNA was going not very well. But the good news was that at the end of this vocation, the rest of the parts we were willing to send were assembled and sequenced.
Week 23 (October 8th to October15th) ▲ back
Finally, we finished constructing the bacteria strain that could produce PHB under green light, and the new strain was tested by measuring the relative weight of PHB through Gas Chromatography and HPLC following the protoco l. Since the effect of type I-E CRISPRi was not ideal when induced by light, so we chose to test it with arabinose as its inducer.
mailto:sdubioigem@163.com
www.instagram.com/sdubiochina_2018/
