<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SDU-CHINA_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SDU-CHINA/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="body"><DIV id="header"><DIV id="menu"><UL><LI><A href="#">PROJECT ▾</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Demonstrate">Demonstrate</A></LI></UL><LI><A href="#">WETLAB ▾</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Experiments">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Safety">Safety</A></LI><LI><A href="#">DRYLAB ▾</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Hardware">Hardware</A></LI><LI><A href="#">TEAM ▾</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Attributions">Attributoins</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/InterLab">INTERLAB</A></LI><LI><A href="#">PARTS ▾</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Human_Practices">HUMAN PRACTICES ▾</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Human_Practices#Silver">Silver</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Human_Practices#integrated">Gold &amp; Integrated</A></LI><LI><A href="https://2018.igem.org/Team:SDU-CHINA/Public_Engagement">Engagement</A></LI></DIV></DIV><DIV class="conteudo"><DIV class="paragraphs"><H1 id="main-title">Design</H1><H4 id="main-title">Overview</H4><P>To solve the conflicts existing between engineered and endogenous pathways competing for metabolite precursors, this year our team introduced light control in <I>E. coli</I> to construct a metabolic flux regulation platform through inhibiting TCA cycle. Through this approach, we dynamically switched the bacteria from growth phase to production phase accumulating acetyl-CoA. Composed of light sensor (opto T7 RNA polymerase or CcaS/CcaR system) and toggle switch (type I-E CRISPRi system), cells respond different wavelength light signal, then translate crRNA targeting <I>gltA</I> gene to inhibit acetyl-CoA turn to TCA cycle, which make cell accumulate more substrate for production. Then we took polyhydroxybutyrate(PHB) as an example to test our switchable platform.</P><DIV style="text-align: center; font-size: 15px">Figure 1. Light-induced metabolic flux regulation.</DIV><DIV style="text-align: center; font-size: 15px">Figure 2. Gene circuits.</DIV><H4 id="main-title" style="line-height:0.7em">Light Sensor</H4><P>We first tried two kinds of photoreceptors: opto T7 RNA polymerase and CcaS/CcaR system as the sensors responding to different wavelength light.</P><H4>Opto T7 RNA polymerase</H4><P>The domains Magnet, derived from the homodimerizing photoreceptor VVD from the filamentous fungus <I>Neurospora crassa</I>, consist of pMag and nMag, which binds each other under blue light (460 nm)<SUP>[1]</SUP>. The T7 RNA polymerase from the T7 bacteriophage with high orthogonality is commonly used for overexpression heterologous protein. In our design, split T7 RNA polymerases are fused to this photoactivated dimerization fragments, and through light-induced protein-protein interactions the split T7 RNA polymerases dimerize and reconstitute its structure, then preforming its normal function. This light-inducible transcription system, with high spatiotemporal accuracy, also has low leakiness of gene expression in darkness and high expression strength when induced upon illumination. And with high spatiotemporal accuracy, this system can allow for precise dynamic control of gene expression<SUP>[2,3]</SUP>. We desire to use this orthogonal blue light-responsive T7 RNA polymerase in our project for precise dynamic control of gene expression.</P><DIV style="text-align: center; font-size: 15px">Figure 3. light-induced T7 RNA polymerases.</DIV><H4>CcaS/CcaR two component system</H4><P>Another optogenetics system applied in our project design is a green/red photoreversible two-component signal transduction system (TCS) with transcriptional outputs in <I>E. coli</I>. The two-component system consists of the membrane-associated histidine kinase CcaS and its response regulator CcaR<SUP>[4]</SUP>. The activating information stored in light is captured by phytochromes in situ. In phytochromes, a bilin-chromophore (in this case phycocyanobilin) binds at a conserved cysteine within an N-terminal GAF (cyclic GMP phosphodiesterase, adenylyl cyclase, FhlA) domain and imparts reversible photoactivation of signaling activity with maximal responses to 535 nm (green) and 672 nm (red) light. Absorption of green light increases the rate of CcaS autophosphorylation, phosphorylation of CcaR by phosphate transferring, and transcription from the promoter of the phycobilisome linker protein cpcG2, while absorption of red light reverses this process. The lower leakiness is reported to be acquired after removing the second putative promoter in cpcG2, which is thought to be constitutive and contributes to leakiness and low dynamic range.</P><P>What’s more, recent studies have showed that the leakiness is got lower while the dynamic range rises sharply by removing two PAS domains of unknown function within the CcaS sensor histidine kinase<SUP>[5]</SUP>. Inspired by their work, we improved the existing part and acquired #3、#4、#10 which were the variants of CcaS.</P><DIV style="text-align: center; font-size: 15px">Figure 4. CcaS/CcaR system.</DIV><H4 id="main-title" style="line-height:1.7em">Toggle Switch</H4><P>Clustered regularly interspaced short palindromic repeats (CRISPR) is a family of DNA sequences found within the genomes of prokaryotic organisms such as bacteria and archaea, which have the function in binding and cutting DNA<SUP>[6]</SUP>. There are three types of CRISPR system found in microorganisms to date: type I, type II and type III. In type I and type III systems the long precursor CRISPR RNA (pre-crRNA) is processed by CRISPR specific endoribonucleases into small CRISPR RNAs (crRNAs) that contain a repeat sequence flaked by portions of the adjacent CRISPR repeat sequence<SUP>[7-12]</SUP>. In contrast, the pre-crRNA in type II systems is processed by RNase III<SUP>[13]</SUP>.</P><P>Similar to the other two CRISPR systems, crRNA in the type I-E system sequences are recognized by ribonucleoprotein complex Cascade during target DNA binding<SUP>[14,15]</SUP>. The ribonucleoprotein complex Cascade is composed of a 61 nt crRNA, and five different Cas proteins in an uneven stoichiometry: Cse1<SUB>1</SUB>Cse2<SUB>2</SUB>Cas7<SUB>6</SUB>Cas5<SUB>1</SUB>Cas6e<SUB>1</SUB>, encoded by 8 genes separately (cas3, cse1, cse2, cas7, cas5, cas6e, cas1, cas2)<SUP>[16]</SUP>. The interference function of this system requires Cas3, a large protein with nuclease and helicase activities<SUP>[17,18]</SUP>. Therefore, by knocking out cas3 and activating the expression of CRISPR-associated complex for antiviral defence (Cascade), the CRISPR system is inactivated for its DNA-cutting function, with its DNA-binding function maintained, which can interference the expression of the target genes<SUP>[19]</SUP>.</P><DIV style="text-align: center; font-size: 15px">Figure 5. The principle of type I-E CRISPRi system.</DIV><P>In <I>Escherichia coli</I>, the type I-E CRISPRi system expressed endogenously shall be easy for internal regulation without causing metabolic burden in compared with the widely used type II system, which expressed dCas9 as an additional plasmid.</P><P>Thus we can use this CRISPRi system as a switch to regulate the gene of interest in <I>Escherichia coli</I>. In our project, the target gene is <I>gltA</I> which encodes citrate synthase. The process catalysed by citrate synthase is irreversible and rate-limiting step in the TCA cycle. Furthermore, its substrate - acetyl-CoA is one of the most significant intermediates used to produce important chemicals such as 1,3-isopropanol<SUP>[20]</SUP>, 1-butanol<SUP>[21]</SUP>. In principle, the type I-E CRISPRi system can redirect the metabolic flux from TCA cycle to target pathway via switching <I>gltA</I> off<SUP>[22]</SUP>.</P><DIV style="text-align: center; font-size: 15px">Figure 6. Using type I-E CRISPR system switching <I>gltA</I> off.</DIV><H4 id="main-title">Bioplastics, polyhydroxybutyrate (PHB)</H4><P>As we all know acetyl-CoA is a prominent precursor for the TCA cycle, which is regarded as the most efficient pathway of energy production for cell growth. However, acetyl-CoA is also required in many synthetic pathways, like biofuel 1-butanol<SUP>[23]</SUP>, solvent acetone<SUP>[24]</SUP> and bioplastic Polyhydroxybutyrate (PHB). PHB, a kind of biosynthesized polyester, is nowadays, one of the most promising bio-materials as the replacement of traditional petrochemical plastics for its outstanding biodegradable as well as other physical properties. So here, we intend to introduce our CcaS/CcaR system and type I CRISPR into PHB fermentation to produce PHB more efficiently as an example of our growth-producing switching system.</P><DIV style="text-align: center; font-size: 15px">Figure 7. The production of PHB<SUP>[25]</SUP>.</DIV><P>Naturally, the synthesis of PHB requires three enzymes, phbA, phbB and phbC, by which acetyl-CoA can ultimately be converted into PHB. But the conflict between cell growing demands and the usage of acetyl-CoA for PHB fermentation is obviously the obstacle for PHB production. So we hope that PHB production can be improved with our growth-producing switching system and accelerate the process of taking PHB plastics into practice. </P><H4 id="main-title">References</H4><P>[1] Balzer, Grant J., et al. &quot;Metabolic engineering of Escherichia coli to minimize byproduct formate and improving succinate productivity through increasing NADH availability by heterologous expression of NAD+-dependent formate dehydrogenase.&quot; Metabolic engineering 20 (2013): 1-8.
[2] Kawano, Fuun, et al. &quot;Engineered pairs of distinct photoswitches for optogenetic control of cellular proteins.&quot; Nature Communications 6(2015):6256.
[3] Baumschlager, A, S. K. Aoki, and M. Khammash. &quot;Dynamic Blue Light-Inducible T7 RNA Polymerases (Opto-T7RNAPs) for Precise Spatiotemporal Gene Expression Control.&quot; Acs Synthetic Biology 6.11(2017).
[4] Fernandez-Rodriguez, J, et al. &quot;Engineering RGB color vision into Escherichia coli.&quot; Nature Chemical Biology 13.7(2017):706-708.
[5] Nakajima, Mitsuharu, et al. &quot;Construction of a Miniaturized Chromatic Acclimation Sensor from Cyanobacteria with Reversed Response to a Light Signal.&quot; Scientific Reports 6(2016):37595.
[6] Barrangou, R. &quot;The roles of CRISPR-Cas systems in adaptive immunity and beyond. &quot; Current Opinion in Immunology 32(2015):36-41.
[7] Brouns, Stan J. J., et al. &quot;Small CRISPR RNAs Guide Antiviral Defense in Prokaryotes.&quot; Science 321.5891(2008):960-964.
[8] Haurwitz, Rachel E., et al. &quot;Sequence- and structure-specific RNA processing by a CRISPR endonuclease.&quot; Science 329.5997(2010):1355-1358.
[9] Carte, Jason, et al. &quot;Cas6 is an endoribonuclease that generates guide RNAs for invader defense in prokaryotes.&quot; Genes &amp; Development 22.24(2008):3489.
[10] Przybilski, R, et al. &quot;Csy4 is responsible for CRISPR RNA processing in Pectobacterium atrosepticum. &quot; Rna Biology 8.3(2011):517-528.
[11] Nam, Ki Hyun, et al. &quot;Cas5d Protein Processes Pre-crRNA and Assembles into a Cascade-like Interference Complex in Subtype I-C/Dvulg CRISPR-Cas System.&quot; Structure 20.9(2012):1574-1584.
[12] Garside, E. L., et al. &quot;Cas5d processes pre-crRNA and is a member of a larger family of CRISPR RNA endonucleases. &quot; Rna-a Publication of the Rna Society 18.11(2012):2020-8.
[13] Deltcheva, E, et al. &quot;CRISPR RNA maturation by trans-encoded small RNA and host factor RNase III.&quot; Nature 471.7340(2011):602-607.
[14] Westra, Edze R., et al. &quot;CRISPR immunity relies on the consecutive binding and degradation of negatively supercoiled invader DNA by Cascade and Cas3.&quot; Molecular Cell 46.5(2012):595-605.
[15] Semenova, Ekaterina, and K. Severinov. &quot;Interference by clustered regularly interspaced short palindromic repeat (CRISPR) RNA is governed by a seed sequence.&quot; Proceedings of the National Academy of Sciences of the United States of America 108.25(2011):10098-10103.
[16] Kunin, Victor, R. Sorek, and P. Hugenholtz. &quot;Evolutionary conservation of sequence and secondary structures in CRISPR repeats.&quot; Genome Biology 8.4(2007):R61.
[17] Westra, Edze R., et al. &quot;CRISPR immunity relies on the consecutive binding and degradation of negatively supercoiled invader DNA by Cascade and Cas3.&quot; Molecular Cell 46.5(2012):595-605.
[18] Sinkunas, Tomas, et al. &quot;Cas3 is a single‐stranded DNA nuclease and ATP‐dependent helicase in the CRISPR/Cas immune system.&quot; Embo Journal 30.7(2014):1335-1342.
[19] Chang, Yizhao, et al. &quot;Easy regulation of metabolic flux in Escherichia coli using an endogenous type I-E CRISPR-Cas system.&quot; Microbial Cell Factories 15.1(2016):195.
[20] Nakamura, Charles E, and G. M. Whited. &quot;Metabolic engineering for the microbial production of 1,3-propanediol.&quot; Current Opinion in Biotechnology 14.5(2003):454-459.
[21] Dellomonaco, Clementina, et al. &quot;Engineered reversal of the &amp;beta-oxidation cycle for the synthesis of fuels and chemicals.&quot; Nature 476.7360 (2011): 355.
[22] Soma, Y, et al. &quot;Metabolic flux redirection from a central metabolic pathway toward a synthetic pathway using a metabolic toggle switch.&quot; Metabolic Engineering 23.5(2014):175-184.
[23] Dellomonaco, Clementina, et al. &quot;Engineered reversal of the &amp;bata-oxidation cycle for the synthesis of fuels and chemicals.&quot; Nature 476.7360(2011):355-359.
[24] Bermejo, L. L., N. E. Welker, and E. T. Papoutsakis. &quot;Expression of Clostridium acetobutylicum ATCC 824 genes in Escherichia coli for acetone production and acetate detoxification. &quot; Appl Environ Microbiol 64.3(1998):1079-1085.
[25] Verlinden, R. A. J., et al. &quot;Bacterial synthesis of biodegradable polyhydroxyalkanoates.&quot; Journal of Applied Microbiology 102.6(2010):1437-1449.</P></DIV></DIV><DIV id="footer"><DIV id="contact"><DIV><A href="mailto:sdubioigem@163.com">mailto:sdubioigem@163.com</A></DIV><DIV><A href="https://www.instagram.com/sdubiochina_2018/">www.instagram.com/sdubiochina_2018/</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>