<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SCUT_Project_Odorant_sensing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SCUT/Project/Odorant_sensing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SCUT/Project/Odorant_sensing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SCUT/Project/Odorant_sensing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SCUT/Project/Odorant_sensing&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SCUT/Project/Odorant_sensing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SCUT/Project/Odorant sensing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="frame01"><A href="https://2013.igem.org/Team:SCUT" class="m1link" style="padding:5px 15px">Home</A><LI><A href="https://2013.igem.org/Team:SCUT/Team" class="m1link" style="padding:5px 15px;letter-spacing:1px">Team</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Team" class="m2link">Members</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Team/Our_University" class="m2link">Our University</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Team/Attributions" class="m2link">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Team/Notebook" class="m2link">Notebook</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=SCUT" class="m2link">Team Profile</A></LI><LI><A href="#" class="m2link">Gallery</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Project" class="m1link" style="padding:5px 14px">Project</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Project" class="m2link">Overview</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Project/Oscillating_odorant" class="m2link">Oscillating Odorant</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Project/Odorant_sensing" class="m2link">Odorant Sensing</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Project/Future_work" class="m2link">Future Work</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Project/Safety" class="m2link">Safety</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Project/Protocol" class="m2link">Protocol</A></LI><LI><A href="#" class="m1link" style="padding:5px 15px;letter-spacing:-1px;">Data Page</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Parts" class="m2link">Parts</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Data_page/Achievements" class="m2link">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Modeling" class="m1link" style="letter-spacing:-1px">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Modeling" class="m2link">Overview</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Modeling/Diacetyl_producer" class="m2link">Diacetyl Producer</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Modeling/Oscillation" class="m2link">Oscillation</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Modeling/Odr-10_Pathway" class="m2link">Odr-10 Pathway</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Human_Practice" class="m1link" style="padding:5px 14px">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Human_Practice" class="m2link">Overview</A></LI><LI><A href="https://2013.igem.org/Team:SCUT/Human_Practice/IGEM_everywhere" class="m2link">IGEM everywhere</A></LI></DIV></DIV><DIV id="textbody"><DIV id="navi"><DIV class="navimenu"><DIV class="navimenu_body"><DIV class="item" style="background-image:url('https://static.igem.org/mediawiki/igem.org/a/a3/Scut_navblock01.png');"><A href="https://2013.igem.org/Team:SCUT/Project" style="line-height:40px;">Overview</A></DIV><DIV class="item" style="background-image:url('https://static.igem.org/mediawiki/igem.org/b/b7/Scut_navblock02.png');"><A href="https://2013.igem.org/Team:SCUT/Project/Oscillating_odorant" style="font-size:19px;letter-spacing:-1px;">Oscillating odorant</A></DIV><DIV class="item" style="background-image:url('https://static.igem.org/mediawiki/igem.org/6/66/Scut_navblock03.png');"><A href="https://2013.igem.org/Team:SCUT/Project/Odorant_sensing" style="letter-spacing:-1px;">Odorant sensing</A></DIV><DIV class="item" style="background-image:url('https://static.igem.org/mediawiki/igem.org/5/51/Scut_navblock04.png');"><A href="https://2013.igem.org/Team:SCUT/Project/Safety" style="line-height:40px;">Safety</A></DIV><DIV class="item" style="background-image:url('https://static.igem.org/mediawiki/igem.org/e/e2/Scut_navblock05.png');"><A href="https://2013.igem.org/Team:SCUT/Project/Protocol" style="line-height:20px;letter-spacing:-1px;line-height:40px;">Protocol</A></DIV></DIV><DIV class="navimenu_head" style="background-image:url('https://static.igem.org/mediawiki/igem.org/4/45/Scut_navibottom.png');width:100px;height:50px;"><A href="https://2013.igem.org/Team:SCUT/Project">Project</A></DIV></DIV></DIV></DIV><DIV id="textbody"><DIV id="container"><DIV id="tblock1"><H1>Introduction</H1><P class="text"><SPAN class="first">W</SPAN>e aim to reconstruct and model the simplest communication in nature, like a flow of information between eukaryotes and prokaryotes through odorant. To accomplish this mission that <I>S.cerecisiae</I> communicate with <I>E.coli</I>, we require to fabricate a kind of dedicated biomimetic nose for our yeast and single out a special and adapted volatile to serve as their “language”.
  After a long discussion, argument and selection, finally we decided to select a complex and 7-transmembrane receptors, GPCR, Odr-10, as nose and specific odorant, diacetyl, as the language. In order to construct the system precisely and efficiently, we constructed the Odr-10 expression system and modified the downstream pathway, deleting Gpa1 and Far1 gene then reconstructing and expressing chimeric Gpa1.
  </P></DIV><DIV id="label"><DIV id="mark1" class="labelmenu" style="background-color:#d6ee5a;"><P onclick="scroll_1()" style="letter-spacing:-1px;">Introduce..</P></DIV><DIV id="mark2" class="labelmenu" style="background-color:#cbf4bb;"><P onclick="scroll_2()" style="padding:2px 0;">Pathway overview</P></DIV><DIV id="mark3" class="labelmenu" style="background-color:#efd9cb;"><P onclick="scroll_3()" style="letter-spacing:-1px;">Reconstruct</P></DIV><DIV id="mark4" class="labelmenu" style="background-color:#f979a4;"><P onclick="scroll_4()">Optimize</P></DIV><DIV id="mark5" class="labelmenu" style="background-color:#d3c902;"><P onclick="scroll_5()">Result</P></DIV><DIV id="mark6" class="labelmenu" style="background-color:#F5F978;"><P onclick="scroll_6()">Reference</P></DIV></DIV><DIV id="tblock2"><H1>Pathway Overview</H1><P class="text"><I>S.cerevisiae</I> is selected as the protagonist of our system owing to the clear genetic background and its natural advantages to express G protein family. The endogenous receptors, Step2, a G protein coupled receptors (GPCRs), will be activated when they sense a/αpheromone, as description of figure 1.
   In yeast, the pheromones are bonds to a GPCR, Sted2, and active downstream pathway. It is composed of Gpa1p (Gα), Ste4p (Gβ) and Ste18p (Gγ) subunits. The GPCR induces the complex of Gα and Gβγ, resulting in the separation of Gα and Gβγ. The Gβγ particle subsequently transmits the signal to a MAP kinase pathway, comprising Ste11p, Ste7p, and Fus3p held in a complex with the Ste5 protein. Fus3p then activates the cyclin-dependent kinase inhibitor Far1p and the transcription factor Ste12p to bring about cell-cycle arrest.
   It is worthy to announce that a major negative regulator of the pathway, Sst2p, is a GTPase activating factors that return Gα to its inactive state which generally would be deleted to increase the sensitivity. Finally, a reporter of this pathway was under the control of a Ste12p inducible promoter, Fus1 linking BFP, GFP or HIS3.
  </P><DIV class="imgC"><P class="tag"><STRONG>Figure 1.</STRONG> Comparison between endogenous GPCR pathway in yeast and motified Odr10 pathway.</P></DIV><P class="text">
   Diacetyl is utilized as our &quot;language&quot; for our communication between <I>E.coli</I> and <I>S.cerevisiae</I>. Diacetyl can be released by modified <I>E.coli</I> as a kind of special intermediate in metabolism pathway which was described in Oscillating odorant part. We take advantage of the modeling olfactory receptors (ORs) odr-10, which is from <I>caenorhabditis elegans</I> and specific to diacetyl. Additionally, diacetyl may be harmful to humans when inhaled frequently and diacetyl-induced bronchiolitis obliterans is one of the representative diseases. Consequently, our biomimetic nose can be utilized to detect the diacetyl in our surroundings more sensitively. 
  </P></DIV><DIV id="tblock3"><H1>Pathway reconstruction</H1><P class="text">
   To express exogenous GPCR，our plasmid manipulation is performed in E.coli strain top10. Odr-10 CDS sequence, which is optimized in GenScript, was inserted into the YEP352 shuttle vector with GAL1 promoter and ADH1 terminator.
   However the odr-10 fails to contain the N-terminal import sequence, which results in the difficulty of localization on plasma membrane when express odr-10 in heterogeneous cells. The overexpressed and inefficient folding odr-10 will stimulate the endoplasmic reticulum stress (ERS) and autophagy pathway. To solve the difficulty of odr-10 localization on the plasma membrane, we try to insert the sequence of rhodopsin signal peptide (rho) into the upstream of odr-10 CDS. Our rho sequence has also been optimized. Additionally, to detect whether our odr-10 has been transcribed and located on the plasma membrane or not, we insert the sequence of GPF in downstream of odr-10 and the sequence of flag-tagged into the position between our rho and odr-10 respectively.
   In the endogenous MAP-kinase pathway in S.cerevisiae, third messenger will stimulate fus1 promoter, which results in mating of the yeast. In like manner, we create the signal reporter device as “fus1+BFP+ADH1 terminator”. However the sequence of fus1 contains Pst1 restrict enzyme site, we use the method of overlap extension to achieve the purpose of site-directed mutagenesis (&quot;CTGCAG&quot; to &quot;CTGGAG&quot;).
  </P></DIV><DIV id="tblock4"><H1>Pathway optimization</H1><P class="text">
   There is no doubt that signal sensing part and signal reporter part are prerequisite, however, things cannot be that simple because the poor affinity between the  Gα(Gpa1) and heterogeneous GPCRs will slash the intensity and sensibility of our signal pathway. Similarly, transforming the yeast with full-length odr-3, which is the Gα in the <I>caenorhabditis elegans</I>, also has limited affinity for Gβγ of yeast.
   A preferential method is that we can create a chimera of Gpa1/odr-3 and only the carboxy-terminal five amino acids of Gpa1 are replaced with corresponding residues from odr-3subunit. This design is owing to the following two detections: first of all, the principal receptor specificity determinants reside at the very carboxy-terminus of Gαsubunits; secondly, The last eight amino acids of Gα are not resolved on the Gαβγ crystal structures, meaning that the exchanged regions have an undefined conformation extending from the end of the 5α helix Figure 2.</P><DIV class="imgC"><P class="tag"><STRONG>Figure 2.</STRONG> Gprotein heterotrimer. G-proteins are signaling molecules found on the inner side of cell membranes. This structure shows a complete complex of three chains with GDP bound.</P></DIV><P class="text">
   In order to achieve the communication between E.coli and S.cerevisiae, we need the heterologous receptor, odr-10, successfully express in the S.cerevisiae to detect the diacetyl produced by E.coli. The functional coupling of GPCRs has been achieved typically by modifications to the G protein alpha subunit, Gpa1p and Gβγ signals to a MAP kinase cascade to induce cell-cycle arrest by expressing FAR1 gene and transcription of mating genes. Therefore, successful coupling of foreign GPCRs need to only express chimeric Gαsubunit by deletion of Gpa1 and disable the cell-cycle arrest mechanism by deletion of FAR1. This year, we adopt Fusion PCR method to construct linear recombinant fragment, as description of Figure 3.</P><DIV class="imgC"><P class="tag" style="width:660px;"><STRONG>Figure 3. A schematic diagram of the use of fusion PCR to synthesize a fragment for gene knockout.</STRONG> Flanking DNA fragments are amplified with primers P1 and P2 and with P5 and P6. Primers P3 and P4 are designed to amplify zeocin. The two flanking fragments and the zeocin fragment are mixed and fused together by PCR creating a linear fragment used for transformation. Finally, transforming the linear fragment into S.cerevisiae and homologous recombination will occur and delete the target gene.</P></DIV></DIV><DIV id="tblock5"><H1>Results</H1><P class="text">We delete the FAR1 gene successfully as shown in the figure 4.
  </P><DIV class="imgC"><P class="tag" style="width:660px;"><STRONG>Figure 4: An ethidium-biomide-stained agarose gel showing results of FAR1 gene knockout.</STRONG><STRONG>a: </STRONG>The first lane is a 250bp DNA marker ladder. The lanes 2-5 are products of PCR using p3 and p4 as the primers using the knockout Far1 yeast genome as the template but the lane 6 uses the original yeast genome as the template. The bands are correct size of zeocin which is not in the original S.cerevisiae BY4741.<STRONG>b: </STRONG>The lane 6 is a 250bp DNA marker ladder. The lane 5 is product of PCR using primers inside Far1 using original yeast genome as the template but the remaining lanes use the knockout Far1 yeast genome as the template. The band is correct size of fragment inside Far1 gene.
   </P></DIV><P class="text">Until now, we have successfully constructed 14 expression plasmids. The sequence of flag-tag, Rho and our olfactory receptor odr-10 has been optimized in genscript.
  We have constructed three devices (DⅠ, DⅡ&amp; DⅢ) to express the olfactory receptor(OR).
  DⅠ: <A href="http://parts.igem.org/Part:BBa_K1072013" class="brick">BBa_K1072013</A>
  DⅡ: <A href="http://parts.igem.org/Part:BBa_K1072014" class="brick">BBa_K1072014</A>
  DⅢ: <A href="http://parts.igem.org/Part:BBa_K1072015" class="brick">BBa_K1072015</A>
  And then three odorant sensing systems(SⅠ, SⅡ&amp; SⅢ) were inserted into shuttle plasmid YEplac181.  Reporter device was <A href="http://parts.igem.org/Part:BBa_K1072020" class="brick">BBa_K1072020</A>, pFus1 + BFP + ADH1 terminator.
  SⅠ：<A href="http://parts.igem.org/Part:BBa_K1072016" class="brick">BBa_K1072016</A>
  SⅡ: <A href="http://parts.igem.org/Part:BBa_K1072017" class="brick">BBa_K1072017</A>
  SⅢ: <A href="http://parts.igem.org/Part:BBa_K1072018" class="brick">BBa_K1072018</A>
  Our olfactory receptor odr-10 has been transformed into <I>S.cerevisiae</I>. we transformed SⅠ<A href="http://parts.igem.org/Part:BBa_K1072016" class="brick">BBa_K1072016</A> into <I>S.cerevisiae</I> and cultured the cells until they reached the exponential phase(OD600, in the range of 1-2), then we diluted broth to OD600 0.5 and cultured in the selection media containing 2% galactose at 30℃ for 18h.
  Finally, we found that GFP, Flag tag signal and BFP were bright by confocal microscope，which means that Odr-10 gene have be translated, located and induced MAPK pathway successfully, as showing in figure 5.Flag tag was detected around plasma membrane, as we can see in figure 6.
  </P><DIV class="imgC"><P class="tag" style="width:660px;"><STRONG>Figure 5:</STRONG> Transforming Odr-10 with flag tag and reporter gene into yeast,GFP, flag tag signal and BFP was detected successfully, meaning that the whole pathway was stimulated by diacetyl.</P></DIV><DIV class="imgC"><P class="tag" style="width:660px;"><STRONG>Figure 6:</STRONG> FLAG immunolocalization of yeast cells expressing Odr-10 with flag tag, which means that Odr-10 protein locate at plasma membrane successfully.</P></DIV><P class="text">Additionally, we transformed SⅡ system into <STRONG>S.cerevisiae</STRONG> and cultured the cells as above method. We found that SⅡsystem, <A href="http://parts.igem.org/Part:BBa_K1072017" class="brick">BBa_K1072017</A>, expresses bright blue fluorescence under UV field under 400ms exposure time Figure 7. Both BBa_K1072016 and BBa_K1072017 work meaning that Brho isn’t the necessary or don’t work in yeast. Odr-10 successful localization may result from the interior signal peptide of cDNA of Odr-10.
  </P><DIV class="imgC"><P class="tag" style="width:580px;"><STRONG>Figure 7.</STRONG>Transforming Odr-10 with flag tag and Brho into yeast, BFP brighten,meaning that the whole pathway work and demonstrate that Brho isn't the necessary factors expressing Odr-10 in yeast.</P></DIV><P class="text">
   In order to optimize the pathway and achieve our expectation, we constructed Gpa1-odr3 chimera (chi) device to enhance the intension of signal transduction.
   The chimera, carboxy-terminal five amino acids of Gpa1 are replaced with corresponding residues from odr-3 subunit, will not only enhance the affinity between Gpa1 with odr-10 from <I>caenorhabditis elegans</I>, but also not influence the affinity between Gpa1 with Gβγ. We utilize the sequence (1000bp) upstream of Gpa1 to serve as the pGpa1 and create a novel biobrick, <A href="http://parts.igem.org/Part:BBa_K1072011" class="brick">BBa_K1072011</A>. To make it can be adapted to principle of IGEM, we successfully mutate the restriction sites of EcoR1 and Spe1in the sequence through site-specific mutagenesis.
   The chi device: </P></DIV><DIV id="tblock6"><H1>Reference</H1><P class="text">
   [1] Minic J, Persuy MA, Godel E et al. Functional expression of olfactory receptors in yeast and development of a bioassay for odorant screening. FEBS J. 2005 Jan;272(2):524-37.
   [2] S. J. Dowell, A. J. Brown. Yeast Assays for G-Protein–Coupled Receptors. Receptors and Channels. 2002, Vol. 8, No. 5-6 , Pages 343-352.
   [3] Audet M, Bouvier M. Restructuring G-protein-coupled receptor activation. Cell. 2012 Sep 28;151(1):14-23. 
   [4] Helen Dacres, Jian Wang,Virginia Leitch et al. Greatly enhanced detection of a volatile ligand   at femtomolar levels using bioluminescence resonance energy transfer (BRET). Biosens Bioelectron. 2011 Nov 15;29(1):119-24.
   [5] Radhika V, Proikas-Cezanne T, Jayaraman M et al. Chemical sensing of DNT by engineered olfactory yeast strain. Nat Chem Biol. 2007 Jun;3(6):325-30.
   [6] Szewczyk E, Nayak T, Oakley CE, et al. Fusion PCR and gene targeting in Aspergillus nidulans. Nat Protoc. 2006;1(6):3111-20.
   [7] M. S. Ferry,  I. A. Razinkov,  and J. Hasty. Microfluidics for Synthetic Biology:From Design to Execution. Book, 2011.
  </P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:SCUT/Project/Odorant_sensing">http://2013.igem.org/Team:SCUT/Project/Odorant_sensing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SCUT/Project/Odorant_sensing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SCUT/Project/Odorant_sensing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SCUT/Project/Odorant_sensing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SCUT/Project/Odorant_sensing&amp;oldid=236502" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>