"
Page
Discussion
View source
History
teams
Log in
 
Team:SCUT/Project/Oscillating odorant
From 2013.igem.org
(Redirected from Team:SCUT/javascript )
Home
Team
Members
Our University
Attributions
Notebook
Team Profile
Gallery
Project
Overview
Oscillating Odorant
Odorant Sensing
Future Work
Safety
Protocol
Data Page
Parts
Achievements
Modeling
Overview
Diacetyl Producer
Oscillation
Odr-10 Pathway
Human Practice
Overview
IGEM everywhere
Overview
Oscillating odorant
Odorant sensing
Safety
Protocol
Project
Introduction
Oscillating odorant contains two parts: the Producer and the Oscillator. We choose diacetyl as our desire odorant which is widely used as a food additive to emit buttery flavor. In order to enhance the production, we construct the E.coli as the producer for it hasn’t ALDC (acetolactate decarboxylase) to catalyze acetoin production，having excellent advantages in producing diacetyl. Additionally, we add the synchronized oscillation system on the basis of odorant producing system, together, distributing a special volatile—diacetyl periodically.
Introduce..
Design
Results
Reference
Design
Oscillator
Nowadays there has dramatic progress in the development of synthetic biology engineering, however, an actual challenge is the construction of robust circuits in a noisy cellular environment. Such environment leads to considerable intercellular variability in circuit behavior, which can hinder functionality at the colony level.
Here, we designed an oscillator to distribute diacetyl periodically, utilizing synergistic intercellular coupling involving quorum sensing machineries in Vibrio fischeri and Bacillus Thurigensis within a colony and gas-phase redox signaling between colonies that is capable of generating synchronized oscillations in a growing population of cells.
Figure 1. Five-pointed star gene circuit of oscillating odorant. The purple lux pR promoter control the expression of luxI, aiiA, ndh, GFP and α-ALS by five identical transcription modules while red lux pL promoter drives the expression of luxR. The whole gene circuit includes Oscillator, Coupling, Producer and Reporter four parts.
Designed as Fig.1, oscillating odorant includes four modules: Oscillator, Producer, Coupling and Reporter. We placed the luxI (from Vibrio fischeri), aiiA (from Bacillus Thurigensis), α-ALS(from Lactococcus lactis) , Ndh(from E.coli) and GFP genes under the control of five identical copies of the luxI promoter standing for the five angles.
The Oscillator part contains gene luxR, luxI and aiiA, generating synchronized oscillation by quorum sensing machineries. And α-ALS gene can express acetolactate synthetases that contribute to producing diacetyl as the Producer. Besides, gene ndh generating H2O2 vapour can promote Coupling between synchronized colonies. GFP represents the Reporter.
The LuxI synthase enzymatically produces an acyl-homoserine lactone (AHL), which is a small molecule that can diffuse across the cell membrane and mediates intercellular coupling. It binds intracellularly to the constitutively produced LuxR, and the AHL-luxR complex is a transcriptional activator for the luxI promoter. However, aiiA would negatively regulate the promoter by catalyzing the degradation of AHL. The ndh gene can express enzyme NDH-2 to generate H2O2 vapor which is an additional activator of the luxI promoter by changing cells redox state transiently. H2O2 vapor can migrate between colonies and then synchronize them. What’s more, GFP gene controlled by luxI promoter is a visualized and easy-measured reporter. This network architecture, whereby an activator activates its own protease or repressor, is similar to the motif used in other synthetic oscillator designs, constructing the core regulatory model for many circadian clock networks.
Producer
As one of the most important compounds in fermentation industry, diacetyl could ameliorate the flavor and aroma in fermented milk. Therefore, the significant aim of following scientific research should concentrate on how to enhance the capability of Lactococcus lactis to metabolize diacetyl. Since Lactococcus lactis is extensively used to produce diacetyl, its genome was utilized as a template to amplify the gene encoding α-Acetolactate synthase. In the meantime, Escherichin coli has many advantages in generating diacetyl, such as, lacking ALDC (acetolacetate decearboxylase) to catalyze acetoin production, metabolism is easy to control and its extensive cultural conditions.
For these reasons, our team decided to develop a preliminary study on generating diacetyl with engineered E.coli. As description of figure 2, the metabolism pathway of diacetyl, beginning with the central pathway metabolite pyruvate as the precursor molecule, the first step in the diacetyl biosynthetic pathway involves the condensation of two molecules of pyruvate to yield one molecule of acetolactate, and is catalyzed by acetolactate synthase (ALS).The stability of acetolactate is known to be affected by the presence of oxygen, and ample oxygen can cause its spontaneous decarboxylation to produce diacetyl.
Figure 2. The metabolism pathway of diacetyl in E.coli. Acetolactate synthase (ALS) catalyze two molecules of pyruvate to produce one molecule of acetolactate which was causes spontaneous decarboxylation to produce diacetyl by the presence of oxygen.
Plasmid construct
We have constructed 4 plasmids to create oscillating odorant, three for oscillating odorant and one for odorant production testing. pSB1K3, pSB3C5 and pSB4A5 are the vectors of oscillating odorant while pET22b(+) as Figure 3, of production testing.
Vector use for testing acetolactate synthase (ALS) expression is pET22b(+), a low copy plasmid that avoids causing inclusion body. After reading different kinds of papers, we find out that high copy plasmid may cause inclusion body when expressing oxidoreductase like our α-ALS then need to use molecular chaperones to help correctly folding the protein. That’s why we choose low copy plasmid, however, the diacetyl output is satisfied according to our results shown below.
In view of the combination of oscillator and producer, we link part BBa_K1072002 and BBa_1072006 together into pSB4A5. Besides, pSB1K3 with part BBa_K1072000 , pSB3C5 with part BBa_K546001 and pSB4A5 with part BBa_K1072002 and part BBa_1072006 correspond to high, medium and low copy replicon respectively in order to control the oscillation to be robust. Together, the three plasmids transfer into E.coli( Top 10) competent cells to create the oscillating odorant.
Figure 3. Map of pET-22b(+)
a.
b.
c.
Figure 4. Biobrick feature of parts containing in the three expression plasmids.
a. Part BBa_K1072000 on plasmid pSB1K3 correspond to high copy replicon
b. Part BBa_K546001 on plasmid pSB3C5 correspond to medium copy replicon
c. Part BBa_K546001 on plasmid pSB3C5 correspond to medium copy replicon
Results
Oscillating odorant
We have successfully constructed and expressed the three plasmids pSB1K3 with part BBa_K1072000; pSB3C5 with part BBa_K546001 and pSB4A5 with part BBa_K1072002 and part BBa_1072006 shown as figure 4. Also, we can know from the figure that the expression of GFP representing the production of α-ALS.
Figure 5. Green fluorescence from expression strain, representing the production of α-ALS
Oscillator
Until now, we have successfully constructed 9 expression plasmids.
Shown as Fg.6, part BBa_K1072000 (PluxR + RBS + LuxR + 2TM + PluxI  +RBS + LuxI + 2TM + PluxI + RBS + GFP + 2TM) expresses bright green fluorescence under UV field under 500ms exposure time.
Figure 6. Green fluorescence proving part BBa_K1072000 luxR+luxI+GFP work as expected.
Shown as Fg.7, part BBa_K1072022 also successfully expresses green fluorescence.
Figure 7. part BBa_K1072022 expresses green fluorescence as expected
In order to analyze the function of part BBa_K1072001 , BBa_K1072002, BBa_K1072003 and Ba_K1072022 , we utilize the characteristic that H2O2 has absorbance at 240nm to complete the linear chart of standard H2O2 concentration as Figure 8 and detect the H2O2 produced from part BBa_K1072002 and BBa_K1072022 by UV spectrophotometer. We choose part BBa_K546000 (pluxR + RBS + luxR + 2TM + pluxI +  RBS + luxI + 2TM) as the blank for part BBa_K1072002 (pluxR + RBS + luxR + 2TM + pluxI + RBS + luxI + 2TM + pluxI + RBS + ndh + 2TM), and get the concentration of H2O2 which is 7.21*10-2 mol/L. Besides, we also detect the H2O2 concentration of part BBa_K1072022 whose blank is part BBa_K1072000 and the results reach to 2.19*10-1 mol/L. Both data prove the functionality of the three parts that can emit H2O2 for next stage's experiments of constructing and adjusting oscillation.
Figure 8. Results of H2O2 detection. Tube 1 to 8 contain the standard concentration of H2O2 to draw the standard linear chart while S1 and S2 are the samples. S1 is part BBa_K1072003 whose blank control is part BBa_546000; S2 is part BBa_K1072022 whose blank control is part BBa_K1072000.
Figure 9. Linear chart of standard H2O2 concentration
Figure 10. H2O2 concentration chart
Producer
We utilize UV spectrophotometry, which is a practical and economical method to detect the concentration of diacetyl. In order to control the disturbance of LB media, we use M9+10%LB as our media and use 1M IPTG, 1ul/ml to induce expression in 30 ℃. After induction, 1 volume 16% TCA is used to retrieve diacetyl, then adding 1% O-phenylenediamine(OPG) to react with retrieved diacetyl. The end product has a characteristic absorbance in 335nm that forms our detecting method; this method may be contributive to the ferment milk research and development.
Figure 11. Results of diacetyl detection. Retrieved diacetyl do not react with OPG as the blank control group. MA, MA-, MB, MB- are all media M9+1% LB, but MA and MA- have induced by IPTG while MB and MB- have not. MA and MB have reacted with OPG while MA- and MB- have not as references. Additionally, four samples MA, MA-, MB, MB- are diluted one times.
According to the data above, we draw the linear chart of standard diacetyl concentration shown as Fg.8. Base on the chart, we can calculate the diacetyl concentration expressed from our engineered E.coli is 21.56mg/L.
Figure 12. Linear chart of standard diacetyl concentration
Figure 13. Diacetyl concentration chart
Reference
[1] Arthur Prindle, Phillip Samayoa, Ivan Razinkov, et al.A sensing array of radically coupled genetic ‘biopixels’. Nature. 2011 Dec 18;481(7379):39-44.
[2] Danino, T., Mondragon-Palomino, O. ,Tsimring, L. & Hasty, J. A synchronized quorum of genetic clocks. Nature 463, 326–330 (2010).
[3] Stricker, J. et al. A fast, robust and tunable synthetic gene oscillator. Nature 456, 516–519 (2008).
[4] Mondragon-Palomino, O., Danino, T., Selimkhanov, J., Tsimring, L. & Hasty, J. Entrainment of a population of synthetic genetic oscillators. Science 333, 1315–1319 (2011).
[5] Tigges, M., Marquez-Lago, T., Stelling, J. & Fussenegger, M. A tunable synthetic mammalian oscillator. Nature 457, 309–312 (2009).
[6] Ferry,M., Razinkov, I. & Hasty, J. Microfluidics for synthetic biology fromdesign to execution. Methods Enzymol 497, 295 (2011).
[7] Mondragon-Palomino, O., Danino, T., Selimkhanov, J., Tsimring, L. & Hasty, J. Entrainment of a population of synthetic genetic oscillators. Science 333, 1315–1319 (2011).
Retrieved from " http://2013.igem.org/Team:SCUT/Project/Oscillating_odorant "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
