<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SHPH-Shanghai_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SHPH-Shanghai/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:SHPH-Shanghai"><DIV class="menu_item direct_link" style="background:linear-gradient(#000000,#181818);height:4%;color:white">
				HOME
			</DIV></A><A href="https://2018.igem.org/Team:SHPH-Shanghai/Team"><DIV class="menu_item direct_link" style="background:linear-gradient(#181818,#303030);height:4%;color:white">
			TEAM
		</DIV></A><A href="https://2018.igem.org/Team:SHPH-Shanghai/Experiments"><DIV class="menu_item direct_link" style="background:linear-gradient(#303030,#484848);height:4%;color:white">
			EXPERIMENT
		</DIV></A><A href="https://2018.igem.org/Team:SHPH-Shanghai/Collaborations"><DIV class="menu_item direct_link" style="background:linear-gradient(#484848,#606060);height:4%;color:white">
				COLLABORATIONS
			</DIV></A><A href="https://2018.igem.org/Team:SHPH-Shanghai/Attributions"><DIV class="menu_item direct_link" style="background:linear-gradient(#606060,#787878);height:4%;color:white">
			ATTRIBUTIONS
		</DIV></A><A href="https://2018.igem.org/Team:SHPH-Shanghai/Safety"><DIV class="menu_item direct_link" style="background:linear-gradient(#787878,#909090);height:4%;color:white">
				SAFETY
			</DIV></A><A href="https://2018.igem.org/Team:SHPH-Shanghai/Human_Practices"><DIV class="menu_item direct_link" style="background:linear-gradient(#909090,#A8A8A8);height:4%;color:white">
			HUMAN PRACTICES 
		</DIV></A><A href="https://2018.igem.org/Team:SHPH-Shanghai/Parts"><DIV class="menu_item direct_link" style="background:linear-gradient(#A8A8A8,#C0C0C0);height:4%;color:white">
			PARTS
		</DIV></A><A href="https://igem.org/2018_Judging_Form?team=SHPH-Shanghai"><DIV class="menu_item direct_link" style="background-color:#C0C0C0;height:4%;color:white">
				JUDGING FORM
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper.column.full_size" style="margin-top:-44px"><DIV><A href="#pl" style="color:#ffffff"><DIV style="border-radius:30px;width:300px;height:60px;background-color:#808080;position:absolute;left:3%;top:100px;z-index:-1"><H2 style="color:#ffffff;text-align:center;height:30px">1.Pre-Lab
</H2></DIV></A><A href="#dp" style="color:#ffffff"><DIV style="border-radius:30px;width:300px;height:60px;background-color:#808080;position:absolute;left:3%;top:300px;z-index:-1"><H2 style="color:#ffffff;text-align:center">2.Degradation Plan
</H2></DIV></A><A href="#cbs" style="color:#ffffff"><DIV style="border-radius:30px;width:300px;height:60px;background-color:#808080;position:absolute;left:60%;top:100px;z-index:-1"><H2 style="color:#ffffff;text-align:center">3.Chassis Bacteria Selection
</H2></DIV></A><A href="#gmp" style="color:#ffffff"><DIV style="border-radius:30px;width:300px;height:60px;background-color:#808080;position:absolute;left:60%;top:300px;z-index:-1"><H2 style="color:#ffffff;text-align:center">4.Genetically Modified Parts
</H2></DIV></A><A href="#pd" style="color:#ffffff"><DIV style="border-radius:30px;width:300px;height:60px;background-color:#808080;position:absolute;left:3%;top:500px;z-index:-1"><H2 style="color:#ffffff;text-align:center">5.Primer Design
</H2></DIV></A><A href="#pcr" style="color:#ffffff"><DIV style="border-radius:30px;width:300px;height:60px;background-color:#808080;position:absolute;left:60%;top:500px;z-index:-1"><H2 style="color:#ffffff;text-align:center">6.PCR Plan
</H2></DIV></A><DIV style="width:85%;height:1000px;position:absolute;left:0%;top:700px"><A name="pl"><DIV style="height:100%;background-color:rgba(10,10,10,0.5)"><P style="color:white; position:absolute;top:5%;left:30%;font-size:40px">Preliminary  Experimentation</P><P style="color:white;position:absolute;right:13%;left:15%;top:15%;font-size:20px"> 
Purpose: Test whether acid has a positive effect on biofilm degradation during all growing stages of biofilm. 

<UL style="color:white;position:absolute;right:13%;left:15%;top:25%;font-size:20px">
Two variables are being investigated during the experiment: 
<LI>Number of days which biofilm has grown in condition of neutral nutrient solution (#=3,4,5)</LI><LI>The acidity of nutrient solution (pH=1,3,5 and 7, as a controlled group)</LI></UL><UL style="color:white;position:absolute;right:13%;left:15%;top:40%;font-size:20px">
Other variables are controlled:
<LI>Temperature of culturing</LI><LI>Number of days which biofilm is placed in acidic nutrient solutions (2 Days)</LI></UL></P><P style="color:white; position:absolute;top:50%;left:15%;font-size:30px">1.Procedure of Pre-experiment</P><DIV><DIV style="width:300px;height:60px;background-color:rgba(90,90,90,0.5);position:absolute;left:40%;top:60%"><P style="color:#ffffff;text-align:center;margin-top:20px">(1) Culturing in neutral medium for 3,4,5 days</P></DIV><DIV style="width:300px;height:60px;background-color:rgba(90,90,90,0.5);position:absolute;left:40%;top:70%"><P style="color:#ffffff;text-align:center;margin-top:20px">(2) Culturing in acidic medium (pH=1,3,5,7) for 2 days
</P></DIV><DIV style="width:300px;height:60px;background-color:rgba(90,90,90,0.5);position:absolute;left:40%;top:80%"><P style="color:#ffffff;text-align:center;margin-top:15px">(3) Measuring of the level of degradation (Optical Density Value)
</P></DIV></DIV></DIV></A><A name="dp"><DIV style="height:60%;background-color:rgba(25,25,25,0.5)"><P style="color:white; position:absolute;top:105%;left:40%;font-size:40px">Degradation Plan</P><P style="color:white;position:absolute;right:13%;left:15%;top:115%;font-size:20px"> 
Our first expectation on the genetically modified bacteria is that it can produce acid to reach a pH low as 3, then the structure of the biofilm can be damaged and therefore degraded.
</P><P style="color:white;position:absolute;right:13%;left:15%;top:125%;font-size:20px"> 
However, after reading plenty of papers, we started to notice that acids are too rare to be expressed. In the natural organisms, most of these organic acids are produced as a side-product of glycolysis and fermentation. (e.g. Lactobacillus bacteria can transform carbon sources into lactic acid via metabolism)
</P><P style="color:white;position:absolute;right:13%;left:15%;top:140%;font-size:20px"> 
So, we checked out those microorganisms which can produce acid through metabolism of saccharides. 
</P></DIV></A><A name="cbs"><DIV style="height:100%;background-color:rgba(40,40,40,0.5)"><P style="color:white; position:absolute;top:165%;left:30%;font-size:40px">Classis Bacteria Selection</P><OL style="color:white;position:absolute;right:13%;left:12%;top:175%;font-size:20px">
The chassis bacteria needs to have the following properties:
<LI>Alive functionally under a low pH value (pH=3)</LI><LI>Having the metabolism of glycolysis, turning monosaccharides into acids</LI><LI>Had been investigated and tested as a chassis bacteria</LI><LI>A rather safe and harmless bacteria which will not cause a secondary pollution</LI></OL><P style="color:white;position:absolute;right:13%;left:15%;top:195%;font-size:20px"> 
After reviewing for tons of species, we finally confirmed Lactobacillus Delbreucki as our chassis bacteria. 
</P><P style="color:white;position:absolute;right:13%;left:15%;top:205%;font-size:20px"> 
Lactobacillus species can be found naturally inside the human body: intestine and vagina. They help to maintain the healthy of vagina by keeping a low pH(around 3). 
</P><P style="color:white;position:absolute;right:13%;left:15%;top:215%;font-size:20px"> 
Lactobacillus species are also used in food producing processes. So, they are considered least harmful to both the environment and people’s health.
</P><P style="color:white;position:absolute;right:13%;left:15%;top:225%;font-size:20px"> 
The reason for choosing Lac. Delbreucki is because it is the only species we are able to get from the bacteria company. 
</P><P style="color:white;position:absolute;right:13%;left:15%;top:235%;font-size:20px"> 
There is also a very helpful property which Lactobacillus species have: when they are under a low pH (e.g. 3.5), they will produce hydrogen peroxide [1]. Hydrogen peroxide is used as a popular disinfector, which kill bacteria and is supposed to have a positive effect on the degradation of biofilm. 
</P><P style="color:white;position:absolute;right:13%;left:15%;top:250%;font-size:10px"> 
[1]Gregory E M,Fridovich I. Oxygen metabolism in Lactobacillus plantarum.[J]. Journal of Bacteriology,1974,117(1).
</P></DIV></A><A name="gmp"><DIV style="height:170%;background-color:rgba(55,55,55,0.5)"><P style="color:white; position:absolute;top:260%;left:27%;font-size:40px">Genetically Modified Parts Design</P><P style="color:white;position:absolute;right:13%;left:15%;top:270%;font-size:20px"> 
Instead of using direct secretion of acids, we choose to let our Lac. delbrueckii secrete lysozyme for decomposing polysaccharide structures in the biofilm into monosaccharides, which is a raw material for glycolysis. 
<OL style="color:white;position:absolute;right:13%;left:12%;top:280%;font-size:20px">
2 things we need to consider:
<LI>a proper lysozyme which can decompose polysaccharides</LI><LI>a sequence which can promote the secretion of the lysozyme</LI></OL></P><P style="color:white; position:absolute;top:295%;left:15%;font-size:30px">1.Lysozyme</P><P style="color:white;position:absolute;right:13%;left:15%;top:300%;font-size:20px"> 
As we know, one of the main structural component in biofilm is polysaccharides. There are tons of kinds of polysaccharides, but they all have one same property: they are polymers of monosaccharides through the connection of glucosidic bonds. 
</P><P style="color:white;position:absolute;right:13%;left:15%;top:310%;font-size:20px"> 
The purpose of the lysozyme is to break the glucosidic bonds. 
</P><P style="color:white;position:absolute;right:13%;left:15%;top:315%;font-size:20px"> 
The lysozyme we selected is: NAG (N-Acetyl-β-D Glucosaminidase) 
</P><P style="color:white;position:absolute;right:13%;left:15%;top:320%;font-size:20px"> 
The gene of NAG is present in the human body, and NAG itself is used as a indicator for health. We collected useless tissue during the liver transferring and extracted the NAG gene from the part of wasted liver tissue. </P><P style="color:white;position:absolute;right:13%;left:15%;top:335%;font-size:10px"> 
Ref: https://pacbio.com/biomarker/assay-detail/145/</P><P style="color:white; position:absolute;top:340%;left:15%;font-size:30px">2.Secretion Promoter</P><P style="color:white;position:absolute;right:13%;left:15%;top:345%;font-size:20px"> 
After read lots of papers, we are attracted by the special function of the s-layer protein signal peptide. The signal peptide can promote the secretion of the whole s-layer protein. For Lac. brevis, its s-layer protein has a property of adherence to epidermal cells. At first we only want to have the gene for the signal peptide since it is the part which promotes the secretion, and with it our cell will be able to secrete NAG. But after comparing the size of signal peptide (370bp) and that of NAG (1881bp), we are afraid whether the signal peptide is able to “pull out” such a huge molecule.
</P><P style="color:white;position:absolute;right:13%;left:15%;top:370%;font-size:20px"> 
Therefore, we kept all genes of s-layer protein (protein + signal peptide).
</P><P style="color:white; position:absolute;top:380%;left:15%;font-size:30px">3.Combination</P><P style="color:white;position:absolute;right:13%;left:15%;top:415%;font-size:20px"> 
As we suppose, the s-layer protein will be connected with NAG with a linker (made up of 5 repeated amino acid). By using a linker as a connector, NAG will be more likely to have it active site unaffected by the s-layer protein. 
</P></DIV></A><A name="pd"><DIV style="height:110%;background-color:rgba(70,70,70,0.5)"><P style="color:white; position:absolute;top:430%;left:40%;font-size:40px">Primer Design</P><P style="color:white;position:absolute;right:13%;left:15%;top:440%;font-size:20px"> 
According to the requirement, we choose two restriction site —Xbal and Hinder III—. The primers we designed stick to the former two restriction sites. 
</P><P style="color:white; position:absolute;top:450%;left:15%;font-size:30px">1.Primer for S-layer protein</P><P style="color:white;position:absolute;right:13%;left:15%;top:480%;font-size:20px">The bases written in red are complementary to the restriction enzyme cutting site XbaI.
</P><P style="color:white; position:absolute;top:490%;left:15%;font-size:30px">2.Primer for NAG</P><P style="color:white;position:absolute;right:13%;left:15%;top:520%;font-size:20px">The bases written in red are complementary to the restriction enzyme cutting site Hind III.</P><P style="color:white;position:absolute;right:13%;left:15%;top:525%;font-size:20px">The bases written in blue are the linker between NAG and s-layer protein. The bases before the linker is complementary with the end of the s-layer protein’s gene, which is for the latter step: fusion PCR, I will introduce it in a few slides later.
</P></DIV></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>