<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Hong_Kong-CUHK_Project_light"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Hong_Kong-CUHK/Project/light">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Hong_Kong-CUHK/Project/light&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Hong_Kong-CUHK/Project/light&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Hong_Kong-CUHK/Project/light&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Hong_Kong-CUHK/Project/light" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Hong Kong-CUHK/Project/light</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="header"><DIV class="span-10 prepend-1" style="height:380px;" alt="CUHK iGEM"><P id="uni">Chinese University of Hong Kong iGEM 2011</P></DIV></DIV><DIV class="container" id="project"><DIV class="span-7"><DIV><UL class="nav-list" id="project-list"><LI><A href="/Team:Hong_Kong-CUHK/Project/overview" id="overview">Overview</A></LI><LI><A href="/Team:Hong_Kong-CUHK/Project/background">Background</A></LI><LI><A class="list-2" href="/Team:Hong_Kong-CUHK/Project/Halorhodopsin">Halorhodopsin</A></LI><LI><A class="list-2" href="/Team:Hong_Kong-CUHK/Project/Chloride Sensing Unit">Chloride Sensing Unit</A></LI><LI><A class="list-2" href="/Team:Hong_Kong-CUHK/Project/Mixing Entropy Battery">Mixing Entropy Battery</A></LI><LI><A href="javascript:void(0)">Results</A></LI><LI><A class="list-2" href="/Team:Hong_Kong-CUHK/Project/Data_page">Data Page</A></LI><LI><A class="list-2 selected" href="/Team:Hong_Kong-CUHK/Project/light">Light Intra-tunable System</A></LI><LI><A class="list-2" href="/Team:Hong_Kong-CUHK/Project/electricity">Solar Electricity Generation</A></LI><LI><A href="/Team:Hong_Kong-CUHK/Project/further">Future Applications</A></LI><LI><A href="/Team:Hong_Kong-CUHK/Project/Judging Form">Judging Form</A></LI></UL></DIV></DIV><DIV class="span-16" id="light"><H2>Light Intra-tunable System</H2><H4><STRONG>Quick Access:</STRONG></H4><H5>A. <A href="#Validation of halorhodopsin system">Validation of halorhodopsin system</A></H5>
1. <A href="#A1">Halorhodopsin was correctly inserted to pSB1C3 and expressed in BL21(DE) properly.</A>
2. <A href="#A2">BL21(DE) transformed with complete halorhodopsin system (BBa_K559010) grows normally in LB medium with additional sodium chloride concentrations ranged from 0 M to 0.4 M.</A>
3. <A href="#A3">Halorhodopsin functions properly in BL21(DE).</A><H5>B. <A href="#Light-coupled expression platform">Light-coupled expression platform</A></H5>
1. <A href="#B1">Intracellular chloride ion concentration of halorhodopsin-expressed bacteria is controllable by regulating the wavelength, intensity and duration of light illumination.</A>
2. <A href="#B2">Mixing-entropy electrodes can monitor extracellular change of sodium chloride concentration.</A>
3. <A href="#B3">Pgad has the capability to control downstream gene expression according to intracellular chloride concentration.</A><H5>C. <A href="#C">Modeling of light intra-tunable system</A></H5>
1. <A href="#C1">Model of chloride uptake of halorhodopsin</A><H4><STRONG>A. Validation of halorhodopsin System</STRONG></H4><P><I><A href="#top">(Back to top)</A></I></P><P>
We successfully construct three biobricks: <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> (BBa_K559000), <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> under <I>T7</I> promoter (BBa_K590001) and <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> complete system (BBa_K559010) and all biobricks were sent to part-registry.  In this session, we are going to show the data proving that our biobricks function properly.</P><P><STRONG>1.	Halorhodopsin was correctly inserted to pSB1C3 and expressed in <I>BL21(DE)</I> properly.</STRONG></P><P><I><A href="#top">(Back to top)</A></I></P><P>We use PCR with six pairs of primers and gel electrophoresis to confirm that we have properly constructed <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> complete system (BBa_K559010) (Fig. 1). All bands in each lane match their theoretical sizes. <A href="http://en.wikipedia.org/wiki/Halorhodopsin">Halorhodopsin</A> complete system then was transformed to <I>BL21(DE)</I>. Since our <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> has been HIS-tagged, we performed western blot to confirm the expression of <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> in <I>BL21(DE)</I>(Fig. 2). The PCR and western blot results indicate that our biobricks is successfully constructed and <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> can be properly expressed in <I>BL21(DE)</I>.</P><P>Fig. 1 <A href="http://en.wikipedia.org/wiki/Halorhodopsin">Halorhodopsin</A> complete system (BBa_K559010) was correctly constructed.   Components in the plasmid were amplified by standard PCR using corresponding primers. The PCR conditions were 95°C denature for 30s, 54 °C annealing for 15s and 72°C extension for 30s. The number of PCR cycle was 30. The PCR products of corresponding lanes are shown on the table. T7 was amplified by T7 F and T7 R, HR was amplified by HR F and HR R, 24C was amplified by 24C F and 24C R, T7 HR was amplified by T7 F and HR R, T7 24C was amplified by T7 F and 24C R and HR 24C was amplified by HR F and 24C R.</P><P>  To optimize the expression of <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A>, induction by different concentration of IPTG was carried out and SDS-PAGE was done to investigate the results. 0.10mM concentration of IPTG was found to be optimum to induce <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> expression. We thus decided to use this condition to induce expression afterwards.</P><P>Fig.2 Optimization of <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> expression in <I>BL21(DE)</I>. <I>BL21(DE)</I> was transformed according to standard protocol. Bacterial culture grew from a single colony at 37℃ in the presence of chloramphenicol. Different concentration of IPTG was used to induce gene expression of <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> when OD600 reached 0.4. After IPTG induction for 4 </P><P><STRONG>2.	<I>BL21(DE)</I> transformed with complete halorhodopsin system (BBa_K559010)   grows normally in LB medium with additional sodium chloride concentrations ranged from 0 M to 0.4 M.</STRONG></P><P><I><A href="#top">(Back to top)</A></I></P><P>In order to determine whether <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> would affect the survival of <A href="http://en.wikipedia.org/wiki/E._coli"><I>E. coli</I></A>, we performed an assay to examine the growth curve of <I>BL21(DE)</I> transformed with the complete <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> system (BBa_K559010) in different sodium chloride concentrations (Fig. 5). Our data show that the bacteria expressing <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> grew normally in LB medium with additional sodium chloride concentration ranged from 0 M to 0.4 M. When the sodium chloride concentration was above 0.4 M, the growth of the bacteria was inhibited due to extremely high salinity conditions. <A href="http://en.wikipedia.org/wiki/Halorhodopsin">Halorhodopsin</A> does not affect the growth and survival of <I><A href="http://en.wikipedia.org/wiki/E._coli">E. coli</A></I>.</P><P>Fig. 3  Growth curve of bacteria under different NaCl concentration. <I>BL21(DE)</I> was transformed according to standard protocol. Bacterial culture grew from a single colony at 37°C. 0.1 mM of IPTG was used to induce <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> expression. 1% (v/v) inoculum was used as start culture at 0 hour. The change in OD600 was monitored.</P><P><STRONG>3.	Halorhodopsin functions properly in BL21(DE).</STRONG></P><P><I><A href="#top">(Back to top)</A></I></P><P>To test the function of <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A>, we performed MQAE assay to measure intracellular chloride concentration right after light illumination to bacterial samples (Fig. 6, Fig. 7). After light illumination, the bacteria expressing <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> have significantly higher intracellular chloride concentrations compared with the bacteria without <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A>. Our data show that <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> pumps chloride ions from medium into bacteria during light illumination. In conclusion, our biobricks can function properly in <A href="http://en.wikipedia.org/wiki/E._coli"><I>E. coli</I></A>. In additional, we also proved that function of <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> is light depended (Fig. 7). </P><P> Fig. 4 <A href="http://en.wikipedia.org/wiki/Halorhodopsin">Halorhodopsin</A> expressing bacteria absorbed chloride ions. BL21(DE) was transformed and bacterial culture grew from a single colony at 37°C in the presence of light. 0.1 mM of IPTG and 0.4 M of NaCl were included in the culture medium. Intracellular chloride ion concentration was determined by MQAE after 4 hours induction. Error bar represents SEM.</P><P>Fig. 5 <A href="http://en.wikipedia.org/wiki/Halorhodopsin">Halorhodopsin</A> expressing bacteria absorbed chloride ions under different condition. <I>BL21(DE)</I> was transformed and bacterial culture grew from a single colony at 37°C under different conditions. Bacterial samples were collected when OD600 reached 0.4. Bacteria from different growth conditions were disrupted. The intracellular chloride ion concentration was determined by MQAE. Light: bacteria grew in the presence of light. IPTG: 0.1 mM of IPTG was used to induce gene expression. NaCl: the LB contained 0.4 M of NaCl. “+” indicates the corresponding component was included. “-” indicates the corresponding component was not included. Error bar represents SEM.</P><P>From the results, light, IPTG and NaCl are all required for the large amount increase in the chloride ion absorption by <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A>. The comparative increase in set up 4, 5 and 7 are due to the addition of NaCl, which causes a greater diffusion of chloride ions into the cell.</P><H4><STRONG>B. Light-coupled expression platform</STRONG></H4><P><I><A href="#top">(Back to top)</A></I></P><P>Fig. 6 Computer-aided light-coupled gene expression regulation platform</P><P>We propose this computer-aided light-coupled gene expression regulation platform for the purpose of dynamically controlling the expression and function of construct genes (Fig. 6). In this platform, we utilize <A href="http://en.wikipedia.org/wiki/E._coli"><I>E. coli</I></A> transformed with <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> to modulate intracellular chloride level by light, and Pgad expression cassette to induce gene expression according to the variation in intracellular chloride level.</P><P>This platform consists of three components: computer instruction, bacteria expression and signal feedback. In computer instruction, a laser power device is controlled by computer and illuminates towards cell culture with desirable intensity, wavelength and duration. When genetically modified <A href="http://en.wikipedia.org/wiki/E._coli"><I>E. coli</I></A> receives light illumination, <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> pumps chloride ions from medium into cytoplasm. Thereafter intracellular chloride ion level keeps increasing and chloride concentration in medium keeps decreasing. Inside the bacteria, gadR up-regulates target genes and feedback signal (e.g. green fluorescence protein) downstream of Pgad according to the increasing intracellular chloride level. Detector thus can sense the feedback signal as well as the regulated amount of target genes expression back to computer. Meanwhile, the pair of ion absorption electrodes can also detect the change in chloride ion concentration in the medium and return this information to computer, which could analyze intracellular chloride level and target gene expression speed. Therefore the target gene expression in this platform becomes dynamically tunable and programmable with the aid of computer.</P><P>To illustrate the feasibility of this design, we performed the following assays:</P><P><STRONG>1.	Intracellular chloride ion concentration of halorhodopsin-expressed bacteria is controllable by regulating the wavelength, intensity and duration of light illumination.</STRONG></P><P><I><A href="#top">(Back to top)</A></I></P><P>We followed MQAE method to determine intracellular chloride ion concentrations after the cells were treated with different wavelength, intensity and duration of illumination. Previous study shows that <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> mainly absorbs photons with wavelength around 600 nm1. Our data agree with this conclusion and we find that <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> functions with maximum efficiency at 530 nm (Fig. 7) and changing the wavelength can significantly reduce its efficiency. </P><P>Then we fixed the wavelength to 530 nm and measured the pumping efficiency of <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> under different light intensity. First we tested the survival of <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A>-transformed <A href="http://en.wikipedia.org/wiki/E._coli"><I>E. coli</I></A> under different light intensities (Fig. 8). The light intensity was indicated by the percentage of full LASER power supply under confocal microscope. The growth of transformed <A href="http://en.wikipedia.org/wiki/E._coli"><I>E. coli</I></A> was not significantly affected when they were exposed to the light intensity below 25 percent of full power. <A href="http://en.wikipedia.org/wiki/E._coli"><I>E. coli</I></A> could still grow normally when they were exposed under the light with intensity between 30 percent and 40 percent of the full power. However, when the LASER power exceeded 40 percent, the bacterial growth was significantly inhibited.  In the next step, we examined the chloride absorption efficiency of <A href="http://en.wikipedia.org/wiki/Halorhodopsin">halorhodopsin</A> under different light intensity (Fig. 9). We found that the highest chloride pumping efficiency appeared at 25 percent of full power. When the intensity kept increasing, the intracellular concentration dropped due to death of <A href="http://en.wikipedia.org/wiki/E._coli"><I>E. coli</I></A>. Furthermore, we studied the relation between intracellular chloride ion concentration and illumination duration under 25 percent of full power (Fig. 10). The highest intracellular chloride concentration was 0.55 M after samples was illuminated for 210 seconds. Afterwards intracellular chloride ions kept decreasing due to cell death caused by high LASER power. </P><P>In conclusion, our data present the feasibility where intracellular chloride concentration can be accurately controlled by the wavelength, intensity and duration of light illumination.</P><P>Fig. 7 <A href="http://en.wikipedia.org/wiki/Halorhodopsin">Halorhodopsin</A> expressing bacteria absorbed chloride ions induced by laser with
different wavelengths. <I>BL21(DE)</I> was transformed by BBa_K559010 and the bacterial culture grew from a single colony in LB with 0.4 M of NaCl at 37°C in the absence of light. Bacterial samples were collected when OD600 reached 0.4. 500 μl of bacterial sample was excited by 20% laser power with different wavelength for 2 minutes under confocal microscope. 200 μl of excited bacteria was collected and disrupted. Its intracellular chloride ion concentration was measured by MQAE. Error bar represents SEM.
</P><P>Fig. 8 Bacterial growth after exposure to different power of laser with wavelength 530 nm. Transformed <I>BL21(DE)</I> grew in LB with 0.4 M of NaCl. 500 μl of bacteria culture was used for laser exposure when OD600 reach 0.4. Bacteria were exposed to different laser power for 2 minutes. After the exposure, 200 μl of the bacteria was cultured in 2 ml of LB. OD600 was determined immediately after inoculation (0 hour) and after 2 hour grew in LB. Table shows the value of OD600 at corresponding time point. Data was expressed as mean ± SEM.</P><P>Fig. 9 Laser (530 nm) induced Chloride ion absorption. <I>BL21(DE)</I> was transformed and a single colony  was picked to grow in LB with 0.4 M of NaCl. 500 μl of bacterial sample with OD600 reached 0.4 was excited by laser (530 nm) with different LASER power (%). 200 μl of excited bacteria was collected and disrupted. The intracellular concentration of chloride ion was determined by MQAE. Error bar represents SEM.
</P><P>Fig. 10 Chloride ions absorption responded differently with the duration of laser (530 nm) exposure. <I>BL21(DE)</I> was transformed and a single colony  was picked to grow in LB with 0.4 M of NaCl. 500 μl of bacterial sample with OD600 reached 0.4 was excited by LASER (530 nm) with different exposure time. The LASER power was fixed at 25%. 200 μl of excited bacteria was collected and disrupted. The intracellular concentration of chloride ion was determined by MQAE. Error bar represents SEM.</P><P><STRONG>2.	Mixing-entropy electrodes can monitor extracellular change of sodium chloride concentration.</STRONG></P><P><I><A href="#top">(Back to top)</A></I></P><P>In our design, we utilize the voltage between mixing-entropy electrodes (For more details, please see &quot;<A href="https://2011.igem.org/Team:Hong_Kong-CUHK/Project/electricity">Solar Electricity Generation</A>&quot; session)  to monitor the extracellular change in sodium chloride concentration. Thus we measured the steady voltage against sodium chloride concentration to prove our concept (Fig. 11). In these data, the steady voltage increases along with the increase of sodium chloride concentration. It shows that the voltage between the electrodes could be utilized as a feedback signal to monitor extracellular sodium chloride concentration. However, the sensitivity of voltmeter would limit the resolution of sodium chloride concentration, which should be improved in the future.</P><P>Fig. 11 Voltage against sodium chloride concentration test shows the capability using entropy-mixing electrodes to monitor extracellular sodium chloride concentration. The electrodes were immersed in 25 ml sodium chloride solutions with different concentrations until the voltage achieved a steady level. The steady voltages were recorded and plotted.</P><P><STRONG>3.	Pgad has the capability to control downstream gene expression according to intracellular chloride concentration.</STRONG></P><P><I><A href="#top">(Back to top)</A></I></P><P>Sanders J.W. et al. succeeded in constructing P<SUB>gad</SUB> expression cassette in L. lactis in 1997<SUP>1</SUP> (For more details, please see &quot;<A href="https://2011.igem.org/Team:Hong_Kong-CUHK/Project/Chloride_Sensing_Unit">Chloride Sensing Unit</A>&quot; session). We borrowed their design and tried to apply it on <I>E. coli</I>. </P><P>P<SUB>gad</SUB> expression cassette was synthesized according the sequence reported by Sanders J.W. et al.<SUP>1</SUP> However there were 1 EcoRI and 2 XbaI restriction sites. Two of the restriction sites were in the protein coding region (gadR) and one in the intergenic region. The two restriction sites in protein coding region were eliminated by changing the base to synonymous frequent codon and the intergenic restriction site was also eliminated by changing to another base. P<SUB>gad</SUB> expression cassette was ligated to pSB1C3. It was confirmed by restriction cut (Fig. 12) and PCR (Fig. 13).

</P><P>Fig. 12 Gel electrophoresis after restriction cut to confirm that the P<SUB>gad</SUB> was correctly inserted to pSB1C3. P<SUB>gad</SUB> was cutted by restriction enzymes EcoRI and PstI and the size of insert and vector showed are around 1300 bp and 2000 bp, respectively.


</P><P>Fig. 13 PCR confirmed that the P<SUB>gad</SUB> was correctly inserted to pSB1C3. P<SUB>gad</SUB> was successfully amplified by primer 1 (red circle).</P><P>Fig.  14 Qualititive measurement shows EGFP expression varies with extracellular NaCl concentrations under P<SUB>gad</SUB> control. We used PCR (row 4 and 5) and western blotting (row 6 and 7) to qualitatively investigate the change of EGFP expression. As the extracellular NaCl concentration increases, the amount of EGFP was elevated significantly in both mRNA level and protein level.</P><P>EGFP was ligated downstream of P<SUB>gad</SUB> expression cassette in pSB1C3 plasmid, and transformed it to BL21(DE) together with halorhodopsin complete system (Fig. 12). BL21(DE)samples were grew in mediums of different sodium chloride concentration. After 210 seconds light excitation, samples were withdrawn. Then PCR and western blotting were performed to measure the expression of EGFP. The expression of EGFP changed significantly in both mRNA level and protein level according to extracellular sodium chloride concentration. Our result proves that under the same illumination condition, with the presence of halorhodopsin, E. coli absorbed increasing amount of chloride ion when the extracellular chloride concentration is elevating, and with the regulation of P<SUB>gad</SUB>, the bacteria generates increasing amount of EGFP, which could be any other genes downstream of P<SUB>gad</SUB>. Halorhodopsin-P<SUB>gad</SUB> system shows a great potential to become a light-tunable platform to manually control gene expression.</P><P>Our next step is to incorporate P<SUB>gad</SUB> into pSB1C3 to become a functional biobrick, and perform quantitative measurement to further characterize and model this photo-tunable gene expression system.</P><H4><STRONG>C. Modeling of light intra-tunable system</STRONG></H4><P><STRONG>1. Model of chloride uptake of halorhodopsin</STRONG></P><P><I><A href="#top">(Back to top)</A></I></P><P>The chloride uptake of halorhodopsin is modeled as following equation:</P><P>Where C<SUB>i</SUB> is measured intracellular chloride concentration, f(t) is intracellular concentration assuming that bacteria would not die when chloride ions keep flowing in at time t of light illumination, and g(t, I, λ, C<SUB>e</SUB>) is bacteria survival rate at time t under light illumination with wavelength λ and intensity I, when the cell is grown in medium of C<SUB>e</SUB>  NaCl concentration.</P><P>The factor f(t) could be modeled according to logistic function:</P><P>Where C<SUB>0</SUB> is the initial intracellular chloride concentration, C<SUB>m</SUB> is theoretical maximum intracellular chloride concentration, r is halorhodopsin pumping rate and t is light illumination time. The rate of chloride uptake is thus to be:</P><P>The halorhodopsin pumping rate (r) relates to the intensity and the wavelength of light illumination on the system. The function of halorhodopsin can be considered as an enzyme to mediate chloride ions flowing into cell, that is:</P><P>Therefore the halorhodopsin pumping rate (r) could be modeled similar to Michaelis–Menten kinetics:</P><P>Where r<SUB>max</SUB> is the maximum halorhodopsin pumping rate, I is light intensity and β is light efficiency constant.</P><P>r<SUB>max</SUB> and β are determined by the wavelength of input light and the number of halorhodopsin per cell.</P><P>Where λ is the wavelength of input light and n is the number of halorhodopsin per cell.</P><P>Our model assumes that there is a maximum intracellular chloride capacity C<SUB>m</SUB> due to the fact that while chloride ions are pumped into the cell by halorhodopsin, some chloride ions leak out because of the increasing osmolality. The larger the osmolality potential across the membrane, the faster the chloride leaking out of the cell. Finally the intracellular chloride concentration achieves equilibrium and there is no net chloride ions flowing into the cell. </P><P>However, in our system, cell might die due to high light intensity, long-term light illumination and high salinity inside the cell, and finally release chloride ions back to the solution. Therefore in the model, cell death rate is included as a function g, which relates to illumination time t, light intensity I, light wavelength λ and extracellular chloride concentration C<SUB>e</SUB>:</P><P>We are going to use real experimental data to estimate unknown distribution and improve our model.</P><P><I><A href="#top">(Back to top)</A></I></P><H4>References</H4><P>1.	Sanders, J.W., Venema, G. &amp; Kok, J. A chloride-inducible gene expression cassette and its use in induced lysis of Lactococcus lactis. Applied and environmental microbiology 63, 4877(1997). </P></DIV></DIV><DIV><DIV class="span-24" style="text-align:left;"><P>&quot;Creativity is thinking up new things. Innovation is doing new things.&quot; - Theodore Levitt
</P><P>©Copyright CUHK IGEM Team 2011</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Hong_Kong-CUHK/Project/light">http://2011.igem.org/Team:Hong_Kong-CUHK/Project/light</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Hong_Kong-CUHK/Project/light" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Hong_Kong-CUHK/Project/light" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Hong_Kong-CUHK/Project/light&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Hong_Kong-CUHK/Project/light&amp;oldid=254414" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>