<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Duesseldorf_Proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Duesseldorf/Proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content"><DIV class="nav"><TABLE style="margin: 0; border: none; width: 100%;"><TBODY><TR style="border:none;"><TD style="border:none;"><UL><LI><A href="https://2016.igem.org/Team:Duesseldorf">Home</A></LI><LI class="dropdown"><A href="#" style="color: white;">Project</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A style="color: white;" href="https://2016.igem.org/Team:Duesseldorf/Proof">Results</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Future_Applications">Future Applications</A></LI></UL><LI><A href="https://2016.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach#HP">Education</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach#IHP">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI></TD></TR><TR style="border:none;"><TD style="border:none;">
                     Heinrich-Heine University 
                     Universitätsstraße 1 
                     40225 Düsseldorf  
                     Design by Marvin van Aalst </TD></TR></TBODY></TABLE></DIV><DIV class="page"><DIV class="paper"><DIV id="Navigation" class="navigation"><UL class="topnav"><LI><A href="https://2016.igem.org/Team:Duesseldorf">Home</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A style="color: black;" href="https://2016.igem.org/Team:Duesseldorf/Proof">Results</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Future_Applications">Future Applications</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach#HP">Education</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach#IHP">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI></UL></DIV><DIV class="attention"><H1 style="color: white;">Results</H1><P style="color:white;"><LI><A style="color:white;" href="#Introduction">Introduction</A></LI><LI><A style="color:white;" href="#Mammals">Team Mammals</A></LI><LI><A style="color:white;" href="#Team_Red">Team Red Results</A></LI><LI><A style="color:white;" href="#Yeast">Yeast</A></LI><LI><A style="color:white;" href="#References">References</A></LI></P></DIV><H2 id="Introduction">Introduction</H2><H3>Saccharomyces cerevisiae - More than just the Baker’s Yeast to us</H3><P>
When we started planning our project we wanted to use an easy-to-manipulate, single-celled eukaryote as our model system. Consequently we found out that cancer researchers use yeast for their basic research. 
</P><P>
At the first glance humans and yeast are very different and also inhabit completely different branches of the evolutionary tree. But as we took a closer look we could see that we have many similarities with this unicellular fungal microorganism. For example around forty per cent of human genes are also found in yeast. <SUP>[1]</SUP></P><P>
It is especially important to note that the same genes that control the cell cycle in baker's yeast (and that malfunction in tumor cells) exist in more or less the same capacity in human cells. <SUP>[2]</SUP> Additionally you only need some nutrient medium and a warm incubator to keep them happy. This makes Baker's yeast so sufficient for our project and really easy to handle.
</P><H3>Why we decided to build our construct in mammalian cells</H3><P>
Halfway through the iGEM competition, halfway done with our yeast constructs, an idea took shape in our minds.
Additionally to our model organism yeast, we wanted to validate our construct in mammalian cells. To show the world, that OPTOPTOSIS was indeed paving the way towards a future application in research. 
</P><P>
We knew that working with mammalian cells would be a challenging experience for bachelor students. Nonetheless, we accepted the challenge. A part of our team became acquainted with all the necessary procedures and methods for working with HeLa and CHO, especially in methods of cell culture and transfection, as well as illumination conditions for our transfected constructs <SUP>[3]</SUP>.  
</P><P>
HeLa cells are derived from the cervical cancer cells obtained from Henrietta Lacks <SUP>[4]</SUP> and were established as one of the most important immortal cancer cell lines for cancer research. Additionally, we work with CHO cells (chinese hamster ovary cells) to test our system in more than one cell type and for comparison of these different cell types. 
</P><H2 id="Mammals">Results Mammals</H2><P>
Our first experiment was the constitutive expression of PX (ePDZb1-mCherry-BaxE184) and LX (TOM5-GFP-LOV2) with the viral SV40 promoter in HeLa cells. Expression of both plasmids was shown by the fluorescent reporters in our fusion proteins. Figure 1 displays, that the cells show emission by the fluorescent reporters. 
</P><P><I>Figure 1: Confocal image of constitutive expression of PX and LX shows expression of fluorescent reporters mCherry and GFP</I></P><P>
First we tested the transfected cells via excitation at a wavelength of 587 nm to excite mCherry. Both tested cells showed red light emission with a wavelength of 610 nm, which is not only evidence for positive transfection but also for successful gene expression. 
</P><P>
Secondly we showed the expression of LX  via excitation at 395 nm. GFP emitted green light with a wavelength of 509 nm which proofs the gene expression of the second construct. Because of the TOM5 mitochondrial anchor in the LX construct, GFP should be localized at the outer mitochondrial membrane. We assume that the stronger emission as well as the concentration of GFP at the blue marked position in Figure 1 supports this interpretation.
</P><P>
Furthermore we illuminated the transfected cells with blue light in order to activate the blue light switch, as well as the localization of the PX fusion protein at the outer mitochondrial membrane as a result of the interaction between LOV2’s Jα helix and PX’s ePDZb1. We assumed the yellow marked position in Figure 1 shows a concentration of red light emitting mCherry located at the mitochondrion, which would be a demonstration of exact interaction and recruitment between LOV2 and PDZ. The signal from mCherry and GFP showed significant colocalization, showing that recruitment has taken place (see figure 2, both GFP and mCherry emission together).
</P><P><I>Figure 2: Confocal image of constitutive expression of PX and LX shows expression of fluorescent reporters mCherry and GFP as in overlay of both channels</I></P><P>
In response to blue light cells showed apoptotic cell morphology, including apoptotic bodies (see red arrow figure 1). This lets us assume that the illumination with blue light induced apoptosis in the cell cultures. Yet, due to the high concentration of Bax (a result of its constitutive expression) it may be possible that necrosis occurred as well.
</P><P>
We are planning to conduct an apoptosis assay to distinguish between apoptotic and necrotic cells, using an annexinV/propidium iodide assay and FACS analysis. 
</P><P>
Our next step was the light controlled expression of our construct 15, which contains tetO-ePDZb1-mCherry-BaxE184. This construct is regulated by the red light dependent PhyB construct 022, which contains tetR-PIF6 and PhyB-VP16. We illuminated our cell cultures with red light in order to excite mCherry. We then measured the emission and correlated it to the concentration of mCherry. Analysis of emission intensity showed that biosynthesis of mCherry was successful, when mCherry delivered an emission signal.
</P><P>
Our control culture containing only construct 15 and no photoactivatable red-light-switch showed only insignificant emission values, so we can conclude that no relevant mCherry concentrations were synthesized, hence there was no expression of construct 15.
</P><P>
The cultures containing the triple transfection of constructs 15, 022 and LX that were left in the dark and showed no significant mCherry concentrations either, only a high GFP concentration, as LX is expressed constitutively. 
</P><P><I>Figure 3: Confocal image of light- dependent expression of 15, 022 and LX containing the fluorescent reporters mCherry and GFP</I></P><P>
The cultures containing the same triple transfection, which were illuminated consecutively with red and then blue light showed high mCherry and GFP concentrations. As you can see in figure 3, both fluorescent proteins showed areas of highly concentrated emission, hence we can assume that both GFP and mCherry were located at their target site, the outer mitochondrial membrane. Therefore this data leads us to the conclusion that both our constructs have been expressed correctly.
</P><P><I>Figure 4: The diagram displays the relative mCherry production in our three different cell cultures. As you can see the expression in both the control and the dark triple transfected cell culture was insignificantly low, while expression in the illuminated triple transfected cell culture was high. This effect is likely to be caused by the red-light induced expression of construct 15 as intended. </I></P><P>
As one of our next steps we will test an alternative apoptotic protein. In these experiments Bax will be replaced by the proapoptotic protein PUMA in construct 15. This will show the strength of apoptotic induction by PUMA in comparison to Bax based on both constitutive and light-dependent expression. In order to distinguish clearly between apoptosis and necrosis we also plan to conduct an apoptotic assay as described above.
</P><H2 id="Team_Red">Team Red Results </H2><P>
The task of team red was the construction of the red-light-switch system based on Phytochrome B. Its purpose is the red-light-induced synthesis of hBAX including a fluorescent marker, mCherry and a PDZ-domain.
</P><P>
The first part of the construct we built was the fusion protein of PhyB and the transcription factor VP16, which allows red-light-induced recruitment of the DNA polymerase II. The two components were isolated via enzymatic restriction, made compatible via PCR and then fused together in a Fusion PCR. 
</P><P>
Test restrictions showed acceptable results, yet the sequencing revealed that our product was incorrect. We suppose that our fusion PCR was faulty and our allegedly correct products were actually oligomers of our used primers.

<A href="https://static.igem.org/mediawiki/2016/c/cd/T--duesseldorf--PhyB%2BVp16.jpg">PhyB+Vp16 gel picture</A></P><P>
The second part of the construct was tetR+PIF6, which would bind PhyB after red-light induction. These two components could be directly isolated together via PCR. Only an illegal restriction site located at the end had to be removed beforehand. We cut the part with the corresponding restriction enzymes EcoRI and PstI and then conducted the PCR, using the primer to alter the sequence by one base pair and so removing the restriction site. The construct was then cloned with a Fry 2 Promotor into a yeast shuttle backbone (pTUM100) to transform <I>S. cerevisiae</I> and test the expression. The construct was sequenced and found to be correct.

<A href="https://static.igem.org/mediawiki/2016/8/82/T--duesseldorf--tetR%2BPif6_Sequencing.pdf">Sequencing tetR+Pif6</A></P><P>
The third part of our construct was tetO+Pmin, which is a Vp16 dependent promotor and was produced via PCR. Sequencing showed that it was correct, but the part can not be tested without a functioning PhyB+Vp16.
<A href="https://static.igem.org/mediawiki/2016/7/7b/T--duesseldorf--tetO_Sequencing.pdf">Sequencing tetO+Pmin</A></P><H2 id="Yeast">Team yeast results </H2><P>
The task of team yeast was to proof our concept. In order to do that, we planned to build and express the entire construct in yeast. The 3A-assembly method was used to build each construct. In order to be able to do so we splitted the blue-construct into four seperate constructs.
</P><P>
The first construct is a constutive promoter, <A href="http://parts.igem.org/Part:BBa_K530008">BBa_K530008</A> TDH3 , fused with a <A href="”http://parts.igem.org/Part:BBa_K165002”">BBa_165002</A> kozak rbs sequence ,  which is essential for the expression and validation of our Lov2-GFP-Tom5 fusion-protein.
</P><P>
The second construct consists of our Lov2-eGFP-Tom5, synthesized by IDT and the <A href="http://parts.igem.org/Part:BBa_K530019">HH01 terminator</A>. This construct was analysed by an analytic digest and was shown to be correct. 
</P><P>
In the third construct we fused an arabinose inducible promoter <A href="http://parts.igem.org/Part:BBa_I0500">AraC</A> with a <A href="http://parts.igem.org/Part:BBa_K165002">Kozak-RBS-sequence</A>. This construct would then control expression of the fourth construct, which is localized downstream.
</P><P>
The fourth and last partial construct consists of the IDT synthesized PDZ-mCherry-BaxS184E and  the <A href="http://parts.igem.org/Part:BBa_K530019">HH01 terminator</A>.
</P><P>
In fusion, the first two constructs would have played an important role for the validation of the part that is responsible for the localization to the mitochondria in yeast cells. The constructs three and four also would have been fused together in order to control the expression of the lethal Bax protein through an inducible promoter, that can be induced independently from all other promoters in our construct. 
</P><P>
The  final construct would then be isolated via gel purification and inserted into one of these yeast shuttle vectors: <A href="http://parts.igem.org/Part:BBa_K1680015"> pRS315 </A>, <A href="http://parts.igem.org/Part:BBa_K801000">pTUM100</A> which we received from iGEM-Team Tübingen, and afterwards transformed into yeast cells to test the entire and the partial construct.
</P><H4 id="References">References</H4><SUP>[1]</SUP>= From beer to eternity – how yeast is revealing cancer’s secrets, Kat Arney, <A href="http://scienceblog.cancerresearchuk.org/2013/06/21/from-beer-to-eternity-how-yeast-is-revealing-cancers-secrets/">http://scienceblog.cancerresearchuk.org/2013/06/21/from-beer-to-eternity-how-yeast-is-revealing-cancers-secrets/ </A> 18.10.2016  <SUP>[2]</SUP>= L. H. Hartwell's Yeast: A Model Organism for Studying Somatic Mutations and Cancer
By: Leslie Pray, Ph.D. © 2008 Nature Education 
Citation: Pray, L. (2008) L. H. Hartwell's yeast: A model organism for studying somatic mutations and cancer. Nature Education 1(1):183<SUP>[3]</SUP>= Muller K., Engesser R., Timmer J., Zurbriggen MD., Weber W.(2014), Orthogonal Optogenetic Triple-Gene Control in Mammalian Cells. Pub ACS Vol.3 (11), pp 796–801. DOI: 10.1021/sb500305v<SUP>[4]</SUP>= Resendes KK. (2015), Using HeLa cell stress response to introduce first year students to the scientific method, laboratory techniques, primary literature, and scientific writing. The International Union of Biochemistry and Molecular Biology, Vol.43(2), pp 110-120. DOI: 10.1002/bmb.20852






</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>