<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Duesseldorf_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Duesseldorf/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content"><DIV class="nav"><TABLE style="margin: 0; border: none; width: 100%;"><TBODY><TR style="border:none;"><TD style="border:none;"><UL><LI><A href="https://2016.igem.org/Team:Duesseldorf">Home</A></LI><LI class="dropdown"><A href="#" style="color: white;">Project</A></LI><LI><A style="color: white;" href="https://2016.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Proof">Results</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Future_Applications">Future Applications</A></LI></UL><LI><A href="https://2016.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach#HP">Education</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach#IHP">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI></TD></TR><TR style="border:none;"><TD style="border:none;">
                     Heinrich-Heine University 
                     Universitätsstraße 1 
                     40225 Düsseldorf  
                     Design by Marvin van Aalst </TD></TR></TBODY></TABLE></DIV><DIV class="page"><DIV class="paper"><DIV id="Navigation" class="navigation"><UL class="topnav"><LI><A href="https://2016.igem.org/Team:Duesseldorf">Home</A></LI><LI><A style="color: black;" href="https://2016.igem.org/Team:Duesseldorf/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Proof">Results</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Future_Applications">Future Applications</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach#HP">Education</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Outreach#IHP">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Duesseldorf/Experiments">Experiments</A></LI></UL></DIV><DIV class="article"><H1 style="font-size:70px">Light on - Cancer gone</H1><DIV class="attention"><H1 style="color:white;">An Optogenetic System to Induce Apoptosis in Cancer Cells</H1><P style="color:white">
The optogenetic induction of apoptosis in cell cultures (HeLa and CHO) serves as a model for the future application in vivo. The application of optogenetic switches enables us to induct extremely precise and highly regulated elimination of malignant cells by combining the precision of light with the accuracy of viral vectors. 
The sequential utilization of two optogenetic switches, namely a Phytochrome-based gene expression system and a LOV2-based switch needed for the localization of apoptotic proteins to the outer mitochondrial membrane allows the attainment of a very high level of spatiotemporal specificity for the activation of apoptosis.
</P><UL><LI><A style="color:white;" href="#Cancer">Every second counts</A></LI><LI><A style="color:white;" href="#Phytochrome">Mechanism of the Phytochrome-based expression system</A></LI><LI><A style="color:white;" href="#Promoter">Expression of fusion proteins utilizing a constitutive promoter</A></LI><LI><A style="color:white;" href="#LOV2">The LOV2-based optogenetic switch allows localization of apoptotic proteins to the outer mitochondrial membrane</A></LI><LI><A style="color:white;" href="#Why_BAX">Why do we use BaxS184E?</A></LI><LI><A style="color:white;" href="#Special_LOV">What makes our LOV2 special?</A></LI><LI><A style="color:white;" href="#ePDZ">ePDZ-b1 binding to AsLOV2pep</A></LI><LI><A style="color:white;" href="#Our_final_construct">Our final construct</A></LI><LI><A style="color:white;" href="#References">References</A></LI><LI><A style="color:white;" href="https://2016.igem.org/Team:Duesseldorf/Test3">TRY IT YOURSELF!</A></LI></UL></DIV><H2 id="Cancer">Every second counts</H2><P>
From the beginning, the goal of OPTOPTOSIS was to fight cancer and to ameliorate commonly known and used cancer therapies. Decreasing the patient's suffering and improving their quality of living was our driving incentive. </P><P>

Treatments with state of the art cancer therapies are painful for the patients and they do not only have to suffer from the disease, but also from the side effects of the therapy which may last up to a lifetime.
The number of mortal cancer cases worldwide sums up to 8.2 million deaths per year. 
That means, every four seconds one person dies. 
Current <A href="https://2016.igem.org/Team:Duesseldorf/Therapies"> cancer therapies</A> fail in stopping the mortality rate. 
</P><P>
Our innovative approach can help patients to recover less painfully by specifically reestablishing the lost function of self-induced cell death in cancer cells. 
</P><P>
The process of apoptosis can be learned <A href="https://2016.igem.org/Team:Duesseldorf/Apoptosis"> here. </A></P><H2 id="Phytochrome">Mechanism of the Phytochrome-based expression system</H2><P>
Phytochrome B (PhyB) is the main functional element of the first optogenetic switch. PhyB is an optogenetic protein derived from <I>Arabidopsis thaliana</I>(see image 1). It is required for control of floral induction and germination, but was found to be useful as an element in the toolkit of synthetic biology. As a photosensory protein, it changes its conformation in response to red (activating) or far-red light (deactivating). The binding of interacting factors such as PIF6 (phytochrome interacting factor 6), also derived from <I>A. thaliana</I>, can only take place in the active state conformation after irradiation with red light. 
</P><P>
The protein requires a chromophore in order to react to light. The natural chromophore of PhyB is Phytochromobilin, a tetrapyrrole. This molecule cannot be biosynthesized by mammalian cells, but an alternative chromophore namely Phycocyanobilin extracted from Cyanobacteria can be delivered to the cells artificially. <SUP>[1]</SUP></P><P>
There are two ways by which mammalian cells can be supplied with Phycocyanobilin: It can be synthesized by a artificially introduced biosynthesis-pathway in the cells themselves or it may be  administered to the patients. 
</P><P>
Structurally Phycocyanobilin forms a covalent bond to the photosensory domain at the N-Terminus of PhyB which is responsible for conformational change upon photoexcitation. As noted above PhyB transits into its PhyBfr-conformation (active state) in response to red light (λ = 660nm) and binds to PIF6. The rapid process of binding can be accomplished in 6.9 seconds.<SUP>[2]</SUP><SUP>[3]</SUP><SUP>[4]</SUP><SUP>[5]</SUP></P><P><I>Image 1: Our red light switch is derived from </I>Arabidopsis thaliana
</P><P>
In our system we have created fusions of the N-terminus of PIF6  with tetR (tetracycline repressor), which constitutively binds the operator tetO upstream of a minimal promoter (Pmin). We have also fused PhyB to the transcription factor VP16. When the red light switch is activated, VP16 is recruited to the promoter region, so that the vicinity of VP16 to the promoter region allows initiation of transcription. Far-red light (λ = 740nm) is applied to the system in order to deactivate the switch. Under far-red light PhyB reverts back to its PhyBr-state (inactive) and interaction with PIF6 is terminated. Termination of binding requires 46.9 seconds. <SUP>[5]</SUP><SUP>[4]</SUP></P><P><I>Figure 1: The phytochrome-based expression system</I></P><P>
The PDZ-mCherry-Bax S184E construct, which expression is regulated by the PhyB-based switch, represents a component of the second optogenetic switch that is based on the photosensory protein LOV2. Bax S184E lays in a fusion with the fluorescent protein mCherry and the Jα-binding PDZ-domain. In contrast to wildtype Bax, the mutant Bax S184E was utilized in our system <SUP>[6]</SUP>, since this mutation makes pore formation dependent on recruitment of Bax to the OMM (outer mitochondrial membrane) via activation of our blue-light switch. </P><P><I>Figure 2: Expression of the component of the LOV2-based optogenetic switch</I></P><H2 id="Promoter">Expression of fusion proteins utilizing a constitutive promoter</H2><P>
Another construct needed for the LOV2-based optogenetic switch is expressed constitutively in the cells. For this purpose, expression of this construct is brought under control of the pSV40 viral promoter.
</P><P>
Our second blue-light switch is a fusion protein and consists of the mitochondrial anchor TOM5 (translocase of the outer membrane 5), the fluorescent protein GFP (green fluorescent protein) and the optogenetic protein LOV2 (light-oxygen- voltage-sensing 2) derived from <I>Avena sativa</I> (see fig. 3). The C-terminus of LOV2 contains the so called Jα-helix (see fig. 3), which allows binding with ePDZ (see fig. 2).
</P><P><I>Figure 3:  Constitutive expression system for the expression of the LOV2 -based optogenetic switch </I></P><P>
TOM5 is a mitochondrial protein that is responsible for recognition and initial import of all proteins directed to the mitochondria. TOM5 serves as an anchor of our blue light switch to the OMM.
</P><P>
LOV2 functions in photosensing in its natural context allowing subsequent activation of regulatory pathways in response to light. We used the ortholog from <I>Avena sativa</I>. An important structure of the domain is its Jα-helix. In our approach, we use the LOV2 domain to control the localization of the apoptotic construct to the outer mitochondrial membrane. More precisely, the structure of the double-mutated version LOV2pep allows the binding of the ePDZ domain (see fig. 2).  
[For more information on what our LOV2pep makes special, <A href="#AsLOV2pep">click here</A>.]
</P><H2 id="LOV2">The LOV2-based optogenetic switch allows localization of apoptotic proteins to the outer mitochondrial membrane</H2><P>
Once both components of the LOV2-switch have been synthesized and brought to their target site they are ready to interact. In order to absorb light, the LOV2 protein requires the chromophore FMN, a compound naturally produced in mammalian cells. <SUP>[8]</SUP></P><P>
Our LOV2 is flanked with α-helices on the N- and C-terminals. Upon photoexcitation with blue light (λ = 473nm) the C-terminal Jα-helix from the LOV2-core undocks and unfolds slightly <SUP>[8]</SUP>.
</P><I> Image 2: Our blue light switch is derived from </I>Avena sativa
<P>
The exposure of the Jα-helix allows the interaction with a binding partner. The addition of a  peptide epitope tag enables the Jα-helix to bind to ePDZ. ePDZ is derived from mice, while LOV2 is derived from <I>Avena sativa</I><SUP>[8]</SUP>
[For more information on ePDZ click here]
LOV2 is bound to the OMM (outer mitochondrial membrane) due to its mitochondrial anchor TOM5. Therefore, binding between Jα and ePDZ causes recruitment of BaxS184E to the OMM (fig. 4).
</P><P><I>Figure 4: The LOV2-based optogenetic switch is activated by blue light</I></P><P>
Here BaxS184E forms pores in the OMM allowing the release of cytochrome c, inducing apoptosis. So BaxS184E is only capable of fulfilling its function when its expression has firstly been activated by the PhyB-based switch and secondly, when it has been recruited to the mitochondria by activation of the LOV2-based switch. An autonomous localization of BaxS184E to the mitochondria does not occur. Thus BaxS184E will only be found at its target site after activation of the blue light regulated switch. The fluorescent proteins GFP and mCherry serve as markers.
</P><H2 id="Why_BAX">Why do we use BaxS184E?</H2><P>
One natural apoptotic protein in humans is called hBax. Since OPTOPTOSIS should be applied as a therapy against human disease in the future, we searched for a suitable substitution and found one. This powerful activator of the intrinsic apoptotic pathway would not allow regulated killing of cancer cells if applied in our system, because it would induce apoptosis independently. Hence we utilized the mutant Bax S184E mutant, as described above. 
</P><P>
Bax S184E is not as deadly as the natural human Bax because it is binding less effectively to the mitochondrial membrane due to a substitution of  serine to glutamate in the residue S184 in the C-terminus of Bax. Due to the substitution at S184 the residue cannot be phosphorylated resulting in conformational changes decreasing the ability of the protein to integrate into the OMM. Also, an autonomous localization of Bax S184E to the mitochondria does not occur, because its ability to localize to the OMM is impeded. In order to induce apoptosis Bax S184E has to be brought to the vicinity to the OMM. <SUP>[7]</SUP></P><H2 id="Special_LOV">What makes our LOV2 special? AsLOV2pep mutant</H2><P>
For our project, we used a double mutant version of LOV2 derived from <I>Avena sativa’s</I> phototrophin 1 fused with a peptide epitope on the Jα-helix called AsLOV2pep. We used this mutant form of LOV2, because it has a higher dark-state binding affinity to the Jα-helix. The higher affinity is caused by the substitutions T406A and T407A. Through these changes the possibility that the Jα-helix epitope is exposed during dark state, is greatly reduced <SUP>[8]</SUP><SUP>[9]</SUP>.  
</P><P><I>Figure 5: Model of AsLOV2pep mutant </I><I>AsLOV2 structure (Protein Data Bank: 2v0u) showing the location of the ePDZ epitope (green) and the caging mutations used in the study of Strickland et al. (2012)<SUP>[8]</SUP></I></P><H2 id="ePDZ">ePDZ-b1 binding to AsLOV2pep</H2><P>
The variation ePDZ-b1 is used in our construct, this PDZ-domain is engineered to have a higher affinity to the binding peptide. The binding kinetics were measured and appeared to have a 500-fold increased affinity. 
The structure of the Erbin PDZ bound to a peptide (PDB entry 1MFG). The N and C termini are indicated. The positions for the new termini of the circularly permutated PDZ (cpPDZ) are shown with a triangle and residue numbers. Right shows the surface of the PDZ domain with the peptide as a stick model, illustrating the shallow binding pocket. 
</P><H2 id="Our_final_construct">Our final construct</H2><H1 id="“TRY&quot;"><A href="https://2016.igem.org/Team:Duesseldorf/Test3">TRY IT YOURSELF!<P>
Klick me!</P></A></H1><H2 id="References">References</H2><SUP>[1]</SUP> Müller K., Zurbriggen MD., Weber W. (2014), Control of gene expression using a red- and far-red light-responsive bi-stable toggle switch. Nature Protocols 9, pp 622-632. doi:10.1038/nprot.2014.038
<SUP>[2]</SUP> Müller K, Engesser R, Timmer J, Nagy F, Zurbriggen MD, Weber W (2013). Synthesis of phycocyanobilin in mammalian cells. Chem. Commun. 49, 8970. doi:10.1039/c3cc45065a 
<SUP>[3]</SUP>= Khanna,R., Huq,E., Kikis,E.A., Al-Sady,B., Lanzatella,C. and Quail,P.H. (2004) A novel molecular recognition motif necessary for targeting photoactivated phytochrome signaling to specific basic helix-loop-helix transcription factors. Plant Cell, 16, 3033–3044.  
<SUP>[4]</SUP> Levskaya,A., Weiner,O.D., Lim,W.A. and Voigt,C.A. (2009), Spatiotemporal control of cell signalling using a light-switchable protein interaction. Nature, 461, 997–1001.
<SUP>[5]</SUP> Buckley CE, Moore RE, Reade A, Goldberg AR, Weiner OD, Clarke JDW (2016). Reversible Optogenetic Control of Subcellular Protein Localization in a Live Vertebrate Embryo. Dev Cell 36(1), pp 117-126. doi:10.1016/j.devcel.2015.12.011
<SUP>[6]</SUP>= Müller et al.(2013) A red/far-red light-responsive bi-stable toggle switch to control gene expression in mammalian cells, Nucleic Acids Research, 2013, Vol. 41, No. 7 e77 doi:10.1093/nar/gkt002 
<SUP>[7]</SUP> Wang Q., Sun SY, Khuro F, Curran WJ, Deng X (2010). Mono- or double-site phosphorylation distinctly regulates the proapoptotic function of Bax. PLos One, p .doi: 10.1371/journal.pone.0013393.
<SUP>[8]</SUP> = Strickland D., Lin Y., Wagner E., Hope M., Zayner J., Antonious C., Sosnick T.R., Weiss E.L., Glotzer M. (2012), TULIPS: tunable, light-controlled interacting protein tags for cell biology, Nature Vol.9(4), doi:10.1038/nmeth.1904
<SUP>[9]</SUP> = Hallet RA, Zimmermann SP, Yumerefendi H, Bear JE, Kuhlmann B (2016). Correlating in vitro and in vivo Activities of Light Inducible Dimers: a Cellular Optogenetics Guide. ACS Synth Biol. 5(1), pp 53-64. doi: 10.1021/acssynbio.5b00119 
<SUP>[10]</SUP> = Huang J, Koide A, Makabe K, Koide S (2008). Design of protein function leaps by directed domain interface evolution. Proc Natl Acad Sci USA 105, pp 6578-6583. doi: 10.1073/pnas.0801097105
</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>