"
Page
Discussion
View source
History
teams
Log in
 
Team:ULB-Brussels/mat
From 2011.igem.org
Home Project Modelling Human practice Discussion Parts Safety Team Sponsors
Introduction Materials & Method Results Experimental conclusion
Materials & Method
Strains:
E. coli strains:
MC1061 : F− araD139, Δ(ara-leu)7696,galE15, galK16, Δ(lac)X74, rpsL (Strr), hsdR2 (rK−mK+), mcrA mcrB1
DG1(Delphi genetics): mcrA Δ (mrr-hsdRMS-mcrBC, modification-,
restriction-) F80lacZDM15 Δ lacX74 recA1 araD139 Δ (ara-leu)7697 galU galK rpsL endA1 nupG
TOP10 (Invitrogen) : F- mcrA Δ(mrr-hsdRMS-mcrBC) φ80lacZΔM15 ΔlacX74 nupG recA1 araD139 Δ(ara-leu)7697 galE15 galK16 rpsL(StrR) endA1 λ-
MG1655 : F- λ- ilvG- rfb-50 rph-1
MG1655 ΔtldD::frt-cm-frt
Saccharomyces cerevisiae strain:
23344C :ura3
Plasmids:
pCP20:
pCP20 has the Flp recombinase gene repressed by CI857ts which is not functional at a temperature higher or equal to 42°C
pFL44S :
pFL44S is a shuttle plasmid between yeast and bacteria. It has the ampicillin resistance gene and the bacterial origin of replication colE1. For selection in yeast it has the ura3 gene. It has also the 2micron yeast origin of replication.
pKD46:
pKD46 expresses the Red system under control of the well-regulated promoter pBAD (induced by arabinose and repressed by glucose). The replication origin of the plasmid is also thermo sensitive. It is not replicated at a temperature higher or equal to 42°C.
pSB1A3:
pSB1A3 is one of the standard iGEM plasmid which has a resistance gene to ampicillin. It is composed of four unique restriction sites:
EcoRI (prefix)
XbaI (prefix)
SpeI (suffix)
PstI (suffix)
PCR Topo® XL plasmid (Invitrogen):
This plasmid is used in the Topo® XL PCR Cloning Kit. For the cloning the plasmid is linearized and topoisomerase l is activated. The vector includes the following features:
ccdB gene for positive selection
Kanamycin and Zeocin™ resistance genes
EcoRI sites flanking the PCR product insertion site for easy excision of inserts
M13 forward and reverse primer sites for sequencing
Primers:
The following primers were produced by Sigma-Aldrich (option desalt)
Yeast cloning:
PKD46-FOR:
5’-gagcaaaaggccagcaaaaggccaggaaccgtaaaaaggctgccacctgcatcgatttat-3’
PKD46-REV:
5’-cgtgagttttcgttccactgagcgtcagaccccgtagaaagagttttcgttccactgagc-3’
PFL-FOR:
5’-gcctttttacggttcctggc-3’
PFL-REV:
5’-tttctacgggggtctgacgc-3’
PCP-FOR:
5’-tggctcttgtatctatcagtgaagcatcaagactaacaaatcagccaaacgtctcttcag-3’
PCP-REV:
5’-ggggctgtatgcacaaagcatcttctgttgagttaagaacttatatgcgtctatttatgtagg-3’
In green are the 40 homologous nucleotides need for the yeast cloning.
Yeast cloning verification:
FLP/CI-FOR:
5’-acatggcgagttttgacgag-3’
FLP/CI-REV:
5’-accacactagagaacatactg-3’
pINDEL sequencing:
pID-seq1:
5’-aggatcttcacctagatcctt-3’
pID-seq2:
 5’-gatgggctagtcaatgataatta-3’
pID-seq3:
5’-ccgttacgtaggtaggaatc-3’
pID-seq4:
5’-agatggggatggggcagtc-3’
pID-seq5:
5’-gatttcggatcaacgttcttaat-3’
pID-seq6:
5’-caatcactttcgtctactcc-3’
pID-seq7:
 5’-ccagatatttcgccgcgac-3’
pID-seq8:
5’-cggggccagcaaaaaatcca-3’
pID-seq9:
5’-ccctgatttttcaccacccc-3’
pID-seq10:
5’-cttccgaaaatgcaacgcga-3’
Biobricks FRT'-CM-FRT':
FRT’-CM-FRT’-FOR:
5’-gaagttcctatactttttagagaataggaacttcgttgatcgggcacgtaagagg-3’
FRT'-CM-FRT'-REV :
5’-gaagttcctattctctaaaaagtataggaacttcttattacgccccgccctgcc-3’
In green is the frt’ sequence which is not homologous with the chloramphenicol gene.
TOPO-FRT'-CM-FRT'-FOR:
5'-tccggcaaaaaagggcaaggtgtcaccaccctgcccttttcgccagtgtgctggatttc-3'
TOPO-FRT'-CM-FRT'-REV:
5’-tgcagcggccgctactagtactctagaagcggccgcgaatgatggatatctgcagatttc-3’
In red are the iGEM restriction sites (prefix and suffix) and in green are the mutation made to suppress the EcoRI restriction site (in the Topo® XL PCR plasmid).
Biobrick FRT'-CM-FRT' sequencing (primer M13 as described in Topo® XL PCR Kit):
FRT’-CM-FRT’-SEQ-FOR:
5’-gaggaaacagctatga-3’
FRT’-CM-FRT’-SEQ-REV:
 5’-gaccggcagcaaatg-3’
Insertion primers
LaCZ-FRT’-CM-FRT’-FOR:
5’- gcaacgcaattaatgtgagttagctcactcattaggcacccttgcccttttttgccgga -3’
LaCZ-FRT’-CM-FRT’-REV:
5’-cggtcggattctccgtgggaacaaacggcggattgaccgtagcgaattatgcagatatccat-3’
Media:
Bacteria, Luria-Bertani (LB):
10g/l tryptone;
5g/l yeast extract;
5g/l NaCl
(+12g/l of Agar for solid medium)
Yeast:
Rich medium:
10g/l of Yeast Extract
10g/l of bactopeptone
20g/l of glucose
Minimal medium:
20g/l of glucose
0.67 g/l Difco Yeast Nitrogen Base w/o Amino acid
15g/l of agar
Solutions:
Mg(SO4)2 (10-2M)
TSS composed of (for 10ml): 8,5ml of LB medium; 500µl of dimethylsulfoxide; 500µl of MgCl2 (1M); 1g polyethylene glycol 8000
Isopropyl-ß-D-galactoside (IPTG) (1000X): 1M
Dimethylsulfoxide =DMSO (Sigma-Aldrich D2650)
5-bromo-4-chloro-3-indolyl-beta-D-galactopyranoside (X-gal) 40mg/ml
Antibiotics (stock):
Ampiciline 100mg/ml
Kanamycine 50 mg/ml
Chloremphenicol 20 mg/ml
Tetracycline 15mg/ml
The antibiotics were diluted 1000X in appropriate medium.
Enzymes:
EcoRI, XbaI, SpeI, PstI, BstXI 10U/µl (Roche)
rAPID Alkaline Phosphatase 1U/µl (Roche)
Taq polymerase Gotaq 5U/µl (Promega)
Phu polymerase Phusion 2U/µl (Finnzymes)
T4 DNA ligase 1U/µl  (Roche)
PCR:
Finnzymes, Phusion® Hot Start High-Fidelity DNA polymerase (Ref: f-540S):
Reaction solution:
10µl of buffer5X
1µl of dNTPs (10mM each)
0.25µl of primer 1 (20µM)
0.25µl of primer 2 (20µM)
1µl of DNA
0.5µl of Taq polymerase Phusion
1.5µl of DMSO (optional)
37 (or 35.5 if DMSO) µl of H20
Program:
1:98°C 30sec
2:98°C 10sec
3:58°C 30Sec
4:72°C X min, 30sec/kb (30 cycles from step2)
5:72°C 10min
Promega, GoTaq® Hot Start Polymerase (ref: M5001):
Reaction solution:
10µl of buffer10X (green or greenless)
1µl of dNTPs (10mM each)
3µl of MgCl2
0.5µl of primer 1 (20µM)
0.5µl of primer 2 (20µM)
1µl of DNA
0.25µl of Taq polymerase Phusion
1.5µl of DMSO (optional)
33.75 (or 32.25 if DMSO) µl of H20
Program:
1:94°C 5min
2:94°C 30sec
3:60°C 30sec
4:72°C X min, 1min/kb (30 cycles from step2)
5:72°C 10min
Mini prep kit:
 Zymo Research, ZyppyTM Plasmid Miniprep Kit (ref: D4036)
Midi prep kit:
Sigma-Adrich, GenEluteTm HP Plasmid Midiprep Kit (ref:Na0200-1KT)
Gel purification kit:
 Sigma-Aldrich, GenEluteTm PCR Clean-up (ref:NA1020)
Yeast transformation by lithium acetate : Gietz version. (culture for 100ml)
Cells preparation:
From one fresh dish make a preculture of your strain on rich medium (10ml) and incubate it overnight at 30°C and preincubate 100ml of rich medium at 30°C.
In the morning dilute your preculture in the rich medium and incubate it at 30°C during three hours in a way your OD600nm is towards 0.5-0.7.
Centrifuge your culture at 3500g for 5 minutes and remove supernatant.
Resuspend pellet in 1ml steril H20, transfer in a 2ml eppendorf, centrifuge 10 seconds, remove supernatant, resuspend in 1ml water, recentrifuge 10 seconds and remove water.
Resuspend pellet in 1ml AcLi/TE, centrifuge 10 seconds, remove supernatant and resuspend the pellet in 0.5ml AcLi/TE.
DNA transformation:
At 50mL of suspension add in order:
5mL sonicated DNA at 10mg/ml, mix delicately.
±1mg DNA to transform, mix delicately.
300ml PEG/AcLi/TE 40% solution, homogenize.
Incubate 30 minutes at 30°C.
Incubate 15 minutes at 42°C
Centrifuge 10 seconds, empty the tube, add 1ml water and directly empty the tube. Then resuspend the cells in 100µl of water.
Spread on yeast minimal plates and incubate at 30°C for 3 days.
Used Solutions:
10x AcLi: 1M AcLi pH 7.5.
10x TE: 0.1M Tris pH 7.5: 0.01M EDTA pH 7.5.
PEG 4000 50%.
AcLi/TE: 1/10 vol 10X AcLi + 1/10 vol 10x TE in steril water.
PEG 40%/AcLi/TE: 8/10 vol PEG 4000 50% + 1/10 vol 10x AcLi + 1/10 vol 10x TE.
Yeast genomic DNA Extraction:
Resuspend your colonies in 100µl lithium acetate 200mM + SDS 1%
Incubate 15 minutes at 70°C.
Add 300µl of ethanol 96%, vortex briefly.
Centrifuge 3 minutes at 15,000g.
Remove supernatant.
Add 300µl of ethanol 75%.
Centrifuge 3 minutes at 15,000g.
Remove supernatant and dry pellet.
Dissolve DNA pellet in 100µl distilled water.
Centrifuge 1 minute at 15,000g.
Transfer supernatant in a new eppendorf. Store at -20°C.
Chemo-transformation (DG1 strain from delphigenetics):
Start thawing the competent cell on crushed ice.
Add 50µl of thawed competent cells and 1-2µl of the resuspend DNA to the labelled tubes. Make sure to keep the competent cells on ice.
Incubate the cells on ice for 30 minutes.
Heat shock the cells by immersion in a pre-heated water bath at 42°C for 60 seconds. A water bath improves heat transfer to the cells.
Incubate the cells on ice for 5 minutes.
Add 200µl of LB.
Incubate the cells at 37°C for 2 hours while the tubes are rotating or shaking. Important: 2 hours recovery time helps in transformation efficiency, especially for plasmids with antibiotic resistance other than Ampicillin.
Label two petri dishes with LB agar and the appropriate antibiotic(s) with the part number, plasmid, and antibiotic resistance. Plate 20µl and 200µl of the transformation onto the dishes, and spread. This ensures that you will be able to pick out a single colony.
Incubate the plate at 37°C for 12-14 hours, making sure the agar side of the plate is up. If incubated for too long the antibiotics start to break down and un-transformed cells will begin to grow. This is especially true for Ampicillin – because the resistance enzyme is excreted by the bacteria, and inactivates the antibiotic outside of the bacteria.
Preparation of ready-to-use electrocompetent cells:
Dilute 100x an overnight culture in LB medium and incubate at 30°C with shaking until OD600nm reaches 0.4.
Centrifuge in a cold rotor at 3500gand 4°C for 15 minutes.
Keep cells at 4°C throughout the whole process.
Remove supernatant, resuspend in 20ml cold H20 centrifuge as above. Repeat this step twice.
Remove supernatant, resuspend in 2ml cold H20 centrifuge as above.
Remove final supernatant carefully, resuspend in 60µl cold water.
Store it on ice.
Electroporation:
Dialyse the DNA by placing it on a round 0.025µm Millipore dialysis filter to float in dH2O on a small petri dish for 20min.
Add dialysed DNA in 50µl of electrocompetent cells.
Put mixture in an electroporation cuvette.
Electroporate at 2,500 volt, 200Ω and 25µFD.
Add quickly 1ml of LB.
Put the bacteria 1 hours at 37°C.
Plate the volume you need on dish with appropriate antibiotics.
TSS transformation in bacteria:
Dilute 100x an overnight culture in 1ml of LB
Put the cells at 37°C with agitation during 2 hours.
Centrifuge in a cold rotor at 3500g for 5 minutes.
Remove supernatant.
Resuspend the pellet in 100µl of cold TSS.
Add 100ng of plasmid.
Incubate on ice 30 minutes.
Add 100µl of LB.
Incubate at 37°C (30° for thermosensitive plasmid) during 1 hour.
Streak 100µl on dishes with appropriate antibiotics.
Incubate overnight at appropriate temperature.
Topo cloning:
Invitrogen, TOPO® XL PCR Cloning Kit with One Shot® TOP10 Electrocomp™ E. coli : K4700-20
Direct cloning:
Restriction:
The first step is to realise a PCR to amplify the genes of interest with floating restriction sites.
Digest the PCR product and the vector by the appropriate restriction enzymes.
Incubate one hour at 37°C
Restriction mix for the PCR products:
Xµl of PCR products.
1µl of each restriction enzyme.
2µl appropriate buffer 10X.
Add Yµl bi-distilled water to reach a final volume of 20µl.
(X corresponds to a necessary quantity in the reaction of ligation.)
Restriction mix for the vector:
200ng of vector.
1µl of each restriction enzyme.
2µl appropriate buffer 10X.
Add Xµl bi-distilled water to reach a final volume of 20µl.
Incubate one our at 37°C
Ligation:
The ligation reaction is made with 100ng restricted vector. The quantity of the PCR products is calculated according to the following formula: (100ng vector x PCR product size x 3)/(vector size) = PCR product quantity in ng.
Incubate the ligation reaction overnight at room temperature.
Electroporate the ligation on bacteria.
Ligation mix:
1µl Ligase.
2µl buffer (10x).
Add the two restrictions reactions in a final volume of 20µl.
Characterization of the repA101ts replication origin:
The MC1061 strain containing the pIN plasmid was grown overnight at 30°C in LB medium containing Amp (100 mg/ml).  After dilution (100-fold) in LB Amp medium, the MC1061/pIN strain was grown at 30°C to an OD600nm of about 0.1.  The culture was washed with LB and then resuspended in LB medium.  The culture was divided in two sub-cultures, one was grown at 30°C and the other at 42°C. At the times indicated in the Figure 10, samples were withdrawn and appropriate dilutions were plated on LB and LB Amp (100 mg/ml). When OD600nm is around 1.2, cultures were diluted 10-fold in the appropriate pre-warmed medium to maintain the bacteria in logarithmic growth. Plates were incubated overnight at 30°C and the CFU/ml were determined. The percentage of bacteria containing the pIN plasmid was obtained by dividing the CFU/ml on LB Amp medium by the CFU/ml on LB medium and multipliate the results by 100.
Deletion test of pindel:
Transform pINDEL and pIN in MG1655DtdlD::FRT-cat-FRT strain. Select candidates on LB Ampicillin (100µg/ml) and Chloramphenicol (20µg/ml) plates at 30°C. Then streak a few clones on LB plates in a way to obtain isolate colonies and incubate plates overnight at 42°C. Check antibiotic resistance removal by restreaking colonies on LB and LB-chloramphenicol plates and incubate them overnight at 42°C.
Growth test on the strain MC1061 - pINDEL # 2:
Cultivate at 30°C the strain MC1061–pINDEL and MC1061–pIN overnight at 30°C in LB with Ampicillin (100µg/ml) and LB with Ampicillin (100µg/ml) and arabinose 1%. The next day, dilute the cultures in the same medium at an OD600nm of 0.01. Incubate the cultures at 30°C with shaking and measure the OD600nm every 30 minutes.
Insertion test of pINDEL:
pIN and pINDEL were transformed in MG1655 strain. Candidates were selected on LB Ampicillin (100µg/ml) plates at 30°C. Electrocompetent cells were prepared as described in material and methods except that the temperature of growth is 30°C and that when diluted cultures reached an OD600nm of 0.2, arabinose was added at a final concentration of 0.2% and then cultures were incubated at 30°C with shaking until OD600nm reached 1 before washing it with cold water.
Electroporate FRT’-Cm-FRT’ fragment flanked by sequences homologous to lacZ gene in electrocomptent cells as described in material and methods
Candidates were selected on LB Ampicillin (100µg/ml) and Chloramphenicol (20µg/ml) plates at 30°C.
Insertion of FRT’-CmR-FRT’ cassette in lacZ gene was checked by streaking a few candidates on LB X-gal (40 µg/ml) and IPTG (1mM) and incubate plates overnight at 30°C.
iGEM ULB Brussels Team - Contact us
Retrieved from " http://2011.igem.org/Team:ULB-Brussels/mat "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
