"
Page
Discussion
View source
History
teams
Log in
 
Team:ULB-Brussels/Results
From 2011.igem.org
Home Project Modelling Human practice Discussion Parts Safety Team Sponsors
Introduction Materials & Method Results Experimental conclusion
Results
Construction of the pINDEL plasmid:
The pINDEL plasmid is composed of 2 functional units: (i) the IN function which is composed of the gam, exo and bet genes coding for the l Red recombinase system (REF: Datsenko and Wanner, PNAS, 2000; Yu et al., PNAS 2000) and (ii) the DEL function which is based on the flp gene encoding the FLP site-specific recombinase and expressed at 42°C (REF: Datsenko and Wanner, PNAS, 2000; Yu et al., PNAS 2000).  The IN function allows the insertion of gene of interest coupled to an antibiotic resistance gene flanked of the FRT’ sites in the E. coli chromosome at a specific location (see below) or the replacement of an E. coli specific chromosomal gene by an antibiotic resistance gene flanked of the FRT’ sites. The antibiotic resistance gene allows the selection of the recombination event.  The DEL function will be used to get rid of the antibiotic resistance gene by site-specific recombination mediated by the FLP recombinase. 
Another feature of pINDEL is that its replication origin is not functional at 42°C due to the thermosensitivity of the RepA101Ts protein.  This protein initiates replication at the ori site in permissive conditions (30°C) and is inactive at 42°C.  As a consequence, replication of pINDEL stops at 42°C and the plasmid-copies are diluted with time in the bacterial population.  Thus, the growth at 42°C allows to get rid of the pINDEL plasmid as well as of the antibiotic resistance gene in the recombinant bacteria. 
To construct pINDEL, we have proceeded in 2 steps and used the high capacity of yeast to perform homologous recombination.  The first step was to recombine a large IN fragment containing the repA101Ts thermosensitive replication origin, the exo, gam and bet genes under the control of the pBAD promoter as well as the araC gene encoding the AraC transcriptional regulator with another large fragment containing a yeast replication origin, the yeast selection marker ura3 and the Amp resistance gene.  This first construct was named pIN.
In a second step, we recombined the DEL unit composed of the flp gene under the control of the l pR promoter and of the cI857 gene encoding the thermosensitive CI repressor of pR to the pIN plasmid in yeast to finally obtain the pINDEL plasmid. 
pIN assembly :
In a first PCR reaction, we used the pKD46 plasmid as template (REF: Datsenko and Wanner, PNAS, 2000).  The region containing the bacterial repA101Ts thermosensisitive replication origin, the exo, gam and bet genes under the control of the pBAD promoter as well as the araC gene encoding the AraC transcriptional regulator were amplified by PCR (Figure 1 A).  In a second PCR reaction, a large fragment of the yeast pFL44S plasmid containing the yeast replication origin, the yeast ura3 selection marker and the Amp resistance gene was amplified (Figure 1 B). Figure 1  A and B shows that the PCR reactions gave 2 fragments of appropriate size (3,699 bp and 5,229 bp for the PFL and IN fragments, respectively).
Figure 1  PCR amplifications of the PFL and IN fragments. (A) A fragment of 5,229 bp of the pKD46 plasmid was amplified by PCR as described in Materials and Methods using the PKD46-FOR and PKD46-REV primers. (B) A fragment of 3,699 bp of the PFL44S was amplified by PCR as described in Materials and Methods using the PFL-FOR and PFL-REV primers. 1: PCR amplification, Bl: negative control, SL: Molecular weight marker (Smart ladder).
 
The 2 linear fragments were transformed into the 23344c yeast strain and recombinant plasmids were obtained at high frequency.  Total DNA of candidates was extracted and transformed in the E. coli MC1061 strain.  The pIN candidates were then extracted by mini-prep and checked by restriction. The PFL and PKD46 fragments containing 1 and 2 EcoRI sites respectively, an EcoRI digestion was used to discriminate positive from negative candidates.  An empty vector (PFL44 fragment) would result in the generation of 1 DNA fragment while a recombinant plasmid would generate 3 DNA fragments (Figure 1 ).
Figure 2  EcoRI restriction of the pIN plasmid candidates.  Mini-prep of the pIN candidates were digested by EcoRI in the appropriate conditions.  1: pIN #1, 2: pIN #2, 3: pIN #8, SL: Molecular weight marker (Smart ladder).
 
Four positive candidates were obtained (3 of them are shown on Figure 2 ) and we selected the pIN #2 to construct pINDEL.
pINDEL assembly :
The pINDEL plasmid was obtained by recombining the flp gene under the control of the λ pR promoter and the cI857 gene to the pIN #2 plasmid in yeast. A fragment of 2,166 bp containing these 2 genes was amplified by PCR using the pCP20 plasmid as template (Figure 3 ) (Datsenko and Wanner, 2000).
Figure 3  PCR amplification of the PCP20 fragment. A fragment of XX bp of the pCP20 plasmid was amplified by PCR as described in Materials and Methods using the PCP-FOR and PCP-REV primers. 2: PCR amplification, Bl: negative control, SL: Molecular weight marker (Smart ladder).
 
The pIN #2 plasmid was linearized by restriction using BstX1 (data not shown).  As described for pIN construction, the 2 linear fragments were transformed into the 23344c yeast strain and recombinant plasmids were obtained at high frequency.  Total DNA of candidates was extracted and transformed in the E. coli MC1061 strain.  The pINDEL candidates were then extracted by mini-prep and checked by PCR amplification using primers complementary to the sequences flanking the BstX1 site in the pIN construct (Figure 4 ).  Positive candidates show an amplification of 2,547 bp.
Figure 4  PCR amplification of the DEL insert in pINDEL plasmid candidates. PCR reactions were performed as described in Materials and Methods using the FLP/CI-FOR and FLP/CI-REV primers. 1: pINDEL #2.1, 2: pINDEL #2.2, 3: pINDEL #2.3, B: negative control, SL: Molecular weight marker (Smart ladder).
 
We selected pINDEL #2.1 for further characterization and experiments.  At the time of writing, we have sent the plasmid for sequencing and are waiting for the data.
 
Construction of the FRT’-Cm-FRT’ biobrick (BBa_K551000) :
As a marker to select either gene disruption or gene insertion in the E. coli chromosome, we thought to use the BBa_J61025 biobrick containing the chloramphenicol resistance gene flanked by FRT’ sites. The Cm resistance gene can be subsequently excised by the FLP recombinase. 
After transformation of BBa_J61025 in the E. coli DG1 strain and selection on LB medium containing Amp, twelve transformants were streaked on LB Amp, LB Cm and LB containing the 2 antibiotics.  After ON incubation at 37°C, although the transformants were able to grow on LB Amp, they were unable to grow on neither on LB Cm nor on LB Amp Cm plates.  We repeated this experiment and obtained the same result confirming that the BBa_J61025 biobrick is not functional. We then performed a BLASTn (http://blast.ncbi.nlm.nih.gov/) with the DNA sequence provided by the Arkin lab in the Registry of Standard Biological Parts. The BLASTn result did not show any similarity between the BBa_J61025 sequence and the Cm resistance gene.  We therefore decided to build a novel biobrick containing the Cm resistance gene flanked by the FRT’ sites.
We started using a Cm resistance cassette provided by the Registry of Standard Biological Parts (Part: BBa_P1004). We transformed the BBa_P1004 plasmid designed by the Knight lab in DG1 and checked the Cm resistance.  We also compared the DNA sequence of BBa_P1004 to the Genbank database and retrieved the wild-type sequence of the chloramphenicol acetyl transferase gene (cat).  We thus decided to construct a novel FRT’-Cm-FRT’ biobrick using the Cm resistance gene of BBa_P1004.  This Cm resistance gene of BBa_P1004 was amplified by PCR using primers containing a mutated FRT’ sequence (Figure 5 A).  As the wild-type FRT sequence contains a Xba1restriction site, a mutation was introduced to mutate the Xba1 site in order to be able to use this biobrick in iGEM standards. After amplification, the PCR product was checked on a 1% agarose gel and showed the expected size (837 bp,Figure 5 B).
Figure 5  Construction of the novel FRT’-Cm-FRT’ biobrick.  (A) Sequence of the mutated FRT’ sites.  In bold, the Xba1 restriction site with the C->T mutation in bold. (B) The Cm resistance gene was amplified by PCR as described in Materials and Methods using BBa_P1004 as template and the FRT’-Cm-FRT’-FOR and FRT’-Cm-FRT’-REV primers.  1: PCR product obtained with a 100-fold dilution of the BBa_P1004 mini-prep, 2: PCR product obtained with a 1000-fold dilution of the BBa_P1004 mini-prep, B: negative control, SL: Molecular weight marker (Smart ladder).
 
The PCR product was cloned in a TOPO® plasmid. After transformation in the MC1061 strain and selection on LB Kan medium, 10 candidates were selected and checked by PCR amplification (Figure 6 ).
Figure 6  PCR amplification of the FRT’-Cm-FRT’ insert in TOPO®- FRT’-Cm-FRT’ plasmid candidates. PCR reactions were performed as described in Materials and Methods using using the M13-FOR and M13-REV primers. 1->10: TOPO®- FRT’-Cm-FRT’ candidates, Bl: negative control, SL: Molecular weight marker (Smart ladder).
 
The Topo-FRT’-CM-FRT’ #5 was selected for further characterization and sequencing. The sequence file is presented in Annex 1 .
 
The next step consisted in sub-cloning the FRT’-Cm-FRT’ in pSB1A3 plasmid. To this end, we designed primers complementary to the TOPO® sequences flanking the insert to be able to amplify the FRT’ sequences. However, the flanking sequences contain an EcoR1 restriction site. We therefore designed primers with a mutated EcoR1 sequence in order to comply with the iGEM standards (Figure 7 ).
Figure 7  Sequence of the TOPO® flanking the insertion site of the FRT’-Cm-FRT’ cassette. The sequence of the primers are indicated as well as the mutation (highlighted in green) that mutate the EcoRI restriction site sequence (highlighted in orange).
 
In addition, we added the IGEM prefix or suffix sequences to the 5’-end of the primers as a non complementary sequence. We obtained a PCR amplification of 889bp (Figure 8 ).
Figure 8  PCR amplification of the FRT’-Cm-FRT’ fragment cloned in the TOPO® vector. A fragment of 889 bp of the TOPO®- FRT’-Cm-FRT’  plasmid was amplified by PCR as described in Materials and Methods using the topo-frt’-cm-frt’-for and topo-frt’-cm-frt’-rev primers. 1: PCR amplification, Bl: negative control, SL: Molecular weight marker (Smart ladder).
 
The PCR product was digested by XbaI and SpeI and cloned in the linearized pSB1A3 plasmid digested with the same enzymes. The ligation product was transformed in MC1061 strain and candidates were selected on plates containing Amp and Cm. Five pSB1A3-FRT’-Cm-FRT’ candidates were then extracted by mini-prep and checked by PCR amplification using the same primers used for PCR amplification of the TOPO® fragment. We also used the TOPO®-FRT’-CM-FRT’ #5 as a positive control (Figure 9 ). Positive candidates show an amplification of 889 bp.
Figure 9  PCR amplification of the FRT’-Cm-FRT’ insert in pSB1A3 plasmid candidates. PCR reactions were performed as described in Materials and Methods using the TOPO-FRT’-CM-FRT’-FOR and TOPO-FRT’-CN-FRT’-REV primers. 1->5 : pSB1A3-FRT’-Cm-FRT’ candidates, 6 : TOPO®-FRT’-CM-FRT’ #5, positive control, BI : negative control, SL : Molecular weight marker (Smart ladder).
 
We selected pSB1A3-FRT’-Cm-FRT’ #1 and sent it as biobrick BBa_K551000.  At the time of writing, the BBa_K551000 was sent for sequencing but we did not get the results yet. 
 
FRT’-Cm-FRT’ excision assay :
We performed an excision assay using the TOPO®-FRT’-Cm-FRT’ since at the time of experiments, we did not have yet the BBa_K551000. 
The E. coli Top10/ TOPO®-FRT’-Cm-FRT’ strain was transformed with pCP20 plasmid. Candidates were selected on LB Kan Cm Amp plates at 30°C. Twelve independent transformants were streaked on LB plates at 42°C to obtain isolated colonies. Four colonies of each streak were stabbed on LB Kan and LB Kan Cm plates at 30°C.  Among the 48 stabbed colonies, 8 grew on LB Kan plate but not on LB Kan Cm plates indicating that the FRT’-Cm-FRT’ cassette has been lost, although at a low frequency.  This might be due to the high copy-number of the TOPO® (ColEI origin of replication)
Characterization of the repA101ts origin of replication:
Before obtaining a final pINDEL construct we tested the thermosensitivity of the repA101ts origin of the pIN construct. This experiment should be repeated with the final construction of pINDEL.  
The pIN plasmid was transformed in the MC1061 strain and transformants were selected on LB Amp medium at 30°C. Colonies were streaked on LB and LB Amp (100 mg/ml) and grown overnight at 30°C and 42°C.  As expected, colonies grew on LB and LB Amp plates at 30°C.  At 42°C, colonies were unable to grow on LB Amp plates while they grew on LB plates (data not shown).  This strongly suggest that the repA101ts origin is not functional at 42°C.
To confirm the results obtained on plates, we performed an experiment in liquid medium in which we measured the plasmid loss frequency as a function of time of culture at 42°C.  We compared the viability (CFU/ml: colony forming units per ml of culture) of MC1061/pIN grown at 30°C and 42°C and plated on LB and LB Amp medium. 
Figure 10  The repA101ts replication origin of the pIN plasmid is thermosensitive. The MC1061 strain containing the pIN plasmid was grown in LB medium at 30°C (blue triangle) and 42°C (red square).  At times indicated in the figure, a sample of the cultures was diluted and plated on LB and LB Amp plates as described in Materials and Methods. Colonies were counted after ON incubation at 30°C. The percentage of bacteria containing pIN is the ratio between the CFU/ml on LB Amp plates and the CFU/ml on LB plates multiplied by 100.
 
Figure 10  shows that the replication of pIN is impaired at 42°C.  After 150 min of culture at 42°C, 60% of the bacteria have lost the pIN plasmid.  After 275 min, only a very small proportion of bacteria retained the pIN plasmid (< 10%).  Note that at time 0 of the experiment, around 70% of the bacteria retained the pIN plasmid suggesting that the repA101ts origin is not fully active even at 30°C.  With time, the pIN plasmid is maintained in about 80% of the bacteria at 30°C. 
This experiment was performed only once due to time limitations and should be repeated to confirm the data. 
Deletion assay using the pINDEL plasmid :
To test the deletion capacity of the pINDEL plasmid, we transformed pINDEL and pIN in the MG1655 ∆tldD ::frt-cm-frt  strain.  The transformants were selected on LB Amp Cm medium at 30°C. Four independent colonies containing pIN or pINDEL were streaked on LB plates.  Plates were incubated ON at 42°C, condition in which the replication of pIN and pINDEL ceases flp is expressed. For each independant transformants, we stabbed 8 colonies on LB and LB Cm plates and incubated the plates ON at 37°C.
 All 32 candidates that contained pIN grew on LB plates and LB Cm plates indicating that none of them had lost the antibiotic resistance cassette.  All 32 candidates that contained pINDEL grew on LB but none of them grew on LB Cm plates indicating that all of them have lost the antibiotic resistance cassette, sgowing the the DEL function of pINDEL functions properly. 
Insertion assay using the pINDEL plasmid :
To  test the IN function of the pINDEL plasmid, we transformed pINDEL and pIN in the MG1655 strain. The transformants were selected on LB Amp Cm medium at 30°C.  Electrocomptent MG1655/pIN and MG1655/pINDEL were prepared as indicated in the Material and Methods.
FRT’-Cm-FRT’ cassette was amplified by PCR using the LACZ-frt’-cm-frt’-for and LACZ-frt’-cm-frt’-rev primers.  The LACZ-frt’-cm-frt’-for primers is composed of 40 nt homologous to the lacZ promoter sequence at its 5’-end and 20 nt homologous to region flanking FRT’-Cm-FRT’ at its 3’-end. The LACZ-frt’-cm-frt’-rev primer is composed of 40 nt homologous to the lacZ gene sequence at its 5’-end and 20 nt  homologous to region flanking FRT’-Cm-FRT’ at its 3’-end.
Figure11  PCR amplification of the LACZ-FRT’-Cm-FRT’.  PCR reactions were performed as described in Materials and Methods using the LACZ-frt’-cm-frt’-for and LACZ-frt’-cm-frt’-rev primers and pSB1A3-FRT’-Cm-FRT’ plasmid as template.  1: LacZ-FRT’-Cm-FRT’-LacZ fragment, Bl: negative control, SL: Molecular weight marker (Smart ladder).
 
The PCR product we obtained was showing the expected size (969 nt) (Figure 11 ). It was electroporated as described in the Material and Methods section. The electroporation mix was plated on LB Amp and Cm plates.  Plates were incubated ON at 30°C.  Unfortunately, we did not obtain candidates LB Amp Cm plates.
Growth test of the MC1061/pINDEL strain:
In order to test whether the IN and/or the DEL function might interfere with E. coli growth, we measured the OD600nm of a culture of the MC1061 strain containing either pIN or pINDEL.
These bacteria were grown overnight at 30°C in LB and in LB with 1% arabinose. The overnight cultures were diluted at an OD600nm of 0.01 in the same medium and incubated at 30°C with shaking. The OD600nm was measured.
Figure 12  Influence of Red gene expression on bacterial growth.  MC1061/pIN (square) and MC1061/pINDEL (triangle) strains were cultivated ON at 30°C in LB Amp (green symbol) and in LB Amp supplemented with 1% arabinose (red symbols). The ON cultures were diluated at an OD600nm of 0.01 in the same medium and incubated at 30°C with shaking. The OD600nm was measured every 30 minutes.
Figure 12  shows that the strains containing the pIN plasmid or the pINDEL plasmid have a similar growth rate in the same medium.  However, in the presence of arabinose, the gowth rate was drastically affected, indicating that expression of the Red recombinase has a detrimental effect on bacterial growth. 
Annexes:
Annex 1 Sequence Topo-FRT'-Cm-FRT'5
iGEM ULB Brussels Team - Contact us
Retrieved from " http://2011.igem.org/Team:ULB-Brussels/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
