"
Page
Discussion
View source
History
teams
Log in
 
Team:ULB-Brussels/Discussion
From 2011.igem.org
Home Project Modelling Human practice Discussion Parts Safety Team Sponsors
Discussion: Wetlab / Modeling
Discussion
Our first goal when constructing pINDEL was to find a cost-effective approach to reduce time and steps for fundamental manipulations to insert or delete genes in the E .coli chromosome.  Homologous recombination is a well-characterized and an easy method to do so.  As compared to the old-fashion ‘cut and paste’ ligation method, there is no need for selecting appropriate restriction sites.  As compared to in vitro methods such as the Gateway or Gibson methods, relying on in vivo methods is cheaper and technically easier.  
Cloning by in vivo recombination works very efficiently in yeast and is routinely used in yeast labs.  Despite this advantage, this technique did not cross yet the frontiers of these specialized labs and using yeast as a cloning organism presents some drawbacks. Notably, yeast has a longer generation time compared to E. coli and there is no method allowing the selective extraction of plasmid DNA.
E. coli on the other hand is not proficient for homologous recombination of transformed linear DNA fragments mainly because of high rate of linear DNA degradation.  However, the generation time of about 20 minutes remains a major advantage. 
Cloning systems based on homologous recombination have been recently developed in E. coli. These innovative systems are based on the use of the l Red recombinase system that involves enzymes inhibiting the exonuclease activity responsible for linear DNA degradation and therefore promotes efficient homologous recombination; and on the FLP recombinase which is used to excise the selection marker flanked of specific sites recognized by this recombinase (ref: Datsenko and Wanner PNAS 2000; Yu et al., PNAS 2000). 
The objective of our project was to implement both functions in a single genetic tool that we named ‘pINDEL’, in order to decrease the number of different steps.
Because of the toxicity of these enzymes, especially that of Gam – one of the RED-Recombinase system enzymes, pINDEL was designed in a way that the expression of both functions is tightly controlled regarding the promoters that were used.  The Red function were placed under the control of the pBAD promoter which is induced by arabinose addition, and the flp function was placed under the control of the l pR promoter which is induced at 42°C.  Our data confirmed that expression of the Red recombinase is toxic for the cells.  We also took advantage of a phenomenon called ‘transcriptional interference’ by placing the 2 transcription units in a convergent manner.  In this configuration, the transcriptional activity of pBAD promoter should drastically reduce the transcription of the flp gene.  Because of a time limitations, we were unable to test experimentally this phenomenon but we modelled it.  The model predicted that the interference was effective. 
The other important feature of pINDEL is that it allow the excision of the antibiotic resistance gene that is needed to select the insertion event as well as the loss of the plasmid by growing the recombinant cells at 42°C, therefore generating a recombinant bacteria free of antibiotic resistance genes. 
Due to time limitations, we were unable to characterize some essential aspects of pINDEL.  In the future, we plan to characterize and quantify the insertion capacity of pINDEL. We will have to determine the optimal growth conditions in which expression of Gam allows sufficient growth and efficient homologous recombination. To do so, we will grow E. coli strains containing pINDEL at different arabinose concentration and in the same experiment, test the homologous recombination efficiency at the different arabinose concentration. 
We will also evaluate experimentally the transcriptional interference by measuring the transcriptional activity of the l R promoter in conditions in which the Red recombianse is expressed (i.e. various arabinose concentration).  We will use northern blots and/or RT-qPCR techniques. 
To improve the versatility of our tool, we will also construct new selection cassettes with different antibiotic resistance genes flanked by FRT’ sites. 
Finally, we want to expand the Biobrick collection notably by standardizing the thermo-sensitive replication origin, the promoters and their regulators as well as the Red system.
pINDEL applications
The use of the antibiotic resistance markers represents a major problem in the industrial production of recombinant molecules. In fact, we cannot allow antibiotic resistance genes to spread in the environment as antibiotic resistance represent a major problem in public health.  Moreover, in the industrial point of view, antibiotic resistant genes influence bacterial metabolism and growth and represent a metabolic burden.
In addition to this global consideration, pINDEL could be implemented for in vivo cloning in E. coli by the insertion of any gene of interest in any plasmid vector. This could represent a major advance in the synthetic biology field. The homologous recombination offers the advantage to be precise and to have no restriction site requirements on the contrary to the ‘cut and paste’ cloning method.
iGEM ULB Brussels Team - Contact us
Retrieved from " http://2011.igem.org/Team:ULB-Brussels/Discussion "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
