"
Page
Discussion
View source
History
teams
Log in
 
Team:ULB-Brussels/project
From 2011.igem.org
Home Project Modelling Human practice Discussion Parts Safety Team Sponsors
Introduction Materials & Method Results Experimental conclusion
Introduction
General introduction, the need for genetic Tools in synthetic biology:
As in any construction-oriented disciplines, the advance in synthetic biology depends on the availability of tools and modular parts with desired basic function.  The Registry of Standard Biological Parts contains multiple biobricks that can be mixed and matched to construct complex genetic circuits. 
The ‘One Step Gene Insertion or Deletion System’ project of the ULB-Brussels team aims at providing the synthetic biology community with a novel tool allowing the insertion and/or deletion of genes in the E. coli chromosome in a minimal number of steps.  Inserting and deleting genes is a key process in synthetic biology, not only to characterize the function and regulation of a specific gene but also to implement genetic circuits in a synthetic way. 
Our system will allow notably the insertion of the biobricks that are available at the Registry of Standard Biological Parts in the E. coli chromosome at chosen chromosomal locations.
Homologous recombination:
In yeast:
In the yeast Saccharomyces cerevisiae, the insertion or deletion of a given gene is enabled by a highly powerful homologous recombination system.  This type of genetic recombination involves the exchange of two identical or similar nucleotide sequences located either on different DNA molecules or distant from each other on the same DNA molecule. This process steps in when the cells are confronted to harmful breaks occurring on both strands of DNA, known as double-strand breaks, allowing an accurate repair of the DNA molecules. It also produces new combinations of DNA sequences during meiosis in eukaryotic cells.
 
When a DNA double-strand break occurs, nucleotides located at the 5’-end of the cleavage site are removed. During the invasion step that follows, the resulting overhanging 3’-end of the DNA strand “invades” a similar or identical DNA molecule. This results in the generation of one or two cross-shaped structures denoted as the Holliday junctions, which connect both the invaded and the damaged DNA strands. The resolution of the Holliday junction gives rise to homologous recombination, implying or not a crossing over.
Figure 1 Diagram of homologous recombination by Holiday
As the DNA double-strand break and repair recombination pathway being very efficient in yeast, this organism is widely used to engineer large DNA molecules in a very precise way. Indeed, inserting a PCR-amplified product containing a 40 nucleotides sequence homologous to a given area of the yeast chromosome will result in the insertion of the DNA fragment in the targeted site (Baudin A. et al, 1993).  We took advantage of yeast homologous recombination in our project (see Results).
In E.coli:
Homologous recombination is a major DNA repair mechanism in bacteria as well. Most of the pathways and enzymes involved in this process are well-characterized. In E. coli, the major pathway of homologous recombination relies on the RecA protein. RecA protein recognizes and binds to single-strand DNA, forming DNA-protein filaments. These protein-DNA filaments detect DNA sequence presenting homology. The RecA filament then pairs and exchange strands with the homologous segment. Other recombination systems operate in E. coli and both require RecA function. The RecBCD system is of particular interest for our project. Indeed, the RecBCD enzyme initiates recombination at double-strand DNA ends by generating 3’-single-strand DNA overhangs (Myers RS. et al, 1994). This nuclease activity is responsible for the rapid degradation of any double-strand linear DNA fragment that is transformed in E. coli (Benzinger R. et al, 1975).
 
Thus, most existing cloning methods in bacteria take advantage of a ‘cut and paste’ system where the fragment of interest is inserted in a plasmid in vitro by the concerted action of restriction enzymes and DNA ligase. Here, the construction of recombinant DNA molecules occurs in vitro, E. coli being used to amplify the resulting construct.
In phages:
Phages systems of homologous recombination have recently been exploited as cloning techniques. These systems are interesting from an engineering point of view since they can catalyze highly efficient recombination using small regions of homology, typically less than 50 bp (Muyrers JP.  et al, 2001). As a consequence and in contrast to classical genetic engineering techniques, the use of phage recombination systems does not require the construction of intermediate plasmids containing the homology segments. Only the design of oligonucleotides containing a sequence homologous to the target sequence is required, allowing the generation of a PCR fragment, further used for homologous recombination.
 
The Red recombinase of the λ phage is well-characterized and routinely used in many labs to generate gene deletion in E. coli and related bacterial species. In vivo genetic engineering using this system and short DNA homologies has been termed ‘recombineering’ (recombination‐mediated genetic engineering) (Ellis et al., 2001).
 
The Red system involves three genes: gam, bet and exo, whichproductsare necessary for homologous recombination. Thegam gene codes for the Gam protein, which inhibits the RecBCD nuclease from attacking linear DNA and thus allowing it to be used as a substrate for recombination (Murphy et al, 1991). Exo is a double strand DNA‐dependent exonuclease, which processes linear double-strand DNA. It remains bound to one strand while degrading the other in a 5’‐3’ direction, leaving behind the intact 3’-overhang. This result in double-strand DNA with a 3’ single-strand DNA overhang, the substrate required for the Beta protein to bind. Finally, Beta is a single-strand DNA binding protein that promotes the annealing of complementary DNA strands. (Donald L. Court et al, 2002).
 
Thus, linear DNA is required for Red mediated recombination. Two possibilities can be used, either a linear double‐strand DNA generated by PCR or a short single-stranded DNA oligonucleotide, both have to be homologous to the target sequence (Court et al., 2002).
Site-specific recombinaison:
Site-specific recombination is another type of mechanism involved in DNA rearrangement. This process is driven by a particular type of enzymes which bind to specific sequences and catalyze the cleavage of the DNA molecules, the exchange of the two strands involved, and their ligation.  The FLP recombination system belongs to the site-specific recombination enzyme family.  It is expressed in most yeast strains and encoded by a replicative plasmid called 2-µm. This plasmid is stably maintained in yeast populations. This system presents the advantage of being the less restrictive site-specific recombination mechanism in terms of host range and therefore can be implemented in many different organisms, including E. coli (Schweizer HP, 2003).
 
The FLP recombination system consists of the flp gene encoding the FLP recombinase. This enzyme recognizes small sequences called Flippase Recognition Targets (FRT). This element contains symetric repeats of 13 bp enclosing an asymmetric core. Upon binding to the FRT sequences, FLP catalyzes a recombinationreaction leading to the excision of the DNA fragment located between the FRT sites. (Schweizer HP, 2003).
Existing E.coli genetic Tools based on homologous and site-specific recombination
Two research groups have developed genetic tools to delete genes in the E. coli chromosome (Datsenko and wanner, PNAS, 2000; YU et al., PNAS, 2000).  In both cases, the Red recombinase system of the l phage was implemented as tool for homologous recombination.  In one of the cases, 
a plasmid containing the exo, gam and bet genes under the control of the tightly regulated pBAD promoter was constructed (pKD46, Datsenko and Wanner, PNAS, 2000). Expression of the Red recombinase is induced by addition of arabinose.  The pKD46 is eliminated by growing the cells at 42°C since the pKD46 replication is thermosensitive.  After recombination of a selection antibiotic resistance gene flanked by FRT sites at a chosen chromosomal location, a plasmid encoding the flp gene is transformed in the E. coli strain. The excision step occurs at 42°C as the expression of flp is under the control of the l pR promoter which is negatively regulated by the CI857 thermosensitive repressor(Datsenko Kirill A. and Wanner Barry L., 2000).  Therefore, this method requires 2 transformation steps and 2 passages at 42°C.
Transcriptional interférence:
Transcriptional interference (TI) is defined as the suppressive influence of one transcriptional process, directly and in cis, on a second transcriptional process (Shearwin et al., 2003, trends in genetics). Models describing the different molecular mechanisms of TI have been proposed (Figure 2 ).
Figure2  Mechanisms for transcriptional interférence Five possible mechanisms are proposed.  (a) Promoter competition : occupation the pA promoter by RNA-polymerase precludes the occupation of the pS promoter. (b) Sitting duck mechanism : if the RNA-polymerase at the pS promoter is slow to start the elongation phase of transcription, it could cosntitute a block that would be dislodged by the elongation from the pA promoter. (c) Occlusion : the progression of the RNA-polymerase that initiated transcription at pA could impair the initiaiton at pS. (d) Physical collision between the 2 convergent elongation complex leads premature termination of one or the 2 transcriptional units. (e) Roadblock : an initiation complex at pA might block the progression of an elongation complex that initiated at pS.  From (Shearwin et al., 2003, trends in genetics).
In this project, we will focus on the ‘collison’ model as we believe that it is the more likely occur in the case we are interested in (see below).  This model proposes that physical collision between converging elongation complexes leads to premature transcriptional termination. Note that in bacteriophages, some evidences have been obtained indicating that convergent promoters might interfere by such a mechanism.  In this model, it is not known whether both RNA-polymerases stall during collision and boh ‘fall off’ or whether only one RNA-polymerase ‘falls off’.  In our model (see below), we will favor the second hypothesis.
Objectif:
The ULB-Brussels project for the iGEM2011 competition aims at combining well-characterized functions for the insertion and deletion of genes in a single a genetic tool.  This tool, called pINDEL, consists of the Red and FLP recombinase systems combined on a single plasmid.  The IN function relies on the l Red recombinase which allows gene insertion by homologous recombination.  The DEL function relies on the FLP site-specific recombinase which will remove specifically the antibiotic resistance gene used to select the insertion event.  The goal is to  design a tool in which the two functions are tightly controlled in order to avoid any interference and that will allow their sequential expression in a sequential manner (expression of the IN function prior that of that of DEL). 
 
We also propose to construct a system allowing genetic ‘coupling’ between any biobrick provided by the Registry of Standard Biological Parts and a antibiotic resistance flanked by the FRT sites and by the iGEM prefix and suffix.  This system will allow selecting the insertion of the biobrick in the E. coli chromosome at a chosen location and the subsequent excision of the selection marker in order to obtain a recombinant bacteria free of any antibiotic resistance gene. 
iGEM ULB Brussels Team - Contact us
Retrieved from " http://2011.igem.org/Team:ULB-Brussels/project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
