<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ULB-Brussels_modeling_excision"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ULB-Brussels/modeling/excision">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ULB-Brussels/modeling/excision&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ULB-Brussels/modeling/excision&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ULB-Brussels/modeling/excision&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ULB-Brussels/modeling/excision" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ULB-Brussels/modeling/excision</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="header"><DIV id="menu"><A href="https://2011.igem.org/Team:ULB-Brussels">Home</A><A href="https://2011.igem.org/Team:ULB-Brussels/project">Project</A><A id="couleur" href="https://2011.igem.org/Team:ULB-Brussels/modeling">Modelling</A><A href="https://2011.igem.org/Team:ULB-Brussels/human">Human practice</A><A href="https://2011.igem.org/Team:ULB-Brussels/Discussion">Discussion</A><A href="https://2011.igem.org/Team:ULB-Brussels/parts">Parts</A><A href="https://2011.igem.org/Team:ULB-Brussels/safety">Safety</A><A href="https://2011.igem.org/Team:ULB-Brussels/team">Team</A><A href="https://2011.igem.org/Team:ULB-Brussels/sponsors">Sponsors</A></DIV><DIV id="sousm"><A href="https://2011.igem.org/Team:ULB-Brussels/modeling">Introduction</A><A href="https://2011.igem.org/Team:ULB-Brussels/modeling/30">Transcriptional interference</A><A href="https://2011.igem.org/Team:ULB-Brussels/modeling/42">Insertion model</A><A href="https://2011.igem.org/Team:ULB-Brussels/modeling/excision">Excision model</A><A href="https://2011.igem.org/Team:ULB-Brussels/modeling/loss">Loss of the pINDEL plasmid at 42°C</A><A href="https://2011.igem.org/Team:ULB-Brussels/modeling/comparison">Comparison with data</A><A href="https://2011.igem.org/Team:ULB-Brussels/modeling/conclusion">Conclusion</A></DIV></DIV><DIV id="main"><DIV id="maintext"><DIV id="hmaint">
			Modelling : Excision model			</DIV><DIV id="maint"><H1>Excision model</H1><H2>Experimental design of the excision step</H2><P>After the insertion step, <EM>E. coli</EM> containing pINDEL and the gene of interest (X) and the FRT'-Cm-FRT' biobrick inserted at the proper location in the chromosome are obtained on LB medium containing Cm.  Colonies are streaked on LB medium and the plates are incubated ON at $42^\circ$C in order to express the FLP recombinase and therefore excise the FRT'-Cm-FRT' cassette. In the meantime, replication of pINDEL will cease, as the REPA101Ts replication protein is thermosensitive. Thus, in this step, the FRT'-Cm-FRT' is excised and the pINDEL plasmid is lost. The next day, colonies are re-streaked on LB medium containing Cm to check the FRT'-Cm-FRT' excision and on LB medium containing Amp to check the loss of pINDEL. Plates are incubated ON at $30^\circ$C.
</P><P>Based on the experiments performed in the host laboratory as we did not have the time to perform this experiment with pINDEL, we consider the pINDEL is lost in $100\%$ of the bacteria and that the FRT'-Cm-FRT' cassette excised in about $80\%$ of the bacteria. The efficiency of excision varies according to the chromosomal location of the insertion. According to the experiment we performed with the pIN construct in liquid medium, pINDEL should be efficiently lost at $42^\circ$C. We observed that after $275$min of liquid culture at $42^\circ$C, more than $90\%$ of the bacteria do not contain pINDEL.
</P><H2>Definitions</H2><P>Let us begin with a proper definition of the different biological functions that are considered in our model:
<UL><LI>$N$: total number of bacteria in the considered colony;
</LI><LI>$P$: average number of pINDEL plasmid copies per bacterium;
</LI><LI>$E$: average amount of active RepA101 protein per bacterium;
</LI><LI>$F$: average amount of active FLP per bacterium;
</LI><LI>$G_i  (i=1,2,3)$: average amount of the Red recombinase protein $i$ (1 is Gam, 2 is Exo and 3 is Bet), per bacterium.
</LI></UL></P><H2>Getting the equations of the excision step</H2><P>At initial time ($t = 0$), i.e. when plates with re-streaked colonies are incubated ON at $30^\circ$C, the number of bacteria in the considered new colony is $N_0:=N(0)\approx1$. As in section (\ref{Mod30}), we postulate a logistic model:
\begin{equation}
\dot N=k_NN\left(1-\frac N{N_{max}}\right)
\label{N42}\end{equation}
where $N_{max}$ is the maximal number of bacteria in a colony and where $k_N$ corresponds to the growth rate one would observe in the limit where the saturation would be inexistent. The number of bacteria in a colony at saturation point is approximatively $N_{max}\approx10^9$. As in section (\ref{Mod30}), $k_N\approx \frac{\log{2}}{20\cdot60}\mbox{s}$.
</P><P>Let us observe how $P$ evolves in time. At initial time in this step, the average amount of pINDEL plasmids per bacterium is $P_0:=P(0)\approx19$, that is slightly less than the maximum: immediately after the night culture we must have theoretically $P=P_{max}$, but we have to take into account the possible accidents during the manipulations before the beginning of the excision step. At 30¡C, the replication of pINDEL is optimum. RepA101Ts is constitutively produced by pINDEL but becomes quickly inactive at $42^\circ$C. Once it is unfolded, it is likely that it will rapidly be degraded by the host ATP-dependent proteases that are in charge of misfolded protein degration such as Lon. We thus obtain for the total amount of active RepA101 (<EM>i.e.</EM> $E\cdot N$) the relation:
\begin{align}
&amp;\frac d{dt} (EN)=C_EPN-A_EEN\\
\Leftrightarrow\quad&amp;\dot E=C_EP-A_EE-\frac{\dot N}NE
\label{E42}\end{align}
where $D_E$ is the desactivation rate of RepA101Ts at $42\circ$C and $C_E$ is its production rate. We estimate that a pINDEL plasmid produce one RepA101 every $18$s, so that $C_E\approx\frac1{18}\mbox{s}^{-1}$. Furthermore, as RepA101 half-life time at $42^\circ$C is about $2$min, we then obtain (by a similar reasoning as in (eq(\ref{k_N}))) $A_E\approx\frac{\log2}{30}\mbox{s}^{-1}$. At initial time, we can estimate that $E_0:=E(0)= \approx 5 \cdot 10^3$.
</P><P>The replication rate of the pINDEL plasmids is proportional to an increasing function of $E$. As we expect $E$ to cancel out very fast, that rate is approximately proportional to $E$ itself. The saturation being at $P=P_{max}\approx20$ (the origin of replication of the plasmid is of the type <EM>low copy</EM>), we postulate for the total number of palsmid copies (<EM>i.e.</EM> $PN$), exactly like in section (\ref{Mod30}), a logistic model, but where the saturation is due only to $P$:
\begin{align}
&amp;\frac d{dt}(PN)=k_P\frac E{E_0}PN\left(1-\frac P{P_{max}}\right)\\
\Leftrightarrow\quad&amp;\dot P=k_P\frac E{E_0}P\left(1-\frac P{P_{max}}\right)-\frac{\dot N}NP\label{P42}
\end{align}
where $k_P\approx\frac{\log{2}}{14.4}\mbox{s}^{-1}$ (see section (\ref{Mod30})) corresponds to the growth rate one would observe in the limit where the saturation would be inexistent and where $E=E_0$
</P><P>In the absence of arabinose, the pBAD promotor is repressed by the AraC protein and the expression of the Red recombinase ceases. The proteins $i$ will eventally be diluted in the growing colony. For the total amount of the Red recombinase protein $i$ (<EM>i.e.</EM> $G_i\cdot N$), we have the equation:
\begin{align}
&amp;\frac d{dt}(G_iN)=-D_iG_iN \quad (i=1,2,3)\\
\Leftrightarrow\quad&amp;\dot{G_i}=-\left(\frac{\dot N}N+D_i\right)G_i \quad (i=1,2,3)
\label{Gi42}\end{align}
We estimate that a pINDEL plasmid produce one protein $i$ every $40$s (exactly as was explained in section (\ref{Mod30})), so that $C_i\approx\frac1{40}\mbox{s}^{-1}$. Furthermore, as that protein at $30^\circ$C is stable, we can estimate its half-life time to be around $60$min; we then obtain (by a similar reasoning as in (eq(\ref{k_N}))) $D_i\approx\frac{\log2}{60\cdot60}\mbox{s}^{-1}$. At the initial time of this step, $G_{i,0}:=G_i(0)\approx\frac{C_iP_{max}}{D_i}\approx 2.59 \cdot 10^3$, that is the asymptotic value at the end of the insertion phase.
</P><P>At $42^\circ$C, the $\lambda$ pR promoter controlling the <EM>flp</EM> is fully active, since the CI857 repressor is thermosensitive and not active at this temperature. As the transcription of the genes $i$ ceases, transcriptional interference does not occur and <EM>flp</EM> is fully expressed. However, the activity of FLP is drastically reduced at $42^\circ$C. Our equation for the total amount of active FLP per bacterium (<EM>i.e.</EM> $F\cdot N$) reads
\begin{align}
&amp;\frac d{dt}(FN)=C_FPN-A_FFN\\
\Leftrightarrow\quad&amp;\dot{F}=C_FP-A_FF-\frac{\dot N}NF
\label{F42}\end{align}
where $C_F$ is the production rate of FLP by Pindel and $A_F$ the deactivation (denaturation) rate at $42^\circ$C.
We estimate that a pINDEL plasmid produce one FLP every $24$s (exactly as was explained in section (\ref{Mod30})), so that $C_F\approx\frac1{24}\mbox{s}^{-1}$. Furthermore, as FLP half-life time at $42^\circ$C is about$30$sec, we then obtain (by a similar reasoning as in (eq(\ref{k_N}))) $A_F\approx\frac{\log2}{30}\mbox{s}^{-1}$. At the initial time of this step, $F_0:=F(0)\approx\frac{10\%p_{simul}C_FP_{max}}{D_F}\approx 4.33$, that is the asymptotic value at the end of the insertion phase.
</P><P>That way we obtained the following system (see eqs (\ref{N42}), (\ref{E42}), (\ref{P42}), (\ref{Gi42}), (\ref{F42})):
$$
\left\{
\begin{array}{c}
\dot N=k_NN\left(1-\frac N{N_{max}}\right)\label{N42f}\\
\dot E=C_EP-A_EE-\frac{\dot N}NE\label{E42f}\\
\dot P=k_P\frac E{E_0}P\left(1-\frac P{P_{max}}\right)-\frac{\dot N}NP\label{P42f}\\
\dot{G_i}=-\left(\frac{\dot N}N+D_i\right)G_i \qquad (i=1,2,3)\label{Gi42f}\\
\dot{F}=C_FP-A_FF-\frac{\dot N}NF\label{F42f}
\end{array}
\right.
$$
</P><H2>Solving the equations </H2><P>The first equation (eq(\ref{N42f})) is solved like we did previously (see section (\ref{Mod30})) and gives us
\begin{align}
N(t)&amp;=\frac{N_{max}N_0e^{k_Nt}}{N_0e^{k_Nt}+(N_{max}-N_0)}=N_0e^{k_Nt}\frac1{1+\frac{N_0}{N_{max}}\left(e^{k_Nt}-1\right)}\label{Nsol42}\\
&amp;\approx N_0e^{k_Nt}\label{approx42}
\end{align}
where once again the approximation remains valid for short times:
\begin{equation}
t\ll\frac1{k_N}\log{(\frac{N_{max}}{N_0}+1)}\approx35877\mbox{s}=9\mbox{h}57\mbox{min}57\mbox{s}.
\end{equation}
Saturation is reached after $t\approx40000\mbox{s}=11\mbox{h}6\mbox{min}40\mbox{s}$, as we can observe on the graph (fig(\ref{graph5})), corresponding to realistic values for the parameters.

In blue is shown the exact solution for $N$, while in red is the exponential approximation (eq(\ref{approx42})). This is obtained for $N_{max}=10^9$bact, $N_0=1$bact and $k_N=\frac{\log2}{20\cdot60}\mbox{s}^{-1}$.}

</P><P>The equations for $E$ and $P$ (eqs (\ref{E42f}) and (\ref{P42f})) then form a coupled system of equations, which rewrite, using the solution for $N$ (eq(\ref{Nsol42})):

$$
\left\{
\begin{array}{}
\dot E=C_EP-A_EE-k_N\frac{N_{max}-N_0}{N_0e^{k_Nt}+(N_{max}-N_0)}E\label{E42f}\\
\dot P=k_P\frac E{E_0}P\left(1-\frac P{P_{max}}\right)-k_N\frac{N_{max}-N_0}{N_0e^{k_Nt}+(N_{max}-N_0)}P\label{P42f}
\end{array}
\right.
$$

that can easily be solved numerically using <EM>Mathematica</EM>. For realistic values of the parameters, we obtain (fig(\ref{graphe6})) and (fig(\ref{graphe7})) for $E$ and $P$ respectively.

This is obtained for $N_{max}=10^9$bact, $N_0=1$bact, $k_N=\frac{\log2}{20\cdot60}\mbox{s}^{-1}$, $C_E=\frac1{18}\mbox{s}^{-1}$, $A_E=\frac{\log{2}}{30}$, $k_P=\frac{\log2}{14.4}\mbox{s}^{-1}$, $E_0=5\cdot10^3$, $P_{max}=20$ and $P_0=19$.}
</P><P>We can observe that $E$ decreases very rapidily and cancels out, in such a way that the equation for $P$ rewrite
\begin{align}
&amp;\dot P\approx-k_N\frac{N_{max}-N_0}{N_0e^{k_Nt}+(N_{max}-N_0)}P=-\frac{\dot N}NP\\
\Leftrightarrow\quad&amp;\frac{\dot P}P=-\frac{\dot N}N\\
\Leftrightarrow\quad&amp;P(t)\approx\frac{P_0N_0}{N(t)}=\frac{P_0}{N_{max}}\left(N_0+(N_{max}-N_0)e^{-k_Nt}\right)\\
&amp;\qquad\qquad\overset{t\rightarrow\infty}{\longrightarrow}P_0\frac{N_0}{N_{max}}\approx1.9\cdot10^{-8}.
\end{align}
Almost all bacteria will then have lost pINDEL, as is observed in the experiments.
</P><P>The equation for $F$ (eq(\ref{F42f})) can be solved numerically, for example with <EM>Mathematica</EM>; for realistic values of the parameters, we obtain the following graphic (fig(\ref{graphe8})).

This is obtained for $N_{max}=10^9$bact, $N_0=1$bact, $k_N=\frac{\log2}{20\cdot60}\mbox{s}^{-1}$, $C_E=\frac1{18}\mbox{s}^{-1}$, $A_E=\frac{\log{2}}{30}\mbox{s}^{-1}$, $k_P=\frac{\log2}{14.4}\mbox{s}^{-1}$, $E_0=5\cdot10^3$, $P_{max}=20$, $P_0=19$, $C_F=\frac1{24}\mbox{s}^{-1}$ and $A_F=\frac{\log{2}}{30}\mbox{s}^{-1}$.}

We consequently observe that the amount of FLP per bacterium is increasing until a given maximum, $F\approx30$ at time $t\approx150$s. This amount of FLP is sufficient for the excision of the Cm resistance gene by site specific recombination of the FRT'-Cm-FRT'. After a while, the amount of active FLP will drastically decreased and reached zero as the pINDEL does not replicate at $42^\circ$C and as the activity of FLP at this temperature decreases. It is likely that FLP unfolds at this temperature and gets rapidly degraded by the host ATP-dependent proteases such as Lon.
</P><P>Finally, the equation for the $G_i$ (eq (\ref{Gi42f})) solves immediately:
\begin{align}
&amp;\dot{G_i}=-\left(\frac{\dot N}N+D_i\right)G_i\\
\Leftrightarrow\quad&amp;\frac d{dt}(\log{G_i})=\frac d{dt}(-\log{N}-D_it)\\
\Leftrightarrow\quad&amp;G_i(t)=\frac{G_{i,0}N_0}{N(t)}e^{-D_it}\qquad\overset{t\rightarrow\infty}{\longrightarrow}0.
\end{align}
For realistic values of the parameters, we obtain the graph that follows (fig(\ref{graphe9})).

This is obtained for $N_{max}=10^9$bact, $N_0=1$bact, $k_N=\frac{\log2}{20\cdot60}\mbox{s}^{-1}$, $D_i=\frac{\log2}{40}\mbox{s}^{-1}$ and $G_{i,0}=5\cdot10^3$.


We then observe that the proteins $i$ deteriorate and disappear.
</P><P>It is necessary to point out that, again, the solution of our model only shows a small sensitivity to the parameters, around the estimated values: a small error on the parameters only leads to a small error in the solution, as we can easily notice by moving the parameters around the estimated values.</P></DIV></DIV></DIV><DIV id="bas"><DIV id="basi">
	iGEM ULB Brussels Team - <A href="mailto:lvmelder@ulb.ac.be">Contact us</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:ULB-Brussels/modeling/excision">http://2011.igem.org/Team:ULB-Brussels/modeling/excision</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ULB-Brussels/modeling/excision" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ULB-Brussels/modeling/excision" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ULB-Brussels/modeling/excision&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ULB-Brussels/modeling/excision&amp;oldid=169343" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>