<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF_Lausanne_Cell_surface_display"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF_Lausanne/Cell_surface_display">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF_Lausanne/Cell_surface_display&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Cell_surface_display&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Cell_surface_display&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF_Lausanne/Cell_surface_display" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF Lausanne/Cell surface display</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="all"><DIV class="menu"><NAV><DIV class="menu-item first-menu-item"><A href="/Team:EPF_Lausanne" class="main-item">Home</A></DIV><DIV class="menu-item"><A href="/Team:EPF_Lausanne/Overview" class="main-item">Project</A><UL><LI><A href="/Team:EPF_Lausanne/Overview">Overview</A></LI><LI><A href="/Team:EPF_Lausanne/Nanoparticles">Nanoparticles</A></LI><LI><A href="/Team:EPF_Lausanne/Cell_surface_display">Cell surface display</A></LI><LI><A href="/Team:EPF_Lausanne/Sensing-Effector">Sensing-Effector</A></LI><LI><A href="/Team:EPF_Lausanne/Results_summary">Results summary</A></LI><LI><A href="/Team:EPF_Lausanne/Next steps">Next steps</A></LI><LI><A href="/Team:EPF_Lausanne/Perspectives">Perspectives</A></LI></UL></DIV><DIV class="menu-item"><A href="/Team:EPF_Lausanne/Calendar" class="main-item">Notebook</A><UL><LI><A href="/Team:EPF_Lausanne/Calendar">Calendar</A></LI><LI><A href="/Team:EPF_Lausanne/Protocols">Protocols</A></LI><LI><A href="http://parts.igem.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2013&amp;group=EPF_Lausanne">Submitted parts</A></LI></UL></DIV><DIV class="menu-item"><A href="/Team:EPF_Lausanne/Team" class="main-item">Team</A><UL><LI><A href="/Team:EPF_Lausanne/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi">Official Team profile</A></LI><LI><A href="/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="/Team:EPF_Lausanne/Acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:EPF_Lausanne/Gallery">Gallery</A></LI></UL></DIV><DIV class="menu-item"><A href="/Team:EPF_Lausanne/Outreach_Overview" class="main-item">Outreach</A><UL><LI><A href="/Team:EPF_Lausanne/Outreach_Overview">Overview</A></LI><LI><A href="/Team:EPF_Lausanne/Kit">Kit</A></LI><LI><A href="/Team:EPF_Lausanne/Woman_and_engineers_day">Woman &amp; Engineer's day</A></LI></UL></DIV><DIV class="menu-item last-menu-item"><A href="/Team:EPF_Lausanne/Safety" class="main-item">Safety</A></DIV></NAV></DIV><DIV class="page"><DIV class="page-content"><H1 id="pageTitle">Header</H1><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Our_Biobricks"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Our Biobricks</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#INP-YFP_Characterization"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">INP-YFP Characterization</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#New_Biobricks.2FMicroscopy_after_the_Jamboree_in_Lyon"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">New Biobricks/Microscopy after the Jamboree in Lyon</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><DIV align="justify"><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 1:Scheme of the cell surface display of streptavidin</DIV></DIV></DIV><P>We knew we wanted to bind Nanoparticles to the Bacteria using the very strong interaction between biotin and streptavidin.

Next we needed to decide which strategy would be the best so the E.coli expresses streptavidin on the outer membrane. 

E.Coli are Gram-negative bacteria and they have two membranes, an inner one and an outer one, separated by a periplasmic space. Furthermore, the outer membrane is covered with a glycocalix, a layer of sugars. 
</P><P>
There are different ways to express a heterologous protein on the outer membrane of E.coli. Peptides can be fused with outer membrane proteins or on fimbriae, fused to beta-autotransporters or lipoproteins, as for example the (Lpp)-OmpA protein. But then there is also another promising display system based on the use of ice nucleation protein (INP) from Pseudomonas syringae. Which we decided to use.
</P><H2><SPAN class="mw-headline" id="Our_Biobricks">Our Biobricks</SPAN></H2><DIV class="biobrick"><P id="brickDescription">For Details, please move the cursor onto the image.</P><P class="areaDescription" id="areaRot" style="left:40px;top:140px"><B>Ice nucleation Protein </B> INP originates from the bacteria Pseudomonas syringae. It is an outer membrane protein. That has successfully been used for cell surface display of enzymes, antibodies and antigens.</P><P class="areaDescription" id="areaBlau" style="left:250px;top:100px"><B>Streptavidin</B>Streptavidin originates from the bacterium Streptomyces avidinii. It is a tetrameric protein, each of the four subunit can bind one biotin molecule with a very high affinity. 

For our Biobricks we used three different kinds of streptavidin.
</P><P class="areaDescription" id="areaSchwarz" style="left:120px;top:270px"><B>pBS1C3</B>The three biobricks were cloned into the standard iGEM backbone plasmid, which contains a chloramphenicol resistance for selection.</P></DIV><P>We decided to engineer a fusion protein between Ice nucleation protein (INP) and Streptavidin. Because such fusion proteins have been successfully used for cell surface display of enzymes, single-chain antibodies and antigens. So we assumed it would work for streptavidin as well. Especially because Jung et al (<A href="http://www.ncbi.nlm.nih.gov/pubmed/9624691" class="external text" rel="nofollow">1</A>) even succeeded to express a functional surface enzyme, that was much bigger than streptavidin.

Ice nucleation protein is a glycosyl-phosphatidyliositol anchored outer membrane protein, this motif is quite unique in prokaryotes. When it is expressed in E.coli over 90% of INP is found in the outer membrane, Wolber et al (<A href="http://www.pnas.org/content/83/19/7256.short" class="external text" rel="nofollow">2</A>).
</P><P><B>Streptavidin</B>
Streptavidin originates from the bacterium Streptomyces avidinii (<A href="http://www.ncbi.nlm.nih.gov/nuccore/A00743.1" class="external text" rel="nofollow">NCBI</A>). It is a tetrameric protein, and each of the four subunits can bind biotin with a very high affinity.
</P><P><I>We cloned three Biobricks that contain different versions of streptavidin.</I></P><P><B>BBa_K1111012</B>
Encodes a fusion protein between INP and a monovalent streptavidin provided by the laboratory of Mark Howarth at Oxford university Department of Biochemistry (<A href="http://users.ox.ac.uk/~bioc0756/MyWebs/activesite/index.htm" class="external text" rel="nofollow">3</A>)
<B>BBa_K1111013</B>
Encodes a fusion protein between INP and a dead version of streptavidin. It was designed to serve as a negative control for the microscopy functional assays, because its affinity to biotin is much lower.
<B>BBa_K1111014</B>
Encodes a fusion protein between INP and streptavidin we received in the Biobrick BBa_K283010 (<A href="http://parts.igem.org/Part:BBa_K283010" class="external text" rel="nofollow">BBa_K283012, Main Page</A>).
</P><H2><SPAN class="mw-headline" id="INP-YFP_Characterization">INP-YFP Characterization</SPAN></H2><P>There is a Biobrick that contains INP in the iGEM registry (BBa_K523013). It was designed by the  <A href="https://2011.igem.org/Team:Edinburgh/Cell_Display" class="external text" rel="nofollow">Edinburgh 2011 iGEM team</A>. This Biobrick encodes a fusion protein of a truncated and codon optimized INP and yellow fluorescent protein (YFP) under control of the Lac promoter. The INP is truncated, it only contains the first 211 and last 97 amino acids that are normally encoded by the inaK gene. This is enough to target it to the outer membrane.
</P><P>
We improved the Biobrick  <A href="http://parts.igem.org/Part:BBa_K523013">BBa_523013</A>   by extensively characterizing it.
</P><P>
We used confocal microscopy to see if it was expressed and eventually show surface localization.

Thanks to Thierry Laroche for taking the images with the confocal microscope.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 2:<B>Confocal image of DH5alpha E.coli  transformed with BBa_K523013</B>A: shows YFP fluorescence (514 nm) B: zoom of one Bacteria that shows spot of clustered YFP</DIV></DIV></DIV><P>Looking at E.coli we noticed that fluorescence was localized in specific specks, that the team of Edinburgh didnât describe. We started looking for an explanation for this phenomena and found in the literature that INP forms aggregates in the cell membranes <A href="http://www.ncbi.nlm.nih.gov/pubmed/2203606" class="external text" rel="nofollow">[3</A>] This suggests that INP also forms aggregates when it is fused to other proteins. We continued characterizing this biobrick by first determining the percentage of bacteria that were fluorescent. We found that approximately 30% of E.coli expressed YFP.
</P><P>The team of Edinburgh grew bacteria and lysed them. After centrifugation, they looked where the fluorescence seemed concentrated. On the transformed bacteria, it was mainly the pellet that appeared fluorescent, meaning that the fusion protein was embedded in the membrane. The negative control, in contrast, showed fluorescence in the pellet as well as in the supernatant with the same intensity. This method was used because their microscopy was not good enough. 

The confocal microscope was really powerful. But we too, didn't succeed to proof surface localization of the fusion protein by simply looking at the intrinsic fluorescence of the YFP, because the signal was to diffuse. 

So we came up with an alternative strategy: <B>Immunofluorescent assay</B> using an anti-YFP antibody and visualizing it with an inverted microscope.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 3:<B>Inverted microscope composite images of BBa_K523013 transformed DH5alpha E.coli</B> A: shows YFP fluorescence (514 nm) B: stained with biotinylated anti-YFP antibody and avidin dylight (650 nm) C: Merge of A and B, shows colocalization of fluorescent signal</DIV></DIV></DIV><P>We did an immunofluorescent assay with an anti YFP antibody (Thanks Francesca Volpetti for giving us the antibody). In Figure 3 you can see the results we obtained with microscopy. The YFP signal (Figure 3 A) and the Cy5 signal (The antibody was tagged with NeutrAvidin DylLight Fluorescent Conjugate red, 650 nm from pierce) (Figure 3 B) superimpose nicely (Figure 3 C). Thus proofing that the fusion protein of INP and YFP is exported to the outer membrane.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>In the following section, you will find the results concerning our streptavidin cell surface display work.
</P><P>Our first experimental steps were to perform Polymerase Chain Reactions (PCR) in order to isolate the different streptavidins.
</P><P><I><B>Cloning</B></I></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Gel A: 0.8% Gel electrophoresis to verify PCR of Streptavidin Alive coding sequence. Expected fragment size is 384bp and nothing in the reagent negative control. Gel B: 0.8% Gel electrophoresis to verify PCR of Streptavidin Dead coding sequence. Expected fragment size is 387bp and nothing in the reagent negative control Gel C: 0.8% Gel electrophoresis to verify PCR of Streptavidin iGEM coding sequence. Expected fragment size is 387bp and nothing in the reagent negative control</DIV></DIV></DIV></DIV><P>
The previous Gels show that our PCR reactions (68Â°C annealing temp, primers in registry) to amplify the streptavidin coding sequences went fine. The DNA fragments migrated at the expected sizes written under each images. The PCR to amplify our backbone (see primers in the registry) from the BBa_K523013 biobrick and to add overhangs complementary to the edges of the streptavidin sequences also worked.

With the PCR products we did a Gibson Assembly to obtained our desired constructs.
The Gibson Assembly reaction worked as expected, we obtained a product around the expected size 4171bp. 
Then another way to verify that our Gibson assembly reaction had worked was to transform cells with the Gibson products. The transformed DH5Î± competent cells should be able to grow on an antibiotic medium because of the resistance provided in the plasmid. The only problem with this second verification is that the Gibson product could still contain parental plasmid, whose insertion in cells can also lead to cells resistance and survival. So to verify if there is a carry over of the initial plasmids, we made negative controls. 
You can see bellow plates containing cells from the glycerol stocks we made of our gibson products .
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:242px;"><DIV class="thumbcaption">Image 1: INP_Streptavidin Alive Gibson transformed cells plated from the glycerol stock</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:242px;"><DIV class="thumbcaption">Image 2: INP_Streptavidin Dead Gibson  transformed cells plated from the glycerol stock</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:242px;"><DIV class="thumbcaption">Image 3: INP_Streptavidin iGEM Gibson  transformed cells plated from the glycerol stock</DIV></DIV></DIV><P>As the other experiments did not provide information about possible mutations, we also sent our constructs for sequencing, using VR and VF2 iGEM primers but also another forward primer we designed ourselves. The sequencing results were good, the protein is well inserted and there is no frame shift. You can see the sequencing results using the following iGEM registry links: <A href="http://parts.igem.org/Part:BBa_K1111012:Experience">results_ISA </A>; <A href="http://parts.igem.org/Part:BBa_K1111013:Experience">results_ISD </A>;<A href="http://parts.igem.org/Part:BBa_K1111014:Experience">results_ISI </A>.
</P><P>So up to now, with the previous information we had on our constructs, we could simply conclude that our basic cloning strategy worked.
</P><P>
Then moving on, we needed to test if the protein was expressed, if it was well localized at the outer membrane and that it was functional.
We thus designed immunostaining assays to verify localization at the outer membrane and biotin-staining assay for the protein functionality (<A href="https://2013.igem.org/Team:EPF_Lausanne/Protocols" class="external text" rel="nofollow">see protocols</A>).
Then we visualized the samples using a NIKON Eclipse TI inverted microscope <A href="http://en.wikipedia.org/wiki/Inverted_microscope" class="external autonumber" rel="nofollow">[1]</A> and also a Confocal microscope <A href="http://en.wikipedia.org/wiki/Confocal_microscopy" class="external autonumber" rel="nofollow">[2]</A>, which is one of the most powerful microscopes nowadays.
Unfortunately for all these assays, we didn't have an appropriate positive control as our constructs were newly designed by ourselves.
</P><P><I><B>Confocal Microscopy</B></I>Focusing on the biotin assays, we did the confocal experiment to better characterize this observed phenomena. You can observe the results below (Figure 1 to 4). 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 1: confocal microscopy image showing ISA cells incubated 1hr with biotin conjugated to FITC reporter</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 2: confocal microscopy image showing ISD cells incubated 1hr with biotin conjugated to FITC reporter</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 3:  confocal microscopy image showing Competent cells incubated 1hr with biotin conjugated to FITC reporter</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 4: confocal microscopy image showing streptavidin coated magnetic beads incubated 1hr with biotin conjugated to FITC reporter</DIV></DIV></DIV><P>With the confocal microscopy, we were not able to distinguish weather the biotin molecules were inside or on the outside of the cells. And also we can see that some cells in the negative control are fluorescent. But these results were promising so we decided to repeat the experiments, trying to optimize the biotin staining protocols we were using. 
</P><P><I><B>Inverted Microscopy</B></I></P><P>The four following images represent only the construct ISA. You can observe the bright field images followed by their corresponding composite images (bright field file merged with fluorescent file).
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 5.1: Bright field image of ISA cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 5.2: Composite image of ISA cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 6.1: Bright field image of ISA cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 6.2: Composite image of ISA cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter</DIV></DIV></DIV><P>You can observe localized specks on the cells surface. As expected, this pattern corresponds to the one we had while characterizing the INP_YFP fusion protein. 
The next images show the competent cells we use as negative controls, in the same conditions as explained above.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 7.1: Bright field image of competent cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 7.2: Composite image of competent cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter</DIV></DIV></DIV><P> It was not expected to find a pattern similar to the one found in the cells transformed with our constructs.
The next images corresponds to the two other constructs ISD and ISI. The cells look exactly like the ISA cells of Figures 1 and 2.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 9.1: Bright field image of ISD cells after 1hr incubation with biotin conjugated to FITC reporter</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 9.2: Composite image of ISD cells 1hr incubation with biotin conjugated to FITC reporter</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 10.1: Bright field image of ISI cells after 1hr incubation with biotin conjugated to FITC reporter</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 10.2: Composite image of ISI cells after 1hr incubation with biotin conjugated to FITC reporter</DIV></DIV></DIV><P>We thus decided to incubate ISA, ISD, ISI cells and competent cells with the some biotinylated nanoparticles conjugated with TexRed reporter. 
This time again you can observe in the following pictures, nanoparticles (red) and cells, but we cannot tell if they are attached or not. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 11: Composite image of ISA cells 1hr incubation with biotin coated nanoparticules conjugated to TexRed reporter</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 12: Composite image of ISD cells 1hr incubation with biotin coated nanoparticules conjugated to TexRed reporter</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 13: Composite image of ISI cells 1hr incubation with biotin coated nanoparticules conjugated to TexRed reporter</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 14: Composite image of competent cells 1hr incubation with biotin coated nanoparticules conjugated to TexRed reporter</DIV></DIV></DIV><P>With the previous results we cannot make a conclusion on the functionality and localization of our fusion proteins.
 We also  did immunostaining assays, consisting in incubating the cells with FITC fluorescent antibody but we didn't have a fluorescent signal. 
With all this even if we know that our cloning strategy was good, we cannot really conclude anything about the protein's expression and functionality.
</P><P><I><B>Growth curve and Western Blot</B></I> Finally we did a Western Blot and growth curve of our transformed cells to at least prove that our protein is indeed expressed. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:329px;"><DIV class="thumbcaption">Figure15Â : WB of the different INP - streptavidin constructs</DIV></DIV></DIV><P>We did a western blot expecting a signal at the ~50kDa bands corresponding to our protein. For this experiment we used a FITC conjugated, anti-streptavidin polyclonal Antibody. Despite the gel quality, you can see in the following picture a band of the desired size.
</P><P>Finally we did a growth curve of the cells containing our constructs. For this last experiment, the cells were inoculated overnight with 1mM IPTG. In the morning they were initially diluted to a certain OD and then the OD was regularly measured. We can see that the insertion of our plasmid affects the growth of E. coli cells.
</P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P> Our idea was to clone a fusion protein made of INP and Streptavidin  that will be able to target streptavidin at the cell surface. This streptavidin could then further be used to attached biotinylated nanoparticules to the cells. We thus designed assays to clone this fusion protein and verify its functionality and localization. For the biotin assay, we were expecting ISA, ISI cells to bind biotin and ISD, competent cells to not bind it. We had the right patterns for ISA and ISI. We thus though that we had things working, but by repeating experiments several times and looking carefully at the competent and the ISD cells we found fluorescent cells also having the same characteristics as the ISA and ISI cells, which is unfortunately a contradictory result. We also had our immunostaining experiments that didnât work, but the western blot results were positive.
So knowing that we did all this very carefully with a strict scientific approach we came to the conclusion that our protein is expressed but might be localized inside the cells. It could be that it is not well folded preventing the fusion product to behave exactly like BBa_K523013 protein.
</P><H2><SPAN class="mw-headline" id="New_Biobricks.2FMicroscopy_after_the_Jamboree_in_Lyon">New Biobricks/Microscopy after the Jamboree in Lyon</SPAN></H2><P>With the additional time we designed new Biobricks <A href="http://parts.igem.org/Part:BBa_K1111015">BBa_K1111015</A>&amp;<A href="http://parts.igem.org/Part:BBa_K1111016">BBa_K1111016</A>.
in order to prove surface localization of streptavidin. We designed a plasmid that contained INP-Streptavidin-YFP and one plasmid that encodes INP-YFP-Streptavidin. Indeed YFP should act as a reporter is the protein is expressed and could be target by an antibody as we did to characterize BBa_K523013. In addition an anti-Streptavidin antibody could be used to see colocalistion of the signals thus showing that the streptavidin is also at the outer membrane. In the following you see the results of the microscopy.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:587px;"><DIV class="thumbcaption">Figure X:<B>Inverted microscope composite images of INP-Streptavidin-YFP construct transformed DH5alpha E.coli</B> A: shows YFP fluorescence (514 nm) B: stained with biotinylated anti-YFP antibody and avidin dylight (650 nm) C: Merge of A and B, shows colocalization of fluorescent signal</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:588px;"><DIV class="thumbcaption">Figure X:<B>Inverted microscope composite images of INP-YFP-Streptavidin construct transformed DH5alpha E.coli</B> A: shows YFP fluorescence (514 nm) B: stained with biotinylated anti-YFP antibody and avidin dylight (650 nm) C: Merge of A and B, shows colocalization of fluorescent signal</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:587px;"><DIV class="thumbcaption">Figure X:<B>Inverted microscope composite images of DH5alpha E.coli, serving as negative control</B> A: shows YFP fluorescence (514 nm) B: stained with biotinylated anti-YFP antibody and avidin dylight (650 nm) C: Merge of A and B, no signall</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:587px;"><DIV class="thumbcaption">Figure X:<B>Inverted microscope composite images of INP-YFP transformed DH5alpha E.coli</B> A: shows YFP fluorescence (514 nm) B: stained with biotinylated anti-YFP antibody and avidin dylight (650 nm) C: Merge of A and B, shows colocalization of fluorescent signal</DIV></DIV></DIV></DIV><P><I><B>Discussion</B></I>
Clearly we see some colocalization of the red and yellow fluorescence with the anti-YFP antibody, but as we were short of time, we weren't able to incubate the antibody long enough, we only incubated for 15 minutes. We will also need to sequence verify the construct to support the colony PCR we did, <B>but still results are promising</B>.
</P><P>Here is the nomenclature we will further use:


1)Initially used plasmid:
INP_EYFP Biobrick: <A href="http://parts.igem.org/Part:BBa_K523013">BBa_K523013</A>Streptavidin Alive: <A href="http://parts.igem.org/Part:BBa_K1111010">BBa_K1111010</A>Streptavidin Dead: <A href="http://parts.igem.org/Part:BBa_K1111011">BBa_K1111011</A>Streptavidin iGEM: <A href="http://parts.igem.org/Part:BBa_K283010">BBa_K283010</A>.


2) PCR reactions products
INP_construct = BBa_K523013 after PCR without EYFP
SA = Streptavidin Alive; SD = Streptavidin Dead
SI = Streptavidin iGEM.


3) Gibson reaction product names:
ISA = INP_Streptavidin Alive gibson product <A href="http://parts.igem.org/Part:BBa_K1111012">BBa_K1111012</A>ISD =  INP_Streptavidin Dead gibson product<A href="http://parts.igem.org/Part:BBa_K1111013">BBa_K1111013</A>ISI = INP_Streptavidin iGEM BBa_K283010 gibson product<A href="http://parts.igem.org/Part:BBa_K1111014">BBa_K1111014</A>
To learn more about how we would continue, see <A href="https://2013.igem.org/Team:EPF_Lausanne/Next_steps" class="external text" rel="nofollow">Next Steps</A> or <A href="https://2013.igem.org/Team:EPF_Lausanne/Perspectives" class="external text" rel="nofollow">Perspectives</A></P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>[1] Surface display of Zymomonas mobilis levansucrase by using the ice-nucleation protein of Pseudomonas syringae, Jung HC, Lebeault JM, Pan JG. (1998)

[2] Identification and purification of a bacterial ice-nucleation protein, Wolber PK, Deininger CA, Southworth MW, Vanderkerckhove J, van Montagu M and Warren GJ (1986)

[3] Laboratory of Mark Howarth at Oxford university Department of Biochemistry, for providing us with the cloning plasmids for Streptavidin.
[4] Clustering of ice nucleation protein correlates with ice nucleation activity, Mueller GM, Wolber PK, Warren GJ. (1990)
</P></DIV></DIV><DIV class="footer"><DIV class="row-fluid"><DIV class="span4"><H4>Wiki Links</H4><UL><LI><A href="?action=edit">Edit this page</A></LI><LI><A href="/Special:RecentChanges">Recent Changes</A></LI><LI><A href="/Special:Upload?wpDestFile=Team-EPF-Lausanne_">Upload file</A></LI><LI><A href="/Special:SpecialPages">Special Pages</A></LI><LI><A href="/Special:Preferences">My Preferences</A></LI><LI><A href="/Special:Privacy_policy">Privacy Policy</A></LI><LI><A href="/Special:General_disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:EPF_Lausanne/Cell_surface_display">http://2013.igem.org/Team:EPF_Lausanne/Cell_surface_display</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF_Lausanne/Cell_surface_display" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF_Lausanne/Cell_surface_display" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Cell_surface_display&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Cell_surface_display&amp;oldid=354422" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>