"
Page
Discussion
View source
History
teams
Log in
 
Team:EPF Lausanne/Cell surface display
From 2013.igem.org
Home
Project
Overview
Nanoparticles
Cell surface display
Sensing-Effector
Results summary
Next steps
Perspectives
Notebook
Calendar
Protocols
Submitted parts
Team
Team
Official Team profile
Attributions
Acknowledgements
Gallery
Outreach
Overview
Kit
Woman & Engineer's day
Safety
Header
Contents
1 Overview
2 Our Biobricks
3 INP-YFP Characterization
4 Results
5 Discussion
6 New Biobricks/Microscopy after the Jamboree in Lyon
7 References
Overview
Figure 1:
Scheme of the cell surface display of streptavidin
We knew we wanted to bind Nanoparticles to the Bacteria using the very strong interaction between biotin and streptavidin.
Next we needed to decide which strategy would be the best so the E.coli expresses streptavidin on the outer membrane.
E.Coli are Gram-negative bacteria and they have two membranes, an inner one and an outer one, separated by a periplasmic space. Furthermore, the outer membrane is covered with a glycocalix, a layer of sugars.
There are different ways to express a heterologous protein on the outer membrane of E.coli. Peptides can be fused with outer membrane proteins or on fimbriae, fused to beta-autotransporters or lipoproteins, as for example the (Lpp)-OmpA protein. But then there is also another promising display system based on the use of ice nucleation protein (INP) from Pseudomonas syringae. Which we decided to use.
Our Biobricks
For Details, please move the cursor onto the image.
Ice nucleation Protein
INP originates from the bacteria Pseudomonas syringae. It is an outer membrane protein. That has successfully been used for cell surface display of enzymes, antibodies and antigens.
Streptavidin
Streptavidin originates from the bacterium Streptomyces avidinii. It is a tetrameric protein, each of the four subunit can bind one biotin molecule with a very high affinity.
For our Biobricks we used three different kinds of streptavidin.
pBS1C3
The three biobricks were cloned into the standard iGEM backbone plasmid, which contains a chloramphenicol resistance for selection.
We decided to engineer a fusion protein between Ice nucleation protein (INP) and Streptavidin. Because such fusion proteins have been successfully used for cell surface display of enzymes, single-chain antibodies and antigens. So we assumed it would work for streptavidin as well. Especially because Jung et al ( 1 ) even succeeded to express a functional surface enzyme, that was much bigger than streptavidin.
Ice nucleation protein is a glycosyl-phosphatidyliositol anchored outer membrane protein, this motif is quite unique in prokaryotes. When it is expressed in E.coli over 90% of INP is found in the outer membrane, Wolber et al ( 2 ).
Streptavidin
Streptavidin originates from the bacterium Streptomyces avidinii ( NCBI ). It is a tetrameric protein, and each of the four subunits can bind biotin with a very high affinity.
We cloned three Biobricks that contain different versions of streptavidin.
BBa_K1111012
Encodes a fusion protein between INP and a monovalent streptavidin provided by the laboratory of Mark Howarth at Oxford university Department of Biochemistry ( 3 )
BBa_K1111013
Encodes a fusion protein between INP and a dead version of streptavidin. It was designed to serve as a negative control for the microscopy functional assays, because its affinity to biotin is much lower.
BBa_K1111014
Encodes a fusion protein between INP and streptavidin we received in the Biobrick BBa_K283010 ( BBa_K283012, Main Page ).
INP-YFP Characterization
There is a Biobrick that contains INP in the iGEM registry (BBa_K523013). It was designed by the Edinburgh 2011 iGEM team . This Biobrick encodes a fusion protein of a truncated and codon optimized INP and yellow fluorescent protein (YFP) under control of the Lac promoter. The INP is truncated, it only contains the first 211 and last 97 amino acids that are normally encoded by the inaK gene. This is enough to target it to the outer membrane.
We improved the Biobrick BBa_523013 by extensively characterizing it.
We used confocal microscopy to see if it was expressed and eventually show surface localization.
Thanks to Thierry Laroche for taking the images with the confocal microscope.
Figure 2:
Confocal image of DH5alpha E.coli  transformed with BBa_K523013
A: shows YFP fluorescence (514 nm)
B: zoom of one Bacteria that shows spot of clustered YFP
Looking at E.coli we noticed that fluorescence was localized in specific specks, that the team of Edinburgh didn’t describe. We started looking for an explanation for this phenomena and found in the literature that INP forms aggregates in the cell membranes [3 ] This suggests that INP also forms aggregates when it is fused to other proteins. We continued characterizing this biobrick by first determining the percentage of bacteria that were fluorescent. We found that approximately 30% of E.coli expressed YFP.
The team of Edinburgh grew bacteria and lysed them. After centrifugation, they looked where the fluorescence seemed concentrated. On the transformed bacteria, it was mainly the pellet that appeared fluorescent, meaning that the fusion protein was embedded in the membrane. The negative control, in contrast, showed fluorescence in the pellet as well as in the supernatant with the same intensity. This method was used because their microscopy was not good enough.
The confocal microscope was really powerful. But we too, didn't succeed to proof surface localization of the fusion protein by simply looking at the intrinsic fluorescence of the YFP, because the signal was to diffuse.
So we came up with an alternative strategy: Immunofluorescent assay using an anti-YFP antibody and visualizing it with an inverted microscope.
Figure 3:
Inverted microscope composite images of BBa_K523013 transformed DH5alpha E.coli
A: shows YFP fluorescence (514 nm)
B: stained with biotinylated anti-YFP antibody and avidin dylight (650 nm)
C: Merge of A and B, shows colocalization of fluorescent signal
We did an immunofluorescent assay with an anti YFP antibody (Thanks Francesca Volpetti for giving us the antibody). In Figure 3 you can see the results we obtained with microscopy. The YFP signal (Figure 3 A) and the Cy5 signal (The antibody was tagged with NeutrAvidin DylLight Fluorescent Conjugate red, 650 nm from pierce) (Figure 3 B) superimpose nicely (Figure 3 C). Thus proofing that the fusion protein of INP and YFP is exported to the outer membrane.
Results
In the following section, you will find the results concerning our streptavidin cell surface display work.
Our first experimental steps were to perform Polymerase Chain Reactions (PCR) in order to isolate the different streptavidins.
Cloning
Gel A: 0.8% Gel electrophoresis to verify PCR of Streptavidin Alive coding sequence. Expected fragment size is 384bp and nothing in the reagent negative control.
Gel B: 0.8% Gel electrophoresis to verify PCR of Streptavidin Dead coding sequence. Expected fragment size is 387bp and nothing in the reagent negative control
Gel C: 0.8% Gel electrophoresis to verify PCR of Streptavidin iGEM coding sequence. Expected fragment size is 387bp and nothing in the reagent negative control
The previous Gels show that our PCR reactions (68°C annealing temp, primers in registry) to amplify the streptavidin coding sequences went fine. The DNA fragments migrated at the expected sizes written under each images. The PCR to amplify our backbone (see primers in the registry) from the BBa_K523013 biobrick and to add overhangs complementary to the edges of the streptavidin sequences also worked.
With the PCR products we did a Gibson Assembly to obtained our desired constructs. The Gibson Assembly reaction worked as expected, we obtained a product around the expected size 4171bp.
Then another way to verify that our Gibson assembly reaction had worked was to transform cells with the Gibson products. The transformed DH5α competent cells should be able to grow on an antibiotic medium because of the resistance provided in the plasmid. The only problem with this second verification is that the Gibson product could still contain parental plasmid, whose insertion in cells can also lead to cells resistance and survival. So to verify if there is a carry over of the initial plasmids, we made negative controls.
You can see bellow plates containing cells from the glycerol stocks we made of our gibson products .
Image 1: INP_Streptavidin Alive Gibson transformed cells plated from the glycerol stock
Image 2: INP_Streptavidin Dead Gibson  transformed cells plated from the glycerol stock
Image 3: INP_Streptavidin iGEM Gibson  transformed cells plated from the glycerol stock
As the other experiments did not provide information about possible mutations, we also sent our constructs for sequencing, using VR and VF2 iGEM primers but also another forward primer we designed ourselves. The sequencing results were good, the protein is well inserted and there is no frame shift. You can see the sequencing results using the following iGEM registry links: results_ISA ; results_ISD ; results_ISI .
So up to now, with the previous information we had on our constructs, we could simply conclude that our basic cloning strategy worked.
Then moving on, we needed to test if the protein was expressed, if it was well localized at the outer membrane and that it was functional. We thus designed immunostaining assays to verify localization at the outer membrane and biotin-staining assay for the protein functionality ( see protocols ). Then we visualized the samples using a NIKON Eclipse TI inverted microscope [1] and also a Confocal microscope [2] , which is one of the most powerful microscopes nowadays. Unfortunately for all these assays, we didn't have an appropriate positive control as our constructs were newly designed by ourselves.
Confocal Microscopy
Focusing on the biotin assays, we did the confocal experiment to better characterize this observed phenomena. You can observe the results below (Figure 1 to 4).
Figure 1: confocal microscopy image showing ISA cells incubated 1hr with biotin conjugated to FITC reporter
Figure 2: confocal microscopy image showing ISD cells incubated 1hr with biotin conjugated to FITC reporter
Figure 3:  confocal microscopy image showing Competent cells incubated 1hr with biotin conjugated to FITC reporter
Figure 4: confocal microscopy image showing streptavidin coated magnetic beads incubated 1hr with biotin conjugated to FITC reporter
With the confocal microscopy, we were not able to distinguish weather the biotin molecules were inside or on the outside of the cells. And also we can see that some cells in the negative control are fluorescent. But these results were promising so we decided to repeat the experiments, trying to optimize the biotin staining protocols we were using.
Inverted Microscopy
The four following images represent only the construct ISA. You can observe the bright field images followed by their corresponding composite images (bright field file merged with fluorescent file).
Figure 5.1: Bright field image of ISA cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter
Figure 5.2: Composite image of ISA cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter
Figure 6.1: Bright field image of ISA cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter
Figure 6.2: Composite image of ISA cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter
You can observe localized specks on the cells surface. As expected, this pattern corresponds to the one we had while characterizing the INP_YFP fusion protein.
The next images show the competent cells we use as negative controls, in the same conditions as explained above.
Figure 7.1: Bright field image of competent cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter
Figure 7.2: Composite image of competent cells after overnight incubation with 1mM IPTG and 1hr incubation with biotin conjugated to FITC reporter
It was not expected to find a pattern similar to the one found in the cells transformed with our constructs.
The next images corresponds to the two other constructs ISD and ISI. The cells look exactly like the ISA cells of Figures 1 and 2.
Figure 9.1: Bright field image of ISD cells after 1hr incubation with biotin conjugated to FITC reporter
Figure 9.2: Composite image of ISD cells 1hr incubation with biotin conjugated to FITC reporter
Figure 10.1: Bright field image of ISI cells after 1hr incubation with biotin conjugated to FITC reporter
Figure 10.2: Composite image of ISI cells after 1hr incubation with biotin conjugated to FITC reporter
We thus decided to incubate ISA, ISD, ISI cells and competent cells with the some biotinylated nanoparticles conjugated with TexRed reporter.
This time again you can observe in the following pictures, nanoparticles (red) and cells, but we cannot tell if they are attached or not.
Figure 11: Composite image of ISA cells 1hr incubation with biotin coated nanoparticules conjugated to TexRed reporter
Figure 12: Composite image of ISD cells 1hr incubation with biotin coated nanoparticules conjugated to TexRed reporter
Figure 13: Composite image of ISI cells 1hr incubation with biotin coated nanoparticules conjugated to TexRed reporter
Figure 14: Composite image of competent cells 1hr incubation with biotin coated nanoparticules conjugated to TexRed reporter
With the previous results we cannot make a conclusion on the functionality and localization of our fusion proteins.
We also  did immunostaining assays, consisting in incubating the cells with FITC fluorescent antibody but we didn't have a fluorescent signal.  With all this even if we know that our cloning strategy was good, we cannot really conclude anything about the protein's expression and functionality.
Growth curve and Western Blot
Finally we did a Western Blot and growth curve of our transformed cells to at least prove that our protein is indeed expressed.
Figure15 : WB of the different INP - streptavidin constructs
We did a western blot expecting a signal at the ~50kDa bands corresponding to our protein. For this experiment we used a FITC conjugated, anti-streptavidin polyclonal Antibody. Despite the gel quality, you can see in the following picture a band of the desired size.
Finally we did a growth curve of the cells containing our constructs. For this last experiment, the cells were inoculated overnight with 1mM IPTG. In the morning they were initially diluted to a certain OD and then the OD was regularly measured. We can see that the insertion of our plasmid affects the growth of E. coli cells.
Discussion
Our idea was to clone a fusion protein made of INP and Streptavidin  that will be able to target streptavidin at the cell surface. This streptavidin could then further be used to attached biotinylated nanoparticules to the cells. We thus designed assays to clone this fusion protein and verify its functionality and localization. For the biotin assay, we were expecting ISA, ISI cells to bind biotin and ISD, competent cells to not bind it. We had the right patterns for ISA and ISI. We thus though that we had things working, but by repeating experiments several times and looking carefully at the competent and the ISD cells we found fluorescent cells also having the same characteristics as the ISA and ISI cells, which is unfortunately a contradictory result. We also had our immunostaining experiments that didn’t work, but the western blot results were positive. So knowing that we did all this very carefully with a strict scientific approach we came to the conclusion that our protein is expressed but might be localized inside the cells. It could be that it is not well folded preventing the fusion product to behave exactly like BBa_K523013 protein.
New Biobricks/Microscopy after the Jamboree in Lyon
With the additional time we designed new Biobricks BBa_K1111015 & BBa_K1111016 . in order to prove surface localization of streptavidin. We designed a plasmid that contained INP-Streptavidin-YFP and one plasmid that encodes INP-YFP-Streptavidin. Indeed YFP should act as a reporter is the protein is expressed and could be target by an antibody as we did to characterize BBa_K523013. In addition an anti-Streptavidin antibody could be used to see colocalistion of the signals thus showing that the streptavidin is also at the outer membrane. In the following you see the results of the microscopy.
Figure X:
Inverted microscope composite images of INP-Streptavidin-YFP construct transformed DH5alpha E.coli
A: shows YFP fluorescence (514 nm)
B: stained with biotinylated anti-YFP antibody and avidin dylight (650 nm)
C: Merge of A and B, shows colocalization of fluorescent signal
Figure X:
Inverted microscope composite images of INP-YFP-Streptavidin construct transformed DH5alpha E.coli
A: shows YFP fluorescence (514 nm)
B: stained with biotinylated anti-YFP antibody and avidin dylight (650 nm)
C: Merge of A and B, shows colocalization of fluorescent signal
Figure X:
Inverted microscope composite images of DH5alpha E.coli, serving as negative control
A: shows YFP fluorescence (514 nm)
B: stained with biotinylated anti-YFP antibody and avidin dylight (650 nm)
C: Merge of A and B, no signall
Figure X:
Inverted microscope composite images of INP-YFP transformed DH5alpha E.coli
A: shows YFP fluorescence (514 nm)
B: stained with biotinylated anti-YFP antibody and avidin dylight (650 nm)
C: Merge of A and B, shows colocalization of fluorescent signal
Discussion
Clearly we see some colocalization of the red and yellow fluorescence with the anti-YFP antibody, but as we were short of time, we weren't able to incubate the antibody long enough, we only incubated for 15 minutes. We will also need to sequence verify the construct to support the colony PCR we did, but still results are promising.
Here is the nomenclature we will further use:
1)Initially used plasmid:
INP_EYFP Biobrick: BBa_K523013
Streptavidin Alive: BBa_K1111010
Streptavidin Dead: BBa_K1111011
Streptavidin iGEM: BBa_K283010 .
2) PCR reactions products
INP_construct = BBa_K523013 after PCR without EYFP
SA = Streptavidin Alive; SD = Streptavidin Dead
SI = Streptavidin iGEM.
3) Gibson reaction product names:
ISA = INP_Streptavidin Alive gibson product BBa_K1111012
ISD =  INP_Streptavidin Dead gibson product BBa_K1111013
ISI = INP_Streptavidin iGEM BBa_K283010 gibson product BBa_K1111014
To learn more about how we would continue, see Next Steps or Perspectives
References
[1] Surface display of Zymomonas mobilis levansucrase by using the ice-nucleation protein of Pseudomonas syringae, Jung HC, Lebeault JM, Pan JG. (1998)
[2] Identification and purification of a bacterial ice-nucleation protein, Wolber PK, Deininger CA, Southworth MW, Vanderkerckhove J, van Montagu M and Warren GJ (1986)
[3] Laboratory of Mark Howarth at Oxford university Department of Biochemistry, for providing us with the cloning plasmids for Streptavidin.
[4] Clustering of ice nucleation protein correlates with ice nucleation activity, Mueller GM, Wolber PK, Warren GJ. (1990)
Wiki Links
Edit this page
Recent Changes
Upload file
Special Pages
My Preferences
Privacy Policy
Disclaimers
Retrieved from " http://2013.igem.org/Team:EPF_Lausanne/Cell_surface_display "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
