<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF_Lausanne_Sensing-Effector"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF_Lausanne/Sensing-Effector">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF_Lausanne/Sensing-Effector&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Sensing-Effector&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF_Lausanne/Sensing-Effector&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF_Lausanne/Sensing-Effector" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF Lausanne/Sensing-Effector</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="all"><DIV class="menu"><NAV><DIV class="menu-item first-menu-item"><A href="/Team:EPF_Lausanne" class="main-item">Home</A></DIV><DIV class="menu-item"><A href="/Team:EPF_Lausanne/Overview" class="main-item">Project</A><UL><LI><A href="/Team:EPF_Lausanne/Overview">Overview</A></LI><LI><A href="/Team:EPF_Lausanne/Nanoparticles">Nanoparticles</A></LI><LI><A href="/Team:EPF_Lausanne/Cell_surface_display">Cell surface display</A></LI><LI><A href="/Team:EPF_Lausanne/Sensing-Effector">Sensing-Effector</A></LI><LI><A href="/Team:EPF_Lausanne/Results_summary">Results summary</A></LI><LI><A href="/Team:EPF_Lausanne/Next steps">Next steps</A></LI><LI><A href="/Team:EPF_Lausanne/Perspectives">Perspectives</A></LI></UL></DIV><DIV class="menu-item"><A href="/Team:EPF_Lausanne/Calendar" class="main-item">Notebook</A><UL><LI><A href="/Team:EPF_Lausanne/Calendar">Calendar</A></LI><LI><A href="/Team:EPF_Lausanne/Protocols">Protocols</A></LI><LI><A href="http://parts.igem.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2013&amp;group=EPF_Lausanne">Submitted parts</A></LI></UL></DIV><DIV class="menu-item"><A href="/Team:EPF_Lausanne/Team" class="main-item">Team</A><UL><LI><A href="/Team:EPF_Lausanne/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi">Official Team profile</A></LI><LI><A href="/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="/Team:EPF_Lausanne/Acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:EPF_Lausanne/Gallery">Gallery</A></LI></UL></DIV><DIV class="menu-item"><A href="/Team:EPF_Lausanne/Outreach_Overview" class="main-item">Outreach</A><UL><LI><A href="/Team:EPF_Lausanne/Outreach_Overview">Overview</A></LI><LI><A href="/Team:EPF_Lausanne/Kit">Kit</A></LI><LI><A href="/Team:EPF_Lausanne/Woman_and_engineers_day">Woman &amp; Engineer's day</A></LI></UL></DIV><DIV class="menu-item last-menu-item"><A href="/Team:EPF_Lausanne/Safety" class="main-item">Safety</A></DIV></NAV></DIV><DIV class="page"><DIV class="page-content"><H1 id="pageTitle">Header</H1><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Sensing"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Sensing</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Overview"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Experiments"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Experiments</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Restriction_Digest"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Restriction Digest</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#PCRs_and_Gibson_Assemblies"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">PCRs and Gibson Assemblies</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#OD_measurements"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">OD measurements</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#GFP_Measurements"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">GFP Measurements</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Microscopy"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Microscopy</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Expected_outcome"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Expected outcome</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Results"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Discussion"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Effecting"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Effecting</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Overview_2"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Experiments_2"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Experiments</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#PCR"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Gibson_Assembly"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Gibson Assembly</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Transformation.2C_control_PCR_and_sequencing"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Transformation, control PCR and sequencing</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Western_Blot"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">Western Blot</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Protein_purification_and_SDS-PAGE"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">Protein purification and SDS-PAGE</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Discussion_and_Future"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">Discussion and Future</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Sensing"><B>Sensing</B></SPAN></H2><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>The idea of this module was to transform bacteria with a plasmid that contains a promoter which senses a specific signal. Once this promoter senses the signal, it would initiate transcription of an enzyme which degrades the nanoparticle, thus releasing its contents.
We decided to use pH as the specific trigger that activates the promoter. As a proof of principle, we inserted three different promoters into three plasmids in front of the BioBrick BBa_I746916 (<A href="http://parts.igem.org/Part:BBa_I746908" class="external text" rel="nofollow">BBa_I746916, Main Page</A>) which encodes superfolded GFP. Then, we transformed cells with these plasmids and let them grow in media with different pHs in order to check the expression.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 1: A Graphical summary of what the sensing module contained. Each promoter was inserted into the same plasmid in front of GFP</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Experiments">Experiments</SPAN></H2><P>We chose the following three pH sensitive promoters: 
1.) Hya-promoter, isolated from the Escherichia Coli K-12 MG1655 strain <A href="http://parts.igem.org/Part:BBa_K1111002" class="external text" rel="nofollow">BBa_K1111002</A>
2.) Cad-promoter, isolated from the Escherichia Coli K-12 MG1655 strain <A href="http://parts.igem.org/Part:BBa_K1111004" class="external text" rel="nofollow">BBa_K1111004</A>
3.) BioBrick BBa_J23119, a constitutive promoter that was made by the 2006 Berkley team. <A href="http://parts.igem.org/Part:BBa_K1111005" class="external text" rel="nofollow">BBa_K1111005</A></P><P><B>Promoter Sequences</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Figure 2: Sequence of the Hya promoter</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Figure 3: Sequence of the Cad promoter</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Figure 4: Sequence of the constitutive promoter BBa_J23119</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Restriction_Digest">Restriction Digest</SPAN></H2><P>We digested the plasmid containing the biobrick BBa_I746908 (<A href="http://parts.igem.org/Part:BBa_I746908" class="external text" rel="nofollow">BBa_I746908</A>) which would serve as backbone for our constructs.  </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 5: 0,8% Gel, Restriction digest of the BioBrick BBa_I74608 with BamHI</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="PCRs_and_Gibson_Assemblies">PCRs and Gibson Assemblies</SPAN></H2><P>All the promoters were isolated by PCR and then assembled into the pSB1C3 Plasmid in front of the superfolded GFP. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 6: 0.8% Gel, PCRs of the three backbones</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 7: 0.8% Gel, PCRs of the Hya and the Cad promoter</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 8: 2% Gel, PCR of the constitutive promoter BBa_J23119</DIV></DIV></DIV><P>
Then, each of the constructs was used to transform DH5-alpha competent cells, which were first plated on chloramphenicol containing LB agar plates for selection of positive transformants, and then incubated into media with different pHs.
</P><P>We used four media: 
1.) LB-Chloramphenicol with a final pH of 5 (Adjusted with 10X MOPS+HCl) 
2.) LB-Chloramphenicol with a final pH of 6 (Adjusted with 10X MOPS)  
3.) LB-Chloramphenicol with a final pH of 7 (Adjusted with water) 
4.) LB-Chloramphenicol with a final pH of 8.5 (with 10X HEPES)</P><P>For each medium, we measured the OD of the cells. This helped us establish whether a low expression of GFP was really due to the fact that the promoters did not work, or simply resulting from the cells dying before they could initiate transcription and translation of the protein. It would also tell us whether an increase of fluorescence was only due to an accumulation of bacteria or to the actual accumulation of GFP.
</P><H2><SPAN class="mw-headline" id="OD_measurements">OD measurements</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 9: OD measurements of Bacteria with the hya promoter in different media</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 10: OD measurements of Bacteria with the cad promoter in different media</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 11: OD measurements of Bacteria with the constitutive promoter in different media</DIV></DIV></DIV><P>GFP expression was measured using a plate reader. We made three replicates for each promoter with each pH. Both the fluorescence and the 1:600 absorbance were measured, in order to normalize the obtained fluorescence. Then, the average for each replicate was calculated and used to establish a curve depicting GFP expression within each medium. </P><H2><SPAN class="mw-headline" id="GFP_Measurements">GFP Measurements</SPAN></H2><P><B>Hya Promoter</B></P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 12: Normalized Fluorescence at pH 5.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 13: Normalized Fluorescence at pH 6.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 14: Normalized Fluorescence at pH 7.0</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 15: Normalized Fluorescence at pH 8.5</DIV></DIV></DIV><P><B>Cad Promoter</B></P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 16: Normalized Fluorescence at pH 5.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 17: Normalized Fluorescence at pH 6.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 18: Normalized Fluorescence at pH 7.0</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 19: Normalized Fluorescence at pH 8.5</DIV></DIV></DIV><P><B>Constitutive Promoter</B></P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 20:  Normalized Fluorescence at pH 5.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 21:  Normalized Fluorescence at pH 6.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 22:  Normalized Fluorescence at pH 7.0</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 23:  Normalized Fluorescence at pH 8.5</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Microscopy">Microscopy</SPAN></H2><P>The Cells were also looked at under a microscope to qualitatively study their GFP expression.
</P><P><B>Hya Promoter</B></P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 24: FRAC_image, exposure: 400ms, Multiplier 1, pH 6.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 25: FRAC_image, exposure: 400ms, Multiplier 1, pH 8.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 26: FRAC_image, exposure: 550ms, Multiplier 1, pH 7</DIV></DIV></DIV><P><B>Cad Promoter</B></P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 27: FRAC_image, exposure: 400ms, Multiplier 50, pH 6.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 28: FRAC_image, exposure: 400ms, Multiplier 50, pH 8.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 29: FRAC_image, exposure: 400ms, Multiplier 50, pH 7</DIV></DIV></DIV><P><B>Constitutive Promoter (Positive Control)</B></P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 30:FRAC_image, exposure: 400ms, Multiplier 1, pH 6.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 31: FRAC_image, exposure: 400ms, Multiplier 1, pH 8.5</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 32: FRAC_image, exposure: 400ms, Multiplier 1, pH 7</DIV></DIV></DIV><P><I>Note</I>
We overexposed some of the images to be sure that there was no fluorescence in the bacteria in these media.
</P><H2><SPAN class="mw-headline" id="Expected_outcome">Expected outcome</SPAN></H2><P>The promoters Hya and Cad were supposed to initiate transcription upon external acidification, which would then cause the bacteria to express superfolded GFP and turn bright green. The biobrick BBa_J23119, being a constitutive promoter, would turn the bacteria green independently of their medium.
For our project, the promoters were inserted in front of a gelatinase so that the later would be expressed upon arrival in an acidic medium. The bacterium would then secrete the gelatinase causing the degradation of the nanoparticle and the release of its contents.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>We successfully managed to PCR amplify each of the three promoters as well as to insert them by Gibson Assembly into the pSB1C3 plasmid so that they would control the expression of superfolded GFP. These assemblies were all successful, which was determined by sequencing the plasmid isolated from the positive transformants on the plate. This sequencing result showed a 100% match between the reference promoter sequence and the inserted sequence that the bacteria contained.
Also, the bacteria containing the biobrick promoter turned green on the plates as well as in liquid culture, as expected.
</P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P><B>GFP expression under the hya and the cad promoters</B>
From the measurements we made on the plate reader, we were not able to conclude whether the promoters are really induced at low pH as the cells died very quickly. This was also supported by the OD measurements we took. On those graphs one can see very clearly that the cells which are in the acidic media die almost immediately. Furthermore, there was some expression in both the neutral and the basic medium. This would indicate that the promoter is either a constitutive promoter or that it is inducible but leaky. 
Comparing these graphs with the microscopy pictures, one cannot say conclusively that the promoter is inducible by pH. Even though under the microscope there are only fluorescent bacteria at acidic pH, the results from the plate reader clearly indicate that there are also fluorescent bacteria at neutral and high pH. However, one can see a trend that for both the hya and the cad promoter, the fluorescence is strongest at acidic pH and weakest at basic pH under the microscope. This trend cannot be seen in the Plate Reader measurements, as the bacteria at low pH die before being able to express GFP. 
Using the constitutive promoter as a positive control one can see that both hya and cad are weak promoters, as the fluorescence in the bacteria with the constitutive promoter is much stronger in acidic, neutral and basic media than the fluorescence in bacteria containing the hya or the cad promoter. Furthermore, a comparison with the constitutive promoter again supports the hypothesis that the bacteria grow best at neutral pH, as there are much more bacteria at pH 7 and at pH 6.5 or 8.5. <B>Conclusion</B>
Thus, we can conclude that the hya promoter and the cad promoter work, in the sense that they in fact induce expression of GFP. But from our data, we were not able to conclusively say that they are inducible by low pH.
Nevertheless, we were able to show that they are weak promoters that can be used to express a protein that should be present at low concentrations.
</P><P><B>With respect to our project</B>
In order to be sure that hya and cad are induced only at low pH, we would need to do further testing. As soon as their inducibility would be confirmed, we would insert them by gibson assembly in front of either GelE gelatinase or MMP2 gelatinase. Upon arrival in a medium with pH below 7 the enzyme would be synthesized and released.
</P><P><I>References:</I></P><P><A href="http://jb.asm.org/content/181/17/5250.full" class="external text" rel="nofollow">[1</A>] Journal of Bacteriology 
Response of hya Expression to External pH in Escherichia coli 
Paul W. King and Alan E. Przybyla 
J.Bacteriol. 1990 </P><P><A href="http://jb.asm.org/content/173/15/4851.abstract?sid=2a119bc7-0c26-4543-a671-d3a50521253f" class="external text" rel="nofollow">[2</A>] Journal of Bacteriology 
Mutational analysis and characterization of the Escherichia coli hya operin, which encodes [NiFe] hydrogenase1 
N K Menon, J Robbins, J C Wendt, K T Shanmuagam and A E Przybyla  J. Bacteriol 1991 </P><P><A href="http://jb.asm.org/content/174/8/2659.short" class="external text" rel="nofollow">[3</A>] Journal of Bacteriology
Nucleotode Sequence of the Escherichiea coli cad Operon: a system for neutralization of Low Extracellular pH 
Shi-Yuan meng and George N. Bennett
J. Bacteriol 1992</P><P><A href="http://jb.asm.org/content/174/8/2670.short" class="external text" rel="nofollow">[4</A>] Journal of Bacteriology 
Regulation of the Escherichia coli cad Operon: Location of a Site Required for Acid Induction 
Shi-Yuan meng and George N. Bennett 
J. Bacteriol 1992 </P><H2><SPAN class="mw-headline" id="Effecting"><B>Effecting</B></SPAN></H2><H2><SPAN class="mw-headline" id="Overview_2">Overview</SPAN></H2><P>The goal of the effecting part was to build a plasmid containing an arabinose-inducible promoter driving a tagged enzyme able to cleave gelatin, the polymer used to build our nanoparticles. We chose three different enzymes that were inserted into part BBa_I746908(<A href="http://parts.igem.org/Part:BBa_I746908" class="external text" rel="nofollow">[BBa_I746908, Main Page</A>) either between the promoter and GFP or replacing the GFP. All of the constructs were designed to have a His tag at the beginning of the protein so we could extract and purify our expressed protein.

We would then transform cells with our construct, make them express the gelatinase by inducing the promoter and then isolate and purify the protein.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 33: Gibson assembly: Protein without GFP</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 34: Gibson assembly: Protein with GFP</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Experiments_2">Experiments</SPAN></H2><P>The three different enzymes that we chose to use were gelatinase E (gelE) from Gram positive bacterium E.faecalis , metalloprotease 2 (MMP2) from H. sapiens <A href="http://plasmid.med.harvard.edu/PLASMID/GetCloneDetail.do?cloneid=2849&amp;species=Homo%20sapiens " class="external text" rel="nofollow">[1</A>] and metalloprotease 9 (MMP9) from M. musculus. We ordered the genomic DNA of E.faecalis as well as two plasmids containing the CDSs of MMP2 and MMP9 from plasmid.med.harvard.edu.
</P><H2><SPAN class="mw-headline" id="PCR">PCR</SPAN></H2><P>We first amplified the CDS of our three proteins by PCR. Because we had introduced a frame shift during the PCR due to our primers, we had constructs that had a stop codon in front of GFP. Thus we decided to continue with these construct knowing that we could still purify the protein because of the His-Tag we added.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 35: 0.8% Gel, PCR of the GelE insert</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 36: 0.8% Gel, PCR of the GelE backbone</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 37: 0.8% Gel, PCR of the MMP2 insert</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 38: 0.8% Gel,PCR of the MMP2 &amp; MMP9 backbones</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 39: 0.8% Gel, PCR of teh MMP9 insert</DIV></DIV></DIV><P>
We then purified the PCR products.
</P><H2><SPAN class="mw-headline" id="Gibson_Assembly">Gibson Assembly</SPAN></H2><P>The next step was the Gibson assembly reaction which would put together the inserts and their corresponding backbones. The reaction was performed only for the constructs that should have GFP as a reporter but had a stop in front of the latter because of the above mentioned frame shift. The three other constructs that were not supposed to have GFP after the gelatinase gene were not done, as the result would have been the same. 
The assembly worked for both the GelE and MMP2 constructs, but not for MMP9.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 40: Gibson Assembly: GelE-GFP construct</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 41: Gibson Assembly: MMP2-GFP construct</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Transformation.2C_control_PCR_and_sequencing">Transformation, control PCR and sequencing</SPAN></H2><P>We then transformed bacteria with the two constructs. Before sending them for sequencing we did a control PCR on each using primers that were specific to the insert, i.e. the GelE and the MMP2 gene.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Figure 42: Minipreps: GelE-GFP and MMP2-GFP constructs</DIV></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 43: Sequence analysis of the GelE gene with a forward Primer</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 44: Sequence analysis of the GelE gene with a reverse Primer</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 45: GelE sequencing overlaps</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 46: MMP2 sequencing overlaps</DIV></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 47: Figure 43: Sequence analysis of the MMP2 gene with a forward Primer</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 43: Sequence analysis of the MMP2 gene with a reverse Primer</DIV></DIV></DIV><P>
We also incubated the positive transformants with arabinose and looked at them under the microscope. As expected they did not express GFP.
</P><H2><SPAN class="mw-headline" id="Western_Blot">Western Blot</SPAN></H2><P>The next assay to test the expression and secretion of our proteins was a Western Blot, performed on the supernatant and cell lysate fractions of our liquid cell culture. The antibody used to detect our protein was an anti-His tag antibody, which was supposed to detect the His-tag that had been added to the CDS of our proteins. This assay unfortunately came out negative. We were unable to check for the presence of the His tag with the sequencing because the primers we used to do it did not initiate sequencing at the beginning of the protein as the sequencing starts a bit downstream of the primer binding site. The sequencing of the middle sequence showed a 97%-98% match between the insert and the reference sequence.
</P><H2><SPAN class="mw-headline" id="Protein_purification_and_SDS-PAGE">Protein purification and SDS-PAGE</SPAN></H2><P>A protein purification method under native conditions using a Ni-NTA spin kit was performed. The native conditions were chosen because we wanted to perform a digestion assay with the purified protein on the nanoparticles to assess their digestion capacities. 
The lysates, flow-through, wash and final eluate (supposed to contain the purified protein) were kept and analyzed with a NanoDrop before being used for an SDS-PAGE. The NanoDrop revealed that the eluates contained no protein, but that the wash fraction contained about 0.4 mg/ml of protein. Possible reasons for this were: 
1) the His tag was not properly added to the protein CDS 
2) the His tag was hidden in the protein structure and could not bind to the column since the purification was done under native conditions
3.) As the Kit was left at room temperature it is possible that the binding columns were defective 
When we did the SDS Page did not work due to a Coomassie stain issue.
</P><H2><SPAN class="mw-headline" id="Discussion_and_Future">Discussion and Future</SPAN></H2><P>We unfortunately could not come to a satisfying final result in the effecting part of the project because we could not prove that the plasmids actually conferred to the bacteria the ability to secrete a gelatin-degrading protein. We can later perform the digestion assay with the wash fraction of the Ni-NTA purification to see whether a gelatin-degrading protein is indeed present in it. Also, the next experiment would be to redo the SDS-PAGE on the remains of the Ni-NTA purification.
</P><P>To learn more about how we would continue, see <A href="https://2013.igem.org/Team:EPF_Lausanne/Next_steps" class="external text" rel="nofollow">Next Steps</A> or <A href="https://2013.igem.org/Team:EPF_Lausanne/Perspectives" class="external text" rel="nofollow">Perspectives</A></P></DIV><DIV class="footer"><DIV class="row-fluid"><DIV class="span4"><H4>Wiki Links</H4><UL><LI><A href="?action=edit">Edit this page</A></LI><LI><A href="/Special:RecentChanges">Recent Changes</A></LI><LI><A href="/Special:Upload?wpDestFile=Team-EPF-Lausanne_">Upload file</A></LI><LI><A href="/Special:SpecialPages">Special Pages</A></LI><LI><A href="/Special:Preferences">My Preferences</A></LI><LI><A href="/Special:Privacy_policy">Privacy Policy</A></LI><LI><A href="/Special:General_disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:EPF_Lausanne/Sensing-Effector">http://2013.igem.org/Team:EPF_Lausanne/Sensing-Effector</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF_Lausanne/Sensing-Effector" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF_Lausanne/Sensing-Effector" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Sensing-Effector&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF_Lausanne/Sensing-Effector&amp;oldid=338974" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>