<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU-Eindhoven_Team_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU-Eindhoven/Team/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Menu</TITLE><UL class="menu"><LI><A href="https://2017.igem.org/Team:TU-Eindhoven">Home</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project">Project</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project/Background" class="description">Background</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project/Design" class="description">Design &amp; Parts</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Demonstrate" class="proof of concept">Proof of 
                   Concept</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/InterLab" class="interlab study">Interlab 
                   study</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project/Protocols" class="protocols">Protocols</A></LI></UL><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Model/2D_Simulation" class="2D simulation">2D Simulation</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Model/Rule_based_model" class="rule based model">Rule based 
                   model</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Software" class="software tool">Software Tool</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results/DNA" class="dry lab">DNA Results</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results/Model" class="dry lab">Model Results</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results/Medal_criteria" class="medial criteria">Medal Criteria</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Applied_Design" class="human practices">Application Scenario</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Collaborations" class="collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Engagement" class="outreach">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Human_Practices/Safe_by_Design" class="safe by design">Safe by Design</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Human_Practices/Stakeholders" class="stakeholders">Stakeholders</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Members" class="members">Members</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Advisors" class="advisors">Advisors</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Attributions" class="attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Sponsors" class="sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Notebook" class="notebook">Notebook</A></LI></P><P><TITLE>Menu</TITLE><UL class="topbar"><LI><A href="#">Notebook</A></LI></UL></P><H6>Throughout the past few months we have tracked our progress in our notebooks, below you can find notebooks for wet lab experiments and for the modeling part. Click on the banners to see what we've done! For each notebook, a short overview per month is shown and the arrows can be used to dig deeper into detail.
</H6><LABEL class="collapsea" for="_a">Notebook wet lab</LABEL><DIV><H2>March - Building the Team</H2><H6>During this first month we divided the team roles and we started to brainstorm about project ideas and read a lot of literature for inspiration. Every week we had a meeting to discuss our progress and exchange ideas.</H6><H2>April - Constructing the Constructs</H2><H6>After a lot of brainstorm sessions we started to design our constructs and had meetings with our supervisors to finetune the idea. During the end of the month we started working on designing the DNA of the constructs. We also had a first session for our Human Practices as a preparation for SYNENERGENE at the end of the month.
The first round of gBlocks were designed in such a way that they can be used with the gibson assembly method. We used the cloning guide from iGEM team Eindhoven 2015 as a starting point. For detailed information on our protein design click <A href="https://2017.igem.org/Team:TU-Eindhoven/Project/Design" target="_blank">here.</A></H6><DIV><H3>Week 1 - 03/04 - 07/04</H3><H6>The first draft of our idea was made and the two constructs were designed:
a 14-3-3 tetramer with a n-term GFP and c-term ExoS peptide &amp; CT33 fused to a strep-tag and mCherry.</H6><H3>Week 2 - 10/04 - 14/04</H3><H6>Due to busy exam weeks we did not have a lot progress this week. However, we did have some meetings with professors to talk about our project and to finetune the idea. Especially developing a model to predict the behaviour of network formation was something that they were interested in.</H6><H3>Week 3 - 17/07 - 21/04</H3><H6>This week we had our first meeting with our supervisors about our two constructs. Several changes were made and we also had a small brainstorm about the eventual applications of our network. On the 21st we also went to The Hague for SYNENERGENE, more information can be found <A href="https://2017.igem.org/Team:TU-Eindhoven/Engagement" target="_blank">here.</A></H6><H3>Week 4 - 24/04 - 28/04</H3><H6>This week we started with designing the DNA of our constructs, in which we already were incorporating restriction sites to change the constructs if necessary. We also designed the constructs for different vectors, to be able to get a double expression.</H6></DIV><H2>May - First gBlocks ordered!</H2><H6>After a lot of work, redesign and adding details we ordered our gBlocks! We also got into contact with iGEM Potsdam, to see whether we could collaborate.</H6><DIV><H3>Week 5 - 01/05 - 05/05</H3><H6>This week we had a meeting with our professors to discuss the constructs we had designed so far. We got some feedback and we integrated this feedback into our constructs. We also looked into the literature to get more feeling for an eventual application.</H6><H3>Week 6 - 08/05 - 12/05</H3><H6>We also were busy working on the design of our constructs and had another meeting with our professors for some feedback. This week two of our team members also went to BCF career event, a career event for Bio, Chemistry, Food &amp; Farma to look voor potential sponsors.</H6><H3>Week 7 - 15/05 - 19/05</H3><H6>We checked our constructs for secondary structures and we scrambled the 14-3-3 construct to make sure that we get 4 completely different monomers, to make mutation possible. Some members of our team also talked to Patricia Dankers (an expert at our university in supramolecular biomaterials, including hydrogels), about the feasibility of our gel as an application.</H6><H3>Week 8 - 22/05 - 26/05</H3><H6>We ordered our gBlocks! We also started designing primers for the linearization of our vectors. Also, a start was made for looking for literature for the model. Via Traci Haddock from iGEM HQ we also contacted iGEM Potsdam, to see if a collaboration would be possible.</H6><H3>Week 9 - 29/05 - 02/06</H3><H6>Testing of restriction enzymes already available on the lab.</H6></DIV><H2>June - Prepping the Lab</H2><H6>During the busy month of June we started with some preparatory work, because we had to wait for our gBlocks to arrive. However, towards the end of the month they arrived! For some reason the Gibson Assembly of the 14-3-3 construct did not work, but the Gibson Assembly of the CT33 construct showed promising results.</H6><DIV><H3>Week 10 - 05/06 - 09/06</H3><H6>Preparation of antibiotic stock solutions and LB medium.</H6><H3>Week 11 - 12/06 - 16/06</H3><H6><I>14-3-3</I><UL><LI><SPAN>Plasmid amplification of pET28a into NovaBlue</SPAN></LI><LI><SPAN>pET28a miniprep</SPAN></LI><LI><SPAN>pET28a linearization</SPAN></LI><LI><SPAN>Designed and ordered primers for colony PCR &amp; sequencing</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>pBAD linearization</SPAN></LI><LI><SPAN>Designed and ordered primers for colony PCR &amp; sequencing</SPAN></LI></UL></H6><H3>Week 12 - 19/06 - 23/06</H3><H6><I>14-3-3</I><UL><LI><SPAN>pET28a linearization to obtain higher concentration</SPAN></LI><LI><SPAN>First Gibson assembly was set up with a to obtain our 14-3-3 tetramer</SPAN></LI><LI><SPAN>Transformation of product from Gibson assembly into competent cells (NEB 5 alpha) and plating. Grown overnight.</SPAN></LI><LI><SPAN>Colony PCR, showing somewhat promising results. Therefore, new primers were ordered for colony PCR.</SPAN></LI><LI><SPAN>Culturing of the colonies with (maybe) the correct plasmid.</SPAN></LI><LI><SPAN>Colony PCR with new primers. However, no better results were yielded.</SPAN></LI><LI><SPAN>Miniprep and sequencing of colonies.</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>pBAD linearization to obtain higher concentration. Failed, purity was also not suitable.</SPAN></LI><LI><SPAN>First Gibson assembly was set up with a gBlock for CT33. IDT had some problems with delivering the gBlock for CT52.</SPAN></LI><LI><SPAN>Transformation of product from Gibson assembly into competent cells (NEB 5-alpha) and plating. Store overnight.</SPAN></LI><LI><SPAN>Colony PCR, showing promising results</SPAN></LI><LI><SPAN>Culturing of the colonies with the correct plasmid</SPAN></LI><LI><SPAN>Miniprep and sequencing of colonies.</SPAN></LI></UL></H6><H3>Week 13 - 26/06 - 30/06</H3><H6><I>14-3-3</I><UL><LI><SPAN>A new Gibson assembly with adjusted parameters was set up.</SPAN></LI><LI><SPAN>Transformation of products from Gibson assembly into competent cells (NEB 5 alpha) and plating. Grown overnight.</SPAN></LI><LI><SPAN>Colony PCR, again no conclusive results.</SPAN></LI><LI><SPAN>Sequencing: linearisation of pET28a seems to give problems.</SPAN></LI><LI><SPAN>New vector linearisation of pET28a. Several temperatures were used since different calculators yielded different Tm's for the primers.</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>Sequencing results came in and showed a positive result: everything was built in for the CT33 construct.</SPAN></LI></UL></H6><H2>July - Summer of Troubleshooting</H2><H6>We decided to take some different approaches to the Gibson Assembly of 14-3-3 like freeze drying and resuspension into reduced volumes to obtain high gblock concentration.  We decided to take another approach for the 14-3-3 construct: add a dimer with a GFP to a vector with a dimer already present in the lab via double digestion and ligation. The CT33 construct showed some trouble during the purification. The CT52 gBlock could not be delivered, so we decided to exchange CT33 present in a vector with CT52 via double digestion and ligation. For the expression of CT33, we started to optimise the arabinose concentration.</H6><DIV><H3>Week 14 - 03/07 - 07/07</H3><H6><I>14-3-3</I><UL><LI><SPAN>gBlocks were freeze-dried and resuspended in half of the original volume to obtain higher concentrations.</SPAN></LI><LI><SPAN>Transformation of products from Gibson assembly into competent cells (NEB 5 alpha) and plating. Grown overnight.</SPAN></LI><LI><SPAN>Colony PCR</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>Transformation into BL21 &amp; small culturing.</SPAN></LI><LI><SPAN>Glycerol stock of the small culture was made and a large culture was set up overnight with arabinose after reaching an OD600 of 0.761.</SPAN></LI><LI><SPAN>Bacteria from large culture were collected by spinning down &amp; stored at -80oC over the weekend.</SPAN></LI></UL></H6><H3>Week 15 - 10/07 - 14/07</H3><H6><I>CT33/CT52</I><UL><LI><SPAN>Protein purification of CT33. Analysis of the fractions during purifications showed protein fractions flowing through at the loading step, so something is wrong.</SPAN></LI><LI><SPAN>Small culture of CT33 again.</SPAN></LI><LI><SPAN>Large culture of CT33 again, with an OD600 of 0.6.</SPAN></LI><LI><SPAN>Since the CT52 gBlock could not be delivered, we will use constructs available in the lab to replace the CT33 by CT52. </SPAN></LI><LI><SPAN>PCR reaction &amp; purification to introduce restriction sites to CT52 and CT33 construct.</SPAN></LI><LI><SPAN>Double digestion. Analysis showed very low concentration, so double digestion probably failed.</SPAN></LI></UL></H6><H3>Week 16 - 17/07 - 21/07</H3><H6><I>14-3-3</I><UL><LI><SPAN>Since we were having some trouble with our gBlocks, we decided to use constructs available in the lab to insert a 14-3-3 dimer with GFP into a vector with already a 14-3-3 dimer present. </SPAN></LI><LI><SPAN>PCR reaction &amp; purification of gBlock 1 (14-3-3 dimer + GFP).</SPAN></LI><LI><SPAN>Double digestion &amp; ligation.</SPAN></LI><LI><SPAN>Colony picking &amp; Colony PCR.</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>First attempt of the double digestion showed inconclusive results, so a second PCR reaction was set up.</SPAN></LI><LI><SPAN>Colony PCR of ligation 12-07</SPAN></LI><LI><SPAN>Expression of CT33 did not go as smoothly as planned, so an arabinose tune experiment was set up to test several temperatures for expression.</SPAN></LI></UL></H6><H3>Week 17 - 24/07 - 28/07</H3><H6><I>14-3-3</I><UL><LI><SPAN>Small culture of 14-3-3 tetramer with GFP from week 16.</SPAN></LI><LI><SPAN>Sequencing of culture of 14-3-3 tetramer with GFP from week 16.</SPAN></LI><LI><SPAN>Transformation into BL21 and NovaBlue.</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>New attempt of double digestion, but with higher concentrations.</SPAN></LI><LI><SPAN>Protein expression of CT33 with conditions as found in the arabinose tune experiment in week 16.</SPAN></LI></UL></H6><H2>August - Work Work Work</H2><H6>New design for the CT33 construct: a hairpin might be the problem, because of this the strep-tag is not expressed, making protein purification impossible. Addition of amino acids to prevent this hairpin via Liu PCR, we also designed a new gBlock. New gBlocks for 14-3-3 (no GFP, shorter constructs, so maybe this is easier to work with) also came in and we tried to make this Gibson Assembly work.</H6><DIV><H3>Week 18 - 31/07 - 04/08</H3><H6><I>CT33/CT52</I><UL><LI><SPAN>Protein purification of expression of CT33 from week 17. However, SDS analysis showed that the protein was not in the elution fraction, but more in the washing and loading fractions.</SPAN></LI><LI><SPAN>Again, double digestion. This time including negative control and taking more samples to see where it went wrong. SDS page shows promising results for CT52.</SPAN></LI><LI><SPAN>Transformation of CT33 plasmid into NovaBlue for amplification. However, this failed several times. No colonies were seen on the plate.</SPAN></LI><LI><SPAN>Design of new primers for LIU PCR. A hairpin is probably formed, preventing the streptag to be translated and thus giving a protein without a tag -&gt; hard to purify. 1 amino acid and 5 amino acids are added via LIU PCR.</SPAN></LI><LI><SPAN>Design of new gBlocks for CT33: CT33-mCherry-StrepTag and CT33-StrepTag-mCherry to prevent hairpin problems.</SPAN></LI></UL></H6><H3>Week 19 - 07/08 - 11/08</H3><H6><I>14-3-3</I><UL><LI><SPAN>Introduction of restriction sites and GFP, via PCR using primers, to gBlock 1, a 14-3-3 dimer</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>Protein purification of CT33. Analysis of the fractions during purifications showed protein fractions flowing through at the loading step, so something is wrong.</SPAN></LI><LI><SPAN>LIU PCR to add 1 and 5 amino acids respectively before the streptag to prevent hairpins.</SPAN></LI></UL></H6><H3>Week 20 - 14/08 - 18/08</H3><H6><I>14-3-3</I><UL><LI><SPAN>Gibson Assembly was performed with the original gBlocks, normal samples and freeze dried samples with higher concentrations.</SPAN></LI><LI><SPAN>Amplification of Gibson Assembly product with PCR.</SPAN></LI><LI><SPAN>Small culture and miniprep of pET28 empty, and with dT.</SPAN></LI><LI><SPAN>Transformation of pET28 empty and pET28 dT into NovaBlue.</SPAN></LI><LI><SPAN>Small culturing of transformed plasmides.</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>Small culture and miniprep of PBAD with CT33.</SPAN></LI><LI><SPAN>Testing of temperature range for primer linearization of pBAD to implement CT52 in a pBAD CT33 plasmid.</SPAN></LI></UL></H6><H3>Week 21 - 21/08 - 25/08</H3><H6><I>14-3-3</I><UL><LI><SPAN>Arrival of new gBlocks: additional GFP and ExoS were removed due to errors from IDT.</SPAN></LI><LI><SPAN>Gibson Assembly of new gBlocks.</SPAN></LI><LI><SPAN>Testing of restriction enzymes for classic cloning of new gBlocks into pET28 vector. Results looked good.</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>Check for abnormalities in CT33 plasmids, since linearisation is not going as planned. Results did not show abnormalities.</SPAN></LI><LI><SPAN>Test denaturation time for CT33 plasmid linearisation. Primers do not anneal sufficiently.</SPAN></LI><LI><SPAN>Temperature range test for LIU PCR with 5 AA insert. Amplification was unsuccesful.</SPAN></LI><LI><SPAN>LIU PCR for 1 AA insert, transformation, plating and subsequently gel electrophoreses showed no results -&gt; PCR did not work.</SPAN></LI><LI><SPAN>LIU PCR for 5 AA insert, transformation, plating and subsequently gel electrophoreses showed no results -&gt; PCR did not work.</SPAN></LI></UL></H6><H3>Week 22 - 28/08 - 01/09</H3><H6><I>14-3-3</I><UL><LI><SPAN>Transformation of Gibson Assembly product.</SPAN></LI><LI><SPAN>Double digestion and ligation of gBlocks into pET28 &amp; transformation -&gt; no colonies.</SPAN></LI><LI><SPAN>Redo ligation &amp; redo transformation with already ligated product.</SPAN></LI><LI><SPAN>PCR amplification of gBlock 1 for obtaining 14-3-3 dimer. Different concentrations of DMSO used to prevent formation of secondary structures.</SPAN></LI><LI><SPAN>Colony PCR of transformed ligated product -&gt; failed.</SPAN></LI><LI><SPAN>Linearization of pET28 vector.</SPAN></LI></UL></H6><H2>September - Fresh Start</H2><H6>As the new academic year began and the new gBlocks for CT33 came in, we had a lot of work to do. A new vector was used for this (pET28 instead of pBAD) to make expression easier. For 14-3-3, we had some positive results for the tetramer+GFP. Yay! This month we also finished the Interlab study, after coping with some problems with the protocol. Furthermore, we started with preparations for BioBricking halfway the month. </H6><DIV><H3>Week 23 - 04/09 - 08/09</H3><H6><I>14-3-3</I><UL><LI><SPAN>Small culture of pET28 vector and pET28 (dimer) for stock.</SPAN></LI><LI><SPAN>Double digestion and ligation of 14-3-3 dimer +GFP into pET28(dimer). Good results -&gt; transformation.</SPAN></LI><LI><SPAN>Colony PCR.</SPAN></LI><LI><SPAN>Miniprep of pET28 and pET28(dimer).</SPAN></LI><LI><SPAN>Glycerol stock of pET28 tetramer + GFP &amp; miniprep.</SPAN></LI><LI><SPAN>Preparation of sequencing.</SPAN></LI><LI><SPAN>Transformation of tetramer into NovaBlue.</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>New gBlocks arrived: CT33-mCherry-ST and CT33-ST-mCherry. To avoid problems with pBAD vector, pET28 vector was used.</SPAN></LI><LI><SPAN>Double digestion and ligation into pET28. Good results -&gt; transformation.</SPAN></LI><LI><SPAN>Colony PCR</SPAN></LI><LI><SPAN>Small culture of transformed products.</SPAN></LI><LI><SPAN>Preparation for sequencing.</SPAN></LI><LI><SPAN>pBAD(ST-mCherry-CT33) small culture &amp; miniprep.</SPAN></LI></UL><I>Interlab</I><UL><LI><SPAN>Transformation, not successful for every sample.</SPAN></LI></UL></H6><H3>Week 24 - 11/09 - 15/09</H3><H6><I>14-3-3</I><UL><LI><SPAN>Amplification of newest gBlocks -&gt; failed</SPAN></LI><LI><SPAN>Colony picking and colony PCR of transformed tetramer into NovaBlue -&gt;failed</SPAN></LI><LI><SPAN>Transformation of of pET28(tetramer+GFP) into BL21 &amp; small culture</SPAN></LI><LI><SPAN>Large culture of pET28(tetramer+GFP)</SPAN></LI><LI><SPAN>Again amplification of newest gBlocks -&gt; failed</SPAN></LI><LI><SPAN>Again amplification of newest gBlocks, with lower Tm values -&gt; failed</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>Transformation of pET28(CT33-mCherry-ST) and pET28(CT33-ST-mCherry) into BL21 and also NovaBlue</SPAN></LI><LI><SPAN>Transformation of pBAD(ST-mCherry-CT33) into NovaBlue</SPAN></LI></UL><I>Interlab</I><UL><LI><SPAN>Transformation with different protocol, shorter heatshock-&gt; still not succesful for every sample</SPAN></LI></UL></H6><H3>Week 25 - 18/09 - 22/09</H3><H6><I>Biobricking</I><UL><LI><SPAN>Start preparations.</SPAN></LI><LI><SPAN>Miniprep pSB1C3 vector.</SPAN></LI></UL><I>14-3-3</I><UL><LI><SPAN>Several conditions for small culture, to test induction and purification parameters for pET28(tetramer+GFP).</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>Several conditions for small culture, to test induction and purification parameters for pET28(CT33-mCherry-ST) and pET28(CT33-ST-mCherry).</SPAN></LI></UL><I>Interlab</I><UL><LI><SPAN>Transformation with different protocol, longer incubation time after addition of SOC medium .</SPAN></LI><LI><SPAN>Cultures seen on every plate -&gt; successful transformation.</SPAN></LI><LI><SPAN>Small culture.</SPAN></LI><LI><SPAN>Glycerol stock.</SPAN></LI></UL></H6><H3>Week 26 - 25/09 - 29/09</H3><H6><I>Biobricking</I><UL><LI><SPAN>Digest pSB1C3 vector with SeI-HF and XbaI -&gt; successful.</SPAN></LI><LI><SPAN>Digestion and ligation of CT33-mCherry-ST and CT33-ST-mCherry.</SPAN></LI><LI><SPAN>Transformation.</SPAN></LI><LI><SPAN>Colony PCR -&gt; promising results -&gt; small culture of good colony &amp; miniprep.</SPAN></LI><LI><SPAN>Miniprep samples sent for sequencing.</SPAN></LI></UL><I>14-3-3</I><UL><LI><SPAN>Sequencing results for tetramer+GFP came in and looked promising. However, the primers did not amplify everything, so extra primers were added to sequence the whole construct.</SPAN></LI><LI><SPAN>Small culture of tetramer+GFP in NovaBlue.</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>Colony PCR of CT33-mCherry-ST -&gt; no promising results.</SPAN></LI><LI><SPAN>LIU PCR to insert 1 AA &amp; 5 AA. Previous time it did not work because the wrong plasmid was used. For 5 AA insert, two different Tm's were used.</SPAN></LI></UL><I>Interlab</I><UL><LI><SPAN>Small culture from glycerol stock from last week.</SPAN></LI><LI><SPAN>Measurements of standard curve &amp; OD600/Abs600.</SPAN></LI></UL></H6></DIV><H2>October - Wrapping Things Up</H2><H6>As the deadlines were getting closer, we really had to work hard on the lab to make sure that we would have enough results and enough BioBricks to send in. To do this, CT33 and 14-3-3 were expressed at higher quantities and had to be purified. Additionally we tried to obtain a dimer and trimer of 14-3-3 with GFP to have more reference material. However, we obtained too little material to work with. Protein purification also gave some problems. Biobricking of the CT33 was successful, but we did not manage to BioBrick our 14-3-3 tetramer on time. But, at the end of the month we had some great results: gelation occurred!</H6><DIV><H3>Week 27 - 02/10 - 06/10</H3><H6><I>BioBricking</I><UL><LI><SPAN>Primer design to introduce restriction sites.</SPAN></LI><LI><SPAN>Design primers for multi site-directed mutagenesis to remove PstI and EcoRI site from pET(dimer).</SPAN></LI></UL><I>14-3-3</I><UL><LI><SPAN>Gibson Assembly.</SPAN></LI><LI><SPAN>Transformation of products of Gibson Assembly.</SPAN></LI><LI><SPAN>14-3-3 dimer transformed into Nova Blue.</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>Colony PCR of Liu &amp; of CT33-StrepTag-mCherry in PET28. Small colony of cultures with good results.</SPAN></LI><LI><SPAN>Glycerol stock, miniprep and sequencing.</SPAN></LI><LI><SPAN>Transformation of samples from miniprep into BL21.</SPAN></LI></UL></H6><H3>Week 28 - 09/10 - 13/10</H3><H6><I>BioBricking</I><UL><LI><SPAN>Double digestion &amp; ligation of tetramer+GFP into pSB1C3 vector.</SPAN></LI><LI><SPAN>Transformation into Nova Blue.</SPAN></LI><LI><SPAN>Double digestion &amp; ligation of CT33-StrepTag-mCherry into pSB1C3 vector.</SPAN></LI><LI><SPAN>Transformation into Nova Blue.</SPAN></LI><LI><SPAN>Digestion of vector.</SPAN></LI><LI><SPAN>Miniprep of CT33-StrepTag-mCherry and pSB1C3.</SPAN></LI><LI><SPAN>Digestion of CT33-StrepTag-mCherry and ligation.</SPAN></LI><LI><SPAN>Colony picking &amp; PCR on tetramer+GFP -&gt; no good results. Other colonies tried.</SPAN></LI></UL><I>14-3-3</I><UL><LI><SPAN>Colony picking &amp; PCR of Gibson Assembly from last week -&gt; failed.</SPAN></LI><LI><SPAN>Small culture and subsequently large culture set up for 14-3-3 monomer.</SPAN></LI></UL></H6><H3>Week 29 - 16/10 - 20/10</H3><H6><I>Biobricking</I><UL><LI><SPAN>14-3-3 trimer and dimer with GFP: PCR, double digestion and ligation into pSB1C3 via XbaI and SpeI -&gt; failed.</SPAN></LI></UL><I>14-3-3</I><UL><LI><SPAN>Constructing a dimer and trimer using the tetramer and PCR.</SPAN></LI><LI><SPAN>Transformation into NovaBlue -&gt; dimer succeeded, trimer failed.</SPAN></LI><LI><SPAN>Small culturing of dimer and transformation into BL21.</SPAN></LI><LI><SPAN>Large culture of 14-3-3 tetramer and dimer with GFP in BL21</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>Large culture of CT33 in BL21.</SPAN></LI></UL></H6><H3>Week 30 - 23/10 - 27/10</H3><H6><I>Biobricking</I><UL><LI><SPAN>PCR and sequential digestion and ligation using XbaI and SpeI -&gt; still failing. Maybe the construct is too large.</SPAN></LI><LI><SPAN>CT33 part in pSB1C3 vector and 14-3-3 tetramer + GFP and 14-3-3 dimer + GFP parts in pET28 vector dried and sent to Boston.</SPAN></LI></UL><I>14-3-3</I><UL><LI><SPAN>Purified 14-3-3 tetramer via Ni column and buffer exchange to prevent protein aggegrates. Results show a not 100% pure protein, but target compound was present.</SPAN></LI></UL><I>CT33/CT52</I><UL><LI><SPAN>Purified via Strep column. Results show a not 100% pure protein, but target compound was present.</SPAN></LI></UL><I>Assays</I><UL><LI><SPAN>FRET was tested by mixing the main components, transferring them to a well plate and exciting them with a wavelength for GFP and reading the emission of mCherry. Results looked good but can be optimized since a background signal of mCherry activation is still present.</SPAN></LI></UL></H6><H3>Week 31 - 30/10 - 03/11</H3><H6><I>14-3-3 &amp; CT33/CT52</I><UL><LI><SPAN>To further purify both proteins SEC was used, but results showed that still some remaining extra compounds are present.</SPAN></LI></UL><I>Assays</I><UL><LI><SPAN>FRET was optimized by comparing the excitation spectra of GFP and mCherry and finding a wavelength where GFP still is excitated but mCherry is not. Results showed that this optimum was found.</SPAN></LI></UL></H6></DIV></DIV><LABEL class="collapsea" for="_b">Notebook modeling</LABEL><DIV><H2>March - Exploring options</H2><H6>In the first month we divided the roles of the team members, and the team members with experience and interest in modeling were became members of the modeling crew, with the most experienced one as captain. The first tasks were to read literature and brainstorm about way to simulate the system.</H6><H2>April - Contacting Experts</H2><H6>As the idea of the constructs became clearer, it became time to search for people that could advise us about how to start with the model. The paper ‘Compositional Control of Phase-Separated Cellular Bodies’ <SUP>[1]</SUP> published in Cell by Michael Rosen looked very promising as they used Matlab to simulate their system. We got a very enthusiastic response and they were happy to help us. He forwarded our mail to the person responsible for the model, Salman Rosen, and he send us some functions together with an explanation. The functions were a nice source of inspiration, but unfortunately we had to develop functions of our own.</H6><H2>May - New Insights</H2><H6>In the next period we had conversation with the modeling captains of the TU Eindhoven team of 2016 and 2015 and asked for their experience. We explained what we were thinking about and how we wanted to execute our plan and got some feedback as response. Furthermore, we were inspired by the course “Molecular Modeling” that let us simulate lipids by using a Atomistic Molecular Dynamics Simulation method. Here we had to apply Monte Carlo and Glauber dynamics, of which Monte Carlo also seemed to be useful for our designed constructs. </H6><H2>June - First Attempts</H2><H6>As the lab was preparing for the delivery of the gBlocks, we started with working out the formulas we wanted to use and developed our first model. The first model was based on concentration dependent reaction rates by using an ODE solver. Here we had to define all the possible reactions that could occur in the system. When looking critically to this method, it appeared to be insufficient, as it was impossible to define all the possible reactions.The second attempt included a Molecular Dynamics simulation based on Monte Carlo, followed by investigating the possibilities of a Gillespie algorithm.</H6><H2>July - 2D Glasses</H2><H6>After having some discussions with experts of the Gillespie algorithm, we could conclude that it had the same problem as the ODE solver. This meant we had to continue with expanding the Molecular Dynamics simulation. The first step was to define the molecules and let them move and rotate separately. Also all the other constraints and interaction between the molecules were added, leading to a simulations that showed network formation. The drawback was that it only was a 2D simulation, while in reality it will be in 3D. Additionally, the scale was too low and the constraints needed to many approximations to use it for a useful prediction and verification of the designed system. The simulation yielded a nice short movie, but we had to continue with searching for a different approach, which we found in the BioNetGen Language in combination with NFsim.</H6><H2>August - Virtual Reality</H2><H6>Every (coding) language is different and has its own way of functioning and needs certain requirements. We discovered that when we started with the installation of NFsim, advised by one of our supervisors. He himself hadn’t used it, but knew someone who did, and he needed to use Linux to get it functional. We, having a Windows system, therefore needed to instal a Virtual Box, together with Linux, NFsim and Matlab. This took quite some time and was very frustrating and unrewarding. Especially as we discovered that we could still use Windows by using NFsim somewhat differently. After the installation was successful, we walked through the manual and tried to understand the new language and experimented with example models.</H6><H2>September - Trial and Error</H2><H6>Learning a new language takes some time, but luckily, it was doable in the limited time given by the iGEM deadlines. Of course it took some time to implement the system, but after some trial and error, we ended up with a model that could be simulated with NFsim. However, there were still some problems left, including generating an output in the desired format and having reliable parameters of our system. This led us to dive back into the literature and contact some experts. Both led to success, but then a new problem arose: we only saw a equilibrium state of our system, but no transition form every unbound to a (small) network formation. Different simulation times, steps, rates and so on were tested, but an improvement was not found yet.</H6><H2>October - Enlightening</H2><H6>Units, remember, those are very important!. After doing a unit control, we discovered that we made a small mistake, with large influences on our simulation. So that you know, NFsim works with molecules instead of concentrations. Being used to an ODE solver, we missed the fact that we had to translate the association rate to a molecule dependent value instead of a concentration dependent value. We knew that we were working with molecules instead of concentrations, so we already implemented a volume definition, and together with the number of Avogadro, converted them. After also converting the association rate, the simulation worked much faster. Another insight we gained was that most times, the process is better visualized on a logarithmic time scale. Doing some simulations with different time steps, we could make such a logarithmic time scale plot, and indeed, we saw the transition from the begin state to an equilibrium! 
Having the simulation working for our system, we could work on simulating the system of the iGEM team from Potsdam.
</H6><DIV class="sources"><SUP>[1] </SUP>S. F. Banani, A. M. Rice, W. B. Peeples, Y. Lin, S. Jain, R. Parker, M. K. Rosen, S. F. Banani, A. M. Rice, W. B. Peeples, Y. Lin, S. Jain, R. Parker, and M. K. Rosen, “Compositional Control of Phase-Separated Cellular Bodies,” Cell, vol. 166, pp. 651–663, 2016. </DIV></DIV><P><TITLE>Footer</TITLE></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>