Team:TU-Eindhoven/Project/Background
Menu
Home
Project
Background
Design & Parts
Proof of                     Concept
Interlab                     study
Protocols
Model
2D Simulation
Rule based                     model
Software Tool
Results
DNA Results
Model Results
Medal Criteria
Human Practices
Application Scenario
Collaborations
Outreach
Safe by Design
Stakeholders
Safety
Team
Members
Advisors
Attributions
Sponsors
Notebook
Menu
Background
Our project includes different proteins and an underlying concept seen in nature. In this section we will provide some additional information about the parts and principles on which our project is based.
Gelation and Liquid-Liquid Phase Separation
As the name of our project (GUPPI) states, we are making a system that initiates gell formation based  multivalent protein protein interactions. In the design process we took inspiration from a common cellular organiser; Membraneless organelles.[1]
Classical views on cellular organisation show that this is carried out by membrane surrounded organelles like the Golgi and Endoplasmic Reticulum. In recent years a lot has become clear about a new type of organizer, called membraneless organelles. These organelles are phase separated from their surrounding environment and function to separate or concentrate biological macromolecules.
Examples of membraneless organelles are the nucleolus, P bodies and stress granules. There is a wide range of the biological functions that membraneless organelles facilitate. The nucleolus is an example of a membraneless organelle and it produces ribosomes. P-bodies, also membraneless, are required for the translation and transport of mRNA (lack of P-bodies results in large amounts of untranslated mRNA). Next to the protein processing, protein storage and transport, the organelles are also important for signalling. An example is the activation of MAP kinase by the clustering of binding partners to a phosphorylated TCR.[2]
Phase separation is a phenomenon seen when molecules reach their miscibility limit in solution. i.e. if a molecule concentration is too high it will phase separate. What we know of membraneless organelles is that these often contain multivalent macromolecules which have either intra- or inter-molecular interactions. Multivalent molecules inherently assembly into larger structures leading to high local concentrations driving phase separation of membraneless organelles into liquid-like droplets [1,3-4]. You can think of this like oil droplets in water. In other words, phase separation is a consequence of high molecular concentration i.e. clustering of molecules. Since our system consists out of two proteins which undergo intermolecular interactions, we expect them to cluster and phase separate.
Figure 1: Water in Oil
The Scaffold Protein 14-3-3
Over 15 years ago, scientists discovered the first protein scaffolds: proteins that regulate the assembly of other proteins. Scaffold proteins have numerous functions in the cell, one of which is mediating signaling transduction or other networks. [5]These scaffold proteins bind components of a signaling pathway, to help localize signaling to a specific part of the cell or to increase the efficiency of the signaling cascade.[6]
One of the many scaffold proteins is the 14-3-3 protein. It is becoming more popular to use this protein in synthetic biology. The 14-3-3 proteins are a family of proteins which are well preserved in evolution and are present in all eukaryotic cells. Many organisms contain isoforms: smaller and more simplistic eukaryotes like yeast contain only two 14-3-3 genes, however, the bigger and more complex eukaryotes can contain up to fifteen different isoforms. Mammalian cells contain seven different isoforms, namely β, ε, γ, η, σ, τ, ζ.[7]
One of the 14-3-3 proteins that is used in synthetic biology is from the Nicotiana plumbaginifolia (Tobacco) plant (T14-3-3). [8] T14-3-3 proteins dimerize to form a functional scaffold and every monomer contains a bundle of nine antiparallel alfa-helices. Helices H3, H5, H7 and H9 form the “amphipathic ligand-binding groove” (see Figure 2 ), in which other proteins can bind and interact with their co-protein.[7] There are different T14-3-3 variants. The variant which will be used and mentioned as 14-3-3 is the T14-3-3cΔC. The schematic overview of this interaction can be seen in figure 3.
Figure 2: Structural information on the 14-3-3 isoform (blue) that we used and its interaction with CT33 (orange) stabilized by fusicoccin (FC) (green). A) The 14-3-3 monomer has a hydrophobic side (circled red) that interacts with other 14-3-3 monomers creating a dimer. B) 14-3-3 dimer shown. The left monomer is depicted in a surface model and the right monomer shows the 9 alpha helixes from with the monomers are built up. C) a CT33 terminus in a 14-3-3 monomer for clarification, note how the CT33 terminus fits nicely in the 14-3-3 groove. D) 14-3-3 dimer with two interacting CT33 termini. E) zoomed view of the FC stabilizer seen in close proximity to the CT33 c-terminus stabilizing the interaction
Figure 3: Schematic overview of interaction between 14-3-3 and CT33
The Binding Partner CT33 and the Inducer Fusicoccin
Fusicoccin is a small molecule that is produced by the fungus Fusicoccum amygali. [9] It is an organic compound with varying ring structures and because of its size, able to diffuse through cell membranes. Fusicoccin is a phytotoxic molecule, meaning it has detrimental effects on plants.[10] It is unknown yet if fusicoccin could have any detrimental effects on the human body. However, Skwarczynska, Molzan, and Ottmann experienced no deleterious effects of fusicoccin in HEK293T or in HeLa cells up to a concentration of 60 µM.[11] One of the binding partners of 14-3-3 is plant plasma membrane H+-ATPase, this protein transports protons over the plasma membrane. This interaction is greatly stabilized by fusicoccin, which binds into a gap between 14-3-3 and H+-ATPase (see Figure 2 ).[12]
Thus the activity of the H+-ATPase is increased when fusicoccin is present, leading to an increase of the membrane potential and metabolic processes.[13-14]] Those C-terminal regions to which 14-3-3 proteins bind have been isolated and are called CT33 (the last 33 amino acids of the C-terminal region of the H+-ATPase. CT33 is thus a 33 amino acid long protein, which needs a free C-terminal end in order to achieve a stabilized binding with 14-3-3. The N-terminus can be used to link other proteins to, which in our case is a Strep-tag® II.
The Signaling Components
In each construct we incorporated a fluorophore, to visualize the location of our proteins and to detect if the two constructs are in close proximity by measuring Förster Resonance Energy Transfer (FRET) between the proteins GFP and mCherry. [15]
GFP is a Green Fluorescent Protein that can be excitated with a wavelength of 395 nm and has an emission peak around 509 nm. mCherry is a very stable red fluorescent protein with can be excitated with a wavelength of 587 nm and has an emission peak around 610 nm. mCherry can also absorb light of a lower wavelength, for example that emitted by GFP, making GFP and mCherry a couple that can be used to detect FRET, even though it is not the most optimal couple. The reason we choose this couple instead of a combination of fluorophores with a more efficient FRET, is that GFP and mCherry are better if you want to localize the protein under a microscope.
Figure 4: GFP structure
Figure 5: mCherry structure
[1] S. F. Banani, H. O. Lee, A. A. Hyman, and M. K. Rosen, “Biomolecular condensates: organizers of cellular biochemistry,” Nat. Rev. Mol. Cell Biol., vol 18, pp 285-298, 2017.
[2] X. Su, X. Su, J. A. Ditlev, E. Hui, W. Xing, S. Banjade, J. Okrut, D. S. King, J. Taunton, M. K. Rosen, and R. D. Vale, “Phase separation of signaling molecules promotes T cell receptor signal transduction,” Nature, vol. 9964, no. April, pp. 1–9, 2016.
[3] L. Bergeron-Sandoval, N. Safaee and S.W. Michnick, “Mechanisms and Consequences of Macromolecular Phase Separation”, Cell, vol. 165, issue 5, pp. 1067-1079, 2016.
[4] Y. Lin, D.S.W. Protter, M.K. Rosen and R. Parker, “Formation and Maturation of Phase-Separated Liquid Droples by RNA-Binding Proteins”, Molecular Cell, vol. 60, no. 2, pp. 208-219, 2015.
[5] M. Good, J. Zalatan and W. Lim, “Scaffold Proteins: Hubs for Controlling the Flow of Cellular Information”, Science, vol. 332, no. 6030, pp.680-686, 2011.
[6] A. Shaw and E. Filbert, ”Scaffold proteins and immune-cell signalling”, Nat Rev Immunol, vol. 9, no. 1, pp.47-56, 2009.
[7] T. Obsil and V. Obsilova, ”Structural basis of 14-3-3 protein functions”, Seminars in Cell & Developmental Biology, vol. 22, no. 7, pp.663-672, 2009.
[8] C. Ottmann, S. Marco, N. Jaspert, C. Marcon, N. Schauer, M. Weyand, C. Vandermeeren, G. Duby, M. Boutry, A. Wittinghofer, J. Rigaud and C. Oecking, “Structure of a 14-3-3 Coordinated Hexamer of the Plant Plasma Membrane H+-ATPase by Combining X-Ray Crystallography and Electron Cryomicroscopy”, Molecular Cell, vol. 25, no. 3, pp.427-440, 2007.
[9] M. Bury, A. Andolfi, B. Rogister, A. Cimmino, V. Mégalizzi, V. Mathieu, O. Feron, A. Evidente and R. Kiss, “Fusicoccin A, a Phytotoxic Carbotricyclic Diterpene Glucoside of Fungal Origin, Reduces Proliferation and Invasion of Glioblastoma Cells by Targeting Multiple Tyrosine Kinases”, Translational Oncology, vol. 6, no. 2, pp 112-123, 2013.
[10] F. Johansson, M. Sommarin and C. Larsson, “Fusicoccin Activates the Plasma Membrane H + -ATPase by a Mechanism Involving the C-Terminal Inhibitory Domain”, The Plant Cell, vol. 5, no. 3, pp 321, 1993.
[11] A. Ballio, E.B. Chain, P. De Leo, B.F. Erlanger, M. Mauri and A. Tonolo, “Fusicoccin: a new wilting toxin produced by Fusicoccum amygdali”, Nature, vol. 203, no. 4942, pp 297, 1964.
[12] L.A. Banaszynski, C.W. Liu and T.J. Wandless, “Characterization of the FKBP⊙ Rapamycin⊙ FRB Ternary Complex”, Journal of the American Chemical Society, vol. 127, no. 13, pp 4715-4721, 2005.
[13] R. Upadhyay, “Advances in microbial toxin research and its biotechnological exploitation”, New York: Kluwer Academic/Plenum Pub., pp 245, 2002.
[14] F. Giordanetto, A. Schäfer and C. Ottmann, “Stabilization of protein–protein interactions by small molecules”, Drug discovery today, vol. 19, no. 11, pp 1812-1821, 2014.
[15] B.T. Bajar, E.S. Wang, S. Zhang, M.Z. Lin and J. Chu, "A Guide to Fluorescent Protein FRET Pairs", Sensors (Basel), vol 16, no. 9, pp 1488, 2016
Footer
