<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU-Eindhoven_Results_DNA skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU-Eindhoven/Results/DNA</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Menu</TITLE><UL class="menu"><LI><A href="https://2017.igem.org/Team:TU-Eindhoven">Home</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project">Project</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project/Background" class="description">Background</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project/Design" class="description">Design &amp; Parts</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Demonstrate" class="proof of concept">Proof of 
                   Concept</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/InterLab" class="interlab study">Interlab 
                   study</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project/Protocols" class="protocols">Protocols</A></LI></UL><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Model/2D_Simulation" class="2D simulation">2D Simulation</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Model/Rule_based_model" class="rule based model">Rule based 
                   model</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Software" class="software tool">Software Tool</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results/DNA" class="dry lab">DNA Results</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results/Model" class="dry lab">Model Results</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results/Medal_criteria" class="medial criteria">Medal Criteria</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Applied_Design" class="human practices">Application Scenario</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Collaborations" class="collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Engagement" class="outreach">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Human_Practices/Safe_by_Design" class="safe by design">Safe by Design</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Human_Practices/Stakeholders" class="stakeholders">Stakeholders</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Members" class="members">Members</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Advisors" class="advisors">Advisors</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Attributions" class="attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Sponsors" class="sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Notebook" class="notebook">Notebook</A></LI><TITLE>Menu</TITLE><UL class="topbar"><LI><A href="#">DNA Results</A></LI></UL></P><H6>The DNA constructs have been designed and assembled on multiple ways, which will briefly be described below. To get an overview of all the steps we performed in the lab, go see our <A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Notebook">&quot;Notebook&quot;</A> page.</H6><H1>14-3-3 tetramer</H1><H2>First approach</H2><H6>The complete designed construct was quite large. comprising over 4000 DNA basepairs. The construct is also made up from different protein domains, hence it was necessary to allow exchanging of the separate parts. As a result, we decided to divide our construct into 3 gBlocks, which could be connected to each other and placed in a pET28a(+) vector using Gibson Assembly. All gBlocks were synthesized by IDT.<UL><LI><SPAN>gBlock 1 contained a GFP sequence, followed by the first two monomers of 14-3-3.</SPAN></LI><LI><SPAN>gBlock 2 contained a sequence encoding for a TEV cleavable linker, to allow protease cleavage after protein expression.</SPAN></LI><LI><SPAN>gBlock 3 contained the third and fourth monomer of 14-3-3, followed by a His tag for purification and an ExoS sequence that would allow inhibition of the last monomer.</SPAN></LI></UL>

The Gibson assembly was attempted using NEBuilder HiFi DNA Assembly Cloning Kit, but this proved unsuccessful.</H6><H2>Second approach</H2><H6>We hypothesized that the unsuccessful Gibson assembly was due to the large gBlocks and large amount of inserts, so we decided to redesign our construct, by dividing it into two gBlocks: one containing the first two monomers and one containing a TEV-cleavable linker and the last two monomers. This time they did not contain ExoS or GFP groups.
The Gibson assembly of these constructs into the pET28a(+) vector once again proved unsuccessful. </H6><H2>Third approach</H2><H6>We then decided we could use a pET28a(+) vector of our supervisor that already contained a 14-3-3 dimer. A new gBlock, comprising the third and fourth monomers and GFP, was placed behind our supervisor’s dimer using SacI and HindIII restriction sites and this finally resulted in a successful tetramer with GFP.
Gel electrophoresis after colony PCR showed that most colonies merely contained the original dimer (around 1700), but two colonies did give good results (around 4200). The sequencing services of StarSEQ confirmed that the formation of a tetrameric 14-3-3 with GFP was finally successful.</H6><H6><DIV class="Figure_1"><FIGCAPTION>Figure 1: 14-3-3 tetramer with GFP</FIGCAPTION></DIV></H6><H1>CT33 with Strep-tag®II</H1><H2>First approach</H2><H6>We designed a gBlock encoding for an N-terminal Strep-tag®II, followed by the mCherry fluorophore and the C-terminal CT33 sequence, in such a way that it can be placed into pBAD-DEST49 vector. This was performed using NEBuilder HiFi DNA Assembly Cloning Kit. With an expected length of approximately 1100 bp this proved successful in multiple colonies after colony PCR. This was also confirmed by sequencing.</H6><H6><DIV class="Figure_2"><FIGCAPTION>Figure 2: Construct with Strep-tag®II, mCherry and CT33 in pBAD-DEST49</FIGCAPTION></DIV></H6><H6>However, the expression of this vector and subsequent protein purification came with multiple difficulties. We hypothesized that this could be due to hairpin formation in the mRNA (Tm = 45°C according to IDT OligoAnalyzer), meaning that during translation the ATG of mCherry is recognized as start codon instead of the ATG before Strep-tag®II. This way the Strep-tag®II will not be expressed and protein purification is impossible. We therefore designed two sets of primers: one would insert 3 bases between the first ATG and Strep-tag®II and one would insert 15 amino acids (shown in figure). This was done according to a protocol by Liu and Naismith.<SUP>[1]</SUP> We hoped that the first ATG would then be located outside the hairpin, so that it can be translated. Such small inserts can not be visualized on agarose gel, but sequencing confirmed that this approach was successful.</H6><H6><DIV class="Figure_3"><FIGCAPTION>Figure 3: First ATG incorporated into possible hairpin formation</FIGCAPTION></DIV></H6><H6><DIV class="Figure_3"><FIGCAPTION>Figure 4: Primers insert extra 15 amino acids</FIGCAPTION></DIV></H6><H2>Second approach</H2><H6>We also planned on placing the construct in a pET28a(+) vector, with which we have more experience when it comes to expression. We designed two new gBlocks: one was quite similar to the one in pBAD-DEST49, which means it starts with Strep-tag®II, followed by mCherry, ending with CT33. The other one starts with mCherry, followed by Strep-tag®II, ending with CT33. These are called samples SM and MS respectively. These were placed into the pET28a(+) vector using NcoI and HindIII restriction sites and these also proved to be successful in gel electrophoresis of colony PCR and sequencing. The expected lengths are around 1200 bp, with sample SM being slightly shorter than MS.</H6><H6><DIV class="Figure_3"><FIGCAPTION>Figure 5: Two different CT33 constructs (MS and SM) cloned into pET28a(+) vector</FIGCAPTION></DIV></H6><H1>BioBricking CT33</H1><H6>Including the entire prefix and suffix in the gBlocks lead to synthesis problems, which is why only the XbaI and SpeI sites were added, which should allow clonation of the gBlock into the pSB1C3 vector at the correct location. Due to similar overhang of XbaI and SpeI, the vector could also ligate to itself, rather than to our gBlock, making this process quite inefficient. Despite this problem, we did manage to successfully create a BioBrick containing the gBlock comprising mCherry, Strep-tag®II and CT33. This was confirmed by gel electrophoresis and sequencing.
Unfortunately the 14-3-3 tetramer with GFP is much larger and due to the high inefficiency we did not manage to properly clone it into pSB1C3. An alternative approach that we tried was the sequential digestion and ligation, starting with XbaI digestion and ligation, followed by SpeI digestion and ligation. The vector however, still preferred to ligate to itself.</H6><H6><DIV class="Figure_3"><FIGCAPTION>Figure 6: CT33 construct cloned into pSB1C3 vector</FIGCAPTION></DIV></H6><H6><DIV class="sources"><SUP>[1] </SUP> H. Liu and J. H. Naismith, “multiple-site plasmid mutagenesis protocol,” vol. 10, pp. 1–10, 2008.
</DIV></H6><TITLE>Footer</TITLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>