<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU-Eindhoven_Project_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU-Eindhoven/Project/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Menu</TITLE><UL class="menu"><LI><A href="https://2017.igem.org/Team:TU-Eindhoven">Home</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project">Project</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project/Background" class="description">Background</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project/Design" class="description">Design &amp; Parts</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Demonstrate" class="proof of concept">Proof of 
                   Concept</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/InterLab" class="interlab study">Interlab 
                   study</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Project/Protocols" class="protocols">Protocols</A></LI></UL><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Model/2D_Simulation" class="2D simulation">2D Simulation</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Model/Rule_based_model" class="rule based model">Rule based 
                   model</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Software" class="software tool">Software Tool</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results/DNA" class="dry lab">DNA Results</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results/Model" class="dry lab">Model Results</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Results/Medal_criteria" class="medial criteria">Medal Criteria</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Applied_Design" class="human practices">Application Scenario</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Collaborations" class="collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Engagement" class="outreach">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Human_Practices/Safe_by_Design" class="safe by design">Safe by Design</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Human_Practices/Stakeholders" class="stakeholders">Stakeholders</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Members" class="members">Members</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Advisors" class="advisors">Advisors</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Attributions" class="attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Sponsors" class="sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TU-Eindhoven/Team/Notebook" class="notebook">Notebook</A></LI></P><P><TITLE>Menu</TITLE><UL class="topbar"><LI><A href="#">Design &amp; Parts</A></LI></UL></P><H1>The Scaffold Construct: 14-3-3</H1><H6>The sequence is designed by us and starts with DNA coding for a His-tag, used for purification purposes, followed by a 14-3-3 dimer, a well researched and described protein. This is then followed by a linker with another 14-3-3 dimer, of which the last monomer is mutated, resulting in loss of binding capability. This domain is subsequently followed by a linker with GFP, a fluorophore. Expression of this protein results in a functional trivalent scaffold that can fulfill multiple roles in protein-protein interaction (PPI) networks. The fluorophore allows localization and can be used to study the protein's behavior. In short:<UL><LI><SPAN>3978 DNA basepairs</SPAN></LI><LI><SPAN>1315 AA protein (146 kDa)</SPAN></LI><LI><SPAN>Binds to CT33 / CT52</SPAN></LI><LI><SPAN>Binding possibilities to other molecules</SPAN></LI><LI><SPAN>Green fluorescent</SPAN></LI><LI><SPAN>His-tag purification</SPAN></LI></UL><DIV class="Figure_1"><FIGCAPTION>Figure 1: Construct consisting of 14-3-3 monomers and GFP</FIGCAPTION></DIV></H6><H2>About 14-3-3</H2><H6>Short background information</H6><DIV><H6>Proteins belonging to the 14-3-3 family are dimers, where each monomer consists out of nine anti-parallel alpha-helices. This causes the dimer to obtain a cup-like shape with two amphipathic binding grooves. The structure forms a rigid scaffold that is capable of anchoring proteins. 14-3-3 proteins are involved in multiple cellular processes and are mostly known to bind phosphorylated peptide motifs, especially those containing phosphoserine and phosphothreonine sequences. Most regions are conserved among different 14-3-3 isoforms, but the C-terminus appears to show more variability and is important in binding different target proteins.<SUP>[1]</SUP></H6></DIV></DIV></DIV><H6>The 14-3-3 protein in this part is the specific tobacco isoform 14-3-3c and it is stripped of its last 18 C-terminal amino acids, called T14-3cΔC. This allows for higher affinity towards the CT33 peptide, more specifically towards the YDI tail, in the presence of small molecule fusicoccin.<SUP>[2]</SUP> Next to the shortening of 14-3-3, this part also connects two 14-3-3 dimers, forming a tetramer scaffold. Mutation of one or more monomers consecutively allows varying the amount of available binding pockets. Tunability of the number of binding pockets can be useful to create a valency that is ideal for phase separation or other Protein-Protein Interactions (PPIs). In this part, the 4th monomer is mutated, preventing it from binding CT33, yielding a trimeric scaffold. These three binding pockets can then be blocked using covalently attached ExoS domains, which can be cleaved by aforementioned MMPs. This would mean that PPIs will be induced only in the presence of these MMPs.<SUP>[3]</SUP> ExoS is not yet included in this part, but could be added in a further stadium. </H6><H2>Connection to CT33</H2><H6>One motif that is known to bind to 14-3-3 is the phosphorylated C-terminus of H<SUP>+</SUP>-ATPase, an enzyme that catalyzes the hydrolysis of ATP to ADP.<SUP>[4]</SUP> The CT33 peptide comprises the final 33 amino acids of this C-terminus, which is referred. The binding of unphosphorylated CT33 and CT52 with YDI mutation to 14-3-3 family has extensively been researched and it was shown that this binding was particularly strong to the specific T14-3cΔC protein.<SUP>[2]</SUP> This happened in the presence of a small molecule, called fusicoccin, which functions as stabilizer and resulted in a Kd of 0.25 µM.<SUP>[5]</SUP> Due to this low Kd value and tunability of fusicoccin this binding is interesting for contributing to a PPI network based on 14-3-3 scaffolds, especially when the valency of 14-3-3 can be altered.</H6><H2>GFP</H2><H6>In many biological or chemical processes it is convenient to allow visualization of the behavior of molecules. One facile approach to such visualization is the attachment of a fluorophore, such as Green Fluorescent Protein (GFP). This protein is often used for this purpose, showing a major and a minor excitation peak at 395 and 475 nm, respectively, while the emission peak lies around 509 nm.<SUP>[6]</SUP> Using this domain in a protein network, where other proteins comprise different fluorophores, may yield significant information on interactions and localization.
</H6><H2>His-tag</H2><H6>The very beginning of the part contains a sequence that encodes for the famous His-tag, comprising 6 consecutive Histidine amino acids, that binds to e.g. Nickel ions. This allows the protein to be purified through Immobilzed Metal Affinity Chromatography (IMAC).<SUP>[7]</SUP>The info can also be found <A href="http://parts.igem.org/Part:BBa_K2356001">here (Part:BBa_K2356001).</A></H6><H6><DIV class="Figure_1"><FIGCAPTION>Figure 2: 14-3-3 interacts with 3 CT33 constructs</FIGCAPTION></DIV></H6><H1>The Binding Partner: CT33</H1><H6>The sequence is designed by us and starts with DNA coding for mCherry, a fluorophore. This is followed by DNA coding for Strep-tag®II, allowing it to bind to Strep-Tactin®XT or other Streptavidin variants. The last part of the sequence encodes for CT33, a protein domain comprising the final 33 amino acids of the C-terminus of H+-ATPase, a known binding partner of 14-3-3 scaffolds. The parts are connected via linkers, consisting mostly of Glycine and Serine. Expression of the part was successful and led to the creation of the desired protein. This protein should be able to be used to bind 14-3-3 protein scaffolds to tetrameric Streptavidin proteins.
In short:<UL><LI><SPAN>1018 DNA basepairs</SPAN></LI><LI><SPAN>334 AA protein (36 kDa)</SPAN></LI><LI><SPAN>Binds to Streptavidin</SPAN></LI><LI><SPAN>Binds to 14-3-3</SPAN></LI><LI><SPAN>Red fluorescent</SPAN></LI><LI><SPAN>Flexible linkers</SPAN></LI></UL><DIV class="Figure_3"><FIGCAPTION>Figure 3: Construct consisting of CT33, Strep-tag®II and mCherry</FIGCAPTION></DIV></H6><H2>mCherry</H2><H6>In many biological or chemical processes it is convenient to allow visualization of the behavior of molecules. One facile approach to such visualization is the attachment of a fluorophore, such as mCherry. This red, monomeric protein is often used for this purpose and exhibits excitation and emission peaks at 587 and 610 nm, respectively.<SUP>[8]</SUP> Using this domain in a protein network, where other proteins comprise different fluorophores, may yield significant information on interactions and localization.
</H6><H2>Strep-tag®II</H2><H6>The middle part of the sequence encodes for the so called &quot;Strep-tag®II&quot;, consisting of the peptide sequence <I>WSHPQFEK</I>. This sequence has proven to exhibit a high binding affinity towards streptavidin.<SUP>[9]</SUP> This binding can be utilized for multiple purposes, which is why this short peptide sequence is so essential. At first the binding to streptavidin can be utilized in the formation of large Protein-Protein Interaction (PPI) networks, due to the tetrameric structure of streptavidin. This allows supramolecular assembly of four CT33 constructs. The creation of such large network could have many different purposes, such as gelation and/or phase separation. Meanwhile, the Strep-tag®II sequence is also extensively used in protein purification purposes. The Strep-tag®II is able to selectively bind to columns containing Strep-Tactin®XT, a variant of streptavidin engineering by IBA Life Sciences. Since the tag is so small, it does not interfere with the folding of the protein.<SUP>[10]</SUP></H6><H2>CT33</H2><H6>The 14-3-3 protein family is a well known group of dimeric proteins that are capable of binding multiple different molecules. One motif that is known to bind to 14-3-3 is the phosphorylated C-terminus of H+-ATPase, an enzyme that catalyzes the hydrolysis of ATP to ADP.<SUP>[4]</SUP> The last 33 amino acids of this part are the same as the last 33 amino acids of H+ATPase, except the mutation of the last three to YDI, allowing unphosphorylated binding as well. Because this group is C-terminal, it has been named &quot;CT33&quot;. It should noted that in other research, sometimes the last 52 amino acids are used, which is called &quot;CT52&quot;. The domain is flanked by SalI and SacI restriction sites, allowing exchange of CT33 with CT52.
The binding of unphosphorylated CT33 and CT52 with YDI mutation to 14-3-3 family has extensively been researched and it was shown that this binding was particularly strong to the specific T14-3cΔC protein. This happened in the presence of a small molecule, called fusicoccin, which functions as stabilizer and resulted in a Kd of 0.25 µM.<SUP>[5]</SUP>
Due to this low Kd value and tunability of fusicoccin this binding is interesting for contributing to a PPI network based on 14-3-3 scaffolds, especially when the valency of 14-3-3 can be altered.

The info can also be found <A href="http://parts.igem.org/Part:BBa_K2356000">here (Part:BBa_K2356000)</A> and a slightly different version can be found <A href="http://parts.igem.org/Part:BBa_K2356003">here (Part:BBa_K2356003)</A>. </H6><H1>Combining our 14-3-3 with CT33</H1><H6>Because the 14-3-3 tetramer is mutated on the last monomer, it only has three binding pockets left, while one Strep-Tactin®XT can bind four CT33 constructs simultaneously. This 4:3 ratio prevents a one-on-one binding and forces the constructs to form a supramolecular network upon addition of fusicoccin, which can then exhibit gel-like behavior.</H6><H6><DIV class="Figure_3"><FIGCAPTION>Figure 4: Formation of large network through PPIs between 14-3-3 and CT33, and between Strep-tag®II and Strep-Tactin®XT</FIGCAPTION></DIV></H6><H6><DIV class="Figure_3"><FIGCAPTION>Figure 5: Close look at interactions between 14-3-3 and CT33, and between Strep-tag®II and Strep-Tactin®XT</FIGCAPTION></DIV></H6><DIV class="sources"><SUP>[1]</SUP> V. Obsilova, M. Kopecka, D. Kosek, M. Kacirova, and S. Kylarova, “Mechanisms of the 14-3-3 Protein Function : Regulation of Protein Function Through Conformational Modulation,” vol. 63, 2014.<SUP>[2]</SUP> C. Ottmann, S. Marco, N. Jaspert, C. Marcon, N. Schauer, M. Weyand, C. Vandermeeren, G. Duby, M. Boutry, A. Wittinghofer, and J. Rigaud, “Article Structure of a 14-3-3 Coordinated Hexamer of the Plant Plasma Membrane H+-ATPase by Combining X-Ray Crystallography and Electron Cryomicroscopy,” pp. 427–440, 2007.<SUP>[3]</SUP> S.J.A. Aper, “Engineering protein switches for sensing and actuation“, Ph.D. dissertation, Dept. Biomed. Eng., TU/e, 2016.<SUP>[4]</SUP> P. Morsomme and M. Boutry, “The plant plasma membrane H+-ATPase : structure , function and regulation,” vol. 1465, 2000.<SUP>[5]</SUP> A. Den Hamer, L. J. M. Lemmens, M. A. D. Nijenhuis, C. Ottmann, M. Merkx, T. F. A. De Greef, and L. Brunsveld, “Small-Molecule-Induced and Cooperative Enzyme Assembly on a 14-3-3 Scaffold,” pp. 331–335, 2017.<SUP>[6]</SUP> K. K. Turoverov, “NIH Public Access,” vol. 9, no. 4, pp. 338–369, 2010.<SUP>[7]</SUP> ThermoFisher Scientific - &quot;His-tagged Proteins – Production and Purification&quot;<SUP>[8]</SUP> N.C. Shaner et al., &quot;Improved monomeric red, orange and yellow fluorescent proteins derived from Discosoma sp. red fluorescent protein,&quot; vol. 22, pp. 1567-1572, 2004.<SUP>[9]</SUP> G.M. Schmidt, A. Skerra, &quot; The Strep-tag system for one-step purification and high-affinity detection or capturing of proteins,&quot; pp. 1528-1535, 2007.<SUP>[10]</SUP> IBA Life Sciences - &quot;Protein Purification with Strep-Tactin&quot;</DIV><P><TITLE>Footer</TITLE></P></DIV></DIV></DIV></BODY></HTML>