Team:TU-Eindhoven/Background
Untitled Document
Members Advisors Collaborations Sponsors Attributions
Description Background Design Experiments Timeline Protocols
Background
14-3-3
Proteins belonging to the 14-3-3 family are dimers, where each monomer consists out of nine anti-parallel α-helices. This causes the dimer to obtain a cup-like shape with two amphipathic binding grooves. The structure forms a rigid scaffold that is capable of anchoring proteins. 14-3-3 proteins are involved in multiple cellular processes and are mostly known to bind phosphorylated peptide motifs, especially those containing phosphoserine and phosphothreonine sequences. Most regions are conserved among different 14-3-3 isoforms, but the C-terminus appears to show more variability and is important in binding different target proteins.[1] In this project, the specific tobacco isoform 14-3-3c is used and stripped of its last 18 C-terminal amino acids, called T14-3cΔC. This allows higher affinity towards the CT52 peptide, more specifically the YDI tail (see Figure 1), in the presence of small molecule fusicoccin.[2] Next to the shortening of 14-3-3Δc, this project also connects two T14-3cΔC dimers, forming a tetramer scaffold. Mutation of one or more monomers consecutively allows varying the amount of available binding pockets. Tunability of the amount of binding pockets can be useful to create a valency that is ideal for phase separation. It is expected that a valency of 3 for 14-3-3 is optimal in combination with a valency of 4 for the CT33/52 construct with Strep-tactin.
CT33/52
One motif that is known to bind to 14-3-3 is the phosphorylated C-terminus of H+-ATPase, an enzyme that catalyzes the hydrolysis of ATP to ADP.[3] In this project we use peptides compromising the final 33 and 52 amino acids of this C-terminus, which are referred to as CT33 and CT52, respectively. In previous research the binding of unphosphorylated CT52 to T14-3cΔC was established by mutation of the last three amino acids of CT52 to YDI and addition of fusicoccin, yielding a Kd of 0.85 nM.[2] Due to this low value and tunability of fusicoccin this binding is interesting for contributing to a PPI network based on 14-3-3 scaffolds.
Fusicoccin
Fusicoccin is a toxin produced by the fungus Fusicoccum amygdali, which is mainly active in almond and peach trees.[4] Fusicoccin activates the plant plasma membrane H+-ATPase by binding to 14-3-3. The binding between 14-3-3 and fusicoccin stabilizes the interaction between H+-ATPase and 14-3-3 by closing the gap in the 14-3-3 groove that remains after binding phosphopeptides.[5] This leads to hyperpolarization of the membrane and alteration of ionic gradients. Applications of fusicoccin can be the targeting of 14-3-3 proteins in cancer cells and inducing apoptosis in tumor cells. The last application is accomplished by the activation of the tumor necrosis factor-related apoptosis-inducing ligand pathway.[4] In our project, fusicoccin is mainly used for its capability to stabilize the binding between 14-3-3 and H+-ATPase.
[1] Obsilova V, Kopecka M, Kosek D, Kacirova M, Kylarova S. Mechanisms of the 14-3-3 Protein Function : Regulation of Protein Function Through Conformational Modulation. 2014;63. [2] Ottmann C, Marco S, Jaspert N, et al. Article Structure of a 14-3-3 Coordinated Hexamer of the Plant Plasma Membrane H + -ATPase by Combining X-Ray Crystallography and Electron Cryomicroscopy. 2007:427-440. doi:10.1016/j.molcel.2006.12.017. [3] Morsomme P, Boutry M. The plant plasma membrane H  -ATPase : structure , function and regulation. 2000;1465. [4] T. Kinases, M. Bury, A. Andolfi, and B. Rogister, “Fusicoccin A , a Phytotoxic Carbotricyclic Diterpene Glucoside of Fungal Origin , Reduces Proliferation and Invasion of Glioblastoma Cells by Targeting Multiple,” vol. 6, no. 2, pp. 112–123, 2013. [5] M. R. Roberts and D. J. Bowles, “Fusicoccin , 14-3-3 Proteins , and Defense Responses in Tomato Plants 1,” vol. 119, no. April, pp. 1243–1250, 1999.
