<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ANU-Canberra_background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ANU-Canberra/background</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="page-wrapper"><DIV id="header-wrapper"><DIV id="header"><NAV id="nav"><UL><LI><A href="https://2015.igem.org/Team:ANU-Canberra">Home</A></LI><LI><A href="https://2015.igem.org/Team:ANU-Canberra/overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ANU-Canberra/team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ANU-Canberra/project">Project</A></LI><LI><A href="https://2015.igem.org/Team:ANU-Canberra/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:ANU-Canberra/labbook">Results</A></LI><LI><A href="https://2015.igem.org/Team:ANU-Canberra/Practices">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:ANU-Canberra/Attributions">Attributions</A></LI></UL></NAV></DIV></DIV><DIV id="main-wrapper"><DIV class="container"><ARTICLE class="box post"><HEADER><H2>Project Background</H2></HEADER><P>
In the International Year of Light, it is apt to consider the growing field of optogenetics - the control of cellular dynamics using light. In its early stages, optogenetics focused on light-gated ion channels to control neuron excitability, but it is now expanding to broader applications. 
                                        </P><P>
The biosynthesis of valuable metabolites such as NAD are areas of important research. A considerable stumbling block towards overexpression of these metabolites for commercial purposes arises when high concentrations of these compounds can become toxic to cells. One strategy to mitigate these issues is to restrict the timing of over-expression of the compound until production is most efficient. We can use optogenetic tools to create an irreversible switch allows for the expression of a late-stage enzyme necessary for the production of the final metabolite. 
                                        </P><P>
                                        In this project we aim to test for the functionality of a known eukaryotic optogenetic system, CRY2/CIB1 in <I>E. coli.</I> We will further test this system in conjunction with a split-Cre-recombinase construct that will enable the excision of targeted DNA constructs between parallel loxP sites only in the presence of blue light. Such a system could be used to control the expression of an end-stage enzyme required for the expression of a toxic metabolite such as NAD.  
                                        </P><HEADER><H2>Optogenetics vs. Ligand Induction</H2></HEADER><P>
Light-based induction offers a more effective option to activate specific cellular activities than traditional chemical induction:
<TABLE style="width:100%"><TBODY><TR><TD><B>Chemical</B></TD><TD><B>Light</B></TD></TR><TR><TD><B>Spatial control and target specificity</B></TD><TD>Low</TD><TD>High</TD></TR><TR><TD><B>Temporal control</B></TD><TD>Limited by time taken for distribution around the body, permeation into target cells and unwanted persistence of inducers in target cells</TD><TD>Precise - light-activation and dark-deactivation within minutes or seconds</TD></TR><TR><TD><B>Cost</B></TD><TD>Expensive</TD><TD>Cheap</TD></TR><TR><TD><B>Toxicity</B></TD><TD>Possible</TD><TD>None</TD></TR></TBODY></TABLE></P><H2>Light-Inducible Protein-Interaction Module</H2></ARTICLE><P>
Cryptochrome 2 (CRY2) is a photosensitive protein found in all kingdoms of life. Human CRY2 helps modulate circadian rhythm; in plants CRY2 regulates the timing of flowering. Upon irradiation with blue-light, CRY2 from <I>Arabidopsis thaliana</I> reversibly binds to cryptochrome-interacting basic helix-loop-helix 1 (CIB1). These molecular light-sensors are useful tools for synthetic biologists to control specific cellular activity. </P><P>
Blue-light illumination (405-488nm) activates CRY2 within seconds via reduction of the FAD chromophore, causing a conformational change that allows it to bind its interacting partner, cryptochrome-interacting basic helix-loop-helix1 (CIB1). After return to the dark, the CRY2/CIB1 complex dissociates in a matter of minutes. 
</P><P><B>CRY2 Absorption spectrum.</B><I>Black line shows absorption spectra of the dark-adapted CRY2 sample (Banerjee et al, 2007).</I></P><P> The basic strategy of an inducible, reversible CRY2/CIB1 optogenetic system is to fuse fragments of inactive, split effector protein of interest to CRY2 and CIB1, such that blue-light illumination brings together CRY2 and CIB1, thus reconstituting the active protein. </P><P><B>Basic scheme for light-regulation of protein activity using a CRY2/CIB1 system. </B><I>The target protein is split into two inactive fragments, A and B, which are fused to CRY2 and CIB1. Irradiation of CRY2 by blue light causes a conformational change which allows interaction with CIB1, allowing the fused protein fragments to dimerise and reconstitute the active protein. Return of the system to the dark results in dissociation of both the CRY2/CIB1 complex and the reconstituted protein. </I></P><P> This versatile strategy has been used to regulate transcriptional activation by split transcription factors, protein clustering to regulate cell signalling via oligomerisation of CRY2, and protein translocation through fusion of CIB1 with a form of eGFP which localises to the plasma membrane. </P><P>
                                       CRY2 systems have been primarily tested in eukaryotic systems, such as yeast, zebrafish and chinese hamster ovary cells. This is in part due to the origins of optogenetics in neural cells and the high value of potential applications in human cells. However, we aimed to express the CRY2/CIB1 system in <I>E. coli.</I> to show that this system has diverse and valuable applications in prokaryotic systems as well. An advantage of CRY2 is that it uses a chromophore endogenous to eukaryotes, flavin adenine dinucleotide (FAD) unlike other photoreceptors such as PhyB.                                 </P><HEADER><H2>Testing for CRY2/CIB1 Activity in Prokaryotic Cells</H2></HEADER><P> Expressing eukaryotic genes within prokaryotes poses many difficulties. Eukaryotic genes may not express well in prokaryotes, often due to misfolding which can occur due to many reasons. Codon bias by prokaryotes may also heavily dictate possible proteins expressed in <I>E. coli.</I>  Furthermore post-translational machinery, such as for disulfide-bond formation, essential to protein function may be missing in prokaryotes. </P><P>
                                    We can determine the extent to which CRY2/CIB1 are expressed and associate in prokaryotes under blue light through SDS PAGE/blot analysis and FRET analysis. Fluorescent proteins fused to each CRY2/CIB1 domain will show a change in fluorescence of the two fluorescent proteins that is dependent on the distance between the two proteins. 
                                    </P><HEADER><H2>CRE-lox recombination</H2></HEADER><P>
                                Cre-recombinase is an enzyme derived from P1 bacteriophage that is utilised in site-specific recombinase technology in both prokaryotes and eukaryotes. Cre-recombinase targets DNA recognition sites termed 34bp DNA loxP sequences. DNA between two parallel loxP is excised by active Cre-recombinase, whilst loxP sites inversely orientated inverts the intervening DNA. 
                                </P><P>
Temporal control of Cre-recombinase can be manipulated by splitting the protein at specific sites into two terminal domains (CRE-N, CRE-C) such that either moiety is inactive whilst forming a functional protein by ligand-induced dimerization. Past studies have used light-induced fusion of CRY2-Cre-N and CIB1-Cre-C constructs to form functional Cre-recombinase. The ability to initiate the excision of specific DNA sites with light enables the control over the expression of genes using light. 
                                </P><HEADER><H2>Potential Functions of a Split Cre-Recombinase CRY2/CIB1 construct</H2></HEADER><P>
                                We hope to test for the activity of a reliable, non-reversible, blue light activated switch in E. coli. using split Cre-recombinase CRY2/CIB1 construct. This could potentially be used induce the production of alternative genes coding enzymatic pathways that contain a final component that is otherwise toxic to E. coli growth.
                                </P><P>Overexpression of certain organic molecules or proteins, due to genetic engineering, can result in toxic levels of intermediate or end products resulting in decreased cell fitness. In particular, we are focusing on nadB and nadA bacterial genes (aspartate oxidase and quinolinate synthase respectively), that encode for enzymes required during nicotinamide adenine dinucleotide (NAD) <I>de novo </I>synthesis. 
                                </P><P>
                                NAD can be produced via a six-step <I>de novo</I> biosynthetic pathway in bacteria. However the accumulation of high levels of NAD can be toxic to bacterial cells, thus limiting the concentrations of NAD that can be achieved in bacteria for biomanufacturing purposes. We propose that suspending <I>de novo</I> production of NAD at a precursor stage until the activation of the CRY2/CIB1 Cre-recombinase complex induces the production of a subsequent enzyme necessary for the production of NAD. This could allow higher concentrations of NAD to be achieved in transformed cells before death due to toxicity. 
                                </P><HEADER><H2>Excision of the nadB gene with a split Cre-Recombinase CRY2/CIB1 construct</H2></HEADER><P><I>CRY2/Cre-N and CIB1-CreC constructs do not form active Cre-recombinase in the dark. In this environment, the gene nadB can be expressed, whilst  nadA is not due to the presence of a terminator upstream of the gene. Upon blue light stimulation, co-association of CRY2/CIB1 domains allows a functional Cre-recombinase enzyme to form. This enables the excision of nadB and the transcription of nadA. </I></P><P>
                                This concept has not been trialled using the CRY2/CIB1 device before. However an analogous strategy is the commercial production of astaxanthin, a carotenoid in high demand globally for use as pigments in feedstock and as a potential anti-cancer and immunostimulatory compound. Production in macroalgae provides an economical alternative to chemical synthesis and a two stage process maximises yield.</P><P><B>1.</B>&quot;Green stage” – or the vegetative stage, where biomass is produced under optimal conditions.</P><P><B>2.</B>
    &quot;Red stage” – astaxanthin is accumulated under adverse environmental conditions. Excess photooxidation by intense radiation produce active oxygen radicals; production of astaxanthin may be a protective strategy in response to oxidative stress. The ratio of carotenoids:chlorophyll may increase by an order of magnitude during the red stage.
                                </P><P>
                                This resembles our strategy in the accumulation of a precursor (in this case, biomass capable of synthesising the product) and the subsequent induction of high-yield synthesis of the desired product (necessary in this case as high levels are only produced under certain conditions). </P><HEADER><H2>REFERENCES</H2></HEADER><P>Banerjee, R., Schleicher, E., Meier, S., Viana, R. F., Pokorny, R., Ahmad, M., Bittl, R. &amp; Batschauer, A. The signaling state of Arabidopsis Cryptochrome 2 contains flavin semiquinone. <I>J. Biol. Chem.</I><B>282</B>, 14916-14922 (2007).</P><P>Begley, T.P., Kinsland, C., Mehl, R.A., Osterman, A., Dorrestein, P. The Biosynthesis of Nicotinamide Adenine Dinucleotides in Bacteria: Vitamins and Hormones. Academic Press 61: 119 (2001).</P><P>Bugaj, L. J. et al. Optogenetic protein clustering and signaling activation in mammalian cells. <I>Nat. Methods.</I><B>10</B>, 249-252 (2013). </P><P>Beyer, H. M., Naumann, S., Weber, W. &amp; Radziwill, G. Optogenetic control of signaling in mammalian cells. <I>Biotechnol. J.</I><B>10</B>, 273-283 (2015).</P><P>Guedes, A. C., Amaro, H. M. &amp; Malcata, F. X. Microalgae as sources of carotenoids. <I>Marine Drugs.</I><B>9</B>, 625 (2011).</P><P>Idevall-Hagren, O., Dickson, E. J., Hille, B., Toomre, D. K. &amp; De Camilli, P. Optogenetic control of phosphoinositide metabolism. <I>Proc. Natl. Acad. Sci. USA.</I><B>109</B>, e2316-2323 (2012).</P><P>Kennedy, M. J., Hughes, R. M., Peteya, L. A., Schwartz, J. W., Ehlers, M. D. &amp; Tucker, C. L. Rapid blue light induction of protein interactions in living cells. <I>Nat. Methods.</I><B>7</B>, 973-975 (2010).</P><P>Liu, H., Gomez, G., Lin, S., Lin, S. &amp; Lin, C. Optogenetic control of transcription in zebrafish. <I>PLoS ONE.</I><B>7</B>, e50738 (2012).</P><P>Liu, H., Wang, Q., Liu, Y., Zhao, X., Imaizumi, T., Somers, D. E., Tobin, E. M. &amp; Lin, C. Arabidopsis CRY2 and ZTL mediate blue-light regulation of the transcription factor CIB1 by distinct mechanisms. <I>Proc. Natl. Acad. Sci. USA.</I><B>110</B>, 17582-17587 (2013).</P><P>O’Brien, S.P., DeLisa, M.P. Split-Cre recombinase effectively monitors protein-protein interactions in living bacteria. <I>J. Biotechnol.</I><B>9</B>, 355 (2014).</P><P>Posten, C. &amp; Walker, C. Microalgal technology: Integration and Economy. Walter de Gryter (2013). </P><P>Tischer, D. &amp; Weiner, O. D. Illuminating cell signalling with optogenetic tools. <I>Nat. Rev.</I><B>15</B>, 551-558 (2014).</P><P>van der Horst, G. T. J., Muijtjens, M., Kobayashi, K., Takano, R., Kanno, S., Takao, M., de Wit, J., Verkek, A., Eker, A. P. M., van Leenen, D., Buijs, R., Bootsma, D., Hoeijmakers, J. H. J. &amp; Yasui, A. Mammalian Cry1 and Cry2 are essential for maintenance of circadian rhythms. <I>Nature.</I><B>398</B>, 627-630 (1999).</P><P>Yizhar, O., Fenno, L. E., Davidson, T. J., Mogri, M. &amp; Deisseroth, K. Optogenetics in neural systems. <I>Neuron.</I><B>71</B>, 9-34 (2011).</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>