<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SJTU-BioX-Shanghai_Project_design"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:SJTU-BioX-Shanghai/Project_design">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SJTU-BioX-Shanghai/Project_design&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Project_design&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Project_design&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SJTU-BioX-Shanghai/Project_design" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SJTU-BioX-Shanghai/Project design</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="mainDiv"><DIV id="logoDiv"><DIV id="logoContent"><DIV id="projectCard"><P>Project introduction.
				Inspired by the natural regulator of circadian bioclock exhibited in most eukaryotic organisms, our team has designed an E.coli-based genetic network with the toxin-antitoxin system so that the bacterium oscillates between two states of dormancy and activity <A href="/Team:SJTU-BioX-Shanghai/The_project#bannerTop">(more...)</A></P></DIV></DIV></DIV><DIV id="sideMenu"><H3><A href="/Team:SJTU-BioX-Shanghai#bannerTop">Home</A></H3><H3><A href="/Team:SJTU-BioX-Shanghai/The_team#bannerTop">The team</A></H3><UL><LI><A href="/Team:SJTU-BioX-Shanghai/The_team#bannerTop">Introduction</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Members#bannerTop">Members</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/The_album#bannerTop">The album</A></LI></UL><H3><A href="/Team:SJTU-BioX-Shanghai/The_project#bannerTop">The project</A></H3><UL><LI><A href="/Team:SJTU-BioX-Shanghai/The_project#bannerTop">Idea &amp; Background</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project_design#bannerTop">Project design</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project_network#bannerTop">Network</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Modelling#bannerTop">Modelling</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Protocols#bannerTop">Protocols</A></LI></UL><H3><A href="/Team:SJTU-BioX-Shanghai/Results#bannerTop">Results</A></H3><UL><LI><A href="/Team:SJTU-BioX-Shanghai/Results#bannerTop">Overview</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Result_eucaryote#bannerTop">Expanding study</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Parts#bannerTop">Parts &amp; Devices</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Characterization#bannerTop">Characterization</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Human_practice#bannerTop">Human practice</A></LI></UL><H3><A href="/Team:SJTU-BioX-Shanghai/Calendar#bannerTop">Calendar</A></H3><UL><LI><A href="/Team:SJTU-BioX-Shanghai/Events#bannerTop">Events</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Lablog#bannerTop">Lablog</A></LI></UL><H3><A href="/Team:SJTU-BioX-Shanghai/Others#bannerTop">Others</A></H3><UL><LI><A href="/Team:SJTU-BioX-Shanghai/Others#bannerTop">Tools for Wiki</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Safety#bannerTop">Biological safety</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Collaboration#bannerTop">Team collaboration</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Acknowledgements#bannerTop">Acknowledgements</A></LI></UL></DIV><DIV id="contextDiv"><DIV id="title">Project design</DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#On_translation_level"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">On translation level</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#On_transcription_level"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">On transcription level</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Overall_output"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Overall output</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#One_concern:How_to_erase_the_toxic_effect_of_relE_on_chromosomal_housekeeping_genes.3F_------the_answer_is_tmRNA."><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">One concern:How to erase the toxic effect of relE on chromosomal housekeeping genes? ------the answer is tmRNA.</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><DL><DT><CENTER><FONT size="4"><I><B>E.coli</B></I> the napper</FONT></CENTER></DT></DL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B><I>E.coli </I>the napper:</B> our original idea is to build up an inner bioclock mastering sleep rhythem (which can only be found in higher organisms) in <I>E.coli</I></DIV></DIV></DIV></DIV><P>Bacteria and at least some archaea harbor ‘suicide systems’, the existence of which is a mystery to those who hear about them for the first time. These systems consist of a lethal toxin that can be neutralized by an antitoxin. The production of labile antitoxin must at least parallel that of the stable toxin to circumvent the induction of cell poison. To achieve this, both proteins are encoded within a single operon and the genes are expressed with the same mRNA transcript. The toxins seem to target specific, essential cellular processes. Perhaps the best characterized of these system is relBE system, the genes of which are present on the <I><B>Escherichia coli</B></I> chromosome. The relE toxin is an RNase that preferentially cleaves mRNAs at the ribosomal A site between the 2nd and 3rd nucleotide of stop codons. Stop codons not only signal the end of the protein coding sequence but also serve as the binding site for release factors, which promote release of the nascent polypeptide and facilitate recycling of ribosomes for further rounds of translation. Thus truncated mRNA by cleavage of relE lacks appropriate termination signals, which causes the accumulation of stalled ribosomes and these mRNAs are unable to promote release factor binding, nascent polypeptide release, and ribosome recycling.
</P><P>As a result, expression of the <I><B>relE</B></I> gene has been shown to severely inhibit translation and prevent colony formation.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:510px;"><DIV class="thumbcaption"><B>fig 1. </B>Expression of relE on growth of <I><B>E.coli</B></I> cells. From <I>Identification of a Pair of Toxin-antitoxin(TA) Gene in the Chromosome of Cyanobacteria synechocystis sp.PCC6803</I></DIV></DIV></DIV></DIV><P>However, the growth of <I><B>E.coli</B></I> can be restored by coexpression of the cognate relB antitoxin, which can form a heterotetrameric (relB-relE)<SUB>2</SUB> structure when binding with relE. Thus,this structure is too large to fit into the A site, so the toxic relE can be neutralized. Overexpression of relB restored protein synthesis and colony formation. From this followed that inhibition of protein synthesis by relE is bacteriostatic, a state of low metabolic rate and cell growth like the hibernation behavior of bears and hedgehogs in chilly winter, rather than bacteriocidal. But the unbound relB is an easy prey for the ATP-dependent Lon protease.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:518px;"><DIV class="thumbcaption"><B>fig 2. </B>Genetic organization and regulatory components of the relBE Toxin-Antitoxin operon(left), the site of mRNA cleavage by relE(middle), and ribosome rescue by tmRNA after mRNA truncation by relE(right). From <I>The Bacterial Toxin RELE Displays Codon-Specific Cleavage of mRNAs in the Ribosomal A site.</I></DIV></DIV></DIV></DIV><P>What is most intricating is that relE display codon-specific cleavage of mRNAs in the ribosomal A site, that is to say, among stop codons UAG is cleaved with fast, UAA intermediate and UGA slow rate(UAG&gt;UAA&gt;UGA). The rates of the inhibition reactions in all cases were proportional to the relE concentration and thus determined by k<SUB>cat</SUB>/K<SUB>m</SUB>. The k<SUB>cat</SUB>/K<SUB>m</SUB> Values (s<SUP>-1</SUP>μM<SUP>-1</SUP>) for RelE Cleavage of 3 stop codons in the Ribosomal A Site have been measured previously.(UGA:0.078, UAA:2.2, UAG:26). The physical rationale of the measurement is that k<SUB>cat</SUB>/K<SUB>m</SUB> is determined by the standard free energy difference between relE in activated complex with a ribosome and a ground state with free relE and ribosome.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:572px;"><DIV class="thumbcaption"><B>fig 3. </B>RelE inhibition of peptide release from ribosomes with a tetrapeptidyl-tRNA in the P site and a stop codon in the A site. From <I>The Bacterial Toxin RELE Displays Codon-Specific Cleavage of mRNAs in the Ribosomal A site.</I></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="On_translation_level">On translation level</SPAN></H2><P>In our project, this unique <I><B>relBE</B></I> operon is separated into two independent genes, that is , <I><B>relE</B></I> and <I><B>relB</B></I> are located on two plasmid vectors pSB1A2 and pSB1AK3 respectively. Thus, expression of <I><B>relE</B></I> can be induced by substrate IPTG and that of <I><B>relB</B></I> by Trp. And the ATP-dependent protease <I><B>Lon</B></I> gene follows <I><B>relE</B></I> to form a polycistron in the same operon, which means Lon can coexpress with relE once induced by IPTG. When two operons are induced simultaneously, in the protein level, relE, relB, and Lon will fight against each other. On the one hand, relB can prevent cell from dormancy by neutralizing relE while on the other hand, relB can be easily degraded by Lon. It seems that with the coexpression of Lon, the toxic effect of relE can not be offset by relB. Thus it is a finality that <I><B>E.Coli</B></I> will hibernate to survive this stage.
</P><H2><SPAN class="mw-headline" id="On_transcription_level">On transcription level</SPAN></H2><P>To add suspense to this play, we decide to create a transcriptional regulation pathway to control the rate of synthesis of 3 proteins. We modify 3 genes’ stop codons artificially via primer design in PCR procedure. In detail, we alter the stop codons of <I><B>relE</B></I>, <I><B>Lon</B></I> and <I><B>relB</B></I> to UAA, UAG, UGA respectively, which means on the transcriptional level, the mRNA of <I><B>Lon</B></I> will be the most severe victim of <I><B>relE</B></I> cleavage whereas the mRNA of <I><B>relB</B></I> can almost be intact based on a pretty low <I><B>relE</B></I> cleavage rate. relE cuts its own mRNA at a medium rate. On this level of competiton, mRNA of <I><B>relB</B></I> is the biggest winner, which grants an accordingly high level of relB product via translation. As relB accumulate to an amount surpassing that of relE, the physiological state can alter from hibernation to activity, where the metabolism of bacteria returns normal and colonies can easily form.
</P><H2><SPAN class="mw-headline" id="Overall_output">Overall output</SPAN></H2><P>If we put all these together, who can tell the final champion? relE or relB? At first stage, once induced by IPTG and arabinose simultaneously, relE, Lon are coexpressed, and relB is hydrolyzed rapidly by Lon once produced. The bacteria may keep in low metabolic rate and have weak capacity to form colonies during this phase.
</P><P>Nevertheless, as relE cleaves the 3 kinds of mRNAs with different stop codons, relB has the fastest synthesis rate because of a high level of intact mRNAs. In contrast, Lon are cornered to a rather low synthesis rate based on a small amount of available mRNAs. This is a perfect moment for relB to accumulate its products. When relB expression outstrips that of relE, the toxic effect of relE can be halted completely. During this stage, the microbe can regain the capacity of colony formation. It’s time to break the nightmare of Hypnos’ Curse!!!
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>fig 4. General mechanisms for oscillation: (1)</B>on transcription level relE toxic inhibition of relB, relE and Lon is minimal, medium and intensive respectively. <B>(2)</B>On protein level relE is the prey of relB whereas relB is also the prey of Lon. These are the prerequisites for our bioclock oscillating. Thus, ideally, our <I>E.coli</I> can live regularly like our human beings who wake up to work in the daytime and sleep home at night.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="One_concern:How_to_erase_the_toxic_effect_of_relE_on_chromosomal_housekeeping_genes.3F_------the_answer_is_tmRNA.">One concern:How to erase the toxic effect of relE on chromosomal housekeeping genes? ------the answer is tmRNA.</SPAN></H2><P>Although some previous job has certify the fact that later induction of relB transcription in RelE-inhibited cells mediated complete recovery of colony formation.
</P><P>Some may doubt that in the whole <I><B>E.Coli</B></I> system there are lots of key housekeeping genes such as those of RNA polymerase, peptide release factors and some catalytic enzymes in respiration, so how can bacteria recover from dormancy without these genes’ proper work, which are also affected by relE toxicity? One essential for recovery is that ribosomal complexes with peptidyl-tRNA in P site and an mRNA truncated in the A site codon can be brought back initiation. So tmRNA is required for rapid recovery of translation in RelE-inhibited cells. tmRNA is a versatile and highly conserved bacterial molecule endowed with the combined structural and functional properties of both a tRNA and a mRNA. The tmRNA system orchestrates three key biological functions: (1) recognition and rescue of ribosomes stalled on aberrant mRNAs, (2) disposal of the causative defective mRNAs, and (3)addition of a degradation tag to ribosome-associated protein fragments for directed proteolysis.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:190px;"><DIV class="thumbcaption"><B>fig 5.</B> Proposed secondary structure of E.coli tmRNA. The tRNA- and mRNA-like properties are highlighted. From <I>Trans-Translation: The tmRNA-Mediated Surveillance Mechanism for Ribosome Rescue, Directed Protein Degradation, and Nonstop mRNA Decay.</I></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:335px;"><DIV class="thumbcaption"><B>fig 6.</B> Trans-translation model for tmRNA-mediated protein tagging and ribosome rescue. From <I>Trans-Translation: The tmRNA-Mediated Surveillance Mechanism for Ribosome Rescue, Directed Protein Degradation, and Nonstop mRNA Decay.</I></DIV></DIV></DIV><P>So tmRNA is also taken into our circuit to assist relB rescue the dormant cells. It does accelerate the rescue efficiency of relB. Overproduction of tmRNA efficiently counteracts relE toxicity.
</P><PRE>Next, go to <A href="/Team:SJTU-BioX-Shanghai/Project_network#bannerTop" title="Team:SJTU-BioX-Shanghai/Project network">Network</A>.
</PRE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:SJTU-BioX-Shanghai/Project_design">http://2009.igem.org/Team:SJTU-BioX-Shanghai/Project_design</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SJTU-BioX-Shanghai/Project_design" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SJTU-BioX-Shanghai/Project_design" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Project_design&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Project_design&amp;oldid=146636" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>