"
Page
Discussion
View source
History
teams
Log in
 
Team:SJTU-BioX-Shanghai/Results
From 2009.igem.org
Project introduction.     Inspired by the natural regulator of circadian bioclock exhibited in most eukaryotic organisms, our team has designed an E.coli-based genetic network with the toxin-antitoxin system so that the bacterium oscillates between two states of dormancy and activity (more...)
Home
The team
Introduction
Members
The album
The project
Idea & Background
Project design
Network
Modelling
Protocols
Results
Overview
Expanding study
Parts & Devices
Characterization
Human practice
Calendar
Events
Lablog
Others
Tools for Wiki
Biological safety
Team collaboration
Acknowledgements
Results overview
As our genetic circuit is complicated in vector designing and logical understanding, involing various influential factors such as RelE-toxin, RelB-antitoxin, Lon protease and tmRNA(ssrA) gene. All the factors interact with each other in a specific method. Among them, TA system (toxin-antitoxin system)works as a vital and esscential part. Therefore, we tested and verified the function of TA system.
Contents
1 Part I: Construction of RelE-Toxin Generator
2 Part II: Insight into Growth Acticity of RelE Strain
3 Part III: Recovering Function Test of RelB Antitoxin
4 Part IV: RelE Strain Growth Curve
5 Part V: Research and Discussion on Bacteria Life Span
6 Part VI : Appendix - Different combination of promoter and RBS
Part I: Construction of RelE-Toxin Generator
RelE-toxin and RelB-antitoxin are the basic parts in our project, the genes of which are present on the Escherichia coli chromosome. First of all, we cloned the RelE and RelB gene in Escherichia coli str. K-12 substr. DH10B through PCR technology. To meet our design, we have made some modifications to their PCR primers : (1)The stop condon of RelE gene is mutated to TAA (2)A 18bp His-tag is linked before the stop codon to facilliate its detection.
The primers of RelE and RelB:
sepTA102-RelE-F:
5’- CGGAATTCGCGGCCGCTTCTAGATGGCGTATTTTCTGGATTTTGAC-3’
sepTA102-RelE-R:
5’-GGACTAGTATTAGTGATGATGATGATGATGGAGAATGCGTTTGACCGCCTCGC-3’
sepTA102-RelB-F:
5’-CCGGAATTCGCGGCCGCTTCTAGATGGGTAGCATTAACCTGCGTAT-3’
sepTA102-RelB-R:
5’-GGACTAGTACTCAGTGATGATGATGATGATGGAGTTCATCCAGCGTCACAC-3’
The blue sequence is biobrick prefix and surfix; Green sequence is His-tag; Red sequence is mutated RelE stop codon; Black sequence is matching nucleotide.
Then we added transcription terminator(here we chose BBa_B0015) to RelE and RelB gene, converting them into complete transcriptional unit. We choose pSB1A2 as the vector. After recombinition of RelE and Terminator, a typical Lac Promoter is inserted before RelE-Terminator complex. Eventually, a basic RelE-Toxin Generater turned out.The following picture shows the digestion results of agar electrophoresis.
Fig 1.Construct results of RelE+Double terminator and RelE+Rbs
Lane2: BBa_K185000 RelE toxin+Rbs30 311bp
Lane3: BBa_K185004 RelE toxin+Double terminator 443bp
Part II: Insight into Growth Acticity of RelE Strain
After identification of reconstruced plasmid, we transfected the plasmid into E.coli BL21 strain. In the E.coli strain containing RelE-toxin gene, after adding 0.1% IPTG onto the LB plate to induce its expression, it grows in a low metabolism rate compared with negative control group. The most obvious evidence is the phenomenon that it can hardly form colonies after induction., meanwhile the negative control group developed a lot of colonies after 20 hours.
Fig 2. expression of relE on growth of E.coli cells
Fig 2. expression of relE on growth of E.coli cells
The left plate contains 0.1% IPTG, which is used to induce RelE-toxin protein. Onced RelE is expressed, bacteria can hardly form colonies even after 20 hours’ cultivation. The plate in the right lacks inducer(IPTG), we can still see the RelE+ strain grows a bit slower than negative control group. It can be explained by the basal transcription of the Lac promoter.
Part III: Recovering Function Test of RelB Antitoxin
In bacteria chromosome, RelB is located in the same operon with RelE. RelB is also a small protein like RelE, about 10 kD. It forms a heterotetrameric (RelB-RelE)2 structure when binding with RelE. It is difficult to detect the function of RelB through plate oberservation, since overexpression of RelB-antitoxin has no influence on bacteria growth. We have to put the RelE+RelB- strain and RelE+RelB+ strain togther to verify the recovering function of RelB antitoxin. As mentioned before, a His-tag has been linked at the end of RelB protein. Firstly, we used anti-His antibody to detect whether RelB protein has been expressed. Then we streaked the RelE+RelB+, RelE+RelB- and negative control group on a M9 plate. To induce the promoter, 0.1%IPTG and 0.2% L-arabinose is respectively added onto the M9 plate.
The Western-blotting picture(Figure 3) shows that RelB protein is expressed once induced by L-arabinose, whereas no RelB protein is produced under normal enviroment (with no induction). Figure 4 demonstrates the growth activity of RelE+RelB+. The bacteria expressing both RelE and RelB proliferate like negative control group, while the bacteria expressing RelE can hardly form colonies. The fact we see on the plate illustrates the recovering function of RelB Antitoxin.
Fig 3. Western -blotting results when RelB is induced
When L-arabinose is added in the culture, the RelB+ strain is activated, starting to produce RelB proteins(Left ). If no iducer is provided, consequently it turns out no RelB protein.
Fig 4. Plate colony oberservation of RelE+RelB+ and RelE+ RelB-
The plate is devided into three regions: RelE+RelB-(only expressing RelE-toxin), RelE+RelB+(expressing both RelE-toxin and RelB-antitoxin), NC group(BL21 containing Amp resistance). The phenotype of RelE+RelB- demonstrates RelE’s inhibition in growth, whereas the E.coli strain expressing both RelE-toxin and RelB-antitoxin restored from inhibition.
Part IV: RelE Strain Growth Curve
To research in depth, we detected the bacteria growth curve of RelE+ strain and negative control group. All the liquid M9 medium contains 0.1% IPTG and 0.1% Amp, in order to keep the same growing conditions. The total amount of bateria is detected by spectrophotometer. It provides the real-time detecting data of bacteria amount under OD600.
In IPTG experimental group, the growth curve of both NC and RelE strain fit the typical bacteria growth pattern, including the lag phase, exponential phase, stationary phase, and death phase. However, the exponential phase (the most rapid phase in which bateria grows) in RelE strain shows a shorter period than in NC group. The shorter exponential period eventually results in lower concentration of bacteria in stationary phase.
Fig 5: IPTG-induced RelE strain growth curve
The green line represents the OD value of NC, whereas the red one represents RelE strain.
The possible interference factor is the toxin effect of IPTG, which cannot be avoided in the IPTG-induced RelE generating system. To exclude the influence of toxin inducer, we construct another plasmid-- BBa_K185001 . It is an arabinose-induced RelE Generator, in which a pBAD promoter is imported. We add 0.2% L-arabinose and 0.1% Amp in M9 liquid medium. The avarage OD value in Arabinose experimental group is higher than IPTG experimental group. The growth curve verified our suspection: IPTG has slight toxin effect.
Fig 6: L-arabinose-induced RelE strain growth curve
The green line represents the OD value of NC, whereas the red one represents RelE strain.
The most important thing the curve tells us is not bacteria growth tendency, but it provides an equation through which we can calculate the bacteria division cycle. In the exponential phage, the bacteria grows rapidly through binary fission. The rate of bacteria splitting can be interpreted as time assumption in one division cycle . Therefore, we used the exponential period of the curve to calculate the proliferating rate.
To put it simply, in IPTG group, the splitting time in one generation of RelE strain is 0.8795 hour, whereas the NC is 0.5125 hour. RelE strain takes more time to split into next generation. The result is possibly due to the fact that stalled ribosomes become inactive in translation process, which leads to torpid accumulation of protein in splitting preparaion.
The enlongation of splitting time can be easily associated with extention of life span. However, whether the splitting time can represent bacteria life span remains to be a question. If not, what else can be interpreted as life span of E.coli?
Therefore, a new concept has been imported in our story – Life Span.
Part V: Research and Discussion on Bacteria Life Span
It is difficult to endow bacteria with the concept of life span. How can we define the moment of birth and death in E.coli?
For an individual bacterium, we can only analysis from splitting time. However, for a large amount of bacteria in plate or flask, the life span of an individual bacterium is meaningless because bacteria can only live as a population. In result, our focus of research is the growth cycle of a microbial population.
We designed another experiment to detect the lifetime of a microbial population. The experiment condition is the same as Detection of Arabinose-induced RelE strain growth curve ( as mentioned previously in Part IV). The difference is that we detect more points after bacteria comes into stationary phase. We try to position a turning point in which the bacteria begin decreasing.
Allowing for the systematic error which cannot be avoided, we determined the turning point of two bacteria growth curve. After turning point, the amount of bacteria starts to decline, which is a possible symbol for entering into death phase. The turning point is much delayed in RelE strain under the same culture condition so that it can not be deteced in the curve. The NC group has entered into death phase (pass through turning point) whereas RelE strain remains in the stationary phase. In other words, the recession period comes later and the active period keeps longer.
Fig 7: L-arabinose-induced RelE strain growth curve (death phase)
The blue circle in the figure features turning point.It does not appear in the RelE strain   curve. The red curve represents the growth status of NC, while the green one represents RelE strain.
Why do we senesce? This question probably has puzzled humans ever since we started thinking. Why did nature fail to evolve elaborate systems of protection and repair that could adequately prevent cellular dysfunction? Such a state of eternal youth seems, at least, indefinitely transmittable through the germ line. The variation in maximum life span is enormous, ranging from about 3 weeks in C. elegans to 120 years in humans. The modern evolutionary explanation of aging links life expectancy with the quality of maintenance and repair systems. One genetic study of aging in C. elegans has identified a gene clk-1, which appears to slow the metabolic rate of animals and extend their life span (Lakowski and Hekimi, 1996; Ewbank et al.,1997). Additional studies in C.elegans point to the importance of a genetic pathway dedicated to channeling worms to a dormant state early in development termed Dauer. The activation of this pathway in adults extends their life span significantly.
E.coli god of longevity. The life span of bacteria with inner clock is expected to suspend significantly.
Since our bioclock system in E.coli endows it an alternative life between dormancy and revivification. It may ring a bell to us that the metabolic rate is considerably decreased during dormancy due to a global translation inhibition in E.coli. This phenomenon resembles the dormant state of C.elegans in Dauer although the latter attributes to a much more complicated mechanism. It still inspires us to wonder if this inner bioclock has the potential to extend life span of E.coli.
It seems a little bit curious to research into the life expectancy of prokaryotes since the life span of an individual bacterium is meaningless because bacteria can only live as a population. In result, our focus of research is the growth cycle of a microbial population. When a microbial population is inoculated into a fresh medium, the growth curve usually includes the lag phase, exponential phase, stationary phase, and death phase. How can we tell the life span of a population from the growth curve？The answer is the duration of the curve. By drawing growth curve we can research into the life span of E.coli with an inner bioclock. It may prove a universal principle that metabolic rate does correlate to life span of organism arranging from prokaryotes to eukaryotes.
Typical growth curve for bacterial populations of negative strain and relB-relE strain.
Part VI : Appendix - Different combination of promoter and RBS
The propose of our project is to creat a bio clock in E coli by adjusting the concentration of RelE protein and RelB protein in E coli cell. So that in order to regulate concentration of RelE oscillating with a specific periodicity, we need to use different combination of promoter and RBS. This is the reason why we assemble several similar device. For example:
BBa_K185010 Promoter116+RBS34+LacI
BBa_K185017 Promoter118+LacI+RBS34
BBa_K185011 Promoter116+QPI+RBS31+RelB+Double terminator
BBa_K185018 Promoter118+QPI+RBS30+RelB+Double terminator
BBa_K185036 Promoter110+QPI+RBS34+RelB+Double terminator
BBa_K185016 Promoter118+RBS34+LacI+Double terminator+Plac+RBS31
BBa_K185019 Promoter118+RBS34+LacI+Double terminator+Plac+RBS34 BBa_K185012 Promoter116+RBS34+LacI+Double terminator+Plac+RBS34
BBa_K185007 Promoter116+RBS34+LacI+Double terminator+Plac+RBS31
According to the information of these parts in http://partsregistry.org/Main_Page , the test of Promoter110(J23110), Promoter116(J23116) and Promoter118(J23118) is
Image 1
Image 2
And the strength of RBS is also different, which can see image 2 above. So if we assemble different promoter and RBS together, we can express protein with different quantity.
Here is the data for Fig 5 ~ Fig 7: SJTU09_data_for_3_pictures.zip .
Next, go to Expanding study on eukaryotic cells .
Retrieved from " http://2009.igem.org/Team:SJTU-BioX-Shanghai/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
