"
Page
Discussion
View source
History
teams
Log in
 
Team:SJTU-BioX-Shanghai/Project design
From 2009.igem.org
Project introduction.     Inspired by the natural regulator of circadian bioclock exhibited in most eukaryotic organisms, our team has designed an E.coli-based genetic network with the toxin-antitoxin system so that the bacterium oscillates between two states of dormancy and activity (more...)
Home
The team
Introduction
Members
The album
The project
Idea & Background
Project design
Network
Modelling
Protocols
Results
Overview
Expanding study
Parts & Devices
Characterization
Human practice
Calendar
Events
Lablog
Others
Tools for Wiki
Biological safety
Team collaboration
Acknowledgements
Project design
Contents
1 Overview
2 On translation level
3 On transcription level
4 Overall output
5 One concern:How to erase the toxic effect of relE on chromosomal housekeeping genes? ------the answer is tmRNA.
Overview
E.coli the napper
E.coli the napper: our original idea is to build up an inner bioclock mastering sleep rhythem (which can only be found in higher organisms) in E.coli
Bacteria and at least some archaea harbor ‘suicide systems’, the existence of which is a mystery to those who hear about them for the first time. These systems consist of a lethal toxin that can be neutralized by an antitoxin. The production of labile antitoxin must at least parallel that of the stable toxin to circumvent the induction of cell poison. To achieve this, both proteins are encoded within a single operon and the genes are expressed with the same mRNA transcript. The toxins seem to target specific, essential cellular processes. Perhaps the best characterized of these system is relBE system, the genes of which are present on the Escherichia coli chromosome. The relE toxin is an RNase that preferentially cleaves mRNAs at the ribosomal A site between the 2nd and 3rd nucleotide of stop codons. Stop codons not only signal the end of the protein coding sequence but also serve as the binding site for release factors, which promote release of the nascent polypeptide and facilitate recycling of ribosomes for further rounds of translation. Thus truncated mRNA by cleavage of relE lacks appropriate termination signals, which causes the accumulation of stalled ribosomes and these mRNAs are unable to promote release factor binding, nascent polypeptide release, and ribosome recycling.
As a result, expression of the relE gene has been shown to severely inhibit translation and prevent colony formation.
fig 1. Expression of relE on growth of E.coli cells. From Identification of a Pair of Toxin-antitoxin(TA) Gene in the Chromosome of Cyanobacteria synechocystis sp.PCC6803
However, the growth of E.coli can be restored by coexpression of the cognate relB antitoxin, which can form a heterotetrameric (relB-relE)2 structure when binding with relE. Thus,this structure is too large to fit into the A site, so the toxic relE can be neutralized. Overexpression of relB restored protein synthesis and colony formation. From this followed that inhibition of protein synthesis by relE is bacteriostatic, a state of low metabolic rate and cell growth like the hibernation behavior of bears and hedgehogs in chilly winter, rather than bacteriocidal. But the unbound relB is an easy prey for the ATP-dependent Lon protease.
fig 2. Genetic organization and regulatory components of the relBE Toxin-Antitoxin operon(left), the site of mRNA cleavage by relE(middle), and ribosome rescue by tmRNA after mRNA truncation by relE(right). From The Bacterial Toxin RELE Displays Codon-Specific Cleavage of mRNAs in the Ribosomal A site.
What is most intricating is that relE display codon-specific cleavage of mRNAs in the ribosomal A site, that is to say, among stop codons UAG is cleaved with fast, UAA intermediate and UGA slow rate(UAG>UAA>UGA). The rates of the inhibition reactions in all cases were proportional to the relE concentration and thus determined by kcat/Km. The kcat/Km Values (s-1μM-1) for RelE Cleavage of 3 stop codons in the Ribosomal A Site have been measured previously.(UGA:0.078, UAA:2.2, UAG:26). The physical rationale of the measurement is that kcat/Km is determined by the standard free energy difference between relE in activated complex with a ribosome and a ground state with free relE and ribosome.
fig 3. RelE inhibition of peptide release from ribosomes with a tetrapeptidyl-tRNA in the P site and a stop codon in the A site. From The Bacterial Toxin RELE Displays Codon-Specific Cleavage of mRNAs in the Ribosomal A site.
On translation level
In our project, this unique relBE operon is separated into two independent genes, that is , relE and relB are located on two plasmid vectors pSB1A2 and pSB1AK3 respectively. Thus, expression of relE can be induced by substrate IPTG and that of relB by Trp. And the ATP-dependent protease Lon gene follows relE to form a polycistron in the same operon, which means Lon can coexpress with relE once induced by IPTG. When two operons are induced simultaneously, in the protein level, relE, relB, and Lon will fight against each other. On the one hand, relB can prevent cell from dormancy by neutralizing relE while on the other hand, relB can be easily degraded by Lon. It seems that with the coexpression of Lon, the toxic effect of relE can not be offset by relB. Thus it is a finality that E.Coli will hibernate to survive this stage.
On transcription level
To add suspense to this play, we decide to create a transcriptional regulation pathway to control the rate of synthesis of 3 proteins. We modify 3 genes’ stop codons artificially via primer design in PCR procedure. In detail, we alter the stop codons of relE, Lon and relB to UAA, UAG, UGA respectively, which means on the transcriptional level, the mRNA of Lon will be the most severe victim of relE cleavage whereas the mRNA of relB can almost be intact based on a pretty low relE cleavage rate. relE cuts its own mRNA at a medium rate. On this level of competiton, mRNA of relB is the biggest winner, which grants an accordingly high level of relB product via translation. As relB accumulate to an amount surpassing that of relE, the physiological state can alter from hibernation to activity, where the metabolism of bacteria returns normal and colonies can easily form.
Overall output
If we put all these together, who can tell the final champion? relE or relB? At first stage, once induced by IPTG and arabinose simultaneously, relE, Lon are coexpressed, and relB is hydrolyzed rapidly by Lon once produced. The bacteria may keep in low metabolic rate and have weak capacity to form colonies during this phase.
Nevertheless, as relE cleaves the 3 kinds of mRNAs with different stop codons, relB has the fastest synthesis rate because of a high level of intact mRNAs. In contrast, Lon are cornered to a rather low synthesis rate based on a small amount of available mRNAs. This is a perfect moment for relB to accumulate its products. When relB expression outstrips that of relE, the toxic effect of relE can be halted completely. During this stage, the microbe can regain the capacity of colony formation. It’s time to break the nightmare of Hypnos’ Curse!!!
fig 4. General mechanisms for oscillation: (1)on transcription level relE toxic inhibition of relB, relE and Lon is minimal, medium and intensive respectively. (2)On protein level relE is the prey of relB whereas relB is also the prey of Lon. These are the prerequisites for our bioclock oscillating. Thus, ideally, our E.coli can live regularly like our human beings who wake up to work in the daytime and sleep home at night.
One concern:How to erase the toxic effect of relE on chromosomal housekeeping genes? ------the answer is tmRNA.
Although some previous job has certify the fact that later induction of relB transcription in RelE-inhibited cells mediated complete recovery of colony formation.
Some may doubt that in the whole E.Coli system there are lots of key housekeeping genes such as those of RNA polymerase, peptide release factors and some catalytic enzymes in respiration, so how can bacteria recover from dormancy without these genes’ proper work, which are also affected by relE toxicity? One essential for recovery is that ribosomal complexes with peptidyl-tRNA in P site and an mRNA truncated in the A site codon can be brought back initiation. So tmRNA is required for rapid recovery of translation in RelE-inhibited cells. tmRNA is a versatile and highly conserved bacterial molecule endowed with the combined structural and functional properties of both a tRNA and a mRNA. The tmRNA system orchestrates three key biological functions: (1) recognition and rescue of ribosomes stalled on aberrant mRNAs, (2) disposal of the causative defective mRNAs, and (3)addition of a degradation tag to ribosome-associated protein fragments for directed proteolysis.
fig 5. Proposed secondary structure of E.coli tmRNA. The tRNA- and mRNA-like properties are highlighted. From Trans-Translation: The tmRNA-Mediated Surveillance Mechanism for Ribosome Rescue, Directed Protein Degradation, and Nonstop mRNA Decay.
fig 6. Trans-translation model for tmRNA-mediated protein tagging and ribosome rescue. From Trans-Translation: The tmRNA-Mediated Surveillance Mechanism for Ribosome Rescue, Directed Protein Degradation, and Nonstop mRNA Decay.
So tmRNA is also taken into our circuit to assist relB rescue the dormant cells. It does accelerate the rescue efficiency of relB. Overproduction of tmRNA efficiently counteracts relE toxicity.
Next, go to Network .
Retrieved from " http://2009.igem.org/Team:SJTU-BioX-Shanghai/Project_design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
