Team:Tsinghua/Description
Project
Home
Team
Team members
Attributions
Acknowledgement
Sponsorship
Project
Description
Design
Results
Proof of Concepts
Notebook
Modeling
Safety
Parts
Parts Summary
Basic Parts
Composite Parts
Human Practices
Overview
Public Engagement
Collaboration
Community Meetup
Integrated Practices
Medal Checklist
Bronze
Silver
Gold
Home
Team
Project
Parts
Human practice
Description
Design
Results
Proof of concepts
Notebook
Modeling
Safety
description
Background
The fidelity of genomic sequence is the underpinning aspect of normal cellular activities. Even a minor
disruption to DNA, the storage center of genetic information, can lead to detrimental consequences,
including excessive cell death, abnormal proliferation, untimely cell senescence, etc. For example,
genomic instability and mutation is considered as one of the ten hallmarks of cancer, the occurrence
of which generally leads to over-activation of signaling pathways, in turn triggering malignant
proliferation of cells[1]. Recurrent hot-spot DNA mutations and chromosomal rearrangements are
commonly observed in cancer samples. Genomic mutation is therefore a primary concern in terms of both
basic research and clinical treatment, demanding multi-disciplinary efforts.
In order to address this imperative issue, a gene mutation surveillance system that can sensitively and
efficiently detect and remove such aberrations to achieve real-time monitoring in vivo is required.
Essentially by designing such a system, a tool that can recognize aberrant mutation on mRNA sequences,
or more generally at the transcriptomics level, is desired, because transcripts are derived from
genomic sequences and hence are representatives of genomic instability and mutations. More importantly,
their availability and abundance in the cytoplasm allow them to be targeted. Nonetheless, few such
satisfying molecular tools have been developed yet. Consequently, to design such a surveillance
system is not only important but also necessary.
To serve such purpose, an RNA binding factor with high efficiency and sensitivity is a must, which
is able to distinguish RNA transcripts produced by mutated genes and switch on the suicidal gene
expression afterwards. There are three major RNA targeting methods currently: PUF proteins[2][3][4],
molecular beacons[5] and RNA aptamers[6]. Due to lack of specificity or stability in vivo, those existing
methods are not suitable for the potential application in our gene mutation surveillance system.
However, it is newly reported that dCas9[7], capable of recognizing specific endogenous mRNA
sequences, turns out to be an ideal candidate. Surveillance Cas9 (suvCas9), termed for the first time
by our team, is a variant of Cas9 without DNA cleavage activity and can target specific mRNA sequences
by manually designed sgRNA and PAM sequences. With this remarkable ability to target mRNA, suvCas9
can hopefully be adopted into our surveillance system as the monitor as well as the executor
We propose to construct suvCas9 system in Saccharomyces cerevisiae to function as the monitor
of genomic sequence. Saccharomyces cerevisiae is one of the most commonly used model organisms,
representing a simple eukaryote with a highly manipulatable genome. Apart from many physical properties
and technical advantages, such as rapid growth, minimal pathogenicity, the ease of phenotype identification,
a well-defined genetic system, and most importantly, a highly versatile DNA transformation system[11],
the yeast has been reported to share a similar characteristics of in vivo ssDNA generation by reverse
transcriptase with prokaryotes[9], which will guide suvCas9 to the targeted mRNA.
Project Overview
Genomic stability and DNA sequence fidelity are critical for normal cellular activities. If such storage
center of genetic information is disrupted, detrimental consequences will take place, including but not
limited to excessive cell death, abnormal proliferation, untimely cell senescence, etc. Therefore, a gene
mutation surveillance system that can sensitively and efficiently detect and remove such aberrations
in vivois required.
Saccharomyces cerevisiae is chosen as the model organism for designing such a gene surveillance system.
As one of the most commonly used tools, it not only has a highly manipulatable genome, rapid growth
speed, minimal pathogenicity, an ensemble of selectable markers, a well-defined genetic system, and most
importantly, a highly versatile DNA transformation system.
Essentially by designing such a system, a tool that can recognize aberrant mutation on mRNA sequences, or
more generally at the transcriptomics level, is desired, because transcripts are derived from genomic
sequences and hence are representatives of genomic instability and mutations. Surveillance Cas9 (suvCas9),
termed for the first time by our team, is a variant of Cas9 without DNA cleavage activity and can target
specific mRNA sequences by manually designed sgRNA and PAM sequences.
That being said, after mutation is detected on mRNAs, how can our system respond? In a nutshell, when
mutations occur, the surveillance system will detect and trigger the suicidal system. The surveillance
system can be further quantitatively optimized to maximize its sensitivity and minimize potential side
effects.
Contribution
This year we've also characterized and improved several parts, including the following three:
BBa_K1323002, BBa_K1493504, BBa_K1470002. Please refer to their Registry pages for detailed
documentation.
Reference
[1] Hanahan, D., & Weinberg, R. A. (2011). Hallmarks of cancer: the next generation. Cell, 144(5), 646-674.
[2] Filipovska, A., Razif, M. F., Nygård, K. K., & Rackham, O. (2011). A universal code for RNA recognition
by PUF proteins. Nature chemical biology, 7(7), 425-427.
[3] Ozawa, T., Natori, Y., Sato, M., and Umezawa, Y. (2007). Imaging dynamics of endogenous mitochondrial
RNA in single living cells. Nat. Methods 4, 413–419.
[4] Wang, Y., Cheong, C.G., Hall, T.M., and Wang, Z. (2009). Engineering splicing factors with designed
specificities. Nat. Methods 6, 825–830.
[5] Sokol, D.L., Zhang, X., Lu, P., and Gewirtz, A.M. (1998). Real time detection of DNA.RNA hybridization
in living cells. Proc. Natl. Acad. Sci. USA 95, 11538–11543.
[6] Fouts, D.E., True, H.L., and Celander, D.W. (1997). Functional recognition of fragmented operator sites
by R17/MS2 coat protein, a translational repressor. Nucleic Acids Res. 25, 4464–4473.
[7] Nelles, D. A., Fang, M. Y., O’Connell, M. R., Xu, J. L., Markmiller, S. J., Doudna, J. A., & Yeo,
G. W. (2016). Programmable RNA Tracking in Live Cells with CRISPR/Cas9. Cell, 165(2), 488-496.
[8] Farzadfard, F., & Lu, T. K. (2014). Genomically encoded analog memory with precise in vivo DNA writing
in living cell populations. Science, 346(6211), 1256272.
[9] Miyata, S., Ohshima, A., Inouye, S., & Inouye, M. (1992). In vivo production of a stable single-stranded
cDNA in Saccharomyces cerevisiae by means of a bacterial retron. Proceedings of the National Academy
of Sciences, 89(13), 5735-5739.
[10] Ran, F. A., Cong, L., Yan, W. X., Scott, D. A., Gootenberg, J. S., Kriz, A. J., ... & Koonin, E. V.
(2015). In vivo genome editing using Staphylococcus aureus Cas9. Nature, 520(7546), 186-191.
[11] Sherman, F. (1991). [1] Getting started with yeast. Methods in enzymology, 194, 3-21.
[12] O’Connell, M.R., Oakes, B.L., Sternberg, S.H., East-Seletsky, A., Kaplan, M., and Doudna, J.A. (2014).
Programmable RNA recognition and cleavage by CRISPR/Cas9. Nature 516, 263–266.
[13] Vidal, M., Brachmann, R. K., Fattaey, A., Harlow, E., & Boeke, J. D. (1996). Reverse two-hybrid and
one-hybrid systems to detect dissociation of protein-protein and DNA-protein interactions. Proceedings
of the National Academy of Sciences, 93(19), 10315-10320.
[14] Alexander, W. G., Doering, D. T., & Hittinger, C. T. (2014). High-efficiency genome editing and allele
replacement in prototrophic and wild strains of Saccharomyces. Genetics, 198(3), 859-866.
[15] Boeke, J. D., La Croute, F., & Fink, G. R. (1984). A positive selection for mutants lacking
orotidine-5′-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance. Molecular
and General Genetics MGG, 197(2), 345-346.
2016 iGEM Tsinghua - All rights reserved Adapted from Bootstrap and FIVB Follow us: igem2016_tsinghua@126.com
