Team:Tsinghua/Experiments
Project
Home
Team
Team members
Attributions
Acknowledgement
Sponsorship
Project
Description
Design
Results
Proof of Concepts
Notebook
Modeling
Safety
Parts
Parts Summary
Basic Parts
Composite Parts
Human Practices
Overview
Public Engagement
Collaboration
Community Meetup
Integrated Practices
Medal Checklist
Bronze
Silver
Gold
Home
Team
Project
Parts
Human practice
Description
Design
Results
Proof of concepts
Notebook
Modeling
Safety
design
This project aims to establish a well-controlled gene mutation surveillance system to monitor
possible harmful gene mutations. When mutations occur, the surveillance system will detect and
trigger the suicidal system. The surveillance system can be further quantitatively optimized to
maximize its sensitivity and minimize potential side effects.
RNA targeting by CRISPR/Cas9 system
To achieve the surveilling functions of the whole circuit, a suvCas9 is facilitated by an sgRNA
and a deoxynucleotide oligonucleotide (PAMmer) that hybridizes to the target RNA, providing
the PAM sequence[12]. suvCas9 plays an indispensable part in the coupling of mRNA recognition
system and the suicidal gene expression. suvCas9 performs high-fidelity mRNA binding activity,
of which the specificity is double-determined by sgRNA and PAMmer. This surveillance system is
composed of the dCas9 protein, sgRNA and PAMmer, forming a complex that can stably exist in the
cytoplasm and tightly bind to its targeted mRNA. When a mutation of the genes under surveillance
is detected by suvCas9, it can immediately trigger the suicidal system to eliminate the mutated
cell, avoiding further catastrophic consequences.
The composition of suvCas9 designed in our project
In vivo generation of ssDNA by SCRIBE system
In order to target mRNA in the cytoplasm, several concerns should be noticed[7]. First, the PAM
motif, essentially an ssDNA in our project, should be provided together with sgRNA pairs, the
presence of which are prerequisites for suvCas9 to acquire its activity. Second, the genomic
counterpart of transcripts should be prevented from being recognized and disrupted by suvCas9,
otherwise the surveillance system will contradict the purpose of maintaining high genomic fidelity.
The newly developed RNA targeting system[7] meets both criteria mentioned above: it creates the PAM motif
by introducing a special PAMmer sequence which is a mixed DNA and 2’-O-methyl RNA oligonucleotide,
and it avoids any unwanted targeting in the nucleus. However, the synthesis of such sequence is not
convenient, and the application in yeasts has not been tested before, not to mention such single-stranded
nucleotides are hard to be generated and replicated in vivo. Illustration is placed at the bottom
of this page.
The limitations of direct introduction of PAMmer sequence into yeast create a niche for us to find
alternative solutions: can we instead generate ssDNA within yeasts, as an integrated part of their
metabolic activities? It turns out that generating ssDNA in yeast cells is viable. SCRIBE (Synthetic
Cellular Recorders Integrating Biological Events)[8], intrinsically an in vivo reverse transcription
system, can reversely transcribe ssDNA with plasmids as the template[9]. More importantly, the RNA
templates for reverse transcription are highly modularized, enabling effortless ssDNA switching and
thus the flexibility of mRNA targeting. In practice, the design of molecular cloning is relatively
convenient as well: a retron and an RNA template of the wanted ssDNA sequence can be assembled on the
same plasmid, enabling a one-step transformation in yeasts.
Coupling with cellular suicidal system
Mutations in the targeted gene will cause instant translocation of suvCas9 complex, switching on the
suicidal gene expression to kill the mutated cell. Here we design to fuse dCas9 with the GAL4 binding
domain and its activation domain on the foundation of yeast one-hybrid system[13] that allows
the interaction of DNA-binding domain and the isolation of the targeted promoter and the downstream
gene. With the engineered Saccharomyces cerevisiae background strain[13] for one-hybrid system, the
interaction between the DNA-binding domain and the promoter containing recognition site will drive the
transcription of Thymine Kinase (TK) and RFP. Thymine Kinase[14] is a novel selection marker in yeast,
which is analogous to the widely used URA3 auxotrophy marker[15], allowing both selection and
counter-selection in respective media. In the selection media, which is YP-glycerol with antifolates,
only yeast populations that produce thymine kinase can survive, while on the counter-selection media,
which is SC media with FUdR, only the absence of TK can ensure the viability of yeast. Mutations
in targeted genes will cause sequence changes in mRNA, which significantly decrease the binding
activity of suvCas9 onto mRNA. The previously sequestered suvCas9 complex will be transported into
nuclear immediately in the guidance of GAL4 binding domain and the exogenous Nuclear Localization
Sequence (NLS), and turn on the transcription of TK and RFP gene, which in turn cause the death of
cell on the counter-selection media and the fluorescence expression that can be quantitatively detected
by FACS for further optimization of our surveillance system.
Figure 1A. A schematic overview of the targeting mechanism of suvCas9. suvCas9 can bind to a correct
mRNA in complex with sgRNA and PAMmer (illustrated above), which can then be translocated into the
nucleus and initiate a suicidal program whenever the mRNA is mutated (not shown). Specially designed
PAMmers will not disrupt the genomic sequence.
Figure 1B. Mechanism of the generation of ssDNA in vivo. A single strand DNA can be derived
from the reversely transcribed product by the retron (RT). Figures are adapted from (7) and (8).
Reference
[1] Hanahan, D., & Weinberg, R. A. (2011). Hallmarks of cancer: the next generation. Cell, 144(5), 646-674.
[2] Filipovska, A., Razif, M. F., Nygård, K. K., & Rackham, O. (2011). A universal code for RNA recognition
by PUF proteins. Nature chemical biology, 7(7), 425-427.
[3] Ozawa, T., Natori, Y., Sato, M., and Umezawa, Y. (2007). Imaging dynamics of endogenous mitochondrial
RNA in single living cells. Nat. Methods 4, 413–419.
[4] Wang, Y., Cheong, C.G., Hall, T.M., and Wang, Z. (2009). Engineering splicing factors with designed
specificities. Nat. Methods 6, 825–830.
[5] Sokol, D.L., Zhang, X., Lu, P., and Gewirtz, A.M. (1998). Real time detection of DNA.RNA hybridization
in living cells. Proc. Natl. Acad. Sci. USA 95, 11538–11543.
[6] Fouts, D.E., True, H.L., and Celander, D.W. (1997). Functional recognition of fragmented operator sites
by R17/MS2 coat protein, a translational repressor. Nucleic Acids Res. 25, 4464–4473.
[7] Nelles, D. A., Fang, M. Y., O’Connell, M. R., Xu, J. L., Markmiller, S. J., Doudna, J. A., & Yeo,
G. W. (2016). Programmable RNA Tracking in Live Cells with CRISPR/Cas9. Cell, 165(2), 488-496.
[8] Farzadfard, F., & Lu, T. K. (2014). Genomically encoded analog memory with precise in vivo DNA writing
in living cell populations. Science, 346(6211), 1256272.
[9] Miyata, S., Ohshima, A., Inouye, S., & Inouye, M. (1992). In vivo production of a stable single-stranded
cDNA in Saccharomyces cerevisiae by means of a bacterial retron. Proceedings of the National Academy
of Sciences, 89(13), 5735-5739.
[10] Ran, F. A., Cong, L., Yan, W. X., Scott, D. A., Gootenberg, J. S., Kriz, A. J., ... & Koonin, E. V.
(2015). In vivo genome editing using Staphylococcus aureus Cas9. Nature, 520(7546), 186-191.
[11] Sherman, F. (1991). [1] Getting started with yeast. Methods in enzymology, 194, 3-21.
[12] O’Connell, M.R., Oakes, B.L., Sternberg, S.H., East-Seletsky, A., Kaplan, M., and Doudna, J.A. (2014).
Programmable RNA recognition and cleavage by CRISPR/Cas9. Nature 516, 263–266.
[13] Vidal, M., Brachmann, R. K., Fattaey, A., Harlow, E., & Boeke, J. D. (1996). Reverse two-hybrid and
one-hybrid systems to detect dissociation of protein-protein and DNA-protein interactions. Proceedings
of the National Academy of Sciences, 93(19), 10315-10320.
[14] Alexander, W. G., Doering, D. T., & Hittinger, C. T. (2014). High-efficiency genome editing and allele
replacement in prototrophic and wild strains of Saccharomyces. Genetics, 198(3), 859-866.
[15] Boeke, J. D., La Croute, F., & Fink, G. R. (1984). A positive selection for mutants lacking
orotidine-5′-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance. Molecular
and General Genetics MGG, 197(2), 345-346.
2016 iGEM Tsinghua - All rights reserved Adapted from Bootstrap and FIVB Follow us: igem2016_tsinghua@126.com
