Team:Tsinghua-A/test
Team:Tsinghua-A - 2016.igem.org
Home
Project
Modeling
Human Practice
Collaboration
Team&Attribution
Tsinghua-A 2016
PROJECT
In this column we are going to present the description,experiment and result of our project
Abstract
When information flows through gene-regulatory networks, noise is introduced, and fidelity suffers. A cell unable to correctly infer the environment signals from the noisy inputs may be hard to make right responses. But there are so many “parallel circuit” in gene-regulatory networks where independently transcribed monomers assemble into functional complexes for downstream regulation. As we know, when we talking with two person at the same time, we could not fully understand them. But if things they talked about were more similar, we could understand their words much better. So that would tighter connection of monomers benefit the quality of gene-regulatory networks?
Inspired by these thoughts, we construct synthetic biology circuit by using split florescent proteins. And we add inteins to split florescent proteins to the make the connection tighter. Then, we quantitatively measure the capacity of these information channels. Computation and wet lab work are combined to optimize our understanding of such systems, and to interpret potential biological significance of reoccurring parallel designs in nature.
Description
From information theory to our project
As we can learn from Wikipedia[1], information theory studies the quantification, storage, and communication of information which was originally proposed by Claude E. Shannon in 1948. The theory has developed amazingly and has found applications in many areas. It’s not exaggerated to say that we can see the power of information theory all the time.
Based on this, we would like to explore something originally in biological pathway using information theory.
Our project has a close connection with two terms in information theory, mutual Information and channel capacity. And what are them?
In probability theory and information theory, the mutual information (MI) of two random variables is a measure of the mutual dependence between the two variables. More specifically, it quantifies the "amount of information" (in units such as bits) obtained about one random variable, through the other random variable. The concept of mutual information is intricately linked to that of entropy of a random variable, a fundamental notion in information theory, that defines the "amount of information" held in a random variable. You can understand it trough from the figure below.
Formally, the mutual information of two discrete random variables X and Y can be defined as:\[I\left( {X;Y} \right) = \sum\limits_{y \in Y} {\sum\limits_{x \in X} {p\left( {x,y} \right)\log \left( {\frac{{p\left( {x,y} \right)}}{{p\left( x \right)p\left( y \right)}}} \right)} } \]
And the formula can also be proofed:\[I\left( {X;Y} \right) = H\left( X \right) - H\left( {X\left| Y \right.} \right) = H\left( Y \right) - H\left( {Y\left| X \right.} \right)\]
From the formula you can easily think that mutual information is the reduction of uncertainty in $X$ when you know $Y$.
And when you understand the mutual information, you can easily understand what the channel capacity is.
In electrical engineering, computer science and information theory, channel capacity is the tight upper bound on the rate at which information can be reliably transmitted over a communications channel. And by the noisy-channel coding theorem, the channel capacity of a given channel is the limiting information rate (in units of information per unit time) that can be achieved with arbitrarily small error probability. And you can also see the figure below.
The channel capacity is defined as:\[C = \mathop {\sup }\limits_{{p_X}\left( x \right)} I\left( {X;Y} \right)\]
where the supremum is taken over all possible choices of ${p_X}\left( x \right)$.
After you know about all the concept above, now we are glad to tell you that you can easily follow us and find out many interesting and inspiring things in our project. Congratulations!
miRNA as Potential Biomarkers
Studies have confirmed that miRNA expression is highly concordant cross individuals. And aberrant expression of miRNA may relate to diseases. Some studies have reported that specific miRNAs in tissue and plasma can be discriminatory bio-makers for detecting cancers. However, getting access to tissue of plasma means physical harm to the human body. So we turn to the easily accessible saliva. Since saliva is considered to be a terminal product of blood circulation, components like proteins and RNAs which are present in plasma are also present in saliva. In fact, both coding RNAs and non-coding RNAs, including some miRNAs, have been found in human saliva. Although mRNAs are highly degraded in saliva, miRNAs are stably and abundantly present in saliva. Recently, there are many reports on cancer-related miRNA expression in saliva. Featured miRNA expression are reported to be found in oral squamous cell carcinoma, parotid gland tumors and esophageal cancer, indicating potential salivary miRNA to be biomarkers for detecting these diseases. According to the test done by Zijun Xie group, there are three type of miRNAs significantly upregulated in the whole saliva from the esophageal cancer patient group in contract to normal control group – miR-10b, miR-144, and miR-451 (p value 0.001, 0.012 and 0.002, respectively; AUC 0.762, 0.706 and 0.756, respectively). And four miRNAs are significantly upregulated in saliva supernatants from the esophageal cancer patient group – miR-10b, miR-144, miR-21 and miR-451. Among them, miR-21 is the most frequently reported one for its high performance in specific expression level related to esophageal cancer (according to one of the reports, p < 0.05, AUC = 0.8820, sensitivity = 90.20% and specificity 70.69%; different tests may report different results).
Comprehensively considering the performance of each miRNA, we finally chose miR-144 as our biomarkers for esophageal cancer. To make our detection quick and convenient, we designed synthetic gene pathways based on paper. It will only require some saliva to complete the detection, which will do no harm to human body. And this techniques can be expanded to be used in the detection of many other diseases which has specific miRNA expression pattern in saliva.
To make our detection quick and convenient, we designed synthetic gene pathways. It will only require some saliva to complete the detection, which will do no harm to human body. And this technique can be expanded to be used in the detection of many other diseases which has specific miRNA expression pattern in saliva.
What is Toehold Switch
We choose toehold switch as our miRNA detector. The structure of toehold switch is similar to hairpin, except it has a loop at the top as ‘toehold’. Toehold switch functions as riboregulator through linear-linear interaction between RNAs. When target RNA appears, it will bind one of the toehold switch stems and open the loop, exposing the RBS.
Toehold switch systems are composed of two RNA strands referred to as the switch and trigger. The switch RNA contains the coding sequence of the gene being regulated. Upstream of this coding sequence is a hairpin-based processing module containing both a strong RBS and a start codon that is followed by a common 21 nt linker sequence coding for low-molecular-weight amino acids added to the N terminus of the gene of interest. A single-stranded toehold sequence at the 50 end of the hairpin module provides the initial binding site for the trigger RNA strand. This trigger molecule contains an extended single-stranded region that completes a branch migration process with the hairpin to expose the RBS and start codon, thereby initiating translation of the gene of interest.
Our Circuit to Detect miRNA-144
There are 3 parts in our circuit in total.
Part 1 includes toehold switch for miRNA-144 and GFP coding sequence. When miRNA-144 exists, the switch is on and mRNA for GFP is transcribed.
Part 2 contains toehold switch for GFP mRNA and T3 RNA polymerase coding sequence (BBa_K346000). Note that, since the maximum length of trigger RNA for toehold switch is about 25nt, so we analyzed GFP mRNA’s structure and choose a small piece from it which ensures binding specificity and stability.
Part 3 is simply a GFP generator (BBa_E0840), with T3 promoter. As we know, T3 promoter can only function when bound with T3 RNA polymerase.
So now it’s clear that, part 2 and part 3 are designed for amplification. They form a positive feedback loop. So the whole process is as follow: when miRNA-144 exists, it triggers part 1 and GFP mRNAs are transcribed. These mRNA on the one hand can be directly translated to GFP and show green fluorescence, on the other hand, can trigger part 2 and T3 RNA polymerase is transcribed and translated, which enables part 3 to work, thus transcribe more GFP mRNAs. And these GFP mRNAs can also be used to active more part 2.
Experiment & Results
Experiment
Protocols:
Experiments:
We transfect HEK-293 human cells with our plasmid constructions as described in the form [ref: table]. Different concentrations of Dox are applied to cell culture at the same time.
Transfected cells are cultured for 48 hours before performing flow cytometry, long enough for protein expression level to achieve steady state. FACS examination measures florescent intensity emitted by each cell, from which we obtain a large sample of florescent protein expression level, tens of thousands of cells for each experiment group.
Data collected from flow cytometry are later analyzed on computers. We estimated probability density function (p.d.f.) from data using kernel density estimation, a nonparametric statistics method. Given high and low Dox concentration input, cells exhibit different probability distributions, as illustrated in the example below [ref: fig].
What we have in hand is the conditional distribution $p\left( {Y\left| {X = x} \right.} \right)$ , given a known level of input $x$ . In order to calculate mutual information $I\left( {X;Y} \right) = \iint {p\left( {x,y} \right){{\log }_2}\frac{{p\left( {x,y} \right)}}{{p\left( x \right)p\left( y \right)}}dxdy}$  and estimate channel capacity, which is $C = \sup I\left( {X;Y} \right)$ , we need to find the input distribution $p\left( X \right)$ and joint distribution $p\left( {X,Y} \right)$ that optimizes the equation. $p\left( X \right)$ , however, is not known in the first place. We first randomly pick a stochastic vector as the initial input distribution and then use an optimization algorithm to iterate the function and maximize $I\left( {X;Y} \right)$ . The final result is the channel capacity.
Results
Do our circuits work?
Yes, they do sense the input level of Dox concentration. Figure. illustrates the changing distribution of EBFP2 florescent intensity in response to Dox gradient. With higher concentration, the distribution shifts to the right till reaching saturation. (TRE-EBFP2N:IntN and TRE-EBFP2C:IntC group is displayed as an example)
The shift, however, is only intuitive. We need more accurate methods to study the quantitative properties. To do this, we plot transfer functions of each group. Transfer function demonstrates the relation between the input level (Dox concentration) and the output level (amount of florescent protein). Plot the function, and the shape of the curve is highly informative.
The transfer functions of all seven groups are illustrated below. All values are in logarithm space. Note that for the convenience of plotting, the points where Dox=0 are plotted at Dox=0.01. (or the point will fly out far to negative infinity)
In the leftmost figure, EBFP2 without intein sequence show relatively low affinity and thus low expression level. Nevertheless, their leakage level is low as well, and Dox induction leads to approximately  fold change. As for the middle and right figures, both split EBFP2 with intein and intact EBFP2 have about  fold change when induced by Dox, but split EBFP2 have lower leakage level.
Meanwhile, if one half of EBFP is driven by constitutive promotor CMV, the leakage level remains the same but the induced multiple suffers. This is expected beforehand because with one constitutively-expressed part, the circuit can only sense the input with one half of the split proteins, thus becoming slightly less inducible.
Normalizing the curves lead to more interesting discoveries. Even though TRE-EBFP2N + CMV-EBFP2C leads to poor fold change, the transfer curve is significantly steeper when the dimerization process is reversible. This means better switch-like properties. With the presence of intein, the effect is weaker but still visible.
Normalize transfer curves to the range of 0 to 1, we can find that the shapes are different. Lines representing split proteins are later to rise and steeper.
If we normalize the initial EBFP2 level to 1, split EBFP2 with intein displays better properties than the other two settings. From fig. we can clearly see that it has the highest multiple among the three, even significantly higher than that of the intact EBFP2. The result shows that split proteins, with high binding affinity, can defeat original undivided proteins for their low leakage level and high induced multiples, that is, high sensibility to inputs.
How well do circuits perform as evaluated by channel capacity?
I Seven circuits are evaluated in our experiment. Calculated channel capacities are displayed in fig.
Come on, this not as dizzying as it is at the first glance. Let’s look at it step by step.
More information is transmitted when both parts of the split protein are inducible.
When both promotors are TREs, both split parts are inducible, and channel capacity is relatively higher than that of channels with un-inducible CMVs. In the absence of intein, the two peptides find it hard to dimerize, giving rise to low channel capacity.
Upon addition of intein sequence, the binding process becomes irreversible since the two halves assemble into one intact protein through splicing. As a consequence, channel capacity greatly increases. Double-inducible group with two TRE promotors still win the competition speaking of channel capacity.
Comparing three inducible groups leads to the conclusion, that splitting leads to decrease in channel capacity, but adding intein sequence to peptides rescue the effect, elevating the channel capacity to even higher level.
What can the result teach us?
Inspirations for Synthetic Biology Engineering:
For synthetic biologists, it is crucial and challenging to construct AND gate. Split up a regulatory protein such as transcription factor, express two halves independently, and an AND gate is born.
Nonetheless, the act of splitting up can bring about unexpected side effects. Gene regulatory circuits are highly dependent on quantitative properties, its complexity and nonlinearity contributing to hard-to-predict behaviors of biological systems. Once an important part in the system is chopped up, who knows what will happen next?
Our program quantitatively studies the behavior of such systems. Splitting up changes the circuit’s output-input function, alleviates leakage phenomenon, improving switch-like property, and increases fold change when induced by circuit inputs. Moreover, we use channel capacity from information theory to describe how well can they transmit signals. We find adding intein sequence tremendously beneficial in that it shifts the channel capacity to a higher level, thus ameliorating uncertainty.
When it comes to designing logic gates, our findings can lead the way. Not only can splitting achieve logic gate effect, but also can it improve sensibility to inputs and defend the system against detrimental interferences of noise when intein is added. Future work shall benefit from this fundamental investigation of basic synthetic biology blocks.
Highlighting the biological significance of dimerization:
Dimerization is only too common in cells. Monomers assembly into dimers for further functions all the time, some interactions strong, some interactions weak. Function-less newborn peptides piece together and get to work, forming so-called tertiary structure; activated kinases reach each other and mutually phosphorylate; transcription factors, when forming homo- or hetero-dimers according to different stoichiometry, leads to varied downstream responses and distinct cellular fates…
Yes, we know which proteins dimerize. We understand how proteins dimerize as well, by interaction of domains like leucine zippers and so forth. But why? What is the point of dimerization?
Previous researches have underlined the important advantages of dimerization, including differential regulation, specificity, facilitated proximity and so on. [citation needed] The influence of dimerization in noise propagation is hardly touched due to the difficulty in controlling experiment variables. Synthetic biology provides powerful tools to carry out experiments otherwise impossible in designed systems. This is exactly what we do.
Traditionally, we evaluate the impacts of noise using variance-related statistics, such as coefficient of variance. These quantities can only describe how concentrated the output is around the mean value, but cannot tell us how well we can infer one of the correlated random variables from the other. Channel capacity makes a better criteria of noise because it more scientifically depicts the information dissemination process.
Reference
1. Jörn M. Schmiedel et al. MicroRNA control of protein expression noise. Science 348, 128 (2015); DOI: 10.1126/science.aaa1738
2. Christian M Metallo and Victor Sourjik. Environmental sensing, information transfer, and cellular decision-making. Current Opinion in Biotechnology 2014, 28:149–155
3. Rouillard J M, Lee W, Truan G, et al. Gene2Oligo: oligonucleotide design for in vitro gene synthesis[J]. Nucleic acids research, 2004, 32(suppl 2): W176-W180.
4. Shimizu Y, Inoue A, Tomari Y, et al. Cell-free translation reconstituted with purified components[J]. Nature biotechnology, 2001, 19(8): 751-755.
5. Gibson D G, Young L, Chuang R Y, et al. Enzymatic assembly of DNA molecules up to several hundred kilobases[J]. Nature methods, 2009, 6(5): 343-345.
6. http://www.snapgene.com/resources/plasmid_files/your_time_is_valuable/
7. http://www.addgene.org/plasmid-protocols/gibson-assembly/
8. http://www.snapgene.com/resources/gibson_assembly/
9. Lin X, Lo H C, Wong D T, et al. Noncoding RNAs in Human Saliva as Potential Disease Biomarkers[J]. Frontiers in Genetics, 2015, 6.
10. Zijun, Xie, Gang, Chen, Xuchao, Zhang, et al. Salivary MicroRNAs as Promising Biomarkers for Detection of Esophageal Cancer[J]. Plos One, 2013, 8(4):e57502.
11. Minhua Y E, Penghui Y E, Zhang W, et al. [Diagnostic values of salivary versus and plasma microRNA-21 for early esophageal cancer].[J]. Journal of Southern Medical University, 2014, 34(6):885-889.
12. http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi
13. http://www.nupack.org
14. http://helixweb.nih.gov/dnaworks
15. http://berry.engin.umich.edu/gene2oligo
Acknowledgement
Tsinghua Team help us dry our part for part submission with their machines.
MODELING
We attempt to describe, in a precise way, an understanding of the elements of a system of interest, their states, and their interactions with other elements.
Overview
We perform deterministic and stochastic mathematical modeling alongside wet lab experiments, and the calculated channel capacity values from models and experimental data fit well with each other. The ability to precisely transmit information is crucial for biological process, which is quantitatively evaluated by channel capacity, a novel concept in information theory. Structure differences of gene regulatory circuits give rise to different capacities and, in turn, different levels of information certainty and reliability. Specifically, although splitting the expressed protein into two halves that can assembly to form dimer provides more flexible functions and other regulatory benefits, it curtails the information capacity compared to expressing the undivided protein. Adding intein sequence to the two parts, however, compensated for the loss of capacity by making the association process irreversible. Simulation results tie in with data measured from synthetic gene regulatory circuits, showing the same conclusions. Furthermore, our models predict several conclusions under conditions we not yet have time to carry out, or even hard to achieve experimentally, and illuminates the way for future research and design.
Gillespie Algorithm Simulation
Gillespie Algorithm, a Stochastic Simulation Algorithm (SSA), can simulate biochemical reactions with small number of molecules accurately. Reactions in cells are full of random fluctuations due to low number of molecules present. For instance, there can be only one or two copies of a gene in one cell. (see https://en.wikipedia.org/wiki/Gillespie_algorithm for more details)
TCaption: an example simulation result of our script using Gillespie Algorithm. 100 cells are generated in this figure. We can see how the state of the cell (in this case, a one-dimension state space )
We use this algorithm to generate hundreds of sample trajectories at high and low inputs, each representing the dynamic changes in the amount of protein in a single cell. Afterwards, we find a time point when all reactions in cells have reached steady state, and ink down the states of the cells at that very moment. The approach resembles the flow cytometry experiment, in both of which we get a sample composed of cells with dissimilar protein levels.
From the simulated data, we can use methods in experiment data processing to calculate channel capacities.
We use ${k_{on}}$ and ${k_{off}}$, two parameters of dimer forming and disassociation, to characterize the process of dimerization. For split EBFP2 without intein, kon is relatively small due to low affinity, and ${k_{off}} > 0$. For split EBFP2 with intein, kon is larger and ${k_{off}} = 0$ because once they interact, they will not fall apart.
Under suitable parameter set, we obtained the following results shown in the table below. When ${k_{off}}$  comes down to zero, channel capacity increases in accordance to experimental results.
We computationally scan the two parameters in two-dimensional parameter space to explore the impacts of those kinetic constants.
In the heat map, warmer color stands for higher channel capacity, vice versa. The figure on the left stands for the case where both halves are driven by inducible promotors, the right for that where one of them is driven by constitutive promotor. The latter resemble chemical titration, so we legend it as above.
The lower left part of the map is warmer. In this region, ${k_{on}}$ is high and ${k_{off}}$ is low, which means that two peptides are prone to associate. This is the very district where the state of split EBFP2 with intein lies in.
The higher right part, on the contrary, is chilling to the bones. Here, ${k_{on}}$ is low and ${k_{off}}$ is high, which means that the two parts tend to disassociate. The state of spilt EBFP2 without intein falls into this region.
If we compare the two figures in parallel, we can see that the values in the “titration” map is lower than those in the “dimer” map. This is in accordance with our experimental data. Wet lab work shows that with a constitutively-expressed part, channel capacity is lower. The model match and explain the phenomenon.
To sum up, mathematical modeling reveals the impacts of thermodynamic constants on the gene regulatory circuit’s ability to convey information.
Model2
Using CV Technology to Quantitative Forecasting
Hypothetically, you use the test paper we design and you can see three lines (Which means there are three kinds of mi-RNA have been found in your body fluid). Don’t worry, it doesn’t necessarily mean you got the cancer. Because the amount of those three mi-RNA has to satisfy this equation：
So, we have to turn the GFP brightness information into the possibility of patient being afflicted with cancer. And we use the CV technology to help us achieve that goal.
For example, we now have the photo of the test paper(The color of test paper itself might be light yellow due to the ambient light or the paper quality).
We use Matlab to read this file and we can get an array of the RGB value of this image.Then,we can use this function to calculate the Brightness of each pixel:Y=0.299*R+0.587*G+0.114*B
Then we have to normalize these data to eliminate the effect of the ambient light.We pick an pixel in the “white” area and calculate its brightness/255, set this as constant C, and use it to multiply every pixel’s brightness.
After that, we will get a new array of the normalized brightness information. We can use the relative position to pick 10 pixel in each line, and calculate the mean value of each line’s brightness. We assume that there is a linear relationship between the brightness of the system and the amount of GFP in the system. Now we can get a relative stoichiometric relation of three mi-RNA. We use the function mentioned above to sum the data proportionately. We set B2P(Brightness to Possibility) as the symbol of this value.
Finally we use this data to calculate the possibility of having cancer.(Of course we have to get the threshold of B2P value. But that can be easily calculated through a machine learning or a basic statistical method if we can get enough samples.)
Outlook
Exactly，all of the experiments we have done is to improve the channel capacity and reduce the disturbance from the noise at the same time. Anyway our final target is to make sure that we can transfer more information when the environment is noisy. Our team focuses on comparing performance between two ways to get fluorescent protein (EBFP), namely, to express the protein directly from the gene, or to independently express two proteins and let them assembly afterwards to become the same protein as the one in the first way. Beside analyzing the experimental data by calculating channel capacity of two systems we wanted to know if there are any difference between two circuits in amplifying the inputs or responding to the noise. And we also want to know which frequency of noise makes our circuit worse and which signal-to-noise ratio may have a higher channel capacity?
So we have these simulations first and we will conduct more experiments to collect data and confirm our conclusion.
Paper of this model(Click here)
HUMAN PRACTICE
We try to "go beyond the lab" to imagine our projects in a social context, to better understand issues that might influence the design and use of our technologies.
Synthetic Biology
Meets
Daily Life
Overview
Our project is about information flowing through gene-regulatory networks and it contains both theory of information and biology. The human practice we did fit our project very well. We show the students who have no idea about what synthetic biology is the fun it have and also make the cross subjects knew by more people. We all think we have did a meaningful practice and help promote development of the cross subjects which is also the aim of iGEM. In a word, human practice gives us a chance to tell more people about our idea and the makes more people realize the charm of synthetic biology.
Eliminate Chinese’s prejudice about biotechnology
After hearing the community's concern about biotechnology, we held two lectures to clear their prejudice. Meanwhile, we printed 200 booklets to expand publicity.
Background
In China, because of the UN human issue that scientists used 60 children to examine their new transgenic product, Golden rice last year, the question about the safety of transgenic technology remains indistinct. The masses even became scared of the transgenic technology as well as the total biotechnology. As a result, transgenic products were resisted all over China.
Lectures
Although many biologists came out to prove their justice, many Chinese still have stereotypes of biotechnology. We hosted two lectures about the correctness of biotechnology for college freshmen and high school students. We introduced the current stereotype phenomenon, then presented the biological applications that have made great contributions, such as bio-fuel, Artemisinin (kind of drug for malaria, first extracted by Chinese pharmacist Youyou Tu) produced by yeast and microorganism which can curb environmental pollution. Because our project involve the knowledge of both biological process and the lecture also involved the development of Synthetic Biology, which is a course of cross-disciplines, requiring us to look at a living organism from the perspective of information system. These contents attracted the attention of students from College of Information.
Brochures
To circulate synthetic biology knowledge and inspiration, we demonstrated key ideas in the discipline in a booklet and handed out the booklets to more high school students. Two students e-mailed us to express their interest in synthetic biology and consulted to us about how to be admitted by Tsinghua University.
COLLABORATION
Sharing and collaboration are core values of iGEM.
Communication & Collabroation
With University of Science & Technology Beijing (USTB)
We cooperated with the iGEM team from University of Science & Technology Beijing (USTB) and we both learned a lot of thing from each other.
We often had friendly conversions through WeChat and told each other what our projects is about and how our projects is going. They told us that they had some trouble in assembling a plasmid they needed. They tried a lot of ways to assemble the plasmid but failed and they asked us for help. They sent us the photo of their unsuccessful plasmid and after some analyses we suggested them using a method called Gibson Assembly. They adopted our opinions and did some try again. Later, they told us that they had made it successfully and we were all glad to hear that.
We also invited them to our school and talked about our projects. We learned about that their project is about a kind of compound named notoginseng saponin. They told us about the process and purpose of their experiment in detail and we noticed that there are some cases they ignored. And also, we help them optimize the process of their experiment and gave them a hand on the analyzing of their result. At the same time, we invited them to our laboratory, showed them how our experiment goes on and also, gave them some advice on the operations they should take care of when they did their experiment. And they also gave us a lot of useful suggestions and all of us had a great and meaningful day!
With ShanghaiTech University
Computing Resource Support for Modeling——Simulating our system with Gillespie Algorithm, we fell short of computing power when it comes to scanning key parameters. ShanghaithchChina generously helped us out by running our code on their servers, which significantly boosted efficiency.
Illustrating Science——We illustrated ShanghaitechChina's project with cartoon, visualizing the process. The image is used in [where it is used]. When art encounters science...
Team&Attribution
......
Meet Our Amazing Team
Instructors
Zhen Xie
Professor
Advice on Wet Lab
Xiaowo Wang
Professor
Advice on Dry Lab
Weixi Liao
Phd Candidate
Advice on Wet Lab
Lei Wei
Phd Candidate
Advice on Dry Lab
Students
Yuxi Ke
Biological Science
Team Leader/Data Processing/Wet Lab
Heng Yu
Automation
Team Leader/Human Practice
Zizhuo Wang
Automation
Modeling
Ao Yu
Automation
Human Practice
Ruogu Lin
Automation
Wiki Building
Qiaoyu Lu
Automation
Wet Lab
Xingyu Sha
Automation
Human Practice
Yiyang Zhang
Automation
Wet Lab
Jie Zheng
Automation
Modeling
Ran Li
Automation
Wet Lab
Siyu Li
Automation
Safety
Yubo Zhang
Automation
Modeling
Linghan Wang
Biological Science
Wet Lab
2016 iGEM
Tsinghua-A
