<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tsinghua_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tsinghua/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Project</TITLE></P><FORM method="post" action="/" id="MainForm"><DIV id="AngularPanel" class="main-wrapper" ng-app="fivb"><DIV id="galaxy-bar" class="gb-styling" style="margin:15px 0px 0px 0px;"><DIV class="navigation"><DIV class="hamburger-submenu"><UL><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_0" href="https://2016.igem.org/Team:Tsinghua">Home</A></SPAN></LI><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_1" href="https://2016.igem.org/Team:Tsinghua/Team">Team</A></SPAN></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_1_Link_0" href="https://2016.igem.org/Team:Tsinghua/Team">Team members</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_1_Link_1" href="https://2016.igem.org/Team:Tsinghua/Team">Attribution</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_1_Link_2" href="https://2016.igem.org/Team:Tsinghua/Team">Acknowledgement</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_1_Link_3" href="https://2016.igem.org/Team:Tsinghua/Team">Sponsorship</A></LI></UL><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_2" href="https://2016.igem.org/Team:Tsinghua/Project">Project</A></SPAN></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_0" href="https://2016.igem.org/Team:Tsinghua/Project">Description</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_1" href="https://2016.igem.org/Team:Tsinghua/Project">Design</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_2" href="https://2016.igem.org/Team:Tsinghua/Project">Experimental Results</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_3" href="https://2016.igem.org/Team:Tsinghua/Project">Proof of concepts</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_4" href="https://2016.igem.org/Team:Tsinghua/Project">Notebooks</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_5" href="https://2016.igem.org/Team:Tsinghua/Project">Modeling</A></LI><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_3" href="https://2016.igem.org/Team:Tsinghua/Parts">Parts</A></SPAN></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_0" href="https://2016.igem.org/Team:Tsinghua/Parts" target="_blank">Parts Summary</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_1" href="https://2016.igem.org/Team:Tsinghua/Parts">Basic Parts</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_2" href="https://2016.igem.org/Team:Tsinghua/Parts">Composite Parts</A></LI><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_3" href="https://2016.igem.org/Team:Tsinghua/HumanPractices">Human Practices</A></SPAN></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_0" href="https://2016.igem.org/Team:Tsinghua/HumanPractices" target="_blank">Collaboration</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_1" href="https://2016.igem.org/Team:Tsinghua/HumanPractices">Public Engagement</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_2" href="https://2016.igem.org/Team:Tsinghua/HumanPractices">Integrated Human Practices</A></LI><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_4" href="https://2016.igem.org/Team:Tsinghua/MedalChecklist">Medal Checklist</A></SPAN></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_4_Link_0" href="https://2016.igem.org/Team:Tsinghua/MedalChecklist#bronze" target="_blank">Bronze</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_4_Link_1" href="https://2016.igem.org/Team:Tsinghua/MedalChecklist#silver">Silver</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_4_Link_2" href="https://2016.igem.org/Team:Tsinghua/MedalChecklist#gold">Gold</A></LI></DIV><DIV class="galaxy-main-navigation"><NAV><UL class="quicklinks"><LI><A href="https://2016.igem.org/Team:Tsinghua">Home</A></LI><LI><A href="https://2016.igem.org/Team:Tsinghua/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Tsinghua/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:Tsinghua/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Tsinghua/HumanPractice">Human practice</A></LI></UL></NAV></DIV></DIV></DIV></DIV><DIV class="subnavigator" id="subnavigator" style="position:absolute;left:30px;top:330px;"><A href="https://2016.igem.org/Team:Tsinghua/Project#tag_description">Description</A><A href="https://2016.igem.org/Team:Tsinghua/Project#tag_design">Design</A><A href="https://2016.igem.org/Team:Tsinghua/Project#tag_results">Results</A><A href="https://2016.igem.org/Team:Tsinghua/Project#tag_proofOfConcepts">Proof of concepts</A><A href="https://2016.igem.org/Team:Tsinghua/Project#tag_notebook">Notebook</A><A href="https://2016.igem.org/Team:Tsinghua/Model" target="_blank">Modeling</A><A href="https://2016.igem.org/Team:Tsinghua/Project#tag_safety">Safety</A></DIV><DIV class="contentPage2"><DIV class="contentPage" style="margin-left:200px;margin-top:20px;width:1130px;"><H1 class="contentPage" id="tag_description">description</H1><H2 class="contentPage">Background</H2>
             The fidelity of genomic sequence is the underpinning aspect of normal cellular activities. Even a minor
             disruption to DNA, the storage center of genetic information, can lead to detrimental consequences,
             including excessive cell death, abnormal proliferation, untimely cell senescence, etc. For example,
             genomic instability and mutation is considered as one of the ten hallmarks of cancer, the occurrence
             of which generally leads to over-activation of signaling pathways, in turn triggering malignant
              proliferation of cells<SUP>[1]</SUP>. Recurrent hot-spot DNA mutations and chromosomal rearrangements are
             commonly observed in cancer samples. Genomic mutation is therefore a primary concern in terms of both
             basic research and clinical treatment, demanding multi-disciplinary efforts.
             In order to address this imperative issue, a gene mutation surveillance system that can sensitively and
             efficiently detect and remove such aberrations to achieve real-time monitoring <I>in vivo</I> is required.
             Essentially by designing such a system, a tool that can recognize aberrant mutation on mRNA sequences,
              or more generally at the transcriptomics level, is desired, because transcripts are derived from
             genomic sequences and hence are representatives of genomic instability and mutations. More importantly,
              their availability and abundance in the cytoplasm allow them to be targeted. Nonetheless, few such
              satisfying molecular tools have been developed yet. Consequently, to design such a surveillance
             system is not only important but also necessary.
             To serve such purpose, an RNA binding factor with high efficiency and sensitivity is a must, which
              is able to distinguish RNA transcripts produced by mutated genes and switch on the suicidal gene
              expression afterwards. There are three major RNA targeting methods currently: PUF proteins<SUP>[2][3][4]</SUP>,
             molecular beacons<SUP>[5]</SUP> and RNA aptamers<SUP>[6]</SUP>. Due to lack of specificity or stability in vivo, those existing
              methods are not suitable for the potential application in our gene mutation surveillance system.
              However, it is newly reported that dCas9<SUP>[7]</SUP>, capable of recognizing specific endogenous mRNA
             sequences, turns out to be an ideal candidate. Surveillance Cas9 (suvCas9), termed for the first time
              by our team, is a variant of Cas9 without DNA cleavage activity and can target specific mRNA sequences
              by manually designed sgRNA and PAM sequences. With this remarkable ability to target mRNA, suvCas9
              can hopefully be adopted into our surveillance system as the monitor as well as the executor
             We propose to construct suvCas9 system in <I>Saccharomyces cerevisiae</I> to function as the monitor
              of genomic sequence. <I>Saccharomyces cerevisiae</I> is one of the most commonly used model organisms,
              representing a simple eukaryote with a highly manipulatable genome. Apart from many physical properties
              and technical advantages, such as rapid growth, minimal pathogenicity, the ease of phenotype identification,
              a well-defined genetic system, and most importantly, a highly versatile DNA transformation system<SUP>[11]</SUP>,
              the yeast has been reported to share a similar characteristics of <I>in vivo</I> ssDNA generation by reverse
              transcriptase with prokaryotes<SUP>[9]</SUP>, which will guide suvCas9 to the targeted mRNA.
           <H2 class="contentPage">Project Overview</H2>
         Genomic stability and DNA sequence fidelity are critical for normal cellular activities. If such storage
          center of genetic information is disrupted, detrimental consequences will take place, including but not
          limited to excessive cell death, abnormal proliferation, untimely cell senescence, etc. Therefore, a gene
          mutation surveillance system that can sensitively and efficiently detect and remove such aberrations
         <I>in vivo</I>is required.
         <I>Saccharomyces cerevisiae</I> is chosen as the model organism for designing such a gene surveillance system.
          As one of the most commonly used tools, it not only has a highly manipulatable genome, rapid growth
          speed, minimal pathogenicity, an ensemble of selectable markers, a well-defined genetic system, and most
          importantly, a highly versatile DNA transformation system.
         Essentially by designing such a system, a tool that can recognize aberrant mutation on mRNA sequences, or
          more generally at the transcriptomics level, is desired, because transcripts are derived from genomic
          sequences and hence are representatives of genomic instability and mutations. Surveillance Cas9 (suvCas9),
          termed for the first time by our team, is a variant of Cas9 without DNA cleavage activity and can target
          specific mRNA sequences by manually designed sgRNA and PAM sequences.
          That being said, after mutation is detected on mRNAs, how can our system respond? In a nutshell, when
          mutations occur, the surveillance system will detect and trigger the suicidal system. The surveillance
          system can be further quantitatively optimized to maximize its sensitivity and minimize potential side
          effects.</DIV></DIV><DIV class="contentPage2"><DIV class="contentPage" style="margin-left:200px;margin-top:20px;width:1130px;"><H1 class="contentPage" id="tag_design">design</H1>This project aims to establish a well-controlled gene mutation surveillance system to monitor
            possible harmful gene mutations. When mutations occur, the surveillance system will detect and
            trigger the suicidal system. The surveillance system can be further quantitatively optimized to
           maximize its sensitivity and minimize potential side effects.
           <H2 class="contentPage">RNA targeting by CRISPR/Cas9 system</H2>
           To achieve the surveilling functions of the whole circuit, a suvCas9 is facilitated by an sgRNA
            and a deoxynucleotide oligonucleotide (PAMmer) that hybridizes to the target RNA, providing
            the PAM sequence<SUP>[12]</SUP>. suvCas9 plays an indispensable part in the coupling of mRNA recognition
            system and the suicidal gene expression. suvCas9 performs high-fidelity mRNA binding activity,
           of which the specificity is double-determined by sgRNA and PAMmer. This surveillance system is
            composed of the dCas9 protein, sgRNA and PAMmer, forming a complex that can stably exist in the
            cytoplasm and tightly bind to its targeted mRNA. When a mutation of the genes under surveillance
            is detected by suvCas9, it can immediately trigger the suicidal system to eliminate the mutated
            cell, avoiding further catastrophic consequences.
           <H2 class="contentPage"><I>In vivo</I> generation of ssDNA by SCRIBE system</H2>
           In order to target mRNA in the cytoplasm, several concerns should be noticed<SUP>[7]</SUP>. First, the PAM
            motif, essentially an ssDNA in our project, should be provided together with sgRNA pairs, the
            presence of which are prerequisites for suvCas9 to acquire its activity. Second, the genomic
            counterpart of transcripts should be prevented from being recognized and disrupted by suvCas9,
           otherwise the surveillance system will contradict the purpose of maintaining high genomic fidelity.
           The newly developed RNA targeting system<SUP>[7]</SUP> meets both criteria mentioned above: it creates the PAM motif
            by introducing a special PAMmer sequence which is a mixed DNA and 2’-O-methyl RNA oligonucleotide,
            and it avoids any unwanted targeting in the nucleus. However, the synthesis of such sequence is not
            convenient, and the application in yeasts has not been tested before, not to mention such single-stranded
            nucleotides are hard to be generated and replicated <I>in vivo</I>.
           The limitations of direct introduction of PAMmer sequence into yeast create a niche for us to find
            alternative solutions: can we instead generate ssDNA within yeasts, as an integrated part of their
            metabolic activities? It turns out that generating ssDNA in yeast cells is viable. SCRIBE (Synthetic
            Cellular Recorders Integrating Biological Events)<SUP>[8]</SUP>, intrinsically an <I>in vivo</I> reverse transcription
            system, can reversely transcribe ssDNA with plasmids as the template<SUP>[9]</SUP>. More importantly, the RNA
            templates for reverse transcription are highly modularized, enabling effortless ssDNA switching and
            thus the flexibility of mRNA targeting. In practice, the design of molecular cloning is relatively
            convenient as well: a retron and an RNA template of the wanted ssDNA sequence can be assembled on the
            same plasmid, enabling a one-step transformation in yeasts.
           <H2 class="contentPage">Coupling with cellular suicidal system</H2>
           Mutations in the targeted gene will cause instant translocation of suvCas9 complex, switching on the
            suicidal gene expression to kill the mutated cell. Here we design to fuse dCas9 with the GAL4 binding
            domain and its activation domain on the foundation of yeast one-hybrid system<SUP>[13]</SUP> that allows
           the interaction of DNA-binding domain and the isolation of the targeted promoter and the downstream
            gene. With the engineered <I>Saccharomyces cerevisiae</I> background strain<SUP>[13]</SUP> for one-hybrid system, the
            interaction between the DNA-binding domain and the promoter containing recognition site will drive the
            transcription of Thymine Kinase (TK) and RFP. Thymine Kinase<SUP>[14]</SUP> is a novel selection marker in yeast,
           which is analogous to the widely used URA3 auxotrophy marker<SUP>[15]</SUP>, allowing both selection and
            counter-selection in respective media. In the selection media, which is YP-glycerol with antifolates,
            only yeast populations that produce thymine kinase can survive, while on the counter-selection media,
            which is SC media with FUdR, only the absence of TK can ensure the viability of yeast. Mutations
            in targeted genes will cause sequence changes in mRNA, which significantly decrease the binding
            activity of suvCas9 onto mRNA. The previously sequestered suvCas9 complex will be transported into
            nuclear immediately in the guidance of GAL4 binding domain and the exogenous Nuclear Localization
            Sequence (NLS), and turn on the transcription of TK and RFP gene, which in turn cause the death of
            cell on the counter-selection media and the fluorescence expression that can be quantitatively detected
           by FACS for further optimization of our surveillance system.Figure 1A. A schematic overview of the targeting mechanism of suvCas9. suvCas9 can bind to a correct
            mRNA in complex with sgRNA and PAMmer (illustrated above), which can then be translocated into the
            nucleus and initiate a suicidal program whenever the mRNA is mutated (not shown). Specially designed
            PAMmers will not disrupt the genomic sequence.
           Figure 1B. Mechanism of the generation of ssDNA <I>in vivo</I>. A single strand DNA can be derived
            from the reversely transcribed product by the retron (RT). Figures are adapted from (7) and (8).
           <H2 class="contentPage">Reference</H2><DIV id="reference" style="display:none;">
             [1] Hanahan, D., &amp; Weinberg, R. A. (2011). Hallmarks of cancer: the next generation. Cell, 144(5), 646-674.
             [2] Filipovska, A., Razif, M. F., Nygård, K. K., &amp; Rackham, O. (2011). A universal code for RNA recognition
              by PUF proteins. Nature chemical biology, 7(7), 425-427.
             [3] Ozawa, T., Natori, Y., Sato, M., and Umezawa, Y. (2007). Imaging dynamics of endogenous mitochondrial
              RNA in single living cells. Nat. Methods 4, 413–419.
             [4] Wang, Y., Cheong, C.G., Hall, T.M., and Wang, Z. (2009). Engineering splicing factors with designed
              specificities. Nat. Methods 6, 825–830.
             [5] Sokol, D.L., Zhang, X., Lu, P., and Gewirtz, A.M. (1998). Real time detection of DNA.RNA hybridization
              in living cells. Proc. Natl. Acad. Sci. USA 95, 11538–11543.
             [6] Fouts, D.E., True, H.L., and Celander, D.W. (1997). Functional recognition of fragmented operator sites
              by R17/MS2 coat protein, a translational repressor. Nucleic Acids Res. 25, 4464–4473.[7] Nelles, D. A., Fang, M. Y., O’Connell, M. R., Xu, J. L., Markmiller, S. J., Doudna, J. A., &amp; Yeo,
              G. W. (2016). Programmable RNA Tracking in Live Cells with CRISPR/Cas9. Cell, 165(2), 488-496.[8] Farzadfard, F., &amp; Lu, T. K. (2014). Genomically encoded analog memory with precise in vivo DNA writing
              in living cell populations. Science, 346(6211), 1256272.
             [9] Miyata, S., Ohshima, A., Inouye, S., &amp; Inouye, M. (1992). In vivo production of a stable single-stranded
              cDNA in Saccharomyces cerevisiae by means of a bacterial retron. Proceedings of the National Academy
              of Sciences, 89(13), 5735-5739.
             [10] Ran, F. A., Cong, L., Yan, W. X., Scott, D. A., Gootenberg, J. S., Kriz, A. J., ... &amp; Koonin, E. V.
              (2015). In vivo genome editing using Staphylococcus aureus Cas9. Nature, 520(7546), 186-191.
             [11] Sherman, F. (1991). [1] Getting started with yeast. Methods in enzymology, 194, 3-21.[12] O’Connell, M.R., Oakes, B.L., Sternberg, S.H., East-Seletsky, A., Kaplan, M., and Doudna, J.A. (2014).
              Programmable RNA recognition and cleavage by CRISPR/Cas9. Nature 516, 263–266. [13] Vidal, M., Brachmann, R. K., Fattaey, A., Harlow, E., &amp; Boeke, J. D. (1996). Reverse two-hybrid and
              one-hybrid systems to detect dissociation of protein-protein and DNA-protein interactions. Proceedings
              of the National Academy of Sciences, 93(19), 10315-10320.
             [14] Alexander, W. G., Doering, D. T., &amp; Hittinger, C. T. (2014). High-efficiency genome editing and allele
              replacement in prototrophic and wild strains of Saccharomyces. Genetics, 198(3), 859-866.
             [15] Boeke, J. D., La Croute, F., &amp; Fink, G. R. (1984). A positive selection for mutants lacking
              orotidine-5′-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance. Molecular
              and General Genetics MGG, 197(2), 345-346.
           </DIV></DIV></DIV><DIV class="contentPage2"><DIV class="contentPage" style="margin-left:200px;margin-top:20px;width:1130px;"><H1 class="contentPage" id="tag_results">Results</H1><H2 class="contentPage"><I>In vivo</I> experiment of dCas9 localization guided by sgRNA in <I>S. cerevisiae</I></H2>
           In order to achieve the surveillance function of the whole circuitry, it is of great significance to indicate
            the precise spatial distribution of suvCas9. With suvCas9 fused with GFP, we can easily detect the cellular
            localization of suvCas9 by fluorescent microscopy. We observed perfect nucleus localization of suvCas9
           without the presence of sgRNA or PAMmer. However, after the co-transformation of actin transcript targeted
            sgRNA expression plasmid and the electroporation of PAMmer, a certain population of yeast cells showed
            obvious appearance of cytoplasmic suvCas9 complex, which implied that the specificity of suvCas9 is
            determined by sgRNA as well as PAMmer. More importantly, suvCas9 protein, sgRNA and PAMmer form a
            complex that can be trapped into the cytoplasm by sgRNA-targeted mRNA.
           <H2 class="contentPage">Yeast one-hybrid system for suicide gene activation (URA3)</H2>
           To test if the GAL4 binding domain and its activation domain fused dCas9 can actually activate
            downstream gene expression. The background yeast strain we used contains SPAL:URA3 engineered
            into S. cerevisiae genome. Interaction between suvCas9 protein complex and SPAL promoter will drive
           the expression of URA3, therefore ensure the viability on autotrophy selection plates. The results
            indicated that successful expression of as well as the nucleus localization of suvCas9 protein complex,
            would guarantee downstream gene activation based on yeast one-hybrid system, which is the premise of
            suicidal gene activation once the potential mutation occurs and suvCas9 recover its nuclear targeting
            capability.
           <H2 class="contentPage">Validation of TK lethality and viability in different culture media</H2>
           Thymine Kinase (TK), as a novel selection marker in yeast, allows both selection and counter-selection
            in respective media. We validated the lethality and viability in different culture media of TK expressing
            strains. In YP-glycerol with antifolates, the selection media, yeast populations that produce TK can
            survive, while on the counter-selection media, SC media with FUdR, TK would cause exclusive lethality
            in those populations. Therefore, TK can act as a candidate downstream suicidal gene to be activated
            in the mutated cells.
           </DIV></DIV><DIV class="contentPage2"><DIV class="contentPage" style="margin-left:200px;margin-top:20px;width:1130px;"><H1 class="contentPage" id="tag_proofOfConcepts">proof of concepts</H1><H2 class="contentPage">suvCas9 relocalization</H2>
           In the presence of suvCas9s alone, GFP signal (indicating suvCas9) can only be observed in the nucleus,
            indicating an enrichment of nuclear localization of suvCas9s. However, when small guide RNA
            targeting the Actin message (sgActin) is co-expressed with suvCas9, GFP signal is relocated into
            the cytoplasm. In contrast, after introducing PAMmers, which function as single-stranded DNA mimics,
            the majority of suvCas9 proteins can now be stably sequestered in the cytoplasm.
           Figure 1. Relocation into the cytoplasm of suvCas9 proteins can be orchestrated by single guide
            RNAs targeting the Actin message (sgActin) and PAMmers.

           
           As negative controls, when suvCas9 is expressed alone, or co-expressed with small guide RNA targeting
            anti-sense message Actin mRNA sequence (sgNC), the GFP signal (indicating suvCas9) are strictly sequestered
            in the nucleus, suggesting without the companion of sgRNA targeting sense Actin message, suvCas9 will be
            translocated into the nucleus by its engineered NLS (nucleus localization sequence). As a positive control,
            when yeasts are only transformed with a GFP expression construct, without the guidance of the NLS, the GFP
           signal is limited within the cytoplasmic region.
           Figure 2. Negative control and positive control for the relocation assay.

      </DIV></DIV><DIV class="contentPage2"><DIV class="contentPage" style="margin-left:200px;margin-top:20px;width:1130px;"><H1 class="contentPage" id="tag_notebooks">notebooks</H1><H2 class="contentPage">akljfd;hskgslkghkldsghaklghadskgl;had;gklhadklghadlgh</H2>
           adsgejiha;oriej'ekEy;hstejaglh`kdhgfjkhl/af`djsfghuqi;eofLKJDJH;I
           ak;ldsghasgushyghfjdaygsuhigjhkodsrtfyguhjgyftdrseadtfyugrihtfoypuoyhis
           afhdjkgh
           akljfd;hskgslkghkldsghaklghadskgl;had;gklhadklghadlgh
           ak;ldsghasgushyghfjdaygsuhigjhkodsrtfyguhjgyftdrseadtfyugrihtfoypuoyhisg
           afhdjkgh</DIV></DIV><DIV class="contentPage2"><DIV class="contentPage" style="margin-left:200px;margin-top:20px;width:1130px;"><H1 class="contentPage" id="tag_safety">Safety</H1><H2 class="contentPage">Overview</H2>
           This year, Team Tsinghua proposed a CRISPR/Cas9-based gene surveillance system and
            attempted to realize such design in baking yeasts. Even though we have deliberately
            chosen E. coli and S. cerevisiae as chassis organisms, of which biosafety levels are
           well recognized and the protocol for experimental manipulation is well established, there
            could have been unsolicited consequences if no attention for safe experiments is heeded.
           Therefore, in order to minimize any unwanted accident, we have received proper training
           and adopted several safety regulations in the laboratory as follows.
           <H2 class="contentPage">Lab safety training</H2>
           First, before we started our project, all of our team members had received strict lab
            safety training, including standard experimental protocols, proper disposal of biological
            and chemical waste, information on hazardous chemicals and use of biosafety cabinets etc
           from two senior members Liu Yan and Qin Yiran in Prof. Dai’s laboratory. Such guidance were
            documented on the online portal in detail. Team members involved in wet works all went through
            orientations with mentors or managers specifying proper equipment usage and safety before
           gaining access to that lab.
           <H2 class="contentPage">Appropriate protection</H2>
           Second, risks to team members brought about by potentially hazardous chemicals were minimized
            by wearing appropriate personal protective equipments (PPEs) throughout lab time and
            following lab safety regulations of Tsinghua University. Gloves, lab coats and close-toed
            shoes are required when working with chassis organisms. Hazardous chemicals such as ethidium
     bromide (EB) were carefully used and disposed according to protocols. No food or drink is
            allowed in the working area.
           <H2 class="contentPage">chassis organism management</H2>
           Third, E. coli strains that are used by our project (DH5α and BL21) as well as the yeast strain
            are harmless and commonly accepted as safe organisms to work with. Our modified chassis
            organism will be properly used and disposed according to the safety manual of the lab and
            will not be released to the natural environment, a crucial principal of Prof Dai’s laboratory.
            We amplified all plasmids to be used in yeast-related experiments in E. coli.
           <H2 class="contentPage">Safety in future</H2>
           Fourth, in terms of future application, we also have a clear vision: we want to test our system
            in yeasts as a proof-of-concept, and if such system is sensitive and safe enough, we want
            to advance its application in mammalian cell lines. In the future, such system can be applied
            in gene therapy to treat diseases like cancer with the help of viral vectors. Since we want to
            combine the viral vector with our surveillance system, any potential risk that a viral vector
            may bring in is worth consideration with scrutiny, such as unwanted genomic insertion of viral
            vectors. Therefore, in order to achieve our goal, we will first test the efficacy and safety level
            of the viral vector used in cell lines and then in mice. Also, we hope to introduce a well-controlled
            molecular switch that only allows for bacterial and yeast growth in an artificial setting, therefore
            to prevent any accidental release of the organisms we use.
           </DIV></DIV></FORM></DIV></DIV></DIV></DIV></DIV></BODY></HTML>