<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tsinghua_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tsinghua/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Project</TITLE></P><FORM method="post" action="/" id="MainForm"><DIV id="AngularPanel" class="main-wrapper" ng-app="fivb"><DIV id="galaxy-bar" class="gb-styling" style="margin:15px 0px 0px 0px;"><DIV class="navigation"><DIV class="hamburger-submenu"><UL><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_0" href="https://2016.igem.org/Team:Tsinghua">Home</A></SPAN></LI><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_1" href="https://2016.igem.org/Team:Tsinghua/Team">Team</A></SPAN></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_1_Link_0" href="https://2016.igem.org/Team:Tsinghua/Team">Team members</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_1_Link_1" href="https://2016.igem.org/Team:Tsinghua/Attributions">Attributions</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_1_Link_2" href="https://2016.igem.org/Team:Tsinghua/Acknowledgement">Acknowledgement</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_1_Link_3" href="https://2016.igem.org/Team:Tsinghua/Acknowledgement#tag_sponsorship">Sponsorship</A></LI></UL><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_2" href="https://2016.igem.org/Team:Tsinghua/Description">Project</A></SPAN></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_0" href="https://2016.igem.org/Team:Tsinghua/Description">Description</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_1" href="https://2016.igem.org/Team:Tsinghua/Experiments">Design</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_2" href="https://2016.igem.org/Team:Tsinghua/Results">Results</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_3" href="https://2016.igem.org/Team:Tsinghua/Proof">Proof of Concepts</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_4" href="https://2016.igem.org/Team:Tsinghua/Notebook">Notebook</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_5" href="https://2016.igem.org/Team:Tsinghua/Model">Modeling</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_2_Link_5" href="https://2016.igem.org/Team:Tsinghua/Safety">Safety</A></LI><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_3" href="https://2016.igem.org/Team:Tsinghua/Parts">Parts</A></SPAN></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_0" href="https://2016.igem.org/Team:Tsinghua/Parts#tag_summary">Parts Summary</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_1" href="https://2016.igem.org/Team:Tsinghua/Basic_Part#tag_basic">Basic Parts</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_2" href="https://2016.igem.org/Team:Tsinghua/Composite_Part">Composite Parts</A></LI><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_3" href="https://2016.igem.org/Team:Tsinghua/Human_Practices">Human Practices</A></SPAN></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_0" href="https://2016.igem.org/Team:Tsinghua/Human_Practices">Overview</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_1" href="https://2016.igem.org/Team:Tsinghua/Human_Practices#tag_engagement">Public Engagement</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_0" href="https://2016.igem.org/Team:Tsinghua/Collaborations">Collaboration</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_1" href="https://2016.igem.org/Team:Tsinghua/Community_meetup">Community Meetup</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_3_Link_2" href="https://2016.igem.org/Team:Tsinghua/Integrated_Practices">Integrated Practices</A></LI><LI class="lvl1"><SPAN class="header"><A id="Galaxy1_Hamburger1_Burgers_Link_4" href="https://2016.igem.org/Team:Tsinghua/MedalChecklist">Medal Checklist</A></SPAN></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_4_Link_0" href="https://2016.igem.org/Team:Tsinghua/MedalChecklist#bronze">Bronze</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_4_Link_1" href="https://2016.igem.org/Team:Tsinghua/MedalChecklist#silver">Silver</A></LI><LI><A id="Galaxy1_Hamburger1_Burgers_Children_4_Link_2" href="https://2016.igem.org/Team:Tsinghua/MedalChecklist#gold">Gold</A></LI></DIV><DIV class="galaxy-main-navigation"><NAV><UL class="quicklinks"><LI><A href="https://2016.igem.org/Team:Tsinghua">Home</A></LI><LI><A href="https://2016.igem.org/Team:Tsinghua/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Tsinghua/Description">Project</A></LI><LI><A href="https://2016.igem.org/Team:Tsinghua/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Tsinghua/Human_Practices">Human practice</A></LI></UL></NAV></DIV></DIV></DIV></DIV><DIV class="subnavigator" id="subnavigator" style="position:absolute;left:30px;top:330px;"><A href="https://2016.igem.org/Team:Tsinghua/Description">Description</A><A href="https://2016.igem.org/Team:Tsinghua/Experiments">Design</A><A href="https://2016.igem.org/Team:Tsinghua/Results">Results</A><A href="https://2016.igem.org/Team:Tsinghua/Proof">Proof of concepts</A><A href="https://2016.igem.org/Team:Tsinghua/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Tsinghua/Model" target="_blank">Modeling</A><A href="https://2016.igem.org/Team:Tsinghua/Safety">Safety</A></DIV><DIV class="contentPage2"><DIV class="contentPage" style="margin-left:200px;margin-top:20px;width:1130px;"><H1 class="contentPage" id="tag_description">description</H1><H2 class="contentPage">Background</H2>
             The fidelity of genomic sequence is the underpinning aspect of normal cellular activities. Even a minor
             disruption to DNA, the storage center of genetic information, can lead to detrimental consequences,
             including excessive cell death, abnormal proliferation, untimely cell senescence, etc. For example,
             genomic instability and mutation is considered as one of the ten hallmarks of cancer, the occurrence
             of which generally leads to over-activation of signaling pathways, in turn triggering malignant
              proliferation of cells<SUP>[1]</SUP>. Recurrent hot-spot DNA mutations and chromosomal rearrangements are
             commonly observed in cancer samples. Genomic mutation is therefore a primary concern in terms of both
             basic research and clinical treatment, demanding multi-disciplinary efforts.
             In order to address this imperative issue, a gene mutation surveillance system that can sensitively and
             efficiently detect and remove such aberrations to achieve real-time monitoring <I>in vivo</I> is required.
             Essentially by designing such a system, a tool that can recognize aberrant mutation on mRNA sequences,
              or more generally at the transcriptomics level, is desired, because transcripts are derived from
             genomic sequences and hence are representatives of genomic instability and mutations. More importantly,
              their availability and abundance in the cytoplasm allow them to be targeted. Nonetheless, few such
              satisfying molecular tools have been developed yet. Consequently, to design such a surveillance
             system is not only important but also necessary.
             To serve such purpose, an RNA binding factor with high efficiency and sensitivity is a must, which
              is able to distinguish RNA transcripts produced by mutated genes and switch on the suicidal gene
              expression afterwards. There are three major RNA targeting methods currently: PUF proteins<SUP>[2][3][4]</SUP>,
             molecular beacons<SUP>[5]</SUP> and RNA aptamers<SUP>[6]</SUP>. Due to lack of specificity or stability in vivo, those existing
              methods are not suitable for the potential application in our gene mutation surveillance system.
              However, it is newly reported that dCas9<SUP>[7]</SUP>, capable of recognizing specific endogenous mRNA
             sequences, turns out to be an ideal candidate. Surveillance Cas9 (suvCas9), termed for the first time
              by our team, is a variant of Cas9 without DNA cleavage activity and can target specific mRNA sequences
              by manually designed sgRNA and PAM sequences. With this remarkable ability to target mRNA, suvCas9
              can hopefully be adopted into our surveillance system as the monitor as well as the executor
             We propose to construct suvCas9 system in <I>Saccharomyces cerevisiae</I> to function as the monitor
              of genomic sequence. <I>Saccharomyces cerevisiae</I> is one of the most commonly used model organisms,
              representing a simple eukaryote with a highly manipulatable genome. Apart from many physical properties
              and technical advantages, such as rapid growth, minimal pathogenicity, the ease of phenotype identification,
              a well-defined genetic system, and most importantly, a highly versatile DNA transformation system<SUP>[11]</SUP>,
              the yeast has been reported to share a similar characteristics of <I>in vivo</I> ssDNA generation by reverse
              transcriptase with prokaryotes<SUP>[9]</SUP>, which will guide suvCas9 to the targeted mRNA.
           <H2 class="contentPage">Project Overview</H2><VIDEO style="width:768px;height:432px;margin-left:60px" controls="">
         Genomic stability and DNA sequence fidelity are critical for normal cellular activities. If such storage
          center of genetic information is disrupted, detrimental consequences will take place, including but not
          limited to excessive cell death, abnormal proliferation, untimely cell senescence, etc. Therefore, a gene
          mutation surveillance system that can sensitively and efficiently detect and remove such aberrations
         <I>in vivo</I>is required.
         <I>Saccharomyces cerevisiae</I> is chosen as the model organism for designing such a gene surveillance system.
          As one of the most commonly used tools, it not only has a highly manipulatable genome, rapid growth
          speed, minimal pathogenicity, an ensemble of selectable markers, a well-defined genetic system, and most
          importantly, a highly versatile DNA transformation system.
         Essentially by designing such a system, a tool that can recognize aberrant mutation on mRNA sequences, or
          more generally at the transcriptomics level, is desired, because transcripts are derived from genomic
          sequences and hence are representatives of genomic instability and mutations. Surveillance Cas9 (suvCas9),
          termed for the first time by our team, is a variant of Cas9 without DNA cleavage activity and can target
          specific mRNA sequences by manually designed sgRNA and PAM sequences.
          That being said, after mutation is detected on mRNAs, how can our system respond? In a nutshell, when
          mutations occur, the surveillance system will detect and trigger the suicidal system. The surveillance
          system can be further quantitatively optimized to maximize its sensitivity and minimize potential side
          effects.
         <H2 class="contentPage">Contribution</H2>
         This year we've also characterized and improved several parts, including the following three:
          BBa_K1323002, BBa_K1493504, BBa_K1470002. Please refer to their Registry pages for detailed
          documentation.

       </VIDEO></DIV></DIV><DIV class="contentPage2"><DIV class="contentPage" style="margin-left:200px;margin-top:20px;width:1130px;"><H2 class="contentPage">Reference</H2><DIV id="reference" style="display:none;">
             [1] Hanahan, D., &amp; Weinberg, R. A. (2011). Hallmarks of cancer: the next generation. Cell, 144(5), 646-674.
             [2] Filipovska, A., Razif, M. F., Nygård, K. K., &amp; Rackham, O. (2011). A universal code for RNA recognition
              by PUF proteins. Nature chemical biology, 7(7), 425-427.
             [3] Ozawa, T., Natori, Y., Sato, M., and Umezawa, Y. (2007). Imaging dynamics of endogenous mitochondrial
              RNA in single living cells. Nat. Methods 4, 413–419.
             [4] Wang, Y., Cheong, C.G., Hall, T.M., and Wang, Z. (2009). Engineering splicing factors with designed
              specificities. Nat. Methods 6, 825–830.
             [5] Sokol, D.L., Zhang, X., Lu, P., and Gewirtz, A.M. (1998). Real time detection of DNA.RNA hybridization
              in living cells. Proc. Natl. Acad. Sci. USA 95, 11538–11543.
             [6] Fouts, D.E., True, H.L., and Celander, D.W. (1997). Functional recognition of fragmented operator sites
              by R17/MS2 coat protein, a translational repressor. Nucleic Acids Res. 25, 4464–4473.[7] Nelles, D. A., Fang, M. Y., O’Connell, M. R., Xu, J. L., Markmiller, S. J., Doudna, J. A., &amp; Yeo,
              G. W. (2016). Programmable RNA Tracking in Live Cells with CRISPR/Cas9. Cell, 165(2), 488-496.[8] Farzadfard, F., &amp; Lu, T. K. (2014). Genomically encoded analog memory with precise in vivo DNA writing
              in living cell populations. Science, 346(6211), 1256272.
             [9] Miyata, S., Ohshima, A., Inouye, S., &amp; Inouye, M. (1992). In vivo production of a stable single-stranded
              cDNA in Saccharomyces cerevisiae by means of a bacterial retron. Proceedings of the National Academy
              of Sciences, 89(13), 5735-5739.
             [10] Ran, F. A., Cong, L., Yan, W. X., Scott, D. A., Gootenberg, J. S., Kriz, A. J., ... &amp; Koonin, E. V.
              (2015). In vivo genome editing using Staphylococcus aureus Cas9. Nature, 520(7546), 186-191.
             [11] Sherman, F. (1991). [1] Getting started with yeast. Methods in enzymology, 194, 3-21.[12] O’Connell, M.R., Oakes, B.L., Sternberg, S.H., East-Seletsky, A., Kaplan, M., and Doudna, J.A. (2014).
              Programmable RNA recognition and cleavage by CRISPR/Cas9. Nature 516, 263–266. [13] Vidal, M., Brachmann, R. K., Fattaey, A., Harlow, E., &amp; Boeke, J. D. (1996). Reverse two-hybrid and
              one-hybrid systems to detect dissociation of protein-protein and DNA-protein interactions. Proceedings
              of the National Academy of Sciences, 93(19), 10315-10320.
             [14] Alexander, W. G., Doering, D. T., &amp; Hittinger, C. T. (2014). High-efficiency genome editing and allele
              replacement in prototrophic and wild strains of Saccharomyces. Genetics, 198(3), 859-866.
             [15] Boeke, J. D., La Croute, F., &amp; Fink, G. R. (1984). A positive selection for mutants lacking
              orotidine-5′-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance. Molecular
              and General Genetics MGG, 197(2), 345-346.
           </DIV></DIV></DIV><FOOTER class="main-footer"><DIV class="visual"><DIV class="copy"><DIV class="grid grid-pad"><DIV class="col-1-1" style="margin-left:17px"><SPAN>2016 iGEM Tsinghua - All rights reserved</SPAN><SPAN>Adapted from Bootstrap and FIVB</SPAN><SPAN>Follow us: igem2016_tsinghua@126.com</SPAN></DIV></DIV></DIV></DIV></FOOTER></FORM></DIV></DIV></DIV></DIV></DIV></BODY></HTML>