Team:Tsinghua/Proof
Project
Home
Team
Team members
Attributions
Acknowledgement
Sponsorship
Project
Description
Design
Results
Proof of Concepts
Notebook
Modeling
Safety
Parts
Parts Summary
Basic Parts
Composite Parts
Human Practices
Overview
Public Engagement
Collaboration
Community Meetup
Integrated Practices
Medal Checklist
Bronze
Silver
Gold
Home
Team
Project
Parts
Human practice
Description
Design
Results
Proof of concepts
Notebook
Modeling
Safety
Results
In vivo experiment of dCas9 localization guided by sgRNA in S. cerevisiae
In order to achieve the surveillance function of the whole circuitry, it is of great significance to indicate
the precise spatial distribution of suvCas9. With suvCas9 fused with GFP, we can easily detect the cellular
localization of suvCas9 by fluorescent microscopy. We observed perfect nucleus localization of suvCas9
without the presence of sgRNA or PAMmer. However, after the co-transformation of actin transcript targeted
sgRNA expression plasmid and the electroporation of PAMmer, a certain population of yeast cells showed
obvious appearance of cytoplasmic suvCas9 complex, which implied that the specificity of suvCas9 is
determined by sgRNA as well as PAMmer. More importantly, suvCas9 protein, sgRNA and PAMmer form a
complex that can be trapped into the cytoplasm by sgRNA-targeted mRNA.
Yeast one-hybrid system for suicide gene activation (URA3)
To test if the GAL4 binding domain and its activation domain fused dCas9 can actually activate
downstream gene expression. The background yeast strain we used contains SPAL:URA3 engineered
into S. cerevisiae genome. Interaction between suvCas9 protein complex and SPAL promoter will drive
the expression of URA3, therefore ensure the viability on autotrophy selection plates. The results
indicated that successful expression of as well as the nucleus localization of suvCas9 protein complex,
would guarantee downstream gene activation based on yeast one-hybrid system, which is the premise of
suicidal gene activation once the potential mutation occurs and suvCas9 recover its nuclear targeting
capability.
Validation of TK lethality and viability in different culture media
Thymine Kinase (TK), as a novel selection marker in yeast, allows both selection and counter-selection
in respective media. We validated the lethality and viability in different culture media of TK expressing
strains. In YP-glycerol with antifolates, the selection media, yeast populations that produce TK can
survive, while on the counter-selection media, SC media with FUdR, TK would cause exclusive lethality
in those populations. Therefore, TK can act as a candidate downstream suicidal gene to be activated
in the mutated cells.
proof of concepts
suvCas9 relocalization
In the presence of suvCas9s alone, GFP signal (indicating suvCas9) can only be observed in the nucleus,
indicating an enrichment of nuclear localization of suvCas9s. However, when small guide RNA
targeting the Actin message (sgActin) is co-expressed with suvCas9, GFP signal is relocated into
the cytoplasm. In contrast, after introducing PAMmers, which function as single-stranded DNA mimics,
the majority of suvCas9 proteins can now be stably sequestered in the cytoplasm.
Figure 1. Relocation into the cytoplasm of suvCas9 proteins can be orchestrated by single guide
RNAs targeting the Actin message (sgActin) and PAMmers.
As negative controls, when suvCas9 is expressed alone, or co-expressed with small guide RNA targeting
anti-sense message Actin mRNA sequence (sgNC), the GFP signal (indicating suvCas9) are strictly sequestered
in the nucleus, suggesting without the companion of sgRNA targeting sense Actin message, suvCas9 will be
translocated into the nucleus by its engineered NLS (nucleus localization sequence). As a positive control,
when yeasts are only transformed with a GFP expression construct, without the guidance of the NLS, the GFP
signal is limited within the cytoplasmic region.
Figure 2. Negative control and positive control for the relocation assay.
2016 iGEM Tsinghua - All rights reserved Adapted from Bootstrap and FIVB Follow us: igem2016_tsinghua@126.com
