"
Page
Discussion
View source
History
teams
Log in
 
Team:Arizona State/Project/Future
From 2011.igem.org
A Vision for the Future
Home iGEM 2011 Home Project
Introduction
CRISPR
E. coli
B. halodurans
L. innocua
Software
Future
References
Lab
Team
Photos
Protocols
Safety
Acknowledgements
Results
Data
BioBricks
Notebook
April/May
June
July
August
September
Cas PCR Log
Sequence Information
Other Documents
Human Practices
Events
Collaborations
Exploring Synthetic Biology
Outreach in Practice
CRISPR: Harnessing its Unlimited Potential
CRISPR, a relatively new scientific discovery, has the ability to revolutionize modern biotechnology techniques and pave a new direction in the field of synthetic biology. By taking advantage of CRISPR's potential to become the prokaryotic analogue of RNAi, we aim to develop a new standard for controlled prokaryotic gene expression using an artificial CRISPR design and construction methodology. In doing so, the applications and future innovations in the CRISPR-Cas system and its manipulation are ever-expanding. Our project takes the first step in developing an innovative modular platform that would standardize the way that researchers control gene transfer and expression. The possibilities are endless.
Present Applications of CRISPR-Cas Systems
Protection of Domesticated Bacteria
From dairy to biotechnology, domesticated bacteria are used in a variety of industries: T. thermophilis, the microbe responsible for yoghurts and cheeses, is responsible for producing $40 billion in product per year alone [77] . Unfortunately, much time and money is wasted every year when these highly specialized and valued bacteria cultures are decimated by phage attack. The Crispr-Cas system’s natural role as defense against foreign DNA has already been exploited by engineering Crispr arrays that have conferred phage-resistance in its host (A patent has been filed by Danisco relating to the commercial use of CRISPR phage protection systems). Better characterized and varied Crispr-cas systems will provide additional tools by which domesticated bacterial cultures can be protected against the bacteriophages.
Future Applications of CRISPR-Cas Systems
Combinatoric Gene Silencing
Crispr-Cas systems have been found to target both exogenous DNA as well as RNA. If spacers from a particular type of Crispr-Cas system can be found to successfully target RNA, the system could be utilized as a novel platform for gene silencing by the use of arrays integrating synthesized spacers complement to target RNA. The repeated structure of Crispr arrays lends itself to the targeting of multiple genes at the same time as well as multiple targeting of the same genes. Varying the combination and multiplicity of each of the genes in synthetic Crispr arrays could lead to better production optimization and inform modeling of entire synthetic pathways. Currently, ribosomal binding sites must be laboriously introduced to vary protein expression levels [78] . Crispr-based systems could provide an alternative for genetic tuning in the future.
Modified Phage-assisted Continuous Evolution (PACE) with the CRISPR-Cas System
Similar to the previous application, scenarios can be envisioned where Crispr-Cas systems are used to modify existing genes. Phage-assisted continuous evolution (known as PACE) is a method that has been used to continuously evolve biomolecules with unique characteristics, such DNA binding proteins that recognize unique sequences [79] . Briefly, PACE works when a particular gene of interest is placed in a bacteriophage genome that has a protein necessary for replication, p III, deleted from it. The p III gene is placed on an accessory plasmid in the bacterial cells where it is directly linked to the expression of the gene of interest. This causes only phage vectors with sufficient expression levels of the gene of interest propagate onward. Mutations can build up in the gene of interest after many cycles of evolution, shifting the function of the protein. A Crispr-Cas system could be introduced into the host bacterial cells such that spacers are made that target the entire length of the gene of interest. The bacteria would be resistant to all phages that do not mutate in the targeted gene of interest. In effect, the introduction of the Crispr-Cas system to a PACE evolutionary set-up would selectively induce mutations in the gene of interest, thus decreasing the total number of rounds necessary to perform (or increase the variation per round). The use of the Crispr-Cas system in a PACE evolutionary set-up could also help elucidate the population-level dynamics of the Crispr-Cas system as protection by certain spacers spreads among surviving bacteria.
In-vitro CRISPR Studies
Some aspects of the Crispr-Cas system could be further studied by using in-vitro set-up. This can be done by tagging all of the relevant Cas proteins with tags for purification purposes. In type II Crispr-Cas systems, such as our Listeria innocua project, only the Cas9 protein needs to be tagged. Cas9 can be combined with guide RNA from the Crispr array in such a way as to function as a 30bp specific endonuclease. In-vitro studies could be vitally important in confirming whether engineered Crispr-Cas systems actually are adequate for functionality.
Contact Us
Arizona State University
ECG 334, PO BOX 9709
Tempe, Arizona 85287
Email
Retrieved from " http://2011.igem.org/Team:Arizona_State/Project/Future "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
