<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Arizona_State_Notebook_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Arizona_State/Notebook/July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Arizona_State/Notebook/July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Arizona_State/Notebook/July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Arizona_State/Notebook/July&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Arizona_State/Notebook/July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Arizona State/Notebook/July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV style="font-size:18pt;"><FONT face="Calibri" style="color:#000000"><CENTER>Notebook: July
</CENTER></FONT></DIV><DIV><TABLE cellspacing="0"><TBODY><TR><TD style="background-color: #ffffff" width="75px" valign="top"><DIV class="glossymenu"><A class="menuitem" href="https://2011.igem.org/Team:Arizona_State">Home</A><A class="menuitem" href="https://2011.igem.org/Main_Page">iGEM 2011 Home</A><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Project">Project</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/Introduction">Introduction</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/CRISPR">CRISPR</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/E_coli">E. coli</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/B_halodurans">B. halodurans</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/L_innocua">L. innocua</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/Future">Future</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/References">References</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Lab">Lab</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Acknowledgements">Acknowledgements</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Results">Results</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Results/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Parts">BioBricks</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Lab">Notebook</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/AprilMay">April/May</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/June">June</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/July">July</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/August">August</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/September">September</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/PCRLog">Cas PCR Log</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/Sequences">Sequence Information</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/Other">Other Documents</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Outreach">Human Practices</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Events">Events</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Collaborations">Collaborations</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Exploring_Synthetic_Biology">Exploring Synthetic Biology</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Outreach_in_Practice">Outreach in Practice</A></LI></UL></DIV></DIV></TD><TD width="880" valign="top" style="padding: 10px; border: 5px solid #ffffff; color: #000; background-color: white"><DIV style="position: relative; margin-top: -450px; min-height: 450px"><DIV> <H2><SPAN class="mw-headline" id="Friday.2C_July_1"> Friday, July 1 </SPAN></H2><UL><LI> Ran a gel of PCRed CAS genes from last night
</LI></UL><DL><DD><UL><LI> A1, B2, C3, D4, E5, F6, ladder 1; 10 uL of each sample placed in wells
</LI></UL></DD></DL><UL><LI> Conducted a ligation
</LI><LI> Transformation of NEB cells:
</LI></UL><DL><DD><UL><LI> DA, double plated on LB+Kan plates
</LI><LI> DB, double plated on LB+Kan plates
</LI><LI> Seq1, double plated on LB+Kan plates
</LI><LI> CMR, double plated on LB+Kan plates
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Saturday.2C_July_2"> Saturday, July 2 </SPAN></H2><UL><LI> Plates from yesterday: colonies visible on all plates except CMR
</LI></UL><DL><DD><UL><LI> Made overnight cultures of colonies
</LI></UL></DD></DL><UL><LI> Conducted ligation + transformation of RA, RB onto LB kan plates
</LI><LI> Ran gel of more pcr products
</LI></UL><DL><DD><UL><LI> Big gel: c1, c2, c3, c4, c5, a1, a2, a3
</LI><LI> Small gel: e1, e2, e3, e4, e5, e6, f1, f2, f3, f4, f5, f6
</LI></UL></DD><DD><UL><LI> No significant or usable bands visible on gels
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Tuesday.2C_July_5"> Tuesday, July 5 </SPAN></H2><UL><LI> Another PCR attempt at Cas genes, 2nd attempt with new primers:
</LI></UL><DL><DD><UL><LI> A, C: didn't work
</LI><LI> Attempted gel extraction
</LI></UL></DD></DL><UL><LI> Conducted miniprep
</LI><LI> Conducted restriction:
</LI></UL><DL><DD><UL><LI> Seq1, DA, RA (ES, EX) in PSB1A3
</LI></UL></DD></DL><UL><LI> Conducted ligation attempt to make tandem RSR, RS, and SR constructs:
</LI></UL><DL><DD><UL><LI> Seq1 + Seq1, DA + DA, RA + RA; all into pSB1K3
</LI><LI> Transformation of ligation products, plated on LB+Kan (NEB cells)
</LI></UL></DD></DL><UL><LI> Also, another conducted ligation:
</LI></UL><DL><DD><UL><LI> Seq1 (ES, XP) + PSB1A3 (EP)
</LI><LI> Transformation, plated on LB+Amp (NEB cells)
</LI></UL></DD></DL><UL><LI> Made glycerol stocks of Seq1, DA, DB, RA, RB in PSB1K3
</LI></UL><H2><SPAN class="mw-headline" id="Wednesday.2C_July_6"> Wednesday, July 6 </SPAN></H2><UL><LI> Observed transformation results:
</LI></UL><DL><DD><UL><LI> Kan plates:
</LI></UL></DD><DD><UL><LI> All successful (?) (Seq1 + Seq1, DA + DA, RA + RA), will be sequenced (ordered biobrick forward / reverse primers)
</LI></UL></DD><DD><UL><LI> Made liquid culture for miniprep tomorrow
</LI></UL></DD></DL><UL><LI> Amp plates:
</LI></UL><DL><DD><UL><LI> No successes
</LI></UL></DD></DL><UL><LI> PCR:
</LI></UL><DL><DD><UL><LI> Cas_b, Cas_c both unsuccessful
</LI></UL></DD></DL><DL><DD><UL><LI> Ran Cas_d on gel, will visualize tomorrow morning
</LI><LI> Adjusted settings for cas_e_f, cas_3_r: 2 parallel runs on thermocycler
</LI></UL></DD><DD><UL><LI> touchdown to 70 using builtin touchdown
</LI><LI> constant 70
</LI></UL></DD></DL><UL><LI> Primer design - do we need new primers???
</LI></UL><DL><DD><UL><LI> Is there melting temp mismatch?
</LI></UL></DD></DL><UL><LI> More lab supplies ordered (#4)
</LI><LI> Made new amp plates
</LI></UL><H2><SPAN class="mw-headline" id="Thursday.2C_July_7"> Thursday, July 7 </SPAN></H2><UL><LI> Miniprep of:
</LI></UL><DL><DD><UL><LI> DADA (kit buffer)
</LI><LI> RARA (kit buffer)
</LI><LI> Seq1Seq1 (kit buffer, dan buffer) all in PSB1K3
</LI></UL></DD></DL><UL><LI> Consensus: Dan's buffer that he created for lysis works
</LI><LI> Nanodrop results: (DADA 48 ng/uL, RARA 68 ng/uL, Seq1Seq1 Dan 73.5 ng/uL, Seq1Seq1 Kit 98 ng/uL)
</LI><LI> Glycerol stock of DADA, RARA, Seq1Seq1 all in PSB1K3
</LI><LI> Gel of &quot;homerun&quot; attempt from yesterday, Block A touchdown + Block B constant 70 degC
</LI></UL><DL><DD><UL><LI> Unsuccessful, deformed gel near top, nonspecific binding, weird band at ~800bp
</LI></UL></DD></DL><UL><LI> Met w/ 3 grad students (Kurt, Jack, Bo) and had good discussion
</LI></UL><DL><DD><UL><LI> Gel techniques:  doing gels in 4 degree room, using X-tracta gel removers for gel band extraction
</LI><LI> Assembly:  problems with large insertions
</LI><LI> Infusion (clonetech)
</LI><LI> Transformation of biobrick promoter J23101 onto amp (2 plates)
</LI></UL></DD></DL><UL><LI> Streak plated E0840 GFP part onto amp from glycerol stock (3 plates)
</LI><LI> Determined theoretical 3 part PCR (casEDC, casBA, case), details in today's folder
</LI><LI> PCR attempt (see photo):
</LI></UL><DL><DD><UL><LI> Block A - CasE forward, Cas3 reverse again but w/varying template DNA concentrations
</LI></UL></DD><DD><UL><LI> 225 ng/uL, 112.5 ng/uL, 56 ng/uL (previous starting conc. was 450, which is too high and could have caused problems)
</LI><LI> Settings stored as TCHDOWN (started @ 80deg, increment -.3)
</LI></UL></DD></DL><UL><LI> Block B - Cas3 forward/reverse w/same varying template DNA concentrations
</LI></UL><DL><DD><UL><LI> Settings stored as IGEM707
</LI></UL></DD></DL><UL><LI> Restriction:
</LI></UL><DL><DD><UL><LI> Seq1Seq1 (ES, EX)
</LI><LI> DADA (ES, EX)
</LI><LI> RARA (ES, EX)
</LI></UL></DD></DL><UL><LI> Ligation (protocol from OpenWetWare):
</LI></UL><DL><DD><UL><LI> 2x Seq1Seq1
</LI><LI> 2x DADA
</LI><LI> 2x RARA
</LI></UL></DD><DD><UL><LI> ((used 22.5 uL water, 15 uL insert, 5 uL vector, 5 uL T4 buffer, 2.5 uL T4 ligase))
</LI></UL></DD></DL><UL><LI> Plated onto 3 kan plates using NEB protocol.
</LI></UL><DL><DD><UL><LI> 2 problems:
</LI></UL></DD><DD> 1. did not denature ligase before transformation
</DD><DD> 2. put cells in 37 room for 1 hour without adding SOC. SOC was added and the cells were shaken for another hour in the 37 room.
</DD></DL><H2><SPAN class="mw-headline" id="Friday.2C_July_8"> Friday, July 8 </SPAN></H2><UL><LI> GFP promoter transformation:
</LI></UL><DL><DD><UL><LI> No success
</LI></UL></DD></DL><UL><LI> GFP streak plate:
</LI></UL><DL><DD><UL><LI> Grew successfully
</LI></UL></DD></DL><UL><LI> DAx4, RAx4, Seq1 transformation:
</LI></UL><DL><DD><UL><LI> Successful, but very small colonies
</LI></UL></DD></DL><UL><LI> Received order from iGEM (pSB1A3 and other agar stab)
</LI></UL><DL><DD><UL><LI> transformed pSB1A3 plasmid to make stock
</LI><LI> streak plated agar stab of J04430
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Sunday.2C_July_10"> Sunday, July 10 </SPAN></H2><UL><LI> Ran gel of array (1x, 4x):
</LI></UL><DL><DD><UL><LI> Backbones visible for all
</LI><LI> No visible inserts in plasmid backbone
</LI></UL></DD></DL><UL><LI> PCR of:
</LI></UL><DL><DD><UL><LI> Homerun, EDC, AB, 3
</LI><LI> None of these attempts work
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Monday.2C_July_11"> Monday, July 11 </SPAN></H2><UL><LI> Transformation results:
</LI></UL><DL><DD><UL><LI> pSB1A3: DNA contamination? Red colononies as well as normal, uncolored colonies growing on plates
</LI></UL></DD><DD><UL><LI> Possible contamination? We will need to sequence when we get biobrick primers
</LI></UL></DD></DL><UL><LI> Agar stab of part J04430: normal (glowing)
</LI></UL><UL><LI> We made liquid cultures of:
</LI></UL><DL><DD><UL><LI> SEQ1, DA, RA (8x) in PSB1k3
</LI><LI> PSB1A3 (pink, white colonies)
</LI><LI> J04430
</LI></UL></DD></DL><UL><LI> Restriction:
</LI></UL><DL><DD><UL><LI> Restarting from 1x in PSB1K3
</LI><LI> Ginkgo:
</LI></UL></DD><DD><UL><LI> Seq1: ES, XP
</LI><LI> DA: ES, XP
</LI><LI> RA: ES, XP
</LI><LI> E0840: EP to get PSB1A3 backbone
</LI></UL></DD></DL><UL><LI> Normal:
</LI></UL><DL><DD><UL><LI> Seq1: ES, EX
</LI><LI> DA: ES, EX
</LI><LI> RA: ES, EX
</LI></UL></DD></DL><UL><LI> ligation:
</LI></UL><DL><DD><UL><LI> 2x Seq1, DA, RA 2 ways:
</LI></UL></DD><DD><UL><LI> pSB1K3
</LI><LI> pSB1A3	
</LI></UL></DD></DL><UL><LI> Transformation of 14 plates:
</LI></UL><DL><DD><UL><LI> New promoter (g18 on plate 1) in duplicate
</LI><LI> Ligation products in duplicate
</LI></UL></DD></DL><UL><LI> Planning:
</LI></UL><DL><DD><UL><LI> Planned assembly of plasmid construct (cas genes, leader sequence, array) using pRSF-Duet plasmid
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Tuesday.2C_July_12"> Tuesday, July 12 </SPAN></H2><UL><LI> transformation results: all successful
</LI></UL><DL><DD> 1. 2x seq1 normal in psb1a3
</DD><DD> 2. 2x seq1 gingko
</DD><DD> 3. 2x ra normal in psb1a3
</DD><DD> 4. 2x ra gingko in psb1k3
</DD><DD> 5. 2x da normal in psb1a3
</DD><DD> 6. 2x da gingko  in psb1k3
</DD><DD> 7. 2x seq1 normal in psb1a3
</DD><DD> 8. 2x seq1 gingko  in psb1k3
</DD><DD> 9. 2x ra normal in psb1a3
</DD><DD> 10. 2x ra gingko  in psb1k3
</DD><DD> 11. 2x da normal in psb1a3
</DD><DD> 12. 2x da gingko  in psb1k3
</DD><DD> 13. promoter in psb1a3
</DD><DD> 14. promoter in psb1a3
</DD></DL><UL><LI> liquid cultures of:
</LI></UL><DL><DD><UL><LI> promoter
</LI><LI> e0840
</LI><LI> 2x seq1, da, ra
</LI></UL></DD></DL><UL><LI> miniprep using new ethanol protocol:
</LI></UL><DL><DD><UL><LI> 8x (ra, da, seq1)
</LI></UL></DD><DD><UL><LI> nanodropped
</LI><LI> yields: 50-1000 ng / ul
</LI></UL></DD></DL><UL><LI> psb1a3 red colony
</LI></UL><UL><LI> sequencing orders:
</LI></UL><DL><DD><UL><LI> 1x, 2x, 4x (3 times each)
</LI></UL></DD></DL><UL><LI> submitted primers / synthesis order:
</LI></UL><DL><DD><UL><LI> new cmr
</LI><LI> new cas3
</LI><LI> new casabcde
</LI><LI> adapter plasmids for leader sequences for cmr, cas
</LI></UL></DD></DL><UL><LI> ordered new restriction enzymes
</LI></UL><H2><SPAN class="mw-headline" id="Wednesday.2C_July_13"> Wednesday, July 13 </SPAN></H2><UL><LI> We haven't been using alkaline phosphatase
</LI></UL><DL><DD><UL><LI> Haven't been preventing vector self ligation
</LI></UL></DD><DD><UL><LI> We have nothing! probably
</LI></UL></DD></DL><UL><LI> ordered:
</LI></UL><DL><DD><UL><LI> AAP
</LI></UL></DD></DL><UL><LI> Answered safety questions
</LI><LI> Miniprepped promoter
</LI><LI> Still waiting on sequencing results
</LI></UL><H2><SPAN class="mw-headline" id="Thursday.2C_July_14"> Thursday, July 14 </SPAN></H2><UL><LI> Miniprep:
</LI></UL><DL><DD><UL><LI> All constructs in puc57
</LI></UL></DD></DL><UL><LI> Restriction:
</LI></UL><DL><DD><UL><LI> seq1: es, xp
</LI><LI> da: es, xp
</LI><LI> ra: es, xp
</LI><LI> psb1K3: ep
</LI><LI> promoter: sp
</LI><LI> e0840: xp
</LI></UL></DD></DL><UL><LI> Ligation:
</LI></UL><DL><DD><UL><LI> da + da
</LI><LI> ra + ra
</LI><LI> seq1 + seq1
</LI><LI> e0840 + promoter
</LI></UL></DD></DL><UL><LI> Transformation:
</LI></UL><DL><DD><UL><LI> 2x da
</LI><LI> 2x ra
</LI><LI> 2x seq1
</LI><LI> E0840 + GFP promoter
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Friday.2C_July_15"> Friday, July 15 </SPAN></H2><UL><LI> Liquid cultures:
</LI></UL><DL><DD><UL><LI> LB + Kan:
</LI></UL></DD><DD><UL><LI> 2x Seq1, DA, RA
</LI></UL></DD></DL><UL><LI> Transformation:
</LI></UL><DL><DD><UL><LI> Seq1, ra, da in puc57 (original from biobasic)
</LI></UL></DD></DL><UL><LI> Sequencing results: 3:
</LI></UL><DL><DD><UL><LI> Did not show inserts we want
</LI><LI> A piece of puc57, E. Coli genome was sequenced
</LI></UL></DD><DD><UL><LI> Possible contamination?
</LI></UL></DD></DL><UL><LI> Use of AAP will prevent this in the future
</LI></UL><UL><LI> Fixed and submitted IDT leader sequence order
</LI><LI> Restriction/ligation of:
</LI></UL><DL><DD><UL><LI> E0840, GFP promoter
</LI><LI> normal (SP, XP):
</LI></UL></DD><DD><UL><LI> used AAP on both E0840 (XP) and promoter (SP)
</LI><LI> EP vector (kan) was added that should not have been added
</LI><LI> Was subsequently transformed onto kan plate, should have been amp 
</LI></UL></DD></DL><UL><LI> Ginkgo (ES, XP, EP):
</LI></UL><DL><DD><UL><LI> Used AAP on psb1k3 vector (EP)
</LI><LI> Transformed and grown on kan plates
</LI></UL></DD></DL><UL><LI> Primers arrived!
</LI><LI> PCR
</LI></UL><DL><DD><UL><LI> CasABCDE
</LI></UL></DD><DD><UL><LI> Template concentration gradient (225 ng/uL, 112.5 ng/uL, 56 ng/uL)
</LI><LI> For each concentration, mgcl2 gradient (null, 1, 2, 4, 8)
</LI></UL></DD></DL><UL><LI> Cas3
</LI></UL><DL><DD><UL><LI> Template concentration gradient (225 ng/uL, 112.5 ng/uL, 56 ng/uL)
</LI><LI> For each concentration, mgcl2 gradient (null, 1, 2, 4, 8)
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Saturday.2C_July_16"> Saturday, July 16 </SPAN></H2><UL><LI> Check transformants
</LI></UL><DL><DD><UL><LI> GFP construct (e0840, promoter) did not grow 3:
</LI><LI> 2x seq1, da, ra in psb1k3
</LI></UL></DD></DL><UL><LI> Liquid cultures 
</LI></UL><DL><DD><UL><LI> Seq1, DA, RA in puc57, LB+Amp
</LI><LI> Original k12, LB
</LI></UL></DD></DL><UL><LI> Miniprep
</LI></UL><DL><DD><UL><LI> 2x seq1, da, ra in psb1k3
</LI><LI> Kylie will nanodrop this afternoon
</LI></UL></DD></DL><UL><LI> Glycerol stock
</LI></UL><DL><DD><UL><LI> 2x seq1, da, ra in psb1k3
</LI><LI> Threw away glycerol stocks of old 8x, 4x, 2x
</LI></UL></DD></DL><UL><LI> Gel electrophoresis
</LI></UL><DL><DD><UL><LI> CasABCDE
</LI></UL></DD><DD><UL><LI> Small gel, absolutely nothing visible
</LI></UL></DD></DL><UL><LI> Cas3
</LI></UL><DL><DD><UL><LI> Big gel, also absolutely nothing 3:
</LI></UL></DD></DL><UL><LI> CMR
</LI></UL><DL><DD><UL><LI> SUCCESS - CMR is the thermocycler favorite
</LI></UL></DD></DL><DL><DD><UL><LI> Extracted by Ethan
</LI><LI> This means our polymerase is fine, probably the template k12 that was wack
</LI></UL></DD></DL><UL><LI> Potential issues:
</LI></UL><DL><DD><UL><LI> Template DNA (which is why we are doing a new liquid culture of k12 for new genome prep)
</LI><LI> Polymerase problems?
</LI><LI> Something w/ master mix?
</LI><LI> Trinette the PCR machine didn't like the train of thermocycles running on her all day
</LI></UL></DD><DD><UL><LI>Choo Choo!
</LI></UL></DD></DL><UL><LI> PCR gnomes (DNA trafficking)
</LI></UL><UL><LI> PCR:
</LI></UL><DL><DD><UL><LI> CMR (using new primers)
</LI><LI> 220ng/ul	null, 1, 2, 4, 8, 10
</LI><LI> 181ng/ul	      1, 2, 4, 10
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Sunday.2C_July_17"> Sunday, July 17 </SPAN></H2><UL><LI> Did a genome extraction of K-12
</LI><LI> Miniprep:
</LI></UL><DL><DD><UL><LI> Seq1, ra, da in puc57
</LI><LI> Very low concentrations likely due to inclusion of precipitate -- be careful w/ this step!
</LI></UL></DD></DL><UL><LI> Restriction:
</LI></UL><DL><DD><UL><LI> promoter: sp + aap
</LI><LI> e0840: xp + aap (accident)
</LI><LI> 2x seq1: es xp + aap (accident)
</LI><LI> 2x ra: es xp
</LI><LI> 2x da: es xp
</LI><LI> psb1a3: ep + aap
</LI></UL></DD></DL><UL><LI> Ligation:
</LI></UL><DL><DD><UL><LI> promoter + e0840
</LI><LI> 2x seq1 -&gt; 4x
</LI><LI> 2x ra -&gt; 4x
</LI><LI> 2x da -&gt; 4x
</LI></UL></DD></DL><UL><LI> Transformation:
</LI></UL><DL><DD><UL><LI> Using CCMB80 cells (old)
</LI></UL></DD><DD><UL><LI> old cells may not be competent / alive
</LI></UL></DD></DL><UL><LI> PCR:
</LI></UL><DL><DD><UL><LI> ABCDE:
</LI></UL></DD><DD><UL><LI> start 70, -0.2 / cycle
</LI><LI> final: 64
</LI></UL></DD></DL><UL><LI> CAS3
</LI></UL><DL><DD><UL><LI> start 63, -0.2 / cycle
</LI><LI> final: 57
</LI></UL></DD></DL><UL><LI> gel results:
</LI></UL><DL><DD><UL><LI> ABCDE: very faint band
</LI><LI> cas3: no bands
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Monday.2C_July_18"> Monday, July 18 </SPAN></H2><UL><LI> Made new TSS cells
</LI><LI> Met with Catherine, got some good resources for modeling
</LI></UL><DL><DD><UL><LI> She will be less busy in August and can help us further
</LI></UL></DD></DL><UL><LI> PCR:
</LI></UL><DL><DD><UL><LI> CasABCDE using a lower starting temp by 5 degrees, still touchdown
</LI><LI> Gel results: no bands at all
</LI></UL></DD></DL><UL><LI> RLT:
</LI></UL><DL><DD><UL><LI> Traditional GFP construct
</LI></UL></DD><DD><UL><LI> GFP promoter (SP + AAP)
</LI><LI> E0840 (XP + gel extract)
</LI></UL></DD></DL><UL><LI> Ginkgo GFP construct
</LI></UL><DL><DD><UL><LI> GFP promoter (ES)
</LI><LI> E0840 (XP)
</LI><LI> psb1k3 (EP + aap)
</LI></UL></DD></DL><UL><LI> 4x constructs (ligation)
</LI></UL><DL><DD><UL><LI> seq1: ex, xp
</LI><LI> da: ex, xp
</LI><LI> ra: ex, xp
</LI><LI> psb1a3: ep + aap
</LI></UL></DD></DL><UL><LI> submitted sequencing:
</LI></UL><DL><DD><UL><LI> 2x: da, ra, seq1
</LI><LI> CMR (f, r)
</LI></UL></DD></DL><UL><LI> Nanodropped 1x in puc57 miniprep by Madeline
</LI></UL><DL><DD><UL><LI> Horrible results!
</LI><LI> Nisarg redid this miniprep today, will nanodrop tomorrow
</LI></UL></DD><DD><UL><LI> ((update: much better!))
</LI></UL></DD></DL><UL><LI> Made TSS cells w/ old BL21 DE3
</LI></UL><DL><DD><UL><LI> We have asked Jon for fresh cells to make better comp cells
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Tuesday.2C_July_19"> Tuesday, July 19 </SPAN></H2><UL><LI> Transformation results:
</LI></UL><DL><DD><UL><LI> 4x Seq1 successful
</LI></UL></DD><DD><UL><LI> However, we know this isn't actually 4x Seq1
</LI></UL></DD></DL><UL><LI> 4x DA, RA unsuccessful
</LI><LI> GFP construct unsuccessful
</LI></UL><UL><LI> Sequencing results:
</LI></UL><DL><DD><UL><LI> CMR: good =3
</LI><LI> All else: nothing that we need 3:
</LI></UL></DD><DD><UL><LI> Just vector sequences
</LI></UL></DD></DL><UL><LI> New sequencing order:
</LI></UL><DL><DD><UL><LI> Putative RFP
</LI><LI> Original biobasic synthesis products in puc57 (with new puc57 primers)
</LI></UL></DD><DD><UL><LI> da, ra, seq1
</LI></UL></DD></DL><UL><LI> Our 1x transformation of biobasic synthesis products in puc57 (with new puc57 primers)
</LI></UL><DL><DD><UL><LI> 1xda, 1xra, 1xseq1
</LI></UL></DD></DL><UL><LI> Genomic DNA from mg1655 (k-12) to see if they can pull out cas gene sequences that we can't
</LI></UL><DL><DD><UL><LI> abcde f/r and cas3 f/r
</LI></UL></DD></DL><UL><LI> PCR:
</LI></UL><DL><DD><UL><LI> Block A:
</LI></UL></DD><DD><UL><LI> Settings we used to get faint bands
</LI><LI> CasABCDE
</LI><LI> Touchdown: 70, -0.2 / cycle -&gt; 63
</LI></UL></DD></DL><UL><LI> Block B:
</LI></UL><DL><DD><UL><LI> Cas3
</LI><LI> Gradient with 3 rows from 72 to 55
</LI></UL></DD></DL><UL><LI> Late night gel results: (Madeline and Joseph)
</LI></UL><DL><DD><UL><LI> CasABCDE: broad bands, perhaps desired band at ~4300 faintly @ higher magnesium concentrations
</LI></UL></DD></DL><DL><DD><UL><LI> Cas3: broad bands, also no ladder
</LI></UL></DD><DD><UL><LI> So we THINK we got the desired band in the &quot;midrange&quot; temperature, which corresponds to &quot;third up&quot; row and relatively high magnesium chloride concentration
</LI><LI> SO we should do another PCR at approximately slightly above 60 somewhere
</LI><LI> Suggestion: gradient between 65 and 60
</LI><LI> Also: what does it mean that it is curved?
</LI><LI> Also: what is the brightness at the bottom? happened in both gels...
</LI></UL></DD></DL><UL><LI> Restriction, Ligation and Transformation (RLT):
</LI><LI> Tried to gel E0840 cut w/ XP to get higher concentration of insert for ligation
</LI></UL><DL><DD><UL><LI> No bands at all 3:
</LI></UL></DD></DL><UL><LI> Instead, ligated / transformed purified GFP insert from yesterday
</LI><LI> @ 3ng/uL with approximately 40 uL, therefore ~120 ng of GFP insert (E0840)
</LI></UL><DL><DD><UL><LI> w/ 5 uL of vector (promoter cut at SP and w/ AAP)
</LI><LI> 5 uL of T4 ligase buffer
</LI><LI> 2.5 uL of T4 ligase
</LI><LI> total: 52.5 uL reaction
</LI></UL></DD></DL><UL><LI> Other news:
</LI></UL><DL><DD><UL><LI> Got new bacterial strains from Jon
</LI><LI> Submitted Barrett reimbursement forms for gas, food
</LI><LI> Met w/ Alana Labelle about safety, who told us to complete a &quot;Preliminary Hazard Assessment&quot; form
</LI><LI> New primers for duet vector and puc57 arrived
</LI><LI> GOT ICE ACCESS!!!
</LI><LI> Kylie/Keith battle has begun
</LI><LI> Xiao is proud of us :3
</LI></UL></DD></DL><UL><LI> To do:
</LI></UL><DL><DD><UL><LI> Email iGEM to see what's up w/ Indianapolis accommodations
</LI><LI> Fill out form
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Wednesday.2C_July_20"> Wednesday, July 20 </SPAN></H2><UL><LI> Transformation results:
</LI></UL><DL><DD><UL><LI> 2 tiny colonies on one plate (GFP)
</LI><LI> Made liquid culture (lb amp)
</LI></UL></DD><DD><UL><LI> Did not grow
</LI></UL></DD></DL><UL><LI> PCR:
</LI></UL><DL><DD><UL><LI> ABCDE: starting: 71, -0.2/cycle -&gt; 64
</LI><LI> Cas3: another gradient using tighter range around best temp from yesterday
</LI></UL></DD><DD><UL><LI> 65.6
</LI><LI> 63.1
</LI><LI> 61.2
</LI><LI> 58.5
</LI></UL></DD></DL><UL><LI> Results: 
</LI></UL><P>
ABCDE: nothing usable
</P><DL><DD><UL><LI> CAS3: small band at correct location
</LI></UL></DD></DL><UL><LI> Need to order new primers with less dimerization potential!
</LI></UL><H2><SPAN class="mw-headline" id="Thursday.2C_July_21"> Thursday, July 21 </SPAN></H2><UL><LI> Heat shocked overnight RE digests (4 tubes of each for maximal gel results)
</LI></UL><DL><DD><UL><LI> E0840
</LI><LI> Seq1, RA, DA
</LI></UL></DD></DL><UL><LI> Gel of RE results
</LI></UL><DL><DD><UL><LI> Very clear bright bands for (ES) E0840 vector, insert
</LI></UL></DD><DD><UL><LI> Extracted insert
</LI></UL></DD></DL><UL><LI> No visible insert for Seq1, but backbone visible
</LI></UL><UL><LI> PCR Insert Amplification, Attempt #1
</LI></UL><DL><DD><UL><LI> Taq polymerase from Bioline
</LI><LI> Notable: 30 sec elongation time
</LI></UL></DD></DL><UL><LI> Gel of PCR Results
</LI></UL><DL><DD><UL><LI> faint bands at ~1000bp
</LI><LI> likely due to 30sec elongation, will try again w/shorter time
</LI></UL></DD></DL><UL><LI> Late Night:
</LI><LI> Gel of Restricted (ES) DA, RA
</LI></UL><DL><DD><UL><LI> 1% Agarose, 40mL total
</LI><LI> Showed very strong, clear bands for backbone, no inserts
</LI></UL></DD></DL><UL><LI> PCR Insert Amplification, Attempt #2
</LI></UL><DL><DD><UL><LI> Used Phusion instead of Biolase product
</LI><LI> Annealing temp: 58 deg, based off of NEB calculator as recommended by Phusion protocol
</LI><LI> Annealing time: 10 sec (low end)
</LI><LI> Elongation @ 72 degrees for 5 seconds (much lower than previous attempt, corresponds to 170-330 bp)
</LI><LI> 25 total cycles (low end)
</LI><LI> 5 minute extension (low end)
</LI></UL></DD></DL><UL><LI> Gel Result:
</LI></UL><DL><DD><UL><LI> Very strong, medium thick bands
</LI><LI> Seq1 ~ 130-180bp
</LI><LI> DA/RA ~ 190-220bp
</LI></UL></DD></DL><UL><LI> Other notes:
</LI></UL><DL><DD><UL><LI> Met with Kurt and a new grad student today at 11 as usual. We discussed various methods to improve our yields in assembling the RSR array, including using PCR amplification with the pUC57 primers we used for sequencing to create the insert in excess prior to gel extraction. Also discussed PCRing PCR products to increase yield (we will likely try this for CasABCDE and Cas3 w/ the current primers). 
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Friday.2C_July_22"> Friday, July 22 </SPAN></H2><UL><LI> PCR Amplification of pUC57 inserts
</LI></UL><DL><DD><UL><LI> 4 second elongation (instead of 5 sec), 35 cycles (instead of 25)
</LI></UL></DD></DL><UL><LI> Gel Results
</LI></UL><DL><DD><UL><LI> same major bands as first attempt (~150 for seq1, ~200 for DA, RA)
</LI><LI> more minor bands than before
</LI><LI> might be contamination in second DA well
</LI></UL></DD></DL><UL><LI> Try again
</LI></UL><DL><DD><UL><LI> PCR with 4 second elongation, but 25 cycles
</LI><LI> Will gel small amount and if only 1 band is present, will restrict directly
</LI></UL></DD></DL><UL><LI> Nanodropped
</LI></UL><DL><DD><UL><LI> Seq1, RA from per amp + gel extraction (~22ng/uL, ~12ng/uL)
</LI><LI> E0840 gel extraction (~6ng/uL)
</LI></UL></DD></DL><UL><LI> GFP Restriction, Ligation
</LI></UL><DL><DD><UL><LI> Previously restricted (XP) E0840 insert from gel extraction
</LI><LI> (SP) Promoter in pSB1A3
</LI><LI> RAGE, Seq1RE into pSB1A3
</LI></UL></DD><DD><UL><LI> Problem: water bath was too hot (~65) and enzymes were denatured
</LI><LI> Will try again tomorrow
</LI></UL></DD></DL><UL><LI> Ordered new primers
</LI></UL><DL><DD><UL><LI> CasABCDE &amp; Cas3
</LI><LI> Much lower affinity for dimerization
</LI><LI> Nice, close melting points
</LI></UL></DD></DL><UL><LI> Ordered Phusion, enzymes (SpeI, DpnI), PCR cleanup kit
</LI><LI> Sent in sequencing requests
</LI><LI> Seq1, RA gel extracts (Forward and reverse primers separately)
</LI></UL><H2><SPAN class="mw-headline" id="Saturday.2C_July_23"> Saturday, July 23 </SPAN></H2><UL><LI> Ran gel of PCR amplification results
</LI></UL><DL><DD><UL><LI> (ran out of Seq1, so RA and DA were gelled)
</LI><LI> 10 uL in each well (2uL buffer, 2uL product, 6uL water)
</LI></UL></DD></DL><UL><LI> Results
</LI></UL><DL><DD><UL><LI> very clear bands (except for DA w/4 mgcl2, which was faint/blurry)
</LI></UL></DD></DL><UL><LI> Restriction Digest
</LI></UL><DL><DD><UL><LI> DA, RA from PCR amplification (no gel extraction)
</LI><LI> E, P digest
</LI><LI> Stored at -20
</LI></UL></DD></DL><UL><LI> Check transformant from Friday
</LI></UL><DL><DD><UL><LI> A single small colony visible on the GFP plate
</LI><LI> No glow visible, but will keep growing it up in case
</LI></UL></DD></DL><UL><LI> Miniprep of Seq1, RA, DA in puc57
</LI></UL><DL><DD><UL><LI> added another round of centrifugation in step 11
</LI><LI> NANODROP RECORD!!! 3412 ng/uL by Joseph Flay
</LI></UL></DD></DL><UL><LI> Glycerol stocks of BL21 DE3, MG1655
</LI></UL><H2><SPAN class="mw-headline" id="Sunday.2C_July_24"> Sunday, July 24 </SPAN></H2><UL><LI> Checked GFP transformation colony from Friday again
</LI></UL><DL><DD><UL><LI> Single colony that hadn't glowed before was much bigger
</LI><LI> IT GLOWS
</LI></UL></DD><DD> --&gt; Liquid culture
<UL><LI> PCR of CasABCDE, Cas3 with new settings
</LI></UL></DD></DL><DL><DD><UL><LI> Preparation for extraction, &quot;PCRception&quot; (PCRing the PCR results)
</LI><LI> Included an elongation step in each cycle, which was not done for the previous runs
</LI><LI> Settings stored in PCR machine as ABCDE724, CAS3724
</LI></UL></DD></DL><UL><LI> Restriction
</LI></UL><DL><DD><UL><LI> RAGE (code for RA gel extraction), Seq1 GE (Seq1 gel extraction) (EP)
</LI><LI> RFP in pSB1A3 (EP) + AAP
</LI></UL></DD></DL><UL><LI> Ligation
</LI></UL><DL><DD><UL><LI> RAGE + pSB1A3
</LI><LI> Se1GE + pSB1A3
</LI><LI> PCR RA + pSB1A3 (messed this one up, probably won't grow)
</LI><LI> PCR DA + pSB1A3
</LI></UL></DD></DL><UL><LI> Transformation of ligation products
</LI><LI> Did dishes
</LI></UL><H2><SPAN class="mw-headline" id="Monday.2C_July_25"> Monday, July 25 </SPAN></H2><UL><LI> Autoclaved and finished dishes
</LI><LI> Checked transformants
</LI></UL><DL><DD><UL><LI> RAGE + pSB1A3 (growth)
</LI><LI> Se1GE + pSB1A3 (growth)
</LI><LI> PCR RA + pSB1A3 (no grow, but was messed up)
</LI><LI> PCR DA + pSB1A3 (growth)
</LI></UL></DD></DL><UL><LI> Liquid cultures
</LI></UL><DL><DD><UL><LI> 3 tubes total
</LI><LI> RAGE, Se1GE, PCR DA in A3 (Amp broth)
</LI></UL></DD></DL><UL><LI> Gel of PCR results
</LI></UL><DL><DD><UL><LI> CasABCDE: not much visible, very faint and nonspecific
</LI></UL></DD><DD><UL><LI> Could be a result of changing the settings too much
</LI><LI> We expect our new primers today/tomorrow so we will likely wait for that
</LI></UL></DD></DL><UL><LI> Cas3: lots of bands visible, looks as though there's a clear band at ~2600
</LI></UL><DL><DD><UL><LI> Gel extraction
</LI></UL></DD></DL><UL><LI> Miniprep of successful constitutive GFP generator
</LI><LI> To nanodrop
</LI></UL><DL><DD><UL><LI> GFP generator
</LI><LI> E0840 gel extract
</LI></UL></DD></DL><UL><LI> Restriction
</LI></UL><DL><DD><UL><LI> Preparation of pSB1K3 backbone
</LI></UL></DD><DD><UL><LI> Protocol from Standard Parts Registry (E, P, DpnI)
</LI></UL></DD></DL><UL><LI> RFP in pSB1A3 (EP)
</LI></UL><UL><LI> Ligation/Transformation
</LI></UL><DL><DD><UL><LI> RFP + pSB1K3
</LI></UL></DD></DL><UL><LI> Notes:
</LI></UL><DL><DD><UL><LI> Still waiting on sequencing results to confirm that the things we have amplified are indeed RA, DA, and Seq1 ((though we won't be sure about DA because we didn't submit it yet))
</LI><LI> We can submit our 1x in A3 for sequencing tomorrow, after it grows up
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Tuesday.2C_July_26"> Tuesday, July 26 </SPAN></H2><UL><LI> Discussed USC CRISPER Conundrum
</LI><LI> Discussed alternate spacers to assemble in parallel
</LI></UL><DL><DD><UL><LI> RFP
</LI><LI> Nickase
</LI><LI> Biofilm &quot;Isr&quot;
</LI></UL></DD></DL><UL><LI> Miniprep
</LI></UL><DL><DD><UL><LI> 1xSeq1, RA, DA in psb1A3
</LI></UL></DD></DL><UL><LI> Genome Prep
</LI><LI> PCR w/ new primers
</LI></UL><DL><DD><UL><LI> CasABCDE - 68 degree annealing temp, 2 minute elongation
</LI><LI> Cas3 - 62 degree annealing temp, 1:20 elongation
</LI></UL></DD></DL><UL><LI> Gel results:
</LI></UL><DL><DD><UL><LI> ABCDE - mostly blank
</LI><LI> Cas3 - broad nonspecific bands around desired area
</LI></UL></DD></DL><UL><LI> Nanodrop
</LI></UL><DL><DD><UL><LI> 1x Seq1, RA, DA (400 ng/uL or so)
</LI><LI> genome prep (77 ng/uL)
</LI><LI> extracted cas3 (11 ng/uL)
</LI><LI> GFP (4000 ng/uL - NEW FLAYPREP RECORD)
</LI></UL></DD></DL><UL><LI> Checked transformants
</LI></UL><DL><DD><UL><LI> 1 of 2 gfp streak plates glows green, the other is partial
</LI><LI> RFP plasmids are not red
</LI></UL></DD><DD><UL><LI> could be a result of improper AAP use, will try again
</LI><LI> grew up in a liquid culture (tried Kan and Amp separately)
</LI></UL></DD><DD><UL><LI> Did not grow in amp broth - no vector/vector ligation
</LI></UL></DD></DL><UL><LI> Restriction Digest
</LI></UL><DL><DD><UL><LI> (EP) pSB1K3 backbone + AAP
</LI><LI> (EP) RFP in pSB1A3
</LI></UL></DD></DL><UL><LI> Ligation
</LI></UL><DL><DD><UL><LI> 1:1 insert:vector ratio according to linearized plasmid backbone protocol
</LI><LI> 3:1 insert to vector ratio according to OWW protocol
</LI><LI> Both methods transformed/plated
</LI></UL></DD></DL><UL><LI> Overnight RE Digest
</LI></UL><DL><DD><UL><LI> Seq1 in pSB1A3 (ES, XP)
</LI><LI> DA in pSB1A3 (ES, XP)
</LI><LI> RA in pSB1A3 (ES, XP)
</LI><LI> (EP) pSB1K3 from &quot;DA in pSB1K3&quot;, which is not really DA but is really the vector
</LI></UL></DD><DD><UL><LI> need to AAP this tomorrow
</LI></UL></DD></DL><UL><LI> Sequencing Results
</LI></UL><DL><DD><UL><LI> Gel extracts of PCR amplified Seq1 and RA are good
</LI></UL></DD></DL><UL><LI> Sequencing Submission
</LI><LI> GFP forward
</LI></UL><DL><DD><UL><LI> Cas3 F/R
</LI><LI> Seq1, DA, RA 1x in psb1A3 (Forward/Reverse)
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Wednesday.2C_July_27"> Wednesday, July 27 </SPAN></H2><UL><LI> Ligate 2x in psb1k3
</LI></UL><DL><DD><UL><LI> Using overnight digests
</LI></UL></DD><DD><UL><LI> Seq1 in pSB1A3 (ES, XP)
</LI><LI> DA in pSB1A3 (ES, XP)
</LI><LI> RA in pSB1A3 (ES, XP)
</LI><LI> (EP) pSB1K3 from &quot;DA in pSB1K3&quot;, which is not really DA
</LI></UL></DD></DL><UL><LI> Transformation of this
</LI><LI> Begin duet construction:
</LI></UL><DL><DD><UL><LI> Transformed duet plasmid
</LI></UL></DD></DL><UL><LI> Streak plate RFP plates
</LI><LI> PCR:
</LI></UL><DL><DD><UL><LI> CasABCDE with new and old primers and a temp gradient for both
</LI><LI> New: 64-&gt;69, three rows of 3 tubes
</LI><LI> Old: 63-&gt;66, three rows of 3 tubes
</LI></UL></DD></DL><UL><LI> Gel results:
</LI></UL><DL><DD><UL><LI> New -- no good
</LI><LI> Old -- mostly blank with some dimerization
</LI></UL></DD></DL><UL><LI> Made more LB + kan, amp, no antibiotic plates
</LI><LI> Talked about CMR things
</LI></UL><DL><DD><UL><LI> We can buy P. furiosus DNA at 2 ug for $300
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Thursday.2C_July_28"> Thursday, July 28 </SPAN></H2><UL><LI> Plates from yesterday:
</LI></UL><DL><DD><UL><LI> Negative control with amp: growing
</LI><LI> RFP streak plates: correct
</LI><LI> Duet vector: correct
</LI><LI> 2x ra, da, seq1: did not grow
</LI></UL></DD></DL><UL><LI> Gel of restrictions from yesterday
</LI></UL><DL><DD><UL><LI> 1st try: no success, ran off
</LI><LI> 2nd try: no success, only backbones visible...
</LI></UL></DD></DL><UL><LI> PCR cleanup of DA PCRPCR to nanodrop, transform
</LI><LI> PCR:
</LI></UL><DL><DD><UL><LI> CasABCDE (old primers)
</LI></UL></DD><DD><UL><LI> TDWN728B
</LI><LI> 71 to 68, -.1/cycle, 30 cycles
</LI><LI> 2:10 elongation
</LI></UL></DD></DL><UL><LI> CasABCDE (new primers)
</LI></UL><DL><DD><UL><LI> TDWN728A
</LI><LI> 72 to 65, -.2/cycle, 35 cycles
</LI><LI> 2:10 elongation
</LI></UL></DD></DL><UL><LI> Gel results:
</LI></UL><DL><DD><UL><LI> No good! ran these at night, all on one gel, and only one lane had anything at all and even then it was one large section of nonspecific amplification
</LI></UL></DD></DL><UL><LI> Meeting with grad students:
</LI></UL><DL><DD><UL><LI> Homology between casA and human immune proteins (which ones?)
</LI></UL></DD><DD><UL><LI> Could be an evolutionary thing
</LI></UL></DD></DL><UL><LI> Look into this could also be
</LI></UL><DL><DD><UL><LI> Conserved functional domains
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Friday.2C_July_29"> Friday, July 29 </SPAN></H2><UL><LI> Transformation results from yesterday
</LI></UL><DL><DD><UL><LI> One colony of 1xDA was not pink, yay
</LI></UL></DD></DL><UL><LI> Remade liquid culture of 1xDA because morning miniprep had goopy crap
</LI></UL><DL><DD><UL><LI> Will miniprep later
</LI></UL></DD></DL><UL><LI> RLT?
</LI></UL><DL><DD><UL><LI> Will wait until we miniprep 1xDA
</LI></UL></DD></DL><UL><LI> Gibson Assembly? Do we have all materials?
</LI></UL><DL><DD><UL><LI> No, we'll have to order a couple of things
</LI><LI> Will plan on doing this and trying Gibson in parallel
</LI></UL></DD></DL><UL><LI> Made liquid culture of Duet plasmid
</LI><LI> Made more LB Kan media
</LI><LI> PCR
</LI></UL><DL><DD><UL><LI> Retry original 7/19 run that was semi-successful
</LI><LI> Another ABCDE
</LI></UL></DD></DL><UL><LI> Night Results
</LI></UL><DL><DD><UL><LI> Gels: No good. Nothing. &quot;Dimer Central&quot;
</LI></UL></DD></DL><DL><DD><UL><LI> Miniprep of 1xDA
</LI><LI> Made SOC mini-stocks
</LI><LI> Glycerol stocks of something maybe
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Saturday.2C_July_30"> Saturday, July 30 </SPAN></H2><UL><LI> PCR
</LI></UL><DL><DD><UL><LI> &quot;Touchup&quot; PCR
</LI></UL></DD></DL><UL><LI> RLT
</LI></UL><DL><DD><UL><LI> 2x DA, Seq1, RA (Ginkgo)
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Sunday.2C_July_31"> Sunday, July 31 </SPAN></H2><UL><LI> Gel Results of &quot;Touchup&quot;
</LI></UL><DL><DD><UL><LI> Worthless
</LI></UL></DD></DL><UL><LI> Check plates
</LI></UL><DL><DD><UL><LI> No transformants but perhaps a few small ones
</LI></UL></DD></DL><UL><LI> Redo transformation
</LI><LI> Redigest RFP pSB1K3 for another RLT</LI></UL></DIV></DIV></TD></TR></TBODY></TABLE><CENTER><DIV style="color: #000000; background-color: #EBEBEB"><DIV id="content-footer-box"><DIV class="footer-left-half"><DIV class="footer-left-col"><SMALL><P><B>Contact Us</B></P><P>Arizona State University</P><P>ECG 334, PO BOX 9709</P><P>Tempe, Arizona 85287</P><P><A href="mailto:asu.igem@gmail.com">Email</A></P></SMALL></DIV></DIV></DIV></DIV></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Arizona_State/Notebook/July">http://2011.igem.org/Team:Arizona_State/Notebook/July</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Arizona_State/Notebook/July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Arizona_State/Notebook/July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Arizona_State/Notebook/July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Arizona_State/Notebook/July&amp;oldid=200802" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>