<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Arizona_State_Project_CRISPR"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Arizona_State/Project/CRISPR">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Arizona_State/Project/CRISPR&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Arizona_State/Project/CRISPR&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Arizona_State/Project/CRISPR&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Arizona_State/Project/CRISPR" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Arizona State/Project/CRISPR</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV style="font-size:18pt;"><FONT face="Calibri" style="color:#000000"><CENTER>CRISPR
</CENTER></FONT></DIV><DIV><TABLE cellspacing="0"><TBODY><TR><TD style="background-color: #ffffff" width="75px" valign="top"><DIV class="glossymenu"><A class="menuitem" href="https://2011.igem.org/Team:Arizona_State">Home</A><A class="menuitem" href="https://2011.igem.org/Main_Page">iGEM 2011 Home</A><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Project">Project</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/Introduction">Introduction</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/CRISPR">CRISPR</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/E_coli">E. coli</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/B_halodurans">B. halodurans</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/L_innocua">L. innocua</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/Future">Future</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/References">References</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Lab">Lab</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Acknowledgements">Acknowledgements</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Results">Results</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Results/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Parts">BioBricks</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Lab">Notebook</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/AprilMay">April/May</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/June">June</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/July">July</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/August">August</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/September">September</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/PCRLog">Cas PCR Log</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/Sequences">Sequence Information</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/Other">Other Documents</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Outreach">Human Practices</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Events">Events</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Collaborations">Collaborations</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Exploring_Synthetic_Biology">Exploring Synthetic Biology</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Outreach_in_Practice">Outreach in Practice</A></LI></UL></DIV></DIV></TD><TD width="880" valign="top" style="padding: 10px; border: 5px solid #ffffff; color: #000; background-color: white"><DIV style="position: relative; margin-top: -450px; min-height: 450px"><DIV> <P><I>See <A href="/Team:Arizona_State/Glossary" title="Team:Arizona State/Glossary">glossary</A> for explanation of various abbreviations used on this page.</I></P><P><B>C</B>lustered <B>R</B>egularly <B>I</B>nterspaced <B>S</B>hort <B>P</B>alindromic <B>R</B>epeats (CRISPR) are a genomic feature of many prokaryotic and archaeal species. 40% of sequenced bacterial genomes and 90% of archaeal genomes contain at least one CRISPR array<SUP><A href="/Team:Arizona_State/Project/References#ref20" title="Team:Arizona State/Project/References">[20]</A></SUP>. It is possible that many laboratory strains of bacteria, which are the sources of many available genome sequences, have lost CRISPR due to a lack of exposure to phages<SUP><A href="/Team:Arizona_State/Project/References#ref42" title="Team:Arizona State/Project/References">[42]</A></SUP>.</P><P>CRISPR functions as an adaptive and inheritable immune system<SUP><A href="/Team:Arizona_State/Project/References#ref40" title="Team:Arizona State/Project/References">[40]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref53" title="Team:Arizona State/Project/References">[53]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref34" title="Team:Arizona State/Project/References">[34]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref38" title="Team:Arizona State/Project/References">[38]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref42" title="Team:Arizona State/Project/References">[42]</A></SUP>. A CRISPR locus consists of a set of Cas (CRISPR associated) genes, a leader, or promoter, sequence, and an array. This array consists of repeating elements along with &quot;spacers&quot;. These spacer regions direct the CRISPR machinery to degrade or otherwise inactivate a complementary sequence in the cell.</P></DIV></DIV></TD></TR><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#The_CRISPR_array"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">The CRISPR array</SPAN></A></LI><LI class="toclevel-1"><A href="#Engineered_arrays"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Engineered arrays</SPAN></A></LI><LI class="toclevel-1"><A href="#CRISPR_in_Escherichia_coli_K-12_substr._MG1655"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">CRISPR in Escherichia coli K-12 substr. MG1655</SPAN></A></LI><LI class="toclevel-1"><A href="#CRISPR_in_Pyrococcus_furiosus_DSM_3638"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">CRISPR in Pyrococcus furiosus DSM 3638</SPAN></A></LI><LI class="toclevel-1"><A href="#CRISPR_in_Bacillus_halodurans_C-125"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">CRISPR in Bacillus halodurans C-125</SPAN></A></LI><LI class="toclevel-1"><A href="#CRISPR_in_Listeria_innocua_Clip11262"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">CRISPR in Listeria innocua Clip11262</SPAN></A></LI><LI class="toclevel-1"><A href="#Stages_of_the_CRISPR_pathway"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Stages of the CRISPR pathway</SPAN></A></LI><LI class="toclevel-2"><A href="#Integration_.2F_Adaptation"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Integration / Adaptation</SPAN></A></LI><LI class="toclevel-2"><A href="#Expression"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Expression</SPAN></A></LI><LI class="toclevel-2"><A href="#Interference"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Interference</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Core_Cas_genes"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Core Cas genes</SPAN></A></LI><LI class="toclevel-2"><A href="#Cas1.2C_Cas2"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Cas1, Cas2</SPAN></A></LI><LI class="toclevel-2"><A href="#Cas3"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Cas3</SPAN></A></LI><LI class="toclevel-1"><A href="#Prevention_of_self_targeting_.28autoimmunity.29"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Prevention of self targeting (autoimmunity)</SPAN></A></LI><LI class="toclevel-1"><A href="#CRISPR_regulation"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">CRISPR regulation</SPAN></A></LI><LI class="toclevel-1"><A href="#Classification_of_CRISPR_systems"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Classification of CRISPR systems</SPAN></A></LI><LI class="toclevel-2"><A href="#Type_I.2C_II_and_III_systems"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Type I, II and III systems</SPAN></A></LI><LI class="toclevel-1"><A href="#CRISPR_resources"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">CRISPR resources</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="The_CRISPR_array"> The CRISPR array </SPAN></H2><P>Genetic information from previous encounters is stored in the array as spacers. These spacers are consistent in length (30-40 bp), and are flanked by repeating elements (also 30-40 bp). The repeating elements are usually partially palindromic, and form secondary structures when transcribed into pre-crRNA. These structures may be necessary for recognition and cleavage.</P><H2><SPAN class="mw-headline" id="Engineered_arrays"> Engineered arrays </SPAN></H2><P>By engineering a spacer complementary to T3 phage, increased survival was demonstrated<SUP><A href="/Team:Arizona_State/Project/References#ref17" title="Team:Arizona State/Project/References">[17]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref25" title="Team:Arizona State/Project/References">[25]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref28" title="Team:Arizona State/Project/References">[28]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref51" title="Team:Arizona State/Project/References">[51]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref59" title="Team:Arizona State/Project/References">[59]</A></SUP>. A customized spacer can prevent transformation of PC194 plasmids with a matching sequence<SUP><A href="/Team:Arizona_State/Project/References#ref28" title="Team:Arizona State/Project/References">[28]</A></SUP>.</P><H2><SPAN class="mw-headline" id="CRISPR_in_Escherichia_coli_K-12_substr._MG1655"> CRISPR in <I>Escherichia coli K-12 substr. MG1655</I></SPAN></H2><P><B><A href="/Team:Arizona_State/Project/E_coli" title="Team:Arizona State/Project/E coli">Project page</A></B></P><P><I>E. coli</I> contains a type I CRISPR system. There are four CRISPR loci in this organism. CRISPR1, the largest, is associated with eight Cas genes<SUP><A href="/Team:Arizona_State/Project/References#ref73" title="Team:Arizona State/Project/References">[73]</A></SUP>. In the classification scheme presented by Haft et al<SUP><A href="/Team:Arizona_State/Project/References#ref15" title="Team:Arizona State/Project/References">[15]</A></SUP>, these genes form the Cse family: casA, casB, casC, casD, casE, aka cse1, cse2, cse3, cse4, cas5e<SUP><A href="/Team:Arizona_State/Project/References#ref15" title="Team:Arizona State/Project/References">[15]</A></SUP>. These 5 proteins combine to form the Cascade complex<SUP><A href="/Team:Arizona_State/Project/References#ref63" title="Team:Arizona State/Project/References">[63]</A></SUP>. This is a protein complex of all 5 Cse genes, resembling a seahorse in shape<SUP><A href="/Team:Arizona_State/Project/References#ref63" title="Team:Arizona State/Project/References">[63]</A></SUP>. Its full composition is 1x casA, 2x casB, 6x casC, 1x casD, 1x casE<SUP><A href="/Team:Arizona_State/Project/References#ref63" title="Team:Arizona State/Project/References">[63]</A></SUP>. Specifically, casE cleaves pre-crRNA<SUP><A href="/Team:Arizona_State/Project/References#ref25" title="Team:Arizona State/Project/References">[25]</A></SUP>, and casA and casB can be omitted without affecting crRNA generation, but are necessary for phage resistance<SUP><A href="/Team:Arizona_State/Project/References#ref63" title="Team:Arizona State/Project/References">[63]</A></SUP>. This complex binds double stranded target DNA without need or enhancement by cofactors such as metal ions or ATP<SUP><A href="/Team:Arizona_State/Project/References#ref63" title="Team:Arizona State/Project/References">[63]</A></SUP>. It also undergoes conformational changes when binding DNA<SUP><A href="/Team:Arizona_State/Project/References#ref63" title="Team:Arizona State/Project/References">[63]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref76" title="Team:Arizona State/Project/References">[76]</A></SUP>.</P><P>Cas gene transcription is repressed by H-NS<SUP><A href="/Team:Arizona_State/Project/References#ref70" title="Team:Arizona State/Project/References">[70]</A></SUP>, and de-repressed by leuO<SUP><A href="/Team:Arizona_State/Project/References#ref48" title="Team:Arizona State/Project/References">[48]</A></SUP> or baeR<SUP><A href="/Team:Arizona_State/Project/References#ref57" title="Team:Arizona State/Project/References">[57]</A></SUP></P>.
<DL><DD>Coordinates:
</DD><DD> CrisprI: 2875723-2876485
</DD><DD> Leader: 2876485-2876592
</DD><DD> Cas2 (ygbF): 2876592-2876875
</DD><DD> Cas1 (ygbT): 2876877-2877794
</DD><DD> CasE: 2877810-2878409
</DD><DD> CasD: 2878396-2879070
</DD><DD> CasC: 2879073-2880164
</DD><DD> CasB: 2880177-2880659
</DD><DD> CasA: 2880652-2882160
</DD><DD> IGLB regulatory region: 2882160-2882575
</DD><DD> Cas3 (ygcB): 2882575-2885241
</DD></DL><P>Structure of the CRISPR I locus in <I>E. coli</I>. 3 promoters have been characterized<SUP><A href="/Team:Arizona_State/Project/References#ref43" title="Team:Arizona State/Project/References">[43]</A></SUP>: Pcrispr1, Pcas, and anti-Pcas.</P><H2><SPAN class="mw-headline" id="CRISPR_in_Pyrococcus_furiosus_DSM_3638"> CRISPR in <I>Pyrococcus furiosus DSM 3638</I></SPAN></H2><P>This organism is notable due to the diversity of its Cas genes, as well as its possible RNA targeting. <I>P. furiosus</I> contains 7 CRISPR loci, along with 29 Cas genes in 2 gene clusters<SUP><A href="/Team:Arizona_State/Project/References#ref35" title="Team:Arizona State/Project/References">[35]</A></SUP>. All 6 core Cas genes (cas1-cas6), as well as genes from the Cmr (type III), Cst (type I), and Csa (type I) families are present. Cmr1-6 have been found to form a Cascade-like complex that targets RNA in in-vitro experiments<SUP><A href="/Team:Arizona_State/Project/References#ref35" title="Team:Arizona State/Project/References">[35]</A></SUP>.</P><H2><SPAN class="mw-headline" id="CRISPR_in_Bacillus_halodurans_C-125"> CRISPR in <I>Bacillus halodurans C-125</I></SPAN></H2><P><B><A href="/Team:Arizona_State/Project/B_halodurans" title="Team:Arizona State/Project/B halodurans">Project page</A></B></P><P><I>B. halodurans</I> contains 6 Cmr genes (Cmr1-6) in a single locus. This is a type III CRISPR system. The organism also contains Csd1 and Csd2 (Dvulg subtype I-C) along with Cas1, Cas2, Cas3, Cas4, and Cas5 in another locus.</P><P><SMALL>*Note: These regions have not been characterized in the literature and are speculative. The two regions shown are homologous.
</SMALL></P><DL><DD>Coordinates:
</DD><DD> CrisprI: 339740-340773
</DD><DD> Leader: 345821-345991
</DD><DD> CrisprII: 345991-347087
</DD><DD> Cmr1 (RAMP): 347483-348859
</DD><DD> Cmr2 (RAMP): 348829-350643
</DD><DD> Cmr3 (RAMP): 350603-351874
</DD><DD> Cmr4 (RAMP): 351874-352785
</DD><DD> Cmr5 (RAMP): 352782-353159
</DD><DD> Cmr6 (RAMP): 353156-353902
</DD><DD> Leader: 355127-355361
</DD><DD> CrisprIII: 355361-356328
</DD><DD> Cas3 (Dvulg): 358059-360461
</DD><DD> Cas5 (Dvulg): 360504-361214
</DD><DD> Csd1 (Dvulg): 361216-363099
</DD><DD> Csd2 (Dvulg): 363096-363947
</DD><DD> Cas4 (Dvulg): 364072-364596
</DD><DD> Cas1 (Dvulg): 364593-365624
</DD><DD> Cas2 (Dvulg): 365633-365923
</DD><DD> CrisprIV: 366106-368458
</DD><DD> CrisprV: 378211-378913
</DD></DL><H2><SPAN class="mw-headline" id="CRISPR_in_Listeria_innocua_Clip11262"> CRISPR in <I>Listeria innocua Clip11262</I></SPAN></H2><P><B><A href="/Team:Arizona_State/Project/L_innocua" title="Team:Arizona State/Project/L innocua">Project page</A></B></P><P><I>L. innocua</I> contains a type II CRISPR system. A single gene (Cas9 / Csn1) has been shown to be necessary for the expression and inactivation stages of the pathway<SUP><A href="/Team:Arizona_State/Project/References#ref65" title="Team:Arizona State/Project/References">[65]</A></SUP>. A separate trans-encoded small RNA (tracrRNA) binds with the repeat segment of the pre-crRNA<SUP><A href="/Team:Arizona_State/Project/References#ref62" title="Team:Arizona State/Project/References">[62]</A></SUP>, followed by cleavage by RNase III and binding with Cas9.</P><DL><DD>Coordinates:
</DD><DD> CrisprI: 2768992-2769687
</DD><DD> Leader: 2769687-2769814
</DD><DD> Cas2 (lin2742): 2769814-2770404
</DD><DD> Cas1 (lin2743): 2770410-2770706
</DD><DD> Csn1 (lin2744): 2770707-2774711
</DD><DD> TracrRNA: 2774711-2774865
</DD></DL><H2><SPAN class="mw-headline" id="Stages_of_the_CRISPR_pathway"> Stages of the CRISPR pathway </SPAN></H2><P>There are 3 distinct stages of the CRISPR pathway: integration<SUP><A href="/Team:Arizona_State/Project/References#ref17" title="Team:Arizona State/Project/References">[17]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref54" title="Team:Arizona State/Project/References">[54]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref22" title="Team:Arizona State/Project/References">[22]</A></SUP>, expression, and adaptation.</P><H3><SPAN class="mw-headline" id="Integration_.2F_Adaptation"> Integration / Adaptation </SPAN></H3><P>In this step, DNA, commonly derived from phages and plasmids<SUP><A href="/Team:Arizona_State/Project/References#ref48" title="Team:Arizona State/Project/References">[48]</A></SUP>, is recognized and processed by Cas proteins. Information from outside of the genome is recognized and incorporated into the leader end of an existing array. This involves cas1 and cas2<SUP><A href="/Team:Arizona_State/Project/References#ref32" title="Team:Arizona State/Project/References">[32]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref58" title="Team:Arizona State/Project/References">[58]</A></SUP>. The integration stage is currently the least understood aspect of the pathway.</P><H3><SPAN class="mw-headline" id="Expression"> Expression </SPAN></H3><P>In the expression stage, the CRISPR array is transcribed in its entirety, yielding pre-crRNA. This pre-crRNA is cleaved at repeat regions<SUP><A href="/Team:Arizona_State/Project/References#ref73" title="Team:Arizona State/Project/References">[73]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref9" title="Team:Arizona State/Project/References">[9]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref27" title="Team:Arizona State/Project/References">[27]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref31" title="Team:Arizona State/Project/References">[31]</A></SUP> to yield crRNA. In <I>E. coli</I>, this crRNA is 61 bp long, consisting of a 31 bp spacer, flanked by repeat-derived segments on both ends<SUP><A href="/Team:Arizona_State/Project/References#ref63" title="Team:Arizona State/Project/References">[63]</A></SUP> (8 bp at 5'<SUP><A href="/Team:Arizona_State/Project/References#ref25" title="Team:Arizona State/Project/References">[25]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref28" title="Team:Arizona State/Project/References">[28]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref27" title="Team:Arizona State/Project/References">[27]</A></SUP>, 21 bp forming a hairpin at 3', with a 5' hydroxyl group). crRNA is then typically bound to a protein complex (known as Cascade in <I>E. coli</I><SUP><A href="/Team:Arizona_State/Project/References#ref63" title="Team:Arizona State/Project/References">[63]</A></SUP>).</P><H3><SPAN class="mw-headline" id="Interference"> Interference </SPAN></H3><P>This stage requires bound crRNA, as well as Cas3 in <I>E. coli</I><SUP><A href="/Team:Arizona_State/Project/References#ref63" title="Team:Arizona State/Project/References">[63]</A></SUP>. The interference stage targets DNA in most organisms<SUP><A href="/Team:Arizona_State/Project/References#ref25" title="Team:Arizona State/Project/References">[25]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref28" title="Team:Arizona State/Project/References">[28]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref54" title="Team:Arizona State/Project/References">[54]</A></SUP>, but RNA targeting has been demonstrated in the case of <I>P. furiosus</I><SUP><A href="/Team:Arizona_State/Project/References#ref35" title="Team:Arizona State/Project/References">[35]</A></SUP>. Recognition of target DNA is thought to take place by means of R-loops<SUP><A href="/Team:Arizona_State/Project/References#ref63" title="Team:Arizona State/Project/References">[63]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref73" title="Team:Arizona State/Project/References">[73]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref1" title="Team:Arizona State/Project/References">[1]</A></SUP>. An R-loop is an RNA strand that has base paired with a complementary DNA strand, displacing the other identical DNA strand<SUP><A href="/Team:Arizona_State/Project/References#ref1" title="Team:Arizona State/Project/References">[1]</A></SUP>. This base pairing between the crRNA spacer sequence and target strand may mark the region for interference by other proteins such as cas3<SUP><A href="/Team:Arizona_State/Project/References#ref63" title="Team:Arizona State/Project/References">[63]</A></SUP>.</P><P>In <I>Streptococcus thermophilus</I>, only Cas9 is necessary for CRISPR functionality<SUP><A href="/Team:Arizona_State/Project/References#ref74" title="Team:Arizona State/Project/References">[74]</A></SUP>. However, a specific sequence, known as a proto-adjacent-motif (PAM) was found to be required for interference. The predicted sequence is 5'-NGGNG-3'. This sequence is found several base pairs upstream of the proto-spacer (target DNA). Single base pair mutations in the PAM completely abolish CRISPR interference<SUP><A href="/Team:Arizona_State/Project/References#ref74" title="Team:Arizona State/Project/References">[74]</A></SUP>.</P><H2><SPAN class="mw-headline" id="Core_Cas_genes"> Core Cas genes </SPAN></H2><P>There are 6 “core” Cas genes, found in a wide variety of organisms and here referred to as Cas1-Cas6<SUP><A href="/Team:Arizona_State/Project/References#ref15" title="Team:Arizona State/Project/References">[15]</A></SUP>.</P><H3><SPAN class="mw-headline" id="Cas1.2C_Cas2"> Cas1, Cas2 </SPAN></H3><P>Cas1 is nearly universally conserved throughout organisms with CRISPR<SUP><A href="/Team:Arizona_State/Project/References#ref32" title="Team:Arizona State/Project/References">[32]</A></SUP>. It is strongly implicated in the integration stage of the pathway<SUP><A href="/Team:Arizona_State/Project/References#ref32" title="Team:Arizona State/Project/References">[32]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref58" title="Team:Arizona State/Project/References">[58]</A></SUP>. Cas1 is a metal-dependent (Mg, Mn) DNA-specific endonuclease that generates an 80 bp fragment<SUP><A href="/Team:Arizona_State/Project/References#ref32" title="Team:Arizona State/Project/References">[32]</A></SUP>. How this is converted into an ~32 bp spacer is unknown.</P><P>Cas2 is also involved in integration<SUP><A href="/Team:Arizona_State/Project/References#ref32" title="Team:Arizona State/Project/References">[32]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref58" title="Team:Arizona State/Project/References">[58]</A></SUP>, and is a metal dependent endoribonuclease<SUP><A href="/Team:Arizona_State/Project/References#ref24" title="Team:Arizona State/Project/References">[24]</A></SUP>.</P><H3><SPAN class="mw-headline" id="Cas3"> Cas3 </SPAN></H3><P>Cas3  is not regulated by H-NS<SUP><A href="/Team:Arizona_State/Project/References#ref41" title="Team:Arizona State/Project/References">[41]</A></SUP>. It cooperates with the Cascade complex<SUP><A href="/Team:Arizona_State/Project/References#ref25" title="Team:Arizona State/Project/References">[25]</A></SUP> in the interference stage. Cas3 has predicted ATP-dependent helicase activity<SUP><A href="/Team:Arizona_State/Project/References#ref4" title="Team:Arizona State/Project/References">[4]</A></SUP>, as well as demonstrated ATP independent annealing of RNA to DNA<SUP><A href="/Team:Arizona_State/Project/References#ref73" title="Team:Arizona State/Project/References">[73]</A></SUP>. It forms an R-loop with DNA, requiring magnesium or manganese as a co-factor<SUP><A href="/Team:Arizona_State/Project/References#ref73" title="Team:Arizona State/Project/References">[73]</A></SUP>, but has an antagonistic function in the presence of ATP, dissociating the R-loop.</P><H2><SPAN class="mw-headline" id="Prevention_of_self_targeting_.28autoimmunity.29"> Prevention of self targeting (autoimmunity) </SPAN></H2><P>The 5' handle of crRNA allows self / nonself discrimination in the csm subtype<SUP><A href="/Team:Arizona_State/Project/References#ref39" title="Team:Arizona State/Project/References">[39]</A></SUP>. In the Cse subtype, regions flanking the proto spacer contain PAMs<SUP><A href="/Team:Arizona_State/Project/References#ref39" title="Team:Arizona State/Project/References">[39]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref14" title="Team:Arizona State/Project/References">[14]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref22" title="Team:Arizona State/Project/References">[22]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref30" title="Team:Arizona State/Project/References">[30]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref21" title="Team:Arizona State/Project/References">[21]</A></SUP>, which may be necessary for interference. In general, it is thought that mismatches at positions outside of the spacer sequence allow for targeting, while extended base pairing with the surrounding repeats prevents targeting<SUP><A href="/Team:Arizona_State/Project/References#ref39" title="Team:Arizona State/Project/References">[39]</A></SUP>.</P><H2><SPAN class="mw-headline" id="CRISPR_regulation"> CRISPR regulation </SPAN></H2><P>In E. coli (Cse subtype), transcription of the Cascade genes and CRISPR array is repressed by H-NS<SUP><A href="/Team:Arizona_State/Project/References#ref48" title="Team:Arizona State/Project/References">[48]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref44" title="Team:Arizona State/Project/References">[44]</A></SUP>. H-NS is a global repressor of transcription in many gram negative bacteria that binds AT rich sequences<SUP><A href="/Team:Arizona_State/Project/References#ref16" title="Team:Arizona State/Project/References">[16]</A></SUP>. This repression is mediated by &quot;DNA stiffening&quot;<SUP><A href="/Team:Arizona_State/Project/References#ref37" title="Team:Arizona State/Project/References">[37]</A></SUP>, as well as formation of &quot;DNA-protein-DNA&quot; bridges<SUP><A href="/Team:Arizona_State/Project/References#ref11" title="Team:Arizona State/Project/References">[11]</A></SUP>. The creation of an H-NS knockout can be shown to increase expression of cas genes<SUP><A href="/Team:Arizona_State/Project/References#ref48" title="Team:Arizona State/Project/References">[48]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref5" title="Team:Arizona State/Project/References">[5]</A></SUP>. This correlates with phage sensitivity<SUP><A href="/Team:Arizona_State/Project/References#ref48" title="Team:Arizona State/Project/References">[48]</A></SUP>.</P><P>Transcription is antagonistically<SUP><A href="/Team:Arizona_State/Project/References#ref26" title="Team:Arizona State/Project/References">[26]</A></SUP> de-repressed by LeuO<SUP><A href="/Team:Arizona_State/Project/References#ref48" title="Team:Arizona State/Project/References">[48]</A></SUP>, a protein of the lysR transcription factor family<SUP><A href="/Team:Arizona_State/Project/References#ref26" title="Team:Arizona State/Project/References">[26]</A></SUP> near the leuABCD (leucine synthesis<SUP><A href="/Team:Arizona_State/Project/References#ref2" title="Team:Arizona State/Project/References">[2]</A></SUP>) operon<SUP><A href="/Team:Arizona_State/Project/References#ref13" title="Team:Arizona State/Project/References">[13]</A></SUP>. LeuO expression is also repressed by H-NS<SUP><A href="/Team:Arizona_State/Project/References#ref3" title="Team:Arizona State/Project/References">[3]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref6" title="Team:Arizona State/Project/References">[6]</A></SUP>. Expression of H-NS repressed proteins can be manipulated by plasmid-encoded leuO in a constitutive promoter<SUP><A href="/Team:Arizona_State/Project/References#ref33" title="Team:Arizona State/Project/References">[33]</A></SUP>. Plasmids: pCA24N (lac1 promoter), pKEDR13 (pTac promoter), pNH41 (IPTG). Increased LeuO expression leads to increased expression of casABCDE, cas1, and cas2<SUP><A href="/Team:Arizona_State/Project/References#ref48" title="Team:Arizona State/Project/References">[48]</A></SUP><SUP><A href="/Team:Arizona_State/Project/References#ref33" title="Team:Arizona State/Project/References">[33]</A></SUP>, but does not affect cas3 expression<SUP><A href="/Team:Arizona_State/Project/References#ref48" title="Team:Arizona State/Project/References">[48]</A></SUP>. Constitutively expressing leuO had a stronger affect than knocking out H-NS<SUP><A href="/Team:Arizona_State/Project/References#ref48" title="Team:Arizona State/Project/References">[48]</A></SUP>.</P><H2><SPAN class="mw-headline" id="Classification_of_CRISPR_systems"> Classification of CRISPR systems </SPAN></H2><P>For a comprehensive listing of Cas genes, see <A href="ftp://ftp.ncbi.nih.gov/pub/wolf/_suppl/CRISPRclass/crisprPro.html" class="external autonumber" rel="nofollow">[1]</A>.</P><P>Haft (2005) <SUP><A href="/Team:Arizona_State/Project/References#ref15" title="Team:Arizona State/Project/References">[15]</A></SUP>: Recognition of core Cas genes (1-6). Organized remaining genes into 9 subtypes: Ecoli, Ypest, Nmeni, Dvulg, Tneap, Hmari, Apern, Mtube, RAMP.</P><P>Makarova (2011) <SUP><A href="/Team:Arizona_State/Project/References#ref65" title="Team:Arizona State/Project/References">[65]</A></SUP>: Classification into I, II, and III subtypes, based on mechanism of action as well as homology. These subtypes correspond with the 9 given by Haft to a large extent:
<DL><DD><UL><LI> I-A: Apern (Csa)
</LI><LI> I-B: Tneap (Cst) / Hmari (Csh)
</LI><LI> I-C: Dvulg (Csd)
</LI><LI> I-D
</LI><LI> I-E: Ecoli (Cse)
</LI><LI> I-F: Ypest (Csy)
</LI><LI> II-A: Nmeni (Csn)
</LI><LI> II-B: Nmeni (Csn)
</LI><LI> III-A: Mtube (Csm)
</LI><LI> III-B: Polymerase-RAMP (Cmr)</LI></UL></DD></DL></P><H3><SPAN class="mw-headline" id="Type_I.2C_II_and_III_systems"> Type I, II and III systems </SPAN></H3><P>This classification takes into account differing mechanisms at all three stages of the pathway<SUP><A href="/Team:Arizona_State/Project/References#ref65" title="Team:Arizona State/Project/References">[65]</A></SUP>.</P><DL><DD><P> Integration: In type I and II systems, the integration of proto-spacers depends on a proto-adjacent-motif. Cas1 and Cas2 are involved in this stage in all three subtypes.</P></DD><DD><P> Expression: The CRISPR locus is transcribed into pre-crRNA. In type I systems, the Cascade complex binds to pre-crRNA, which is then cleaved by the Cas6e or Cas6f. Type II systems use a trans-encoded small RNA (tracrRNA) that binds with the repeat segment of pre-crRNA<SUP><A href="/Team:Arizona_State/Project/References#ref62" title="Team:Arizona State/Project/References">[62]</A></SUP>, followed by cleavage by RNase III with Cas9. Cas6 cleaves pre-crRNA in Type III systems. The crRNAs are then transferred to a distinct Cas complex (Csm in subtype III-A and Cmr in subtype III-B).</P></DD><DD><P>Interference: In type I systems, the Cascade complex is guided by the crRNA to the target strand. Cas3 then cleaves the DNA. In type II systems, Cas9 directly targets and cleaves the DNA without any additional proteins. Type I and type II systems both probably require a specific PAM for this stage. The Csm or Cmr proteins in type III systems also directly target DNA without additional proteins.</P></DD></DL><H2><SPAN class="mw-headline" id="CRISPR_resources"> CRISPR resources </SPAN></H2><UL><LI> The <A href="http://crispr.u-psud.fr/crispr/" class="external text" rel="nofollow">CRISPR database</A> analyzes archaeal and bacterial genomes for CRISPR arrays. The sequence of each locus can be displayed. This database is regularly updated.
</LI><LI> The <A href="http://cmr.jcvi.org/cgi-bin/CMR/shared/GenomePropDefinition.cgi?prop_acc=GenProp0021" class="external text" rel="nofollow">J. Craig Venter Institute</A> CMR genome properties database contains Cas gene information for several hundred genomes. The CMR database is currently without direct funding and is not being actively maintained.
</LI><LI><A href="http://crispi.genouest.org" class="external text" rel="nofollow">CRISPI: a CRISPR Interactive database</A> analyzes archaeal and bacterial genomes for CRISPR arrays and Cas genes. This database should be used with caution, as many of the purported repeats and spacers are several hundred base pairs, which is in conflict with the literature. Last updated 2008-11-04.
</LI><LI><A href="http://www.drive5.com/pilercr/" class="external text" rel="nofollow">PILER-CR</A>, a software tool for detecting CRISPR arrays.
</LI><LI><A href="http://www.room220.com/crt/" class="external text" rel="nofollow">CRISPR Recognition Tool</A>, another array recognition tool.</LI></UL></DIV><CENTER><DIV style="color: #000000; background-color: #EBEBEB"><DIV id="content-footer-box"><DIV class="footer-left-half"><DIV class="footer-left-col"><SMALL><P><B>Contact Us</B></P><P>Arizona State University</P><P>ECG 334, PO BOX 9709</P><P>Tempe, Arizona 85287</P><P><A href="mailto:asu.igem@gmail.com">Email</A></P></SMALL></DIV></DIV></DIV></DIV></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Arizona_State/Project/CRISPR">http://2011.igem.org/Team:Arizona_State/Project/CRISPR</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Arizona_State/Project/CRISPR" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Arizona_State/Project/CRISPR" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Arizona_State/Project/CRISPR&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Arizona_State/Project/CRISPR&amp;oldid=200760" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>