"
Page
Discussion
View source
History
teams
Log in
 
Team:Arizona State/Project/References
From 2011.igem.org
References
Home iGEM 2011 Home Project
Introduction
CRISPR
E. coli
B. halodurans
L. innocua
Software
Future
References
Lab
Team
Photos
Protocols
Safety
Acknowledgements
Results
Data
BioBricks
Notebook
April/May
June
July
August
September
Cas PCR Log
Sequence Information
Other Documents
Human Practices
Events
Collaborations
Exploring Synthetic Biology
Outreach in Practice
 
M. Thomas et al, “Hybridization of RNA to double-stranded DNA: formation of R-loops,” PNAS, vol. 73, no. 7, pp. 2294-2298, 1 July 1976.
NB Vartak et al, “A functional leuABCD operon is required for leucine synthesis by the tyrosine-repressible transaminase in Escherichia coli K-12,” J Bacteriol., vol. 173, no. 12, pp. 3864-3871, June 1991.
E. Klauck et al, “The LysR-like regulator LeuO in Escherichia coli is involved in the translational regulation of rpoS by affecting the expression of the small regulatory DsrA-RNA,” Mol. Microbio, vol. 25, no. 3, pp. 559-569, August 1997.
L. Aravind and EV Koonin, “The HD domain defines a new superfamily of metaldependent phosphohydrolases,” Trends Biochem Sci., vol. 23, no. 12, pp. 469-472, December 1998.
F. Hommais et al, “Large-scale monitoring of pleiotropic regulation of gene expression by the prokaryotic nucleoid-associated protein, H-NS,” Mol. Microbio, vol. 40, no. 1, pp. 20-36, April 2001.
A. Majumder et al, “LeuO expression in response to starvation for branched-chain amino acids,” J Biol Chem., vol. 276, no. 22, pp. 19046-19051, 1 June 2001.
K. S. Makarova et al, “A DNA repair system specific for thermophilic Archaea and bacteria predicted by genomic context analysis,” Nucl. Acids Res., vol. 30, no. 2, 1 November 2001.
R. Jansen et al, “Identification of genes that are associated with DNA repeats in prokaryotes,” Mol Microbiol., vol. 43, no. 6, March 2002.
T.H. Tang et al, “Identification of 86 candidates for small non-messenger RNAs from the archaeon  Archaeoglobus fulgidus,” PNAS, vol. 99, no. 11, pp. 7536-7541, 28 May 2002.
FJ Mojica et al, “Intervening sequences of regularly spaced prokaryotic repeats derive from foreign genetic elements,” J Mol Evol., vol. 60, no. 2, pp. 174-182, February 2005.
R. Dame et al, “DNA bridging: a property shared among H-NS-like proteins,” J. Bacteriol., vol. 187, no. 5, pp. 1845-1848, March 2005.
C. Pourcel et al, “CRISPR elements in Yersinia pestis acquire new repeats by preferential uptake of bacteriophage DNA, and provide additional tools for evolutionary studies,” Microbiology, vol. 151, no. 3, pp. 653-663, March 2005.
CC Chen and HY Wu, “LeuO protein delimits the transcriptionally active and repressive domains on the bacterial chromosome,” J Biol Chem., vol. 280, no. 15, pp. 15111-15112, 15 April 2005.
A. Bolotin et al, “Clustered regularly interspaced short palindrome repeats (CRISPRs) have spacers of extrachromosomal origin,” Microbiology, vol. 151, no. 8, pp. 2551-2561, August 2005.
D. Haft et al, “A Guild of 45 CRISPR-Associated (Cas) Protein Families and Multiple CRISPR/Cas Subtypes Exist in Prokaryotic Genomes,” PLoS Computational Biology, vol. 1, no. 6, pp. 474-483, 1 November 2005.
W. Navarre et al, “Selective silencing of foreign DNA with low GC content by the H-NS protein in Salmonella,” Science, vol. 313, no. 5784, pp. 236-238, 14 July 2006.
R. Barrangou et al, “CRISPR Provides Acquired Resistance Against Viruses in Prokaryotes,” Science, vol. 315, pp. 1709-1712, 23 March 2007.
V. Kunin et al, “Evolutionary conservation of sequence and secondary structures in CRISPR repeats,” Genome Biology, vol. 8, 18 April 2007.
V. Kunin et al, “Evolutionary conservation of sequence and secondary structures in CRISPR repeats,” Genome Biology, vol. 8, 18 April 2007.
I. Grissa et al, “The CRISPRdb database and tools to display CRISPRs and to generate dictionaries of spacers and repeat,” BMC Bioinformatics, vol. 8, no. 172, 23 May 2007.
H. Deveau et al, “Phage response to CRISPR-encoded resistance in Streptococcus thermophilus,” J. Bacteriol., vol. 190, no. 4, pp. 1390-1400, February 2008.
P. Horvath et al, “Diversity, activity, and evolution of CRISPR loci in Streptococcus thermophilus,” J. Bacteriol., vol. 190, no. 4, pp. 1401-1412, February 2008.
R. Sorek et al, “CRISPR — a widespread system that provides acquired resistance against phages in bacteria and archaea,” Nature Reviews Microbiology, no. 6, pp. 181-186, March 2008.
N. Beloglazova et al, “A novel family of sequence-speciﬁc endoribonucleases associated with the clustered regularly interspaced short palindromic repeats.,” J Biol Chem., vol. 283, no. 29, pp. 20361-20371, 18 July 2008.
S. Brouns et al, “Small CRISPR RNAs Guide Antiviral Defense in Prokaryotes,” Science, vol. 321, pp. 960-964, 15 August 2008.
D. Stoebel et al, “Antisilencing: overcoming H-NS-mediated repression of transcription in Gram-negative enteric bacteria.,” Microbiology, vol. 154, no. 9, pp. 2533-2545, September 2008.
C. Hale et al, “Prokaryotic silencing (psi)RNAs in Pyrococcus furiosus,” RNA, vol. 14, pp. 2572-2579, 29 October 2008.
S. Marraffini and E. Sontheimer, ““CRISPR Interference Limits Horizontal Gene Transfer in Staphylococci by Targeting DNA,” Science, no. 322, pp. 1843-1845, 19 December 2008.
J. Heidelberg et al, “Germ Warfare in a Microbial Mat Community: CRISPRs Provide Insights into the Co-Evolution of Host and Viral Genomes,” PloS ONE, vol. 4, no. 1, 9 January 2009.
F. J. M. Mojica et al, “Short motif sequences determine the targets of the prokaryotic CRISPR defence system,” Microbiology, vol. 155, no. 3, pp. 733-740, March 2009.
RK Lillestøl et al, “CRISPR families of the crenarchaeal genus Sulfolobus: bidirectional transcription and dynamic properties.,” Mol. Microbio, vol. 72, no. 1, pp. 259-272, April 2009.
B. Wiedenheft et al, “Structural basis for DNase activity of a conserved protein implicated in CRISPR-mediated genome defense,” Structure, vol. 17, pp. 904-912, 10 June 2009.
T. Shimada et al, “Involvement of the leucine response transcription factor LeuO in regulation of the genes for sulfa drug efflux,” J Bacteriol., vol. 191, no. 14, pp. 4562-4571, July 2009.
J. van der Oost et al, “CRISPR based adaptive and heritable immunity in prokaryotes,” Trends in Biochemical Sciences, vol. 34, no. 8, pp. 401-407, 30 July 2009.
C. Hale et al, “RNA-Guided RNA Cleavage by a CRISPR RNA-Cas Protein Complex,” Cell, vol. 139, pp. 945-956, 25 November 2009.
J. van der Oost and S. Brouns, “RnAi: Prokaryotes get in on the Act,” Cell, vol. 139, pp. 863-865, 25 November 2009.
Y. Liu et al, “A divalent switch drives H-NS/DNA-binding conformations between stiffening and bridging modes,” Genes & Dev., vol. 24, pp. 339-344, 18 December 2009.
P. Horvath and R. Barrangou, “CRISPR/Cas, the immune system of bacteria and Archaea.,” Science, vol. 327, no. 5962, pp. 167-170, 8 January 2010.
L. Marraffini and E. Sontheimer, “Self vs. non-self discrimination during CRISPR RNA-directed immunity,” Nature, vol. 463, pp. 568-571, 13 January 2010.
F. Karginov and G. Hannon, “The CRISPR system: small RNA-guided defense in bacteria and archaea,” Mol. Cell, vol. 37, no. 1, pp. 7-19, 15 January 2010.
Y. Agari et al, “Transcription profile of Thermus thermophilus CRISPR systems after phage infection.,” Journal of Molecular Biology, vol. 395, no. 2, pp. 270-281, 15 January 2010.
L. Marraffini and E. Sontheimer, “CRISPR interference: RNA-directed adaptive immunity in bacteria and archaea.,” Nature Reviews Genetics, vol. 11, pp. 181-190, 2 February 2010.
Ü. Pul et al, “Identiﬁcation and characterization of E. coli CRISPR-cas promoters and their silencing by H-NS,“ Mol. Microbio, vol. 75, no. 6, 17 February 2010.
U. Pul et al, “Identiﬁcation and characterization of E.coli CRISPR-cas promoters and their silencing by H-NS,” Mol. Microbio, vol. 75, no. 6, pp. 1495-1512, March 2010.
A. Stern et al, “Self-targeting by CRISPR: gene regulation or autoimmunity?,” Trends in Genetics, vol. 26, no. 8, pp. 335-340, 1 July 2010.
M. Aklujkar and D. Lovley, “Interference with histidyl-tRNA synthetase by a CRISPR spacer sequence as a factor in the evolution of Pelobacter carbinolicus,” BMC Evolutionary Biology, vol. 10, 28 July 2010.
J. He and M. Deen, “Heterogeneous diversity of spacers within CRISPR,” arXiv, 16 August 2010.
E. Westra et al, “H-NS-mediated repression of CRISPR-based immunity in Escherichia coli K12 can be relieved by the transcription activator LeuO,” Molecular Microbiology, vol. 77, no. 6, pp. 1380-1393, 18 August 2010.
N. Held et al, “CRISPR Associated Diversity within a Population of Sulfolobus islandicus,” PloS ONE, vol. 5, no. 9, 28 September 2010.
J. Carte et al, “Binding and cleavage of CRISPR RNA by Cas6,” RNA, 30 September 2010.
R. Edgar and U. Qimron, “The Escherichia coli CRISPR system protects from lambda lysogenisation, lysogens and prophage induction,” J. bact., vol. 192, pp. 6291-6294, 1 October 2010.
K. Palmer and M. Gilmore, “Multidrug-Resistant Enterococci Lack CRISPR-cas,” mBio, vol. 1, no. 4, 12 October 2010.
H. Deveau et a, “CRISPR/Cas System and Its Role in Phage-Bacteria Interactions,” Annu. Rev. Microbiol., 13 October 2010.
J. Garneau et al, “The CRISPR/Cas bacterial immune system cleaves bacteriophage and plasmid DNA,” Nature, vol. 468, pp. 67-71, 3 November 2010.
E. Sontheimer and L. Marraffini, “Slicer for DNA,” Nature, vol. 468, pp. 45-46, 4 November 2010.
D. Pride et al, “Analysis of streptococcal CRISPRs from human saliva reveals substantial sequence diversity within and between subjects over time,” Genome Research, vol. 21, pp. 126-136, 13 December 2010.
R. Perez-Rodriguez et al, “Envelope stress is a trigger of CRISPR RNA-mediated DNA silencing in Escheria Coli,” Mol. Microbio, vol. 79, pp. 584-599, 13 December 2010.
M. Babu et al, “A dual function of the CRISPR-Cas system in bacterial antivirus immunity and DNA repair,” Mol. Microbio, vol. 79, no. 2, pp. 484-502, January 2011.
K. Pougach et al, “Transcription, processing and function of CRISPR cassettes in Escherichia coli.,” Mol. Microbio, vol. 77, pp. 1367-1379, 1 January 2011.
R. Garrett et al, “CRISPR-based immune systems of the Sulfolobales: complexity and diversity,” Biochem. Soc. Trans., vol. 39, pp. 51-57, 19 January 2011.
F. Rezzonico et al, “Diversity, Evolution, and Functionality of Clustered Regularly Interspaced Short Palindromic Repeat (CRISPR) Regions in the Fire Blight Pathogen Erwinia amylovora,” Applied and Environmental Microbiology, vol. 77, no. 11, pp. 3819-3829, 24 March 2011.
E. Deltcheva et al, “CRISPR RNA maturation by trans-encoded small RNA and host factor RNase III,” Nature, vol. 471, no. 7340, pp. 602-607, 30 March 2011.
M. Jore et al, “Structural basis for CRISPR RNA-guided DNA recognition by Cascade,” Nature Structural & Molecular Biology, vol. 18, pp. 529-536, 3 April 2011.
T. Nozawa et al, “CRISPR Inhibition of Prophage Acquisition in Streptococcus pyogenes,” PLoS ONE, vol. 6, no. 5, 6 May 2011.
K. S. Makarova et al, “Evolution and classification of the CRISPR–Cas systems,” Nature Reviews Microbiology, vol. 9, pp. 467-477, 9 May 2011.
D. Sashital et al, “An RNA-induced conformational change required for CRISPR RNA cleavage by the endoribonuclease Cse3,” Nature Structural & Molecular Biology, vol. 18, pp. 680-687, 15 May 2011.
E. Gesner et al, “Recognition and maturation of effector RNAs in a CRISPR interference pathway,” Nature Structural & Molecular Biology, vol. 18, no. 6, pp. 688-692, 15 May 2011.
C. Skennerton et al, “Phage Encoded H-NS: A Potential Achilles Heel in the Bacterial Defence System,” PLoS ONE, vol. 6, no. 5, 18 May 2011.
W. Fricke et al, “Comparative Genomics of 28 Salmonella enterica Isolates: Evidence for CRISPR-mediated Adaptive Sublineage Evolution,” J. Bacteriology, 20 May 2011.
E. Semenova et al, “Interference by clustered regularly interspaced short palindromic repeat (CRISPR) RNA is governed by a seed sequence,” PNAS, 6 June 2011.
K. Phok el al, “Identification of CRISPR and riboswitch related RNAs among novel non-coding RNAs of the euryarchaeon Pyrococcus abyss,” BMC Genomics, 13 June 2011.
K. Kyun et al, “Crystal Structure of Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-associated Csn2 Protein Revealed Ca2+-dependent Double-stranded DNA-binding Activity,” J Biol Chem., 21 June 2011.
J. Howard et al, “Helicase dissociation and annealing of RNA-DNA hybrids by Escherichia coli Cas3 protein,” Biochem. J., 23 June 2011.
R. Sapranauskas et al, “The Streptococcus thermophilus CRISPR/Cas system provides immunity in Escherichia coli,” Nucl. Acids Res., 8 July 2011.
K. S. Makarova et al, “Unification of Cas protein families and a simple scenario for the origin and evolution of CRISPR-Cas systems,” Biol. Direct, vol. 6, no. 1, 14 July 2011.
B. Wiedenheft et al, “Structures of the RNA-guided surveillance complex from a bacterial immune system,” Nature, vol. 477, 22 September 2011.
T. Blomqvist et al, “Natural Genetic Transformation: a novel tool for efficient genetic engineering of the dairy bacterium Streptococcus thermophilus,” Applied and Environmental Microbiology, vol. 72, pp. 6751-6756, October 2006.
Salis et al, “Automated design of synthetic ribosome binding sites to control protein expression,” Nature Biotechnology, vol. 27, pp. 946-950, 4 October 2009.
Esvelt et al, “A system for the continuous directed evolution of biomolecules,” Nature, vol. 462, pp. 499-503, 11 February 2011.
Makarova et al, “A putative RNA-interference-based immune system in prokaryotes: computational analysis of the predicted enzymatic machinery, functional analogies with eukaryotic RNAi, and hypothetical mechanisms of action,” Biology Direct, vol. 1 no. 7, 16 March 2006.
Contact Us
Arizona State University
ECG 334, PO BOX 9709
Tempe, Arizona 85287
Email
Retrieved from " http://2011.igem.org/Team:Arizona_State/Project/References "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
